<html xmlns="http://www.w3.org/1999/xhtml"><head> <style type="text/css">
         	     		 
         	    a                          {background : #ffffff; }
         	    article                    {border-style : dotted; border-width : 2px; }
         		 	
         		 	div                        {background : #ffffcc;}
         		 	div.abstract-title         {font-weight : bold ; font-size : 16pt;}
         		 	div.ack                    {border-style : solid ; border-color : red; margin
         : 2em; }
         		 	div.article-meta           {border-style : solid; border-width : 1 pt; margin
         1em;}
         		 	div.boxed-text             {margin : 5em; border-style : solid;}
         		 	div.contrib-group          {margin : 1em; }
         		 	div.fig                    {border-style : solid; border-width : 2px; margin :
         2em; }
         		 	div.funding                {font-weight : bold ; font-size : 16pt;}
         		 	div.given-names            {font-style : italic;}
         		 	div.intro                  {border-style : inset; margin : 5px;}
         		 	div.introduction           {border-style : inset; margin : 5px;}
         		 	div.journal-meta           {border-style : solid; border-width : 1 pt; margin
         1em;}
         		 	div.journal-title-group    {background : #ffeeee;}
         		 	div.article-title          {font-weight : bold ; font-size : 18pt;}
         		 	div.kwd                    {font-style : italic}
         		 	div.meta-name              {font-weight : bold ; font-size : 16pt;}
         		 	div.name                   {font-weight : bold;}
         		 	div.alt-title              {font-style : italic; font-size : 12px;}
         		 	
         		 	div.sec                    {border : 2px; margin 5px; padding 2px;}
         		 	div.title                  {font-family : courier; font-weight : bold;
         		 	                            font-size : 16pt; margin : 5px;}
         		 	
         		 	div.materials_methods      {border-style : double; margin : 5px; }
         		 	div.methods                {border-style : double; margin : 5px; }
         		 	
         		 	div.results                {border-style : solid; margin : 5px;}
         		 	div.background             {border-style : dotted; margin : 5px;}
         		 	
         		 	div.discussion             {border-style : groove; margin : 5px;}
         		 	div.conclusion             {border-style : ridge; margin : 5px;}
         		 	div.conclusions             {border-style : ridge; margin : 5px;}
         		 	div.supplementary-material {border-style : inset; margin : 5px;}
         		 	div.abbreviations          {border-style : double; border-color : red; margin
         : 5px;}
         		 	div.competinginterests     {border-style : double; border-color : blue; margin
         : 5px;}
         		 	div.acknowledgements       {border-style : double; border-color : green; margin
         : 5px;}
         		 	div.authors_contributions   {border-style : double; border-color : purple; margin
         : 5px;}
         		 	
         		 	div.publisher              {border-style : outset; margin : 5px;}
         		 	div.fn-type-conflict       {background : #f88; }
         		 	div.fn-type-con            {background : #ddf; }
         		 	div.fn-type-other          {background : #ddd; }
         		 	
         		 	div.unknown                {background : #ffd;
         									 	  border-style : solid;
         									 	  border-width : 1px;
         									 	  padding : 2 px;}
         		 	  
         	    table                      {background : #ffffdd;}
         		 	tr                         {background : #ddddff; padding : 1px;}
         		 	
         		 	span                       {background : #ffcccc;}
         		 	
         		 	span.citation-author       {font-family : helvetica; background : #ffeeee;}
         		 	span.collab                {background : #ddffff; }
         		 	span.comment               {font-family : courier; font-size : 6px; background
         : #ffaaff;}
         		 	span.contrib               {background : #ffffff;}
         		 	span.corresp               {background : #ddffdd; }
         		 	span.doi                   {background : #ffffff;}
         		 	span.email                 {font-family : courier; }
         		 	span.etal                  {font-style : italic;}
         		 	span.fpage                 {font-family : courier;}
         		 	span.given-names           {background : #ffffff;}
         		 	span.iso-abbrev            {background : #ffffff;}
         		 	span.issn-epub             {background : #ffffff;}
         		 	span.issn-ppub             {background : #ffffff;}
         		 	span.journal-title         {background : #ffffff;}
         		 	span.lpage                 {font-family : courier;}
         		 	span.mixed-article-title   {font-style : italic ;}
         		 	span.nlm-ta                {background : #ffffff;}
         		 	span.pmc                   {background : #ffffff;}
         		 	span.pmcid                 {background : #ffffff;}
         		 	span.pmid                  {background : #ffffff;}
         		 	span.publisher             {background : #ffffff;}
         		 	span.publisher-id          {background : #ffffff;}
         		 	span.publisher-name        {background : #ffffff;}
         		 	span.source                {background : #ffffff;}
         		 	span.subject               {background : #ffffff;}
         		 	span.surname               {background : #ffffff;}
         		 	span.volume                {font-family : courier; font-weight : bold;}
         		 	span.year                  {font-family : courier ; font-style : italic;}
         			
      </style> </head> <body> <div class="front" title="front"> <div class="journal-meta" tagx="journal-meta" title="journal-meta"> <span class="nlm-ta" title="nlm-ta">BMC Biol</span> <span class="journal-title" tagx="journal-title" title="journal-title">BMC Biology</span> <span class="issn-epub" tagx="issn" title="issn-epub">1741-7007</span> <div class="publisher" tagx="publisher" title="publisher"> <span class="publisher-name" tagx="publisher-name" title="publisher-name">BioMed Central</span> <span class="publisher-loc" tagx="publisher-loc" title="publisher-loc">London</span> </div> </div> <div class="article-meta" tagx="article-meta" title="article-meta"> <span class="pmcid" title="pmcid"> pmcid: <a href="http://www.ncbi.nlm.nih.gov/pubmed/1570145">1570145</a> </span> <span class="publisher-id" title="publisher-id">1741-7007-4-31</span> <span class="pmid" title="pmid"> pmid: <a href="http://www.ncbi.nlm.nih.gov/pubmed/16956407">16956407</a> </span> <span class="doi" title="doi"> doi: <a href="https://doi.org/10.1186/1741-7007-4-31">10.1186/1741-7007-4-31</a> </span> <div class="article-categories" title="article-categories">
: <span class="subject" title="subject">Research Article</span> </div> <div class="title-group"> <div class="article-title" title="article-title">
An exceptional horizontal gene transfer in plastids: gene replacement by a distant bacterial paralog and evidence that haptophyte and cryptophyte plastids are sisters </div> </div> <div class="contrib-group" title="contrib-group"> <span class="contrib" title="contrib"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Rice</span> <span class="given-names" tagx="given-names" title="given-names">Danny W</span> </span> <a href="#I1">1</a> <span class="email" tagx="email" title="email">dwrice@indiana.edu</span> </span> <span class="contrib" title="contrib"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Palmer</span> <span class="given-names" tagx="given-names" title="given-names">Jeffrey D</span> </span> <a href="#I1">1</a> <span class="email" tagx="email" title="email">jpalmer@bio.indiana.edu</span> </span> </div> <span class="citation_author_institution" id="I1">[1], </span> <span class="pub-date-collection" title="pub-date-collection">collection: <span>2006</span> </span> <span class="pub-date-epub" title="pub-date-epub">epub: <span>2006-9-9</span> </span> <span class="volume" tagx="volume" title="volume">4</span> <span class="fpage" tagx="fpage" title="fpage">31</span> <span class="lpage" tagx="lpage" title="lpage">31</span> <a href="http://www.biomedcentral.com/1741-7007/4/31" /> <span class="history" title="history"> <span class="received" title="received">received: 2006-6-5</span> <span class="accepted" title="accepted">accepted: 2006-9-6</span> </span> <div class="permissions"> <span class="copyright" title="copyright">(C) <span class="copyright-holder" tagx="copyright-holder" title="copyright-holder">Rice and Palmer; licensee BioMed Central Ltd.</span>, <span class="copyright-year" tagx="copyright-year" title="copyright-year">2006</span> </span> <span class="license" title="license"> <p>This is an Open Access article distributed under the terms of the Creative Commons Attribution License ( <a href="http://creativecommons.org/licenses/by/2.0" />), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited. </p> </span> </div> <div class="abstract" title="abstract"> <div class="abstract-title" title="abstract-title">
Abstract</div> <div class="background" title="sec"> <div class="title" tagx="title" title="title">
Background</div> <p>Horizontal gene transfer (HGT) to the plant mitochondrial genome has recently been shown to occur at a surprisingly high rate; however, little evidence has been found for HGT to the plastid genome, despite extensive sequencing. In this study, we analyzed all genes from sequenced plastid genomes to unearth any neglected cases of HGT and to obtain a measure of the overall extent of HGT to the plastid. </p> </div> <div class="results" title="sec"> <div class="title" tagx="title" title="title">
Results</div> <p>Although several genes gave strongly supported conflicting trees under certain conditions, we are confident of HGT in only a single case beyond the rubisco HGT already reported. Most of the conflicts involved near neighbors connected by long branches (e.g. red algae and their secondary hosts), where phylogenetic methods are prone to mislead. However, three genes – <i>clpP</i>, <i>ycf2</i>, and <i>rpl36 </i>– provided strong support for taxa moving far from their organismal position. Further taxon sampling of <i>clpP </i>and <i>ycf2 </i>resulted in rejection of HGT due to long-branch attraction and a serious error in the published plastid genome sequence of <i>Oenothera elata</i>, respectively. A single new case, a bacterial <i>rpl36 </i>gene transferred into the ancestor of the cryptophyte and haptophyte plastids, appears to be a true HGT event. Interestingly, this <i>rpl36 </i>gene is a distantly related paralog of the <i>rpl36 </i>type found in other plastids and most eubacteria. Moreover, the transferred gene has physically replaced the native <i>rpl36 </i>gene, yet flanking genes and intergenic regions show no sign of HGT. This suggests that gene replacement somehow occurred by recombination at the very ends of <i>rpl36</i>, without the level and length of similarity normally expected to support recombination. </p> </div> <div class="conclusion" title="sec"> <div class="title" tagx="title" title="title">
Conclusion</div> <p>The <i>rpl36 </i>HGT discovered in this study is of considerable interest in terms of both molecular mechanism and phylogeny. The plastid acquisition of a bacterial <i>rpl36 </i>gene via HGT provides the first strong evidence for a sister-group relationship between haptophyte and cryptophyte plastids to the exclusion of heterokont and alveolate plastids. Moreover, the bacterial gene has replaced the native plastid <i>rpl36 </i>gene by an uncertain mechanism that appears inconsistent with existing models for the recombinational basis of gene conversion. </p> </div> </div> </div> </div> <div class="body" title="body"> <div class="background" title="sec"> <div class="title" tagx="title" title="title">
Background</div> <p>Unlike the dynamic mitochondrial genome of flowering plants, which frequently incorporates plastid and nuclear sequences via intracellular gene transfer [ <a href="#B1">1</a>- <a href="#B3">3</a>], the plastid genome is highly resistant to the uptake of intracellular DNA [ <a href="#B4">4</a>, <a href="#B5">5</a>]. Recently, a large number of discoveries of HGT involving mitochondrial genes of land plants have been reported [ <a href="#B6">6</a>- <a href="#B15">15</a>]. Most, if not all of these transfers seem to be the result of a gene being transferred from the mitochondrial genome of one species to that of another. No analogous case of plastid-to-plastid transfer has been reported, but these mitochondrial discoveries recommend a thorough assessment of plastid HGT. </p> <p>To date, only a single non-intron example of HGT to the plastid has been found. This is the ancient transfer of the rubisco operon ( <i>rbcL </i>and <i>rbcS</i>) from a proteobacterium into the common ancestor of red algal plastids and their secondary derivatives [ <a href="#B16">16</a>], a case that is revisited in this study. In contrast to transfers of constituent genes, acquisition of new introns may be relatively common in plastids [ <a href="#B17">17</a>- <a href="#B25">25</a>], based on their disparate phylogenetic distribution among plastid genomes, especially in green algae, and the fact that some introns are mobile elements. </p> <p>The evidence found thus far for HGT to the plastid proceeded from studies of a particular gene or intron. To quantify the overall extent of HGT in plastid genomes, we searched exhaustively for HGT among the 42 sequenced plastid genomes available when this study began. Our search relied primarily on phylogenetic analyses, but also involved scrutiny of each potential case (including generation of new gene sequences from phylogenetically relevant taxa) to rule out artifacts and various types of homoplasy. </p> </div> <div class="results" title="sec"> <div class="title" tagx="title" title="title">
Results</div> <p>Of the 204 protein genes present in four or more of the 42 examined plastid genomes, 34 produced maximum likelihood (ML) trees that had at least one node that conflicted (see Methods) with the reference plastid tree ( <a href="#S1">Additional File 1</a>), with bootstrap proportion (BP) ≥ 80%. Fifteen had conflicts with BP ≥ 90%, and 11 had conflicts with BP ≥ 95%. Thirteen of the genes with BP ≥ 80% involved rhodophyte/ <i>Odontella</i>/ <i>Guillardia </i>relationships, eight involved conflicts within the four grass taxa, and the rest were various other conflicts. In eight trees, multiple nodes had well-supported conflicts. Usually these pointed to a single rearrangement in the tree, but three trees had well-supported conflicts in different regions of the tree. </p> <p>After closer analysis, in some cases requiring the generation of additional sequences for key taxa, none of these conflicts proved to be strong cases of HGT. Ironically, one case that was not detected by this phylogenetic filter involves a very short gene that nonetheless offers strong support for a bacterium-to-plastid HGT. </p> <div class="hgtofrpl36" title="sec"> <div class="title" tagx="title" title="title">
HGT of rpl36</div> <p>The <i>Guillardia theta rpl36 </i>gene is very divergent from the <i>rpl36 </i>genes present in the surveyed plastid genomes and in cyanobacteria. In trees, it branches with a paralogous <i>rpl36 </i>type with strong support regardless of the phylogenetic method used (Figure <a href="#F1">1</a>). Here we refer to the type found in <i>Guillardia </i>as <i>rpl36</i>-c (for cryptophyte), and the type found in most plastids and most cyanobacteria as <i>rpl36</i>-p (for plastid). The 144-bp-long <i>Guillardia </i>gene shares, with all <i>rpl36</i>-c genes relative to <i>rpl36</i>-p, three indels (insertions of one and six amino acids, and a deletion of three amino acids), as well as an overall amino-acid and nucleotide similarity (Figure <a href="#F1">1</a> and <a href="#S2">Additional File 2</a>). <i>Guillardia rpl36</i>-c has a 7 amino-acid 3' extension present in 18 gamma-proteobacterial species, in the planctomycete <i>Rhodopirellula baltica</i>, and in the cyanobacterium <i>Crocosphaera watsonii </i>( <a href="#S2">Additional File 2</a>). The <i>rpl36 </i>HGT was not detected by our initial phylogenetic filter because our trees sampled only plastid-containing taxa, and this gene is too short to give strongly supported groupings within the plastids ( <a href="#S3">Additional File 3</a>). We detected this conflict only after building trees containing a broader sampling of <i>rpl36 </i>genes. </p> <p>In addition to the plastids and cyanobacteria, <i>rpl36</i>-p is found across many groups of bacteria, including diverse proteobacteria, and in fungal nuclear genes targeted to the mitochondrion. Most gamma-proteobacteria and a few beta-proteobacteria and actinobacteria contain both forms of the <i>rpl36 </i>gene (e.g. Figure <a href="#F1">1</a>). <i>Crocosphaera watsonii </i>has an <i>rpl36</i>-p with a frame-shift insertion near the 5' end, suggesting that it has been functionally replaced by a horizontally transferred <i>rpl36</i>-c. </p> <p>The <i>rpl36 </i>gene is located between <i>secY </i>and <i>rps13 </i>in all six sequenced plastid genomes from red algae and their secondary photosynthetic derivatives, including <i>Guillardia theta </i>( <a href="#S2">Additional File 2</a>). This is within a larger syntenic group of 22 genes conserved in the red algal plastids and diverse bacterial lineages. None of the <i>rpl36</i>-c genes in bacteria are adjacent to <i>secY </i>or <i>rps13</i>, nor are any located within the larger syntenic region. </p> <p>To identify the approximate time/phylogenetic boundary of transfer and to confirm the validity of the <i>Guillardia </i>gene, we sequenced <i>rpl36 </i>from three additional, diverse [ <a href="#B26">26</a>] cryptophytes: <i>Hanusia phi</i>, <i>Chroomonas mesostigmatica</i>, and <i>Cryptomonas tetrapyrenoidosa</i>. Using PCR, we isolated only <i>rpl36</i>-c from all three cryptophytes (and only <i>rpl36</i>-c was found in the unpublished plastid genome sequence of the cryptophyte <i>Rhodomonas salina </i>CCMP1319; H. Khan and J. Archibald, personal communication). These genes possess high sequence similarity (nucleotide identity between 79% and 96%) to <i>rpl36 </i>of <i>Guillardia theta</i>. We obtained high-quality sequence for a region comprising all of <i>rpl36</i>, both of its flanking spacers, 219 bp at the 3' end of <i>secY</i>, and 300 bp at the 5' end of <i>rps13 </i>(Figure <a href="#F1">1</a> and <a href="#S2">Additional File 2</a>). </p> <p>The plastid genome of the haptophyte <i>Emiliania huxleyi</i>, which was sequenced too recently [ <a href="#B27">27</a>] to be included in this study, also contains <i>rpl36</i>-c in place of <i>rpl36</i>-p (Figure <a href="#F1">1</a> and <a href="#S2">Additional File 2</a>). <i>Emiliania rpl36 </i>shares the c-type indels and 3' extension with the cryptophyte <i>rpl36 </i>genes and contains no additional indels over its entire length. Its amino-acid identities to the cryptophyte <i>rpl36</i>-c genes range from 85 to 90%, and its nucleotide identities range from 72 to 79%. It too is located between <i>secY </i>and <i>rps13</i>, with 5' and 3' intergenic spacers of length 139 bp and 14 bp, respectively. The <i>Emiliania </i>sequence groups as sister to the cryptophyte <i>rpl36</i>-c genes with good support (Figure <a href="#F1">1</a>). In addition, an EST sequence <a href="http://tbestdb.bcm.umontreal.ca" /> from the dinoflagellate <i>Karlodinium micrum</i>, which possesses a tertiary plastid of haptophyte origin [ <a href="#B28">28</a>- <a href="#B30">30</a>], also contains the <i>rpl36</i>-c gene. Furthermore, the <i>Karlodinium </i>sequence is sister to the <i>Emiliania </i>sequence in phylogenetic analyses (data not shown, but see <a href="#S2">Additional File 2</a>). </p> </div> <div class="theclppconflict" title="sec"> <div class="title" tagx="title" title="title">
The <i>clpP </i>conflict </div> <p>In what follows, we describe and discuss genes that initially gave conflicting trees with relatively high bootstrap support, but which for various reasons were either strongly rejected or brought into question as potential horizontal transfers. </p> <p>The <i>clpP </i>gene from <i>Oenothera elata </i>is highly divergent. With the taxon sampling used in this study ( <a href="#S1">Additional File 1</a>), it branches, with 84% BP (see Figure <a href="#F2">2</a> legend), as the sister to the grasses, which are also a long-branched group (Figure <a href="#F2">2A</a>). Suspecting long-branch attraction (LBA), we obtained more genes (provided by L. Goertzen and C. Long) from the order Myrtales, to which <i>Oenothera </i>belongs (including <i>Clarkia</i>, <i>Fuschia</i>, <i>Eucalyptus</i>, <i>Punica</i>, <i>Callistemon</i>, and <i>Oenothera organensis</i>) and from other non-grass monocots (including <i>Acorus </i>and <i>Flagellaria</i>) to test if the grouping with the grasses was an artifact. The resulting tree (Figure <a href="#F2">2B</a>) strongly suggests that the original result was an LBA artifact, as <i>Oenothera </i>goes within the Myrtales (and within its family Onagraceae) with the better sampling. </p> </div> <div class="theycf2conflict" title="sec"> <div class="title" tagx="title" title="title">
The <i>ycf2 </i>conflict </div> <p>The published <i>Oenothera elata ycf2 </i>gene branched as sister to the asterids <i>Atropa </i>and <i>Nicotiana </i>with BP of 100%, instead of with other rosid sequences (e.g. <i>Arabidopsis </i>and <i>Lotus</i>). To verify this, we sequenced <i>ycf2 </i>from a number of diverse Myrtales including <i>O. elata </i>itself. This new <i>Oenothera </i>sequence did not match the published <i>O. elata </i>plastid genome sequence [ <a href="#B31">31</a>], several regions of which (up to 1.5 kb in length) have 100% sequence identity with the <i>Nicotiana ycf2 </i>gene (Figure <a href="#F3">3A</a>). These cover regions that have long insertions in our <i>O. elata </i>sequence but that are missing in the <i>O. elata </i>genome sequence. This latter sequence also contains insertions shared with <i>Nicotiana </i>but not with our <i>O. elata ycf2 </i>sequence. Regions in the published sequence that do match our sequence appear to have single base errors, given that our <i>O. biennis </i>sequence is more similar to our <i>O. elata </i>sequence than is the published sequence in these regions (Figure <a href="#F3">3A</a>). Although very divergent, the new <i>O. elata </i>sequence branches in the expected position with other Myrtales (Figure <a href="#F3">3B</a>). We strongly suspect that this conflict is due to extensive error in the <i>ycf2 </i>region of the published <i>O. elata </i>genome [ <a href="#B31">31</a>], perhaps due to inadvertent incorporation of <i>Nicotiana </i>sequence during genome assembly (see <a href="#S4">Additional File 4</a>). </p> </div> <div class="therhodophyte/chromalveolateconflicts" title="sec"> <div class="title" tagx="title" title="title">
The rhodophyte/chromalveolate conflicts</div> <p>The chromalveolates [ <i>Guillardia </i>(cryptophytes), <i>Odontella </i>(heterokonts), and apicomplexans in our sampling] are a putatively monophyletic group that, with respect to plastid phylogeny, branch within the red algae, with the Cyanidiales being sister to the <i>Porphyra</i>/ <i>Gracilaria</i>/chromalveolate clade ( <a href="#S1">Additional File 1</a>). Relationships among the different chromalveolate groups are not well established [ <a href="#B32">32</a>], but the consensus topology provides a reasonable working hypothesis. Most of the conflicts that we found relative to this topology involve <i>Guillardia </i>and/or <i>Odontella </i>branching as sister to all the rhodophytes instead of as sister to <i>Porphyra</i>/ <i>Gracilaria</i>. Although some of these conflicts could in principle be true cases of HGT, the combination of long branches and near-neighbor exchanges makes these conflicts suspect, even given high bootstrap support. For some of these we have seen evidence (assuming our plastid tree is correct) of codon-usage bias. For example, the <i>psbB </i>gene tree goes from 100% BP for rhodophytes being monophyletic to the exclusion of <i>Guillardia </i>and <i>Odontella</i>, using all three codon positions, to weak support for a topological change to the consensus tree when only second positions are used. First and second positions together still give strong support for the conflicting tree, indicating that first positions may also contribute a significant bias for <i>psbB </i>[ <a href="#B33">33</a>]. </p> <p>Another well-supported conflict, psbA, is discussed in <a href="#S5">Additional File 5</a>. The remaining genes that had conflicts supported by BP of 80% or higher are <i>psbC</i>, <i>atpH</i>, <i>psaK</i>, <i>dnaK</i>, <i>atpF</i>, <i>atpB</i>, <i>rpl31</i>, <i>ycf4</i>, <i>ycf17</i>, <i>ycf45</i>, and <i>ycf37</i>. As above, we could not reject the conflicts outright, but we could show weakened support or induce topological changes with alternative data filtering such as using second codon positions alone or amino acids. In no case did we observe any telltale signals such as uniquely shared indels in the conflicting clades. </p> </div> <div class="thegrassconflicts" title="sec"> <div class="title" tagx="title" title="title">
The grass conflicts</div> <p>Eight gene trees conflicted with the consensus tree ( <a href="#S1">Additional File 1</a>) with respect to relationships among the four grasses examined. Four gene trees ( <i>atpI</i>, <i>psbH</i>, <i>atpF</i>, <i>rpl16</i>) supported the monophyly of <i>Triticum</i>/ <i>Saccharum</i>/ <i>Zea</i>, while the other four ( <i>rpl22</i>, <i>ndhD</i>, <i>psaA</i>, <i>psbA</i>) supported monophyly of <i>Oryza</i>/ <i>Saccharum</i>/ <i>Zea</i>. In all cases there is a long branch leading to the grasses, which reflects both the lack of any close outgroups (no other monocots were included) and the well-established rapid evolution of the chloroplast genome in the stem group leading to grasses (Stefanovic et al [ <a href="#B34">34</a>] and references therein). We hypothesize that all eight gene trees whose within-grass topologies conflicted with the consensus topology reflect spurious results stemming from the lack of close outgroups to the grass sequences. To test this hypothesis, we reanalyzed the two genes ( <i>atpI </i>and <i>psbH</i>) that showed the highest level of conflict (BP = 96%), using all other monocot sequences available (from <i>Sorghum, Hordeum, Phyllostachys, Typha, Yucca, Phalaenopsis, and Acorus</i>) for these two genes. With this improved sample, the <i>atpI </i>phylogeny was no longer in significant conflict with the organismal tree; instead, relationships among <i>Triticum</i>, <i>Oryza</i>, and <i>Saccharum/Zea </i>were entirely unresolved under an all nucleotide position ML model ( <a href="#S6">Additional File 6</a>). For <i>psbH</i>, however, the situation did not change markedly; we obtained a BP of 91% for <i>Oryza </i>being sister to <i>Triticum</i>/ <i>Saccharum</i>/ <i>Zea</i>, but this corresponds to only three parsimony informative characters. Such a small number of informative characters could easily be homoplasious, and better taxon sampling is required to resolve these conflicts firmly. </p> </div> <div class="otherconflicts" title="sec"> <div class="title" tagx="title" title="title">
Other conflicts</div> <p>The remaining conflicts with BP ≥ 80% were all brought into question using alternative phylogenetic analyses that led to reduction of bootstrap support or topology change. All of these involved conflicts with branches that are near each other in the consensus tree. </p> </div> <div class="rrnaandtrnagenes" title="sec"> <div class="title" tagx="title" title="title">
rRNA and tRNA genes</div> <p>The small and large subunit ribosomal RNA genes had strong support for the euglenids going outside the green algae (sister to red/green algae for 16S and within the red algae for 23S). Increasing taxon sampling using other available rRNA genes [ <a href="#B35">35</a>] gave a weakly supported placement of the euglenids within the green algae. No clear cases of HGT in tRNA genes were detected, but interpretation of these alignments and trees is difficult, owing to the short length and extensive paralogy of these genes. </p> </div> <div class="scrutinyoflong-branchedlineages" title="sec"> <div class="title" tagx="title" title="title">
Scrutiny of long-branched lineages</div> <p>The approach used here is limited by the taxon sampling. Because our initial trees included only plastid genes, we could essentially detect HGT only from one plastid genome to another, but not transfers from other genomes into plastids. We reasoned that transfers of non-plastid genes should normally result in a long branch leading to the donee taxon within the plastid gene trees. A description of our analysis of these long-branch lineages is presented in <a href="#S7">Additional File 7</a>. No additional cases of HGT were detected in this analysis. </p> </div> </div> <div class="discussion" title="sec"> <div class="title" tagx="title" title="title">
Discussion</div> <div class="therpl36transfer:thechromalveolatehypothesisandalgalphylogeny" title="sec"> <div class="title" tagx="title" title="title">
The <i>rpl36 </i>transfer: the chromalveolate hypothesis and algal phylogeny </div> <p>The unique, derived presence of the horizontally transferred <i>rpl36</i>-c gene in haptophyte and cryptophyte plastids, but not in heterokont and alveolate plastids, provides the first strong evidence for the "sisterhood" of haptophyte and cryptophyte plastids. The most parsimonious scenario is that the <i>rpl36</i>-c gene was transferred once to the ancestral plastid of the haptophytes and cryptophytes after this plastid lineage and the lineage(s) leading to heterokonts and alveolates diverged. Less parsimonious alternative scenarios can be imagined, but the similarity between the haptophyte and cryptophyte <i>rpl36 </i>genes, their position as sister lineages among the <i>rpl36</i>-c genes (Figure <a href="#F1">1</a>), and the fact that the transfer appears to have occurred via an improbable recombination event make alternative explanations unlikely. </p> <p>The haptophytes and heterokonts have been recognized as sister groups based on ultrastructural and pigment similarities [ <a href="#B36">36</a>, <a href="#B37">37</a>], and named 'chromobiotes'. In addition, phylogenies based on concatenated plastid genes tend to group the haptophytes and heterokonts [ <a href="#B38">38</a>, <a href="#B39">39</a>] (see <a href="#S8">Additional File 8</a> for further discussion of chromalveolate phylogeny). However, on a per-gene basis, the signal is mixed, and nearly half the plastid genes actually group haptophytes and cryptophytes as sisters ( <a href="#S9">Additional File 9</a> and <a href="#S10">Additional File 10</a>). The morphological characters linking haptophytes to heterokonts could all be ancestral to the chromophytes (chromobiotes plus cryptophytes) or even the chromalveolates (chromophytes plus alveolates) [ <a href="#B32">32</a>, <a href="#B40">40</a>]) and lost differentially. For example, chlorophyll c3 and autofluorescence of the rear cilium could have been lost in the cryptophytes, and the nucleomorph (present in cryptophytes) could have been lost independently in the haptophytes and heterokonts (it is well established that the nucleomorph has been independently lost in the secondary, green-derived plastid of euglenids). In contrast, the presence of the c-type <i>rpl36 </i>in only haptophytes and cryptophytes cannot be explained by differential loss unless one posits its unlikely insertion via HGT immediately adjacent to, rather than in place of, the ancestral p-type gene. It remains to be seen whether the hypothesis of haptophyte/cryptophyte plastid monophyly is supported or rejected by future phylogenetic analyses involving many more plastid and nuclear genes and/or taxa from across the chromalveolates. One possibility, which is based on the serial symbiosis models developed by Bachvaroff et al [ <a href="#B38">38</a>, <a href="#B39">39</a>], is that the cryptophyte and haptophyte plastids, but not their nuclear lineages, will turn out to be sister groups. This would be the case if, say, the cryptophyte plastid was of secondary, red-algal origin and the haptophyte plastid of tertiary, cryptophyte origin. However, a study using six nuclear cytosolic protein genes did group haptophytes and cryptophytes with weak support [ <a href="#B41">41</a>], suggesting that their nuclear genes may also be monophyletic. </p> </div> <div class="thedonoroftherpl36-cgene" title="sec"> <div class="title" tagx="title" title="title">
The donor of the <i>rpl36</i>-c gene </div> <p> <i>rpl36</i>-c was probably transferred to the ancestral plastid of haptophytes and cryptophytes directly from a bacterium rather than from the mitochondrion or nucleus, as there is no evidence of a <i>rpl36</i>-c in these compartments. At the amino-acid level, plastid <i>rpl36</i>-c is most similar to <i>Rhodopirellula baltica</i>. Interestingly, the complete shotgun sequences from two other planctomycetes ( <i>Blastopirellula marina </i>and <i>Gemmata obscuriglobus</i>) contain only <i>rpl36</i>-p. Thus, a potential transfer between the <i>Rhodopirellula </i>lineage and the cryptophyte lineage would likely postdate the <i>Rhodopirellula</i>/ <i>Blastopirellula</i>/ <i>Gemmata </i>divergence. However, a recent HGT from an unknown donor into <i>Rhodopirellula </i>is also possible. On balance, based on the current bacterial sampling, the donor of the plastid <i>rpl36</i>-c was most likely a planctomycete related to <i>Rhodopirellula </i>or a proteobacterium. A cyanobacterium related to <i>Crocosphaera watsonii </i>is a less likely but potential donor since the <i>Crocosphaera </i>branches within the gamma-proteobacterial c-type group (tree not shown; but see <a href="#S2">Additional File 2</a>), but was probably recently acquired from this group via HGT (see Results). </p> </div> <div class="therpl36transfer:mechanismandfunctionalconsequence" title="sec"> <div class="title" tagx="title" title="title">
The <i>rpl36 </i>transfer: mechanism and functional consequence </div> <p>Because plastid <i>rpl36</i>-c and <i>rpl36</i>-p are both located between <i>secY </i>and <i>rps13 </i>in the same orientation, we suspect that the <i>rpl36 </i>HGT was mediated by homologous recombination. This would be extraordinary, because the <i>Guillardia </i>and <i>Porphyra rpl36 </i>genes are only 49% identical in nucleotide sequence in non-gap regions. At this level of divergence, homologous recombination is thought to be highly unlikely [ <a href="#B42">42</a>, <a href="#B43">43</a>]. It is implausible that flanking sequence could have been used to initiate gene conversion, as intergenic regions between distant taxa are essentially random, and no bacterial c-type <i>rpl36 </i>genes are flanked by <i>secY </i>and <i>rps13</i>. Additionally, the 3' end of <i>secY </i>and the 5' end of <i>rps13 </i>in <i>Guillardia </i>do not appear to have been replaced by divergent sequence, as they are still highly similar to the red algal and cyanobacterial genes relative to <i>Rhodopirellula </i>and proteobacteria ( <a href="#S2">Additional File 2A</a>). Even the last 30 bases of <i>Guillardia secY </i>have a higher sequence identity to red algal and cyanobacterial homologs than to all other known sequences. As the 3' end of the <i>rps13 </i>first and second position alignment is iteratively removed, <i>Guillardia </i>continues to group with the red algae and cyanobacteria until about 40 bp are left, at which point phylogenetic resolution is lost, owing to relatively high sequence conservation. There is no significant sequence similarity between the <i>Guillardia rpl36 </i>intergenic regions and those from any available c-type-containing bacteria. In fact, there is only modest conservation among the intergenic regions of the additional cryptophyte genomes we examined. The cryptophyte intergenic spacers 3' to <i>rpl36 </i>ranged in length from 42 to 53 bp with sequence identities, in non-gap regions, of 68–80%, while the 5' spacers ranged in length from 30 to 150 bp with sequence identities of 59–74%. </p> <p>This leads to the hypothesis that recombination may have been initiated by very short regions of conservation between the <i>rpl36</i>-c and <i>rpl36</i>-p genes themselves. Most reported recombination events between bacterial species tend to be among highly similar sequences [ <a href="#B44">44</a>]. However, this may not be entirely due to the level of sequence similarity but also to interspecies barriers, such as mismatch repair [ <a href="#B45">45</a>]. The minimal sequence identity required to initiate recombination varies depending on the system and species being tested, but has been shown to involve 20 or fewer consecutive identical nucleotides for some types of recombination [ <a href="#B46">46</a>, <a href="#B47">47</a>]. </p> <p>Although the overall similarity between <i>rpl36</i>-c and <i>rpl36</i>-p is very low (Figure <a href="#F1">1</a>), the 5' and 3' ends are more conserved than the rest of the gene. Specifically, the plastid <i>rpl36</i>-c genes share an 8-bp 5' sequence (AGTAAAGT) with all <i>rpl36</i>-p genes in the red algal lineage, as well as with several green algae and land plants and with several <i>rpl36</i>-p and <i>rpl36</i>-c bacterial genes. A less well-conserved 6-bp 3' sequence (CAAGGT) exists in cryptophyte and some alpha-proteobacterial <i>rpl36</i>-c genes and in the <i>rpl36</i>-p gene from some cyanobacteria, land plants, green algae, and red algae. This identity extends leftwards by a further 3 bp (CGTCAAGGT) in <i>Cryptomonas</i>, some cyanobacteria, and <i>Odontella</i>. These similarities between the plastidal and bacterial <i>rpl36</i>-c and the <i>rpl36</i>-p genes in the red-plastid lineage are consistent with one or both ends being involved in recombination. A gene replacement along these lines would represent, to our knowledge, an unprecedented recombination event in terms of sequence distance. Although it is conceivable that the <i>rpl36</i>-c and <i>rpl36</i>-p genes involved in this putative recombination some 1 billion years ago shared more sequence similarity than the extant genes, <i>rpl36</i>-c has not diverged greatly among haptophytes and cryptophytes, and <i>rpl36</i>-p is quite conserved among plastids (Figure <a href="#F1">1</a>). </p> <p>This being the case, possible alternatives to recombination, and HGT itself, should be considered, such as convergent evolution. However, even though <i>rpl36 </i>is very short, convergence is unlikely, given the sequence divergence between the p- and c-type <i>rpl36 </i>genes and that the algal c-type genes emerge as a nested clade from within the larger group of c-type genes (Figure <a href="#F1">1</a>). The convergence hypothesis would require a staggering and unprecedented number of convergent events. At the amino-acid level, the <i>Guillardia rpl36 </i>shares 36 identities with <i>Rhodopirellula</i>, but only 13–18 identities with its red-algal relatives. It also shares three gaps and a 7 amino-acid 3' extension with bacterial <i>rpl36</i>-c (Figure <a href="#F1">1</a> and <a href="#S2">Additional File 2</a>). Although functional convergence does occur in protein genes [ <a href="#B48">48</a>], nothing approaching the extent that must be invoked for <i>rpl36 </i>has been shown. Protein functional convergence usually occurs at very short, key areas of the protein; for example, within active site regions of an enzyme. </p> <p>In contrast to convergent positive selection for function, GC content can have a dramatic effect on amino-acid and codon usage [ <a href="#B49">49</a>]. However, plastids, including <i>Guillardia </i>and <i>Emiliania</i>, have a low genomic GC content relative to the bacterial genomes of the taxa shown in Figure <a href="#F1">1</a>. The <i>rpl36</i>-c genes mirror the genomic GC content. For example, <i>Guillardia rpl36</i>-c is 31% GC (genome is 33%) while <i>Rhodopirellula rpl36</i>-c is 54% (genome is 55%). In contrast to convergence, these differences in GC content probably account for much of the divergence between the plastid and bacterial <i>rpl36</i>-c genes and make chance convergence much less likely. </p> <p> <i>rpl36</i>-c almost certainly physically replaced <i>rpl36</i>-p in certain algae; but did it also functionally replace it? The two types are highly divergent from one another (Figure <a href="#F1">1</a> and <a href="#S2">Additional File 2</a>), and to our knowledge, <i>rpl36</i>-c has never been shown to play the equivalent ribosomal function in any organism. It is therefore conceivable that a nuclear-encoded, plastid-targeted <i>rpl36</i>-p exists in the cryptophytes and haptophytes, while <i>rpl36</i>-c serves some other function. However, upon consideration of the crystal structure of the 50S ribosomal subunit from <i>E. coli </i>[ <a href="#B50">50</a>] it is plausible that <i>rpl36</i>-c could functionally replace <i>rpl36</i>-p. First, the amino acids making van der Waals and hydrogen bond contacts with the 23S ribosomal RNA are fairly well conserved between the two <i>rpl36 </i>types ( <a href="#S2">Additional File 2</a>). Second, the region where the 3 amino-acid insertion exists in the c-type makes no intermolecular contacts, but instead protrudes into the solvent where an insertion is unlikely to cause a functional problem. Interestingly, this insertion creates potential N-myristoylation and N-glycosylation sites that could have functional importance. Third, the crystal structure reveals a large empty space, in contact with the C-terminal glycine, that could easily accommodate the 7 amino-acid C-terminal extension in <i>rpl36</i>-c. Thus, it is reasonable to expect that <i>rpl36</i>-c could functionally replace <i>rpl36</i>-p without any major steric interference or loss of intermolecular contacts. In addition, <i>rpl36</i>-c is highly conserved between the haptophyte and cryptophytes as would be expected for a functional ribosomal protein. </p> </div> <div class="rubiscohgtrevisited" title="sec"> <div class="title" tagx="title" title="title">
Rubisco HGT revisited</div> <p>It was first recognized nearly 20 years ago [ <a href="#B51">51</a>- <a href="#B53">53</a>] that red algae and their secondary symbiotic derivatives possess a rubisco operon ( <i>rbcLS</i>) of highly unusual evolutionary origin. Whereas all green plastids and those of glaucophytes contain <i>rbcLS </i>of expected cyanobacterial origin, red plastids possess rubisco genes of apparent proteobacterial origin. Based on phylogenetic considerations, Delwiche and Palmer [ <a href="#B16">16</a>] argued in 1996 that the red algal rubisco was acquired from proteobacteria by horizontal transfer in the common ancestor of all red algae. In addition, they provided evidence for several other <i>rbcLS </i>transfers, all among eubacteria. Martin and Schnarrenberger [ <a href="#B53">53</a>] argued that the cyanobacterial endosymbiont instead carried both the red-like and green-like rubisco genes from a duplication predating cyanobacteria and proteobacteria, and that differential loss in the plastid lineages and loss in all cyanobacterial lineages resulted in the observed pattern. </p> <p>In the context of the current study, and with the passage of some 10 years and the accumulation of hundreds of bacterial genome sequences, we revisit this issue. Figure <a href="#F4">4</a> shows an <i>rbcL </i>phylogeny for a representative sampling of currently available sequences. The overall structure of this tree is very similar to that of Figure <a href="#F2">2</a> of the paper by Delwiche and Palmer [ <a href="#B16">16</a>]. Importantly, however, a number of new proteobacterial <i>rbcL </i>sequences have become available that show even greater similarity to red algal <i>rbcL </i>than those considered by that study [ <a href="#B16">16</a>]. For example, the recently sequenced genome of <i>Nitrosospira multiformis </i>shares 86% amino-acid identity with <i>Gracilaria </i>over a contiguous 381 amino-acid region of <i>rbcL</i>. This is within the range of identities among the red algae over this same region. The <i>rbcL </i>tree (Figure <a href="#F4">4</a>) groups the red algal <i>rbcL </i>genes with those of <i>Nitrosospira </i>and <i>Nitrosococcus</i>. With the advent of these and other related bacterial sequences, the red algal <i>rbcL </i>clade is now two steps nested within the overall clade of red-like proteobacterial <i>rbcL </i>sequences, whereas previously [ <a href="#B16">16</a>] it was simply sister to a more limited set of proteobacteria. </p> <p>Now, with many more rubisco sequences in hand, and with complete genomes available for many of these organisms, the duplication/loss model strongly conflicts with the phylogenetic and presence/absence data. None of the 15 or more sequenced cyanobacterial genomes contains a red-like <i>rbcL</i>. Furthermore, out of the many bacterial <i>rbcL </i>sequences now available, only a single organism, <i>Rhodobacter azotoformans</i>, has been found that contains both red-like and green-like <i>rbcL </i>[ <a href="#B54">54</a>], and this is clearly due to a bacterial HGT event instead of retention of both copies from an ancient duplication. So the hypothesis of an ancient duplication and differential loss of paralogs is becoming increasingly untenable. </p> <p>Alternatively, instead of a transfer to the recent ancestor of the red algae, a recent ancestor of the cyanobacterial endosymbiont could have received a red-like <i>rbcL </i>from a proteobacterium followed by differential losses in the plastid lineages [ <a href="#B55">55</a>]. This possibility is less parsimonious, however, as it still requires one horizontal transfer, plus at least two independent losses in the plastid lineage and at least one in cyanobacteria. In conclusion, it is likely that the <i>rbcLS </i>operon of red algae represents a genuine HGT event to the plastid genome. </p> </div> <div class="hgtinplastids:rarebutchoice" title="sec"> <div class="title" tagx="title" title="title">
HGT in plastids: rare but choice</div> <p>Comprehensive examination of all 204 genes present in four or more of the 42 examined plastid genomes has revealed but a single new, well-supported case of HGT. This <i>rpl36 </i>transfer and the <i>rbcL</i>S transfer described years ago [ <a href="#B16">16</a>] and revisited above share several features: (i) they both involve bacterial donors; (ii) they are both relatively ancient [ <a href="#B56">56</a>- <a href="#B58">58</a>], having occurred in the common ancestor of red algae ( <i>rbcLS</i>), perhaps 1.0–1.5 billion years ago, or in the common ancestor of cryptophytes and haptophytes ( <i>rpl36</i>), probably not much more recently; and (iii) in both cases, the transferred genes are known ( <i>rbcLS</i>; see Delwiche and Palmer [ <a href="#B16">16</a>, <a href="#B51">51</a>- <a href="#B53">53</a>], Valentin and Zetche [ <a href="#B16">16</a>, <a href="#B51">51</a>- <a href="#B53">53</a>], Boczar et al [ <a href="#B16">16</a>, <a href="#B51">51</a>- <a href="#B53">53</a>], Martin and Schnarrenberger [ <a href="#B16">16</a>, <a href="#B51">51</a>- <a href="#B53">53</a>], and references therein) or thought ( <i>rpl36</i>) to have functionally replaced native homologs, which would explain their retention for eons as intact genes. The <i>rpl36 </i>transfer also serves as an important phylogenetic marker and is intriguing from a mechanistic standpoint. Thus, although HGT in plastids is extremely rare, when it happens it can be of considerable consequence and interest. </p> <p>Both cases of plastid horizontal gene transfer occurred anciently in red algae or their secondary derivatives, while several cases of potential [ <a href="#B22">22</a>] or likely [ <a href="#B20">20</a>, <a href="#B21">21</a>] horizontal acquisition of introns are evident in green algae plastids. In contrast, no cases of HGT were evident in our analyses of sequenced plastid genomes of land plants, nor has HGT been reported for any of the many plastid genes that have been widely sequenced (in hundreds to thousands of plants) for phylogenetic purposes. This contrast is noteworthy because far more plastid sequencing has been performed in land plants (99671 entries from an NCBI Entrez search for plastid genes) than in algae (6731 entries). Are algal plastids, an admittedly paraphyletic group, somehow more amenable to HGT than plant plastids? </p> </div> <div class="hgtandgettingtherighttree" title="sec"> <div class="title" tagx="title" title="title">
HGT and getting the right tree</div> <p>We initially constructed many phylogenetic trees that gave well-supported, conflicting results suggestive of HGT, but which were ultimately deemed wrong or showed weakened support under closer scrutiny. The largest source of conflicts arose within the red algal lineage and their secondary descendants (and to a lesser extent with the green algae), where limited taxon sampling and early diversification of lineages led to a series of long terminal branches connected by short internal branches. This is where phylogenetic reconstruction is most prone to fail due to LBA [ <a href="#B59">59</a>]. Subsequent analyses with improved taxon sampling and/or filtering of fast-evolving codon positions caused us to reject all of these cases as potential HGTs. Some of these conflicts might still represent actual HGT events, but further taxon sampling will be required to resolve the issue completely. Sequencing errors should be considered a possibility and the anomalous sequences verified where appropriate. </p> </div> <div class="whyishgtsolesscommoninplastidsthaninmitochondriainlandplants?" title="sec"> <div class="title" tagx="title" title="title">
Why is HGT so less common in plastids than in mitochondria in land plants?</div> <p>Some 40 cases of HGT have now been reported in plant mitochondrial genomes [ <a href="#B6">6</a>- <a href="#B15">15</a>] versus none in plant plastids. This is despite far less sequencing of plant mitochondrial genes (7075 Genbank entries) than plastid genes (99671 Genbank entries). Similarly, plant mitochondrial genomes are rich in plastid and nuclear sequences acquired via intracellular gene transfer [ <a href="#B1">1</a>- <a href="#B3">3</a>], whereas plastid genomes entirely lack such sequences [ <a href="#B4">4</a>, <a href="#B5">5</a>]. What could account for such differences? To be sure, plant mitochondrial genomes are less compact (72–89% noncoding DNA in sequenced angiosperm genomes) and less constrained in size (varying over 10-fold in size). Nevertheless, angiosperm plastids contain considerable noncoding DNA (generally 40–45%), suggesting that the greater propensity for mitochondrial HGT is not simply a function of the total amount of "junk" DNA. Rather, the differences may be how efficiently the organelles take up exogenous DNA. Plant mitochondria possess an active DNA uptake system [ <a href="#B2">2</a>]; no similar activity has been reported for plastids, but it is also unclear whether this has been assayed for. This uptake system may lower a rate-limiting barrier in the incorporation of both foreign and native DNA. A major, well-documented difference between the two organelles is the tendency of mitochondria to fuse. This may account for some of the observed mitochondrion-to-mitochondrion HGT. Plant mitochondria regularly fuse [ <a href="#B60">60</a>, <a href="#B61">61</a>], promoting recombination between parental mitochondrial genomes in the case of somatic hybrid plants generated by protoplast fusion, whereas chloroplasts virtually never fuse under similar conditions [ <a href="#B62">62</a>, <a href="#B63">63</a>]. </p> </div> </div> <div class="conclusion" title="sec"> <div class="title" tagx="title" title="title">
Conclusion</div> <p>This study confirms and quantifies the hypothesis that HGT is rare in plastids. Only <i>rpl36 </i>and the rubisco operon are clear cases of HGT to the plastid genome. Both are ancient transfers, whereby bacterial genes have replaced native homologs and have become permanent, functional residents in their respective lineages. In contrast, the frequent (and recent) transfers in plant mitochondria occur by plant-to-plant transfer and are essentially ephemeral events, few of which seem to be of functional significance. The horizontal transfer of bacterial <i>rpl36</i>-c into the plastid genome represents an unprecedented example of apparent homologous recombination that defies current concepts of the sequence relatedness required to allow gene conversion/replacement to occur. The <i>rpl36</i>-c HGT also serves as a striking phylogenetic character that establishes an important new phylogenetic connection, linking haptophyte and cryptophyte plastids as sister groups to the exclusion of heterokont and alveolate plastids. </p> </div> <div class="methods" title="methods"> <div class="title" tagx="title" title="title">
Methods</div> <div class="plastidgenomes" title="sec"> <div class="title" tagx="title" title="title">
Plastid genomes</div> <p>EMBL-Bank files for the following 40 plastid genomes were retrieved from the European Bioinformatics Institute: <i>Eimeria tenella </i>( <a href="AY217738">AY217738</a>), <i>Euglena gracilis </i>( <a href="X70810">X70810</a>), <i>Euglena longa </i>( <a href="AJ294725">AJ294725</a>), <i>Guillardia theta </i>( <a href="AF041468">AF041468</a>), <i>Toxoplasma gondii </i>( <a href="U87145">U87145</a>), <i>Cyanophora paradoxa </i>( <a href="U30821">U30821</a>), <i>Cyanidioschyzon merolae </i>( <a href="AB002583">AB002583</a>), <i>Cyanidium caldarium </i>( <a href="AF022186">AF022186</a>), <i>Gracilaria tenuistipitata </i>( <a href="AY673996">AY673996</a>), <i>Porphyra purpurea </i>( <a href="U38804">U38804</a>), <i>Odontella sinensis </i>( <a href="Z67753">Z67753</a>), <i>Adiantum capillus </i>( <a href="AY178864">AY178864</a>), <i>Amborella trichopoda </i>( <a href="AJ506156">AJ506156</a>), <i>Anthoceros formosae </i>( <a href="AB086179">AB086179</a>), <i>Arabidopsis thaliana </i>( <a href="AP000423">AP000423</a>), <i>Atropa belladonna </i>( <a href="AJ316582">AJ316582</a>), <i>Calycanthus floridus </i>( <a href="AJ428413">AJ428413</a>), <i>Chaetosphaeridium globosum </i>( <a href="AF494278">AF494278</a>), <i>Chlamydomonas reinhardtii </i>( <a href="BK000554">BK000554</a>), <i>Chlorella vulgaris </i>( <a href="AB001684">AB001684</a>), <i>Epifagus virginiana </i>( <a href="M81884">M81884</a>), <i>Lotus japonicus </i>( <a href="AP002983">AP002983</a>), <i>Marchantia polymorpha </i>( <a href="X04465">X04465</a>), <i>Medicago truncatula </i>( <a href="AC093544">AC093544</a>), <i>Mesostigma viride </i>( <a href="AF166114">AF166114</a>), <i>Nephroselmis olivacea </i>( <a href="AF137379">AF137379</a>), <i>Nicotiana tabacum </i>( <a href="Z00044">Z00044</a>), <i>Nymphaea alba </i>( <a href="AJ627251">AJ627251</a>), <i>Oenothera elata </i>( <a href="AJ271079">AJ271079</a>), <i>Oryza sativa </i>( <a href="X15901">X15901</a>), <i>Physcomitrella patens </i>( <a href="AP005672">AP005672</a>), <i>Pinus koraiensis </i>( <a href="AY228468">AY228468</a>), <i>Pinus thunbergii </i>( <a href="D17510">D17510</a>), <i>Psilotum nudum </i>( <a href="AP004638">AP004638</a>), <i>Saccharum officinarum </i>( <a href="AP006714">AP006714</a>), <i>Spinacia oleracea </i>( <a href="AJ400848">AJ400848</a>), <i>Triticum aestivum </i>( <a href="AB042240">AB042240</a>), <i>Zea mays </i>( <a href="X86563">X86563</a>), <i>Plasmodium falciparum </i>( <a href="X95275">X95275</a>, <a href="X95276">X95276</a>), <i>Panax ginseng </i>( <a href="AY582139">AY582139</a>). In addition, the sequence of the <i>Pisum sativum </i>plastid genome was provided by John C. Gray (unpublished data), and the <i>Vitis vinifera </i>coding sequences were extracted and pieced together (unpublished result) from all NCBI nucleotide databases including dbEST [ <a href="#B64">64</a>]. A combination of BLAST searches with closely related genomes, Perl scripts for parsing output, and hand editing was used to define the protein and RNA genes in the unannotated genomes of <i>Pisum sativum</i>, <i>Vitis vinifera</i>, and <i>Medicago truncatula</i>. </p> </div> <div class="geneclustering" title="sec"> <div class="title" tagx="title" title="title">
Gene clustering</div> <p>In total, 5676 protein, tRNA and rRNA genes were extracted from the 42 plastid genome sequences. A BLAST [ <a href="#B65">65</a>] database was created with these DNA sequences, and then each sequence was used as a BLAST query against the database. From the BLAST output, a pairwise distance matrix was constructed based on the best BLAST expectation value for each query/hit pair. For a pair to be considered, the BLAST expectation value had to be ≤0.1, and at least 20% of the longer sequence of a pair had to be included in the alignment. Pairs for which these criteria were not met received a large distance value of 1.1. A huge neighbor-joining tree was constructed with PAUP* software, [ <a href="#B66">66</a>] using this distance matrix. Gene families and superfamilies were easily identifiable in the resulting tree. Unrelated gene families formed a polytomy of long branches at the root node. From this tree, 204 protein gene families containing four or more genes were hand selected by visual inspection. Ribosomal RNA genes were easily resolved in the tree, but transfer RNA genes were clustered into many hard-to-resolve paralogous families. The distinct tRNA clusters were separated into groups for further clustering using maximum parsimony (MP) and ML analyses. </p> </div> <div class="genealignment" title="sec"> <div class="title" tagx="title" title="title">
Gene alignment</div> <p>Protein and nucleotide alignments were made for each of the gene families using MUSCLE software [ <a href="#B67">67</a>] with unlimited iterations, and were inspected manually to correct errors. Initially, amino-acid alignments were constructed for sequences whose translation could be obtained. The protein alignment was then back-translated to nucleotides using the known nucleotide sequences. Sequences that could not be translated (such as pseudogenes and RNA genes) were aligned based on nucleotides. Positions containing mostly gaps, especially where homology was deemed questionable, were excluded from phylogenetic analyses. </p> </div> <div class="phylogeneticanalyses" title="sec"> <div class="title" tagx="title" title="title">
Phylogenetic analyses</div> <p>ML models for each gene family were determined using the likelihood-ratio test criterion of MODELTEST [ <a href="#B68">68</a>] except where specified. Final model parameter values were estimated by iteratively building ML trees and recalculating parameter values until the best trees converged. Heuristic ML searches using tree bisection-reconnection branch swapping in PAUP* were performed to find these trees. All three codon positions were included for the initial phylogenetic screening of gene families, but first and second position and other character-sampling strategies and software were used secondarily if needed to clarify the phylogenetic support for a conflicting tree. One hundred ML bootstrap replicates were performed using the same model and search method as used for searches for the best trees. Neighbor-joining and MP analyses were also carried out to allow for comparison to the ML results. Additional phylogenetic methods and programs (e.g. MrBayes, PAML and PROML) are indicated where used. </p> </div> <div class="phylogeneticconflictevaluation" title="sec"> <div class="title" tagx="title" title="title">
Phylogenetic conflict evaluation</div> <p>A consensus plastid tree ( <a href="#S1">Additional File 1</a>) was used as the working hypothesis topology for finding conflicts in gene trees. This tree was compiled from the current literature and our unpublished work using entire genomes. Nodes in a gene tree that conflicted with the plastid tree by addition of a taxon not part of that clade or subtraction of a taxon that is part of that clade were marked as in conflict. Conflicting nodes were ordered by their ML BP values, using a PERL script developed for finding these conflicts. Trees were viewed graphically, with conflicting clades highlighted to determine whether further processing was necessary. To rule out HGT in well-supported conflicts, we scrutinized the alignment in more detail, increased taxon sampling, and tried other models and phylogenetic methods. </p> </div> <div class="newgenesequences" title="sec"> <div class="title" tagx="title" title="title">
New gene sequences</div> <p>Several new sequences for <i>rpl36 </i>( <a href="DQ365944">DQ365944</a>– <a href="DQ365946">DQ365946</a>) and <i>ycf2 </i>( <a href="DQ370441">DQ370441</a>– <a href="DQ370447">DQ370447</a>) were obtained using standard PCR and sequencing protocols [ <a href="#B69">69</a>]. Cryptophyte genomic DNAs (for <i>rpl36 </i>isolation) were generously provided by Chris Lane and John Archibald. Angiosperm DNAs (for <i>ycf2 </i>isolation) were isolated [ <a href="#B70">70</a>] directly from young plant leaves or were taken from lab stocks. </p> </div> </div> <div class="authors'contributions" title="sec"> <div class="title" tagx="title" title="title">
Authors' contributions</div> <p>DWR carried out all the analyses, wrote the new software used in this study, generated the new <i>rpl36 </i>and <i>ycf2 </i>sequence data, and drafted the manuscript (text and figures). DWR and JDP jointly directed the project, and JDP contributed substantially to the final manuscript. Both authors read and approved the final manuscript. </p> </div> <div class="supplementary-material" title="supplementary-material"> <div class="title" tagx="title" title="title">
Supplementary Material</div> <div class="local-data" title="local-data">  <div class="title" tagx="title" title="title">
Additional file 1</div> <p>Consensus plastid phylogeny. Shown is the plastid phylogeny used in this study, based on the current literature. Dashed and solid vertical brackets denote paraphyletic and monophyletic groups, respectively. Although the tree is shown as entirely resolved, some parts are not well supported (e.g., relationships among <i>Cyanophora</i>, green algae, and red algae; among bryophytes; among chromalveolates; and whether <i>Amborella </i>plus <i>Nymphaea </i>or <i>Amborella </i>alone is the sister of all other angiosperms). </p>  <div class="media" title="media"> <a href="1741-7007-4-31-S1.pdf">LINK</a>  <p>Click here for file</p>  </div> </div> <div class="local-data" title="local-data">  <div class="title" tagx="title" title="title">
Additional file 2</div> <p>Amino-acid alignment of <i>secY</i>, <i>rpl36 </i>and <i>rps13 </i>and the rp <i>l</i>36 amino-acid alignment showing the C-terminal extensions and atomic contacts in the ribosome crystal structure. <b>(A) </b>This shows that <i>secY </i>and <i>rps13 </i>from plastids which contain the c-type <i>rpl36 </i>are more similar to <i>secY </i>and <i>rps13 </i>genes from red algae and cyanobacteria than to potential <i>rpl36 </i>donors, <i>Rhodopirellula </i>and proteobacteria. Amino acids that conflict with the consensus amino acid at each position are colored according to the key. The similarity between the c-type <i>rpl36 </i>genes in the haptophyte and cryptophyte plastids and those from <i>Rhodopirellula </i>and proteobacteria is also apparent. Note that <i>rpl36 </i>is flanked by <i>secY </i>and <i>rps13 </i>in plastids of red algal origin, but the c-type <i>rpl36 </i>in bacteria is not flanked by these genes. <b>(B) </b>Amino-acid alignment of c-type and p-type <i>rpl36 </i>genes. Note the three apicomplexans included in the alignment ( <i>Plasmodium</i>, <i>Toxoplasma</i>, and <i>Theileria</i>). At the bottom is shown which residues make contact with the 23S rRNA in the ribosome crystal structure of <i>Escherichia coli</i>, which has a p-type <i>rpl36</i>. </p>  <div class="media" title="media"> <a href="1741-7007-4-31-S2.pdf">LINK</a>  <p>Click here for file</p>  </div> </div> <div class="local-data" title="local-data">  <div class="title" tagx="title" title="title">
Additional file 3</div> <p>The poorly resolved <i>rpl36 </i>tree obtained in the initial phylogenetic screen. This shows the <i>Guillardia rpl36 </i>gene going arbitrarily with <i>Pisum </i>with weak support given the poor taxon sampling and low information content of this gene. See Figure <a href="#F1">1</a> for clarification. </p>  <div class="media" title="media"> <a href="1741-7007-4-31-S3.pdf">LINK</a>  <p>Click here for file</p>  </div> </div> <div class="local-data" title="local-data">  <div class="title" tagx="title" title="title">
Additional file 4</div> <p>Discussion of the <i>Oenothera elata ycf2 </i>gene error in the genome sequence. Discussion of the <i>Oenothera elata ycf2 </i>gene error in the genome sequence. </p>  <div class="media" title="media"> <a href="1741-7007-4-31-S4.pdf">LINK</a>  <p>Click here for file</p>  </div> </div> <div class="local-data" title="local-data">  <div class="title" tagx="title" title="title">
Additional file 5</div> <p>Results of the <i>psbA </i>conflict in the red algal lineage. Results of the <i>psbA </i>conflict in the red algal lineage. </p>  <div class="media" title="media"> <a href="1741-7007-4-31-S5.pdf">LINK</a>  <p>Click here for file</p>  </div> </div> <div class="local-data" title="local-data">  <div class="title" tagx="title" title="title">
Additional file 6</div> <p>Resolution of conflicts within grasses by adding taxa for the <i>atpI </i>gene. <b>(A) </b>The topology and support values with the original taxon sampling. The conflicting node is indicated by the bold 96; <b>(B) </b>after more monocot taxa are added <i>Triticum </i>and <i>Zea </i>move apart. </p>  <div class="media" title="media"> <a href="1741-7007-4-31-S6.pdf">LINK</a>  <p>Click here for file</p>  </div> </div> <div class="local-data" title="local-data">  <div class="title" tagx="title" title="title">
Additional file 7</div> <p>List of gene trees having branches ≥ 4 SD above the mean branch length for a given tree and text describing analysis of long branch taxa. The text describes the analysis of long-branch taxa. The table gives the number of standard deviations above the mean, the corresponding gene and the group that the long-branch leads to. </p>  <div class="media" title="media"> <a href="1741-7007-4-31-S7.pdf">LINK</a>  <p>Click here for file</p>  </div> </div> <div class="local-data" title="local-data">  <div class="title" tagx="title" title="title">
Additional file 8</div> <p>Discussion of chromalveolate hypothesis and algal phylogeny. Discussion of chromalveolate hypothesis and algal phylogeny. </p>  <div class="media" title="media"> <a href="1741-7007-4-31-S8.pdf">LINK</a>  <p>Click here for file</p>  </div> </div> <div class="local-data" title="local-data">  <div class="title" tagx="title" title="title">
Additional file 9</div> <p>Multigene chromist plastid phylogeny. Results of the phylogenetic placement analysis of the haptophyte <i>Emiliania </i>based on all plastid genes. </p>  <div class="media" title="media"> <a href="1741-7007-4-31-S9.pdf">LINK</a>  <p>Click here for file</p>  </div> </div> <div class="local-data" title="local-data">  <div class="title" tagx="title" title="title">
Additional file 10</div> <p>Plastid genes supporting the sisterhood of either haptophytes and heterokonts or haptophytes and cryptophytes. The histograms list the genes favoring the given topology over the other according to the codeml amino-acid ML score with the WAG matrix [ <a href="#B71">71</a>] and gamma distributed rates. The site-specific likelihood values were calculated using parameter estimates and branch lengths based on the concatenated alignment with all the genes. The height of the histogram bars represent the log likelihood preference for the given topology over the other for a particular gene (i.e. the sum of site likelihoods in the concatenated alignment corresponding to a particular gene). <b>(A) </b>The topology found using MrBayes with 21,659 plastid amino-acid positions shared among the chromophytes, <i>Gracilaria</i>, and <i>Porphyra</i>. Invariant sites and gamma distributed rates were used with the Cprev model [ <a href="#B72">72</a>]. The posterior probabilities were 1.0 for all nodes. <b>(B) </b>The topology was found as in <b>(A) </b>except that <i>Guillardia </i>and <i>Emiliania </i>were constrained to be monophyletic. The node corresponding to the chromist clade had a posterior probability of 0.98 and the other nodes had 1.0. </p>  <div class="media" title="media"> <a href="1741-7007-4-31-S10.pdf">LINK</a>  <p>Click here for file</p>  </div> </div> </div> </div> <div class="back" title="back"> <div class="ack" title="ack"> <div class="acknowledgements" title="sec"> <div class="title" tagx="title" title="title">
Acknowledgements</div> <p>We thank Christa Long and Les Goertzen for contributing unpublished <i>clpP </i>sequences for use in this study; Chris Lane, Hameed Khan, and John Archibald for providing several cryptophyte DNA samples and unpublished information about the Rhodomonas plastid genome; John Gray for providing the unpublished sequence of <i>Pisum sativum </i>plastid genome; and Virginia Sanchez-Puerta for helpful suggestions regarding the manuscript. This work was supported by NIH research grants GM-35087 and GM-70612 (to J.D.P.). </p> </div> </div> <div class="references">
References</div> <div tag="ref-list"> <ul> <li tag="ref"> <a name="B1" /> <div class="citation" tagx="citation" title="citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Burger</span> <span class="given-names" tagx="given-names" title="given-names">G</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Gray</span> <span class="given-names" tagx="given-names" title="given-names">MW</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Lang</span> <span class="given-names" tagx="given-names" title="given-names">BF</span> </span> </span> <div class="article-title" title="article-title">
Mitochondrial genomes: anything goes</div> <span class="source" tagx="source" title="source">Trends Genet</span> <span class="year" tagx="year" title="year">2003</span> <span class="volume" tagx="volume" title="volume">19</span> <span class="fpage" tagx="fpage" title="fpage">709</span> <span class="lpage" tagx="lpage" title="lpage">716</span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/14642752">14642752</a> </span> <span class="pub-id"> <a href="https://doi.org/10.1016/j.tig.2003.10.012">10.1016/j.tig.2003.10.012</a> </span> </div> </li> <li tag="ref"> <a name="B2" /> <div class="citation" tagx="citation" title="citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Koulintchenko</span> <span class="given-names" tagx="given-names" title="given-names">M</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Konstantinov</span> <span class="given-names" tagx="given-names" title="given-names">Y</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Dietrich</span> <span class="given-names" tagx="given-names" title="given-names">A</span> </span> </span> <div class="article-title" title="article-title">
Plant mitochondria actively import DNA via the permeability transition pore complex</div> <span class="source" tagx="source" title="source">EMBO J</span> <span class="year" tagx="year" title="year">2003</span> <span class="volume" tagx="volume" title="volume">22</span> <span class="fpage" tagx="fpage" title="fpage">1245</span> <span class="lpage" tagx="lpage" title="lpage">1254</span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/12628917">12628917</a> </span> <span class="pub-id"> <a href="https://doi.org/10.1093/emboj/cdg128">10.1093/emboj/cdg128</a> </span> </div> </li> <li tag="ref"> <a name="B3" /> <div class="citation" tagx="citation" title="citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Knoop</span> <span class="given-names" tagx="given-names" title="given-names">V</span> </span> </span> <div class="article-title" title="article-title">
The mitochondrial DNA of land plants: peculiarities in phylogenetic perspective</div> <span class="source" tagx="source" title="source">Curr Genet</span> <span class="year" tagx="year" title="year">2004</span> <span class="volume" tagx="volume" title="volume">46</span> <span class="fpage" tagx="fpage" title="fpage">123</span> <span class="lpage" tagx="lpage" title="lpage">139</span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/15300404">15300404</a> </span> <span class="pub-id"> <a href="https://doi.org/10.1007/s00294-004-0522-8">10.1007/s00294-004-0522-8</a> </span> </div> </li> <li tag="ref"> <a name="B4" /> <div class="citation" tagx="citation" title="citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Palmer</span> <span class="given-names" tagx="given-names" title="given-names">JD</span> </span> </span> <div class="article-title" title="article-title">
Contrasting modes and tempos of genome evolution in land plantorganelles</div> <span class="source" tagx="source" title="source">Trends Genet</span> <span class="year" tagx="year" title="year">1990</span> <span class="volume" tagx="volume" title="volume">6</span> <span class="fpage" tagx="fpage" title="fpage">115</span> <span class="lpage" tagx="lpage" title="lpage">120</span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/2132730">2132730</a> </span> <span class="pub-id"> <a href="https://doi.org/10.1016/0168-9525(90)90125-P">10.1016/0168-9525(90)90125-P</a> </span> </div> </li> <li tag="ref"> <a name="B5" /> <div class="citation" tagx="citation" title="citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Lemieux</span> <span class="given-names" tagx="given-names" title="given-names">C</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Otis</span> <span class="given-names" tagx="given-names" title="given-names">C</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Turmel</span> <span class="given-names" tagx="given-names" title="given-names">M</span> </span> </span> <div class="article-title" title="article-title">
Ancestral chloroplast genome in <i>Mesostigma viride </i>reveals an early branch of green plant evolution </div> <span class="source" tagx="source" title="source">Nature</span> <span class="year" tagx="year" title="year">2000</span> <span class="volume" tagx="volume" title="volume">403</span> <span class="fpage" tagx="fpage" title="fpage">649</span> <span class="lpage" tagx="lpage" title="lpage">652</span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/10688199">10688199</a> </span> <span class="pub-id"> <a href="https://doi.org/10.1038/35001059">10.1038/35001059</a> </span> </div> </li> <li tag="ref"> <a name="B6" /> <div class="citation" tagx="citation" title="citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Cho</span> <span class="given-names" tagx="given-names" title="given-names">Y</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Qiu</span> <span class="given-names" tagx="given-names" title="given-names">YL</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Kuhlman</span> <span class="given-names" tagx="given-names" title="given-names">P</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Palmer</span> <span class="given-names" tagx="given-names" title="given-names">JD</span> </span> </span> <div class="article-title" title="article-title">
Explosive invasion of plant mitochondria by a group I intron</div> <span class="source" tagx="source" title="source">Proc Natl Acad Sci USA</span> <span class="year" tagx="year" title="year">1998</span> <span class="volume" tagx="volume" title="volume">95</span> <span class="fpage" tagx="fpage" title="fpage">14244</span> <span class="lpage" tagx="lpage" title="lpage">14249</span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/9826685">9826685</a> </span> <span class="pub-id"> <a href="https://doi.org/10.1073/pnas.95.24.14244">10.1073/pnas.95.24.14244</a> </span> </div> </li> <li tag="ref"> <a name="B7" /> <div class="citation" tagx="citation" title="citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Cho</span> <span class="given-names" tagx="given-names" title="given-names">Y</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Palmer</span> <span class="given-names" tagx="given-names" title="given-names">JD</span> </span> </span> <div class="article-title" title="article-title">
Multiple acquisitions via horizontal transfer of a group I intron in the mitochondrial <i>cox1 </i>gene during evolution of the Araceae family </div> <span class="source" tagx="source" title="source">Mol Biol Evol</span> <span class="year" tagx="year" title="year">1999</span> <span class="volume" tagx="volume" title="volume">16</span> <span class="fpage" tagx="fpage" title="fpage">1155</span> <span class="lpage" tagx="lpage" title="lpage">1165</span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/10486971">10486971</a> </span> </div> </li> <li tag="ref"> <a name="B8" /> <div class="citation" tagx="citation" title="citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Bergthorsson</span> <span class="given-names" tagx="given-names" title="given-names">U</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Adams</span> <span class="given-names" tagx="given-names" title="given-names">KL</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Thomason</span> <span class="given-names" tagx="given-names" title="given-names">B</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Palmer</span> <span class="given-names" tagx="given-names" title="given-names">JD</span> </span> </span> <div class="article-title" title="article-title">
Widespread horizontal transfer of mitochondrial genes in flowering plants</div> <span class="source" tagx="source" title="source">Nature</span> <span class="year" tagx="year" title="year">2003</span> <span class="volume" tagx="volume" title="volume">424</span> <span class="fpage" tagx="fpage" title="fpage">197</span> <span class="lpage" tagx="lpage" title="lpage">201</span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/12853958">12853958</a> </span> <span class="pub-id"> <a href="https://doi.org/10.1038/nature01743">10.1038/nature01743</a> </span> </div> </li> <li tag="ref"> <a name="B9" /> <div class="citation" tagx="citation" title="citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Won</span> <span class="given-names" tagx="given-names" title="given-names">H</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Renner</span> <span class="given-names" tagx="given-names" title="given-names">SS</span> </span> </span> <div class="article-title" title="article-title">
Horizontal gene transfer from flowering plants to <i>Gnetum</i> </div> <span class="source" tagx="source" title="source">Proc Natl Acad Sci USA</span> <span class="year" tagx="year" title="year">2003</span> <span class="volume" tagx="volume" title="volume">100</span> <span class="fpage" tagx="fpage" title="fpage">10824</span> <span class="lpage" tagx="lpage" title="lpage">10829</span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/12963817">12963817</a> </span> <span class="pub-id"> <a href="https://doi.org/10.1073/pnas.1833775100">10.1073/pnas.1833775100</a> </span> </div> </li> <li tag="ref"> <a name="B10" /> <div class="citation" tagx="citation" title="citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Nickrent</span> <span class="given-names" tagx="given-names" title="given-names">DL</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Blarer</span> <span class="given-names" tagx="given-names" title="given-names">A</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Qiu</span> <span class="given-names" tagx="given-names" title="given-names">YL</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Vidal-Russell</span> <span class="given-names" tagx="given-names" title="given-names">R</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Anderson</span> <span class="given-names" tagx="given-names" title="given-names">FE</span> </span> </span> <div class="article-title" title="article-title">
Phylogenetic inference in Rafflesiales: the influence of rate heterogeneity and horizontal gene transfer </div> <span class="source" tagx="source" title="source">BMC Evol Biol</span> <span class="year" tagx="year" title="year">2004</span> <span class="volume" tagx="volume" title="volume">4</span> <span class="fpage" tagx="fpage" title="fpage">40</span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/15496229">15496229</a> </span> <span class="pub-id"> <a href="https://doi.org/10.1186/1471-2148-4-40">10.1186/1471-2148-4-40</a> </span> </div> </li> <li tag="ref"> <a name="B11" /> <div class="citation" tagx="citation" title="citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Mower</span> <span class="given-names" tagx="given-names" title="given-names">JP</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Stefanovic</span> <span class="given-names" tagx="given-names" title="given-names">S</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Young</span> <span class="given-names" tagx="given-names" title="given-names">GJ</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Palmer</span> <span class="given-names" tagx="given-names" title="given-names">JD</span> </span> </span> <div class="article-title" title="article-title">
Plant genetics: gene transfer from parasitic to host plants</div> <span class="source" tagx="source" title="source">Nature</span> <span class="year" tagx="year" title="year">2004</span> <span class="volume" tagx="volume" title="volume">432</span> <span class="fpage" tagx="fpage" title="fpage">165</span> <span class="lpage" tagx="lpage" title="lpage">166</span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/15538356">15538356</a> </span> <span class="pub-id"> <a href="https://doi.org/10.1038/432165b">10.1038/432165b</a> </span> </div> </li> <li tag="ref"> <a name="B12" /> <div class="citation" tagx="citation" title="citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Davis</span> <span class="given-names" tagx="given-names" title="given-names">CC</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Wurdack</span> <span class="given-names" tagx="given-names" title="given-names">KJ</span> </span> </span> <div class="article-title" title="article-title">
Host-to-parasite gene transfer in flowering plants: phylogenetic evidence from Malpighiales</div> <span class="source" tagx="source" title="source">Science</span> <span class="year" tagx="year" title="year">2004</span> <span class="volume" tagx="volume" title="volume">305</span> <span class="fpage" tagx="fpage" title="fpage">676</span> <span class="lpage" tagx="lpage" title="lpage">678</span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/15256617">15256617</a> </span> <span class="pub-id"> <a href="https://doi.org/10.1126/science.1100671">10.1126/science.1100671</a> </span> </div> </li> <li tag="ref"> <a name="B13" /> <div class="citation" tagx="citation" title="citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Bergthorsson</span> <span class="given-names" tagx="given-names" title="given-names">U</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Richardson</span> <span class="given-names" tagx="given-names" title="given-names">AO</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Young</span> <span class="given-names" tagx="given-names" title="given-names">GJ</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Goertzen</span> <span class="given-names" tagx="given-names" title="given-names">LR</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Palmer</span> <span class="given-names" tagx="given-names" title="given-names">JD</span> </span> </span> <div class="article-title" title="article-title">
Massive horizontal transfer of mitochondrial genes from diverse land plant donors to the basal angiosperm <i>Amborella</i> </div> <span class="source" tagx="source" title="source">Proc Natl Acad Sci USA</span> <span class="year" tagx="year" title="year">2004</span> <span class="volume" tagx="volume" title="volume">101</span> <span class="fpage" tagx="fpage" title="fpage">17747</span> <span class="lpage" tagx="lpage" title="lpage">17752</span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/15598737">15598737</a> </span> <span class="pub-id"> <a href="https://doi.org/10.1073/pnas.0408336102">10.1073/pnas.0408336102</a> </span> </div> </li> <li tag="ref"> <a name="B14" /> <div class="citation" tagx="citation" title="citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Davis</span> <span class="given-names" tagx="given-names" title="given-names">CC</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Anderson</span> <span class="given-names" tagx="given-names" title="given-names">WR</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Wurdack</span> <span class="given-names" tagx="given-names" title="given-names">KJ</span> </span> </span> <div class="article-title" title="article-title">
Gene transfer from a parasitic flowering plant to a fern</div> <span class="source" tagx="source" title="source">Proc Biol Sci</span> <span class="year" tagx="year" title="year">2005</span> <span class="volume" tagx="volume" title="volume">272</span> <span class="fpage" tagx="fpage" title="fpage">2237</span> <span class="lpage" tagx="lpage" title="lpage">2242</span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/16191635">16191635</a> </span> <span class="pub-id"> <a href="https://doi.org/10.1098/rspb.2005.3226">10.1098/rspb.2005.3226</a> </span> </div> </li> <li tag="ref"> <a name="B15" /> <div class="citation" tagx="citation" title="citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Schonenberger</span> <span class="given-names" tagx="given-names" title="given-names">J</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Anderberg</span> <span class="given-names" tagx="given-names" title="given-names">AA</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Sytsma</span> <span class="given-names" tagx="given-names" title="given-names">KJ</span> </span> </span> <div class="article-title" title="article-title">
Molecular phylogenetics and patterns of floral evolution in the Ericales</div> <span class="source" tagx="source" title="source">Int J Plant Sci</span> <span class="year" tagx="year" title="year">2005</span> <span class="volume" tagx="volume" title="volume">166</span> <span class="fpage" tagx="fpage" title="fpage">265</span> <span class="lpage" tagx="lpage" title="lpage">288</span> <span class="pub-id"> <a href="https://doi.org/10.1086/427198">10.1086/427198</a> </span> </div> </li> <li tag="ref"> <a name="B16" /> <div class="citation" tagx="citation" title="citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Delwiche</span> <span class="given-names" tagx="given-names" title="given-names">CF</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Palmer</span> <span class="given-names" tagx="given-names" title="given-names">JD</span> </span> </span> <div class="article-title" title="article-title">
Rampant horizontal transfer and duplication of Rubisco genes in eubacteria and plastids</div> <span class="source" tagx="source" title="source">Mol Biol Evol</span> <span class="year" tagx="year" title="year">1996</span> <span class="volume" tagx="volume" title="volume">31</span> <span class="fpage" tagx="fpage" title="fpage">873</span> <span class="lpage" tagx="lpage" title="lpage">882</span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/8754222">8754222</a> </span> </div> </li> <li tag="ref"> <a name="B17" /> <div class="citation" tagx="citation" title="citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Turmel</span> <span class="given-names" tagx="given-names" title="given-names">M</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Gutell</span> <span class="given-names" tagx="given-names" title="given-names">RR</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Mercier</span> <span class="given-names" tagx="given-names" title="given-names">JP</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Otis</span> <span class="given-names" tagx="given-names" title="given-names">C</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Lemieux</span> <span class="given-names" tagx="given-names" title="given-names">C</span> </span> </span> <div class="article-title" title="article-title">
Analysis of the chloroplast large subunit ribosomal RNA gene from 17 <i>Chlamydomonas </i>taxa. Three internal transcribed spacers and 12 group I intron insertion sites </div> <span class="source" tagx="source" title="source">J Mol Biol</span> <span class="year" tagx="year" title="year">1993</span> <span class="volume" tagx="volume" title="volume">232</span> <span class="fpage" tagx="fpage" title="fpage">446</span> <span class="lpage" tagx="lpage" title="lpage">467</span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/8393936">8393936</a> </span> <span class="pub-id"> <a href="https://doi.org/10.1006/jmbi.1993.1402">10.1006/jmbi.1993.1402</a> </span> </div> </li> <li tag="ref"> <a name="B18" /> <div class="citation" tagx="citation" title="citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Maier</span> <span class="given-names" tagx="given-names" title="given-names">UG</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Rensing</span> <span class="given-names" tagx="given-names" title="given-names">SA</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Igloi</span> <span class="given-names" tagx="given-names" title="given-names">GL</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Maerz</span> <span class="given-names" tagx="given-names" title="given-names">M</span> </span> </span> <div class="article-title" title="article-title">
Twintrons are not unique to the <i>Euglena </i>chloroplast genome – structure and evolution of a plastome <i>cpn60 </i>gene from a cryptomonad </div> <span class="source" tagx="source" title="source">Mol Gen Genet</span> <span class="year" tagx="year" title="year">1995</span> <span class="volume" tagx="volume" title="volume">246</span> <span class="fpage" tagx="fpage" title="fpage">128</span> <span class="lpage" tagx="lpage" title="lpage">131</span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/7823908">7823908</a> </span> <span class="pub-id"> <a href="https://doi.org/10.1007/BF00290141">10.1007/BF00290141</a> </span> </div> </li> <li tag="ref"> <a name="B19" /> <div class="citation" tagx="citation" title="citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Nozaki</span> <span class="given-names" tagx="given-names" title="given-names">H</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Ohta</span> <span class="given-names" tagx="given-names" title="given-names">N</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Yamada</span> <span class="given-names" tagx="given-names" title="given-names">T</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Takano</span> <span class="given-names" tagx="given-names" title="given-names">H</span> </span> </span> <div class="article-title" title="article-title">
Characterization of <i>rbcL </i>group IA introns from two colonial volvocalean species (Chlorophyceae) </div> <span class="source" tagx="source" title="source">Plant Mol Biol</span> <span class="year" tagx="year" title="year">1998</span> <span class="volume" tagx="volume" title="volume">37</span> <span class="fpage" tagx="fpage" title="fpage">77</span> <span class="lpage" tagx="lpage" title="lpage">85</span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/9620266">9620266</a> </span> <span class="pub-id"> <a href="https://doi.org/10.1023/A:1005904410345">10.1023/A:1005904410345</a> </span> </div> </li> <li tag="ref"> <a name="B20" /> <div class="citation" tagx="citation" title="citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Sheveleva</span> <span class="given-names" tagx="given-names" title="given-names">EV</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Hallick</span> <span class="given-names" tagx="given-names" title="given-names">RB</span> </span> </span> <div class="article-title" title="article-title">
Recent horizontal intron transfer to a chloroplast genome</div> <span class="source" tagx="source" title="source">Nucleic Acids Res</span> <span class="year" tagx="year" title="year">2004</span> <span class="volume" tagx="volume" title="volume">32</span> <span class="fpage" tagx="fpage" title="fpage">803</span> <span class="lpage" tagx="lpage" title="lpage">810</span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/14762207">14762207</a> </span> <span class="pub-id"> <a href="https://doi.org/10.1093/nar/gkh225">10.1093/nar/gkh225</a> </span> </div> </li> <li tag="ref"> <a name="B21" /> <div class="citation" tagx="citation" title="citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Odom</span> <span class="given-names" tagx="given-names" title="given-names">OW</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Shenkenberg</span> <span class="given-names" tagx="given-names" title="given-names">DL</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Garcia</span> <span class="given-names" tagx="given-names" title="given-names">JA</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Herrin</span> <span class="given-names" tagx="given-names" title="given-names">DL</span> </span> </span> <div class="article-title" title="article-title">
A horizontally acquired group II intron in the chloroplast <i>psbA </i>gene of a psychrophilic <i>Chlamydomonas</i>: in vitro self-splicing and genetic evidence for maturase activity </div> <span class="source" tagx="source" title="source">RNA</span> <span class="year" tagx="year" title="year">2004</span> <span class="volume" tagx="volume" title="volume">10</span> <span class="fpage" tagx="fpage" title="fpage">1097</span> <span class="lpage" tagx="lpage" title="lpage">1107</span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/15208445">15208445</a> </span> <span class="pub-id"> <a href="https://doi.org/10.1261/rna.7140604">10.1261/rna.7140604</a> </span> </div> </li> <li tag="ref"> <a name="B22" /> <div class="citation" tagx="citation" title="citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Thompson</span> <span class="given-names" tagx="given-names" title="given-names">MD</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Copertino</span> <span class="given-names" tagx="given-names" title="given-names">DW</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Thompson</span> <span class="given-names" tagx="given-names" title="given-names">E</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Favreau</span> <span class="given-names" tagx="given-names" title="given-names">MR</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Hallick</span> <span class="given-names" tagx="given-names" title="given-names">RB</span> </span> </span> <div class="article-title" title="article-title">
Evidence for the late origin of introns in chloroplast genes from an evolutionary analysis of the genus <i>Euglena</i> </div> <span class="source" tagx="source" title="source">Nucleic Acids Res</span> <span class="year" tagx="year" title="year">1995</span> <span class="volume" tagx="volume" title="volume">23</span> <span class="fpage" tagx="fpage" title="fpage">4745</span> <span class="lpage" tagx="lpage" title="lpage">4752</span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/8532514">8532514</a> </span> </div> </li> <li tag="ref"> <a name="B23" /> <div class="citation" tagx="citation" title="citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Doetsch</span> <span class="given-names" tagx="given-names" title="given-names">NA</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Thompson</span> <span class="given-names" tagx="given-names" title="given-names">MD</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Favreau</span> <span class="given-names" tagx="given-names" title="given-names">MR</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Hallick</span> <span class="given-names" tagx="given-names" title="given-names">RB</span> </span> </span> <div class="article-title" title="article-title">
Comparison of psbK operon organization and group III intron content in chloroplast genomes of 12 Euglenoid species </div> <span class="source" tagx="source" title="source">Mol Gen Genet</span> <span class="year" tagx="year" title="year">2001</span> <span class="volume" tagx="volume" title="volume">264</span> <span class="fpage" tagx="fpage" title="fpage">682</span> <span class="lpage" tagx="lpage" title="lpage">690</span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/11212923">11212923</a> </span> <span class="pub-id"> <a href="https://doi.org/10.1007/s004380000355">10.1007/s004380000355</a> </span> </div> </li> <li tag="ref"> <a name="B24" /> <div class="citation" tagx="citation" title="citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Hanyuda</span> <span class="given-names" tagx="given-names" title="given-names">T</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Arai</span> <span class="given-names" tagx="given-names" title="given-names">S</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Ueda</span> <span class="given-names" tagx="given-names" title="given-names">K</span> </span> </span> <div class="article-title" title="article-title">
Variability in the <i>rbcL </i>introns of Caulerpalean algae (Chlorophyta, Ulvophyceae) </div> <span class="source" tagx="source" title="source">J Plant Res</span> <span class="year" tagx="year" title="year">2000</span> <span class="volume" tagx="volume" title="volume">113</span> <span class="fpage" tagx="fpage" title="fpage">403</span> <span class="lpage" tagx="lpage" title="lpage">413</span> <span class="pub-id"> <a href="https://doi.org/10.1007/PL00013948">10.1007/PL00013948</a> </span> </div> </li> <li tag="ref"> <a name="B25" /> <div class="citation" tagx="citation" title="citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Pombert</span> <span class="given-names" tagx="given-names" title="given-names">JF</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Otis</span> <span class="given-names" tagx="given-names" title="given-names">C</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Lemieux</span> <span class="given-names" tagx="given-names" title="given-names">C</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Turmel</span> <span class="given-names" tagx="given-names" title="given-names">M</span> </span> </span> <div class="article-title" title="article-title">
Chloroplast genome sequence of the green alga <i>Pseudendoclonium akinetum </i>(Ulvophyceae) reveals unusual structural features and new insights into the branching order of chlorophyte lineages </div> <span class="source" tagx="source" title="source">Mol Biol Evol</span> <span class="year" tagx="year" title="year">2005</span> <span class="volume" tagx="volume" title="volume">22</span> <span class="fpage" tagx="fpage" title="fpage">1903</span> <span class="lpage" tagx="lpage" title="lpage">1918</span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/15930151">15930151</a> </span> <span class="pub-id"> <a href="https://doi.org/10.1093/molbev/msi182">10.1093/molbev/msi182</a> </span> </div> </li> <li tag="ref"> <a name="B26" /> <div class="citation" tagx="citation" title="citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Hoef-Emden</span> <span class="given-names" tagx="given-names" title="given-names">K</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Marin</span> <span class="given-names" tagx="given-names" title="given-names">B</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Melkonian</span> <span class="given-names" tagx="given-names" title="given-names">M</span> </span> </span> <div class="article-title" title="article-title">
Nuclear and nucleomorph SSU rDNA phylogeny in the Cryptophyta and the evolution of cryptophyte diversity </div> <span class="source" tagx="source" title="source">J Mol Evol</span> <span class="year" tagx="year" title="year">2002</span> <span class="volume" tagx="volume" title="volume">55</span> <span class="fpage" tagx="fpage" title="fpage">161</span> <span class="lpage" tagx="lpage" title="lpage">179</span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/12107593">12107593</a> </span> <span class="pub-id"> <a href="https://doi.org/10.1007/s00239-002-2313-5">10.1007/s00239-002-2313-5</a> </span> </div> </li> <li tag="ref"> <a name="B27" /> <div class="citation" tagx="citation" title="citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Sanchez-Puerta</span> <span class="given-names" tagx="given-names" title="given-names">MV</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Bachvaroff</span> <span class="given-names" tagx="given-names" title="given-names">TR</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Delwiche</span> <span class="given-names" tagx="given-names" title="given-names">CF</span> </span> </span> <div class="article-title" title="article-title">
The complete plastid genome sequence of the haptophyte <i>Emiliania huxleyi</i>: a comparison to other plastid genomes </div> <span class="source" tagx="source" title="source">DNA Res</span> <span class="year" tagx="year" title="year">2005</span> <span class="volume" tagx="volume" title="volume">12</span> <span class="fpage" tagx="fpage" title="fpage">151</span> <span class="lpage" tagx="lpage" title="lpage">156</span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/16303746">16303746</a> </span> <span class="pub-id"> <a href="https://doi.org/10.1093/dnares/12.2.151">10.1093/dnares/12.2.151</a> </span> </div> </li> <li tag="ref"> <a name="B28" /> <div class="citation" tagx="citation" title="citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Ishida</span> <span class="given-names" tagx="given-names" title="given-names">K</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Green</span> <span class="given-names" tagx="given-names" title="given-names">BR</span> </span> </span> <div class="article-title" title="article-title">
Second- and third-hand chloroplasts in dinoflagellates: phylogeny of oxygen-evolving enhancer 1 (PsbO) protein reveals replacement of a nuclear-encoded plastid gene by that of a haptophyte tertiary endosymbiont </div> <span class="source" tagx="source" title="source">Proc Natl Acad Sci USA</span> <span class="year" tagx="year" title="year">2002</span> <span class="volume" tagx="volume" title="volume">99</span> <span class="fpage" tagx="fpage" title="fpage">9294</span> <span class="lpage" tagx="lpage" title="lpage">9299</span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/12089328">12089328</a> </span> <span class="pub-id"> <a href="https://doi.org/10.1073/pnas.142091799">10.1073/pnas.142091799</a> </span> </div> </li> <li tag="ref"> <a name="B29" /> <div class="citation" tagx="citation" title="citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Yoon</span> <span class="given-names" tagx="given-names" title="given-names">HS</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Hackett</span> <span class="given-names" tagx="given-names" title="given-names">JD</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Van Dolah</span> <span class="given-names" tagx="given-names" title="given-names">FM</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Nosenko</span> <span class="given-names" tagx="given-names" title="given-names">T</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Lidie</span> <span class="given-names" tagx="given-names" title="given-names">KL</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Bhattacharya</span> <span class="given-names" tagx="given-names" title="given-names">D</span> </span> </span> <div class="article-title" title="article-title">
Tertiary endosymbiosis driven genome evolution in dinoflagellate algae</div> <span class="source" tagx="source" title="source">Mol Biol Evol</span> <span class="year" tagx="year" title="year">2005</span> <span class="volume" tagx="volume" title="volume">22</span> <span class="fpage" tagx="fpage" title="fpage">1299</span> <span class="lpage" tagx="lpage" title="lpage">1308</span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/15746017">15746017</a> </span> <span class="pub-id"> <a href="https://doi.org/10.1093/molbev/msi118">10.1093/molbev/msi118</a> </span> </div> </li> <li tag="ref"> <a name="B30" /> <div class="citation" tagx="citation" title="citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Patron</span> <span class="given-names" tagx="given-names" title="given-names">NJ</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Waller</span> <span class="given-names" tagx="given-names" title="given-names">RF</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Keeling</span> <span class="given-names" tagx="given-names" title="given-names">PJ</span> </span> </span> <div class="article-title" title="article-title">
A tertiary plastid uses genes from twoendosymbionts</div> <span class="source" tagx="source" title="source">J Mol Biol</span> <span class="year" tagx="year" title="year">2006</span> <span class="volume" tagx="volume" title="volume">357</span> <span class="fpage" tagx="fpage" title="fpage">1373</span> <span class="lpage" tagx="lpage" title="lpage">1382</span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/16490209">16490209</a> </span> <span class="pub-id"> <a href="https://doi.org/10.1016/j.jmb.2006.01.084">10.1016/j.jmb.2006.01.084</a> </span> </div> </li> <li tag="ref"> <a name="B31" /> <div class="citation" tagx="citation" title="citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Hupfer</span> <span class="given-names" tagx="given-names" title="given-names">H</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Swiatek</span> <span class="given-names" tagx="given-names" title="given-names">M</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Hornung</span> <span class="given-names" tagx="given-names" title="given-names">S</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Herrmann</span> <span class="given-names" tagx="given-names" title="given-names">RG</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Maier</span> <span class="given-names" tagx="given-names" title="given-names">RM</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Chiu</span> <span class="given-names" tagx="given-names" title="given-names">WL</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Sears</span> <span class="given-names" tagx="given-names" title="given-names">B</span> </span> </span> <div class="article-title" title="article-title">
Complete nucleotide sequence of the <i>Oenothera elata </i>plastid chromosome, representing plastome I of the five distinguishable euoenothera plastomes </div> <span class="source" tagx="source" title="source">Mol Gen Genet</span> <span class="year" tagx="year" title="year">2000</span> <span class="volume" tagx="volume" title="volume">263</span> <span class="fpage" tagx="fpage" title="fpage">581</span> <span class="lpage" tagx="lpage" title="lpage">585</span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/10852478">10852478</a> </span> </div> </li> <li tag="ref"> <a name="B32" /> <div class="citation" tagx="citation" title="citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Keeling</span> <span class="given-names" tagx="given-names" title="given-names">PJ</span> </span> </span> <div class="article-title" title="article-title">
Diversity and evolutionary history of plastids and their hosts</div> <span class="source" tagx="source" title="source">Am J Bot</span> <span class="year" tagx="year" title="year">2004</span> <span class="volume" tagx="volume" title="volume">91</span> <span class="fpage" tagx="fpage" title="fpage">1481</span> <span class="lpage" tagx="lpage" title="lpage">1493</span> </div> </li> <li tag="ref"> <a name="B33" /> <div class="citation" tagx="citation" title="citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Inagaki</span> <span class="given-names" tagx="given-names" title="given-names">Y</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Simpson</span> <span class="given-names" tagx="given-names" title="given-names">AGB</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Dacks</span> <span class="given-names" tagx="given-names" title="given-names">JB</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Roger</span> <span class="given-names" tagx="given-names" title="given-names">AJ</span> </span> </span> <div class="article-title" title="article-title">
Phylogenetic artifacts can be caused by leucine, serine, and arginine codon usage heterogeneity: Dinoflagellate plastid origins as a case study </div> <span class="source" tagx="source" title="source">Syst Biol</span> <span class="year" tagx="year" title="year">2004</span> <span class="volume" tagx="volume" title="volume">53</span> <span class="fpage" tagx="fpage" title="fpage">582</span> <span class="lpage" tagx="lpage" title="lpage">593</span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/15371248">15371248</a> </span> <span class="pub-id"> <a href="https://doi.org/10.1080/10635150490468756">10.1080/10635150490468756</a> </span> </div> </li> <li tag="ref"> <a name="B34" /> <div class="citation" tagx="citation" title="citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Stefanovic</span> <span class="given-names" tagx="given-names" title="given-names">S</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Rice</span> <span class="given-names" tagx="given-names" title="given-names">DW</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Palmer</span> <span class="given-names" tagx="given-names" title="given-names">JD</span> </span> </span> <div class="article-title" title="article-title">
Long branch attraction, taxon sampling, and the earliest angiosperms: <i>Amborella </i>or monocots? </div> <span class="source" tagx="source" title="source">BMC Evol Biol</span> <span class="year" tagx="year" title="year">2004</span> <span class="volume" tagx="volume" title="volume">4</span> <span class="fpage" tagx="fpage" title="fpage">35</span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/15453916">15453916</a> </span> <span class="pub-id"> <a href="https://doi.org/10.1186/1471-2148-4-35">10.1186/1471-2148-4-35</a> </span> </div> </li> <li tag="ref"> <a name="B35" /> <div class="citation" tagx="citation" title="citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Turmel</span> <span class="given-names" tagx="given-names" title="given-names">M</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Ehara</span> <span class="given-names" tagx="given-names" title="given-names">M</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Otis</span> <span class="given-names" tagx="given-names" title="given-names">C</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Lemieux</span> <span class="given-names" tagx="given-names" title="given-names">C</span> </span> </span> <div class="article-title" title="article-title">
Phylogenetic relationships among streptophytes as inferred from chloroplast small and large subunit rRNA gene sequences </div> <span class="source" tagx="source" title="source">J Phycol</span> <span class="year" tagx="year" title="year">2002</span> <span class="volume" tagx="volume" title="volume">38</span> <span class="fpage" tagx="fpage" title="fpage">364</span> <span class="lpage" tagx="lpage" title="lpage">375</span> <span class="pub-id"> <a href="https://doi.org/10.1046/j.1529-8817.2002.01163.x">10.1046/j.1529-8817.2002.01163.x</a> </span> </div> </li> <li tag="ref"> <a name="B36" /> <div class="citation" tagx="citation" title="citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Cavalier-Smith</span> <span class="given-names" tagx="given-names" title="given-names">T</span> </span> </span> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Green JC, Leadbeater BSC</span> </span> </span> <div class="article-title" title="article-title">
Origin and relationships of Haptophyta</div> <span class="source" tagx="source" title="source">The Haptophyte algae</span> <span class="year" tagx="year" title="year">1994</span> <span class="publisher-name" tagx="publisher-name" title="publisher-name">Oxford: Clarendon Press</span> <span class="fpage" tagx="fpage" title="fpage">413</span> <span class="lpage" tagx="lpage" title="lpage">435</span> </div> </li> <li tag="ref"> <a name="B37" /> <div class="citation" tagx="citation" title="citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Cavalier-Smith</span> <span class="given-names" tagx="given-names" title="given-names">T</span> </span> </span> <div class="article-title" title="article-title">
Genomic reduction and evolution of novel genetic membranes and protein-targeting machinery in eukaryote-eukaryote chimaeras (meta-algae) </div> <span class="source" tagx="source" title="source">Philos Trans R Soc Lond B Biol Sci</span> <span class="year" tagx="year" title="year">2003</span> <span class="volume" tagx="volume" title="volume">358</span> <span class="fpage" tagx="fpage" title="fpage">109</span> <span class="lpage" tagx="lpage" title="lpage">133</span> <span class="comment">discussion 133–140</span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/12594921">12594921</a> </span> <span class="pub-id"> <a href="https://doi.org/10.1098/rstb.2002.1194">10.1098/rstb.2002.1194</a> </span> </div> </li> <li tag="ref"> <a name="B38" /> <div class="citation" tagx="citation" title="citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Bachvaroff</span> <span class="given-names" tagx="given-names" title="given-names">TR</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Sanchez-Puerta</span> <span class="given-names" tagx="given-names" title="given-names">MV</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Delwiche</span> <span class="given-names" tagx="given-names" title="given-names">CF</span> </span> </span> <div class="article-title" title="article-title">
Chlorophyll c-containing plastid relationships based on analyses of a multigene data set with all four chromalveolate lineages </div> <span class="source" tagx="source" title="source">Mol Biol Evol</span> <span class="year" tagx="year" title="year">2005</span> <span class="volume" tagx="volume" title="volume">22</span> <span class="fpage" tagx="fpage" title="fpage">1772</span> <span class="lpage" tagx="lpage" title="lpage">1782</span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/15917498">15917498</a> </span> <span class="pub-id"> <a href="https://doi.org/10.1093/molbev/msi172">10.1093/molbev/msi172</a> </span> </div> </li> <li tag="ref"> <a name="B39" /> <div class="citation" tagx="citation" title="citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Yoon</span> <span class="given-names" tagx="given-names" title="given-names">HS</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Hackett</span> <span class="given-names" tagx="given-names" title="given-names">JD</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Pinto</span> <span class="given-names" tagx="given-names" title="given-names">G</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Bhattacharya</span> <span class="given-names" tagx="given-names" title="given-names">D</span> </span> </span> <div class="article-title" title="article-title">
The single, ancient originof chromist plastids</div> <span class="source" tagx="source" title="source">Proc Natl Acad Sci USA</span> <span class="year" tagx="year" title="year">2002</span> <span class="volume" tagx="volume" title="volume">99</span> <span class="fpage" tagx="fpage" title="fpage">15507</span> <span class="lpage" tagx="lpage" title="lpage">15512</span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/12438651">12438651</a> </span> <span class="pub-id"> <a href="https://doi.org/10.1073/pnas.242379899">10.1073/pnas.242379899</a> </span> </div> </li> <li tag="ref"> <a name="B40" /> <div class="citation" tagx="citation" title="citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Cavalier-Smith</span> <span class="given-names" tagx="given-names" title="given-names">T</span> </span> </span> <div class="article-title" title="article-title">
Principles of protein and lipid targeting in secondarysymbiogenesis: Euglenoid, dinoflagellate, and sporozoan plastid origins and the eukaryote family tree </div> <span class="source" tagx="source" title="source">J Eukaryot Microbiol</span> <span class="year" tagx="year" title="year">1999</span> <span class="volume" tagx="volume" title="volume">46</span> <span class="fpage" tagx="fpage" title="fpage">347</span> <span class="lpage" tagx="lpage" title="lpage">366</span> </div> </li> <li tag="ref"> <a name="B41" /> <div class="citation" tagx="citation" title="citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Harper</span> <span class="given-names" tagx="given-names" title="given-names">JT</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Waanders</span> <span class="given-names" tagx="given-names" title="given-names">E</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Keeling</span> <span class="given-names" tagx="given-names" title="given-names">PJ</span> </span> </span> <div class="article-title" title="article-title">
On the monophyly of chromalveolates using a six-protein phylogeny of eukaryotes</div> <span class="source" tagx="source" title="source">Int J Syst Evol Microbiol</span> <span class="year" tagx="year" title="year">2005</span> <span class="volume" tagx="volume" title="volume">55</span> <span class="fpage" tagx="fpage" title="fpage">487</span> <span class="lpage" tagx="lpage" title="lpage">496</span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/15653923">15653923</a> </span> <span class="pub-id"> <a href="https://doi.org/10.1099/ijs.0.63216-0">10.1099/ijs.0.63216-0</a> </span> </div> </li> <li tag="ref"> <a name="B42" /> <div class="citation" tagx="citation" title="citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Majewski</span> <span class="given-names" tagx="given-names" title="given-names">J</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Cohan</span> <span class="given-names" tagx="given-names" title="given-names">FM</span> </span> </span> <div class="article-title" title="article-title">
DNA sequence similarity requirements for interspecific recombination in <i>Bacillus</i> </div> <span class="source" tagx="source" title="source">Genetics</span> <span class="year" tagx="year" title="year">1999</span> <span class="volume" tagx="volume" title="volume">153</span> <span class="fpage" tagx="fpage" title="fpage">1525</span> <span class="lpage" tagx="lpage" title="lpage">1533</span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/10581263">10581263</a> </span> </div> </li> <li tag="ref"> <a name="B43" /> <div class="citation" tagx="citation" title="citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Thomas</span> <span class="given-names" tagx="given-names" title="given-names">CM</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Nielsen</span> <span class="given-names" tagx="given-names" title="given-names">KM</span> </span> </span> <div class="article-title" title="article-title">
Mechanisms of, and barriers to, horizontal gene transfer between bacteria</div> <span class="source" tagx="source" title="source">Nat Rev Microbiol</span> <span class="year" tagx="year" title="year">2005</span> <span class="volume" tagx="volume" title="volume">3</span> <span class="fpage" tagx="fpage" title="fpage">711</span> <span class="lpage" tagx="lpage" title="lpage">721</span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/16138099">16138099</a> </span> <span class="pub-id"> <a href="https://doi.org/10.1038/nrmicro1234">10.1038/nrmicro1234</a> </span> </div> </li> <li tag="ref"> <a name="B44" /> <div class="citation" tagx="citation" title="citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Boucher</span> <span class="given-names" tagx="given-names" title="given-names">Y</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Douady</span> <span class="given-names" tagx="given-names" title="given-names">CJ</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Sharma</span> <span class="given-names" tagx="given-names" title="given-names">AK</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Kamekura</span> <span class="given-names" tagx="given-names" title="given-names">M</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Doolittle</span> <span class="given-names" tagx="given-names" title="given-names">WF</span> </span> </span> <div class="article-title" title="article-title">
Intragenomic heterogeneity and intergenomic recombination among haloarchaeal rRNA genes </div> <span class="source" tagx="source" title="source">J Bacteriol</span> <span class="year" tagx="year" title="year">2004</span> <span class="volume" tagx="volume" title="volume">186</span> <span class="fpage" tagx="fpage" title="fpage">3980</span> <span class="lpage" tagx="lpage" title="lpage">3990</span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/15175312">15175312</a> </span> <span class="pub-id"> <a href="https://doi.org/10.1128/JB.186.12.3980-3990.2004">10.1128/JB.186.12.3980-3990.2004</a> </span> </div> </li> <li tag="ref"> <a name="B45" /> <div class="citation" tagx="citation" title="citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Inagaki</span> <span class="given-names" tagx="given-names" title="given-names">Y</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Susko</span> <span class="given-names" tagx="given-names" title="given-names">E</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Roger</span> <span class="given-names" tagx="given-names" title="given-names">AJ</span> </span> </span> <div class="article-title" title="article-title">
Recombination between elongation factor 1-alpha genes from distantly-related archaeal lineages </div> <span class="source" tagx="source" title="source">Proc Natl Acad Sci USA</span> <span class="year" tagx="year" title="year">2006</span> <span class="comment" /> </div> </li> <li tag="ref"> <a name="B46" /> <div class="citation" tagx="citation" title="citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Ikeda</span> <span class="given-names" tagx="given-names" title="given-names">H</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Shiraishi</span> <span class="given-names" tagx="given-names" title="given-names">K</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Ogata</span> <span class="given-names" tagx="given-names" title="given-names">Y</span> </span> </span> <div class="article-title" title="article-title">
Illegitimate recombination mediated by double-strand break and end-joining in <i>Escherichia coli</i> </div> <span class="source" tagx="source" title="source">Adv Biophys</span> <span class="year" tagx="year" title="year">2004</span> <span class="volume" tagx="volume" title="volume">38</span> <span class="fpage" tagx="fpage" title="fpage">3</span> <span class="lpage" tagx="lpage" title="lpage">20</span> <span class="pub-id"> <a href="https://doi.org/10.1016/S0065-227X(04)80031-5">10.1016/S0065-227X(04)80031-5</a> </span> </div> </li> <li tag="ref"> <a name="B47" /> <div class="citation" tagx="citation" title="citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Cohan</span> <span class="given-names" tagx="given-names" title="given-names">FM</span> </span> </span> <div class="article-title" title="article-title">
Bacterial species and speciation</div> <span class="source" tagx="source" title="source">Syst Biol</span> <span class="year" tagx="year" title="year">2001</span> <span class="volume" tagx="volume" title="volume">50</span> <span class="fpage" tagx="fpage" title="fpage">513</span> <span class="lpage" tagx="lpage" title="lpage">524</span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/12116650">12116650</a> </span> </div> </li> <li tag="ref"> <a name="B48" /> <div class="citation" tagx="citation" title="citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Bajaj</span> <span class="given-names" tagx="given-names" title="given-names">M</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Blundell</span> <span class="given-names" tagx="given-names" title="given-names">T</span> </span> </span> <div class="article-title" title="article-title">
Evolution and the tertiary structure of proteins</div> <span class="source" tagx="source" title="source">Annu Rev Biophys Bioeng</span> <span class="year" tagx="year" title="year">1984</span> <span class="volume" tagx="volume" title="volume">13</span> <span class="fpage" tagx="fpage" title="fpage">453</span> <span class="lpage" tagx="lpage" title="lpage">492</span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/6378074">6378074</a> </span> <span class="pub-id"> <a href="https://doi.org/10.1146/annurev.bb.13.060184.002321">10.1146/annurev.bb.13.060184.002321</a> </span> </div> </li> <li tag="ref"> <a name="B49" /> <div class="citation" tagx="citation" title="citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Knight</span> <span class="given-names" tagx="given-names" title="given-names">RD</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Freeland</span> <span class="given-names" tagx="given-names" title="given-names">SJ</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Landweber</span> <span class="given-names" tagx="given-names" title="given-names">LF</span> </span> </span> <div class="article-title" title="article-title">
A simple model based on mutation and selection explains trends in codon and amino-acid usage and GC composition within and across genomes </div> <span class="source" tagx="source" title="source">Genome Biol</span> <span class="year" tagx="year" title="year">2001</span> <span class="volume" tagx="volume" title="volume">2</span> <span class="fpage" tagx="fpage" title="fpage">RESEARCH0010</span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/11305938">11305938</a> </span> </div> </li> <li tag="ref"> <a name="B50" /> <div class="citation" tagx="citation" title="citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Schuwirth</span> <span class="given-names" tagx="given-names" title="given-names">BS</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Borovinskaya</span> <span class="given-names" tagx="given-names" title="given-names">MA</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Hau</span> <span class="given-names" tagx="given-names" title="given-names">CW</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Zhang</span> <span class="given-names" tagx="given-names" title="given-names">W</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Vila-Sanjurjo</span> <span class="given-names" tagx="given-names" title="given-names">A</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Holton</span> <span class="given-names" tagx="given-names" title="given-names">JM</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Cate</span> <span class="given-names" tagx="given-names" title="given-names">JH</span> </span> </span> <div class="article-title" title="article-title">
Structures of the bacterial ribosome at 3.5 A resolution</div> <span class="source" tagx="source" title="source">Science</span> <span class="year" tagx="year" title="year">2005</span> <span class="volume" tagx="volume" title="volume">310</span> <span class="fpage" tagx="fpage" title="fpage">827</span> <span class="lpage" tagx="lpage" title="lpage">834</span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/16272117">16272117</a> </span> <span class="pub-id"> <a href="https://doi.org/10.1126/science.1117230">10.1126/science.1117230</a> </span> </div> </li> <li tag="ref"> <a name="B51" /> <div class="citation" tagx="citation" title="citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Valentin</span> <span class="given-names" tagx="given-names" title="given-names">K</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Zetsche</span> <span class="given-names" tagx="given-names" title="given-names">K</span> </span> </span> <div class="article-title" title="article-title">
The genes of both subunits of ribulose-1,5-bisphosphate carboxylase constitute an operon on the plastome of a red alga </div> <span class="source" tagx="source" title="source">Curr Genet</span> <span class="year" tagx="year" title="year">1989</span> <span class="volume" tagx="volume" title="volume">16</span> <span class="fpage" tagx="fpage" title="fpage">203</span> <span class="lpage" tagx="lpage" title="lpage">209</span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/2598276">2598276</a> </span> <span class="pub-id"> <a href="https://doi.org/10.1007/BF00391478">10.1007/BF00391478</a> </span> </div> </li> <li tag="ref"> <a name="B52" /> <div class="citation" tagx="citation" title="citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Boczar</span> <span class="given-names" tagx="given-names" title="given-names">BA</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Delaney</span> <span class="given-names" tagx="given-names" title="given-names">TP</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Cattolico</span> <span class="given-names" tagx="given-names" title="given-names">RA</span> </span> </span> <div class="article-title" title="article-title">
Gene for the ribulose-1,5-bisphosphate carboxylase small subunit protein of the marine chromophyte <i>Olisthodiscus luteus </i>is similar to that of a chemoautotrophic bacterium </div> <span class="source" tagx="source" title="source">Proc Natl Acad Sci USA</span> <span class="year" tagx="year" title="year">1989</span> <span class="volume" tagx="volume" title="volume">86</span> <span class="fpage" tagx="fpage" title="fpage">4996</span> <span class="lpage" tagx="lpage" title="lpage">4999</span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/2740337">2740337</a> </span> <span class="pub-id"> <a href="https://doi.org/10.1073/pnas.86.13.4996">10.1073/pnas.86.13.4996</a> </span> </div> </li> <li tag="ref"> <a name="B53" /> <div class="citation" tagx="citation" title="citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Martin</span> <span class="given-names" tagx="given-names" title="given-names">W</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Schnarrenberger</span> <span class="given-names" tagx="given-names" title="given-names">C</span> </span> </span> <div class="article-title" title="article-title">
The evolution of the Calvin cycle from prokaryotic to eukaryotic chromosomes: a case study of functional redundancy in ancient pathways through endosymbiosis </div> <span class="source" tagx="source" title="source">Curr Genet</span> <span class="year" tagx="year" title="year">1997</span> <span class="volume" tagx="volume" title="volume">32</span> <span class="fpage" tagx="fpage" title="fpage">1</span> <span class="lpage" tagx="lpage" title="lpage">18</span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/9309164">9309164</a> </span> <span class="pub-id"> <a href="https://doi.org/10.1007/s002940050241">10.1007/s002940050241</a> </span> </div> </li> <li tag="ref"> <a name="B54" /> <div class="citation" tagx="citation" title="citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Uchino</span> <span class="given-names" tagx="given-names" title="given-names">Y</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Yokota</span> <span class="given-names" tagx="given-names" title="given-names">A</span> </span> </span> <div class="article-title" title="article-title">
"Green-like" and "red-like" RubisCO <i>cbbL </i>genes in <i>Rhodobacter azotoformans</i> </div> <span class="source" tagx="source" title="source">Mol Biol Evol</span> <span class="year" tagx="year" title="year">2003</span> <span class="volume" tagx="volume" title="volume">20</span> <span class="fpage" tagx="fpage" title="fpage">821</span> <span class="lpage" tagx="lpage" title="lpage">830</span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/12679539">12679539</a> </span> <span class="pub-id"> <a href="https://doi.org/10.1093/molbev/msg100">10.1093/molbev/msg100</a> </span> </div> </li> <li tag="ref"> <a name="B55" /> <div class="citation" tagx="citation" title="citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Assali</span> <span class="given-names" tagx="given-names" title="given-names">NE</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Martin</span> <span class="given-names" tagx="given-names" title="given-names">WF</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Sommerville</span> <span class="given-names" tagx="given-names" title="given-names">CC</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Loiseaux-de Goër</span> <span class="given-names" tagx="given-names" title="given-names">S</span> </span> </span> <div class="article-title" title="article-title">
Evolution of the Rubisco operon from prokaryotes to algae: Structure and analysis of the <i>rbcS </i>gene of the brown alga <i>Pylaiella littoralis</i> </div> <span class="source" tagx="source" title="source">Plant Mol Biol</span> <span class="year" tagx="year" title="year">1991</span> <span class="volume" tagx="volume" title="volume">17</span> <span class="fpage" tagx="fpage" title="fpage">853</span> <span class="lpage" tagx="lpage" title="lpage">863</span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/1840691">1840691</a> </span> <span class="pub-id"> <a href="https://doi.org/10.1007/BF00037066">10.1007/BF00037066</a> </span> </div> </li> <li tag="ref"> <a name="B56" /> <div class="citation" tagx="citation" title="citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Douzery</span> <span class="given-names" tagx="given-names" title="given-names">EJP</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Snell</span> <span class="given-names" tagx="given-names" title="given-names">EA</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Bapteste</span> <span class="given-names" tagx="given-names" title="given-names">E</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Delsuc</span> <span class="given-names" tagx="given-names" title="given-names">F</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Philippe</span> <span class="given-names" tagx="given-names" title="given-names">H</span> </span> </span> <div class="article-title" title="article-title">
The timing of eukaryotic evolution: Does a relaxed molecular clock reconcile proteins and fossils? </div> <span class="source" tagx="source" title="source">Proc Natl Acad Sci USA</span> <span class="year" tagx="year" title="year">2004</span> <span class="volume" tagx="volume" title="volume">101</span> <span class="fpage" tagx="fpage" title="fpage">15386</span> <span class="lpage" tagx="lpage" title="lpage">15391</span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/15494441">15494441</a> </span> <span class="pub-id"> <a href="https://doi.org/10.1073/pnas.0403984101">10.1073/pnas.0403984101</a> </span> </div> </li> <li tag="ref"> <a name="B57" /> <div class="citation" tagx="citation" title="citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Yoon</span> <span class="given-names" tagx="given-names" title="given-names">HS</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Hackett</span> <span class="given-names" tagx="given-names" title="given-names">JD</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Ciniglia</span> <span class="given-names" tagx="given-names" title="given-names">C</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Pinto</span> <span class="given-names" tagx="given-names" title="given-names">G</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Bhattacharya</span> <span class="given-names" tagx="given-names" title="given-names">D</span> </span> </span> <div class="article-title" title="article-title">
A molecular timeline for the origin of photosynthetic eukaryotes</div> <span class="source" tagx="source" title="source">Mol Biol Evol</span> <span class="year" tagx="year" title="year">2004</span> <span class="volume" tagx="volume" title="volume">21</span> <span class="fpage" tagx="fpage" title="fpage">809</span> <span class="lpage" tagx="lpage" title="lpage">818</span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/14963099">14963099</a> </span> <span class="pub-id"> <a href="https://doi.org/10.1093/molbev/msh075">10.1093/molbev/msh075</a> </span> </div> </li> <li tag="ref"> <a name="B58" /> <div class="citation" tagx="citation" title="citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Butterfield</span> <span class="given-names" tagx="given-names" title="given-names">NJ</span> </span> </span> <div class="article-title" title="article-title"> <i>Bangiomorpha pubescens </i>n. gen., n. sp.: implications for the evolution of sex, multicellularity, and the Mesoproterozoic/Neoproterozoic radiation of eukaryotes </div> <span class="source" tagx="source" title="source">Paleobiology</span> <span class="year" tagx="year" title="year">2000</span> <span class="volume" tagx="volume" title="volume">26</span> <span class="fpage" tagx="fpage" title="fpage">386</span> <span class="lpage" tagx="lpage" title="lpage">404</span> <span class="pub-id"> <a href="https://doi.org/10.1666/0094-8373(2000)026&lt;0386:BPNGNS&gt;2.0.CO;2">10.1666/0094-8373(2000)026&amp;lt;0386:BPNGNS&amp;gt;2.0.CO;2</a> </span> </div> </li> <li tag="ref"> <a name="B59" /> <div class="citation" tagx="citation" title="citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Schulmeister</span> <span class="given-names" tagx="given-names" title="given-names">S</span> </span> </span> <div class="article-title" title="article-title">
Inconsistency of maximum parsimony revisited</div> <span class="source" tagx="source" title="source">Syst Biol</span> <span class="year" tagx="year" title="year">2004</span> <span class="volume" tagx="volume" title="volume">53</span> <span class="fpage" tagx="fpage" title="fpage">521</span> <span class="lpage" tagx="lpage" title="lpage">528</span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/15371243">15371243</a> </span> <span class="pub-id"> <a href="https://doi.org/10.1080/10635150490445788">10.1080/10635150490445788</a> </span> </div> </li> <li tag="ref"> <a name="B60" /> <div class="citation" tagx="citation" title="citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Arimura</span> <span class="given-names" tagx="given-names" title="given-names">S</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Yamamoto</span> <span class="given-names" tagx="given-names" title="given-names">J</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Aida</span> <span class="given-names" tagx="given-names" title="given-names">GP</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Nakazono</span> <span class="given-names" tagx="given-names" title="given-names">M</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Tsutsumi</span> <span class="given-names" tagx="given-names" title="given-names">N</span> </span> </span> <div class="article-title" title="article-title">
Frequent fusion and fission of plant mitochondria with unequal nucleoid distribution</div> <span class="source" tagx="source" title="source">Proc Natl Acad Sci USA</span> <span class="year" tagx="year" title="year">2004</span> <span class="volume" tagx="volume" title="volume">101</span> <span class="fpage" tagx="fpage" title="fpage">7805</span> <span class="lpage" tagx="lpage" title="lpage">7808</span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/15136720">15136720</a> </span> <span class="pub-id"> <a href="https://doi.org/10.1073/pnas.0401077101">10.1073/pnas.0401077101</a> </span> </div> </li> <li tag="ref"> <a name="B61" /> <div class="citation" tagx="citation" title="citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Sheahan</span> <span class="given-names" tagx="given-names" title="given-names">MB</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">McCurdy</span> <span class="given-names" tagx="given-names" title="given-names">DW</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Rose</span> <span class="given-names" tagx="given-names" title="given-names">RJ</span> </span> </span> <div class="article-title" title="article-title">
Mitochondria as a connected population:ensuring continuity of the mitochondrial genome during plant cell dedifferentiation through massive mitochondrial fusion </div> <span class="source" tagx="source" title="source">Plant J</span> <span class="year" tagx="year" title="year">2005</span> <span class="volume" tagx="volume" title="volume">44</span> <span class="fpage" tagx="fpage" title="fpage">744</span> <span class="lpage" tagx="lpage" title="lpage">755</span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/16297067">16297067</a> </span> </div> </li> <li tag="ref"> <a name="B62" /> <div class="citation" tagx="citation" title="citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Mohapatra</span> <span class="given-names" tagx="given-names" title="given-names">T</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Kirti</span> <span class="given-names" tagx="given-names" title="given-names">PB</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Kumar</span> <span class="given-names" tagx="given-names" title="given-names">VD</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Prakash</span> <span class="given-names" tagx="given-names" title="given-names">S</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Chopra</span> <span class="given-names" tagx="given-names" title="given-names">VL</span> </span> </span> <div class="article-title" title="article-title">
Random chloroplast segregation and mitochondrial genome recombination in somatic: hybrid plants of <i>Diplotaxis catholica </i>+ <i>Brassica juncea</i> </div> <span class="source" tagx="source" title="source">Plant Cell Rep</span> <span class="year" tagx="year" title="year">1998</span> <span class="volume" tagx="volume" title="volume">17</span> <span class="fpage" tagx="fpage" title="fpage">814</span> <span class="lpage" tagx="lpage" title="lpage">818</span> <span class="pub-id"> <a href="https://doi.org/10.1007/s002990050489">10.1007/s002990050489</a> </span> </div> </li> <li tag="ref"> <a name="B63" /> <div class="citation" tagx="citation" title="citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Kanno</span> <span class="given-names" tagx="given-names" title="given-names">A</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Kanzaki</span> <span class="given-names" tagx="given-names" title="given-names">H</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Kameya</span> <span class="given-names" tagx="given-names" title="given-names">T</span> </span> </span> <div class="article-title" title="article-title">
Detailed analyses of chloroplast and mitochondrial DNAs from the hybrid plant generated by asymmetric protoplast fusion between radish and cabbage </div> <span class="source" tagx="source" title="source">Plant Cell Rep</span> <span class="year" tagx="year" title="year">1997</span> <span class="volume" tagx="volume" title="volume">16</span> <span class="fpage" tagx="fpage" title="fpage">479</span> <span class="lpage" tagx="lpage" title="lpage">484</span> </div> </li> <li tag="ref"> <a name="B64" /> <div class="citation" tagx="citation" title="citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Boguski</span> <span class="given-names" tagx="given-names" title="given-names">MS</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Lowe</span> <span class="given-names" tagx="given-names" title="given-names">TM</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Tolstoshev</span> <span class="given-names" tagx="given-names" title="given-names">CM</span> </span> </span> <div class="article-title" title="article-title">
dbEST – database for "expressed sequence tags"</div> <span class="source" tagx="source" title="source">Nat Genet</span> <span class="year" tagx="year" title="year">1993</span> <span class="volume" tagx="volume" title="volume">4</span> <span class="fpage" tagx="fpage" title="fpage">332</span> <span class="lpage" tagx="lpage" title="lpage">333</span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/8401577">8401577</a> </span> <span class="pub-id"> <a href="https://doi.org/10.1038/ng0893-332">10.1038/ng0893-332</a> </span> </div> </li> <li tag="ref"> <a name="B65" /> <div class="citation" tagx="citation" title="citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Altschul</span> <span class="given-names" tagx="given-names" title="given-names">SF</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Madden</span> <span class="given-names" tagx="given-names" title="given-names">TL</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Schaffer</span> <span class="given-names" tagx="given-names" title="given-names">AA</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Zhang</span> <span class="given-names" tagx="given-names" title="given-names">J</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Zhang</span> <span class="given-names" tagx="given-names" title="given-names">Z</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Miller</span> <span class="given-names" tagx="given-names" title="given-names">W</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Lipman</span> <span class="given-names" tagx="given-names" title="given-names">DJ</span> </span> </span> <div class="article-title" title="article-title">
Gapped BLAST and PSI-BLAST: a new generation of protein database search programs</div> <span class="source" tagx="source" title="source">Nucleic Acids Res</span> <span class="year" tagx="year" title="year">1997</span> <span class="volume" tagx="volume" title="volume">25</span> <span class="fpage" tagx="fpage" title="fpage">3389</span> <span class="lpage" tagx="lpage" title="lpage">3402</span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/9254694">9254694</a> </span> <span class="pub-id"> <a href="https://doi.org/10.1093/nar/25.17.3389">10.1093/nar/25.17.3389</a> </span> </div> </li> <li tag="ref"> <a name="B66" /> <div class="citation" tagx="citation" title="citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Swofford</span> <span class="given-names" tagx="given-names" title="given-names">DL</span> </span> </span> <span class="source" tagx="source" title="source">PAUP*: Phylogenetic analysis using parsimony (* and other methods) Version 40b10</span> <span class="year" tagx="year" title="year">2003</span> <span class="publisher-name" tagx="publisher-name" title="publisher-name">Sunderland, Massachusetts: Sinauer Associates</span> </div> </li> <li tag="ref"> <a name="B67" /> <div class="citation" tagx="citation" title="citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Edgar</span> <span class="given-names" tagx="given-names" title="given-names">RC</span> </span> </span> <div class="article-title" title="article-title">
MUSCLE: multiple sequence alignment with high accuracy and high throughput</div> <span class="source" tagx="source" title="source">Nucleic Acids Res</span> <span class="year" tagx="year" title="year">2004</span> <span class="volume" tagx="volume" title="volume">32</span> <span class="fpage" tagx="fpage" title="fpage">1792</span> <span class="lpage" tagx="lpage" title="lpage">1797</span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/15034147">15034147</a> </span> <span class="pub-id"> <a href="https://doi.org/10.1093/nar/gkh340">10.1093/nar/gkh340</a> </span> </div> </li> <li tag="ref"> <a name="B68" /> <div class="citation" tagx="citation" title="citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Posada</span> <span class="given-names" tagx="given-names" title="given-names">D</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Crandall</span> <span class="given-names" tagx="given-names" title="given-names">KA</span> </span> </span> <div class="article-title" title="article-title">
MODELTEST: testing the model of DNA substitution</div> <span class="source" tagx="source" title="source">Bioinformatics</span> <span class="year" tagx="year" title="year">1998</span> <span class="volume" tagx="volume" title="volume">14</span> <span class="fpage" tagx="fpage" title="fpage">817</span> <span class="lpage" tagx="lpage" title="lpage">818</span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/9918953">9918953</a> </span> <span class="pub-id"> <a href="https://doi.org/10.1093/bioinformatics/14.9.817">10.1093/bioinformatics/14.9.817</a> </span> </div> </li> <li tag="ref"> <a name="B69" /> <div class="citation" tagx="citation" title="citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Cho</span> <span class="given-names" tagx="given-names" title="given-names">Y</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Mower</span> <span class="given-names" tagx="given-names" title="given-names">JP</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Qiu</span> <span class="given-names" tagx="given-names" title="given-names">YL</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Palmer</span> <span class="given-names" tagx="given-names" title="given-names">JD</span> </span> </span> <div class="article-title" title="article-title">
Mitochondrial substitution rates are extraordinarily elevated and variable in a genus of flowering plants </div> <span class="source" tagx="source" title="source">Proc Natl Acad Sci USA</span> <span class="year" tagx="year" title="year">2004</span> <span class="volume" tagx="volume" title="volume">101</span> <span class="fpage" tagx="fpage" title="fpage">17741</span> <span class="lpage" tagx="lpage" title="lpage">17746</span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/15598738">15598738</a> </span> <span class="pub-id"> <a href="https://doi.org/10.1073/pnas.0408302101">10.1073/pnas.0408302101</a> </span> </div> </li> <li tag="ref"> <a name="B70" /> <div class="citation" tagx="citation" title="citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Doyle</span> <span class="given-names" tagx="given-names" title="given-names">JJ</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Doyle</span> <span class="given-names" tagx="given-names" title="given-names">JL</span> </span> </span> <div class="article-title" title="article-title">
A rapid DNA isolation procedure for small quantitiesof fresh leaf tissue</div> <span class="source" tagx="source" title="source">Phytochem Bull</span> <span class="year" tagx="year" title="year">1987</span> <span class="volume" tagx="volume" title="volume">19</span> <span class="fpage" tagx="fpage" title="fpage">11</span> <span class="lpage" tagx="lpage" title="lpage">15</span> </div> </li> <li tag="ref"> <a name="B71" /> <div class="citation" tagx="citation" title="citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Whelan</span> <span class="given-names" tagx="given-names" title="given-names">S</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Goldman</span> <span class="given-names" tagx="given-names" title="given-names">N</span> </span> </span> <div class="article-title" title="article-title">
A general empirical model of protein evolution derived from multiple protein families using a maximum-likelihood approach </div> <span class="source" tagx="source" title="source">Mol Biol Evol</span> <span class="year" tagx="year" title="year">2001</span> <span class="volume" tagx="volume" title="volume">18</span> <span class="fpage" tagx="fpage" title="fpage">691</span> <span class="lpage" tagx="lpage" title="lpage">699</span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/11319253">11319253</a> </span> </div> </li> <li tag="ref"> <a name="B72" /> <div class="citation" tagx="citation" title="citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Adachi</span> <span class="given-names" tagx="given-names" title="given-names">J</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Waddell</span> <span class="given-names" tagx="given-names" title="given-names">PJ</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Martin</span> <span class="given-names" tagx="given-names" title="given-names">W</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Hasegawa</span> <span class="given-names" tagx="given-names" title="given-names">M</span> </span> </span> <div class="article-title" title="article-title">
Plastid genome phylogeny anda model of amino acid substitution for proteins encoded by chloroplast DNA </div> <span class="source" tagx="source" title="source">J Mol Evol</span> <span class="year" tagx="year" title="year">2000</span> <span class="volume" tagx="volume" title="volume">50</span> <span class="fpage" tagx="fpage" title="fpage">348</span> <span class="lpage" tagx="lpage" title="lpage">358</span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/10795826">10795826</a> </span> </div> </li> </ul> </div> <div class="display-objects" title="display-objects"> <div class="title" tagx="title" title="title">
Figures and Tables</div> <div class="fig" title="fig"> <div class="figure" title="figure"> <a href="https://doi.org/"> <span class="label" tagx="label" title="label">Figure 1</span> </a> </div>   <p> <b> <i>rpl36 </i>tree and alignment </b>. The M3 codon model in MrBayes was used to calculate the tree using the alignment shown. Nodes with posterior probability &amp;lt;0.95 are collapsed. Posterior probabilities (left) and PROML BP values &amp;gt;50% (right) are shown on the remaining nodes. The PROML bootstraps were run with four rate categories (estimated with PUZZLE) and global rearrangements. Nucleotide and amino-acid based ML analyses using PAUP* and MrBayes also gave 100% support for the division between the c-type and p-type <i>rpl36 </i>genes. This support is maintained when all positions containing gaps are removed. Because the 3' extension unique to some c-type <i>rpl36 </i>genes (see <a href="#S2">Additional File 2</a>) was excluded from this phylogenetic analysis, it is not shown in the alignment. In the alignment, each base is colored according to the key. Taxa in red include the red algae and their secondary plastid containing relatives. A subset of the many proteobacterial species which contain both the p-type and c-type genes is shown in purple. The p-type <i>Pseudomonas</i>, <i>Photobacterium</i>, and <i>Vibrio </i>genes are not shown here. </p>   </div> <div class="fig" title="fig"> <div class="figure" title="figure"> <a href="https://doi.org/"> <span class="label" tagx="label" title="label">Figure 2</span> </a> </div>   <p> <b> <i>clpP </i>phylogeny before and after taxon addition </b>. ML analysis was performed on an all-position nucleotide alignment using PAUP* as described in Methods with the TVM+G model used for both trees. A 60-bp 3' extension with questionable homology across taxa was removed in this analysis but was included in the original analysis. This is probably responsible for the change in the BP from 84% originally (not shown) to 70% here for <i>Oenothera </i>going with grasses in tree <b>(A)</b>. Bootstrap values &amp;lt;50% are not shown. <b>(A) </b>Original taxon sampling; <b>(B) </b>after new taxa added. </p>   </div> <div class="fig" title="fig"> <div class="figure" title="figure"> <a href="https://doi.org/"> <span class="label" tagx="label" title="label">Figure 3</span> </a> </div>   <p> <b>Error in the published <i>Oenothera elata ycf2</i> </b>. <b>(A) </b>Alignment of <i>ycf2 </i>nucleotide sequences: The top two sequences, <i>Oenothera biennis </i>and <i>O. elata</i>, were sequenced as part of this study. We did not sequence the first ~1600 bp. The bottom two sequences correspond to the published <i>Oenothera elata </i>and <i>Nicotiana tabacum </i>sequences. The bottom three sequences were used to determine a consensus base at each position, and positions that did not match this consensus are colored as denoted in the key. <b>(B) </b>All codon position ML tree using the TVM+G model in PAUP* with 100 bootstrap replicates. Only the 3' region of <i>ycf2 </i>starting at position 4023 of the published <i>Oenothera </i>sequence [31] was obtained for the four Myrtales taxa ( <i>Eucalyptus</i>, <i>Fuchsia</i>, <i>Clarkia </i>and <i>Epilobium</i>) and the analysis was performed using this region of aligned positions as indicated in <b>(A)</b>. Within this region, gappy positions were removed prior to phylogenetic analysis, which resulted in 2567 positions. When the entire gene was used with the published <i>Oenothera </i>sequence excluded, the topology was the same except that the <i>Lotus </i>and <i>Arabidopsis </i>branches were switched. When the published <i>Oenothera </i>is included in the full-length analysis, its strong chimerism pulled <i>Clarkia</i>, <i>Epilobium </i>and our <i>elata </i>sequence into an artifactual clade with the published <i>elata </i>gene at the base of the Solanaceae. </p>   </div> <div class="fig" title="fig"> <div class="figure" title="figure"> <a href="https://doi.org/"> <span class="label" tagx="label" title="label">Figure 4</span> </a> </div>   <p> <b>Phylogenetic tree of red-like and green-like <i>rbcL </i>sequences </b>. The amino-acid Bayesian tree was generated using MrBayes with the following parameters: <i>rates </i>= invgamma; <i>aamodelpr </i>= mixed; <i>ngen </i>= 500000; <i>nchains </i>= 4. The <i>burnin </i>was set to 100 to generate the tree and this <i>burnin </i>gave a convergence diagnostic of 0.017. The nodal support values are PROML bootstrap support values obtained using global rearrangements, and four rate categories and an invariant category estimated using PUZZLE. Support values are shown on nodes with BP ≥ 50. </p>   </div> </div> </div>  </body></html>