<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<link rel="dns-prefetch" href="//d33xdlntwy0kbs.cloudfront.net" />
<link rel="shortcut icon" href="https://www.biorxiv.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<meta name="article_thumbnail" content="https://www.biorxiv.org/content/biorxiv/early/2019/09/20/776708/embed/graphic-9.gif" />
<meta name="type" content="article" />
<meta name="category" content="article" />
<meta name="HW.identifier" content="/biorxiv/early/2019/09/20/776708.atom" />
<meta name="HW.pisa" content="biorxiv;776708v1" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="The Evolution of Social Organization: Climate Influences Variation in Drosophilid Social Networks" />
<meta name="DC.Identifier" content="10.1101/776708" />
<meta name="DC.Date" content="2019-09-20" />
<meta name="DC.Publisher" content="Cold Spring Harbor Laboratory" />
<meta name="DC.Rights" content="© 2019, Posted by Cold Spring Harbor Laboratory. The copyright holder for this pre-print is the author. All rights reserved. The material may not be redistributed, re-used or adapted without the author's permission." />
<meta name="DC.AccessRights" content="restricted" />
<meta name="DC.Description" content="Cultural norms, collective decisions, reproductive behaviour, and pathogen transmission all emerge from interaction patterns within animal social groups. These patterns of interaction support group-level phenomena that can influence an individual’s fitness. The aim of this study is to understand the evolution of social organization in Drosophila. Using a comparative ecological, phylogenetic and behavioural approach, we studied the different properties of social interaction networks (SINs) formed by 20 drosophilids and the different ways these species interact. We investigate whether animal network structures arise from common ancestry, a response to the species’ past ecological environment, other social behaviours, or a combination of these factors. We demonstrate that differences in past climate predicted the species’ current SIN properties. The drosophilid phylogeny offered no value to predicting species’ differences in SINs through phylogenetic signal tests. This suggests group-level social behaviours in drosophilid species are shaped by divergent climates. However, we find that the distance at which flies interact correlated with the drosophilid phylogeny, indicating that behavioural elements that comprise SINs have remained largely unchanged in their recent evolutionary history. We find a significant correlation of leg length to social distance, outlining the interdependence of anatomy and complex social structures. Although SINs display a complex evolutionary relationship across drosophilids, this study provides evidence of selective pressures acting on social behaviour in Drosophila. We speculate that conserved molecular mechanisms may be shared across drosophilids deep in their evolutionary history, similar to other pervasive mechanisms, like biological clocks." />
<meta name="DC.Contributor" content="Jacob A. Jezovit" />
<meta name="DC.Contributor" content="Rebecca Rooke" />
<meta name="DC.Contributor" content="Jonathan Schneider" />
<meta name="DC.Contributor" content="Joel D. Levine" />
<meta name="article:published_time" content="2019-09-20" />
<meta name="article:section" content="New Results" />
<meta name="citation_title" content="The Evolution of Social Organization: Climate Influences Variation in Drosophilid Social Networks" />
<meta name="citation_abstract" lang="en" content="&lt;h3&gt;Summary&lt;/h3&gt;
&lt;p&gt;Cultural norms, collective decisions, reproductive behaviour, and pathogen transmission all emerge from interaction patterns within animal social groups. These patterns of interaction support group-level phenomena that can influence an individual’s fitness. The aim of this study is to understand the evolution of social organization in Drosophila. Using a comparative ecological, phylogenetic and behavioural approach, we studied the different properties of social interaction networks (SINs) formed by 20 drosophilids and the different ways these species interact. We investigate whether animal network structures arise from common ancestry, a response to the species’ past ecological environment, other social behaviours, or a combination of these factors. We demonstrate that differences in past climate predicted the species’ current SIN properties. The drosophilid phylogeny offered no value to predicting species’ differences in SINs through phylogenetic signal tests. This suggests group-level social behaviours in drosophilid species are shaped by divergent climates. However, we find that the distance at which flies interact correlated with the drosophilid phylogeny, indicating that behavioural elements that comprise SINs have remained largely unchanged in their recent evolutionary history. We find a significant correlation of leg length to social distance, outlining the interdependence of anatomy and complex social structures. Although SINs display a complex evolutionary relationship across drosophilids, this study provides evidence of selective pressures acting on social behaviour in Drosophila. We speculate that conserved molecular mechanisms may be shared across drosophilids deep in their evolutionary history, similar to other pervasive mechanisms, like biological clocks.&lt;/p&gt;" />
<meta name="citation_journal_title" content="bioRxiv" />
<meta name="citation_publisher" content="Cold Spring Harbor Laboratory" />
<meta name="citation_publication_date" content="2019/01/01" />
<meta name="citation_mjid" content="biorxiv;776708v1" />
<meta name="citation_id" content="776708v1" />
<meta name="citation_public_url" content="https://www.biorxiv.org/content/10.1101/776708v1" />
<meta name="citation_abstract_html_url" content="https://www.biorxiv.org/content/10.1101/776708v1.abstract" />
<meta name="citation_full_html_url" content="https://www.biorxiv.org/content/10.1101/776708v1.full" />
<meta name="citation_pdf_url" content="https://www.biorxiv.org/content/biorxiv/early/2019/09/20/776708.full.pdf" />
<meta name="citation_doi" content="10.1101/776708" />
<meta name="citation_num_pages" content="51" />
<meta name="citation_article_type" content="Article" />
<meta name="citation_section" content="New Results" />
<meta name="citation_firstpage" content="776708" />
<meta name="citation_author" content="Jacob A. Jezovit" />
<meta name="citation_author_institution" content="Department of Biology, University of Toronto at Mississauga" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0003-3037-5738" />
<meta name="citation_author" content="Rebecca Rooke" />
<meta name="citation_author_institution" content="Department of Biology, University of Toronto at Mississauga" />
<meta name="citation_author" content="Jonathan Schneider" />
<meta name="citation_author_institution" content="Department of Biology, University of Toronto at Mississauga" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0003-0468-0763" />
<meta name="citation_author" content="Joel D. Levine" />
<meta name="citation_author_institution" content="Department of Biology, University of Toronto at Mississauga" />
<meta name="citation_author_email" content="joel.levine@utoronto.ca" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0002-6254-6274" />
<meta name="citation_reference" content="Scott, J.P. (1956). The analysis of social organization in animals. Ecology 37, 213–221." />
<meta name="citation_reference" content="Krause, J., and Ruxton, G.D. (2002). Living in groups, (Oxford University Press)." />
<meta name="citation_reference" content="Drewe, J.A. (2009). Who infects whom? Social networks and tuberculosis transmission in wild meerkats. Proceedings of the Royal Society B: Biological Sciences 277, 633–642." />
<meta name="citation_reference" content="Oh, K.P., and Badyaev, A.V. (2010). Structure of social networks in a passerine bird: consequences for sexual selection and the evolution of mating strategies. The American Naturalist 176, E80–E89." />
<meta name="citation_reference" content="Pasquaretta, C., Battesti, M., Klenschi, E., Bousquet, C.A., Sueur, C., and Mery, F. (2016). How social network structure affects decision-making in Drosophila melanogaster. Proc Biol Sci 283, 20152954." />
<meta name="citation_reference" content="Battesti, M., Moreno, C., Joly, D., and Mery, F. (2012). Spread of social information and dynamics of social transmission within Drosophila groups. Curr Biol 22, 309–313." />
<meta name="citation_reference" content="Sarin, S., and Dukas, R. (2009). Social learning about egg-laying substrates in fruitflies. Proc Biol Sci 276, 4323–4328." />
<meta name="citation_reference" content="Danchin, E., Nöbel, S., Pocheville, A., Dagaeff, A.C., Demay, L., Alphand, M., Ranty-Roby, S., van Renssen, L., Monier, M., Gazagne, E., et al. (2018). Cultural flies: Conformist social learning in fruitflies predicts long-lasting mate-choice traditions. Science 362, 1025–1030." />
<meta name="citation_reference" content="Lihoreau, M., Poissonnier, L.A., Isabel, G., and Dussutour, A. (2016). Drosophila females trade off good nutrition with high-quality oviposition sites when choosing foods. J Exp Biol 219, 2514–2524." />
<meta name="citation_reference" content="Kacsoh, B.Z., Bozler, J., Ramaswami, M., and Bosco, G. (2015). Social communication of predator-induced changes in Drosophila behavior and germ line physiology. Elife 4." />
<meta name="citation_reference" content="Kacsoh, B.Z., Bozler, J., and Bosco, G. (2018). Correction: Drosophila species learn dialects through communal living. PLoS Genet 14, e1007825." />
<meta name="citation_reference" content="Ramdya, P., Lichocki, P., Cruchet, S., Frisch, L., Tse, W., Floreano, D., and Benton, R. (2015). Mechanosensory interactions drive collective behaviour in Drosophila. Nature 519, 233–236." />
<meta name="citation_reference" content="Schneider, J., Dickinson, M.H., and Levine, J.D. (2012). Social structures depend on innate determinants and chemosensory processing in Drosophila. Proc Natl Acad Sci U S A 109 Suppl 2, 17174–17179." />
<meta name="citation_reference" content="Newman, M. (2010). Networks: an introduction, (Oxford University Press)." />
<meta name="citation_reference" content="Fowler, J.H., Dawes, C.T., and Christakis, N.A. (2009). Model of genetic variation in human social networks. Proc Natl Acad Sci U S A 106, 1720–1724." />
<meta name="citation_reference" content="Jezovit, J.A., Levine, J.D., and Schneider, J. (2017). Phylogeny, environment and sexual communication across the Drosophila genus. J Exp Biol 220, 42–52." />
<meta name="citation_reference" content="Markow, T.A., and O’Grady, P.M. (2006). Drosophila: A guide to species identification and use, (San Diego, California, USA: Elsevier, Inc)." />
<meta name="citation_reference" content="Foster, E.A., Franks, D.W., Morrell, L.J., Balcomb, K.C., Parsons, K.M., van Ginneken, A., and Croft, D.P. (2012). Social network correlates of food availability in an endangered population of killer whales, Orcinus orca. Animal Behaviour 83, 731–736." />
<meta name="citation_reference" content="Kelley, J.L., Morrell, L.J., Inskip, C., Krause, J., and Croft, D.P. (2011). Predation risk shapes social networks in fission-fusion populations. PloS one 6, e24280." />
<meta name="citation_reference" content="Grimaldi, D.A. (1987). Amber fossil Drosophilidae (Diptera), with particular reference to the Hispaniolan taxa. American museum novitates (USA)." />
<meta name="citation_reference" content="van der Linde, K., Houle, D., Spicer, G.S., and Steppan, S.J. (2010). A supermatrix-based molecular phylogeny of the family Drosophilidae. Genetics research 92, 25–38." />
<meta name="citation_reference" content="Kellermann, V., Loeschcke, V., Hoffmann, A.A., Kristensen, T.N., Fløjgaard, C., David, J.R., Svenning, J.C., and Overgaard, J. (2012). Phylogenetic constraints in key functional traits behind species’ climate niches: patterns of desiccation and cold resistance across 95 Drosophila species. Evolution 66, 3377–3389." />
<meta name="citation_reference" content="Kellermann, V., Overgaard, J., Hoffmann, A.A., Fløjgaard, C., Svenning, J.C., and Loeschcke, V. (2012). Upper thermal limits of Drosophila are linked to species distributions and strongly constrained phylogenetically. Proc Natl Acad Sci U S A 109, 16228–16233." />
<meta name="citation_reference" content="Blomberg, S.P., Garland, T., and Ives, A.R. (2003). Testing for phylogenetic signal in comparative data: behavioral traits are more labile. Evolution 57, 717–745." />
<meta name="citation_reference" content="Pagel, M. (1999). Inferring the historical patterns of biological evolution. Nature 401, 877–884." />
<meta name="citation_reference" content="Hijmans, R.J., Cameron, S.E., Parra, J.L., Jones, P.G., and Jarvis, A. (2005). Very high resolution interpolated climate surfaces for global land areas. International Journal of Climatology: A Journal of the Royal Meteorological Society 25, 1965–1978." />
<meta name="citation_reference" content="Ives, A.R., Midford, P.E., and Garland Jr, T. (2007). Within-species variation and measurement error in phylogenetic comparative methods. Systematic Biology 56, 252–270." />
<meta name="citation_reference" content="Durisko, Z., Kemp, R., Mubasher, R., and Dukas, R. (2014). Dynamics of social behavior in fruit fly larvae. PLoS One 9, e95495." />
<meta name="citation_reference" content="Lihoreau, M., Clarke, I.M., Buhl, J., Sumpter, D.J., and Simpson, S.J. (2016). Collective selection of food patches in Drosophila. J Exp Biol 219, 668–675." />
<meta name="citation_reference" content="Simon, A.F., Chou, M.T., Salazar, E.D., Nicholson, T., Saini, N., Metchev, S., and Krantz, D.E. (2012). A simple assay to study social behavior in Drosophila: measurement of social space within a group. Genes Brain Behav 11, 243–252." />
<meta name="citation_reference" content="Vosshall, L.B., and Stocker, R.F. (2007). Molecular architecture of smell and taste in Drosophila. Annu Rev Neurosci 30, 505–533." />
<meta name="citation_reference" content="Bontonou, G., and Wicker-Thomas, C. (2014). Sexual Communication in the Drosophila Genus. Insects 5, 439–458." />
<meta name="citation_reference" content="Ferveur, J.F. (2005). Cuticular hydrocarbons: their evolution and roles in Drosophila pheromonal communication. Behav Genet 35, 279–295." />
<meta name="citation_reference" content="Krupp, J.J., Kent, C., Billeter, J.C., Azanchi, R., So, A.K., Schonfeld, J.A., Smith, B.P., Lucas, C., and Levine, J.D. (2008). Social experience modifies pheromone expression and mating behavior in male Drosophila melanogaster. Curr Biol 18, 1373–1383." />
<meta name="citation_reference" content="Billeter, J.C., Atallah, J., Krupp, J.J., Millar, J.G., and Levine, J.D. (2009). Specialized cells tag sexual and species identity in Drosophila melanogaster. Nature 461, 987–991." />
<meta name="citation_reference" content="Sexton, O.J., and Stalker, H.D. (1961). Spacing patterns of female Drosophila paramelanica. Animal Behaviour 9, 77–81." />
<meta name="citation_reference" content="Bergallo, H.G., and Magnusson, W.E. (1999). Effects of climate and food availability on four rodent species in southeastern Brazil. Journal of Mammalogy 80, 472–486." />
<meta name="citation_reference" content="Craine, J.M. (2013). Long-term climate sensitivity of grazer performance: a cross-site study. PLoS One 8, e67065." />
<meta name="citation_reference" content="Currie, D.J., Mittelbach, G.G., Cornell, H.V., Field, R., Guégan, J.F., Hawkins, B.A., Kaufman, D.M., Kerr, J.T., Oberdorff, T., and O’Brien, E. (2004). Predictions and tests of climate-based hypotheses of broad-scale variation in taxonomic richness. Ecology letters 7, 1121–1134." />
<meta name="citation_reference" content="McInerny, M.C., and Cross, T.K. (1999). Effects of lake productivity, climate warming, and intraspecific density on growth and growth patterns of black crappie in southern Minnesota lakes. Journal of Freshwater Ecology 14, 255–264." />
<meta name="citation_reference" content="Sánchez-Bayo, F., and Wyckhuys, K.A. (2019). Worldwide decline of the entomofauna: A review of its drivers. Biological conservation 232, 8–27." />
<meta name="citation_reference" content="Hallmann, C.A., Sorg, M., Jongejans, E., Siepel, H., Hofland, N., Schwan, H., Stenmans, W., Müller, A., Sumser, H., and Hörren, T. (2017). More than 75 percent decline over 27 years in total flying insect biomass in protected areas. PloS one 12, e0185809." />
<meta name="citation_reference" content="Krause, J., James, R., Franks, D.W., and Croft, D.P. (2015). Animal social networks, (Oxford University Press, USA)." />
<meta name="citation_reference" content="Barocas, A., Ilany, A., Koren, L., Kam, M., and Geffen, E. (2011). Variance in centrality within rock hyrax social networks predicts adult longevity. PLoS One 6, e22375." />
<meta name="citation_reference" content="Ma, S., Avanesov, A.S., Porter, E., Lee, B.C., Mariotti, M., Zemskaya, N., Guigo, R., Moskalev, A.A., and Gladyshev, V.N. (2018). Comparative transcriptomics across 14 Drosophila species reveals signatures of longevity. Aging cell 17, e12740." />
<meta name="citation_reference" content="Matos, R.J., and Schlupp, I. (2005). Performing in front of an audience: signalers and the social environment. Animal communication networks, 63–83." />
<meta name="citation_reference" content="Markow, T.A. (2015). The secret lives of Drosophila flies. Elife 4." />
<meta name="citation_reference" content="Dickinson, M.H. (2014). Death valley, Drosophila, and the Devonian toolkit. Annual review of entomology 59, 51–72." />
<meta name="citation_reference" content="Peel, M.C., Finlayson, B.L., and McMahon, T.A. (2007). Updated world map of the Köppen-Geiger climate classification. Hydrology and earth system sciences discussions 4, 439–473." />
<meta name="citation_reference" content="Schneider, J., and Levine, J.D. (2014). Automated identification of social interaction criteria in Drosophila melanogaster. Biol Lett 10, 20140749." />
<meta name="citation_reference" content="Paradis, E., and Schliep, K. (2018). ape 5.0: an environment for modern phylogenetics and evolutionary analyses in R. Bioinformatics 35, 526–528." />
<meta name="citation_reference" content="Revell, L.J. (2012). phytools: an R package for phylogenetic comparative biology (and other things). Methods in Ecology and Evolution 3, 217–223." />
<meta name="citation_reference" content="Felsenstein, J. (1985). Phylogenies and the comparative method. The American Naturalist 125, 1–15." />
<meta name="citation_reference" content="Revell, L.J., Harmon, L.J., and Collar, D.C. (2008). Phylogenetic signal, evolutionary process, and rate. Syst Biol 57, 591–601." />
<meta name="citation_reference" content="Spicer, G.S., and Bell, C. (2002). Molecular phylogeny of the Drosophila virilis species group (Diptera: Drosophilidae) inferred from mitochondrial 12S and 16S ribosomal RNA genes. Annals of the Entomological Society of America 95, 156–161." />
<meta name="citation_reference" content="Sandweiss, D.H., Maasch, K.A., and Anderson, D.G. (1999). Transitions in the mid-Holocene. Science 283, 499–500." />
<meta name="citation_reference" content="Revell, L.J. (2013). Two new graphical methods for mapping trait evolution on phylogenies. Methods in Ecology and Evolution 4, 754–759." />
<meta name="twitter:title" content="The Evolution of Social Organization: Climate Influences Variation in Drosophilid Social Networks" />
<meta name="twitter:site" content="@biorxivpreprint" />
<meta name="twitter:card" content="summary" />
<meta name="twitter:image" content="https://www.biorxiv.org/sites/default/files/images/biorxiv_logo_homepage7-5-small.png" />
<meta name="twitter:description" content="Cultural norms, collective decisions, reproductive behaviour, and pathogen transmission all emerge from interaction patterns within animal social groups. These patterns of interaction support group-level phenomena that can influence an individual’s fitness. The aim of this study is to understand the evolution of social organization in Drosophila. Using a comparative ecological, phylogenetic and behavioural approach, we studied the different properties of social interaction networks (SINs) formed by 20 drosophilids and the different ways these species interact. We investigate whether animal network structures arise from common ancestry, a response to the species’ past ecological environment, other social behaviours, or a combination of these factors. We demonstrate that differences in past climate predicted the species’ current SIN properties. The drosophilid phylogeny offered no value to predicting species’ differences in SINs through phylogenetic signal tests. This suggests group-level social behaviours in drosophilid species are shaped by divergent climates. However, we find that the distance at which flies interact correlated with the drosophilid phylogeny, indicating that behavioural elements that comprise SINs have remained largely unchanged in their recent evolutionary history. We find a significant correlation of leg length to social distance, outlining the interdependence of anatomy and complex social structures. Although SINs display a complex evolutionary relationship across drosophilids, this study provides evidence of selective pressures acting on social behaviour in Drosophila. We speculate that conserved molecular mechanisms may be shared across drosophilids deep in their evolutionary history, similar to other pervasive mechanisms, like biological clocks." />
<meta name="og-title" property="og:title" content="The Evolution of Social Organization: Climate Influences Variation in Drosophilid Social Networks" />
<meta name="og-url" property="og:url" content="https://www.biorxiv.org/content/10.1101/776708v1" />
<meta name="og-site-name" property="og:site_name" content="bioRxiv" />
<meta name="og-description" property="og:description" content="Cultural norms, collective decisions, reproductive behaviour, and pathogen transmission all emerge from interaction patterns within animal social groups. These patterns of interaction support group-level phenomena that can influence an individual’s fitness. The aim of this study is to understand the evolution of social organization in Drosophila. Using a comparative ecological, phylogenetic and behavioural approach, we studied the different properties of social interaction networks (SINs) formed by 20 drosophilids and the different ways these species interact. We investigate whether animal network structures arise from common ancestry, a response to the species’ past ecological environment, other social behaviours, or a combination of these factors. We demonstrate that differences in past climate predicted the species’ current SIN properties. The drosophilid phylogeny offered no value to predicting species’ differences in SINs through phylogenetic signal tests. This suggests group-level social behaviours in drosophilid species are shaped by divergent climates. However, we find that the distance at which flies interact correlated with the drosophilid phylogeny, indicating that behavioural elements that comprise SINs have remained largely unchanged in their recent evolutionary history. We find a significant correlation of leg length to social distance, outlining the interdependence of anatomy and complex social structures. Although SINs display a complex evolutionary relationship across drosophilids, this study provides evidence of selective pressures acting on social behaviour in Drosophila. We speculate that conserved molecular mechanisms may be shared across drosophilids deep in their evolutionary history, similar to other pervasive mechanisms, like biological clocks." />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.biorxiv.org/sites/default/files/images/biorxiv_logo_homepage7-5-small.png" />
<meta name="citation_date" content="2019-09-20" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/10.1101/776708v1.full.pdf" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/10.1101/776708v1.full.txt" />
<link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/10.1101/776708v1.ppt" />
<meta name="description" content="bioRxiv - the preprint server for biology, operated by Cold Spring Harbor Laboratory, a research and educational institution" />
<meta name="generator" content="Drupal 7 (http://drupal.org)" />
<link rel="canonical" href="https://www.biorxiv.org/content/10.1101/776708v1" />
<link rel="shortlink" href="https://www.biorxiv.org/node/916114" />
    <title>The Evolution of Social Organization: Climate Influences Variation in Drosophilid Social Networks | bioRxiv</title>  
    <link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__jMRAK66KMC1e4TQlwUNn3KiWVDC5AjueUAYEm1xBY_U__KSUjdT4jdcJ4qJz6fKY1K9WkYh1a5EcaWZxt_-zbTis__QJacVmveFyLIBSPkp21xL0ZvuDW9DMBrD2HB1P4Ry5I.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q5p1rr" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__icrP-Y0qw2NP73rJxxkUtrJta0z3VLAABtNjfSdEghA__rlN3rx0wOoswGNHZMeZM96bNn-3cLWHXTztGDoThNlw__QJacVmveFyLIBSPkp21xL0ZvuDW9DMBrD2HB1P4Ry5I.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__q48SDxVDWut_cH6dTPyQe7HjADuSLLmJVFztuiJEFrU__DJqddQGjm5dXCHsaiwNriGytKN5eevIQqyjdq65TZtE__QJacVmveFyLIBSPkp21xL0ZvuDW9DMBrD2HB1P4Ry5I.css" media="all" />
<style type="text/css" media="all">
/* <![CDATA[ */
#sliding-popup.sliding-popup-bottom,#sliding-popup.sliding-popup-bottom .eu-cookie-withdraw-banner,.eu-cookie-withdraw-tab{background:gray}#sliding-popup.sliding-popup-bottom.eu-cookie-withdraw-wrapper{background:transparent}#sliding-popup .popup-content #popup-text h1,#sliding-popup .popup-content #popup-text h2,#sliding-popup .popup-content #popup-text h3,#sliding-popup .popup-content #popup-text p,.eu-cookie-compliance-secondary-button,.eu-cookie-withdraw-tab{color:#fff !important}.eu-cookie-withdraw-tab{border-color:#fff}.eu-cookie-compliance-more-button{color:#fff !important}
/* ]]> */
</style>

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__QJacVmveFyLIBSPkp21xL0ZvuDW9DMBrD2HB1P4Ry5I.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__B6tGdsQSt1vT8d8j-19-oS4FKJe83RrSNBkjPs9jfos__6VkE5WcQmmQpSVTQx-xS2-zn_nlwFlDm-EN1HLJNjrA__QJacVmveFyLIBSPkp21xL0ZvuDW9DMBrD2HB1P4Ry5I.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__I6pBwZrmO51afTZVOjN83FdGH1QRb9t15lSLso0JVV4__F3sVKlGyHKCaAPI_WdiHD7l_-05B6b9bLdUwzi1me90__QJacVmveFyLIBSPkp21xL0ZvuDW9DMBrD2HB1P4Ry5I.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__QJacVmveFyLIBSPkp21xL0ZvuDW9DMBrD2HB1P4Ry5I.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__QJacVmveFyLIBSPkp21xL0ZvuDW9DMBrD2HB1P4Ry5I.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__CRdjIL0HKDL2T9slYvkTSTLypBBPjyoDhwBezBuDy7c__cHebPmPNdg2jyPmpyQpZWssJQ4cPrqfaabp5Or9KlkU__QJacVmveFyLIBSPkp21xL0ZvuDW9DMBrD2HB1P4Ry5I.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://d33xdlntwy0kbs.cloudfront.net/cshl_custom.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/all/libraries/modernizr/modernizr.min.js?q5p1rr"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__yOhrJUBmMpN-jsuQlJYmS8ovkQU6I6KInr295HrSTb0__c--zHhbuOWmkdWkJYXiNWDmkpDRPesmbBjGlQcyqBOY__QJacVmveFyLIBSPkp21xL0ZvuDW9DMBrD2HB1P4Ry5I.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var _prum=[['id', '52e95df3abe53d2e33000000'], ['mark', 'firstbyte',
      (new Date()).getTime()]]; (function() {
      var s=document.getElementsByTagName('script')[0],
      p=document.createElement('script');
      p.async='async'; p.src='//rum-static.pingdom.net/prum.min.js';s.parentNode.insertBefore(p,s);})();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js___KyvuEvfmewWxdOLBXTYsSUo8Pfm_DijV6fWtGvSL_k__HSKkTgIr7nDcdvQYewipf6v4ak6ZLUOQkwPj44wKouA__QJacVmveFyLIBSPkp21xL0ZvuDW9DMBrD2HB1P4Ry5I.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__3uCzzYE5jr6ly9nvsM8PRBRkhh4p0NWCMDja4EKBy4M__dmGGVYcB8LhnbR2iAmVW0ZyXI6QuFVYFRORa5XxglXs__QJacVmveFyLIBSPkp21xL0ZvuDW9DMBrD2HB1P4Ry5I.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__0zdUME0GLauYYyEch2tGPydIw-IG7Uizip0Qb4wUsig__3RgszrCgk4bOnTzo4vaQ-hrWzaG6oD19l-R9CQ1YQ-o__QJacVmveFyLIBSPkp21xL0ZvuDW9DMBrD2HB1P4Ry5I.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__W6wAjopaALytcBam7oT3WhEzllkWaB9R-8hazHyvShM__rK5YBIqBy9qt0di0Sn4DriBx9TJupdmv6uGmLJyZvDs__QJacVmveFyLIBSPkp21xL0ZvuDW9DMBrD2HB1P4Ry5I.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-45638026-1", {"cookieDomain":"auto"});ga("set", "page", location.pathname + location.search + location.hash);ga("send", "pageview");ga('create', 'UA-189672-38', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__1PtAOZ8x7l6QAi8D7X6Kj_Y7HcUOFzvWRvvYHUcCT98__U8IB92u_CnnGwJ-X_xaUa9lzsC8OzUfN0WkK2JQfirg__QJacVmveFyLIBSPkp21xL0ZvuDW9DMBrD2HB1P4Ry5I.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings,{"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"SeI34RJu96Xp9gHkLirLc4Kr7m21RSp2o_ngj5UTM8g","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"modules\/comment\/comment.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"modules\/forum\/forum.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"sites\/default\/modules\/biorxiv\/css\/biorxiv.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/flexible\/flexible.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_collection\/css\/highwire_subject_collection.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire.highwire-list-expand.css":1,"public:\/\/ctools\/css\/bab59090150caf0c64e27afa1b36eef1.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/contrib\/eu_cookie_compliance\/css\/eu_cookie_compliance.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"0":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_biorxiv_styles\/css\/biorxiv-styles.css.less":1,"https:\/\/d33xdlntwy0kbs.cloudfront.net\/cshl_custom.css":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_figures.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_tables.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"http:\/\/d33xdlntwy0kbs.cloudfront.net\/cshl_custom.js":1,"sites\/all\/modules\/contrib\/eu_cookie_compliance\/js\/eu_cookie_compliance.js":1,"sites\/default\/modules\/jnl_biorxiv_styles\/js\/adv-search-collection-toggle.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/eu_cookie_compliance\/js\/jquery.cookie-1.4.1.min.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/default\/modules\/jnl_biorxiv_styles\/js\/biorxiv-scripts.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_nps_survey\/js\/nps_survey.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"sites\/default\/modules\/contrib\/disqus\/disqus.js":1,"misc\/textarea.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/contrib\/service_links\/js\/twitter_button.js":1,"sites\/all\/modules\/contrib\/service_links\/js\/facebook_like.js":1,"sites\/all\/modules\/contrib\/service_links\/js\/google_plus_one.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.highwire-list-expand.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"916114","apath":"\/biorxiv\/early\/2019\/09\/20\/776708.atom","pisa":"biorxiv;776708v1","processed":["highwire_math"],"markup":[{"requested":"full-text","variant":"full-text","view":"full","pisa":"biorxiv;776708v1"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"disqus":{"domain":"biorxivstage","url":"https:\/\/www.biorxiv.org\/content\/10.1101\/776708v1","title":"The Evolution of Social Organization: Climate Influences Variation in Drosophilid Social Networks","identifier":"node\/916114"},"panels_ajax_pane":{"new-41":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u00228L+qj\\\\\\\/U+7Sh+c3rOSJtHiE9WpvY4rc4GZPssZEohxq95s9n9h3um1NAkvID25aHguTqeY9SUhuNzsbdSQ5x2xelrE610tlYFWXW0kvpJCI6vxIzvoWzft5I6BXUFdHdc6HrCHYmPpVy+V6Tb4dLRQD02Q2GrDUKyYyiYTrAf+D+vBMCw4VGoXRtTTSDKC40cZWe+oYK0jhgfcBg7GX\\\\\\\/YcrApg3pKWEe0Y89akg5z6ymQRbaRhELAV3NijLo4hhm8C8N0H2XrKNq\\\\\\\/zxfH+2KBkzD5pvPw+G5EpsfMwQV9nCTS5IdOOLX8GCZy8RVVuriMrG5qTMV2Mpj8Z14oj5PZSaN7YvhpTYCURzXZrqSfbxskQCAt31qiFhyMPNd3PsAoCUAMq+hYepMCF\\\\\\\/7nVjqZ4nsoRWhOaOxDpfhVqn6ypKpkDbV7SKcu\\\\\\\/bMxUgttCCwqfXNDOppDC9uS4md7gYt6nAlpMcO2NJeTDHCuzt92edjOHhc2ZYhiGahRvcLo8Al08n6CBaqJqXPSLjo\\\\\\\/NdomLAmrLwla6yuqhshfLmPNSAUJZCuqki3a+iADTGxrwrLsZy1R1cJJuxLztlyfn4sqGlBuFvpVbtLbWHf4w+mYfX8e+3IhCy6BUb3yjzA0CcvXZ9A4Gaspwwe0Y8V8YES9YDDNapLXVxiaIpqaDy+wQSjoWqVQVg2KsN2a+NO6fnMk8E2TYQvM9uR52gbZ0MPe+9qcQu\\\\\\\/\\\\\\\/k2KYAY+eqS5gsgBhmzZvVbPnaryFVqGBddSkCkuLlVvlRilVhLarAoKjcvf6m68\\\\\\\/FW5UV3BJ2A2wF4g8oe\\\\\\\/fRXzqNW4REhypvxuNpsM+UuVHpvalZye+fICBIzjqCXDLYk1AZ3J1ZMX0\\\\\\\/IGdaDzdB9WDeMkiICFa1hTZgwFb9Ql6Ff9HmWRhpAFHwhvekZdo1YAF2xQvavtC71xmo4ajLFAv\\\\\\\/uOZRiQ\\\\\\\/H\\\\\\\/sj6JFBqS5QFSmcxPeuaYIDTjNO6Pwg\\\\\\\/mQcoV6sSkQ69fmns4Sb12ifDlCzrigf9TmhVqCcl6G7Vpc\\\\\\\/BegCgx462Ce2i5TAJPIR1ZeXPKmWuaFmG8W+T5hpUFgNn2OPO1tmoQ5gsyEyaOcFVf1dMvl4vHpOCMo7CsB\\\\\\\/WpgCBr3CKU+rlTYPdm2NAXDfbjeknfMsPmZuSGjGhPdJ5cMKJfUtyt57eRP2hojnzZS1ndON9ZRzDOZF68FpEh11eFZ7HAZWHP9wRzm9MLZ5\\\\\\\/dzuietvdxgOzWHbbjOyZeBqQNNTqhMEeJwshjRUHCe6HgmwP5Cs8bA0g6u+ecrn1i5TA4l6XqeSkY1N1V\\\\\\\/6FKUAFSqKQ80=\\\u0022,\\\u0022iv\\\u0022:\\\u0022Uo5bkUk0AsH6ae7z9YwmTg==\\\u0022,\\\u0022salt\\\u0022:\\\u0022f9d8845712f72c9fee5db7d9ff111eab\\\u0022}\u0022,\u0022hmac\u0022:\u0022f4bb6422efd3581980065d2efb4c8ddd03e21df476e2c9552f87609022864c11\u0022}"},"urlIsAjaxTrusted":{"\/content\/10.1101\/776708v1.full":true},"ws_fl":{"width":100,"height":21},"ws_gpo":{"size":"","annotation":"","lang":"","callback":"","width":300},"color":{"logo":"https:\/\/www.biorxiv.org\/sites\/default\/files\/bioRxiv_article.jpg"},"highwire_list_expand":{"is_collapsed":"1"},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"eu_cookie_compliance":{"popup_enabled":1,"popup_agreed_enabled":0,"popup_hide_agreed":0,"popup_clicking_confirmation":1,"popup_scrolling_confirmation":false,"popup_html_info":"\u003Cdiv\u003E\n  \u003Cdiv class =\u0022popup-content info\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n      \u003Cp\u003EWe use cookies on this site to enhance your user experience. By clicking any link on this page you are giving your consent for us to set cookies.\u003C\/p\u003E\n    \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022agree-button eu-cookie-compliance-default-button\u0022\u003EContinue\u003C\/button\u003E\n                    \u003Cbutton type=\u0022button\u0022 class=\u0022find-more-button eu-cookie-compliance-more-button\u0022\u003EFind out more\u003C\/button\u003E\n          \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E","use_mobile_message":false,"mobile_popup_html_info":"\u003Cdiv\u003E\n  \u003Cdiv class =\u0022popup-content info\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n          \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022agree-button eu-cookie-compliance-default-button\u0022\u003EContinue\u003C\/button\u003E\n                    \u003Cbutton type=\u0022button\u0022 class=\u0022find-more-button eu-cookie-compliance-more-button\u0022\u003EFind out more\u003C\/button\u003E\n          \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E\n","mobile_breakpoint":"768","popup_html_agreed":"\u003Cdiv\u003E\n  \u003Cdiv class=\u0022popup-content agreed\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n      \u003Ch2\u003EThank you for accepting cookies\u003C\/h2\u003E\u003Cp\u003EYou can now hide this message or find out more about cookies.\u003C\/p\u003E    \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022hide-popup-button eu-cookie-compliance-hide-button\u0022\u003EHide\u003C\/button\u003E\n              \u003Cbutton type=\u0022button\u0022 class=\u0022find-more-button eu-cookie-compliance-more-button-thank-you\u0022 \u003EMore info\u003C\/button\u003E\n          \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E","popup_use_bare_css":false,"popup_height":"auto","popup_width":"100%","popup_delay":1000,"popup_link":"\/help\/cookie-policy","popup_link_new_window":1,"popup_position":null,"popup_language":"en","store_consent":false,"better_support_for_screen_readers":0,"reload_page":0,"domain":"","popup_eu_only_js":0,"cookie_lifetime":"365","cookie_session":false,"disagree_do_not_show_popup":0,"method":"default","whitelisted_cookies":"","withdraw_markup":"\u003Cbutton type=\u0022button\u0022 class=\u0022eu-cookie-withdraw-tab\u0022\u003EPrivacy settings\u003C\/button\u003E\n\u003Cdiv class=\u0022eu-cookie-withdraw-banner\u0022\u003E\n  \u003Cdiv class=\u0022popup-content info\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n      \u003Cp\u003E\u0026lt;h2\u0026gt;We use cookies on this site to enhance your user experience\u0026lt;\/h2\u0026gt;\u0026lt;p\u0026gt;You have given your consent for us to set cookies.\u0026lt;\/p\u0026gt;\u003C\/p\u003E\n    \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022eu-cookie-withdraw-button\u0022\u003EWithdraw consent\u003C\/button\u003E\n    \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E\n","withdraw_enabled":false},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1,"trackUrlFragments":1},"jnl_biorxiv_styles":{"defaultJCode":"biorxiv"},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-916114 node-type-highwire-article context-content hw-default-jcode-biorxiv hw-article-type-article hw-article-category-new-results">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-M677548" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-M677548');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
        <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
    
  <div id="zone-branding" class="zone zone-branding clearfix print-display-block container-30">
    <div class="grid-15 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="bioRxiv" src="https://www.biorxiv.org/sites/default/files/bioRxiv_article.jpg" /></a>      </div>
                </div>
          </div>
</div><div class="grid-11 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-system block-menu block-main-menu block-system-main-menu odd block-without-title" id="block-system-main-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/" title="" class="" data-icon-position="" data-hide-link-title="0">Home</a></li>
<li class="leaf" role="menuitem"><a href="/about-biorxiv" class="" data-icon-position="" data-hide-link-title="0">About</a></li>
<li class="leaf" role="menuitem"><a href="/submit-a-manuscript" class="" data-icon-position="" data-hide-link-title="0">Submit</a></li>
<li class="last leaf" role="menuitem"><a href="/content/alertsrss" title="" class="" data-icon-position="" data-hide-link-title="0">ALERTS / RSS</a></li>
</ul></nav>    </div>
  </div>
</div><div class="block block-panels-mini block-biorxiv-search-box block-panels-mini-biorxiv-search-box even block-without-title" id="block-panels-mini-biorxiv-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-biorxiv_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/10.1101/776708v1.full" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--2" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-3QHF6U5Z2R-GQHeGNaF7d0lt2nJbLVD9OWz2HTIm-34" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2 advanced-search-link" >
  
      
  
  <div class="pane-content">
    <a href="/search">Advanced Search</a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix container-30">
  	      </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="916114" id="node916114" data-pisa="biorxiv;776708v1" data-pisa-master="biorxiv;776708" data-apath="/biorxiv/early/2019/09/20/776708.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div  class="highwire-cite highwire-cite-highwire-article highwire-citation-biorxiv-article-top clearfix has-author-tooltip" >

      <span class="biorxiv-article-type">
        New Results    </span>
  
        <h1 class="highwire-cite-title" id="page-title">The Evolution of Social Organization: Climate Influences Variation in Drosophilid Social Networks</h1>  
      <div  class="highwire-cite-authors" ><span  class="highwire-citation-authors"><span class="highwire-citation-author first hw-author-orcid-logo-wrapper" data-delta="0"><a href="http://orcid.org/0000-0003-3037-5738" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a><span class="nlm-given-names">Jacob A.</span> <span class="nlm-surname">Jezovit</span></span>, <span class="highwire-citation-author" data-delta="1"><span class="nlm-given-names">Rebecca</span> <span class="nlm-surname">Rooke</span></span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="2"><a href="http://orcid.org/0000-0003-0468-0763" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a><span class="nlm-given-names">Jonathan</span> <span class="nlm-surname">Schneider</span></span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="3"><a href="http://orcid.org/0000-0002-6254-6274" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a><span class="nlm-given-names">Joel D.</span> <span class="nlm-surname">Levine</span></span></span></div>
  
      <div  class="highwire-cite-metadata" ><span  class="highwire-cite-metadata-doi highwire-cite-metadata"><span class="label">doi:</span> https://doi.org/10.1101/776708 </span></div>
  
  
  </div>
<div id="hw-article-author-popups-node916114" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Jacob A. Jezovit </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-institution'>Department of Biology, University of Toronto at Mississauga</span>, Mississauga, Ontario, <span class='nlm-country'>Canada</span> L5L 1C6</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jacob%2BA.%2BJezovit%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Jezovit%20JA&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AJacob%2BA.%2BJezovit%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0003-3037-5738" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Jacob A. Jezovit</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Rebecca Rooke </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-institution'>Department of Biology, University of Toronto at Mississauga</span>, Mississauga, Ontario, <span class='nlm-country'>Canada</span> L5L 1C6</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Rebecca%2BRooke%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Rooke%20R&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ARebecca%2BRooke%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Jonathan Schneider </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-institution'>Department of Biology, University of Toronto at Mississauga</span>, Mississauga, Ontario, <span class='nlm-country'>Canada</span> L5L 1C6</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jonathan%2BSchneider%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Schneider%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AJonathan%2BSchneider%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0003-0468-0763" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Jonathan Schneider</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Joel D. Levine </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-institution'>Department of Biology, University of Toronto at Mississauga</span>, Mississauga, Ontario, <span class='nlm-country'>Canada</span> L5L 1C6</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Joel%2BD.%2BLevine%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Levine%20JD&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AJoel%2BD.%2BLevine%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0002-6254-6274" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Joel D. Levine</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:joel.levine@utoronto.ca" class="" data-icon-position="" data-hide-link-title="0">joel.levine@utoronto.ca</a></span></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/10.1101/776708v1" class="panels-ajax-tab-tab" data-panel-name="biorxiv_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:916114" data-trigger="" data-url-enabled="1">Abstract</a><a href="/panels_ajax_tab/biorxiv_tab_art/node:916114/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/10.1101/776708v1.full-text" class="panels-ajax-tab-tab" data-panel-name="article_tab_full_text" data-target-id="highwire_article_tabs" data-entity-context="node:916114" data-trigger="full-text" data-url-enabled="1">Full Text</a><a href="/panels_ajax_tab/article_tab_full_text/node:916114/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/10.1101/776708v1.article-info" class="panels-ajax-tab-tab" data-panel-name="biorxiv_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:916114" data-trigger="article-info" data-url-enabled="1">Info/History</a><a href="/panels_ajax_tab/biorxiv_tab_info/node:916114/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/10.1101/776708v1.article-metrics" class="panels-ajax-tab-tab" data-panel-name="article_tab_metrics" data-target-id="highwire_article_tabs" data-entity-context="node:916114" data-trigger="article-metrics" data-url-enabled="1">Metrics</a><a href="/panels_ajax_tab/article_tab_metrics/node:916114/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/10.1101/776708v1.supplementary-material" class="panels-ajax-tab-tab" data-panel-name="biorxiv_tab_data" data-target-id="highwire_article_tabs" data-entity-context="node:916114" data-trigger="supplementary-material" data-url-enabled="1">Supplementary material</a><a href="/panels_ajax_tab/biorxiv_tab_data/node:916114/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/10.1101/776708v1.full.pdf+html" class="panels-ajax-tab-tab" data-panel-name="biorxiv_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:916114" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i> Preview PDF</a><a href="/panels_ajax_tab/biorxiv_tab_pdf/node:916114/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="article_tab_full_text" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.biorxiv.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-article_tab_full_text"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-markup" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2 class="">Summary</h2><p id="p-2">Cultural norms, collective decisions, reproductive behaviour, and pathogen transmission all emerge from interaction patterns within animal social groups. These patterns of interaction support group-level phenomena that can influence an individual’s fitness. The aim of this study is to understand the evolution of social organization in Drosophila. Using a comparative ecological, phylogenetic and behavioural approach, we studied the different properties of social interaction networks (SINs) formed by 20 drosophilids and the different ways these species interact. We investigate whether animal network structures arise from common ancestry, a response to the species’ past ecological environment, other social behaviours, or a combination of these factors. We demonstrate that differences in past climate predicted the species’ current SIN properties. The drosophilid phylogeny offered no value to predicting species’ differences in SINs through phylogenetic signal tests. This suggests group-level social behaviours in drosophilid species are shaped by divergent climates. However, we find that the distance at which flies interact correlated with the drosophilid phylogeny, indicating that behavioural elements that comprise SINs have remained largely unchanged in their recent evolutionary history. We find a significant correlation of leg length to social distance, outlining the interdependence of anatomy and complex social structures. Although SINs display a complex evolutionary relationship across drosophilids, this study provides evidence of selective pressures acting on social behaviour in Drosophila. We speculate that conserved molecular mechanisms may be shared across drosophilids deep in their evolutionary history, similar to other pervasive mechanisms, like biological clocks.</p></div><div class="section" id="sec-1"><h2 class="">Introduction</h2><p id="p-3">Virtually all animals communicate and assemble into structured groups through a process called social organization [<a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">1</a>]. Cooperating in these groups helps many animals overcome environmental stress and increase their fitness. For example, animals may avoid predation, increase foraging efficiency, access mates and locate optimal migration routes by collectively moving as a herd [<a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">2</a>]. Understanding the nuances of social organization across animals has been enhanced by the application of social network analysis, since the structure of the network can predict and provide insight into group-level behaviours. For example, meerkat grooming networks can predict the likelihood of tuberculosis infection [<a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">3</a>], and a finch’s position in their social network can influence how attractive they are to potential mates [<a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">4</a>]. In <em>Drosophila melanogaster</em>, researchers have demonstrated that where a female lays her eggs is influenced by the structure of her social network [<a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">5</a>]. Taken together, animal social networks can impact the health and behaviour of individual group members. Studying social networks can, therefore, provide a valuable tool for understanding the dynamics of social organization.</p><p id="p-4">Despite its common label as a solitary insect, <em>D. melanogaster</em> displays a variety of social and collective behaviours. These include coordinating egg-laying sites [<a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">6</a>, <a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">7</a>], trans-generational mate preference [<a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">8</a>], collective feeding strategies [<a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">9</a>], communicating the presence of predators/parasites to group members [<a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">10</a>, <a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">11</a>] and collective escape responses [<a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">12</a>]. Additionally, groups of <em>D. melanogaster</em> socially interact in a group setting that is structured and non-random. This was demonstrated through the previously published Social Interaction Network (SIN) assay [<a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">13</a>]. A major finding from this assay was the intraspecific variation of a SIN property called betweeness centrality, a measurement of social network cohesiveness [<a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">14</a>]. Betweeness centrality has also been reported to be heritable in humans [<a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">15</a>], suggesting that the SIN assay may capture group-level phenotypes that are heritable in <em>Drosophila</em> and other organisms. With the SIN assay, we can measure distributions of replicated social networks, providing an excellent tool to study the sociality of <em>Drosophila</em>.</p><p id="p-5">The <em>Drosophila</em> genus of insects is widespread throughout the world where groups of species have radiated into arid, temperate and tropical environments [<a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">16</a>, <a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">17</a>]. The type of environment in which animals reside can correspond to differences in food availability, predation and species richness, among other things. These ecological variables have been shown to influence social network structure. For example, food abundance and predation pressure influence the interconnectedness of social networks in whales and guppies [<a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">18</a>, <a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">19</a>]. Additionally, the taxonomic and phylogenetic relationships across drosophilids are well-established [<a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">20</a>, <a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">21</a>]. One group measured thermal, cold and desiccation stress across 95 drosophilid species and found a strong phylogenetic correlation between cold tolerance and desiccation tolerance [<a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">22</a>, <a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">23</a>]. Thus, this genus is well-suited for studying trait evolution in a macroevolutionary context.</p><p id="p-6">Here, our aim is to apply comparative ecological, phylogenetic and behavioural approaches to understand the evolution of social organization in <em>Drosophila</em>. We hypothesized that drosophilid species vary in their social organization, measured through SINs, and that the species’ variation in SINs can result from three categorical influences (<a id="xref-fig-1-1" class="xref-fig" href="#F1">Figure 1</a>): <em>behavioural elements of the network</em> (movement, social spacing, pairwise interactions; see <a id="xref-table-wrap-1-1" class="xref-table" href="#T1">Table 1</a>), <em>phylogenetic signal</em> [<a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">24</a>, <a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">25</a>], and <em>climatic selective pressures</em> [<a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">26</a>]. In this report, we demonstrate how a combination of these three categories predicts drosophilid species’ variation in social behaviour. We reason that a broader comparative study of SINs across ecologically diverse drosophilid species enables us to investigate which factors have shaped social organization over macroevolutionary time. We show that the ability to form social networks in <em>Drosophila</em> is conserved and that their past environment predicts their present social structure. We suggest that the evolution of group-level interaction patterns may be supported by conserved mechanisms analogous to other pervasive mechanisms, such as biological clocks.</p><div id="F1" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/09/20/776708/F1.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Hypothetical sources of influence on variation in drosophilid group level behaviours. Variation in Social Interaction Networks (SINs), may be influenced by past environmental selective pressures, differential common ancestry (phylogeny) and through a hierarchy of behavioural elements in order of increasing complexity: movement (level 1), the social spacing between interacting flies (level 2), and pairwise social interactions (level 3). See Table 1 for definitions of each of these elements. Arrows show how phylogeny, environment and behaviour are connected and indicate the relationships of phylogenetic and environmental influences on each behavioural variable. The phylogeny [21] lists the evolutionary history of the 20 drosophilid species we studied. The world map outlines the coordinates of the geographic origin of each species stock we studied. The colouration on the map indicate tropical regions (green), arid regions (red), and temperate regions (blue), based on the Koppen-Geiger climate classification [49]. See Table 1 for definitions of each behavioural measure within these three categories." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-988956387" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;span xmlns=&quot;http://www.w3.org/1999/xhtml&quot; class=&quot;caption-title&quot;&gt;Hypothetical sources of influence on variation in drosophilid group level behaviours.&lt;/span&gt; Variation in Social Interaction Networks (SINs), may be influenced by past environmental selective pressures, differential common ancestry (phylogeny) and through a hierarchy of behavioural elements in order of increasing complexity: movement (level 1), the social spacing between interacting flies (level 2), and pairwise social interactions (level 3). See Table 1 for definitions of each of these elements. Arrows show how phylogeny, environment and behaviour are connected and indicate the relationships of phylogenetic and environmental influences on each behavioural variable. The phylogeny [21] lists the evolutionary history of the 20 drosophilid species we studied. The world map outlines the coordinates of the geographic origin of each species stock we studied. The colouration on the map indicate tropical regions (green), arid regions (red), and temperate regions (blue), based on the Koppen-Geiger climate classification [49]. See Table 1 for definitions of each behavioural measure within these three categories.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 1:" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2019/09/20/776708/F1.medium.gif" width="358" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 1:" src="https://www.biorxiv.org/content/biorxiv/early/2019/09/20/776708/F1.medium.gif" width="358" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/09/20/776708/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1:" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/09/20/776708/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml"><span class="fig-label">Figure 1:</span> <span class="caption-title">Hypothetical sources of influence on variation in drosophilid group level behaviours.</span><p id="p-7" class="first-child">Variation in Social Interaction Networks (SINs), may be influenced by past environmental selective pressures, differential common ancestry (phylogeny) and through a hierarchy of behavioural elements in order of increasing complexity: movement (level 1), the social spacing between interacting flies (level 2), and pairwise social interactions (level 3). See <a id="xref-table-wrap-1-2" class="xref-table" href="#T1">Table 1</a> for definitions of each of these elements. Arrows show how phylogeny, environment and behaviour are connected and indicate the relationships of phylogenetic and environmental influences on each behavioural variable. The phylogeny [<a id="xref-ref-21-2" class="xref-bibr" href="#ref-21">21</a>] lists the evolutionary history of the 20 drosophilid species we studied. The world map outlines the coordinates of the geographic origin of each species stock we studied. The colouration on the map indicate tropical regions (green), arid regions (red), and temperate regions (blue), based on the Koppen-Geiger climate classification [<a id="xref-ref-49-1" class="xref-bibr" href="#ref-49">49</a>]. See <a id="xref-table-wrap-1-3" class="xref-table" href="#T1">Table 1</a> for definitions of each behavioural measure within these three categories.</p><div class="sb-div caption-clear"></div></div></div><div id="T1" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/919988/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup"><a href="/highwire/markup/919988/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/919988" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 1:</span> <p id="p-8" class="first-child">A glossary of all behavioural measurements discussed in this article. For a more comprehensive definition of each measure, see STAR Methods.</p><div class="sb-div caption-clear"></div></div></div></div><div class="section" id="sec-2"><h2 class="">Results</h2><div id="sec-3" class="subsection"><h3>Variation in social behaviour</h3><p id="p-9">The movement of some species, such as <em>D. mojavensis, D. immigrans</em>, <em>D. virilis</em>, <em>D. santomea</em> and <em>D. mauritiana</em>, were noticeably sedentary, compared to more active species such as <em>D. melanogaster</em> (<a id="xref-fig-2-1" class="xref-fig" href="#F2">Figure 2</a>). Overall, the male flies in all species were more active than the female flies, except for <em>D. novamexicana</em> and the outgroup species <em>Chymomyza procnemis</em> (<a id="xref-fig-2-2" class="xref-fig" href="#F2">Figure 2</a>). In <em>D. yakuba</em>, <em>D. erecta</em> and <em>D. hydei</em>, the males were markedly more active than their female counterparts. Through a two-way ANOVA, we found significant species-by-sex interaction effects for movement (p &lt; 0.0001, <a id="xref-fig-2-3" class="xref-fig" href="#F2">Figure 2</a>).</p><div id="F2" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/09/20/776708/F2.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Movement, interaction duration, reciprocation and number of interactions vary across species and between groups of male flies and female flies. The y-axis lists each species, along with a phylogeny mapping their evolutionary relationships. Each phylogeny is coloured using the &#x2018;contMap&#x2019; function (phytools package), according to a scale where red indicates the minimum mean measure observed and blue indicates the maximum mean measure observed. Error bars indicate standard error of the mean. All graphs on the left with red bars refer to the data acquired from groups of male flies and all graphs on the right with blue bars refers to the data acquired from groups of female flies. A) Movement: Significant species-specific effects were observed (p &lt; 2 x 10-16, two-way ANOVA). Significant sex specific effects and sex by species interaction effects were observed (p = 2 x 10-16 for both effects; two-way ANOVA). B) Interaction duration: Significant species-specific effects were observed (p &lt; 2 x 10-16, two-way ANOVA). Significant sex specific effects and sex by species interaction effects were observed (p = 2 x 10-16 for both effects; two-way ANOVA). C) Reciprocation: Significant species-specific effects were observed (p &lt; 2 x 10-16, two-way ANOVA). Significant sex specific effects and sex by species interaction effects were observed (p = 2 x 10-16 for both effects; two-way ANOVA). D) Number of interactions: Significant species-specific effects were observed (p &lt; 2.24 x 10-11, two-way ANOVA). Significant sex specific effects and sex by species interaction effects were observed (p = 2 x 10-16 for both effects; two-way ANOVA)." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-988956387" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;&lt;span class=&quot;caption-title&quot;&gt;Movement, interaction duration, reciprocation and number of interactions vary across species and between groups of male flies and female flies.&lt;/span&gt; The y-axis lists each species, along with a phylogeny mapping their evolutionary relationships. Each phylogeny is coloured using the &#x2018;contMap&#x2019; function (phytools package), according to a scale where red indicates the minimum mean measure observed and blue indicates the maximum mean measure observed. Error bars indicate standard error of the mean. All graphs on the left with red bars refer to the data acquired from groups of male flies and all graphs on the right with blue bars refers to the data acquired from groups of female flies. A) Movement: Significant species-specific effects were observed (p &lt; 2 x 10&lt;sup&gt;-16&lt;/sup&gt;, two-way ANOVA). Significant sex specific effects and sex by species interaction effects were observed (p = 2 x 10&lt;sup&gt;-16&lt;/sup&gt; for both effects; two-way ANOVA). B) Interaction duration: Significant species-specific effects were observed (p &lt; 2 x 10&lt;sup&gt;-16&lt;/sup&gt;, two-way ANOVA). Significant sex specific effects and sex by species interaction effects were observed (p = 2 x 10&lt;sup&gt;-16&lt;/sup&gt; for both effects; two-way ANOVA). C) Reciprocation: Significant species-specific effects were observed (p &lt; 2 x 10&lt;sup&gt;-16&lt;/sup&gt;, two-way ANOVA). Significant sex specific effects and sex by species interaction effects were observed (p = 2 x 10&lt;sup&gt;-16&lt;/sup&gt; for both effects; two-way ANOVA). D) Number of interactions: Significant species-specific effects were observed (p &lt; 2.24 x 10&lt;sup&gt;-11&lt;/sup&gt;, two-way ANOVA). Significant sex specific effects and sex by species interaction effects were observed (p = 2 x 10&lt;sup&gt;-16&lt;/sup&gt; for both effects; two-way ANOVA).&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 2:" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2019/09/20/776708/F2.medium.gif" width="267" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 2:" src="https://www.biorxiv.org/content/biorxiv/early/2019/09/20/776708/F2.medium.gif" width="267" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/09/20/776708/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2:" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/09/20/776708/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 2:</span> <span class="caption-title">Movement, interaction duration, reciprocation and number of interactions vary across species and between groups of male flies and female flies.</span><p id="p-10" class="first-child">The y-axis lists each species, along with a phylogeny mapping their evolutionary relationships. Each phylogeny is coloured using the ‘contMap’ function (phytools package), according to a scale where red indicates the minimum mean measure observed and blue indicates the maximum mean measure observed. Error bars indicate standard error of the mean. All graphs on the left with red bars refer to the data acquired from groups of male flies and all graphs on the right with blue bars refers to the data acquired from groups of female flies. A) Movement: Significant species-specific effects were observed (p &lt; 2 x 10<sup>-16</sup>, two-way ANOVA). Significant sex specific effects and sex by species interaction effects were observed (p = 2 x 10<sup>-16</sup> for both effects; two-way ANOVA). B) Interaction duration: Significant species-specific effects were observed (p &lt; 2 x 10<sup>-16</sup>, two-way ANOVA). Significant sex specific effects and sex by species interaction effects were observed (p = 2 x 10<sup>-16</sup> for both effects; two-way ANOVA). C) Reciprocation: Significant species-specific effects were observed (p &lt; 2 x 10<sup>-16</sup>, two-way ANOVA). Significant sex specific effects and sex by species interaction effects were observed (p = 2 x 10<sup>-16</sup> for both effects; two-way ANOVA). D) Number of interactions: Significant species-specific effects were observed (p &lt; 2.24 x 10<sup>-11</sup>, two-way ANOVA). Significant sex specific effects and sex by species interaction effects were observed (p = 2 x 10<sup>-16</sup> for both effects; two-way ANOVA).</p><div class="sb-div caption-clear"></div></div></div><p id="p-11">Next, we observed differences in the social spacing parameters across the drosophilid species in both the male and female data sets (Table S2). The distance and time parameter displayed the least variation for both the male dataset (distance: 1.25-3.25 body lengths; time: 0.2-2.65 seconds) and the female dataset (distance: 1.25-2.75 body lengths; time: 0.35-1.85 seconds). In the male dataset, the species with the estimated distance parameter of 3.25 body lengths was the outgroup species <em>Ch. procnemis</em>. Thus, in the male dataset, all <em>Drosophila</em> species’ distance parameter ranged from 1.25 to 2.50 body lengths (Table S2). The angle parameter showed the most variation in both the male dataset (20-147.5 degrees) and the female dataset (30-140 degrees). Most female species interact at a wider angle than their male counterparts. This is especially evident in <em>D. santomea</em>, <em>D. yakuba</em>, <em>D. willistoni</em>, <em>D. persimilis</em> and <em>D. buzzatii</em>, where the females have an estimated angle parameter nearly double the parameter of the males (Table S2). Overall, this data shows that most males and females of the same species interact differently and that most females interact at a wider angle than the males of the same species.</p><p id="p-12">When we accounted for the different social spacing parameters, we found that there were significant species-by-sex interaction effects for interaction duration, reciprocation and number of interactions (p &lt; 0.0001 for all measures, two-way ANOVA, <a id="xref-fig-2-4" class="xref-fig" href="#F2">Figure 2</a>). For reciprocation, female flies tended to reciprocate interactions more frequently than male flies. This may be attributable to the wider angle parameters estimated in the female social spacing data (Table S2). For interaction duration, there is a clear inverse relationship with movement, especially in the female dataset where sedentary species spent more time socially interacting on average. Species that were more active also interacted more on average. Although some sedentary species interacted less than active species, all species interacted at least hundreds of times on average (<a id="xref-fig-2-5" class="xref-fig" href="#F2">Figure 2</a>), providing relatively large samples of SINs to analyze. Taken together, these data suggest that species vary in the characteristics of their pairwise social interactions.</p><p id="p-13">Despite the variation in movement, social spacing and pairwise interaction patterns, all 20 species form SINs in at least 80% of the video trials in both the male and female datasets (Figure S1). We found significant species-by-sex interaction effects for assortativity (p = 0.0007), clustering coefficient (p &lt; 0.0001), betweeness centrality (p &lt; 0.0001), and global efficiency (p &lt; 0.0001; two-way ANOVA). Qualitatively across all SIN measures, the relative species differences appear very consistent between the male and female datasets (<a id="xref-fig-3-1" class="xref-fig" href="#F3">Figure 3</a>): clustering coefficient and global efficiency display the largest range of species’ variation, compared to betweeness centrality and assortativity (<a id="xref-fig-3-2" class="xref-fig" href="#F3">Figure 3</a>). Prior to collecting the 20 species-wide data, we collected data for seven species. Five common species were compared between these two independent data sets. The estimated SIN measures (Figures S5-S6) were consistent between these independent data collections. Overall, these data suggest that SIN structure varies across species and sex and that species’ SIN structures are stable across time.</p><div id="F3" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/09/20/776708/F3.large.jpg?width=800&amp;height=600&amp;carousel=1" title="SINs vary across species and between groups of male flies and female flies. For each measure, the x-axis is expressed as a z-score, normalized from the values of virtual SINs (see Methods). The y-axis lists each species, along with a phylogeny mapping their evolutionary relationships. Each phylogeny is coloured using the &#x2018;contMap&#x2019; function (phytools package), according to a scale where red indicates the minimum mean measure observed and blue indicates the maximum mean measure observed. Error bars indicate standard error of the mean. All graphs on the left with red bars refer to the data acquired from groups of male flies and all graphs on the right with blue bars refers to the data acquired from groups of female flies. A) Assortativity: Significant species-specific effects were observed (p &lt; 2 x 10-16, two-way ANOVA). Significant sex-specific and sex by species interaction effects were observed (p = 0.0002, p = 0.0007 respectively, two-way ANOVA). B) Clustering coefficient: Significant species-specific effects were observed (p &lt; 2 x 10-16, two-way ANOVA). No significant sex specific effects were observed (p = 0.248, two-way ANOVA) but a significant sex by species interaction effect was observed (p &lt; 2 x 10-16, two-way ANOVA). C) Betweeness centrality: Significant species-specific effects are observed (p &lt; 2 x 10-16, two-way ANOVA). No sex-specific effects were observed following multiple test correction (p = 0.0151, two-way ANOVA). A significant sex by species interaction effect was observed (p = 7.65 x 10-15, two-way ANOVA). D) Global efficiency: Significant species specific, sex specific, and sex by species interaction effects were observed (p = 7.23 x 10-9, p &lt; 2 x 10-16, p &lt; 2 x 10-16 respectively; two-way ANOVA). See also Figures S1 for the percentage of SINs formed across all video trials and Figures S5-S6 for replication of SIN measures from pilot data." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-988956387" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;&lt;span class=&quot;caption-title&quot;&gt;SINs vary across species and between groups of male flies and female flies.&lt;/span&gt; For each measure, the x-axis is expressed as a z-score, normalized from the values of virtual SINs (see Methods). The y-axis lists each species, along with a phylogeny mapping their evolutionary relationships. Each phylogeny is coloured using the &#x2018;contMap&#x2019; function (phytools package), according to a scale where red indicates the minimum mean measure observed and blue indicates the maximum mean measure observed. Error bars indicate standard error of the mean. All graphs on the left with red bars refer to the data acquired from groups of male flies and all graphs on the right with blue bars refers to the data acquired from groups of female flies. A) Assortativity: Significant species-specific effects were observed (p &lt; 2 x 10&lt;sup&gt;-16&lt;/sup&gt;, two-way ANOVA). Significant sex-specific and sex by species interaction effects were observed (p = 0.0002, p = 0.0007 respectively, two-way ANOVA). B) Clustering coefficient: Significant species-specific effects were observed (p &lt; 2 x 10&lt;sup&gt;-16&lt;/sup&gt;, two-way ANOVA). No significant sex specific effects were observed (p = 0.248, two-way ANOVA) but a significant sex by species interaction effect was observed (p &lt; 2 x 10&lt;sup&gt;-16&lt;/sup&gt;, two-way ANOVA). C) Betweeness centrality: Significant species-specific effects are observed (p &lt; 2 x 10&lt;sup&gt;-16&lt;/sup&gt;, two-way ANOVA). No sex-specific effects were observed following multiple test correction (p = 0.0151, two-way ANOVA). A significant sex by species interaction effect was observed (p = 7.65 x 10&lt;sup&gt;-15&lt;/sup&gt;, two-way ANOVA). D) Global efficiency: Significant species specific, sex specific, and sex by species interaction effects were observed (p = 7.23 x 10&lt;sup&gt;-9&lt;/sup&gt;, p &lt; 2 x 10&lt;sup&gt;-16&lt;/sup&gt;, p &lt; 2 x 10&lt;sup&gt;-16&lt;/sup&gt; respectively; two-way ANOVA). See also Figures S1 for the percentage of SINs formed across all video trials and Figures S5-S6 for replication of SIN measures from pilot data.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 3:" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2019/09/20/776708/F3.medium.gif" width="278" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 3:" src="https://www.biorxiv.org/content/biorxiv/early/2019/09/20/776708/F3.medium.gif" width="278" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/09/20/776708/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 3:" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/09/20/776708/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 3:</span> <span class="caption-title">SINs vary across species and between groups of male flies and female flies.</span><p id="p-14" class="first-child">For each measure, the x-axis is expressed as a z-score, normalized from the values of virtual SINs (see Methods). The y-axis lists each species, along with a phylogeny mapping their evolutionary relationships. Each phylogeny is coloured using the ‘contMap’ function (phytools package), according to a scale where red indicates the minimum mean measure observed and blue indicates the maximum mean measure observed. Error bars indicate standard error of the mean. All graphs on the left with red bars refer to the data acquired from groups of male flies and all graphs on the right with blue bars refers to the data acquired from groups of female flies. A) Assortativity: Significant species-specific effects were observed (p &lt; 2 x 10<sup>-16</sup>, two-way ANOVA). Significant sex-specific and sex by species interaction effects were observed (p = 0.0002, p = 0.0007 respectively, two-way ANOVA). B) Clustering coefficient: Significant species-specific effects were observed (p &lt; 2 x 10<sup>-16</sup>, two-way ANOVA). No significant sex specific effects were observed (p = 0.248, two-way ANOVA) but a significant sex by species interaction effect was observed (p &lt; 2 x 10<sup>-16</sup>, two-way ANOVA). C) Betweeness centrality: Significant species-specific effects are observed (p &lt; 2 x 10<sup>-16</sup>, two-way ANOVA). No sex-specific effects were observed following multiple test correction (p = 0.0151, two-way ANOVA). A significant sex by species interaction effect was observed (p = 7.65 x 10<sup>-15</sup>, two-way ANOVA). D) Global efficiency: Significant species specific, sex specific, and sex by species interaction effects were observed (p = 7.23 x 10<sup>-9</sup>, p &lt; 2 x 10<sup>-16</sup>, p &lt; 2 x 10<sup>-16</sup> respectively; two-way ANOVA). See also Figures S1 for the percentage of SINs formed across all video trials and Figures S5-S6 for replication of SIN measures from pilot data.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-4" class="subsection"><h3>Phylogenetic signal</h3><p id="p-15">To test for phylogenetic influence on comparative trait data, two metrics were implemented: Blomberg’s K [<a id="xref-ref-24-2" class="xref-bibr" href="#ref-24">24</a>] and Pagel’s λ [<a id="xref-ref-25-2" class="xref-bibr" href="#ref-25">25</a>]. High phylogenetic signal implies traits are conserved within a given phylogenetic tree. Low phylogenetic signal can be explained by rapidly changing evolutionary events and we favor the explanation that the trait is conserved beyond the root of the given phylogenetic tree. We applied these phylogenetic signal tests across all the behavioural elements (movement, social spacing, pairwise interactions and SIN variables). We found the strongest evidence of high phylogenetic signal for the social spacing distance parameter in both the male and female datasets (<a id="xref-table-wrap-2-1" class="xref-table" href="#T2">Table 2</a>). Both the K and λ tests agree on this statistically significant result, except for the K value in the male dataset (K = 0.20, p = 0.21). This disagreement between the two tests may result from statistical noise because a sample size of 20 species is considered minimal to detect phylogenetic signal [<a id="xref-ref-24-3" class="xref-bibr" href="#ref-24">24</a>, <a id="xref-ref-27-1" class="xref-bibr" href="#ref-27">27</a>].</p><div id="T2" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/919987/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup"><a href="/highwire/markup/919987/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/919987" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 2:</span> <p id="p-16" class="first-child">Phylogenetic signal test results of all behavioural variables for K and λ Bolded cells indicate statistical significance for the presence of phylogenetic signal.</p><div class="sb-div caption-clear"></div></div></div><p id="p-17">To illustrate the phylogenetic signal for the distance parameter, hereafter referred to as social distance, we mapped each species’ distance parameter (Table S2) onto the drosophilid phylogeny. This produced an evolutionary trait map that showed strong phenotypic divergence of social distance between the <em>Drosophila</em> and <em>Sophophora</em> subgenera (<a id="xref-fig-4-1" class="xref-fig" href="#F4">Figure 4</a>). We hypothesized that this divergence in social distance between the two subgenera may correlate with differences in leg size since: 1) most of the <em>Drosophila</em> species in our sample were noticeably larger than the <em>Sophophora</em> species; 2) all species commonly extended their legs to touch conspecifics during social interactions. To test this, we measured the legs and bodies of male flies for all 20 species. To control for the differences in body sizes across all species, we calculated a relative leg length measure (total leg length ÷ body size; Table S3 and S4). We generated a trait map of relative leg length which shows that the <em>Sophophora</em> subgenus has a longer social distance with longer relative leg lengths and the <em>Drosophila</em> subgenus has a shorter social distance with shorter relative leg lengths (<a id="xref-fig-4-2" class="xref-fig" href="#F4">Figure 4</a>). Next, we tested whether the relative leg length predicts social distance within the entire <em>Drosophila</em> genus by generating a simple linear regression model (<a id="xref-fig-4-3" class="xref-fig" href="#F4">Figure 4</a>). These results show a positive correlation (R<sup>2</sup>=0.259) between relative leg length and social distance.</p><div id="F4" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/09/20/776708/F4.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Social distance and relative leg length display phylogenetic signal and are correlated. A) Phylogenetic trait map of drosophilid species differences in social distance. The phylogeny is coloured using the &#x2018;contMap&#x2019; function (phytools package) to help visualize species differences and ancestral nodes. Red indicates the minimum value and blue indicates the maximum value observed across the species. The social distance displays divergence between the Drosophila and Sophophora subgenera. B) Boxplot displaying drosophilid species differences across the relative leg length (total leg length / relative body size). Circles depict outliers, whiskers represent the maximum and minimum values and the bolded line within each box represents the median. The phylogeny on the y-axis is coloured using the &#x2018;contMap&#x2019; function (phytools). C) Simple linear regression outlining the relationship between relative leg length (x-axis) and social distance (y-axis). See also Tables S3-S4 for measurements of leg length and body size across all species." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-988956387" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;&lt;span class=&quot;caption-title&quot;&gt;Social distance and relative leg length display phylogenetic signal and are correlated.&lt;/span&gt; A) Phylogenetic trait map of drosophilid species differences in social distance. The phylogeny is coloured using the &#x2018;contMap&#x2019; function (phytools package) to help visualize species differences and ancestral nodes. Red indicates the minimum value and blue indicates the maximum value observed across the species. The social distance displays divergence between the &lt;em&gt;Drosophila&lt;/em&gt; and &lt;em&gt;Sophophora&lt;/em&gt; subgenera. B) Boxplot displaying drosophilid species differences across the relative leg length (total leg length / relative body size). Circles depict outliers, whiskers represent the maximum and minimum values and the bolded line within each box represents the median. The phylogeny on the y-axis is coloured using the &#x2018;contMap&#x2019; function (phytools). C) Simple linear regression outlining the relationship between relative leg length (x-axis) and social distance (y-axis). See also Tables S3-S4 for measurements of leg length and body size across all species.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 4:" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2019/09/20/776708/F4.medium.gif" width="325" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 4:" src="https://www.biorxiv.org/content/biorxiv/early/2019/09/20/776708/F4.medium.gif" width="325" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/09/20/776708/F4.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 4:" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/09/20/776708/F4.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 4:</span> <span class="caption-title">Social distance and relative leg length display phylogenetic signal and are correlated.</span><p id="p-18" class="first-child">A) Phylogenetic trait map of drosophilid species differences in social distance. The phylogeny is coloured using the ‘contMap’ function (phytools package) to help visualize species differences and ancestral nodes. Red indicates the minimum value and blue indicates the maximum value observed across the species. The social distance displays divergence between the <em>Drosophila</em> and <em>Sophophora</em> subgenera. B) Boxplot displaying drosophilid species differences across the relative leg length (total leg length / relative body size). Circles depict outliers, whiskers represent the maximum and minimum values and the bolded line within each box represents the median. The phylogeny on the y-axis is coloured using the ‘contMap’ function (phytools). C) Simple linear regression outlining the relationship between relative leg length (x-axis) and social distance (y-axis). See also Tables S3-S4 for measurements of leg length and body size across all species.</p><div class="sb-div caption-clear"></div></div></div><p id="p-19">To sum up, these data show that most properties of self-organization in drosophilids have low phylogenetic signal, except social distance which has strong phylogenetic signal. We also show the physical characteristics, leg length and body size, correlates with social distance in <em>Drosophila</em>.</p></div><div id="sec-5" class="subsection"><h3>Environmental models of social behaviour</h3><p id="p-20">To investigate whether past environmental selective pressures predict current social structure across drosophilids, we extracted 19 climate variables from WorldClim [<a id="xref-ref-26-2" class="xref-bibr" href="#ref-26">26</a>]. We simplified the 19 climate variables to 5 principal components and used them as predictors to generate environmental models, for each SIN measure, through stepwise regressions. Temperature range, precipitation of the wettest quarter and temperature of the coldest quarter were strong proponents in each of the climatic principal components (data not shown). We found the resultant environmental models to be surprisingly predictive for assortativity (R<sup>2</sup> = 0.444), clustering coefficient (R<sup>2</sup> = 0.452), betweeness centrality (R<sup>2</sup> = 0.314), and global efficiency (R<sup>2</sup> = 0.352; <a id="xref-fig-5-1" class="xref-fig" href="#F5">Figure 5</a>). We generated additional environmental models via stepwise regression by fitting the 5 principal components to the behavioural element variables. The subsequent environmental models were less predictive than the models for SIN measures: movement (R<sup>2</sup> = 0.250), the social spacing parameters (distance: R<sup>2</sup> = 0.255, angle: R<sup>2</sup> = 0.145, time: R<sup>2</sup> = 0.160) and the pairwise interaction variables (number of interactions: R<sup>2</sup> = 0.208, interaction duration: R<sup>2</sup> = 0.382, reciprocation: R<sup>2</sup> = 0.122; Figure S2-S4). Together this indicates that climatic selective pressures correlate to group-level behaviours better than they correlate to the individual behavioural elements.</p><div id="F5" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/09/20/776708/F5.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Environmental, behavioural and combined stepwise regression models of assortativity, clustering coefficient, betweeness centrality and global efficiency. For all regressions, each data point represents the mean SIN measure for a single species. The mean SIN measure for groups of male flies and female flies were pooled into each regression and are labeled with red and blue points, respectively. Each solid trend line indicates line of best fit and dashed lines indicate 95% confidence interval of the model. For each model, the multivariate linear equation can be found below. A) The environmental model (Equation 1) significantly predicts assortativity (R2 = 0.44447, p &lt; 0.0001). The behavioural model (Equation 2) significantly predicts assortativity (R2 = 0.49493, p &lt; 0.0001). The combined model (Equation 3) significantly improves the prediction of assortativity compared to behavioural model alone (Combined model AICc &lt; Behavioural model AICc; p = 0.0001, log likelihood ratio test). B) The environmental model (Equation 4) significantly predicts clustering coefficient (R2 = 0.45258. p &lt; 0.0001). The behavioural model (Equation 5) significantly predicts clustering coefficient (R2 = 0.56293, p &lt; 0.0001). The combined model (Equation 6) significantly improves the prediction of clustering coefficient compared to behavioural model alone (Combined model AICc &lt; Behavioural model AICc; p &lt; 0.0001, log likelihood ratio test). C) The environmental model (Equation 7) significantly predicts betweeness centrality (R2 = 0.31432, p= 0.0008). The behavioural model (Equation 8) significantly predicts betweeness centrality (R2 = 0.52325, p &lt; 0.0001). The combined model (Equation 9) does not significantly improve the prediction of betweeness centrality compared to behavioural model alone (Combined model AICc &gt; Behavioural model AICc; p = 0.0504, log likelihood ratio test). D) The environmental model (Equation 10) significantly predicts global efficiency (R2 = 0.35261, p = 0.0003). The behavioural model (Equation 11) significantly predicts global efficiency (R2 = 0.74972). The combined model (Equation 12) significantly improves prediction of global efficiency (Combined AICc &lt; Behavioural AICc; p &lt; 0.0001, log likelihood ratio test). Equation 1: y = 0.63 &#x2013; 0.092*[PC2] + 0.064*[PC3] + 0.11*[PC5]. Equation 2: y = 0.53 &#x2013; 0.081*[movement] + 0.023*[interaction duration] + 0.00018*[number of interactions]. Equation 3: y = 0.56 &#x2013; 0.068*[PC2] + 0.045*[PC3] + 0.086*[PC5] &#x2013; 0.071*[movement] + 0.11 *[interaction duration] + 0.00019*[number of interactions]. Equation 4: y = 0.047 &#x2013; 0.069*[PC1] &#x2013; 0.18*[PC2] + 0.21*[PC5]. Equation 5: y = 0.73 &#x2013; 0.19*[movement] &#x2013; 0.0072*[angle] + 0.04*[interaction duration] + 0.00038*[number of interactions]. Equation 6: y = 0.71 &#x2013; 0.014*[PC1] &#x2013; 0.15*[PC2] + 0.14*[PC5] &#x2013; 0.16*[movement] &#x2013; 0.0067*[angle] + 0.021*[duration] + 0.00038*[number of interactions]. Equation 7: y = 0.99 + 0.07*[PC1] + 0.057*[PC2] &#x2013; 0.13*[PC5]. Equation 8: y = 0.096 &#x2013; 0.28*[social distance] + 0.25*[time] &#x2013; 0.031*[interaction duration] + 0.0002*[number of interactions] + 2.91*[reciprocation]. Equation 9: y = 0.54 + 0.0065*[PC5] + 0.055*[PC2] &#x2013; 0.11*[PC5] &#x2013; 0.39*[social distance] + 0.23*[time] &#x2013; 0.01*[interaction duration] + 0.0002*[number of interactions] + 1.94*[reciprocation]. Equation 10: y = &#x2212;3.64 + 0.55[PC2] &#x2013; 0.34[PC3] &#x2013; 0.58[PC5]. Equation 11: y = 2.03 + 0.31*[movement] &#x2013; 0.14*[interaction duration] &#x2013; 0.0007*[number of interactions] &#x2013; 13*[reciprocation]. Equation 12: y = 2.47 + 0.33*[PC3] &#x2013; 0.38*[PC5] + 0.26*[movement] &#x2013; 0.078*[interaction duration] &#x2013; 0.0007*[number of interactions] &#x2013; 15*[reciprocation]. See also Figures S2-S4 for environmental and behavioural models of other behavioural elements." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-988956387" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;&lt;span class=&quot;caption-title&quot;&gt;Environmental, behavioural and combined stepwise regression models of assortativity, clustering coefficient, betweeness centrality and global efficiency.&lt;/span&gt; For all regressions, each data point represents the mean SIN measure for a single species. The mean SIN measure for groups of male flies and female flies were pooled into each regression and are labeled with red and blue points, respectively. Each solid trend line indicates line of best fit and dashed lines indicate 95% confidence interval of the model. For each model, the multivariate linear equation can be found below. A) The environmental model (Equation 1) significantly predicts assortativity (R&lt;sup&gt;2&lt;/sup&gt; = 0.44447, p &lt; 0.0001). The behavioural model (Equation 2) significantly predicts assortativity (R&lt;sup&gt;2&lt;/sup&gt; = 0.49493, p &lt; 0.0001). The combined model (Equation 3) significantly improves the prediction of assortativity compared to behavioural model alone (Combined model AICc &lt; Behavioural model AICc; p = 0.0001, log likelihood ratio test). B) The environmental model (Equation 4) significantly predicts clustering coefficient (R&lt;sup&gt;2&lt;/sup&gt; = 0.45258. p &lt; 0.0001). The behavioural model (Equation 5) significantly predicts clustering coefficient (R&lt;sup&gt;2&lt;/sup&gt; = 0.56293, p &lt; 0.0001). The combined model (Equation 6) significantly improves the prediction of clustering coefficient compared to behavioural model alone (Combined model AICc &lt; Behavioural model AICc; p &lt; 0.0001, log likelihood ratio test). C) The environmental model (Equation 7) significantly predicts betweeness centrality (R&lt;sup&gt;2&lt;/sup&gt; = 0.31432, p= 0.0008). The behavioural model (Equation 8) significantly predicts betweeness centrality (R&lt;sup&gt;2&lt;/sup&gt; = 0.52325, p &lt; 0.0001). The combined model (Equation 9) does not significantly improve the prediction of betweeness centrality compared to behavioural model alone (Combined model AICc &gt; Behavioural model AICc; p = 0.0504, log likelihood ratio test). D) The environmental model (Equation 10) significantly predicts global efficiency (R&lt;sup&gt;2&lt;/sup&gt; = 0.35261, p = 0.0003). The behavioural model (Equation 11) significantly predicts global efficiency (R&lt;sup&gt;2&lt;/sup&gt; = 0.74972). The combined model (Equation 12) significantly improves prediction of global efficiency (Combined AICc &lt; Behavioural AICc; p &lt; 0.0001, log likelihood ratio test). Equation 1: y = 0.63 &#x2013; 0.092*[PC2] + 0.064*[PC3] + 0.11*[PC5]. Equation 2: y = 0.53 &#x2013; 0.081*[movement] + 0.023*[interaction duration] + 0.00018*[number of interactions]. Equation 3: y = 0.56 &#x2013; 0.068*[PC2] + 0.045*[PC3] + 0.086*[PC5] &#x2013; 0.071*[movement] + 0.11 *[interaction duration] + 0.00019*[number of interactions]. Equation 4: y = 0.047 &#x2013; 0.069*[PC1] &#x2013; 0.18*[PC2] + 0.21*[PC5]. Equation 5: y = 0.73 &#x2013; 0.19*[movement] &#x2013; 0.0072*[angle] + 0.04*[interaction duration] + 0.00038*[number of interactions]. Equation 6: y = 0.71 &#x2013; 0.014*[PC1] &#x2013; 0.15*[PC2] + 0.14*[PC5] &#x2013; 0.16*[movement] &#x2013; 0.0067*[angle] + 0.021*[duration] + 0.00038*[number of interactions]. Equation 7: y = 0.99 + 0.07*[PC1] + 0.057*[PC2] &#x2013; 0.13*[PC5]. Equation 8: y = 0.096 &#x2013; 0.28*[social distance] + 0.25*[time] &#x2013; 0.031*[interaction duration] + 0.0002*[number of interactions] + 2.91*[reciprocation]. Equation 9: y = 0.54 + 0.0065*[PC5] + 0.055*[PC2] &#x2013; 0.11*[PC5] &#x2013; 0.39*[social distance] + 0.23*[time] &#x2013; 0.01*[interaction duration] + 0.0002*[number of interactions] + 1.94*[reciprocation]. Equation 10: y = &#x2212;3.64 + 0.55[PC2] &#x2013; 0.34[PC3] &#x2013; 0.58[PC5]. Equation 11: y = 2.03 + 0.31*[movement] &#x2013; 0.14*[interaction duration] &#x2013; 0.0007*[number of interactions] &#x2013; 13*[reciprocation]. Equation 12: y = 2.47 + 0.33*[PC3] &#x2013; 0.38*[PC5] + 0.26*[movement] &#x2013; 0.078*[interaction duration] &#x2013; 0.0007*[number of interactions] &#x2013; 15*[reciprocation]. See also Figures S2-S4 for environmental and behavioural models of other behavioural elements.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 5:" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2019/09/20/776708/F5.medium.gif" width="297" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 5:" src="https://www.biorxiv.org/content/biorxiv/early/2019/09/20/776708/F5.medium.gif" width="297" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/09/20/776708/F5.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 5:" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/09/20/776708/F5.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 5:</span> <span class="caption-title">Environmental, behavioural and combined stepwise regression models of assortativity, clustering coefficient, betweeness centrality and global efficiency.</span><p id="p-21" class="first-child">For all regressions, each data point represents the mean SIN measure for a single species. The mean SIN measure for groups of male flies and female flies were pooled into each regression and are labeled with red and blue points, respectively. Each solid trend line indicates line of best fit and dashed lines indicate 95% confidence interval of the model. For each model, the multivariate linear equation can be found below. A) The environmental model (Equation 1) significantly predicts assortativity (R<sup>2</sup> = 0.44447, p &lt; 0.0001). The behavioural model (Equation 2) significantly predicts assortativity (R<sup>2</sup> = 0.49493, p &lt; 0.0001). The combined model (Equation 3) significantly improves the prediction of assortativity compared to behavioural model alone (Combined model AICc &lt; Behavioural model AICc; p = 0.0001, log likelihood ratio test). B) The environmental model (Equation 4) significantly predicts clustering coefficient (R<sup>2</sup> = 0.45258. p &lt; 0.0001). The behavioural model (Equation 5) significantly predicts clustering coefficient (R<sup>2</sup> = 0.56293, p &lt; 0.0001). The combined model (Equation 6) significantly improves the prediction of clustering coefficient compared to behavioural model alone (Combined model AICc &lt; Behavioural model AICc; p &lt; 0.0001, log likelihood ratio test). C) The environmental model (Equation 7) significantly predicts betweeness centrality (R<sup>2</sup> = 0.31432, p= 0.0008). The behavioural model (Equation 8) significantly predicts betweeness centrality (R<sup>2</sup> = 0.52325, p &lt; 0.0001). The combined model (Equation 9) does not significantly improve the prediction of betweeness centrality compared to behavioural model alone (Combined model AICc &gt; Behavioural model AICc; p = 0.0504, log likelihood ratio test). D) The environmental model (Equation 10) significantly predicts global efficiency (R<sup>2</sup> = 0.35261, p = 0.0003). The behavioural model (Equation 11) significantly predicts global efficiency (R<sup>2</sup> = 0.74972). The combined model (Equation 12) significantly improves prediction of global efficiency (Combined AICc &lt; Behavioural AICc; p &lt; 0.0001, log likelihood ratio test). Equation 1: y = 0.63 – 0.092*[PC2] + 0.064*[PC3] + 0.11*[PC5]. Equation 2: y = 0.53 – 0.081*[movement] + 0.023*[interaction duration] + 0.00018*[number of interactions]. Equation 3: y = 0.56 – 0.068*[PC2] + 0.045*[PC3] + 0.086*[PC5] – 0.071*[movement] + 0.11 *[interaction duration] + 0.00019*[number of interactions]. Equation 4: y = 0.047 – 0.069*[PC1] – 0.18*[PC2] + 0.21*[PC5]. Equation 5: y = 0.73 – 0.19*[movement] – 0.0072*[angle] + 0.04*[interaction duration] + 0.00038*[number of interactions]. Equation 6: y = 0.71 – 0.014*[PC1] – 0.15*[PC2] + 0.14*[PC5] – 0.16*[movement] – 0.0067*[angle] + 0.021*[duration] + 0.00038*[number of interactions]. Equation 7: y = 0.99 + 0.07*[PC1] + 0.057*[PC2] – 0.13*[PC5]. Equation 8: y = 0.096 – 0.28*[social distance] + 0.25*[time] – 0.031*[interaction duration] + 0.0002*[number of interactions] + 2.91*[reciprocation]. Equation 9: y = 0.54 + 0.0065*[PC5] + 0.055*[PC2] – 0.11*[PC5] – 0.39*[social distance] + 0.23*[time] – 0.01*[interaction duration] + 0.0002*[number of interactions] + 1.94*[reciprocation]. Equation 10: y = −3.64 + 0.55[PC2] – 0.34[PC3] – 0.58[PC5]. Equation 11: y = 2.03 + 0.31*[movement] – 0.14*[interaction duration] – 0.0007*[number of interactions] – 13*[reciprocation]. Equation 12: y = 2.47 + 0.33*[PC3] – 0.38*[PC5] + 0.26*[movement] – 0.078*[interaction duration] – 0.0007*[number of interactions] – 15*[reciprocation]. See also Figures S2-S4 for environmental and behavioural models of other behavioural elements.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-6" class="subsection"><h3>Behavioural models of social behaviour</h3><p id="p-22">To investigate how behavioural elements predict SINs, we fit all the behavioural element variables (movement, social spacing, pairwise interactions) to each SIN measure through stepwise regressions. We found the resultant behavioural models to be strongly predictive for assortativity (R<sup>2</sup> = 0.495), clustering coefficient (R<sup>2</sup> = 0.587), betweeness centrality (R<sup>2</sup> = 0.523) and global efficiency (R<sup>2</sup> = 0.760; <a id="xref-fig-5-2" class="xref-fig" href="#F5">Figure 5</a>). Interestingly, movement contributed to all behavioural models for SIN measures, except betweeness centrality (<a id="xref-fig-5-3" class="xref-fig" href="#F5">Figure 5</a>). We decided to investigate our behavioural hierarchy by correlating higher-order behaviours to their respective lower-order behaviours (<a id="xref-fig-1-2" class="xref-fig" href="#F1">Figure 1</a>; see Methods for explanation of lower- and higher-order behaviours). We found these behavioural models to be strongly predictive (number of interactions: R<sup>2</sup> = 0.538, interaction duration: R<sup>2</sup> = 0.553, and reciprocation: R<sup>2</sup> = 0.879; Figure S3-S4). Together, this suggests that more complex higher-order behaviours in <em>Drosophila</em> can be predicted by some lower-order behaviours but not all lower-order behaviours (such as movement) can predict complex social structures.</p></div><div id="sec-7" class="subsection"><h3>Combined models of social behaviour</h3><p id="p-23">We tested how much variation in each SIN measure can be explained by both the environmental and behavioural models simultaneously in a “combined model”. We hypothesized that comparing the combined model to the behavioural model through a likelihood ratio test would determine whether the combination of the two models outperforms the behavioural model alone. We found that the combined models significantly improve the fit over the behavioural models for assortativity (R<sup>2</sup> = 0.673, p = 0.0001; likelihood ratio test), clustering coefficient (R<sup>2</sup> = 0.757, p &lt; 0.0001) and global efficiency (R<sup>2</sup> = 0.846, p &lt; 0.0001; <a id="xref-fig-5-4" class="xref-fig" href="#F5">Figure 5</a>). The combined model for betweeness centrality remains nearly identical to the behavioural model (Combined: R<sup>2</sup> = 0.569, Behavioural: R<sup>2</sup> = 0.523), suggesting that the behavioural data alone is sufficient at predicting betweeness centrality. We repeated this analysis by generating combined models for the pairwise interaction variables (level 3; interaction duration, number of interactions and reciprocation). We found that the combined models do not significantly outperform the behavioural models for any of these variables (Figure S4), indicating that movement (level one) and social space variables (level two) are sufficient at predicting pairwise interactions (level three). Altogether, this suggests that the best prediction of SIN measures requires the consideration of both environmental and behavioural factors. However, betweeness centrality and pairwise interactions can be sufficiently predicted by behaviour alone.</p></div></div><div class="section" id="sec-8"><h2 class="">Discussion</h2><p id="p-24">In this study, we set out to determine which factors shape social organization by studying social networks in <em>Drosophila</em>. To do this, we used a species-wide comparative method to model climate and behaviour to the evolution of SINs. Here, we show that all 20 drosophilid species used in this study form SINs (Figure S1) and that the properties of these SINs vary across species (<a id="xref-fig-3-3" class="xref-fig" href="#F3">Figure 3</a>). This suggests that all drosophilids self-organize but not all drosophilids self-organize similarly. We show the same results from a pilot experiment we performed, suggesting we are capturing a reproducible, stable phenotype (Figure S5-S6). We found low phylogenetic signal for the measured SIN properties and speculate that group organization in drosophilids is the result of conserved mechanisms stemming deep in evolutionary time.</p><p id="p-25">We found that social distance, which represents the typical distance at which flies communicate, displays evidence of high phylogenetic signal according to Pagel’s λin males and females and Blomberg’s K in females (<a id="xref-table-wrap-2-2" class="xref-table" href="#T2">Table 2</a>). According to our data, most species tend to interact within 1-3 body lengths, and this range of interaction distance has been reported and applied by other researchers [<a id="xref-ref-5-2" class="xref-bibr" href="#ref-5">5</a>, <a id="xref-ref-13-2" class="xref-bibr" href="#ref-13">13</a>, <a id="xref-ref-28-1" class="xref-bibr" href="#ref-28">28</a>–<a id="xref-ref-30-1" class="xref-bibr" href="#ref-30">30</a>]. The distance at which flies socially interact is conserved within different clades of our tree, suggesting that closely related drosophilids evolved similar mechanisms for social communication (<a id="xref-fig-4-4" class="xref-fig" href="#F4">Figure 4</a>). We hypothesized that the social distance at which flies interact may correlate with the physical limitation of a fly’s anatomical ability to extend its legs towards another fly, which contain gustatory receptors [<a id="xref-ref-31-1" class="xref-bibr" href="#ref-31">31</a>]. In flies, it is thought that touching conspecifics leads to the exchange of pheromones [<a id="xref-ref-32-1" class="xref-bibr" href="#ref-32">32</a>, <a id="xref-ref-33-1" class="xref-bibr" href="#ref-33">33</a>], which can serve as cues about a flies’ social environment [<a id="xref-ref-34-1" class="xref-bibr" href="#ref-34">34</a>, <a id="xref-ref-35-1" class="xref-bibr" href="#ref-35">35</a>]. It is also possible that flies communicate with each other using both touch and taste. We show that relative leg length does positively correlate with social distance (<a id="xref-fig-4-5" class="xref-fig" href="#F4">Figure 4</a>). This suggests that how flies space themselves in a social setting may be determined by the length of their legs relative to their body size and we postulate that their leg length determines how close they need to be to conspecifics to touch. Evidence of flies touching has been documented in <em>D. paramelanica</em> [<a id="xref-ref-36-1" class="xref-bibr" href="#ref-36">36</a>], <em>D. melanogaster</em> [<a id="xref-ref-13-3" class="xref-bibr" href="#ref-13">13</a>] and witnessed in this study across all species. We reason that social interactions within a species may influence the evolution of their anatomy, such as leg length. By maintaining the ability to interact and communicate with conspecifics, individuals are better equipped to survey their environment [<a id="xref-ref-34-2" class="xref-bibr" href="#ref-34">34</a>, <a id="xref-ref-35-2" class="xref-bibr" href="#ref-35">35</a>], locate potential mates and find food [<a id="xref-ref-2-2" class="xref-bibr" href="#ref-2">2</a>]. The evolution of leg length and social distance in <em>Drosophila</em> may be one example of this.</p><p id="p-26">We found that the past climatic pressures, behavioural elements, and a combination of these variables can predict SIN structure in drosophilids. Although the behavioural elements we investigate in this study are also behaviours that comprise SINs, they are not necessarily predictive of SIN structure. For example, different sensory mutants in <em>D. melanogaster</em> show significantly different movement and rates of interaction. However, the sensory mutants form SINs with the same clustering coefficient, assortativity, and betweeness centrality [<a id="xref-ref-13-4" class="xref-bibr" href="#ref-13">13</a>]. In this study, we show that different combinations of behavioural elements can be used to predict SIN structure across different <em>Drosophilia</em> species.</p><p id="p-27">We found that past climatic pressures correlate to the behavioural elements that comprise drosophilid SINs and to the different properties of their SINs (Figure S2-S4, <a id="xref-fig-5-5" class="xref-fig" href="#F5">Figure 5</a>, respectively). These climatic variables could be indicative of other past pressures of their environment, such as food availability [<a id="xref-ref-37-1" class="xref-bibr" href="#ref-37">37</a>, <a id="xref-ref-38-1" class="xref-bibr" href="#ref-38">38</a>], species richness [<a id="xref-ref-39-1" class="xref-bibr" href="#ref-39">39</a>] and intraspecific density [<a id="xref-ref-40-1" class="xref-bibr" href="#ref-40">40</a>]. Thus, we argue that the ways in which drosophilids behave and the structures of their groups have evolved, at least in part, by pressures exerted by their environments. It has been shown that how individuals organize in groups can influence their fitness and health [<a id="xref-ref-3-2" class="xref-bibr" href="#ref-3">3</a>–<a id="xref-ref-5-3" class="xref-bibr" href="#ref-5">5</a>]. Thus, understanding the influences on social organization is imperative for recognizing that disrupting these influences can impact the survival of a species. It has been speculated that 40% of the insect populations on earth will become extinct in the next few decades and that climate change will be one factor contributing to their destruction [<a id="xref-ref-41-1" class="xref-bibr" href="#ref-41">41</a>]. However, not all declining insect populations can be attributable to climate change alone and, in some cases, factors contributing to the observed differences in insect populations remain unexplained [<a id="xref-ref-41-2" class="xref-bibr" href="#ref-41">41</a>, <a id="xref-ref-42-1" class="xref-bibr" href="#ref-42">42</a>]. Seeing as social organization has been shown to contribute to health and reproductive success, it is rational to speculate that the breakdown of social organization, brought on by climate change, is contributing to the extinction of these insects.</p><p id="p-28">In this study, we found that movement varies across drosophilid species. Interestingly, the most sedentary species in this study tend to organize themselves into SINs with high clustering coefficient (<a id="xref-fig-3-4" class="xref-fig" href="#F3">Figure 3</a>). This could lead to a social structure where flies interact more equally [<a id="xref-ref-43-1" class="xref-bibr" href="#ref-43">43</a>]. Some studies suggest that living in more egalitarian groups correlates with longer lifespans [<a id="xref-ref-44-1" class="xref-bibr" href="#ref-44">44</a>] and, interestingly, some of the most sedentary species in our study, such as <em>D. mojavensis</em> and <em>D. virilis</em>, have been reported to have some of the longest life spans across the <em>Drosophila</em> genus [<a id="xref-ref-45-1" class="xref-bibr" href="#ref-45">45</a>]. We argue that the differences in group organization seen across drosophilid species is influenced by their different geographic origins and that these differences may correlate with life span. In the future, we may be able to evaluate properties of social networks and predict health outcomes of populations, such as longevity.</p><p id="p-29">Although movement correlated with three SIN measures, it did not correlate with betweeness centrality (<a id="xref-fig-5-6" class="xref-fig" href="#F5">Figure 5</a>), a SIN measure previously reported to be heritable in <em>Drosophila</em> [<a id="xref-ref-13-5" class="xref-bibr" href="#ref-13">13</a>] and humans [<a id="xref-ref-15-2" class="xref-bibr" href="#ref-15">15</a>]. Moreover, when we combined the environmental and behavioural models for betweeness centrality, the combined model did not significantly improve the correlation compared to the behavioural model alone (<a id="xref-fig-5-7" class="xref-fig" href="#F5">Figure 5</a>). It appears that the climate data adds no value to maximizing the prediction of betweeness centrality, and that the behavioural elements are sufficient. We speculate that betweeness centrality, as a phenotype, may respond primarily to selective pressures of the social environment. In one study, researchers found that the willingness of individuals to share or withhold information from competitors is influenced by the composition of the social environment, which they coined “audience effects” [<a id="xref-ref-46-1" class="xref-bibr" href="#ref-46">46</a>]. In <em>Drosophila</em>, audience effects may explain why flies signal fertile oviposition sites [<a id="xref-ref-6-2" class="xref-bibr" href="#ref-6">6</a>, <a id="xref-ref-7-2" class="xref-bibr" href="#ref-7">7</a>] and the presence of predators to naïve flies [<a id="xref-ref-10-2" class="xref-bibr" href="#ref-10">10</a>, <a id="xref-ref-11-2" class="xref-bibr" href="#ref-11">11</a>]. In the wild, flies may readily organize and transmit valuable information about their environment if the benefits of cooperation exceed the cost of competition. This may facilitate dynamic behavioural strategies which vary across individuals, populations and species. Overall, our results suggest that betweeness centrality may measure a group-level phenotype that is likely genetically heritable and has been selected by pressures of the social environment. Future network studies on <em>Drosophila</em>, or in other animals, may be able to determine gene(s) responsible for regulating the betweeness centrality phenotype, which could lead to the discovery of conserved social mechanisms that may be found in other organisms.</p><p id="p-30">We highlight how phylogeny, environment and simple behaviours influence the social organization of drosophilids. All 20 of the drosophilids we studied socially organize and form SINs, allowing for the possibility of dissecting the evolution of sociality in the future. We speculate that social communication across drosophilids is conserved, given that drosophilids are known to encounter and interact with other species [<a id="xref-ref-47-1" class="xref-bibr" href="#ref-47">47</a>]. Recent work uncovered that drosophilids can socially signal the presence of predators to an unrelated heterospecific species [<a id="xref-ref-11-3" class="xref-bibr" href="#ref-11">11</a>], suggesting that flies communicate in “dialects” with each other. When considering complex social organization behaviours, the ecological pressures of a species’ environment play a large role in shaping these phenotypes. Perhaps, one day, genes that influence social organization will be uncovered in <em>Drosophila</em> and we will consider social organization as a behavioural phenotype that emerges from a deeply conserved genetic toolkit [<a id="xref-ref-48-1" class="xref-bibr" href="#ref-48">48</a>].</p></div><div class="section" id="sec-10"><h2 class="">Author Contributions</h2><p id="p-32">J.A.J. acquired the data, analyzed the data and wrote the first draft of this manuscript. R.R. assisted with some experiments and participated in editing and writing the final draft of this manuscript. J.S. wrote some of the scripts for data analysis, provided statistical insights and participated in data analysis. J.A.J., J.S., J.D.L. designed the experiments and edited the manuscript.</p></div><div class="section" id="sec-11"><h2 class="">Declaration of Interests</h2><p id="p-33">The authors declare no competing interests.</p></div><div class="section" id="sec-12"><h2 class="">STAR⍰Methods</h2><div class="section" id="sec-13"><h3>KEY RESOURCES TABLE</h3><div id="T3" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/919985/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup"><a href="/highwire/markup/919985/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/919985" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div></div></div></div><div class="section" id="sec-14"><h3>LEAD CONTACT AND MATERIALS AVAILABILITY</h3><p id="p-34">Further information and requests for resources and reagents should be directed to and will be fulfilled by the lead contact Joel Levine (<span class="em-link"><span class="em-addr">joel.levine{at}utoronto.ca</span></span>).</p><p id="p-35">This study did not generate new unique reagents.</p></div><div class="section" id="sec-15"><h3>EXPERIMENTAL MODEL AND SUBJECT DETAILS</h3><div class="section" id="sec-16"><h4>Fly stocks</h4><p id="p-36">The majority of the species used for the SIN assay were purchased from the <em>Drosophila</em> Species Stock Centre (<a href="http://blogs.cornell.edu/drosophila/orders/">http://blogs.cornell.edu/drosophila/orders/</a>) as shown in Table S1. The Canton-S strain was used to represent <em>D. melanogaster</em>. The <em>D. yakuba</em> stock was a gift from N. Gompel. The source of the <em>D. mojavensis</em> and <em>D. sechellia</em> stocks are unknown, but these stocks were confirmed to be the correct species through documented phenotypic markers [<a id="xref-ref-17-2" class="xref-bibr" href="#ref-17">17</a>].</p></div></div><div class="section" id="sec-17"><h3>METHOD DETAILS</h3><div class="section" id="sec-18"><h4>Video acquisition</h4><p id="p-37">SIN data were acquired as outlined previously [<a id="xref-ref-13-6" class="xref-bibr" href="#ref-13">13</a>]. To summarize, all fly stocks were maintained in bottles with 40 mL of media containing cornmeal, wheat germ, soy flour, molasses, sucrose, glucose, yeast, agar, propionic acid and Tegosept. These bottles were stored in a Percival incubator (model I-36LL) set to 25°C with a 12 h L:D cycle. For all species, virgin male and female flies were collected through light CO<sub>2</sub> anaesthesia. Male and female flies were housed separately in groups of 12-16 individuals within vials containing 8 mL of media. All flies were reared for 3 days in an incubator controlled under the same temperature and photoperiod described above. Afterwards, 12 flies from each vial were gently aspirated into circular arenas (60 mm diameter, 2 mm depth) and filmed for 30 minutes using Firefly MV cameras (Point Grey). All videos were filmed at the same time of day (9.5-10.5 h after initiation of light phase during photoperiod) to control for the clock-controlled locomotor activity of flies. All filming was completed inside a Biochamber with controlled temperature (25°C) and humidity (65% RH). After filming, all flies were anaesthetized with CO<sub>2</sub> and discarded. The position, orientation and identity of all flies in each video were tracked through machine vision software (Ctrax; versions 0.4.2 &amp; 0.5.19b). Errors in tracking were manually corrected through the fixerrors (version 0.2.3) MATLAB package to correct for: swapping of fly identities, errors in the orientation of the head to abdomen region of the fly tracks and drastic fluctuations in the large major axis of the fly tracks. Data collection was performed separately for groups of male flies and for groups of female flies and we will use the terms “male dataset” and “female dataset”, respectively. In total, we collected, tracked and fixed the errors in 1000 video trials, totaling approximately forty million frames of video, across the male and female datasets of the 20 drosophilid species.</p></div><div class="section" id="sec-19"><h4>Pilot data</h4><p id="p-38">An additional male and female dataset containing 399 videos across 7 species was collected and analyzed. This collection of videos was acquired between July 2014 and August 2015. The following species are represented in this pilot data: <em>D. melanogaster</em> (Canton-S), <em>D. simulans</em>, <em>D. sechellia</em>, <em>D. yakuba</em>, <em>D. pseudoobscura</em>, <em>D. mojavensis</em>, and <em>D. paramelanica</em>. With this pilot data, we were able to test the robustness of SIN measures between 5 species common to both independent data collections. The stocks for these species were the same as those used in the 20 species-wide dataset. The relative species differences, for each dataset, were statistically tested through a Kruskal-Wallis one-way ANOVA in MATLAB. The replication of these datasets were visualized through boxplots that display the distribution of the SIN scores for each species (Figure S5-S6).</p></div><div class="section" id="sec-20"><h4>Automated estimation of social spacing interaction criteria</h4><p id="p-39">All interaction criteria used to generate SINs in the 20 species-wide data (referred to as extended data below) are shown in Table S2. The distance, angle and time interaction criteria were estimated for each species using an algorithm published by Schneider and Levine [<a id="xref-ref-50-1" class="xref-bibr" href="#ref-50">50</a>]. Stated briefly, the algorithm analyzes the spatial positions of every fly in all tracked videos. The algorithm eliminates background noise from flies by analyzing spatial positions of “virtual trials” which consist of fly tracks randomly sampled from separate videos. With that background subtraction, the algorithm identifies distance, angle and time criteria that are over-represented in videos of flies socially interacting compared to the non-social virtual trials. Initially, the algorithm provided interaction criteria that contradicted personal observations of each species’ spatial patterning. The following adjustments were made to improve the performance of the algorithm:</p><ol class="list-ord " id="list-1"><li id="list-item-1"><p id="p-40">Previously the algorithm computed an “inter-fly distance” defined as the interval where the distribution for distance measures was positive when subtracted from the virtual trials. This interval was the starting point to locate interaction hotspots on the social space heat map (see S1-S25 for a visualization of these heat maps). Here, we did not compute the inter-fly distance and established the initial interaction hotspots by targeting the 95<sup>th</sup> percentile instead of the third quartile. As reported previously [<a id="xref-ref-50-2" class="xref-bibr" href="#ref-50">50</a>], angle and distance bins were increased until the mean of the enclosed region on the heat maps began to decrease.</p></li><li id="list-item-2"><p id="p-41">Previously the algorithm computed the time criteria by recording the time elapsed when flies fulfilled the calculated distance and angle criteria. Background noise would be removed by subtracting the distribution of time values from the virtual trial time distribution. The first positive time bin, representing the minimal time value over-represented in videos of socially interacting flies, was utilized as the time criteria. Here, the time criteria were acquired by increasing the number of bins on the normalized time frequency distribution until the mean of the enclosed bins decreases.</p></li></ol><p id="p-42">For each species, this algorithm was performed 500 times where 15 videos were randomly sampled with replacement in order to generate a 95% confidence interval and median estimate of each species’ interaction criteria. Both the pilot data (see below) and extended data were analyzed with this algorithm. For a few species, the estimated interaction criteria produced questionable results that contradicted our own observations. The following changes were made to amend the computerized estimates:</p><ol class="list-ord " id="list-2"><li id="list-item-3"><p id="p-43"><em>D. mauritiana</em> males: the angle criterion was relaxed to 121.5 degrees since that angle encompasses more hotspots in their social space heat map (data not shown).</p></li><li id="list-item-4"><p id="p-44"><em>D. santomea</em> females: the distance was restricted to the lower limit of the 95% confidence interval, the angle was relaxed to the upper limit of the 95% confidence interval, and time was restricted to the lower limit of the 95% confidence interval (data not shown).</p></li><li id="list-item-5"><p id="p-45"><em>D. immigrans</em> males: The automated time criterion was estimated to be 0 s, clearly a failure of the algorithm to provide a meaningful criterion. As a result, 103 social interactions of male <em>D. immigrans</em> were randomly flagged throughout the videos and the duration of these interactions (in units of seconds) was recorded in a histogram. The minimum value of 0.53 s was used to generate SINs. This value falls in a bin that contains ∼ 10% of the interactions observed, making it a reliable approximation of a minimum social interaction time criterion (data not shown).</p></li><li id="list-item-6"><p id="p-46"><em>D. mojavensis</em> males and females: the angle criterion computed in the extended data was narrower than the angle criterion estimated from the pilot data. Because the pilot data angle criterion encompasses a larger distribution of hot spots that are consistent with the extended data heat maps, the pilot data angle criterion was used for SIN analysis in the extended data (Table S2; data not shown).</p></li></ol></div><div class="section" id="sec-21"><h4>Estimation of social interaction networks and other behavioural measures</h4><p id="p-47">We considered that different drosophilid species may interact differently. We considered how the inter-individual distance, angle and time, collectively referred to as the social spacing parameters (Figure S2), may vary across species. To account for this, we utilized an algorithm that estimates these parameters based on tracked videos (see above). Each species’ distance, angle and time parameters for both the male and female datasets are listed in Table S2. These species-specific and sex-specific social spacing parameters allowed us to control for variation in social interactions when generating SINs. Additionally, the social spacing parameters can also be used as a measure to classify the ways that different species interact.</p><p id="p-48">All SINs were generated as previously described [<a id="xref-ref-13-7" class="xref-bibr" href="#ref-13">13</a>]. To summarize, all SINs are iterative networks that comprise 33 unique interactions, which represents a network density of 25% (25% of 132 unique possible interactions in a group of 12 individuals). The number of iterative networks in a single 30-minute video may vary and is proportional to the number of unique interactions in a video. Most species formed at least one SIN iteration in over 80% of their respective video trials (Figure S1). We assess four SIN measures that we view as social organization phenotypes: i) <em>assortativity</em>, defined as the probability of an individual interacting with another individual with a similar degree (degree is defined as the number of incoming and outgoing connections to a single node); ii) <em>clustering coefficient</em>, defined as a measure of how interconnected neighbours are to one another; iii) <em>betweeness centrality</em>, defined as the number of shortest paths that traverse an individual, indicating the relative importance of an individual for maintaining the cohesion of the network; and iv) <em>global efficiency</em>, defined as a measure of redundant pathways, indicating the efficiency of information flow throughout the network [<a id="xref-ref-13-8" class="xref-bibr" href="#ref-13">13</a>, <a id="xref-ref-14-2" class="xref-bibr" href="#ref-14">14</a>]. Explanations of these SIN measures that correspond to the iterative approach are described in <a id="xref-table-wrap-1-4" class="xref-table" href="#T1">Table 1</a>. Here we speak about these network measures as they apply to a group on average. The four SIN measures are expressed as z-scores, which normalizes the networks to control for degree distribution. To do this, we generated 10000 random networks and calculated the z-score as follows:</p><span class="disp-formula" id="disp-formula-1"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2019/09/20/776708/embed/graphic-9.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2019/09/20/776708/embed/graphic-9.gif"/></noscript></span>

</span><p id="p-49">For each SIN measure, we present the mean z-score, averaged from the distribution of videos for each species in the male and female dataset (<a id="xref-fig-3-5" class="xref-fig" href="#F3">Figure 3</a>; Schneider et al., 2012 methods [<a id="xref-ref-13-9" class="xref-bibr" href="#ref-13">13</a>]).</p><p id="p-50">In addition to the SIN measures, other behavioural elements were estimated from the tracked videos: i) <em>movement</em>, defined as the mean locomotor activity (millimetres/second) of the flies in a single video; ii) <em>interaction duration</em>, defined as the mean duration of the social interactions (based on the distance, angle and time parameters) in a single video; iii) <em>reciprocation</em>, defined as the proportion of social interactions reciprocated in a single video; and iv) <em>number of interactions</em>, defined as the total number of social interactions in a single video. See <a id="xref-table-wrap-1-5" class="xref-table" href="#T1">Table 1</a> for a summary of these behavioural elements. Like the SIN measures, the mean of each behavioural element is calculated from the distribution of videos for each species in the male and female dataset (<a id="xref-fig-2-6" class="xref-fig" href="#F2">Figure 2</a>).</p></div><div class="section" id="sec-22"><h4>Phylogenetic signal</h4><p id="p-51">To determine whether the drosophilid species’ social behaviour exhibit phylogenetic signal (i.e. differences in a given trait reflect the species’ degree of common ancestry), we applied two tests: Blomberg’s K [<a id="xref-ref-24-4" class="xref-bibr" href="#ref-24">24</a>] and Pagel’s λ [<a id="xref-ref-25-3" class="xref-bibr" href="#ref-25">25</a>]. Both phylogenetic signal tests evaluate the extent to which the observed average trait data across species adhere to a model of Brownian motion across a phylogenetic tree, which approximates the expectation under evolution due to random genetic drift [<a id="xref-ref-53-1" class="xref-bibr" href="#ref-53">53</a>, <a id="xref-ref-54-1" class="xref-bibr" href="#ref-54">54</a>]. Both the K and λ values typically range on a scale from 0-1, where 0 indicates low phylogenetic signal (trait evolution does not adhere to Brownian motion) and 1 indicates high phylogenetic signal (trait evolution adheres to Brownian motion; see Blomberg et al. 2003 [<a id="xref-ref-24-5" class="xref-bibr" href="#ref-24">24</a>]). The phylogenetic tree applied to these analyses was a published drosophilid phylogeny that we pruned to include our species sample [<a id="xref-ref-21-3" class="xref-bibr" href="#ref-21">21</a>]. Since <em>D. novamexicana</em> was not included in this tree, its placement and branch length was standardized based on another published phylogenetic tree of the <em>virilis</em> group species [<a id="xref-ref-55-1" class="xref-bibr" href="#ref-55">55</a>]. The tree was made ultrametric in R through the ‘chronos’ function (ape package) for all subsequent phylogenetic comparative analyses.</p></div><div class="section" id="sec-23"><h4>Geographic coordinate acquisition for the 20 species</h4><p id="p-52">To test the influence of climatic environment on species variation in SINs, climate data was acquired from each species geographic distribution. Many of the species purchased from the <em>Drosophila</em> Species Stock Centre had records of the geographic coordinates, and/or the city and country the stock was collected from. For the stocks that did not have precise coordinates listed on the stock center, the rough latitude and longitude of the recorded locations were noted from a search on Google Maps. The coordinates estimating each species’ stock geographic origin are listed in Table S1. However, we were unsure of the precise geographic origin for the following stocks: <em>D. sechellia</em>, <em>D. yakuba</em>, <em>D. erecta</em>, <em>D. mojavensis</em>. As a result, we utilized Taxodros, a database that contains records of coordinates pin-pointing thousands of collection sites for drosophilid species (<a href="http://www.taxodros.uzh.ch/">http://www.taxodros.uzh.ch/</a>). We acquired all the Taxodros coordinates listed for each of our 20 species through custom MATLAB scripts. For the species stocks that had known coordinates from the stock centre or Google Maps, we filtered out all Taxodros coordinates beyond +/- 2 latitudinal and longitudinal units from the known coordinates (Table S1). Fortunately, the species without known coordinates (<em>D. mojavensis</em>, <em>D. yakuba</em>, <em>D. erecta</em>, <em>D. sechellia</em>), have narrow geographic distributions confined to one continent. Therefore, the mean latitude and longitude were calculated from all Taxodros coordinates to approximate the center of their geographic distribution, and these coordinates are listed in Table S1.</p><p id="p-53">The Taxodros database also aided in acquiring more accurate estimates of the climate variables used in the environmental models (<a id="xref-fig-5-8" class="xref-fig" href="#F5">Figure 5</a>). Rather than estimating a single measure for all 19 climate variables using the coordinates in Table S1, we acquired distributions for the 19 climate variables from the filtered Taxodros coordinates. For all species, the mean value was calculated from the distribution of each climate variable, and these mean measures were incorporated into the principal component analysis. The first 5 principal components accounted for 92% of the variance across all the climate data (Figure S7).</p></div><div class="section" id="sec-24"><h4>Estimation of environmental variables</h4><p id="p-54">We estimated climate data from latitudinal and longitudinal coordinates specific to the capture site of each drosophilid species stock (<a id="xref-fig-1-3" class="xref-fig" href="#F1">Figure 1</a>, Table S1). For each set of coordinates, we obtained 19 climate variables from the WorldClim database (<a href="http://www.worldclim.org">http://www.worldclim.org</a>; [<a id="xref-ref-26-3" class="xref-bibr" href="#ref-26">26</a>]). All climate variables represent the predicted climate from the mid-Holocene period, approximately 3000 to 8000 years ago [<a id="xref-ref-56-1" class="xref-bibr" href="#ref-56">56</a>]. We emphasize that this climate data best captures past selective pressures that each drosophilid species adapted to. The following variables, representing temperature and precipitation patterns, were obtained: <em>annual mean temperature</em> (BIO1)<em>, mean diurnal range</em> (BIO2; calculated as mean of monthly(max temp – min temp)), <em>isothermability</em> (BIO3; calculated as BIO2/BIO7 * 100), <em>temperature seasonality</em> (BIO4), <em>maximum temperature of warmest month</em> (BIO5), <em>minimum temperature of coldest month</em> (BIO6), <em>temperature annual range</em> (BIO7; calculated as BIO5-BIO6), <em>mean temperature of wettest quarter</em> (BIO8), <em>mean temperature of driest quarter</em> (BIO9), <em>mean temperature of warmest quarter</em> (BIO10), <em>mean temperature of coldest quarter</em> (BIO11), <em>annual precipitation</em> (BIO12), <em>precipitation of wettest month</em> (BIO13), <em>precipitation of driest month</em> (BIO14), <em>precipitation seasonality</em> (BIO15), <em>precipitation of wettest quarter</em> (BIO16), <em>precipitation of driest quarter</em> (BIO 17), <em>precipitation of warmest quarter</em> (BIO 18), <em>precipitation of coldest quarter</em> (BIO 19).</p></div><div class="section" id="sec-25"><h4>Leg measurements</h4><p id="p-55">The front, middle and rear legs were measured from a minimum of 10 individuals for all 20 drosophilid species (Table S3). First, we anaesthetized the flies with carbon dioxide and carefully severed each leg above the femur using forceps. Each leg was placed on a flat surface and covered with a coverslip. An image was acquired for each leg using a Zeiss SteREO Discovery.V12 microscope and Zeiss ZEN (2011) software. The leg was measured in µm using Zeiss ZEN (2011) built-in measurement tools. The femur measurement began where the trochanter met the femur and ended at the tibia. The tibia was measured from the beginning of the tibia to the end of the tibia. The tarsi were measured from the beginning of the tarsi and ended in the middle of the claw. For each leg, the length of the femur, tibia and tarsal fragments were summed. The sum of the three legs were then averaged to provide us with the total leg length.</p></div><div class="section" id="sec-26"><h4>Body size measurements</h4><p id="p-56">The body sizes for each species were measured using the tracked videos (see <em>Video acquisition</em> above). For each 30-minute tracked video, the mean large major axis was calculated in µm for each fly across all frames. Then, the mean of all individual flies within each video was calculated. Therefore, each video generated a single mean body size measurement. Each species’ reported body size (Table S4) was calculated by averaging the mean body size measurements from all video trials. Since the body sizes varied for each species, we calculated the relative leg length by dividing the total leg length by body size.</p></div></div><div class="section" id="sec-27"><h3>QUANTIFICATION AND STATISTICAL ANALYSIS</h3><div class="section" id="sec-28"><h4>SINs and behavioural elements</h4><p id="p-57">Social interaction network (SIN) data was acquired from 19 <em>Drosophila</em> species and one outgroup species, <em>Chymomyza procnemis</em>. We gathered data for 20 species since a power analysis reported this as a reliable minimum sample size for phylogenetic comparative methods [<a id="xref-ref-24-6" class="xref-bibr" href="#ref-24">24</a>, <a id="xref-ref-27-2" class="xref-bibr" href="#ref-27">27</a>]. Approximately 20-25 videos were acquired for each species in both the male and female datasets (see Figure S1 for precise sample sizes of each species).</p><p id="p-58">A two-way ANOVA was used in MATLAB to test whether SINs and the behavioural elements differ by sex and across species. Due to the repeated tests on 8 measures, a Bonferroni correction was employed and all p-values below 0.00625 were considered significant. Statistical details can be found in the results, figures and figure legends.</p><p id="p-59">Outliers were removed from all analyses. For the SIN measures and behavioural measures, a data was considered an outlier if it was greater than the 75<sup>th</sup> quartile+(1.5*IQR) or lower than the 25<sup>th</sup> quartile-(1.5*IQR).</p></div><div class="section" id="sec-29"><h4>Phylogenetic signal</h4><p id="p-60">The K and λ tests were implemented in R using the ‘phylosig’ function (phytools package). We tested phylogenetic signal on the four SIN measures and the four behavioural elements where each species’ mean and standard error of the mean were incorporated to represent the species’ average and intraspecific variation for the trait, respectively [<a id="xref-ref-27-3" class="xref-bibr" href="#ref-27">27</a>]. We also tested phylogenetic signal for the distance, angle and time social spacing parameters. Since some of these variables were manually altered, we did not incorporate any measure of intraspecific variation into the phylogenetic signal tests for these measures. All phylogenetic signal tests were performed separately on the male and female data sets. The estimated K values were statistically evaluated using R through permutation tests where the observed K value was compared to a null distribution of 1000 K values, each computed by randomly shuffling the tips of the phylogenetic tree. All estimated λ values were statistically evaluated using R through likelihood ratio tests against the null hypothesis that λ = 0. All measures of phylogenetic signal and the associated p-values are present in <a id="xref-table-wrap-2-3" class="xref-table" href="#T2">Table 2</a>. Statistical details can be found in the results, figures and figure legends. Each phylogenetic signal test had a sample size of 20, equal to the number of species. All p-values &lt; 0.05 were considered statistically significant.</p><p id="p-61">To visualize the phylogenetic signal tests for each variable, trait data was mapped onto the drosophilid phylogeny using the ‘contMap’ function (phytools package) in R. This function maps ancestral states for the internal nodes of the phylogeny using maximum likelihood, given the trait data for each species at the tips of the tree [<a id="xref-ref-57-1" class="xref-bibr" href="#ref-57">57</a>]. These phylogenetic trait maps are present in <a id="xref-fig-2-7" class="xref-fig" href="#F2">Figures 2</a>-<a id="xref-fig-4-6" class="xref-fig" href="#F4">4</a>.</p></div><div class="section" id="sec-30"><h4>Regression analyses</h4><p id="p-62">To explore how environment and behavioural elements influence drosophilid species’ variation in SINs, we produced three different types of models through multivariate stepwise regressions: i) environmental models, ii) behavioural models and iii) combined models (<a id="xref-fig-5-9" class="xref-fig" href="#F5">Figure 5</a>). The environmental models were derived from the mid-Holocene climatic data described above. Through a principal component analysis implemented in MATLAB, we reduced the 19 climate variables to 5 principal components that account for 92% of the observed variance in the climate data (Figure S7). For each environmental model, the 5 principal components served as the initial predictors, with one SIN measure as a response variable. To generate each environmental model, the “stepwiselm” function, was implemented in MATLAB with default parameters. The optimal predictors that remained after forward and backward stepwise selection are listed below the x-axis of each regression (<a id="xref-fig-5-10" class="xref-fig" href="#F5">Figure 5</a>).</p><p id="p-63">The behavioural models were produced through stepwise regressions, similar to the environmental models. The following variables served as predictors and were fit to each SIN measure: movement, social spacing parameters (distance, angle and time) and the pairwise interaction variables (interaction duration, reciprocation, number of interactions; <a id="xref-fig-1-4" class="xref-fig" href="#F1">Figure 1</a>). The purpose of these models was to explore how behavioural elements predict the complex SIN measures. Like the environmental models, the predictors that remained after forward and backward selection are listed below the x-axis of each regression (<a id="xref-fig-5-11" class="xref-fig" href="#F5">Figure 5</a>).</p><p id="p-64">Finally, the combined models were generated by combining and fitting the predictors of the associated environmental and behavioural models to each SIN measure (<a id="xref-fig-5-12" class="xref-fig" href="#F5">Figure 5</a>). This was done to test if the environmental data, together with the behavioural data, improves the correlation to each SIN measure. To test this, we compared the fit of the combined model to the behavioural model for each SIN measure through a likelihood ratio test. We justify that the comparison can be done against only the behavioural models because their R<sup>2</sup> value was consistently better than the environmental models for each SIN measure. Significant results indicate the combination of behavioural and environmental variables are better at modeling the SIN measures than either the behavioural variables or environmental variables alone. In total, 10 variables had behavioural and combined models generated and compared (see <a id="xref-fig-5-13" class="xref-fig" href="#F5">Figure 5</a>, Figures S2-S4). Therefore, we considered α ≤ 0.005 as significant through a Bonferroni correction. In addition, we also validated the quality of the combined models, relative to the behavioural models, by comparing their Akaike Information Criterion (AICc) values.</p><p id="p-65">We characterized the behavioural elements in a hierarchy, where movement is level one, social spacing is level two, pairwise interactions are level three, and SINs are level four (<a id="xref-fig-1-5" class="xref-fig" href="#F1">Figure 1</a>). Separate environmental models, behavioural models and combined models were generated using level two and three variables as response variables. To generate level three behavioural models, level one and two variables (movement, social spacing) served as predictors. To generate level two behavioural models, the level one variable (movement) served as a predictor. Both male and female data were pooled in these regression analyses since the SIN and behavioural measures contain significant sex by species interaction effects (<a id="xref-fig-2-8" class="xref-fig" href="#F2">Figure 2</a>-<a id="xref-fig-3-6" class="xref-fig" href="#F3">3</a>). All data points in the regression are the mean values for each behavioural measure.</p><p id="p-66">To explore whether total leg length and relative leg length predicts the level two distance parameter (social distance), we generated simple linear regressions in R. For each regression we excluded the outgroup, <em>Ch. procnemis</em>, since we were interested in whether social distance correlated with the total and relative leg lengths of species belonging to the <em>Drosophila</em> genus.</p></div></div><div class="section" id="sec-31"><h3>DATA AND CODE AVAILABILITY</h3><p id="p-67">The behavioural videos supporting the current study have not been deposited in a public repository because of the immense size of the ∼1000 videos but are available from the corresponding author on request.</p></div></div><div class="section" id="sec-32"><h2 class="">Supplemental Information</h2><p id="p-68"><strong>Table S1-Related to Method Details: Summary of species stocks used for all experiments.</strong> All stocks purchased from the <em>Drosophila</em> Species Stock Centre are indicated. Where applicable, the geographic origin of the capture site of each stock is indicated. The approximate geographic coordinates of each stock are also indicated, and these were utilized for acquisition of climate data.</p><p id="p-69"><strong>Table S2-Related to Method Details: Interaction criteria of all 20 species for both male and female flies.</strong> Interaction criteria in bold are values that were used to over-ride the automated estimations (see methods).</p><p id="p-70"><strong>Table S3-Related to Figure 4: Leg lengths of all 20 species for male flies.</strong> Mean front, middle, and rear legs were measured and a total mean leg length was calculated.</p><p id="p-71"><strong>Table S4-Related to Figure 4: Mean body size of all 20 species for male flies.</strong> Body sizes were calculated from tracked videos and averaged for each species.</p><p id="p-72"><strong>Figure S1-Related to Method Details and Figure 3: Ability to form SINs for the male dataset (blue) and female dataset (red).</strong> The x-axis is expressed as a percentage of the number of videos with at least one SIN iteration divided by the total number of videos acquired for each species. There are no differences across species in their ability to form networks (p = 1; χ goodness of fit test). The total number of videos that formed SINs are as follows: <em>D. melanogaster</em>: n= 46 (male), n=48 (female); <em>D.sechellia</em>: n=22 (male), n=22 (female); <em>D. mauritiana</em>: n=26 (male), n=27 (female); <em>D. santomea</em>: n=21 (male), n=23 (female); <em>D. yakuba</em>: n=24 (male), n=26 (female); <em>D.erecta</em>: n=23 (male), n=21 (female); <em>D. bipectinata</em>: n=22 (male), n=24 (female); <em>D. ananassae</em>: n=23 (male), n=25 (female); <em>D. persimilis</em>: n=23 (male), n=25 (female); <em>D. willistoni</em>: n=23 (male), n=24 (female); <em>D.paramelanica</em>: n=22 (male), n=18 (female); <em>D. melanica</em>: n=22 (male), n=21 (female); <em>D. mojavensis</em>: n=25 (male), n=25 (female); <em>D. buzzatii</em>: n=23 (male), n=21 (female); <em>D. hydei</em>: n=25 (male), n=24 (female); <em>D. novamexicana</em>: n=24 (male), n=22 (female); <em>D. virilis</em>: n=25 (male), n=27 (female); <em>D. immigrans</em>: n=23 (male), n=22 (female); <em>Ch. procnemis</em>: n=20 (male), n=21 (female).</p><p id="p-73"><strong>Figure S2-Related to Figure 1: Environmental model for the level 1 variable Movement.</strong> For the regression, each data point represents the mean SIN measure for a single species. The mean SIN measure for groups of male flies and female flies were pooled into each regression and are labeled with red and blue points, respectively. The solid trend line indicates line of best fit and dashed lines indicate 95% confidence interval of the model. The equation for this model is listed below the x-axis.</p><p id="p-74"><strong>Figure S3-Related to Figure 1: Environmental, behavioural and combined models for the level 2 social spacing variables: Distance, Angle and Time.</strong> For all regressions, each data point represents the mean SIN measure for a single species. The mean SIN measure for groups of male flies and female flies were pooled into each regression and are labeled with red and blue points, respectively. Each solid trend line indicates line of best fit and dashed lines indicate 95% confidence interval of the model. <strong>A)</strong> No significant behavioural model formed for distance, therefore no combined model to report. The environmental model (Equation 1 predicts social distance (R<sup>2</sup> = 0.255, p = 0.00164). <strong>B)</strong> No significant behavioural model formed for angle, therefore no combined model to report. The environmental model (Equation 2) predicts angle (R<sup>2</sup> = 0.145, p = 0.0087). <strong>C)</strong> The environmental model (Equation 3) predicts time (R<sup>2</sup> = 0.16, p = 0.0062). The behavioural model (Equation 4) predicts time (R<sup>2</sup> = 0.0837, p = 0.039). The combined model significantly improves prediction of time compared to the behavioural model alone (Combined AICc &lt; Behavioural AICc; p = 0.0003, likelihood ratio test). <strong>Equation 1:</strong> y = 1.80 – 0.071*[PC1] – 0.13*[PC5]. <strong>Equation 2:</strong> y = 106.3 + 4.71*[PC1]. <strong>Equation 3:</strong> y = 0.99 + 0.077*[PC1]. <strong>Equation 4:</strong> y = 1.21 – 0.061*[movement]. <strong>Equation 5:</strong> y = 1.28 + 0.091*[PC1] – 0.078*[movement].</p><p id="p-75"><strong>Figure S4-Related to Figure 1: Environmental, behavioural and combined models for the level 3 variables interaction duration (Interaction</strong> Δ<strong>t), number of interactions (Interaction #), and reciprocation.</strong> For all regressions, each data point represents the mean SIN measure for a single species. The mean SIN measure for groups of male flies and female flies were pooled into each regression and are labeled with red and blue points, respectively. Each solid trend line indicates line of best fit and dashed lines indicate 95% confidence interval of the model. <strong>A)</strong> The environmental model (Equation 1) predicts interaction duration (R<sup>2</sup> = 0.382, p = 0.0003). The behavioural model (Equation 2) predicts interaction duration (R<sup>2</sup> = 0.553, p &lt; 0.0001). The combined model (Equation 3) does not significantly improve the prediction of interaction duration compared to the behavioural model alone (Combined AICc =∼ Behavioural AICc; p = 0.0182, likelihood ratio test). <strong>B)</strong> The environmental model (Equation 4) predicts number of interactions (R<sup>2</sup> = 0.20772, p = 0.005). The behavioural model (Equation 5) predicts number of interactions (R<sup>2</sup> = 0.538, p &lt; 0.0001). The combined model does not significantly improve the prediction of number of interactions compared to the behavioural model alone (Combined AICc =∼ Behavioural AICc; p = 0.1594, likelihood ratio test). <strong>C)</strong> The environmental model (Equation 7) predicts reciprocation (R<sup>2</sup> = 0.122, p = 0.034). The behavioural model (Equation 8) predicts reciprocation (R<sup>2</sup> = 0.879, p &lt; 0.0001). The combined model (Equation 9) does not significantly improve the prediction of reciprocation compared to the behavioural model alone (Combined AICc =∼ Behavioural AICc; p = 0.153, likelihood ratio test). <strong>Equation 1:</strong> y = 8.17 – 0.76*[PC2] + 1.02*[PC3] + 2.31*[PC5]. <strong>Equation 2:</strong> y = 6.54 – 1.25*[movement] + 0.045*[angle]. <strong>Equation 3:</strong> y = 6.99 – 0.23*[PC2] + 0.47*[PC3] + 1.47*[PC5] – 0.93*[movement] + 0.038*[angle]. <strong>Equation 4:</strong> y = 1398 + 220*[PC4] – 336*[PC5]. <strong>Equation 5:</strong> y = 1001 + 213*[movement] – 406*[time]. <strong>Equation 6:</strong> y = 1100 + 91.5*[PC4] – 151*[PC5] + 183*[movement] – 387*[time]. <strong>Equation 7:</strong> y = 0.39 + 0.011*[PC3]. <strong>Equation 8:</strong> y = 0.21 – 0.0065*[movement] + 0.014*[social distance] + 0.0017*[angle] – 0.025*[time]. <strong>Equation 9:</strong> y = 0.22 + 0.0028*[PC3] – 0.0061*[movement] + 0.01*[social distance] + 0.0017*[angle] – 0.025*[time].</p><p id="p-76"><strong>Figure S5-Related to Figure 3: Relative species differences between <em>D. melanogaster</em> (CS), <em>D. sechellia</em> (SEC), <em>D. yakuba</em> (YAK), <em>D. mojavensis</em> (MOV), <em>D. paramelanica</em> (PARA) are replicated in male flies between pilot data (Experiment 1; left panels) and extended data (Experiment 2; right panels).</strong> All figures are boxplots which outline the distribution of the z-scores for all four SIN measures. Letters above each box indicate statistically distinct groups from a Kruskal-Wallis one-way ANOVA followed Tukey-Kramer post-hoc tests.</p><p id="p-77"><strong>Figure S6-Related to Figure 3: Relative species differences between <em>D. melanogaster</em> (CS), <em>D. sechellia</em> (SEC), <em>D. yakuba</em> (YAK), <em>D. mojavensis</em> (MOV), <em>D. paramelanica</em> (PARA) are replicated in female flies between pilot data (Experiment 1; left panels) and extended data (Experiment 2; right panels).</strong> All figures are boxplots which outline the distribution of the z-scores for all four SIN measures. Letters above each box indicate statistically distinct groups from a Kruskal-Wallis one-way ANOVA followed Tukey-Kramer post-hoc tests.</p><p id="p-78"><strong>Figure S7-Related to Figure 5: The first 5 principal components of the climatic measures, extracted from each species geographic origin, account for 92% of the variance across the 19 variables.</strong> Each bar represents the percentage of variance explained by the first 10 dimensions. Because the rate of decrease reduces after dimension 5, the first 5 dimensions were used to represent the environmental variables for all regression analyses.</p></div><div class="section ack" id="ack-1"><h2 class="">Acknowledgements</h2><p id="p-31">Funding for this study was graciously provided to JDL by NSERC, CIHR, the Canada Research Chair, and Canadian Institute for Advanced Research. The authors wish to thank Leslie Vosshall, Michael Dickinson, John Ratcliffe, Rob Ness, Marc Johnson, and Luke Mahler for helpful discussions along the way.</p></div><div class="section ref-list" id="ref-list-1"><h2 class="">References</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label">1.</span><a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference 1. in text" id="ref-1">↵</a><div class="cit ref-cit ref-journal" id="cit-776708v1.1" data-doi="10.2307/1933133"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Scott</span>,  <span class="cit-name-given-names">J.P</span></span>. (<span class="cit-pub-date">1956</span>). <span class="cit-article-title">The analysis of social organization in animals</span>. <abbr class="cit-jnl-abbrev">Ecology</abbr> <span class="cit-vol">37</span>, <span class="cit-fpage">213</span>–<span class="cit-lpage">221</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEcology%26rft.volume%253D37%26rft.spage%253D213%26rft_id%253Dinfo%253Adoi%252F10.2307%252F1933133%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2307/1933133&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=A1956WR62100002&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">2.</span><a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference 2. in text" id="ref-2">↵</a><div class="cit ref-cit ref-book" id="cit-776708v1.2"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Krause</span>,  <span class="cit-name-given-names">J.</span></span>, and <span class="cit-auth"><span class="cit-name-surname">Ruxton</span>,  <span class="cit-name-given-names">G.D.</span></span> (<span class="cit-pub-date">2002</span>). <span class="cit-source">Living in groups</span>, (<span class="cit-publ-name">Oxford University Press</span>).</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">3.</span><a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference 3. in text" id="ref-3">↵</a><div class="cit ref-cit ref-journal" id="cit-776708v1.3"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Drewe</span>,  <span class="cit-name-given-names">J.A</span></span>. (<span class="cit-pub-date">2009</span>). <span class="cit-article-title">Who infects whom? Social networks and tuberculosis transmission in wild meerkats</span>. <abbr class="cit-jnl-abbrev">Proceedings of the Royal Society B: Biological Sciences</abbr> <span class="cit-vol">277</span>, <span class="cit-fpage">633</span>–<span class="cit-lpage">642</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProceedings%2Bof%2Bthe%2BRoyal%2BSociety%2BB%253A%2BBiological%2BSciences%26rft.volume%253D277%26rft.spage%253D633%26rft_id%253Dinfo%253Apmid%252F19889705%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=19889705&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F09%2F20%2F776708.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000273459600016&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">4.</span><a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference 4. in text" id="ref-4">↵</a><div class="cit ref-cit ref-journal" id="cit-776708v1.4" data-doi="10.1086/655216"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Oh</span>,  <span class="cit-name-given-names">K.P.</span></span>, and <span class="cit-auth"><span class="cit-name-surname">Badyaev</span>,  <span class="cit-name-given-names">A.V</span></span>. (<span class="cit-pub-date">2010</span>). <span class="cit-article-title">Structure of social networks in a passerine bird: consequences for sexual selection and the evolution of mating strategies</span>. <abbr class="cit-jnl-abbrev">The American Naturalist</abbr> <span class="cit-vol">176</span>, <span class="cit-fpage">E80</span>–<span class="cit-lpage">E89</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DAm%2BNat%26rft.aulast%253DOh%26rft.auinit1%253DK.%2BP.%26rft.volume%253D176%26rft.issue%253D3%26rft.spage%253DE80%26rft.epage%253DE89%26rft.atitle%253DStructure%2Bof%2Bsocial%2Bnetworks%2Bin%2Ba%2Bpasserine%2Bbird%253A%2Bconsequences%2Bfor%2Bsexual%2Bselection%2Band%2Bthe%2Bevolution%2Bof%2Bmating%2Bstrategies.%26rft_id%253Dinfo%253Adoi%252F10.1086%252F655216%26rft_id%253Dinfo%253Apmid%252F20608873%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1086/655216&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20608873&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F09%2F20%2F776708.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000280621000013&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">5.</span><a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference 5. in text" id="ref-5">↵</a><div class="cit ref-cit ref-journal" id="cit-776708v1.5" data-doi="10.1098/rspb.2015.2954"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Pasquaretta</span>,  <span class="cit-name-given-names">C.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Battesti</span>,  <span class="cit-name-given-names">M.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Klenschi</span>,  <span class="cit-name-given-names">E.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Bousquet</span>,  <span class="cit-name-given-names">C.A.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Sueur</span>,  <span class="cit-name-given-names">C.</span></span>, and <span class="cit-auth"><span class="cit-name-surname">Mery</span>,  <span class="cit-name-given-names">F</span></span>. (<span class="cit-pub-date">2016</span>). <span class="cit-article-title">How social network structure affects decision-making in Drosophila melanogaster</span>. <abbr class="cit-jnl-abbrev">Proc Biol Sci</abbr> <span class="cit-vol">283</span>, <span class="cit-fpage">20152954</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc%2BBiol%2BSci%26rft_id%253Dinfo%253Adoi%252F10.1098%252Frspb.2015.2954%26rft_id%253Dinfo%253Apmid%252F26936247%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1098/rspb.2015.2954&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=26936247&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F09%2F20%2F776708.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">6.</span><a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference 6. in text" id="ref-6">↵</a><div class="cit ref-cit ref-journal" id="cit-776708v1.6" data-doi="10.1016/j.cub.2011.12.050"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Battesti</span>,  <span class="cit-name-given-names">M.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Moreno</span>,  <span class="cit-name-given-names">C.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Joly</span>,  <span class="cit-name-given-names">D.</span></span>, and <span class="cit-auth"><span class="cit-name-surname">Mery</span>,  <span class="cit-name-given-names">F</span></span>. (<span class="cit-pub-date">2012</span>). <span class="cit-article-title">Spread of social information and dynamics of social transmission within Drosophila groups</span>. <abbr class="cit-jnl-abbrev">Curr Biol</abbr> <span class="cit-vol">22</span>, <span class="cit-fpage">309</span>–<span class="cit-lpage">313</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCurrent%2Bbiology%2B%253A%2B%2BCB%26rft.stitle%253DCurr%2BBiol%26rft.aulast%253DBattesti%26rft.auinit1%253DM.%26rft.volume%253D22%26rft.issue%253D4%26rft.spage%253D309%26rft.epage%253D313%26rft.atitle%253DSpread%2Bof%2Bsocial%2Binformation%2Band%2Bdynamics%2Bof%2Bsocial%2Btransmission%2Bwithin%2BDrosophila%2Bgroups.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.cub.2011.12.050%26rft_id%253Dinfo%253Apmid%252F22264604%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.cub.2011.12.050&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22264604&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F09%2F20%2F776708.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">7.</span><a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference 7. in text" id="ref-7">↵</a><div class="cit ref-cit ref-journal" id="cit-776708v1.7" data-doi="10.1098/rspb.2009.1294"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Sarin</span>,  <span class="cit-name-given-names">S.</span></span>, and <span class="cit-auth"><span class="cit-name-surname">Dukas</span>,  <span class="cit-name-given-names">R</span></span>. (<span class="cit-pub-date">2009</span>). <span class="cit-article-title">Social learning about egg-laying substrates in fruitflies</span>. <abbr class="cit-jnl-abbrev">Proc Biol Sci</abbr> <span class="cit-vol">276</span>, <span class="cit-fpage">4323</span>–<span class="cit-lpage">4328</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc%2BBiol%2BSci%26rft_id%253Dinfo%253Adoi%252F10.1098%252Frspb.2009.1294%26rft_id%253Dinfo%253Apmid%252F19759037%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1098/rspb.2009.1294&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19759037&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F09%2F20%2F776708.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">8.</span><a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference 8. in text" id="ref-8">↵</a><div class="cit ref-cit ref-journal" id="cit-776708v1.8" data-doi="10.1126/science.aat1590"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Danchin</span>,  <span class="cit-name-given-names">E.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Nöbel</span>,  <span class="cit-name-given-names">S.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Pocheville</span>,  <span class="cit-name-given-names">A.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Dagaeff</span>,  <span class="cit-name-given-names">A.C.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Demay</span>,  <span class="cit-name-given-names">L.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Alphand</span>,  <span class="cit-name-given-names">M.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Ranty-Roby</span>,  <span class="cit-name-given-names">S.</span></span>, <span class="cit-auth"><span class="cit-name-surname">van Renssen</span>,  <span class="cit-name-given-names">L.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Monier</span>,  <span class="cit-name-given-names">M.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Gazagne</span>,  <span class="cit-name-given-names">E.</span></span>, <span class="cit-etal">et al.</span> (<span class="cit-pub-date">2018</span>). <span class="cit-article-title">Cultural flies: Conformist social learning in fruitflies predicts long-lasting mate-choice traditions</span>. <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">362</span>, <span class="cit-fpage">1025</span>–<span class="cit-lpage">1030</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DDanchin%26rft.auinit1%253DE.%26rft.volume%253D362%26rft.issue%253D6418%26rft.spage%253D1025%26rft.epage%253D1030%26rft.atitle%253DCultural%2Bflies%253A%2BConformist%2Bsocial%2Blearning%2Bin%2Bfruitflies%2Bpredicts%2Blong-lasting%2Bmate-choice%2Btraditions%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.aat1590%26rft_id%253Dinfo%253Apmid%252F30498121%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEzOiIzNjIvNjQxOC8xMDI1IjtzOjQ6ImF0b20iO3M6Mzc6Ii9iaW9yeGl2L2Vhcmx5LzIwMTkvMDkvMjAvNzc2NzA4LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">9.</span><a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference 9. in text" id="ref-9">↵</a><div class="cit ref-cit ref-journal" id="cit-776708v1.9" data-doi="10.1242/jeb.142257"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Lihoreau</span>,  <span class="cit-name-given-names">M.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Poissonnier</span>,  <span class="cit-name-given-names">L.A.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Isabel</span>,  <span class="cit-name-given-names">G.</span></span>, and <span class="cit-auth"><span class="cit-name-surname">Dussutour</span>,  <span class="cit-name-given-names">A</span></span>. (<span class="cit-pub-date">2016</span>). <span class="cit-article-title">Drosophila females trade off good nutrition with high-quality oviposition sites when choosing foods</span>. <abbr class="cit-jnl-abbrev">J Exp Biol</abbr> <span class="cit-vol">219</span>, <span class="cit-fpage">2514</span>–<span class="cit-lpage">2524</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ%2BExp%2BBiol%26rft_id%253Dinfo%253Adoi%252F10.1242%252Fjeb.142257%26rft_id%253Dinfo%253Apmid%252F27284071%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiamV4YmlvIjtzOjU6InJlc2lkIjtzOjExOiIyMTkvMTYvMjUxNCI7czo0OiJhdG9tIjtzOjM3OiIvYmlvcnhpdi9lYXJseS8yMDE5LzA5LzIwLzc3NjcwOC5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">10.</span><a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference 10. in text" id="ref-10">↵</a><div class="cit ref-cit ref-journal" id="cit-776708v1.10"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Kacsoh</span>,  <span class="cit-name-given-names">B.Z.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Bozler</span>,  <span class="cit-name-given-names">J.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Ramaswami</span>,  <span class="cit-name-given-names">M.</span></span>, and <span class="cit-auth"><span class="cit-name-surname">Bosco</span>,  <span class="cit-name-given-names">G</span></span>. (<span class="cit-pub-date">2015</span>). <span class="cit-article-title">Social communication of predator-induced changes in Drosophila behavior and germ line physiology</span>. <abbr class="cit-jnl-abbrev">Elife</abbr> <span class="cit-vol">4</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">11.</span><a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference 11. in text" id="ref-11">↵</a><div class="cit ref-cit ref-journal" id="cit-776708v1.11"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Kacsoh</span>,  <span class="cit-name-given-names">B.Z.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Bozler</span>,  <span class="cit-name-given-names">J.</span></span>, and <span class="cit-auth"><span class="cit-name-surname">Bosco</span>,  <span class="cit-name-given-names">G</span></span>. (<span class="cit-pub-date">2018</span>). <span class="cit-article-title">Correction: Drosophila species learn dialects through communal living</span>. <abbr class="cit-jnl-abbrev">PLoS Genet</abbr> <span class="cit-vol">14</span>, <span class="cit-fpage">e1007825</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BGenet%26rft.volume%253D14%26rft.spage%253De1007825%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">12.</span><a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference 12. in text" id="ref-12">↵</a><div class="cit ref-cit ref-journal" id="cit-776708v1.12" data-doi="10.1038/nature14024"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Ramdya</span>,  <span class="cit-name-given-names">P.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Lichocki</span>,  <span class="cit-name-given-names">P.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Cruchet</span>,  <span class="cit-name-given-names">S.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Frisch</span>,  <span class="cit-name-given-names">L.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Tse</span>,  <span class="cit-name-given-names">W.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Floreano</span>,  <span class="cit-name-given-names">D.</span></span>, and <span class="cit-auth"><span class="cit-name-surname">Benton</span>,  <span class="cit-name-given-names">R</span></span>. (<span class="cit-pub-date">2015</span>). <span class="cit-article-title">Mechanosensory interactions drive collective behaviour in Drosophila</span>. <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">519</span>, <span class="cit-fpage">233</span>–<span class="cit-lpage">236</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.volume%253D519%26rft.spage%253D233%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature14024%26rft_id%253Dinfo%253Apmid%252F25533959%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature14024&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25533959&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F09%2F20%2F776708.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">13.</span><a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference 13. in text" id="ref-13">↵</a><div class="cit ref-cit ref-journal" id="cit-776708v1.13" data-doi="10.1073/pnas.1121252109"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Schneider</span>,  <span class="cit-name-given-names">J.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Dickinson</span>,  <span class="cit-name-given-names">M.H.</span></span>, and <span class="cit-auth"><span class="cit-name-surname">Levine</span>,  <span class="cit-name-given-names">J.D</span></span>. (<span class="cit-pub-date">2012</span>). <span class="cit-article-title">Social structures depend on innate determinants and chemosensory processing in Drosophila</span>. <abbr class="cit-jnl-abbrev">Proc Natl Acad Sci U S A</abbr> <span class="cit-vol">109</span> <span class="cit-issue">Suppl 2</span>, <span class="cit-fpage">17174</span>–<span class="cit-lpage">17179</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc%2BNatl%2BAcad%2BSci%2BU%2BS%2BA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.1121252109%26rft_id%253Dinfo%253Apmid%252F22802679%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoyMjoiMTA5L1N1cHBsZW1lbnRfMi8xNzE3NCI7czo0OiJhdG9tIjtzOjM3OiIvYmlvcnhpdi9lYXJseS8yMDE5LzA5LzIwLzc3NjcwOC5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">14.</span><a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference 14. in text" id="ref-14">↵</a><div class="cit ref-cit ref-book" id="cit-776708v1.14"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Newman</span>,  <span class="cit-name-given-names">M</span></span>. (<span class="cit-pub-date">2010</span>). <span class="cit-source">Networks: an introduction</span>, (<span class="cit-publ-name">Oxford University Press</span>).</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">15.</span><a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference 15. in text" id="ref-15">↵</a><div class="cit ref-cit ref-journal" id="cit-776708v1.15" data-doi="10.1073/pnas.0806746106"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Fowler</span>,  <span class="cit-name-given-names">J.H.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Dawes</span>,  <span class="cit-name-given-names">C.T.</span></span>, and <span class="cit-auth"><span class="cit-name-surname">Christakis</span>,  <span class="cit-name-given-names">N.A</span></span>. (<span class="cit-pub-date">2009</span>). <span class="cit-article-title">Model of genetic variation in human social networks</span>. <abbr class="cit-jnl-abbrev">Proc Natl Acad Sci U S A</abbr> <span class="cit-vol">106</span>, <span class="cit-fpage">1720</span>–<span class="cit-lpage">1724</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc%2BNatl%2BAcad%2BSci%2BU%2BS%2BA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.0806746106%26rft_id%253Dinfo%253Apmid%252F19171900%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMDoiMTA2LzYvMTcyMCI7czo0OiJhdG9tIjtzOjM3OiIvYmlvcnhpdi9lYXJseS8yMDE5LzA5LzIwLzc3NjcwOC5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">16.</span><a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference 16. in text" id="ref-16">↵</a><div class="cit ref-cit ref-journal" id="cit-776708v1.16" data-doi="10.1242/jeb.143008"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Jezovit</span>,  <span class="cit-name-given-names">J.A.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Levine</span>,  <span class="cit-name-given-names">J.D.</span></span>, and <span class="cit-auth"><span class="cit-name-surname">Schneider</span>,  <span class="cit-name-given-names">J</span></span>. (<span class="cit-pub-date">2017</span>). <span class="cit-article-title">Phylogeny, environment and sexual communication across the Drosophila genus</span>. <abbr class="cit-jnl-abbrev">J Exp Biol</abbr> <span class="cit-vol">220</span>, <span class="cit-fpage">42</span>–<span class="cit-lpage">52</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BExperimental%2BBiology%26rft.stitle%253DJ.%2BExp.%2BBiol.%26rft.aulast%253DJezovit%26rft.auinit1%253DJ.%2BA.%26rft.volume%253D220%26rft.issue%253D1%26rft.spage%253D42%26rft.epage%253D52%26rft.atitle%253DPhylogeny%252C%2Benvironment%2Band%2Bsexual%2Bcommunication%2Bacross%2Bthe%2BDrosophila%2Bgenus%26rft_id%253Dinfo%253Adoi%252F10.1242%252Fjeb.143008%26rft_id%253Dinfo%253Apmid%252F28057827%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiamV4YmlvIjtzOjU6InJlc2lkIjtzOjg6IjIyMC8xLzQyIjtzOjQ6ImF0b20iO3M6Mzc6Ii9iaW9yeGl2L2Vhcmx5LzIwMTkvMDkvMjAvNzc2NzA4LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">17.</span><a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference 17. in text" id="ref-17">↵</a><div class="cit ref-cit ref-book" id="cit-776708v1.17"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Markow</span>,  <span class="cit-name-given-names">T.A.</span></span>, and <span class="cit-auth"><span class="cit-name-surname">O’Grady</span>,  <span class="cit-name-given-names">P.M</span></span>. (<span class="cit-pub-date">2006</span>). <span class="cit-source">Drosophila: A guide to species identification and use</span>, (<span class="cit-publ-loc">San Diego, California, USA</span>: <span class="cit-publ-name">Elsevier, Inc</span>).</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">18.</span><a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference 18. in text" id="ref-18">↵</a><div class="cit ref-cit ref-journal" id="cit-776708v1.18" data-doi="10.1016/j.anbehav.2011.12.021"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Foster</span>,  <span class="cit-name-given-names">E.A.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Franks</span>,  <span class="cit-name-given-names">D.W.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Morrell</span>,  <span class="cit-name-given-names">L.J.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Balcomb</span>,  <span class="cit-name-given-names">K.C.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Parsons</span>,  <span class="cit-name-given-names">K.M.</span></span>, <span class="cit-auth"><span class="cit-name-surname">van Ginneken</span>,  <span class="cit-name-given-names">A.</span></span>, and <span class="cit-auth"><span class="cit-name-surname">Croft</span>,  <span class="cit-name-given-names">D.P.</span></span> (<span class="cit-pub-date">2012</span>). <span class="cit-article-title">Social network correlates of food availability in an endangered population of killer whales, Orcinus orca</span>. <abbr class="cit-jnl-abbrev">Animal Behaviour</abbr> <span class="cit-vol">83</span>, <span class="cit-fpage">731</span>–<span class="cit-lpage">736</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAnimal%2BBehaviour%26rft.volume%253D83%26rft.spage%253D731%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.anbehav.2011.12.021%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.anbehav.2011.12.021&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000300618100018&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">19.</span><a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference 19. in text" id="ref-19">↵</a><div class="cit ref-cit ref-journal" id="cit-776708v1.19" data-doi="10.1371/journal.pone.0024280"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Kelley</span>,  <span class="cit-name-given-names">J.L.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Morrell</span>,  <span class="cit-name-given-names">L.J.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Inskip</span>,  <span class="cit-name-given-names">C.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Krause</span>,  <span class="cit-name-given-names">J.</span></span>, and <span class="cit-auth"><span class="cit-name-surname">Croft</span>,  <span class="cit-name-given-names">D.P</span></span>. (<span class="cit-pub-date">2011</span>). <span class="cit-article-title">Predation risk shapes social networks in fission-fusion populations</span>. <abbr class="cit-jnl-abbrev">PloS one</abbr> <span class="cit-vol">6</span>, <span class="cit-fpage">e24280</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BONE%26rft.aulast%253DKelley%26rft.auinit1%253DJ.%2BL.%26rft.volume%253D6%26rft.issue%253D8%26rft.spage%253De24280%26rft.epage%253De24280%26rft.atitle%253DPredation%2Brisk%2Bshapes%2Bsocial%2Bnetworks%2Bin%2Bfission-fusion%2Bpopulations.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0024280%26rft_id%253Dinfo%253Apmid%252F21912627%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0024280&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21912627&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F09%2F20%2F776708.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">20.</span><a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference 20. in text" id="ref-20">↵</a><div class="cit ref-cit ref-journal" id="cit-776708v1.20"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Grimaldi</span>,  <span class="cit-name-given-names">D.A.</span></span> (<span class="cit-pub-date">1987</span>). <span class="cit-article-title">Amber fossil Drosophilidae (Diptera), with particular reference to the Hispaniolan taxa</span>. <abbr class="cit-jnl-abbrev">American museum novitates (USA)</abbr>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">21.</span><a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference 21. in text" id="ref-21">↵</a><div class="cit ref-cit ref-journal" id="cit-776708v1.21"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">van der Linde</span>,  <span class="cit-name-given-names">K.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Houle</span>,  <span class="cit-name-given-names">D.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Spicer</span>,  <span class="cit-name-given-names">G.S.</span></span>, and <span class="cit-auth"><span class="cit-name-surname">Steppan</span>,  <span class="cit-name-given-names">S.J.</span></span> (<span class="cit-pub-date">2010</span>). <span class="cit-article-title">A supermatrix-based molecular phylogeny of the family Drosophilidae</span>. <abbr class="cit-jnl-abbrev">Genetics research</abbr> <span class="cit-vol">92</span>, <span class="cit-fpage">25</span>–<span class="cit-lpage">38</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%2Bresearch%26rft.volume%253D92%26rft.spage%253D25%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">22.</span><a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference 22. in text" id="ref-22">↵</a><div class="cit ref-cit ref-journal" id="cit-776708v1.22" data-doi="10.1111/j.1558-5646.2012.01685.x"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Kellermann</span>,  <span class="cit-name-given-names">V.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Loeschcke</span>,  <span class="cit-name-given-names">V.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Hoffmann</span>,  <span class="cit-name-given-names">A.A.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Kristensen</span>,  <span class="cit-name-given-names">T.N.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Fløjgaard</span>,  <span class="cit-name-given-names">C.</span></span>, <span class="cit-auth"><span class="cit-name-surname">David</span>,  <span class="cit-name-given-names">J.R.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Svenning</span>,  <span class="cit-name-given-names">J.C.</span></span>, and <span class="cit-auth"><span class="cit-name-surname">Overgaard</span>,  <span class="cit-name-given-names">J</span></span>. (<span class="cit-pub-date">2012</span>). <span class="cit-article-title">Phylogenetic constraints in key functional traits behind species’ climate niches: patterns of desiccation and cold resistance across 95 Drosophila species</span>. <abbr class="cit-jnl-abbrev">Evolution</abbr> <span class="cit-vol">66</span>, <span class="cit-fpage">3377</span>–<span class="cit-lpage">3389</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEvolution%26rft.stitle%253DEvolution%26rft.aulast%253DKellermann%26rft.auinit1%253DV.%26rft.volume%253D66%26rft.issue%253D11%26rft.spage%253D3377%26rft.epage%253D3389%26rft.atitle%253DPHYLOGENETIC%2BCONSTRAINTS%2BIN%2BKEY%2BFUNCTIONAL%2BTRAITS%2BBEHIND%2BSPECIES%2527%2BCLIMATE%2BNICHES%253A%2BPATTERNS%2BOF%2BDESICCATION%2BAND%2BCOLD%2BRESISTANCE%2BACROSS%2B95%2BDROSOPHILA%2BSPECIES.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1558-5646.2012.01685.x%26rft_id%253Dinfo%253Apmid%252F23106704%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1558-5646.2012.01685.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23106704&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F09%2F20%2F776708.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000310477600006&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">23.</span><a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference 23. in text" id="ref-23">↵</a><div class="cit ref-cit ref-journal" id="cit-776708v1.23" data-doi="10.1073/pnas.1207553109"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Kellermann</span>,  <span class="cit-name-given-names">V.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Overgaard</span>,  <span class="cit-name-given-names">J.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Hoffmann</span>,  <span class="cit-name-given-names">A.A.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Fløjgaard</span>,  <span class="cit-name-given-names">C.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Svenning</span>,  <span class="cit-name-given-names">J.C.</span></span>, and <span class="cit-auth"><span class="cit-name-surname">Loeschcke</span>,  <span class="cit-name-given-names">V</span></span>. (<span class="cit-pub-date">2012</span>). <span class="cit-article-title">Upper thermal limits of Drosophila are linked to species distributions and strongly constrained phylogenetically</span>. <abbr class="cit-jnl-abbrev">Proc Natl Acad Sci U S A</abbr> <span class="cit-vol">109</span>, <span class="cit-fpage">16228</span>–<span class="cit-lpage">16233</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc%2BNatl%2BAcad%2BSci%2BU%2BS%2BA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.1207553109%26rft_id%253Dinfo%253Apmid%252F22988106%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMjoiMTA5LzQwLzE2MjI4IjtzOjQ6ImF0b20iO3M6Mzc6Ii9iaW9yeGl2L2Vhcmx5LzIwMTkvMDkvMjAvNzc2NzA4LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">24.</span><a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference 24. in text" id="ref-24">↵</a><div class="cit ref-cit ref-journal" id="cit-776708v1.24" data-doi="10.1554/0014-3820(2003)057[0717:TFPSIC]2.0.CO;2"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Blomberg</span>,  <span class="cit-name-given-names">S.P.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Garland</span>,  <span class="cit-name-given-names">T.</span></span>, and <span class="cit-auth"><span class="cit-name-surname">Ives</span>,  <span class="cit-name-given-names">A.R</span></span>. (<span class="cit-pub-date">2003</span>). <span class="cit-article-title">Testing for phylogenetic signal in comparative data: behavioral traits are more labile</span>. <abbr class="cit-jnl-abbrev">Evolution</abbr> <span class="cit-vol">57</span>, <span class="cit-fpage">717</span>–<span class="cit-lpage">745</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEvolution%26rft.stitle%253DEvolution%26rft.aulast%253DBlomberg%26rft.auinit1%253DS.%2BP.%26rft.volume%253D57%26rft.issue%253D4%26rft.spage%253D717%26rft.epage%253D745%26rft.atitle%253DTesting%2Bfor%2Bphylogenetic%2Bsignal%2Bin%2Bcomparative%2Bdata%253A%2Bbehavioral%2Btraits%2Bare%2Bmore%2Blabile.%26rft_id%253Dinfo%253Adoi%252F10.1554%252F0014-3820%25282003%2529057%255B0717%253ATFPSIC%255D2.0.CO%253B2%26rft_id%253Dinfo%253Apmid%252F12778543%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1554/0014-3820(2003)057[0717:TFPSIC]2.0.CO;2&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=12778543&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F09%2F20%2F776708.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000182815100003&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">25.</span><a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference 25. in text" id="ref-25">↵</a><div class="cit ref-cit ref-journal" id="cit-776708v1.25" data-doi="10.1038/44766"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Pagel</span>,  <span class="cit-name-given-names">M</span></span>. (<span class="cit-pub-date">1999</span>). <span class="cit-article-title">Inferring the historical patterns of biological evolution</span>. <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">401</span>, <span class="cit-fpage">877</span>–<span class="cit-lpage">884</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.volume%253D401%26rft.issue%253D6756%26rft.spage%253D877%26rft.epage%253D884%26rft.atitle%253DInferring%2Bthe%2Bhistorical%2Bpatterns%2Bof%2Bbiological%2Bevolution%26rft_id%253Dinfo%253Adoi%252F10.1038%252F44766%26rft_id%253Dinfo%253Apmid%252F10553904%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/44766&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=2000007066&amp;link_type=GEOREF" class="cit-ref-sprinkles cit-ref-sprinkles-georef"><span>GeoRef</span></a><a href="/lookup/external-ref?access_num=10553904&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F09%2F20%2F776708.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000083464700048&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">26.</span><a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference 26. in text" id="ref-26">↵</a><div class="cit ref-cit ref-journal" id="cit-776708v1.26"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Hijmans</span>,  <span class="cit-name-given-names">R.J.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Cameron</span>,  <span class="cit-name-given-names">S.E.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Parra</span>,  <span class="cit-name-given-names">J.L.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Jones</span>,  <span class="cit-name-given-names">P.G.</span></span>, and <span class="cit-auth"><span class="cit-name-surname">Jarvis</span>,  <span class="cit-name-given-names">A</span></span>. (<span class="cit-pub-date">2005</span>). <span class="cit-article-title">Very high resolution interpolated climate surfaces for global land areas</span>. <abbr class="cit-jnl-abbrev">International Journal of Climatology: A Journal of the Royal Meteorological Society</abbr> <span class="cit-vol">25</span>, <span class="cit-fpage">1965</span>–<span class="cit-lpage">1978</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DInternational%2BJournal%2Bof%2BClimatology%253A%2BA%2BJournal%2Bof%2Bthe%2BRoyal%2BMeteorological%2BSociety%26rft.volume%253D25%26rft.spage%253D1965%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">27.</span><a class="rev-xref-ref" href="#xref-ref-27-1" title="View reference 27. in text" id="ref-27">↵</a><div class="cit ref-cit ref-journal" id="cit-776708v1.27" data-doi="10.1080/10635150701313830"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Ives</span>,  <span class="cit-name-given-names">A.R.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Midford</span>,  <span class="cit-name-given-names">P.E.</span></span>, and <span class="cit-auth"><span class="cit-name-surname">Garland</span>   <span class="cit-name-suffix">Jr</span>,  <span class="cit-name-given-names">T</span></span>. (<span class="cit-pub-date">2007</span>). <span class="cit-article-title">Within-species variation and measurement error in phylogenetic comparative methods</span>. <abbr class="cit-jnl-abbrev">Systematic Biology</abbr> <span class="cit-vol">56</span>, <span class="cit-fpage">252</span>–<span class="cit-lpage">270</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DSystematic%2BBiology%26rft.stitle%253DSyst%2BBiol%26rft.aulast%253DIves%26rft.auinit1%253DA.%2BR.%26rft.volume%253D56%26rft.issue%253D2%26rft.spage%253D252%26rft.epage%253D270%26rft.atitle%253DWithin-species%2Bvariation%2Band%2Bmeasurement%2Berror%2Bin%2Bphylogenetic%2Bcomparative%2Bmethods.%26rft_id%253Dinfo%253Adoi%252F10.1080%252F10635150701313830%26rft_id%253Dinfo%253Apmid%252F17464881%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1080/10635150701313830&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17464881&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F09%2F20%2F776708.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000246448300008&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">28.</span><a class="rev-xref-ref" href="#xref-ref-28-1" title="View reference 28. in text" id="ref-28">↵</a><div class="cit ref-cit ref-journal" id="cit-776708v1.28" data-doi="10.1371/journal.pone.0095495"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Durisko</span>,  <span class="cit-name-given-names">Z.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Kemp</span>,  <span class="cit-name-given-names">R.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Mubasher</span>,  <span class="cit-name-given-names">R.</span></span>, and <span class="cit-auth"><span class="cit-name-surname">Dukas</span>,  <span class="cit-name-given-names">R</span></span>. (<span class="cit-pub-date">2014</span>). <span class="cit-article-title">Dynamics of social behavior in fruit fly larvae</span>. <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">9</span>, <span class="cit-fpage">e95495</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BOne%26rft.volume%253D9%26rft.spage%253De95495%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0095495%26rft_id%253Dinfo%253Apmid%252F24740198%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0095495&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24740198&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F09%2F20%2F776708.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">29.</span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-776708v1.29" data-doi="10.1242/jeb.127431"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Lihoreau</span>,  <span class="cit-name-given-names">M.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Clarke</span>,  <span class="cit-name-given-names">I.M.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Buhl</span>,  <span class="cit-name-given-names">J.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Sumpter</span>,  <span class="cit-name-given-names">D.J.</span></span>, and <span class="cit-auth"><span class="cit-name-surname">Simpson</span>,  <span class="cit-name-given-names">S.J</span></span>. (<span class="cit-pub-date">2016</span>). <span class="cit-article-title">Collective selection of food patches in Drosophila</span>. <abbr class="cit-jnl-abbrev">J Exp Biol</abbr> <span class="cit-vol">219</span>, <span class="cit-fpage">668</span>–<span class="cit-lpage">675</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ%2BExp%2BBiol%26rft_id%253Dinfo%253Adoi%252F10.1242%252Fjeb.127431%26rft_id%253Dinfo%253Apmid%252F26747899%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiamV4YmlvIjtzOjU6InJlc2lkIjtzOjk6IjIxOS81LzY2OCI7czo0OiJhdG9tIjtzOjM3OiIvYmlvcnhpdi9lYXJseS8yMDE5LzA5LzIwLzc3NjcwOC5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">30.</span><a class="rev-xref-ref" href="#xref-ref-30-1" title="View reference 30. in text" id="ref-30">↵</a><div class="cit ref-cit ref-journal" id="cit-776708v1.30" data-doi="10.1111/j.1601-183X.2011.00740.x"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Simon</span>,  <span class="cit-name-given-names">A.F.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Chou</span>,  <span class="cit-name-given-names">M.T.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Salazar</span>,  <span class="cit-name-given-names">E.D.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Nicholson</span>,  <span class="cit-name-given-names">T.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Saini</span>,  <span class="cit-name-given-names">N.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Metchev</span>,  <span class="cit-name-given-names">S.</span></span>, and <span class="cit-auth"><span class="cit-name-surname">Krantz</span>,  <span class="cit-name-given-names">D.E</span></span>. (<span class="cit-pub-date">2012</span>). <span class="cit-article-title">A simple assay to study social behavior in Drosophila: measurement of social space within a group</span>. <abbr class="cit-jnl-abbrev">Genes Brain Behav</abbr> <span class="cit-vol">11</span>, <span class="cit-fpage">243</span>–<span class="cit-lpage">252</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenes%252C%2Bbrain%252C%2Band%2Bbehavior%26rft.stitle%253DGenes%2BBrain%2BBehav%26rft.aulast%253DSimon%26rft.auinit1%253DA.%2BF.%26rft.volume%253D11%26rft.issue%253D2%26rft.spage%253D243%26rft.epage%253D252%26rft.atitle%253DA%2Bsimple%2Bassay%2Bto%2Bstudy%2Bsocial%2Bbehavior%2Bin%2BDrosophila%253A%2Bmeasurement%2Bof%2Bsocial%2Bspace%2Bwithin%2Ba%2Bgroup.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1601-183X.2011.00740.x%26rft_id%253Dinfo%253Apmid%252F22010812%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1601-183X.2011.00740.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22010812&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F09%2F20%2F776708.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000299634400014&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">31.</span><a class="rev-xref-ref" href="#xref-ref-31-1" title="View reference 31. in text" id="ref-31">↵</a><div class="cit ref-cit ref-journal" id="cit-776708v1.31" data-doi="10.1146/annurev.neuro.30.051606.094306"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Vosshall</span>,  <span class="cit-name-given-names">L.B.</span></span>, and <span class="cit-auth"><span class="cit-name-surname">Stocker</span>,  <span class="cit-name-given-names">R.F</span></span>. (<span class="cit-pub-date">2007</span>). <span class="cit-article-title">Molecular architecture of smell and taste in Drosophila</span>. <abbr class="cit-jnl-abbrev">Annu Rev Neurosci</abbr> <span class="cit-vol">30</span>, <span class="cit-fpage">505</span>–<span class="cit-lpage">533</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAnnual%2Breview%2Bof%2Bneuroscience%26rft.stitle%253DAnnu%2BRev%2BNeurosci%26rft.aulast%253DVosshall%26rft.auinit1%253DL.%2BB.%26rft.volume%253D30%26rft.spage%253D505%26rft.epage%253D533%26rft.atitle%253DMolecular%2Barchitecture%2Bof%2Bsmell%2Band%2Btaste%2Bin%2BDrosophila.%26rft_id%253Dinfo%253Adoi%252F10.1146%252Fannurev.neuro.30.051606.094306%26rft_id%253Dinfo%253Apmid%252F17506643%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1146/annurev.neuro.30.051606.094306&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17506643&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F09%2F20%2F776708.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000248735400020&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">32.</span><a class="rev-xref-ref" href="#xref-ref-32-1" title="View reference 32. in text" id="ref-32">↵</a><div class="cit ref-cit ref-journal" id="cit-776708v1.32"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Bontonou</span>,  <span class="cit-name-given-names">G.</span></span>, and <span class="cit-auth"><span class="cit-name-surname">Wicker-Thomas</span>,  <span class="cit-name-given-names">C</span></span>. (<span class="cit-pub-date">2014</span>). <span class="cit-article-title">Sexual Communication in the Drosophila Genus</span>. <abbr class="cit-jnl-abbrev">Insects</abbr> <span class="cit-vol">5</span>, <span class="cit-fpage">439</span>–<span class="cit-lpage">458</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DInsects%26rft.volume%253D5%26rft.spage%253D439%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">33.</span><a class="rev-xref-ref" href="#xref-ref-33-1" title="View reference 33. in text" id="ref-33">↵</a><div class="cit ref-cit ref-journal" id="cit-776708v1.33" data-doi="10.1007/s10519-005-3220-5"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Ferveur</span>,  <span class="cit-name-given-names">J.F</span></span>. (<span class="cit-pub-date">2005</span>). <span class="cit-article-title">Cuticular hydrocarbons: their evolution and roles in Drosophila pheromonal communication</span>. <abbr class="cit-jnl-abbrev">Behav Genet</abbr> <span class="cit-vol">35</span>, <span class="cit-fpage">279</span>–<span class="cit-lpage">295</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBehavior%2Bgenetics%26rft.stitle%253DBehav%2BGenet%26rft.aulast%253DFerveur%26rft.auinit1%253DJ.%2BF.%26rft.volume%253D35%26rft.issue%253D3%26rft.spage%253D279%26rft.epage%253D295%26rft.atitle%253DCuticular%2Bhydrocarbons%253A%2Btheir%2Bevolution%2Band%2Broles%2Bin%2BDrosophila%2Bpheromonal%2Bcommunication.%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs10519-005-3220-5%26rft_id%253Dinfo%253Apmid%252F15864443%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s10519-005-3220-5&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15864443&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F09%2F20%2F776708.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000228861200006&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">34.</span><a class="rev-xref-ref" href="#xref-ref-34-1" title="View reference 34. in text" id="ref-34">↵</a><div class="cit ref-cit ref-journal" id="cit-776708v1.34" data-doi="10.1016/j.cub.2008.07.089"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Krupp</span>,  <span class="cit-name-given-names">J.J.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Kent</span>,  <span class="cit-name-given-names">C.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Billeter</span>,  <span class="cit-name-given-names">J.C.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Azanchi</span>,  <span class="cit-name-given-names">R.</span></span>, <span class="cit-auth"><span class="cit-name-surname">So</span>,  <span class="cit-name-given-names">A.K.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Schonfeld</span>,  <span class="cit-name-given-names">J.A.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Smith</span>,  <span class="cit-name-given-names">B.P.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Lucas</span>,  <span class="cit-name-given-names">C.</span></span>, and <span class="cit-auth"><span class="cit-name-surname">Levine</span>,  <span class="cit-name-given-names">J.D</span></span>. (<span class="cit-pub-date">2008</span>). <span class="cit-article-title">Social experience modifies pheromone expression and mating behavior in male Drosophila melanogaster</span>. <abbr class="cit-jnl-abbrev">Curr Biol</abbr> <span class="cit-vol">18</span>, <span class="cit-fpage">1373</span>–<span class="cit-lpage">1383</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCurrent%2Bbiology%2B%253A%2B%2BCB%26rft.stitle%253DCurr%2BBiol%26rft.aulast%253DKrupp%26rft.auinit1%253DJ.%2BJ.%26rft.volume%253D18%26rft.issue%253D18%26rft.spage%253D1373%26rft.epage%253D1383%26rft.atitle%253DSocial%2Bexperience%2Bmodifies%2Bpheromone%2Bexpression%2Band%2Bmating%2Bbehavior%2Bin%2Bmale%2BDrosophila%2Bmelanogaster.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.cub.2008.07.089%26rft_id%253Dinfo%253Apmid%252F18789691%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.cub.2008.07.089&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18789691&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F09%2F20%2F776708.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000259523600022&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">35.</span><a class="rev-xref-ref" href="#xref-ref-35-1" title="View reference 35. in text" id="ref-35">↵</a><div class="cit ref-cit ref-journal" id="cit-776708v1.35" data-doi="10.1038/nature08495"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Billeter</span>,  <span class="cit-name-given-names">J.C.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Atallah</span>,  <span class="cit-name-given-names">J.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Krupp</span>,  <span class="cit-name-given-names">J.J.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Millar</span>,  <span class="cit-name-given-names">J.G.</span></span>, and <span class="cit-auth"><span class="cit-name-surname">Levine</span>,  <span class="cit-name-given-names">J.D</span></span>. (<span class="cit-pub-date">2009</span>). <span class="cit-article-title">Specialized cells tag sexual and species identity in Drosophila melanogaster</span>. <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">461</span>, <span class="cit-fpage">987</span>–<span class="cit-lpage">991</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DBilleter%26rft.auinit1%253DJ.%2BC.%26rft.volume%253D461%26rft.issue%253D7266%26rft.spage%253D987%26rft.epage%253D991%26rft.atitle%253DSpecialized%2Bcells%2Btag%2Bsexual%2Band%2Bspecies%2Bidentity%2Bin%2BDrosophila%2Bmelanogaster.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature08495%26rft_id%253Dinfo%253Apmid%252F19829381%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature08495&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19829381&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F09%2F20%2F776708.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000270817700049&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">36.</span><a class="rev-xref-ref" href="#xref-ref-36-1" title="View reference 36. in text" id="ref-36">↵</a><div class="cit ref-cit ref-journal" id="cit-776708v1.36" data-doi="10.1016/0003-3472(61)90053-7"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Sexton</span>,  <span class="cit-name-given-names">O.J.</span></span>, and <span class="cit-auth"><span class="cit-name-surname">Stalker</span>,  <span class="cit-name-given-names">H.D</span></span>. (<span class="cit-pub-date">1961</span>). <span class="cit-article-title">Spacing patterns of female Drosophila paramelanica</span>. <abbr class="cit-jnl-abbrev">Animal Behaviour</abbr> <span class="cit-vol">9</span>, <span class="cit-fpage">77</span>–<span class="cit-lpage">81</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAnimal%2BBehaviour%26rft.volume%253D9%26rft.spage%253D77%26rft_id%253Dinfo%253Adoi%252F10.1016%252F0003-3472%252861%252990053-7%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/0003-3472(61)90053-7&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label">37.</span><a class="rev-xref-ref" href="#xref-ref-37-1" title="View reference 37. in text" id="ref-37">↵</a><div class="cit ref-cit ref-journal" id="cit-776708v1.37" data-doi="10.2307/1383294"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Bergallo</span>,  <span class="cit-name-given-names">H.G.</span></span>, and <span class="cit-auth"><span class="cit-name-surname">Magnusson</span>,  <span class="cit-name-given-names">W.E</span></span>. (<span class="cit-pub-date">1999</span>). <span class="cit-article-title">Effects of climate and food availability on four rodent species in southeastern Brazil</span>. <abbr class="cit-jnl-abbrev">Journal of Mammalogy</abbr> <span class="cit-vol">80</span>, <span class="cit-fpage">472</span>–<span class="cit-lpage">486</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BMammalogy%26rft_id%253Dinfo%253Adoi%252F10.2307%252F1383294%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2307/1383294&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000080639400013&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">38.</span><a class="rev-xref-ref" href="#xref-ref-38-1" title="View reference 38. in text" id="ref-38">↵</a><div class="cit ref-cit ref-journal" id="cit-776708v1.38"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Craine</span>,  <span class="cit-name-given-names">J.M</span></span>. (<span class="cit-pub-date">2013</span>). <span class="cit-article-title">Long-term climate sensitivity of grazer performance: a cross-site study</span>. <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">8</span>, <span class="cit-fpage">e67065</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BOne%26rft.volume%253D8%26rft.spage%253De67065%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">39.</span><a class="rev-xref-ref" href="#xref-ref-39-1" title="View reference 39. in text" id="ref-39">↵</a><div class="cit ref-cit ref-journal" id="cit-776708v1.39" data-doi="10.1111/j.1461-0248.2004.00671.x"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Currie</span>,  <span class="cit-name-given-names">D.J.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Mittelbach</span>,  <span class="cit-name-given-names">G.G.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Cornell</span>,  <span class="cit-name-given-names">H.V.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Field</span>,  <span class="cit-name-given-names">R.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Guégan</span>,  <span class="cit-name-given-names">J.F.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Hawkins</span>,  <span class="cit-name-given-names">B.A.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Kaufman</span>,  <span class="cit-name-given-names">D.M.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Kerr</span>,  <span class="cit-name-given-names">J.T.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Oberdorff</span>,  <span class="cit-name-given-names">T.</span></span>, and <span class="cit-auth"><span class="cit-name-surname">O’Brien</span>,  <span class="cit-name-given-names">E</span></span>. (<span class="cit-pub-date">2004</span>). <span class="cit-article-title">Predictions and tests of climate-based hypotheses of broad-scale variation in taxonomic richness</span>. <abbr class="cit-jnl-abbrev">Ecology letters</abbr> <span class="cit-vol">7</span>, <span class="cit-fpage">1121</span>–<span class="cit-lpage">1134</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEcology%2Bletters%26rft.volume%253D7%26rft.spage%253D1121%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1461-0248.2004.00671.x%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1461-0248.2004.00671.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000225078000001&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">40.</span><a class="rev-xref-ref" href="#xref-ref-40-1" title="View reference 40. in text" id="ref-40">↵</a><div class="cit ref-cit ref-journal" id="cit-776708v1.40"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">McInerny</span>,  <span class="cit-name-given-names">M.C.</span></span>, and <span class="cit-auth"><span class="cit-name-surname">Cross</span>,  <span class="cit-name-given-names">T.K</span></span>. (<span class="cit-pub-date">1999</span>). <span class="cit-article-title">Effects of lake productivity, climate warming, and intraspecific density on growth and growth patterns of black crappie in southern Minnesota lakes</span>. <abbr class="cit-jnl-abbrev">Journal of Freshwater Ecology</abbr> <span class="cit-vol">14</span>, <span class="cit-fpage">255</span>–<span class="cit-lpage">264</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BFreshwater%2BEcology%26rft.volume%253D14%26rft.spage%253D255%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">41.</span><a class="rev-xref-ref" href="#xref-ref-41-1" title="View reference 41. in text" id="ref-41">↵</a><div class="cit ref-cit ref-journal" id="cit-776708v1.41"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Sánchez-Bayo</span>,  <span class="cit-name-given-names">F.</span></span>, and <span class="cit-auth"><span class="cit-name-surname">Wyckhuys</span>,  <span class="cit-name-given-names">K.A</span></span>. (<span class="cit-pub-date">2019</span>). <span class="cit-article-title">Worldwide decline of the entomofauna: A review of its drivers</span>. <abbr class="cit-jnl-abbrev">Biological conservation</abbr> <span class="cit-vol">232</span>, <span class="cit-fpage">8</span>–<span class="cit-lpage">27</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBiological%2Bconservation%26rft.volume%253D232%26rft.spage%253D8%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">42.</span><a class="rev-xref-ref" href="#xref-ref-42-1" title="View reference 42. in text" id="ref-42">↵</a><div class="cit ref-cit ref-journal" id="cit-776708v1.42" data-doi="10.1371/journal.pone.0185809"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Hallmann</span>,  <span class="cit-name-given-names">C.A.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Sorg</span>,  <span class="cit-name-given-names">M.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Jongejans</span>,  <span class="cit-name-given-names">E.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Siepel</span>,  <span class="cit-name-given-names">H.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Hofland</span>,  <span class="cit-name-given-names">N.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Schwan</span>,  <span class="cit-name-given-names">H.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Stenmans</span>,  <span class="cit-name-given-names">W.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Müller</span>,  <span class="cit-name-given-names">A.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Sumser</span>,  <span class="cit-name-given-names">H.</span></span>, and <span class="cit-auth"><span class="cit-name-surname">Hörren</span>,  <span class="cit-name-given-names">T</span></span>. (<span class="cit-pub-date">2017</span>). <span class="cit-article-title">More than 75 percent decline over 27 years in total flying insect biomass in protected areas</span>. <abbr class="cit-jnl-abbrev">PloS one</abbr> <span class="cit-vol">12</span>, <span class="cit-fpage">e0185809</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPloS%2Bone%26rft.volume%253D12%26rft.spage%253De0185809%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0185809%26rft_id%253Dinfo%253Apmid%252F29045418%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0185809&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=29045418&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F09%2F20%2F776708.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">43.</span><a class="rev-xref-ref" href="#xref-ref-43-1" title="View reference 43. in text" id="ref-43">↵</a><div class="cit ref-cit ref-book" id="cit-776708v1.43"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Krause</span>,  <span class="cit-name-given-names">J.</span></span>, <span class="cit-auth"><span class="cit-name-surname">James</span>,  <span class="cit-name-given-names">R.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Franks</span>,  <span class="cit-name-given-names">D.W.</span></span>, and <span class="cit-auth"><span class="cit-name-surname">Croft</span>,  <span class="cit-name-given-names">D.P</span></span>. (<span class="cit-pub-date">2015</span>). <span class="cit-source">Animal social networks</span>, (<span class="cit-publ-name">Oxford University Press</span>, <span class="cit-publ-loc">USA</span>).</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">44.</span><a class="rev-xref-ref" href="#xref-ref-44-1" title="View reference 44. in text" id="ref-44">↵</a><div class="cit ref-cit ref-journal" id="cit-776708v1.44" data-doi="10.1371/journal.pone.0022375"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Barocas</span>,  <span class="cit-name-given-names">A.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Ilany</span>,  <span class="cit-name-given-names">A.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Koren</span>,  <span class="cit-name-given-names">L.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Kam</span>,  <span class="cit-name-given-names">M.</span></span>, and <span class="cit-auth"><span class="cit-name-surname">Geffen</span>,  <span class="cit-name-given-names">E</span></span>. (<span class="cit-pub-date">2011</span>). <span class="cit-article-title">Variance in centrality within rock hyrax social networks predicts adult longevity</span>. <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">6</span>, <span class="cit-fpage">e22375</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BONE%26rft.aulast%253DBarocas%26rft.auinit1%253DA.%26rft.volume%253D6%26rft.issue%253D7%26rft.spage%253De22375%26rft.epage%253De22375%26rft.atitle%253DVariance%2Bin%2Bcentrality%2Bwithin%2Brock%2Bhyrax%2Bsocial%2Bnetworks%2Bpredicts%2Badult%2Blongevity.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0022375%26rft_id%253Dinfo%253Apmid%252F21818314%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0022375&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21818314&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F09%2F20%2F776708.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">45.</span><a class="rev-xref-ref" href="#xref-ref-45-1" title="View reference 45. in text" id="ref-45">↵</a><div class="cit ref-cit ref-journal" id="cit-776708v1.45"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Ma</span>,  <span class="cit-name-given-names">S.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Avanesov</span>,  <span class="cit-name-given-names">A.S.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Porter</span>,  <span class="cit-name-given-names">E.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Lee</span>,  <span class="cit-name-given-names">B.C.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Mariotti</span>,  <span class="cit-name-given-names">M.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Zemskaya</span>,  <span class="cit-name-given-names">N.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Guigo</span>,  <span class="cit-name-given-names">R.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Moskalev</span>,  <span class="cit-name-given-names">A.A.</span></span>, and <span class="cit-auth"><span class="cit-name-surname">Gladyshev</span>,  <span class="cit-name-given-names">V.N</span></span>. (<span class="cit-pub-date">2018</span>). <span class="cit-article-title">Comparative transcriptomics across 14 Drosophila species reveals signatures of longevity</span>. <abbr class="cit-jnl-abbrev">Aging cell</abbr> <span class="cit-vol">17</span>, <span class="cit-fpage">e12740</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAging%2Bcell%26rft.volume%253D17%26rft.spage%253De12740%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">46.</span><a class="rev-xref-ref" href="#xref-ref-46-1" title="View reference 46. in text" id="ref-46">↵</a><div class="cit ref-cit ref-journal" id="cit-776708v1.46"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Matos</span>,  <span class="cit-name-given-names">R.J.</span></span>, and <span class="cit-auth"><span class="cit-name-surname">Schlupp</span>,  <span class="cit-name-given-names">I</span></span>. (<span class="cit-pub-date">2005</span>). <span class="cit-article-title">Performing in front of an audience: signalers and the social environment</span>. <abbr class="cit-jnl-abbrev">Animal communication networks</abbr>, <span class="cit-fpage">63</span>–<span class="cit-lpage">83</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">47.</span><a class="rev-xref-ref" href="#xref-ref-47-1" title="View reference 47. in text" id="ref-47">↵</a><div class="cit ref-cit ref-journal" id="cit-776708v1.47"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Markow</span>,  <span class="cit-name-given-names">T.A</span></span>. (<span class="cit-pub-date">2015</span>). <span class="cit-article-title">The secret lives of Drosophila flies</span>. <abbr class="cit-jnl-abbrev">Elife</abbr> <span class="cit-vol">4</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">48.</span><a class="rev-xref-ref" href="#xref-ref-48-1" title="View reference 48. in text" id="ref-48">↵</a><div class="cit ref-cit ref-journal" id="cit-776708v1.48" data-doi="10.1146/annurev-ento-011613-162041"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Dickinson</span>,  <span class="cit-name-given-names">M.H</span></span>. (<span class="cit-pub-date">2014</span>). <span class="cit-article-title">Death valley, Drosophila, and the Devonian toolkit</span>. <abbr class="cit-jnl-abbrev">Annual review of entomology</abbr> <span class="cit-vol">59</span>, <span class="cit-fpage">51</span>–<span class="cit-lpage">72</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAnnual%2Breview%2Bof%2Bentomology%26rft.volume%253D59%26rft.spage%253D51%26rft_id%253Dinfo%253Adoi%252F10.1146%252Fannurev-ento-011613-162041%26rft_id%253Dinfo%253Apmid%252F24160432%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1146/annurev-ento-011613-162041&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24160432&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F09%2F20%2F776708.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">49.</span><a class="rev-xref-ref" href="#xref-ref-49-1" title="View reference 49. in text" id="ref-49">↵</a><div class="cit ref-cit ref-journal" id="cit-776708v1.49" data-doi="10.5194/hessd-4-439-2007"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Peel</span>,  <span class="cit-name-given-names">M.C.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Finlayson</span>,  <span class="cit-name-given-names">B.L.</span></span>, and <span class="cit-auth"><span class="cit-name-surname">McMahon</span>,  <span class="cit-name-given-names">T.A</span></span>. (<span class="cit-pub-date">2007</span>). <span class="cit-article-title">Updated world map of the Köppen-Geiger climate classification</span>. <abbr class="cit-jnl-abbrev">Hydrology and earth system sciences discussions</abbr> <span class="cit-vol">4</span>, <span class="cit-fpage">439</span>–<span class="cit-lpage">473</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DHydrology%2Band%2Bearth%2Bsystem%2Bsciences%2Bdiscussions%26rft.volume%253D4%26rft.spage%253D439%26rft_id%253Dinfo%253Adoi%252F10.5194%252Fhessd-4-439-2007%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.5194/hessd-4-439-2007&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label">50.</span><a class="rev-xref-ref" href="#xref-ref-50-1" title="View reference 50. in text" id="ref-50">↵</a><div class="cit ref-cit ref-journal" id="cit-776708v1.50" data-doi="10.1098/rsbl.2014.0749"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Schneider</span>,  <span class="cit-name-given-names">J.</span></span>, and <span class="cit-auth"><span class="cit-name-surname">Levine</span>,  <span class="cit-name-given-names">J.D</span></span>. (<span class="cit-pub-date">2014</span>). <span class="cit-article-title">Automated identification of social interaction criteria in Drosophila melanogaster</span>. <abbr class="cit-jnl-abbrev">Biol Lett</abbr> <span class="cit-vol">10</span>, <span class="cit-fpage">20140749</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBiol%2BLett%26rft_id%253Dinfo%253Adoi%252F10.1098%252Frsbl.2014.0749%26rft_id%253Dinfo%253Apmid%252F25354920%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1098/rsbl.2014.0749&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25354920&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F09%2F20%2F776708.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">51.</span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-776708v1.51"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Paradis</span>,  <span class="cit-name-given-names">E.</span></span>, and <span class="cit-auth"><span class="cit-name-surname">Schliep</span>,  <span class="cit-name-given-names">K</span></span>. (<span class="cit-pub-date">2018</span>). <span class="cit-article-title">ape 5.0: an environment for modern phylogenetics and evolutionary analyses in R</span>. <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">35</span>, <span class="cit-fpage">526</span>–<span class="cit-lpage">528</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft.volume%253D35%26rft.spage%253D526%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">52.</span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-776708v1.52"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Revell</span>,  <span class="cit-name-given-names">L.J</span></span>. (<span class="cit-pub-date">2012</span>). <span class="cit-article-title">phytools: an R package for phylogenetic comparative biology (and other things)</span>. <abbr class="cit-jnl-abbrev">Methods in Ecology and Evolution</abbr> <span class="cit-vol">3</span>, <span class="cit-fpage">217</span>–<span class="cit-lpage">223</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMethods%2Bin%2BEcology%2Band%2BEvolution%26rft.volume%253D3%26rft.spage%253D217%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">53.</span><a class="rev-xref-ref" href="#xref-ref-53-1" title="View reference 53. in text" id="ref-53">↵</a><div class="cit ref-cit ref-journal" id="cit-776708v1.53" data-doi="10.1086/284325"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Felsenstein</span>,  <span class="cit-name-given-names">J</span></span>. (<span class="cit-pub-date">1985</span>). <span class="cit-article-title">Phylogenies and the comparative method</span>. <abbr class="cit-jnl-abbrev">The American Naturalist</abbr> <span class="cit-vol">125</span>, <span class="cit-fpage">1</span>–<span class="cit-lpage">15</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DThe%2BAmerican%2BNaturalist%26rft.volume%253D125%26rft.spage%253D1%26rft_id%253Dinfo%253Adoi%252F10.1086%252F284325%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1086/284325&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=A1985AAK4600001&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">54.</span><a class="rev-xref-ref" href="#xref-ref-54-1" title="View reference 54. in text" id="ref-54">↵</a><div class="cit ref-cit ref-journal" id="cit-776708v1.54" data-doi="10.1080/10635150802302427"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Revell</span>,  <span class="cit-name-given-names">L.J.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Harmon</span>,  <span class="cit-name-given-names">L.J.</span></span>, and <span class="cit-auth"><span class="cit-name-surname">Collar</span>,  <span class="cit-name-given-names">D.C</span></span>. (<span class="cit-pub-date">2008</span>). <span class="cit-article-title">Phylogenetic signal, evolutionary process, and rate</span>. <abbr class="cit-jnl-abbrev">Syst Biol</abbr> <span class="cit-vol">57</span>, <span class="cit-fpage">591</span>–<span class="cit-lpage">601</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DSyst%2BBiol%26rft_id%253Dinfo%253Adoi%252F10.1080%252F10635150802302427%26rft_id%253Dinfo%253Apmid%252F18709597%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1080/10635150802302427&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18709597&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F09%2F20%2F776708.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000263702800005&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">55.</span><a class="rev-xref-ref" href="#xref-ref-55-1" title="View reference 55. in text" id="ref-55">↵</a><div class="cit ref-cit ref-journal" id="cit-776708v1.55" data-doi="10.1603/0013-8746(2002)095[0156:MPOTDV]2.0.CO;2"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Spicer</span>,  <span class="cit-name-given-names">G.S.</span></span>, and <span class="cit-auth"><span class="cit-name-surname">Bell</span>,  <span class="cit-name-given-names">C</span></span>. (<span class="cit-pub-date">2002</span>). <span class="cit-article-title">Molecular phylogeny of the Drosophila virilis species group (Diptera: Drosophilidae) inferred from mitochondrial 12S and 16S ribosomal RNA genes</span>. <abbr class="cit-jnl-abbrev">Annals of the Entomological Society of America</abbr> <span class="cit-vol">95</span>, <span class="cit-fpage">156</span>–<span class="cit-lpage">161</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAnnals%2Bof%2Bthe%2BEntomological%2BSociety%2Bof%2BAmerica%26rft_id%253Dinfo%253Adoi%252F10.1603%252F0013-8746%25282002%2529095%255B0156%253AMPOTDV%255D2.0.CO%253B2%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1603/0013-8746(2002)095[0156:MPOTDV]2.0.CO;2&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label">56.</span><a class="rev-xref-ref" href="#xref-ref-56-1" title="View reference 56. in text" id="ref-56">↵</a><div class="cit ref-cit ref-journal" id="cit-776708v1.56" data-doi="10.1126/science.283.5401.499"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Sandweiss</span>,  <span class="cit-name-given-names">D.H.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Maasch</span>,  <span class="cit-name-given-names">K.A.</span></span>, and <span class="cit-auth"><span class="cit-name-surname">Anderson</span>,  <span class="cit-name-given-names">D.G</span></span>. (<span class="cit-pub-date">1999</span>). <span class="cit-article-title">Transitions in the mid-Holocene</span>. <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">283</span>, <span class="cit-fpage">499</span>–<span class="cit-lpage">500</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DSandweiss%26rft.auinit1%253DD.%2BH.%26rft.volume%253D283%26rft.issue%253D5401%26rft.spage%253D499%26rft.epage%253D500%26rft.atitle%253DCLIMATE%2BAND%2BCULTURE%253ATransitions%2Bin%2Bthe%2BMid-Holocene%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.283.5401.499%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiRlVMTCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEyOiIyODMvNTQwMS80OTkiO3M6NDoiYXRvbSI7czozNzoiL2Jpb3J4aXYvZWFybHkvMjAxOS8wOS8yMC83NzY3MDguYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">57.</span><a class="rev-xref-ref" href="#xref-ref-57-1" title="View reference 57. in text" id="ref-57">↵</a><div class="cit ref-cit ref-journal" id="cit-776708v1.57"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Revell</span>,  <span class="cit-name-given-names">L.J</span></span>. (<span class="cit-pub-date">2013</span>). <span class="cit-article-title">Two new graphical methods for mapping trait evolution on phylogenies</span>. <abbr class="cit-jnl-abbrev">Methods in Ecology and Evolution</abbr> <span class="cit-vol">4</span>, <span class="cit-fpage">754</span>–<span class="cit-lpage">759</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMethods%2Bin%2BEcology%2Band%2BEvolution%26rft.volume%253D4%26rft.spage%253D754%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.biorxiv.org/content/10.1101/776708v1.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-disqus-comment" >
  
      
  
  <div class="pane-content">
    <div id="disqus_thread"><noscript><p><a href="http://biorxivstage.disqus.com/?url=https%3A%2F%2Fwww.biorxiv.org%2Fcontent%2F10.1101%2F776708v1" class="" data-icon-position="" data-hide-link-title="0">View the discussion thread.</a></p></noscript></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/10.1101/776773v1" title="Nest microbiota and pathogen abundance impact hatching success in sea turtle conservation" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous</span></a></span><span class="pager-next"><a href="/content/10.1101/569319v4" title="Neonatal morphometric similarity mapping for predicting brain age and characterizing neuroanatomic variation associated with preterm birth" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    Posted&nbsp;September 20, 2019.  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-biorxiv-art-tools" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-biorxiv_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/10.1101/776708v1.full.pdf" target="_self" class="article-dl-pdf-link link-icon"><i class="icon-external-link-sign"></i> <span class="title">Download PDF</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-biorxiv-supplementary-fragment" >
  
      
  
  <div class="pane-content">
    <p><a href="/content/10.1101/776708v1.supplementary-material"><i class="icon-file"></i>Supplementary Material </a></p>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-biorxiv-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="Email this Article" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-biorxiv_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/10.1101/776708v1.full" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for your interest in spreading the word about bioRxiv.</p><p>NOTE: Your email address is requested solely to identify you as the sender of this article.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/10.1101/776708v1" class="active" data-icon-position="" data-hide-link-title="0">The Evolution of Social Organization: Climate Influences Variation in Drosophilid Social Networks</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from bioRxiv
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would like to see this page from the bioRxiv website.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/916114" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-BuBIWuNUv-Xunu6crm-qY0yruLJdcWUPINk1Sl5MYik" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-biorxiv-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-biorxiv_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="916114" id="node916114--2" data-pisa="biorxiv;776708v1" data-pisa-master="biorxiv;776708" data-seqnum="916114" data-apath="/biorxiv/early/2019/09/20/776708.atom"><div  class="highwire-cite highwire-cite-highwire-article highwire-citation-biorxiv-article-pap-list clearfix" >

      <div class="highwire-cite-title" >
      <div class="highwire-cite-title">The Evolution of Social Organization: Climate Influences Variation in Drosophilid Social Networks</div>    </div>
  
      <div  class="highwire-cite-authors" ><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0"><span class="nlm-given-names">Jacob A.</span> <span class="nlm-surname">Jezovit</span></span>, <span class="highwire-citation-author" data-delta="1"><span class="nlm-given-names">Rebecca</span> <span class="nlm-surname">Rooke</span></span>, <span class="highwire-citation-author" data-delta="2"><span class="nlm-given-names">Jonathan</span> <span class="nlm-surname">Schneider</span></span>, <span class="highwire-citation-author" data-delta="3"><span class="nlm-given-names">Joel D.</span> <span class="nlm-surname">Levine</span></span></span></div>
  
      <div  class="highwire-cite-metadata" ><span  class="highwire-cite-metadata-journal highwire-cite-metadata">bioRxiv </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">776708; </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata"><span class="doi_label">doi:</span> https://doi.org/10.1101/776708 </span></div>
  
  
  </div>
</div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.biorxiv.org/content/10.1101/776708v1" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.biorxiv.org/content/10.1101/776708v1&amp;title=The%20Evolution%20of%20Social%20Organization%3A%20Climate%20Influences%20Variation%20in%20Drosophilid%20Social%20Networks" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.biorxiv.org/content/10.1101/776708v1&amp;title=The%20Evolution%20of%20Social%20Organization%3A%20Climate%20Influences%20Variation%20in%20Drosophilid%20Social%20Networks" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.biorxiv.org/content/10.1101/776708v1&amp;text=The%20Evolution%20of%20Social%20Organization%3A%20Climate%20Influences%20Variation%20in%20Drosophilid%20Social%20Networks" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.biorxiv.org/content/10.1101/776708v1&amp;title=The%20Evolution%20of%20Social%20Organization%3A%20Climate%20Influences%20Variation%20in%20Drosophilid%20Social%20Networks" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.biorxiv.org/content/10.1101/776708v1&amp;t=The%20Evolution%20of%20Social%20Organization%3A%20Climate%20Influences%20Variation%20in%20Drosophilid%20Social%20Networks" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.biorxiv.org/content/10.1101/776708v1&amp;title=The%20Evolution%20of%20Social%20Organization%3A%20Climate%20Influences%20Variation%20in%20Drosophilid%20Social%20Networks" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.biorxiv.org/content/10.1101/776708v1&amp;title=The%20Evolution%20of%20Social%20Organization%3A%20Climate%20Influences%20Variation%20in%20Drosophilid%20Social%20Networks" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-biorxiv-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title="Citation Tools"><i class="icon-globe"></i> <span class="title">Citation Tools</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-biorxiv_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="916114" id="node916114--3" data-pisa="biorxiv;776708v1" data-pisa-master="biorxiv;776708" data-seqnum="916114" data-apath="/biorxiv/early/2019/09/20/776708.atom"><div  class="highwire-cite highwire-cite-highwire-article highwire-citation-biorxiv-article-pap-list clearfix" >

      <div class="highwire-cite-title" >
      <div class="highwire-cite-title">The Evolution of Social Organization: Climate Influences Variation in Drosophilid Social Networks</div>    </div>
  
      <div  class="highwire-cite-authors" ><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0"><span class="nlm-given-names">Jacob A.</span> <span class="nlm-surname">Jezovit</span></span>, <span class="highwire-citation-author" data-delta="1"><span class="nlm-given-names">Rebecca</span> <span class="nlm-surname">Rooke</span></span>, <span class="highwire-citation-author" data-delta="2"><span class="nlm-given-names">Jonathan</span> <span class="nlm-surname">Schneider</span></span>, <span class="highwire-citation-author" data-delta="3"><span class="nlm-given-names">Joel D.</span> <span class="nlm-surname">Levine</span></span></span></div>
  
      <div  class="highwire-cite-metadata" ><span  class="highwire-cite-metadata-journal highwire-cite-metadata">bioRxiv </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">776708; </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata"><span class="doi_label">doi:</span> https://doi.org/10.1101/776708 </span></div>
  
  
  </div>
</div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage&amp;rft.epage&amp;rft.atitle=The%20Evolution%20of%20Social%20Organization%3A%20Climate%20Influences%20Variation%20in%20Drosophilid%20Social%20Networks&amp;rft.volume&amp;rft.issue&amp;rft.date=2019-01-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=bioRxiv&amp;rft.au=Jezovit%2C+Jacob+A.&amp;rft.au=Rooke%2C+Rebecca&amp;rft.au=Schneider%2C+Jonathan&amp;rft.au=Levine%2C+Joel+D."></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/916114/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/916114/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/916114/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/916114/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/916114/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/916114/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/916114/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/916114/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/916114/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/916114/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/916114/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/916114/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><div class="item-list"><ul><li class="first"><a href="http://twitter.com/share?url=https%3A//www.biorxiv.org/content/10.1101/776708v1&amp;count=horizontal&amp;via=&amp;text=The%20Evolution%20of%20Social%20Organization%3A%20Climate%20Influences%20Variation%20in%20Drosophilid%20Social%20Networks&amp;counturl=https%3A//www.biorxiv.org/content/10.1101/776708v1" class="twitter-share-button service-links-twitter-widget" id="twitter_widget" title="Tweet This" rel="nofollow" data-icon-position="" data-hide-link-title="0"><span class="element-invisible">Tweet Widget</span></a></li><li><a href="http://www.facebook.com/plugins/like.php?href=https%3A//www.biorxiv.org/content/10.1101/776708v1&amp;layout=button_count&amp;show_faces=false&amp;action=like&amp;colorscheme=light&amp;width=100&amp;height=21&amp;font=&amp;locale=" id="facebook_like" title="I Like it" class="service-links-facebook-like" rel="nofollow" data-icon-position="" data-hide-link-title="0"><span class="element-invisible">Facebook Like</span></a></li><li class="last"><a href="https://www.biorxiv.org/content/10.1101/776708v1" id="google_plus_one" title="Plus it" class="service-links-google-plus-one" rel="nofollow" data-icon-position="" data-hide-link-title="0"><span class="element-invisible">Google Plus One</span></a></li></ul></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-collections" >
  
        <h2 class="pane-title">Subject Area</h2>
    
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-collections"><div class="highwire-list"><ul class="highwire-article-collection-term-list"><li class="first last odd"><span class="highwire-article-collection-term"><a href="/collection/evolutionary-biology" class="highlight" data-icon-position="" data-hide-link-title="0">Evolutionary Biology<i class="icon-caret-right"></i>
</a></span></li></ul></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-biorxiv-subject-collections block-style-col2" >
  
      
  
  <div class="pane-content">
    <div class="panel-flexible panels-flexible-8 clearfix" id="mini-panel-biorxiv_subject_collections">
<div class="panel-flexible-inside panels-flexible-8-inside">
<div class="panels-flexible-region panels-flexible-region-8-center panels-flexible-region-first panels-flexible-region-last">
  <div class="inside panels-flexible-region-inside panels-flexible-region-8-center-inside panels-flexible-region-inside-first panels-flexible-region-inside-last">
<div class="panel-pane pane-snippet" >
  
      
  
  <div class="pane-content">
    <div class="snippet biorxiv-subject-areas-table-title" id="biorxiv-subject-areas-table-title">
  
      
  <div class="snippet-content">
    <b>Subject Areas</b>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-snippet" >
  
      
  
  <div class="pane-content">
    <div class="snippet biorxiv-subject-areas-view-papers" id="biorxiv-subject-areas-view-papers">
  
      
  <div class="snippet-content">
    <a href="/content/early/recent"><strong>All Articles</strong></a>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-subject-collections" >
  
      
  
  <div class="pane-content">
    <ul id="collection" class="collection highwire-list-expand"><li class="outer collection depth-2  child first"><div class = "data-wrapper"><a href="/collection/animal-behavior-and-cognition" class="" data-icon-position="" data-hide-link-title="0">Animal Behavior and Cognition</a> <span class = "article-count">(1662)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/biochemistry" class="" data-icon-position="" data-hide-link-title="0">Biochemistry</a> <span class = "article-count">(2789)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/bioengineering" class="" data-icon-position="" data-hide-link-title="0">Bioengineering</a> <span class = "article-count">(1946)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/bioinformatics" class="" data-icon-position="" data-hide-link-title="0">Bioinformatics</a> <span class = "article-count">(10400)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/biophysics" class="" data-icon-position="" data-hide-link-title="0">Biophysics</a> <span class = "article-count">(4243)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/cancer-biology" class="" data-icon-position="" data-hide-link-title="0">Cancer Biology</a> <span class = "article-count">(3272)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/cell-biology" class="" data-icon-position="" data-hide-link-title="0">Cell Biology</a> <span class = "article-count">(4641)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/clinical-trials" class="" data-icon-position="" data-hide-link-title="0">Clinical Trials</a> <span class = "article-count">(135)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/developmental-biology" class="" data-icon-position="" data-hide-link-title="0">Developmental Biology</a> <span class = "article-count">(2889)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/ecology" class="" data-icon-position="" data-hide-link-title="0">Ecology</a> <span class = "article-count">(4544)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/epidemiology" class="" data-icon-position="" data-hide-link-title="0">Epidemiology</a> <span class = "article-count">(2041)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/evolutionary-biology" class="" data-icon-position="" data-hide-link-title="0">Evolutionary Biology</a> <span class = "article-count">(7288)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/genetics" class="" data-icon-position="" data-hide-link-title="0">Genetics</a> <span class = "article-count">(5540)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/genomics" class="" data-icon-position="" data-hide-link-title="0">Genomics</a> <span class = "article-count">(6893)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/immunology" class="" data-icon-position="" data-hide-link-title="0">Immunology</a> <span class = "article-count">(2434)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/microbiology" class="" data-icon-position="" data-hide-link-title="0">Microbiology</a> <span class = "article-count">(7616)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/molecular-biology" class="" data-icon-position="" data-hide-link-title="0">Molecular Biology</a> <span class = "article-count">(3051)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/neuroscience" class="" data-icon-position="" data-hide-link-title="0">Neuroscience</a> <span class = "article-count">(18885)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/paleontology" class="" data-icon-position="" data-hide-link-title="0">Paleontology</a> <span class = "article-count">(142)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/pathology" class="" data-icon-position="" data-hide-link-title="0">Pathology</a> <span class = "article-count">(477)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/pharmacology-and-toxicology" class="" data-icon-position="" data-hide-link-title="0">Pharmacology and Toxicology</a> <span class = "article-count">(791)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/physiology" class="" data-icon-position="" data-hide-link-title="0">Physiology</a> <span class = "article-count">(1178)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/plant-biology" class="" data-icon-position="" data-hide-link-title="0">Plant Biology</a> <span class = "article-count">(2771)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/scientific-communication-and-education" class="" data-icon-position="" data-hide-link-title="0">Scientific Communication and Education</a> <span class = "article-count">(686)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/synthetic-biology" class="" data-icon-position="" data-hide-link-title="0">Synthetic Biology</a> <span class = "article-count">(905)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/systems-biology" class="" data-icon-position="" data-hide-link-title="0">Systems Biology</a> <span class = "article-count">(2881)</span></div></li>
<li class="outer collection depth-2  child last"><div class = "data-wrapper"><a href="/collection/zoology" class="" data-icon-position="" data-hide-link-title="0">Zoology</a> <span class = "article-count">(472)</span></div></li>
</ul>  </div>

  
  </div>
  </div>
</div>
</div>
</div>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__VNH5GD8Zz7g6_hGCOZjVjdMndxxC6naiExSSJKX3k_A__kzmEdtbqCsJx5Z9yg2Qy0PptB__ufXm-TxUOl0KZzUw__QJacVmveFyLIBSPkp21xL0ZvuDW9DMBrD2HB1P4Ry5I.js"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__QJacVmveFyLIBSPkp21xL0ZvuDW9DMBrD2HB1P4Ry5I.js" defer="defer"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__N7ERJBYsOWyRnJgyoM125_Aiez2MOJGaUofG1JdWWBg__bLlcwhzZhT-IwoQ_dvm1_xPxU7E7gecFLC6Fyeqtv8s__QJacVmveFyLIBSPkp21xL0ZvuDW9DMBrD2HB1P4Ry5I.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
function euCookieComplianceLoadScripts() {}
//--><!]]>
</script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var eu_cookie_compliance_cookie_name = "";
//--><!]]>
</script>
<script type="text/javascript" src="//d33xdlntwy0kbs.cloudfront.net/cshl_custom.js"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__BmqjBnkz3MgYCAoc25s1lDRMEjLhC3mEPVonUFIHi08__Unwv5-ZIuHBfFwytsjEx1niBVJ7n1T4lPws7VrkHXM4__QJacVmveFyLIBSPkp21xL0ZvuDW9DMBrD2HB1P4Ry5I.js"></script>
  </body>
</html>
