<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<link rel="dns-prefetch" href="//d33xdlntwy0kbs.cloudfront.net" />
<link rel="shortcut icon" href="https://www.biorxiv.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<meta name="type" content="article" />
<meta name="category" content="article" />
<meta name="HW.identifier" content="/biorxiv/early/2020/02/13/2020.02.12.939041.atom" />
<meta name="HW.pisa" content="biorxiv;2020.02.12.939041v1" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="Climate and urbanization drive mosquito preference for humans" />
<meta name="DC.Identifier" content="10.1101/2020.02.12.939041" />
<meta name="DC.Date" content="2020-02-13" />
<meta name="DC.Publisher" content="Cold Spring Harbor Laboratory" />
<meta name="DC.Rights" content="© 2020, Posted by Cold Spring Harbor Laboratory. This pre-print is available under a Creative Commons License (Attribution-NonCommercial-NoDerivs 4.0 International), CC BY-NC-ND 4.0, as described at http://creativecommons.org/licenses/by-nc-nd/4.0/" />
<meta name="DC.AccessRights" content="restricted" />
<meta name="DC.Description" content="The majority of mosquito-borne illness is spread by a few mosquito species that have evolved to specialize in biting humans, yet the precise causes of this behavioral shift are poorly understood. We address this gap in the arboviral vector Aedes aegypti . We first characterize the behaviour of mosquitoes from 27 sites scattered across the species’ ancestral range in sub-Saharan Africa, revealing previously unrecognized diversity in female preference for human versus animal odor. We then use modelling to show that this diversity can be almost fully predicted by two ecological factors – dry season intensity and human population density. Finally we integrate this information with whole genome sequence data from 345 individual mosquitoes to identify a single underlying ancestry component linked to human preference, with genetic changes concentrated in a few key chromosomal regions. Our findings strongly suggest that human-biting in this important disease vector originally evolved as a by-product of breeding in human-stored water in areas where doing so provided the only means to survive the long, hot dry season. Our model also predicts that changes in human population density are likely to drive future mosquito evolution. Rapid urbanization may drive a shift to human-biting in many cities across Africa by 2050." />
<meta name="DC.Contributor" content="Noah H. Rose" />
<meta name="DC.Contributor" content="Massamba Sylla" />
<meta name="DC.Contributor" content="Athanase Badolo" />
<meta name="DC.Contributor" content="Joel Lutomiah" />
<meta name="DC.Contributor" content="Diego Ayala" />
<meta name="DC.Contributor" content="Ogechukwu B. Aribodor" />
<meta name="DC.Contributor" content="Nnenna Ibe" />
<meta name="DC.Contributor" content="Jewelna Akorli" />
<meta name="DC.Contributor" content="Sampson Otoo" />
<meta name="DC.Contributor" content="John-Paul Mutebi" />
<meta name="DC.Contributor" content="Alexis L. Kriete" />
<meta name="DC.Contributor" content="Eliza G. Ewing" />
<meta name="DC.Contributor" content="Rosemary Sang" />
<meta name="DC.Contributor" content="Andrea Gloria-Soria" />
<meta name="DC.Contributor" content="Jeffrey R. Powell" />
<meta name="DC.Contributor" content="Rachel E. Baker" />
<meta name="DC.Contributor" content="Bradley J. White" />
<meta name="DC.Contributor" content="Jacob E. Crawford" />
<meta name="DC.Contributor" content="Carolyn S. McBride" />
<meta name="article:published_time" content="2020-02-13" />
<meta name="article:section" content="New Results" />
<meta name="citation_title" content="Climate and urbanization drive mosquito preference for humans" />
<meta name="citation_abstract" lang="en" content="&lt;p&gt;The majority of mosquito-borne illness is spread by a few mosquito species that have evolved to specialize in biting humans, yet the precise causes of this behavioral shift are poorly understood. We address this gap in the arboviral vector &lt;i&gt;Aedes aegypti&lt;/i&gt;. We first characterize the behaviour of mosquitoes from 27 sites scattered across the species9 ancestral range in sub-Saharan Africa, revealing previously unrecognized diversity in female preference for human versus animal odor. We then use modelling to show that this diversity can be almost fully predicted by two ecological factors -- dry season intensity and human population density. Finally we integrate this information with whole genome sequence data from 345 individual mosquitoes to identify a single underlying ancestry component linked to human preference, with genetic changes concentrated in a few key chromosomal regions. Our findings strongly suggest that human-biting in this important disease vector originally evolved as a by-product of breeding in human-stored water in areas where doing so provided the only means to survive the long, hot dry season. Our model also predicts that changes in human population density are likely to drive future mosquito evolution. Rapid urbanization may drive a shift to human-biting in many cities across Africa by 2050.&lt;/p&gt;" />
<meta name="citation_journal_title" content="bioRxiv" />
<meta name="citation_publisher" content="Cold Spring Harbor Laboratory" />
<meta name="citation_publication_date" content="2020/01/01" />
<meta name="citation_mjid" content="biorxiv;2020.02.12.939041v1" />
<meta name="citation_id" content="2020.02.12.939041v1" />
<meta name="citation_public_url" content="https://www.biorxiv.org/content/10.1101/2020.02.12.939041v1" />
<meta name="citation_abstract_html_url" content="https://www.biorxiv.org/content/10.1101/2020.02.12.939041v1.abstract" />
<meta name="citation_full_html_url" content="https://www.biorxiv.org/content/10.1101/2020.02.12.939041v1.full" />
<meta name="citation_pdf_url" content="https://www.biorxiv.org/content/biorxiv/early/2020/02/13/2020.02.12.939041.full.pdf" />
<meta name="citation_doi" content="10.1101/2020.02.12.939041" />
<meta name="citation_num_pages" content="42" />
<meta name="citation_article_type" content="Article" />
<meta name="citation_section" content="New Results" />
<meta name="citation_firstpage" content="2020.02.12.939041" />
<meta name="citation_author" content="Noah H. Rose" />
<meta name="citation_author_institution" content="Department of Ecology &amp; Evolutionary Biology, Princeton University" />
<meta name="citation_author_institution" content="Princeton Neuroscience Institute, Princeton University" />
<meta name="citation_author_email" content="noahr@princeton.edu" />
<meta name="citation_author_email" content="csm7@princeton.edu" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0001-7129-4753" />
<meta name="citation_author" content="Massamba Sylla" />
<meta name="citation_author_institution" content="Unité d’Entomologie, de Bactériologie, de Virologie, Département de Biologie Animale, Faculté des Sciences et Techniques, Université Cheikh Anta DIOP" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0003-2893-9725" />
<meta name="citation_author" content="Athanase Badolo" />
<meta name="citation_author_institution" content="Laboratory of Fundamental and Applied Entomology, Université Joseph Ki-Zerbo" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0002-6652-4240" />
<meta name="citation_author" content="Joel Lutomiah" />
<meta name="citation_author_institution" content="Arbovirus/Viral Hemorrhagic Fevers Laboratory, Center for Virus Research, Kenya Medical Research Institute (KEMRI)" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0002-8507-2024" />
<meta name="citation_author" content="Diego Ayala" />
<meta name="citation_author_institution" content="UMR MIVEGEC, IRD, CNRS, Univ. Montpellier" />
<meta name="citation_author_institution" content="Le Centre International de Recherches Médicales de Franceville" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0003-4726-580X" />
<meta name="citation_author" content="Ogechukwu B. Aribodor" />
<meta name="citation_author_institution" content="Department of Zoology, Nnamdi Azikiwe University" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0002-2530-0865" />
<meta name="citation_author" content="Nnenna Ibe" />
<meta name="citation_author_institution" content="Department of Ecology &amp; Evolutionary Biology, Princeton University" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0002-6609-8173" />
<meta name="citation_author" content="Jewelna Akorli" />
<meta name="citation_author_institution" content="Department of Parasitology, Noguchi Memorial Institute for Medical Research, University of Ghana" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0002-3972-0860" />
<meta name="citation_author" content="Sampson Otoo" />
<meta name="citation_author_institution" content="Department of Parasitology, Noguchi Memorial Institute for Medical Research, University of Ghana" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0003-0548-8466" />
<meta name="citation_author" content="John-Paul Mutebi" />
<meta name="citation_author_institution" content="Centers for Disease Control and Prevention" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0001-7254-6537" />
<meta name="citation_author" content="Alexis L. Kriete" />
<meta name="citation_author_institution" content="Department of Ecology &amp; Evolutionary Biology, Princeton University" />
<meta name="citation_author_institution" content="Princeton Neuroscience Institute, Princeton University" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0002-4783-8367" />
<meta name="citation_author" content="Eliza G. Ewing" />
<meta name="citation_author_institution" content="Department of Ecology &amp; Evolutionary Biology, Princeton University" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0003-2960-6500" />
<meta name="citation_author" content="Rosemary Sang" />
<meta name="citation_author_institution" content="Arbovirus/Viral Hemorrhagic Fevers Laboratory, Center for Virus Research, Kenya Medical Research Institute (KEMRI)" />
<meta name="citation_author" content="Andrea Gloria-Soria" />
<meta name="citation_author_institution" content="Department of Ecology and Evolutionary Biology, Yale University" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0002-5401-3988" />
<meta name="citation_author" content="Jeffrey R. Powell" />
<meta name="citation_author_institution" content="Department of Ecology and Evolutionary Biology, Yale University" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0002-7651-5153" />
<meta name="citation_author" content="Rachel E. Baker" />
<meta name="citation_author_institution" content="Princeton Environmental Institute, Princeton University" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0002-2661-8103" />
<meta name="citation_author" content="Bradley J. White" />
<meta name="citation_author_institution" content="Verily Life Sciences" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0002-6444-4159" />
<meta name="citation_author" content="Jacob E. Crawford" />
<meta name="citation_author_institution" content="Verily Life Sciences" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0002-6745-9145" />
<meta name="citation_author" content="Carolyn S. McBride" />
<meta name="citation_author_institution" content="Department of Ecology &amp; Evolutionary Biology, Princeton University" />
<meta name="citation_author_institution" content="Princeton Neuroscience Institute, Princeton University" />
<meta name="citation_author_email" content="noahr@princeton.edu" />
<meta name="citation_author_email" content="csm7@princeton.edu" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0002-8898-1768" />
<meta name="citation_reference" content="WHO | Vector-borne diseases. WHO http://www.who.int/mediacentre/factsheets/fs387/en/." />
<meta name="citation_reference" content="Clements, A. N. The biology of mosquitoes. The biology of mosquitoes. (1999)." />
<meta name="citation_reference" content="McBride, C. S. Genes and Odors Underlying the Recent Evolution of Mosquito Preference for Humans. Current Biology 26, R41–R46 (2016)." />
<meta name="citation_reference" content="Takken, W. &amp; Verhulst, N. O. Host Preferences of Blood-Feeding Mosquitoes. Annual Review of Entomology 58, 433–453 (2013)." />
<meta name="citation_reference" content="Christophers, S. R. Aedes aegypti: The yellow fever mosquito. (Cambridge University Press Archive, 1960)." />
<meta name="citation_reference" content="Kotsakiozi, P. et al. Population structure of a vector of human diseases: Aedes aegypti in its ancestral range, Africa. Ecology and evolution 8, 7835–7848 (2018)." />
<meta name="citation_reference" content="Crawford, J. E. et al. Population genomics reveals that an anthropophilic population of Aedes aegypti mosquitoes in West Africa recently gave rise to American and Asian populations of this major disease vector. BMC Biology 15, 16 (2017)." />
<meta name="citation_reference" content="McClelland, G. A. H. &amp; Weitz, B. Serological identification of the natural hosts of Aedes aegypti (L.) and some other mosquitoes (Diptera, Culicidae) caught resting in vegetation in Kenya and Uganda. Annals of Tropical Medicine &amp; Parasitology 57, 214–224 (1963)." />
<meta name="citation_reference" content="Teesdale, C. Studies on the Bionomics of Aëdes aegypti (L.) in its Natural Habitats in a Coastal Region of Kenya. Bulletin of Entomological Research 46, 711–742 (1955)." />
<meta name="citation_reference" content="McBride, C. S. et al. Evolution of mosquito preference for humans linked to an odorant receptor. Nature 515, 222–227 (2014)." />
<meta name="citation_reference" content="Gouck, H. K. Host preferences of various strains of Aedes aegypti and A. simpsoni as determined by an olfactometer. Bulletin of the World Health Organization 47, 680 (1972)." />
<meta name="citation_reference" content="Fick, S. E. &amp; Hijmans, R. J. WorldClim 2: new 1-km spatial resolution climate surfaces for global land areas. International journal of climatology 37, 4302–4315 (2017)." />
<meta name="citation_reference" content="Silver, J. B. Mosquito ecology: field sampling methods. (springer science &amp; business media, 2007)." />
<meta name="citation_reference" content="Trpiš, M. Dry season survival of Aedes aegypti eggs in various breeding sites in the Dar es Salaam area, Tanzania. Bulletin of the World Health Organization 47, 433 (1972)." />
<meta name="citation_reference" content="Paupy, C. et al. Morphological and genetic variability within Aedes aegypti in Niakhar, Senegal. Infection, Genetics and Evolution 10, 473–480 (2010)." />
<meta name="citation_reference" content="Powell, J. R., Gloria-Soria, A. &amp; Kotsakiozi, P. Recent history of Aedes aegypti: vector Genomics and Epidemiology records. BioScience 68, 854–860 (2018)." />
<meta name="citation_reference" content="Mattingly, P. F. Genetical Aspects of the Aëdes Aegypti Problem: I.—Taxonomy and Bionomics. Annals of Tropical Medicine &amp; Parasitology 51, 392–408 (1957)." />
<meta name="citation_reference" content="McClelland, G. A. H. A worldwide survey of variation in scale pattern of the abdominal tergum of Aedes aegypti (L.)(Diptera: Culicidae). Transactions of the Royal Entomological Society of London 126, 239–259 (1974)." />
<meta name="citation_reference" content="Sylla, M., Bosio, C., Urdaneta-Marquez, L., Ndiaye, M. &amp; Iv, W. C. B. Gene Flow, Subspecies Composition, and Dengue Virus-2 Susceptibility among Aedes aegypti Collections in Senegal. PLOS Neglected Tropical Diseases 3, e408 (2009)." />
<meta name="citation_reference" content="Gloria-Soria, A. et al. Global genetic diversity of Aedes aegypti. Molecular Ecology 25, 5377–5395 (2016)." />
<meta name="citation_reference" content="Alexander, D. H., Novembre, J. &amp; Lange, K. Fast model-based estimation of ancestry in unrelated individuals. Genome research 19, 1655–1664 (2009)." />
<meta name="citation_reference" content="Yi, X. et al. Sequencing of 50 human exomes reveals adaptation to high altitude. Science 329, 75–78 (2010)." />
<meta name="citation_reference" content="Martin, S. H., Davey, J. W. &amp; Jiggins, C. D. Evaluating the use of ABBA–BABA statistics to locate introgressed loci. Molecular biology and evolution 32, 244–257 (2014)." />
<meta name="citation_reference" content="Cruickshank, T. E. &amp; Hahn, M. W. Reanalysis suggests that genomic islands of speciation are due to reduced diversity, not reduced gene flow. Molecular ecology 23, 3133–3157 (2014)." />
<meta name="citation_reference" content="Luu, K., Bazin, E. &amp; Blum, M. G. pcadapt: an R package to perform genome scans for selection based on principal component analysis. Molecular ecology resources 17, 67–77 (2017)." />
<meta name="citation_reference" content="Gerland, P. et al. World population stabilization unlikely this century. Science 346, 234–237 (2014)." />
<meta name="citation_reference" content="Taylor, K. E., Stouffer, R. J. &amp; Meehl, G. A. An Overview of CMIP5 and the Experiment Design. Bull. Amer. Meteor. Soc. 93, 485–498 (2011)." />
<meta name="citation_reference" content="Futuyma, D. J. &amp; Moreno, G. The Evolution of Ecological Specialization. Annual Review of Ecology and Systematics 19, 207–233 (1988)." />
<meta name="citation_reference" content="Machado-Allison, C. E. &amp; Craig, G. B. Geographic Variation in Resistance to Desiccation in Aedes aegypti and A. atropalpus (Diptera: Culicidae). Ann Entomol Soc Am 65, 542–547 (1972)." />
<meta name="citation_reference" content="Petersen, J. Behavioral Differences in Two Sub-species of Aedes Aegypti (L.)(diptera, Culicidae) in East Africa. (1977)." />
<meta name="citation_reference" content="Dao, A. et al. Signatures of aestivation and migration in Sahelian malaria mosquito populations. Nature 516, 387–390 (2014)." />
<meta name="citation_reference" content="Tripet, F. et al. Competitive Reduction by Satyrization? Evidence for Interspecific Mating in Nature and Asymmetric Reproductive Competition between Invasive Mosquito Vectors. The American Journal of Tropical Medicine and Hygiene 85, 265–270 (2011)." />
<meta name="citation_reference" content="Gloria-Soria, A., Soghigian, J., Kellner, D. &amp; Powell, J. R. Genetic diversity of laboratory strains and implications for research: The case of Aedes aegypti. PLOS Neglected Tropical Diseases 13, e0007930 (2019)." />
<meta name="citation_reference" content="Team, R. C. R: A language and environment for statistical computing. (2013)." />
<meta name="citation_reference" content="Brooks, M. E. et al. glmmTMB balances speed and flexibility among packages for zero-inflated generalized linear mixed modeling. The R journal 9, 378–400 (2017)." />
<meta name="citation_reference" content="Lenth, R., Singmann, H. &amp; Love, J. Emmeans: Estimated marginal means, aka least-squares means. R package version 1, (2018)." />
<meta name="citation_reference" content="Turlach, B. A., Murray, K. &amp; Turlach, M. B. A. Package ‘MonoPoly’. (2019)." />
<meta name="citation_reference" content="Matthews, B. J. et al. Improved reference genome of Aedes aegypti informs arbovirus vector control. Nature 563, 501 (2018)." />
<meta name="citation_reference" content="Danecek, P. et al. The variant call format and VCFtools. Bioinformatics 27, 2156–2158 (2011)." />
<meta name="citation_reference" content="Li, H. statistical framework for SNP calling, mutation discovery, association mapping and population genetical parameter estimation from sequencing data. Bioinformatics 27, 2987–2993 (2011)." />
<meta name="citation_reference" content="http://broadinstitute.github.io/picard." />
<meta name="citation_reference" content="McKenna, A. et al. The Genome Analysis Toolkit: a MapReduce framework for analyzing next-generation DNA sequencing data. Genome research 20, 1297–1303 (2010)." />
<meta name="citation_reference" content="Korneliussen, T. S., Albrechtsen, A. &amp; Nielsen, R. ANGSD: analysis of next generation sequencing data. BMC bioinformatics 15, 356 (2014)." />
<meta name="citation_reference" content="Purcell, S. et al. PLINK: a tool set for whole-genome association and population-based linkage analyses. The American journal of human genetics 81, 559–575 (2007)." />
<meta name="citation_reference" content="Malinsky, M. Dsuite-fast D-statistics and related admixture evidence from VCF files. BioRxiv 634477 (2019)." />
<meta name="citation_reference" content="Fumagalli, M., Vieira, F. G., Linderoth, T. &amp; Nielsen, R. ngsTools: methods for population genetics analyses from next-generation sequencing data. Bioinformatics 30, 1486–1487 (2014)." />
<meta name="citation_reference" content="United Nations, Department of Economic and Social Affairs, Population Division (2019). World Population Prospects 2019, Online Edition." />
<meta name="citation_reference" content="United Nations, Department of Economic and Social Affairs, Population Division. World Population Prospects 2019: Highlights (ST/ESA/SER. A/423). (2019)." />
<meta name="twitter:title" content="Climate and urbanization drive mosquito preference for humans" />
<meta name="twitter:site" content="@biorxivpreprint" />
<meta name="twitter:card" content="summary" />
<meta name="twitter:image" content="https://www.biorxiv.org/sites/default/files/images/biorxiv_logo_homepage7-5-small.png" />
<meta name="twitter:description" content="The majority of mosquito-borne illness is spread by a few mosquito species that have evolved to specialize in biting humans, yet the precise causes of this behavioral shift are poorly understood. We address this gap in the arboviral vector Aedes aegypti . We first characterize the behaviour of mosquitoes from 27 sites scattered across the species’ ancestral range in sub-Saharan Africa, revealing previously unrecognized diversity in female preference for human versus animal odor. We then use modelling to show that this diversity can be almost fully predicted by two ecological factors – dry season intensity and human population density. Finally we integrate this information with whole genome sequence data from 345 individual mosquitoes to identify a single underlying ancestry component linked to human preference, with genetic changes concentrated in a few key chromosomal regions. Our findings strongly suggest that human-biting in this important disease vector originally evolved as a by-product of breeding in human-stored water in areas where doing so provided the only means to survive the long, hot dry season. Our model also predicts that changes in human population density are likely to drive future mosquito evolution. Rapid urbanization may drive a shift to human-biting in many cities across Africa by 2050." />
<meta name="og-title" property="og:title" content="Climate and urbanization drive mosquito preference for humans" />
<meta name="og-url" property="og:url" content="https://www.biorxiv.org/content/10.1101/2020.02.12.939041v1" />
<meta name="og-site-name" property="og:site_name" content="bioRxiv" />
<meta name="og-description" property="og:description" content="The majority of mosquito-borne illness is spread by a few mosquito species that have evolved to specialize in biting humans, yet the precise causes of this behavioral shift are poorly understood. We address this gap in the arboviral vector Aedes aegypti . We first characterize the behaviour of mosquitoes from 27 sites scattered across the species’ ancestral range in sub-Saharan Africa, revealing previously unrecognized diversity in female preference for human versus animal odor. We then use modelling to show that this diversity can be almost fully predicted by two ecological factors – dry season intensity and human population density. Finally we integrate this information with whole genome sequence data from 345 individual mosquitoes to identify a single underlying ancestry component linked to human preference, with genetic changes concentrated in a few key chromosomal regions. Our findings strongly suggest that human-biting in this important disease vector originally evolved as a by-product of breeding in human-stored water in areas where doing so provided the only means to survive the long, hot dry season. Our model also predicts that changes in human population density are likely to drive future mosquito evolution. Rapid urbanization may drive a shift to human-biting in many cities across Africa by 2050." />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.biorxiv.org/sites/default/files/images/biorxiv_logo_homepage7-5-small.png" />
<meta name="citation_date" content="2020-02-13" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/10.1101/2020.02.12.939041v1.full.pdf" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/10.1101/2020.02.12.939041v1.full.txt" />
<link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/10.1101/2020.02.12.939041v1.ppt" />
<meta name="description" content="bioRxiv - the preprint server for biology, operated by Cold Spring Harbor Laboratory, a research and educational institution" />
<meta name="generator" content="Drupal 7 (http://drupal.org)" />
<link rel="canonical" href="https://www.biorxiv.org/content/10.1101/2020.02.12.939041v1" />
<link rel="shortlink" href="https://www.biorxiv.org/node/1149559" />
    <title>Climate and urbanization drive mosquito preference for humans | bioRxiv</title>  
    <link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__jMRAK66KMC1e4TQlwUNn3KiWVDC5AjueUAYEm1xBY_U__KSUjdT4jdcJ4qJz6fKY1K9WkYh1a5EcaWZxt_-zbTis__QJacVmveFyLIBSPkp21xL0ZvuDW9DMBrD2HB1P4Ry5I.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q5p1rr" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__icrP-Y0qw2NP73rJxxkUtrJta0z3VLAABtNjfSdEghA__rlN3rx0wOoswGNHZMeZM96bNn-3cLWHXTztGDoThNlw__QJacVmveFyLIBSPkp21xL0ZvuDW9DMBrD2HB1P4Ry5I.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__q48SDxVDWut_cH6dTPyQe7HjADuSLLmJVFztuiJEFrU__DJqddQGjm5dXCHsaiwNriGytKN5eevIQqyjdq65TZtE__QJacVmveFyLIBSPkp21xL0ZvuDW9DMBrD2HB1P4Ry5I.css" media="all" />
<style type="text/css" media="all">
/* <![CDATA[ */
#sliding-popup.sliding-popup-bottom,#sliding-popup.sliding-popup-bottom .eu-cookie-withdraw-banner,.eu-cookie-withdraw-tab{background:gray}#sliding-popup.sliding-popup-bottom.eu-cookie-withdraw-wrapper{background:transparent}#sliding-popup .popup-content #popup-text h1,#sliding-popup .popup-content #popup-text h2,#sliding-popup .popup-content #popup-text h3,#sliding-popup .popup-content #popup-text p,.eu-cookie-compliance-secondary-button,.eu-cookie-withdraw-tab{color:#fff !important}.eu-cookie-withdraw-tab{border-color:#fff}.eu-cookie-compliance-more-button{color:#fff !important}
/* ]]> */
</style>

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__QJacVmveFyLIBSPkp21xL0ZvuDW9DMBrD2HB1P4Ry5I.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__B6tGdsQSt1vT8d8j-19-oS4FKJe83RrSNBkjPs9jfos__6VkE5WcQmmQpSVTQx-xS2-zn_nlwFlDm-EN1HLJNjrA__QJacVmveFyLIBSPkp21xL0ZvuDW9DMBrD2HB1P4Ry5I.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__I6pBwZrmO51afTZVOjN83FdGH1QRb9t15lSLso0JVV4__F3sVKlGyHKCaAPI_WdiHD7l_-05B6b9bLdUwzi1me90__QJacVmveFyLIBSPkp21xL0ZvuDW9DMBrD2HB1P4Ry5I.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__QJacVmveFyLIBSPkp21xL0ZvuDW9DMBrD2HB1P4Ry5I.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__QJacVmveFyLIBSPkp21xL0ZvuDW9DMBrD2HB1P4Ry5I.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__CRdjIL0HKDL2T9slYvkTSTLypBBPjyoDhwBezBuDy7c__cHebPmPNdg2jyPmpyQpZWssJQ4cPrqfaabp5Or9KlkU__QJacVmveFyLIBSPkp21xL0ZvuDW9DMBrD2HB1P4Ry5I.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://d33xdlntwy0kbs.cloudfront.net/cshl_custom.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/all/libraries/modernizr/modernizr.min.js?q5p1rr"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__yOhrJUBmMpN-jsuQlJYmS8ovkQU6I6KInr295HrSTb0__c--zHhbuOWmkdWkJYXiNWDmkpDRPesmbBjGlQcyqBOY__QJacVmveFyLIBSPkp21xL0ZvuDW9DMBrD2HB1P4Ry5I.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var _prum=[['id', '52e95df3abe53d2e33000000'], ['mark', 'firstbyte',
      (new Date()).getTime()]]; (function() {
      var s=document.getElementsByTagName('script')[0],
      p=document.createElement('script');
      p.async='async'; p.src='//rum-static.pingdom.net/prum.min.js';s.parentNode.insertBefore(p,s);})();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js___KyvuEvfmewWxdOLBXTYsSUo8Pfm_DijV6fWtGvSL_k__HSKkTgIr7nDcdvQYewipf6v4ak6ZLUOQkwPj44wKouA__QJacVmveFyLIBSPkp21xL0ZvuDW9DMBrD2HB1P4Ry5I.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__3uCzzYE5jr6ly9nvsM8PRBRkhh4p0NWCMDja4EKBy4M__dmGGVYcB8LhnbR2iAmVW0ZyXI6QuFVYFRORa5XxglXs__QJacVmveFyLIBSPkp21xL0ZvuDW9DMBrD2HB1P4Ry5I.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__0zdUME0GLauYYyEch2tGPydIw-IG7Uizip0Qb4wUsig__3RgszrCgk4bOnTzo4vaQ-hrWzaG6oD19l-R9CQ1YQ-o__QJacVmveFyLIBSPkp21xL0ZvuDW9DMBrD2HB1P4Ry5I.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__W6wAjopaALytcBam7oT3WhEzllkWaB9R-8hazHyvShM__rK5YBIqBy9qt0di0Sn4DriBx9TJupdmv6uGmLJyZvDs__QJacVmveFyLIBSPkp21xL0ZvuDW9DMBrD2HB1P4Ry5I.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-45638026-1", {"cookieDomain":"auto"});ga("set", "page", location.pathname + location.search + location.hash);ga("send", "pageview");ga('create', 'UA-189672-38', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__1PtAOZ8x7l6QAi8D7X6Kj_Y7HcUOFzvWRvvYHUcCT98__U8IB92u_CnnGwJ-X_xaUa9lzsC8OzUfN0WkK2JQfirg__QJacVmveFyLIBSPkp21xL0ZvuDW9DMBrD2HB1P4Ry5I.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings,{"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"4Gj2js9ZhjWu85XydfSXcftyNCqp6NjqVm2hFRc3OTM","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"modules\/comment\/comment.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"modules\/forum\/forum.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"sites\/default\/modules\/biorxiv\/css\/biorxiv.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/flexible\/flexible.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_collection\/css\/highwire_subject_collection.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire.highwire-list-expand.css":1,"public:\/\/ctools\/css\/bab59090150caf0c64e27afa1b36eef1.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/contrib\/eu_cookie_compliance\/css\/eu_cookie_compliance.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"0":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_biorxiv_styles\/css\/biorxiv-styles.css.less":1,"https:\/\/d33xdlntwy0kbs.cloudfront.net\/cshl_custom.css":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_figures.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_tables.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"http:\/\/d33xdlntwy0kbs.cloudfront.net\/cshl_custom.js":1,"sites\/all\/modules\/contrib\/eu_cookie_compliance\/js\/eu_cookie_compliance.js":1,"sites\/default\/modules\/jnl_biorxiv_styles\/js\/adv-search-collection-toggle.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/eu_cookie_compliance\/js\/jquery.cookie-1.4.1.min.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/default\/modules\/jnl_biorxiv_styles\/js\/biorxiv-scripts.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_nps_survey\/js\/nps_survey.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"sites\/default\/modules\/contrib\/disqus\/disqus.js":1,"misc\/textarea.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/contrib\/service_links\/js\/twitter_button.js":1,"sites\/all\/modules\/contrib\/service_links\/js\/facebook_like.js":1,"sites\/all\/modules\/contrib\/service_links\/js\/google_plus_one.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.highwire-list-expand.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"1149559","apath":"\/biorxiv\/early\/2020\/02\/13\/2020.02.12.939041.atom","pisa":"biorxiv;2020.02.12.939041v1","processed":["highwire_math"],"markup":[{"requested":"full-text","variant":"full-text","view":"full","pisa":"biorxiv;2020.02.12.939041v1"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"disqus":{"domain":"biorxivstage","url":"https:\/\/www.biorxiv.org\/content\/10.1101\/2020.02.12.939041v1","title":"Climate and urbanization drive mosquito preference for humans","identifier":"node\/1149559"},"panels_ajax_pane":{"new-41":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022n6T6fmWPsAyRrdqwzCxlbdWC9Z6hb0hEe6yslMo1no8T5wXkS7B4o7zS2DIYuFPoIXj1hgEbrrDItWObteXdN+GyAHRuSNnF\\\\\\\/iNoK9nGR7aXGeEytBOg+EVzDGbnAYB2vH+Jw6rH+EZpgwXVUjQT4gPlVPV5ELsRGyboFeQKfGW9ATfhyFICKJB5JCtkS7o\\\\\\\/oQaL\\\\\\\/JLAQkLiUrmijo6qdIl+oU12OleVJ93SbL6CcrAm+OmzhdNMidTjZN1T\\\\\\\/oJhIFAm61nGDq2DyaDd\\\\\\\/OJPUZbZ2qF3zPshvF7Q0\\\\\\\/xwgWXFuO8E\\\\\\\/xTCcjO37B7BoyKStVOkRiSsRiJA5xHjR06X20sg6EwzzZgy4lxm2Czr\\\\\\\/mCTFdMXkim2boQaxc4eRBL+EEKCD1NDxG2gKG68LV35VfmUHtRhmtSTqSRp7kE5a4i5oeeJOrStpM+fK4QbqOOeCsgXviYh+dpmzwxZBZiiCRc9dhzkKRg2N8jv8CA3fc9XR4SuI6WIaLi7uuS\\\\\\\/hNfp0i5xxHCdx\\\\\\\/hGcnbzJSexCd9xMdrwSlEoUR2lJnvR8d0biyn\\\\\\\/eUnfr5v9XD4RBExoybN7SgEY5CmWtCUrgN6590C0GB+TcY92aow6kzcBUwD+wpNTamo50SLB2N6HXsZGEtx7eB\\\\\\\/llvceCk\\\\\\\/NsJzrFaQcaj1GDa2XkfLCgXeB2SZKFruLaebAL8ZXMshnWNRuEiq1UkKC0EajTeg8+M54WGYwckW1I3DaxiBBVnW8N1cpyMHo2HinVll6dMNEyAKooKnD3VBHWn2CMNBOUNB2uFNu157exk98HZqOwRnrjiJEQz6XOp0HpVcTWooDZezO7jrjXXgkoivjzZ4z0kEU4KMAc0EWcNerqUWGAt7XpnTbTa43RLLFLaawm5OtPORG1Erh5AYqVV\\\\\\\/cEoFKbDiWgP3tPHQv1eL9j3ouDYuKX53VMGwSs1rUJjjSlYnc6BclKDV\\\\\\\/g70Ke45+9f7A1qnqGuObTriWmnlQ9lrLh74yiShuPI+GRw38XP0fr10utJ5fnyNFj2kdH20Y\\\\\\\/Y7eJLEgWYTNk7+d7YhaMfsKFzIGZMWlHdYGMg28Dhk0XDH5SzZPkgdxLZGG+hDhIiKA5OyqRam5PMQ185iTiE7DKj9xG1NSRbwWBSK57bGBokoxDo9MU78qs5xSAXSHuU2UkTVvCi\\\\\\\/xJTUTwI+qzDxJizISQlinBRbpmTU6wh5NirMhxEjC5\\\\\\\/fixTXPvuQrCj6FI0w8C5fQ94yOo\\\\\\\/QXoYt5cAfVeLvQX8+P0WnvFgsNn8xqcyCgpQWFZdfuiTieqFUW\\\\\\\/K34CpoiytUU6dwtrU\\\\\\\/AMeQ21DbyquJZ85\\\\\\\/RD2SkHNNDRbXb\\\u0022,\\\u0022iv\\\u0022:\\\u0022Q2iC5uNYh+nz7SkhTZrDFw==\\\u0022,\\\u0022salt\\\u0022:\\\u0022f9d8845712f72c9fee5db7d9ff111eab\\\u0022}\u0022,\u0022hmac\u0022:\u0022d58a0093f42659a25e29786805e111911698f2e5e4b1cc72c4531c2a984f117d\u0022}"},"urlIsAjaxTrusted":{"\/content\/10.1101\/2020.02.12.939041v1.full":true},"ws_fl":{"width":100,"height":21},"ws_gpo":{"size":"","annotation":"","lang":"","callback":"","width":300},"color":{"logo":"https:\/\/www.biorxiv.org\/sites\/default\/files\/bioRxiv_article.jpg"},"highwire_list_expand":{"is_collapsed":"1"},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"eu_cookie_compliance":{"popup_enabled":1,"popup_agreed_enabled":0,"popup_hide_agreed":0,"popup_clicking_confirmation":1,"popup_scrolling_confirmation":false,"popup_html_info":"\u003Cdiv\u003E\n  \u003Cdiv class =\u0022popup-content info\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n      \u003Cp\u003EWe use cookies on this site to enhance your user experience. By clicking any link on this page you are giving your consent for us to set cookies.\u003C\/p\u003E\n    \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022agree-button eu-cookie-compliance-default-button\u0022\u003EContinue\u003C\/button\u003E\n                    \u003Cbutton type=\u0022button\u0022 class=\u0022find-more-button eu-cookie-compliance-more-button\u0022\u003EFind out more\u003C\/button\u003E\n          \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E","use_mobile_message":false,"mobile_popup_html_info":"\u003Cdiv\u003E\n  \u003Cdiv class =\u0022popup-content info\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n          \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022agree-button eu-cookie-compliance-default-button\u0022\u003EContinue\u003C\/button\u003E\n                    \u003Cbutton type=\u0022button\u0022 class=\u0022find-more-button eu-cookie-compliance-more-button\u0022\u003EFind out more\u003C\/button\u003E\n          \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E\n","mobile_breakpoint":"768","popup_html_agreed":"\u003Cdiv\u003E\n  \u003Cdiv class=\u0022popup-content agreed\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n      \u003Ch2\u003EThank you for accepting cookies\u003C\/h2\u003E\u003Cp\u003EYou can now hide this message or find out more about cookies.\u003C\/p\u003E    \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022hide-popup-button eu-cookie-compliance-hide-button\u0022\u003EHide\u003C\/button\u003E\n              \u003Cbutton type=\u0022button\u0022 class=\u0022find-more-button eu-cookie-compliance-more-button-thank-you\u0022 \u003EMore info\u003C\/button\u003E\n          \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E","popup_use_bare_css":false,"popup_height":"auto","popup_width":"100%","popup_delay":1000,"popup_link":"\/help\/cookie-policy","popup_link_new_window":1,"popup_position":null,"popup_language":"en","store_consent":false,"better_support_for_screen_readers":0,"reload_page":0,"domain":"","popup_eu_only_js":0,"cookie_lifetime":"365","cookie_session":false,"disagree_do_not_show_popup":0,"method":"default","whitelisted_cookies":"","withdraw_markup":"\u003Cbutton type=\u0022button\u0022 class=\u0022eu-cookie-withdraw-tab\u0022\u003EPrivacy settings\u003C\/button\u003E\n\u003Cdiv class=\u0022eu-cookie-withdraw-banner\u0022\u003E\n  \u003Cdiv class=\u0022popup-content info\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n      \u003Cp\u003E\u0026lt;h2\u0026gt;We use cookies on this site to enhance your user experience\u0026lt;\/h2\u0026gt;\u0026lt;p\u0026gt;You have given your consent for us to set cookies.\u0026lt;\/p\u0026gt;\u003C\/p\u003E\n    \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022eu-cookie-withdraw-button\u0022\u003EWithdraw consent\u003C\/button\u003E\n    \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E\n","withdraw_enabled":false},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1,"trackUrlFragments":1},"jnl_biorxiv_styles":{"defaultJCode":"biorxiv"},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-1149559 node-type-highwire-article context-content hw-default-jcode-biorxiv hw-article-type-article hw-article-category-new-results">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-M677548" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-M677548');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
        <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
    
  <div id="zone-branding" class="zone zone-branding clearfix print-display-block container-30">
    <div class="grid-15 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="bioRxiv" src="https://www.biorxiv.org/sites/default/files/bioRxiv_article.jpg" /></a>      </div>
                </div>
          </div>
</div><div class="grid-11 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-system block-menu block-main-menu block-system-main-menu odd block-without-title" id="block-system-main-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/" title="" class="" data-icon-position="" data-hide-link-title="0">Home</a></li>
<li class="leaf" role="menuitem"><a href="/about-biorxiv" class="" data-icon-position="" data-hide-link-title="0">About</a></li>
<li class="leaf" role="menuitem"><a href="/submit-a-manuscript" class="" data-icon-position="" data-hide-link-title="0">Submit</a></li>
<li class="last leaf" role="menuitem"><a href="/content/alertsrss" title="" class="" data-icon-position="" data-hide-link-title="0">ALERTS / RSS</a></li>
</ul></nav>    </div>
  </div>
</div><div class="block block-panels-mini block-biorxiv-search-box block-panels-mini-biorxiv-search-box even block-without-title" id="block-panels-mini-biorxiv-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-biorxiv_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/10.1101/2020.02.12.939041v1.full" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--2" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-vc63Ct693DL8Y3ZEsdZ71UI8hsGRzgYXdZRbn1CQPao" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2 advanced-search-link" >
  
      
  
  <div class="pane-content">
    <a href="/search">Advanced Search</a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix container-30">
  	      </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="1149559" id="node1149559" data-pisa="biorxiv;2020.02.12.939041v1" data-pisa-master="biorxiv;2020.02.12.939041" data-apath="/biorxiv/early/2020/02/13/2020.02.12.939041.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div  class="highwire-cite highwire-cite-highwire-article highwire-citation-biorxiv-article-top clearfix has-author-tooltip" >

      <span class="biorxiv-article-type">
        New Results    </span>
  
        <h1 class="highwire-cite-title" id="page-title">Climate and urbanization drive mosquito preference for humans</h1>  
      <div  class="highwire-cite-authors" ><span  class="highwire-citation-authors"><span class="highwire-citation-author first hw-author-orcid-logo-wrapper" data-delta="0"><a href="http://orcid.org/0000-0001-7129-4753" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a><span class="nlm-given-names">Noah H.</span> <span class="nlm-surname">Rose</span></span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="1"><a href="http://orcid.org/0000-0003-2893-9725" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a><span class="nlm-given-names">Massamba</span> <span class="nlm-surname">Sylla</span></span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="2"><a href="http://orcid.org/0000-0002-6652-4240" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a><span class="nlm-given-names">Athanase</span> <span class="nlm-surname">Badolo</span></span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="3"><a href="http://orcid.org/0000-0002-8507-2024" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a><span class="nlm-given-names">Joel</span> <span class="nlm-surname">Lutomiah</span></span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="4"><a href="http://orcid.org/0000-0003-4726-580X" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a><span class="nlm-given-names">Diego</span> <span class="nlm-surname">Ayala</span></span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="5"><a href="http://orcid.org/0000-0002-2530-0865" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a><span class="nlm-given-names">Ogechukwu B.</span> <span class="nlm-surname">Aribodor</span></span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="6"><a href="http://orcid.org/0000-0002-6609-8173" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a><span class="nlm-given-names">Nnenna</span> <span class="nlm-surname">Ibe</span></span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="7"><a href="http://orcid.org/0000-0002-3972-0860" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a><span class="nlm-given-names">Jewelna</span> <span class="nlm-surname">Akorli</span></span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="8"><a href="http://orcid.org/0000-0003-0548-8466" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a><span class="nlm-given-names">Sampson</span> <span class="nlm-surname">Otoo</span></span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="9"><a href="http://orcid.org/0000-0001-7254-6537" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a><span class="nlm-given-names">John-Paul</span> <span class="nlm-surname">Mutebi</span></span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="10"><a href="http://orcid.org/0000-0002-4783-8367" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a><span class="nlm-given-names">Alexis L.</span> <span class="nlm-surname">Kriete</span></span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="11"><a href="http://orcid.org/0000-0003-2960-6500" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a><span class="nlm-given-names">Eliza G.</span> <span class="nlm-surname">Ewing</span></span>, <span class="highwire-citation-author" data-delta="12"><span class="nlm-given-names">Rosemary</span> <span class="nlm-surname">Sang</span></span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="13"><a href="http://orcid.org/0000-0002-5401-3988" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a><span class="nlm-given-names">Andrea</span> <span class="nlm-surname">Gloria-Soria</span></span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="14"><a href="http://orcid.org/0000-0002-7651-5153" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a><span class="nlm-given-names">Jeffrey R.</span> <span class="nlm-surname">Powell</span></span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="15"><a href="http://orcid.org/0000-0002-2661-8103" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a><span class="nlm-given-names">Rachel E.</span> <span class="nlm-surname">Baker</span></span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="16"><a href="http://orcid.org/0000-0002-6444-4159" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a><span class="nlm-given-names">Bradley J.</span> <span class="nlm-surname">White</span></span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="17"><a href="http://orcid.org/0000-0002-6745-9145" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a><span class="nlm-given-names">Jacob E.</span> <span class="nlm-surname">Crawford</span></span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="18"><a href="http://orcid.org/0000-0002-8898-1768" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a><span class="nlm-given-names">Carolyn S.</span> <span class="nlm-surname">McBride</span></span></span></div>
  
      <div  class="highwire-cite-metadata" ><span  class="highwire-cite-metadata-doi highwire-cite-metadata"><span class="label">doi:</span> https://doi.org/10.1101/2020.02.12.939041 </span></div>
  
  
  </div>
<div id="hw-article-author-popups-node1149559" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Noah H. Rose </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-institution'>Department of Ecology &#38; Evolutionary Biology, Princeton University</span>, Princeton, NJ 08544, <span class='nlm-country'>USA</span></div><div class='author-affiliation'><span class='nlm-institution'>Princeton Neuroscience Institute, Princeton University</span>, Princeton, NJ 08544, <span class='nlm-country'>USA</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Noah%2BH.%2BRose%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Rose%20NH&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ANoah%2BH.%2BRose%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0001-7129-4753" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Noah H. Rose</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:noahr@princeton.edu" class="" data-icon-position="" data-hide-link-title="0">noahr@princeton.edu</a>
<a href="mailto:csm7@princeton.edu" class="" data-icon-position="" data-hide-link-title="0">csm7@princeton.edu</a></span></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Massamba Sylla </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-institution'>Unité d’Entomologie, de Bactériologie, de Virologie, Département de Biologie Animale, Faculté des Sciences et Techniques, Université Cheikh Anta DIOP</span>, BP 5005 Dakar, <span class='nlm-country'>Senegal</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Massamba%2BSylla%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Sylla%20M&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AMassamba%2BSylla%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0003-2893-9725" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Massamba Sylla</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Athanase Badolo </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-institution'>Laboratory of Fundamental and Applied Entomology, Université Joseph Ki-Zerbo</span>, 03 BP 7021 Ouagadougou, <span class='nlm-country'>Burkina Faso</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Athanase%2BBadolo%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Badolo%20A&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AAthanase%2BBadolo%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-6652-4240" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Athanase Badolo</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Joel Lutomiah </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-institution'>Arbovirus/Viral Hemorrhagic Fevers Laboratory, Center for Virus Research, Kenya Medical Research Institute (KEMRI)</span>, Nairobi, <span class='nlm-country'>Kenya</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Joel%2BLutomiah%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Lutomiah%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AJoel%2BLutomiah%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-8507-2024" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Joel Lutomiah</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Diego Ayala </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-institution'>UMR MIVEGEC, IRD, CNRS, Univ. Montpellier</span>, 911 avenue Agropolis, BP 64501, 34394 Montpellier, <span class='nlm-country'>France</span></div><div class='author-affiliation'><span class='nlm-institution'>Le Centre International de Recherches Médicales de Franceville</span>, BP 769, Franceville, <span class='nlm-country'>Gabon</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Diego%2BAyala%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Ayala%20D&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ADiego%2BAyala%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0003-4726-580X" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Diego Ayala</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Ogechukwu B. Aribodor </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-institution'>Department of Zoology, Nnamdi Azikiwe University</span>, Awka, Anambra State, <span class='nlm-country'>Nigeria</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Ogechukwu%2BB.%2BAribodor%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Aribodor%20OB&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AOgechukwu%2BB.%2BAribodor%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-2530-0865" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Ogechukwu B. Aribodor</a></li></ul></div><div class="author-tooltip-6"><div class="author-tooltip-name">Nnenna Ibe </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-institution'>Department of Ecology &#38; Evolutionary Biology, Princeton University</span>, Princeton, NJ 08544, <span class='nlm-country'>USA</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Nnenna%2BIbe%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Ibe%20N&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ANnenna%2BIbe%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-6609-8173" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Nnenna Ibe</a></li></ul></div><div class="author-tooltip-7"><div class="author-tooltip-name">Jewelna Akorli </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-institution'>Department of Parasitology, Noguchi Memorial Institute for Medical Research, University of Ghana</span>, Accra, <span class='nlm-country'>Ghana</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jewelna%2BAkorli%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Akorli%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AJewelna%2BAkorli%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-3972-0860" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Jewelna Akorli</a></li></ul></div><div class="author-tooltip-8"><div class="author-tooltip-name">Sampson Otoo </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-institution'>Department of Parasitology, Noguchi Memorial Institute for Medical Research, University of Ghana</span>, Accra, <span class='nlm-country'>Ghana</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Sampson%2BOtoo%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Otoo%20S&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ASampson%2BOtoo%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0003-0548-8466" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Sampson Otoo</a></li></ul></div><div class="author-tooltip-9"><div class="author-tooltip-name">John-Paul Mutebi </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-institution'>Centers for Disease Control and Prevention</span>, Fort Collins, CO 80521, <span class='nlm-country'>USA</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=John-Paul%2BMutebi%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Mutebi%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AJohn-Paul%2BMutebi%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0001-7254-6537" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for John-Paul Mutebi</a></li></ul></div><div class="author-tooltip-10"><div class="author-tooltip-name">Alexis L. Kriete </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-institution'>Department of Ecology &#38; Evolutionary Biology, Princeton University</span>, Princeton, NJ 08544, <span class='nlm-country'>USA</span></div><div class='author-affiliation'><span class='nlm-institution'>Princeton Neuroscience Institute, Princeton University</span>, Princeton, NJ 08544, <span class='nlm-country'>USA</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Alexis%2BL.%2BKriete%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Kriete%20AL&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AAlexis%2BL.%2BKriete%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-4783-8367" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Alexis L. Kriete</a></li></ul></div><div class="author-tooltip-11"><div class="author-tooltip-name">Eliza G. Ewing </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-institution'>Department of Ecology &#38; Evolutionary Biology, Princeton University</span>, Princeton, NJ 08544, <span class='nlm-country'>USA</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Eliza%2BG.%2BEwing%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Ewing%20EG&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AEliza%2BG.%2BEwing%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0003-2960-6500" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Eliza G. Ewing</a></li></ul></div><div class="author-tooltip-12"><div class="author-tooltip-name">Rosemary Sang </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-institution'>Arbovirus/Viral Hemorrhagic Fevers Laboratory, Center for Virus Research, Kenya Medical Research Institute (KEMRI)</span>, Nairobi, <span class='nlm-country'>Kenya</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Rosemary%2BSang%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Sang%20R&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ARosemary%2BSang%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-13"><div class="author-tooltip-name">Andrea Gloria-Soria </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-institution'>Department of Ecology and Evolutionary Biology, Yale University</span>, 21 Sachem Street, New Haven, CT 06511, <span class='nlm-country'>USA</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Andrea%2BGloria-Soria%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Gloria-Soria%20A&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AAndrea%2BGloria-Soria%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-5401-3988" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Andrea Gloria-Soria</a></li></ul></div><div class="author-tooltip-14"><div class="author-tooltip-name">Jeffrey R. Powell </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-institution'>Department of Ecology and Evolutionary Biology, Yale University</span>, 21 Sachem Street, New Haven, CT 06511, <span class='nlm-country'>USA</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jeffrey%2BR.%2BPowell%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Powell%20JR&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AJeffrey%2BR.%2BPowell%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-7651-5153" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Jeffrey R. Powell</a></li></ul></div><div class="author-tooltip-15"><div class="author-tooltip-name">Rachel E. Baker </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-institution'>Princeton Environmental Institute, Princeton University</span>, Princeton, NJ 08544, <span class='nlm-country'>USA</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Rachel%2BE.%2BBaker%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Baker%20RE&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ARachel%2BE.%2BBaker%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-2661-8103" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Rachel E. Baker</a></li></ul></div><div class="author-tooltip-16"><div class="author-tooltip-name">Bradley J. White </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-institution'>Verily Life Sciences</span>, 259 East Grand Avenue, South San Francisco, CA 94080, <span class='nlm-country'>USA</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Bradley%2BJ.%2BWhite%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=White%20BJ&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ABradley%2BJ.%2BWhite%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-6444-4159" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Bradley J. White</a></li></ul></div><div class="author-tooltip-17"><div class="author-tooltip-name">Jacob E. Crawford </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-institution'>Verily Life Sciences</span>, 259 East Grand Avenue, South San Francisco, CA 94080, <span class='nlm-country'>USA</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jacob%2BE.%2BCrawford%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Crawford%20JE&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AJacob%2BE.%2BCrawford%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-6745-9145" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Jacob E. Crawford</a></li></ul></div><div class="author-tooltip-18"><div class="author-tooltip-name">Carolyn S. McBride </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-institution'>Department of Ecology &#38; Evolutionary Biology, Princeton University</span>, Princeton, NJ 08544, <span class='nlm-country'>USA</span></div><div class='author-affiliation'><span class='nlm-institution'>Princeton Neuroscience Institute, Princeton University</span>, Princeton, NJ 08544, <span class='nlm-country'>USA</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Carolyn%2BS.%2BMcBride%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=McBride%20CS&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ACarolyn%2BS.%2BMcBride%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0002-8898-1768" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Carolyn S. McBride</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:noahr@princeton.edu" class="" data-icon-position="" data-hide-link-title="0">noahr@princeton.edu</a>
<a href="mailto:csm7@princeton.edu" class="" data-icon-position="" data-hide-link-title="0">csm7@princeton.edu</a></span></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/10.1101/2020.02.12.939041v1" class="panels-ajax-tab-tab" data-panel-name="biorxiv_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:1149559" data-trigger="" data-url-enabled="1">Abstract</a><a href="/panels_ajax_tab/biorxiv_tab_art/node:1149559/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/10.1101/2020.02.12.939041v1.full-text" class="panels-ajax-tab-tab" data-panel-name="article_tab_full_text" data-target-id="highwire_article_tabs" data-entity-context="node:1149559" data-trigger="full-text" data-url-enabled="1">Full Text</a><a href="/panels_ajax_tab/article_tab_full_text/node:1149559/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/10.1101/2020.02.12.939041v1.article-info" class="panels-ajax-tab-tab" data-panel-name="biorxiv_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:1149559" data-trigger="article-info" data-url-enabled="1">Info/History</a><a href="/panels_ajax_tab/biorxiv_tab_info/node:1149559/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/10.1101/2020.02.12.939041v1.article-metrics" class="panels-ajax-tab-tab" data-panel-name="article_tab_metrics" data-target-id="highwire_article_tabs" data-entity-context="node:1149559" data-trigger="article-metrics" data-url-enabled="1">Metrics</a><a href="/panels_ajax_tab/article_tab_metrics/node:1149559/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/10.1101/2020.02.12.939041v1.external-links" class="panels-ajax-tab-tab" data-panel-name="article_tab_data_code" data-target-id="highwire_article_tabs" data-entity-context="node:1149559" data-trigger="external-links" data-url-enabled="1">Data/Code</a><a href="/panels_ajax_tab/article_tab_data_code/node:1149559/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/10.1101/2020.02.12.939041v1.full.pdf+html" class="panels-ajax-tab-tab" data-panel-name="biorxiv_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:1149559" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i> Preview PDF</a><a href="/panels_ajax_tab/biorxiv_tab_pdf/node:1149559/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="article_tab_full_text" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.biorxiv.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-article_tab_full_text"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-markup" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2 class="">Summary</h2><p id="p-4">The majority of mosquito-borne illness is spread by a few mosquito species that have evolved to specialize in biting humans, yet the precise causes of this behavioral shift are poorly understood. We address this gap in the arboviral vector <em>Aedes aegypti</em>. We first characterize the behaviour of mosquitoes from 27 sites scattered across the species’ ancestral range in sub-Saharan Africa, revealing previously unrecognized diversity in female preference for human <em>versus</em> animal odor. We then use modelling to show that this diversity can be almost fully predicted by two ecological factors – dry season intensity and human population density. Finally we integrate this information with whole genome sequence data from 345 individual mosquitoes to identify a single underlying ancestry component linked to human preference, with genetic changes concentrated in a few key chromosomal regions. Our findings strongly suggest that human-biting in this important disease vector originally evolved as a by-product of breeding in human-stored water in areas where doing so provided the only means to survive the long, hot dry season. Our model also predicts that changes in human population density are likely to drive future mosquito evolution. Rapid urbanization may drive a shift to human-biting in many cities across Africa by 2050.</p></div><div id="sec-1" class="subsection"><p id="p-7">Mosquitoes spread pathogens that make approximately 100 million people sick every year<sup><a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">1</a></sup>. There are roughly 3,500 mosquito species worldwide<sup><a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">2</a></sup>, but most cases of human disease are caused by the bites of just a few taxa that specifically target humans<sup><a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">3</a>,<a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">4</a></sup>. Understanding where and why mosquitoes evolve to specialize in biting humans is therefore critical for controlling and predicting disease spread.</p><p id="p-8"><em>Aedes aegypti</em> provides a key opportunity to investigate this phenomenon. The globally invasive subspecies, <em>Ae. aegypti aegypti</em>, thrives in urban habitats across the American and Asian tropics, where its proclivity for biting humans makes it the primary vector of dengue, Zika, chikungunya, and yellow fever<sup><a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">5</a></sup>. Host-seeking females take up to 95% of their blood meals from humans in nature<sup><a id="xref-ref-3-2" class="xref-bibr" href="#ref-3">3</a></sup>. This human-biting specialist is thought to have evolved from generalist ancestors in Africa approximately 5,000-10,000 years ago, possibly in northern Senegal or Angola<sup><a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">6</a>,<a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">7</a></sup>. However, in at least a few places in East Africa, the contemporary African subspecies <em>Ae. aegypti formosus</em> remains a generalist, biting a wide variety of vertebrate animals<sup><a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">8</a>,<a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">9</a></sup>. Little is known about the host-seeking behavior of <em>Ae. aegypti</em> in other parts of Africa, and no work to date has explicitly examined the ultimate drivers of human-biting in mosquitoes.</p><p id="p-9">To address this gap, we used ovitraps to collect <em>Ae. aegypti</em> eggs from multiple outdoor sites in each of 27 locations across sub-Saharan Africa (<a id="xref-fig-1-1" class="xref-fig" href="#F1">Fig. 1a-c</a>, <a id="xref-table-wrap-1-1" class="xref-table" href="#T1">Extended Data Table 1</a>). The collections spanned a wide range of human population densities, including egg traps placed among assemblages of plastic and concrete in large cities with over 2,000 people per square kilometer, ranging to traps placed among trees and undergrowth in wild areas where mosquitoes rarely encounter human hosts (<a id="xref-fig-1-2" class="xref-fig" href="#F1">Fig. 1a,c</a>, <a id="xref-table-wrap-1-2" class="xref-table" href="#T1">Extended Data Table 1</a>). They also spanned a wide range of climates, from highly seasonal, semi-arid habitats in the northwest to forest ecosystems with year-round rain in Central Africa (<a id="xref-fig-1-3" class="xref-fig" href="#F1">Fig. 1b-c</a>, <a id="xref-table-wrap-1-3" class="xref-table" href="#T1">Extended Data Table 1</a>). We used eggs from independent traps to establish two replicate laboratory colonies for each of 23 populations, and a single colony for the remaining 4 populations (n=50 colonies total, <a id="xref-table-wrap-2-1" class="xref-table" href="#T2">Extended Data Table 2</a>).</p><div id="F1" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/02/13/2020.02.12.939041/F1.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Preference for human odor varies widely in Aedes aegypti mosquitoes across Africa. a-b, Satellite images or photographs of mosquito collection localities with different human population densities (a) or levels of precipitation seasonality (b). Satellite images are from Google Earth, copyright Maxar Technologies and CNES/Airbus. c, Map of collection localities. Diagonal hatched lines mark the Sahel ecoclimatic zone. Extended Data Table 1 has full location names. d, Host preference measured in a two-port olfactometer (inset) for all African localities plus a reference colony from Thailand (T51) and a lab colony most likely to have originated in the United States (ORL) (n=3-14 trials with 25-110 females per trial; details in Extended Data Table 2). Bars indicate 95% confidence intervals. Grey boxes around location names highlight adjacent forest-town pairs (forest in green text). Circle sizes and bar colors show population density and climate, respectively, as in (c). e, Females from adjacent forest and town habitats did not differ in preference (P&gt;0.05, ns=not significant)." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-2014705" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;&lt;span class=&quot;caption-title&quot;&gt;Preference for human odor varies widely in &lt;em&gt;Aedes aegypti&lt;/em&gt; mosquitoes across Africa.&lt;/span&gt; &lt;strong&gt;a-b&lt;/strong&gt;, Satellite images or photographs of mosquito collection localities with different human population densities (a) or levels of precipitation seasonality (b). Satellite images are from Google Earth, copyright Maxar Technologies and CNES/Airbus. &lt;strong&gt;c&lt;/strong&gt;, Map of collection localities. Diagonal hatched lines mark the Sahel ecoclimatic zone. Extended Data Table 1 has full location names. &lt;strong&gt;d&lt;/strong&gt;, Host preference measured in a two-port olfactometer (inset) for all African localities plus a reference colony from Thailand (T51) and a lab colony most likely to have originated in the United States (ORL) (n=3-14 trials with 25-110 females per trial; details in Extended Data Table 2). Bars indicate 95% confidence intervals. Grey boxes around location names highlight adjacent forest-town pairs (forest in green text). Circle sizes and bar colors show population density and climate, respectively, as in (c). &lt;strong&gt;e&lt;/strong&gt;, Females from adjacent forest and town habitats did not differ in preference (&lt;em&gt;P&lt;/em&gt;&gt;0.05, ns=not significant).&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Fig. 1." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2020/02/13/2020.02.12.939041/F1.medium.gif" width="440" height="405"/><noscript><img class="highwire-fragment fragment-image" alt="Fig. 1." src="https://www.biorxiv.org/content/biorxiv/early/2020/02/13/2020.02.12.939041/F1.medium.gif" width="440" height="405"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/02/13/2020.02.12.939041/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Fig. 1." data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/02/13/2020.02.12.939041/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml"><span class="fig-label">Fig. 1.</span> <span class="caption-title">Preference for human odor varies widely in <em>Aedes aegypti</em> mosquitoes across Africa.</span><p id="p-10" class="first-child"><strong>a-b</strong>, Satellite images or photographs of mosquito collection localities with different human population densities (a) or levels of precipitation seasonality (b). Satellite images are from Google Earth, copyright Maxar Technologies and CNES/Airbus. <strong>c</strong>, Map of collection localities. Diagonal hatched lines mark the Sahel ecoclimatic zone. <a id="xref-table-wrap-1-4" class="xref-table" href="#T1">Extended Data Table 1</a> has full location names. <strong>d</strong>, Host preference measured in a two-port olfactometer (inset) for all African localities plus a reference colony from Thailand (T51) and a lab colony most likely to have originated in the United States (ORL) (n=3-14 trials with 25-110 females per trial; details in <a id="xref-table-wrap-2-2" class="xref-table" href="#T2">Extended Data Table 2</a>). Bars indicate 95% confidence intervals. Grey boxes around location names highlight adjacent forest-town pairs (forest in green text). Circle sizes and bar colors show population density and climate, respectively, as in (c). <strong>e</strong>, Females from adjacent forest and town habitats did not differ in preference (<em>P</em>&gt;0.05, ns=not significant).</p><div class="sb-div caption-clear"></div></div></div><div id="sec-2" class="subsection"><h3>Preference for human odor varies widely across sub-Saharan Africa</h3><p id="p-11">Mosquitoes choose hosts based largely on body odor<sup><a id="xref-ref-4-2" class="xref-bibr" href="#ref-4">4</a></sup>. <em>Ae. aegypti</em> females from human-biting populations show a robust preference for human odor, while those from generalist populations often prefer the odor of non-human animals<sup><a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">10</a></sup>. We tested the odor preference of colony females from each population in a two-port olfactometer and estimated preference using a beta-binomial mixed model that accounts for trial structure (<a id="xref-fig-1-4" class="xref-fig" href="#F1">Fig. 1d</a>, <a id="xref-fig-5-1" class="xref-fig" href="#F5">Extended Data Fig. 1a-b</a>, <a id="xref-table-wrap-2-3" class="xref-table" href="#T2">Extended Data Table 2</a>, see Methods). The results were generalizable across different individual humans and different animal species used as stimuli (<a id="xref-fig-5-2" class="xref-fig" href="#F5">Extended Data Fig. 1c</a>).</p><p id="p-12">Most populations preferred animals, but one population from Central Africa stood out as having an extreme animal preference (Franceville, Gabon; FCV), and three from West Africa showed either no preference (Ouagadougou, Burkina Faso; OGD) or clear human preference (Thies and Ngoye, Senegal; THI, NGO) (<a id="xref-fig-1-5" class="xref-fig" href="#F1">Fig. 1d</a>). The significance of this geographic variation (Likelihood Ratio Test <em>P</em>&lt;2.2×10<sup>−16</sup>) was further supported by a strong correlation between the preference of replicate colonies from the same location (Pearson <em>R</em><sup><em>2</em></sup>=0.60, <em>P</em>=1.5×10<sup>−5</sup>, <a id="xref-fig-5-3" class="xref-fig" href="#F5">Extended Data Fig. 1d</a>). As seen in previous work, overall response rates mirrored preference, with females from animal-preferring colonies being less likely to choose either host in the assay (<a id="xref-fig-5-4" class="xref-fig" href="#F5">Extended Data Fig. 1e-f</a>)<sup><a id="xref-ref-10-2" class="xref-bibr" href="#ref-10">10</a>,<a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">11</a></sup>.</p></div><div id="sec-3" class="subsection"><h3>Preference variation is explained by two ecological factors</h3><p id="p-13">Human-biting may be favored in areas with a high density of humans. Whether or not humans were living in the immediate vicinity of collection sites had no effect on behavior across a set of paired forest and town locations (<a id="xref-fig-1-6" class="xref-fig" href="#F1">Fig. 1e</a>). However, linear modeling of preference variation across all locations revealed a clear effect when humans were counted within a circle of radius 20-50km (Likelihood ratio test all <em>P</em>≤0.002; compare grey and black lines in <a id="xref-fig-6-1" class="xref-fig" href="#F6">Extended Data Fig. 2a</a>). This result supports the idea that high regional human population density helps drive mosquito preference for humans.</p><p id="p-14">The population density model included latitude and longitude as covariates to control for clear geographic trends in the data (higher preference for humans in the northwest, <a id="xref-fig-1-7" class="xref-fig" href="#F1">Fig. 1c</a>). We wondered whether climate might explain some of this additional variation. Surprisingly, when we used stepwise model selection to replace latitude and longitude with ecologically relevant climate variables (Bio1-Bio19 from the WorldClim 2 dataset; <a id="xref-fig-6-2" class="xref-fig" href="#F6">Extended Data Fig. 2b-c</a>)<sup><a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">12</a></sup>, the best climate variables explained more behavioral variation than human population density itself. In the final model, human population density explained 18% of variation (<a id="xref-fig-2-1" class="xref-fig" href="#F2">Fig. 2a</a>, Likelihood Ratio Test <em>P</em>=1.0×10<sup>−5</sup>, density measured within 20 km radius). The strongest climate predictor was precipitation seasonality (<a id="xref-fig-2-2" class="xref-fig" href="#F2">Fig. 2b</a>, <a id="xref-fig-6-3" class="xref-fig" href="#F6">Extended Data Fig. 2b</a>, Likelihood Ratio Test <em>P</em>=1.2×10<sup>−8</sup>), a measure of how variable rainfall is from month to month. A third-degree polynomial provided the best fit (<a id="xref-fig-6-4" class="xref-fig" href="#F6">Extended Data Fig. 2a-b</a>), helping to predict the abrupt emergence of preference for humans in the Sahel ecoclimatic zone of West Africa, where it is dry for 9 months of the year and all rainfall comes during a short, intense rainy season (<a id="xref-fig-1-8" class="xref-fig" href="#F1">Fig. 1c</a>, <a id="xref-fig-2-3" class="xref-fig" href="#F2">2b</a>). A second variable, level of precipitation during the warmest quarter of the year, also contributed significantly to our model (<a id="xref-fig-2-4" class="xref-fig" href="#F2">Fig. 2c</a>, <a id="xref-fig-6-5" class="xref-fig" href="#F6">Extended Data Fig. 2c</a>, Likelihood Ratio Test <em>P</em>=0.014) and helped explain behavior across populations in the animal-preferring range – the bulk of our sample. Animal preference was weaker in places with less rain at the hottest time of year (<a id="xref-fig-2-5" class="xref-fig" href="#F2">Fig. 2c</a>, <a id="xref-fig-6-6" class="xref-fig" href="#F6">Extended Data Fig. 2d</a>).</p><div id="F2" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/02/13/2020.02.12.939041/F2.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Human population density and dry season intensity explain variation in preference a, Human population density explains 18% of variation in preference (linear fit, density calculated within 20km radius, Likelihood Ratio Test [LRT] P=1.0&#xD7;10&#x2212;5). b-c, Preference for humans increases in habitats with highly seasonal rainfall (b, cubic monotonic polynomial fit, LRT P=1.2&#xD7;10&#x2212;8) and decreases in habitats with more rain at the warmest time of year (c, linear fit, LRT P=0.016 for model that already includes seasonality). d, Overall dry season intensity (combination of variables shown in b-c) explains 65% of variation in preference (LRT P=3.0&#xD7;10&#x2212;9). All analyses were carried out with logit-transformed preference indices subsequently back-transformed for plotting." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-2014705" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;&lt;span class=&quot;caption-title&quot;&gt;Human population density and dry season intensity explain variation in preference&lt;/span&gt; &lt;strong&gt;a&lt;/strong&gt;, Human population density explains 18% of variation in preference (linear fit, density calculated within 20km radius, Likelihood Ratio Test [LRT] &lt;em&gt;P&lt;/em&gt;=1.0&#xD7;10&lt;sup&gt;&#x2212;5&lt;/sup&gt;). &lt;strong&gt;b-c&lt;/strong&gt;, Preference for humans increases in habitats with highly seasonal rainfall (b, cubic monotonic polynomial fit, LRT &lt;em&gt;P&lt;/em&gt;=1.2&#xD7;10&lt;sup&gt;&#x2212;8&lt;/sup&gt;) and decreases in habitats with more rain at the warmest time of year (c, linear fit, LRT &lt;em&gt;P&lt;/em&gt;=0.016 for model that already includes seasonality). &lt;strong&gt;d&lt;/strong&gt;, Overall dry season intensity (combination of variables shown in b-c) explains 65% of variation in preference (LRT &lt;em&gt;P&lt;/em&gt;=3.0&#xD7;10&lt;sup&gt;&#x2212;9&lt;/sup&gt;). All analyses were carried out with logit-transformed preference indices subsequently back-transformed for plotting.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Fig. 2." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2020/02/13/2020.02.12.939041/F2.medium.gif" width="440" height="151"/><noscript><img class="highwire-fragment fragment-image" alt="Fig. 2." src="https://www.biorxiv.org/content/biorxiv/early/2020/02/13/2020.02.12.939041/F2.medium.gif" width="440" height="151"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/02/13/2020.02.12.939041/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Fig. 2." data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/02/13/2020.02.12.939041/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Fig. 2.</span> <span class="caption-title">Human population density and dry season intensity explain variation in preference</span><p id="p-15" class="first-child"><strong>a</strong>, Human population density explains 18% of variation in preference (linear fit, density calculated within 20km radius, Likelihood Ratio Test [LRT] <em>P</em>=1.0×10<sup>−5</sup>). <strong>b-c</strong>, Preference for humans increases in habitats with highly seasonal rainfall (b, cubic monotonic polynomial fit, LRT <em>P</em>=1.2×10<sup>−8</sup>) and decreases in habitats with more rain at the warmest time of year (c, linear fit, LRT <em>P</em>=0.016 for model that already includes seasonality). <strong>d</strong>, Overall dry season intensity (combination of variables shown in b-c) explains 65% of variation in preference (LRT <em>P</em>=3.0×10<sup>−9</sup>). All analyses were carried out with logit-transformed preference indices subsequently back-transformed for plotting.</p><div class="sb-div caption-clear"></div></div></div><p id="p-16">Taken together, these two climate variables capture the challenges mosquitoes face during the dry season. <em>Ae. aegypti</em> lay their eggs on wet substrate just above the water line in tree holes, rock pools, or artificial containers<sup><a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">13</a></sup>. If the eggs remain wet, they can hatch immediately. However, eggs laid in wild areas at the end of the rains must pause development and survive the duration of the dry season until rain returns – a particularly difficult challenge when the dry season is long (<em>i.e.</em> precipitation seasonality is high) and hot (<em>i.e.</em> precipitation is low at the warmest time of year)<sup><a id="xref-ref-13-2" class="xref-bibr" href="#ref-13">13</a>,<a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">14</a></sup>. Human water storage helps <em>Ae. aegypti</em> in harsh environments by providing a year-round aquatic niche for larval development. We put the two climate variables together into a single index of dry season intensity that explains 65% of variation in host odor preference across Africa (<a id="xref-fig-2-6" class="xref-fig" href="#F2">Fig. 2d</a>, Likelihood Ratio Test <em>P</em>=3.0×10<sup>−9</sup>). These findings point to long, hot dry seasons as a key selective factor driving <em>Ae. aegypti</em> specialization on human hosts, likely as a by-product of dependence on human-stored water for breeding<sup><a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">15</a>,<a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">16</a></sup>.</p></div><div id="sec-4" class="subsection"><h3>Preference for humans within and outside Africa has a single genomic origin</h3><p id="p-17">Females of the globally invasive human specialist subspecies are characterized by light scaling on the back of the abdomen (first tergite, <a id="xref-fig-3-1" class="xref-fig" href="#F3">Fig. 3a</a> inset, <a id="xref-fig-3-2" class="xref-fig" href="#F3">Fig. 3b</a> grey populations)<sup><a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">17</a>,<a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">18</a></sup>, and previous work documented this trait in the Sahel of northern Senegal where we observed preference for humans<sup><a id="xref-ref-15-2" class="xref-bibr" href="#ref-15">15</a>,<a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">19</a></sup>. We therefore wondered whether it might be linked to behavior in a continuous way across our full sample set. Indeed, abdominal scaling was strongly correlated with preference for humans (<em>R</em><sup><em>2</em></sup>=0.81, <em>P</em>=6.7×10<sup>−10</sup>, <a id="xref-fig-3-3" class="xref-fig" href="#F3">Fig. 3a-b</a>). The trend was driven not only by the most extreme variation in Senegal and outside of Africa but also by more modest variation in other regions (<em>R</em><sup><em>2</em></sup>=0.46, <em>P</em>=0.002, Senegal and non-African reference populations excluded).</p><div id="F3" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/02/13/2020.02.12.939041/F3.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Specialization on humans has a single genomic origin and is associated with changes concentrated in a few key chromosomal regions. a, Variation in percentage of the first abdominal tergite (inset, right) covered in white scales. Circle colors indicate host preference as in Fig. 1. b, Morphology is strongly correlated with host preference. Dark grey dots represent non-African reference colonies (see also Fig. 1d). c-d, Bar plots showing proportion ancestry derived from k=3 (c) or k=6 (d) ancestry components for 375 mosquito genomes. e, Average proportion ancestry from the red cluster (k=3 analysis) is strongly correlated with preference for humans across populations. f, Polarized genomic divergence (Population Branch Statistic, PBS) along the lineages leading to three human-seeking populations (top; NGO, THI, OGD) and two animal-seeking populations (bottom; KED, OHI). Grey shading indicates regions that were outliers (permutation FDR" class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-2014705" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;&lt;span class=&quot;caption-title&quot;&gt;Specialization on humans has a single genomic origin and is associated with changes concentrated in a few key chromosomal regions.&lt;/span&gt; &lt;strong&gt;a&lt;/strong&gt;, Variation in percentage of the first abdominal tergite (inset, right) covered in white scales. Circle colors indicate host preference as in Fig. 1. &lt;strong&gt;b&lt;/strong&gt;, Morphology is strongly correlated with host preference. Dark grey dots represent non-African reference colonies (see also Fig. 1d). &lt;strong&gt;c&lt;/strong&gt;-&lt;strong&gt;d&lt;/strong&gt;, Bar plots showing proportion ancestry derived from k=3 (c) or k=6 (d) ancestry components for 375 mosquito genomes. &lt;strong&gt;e&lt;/strong&gt;, Average proportion ancestry from the red cluster (k=3 analysis) is strongly correlated with preference for humans across populations. &lt;strong&gt;f&lt;/strong&gt;, Polarized genomic divergence (Population Branch Statistic, &lt;em&gt;PBS&lt;/em&gt;) along the lineages leading to three human-seeking populations (top; NGO, THI, OGD) and two animal-seeking populations (bottom; KED, OHI). Grey shading indicates regions that were outliers (permutation FDR&lt;0.05) in all three human-seeking populations (dark grey) or in just the two populations showing strong preference for humans (NGO, THI; light grey). &lt;strong&gt;g&lt;/strong&gt;, Proportion of derived variation shared by African human-seeking populations with non-African human specialists (&lt;em&gt;f&lt;/em&gt;&lt;sub&gt;&lt;em&gt;D&lt;/em&gt;&lt;/sub&gt;). Grey shading as in (f). Grey triangle in (f) and (g) mark the location of &lt;em&gt;Or4&lt;/em&gt;, an odorant receptor previously linked to preference for human odor&lt;sup&gt;10&lt;/sup&gt;.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Fig. 3." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2020/02/13/2020.02.12.939041/F3.medium.gif" width="440" height="354"/><noscript><img class="highwire-fragment fragment-image" alt="Fig. 3." src="https://www.biorxiv.org/content/biorxiv/early/2020/02/13/2020.02.12.939041/F3.medium.gif" width="440" height="354"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/02/13/2020.02.12.939041/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Fig. 3." data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/02/13/2020.02.12.939041/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Fig. 3.</span> <span class="caption-title">Specialization on humans has a single genomic origin and is associated with changes concentrated in a few key chromosomal regions.</span><p id="p-18" class="first-child"><strong>a</strong>, Variation in percentage of the first abdominal tergite (inset, right) covered in white scales. Circle colors indicate host preference as in <a id="xref-fig-1-9" class="xref-fig" href="#F1">Fig. 1</a>. <strong>b</strong>, Morphology is strongly correlated with host preference. Dark grey dots represent non-African reference colonies (see also <a id="xref-fig-1-10" class="xref-fig" href="#F1">Fig. 1d</a>). <strong>c</strong>-<strong>d</strong>, Bar plots showing proportion ancestry derived from k=3 (c) or k=6 (d) ancestry components for 375 mosquito genomes. <strong>e</strong>, Average proportion ancestry from the red cluster (k=3 analysis) is strongly correlated with preference for humans across populations. <strong>f</strong>, Polarized genomic divergence (Population Branch Statistic, <em>PBS</em>) along the lineages leading to three human-seeking populations (top; NGO, THI, OGD) and two animal-seeking populations (bottom; KED, OHI). Grey shading indicates regions that were outliers (permutation FDR&lt;0.05) in all three human-seeking populations (dark grey) or in just the two populations showing strong preference for humans (NGO, THI; light grey). <strong>g</strong>, Proportion of derived variation shared by African human-seeking populations with non-African human specialists (<em>f</em><sub><em>D</em></sub>). Grey shading as in (f). Grey triangle in (f) and (g) mark the location of <em>Or4</em>, an odorant receptor previously linked to preference for human odor<sup><a id="xref-ref-10-3" class="xref-bibr" href="#ref-10">10</a></sup>.</p><div class="sb-div caption-clear"></div></div></div><p id="p-19">The morphological resemblance of human-preferring mosquitoes within and outside Africa suggests shared ancestry. To test this hypothesis, we sequenced the genomes of ∼15 individuals from 24 sites in our current study plus one site in South America and one in Asia (n=366 genomes after exclusion of relatives, ∼15x coverage). We also sequenced 9 previously collected individuals of the human-biting domestic form from Rabai<sup><a id="xref-ref-10-4" class="xref-bibr" href="#ref-10">10</a></sup>, which was not present when we carried out the fieldwork for this study in 2017. Analyses of overall population structure were consistent with earlier work<sup><a id="xref-ref-6-2" class="xref-bibr" href="#ref-6">6</a>,<a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">20</a></sup>. The program ADMIXTURE<sup><a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">21</a></sup> revealed strong support for a model with three genomic clusters or ancestry components corresponding to coastal East Africa, Central/West Africa, and globally invasive human specialists (<a id="xref-fig-3-4" class="xref-fig" href="#F3">Fig. 3c</a>, <a id="xref-fig-7-1" class="xref-fig" href="#F7">Extended Data Fig. 3a-b</a>). The Rabai domestic form was the only African population to group unambiguously with non-African human specialists, consistent with its putative origin as a recent, localized reintroduction of non-African mosquitoes<sup><a id="xref-ref-20-2" class="xref-bibr" href="#ref-20">20</a></sup>. However, many populations across sub-Saharan Africa showed some level of ancestry from the human specialist component (red in <a id="xref-fig-3-5" class="xref-fig" href="#F3">Fig. 3c</a>), and this signal was strongly correlated with preference for humans (<a id="xref-fig-3-6" class="xref-fig" href="#F3">Fig. 3e</a>, <em>R</em><sup><em>2</em></sup>=0.76, <em>P</em>=2.7×10<sup>−8</sup>) – supporting a single, shared origin for the behavior.</p><p id="p-20">The shared ancestry of human-preferring mosquitoes within and outside African has two potential explanations – contemporary admixture due to back-to-Africa gene flow or ancestral relationships present before the species left Africa. Recent admixture has almost certainly occurred in coastal East Africa, where the reintroduced domestic form once thrived. However, a recent exome study suggested that a supposedly highly ‘admixed’ population from the Sahel region of West Africa may instead be ancestral to bottlenecked, non-African populations<sup><a id="xref-ref-7-2" class="xref-bibr" href="#ref-7">7</a></sup>. Consistent with this interpretation, the three most human-seeking Sahelian populations in our dataset (NGO, THI, OGD, <a id="xref-fig-1-11" class="xref-fig" href="#F1">Fig. 1c-d</a>) formed a unified genomic cluster, distinct from both the globally invasive subspecies and nearby animal-preferring populations, in an ADMIXTURE analysis with six clusters (<a id="xref-fig-3-7" class="xref-fig" href="#F3">Fig. 3d</a>, <a id="xref-fig-7-2" class="xref-fig" href="#F7">Extended Data Fig. 3a</a>). Principal components analysis also shows these populations extending away from the West African cluster towards the globally invasive specialists (<a id="xref-fig-7-3" class="xref-fig" href="#F7">Extended Data Fig. 3c-d</a>).</p></div><div id="sec-5" class="subsection"><h3>Loci associated with specialization are clustered in the genome</h3><p id="p-21">Human-seeking mosquitoes in the Sahel have likely experienced selection for human-biting and other traits that help them live and breed in human environments – a process expected to drive and maintain divergence in parts of the genome harboring relevant loci. We used the Population Branch Statistic (<em>PBS</em>)<sup><a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">22</a></sup> to identify chromosomal regions with enhanced divergence along the branches leading to Ngoye (NGO), Thies (THI), and Ouagadougou (OGD) relative to nearby animal-preferring mosquitoes from Bantata (BTT) (<a id="xref-fig-3-8" class="xref-fig" href="#F3">Fig. 3f</a>, FCV used as outgroup). Several regions stood out in all three populations (dark grey regions in <a id="xref-fig-3-9" class="xref-fig" href="#F3">Fig. 3f</a>, permutation false discovery rate&lt;0.05), including a large area at the distal end of the first chromosome containing an odorant receptor previously linked to preference for humans<sup><a id="xref-ref-10-5" class="xref-bibr" href="#ref-10">10</a></sup>. Interestingly, this same region showed signs of recent, strong selection and/or a chromosomal rearrangement in the geographic transition zone to highly seasonal Sahelian climates in Senegal (<a id="xref-fig-8-1" class="xref-fig" href="#F8">Extended Data Fig. 4</a>).</p><p id="p-22">Elevated divergence in key genomic regions might reflect the maintenance of human specialist ancestry in the face of gene flow from nearby animal-preferring mosquitoes. Using the <em>f</em><sub><em>D</em></sub> statistic<sup><a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">23</a></sup> we found that human-seeking Sahelian populations were indeed most likely to share derived alleles with non-African populations in outlier regions (<a id="xref-fig-3-10" class="xref-fig" href="#F3">Fig. 3g</a>; Fisher’s exact test, all <em>P</em>&lt;0.0001). Separate analyses of absolute differentiation between populations (<em>d</em><sub><em>xy</em></sub>) and diversity within populations (<em>π</em>) were also consistent with the idea that outlier regions have been affected by strong selection during the establishment of human specialist ecology, followed by the accumulation of sequence differences via subsequent selection against gene flow (<a id="xref-fig-9-1" class="xref-fig" href="#F9">Extended Data Fig. 5</a>)<sup><a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">24</a></sup>.</p><p id="p-23">While the most divergent chromosomal regions may harbor key loci of large effect, specialization on humans is likely to involve a diverse suite of behavioral and physiological traits with complex underlying genetics. Consistent with this hypothesis, a screen for genetic outliers more strongly associated with a human specialist ancestry component than expected under neutral evolution<sup><a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">25</a></sup> revealed thousands of significant single nucleotide variants scattered across the entire genome (n=16,782 SNPs at Bonferroni-adjusted <em>P</em>&lt;0.05, <a id="xref-fig-10-1" class="xref-fig" href="#F10">Extended Data Fig. 6</a>). However, the highest peaks fell in the same regions that stood out in the Population Branch Statistic scan.</p></div><div id="sec-6" class="subsection"><h3>Rapid urbanization may drive a shift towards human-biting by 2050</h3><p id="p-24">Both climate and human population density are changing rapidly in Africa<sup><a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">26</a>,<a id="xref-ref-27-1" class="xref-bibr" href="#ref-27">27</a></sup>. We therefore incorporated publicly available climate and human population projections into our model to explore how the behavior of African <em>Ae. aegypti</em> mosquitoes might be expected to evolve over the next 30-50 years (see Methods). Projected changes in relevant precipitation variables are modest (<a id="xref-fig-11-1" class="xref-fig" href="#F11">Extended Data Fig. 7</a>) and unlikely to drive substantial shifts in preference (<a id="xref-fig-4-1" class="xref-fig" href="#F4">Fig. 4a</a>, <a id="xref-fig-12-1" class="xref-fig" href="#F12">Extended Data Fig. 8</a>). Rapid urbanization, in contrast, may trigger transitions to human-biting in many cities across the continent by 2050 (<a id="xref-fig-4-2" class="xref-fig" href="#F4">Fig. 4a</a>, <a id="xref-fig-12-2" class="xref-fig" href="#F12">Extended Data Fig. 8</a>), with the important caveat that the highest projected population densities are well above those used to fit our model (<a id="xref-fig-4-3" class="xref-fig" href="#F4">Fig. 4a</a>).</p><div id="F4" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/02/13/2020.02.12.939041/F4.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Rapid urbanization may favor a shift towards human-biting in large African cities by 2050. a, Current (circles) and projected (arrows) environmental parameters for each site. Circle colors indicate current host preference. Plot background color indicates preference predicted by model described in Fig. 2. b, Map from Fig. 1c showing projected (rather than current) human population densities (circle size), precipitation seasonality (map background color), and host odor preference (circle color) for the year 2050." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-2014705" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;&lt;span class=&quot;caption-title&quot;&gt;Rapid urbanization may favor a shift towards human-biting in large African cities by 2050.&lt;/span&gt; &lt;strong&gt;a&lt;/strong&gt;, Current (circles) and projected (arrows) environmental parameters for each site. Circle colors indicate current host preference. Plot background color indicates preference predicted by model described in Fig. 2. &lt;strong&gt;b&lt;/strong&gt;, Map from Fig. 1c showing projected (rather than current) human population densities (circle size), precipitation seasonality (map background color), and host odor preference (circle color) for the year 2050.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Fig. 4." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2020/02/13/2020.02.12.939041/F4.medium.gif" width="440" height="222"/><noscript><img class="highwire-fragment fragment-image" alt="Fig. 4." src="https://www.biorxiv.org/content/biorxiv/early/2020/02/13/2020.02.12.939041/F4.medium.gif" width="440" height="222"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/02/13/2020.02.12.939041/F4.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Fig. 4." data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/02/13/2020.02.12.939041/F4.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Fig. 4.</span> <span class="caption-title">Rapid urbanization may favor a shift towards human-biting in large African cities by 2050.</span><p id="p-25" class="first-child"><strong>a</strong>, Current (circles) and projected (arrows) environmental parameters for each site. Circle colors indicate current host preference. Plot background color indicates preference predicted by model described in <a id="xref-fig-2-7" class="xref-fig" href="#F2">Fig. 2</a>. <strong>b</strong>, Map from <a id="xref-fig-1-12" class="xref-fig" href="#F1">Fig. 1c</a> showing projected (rather than current) human population densities (circle size), precipitation seasonality (map background color), and host odor preference (circle color) for the year 2050.</p><div class="sb-div caption-clear"></div></div></div></div></div><div class="section" id="sec-7"><h2 class="">Discussion</h2><p id="p-26">Our findings suggest that human-biting <em>Ae. aegypti</em> are favored in highly seasonal climates, such as the western Sahel, where laying eggs in human-stored water may be the only way to survive the dry season. Once dependent on humans for breeding sites, mosquitoes may evolve preference for humans due to trade-offs between traits that promote effective use of locally abundant human targets and those necessary for finding and biting animals<sup><a id="xref-ref-28-1" class="xref-bibr" href="#ref-28">28</a></sup>. There are other indications that dry season dynamics played an important role in human specialization. A decades-old study found that globally invasive specialists from Asia and the Americas, as well as a single population from the Sahel, were all more resistant to desiccation than strains from other parts of Africa<sup><a id="xref-ref-29-1" class="xref-bibr" href="#ref-29">29</a></sup>. Moreover, human-biting is accompanied by preference for laying eggs in human water storage vessels in at least some areas – this preference was shown to allow the introduced domestic form in Rabai, Kenya to breed throughout the dry season<sup><a id="xref-ref-30-1" class="xref-bibr" href="#ref-30">30</a></sup>. Beyond <em>Aedes</em> mosquitoes, dry season dynamics may have played a key role in the evolution and divergence of <em>Anopheles</em> malaria vectors in Africa<sup><a id="xref-ref-31-1" class="xref-bibr" href="#ref-31">31</a></sup>.</p><p id="p-27">Human population density also helps predict contemporary mosquito behavior, and rapid urbanization will likely make it the most important driver of future change in Africa. However, it is unclear what will happen when human densities grow beyond the highest observed in this study, especially in the wetter and less seasonal parts of Central and East Africa. These sites may evolve strong preference for humans (<a id="xref-fig-4-4" class="xref-fig" href="#F4">Fig. 4a</a>), as suggested by the linear trend across contemporary sites (<a id="xref-fig-2-8" class="xref-fig" href="#F2">Fig. 2a</a>). Alternatively, the effect of human density could plateau at intermediate preference (willingness to bite humans) without driving the true specialization seen in highly seasonal environments. Despite these ambiguities, the speed and scale of ongoing urbanization argue strongly for careful monitoring of potential shifts in <em>Ae. aegypti</em> behavior (or correlated morphology/genetics) across Africa.</p><p id="p-28">Our genomic data are consistent with the hypothesis that human specialization is not only favored in the seasonal Sahel, but also first arose there before seeding globally invasive populations<sup><a id="xref-ref-7-3" class="xref-bibr" href="#ref-7">7</a></sup>. Further work is needed to test this hypothesis – incorporating genome-wide data from a wider range of global populations. <em>Ae. aegypti</em> from Angola<sup><a id="xref-ref-6-3" class="xref-bibr" href="#ref-6">6</a></sup> and northern Argentina<sup><a id="xref-ref-20-3" class="xref-bibr" href="#ref-20">20</a></sup>, for example, show similar patterns of ancestry to populations in the Sahel. Regardless, adaptation to humans in this mosquito is clearly associated with shifts in the frequency of a large number of variants, scattered throughout the genome but especially concentrated in a few key regions. More broadly, the tight correlations between ancestry, behavior, and environment reveal a dynamic situation playing out across the continent as a whole, with selection and gene flow fine-tuning the frequency of human-adaptive alleles, and thus levels of attraction to human hosts, according to local climate and human population density.</p><p id="p-29">Our results synthesize behavioral, ecological, and genomic data to demonstrate significant, environmentally structured variation in a key disease vector. We urgently need to incorporate such variation into epidemiological models and other efforts to predict and manage the transmission of <em>Ae. aegypti</em>-borne disease in Africa.</p></div><div class="section" id="sec-8"><h2 class="">Methods</h2><div class="section" id="sec-9"><h3>Ethics and regulatory information</h3><p id="p-30">Mosquito eggs were collected and exported with permission from local institutions and/or governments as required (Kenya SERU No. 3433; Uganda permit 2014-12-134; Gabon AR0013/16/MESRS/CENAREST/CG/CST/CSAR and AE16008/PR/ANPN/SE/CS/AEPN) and imported to the USA under USDA permit 129920. The use of non-human animals in olfactometer trials was approved and monitored by the Princeton University Institutional Animal Care and Use Committee (protocols 1998-17 and 2113-17). The participation of humans in olfactometer trials was approved and monitored by the Princeton University Institutional Review Board (protocol 8170). All human subjects gave their informed consent to participate in work carried out at Princeton University. Human-blood feeding conducted for colony maintenance did not meet the definition of human subjects research, as determined by the Princeton University IRB (Non Human-Subjects Research Determination 6870).</p></div><div class="section" id="sec-10"><h3>Field collections</h3><p id="p-31">We collected mosquito eggs in each African sampling location by distributing 20-60 ‘ovitraps’ at regular intervals across the landscape. Ovitraps consisted of 32 oz black plastic cups (The Executive Advertising), each lined with a 38 x 15 cm piece of 76 lb (34.5 kg) seed germination paper (Anchor Paper Co.) and filled with 3-8 cm of water. In all locations except coastal Kenya, the water was infused with a mixture of fresh or dry mango leaves collected from the leaf litter (n=∼20 leaves per 10 liters of water) for 1-2 days before use. In coastal Kenya and Uganda, we used tap water or made a similar infusion with twigs, bark, and leaves from unidentified broad-leafed trees. Anecdotally, water source did not appear to affect egg numbers. Each ovitrap had a hole in the side at a height of ∼3 inches to allow rainwater to drain from the trap. We attempted to spread ovitraps at approximately 100 meter intervals, but placed them at intervals as small as 10 meters in areas with limited access. We left ovitraps in the field for two nights before returning to collect egg-impregnated seed papers. We then dried the papers slowly on beds of paper towels over the course of 24 hours and stored them in airtight, whirl-pak bags during transport back to the laboratory. The only exception to this approach was at Bantata, Senegal (BTT), where we collected <em>Saba senegalensis</em> husks from the forest floor, flooded them with water, and collected hatchling larvae over the following few days. Most collections were carried out in 2017 and 2018, but Ugandan collections were carried out in 2015.</p></div><div class="section" id="sec-11"><h3>Generation and maintenance of laboratory colonies</h3><p id="p-32">We hatched egg-impregnated papers from each ovitrap in separate pans of hatch broth, made by dissolving finely ground Tetramin Tropical Tablets fish food (Spectrum Brands, Inc.) in deoxygenated water (¼ tablet/liter). We continued to feed larvae Tetramin <em>ad libitum</em> through to pupation, and transferred pupae from each seed paper to separate 32 oz HDPE plastic cages (VWR). Eclosing males and females were able to mate with each other in the cages and had access to 10% sucrose solution. Other mosquito species sometimes hatched from papers alongside <em>Ae. aegypti</em> and were eventually removed from cages without hindering our breeding efforts. However, <em>Ae. albopictus</em> males are known to satyrize <em>Ae. aegypti</em> females, rendering them infertile<sup><a id="xref-ref-32-1" class="xref-bibr" href="#ref-32">32</a></sup>. In areas where <em>Ae. albopictus</em> was present (Nigeria and Gabon), we therefore separated the male and female pupae reared from any given seed paper and let them eclose separately before identifying adults to species and recombining <em>Ae. aegypti</em> males and females only. We set aside 2-20 adults from each population for genome sequencing, using only a single individual per ovitrap/cage where possible to reduce the probability of sequencing siblings (n=10-20 individuals for 20 locations; n=2-9 for an additional 4 locations).</p><p id="p-33">We used mosquitoes from independent ovitraps to establish two replicate laboratory colonies for 23 locations and a single colony for the remaining 4 locations (<a id="xref-table-wrap-2-4" class="xref-table" href="#T2">Extended Data Table 2</a>). Each colony was founded using eggs from 4-43 females, except the Lope Forest colony which was founded with eggs from a single female (<a id="xref-table-wrap-2-5" class="xref-table" href="#T2">Extended Data Table 2</a>). Founding females were fed on human volunteers (see ethics subsection) and allowed to lay eggs individually on wet filter paper cones (Whatman 55 mm Grade 1 filter paper) in small shell vials (Applied Scientific Drosophila Vials, 28.5 mm diameter, 95 mm height). We gave females multiple opportunities to feed in order to ensure high feeding rates (typically &gt;90%) and thus reduce the potential for selection on host preference. However, it was sometimes difficult to coax recalcitrant females to lay eggs in the lab. Oviposition rates ranged from 35 to 100% in the first generation. In subsequent generations, we maintained population sizes of 300-600 individuals per colony, continued to ensure blood-feeding rates &gt;90%, and tried to maximize oviposition by forcing females into contact with wet, potting soil-infused, seed germination paper cones in small 8.5 oz HDPE plastic cups (VWR) for 2 days (30 females/cup). Eggs were dried and stored at 16°C, 80%RH for up to 6 months between generations. The only exception to these breeding procedures applied to the first 2 generations of the colony from Zika, Uganda (ZIK), which was fed on a membrane and laid eggs on cups of water placed inside a large breeding cage.</p><p id="p-34">We included two reference colonies of non-African origin in behavioral and morphological studies (<a id="xref-table-wrap-2-6" class="xref-table" href="#T2">Extended Data Table 2</a>). These were a colony from Thailand (T51) generated as described above and a laboratory colony originally maintained at the USDA labs in Orlando, Florida (ORL) that is of uncertain origin but was most likely supplemented for decades with local Floridian mosquitoes<sup><a id="xref-ref-33-1" class="xref-bibr" href="#ref-33">33</a></sup>.</p></div><div class="section" id="sec-12"><h3>Behavior</h3><p id="p-35">We tested the host odor preference of 7-14 day old females that had been housed over night with access to water only (no sucrose). Different colonies were hatched on the same day, females mated freely with males after eclosion, and females were matched for age on testing days. We used a two-port olfactometer as previously described<sup><a id="xref-ref-10-6" class="xref-bibr" href="#ref-10">10</a>,<a id="xref-ref-11-2" class="xref-bibr" href="#ref-11">11</a></sup> (<a id="xref-fig-1-13" class="xref-fig" href="#F1">Fig. 1d</a> inset), with small modifications. Instead of using a large box fan to pull air through the device from the back of the olfactometer, we used a smaller fan to pull air through an 10.2 x 10.2 cm opening in the back panel. Instead of pulling air from the room, carbon-filtered, conditioned air was supplied to the two olfactometer ports from an independent building source. Inflow and outflow was balanced to achieve a rate of approximately 0.3m/s as measured at the traps. In each trial, 25-110 females were allowed to acclimate for 5 minutes in the large holding chamber before turning on the fan and opening a sliding door to expose them to streams of air coming from two alternative cylindrical traps and host chambers. One host chamber contained an awake guinea pig (<em>Cavia porcellus</em>, pigmented breed) or button quail (<em>Coturnix coturnix</em>). The other contained a section of the arm of a human volunteer (middle of forearm to middle of upper arm; silicone sheeting used to seal the holes through which the arm was inserted). The breath of the animal mixed with its odor in the animal odor stream. To add human breath on the human side, we asked the human subject to breathe gently through a nasal mask into the host chamber every 30 seconds. Trials lasted 10 minutes, and mosquitoes choosing to fly upwind towards either host odor stream during this time were trapped in small ports and counted at the end.</p><p id="p-36">We carried out host preference trials in two main waves, with a 28-year old, European-American male serving as the human subject and one of two female guinea pigs serving as the animal subject. In the first wave, we tested second-generation colonies from Kenya and Gabon. In the second wave, we tested second generation colonies from Nigeria, Ghana, Burkina Faso, and Senegal, eighth- or ninth-generation colonies from Uganda, and two reference colonies of non-African origin. We also repeat-tested a representative set of first wave Kenyan and Gabonese colonies (RAB, VMB, FCV, LBV; by then in their fourth generation) in the second wave to ensure that results were comparable between waves. At least one of two colonies from every population included in a given wave was tested on every experimental day in order to balance random day-to-day variation with the population effects we were trying to estimate (<a id="xref-fig-5-5" class="xref-fig" href="#F5">Extended Data Fig. 1a-b</a>). Overall, we carried out 3-4 trials for each colony, except one of two replicate colonies from BTT, KED, and KUM, which were tested only twice. This resulted in a total of 7 trials for most populations, 3-6 trials for the six populations represented by only a single colony or for which one of two colonies had fewer trials, and 14 trials for the four populations tested in both waves. In total, we carried out 206 trials including 17,856 female mosquitoes, of which 7,385 responded to one or the other host odor.</p><p id="p-37">After the two main waves, we carried out a smaller set of trials with one colony from each of four representative African populations (FCV, OGD, AWK, NGO) and a wider array of host comparisons. In one set of trials, we substituted a 22-year old Nigerian-American female for the original 28-year old European-American male, and in another set we substituted a button quail for the guinea pig (<a id="xref-fig-5-6" class="xref-fig" href="#F5">Extended Data Fig. 1c</a>, n=3-5 trials per colony x host combination).</p><p id="p-38">We used a beta-binomial mixed generalized linear model as implemented in the R<sup><a id="xref-ref-34-1" class="xref-bibr" href="#ref-34">34</a></sup> package <em>glmmTMB</em><sup><a id="xref-ref-35-1" class="xref-bibr" href="#ref-35">35</a></sup> to model the probability of choosing a human versus animal host for each population. This model assumes independence of individual females within trials but accounts for trial structure and the fact that preference varies more from trial to trial than is expected for a binomial model (is overdispersed) due to random sources variation (e.g. exact starting position of females within the acclimation chamber at the start of a trial, small differences in airflow between right and left ports, uncontrollable trial-to-trial variation in live host stimuli etc.). Replicate colonies and trial day were included as random factors, while population was modelled as a fixed factor. We switched the guinea pig used and the side of the human versus animal host between days such that these effects would be subsumed under the trial day random factor. We used the R package <em>emmeans</em><sup><a id="xref-ref-36-1" class="xref-bibr" href="#ref-36">36</a></sup> to extract from our glm the fitted probability of choosing a human host with 95% confidence intervals. For purposes of data visualization, we transformed each probability (p) into a preference index (PI) ranging from −1 to 1 using the formula PI=2p-1. An index of zero means the mosquitoes were equally likely to choose either host (no preference), while an index above or below zero means the mosquitoes were more likely to choose the human or animal, respectively. We used a likelihood ratio test to compare our glm to a null model accounting for day-to-day variation but not population of origin. The same beta-binomial mixed generalized linear model was used to model the probability of responding to either host (overall response rates, <a id="xref-fig-5-7" class="xref-fig" href="#F5">Extended Data Fig. 1f</a>).</p></div><div class="section" id="sec-13"><h3>Ecological modeling</h3><p id="p-39">We first compared the behavior of mosquitoes from paired forest and town populations within 5-60km of each other. In one case, a single town population (KED) was paired with two forest sites (BTT and PKT). We estimated the effect of forest habitat using a linear model that estimated preference for each pair (or group for KED, BTT, and PKT) and a coefficient for forest or town habitat. This is conceptually very similar to carrying out a paired t-test, except it allowed us to take into account the two different forest sites near KED.</p><p id="p-40">We next explored the ecological factors associated with preference for humans across all populations in the sample set, again using a linear modelling framework. In this set of analyses, each population was represented by a single logit-transformed preference probability (generated by the beta-binomial model described in the previous section) and a single estimate of each ecological descriptor extracted from public datasets using the mean latitude and longitude of the ovitraps that contributed to the corresponding colony (or the mean of the two independent colony means for populations with two colonies).</p><p id="p-41">While immediate habitat had no effect on behavior, we hypothesized that human population density might be relevant when calculated across a larger spatial scale. We therefore used a 2.5-minute resolution population density raster from the United Nations World Population Prospects (UNWPP, 2015 population densities adjusted to country totals)<sup><a id="xref-ref-37-1" class="xref-bibr" href="#ref-37">37</a></sup> to compare the effect of density across buffers of the following radiuses: 5, 10, 15, 20, 25, 30, 35, 40, 45, 50, 60, 70, 80, 90, 100, 200, and 300 km. In a simple human population density model that also takes regional variation into account (logit(prob) ∼ human_pop_density + Latitude*Longitude), human population density had a significant effect across a wide range of spatial scales, but was strongest with ∼20-50km buffers (black line in <a id="xref-fig-6-7" class="xref-fig" href="#F6">Extended Data Fig. 2a</a>).</p><p id="p-42">To better understand the regional drivers of variation in preference, we used the WorldClim 2 bioclimatic variables (Bio1-19) as a set of candidate predictors. Because some of these variables are correlated, we also considered the predictive value of the first three principal components from a PCA analysis of Bio1-19 variation across our populations. In preliminary tests, Bio15 (precipitation seasonality) clearly showed the strongest single-variable association with preference. This was true both in a comparison of simple correlations between each variable and preference (Bio15 r=0.65) and when we included each variable in a linear model with human population density (20km buffer) (red circles in <a id="xref-fig-6-8" class="xref-fig" href="#F6">Extended Data Fig. 2b</a>). However, the relationship with Bio15 appeared to be strongly nonlinear, if still monotonic (<a id="xref-fig-2-9" class="xref-fig" href="#F2">Fig. 2b</a>). We therefore used a two-step procedure to model the nonlinearity. We first used the R package MonoPoly<sup><a id="xref-ref-37-2" class="xref-bibr" href="#ref-37">37</a></sup> to fit monotonic polynomials of different degrees to logit-transformed preference probabilities, and then included the fitted values as an offset (i.e. removed their effects before fitting) in the linear model (logit(prob) ∼ human_pop_density_20km + offset(fitted(monotonic polynomial Bio15))). We found that a third-order monotonic polynomial significantly improved model performance, minimizing the Akaike Information Criterion (AIC). Rechecking the performance of different human population density buffers in this new model context showed that 20km yielded a much lower AIC than other buffers (light blue line in <a id="xref-fig-6-9" class="xref-fig" href="#F6">Extended Data Fig. 2a</a>). Note that this buffer most likely reflects the balance between selection and dispersal, and is not a direct reflection of adult dispersal patterns <em>per se</em>.</p><p id="p-43">We were concerned that nonlinear relationships could have obscured another better predictor in our initial survey of single-variable correlations. However, after fitting monotonic polynomials of degree 1-4 for all 19 bioclim variables and the first three PC axes, a third-order monotonic polynomial fit for Bio15 still had the lowest AIC (<a id="xref-fig-6-10" class="xref-fig" href="#F6">Extended Data Fig. 2b</a>).</p><p id="p-44">To check whether additional climate variables could further improve our model. We regressed logit-transformed preference and the other bioclimate variables on our third-order fit for Bio15 and tested if residual variation in preference could be explained by the other variables. We again used a two-step procedure to model these effects. We used the R package MonoPoly to fit monotonic polynomials of different degrees to logit-transformed preference probability residuals, and then included the fitted values as an offset (i.e. removed their effects before fitting) in the linear model (logit(prob) ∼ human_pop_density_20km + offset(fitted(monotonic polynomial Bio15)) + offset(fitted(monotonic polynomial BioX))). We found that including a linear Bio18 (precipitation in the warmest quarter) term further reduced AIC (<a id="xref-fig-6-11" class="xref-fig" href="#F6">Extended Data Fig. 2c</a>). Because Bio18 and our fitted Bio15 polynomial relationship were modestly correlated (r=-.29) and we had selected the variables in a stepwise way, we wondered if including them in a single linear model would change our estimates of their effects. In this full, final model (logit(prob) ∼ human_pop_density_20km + fitted(monotonic polynomial Bio15) + Bio18), the estimated coefficient for the fitted Bio15 component was close to 1 (1.04), indicating that fitting the effects sequentially or together didn’t make a major difference, but we used the model where both were fit together going forward. Using both Bio15 and Bio18 as covariates, we again found that using a buffer of 20km for calculating population density yielded a much lower AIC than other buffers (<a id="xref-fig-6-12" class="xref-fig" href="#F6">Extended Data Fig. 2a</a>).</p><p id="p-45">Bio15 and Bio18 both have clear connections to the length and temperature of the dry season - an important factor in survival of dormant <em>Aedes aegypti</em> eggs. We therefore combined them into a single Dry Season Intensity Index by adding together the fitted Bio15 and Bio18 terms for each location. This simple transformation yields a single linear climate term that predicts the host odor preference of mosquitoes from a given location.</p></div><div class="section" id="sec-14"><h3>Morphological analyses</h3><p id="p-46">We pinned 7-29 female mosquitoes from each location (field-collected [AWK, BOA, FCV, KED, KIN, KUM, LBV, LPV, MIN, NGO, OGD, OHI, PKT, THI] or lab colony [ABK, BTT, ENT, GND, KAK, KBO, KWA, LPF, ORL, RAB, SHM, T51, VMB, ZIK]) as previously described (<em>4</em>) and captured light microscope images of the dorsal abdomen under constant lighting and magnification (3X) on a Nikon SMZ1270 microscope. We then estimated the proportion of white scales on the first abdominal tergite by converting each image to 8-bit grayscale in ImageJ, selecting the region of interest, and calculating the area with brightness values above 128. Area estimates were only weakly sensitive to the precise cutoff since the white and black scales differ markedly in brightness; we therefore chose a value in the middle of the range. We used the R function <em>lm</em> to fit a linear model with logit-transformed scaling proportion as the response variable and population of origin as the predictor. We then used the R package <em>emmeans</em> to calculate 95% confidence intervals for each population.</p></div><div class="section" id="sec-15"><h3>Whole genome resequencing and variant calling</h3><p id="p-47">As part of an ongoing 1200 <em>Aedes aegypti</em> genomes project, we extracted gDNA from 480 field-collected mosquitoes using the Chemagic DNA tissue protocol and sequenced them to 15x coverage with PE 151bp reads using the Illumina HiSeqX platform. The sequenced mosquitoes included 397 individuals from 24 sub-Saharan African populations collected for this study, 29 additional individuals from sites in Uganda, Kenya, and Burkina Faso that were not included in the main study, 12 individuals of the domestic form collected in 2009 or 2011 in Rabai, Kenya<sup><a id="xref-ref-10-7" class="xref-bibr" href="#ref-10">10</a></sup>, 20 individuals from Bangkok, Thailand, 18 individuals from Santarem, Brazil, and 4 <em>Ae. mascarensis</em> mosquitoes for use as an outgroup.</p><p id="p-48">We initially mapped all sequence data to the L5 reference genome<sup><a id="xref-ref-38-1" class="xref-bibr" href="#ref-38">38</a></sup>. We identified and removed close relatives from our sample as follows. First, we generated a matrix of relatedness coefficients using the --relatedness2 subprogram from VCFtools<sup><a id="xref-ref-39-1" class="xref-bibr" href="#ref-39">39</a></sup> with a set of randomly selected 109,267 biallelic SNPs (MAF&gt;0.05, &gt;1 read in 90% of individuals) preliminarily called with <em>bcftools</em>. Second, we hierarchically clustered the coefficients using the R function <em>hclust</em> (method=‘average’). Third, we grouped close relatives using the R function <em>cutree</em>, with a relatedness cutoff of 0.05 for African samples (corresponding to first cousin or closer relationships) and 0.2 for non-African populations (corresponding to siblings). The more permissive cutoff was used for non-African populations because they are more inbred/bottlenecked, with many individuals showing cousin-like relationships. Finally, we removed all but one randomly-chosen individual from each group of relatives. This left us with 345 sub-Saharan African <em>Ae. aegypti</em> genomes from our 24 focal study sites, 14 <em>Ae. aegypti</em> from other sites in sub-Saharan Africa, 30 <em>Ae. aegypti</em> genomes from outside continental Africa and 4 <em>Ae. mascarensis</em> genomes. Most relatives came from the same ovitrap (we sequenced more than one individual from a single ovitrap when ovitrap/egg limited). A smaller number came from nearby ovitraps in the same general location. The 14 <em>Ae. aegypti</em> genomes from non-focal sites were used for variant discovery and included in ADMIXTURE and PCA-based analyses (see below) in order to ensure we were sampling as much diversity as possible, but they are not plotted in figures.</p><p id="p-49">We then used three iterative mapping steps to construct an updated African reference based on data from a geographically distributed (Africa only) set of 100 unrelated male mosquitoes (<a id="xref-fig-13-1" class="xref-fig" href="#F13">Extended Data Fig. 9</a>). We chose to use males for the update because the L5 reference was constructed using data from males. In each of three iterative mapping steps, we (1) mapped sequence data from the mapping set to the reference using <em>bwa mem</em> (MAPQ cutoff of 10), (2) called consensus biallelic SNP genotypes using <em>bcftools</em> (<em>“bcftools mpileup -BI</em> | <em>bcftools call -vmOu</em> | <em>bcftools view -v snps -q 0.5:alt1</em> | <em>bcftools norm -Ou -m -</em> | <em>bcftools norm -Oz -d snps”</em>), and (3) substituted the consensus base into our reference sequence using <em>bcftools consensus</em><sup><a id="xref-ref-40-1" class="xref-bibr" href="#ref-40">40</a>,<a id="xref-ref-41-1" class="xref-bibr" href="#ref-41">41</a></sup>. We used PicardTools<sup><a id="xref-ref-42-1" class="xref-bibr" href="#ref-42">42</a></sup> to characterize read mapping quality after each iteration on a set of individuals not used for alternate reference construction (20 individuals; male-female pairs from 10 African populations) (<a id="xref-fig-13-2" class="xref-fig" href="#F13">Extended Data Fig. 9</a>). We used a permissive MAPQ cutoff of 10 for the mapping steps because analyses suggested that high levels of sequence divergence from the L5 reference were disrupting initial alignments (<a id="xref-fig-13-3" class="xref-fig" href="#F13">Extended Data Fig. 9a-d</a>). Finally, we remapped data from all 480 genomes to the updated third-iteration reference; this included non-African and outgroup samples, which also mapped well to the updated reference (<a id="xref-fig-13-4" class="xref-fig" href="#F13">Extended Data Fig. 9e-f</a>). Males and females mapped similarly, except in the region around the sex-determining M-locus (<a id="xref-fig-13-5" class="xref-fig" href="#F13">Extended Data Fig. 9g-h</a>). After remapping, we realigned reads near insertions and deletions to improve variant discovery in these regions using GATK IndelRealigner<sup><a id="xref-ref-43-1" class="xref-bibr" href="#ref-43">43</a></sup>.</p><p id="p-50">We took two different approaches to variant calling - both of which were confined to regions of the genome we inferred to be non-repetitive (repeat masked using the RepeatMasker intervals from the L5 genome) and single copy (mean coverage between 5-30X across individuals). We used the program <em>ANGSD</em><sup><a id="xref-ref-44-1" class="xref-bibr" href="#ref-44">44</a></sup> to calculate population-level allele frequencies and genetic diversity, as well as to carry out genotype-likelihood based analyses (see below) for 161,713,099 biallelic single nucleotide polymorphisms (SNPs, <em>P</em>&lt;10<sup>−6</sup>). We also called individual genotypes for a filtered set of 14,045,728 high quality biallelic SNPs using bcftools. These SNPs were filtered for coverage across the entire sample (covered by at least 1 read in 90% of individuals) and then called for any individual with sample depth &gt; 8 reads and genotype quality score &gt;30. After individual genotype calling we implemented a further filter for the fraction of individuals genotyped (&gt;75% at any given SNP) and minor allele frequency (MAF&gt;1%). We used the same permissive MAPQ cutoff of 10 for variant calling as used for generating the updated reference genome (see above) in order to minimize potential problems with aligning alternate haplotypes. Note that our additional MAF and genotyping filters help protect against SNP calls from false positive alignments. Hard genotype calls (or subsets thereof) were used for <em>ADMIXTURE</em>, principal components analyses (PCA), <em>PCAdapt</em>, and <em>Dsuite</em> analyses (see below).</p></div><div class="section" id="sec-16"><h3>Population structure analyses</h3><p id="p-51">We characterized population structure using two alternative approaches based on a set of 1,000,000 unlinked SNPs selected in <em>PLINK</em> (step size 100, cutoff 0.1, --thin-count 1000000)<sup><a id="xref-ref-45-1" class="xref-bibr" href="#ref-45">45</a></sup>. First, we used <em>ADMIXTURE</em><sup><a id="xref-ref-21-2" class="xref-bibr" href="#ref-21">21</a></sup> to assign individuals to variable numbers of population clusters for K=2-10, with K=3 minimizing cross-validation error (<a id="xref-fig-7-4" class="xref-fig" href="#F7">Extended Data Fig. 3b</a>). Second, we used <em>PLINK</em> to carry out principal components analysis (PCA). One sample from Ngoye, Senegal (NGO) was a clear outlier in ancestry, showing strong affinity with West African generalist populations while all other individuals from this population showed consistent affiliation with human specialists (<a id="xref-fig-3-11" class="xref-fig" href="#F3">Fig. 3c-d</a>); we excluded this putative recent migrant from subsequent <em>F</em><sub><em>ST</em></sub>, <em>PBS, ABBA-BABA, π</em>, and <em>d</em><sub><em>xy</em></sub> analyses involving Ngoye (see below).</p></div><div class="section" id="sec-17"><h3>Gene flow and divergence analyses</h3><p id="p-52">We used <em>ANGSD</em> (subprogram <em>realSFS</em>) to calculate pairwise <em>F</em><sub><em>ST</em></sub> between populations and a custom script to turn these FST values into the Population Branch Statistic (<em>PBS</em>, essentially polarized <em>F</em><sub><em>ST</em></sub>) for NGO, THI, OGD, KED, and OHI, using BTT as a nearby generalist reference population and FCV as an outgroup<sup><a id="xref-ref-22-2" class="xref-bibr" href="#ref-22">22</a></sup>. Using alternative reference and outgroup populations yielded similar results. We used a permutation testing approach to construct a distribution of 5Mb <em>PBS</em> values under the null hypothesis of homogeneous differentiation across the genome. More specifically we shuffled the genomic locations of 10kb windows (which preserves local linkage patterns) with the constraint that shuffling could only occur among regions matched for average genetic diversity (average of BTT and focal population). We then used this null distribution to identify ≥5Mb regions of elevated <em>PBS</em> at a false discovery rate of 0.05 using a two-tailed test. The choice of a 5Mb window restricts us to largish regions that are either relatively new (giving recombination limited time to break up divergent chromosomal regions) or contain several tightly linked loci. However, using smaller regions (e.g. 1Mb or 100kb) identified similar patterns.</p><p id="p-53">We used <em>ABBA-BABA</em>-related statistics to further explore patterns of divergence between populations. These statistics test for an excess of shared derived variation between lineages in order to distinguish gene flow from the incomplete lineage sorting (ILS) that can occur during a simple tree-like branching process. For more on expected genome-wide and locus-specific patterns of derived allele sharing under ILS and gene flow, see<sup><a id="xref-ref-23-2" class="xref-bibr" href="#ref-23">23</a></sup>. First, we used <em>Dsuite</em><sup><a id="xref-ref-46-1" class="xref-bibr" href="#ref-46">46</a></sup> to confirm that the populations in our dataset did not conform to the strict tree-like model, which is expected since all populations belong to the same species and almost certainly exchange genes. Indeed, we strongly rejected the null tree-like hypothesis (block-jacknife <em>P</em>&lt;10<sup>−7</sup>) for all three-population trees with <em>Ae. mascarensis</em> as an outgroup.</p><p id="p-54">We then explored potential heterogeneity in gene flow across the genome using the <em>f</em><sub><em>D</em></sub> statistic (calculated in 5Mb windows with a 10kb step). The <em>f</em><sub><em>D</em></sub> statistic uses shared derived genetic variation to estimate the fraction of ancestry at a specific locus derived from gene flow between branches in a specified tree<sup><a id="xref-ref-23-3" class="xref-bibr" href="#ref-23">23</a></sup>. We calculated <em>f</em><sub><em>D</em></sub> from <em>ANGSD</em> population allele frequencies using a custom python script for the tree (BTT, X; BKK, <em>mascarensis</em>) to identify regions of the genome showing elevated levels of shared derived variation between the focal population (X=NGO, THI, or OGD) and non-African human specialists (BKK). We do not think such shared derived variation is necessarily derived from introgression back to Africa from non-African populations. Instead, it may reflect relationships present in ancestral populations, before <em>Ae. aegypti</em> left Africa. Differentiating between these two hypotheses is out-of-scope for this study but will be addressed in a future study incorporating much more genomic data from outside Africa. Regardless, we expect shared derived variation between human-preferring populations within and outside Africa to be present in regions that code for human-adaptive traits and thus experience reduced gene flow between human- and animal-preferring populations within Africa. We used Fisher’s exact test to test whether the top 10% of non-overlapping 5Mb <em>f</em><sub><em>D</em></sub> windows were significantly enriched in our <em>PBS</em> outlier regions for each focal population.</p><p id="p-55">To help interpret measures of between-population divergence (i.e. <em>F</em><sub><em>ST</em></sub> and <em>PBS</em>), we used <em>ANGSD</em> to estimate levels of genetic diversity (<em>π</em>) across the genome for each population and the perl script getDxy.pl (modified to skip variant sites not covered in one population) from ngsTools<sup><a id="xref-ref-47-1" class="xref-bibr" href="#ref-47">47</a></sup> to calculate <em>d</em><sub><em>xy</em></sub> (<a id="xref-fig-9-2" class="xref-fig" href="#F9">Extended Data Fig. 5</a>). We also calculated normalized <em>d</em><sub><em>xy</em></sub> (<a id="xref-fig-9-3" class="xref-fig" href="#F9">Extended Data Fig. 5c</a>) by dividing <em>d</em><sub><em>xy</em></sub> for a given population pair by mean <em>d</em><sub><em>xy</em></sub> for all pairs of NGO, THI, OGD, BTT, and FCV. We calculated normalized <em>π</em> (<a id="xref-fig-9-4" class="xref-fig" href="#F9">Extended Data Fig. 5c</a>) by dividing population <em>π</em> by mean <em>π</em> across all populations.</p><p id="p-56">We used PCAdapt<sup><a id="xref-ref-25-2" class="xref-bibr" href="#ref-25">25</a></sup> to test whether specific SNPs were associated with specialist ancestry across the subset of high-quality, biallelic SNPs from our hard-called set that had minor allele frequency &gt;0.05 (n=5,369,564 SNPs) (PCAdapt parameters: K=3, method “componentwise”, and LD clumping with a size of 200 and a cutoff of 0.1).</p></div><div class="section" id="sec-18"><h3>Climate and population projections</h3><p id="p-57">We predicted future changes in host odor preference at each sampling location by plugging climate and human population density change projections for 2050 into our final, fitted, ecological model – including human population density (calculated within 20km radius), precipitation seasonality (Bio15, third degree monotonic polynomial) and precipitation in the warmest quarter (Bio18, linear).</p><p id="p-58">Climate change projection data came from the Coupled Model Intercomparison Project Phase 5 (CMIP5) based on Representative Concentration Pathway 8.5 (RCP8.5) scenarios<sup><a id="xref-ref-27-2" class="xref-bibr" href="#ref-27">27</a></sup>. RCP8.5 is considered the business-as-usual scenario for future greenhouse gas concentrations, reflecting minimal mitigation efforts. The CMIP5 effort contributed to the International Panel on Climate Change (IPCC) Fifth Assessment Report. A new modeling effort, CMIP6, is currently underway but complete data are not yet available. Projected climate data from a global climate model (GCM) cannot be directly compared to present-day observational climate data due to model biases and measurement error. Failing to account for these biases can result in misinterpreting structural differences between the two datasets as potential climate change effects. Instead, the projection data must first be bias-corrected by calculating the relative or absolute change between current and future climates for the variable of interest, using solely the GCM output. This relative or absolute change can then be applied to observational data. Depending on the resolution of the observational climate data, projections may also be downscaled – i.e. the resolution of the model output improved. The Worldclim projection data has undergone both downscaling and bias-correction processes such that it can be compared with the observational data used in our present-day analysis. Absolute changes were used for temperature and relative changes were used for precipitation. Further details of these processes are available at <a href="https://www.worldclim.org/downscaling">https://www.worldclim.org/downscaling</a>. There is relatively high agreement across models in terms of the spatial distribution of projected Bio15 and Bio18 changes (<a id="xref-fig-11-2" class="xref-fig" href="#F11">Extended Data Fig. 7</a>).</p><p id="p-59">Population projection data came from the United Nations medium-variant scenario<sup><a id="xref-ref-48-1" class="xref-bibr" href="#ref-48">48</a></sup>. This scenario assumes existing high fertility populations will experience a fertility decline over the coming century, as economic development increases. Despite falling fertility, sub-Saharan Africa is expected to see an increase in the total number of births over the next several decades relative to the recent past. High birth numbers coupled with increasing life expectancy will lead to 1.05 billion increase in population in sub-Saharan African countries by 2050, 52% of the additional global population in this timeline<sup><a id="xref-ref-49-1" class="xref-bibr" href="#ref-49">49</a></sup>. We used urban and rural, medium-variant projections for each country and calculated growth rates by comparing 2050 numbers with those from 2015. Note, our field collections were conducted between 2015 and 2018 (mostly 2017-2018) making 2015 numbers more applicable than any other available estimates. We then applied these growth rates to the baseline population data for each location. Urban and rural locations were considered separately because urban populations are expected to grow at a faster rate than rural populations over this time period. Urban populations were defined as those with current population density &gt; 400 humans/km<sup>2</sup> calculated with a 20km buffer (<a id="xref-table-wrap-1-5" class="xref-table" href="#T1">Extended Data Table 1</a>). These populations were all from areas that we observed to be dominated by human structures and activities (<a id="xref-table-wrap-1-6" class="xref-table" href="#T1">Extended Data Table 1</a>). A few intermediate density locations fell below this cutoff and were classified as rural. More specifically, KWA, OHI and NGO are rural towns, while ABK and SHM are wild areas on the far outskirts of what most would consider urban areas (densities 173-288 humans/km<sup>2</sup>; <a id="xref-table-wrap-1-7" class="xref-table" href="#T1">Extended Data Table 1</a>). The other sites classified as rural had much lower densities (1-67 humans/km<sup>2</sup>; <a id="xref-table-wrap-1-8" class="xref-table" href="#T1">Extended Data Table 1</a>).</p></div></div><div class="section" id="sec-19"><h2 class="">Data and code availability</h2><p id="p-60">Raw genomic data are deposited in the NCBI SRA under the accession code PRJNA602495 and will be made available upon publication. Other raw data and scripts are available at github.com/noahrose.</p></div><div class="section" id="sec-20"><h2 class="">Funding</h2><p id="p-62">This work was funded by Pew Scholars, Searle Scholars, Klingenstein-Simons, and Rosalind Franklin/Gruber Foundation awards (to C.S.M.), the National Institutes of Health (R00DC012069 to C.S.M.; R01AI101112 and U01AI115595 to J.R.P.), a Helen Hay Whitney Postdoctoral Fellowship (to N.H.R.) and undergraduate thesis funding from the Princeton University Department of Ecology and Evolutionary Biology and African Studies Program (to N.I and E.G.E.); Verily Life Sciences funded all genome sequencing. The research was also supported by the New York Stem Cell Foundation. C.S.M. is a New York Stem Cell Foundation – Robertson Investigator</p></div><div class="section" id="sec-21"><h2 class="">Author contributions</h2><p id="p-63">N.H.R., M.S., A.B., J.L., D.A., O.B.A., N.I., S.O., J.A., J.P.M., R.S., and C.S.M. planned and carried out field collections. N.H.R., A.L.K., M.S., and J.R.P. established laboratory colonies. N.H.R, N.I., and A.L.K carried out behavioral experiments. N.H.R. and E.G.E. carried out morphological quantification. B.J.W. and J.E.C. carried out genome sequencing and alignment. J.R.P. and A.G.S. provided non-African samples for sequencing. N.H.R., B.J.W., J.E.C. and C.S.M analyzed genomic data. R.E.B. modeled future conditions. N.H.R. and C.S.M. conceived the project and wrote the manuscript, with input at all stages from many other authors.</p></div><div class="section" id="sec-22"><h2 class="">Competing interests</h2><p id="p-64">the Authors declare no competing interests.</p></div><div class="section" id="sec-23"><h2 class="">Extended Data</h2><div id="F5" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/02/13/2020.02.12.939041/F5.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Additional details for host preference assays. a-b, Raw preference indices for all trials plotted by location (a) or trial day (b). Color indicates location and symbols indicate trial day. c, Preference estimates for four populations derived from tests with alternative human subjects (top color of each circle) and animal species (bottom color of each circle) (n=3-8 trials, each including 37-110 females per colony/host combination). Data for one human versus either of two guinea pigs (red/light blue and red/dark blue) come from same experiments summarized in Fig. 1d. Data for same human versus quail (red/green) and second human versus guinea pig (pink/dark blue) are new. d, Host preference was strongly correlated for replicate colonies from the same location (Pearson R2=0.60, P=1.5&#xD7;10&#x2212;5). e, Preference and overall response rate (rate of choosing either host) were strongly correlated across locations (R2=0.73, P=4.4&#xD7;10&#x2212;9). f, Overall response rates for each location plotted and analyzed as in Fig. 1d." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-2014705" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;&lt;span class=&quot;caption-title&quot;&gt;Additional details for host preference assays.&lt;/span&gt; &lt;strong&gt;a&lt;/strong&gt;-&lt;strong&gt;b&lt;/strong&gt;, Raw preference indices for all trials plotted by location (a) or trial day (b). Color indicates location and symbols indicate trial day. &lt;strong&gt;c&lt;/strong&gt;, Preference estimates for four populations derived from tests with alternative human subjects (top color of each circle) and animal species (bottom color of each circle) (n=3-8 trials, each including 37-110 females per colony/host combination). Data for one human versus either of two guinea pigs (red/light blue and red/dark blue) come from same experiments summarized in Fig. 1d. Data for same human versus quail (red/green) and second human versus guinea pig (pink/dark blue) are new. &lt;strong&gt;d&lt;/strong&gt;, Host preference was strongly correlated for replicate colonies from the same location (Pearson &lt;em&gt;R&lt;/em&gt;&lt;sup&gt;&lt;em&gt;2&lt;/em&gt;&lt;/sup&gt;=0.60, &lt;em&gt;P&lt;/em&gt;=1.5&#xD7;10&lt;sup&gt;&#x2212;5&lt;/sup&gt;). &lt;strong&gt;e&lt;/strong&gt;, Preference and overall response rate (rate of choosing either host) were strongly correlated across locations (&lt;em&gt;R&lt;/em&gt;&lt;sup&gt;&lt;em&gt;2&lt;/em&gt;&lt;/sup&gt;=0.73, &lt;em&gt;P&lt;/em&gt;=4.4&#xD7;10&lt;sup&gt;&#x2212;9&lt;/sup&gt;). &lt;strong&gt;f&lt;/strong&gt;, Overall response rates for each location plotted and analyzed as in Fig. 1d.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Extended Data Fig. 1." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2020/02/13/2020.02.12.939041/F5.medium.gif" width="440" height="406"/><noscript><img class="highwire-fragment fragment-image" alt="Extended Data Fig. 1." src="https://www.biorxiv.org/content/biorxiv/early/2020/02/13/2020.02.12.939041/F5.medium.gif" width="440" height="406"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/02/13/2020.02.12.939041/F5.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Extended Data Fig. 1." data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/02/13/2020.02.12.939041/F5.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Extended Data Fig. 1.</span> <span class="caption-title">Additional details for host preference assays.</span><p id="p-65" class="first-child"><strong>a</strong>-<strong>b</strong>, Raw preference indices for all trials plotted by location (a) or trial day (b). Color indicates location and symbols indicate trial day. <strong>c</strong>, Preference estimates for four populations derived from tests with alternative human subjects (top color of each circle) and animal species (bottom color of each circle) (n=3-8 trials, each including 37-110 females per colony/host combination). Data for one human versus either of two guinea pigs (red/light blue and red/dark blue) come from same experiments summarized in <a id="xref-fig-1-14" class="xref-fig" href="#F1">Fig. 1d</a>. Data for same human versus quail (red/green) and second human versus guinea pig (pink/dark blue) are new. <strong>d</strong>, Host preference was strongly correlated for replicate colonies from the same location (Pearson <em>R</em><sup><em>2</em></sup>=0.60, <em>P</em>=1.5×10<sup>−5</sup>). <strong>e</strong>, Preference and overall response rate (rate of choosing either host) were strongly correlated across locations (<em>R</em><sup><em>2</em></sup>=0.73, <em>P</em>=4.4×10<sup>−9</sup>). <strong>f</strong>, Overall response rates for each location plotted and analyzed as in <a id="xref-fig-1-15" class="xref-fig" href="#F1">Fig. 1d</a>.</p><div class="sb-div caption-clear"></div></div></div><div id="F6" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/02/13/2020.02.12.939041/F6.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Additional details for modelling effects of human population density and climate on host preference. a, Akaike Information Criterion (AIC) for linear models of host preference as a function of various combinations of region (latitude/longitude), specific climate variables, and human population density estimated across different scales (x-axis). All models incorporating population density perform better than a null model that includes region only (Lat*Long, grey line). Dotted vertical line highlights the spatial scale at which population density best predicted behavior in our final model (purple). b, AIC for linear models of host preference as a function of human population density plus individual bioclimate variables. Precipitation seasonality (pink) provided the best fit, especially when modeled as a monotonic cubic polynomial. c, After removing the effects of population density and precipitation seasonality, a linear effect of precipitation in the warmest quarter (pink) further decreases AIC. No further variable improves the model after including this second climate effect. d, Map of collection localities equivalent to Fig. 1c except on background map of variation in precipitation in the warmest quarter (rather than precipitation seasonality)." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-2014705" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;&lt;span class=&quot;caption-title&quot;&gt;Additional details for modelling effects of human population density and climate on host preference.&lt;/span&gt; &lt;strong&gt;a&lt;/strong&gt;, Akaike Information Criterion (AIC) for linear models of host preference as a function of various combinations of region (latitude/longitude), specific climate variables, and human population density estimated across different scales (x-axis). All models incorporating population density perform better than a null model that includes region only (Lat*Long, grey line). Dotted vertical line highlights the spatial scale at which population density best predicted behavior in our final model (purple). &lt;strong&gt;b&lt;/strong&gt;, AIC for linear models of host preference as a function of human population density plus individual bioclimate variables. Precipitation seasonality (pink) provided the best fit, especially when modeled as a monotonic cubic polynomial. &lt;strong&gt;c&lt;/strong&gt;, After removing the effects of population density and precipitation seasonality, a linear effect of precipitation in the warmest quarter (pink) further decreases AIC. No further variable improves the model after including this second climate effect. &lt;strong&gt;d&lt;/strong&gt;, Map of collection localities equivalent to Fig. 1c except on background map of variation in precipitation in the warmest quarter (rather than precipitation seasonality).&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Extended Data Fig. 2." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2020/02/13/2020.02.12.939041/F6.medium.gif" width="440" height="358"/><noscript><img class="highwire-fragment fragment-image" alt="Extended Data Fig. 2." src="https://www.biorxiv.org/content/biorxiv/early/2020/02/13/2020.02.12.939041/F6.medium.gif" width="440" height="358"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/02/13/2020.02.12.939041/F6.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Extended Data Fig. 2." data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/02/13/2020.02.12.939041/F6.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Extended Data Fig. 2.</span> <span class="caption-title">Additional details for modelling effects of human population density and climate on host preference.</span><p id="p-66" class="first-child"><strong>a</strong>, Akaike Information Criterion (AIC) for linear models of host preference as a function of various combinations of region (latitude/longitude), specific climate variables, and human population density estimated across different scales (x-axis). All models incorporating population density perform better than a null model that includes region only (Lat*Long, grey line). Dotted vertical line highlights the spatial scale at which population density best predicted behavior in our final model (purple). <strong>b</strong>, AIC for linear models of host preference as a function of human population density plus individual bioclimate variables. Precipitation seasonality (pink) provided the best fit, especially when modeled as a monotonic cubic polynomial. <strong>c</strong>, After removing the effects of population density and precipitation seasonality, a linear effect of precipitation in the warmest quarter (pink) further decreases AIC. No further variable improves the model after including this second climate effect. <strong>d</strong>, Map of collection localities equivalent to <a id="xref-fig-1-16" class="xref-fig" href="#F1">Fig. 1c</a> except on background map of variation in precipitation in the warmest quarter (rather than precipitation seasonality).</p><div class="sb-div caption-clear"></div></div></div><div id="F7" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/02/13/2020.02.12.939041/F7.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Additional details for analyses of population structure. a, Ancestry proportions from ADMIXTURE analysis for K=2-6. b, Cross-validation error for analysis in (a) is minimized for K=3. c-d, Principal components analysis shows three main clusters, with sub-structure corresponding to the clusters found at higher K values (c) and, remarkably, to behavior (d). The non-African and Rabai domestic samples are shown in grey in panel (d) because their behavior was not tested in this study. However, we expect them to be strongly human-preferring based on previous work 4 as well as testing of the T51 colony (from Thailand) and ORL colony (likely of largely American descent) in this study (Fig. 1d)." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-2014705" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;&lt;span class=&quot;caption-title&quot;&gt;Additional details for analyses of population structure.&lt;/span&gt; &lt;strong&gt;a&lt;/strong&gt;, Ancestry proportions from ADMIXTURE analysis for K=2-6. &lt;strong&gt;b&lt;/strong&gt;, Cross-validation error for analysis in (a) is minimized for K=3. &lt;strong&gt;c&lt;/strong&gt;-&lt;strong&gt;d&lt;/strong&gt;, Principal components analysis shows three main clusters, with sub-structure corresponding to the clusters found at higher K values (c) and, remarkably, to behavior (d). The non-African and Rabai domestic samples are shown in grey in panel (d) because their behavior was not tested in this study. However, we expect them to be strongly human-preferring based on previous work &lt;sup&gt;4&lt;/sup&gt; as well as testing of the T51 colony (from Thailand) and ORL colony (likely of largely American descent) in this study (Fig. 1d).&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Extended Data Fig. 3." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2020/02/13/2020.02.12.939041/F7.medium.gif" width="440" height="431"/><noscript><img class="highwire-fragment fragment-image" alt="Extended Data Fig. 3." src="https://www.biorxiv.org/content/biorxiv/early/2020/02/13/2020.02.12.939041/F7.medium.gif" width="440" height="431"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/02/13/2020.02.12.939041/F7.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Extended Data Fig. 3." data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/02/13/2020.02.12.939041/F7.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Extended Data Fig. 3.</span> <span class="caption-title">Additional details for analyses of population structure.</span><p id="p-67" class="first-child"><strong>a</strong>, Ancestry proportions from ADMIXTURE analysis for K=2-6. <strong>b</strong>, Cross-validation error for analysis in (a) is minimized for K=3. <strong>c</strong>-<strong>d</strong>, Principal components analysis shows three main clusters, with sub-structure corresponding to the clusters found at higher K values (c) and, remarkably, to behavior (d). The non-African and Rabai domestic samples are shown in grey in panel (d) because their behavior was not tested in this study. However, we expect them to be strongly human-preferring based on previous work <sup><a id="xref-ref-4-3" class="xref-bibr" href="#ref-4">4</a></sup> as well as testing of the T51 colony (from Thailand) and ORL colony (likely of largely American descent) in this study (<a id="xref-fig-1-17" class="xref-fig" href="#F1">Fig. 1d</a>).</p><div class="sb-div caption-clear"></div></div></div><div id="F8" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/02/13/2020.02.12.939041/F8.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Mindin, Senegal (MIN) shows signs of a selective sweep and/or structural rearrangement in the chromosome 1 outlier region. a, Map of sites addressed in this figure (taken from Fig. 1c with precipitation seasonality scale adjusted to better illustrate climate gradient in Senegal and Burkina Faso. Mindin is an animal-seeking population at the bleeding edge of the geographic transition to human-seeking in the Sahel region of Senegal. b, Mosquitoes from Mindin show elevated divergence from human-seeking populations at the distal end of chromosome 1, as do other animal-seeking populations (e.g. BTT; Extended Data Fig. 5b). Grey shading indicates PBS outlier regions. c, Unexpectedly, mosquitoes from Mindin also show step-like, elevated divergence in this chromosomal region from other animal-seeking populations, despite having similar host preference. Elevated divergence in (c) is a function of variably elevated absolute differentiation (dxy) (d) and markedly reduced genetic diversity (&#x3C0;) in Mindin (e)." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-2014705" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;&lt;span class=&quot;caption-title&quot;&gt;Mindin, Senegal (MIN) shows signs of a selective sweep and/or structural rearrangement in the chromosome 1 outlier region.&lt;/span&gt; &lt;strong&gt;a&lt;/strong&gt;, Map of sites addressed in this figure (taken from Fig. 1c with precipitation seasonality scale adjusted to better illustrate climate gradient in Senegal and Burkina Faso. Mindin is an animal-seeking population at the bleeding edge of the geographic transition to human-seeking in the Sahel region of Senegal. &lt;strong&gt;b&lt;/strong&gt;, Mosquitoes from Mindin show elevated divergence from human-seeking populations at the distal end of chromosome 1, as do other animal-seeking populations (e.g. BTT; Extended Data Fig. 5b). Grey shading indicates &lt;em&gt;PBS&lt;/em&gt; outlier regions. &lt;strong&gt;c&lt;/strong&gt;, Unexpectedly, mosquitoes from Mindin also show step-like, elevated divergence in this chromosomal region from other animal-seeking populations, despite having similar host preference. Elevated divergence in (c) is a function of variably elevated absolute differentiation (&lt;em&gt;d&lt;/em&gt;&lt;sub&gt;&lt;em&gt;xy&lt;/em&gt;&lt;/sub&gt;) (&lt;strong&gt;d&lt;/strong&gt;) and markedly reduced genetic diversity (&lt;em&gt;&#x3C0;&lt;/em&gt;) in Mindin (&lt;strong&gt;e&lt;/strong&gt;).&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Extended Data Fig. 4." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2020/02/13/2020.02.12.939041/F8.medium.gif" width="440" height="413"/><noscript><img class="highwire-fragment fragment-image" alt="Extended Data Fig. 4." src="https://www.biorxiv.org/content/biorxiv/early/2020/02/13/2020.02.12.939041/F8.medium.gif" width="440" height="413"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/02/13/2020.02.12.939041/F8.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Extended Data Fig. 4." data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/02/13/2020.02.12.939041/F8.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Extended Data Fig. 4.</span> <span class="caption-title">Mindin, Senegal (MIN) shows signs of a selective sweep and/or structural rearrangement in the chromosome 1 outlier region.</span><p id="p-68" class="first-child"><strong>a</strong>, Map of sites addressed in this figure (taken from <a id="xref-fig-1-18" class="xref-fig" href="#F1">Fig. 1c</a> with precipitation seasonality scale adjusted to better illustrate climate gradient in Senegal and Burkina Faso. Mindin is an animal-seeking population at the bleeding edge of the geographic transition to human-seeking in the Sahel region of Senegal. <strong>b</strong>, Mosquitoes from Mindin show elevated divergence from human-seeking populations at the distal end of chromosome 1, as do other animal-seeking populations (e.g. BTT; <a id="xref-fig-9-5" class="xref-fig" href="#F9">Extended Data Fig. 5b</a>). Grey shading indicates <em>PBS</em> outlier regions. <strong>c</strong>, Unexpectedly, mosquitoes from Mindin also show step-like, elevated divergence in this chromosomal region from other animal-seeking populations, despite having similar host preference. Elevated divergence in (c) is a function of variably elevated absolute differentiation (<em>d</em><sub><em>xy</em></sub>) (<strong>d</strong>) and markedly reduced genetic diversity (<em>π</em>) in Mindin (<strong>e</strong>).</p><div class="sb-div caption-clear"></div></div></div><div id="F9" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/02/13/2020.02.12.939041/F9.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Additional details for analyses of genomic divergence between human- and animal-seeking populations. a, Polarized genomic divergence (Population Branch Statistic, PBS) as shown in Fig. 3f with the addition of a human specialist population from outside Africa (Bangkok, BKK). BKK shows elevated differentiation across the genome. b, Non-polarized divergence (FST) from the animal-seeking population Bantata (BTT). Human-seeking populations show elevated differentiation in PBS outlier regions. Animal-seeking populations show minimal differentiation even when geographically distant from BTT (e.g. Ouahigouya, OHI). BKK again shows higher differentiation across the genome. Patterns in (b) are driven by variation in (c) absolute genetic differentiation (dxy) and (d) within-population diversity (&#x3C0;) in the same populations. e, Analyses of normalized dxy and normalized &#x3C0; indicate that human-seeking populations have both elevated absolute divergence and reduced diversity within PBS outlier regions (pink, green, black), suggesting that both sweeps and subsequent selection against gene flow have driven divergence in these regions. Conversely, human-seeking populations inside Africa show reduced levels of absolute divergence from the non-African population (BKK) in these regions (last three subpanels), consistent with shorter coalescence times and a common origin." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-2014705" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;&lt;span class=&quot;caption-title&quot;&gt;Additional details for analyses of genomic divergence between human- and animal-seeking populations.&lt;/span&gt; &lt;strong&gt;a&lt;/strong&gt;, Polarized genomic divergence (Population Branch Statistic, &lt;em&gt;PBS&lt;/em&gt;) as shown in Fig. 3f with the addition of a human specialist population from outside Africa (Bangkok, BKK). BKK shows elevated differentiation across the genome. &lt;strong&gt;b&lt;/strong&gt;, Non-polarized divergence (&lt;em&gt;F&lt;/em&gt;&lt;sub&gt;&lt;em&gt;ST&lt;/em&gt;&lt;/sub&gt;) from the animal-seeking population Bantata (BTT). Human-seeking populations show elevated differentiation in &lt;em&gt;PBS&lt;/em&gt; outlier regions. Animal-seeking populations show minimal differentiation even when geographically distant from BTT (e.g. Ouahigouya, OHI). BKK again shows higher differentiation across the genome. Patterns in (b) are driven by variation in (&lt;strong&gt;c&lt;/strong&gt;) absolute genetic differentiation (&lt;em&gt;d&lt;/em&gt;&lt;sub&gt;&lt;em&gt;xy&lt;/em&gt;&lt;/sub&gt;) and (&lt;strong&gt;d&lt;/strong&gt;) within-population diversity (&lt;em&gt;&#x3C0;&lt;/em&gt;) in the same populations. &lt;strong&gt;e&lt;/strong&gt;, Analyses of normalized &lt;em&gt;d&lt;/em&gt;&lt;sub&gt;&lt;em&gt;xy&lt;/em&gt;&lt;/sub&gt; and normalized &lt;em&gt;&#x3C0;&lt;/em&gt; indicate that human-seeking populations have both elevated absolute divergence and reduced diversity within &lt;em&gt;PBS&lt;/em&gt; outlier regions (pink, green, black), suggesting that both sweeps and subsequent selection against gene flow have driven divergence in these regions. Conversely, human-seeking populations inside Africa show reduced levels of absolute divergence from the non-African population (BKK) in these regions (last three subpanels), consistent with shorter coalescence times and a common origin.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Extended Data Fig. 5." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2020/02/13/2020.02.12.939041/F9.medium.gif" width="339" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Extended Data Fig. 5." src="https://www.biorxiv.org/content/biorxiv/early/2020/02/13/2020.02.12.939041/F9.medium.gif" width="339" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/02/13/2020.02.12.939041/F9.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Extended Data Fig. 5." data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/02/13/2020.02.12.939041/F9.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Extended Data Fig. 5.</span> <span class="caption-title">Additional details for analyses of genomic divergence between human- and animal-seeking populations.</span><p id="p-69" class="first-child"><strong>a</strong>, Polarized genomic divergence (Population Branch Statistic, <em>PBS</em>) as shown in <a id="xref-fig-3-12" class="xref-fig" href="#F3">Fig. 3f</a> with the addition of a human specialist population from outside Africa (Bangkok, BKK). BKK shows elevated differentiation across the genome. <strong>b</strong>, Non-polarized divergence (<em>F</em><sub><em>ST</em></sub>) from the animal-seeking population Bantata (BTT). Human-seeking populations show elevated differentiation in <em>PBS</em> outlier regions. Animal-seeking populations show minimal differentiation even when geographically distant from BTT (e.g. Ouahigouya, OHI). BKK again shows higher differentiation across the genome. Patterns in (b) are driven by variation in (<strong>c</strong>) absolute genetic differentiation (<em>d</em><sub><em>xy</em></sub>) and (<strong>d</strong>) within-population diversity (<em>π</em>) in the same populations. <strong>e</strong>, Analyses of normalized <em>d</em><sub><em>xy</em></sub> and normalized <em>π</em> indicate that human-seeking populations have both elevated absolute divergence and reduced diversity within <em>PBS</em> outlier regions (pink, green, black), suggesting that both sweeps and subsequent selection against gene flow have driven divergence in these regions. Conversely, human-seeking populations inside Africa show reduced levels of absolute divergence from the non-African population (BKK) in these regions (last three subpanels), consistent with shorter coalescence times and a common origin.</p><div class="sb-div caption-clear"></div></div></div><div id="F10" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/02/13/2020.02.12.939041/F10.large.jpg?width=800&amp;height=600&amp;carousel=1" title="PCAdapt analysis identifies genome-wide variants associated with human specialist ecology. a, The second principle component from a PCAdapt analysis of African genomes shows a striking correlation with host preference. This result is similar to the outcome of the principal components analysis of population structure shown in Extended Data Fig. 3d, where the second PC is clearly also strongly correlated with behavior within Africa. b, Thousands of SNPs were more strongly associated with PC2 than expected under neutral evolution, suggesting positive selection (n=16,782 SNPs in red, Bonferroni-adjusted P0.01 not plotted." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-2014705" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;&lt;span class=&quot;caption-title&quot;&gt;PCAdapt analysis identifies genome-wide variants associated with human specialist ecology.&lt;/span&gt; &lt;strong&gt;a&lt;/strong&gt;, The second principle component from a PCAdapt analysis of African genomes shows a striking correlation with host preference. This result is similar to the outcome of the principal components analysis of population structure shown in Extended Data Fig. 3d, where the second PC is clearly also strongly correlated with behavior within Africa. &lt;strong&gt;b&lt;/strong&gt;, Thousands of SNPs were more strongly associated with PC2 than expected under neutral evolution, suggesting positive selection (n=16,782 SNPs in red, Bonferroni-adjusted &lt;em&gt;P&lt;/em&gt;&lt;0.05). Consistent with analyses of genomic divergence between human- and animal-preferring populations in West Africa (Fig. 3f), the most significant SNPs are concentrated in &lt;em&gt;PBS&lt;/em&gt; outlier regions (grey shading). Points with unadjusted &lt;em&gt;P&lt;/em&gt;&gt;0.01 not plotted.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Extended Data Fig. 6." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2020/02/13/2020.02.12.939041/F10.medium.gif" width="440" height="236"/><noscript><img class="highwire-fragment fragment-image" alt="Extended Data Fig. 6." src="https://www.biorxiv.org/content/biorxiv/early/2020/02/13/2020.02.12.939041/F10.medium.gif" width="440" height="236"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/02/13/2020.02.12.939041/F10.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Extended Data Fig. 6." data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/02/13/2020.02.12.939041/F10.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Extended Data Fig. 6.</span> <span class="caption-title">PCAdapt analysis identifies genome-wide variants associated with human specialist ecology.</span><p id="p-70" class="first-child"><strong>a</strong>, The second principle component from a PCAdapt analysis of African genomes shows a striking correlation with host preference. This result is similar to the outcome of the principal components analysis of population structure shown in <a id="xref-fig-7-5" class="xref-fig" href="#F7">Extended Data Fig. 3d</a>, where the second PC is clearly also strongly correlated with behavior within Africa. <strong>b</strong>, Thousands of SNPs were more strongly associated with PC2 than expected under neutral evolution, suggesting positive selection (n=16,782 SNPs in red, Bonferroni-adjusted <em>P</em>&lt;0.05). Consistent with analyses of genomic divergence between human- and animal-preferring populations in West Africa (<a id="xref-fig-3-13" class="xref-fig" href="#F3">Fig. 3f</a>), the most significant SNPs are concentrated in <em>PBS</em> outlier regions (grey shading). Points with unadjusted <em>P</em>&gt;0.01 not plotted.</p><div class="sb-div caption-clear"></div></div></div><div id="F11" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/02/13/2020.02.12.939041/F11.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Projected change in preference-related climate variables from 2000 to 2050. a, Mean projected change in precipitation seasonality across all climate models. b, Individual model projections for precipitation seasonality. Note that scale differs from panel A. c-d, same as a-b except for precipitation in the warmest quarter." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-2014705" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;&lt;span class=&quot;caption-title&quot;&gt;Projected change in preference-related climate variables from 2000 to 2050.&lt;/span&gt; &lt;strong&gt;a&lt;/strong&gt;, Mean projected change in precipitation seasonality across all climate models. &lt;strong&gt;b&lt;/strong&gt;, Individual model projections for precipitation seasonality. Note that scale differs from panel A. &lt;strong&gt;c&lt;/strong&gt;-&lt;strong&gt;d&lt;/strong&gt;, same as a-b except for precipitation in the warmest quarter.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Extended Data Fig. 7." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2020/02/13/2020.02.12.939041/F11.medium.gif" width="363" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Extended Data Fig. 7." src="https://www.biorxiv.org/content/biorxiv/early/2020/02/13/2020.02.12.939041/F11.medium.gif" width="363" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/02/13/2020.02.12.939041/F11.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Extended Data Fig. 7." data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/02/13/2020.02.12.939041/F11.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Extended Data Fig. 7.</span> <span class="caption-title">Projected change in preference-related climate variables from 2000 to 2050.</span><p id="p-71" class="first-child"><strong>a</strong>, Mean projected change in precipitation seasonality across all climate models. <strong>b</strong>, Individual model projections for precipitation seasonality. Note that scale differs from panel A. <strong>c</strong>-<strong>d</strong>, same as a-b except for precipitation in the warmest quarter.</p><div class="sb-div caption-clear"></div></div></div><div id="F12" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/02/13/2020.02.12.939041/F12.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Additional details on host preference projections for 2050. Expected effects of changes in human population density (black) and climate (grey) are shown separately. Net effects are shown in red (increases) or blue (decreases). Note that publicly available human population growth rate projections are different for urban (a) and rural (b) sites. We classified sites as urban if current population density exceeded 300 humans km-2." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-2014705" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;&lt;span class=&quot;caption-title&quot;&gt;Additional details on host preference projections for 2050.&lt;/span&gt; Expected effects of changes in human population density (black) and climate (grey) are shown separately. Net effects are shown in red (increases) or blue (decreases). Note that publicly available human population growth rate projections are different for urban (&lt;strong&gt;a&lt;/strong&gt;) and rural (&lt;strong&gt;b&lt;/strong&gt;) sites. We classified sites as urban if current population density exceeded 300 humans km&lt;sup&gt;-2&lt;/sup&gt;.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Extended Data Fig. 8." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2020/02/13/2020.02.12.939041/F12.medium.gif" width="440" height="190"/><noscript><img class="highwire-fragment fragment-image" alt="Extended Data Fig. 8." src="https://www.biorxiv.org/content/biorxiv/early/2020/02/13/2020.02.12.939041/F12.medium.gif" width="440" height="190"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/02/13/2020.02.12.939041/F12.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Extended Data Fig. 8." data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/02/13/2020.02.12.939041/F12.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Extended Data Fig. 8.</span> <span class="caption-title">Additional details on host preference projections for 2050.</span><p id="p-72" class="first-child">Expected effects of changes in human population density (black) and climate (grey) are shown separately. Net effects are shown in red (increases) or blue (decreases). Note that publicly available human population growth rate projections are different for urban (<strong>a</strong>) and rural (<strong>b</strong>) sites. We classified sites as urban if current population density exceeded 300 humans km<sup>-2</sup>.</p><div class="sb-div caption-clear"></div></div></div><div id="F13" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/02/13/2020.02.12.939041/F13.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Construction of an alternate reference genome improved read mapping and variant discovery. a-d, Comparison of four mapping statistics across three iterative updates to the AaegL5 genome for a panel of 20 individuals (one male and one female from each of 10 African populations): a, Total number of reads mapped with MAPQ&gt;20. b, fraction of mismatched bases. c, mean number of polymorphic sites per individual covered by at least 10 reads with MAPQ&gt;10 (polymorphic sites defined as those with at least 5 alternate allele reads present among summed reads from all samples). d, average number of heterozygous sites per individual covered by &gt;10 reads with MAPQ&gt;10. e-f, Improvements in high quality mapping rates (MAPQ&gt;20) (e) and mismatch rates (f) were most apparent for African mosquitoes (black). Use of the alternate reference also improved high quality mapping rates for non-African mosquitoes (red, e), but increased mismatch rates (f). g-h, Males and females had similar coverage (g) and mismatch rates (h) when mapped to the alternate reference, except around the sex-determining M locus. At the M locus, males had higher coverage and fewer mismatches. Note the AaegL5 assembly and alternate reference were both constructed using male mosquitoes. In panel h, the pair from Franceville, Gabon (FCV) is excluded due to a male outlier showing elevated genome-wide mismatch rates." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-2014705" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;&lt;span class=&quot;caption-title&quot;&gt;Construction of an alternate reference genome improved read mapping and variant discovery.&lt;/span&gt; &lt;strong&gt;a&lt;/strong&gt;-&lt;strong&gt;d&lt;/strong&gt;, Comparison of four mapping statistics across three iterative updates to the AaegL5 genome for a panel of 20 individuals (one male and one female from each of 10 African populations): &lt;strong&gt;a&lt;/strong&gt;, Total number of reads mapped with MAPQ&gt;20. &lt;strong&gt;b&lt;/strong&gt;, fraction of mismatched bases. &lt;strong&gt;c&lt;/strong&gt;, mean number of polymorphic sites per individual covered by at least 10 reads with MAPQ&gt;10 (polymorphic sites defined as those with at least 5 alternate allele reads present among summed reads from all samples). &lt;strong&gt;d&lt;/strong&gt;, average number of heterozygous sites per individual covered by &gt;10 reads with MAPQ&gt;10. &lt;strong&gt;e&lt;/strong&gt;-&lt;strong&gt;f&lt;/strong&gt;, Improvements in high quality mapping rates (MAPQ&gt;20) (e) and mismatch rates (f) were most apparent for African mosquitoes (black). Use of the alternate reference also improved high quality mapping rates for non-African mosquitoes (red, e), but increased mismatch rates (f). &lt;strong&gt;g&lt;/strong&gt;-&lt;strong&gt;h&lt;/strong&gt;, Males and females had similar coverage (g) and mismatch rates (h) when mapped to the alternate reference, except around the sex-determining M locus. At the M locus, males had higher coverage and fewer mismatches. Note the AaegL5 assembly and alternate reference were both constructed using male mosquitoes. In panel h, the pair from Franceville, Gabon (FCV) is excluded due to a male outlier showing elevated genome-wide mismatch rates.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Extended Data Fig. 9." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2020/02/13/2020.02.12.939041/F13.medium.gif" width="440" height="425"/><noscript><img class="highwire-fragment fragment-image" alt="Extended Data Fig. 9." src="https://www.biorxiv.org/content/biorxiv/early/2020/02/13/2020.02.12.939041/F13.medium.gif" width="440" height="425"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/02/13/2020.02.12.939041/F13.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Extended Data Fig. 9." data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/02/13/2020.02.12.939041/F13.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Extended Data Fig. 9.</span> <span class="caption-title">Construction of an alternate reference genome improved read mapping and variant discovery.</span><p id="p-73" class="first-child"><strong>a</strong>-<strong>d</strong>, Comparison of four mapping statistics across three iterative updates to the AaegL5 genome for a panel of 20 individuals (one male and one female from each of 10 African populations): <strong>a</strong>, Total number of reads mapped with MAPQ&gt;20. <strong>b</strong>, fraction of mismatched bases. <strong>c</strong>, mean number of polymorphic sites per individual covered by at least 10 reads with MAPQ&gt;10 (polymorphic sites defined as those with at least 5 alternate allele reads present among summed reads from all samples). <strong>d</strong>, average number of heterozygous sites per individual covered by &gt;10 reads with MAPQ&gt;10. <strong>e</strong>-<strong>f</strong>, Improvements in high quality mapping rates (MAPQ&gt;20) (e) and mismatch rates (f) were most apparent for African mosquitoes (black). Use of the alternate reference also improved high quality mapping rates for non-African mosquitoes (red, e), but increased mismatch rates (f). <strong>g</strong>-<strong>h</strong>, Males and females had similar coverage (g) and mismatch rates (h) when mapped to the alternate reference, except around the sex-determining M locus. At the M locus, males had higher coverage and fewer mismatches. Note the AaegL5 assembly and alternate reference were both constructed using male mosquitoes. In panel h, the pair from Franceville, Gabon (FCV) is excluded due to a male outlier showing elevated genome-wide mismatch rates.</p><div class="sb-div caption-clear"></div></div></div><div id="T1" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/1152586/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/1152586/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Extended Data Table 1.</span> <p id="p-74" class="first-child">Details of collection localities. Climate variables and population densities are calculated within a 20km radius. Latitude and Longitude are the average coordinates of egg-positive ovitraps from collections.</p><div class="sb-div caption-clear"></div></div></div><div id="T2" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/1152581/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/1152581/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Extended Data Table 2.</span> <span class="caption-title">Details of laboratory colony establishment and behavioral testing.</span><div class="sb-div caption-clear"></div></div></div></div><div class="section ack" id="ack-1"><h2 class="">Acknowledgements</h2><p id="p-61">The authors thank Siyang Xia, Christophe Paupy, and Bryan Grenfell for valuable feedback on early results, Boy Ponlawat for generously providing mosquito eggs used to establish a laboratory reference colony from Thailand, Francis Mulwa, Gilbert Rotich, Gilbert Bianquinche, Marc F. Ngangué for field assistance, the National Park Services and rangers of Kenya, Gabon, and Ghana for providing access to forest areas, and a large number of local residents who gave advice and assistance at all field sites. The findings and conclusions in this report are those of the authors and do not necessarily represent the official position of CDC.</p></div><div class="section fn-group" id="fn-group-1"><h2>Footnotes</h2><ul><li class="fn-dataset fn-group-external-links" id="fn-3"><p id="p-5">
<a href="https://www.ncbi.nlm.nih.gov/sra/PRJNA602495">https://www.ncbi.nlm.nih.gov/sra/PRJNA602495</a>
</p></li><li class="fn-dataset fn-group-external-links" id="fn-4"><p id="p-6">
<a href="https://github.com/noahrose/aaeg_adaptation_human_hosts">https://github.com/noahrose/aaeg_adaptation_human_hosts</a>
</p></li></ul></div><div class="section ref-list" id="ref-list-1"><h2 class="">References</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label">1.</span><a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference 1. in text" id="ref-1">↵</a><div class="cit ref-cit ref-web" id="cit-2020.02.12.939041v1.1"><div class="cit-metadata"><cite><span class="cit-auth cit-collab">WHO | Vector-borne diseases</span>. <span class="cit-source">WHO</span> <a href="http://www.who.int/mediacentre/factsheets/fs387/en/">http://www.who.int/mediacentre/factsheets/fs387/en/</a>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">2.</span><a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference 2. in text" id="ref-2">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.02.12.939041v1.2"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Clements</span>,  <span class="cit-name-given-names">A. N.</span></span> <span class="cit-article-title">The biology of mosquitoes</span>. <abbr class="cit-jnl-abbrev">The biology of mosquitoes</abbr>. (<span class="cit-pub-date">1999</span>).</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">3.</span><a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference 3. in text" id="ref-3">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.02.12.939041v1.3" data-doi="10.1016/j.cub.2015.11.032"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">McBride</span>,  <span class="cit-name-given-names">C. S.</span></span> <span class="cit-article-title">Genes and Odors Underlying the Recent Evolution of Mosquito Preference for Humans</span>. <abbr class="cit-jnl-abbrev">Current Biology</abbr> <span class="cit-vol">26</span>, <span class="cit-fpage">R41</span>–<span class="cit-lpage">R46</span> (<span class="cit-pub-date">2016</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCurrent%2BBiology%26rft.volume%253D26%26rft.spage%253DR41%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.cub.2015.11.032%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.cub.2015.11.032&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label">4.</span><a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference 4. in text" id="ref-4">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.02.12.939041v1.4" data-doi="10.1146/annurev-ento-120811-153618"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Takken</span>,  <span class="cit-name-given-names">W.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Verhulst</span>,  <span class="cit-name-given-names">N. O.</span></span> <span class="cit-article-title">Host Preferences of Blood-Feeding Mosquitoes</span>. <abbr class="cit-jnl-abbrev">Annual Review of Entomology</abbr> <span class="cit-vol">58</span>, <span class="cit-fpage">433</span>–<span class="cit-lpage">453</span> (<span class="cit-pub-date">2013</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAnnual%2BReview%2Bof%2BEntomology%26rft.volume%253D58%26rft.spage%253D433%26rft_id%253Dinfo%253Adoi%252F10.1146%252Fannurev-ento-120811-153618%26rft_id%253Dinfo%253Apmid%252F23020619%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1146/annurev-ento-120811-153618&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23020619&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F02%2F13%2F2020.02.12.939041.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000316406000023&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">5.</span><a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference 5. in text" id="ref-5">↵</a><div class="cit ref-cit ref-book" id="cit-2020.02.12.939041v1.5"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Christophers</span>,  <span class="cit-name-given-names">S. R.</span></span> <span class="cit-source">Aedes aegypti: The yellow fever mosquito</span>. (<span class="cit-publ-name">Cambridge University Press Archive</span>, <span class="cit-pub-date">1960</span>).</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">6.</span><a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference 6. in text" id="ref-6">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.02.12.939041v1.6"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Kotsakiozi</span>,  <span class="cit-name-given-names">P.</span></span> <span class="cit-etal">et al.</span> <span class="cit-article-title">Population structure of a vector of human diseases: Aedes aegypti in its ancestral range, Africa</span>. <abbr class="cit-jnl-abbrev">Ecology and evolution</abbr> <span class="cit-vol">8</span>, <span class="cit-fpage">7835</span>–<span class="cit-lpage">7848</span> (<span class="cit-pub-date">2018</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEcology%2Band%2Bevolution%26rft.volume%253D8%26rft.spage%253D7835%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">7.</span><a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference 7. in text" id="ref-7">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.02.12.939041v1.7"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Crawford</span>,  <span class="cit-name-given-names">J. E.</span></span> <span class="cit-etal">et al.</span> <span class="cit-article-title">Population genomics reveals that an anthropophilic population of Aedes aegypti mosquitoes in West Africa recently gave rise to American and Asian populations of this major disease vector</span>. <abbr class="cit-jnl-abbrev">BMC Biology</abbr> <span class="cit-vol">15</span>, <span class="cit-fpage">16</span> (<span class="cit-pub-date">2017</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2BBiology%26rft.volume%253D15%26rft.spage%253D16%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">8.</span><a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference 8. in text" id="ref-8">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.02.12.939041v1.8"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">McClelland</span>,  <span class="cit-name-given-names">G. A. H.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Weitz</span>,  <span class="cit-name-given-names">B.</span></span> <span class="cit-article-title">Serological identification of the natural hosts of Aedes aegypti (L.) and some other mosquitoes (Diptera, Culicidae) caught resting in vegetation in Kenya and Uganda</span>. <abbr class="cit-jnl-abbrev">Annals of Tropical Medicine &amp; Parasitology</abbr> <span class="cit-vol">57</span>, <span class="cit-fpage">214</span>–<span class="cit-lpage">224</span> (<span class="cit-pub-date">1963</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAnnals%2Bof%2BTropical%2BMedicine%2B%2526%2BParasitology%26rft.volume%253D57%26rft.spage%253D214%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">9.</span><a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference 9. in text" id="ref-9">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.02.12.939041v1.9" data-doi="10.1017/S0007485300039614"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Teesdale</span>,  <span class="cit-name-given-names">C.</span></span> <span class="cit-article-title">Studies on the Bionomics of Aëdes aegypti (L.) in its Natural Habitats in a Coastal Region of Kenya</span>. <abbr class="cit-jnl-abbrev">Bulletin of Entomological Research</abbr> <span class="cit-vol">46</span>, <span class="cit-fpage">711</span>–<span class="cit-lpage">742</span> (<span class="cit-pub-date">1955</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBulletin%2Bof%2BEntomological%2BResearch%26rft.volume%253D46%26rft.spage%253D711%26rft_id%253Dinfo%253Adoi%252F10.1017%252FS0007485300039614%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1017/S0007485300039614&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label">10.</span><a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference 10. in text" id="ref-10">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.02.12.939041v1.10" data-doi="10.1038/nature13964"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">McBride</span>,  <span class="cit-name-given-names">C. S.</span></span> <span class="cit-etal">et al.</span> <span class="cit-article-title">Evolution of mosquito preference for humans linked to an odorant receptor</span>. <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">515</span>, <span class="cit-fpage">222</span>–<span class="cit-lpage">227</span> (<span class="cit-pub-date">2014</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.volume%253D515%26rft.spage%253D222%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature13964%26rft_id%253Dinfo%253Apmid%252F25391959%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature13964&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25391959&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F02%2F13%2F2020.02.12.939041.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000344631400040&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">11.</span><a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference 11. in text" id="ref-11">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.02.12.939041v1.11"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Gouck</span>,  <span class="cit-name-given-names">H. K.</span></span> <span class="cit-article-title">Host preferences of various strains of Aedes aegypti and A. simpsoni as determined by an olfactometer</span>. <abbr class="cit-jnl-abbrev">Bulletin of the World Health Organization</abbr> <span class="cit-vol">47</span>, <span class="cit-fpage">680</span> (<span class="cit-pub-date">1972</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBulletin%2Bof%2Bthe%2BWorld%2BHealth%2BOrganization%26rft.stitle%253DBull%2BWorld%2BHealth%2BOrgan%26rft.aulast%253DGouck%26rft.auinit1%253DH.%2BK.%26rft.volume%253D47%26rft.issue%253D5%26rft.spage%253D680%26rft.epage%253D683%26rft.atitle%253DHost%2Bpreferences%2Bof%2Bvarious%2Bstrains%2Bof%2BAedes%2Baegypti%2Band%2BA.%2Bsimpsoni%2Bas%2Bdetermined%2Bby%2Ban%2Bolfactometer.%26rft_id%253Dinfo%253Apmid%252F4540689%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=4540689&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F02%2F13%2F2020.02.12.939041.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">12.</span><a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference 12. in text" id="ref-12">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.02.12.939041v1.12" data-doi="https://doi.org/10.1002/joc.5086"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Fick</span>,  <span class="cit-name-given-names">S. E.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Hijmans</span>,  <span class="cit-name-given-names">R. J.</span></span> <span class="cit-article-title">WorldClim 2: new 1-km spatial resolution climate surfaces for global land areas</span>. <abbr class="cit-jnl-abbrev">International journal of climatology</abbr> <span class="cit-vol">37</span>, <span class="cit-fpage">4302</span>–<span class="cit-lpage">4315</span> (<span class="cit-pub-date">2017</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DInternational%2Bjournal%2Bof%2Bclimatology%26rft.volume%253D37%26rft.spage%253D4302%26rft_id%253Dinfo%253Adoi%252Fhttps%253A%252F%252Fdoi.org%252F10.1002%252Fjoc.5086%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=https://doi.org/10.1002/joc.5086&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label">13.</span><a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference 13. in text" id="ref-13">↵</a><div class="cit ref-cit ref-book" id="cit-2020.02.12.939041v1.13"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Silver</span>,  <span class="cit-name-given-names">J. B.</span></span> <span class="cit-source">Mosquito ecology: field sampling methods</span>. (<span class="cit-publ-name">springer science &amp; business media</span>, <span class="cit-pub-date">2007</span>).</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">14.</span><a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference 14. in text" id="ref-14">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.02.12.939041v1.14"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Trpiš</span>,  <span class="cit-name-given-names">M.</span></span> <span class="cit-article-title">Dry season survival of Aedes aegypti eggs in various breeding sites in the Dar es Salaam area, Tanzania</span>. <abbr class="cit-jnl-abbrev">Bulletin of the World Health Organization</abbr> <span class="cit-vol">47</span>, <span class="cit-fpage">433</span> (<span class="cit-pub-date">1972</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBulletin%2Bof%2Bthe%2BWorld%2BHealth%2BOrganization%26rft.stitle%253DBull%2BWorld%2BHealth%2BOrgan%26rft.aulast%253DTrpis%26rft.auinit1%253DM.%26rft.volume%253D47%26rft.issue%253D3%26rft.spage%253D433%26rft.epage%253D437%26rft.atitle%253DDry%2Bseason%2Bsurvival%2Bof%2BAedes%2Baegypti%2Beggs%2Bin%2Bvarious%2Bbreeding%2Bsites%2Bin%2Bthe%2BDar%2Bes%2BSalaam%2Barea%252C%2BTanzania.%26rft_id%253Dinfo%253Apmid%252F4539825%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=4539825&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F02%2F13%2F2020.02.12.939041.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1972O655800018&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">15.</span><a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference 15. in text" id="ref-15">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.02.12.939041v1.15"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Paupy</span>,  <span class="cit-name-given-names">C.</span></span> <span class="cit-etal">et al.</span> <span class="cit-article-title">Morphological and genetic variability within Aedes aegypti in Niakhar, Senegal</span>. <abbr class="cit-jnl-abbrev">Infection, Genetics and Evolution</abbr> <span class="cit-vol">10</span>, <span class="cit-fpage">473</span>–<span class="cit-lpage">480</span> (<span class="cit-pub-date">2010</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DInfection%252C%2BGenetics%2Band%2BEvolution%26rft.volume%253D10%26rft.spage%253D473%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">16.</span><a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference 16. in text" id="ref-16">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.02.12.939041v1.16" data-doi="10.1093/biosci/biy119"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Powell</span>,  <span class="cit-name-given-names">J. R.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Gloria-Soria</span>,  <span class="cit-name-given-names">A.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Kotsakiozi</span>,  <span class="cit-name-given-names">P.</span></span> <span class="cit-article-title">Recent history of Aedes aegypti: vector Genomics and Epidemiology records</span>. <abbr class="cit-jnl-abbrev">BioScience</abbr> <span class="cit-vol">68</span>, <span class="cit-fpage">854</span>–<span class="cit-lpage">860</span> (<span class="cit-pub-date">2018</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioScience%26rft.volume%253D68%26rft.spage%253D854%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbiosci%252Fbiy119%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/biosci/biy119&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label">17.</span><a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference 17. in text" id="ref-17">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.02.12.939041v1.17"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Mattingly</span>,  <span class="cit-name-given-names">P. F.</span></span> <span class="cit-article-title">Genetical Aspects of the Aëdes Aegypti Problem: I.—Taxonomy and Bionomics</span>. <abbr class="cit-jnl-abbrev">Annals of Tropical Medicine &amp; Parasitology</abbr> <span class="cit-vol">51</span>, <span class="cit-fpage">392</span>–<span class="cit-lpage">408</span> (<span class="cit-pub-date">1957</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAnnals%2Bof%2BTropical%2BMedicine%2B%2526%2BParasitology%26rft.volume%253D51%26rft.spage%253D392%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">18.</span><a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference 18. in text" id="ref-18">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.02.12.939041v1.18"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">McClelland</span>,  <span class="cit-name-given-names">G. A. H.</span></span> <span class="cit-article-title">A worldwide survey of variation in scale pattern of the abdominal tergum of Aedes aegypti (L.)(Diptera: Culicidae)</span>. <abbr class="cit-jnl-abbrev">Transactions of the Royal Entomological Society of London</abbr> <span class="cit-vol">126</span>, <span class="cit-fpage">239</span>–<span class="cit-lpage">259</span> (<span class="cit-pub-date">1974</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTransactions%2Bof%2Bthe%2BRoyal%2BEntomological%2BSociety%2Bof%2BLondon%26rft.volume%253D126%26rft.spage%253D239%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=A1974U650200005&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">19.</span><a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference 19. in text" id="ref-19">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.02.12.939041v1.19"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Sylla</span>,  <span class="cit-name-given-names">M.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Bosio</span>,  <span class="cit-name-given-names">C.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Urdaneta-Marquez</span>,  <span class="cit-name-given-names">L.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Ndiaye</span>,  <span class="cit-name-given-names">M.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Iv</span>,  <span class="cit-name-given-names">W. C. B.</span></span> <span class="cit-article-title">Gene Flow, Subspecies Composition, and Dengue Virus-2 Susceptibility among Aedes aegypti Collections in Senegal</span>. <abbr class="cit-jnl-abbrev">PLOS Neglected Tropical Diseases</abbr> <span class="cit-vol">3</span>, <span class="cit-fpage">e408</span> (<span class="cit-pub-date">2009</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLOS%2BNeglected%2BTropical%2BDiseases%26rft.volume%253D3%26rft.spage%253D408e%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">20.</span><a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference 20. in text" id="ref-20">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.02.12.939041v1.20" data-doi="10.1111/mec.13866"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Gloria-Soria</span>,  <span class="cit-name-given-names">A.</span></span> <span class="cit-etal">et al.</span> <span class="cit-article-title">Global genetic diversity of Aedes aegypti</span>. <abbr class="cit-jnl-abbrev">Molecular Ecology</abbr> <span class="cit-vol">25</span>, <span class="cit-fpage">5377</span>–<span class="cit-lpage">5395</span> (<span class="cit-pub-date">2016</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2BEcology%26rft.volume%253D25%26rft.spage%253D5377%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fmec.13866%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/mec.13866&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label">21.</span><a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference 21. in text" id="ref-21">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.02.12.939041v1.21" data-doi="10.1101/gr.094052.109"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Alexander</span>,  <span class="cit-name-given-names">D. H.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Novembre</span>,  <span class="cit-name-given-names">J.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Lange</span>,  <span class="cit-name-given-names">K.</span></span> <span class="cit-article-title">Fast model-based estimation of ancestry in unrelated individuals</span>. <abbr class="cit-jnl-abbrev">Genome research</abbr> <span class="cit-vol">19</span>, <span class="cit-fpage">1655</span>–<span class="cit-lpage">1664</span> (<span class="cit-pub-date">2009</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2Bresearch%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.094052.109%26rft_id%253Dinfo%253Apmid%252F19648217%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjk6IjE5LzkvMTY1NSI7czo0OiJhdG9tIjtzOjQ4OiIvYmlvcnhpdi9lYXJseS8yMDIwLzAyLzEzLzIwMjAuMDIuMTIuOTM5MDQxLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">22.</span><a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference 22. in text" id="ref-22">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.02.12.939041v1.22" data-doi="10.1126/science.1190371"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Yi</span>,  <span class="cit-name-given-names">X.</span></span> <span class="cit-etal">et al.</span> <span class="cit-article-title">Sequencing of 50 human exomes reveals adaptation to high altitude</span>. <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">329</span>, <span class="cit-fpage">75</span>–<span class="cit-lpage">78</span> (<span class="cit-pub-date">2010</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DYi%26rft.auinit1%253DX.%26rft.volume%253D329%26rft.issue%253D5987%26rft.spage%253D75%26rft.epage%253D78%26rft.atitle%253DSequencing%2Bof%2B50%2BHuman%2BExomes%2BReveals%2BAdaptation%2Bto%2BHigh%2BAltitude%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1190371%26rft_id%253Dinfo%253Apmid%252F20595611%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjExOiIzMjkvNTk4Ny83NSI7czo0OiJhdG9tIjtzOjQ4OiIvYmlvcnhpdi9lYXJseS8yMDIwLzAyLzEzLzIwMjAuMDIuMTIuOTM5MDQxLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">23.</span><a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference 23. in text" id="ref-23">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.02.12.939041v1.23" data-doi="10.1101/001347"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Martin</span>,  <span class="cit-name-given-names">S. H.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Davey</span>,  <span class="cit-name-given-names">J. W.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Jiggins</span>,  <span class="cit-name-given-names">C. D.</span></span> <span class="cit-article-title">Evaluating the use of ABBA–BABA statistics to locate introgressed loci</span>. <abbr class="cit-jnl-abbrev">Molecular biology and evolution</abbr> <span class="cit-vol">32</span>, <span class="cit-fpage">244</span>–<span class="cit-lpage">257</span> (<span class="cit-pub-date">2014</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2Bbiology%2Band%2Bevolution%26rft.volume%253D32%26rft.spage%253D244%26rft_id%253Dinfo%253Adoi%252F10.1101%252F001347%26rft_id%253Dinfo%253Apmid%252F25246699%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1101/001347&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25246699&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F02%2F13%2F2020.02.12.939041.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">24.</span><a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference 24. in text" id="ref-24">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.02.12.939041v1.24" data-doi="10.1111/mec.12796"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Cruickshank</span>,  <span class="cit-name-given-names">T. E.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Hahn</span>,  <span class="cit-name-given-names">M. W.</span></span> <span class="cit-article-title">Reanalysis suggests that genomic islands of speciation are due to reduced diversity, not reduced gene flow</span>. <abbr class="cit-jnl-abbrev">Molecular ecology</abbr> <span class="cit-vol">23</span>, <span class="cit-fpage">3133</span>–<span class="cit-lpage">3157</span> (<span class="cit-pub-date">2014</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2Becology%26rft.volume%253D23%26rft.spage%253D3133%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fmec.12796%26rft_id%253Dinfo%253Apmid%252F24845075%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/mec.12796&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24845075&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F02%2F13%2F2020.02.12.939041.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000338014900003&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">25.</span><a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference 25. in text" id="ref-25">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.02.12.939041v1.25"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Luu</span>,  <span class="cit-name-given-names">K.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Bazin</span>,  <span class="cit-name-given-names">E.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Blum</span>,  <span class="cit-name-given-names">M. G.</span></span> <span class="cit-article-title">pcadapt: an R package to perform genome scans for selection based on principal component analysis</span>. <abbr class="cit-jnl-abbrev">Molecular ecology resources</abbr> <span class="cit-vol">17</span>, <span class="cit-fpage">67</span>–<span class="cit-lpage">77</span> (<span class="cit-pub-date">2017</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2Becology%2Bresources%26rft.volume%253D17%26rft.spage%253D67%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">26.</span><a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference 26. in text" id="ref-26">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.02.12.939041v1.26" data-doi="10.1126/science.1257469"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Gerland</span>,  <span class="cit-name-given-names">P.</span></span> <span class="cit-etal">et al.</span> <span class="cit-article-title">World population stabilization unlikely this century</span>. <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">346</span>, <span class="cit-fpage">234</span>–<span class="cit-lpage">237</span> (<span class="cit-pub-date">2014</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1257469%26rft_id%253Dinfo%253Apmid%252F25301627%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEyOiIzNDYvNjIwNi8yMzQiO3M6NDoiYXRvbSI7czo0ODoiL2Jpb3J4aXYvZWFybHkvMjAyMC8wMi8xMy8yMDIwLjAyLjEyLjkzOTA0MS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">27.</span><a class="rev-xref-ref" href="#xref-ref-27-1" title="View reference 27. in text" id="ref-27">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.02.12.939041v1.27"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Taylor</span>,  <span class="cit-name-given-names">K. E.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Stouffer</span>,  <span class="cit-name-given-names">R. J.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Meehl</span>,  <span class="cit-name-given-names">G. A.</span></span> <span class="cit-article-title">An Overview of CMIP5 and the Experiment Design</span>. <abbr class="cit-jnl-abbrev">Bull. Amer. Meteor. Soc</abbr>. <span class="cit-vol">93</span>, <span class="cit-fpage">485</span>–<span class="cit-lpage">498</span> (<span class="cit-pub-date">2011</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBull.%2BAmer.%2BMeteor.%2BSoc%26rft.volume%253D93%26rft.spage%253D485%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">28.</span><a class="rev-xref-ref" href="#xref-ref-28-1" title="View reference 28. in text" id="ref-28">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.02.12.939041v1.28" data-doi="10.1146/annurev.es.19.110188.001231"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Futuyma</span>,  <span class="cit-name-given-names">D. J.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Moreno</span>,  <span class="cit-name-given-names">G.</span></span> <span class="cit-article-title">The Evolution of Ecological Specialization</span>. <abbr class="cit-jnl-abbrev">Annual Review of Ecology and Systematics</abbr> <span class="cit-vol">19</span>, <span class="cit-fpage">207</span>–<span class="cit-lpage">233</span> (<span class="cit-pub-date">1988</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAnnual%2BReview%2Bof%2BEcology%2Band%2BSystematics%26rft.stitle%253DAnnu.%2BRev.%2BEcol.%2BSyst.%26rft.aulast%253DFutuyma%26rft.auinit1%253DD.%26rft.volume%253D19%26rft.issue%253D1%26rft.spage%253D207%26rft.epage%253D233%26rft.atitle%253DThe%2BEvolution%2BOf%2BEcological%2BSpecialization%26rft_id%253Dinfo%253Adoi%252F10.1146%252Fannurev.es.19.110188.001231%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1146/annurev.es.19.110188.001231&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=A1988R137700009&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">29.</span><a class="rev-xref-ref" href="#xref-ref-29-1" title="View reference 29. in text" id="ref-29">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.02.12.939041v1.29" data-doi="10.1093/aesa/65.3.542"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Machado-Allison</span>,  <span class="cit-name-given-names">C. E.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Craig</span>,  <span class="cit-name-given-names">G. B.</span></span> <span class="cit-article-title">Geographic Variation in Resistance to Desiccation in Aedes aegypti and A. atropalpus (Diptera: Culicidae)</span>. <abbr class="cit-jnl-abbrev">Ann Entomol Soc Am</abbr> <span class="cit-vol">65</span>, <span class="cit-fpage">542</span>–<span class="cit-lpage">547</span> (<span class="cit-pub-date">1972</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAnn%2BEntomol%2BSoc%2BAm%26rft_id%253Dinfo%253Adoi%252F10.1093%252Faesa%252F65.3.542%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/aesa/65.3.542&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label">30.</span><a class="rev-xref-ref" href="#xref-ref-30-1" title="View reference 30. in text" id="ref-30">↵</a><div class="cit ref-cit ref-book" id="cit-2020.02.12.939041v1.30"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Petersen</span>,  <span class="cit-name-given-names">J.</span></span> <span class="cit-source">Behavioral Differences in Two Sub-species of Aedes Aegypti (L</span>.)(<span class="cit-publ-name">diptera, Culicidae</span>) <span class="cit-publ-loc">in East Africa</span>. (<span class="cit-pub-date">1977</span>).</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">31.</span><a class="rev-xref-ref" href="#xref-ref-31-1" title="View reference 31. in text" id="ref-31">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.02.12.939041v1.31" data-doi="10.1038/nature13987"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Dao</span>,  <span class="cit-name-given-names">A.</span></span> <span class="cit-etal">et al.</span> <span class="cit-article-title">Signatures of aestivation and migration in Sahelian malaria mosquito populations</span>. <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">516</span>, <span class="cit-fpage">387</span>–<span class="cit-lpage">390</span> (<span class="cit-pub-date">2014</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.volume%253D516%26rft.spage%253D387%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature13987%26rft_id%253Dinfo%253Apmid%252F25470038%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature13987&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25470038&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F02%2F13%2F2020.02.12.939041.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li></ol></div><div class="section ref-list" id="ref-list-2"><h2 class="">Methods references</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label">32.</span><a class="rev-xref-ref" href="#xref-ref-32-1" title="View reference 32. in text" id="ref-32">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.02.12.939041v1.32" data-doi="10.4269/ajtmh.2011.10-0677"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Tripet</span>,  <span class="cit-name-given-names">F.</span></span> <span class="cit-etal">et al.</span> <span class="cit-article-title">Competitive Reduction by Satyrization? Evidence for Interspecific Mating in Nature and Asymmetric Reproductive Competition between Invasive Mosquito Vectors</span>. <abbr class="cit-jnl-abbrev">The American Journal of Tropical Medicine and Hygiene</abbr> <span class="cit-vol">85</span>, <span class="cit-fpage">265</span>–<span class="cit-lpage">270</span> (<span class="cit-pub-date">2011</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DThe%2BAmerican%2BJournal%2Bof%2BTropical%2BMedicine%2Band%2BHygiene%26rft.stitle%253DAm%2BJ%2BTrop%2BMed%2BHyg%26rft.aulast%253DTripet%26rft.auinit1%253DF.%26rft.volume%253D85%26rft.issue%253D2%26rft.spage%253D265%26rft.epage%253D270%26rft.atitle%253DCompetitive%2BReduction%2Bby%2BSatyrization%253F%2BEvidence%2Bfor%2BInterspecific%2BMating%2Bin%2BNature%2Band%2BAsymmetric%2BReproductive%2BCompetition%2Bbetween%2BInvasive%2BMosquito%2BVectors%26rft_id%253Dinfo%253Adoi%252F10.4269%252Fajtmh.2011.10-0677%26rft_id%253Dinfo%253Apmid%252F21813845%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoidHJvcG1lZCI7czo1OiJyZXNpZCI7czo4OiI4NS8yLzI2NSI7czo0OiJhdG9tIjtzOjQ4OiIvYmlvcnhpdi9lYXJseS8yMDIwLzAyLzEzLzIwMjAuMDIuMTIuOTM5MDQxLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">33.</span><a class="rev-xref-ref" href="#xref-ref-33-1" title="View reference 33. in text" id="ref-33">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.02.12.939041v1.33"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Gloria-Soria</span>,  <span class="cit-name-given-names">A.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Soghigian</span>,  <span class="cit-name-given-names">J.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Kellner</span>,  <span class="cit-name-given-names">D.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Powell</span>,  <span class="cit-name-given-names">J. R.</span></span> <span class="cit-article-title">Genetic diversity of laboratory strains and implications for research: The case of Aedes aegypti</span>. <abbr class="cit-jnl-abbrev">PLOS Neglected Tropical Diseases</abbr> <span class="cit-vol">13</span>, <span class="cit-fpage">e0007930</span> (<span class="cit-pub-date">2019</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLOS%2BNeglected%2BTropical%2BDiseases%26rft.volume%253D13%26rft.spage%253De0007930%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">34.</span><a class="rev-xref-ref" href="#xref-ref-34-1" title="View reference 34. in text" id="ref-34">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.02.12.939041v1.34"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Team</span>,  <span class="cit-name-given-names">R. C.</span></span> <abbr class="cit-jnl-abbrev">R: A language and environment for statistical computing</abbr>. (<span class="cit-pub-date">2013</span>).</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">35.</span><a class="rev-xref-ref" href="#xref-ref-35-1" title="View reference 35. in text" id="ref-35">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.02.12.939041v1.35" data-doi="10.3929/ethz-b-000240890"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Brooks</span>,  <span class="cit-name-given-names">M. E.</span></span> <span class="cit-etal">et al.</span> <span class="cit-article-title">glmmTMB balances speed and flexibility among packages for zero-inflated generalized linear mixed modeling</span>. <abbr class="cit-jnl-abbrev">The R journal</abbr> <span class="cit-vol">9</span>, <span class="cit-fpage">378</span>–<span class="cit-lpage">400</span> (<span class="cit-pub-date">2017</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DThe%2BR%2Bjournal%26rft.volume%253D9%26rft.spage%253D378%26rft_id%253Dinfo%253Adoi%252F10.3929%252Fethz-b-000240890%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3929/ethz-b-000240890&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label">36.</span><a class="rev-xref-ref" href="#xref-ref-36-1" title="View reference 36. in text" id="ref-36">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.02.12.939041v1.36"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Lenth</span>,  <span class="cit-name-given-names">R.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Singmann</span>,  <span class="cit-name-given-names">H.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Love</span>,  <span class="cit-name-given-names">J.</span></span> <span class="cit-article-title">Emmeans: Estimated marginal means, aka least-squares means</span>. <abbr class="cit-jnl-abbrev">R package version</abbr> <span class="cit-vol">1</span>, (<span class="cit-pub-date">2018</span>).</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">37.</span><a class="rev-xref-ref" href="#xref-ref-37-1" title="View reference 37. in text" id="ref-37">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.02.12.939041v1.37"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Turlach</span>,  <span class="cit-name-given-names">B. A.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Murray</span>,  <span class="cit-name-given-names">K.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Turlach</span>,  <span class="cit-name-given-names">M. B. A.</span></span> <abbr class="cit-jnl-abbrev">Package ‘MonoPoly’</abbr>. (<span class="cit-pub-date">2019</span>).</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">38.</span><a class="rev-xref-ref" href="#xref-ref-38-1" title="View reference 38. in text" id="ref-38">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.02.12.939041v1.38" data-doi="10.1038/s41586-018-0692-z"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Matthews</span>,  <span class="cit-name-given-names">B. J.</span></span> <span class="cit-etal">et al.</span> <span class="cit-article-title">Improved reference genome of Aedes aegypti informs arbovirus vector control</span>. <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">563</span>, <span class="cit-fpage">501</span> (<span class="cit-pub-date">2018</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.volume%253D563%26rft.spage%253D501%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fs41586-018-0692-z%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/s41586-018-0692-z&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label">39.</span><a class="rev-xref-ref" href="#xref-ref-39-1" title="View reference 39. in text" id="ref-39">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.02.12.939041v1.39" data-doi="10.1093/bioinformatics/btr330"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Danecek</span>,  <span class="cit-name-given-names">P.</span></span> <span class="cit-etal">et al.</span> <span class="cit-article-title">The variant call format and VCFtools</span>. <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">27</span>, <span class="cit-fpage">2156</span>–<span class="cit-lpage">2158</span> (<span class="cit-pub-date">2011</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtr330%26rft_id%253Dinfo%253Apmid%252F21653522%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/btr330&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21653522&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F02%2F13%2F2020.02.12.939041.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000292778700023&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">40.</span><a class="rev-xref-ref" href="#xref-ref-40-1" title="View reference 40. in text" id="ref-40">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.02.12.939041v1.40"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Li</span>,  <span class="cit-name-given-names">H.</span></span> <span class="cit-article-title">Aligning sequence reads, clone sequences and assembly contigs with BWA-MEM</span>. <abbr class="cit-jnl-abbrev">arXiv preprint</abbr><span class="cit-pub-id-sep cit-pub-id-arxiv-sep"> </span><span class="cit-pub-id cit-pub-id-arxiv"><span class="cit-pub-id-scheme-arxiv">arxiv:</span>1303.3997</span> (<span class="cit-pub-date">2013</span>).</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">41.</span><a class="rev-xref-ref" href="#xref-ref-41-1" title="View reference 41. in text" id="ref-41">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.02.12.939041v1.41" data-doi="10.1093/bioinformatics/btr509"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Li</span>,  <span class="cit-name-given-names">H.</span></span> <span class="cit-article-title">statistical framework for SNP calling, mutation discovery, association mapping and population genetical parameter estimation from sequencing data</span>. <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">27</span>, <span class="cit-fpage">2987</span>–<span class="cit-lpage">2993</span> (<span class="cit-pub-date">2011</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtr509%26rft_id%253Dinfo%253Apmid%252F21903627%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/btr509&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21903627&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F02%2F13%2F2020.02.12.939041.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000296099300009&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">42.</span><a class="rev-xref-ref" href="#xref-ref-42-1" title="View reference 42. in text" id="ref-42">↵</a><div class="cit ref-cit ref-web" id="cit-2020.02.12.939041v1.42"><div class="cit-metadata unstructured"><a href="http://broadinstitute.github.io/picard">http://broadinstitute.github.io/picard</a>.</div><div class="cit-extra"></div></div></li><li><span class="ref-label">43.</span><a class="rev-xref-ref" href="#xref-ref-43-1" title="View reference 43. in text" id="ref-43">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.02.12.939041v1.43" data-doi="10.1101/gr.107524.110"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">McKenna</span>,  <span class="cit-name-given-names">A.</span></span> <span class="cit-etal">et al.</span> <span class="cit-article-title">The Genome Analysis Toolkit: a MapReduce framework for analyzing next-generation DNA sequencing data</span>. <abbr class="cit-jnl-abbrev">Genome research</abbr> <span class="cit-vol">20</span>, <span class="cit-fpage">1297</span>–<span class="cit-lpage">1303</span> (<span class="cit-pub-date">2010</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2Bresearch%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.107524.110%26rft_id%253Dinfo%253Apmid%252F20644199%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjk6IjIwLzkvMTI5NyI7czo0OiJhdG9tIjtzOjQ4OiIvYmlvcnhpdi9lYXJseS8yMDIwLzAyLzEzLzIwMjAuMDIuMTIuOTM5MDQxLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">44.</span><a class="rev-xref-ref" href="#xref-ref-44-1" title="View reference 44. in text" id="ref-44">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.02.12.939041v1.44" data-doi="10.1186/s12859-014-0356-4"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Korneliussen</span>,  <span class="cit-name-given-names">T. S.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Albrechtsen</span>,  <span class="cit-name-given-names">A.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Nielsen</span>,  <span class="cit-name-given-names">R.</span></span> <span class="cit-article-title">ANGSD: analysis of next generation sequencing data</span>. <abbr class="cit-jnl-abbrev">BMC bioinformatics</abbr> <span class="cit-vol">15</span>, <span class="cit-fpage">356</span> (<span class="cit-pub-date">2014</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2Bbioinformatics%26rft.volume%253D15%26rft.spage%253D356%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fs12859-014-0356-4%26rft_id%253Dinfo%253Apmid%252F25420514%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/s12859-014-0356-4&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25420514&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F02%2F13%2F2020.02.12.939041.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">45.</span><a class="rev-xref-ref" href="#xref-ref-45-1" title="View reference 45. in text" id="ref-45">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.02.12.939041v1.45" data-doi="10.1086/519795"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Purcell</span>,  <span class="cit-name-given-names">S.</span></span> <span class="cit-etal">et al.</span> <span class="cit-article-title">PLINK: a tool set for whole-genome association and population-based linkage analyses</span>. <abbr class="cit-jnl-abbrev">The American journal of human genetics</abbr> <span class="cit-vol">81</span>, <span class="cit-fpage">559</span>–<span class="cit-lpage">575</span> (<span class="cit-pub-date">2007</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAmerican%2Bjournal%2Bof%2Bhuman%2Bgenetics%26rft.stitle%253DAm%2BJ%2BHum%2BGenet%26rft.aulast%253DPurcell%26rft.auinit1%253DS.%26rft.volume%253D81%26rft.issue%253D3%26rft.spage%253D559%26rft.epage%253D575%26rft.atitle%253DPLINK%253A%2Ba%2Btool%2Bset%2Bfor%2Bwhole-genome%2Bassociation%2Band%2Bpopulation-based%2Blinkage%2Banalyses.%26rft_id%253Dinfo%253Adoi%252F10.1086%252F519795%26rft_id%253Dinfo%253Apmid%252F17701901%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1086/519795&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17701901&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F02%2F13%2F2020.02.12.939041.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">46.</span><a class="rev-xref-ref" href="#xref-ref-46-1" title="View reference 46. in text" id="ref-46">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.02.12.939041v1.46"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Malinsky</span>,  <span class="cit-name-given-names">M.</span></span> <span class="cit-article-title">Dsuite-fast D-statistics and related admixture evidence from VCF files</span>. <abbr class="cit-jnl-abbrev">BioRxiv 634477</abbr> (<span class="cit-pub-date">2019</span>).</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">47.</span><a class="rev-xref-ref" href="#xref-ref-47-1" title="View reference 47. in text" id="ref-47">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.02.12.939041v1.47" data-doi="10.1093/bioinformatics/btu041"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Fumagalli</span>,  <span class="cit-name-given-names">M.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Vieira</span>,  <span class="cit-name-given-names">F. G.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Linderoth</span>,  <span class="cit-name-given-names">T.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Nielsen</span>,  <span class="cit-name-given-names">R.</span></span> <span class="cit-article-title">ngsTools: methods for population genetics analyses from next-generation sequencing data</span>. <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">30</span>, <span class="cit-fpage">1486</span>–<span class="cit-lpage">1487</span> (<span class="cit-pub-date">2014</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtu041%26rft_id%253Dinfo%253Apmid%252F24458950%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/btu041&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24458950&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F02%2F13%2F2020.02.12.939041.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">48.</span><a class="rev-xref-ref" href="#xref-ref-48-1" title="View reference 48. in text" id="ref-48">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.02.12.939041v1.48"><div class="cit-metadata"><cite><span class="cit-auth cit-collab">United Nations</span>, <span class="cit-article-title">Department of Economic and Social Affairs</span>, <abbr class="cit-jnl-abbrev">Population Division</abbr> (<span class="cit-pub-date">2019</span>). World Population Prospects 2019, Online Edition.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">49.</span><a class="rev-xref-ref" href="#xref-ref-49-1" title="View reference 49. in text" id="ref-49">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.02.12.939041v1.49"><div class="cit-metadata"><cite><span class="cit-auth cit-collab">United Nations</span>, <span class="cit-article-title">Department of Economic and Social Affairs</span>, <abbr class="cit-jnl-abbrev">Population Division. World Population Prospects 2019: Highlights (ST/ESA/SER. A/423)</abbr>. (<span class="cit-pub-date">2019</span>).</cite></div><div class="cit-extra"></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.biorxiv.org/content/10.1101/2020.02.12.939041v1.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-disqus-comment" >
  
      
  
  <div class="pane-content">
    <div id="disqus_thread"><noscript><p><a href="http://biorxivstage.disqus.com/?url=https%3A%2F%2Fwww.biorxiv.org%2Fcontent%2F10.1101%2F2020.02.12.939041v1" class="" data-icon-position="" data-hide-link-title="0">View the discussion thread.</a></p></noscript></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/10.1101/2020.02.13.948083v1" title="Gene tree discord, simplex plots, and statistical tests under the coalescent" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous</span></a></span><span class="pager-next"><a href="/content/10.1101/2020.02.13.948141v1" title="Inflammation promotes tumor aggression by stimulating stromal cell-dependent collagen crosslinking and stromal stiffening" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    Posted&nbsp;February 13, 2020.  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-biorxiv-art-tools" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-biorxiv_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/10.1101/2020.02.12.939041v1.full.pdf" target="_self" class="article-dl-pdf-link link-icon"><i class="icon-external-link-sign"></i> <span class="title">Download PDF</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/early/2020/02/13/2020.02.12.939041.external-links" target="_self" class="link-icon"><i class="icon-file"></i> <span class="title">Data/Code</span></a>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-biorxiv-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="Email this Article" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-biorxiv_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/10.1101/2020.02.12.939041v1.full" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for your interest in spreading the word about bioRxiv.</p><p>NOTE: Your email address is requested solely to identify you as the sender of this article.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/10.1101/2020.02.12.939041v1" class="active" data-icon-position="" data-hide-link-title="0">Climate and urbanization drive mosquito preference for humans</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from bioRxiv
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would like to see this page from the bioRxiv website.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/1149559" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-mTefYL_oax40qmXlPbUTS6PgOEVWyWhbq0QSM3_WJ_4" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-biorxiv-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-biorxiv_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="1149559" id="node1149559--2" data-pisa="biorxiv;2020.02.12.939041v1" data-pisa-master="biorxiv;2020.02.12.939041" data-seqnum="1149559" data-apath="/biorxiv/early/2020/02/13/2020.02.12.939041.atom"><div  class="highwire-cite highwire-cite-highwire-article highwire-citation-biorxiv-article-pap-list clearfix" >

      <div class="highwire-cite-title" >
      <div class="highwire-cite-title">Climate and urbanization drive mosquito preference for humans</div>    </div>
  
      <div  class="highwire-cite-authors" ><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0"><span class="nlm-given-names">Noah H.</span> <span class="nlm-surname">Rose</span></span>, <span class="highwire-citation-author" data-delta="1"><span class="nlm-given-names">Massamba</span> <span class="nlm-surname">Sylla</span></span>, <span class="highwire-citation-author" data-delta="2"><span class="nlm-given-names">Athanase</span> <span class="nlm-surname">Badolo</span></span>, <span class="highwire-citation-author" data-delta="3"><span class="nlm-given-names">Joel</span> <span class="nlm-surname">Lutomiah</span></span>, <span class="highwire-citation-author" data-delta="4"><span class="nlm-given-names">Diego</span> <span class="nlm-surname">Ayala</span></span>, <span class="highwire-citation-author" data-delta="5"><span class="nlm-given-names">Ogechukwu B.</span> <span class="nlm-surname">Aribodor</span></span>, <span class="highwire-citation-author" data-delta="6"><span class="nlm-given-names">Nnenna</span> <span class="nlm-surname">Ibe</span></span>, <span class="highwire-citation-author" data-delta="7"><span class="nlm-given-names">Jewelna</span> <span class="nlm-surname">Akorli</span></span>, <span class="highwire-citation-author" data-delta="8"><span class="nlm-given-names">Sampson</span> <span class="nlm-surname">Otoo</span></span>, <span class="highwire-citation-author" data-delta="9"><span class="nlm-given-names">John-Paul</span> <span class="nlm-surname">Mutebi</span></span>, <span class="highwire-citation-author" data-delta="10"><span class="nlm-given-names">Alexis L.</span> <span class="nlm-surname">Kriete</span></span>, <span class="highwire-citation-author" data-delta="11"><span class="nlm-given-names">Eliza G.</span> <span class="nlm-surname">Ewing</span></span>, <span class="highwire-citation-author" data-delta="12"><span class="nlm-given-names">Rosemary</span> <span class="nlm-surname">Sang</span></span>, <span class="highwire-citation-author" data-delta="13"><span class="nlm-given-names">Andrea</span> <span class="nlm-surname">Gloria-Soria</span></span>, <span class="highwire-citation-author" data-delta="14"><span class="nlm-given-names">Jeffrey R.</span> <span class="nlm-surname">Powell</span></span>, <span class="highwire-citation-author" data-delta="15"><span class="nlm-given-names">Rachel E.</span> <span class="nlm-surname">Baker</span></span>, <span class="highwire-citation-author" data-delta="16"><span class="nlm-given-names">Bradley J.</span> <span class="nlm-surname">White</span></span>, <span class="highwire-citation-author" data-delta="17"><span class="nlm-given-names">Jacob E.</span> <span class="nlm-surname">Crawford</span></span>, <span class="highwire-citation-author" data-delta="18"><span class="nlm-given-names">Carolyn S.</span> <span class="nlm-surname">McBride</span></span></span></div>
  
      <div  class="highwire-cite-metadata" ><span  class="highwire-cite-metadata-journal highwire-cite-metadata">bioRxiv </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">2020.02.12.939041; </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata"><span class="doi_label">doi:</span> https://doi.org/10.1101/2020.02.12.939041 </span></div>
  
  
  </div>
</div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.biorxiv.org/content/10.1101/2020.02.12.939041v1" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.biorxiv.org/content/10.1101/2020.02.12.939041v1&amp;title=Climate%20and%20urbanization%20drive%20mosquito%20preference%20for%20humans" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.biorxiv.org/content/10.1101/2020.02.12.939041v1&amp;title=Climate%20and%20urbanization%20drive%20mosquito%20preference%20for%20humans" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.biorxiv.org/content/10.1101/2020.02.12.939041v1&amp;text=Climate%20and%20urbanization%20drive%20mosquito%20preference%20for%20humans" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.biorxiv.org/content/10.1101/2020.02.12.939041v1&amp;title=Climate%20and%20urbanization%20drive%20mosquito%20preference%20for%20humans" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.biorxiv.org/content/10.1101/2020.02.12.939041v1&amp;t=Climate%20and%20urbanization%20drive%20mosquito%20preference%20for%20humans" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.biorxiv.org/content/10.1101/2020.02.12.939041v1&amp;title=Climate%20and%20urbanization%20drive%20mosquito%20preference%20for%20humans" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.biorxiv.org/content/10.1101/2020.02.12.939041v1&amp;title=Climate%20and%20urbanization%20drive%20mosquito%20preference%20for%20humans" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-biorxiv-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title="Citation Tools"><i class="icon-globe"></i> <span class="title">Citation Tools</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-biorxiv_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="1149559" id="node1149559--3" data-pisa="biorxiv;2020.02.12.939041v1" data-pisa-master="biorxiv;2020.02.12.939041" data-seqnum="1149559" data-apath="/biorxiv/early/2020/02/13/2020.02.12.939041.atom"><div  class="highwire-cite highwire-cite-highwire-article highwire-citation-biorxiv-article-pap-list clearfix" >

      <div class="highwire-cite-title" >
      <div class="highwire-cite-title">Climate and urbanization drive mosquito preference for humans</div>    </div>
  
      <div  class="highwire-cite-authors" ><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0"><span class="nlm-given-names">Noah H.</span> <span class="nlm-surname">Rose</span></span>, <span class="highwire-citation-author" data-delta="1"><span class="nlm-given-names">Massamba</span> <span class="nlm-surname">Sylla</span></span>, <span class="highwire-citation-author" data-delta="2"><span class="nlm-given-names">Athanase</span> <span class="nlm-surname">Badolo</span></span>, <span class="highwire-citation-author" data-delta="3"><span class="nlm-given-names">Joel</span> <span class="nlm-surname">Lutomiah</span></span>, <span class="highwire-citation-author" data-delta="4"><span class="nlm-given-names">Diego</span> <span class="nlm-surname">Ayala</span></span>, <span class="highwire-citation-author" data-delta="5"><span class="nlm-given-names">Ogechukwu B.</span> <span class="nlm-surname">Aribodor</span></span>, <span class="highwire-citation-author" data-delta="6"><span class="nlm-given-names">Nnenna</span> <span class="nlm-surname">Ibe</span></span>, <span class="highwire-citation-author" data-delta="7"><span class="nlm-given-names">Jewelna</span> <span class="nlm-surname">Akorli</span></span>, <span class="highwire-citation-author" data-delta="8"><span class="nlm-given-names">Sampson</span> <span class="nlm-surname">Otoo</span></span>, <span class="highwire-citation-author" data-delta="9"><span class="nlm-given-names">John-Paul</span> <span class="nlm-surname">Mutebi</span></span>, <span class="highwire-citation-author" data-delta="10"><span class="nlm-given-names">Alexis L.</span> <span class="nlm-surname">Kriete</span></span>, <span class="highwire-citation-author" data-delta="11"><span class="nlm-given-names">Eliza G.</span> <span class="nlm-surname">Ewing</span></span>, <span class="highwire-citation-author" data-delta="12"><span class="nlm-given-names">Rosemary</span> <span class="nlm-surname">Sang</span></span>, <span class="highwire-citation-author" data-delta="13"><span class="nlm-given-names">Andrea</span> <span class="nlm-surname">Gloria-Soria</span></span>, <span class="highwire-citation-author" data-delta="14"><span class="nlm-given-names">Jeffrey R.</span> <span class="nlm-surname">Powell</span></span>, <span class="highwire-citation-author" data-delta="15"><span class="nlm-given-names">Rachel E.</span> <span class="nlm-surname">Baker</span></span>, <span class="highwire-citation-author" data-delta="16"><span class="nlm-given-names">Bradley J.</span> <span class="nlm-surname">White</span></span>, <span class="highwire-citation-author" data-delta="17"><span class="nlm-given-names">Jacob E.</span> <span class="nlm-surname">Crawford</span></span>, <span class="highwire-citation-author" data-delta="18"><span class="nlm-given-names">Carolyn S.</span> <span class="nlm-surname">McBride</span></span></span></div>
  
      <div  class="highwire-cite-metadata" ><span  class="highwire-cite-metadata-journal highwire-cite-metadata">bioRxiv </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">2020.02.12.939041; </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata"><span class="doi_label">doi:</span> https://doi.org/10.1101/2020.02.12.939041 </span></div>
  
  
  </div>
</div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage&amp;rft.epage&amp;rft.atitle=Climate%20and%20urbanization%20drive%20mosquito%20preference%20for%20humans&amp;rft.volume&amp;rft.issue&amp;rft.date=2020-01-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=bioRxiv&amp;rft.au=Rose%2C+Noah+H.&amp;rft.au=Sylla%2C+Massamba&amp;rft.au=Badolo%2C+Athanase&amp;rft.au=Lutomiah%2C+Joel&amp;rft.au=Ayala%2C+Diego&amp;rft.au=Aribodor%2C+Ogechukwu+B.&amp;rft.au=Ibe%2C+Nnenna&amp;rft.au=Akorli%2C+Jewelna&amp;rft.au=Otoo%2C+Sampson&amp;rft.au=Mutebi%2C+John-Paul&amp;rft.au=Kriete%2C+Alexis+L.&amp;rft.au=Ewing%2C+Eliza+G.&amp;rft.au=Sang%2C+Rosemary&amp;rft.au=Gloria-Soria%2C+Andrea&amp;rft.au=Powell%2C+Jeffrey+R.&amp;rft.au=Baker%2C+Rachel+E.&amp;rft.au=White%2C+Bradley+J.&amp;rft.au=Crawford%2C+Jacob+E.&amp;rft.au=McBride%2C+Carolyn+S."></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/1149559/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/1149559/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/1149559/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/1149559/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/1149559/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/1149559/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/1149559/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/1149559/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/1149559/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/1149559/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/1149559/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/1149559/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><div class="item-list"><ul><li class="first"><a href="http://twitter.com/share?url=https%3A//www.biorxiv.org/content/10.1101/2020.02.12.939041v1&amp;count=horizontal&amp;via=&amp;text=Climate%20and%20urbanization%20drive%20mosquito%20preference%20for%20humans&amp;counturl=https%3A//www.biorxiv.org/content/10.1101/2020.02.12.939041v1" class="twitter-share-button service-links-twitter-widget" id="twitter_widget" title="Tweet This" rel="nofollow" data-icon-position="" data-hide-link-title="0"><span class="element-invisible">Tweet Widget</span></a></li><li><a href="http://www.facebook.com/plugins/like.php?href=https%3A//www.biorxiv.org/content/10.1101/2020.02.12.939041v1&amp;layout=button_count&amp;show_faces=false&amp;action=like&amp;colorscheme=light&amp;width=100&amp;height=21&amp;font=&amp;locale=" id="facebook_like" title="I Like it" class="service-links-facebook-like" rel="nofollow" data-icon-position="" data-hide-link-title="0"><span class="element-invisible">Facebook Like</span></a></li><li class="last"><a href="https://www.biorxiv.org/content/10.1101/2020.02.12.939041v1" id="google_plus_one" title="Plus it" class="service-links-google-plus-one" rel="nofollow" data-icon-position="" data-hide-link-title="0"><span class="element-invisible">Google Plus One</span></a></li></ul></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-collections" >
  
        <h2 class="pane-title">Subject Area</h2>
    
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-collections"><div class="highwire-list"><ul class="highwire-article-collection-term-list"><li class="first last odd"><span class="highwire-article-collection-term"><a href="/collection/evolutionary-biology" class="highlight" data-icon-position="" data-hide-link-title="0">Evolutionary Biology<i class="icon-caret-right"></i>
</a></span></li></ul></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-biorxiv-subject-collections block-style-col2" >
  
      
  
  <div class="pane-content">
    <div class="panel-flexible panels-flexible-8 clearfix" id="mini-panel-biorxiv_subject_collections">
<div class="panel-flexible-inside panels-flexible-8-inside">
<div class="panels-flexible-region panels-flexible-region-8-center panels-flexible-region-first panels-flexible-region-last">
  <div class="inside panels-flexible-region-inside panels-flexible-region-8-center-inside panels-flexible-region-inside-first panels-flexible-region-inside-last">
<div class="panel-pane pane-snippet" >
  
      
  
  <div class="pane-content">
    <div class="snippet biorxiv-subject-areas-table-title" id="biorxiv-subject-areas-table-title">
  
      
  <div class="snippet-content">
    <b>Subject Areas</b>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-snippet" >
  
      
  
  <div class="pane-content">
    <div class="snippet biorxiv-subject-areas-view-papers" id="biorxiv-subject-areas-view-papers">
  
      
  <div class="snippet-content">
    <a href="/content/early/recent"><strong>All Articles</strong></a>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-subject-collections" >
  
      
  
  <div class="pane-content">
    <ul id="collection" class="collection highwire-list-expand"><li class="outer collection depth-2  child first"><div class = "data-wrapper"><a href="/collection/animal-behavior-and-cognition" class="" data-icon-position="" data-hide-link-title="0">Animal Behavior and Cognition</a> <span class = "article-count">(1662)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/biochemistry" class="" data-icon-position="" data-hide-link-title="0">Biochemistry</a> <span class = "article-count">(2789)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/bioengineering" class="" data-icon-position="" data-hide-link-title="0">Bioengineering</a> <span class = "article-count">(1946)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/bioinformatics" class="" data-icon-position="" data-hide-link-title="0">Bioinformatics</a> <span class = "article-count">(10400)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/biophysics" class="" data-icon-position="" data-hide-link-title="0">Biophysics</a> <span class = "article-count">(4243)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/cancer-biology" class="" data-icon-position="" data-hide-link-title="0">Cancer Biology</a> <span class = "article-count">(3272)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/cell-biology" class="" data-icon-position="" data-hide-link-title="0">Cell Biology</a> <span class = "article-count">(4641)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/clinical-trials" class="" data-icon-position="" data-hide-link-title="0">Clinical Trials</a> <span class = "article-count">(135)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/developmental-biology" class="" data-icon-position="" data-hide-link-title="0">Developmental Biology</a> <span class = "article-count">(2889)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/ecology" class="" data-icon-position="" data-hide-link-title="0">Ecology</a> <span class = "article-count">(4544)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/epidemiology" class="" data-icon-position="" data-hide-link-title="0">Epidemiology</a> <span class = "article-count">(2041)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/evolutionary-biology" class="" data-icon-position="" data-hide-link-title="0">Evolutionary Biology</a> <span class = "article-count">(7288)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/genetics" class="" data-icon-position="" data-hide-link-title="0">Genetics</a> <span class = "article-count">(5540)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/genomics" class="" data-icon-position="" data-hide-link-title="0">Genomics</a> <span class = "article-count">(6893)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/immunology" class="" data-icon-position="" data-hide-link-title="0">Immunology</a> <span class = "article-count">(2434)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/microbiology" class="" data-icon-position="" data-hide-link-title="0">Microbiology</a> <span class = "article-count">(7616)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/molecular-biology" class="" data-icon-position="" data-hide-link-title="0">Molecular Biology</a> <span class = "article-count">(3051)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/neuroscience" class="" data-icon-position="" data-hide-link-title="0">Neuroscience</a> <span class = "article-count">(18885)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/paleontology" class="" data-icon-position="" data-hide-link-title="0">Paleontology</a> <span class = "article-count">(142)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/pathology" class="" data-icon-position="" data-hide-link-title="0">Pathology</a> <span class = "article-count">(477)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/pharmacology-and-toxicology" class="" data-icon-position="" data-hide-link-title="0">Pharmacology and Toxicology</a> <span class = "article-count">(791)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/physiology" class="" data-icon-position="" data-hide-link-title="0">Physiology</a> <span class = "article-count">(1178)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/plant-biology" class="" data-icon-position="" data-hide-link-title="0">Plant Biology</a> <span class = "article-count">(2771)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/scientific-communication-and-education" class="" data-icon-position="" data-hide-link-title="0">Scientific Communication and Education</a> <span class = "article-count">(686)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/synthetic-biology" class="" data-icon-position="" data-hide-link-title="0">Synthetic Biology</a> <span class = "article-count">(905)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/systems-biology" class="" data-icon-position="" data-hide-link-title="0">Systems Biology</a> <span class = "article-count">(2881)</span></div></li>
<li class="outer collection depth-2  child last"><div class = "data-wrapper"><a href="/collection/zoology" class="" data-icon-position="" data-hide-link-title="0">Zoology</a> <span class = "article-count">(472)</span></div></li>
</ul>  </div>

  
  </div>
  </div>
</div>
</div>
</div>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__VNH5GD8Zz7g6_hGCOZjVjdMndxxC6naiExSSJKX3k_A__kzmEdtbqCsJx5Z9yg2Qy0PptB__ufXm-TxUOl0KZzUw__QJacVmveFyLIBSPkp21xL0ZvuDW9DMBrD2HB1P4Ry5I.js"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__QJacVmveFyLIBSPkp21xL0ZvuDW9DMBrD2HB1P4Ry5I.js" defer="defer"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__N7ERJBYsOWyRnJgyoM125_Aiez2MOJGaUofG1JdWWBg__bLlcwhzZhT-IwoQ_dvm1_xPxU7E7gecFLC6Fyeqtv8s__QJacVmveFyLIBSPkp21xL0ZvuDW9DMBrD2HB1P4Ry5I.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
function euCookieComplianceLoadScripts() {}
//--><!]]>
</script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var eu_cookie_compliance_cookie_name = "";
//--><!]]>
</script>
<script type="text/javascript" src="//d33xdlntwy0kbs.cloudfront.net/cshl_custom.js"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__BmqjBnkz3MgYCAoc25s1lDRMEjLhC3mEPVonUFIHi08__Unwv5-ZIuHBfFwytsjEx1niBVJ7n1T4lPws7VrkHXM4__QJacVmveFyLIBSPkp21xL0ZvuDW9DMBrD2HB1P4Ry5I.js"></script>
  </body>
</html>
