<html xmlns="http://www.w3.org/1999/xhtml"><head> <style type="text/css">
         	     		 
         	    a                          {background : #ffffff; }
         	    article                    {border-style : dotted; border-width : 2px; }
         		 	
         		 	div                        {background : #ffffcc;}
         		 	div.abstract-title         {font-weight : bold ; font-size : 16pt;}
         		 	div.ack                    {border-style : solid ; border-color : red; margin
         : 2em; }
         		 	div.article-meta           {border-style : solid; border-width : 1 pt; margin
         1em;}
         		 	div.boxed-text             {margin : 5em; border-style : solid;}
         		 	div.contrib-group          {margin : 1em; }
         		 	div.fig                    {border-style : solid; border-width : 2px; margin :
         2em; }
         		 	div.funding                {font-weight : bold ; font-size : 16pt;}
         		 	div.given-names            {font-style : italic;}
         		 	div.intro                  {border-style : inset; margin : 5px;}
         		 	div.introduction           {border-style : inset; margin : 5px;}
         		 	div.journal-meta           {border-style : solid; border-width : 1 pt; margin
         1em;}
         		 	div.journal-title-group    {background : #ffeeee;}
         		 	div.article-title          {font-weight : bold ; font-size : 18pt;}
         		 	div.kwd                    {font-style : italic}
         		 	div.meta-name              {font-weight : bold ; font-size : 16pt;}
         		 	div.name                   {font-weight : bold;}
         		 	div.alt-title              {font-style : italic; font-size : 12px;}
         		 	
         		 	div.sec                    {border : 2px; margin 5px; padding 2px;}
         		 	div.title                  {font-family : courier; font-weight : bold;
         		 	                            font-size : 16pt; margin : 5px;}
         		 	
         		 	div.materials_methods      {border-style : double; margin : 5px; }
         		 	div.methods                {border-style : double; margin : 5px; }
         		 	
         		 	div.results                {border-style : solid; margin : 5px;}
         		 	div.background             {border-style : dotted; margin : 5px;}
         		 	
         		 	div.discussion             {border-style : groove; margin : 5px;}
         		 	div.conclusion             {border-style : ridge; margin : 5px;}
         		 	div.conclusions             {border-style : ridge; margin : 5px;}
         		 	div.supplementary-material {border-style : inset; margin : 5px;}
         		 	div.abbreviations          {border-style : double; border-color : red; margin
         : 5px;}
         		 	div.competinginterests     {border-style : double; border-color : blue; margin
         : 5px;}
         		 	div.acknowledgements       {border-style : double; border-color : green; margin
         : 5px;}
         		 	div.authors_contributions   {border-style : double; border-color : purple; margin
         : 5px;}
         		 	
         		 	div.publisher              {border-style : outset; margin : 5px;}
         		 	div.fn-type-conflict       {background : #f88; }
         		 	div.fn-type-con            {background : #ddf; }
         		 	div.fn-type-other          {background : #ddd; }
         		 	
         		 	div.unknown                {background : #ffd;
         									 	  border-style : solid;
         									 	  border-width : 1px;
         									 	  padding : 2 px;}
         		 	  
         	    table                      {background : #ffffdd;}
         		 	tr                         {background : #ddddff; padding : 1px;}
         		 	
         		 	span                       {background : #ffcccc;}
         		 	
         		 	span.citation-author       {font-family : helvetica; background : #ffeeee;}
         		 	span.collab                {background : #ddffff; }
         		 	span.comment               {font-family : courier; font-size : 6px; background
         : #ffaaff;}
         		 	span.contrib               {background : #ffffff;}
         		 	span.corresp               {background : #ddffdd; }
         		 	span.doi                   {background : #ffffff;}
         		 	span.email                 {font-family : courier; }
         		 	span.etal                  {font-style : italic;}
         		 	span.fpage                 {font-family : courier;}
         		 	span.given-names           {background : #ffffff;}
         		 	span.iso-abbrev            {background : #ffffff;}
         		 	span.issn-epub             {background : #ffffff;}
         		 	span.issn-ppub             {background : #ffffff;}
         		 	span.journal-title         {background : #ffffff;}
         		 	span.lpage                 {font-family : courier;}
         		 	span.mixed-article-title   {font-style : italic ;}
         		 	span.nlm-ta                {background : #ffffff;}
         		 	span.pmc                   {background : #ffffff;}
         		 	span.pmcid                 {background : #ffffff;}
         		 	span.pmid                  {background : #ffffff;}
         		 	span.publisher             {background : #ffffff;}
         		 	span.publisher-id          {background : #ffffff;}
         		 	span.publisher-name        {background : #ffffff;}
         		 	span.source                {background : #ffffff;}
         		 	span.subject               {background : #ffffff;}
         		 	span.surname               {background : #ffffff;}
         		 	span.volume                {font-family : courier; font-weight : bold;}
         		 	span.year                  {font-family : courier ; font-style : italic;}
         			
      </style> </head> <body> <div class="front" title="front"> <div class="journal-meta" tagx="journal-meta" title="journal-meta"> <span class="nlm-ta" title="nlm-ta">Sci Rep</span> <span class="iso-abbrev" title="iso-abbrev">Sci Rep</span> <div class="journal-title-group" tagx="journal-title-group" title="journal-title-group"> <span class="journal-title" tagx="journal-title" title="journal-title">Scientific Reports</span> </div> <span class="issn-epub" tagx="issn" title="issn-epub">2045-2322</span> <div class="publisher" tagx="publisher" title="publisher"> <span class="publisher-name" tagx="publisher-name" title="publisher-name">Nature Publishing Group UK</span> <span class="publisher-loc" tagx="publisher-loc" title="publisher-loc">London</span> </div> </div> <div class="article-meta" tagx="article-meta" title="article-meta"> <span class="pmcid" title="pmcid"> pmcid: <a href="http://www.ncbi.nlm.nih.gov/pubmed/5505980">5505980</a> </span> <span class="publisher-id" title="publisher-id">5309</span> <span class="doi" title="doi"> doi: <a href="https://dx.doi.org/10.1038/s41598-017-05309-w">10.1038/s41598-017-05309-w</a> </span> <div class="article-categories" title="article-categories">
: <span class="subject" title="subject">Article</span> </div> <div class="title-group"> <div class="article-title" title="article-title">
An intragenic mutagenesis strategy in <i>Physcomitrella patens</i> to preserve intron splicing </div> </div> <div class="contrib-group" title="contrib-group"> <span class="contrib" title="contrib"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Ako</span> <span class="given-names" tagx="given-names" title="given-names">Ako Eugene</span> </span> <a href="#Aff1">1</a> </span> <span class="contrib" title="contrib"> <span class="contrib-id" tagx="contrib-id" title="contrib-id">http://orcid.org/0000-0001-7607-3618</span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Perroud</span> <span class="given-names" tagx="given-names" title="given-names">Pierre-François</span> </span> <a href="#Aff2">2</a> </span> <span class="contrib" title="contrib"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Innocent</span> <span class="given-names" tagx="given-names" title="given-names">Joseph</span> </span> <a href="#Aff1">1</a> </span> <span class="contrib" title="contrib"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Demko</span> <span class="given-names" tagx="given-names" title="given-names">Viktor</span> </span> <a href="#Aff3">3</a> </span> <span class="contrib" title="contrib"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Olsen</span> <span class="given-names" tagx="given-names" title="given-names">Odd-Arne</span> </span> <span class="address" title="address"> <span class="email" tagx="email" title="email">odd-arne.olsen@nmbu.no</span> </span> <a href="#Aff3">3</a> </span> <span class="contrib" title="contrib"> <span class="contrib-id" tagx="contrib-id" title="contrib-id">http://orcid.org/0000-0002-9240-586X</span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Johansen</span> <span class="given-names" tagx="given-names" title="given-names">Wenche</span> </span> <span class="address" title="address"> <span class="email" tagx="email" title="email">wenche.johansen@inn.no</span> </span> <a href="#Aff1">1</a> </span> <span class="citation_author_institution" id="Aff1">[1], <div class="institution-wrap" title="institution-wrap"> <span class="institution-id" title="institution-id">grid.477237.2</span> <span class="institution" title="institution" /> <span class="institution" title="institution">Inland Norway University of Applied Sciences, </span> </div> </span> <span class="citation_author_institution" id="Aff2">[2], <div class="institution-wrap" title="institution-wrap"> <span class="institution-id" title="institution-id">0000 0004 1936 9756</span> <span class="institution-id" title="institution-id">grid.10253.35</span> <span class="institution" title="institution" /> <span class="institution" title="institution">Philipps University Marburg, </span> </div> </span> <span class="citation_author_institution" id="Aff3">[3], <div class="institution-wrap" title="institution-wrap"> <span class="institution-id" title="institution-id">0000 0004 0607 975X</span> <span class="institution-id" title="institution-id">grid.19477.3c</span> <span class="institution" title="institution" /> <span class="institution" title="institution">Norwegian University of Life Sciences, </span> </div> </span> </div> <span class="pub-date-epub" title="pub-date-epub">epub: <span>2017-7-7</span> </span> <span class="pub-date-pmc-release" title="pub-date-pmc-release">pmc-release: <span>2017-7-7</span> </span> <span class="pub-date-collection" title="pub-date-collection">collection: <span>2017</span> </span> <span class="volume" tagx="volume" title="volume">7</span> <span class="elocation-id" tagx="elocation-id" title="elocation-id">5111</span> <span class="history" title="history"> <span class="received" title="received">received: 2017-1-4</span> <span class="accepted" title="accepted">accepted: 2017-5-26</span> </span> <div class="permissions"> <span class="copyright" title="copyright">(C) , </span> <span class="license" title="license"> <span class="license-p" title="license-p"> <b>Open Access</b> This article is licensed under a Creative Commons Attribution 4.0 International License, which permits use, sharing, adaptation, distribution and reproduction in any medium or format, as long as you give appropriate credit to the original author(s) and the source, provide a link to the Creative Commons license, and indicate if changes were made. The images or other third party material in this article are included in the article’s Creative Commons license, unless indicated otherwise in a credit line to the material. If material is not included in the article’s Creative Commons license and your intended use is not permitted by statutory regulation or exceeds the permitted use, you will need to obtain permission directly from the copyright holder. To view a copy of this license, visit <a href="http://creativecommons.org/licenses/by/4.0/">http://creativecommons.org/licenses/by/4.0/</a>. </span> </span> </div> <div class="abstract" title="abstract"> <div class="abstract-title" title="abstract-title">
Abstract</div> <p>Gene targeting is a powerful reverse genetics technique for site-specific genome modification. Intrinsic homologous recombination in the moss <i>Physcomitrella patens</i> permits highly effective gene targeting, a characteristic that makes this organism a valuable model for functional genetics. Functional characterization of domains located within a multi-domain protein depends on the ability to generate mutants harboring genetic modifications at internal gene positions while maintaining the reading-frames of the flanking exons. In this study, we designed and evaluated different gene targeting constructs for targeted gene manipulation of sequences corresponding to internal domains of the DEFECTIVE KERNEL1 protein in <i>Physcomitrella patens</i>. Our results show that gene targeting-associated mutagenesis of introns can have adverse effects on splicing, corrupting the normal reading frame of the transcript. We show that successful genetic modification of internal sequences of multi-exon genes depends on gene-targeting strategies which insert the selection marker cassette into the 5′ end of the intron and preserve the nucleotide sequence of the targeted intron. </p> </div> <div class="custom-meta-group" title="custom-meta-group"> <meta name="issue-copyright-statement" value="© The Author(s) 2017" /> </div> </div> </div> <div class="body" title="body"> <div class="introduction" title="introduction"> <div class="title" tagx="title" title="title">
Introduction</div> <p>Engineering loss- and gain-of-function alleles by reverse genetic studies is a powerful approach for dissecting the biochemical function of a gene product <i>in vivo</i>. Several different tools and resources are available for reverse genetic approaches in plants, including RNA-mediated interference (RNAi) <sup> <a href="#CR1">1</a> </sup>, virus-induced gene silencing (VIGS) <sup> <a href="#CR2">2</a> </sup>, targeted induced local lesions in genomes (TILLING) <sup> <a href="#CR3">3</a> </sup>, clustered regularly interspaced short palindromic repeats (CRISPR)/CRISPR-associated protein 9 (Cas9) (CRISPR/Cas9) <sup> <a href="#CR4">4</a> </sup> and gene targeting (GT) <sup> <a href="#CR5">5</a> </sup>. Gene targeting is a technique for introducing site-specific genome modifications at an endogenous locus via homologous recombination (HR) with exogenous DNA. Because of its site-specific precision and the almost unlimited types of DNA modifications that can be introduced, e.g. deletions, inversions, insertions and point mutations, GT remains one of the most powerful methods available to analyze gene function. However, in angiosperm plants as well as in most higher eukaryotes, GT is inefficient due to the fact that the frequency of random DNA integration by illegitimate recombination (IR) exceeds that of targeted integration by HR by several orders of magnitude <sup> <a href="#CR6">6</a>, <a href="#CR7">7</a> </sup>. Mosses, particularly <i>Physcomitrella patens</i>, where GT is as efficient as in <i>Saccharomyces cerevisiae</i> <sup> <a href="#CR5">5</a>, <a href="#CR8">8</a> </sup>, and much more efficient than in mouse ES cells <sup> <a href="#CR9">9</a> </sup> and in the green alga <i>Chlamydomonas reinhardtii</i> <sup> <a href="#CR10">10</a> </sup>, represent an exception to this rule in plants. Recently, it was demonstrated that GT-efficiency could be significantly improved in moss using the CRISPR/Cas9 system <sup> <a href="#CR11">11</a>, <a href="#CR12">12</a> </sup>. Highly efficient GT in <i>P</i>. <i>patens</i> also permits complex <i>in vivo</i> assembly of multiple DNA fragments with overlapping sequences of as short as 12 bp <sup> <a href="#CR13">13</a> </sup>. This also expands the potential of this model organism for biotechnological applications similar to that of yeast <sup> <a href="#CR14">14</a>, <a href="#CR15">15</a> </sup>. </p> <p>The intrinsic efficiency of HR in <i>P</i>. <i>patens</i> has made GT the predominant method for functional genomics in this species. Knock-out of a gene in <i>P</i>. <i>patens</i> is routinely achieved using a construct harboring a positive selection marker gene, such as those conferring neomycin ( <i>neo</i>) or hygromycin ( <i>hpt</i>) resistance, flanked by targeting sequences (TS) homologous to the 5′ and 3′ flanking regions of the sequence to be modified <sup> <a href="#CR16">16</a> </sup>. Upon transformation, homologous exchange between the TSs in the vector and their genome counterparts results in a targeted gene deletion and insertion of the marker cassette. Constructs for homology-dependent gene insertion (knock-in) are similar in design to knock-out vectors, except that the sequence to be introduced is fused in-frame with one of the TSs in the vector. The selection marker gene used in GT vectors can be flanked by <i>loxP</i> sequences that later can be exploited to obtain marker-free mutant lines by Cre/lox-mediated recombination <sup> <a href="#CR17">17</a>– <a href="#CR22">22</a> </sup>. The site-specific Cre recombinase, which is introduced by transformation of a Cre-expressing vector <sup> <a href="#CR23">23</a> </sup>, specifically recognizes the recombining <i>loxP</i> sites and mediates deletion of the marker gene, leaving a singular <i>loxP</i> site at the recombined locus <sup> <a href="#CR23">23</a> </sup>. </p> <p>The majority of HR-promoted gene disruptions in <i>P</i>. <i>patens</i> represent single gene knock-out mutants generated to study gene-function relationships, in addition to gene tagging experiments allowing <i>in vivo</i> monitoring of target gene expression and fusion protein localization. However, since most eukaryotic proteins are composed of multiple domains, often with specific functions, mutants harboring specific partial gene deletions are often desirable for functional studies. Design of the vector for targeted gene manipulations of individual domains is a critical step, and requires knowledge of the intron-exon borders in order to precisely modify the nucleotide sequence corresponding to the domain of interest while preserving the reading frame of the gene. In addition, deletion of individual domains located internally in a protein and/or for which the coding sequence spans two or more exons are further complicated by the insertion of the selection marker into intronic sequences, which potentially can interfere with splicing. <i>P</i>. <i>patens</i> mutants harboring partial gene deletions have been reported, the deleted domains being either located at the N- or C-terminus of the protein <sup> <a href="#CR24">24</a> </sup>. Although common practice in yeast <sup> <a href="#CR25">25</a>, <a href="#CR26">26</a> </sup> and mice <sup> <a href="#CR27">27</a>, <a href="#CR28">28</a> </sup>, reports of targeted partial deletion of sequences corresponding to internal protein domains are rare in plants. </p> <p>DEK1 is a large multi-domain membrane-anchored cysteine protease of approximately 240 kDa composed of an N-terminal membrane domain (MEM) harboring 23 transmembrane segments linked to the C-terminal calpain domain (CysPc-C2L) by a Linker segment, containing a Laminin-G3 (LG3) domain (Fig. <a href="#Fig1">1</a>) <sup> <a href="#CR29">29</a> </sup>. Previously, we created the <i>P</i>. <i>patens DEK1</i> knockout mutant ( <i>Δdek1</i>), in which the entire 14,569 bp <i>DEK1</i> gene of 30 coding exons was deleted using GT. The <i>Δdek1</i> mutant develops protonemata, filamentous cells exhibiting polar tip growth; however, leafy gametophores fail to form due to an early arrest of bud development <sup> <a href="#CR30">30</a> </sup>. The same phenotype is obtained in a mutant ( <i>dek1</i> <sup> <i>0</i> </sup>) <sup> <a href="#CR29">29</a> </sup> in which the active site cysteine residue of the DEK1 protease core domain is substituted with serine, a mutation that also inactivates DEK1 calpain activity <i>in vitro</i> <sup> <a href="#CR31">31</a> </sup>. Thus, mutant plants with calpain-null alleles display a clearly detectable phenotype with filamentous growth without gametophores. A much milder phenotype is observed in the <i>P</i>. <i>patens dek1Δlg3</i> mutant, generated by in-frame removal of the sequence corresponding to the DEK1-LG3 domain. This mutant forms reduced gametophores with narrow phyllids lacking several features of the WT phyllids, including the midrib <sup> <a href="#CR29">29</a> </sup>. </p><div class="fig" title="fig"> <div class="figure" title="figure"> <a href="http://doi.org/"> <span class="label" tagx="label" title="label">Figure 1</span> </a> </div>   <p>Intron-exon structure of the <i>Physcomitrella patens DEK1</i> gene. <i>DEK1</i> ORF consists of 30 exons (grey rectangles) encoding the MEM domain (blue line), the LINKER segment (red line), the LG3 domain (blue rectangle) and the CALPAIN domain (green line). The numbers identify the exons of the <i>DEK1</i> ORF. </p>   </div> <p /> <p>In this report, we evaluate the performance of different <i>DEK1</i> GT genome editing constructs for the production of mutants harboring specific internal gene deletions and insertions in <i>P</i>. <i>patens</i>. Generation of mutagenized <i>DEK1</i> transcripts with an intact open-reading frame was used as a criterion for successful gene modification. Our results show that intragenic mutagenesis in <i>P</i>. <i>patens</i> is critically dependent on a vector design that conserves intron splicing signals. We propose useful guidelines and discuss important considerations in the design of GT vectors for targeting internal sequences of multi-exon genes in <i>P</i>. <i>patens</i>. </p> </div> <div class="results" title="results"> <div class="title" tagx="title" title="title">
Results</div> <div class="designofgenetargetingvectorsforinternalexondeletions" title="sec"> <div class="title" tagx="title" title="title">
Design of gene targeting vectors for internal exon deletions</div> <p>Two different GT vector designs G1 and G2 were tested to delete the <i>Linker</i> encoded by <i>DEK1</i> exons 17–24 (Fig. <a href="#Fig1">1</a>) in order to create a <i>P</i>. <i>patens</i> ( <i>Pp</i>) mutant expressing a truncated DEK1 protein lacking the internal Linker segment. The G1 vector was designed to insert the 5′ and 3′ ends of the <i>loxP</i>-flanked <i>Hygromycin Resistance Cassette</i> ( <i>HRC</i>) into <i>DEK1</i> introns 17 and 23, respectively. With this vector, we expected to produce a mutant with targeted deletion of a 3198 bp <i>DEK1</i> fragment (nucleotides (nt) 8347–11544; XP_001774206.1) of the <i>Linker</i> sequence. In addition, transformation was expected to yield a <i>DEK1</i> hybrid intron composed of the <i>HRC</i> flanked by the 5′ and 3′ ends of introns 17 and 23, respectively (Fig. <a href="#Fig2">2a</a>). The G2 vector was designed to insert the <i>loxP</i>-flanked <i>HRC</i> into <i>DEK1</i> intron 17 without affecting the intron nucleotide composition (Fig. <a href="#Fig2">2b</a>). To this end, the G2 5′ targeting sequence (TS) contained ~1.3 kb genomic sequence (nt 7249–8404) ending in <i>DEK1</i> intron 17. The ~1.2 kb 3′ TS was a mutagenized genomic sequence (nt 8402–12893, Δ8462–11718) composed of <i>DEK1</i> nucleotides from intron 17 followed by the first 12 nucleotides of exon 18 fused in frame with the 3′ end of exon 24, to intron 26. Using the G2 vector for transformation, we expected targeted deletion of the <i>DEK1</i> sequence from within exon 18 to within exon 24, in addition to insertion of the <i>HRC</i> into the full-length intron 17 (Fig. <a href="#Fig2">2b</a>). </p><div class="fig" title="fig"> <div class="figure" title="figure"> <a href="http://doi.org/"> <span class="label" tagx="label" title="label">Figure 2</span> </a> </div>   <p> <i>DEK1-Linker</i> knock-out. Schematic representation of the <i>DEK1-Linker</i> knock-out strategies using the ( <b>a</b>) G1 and ( <b>b</b>) G2 vectors and elimination of the hygromycin resistance cassette ( <i>HRC</i>) (light blue box) by Cre/loxP-mediated recombination. The grey boxes correspond to exons in the <i>P</i>. <i>patens DEK1</i> gene. Introns are colored in orange, with the exception of introns 17 (blue) and 23 (red). The 5′ and 3′ targeting sequences (TS) are boxed. The <i>loxP</i> sites are shown as vertical black lines. Numbers identify the exons of the <i>DEK1</i> gene. ( <b>c</b>) cDNA sequencing electropherogram of the <i>dek1∆linkerG1</i>/ <i>cre DEK1</i> transcript showing deletion of exon 18–23 and retention of the <i>loxP</i>-harboring hybrid intron 17/23, creating a premature termination code (red box). ( <b>d</b>) cDNA sequencing electropherogram of the <i>dek1∆linkerG2</i>/ <i>cre DEK1</i> transcript showing in-frame fusion of exons 17, 18 and 24. ( <b>e</b>) Phenotype of 3-week-old <i>dek1∆linkerG1</i>/ <i>cre</i> and <i>dek1∆linkerG2</i>/ <i>cre</i> plants. For comparison, wild-type (WT) and <i>∆dek1</i> are also shown. WT plants with developing gametophores. <i>dek1∆linkerG1</i>/ <i>cre</i> and <i>dek1∆linkerG2</i>/ <i>cre</i> mutant plants are showing filamentous growth and aborted buds (arrows). <i>dek1∆linkerG1</i>/ <i>cre</i> bud development stops at an early stage leading to small arrested buds, similar to <i>∆dek1</i>; the buds of <i>dek1∆linkerG2</i>/ <i>cre</i> mutant continue to divide, giving rise to larger disorganized bud structures. Scale bars: grey = 2 mm, black = 500 μm and white = 25 μm. </p>   </div> <p /> <p>Following transformation of wild-type <i>P</i>. <i>patens</i> protoplasts with the G1 and G2 vectors, six <i>dek1ΔlinkerG1</i> and one <i>dek1ΔlinkerG2</i> hygromycin-resistant (Hyg <sup>R</sup>) on-locus targeted mutant lines, respectively, were identified by PCR (data not shown). These plants all displayed a phenotype similar to the <i>dek1</i> <sup> <i>0</i> </sup> mutant <sup> <a href="#CR29">29</a> </sup> with filamentous growth and lacking gametophores, as shown for the <i>dek1ΔlinkerG1#20</i> and <i>dek1ΔlinkerG2#1</i> lines in Supplementary Fig. <a href="#MOESM1">S1</a>. Insertion of the 1.8 kb <i>HRC</i> in an intron may potentially cause a null mutant phenotype by interfering with posttranscriptional modification of the transcript, preventing expression of the active protein. This was subsequently confirmed by reverse-transcription (RT)-PCR analysis using primers annealing outside TS and within the resistance marker cassette, with positive amplification signals for the presence of the <i>HRC</i> in the cDNA amplified from the two mutant lines (data not shown). Thus, in order to eliminate the resistant marker from the <i>DEK1</i> locus, <i>dek1ΔlinkerG1#20</i> and <i>dek1ΔlinkerG2#1</i> were subjected to Cre/loxP-mediated recombination, resulting in the hygromycin-sensitive (Hyg <sup>S</sup>) lines named <i>dek1ΔlinkerG1</i>/ <i>cre</i> and <i>dek1ΔlinkerG2</i>/ <i>cre</i>. The mutants were verified by Southern hybridization confirming targeted deletion of the <i>Linker</i>, no sign of secondary non-specific integration, and loss of the <i>HRC</i> from the locus (Supplementary Fig. <a href="#MOESM1">S2a</a>). To investigate <i>DEK1</i> transcript processing in <i>dek1ΔlinkerG1</i>/ <i>cre</i> and <i>dek1ΔlinkerG2</i>/ <i>cre</i>, a cDNA spanning exon 11 to exon 27 was amplified by RT-PCR using primers annealing outside the TSs, resulting in the generation of a single PCR product (Supplementary Fig. <a href="#MOESM1">S2b</a>) which was subsequently sequenced (Fig. <a href="#Fig2">2c,d</a>). Analysis of the <i>dek1ΔlinkerG1</i>/ <i>cre</i> cDNA revealed retention of the hybrid intron 17/23 in the truncated <i>DEK1</i> transcript (Fig. <a href="#Fig2">2c</a>). Retention of this intron introduced a premature stop codon 5′ to the calpain coding sequence (Fig. <a href="#Fig2">2c</a>), caused by a frame-shift mutation. In contrast, analysis of the <i>dek1ΔlinkerG2</i>/ <i>cre</i> cDNA confirmed the expected splicing product of the truncated <i>DEK1</i> transcript, with the 5′ end of exon 18 fused in frame with the 3′ end of exon 24 (Fig. <a href="#Fig2">2d</a>). Consistent with the molecular data, <i>dek1ΔlinkerG1</i>/ <i>cre</i> displayed a typical DEK1 calpain-null mutant phenotype <sup> <a href="#CR29">29</a>, <a href="#CR30">30</a> </sup> with lack of gametophores and aborted buds (Fig. <a href="#Fig2">2e</a>). In contrast, <i>dek1ΔlinkerG2</i>/ <i>cre</i> mutant buds divide beyond those of <i>Δdek1</i>, forming larger disorganized buds with abortive growth (Fig. <a href="#Fig2">2e</a>), suggesting the presence of residual levels of DEK1 calpain activity in the mutant <sup> <a href="#CR29">29</a> </sup>. </p> </div> <div class="designofgenetargetingvectorsforinternalexoninsertions" title="sec"> <div class="title" tagx="title" title="title">
Design of gene targeting vectors for internal exon insertions</div> <p>We next tested two different GT vector designs (In23-5′ and In23-3′) for knock-in of wild-type, heterologous and mutated <i>DEK1-LG3</i> sequences into the <i>Pp dek1Δlg3</i> mutant. The LG3 domain is encoded within exon 22 and 23 of the <i>PpDEK1</i> gene (Fig. <a href="#Fig1">1</a>), and the <i>dek1Δlg3</i> mutant contains a deletion in the genomic region spanning the LG3 domain <sup> <a href="#CR29">29</a> </sup>. Both for the In23-5′ and the In23-3′ designs, the complementing <i>LG3</i> sequence is fused in frame with the 5′ TS, and the <i>loxP</i> flanked <i>HRC</i> are targeted to intron 23, however the position of the <i>HRC</i> insertion within intron 23 differed between the two (Fig. <a href="#Fig3">3a,b</a>). </p><div class="fig" title="fig"> <div class="figure" title="figure"> <a href="http://doi.org/"> <span class="label" tagx="label" title="label">Figure 3</span> </a> </div>   <p> <i>DEK1-LG3</i> knock-in. Schematic representation of the <i>DEK1-LG3</i> knock-in strategies using the ( <b>a</b>) In23-3′ and ( <b>b</b>) In23-5′ vectors, and elimination of the <i>hygromycin</i> resistance cassette ( <i>HRC</i>) (light blue box) by Cre/loxP-mediated recombination. The grey, blue and green boxes correspond to the exons in the <i>P</i>. <i>patens DEK1</i> gene. The blue and green boxes represent the fused exon 22 and 23, respectively, of the <i>dek1Δlg3</i> mutant. Introns are colored in orange, with the exception of intron 23, which is colored red. The 5′ and 3′ targeting sequences (TS) are boxed. The <i>loxP</i> sites are shown as vertical black lines. The numbers correspond to the exons of the <i>DEK1</i> gene. For simplicity, the <i>PpLG3-M1</i>, <i>PpLG3-M2</i> and <i>PpLG3-M3</i> mutant lines are collectively denoted as <i>Pp</i>* <i>LG3</i>. ( <b>c</b>) cDNA sequencing electropherogram of the <i>PpLG3-3</i>′ <i>DEK1</i> transcript showing retention of the <i>loxP</i>-harboring intron 23, resulting in a premature stop codon (red box). The slanted lines denote omitted sequences due to space limitation. ( <b>d</b>) Phenotype of 3-week-old <i>PpLG3-3</i>′ mutant plant showing filamentous growth without gametophores. ( <b>e</b>) cDNA sequencing electropherogram of the <i>PpLG3-5</i>′ <i>DEK1</i> transcript showing in-frame fusion of exon 23 and exon 24. ( <b>f</b>) Phenotype of 3-week-old <i>PpLG3-5</i>′ mutant plant showing well-developed gametophores, demonstrating successful complementation of the <i>dek1Δlg3</i> mutant phenotype. Scale bars: 2 mm. </p>   </div> <p /> <p>In the In23-3′ design, the <i>HRC</i> was targeted to the end of intron 23, 27 bp upstream of the 3′ end of the intron (Fig. <a href="#Fig3">3a</a> and Supplementary Fig. <a href="#MOESM1">S3a</a>). In this set of experiments, wild-type genomic <i>LG3</i> sequences from <i>Pp</i>, <i>Arabidopsis thaliana</i> ( <i>At</i>) and <i>Marchantia polymorpha</i> ( <i>Mp</i>) were targeted to the <i>Pp DEK1ΔLG3</i> locus. In the In23-5′ design, the <i>HRC</i> was targeted to the start of intron 23, 28 bp downstream of the 5′ end of the intron (Fig. <a href="#Fig3">3b</a> and Supplementary Fig. <a href="#MOESM1">S3b</a>), and in this set of experiments, we targeted the wild-type and three mutated <i>Pp LG3</i> versions (LG3/M1, LG3/M2 and LG3/M3, see Methods) to the <i>DEK1ΔLG3</i> locus. Following transformation of <i>dek1Δlg3</i> protoplasts with the vectors built using the In23-3′ design, we obtained 40 <i>PpLG3-3</i>′, 105 <i>MpLG3</i> and 74 <i>AtLG3</i> Hyg <sup>R</sup> lines in which four, five and one, respectively were on-locus targeted as demonstrated by PCR genotyping (data not shown). The <i>HRC</i> was subsequently removed from one representative of each of the <i>PpLG3-3</i>′, <i>AtLG3</i> and <i>MpLG3</i> lines using Cre/loxP-mediated recombination. <i>DEK1</i> cDNA of marker-free <i>PpLG3-3</i>′/ <i>cre</i>, <i>AtLG3</i>/ <i>cre</i> and <i>MpLG3</i>/ <i>cre</i> lines were amplified and analyzed as described in the preceding section, confirming insertion of the wild-type (Supplementary Fig. <a href="#MOESM1">S3c(i)</a>) and heterologous <i>LG3</i> sequences (Supplementary Fig. <a href="#MOESM1">S3c(ii,iii)</a>) at the <i>DEK1</i> locus. In all lines, however, intron 23 was retained in the <i>DEK1</i> transcript, as shown by DNA sequencing of the <i>DEK1</i> cDNA amplification-product of line <i>PpLG3-3</i>′/ <i>cre</i> in Fig. <a href="#Fig3">3c</a>. Retention of the <i>loxP</i>-harboring intron in the <i>DEK1</i> transcript caused a frame-shift mutation leading to a premature termination codon and a putative truncated DEK1 protein lacking the catalytic calpain domain. Consistently, all mutant lines displayed a calpain-null mutant phenotype similar to <i>Δdek1</i> and <i>dek1</i> <sup> <i>0</i> </sup> with the lack of gametophores as shown for <i>PpLG3-3</i>′/ <i>cre</i> in Fig. <a href="#Fig3">3d</a>, and for <i>AtLG3</i>/ <i>cre</i> and <i>MpLG3</i>/ <i>cre</i> in Supplementary Fig. <a href="#MOESM1">S3d</a>. In addition to retention of <i>Pp DEK1</i> intron 23, the <i>AtLG3</i>/ <i>cre DEK1</i> transcript displayed additional pre-mRNA splicing errors, specifically the loss of 64 exonic nucleotides corresponding to 61 nucleotides at the 3′ end of <i>At DEK1</i> exon 21 followed by the first three nucleotides of <i>At</i> exon 22 (Supplementary Fig. <a href="#MOESM1">S3c(ii)</a>). In contrast, sequencing of the <i>DEK1</i> cDNA from <i>MpLG3</i>/ <i>cre</i> confirmed proper splicing of the heterologous <i>Mp LG3</i> sequence (Supplementary Fig. <a href="#MOESM1">S3c(iii)</a>). </p> <p>Transformation of <i>dek1Δlg3</i> protoplasts with vectors built using the In23-5′ strategy (Fig. <a href="#Fig3">3b</a> and Supplementary Fig. <a href="#MOESM1">S3b</a>), yielded 3 <i>PpLG3-5</i>′, 3 <i>PpLG3-M1</i>, 5 <i>PpLG3-M2</i> and 8 <i>PpLG3-M3</i> Hyg <sup>R</sup> on-locus targeted lines (data not shown) with the <i>HRC</i> from one representative line for each having been removed by Cre/lox-mediated excision. Sequence analysis of <i>DEK1</i> cDNA amplified from marker-free lines showed in-frame re-targeting of the <i>Pp LG3</i> sequences and proper splicing at the locus, as shown for mutant <i>PpLG3-5</i>′/ <i>cre</i> in Fig. <a href="#Fig3">3e</a>. Sequencing also confirmed the presence of the desired <i>LG3</i> mutations in the <i>DEK1</i> transcript of <i>PpLG3-M1</i>/ <i>cre</i>, <i>PpLG3-M2</i>/ <i>cre</i> and <i>PpLG3-M3</i>/ <i>cre</i> (Supplementary Fig. <a href="#MOESM1">S3e</a>). The <i>PpLG3-5</i>′/ <i>cre</i> (Fig. <a href="#Fig3">3f</a>) and <i>PpLG3-M1</i>/ <i>cre</i>, <i>PpLG3-M2</i>/ <i>cre</i> and <i>PpLG3-M3</i>/ <i>cre</i> mutants (Supplementary Fig. <a href="#MOESM1">S3f</a>) all developed gametophores with fully expanded phyllids similar to wild-type plants, and clearly distinguishable from the narrow phyllid phenotype of the <i>dek1Δlg3</i> mutant <sup> <a href="#CR29">29</a> </sup> (Supplementary Fig. <a href="#MOESM1">S3f</a>), thus confirming successful complementation at the <i>DEK1ΔLG3</i> locus using the In23-5′ vector design. </p> </div> </div> <div class="discussion" title="discussion"> <div class="title" tagx="title" title="title">
Discussion</div> <p> <i>In vivo</i> genetic engineering of internal exonic sequences is complicated by the targeting of intronic sequences with a selection marker cassette, which may interfere with post-transcriptional modifications, disrupting gene expression. Although the resistance marker cassette can be efficiently eliminated from the targeted intron using e.g. the Cre/lox system, one <i>loxP</i> site will be retained at the locus following recombination. In this study, we show that successful genetic modifications of internal protein domains is dependent on the critical and rational design of the GT construct in order to avoid gene targeting-associated mutagenesis of introns having detrimental effects on splicing of the targeted transcript. </p> <p>Two different knock-out vectors (G1 and G2) for targeted deletion of the internal <i>DEK1 Linker</i> were constructed and tested in <i>P</i>. <i>patens</i> plants. Transformation with both vectors resulted in mutants with targeted deletion of the <i>Linker</i>. However, while the G2 strategy resulted in a mutant plant ( <i>dek1ΔlinkerG2</i>/ <i>cre</i>) expressing a <i>DEK1</i> transcript with in-frame deletion of the <i>Linker</i>, the G1 strategy resulted in a mutant plant ( <i>dek1ΔlinkerG1</i>/ <i>cre</i>) expressing a <i>DEK1</i> transcript containing a frame-shift mutation leading to a premature termination code. The frame-shift mutation was caused by retention of the <i>loxP</i>-harboring hybrid intron 17/23, impairing <i>DEK1</i> pre-mRNA splicing. Thus, the G1, but not the G2 strategy resulted in the introduction of mutations causing splicing defects of the targeted intron. Three structural elements are used to correctly excise introns from primary transcripts. These include the 5′ and 3′ splicing sites spanning the 5′ exon/intron and the 3′ intron/exon borders respectively, and the internal branch point (BP) sequence located 19–50 nt upstream of the 3′ splicing site <sup> <a href="#CR32">32</a> </sup>. The hybrid <i>DEK1</i> 17/23 intron of <i>dek1ΔlinkerG1</i>/ <i>cre</i> contains both the heterologous 5′ and 3′ splicing sites, but lacks the internal sequences of the native introns since only the 5′ and 3′ portions of intron 17 and 23, respectively, are retained after recombination between the G1 donor vector and the <i>DEK1</i> locus (Fig. <a href="#Fig2">2a</a>). This likely resulted in a loss of internal splicing signals, e.g the BP sequence within the targeted intron, thus abolishing splicing of exon 17 and 24. In contrast, splicing of the mutagenized, <i>loxP</i>-harboring <i>DEK1</i> intron 17 of the <i>dek1ΔlinkerG2</i>/ <i>cre</i> mutant occurred properly. In this intron, no nucleotides were deleted during targeted insertion of the <i>HRC</i>. These results demonstrate that when designing GT knock-out vectors aimed at deleting internal exonic sequences, a strategy that preserves the native sequence of the targeted intron(s) and which minimizes or eliminates the loss of intronic nucleotides, should be chosen. This is achieved by using a strategy (G2) as described in this study; the selection marker cassette in the donor vector is placed within the sequence of the native intron (Fig. <a href="#Fig2">2b</a>). The 3′ end of this intron is followed by a sequence of the adjacent (3′) exon, which is fused in frame to the 3′ TS starting with a downstream exon. This design thus creates a heterologous exon at the targeted locus, containing a mutagenized sequence harboring a deletion of the genomic sequence intervening the original exons of the heterologous exon. Using this strategy, intronic sequences are preserved. </p> <p>We also designed and evaluated two different GT knock-in vectors (termed In23-5′ and In23-3′) for targeted insertion of wild-type, heterologous and mutated versions of <i>DEK1-LG3</i> sequences into the <i>P</i>. <i>patens dek1Δlg3</i> mutant. Knock-in of a gene fragment is commonly performed in genetic complementation tests to confirm that a mutant phenotype is caused by a specific gene deletion, or to introduce modified versions of the gene for functional studies. In complementation tests where the outcome of the gene modification cannot be predicted, possibly giving a phenotype indistinguishable from that of the background line used for transformation, a positive selection marker is needed to identify transformed plants. Similar to gene knock-out approaches, knock-in of DNA fragments into internal regions of a gene is complicated by the necessity of targeting the selection marker gene to introns. Transformation of <i>dek1Δlg3</i> protoplasts with In23-5′-based vectors, where the resistant marker was inserted 28 bp downstream of the start of intron 23, expressed <i>DEK1</i> transcripts in which the reading-frame was maintained, and the plants reverted to a wild-type gametophore phenotype, confirming successful knock-in of <i>DEK1-LG3</i> into the <i>DEK1ΔLG3</i> locus. In contrast, <i>dek1Δlg3</i> protoplasts transformed with In23-3′-based vectors, where the resistance marker was inserted 27 bp upstream of the end of intron 23, expressed <i>DEK1</i> transcripts containing the <i>loxP-</i>harboring intron 23. These plants displayed a calpain-null mutant phenotype, as expected for DEK1 plants lacking the catalytic calpain domain. These results demonstrate that <i>HRC</i> insertion into the 3′ end, but not the 5′ end, of <i>DEK1</i> intron 23 impairs splicing, suggesting that a residing <i>loxP</i> sequence located 3′, but not 5′, corrupts the internal splicing signals of the intron. To explore this possibility further, we used the ERISdb program (http://lemur.amu.edu.pl/share/ERISdb/home.html) <sup> <a href="#CR33">33</a> </sup>, which predicts splice sites and splicing signals in various plant species, including <i>P</i>. <i>patens</i>. Using this tool, we predicted a putative branch site 36 bp upstream of the 3′ splice site of <i>DEK1</i> intron 23, close to the position in which the <i>loxP</i> sequence is retained in the mis-spliced mutagenized <i>DEK1</i> intron 23. The BP sequence is important for splicing in plants <sup> <a href="#CR34">34</a> </sup>, the sequence between the BP and the 3′ splicing site, tending to be U-rich, strongly influences splicing efficiency and splice site selection in plants <sup> <a href="#CR35">35</a> </sup>. Thus, insertion of a <i>loxP</i> site into this region may have corrupted the splicing signal of the intron, causing intron 23 to evade spliceosome activity. Our results demonstrate that selection of the marker cassette insertion point within the intron is important when designing GT vectors. BP sequences are normally located 19–50 nucleotides upstream of the 3′ splicing site. In order to avoid interfering with essential BP signals, we recommend to insert the selection marker close to the 5′ end of the intron and, if possible, at least 50 nucleotides upstream of the 3′ end of the intron. Correspondingly, the G2 donor vector that was successfully used to create the mutant plant expressing the <i>DEK1</i> transcript harboring an in-frame deletion of the <i>Linker</i>, was also designed to insert the resistant marker to the 5′ end of the targeted intron. Thus, our results show that targeting of the resistance marker cassette to the 5′ end of the intron should be preferred. It should be mentioned that choosing an appropriate position to insert the marker cassette may be problematic if the targeted intron is small. In such case, identifying a position where GT-associated mutagenesis ( <i>loxP</i> retention) will not affect splicing, may be challenging. Therefore, if the insertion position within the gene is not critical for the experiment, we recommend the targeting of larger sized introns. </p> <p>Recently it was shown that the <i>piggyBAC</i> transposon could be used for the precise and effective excision of selection marker cassettes from the genome without leaving any residual ectopic sequences both in mammalian cells <sup> <a href="#CR36">36</a> </sup> and rice <sup> <a href="#CR37">37</a>, <a href="#CR38">38</a> </sup>. The transposon, which derives from the cabbage looper <i>Trichoplusia ni</i> <sup> <a href="#CR39">39</a> </sup>, integrates into the host genome at TTAA elements and has been used efficiently for transgenesis in animals <sup> <a href="#CR40">40</a> </sup>. The <i>piggyBAC</i> transposon thus offers an alternative method to the Cre/loxP system for elimination of a selection marker cassette from genomes. However, with the exception that the piggyBAC system has yet to be tested in <i>P</i>. <i>patens</i>, the main limitation of the system is the requirement for specific insertion into a TTAA sequence, maxing the system less flexible especially for precise site-directed mutagenesis. </p> <p>An attractive and powerful alternative to classical GT methods in <i>P</i>. <i>patens</i> is the CRISPR/Cas9 system <sup> <a href="#CR12">12</a> </sup>. In <i>P</i>. <i>patens</i> both transformation frequency and GT efficiency have been found to be significantly enhanced using CRISPR/Cas9 when compared to the classical method using GT donor vectors only <sup> <a href="#CR11">11</a> </sup>. However, no reports have been published on the use of the CRISPR/Cas9 system for precise partial deletions or insertions of gene sequences at internal positions. Ultimately, the CRISPR/Cas9 approach may be efficient enough to bypass the need of the resistance cassette for such partial deletions. But for now, as with classical GT approaches, insertion of a resistance marker cassette to internal intronic sequences is necessary in order to select for transformants receiving the donor DNA. Thus, the results reported in the present study would also apply to the design of the donor vector using the CRISPR/Cas9 system. </p> <p>In this study, we also demonstrate that <i>P</i>. <i>patens</i> has the capacity to properly splice a genomic fragment from the liverwort <i>M</i>. <i>polymorpha</i>, suggesting conservation of splicing signals between mosses and liverworts. In contrast, the genomic <i>A</i>. <i>thaliana DEK1</i> sequence in <i>P</i>. <i>patens</i> is incorrectly processed, suggesting the possibility that the moss spliceosome machinery does not properly recognize the splicing signals from dicot plants. This may stem from differences in the UA content of the introns; <i>P</i>. <i>patens</i> introns display a significantly lower UA content, 60%, compared to 67% in <i>A</i>. <i>thaliana</i> <sup> <a href="#CR41">41</a> </sup>. Notably, Perroud and Quatrano <sup> <a href="#CR42">42</a> </sup> showed that <i>P</i>. <i>patens</i> correctly splices the <i>BRK1</i> gene from <i>A</i>. <i>thaliana</i>, indicating that accurate processing of certain <i>A</i>. <i>thaliana</i> introns occurs in the moss. </p> <p>It is worth noting that another knock-in GT strategy can be used when the expected phenotype of the resulting mutant is morphologically distinguishable from that of the transformation background. In this strategy, the GT vector does not carry a resistance marker and the complementing DNA is flanked by the 5′ and 3′ TSs. After transformation with the resistance marker-free vector, transformed mutants harboring single copy on-locus GT events can easily be identified based on the phenotype. The use of this approach offers two advantages; elimination of the time-consuming Cre/loxP-mediated transformation step to remove the resistance marker and no introduction of <i>loxP</i>-associated mutations affecting splicing integrity of the targeted intron. </p> <p>Finally, in this study we investigated the functional significance of highly conserved amino acid residues in the DEK1-LG3 domain by performing site-directed mutagenesis using the In23-5′ strategy to knock-in mutated versions of <i>LG3</i> into the <i>dek1Δlg3</i> mutant. <i>HRC</i>-free <i>PpLG3-M1</i>, <i>PpLG3-M2</i> and <i>PpLG3-M3</i> mutant lines, harboring substitution of amino acids E <sup>1477</sup>Q <sup>1478</sup>(M1), E <sup>1481</sup>(M2) and S <sup>1497</sup>(M3) to alanine, respectively, all developed wild-type like gametophores indicating that the targeted amino acids are not critically important for DEK1-LG3 function during gametophore development. Also, worth noting is that we have observed that transformation of wild-type protoplasts with constructs designed to introduce site-directed amino acid substitutions may result in selection of marker-resistant plants harboring the wild-type sequence at the targeted position. Therefore, we recommend the use of deletion mutants (lacking the sequence for which the substitution is to be made), rather than wild-type plants, as the transformation background for creating mutants harboring single (or few) amino acid substitutions. </p> </div> <div class="materials_methods" title="materials|methods"> <div class="title" tagx="title" title="title">
Methods</div> <div class="plantmaterial,growthconditionsandtransformation" title="sec"> <div class="title" tagx="title" title="title">
Plant material, growth conditions and transformation</div> <p>The <i>Physcomitrella patens</i> Gransden strain and the <i>P</i>. <i>patens dek1Δlg3</i> mutant <sup> <a href="#CR29">29</a> </sup> were used throughout this study. Tissue maintenance and production was performed on BCDA medium as described by Cove and co-workers <sup> <a href="#CR43">43</a> </sup>. <i>P</i>. <i>patens</i> tissue and protoplasts were grown under long day conditions (16 hours light [70 to 80 µmolm <sup>−2</sup> s <sup>−1</sup>]/8 hours dark) at 25 °C. <i>P</i>. <i>patens</i> protoplast transformation was performed as described in ref. <a href="#CR43">43</a>. Growth medium was supplemented with 20 µgl <sup>−1</sup> of Hygromycin B for selection of transformed plants. Tissue for phenotypic characterizations was grown on BCD medium. Tissue for reverse transcription PCR (RT-PCR) and Southern blot analysis was grown on BCDA medium overlaid with cellophane discs under long day conditions. The tissue was harvested after 7 days, snap-frozen in liquid nitrogen, ground in a mortar and used for RNA and DNA extraction. </p> </div> <div class="constructionofgenetargetingvectors" title="sec"> <div class="title" tagx="title" title="title">
Construction of gene targeting vectors</div> <p>The sequences of the primers used for vector constructions are listed in Supplementary Table <a href="#MOESM1">S1</a>. Plasmid cloning was performed using the In-Fusion® HD Cloning kit (Clontech Laboratories) according to the manufacturer’s instructions, unless otherwise stated. The PCR-generated fragments used in the In-Fusion reactions were amplified with Thermo Scientific® Phusion High-Fidelity DNA Polymerase. To verify the constructed vectors, restriction digestion and DNA sequencing analyses were performed. All nucleotide (nt) numberings are relative to the A <sup>1</sup>TG start site of the <i>P</i>. <i>patens</i> ( <i>Pp</i>) <i>DEK1</i> genomic sequence (Pp3c17_17550; <a href="http://www.phytozome.net">www.phytozome.net</a>), unless otherwise stated. </p> <p>Knock-out vectors: Construction of the pBHRF∆LinkerG1 vector is described by Johansen <i>et al</i>. <sup> <a href="#CR29">29</a> </sup>. In short, fragments spanning nt 7249–8346 and nt 11545–1254 as the 5′ and 3′ TSs, respectively, flank the hygromycin resistance cassette in pBHRF∆LinkerG1 (Fig. <a href="#Fig2">2a</a>). GenScript Inc. (USA) performed fragment synthesis and cloning of the pBHRFΔLinkerG2 vector. The following fragments were synthesized de novo: the 5′ TS, nt 7249–8404 flanked by HindIII and NruI recognition sequences and the 3′ TS, a mutagenized fragment spanning from nt 8402–12893 with a 3257 bp deletion from nt 8462–11718 and flanked by 5′ SpeI and 3′ NsiI recognition sequences. The 5′ and 3′ TSs were cloned sequentially into pBHRF <sup> <a href="#CR20">20</a> </sup> using HindIII/NruI and SpeI/NsiI restriction enzyme pairs, respectively, resulting in vector pBHRF∆LinkerG2 (Fig. <a href="#Fig2">2b</a>). Prior to <i>P</i>. <i>patens</i> protoplast transformation, the pBHRF∆LinkerG1 and pBHRF∆LinkerG2 vectors were digested with the restriction enzymes BmrI and PacI. </p> <p>Knock-in vectors: The vectors to re-insert WT and heterologous DEK1-LG3 sequences into the <i>DEK1ΔLG3</i> locus in the <i>dekΔlg3</i> mutant using the In23-3′ strategy (Fig. <a href="#Fig3">3</a>) were constructed from vector pBHRF∆LinkerG1 <sup> <a href="#CR29">29</a> </sup>. To make the vector containing the WT LG3 sequence, nt 8347–11544 was first PCR-amplified using primers SP_Inf_6 N and ASP_Inf_6 N from genomic <i>P</i>. <i>patens</i> DNA. The resulting PCR product was mixed with linearized (at nt position 8346) pBHRF <i>Δ</i>LinkerG1 DNA, produced by PCR using primers SP_Inf_5 N and ASP_Inf_5 N. The linearized vector and insert were ligated using an In-Fusion reaction, resulting in the WT LG3 complementation vector pBHRF_PpLG3_In23-3′ (Fig. <a href="#Fig3">3a</a>). To make the construct for DEK1 LG3 cross-species complementation tests, the WT genomic <i>DEK1 LG3</i> sequences from <i>Arabidopsis thaliana</i> ( <i>At</i>) (nt 6741–7450; relative to the A <sup>1</sup>TG start site in AT1G55350) and <i>Marchantia polymorpha</i> ( <i>Mp</i>) (nt 1258573–1259287; scaffold 26, JGI scaffold genomic data v3.1) were first PCR-amplified using primer sets MpLG3_Inf-F/MpLG3_Inf-R and AtLG3_Inf-F/AtLG3_Inf-R, respectively. Vector pBHRF_PpLG3_In23-3′ was PCR-amplified to delete nt 10567–11239, corresponding to the <i>DEK1 LG3</i> sequence, using primers SP_Inf_2 and ASP_Inf_2X. Finally, the linearized vector and the <i>AtLG3</i> and <i>MpLG3</i> amplicons were ligated in two separate In-Fusion reactions creating the cross-species complementation vectors pBHRF_AtLG3_In23-3′ and pBHRF_MpLG3_In23-3′, respectively. The vectors to re-insert <i>P</i>. <i>patens</i> DEK1 LG3 WT and mutated sequences using the In23-5′ strategy (Fig. <a href="#Fig3">3b</a>) were built from pBHRF <sup> <a href="#CR20">20</a> </sup>. First, the 3′ TS (nt 11334–12511) was PCR-amplified using primers 3TS-F and 3TS-R. The resulting product was cloned into linearized plasmid pBHRF, produced by PCR using primers pBHRF-F and pBHRF-R, resulting in plasmid pBHRF_3′ TS. The 5′ TS ( <i>DEK1</i> nt 9568–11333), which harbors the <i>LG3</i> coding sequence, was PCR-amplified from genomic <i>P</i>. <i>patens</i> DNA using primers 5TS-F and 5TS-R and subsequently cloned into the pCR-Blunt vector (Invitrogen) to obtain the pCR_5′ TS vector. To make mutated versions of the <i>LG3</i> sequence, the pCR_5′ TS plasmid was subjected to mutagenesis reactions using the GeneArt® Site-Directed Mutagenesis PLUS kit (Invitrogen) with the mutagenic primer pairs M1-F/M1-R, M2-F/M2-R and M3-F/M3-R, resulting in three mutated 5′ TSs containing the M1 (E1477A;Q1478A), M2 (E <sup>1481</sup>A) and M3 (S <sup>1497</sup>A) substitutions, respectively. The WT and mutagenized versions of the 5′ TS were each amplified with primers 5TS_Inf-F and 5TS_Inf-R, and sub-cloned into XhoI-linearized vector pBHRF_3′ TS, resulting in the generation of the complementation vectors pBHRF_PpLG3_In23-5′, pBHRF_PpLG3_M1_In23-5′ pBHRF_PpLG3_M2_In23-5′ and pBHRF_PpLG3_M3_In23-5′. Prior to <i>P</i>. <i>patens</i> protoplast transformation, these vectors were digested with the restriction enzymes BmrI and PacI. </p> </div> <div class="pcrgenotypingoftransformants" title="sec"> <div class="title" tagx="title" title="title">
PCR genotyping of transformants</div> <p>Each mutant was subjected to three rounds of PCR genotyping using the Phire Plant Direct PCR kit (Thermo Fisher Scientific) according to the manufacturer’s instructions. The sequences of the primers used for genotyping each set of transformants are provided in Supplementary Table <a href="#MOESM1">S1</a>. For primary genotyping, knock-out and knock-in mutants were genotyped for the loss and gain of genomic sequence, respectively. Secondary genotyping PCR was performed to verify proper 5′ and 3′ targeting at the locus using a primer pair annealing within the DNA fragment used for transformation and 5′ or 3′ to the 5′ and 3′ TSs, respectively. Finally, tertiary genotyping PCR, mutants harboring single-copy insertion events at the locus were identified using primers annealing 5′ and 3′ to the 5′ and 3′ TSs, respectively. </p> <p>For <i>dek1∆linkerG1</i> and <i>dek1∆linkerG2</i>, primary genotyping was performed using primers Linker-F and Linker-R. Lines without a detectable PCR product were subjected to the second PCR using primer pairs Linker-2F and 35S_rev1 for the 5′ targeting and, Term-fwd and Linker-2R for the 3′ targeting. Lines with both 5′ and 3′ targeting were assessed for single copy insertion at each locus using the primers Linker-2F and Linker 2 R. For the In23-3′ lines, gain of sequence was assessed using primers ARMSeq 6 and ASP_PpARM_inf 1. Positive lines were genotyped for on-locus targeting using primers TM2Seq1 and 35S_rev1 for 5′ targeting, and Term_fwd and Ex30 R for 3′ targeting. Single copy insertion was assessed using TM2seq1 and Ex30 R. For In23-5′ lines, primary genotyping was performed using primers LG3_fwd and LG3_rev; secondary genotyping: ARMSeq3 and 35S_rev1 for 5′ targeting and, Term_fwd and Ex30 R for 3′ targeting, respectively. Tertiary genotyping was performed using primers ARMSeq3 and Ex30 R. </p> </div> <div class="molecularcharacterizationoftransformants" title="sec"> <div class="title" tagx="title" title="title">
Molecular characterization of transformants</div> <p>Genomic DNA for Southern Blot analysis was extracted using the Nucleon™ PhytoPure™ Genomic DNA Extraction Kit (GE Healthcare). Southern Blot was performed as described <sup> <a href="#CR19">19</a> </sup> using ca. 1 µg DNA per digestion. Probes were DIG-labelled using the DIG Probe PCR synthesis kit (Roche, Indianapolis, USA) according to the manufacturer’s instructions. pBHRF∆LinkerG1 DNA was used as template for PCR amplification of the TS and hygromycin-resistance probes; primers for amplification of the; 5′ TS probe: Δarm5′-fwd and Δarm5′-rev; 3′ TS probe: Δarm3′-fwd and Δarm3′-rev; HRC probe: HRC-fwd and HRC-rev. The sequences of the primers used to generate the probes are provided in Supplementary Table <a href="#MOESM1">S1</a>. </p> <p>RT-PCR and DNA sequencing were employed to amplify and analyze the <i>DEK1</i> cDNA to verify proper deletion/insertion and splicing of the transcript. Total RNA was isolated from protonema tissue using the Plant RNeasy Kit (Qiagen). Five hundred ng DNaseI-treated total RNA was reverse-transcribed using 100 U Superscript™ III Reverse Transcriptase (Invitrogen) primed with 200 ng random hexamers at 55 °C for 60 min. Phusion® High-Fidelity DNA polymerase was used to amplify the DEK1 cDNA target using primers TM2Seq1 and ASP_PpCALP, annealing in exons 11 and 27, respectively. After Exo-SAP treatment to remove unincorporated primers, the PCR product was sequenced with appropriate primers using BigDye® v3.1 chemistry (ABI) according to the SteP method <sup> <a href="#CR44">44</a> </sup>. Extension products were precipitated using sodium-acetate/ethanol, dissolved in Hi-Di™ Formamide (ABI) and finally sequenced by Capillary Electrophoresis using the 3130xL Genetic Analyzer. DNA sequences were analyzed using Genomic Workbench v7. </p> </div> <div class="microscopy" title="sec"> <div class="title" tagx="title" title="title">
Microscopy</div> <p>The mutants were observed by light microscopy.</p> </div> </div> <div class="supplementary-material" title="supplementary-material"> <div class="title" tagx="title" title="title">
Electronic supplementary material</div> <div class="sec" title="sec"> <p> </p><div class="local-data" title="local-data"> <div class="media" title="media"> <a href="41598_2017_5309_MOESM1_ESM.pdf">LINK</a>  <p>Supplementary Information</p>  </div> </div> <p /> </div> </div> </div> <div class="back" title="back"> <div class="fn-group" title="fn-group"> <div class="fn-type-" title=""> <p> <b>Electronic supplementary material</b> </p> <p> <b>Supplementary information</b> accompanies this paper at doi:10.1038/s41598-017-05309-w </p> </div> <div class="fn-type-" title=""> <p> <b>Publisher's note:</b> Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations. </p> </div> </div> <div class="ack" title="ack"> <div class="title" tagx="title" title="title">
Acknowledgements</div> <p>This research was made possible by grants from the Norwegian Research Council (grant FRIMEDBIO 240343) and from Inland Norway University of Applied Sciences. The authors wish to thank the Imaging Centre at The Norwegian University of Life Sciences for providing access to microscopes. The authors also thank Robert Wilson for review of the manuscript. </p> </div> <div class="title" tagx="title" title="title">
Author Contributions</div> <p>A.E.A. and W.J. planned and performed the experiments, analyzed the data, and wrote the manuscript together with O.A.O. P.F.P. performed plant transformation and reviewed the manuscript. J.I.M. contributed to plant transformation and to vector construction. V.D. contributed to vector design and reviewed the manuscript. </p> <div class="competinginterests" title="sec"> <div class="title" tagx="title" title="title">
Competing Interests</div> <p>The authors declare that they have no competing interests.</p> </div> <div class="references">
References</div> <div tag="ref-list"> <ul> <div class="title" tagx="title" title="title">
References</div> <li tag="ref"> <a name="CR1" /> <span class="label" tagx="label" title="label">1.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Wilson</span> <span class="given-names" tagx="given-names" title="given-names">RC</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Doudna</span> <span class="given-names" tagx="given-names" title="given-names">JA</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Molecular mechanisms of RNA interference</span> <span class="source" tagx="source" title="source">Annual review of biophysics</span> <span class="year" tagx="year" title="year">2013</span> <span class="volume" tagx="volume" title="volume">42</span> <span class="fpage" tagx="fpage" title="fpage">217</span> <span class="lpage" tagx="lpage" title="lpage">239</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1146/annurev-biophys-083012-130404">10.1146/annurev-biophys-083012-130404</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/23654304">23654304</a> </span> </span> </li> <li tag="ref"> <a name="CR2" /> <span class="label" tagx="label" title="label">2.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Lange</span> <span class="given-names" tagx="given-names" title="given-names">M</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Yellina</span> <span class="given-names" tagx="given-names" title="given-names">AL</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Orashakova</span> <span class="given-names" tagx="given-names" title="given-names">S</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Becker</span> <span class="given-names" tagx="given-names" title="given-names">A</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Virus-induced gene silencing (VIGS) in plants: an overview of target species and the virus-derived vector systems </span> <span class="source" tagx="source" title="source">Methods in molecular biology (Clifton, N.J.)</span> <span class="year" tagx="year" title="year">2013</span> <span class="volume" tagx="volume" title="volume">975</span> <span class="fpage" tagx="fpage" title="fpage">1</span> <span class="lpage" tagx="lpage" title="lpage">14</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1007/978-1-62703-278-0_1">10.1007/978-1-62703-278-0_1</a> </span> </span> </li> <li tag="ref"> <a name="CR3" /> <span class="label" tagx="label" title="label">3.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Kurowska</span> <span class="given-names" tagx="given-names" title="given-names">M</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span> </span> <span class="mixed-article-title" title="mixed-article-title">TILLING - a shortcut in functional genomics</span> <span class="source" tagx="source" title="source">Journal of Applied Genetics</span> <span class="year" tagx="year" title="year">2011</span> <span class="volume" tagx="volume" title="volume">52</span> <span class="fpage" tagx="fpage" title="fpage">371</span> <span class="lpage" tagx="lpage" title="lpage">390</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1007/s13353-011-0061-1">10.1007/s13353-011-0061-1</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/21912935">21912935</a> </span> </span> </li> <li tag="ref"> <a name="CR4" /> <span class="label" tagx="label" title="label">4.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Bortesi</span> <span class="given-names" tagx="given-names" title="given-names">L</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Fischer</span> <span class="given-names" tagx="given-names" title="given-names">R</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">The CRISPR/Cas9 system for plant genome editing and beyond</span> <span class="source" tagx="source" title="source">Biotechnology advances</span> <span class="year" tagx="year" title="year">2015</span> <span class="volume" tagx="volume" title="volume">33</span> <span class="fpage" tagx="fpage" title="fpage">41</span> <span class="lpage" tagx="lpage" title="lpage">52</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1016/j.biotechadv.2014.12.006">10.1016/j.biotechadv.2014.12.006</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/25536441">25536441</a> </span> </span> </li> <li tag="ref"> <a name="CR5" /> <span class="label" tagx="label" title="label">5.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Schaefer</span> <span class="given-names" tagx="given-names" title="given-names">DG</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Gene targeting in Physcomitrella patens</span> <span class="source" tagx="source" title="source">Current opinion in plant biology</span> <span class="year" tagx="year" title="year">2001</span> <span class="volume" tagx="volume" title="volume">4</span> <span class="fpage" tagx="fpage" title="fpage">143</span> <span class="lpage" tagx="lpage" title="lpage">150</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1016/S1369-5266(00)00150-3">10.1016/S1369-5266(00)00150-3</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/11228437">11228437</a> </span> </span> </li> <li tag="ref"> <a name="CR6" /> <span class="label" tagx="label" title="label">6.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Britt</span> <span class="given-names" tagx="given-names" title="given-names">AB</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">May</span> <span class="given-names" tagx="given-names" title="given-names">GD</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Re-engineering plant gene targeting</span> <span class="source" tagx="source" title="source">Trends Plant Sci</span> <span class="year" tagx="year" title="year">2003</span> <span class="volume" tagx="volume" title="volume">8</span> <span class="fpage" tagx="fpage" title="fpage">90</span> <span class="lpage" tagx="lpage" title="lpage">95</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1016/S1360-1385(03)00002-5">10.1016/S1360-1385(03)00002-5</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/12597876">12597876</a> </span> </span> </li> <li tag="ref"> <a name="CR7" /> <span class="label" tagx="label" title="label">7.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Hanin</span> <span class="given-names" tagx="given-names" title="given-names">M</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Paszkowski</span> <span class="given-names" tagx="given-names" title="given-names">J</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Plant genome modification by homologous recombination</span> <span class="source" tagx="source" title="source">Current opinion in plant biology</span> <span class="year" tagx="year" title="year">2003</span> <span class="volume" tagx="volume" title="volume">6</span> <span class="fpage" tagx="fpage" title="fpage">157</span> <span class="lpage" tagx="lpage" title="lpage">162</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1016/S1369-5266(03)00016-5">10.1016/S1369-5266(03)00016-5</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/12667873">12667873</a> </span> </span> </li> <li tag="ref"> <a name="CR8" /> <span class="label" tagx="label" title="label">8.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Rothstein</span> <span class="given-names" tagx="given-names" title="given-names">R</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Targeting, disruption, replacement, and allele rescue: integrative DNA transformation in yeast </span> <span class="source" tagx="source" title="source">Methods in enzymology</span> <span class="year" tagx="year" title="year">1991</span> <span class="volume" tagx="volume" title="volume">194</span> <span class="fpage" tagx="fpage" title="fpage">281</span> <span class="lpage" tagx="lpage" title="lpage">301</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1016/0076-6879(91)94022-5">10.1016/0076-6879(91)94022-5</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/2005793">2005793</a> </span> </span> </li> <li tag="ref"> <a name="CR9" /> <span class="label" tagx="label" title="label">9.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Deng</span> <span class="given-names" tagx="given-names" title="given-names">C</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Capecchi</span> <span class="given-names" tagx="given-names" title="given-names">MR</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Reexamination of gene targeting frequency as a function of the extent of homology between the targeting vector and the target locus </span> <span class="source" tagx="source" title="source">Mol Cell Biol</span> <span class="year" tagx="year" title="year">1992</span> <span class="volume" tagx="volume" title="volume">12</span> <span class="fpage" tagx="fpage" title="fpage">3365</span> <span class="lpage" tagx="lpage" title="lpage">3371</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1128/MCB.12.8.3365">10.1128/MCB.12.8.3365</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/1321331">1321331</a> </span> </span> </li> <li tag="ref"> <a name="CR10" /> <span class="label" tagx="label" title="label">10.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Sodeinde</span> <span class="given-names" tagx="given-names" title="given-names">OA</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Kindle</span> <span class="given-names" tagx="given-names" title="given-names">KL</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Homologous recombination in the nuclear genome of Chlamydomonas reinhardtii</span> <span class="source" tagx="source" title="source">Proceedings of the National Academy of Sciences</span> <span class="year" tagx="year" title="year">1993</span> <span class="volume" tagx="volume" title="volume">90</span> <span class="fpage" tagx="fpage" title="fpage">9199</span> <span class="lpage" tagx="lpage" title="lpage">9203</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1073/pnas.90.19.9199">10.1073/pnas.90.19.9199</a> </span> </span> </li> <li tag="ref"> <a name="CR11" /> <span class="label" tagx="label" title="label">11.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Collonnier</span> <span class="given-names" tagx="given-names" title="given-names">C</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span> </span> <span class="mixed-article-title" title="mixed-article-title">CRISPR-Cas9-mediated efficient directed mutagenesis and RAD51-dependent and RAD51-independent gene targeting in the moss Physcomitrella patens </span> <span class="source" tagx="source" title="source">Plant Biotechnol J</span> <span class="year" tagx="year" title="year">2016</span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/27368642">27368642</a> </span> </span> </li> <li tag="ref"> <a name="CR12" /> <span class="label" tagx="label" title="label">12.</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation">Lopez-Obando, M. <i>et al</i>. Simple and Efficient Targeting of Multiple Genes Through CRISPR-Cas9 in Physcomitrella patens. <i>G3</i> ( <i>Bethesda</i>, <i>Md</i>.), doi:10.1534/g3.116.033266 (2016). </span> </li> <li tag="ref"> <a name="CR13" /> <span class="label" tagx="label" title="label">13.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">King</span> <span class="given-names" tagx="given-names" title="given-names">BC</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span> </span> <span class="mixed-article-title" title="mixed-article-title"> <i>In vivo</i> assembly of DNA-fragments in the moss, Physcomitrella patens </span> <span class="source" tagx="source" title="source">Scientific reports</span> <span class="year" tagx="year" title="year">2016</span> <span class="volume" tagx="volume" title="volume">6</span> <span class="fpage" tagx="fpage" title="fpage">25030</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1038/srep25030">10.1038/srep25030</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/27126800">27126800</a> </span> </span> </li> <li tag="ref"> <a name="CR14" /> <span class="label" tagx="label" title="label">14.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Gibson</span> <span class="given-names" tagx="given-names" title="given-names">DG</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Complete chemical synthesis, assembly, and cloning of a Mycoplasma genitalium genome</span> <span class="source" tagx="source" title="source">Science (New York, N.Y.)</span> <span class="year" tagx="year" title="year">2008</span> <span class="volume" tagx="volume" title="volume">319</span> <span class="fpage" tagx="fpage" title="fpage">1215</span> <span class="lpage" tagx="lpage" title="lpage">1220</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1126/science.1151721">10.1126/science.1151721</a> </span> </span> </li> <li tag="ref"> <a name="CR15" /> <span class="label" tagx="label" title="label">15.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Shao</span> <span class="given-names" tagx="given-names" title="given-names">Z</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Zhao</span> <span class="given-names" tagx="given-names" title="given-names">H</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Zhao</span> <span class="given-names" tagx="given-names" title="given-names">H</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">DNA assembler, an <i>in vivo</i> genetic method for rapid construction of biochemical pathways </span> <span class="source" tagx="source" title="source">Nucleic acids research</span> <span class="year" tagx="year" title="year">2009</span> <span class="volume" tagx="volume" title="volume">37</span> <span class="fpage" tagx="fpage" title="fpage">e16</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1093/nar/gkn991">10.1093/nar/gkn991</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/19074487">19074487</a> </span> </span> </li> <li tag="ref"> <a name="CR16" /> <span class="label" tagx="label" title="label">16.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Nandi</span> <span class="given-names" tagx="given-names" title="given-names">AK</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Roginski</span> <span class="given-names" tagx="given-names" title="given-names">RS</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Gregg</span> <span class="given-names" tagx="given-names" title="given-names">RG</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Smithies</span> <span class="given-names" tagx="given-names" title="given-names">O</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Skoultchi</span> <span class="given-names" tagx="given-names" title="given-names">AI</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Regulated expression of genes inserted at the human chromosomal beta-globin locus by homologous recombination </span> <span class="source" tagx="source" title="source">Proceedings of the National Academy of Sciences of the United States of America</span> <span class="year" tagx="year" title="year">1988</span> <span class="volume" tagx="volume" title="volume">85</span> <span class="fpage" tagx="fpage" title="fpage">3845</span> <span class="lpage" tagx="lpage" title="lpage">3849</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1073/pnas.85.11.3845">10.1073/pnas.85.11.3845</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/3375244">3375244</a> </span> </span> </li> <li tag="ref"> <a name="CR17" /> <span class="label" tagx="label" title="label">17.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Charlot</span> <span class="given-names" tagx="given-names" title="given-names">F</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span> </span> <span class="mixed-article-title" title="mixed-article-title">RAD51B plays an essential role during somatic and meiotic recombination in Physcomitrella</span> <span class="source" tagx="source" title="source">Nucleic acids research</span> <span class="year" tagx="year" title="year">2014</span> <span class="volume" tagx="volume" title="volume">42</span> <span class="fpage" tagx="fpage" title="fpage">11965</span> <span class="lpage" tagx="lpage" title="lpage">11978</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1093/nar/gku890">10.1093/nar/gku890</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/25260587">25260587</a> </span> </span> </li> <li tag="ref"> <a name="CR18" /> <span class="label" tagx="label" title="label">18.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Kamisugi</span> <span class="given-names" tagx="given-names" title="given-names">Y</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span> </span> <span class="mixed-article-title" title="mixed-article-title">MRE11 and RAD50, but not NBS1, are essential for gene targeting in the moss Physcomitrella patens </span> <span class="source" tagx="source" title="source">Nucleic acids research</span> <span class="year" tagx="year" title="year">2012</span> <span class="volume" tagx="volume" title="volume">40</span> <span class="fpage" tagx="fpage" title="fpage">3496</span> <span class="lpage" tagx="lpage" title="lpage">3510</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1093/nar/gkr1272">10.1093/nar/gkr1272</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/22210882">22210882</a> </span> </span> </li> <li tag="ref"> <a name="CR19" /> <span class="label" tagx="label" title="label">19.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Perroud</span> <span class="given-names" tagx="given-names" title="given-names">PF</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Quatrano</span> <span class="given-names" tagx="given-names" title="given-names">RS</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">The role of ARPC4 in tip growth and alignment of the polar axis in filaments of Physcomitrella patens </span> <span class="source" tagx="source" title="source">Cell motility and the cytoskeleton</span> <span class="year" tagx="year" title="year">2006</span> <span class="volume" tagx="volume" title="volume">63</span> <span class="fpage" tagx="fpage" title="fpage">162</span> <span class="lpage" tagx="lpage" title="lpage">171</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1002/cm.20114">10.1002/cm.20114</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/16450411">16450411</a> </span> </span> </li> <li tag="ref"> <a name="CR20" /> <span class="label" tagx="label" title="label">20.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Schaefer</span> <span class="given-names" tagx="given-names" title="given-names">DG</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span> </span> <span class="mixed-article-title" title="mixed-article-title">RAD51 loss of function abolishes gene targeting and de-represses illegitimate integration in the moss Physcomitrella patens </span> <span class="source" tagx="source" title="source">DNA repair</span> <span class="year" tagx="year" title="year">2010</span> <span class="volume" tagx="volume" title="volume">9</span> <span class="fpage" tagx="fpage" title="fpage">526</span> <span class="lpage" tagx="lpage" title="lpage">533</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1016/j.dnarep.2010.02.001">10.1016/j.dnarep.2010.02.001</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/20189889">20189889</a> </span> </span> </li> <li tag="ref"> <a name="CR21" /> <span class="label" tagx="label" title="label">21.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Spinner</span> <span class="given-names" tagx="given-names" title="given-names">L</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span> </span> <span class="mixed-article-title" title="mixed-article-title">The function of TONNEAU1 in moss reveals ancient mechanisms of division plane specification and cell elongation in land plants </span> <span class="source" tagx="source" title="source">Development</span> <span class="year" tagx="year" title="year">2010</span> <span class="volume" tagx="volume" title="volume">137</span> <span class="fpage" tagx="fpage" title="fpage">2733</span> <span class="lpage" tagx="lpage" title="lpage">2742</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1242/dev.043810">10.1242/dev.043810</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/20663817">20663817</a> </span> </span> </li> <li tag="ref"> <a name="CR22" /> <span class="label" tagx="label" title="label">22.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Wu</span> <span class="given-names" tagx="given-names" title="given-names">SZ</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Ritchie</span> <span class="given-names" tagx="given-names" title="given-names">JA</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Pan</span> <span class="given-names" tagx="given-names" title="given-names">AH</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Quatrano</span> <span class="given-names" tagx="given-names" title="given-names">RS</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Bezanilla</span> <span class="given-names" tagx="given-names" title="given-names">M</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Myosin VIII regulates protonemal patterning and developmental timing in the moss Physcomitrella patens </span> <span class="source" tagx="source" title="source">Molecular plant</span> <span class="year" tagx="year" title="year">2011</span> <span class="volume" tagx="volume" title="volume">4</span> <span class="fpage" tagx="fpage" title="fpage">909</span> <span class="lpage" tagx="lpage" title="lpage">921</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1093/mp/ssr068">10.1093/mp/ssr068</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/21873296">21873296</a> </span> </span> </li> <li tag="ref"> <a name="CR23" /> <span class="label" tagx="label" title="label">23.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Trouiller</span> <span class="given-names" tagx="given-names" title="given-names">B</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Schaefer</span> <span class="given-names" tagx="given-names" title="given-names">DG</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Charlot</span> <span class="given-names" tagx="given-names" title="given-names">F</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Nogue</span> <span class="given-names" tagx="given-names" title="given-names">F</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">MSH2 is essential for the preservation of genome integrity and prevents homeologous recombination in the moss Physcomitrella patens </span> <span class="source" tagx="source" title="source">Nucleic acids research</span> <span class="year" tagx="year" title="year">2006</span> <span class="volume" tagx="volume" title="volume">34</span> <span class="fpage" tagx="fpage" title="fpage">232</span> <span class="lpage" tagx="lpage" title="lpage">242</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1093/nar/gkj423">10.1093/nar/gkj423</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/16397301">16397301</a> </span> </span> </li> <li tag="ref"> <a name="CR24" /> <span class="label" tagx="label" title="label">24.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Uenaka</span> <span class="given-names" tagx="given-names" title="given-names">H</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Kadota</span> <span class="given-names" tagx="given-names" title="given-names">A</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Functional analyses of the Physcomitrella patens phytochromes in regulating chloroplast avoidance movement </span> <span class="source" tagx="source" title="source">The Plant Journal</span> <span class="year" tagx="year" title="year">2007</span> <span class="volume" tagx="volume" title="volume">51</span> <span class="fpage" tagx="fpage" title="fpage">1050</span> <span class="lpage" tagx="lpage" title="lpage">1061</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1111/j.1365-313X.2007.03202.x">10.1111/j.1365-313X.2007.03202.x</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/17662030">17662030</a> </span> </span> </li> <li tag="ref"> <a name="CR25" /> <span class="label" tagx="label" title="label">25.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Dunø</span> <span class="given-names" tagx="given-names" title="given-names">M</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Bendixen</span> <span class="given-names" tagx="given-names" title="given-names">C</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Krejci</span> <span class="given-names" tagx="given-names" title="given-names">L</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Thomsen</span> <span class="given-names" tagx="given-names" title="given-names">B</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Targeted deletions created in yeast vectors by recombinational excision</span> <span class="source" tagx="source" title="source">Nucleic acids research</span> <span class="year" tagx="year" title="year">1999</span> <span class="volume" tagx="volume" title="volume">27</span> <span class="fpage" tagx="fpage" title="fpage">i</span> <span class="lpage" tagx="lpage" title="lpage">iii</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1093/nar/27.8.i">10.1093/nar/27.8.i</a> </span> </span> </li> <li tag="ref"> <a name="CR26" /> <span class="label" tagx="label" title="label">26.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Silva</span> <span class="given-names" tagx="given-names" title="given-names">JC</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Borges</span> <span class="given-names" tagx="given-names" title="given-names">JC</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Cyr</span> <span class="given-names" tagx="given-names" title="given-names">DM</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Ramos</span> <span class="given-names" tagx="given-names" title="given-names">CHI</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Torriani</span> <span class="given-names" tagx="given-names" title="given-names">IL</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Central domain deletions affect the SAXS solution structure and function of Yeast Hsp40 proteins Sis1 and Ydj1 </span> <span class="source" tagx="source" title="source">BMC Structural Biology</span> <span class="year" tagx="year" title="year">2011</span> <span class="volume" tagx="volume" title="volume">11</span> <span class="fpage" tagx="fpage" title="fpage">40</span> <span class="lpage" tagx="lpage" title="lpage">40</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1186/1472-6807-11-40">10.1186/1472-6807-11-40</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/22011374">22011374</a> </span> </span> </li> <li tag="ref"> <a name="CR27" /> <span class="label" tagx="label" title="label">27.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Henderson</span> <span class="given-names" tagx="given-names" title="given-names">DM</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Belanto</span> <span class="given-names" tagx="given-names" title="given-names">JJ</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Li</span> <span class="given-names" tagx="given-names" title="given-names">B</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Heun-Johnson</span> <span class="given-names" tagx="given-names" title="given-names">H</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Ervasti</span> <span class="given-names" tagx="given-names" title="given-names">JM</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Internal deletion compromises the stability of dystrophin</span> <span class="source" tagx="source" title="source">Human molecular genetics</span> <span class="year" tagx="year" title="year">2011</span> <span class="volume" tagx="volume" title="volume">20</span> <span class="fpage" tagx="fpage" title="fpage">2955</span> <span class="lpage" tagx="lpage" title="lpage">2963</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1093/hmg/ddr199">10.1093/hmg/ddr199</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/21558423">21558423</a> </span> </span> </li> <li tag="ref"> <a name="CR28" /> <span class="label" tagx="label" title="label">28.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Mbikay</span> <span class="given-names" tagx="given-names" title="given-names">M</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span> </span> <span class="mixed-article-title" title="mixed-article-title">A targeted deletion/insertion in the mouse Pcsk1 locus is associated with homozygous embryo preimplantation lethality, mutant allele preferential transmission and heterozygous female susceptibility to dietary fat </span> <span class="source" tagx="source" title="source">Developmental Biology</span> <span class="year" tagx="year" title="year">2007</span> <span class="volume" tagx="volume" title="volume">306</span> <span class="fpage" tagx="fpage" title="fpage">584</span> <span class="lpage" tagx="lpage" title="lpage">598</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1016/j.ydbio.2007.03.523">10.1016/j.ydbio.2007.03.523</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/17490633">17490633</a> </span> </span> </li> <li tag="ref"> <a name="CR29" /> <span class="label" tagx="label" title="label">29.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Johansen</span> <span class="given-names" tagx="given-names" title="given-names">W</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span> </span> <span class="mixed-article-title" title="mixed-article-title">The DEK1 calpain Linker functions in three-dimensional body patterning in Physcomitrella patens </span> <span class="source" tagx="source" title="source">Plant physiology</span> <span class="year" tagx="year" title="year">2016</span> <span class="volume" tagx="volume" title="volume">172</span> <span class="fpage" tagx="fpage" title="fpage">1089</span> <span class="lpage" tagx="lpage" title="lpage">1104</span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/27506240">27506240</a> </span> </span> </li> <li tag="ref"> <a name="CR30" /> <span class="label" tagx="label" title="label">30.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Perroud</span> <span class="given-names" tagx="given-names" title="given-names">PF</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Defective Kernel 1 (DEK1) is required for three-dimensional growth in Physcomitrella patens </span> <span class="source" tagx="source" title="source">The New phytologist</span> <span class="year" tagx="year" title="year">2014</span> <span class="volume" tagx="volume" title="volume">203</span> <span class="fpage" tagx="fpage" title="fpage">794</span> <span class="lpage" tagx="lpage" title="lpage">804</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1111/nph.12844">10.1111/nph.12844</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/24844771">24844771</a> </span> </span> </li> <li tag="ref"> <a name="CR31" /> <span class="label" tagx="label" title="label">31.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Wang</span> <span class="given-names" tagx="given-names" title="given-names">C</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span> </span> <span class="mixed-article-title" title="mixed-article-title">The calpain domain of the maize DEK1 protein contains the conserved catalytic triad and functions as a cysteine proteinase </span> <span class="source" tagx="source" title="source">The Journal of biological chemistry</span> <span class="year" tagx="year" title="year">2003</span> <span class="volume" tagx="volume" title="volume">278</span> <span class="fpage" tagx="fpage" title="fpage">34467</span> <span class="lpage" tagx="lpage" title="lpage">34474</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1074/jbc.M300745200">10.1074/jbc.M300745200</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/12824178">12824178</a> </span> </span> </li> <li tag="ref"> <a name="CR32" /> <span class="label" tagx="label" title="label">32.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Schuler</span> <span class="given-names" tagx="given-names" title="given-names">MA</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Splice site requirements and switches in plants</span> <span class="source" tagx="source" title="source">Current topics in microbiology and immunology</span> <span class="year" tagx="year" title="year">2008</span> <span class="volume" tagx="volume" title="volume">326</span> <span class="fpage" tagx="fpage" title="fpage">39</span> <span class="lpage" tagx="lpage" title="lpage">59</span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/18630746">18630746</a> </span> </span> </li> <li tag="ref"> <a name="CR33" /> <span class="label" tagx="label" title="label">33.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Szcześniak</span> <span class="given-names" tagx="given-names" title="given-names">MW</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Kabza</span> <span class="given-names" tagx="given-names" title="given-names">M</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Pokrzywa</span> <span class="given-names" tagx="given-names" title="given-names">R</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Gudyś</span> <span class="given-names" tagx="given-names" title="given-names">A</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Makałowska</span> <span class="given-names" tagx="given-names" title="given-names">I</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">ERISdb: A Database of Plant Splice Sites and Splicing Signals</span> <span class="source" tagx="source" title="source">Plant and Cell Physiology</span> <span class="year" tagx="year" title="year">2013</span> <span class="volume" tagx="volume" title="volume">54</span> <span class="fpage" tagx="fpage" title="fpage">e10</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1093/pcp/pct001">10.1093/pcp/pct001</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/23299413">23299413</a> </span> </span> </li> <li tag="ref"> <a name="CR34" /> <span class="label" tagx="label" title="label">34.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Simpson</span> <span class="given-names" tagx="given-names" title="given-names">CG</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Mutational analysis of a plant branchpoint and polypyrimidine tract required for constitutive splicing of a mini-exon </span> <span class="source" tagx="source" title="source">RNA (New York, N.Y.)</span> <span class="year" tagx="year" title="year">2002</span> <span class="volume" tagx="volume" title="volume">8</span> <span class="fpage" tagx="fpage" title="fpage">47</span> <span class="lpage" tagx="lpage" title="lpage">56</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1017/S1355838202015546">10.1017/S1355838202015546</a> </span> </span> </li> <li tag="ref"> <a name="CR35" /> <span class="label" tagx="label" title="label">35.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Goodall</span> <span class="given-names" tagx="given-names" title="given-names">GJ</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Filipowicz</span> <span class="given-names" tagx="given-names" title="given-names">W</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">The AU-rich sequences present in the introns of plant nuclear pre-mRNAs are required for splicing </span> <span class="source" tagx="source" title="source">Cell</span> <span class="year" tagx="year" title="year">1989</span> <span class="volume" tagx="volume" title="volume">58</span> <span class="fpage" tagx="fpage" title="fpage">473</span> <span class="lpage" tagx="lpage" title="lpage">483</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1016/0092-8674(89)90428-5">10.1016/0092-8674(89)90428-5</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/2758463">2758463</a> </span> </span> </li> <li tag="ref"> <a name="CR36" /> <span class="label" tagx="label" title="label">36.</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation">Yusa, K. <i>et al</i>. Targeted gene correction of alpha(1)-antitrypsin deficiency in induced pluripotent stem cells. <i>Nature</i> <b>478</b>, 391-+, doi:10.1038/nature10424 (2011). </span> </li> <li tag="ref"> <a name="CR37" /> <span class="label" tagx="label" title="label">37.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Nishizawa-Yokoi</span> <span class="given-names" tagx="given-names" title="given-names">A</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Endo</span> <span class="given-names" tagx="given-names" title="given-names">M</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Ohtsuki</span> <span class="given-names" tagx="given-names" title="given-names">N</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Saika</span> <span class="given-names" tagx="given-names" title="given-names">H</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Toki</span> <span class="given-names" tagx="given-names" title="given-names">S</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Precision genome editing in plants via gene targeting and piggyBac-mediated marker excision </span> <span class="source" tagx="source" title="source">The Plant journal: for cell and molecular biology</span> <span class="year" tagx="year" title="year">2015</span> <span class="volume" tagx="volume" title="volume">81</span> <span class="fpage" tagx="fpage" title="fpage">160</span> <span class="lpage" tagx="lpage" title="lpage">168</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1111/tpj.12693">10.1111/tpj.12693</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/25284193">25284193</a> </span> </span> </li> <li tag="ref"> <a name="CR38" /> <span class="label" tagx="label" title="label">38.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Nishizawa-Yokoi</span> <span class="given-names" tagx="given-names" title="given-names">A</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Endo</span> <span class="given-names" tagx="given-names" title="given-names">M</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Osakabe</span> <span class="given-names" tagx="given-names" title="given-names">K</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Saika</span> <span class="given-names" tagx="given-names" title="given-names">H</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Toki</span> <span class="given-names" tagx="given-names" title="given-names">S</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Precise marker excision system using an animal-derived piggyBac transposon in plants</span> <span class="source" tagx="source" title="source">The Plant journal: for cell and molecular biology</span> <span class="year" tagx="year" title="year">2014</span> <span class="volume" tagx="volume" title="volume">77</span> <span class="fpage" tagx="fpage" title="fpage">454</span> <span class="lpage" tagx="lpage" title="lpage">463</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1111/tpj.12367">10.1111/tpj.12367</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/24164672">24164672</a> </span> </span> </li> <li tag="ref"> <a name="CR39" /> <span class="label" tagx="label" title="label">39.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Cary</span> <span class="given-names" tagx="given-names" title="given-names">LC</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Transposon mutagenesis of baculoviruses - analysis of trichoplusia-ni transposon ifp2 insertions within the fp-locus of nuclear polyhedrosis viruses </span> <span class="source" tagx="source" title="source">Virology</span> <span class="year" tagx="year" title="year">1989</span> <span class="volume" tagx="volume" title="volume">172</span> <span class="fpage" tagx="fpage" title="fpage">156</span> <span class="lpage" tagx="lpage" title="lpage">169</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1016/0042-6822(89)90117-7">10.1016/0042-6822(89)90117-7</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/2549707">2549707</a> </span> </span> </li> <li tag="ref"> <a name="CR40" /> <span class="label" tagx="label" title="label">40.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Ding</span> <span class="given-names" tagx="given-names" title="given-names">S</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Efficient transposition of the piggyBac resource (PB) transposon in mammalian cells and mice </span> <span class="source" tagx="source" title="source">Cell</span> <span class="year" tagx="year" title="year">2005</span> <span class="volume" tagx="volume" title="volume">122</span> <span class="fpage" tagx="fpage" title="fpage">473</span> <span class="lpage" tagx="lpage" title="lpage">483</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1016/j.cell.2005.07.013">10.1016/j.cell.2005.07.013</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/16096065">16096065</a> </span> </span> </li> <li tag="ref"> <a name="CR41" /> <span class="label" tagx="label" title="label">41.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Rensing</span> <span class="given-names" tagx="given-names" title="given-names">SA</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Fritzowsky</span> <span class="given-names" tagx="given-names" title="given-names">D</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Lang</span> <span class="given-names" tagx="given-names" title="given-names">D</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Reski</span> <span class="given-names" tagx="given-names" title="given-names">R</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Protein encoding genes in an ancient plant: analysis of codon usage, retained genes and splice sites in a moss, Physcomitrella patens </span> <span class="source" tagx="source" title="source">BMC Genomics</span> <span class="year" tagx="year" title="year">2005</span> <span class="volume" tagx="volume" title="volume">6</span> <span class="fpage" tagx="fpage" title="fpage">1</span> <span class="lpage" tagx="lpage" title="lpage">13</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1186/1471-2164-6-43">10.1186/1471-2164-6-43</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/15627404">15627404</a> </span> </span> </li> <li tag="ref"> <a name="CR42" /> <span class="label" tagx="label" title="label">42.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Perroud</span> <span class="given-names" tagx="given-names" title="given-names">PF</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Quatrano</span> <span class="given-names" tagx="given-names" title="given-names">RS</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">BRICK1 is required for apical cell growth in filaments of the moss Physcomitrella patens but not for gametophore morphology </span> <span class="source" tagx="source" title="source">Plant Cell</span> <span class="year" tagx="year" title="year">2008</span> <span class="volume" tagx="volume" title="volume">20</span> <span class="fpage" tagx="fpage" title="fpage">411</span> <span class="lpage" tagx="lpage" title="lpage">422</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1105/tpc.107.053256">10.1105/tpc.107.053256</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/18263777">18263777</a> </span> </span> </li> <li tag="ref"> <a name="CR43" /> <span class="label" tagx="label" title="label">43.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Cove</span> <span class="given-names" tagx="given-names" title="given-names">DJ</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span> </span> <span class="mixed-article-title" title="mixed-article-title">The moss Physcomitrella patens: a novel model system for plant development and genomic studies </span> <span class="source" tagx="source" title="source">Cold Spring Harbor protocols</span> <span class="year" tagx="year" title="year">2009</span> <span class="volume" tagx="volume" title="volume">2009</span> <span class="fpage" tagx="fpage" title="fpage">pdb.emo115</span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/20147063">20147063</a> </span> </span> </li> <li tag="ref"> <a name="CR44" /> <span class="label" tagx="label" title="label">44.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Platt</span> <span class="given-names" tagx="given-names" title="given-names">AR</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Woodhall</span> <span class="given-names" tagx="given-names" title="given-names">RW</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">George</span> <span class="given-names" tagx="given-names" title="given-names">AL</span> <span class="suffix" tagx="suffix" title="suffix">Jr.</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Improved DNA sequencing quality and efficiency using an optimized fast cycle sequencing protocol </span> <span class="source" tagx="source" title="source">BioTechniques</span> <span class="year" tagx="year" title="year">2007</span> <span class="volume" tagx="volume" title="volume">43</span> <span class="issue" tagx="issue" title="issue">58</span> <span class="fpage" tagx="fpage" title="fpage">60</span> <span class="lpage" tagx="lpage" title="lpage">62</span> </span> </li> </ul> </div> </div>  </body></html>