<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<link rel="dns-prefetch" href="//d33xdlntwy0kbs.cloudfront.net" />
<link rel="shortcut icon" href="https://www.biorxiv.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<meta name="article_thumbnail" content="https://www.biorxiv.org/content/biorxiv/early/2020/01/14/2020.01.12.903427/embed/inline-graphic-1.gif" />
<meta name="type" content="article" />
<meta name="category" content="article" />
<meta name="HW.identifier" content="/biorxiv/early/2020/01/14/2020.01.12.903427.atom" />
<meta name="HW.pisa" content="biorxiv;2020.01.12.903427v1" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="The fate of standing variation and new mutation under climate change" />
<meta name="DC.Identifier" content="10.1101/2020.01.12.903427" />
<meta name="DC.Date" content="2020-01-14" />
<meta name="DC.Publisher" content="Cold Spring Harbor Laboratory" />
<meta name="DC.Rights" content="© 2020, Posted by Cold Spring Harbor Laboratory. This pre-print is available under a Creative Commons License (Attribution-NonCommercial-NoDerivs 4.0 International), CC BY-NC-ND 4.0, as described at http://creativecommons.org/licenses/by-nc-nd/4.0/" />
<meta name="DC.AccessRights" content="restricted" />
<meta name="DC.Description" content="Many species face existence threat under anthropogenic climate change, and standing genetic variation was proposed as a way for sessile species to adapt to novel environments. However, it is still unknown whether standing genetic variants, being adaptive to current environmental variability, are sufficient to guarantee future survival. Here we investigate the relative importance of standing variation versus new mutations and their relative effect sizes from the past to infer the future. While theoretical and experimental evolution studies provided valuable insights, examples in nature are scarce. In the wild banana Musa itinerans in Taiwan, we found enrichment of standing variation with environmental associations. New mutations exert larger effect size per variant in precipitation-related climatic variables, where Taiwan contains extreme environments beyond the ancestral climatic range, and novel alleles have stronger association with novel environments. This pattern is reversed for temperature-related variables, where Taiwan is within the ancestral environmental range. Despite such differences, anthropogenic climate change impacts both types of variants strongly. The results constitute one of the few examples in nature demonstrating that the effect sizes of adaptive variants differ under distinct environmental pressures, and the patterns support theoretical predictions that natural selection favors new mutations with larger effect sizes in a novel environment where the population is far from the adaptive peak. Despite the importance of new mutations, however, the pace of anthropogenic climate change may not allow the accumulation of such mutations." />
<meta name="DC.Contributor" content="Cheng-Yueh Lu" />
<meta name="DC.Contributor" content="Cheng-Ruei Lee" />
<meta name="article:published_time" content="2020-01-14" />
<meta name="article:section" content="New Results" />
<meta name="citation_title" content="The fate of standing variation and new mutation under climate change" />
<meta name="citation_abstract" lang="en" content="&lt;h3&gt;Abstract&lt;/h3&gt;
&lt;p&gt;Many species face existence threat under anthropogenic climate change, and standing genetic variation was proposed as a way for sessile species to adapt to novel environments. However, it is still unknown whether standing genetic variants, being adaptive to current environmental variability, are sufficient to guarantee future survival. Here we investigate the relative importance of standing variation versus new mutations and their relative effect sizes from the past to infer the future. While theoretical and experimental evolution studies provided valuable insights, examples in nature are scarce. In the wild banana &lt;i&gt;Musa itinerans&lt;/i&gt; in Taiwan, we found enrichment of standing variation with environmental associations. New mutations exert larger effect size per variant in precipitation-related climatic variables, where Taiwan contains extreme environments beyond the ancestral climatic range, and novel alleles have stronger association with novel environments. This pattern is reversed for temperature-related variables, where Taiwan is within the ancestral environmental range. Despite such differences, anthropogenic climate change impacts both types of variants strongly. The results constitute one of the few examples in nature demonstrating that the effect sizes of adaptive variants differ under distinct environmental pressures, and the patterns support theoretical predictions that natural selection favors new mutations with larger effect sizes in a novel environment where the population is far from the adaptive peak. Despite the importance of new mutations, however, the pace of anthropogenic climate change may not allow the accumulation of such mutations.&lt;/p&gt;" />
<meta name="citation_journal_title" content="bioRxiv" />
<meta name="citation_publisher" content="Cold Spring Harbor Laboratory" />
<meta name="citation_publication_date" content="2020/01/01" />
<meta name="citation_mjid" content="biorxiv;2020.01.12.903427v1" />
<meta name="citation_id" content="2020.01.12.903427v1" />
<meta name="citation_public_url" content="https://www.biorxiv.org/content/10.1101/2020.01.12.903427v1" />
<meta name="citation_abstract_html_url" content="https://www.biorxiv.org/content/10.1101/2020.01.12.903427v1.abstract" />
<meta name="citation_full_html_url" content="https://www.biorxiv.org/content/10.1101/2020.01.12.903427v1.full" />
<meta name="citation_pdf_url" content="https://www.biorxiv.org/content/biorxiv/early/2020/01/14/2020.01.12.903427.full.pdf" />
<meta name="citation_doi" content="10.1101/2020.01.12.903427" />
<meta name="citation_num_pages" content="32" />
<meta name="citation_article_type" content="Article" />
<meta name="citation_section" content="New Results" />
<meta name="citation_firstpage" content="2020.01.12.903427" />
<meta name="citation_author" content="Cheng-Yueh Lu" />
<meta name="citation_author_institution" content="Institute of Plant Biology, National Taiwan University" />
<meta name="citation_author" content="Cheng-Ruei Lee" />
<meta name="citation_author_institution" content="Institute of Plant Biology, National Taiwan University" />
<meta name="citation_author_institution" content="Institute of Ecology and Evolutionary Biology, National Taiwan University" />
<meta name="citation_author_institution" content="Genome and Systems Biology Degree Program, National Taiwan University" />
<meta name="citation_author_email" content="chengrueilee@ntu.edu.tw" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0002-1913-9964" />
<meta name="citation_reference" content="citation_title=In Advances in Genetics;citation_author=A. D. Bradshaw;citation_pages=115-155;citation_volume=13;citation_year=1965" />
<meta name="citation_reference" content="Ghalambor, C. K., McKay, J. K., Carroll, S. P. &amp; Reznick, D. N. Adaptive versus non-adaptive phenotypic plasticity and the potential for contemporary adaptation in new environments. Funct. Ecol. 21, 394–407, doi: 10.1111/j.1365-2435.2007.01283.x (2007)." />
<meta name="citation_reference" content="Jump, A. S. &amp; Peñuelas, J. Running to stand still: adaptation and the response of plants to rapid climate change. Ecol. Lett. 8, 1010–1020, doi: 10.1111/j.1461-0248.2005.00796.x (2005)." />
<meta name="citation_reference" content="Franks, S. J., Weber, J. J. &amp; Aitken, S. N. Evolutionary and plastic responses to climate change in terrestrial plant populations. Evol. Appl. 7, 123–139, doi: 10.1111/eva.12112 (2014)." />
<meta name="citation_reference" content="Barrett, R. D. H. &amp; Schluter, D. Adaptation from standing genetic variation. Trends Ecol. Evol. 23, 38–44, doi: 10.1016/j.tree.2007.09.008 (2008)." />
<meta name="citation_reference" content="Razgour, O. et al. Considering adaptive genetic variation in climate change vulnerability assessment reduces species range loss projections. Proc. Natl. Acad. Sci.USA 116, 10418–10423, doi: 10.1073/pnas.1820663116 (2019)." />
<meta name="citation_reference" content="Lai, Y. T. et al. Standing genetic variation as the predominant source for adaptation of a songbird. Proc. Natl. Acad. Sci. USA 116, 2152–2157, doi: 10.1073/pnas.1813597116 (2019)." />
<meta name="citation_reference" content="Fisher, R. A. The Genetical Theory of Natural Selection. (Clarendon Press, Oxford, England, 1930)." />
<meta name="citation_reference" content="Kimura, M. The Neutral Theory of Molecular Evolution. (Cambridge University Press, Cambridge, England, 1983)." />
<meta name="citation_reference" content="Orr, H. A. The population genetics of adaptation: The distribution of factors fixed during adaptive evolution. Evolution 52, 935–949, doi: 10.2307/2411226 (1998)." />
<meta name="citation_reference" content="Barrick, J. E. et al. Genome evolution and adaptation in a long-term experiment with Escherichia coli. Nature 461, 1243–1247, doi: 10.1038/nature08480 (2009)." />
<meta name="citation_reference" content="Gifford, D. R., Schoustra, S. E. &amp; Kassen, R. The length of adaptive walks is insensitive to starting fitness in Aspergillus nidulans. Evolution 65, 3070–3078, doi: 10.1111/j.1558-5646.2011.01380.x (2011)." />
<meta name="citation_reference" content="Bramwell, D. &amp; Caujapé-Castells, J. The Biology of Island Floras. (Cambridge University Press, Cambridge, 2011)." />
<meta name="citation_reference" content="Janssens, S. B. et al. Evolutionary dynamics and biogeography of Musaceae reveal a correlation between the diversification of the banana family and the geological and climatic history of Southeast Asia. New Phytol. 210, 1453–1465, doi: 10.1111/nph.13856 (2016)." />
<meta name="citation_reference" content="Chiu, H. L. The collection, evaluation and analysis of genetic diversity of Musa formosana (Warb.) Hayata native in Taiwan. (National Taiwan University, 2004)." />
<meta name="citation_reference" content="Phillips, S. J., Anderson, R. P. &amp; Schapire, R. E. Maximum entropy modeling of species geographic distributions. Ecol. Model. 190, 231–259, doi: 10.1016/j.ecolmodel.2005.03.026 (2006)." />
<meta name="citation_reference" content="Nosil, P., Egan, S. P. &amp; Funk, D. J. Heterogeneous genomic differentiation between walking-stick ecotypes: “isolation by adaptation” and multiple roles for divergent selection. Evolution 62, 316–336, doi: 10.1111/j.1558-5646.2007.00299.x (2008)." />
<meta name="citation_reference" content="Gunther, T. &amp; Coop, G. Robust identification of local adaptation from allele frequencies. Genetics 195, 205–220, doi: 10.1534/genetics.113.152462 (2013)." />
<meta name="citation_reference" content="Ellis, N., Smith, S. J. &amp; Pitcher, C. R. Gradient forests: calculating importance gradients on physical predictors. Ecology 93, 156–168, doi: 10.1890/11-0252.1 (2012)." />
<meta name="citation_reference" content="Fitzpatrick, M. C. &amp; Keller, S. R. Ecological genomics meets community-level modelling of biodiversity: mapping the genomic landscape of current and future environmental adaptation. Ecol. Lett. 18, 1–16, doi: 10.1111/ele.12376 (2015)." />
<meta name="citation_reference" content="Barton, N. H. Genetic hitchhiking. Philos. Trans. R. Soc. Lond. Ser. B-Biol. Sci. 355, 1553–1562, doi: 10.1098/rstb.2000.0716 (2000)." />
<meta name="citation_reference" content="Linnen, C. R., Kingsley, E. P., Jensen, J. D. &amp; Hoekstra, H. E. On the origin and spread of an adaptive allele in deer mice. Science 325, 1095–1098, doi: 10.1126/science.1175826 (2009)." />
<meta name="citation_reference" content="Jones, F. C. et al. The genomic basis of adaptive evolution in threespine sticklebacks. Nature 484, 55–61, doi: 10.1038/nature10944 (2012)." />
<meta name="citation_reference" content="Reid, N. M. et al. The genomic landscape of rapid repeated evolutionary adaptation to toxic pollution in wild fish. Science 354, 1305–1308, doi: 10.1126/science.aah4993 (2016)." />
<meta name="citation_reference" content="Ahrens, C. W., Byrne, M. &amp; Rymer, P. D. Standing genomic variation within coding cand regulatory regions contributes to the adaptive capacity to climate in a foundation tree species. Mol. Ecol. 28, 2502–2516, doi: 10.1111/mec.15092 (2019)." />
<meta name="citation_reference" content="Doyle, J. J. &amp; Doyle, J. L. A rapid DNA isolation procedure for small quantities of fresh leaf tissue. Phytochemical Bulletin 19, 11–15 (1987)." />
<meta name="citation_reference" content="citation_title=PCR Protocols;citation_pages=315-322;citation_year=1990" />
<meta name="citation_reference" content="Taberlet, P., Gielly, L., Pautou, G. &amp; Bouvet, J. Universal primers for amplification of three non-coding regions of chloroplast DNA. Plant Mol. Biol. 17, 1105–1109, doi: 10.1007/BF00037152 (1991)." />
<meta name="citation_reference" content="Oxelman, B., Lidén, M. &amp; Berglund, D. Chloroplast rps16 intron phylogeny of the tribe Sileneae (Caryophyllaceae). Plant Syst. Evol. 206, 393–410, doi: 10.1007/BF00987959 (1997)." />
<meta name="citation_reference" content="Hurr, K. A., Lockhart, P. J., Heenan, P. B. &amp; Penny, D. Evidence for the recent dispersal of Sophora (Leguminosae) around the Southern Oceans: molecular data. J. Biogeogr. 26, 565–577, doi: 10.1046/j.1365-2699.1999.00302.x (1999)." />
<meta name="citation_reference" content="Lagoda, P. J. et al. Sequence tagged microsatellite site (STMS) markers in the Musaceae. Mol. Ecol. 7, 659–663 (1998)." />
<meta name="citation_reference" content="Crouch, H. K., Crouch, J. H., Jarret, R. L., Cregan, P. B. &amp; Ortiz, R. Segregation at microsatellite loci in haploid and diploid gametes of Musa. Crop Sci. 38, 211–217, doi: 10.2135/cropsci1998.0011183X003800010035x (1998)." />
<meta name="citation_reference" content="Martin, G. et al. Improvement of the banana “Musa acuminata” reference sequence using NGS data and semi-automated bioinformatics methods. BMC Genomics 17, 243, doi: 10.1186/s12864-016-2579-4 (2016)." />
<meta name="citation_reference" content="Schuelke, M. An economic method for the fluorescent labeling of PCR fragments. Nat. Biotechnol. 18, 233–234, doi: 10.1038/72708 (2000)." />
<meta name="citation_reference" content="Pritchard, J. K., Stephens, M. &amp; Donnelly, P. Inference of population structure using multilocus genotype data. Genetics 155, 945–959 (2000)." />
<meta name="citation_reference" content="Falush, D., Stephens, M. &amp; Pritchard, J. K. Inference of population structure using multilocus genotype data: linked loci and correlated allele frequencies. Genetics 164, 1567–1587 (2003)." />
<meta name="citation_reference" content="Peakall, R. O. D. &amp; Smouse, P. E. Genalex 6: genetic analysis in Excel. Population genetic software for teaching and research. Mol. Ecol. Notes 6, 288–295, doi: 10.1111/j.1471-8286.2005.01155.x (2006)." />
<meta name="citation_reference" content="Peakall, R. &amp; Smouse, P. E. GenAlEx 6.5: genetic analysis in Excel. Population genetic software for teaching and research--an update. Bioinformatics 28, 2537–2539, doi: 10.1093/bioinformatics/bts460 (2012)." />
<meta name="citation_reference" content="Bean, W. T., Stafford, R. &amp; Brashares, J. S. The effects of small sample size and sample bias on threshold selection and accuracy assessment of species distribution models. Ecography 35, 250–258, doi: 10.1111/j.1600-0587.2011.06545.x (2012)." />
<meta name="citation_reference" content="Elith, J. et al. Novel methods improve prediction of species’ distributions from occurrence data. Ecography 29, 129–151, doi: 10.1111/j.2006.0906-7590.04596.x (2006)." />
<meta name="citation_reference" content="Elith, J. et al. A statistical explanation of MaxEnt for ecologists. Divers. Distrib. 17, 43–57, doi: 10.1111/j.1472-4642.2010.00725.x (2011)." />
<meta name="citation_reference" content="Hijmans, R. J., Cameron, S. E., Parra, J. L., Jones, P. G. &amp; Jarvis, A. Very high resolution interpolated climate surfaces for global land areas. Int. J. Climatol. 25, 1965–1978, doi: 10.1002/joc.1276 (2005)." />
<meta name="citation_reference" content="Warren, D. L., Glor, R. E. &amp; Turelli, M. ENMTools: a toolbox for comparative studies of environmental niche models. Ecography 33, 607–611, doi: 10.1111/j.1600-0587.2009.06142.x (2010)." />
<meta name="citation_reference" content="Gent, P. R. et al. The Community Climate System Model Version 4. J. Climate 24, 4973–4991, doi: 10.1175/2011jcli4083.1 (2011)." />
<meta name="citation_reference" content="Watanabe, S. et al. MIROC-ESM 2010: model description and basic results of CMIP5-20c3m experiments. Geosci. Model Dev. 4, 845–872, doi: 10.5194/gmd-4-845-2011 (2011)." />
<meta name="citation_reference" content="Brown, J. L. &amp; Anderson, B. SDMtoolbox: a python-based GIS toolkit for landscape genetic, biogeographic and species distribution model analyses. Methods Ecol. Evol. 5, 694–700, doi: 10.1111/2041-210x.12200 (2014)." />
<meta name="citation_reference" content="Schlotterer, C., Tobler, R., Kofler, R. &amp; Nolte, V. Sequencing pools of individuals - mining genome-wide polymorphism data without big funding. Nat. Rev. Genet. 15, 749–763, doi: 10.1038/nrg3803 (2014)." />
<meta name="citation_reference" content="Cox, M. P., Peterson, D. A. &amp; Biggs, P. J. SolexaQA: At-a-glance quality assessment of Illumina second-generation sequencing data. BMC Bioinformatics 11, 485, doi: 10.1186/1471-2105-11-485 (2010)." />
<meta name="citation_reference" content="Martin, M. Cutadapt removes adapter sequences from high-throughput sequencing reads. EMBnet.journal 17, 10–12, doi: 10.14806/ej.17.1.200 (2011)." />
<meta name="citation_reference" content="Wu, W. et al. Whole genome sequencing of a banana wild relative Musa itinerans provides insights into lineage-specific diversification of the Musa genus. Sci. Rep. 6, 31586, doi: 10.1038/srep31586 (2016)." />
<meta name="citation_reference" content="Li, H. &amp; Durbin, R. Fast and accurate short read alignment with Burrows-Wheeler transform. Bioinformatics 25, 1754–1760, doi: 10.1093/bioinformatics/btp324 (2009)." />
<meta name="citation_reference" content="McKenna, A. et al. The Genome Analysis Toolkit: A MapReduce framework for analyzing next-generation DNA sequencing data. Genome Res. 20, 1297–1303, doi: 10.1101/gr.107524.110 (2010)." />
<meta name="citation_reference" content="Wu, W., Ng, W. L., Yang, J. X., Li, W. M. &amp; Ge, X. J. High cryptic species diversity is revealed by genome-wide polymorphisms in a wild relative of banana, Musa itinerans, and implications for its conservation in subtropical China. BMC Plant Biol. 18, 194, doi: 10.1186/s12870-018-1410-6 (2018)." />
<meta name="twitter:title" content="The fate of standing variation and new mutation under climate change" />
<meta name="twitter:site" content="@biorxivpreprint" />
<meta name="twitter:card" content="summary" />
<meta name="twitter:image" content="https://www.biorxiv.org/sites/default/files/images/biorxiv_logo_homepage7-5-small.png" />
<meta name="twitter:description" content="Many species face existence threat under anthropogenic climate change, and standing genetic variation was proposed as a way for sessile species to adapt to novel environments. However, it is still unknown whether standing genetic variants, being adaptive to current environmental variability, are sufficient to guarantee future survival. Here we investigate the relative importance of standing variation versus new mutations and their relative effect sizes from the past to infer the future. While theoretical and experimental evolution studies provided valuable insights, examples in nature are scarce. In the wild banana Musa itinerans in Taiwan, we found enrichment of standing variation with environmental associations. New mutations exert larger effect size per variant in precipitation-related climatic variables, where Taiwan contains extreme environments beyond the ancestral climatic range, and novel alleles have stronger association with novel environments. This pattern is reversed for temperature-related variables, where Taiwan is within the ancestral environmental range. Despite such differences, anthropogenic climate change impacts both types of variants strongly. The results constitute one of the few examples in nature demonstrating that the effect sizes of adaptive variants differ under distinct environmental pressures, and the patterns support theoretical predictions that natural selection favors new mutations with larger effect sizes in a novel environment where the population is far from the adaptive peak. Despite the importance of new mutations, however, the pace of anthropogenic climate change may not allow the accumulation of such mutations." />
<meta name="og-title" property="og:title" content="The fate of standing variation and new mutation under climate change" />
<meta name="og-url" property="og:url" content="https://www.biorxiv.org/content/10.1101/2020.01.12.903427v1" />
<meta name="og-site-name" property="og:site_name" content="bioRxiv" />
<meta name="og-description" property="og:description" content="Many species face existence threat under anthropogenic climate change, and standing genetic variation was proposed as a way for sessile species to adapt to novel environments. However, it is still unknown whether standing genetic variants, being adaptive to current environmental variability, are sufficient to guarantee future survival. Here we investigate the relative importance of standing variation versus new mutations and their relative effect sizes from the past to infer the future. While theoretical and experimental evolution studies provided valuable insights, examples in nature are scarce. In the wild banana Musa itinerans in Taiwan, we found enrichment of standing variation with environmental associations. New mutations exert larger effect size per variant in precipitation-related climatic variables, where Taiwan contains extreme environments beyond the ancestral climatic range, and novel alleles have stronger association with novel environments. This pattern is reversed for temperature-related variables, where Taiwan is within the ancestral environmental range. Despite such differences, anthropogenic climate change impacts both types of variants strongly. The results constitute one of the few examples in nature demonstrating that the effect sizes of adaptive variants differ under distinct environmental pressures, and the patterns support theoretical predictions that natural selection favors new mutations with larger effect sizes in a novel environment where the population is far from the adaptive peak. Despite the importance of new mutations, however, the pace of anthropogenic climate change may not allow the accumulation of such mutations." />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.biorxiv.org/sites/default/files/images/biorxiv_logo_homepage7-5-small.png" />
<meta name="citation_date" content="2020-01-14" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/10.1101/2020.01.12.903427v1.full.pdf" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/10.1101/2020.01.12.903427v1.full.txt" />
<link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/10.1101/2020.01.12.903427v1.ppt" />
<meta name="description" content="bioRxiv - the preprint server for biology, operated by Cold Spring Harbor Laboratory, a research and educational institution" />
<meta name="generator" content="Drupal 7 (http://drupal.org)" />
<link rel="canonical" href="https://www.biorxiv.org/content/10.1101/2020.01.12.903427v1" />
<link rel="shortlink" href="https://www.biorxiv.org/node/1105568" />
    <title>The fate of standing variation and new mutation under climate change | bioRxiv</title>  
    <link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__jMRAK66KMC1e4TQlwUNn3KiWVDC5AjueUAYEm1xBY_U__KSUjdT4jdcJ4qJz6fKY1K9WkYh1a5EcaWZxt_-zbTis__dPhHATyREQrnntSIgxU8XNoYny4rjInxP7lLKSNxQyY.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q5yzns" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__icrP-Y0qw2NP73rJxxkUtrJta0z3VLAABtNjfSdEghA__rlN3rx0wOoswGNHZMeZM96bNn-3cLWHXTztGDoThNlw__dPhHATyREQrnntSIgxU8XNoYny4rjInxP7lLKSNxQyY.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__q48SDxVDWut_cH6dTPyQe7HjADuSLLmJVFztuiJEFrU__DJqddQGjm5dXCHsaiwNriGytKN5eevIQqyjdq65TZtE__dPhHATyREQrnntSIgxU8XNoYny4rjInxP7lLKSNxQyY.css" media="all" />
<style type="text/css" media="all">
/* <![CDATA[ */
#sliding-popup.sliding-popup-bottom,#sliding-popup.sliding-popup-bottom .eu-cookie-withdraw-banner,.eu-cookie-withdraw-tab{background:gray}#sliding-popup.sliding-popup-bottom.eu-cookie-withdraw-wrapper{background:transparent}#sliding-popup .popup-content #popup-text h1,#sliding-popup .popup-content #popup-text h2,#sliding-popup .popup-content #popup-text h3,#sliding-popup .popup-content #popup-text p,.eu-cookie-compliance-secondary-button,.eu-cookie-withdraw-tab{color:#fff !important}.eu-cookie-withdraw-tab{border-color:#fff}.eu-cookie-compliance-more-button{color:#fff !important}
/* ]]> */
</style>

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__dPhHATyREQrnntSIgxU8XNoYny4rjInxP7lLKSNxQyY.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__B6tGdsQSt1vT8d8j-19-oS4FKJe83RrSNBkjPs9jfos__6VkE5WcQmmQpSVTQx-xS2-zn_nlwFlDm-EN1HLJNjrA__dPhHATyREQrnntSIgxU8XNoYny4rjInxP7lLKSNxQyY.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__Cm80tkaB8yhCBXUaefYJe7GLLFWYsVxjke4GKdtw7XY__F3sVKlGyHKCaAPI_WdiHD7l_-05B6b9bLdUwzi1me90__dPhHATyREQrnntSIgxU8XNoYny4rjInxP7lLKSNxQyY.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__dPhHATyREQrnntSIgxU8XNoYny4rjInxP7lLKSNxQyY.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__dPhHATyREQrnntSIgxU8XNoYny4rjInxP7lLKSNxQyY.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__O9AieMsHBlZYeQh-sAOhdoS_nAEDhkj0jDZjGc_kUug__f_9pwrTE8mmshoJRfurwkQ36QonD_Jiwbop2sscnkrE__dPhHATyREQrnntSIgxU8XNoYny4rjInxP7lLKSNxQyY.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://d33xdlntwy0kbs.cloudfront.net/cshl_custom.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/all/libraries/modernizr/modernizr.min.js?q5yzns"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__yOhrJUBmMpN-jsuQlJYmS8ovkQU6I6KInr295HrSTb0__c--zHhbuOWmkdWkJYXiNWDmkpDRPesmbBjGlQcyqBOY__dPhHATyREQrnntSIgxU8XNoYny4rjInxP7lLKSNxQyY.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var _prum=[['id', '52e95df3abe53d2e33000000'], ['mark', 'firstbyte',
      (new Date()).getTime()]]; (function() {
      var s=document.getElementsByTagName('script')[0],
      p=document.createElement('script');
      p.async='async'; p.src='//rum-static.pingdom.net/prum.min.js';s.parentNode.insertBefore(p,s);})();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js___KyvuEvfmewWxdOLBXTYsSUo8Pfm_DijV6fWtGvSL_k__HSKkTgIr7nDcdvQYewipf6v4ak6ZLUOQkwPj44wKouA__dPhHATyREQrnntSIgxU8XNoYny4rjInxP7lLKSNxQyY.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__3uCzzYE5jr6ly9nvsM8PRBRkhh4p0NWCMDja4EKBy4M__dmGGVYcB8LhnbR2iAmVW0ZyXI6QuFVYFRORa5XxglXs__dPhHATyREQrnntSIgxU8XNoYny4rjInxP7lLKSNxQyY.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__0zdUME0GLauYYyEch2tGPydIw-IG7Uizip0Qb4wUsig__3RgszrCgk4bOnTzo4vaQ-hrWzaG6oD19l-R9CQ1YQ-o__dPhHATyREQrnntSIgxU8XNoYny4rjInxP7lLKSNxQyY.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__W6wAjopaALytcBam7oT3WhEzllkWaB9R-8hazHyvShM__rK5YBIqBy9qt0di0Sn4DriBx9TJupdmv6uGmLJyZvDs__dPhHATyREQrnntSIgxU8XNoYny4rjInxP7lLKSNxQyY.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-45638026-1", {"cookieDomain":"auto"});ga("set", "page", location.pathname + location.search + location.hash);ga("send", "pageview");ga('create', 'UA-189672-38', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__1PtAOZ8x7l6QAi8D7X6Kj_Y7HcUOFzvWRvvYHUcCT98__U8IB92u_CnnGwJ-X_xaUa9lzsC8OzUfN0WkK2JQfirg__dPhHATyREQrnntSIgxU8XNoYny4rjInxP7lLKSNxQyY.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings,{"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"m9yc6lyMHy5XpWenxb4kBC53LR-xShCAlpRGcA0nZbg","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"modules\/comment\/comment.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"modules\/forum\/forum.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"sites\/default\/modules\/biorxiv\/css\/biorxiv.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/flexible\/flexible.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_collection\/css\/highwire_subject_collection.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire.highwire-list-expand.css":1,"public:\/\/ctools\/css\/bab59090150caf0c64e27afa1b36eef1.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/contrib\/eu_cookie_compliance\/css\/eu_cookie_compliance.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"0":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_biorxiv_styles\/css\/biorxiv-styles.css.less":1,"https:\/\/d33xdlntwy0kbs.cloudfront.net\/cshl_custom.css":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_figures.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_tables.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"http:\/\/d33xdlntwy0kbs.cloudfront.net\/cshl_custom.js":1,"sites\/all\/modules\/contrib\/eu_cookie_compliance\/js\/eu_cookie_compliance.js":1,"sites\/default\/modules\/jnl_biorxiv_styles\/js\/adv-search-collection-toggle.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/eu_cookie_compliance\/js\/jquery.cookie-1.4.1.min.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/default\/modules\/jnl_biorxiv_styles\/js\/biorxiv-scripts.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_nps_survey\/js\/nps_survey.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"sites\/default\/modules\/contrib\/disqus\/disqus.js":1,"misc\/textarea.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/contrib\/service_links\/js\/twitter_button.js":1,"sites\/all\/modules\/contrib\/service_links\/js\/facebook_like.js":1,"sites\/all\/modules\/contrib\/service_links\/js\/google_plus_one.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.highwire-list-expand.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"1105568","apath":"\/biorxiv\/early\/2020\/01\/14\/2020.01.12.903427.atom","pisa":"biorxiv;2020.01.12.903427v1","processed":["highwire_math"],"markup":[{"requested":"full-text","variant":"full-text","view":"full","pisa":"biorxiv;2020.01.12.903427v1"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"disqus":{"domain":"biorxivstage","url":"https:\/\/www.biorxiv.org\/content\/10.1101\/2020.01.12.903427v1","title":"The fate of standing variation and new mutation under climate change","identifier":"node\/1105568"},"panels_ajax_pane":{"new-41":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022P1DpAUClGMNLrzSr4DYxriinrygQ4l97Om2N8\\\\\\\/av3HDmKKWF35j8FruUkAq7IANkjcatSGEKoVfjxd9cF8q3eKP3vmNa0jtu9pJSnb55PjMGUI+At8bl715xdyjWl7duyxkquxBppET2DoMTVm9eN+gcdvp2GT9MBOpJmK5s\\\\\\\/NnOyZ\\\\\\\/FWLMEZh3JXrpUxRY78wBu005Jz02KabtuKo8\\\\\\\/IttpErJ\\\\\\\/MHJeF0UrGPav1Ic\\\\\\\/Okt20MQWIp440WfvNqfXr45NfE8G+ITTf5fBuLP60\\\\\\\/nJyW3fUVTWVTgmpATO9zM8eAL0ILCOum3TwnTG26cURwYmLX1FJv0c9SZYndFJRTPslh61gQhU3hDNjSvfcNm4GtLi7N1J2GNMP+WXu7420IPlG+DB\\\\\\\/aBmmRELPiM8frQy3Trl4\\\\\\\/QYxFd8iuS\\\\\\\/X6+GmiQk8H6TZFFqEFAYLOcMA57YyTxqlXzF8W4RFyacJaSHmFwdlXzUkRyBjQkCddv1A62y8weMbyxMAVuXRyal0nytC7zf9amDlFRKrfR9PeQY5ySa6zXcnQMH2A79GZLtUS9kaGOEAR7TW6OdNuphcnRFzXEsLJPRERs\\\\\\\/xZ\\\\\\\/0LaIh0AI8e\\\\\\\/QROcoSQKcQXDBDxawTAIQJ\\\\\\\/uRX5Kg1Fol3hPud4r6oStTZKqH2QaFyhm8\\\\\\\/8HWvJ3iy7ALZm5cAgD0NzDPuyjwgBcJqSX+CJHmWWotwNepmpzLQK4L+lSkGICuYNu+i+lL\\\\\\\/7SdqfZmZbxaU6lCXL7WJbE3dfC0RnEpU7Q1rj95ege53AXYIYCmSWvd\\\\\\\/qBuPFOrsAXt8UdLIrZWU0WkmGlwG3d5Mi8NYnbJ7CsPfkS\\\\\\\/jC8wi3GR5Psp2fBEJwh+PwvYDOlDkBRBAeqOqlM3rB3QXr8HOu6LMGTHfuNN8QVcPNn9bieRK9ZPKxs3m1GqvekSks7AriqgFhdRRYRfJAD+W6GRc2XRvH78RAlnzct36wNbljdvev+qY5r4IO0tgWCWoAyAL4fyuurjjU4g8ueAJjqibFn1a89zeVgOITyAKzW2GciCtkHYe4yO1KtsQmzsmyLjBX5v+DcIi9xOmElKntlHNv\\\\\\\/K3HgmZKozKnAsc82ecYS+l34K7eIJ1m\\\\\\\/tIxBhPY2PWlr3j2DsfNxA\\\\\\\/gLZ0FUWMVk2hZOkOxo4v3EWKleclpv6uIa2OsRXDIs89wZVjHHUPOxso\\\\\\\/gnGIpKOY7tLPaEKqgTajtzpmNn7kOaW3aCLmfTAcrJdPyqxPWGbAOSKiKyMV165tQC47aCfNlyghhD9Ml1vKso3YWrQ9kOZUZWcxMPfwuLsa2RDxHMzeNUr0JeSHIObZulkcH801pFBiAYrzlpg\\\\\\\/gGa\\\u0022,\\\u0022iv\\\u0022:\\\u0022nl6opB9JCZqZl31nQFAmUg==\\\u0022,\\\u0022salt\\\u0022:\\\u0022f9d8845712f72c9fee5db7d9ff111eab\\\u0022}\u0022,\u0022hmac\u0022:\u0022faa282389adb56f1529152df31f867c4ae5925a038c28c08816772e57e151446\u0022}"},"urlIsAjaxTrusted":{"\/content\/10.1101\/2020.01.12.903427v1.full":true},"ws_fl":{"width":100,"height":21},"ws_gpo":{"size":"","annotation":"","lang":"","callback":"","width":300},"color":{"logo":"https:\/\/www.biorxiv.org\/sites\/default\/files\/bioRxiv_article.jpg"},"highwire_list_expand":{"is_collapsed":"1"},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"eu_cookie_compliance":{"popup_enabled":1,"popup_agreed_enabled":0,"popup_hide_agreed":0,"popup_clicking_confirmation":1,"popup_scrolling_confirmation":false,"popup_html_info":"\u003Cdiv\u003E\n  \u003Cdiv class =\u0022popup-content info\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n      \u003Cp\u003EWe use cookies on this site to enhance your user experience. By clicking any link on this page you are giving your consent for us to set cookies.\u003C\/p\u003E\n    \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022agree-button eu-cookie-compliance-default-button\u0022\u003EContinue\u003C\/button\u003E\n                    \u003Cbutton type=\u0022button\u0022 class=\u0022find-more-button eu-cookie-compliance-more-button\u0022\u003EFind out more\u003C\/button\u003E\n          \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E","use_mobile_message":false,"mobile_popup_html_info":"\u003Cdiv\u003E\n  \u003Cdiv class =\u0022popup-content info\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n          \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022agree-button eu-cookie-compliance-default-button\u0022\u003EContinue\u003C\/button\u003E\n                    \u003Cbutton type=\u0022button\u0022 class=\u0022find-more-button eu-cookie-compliance-more-button\u0022\u003EFind out more\u003C\/button\u003E\n          \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E\n","mobile_breakpoint":"768","popup_html_agreed":"\u003Cdiv\u003E\n  \u003Cdiv class=\u0022popup-content agreed\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n      \u003Ch2\u003EThank you for accepting cookies\u003C\/h2\u003E\u003Cp\u003EYou can now hide this message or find out more about cookies.\u003C\/p\u003E    \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022hide-popup-button eu-cookie-compliance-hide-button\u0022\u003EHide\u003C\/button\u003E\n              \u003Cbutton type=\u0022button\u0022 class=\u0022find-more-button eu-cookie-compliance-more-button-thank-you\u0022 \u003EMore info\u003C\/button\u003E\n          \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E","popup_use_bare_css":false,"popup_height":"auto","popup_width":"100%","popup_delay":1000,"popup_link":"\/help\/cookie-policy","popup_link_new_window":1,"popup_position":null,"popup_language":"en","store_consent":false,"better_support_for_screen_readers":0,"reload_page":0,"domain":"","popup_eu_only_js":0,"cookie_lifetime":"365","cookie_session":false,"disagree_do_not_show_popup":0,"method":"default","whitelisted_cookies":"","withdraw_markup":"\u003Cbutton type=\u0022button\u0022 class=\u0022eu-cookie-withdraw-tab\u0022\u003EPrivacy settings\u003C\/button\u003E\n\u003Cdiv class=\u0022eu-cookie-withdraw-banner\u0022\u003E\n  \u003Cdiv class=\u0022popup-content info\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n      \u003Cp\u003E\u0026lt;h2\u0026gt;We use cookies on this site to enhance your user experience\u0026lt;\/h2\u0026gt;\u0026lt;p\u0026gt;You have given your consent for us to set cookies.\u0026lt;\/p\u0026gt;\u003C\/p\u003E\n    \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022eu-cookie-withdraw-button\u0022\u003EWithdraw consent\u003C\/button\u003E\n    \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E\n","withdraw_enabled":false},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1,"trackUrlFragments":1},"jnl_biorxiv_styles":{"defaultJCode":"biorxiv"},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-1105568 node-type-highwire-article context-content hw-default-jcode-biorxiv hw-article-type-article hw-article-category-new-results">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-M677548" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-M677548');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
        <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
    
  <div id="zone-branding" class="zone zone-branding clearfix print-display-block container-30">
    <div class="grid-15 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="bioRxiv" src="https://www.biorxiv.org/sites/default/files/bioRxiv_article.jpg" /></a>      </div>
                </div>
          </div>
</div><div class="grid-11 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-system block-menu block-main-menu block-system-main-menu odd block-without-title" id="block-system-main-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/" title="" class="" data-icon-position="" data-hide-link-title="0">Home</a></li>
<li class="leaf" role="menuitem"><a href="/about-biorxiv" class="" data-icon-position="" data-hide-link-title="0">About</a></li>
<li class="leaf" role="menuitem"><a href="/submit-a-manuscript" class="" data-icon-position="" data-hide-link-title="0">Submit</a></li>
<li class="last leaf" role="menuitem"><a href="/content/alertsrss" title="" class="" data-icon-position="" data-hide-link-title="0">ALERTS / RSS</a></li>
</ul></nav>    </div>
  </div>
</div><div class="block block-panels-mini block-biorxiv-search-box block-panels-mini-biorxiv-search-box even block-without-title" id="block-panels-mini-biorxiv-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-biorxiv_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/10.1101/2020.01.12.903427v1.full" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--2" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-WL-kVxBkJthxfrKn1ulkwd5Oy1uCeWz_ZBtJufJfu-g" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2 advanced-search-link" >
  
      
  
  <div class="pane-content">
    <a href="/search">Advanced Search</a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix container-30">
  	      </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="1105568" id="node1105568" data-pisa="biorxiv;2020.01.12.903427v1" data-pisa-master="biorxiv;2020.01.12.903427" data-apath="/biorxiv/early/2020/01/14/2020.01.12.903427.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div  class="highwire-cite highwire-cite-highwire-article highwire-citation-biorxiv-article-top clearfix has-author-tooltip" >

      <span class="biorxiv-article-type">
        New Results    </span>
  
        <h1 class="highwire-cite-title" id="page-title">The fate of standing variation and new mutation under climate change</h1>  
      <div  class="highwire-cite-authors" ><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0"><span class="nlm-given-names">Cheng-Yueh</span> <span class="nlm-surname">Lu</span></span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="1"><a href="http://orcid.org/0000-0002-1913-9964" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a><span class="nlm-given-names">Cheng-Ruei</span> <span class="nlm-surname">Lee</span></span></span></div>
  
      <div  class="highwire-cite-metadata" ><span  class="highwire-cite-metadata-doi highwire-cite-metadata"><span class="label">doi:</span> https://doi.org/10.1101/2020.01.12.903427 </span></div>
  
  
  </div>
<div id="hw-article-author-popups-node1105568" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Cheng-Yueh Lu </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-institution'>Institute of Plant Biology, National Taiwan University</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Cheng-Yueh%2BLu%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Lu%20C&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ACheng-Yueh%2BLu%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Cheng-Ruei Lee </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-institution'>Institute of Plant Biology, National Taiwan University</span></div><div class='author-affiliation'><span class='nlm-institution'>Institute of Ecology and Evolutionary Biology, National Taiwan University</span></div><div class='author-affiliation'><span class='nlm-institution'>Genome and Systems Biology Degree Program, National Taiwan University</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Cheng-Ruei%2BLee%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Lee%20C&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ACheng-Ruei%2BLee%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0002-1913-9964" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Cheng-Ruei Lee</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:chengrueilee@ntu.edu.tw" class="" data-icon-position="" data-hide-link-title="0">chengrueilee@ntu.edu.tw</a></span></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/10.1101/2020.01.12.903427v1" class="panels-ajax-tab-tab" data-panel-name="biorxiv_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:1105568" data-trigger="" data-url-enabled="1">Abstract</a><a href="/panels_ajax_tab/biorxiv_tab_art/node:1105568/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/10.1101/2020.01.12.903427v1.full-text" class="panels-ajax-tab-tab" data-panel-name="article_tab_full_text" data-target-id="highwire_article_tabs" data-entity-context="node:1105568" data-trigger="full-text" data-url-enabled="1">Full Text</a><a href="/panels_ajax_tab/article_tab_full_text/node:1105568/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/10.1101/2020.01.12.903427v1.article-info" class="panels-ajax-tab-tab" data-panel-name="biorxiv_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:1105568" data-trigger="article-info" data-url-enabled="1">Info/History</a><a href="/panels_ajax_tab/biorxiv_tab_info/node:1105568/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/10.1101/2020.01.12.903427v1.article-metrics" class="panels-ajax-tab-tab" data-panel-name="article_tab_metrics" data-target-id="highwire_article_tabs" data-entity-context="node:1105568" data-trigger="article-metrics" data-url-enabled="1">Metrics</a><a href="/panels_ajax_tab/article_tab_metrics/node:1105568/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/10.1101/2020.01.12.903427v1.full.pdf+html" class="panels-ajax-tab-tab" data-panel-name="biorxiv_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:1105568" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i> Preview PDF</a><a href="/panels_ajax_tab/biorxiv_tab_pdf/node:1105568/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="article_tab_full_text" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.biorxiv.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-article_tab_full_text"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-markup" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2 class="">Abstract</h2><p id="p-2">Many species face existence threat under anthropogenic climate change, and standing genetic variation was proposed as a way for sessile species to adapt to novel environments. However, it is still unknown whether standing genetic variants, being adaptive to current environmental variability, are sufficient to guarantee future survival. Here we investigate the relative importance of standing variation versus new mutations and their relative effect sizes from the past to infer the future. While theoretical and experimental evolution studies provided valuable insights, examples in nature are scarce. In the wild banana <em>Musa itinerans</em> in Taiwan, we found enrichment of standing variation with environmental associations. New mutations exert larger effect size per variant in precipitation-related climatic variables, where Taiwan contains extreme environments beyond the ancestral climatic range, and novel alleles have stronger association with novel environments. This pattern is reversed for temperature-related variables, where Taiwan is within the ancestral environmental range. Despite such differences, anthropogenic climate change impacts both types of variants strongly. The results constitute one of the few examples in nature demonstrating that the effect sizes of adaptive variants differ under distinct environmental pressures, and the patterns support theoretical predictions that natural selection favors new mutations with larger effect sizes in a novel environment where the population is far from the adaptive peak. Despite the importance of new mutations, however, the pace of anthropogenic climate change may not allow the accumulation of such mutations.</p></div><div class="section" id="sec-1"><h2 class="">Introduction</h2><p id="p-3">Anthropogenic climatic change posts an imminent threat to most organisms. For large and sessile plant species with long generation time, the speed of migration may not keep up with environmental change, and therefore phenotypic plasticity and genetic variation in the population may allow their survival under novel environments<sup><a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">1</a>-<a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">4</a></sup>. Adaptive genetic variation originates from standing variants before or new mutations after environmental change. Since anthropogenic climate change greatly outpaces natural mutation, the amount of standing genetic variation is therefore critical for the rapid response of a population to changing environments<sup><a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">5</a>-<a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">7</a></sup>. It remains unclear, however, whether standing variants are sufficient to guarantee future survival, given that they are mostly adaptive to the present range of climatic variability. While it may be difficult to perform manipulative experiments in the field to compare the effects of new mutations (NM) and standing variation (SV), one could investigate NM and SV during climatic change in the past.</p><p id="p-4">Adaptation could happen through genetic variants that differ in their origins (NM or SV) or effect sizes (Mendelian genes with major effects or polygenic variants with minor effects). However, how these factors interact and respond to environmental pressures remains relatively uninvestigated. For example, does SV and NM differ in their relative number or effect sizes towards environmental adaptation, and how does this relationship change with different types of environmental factor? For adaptive new mutations that were fixed when facing environment change, Fisher first predicted primarily small allelic effects<sup><a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">8</a></sup> while Kimura emphasized intermediate effects<sup><a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">9</a></sup>. Orr, later considering the entire adaptive walk, concluded the evolution towards a novel adaptive peak should first happen through fixation of large-effect mutations and later by small-effect polymorphisms<sup><a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">10</a></sup>. While this was supported by some studies, the majority of these are microbial experimental evolution in well-controlled environments<sup><a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">11</a>,<a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">12</a></sup>, and few have specifically compared the effects of NM and SV. To test whether this idea holds in nature, empirical investigations on natural populations are needed.</p><p id="p-5">Taiwan is well-suited for such studies: Unlike oceanic islands such as Hawaii, Taiwan is a continental island where most species originated from the East Asian continent with recurrent gene flow<sup><a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">13</a></sup>. The land bridge between Taiwan and China during the glacial maximum allowed exchange of SV, and the isolation during interglacial periods enabled the development of NM. Here we investigate the genomic basis of environmental adaptation of a wild banana species, <em>Musa itinerans</em>, whose habitats in Taiwan are considered peripheral from ancestral area reconstructions<sup><a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">14</a></sup>, providing an opportunity to distinguish SV from NM, as well as their response to ancestral versus novel adaptive landscapes. We investigated how past events (SV versus NM) influence present adaptation and whether local adaptation may persist under future anthropogenic climate change.</p></div><div class="section" id="sec-2"><h2 class="">Results</h2><div id="sec-3" class="subsection"><h3>Environmental adaptation in <em>Musa itinerans</em></h3><p id="p-6">We first sampled <em>Musa itinerans</em> at 24 populations across Taiwan (<a id="xref-fig-1-1" class="xref-fig" href="#F1">Fig. 1a</a>; <a id="xref-fig-5-1" class="xref-fig" href="#F5">Supplementary Fig. 1a</a>; <a id="xref-table-wrap-1-1" class="xref-table" href="#T1">Supplementary Table 1</a>) and investigated the population structure using 14 microsatellites (<a id="xref-table-wrap-2-1" class="xref-table" href="#T2">Supplementary Table 2</a>). Environmental Niche Modeling (with 483 occurrence points from field survey and Google Street View) reported species distribution (<a id="xref-fig-1-2" class="xref-fig" href="#F1">Fig. 1b</a>) in line with the previous statement that <em>Musa itinerans</em> inhabits sunny valleys, watersheds, and hillsides with gentle slopes<sup><a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">15</a></sup>. Populations differentiated mostly between east and west (<a id="xref-fig-5-2" class="xref-fig" href="#F5">Supplementary Fig. 1b</a>). The most unsuitable environments lay within the Central Mountain Range and the southwestern plains (<a id="xref-fig-1-3" class="xref-fig" href="#F1">Fig. 1b</a>), respectively corresponding to low annual mean temperature (BIO1) and low precipitation of driest quarter (BIO17), the two most important bioclimatic variables determining species distribution (MaxEnt permutation importance<sup><a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">16</a></sup>: 36.7 for BIO1 and 27.3 for BIO17).</p><div id="F1" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/01/14/2020.01.12.903427/F1.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Sample distribution and niche modeling of Musa itinerans. a, Sampling sites are distributed along the latitudinal and altitudinal gradient. Lowland populations are represented as white circles; transect populations are represented as white triangles and squares. b, Suitability is derived from MaxEnt niche modeling. c, Least-cost-corridor landscape is constructed from pairwise least-cost paths among 20 populations from the resistance surface (the reciprocal of niche suitability)." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1212661093" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;&lt;span class=&quot;caption-title&quot;&gt;Sample distribution and niche modeling of &lt;em&gt;Musa itinerans&lt;/em&gt;.&lt;/span&gt; &lt;strong&gt;a&lt;/strong&gt;, Sampling sites are distributed along the latitudinal and altitudinal gradient. Lowland populations are represented as white circles; transect populations are represented as white triangles and squares. &lt;strong&gt;b&lt;/strong&gt;, Suitability is derived from MaxEnt niche modeling. &lt;strong&gt;c&lt;/strong&gt;, Least-cost-corridor landscape is constructed from pairwise least-cost paths among 20 populations from the resistance surface (the reciprocal of niche suitability).&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Fig. 1." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2020/01/14/2020.01.12.903427/F1.medium.gif" width="440" height="134"/><noscript><img class="highwire-fragment fragment-image" alt="Fig. 1." src="https://www.biorxiv.org/content/biorxiv/early/2020/01/14/2020.01.12.903427/F1.medium.gif" width="440" height="134"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/01/14/2020.01.12.903427/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Fig. 1." data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/01/14/2020.01.12.903427/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml"><span class="fig-label">Fig. 1.</span> <span class="caption-title">Sample distribution and niche modeling of <em>Musa itinerans</em>.</span><p id="p-7" class="first-child"><strong>a</strong>, Sampling sites are distributed along the latitudinal and altitudinal gradient. Lowland populations are represented as white circles; transect populations are represented as white triangles and squares. <strong>b</strong>, Suitability is derived from MaxEnt niche modeling. <strong>c</strong>, Least-cost-corridor landscape is constructed from pairwise least-cost paths among 20 populations from the resistance surface (the reciprocal of niche suitability).</p><div class="sb-div caption-clear"></div></div></div><p id="p-8">To test for local adaptation, we examined the pattern of “isolation by adaptation<sup><a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">17</a></sup>”, a process where differential local adaptation restricted effective gene flow and promoted genetic differentiation among populations, by dissecting geographic and environmental effects on genetic differentiation. The strait-line “fly-over” geographical distance, calculated as the straight-line distance between locations, does not explain patterns of genetic differentiation (Mantel’s <em>r</em> = 0.146 and <em>P</em> = 0.062). However, if we considered that Central Mountain Range lacks corridors for <em>M. itinerans</em> to disperse (<a id="xref-fig-1-4" class="xref-fig" href="#F1">Fig. 1b</a>), this fly-over geographical distance could be too unrealistic. We therefore used resistance distance, calculated from the route with least resistance among populations on the niche suitability map (<a id="xref-fig-1-5" class="xref-fig" href="#F1">Fig. 1b</a>), to represent the “realized” geographical distance (<a id="xref-fig-1-6" class="xref-fig" href="#F1">Fig. 1c</a>) and found that genetic differentiation was significantly associated with resistance (Mantel’s <em>r</em> = 0.226 and <em>P</em> = 0.006). The environmental Mahalanobis distance of bioclimatic variables also showed strong association with genetic differentiation (Mantel’s <em>r</em> = 0.298 and <em>P</em> = 0.005). Given that the environmental distance could be strongly dependent on geography, we performed Partial Mantel test to control the geographical effect. After controlling for realized geographic distance (resistance distance), genetic differentiation still correlates with the Mahalanobis environmental distance (Mantel’s <em>r</em> = 0.250 and <em>P</em> = 0.012), suggesting differential local adaptation is associated with genetic variation.</p></div><div id="sec-4" class="subsection"><h3>Standing variation versus new mutations</h3><p id="p-9">To identify genomic regions associated with environmental adaptation, we performed pooled sequencing for each population. SNPs were separated into standing variation (SV: both alleles exist in Taiwan and China) or new mutations (NM, polymorphic only in Taiwan). SV outnumbered NM in both adaptive (identified with Bayenv<sup><a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">18</a></sup>) and non-adaptive SNPs, and after controlling for the overall number of SNPs in SV and NM, SV were further enriched among adaptive polymorphisms (<a id="xref-table-wrap-3-1" class="xref-table" href="#T3">Supplementary Table 3</a>). However, since adaptive SNPs also have higher minor allele frequency (MAF) (<a id="xref-fig-6-1" class="xref-fig" href="#F6">Supplementary Fig. 2a</a>), this pattern could be confounded: SV are more likely to have higher MAF than NM, and SNPs with higher MAF may be more likely detected as adaptive due to higher statistical power. We therefore performed the same test with a subset where the adaptive and non-adaptive SNPs have similar allele frequencies (ranging from the first quantile of adaptive MAF to the third quantile of non-adaptive MAF separately for each bioclimatic variable; <a id="xref-fig-6-2" class="xref-fig" href="#F6">Supplementary Fig. 2a</a>). In this case as well, SV are still disproportionately abundant (<a id="xref-table-wrap-4-1" class="xref-table" href="#T4">Supplementary Table 4</a>), suggesting SV are more likely than NM to become environment-associated SNPs. Another potential confounding factor is the geographic extent of variants: if most NM resulted from mutations restricted to a few local populations, the limited distribution prevents environment association for NM. We therefore compared the number of Taiwanese populations containing the minor alleles for SV and NM, respectively. Contrary to the direction predicted by the aforementioned confounding factor, the geographic extent of minor alleles for SV is slightly smaller than NM (15.8 populations for SV and 16.3 for NM, <em>P</em> &lt; 0.001).</p><p id="p-10">In addition to SNP number, do NM and SV differ in their directions of effect? Under the null hypothesis that (1) the effects of NM are equally likely to facilitate adaptation to the ancestral or novel environments and (2) natural selection is equally likely to fix NM facilitating adaptation towards either direction, we expected no enrichment of new alleles in either environment. When we separated the Taiwanese populations into those within the Chinese ancestral environmental range and those with novel environments, frequencies of putatively adaptive new alleles in NM SNPs were higher in the latter set of populations, with precipitation of driest quarter (BIO17) and precipitation of coldest quarter (BIO19) showing the strongest effect (<a id="xref-fig-2-1" class="xref-fig" href="#F2">Fig. 2a</a>; <a id="xref-fig-6-3" class="xref-fig" href="#F6">Supplementary Fig. 2b</a>). Given that the directions of mutation effects should be random, these results suggest that new variants facilitating adaptation to novel environments are more likely to be retained by selection.</p><div id="F2" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/01/14/2020.01.12.903427/F2.large.jpg?width=800&amp;height=600&amp;carousel=1" title="The environment-dependent enrichment of new alleles and the distinct effect sizes of standing variation (SV) and new mutations (NM) in contrasting climatic factors. BIO1 reflects annual mean temperature, and BIO17 indicates precipitation of driest quarter. a, Mean frequency of new alleles among NM SNPs compared between populations that have local environments within or outside of the ancestral climatic range. New alleles are enriched in novel environments (**P &lt; 0.01, ***P &lt; 0.001, t-test). Values on the horizontal axis denote the number of Taiwanese populations within the ancestral or novel environmental range. b, Gradient forest cumulative r2 importance is shown along environmental gradients. c, Effect sizes as estimated from Bayes factor, gradient forest r2 importance, and gradient forest slope all show that SV exhibit higher effect sizes in BIO1 but the reverse in BIO17 (**P &lt; 0.01, ***P &lt; 0.001, Wilcoxon rank-sum test for Bayes factor and t-test for r2 importance and slope)." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1212661093" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;&lt;span class=&quot;caption-title&quot;&gt;The environment-dependent enrichment of new alleles and the distinct effect sizes of standing variation (SV) and new mutations (NM) in contrasting climatic factors.&lt;/span&gt; BIO1 reflects annual mean temperature, and BIO17 indicates precipitation of driest quarter. &lt;strong&gt;a&lt;/strong&gt;, Mean frequency of new alleles among NM SNPs compared between populations that have local environments within or outside of the ancestral climatic range. New alleles are enriched in novel environments (**&lt;em&gt;P&lt;/em&gt; &lt; 0.01, ***&lt;em&gt;P&lt;/em&gt; &lt; 0.001, &lt;em&gt;t&lt;/em&gt;-test). Values on the horizontal axis denote the number of Taiwanese populations within the ancestral or novel environmental range. &lt;strong&gt;b&lt;/strong&gt;, Gradient forest cumulative &lt;em&gt;r&lt;/em&gt;&lt;sup&gt;&lt;em&gt;2&lt;/em&gt;&lt;/sup&gt; importance is shown along environmental gradients. &lt;strong&gt;c&lt;/strong&gt;, Effect sizes as estimated from Bayes factor, gradient forest &lt;em&gt;r&lt;/em&gt;&lt;sup&gt;&lt;em&gt;2&lt;/em&gt;&lt;/sup&gt; importance, and gradient forest slope all show that SV exhibit higher effect sizes in BIO1 but the reverse in BIO17 (**&lt;em&gt;P&lt;/em&gt; &lt; 0.01, ***&lt;em&gt;P&lt;/em&gt; &lt; 0.001, Wilcoxon rank-sum test for Bayes factor and &lt;em&gt;t&lt;/em&gt;-test for &lt;em&gt;r&lt;/em&gt;&lt;sup&gt;&lt;em&gt;2&lt;/em&gt;&lt;/sup&gt; importance and slope).&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Fig. 2." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2020/01/14/2020.01.12.903427/F2.medium.gif" width="440" height="296"/><noscript><img class="highwire-fragment fragment-image" alt="Fig. 2." src="https://www.biorxiv.org/content/biorxiv/early/2020/01/14/2020.01.12.903427/F2.medium.gif" width="440" height="296"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/01/14/2020.01.12.903427/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Fig. 2." data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/01/14/2020.01.12.903427/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Fig. 2.</span> <span class="caption-title">The environment-dependent enrichment of new alleles and the distinct effect sizes of standing variation (SV) and new mutations (NM) in contrasting climatic factors.</span><p id="p-11" class="first-child">BIO1 reflects annual mean temperature, and BIO17 indicates precipitation of driest quarter. <strong>a</strong>, Mean frequency of new alleles among NM SNPs compared between populations that have local environments within or outside of the ancestral climatic range. New alleles are enriched in novel environments (**<em>P</em> &lt; 0.01, ***<em>P</em> &lt; 0.001, <em>t</em>-test). Values on the horizontal axis denote the number of Taiwanese populations within the ancestral or novel environmental range. <strong>b</strong>, Gradient forest cumulative <em>r</em><sup><em>2</em></sup> importance is shown along environmental gradients. <strong>c</strong>, Effect sizes as estimated from Bayes factor, gradient forest <em>r</em><sup><em>2</em></sup> importance, and gradient forest slope all show that SV exhibit higher effect sizes in BIO1 but the reverse in BIO17 (**<em>P</em> &lt; 0.01, ***<em>P</em> &lt; 0.001, Wilcoxon rank-sum test for Bayes factor and <em>t</em>-test for <em>r</em><sup><em>2</em></sup> importance and slope).</p><div class="sb-div caption-clear"></div></div></div><p id="p-12">The results above suggest SV might be more important than NM in terms of enriched number of variants with environment association, and adaptive NM, while lower in number, are more associated with novel environments. On the other hand, the number of candidate SNPs does not necessarily reflect the overall importance of SV over NM, since the effect size also needs to be considered. To investigate the effect size of SV and NM in environmental adaptation, we compared their Bayes factors from Bayenv and focused on two bioclimatic variables that were important in determining species distribution, annual mean temperature (BIO1) and precipitation of driest quarter (BIO17). The two variables exhibit opposite patterns (<a id="xref-fig-2-2" class="xref-fig" href="#F2">Fig. 2c</a>): While in BIO17 and other related precipitation variables (<a id="xref-fig-7-1" class="xref-fig" href="#F7">Supplementary Fig. 3a</a>) NM consistently had higher Bayes factor and therefore stronger effect size than SV, in BIO1 and other related temperature variables (<a id="xref-fig-7-2" class="xref-fig" href="#F7">Supplementary Fig. 3a</a>) we observed the reverse. The same trend was observed when we estimated the effect size with gradient forest<sup><a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">19</a>,<a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">20</a></sup> (<a id="xref-fig-2-3" class="xref-fig" href="#F2">Fig. 2b, c</a>; <a id="xref-fig-7-3" class="xref-fig" href="#F7">Supplementary Fig. 3b, c</a>): BIO17 was the most important factor for differential local adaptation, and NM had stronger effects than SV. On the other hand, BIO1 was the least important factor where SV had stronger effects. Finally, the “importance” estimated by gradient forest is analogous to <em>r</em><sup><em>2</em></sup>, representing the amount of allele frequency variation explained by environmental gradients. Assuming a simple linear relationship between allele frequency and environment, the value of <em>r</em><sup><em>2</em></sup> only represents how well each data point (a population) fits along the regression line. We were, however, also interested in the regression slope: the amount of allele frequency changes along environmental gradients (<a id="xref-fig-7-4" class="xref-fig" href="#F7">Supplementary Fig. 3d</a>). Again, BIO17 had the largest overall slope among all bioclimatic variables (<a id="xref-fig-7-5" class="xref-fig" href="#F7">Supplementary Fig. 3e</a>), with NM being significantly higher than SV (<a id="xref-fig-2-4" class="xref-fig" href="#F2">Fig. 2c</a>). BIO1 had the lowest overall slope, again with the reversed pattern. Given that the MAF of adaptive NM and SV SNPs are similar, there is no need to control for allele frequency in these tests (<a id="xref-fig-8-1" class="xref-fig" href="#F8">Supplementary Fig. 4a</a>). Therefore, NM with larger effect size per SNP (as estimated by Bayenv Bayes factor, gradient forest <em>r</em><sup><em>2</em></sup> weighted importance, and gradient forest slope) were associated with the adaptation to novel environments outside of the ancestral niche space, consistent with previous population genetics modeling results<sup><a id="xref-ref-8-2" class="xref-bibr" href="#ref-8">8</a>-<a id="xref-ref-10-2" class="xref-bibr" href="#ref-10">10</a></sup>.</p><p id="p-13">The observed patterns could be integrated with the unique climate of Taiwan. In comparison to the rest of the species range, northern Taiwan experiences northeastern monsoons during winter and has higher precipitation during the typical winter dry season (<a id="xref-fig-3-1" class="xref-fig" href="#F3">Fig. 3b</a>). This pattern has been maintained since at least the last glacial maximum (<a id="xref-fig-3-2" class="xref-fig" href="#F3">Fig. 3d</a>). The novel environments might impose novel adaptive optima to the immigrant population from China. The response to selection imposed by these environmental gradients is strong (with highest Bayes factor, <em>r</em><sup><em>2</em></sup>, and slope among all bioclimatic variables; <a id="xref-fig-7-6" class="xref-fig" href="#F7">Supplementary Fig. 3a, c, e</a>) especially for NM, where new alleles are strongly associated with novel environments (<a id="xref-fig-6-4" class="xref-fig" href="#F6">Supplementary Fig. 2b</a>). More importantly, for this major driver of adaptation (BIO17), the greatest increment of gradient forest importance lies between 200 mm and 300 mm (<a id="xref-fig-2-5" class="xref-fig" href="#F2">Fig. 2b</a>), a range also distinguishing the novel Taiwanese versus ancestral Chinese environments (<a id="xref-fig-3-3" class="xref-fig" href="#F3">Fig. 3b</a>). This suggests that the majority of differential local adaptation is associated with such novel-versus-ancestral environmental differences. Annual mean temperature (BIO1) is the other extreme: the environmental gradient within Taiwan is well within the ancestral Chinese environmental range (<a id="xref-fig-3-4" class="xref-fig" href="#F3">Fig. 3a</a>), which can also be traced back to the last glacial maximum (<a id="xref-fig-3-5" class="xref-fig" href="#F3">Fig. 3c</a>). It is likely that SV already contained genetic variants adaptive to such environmental gradients and are therefore more important than NM (<a id="xref-fig-2-6" class="xref-fig" href="#F2">Fig. 2c</a>). In summary, we observed adaptation happening through the assortment of SV for a new territory with similar adaptive landscape and optimum (BIO1; <a id="xref-fig-2-7" class="xref-fig" href="#F2">Figs. 2c</a>, <a id="xref-fig-3-6" class="xref-fig" href="#F3">3a, c</a>). For adaptation to novel environments and a new adaptive landscape (BIO17; <a id="xref-fig-2-8" class="xref-fig" href="#F2">Figs. 2c</a>, <a id="xref-fig-3-7" class="xref-fig" href="#F3">3b, d</a>), NM with larger effect sizes were more likely favored by natural selection. Our results are therefore consistent with Orr’s model<sup><a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">3</a></sup>, providing one of the few examples in nature.</p><div id="F3" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/01/14/2020.01.12.903427/F3.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Annual mean temperature (BIO1) and precipitation of driest quarter (BIO17) for the present and last glacial maximum. a, b, Present environments. c, d, Last-glacial-maximum environments, showing the environments on the extent of land. Maps on all panels have the same range in latitude and longitude." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1212661093" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;&lt;span class=&quot;caption-title&quot;&gt;Annual mean temperature (BIO1) and precipitation of driest quarter (BIO17) for the present and last glacial maximum.&lt;/span&gt; &lt;strong&gt;a, b&lt;/strong&gt;, Present environments. &lt;strong&gt;c, d&lt;/strong&gt;, Last-glacial-maximum environments, showing the environments on the extent of land. Maps on all panels have the same range in latitude and longitude.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Fig. 3." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2020/01/14/2020.01.12.903427/F3.medium.gif" width="440" height="216"/><noscript><img class="highwire-fragment fragment-image" alt="Fig. 3." src="https://www.biorxiv.org/content/biorxiv/early/2020/01/14/2020.01.12.903427/F3.medium.gif" width="440" height="216"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/01/14/2020.01.12.903427/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Fig. 3." data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/01/14/2020.01.12.903427/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Fig. 3.</span> <span class="caption-title">Annual mean temperature (BIO1) and precipitation of driest quarter (BIO17) for the present and last glacial maximum.</span><p id="p-14" class="first-child"><strong>a, b</strong>, Present environments. <strong>c, d</strong>, Last-glacial-maximum environments, showing the environments on the extent of land. Maps on all panels have the same range in latitude and longitude.</p><div class="sb-div caption-clear"></div></div></div><p id="p-15">One key point of this study is the correct designation of SV or NM. It is possible that some SV SNPs were mis-assigned as NM if we missed an allele in China, most likely for SNPs with low MAF. We addressed this issue with the following: (1) The Taiwanese populations were nested within China in the phylogenetic tree (<a id="xref-fig-8-2" class="xref-fig" href="#F8">Supplementary Fig. 4b</a>) and contained much less genetic variation (<a id="xref-fig-8-3" class="xref-fig" href="#F8">Supplementary Fig. 4c</a>). Due to the stronger genetic drift in Taiwan than China, it is less likely that an originally SV SNP would retain both alleles in Taiwan but lose one in China. (2) In the extreme case, assuming 50% of NM SNPs were mis-assigned from SV, we performed 100 new analyses, each randomly assigning 50% of NM SNPs back to SV. These new analyses yielded similar results, with NM having higher effect sizes than SV in precipitation-related variables (<a id="xref-fig-9-1" class="xref-fig" href="#F9">Supplementary Fig. 5a</a>). (3) Since MAF are correlated between Taiwan and China (Spearman’s rank correlation ρ = 0.35, <em>P</em> &lt; 0.001), we performed analyses with top 50% MAF SNPs in Taiwan, thereby reducing the chance of missing minor alleles in China. The results are qualitatively the same (<a id="xref-fig-9-2" class="xref-fig" href="#F9">Supplementary Fig. 5b-d</a>).</p></div><div id="sec-5" class="subsection"><h3>Fate of adaptive variants under future climate change</h3><p id="p-16">In addition to understanding how past events (SV versus NM) affected present adaptation, we are also concerned with how these factors affect the future of this species under anthropogenic climate change. We predicted 16 future outcomes and used Bayenv to investigate the future fate of currently adaptive SNPs. Currently adaptive SNPs retaining high association with future environments are classified as “retention”, while those no longer associated with future environments are “disruption”. Different from the present pattern of SV being enriched in adaptive SNPs, we saw no clear tendency for any set of adaptive SNPs enriched towards retention or disruption (<a id="xref-table-wrap-5-1" class="xref-table" href="#T5">Supplementary Table 5</a>), suggesting both SV and NM will be affected by climate change.</p><p id="p-17">Under the 16 future scenarios we identified two extreme conditions: CCSM4-2070-RCP8.5 as the niche-expansion extreme (<a id="xref-fig-4-1" class="xref-fig" href="#F4">Fig. 4a-d</a>) and MIROC-2070-RCP4.5 as the niche-contraction extreme (<a id="xref-fig-4-2" class="xref-fig" href="#F4">Fig. 4e-h</a>). Intending to investigate whether SV and NM have different fates under the two contrasting future predictions of this species, we used the genetic offset value from the gradient forest results to estimate genetic mismatch for SV and NM separately, which is associated with the magnitude of allele frequency turnover perturbed by future climatic conditions<sup><a id="xref-ref-20-2" class="xref-bibr" href="#ref-20">20</a></sup> (<a id="xref-fig-4-3" class="xref-fig" href="#F4">Fig. 4b, f</a>). The “genetic offset difference” was calculated by subtracting genetic offset caused by SV from that of NM (<a id="xref-fig-4-4" class="xref-fig" href="#F4">Fig. 4c, g</a>). We found no general consensus of whether or where in Taiwan NM or SV SNPs are more affected by climate change (<a id="xref-fig-4-5" class="xref-fig" href="#F4">Fig. 4c, g</a>), suggesting NM and SV might both be affected by future anthropogenic climate change despite their distinct genetic architecture in adaptation to current environments.</p><div id="F4" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/01/14/2020.01.12.903427/F4.large.jpg?width=800&amp;height=600&amp;carousel=1" title="The fate of Musa itinerans under two extreme future conditions. a-d, Niche-expansion extreme (CCSM4-2070-RCP8.5) reports the species-average niche suitability from MaxEnt modeling (a), the genetic mismatch of locally adaptive populations to future environments based on the genetic offset from gradient forest (b), genetic offset difference calculated as the genetic offset experienced by new mutations minus that from standing variation (c), and extinction risk estimated as genetic offset divided by niche suitability (shown as 32 bins with equal grid counts; d). e-h, Niche-contraction extreme (MIROC-2070-RCP4.5) reports the suitability (e), genetic offset (f), genetic offset difference (g), and extinction risk (h). Grids with present niche suitability &lt; 0.2 are excluded and colored in gray in b-d and f-h." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1212661093" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;&lt;span class=&quot;caption-title&quot;&gt;The fate of &lt;em&gt;Musa itinerans&lt;/em&gt; under two extreme future conditions.&lt;/span&gt; &lt;strong&gt;a&lt;/strong&gt;-&lt;strong&gt;d&lt;/strong&gt;, Niche-expansion extreme (CCSM4-2070-RCP8.5) reports the species-average niche suitability from MaxEnt modeling (&lt;strong&gt;a&lt;/strong&gt;), the genetic mismatch of locally adaptive populations to future environments based on the genetic offset from gradient forest (&lt;strong&gt;b&lt;/strong&gt;), genetic offset difference calculated as the genetic offset experienced by new mutations minus that from standing variation (&lt;strong&gt;c&lt;/strong&gt;), and extinction risk estimated as genetic offset divided by niche suitability (shown as 32 bins with equal grid counts; &lt;strong&gt;d&lt;/strong&gt;). &lt;strong&gt;e&lt;/strong&gt;-&lt;strong&gt;h&lt;/strong&gt;, Niche-contraction extreme (MIROC-2070-RCP4.5) reports the suitability (&lt;strong&gt;e&lt;/strong&gt;), genetic offset (&lt;strong&gt;f&lt;/strong&gt;), genetic offset difference (&lt;strong&gt;g&lt;/strong&gt;), and extinction risk (&lt;strong&gt;h&lt;/strong&gt;). Grids with present niche suitability &lt; 0.2 are excluded and colored in gray in &lt;strong&gt;b&lt;/strong&gt;-&lt;strong&gt;d&lt;/strong&gt; and &lt;strong&gt;f&lt;/strong&gt;-&lt;strong&gt;h&lt;/strong&gt;.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Fig. 4." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2020/01/14/2020.01.12.903427/F4.medium.gif" width="440" height="217"/><noscript><img class="highwire-fragment fragment-image" alt="Fig. 4." src="https://www.biorxiv.org/content/biorxiv/early/2020/01/14/2020.01.12.903427/F4.medium.gif" width="440" height="217"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/01/14/2020.01.12.903427/F4.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Fig. 4." data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/01/14/2020.01.12.903427/F4.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Fig. 4.</span> <span class="caption-title">The fate of <em>Musa itinerans</em> under two extreme future conditions.</span><p id="p-18" class="first-child"><strong>a</strong>-<strong>d</strong>, Niche-expansion extreme (CCSM4-2070-RCP8.5) reports the species-average niche suitability from MaxEnt modeling (<strong>a</strong>), the genetic mismatch of locally adaptive populations to future environments based on the genetic offset from gradient forest (<strong>b</strong>), genetic offset difference calculated as the genetic offset experienced by new mutations minus that from standing variation (<strong>c</strong>), and extinction risk estimated as genetic offset divided by niche suitability (shown as 32 bins with equal grid counts; <strong>d</strong>). <strong>e</strong>-<strong>h</strong>, Niche-contraction extreme (MIROC-2070-RCP4.5) reports the suitability (<strong>e</strong>), genetic offset (<strong>f</strong>), genetic offset difference (<strong>g</strong>), and extinction risk (<strong>h</strong>). Grids with present niche suitability &lt; 0.2 are excluded and colored in gray in <strong>b</strong>-<strong>d</strong> and <strong>f</strong>-<strong>h</strong>.</p><div class="sb-div caption-clear"></div></div></div><p id="p-19">Traditional species distribution models build the species-wide niche model from all occurrence records of a species, thereby assuming all populations in a species reacting equally to the environment and overlooking local adaptation due to within-species polymorphism<sup><a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">6</a></sup>. Here we propose a concept, “extinction risk”, integrating species-level and population-level responses to future climate change simply by dividing genetic offset (estimated from gradient forest) by suitability (estimated from MaxEnt). While the two future conditions exhibit slight differences in suitability and genetic offset, the extinction risks are surprisingly similar, with very high risk in western Taiwan (<a id="xref-fig-4-6" class="xref-fig" href="#F4">Fig. 4d, h</a>). In Western Taiwan, the expansion-extreme model suggested high suitability for the species as a whole but also high local population mismatch, thereby having similar trends of extinction risk as the contraction-extreme model. Taken together, our results emphasize the importance of considering both whole-species suitability as well as adaptation of local populations when considering the effect of anthropogenic climate change on the fate of sessile species.</p></div></div><div class="section" id="sec-6"><h2 class="">Discussion</h2><p id="p-20">Assessing the within-species variation in climate association is the crucial first step to understand species susceptibility to fluctuating environment, and the relative importance of standing variation (SV) and new mutations (NM) in adaptation has long been debated<sup><a id="xref-ref-7-2" class="xref-bibr" href="#ref-7">7</a>,<a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">21</a>-<a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">25</a></sup>. In this study, we investigate how past genetic variation (SV and NM) contribute to the adaptation of present environments as well as how these factors together affect the future fate of a species under anthropogenic climate change.</p><p id="p-21">For a population facing environmental change and therefore a novel adaptive landscape, previous population genetics models have documented the effect size distribution of new mutations fixed by natural selection. Considering the entire adaptive walk of a population facing novel environments, Orr’s model predicted the distribution of effect sizes where early substitutions have larger effect than later ones<sup><a id="xref-ref-10-3" class="xref-bibr" href="#ref-10">10</a></sup>. Consistent with Orr’s prediction, we show that NM have stronger effect size than SV in precipitation-related variables, where Taiwan exceeds the ancestral climatic range in China (and therefore the migrating population was far away from the optimum in the new adaptive landscape). This pattern is reversed for temperature-related variables, where Taiwan has similar environmental range as China. Here we provide another perspective to recent research showing that SV contributes to adaptation<sup><a id="xref-ref-7-3" class="xref-bibr" href="#ref-7">7</a>,<a id="xref-ref-25-2" class="xref-bibr" href="#ref-25">25</a></sup>. We show that SV indeed dominate over NM in number<sup><a id="xref-ref-7-4" class="xref-bibr" href="#ref-7">7</a></sup>. However, the effect size of SV and NM hinges on environmental conditions: natural selection may prefer new mutations contributing to the adaptation to the new rather than the old environment, and the effect sizes of NM tend to be higher than SV under such conditions. Our results imply that standing genetic variation in a population may not be sufficient for the adaptation to anthropogenic climate change.</p><p id="p-22">Ecological niche models are widely used to predict future species distribution, but such models often do not account for the differential adaptation of populations within species. Our observed difference between the major determinant of species-wide range distribution (BIO1) and the driver of differential adaptation of populations within species (BIO17) demonstrates the need to consider local adaptation. Incorporating species distribution modeling (predicting the future range of the whole species) and genetic offset (predicting the genetic mismatch of each local population to future environments), we identified the potential risk of western lowland populations despite the seemingly distinct species distribution modeling results in different future climate scenarios. In other words, despite in some future scenarios the range of environments suitable for the whole species might increase, giving the impression that the species benefits from anthropogenic climate change, such change might be detrimental to each local population uniquely adapted to a much narrower environmental range than the whole species. Finally, we separately calculated the genetic offset originated from the mismatch of SV or NM to future environments and showed that both will be affected by anthropogenic climate change, regardless of their distinct genetic architecture towards environmental adaptation. Since we distinguished SNPs into SV or NM based on past environmental change, all of these genetic variants will be standing variation when facing future climate change. Novel mutations facilitating the adaptation to new environments may happen in the future, although they will be strongly outpaced by anthropogenic climate change.</p></div><div class="section" id="sec-7"><h2 class="">Methods</h2><div id="sec-8" class="subsection"><h3>Sample collection and DNA extraction</h3><p id="p-23">Field work was conducted during 2017 (August - December) and 2018 (January - May). We sampled <em>Musa itinerans</em> at 24 sites across Taiwan (<a id="xref-fig-5-3" class="xref-fig" href="#F5">Supplementary Fig. 1a</a>; <a id="xref-table-wrap-1-2" class="xref-table" href="#T1">Supplementary Table 1</a>). Fresh leaves were harvested from nine to fifteen individuals at each site. Total genomic DNA was extracted using the standard CTAB extraction method<sup><a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">26</a></sup>. Since other commercial <em>Musa</em> species were also grown in Taiwan, we developed an indel marker for species delimitation. From previous studies<sup><a id="xref-ref-27-1" class="xref-bibr" href="#ref-27">27</a>-<a id="xref-ref-30-1" class="xref-bibr" href="#ref-30">30</a></sup>, we identified a 6-bp insertion specific for the Taiwanese <em>M. itinerans</em> in the <em>atpB-rbcL</em> region of chloroplast. We designed a primer pair (5’-GAAGGGGTAGGATTGATTCTCA-3’; 5’-CGACTTGGCATGGCACTATT-3’) and used amplicon size to confirm all collected samples are Taiwanese <em>M. itinerans</em>.</p></div><div id="sec-9" class="subsection"><h3>Simple sequence repeat genotyping and analysis</h3><p id="p-24">SSR primer sequences used in this study were originally developed for the genus <em>Musa</em><sup><a id="xref-ref-31-1" class="xref-bibr" href="#ref-31">31</a>,<a id="xref-ref-32-1" class="xref-bibr" href="#ref-32">32</a></sup>, which were then applied on <em>Musa itinerans</em>. Previously documented primer sequences were first searched against the <em>Musa acuminata</em> DH-Pahang genome version 2<sup><a id="xref-ref-33-1" class="xref-bibr" href="#ref-33">33</a></sup> on Banana Genome Hub (<a href="https://banana-genome-hub.southgreen.fr/">https://banana-genome-hub.southgreen.fr/</a>) to check specificity as having only one amplicon, resulting in 26 primer pairs. These primers were then experimented to check specificity on <em>M. itinerans</em>, resulting in 14 pairs (<a id="xref-table-wrap-2-2" class="xref-table" href="#T2">Supplementary Table 2</a>). We modified each pair of primers by capping the 5’ end of forward primers with M13 sequences (CACGACGTTGTAAAACGAC) and inflorescent molecules<sup><a id="xref-ref-34-1" class="xref-bibr" href="#ref-34">34</a></sup>. SSR amplicons were run through capillary electrophoresis and the length of each allele was recorded.</p><p id="p-25">Population structure of 20 populations (<a id="xref-table-wrap-1-3" class="xref-table" href="#T1">Supplementary Table 1</a>) was analyzed with 14 SSR markers (<a id="xref-table-wrap-2-3" class="xref-table" href="#T2">Supplementary Table 2</a>). Lowland populations (C35H, WFL, THNL, PTWT, P199H, MLLYT, HDPG, TTL, NAJY, HLCN, NXIR, and DFR), east transect populations (TPS300, TPS500, TPS700, and TPS900), and west transect populations (XT400, XT700, XT1200, and XT1500) were used in the analysis. We inferred the ancestry of 244 individuals with STRUCTURE 2.3.4<sup><a id="xref-ref-35-1" class="xref-bibr" href="#ref-35">35</a>,<a id="xref-ref-36-1" class="xref-bibr" href="#ref-36">36</a></sup>, parameterizing a run to have (1) run length of burnin and after-burnin period of 100,000, (2) admixture ancestry model, and (3) independent allele frequency model, further setting 20 runs for each <em>K</em> value.</p><p id="p-26">To investigate the association among genetic, geographical, and environmental distance, we generated these distance matrices. Genetic distance was calculated by GenAlEx 6.503<sup><a id="xref-ref-37-1" class="xref-bibr" href="#ref-37">37</a>,<a id="xref-ref-38-1" class="xref-bibr" href="#ref-38">38</a></sup> from 14 SSR markers; straight geographical distance (the fly-over distance) was generated by ArcGIS 10.5 (<a href="http://desktop.arcgis.com/en/">http://desktop.arcgis.com/en/</a>); environmental distance was measured as Mahalanobis distance to address the correlation among nine bioclimatic variables (below). In addition to the fly-over geographical distance which assumes organism dispersal ignores landscapes, we further calculated as resistance distance the cumulative cost along the least cost path (below). Matrix association was examined under Mantel and Partial Mantel tests. Statistical significance was examined with 1,000 permutations. We performed Mantel tests on (1) genetic distance <em>vs</em>. fly-over distance, (2) genetic distance <em>vs</em>. resistance distance, (3) genetic distance <em>vs</em>. Mahalanobis environmental distance, and Partial Mantel tests on (4) genetic distance <em>vs</em>. Mahalanobis environmental distance while controlling for resistance distance.</p></div><div id="sec-10" class="subsection"><h3>Species distribution modeling</h3><p id="p-27">Current and future species distribution models were built for <em>Musa itinerans</em> using presence-only data (483 occurrence points) obtained from field survey and Google Street view. Occurrence points were then reduced to 204 cells by the removal of co-occurring presence data within the same 1× 1 km grid. MaxEnt 3.4.1<sup><a id="xref-ref-16-2" class="xref-bibr" href="#ref-16">16</a></sup>, implemented with the maximum entropy modeling approach, reports an overall niche suitability and the importance of predictors by analyzing the presence-only data as well as background (psuedo-absence) data distribution<sup><a id="xref-ref-39-1" class="xref-bibr" href="#ref-39">39</a>-<a id="xref-ref-41-1" class="xref-bibr" href="#ref-41">41</a></sup>. We downloaded from WorldClim database version 1.4 (<a href="http://worldclim.org/">http://worldclim.org/</a>) spatial layers of 19 present-day bioclimatic variables based on high-resolution monthly temperature and rainfall data<sup><a id="xref-ref-42-1" class="xref-bibr" href="#ref-42">42</a></sup>. Layers were selected at spatial resolution of 30 arc-second and with a mask that ranges 119.25-122.47°E and 21.76-25.49°N covering Taiwan. Variables showing high dependence (Pearson’s correlation coefficient &gt; 0.9 calculated from ENMTools<sup><a id="xref-ref-43-1" class="xref-bibr" href="#ref-43">43</a></sup>) from each other were removed, resulting in nine final variables: BIO1—mean annual temperature, BIO2—mean diurnal range, BIO3—isothermality, BIO7—temperature annual range, BIO12—annual precipitation, BIO15 —precipitation seasonality, BIO16—precipitation of wettest quarter, BIO17—precipitation of driest quarter, and BIO19—precipitation of coldest quarter (<a id="xref-table-wrap-1-4" class="xref-table" href="#T1">Supplementary Table 1</a>).</p><p id="p-28">Present species distribution model was constructed using the default optimization settings in MaxEnt, except the regularization set to three. We tested the predictive model by ten-fold cross-validation which was carried out by randomly partitioning the data into ten equally sized subsets and then replicating models while omitting one subset in turn. In each turn, the predictive model was built using nine subsets as training data and evaluated using the other subset as test data. The output of the predictive model is the probability of presence, or called suitability, and we averaged the ten runs to have an averaged suitability.</p><p id="p-29">To predict the species distribution under different scenarios of future climate change, we projected the present-day model onto eight future climatic conditions combining two periods (2050 and 2070) and four Representative Concentration Pathways (RCP 2.6, RCP 4.5, RCP 6.0, and RCP 8.5). Future climatic layers were obtained from the WorldClim database at spatial resolution of 30 arc-second and were developed based on two general circulation models: the Community Climate System Model<sup><a id="xref-ref-44-1" class="xref-bibr" href="#ref-44">44</a></sup>, CCSM, and the Model for Interdisciplinary Research on Climate<sup><a id="xref-ref-45-1" class="xref-bibr" href="#ref-45">45</a></sup>, MIROC. Species distribution models for the future were carried out using the same settings described above.</p><p id="p-30">To estimate the least cost path between populations, we first generated the resistance surface by taking the reciprocal of suitability. Resistance and suitability is simply a monotonic transformation in which locations with higher suitability exhibit lower resistance. Pairwise least cost path was then measured among 20 populations from the resistance surface, performed by SDM Toolbox v2.3<sup><a id="xref-ref-46-1" class="xref-bibr" href="#ref-46">46</a></sup>. While least cost path is the single line with least overall cost, we also constructed the least cost corridor between populations, allowing 1%, 2%, or 5% higher cost than the least cost value. In essence, the least cost corridors represent the realized dispersal routes of organisms along suitable habitats.</p></div><div id="sec-11" class="subsection"><h3>Sequencing library construction and SNP identification</h3><p id="p-31">We conducted whole genome pooled-sequencing<sup><a id="xref-ref-47-1" class="xref-bibr" href="#ref-47">47</a></sup> for each population (<a id="xref-table-wrap-1-5" class="xref-table" href="#T1">Supplementary Table 1</a>), resulting in 24 pooled-sequencing libraries. Equal amount of DNA from ten individuals at each population were pooled, except for the PTWT population where only nine individuals were available. A library with 300-400 bp insert size for each pool was prepared using NEBNext Ultra II DNA Library Prep Kit (New England Biolabs). Libraries were then sequenced with 150 bp paired-end on the HiSeq X Ten platform.</p><p id="p-32">Illumina reads were then trimmed with SolexaQA<sup><a id="xref-ref-48-1" class="xref-bibr" href="#ref-48">48</a></sup>, followed by the removal of adaptor sequences with cutadapt<sup><a id="xref-ref-49-1" class="xref-bibr" href="#ref-49">49</a></sup>, subsequently mapped to the <em>Musa itinerans</em> reference genome assembly ASM164941v1<sup><a id="xref-ref-50-1" class="xref-bibr" href="#ref-50">50</a></sup> with BWA 0.7.15<sup><a id="xref-ref-51-1" class="xref-bibr" href="#ref-51">51</a></sup>. Picard Tools (<a href="http://broadinstitute.github.io/picard">http://broadinstitute.github.io/picard</a>) was used to mark duplicated read pairs, and the genotypes were called following GATK 3.7 best practice<sup><a id="xref-ref-52-1" class="xref-bibr" href="#ref-52">52</a></sup>.</p><p id="p-33">For the 24 pooled samples, we filtered out sites with (1) more than two alleles, (2) indels, (3) quality (QUAL) &lt; 30, (4) quality by depth (QD) &lt; 2, (5) call rate &lt; 0.74, and (6) depth (DP) &gt; genome-wide average depth plus three standard deviations, resulting in 4,200,177 SNPs. SNPs with (1) minor allele frequency (MAF) &lt; 0.05, (2) missing data in any of the pooled-seq sample, and (3) DP per sample &lt; 20 were further filtered out, resulting in 1,256,894 SNPs.</p><p id="p-34">To investigate the relationship between Taiwanese and Chinese <em>M. itinerans</em>, we downloaded public data from 24 Chinese accessions (SRR6382516 - SRR6382539)<sup><a id="xref-ref-53-1" class="xref-bibr" href="#ref-53">53</a></sup>. SNPs were called using all 24 Chinese and the 24 Taiwanese samples together following the pipeline described above. We did not perform any site filtering for this joint data set since the main objective is to investigate whether specific SNPs in Taiwan also existed in China as SV. This dataset has 18,442,853 SNPs. SRR6382532 was excluded due to high missing rate. Only when evaluating the averaged expected heterozygosity between Taiwanese and Chinese populations did we filter out sites with (1) indels and (2) QUAL &lt; 30, resulting in 15,591,923 SNPs.</p><p id="p-35">To assess the phylogeny of our Taiwanese populations and Chinese accessions, we downloaded <em>Musa acuminata</em> sequence (SRR7013754) as an outgroup. SNPs were called using one <em>M. acuminata</em> species, 24 Chinese and 24 Taiwanese samples together following the pipeline described above. We filtered out sites with (1) more than two alleles, (2) indels, (3) QUAL &lt; 30, and (4) call rate &lt; 0.9, resulting in 12,693,687 SNPs. This dataset also excluded SRR6382532.</p></div><div id="sec-12" class="subsection"><h3>Environmentally-associated SNP identification</h3><p id="p-36">We used Bayenv 2.0<sup><a id="xref-ref-18-2" class="xref-bibr" href="#ref-18">18</a></sup> to search for SNPs highly associated with environmental variables. Bayenv estimates the relationship between SNPs and environments while controlling the whole-genome population structure from a subset of loose linkage-disequilibrium SNPs. Loose linkage-disequilibrium SNPs were formed by sampling (1) one SNP from scaffolds more than 10 kb and less than 100 kb, (2) two SNPs from scaffolds more than 100 kb and less than 500 kb, (3) three SNPs from scaffolds more than 500 kb and less than 1000 kb, and (4) four SNPs from scaffolds more than 1000 kb. We then, for each bioclimatic variable, defined as the adaptive SNPs ones exhibiting top 1% Bayes factor and top 5% rho value (a nonparametric correlation coefficient capable to reduce outlier effects).</p><p id="p-37">We further investigated the fate of currently adaptive SNPs under anthropogenic climate change, performing the same Bayenv analyses of currently adaptive SNPs using future climatic conditions. We included two time periods (2050 and 2070) and four Representative Concentration Pathways (RCP 2.6, RCP 4.5, RCP 6.0, and RCP 8.5) from two general circulation models, CCSM<sup><a id="xref-ref-44-2" class="xref-bibr" href="#ref-44">44</a></sup> and MIROC<sup><a id="xref-ref-45-2" class="xref-bibr" href="#ref-45">45</a></sup>, resulting in 16 future climatic conditions. If a currently adaptive SNP remains strongly associated with environments, it should exhibit Bayes factor above the current threshold. We then defined as “retention” a currently adaptive SNP constantly exhibiting Bayes factor above the current adaptive threshold in all future scenarios, and defined as “disruption” a currently adaptive SNP exhibiting Bayes factor above the current adaptive threshold in none of the future scenarios.</p></div><div id="sec-13" class="subsection"><h3>The gradient forest method and genetic offset</h3><p id="p-38">We used a novel method, gradient forest<sup><a id="xref-ref-19-2" class="xref-bibr" href="#ref-19">19</a>,<a id="xref-ref-20-3" class="xref-bibr" href="#ref-20">20</a></sup>, to estimate the effect of environmental gradients on allele frequency differences among populations. Gradient forest is a regression-tree based machine-learning algorithm using environmental variables to partition SNP allele frequencies. The analysis was done separately for each SNP. The resulting “importance” measures how much of the variation in allele frequency was explained by partitioning the populations based on a specific value in an environmental variable. By making multiple regression trees (thus generating a random forest) for a SNP, the goodness-of-fit <em>r</em><sup><em>2</em></sup> of a random forest is measured as the proportion of variance explained by this random forest, which is then partitioned among environmental variables in proportion to their conditional importance. Such SNP-wise importance of each environmental variable is then averaged across SNPs belonging to the standing variation (SV) or new mutations (NM) group, resulting in the overall importance (of each environmental variable). In the end, one could obtain the relation curve between environmental gradient and cumulative importance (analogous to the cumulative <em>r</em><sup>2</sup>, proportion of allele frequency differences among populations explained by environments). This curve has two properties. First, the highest point of the cumulative importance curve denotes the overall association between a climatic variable and allele frequency, and we used this to represent the effect size of these SNPs. Second, when traversing along the environmental gradient, a sudden increase of cumulative importance at some environmental value (for example, 20°C) denotes populations on either side of this environmental cutoff have very different allele frequency compositions. In other words, this represents a threshold factor for local adaptation.</p><p id="p-39">One can use this cumulative importance curve to estimate the effect of future environmental change on local populations. In the example above, a population’s local temperature increased from 19°C to 21°C due to climate change would require larger allele frequency shift than another population whose local temperature changed from 17°C to 19°C. The “genetic offset”<sup><a id="xref-ref-10-4" class="xref-bibr" href="#ref-10">10</a></sup> could then be calculated as the Euclidean distance between cumulative importance corresponding to the contemporary environmental value and that corresponding to the future environmental value, considering all bioclimatic variables together. Genetic offset can then be considered to be the magnitude of genetic change needed for a population to be still adaptive in the face of climate change.</p></div><div id="sec-14" class="subsection"><h3>Regression slope</h3><p id="p-40">The regression slope is not given by gradient forest, since it only reports the <em>r</em><sup><em>2</em></sup> importance estimate. Thus, we introduced the simple linear regression <em>y=α+βx</em> to measure the regression slope. We took <em>y</em> as the allele frequency, <em>x</em> as the standardized bioclimatic variable, and <em>β</em> (slope) as the measurement of the amount of allele frequency changes along environmental gradients. By fitting simple linear regression with the general least-square approach, <em>β</em> can then be expanded to <span class="inline-formula" id="inline-formula-1"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/01/14/2020.01.12.903427/embed/inline-graphic-1.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/01/14/2020.01.12.903427/embed/inline-graphic-1.gif"/></noscript></span></span>, where <em>r</em><sub><em>xy</em></sub> is the correlation coefficient (the square root of gradient forest measured “importance”) between <em>x</em> (environment value) and <em>y</em> (allele frequency), and <em>s</em><sub><em>x</em></sub> and <em>s</em><sub><em>y</em></sub> are the standard deviation of <em>x</em> and <em>y</em>.</p></div></div><div class="section" id="sec-15"><h2 class="">Data Availability</h2><p id="p-41">Population pooled sequencing reads are available under NCBI BioProject PRJNA575344.</p></div><div class="section" id="sec-16"><h2 class="">Author Contributions</h2><p id="p-43">C-RL designed the research. Both authors performed experiments and analyses and wrote the manuscript.</p></div><div class="section" id="sec-17"><h2 class="">Competing Interest Declaration</h2><p id="p-44">None</p><div id="F5" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/01/14/2020.01.12.903427/F5.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Sampling site and population structure of Taiwanese Musa itinerans under STRUCTURE K = 2. a, Solid circles represent collection locations corresponding to their coordinates and elevation: Gray circles indicate populations of low elevation (LE); blue circles indicate populations of Xitou transect (XT); red circles indicate populations of Taipingshan transect (TPS). b, Individual ancestry is plotted on the right side, while population ancestry is plotted on map with a pie chart. Map template is provided by *Cheng-Tao Lin. *Cheng-Tao Lin (2018) QGIS template for displaying species distribution by horizontal and vertical view in Taiwan. URL: https://github.com/mutolisp/distrmap_tw.qgis. DOI: 10.5281/zenodo.1493690" class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1212661093" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;&lt;span class=&quot;caption-title&quot;&gt;Sampling site and population structure of Taiwanese &lt;em&gt;Musa itinerans&lt;/em&gt; under STRUCTURE &lt;em&gt;K&lt;/em&gt; = 2.&lt;/span&gt; &lt;strong&gt;a&lt;/strong&gt;, Solid circles represent collection locations corresponding to their coordinates and elevation: Gray circles indicate populations of low elevation (LE); blue circles indicate populations of Xitou transect (XT); red circles indicate populations of Taipingshan transect (TPS). &lt;strong&gt;b&lt;/strong&gt;, Individual ancestry is plotted on the right side, while population ancestry is plotted on map with a pie chart. Map template is provided by *Cheng-Tao Lin. *Cheng-Tao Lin (2018) QGIS template for displaying species distribution by horizontal and vertical view in Taiwan. URL: https://github.com/mutolisp/distrmap_tw.qgis. DOI: 10.5281/zenodo.1493690&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Supplementary Fig. 1" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2020/01/14/2020.01.12.903427/F5.medium.gif" width="271" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Supplementary Fig. 1" src="https://www.biorxiv.org/content/biorxiv/early/2020/01/14/2020.01.12.903427/F5.medium.gif" width="271" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/01/14/2020.01.12.903427/F5.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Supplementary Fig. 1" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/01/14/2020.01.12.903427/F5.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Supplementary Fig. 1</span> <span class="caption-title">Sampling site and population structure of Taiwanese <em>Musa itinerans</em> under STRUCTURE <em>K</em> = 2.</span><p id="p-45" class="first-child"><strong>a</strong>, Solid circles represent collection locations corresponding to their coordinates and elevation: Gray circles indicate populations of low elevation (LE); blue circles indicate populations of Xitou transect (XT); red circles indicate populations of Taipingshan transect (TPS). <strong>b</strong>, Individual ancestry is plotted on the right side, while population ancestry is plotted on map with a pie chart. Map template is provided by</p><p id="p-46">*Cheng-Tao Lin.</p><p id="p-47">*Cheng-Tao Lin (2018) QGIS template for displaying species distribution by horizontal and vertical view in Taiwan. URL: <a href="https://github.com/mutolisp/distrmap_tw.qgis">https://github.com/mutolisp/distrmap_tw.qgis</a>. DOI: 10.5281/zenodo.1493690</p><div class="sb-div caption-clear"></div></div></div><div id="F6" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/01/14/2020.01.12.903427/F6.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Allele frequency distribution. a, Minor allele frequency distribution of adaptive and non-adaptive SNPs. b, Frequency of new alleles in Taiwanese populations within the ancestral or novel environmental range. Statistical significance from Student&#x2019;s t-test between the novel and ancestral environmental range for each bioclimatic variable is shown. Values on the horizontal axis denote the number of Taiwanese populations within the ancestral or novel environmental range." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1212661093" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;&lt;span class=&quot;caption-title&quot;&gt;Allele frequency distribution.&lt;/span&gt; &lt;strong&gt;a&lt;/strong&gt;, Minor allele frequency distribution of adaptive and non-adaptive SNPs. &lt;strong&gt;b&lt;/strong&gt;, Frequency of new alleles in Taiwanese populations within the ancestral or novel environmental range. Statistical significance from Student&#x2019;s &lt;em&gt;t&lt;/em&gt;-test between the novel and ancestral environmental range for each bioclimatic variable is shown. Values on the horizontal axis denote the number of Taiwanese populations within the ancestral or novel environmental range.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Supplementary Fig. 2" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2020/01/14/2020.01.12.903427/F6.medium.gif" width="314" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Supplementary Fig. 2" src="https://www.biorxiv.org/content/biorxiv/early/2020/01/14/2020.01.12.903427/F6.medium.gif" width="314" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/01/14/2020.01.12.903427/F6.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Supplementary Fig. 2" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/01/14/2020.01.12.903427/F6.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Supplementary Fig. 2</span> <span class="caption-title">Allele frequency distribution.</span><p id="p-48" class="first-child"><strong>a</strong>, Minor allele frequency distribution of adaptive and non-adaptive SNPs. <strong>b</strong>, Frequency of new alleles in Taiwanese populations within the ancestral or novel environmental range. Statistical significance from Student’s <em>t</em>-test between the novel and ancestral environmental range for each bioclimatic variable is shown. Values on the horizontal axis denote the number of Taiwanese populations within the ancestral or novel environmental range.</p><div class="sb-div caption-clear"></div></div></div><div id="F7" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/01/14/2020.01.12.903427/F7.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Profiling of effect sizes. a, Bayenv Bayes factor distribution for adaptive new mutations and standing variation (*P &lt; 0.05, **P &lt; 0.01, ***P &lt; 0.001, Wilcoxon rank-sum test). b, Cumulative r2 importance from gradient forest along environmental gradients. c, Gradient forest r2 importance distribution (***P &lt; 0.001, Welch&#x2019;s t-test). d, Example figure showing the relationship between r2 importance and slope. r2 indicates the extent that allele frequency fits a linear model, while slope indicates the amount of allele frequency changes along the linear relationship. The graphs indicate one should also investigate regression slopes in addition to the gradient forest r2. Values on the horizontal axis show the range of standardized environmental variables. e, The distribution of regression slopes when one regresses adaptive SNP allele frequency onto environmental gradients (**P &lt; 0.01, ***P &lt; 0.001, Welch&#x2019;s t-test)." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1212661093" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;&lt;span class=&quot;caption-title&quot;&gt;Profiling of effect sizes.&lt;/span&gt; &lt;strong&gt;a&lt;/strong&gt;, Bayenv Bayes factor distribution for adaptive new mutations and standing variation (*&lt;em&gt;P&lt;/em&gt; &lt; 0.05, **&lt;em&gt;P&lt;/em&gt; &lt; 0.01, ***&lt;em&gt;P&lt;/em&gt; &lt; 0.001, Wilcoxon rank-sum test). &lt;strong&gt;b&lt;/strong&gt;, Cumulative &lt;em&gt;r&lt;/em&gt;&lt;sup&gt;&lt;em&gt;2&lt;/em&gt;&lt;/sup&gt; importance from gradient forest along environmental gradients. &lt;strong&gt;c&lt;/strong&gt;, Gradient forest &lt;em&gt;r&lt;/em&gt;&lt;sup&gt;&lt;em&gt;2&lt;/em&gt;&lt;/sup&gt; importance distribution (***&lt;em&gt;P&lt;/em&gt; &lt; 0.001, Welch&#x2019;s &lt;em&gt;t&lt;/em&gt;-test). &lt;strong&gt;d&lt;/strong&gt;, Example figure showing the relationship between &lt;em&gt;r&lt;/em&gt;&lt;sup&gt;&lt;em&gt;2&lt;/em&gt;&lt;/sup&gt; importance and slope. &lt;em&gt;r&lt;/em&gt;&lt;sup&gt;&lt;em&gt;2&lt;/em&gt;&lt;/sup&gt; indicates the extent that allele frequency fits a linear model, while slope indicates the amount of allele frequency changes along the linear relationship. The graphs indicate one should also investigate regression slopes in addition to the gradient forest &lt;em&gt;r&lt;/em&gt;&lt;sup&gt;&lt;em&gt;2&lt;/em&gt;&lt;/sup&gt;. Values on the horizontal axis show the range of standardized environmental variables. &lt;strong&gt;e&lt;/strong&gt;, The distribution of regression slopes when one regresses adaptive SNP allele frequency onto environmental gradients (**&lt;em&gt;P&lt;/em&gt; &lt; 0.01, ***&lt;em&gt;P&lt;/em&gt; &lt; 0.001, Welch&#x2019;s &lt;em&gt;t&lt;/em&gt;-test).&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Supplementary Fig. 3" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2020/01/14/2020.01.12.903427/F7.medium.gif" width="440" height="434"/><noscript><img class="highwire-fragment fragment-image" alt="Supplementary Fig. 3" src="https://www.biorxiv.org/content/biorxiv/early/2020/01/14/2020.01.12.903427/F7.medium.gif" width="440" height="434"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/01/14/2020.01.12.903427/F7.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Supplementary Fig. 3" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/01/14/2020.01.12.903427/F7.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Supplementary Fig. 3</span> <span class="caption-title">Profiling of effect sizes.</span><p id="p-49" class="first-child"><strong>a</strong>, Bayenv Bayes factor distribution for adaptive new mutations and standing variation (*<em>P</em> &lt; 0.05, **<em>P</em> &lt; 0.01, ***<em>P</em> &lt; 0.001, Wilcoxon rank-sum test). <strong>b</strong>, Cumulative <em>r</em><sup><em>2</em></sup> importance from gradient forest along environmental gradients. <strong>c</strong>, Gradient forest <em>r</em><sup><em>2</em></sup> importance distribution (***<em>P</em> &lt; 0.001, Welch’s <em>t</em>-test). <strong>d</strong>, Example figure showing the relationship between <em>r</em><sup><em>2</em></sup> importance and slope. <em>r</em><sup><em>2</em></sup> indicates the extent that allele frequency fits a linear model, while slope indicates the amount of allele frequency changes along the linear relationship. The graphs indicate one should also investigate regression slopes in addition to the gradient forest <em>r</em><sup><em>2</em></sup>. Values on the horizontal axis show the range of standardized environmental variables. <strong>e</strong>, The distribution of regression slopes when one regresses adaptive SNP allele frequency onto environmental gradients (**<em>P</em> &lt; 0.01, ***<em>P</em> &lt; 0.001, Welch’s <em>t</em>-test).</p><div class="sb-div caption-clear"></div></div></div><div id="F8" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/01/14/2020.01.12.903427/F8.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Evolutionary analyses on Taiwanese and Chinese Musa itinerans. a, Minor allele frequency distribution of adaptive SNPs in Taiwan. b, Phylogeny of Taiwanese and Chinese Musa itinerans. The gray-colored indicates Taiwanese lowland populations (LE); the blue-colored indicates Taiwanese populations of Xitou transect (XT); the red-colored indicates Taiwanese populations of Taipingshan transect (TPS); the black-colored indicates Chinese accessions and the outgroup Musa acuminata. c, Table showing the expected heterozygosity among Taiwanese and Chinese populations." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1212661093" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;&lt;span class=&quot;caption-title&quot;&gt;Evolutionary analyses on Taiwanese and Chinese &lt;em&gt;Musa itinerans&lt;/em&gt;.&lt;/span&gt; &lt;strong&gt;a&lt;/strong&gt;, Minor allele frequency distribution of adaptive SNPs in Taiwan. &lt;strong&gt;b&lt;/strong&gt;, Phylogeny of Taiwanese and Chinese &lt;em&gt;Musa itinerans&lt;/em&gt;. The gray-colored indicates Taiwanese lowland populations (LE); the blue-colored indicates Taiwanese populations of Xitou transect (XT); the red-colored indicates Taiwanese populations of Taipingshan transect (TPS); the black-colored indicates Chinese accessions and the outgroup &lt;em&gt;Musa acuminata&lt;/em&gt;. &lt;strong&gt;c&lt;/strong&gt;, Table showing the expected heterozygosity among Taiwanese and Chinese populations.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Supplementary Fig. 4" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2020/01/14/2020.01.12.903427/F8.medium.gif" width="440" height="258"/><noscript><img class="highwire-fragment fragment-image" alt="Supplementary Fig. 4" src="https://www.biorxiv.org/content/biorxiv/early/2020/01/14/2020.01.12.903427/F8.medium.gif" width="440" height="258"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/01/14/2020.01.12.903427/F8.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Supplementary Fig. 4" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/01/14/2020.01.12.903427/F8.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Supplementary Fig. 4</span> <span class="caption-title">Evolutionary analyses on Taiwanese and Chinese <em>Musa itinerans</em>.</span><p id="p-50" class="first-child"><strong>a</strong>, Minor allele frequency distribution of adaptive SNPs in Taiwan. <strong>b</strong>, Phylogeny of Taiwanese and Chinese <em>Musa itinerans</em>. The gray-colored indicates Taiwanese lowland populations (LE); the blue-colored indicates Taiwanese populations of Xitou transect (XT); the red-colored indicates Taiwanese populations of Taipingshan transect (TPS); the black-colored indicates Chinese accessions and the outgroup <em>Musa acuminata</em>. <strong>c</strong>, Table showing the expected heterozygosity among Taiwanese and Chinese populations.</p><div class="sb-div caption-clear"></div></div></div><div id="F9" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/01/14/2020.01.12.903427/F9.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Multi-analyses addressing ascertainment of standing variation and new mutation. a, The distribution of gradient forest r2 importance across 100 re-sampling trials. In each re-sampling trials, a random set of 50% new mutations were assigned as standing variation, and the mean r2 importance was reported for each trial. All comparisons show strong statistical significance from Student&#x2019;s t-test between new mutations and introduced standing variation (***P &lt; 0.001). b-d, Profiling of effect sizes of adaptive SNPs with top 50% minor allele frequency. Distribution of Bayes factor (***P &lt; 0.001, Wilcoxon rank-sum test; b), gradient forest r2 importance (*P &lt; 0.05, ***P &lt; 0.001, Welch&#x2019;s t-test; c), and regression slopes of adaptive SNP allele frequency onto environmental gradients (***P &lt; 0.001, Welch&#x2019;s t-test; d) is analyzed between new mutations and standing variation for each bioclimatic variable." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1212661093" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;&lt;span class=&quot;caption-title&quot;&gt;Multi-analyses addressing ascertainment of standing variation and new mutation.&lt;/span&gt; &lt;strong&gt;a&lt;/strong&gt;, The distribution of gradient forest &lt;em&gt;r&lt;/em&gt;&lt;sup&gt;&lt;em&gt;2&lt;/em&gt;&lt;/sup&gt; importance across 100 re-sampling trials. In each re-sampling trials, a random set of 50% new mutations were assigned as standing variation, and the mean &lt;em&gt;r&lt;/em&gt;&lt;sup&gt;&lt;em&gt;2&lt;/em&gt;&lt;/sup&gt; importance was reported for each trial. All comparisons show strong statistical significance from Student&#x2019;s &lt;em&gt;t&lt;/em&gt;-test between new mutations and introduced standing variation (***&lt;em&gt;P&lt;/em&gt; &lt; 0.001). &lt;strong&gt;b&lt;/strong&gt;-&lt;strong&gt;d&lt;/strong&gt;, Profiling of effect sizes of adaptive SNPs with top 50% minor allele frequency. Distribution of Bayes factor (***&lt;em&gt;P&lt;/em&gt; &lt; 0.001, Wilcoxon rank-sum test; &lt;strong&gt;b&lt;/strong&gt;), gradient forest &lt;em&gt;r&lt;/em&gt;&lt;sup&gt;&lt;em&gt;2&lt;/em&gt;&lt;/sup&gt; importance (*&lt;em&gt;P&lt;/em&gt; &lt; 0.05, ***&lt;em&gt;P&lt;/em&gt; &lt; 0.001, Welch&#x2019;s &lt;em&gt;t&lt;/em&gt;-test; &lt;strong&gt;c&lt;/strong&gt;), and regression slopes of adaptive SNP allele frequency onto environmental gradients (***&lt;em&gt;P&lt;/em&gt; &lt; 0.001, Welch&#x2019;s &lt;em&gt;t&lt;/em&gt;-test; &lt;strong&gt;d&lt;/strong&gt;) is analyzed between new mutations and standing variation for each bioclimatic variable.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Supplementary Fig. 5" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2020/01/14/2020.01.12.903427/F9.medium.gif" width="440" height="337"/><noscript><img class="highwire-fragment fragment-image" alt="Supplementary Fig. 5" src="https://www.biorxiv.org/content/biorxiv/early/2020/01/14/2020.01.12.903427/F9.medium.gif" width="440" height="337"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/01/14/2020.01.12.903427/F9.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Supplementary Fig. 5" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/01/14/2020.01.12.903427/F9.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Supplementary Fig. 5</span> <span class="caption-title">Multi-analyses addressing ascertainment of standing variation and new mutation.</span><p id="p-51" class="first-child"><strong>a</strong>, The distribution of gradient forest <em>r</em><sup><em>2</em></sup> importance across 100 re-sampling trials. In each re-sampling trials, a random set of 50% new mutations were assigned as standing variation, and the mean <em>r</em><sup><em>2</em></sup> importance was reported for each trial. All comparisons show strong statistical significance from Student’s <em>t</em>-test between new mutations and introduced standing variation (***<em>P</em> &lt; 0.001). <strong>b</strong>-<strong>d</strong>, Profiling of effect sizes of adaptive SNPs with top 50% minor allele frequency. Distribution of Bayes factor (***<em>P</em> &lt; 0.001, Wilcoxon rank-sum test; <strong>b</strong>), gradient forest <em>r</em><sup><em>2</em></sup> importance (*<em>P</em> &lt; 0.05, ***<em>P</em> &lt; 0.001, Welch’s <em>t</em>-test; <strong>c</strong>), and regression slopes of adaptive SNP allele frequency onto environmental gradients (***<em>P</em> &lt; 0.001, Welch’s <em>t</em>-test; <strong>d</strong>) is analyzed between new mutations and standing variation for each bioclimatic variable.</p><div class="sb-div caption-clear"></div></div></div><div id="T1" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/1108475/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup"><a href="/highwire/markup/1108475/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/1108475" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div></div><div class="table-caption"><span class="table-label">Supplementary Table 1</span> <span class="caption-title">Population coordinates and bioclimatic information</span><div class="sb-div caption-clear"></div></div></div><div id="T2" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/1108469/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup"><a href="/highwire/markup/1108469/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/1108469" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div></div><div class="table-caption"><span class="table-label">Supplementary Table 2</span> <span class="caption-title">SSR primer information</span><div class="sb-div caption-clear"></div></div></div><div id="T3" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/1108474/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup"><a href="/highwire/markup/1108474/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/1108474" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div></div><div class="table-caption"><span class="table-label">Supplementary Table 3</span> <span class="caption-title">Number of new-mutation (NM) or standing-variation (SV) SNPs with (adaptive) or without (non-adaptive) significant associations with bioclimatic variables</span><div class="sb-div caption-clear"></div></div></div><div id="T4" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/1108457/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup"><a href="/highwire/markup/1108457/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/1108457" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div></div><div class="table-caption"><span class="table-label">Supplementary Table 4</span> <span class="caption-title">Number of new-mutation (NM) or standing-variation (SV) SNPs with (adaptive) or without (non-adaptive) significant associations with bioclimatic variables (controlled for minor allele frequency)</span><div class="sb-div caption-clear"></div></div></div><div id="T5" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/1108466/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup"><a href="/highwire/markup/1108466/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/1108466" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div></div><div class="table-caption"><span class="table-label">Supplementary Table 5</span> <span class="caption-title">Number of currently adaptive new-mutation (NM) or standing-variation (SV) SNPs that remain (retention) or lose (disruption) significant associations with environments under all future climate-change scenarios</span><div class="sb-div caption-clear"></div></div></div></div><div class="section ack" id="ack-1"><h2 class="">Acknowledgements</h2><p id="p-42">We thank Thomas Mitchell-Olds for valuable comments, Hui-Long Chiu for the knowledge of <em>Musa itinerans</em> ecology in Taiwan, Chia-Yu Chen, Zhe-Ting Kuo, and Jo-Wei Hsieh for the assistance of sample preparation, Hao-Chih Kuo for the introduction to species distribution modeling, Cheng-Tao Lin for providing the R template of some maps, the Lee lab member for valuable discussions, and the Computer and Information Networking Center of National Taiwan University for the support of high-performance computing facilities. This work is supported by the Ministry of Science and Technology of Taiwan (106-2628-B-002-001-MY3, 107-2636-B-002-004, and 108-2636-B-002-004 to CRL).</p></div><div class="section ref-list" id="ref-list-1"><h2 class="">References</h2><ol class="cit-list"><li><a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference 1 in text" id="ref-1">↵</a><div class="cit ref-cit ref-book" id="cit-2020.01.12.903427v1.1"><div class="cit-metadata"><ol class="duplicate"><li><span class="cit-ed"><span class="cit-name-surname">Caspari</span>,  <span class="cit-name-given-names">E. W.</span></span> &amp; </li><li><span class="cit-ed"><span class="cit-name-surname">Thoday</span>,  <span class="cit-name-given-names">J. M.</span></span></li></ol><cite><span class="cit-auth"><span class="cit-name-surname">Bradshaw</span>,  <span class="cit-name-given-names">A. D.</span></span> <span class="cit-chapter-title">Evolutionary significance of phenotypic plasticity in plants</span>. <span class="cit-source">In Advances in Genetics</span> <span class="cit-vol">13</span> (eds <span class="cit-ed"><span class="cit-name-surname">Caspari</span>,  <span class="cit-name-given-names">E. W.</span></span> &amp; <span class="cit-ed"><span class="cit-name-surname">Thoday</span>,  <span class="cit-name-given-names">J. M.</span></span>) <span class="cit-fpage">115</span>–<span class="cit-lpage">155</span> (<span class="cit-publ-name">Academic Press</span>, <span class="cit-pub-date">1965</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DIn%2BAdvances%2Bin%2BGenetics%26rft.volume%253D13%26rft.spage%253D115%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><div class="cit ref-cit ref-journal no-rev-xref" id="cit-2020.01.12.903427v1.2" data-doi="10.1111/j.1365-2435.2007.01283.x"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Ghalambor</span>,  <span class="cit-name-given-names">C. K.</span></span>, <span class="cit-auth"><span class="cit-name-surname">McKay</span>,  <span class="cit-name-given-names">J. K.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Carroll</span>,  <span class="cit-name-given-names">S. P.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Reznick</span>,  <span class="cit-name-given-names">D. N.</span></span> <span class="cit-article-title">Adaptive versus non-adaptive phenotypic plasticity and the potential for contemporary adaptation in new environments</span>. <abbr class="cit-jnl-abbrev">Funct. Ecol</abbr>. <span class="cit-vol">21</span>, <span class="cit-fpage">394</span>–<span class="cit-lpage">407</span><span class="cit-pub-id-sep cit-pub-id-doi-sep">, </span><span class="cit-pub-id-scheme">doi: </span><span class="cit-pub-id cit-pub-id-doi">10.1111/j.1365-2435.2007.01283.x</span> (<span class="cit-pub-date">2007</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DFunct.%2BEcol%26rft.volume%253D21%26rft.spage%253D394%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1365-2435.2007.01283.x%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1365-2435.2007.01283.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000246708200002&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference 3 in text" id="ref-3">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.01.12.903427v1.3" data-doi="10.1111/j.1461-0248.2005.00796.x"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Jump</span>,  <span class="cit-name-given-names">A. S.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Peñuelas</span>,  <span class="cit-name-given-names">J.</span></span> <span class="cit-article-title">Running to stand still: adaptation and the response of plants to rapid climate change</span>. <abbr class="cit-jnl-abbrev">Ecol. Lett</abbr>. <span class="cit-vol">8</span>, <span class="cit-fpage">1010</span>–<span class="cit-lpage">1020</span><span class="cit-pub-id-sep cit-pub-id-doi-sep">, </span><span class="cit-pub-id-scheme">doi: </span><span class="cit-pub-id cit-pub-id-doi">10.1111/j.1461-0248.2005.00796.x</span> (<span class="cit-pub-date">2005</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEcol.%2BLett%26rft.volume%253D8%26rft.spage%253D1010%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1461-0248.2005.00796.x%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1461-0248.2005.00796.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000231224600012&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference 4 in text" id="ref-4">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.01.12.903427v1.4" data-doi="10.1111/eva.12112"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Franks</span>,  <span class="cit-name-given-names">S. J.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Weber</span>,  <span class="cit-name-given-names">J. J.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Aitken</span>,  <span class="cit-name-given-names">S. N.</span></span> <span class="cit-article-title">Evolutionary and plastic responses to climate change in terrestrial plant populations</span>. <abbr class="cit-jnl-abbrev">Evol. Appl</abbr>. <span class="cit-vol">7</span>, <span class="cit-fpage">123</span>–<span class="cit-lpage">139</span><span class="cit-pub-id-sep cit-pub-id-doi-sep">, </span><span class="cit-pub-id-scheme">doi: </span><span class="cit-pub-id cit-pub-id-doi">10.1111/eva.12112</span> (<span class="cit-pub-date">2014</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEvol.%2BAppl%26rft.volume%253D7%26rft.spage%253D123%26rft_id%253Dinfo%253Adoi%252F10.1111%252Feva.12112%26rft_id%253Dinfo%253Apmid%252F24454552%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/eva.12112&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24454552&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F01%2F14%2F2020.01.12.903427.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference 5 in text" id="ref-5">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.01.12.903427v1.5" data-doi="10.1016/j.tree.2007.09.008"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Barrett</span>,  <span class="cit-name-given-names">R. D. H.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Schluter</span>,  <span class="cit-name-given-names">D.</span></span> <span class="cit-article-title">Adaptation from standing genetic variation</span>. <abbr class="cit-jnl-abbrev">Trends Ecol. Evol</abbr>. <span class="cit-vol">23</span>, <span class="cit-fpage">38</span>–<span class="cit-lpage">44</span><span class="cit-pub-id-sep cit-pub-id-doi-sep">, </span><span class="cit-pub-id-scheme">doi: </span><span class="cit-pub-id cit-pub-id-doi">10.1016/j.tree.2007.09.008</span> (<span class="cit-pub-date">2008</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTrends%2Bin%2BEcology%2B%2526%2BEvolution%26rft.stitle%253DTrends%2Bin%2BEcology%2B%2526%2BEvolution%26rft.aulast%253DBarrett%26rft.auinit1%253DR.%2BD.%26rft.volume%253D23%26rft.issue%253D1%26rft.spage%253D38%26rft.epage%253D44%26rft.atitle%253DAdaptation%2Bfrom%2Bstanding%2Bgenetic%2Bvariation.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.tree.2007.09.008%26rft_id%253Dinfo%253Apmid%252F18006185%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.tree.2007.09.008&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18006185&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F01%2F14%2F2020.01.12.903427.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000252767300009&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference 6 in text" id="ref-6">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.01.12.903427v1.6" data-doi="10.1073/pnas.1820663116"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Razgour</span>,  <span class="cit-name-given-names">O.</span></span> <span class="cit-etal">et al.</span> <span class="cit-article-title">Considering adaptive genetic variation in climate change vulnerability assessment reduces species range loss projections</span>. <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci.USA</abbr> <span class="cit-vol">116</span>, <span class="cit-fpage">10418</span>–<span class="cit-lpage">10423</span><span class="cit-pub-id-sep cit-pub-id-doi-sep">, </span><span class="cit-pub-id-scheme">doi: </span><span class="cit-pub-id cit-pub-id-doi">10.1073/pnas.1820663116</span> (<span class="cit-pub-date">2019</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.USA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.1820663116%26rft_id%253Dinfo%253Apmid%252F31061126%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMjoiMTE2LzIxLzEwNDE4IjtzOjQ6ImF0b20iO3M6NDg6Ii9iaW9yeGl2L2Vhcmx5LzIwMjAvMDEvMTQvMjAyMC4wMS4xMi45MDM0MjcuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference 7 in text" id="ref-7">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.01.12.903427v1.7" data-doi="10.1073/pnas.1813597116"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Lai</span>,  <span class="cit-name-given-names">Y. T.</span></span> <span class="cit-etal">et al.</span> <span class="cit-article-title">Standing genetic variation as the predominant source for adaptation of a songbird</span>. <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">116</span>, <span class="cit-fpage">2152</span>–<span class="cit-lpage">2157</span><span class="cit-pub-id-sep cit-pub-id-doi-sep">, </span><span class="cit-pub-id-scheme">doi: </span><span class="cit-pub-id cit-pub-id-doi">10.1073/pnas.1813597116</span> (<span class="cit-pub-date">2019</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.1813597116%26rft_id%253Dinfo%253Apmid%252F30659151%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMDoiMTE2LzYvMjE1MiI7czo0OiJhdG9tIjtzOjQ4OiIvYmlvcnhpdi9lYXJseS8yMDIwLzAxLzE0LzIwMjAuMDEuMTIuOTAzNDI3LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference 8 in text" id="ref-8">↵</a><div class="cit ref-cit ref-book" id="cit-2020.01.12.903427v1.8"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Fisher</span>,  <span class="cit-name-given-names">R. A.</span></span> <span class="cit-source">The Genetical Theory of Natural Selection</span>. (<span class="cit-publ-name">Clarendon Press</span>, <span class="cit-publ-loc">Oxford, England</span>, <span class="cit-pub-date">1930</span>).</cite></div><div class="cit-extra"></div></div></li><li><a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference 9 in text" id="ref-9">↵</a><div class="cit ref-cit ref-book" id="cit-2020.01.12.903427v1.9"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Kimura</span>,  <span class="cit-name-given-names">M.</span></span> <span class="cit-source">The Neutral Theory of Molecular Evolution</span>. (<span class="cit-publ-name">Cambridge University Press</span>, <span class="cit-publ-loc">Cambridge, England</span>, <span class="cit-pub-date">1983</span>).</cite></div><div class="cit-extra"></div></div></li><li><a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference 10 in text" id="ref-10">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.01.12.903427v1.10" data-doi="10.2307/2411226"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Orr</span>,  <span class="cit-name-given-names">H. A.</span></span> <span class="cit-article-title">The population genetics of adaptation: The distribution of factors fixed during adaptive evolution</span>. <abbr class="cit-jnl-abbrev">Evolution</abbr> <span class="cit-vol">52</span>, <span class="cit-fpage">935</span>–<span class="cit-lpage">949</span><span class="cit-pub-id-sep cit-pub-id-doi-sep">, </span><span class="cit-pub-id-scheme">doi: </span><span class="cit-pub-id cit-pub-id-doi">10.2307/2411226</span> (<span class="cit-pub-date">1998</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEvolution%26rft.volume%253D52%26rft.spage%253D935%26rft_id%253Dinfo%253Adoi%252F10.2307%252F2411226%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2307/2411226&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000075753100001&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference 11 in text" id="ref-11">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.01.12.903427v1.11" data-doi="10.1038/nature08480"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Barrick</span>,  <span class="cit-name-given-names">J. E.</span></span> <span class="cit-etal">et al.</span> <span class="cit-article-title">Genome evolution and adaptation in a long-term experiment with <em>Escherichia coli</em></span>. <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">461</span>, <span class="cit-fpage">1243</span>–<span class="cit-lpage">1247</span><span class="cit-pub-id-sep cit-pub-id-doi-sep">, </span><span class="cit-pub-id-scheme">doi: </span><span class="cit-pub-id cit-pub-id-doi">10.1038/nature08480</span> (<span class="cit-pub-date">2009</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DBarrick%26rft.auinit1%253DJ.%2BE.%26rft.volume%253D461%26rft.issue%253D7268%26rft.spage%253D1243%26rft.epage%253D1247%26rft.atitle%253DGenome%2Bevolution%2Band%2Badaptation%2Bin%2Ba%2Blong-term%2Bexperiment%2Bwith%2BEscherichia%2Bcoli.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature08480%26rft_id%253Dinfo%253Apmid%252F19838166%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature08480&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19838166&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F01%2F14%2F2020.01.12.903427.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000271190800040&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference 12 in text" id="ref-12">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.01.12.903427v1.12" data-doi="10.1111/j.1558-5646.2011.01380.x"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Gifford</span>,  <span class="cit-name-given-names">D. R.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Schoustra</span>,  <span class="cit-name-given-names">S. E.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Kassen</span>,  <span class="cit-name-given-names">R.</span></span> <span class="cit-article-title">The length of adaptive walks is insensitive to starting fitness in <em>Aspergillus nidulans</em></span>. <abbr class="cit-jnl-abbrev">Evolution</abbr> <span class="cit-vol">65</span>, <span class="cit-fpage">3070</span>–<span class="cit-lpage">3078</span><span class="cit-pub-id-sep cit-pub-id-doi-sep">, </span><span class="cit-pub-id-scheme">doi: </span><span class="cit-pub-id cit-pub-id-doi">10.1111/j.1558-5646.2011.01380.x</span> (<span class="cit-pub-date">2011</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEvolution%26rft.stitle%253DEvolution%26rft.aulast%253DGifford%26rft.auinit1%253DD.%2BR.%26rft.volume%253D65%26rft.issue%253D11%26rft.spage%253D3070%26rft.epage%253D3078%26rft.atitle%253DThe%2Blength%2Bof%2Badaptive%2Bwalks%2Bis%2Binsensitive%2Bto%2Bstarting%2Bfitness%2Bin%2BAspergillus%2Bnidulans.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1558-5646.2011.01380.x%26rft_id%253Dinfo%253Apmid%252F22023575%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1558-5646.2011.01380.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22023575&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F01%2F14%2F2020.01.12.903427.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference 13 in text" id="ref-13">↵</a><div class="cit ref-cit ref-book" id="cit-2020.01.12.903427v1.13"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Bramwell</span>,  <span class="cit-name-given-names">D.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Caujapé-Castells</span>,  <span class="cit-name-given-names">J.</span></span> <span class="cit-source">The Biology of Island Floras</span>. (<span class="cit-publ-name">Cambridge University Press</span>, <span class="cit-publ-loc">Cambridge</span>, <span class="cit-pub-date">2011</span>).</cite></div><div class="cit-extra"></div></div></li><li><a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference 14 in text" id="ref-14">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.01.12.903427v1.14" data-doi="10.1111/nph.13856"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Janssens</span>,  <span class="cit-name-given-names">S. B.</span></span> <span class="cit-etal">et al.</span> <span class="cit-article-title">Evolutionary dynamics and biogeography of Musaceae reveal a correlation between the diversification of the banana family and the geological and climatic history of Southeast Asia</span>. <abbr class="cit-jnl-abbrev">New Phytol</abbr>. <span class="cit-vol">210</span>, <span class="cit-fpage">1453</span>–<span class="cit-lpage">1465</span><span class="cit-pub-id-sep cit-pub-id-doi-sep">, </span><span class="cit-pub-id-scheme">doi: </span><span class="cit-pub-id cit-pub-id-doi">10.1111/nph.13856</span> (<span class="cit-pub-date">2016</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNew%2BPhytol%26rft.volume%253D210%26rft.spage%253D1453%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fnph.13856%26rft_id%253Dinfo%253Apmid%252F26832306%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/nph.13856&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=26832306&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F01%2F14%2F2020.01.12.903427.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference 15 in text" id="ref-15">↵</a><div class="cit ref-cit ref-book" id="cit-2020.01.12.903427v1.15"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Chiu</span>,  <span class="cit-name-given-names">H. L.</span></span> <span class="cit-chapter-title">The collection, evaluation and analysis of genetic diversity of <em>Musa formosana</em> (Warb.)</span> <span class="cit-source">Hayata native in Taiwan</span>. (<span class="cit-publ-name">National Taiwan University</span>, <span class="cit-pub-date">2004</span>).</cite></div><div class="cit-extra"></div></div></li><li><a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference 16 in text" id="ref-16">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.01.12.903427v1.16" data-doi="10.1016/j.ecolmodel.2005.03.026"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Phillips</span>,  <span class="cit-name-given-names">S. J.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Anderson</span>,  <span class="cit-name-given-names">R. P.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Schapire</span>,  <span class="cit-name-given-names">R. E.</span></span> <span class="cit-article-title">Maximum entropy modeling of species geographic distributions</span>. <abbr class="cit-jnl-abbrev">Ecol. Model</abbr>. <span class="cit-vol">190</span>, <span class="cit-fpage">231</span>–<span class="cit-lpage">259</span><span class="cit-pub-id-sep cit-pub-id-doi-sep">, </span><span class="cit-pub-id-scheme">doi: </span><span class="cit-pub-id cit-pub-id-doi">10.1016/j.ecolmodel.2005.03.026</span> (<span class="cit-pub-date">2006</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEcol.%2BModel%26rft.volume%253D190%26rft.spage%253D231%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.ecolmodel.2005.03.026%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.ecolmodel.2005.03.026&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference 17 in text" id="ref-17">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.01.12.903427v1.17" data-doi="10.1111/j.1558-5646.2007.00299.x"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Nosil</span>,  <span class="cit-name-given-names">P.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Egan</span>,  <span class="cit-name-given-names">S. P.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Funk</span>,  <span class="cit-name-given-names">D. J.</span></span> <span class="cit-article-title">Heterogeneous genomic differentiation between walking-stick ecotypes: “isolation by adaptation” and multiple roles for divergent selection</span>. <abbr class="cit-jnl-abbrev">Evolution</abbr> <span class="cit-vol">62</span>, <span class="cit-fpage">316</span>–<span class="cit-lpage">336</span><span class="cit-pub-id-sep cit-pub-id-doi-sep">, </span><span class="cit-pub-id-scheme">doi: </span><span class="cit-pub-id cit-pub-id-doi">10.1111/j.1558-5646.2007.00299.x</span> (<span class="cit-pub-date">2008</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEvolution%26rft.stitle%253DEvolution%26rft.aulast%253DNosil%26rft.auinit1%253DP.%26rft.volume%253D62%26rft.issue%253D2%26rft.spage%253D316%26rft.epage%253D336%26rft.atitle%253DHeterogeneous%2Bgenomic%2Bdifferentiation%2Bbetween%2Bwalking-stick%2Becotypes%253A%2B%2526quot%253Bisolation%2Bby%2Badaptation%2526quot%253B%2Band%2Bmultiple%2Broles%2Bfor%2Bdivergent%2Bselection.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1558-5646.2007.00299.x%26rft_id%253Dinfo%253Apmid%252F17999721%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1558-5646.2007.00299.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17999721&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F01%2F14%2F2020.01.12.903427.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000253373900006&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference 18 in text" id="ref-18">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.01.12.903427v1.18" data-doi="10.1534/genetics.113.152462"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Gunther</span>,  <span class="cit-name-given-names">T.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Coop</span>,  <span class="cit-name-given-names">G.</span></span> <span class="cit-article-title">Robust identification of local adaptation from allele frequencies</span>. <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">195</span>, <span class="cit-fpage">205</span>–<span class="cit-lpage">220</span><span class="cit-pub-id-sep cit-pub-id-doi-sep">, </span><span class="cit-pub-id-scheme">doi: </span><span class="cit-pub-id cit-pub-id-doi">10.1534/genetics.113.152462</span> (<span class="cit-pub-date">2013</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.113.152462%26rft_id%253Dinfo%253Apmid%252F23821598%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTk1LzEvMjA1IjtzOjQ6ImF0b20iO3M6NDg6Ii9iaW9yeGl2L2Vhcmx5LzIwMjAvMDEvMTQvMjAyMC4wMS4xMi45MDM0MjcuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference 19 in text" id="ref-19">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.01.12.903427v1.19" data-doi="10.1890/11-0252.1"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Ellis</span>,  <span class="cit-name-given-names">N.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Smith</span>,  <span class="cit-name-given-names">S. J.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Pitcher</span>,  <span class="cit-name-given-names">C. R.</span></span> <span class="cit-article-title">Gradient forests: calculating importance gradients on physical predictors</span>. <abbr class="cit-jnl-abbrev">Ecology</abbr> <span class="cit-vol">93</span>, <span class="cit-fpage">156</span>–<span class="cit-lpage">168</span><span class="cit-pub-id-sep cit-pub-id-doi-sep">, </span><span class="cit-pub-id-scheme">doi: </span><span class="cit-pub-id cit-pub-id-doi">10.1890/11-0252.1</span> (<span class="cit-pub-date">2012</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEcology%26rft.stitle%253DEcology%26rft.aulast%253DEllis%26rft.auinit1%253DN.%26rft.volume%253D93%26rft.issue%253D1%26rft.spage%253D156%26rft.epage%253D168%26rft.atitle%253DGradient%2Bforests%253A%2Bcalculating%2Bimportance%2Bgradients%2Bon%2Bphysical%2Bpredictors.%26rft_id%253Dinfo%253Adoi%252F10.1890%252F11-0252.1%26rft_id%253Dinfo%253Apmid%252F22486096%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1890/11-0252.1&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22486096&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F01%2F14%2F2020.01.12.903427.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference 20 in text" id="ref-20">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.01.12.903427v1.20" data-doi="10.1111/ele.12376"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Fitzpatrick</span>,  <span class="cit-name-given-names">M. C.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Keller</span>,  <span class="cit-name-given-names">S. R.</span></span> <span class="cit-article-title">Ecological genomics meets community-level modelling of biodiversity: mapping the genomic landscape of current and future environmental adaptation</span>. <abbr class="cit-jnl-abbrev">Ecol. Lett</abbr>. <span class="cit-vol">18</span>, <span class="cit-fpage">1</span>–<span class="cit-lpage">16</span><span class="cit-pub-id-sep cit-pub-id-doi-sep">, </span><span class="cit-pub-id-scheme">doi: </span><span class="cit-pub-id cit-pub-id-doi">10.1111/ele.12376</span> (<span class="cit-pub-date">2015</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEcol.%2BLett%26rft.volume%253D18%26rft.spage%253D1%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fele.12376%26rft_id%253Dinfo%253Apmid%252F25270536%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/ele.12376&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25270536&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F01%2F14%2F2020.01.12.903427.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference 21 in text" id="ref-21">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.01.12.903427v1.21" data-doi="10.1098/rstb.2000.0716"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Barton</span>,  <span class="cit-name-given-names">N. H.</span></span> <span class="cit-article-title">Genetic hitchhiking</span>. <abbr class="cit-jnl-abbrev">Philos. Trans. R. Soc. Lond. Ser. B-Biol. Sci</abbr>. <span class="cit-vol">355</span>, <span class="cit-fpage">1553</span>–<span class="cit-lpage">1562</span><span class="cit-pub-id-sep cit-pub-id-doi-sep">, </span><span class="cit-pub-id-scheme">doi: </span><span class="cit-pub-id cit-pub-id-doi">10.1098/rstb.2000.0716</span> (<span class="cit-pub-date">2000</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPhilosophical%2BTransactions%2Bof%2Bthe%2BRoyal%2BSociety%2BB%253A%2BBiological%2BSciences%26rft.stitle%253DPhil%2BTrans%2BR%2BSoc%2BB%26rft.aulast%253DBarton%26rft.auinit1%253DN.%2BH.%26rft.volume%253D355%26rft.issue%253D1403%26rft.spage%253D1553%26rft.epage%253D1562%26rft.atitle%253DGenetic%2Bhitchhiking%26rft_id%253Dinfo%253Adoi%252F10.1098%252Frstb.2000.0716%26rft_id%253Dinfo%253Apmid%252F11127900%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1098/rstb.2000.0716&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=11127900&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F01%2F14%2F2020.01.12.903427.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000166088400002&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><div class="cit ref-cit ref-journal no-rev-xref" id="cit-2020.01.12.903427v1.22" data-doi="10.1126/science.1175826"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Linnen</span>,  <span class="cit-name-given-names">C. R.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Kingsley</span>,  <span class="cit-name-given-names">E. P.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Jensen</span>,  <span class="cit-name-given-names">J. D.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Hoekstra</span>,  <span class="cit-name-given-names">H. E.</span></span> <span class="cit-article-title">On the origin and spread of an adaptive allele in deer mice</span>. <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">325</span>, <span class="cit-fpage">1095</span>–<span class="cit-lpage">1098</span><span class="cit-pub-id-sep cit-pub-id-doi-sep">, </span><span class="cit-pub-id-scheme">doi: </span><span class="cit-pub-id cit-pub-id-doi">10.1126/science.1175826</span> (<span class="cit-pub-date">2009</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DLinnen%26rft.auinit1%253DC.%2BR.%26rft.volume%253D325%26rft.issue%253D5944%26rft.spage%253D1095%26rft.epage%253D1098%26rft.atitle%253DOn%2Bthe%2BOrigin%2Band%2BSpread%2Bof%2Ban%2BAdaptive%2BAllele%2Bin%2BDeer%2BMice%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1175826%26rft_id%253Dinfo%253Apmid%252F19713521%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEzOiIzMjUvNTk0NC8xMDk1IjtzOjQ6ImF0b20iO3M6NDg6Ii9iaW9yeGl2L2Vhcmx5LzIwMjAvMDEvMTQvMjAyMC4wMS4xMi45MDM0MjcuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><div class="cit ref-cit ref-journal no-rev-xref" id="cit-2020.01.12.903427v1.23" data-doi="10.1038/nature10944"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Jones</span>,  <span class="cit-name-given-names">F. C.</span></span> <span class="cit-etal">et al.</span> <span class="cit-article-title">The genomic basis of adaptive evolution in threespine sticklebacks</span>. <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">484</span>, <span class="cit-fpage">55</span>–<span class="cit-lpage">61</span><span class="cit-pub-id-sep cit-pub-id-doi-sep">, </span><span class="cit-pub-id-scheme">doi: </span><span class="cit-pub-id cit-pub-id-doi">10.1038/nature10944</span> (<span class="cit-pub-date">2012</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DJones%26rft.auinit1%253DF.%2BC.%26rft.volume%253D484%26rft.issue%253D7392%26rft.spage%253D55%26rft.epage%253D61%26rft.atitle%253DThe%2Bgenomic%2Bbasis%2Bof%2Badaptive%2Bevolution%2Bin%2Bthreespine%2Bsticklebacks.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature10944%26rft_id%253Dinfo%253Apmid%252F22481358%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature10944&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22481358&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F01%2F14%2F2020.01.12.903427.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000302343400033&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><div class="cit ref-cit ref-journal no-rev-xref" id="cit-2020.01.12.903427v1.24" data-doi="10.1126/science.aah4993"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Reid</span>,  <span class="cit-name-given-names">N. M.</span></span> <span class="cit-etal">et al.</span> <span class="cit-article-title">The genomic landscape of rapid repeated evolutionary adaptation to toxic pollution in wild fish</span>. <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">354</span>, <span class="cit-fpage">1305</span>–<span class="cit-lpage">1308</span><span class="cit-pub-id-sep cit-pub-id-doi-sep">, </span><span class="cit-pub-id-scheme">doi: </span><span class="cit-pub-id cit-pub-id-doi">10.1126/science.aah4993</span> (<span class="cit-pub-date">2016</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DReid%26rft.auinit1%253DN.%2BM.%26rft.volume%253D354%26rft.issue%253D6317%26rft.spage%253D1305%26rft.epage%253D1308%26rft.atitle%253DThe%2Bgenomic%2Blandscape%2Bof%2Brapid%2Brepeated%2Bevolutionary%2Badaptation%2Bto%2Btoxic%2Bpollution%2Bin%2Bwild%2Bfish%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.aah4993%26rft_id%253Dinfo%253Apmid%252F27940876%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEzOiIzNTQvNjMxNy8xMzA1IjtzOjQ6ImF0b20iO3M6NDg6Ii9iaW9yeGl2L2Vhcmx5LzIwMjAvMDEvMTQvMjAyMC4wMS4xMi45MDM0MjcuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference 25 in text" id="ref-25">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.01.12.903427v1.25" data-doi="10.1111/mec.15092"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Ahrens</span>,  <span class="cit-name-given-names">C. W.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Byrne</span>,  <span class="cit-name-given-names">M.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Rymer</span>,  <span class="cit-name-given-names">P. D.</span></span> <span class="cit-article-title">Standing genomic variation within coding cand regulatory regions contributes to the adaptive capacity to climate in a foundation tree species</span>. <abbr class="cit-jnl-abbrev">Mol. Ecol</abbr>. <span class="cit-vol">28</span>, <span class="cit-fpage">2502</span>–<span class="cit-lpage">2516</span><span class="cit-pub-id-sep cit-pub-id-doi-sep">, </span><span class="cit-pub-id-scheme">doi: </span><span class="cit-pub-id cit-pub-id-doi">10.1111/mec.15092</span> (<span class="cit-pub-date">2019</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BEcol%26rft.volume%253D28%26rft.spage%253D2502%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fmec.15092%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/mec.15092&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference 26 in text" id="ref-26">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.01.12.903427v1.26"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Doyle</span>,  <span class="cit-name-given-names">J. J.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Doyle</span>,  <span class="cit-name-given-names">J. L.</span></span> <span class="cit-article-title">A rapid DNA isolation procedure for small quantities of fresh leaf tissue</span>. <abbr class="cit-jnl-abbrev">Phytochemical Bulletin</abbr> <span class="cit-vol">19</span>, <span class="cit-fpage">11</span>–<span class="cit-lpage">15</span> (<span class="cit-pub-date">1987</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPhytochemical%2BBulletin%26rft.volume%253D19%26rft.spage%253D11%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><a class="rev-xref-ref" href="#xref-ref-27-1" title="View reference 27 in text" id="ref-27">↵</a><div class="cit ref-cit ref-book" id="cit-2020.01.12.903427v1.27"><div class="cit-metadata"><ol class="duplicate"><li><span class="cit-ed"><span class="cit-name-surname">Innis</span>,  <span class="cit-name-given-names">M. A.</span></span>, </li><li><span class="cit-ed"><span class="cit-name-surname">Gelfand</span>,  <span class="cit-name-given-names">D. H.</span></span>, </li><li><span class="cit-ed"><span class="cit-name-surname">Sninsky</span>,  <span class="cit-name-given-names">J. J.</span></span> &amp; </li><li><span class="cit-ed"><span class="cit-name-surname">White</span>,  <span class="cit-name-given-names">T. J.</span></span></li></ol><cite><span class="cit-auth"><span class="cit-name-surname">White</span>,  <span class="cit-name-given-names">T. J.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Bruns</span>,  <span class="cit-name-given-names">T.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Lee</span>,  <span class="cit-name-given-names">S.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Taylor</span>,  <span class="cit-name-given-names">J.</span></span> <span class="cit-chapter-title">Amplification and direct sequencing of fungal ribosomal RNA genes for phylogenetics</span>. in <span class="cit-source">PCR Protocols</span> (eds <span class="cit-ed"><span class="cit-name-surname">Innis</span>,  <span class="cit-name-given-names">M. A.</span></span>, <span class="cit-ed"><span class="cit-name-surname">Gelfand</span>,  <span class="cit-name-given-names">D. H.</span></span>, <span class="cit-ed"><span class="cit-name-surname">Sninsky</span>,  <span class="cit-name-given-names">J. J.</span></span> &amp; <span class="cit-ed"><span class="cit-name-surname">White</span>,  <span class="cit-name-given-names">T. J.</span></span>) <span class="cit-fpage">315</span>–<span class="cit-lpage">322</span> (<span class="cit-publ-name">Academic Press</span>, <span class="cit-publ-loc">San Diego, California</span>, <span class="cit-pub-date">1990</span>).</cite></div><div class="cit-extra"></div></div></li><li><div class="cit ref-cit ref-journal no-rev-xref" id="cit-2020.01.12.903427v1.28" data-doi="10.1007/BF00037152"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Taberlet</span>,  <span class="cit-name-given-names">P.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Gielly</span>,  <span class="cit-name-given-names">L.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Pautou</span>,  <span class="cit-name-given-names">G.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Bouvet</span>,  <span class="cit-name-given-names">J.</span></span> <span class="cit-article-title">Universal primers for amplification of three non-coding regions of chloroplast DNA</span>. <abbr class="cit-jnl-abbrev">Plant Mol. Biol</abbr>. <span class="cit-vol">17</span>, <span class="cit-fpage">1105</span>–<span class="cit-lpage">1109</span><span class="cit-pub-id-sep cit-pub-id-doi-sep">, </span><span class="cit-pub-id-scheme">doi: </span><span class="cit-pub-id cit-pub-id-doi">10.1007/BF00037152</span> (<span class="cit-pub-date">1991</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2Bmolecular%2Bbiology%26rft.stitle%253DPlant%2BMol%2BBiol%26rft.aulast%253DTaberlet%26rft.auinit1%253DP.%26rft.volume%253D17%26rft.issue%253D5%26rft.spage%253D1105%26rft.epage%253D1109%26rft.atitle%253DUniversal%2Bprimers%2Bfor%2Bamplification%2Bof%2Bthree%2Bnon-coding%2Bregions%2Bof%2Bchloroplast%2BDNA.%26rft_id%253Dinfo%253Adoi%252F10.1007%252FBF00037152%26rft_id%253Dinfo%253Apmid%252F1932684%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/BF00037152&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=1932684&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F01%2F14%2F2020.01.12.903427.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1991GM46200017&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><div class="cit ref-cit ref-journal no-rev-xref" id="cit-2020.01.12.903427v1.29" data-doi="10.1007/BF00987959"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Oxelman</span>,  <span class="cit-name-given-names">B.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Lidén</span>,  <span class="cit-name-given-names">M.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Berglund</span>,  <span class="cit-name-given-names">D.</span></span> <span class="cit-article-title">Chloroplast <em>rps16</em> intron phylogeny of the tribe Sileneae (Caryophyllaceae)</span>. <abbr class="cit-jnl-abbrev">Plant Syst. Evol</abbr>. <span class="cit-vol">206</span>, <span class="cit-fpage">393</span>–<span class="cit-lpage">410</span><span class="cit-pub-id-sep cit-pub-id-doi-sep">, </span><span class="cit-pub-id-scheme">doi: </span><span class="cit-pub-id cit-pub-id-doi">10.1007/BF00987959</span> (<span class="cit-pub-date">1997</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BSyst.%2BEvol%26rft.volume%253D206%26rft.spage%253D393%26rft_id%253Dinfo%253Adoi%252F10.1007%252FBF00987959%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/BF00987959&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><a class="rev-xref-ref" href="#xref-ref-30-1" title="View reference 30 in text" id="ref-30">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.01.12.903427v1.30" data-doi="10.1046/j.1365-2699.1999.00302.x"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Hurr</span>,  <span class="cit-name-given-names">K. A.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Lockhart</span>,  <span class="cit-name-given-names">P. J.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Heenan</span>,  <span class="cit-name-given-names">P. B.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Penny</span>,  <span class="cit-name-given-names">D.</span></span> <span class="cit-article-title">Evidence for the recent dispersal of <em>Sophora</em> (Leguminosae) around the Southern Oceans: molecular data</span>. <abbr class="cit-jnl-abbrev">J. Biogeogr</abbr>. <span class="cit-vol">26</span>, <span class="cit-fpage">565</span>–<span class="cit-lpage">577</span><span class="cit-pub-id-sep cit-pub-id-doi-sep">, </span><span class="cit-pub-id-scheme">doi: </span><span class="cit-pub-id cit-pub-id-doi">10.1046/j.1365-2699.1999.00302.x</span> (<span class="cit-pub-date">1999</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BBiogeogr%26rft.volume%253D26%26rft.spage%253D565%26rft_id%253Dinfo%253Adoi%252F10.1046%252Fj.1365-2699.1999.00302.x%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1046/j.1365-2699.1999.00302.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000083517700012&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><a class="rev-xref-ref" href="#xref-ref-31-1" title="View reference 31 in text" id="ref-31">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.01.12.903427v1.31"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Lagoda</span>,  <span class="cit-name-given-names">P. J.</span></span> <span class="cit-etal">et al.</span> <span class="cit-article-title">Sequence tagged microsatellite site (STMS) markers in the Musaceae</span>. <abbr class="cit-jnl-abbrev">Mol. Ecol</abbr>. <span class="cit-vol">7</span>, <span class="cit-fpage">659</span>–<span class="cit-lpage">663</span> (<span class="cit-pub-date">1998</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2BEcology%2B%2528Print%2529%26rft.stitle%253DMolecular%2BEcology%2B%2528Print%2529%26rft.aulast%253DLagoda%26rft.auinit1%253DP.%2BJ.%26rft.volume%253D7%26rft.issue%253D5%26rft.spage%253D659%26rft.epage%253D663%26rft.atitle%253DSequence%2Btagged%2Bmicrosatellite%2Bsite%2B%2528STMS%2529%2Bmarkers%2Bin%2Bthe%2BMusaceae.%26rft_id%253Dinfo%253Apmid%252F9633107%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=9633107&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F01%2F14%2F2020.01.12.903427.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><a class="rev-xref-ref" href="#xref-ref-32-1" title="View reference 32 in text" id="ref-32">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.01.12.903427v1.32" data-doi="10.2135/cropsci1998.0011183X003800010035x"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Crouch</span>,  <span class="cit-name-given-names">H. K.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Crouch</span>,  <span class="cit-name-given-names">J. H.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Jarret</span>,  <span class="cit-name-given-names">R. L.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Cregan</span>,  <span class="cit-name-given-names">P. B.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Ortiz</span>,  <span class="cit-name-given-names">R.</span></span> <span class="cit-article-title">Segregation at microsatellite loci in haploid and diploid gametes of <em>Musa</em></span>. <abbr class="cit-jnl-abbrev">Crop Sci</abbr>. <span class="cit-vol">38</span>, <span class="cit-fpage">211</span>–<span class="cit-lpage">217</span><span class="cit-pub-id-sep cit-pub-id-doi-sep">, </span><span class="cit-pub-id-scheme">doi: </span><span class="cit-pub-id cit-pub-id-doi">10.2135/cropsci1998.0011183X003800010035x</span> (<span class="cit-pub-date">1998</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCrop%2BScience%26rft.stitle%253DCrop%2BSci.%26rft.aulast%253DCrouch%26rft.auinit1%253DH.%2BK.%26rft.volume%253D38%26rft.issue%253D1%26rft.spage%253D211%26rft.epage%253D217%26rft.atitle%253DSegregation%2Bat%2BMicrosatellite%2BLoci%2Bin%2BHaploid%2Band%2BDiploid%2BGametes%2Bof%2BMusa%26rft_id%253Dinfo%253Adoi%252F10.2135%252Fcropsci1998.0011183X003800010035x%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2135/cropsci1998.0011183X003800010035x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000071885100035&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><a class="rev-xref-ref" href="#xref-ref-33-1" title="View reference 33 in text" id="ref-33">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.01.12.903427v1.33" data-doi="10.1186/s12864-016-2579-4."><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Martin</span>,  <span class="cit-name-given-names">G.</span></span> <span class="cit-etal">et al.</span> <span class="cit-article-title">Improvement of the banana “<em>Musa acuminata</em>” reference sequence using NGS data and semi-automated bioinformatics methods</span>. <abbr class="cit-jnl-abbrev">BMC Genomics</abbr> <span class="cit-vol">17</span>, <span class="cit-fpage">243</span><span class="cit-pub-id-sep cit-pub-id-doi-sep">, </span><span class="cit-pub-id-scheme">doi: </span><span class="cit-pub-id cit-pub-id-doi">10.1186/s12864-016-2579-4</span> (<span class="cit-pub-date">2016</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2BGenomics%26rft.volume%253D17%26rft.spage%253D243%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fs12864-016-2579-4.%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/s12864-016-2579-4.&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><a class="rev-xref-ref" href="#xref-ref-34-1" title="View reference 34 in text" id="ref-34">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.01.12.903427v1.34" data-doi="10.1038/72708"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Schuelke</span>,  <span class="cit-name-given-names">M.</span></span> <span class="cit-article-title">An economic method for the fluorescent labeling of PCR fragments</span>. <abbr class="cit-jnl-abbrev">Nat. Biotechnol</abbr>. <span class="cit-vol">18</span>, <span class="cit-fpage">233</span>–<span class="cit-lpage">234</span><span class="cit-pub-id-sep cit-pub-id-doi-sep">, </span><span class="cit-pub-id-scheme">doi: </span><span class="cit-pub-id cit-pub-id-doi">10.1038/72708</span> (<span class="cit-pub-date">2000</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bbiotechnology%26rft.stitle%253DNat%2BBiotechnol%26rft.aulast%253DSchuelke%26rft.auinit1%253DM.%26rft.volume%253D18%26rft.issue%253D2%26rft.spage%253D233%26rft.epage%253D234%26rft.atitle%253DAn%2Beconomic%2Bmethod%2Bfor%2Bthe%2Bfluorescent%2Blabeling%2Bof%2BPCR%2Bfragments.%26rft_id%253Dinfo%253Adoi%252F10.1038%252F72708%26rft_id%253Dinfo%253Apmid%252F10657137%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/72708&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=10657137&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F01%2F14%2F2020.01.12.903427.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000085146900036&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><a class="rev-xref-ref" href="#xref-ref-35-1" title="View reference 35 in text" id="ref-35">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.01.12.903427v1.35" data-doi="10.1111/j.1471-8286.2005.01155.x"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Pritchard</span>,  <span class="cit-name-given-names">J. K.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Stephens</span>,  <span class="cit-name-given-names">M.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Donnelly</span>,  <span class="cit-name-given-names">P.</span></span> <span class="cit-article-title">Inference of population structure using multilocus genotype data</span>. <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">155</span>, <span class="cit-fpage">945</span>–<span class="cit-lpage">959</span> (<span class="cit-pub-date">2000</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DPritchard%26rft.auinit1%253DJ.%2BK.%26rft.volume%253D155%26rft.issue%253D2%26rft.spage%253D945%26rft.epage%253D959%26rft.atitle%253DInference%2Bof%2BPopulation%2BStructure%2BUsing%2BMultilocus%2BGenotype%2BData%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1471-8286.2005.01155.x%26rft_id%253Dinfo%253Apmid%252F10835412%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTU1LzIvOTQ1IjtzOjQ6ImF0b20iO3M6NDg6Ii9iaW9yeGl2L2Vhcmx5LzIwMjAvMDEvMTQvMjAyMC4wMS4xMi45MDM0MjcuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><a class="rev-xref-ref" href="#xref-ref-36-1" title="View reference 36 in text" id="ref-36">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.01.12.903427v1.36" data-doi="PubMed PMID: 88"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Falush</span>,  <span class="cit-name-given-names">D.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Stephens</span>,  <span class="cit-name-given-names">M.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Pritchard</span>,  <span class="cit-name-given-names">J. K.</span></span> <span class="cit-article-title">Inference of population structure using multilocus genotype data: linked loci and correlated allele frequencies</span>. <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">164</span>, <span class="cit-fpage">1567</span>–<span class="cit-lpage">1587</span> (<span class="cit-pub-date">2003</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DFalush%26rft.auinit1%253DD.%26rft.volume%253D164%26rft.issue%253D4%26rft.spage%253D1567%26rft.epage%253D1587%26rft.atitle%253DInference%2Bof%2BPopulation%2BStructure%2BUsing%2BMultilocus%2BGenotype%2BData%253A%2BLinked%2BLoci%2Band%2BCorrelated%2BAllele%2BFrequencies%26rft_id%253Dinfo%253Adoi%252FPubMed%2BPMID%253A%2B88%26rft_id%253Dinfo%253Apmid%252F12930761%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE2NC80LzE1NjciO3M6NDoiYXRvbSI7czo0ODoiL2Jpb3J4aXYvZWFybHkvMjAyMC8wMS8xNC8yMDIwLjAxLjEyLjkwMzQyNy5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><a class="rev-xref-ref" href="#xref-ref-37-1" title="View reference 37 in text" id="ref-37">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.01.12.903427v1.37" data-doi="10.1111/j.1471-8286.2005.01155.x"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Peakall</span>,  <span class="cit-name-given-names">R. O. D.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Smouse</span>,  <span class="cit-name-given-names">P. E.</span></span> <span class="cit-article-title">Genalex 6: genetic analysis in Excel. Population genetic software for teaching and research</span>. <abbr class="cit-jnl-abbrev">Mol. Ecol. Notes</abbr> <span class="cit-vol">6</span>, <span class="cit-fpage">288</span>–<span class="cit-lpage">295</span><span class="cit-pub-id-sep cit-pub-id-doi-sep">, </span><span class="cit-pub-id-scheme">doi: </span><span class="cit-pub-id cit-pub-id-doi">10.1111/j.1471-8286.2005.01155.x</span> (<span class="cit-pub-date">2006</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BEcol.%2BNotes%26rft.volume%253D6%26rft.spage%253D288%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1471-8286.2005.01155.x%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1471-8286.2005.01155.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000235725600089&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><a class="rev-xref-ref" href="#xref-ref-38-1" title="View reference 38 in text" id="ref-38">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.01.12.903427v1.38" data-doi="10.1093/bioinformatics/bts460"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Peakall</span>,  <span class="cit-name-given-names">R.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Smouse</span>,  <span class="cit-name-given-names">P. E.</span></span> <span class="cit-article-title">GenAlEx 6.5: genetic analysis in Excel. Population genetic software for teaching and research--an update</span>. <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">28</span>, <span class="cit-fpage">2537</span>–<span class="cit-lpage">2539</span><span class="cit-pub-id-sep cit-pub-id-doi-sep">, </span><span class="cit-pub-id-scheme">doi: </span><span class="cit-pub-id cit-pub-id-doi">10.1093/bioinformatics/bts460</span> (<span class="cit-pub-date">2012</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbts460%26rft_id%253Dinfo%253Apmid%252F22820204%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/bts460&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22820204&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F01%2F14%2F2020.01.12.903427.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000309687500024&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><a class="rev-xref-ref" href="#xref-ref-39-1" title="View reference 39 in text" id="ref-39">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.01.12.903427v1.39" data-doi="10.1111/j.1600-0587.2011.06545.x"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Bean</span>,  <span class="cit-name-given-names">W. T.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Stafford</span>,  <span class="cit-name-given-names">R.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Brashares</span>,  <span class="cit-name-given-names">J. S.</span></span> <span class="cit-article-title">The effects of small sample size and sample bias on threshold selection and accuracy assessment of species distribution models</span>. <abbr class="cit-jnl-abbrev">Ecography</abbr> <span class="cit-vol">35</span>, <span class="cit-fpage">250</span>–<span class="cit-lpage">258</span><span class="cit-pub-id-sep cit-pub-id-doi-sep">, </span><span class="cit-pub-id-scheme">doi: </span><span class="cit-pub-id cit-pub-id-doi">10.1111/j.1600-0587.2011.06545.x</span> (<span class="cit-pub-date">2012</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEcography%26rft.volume%253D35%26rft.spage%253D250%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1600-0587.2011.06545.x%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1600-0587.2011.06545.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><div class="cit ref-cit ref-journal no-rev-xref" id="cit-2020.01.12.903427v1.40" data-doi="10.1111/j.2006.0906-7590.04596.x"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Elith</span>,  <span class="cit-name-given-names">J.</span></span> <span class="cit-etal">et al.</span> <span class="cit-article-title">Novel methods improve prediction of species’ distributions from occurrence data</span>. <abbr class="cit-jnl-abbrev">Ecography</abbr> <span class="cit-vol">29</span>, <span class="cit-fpage">129</span>–<span class="cit-lpage">151</span><span class="cit-pub-id-sep cit-pub-id-doi-sep">, </span><span class="cit-pub-id-scheme">doi: </span><span class="cit-pub-id cit-pub-id-doi">10.1111/j.2006.0906-7590.04596.x</span> (<span class="cit-pub-date">2006</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEcography%26rft.volume%253D29%26rft.spage%253D129%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.2006.0906-7590.04596.x%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.2006.0906-7590.04596.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000236767000001&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><a class="rev-xref-ref" href="#xref-ref-41-1" title="View reference 41 in text" id="ref-41">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.01.12.903427v1.41" data-doi="10.1111/j.1472-4642.2010.00725.x"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Elith</span>,  <span class="cit-name-given-names">J.</span></span> <span class="cit-etal">et al.</span> <span class="cit-article-title">A statistical explanation of MaxEnt for ecologists</span>. <abbr class="cit-jnl-abbrev">Divers. Distrib</abbr>. <span class="cit-vol">17</span>, <span class="cit-fpage">43</span>–<span class="cit-lpage">57</span><span class="cit-pub-id-sep cit-pub-id-doi-sep">, </span><span class="cit-pub-id-scheme">doi: </span><span class="cit-pub-id cit-pub-id-doi">10.1111/j.1472-4642.2010.00725.x</span> (<span class="cit-pub-date">2011</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DDivers.%2BDistrib%26rft.volume%253D17%26rft.spage%253D43%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1472-4642.2010.00725.x%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1472-4642.2010.00725.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><a class="rev-xref-ref" href="#xref-ref-42-1" title="View reference 42 in text" id="ref-42">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.01.12.903427v1.42" data-doi="10.1002/joc.1276"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Hijmans</span>,  <span class="cit-name-given-names">R. J.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Cameron</span>,  <span class="cit-name-given-names">S. E.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Parra</span>,  <span class="cit-name-given-names">J. L.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Jones</span>,  <span class="cit-name-given-names">P. G.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Jarvis</span>,  <span class="cit-name-given-names">A.</span></span> <span class="cit-article-title">Very high resolution interpolated climate surfaces for global land areas</span>. <abbr class="cit-jnl-abbrev">Int. J. Climatol</abbr>. <span class="cit-vol">25</span>, <span class="cit-fpage">1965</span>–<span class="cit-lpage">1978</span><span class="cit-pub-id-sep cit-pub-id-doi-sep">, </span><span class="cit-pub-id-scheme">doi: </span><span class="cit-pub-id cit-pub-id-doi">10.1002/joc.1276</span> (<span class="cit-pub-date">2005</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DInt.%2BJ.%2BClimatol%26rft.volume%253D25%26rft.spage%253D1965%26rft_id%253Dinfo%253Adoi%252F10.1002%252Fjoc.1276%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1002/joc.1276&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000234519700002&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><a class="rev-xref-ref" href="#xref-ref-43-1" title="View reference 43 in text" id="ref-43">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.01.12.903427v1.43" data-doi="10.1111/j.1600-0587.2009.06142.x"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Warren</span>,  <span class="cit-name-given-names">D. L.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Glor</span>,  <span class="cit-name-given-names">R. E.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Turelli</span>,  <span class="cit-name-given-names">M.</span></span> <span class="cit-article-title">ENMTools: a toolbox for comparative studies of environmental niche models</span>. <abbr class="cit-jnl-abbrev">Ecography</abbr> <span class="cit-vol">33</span>, <span class="cit-fpage">607</span>–<span class="cit-lpage">611</span><span class="cit-pub-id-sep cit-pub-id-doi-sep">, </span><span class="cit-pub-id-scheme">doi: </span><span class="cit-pub-id cit-pub-id-doi">10.1111/j.1600-0587.2009.06142.x</span> (<span class="cit-pub-date">2010</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEcography%26rft.volume%253D33%26rft.spage%253D607%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1600-0587.2009.06142.x%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1600-0587.2009.06142.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000280457500019&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><a class="rev-xref-ref" href="#xref-ref-44-1" title="View reference 44 in text" id="ref-44">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.01.12.903427v1.44" data-doi="10.1175/2011jcli4083.1"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Gent</span>,  <span class="cit-name-given-names">P. R.</span></span> <span class="cit-etal">et al.</span> <span class="cit-article-title">The Community Climate System Model Version 4</span>. <abbr class="cit-jnl-abbrev">J. Climate</abbr> <span class="cit-vol">24</span>, <span class="cit-fpage">4973</span>–<span class="cit-lpage">4991</span><span class="cit-pub-id-sep cit-pub-id-doi-sep">, </span><span class="cit-pub-id-scheme">doi: </span><span class="cit-pub-id cit-pub-id-doi">10.1175/2011jcli4083.1</span> (<span class="cit-pub-date">2011</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BClimate%26rft.volume%253D24%26rft.spage%253D4973%26rft_id%253Dinfo%253Adoi%252F10.1175%252F2011jcli4083.1%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1175/2011jcli4083.1&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><a class="rev-xref-ref" href="#xref-ref-45-1" title="View reference 45 in text" id="ref-45">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.01.12.903427v1.45" data-doi="10.5194/gmd-4-845-2011"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Watanabe</span>,  <span class="cit-name-given-names">S.</span></span> <span class="cit-etal">et al.</span> <span class="cit-article-title">MIROC-ESM 2010: model description and basic results of CMIP5-20c3m experiments</span>. <abbr class="cit-jnl-abbrev">Geosci. Model Dev</abbr>. <span class="cit-vol">4</span>, <span class="cit-fpage">845</span>–<span class="cit-lpage">872</span><span class="cit-pub-id-sep cit-pub-id-doi-sep">, </span><span class="cit-pub-id-scheme">doi: </span><span class="cit-pub-id cit-pub-id-doi">10.5194/gmd-4-845-2011</span> (<span class="cit-pub-date">2011</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGeosci.%2BModel%2BDev%26rft.volume%253D4%26rft.spage%253D845%26rft_id%253Dinfo%253Adoi%252F10.5194%252Fgmd-4-845-2011%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.5194/gmd-4-845-2011&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000298366300001&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><a class="rev-xref-ref" href="#xref-ref-46-1" title="View reference 46 in text" id="ref-46">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.01.12.903427v1.46" data-doi="10.1111/2041-210X.12200"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Brown</span>,  <span class="cit-name-given-names">J. L.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Anderson</span>,  <span class="cit-name-given-names">B.</span></span> <span class="cit-article-title">SDMtoolbox: a python-based GIS toolkit for landscape genetic, biogeographic and species distribution model analyses</span>. <abbr class="cit-jnl-abbrev">Methods Ecol. Evol</abbr>. <span class="cit-vol">5</span>, <span class="cit-fpage">694</span>–<span class="cit-lpage">700</span><span class="cit-pub-id-sep cit-pub-id-doi-sep">, </span><span class="cit-pub-id-scheme">doi: </span><span class="cit-pub-id cit-pub-id-doi">10.1111/2041-210x.12200</span> (<span class="cit-pub-date">2014</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMethods%2BEcol.%2BEvol%26rft.volume%253D5%26rft.spage%253D694%26rft_id%253Dinfo%253Adoi%252F10.1111%252F2041-210X.12200%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/2041-210X.12200&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><a class="rev-xref-ref" href="#xref-ref-47-1" title="View reference 47 in text" id="ref-47">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.01.12.903427v1.47" data-doi="10.1038/nrg3803"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Schlotterer</span>,  <span class="cit-name-given-names">C.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Tobler</span>,  <span class="cit-name-given-names">R.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Kofler</span>,  <span class="cit-name-given-names">R.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Nolte</span>,  <span class="cit-name-given-names">V.</span></span> <span class="cit-article-title">Sequencing pools of individuals - mining genome-wide polymorphism data without big funding</span>. <abbr class="cit-jnl-abbrev">Nat. Rev. Genet</abbr>. <span class="cit-vol">15</span>, <span class="cit-fpage">749</span>–<span class="cit-lpage">763</span><span class="cit-pub-id-sep cit-pub-id-doi-sep">, </span><span class="cit-pub-id-scheme">doi: </span><span class="cit-pub-id cit-pub-id-doi">10.1038/nrg3803</span> (<span class="cit-pub-date">2014</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat.%2BRev.%2BGenet%26rft.volume%253D15%26rft.spage%253D749%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnrg3803%26rft_id%253Dinfo%253Apmid%252F25246196%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nrg3803&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25246196&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F01%2F14%2F2020.01.12.903427.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><a class="rev-xref-ref" href="#xref-ref-48-1" title="View reference 48 in text" id="ref-48">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.01.12.903427v1.48" data-doi="10.1186/1471-2105-11-485"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Cox</span>,  <span class="cit-name-given-names">M. P.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Peterson</span>,  <span class="cit-name-given-names">D. A.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Biggs</span>,  <span class="cit-name-given-names">P. J.</span></span> <span class="cit-article-title">SolexaQA: At-a-glance quality assessment of Illumina second-generation sequencing data</span>. <abbr class="cit-jnl-abbrev">BMC Bioinformatics</abbr> <span class="cit-vol">11</span>, <span class="cit-fpage">485</span><span class="cit-pub-id-sep cit-pub-id-doi-sep">, </span><span class="cit-pub-id-scheme">doi: </span><span class="cit-pub-id cit-pub-id-doi">10.1186/1471-2105-11-485</span> (<span class="cit-pub-date">2010</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2Bbioinformatics%2B%255Belectronic%2Bresource%255D%26rft.stitle%253DBMC%2BBioinformatics%26rft.aulast%253DCox%26rft.auinit1%253DM.%2BP.%26rft.volume%253D11%26rft.spage%253D485%26rft.epage%253D485%26rft.atitle%253DSolexaQA%253A%2BAt-a-glance%2Bquality%2Bassessment%2Bof%2BIllumina%2Bsecond-generation%2Bsequencing%2Bdata.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2105-11-485%26rft_id%253Dinfo%253Apmid%252F20875133%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2105-11-485&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20875133&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F01%2F14%2F2020.01.12.903427.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><a class="rev-xref-ref" href="#xref-ref-49-1" title="View reference 49 in text" id="ref-49">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.01.12.903427v1.49" data-doi="10.14806/ej.17.1.200"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Martin</span>,  <span class="cit-name-given-names">M.</span></span> <span class="cit-article-title">Cutadapt removes adapter sequences from high-throughput sequencing reads</span>. <abbr class="cit-jnl-abbrev">EMBnet.journal</abbr> <span class="cit-vol">17</span>, <span class="cit-fpage">10</span>–<span class="cit-lpage">12</span><span class="cit-pub-id-sep cit-pub-id-doi-sep">, </span><span class="cit-pub-id-scheme">doi: </span><span class="cit-pub-id cit-pub-id-doi">10.14806/ej.17.1.200</span> (<span class="cit-pub-date">2011</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEMBnet.journal%26rft.volume%253D17%26rft.spage%253D10%26rft_id%253Dinfo%253Adoi%252F10.14806%252Fej.17.1.200%26rft_id%253Dinfo%253Apmid%252F25452271%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.14806/ej.17.1.200&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25452271&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F01%2F14%2F2020.01.12.903427.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><a class="rev-xref-ref" href="#xref-ref-50-1" title="View reference 50 in text" id="ref-50">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.01.12.903427v1.50" data-doi="10.1038/srep31586"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Wu</span>,  <span class="cit-name-given-names">W.</span></span> <span class="cit-etal">et al.</span> <span class="cit-article-title">Whole genome sequencing of a banana wild relative <em>Musa itinerans</em> provides insights into lineage-specific diversification of the <em>Musa</em> genus</span>. <abbr class="cit-jnl-abbrev">Sci. Rep</abbr>. <span class="cit-vol">6</span>, <span class="cit-fpage">31586</span><span class="cit-pub-id-sep cit-pub-id-doi-sep">, </span><span class="cit-pub-id-scheme">doi: </span><span class="cit-pub-id cit-pub-id-doi">10.1038/srep31586</span> (<span class="cit-pub-date">2016</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DSci.%2BRep%26rft.volume%253D6%26rft.spage%253D31586%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fsrep31586%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/srep31586&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><a class="rev-xref-ref" href="#xref-ref-51-1" title="View reference 51 in text" id="ref-51">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.01.12.903427v1.51" data-doi="10.1093/bioinformatics/btp324"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Li</span>,  <span class="cit-name-given-names">H.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Durbin</span>,  <span class="cit-name-given-names">R.</span></span> <span class="cit-article-title">Fast and accurate short read alignment with Burrows-Wheeler transform</span>. <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">25</span>, <span class="cit-fpage">1754</span>–<span class="cit-lpage">1760</span><span class="cit-pub-id-sep cit-pub-id-doi-sep">, </span><span class="cit-pub-id-scheme">doi: </span><span class="cit-pub-id cit-pub-id-doi">10.1093/bioinformatics/btp324</span> (<span class="cit-pub-date">2009</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtp324%26rft_id%253Dinfo%253Apmid%252F19451168%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/btp324&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19451168&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F01%2F14%2F2020.01.12.903427.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000267665900006&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><a class="rev-xref-ref" href="#xref-ref-52-1" title="View reference 52 in text" id="ref-52">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.01.12.903427v1.52" data-doi="10.1101/gr.107524.110"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">McKenna</span>,  <span class="cit-name-given-names">A.</span></span> <span class="cit-etal">et al.</span> <span class="cit-article-title">The Genome Analysis Toolkit: A MapReduce framework for analyzing next-generation DNA sequencing data</span>. <abbr class="cit-jnl-abbrev">Genome Res</abbr>. <span class="cit-vol">20</span>, <span class="cit-fpage">1297</span>–<span class="cit-lpage">1303</span><span class="cit-pub-id-sep cit-pub-id-doi-sep">, </span><span class="cit-pub-id-scheme">doi: </span><span class="cit-pub-id cit-pub-id-doi">10.1101/gr.107524.110</span> (<span class="cit-pub-date">2010</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BRes%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.107524.110%26rft_id%253Dinfo%253Apmid%252F20644199%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjk6IjIwLzkvMTI5NyI7czo0OiJhdG9tIjtzOjQ4OiIvYmlvcnhpdi9lYXJseS8yMDIwLzAxLzE0LzIwMjAuMDEuMTIuOTAzNDI3LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><a class="rev-xref-ref" href="#xref-ref-53-1" title="View reference 53 in text" id="ref-53">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.01.12.903427v1.53" data-doi="10.1186/s12870-018-1410-6"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Wu</span>,  <span class="cit-name-given-names">W.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Ng</span>,  <span class="cit-name-given-names">W. L.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Yang</span>,  <span class="cit-name-given-names">J. X.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Li</span>,  <span class="cit-name-given-names">W. M.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Ge</span>,  <span class="cit-name-given-names">X. J.</span></span> <span class="cit-article-title">High cryptic species diversity is revealed by genome-wide polymorphisms in a wild relative of banana, <em>Musa itinerans</em>, and implications for its conservation in subtropical China</span>. <abbr class="cit-jnl-abbrev">BMC Plant Biol</abbr>. <span class="cit-vol">18</span>, <span class="cit-fpage">194</span><span class="cit-pub-id-sep cit-pub-id-doi-sep">, </span><span class="cit-pub-id-scheme">doi: </span><span class="cit-pub-id cit-pub-id-doi">10.1186/s12870-018-1410-6</span> (<span class="cit-pub-date">2018</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2BPlant%2BBiol%26rft.volume%253D18%26rft.spage%253D194%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fs12870-018-1410-6%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/s12870-018-1410-6&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.biorxiv.org/content/10.1101/2020.01.12.903427v1.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-disqus-comment" >
  
      
  
  <div class="pane-content">
    <div id="disqus_thread"><noscript><p><a href="http://biorxivstage.disqus.com/?url=https%3A%2F%2Fwww.biorxiv.org%2Fcontent%2F10.1101%2F2020.01.12.903427v1" class="" data-icon-position="" data-hide-link-title="0">View the discussion thread.</a></p></noscript></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/10.1101/2020.01.12.903419v1" title="Reward-driven enhancements in motor control are robust to TMS manipulation" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous</span></a></span><span class="pager-next"><a href="/content/10.1101/569319v4" title="Neonatal morphometric similarity mapping for predicting brain age and characterizing neuroanatomic variation associated with preterm birth" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    Posted&nbsp;January 14, 2020.  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-biorxiv-art-tools" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-biorxiv_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/10.1101/2020.01.12.903427v1.full.pdf" target="_self" class="article-dl-pdf-link link-icon"><i class="icon-external-link-sign"></i> <span class="title">Download PDF</span></a>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-biorxiv-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="Email this Article" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-biorxiv_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/10.1101/2020.01.12.903427v1.full" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for your interest in spreading the word about bioRxiv.</p><p>NOTE: Your email address is requested solely to identify you as the sender of this article.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/10.1101/2020.01.12.903427v1" class="active" data-icon-position="" data-hide-link-title="0">The fate of standing variation and new mutation under climate change</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from bioRxiv
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would like to see this page from the bioRxiv website.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/1105568" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-M_Vf43kTlFhww8ubSGVbzRSxHDj1Lm9Zy_YEHEDp7YI" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-biorxiv-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-biorxiv_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="1105568" id="node1105568--2" data-pisa="biorxiv;2020.01.12.903427v1" data-pisa-master="biorxiv;2020.01.12.903427" data-seqnum="1105568" data-apath="/biorxiv/early/2020/01/14/2020.01.12.903427.atom"><div  class="highwire-cite highwire-cite-highwire-article highwire-citation-biorxiv-article-pap-list clearfix" >

      <div class="highwire-cite-title" >
      <div class="highwire-cite-title">The fate of standing variation and new mutation under climate change</div>    </div>
  
      <div  class="highwire-cite-authors" ><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0"><span class="nlm-given-names">Cheng-Yueh</span> <span class="nlm-surname">Lu</span></span>, <span class="highwire-citation-author" data-delta="1"><span class="nlm-given-names">Cheng-Ruei</span> <span class="nlm-surname">Lee</span></span></span></div>
  
      <div  class="highwire-cite-metadata" ><span  class="highwire-cite-metadata-journal highwire-cite-metadata">bioRxiv </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">2020.01.12.903427; </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata"><span class="doi_label">doi:</span> https://doi.org/10.1101/2020.01.12.903427 </span></div>
  
  
  </div>
</div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.biorxiv.org/content/10.1101/2020.01.12.903427v1" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.biorxiv.org/content/10.1101/2020.01.12.903427v1&amp;title=The%20fate%20of%20standing%20variation%20and%20new%20mutation%20under%20climate%20change" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.biorxiv.org/content/10.1101/2020.01.12.903427v1&amp;title=The%20fate%20of%20standing%20variation%20and%20new%20mutation%20under%20climate%20change" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.biorxiv.org/content/10.1101/2020.01.12.903427v1&amp;text=The%20fate%20of%20standing%20variation%20and%20new%20mutation%20under%20climate%20change" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.biorxiv.org/content/10.1101/2020.01.12.903427v1&amp;title=The%20fate%20of%20standing%20variation%20and%20new%20mutation%20under%20climate%20change" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.biorxiv.org/content/10.1101/2020.01.12.903427v1&amp;t=The%20fate%20of%20standing%20variation%20and%20new%20mutation%20under%20climate%20change" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.biorxiv.org/content/10.1101/2020.01.12.903427v1&amp;title=The%20fate%20of%20standing%20variation%20and%20new%20mutation%20under%20climate%20change" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.biorxiv.org/content/10.1101/2020.01.12.903427v1&amp;title=The%20fate%20of%20standing%20variation%20and%20new%20mutation%20under%20climate%20change" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-biorxiv-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title="Citation Tools"><i class="icon-globe"></i> <span class="title">Citation Tools</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-biorxiv_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="1105568" id="node1105568--3" data-pisa="biorxiv;2020.01.12.903427v1" data-pisa-master="biorxiv;2020.01.12.903427" data-seqnum="1105568" data-apath="/biorxiv/early/2020/01/14/2020.01.12.903427.atom"><div  class="highwire-cite highwire-cite-highwire-article highwire-citation-biorxiv-article-pap-list clearfix" >

      <div class="highwire-cite-title" >
      <div class="highwire-cite-title">The fate of standing variation and new mutation under climate change</div>    </div>
  
      <div  class="highwire-cite-authors" ><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0"><span class="nlm-given-names">Cheng-Yueh</span> <span class="nlm-surname">Lu</span></span>, <span class="highwire-citation-author" data-delta="1"><span class="nlm-given-names">Cheng-Ruei</span> <span class="nlm-surname">Lee</span></span></span></div>
  
      <div  class="highwire-cite-metadata" ><span  class="highwire-cite-metadata-journal highwire-cite-metadata">bioRxiv </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">2020.01.12.903427; </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata"><span class="doi_label">doi:</span> https://doi.org/10.1101/2020.01.12.903427 </span></div>
  
  
  </div>
</div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage&amp;rft.epage&amp;rft.atitle=The%20fate%20of%20standing%20variation%20and%20new%20mutation%20under%20climate%20change&amp;rft.volume&amp;rft.issue&amp;rft.date=2020-01-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=bioRxiv&amp;rft.au=Lu%2C+Cheng-Yueh&amp;rft.au=Lee%2C+Cheng-Ruei"></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/1105568/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/1105568/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/1105568/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/1105568/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/1105568/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/1105568/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/1105568/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/1105568/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/1105568/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/1105568/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/1105568/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/1105568/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><div class="item-list"><ul><li class="first"><a href="http://twitter.com/share?url=https%3A//www.biorxiv.org/content/10.1101/2020.01.12.903427v1&amp;count=horizontal&amp;via=&amp;text=The%20fate%20of%20standing%20variation%20and%20new%20mutation%20under%20climate%20change&amp;counturl=https%3A//www.biorxiv.org/content/10.1101/2020.01.12.903427v1" class="twitter-share-button service-links-twitter-widget" id="twitter_widget" title="Tweet This" rel="nofollow" data-icon-position="" data-hide-link-title="0"><span class="element-invisible">Tweet Widget</span></a></li><li><a href="http://www.facebook.com/plugins/like.php?href=https%3A//www.biorxiv.org/content/10.1101/2020.01.12.903427v1&amp;layout=button_count&amp;show_faces=false&amp;action=like&amp;colorscheme=light&amp;width=100&amp;height=21&amp;font=&amp;locale=" id="facebook_like" title="I Like it" class="service-links-facebook-like" rel="nofollow" data-icon-position="" data-hide-link-title="0"><span class="element-invisible">Facebook Like</span></a></li><li class="last"><a href="https://www.biorxiv.org/content/10.1101/2020.01.12.903427v1" id="google_plus_one" title="Plus it" class="service-links-google-plus-one" rel="nofollow" data-icon-position="" data-hide-link-title="0"><span class="element-invisible">Google Plus One</span></a></li></ul></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-collections" >
  
        <h2 class="pane-title">Subject Area</h2>
    
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-collections"><div class="highwire-list"><ul class="highwire-article-collection-term-list"><li class="first last odd"><span class="highwire-article-collection-term"><a href="/collection/evolutionary-biology" class="highlight" data-icon-position="" data-hide-link-title="0">Evolutionary Biology<i class="icon-caret-right"></i>
</a></span></li></ul></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-biorxiv-subject-collections block-style-col2" >
  
      
  
  <div class="pane-content">
    <div class="panel-flexible panels-flexible-8 clearfix" id="mini-panel-biorxiv_subject_collections">
<div class="panel-flexible-inside panels-flexible-8-inside">
<div class="panels-flexible-region panels-flexible-region-8-center panels-flexible-region-first panels-flexible-region-last">
  <div class="inside panels-flexible-region-inside panels-flexible-region-8-center-inside panels-flexible-region-inside-first panels-flexible-region-inside-last">
<div class="panel-pane pane-snippet" >
  
      
  
  <div class="pane-content">
    <div class="snippet biorxiv-subject-areas-table-title" id="biorxiv-subject-areas-table-title">
  
      
  <div class="snippet-content">
    <b>Subject Areas</b>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-snippet" >
  
      
  
  <div class="pane-content">
    <div class="snippet biorxiv-subject-areas-view-papers" id="biorxiv-subject-areas-view-papers">
  
      
  <div class="snippet-content">
    <a href="/content/early/recent"><strong>All Articles</strong></a>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-subject-collections" >
  
      
  
  <div class="pane-content">
    <ul id="collection" class="collection highwire-list-expand"><li class="outer collection depth-2  child first"><div class = "data-wrapper"><a href="/collection/animal-behavior-and-cognition" class="" data-icon-position="" data-hide-link-title="0">Animal Behavior and Cognition</a> <span class = "article-count">(1674)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/biochemistry" class="" data-icon-position="" data-hide-link-title="0">Biochemistry</a> <span class = "article-count">(2825)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/bioengineering" class="" data-icon-position="" data-hide-link-title="0">Bioengineering</a> <span class = "article-count">(1963)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/bioinformatics" class="" data-icon-position="" data-hide-link-title="0">Bioinformatics</a> <span class = "article-count">(10456)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/biophysics" class="" data-icon-position="" data-hide-link-title="0">Biophysics</a> <span class = "article-count">(4275)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/cancer-biology" class="" data-icon-position="" data-hide-link-title="0">Cancer Biology</a> <span class = "article-count">(3301)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/cell-biology" class="" data-icon-position="" data-hide-link-title="0">Cell Biology</a> <span class = "article-count">(4686)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/clinical-trials" class="" data-icon-position="" data-hide-link-title="0">Clinical Trials</a> <span class = "article-count">(136)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/developmental-biology" class="" data-icon-position="" data-hide-link-title="0">Developmental Biology</a> <span class = "article-count">(2911)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/ecology" class="" data-icon-position="" data-hide-link-title="0">Ecology</a> <span class = "article-count">(4575)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/epidemiology" class="" data-icon-position="" data-hide-link-title="0">Epidemiology</a> <span class = "article-count">(2041)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/evolutionary-biology" class="" data-icon-position="" data-hide-link-title="0">Evolutionary Biology</a> <span class = "article-count">(7337)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/genetics" class="" data-icon-position="" data-hide-link-title="0">Genetics</a> <span class = "article-count">(5574)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/genomics" class="" data-icon-position="" data-hide-link-title="0">Genomics</a> <span class = "article-count">(6938)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/immunology" class="" data-icon-position="" data-hide-link-title="0">Immunology</a> <span class = "article-count">(2463)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/microbiology" class="" data-icon-position="" data-hide-link-title="0">Microbiology</a> <span class = "article-count">(7696)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/molecular-biology" class="" data-icon-position="" data-hide-link-title="0">Molecular Biology</a> <span class = "article-count">(3078)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/neuroscience" class="" data-icon-position="" data-hide-link-title="0">Neuroscience</a> <span class = "article-count">(19031)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/paleontology" class="" data-icon-position="" data-hide-link-title="0">Paleontology</a> <span class = "article-count">(142)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/pathology" class="" data-icon-position="" data-hide-link-title="0">Pathology</a> <span class = "article-count">(481)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/pharmacology-and-toxicology" class="" data-icon-position="" data-hide-link-title="0">Pharmacology and Toxicology</a> <span class = "article-count">(796)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/physiology" class="" data-icon-position="" data-hide-link-title="0">Physiology</a> <span class = "article-count">(1188)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/plant-biology" class="" data-icon-position="" data-hide-link-title="0">Plant Biology</a> <span class = "article-count">(2805)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/scientific-communication-and-education" class="" data-icon-position="" data-hide-link-title="0">Scientific Communication and Education</a> <span class = "article-count">(699)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/synthetic-biology" class="" data-icon-position="" data-hide-link-title="0">Synthetic Biology</a> <span class = "article-count">(919)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/systems-biology" class="" data-icon-position="" data-hide-link-title="0">Systems Biology</a> <span class = "article-count">(2898)</span></div></li>
<li class="outer collection depth-2  child last"><div class = "data-wrapper"><a href="/collection/zoology" class="" data-icon-position="" data-hide-link-title="0">Zoology</a> <span class = "article-count">(473)</span></div></li>
</ul>  </div>

  
  </div>
  </div>
</div>
</div>
</div>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__VNH5GD8Zz7g6_hGCOZjVjdMndxxC6naiExSSJKX3k_A__kzmEdtbqCsJx5Z9yg2Qy0PptB__ufXm-TxUOl0KZzUw__dPhHATyREQrnntSIgxU8XNoYny4rjInxP7lLKSNxQyY.js"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__dPhHATyREQrnntSIgxU8XNoYny4rjInxP7lLKSNxQyY.js" defer="defer"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__N7ERJBYsOWyRnJgyoM125_Aiez2MOJGaUofG1JdWWBg__bLlcwhzZhT-IwoQ_dvm1_xPxU7E7gecFLC6Fyeqtv8s__dPhHATyREQrnntSIgxU8XNoYny4rjInxP7lLKSNxQyY.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
function euCookieComplianceLoadScripts() {}
//--><!]]>
</script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var eu_cookie_compliance_cookie_name = "";
//--><!]]>
</script>
<script type="text/javascript" src="//d33xdlntwy0kbs.cloudfront.net/cshl_custom.js"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__BmqjBnkz3MgYCAoc25s1lDRMEjLhC3mEPVonUFIHi08__Unwv5-ZIuHBfFwytsjEx1niBVJ7n1T4lPws7VrkHXM4__dPhHATyREQrnntSIgxU8XNoYny4rjInxP7lLKSNxQyY.js"></script>
  </body>
</html>
