<html xmlns="http://www.w3.org/1999/xhtml"> <head> <style type="text/css">
      	     		 
      	    a                          {background : #ffffff; }
      	    article                    {border-style : dotted; border-width : 2px; }
		 	
		 	div                        {background : #ffffcc;}
		 	div.abstract-title         {font-weight : bold ; font-size : 16pt;}
		 	div.ack                    {border-style : solid ; border-color : red; margin : 2em; }
		 	div.article-meta           {border-style : solid; border-width : 1 pt; margin 1em;}
		 	div.boxed-text             {margin : 5em; border-style : solid;}
		 	div.contrib-group          {margin : 1em; }
		 	div.fig                    {border-style : solid; border-width : 2px; margin : 2em; }
		 	div.funding                {font-weight : bold ; font-size : 16pt;}
		 	div.given-names            {font-style : italic;}
		 	div.intro                  {border-style : inset; margin : 5px;}
		 	div.introduction           {border-style : inset; margin : 5px;}
		 	div.journal-meta           {border-style : solid; border-width : 1 pt; margin 1em;}
		 	div.journal-title-group    {background : #ffeeee;}
		 	div.article-title          {font-weight : bold ; font-size : 18pt;}
		 	div.kwd                    {font-style : italic}
		 	div.meta-name              {font-weight : bold ; font-size : 16pt;}
		 	div.name                   {font-weight : bold;}
		 	div.alt-title              {font-style : italic; font-size : 12px;}
		 	
		 	div.sec                    {border : 2px; margin 5px; padding 2px;}
		 	div.title                  {font-family : courier; font-weight : bold;
		 	                            font-size : 16pt; margin : 5px;}
		 	
		 	div.materials_methods      {border-style : double; margin : 5px; }
		 	div.methods                {border-style : double; margin : 5px; }
		 	
		 	div.results                {border-style : solid; margin : 5px;}
		 	div.background             {border-style : dotted; margin : 5px;}
		 	
		 	div.discussion             {border-style : groove; margin : 5px;}
		 	div.conclusion             {border-style : ridge; margin : 5px;}
		 	div.conclusions             {border-style : ridge; margin : 5px;}
		 	div.supplementary-material {border-style : inset; margin : 5px;}
		 	div.abbreviations          {border-style : double; border-color : red; margin : 5px;}
		 	div.competinginterests     {border-style : double; border-color : blue; margin : 5px;}
		 	div.acknowledgements       {border-style : double; border-color : green; margin : 5px;}
		 	div.authors_contributions   {border-style : double; border-color : purple; margin : 5px;}
		 	
		 	div.publisher              {border-style : outset; margin : 5px;}
		 	div.fn-type-conflict       {background : #f88; }
		 	div.fn-type-con            {background : #ddf; }
		 	div.fn-type-other          {background : #ddd; }
		 	
		 	div.unknown                {background : #ffd;
									 	  border-style : solid;
									 	  border-width : 1px;
									 	  padding : 2 px;}
		 	  
      	    table                      {background : #ffffdd;}
		 	tr                         {background : #ddddff; padding : 1px;}
		 	
		 	span                       {background : #ffcccc;}
		 	
		 	span.citation-author       {font-family : helvetica; background : #ffeeee;}
		 	span.collab                {background : #ddffff; }
		 	span.comment               {font-family : courier; font-size : 6px; background : #ffaaff;}
		 	span.contrib               {background : #ffffff;}
		 	span.corresp               {background : #ddffdd; }
		 	span.doi                   {background : #ffffff;}
		 	span.email                 {font-family : courier; }
		 	span.etal                  {font-style : italic;}
		 	span.fpage                 {font-family : courier;}
		 	span.given-names           {background : #ffffff;}
		 	span.iso-abbrev            {background : #ffffff;}
		 	span.issn-epub             {background : #ffffff;}
		 	span.issn-ppub             {background : #ffffff;}
		 	span.journal-title         {background : #ffffff;}
		 	span.lpage                 {font-family : courier;}
		 	span.mixed-article-title   {font-style : italic ;}
		 	span.nlm-ta                {background : #ffffff;}
		 	span.pmc                   {background : #ffffff;}
		 	span.pmcid                 {background : #ffffff;}
		 	span.pmid                  {background : #ffffff;}
		 	span.publisher             {background : #ffffff;}
		 	span.publisher-id          {background : #ffffff;}
		 	span.publisher-name        {background : #ffffff;}
		 	span.source                {background : #ffffff;}
		 	span.subject               {background : #ffffff;}
		 	span.surname               {background : #ffffff;}
		 	span.volume                {font-family : courier; font-weight : bold;}
		 	span.year                  {font-family : courier ; font-style : italic;}
			</style> </head> <body> <div class="front" title="front"> <div class="journal-meta" tagx="journal-meta" title="journal-meta"><span class="nlm-ta" title="nlm-ta">New Phytol</span><span class="iso-abbrev" title="iso-abbrev">New Phytol</span><span class="doi" title="doi">10.1111/(ISSN)1469-8137</span><span class="publisher-id" title="publisher-id">NPH</span><div class="journal-title-group" tagx="journal-title-group" title="journal-title-group"><span class="journal-title" tagx="journal-title" title="journal-title">The New Phytologist</span></div><span class="issn-ppub" tagx="issn" title="issn-ppub">0028-646X</span><span class="issn-epub" tagx="issn" title="issn-epub">1469-8137</span><div class="publisher" tagx="publisher" title="publisher"><span class="publisher-name" tagx="publisher-name" title="publisher-name">John Wiley and Sons Inc.</span><span class="publisher-loc" tagx="publisher-loc" title="publisher-loc">Hoboken</span></div> </div> <div class="article-meta" tagx="article-meta" title="article-meta"><span class="doi" title="doi"> doi: <a href="https://dx.doi.org/10.1111/nph.14557">10.1111/nph.14557</a></span><span class="publisher-id" title="publisher-id">NPH14557</span><span class="other" title="other">2016-22838</span><div class="article-categories" title="article-categories">
: <span class="subject" title="subject">Full Paper</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Research</span>: <span class="subject" title="subject">Full Papers</span></div> </div> <div class="title-group" tagx="title-group" title="title-group"> <div class="article-title" title="article-title">
Finding a partner in the ocean: molecular and evolutionary bases of the response to sexual cues in a planktonic diatom</div> </div> <div class="contrib-group" title="contrib-group"><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Basu</span><span class="given-names" tagx="given-names" title="given-names">Swaraj</span></span><a href="#nph14557-aff-0001"><sup>1</sup></a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Patil</span><span class="given-names" tagx="given-names" title="given-names">Shrikant</span></span><a href="#nph14557-aff-0001"><sup>1</sup></a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Mapleson</span><span class="given-names" tagx="given-names" title="given-names">Daniel</span></span><a href="#nph14557-aff-0002"><sup>2</sup></a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Russo</span><span class="given-names" tagx="given-names" title="given-names">Monia Teresa</span></span><a href="#nph14557-aff-0001"><sup>1</sup></a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Vitale</span><span class="given-names" tagx="given-names" title="given-names">Laura</span></span><a href="#nph14557-aff-0001"><sup>1</sup></a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Fevola</span><span class="given-names" tagx="given-names" title="given-names">Cristina</span></span><a href="#nph14557-aff-0001"><sup>1</sup></a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Maumus</span><span class="given-names" tagx="given-names" title="given-names">Florian</span></span><a href="#nph14557-aff-0003"><sup>3</sup></a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Casotti</span><span class="given-names" tagx="given-names" title="given-names">Raffaella</span></span><a href="#nph14557-aff-0001"><sup>1</sup></a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Mock</span><span class="given-names" tagx="given-names" title="given-names">Thomas</span></span><a href="#nph14557-aff-0004"><sup>4</sup></a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Caccamo</span><span class="given-names" tagx="given-names" title="given-names">Mario</span></span><a href="#nph14557-aff-0002"><sup>2</sup></a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Montresor</span><span class="given-names" tagx="given-names" title="given-names">Marina</span></span><a href="#nph14557-aff-0001"><sup>1</sup></a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Sanges</span><span class="given-names" tagx="given-names" title="given-names">Remo</span></span><span class="address" title="address"><span class="email" tagx="email" title="email">remo.sanges@szn.it</span></span><a href="#nph14557-aff-0005"><sup>5</sup></a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ferrante</span><span class="given-names" tagx="given-names" title="given-names">Maria Immacolata</span></span><span class="address" title="address"><span class="email" tagx="email" title="email">mariella.ferrante@szn.it</span></span><a href="#nph14557-aff-0001"><sup>1</sup></a></span></div><span class="citation_author_institution" id="nph14557-aff-0001">[1], <span class="named-content" title="named-content">Integrative Marine Ecology</span><span class="institution" title="institution">Stazione Zoologica Anton Dohrn</span><span class="named-content" title="named-content">Villa Comunale 1</span><span class="named-content" title="named-content">Naples</span><span class="named-content" title="named-content">80121</span><span class="country" tagx="country" title="country">Italy</span></span><span class="citation_author_institution" id="nph14557-aff-0002">[2], <span class="named-content" title="named-content">Earlham Institute</span><span class="institution" title="institution">Norwich Research Park</span><span class="named-content" title="named-content">Norwich</span><span class="named-content" title="named-content">NR4 7UG</span><span class="country" tagx="country" title="country">UK</span></span><span class="citation_author_institution" id="nph14557-aff-0003">[3], <span class="named-content" title="named-content">URGI</span><span class="named-content" title="named-content">INRA</span><span class="institution" title="institution">Université Paris‐Saclay</span><span class="named-content" title="named-content">78026</span><span class="named-content" title="named-content">Versailles</span><span class="country" tagx="country" title="country">France</span></span><span class="citation_author_institution" id="nph14557-aff-0004">[4], <span class="named-content" title="named-content">School of Environmental Sciences</span><span class="named-content" title="named-content">University of East Anglia</span><span class="institution" title="institution">Norwich Research Park</span><span class="named-content" title="named-content">Norwich</span><span class="named-content" title="named-content">NR4 7TJ</span><span class="country" tagx="country" title="country">UK</span></span><span class="citation_author_institution" id="nph14557-aff-0005">[5], <span class="named-content" title="named-content">Biology and Evolution of Marine Organisms</span><span class="institution" title="institution">Stazione Zoologica Anton Dohrn</span><span class="named-content" title="named-content">Villa Comunale 1</span><span class="named-content" title="named-content">Naples</span><span class="named-content" title="named-content">80121</span><span class="country" tagx="country" title="country">Italy</span></span><div class="author-notes" title="author-notes"> <div class="corresp" title="corresp"><span class="label" tagx="label" title="label">*</span> Authors for correspondence:<br /><br /><i>Maria Immacolata Ferrante</i><br /><br /><i>Tel: +39 0815833268</i><br /><br /><i>Email: </i><span class="email" tagx="email" title="email">mariella.ferrante@szn.it</span><br /><br /><i>Remo Sanges</i><br /><br /><i>Tel: +39 0815833428</i><br /><br /><i>Email: </i><span class="email" tagx="email" title="email">remo.sanges@szn.it</span><br /><br /></div> </div><span class="pub-date-epub" title="pub-date-epub">epub: <span>2017-4-4</span></span><span class="pub-date-ppub" title="pub-date-ppub">ppub: <span>2017-7</span></span><span class="volume" tagx="volume" title="volume">215</span><span class="issue" tagx="issue" title="issue">1</span><span class="issue-id" tagx="issue-id" title="issue-id">10.1111/nph.2017.215.issue-1</span><span class="fpage" tagx="fpage" title="fpage">140</span><span class="lpage" tagx="lpage" title="lpage">156</span><span class="history" title="history"><span class="received" title="received">received: 2016-9-17</span><span class="accepted" title="accepted">accepted: 2017-2-25</span></span><div class="permissions"><span class="copyright" title="copyright">(C) , </span><span class="license" title="license"><span class="license-p" title="license-p">This is an open access article under the terms of the <a href="http://creativecommons.org/licenses/by/4.0/">Creative Commons Attribution</a> License, which permits use, distribution and reproduction in any medium, provided the original work is properly cited.</span></span></div><a class="self-uri" href="file:NPH-215-140.pdf" title="self-uri">file:NPH-215-140.pdf</a><div class="abstract" title="abstract"> <div class="abstract-title" title="abstract-title">
Abstract</div> <div class="title" tagx="title" title="title">
Summary</div> <p> </p><ul> <li> <p>Microalgae play a major role as primary producers in aquatic ecosystems. Cell signalling regulates their interactions with the environment and other organisms, yet this process in phytoplankton is poorly defined. Using the marine planktonic diatom <i>Pseudo‐nitzschia multistriata</i>, we investigated the cell response to cues released during sexual reproduction, an event that demands strong regulatory mechanisms and impacts on population dynamics.</p> </li> <li> <p>We sequenced the genome of <i>P. multistriata</i> and performed phylogenomic and transcriptomic analyses, which allowed the definition of gene gains and losses, horizontal gene transfers, conservation and evolutionary rate of sex‐related genes. We also identified a small number of conserved noncoding elements.</p> </li> <li> <p>Sexual reproduction impacted on cell cycle progression and induced an asymmetric response of the opposite mating types. G protein‐coupled receptors and cyclic guanosine monophosphate (<span class="styled-content" tagx="styled-content" title="styled-content">cGMP</span>) are implicated in the response to sexual cues, which overall entails a modulation of cell cycle, meiosis‐related and nutrient transporter genes, suggesting a fine control of nutrient uptake even under nutrient‐replete conditions.</p> </li> <li> <p>The controllable life cycle and the genome sequence of <i>P. multistriata</i> allow the reconstruction of changes occurring in diatoms in a key phase of their life cycle, providing hints on the evolution and putative function of their genes and empowering studies on sexual reproduction.</p> </li> </ul> <p /> </div> <div class="kwd-group"> <div class="kwd" title="kwd">
algae</div> <div class="kwd" title="kwd">
diatom</div> <div class="kwd" title="kwd">
genomics</div> <div class="kwd" title="kwd">
mating type</div> <div class="kwd" title="kwd">
phytoplankton</div> <div class="kwd" title="kwd"><i>Pseudo‐nitzschia multistriata</i></div> <div class="kwd" title="kwd">
sexual reproduction</div> <div class="kwd" title="kwd">
signal transduction</div> </div> <div class="funding-group" title="funding-group"> <div class="funding" title="funding">
Funding</div> <div class="award-group" title="award-group"> <div class="funding-source" title="funding-source">
Marie Curie FP7‐PEOPLE‐2011‐CIG</div><span class="award-id" tagx="award-id" title="award-id">293887</span></div> </div> <div class="funding-group" title="funding-group"> <div class="funding" title="funding">
Funding</div> <div class="award-group" title="award-group"> <div class="funding-source" title="funding-source">
RITMARE Italian Flagship Project</div> </div> </div> <div class="funding-group" title="funding-group"> <div class="funding" title="funding">
Funding</div> <div class="award-group" title="award-group"> <div class="funding-source" title="funding-source">
Office of Science of the US Department of Energy</div><span class="award-id" tagx="award-id" title="award-id">DE‐AC02‐05CH11231</span></div> </div> <div class="funding-group" title="funding-group"> <div class="funding" title="funding">
Funding</div> <div class="award-group" title="award-group"> <div class="funding-source" title="funding-source">
SZN Open University PhD fellowship</div> </div> </div> <div class="funding-group" title="funding-group"> <div class="funding" title="funding">
Funding</div> <div class="award-group" title="award-group"> <div class="funding-source" title="funding-source">
Stazione Zoologica Anton Dohrn</div> </div> </div> <div class="funding-group" title="funding-group"> <div class="funding" title="funding">
Funding</div> <div class="award-group" title="award-group"> <div class="funding-source" title="funding-source">
NBIP Computing infrastructure for Science (CiS)</div> </div> </div> <div class="counts" title="counts"><span class="fig-count" title="fig-count">fig-count: </span><span class="table-count" title="table-count">table-count: </span><span class="page-count" title="page-count">page-count: </span><span class="word-count" title="word-count">word-count: </span></div> <div class="custom-meta-group" title="custom-meta-group"> <meta name="source-schema-version-number" value="2.0" /> <meta name="component-id" value="nph14557" /> <meta name="cover-date" value="July 2017" /> <meta name="details-of-publishers-convertor" value="Converter:WILEY_ML3GV2_TO_NLMPMC version:5.1.2 mode:remove_FC converted:27.06.2017" /> </div> </div> </div> <div class="body" title="body"> <div class="introduction" title="sec"> <div class="title" tagx="title" title="title">
Introduction</div> <p>Phytoplankton feature prominently in aquatic ecosystems, showing striking morphological and functional diversity and accounting for one‐half of the Earth's primary productivity (Falkowski &amp;amp; Knoll, <a href="#nph14557-bib-0022">2011</a>). Diatoms are a major component of phytoplankton with over 100 000 species (Mann &amp;amp; Vanormelingen, <a href="#nph14557-bib-0043">2013</a>) and contribute substantially to primary production and major biogeochemical cycles (Armbrust, <a href="#nph14557-bib-0003">2009</a>). A high rate of DNA turnover, horizontal gene transfer (HGT) from bacteria and endosymbiotic events are responsible for the chimeric nature of diatom genomes, which have probably contributed to the heterogeneity of their physiological and ecological traits (Bowler <i>et al</i>., <a href="#nph14557-bib-0011">2010</a>).</p> <p>The first assembled genomes of a centric (<i>Thalassiosira pseudonana</i>, Armbrust <i>et al</i>., <a href="#nph14557-bib-0004">2004</a>) and a pennate (<i>Phaeodactylum tricornutum</i>, Bowler <i>et al</i>., <a href="#nph14557-bib-0010">2008</a>) diatom were small in size (27–32 Mb) with 10 000–14 000 genes. They contained only one‐half of the genes with an annotated function, and <i>c</i>. 35% of the genes were reported to be species specific. Further, <i>c</i>. 5% of <i>P. tricornutum</i> genes were predicted to be acquired by HGT from bacteria. These genomes contributed towards an understanding of the genes and pathways involved in nutrient assimilation and metabolism of diatoms. To improve our understanding of the evolution and adaptation of this highly diverse group of organisms, additional diatom genomes were sequenced, such as those of the open‐ocean centric diatom <i>Thalassiosira oceanica</i> (Lommer <i>et al</i>., <a href="#nph14557-bib-0041">2012</a>), the oleaginous <i>Fistulifera solaris</i> (Tanaka <i>et al</i>., <a href="#nph14557-bib-0073">2015</a>) and the polar diatom <i>Fragilariopsis cylindrus</i> (Mock <i>et al</i>., <a href="#nph14557-bib-0047">2017</a>), instrumental for the study of iron physiology, lipid metabolism and adaptation to cold, respectively.</p> <p>The dynamics of planktonic communities are strongly dependent on the life cycle traits of the individual species. Diatoms have a unique life cycle characterized by progressive cell size reduction in the population, imposed by a rigid silica wall. A few exceptions apart, sexual reproduction is an obligate phase in diatom life cycles, important not only to generate genetic diversity, but also to escape the miniaturization process, thus allowing the persistence of populations by restoring the original cell size (Montresor <i>et al</i>., <a href="#nph14557-bib-0049">2016</a>). It has been proposed that some of the unique features of the diatom genomes may reflect the unusual characteristics of diatom life cycles (Bowler <i>et al</i>., <a href="#nph14557-bib-0010">2008</a>). However, the most widely used diatom models are putatively asexual and this has hampered research on the molecular and genomic underpinning of sexual reproduction.</p> <p>The marine planktonic pennate diatom <i>Pseudo‐nitzschia multistriata</i> has a typical, controllable size reduction–restitution life cycle in which cells of opposite mating type (MT+ and MT−) produce gametes when they are below the size threshold for sex (D'Alelio <i>et al</i>., <a href="#nph14557-bib-0015">2009</a>). On gamete conjugation, an expandable zygote is produced, within which the cell of maximum size is formed (Fig. <a href="#nph14557-fig-0001">1</a>). Sexual reproduction requires a threshold cell concentration to start, suggesting that chemical signalling is needed to allow the induction of the sexual phase in this species (Scalco <i>et al</i>., <a href="#nph14557-bib-0066">2014</a>). Diffusible chemical cues have been shown to be responsible for a multi‐step sexualization process, and two sex pheromones have been characterized for the benthic diatom <i>Seminavis robusta</i> (Gillard <i>et al</i>., <a href="#nph14557-bib-0027">2013</a>; Moeys <i>et al</i>., <a href="#nph14557-bib-0048">2016</a>). The availability of transcriptomic data for the latter species and for <i>P. multistriata</i>, coupled with a comparative genomic approach, led us to the identification of the diatom genes involved in meiosis (Patil <i>et al</i>., <a href="#nph14557-bib-0053">2015</a>). Although the meiosis toolkit is well conserved, the sexual cues and the response mechanisms might have diverged substantially between benthic species, which can glide on the substrate to follow attraction cues, and planktonic species, which are suspended in the water column. Indeed, it is a mystery how pennate planktonic diatoms find their partner and the significance of pheromone signalling remains completely unexplored.</p> <div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/"><span class="label" tagx="label" title="label">Figure 1</span></a></div>   <p>Schematic drawing of the life cycle of <i>Pseudo‐nitzschia multistriata</i>. Starting clockwise from the bottom portion of the cycle, the vegetative phase is characterized by progressive cell size reduction of the population imposed by the rigid silica wall, made up of two unequal thecae. During this process, the cells reach the sexualization size threshold (<span class="styled-content" tagx="styled-content" title="styled-content">SST</span>) and can either keep decreasing in size until death, or undergo sexual reproduction and escape the miniaturization process, producing large cells. In the heterothallic <i>P. multistriata</i>, sex can occur only if strains of opposite mating type come into contact. The perception of chemical cues deriving from the mating partner (0–12 h) brings cells of opposite mating type to pair (12–24 h). The formation of gametes (24–36 h) takes place following meiosis. Conjugation of the haploid gametes (24–48 h) produces two expandable zygotes (36–48 h) that develop into auxospores (36–72 h). Within each auxospore, an initial cell of maximum size is synthesized (60–72 h), restoring the vegetative phase of the cycle. The time interval for each stage is indicated. Representative microscopic images of the different stages are shown outside the circle; bar, 10 μm. Thick black arrows mark the sampling time points for the experiments described in this work. <span class="styled-content" tagx="styled-content" title="styled-content">MT</span>, mating type.</p>   </div> <p><i>Pseudo‐nitzschia multistriata</i> is able to produce the neurotoxin domoic acid, a molecule that can contaminate seafood and cause a syndrome called amnesic shellfish poisoning (Trainer <i>et al</i>., <a href="#nph14557-bib-0074">2012</a>). The genome sequence of this toxic species and insights into the mechanisms underlying its life cycle regulation will facilitate investigations on the dynamics of toxic <i>Pseudo‐nitzschia</i> blooms.</p> <p>We chose <i>P. multistriata</i> as a model to study sexual reproduction. We report the assembly and annotation of its genome, which was first exploited to reveal unexplored features of diatom genomes, such as conserved noncoding elements (CNEs) with a potential regulatory function, and transposable element activity. We also assessed the turnover of gene families amongst Stramenopiles through an in‐depth phylogenomic approach to better identify conserved and unique features of <i>P. multistriata</i>, and to provide novel information on HGT. Furthermore, the availability of the <i>P. multistriata</i> genome, coupled with a transcriptomic approach, led us to dissect the signalling pathways employed in the early phases of sexual reproduction. Several mating type (MT)‐specific gene expression changes were observed, highlighting the involvement of different pathways in the response to putative pheromones, whereas other changes were common to both MTs, including growth arrest and the modulation of cell cycle genes and nutrient transporters.</p> </div> <div class="materialsandmethods" title="sec"> <div class="title" tagx="title" title="title">
Materials and Methods</div> <div class="strains" title="sec"> <div class="title" tagx="title" title="title">
Strains</div> <p>A <i>Pseudo‐nitzschia multistriata</i> (Takano) Takano pedigree was built starting from two strains collected in 2009 (Fig. <a href="#nph14557-fig-0002">2</a>). Strain B856, chosen for genome sequencing, was made axenic by treatment with antibiotics (Supporting Information Methods <a href="#nph14557-sup-0001">S1</a>). RNA‐seq reads used to produce the <i>de novo</i> transcriptome were obtained from strains Sy373, Sy379, B856 and B857 (Fig. <a href="#nph14557-fig-0002">2</a>e). For the differential expression studies, strains B856, B857 and B938 were used with B936, B937 and B939, isolated from the LTER (Long TERm) station MareChiara (40°48.5′N, 14°15′E). Cultures were kept at a temperature of 18°C, irradiance of 80 μmol photons m<sup>−1</sup> s<sup>−1</sup> and in a 12 h : 12 h light : dark photoperiod.</p> <div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/"><span class="label" tagx="label" title="label">Figure 2</span></a></div>   <p>Main features of <i>Pseudo‐nitzschia multistriata</i> and its genome. (a, b) Microscopic images showing three cells in a chain in a normal culture with bacteria, in bright field and fluorescence, respectively, and (c, d) four cells in an axenic culture without bacteria. DAPI (4′,6‐diamidino‐2‐phenylindole) stains <span class="styled-content" tagx="styled-content" title="styled-content">DNA</span> in cell nuclei (arrowheads) and bacterial nucleoids (thin arrows). Bars, 10 μm. (e) <i>Pseudo‐nitzschia multistriata</i> pedigree showing four generations. Strain B856 was used to produce the genome sequence. (f) Estimation of species divergence based on amino acid identity of coding genes. The <i>x</i>‐axis represents the average percentage identity of <span class="styled-content" tagx="styled-content" title="styled-content">BLAST</span>p hits with maximum scores for the first species against the second. The <i>y</i>‐axis represents the cumulative proportion of the genes showing a given percentage identity. (g) Distribution of percentage identity for noncoding elements conserved between <i>Pseudo‐nitzschia</i> species (light blue dots), among <i>P. multistriata</i>,<i> Pseudo‐nitzschia multiseries</i> and <i>Fragilariopsis cylindrus</i> (red dots) and in other combinations. The <i>x</i>‐axis represents the identified conserved noncoding elements, stacked for best visualization of their distribution of conservation.</p>   </div> </div> <div class="genomesequencingandassembly" title="sec"> <div class="title" tagx="title" title="title">
Genome sequencing and assembly</div> <p>B856 cells were collected onto 1.2‐μm pore‐size membrane filters (RAWP04700 Millipore) and DNA was extracted with phenol–chloroform as described in Sabatino <i>et al</i>. (<a href="#nph14557-bib-0062">2015</a>). The <i>P. multistriata</i> genome was assembled from a total of 172 million 101‐bp overlapping paired‐end reads with <i>c</i>. 175‐bp inserts, 117 million 100‐bp paired‐end reads with <i>c</i>. 450‐bp inserts, 72 million <i>c</i>. 68‐bp (after trimming) mate pair reads with <i>c</i>. 1.2‐kb inserts and 5.4 million <i>c</i>. 156‐bp (after trimming) mate pair reads with <i>c</i>. 4.5‐kb inserts. Mate pair libraries were processed by N <div class="sc_UNKNOWN">
ext</div>C <div class="sc_UNKNOWN">
lip</div> to remove adapters. Depending on the library, the genome size was estimated to be between 71 and 82 Mb using SGA preqc. Reads from libraries exceeding 100× coverage were randomly subsampled to 100× and then assembled into scaffolds by A <div class="sc_UNKNOWN">
llpaths</div>‐LG (Gnerre <i>et al</i>., <a href="#nph14557-bib-0028">2011</a>) via R <div class="sc_UNKNOWN">
ampart</div> (Mapleson <i>et al</i>., <a href="#nph14557-bib-0044">2015</a>). The completeness of the genome was evaluated using C <div class="sc_UNKNOWN">
egma</div> with the set of 248 core eukaryotic genes (CEGs) (Parra <i>et al</i>., <a href="#nph14557-bib-0052">2007</a>). The assembly (accession number PRJEB9419) can be visualized at <a href="http://apollo.tgac.ac.uk/Pseudo-nitzschia_multistriata_V1_4_browser/sequences">http://apollo.tgac.ac.uk/Pseudo-nitzschia_multistriata_V1_4_browser/sequences</a> (username and password: pnitzschia).</p> </div> <div class="genepredictionandannotation" title="sec"> <div class="title" tagx="title" title="title">
Gene prediction and annotation</div> <p>Protein‐coding genes were predicted using a workflow incorporating RNA‐seq reads, homologous proteins from <i>P. tricornutum</i>,<i> T. pseudonana</i> and a <i>de novo P. multistriata</i> transcriptome assembly. RNA‐seq reads were combined from four different libraries (samples: B856, libraries HCUO and HCUH; B857, libraries HCUN and HATT; available at <a href="http://genomeportal.jgi.doe.gov/pages/dynamicOrganismDownload.jsf?organism=PsenittraphaseII">http://genomeportal.jgi.doe.gov/pages/dynamicOrganismDownload.jsf?organism=PsenittraphaseII</a>) and assembled <i>de novo</i>. The transcripts generated were used as training data for A <div class="sc_UNKNOWN">
ugustus</div> (Stanke <i>et al</i>., <a href="#nph14557-bib-0071">2006</a>). The model built on the training data was applied to the entire repeat masked assembly, together with external support from homologous proteins aligned using E <div class="sc_UNKNOWN">
xonerate</div> (Slater &amp;amp; Birney, <a href="#nph14557-bib-0068">2005</a>). The predicted gene models were annotated using A <div class="sc_UNKNOWN">
nnocript</div> (Musacchia <i>et al</i>., <a href="#nph14557-bib-0050">2015</a>).</p> <p>Repeats were identified using R <div class="sc_UNKNOWN">
epet</div>. The TE <div class="sc_UNKNOWN">
denovo</div> pipeline (Flutre <i>et al</i>., <a href="#nph14557-bib-0024">2011</a>) was used to build a library of consensus sequences of repetitive elements in the genome assembly. The TE <div class="sc_UNKNOWN">
annot</div> pipeline (Quesneville <i>et al</i>., <a href="#nph14557-bib-0058">2005</a>) was employed with default settings using the sequences from the filtered combined library as probes to perform genome annotation.</p> <p>Full‐length complete long terminal repeats (LTRs) were identified using LTRH <div class="sc_UNKNOWN">
arvest</div> and LTRD <div class="sc_UNKNOWN">
igest</div> (Gremme <i>et al</i>., <a href="#nph14557-bib-0030">2013</a>). The relative age of LTR insertion was estimated using the method proposed in previous studies (Kimura, <a href="#nph14557-bib-0037">1980</a>).</p> <p>The statistics for the genomic features in Table <a href="#nph14557-tbl-0001">1</a> were extracted from the GFF files using shell scripts and the BEDT <div class="sc_UNKNOWN">
ools</div> package (Quinlan, <a href="#nph14557-bib-0059">2014</a>). The genome size, N50 value and GC content were taken from the respective publications (Armbrust <i>et al</i>., <a href="#nph14557-bib-0004">2004</a>; Bowler <i>et al</i>., <a href="#nph14557-bib-0010">2008</a>; Cock <i>et al</i>., <a href="#nph14557-bib-0014">2010</a>; Lévesque <i>et al</i>., <a href="#nph14557-bib-0040">2010</a>; Lommer <i>et al</i>., <a href="#nph14557-bib-0041">2012</a>; Tanaka <i>et al</i>., <a href="#nph14557-bib-0073">2015</a>; Mock <i>et al</i>., <a href="#nph14557-bib-0047">2017</a>).</p> <div class="table-wrap_UNKNOWN" id="nph14557-tbl-0001" orientation="portrait" position="float"><span class="label" tagx="label" title="label">Table 1</span> <p>Genome assembly and gene annotation statistics for selected Stramenopiles</p>  <table frame="hsides" rules="groups"> <colgroup><col style="border-right:solid 1px #000000" span="1" /></colgroup> <colgroup><col style="border-right:solid 1px #000000" span="1" /></colgroup> <colgroup><col style="border-right:solid 1px #000000" span="1" /></colgroup> <colgroup><col style="border-right:solid 1px #000000" span="1" /></colgroup> <colgroup><col style="border-right:solid 1px #000000" span="1" /></colgroup> <colgroup><col style="border-right:solid 1px #000000" span="1" /></colgroup> <colgroup><col style="border-right:solid 1px #000000" span="1" /></colgroup> <colgroup><col style="border-right:solid 1px #000000" span="1" /></colgroup> <colgroup><col style="border-right:solid 1px #000000" span="1" /></colgroup> <colgroup><col style="border-right:solid 1px #000000" span="1" /></colgroup> <thead valign="top"> <tr style="border-bottom:solid 1px #000000"> <th align="left" valign="top" rowspan="1" colspan="1">Organism</th> <th align="left" valign="top" rowspan="1" colspan="1"><i>Pseudo‐nitzschia multistriata</i></th> <th align="left" valign="top" rowspan="1" colspan="1"><i>Pseudo‐nitzschia multiseries</i></th> <th align="left" valign="top" rowspan="1" colspan="1"><i>Fragilariopsis cylindrus</i></th> <th align="left" valign="top" rowspan="1" colspan="1"><i>Fistulifera solaris</i></th> <th align="left" valign="top" rowspan="1" colspan="1"><i>Phaeodactylum tricornutum</i></th> <th align="left" valign="top" rowspan="1" colspan="1"><i>Thalassiosira pseudonana</i></th> <th align="left" valign="top" rowspan="1" colspan="1"><i>Thalassiosira oceanica</i></th> <th align="left" valign="top" rowspan="1" colspan="1"><i>Pythium ultimum</i></th> <th align="left" valign="top" rowspan="1" colspan="1"><i>Ectocarpus siliculosus</i></th> </tr> </thead> <tbody> <tr> <td align="left" rowspan="1" colspan="1">Genome size (Mb)</td> <td align="left" rowspan="1" colspan="1">59</td> <td align="left" rowspan="1" colspan="1">219</td> <td align="left" rowspan="1" colspan="1">61</td> <td align="left" rowspan="1" colspan="1">50</td> <td align="left" rowspan="1" colspan="1">27</td> <td align="left" rowspan="1" colspan="1">32</td> <td align="left" rowspan="1" colspan="1">92</td> <td align="left" rowspan="1" colspan="1">45</td> <td align="left" rowspan="1" colspan="1">196</td> </tr> <tr> <td align="left" rowspan="1" colspan="1">N50 (Mb)</td> <td align="left" rowspan="1" colspan="1">0.138</td> <td align="left" rowspan="1" colspan="1">0.147</td> <td align="left" rowspan="1" colspan="1">1.3</td> <td align="left" rowspan="1" colspan="1">na</td> <td align="left" rowspan="1" colspan="1">0.945</td> <td align="left" rowspan="1" colspan="1">1.9</td> <td align="left" rowspan="1" colspan="1">0.04</td> <td align="left" rowspan="1" colspan="1">0.837</td> <td align="left" rowspan="1" colspan="1">0.504</td> </tr> <tr> <td align="left" rowspan="1" colspan="1">G + C (%)</td> <td align="left" rowspan="1" colspan="1">46.3</td> <td align="left" rowspan="1" colspan="1">40.72</td> <td align="left" rowspan="1" colspan="1">39.8</td> <td align="left" rowspan="1" colspan="1">46.1</td> <td align="left" rowspan="1" colspan="1">48.8</td> <td align="left" rowspan="1" colspan="1">47</td> <td align="left" rowspan="1" colspan="1">53.3</td> <td align="left" rowspan="1" colspan="1">52</td> <td align="left" rowspan="1" colspan="1">53.6</td> </tr> <tr> <td align="left" rowspan="1" colspan="1">Repeat (%)</td> <td align="left" rowspan="1" colspan="1">25</td> <td align="left" rowspan="1" colspan="1">73</td> <td align="left" rowspan="1" colspan="1">38</td> <td align="left" rowspan="1" colspan="1">16</td> <td align="left" rowspan="1" colspan="1">6.4</td> <td align="left" rowspan="1" colspan="1">2</td> <td align="left" rowspan="1" colspan="1">na</td> <td align="left" rowspan="1" colspan="1">7</td> <td align="left" rowspan="1" colspan="1">22.7</td> </tr> <tr> <td align="left" rowspan="1" colspan="1">Gene count</td> <td align="left" rowspan="1" colspan="1">12 008</td> <td align="left" rowspan="1" colspan="1">19 703</td> <td align="left" rowspan="1" colspan="1">21 066</td> <td align="left" rowspan="1" colspan="1">20 621</td> <td align="left" rowspan="1" colspan="1">10 402</td> <td align="left" rowspan="1" colspan="1">11 776</td> <td align="left" rowspan="1" colspan="1">34 500</td> <td align="left" rowspan="1" colspan="1">15 291</td> <td align="left" rowspan="1" colspan="1">16 256</td> </tr> <tr> <td align="left" rowspan="1" colspan="1">Av. gene length (bp)</td> <td align="left" rowspan="1" colspan="1">2205</td> <td align="left" rowspan="1" colspan="1">1522</td> <td align="left" rowspan="1" colspan="1">1575</td> <td align="left" rowspan="1" colspan="1">na</td> <td align="left" rowspan="1" colspan="1">1511</td> <td align="left" rowspan="1" colspan="1">1553</td> <td align="left" rowspan="1" colspan="1">1256</td> <td align="left" rowspan="1" colspan="1">1312</td> <td align="left" rowspan="1" colspan="1">1526</td> </tr> <tr> <td align="left" rowspan="1" colspan="1">Av. exon length (bp)</td> <td align="left" rowspan="1" colspan="1">1164</td> <td align="left" rowspan="1" colspan="1">509</td> <td align="left" rowspan="1" colspan="1">625</td> <td align="left" rowspan="1" colspan="1">na</td> <td align="left" rowspan="1" colspan="1">842</td> <td align="left" rowspan="1" colspan="1">612</td> <td align="left" rowspan="1" colspan="1">464</td> <td align="left" rowspan="1" colspan="1">501</td> <td align="left" rowspan="1" colspan="1">242</td> </tr> <tr> <td align="left" rowspan="1" colspan="1">Av. intron length (bp)</td> <td align="left" rowspan="1" colspan="1">341</td> <td align="left" rowspan="1" colspan="1">229</td> <td align="left" rowspan="1" colspan="1">245</td> <td align="left" rowspan="1" colspan="1">na</td> <td align="left" rowspan="1" colspan="1">135</td> <td align="left" rowspan="1" colspan="1">124</td> <td align="left" rowspan="1" colspan="1">149</td> <td align="left" rowspan="1" colspan="1">116</td> <td align="left" rowspan="1" colspan="1">715</td> </tr> <tr> <td align="left" rowspan="1" colspan="1">Av. exon number per gene</td> <td align="left" rowspan="1" colspan="1">1.87</td> <td align="left" rowspan="1" colspan="1">2.38</td> <td align="left" rowspan="1" colspan="1">2.08</td> <td align="left" rowspan="1" colspan="1">na</td> <td align="left" rowspan="1" colspan="1">1.79</td> <td align="left" rowspan="1" colspan="1">2.49</td> <td align="left" rowspan="1" colspan="1">2.3</td> <td align="left" rowspan="1" colspan="1">2.6</td> <td align="left" rowspan="1" colspan="1">8</td> </tr> </tbody> </table> <div class="table-wrap-foot_UNKNOWN"> <div class="fn-type-" title=""> <p>na, data not available.</p> </div> </div> <div class="permissions"><span class="copyright" title="copyright">(C) <span class="copyright-holder" tagx="copyright-holder" title="copyright-holder">John Wiley &amp;amp; Sons, Ltd</span>, </span></div> </div> </div> <div class="identificationofcnes" title="sec"> <div class="title" tagx="title" title="title">
Identification of CNEs</div> <p>The public genomes of sequenced diatoms were aligned pairwise against the reference <i>P. multistriata</i> genome with L <div class="sc_UNKNOWN">
astz</div>. Utilities from the University of California Santa Cruz (UCSC) genome browser source code tree (Speir <i>et al</i>., <a href="#nph14557-bib-0069">2016</a>) were used to generate NET alignments from the raw pairwise alignments.</p> <p>The pairwise NET alignments in MAF format were combined into a single diatom NET alignment file using the roast binary from the M <div class="sc_UNKNOWN">
ultiz</div> package (Blanchette <i>et al</i>., <a href="#nph14557-bib-0007">2004</a>) with <i>P. multistriata</i> as reference. Custom P <div class="sc_UNKNOWN">
erl</div> scripts were used to scan the diatom NET alignment to identify conserved intergenic blocks (window, 20 bp; step, 10 bp) which do not overlap gene/expressed sequence tags in the species conserved. Searches for transcription factor binding sites were performed using J <div class="sc_UNKNOWN">
aspar</div> 2014 (Mathelier <i>et al</i>., <a href="#nph14557-bib-0045">2014</a>).</p> </div> <div class="expansionofgenefamiliesinp. multistriata" title="sec"> <div class="title" tagx="title" title="title">
Expansion of gene families in <i>P. multistriata</i></div> <p>Proteomes of Stramenopiles were compared against hidden Markov models (HMMs) of protein families classified in the SUPERFAMILY database (Wilson <i>et al</i>., <a href="#nph14557-bib-0079">2009</a>). The comparison was performed using Perl scripts provided by the SUPERFAMILY database (<a href="http://supfam.cs.bris.ac.uk/SUPERFAMILY/howto_use_models.html">http://supfam.cs.bris.ac.uk/SUPERFAMILY/howto_use_models.html</a>) and the hmmscan binary from HMMER3 (Eddy, <a href="#nph14557-bib-0019">1995</a>). For each SUPERFAMILY present in <i>P. multistriata</i>, a Z‐score was calculated using the following formula (no. of SUPERFAMILY genes in <i>P. multistriata</i> – mean no. of SUPERFAMILY genes in all proteomes)/SD of SUPERFAMILY genes in all proteomes.</p> </div> <div class="identificationofgenefamiliesbyclusteringofproteinsequences" title="sec"> <div class="title" tagx="title" title="title">
Identification of gene families by clustering of protein sequences</div> <p>An All vs All B <div class="sc_UNKNOWN">
last</div>p search was performed on the combined F <div class="sc_UNKNOWN">
asta</div> file of proteomes from bacteria, archaea and 50 eukaryotes. The results of the B <div class="sc_UNKNOWN">
last</div>p search were provided to the orthAgogue software (Ekseth <i>et al</i>., <a href="#nph14557-bib-0020">2014</a>) for the estimation of homology between the protein sequences. The ‘abc’ format output from <div class="sc_UNKNOWN">
orth</div>A <div class="sc_UNKNOWN">
gogue</div> was given to the MCL software (Enright <i>et al</i>., <a href="#nph14557-bib-0021">2002</a>) for clustering of the proteins into homologous groups.</p> </div> <div class="estimationofgenefamilygainsandlossesinstramenopiles" title="sec"> <div class="title" tagx="title" title="title">
Estimation of gene family gains and losses in Stramenopiles</div> <p>Clusters containing only one‐to‐one orthologues of each stramenopile species (85 clusters, considering 13 species mentioned in the ‘Expansion of gene families in <i>P. multistriata</i>’ subsection) were chosen to generate the species tree for Stramenopiles using M <div class="sc_UNKNOWN">
afft</div> (Katoh &amp;amp; Standley, <a href="#nph14557-bib-0036">2013</a>).</p> <p>The alignments were concatenated and trimmed with trimAl. ProtTest (Darriba <i>et al</i>., <a href="#nph14557-bib-0016">2011</a>) was then run on the trimmed concatenated alignment to determine the best amino acid substitution matrix to generate a phylogenetic tree based on the Bayesian Information Criterion score. The identified model was employed to generate the phylogenetic tree using a maximum likelihood and a Bayesian approach employing RA <div class="sc_UNKNOWN">
x</div>ML and M <div class="sc_UNKNOWN">
r</div>B <div class="sc_UNKNOWN">
ayes</div> (Stamatakis, <a href="#nph14557-bib-0070">2006</a>). In both approaches, <i>Blastocystis hominis</i> was used as outgroup. Protein clusters with at least one member from any stramenopile species were identified to obtain 28 927 clusters. For each stramenopile species, a binary code was established stating the presence or absence of the species in each cluster. The binary file, together with the maximum likelihood tree, was subjected to D <div class="sc_UNKNOWN">
ollo</div> parsimony analysis using the P <div class="sc_UNKNOWN">
hylip</div> package (Felsenstein, <a href="#nph14557-bib-0023">1989</a>). The topologies of the maximum likelihood and Bayesian trees were compared with treedist from the P <div class="sc_UNKNOWN">
hylip</div> software, indicating an identical topology.</p> </div> <div class="identificationofgenesacquiredfromredalgaeandbyhgtfrombacteriainp. multistriata" title="sec"> <div class="title" tagx="title" title="title">
Identification of genes acquired from red algae and by HGT from bacteria in <i>P. multistriata</i></div> <p>Identification of HGT events in <i>P. multistriata</i> was performed with the following steps: identification of protein clusters containing at least one <i>P. multistriata</i> protein; building of a multiple alignment for each cluster using M <div class="sc_UNKNOWN">
afft</div> (Katoh &amp;amp; Standley, <a href="#nph14557-bib-0036">2013</a>); trimming of columns with ≥ 95% gaps in the alignment generated using <div class="sc_UNKNOWN">
trim</div>A <div class="sc_UNKNOWN">
l</div> (Capella‐Gutiérrez <i>et al</i>., <a href="#nph14557-bib-0012">2009</a>); generation of a phylogenetic tree using F <div class="sc_UNKNOWN">
ast</div>T <div class="sc_UNKNOWN">
ree</div>.</p> <p>Phylogenetic trees for each cluster were parsed to identify genes of potential bacterial origin using the following criteria: identification of a clade represented in the majority by bacteria, archaea and diatoms (≥ 90%) without members from metazoa, plantae or fungi; bootstrap cut off at the clade of interest ≥ 0.5 or if the average bootstrap value for the tree is ≥ 0.5 (if one of the bootstrap values is ≤ 0.5, the tree is still retained (if other filters are passed) as a candidate with medium confidence); to add further stringency to the analysis, at least five bacterial members must be present in the clade of interest (10 when <i>P. multistriata</i> is the only eukaryote in the clade) to avoid false positives as a result of misplacement of a single protein within the clade of another taxon, which can be caused by issues such as long branch attraction.</p> </div> <div class="co‐cultureexperiments" title="sec"> <div class="title" tagx="title" title="title">
Co‐culture experiments</div> <p>Three independent co‐culture experiments were performed, two for RNA‐seq (MT+B856xMT‐B939 and MT+B938xMT‐B857) and one for quantitative PCR (MT+B937xMT‐B936). A bipartite glass apparatus (Duran flasks; VWR, Dresden, Germany) (Paul <i>et al</i>., <a href="#nph14557-bib-0054">2012</a>) was used for the co‐culture of strains of opposite MTs. A 0.22‐μm pore size hydrophilic polyvinylidene fluoride membrane (Durapore, Millipore) was placed in between the bottles to keep the cells separate. Control parental strains were grown in monoculture. The cell concentration was 80 000 cells ml<sup>−1</sup> for each strain. The cells were grown in f/2 medium (Guillard, <a href="#nph14557-bib-0031">1975</a>). A 36‐h dark incubation was employed to synchronize the cultures. Samples were collected 2 and 6 h after the start of the experiment. Fifty‐millilitre samples were centrifuged, resuspended in cold methanol and stored at −20°C. They were resuspended in Tris‐EDTA buffer, treated with RNase I (300 μg ml<sup>−1</sup>) for 45 min and stained with SYBR Green (1 : 10 000 dilution of SYBR<sup>®</sup> Green I – 10 000× concentrate, Invitrogen) for 15 min. Cell cycle synchronization was verified with a FACSCalibur flow cytometer (Becton Dickinson BioSciences Inc., Franklin Lakes, NJ, USA) with standard filters and a 488‐nm Ar laser. SYBR Green fluorescence (DNA) was collected through 530 ± 30‐nm optical filters in order to assess the percentage of cells in the different cell cycle phases. Control cells always presented a bimodal distribution of SYBR Green fluorescence, allowing the assessment of cell cycle blockage (one peak) in treated samples. Sample acquisition was realized using BD C <div class="sc_UNKNOWN">
ell</div>Q <div class="sc_UNKNOWN">
uest</div> software, and the relative proportions of cells in the different phases of the cell cycle were assessed using ModFit software (Verity Inc., Palo Alto, CA, USA).</p> </div> <div class="rnaextractionandsequencing" title="sec"> <div class="title" tagx="title" title="title">
RNA extraction and sequencing</div> <p>Samples were collected on 1.2‐μm pore size membrane filters (RAWP04700 Millipore) and extracted with Trizol™ (Invitrogen) according to the manufacturer's instructions; the gDNA contamination was removed by DNase I treatment (Qiagen), followed by purification using an RNeasy Plant Mini Kit (Qiagen). RNA quantity was determined using a Qubit<sup>®</sup> 2.0 Fluorometer (Life Technologies, Thermofisher, Waltham, MA, USA) and integrity using a Bioanalyzer (2100 Bioanalyzer Instruments, Agilent Technologies, Santa Clara, CA, USA).</p> <p>Libraries were prepared using a Beckman Biomek FX and an Illumina<sup>®</sup> TruSeq<sup>®</sup> Stranded Total RNA Sample Preparation kit, with poly‐A selection and starting with 500 ng of total RNA. Samples were sequenced on an Illumina HiSeq2000 producing single‐end 50‐bp reads. Library preparation and sequencing were performed at the Genecore Facility of the European Molecular Biology Laboratory (EMBL), Germany.</p> </div> <div class="rna‐seqfiltering,mappinganddifferentialexpressionanalysis" title="sec"> <div class="title" tagx="title" title="title">
RNA‐seq filtering, mapping and differential expression analysis</div> <p>The raw sequencing reads were processed with T <div class="sc_UNKNOWN">
rimmomatic</div> (Bolger <i>et al</i>., <a href="#nph14557-bib-0009">2014</a>) to trim low‐quality bases and adapters and to filter reads with low quality and smaller than 36 bases. The S <div class="sc_UNKNOWN">
tar</div> aligner (Dobin <i>et al</i>., <a href="#nph14557-bib-0018">2013</a>) was used to map the filtered reads onto the <i>P. multistriata</i> genome. The A <div class="sc_UNKNOWN">
ugustus</div> gene models were associated with the mapped reads from each sample to generate raw counts for each gene as a measure of their expression level. E <div class="sc_UNKNOWN">
dge</div>R (Robinson <i>et al</i>., <a href="#nph14557-bib-0061">2010</a>) was used to obtain the differentially expressed genes. In brief, generalized linear models were used to estimate dispersion considering multiple factors (MT, control/sexualized and species strain), whereas a more classical negative binomial distribution was used to compare only the control and sexualized stages independently in each MT.</p> <p>For quantitative PCR validation, total RNA was extracted from samples collected at 6 h. One microgram of total RNA was reverse transcribed using a QuantiTect<sup>®</sup> Reverse Transcription Kit (Qiagen). Nineteen genes were selected (Table <a href="#nph14557-sup-0003">S1</a>). <i>TUB‐A</i> (Adelfi <i>et al</i>., <a href="#nph14557-bib-0001">2014</a>) was used as reference. Real‐time PCR amplification and analyses were performed as described in Patil <i>et al</i>. (<a href="#nph14557-bib-0053">2015</a>).</p> </div> <div class="identificationofhomologousgenesandka : ksanalysis" title="sec"> <div class="title" tagx="title" title="title">
Identification of homologous genes and Ka : Ks analysis</div> <p>The analysed data included 12 152 and 19 703 coding DNA sequences (CDSs) of <i>P. multistriata</i> and <i>Pseudo‐nitzschia multiseries</i> (Psemu1, downloaded from the Joint Genome Institute (JGI)), respectively. As a first step, a reciprocal best B <div class="sc_UNKNOWN">
last</div> hit (RBH) was used to identify <i>P. multistriata</i> and <i>P. multiseries</i> orthologues. Only alignments covering at least 30% of <i>P. multistriata</i> sequences were retained. The analysis identified 7128 reciprocal best B <div class="sc_UNKNOWN">
last</div> hits. Next, each pair of sequences was aligned with P <div class="sc_UNKNOWN">
rank</div> (Löytynoja, <a href="#nph14557-bib-0042">2014</a>) using the empirical codon model, and the alignments were refined using <div class="sc_UNKNOWN">
trim</div>A <div class="sc_UNKNOWN">
l</div> (Capella‐Gutiérrez <i>et al</i>., <a href="#nph14557-bib-0012">2009</a>). Of the 7128 alignments, 6066 were suitable for Ka : Ks calculation. Ka : Ks calculation was performed with KaKs_Calculator (Wang <i>et al</i>., <a href="#nph14557-bib-0076">2010</a>); the model for the calculation was chosen for each alignment using the corrected akaike information criterion (AICc) model selection method.</p> <p>An extended version of the methods can be found in Methods <a href="#nph14557-sup-0001">S1</a>.</p> </div> </div> <div class="resultsanddiscussion" title="sec"> <div class="title" tagx="title" title="title">
Results and discussion</div> <div class="thep. multistriatagenomesequenceandfirstidentificationofcnesindiatoms" title="sec"> <div class="title" tagx="title" title="title">
The <i>P. multistriata</i> genome sequence and first identification of CNEs in diatoms</div> <p>To sequence the genome of <i>P. multistriata</i>, we used a strain derived from the cross of two siblings, grown under axenic conditions (Fig. <a href="#nph14557-fig-0002">2</a>a–e). The sequencing and assembly yielded a genome of 59 Mb composed of 1099 scaffolds with an N50 of 139 kb. Estimated heterozygosity was 0.18% and the distribution of allele frequencies peaked at <i>c</i>. 0.5, indicating a diploid clonal strain. A total of 99.5% of variable sites presented two alleles; only <i>c</i>. 500 of <i>c</i>. 110 000 variable sites showed more than two alleles, mainly associated with repeats and noncoding regions. A total of 12 008 genes were predicted on the assembled scaffolds. The regions comprising coding genes accounted for 50% of the genome, where <i>c</i>. 80% of genes (9653 genes) were assigned a U <div class="sc_UNKNOWN">
niprot</div> ID and an additional 214 genes were exclusively annotated for the presence of a protein domain. Estimation of genome completion by C <div class="sc_UNKNOWN">
egma</div> identified 221 (89.11%) of CEGs as complete and an additional seven CEGs (3%) as partial, indicating a high‐quality genome assembly and gene build. The statistics and features of the genome assembly and gene prediction for <i>P. multistriata</i> and selected Stramenopiles are summarized in Table <a href="#nph14557-tbl-0001">1</a> and Fig. <a href="#nph14557-sup-0001">S1</a>.</p> <p>Sequence conservation at the amino acid level can be a potential indicator of species divergence; hence, we used the sequence homology between species with known evolutionary history to estimate the divergence within diatom genomes (Fig. <a href="#nph14557-fig-0002">2</a>f). Consistent with its known phylogenetic relationships (Kooistra <i>et al</i>., <a href="#nph14557-bib-0038">2007</a>), <i>P. multistriata</i> shows maximum amino acid identity with the congeneric species <i>P. multiseries</i>, followed by the phylogenetically close <i>F. cylindrus</i> and then by the more distant <i>P. tricornutum</i>. The group of raphid pennates to which the four species belong is thought to have evolved <i>c</i>. 60 million yr ago (Ma) (Kooistra <i>et al</i>., <a href="#nph14557-bib-0038">2007</a>). Yet, the divergence of these diatom pairs is comparable with that between eukaryotic pairs known to have separated earlier (<i>Plasmodium falciparum</i> and <i>P. vivax</i>, reported to have separated <i>c</i>. 90–100 Ma (Perkins &amp;amp; Schall, <a href="#nph14557-bib-0055">2002</a>); <i>Arabidopsis thaliana</i> and <i>Physcomitrella patens</i>, with flowering plants reported to have diverged from the bryopsids <i>c</i>. 400–450 Ma (Rensing <i>et al</i>., <a href="#nph14557-bib-0060">2008</a>)), confirming the rapid evolutionary rates in diatoms (Bowler <i>et al</i>., <a href="#nph14557-bib-0010">2008</a>).</p> <p>In comparison with coding genes, the noncoding part of diatom genomes remains vastly unexplored, with no precise information on noncoding regions that might act as regulatory elements, as reported in animals, plants and unicellular eukaryotes (Vavouri <i>et al</i>., <a href="#nph14557-bib-0075">2007</a>; Piganeau <i>et al</i>., <a href="#nph14557-bib-0056">2009</a>; Haudry <i>et al</i>., <a href="#nph14557-bib-0033">2013</a>). Here, we take a first step towards the identification and classification of CNEs in diatoms using a comparative genomic approach centred on <i>P. multistriata</i>. CNEs play a role in the regulation of gene expression, often being part of promoters or enhancers (Woolfe <i>et al</i>., <a href="#nph14557-bib-0081">2005</a>). We identified a core set of <i>c</i>. 1500 CNEs in the genome of <i>P. multistriata</i> (mean length, 110 bp; mean identity, 73%; Table <a href="#nph14557-sup-0003">S2</a>) when compared with other diatom genomes. As expected, the majority of the predicted CNEs (93%, 1462 CNEs) were conserved exclusively between the <i>Pseudo‐nitzschia</i> species (Fig. <a href="#nph14557-fig-0002">2</a>g). A smaller subset of <i>c</i>. 50 CNEs was conserved between <i>Pseudo‐nitzschia</i> species and <i>F. cylindrus</i> (26), between <i>P. multistriata</i> and <i>P. tricornutum</i> (15), and between <i>P. multistriata</i> and <i>T. pseudonana</i> (10, Fig. <a href="#nph14557-fig-0002">2</a>g), suggesting functional constraints leading to noncoding conservation over large evolutionary distances. The predicted CNEs showed a significant enrichment to be located near transcription start sites (TSSs) (Student's <i>t</i>‐test, <i>P</i> = 0.0034; Fig. <a href="#nph14557-sup-0001">S2</a>a), indicating that they are probably involved in transcriptional regulation. Genes associated with the gene ontology (GO) molecular function terms ‘<i>signal transducer activity</i>’ and ‘<i>sequence‐specific DNA binding transcription factor activity</i>’ were enriched in loci containing CNEs (Fisher test, <i>P</i> ≤ 0.05), further supporting the functionality of the CNE. In addition, a significant enrichment of transcription factor binding sites of major transcription factor families was observed in CNE sequences compared with that observed in random sequences of similar size (Fig. <a href="#nph14557-sup-0001">S2</a>b–d; Table <a href="#nph14557-sup-0004">S3</a>). Thus, the proximity to genes related to the regulation of transcriptional control, together with the binding site propensity for transcription factors, corroborate previous reports which indicate that CNEs are often involved in the regulation of transcription (Inada <i>et al</i>., <a href="#nph14557-bib-0035">2003</a>; Sanges <i>et al</i>., <a href="#nph14557-bib-0063">2013</a>), and provide evidence in support of their functionality in diatoms.</p> <p>Approximately 25% of the <i>P. multistriata</i> genome comprises repetitive elements, 8% of which are in genic regions and 92% of which are in intergenic regions. The repetitive element coverage is significantly higher than that of other diatoms (6.4% in <i>P. tricornutum</i> and 1.9% in <i>T. pseudonana</i>). Of the known classes of repetitive elements, LTR retrotransposons were the most abundant group of annotated repeats (6%) (Fig. <a href="#nph14557-sup-0001">S3</a>a–c), similar to previous reports in other diatoms (Maumus <i>et al</i>., <a href="#nph14557-bib-0046">2009</a>), as well as other Stramenopiles (Cock <i>et al</i>., <a href="#nph14557-bib-0014">2010</a>; Ye <i>et al</i>., <a href="#nph14557-bib-0084">2015</a>), yeast (Bleykasten‐Grosshans &amp;amp; Neuvéglise, <a href="#nph14557-bib-0008">2011</a>) and some flowering plants (Bennetzen, <a href="#nph14557-bib-0006">2005</a>). To investigate the role of LTRs in shaping the genomic structure of <i>P. multistriata</i>, a <i>de novo</i> search specific for complete LTR elements was performed in the genomes of five diatoms and that of <i>Oryza sativa</i>. Intact retroelements have a built‐in molecular clock useful for estimating their insertion times, based on sister LTR divergence. <i>Pseudo‐nitzschia</i> species have, on average, older insertions of LTR elements (45% insertions in the last 0.5 Mya) with respect to <i>F. cylindrus</i>,<i> P. tricornutum</i> and <i>T. pseudonana</i> (80%, 65% and 61% insertions in the last 0.5 Mya, respectively) (Table <a href="#nph14557-sup-0002">S4</a>), indicating an earlier expansion of LTR retroelements in the <i>Pseudo‐nitzschia</i> genus. However, LTR elements of the Copia lineage also showed a recent burst of activity, suggesting that they might still be active in generating genetic variability in <i>Pseudo‐nitzschia</i>, as shown in <i>P. tricornutum</i> (Maumus <i>et al</i>., <a href="#nph14557-bib-0046">2009</a>) (Fig. <a href="#nph14557-sup-0001">S3</a>d).</p> </div> <div class="high‐resolutionphylogenomicanalysesdefinegenefamilyexpansions,genegainsandhgtindiatoms" title="sec"> <div class="title" tagx="title" title="title">
High‐resolution phylogenomic analyses define gene family expansions, gene gains and HGT in diatoms</div> <p>Lineage‐specific gene duplications, losses and pseudogenization, together with genome rearrangements and horizontal transfer of genes between species, have paved the way for the evolution of diversity (Koonin, <a href="#nph14557-bib-0039">2010</a>).</p> <p>In order to identify gene family expansions in diatoms, we compared the proteomes of 13 Stramenopiles (including five diatoms) against the collection of gene families from the SUPERFAMILY database. In total, 11 families showed expansion within the diatom lineage (Fig. <a href="#nph14557-fig-0003">3</a>; Table <a href="#nph14557-sup-0005">S5</a>); none was <i>P. multistriata</i> specific. We confirmed an expansion of a gene family (<i>pseudouridine synthase</i>) (Fig. <a href="#nph14557-fig-0003">3</a>a) reported to be expanded in the <i>P. tricornutum</i> genome (Bowler <i>et al</i>., <a href="#nph14557-bib-0010">2008</a>) and found specific expansion events within the order Bacillariales (Fig. <a href="#nph14557-fig-0003">3</a>a).</p> <div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/"><span class="label" tagx="label" title="label">Figure 3</span></a></div>   <p>Evolution of gene families in diatoms. (a) Expansion of gene families within diatoms. Each column represents a stramenopile species and each row represents a given gene family showing expansion within diatoms as compared with other Stramenopiles. Names of diatom species are given in red, whereas names of other Stramenopiles are given in blue. The colour intensity and size of the circles are proportional to the number of genes falling under the given gene family. (b) A species tree of Stramenopiles derived using a maximum likelihood approach, built using 85 genes showing one‐to‐one orthology among the selected species. The selected genes include genes with a wide range of functions. Branch lengths are drawn to scale. At each branch point, the number of gene family gains and losses is indicated in green and brown, respectively. The number of orphans present in each organism is shown in blue. (c) Phylogenetic tree for a cluster containing proteins annotated with an uncharacterized cystatin‐like domain, conserved in bacteria. The tree topology depicts a potential horizontal gene transfer event which led to the introduction of the gene within diatoms. The regions coloured red, blue and green represent bacteria, diatoms and other Stramenopiles, respectively. Species codes used in the tree: ecsi, <i>Ectocarpus siliculosus</i>; naga, <i>Nannochloropsis gaditana</i>; symi, <i>Symbiodinium minutum</i>; psmu, <i>Pseudo‐nitzschia multistriata</i>; psmus, <i>Pseudo‐nitzschia multiseries</i>; frcy, <i>Fragilariopsis cylindrus</i>; phtr, <i>Phaeodactylum tricornutum</i>; thps, <i>Thalassiosira pseudonana</i>; chre, <i>Chlamydomonas reinhardtii</i>; chva, <i>Chlorella variabilis</i>; BODB suffix is used for all bacterial species. For the correspondence between protein IDs used in this tree and GenBank IDs, see Supporting Information Methods <a href="#nph14557-sup-0001">S1</a>.</p>   </div> <p>In addition to gene family expansions, gene family gains and losses also contribute to the evolution and diversification of species. A study on the <i>Ectocarpus</i> genome showed large‐scale gene gains and losses within Stramenopiles, where lineages giving rise to multicellularity were reported to show a high rate of evolution of new gene families (Cock <i>et al</i>., <a href="#nph14557-bib-0014">2010</a>). Until a few years ago, the limited availability of sequenced genomes resulted in a lack of resolution to identify gene gain/loss events during the divergence of Stramenopiles. Taking advantage of the latest sequenced genomes, we were able to use <i>c</i>. 2 million protein sequences from organisms spanning the tree of life (50 eukaryotes, 1000 bacteria, 170 archaea; Table <a href="#nph14557-sup-0006">S6</a>) to infer, at an unprecedented resolution, gene gains/losses in diatoms. Phylogenetic clustering analysis generated <i>c</i>. 240 000 clusters of putative homologous proteins, 8113 of which contained 9122 <i>P. multistriata</i> proteins. Approximately 26% (2886 singlets + 241 genes in 109 clusters) of the <i>P. multistriata</i> proteome was predicted to be orphan (<i>P. multistriata</i>‐specific) (Fig. <a href="#nph14557-sup-0001">S4</a>), which is slightly less than that reported for other diatoms (<i>P. tricornutum</i>, 35%, Bowler <i>et al</i>., <a href="#nph14557-bib-0010">2008</a>; <i>T. pseudonana</i>, 29%, Armbrust <i>et al</i>., <a href="#nph14557-bib-0004">2004</a>) and similar to estimates deriving from transcriptomic data (Di Dato <i>et al</i>., <a href="#nph14557-bib-0017">2015</a>). Based on these data, we built a comprehensive species tree for 13 Stramenopiles by concatenating the alignments of 85 clusters of one‐to‐one orthologues conserved across the 13 species. The species tree topologies generated independently by maximum likelihood and Bayesian inference were identical and supported by bootstrap values &amp;gt; 90 at all branch points (Fig. <a href="#nph14557-fig-0003">3</a>b). The tree topology is congruent with a previous report (Yang <i>et al</i>., <a href="#nph14557-bib-0082">2012</a>), except for <i>Aureococcus anophagefferens</i> and diatoms forming a monophyletic group separated from the other Stramenopiles (<i>Ectocarpus siliculosus</i> and <i>Nannochloropsis gaditana</i>). Interestingly, a similar topology has been obtained recently using chloroplast genomes to infer phylogeny (Ševčíková <i>et al</i>., <a href="#nph14557-bib-0067">2015</a>), suggesting, in accordance with our results, a comparatively more recent split of pelagophytes and diatoms than previously estimated (Gobler <i>et al</i>., <a href="#nph14557-bib-0029">2011</a>). We then performed a Dollo parsimony analysis considering the presence/absence profiles of the identified protein clusters within the produced stramenopile species tree. Only those clusters containing at least one stramenopile member were considered (<i>c</i>. 27 000 clusters). We observed large‐scale gene family gain and loss events (Fig. <a href="#nph14557-fig-0003">3</a>b), suggesting a high order of genetic diversity among the species compared. A large number of orphan gene families were predicted for each species, confirming extensive species‐specific gains and/or rapid gene divergence as a major feature of stramenopile genomes. We then performed a GO term enrichment analysis on the subset of gene families gained at particular branch points (Figs <a href="#nph14557-fig-0003">3</a>b, <a href="#nph14557-sup-0005">S5</a>). Although gene families significantly gained by autotrophic Stramenopiles (lower clade in Fig. <a href="#nph14557-fig-0003">3</a>b) were mostly related to photosynthesis (Fig. <a href="#nph14557-sup-0001">S5</a>a), gene families associated with terms such as ‘<i>mitotic cell cycle spindle assembly check point</i>’, ‘<i>synaptonemal complex assembly</i>’ and ‘<i>G protein‐coupled receptor signalling pathway</i>’ were enriched in <i>Pseudo‐nitzschia</i> species and <i>F. cylindrus</i>, suggesting that potential novel mechanisms evolved in these diatoms to regulate cell division and, possibly, sexual reproduction (Fig. <a href="#nph14557-sup-0001">S5</a>b).</p> <p>The clusters of homologous genes produced were then used for the identification of putative HGT events from bacteria to diatoms. A total of 32% of the <i>P. multistriata</i> genes were placed in clusters containing putative orthologues from bacteria, red algae, plants, fungi and metazoans. Among these, genes falling exclusively within clusters dominated by bacterial genes might reflect ancient HGT events from bacteria to diatoms. We identified 392 <i>P. multistriata</i> proteins showing homology exclusively with bacteria (Fig. <a href="#nph14557-sup-0001">S6</a>). To refine these results, we developed a stringent classification method which, starting from the clusters of orthologous proteins, generated &amp;gt; 9000 phylogenetic trees containing at least one <i>P. multistriata</i> protein and, based on the topology of each tree, predicted 252 genes of bacterial origin specifically within diatoms. This is &amp;lt; 50% of the number of genes reported to be of bacterial origin in <i>P. tricornutum</i> (587 genes) (Bowler <i>et al</i>., <a href="#nph14557-bib-0010">2008</a>) and more than a previous estimate at lower resolution (Lommer <i>et al</i>., <a href="#nph14557-bib-0041">2012</a>). Repeating the analysis considering HGT events within Stramenopiles and the SAR supergroup (Stramenopiles, Alveolates and Rhizaria), we predicted 353 and 438 <i>P. multistriata</i> genes of potential bacterial origin, respectively. We conclude that the detailed taxonomic resolution introduced in our study allowed for the filtering of candidate genes previously considered to be of bacterial origin via HGT because of a lack of data from related species at the time of analysis. The 252 genes predicted to be of bacterial origin are proposed as a conservative set of genes introduced in the diatom lineage through HGT (Table <a href="#nph14557-sup-0007">S7</a>). They are smaller in size than average (<i>t</i>‐test; <i>P</i> value for gene length, 1e‐04; <i>P</i> value for exon length, 1.4e‐05), with no significant difference in the number of exons/gene (Mann–Whitney test, <i>P</i> = 0.48) and length of introns (<i>t</i>‐test, <i>P</i> = 0.24) (Fig. <a href="#nph14557-sup-0001">S7</a>). These genes are enriched for GO terms involved in processes such as ‘<i>energy metabolism</i>’, ‘<i>oxidative stress response</i>’ and ‘<i>substrate transport</i>’ (Fig. <a href="#nph14557-sup-0001">S8</a>; <i>P</i> &amp;lt; 0.05). In support of our results, the ‘<i>quinone oxidoreductase</i>’ (PSNMU‐V1.4_AUG‐EV‐PASAV3_0025570.1) was already known to be derived from HGT in diatoms (Nosenko &amp;amp; Bhattacharya, <a href="#nph14557-bib-0051">2007</a>). In addition, a significant difference (<i>t</i>‐test, <i>P</i> = 0.014) in the GC content for HGT genes compared with all <i>P. multistriata</i> genes supports their foreign origin (Garcia‐Vallvé <i>et al</i>., <a href="#nph14557-bib-0026">2000</a>) (Fig. <a href="#nph14557-sup-0001">S9</a>). Twenty‐four HGT events are specific to the <i>Pseudo‐nitzschia</i> genus; an example is shown in Fig. <a href="#nph14557-fig-0003">3</a>(c).</p> <p>Apart from bacteria, 123 genes in <i>P. multistriata</i> were classified to be of red algal origin (Table <a href="#nph14557-sup-0002">S8</a>), consistent with the notion that diatom progenitors originated from an ancient secondary endosymbiosis event involving a red alga and a heterotrophic eukaryote (Bowler <i>et al</i>., <a href="#nph14557-bib-0011">2010</a>).</p> </div> <div class="globalgeneexpressionchangesattheonsetofthesexualphasehighlightastrongerresponseinmt−cells" title="sec"> <div class="title" tagx="title" title="title">
Global gene expression changes at the onset of the sexual phase highlight a stronger response in MT− cells</div> <p>We exploited the controllable life cycle of <i>P. multistriata</i> to investigate the changes occurring in diatom cells in a key phase of their life cycle (Fig. <a href="#nph14557-fig-0004">4</a>). To study gene expression changes induced by the perception of chemical cues deriving from the mating partner, each of two <i>P. multistriata</i> strains was placed in one compartment of an apparatus that allowed free exchange of the medium, but not physical contact between the cells (Fig. <a href="#nph14557-fig-0004">4</a>a). Sampling times were 2 h (T1) and 6 h (T2) after co‐culture, the time at which the two MTs are sensing each other (black arrows in Fig. <a href="#nph14557-fig-0001">1</a>; Fig. <a href="#nph14557-sup-0001">S10</a>; Table <a href="#nph14557-sup-0002">S9</a>). Interestingly, although control strains in an isolated monoculture continued to progress through the cell cycle, cells of both MTs in the experimental set‐up arrested their cell cycle in the G1 phase (Fig. <a href="#nph14557-fig-0004">4</a>b). Previous observations over a longer period of time (14 d, Scalco <i>et al</i>., <a href="#nph14557-bib-0066">2014</a>) have revealed a marked decrease in growth of <i>P. multistriata</i> cultures undergoing sexual reproduction. The occurrence of growth arrest concomitantly with sexual reproduction is known in yeast, where the cell cycle is arrested in G1 as a consequence of pheromone signalling (Wilkinson &amp;amp; Pringle, <a href="#nph14557-bib-0078">1974</a>). One of the roles of pheromones is indeed cell synchronization to release gametes, simultaneously increasing the success of sexual reproduction (Frenkel <i>et al</i>., <a href="#nph14557-bib-0025">2014</a>). A total of 1112 genes (9% of the total <i>P. multistriata</i> genes) were differentially expressed in any of the comparisons (all sexualized vs all control samples, MT+ sexualized vs MT+ controls, and MT− sexualized vs MT− controls) (Tables <a href="#nph14557-sup-0008">S10</a>, <a href="#nph14557-sup-0009">S11</a>). A larger number of genes appeared to be regulated in MT− compared with MT+ (596 in MT− and 182 in MT+, Fisher test, <i>P</i> &amp;lt; 0.01) (Table <a href="#nph14557-sup-0002">S12</a>). In a cross, both MTs behave similarly (most of the cells arrest their growth and only a small fraction, <i>c</i>. 20%, undergo meiosis; Scalco <i>et al</i>., <a href="#nph14557-bib-0066">2014</a>), and the gametes produced are morphologically indistinguishable (Fig. <a href="#nph14557-fig-0001">1</a>). However, microscopic and time‐lapse observations have shown that the <i>P. multistriata</i> MT− cell in a pair undergoes meiosis, on average, 30 min earlier than the MT+ cell (Scalco <i>et al</i>., <a href="#nph14557-bib-0065">2016</a>). This observation suggests that the general response of the cells to pheromones is slightly out of phase, with MT− cells initiating the process earlier than MT+ cells, and this partly explains the different gene expression profiles between MT+ and MT−. In addition to this asynchrony, other MT‐specific changes can be explained by the fact that each of the two MTs secretes specific pheromones, produced by different mechanisms and triggering MT‐specific responses. Signalling involving multiple molecules has been demonstrated for <i>S. robusta</i> (Moeys <i>et al</i>., <a href="#nph14557-bib-0048">2016</a>) and postulated for <i>Pseudostaurosira trainorii</i> (Sato <i>et al</i>., <a href="#nph14557-bib-0064">2011</a>). In <i>S. robusta</i>, the compound diproline acts as an attraction pheromone (Gillard <i>et al</i>., <a href="#nph14557-bib-0027">2013</a>), whereas the molecule SIP+, secreted by MT+ cells, triggers both cell cycle arrest and diproline production in MT− cells (Moeys <i>et al</i>., <a href="#nph14557-bib-0048">2016</a>).</p> <div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/"><span class="label" tagx="label" title="label">Figure 4</span></a></div>   <p>Cell cycle and gene expression changes in the early stages of sexual reproduction. (a) Co‐culture glass apparatus containing cultures of opposite mating type (<span class="styled-content" tagx="styled-content" title="styled-content">MT</span>) separated by a membrane held by a metal ring (black arrowhead), and control bottles containing each of the two <span class="styled-content" tagx="styled-content" title="styled-content">MT</span>s. (b) Cell cycle phases of <span class="styled-content" tagx="styled-content" title="styled-content">MT</span>+ and <span class="styled-content" tagx="styled-content" title="styled-content">MT</span>– control (grey) and sexualized (green) samples represented by the relative percentage of cells in G1 (upper) and S + G2 + M (lower) phases, at the beginning of the experiment (0) and 2 and 6 h later. Whiskers in the boxplot extend to ± 1.5 × interquartile range (<span class="styled-content" tagx="styled-content" title="styled-content">IQR</span>). (c) Plot showing the log<sub>FC</sub> (fold change) (<i>y</i>‐axis) of genes differentially expressed, ordered according to the log<sub>CPM</sub> (counts per million) on the <i>x</i>‐axis. (d) Percentage of orphan genes and gene gains in the set of genes differentially expressed at the onset of the sexual phase in <i>Pseudo‐nitzschia multistriata</i> compared with the percentages of the same classes in the entire gene set.</p>   </div> </div> <div class="genesdifferentiallyexpressedduringsexualreproductionareinvolvedinsignalling,metabolism,nutrienttransportandmeiosis" title="sec"> <div class="title" tagx="title" title="title">
Genes differentially expressed during sexual reproduction are involved in signalling, metabolism, nutrient transport and meiosis</div> <p>The largest fraction of genes was found to be regulated in both MTs at T2 (Fig. <a href="#nph14557-fig-0004">4</a>c), where we observed a general tendency to downregulate genes encoding nutrient transporters, such as silicate (PSNMU‐V1.4_AUG‐EV‐PASAV3_0083020.1), ammonium, nitrate/nitrite and formate transporters (Tables <a href="#nph14557-tbl-0002">2</a>, <a href="#nph14557-sup-0008">S10</a>), suggesting that the cells, once the sexual phase is initiated, modulate their nutrient uptake. Interestingly, two <i>P. multistriata</i> genes with significant homology to the <i>P. tricornutum</i> diatom‐specific cyclins <i>dsCYC5</i> and <i>dsCYC4</i> were downregulated at T2 (Tables <a href="#nph14557-tbl-0002">2</a>, <a href="#nph14557-sup-0008">S10</a>). <i>Phaeodactylum tricornutum dsCYC5</i> is known to respond to phosphate addition and is supposed to be involved in signal integration to regulate the cell cycle, whereas <i>dsCYC4</i> is involved in the perception of growth stimuli (Huysman <i>et al</i>., <a href="#nph14557-bib-0034">2013</a>). In our experiment, we used nutrient‐replete medium, and it is unlikely that the cells suffered from nutrient limitation after only 6 h of treatment. Thus, the downregulation of nutrient transporters and cyclins involved in sensing the nutritional status suggests the existence of a complex interplay for the integration of external signals, including mating signals. Upregulation of genes encoding the cohesin complex (<i>SMC1</i>,<i> SMC3</i>,<i> SCC3</i>,<i> RAD21</i>; Patil <i>et al</i>., <a href="#nph14557-bib-0053">2015</a>), required to hold sister chromatids together during the S phase, indicated preparation for meiosis, which will occur a few hours later (Fig. <a href="#nph14557-fig-0001">1</a>; Scalco <i>et al</i>., <a href="#nph14557-bib-0065">2016</a>). This was also supported by the simultaneous upregulation of the meiosis‐related genes <i>RAD51‐A1</i>,<i> RAD51‐C</i>,<i> SMC5</i>,<i> PSD5</i> and Smc‐containing proteins (Tables <a href="#nph14557-tbl-0002">2</a>, <a href="#nph14557-sup-0008">S10</a>).</p> <div class="table-wrap_UNKNOWN" id="nph14557-tbl-0002" orientation="portrait" position="float"><span class="label" tagx="label" title="label">Table 2</span> <p>Selection of genes differentially expressed in <i>Pseudo‐nitzschia multistriata</i> cells 6 h after chemical contact of opposite mating types (MTs)</p>  <table frame="hsides" rules="groups"> <colgroup><col style="border-right:solid 1px #000000" span="1" /></colgroup> <colgroup><col style="border-right:solid 1px #000000" span="1" /></colgroup> <colgroup><col style="border-right:solid 1px #000000" span="1" /></colgroup> <colgroup><col style="border-right:solid 1px #000000" span="1" /></colgroup> <thead valign="top"> <tr style="border-bottom:solid 1px #000000"> <th align="left" valign="top" rowspan="1" colspan="1">Pathway/process</th> <th align="left" valign="top" rowspan="1" colspan="1">GeneModel ID</th> <th align="left" valign="top" rowspan="1" colspan="1">Gene description</th> <th align="left" valign="top" rowspan="1" colspan="1">log<sub>FC</sub></th> </tr> </thead> <tbody> <tr> <td align="left" rowspan="1" colspan="1">ABC transporters</td> <td align="left" rowspan="1" colspan="1">PSNMU‐V1.4_AUG‐EV‐PASAV3_0056660.1</td> <td align="left" rowspan="1" colspan="1">ATP‐binding cassette, subfamily B (MDR/TAP), member 1</td> <td align="left" rowspan="1" colspan="1">2.14</td> </tr> <tr> <td align="left" rowspan="10" colspan="1">Cell cycle and meiosis</td> <td align="left" rowspan="1" colspan="1">PSNMU‐V1.4_AUG‐EV‐PASAV3_0056780.1</td> <td align="left" rowspan="1" colspan="1">DNA repair protein RAD51 homologue 1, RAD51‐A1<a href="#nph14557-note-0002">a</a></td> <td align="left" rowspan="1" colspan="1">7.60</td> </tr> <tr> <td align="left" rowspan="1" colspan="1">PSNMU‐V1.4_AUG‐EV‐PASAV3_0104040.1</td> <td align="left" rowspan="1" colspan="1">DNA repair protein RAD51 homologue 3, RAD51‐C<a href="#nph14557-note-0002">a</a></td> <td align="left" rowspan="1" colspan="1">2.86</td> </tr> <tr> <td align="left" rowspan="1" colspan="1">PSNMU‐V1.4_AUG‐EV‐PASAV3_0089060.1</td> <td align="left" rowspan="1" colspan="1">Pds5<a href="#nph14557-note-0002">a</a></td> <td align="left" rowspan="1" colspan="1">1.95</td> </tr> <tr> <td align="left" rowspan="1" colspan="1">PSNMU‐V1.4_AUG‐EV‐PASAV3_0102810.1</td> <td align="left" rowspan="1" colspan="1">Structural maintenance of chromosomes protein 5</td> <td align="left" rowspan="1" colspan="1">2.02</td> </tr> <tr> <td align="left" rowspan="1" colspan="1">PSNMU‐V1.4_AUG‐EV‐PASAV3_0079810.1</td> <td align="left" rowspan="1" colspan="1">Structural maintenance of chromosomes protein 3</td> <td align="left" rowspan="1" colspan="1">1.67</td> </tr> <tr> <td align="left" rowspan="1" colspan="1">PSNMU‐V1.4_AUG‐EV‐PASAV3_0116990.1</td> <td align="left" rowspan="1" colspan="1">Structural maintenance of chromosomes protein 1</td> <td align="left" rowspan="1" colspan="1">2.60</td> </tr> <tr> <td align="left" rowspan="1" colspan="1">PSNMU‐V1.4_AUG‐EV‐PASAV3_0072170.1</td> <td align="left" rowspan="1" colspan="1">Cohesin complex subunit SCC1 (RAD21)</td> <td align="left" rowspan="1" colspan="1">2.07</td> </tr> <tr> <td align="left" rowspan="1" colspan="1">PSNMU‐V1.4_AUG‐EV‐PASAV3_0079350.1</td> <td align="left" rowspan="1" colspan="1">Cohesin complex subunit SA‐1/2 (SCC3)</td> <td align="left" rowspan="1" colspan="1">2.49</td> </tr> <tr> <td align="left" rowspan="1" colspan="1">PSNMU‐V1.4_AUG‐EV‐PASAV3_0081540.1</td> <td align="left" rowspan="1" colspan="1">G2/mitotic‐specific cyclin S13‐6 (dsCYC5)</td> <td align="left" rowspan="1" colspan="1">−1.66</td> </tr> <tr> <td align="left" rowspan="1" colspan="1">PSNMU‐V1.4_AUG‐EV‐PASAV3_0095090.1</td> <td align="left" rowspan="1" colspan="1">Cyclin‐B2‐2 (dsCYC4)</td> <td align="left" rowspan="1" colspan="1">−2.14</td> </tr> <tr> <td align="left" rowspan="3" colspan="1">Nitrate metabolism</td> <td align="left" rowspan="1" colspan="1">PSNMU‐V1.4_AUG‐EV‐PASAV3_0102470.1</td> <td align="left" rowspan="1" colspan="1">Ferredoxin‐dependent glutamate synthase 2</td> <td align="left" rowspan="1" colspan="1">−1.88</td> </tr> <tr> <td align="left" rowspan="1" colspan="1">PSNMU‐V1.4_AUG‐EV‐PASAV3_0012680.1</td> <td align="left" rowspan="1" colspan="1">Ammonium transporter 1, member 5</td> <td align="left" rowspan="1" colspan="1">−2.01</td> </tr> <tr> <td align="left" rowspan="1" colspan="1">PSNMU‐V1.4_AUG‐EV‐PASAV3_0048930.1</td> <td align="left" rowspan="1" colspan="1">Nitrate/nitrite transporter NarU</td> <td align="left" rowspan="1" colspan="1">−1.88</td> </tr> <tr> <td align="left" rowspan="2" colspan="1">Transcription factors</td> <td align="left" rowspan="1" colspan="1">PSNMU‐V1.4_AUG‐EV‐PASAV3_0036500.1</td> <td align="left" rowspan="1" colspan="1">Transcription factor SKN7</td> <td align="left" rowspan="1" colspan="1">−2.63</td> </tr> <tr> <td align="left" rowspan="1" colspan="1">PSNMU‐V1.4_AUG‐EV‐PASAV3_0061620.1</td> <td align="left" rowspan="1" colspan="1">Transcriptional activator Myb</td> <td align="left" rowspan="1" colspan="1">−2.31</td> </tr> <tr> <td align="left" rowspan="1" colspan="1">Protein processing</td> <td align="left" rowspan="1" colspan="1">PSNMU‐V1.4_AUG‐EV‐PASAV3_0032180.1</td> <td align="left" rowspan="1" colspan="1">Major intracellular serine protease</td> <td align="left" rowspan="1" colspan="1">−1.96</td> </tr> <tr> <td align="left" rowspan="10" colspan="1">Receptor like</td> <td align="left" rowspan="1" colspan="1">PSNMU‐V1.4_AUG‐EV‐PASAV3_0028550.1</td> <td align="left" rowspan="1" colspan="1">Probable leucine‐rich repeat receptor‐like protein kinase At2g33170</td> <td align="left" rowspan="1" colspan="1">2.39</td> </tr> <tr> <td align="left" rowspan="1" colspan="1">PSNMU‐V1.4_AUG‐EV‐PASAV3_0087210.1</td> <td align="left" rowspan="1" colspan="1">Probable leucine‐rich repeat receptor‐like protein kinase At1g35710</td> <td align="left" rowspan="1" colspan="1">−1.84</td> </tr> <tr> <td align="left" rowspan="1" colspan="1">PSNMU‐V1.4_AUG‐EV‐PASAV3_0079570.1</td> <td align="left" rowspan="1" colspan="1">Receptor‐like protein kinase 5</td> <td align="left" rowspan="1" colspan="1">−1.83</td> </tr> <tr> <td align="left" rowspan="1" colspan="1">PSNMU‐V1.4_AUG‐EV‐PASAV3_0113140.1</td> <td align="left" rowspan="1" colspan="1">Probable leucine‐rich repeat receptor‐like serine/threonine protein kinase At4g08850</td> <td align="left" rowspan="1" colspan="1">−1.70</td> </tr> <tr> <td align="left" rowspan="1" colspan="1">PSNMU‐V1.4_AUG‐EV‐PASAV3_0039440.1</td> <td align="left" rowspan="1" colspan="1">Receptor‐like protein kinase HSL1</td> <td align="left" rowspan="1" colspan="1">−1.69</td> </tr> <tr> <td align="left" rowspan="1" colspan="1">PSNMU‐V1.4_AUG‐EV‐PASAV3_0029550.1</td> <td align="left" rowspan="1" colspan="1">Leucine‐rich repeat receptor‐like serine/threonine protein kinase GSO2</td> <td align="left" rowspan="1" colspan="1">−1.62</td> </tr> <tr> <td align="left" rowspan="1" colspan="1">PSNMU‐V1.4_AUG‐EV‐PASAV3_0018920.1</td> <td align="left" rowspan="1" colspan="1">Somatic embryogenesis receptor kinase 1</td> <td align="left" rowspan="1" colspan="1">−1.52</td> </tr> <tr> <td align="left" rowspan="1" colspan="1">PSNMU‐V1.4_AUG‐EV‐PASAV3_0055410.1</td> <td align="left" rowspan="1" colspan="1">Probable leucine‐rich repeat receptor‐like serine/threonine protein kinase At4g26540</td> <td align="left" rowspan="1" colspan="1">−2.32</td> </tr> <tr> <td align="left" rowspan="1" colspan="1">PSNMU‐V1.4_AUG‐EV‐PASAV3_0055070.1<a href="#nph14557-note-0003">b</a></td> <td align="left" rowspan="2" colspan="1">Probable leucine‐rich repeat receptor‐like protein kinase</td> <td align="left" rowspan="2" colspan="1">−2.41</td> </tr> <tr> <td align="left" rowspan="1" colspan="1">PSNMU‐V1.4_AUG‐EV‐PASAV3_0055080.1<a href="#nph14557-note-0003">b</a></td> </tr> <tr> <td align="left" rowspan="1" colspan="1">Signalling</td> <td align="left" rowspan="1" colspan="1">PSNMU‐V1.4_AUG‐EV‐PASAV3_0051110.1</td> <td align="left" rowspan="1" colspan="1">Soluble guanylate cyclase 88E</td> <td align="left" rowspan="1" colspan="1">6.30</td> </tr> <tr> <td align="left" rowspan="4" colspan="1">Miscellaneous</td> <td align="left" rowspan="1" colspan="1">PSNMU‐V1.4_AUG‐EV‐PASAV3_0102760.1</td> <td align="left" rowspan="1" colspan="1">Protein aardvark (adhesion protein)</td> <td align="left" rowspan="1" colspan="1">6.09</td> </tr> <tr> <td align="left" rowspan="1" colspan="1">PSNMU‐V1.4_AUG‐EV‐PASAV3_0112340.1</td> <td align="left" rowspan="1" colspan="1">Tetratricopeptide (TPR) repeat‐containing protein<a href="#nph14557-note-0004">c</a> (protein–protein interactions)</td> <td align="left" rowspan="1" colspan="1">2.94</td> </tr> <tr> <td align="left" rowspan="1" colspan="1">PSNMU‐V1.4_AUG‐EV‐PASAV3_0063230.1</td> <td align="left" rowspan="1" colspan="1">Salicylate carboxymethyltransferase</td> <td align="left" rowspan="1" colspan="1">3.50</td> </tr> <tr> <td align="left" rowspan="1" colspan="1">PSNMU‐V1.4_AUG‐EV‐PASAV3_0078620.1</td> <td align="left" rowspan="1" colspan="1">E3 ubiquitin‐protein ligase Nedd‐4</td> <td align="left" rowspan="1" colspan="1">3.30</td> </tr> <tr> <td align="left" colspan="4" rowspan="1">MT‐specific</td> </tr> <tr> <td align="left" style="padding-left:10%" rowspan="3" colspan="1">Miscellaneous</td> <td align="left" rowspan="1" colspan="1">PSNMU‐V1.4_AUG‐EV‐PASAV3_0041130.1</td> <td align="left" rowspan="1" colspan="1">Heat shock factor protein 3</td> <td align="left" rowspan="1" colspan="1">3.41</td> </tr> <tr> <td align="left" rowspan="1" colspan="1">PSNMU‐V1.4_AUG‐EV‐PASAV3_0103000.1</td> <td align="left" rowspan="1" colspan="1">Cathepsin D</td> <td align="left" rowspan="1" colspan="1">6.96</td> </tr> <tr> <td align="left" rowspan="1" colspan="1">PSNMU‐V1.4_AUG‐EV‐PASAV3_0067710.1</td> <td align="left" rowspan="1" colspan="1">TPR repeat‐containing protein<a href="#nph14557-note-0004">c</a> (protein–protein interactions)</td> <td align="left" rowspan="1" colspan="1">−2.86</td> </tr> </tbody> </table> <div class="table-wrap-foot_UNKNOWN"> <div class="fn-type-" title=""><span class="label" tagx="label" title="label">a</span><p>Gene identity defined in Patil <i>et al</i>. (<a href="#nph14557-bib-0053">2015</a>).</p> </div> <div class="fn-type-" title=""><span class="label" tagx="label" title="label">b</span><p>Gene models to be merged in a single model.</p> </div> <div class="fn-type-" title=""><span class="label" tagx="label" title="label">c</span><p>Automatic annotation yields nephrocystin.</p> </div> </div> <div class="permissions"><span class="copyright" title="copyright">(C) <span class="copyright-holder" tagx="copyright-holder" title="copyright-holder">John Wiley &amp;amp; Sons, Ltd</span>, </span></div> </div> <p>One of the strongest inductions (6.3‐fold) was observed in both MTs for a soluble guanylate cyclase (Tables <a href="#nph14557-tbl-0002">2</a>, <a href="#nph14557-sup-0008">S10</a>). Notably, a bifunctional guanylyl cyclase/phosphodiesterase (<i>GC/PDE</i>) was found to be upregulated in MT− <i>S. robusta</i> cells in response to the SIP+ pheromone (Moeys <i>et al</i>., <a href="#nph14557-bib-0048">2016</a>). Although the <i>S. robusta GC/PDE</i> and the <i>P. multistriata</i> guanylate cyclase show some degree of similarity (25% identity), they are not orthologues (data not shown). The <i>P. multistriata GC/PDE</i> homologue (PSNMU‐V1.4_AUG‐EV‐PASAV3_0076150.1) was also regulated, albeit at a lower level (1.9‐fold). Therefore, through different genes, cyclic guanosine monophosphate (cGMP) synthesis and downstream activation of signalling are common responses to pheromone perception.</p> <p>The presence of G protein‐coupled receptors (GPCRs), one of the largest families of cell surface receptors in eukaryotes, has been reported in <i>P. multiseries</i> (Port <i>et al</i>., <a href="#nph14557-bib-0057">2013</a>), and we also detected gain of this class in <i>P. multistriata</i>. In the RNA‐seq data, PSNMU‐V1.4_AUG‐EV‐PASAV3_0072880.1, homologous to <i>GPCR3</i> (Port <i>et al</i>., <a href="#nph14557-bib-0057">2013</a>), is upregulated, and the inositol phospholipid signalling pathway, which is downstream of GPCRs, also appears to be employed by <i>P. multistriata</i> cells to transduce information (Table <a href="#nph14557-sup-0008">S10</a>). GPCRs bind many signalling molecules and our data indicate that they might be involved in the perception of mating cues in diatoms, as has been reported in yeast (Alvaro &amp;amp; Thorner, <a href="#nph14557-bib-0002">2016</a>).</p> <p>The MT− sexualized strain displayed a seven‐fold increase in Cathepsin D, a pepsin‐like aspartate protease, which has been shown to cleave proteins in the extracellular matrix (Handley <i>et al</i>., <a href="#nph14557-bib-0032">2001</a>). In <i>Saccharomyces cerevisiae</i>, an extracellular peptidase is involved in the degradation of the pheromone, a process that helps to align the pheromone gradient to detect the direction of the nearest mating partner, increasing mating efficiency (Barkai <i>et al</i>., <a href="#nph14557-bib-0005">1998</a>).</p> <p>Nineteen genes were selected for quantitative PCR validations on samples from an independent experiment, and changes were confirmed for 16 of them (Table <a href="#nph14557-sup-0002">S1</a>).</p> </div> <div class="asubsetofgenesdifferentiallyexpressedduringsexualreproductionshowslineage‐specificevolution" title="sec"> <div class="title" tagx="title" title="title">
A subset of genes differentially expressed during sexual reproduction shows lineage‐specific evolution</div> <p>Rapid divergence and positive selection are common for genes involved in sexual reproduction and can contribute to the establishment of reproductive isolation (Swanson &amp;amp; Vacquier, <a href="#nph14557-bib-0072">2002</a>). Our results support this assumption. A significantly higher proportion of genes differentially expressed during the early phases of sexual reproduction (sensing, Fig. <a href="#nph14557-fig-0001">1</a>) were predicted to be <i>P. multistriata</i> orphans (35%) or gene gain events in Bacillariales (20%) with respect to the same proportions in the whole proteome (Fisher test, <i>P</i> &amp;lt; 0.05; Fig. <a href="#nph14557-fig-0004">4</a>d; Tables <a href="#nph14557-sup-0010">S13</a>, <a href="#nph14557-sup-0011">S14</a>). In addition, when comparing the average identity between <i>P. multistriata</i> and <i>P. multiseries</i> orthologues (or between <i>P. multistriata</i> and other diatoms), genes differentially expressed in our experiment showed lower conservation with respect to the entire <i>P. multistriata</i> gene set (Fig. <a href="#nph14557-sup-0001">S11</a>).</p> <p>Furthermore, using our in‐depth clustering of <i>P. multistriata</i> genes with a diverse range of prokaryotes and eukaryotes, we observed that the majority of non‐orphan genes differentially expressed during the sexual phase were specific to diatoms (Figs <a href="#nph14557-fig-0005">5</a>, <a href="#nph14557-sup-0001">S11</a>e).</p> <div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/"><span class="label" tagx="label" title="label">Figure 5</span></a></div>   <p>Conservation of the genes differentially expressed in the experiments described in this work. Conservation is shown as the presence/absence of a horizontal line in 52 different species belonging to Prokaryotes, Rhizarians, Chromalveolates, Excavates, Unikonts and Plantae.</p>   </div> <p>The data indicate that a substantial fraction of the differentially expressed genes are diatom specific, Bacillariales specific or <i>Pseudo‐nitzschia</i> specific, or orphans, consistent with the uniqueness of the diatom life cycle and with the necessity to evolve species‐specific mechanisms to attract and mate with the right partner.</p> <p>Reproductive proteins show a tendency to be under positive selection (Clark <i>et al</i>., <a href="#nph14557-bib-0013">2006</a>). In order to identify the <i>P. multistriata</i> genes which are under positive selection, we selected all one‐to‐one homologues between <i>P. multistriata</i> and <i>P. multiseries</i>, and calculated the Ka : Ks ratio (number of nonsynonymous mutations/number of synonymous mutations) to measure their evolutionary divergence (Yang &amp;amp; Bielawski, <a href="#nph14557-bib-0083">2000</a>). Ka : Ks &amp;gt; 1 indicates a selective advantage to amino acid substitutions in a protein. Of the 6066 homologous pairs identified (Table <a href="#nph14557-sup-0012">S15</a>), 434 were among those regulated during sexual reproduction and included 11 genes showing a strong positive selection (Ka : Ks &amp;gt; 1). This group contains six unknown genes, two peptidases, two leucine‐rich repeat (LRR) receptor‐like protein kinases and a putative DNA helicase (Table <a href="#nph14557-sup-0012">S15</a>). The next gene in the list, with a Ka : Ks value of 0.92, is the homologue of the <i>S. robusta GC/PDE</i>. Further studies will clarify whether any of these genes has a specific role in recognizing the right mating partner, avoiding interspecies breeding. Finally, 20 differentially expressed genes are derived from bacteria by HGT in diatoms (Table <a href="#nph14557-sup-0013">S16</a>); an example (nitrate/nitrite transporter, PSNMU‐V1.4_AUG‐EV‐PASAV3_0048930.1) is shown in Fig. <a href="#nph14557-sup-0002">S12</a>.</p> <p>A schematic summary of the regulated pathways and functions in a <i>P. multistriata</i> cell responding to sexual cues is shown in Fig. <a href="#nph14557-fig-0006">6</a>.</p> <div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/"><span class="label" tagx="label" title="label">Figure 6</span></a></div>   <p>Cell response to sexual cues. Diagrammatic representation of a <i>Pseudo‐nitzschia multistriata</i> cell with the principal genes involved in the response to chemical cues acting at the beginning of sexual reproduction. Green triangles represent upregulation and red triangles downregulation of expression. PLC, Phospholipase C; DAG, diacylglycerol; PIP2, phosphatidylinositol biphosphate; IP3, inositol trisphosphate; GTP, Guanosine‐5′‐triphosphate; N, nucleus; ER, endoplasmic reticulum; M, mitochondrion; Ch, chloroplast; G, Golgi; LRR, leucine‐rich repeat.</p>   </div> <p>These data provide markers for data mining of metatranscriptomic datasets and will improve our ability to understand and monitor toxic <i>Pseudo‐nitzschia</i> blooms.</p> </div> </div> <div class="authorcontributions" title="sec"> <div class="title" tagx="title" title="title">
Author contributions</div> <p>M.I.F. coordinated the project. M.I.F., R.S., M.M. and M.C. designed the study. S.P., C.F. and R.C. performed the experiments. S.B., S.P., D.M., M.T.R., L.V., F.M., T.M., R.C., R.S. and M.I.F. analysed the data. S.B., M.I.F., R.S. and M.M. wrote the paper with contributions from all authors. All authors read and approved the final manuscript.</p> </div> <div class="supplementary-material" title="supplementary-material"> <div class="title" tagx="title" title="title">
Supporting information</div> <div class="local-data" title="local-data"> <p>Please note: Wiley Blackwell are not responsible for the content or functionality of any Supporting Information supplied by the authors. Any queries (other than missing material) should be directed to the <i>New Phytologist</i> Central Office.</p> </div> <div class="local-data" title="local-data">  <p><b>Fig. S1</b> General statistics of the <i>Pseudo‐nitzschia multistriata</i> genome assembly.</p> <p><b>Fig. S2</b> Putative association of conserved noncoding elements in <i>Pseudo‐nitzschia multistriata</i> with regulation of transcription.</p> <p><b>Fig. S3</b> Coverage of repeat elements and estimation of long terminal repeat (LTR) insertion period in the <i>Pseudo‐nitzschia multistriata</i> genome.</p> <p><b>Fig. S4</b> Comparison of the number of protein clusters (potential gene families) in chromalveolates, unikonts, plants and prokaryotes (archaea + bacteria) in relation to the clusters with at least one representative from <i>Pseudo‐nitzschia multistriata</i>.</p> <p><b>Fig. S5</b> Enrichment of gene ontology (GO) molecular function terms for gene families gained in photosynthetic Stramenopiles or in <i>Pseudo‐nitzschia</i> and <i>Fragilariopsis</i>.</p> <p><b>Fig. S6</b> Comparison of the number of <i>Pseudo‐nitzschia multistriata</i> proteins sharing common clusters (potential gene families) with red algae, plants, fungi, metazoans and bacteria.</p> <p><b>Fig. S7</b> General statistics of <i>Pseudo‐nitzschia multistriata</i> genes against those predicted to be of bacterial origin specifically in diatoms.</p> <p><b>Fig. S8</b> Enrichment of gene ontology (GO) molecular function terms for genes of potential bacterial origin specific to diatoms, Stramenopiles or SAR (Stramenopiles, Alveolates and Rhizaria).</p> <p><b>Fig. S9</b> GC content of genes acquired by horizontal gene transfer from bacteria as compared with all genes in <i>Pseudo‐nitzschia multistriata</i>.</p> <p><b>Fig. S10</b> Experimental set‐up for gene expression studies at the onset of sexual reproduction.</p> <p><b>Fig. S11</b> Conservation of genes predicted to be differentially expressed during sexual reproduction in <i>Pseudo‐nitzschia multistriata</i> compared with the same data for the entire <i>P. multistriata</i> gene set.</p> <p><b>Fig. S12</b> Screenshot of the <i>Pseudo‐nitzschia multistriata</i> genome browser.</p> <p><b>Methods S1</b> Supplementary methods.</p>  <div class="media" title="media"><a href="NPH-215-140-s001.pdf">LINK</a> <p>Click here for additional data file.</p>  </div> </div> <div class="local-data" title="local-data">  <p><b>Table S1</b> Validation of a selected subset of genes differentially expressed during sexual reproduction in <i>Pseudo‐nitzschia multistriata</i> by quantitative PCR</p> <p><b>Table S4</b> Insertion period estimation of complete long terminal repeats (LTRs) identified in diatom genomes</p> <p><b>Table S8</b> Annotation for diatom genes of red algal origin</p> <p><b>Table S9</b> Summary statistics of RNA‐seq read mapping results for <i>Pseudo‐nitzschia multistriata</i> samples</p> <p><b>Table S12</b> Statistics of genes differentially regulated during the sexualized stage in both mating types at two different time points</p>  <div class="media" title="media"><a href="NPH-215-140-s002.xlsx">LINK</a> <p>Click here for additional data file.</p>  </div> </div> <div class="local-data" title="local-data">  <p><b>Table S2</b> Genomic coordinates of the <i>Pseudo‐nitzschia multistriata</i> conserved noncoding elements, together with coordinates in other diatom species, where each element remains conserved</p>  <div class="media" title="media"><a href="NPH-215-140-s003.xls">LINK</a> <p>Click here for additional data file.</p>  </div> </div> <div class="local-data" title="local-data">  <p><b>Table S3</b> The core, plant and fungal transcription factor families which show enrichment of binding sites on the <i>Pseudo‐nitzschia multistriata</i> conserved noncoding elements</p>  <div class="media" title="media"><a href="NPH-215-140-s004.xls">LINK</a> <p>Click here for additional data file.</p>  </div> </div> <div class="local-data" title="local-data">  <p><b>Table S5</b> Number of proteins from stramenopile genomes represented by different superfamilies from the SUPERFAMILY database</p>  <div class="media" title="media"><a href="NPH-215-140-s005.xls">LINK</a> <p>Click here for additional data file.</p>  </div> </div> <div class="local-data" title="local-data">  <p><b>Table S6</b> Details of eukaryotic and prokaryotic organisms considered for the generation of the protein clusters for the <i>Pseudo‐nitzschia multistriata</i> proteome</p>  <div class="media" title="media"><a href="NPH-215-140-s006.xls">LINK</a> <p>Click here for additional data file.</p>  </div> </div> <div class="local-data" title="local-data">  <p><b>Table S7</b> Annotation for the <i>Pseudo‐nitzschia</i>/diatom/stramenopile/SAR (Stramenopile, Alveolates and Rhizaria)‐specific genes of bacterial origin</p>  <div class="media" title="media"><a href="NPH-215-140-s007.xls">LINK</a> <p>Click here for additional data file.</p>  </div> </div> <div class="local-data" title="local-data">  <p><b>Table S10</b> Differential expression analyses of all sexualized samples vs all control samples, MT+ sexualized samples against MT+ controls, and MT− sexualized samples against MT− controls, at two different time points</p>  <div class="media" title="media"><a href="NPH-215-140-s008.xlsx">LINK</a> <p>Click here for additional data file.</p>  </div> </div> <div class="local-data" title="local-data">  <p><b>Table S11</b> Log<sub>FC</sub> (fold change) and false discovery rate (FDR) values for all <i>Pseudo‐nitzschia multistriata</i> transcripts for the same conditions as in Table S10</p>  <div class="media" title="media"><a href="NPH-215-140-s009.xls">LINK</a> <p>Click here for additional data file.</p>  </div> </div> <div class="local-data" title="local-data">  <p><b>Table S13</b> Differentially expressed genes predicted to be gene gain events in diatoms post‐divergence from <i>Phaeodactylum tricornutum</i></p>  <div class="media" title="media"><a href="NPH-215-140-s010.xlsx">LINK</a> <p>Click here for additional data file.</p>  </div> </div> <div class="local-data" title="local-data">  <p><b>Table S14</b> Differentially expressed genes predicted to be orphan genes in <i>Pseudo‐nitzschia multistriata</i></p>  <div class="media" title="media"><a href="NPH-215-140-s011.xlsx">LINK</a> <p>Click here for additional data file.</p>  </div> </div> <div class="local-data" title="local-data">  <p><b>Table S15</b> Rate of evolution of homologous pairs of <i>Pseudo‐nitzschia multistriata</i> and <i>Pseudo‐nitzschia multiseries</i></p>  <div class="media" title="media"><a href="NPH-215-140-s012.xlsx">LINK</a> <p>Click here for additional data file.</p>  </div> </div> <div class="local-data" title="local-data">  <p><b>Table S16</b> Genes predicted to be introduced via horizontal gene transfer (HGT) in diatoms, showing differential expression during sexual reproduction in <i>Pseudo‐nitzschia multistriata</i></p>  <div class="media" title="media"><a href="NPH-215-140-s013.xls">LINK</a> <p>Click here for additional data file.</p>  </div> </div> </div> </div> <div class="back" title="back"> <div class="ack" title="ack"> <div class="title" tagx="title" title="title">
Acknowledgements</div> <p>This work was supported by Marie Curie FP7‐PEOPLE‐2011‐CIG (GyPSy, grant no. 293887) to M.I.F. and the RITMARE Italian Flagship Project. The work conducted by the US Department of Energy Joint Genome Institute is supported by the Office of Science of the US Department of Energy under contract no. DE‐AC02‐05CH11231. S.P. and L.V. were supported by a SZN Open University PhD fellowship from Stazione Zoologica Anton Dohrn. This research was partly supported by the NBIP Computing infrastructure for Science (CiS). The authors wish to thank Cecilia Balestra for technical help, Anil Thanki for the TGAC browser, Riccardo Aiese Cigliano and Walter Sanseverino (Sequentia Biotech, <a href="http://www.sequentiabiotech.com">www.sequentiabiotech.com</a>), the International Centre for Theoretical Physics for cooperation, the EMBL GeneCore service and the SZN Molecular Biology and Bioinformatics Unit. E. Scalco kindly provided <i>P. multistriata</i> images.</p> </div> <div class="references">
References</div> <div tag="ref-list"> <ul> <div class="title" tagx="title" title="title">
References</div> <li tag="ref"><a name="nph14557-bib-0001" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Adelfi</span><span class="given-names" tagx="given-names" title="given-names">MG</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Borra</span><span class="given-names" tagx="given-names" title="given-names">M</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Sanges</span><span class="given-names" tagx="given-names" title="given-names">R</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Montresor</span><span class="given-names" tagx="given-names" title="given-names">M</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Fontana</span><span class="given-names" tagx="given-names" title="given-names">A</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Ferrante</span><span class="given-names" tagx="given-names" title="given-names">MI</span></span>. <span class="year" tagx="year" title="year">2014</span><span class="mixed-article-title" title="mixed-article-title">Selection and validation of reference genes for qPCR analysis in the pennate diatoms <i>Pseudo‐nitzschia multistriata</i> and <i>P. arenysensis</i></span>. <span class="source" tagx="source" title="source">Journal of Experimental Marine Biology and Ecology</span><span class="volume" tagx="volume" title="volume">451</span>: <span class="fpage" tagx="fpage" title="fpage">74</span>–<span class="lpage" tagx="lpage" title="lpage">81</span>.</span></li> <li tag="ref"><a name="nph14557-bib-0002" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Alvaro</span><span class="given-names" tagx="given-names" title="given-names">CG</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Thorner</span><span class="given-names" tagx="given-names" title="given-names">J</span></span>. <span class="year" tagx="year" title="year">2016</span><span class="mixed-article-title" title="mixed-article-title">Heterotrimeric G protein‐coupled receptor signaling in yeast mating pheromone response</span>. <span class="source" tagx="source" title="source">Journal of Biological Chemistry</span><span class="volume" tagx="volume" title="volume">291</span>: <span class="fpage" tagx="fpage" title="fpage">7788</span>–<span class="lpage" tagx="lpage" title="lpage">7795</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/26907689">26907689</a></span></span></li> <li tag="ref"><a name="nph14557-bib-0003" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Armbrust</span><span class="given-names" tagx="given-names" title="given-names">EV</span></span>. <span class="year" tagx="year" title="year">2009</span><span class="mixed-article-title" title="mixed-article-title">The life of diatoms in the world's oceans</span>. <span class="source" tagx="source" title="source">Nature</span><span class="volume" tagx="volume" title="volume">459</span>: <span class="fpage" tagx="fpage" title="fpage">185</span>–<span class="lpage" tagx="lpage" title="lpage">192</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19444204">19444204</a></span></span></li> <li tag="ref"><a name="nph14557-bib-0004" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Armbrust</span><span class="given-names" tagx="given-names" title="given-names">EV</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Berges</span><span class="given-names" tagx="given-names" title="given-names">JA</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Bowler</span><span class="given-names" tagx="given-names" title="given-names">C</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Green</span><span class="given-names" tagx="given-names" title="given-names">BR</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Martinez</span><span class="given-names" tagx="given-names" title="given-names">D</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Putnam</span><span class="given-names" tagx="given-names" title="given-names">NH</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Zhou</span><span class="given-names" tagx="given-names" title="given-names">S</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Allen</span><span class="given-names" tagx="given-names" title="given-names">AE</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Apt</span><span class="given-names" tagx="given-names" title="given-names">KE</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Bechner</span><span class="given-names" tagx="given-names" title="given-names">M</span></span><i>et al</i><span class="year" tagx="year" title="year">2004</span><span class="mixed-article-title" title="mixed-article-title">The genome of the diatom <i>Thalassiosira pseudonana</i>: ecology, evolution, and metabolism</span>. <span class="source" tagx="source" title="source">Science</span><span class="volume" tagx="volume" title="volume">306</span>: <span class="fpage" tagx="fpage" title="fpage">79</span>–<span class="lpage" tagx="lpage" title="lpage">86</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15459382">15459382</a></span></span></li> <li tag="ref"><a name="nph14557-bib-0005" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Barkai</span><span class="given-names" tagx="given-names" title="given-names">N</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Rose</span><span class="given-names" tagx="given-names" title="given-names">MD</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Wingreen</span><span class="given-names" tagx="given-names" title="given-names">NS</span></span>. <span class="year" tagx="year" title="year">1998</span><span class="mixed-article-title" title="mixed-article-title">Protease helps yeast find mating partners</span>. <span class="source" tagx="source" title="source">Nature</span><span class="volume" tagx="volume" title="volume">396</span>: <span class="fpage" tagx="fpage" title="fpage">422</span>–<span class="lpage" tagx="lpage" title="lpage">423</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/9853747">9853747</a></span></span></li> <li tag="ref"><a name="nph14557-bib-0006" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Bennetzen</span><span class="given-names" tagx="given-names" title="given-names">JL</span></span>. <span class="year" tagx="year" title="year">2005</span><span class="mixed-article-title" title="mixed-article-title">Transposable elements, gene creation and genome rearrangement in flowering plants</span>. <span class="source" tagx="source" title="source">Current Opinion in Genetics &amp;amp; Development</span><span class="volume" tagx="volume" title="volume">15</span>: <span class="fpage" tagx="fpage" title="fpage">621</span>–<span class="lpage" tagx="lpage" title="lpage">627</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16219458">16219458</a></span></span></li> <li tag="ref"><a name="nph14557-bib-0007" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Blanchette</span><span class="given-names" tagx="given-names" title="given-names">M</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Kent</span><span class="given-names" tagx="given-names" title="given-names">WJ</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Riemer</span><span class="given-names" tagx="given-names" title="given-names">C</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Elnitski</span><span class="given-names" tagx="given-names" title="given-names">L</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Smit</span><span class="given-names" tagx="given-names" title="given-names">AFA</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Roskin</span><span class="given-names" tagx="given-names" title="given-names">KM</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Baertsch</span><span class="given-names" tagx="given-names" title="given-names">R</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Rosenbloom</span><span class="given-names" tagx="given-names" title="given-names">K</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Clawson</span><span class="given-names" tagx="given-names" title="given-names">H</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Green</span><span class="given-names" tagx="given-names" title="given-names">ED</span></span><i>et al</i><span class="year" tagx="year" title="year">2004</span><span class="mixed-article-title" title="mixed-article-title">Aligning multiple genomic sequences with the threaded blockset aligner</span>. <span class="source" tagx="source" title="source">Genome Research</span><span class="volume" tagx="volume" title="volume">14</span>: <span class="fpage" tagx="fpage" title="fpage">708</span>–<span class="lpage" tagx="lpage" title="lpage">715</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15060014">15060014</a></span></span></li> <li tag="ref"><a name="nph14557-bib-0008" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Bleykasten‐Grosshans</span><span class="given-names" tagx="given-names" title="given-names">C</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Neuvéglise</span><span class="given-names" tagx="given-names" title="given-names">C</span></span>. <span class="year" tagx="year" title="year">2011</span><span class="mixed-article-title" title="mixed-article-title">Transposable elements in yeasts</span>. <span class="source" tagx="source" title="source">Comptes Rendus Biologies</span><span class="volume" tagx="volume" title="volume">334</span>: <span class="fpage" tagx="fpage" title="fpage">679</span>–<span class="lpage" tagx="lpage" title="lpage">686</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21819950">21819950</a></span></span></li> <li tag="ref"><a name="nph14557-bib-0009" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Bolger</span><span class="given-names" tagx="given-names" title="given-names">AM</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Lohse</span><span class="given-names" tagx="given-names" title="given-names">M</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Usadel</span><span class="given-names" tagx="given-names" title="given-names">B</span></span>. <span class="year" tagx="year" title="year">2014</span><span class="mixed-article-title" title="mixed-article-title">Trimmomatic: a flexible trimmer for Illumina sequence data</span>. <span class="source" tagx="source" title="source">Bioinformatics</span><span class="volume" tagx="volume" title="volume">30</span>: <span class="fpage" tagx="fpage" title="fpage">2114</span>–<span class="lpage" tagx="lpage" title="lpage">2120</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24695404">24695404</a></span></span></li> <li tag="ref"><a name="nph14557-bib-0010" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Bowler</span><span class="given-names" tagx="given-names" title="given-names">C</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Allen</span><span class="given-names" tagx="given-names" title="given-names">AE</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Badger</span><span class="given-names" tagx="given-names" title="given-names">JH</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Grimwood</span><span class="given-names" tagx="given-names" title="given-names">J</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Jabbari</span><span class="given-names" tagx="given-names" title="given-names">K</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Kuo</span><span class="given-names" tagx="given-names" title="given-names">A</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Maheswari</span><span class="given-names" tagx="given-names" title="given-names">U</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Martens</span><span class="given-names" tagx="given-names" title="given-names">C</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Maumus</span><span class="given-names" tagx="given-names" title="given-names">F</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Otillar</span><span class="given-names" tagx="given-names" title="given-names">RP</span></span><i>et al</i><span class="year" tagx="year" title="year">2008</span><span class="mixed-article-title" title="mixed-article-title">The <i>Phaeodactylum</i> genome reveals the evolutionary history of diatom genomes</span>. <span class="source" tagx="source" title="source">Nature</span><span class="volume" tagx="volume" title="volume">456</span>: <span class="fpage" tagx="fpage" title="fpage">239</span>–<span class="lpage" tagx="lpage" title="lpage">244</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18923393">18923393</a></span></span></li> <li tag="ref"><a name="nph14557-bib-0011" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Bowler</span><span class="given-names" tagx="given-names" title="given-names">C</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Vardi</span><span class="given-names" tagx="given-names" title="given-names">A</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Allen</span><span class="given-names" tagx="given-names" title="given-names">AE</span></span>. <span class="year" tagx="year" title="year">2010</span><span class="mixed-article-title" title="mixed-article-title">Oceanographic and biogeochemical insights from diatom genomes</span>. <span class="source" tagx="source" title="source">Annual Review of Marine Science</span><span class="volume" tagx="volume" title="volume">2</span>: <span class="fpage" tagx="fpage" title="fpage">333</span>–<span class="lpage" tagx="lpage" title="lpage">365</span>.</span></li> <li tag="ref"><a name="nph14557-bib-0012" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Capella‐Gutiérrez</span><span class="given-names" tagx="given-names" title="given-names">S</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Silla‐Martínez</span><span class="given-names" tagx="given-names" title="given-names">JM</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Gabaldón</span><span class="given-names" tagx="given-names" title="given-names">T</span></span>. <span class="year" tagx="year" title="year">2009</span><span class="mixed-article-title" title="mixed-article-title">trimAl: a tool for automated alignment trimming in large‐scale phylogenetic analyses</span>. <span class="source" tagx="source" title="source">Bioinformatics</span><span class="volume" tagx="volume" title="volume">25</span>: <span class="fpage" tagx="fpage" title="fpage">1972</span>–<span class="lpage" tagx="lpage" title="lpage">1973</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19505945">19505945</a></span></span></li> <li tag="ref"><a name="nph14557-bib-0013" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Clark</span><span class="given-names" tagx="given-names" title="given-names">NL</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Aagaard</span><span class="given-names" tagx="given-names" title="given-names">JE</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Swanson</span><span class="given-names" tagx="given-names" title="given-names">WJ</span></span>. <span class="year" tagx="year" title="year">2006</span><span class="mixed-article-title" title="mixed-article-title">Evolution of reproductive proteins from animals and plants</span>. <span class="source" tagx="source" title="source">Reproduction</span><span class="volume" tagx="volume" title="volume">131</span>: <span class="fpage" tagx="fpage" title="fpage">11</span>–<span class="lpage" tagx="lpage" title="lpage">22</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16388004">16388004</a></span></span></li> <li tag="ref"><a name="nph14557-bib-0014" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Cock</span><span class="given-names" tagx="given-names" title="given-names">JM</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Sterck</span><span class="given-names" tagx="given-names" title="given-names">L</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Rouzé</span><span class="given-names" tagx="given-names" title="given-names">P</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Scornet</span><span class="given-names" tagx="given-names" title="given-names">D</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Allen</span><span class="given-names" tagx="given-names" title="given-names">AE</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Amoutzias</span><span class="given-names" tagx="given-names" title="given-names">G</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Anthouard</span><span class="given-names" tagx="given-names" title="given-names">V</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Artiguenave</span><span class="given-names" tagx="given-names" title="given-names">F</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Aury</span><span class="given-names" tagx="given-names" title="given-names">J‐M</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Badger</span><span class="given-names" tagx="given-names" title="given-names">JH</span></span><i>et al</i><span class="year" tagx="year" title="year">2010</span><span class="mixed-article-title" title="mixed-article-title">The <i>Ectocarpus</i> genome and the independent evolution of multicellularity in brown algae</span>. <span class="source" tagx="source" title="source">Nature</span><span class="volume" tagx="volume" title="volume">465</span>: <span class="fpage" tagx="fpage" title="fpage">617</span>–<span class="lpage" tagx="lpage" title="lpage">621</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20520714">20520714</a></span></span></li> <li tag="ref"><a name="nph14557-bib-0015" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">D'Alelio</span><span class="given-names" tagx="given-names" title="given-names">D</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Amato</span><span class="given-names" tagx="given-names" title="given-names">A</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Luedeking</span><span class="given-names" tagx="given-names" title="given-names">A</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Montresor</span><span class="given-names" tagx="given-names" title="given-names">M</span></span>. <span class="year" tagx="year" title="year">2009</span><span class="mixed-article-title" title="mixed-article-title">Sexual and vegetative phases in the planktonic diatom <i>Pseudo‐nitzschia multistriata</i></span>. <span class="source" tagx="source" title="source">Harmful Algae</span><span class="volume" tagx="volume" title="volume">8</span>: <span class="fpage" tagx="fpage" title="fpage">225</span>–<span class="lpage" tagx="lpage" title="lpage">232</span>.</span></li> <li tag="ref"><a name="nph14557-bib-0016" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Darriba</span><span class="given-names" tagx="given-names" title="given-names">D</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Taboada</span><span class="given-names" tagx="given-names" title="given-names">GL</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Doallo</span><span class="given-names" tagx="given-names" title="given-names">R</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Posada</span><span class="given-names" tagx="given-names" title="given-names">D</span></span>. <span class="year" tagx="year" title="year">2011</span><span class="mixed-article-title" title="mixed-article-title">ProtTest 3: fast selection of best‐fit models of protein evolution</span>. <span class="source" tagx="source" title="source">Bioinformatics</span><span class="volume" tagx="volume" title="volume">27</span>: <span class="fpage" tagx="fpage" title="fpage">1164</span>–<span class="lpage" tagx="lpage" title="lpage">1165</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21335321">21335321</a></span></span></li> <li tag="ref"><a name="nph14557-bib-0017" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Di Dato</span><span class="given-names" tagx="given-names" title="given-names">V</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Musacchia</span><span class="given-names" tagx="given-names" title="given-names">F</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Petrosino</span><span class="given-names" tagx="given-names" title="given-names">G</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Patil</span><span class="given-names" tagx="given-names" title="given-names">S</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Montresor</span><span class="given-names" tagx="given-names" title="given-names">M</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Sanges</span><span class="given-names" tagx="given-names" title="given-names">R</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Ferrante</span><span class="given-names" tagx="given-names" title="given-names">MI</span></span>. <span class="year" tagx="year" title="year">2015</span><span class="mixed-article-title" title="mixed-article-title">Transcriptome sequencing of three <i>Pseudo‐nitzschia</i> species reveals comparable gene sets and the presence of Nitric Oxide Synthase genes in diatoms</span>. <span class="source" tagx="source" title="source">Scientific Reports</span><span class="volume" tagx="volume" title="volume">5</span>: <span class="fpage" tagx="fpage" title="fpage">12329</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/26189990">26189990</a></span></span></li> <li tag="ref"><a name="nph14557-bib-0018" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Dobin</span><span class="given-names" tagx="given-names" title="given-names">A</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Davis</span><span class="given-names" tagx="given-names" title="given-names">CA</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Schlesinger</span><span class="given-names" tagx="given-names" title="given-names">F</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Drenkow</span><span class="given-names" tagx="given-names" title="given-names">J</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Zaleski</span><span class="given-names" tagx="given-names" title="given-names">C</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Jha</span><span class="given-names" tagx="given-names" title="given-names">S</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Batut</span><span class="given-names" tagx="given-names" title="given-names">P</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Chaisson</span><span class="given-names" tagx="given-names" title="given-names">M</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Gingeras</span><span class="given-names" tagx="given-names" title="given-names">TR</span></span>. <span class="year" tagx="year" title="year">2013</span><span class="mixed-article-title" title="mixed-article-title">STAR: ultrafast universal RNA‐seq aligner</span>. <span class="source" tagx="source" title="source">Bioinformatics</span><span class="volume" tagx="volume" title="volume">29</span>: <span class="fpage" tagx="fpage" title="fpage">15</span>–<span class="lpage" tagx="lpage" title="lpage">21</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23104886">23104886</a></span></span></li> <li tag="ref"><a name="nph14557-bib-0019" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Eddy</span><span class="given-names" tagx="given-names" title="given-names">SR</span></span>. <span class="year" tagx="year" title="year">1995</span><span class="mixed-article-title" title="mixed-article-title">Multiple alignment using hidden Markov models</span>. <span class="source" tagx="source" title="source">Proceedings/International Conference on Intelligent Systems for Molecular Biology; ISMB. International Conference on Intelligent Systems for Molecular Biology</span><span class="volume" tagx="volume" title="volume">3</span>: <span class="fpage" tagx="fpage" title="fpage">114</span>–<span class="lpage" tagx="lpage" title="lpage">120</span>.</span></li> <li tag="ref"><a name="nph14557-bib-0020" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Ekseth</span><span class="given-names" tagx="given-names" title="given-names">OK</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Kuiper</span><span class="given-names" tagx="given-names" title="given-names">M</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Mironov</span><span class="given-names" tagx="given-names" title="given-names">V</span></span>. <span class="year" tagx="year" title="year">2014</span><span class="mixed-article-title" title="mixed-article-title">orthAgogue: an agile tool for the rapid prediction of orthology relations</span>. <span class="source" tagx="source" title="source">Bioinformatics</span><span class="volume" tagx="volume" title="volume">30</span>: <span class="fpage" tagx="fpage" title="fpage">734</span>–<span class="lpage" tagx="lpage" title="lpage">736</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24115168">24115168</a></span></span></li> <li tag="ref"><a name="nph14557-bib-0021" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Enright</span><span class="given-names" tagx="given-names" title="given-names">AJ</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Van Dongen</span><span class="given-names" tagx="given-names" title="given-names">S</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Ouzounis</span><span class="given-names" tagx="given-names" title="given-names">CA</span></span>. <span class="year" tagx="year" title="year">2002</span><span class="mixed-article-title" title="mixed-article-title">An efficient algorithm for large‐scale detection of protein families</span>. <span class="source" tagx="source" title="source">Nucleic Acids Research</span><span class="volume" tagx="volume" title="volume">30</span>: <span class="fpage" tagx="fpage" title="fpage">1575</span>–<span class="lpage" tagx="lpage" title="lpage">1584</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11917018">11917018</a></span></span></li> <li tag="ref"><a name="nph14557-bib-0022" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Falkowski</span><span class="given-names" tagx="given-names" title="given-names">P</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Knoll</span><span class="given-names" tagx="given-names" title="given-names">AH</span></span>. <span class="year" tagx="year" title="year">2011</span><span class="chapter-title" tagx="chapter-title" title="chapter-title">An introduction to primary producers in the sea: who they are, what they do, and when they evolved</span> In: <span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Falkowski</span><span class="given-names" tagx="given-names" title="given-names">PG</span></span></span>, <span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Knoll</span><span class="given-names" tagx="given-names" title="given-names">AH</span></span></span>, eds. <span class="source" tagx="source" title="source">Evolution of primary producers in the sea</span>. <span class="publisher-loc" tagx="publisher-loc" title="publisher-loc">Burlington, MA, USA</span>: <span class="publisher-name" tagx="publisher-name" title="publisher-name">Academic Press</span>, <span class="fpage" tagx="fpage" title="fpage">1</span>–<span class="lpage" tagx="lpage" title="lpage">7</span>.</span></li> <li tag="ref"><a name="nph14557-bib-0023" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Felsenstein</span><span class="given-names" tagx="given-names" title="given-names">J</span></span>. <span class="year" tagx="year" title="year">1989</span><span class="mixed-article-title" title="mixed-article-title">PHYLIP – phylogeny inference package (Version 3.2)</span>. <span class="source" tagx="source" title="source">Cladistics</span><span class="volume" tagx="volume" title="volume">5</span>: <span class="fpage" tagx="fpage" title="fpage">164</span>–<span class="lpage" tagx="lpage" title="lpage">166</span>.</span></li> <li tag="ref"><a name="nph14557-bib-0024" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Flutre</span><span class="given-names" tagx="given-names" title="given-names">T</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Duprat</span><span class="given-names" tagx="given-names" title="given-names">E</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Feuillet</span><span class="given-names" tagx="given-names" title="given-names">C</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Quesneville</span><span class="given-names" tagx="given-names" title="given-names">H</span></span>. <span class="year" tagx="year" title="year">2011</span><span class="mixed-article-title" title="mixed-article-title">Considering transposable element diversification in de novo annotation approaches</span>. <span class="source" tagx="source" title="source">PLoS ONE</span><span class="volume" tagx="volume" title="volume">6</span>: <span class="fpage" tagx="fpage" title="fpage">e16526</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21304975">21304975</a></span></span></li> <li tag="ref"><a name="nph14557-bib-0025" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Frenkel</span><span class="given-names" tagx="given-names" title="given-names">J</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Vyverman</span><span class="given-names" tagx="given-names" title="given-names">W</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Pohnert</span><span class="given-names" tagx="given-names" title="given-names">G</span></span>. <span class="year" tagx="year" title="year">2014</span><span class="mixed-article-title" title="mixed-article-title">Pheromone signaling during sexual reproduction in algae</span>. <span class="source" tagx="source" title="source">Plant Journal</span><span class="volume" tagx="volume" title="volume">79</span>: <span class="fpage" tagx="fpage" title="fpage">632</span>–<span class="lpage" tagx="lpage" title="lpage">644</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24597605">24597605</a></span></span></li> <li tag="ref"><a name="nph14557-bib-0026" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Garcia‐Vallvé</span><span class="given-names" tagx="given-names" title="given-names">S</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Romeu</span><span class="given-names" tagx="given-names" title="given-names">A</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Palau</span><span class="given-names" tagx="given-names" title="given-names">J</span></span>. <span class="year" tagx="year" title="year">2000</span><span class="mixed-article-title" title="mixed-article-title">Horizontal gene transfer in bacterial and archaeal complete genomes</span>. <span class="source" tagx="source" title="source">Genome Research</span><span class="volume" tagx="volume" title="volume">10</span>: <span class="fpage" tagx="fpage" title="fpage">1719</span>–<span class="lpage" tagx="lpage" title="lpage">1725</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11076857">11076857</a></span></span></li> <li tag="ref"><a name="nph14557-bib-0027" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Gillard</span><span class="given-names" tagx="given-names" title="given-names">J</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Frenkel</span><span class="given-names" tagx="given-names" title="given-names">J</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Devos</span><span class="given-names" tagx="given-names" title="given-names">V</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Sabbe</span><span class="given-names" tagx="given-names" title="given-names">K</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Paul</span><span class="given-names" tagx="given-names" title="given-names">C</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Rempt</span><span class="given-names" tagx="given-names" title="given-names">M</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Inzé</span><span class="given-names" tagx="given-names" title="given-names">D</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Pohnert</span><span class="given-names" tagx="given-names" title="given-names">G</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Vuylsteke</span><span class="given-names" tagx="given-names" title="given-names">M</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Vyverman</span><span class="given-names" tagx="given-names" title="given-names">W</span></span>. <span class="year" tagx="year" title="year">2013</span><span class="mixed-article-title" title="mixed-article-title">Metabolomics enables the structure elucidation of a diatom sex pheromone</span>. <span class="source" tagx="source" title="source">Angewandte Chemie International Edition</span><span class="volume" tagx="volume" title="volume">52</span>: <span class="fpage" tagx="fpage" title="fpage">854</span>–<span class="lpage" tagx="lpage" title="lpage">857</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23315901">23315901</a></span></span></li> <li tag="ref"><a name="nph14557-bib-0028" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Gnerre</span><span class="given-names" tagx="given-names" title="given-names">S</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">MacCallum</span><span class="given-names" tagx="given-names" title="given-names">I</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Przybylski</span><span class="given-names" tagx="given-names" title="given-names">D</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Ribeiro</span><span class="given-names" tagx="given-names" title="given-names">FJ</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Burton</span><span class="given-names" tagx="given-names" title="given-names">JN</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Walker</span><span class="given-names" tagx="given-names" title="given-names">BJ</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Sharpe</span><span class="given-names" tagx="given-names" title="given-names">T</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Hall</span><span class="given-names" tagx="given-names" title="given-names">G</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Shea</span><span class="given-names" tagx="given-names" title="given-names">TP</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Sykes</span><span class="given-names" tagx="given-names" title="given-names">S</span></span><i>et al</i><span class="year" tagx="year" title="year">2011</span><span class="mixed-article-title" title="mixed-article-title">High‐quality draft assemblies of mammalian genomes from massively parallel sequence data</span>. <span class="source" tagx="source" title="source">Proceedings of the National Academy of Sciences, USA</span><span class="volume" tagx="volume" title="volume">108</span>: <span class="fpage" tagx="fpage" title="fpage">1513</span>–<span class="lpage" tagx="lpage" title="lpage">1518</span>.</span></li> <li tag="ref"><a name="nph14557-bib-0029" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Gobler</span><span class="given-names" tagx="given-names" title="given-names">CJ</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Berry</span><span class="given-names" tagx="given-names" title="given-names">DL</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Dyhrman</span><span class="given-names" tagx="given-names" title="given-names">ST</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Wilhelm</span><span class="given-names" tagx="given-names" title="given-names">SW</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Salamov</span><span class="given-names" tagx="given-names" title="given-names">A</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Lobanov</span><span class="given-names" tagx="given-names" title="given-names">AV</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Zhang</span><span class="given-names" tagx="given-names" title="given-names">Y</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Collier</span><span class="given-names" tagx="given-names" title="given-names">JL</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Wurch</span><span class="given-names" tagx="given-names" title="given-names">LL</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Kustka</span><span class="given-names" tagx="given-names" title="given-names">AB</span></span><i>et al</i><span class="year" tagx="year" title="year">2011</span><span class="mixed-article-title" title="mixed-article-title">Niche of harmful alga <i>Aureococcus anophagefferens</i> revealed through ecogenomics</span>. <span class="source" tagx="source" title="source">Proceedings of the National Academy of Sciences, USA</span><span class="volume" tagx="volume" title="volume">108</span>: <span class="fpage" tagx="fpage" title="fpage">4352</span>–<span class="lpage" tagx="lpage" title="lpage">4357</span>.</span></li> <li tag="ref"><a name="nph14557-bib-0030" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Gremme</span><span class="given-names" tagx="given-names" title="given-names">G</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Steinbiss</span><span class="given-names" tagx="given-names" title="given-names">S</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Kurtz</span><span class="given-names" tagx="given-names" title="given-names">S</span></span>. <span class="year" tagx="year" title="year">2013</span><span class="mixed-article-title" title="mixed-article-title">GenomeTools: a comprehensive software library for efficient processing of structured genome annotations</span>. <span class="source" tagx="source" title="source">IEEE/ACM Transactions on Computational Biology and Bioinformatics</span><span class="volume" tagx="volume" title="volume">10</span>: <span class="fpage" tagx="fpage" title="fpage">645</span>–<span class="lpage" tagx="lpage" title="lpage">656</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24091398">24091398</a></span></span></li> <li tag="ref"><a name="nph14557-bib-0031" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Guillard</span><span class="given-names" tagx="given-names" title="given-names">RRL</span></span>. <span class="year" tagx="year" title="year">1975</span><span class="chapter-title" tagx="chapter-title" title="chapter-title">Culture of phytoplankton for feeding marine invertebrates</span> In: <span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Smith</span><span class="given-names" tagx="given-names" title="given-names">WL</span></span></span>, <span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Chanley</span><span class="given-names" tagx="given-names" title="given-names">MH</span></span></span>, eds. <span class="source" tagx="source" title="source">Culture of marine invertebrate animals</span>. <span class="publisher-loc" tagx="publisher-loc" title="publisher-loc">New York, NY, USA</span>: <span class="publisher-name" tagx="publisher-name" title="publisher-name">Plenum Press</span>, <span class="fpage" tagx="fpage" title="fpage">29</span>–<span class="lpage" tagx="lpage" title="lpage">60</span>.</span></li> <li tag="ref"><a name="nph14557-bib-0032" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Handley</span><span class="given-names" tagx="given-names" title="given-names">CJ</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Tuck Mok</span><span class="given-names" tagx="given-names" title="given-names">M</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Ilic</span><span class="given-names" tagx="given-names" title="given-names">MZ</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Adcocks</span><span class="given-names" tagx="given-names" title="given-names">C</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Buttle</span><span class="given-names" tagx="given-names" title="given-names">DJ</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Robinson</span><span class="given-names" tagx="given-names" title="given-names">HC</span></span>. <span class="year" tagx="year" title="year">2001</span><span class="mixed-article-title" title="mixed-article-title">Cathepsin D cleaves aggrecan at unique sites within the interglobular domain and chondroitin sulfate attachment regions that are also cleaved when cartilage is maintained at acid pH</span>. <span class="source" tagx="source" title="source">Matrix Biology</span><span class="volume" tagx="volume" title="volume">20</span>: <span class="fpage" tagx="fpage" title="fpage">543</span>–<span class="lpage" tagx="lpage" title="lpage">553</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11731271">11731271</a></span></span></li> <li tag="ref"><a name="nph14557-bib-0033" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Haudry</span><span class="given-names" tagx="given-names" title="given-names">A</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Platts</span><span class="given-names" tagx="given-names" title="given-names">AE</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Vello</span><span class="given-names" tagx="given-names" title="given-names">E</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Hoen</span><span class="given-names" tagx="given-names" title="given-names">DR</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Leclercq</span><span class="given-names" tagx="given-names" title="given-names">M</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Williamson</span><span class="given-names" tagx="given-names" title="given-names">RJ</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Forczek</span><span class="given-names" tagx="given-names" title="given-names">E</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Joly‐Lopez</span><span class="given-names" tagx="given-names" title="given-names">Z</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Steffen</span><span class="given-names" tagx="given-names" title="given-names">JG</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Hazzouri</span><span class="given-names" tagx="given-names" title="given-names">KM</span></span><i>et al</i><span class="year" tagx="year" title="year">2013</span><span class="mixed-article-title" title="mixed-article-title">An atlas of over 90,000 conserved noncoding sequences provides insight into crucifer regulatory regions</span>. <span class="source" tagx="source" title="source">Nature Genetics</span><span class="volume" tagx="volume" title="volume">45</span>: <span class="fpage" tagx="fpage" title="fpage">891</span>–<span class="lpage" tagx="lpage" title="lpage">898</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23817568">23817568</a></span></span></li> <li tag="ref"><a name="nph14557-bib-0034" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Huysman</span><span class="given-names" tagx="given-names" title="given-names">MJJ</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Vyverman</span><span class="given-names" tagx="given-names" title="given-names">W</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">De Veylder</span><span class="given-names" tagx="given-names" title="given-names">L</span></span>. <span class="year" tagx="year" title="year">2013</span><span class="mixed-article-title" title="mixed-article-title">Molecular regulation of the diatom cell cycle</span>. <span class="source" tagx="source" title="source">Journal of Experimental Botany</span><span class="volume" tagx="volume" title="volume">65</span>: <span class="fpage" tagx="fpage" title="fpage">2573</span>–<span class="lpage" tagx="lpage" title="lpage">2584</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24277280">24277280</a></span></span></li> <li tag="ref"><a name="nph14557-bib-0035" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Inada</span><span class="given-names" tagx="given-names" title="given-names">DC</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Bashir</span><span class="given-names" tagx="given-names" title="given-names">A</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Lee</span><span class="given-names" tagx="given-names" title="given-names">C</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Thomas</span><span class="given-names" tagx="given-names" title="given-names">BC</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Ko</span><span class="given-names" tagx="given-names" title="given-names">C</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Goff</span><span class="given-names" tagx="given-names" title="given-names">SA</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Freeling</span><span class="given-names" tagx="given-names" title="given-names">M</span></span>. <span class="year" tagx="year" title="year">2003</span><span class="mixed-article-title" title="mixed-article-title">Conserved noncoding sequences in the grasses</span>. <span class="source" tagx="source" title="source">Genome Research</span><span class="volume" tagx="volume" title="volume">13</span>: <span class="fpage" tagx="fpage" title="fpage">2030</span>–<span class="lpage" tagx="lpage" title="lpage">2041</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12952874">12952874</a></span></span></li> <li tag="ref"><a name="nph14557-bib-0036" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Katoh</span><span class="given-names" tagx="given-names" title="given-names">K</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Standley</span><span class="given-names" tagx="given-names" title="given-names">DM</span></span>. <span class="year" tagx="year" title="year">2013</span><span class="mixed-article-title" title="mixed-article-title">MAFFT multiple sequence alignment software version 7: improvements in performance and usability</span>. <span class="source" tagx="source" title="source">Molecular Biology and Evolution</span><span class="volume" tagx="volume" title="volume">30</span>: <span class="fpage" tagx="fpage" title="fpage">772</span>–<span class="lpage" tagx="lpage" title="lpage">780</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23329690">23329690</a></span></span></li> <li tag="ref"><a name="nph14557-bib-0037" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Kimura</span><span class="given-names" tagx="given-names" title="given-names">M</span></span>. <span class="year" tagx="year" title="year">1980</span><span class="mixed-article-title" title="mixed-article-title">A simple method for estimating evolutionary rates of base substitutions through comparative studies of nucleotide sequences</span>. <span class="source" tagx="source" title="source">Journal of Molecular Evolution</span><span class="volume" tagx="volume" title="volume">16</span>: <span class="fpage" tagx="fpage" title="fpage">111</span>–<span class="lpage" tagx="lpage" title="lpage">120</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/7463489">7463489</a></span></span></li> <li tag="ref"><a name="nph14557-bib-0038" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Kooistra</span><span class="given-names" tagx="given-names" title="given-names">WHCF</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Gersonde</span><span class="given-names" tagx="given-names" title="given-names">R</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Medlin</span><span class="given-names" tagx="given-names" title="given-names">LK</span></span>. <span class="year" tagx="year" title="year">2007</span><span class="chapter-title" tagx="chapter-title" title="chapter-title">The origin and evolution of the diatoms: their adaptation to a planktonic existence</span> In: <span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Falkowski</span><span class="given-names" tagx="given-names" title="given-names">PG</span></span></span>, <span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Knoll</span><span class="given-names" tagx="given-names" title="given-names">AH</span></span></span>, eds. <span class="source" tagx="source" title="source">Evolution of primary producers in the sea</span>. <span class="publisher-loc" tagx="publisher-loc" title="publisher-loc">Burlington, MA, USA</span>: <span class="publisher-name" tagx="publisher-name" title="publisher-name">Academic Press</span>, <span class="fpage" tagx="fpage" title="fpage">207</span>–<span class="lpage" tagx="lpage" title="lpage">249</span>.</span></li> <li tag="ref"><a name="nph14557-bib-0039" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Koonin</span><span class="given-names" tagx="given-names" title="given-names">EV</span></span>. <span class="year" tagx="year" title="year">2010</span><span class="mixed-article-title" title="mixed-article-title">The origin and early evolution of eukaryotes in the light of phylogenomics</span>. <span class="source" tagx="source" title="source">Genome Biology</span><span class="volume" tagx="volume" title="volume">11</span>: <span class="fpage" tagx="fpage" title="fpage">209</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20441612">20441612</a></span></span></li> <li tag="ref"><a name="nph14557-bib-0040" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Lévesque</span><span class="given-names" tagx="given-names" title="given-names">CA</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Brouwer</span><span class="given-names" tagx="given-names" title="given-names">H</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Cano</span><span class="given-names" tagx="given-names" title="given-names">L</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Hamilton</span><span class="given-names" tagx="given-names" title="given-names">JP</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Holt</span><span class="given-names" tagx="given-names" title="given-names">C</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Huitema</span><span class="given-names" tagx="given-names" title="given-names">E</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Raffaele</span><span class="given-names" tagx="given-names" title="given-names">S</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Robideau</span><span class="given-names" tagx="given-names" title="given-names">GP</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Thines</span><span class="given-names" tagx="given-names" title="given-names">M</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Win</span><span class="given-names" tagx="given-names" title="given-names">J</span></span><i>et al</i><span class="year" tagx="year" title="year">2010</span><span class="mixed-article-title" title="mixed-article-title">Genome sequence of the necrotrophic plant pathogen <i>Pythium ultimum</i> reveals original pathogenicity mechanisms and effector repertoire</span>. <span class="source" tagx="source" title="source">Genome Biology</span><span class="volume" tagx="volume" title="volume">11</span>: <span class="fpage" tagx="fpage" title="fpage">R73</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20626842">20626842</a></span></span></li> <li tag="ref"><a name="nph14557-bib-0041" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Lommer</span><span class="given-names" tagx="given-names" title="given-names">M</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Specht</span><span class="given-names" tagx="given-names" title="given-names">M</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Roy</span><span class="given-names" tagx="given-names" title="given-names">A‐S</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Kraemer</span><span class="given-names" tagx="given-names" title="given-names">L</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Andreson</span><span class="given-names" tagx="given-names" title="given-names">R</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Gutowska</span><span class="given-names" tagx="given-names" title="given-names">MA</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Wolf</span><span class="given-names" tagx="given-names" title="given-names">J</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Bergner</span><span class="given-names" tagx="given-names" title="given-names">SV</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Schilhabel</span><span class="given-names" tagx="given-names" title="given-names">MB</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Klostermeier</span><span class="given-names" tagx="given-names" title="given-names">UC</span></span><i>et al</i><span class="year" tagx="year" title="year">2012</span><span class="mixed-article-title" title="mixed-article-title">Genome and low‐iron response of an oceanic diatom adapted to chronic iron limitation</span>. <span class="source" tagx="source" title="source">Genome Biology</span><span class="volume" tagx="volume" title="volume">13</span>: <span class="fpage" tagx="fpage" title="fpage">R66</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22835381">22835381</a></span></span></li> <li tag="ref"><a name="nph14557-bib-0042" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Löytynoja</span><span class="given-names" tagx="given-names" title="given-names">A</span></span>. <span class="year" tagx="year" title="year">2014</span><span class="mixed-article-title" title="mixed-article-title">Phylogeny‐aware alignment with PRANK</span>. <span class="source" tagx="source" title="source">Methods in Molecular Biology</span><span class="volume" tagx="volume" title="volume">1079</span>: <span class="fpage" tagx="fpage" title="fpage">155</span>–<span class="lpage" tagx="lpage" title="lpage">170</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24170401">24170401</a></span></span></li> <li tag="ref"><a name="nph14557-bib-0043" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Mann</span><span class="given-names" tagx="given-names" title="given-names">DG</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Vanormelingen</span><span class="given-names" tagx="given-names" title="given-names">P</span></span>. <span class="year" tagx="year" title="year">2013</span><span class="mixed-article-title" title="mixed-article-title">An inordinate fondness? The number, distributions, and origins of diatom species</span>. <span class="source" tagx="source" title="source">Journal of Eukaryotic Microbiology</span><span class="volume" tagx="volume" title="volume">60</span>: <span class="fpage" tagx="fpage" title="fpage">414</span>–<span class="lpage" tagx="lpage" title="lpage">420</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23710621">23710621</a></span></span></li> <li tag="ref"><a name="nph14557-bib-0044" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Mapleson</span><span class="given-names" tagx="given-names" title="given-names">D</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Drou</span><span class="given-names" tagx="given-names" title="given-names">N</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Swarbreck</span><span class="given-names" tagx="given-names" title="given-names">D</span></span>. <span class="year" tagx="year" title="year">2015</span><span class="mixed-article-title" title="mixed-article-title">RAMPART: a workflow management system for de novo genome assembly</span>. <span class="source" tagx="source" title="source">Bioinformatics</span><span class="volume" tagx="volume" title="volume">31</span>: <span class="fpage" tagx="fpage" title="fpage">1824</span>–<span class="lpage" tagx="lpage" title="lpage">1826</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25637556">25637556</a></span></span></li> <li tag="ref"><a name="nph14557-bib-0045" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Mathelier</span><span class="given-names" tagx="given-names" title="given-names">A</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Zhao</span><span class="given-names" tagx="given-names" title="given-names">X</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Zhang</span><span class="given-names" tagx="given-names" title="given-names">AW</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Parcy</span><span class="given-names" tagx="given-names" title="given-names">F</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Worsley‐Hunt</span><span class="given-names" tagx="given-names" title="given-names">R</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Arenillas</span><span class="given-names" tagx="given-names" title="given-names">DJ</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Buchman</span><span class="given-names" tagx="given-names" title="given-names">S</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Chen</span><span class="given-names" tagx="given-names" title="given-names">C</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Chou</span><span class="given-names" tagx="given-names" title="given-names">A</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Ienasescu</span><span class="given-names" tagx="given-names" title="given-names">H</span></span><i>et al</i><span class="year" tagx="year" title="year">2014</span><span class="mixed-article-title" title="mixed-article-title">JASPAR 2014: an extensively expanded and updated open‐access database of transcription factor binding profiles</span>. <span class="source" tagx="source" title="source">Nucleic Acids Research</span><span class="volume" tagx="volume" title="volume">42</span>: <span class="fpage" tagx="fpage" title="fpage">D142</span>–<span class="lpage" tagx="lpage" title="lpage">D147</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24194598">24194598</a></span></span></li> <li tag="ref"><a name="nph14557-bib-0046" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Maumus</span><span class="given-names" tagx="given-names" title="given-names">F</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Allen</span><span class="given-names" tagx="given-names" title="given-names">AE</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Mhiri</span><span class="given-names" tagx="given-names" title="given-names">C</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Hu</span><span class="given-names" tagx="given-names" title="given-names">H</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Jabbari</span><span class="given-names" tagx="given-names" title="given-names">K</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Vardi</span><span class="given-names" tagx="given-names" title="given-names">A</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Grandbastien</span><span class="given-names" tagx="given-names" title="given-names">M‐A</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Bowler</span><span class="given-names" tagx="given-names" title="given-names">C</span></span>. <span class="year" tagx="year" title="year">2009</span><span class="mixed-article-title" title="mixed-article-title">Potential impact of stress activated retrotransposons on genome evolution in a marine diatom</span>. <span class="source" tagx="source" title="source">BMC Genomics</span><span class="volume" tagx="volume" title="volume">10</span>: <span class="fpage" tagx="fpage" title="fpage">624</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20028555">20028555</a></span></span></li> <li tag="ref"><a name="nph14557-bib-0047" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Mock</span><span class="given-names" tagx="given-names" title="given-names">T</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Otillar</span><span class="given-names" tagx="given-names" title="given-names">RP</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Strauss</span><span class="given-names" tagx="given-names" title="given-names">J</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">McMullan</span><span class="given-names" tagx="given-names" title="given-names">M</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Paajanen</span><span class="given-names" tagx="given-names" title="given-names">P</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Schmutz</span><span class="given-names" tagx="given-names" title="given-names">J</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Salamov</span><span class="given-names" tagx="given-names" title="given-names">A</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Sanges</span><span class="given-names" tagx="given-names" title="given-names">R</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Toseland</span><span class="given-names" tagx="given-names" title="given-names">A</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Ward</span><span class="given-names" tagx="given-names" title="given-names">BJ</span></span><i>et al</i><span class="year" tagx="year" title="year">2017</span><span class="mixed-article-title" title="mixed-article-title">Evolutionary genomics of the cold‐adapted diatom <i>Fragilariopsis cylindrus</i></span>. <span class="source" tagx="source" title="source">Nature</span><span class="volume" tagx="volume" title="volume">541</span>: <span class="fpage" tagx="fpage" title="fpage">536</span>–<span class="lpage" tagx="lpage" title="lpage">540</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/28092920">28092920</a></span></span></li> <li tag="ref"><a name="nph14557-bib-0048" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Moeys</span><span class="given-names" tagx="given-names" title="given-names">S</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Frenkel</span><span class="given-names" tagx="given-names" title="given-names">J</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Lembke</span><span class="given-names" tagx="given-names" title="given-names">C</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Gillard</span><span class="given-names" tagx="given-names" title="given-names">JTF</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Devos</span><span class="given-names" tagx="given-names" title="given-names">V</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">den Berge</span><span class="given-names" tagx="given-names" title="given-names">KV</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Bouillon</span><span class="given-names" tagx="given-names" title="given-names">B</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Huysman</span><span class="given-names" tagx="given-names" title="given-names">MJJ</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Decker</span><span class="given-names" tagx="given-names" title="given-names">SD</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Scharf</span><span class="given-names" tagx="given-names" title="given-names">J</span></span><i>et al</i><span class="year" tagx="year" title="year">2016</span><span class="mixed-article-title" title="mixed-article-title">A sex‐inducing pheromone triggers cell cycle arrest and mate attraction in the diatom <i>Seminavis robusta</i></span>. <span class="source" tagx="source" title="source">Scientific Reports</span><span class="volume" tagx="volume" title="volume">6</span>: <span class="fpage" tagx="fpage" title="fpage">19252</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/26786712">26786712</a></span></span></li> <li tag="ref"><a name="nph14557-bib-0049" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Montresor</span><span class="given-names" tagx="given-names" title="given-names">M</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Vitale</span><span class="given-names" tagx="given-names" title="given-names">L</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">D'Alelio</span><span class="given-names" tagx="given-names" title="given-names">D</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Ferrante</span><span class="given-names" tagx="given-names" title="given-names">MI</span></span>. <span class="year" tagx="year" title="year">2016</span><span class="mixed-article-title" title="mixed-article-title">Sex in marine planktonic diatoms: insights and challenges</span>. <span class="source" tagx="source" title="source">Perspectives in Phycology</span><span class="volume" tagx="volume" title="volume">3</span>: <span class="fpage" tagx="fpage" title="fpage">61</span>–<span class="lpage" tagx="lpage" title="lpage">75</span>.</span></li> <li tag="ref"><a name="nph14557-bib-0050" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Musacchia</span><span class="given-names" tagx="given-names" title="given-names">F</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Basu</span><span class="given-names" tagx="given-names" title="given-names">S</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Petrosino</span><span class="given-names" tagx="given-names" title="given-names">G</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Salvemini</span><span class="given-names" tagx="given-names" title="given-names">M</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Sanges</span><span class="given-names" tagx="given-names" title="given-names">R</span></span>. <span class="year" tagx="year" title="year">2015</span><span class="mixed-article-title" title="mixed-article-title">Annocript: a flexible pipeline for the annotation of transcriptomes able to identify putative long noncoding RNAs</span>. <span class="source" tagx="source" title="source">Bioinformatics</span><span class="volume" tagx="volume" title="volume">31</span>: <span class="fpage" tagx="fpage" title="fpage">2199</span>–<span class="lpage" tagx="lpage" title="lpage">2201</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25701574">25701574</a></span></span></li> <li tag="ref"><a name="nph14557-bib-0051" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Nosenko</span><span class="given-names" tagx="given-names" title="given-names">T</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Bhattacharya</span><span class="given-names" tagx="given-names" title="given-names">D</span></span>. <span class="year" tagx="year" title="year">2007</span><span class="mixed-article-title" title="mixed-article-title">Horizontal gene transfer in chromalveolates</span>. <span class="source" tagx="source" title="source">BMC Evolutionary Biology</span><span class="volume" tagx="volume" title="volume">7</span>: <span class="fpage" tagx="fpage" title="fpage">173</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17894863">17894863</a></span></span></li> <li tag="ref"><a name="nph14557-bib-0052" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Parra</span><span class="given-names" tagx="given-names" title="given-names">G</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Bradnam</span><span class="given-names" tagx="given-names" title="given-names">K</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Korf</span><span class="given-names" tagx="given-names" title="given-names">I</span></span>. <span class="year" tagx="year" title="year">2007</span><span class="mixed-article-title" title="mixed-article-title">CEGMA: a pipeline to accurately annotate core genes in eukaryotic genomes</span>. <span class="source" tagx="source" title="source">Bioinformatics</span><span class="volume" tagx="volume" title="volume">23</span>: <span class="fpage" tagx="fpage" title="fpage">1061</span>–<span class="lpage" tagx="lpage" title="lpage">1067</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17332020">17332020</a></span></span></li> <li tag="ref"><a name="nph14557-bib-0053" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Patil</span><span class="given-names" tagx="given-names" title="given-names">S</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Moeys</span><span class="given-names" tagx="given-names" title="given-names">S</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">von Dassow</span><span class="given-names" tagx="given-names" title="given-names">P</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Huysman</span><span class="given-names" tagx="given-names" title="given-names">MJJ</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Mapleson</span><span class="given-names" tagx="given-names" title="given-names">D</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">De Veylder</span><span class="given-names" tagx="given-names" title="given-names">L</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Sanges</span><span class="given-names" tagx="given-names" title="given-names">R</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Vyverman</span><span class="given-names" tagx="given-names" title="given-names">W</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Montresor</span><span class="given-names" tagx="given-names" title="given-names">M</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Ferrante</span><span class="given-names" tagx="given-names" title="given-names">MI</span></span>. <span class="year" tagx="year" title="year">2015</span><span class="mixed-article-title" title="mixed-article-title">Identification of the meiotic toolkit in diatoms and exploration of meiosis‐specific SPO11 and RAD51 homologs in the sexual species <i>Pseudo‐nitzschia multistriata</i> and <i>Seminavis robusta</i></span>. <span class="source" tagx="source" title="source">BMC Genomics</span><span class="volume" tagx="volume" title="volume">16</span>: <span class="fpage" tagx="fpage" title="fpage">930</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/26572248">26572248</a></span></span></li> <li tag="ref"><a name="nph14557-bib-0054" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Paul</span><span class="given-names" tagx="given-names" title="given-names">C</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Mausz</span><span class="given-names" tagx="given-names" title="given-names">MA</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Pohnert</span><span class="given-names" tagx="given-names" title="given-names">G</span></span>. <span class="year" tagx="year" title="year">2012</span><span class="mixed-article-title" title="mixed-article-title">A co‐culturing/metabolomics approach to investigate chemically mediated interactions of planktonic organisms reveals influence of bacteria on diatom metabolism</span>. <span class="source" tagx="source" title="source">Metabolomics</span><span class="volume" tagx="volume" title="volume">9</span>: <span class="fpage" tagx="fpage" title="fpage">349</span>–<span class="lpage" tagx="lpage" title="lpage">359</span>.</span></li> <li tag="ref"><a name="nph14557-bib-0055" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Perkins</span><span class="given-names" tagx="given-names" title="given-names">SL</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Schall</span><span class="given-names" tagx="given-names" title="given-names">J</span></span>. <span class="year" tagx="year" title="year">2002</span><span class="mixed-article-title" title="mixed-article-title">A molecular phylogeny of malarial parasites recovered from cytochrome <i>b</i> gene sequences</span>. <span class="source" tagx="source" title="source">Journal of Parasitology</span><span class="volume" tagx="volume" title="volume">88</span>: <span class="fpage" tagx="fpage" title="fpage">972</span>–<span class="lpage" tagx="lpage" title="lpage">978</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12435139">12435139</a></span></span></li> <li tag="ref"><a name="nph14557-bib-0056" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Piganeau</span><span class="given-names" tagx="given-names" title="given-names">G</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Vandepoele</span><span class="given-names" tagx="given-names" title="given-names">K</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Gourbière</span><span class="given-names" tagx="given-names" title="given-names">S</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Van de Peer</span><span class="given-names" tagx="given-names" title="given-names">Y</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Moreau</span><span class="given-names" tagx="given-names" title="given-names">H</span></span>. <span class="year" tagx="year" title="year">2009</span><span class="mixed-article-title" title="mixed-article-title">Unravelling cis‐regulatory elements in the genome of the smallest photosynthetic eukaryote: phylogenetic footprinting in <i>Ostreococcus</i></span>. <span class="source" tagx="source" title="source">Journal of Molecular Evolution</span><span class="volume" tagx="volume" title="volume">69</span>: <span class="fpage" tagx="fpage" title="fpage">249</span>–<span class="lpage" tagx="lpage" title="lpage">259</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19693423">19693423</a></span></span></li> <li tag="ref"><a name="nph14557-bib-0057" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Port</span><span class="given-names" tagx="given-names" title="given-names">JA</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Parker</span><span class="given-names" tagx="given-names" title="given-names">MS</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Kodner</span><span class="given-names" tagx="given-names" title="given-names">RB</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Wallace</span><span class="given-names" tagx="given-names" title="given-names">JC</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Armbrust</span><span class="given-names" tagx="given-names" title="given-names">EV</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Faustman</span><span class="given-names" tagx="given-names" title="given-names">EM</span></span>. <span class="year" tagx="year" title="year">2013</span><span class="mixed-article-title" title="mixed-article-title">Identification of G protein‐coupled receptor signaling pathway proteins in marine diatoms using comparative genomics</span>. <span class="source" tagx="source" title="source">BMC Genomics</span><span class="volume" tagx="volume" title="volume">14</span>: <span class="fpage" tagx="fpage" title="fpage">503</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23883327">23883327</a></span></span></li> <li tag="ref"><a name="nph14557-bib-0058" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Quesneville</span><span class="given-names" tagx="given-names" title="given-names">H</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Bergman</span><span class="given-names" tagx="given-names" title="given-names">CM</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Andrieu</span><span class="given-names" tagx="given-names" title="given-names">O</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Autard</span><span class="given-names" tagx="given-names" title="given-names">D</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Nouaud</span><span class="given-names" tagx="given-names" title="given-names">D</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Ashburner</span><span class="given-names" tagx="given-names" title="given-names">M</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Anxolabehere</span><span class="given-names" tagx="given-names" title="given-names">D</span></span>. <span class="year" tagx="year" title="year">2005</span><span class="mixed-article-title" title="mixed-article-title">Combined evidence annotation of transposable elements in genome sequences</span>. <span class="source" tagx="source" title="source">PLoS Computational Biology</span><span class="volume" tagx="volume" title="volume">1</span>: <span class="fpage" tagx="fpage" title="fpage">166</span>–<span class="lpage" tagx="lpage" title="lpage">175</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16110336">16110336</a></span></span></li> <li tag="ref"><a name="nph14557-bib-0059" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Quinlan</span><span class="given-names" tagx="given-names" title="given-names">AR</span></span>. <span class="year" tagx="year" title="year">2014</span><span class="mixed-article-title" title="mixed-article-title">BEDTools: the Swiss‐Army tool for genome feature analysis</span>. <span class="source" tagx="source" title="source">Current Protocols in Bioinformatics</span><span class="volume" tagx="volume" title="volume">47</span>: <span class="fpage" tagx="fpage" title="fpage">1</span>–<span class="lpage" tagx="lpage" title="lpage">34</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25199789">25199789</a></span></span></li> <li tag="ref"><a name="nph14557-bib-0060" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Rensing</span><span class="given-names" tagx="given-names" title="given-names">SA</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Lang</span><span class="given-names" tagx="given-names" title="given-names">D</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Zimmer</span><span class="given-names" tagx="given-names" title="given-names">AD</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Terry</span><span class="given-names" tagx="given-names" title="given-names">A</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Salamov</span><span class="given-names" tagx="given-names" title="given-names">A</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Shapiro</span><span class="given-names" tagx="given-names" title="given-names">H</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Nishiyama</span><span class="given-names" tagx="given-names" title="given-names">T</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Perroud</span><span class="given-names" tagx="given-names" title="given-names">P‐F</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Lindquist</span><span class="given-names" tagx="given-names" title="given-names">EA</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Kamisugi</span><span class="given-names" tagx="given-names" title="given-names">Y</span></span><i>et al</i><span class="year" tagx="year" title="year">2008</span><span class="mixed-article-title" title="mixed-article-title">The <i>Physcomitrella</i> genome reveals evolutionary insights into the conquest of land by plants</span>. <span class="source" tagx="source" title="source">Science</span><span class="volume" tagx="volume" title="volume">319</span>: <span class="fpage" tagx="fpage" title="fpage">64</span>–<span class="lpage" tagx="lpage" title="lpage">69</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18079367">18079367</a></span></span></li> <li tag="ref"><a name="nph14557-bib-0061" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Robinson</span><span class="given-names" tagx="given-names" title="given-names">MD</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">McCarthy</span><span class="given-names" tagx="given-names" title="given-names">DJ</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Smyth</span><span class="given-names" tagx="given-names" title="given-names">GK</span></span>. <span class="year" tagx="year" title="year">2010</span><span class="mixed-article-title" title="mixed-article-title">edgeR: a Bioconductor package for differential expression analysis of digital gene expression data</span>. <span class="source" tagx="source" title="source">Bioinformatics</span><span class="volume" tagx="volume" title="volume">26</span>: <span class="fpage" tagx="fpage" title="fpage">139</span>–<span class="lpage" tagx="lpage" title="lpage">140</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19910308">19910308</a></span></span></li> <li tag="ref"><a name="nph14557-bib-0062" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Sabatino</span><span class="given-names" tagx="given-names" title="given-names">V</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Russo</span><span class="given-names" tagx="given-names" title="given-names">MT</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Patil</span><span class="given-names" tagx="given-names" title="given-names">S</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">d'Ippolito</span><span class="given-names" tagx="given-names" title="given-names">G</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Fontana</span><span class="given-names" tagx="given-names" title="given-names">A</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Ferrante</span><span class="given-names" tagx="given-names" title="given-names">MI</span></span>. <span class="year" tagx="year" title="year">2015</span><span class="mixed-article-title" title="mixed-article-title">Establishment of genetic transformation in the sexually reproducing diatoms <i>Pseudo‐nitzschia multistriata</i> and <i>Pseudo‐nitzschia arenysensis</i> and inheritance of the transgene</span>. <span class="source" tagx="source" title="source">Marine Biotechnology</span><span class="volume" tagx="volume" title="volume">17</span>: <span class="fpage" tagx="fpage" title="fpage">452</span>–<span class="lpage" tagx="lpage" title="lpage">462</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25869745">25869745</a></span></span></li> <li tag="ref"><a name="nph14557-bib-0063" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Sanges</span><span class="given-names" tagx="given-names" title="given-names">R</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Hadzhiev</span><span class="given-names" tagx="given-names" title="given-names">Y</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Gueroult‐Bellone</span><span class="given-names" tagx="given-names" title="given-names">M</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Roure</span><span class="given-names" tagx="given-names" title="given-names">A</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Ferg</span><span class="given-names" tagx="given-names" title="given-names">M</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Meola</span><span class="given-names" tagx="given-names" title="given-names">N</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Amore</span><span class="given-names" tagx="given-names" title="given-names">G</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Basu</span><span class="given-names" tagx="given-names" title="given-names">S</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Brown</span><span class="given-names" tagx="given-names" title="given-names">ER</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">De Simone</span><span class="given-names" tagx="given-names" title="given-names">M</span></span><i>et al</i><span class="year" tagx="year" title="year">2013</span><span class="mixed-article-title" title="mixed-article-title">Highly conserved elements discovered in vertebrates are present in non‐syntenic loci of tunicates, act as enhancers and can be transcribed during development</span>. <span class="source" tagx="source" title="source">Nucleic Acids Research</span><span class="volume" tagx="volume" title="volume">41</span>: <span class="fpage" tagx="fpage" title="fpage">3600</span>–<span class="lpage" tagx="lpage" title="lpage">3618</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23393190">23393190</a></span></span></li> <li tag="ref"><a name="nph14557-bib-0064" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Sato</span><span class="given-names" tagx="given-names" title="given-names">S</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Beakes</span><span class="given-names" tagx="given-names" title="given-names">G</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Idei</span><span class="given-names" tagx="given-names" title="given-names">M</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Nagumo</span><span class="given-names" tagx="given-names" title="given-names">T</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Mann</span><span class="given-names" tagx="given-names" title="given-names">DG</span></span>. <span class="year" tagx="year" title="year">2011</span><span class="mixed-article-title" title="mixed-article-title">Novel sex cells and evidence for sex pheromones in diatoms</span>. <span class="source" tagx="source" title="source">PLoS ONE</span><span class="volume" tagx="volume" title="volume">6</span>: <span class="fpage" tagx="fpage" title="fpage">e26923</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22046412">22046412</a></span></span></li> <li tag="ref"><a name="nph14557-bib-0065" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Scalco</span><span class="given-names" tagx="given-names" title="given-names">E</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Amato</span><span class="given-names" tagx="given-names" title="given-names">A</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Ferrante</span><span class="given-names" tagx="given-names" title="given-names">MI</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Montresor</span><span class="given-names" tagx="given-names" title="given-names">M</span></span>. <span class="year" tagx="year" title="year">2016</span><span class="mixed-article-title" title="mixed-article-title">The sexual phase of the diatom <i>Pseudo‐nitzschia multistriata</i>: cytological and time‐lapse cinematography characterization</span>. <span class="source" tagx="source" title="source">Protoplasma</span><span class="volume" tagx="volume" title="volume">253</span>: <span class="fpage" tagx="fpage" title="fpage">1421</span>–<span class="lpage" tagx="lpage" title="lpage">1431</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/26494151">26494151</a></span></span></li> <li tag="ref"><a name="nph14557-bib-0066" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Scalco</span><span class="given-names" tagx="given-names" title="given-names">E</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Stec</span><span class="given-names" tagx="given-names" title="given-names">K</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Iudicone</span><span class="given-names" tagx="given-names" title="given-names">D</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Ferrante</span><span class="given-names" tagx="given-names" title="given-names">MI</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Montresor</span><span class="given-names" tagx="given-names" title="given-names">M</span></span>. <span class="year" tagx="year" title="year">2014</span><span class="mixed-article-title" title="mixed-article-title">The dynamics of sexual phase in the marine diatom <i>Pseudo‐nitzschia multistriata</i> (Bacillariophyceae)</span>. <span class="source" tagx="source" title="source">Journal of Phycology</span><span class="volume" tagx="volume" title="volume">50</span>: <span class="fpage" tagx="fpage" title="fpage">817</span>–<span class="lpage" tagx="lpage" title="lpage">828</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/26988637">26988637</a></span></span></li> <li tag="ref"><a name="nph14557-bib-0067" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Ševčíková</span><span class="given-names" tagx="given-names" title="given-names">T</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Horák</span><span class="given-names" tagx="given-names" title="given-names">A</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Klimeš</span><span class="given-names" tagx="given-names" title="given-names">V</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Zbránková</span><span class="given-names" tagx="given-names" title="given-names">V</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Demir‐Hilton</span><span class="given-names" tagx="given-names" title="given-names">E</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Sudek</span><span class="given-names" tagx="given-names" title="given-names">S</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Jenkins</span><span class="given-names" tagx="given-names" title="given-names">J</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Schmutz</span><span class="given-names" tagx="given-names" title="given-names">J</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Přibyl</span><span class="given-names" tagx="given-names" title="given-names">P</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Fousek</span><span class="given-names" tagx="given-names" title="given-names">J</span></span><i>et al</i><span class="year" tagx="year" title="year">2015</span><span class="mixed-article-title" title="mixed-article-title">Updating algal evolutionary relationships through plastid genome sequencing: did alveolate plastids emerge through endosymbiosis of an ochrophyte?</span><span class="source" tagx="source" title="source">Scientific Reports</span><span class="volume" tagx="volume" title="volume">5</span>: <span class="fpage" tagx="fpage" title="fpage">10134</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/26017773">26017773</a></span></span></li> <li tag="ref"><a name="nph14557-bib-0068" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Slater</span><span class="given-names" tagx="given-names" title="given-names">GSC</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Birney</span><span class="given-names" tagx="given-names" title="given-names">E</span></span>. <span class="year" tagx="year" title="year">2005</span><span class="mixed-article-title" title="mixed-article-title">Automated generation of heuristics for biological sequence comparison</span>. <span class="source" tagx="source" title="source">BMC Bioinformatics</span><span class="volume" tagx="volume" title="volume">6</span>: <span class="fpage" tagx="fpage" title="fpage">31</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15713233">15713233</a></span></span></li> <li tag="ref"><a name="nph14557-bib-0069" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Speir</span><span class="given-names" tagx="given-names" title="given-names">ML</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Zweig</span><span class="given-names" tagx="given-names" title="given-names">AS</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Rosenbloom</span><span class="given-names" tagx="given-names" title="given-names">KR</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Raney</span><span class="given-names" tagx="given-names" title="given-names">BJ</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Paten</span><span class="given-names" tagx="given-names" title="given-names">B</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Nejad</span><span class="given-names" tagx="given-names" title="given-names">P</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Lee</span><span class="given-names" tagx="given-names" title="given-names">BT</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Learned</span><span class="given-names" tagx="given-names" title="given-names">K</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Karolchik</span><span class="given-names" tagx="given-names" title="given-names">D</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Hinrichs</span><span class="given-names" tagx="given-names" title="given-names">AS</span></span><i>et al</i><span class="year" tagx="year" title="year">2016</span><span class="mixed-article-title" title="mixed-article-title">The UCSC Genome Browser database: 2016 update</span>. <span class="source" tagx="source" title="source">Nucleic Acids Research</span><span class="volume" tagx="volume" title="volume">44</span>: <span class="fpage" tagx="fpage" title="fpage">D717</span>–<span class="lpage" tagx="lpage" title="lpage">D725</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/26590259">26590259</a></span></span></li> <li tag="ref"><a name="nph14557-bib-0070" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Stamatakis</span><span class="given-names" tagx="given-names" title="given-names">A</span></span>. <span class="year" tagx="year" title="year">2006</span><span class="mixed-article-title" title="mixed-article-title">RAxML‐VI‐HPC: maximum likelihood‐based phylogenetic analyses with thousands of taxa and mixed models</span>. <span class="source" tagx="source" title="source">Bioinformatics</span><span class="volume" tagx="volume" title="volume">22</span>: <span class="fpage" tagx="fpage" title="fpage">2688</span>–<span class="lpage" tagx="lpage" title="lpage">2690</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16928733">16928733</a></span></span></li> <li tag="ref"><a name="nph14557-bib-0071" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Stanke</span><span class="given-names" tagx="given-names" title="given-names">M</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Schöffmann</span><span class="given-names" tagx="given-names" title="given-names">O</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Morgenstern</span><span class="given-names" tagx="given-names" title="given-names">B</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Waack</span><span class="given-names" tagx="given-names" title="given-names">S</span></span>. <span class="year" tagx="year" title="year">2006</span><span class="mixed-article-title" title="mixed-article-title">Gene prediction in eukaryotes with a generalized hidden Markov model that uses hints from external sources</span>. <span class="source" tagx="source" title="source">BMC Bioinformatics</span><span class="volume" tagx="volume" title="volume">7</span>: <span class="fpage" tagx="fpage" title="fpage">62</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16469098">16469098</a></span></span></li> <li tag="ref"><a name="nph14557-bib-0072" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Swanson</span><span class="given-names" tagx="given-names" title="given-names">WJ</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Vacquier</span><span class="given-names" tagx="given-names" title="given-names">VD</span></span>. <span class="year" tagx="year" title="year">2002</span><span class="mixed-article-title" title="mixed-article-title">Reproductive protein evolution</span>. <span class="source" tagx="source" title="source">Annual Review of Ecology and Systematics</span><span class="volume" tagx="volume" title="volume">33</span>: <span class="fpage" tagx="fpage" title="fpage">161</span>–<span class="lpage" tagx="lpage" title="lpage">179</span>.</span></li> <li tag="ref"><a name="nph14557-bib-0073" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Tanaka</span><span class="given-names" tagx="given-names" title="given-names">T</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Maeda</span><span class="given-names" tagx="given-names" title="given-names">Y</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Veluchamy</span><span class="given-names" tagx="given-names" title="given-names">A</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Tanaka</span><span class="given-names" tagx="given-names" title="given-names">M</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Abida</span><span class="given-names" tagx="given-names" title="given-names">H</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Maréchal</span><span class="given-names" tagx="given-names" title="given-names">E</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Bowler</span><span class="given-names" tagx="given-names" title="given-names">C</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Muto</span><span class="given-names" tagx="given-names" title="given-names">M</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Sunaga</span><span class="given-names" tagx="given-names" title="given-names">Y</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Tanaka</span><span class="given-names" tagx="given-names" title="given-names">M</span></span><i>et al</i><span class="year" tagx="year" title="year">2015</span><span class="mixed-article-title" title="mixed-article-title">Oil accumulation by the oleaginous diatom <i>Fistulifera solaris</i> as revealed by the genome and transcriptome</span>. <span class="source" tagx="source" title="source">Plant Cell</span><span class="volume" tagx="volume" title="volume">27</span>: <span class="fpage" tagx="fpage" title="fpage">162</span>–<span class="lpage" tagx="lpage" title="lpage">176</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25634988">25634988</a></span></span></li> <li tag="ref"><a name="nph14557-bib-0074" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Trainer</span><span class="given-names" tagx="given-names" title="given-names">VL</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Bates</span><span class="given-names" tagx="given-names" title="given-names">SS</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Lundholm</span><span class="given-names" tagx="given-names" title="given-names">N</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Thessen</span><span class="given-names" tagx="given-names" title="given-names">AE</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Cochlan</span><span class="given-names" tagx="given-names" title="given-names">WP</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Adams</span><span class="given-names" tagx="given-names" title="given-names">NG</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Trick</span><span class="given-names" tagx="given-names" title="given-names">CG</span></span>. <span class="year" tagx="year" title="year">2012</span><span class="mixed-article-title" title="mixed-article-title"><i>Pseudo‐nitzschia</i> physiological ecology, phylogeny, toxicity, monitoring and impacts on ecosystem health</span>. <span class="source" tagx="source" title="source">Harmful Algae</span><span class="volume" tagx="volume" title="volume">14</span>: <span class="fpage" tagx="fpage" title="fpage">271</span>–<span class="lpage" tagx="lpage" title="lpage">300</span>.</span></li> <li tag="ref"><a name="nph14557-bib-0075" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Vavouri</span><span class="given-names" tagx="given-names" title="given-names">T</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Walter</span><span class="given-names" tagx="given-names" title="given-names">K</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Gilks</span><span class="given-names" tagx="given-names" title="given-names">WR</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Lehner</span><span class="given-names" tagx="given-names" title="given-names">B</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Elgar</span><span class="given-names" tagx="given-names" title="given-names">G</span></span>. <span class="year" tagx="year" title="year">2007</span><span class="mixed-article-title" title="mixed-article-title">Parallel evolution of conserved non‐coding elements that target a common set of developmental regulatory genes from worms to humans</span>. <span class="source" tagx="source" title="source">Genome Biology</span><span class="volume" tagx="volume" title="volume">8</span>: <span class="fpage" tagx="fpage" title="fpage">R15</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17274809">17274809</a></span></span></li> <li tag="ref"><a name="nph14557-bib-0076" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Wang</span><span class="given-names" tagx="given-names" title="given-names">D</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Zhang</span><span class="given-names" tagx="given-names" title="given-names">Y</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Zhang</span><span class="given-names" tagx="given-names" title="given-names">Z</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Zhu</span><span class="given-names" tagx="given-names" title="given-names">J</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Yu</span><span class="given-names" tagx="given-names" title="given-names">J</span></span>. <span class="year" tagx="year" title="year">2010</span><span class="mixed-article-title" title="mixed-article-title">KaKs_Calculator 2.0: a toolkit incorporating gamma‐series methods and sliding window strategies</span>. <span class="source" tagx="source" title="source">Genomics, Proteomics &amp;amp; Bioinformatics</span><span class="volume" tagx="volume" title="volume">8</span>: <span class="fpage" tagx="fpage" title="fpage">77</span>–<span class="lpage" tagx="lpage" title="lpage">80</span>.</span></li> <li tag="ref"><a name="nph14557-bib-0078" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Wilkinson</span><span class="given-names" tagx="given-names" title="given-names">LE</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Pringle</span><span class="given-names" tagx="given-names" title="given-names">JR</span></span>. <span class="year" tagx="year" title="year">1974</span><span class="mixed-article-title" title="mixed-article-title">Transient G1 arrest of <i>S. cerevisiae</i> cells of mating type alpha by a factor produced by cells of mating type <i>a</i></span>. <span class="source" tagx="source" title="source">Experimental Cell Research</span><span class="volume" tagx="volume" title="volume">89</span>: <span class="fpage" tagx="fpage" title="fpage">175</span>–<span class="lpage" tagx="lpage" title="lpage">187</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/4611778">4611778</a></span></span></li> <li tag="ref"><a name="nph14557-bib-0079" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Wilson</span><span class="given-names" tagx="given-names" title="given-names">D</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Pethica</span><span class="given-names" tagx="given-names" title="given-names">R</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Zhou</span><span class="given-names" tagx="given-names" title="given-names">Y</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Talbot</span><span class="given-names" tagx="given-names" title="given-names">C</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Vogel</span><span class="given-names" tagx="given-names" title="given-names">C</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Madera</span><span class="given-names" tagx="given-names" title="given-names">M</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Chothia</span><span class="given-names" tagx="given-names" title="given-names">C</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Gough</span><span class="given-names" tagx="given-names" title="given-names">J</span></span>. <span class="year" tagx="year" title="year">2009</span><span class="mixed-article-title" title="mixed-article-title">SUPERFAMILY – sophisticated comparative genomics, data mining, visualization and phylogeny</span>. <span class="source" tagx="source" title="source">Nucleic Acids Research</span><span class="volume" tagx="volume" title="volume">37</span>: <span class="fpage" tagx="fpage" title="fpage">D380</span>–<span class="lpage" tagx="lpage" title="lpage">D386</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19036790">19036790</a></span></span></li> <li tag="ref"><a name="nph14557-bib-0081" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Woolfe</span><span class="given-names" tagx="given-names" title="given-names">A</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Goodson</span><span class="given-names" tagx="given-names" title="given-names">M</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Goode</span><span class="given-names" tagx="given-names" title="given-names">DK</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Snell</span><span class="given-names" tagx="given-names" title="given-names">P</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">McEwen</span><span class="given-names" tagx="given-names" title="given-names">GK</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Vavouri</span><span class="given-names" tagx="given-names" title="given-names">T</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Smith</span><span class="given-names" tagx="given-names" title="given-names">SF</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">North</span><span class="given-names" tagx="given-names" title="given-names">P</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Callaway</span><span class="given-names" tagx="given-names" title="given-names">H</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Kelly</span><span class="given-names" tagx="given-names" title="given-names">K</span></span><i>et al</i><span class="year" tagx="year" title="year">2005</span><span class="mixed-article-title" title="mixed-article-title">Highly conserved non‐coding sequences are associated with vertebrate development</span>. <span class="source" tagx="source" title="source">PLoS Biology</span><span class="volume" tagx="volume" title="volume">3</span>: <span class="fpage" tagx="fpage" title="fpage">e7</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15630479">15630479</a></span></span></li> <li tag="ref"><a name="nph14557-bib-0082" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Yang</span><span class="given-names" tagx="given-names" title="given-names">EC</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Boo</span><span class="given-names" tagx="given-names" title="given-names">GH</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Kim</span><span class="given-names" tagx="given-names" title="given-names">HJ</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Cho</span><span class="given-names" tagx="given-names" title="given-names">SM</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Boo</span><span class="given-names" tagx="given-names" title="given-names">SM</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Andersen</span><span class="given-names" tagx="given-names" title="given-names">RA</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Yoon</span><span class="given-names" tagx="given-names" title="given-names">HS</span></span>. <span class="year" tagx="year" title="year">2012</span><span class="mixed-article-title" title="mixed-article-title">Supermatrix data highlight the phylogenetic relationships of photosynthetic stramenopiles</span>. <span class="source" tagx="source" title="source">Protist</span><span class="volume" tagx="volume" title="volume">163</span>: <span class="fpage" tagx="fpage" title="fpage">217</span>–<span class="lpage" tagx="lpage" title="lpage">231</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22001261">22001261</a></span></span></li> <li tag="ref"><a name="nph14557-bib-0083" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Yang</span><span class="given-names" tagx="given-names" title="given-names">Z</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Bielawski</span><span class="given-names" tagx="given-names" title="given-names">JP</span></span>. <span class="year" tagx="year" title="year">2000</span><span class="mixed-article-title" title="mixed-article-title">Statistical methods for detecting molecular adaptation</span>. <span class="source" tagx="source" title="source">Trends in Ecology &amp;amp; Evolution</span><span class="volume" tagx="volume" title="volume">15</span>: <span class="fpage" tagx="fpage" title="fpage">496</span>–<span class="lpage" tagx="lpage" title="lpage">503</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11114436">11114436</a></span></span></li> <li tag="ref"><a name="nph14557-bib-0084" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Ye</span><span class="given-names" tagx="given-names" title="given-names">N</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Zhang</span><span class="given-names" tagx="given-names" title="given-names">X</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Miao</span><span class="given-names" tagx="given-names" title="given-names">M</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Fan</span><span class="given-names" tagx="given-names" title="given-names">X</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Zheng</span><span class="given-names" tagx="given-names" title="given-names">Y</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Xu</span><span class="given-names" tagx="given-names" title="given-names">D</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Wang</span><span class="given-names" tagx="given-names" title="given-names">J</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Zhou</span><span class="given-names" tagx="given-names" title="given-names">L</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Wang</span><span class="given-names" tagx="given-names" title="given-names">D</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Gao</span><span class="given-names" tagx="given-names" title="given-names">Y</span></span><i>et al</i><span class="year" tagx="year" title="year">2015</span><span class="mixed-article-title" title="mixed-article-title"><i>Saccharina</i> genomes provide novel insight into kelp biology</span>. <span class="source" tagx="source" title="source">Nature Communications</span><span class="volume" tagx="volume" title="volume">6</span>: <span class="fpage" tagx="fpage" title="fpage">6986</span>.</span></li> </ul> </div> </div>  </body></html>