<html xmlns="http://www.w3.org/1999/xhtml"> <head> <style type="text/css">
      	     		 
      	    a                          {background : #ffffff; }
      	    article                    {border-style : dotted; border-width : 2px; }
		 	
		 	div                        {background : #ffffcc;}
		 	div.abstract-title         {font-weight : bold ; font-size : 16pt;}
		 	div.ack                    {border-style : solid ; border-color : red; margin : 2em; }
		 	div.article-meta           {border-style : solid; border-width : 1 pt; margin 1em;}
		 	div.boxed-text             {margin : 5em; border-style : solid;}
		 	div.contrib-group          {margin : 1em; }
		 	div.fig                    {border-style : solid; border-width : 2px; margin : 2em; }
		 	div.funding                {font-weight : bold ; font-size : 16pt;}
		 	div.given-names            {font-style : italic;}
		 	div.intro                  {border-style : inset; margin : 5px;}
		 	div.introduction           {border-style : inset; margin : 5px;}
		 	div.journal-meta           {border-style : solid; border-width : 1 pt; margin 1em;}
		 	div.journal-title-group    {background : #ffeeee;}
		 	div.article-title          {font-weight : bold ; font-size : 18pt;}
		 	div.kwd                    {font-style : italic}
		 	div.meta-name              {font-weight : bold ; font-size : 16pt;}
		 	div.name                   {font-weight : bold;}
		 	div.alt-title              {font-style : italic; font-size : 12px;}
		 	
		 	div.sec                    {border : 2px; margin 5px; padding 2px;}
		 	div.title                  {font-family : courier; font-weight : bold;
		 	                            font-size : 16pt; margin : 5px;}
		 	
		 	div.materials_methods      {border-style : double; margin : 5px; }
		 	div.methods                {border-style : double; margin : 5px; }
		 	
		 	div.results                {border-style : solid; margin : 5px;}
		 	div.background             {border-style : dotted; margin : 5px;}
		 	
		 	div.discussion             {border-style : groove; margin : 5px;}
		 	div.conclusion             {border-style : ridge; margin : 5px;}
		 	div.conclusions             {border-style : ridge; margin : 5px;}
		 	div.supplementary-material {border-style : inset; margin : 5px;}
		 	div.abbreviations          {border-style : double; border-color : red; margin : 5px;}
		 	div.competinginterests     {border-style : double; border-color : blue; margin : 5px;}
		 	div.acknowledgements       {border-style : double; border-color : green; margin : 5px;}
		 	div.authors_contributions   {border-style : double; border-color : purple; margin : 5px;}
		 	
		 	div.publisher              {border-style : outset; margin : 5px;}
		 	div.fn-type-conflict       {background : #f88; }
		 	div.fn-type-con            {background : #ddf; }
		 	div.fn-type-other          {background : #ddd; }
		 	
		 	div.unknown                {background : #ffd;
									 	  border-style : solid;
									 	  border-width : 1px;
									 	  padding : 2 px;}
		 	  
      	    table                      {background : #ffffdd;}
		 	tr                         {background : #ddddff; padding : 1px;}
		 	
		 	span                       {background : #ffcccc;}
		 	
		 	span.citation-author       {font-family : helvetica; background : #ffeeee;}
		 	span.collab                {background : #ddffff; }
		 	span.comment               {font-family : courier; font-size : 6px; background : #ffaaff;}
		 	span.contrib               {background : #ffffff;}
		 	span.corresp               {background : #ddffdd; }
		 	span.doi                   {background : #ffffff;}
		 	span.email                 {font-family : courier; }
		 	span.etal                  {font-style : italic;}
		 	span.fpage                 {font-family : courier;}
		 	span.given-names           {background : #ffffff;}
		 	span.iso-abbrev            {background : #ffffff;}
		 	span.issn-epub             {background : #ffffff;}
		 	span.issn-ppub             {background : #ffffff;}
		 	span.journal-title         {background : #ffffff;}
		 	span.lpage                 {font-family : courier;}
		 	span.mixed-article-title   {font-style : italic ;}
		 	span.nlm-ta                {background : #ffffff;}
		 	span.pmc                   {background : #ffffff;}
		 	span.pmcid                 {background : #ffffff;}
		 	span.pmid                  {background : #ffffff;}
		 	span.publisher             {background : #ffffff;}
		 	span.publisher-id          {background : #ffffff;}
		 	span.publisher-name        {background : #ffffff;}
		 	span.source                {background : #ffffff;}
		 	span.subject               {background : #ffffff;}
		 	span.surname               {background : #ffffff;}
		 	span.volume                {font-family : courier; font-weight : bold;}
		 	span.year                  {font-family : courier ; font-style : italic;}
			</style> </head> <body> <div class="front" title="front"> <div class="journal-meta" tagx="journal-meta" title="journal-meta"><span class="nlm-ta" title="nlm-ta">eLife</span><span class="iso-abbrev" title="iso-abbrev">Elife</span><span class="publisher-id" title="publisher-id">eLife</span><div class="journal-title-group" tagx="journal-title-group" title="journal-title-group"><span class="journal-title" tagx="journal-title" title="journal-title">eLife</span></div><span class="issn-epub" tagx="issn" title="issn-epub">2050-084X</span><div class="publisher" tagx="publisher" title="publisher"><span class="publisher-name" tagx="publisher-name" title="publisher-name">eLife Sciences Publications, Ltd</span></div> </div> <div class="article-meta" tagx="article-meta" title="article-meta"><span class="pmid" title="pmid"> pmid: <a href="http://www.ncbi.nlm.nih.gov/pubmed/31647412">31647412</a></span><span class="publisher-id" title="publisher-id">49801</span><span class="doi" title="doi"> doi: <a href="https://dx.doi.org/10.7554/eLife.49801">10.7554/eLife.49801</a></span><div class="article-categories" title="article-categories">
: <span class="subject" title="subject">Research Article</span>: <span class="subject" title="subject">Cell Biology</span>: <span class="subject" title="subject">Evolutionary Biology</span></div> <div class="title-group" tagx="title-group" title="title-group"> <div class="article-title" title="article-title">
A unicellular relative of animals generates a layer of polarized cells by actomyosin-dependent cellularization</div> </div> <div class="contrib-group" title="contrib-group"><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Dudin</span><span class="given-names" tagx="given-names" title="given-names">Omaya</span></span><span class="contrib-id" tagx="contrib-id" title="contrib-id">https://orcid.org/0000-0002-6673-3149</span><span class="email" tagx="email" title="email">omaya.dudin@outlook.com</span><a href="#aff1">1</a><a href="#equal-contrib1">†</a><a href="#fund5" /><a href="#fund6" /><a href="#con1" /><a href="#conf1" /></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ondracka</span><span class="given-names" tagx="given-names" title="given-names">Andrej</span></span><span class="contrib-id" tagx="contrib-id" title="contrib-id">https://orcid.org/0000-0003-4193-6027</span><a href="#aff1">1</a><a href="#equal-contrib1">†</a><a href="#fund7" /><a href="#con2" /><a href="#conf1" /></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Grau-Bové</span><span class="given-names" tagx="given-names" title="given-names">Xavier</span></span><span class="contrib-id" tagx="contrib-id" title="contrib-id">http://orcid.org/0000-0003-1978-5824</span><a href="#aff1">1</a><a href="#aff2">2</a><a href="#con3" /><a href="#conf1" /></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Haraldsen</span><span class="given-names" tagx="given-names" title="given-names">Arthur AB</span></span><a href="#aff3">3</a><a href="#con4" /><a href="#conf1" /></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Toyoda</span><span class="given-names" tagx="given-names" title="given-names">Atsushi</span></span><span class="contrib-id" tagx="contrib-id" title="contrib-id">http://orcid.org/0000-0002-0728-7548</span><a href="#aff4">4</a><a href="#fund2" /><a href="#con5" /><a href="#conf1" /></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Suga</span><span class="given-names" tagx="given-names" title="given-names">Hiroshi</span></span><a href="#aff5">5</a><a href="#fund3" /><a href="#con6" /><a href="#conf1" /></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bråte</span><span class="given-names" tagx="given-names" title="given-names">Jon</span></span><span class="contrib-id" tagx="contrib-id" title="contrib-id">http://orcid.org/0000-0003-0490-1175</span><a href="#aff3">3</a><a href="#fund4" /><a href="#con7" /><a href="#conf1" /></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ruiz-Trillo</span><span class="given-names" tagx="given-names" title="given-names">Iñaki</span></span><span class="contrib-id" tagx="contrib-id" title="contrib-id">https://orcid.org/0000-0001-6547-5304</span><span class="email" tagx="email" title="email">inaki.ruiz@ibe.upf-csic.es</span><a href="#aff1">1</a><a href="#aff6">6</a><a href="#aff7">7</a><a href="#fund1" /><a href="#con8" /><a href="#conf1" /></span><span class="citation_author_institution" id="aff1">[1], <span class="institution" title="institution">Institut de Biologia Evolutiva (CSIC-Universitat Pompeu Fabra)</span><span class="addr-line" title="addr-line">Barcelona</span><span class="country" tagx="country" title="country">Spain</span></span><span class="citation_author_institution" id="aff2">[2], <span class="institution" title="institution">Department of Vector Biology</span><span class="institution" title="institution">Liverpool School of Tropical Medicine</span><span class="addr-line" title="addr-line">Liverpool</span><span class="country" tagx="country" title="country">United Kingdom</span></span><span class="citation_author_institution" id="aff3">[3], <span class="institution" title="institution">Section for Genetics and Evolutionary Biology (EVOGENE), Department of Biosciences</span><span class="institution" title="institution">University of Oslo</span><span class="addr-line" title="addr-line">Oslo</span><span class="country" tagx="country" title="country">Norway</span></span><span class="citation_author_institution" id="aff4">[4], <span class="institution" title="institution">Department of Genomics and Evolutionary Biology</span><span class="institution" title="institution">National Institute of Genetics</span><span class="addr-line" title="addr-line">Mishima</span><span class="country" tagx="country" title="country">Japan</span></span><span class="citation_author_institution" id="aff5">[5], <span class="institution" title="institution">Faculty of Life and Environmental Sciences</span><span class="institution" title="institution">Prefectural University of Hiroshima</span><span class="addr-line" title="addr-line">Hiroshima</span><span class="country" tagx="country" title="country">Japan</span></span><span class="citation_author_institution" id="aff6">[6], <span class="institution" title="institution">Departament de Genètica, Microbiologia i Estadística</span><span class="institution" title="institution">Universitat de Barcelona</span><span class="addr-line" title="addr-line">Barcelona</span><span class="country" tagx="country" title="country">Spain</span></span><span class="citation_author_institution" id="aff7">[7], <span class="institution" title="institution">ICREA</span><span class="addr-line" title="addr-line">Barcelona</span><span class="country" tagx="country" title="country">Spain</span></span></div> <div class="contrib-group" title="contrib-group"><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Thattai</span><span class="given-names" tagx="given-names" title="given-names">Mukund</span></span><span class="role" tagx="role" title="role">Reviewing Editor</span><span class="citation_author_institution" id="">[], <span class="institution" title="institution">National Centre for Biological Sciences</span><span class="country" tagx="country" title="country">India</span></span></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">VijayRaghavan</span><span class="given-names" tagx="given-names" title="given-names">K</span></span><span class="role" tagx="role" title="role">Senior Editor</span><span class="citation_author_institution" id="">[], <span class="institution" title="institution">National Centre for Biological Sciences, Tata Institute of Fundamental Research</span><span class="country" tagx="country" title="country">India</span></span></span></div> <div class="author-notes" title="author-notes"> <div class="fn-type-con" title="con"><span class="label" tagx="label" title="label">†</span><p>These authors contributed equally to this work.</p> </div> </div><span class="pub-date-" title="pub-date-">: <span>2019-10-10</span></span><span class="pub-date-collection" title="pub-date-collection">collection: <span>2019</span></span><span class="volume" tagx="volume" title="volume">8</span><span class="elocation-id" tagx="elocation-id" title="elocation-id">e49801</span><span class="history" title="history"><span class="received" title="received">received: 2019-6-29</span><span class="accepted" title="accepted">accepted: 2019-10-23</span></span><div class="permissions"><span class="copyright" title="copyright">(C) <span class="copyright-holder" tagx="copyright-holder" title="copyright-holder">Dudin et al</span>, <span class="copyright-year" tagx="copyright-year" title="copyright-year">2019</span></span><span class="license" title="license"> <div tag="license_ref">
http://creativecommons.org/licenses/by/4.0/</div><span class="license-p" title="license-p">This article is distributed under the terms of the <a href="http://creativecommons.org/licenses/by/4.0/">Creative Commons Attribution License</a>, which permits unrestricted use and redistribution provided that the original author and source are credited.</span></span></div><a class="self-uri" href="elife-49801.pdf" title="self-uri">elife-49801.pdf</a><span class="related-article" tagx="related-article" title="related-article" /><div class="abstract" title="abstract"> <div class="abstract-title" title="abstract-title">
Abstract</div> <p>In animals, cellularization of a coenocyte is a specialized form of cytokinesis that results in the formation of a polarized epithelium during early embryonic development. It is characterized by coordinated assembly of an actomyosin network, which drives inward membrane invaginations. However, whether coordinated cellularization driven by membrane invagination exists outside animals is not known. To that end, we investigate cellularization in the ichthyosporean <i>Sphaeroforma arctica</i>, a close unicellular relative of animals. We show that the process of cellularization involves coordinated inward plasma membrane invaginations dependent on an actomyosin network and reveal the temporal order of its assembly. This leads to the formation of a polarized layer of cells resembling an epithelium. We show that this stage is associated with tightly regulated transcriptional activation of genes involved in cell adhesion. Hereby we demonstrate the presence of a self-organized, clonally-generated, polarized layer of cells in a unicellular relative of animals.</p> </div> <div class="kwd-group"> <div class="kwd" title="kwd">
cellularization</div> <div class="kwd" title="kwd">
coenocyte</div> <div class="kwd" title="kwd">
evolution</div> <div class="kwd" title="kwd">
epithelium</div> <div class="kwd" title="kwd">
actin</div> <div class="kwd" title="kwd">
multicellularity</div> </div> <div class="kwd-group"> <div class="title" tagx="title" title="title">
Research organism</div> <div class="kwd" title="kwd">
Other</div> </div> <div class="funding-group" title="funding-group"> <div class="funding" title="funding">
Funding</div> <div class="award-group" title="award-group"> <div class="funding-source" title="funding-source"> <div class="institution-wrap" title="institution-wrap"><span class="institution-id" title="institution-id">http://dx.doi.org/10.13039/501100000781</span><span class="institution" title="institution">European Research Council</span></div> </div><span class="award-id" tagx="award-id" title="award-id">Consolidator Grant ERC-2012-Co-616960</span><span class="principal-award-recipient" title="principal-award-recipient"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ruiz-Trillo</span><span class="given-names" tagx="given-names" title="given-names">Iñaki</span></span></span></div> <div class="award-group" title="award-group"> <div class="funding-source" title="funding-source"> <div class="institution-wrap" title="institution-wrap"><span class="institution-id" title="institution-id">http://dx.doi.org/10.13039/501100001700</span><span class="institution" title="institution">Ministry of Education, Culture, Sports, Science and Technology</span></div> </div><span class="award-id" tagx="award-id" title="award-id">MEXT KAKENHI 221S0002</span><span class="principal-award-recipient" title="principal-award-recipient"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Toyoda</span><span class="given-names" tagx="given-names" title="given-names">Atsushi</span></span></span></div> <div class="award-group" title="award-group"> <div class="funding-source" title="funding-source"> <div class="institution-wrap" title="institution-wrap"><span class="institution-id" title="institution-id">http://dx.doi.org/10.13039/501100001700</span><span class="institution" title="institution">Ministry of Education, Culture, Sports, Science and Technology</span></div> </div><span class="award-id" tagx="award-id" title="award-id">MEXT KAKENHI 26891021</span><span class="principal-award-recipient" title="principal-award-recipient"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Suga</span><span class="given-names" tagx="given-names" title="given-names">Hiroshi</span></span></span></div> <div class="award-group" title="award-group"> <div class="funding-source" title="funding-source"> <div class="institution-wrap" title="institution-wrap"><span class="institution-id" title="institution-id">http://dx.doi.org/10.13039/501100005416</span><span class="institution" title="institution">Research Council of Norway</span></div> </div><span class="award-id" tagx="award-id" title="award-id">Young Research Talents grant 240284</span><span class="principal-award-recipient" title="principal-award-recipient"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bråte</span><span class="given-names" tagx="given-names" title="given-names">Jon</span></span></span></div> <div class="award-group" title="award-group"> <div class="funding-source" title="funding-source"> <div class="institution-wrap" title="institution-wrap"><span class="institution-id" title="institution-id">http://dx.doi.org/10.13039/501100001711</span><span class="institution" title="institution">Swiss National Science Foundation</span></div> </div><span class="award-id" tagx="award-id" title="award-id">P2LAP3_171815</span><span class="principal-award-recipient" title="principal-award-recipient"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Dudin</span><span class="given-names" tagx="given-names" title="given-names">Omaya</span></span></span></div> <div class="award-group" title="award-group"> <div class="funding-source" title="funding-source"> <div class="institution-wrap" title="institution-wrap"><span class="institution-id" title="institution-id">http://dx.doi.org/10.13039/100010665</span><span class="institution" title="institution">H2020 Marie Skłodowska-Curie Actions</span></div> </div><span class="award-id" tagx="award-id" title="award-id">Individual fellowship MSCA-IF 746044</span><span class="principal-award-recipient" title="principal-award-recipient"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Dudin</span><span class="given-names" tagx="given-names" title="given-names">Omaya</span></span></span></div> <div class="award-group" title="award-group"> <div class="funding-source" title="funding-source"> <div class="institution-wrap" title="institution-wrap"><span class="institution-id" title="institution-id">http://dx.doi.org/10.13039/100010665</span><span class="institution" title="institution">H2020 Marie Skłodowska-Curie Actions</span></div> </div><span class="award-id" tagx="award-id" title="award-id">Individual fellowship MSCA-IF 747086</span><span class="principal-award-recipient" title="principal-award-recipient"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ondracka</span><span class="given-names" tagx="given-names" title="given-names">Andrej</span></span></span></div> <div class="funding-statement" title="funding-statement">
The funders had no role in study design, data collection and interpretation, or the decision to submit the work for publication.</div> </div> <div class="custom-meta-group" title="custom-meta-group"> <meta name="Author impact statement" value="Cellularization in Sphaeroforma arctica generates a self-organized structure that morphologically resembles an epithelium, and is associated with tightly regulated expression of cell adhesion pathways." /> </div> </div> </div> <div class="body" title="body"> <div class="intro" title="intro"> <div class="title" tagx="title" title="title">
Introduction</div> <p>Cellularization of a coenocyte — a multinucleate cell that forms through sequential nuclear divisions without accompanying cytokinesis — is a specialized form of coordinated cytokinesis that results in cleavage into individual cells. Cellularization commonly occurs during development of animals, plants and unicellular eukaryotes (<a href="#bib39">Hehenberger et al., 2012</a>; <a href="#bib61">Mazumdar and Mazumdar, 2002</a>). Despite the similarities, distinct mechanisms are involved in the cellularization of these coenocytes. During endosperm development in most flowering plants, coenocytes cellularize through cell wall formation around individual nuclei, forming a peripheral layer of cells surrounding a central vacuole (<a href="#bib39">Hehenberger et al., 2012</a>). This is coordinated by the radial microtubule system and is dependent on several microtubule-associated proteins (<a href="#bib72">Pignocchi et al., 2009</a>; <a href="#bib85">Sørensen et al., 2002</a>). In apicomplexan parasites, cytokinesis of multinucleate schizonts occurs by budding, which is driven by a polarized microtubule scaffold (<a href="#bib29">Francia and Striepen, 2014</a>).</p> <p>In contrast, in a model animal coenocyte, the syncytial blastoderm of the fruit fly <i>Drosophila melanogaster</i>, cellularization is accomplished through plasma membrane invaginations around equally spaced, cortically positioned nuclei (<a href="#bib27">Farrell and O'Farrell, 2014</a>; <a href="#bib61">Mazumdar and Mazumdar, 2002</a>). This process relies on extensive membrane remodeling (<a href="#bib55">Lecuit, 2004</a>; <a href="#bib84">Sokac and Wieschaus, 2008</a>; <a href="#bib28">Figard et al., 2016</a>) regulated by zygotically transcribed genes (<a href="#bib82">Schweisguth et al., 1990</a>; <a href="#bib80">Schejter and Wieschaus, 1993</a>; <a href="#bib73">Postner and Wieschaus, 1994</a>; <a href="#bib44">Hunter and Wieschaus, 2000</a>; <a href="#bib54">Lecuit et al., 2002</a>) and driven by microtubules and a contractile actomyosin network (<a href="#bib61">Mazumdar and Mazumdar, 2002</a>). It depends on several actin nucleators, such as the Arp2/3 complex (<a href="#bib86">Stevenson et al., 2002</a>) and formins (<a href="#bib3">Afshar et al., 2000</a>). It also requires multiple actin-binding proteins, including myosin II (<a href="#bib79">Royou et al., 2002</a>), which mediates actin cross-linking and contractility, as well as septins (<a href="#bib2">Adam et al., 2000</a>; <a href="#bib16">Cooper and Kiehart, 1996</a>), cofilin (<a href="#bib35">Gunsalus et al., 1995</a>) and profilin (<a href="#bib31">Giansanti et al., 1998</a>). In addition, it depends on cell-cell adhesion proteins including cadherin, and alpha- and beta-catenin (<a href="#bib44">Hunter and Wieschaus, 2000</a>; <a href="#bib90">Wang et al., 2004</a>). This coordinated cellularization results in the formation of a single layer of polarized epithelial tissue, also known as cellular blastoderm (<a href="#bib61">Mazumdar and Mazumdar, 2002</a>). This actomyosin-dependent cellularization is common in early insect embryos and also commonly observed in germline development of many animals (<a href="#bib38">Haglund et al., 2011</a>), such as the nematode <i>C. elegans</i> (<a href="#bib74">Priti et al., 2018</a>), however, whether this mechanism of cellularization is found outside animals, remains unknown.</p> <p>Among holozoans — a clade that includes animals and their closest unicellular relatives (<a href="#fig1">Figure 1A</a>) — ichthyosporeans are the only known lineage besides animals that forms coenocytes during their life cycles (<a href="#bib63">Mendoza et al., 2002</a>; <a href="#bib18">de Mendoza et al., 2015</a>). All characterized ichthyosporeans proliferate through rounds of nuclear divisions within a cell-walled coenocyte, followed by release of newborn cells (<a href="#bib59">Marshall and Berbee, 2011</a>; <a href="#bib60">Marshall and Berbee, 2013</a>; <a href="#bib87">Suga and Ruiz-Trillo, 2013</a>; <a href="#bib91">Whisler, 1968</a>). We have previously suggested that they undergo cellularization (<a href="#bib67">Ondracka et al., 2018</a>; <a href="#bib87">Suga and Ruiz-Trillo, 2013</a>). However, at present nothing is known about the ichthyosporean cellularization, and whether it involves animal-like mechanisms.</p> <div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/"><span class="label" tagx="label" title="label">Figure 1.</span></a></div>   <div class="title" tagx="title" title="title">
Cellularization dynamics in <i>Sphaeroforma arctica</i>.</div> <p>(<b>A</b>) Phylogenetic position of the ichthyosporean <i>Sphaeroforma arctica</i> in the tree of life. (<b>B</b>) Time-lapse images of the life-cycle of <i>S. arctica</i> show cell-size increase prior to cellularization and release of new-born cells. Associated with <a href="#video2">Video 2</a>. Bar, 50 µm. (<b>C</b>) Kymographs of 4 distinct cells undergoing cellularization with the time of flip (yellow) and cell release (orange) indicated for each. An example of the advancing dark front is shown for cell 4. Bar, 50 µm. (<b>D</b>) Mean cell diameter over time of 65 cell traces aligned to Flip reveals distinct cell stages: Growth, Pre and post-flip and cell release. (<b>E</b>) Duration of growth, cellularization and post-flip represented as box-plots (N°cells &amp;gt; 100). MAD (Median absolute deviation) over median is used as a measure of variability.</p>   </div> <p>Here, we comprehensively characterized cellularization in the ichthyosporean <i>Sphaeroforma arctica,</i> in which we have previously shown that coenocytic cycles can be synchronized (<a href="#bib67">Ondracka et al., 2018</a>). We used imaging, transcriptional profiling, and pharmacological inhibition, to study the gene expression dynamics, morphological rearrangements, and mechanisms of cellularization. We found that cellularization is accomplished by inward plasma membrane invaginations driven by an actomyosin network, forming a polarized layer of cells. Time-resolved transcriptomics revealed sharply regulated expression of cell polarity and cell adhesion genes during this stage. Finally, we show that this process depends on actin nucleators and Myosin II, and we reveal the temporal order of the actomyosin network assembly. Together, these findings establish that cellularization of ichthyosporeans proceeds by mechanism conserved between animals and ichthyosporeans.</p> </div> <div class="results" title="results"> <div class="title" tagx="title" title="title">
Results</div> <div class="growthandcellularizationins.arcticaaretemporallyseparatedstagesofthecoenocyticcycle" title="sec"> <div class="title" tagx="title" title="title">
Growth and cellularization in <i>S. arctica</i> are temporally separated stages of the coenocytic cycle</div> <p>To determine the timing of cellularization in synchronized cultures, we established long-term live imaging conditions. Individual coenocytes exhibit growth in cell size until approximately 60 hr, after which they undergo release of newborn cells (<a href="#fig1">Figure 1B</a>, <a href="#video1">Video 1</a>). This was consistent with previous results in bulk cultures (<a href="#bib67">Ondracka et al., 2018</a>), confirming that our experimental setup for long-term live imaging faithfully reproduces culture growth. However, by measuring the diameter of the coenocytes, we observed that newborn cell release occurred with somewhat variable timing (<a href="#fig1s1">Figure 1—figure supplement 1A</a>, <a href="#video1">Video 1</a>).</p> <div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/"><span class="label" tagx="label" title="label">Video 1.</span></a></div>   <div class="title" tagx="title" title="title">
Time lapse of synchronized cells of <i>S. arctica</i> obtained with epifluorescent microscopy.</div> <p>Time interval between frames is 20 min. The movie is played at 7fps. Four distinct cells can be seen undergoing a full life-cycle with the release of new born cells. Bar, 50 µm.</p>   </div> <p>Time-lapse imaging revealed that prior to cell release, the coenocytes darken along the periphery, and the dark front begins to advance towards the center (<a href="#video2">Video 2</a>). Afterwards, we observed an abrupt internal morphological change in the coenocyte, when the front disappears. We termed this event ‘flip’ (<a href="#video2">Video 2</a>). The flip occurred in all the coenocytes and can be reliably detected on kymographs (<a href="#fig1">Figure 1C</a>). Aligning individual coenocyte size traces to this specific common temporal marker, we observe that coenocytes stop growing approximately 9 hr before the flip (<a href="#fig1">Figure 1D and E</a> and <a href="#fig1s1">Figure 1—figure supplement 1B</a>). This shows that the growth stage and cellularization are temporally separated. The cellularization can be divided into a temporally less variable pre-flip phase (~9 hr) and a variable post-flip phase (<a href="#fig1">Figure 1E</a> and <a href="#fig1s1">Figure 1—figure supplement 1B and C</a>). Taken together, these results show that growth and cellularization form temporally distinct stages of the coenocytic cycle of <i>S. arctica.</i> This provides a temporal framework in which to characterize phenotypically distinct stages of cellularization.</p> <div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/"><span class="label" tagx="label" title="label">Video 2.</span></a></div>   <div class="title" tagx="title" title="title">
Time lapse of synchronized cells of <i>S. arctica</i> obtained with epifluorescent microscopy.</div> <p>Time interval between frames is 15 min. The movie is played at 7fps. Three cells can be seen undergoing flip prior to cell release whereas one cell undergo ‘flip’ without cell release during the course of the experiment. The exact timing of cell release is pointed out for each cell. Bar, 50 µm.</p>   </div> </div> <div class="thecorticalactinnetworkestablishessitesofmembraneinvaginationandgeneratesapolarizedlayerofcellsduringcellularization" title="sec"> <div class="title" tagx="title" title="title">
The cortical actin network establishes sites of membrane invagination and generates a polarized layer of cells during cellularization</div> <p>To assess whether cellularization in <i>S. arctica</i> involves encapsulation of nuclei by plasma membranes, we imaged the plasma membrane using live time-lapse imaging in the presence of the lipid dye FM4-64 (<a href="#bib7">Betz et al., 1996</a>). We observed a rapid increase in FM4-64 intensity at the periphery of the coenocyte 30 min prior to flip (<a href="#fig2">Figure 2A</a>, panel II, <a href="#video3">Videos 3</a> and <a href="#video4">4</a>). The plasma membrane invagination sites formed at the periphery and progressed synchronously from the outside toward the center of the coenocyte, forming polarized, polyhedral cells (<a href="#fig2">Figure 2A</a>, panels II-V, <a href="#video3">Videos 3</a> and <a href="#video4">4</a>). Similar to cellularization in <i>Drosophila</i> coenocytes (<a href="#bib56">Lecuit and Wieschaus, 2000</a>; <a href="#bib54">Lecuit et al., 2002</a>), we observe different rates of plasma membrane invaginations throughout the process. During the first 10–15 min, the invaginations advance at a rate of about 0,3 µm/min (<a href="#fig2s1">Figure 2—figure supplement 1A</a>). This period is followed by a rapid phase where membrane invaginates at a rate of about 0,9 µm/min leading to flip (<a href="#fig2s1">Figure 2—figure supplement 1A</a>). Finally, following flip, cells lost their polyhedral shape and became round, suggesting that they were no longer attached to each other (<a href="#fig2">Figure 2A</a>, panel VI, <a href="#video3">Videos 3</a> and <a href="#video4">4</a>).</p> <div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/"><span class="label" tagx="label" title="label">Figure 2.</span></a></div>   <div class="title" tagx="title" title="title">
Actin cytoskeleton and plasma membrane dynamics during cellularization of <i>S. arctica</i>.</div> <p>(<b>A</b>) Dynamics of plasma membrane invaginations during cellularization. Live-cells, pre-grown for 58 hr, were stained with FM4-64 (10 µM) and imaged using epifluorescent microscopy with a 5 min interval. Bar, 50 µm. (<b>B</b>) Spatio-temporal organization of the actin cytoskeleton, nuclei and cells during cellularization. Synchronized cells of <i>S. arctica,</i> pre-grown for 48 hr, were fixed every hour for 14 hr and stained with phalloidin and DAPI to reveal cytoskeletal dynamics during cellularization. All cells were imaged using confocal microscopy. In panel IV, arrows indicate higher actin signal intensity on the internal side and that nuclei are localized close to the cortex indicating that the layer of cells is polarized. Bar, 50 µm. (<b>C</b>) Actin network is established prior to plasma membrane invaginations. Synchronized cells of <i>S. arctica,</i> pre-grown for 54 hr, were fixed every 2 hr for 10 hr and stained with both the membrane dye FM4-64FX and phalloidin. Arrows show sites of colocalization between both markers at the onset of plasma membrane invaginations. Bar, 50 µm.</p>   </div> <div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/"><span class="label" tagx="label" title="label">Video 3.</span></a></div>   <div class="title" tagx="title" title="title">
Time lapse of synchronized cells of <i>S. arctica</i> stained with the plasma membrane dye FM4-64 and obtained with epifluorescent microscopy.</div> <p>Time interval between frames is 5 min. The movie is played at 7fps. Plasma membrane invaginations can be seen occurring from the outside inwards for approximately 40 min prior to flip. Bar, 50 µm.</p>   </div> <div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/"><span class="label" tagx="label" title="label">Video 4.</span></a></div>   <div class="title" tagx="title" title="title">
Time lapse of plasma membrane dynamics during cellularization of <i>S. arctica.</i></div>   </div> <p>In animal coenocytes, plasma membrane invagination is associated with dynamic organization of the actomyosin cytoskeleton (<a href="#bib61">Mazumdar and Mazumdar, 2002</a>). To investigate actin dynamics during cellularization, we took advantage of the timeline described above and imaged coenocytes that were fixed and stained for actin and nuclei (using phalloidin and DAPI, respectively) at different time points during cellularization (<a href="#fig2">Figure 2B</a>, <a href="#fig2s1">Figure 2—figure supplement 1B and C</a>). During growth, actin localized exclusively as small patches at the surface of the coenocyte (<a href="#fig2s1">Figure 2—figure supplement 1B</a>, panel I). Only at the onset of cellularization, multiple actin patches increased in size to form actin nodes (<a href="#fig2">Figure 2B</a>, panel I and II, and <a href="#fig2s1">Figure 2—figure supplement 1B</a>, panel II). This phase was followed by cortical compartmentalization surrounding the nuclei through gradual formation of an actin filament network solely at the cortex of the coenocyte (<a href="#fig2">Figure 2B</a>, panel III, and <a href="#fig2s1">Figure 2—figure supplement 1B</a>, panels III and IV). Following this cortical compartmentalization, a layer of cells was transiently formed by inward growth of the actin filaments from the cortex (<a href="#fig2">Figure 2B</a>, panels IV and <a href="#fig2s1">Figure 2—figure supplement 1B</a>, panels V and VI). During this stage, the actin signal intensity was uneven and higher on the internal side (<a href="#video5">Video 5</a>), and nuclei were localized close to the cortex, indicating that cells are polarized (<a href="#fig2">Figure 2B</a>, panel IV, <a href="#video5">Video 5</a>). These polarized cells progressively grew towards the center of the coenocyte to fill the cavity (<a href="#fig2s1">Figure 2—figure supplement 1B</a>, panel VII). After flip, similar to the plasma membrane organization mentioned above, the layer of cells was reorganized to form spherical cells (<a href="#fig2">Figure 2B</a>, panel VI, and <a href="#fig2s1">Figure 2—figure supplement 1B</a>, panel VIII).</p> <div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/"><span class="label" tagx="label" title="label">Video 5.</span></a></div>   <div class="title" tagx="title" title="title">
Z-projection of the spatial organization of the polarized layer of cells during cellularization of <i>S. arctica</i> obtained with confocal microscopy.</div> <p>The cell is stained with phalloidin and DAPI. The movie is played at 7fps. A polarized layer of cells can be seen. Bar, 10 µm.</p>   </div> <p>To determine the order of actin organization and plasma membrane invaginations, we assessed the localization of both actin and plasma membrane in fixed samples using phalloidin and a fixable analog of FM4-64 (FM4-64FX). We found that the cortical actin network formed prior to the appearance of the membrane dye (<a href="#fig2">Figure 2C</a>, panel II).</p> <p>The intensity of FM4-64FX labeling also increased and co-localized with the underlying actin network around the time of plasma membrane invagination (<a href="#fig2">Figure 2C</a>, panels III and IV). This suggests that the cortical actin network determines the site of plasma membrane invagination.</p> <p>Finally, to determine the timing of cell wall formation, we stained the cells with calcofluor. We observed that labeling co-localized with the membrane marker FM4-64FX around individual cells in the post-flip stage (<a href="#fig2s1">Figure 2—figure supplement 1D</a>). This shows that the newborn cells already build the cell wall before the release, as was suggested previously in other <i>Sphaeroforma</i> species (<a href="#bib60">Marshall and Berbee, 2013</a>).</p> <p>In early insect embryos, cellularization of the syncytial blastoderm occurs through actin-dependent invagination of the plasma membrane. Here, we demonstrate that the cellularization of the ichthyosporean coenocyte also involves active actin reorganization and membrane invagination (slow and fast phases of membrane invagination rates) at the site of actin cytoskeleton. Additionally, this results in the transient formation of a polarized layer of cells with an internal cavity that morphologically resembles simple epithelial structures.</p> </div> <div class="cellularizationisassociatedwithextensivesequentialtranscriptionalwavesandisassociatedwithevolutionarilyyoungertranscripts" title="sec"> <div class="title" tagx="title" title="title">
Cellularization is associated with extensive sequential transcriptional waves and is associated with evolutionarily younger transcripts</div> <p>To gain insight into the regulation of the cellularization of <i>S. arctica</i>, we sequenced the expressed mRNAs of synchronized cultures with high temporal resolution, and comprehensively analyzed the dynamics of transcription, alternative splicing, and long intergenic non-coding RNAs (lincRNAs). Because the published genome assembly of <i>S. arctica</i> (<a href="#bib33">Grau-Bové et al., 2017</a>) was fragmented and likely resulted in incomplete gene models, we first re-sequenced the genome combining the Illumina technology with the PacBio long read sequencing technology. The final assembly sequences comprised 142,721,209 bp, and the metrics were greatly improved compared to the previous assembly (<a href="#bib33">Grau-Bové et al., 2017</a>) (<a href="#fig3sdata1">Figure 3—source data 1</a>). <i>Ab initio</i> gene annotation resulted in the discovery of novel ORFs due to the absence of repetitive regions in the previous assembly. RNA-seq data was used to improve the ORF prediction, to define the 5’ and 3’ untranslated regions, and to discover lincRNAs. In total, 33,682 protein coding genes and 1071 lincRNAs were predicted using this combined pipeline (see Materilas and methods). This final transcriptome assembly was used as the reference transcriptome for further analysis.</p> <p>To perform the time-resolved transcriptomics, we isolated and sequenced mRNA from two independent synchronized cultures at 6 hr time intervals during the entire coenocytic cycle, encompassing time points from early 4-nuclei stage throughout growth and cellularization stages until the release of newborn cells (<a href="#fig3s1">Figure 3—figure supplement 1A</a>).</p> <p>We first analyzed transcript abundance during the time series. The majority of the transcriptome (20,196 out of 34,753 predicted protein-coding and lincRNA genes) was transcribed at very low levels (mean expression throughout the time course &amp;lt;0.5 tpm [transcripts per million]) and were removed for the subsequent clustering analysis. Clustering of transcript abundance data from both biological replicates revealed a clear separation between the transcriptomes of the growth stage (12 hr to 42 hr time points) and the cellularization stage (48 hr to 66 hr time points) (<a href="#fig3">Figure 3A</a>). Furthermore, we observed that the transcriptome patterns between replicates 1 and 2 were shifted by 6 hr from 48 hr onwards (<a href="#fig3">Figure 3A</a>), presumably due to differences in temperature and conditions influencing the kinetics of the coenocytic cycle. We thus adjusted the time of the second replicate by 6 hr according to the clustering results, although we emphasize that the clustering analysis did not depend on time. Among the expressed transcripts (defined as mean expression levels higher than 0.5 tpm across all samples; in total 13,542 coding genes and 1015 lincRNAs), consensus clustering using Clust (<a href="#bib1">Abu-Jamous and Kelly, 2018</a>) extracted 9 clusters of co-expressed transcripts with a total of 4441 protein coding genes (<a href="#fig3">Figure 3B</a>), while the rest of the transcripts were not assigned to any co-expression cluster. The assigned cluster membership was robust to using either of the replicates or averaging (<a href="#fig3s1">Figure 3—figure supplement 1B</a>). Visualization by heatmap and t-SNE plot separated the clusters into two meta-clusters containing the growth stage (clusters 1–3, totaling 2197 genes) and cellularization stage (clusters 4–9, totaling 2314 genes) clusters (<a href="#fig3">Figure 3C and D</a>). Among the cellularization clusters, we obtain three main clusters of genes sharply activated during cellularization (clusters 5, 6 and 7), which contain transcripts expressed at early, mid and late cellularization, respectively (<a href="#fig3">Figure 3B and D</a>). Altogether, this shows that cellularization is associated with extensive sharp transcriptional activation in multiple temporal waves, in total affecting 17% of the expressed transcriptome.</p> <div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/"><span class="label" tagx="label" title="label">Figure 3.</span></a></div>   <div class="title" tagx="title" title="title">
Transcriptional dynamics across the <i>S. arctica</i> life cycle.</div> <p>(<b>A</b>) Hierarchical clustering of time point samples by Euclidian distance of spearman correlation coefficient. The sample T48_2 is missing due to technical reason. (<b>B</b>) A heatmap of 4441 coding genes that were clustered into nine clusters. (<b>C</b>) A t-SNE plot of clustered genes. (<b>D</b>) Mean expression profile of each gene expression cluster.</p> <p> </p><div class="local-data" title="local-data"><span class="label" tagx="label" title="label">Figure 3—source data 1.</span> <div class="title" tagx="title" title="title">
Metrics of the <i>Sphaeroforma arctica</i> genome assemblies.</div>  <div class="media" title="media"><a href="elife-49801-fig3-data1.pdf">LINK</a></div> </div> <p /> <p> </p><div class="local-data" title="local-data"><span class="label" tagx="label" title="label">Figure 3—source data 2.</span> <div class="title" tagx="title" title="title">
Transcript abundance (in tpm) of all the <i>S. arctica</i> transcripts.</div> <p>Used in <a href="#fig3">Figures 3</a>, <a href="#fig4">4</a> and <a href="#fig5">5</a>.</p>  <div class="media" title="media"><a href="elife-49801-fig3-data2.txt">LINK</a></div> </div> <p /> <p> </p><div class="local-data" title="local-data"><span class="label" tagx="label" title="label">Figure 3—source data 3.</span> <div class="title" tagx="title" title="title">
Normalized transcript abundance of the expressed <i>S. arctica</i> transcripts (mean tpm &amp;gt; 0.5).</div> <p>Used in <a href="#fig3">Figures 3</a>, <a href="#fig4">4</a> and <a href="#fig5">5</a>.</p>  <div class="media" title="media"><a href="elife-49801-fig3-data3.txt">LINK</a></div> </div> <p /> <p> </p><div class="local-data" title="local-data"><span class="label" tagx="label" title="label">Figure 3—source data 4.</span> <div class="title" tagx="title" title="title">
Table of transcripts per cluster membership.</div>  <div class="media" title="media"><a href="elife-49801-fig3-data4.txt">LINK</a></div> </div> <p />   </div> <p>In parallel to transcript abundance dynamics, we also assessed alternative splicing (AS) across the time series. This analysis identified 2022 genes affected by intron retention (12.9% of all intron-bearing genes, totaling 4310 introns), 914 by exon skipping (12.3% of genes with &amp;gt;2 exons, 1206 exons) and 44 with mutually exclusive exons (0.7% of genes with &amp;gt;3 exons, involving 118 exon pairs) in all samples (<a href="#fig3s1">Figure 3—figure supplement 1C and D</a>). Overall, neither the number of AS events nor the number of genes affected vary dramatically along the <i>S. arctica</i> growth cycle (<a href="#fig3s1">Figure 3—figure supplement 1E-J</a>). Analysis of AS events over time did not yield any discernible global dynamics, although we found a small number of events differentially present between the growth and cellularization stages (<a href="#fig3s1">Figure 3—figure supplement 1K</a>).</p> <p>Interestingly, skipped exons were more likely to be in-frame (38.63%, compared to 30.33% of in-frame exons in genes with &amp;gt;2 exons, p=4.34e-05, Fisher’s exact test) and yield non-truncated transcripts, a phenomenon commonly observed in animal transcriptomes but not in transcriptomes of other unicellular eukaryotes (<a href="#bib34">Grau-Bové et al., 2018</a>). The transcripts affected by such in-frame exon skipping events are enriched in biological processes such as regulation of multicellular organismal processes, assembly of the focal adhesion complex and cell growth (<a href="#fig3s1">Figure 3—figure supplement 1L</a>). Overall, although pervasive, alternative splicing likely does not play a major role in regulation of the coenocytic cycle and cellularization of <i>S. arctica</i>.</p> <p>Additionally, we analyzed the dynamics of lincRNA expression. Overall, lincRNAs represent ~3% of total transcript abundance across the time series (<a href="#fig4s1">Figure 4—figure supplement 1A</a>). Among the long non-coding RNAs, 70 lincRNA transcripts clustered with coding genes into temporally co-expressed clusters (<a href="#fig4">Figure 4A</a>). Sequence homology searches revealed that 24 of the <i>S. arctica</i> lncRNAs were conserved in distantly related ichthyosporean species such as <i>Creolimax fragrantissima</i>, <i>Pirum gemmata</i> and <i>Abeoforma whisleri</i> (estimated to have diverged ~500 million years ago; <a href="#bib69">Parfrey et al., 2011</a>). This is a remarkable depth of conservation, since animal lincRNAs are not conserved between animal phyla (<a href="#bib11">Bråte et al., 2015</a>; <a href="#bib30">Gaiti et al., 2015</a>; <a href="#bib41">Hezroni et al., 2015</a>). Other lincRNAs were either specific to <i>S. arctica</i> (511) or conserved only in closely related <i>Sphaeroforma</i> species (536). Comparison of lincRNAs by degree of conservation showed no notable difference in GC content (<a href="#fig4s1">Figure 4—figure supplement 1B</a>) or dynamics of expression during the coenocytic cycle (<a href="#fig4s1">Figure 4—figure supplement 1C</a>). However, we found that conserved lincRNAs were on average longer than non-conserved ones (<a href="#fig4s1">Figure 4—figure supplement 1D</a>) and, strikingly, expressed at much higher levels (<a href="#fig4">Figure 4B</a>). Among the 24 deeply conserved lincRNAs, five clustered in the temporally co-expressed clusters, which is higher than expected by chance (p=0.0166, Fisher’s exact test). Among these, three belonged to the cellularization clusters, including, lincRNA asmbl_31839, which has a remarkably high sequence similarity with its homologs from other ichthyosporeans (<a href="#fig4s1">Figure 4—figure supplement 1E</a>). Furthermore, its transcriptional regulation is independent of the transcriptional regulation of its neighboring coding genes (located within 3 kb) (<a href="#fig4">Figure 4C</a>). In summary, we discovered deeply conserved lincRNAs that are expressed at high levels and are transcriptionally activated during cellularization.</p> <div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/"><span class="label" tagx="label" title="label">Figure 4.</span></a></div>   <div class="title" tagx="title" title="title">
Dynamics of lincRNAs, alternative splicing, and gene phylostrata.</div> <p>(<b>A</b>) A heatmap of 70 long intergenic non-coding RNAs (lincRNAs) that co-cluster with coding genes. (<b>B</b>) mean expression level of the lincRNAs, binned by degree of conservation. (<b>C</b>) Expression of the conserved lincRNA, lincRNA: asmbl_31839 and the two coding genes located immediately upstream and downstream of it. (<b>D</b>) A phylogenetic tree indicating the 10 defined phylostrata, and a pie chart of fractions of all expressed genes per phylostratum. (<b>E</b>) Coefficient of variance of gene expression across the <i>S. arctica</i> life cycle, binned by phylostratum. (<b>F</b>) Transcriptome age index (TAI) of gene expression for the <i>S. arctica</i> life cycle. Higher TAI represents a younger transcriptome.</p> <p> </p><div class="local-data" title="local-data"><span class="label" tagx="label" title="label">Figure 4—source data 1.</span> <div class="title" tagx="title" title="title">
Table of Blast e-values of orthologs of <i>S. arctica</i> lincRNAs in other ichthyosporean species.</div>  <div class="media" title="media"><a href="elife-49801-fig4-data1.txt">LINK</a></div> </div> <p /> <p> </p><div class="local-data" title="local-data"><span class="label" tagx="label" title="label">Figure 4—source data 2.</span> <div class="title" tagx="title" title="title">
List of transcriptomes used for lincRNA conservation analysis.</div>  <div class="media" title="media"><a href="elife-49801-fig4-data2.pdf">LINK</a></div> </div> <p /> <p> </p><div class="local-data" title="local-data"><span class="label" tagx="label" title="label">Figure 4—source data 3.</span> <div class="title" tagx="title" title="title">
Table of gene orthogroups.</div> <p>Used in <a href="#fig4">Figures 4</a> and <a href="#fig5">5</a>.</p>  <div class="media" title="media"><a href="elife-49801-fig4-data3.csv">LINK</a></div> </div> <p /> <p> </p><div class="local-data" title="local-data"><span class="label" tagx="label" title="label">Figure 4—source data 4.</span> <div class="title" tagx="title" title="title">
List of transcriptomes used for generation of orthogroups and phylostratigraphy analysis.</div>  <div class="media" title="media"><a href="elife-49801-fig4-data4.pdf">LINK</a></div> </div> <p /> <p> </p><div class="local-data" title="local-data"><span class="label" tagx="label" title="label">Figure 4—source data 5.</span> <div class="title" tagx="title" title="title">
Table of <i>S. arctica</i> transcripts per gene stratum and orthogroup membership.</div> <p>Gene ages range from 1 (<i>S. arctica</i>-specific) to 10 (pan-eukaryotic). Only transcripts that were classified in an orthogroup are listed; the remainder of the transcripts were <i>S. arctica</i>-specific.</p>  <div class="media" title="media"><a href="elife-49801-fig4-data5.csv">LINK</a></div> </div> <p />   </div> <p>Finally, to assess the evolutionary origins of the co-expression clusters, we used a phylostratigraphic analysis to classify genes into evolutionary gene age groups. We carried out orthology analysis of the predicted <i>S. arctica</i> proteome along with 30 representative species from the eukaryotic tree of life to identify ‘orthogroups’ (i.e. groups of putative orthologs between species). <i>S. arctica</i> protein-coding genes clustered in 6149 orthogroups representing 12,527 genes; the rest of the genes did not have an ortholog outside <i>S. arctica</i>.</p> <p>Next, we inferred the age of each gene using Dollo parsimony (<a href="#bib17">Csurös, 2010</a>) to classify them into phylostrata (sets of genes with the same phylogenetic origin) (<a href="#fig4">Figure 4D</a>). Analysis of gene expression by phylostrata revealed a trend toward more variable expression throughout the coenocytic cycle in younger genes (<a href="#fig4">Figure 4E</a>), although their mean expression levels were lower (<a href="#fig4s2">Figure 4—figure supplement 2A</a>). Such a correlation has been observed in animal development, where developmentally regulated genes tend to be of younger origin (<a href="#bib23">Domazet-Lošo and Tautz, 2010</a>). Analysis of enrichment of gene phylostrata in each gene expression cluster (<a href="#fig4s2">Figure 4—figure supplement 2B</a>) showed that the growth clusters are enriched for pan-eukaryotic genes. In contrast, we find that the cellularization clusters were enriched for younger genes (<a href="#fig4s2">Figure 4—figure supplement 2B and C</a>). Importantly, we found that genes with ichthyosporean origins were significantly enriched in all three of the largest cellularization clusters (<a href="#fig4s2">Figure 4—figure supplement 2B and C</a>). Finally, computing the transcriptome age index (<a href="#bib23">Domazet-Lošo and Tautz, 2010</a>) to assess the overall age of the transcriptome across the life cycle revealed a statistically significant hourglass pattern (p=4.4 * 10<sup>−5</sup> and p=0.02 for replicates 1 and 2, Reductive Hourglass test; <a href="#bib24">Drost et al., 2015</a>), with older genes expressed at later stages of growth, and younger genes expressed during early growth and cellularization (<a href="#fig4">Figure 4F</a>). Such an hourglass transcriptomic pattern has previously been observed in animal development, where it reflects the morphological similarities and differences of embryos of different taxa (<a href="#bib23">Domazet-Lošo and Tautz, 2010</a>), and it has been suggested as a conserved logic of embryogenesis across kingdoms (<a href="#bib76">Quint et al., 2012</a>). Despite this, we currently do not have a morphological explanation for this transcriptional hourglass pattern in ichthyosporeans. Altogether, the phylostratigraphic analysis suggests that cellularization is a comparatively younger process, whereas the growth stage represent an evolutionarily ancient process.</p> </div> <div class="temporalco-expressionofactincytoskeleton,celladhesionandcellpolaritypathwaysduringcellularization" title="sec"> <div class="title" tagx="title" title="title">
Temporal co-expression of actin cytoskeleton, cell adhesion and cell polarity pathways during cellularization</div> <p>To functionally assess the gene expression clusters, we also carried out gene ontology (GO) enrichment analysis (<a href="#fig5sdata1">Figure 5—source data 1</a>). The largest growth cluster (cluster 1) was enriched in GO terms related to cell growth and biosynthesis. Early and mid-cellularization clusters 5 and 6 were enriched for GO terms related to membrane organization and actin cytoskeleton. Late cellularization clusters were, in addition to GO terms related to actin, also enriched for GO terms related to cell adhesion and polarity. Given that these processes play a major role during cellularization of the insect blastoderm, we investigated the expression pattern of homologs of known regulators of cellularization in <i>Drosophila</i>.</p> <p>In the <i>Drosophila</i> blastoderm, cellularization is regulated by several zygotically transcribed genes (<a href="#bib61">Mazumdar and Mazumdar, 2002</a>) and relies on extensive membrane trafficking controlled by Rab GTPases (<a href="#bib14">Bucci et al., 1992</a>; <a href="#bib22">Dollar et al., 2002</a>; <a href="#bib71">Pelissier et al., 2003</a>). It also depends on the spatial organization of both the microtubule and actin cytoskeleton. It involves several microtubule and actin binding proteins, including kinesins, Myosin II , Myosin V, profilin (Chickadee), cofilin (Twinstar), formin (Diaphanous) and Septins, and the conserved family of Rho GTPases (<a href="#bib61">Mazumdar and Mazumdar, 2002</a>).</p> <p>In <i>S. arctica,</i> we did not find homologs of any <i>Drosophila</i> zygotically transcribed genes known to regulate cellularization (data not shown). However, we found that the expression of homologs of Rab5 and Rab11 as well as tubulins and kinesins, except for one (<i>S. arctica</i> Kinesin 2), is constant throughout the coenocytic cycle (<a href="#fig5">Figure 5A and B</a>, <a href="#fig5s1">Figure 5—figure supplement 1</a>). On the other hand, we found many actin-associated genes dynamically expressed. All actin nucleators of the formin family and members of the Arp2/3 complex peaked during cellularization (<a href="#fig5">Figure 5C</a>). In contrast to formins, which largely exhibited sharp peaks, the gene expression of the Arp2/3 complex was initiated earlier and increased gradually (<a href="#fig5">Figure 5C</a>). Septins, cofilin, profilin and myosin V were temporally co-expressed during mid-cellularization and peaked at the same time as actin nucleators (<a href="#fig5">Figure 5D</a>), whereas myosin II, which has a role in organizing actin filaments and contractility, peaked later (<a href="#fig5">Figure 5D</a>). We likewise found the expression of three out of four members of the Cdc42/Rho1 orthogroup to be sharply activated during late cellularization (<a href="#fig5">Figure 5E</a>). Temporal co-expression of these genes suggests that the cellular pathways responsible for organizing the cytoskeleton and cell polarity in <i>Drosophila</i> cellularization are also involved in the cellularization process of <i>S. arctica</i>.</p> <div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/"><span class="label" tagx="label" title="label">Figure 5.</span></a></div>   <div class="title" tagx="title" title="title">
Temporal transcript abundance of cytoskeletal, cell polarity and cell adhesion genes.</div> <p>(<b>A–F</b>) Gene expression of indicated genes across the <i>S. arctica</i> life cycle.</p> <p> </p><div class="local-data" title="local-data"><span class="label" tagx="label" title="label">Figure 5—source data 1.</span> <div class="title" tagx="title" title="title">
Gene ontology (GO) enrichment analysis of gene expression clusters.</div>  <div class="media" title="media"><a href="elife-49801-fig5-data1.pdf">LINK</a></div> </div> <p /> <p> </p><div class="local-data" title="local-data"><span class="label" tagx="label" title="label">Figure 5—source data 2.</span> <div class="title" tagx="title" title="title">
A table of reference gene names plotted in <a href="#fig5">Figure 5</a>.</div>  <div class="media" title="media"><a href="elife-49801-fig5-data2.pdf">LINK</a></div> </div> <p />   </div> <p>Since GO enrichment analysis suggested expression of cell adhesion genes, we investigated expression pattern of conserved cell-cell and cell-matrix adhesion pathways. Integrins and the cytoplasmic members of the integrin adhesome mediate cell-matrix adhesion in animal tissues. We observed that in <i>S. arctica</i>, both integrin receptors, alpha and beta, as well as all cytoplasmic members, are temporally co-expressed during late cellularization (<a href="#fig5">Figure 5F</a>). Beta and alpha catenin, together with cadherins in animals, mediate cell-cell adhesion in epithelial tissues (<a href="#bib78">Rodriguez-Boulan and Macara, 2014</a>). In <i>S. arctica</i>, we found three copies of Aardvark, a homolog of beta-catenin (<a href="#bib65">Murray and Zaidel-Bar, 2014</a>), as well as one homolog of alpha-catenin (<a href="#bib64">Miller et al., 2013</a>). We found that the expression of two out of three Aardvark transcripts, as well as the expression of the alpha catenin homolog, peaked during late cellularization (<a href="#fig5">Figure 5F</a>). This suggests that both cell-matrix and cell-cell adhesion pathways, largely conserved between animals and ichthyosporeans, play a role in the establishment of the polarized layer of cells during late cellularization.</p> </div> <div class="theactomyosincytoskeletonisessentialforcellularization" title="sec"> <div class="title" tagx="title" title="title">
The actomyosin cytoskeleton is essential for cellularization</div> <p>Finally, we tested whether disrupting the cytoskeleton would lead to defects in cellularization. In the absence of genetic tools, we used small inhibitory molecules that target specific conserved cytoskeletal components. We synchronized the cultures and added the inhibitors at the onset of cellularization (54 hr time point). We first assessed the role of the microtubule cytoskeleton during cellularization by adding carbendazim (MBC), a microtubule depolymerizing agent (<a href="#bib15">Castagnetti et al., 2007</a>). Microtubule inhibition did not prevent plasma membrane invagination, although it resulted in a delay of flip and release of newborn cells (<a href="#fig6">Figure 6A and B</a>, and <a href="#fig6s1">Figure 6—figure supplement 1A</a>, <a href="#video6">Videos 6</a> and <a href="#video7">7</a>).</p> <div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/"><span class="label" tagx="label" title="label">Figure 6.</span></a></div>   <div class="title" tagx="title" title="title">
The actomyosin network organization is crucial for cellularization of <i>S. arctica</i>.</div> <p>(<b>A</b>) Depolymerization of microtubules and actin, as well as inhibition of Arp2/3, formins and myosin II affect ‘flip’ and release of newborn cells. Synchronized cells of <i>S. arctica,</i> pre-grown for 54 hr, were imaged for 24 hr in presence of multiple pharmacological inhibitors (DMSO as a control, MBC, Latrunculin A, CK666, SMIFH2, Blebbistatin). Cells undergoing flip and cell release throughout the duration of the experiment were measured (N &amp;gt; 400 cells, Error bars are standard deviation from three independent experiments). (<b>B</b>) Temporal functions of the Arp2/3 complex, formins and myosin II in distinct stages of cellularization. Synchronized cells of <i>S. arctica,</i> pre-grown for 54 hr were subject to multiple pharmacological inhibitors treatments (DMSO as a control, MBC, Latrunculin A, CK666, SMIFH2, Blebbistatin) and fixed and stained with phalloidin and DAPI after 24 hr and 48 hr of treatments. Phalloidin staining allows us to measure the fraction of cells exhibiting different actin and cellular structures throughout cellularization. (<b>C</b>) The different actin structures observed when cells are treated with multiple pharmacological inhibitors treatments (DMSO as a control, MBC, Latrunculin A, CK666, SMIFH2, Blebbistatin). Bar, 10 µm. (<b>D</b>) A model representing the actin cytoskeleton, plasma membrane and cell wall at different stages of the cellularization process in <i>S. arctica,</i> indicating sequential steps of actin remodeling mediated by Arp2/3, formins and Myosin II.</p>   </div> <div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/"><span class="label" tagx="label" title="label">Video 6.</span></a></div>   <div class="title" tagx="title" title="title">
Time lapse of synchronized cells of <i>S. arctica,</i> pre-grown for 54 hr and treated with different pharmacological inhibitors.</div> <p>Time interval between frames is 30 min. The movie is played at 7fps. Cellularization is affected when cells are treated with LatA, CK666, SMIFH2 and Blebbistatin. Bar, 10 µm.</p>   </div> <div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/"><span class="label" tagx="label" title="label">Video 7.</span></a></div>   <div class="title" tagx="title" title="title">
Time lapse of synchronized cells of <i>S. arctica,</i> pre-grown for 54 hr, stained with the plasma membrane dye FM4-64 and treated with different pharmacological inhibitors.</div> <p>Time interval is 10 min. The movie is played at 7fps. Plasma membrane invaginations is prevented when cells are treated with LatA, CK666, SMIFH2 and Blebbistatin. Bar, 10 µm.</p>   </div> <p>Furthermore, by staining the MBC-treated cells with DAPI and phalloidin, we observed a loss of the uniform spacing of the nuclei and actin filaments during the cortical compartmentalization stage of cellularization (<a href="#fig6">Figure 6C</a>). After MBC treatment, newborn cells varied in size and number of nuclei (<a href="#fig6s1">Figure 6—figure supplement 1B and C</a>, <a href="#video6">Video 6</a>). These results suggest that the microtubule cytoskeleton is not essential for plasma membrane invagination but is crucial for nucleus and actin filament positioning at the cortex of the coenocyte. This is in contrast to the role of microtubules during cellularization of the blastoderm in <i>Drosophila</i>, where microtubules also directly drive the plasma membrane invagination through formation of inverted baskets covering the nuclei (<a href="#bib61">Mazumdar and Mazumdar, 2002</a>).</p> <p>Next, we sought to disrupt actin polymerization using either the broad actin depolymerizing agent Latrunculin A (LatA) (<a href="#bib10">Braet et al., 1996</a>), the Arp2/3 inhibitor CK666 (<a href="#bib40">Hetrick et al., 2013</a>), or the formin inhibitor SMIFH2 (<a href="#bib50">Kim et al., 2015b</a>). Cells treated with Latrunculin A lacked any actin patches or actin filaments and failed to undergo flip or produce newborn cells (<a href="#fig6">Figure 6A-C</a>, and <a href="#fig6s1">Figure 6—figure supplement 1A</a>, <a href="#video6">Videos 6</a> and <a href="#video7">7</a>). Furthermore, plasma membrane invagination did not occur after LatA treatment (<a href="#video6">Videos 6</a> and <a href="#video7">7</a>). In contrast, CK666-treated cells formed cortical actin patches, but were not able to form actin nodes, and they were unable to generate plasma membrane invaginations (<a href="#fig6">Figure 6A–6C</a>, <a href="#video6">Videos 6</a> and <a href="#video7">7</a>). These results show that the actin cytoskeleton and Arp2/3-mediated actin nucleation are required for the formation of actin nodes, the first step in the cellularization of <i>S. arctica</i>. In contrast, the formin inhibitor SMIFH2 did not block the formation of actin nodes, but it prevented the formation of actin filaments in the later stages (<a href="#fig6">Figure 6A–6C</a>). In addition, the plasma membrane invagination did not occur (<a href="#fig6">Figure 6A-C</a>, and <a href="#fig6s1">Figure 6—figure supplement 1A</a>, <a href="#video6">Videos 6</a> and <a href="#video7">7</a>), although we note that a small fraction of cells was not affected. This suggests that formins play a role in the nucleation of actin filaments after the formation of actin nodes.</p> <p>Finally, we assessed the role of Myosin II, using blebbistatin, an inhibitor of Myosin II ATPase activity (<a href="#bib51">Kovács et al., 2004</a>). Blebbistatin treatment blocked plasma membrane invaginations and prevented cellularization (<a href="#fig6">Figure 6A-C</a>, and <a href="#fig6s1">Figure 6—figure supplement 1A</a>, <a href="#video6">Videos 6</a> and <a href="#video7">7</a>). In coenocytes where plasma membrane invaginations started but the cell layer was not yet formed, we observed a retraction of already present invaginations when treated with blebbistatin (<a href="#fig6s1">Figure 6—figure supplement 1C</a>, <a href="#video8">Video 8</a>). Additionally, in coenocytes where the cell layer was formed but polarized growth was not complete, blebbistatin treatment prevented polarized growth but allowed the release of cells of different cell sizes (<a href="#fig6s1">Figure 6—figure supplement 1C</a>, <a href="#video8">Video 8</a>). Although blebbistatin-treated cells were able to form actin filaments, they had an aberrant wavy cortical actin filament network, suggesting that inhibition of Myosin II causes loss of actin crosslinking and actin network contractility (<a href="#fig6">Figure 6C</a>). Taken together, these results indicate that the actomyosin apparatus is essential for cellularization in <i>S. arctica</i> and reveal a temporal sequence of involvement of Arp2/3 complex, formins and Myosin II (<a href="#fig6">Figure 6D</a>). This temporal sequence is reflected in the relative timing of expression of Arp2/3, formins and myosin II genes (<a href="#fig5">Figure 5C and D</a>).</p> <div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/"><span class="label" tagx="label" title="label">Video 8.</span></a></div>   <div class="title" tagx="title" title="title">
Time lapse of synchronized cells of <i>S. arctica,</i> pre-grown for 62 hr, stained with the plasma membrane dye FM4-64 and treated with blebbistatin.</div> <p>Time interval is 5 min. The movie is played at 7fps. Plasma membrane invaginations and polarized growth are affectd when cells are treated with Blebbistatin. Bar, 10 µm.</p>   </div> </div> </div> <div class="discussion" title="discussion"> <div class="title" tagx="title" title="title">
Discussion</div> <p>To address whether animal-like cellularization exists outside animals, we performed imaging, transcriptomic analysis and pharmacological inhibition experiments in an ichthyosporean <i>Sphaeroforma arctica</i>, a unicellular relative of animals (<a href="#fig6">Figure 6D</a>). We show that at the onset of cellularization, Arp2/3 complex mediates the formation of actin nodes at the cortex of the coenocyte. This is followed by formin-dependent nucleation of actin filaments. These actin filaments are then crosslinked in a Myosin II-dependent manner, which results in formation of a cortical actomyosin network that surrounds evenly spaced nuclei. This spatial organization of both the actomyosin network and nuclei depends on microtubules. The localization of the actomyosin network determines the sites of plasma membrane invaginations and drives their inward growth through a slow and a fast phase, resulting in the formation of a polarized layer of cells. This stage co-occurs with transcriptional activation of genes involved in cell-cell and cell-matrix adhesion. Polarized growth persists until the internal cavity is occupied, after which the polyhedrally-shaped cells undergo flip, lose polarization and detach from each other. At this point, future newborn cells form the cell wall before they are released and initiate a new coenocytic cycle. Overall, our results show that the cellularization of <i>S. arctica</i> morphologically resembles the cellularization of the <i>Drosophila</i> blastoderm and share a common actomyosin pathway for cellularization. This suggests two possible evolutionary scenarios. First, given the widespread presence of coenocytes in opisthokonts, we have suggested that the common ancestor of opisthokonts already had a coenocytic life cycle (<a href="#bib67">Ondracka et al., 2018</a>). In this scenario, cellularization of the coenocyte by this conserved actomyosin mechanism would have already been present in this common ancestor. In the second scenario, coenocytes in animals and ichthyosporeans would have evolved independently, but adopted the same conserved actomyosin mechanism for cellularization.</p> <p>Additionally, our transcriptomic analysis suggests that transcriptional control plays an important role in regulation of the coenocytic cycle of <i>S. arctica</i>. Interestingly, we show that many genes expressed during cellularization of <i>S. arctica</i> emerged at the onset of ichthyosporeans. This strongly suggests that the general mechanisms of cellularization in <i>S. arctica</i> are likely conserved within ichthyosporeans. Altogether, our results argue that cellularization in ichthyosporeans requires both regulatory pathways conserved with animals, as well as regulatory pathways that evolved independently in ichthyosporeans. The later may include lincRNAs that we found to be remarkably conserved among ichthyosporeans and show high expression levels. Overall, our temporal gene expression dataset provides a good resource for further functional studies in <i>S. arctica</i> and other ichthyosporean species.</p> <p>Among pathways transcriptionally upregulated during cellularization are the conserved cell-cell adhesion and cell-matrix adhesion pathways, which occur concomitantly with the formation of polarized layer of cells. These pathways are hallmarks of epithelial tissues in animals (<a href="#bib78">Rodriguez-Boulan and Macara, 2014</a>). The epithelium is the first tissue that is established during embryogenesis and perhaps the first that emerged in evolution, thus representing the basic form of multicellular organization in animals (<a href="#bib13">Brunet and King, 2017</a>). Epithelia are found in all animal lineages, including sponges (<a href="#bib57">Leys et al., 2009</a>). It has previously been suggested that the origin of epithelial structures predates the origin of animals, as a polarized non-cadherin-based epithelium-like structure, regulated by alpha and beta catenins, is also present in the slime mold <i>Dictyostelium discoideum</i> (<a href="#bib19">Dickinson et al., 2011</a>). Based on this indirect evidence from transcriptomics, we hypothesize that cellularization in <i>S. arctica</i> generates a transient epithelium-like layer of cells, lending support to the hypothesis that the origin of epithelial tissues predates the origin of animals (<a href="#bib20">Dickinson et al., 2012</a>). However, unlike the epithelium formation in social amoebae, which originates through aggregation (<a href="#bib9">Bonner, 1998</a>), ichthyosporean epithelium is generated clonally, such as in animals. Further comparison of this ichthyosporean epithelium-like structure with animal epithelia will allow us to address whether <i>S. arctica</i> form a bona fide epithelium during cellularization.</p> </div> <div class="materials_methods" title="materials|methods"> <div class="title" tagx="title" title="title">
Materials and methods</div> <div class="table-wrap_UNKNOWN" id="keyresource" orientation="portrait" position="anchor"><span class="label" tagx="label" title="label">Key resources table</span><table frame="hsides" rules="groups"> <thead> <tr> <th rowspan="1" colspan="1">Reagent type <br /><br />(species) or resource</th> <th rowspan="1" colspan="1">Designation</th> <th rowspan="1" colspan="1">Source or reference</th> <th rowspan="1" colspan="1">Identifiers</th> <th rowspan="1" colspan="1">Additional <br /><br />information</th> </tr> </thead> <tbody> <tr> <td rowspan="1" colspan="1">Commercial assay or kit</td> <td rowspan="1" colspan="1">miRNeasy Mini Kit</td> <td rowspan="1" colspan="1">QIAGEN</td> <td rowspan="1" colspan="1">Cat. #: 217004</td> <td rowspan="1" colspan="1" /> </tr> <tr> <td rowspan="1" colspan="1">Commercial assay or kit</td> <td rowspan="1" colspan="1">RNeasy Mini Kit</td> <td rowspan="1" colspan="1">QIAGEN</td> <td rowspan="1" colspan="1">Cat. #: 74104</td> <td rowspan="1" colspan="1" /> </tr> <tr> <td rowspan="1" colspan="1">Commercial assay or kit</td> <td rowspan="1" colspan="1">QIAamp DNA Blood Midi Kit</td> <td rowspan="1" colspan="1">QIAGEN</td> <td rowspan="1" colspan="1">Cat. #: 51183</td> <td rowspan="1" colspan="1" /> </tr> <tr> <td rowspan="1" colspan="1">Chemical compound, drug</td> <td rowspan="1" colspan="1">FM4-64</td> <td rowspan="1" colspan="1">Invitrogen</td> <td rowspan="1" colspan="1">Cat. #: T3166</td> <td rowspan="1" colspan="1" /> </tr> <tr> <td rowspan="1" colspan="1">Chemical compound, drug</td> <td rowspan="1" colspan="1">FM4-64FX</td> <td rowspan="1" colspan="1">Invitrogen</td> <td rowspan="1" colspan="1">Cat. #: F34653</td> <td rowspan="1" colspan="1" /> </tr> <tr> <td rowspan="1" colspan="1">Chemical compound, drug</td> <td rowspan="1" colspan="1">Latrunculin A</td> <td rowspan="1" colspan="1">Sigma-Aldrich</td> <td rowspan="1" colspan="1">Cat. #: L5163</td> <td rowspan="1" colspan="1" /> </tr> <tr> <td rowspan="1" colspan="1">Chemical compound, drug</td> <td rowspan="1" colspan="1">CK666</td> <td rowspan="1" colspan="1">Sigma-Aldrich</td> <td rowspan="1" colspan="1">Cat. #: SML0006</td> <td rowspan="1" colspan="1" /> </tr> <tr> <td rowspan="1" colspan="1">Chemical compound, drug</td> <td rowspan="1" colspan="1">SMIFH2</td> <td rowspan="1" colspan="1">Sigma-Aldrich</td> <td rowspan="1" colspan="1">Cat. #: S4826</td> <td rowspan="1" colspan="1" /> </tr> <tr> <td rowspan="1" colspan="1">Chemical compound, drug</td> <td rowspan="1" colspan="1">Blebbistatin</td> <td rowspan="1" colspan="1">Sigma-Aldrich</td> <td rowspan="1" colspan="1">Cat. #: B0560</td> <td rowspan="1" colspan="1" /> </tr> <tr> <td rowspan="1" colspan="1">Chemical compound, drug</td> <td rowspan="1" colspan="1">Carbendazim (MBC)</td> <td rowspan="1" colspan="1">Sigma-Aldrich</td> <td rowspan="1" colspan="1">Cat. #: 378674</td> <td rowspan="1" colspan="1" /> </tr> <tr> <td rowspan="1" colspan="1">Software</td> <td rowspan="1" colspan="1">ImageJ (<a href="http://imagej.nih.gov/ij/">http://imagej.nih.gov/ij/</a>)</td> <td rowspan="1" colspan="1">PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/29187165">29187165</a></td> <td rowspan="1" colspan="1" /> <td rowspan="1" colspan="1" /> </tr> <tr> <td rowspan="1" colspan="1">Software</td> <td rowspan="1" colspan="1">FALCON_unzip assembler</td> <td rowspan="1" colspan="1" /> <td rowspan="1" colspan="1" /> <td rowspan="1" colspan="1">v0.4.0</td> </tr> <tr> <td rowspan="1" colspan="1">Software, algorithm</td> <td rowspan="1" colspan="1">SMRT analysis</td> <td rowspan="1" colspan="1" /> <td rowspan="1" colspan="1" /> <td rowspan="1" colspan="1">v2.3.0</td> </tr> <tr> <td rowspan="1" colspan="1">Software, algorithm</td> <td rowspan="1" colspan="1">BWA mem</td> <td rowspan="1" colspan="1" /> <td rowspan="1" colspan="1" /> <td rowspan="1" colspan="1">v0.7.12</td> </tr> <tr> <td rowspan="1" colspan="1">Software, algorithm</td> <td rowspan="1" colspan="1">Pilon</td> <td rowspan="1" colspan="1">PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/25409509">25409509</a></td> <td rowspan="1" colspan="1" /> <td rowspan="1" colspan="1">v1.22</td> </tr> <tr> <td rowspan="1" colspan="1">Software, algorithm</td> <td rowspan="1" colspan="1">BRAKER2 (<a href="https://github.com/Gaius-Augustus/BRAKER">https://github.com/Gaius-Augustus/BRAKER</a>)</td> <td rowspan="1" colspan="1">PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/26559507">26559507</a></td> <td rowspan="1" colspan="1" /> <td rowspan="1" colspan="1" /> </tr> <tr> <td rowspan="1" colspan="1">Software, algorithm</td> <td rowspan="1" colspan="1">STAR 2.7</td> <td rowspan="1" colspan="1">PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/23104886">23104886</a></td> <td rowspan="1" colspan="1" /> <td rowspan="1" colspan="1" /> </tr> <tr> <td rowspan="1" colspan="1">Software, algorithm</td> <td rowspan="1" colspan="1">Augustus 3.2.2</td> <td rowspan="1" colspan="1">PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/21216780">21216780</a></td> <td rowspan="1" colspan="1" /> <td rowspan="1" colspan="1" /> </tr> <tr> <td rowspan="1" colspan="1">Software, algorithm</td> <td rowspan="1" colspan="1">Genemark v4.21</td> <td rowspan="1" colspan="1">PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/16314312">16314312</a></td> <td rowspan="1" colspan="1" /> <td rowspan="1" colspan="1" /> </tr> <tr> <td rowspan="1" colspan="1">Software, <br /><br />algorithm</td> <td rowspan="1" colspan="1">Trimgalore<a href="https://github.com/FelixKrueger/TrimGalore">https://github.com/FelixKrueger/TrimGalore</a></td> <td rowspan="1" colspan="1" /> <td rowspan="1" colspan="1" /> <td rowspan="1" colspan="1">v0.4.5</td> </tr> <tr> <td rowspan="1" colspan="1">Software, <br /><br />algorithm</td> <td rowspan="1" colspan="1">Trimmomatic</td> <td rowspan="1" colspan="1">PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/24695404">24695404</a></td> <td rowspan="1" colspan="1" /> <td rowspan="1" colspan="1">v0.35</td> </tr> <tr> <td rowspan="1" colspan="1">Software, algorithm</td> <td rowspan="1" colspan="1">Trinity</td> <td rowspan="1" colspan="1">PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/21572440">21572440</a></td> <td rowspan="1" colspan="1" /> <td rowspan="1" colspan="1">v2.5.1</td> </tr> <tr> <td rowspan="1" colspan="1">Software, algorithm</td> <td rowspan="1" colspan="1">Hisat2</td> <td rowspan="1" colspan="1">PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/25751142">25751142</a></td> <td rowspan="1" colspan="1" /> <td rowspan="1" colspan="1">v2.1.0</td> </tr> <tr> <td rowspan="1" colspan="1">Software, algorithm</td> <td rowspan="1" colspan="1">PASA pipeline</td> <td rowspan="1" colspan="1">PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/14500829">14500829</a></td> <td rowspan="1" colspan="1" /> <td rowspan="1" colspan="1">v2.3.1</td> </tr> <tr> <td rowspan="1" colspan="1">Software, algorithm</td> <td rowspan="1" colspan="1">Kallisto software</td> <td rowspan="1" colspan="1">PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/27043002">27043002</a></td> <td rowspan="1" colspan="1" /> <td rowspan="1" colspan="1" /> </tr> <tr> <td rowspan="1" colspan="1">Software, algorithm</td> <td rowspan="1" colspan="1">Clust software</td> <td rowspan="1" colspan="1">PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/30359297">30359297</a></td> <td rowspan="1" colspan="1" /> <td rowspan="1" colspan="1" /> </tr> <tr> <td rowspan="1" colspan="1">Software, algorithm</td> <td rowspan="1" colspan="1">eggNOG mapper</td> <td rowspan="1" colspan="1">PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/28460117">28460117</a></td> <td rowspan="1" colspan="1" /> <td rowspan="1" colspan="1" /> </tr> <tr> <td rowspan="1" colspan="1">Software, algorithm</td> <td rowspan="1" colspan="1">Spades</td> <td rowspan="1" colspan="1">PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/24093227">24093227</a></td> <td rowspan="1" colspan="1" /> <td rowspan="1" colspan="1">v3.10.0</td> </tr> <tr> <td rowspan="1" colspan="1">Software, algorithm</td> <td rowspan="1" colspan="1">Count software</td> <td rowspan="1" colspan="1">PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/20551134">20551134</a></td> <td rowspan="1" colspan="1" /> <td rowspan="1" colspan="1" /> </tr> </tbody> </table> </div> <div class="cultureconditions" title="sec"> <div class="title" tagx="title" title="title">
Culture conditions</div> <p><i>Sphaeroforma arctica</i> cultures were grown and synchronized as described previously (<a href="#bib67">Ondracka et al., 2018</a>). Briefly, cultures were grown in Marine Broth (Difco BD, NJ, USA; 37.4 g/L) in flasks at 12°C until saturation, producing small cells. These cells were then diluted into fresh media with low density (1:300 dilution of the saturated culture), resulting in a synchronously growing culture. Cultures of <i>S. gastrica</i>, <i>S. tapetis</i>, <i>S. nootkatensis</i>, <i>P. gemmata</i> and <i>A. whisleri</i> were similarly kept in Marine Broth (Difco BD, NJ, USA; 37.4 g/L) at 12°C in dark conditions.</p> </div> <div class="sphaeroformaarcticagenomesequencingandassembly" title="sec"> <div class="title" tagx="title" title="title"><i>Sphaeroforma arctica</i> genome sequencing and assembly</div> <p>Genomic DNA from <i>S. arctica</i> was extracted using QIAamp DNA Blood Midi Kit (Qiagen) from 300 mL culture incubated at 12 C° for 1 week in six 75 cm<sup>2</sup> flasks. The Qubit (Invitrogen) quantification found ~150 μg genomic DNA in total.</p> <p>A SMRTbell library for P6/C4 chemistry was constructed and was run on 32 SMRT cells in a PacBio RSII system (Pacific Biosciences), generating 2,209,004 subreads with a total of 25,164,714,269 bp. Raw subreads were first assembled using the FALCON_unzip assembler (v0.4.0) and the initial assembled sequences were polished by the Quiver integrated in SMRT analysis (v2.3.0). Genomic DNA was then sheared using a Focused-ultrasonicator (Covaris, Inc). A paired-end library with an insert size of 600 bp was sequenced on an Illumina HiSeq2500 platform, producing 136,566,600 reads with read length of 250 bp. Paired-end reads were mapped against the polished sequences using the BWA mem (v0.7.12) followed by error-correction using Pilon (v1.22) (<a href="#bib89">Walker et al., 2014</a>).</p> <p>The assembled genome is deposited on DDBJ (DNA Data Bank of Japan) under accession numbers BJTW01000001-BJTW01000733.</p> </div> <div class="sphaeroformaarcticageneannotation" title="sec"> <div class="title" tagx="title" title="title"><i>Sphaeroforma arctica</i> gene annotation</div> <p>We annotated predicted gene models in our scaffolds using BRAKER2 (<a href="#bib42">Hoff et al., 2018</a>). First we used STAR 2.7 (<a href="#bib21">Dobin et al., 2013</a>) to map all the RNA-seq samples to the genomic scaffolds, so as to obtain empirical evidence of gene bodies and guide the prediction of gene coordinates. These read mappings were then supplied to BRAKER2 in BAM format, which combied these external evidence with the gene coordinates predicted by Augustus 3.2.2 (<a href="#bib47">Keller et al., 2011</a>) (--noInFrameStop mode) and Genemark v4.21 (<a href="#bib58">Lomsadze et al., 2005</a>) to obtain a consolidated set of gene models in GFF format.</p> <p>The transcriptome of <i>S. arctica</i> was assembled in order to add 5’ and 3’ UTR regions to the genome annotation and to search for long non-coding RNA-seq using data obtained here (see below).</p> <p>The RNA-seq reads from the first time series replicate (excluding timepoint 30) were subjected to quality trimming and adaptor removal using Trimgalore (v0.4.5) (<a href="http://www.bioinformatics.babraham.ac.uk/projects/trim_galore/">http://www.bioinformatics.babraham.ac.uk/projects/trim_galore/</a>) and Trimmomatic v0.35 (<a href="#bib8">Bolger et al., 2014</a>). Trimgalore was first run to remove sequencing adaptors, allowing 0 mismatch and minimum adaptor length of 1 nt. Trimmomatic was run by trimming bases with phred score &amp;lt;20 from both ends. Furthermore, a sliding window of 4 bases was used to trim reads from the 5’ end when the mean phred score dropped below 20. Finally, the IlluminaClip option was used to search for remaining sequencing adaptors, allowing two seed mismatches, a palindrome clip threshold of 20 and minimum single match threshold of 7 nt. The quality filtered RNA reads were subsequently assembled using Trinity v2.5.1 (<a href="#bib32">Grabherr et al., 2011</a>), running both a de novo and a genome-guided assembly. To perform a genome-guided transcriptome assembly, we first mapped the RNA-seq reads against the genome using Hisat2 v2.1.0 (<a href="#bib49">Kim et al., 2015a</a>), in strand-specific mode with default parameters. Both Trinity assemblies were run in strand-specific mode while applying the jaccard clip option and otherwise default parameters. We next evaluated the strand-specificity of the assemblies by mapping RNA-seq reads back to the Trinity assemblies using scripts supplied within the Trinity software. Transcripts were subsequently removed if &amp;gt;80% of the reads mapped in the wrong direction. The PASA pipeline v2.3.1 (<a href="#bib36">Haas et al., 2003</a>) was then used to update the existing genome annotation. First, the assembled transcripts described above were mapped against the genome using the initial PASA script to make a temporary annotation file. This was performed with default parameters, using both Blat v35 (<a href="#bib48">Kent, 2002</a>) and Gmap v2015-09-29 (<a href="#bib92">Wu and Watanabe, 2005</a>) aligners, with transcripts specified as strand-specific. Lastly, the transcriptome-based annotations were compared with the existing genome annotation in a final PASA run, also using default parameters. In this step, the existing genes were expanded with UTR annotations and, in cases where a single RNA transcript covered multiple genes, these become merged into a single gene. The annotated genome and proteome of <i>S. arctica</i> can be found on figshare; <a href="https://figshare.com/authors/Multicellgenome_Lab/2628379">https://figshare.com/authors/Multicellgenome_Lab/2628379</a>.</p> </div> <div class="rnaisolation,librarypreparationandsequencing" title="sec"> <div class="title" tagx="title" title="title">
RNA isolation, library preparation and sequencing</div> <p>Synchronized cultures of <i>Sphaeroforma arctica</i> at 12°C were sampled every 6 hr for a total duration of 72 hr. Total RNA was extracted by Trizol and purified using the miRNeasy Mini Kit (QIAGEN) from ~50 mL of culture at each time point. Libraries were prepared using the TruSeq Stranded mRNA Sample Prep kit. Paired-end 50 bp read length sequencing was carried out at the CRG genomics core unit on an Illumina HiSeq v4 sequencer. We obtained between 19 and 32 M reads per sample. Transcripts were quantified using Kallisto software (<a href="#bib12">Bray et al., 2016</a>) with default parameters. To remove non-expressed genes, we filtered out transcripts that had a mean expression level of &amp;lt;0.5 tpm across all 20 samples. This resulted in a set of 14557 transcripts that were used for clustering.</p> <p>For <i>S. gastrica</i> and <i>S. tapetis</i>, RNA purification was performed using the RNeasy kit (Qiagen), while for <i>S. nootkatensis</i>, <i>P. gemmata</i> and <i>A. whisleri</i>, RNA was purified using Trizol (Invitrogen, CA, USA). Strand-specific sequencing libraries were prepared using Illumina TrueSeq Stranded mRNA Sample Prep kit and sequenced on an Illumina HiSeq3000 machine (150 bp paired end).</p> </div> <div class="clusteringofthegeneexpressionandgeneontologyenrichmentanalysisins.arctica" title="sec"> <div class="title" tagx="title" title="title">
Clustering of the gene expression and gene ontology enrichment analysis in <i>S. arctica</i></div> <p><i>Sphaeroforma arctica</i> transcripts were clustered by their gene expression profiles using Clust software (<a href="#bib1">Abu-Jamous and Kelly, 2018</a>) with default parameters and automatic normalization mode. This yielded 4511 transcripts (4441 coding genes and 70 lncRNA genes) that were clustered into nine clusters, ranging from 41 to 2081 co-expressed genes. Gene expression profiles were visualized using superheat package in R (<a href="#bib6">Barter and Yu, 2018</a>). The tSNE plot was generated using Rtsne package. The code used for the transcriptome analysis and visualization is available from GitHub: <a href="https://github.com/andrejondracka/sphaeroforma_transcriptome">https://github.com/andrejondracka/sphaeroforma_transcriptome</a> (<a href="#bib68">Ondracka, 2019</a>; <a href="https://github.com/elifesciences-publications/sphaeroforma_transcriptome">https://github.com/elifesciences-publications/sphaeroforma_transcriptome</a>). </p> <p>Gene Ontology enrichments based on the GOs annotated with eggNOG mapper (<a href="#bib43">Huerta-Cepas et al., 2017</a>) were computed using the topGO R library (v. 2.34). Specifically, we computed the functional enrichments based on the counts of genes belonging to the group of interest relative to all annotated genes, using Fisher’s exact test and the elim algorithm for ontology weighting (<a href="#bib4">Alexa et al., 2006</a>).</p> </div> <div class="transcriptomeassemblyofotherunicellularholozoans" title="sec"> <div class="title" tagx="title" title="title">
Transcriptome assembly of other unicellular holozoans</div> <p>Raw reads (from sequencing libraries and SRA data) were processed with Trimmomatic (<a href="#bib8">Bolger et al., 2014</a>) to remove adapters and low-quality bases, by trimming bases with phred score &amp;lt;28 from both ends. Furthermore, a sliding window of 4 bases was used to trim reads from the 5’ end when the mean phred score dropped below 28. Finally, the IlluminaClip option was used to search for remaining sequencing adaptors, allowing two seed mismatches, a palindrome clip threshold of 28 and minimum single match threshold of 10 nt. All libraries were assembled denovo with Trinity (v2.3.2–2.5.1) (<a href="#bib32">Grabherr et al., 2011</a>) using default parameters. The assemblies of RNA-seq data were performed with the strand specific option, while assemblies based on SRA data were run in standard mode. For the <i>S. sirkka</i> assembly we also applied the jaccard clip option. Coding regions were predicted using TransDecoder v5.2.0 (<a href="#bib37">Haas et al., 2013</a>), by first extracting the longest possible ORFs (only top strand was searched in the strand-specific assemblies), on which likely coding region was predicted. Only the longest ORF was kept for each transcript.</p> </div> <div class="dnaisolationandgenomeassemblyofs.gastricaands.tapetis" title="sec"> <div class="title" tagx="title" title="title">
DNA isolation and genome assembly of <i>S. gastrica</i> and <i>S. tapetis</i></div> <p>Genomic DNA (gDNA) was isolated from <i>S. gastrica</i> and <i>S. tapetis</i>, by lysing cells on a FastPrep system (MP Biomedicals, CA, USA; 4 m/s, 20 s) followed by gDNA purification using the DNeasy kit (Qiagen, NRW, Germany) and subsequently sequenced on an Illumina HiSeq X system (150 bp paired end). Raw reads were subjected to quality trimming and adaptor removal as described above for RNA-seq data, with a phred score cut-off of 26.</p> <p>The quality trimmed reads were subsequently error-corrected and assembled using Spades v3.10.0 (<a href="#bib66">Nurk et al., 2013</a>) applying kmer values 21, 33, 55, 77, 99 and 121, but otherwise default parameters. The resulting spades assemblies were scaffolded using L_rna_scaffolder (<a href="#bib93">Xue et al., 2013</a>) and polished with Pilon (<a href="#bib89">Walker et al., 2014</a>). L_RNA_Scaffolder was run by first mapping the respective transcriptome assemblies to the genome assemblies using Blat (<a href="#bib48">Kent, 2002</a>) which was inputted to L_rna_scaffolder. Next, we run Pilon by first mapping the quality trimmed genomic Illumina reads to the genome assembly using Bowtie2 (<a href="#bib52">Langmead and Salzberg, 2012</a>). The resulting mapping file was then used in Pilon with default parameters. L_rna_scaffolder and Pilon were run repeatedly 5 times, followed by three final runs using only Pilon. These genome assemblies were used as reference genomes for genome-guided Trinity assemblies and PASA annotation as previously described for <i>S. arctica</i>. The genomic reads for <i>S. gastrica</i> and <i>S. tapetis</i> can be found under the EBI/ENA accession: PRJEB34306.</p> </div> <div class="s.arcticalongintergenicnon-codingrnaidentificationandconservationanalysis" title="sec"> <div class="title" tagx="title" title="title"><i>S. arctica</i> long intergenic non-coding RNA identification and conservation analysis</div> <p>Long intergenic non-coding RNAs (lincRNAs) were identified from the PASA annotation described above. First, transcripts shorter than 200 nt were discarded. Then, coding potential was evaluated using both TransDecoder (<a href="#bib37">Haas et al., 2013</a>) and CPC2 (<a href="#bib46">Kang et al., 2017</a>) with default parameters. All transcripts lacking coding potential were then compared with the Rfam database (<a href="#bib45">Kalvari et al., 2018</a>) to exclude other non-coding RNAs such as rRNAs and tRNAs. To exclude possible UTRs actually belonging to fragmented protein coding genes, a minimum genomic distance of 1000 bp from the closest protein coding gene was required. The remaining transcripts were compared with protein coding genes in the Swissprot database using Blastx (<a href="#bib5">Altschul et al., 1990</a>), and all sequences with a match smaller than e-value 1e-5 were removed.</p> <p>Next, the potential lncRNA transcripts were blasted against the proteomes of <i>S. arctica</i> and other closely related ichthyosporeans (<i>Sphaeroforma tapetis</i>, <i>Sphaeroforma sirkka</i>, <i>Sphaeroforma nootkatensis</i>, <i>Sphaeroforma napiecek</i>, <i>Sphaeroforma gastrica</i>, <i>Creolimax fragrantissima</i>, <i>Pirum gemmata</i> and <i>Abeoforma whisleri</i>), and all transcripts with a match smaller than e-value 1e-5, were discarded. To remove transcripts resulting from potential spurious transcription, we required a minimum expression level of at least one tpm in at least one time series sample. The transcriptome reads for <i>S. sirkka</i> and <i>S. napiecek</i> can be found under the EBI/ENA accession: ERR2729814 and ERR2729813 respectively. The transcriptome reads for <i>S. tapetis</i>, <i>S nootkatensis</i> and <i>S. gastrica</i> can be found under the EBI/ENA accession: PRJEB34306.</p> <p>To search for possible conserved lincRNAs in other species, we performed Blastn searches against transcriptomes and genomes of several holozoan species (<a href="#bib77">Richter et al., 2018</a>; <a href="#bib18">de Mendoza et al., 2015</a>; <a href="#bib83">Sebé-Pedrós et al., 2013</a>) (<a href="#fig4sdata2">Figure 4—source data 2</a>). All transcripts providing hits with an e-value less than 1e-50 were considered to be a conserved homolog.</p> </div> <div class="genome-wideanalysisofalternativesplicing" title="sec"> <div class="title" tagx="title" title="title">
Genome-wide analysis of alternative splicing</div> <p>Each RNA-seq run was independently aligned to the <i>Sphaeroforma</i> genome using Hisat2 (<a href="#bib49">Kim et al., 2015a</a>), using default parameters except for longer anchor lengths to faciliate de novo transcriptome assembly (--dta flag).</p> <p>The resulting alignments (in SAM format) were used to build sample-specific transcriptome assemblies with Stringtie2, using existing gene models (in GFF format, -G flag) as a reference, a minimum isoform abundance of 0.01 (-f flag) and a minimum isoform length of 50 bp (-m flag). For each sample, we only retained transcripts that overlapped with known genes in the final GFF file (using bedtools; <a href="#bib75">Quinlan and Hall, 2010</a>). Then, we built a consolidated set of isoforms by pooling all sample-specific GFF annotations and the reference annotation using Stringtie2 (--merge flag), without imposing any limitation of minimum expression levels (-T flag set to 0), and retaining isoforms with retained introns (-i flag). We also calculated the expression levels at the isoform level using Salmon (<a href="#bib70">Patro et al., 2017</a>) (output in TPM). Then, we used SUPPPA2 (<a href="#bib88">Trincado et al., 2018</a>) to generate a set of alternative splicing events, for which their frequencies were calculated for each sample. Specifically, we used the consolidated set of isoforms (GFF format) to obtain a list of all possible AS events using SUPPA2 <i>generateEvents</i> mode (setting the output to exon skipping [SE], mutually exclusive exons [MX] and intron retention [RI]), and -l 10. Then, we used SUPPA psiPerEvent mode to calculate the PSI values of each AS event for each sample, using the expression levels of each isoform (obtained from Salmon) as a reference. Differential splicing was quantified by calculating the calculating the differential PSI values between the average of each sample group (growth stage [t = 12 hr to t = 48 hr] compared to cellularization stage [t = 54 hr to t = 66 hr]). <i>p-</i>values were obtained using the empirical significance calculation protocol described in SUPPA2.</p> <p>After running SUPPA2, we produced functional annotations of the effect of each AS event on the final transcript. First, we annotated the protein domains in our consesus gene models using Pfam (version 31) and Pfamscan. These protein-level coordinates were converted to genomic coordinates using BLAT alignments (version 36, by aligning protein sequences to 6-frame translations of the genome) (<a href="#bib48">Kent, 2002</a>) and intersected with the genomic coordinates of AS events using the GenomicRanges and IRanges libraries (<i>findOverlapPairs</i> module) (<a href="#bib53">Lawrence et al., 2013</a>) from the R statistical framework (version 3.5.2). Second, we used the genomic coordinates of all AS events to recode each AS event as insertion/deletion variants that were amendable for analysis using the Variant Effect Predictor software (version 92.3) (<a href="#bib62">McLaren et al., 2016</a>) (SE were encoded as deletions, RI as insertions, and MX as insertion/deletion complex events). VEP produced a list of the effects of each AS variant (according to the Sequence Ontology nomenclature; <a href="#bib25">Eilbeck et al., 2005</a>), using our consensus gene models as a reference.</p> </div> <div class="phylostratigraphyanalysis" title="sec"> <div class="title" tagx="title" title="title">
Phylostratigraphy analysis</div> <p>First, we performed gene orthology assignment by searching for orthologs in a representative set of 30 publicly available eukaryotic proteome sequences (animals: <i>H. sapiens, S. kowalewskii, D. melanogaster, T. tribolium, N. vectensis, T. adherens, M. leidii and A. queenslandica</i>; choanoflagellates: <i>S. rosetta</i>; filastereans: <i>C. owczarczaki</i> and <i>M. vibrans</i>; teretosporeans: <i>S. arctica, C. fragrantissima, I. Hoferi, A. whisleri, C. perkinsii, P. gemmata, S. destruens, C. limatocisporum,</i> fungi: <i>F. alba, C. anguillulae, S. punctatus, M. verticillata</i> and <i>S. pombe</i>; other eukaryotes: <i>T. trahens, A. castellanii, D. discoideum, N. gruberi, T. thermophila, E. huxleyi, A. thaliana</i> and <i>P. yezoensis</i>) using orthofinder (<a href="#bib26">Emms and Kelly, 2015</a>) in Diamond mode. The longest isoform of each transcript was used. The generated orthogroups were used to determine orthologs of <i>S. arctica</i> genes in other species, unless a phylogeny of a specific gene family has been published.</p> <p>The orthofinder analysis resulted in 18797 orthogroups. To determine gene age of each orthogroup, the orthofinder output was passed to Count software (<a href="#bib17">Csurös, 2010</a>) to classify the <i>S. arctica</i> proteome into 10 phylostrata according to Dollo parsimony, ranging from paneukaryotic to <i>S. arctica</i>-specific. <i>S. arctica</i> genes that were not present in any orthogroup (i.e. did not have a single ortholog in any of the species) were also classified as <i>S. arctica</i>-specific. The enrichment analyses were performed using Fisher test implemented in R.</p> <p>The transcriptome age index (TAI) at each time point was computed as:</p> <p>TAI = sum<i><sub>i</sub></i>(ps<i><sub>i</sub></i> * e<i><sub>i</sub></i>)/sum<i><sub>i</sub></i>(e<i><sub>i</sub></i>) where ps<i><sub>i</sub></i> is an integer that represent the phylostratum of the gene <i>i</i> (with older genes assigned lower ps; for instance one for genes with paneukaryotic origin and 10 for genes with origin in <i>S. arctica</i>), and e<i><sub>i</sub></i> the expression level of each gene <i>i</i> (in tpm), according to <a href="#bib23">Domazet-Lošo and Tautz (2010)</a>. The reductive hourglass statistical test for the hourglass pattern was implemented using the R package myTAI (<a href="#bib24">Drost et al., 2015</a>), with time points 1 and 2 defined as early, time points 3–6 as mid, and time points 7–10 as late phases.</p> </div> <div class="flowcytometry" title="sec"> <div class="title" tagx="title" title="title">
Flow cytometry</div> <p>Flow cytometry was performed as described previously (<a href="#bib67">Ondracka et al., 2018</a>) Cells were fixed in 4% formaldehyde, 1M sorbitol solution for 15 min at room temperature, washed once with marine PBS (PBS with 35 g/L NaCl), and stained with DAPI (final concentration 0.5 μg/mL) in marine PBS. Samples were analyzed using an LSRII flow cytometer (BD Biosciences, USA) and the data were collected with FACSDiva software. DAPI signal was measured using a 355 nm laser with the 505 nm longpass and 530/30 nm bandpass filters. Approximately 2000 events were recorded in each measurement. The flow cytometry data were processed and analyzed using FloJo software (Ashland, OR).</p> </div> <div class="microscopy" title="sec"> <div class="title" tagx="title" title="title">
Microscopy</div> <p>Confocal microscopy of the spatiotemporal organization of actin in <a href="#fig2">Figure 2A</a> and <a href="#video5">Video 5</a>, was performed using a confocal laser scanning Leica TCS SP5 II microscope with an HC PL APO 63x/1.40 Oil CS2 oil objective. All remaining live and fixed images were obtained using a Zeiss Axio Observer Z.1 Epifluorescence inverted microscope equipped with Colibri LED illumination system and an Axiocam 503 mono camera. A Plan-Apochromat 63X/1.4 oil objective has been used for imaging fixed cells (<a href="#fig2">Figure 2C</a>, <a href="#fig2s1">Figure 2—figure supplements 1A</a>; <a href="#fig6">Figure 6B-D</a>), and an EC Plan-Neofluar 40x/0.75 air objective for <a href="#fig2s1">Figure 2—figure supplement 1C</a>, <a href="#video3">Videos 3</a> and <a href="#video4">4</a> and an N-ACHROPLAN 20x/0.45na Ph2 air objective for live imaging in <a href="#fig1">Figure 1C and B</a>, <a href="#fig1s1">Figure 1—figure supplement 1B</a>, <a href="#video1">Videos 1</a>, <a href="#video2">2</a>, <a href="#video6">6</a>, <a href="#video7">7</a> and <a href="#video8">8</a>.</p> </div> <div class="cellfixationandstaining" title="sec"> <div class="title" tagx="title" title="title">
Cell fixation and staining</div> <p>Cells were fixed using 4% formaldehyde and 250 mM sorbitol for 30 min before being washed twice with PBS. For actin and nuclei staining phalloidin (<a href="#fig2">Figures 2A, C</a> and <a href="#fig6">6C</a>, <a href="#fig2s1">Figure 2—figure supplement 1B and C</a>), cells were span down at 1,000 rpm for 3 min after fixation and washed again three times with PBS before adding 10 µl of Alexa Fluor 488–phalloidin (Invitrogen) and DAPI at a final concentration of 5 µg/mL to 5 µl of concentrated sample. For plasma membrane and cell wall staining (<a href="#fig2s1">Figure 2—figure supplement 1D</a>), cells were incubated for 10 min with FM4-64FX (Invitrogen) at a final concentration of 10 µM from 100 × DMSO diluted stock solution and Calcofluor white (Sigma-Aldrich) at a final concentration of 5 µg/ml from a 200 × stock solution prior to fixation. Cells were then fixed as previously mentioned and concentrated before being disposed between slide and coverslip.</p> <p>For <a href="#fig6">Figure 6A</a>, <a href="#fig6s1">Figure 6—figure supplement 1A and B</a>, cells were pre-grown at 12°C for 48 hr prior to fixing every hour for a total of 14 hr.</p> </div> <div class="livemicroscopyandpharmacologicalinhibitors" title="sec"> <div class="title" tagx="title" title="title">
Live microscopy and pharmacological inhibitors</div> <p>For live-cell imaging (<a href="#fig1">Figures 1C</a>–<a href="#fig1">1F</a>, <a href="#fig2">2B</a> and <a href="#fig6">6A</a>, <a href="#fig1s1">Figure 1—figure supplement 1A</a> to C, <a href="#video1">Videos 1</a>–<a href="#video4">4</a> and <a href="#video6">6</a>–<a href="#video8">8</a>) saturated culture was diluted 300x in fresh marine broth medium 1X inside a μ-Slide 4 or eight well slide (Ibidi) at time zero. To ensure oxygenation during the whole period of the experiment, the cover has been removed. To maintain the temperature at 12°C we used a P-Lab Tek (Pecon GmbH) Heating/Cooling system connected to a Lauda Ecoline E100 circulating water bath. To reduce light toxicity, we used a 495 nm Long Pass Filter (FGL495M- ThorLabs). Kymographs of cells undergoing cellularization in <a href="#fig1">Figure 1D</a> were constructed in ImageJ v1.46 by drawing a 3-pixel-wide (0.39 µm) line crossing the center of each cell.</p> <p>For plasma membrane live staining (<a href="#fig2">Figure 2B</a>, <a href="#fig2s1">Figure 2—figure supplement 1A</a>, <a href="#video3">Videos 3</a>, <a href="#video4">4</a>, <a href="#video7">7</a> and <a href="#video8">8</a>), FM4-64 (Invitrogen) at a final concentration of 10 µM from 100 × DMSO diluted stock solution was added after 58 hr of growth unless indicated otherwise in figure legends.</p> <p>Treatment with pharmacological inhibitors was performed on 54 hr grown cells inside a μ-Slide eight well slide (Ibidi) at 12°C and lasted for 24 hr during which live microscopy was performed. Latrunculin A (Sigma-Aldrich) was used at final concentration of 50 μM from a stock of 20 mM in DMSO. CK666 (Sigma-Aldrich) was used at a final concentration of 100 μM from a stock of 10 mM in DMSO. SMIFH2 (Sigma-Aldrich) was used at a final concentration of 40 μM from a stock of 10 mM in DMSO. Blebbistatin (Sigma-Aldrich) was used at a final concentration of 1.25 μM from a stock of 2.5 mM in DMSO. MBC (Sigma-Aldrich) was used at final concentration of 130 μg/ml from a stock of 2.5 mg/ml in dimethyl sulfoxide (DMSO).</p> </div> <div class="imageanalysis" title="sec"> <div class="title" tagx="title" title="title">
Image analysis</div> <p>Image analysis was done using ImageJ software (version 1.52) (<a href="#bib81">Schneider et al., 2012</a>). For measurements of cell diameter in <a href="#fig1">Figure 1E</a> and <a href="#fig1s1">Figure 1—figure supplement 1A</a>, we cropped movies to ensure having a single cell per movie. We then transformed the movies into binaries to ensure later segmentation. We then used particle analysis function in ImageJ with a circularity parameter set to 0.65–1 to quantify measure cell perimeter. As cells are spherical, we computed cell diameter as:</p> <p>For quantification of fraction of cells in each stage of cellularization <a href="#fig2s1">Figure 2—figure supplement 1B</a>, we used the ObjectJ plugin in ImageJ (National Institutes of Health).</p> <p>All Figures were assembled with Illustrator CC 2017 (Adobe).</p> </div> </div> </div> <div class="back" title="back"> <div class="funding-information" title="funding-information"> <div class="title" tagx="title" title="title">
Funding Information</div> <p>This paper was supported by the following grants:</p> <ul> <li> <p> </p><div class="funding-source" title="funding-source"> <div class="institution-wrap" title="institution-wrap"><span class="institution-id" title="institution-id">http://dx.doi.org/10.13039/501100000781</span><span class="institution" title="institution">European Research Council</span></div> </div><span class="award-id" tagx="award-id" title="award-id">Consolidator Grant ERC-2012-Co-616960</span> to Iñaki Ruiz-Trillo.<p /> </li> <li> <p> </p><div class="funding-source" title="funding-source"> <div class="institution-wrap" title="institution-wrap"><span class="institution-id" title="institution-id">http://dx.doi.org/10.13039/501100001700</span><span class="institution" title="institution">Ministry of Education, Culture, Sports, Science and Technology</span></div> </div><span class="award-id" tagx="award-id" title="award-id">MEXT KAKENHI 221S0002</span> to Atsushi Toyoda.<p /> </li> <li> <p> </p><div class="funding-source" title="funding-source"> <div class="institution-wrap" title="institution-wrap"><span class="institution-id" title="institution-id">http://dx.doi.org/10.13039/501100001700</span><span class="institution" title="institution">Ministry of Education, Culture, Sports, Science and Technology</span></div> </div><span class="award-id" tagx="award-id" title="award-id">MEXT KAKENHI 26891021</span> to Hiroshi Suga.<p /> </li> <li> <p> </p><div class="funding-source" title="funding-source"> <div class="institution-wrap" title="institution-wrap"><span class="institution-id" title="institution-id">http://dx.doi.org/10.13039/501100005416</span><span class="institution" title="institution">Research Council of Norway</span></div> </div><span class="award-id" tagx="award-id" title="award-id">Young Research Talents grant 240284</span> to Jon Bråte.<p /> </li> <li> <p> </p><div class="funding-source" title="funding-source"> <div class="institution-wrap" title="institution-wrap"><span class="institution-id" title="institution-id">http://dx.doi.org/10.13039/501100001711</span><span class="institution" title="institution">Swiss National Science Foundation</span></div> </div><span class="award-id" tagx="award-id" title="award-id">P2LAP3_171815</span> to Omaya Dudin.<p /> </li> <li> <p> </p><div class="funding-source" title="funding-source"> <div class="institution-wrap" title="institution-wrap"><span class="institution-id" title="institution-id">http://dx.doi.org/10.13039/100010665</span><span class="institution" title="institution">H2020 Marie Skłodowska-Curie Actions</span></div> </div><span class="award-id" tagx="award-id" title="award-id">Individual fellowship MSCA-IF 746044</span> to Omaya Dudin.<p /> </li> <li> <p> </p><div class="funding-source" title="funding-source"> <div class="institution-wrap" title="institution-wrap"><span class="institution-id" title="institution-id">http://dx.doi.org/10.13039/100010665</span><span class="institution" title="institution">H2020 Marie Skłodowska-Curie Actions</span></div> </div><span class="award-id" tagx="award-id" title="award-id">Individual fellowship MSCA-IF 747086</span> to Andrej Ondracka.<p /> </li> </ul> </div> <div class="ack" title="ack"> <div class="title" tagx="title" title="title">
Acknowledgements</div> <p>We thank Sophie Martin, Aaron New, Daniel Richter and Sébastien Wielgoss for discussion and comments on the manuscript, Takaaki Kai for discussion on the ichthyosporean development, Eduard Ocaña for advice on the phylostratigraphic analysis, and Meritxell Antó for technical support. We also acknowledge the UPF Flow Cytometry Core Facility for assistance with flow cytometry, CRG Advanced light microscopy unit for support with confocal imaging, and the CRG Genomics Unit for mRNA library preparation and Illumina sequencing. This work is dedicated to the memory of Arthur Haraldsen, our dear friend and colleague, who tragically passed away during the final stages of the manuscript preparation.</p> <p>This work was funded by European Research Council Consolidator Grant (ERC-2012-Co −616960) to IR-T; MEXT KAKENHI (grant Nos. 221S0002 to AT; grants no 26891021 and 16K07468 to HS); and a Young Research Talents grant from the Research Council of Norway (grant no. 240284) to JB. OD was supported by a Swiss National Science Foundation Early PostDoc Mobility fellowship (P2LAP3_171815) and a Marie Sklodowska-Curie individual fellowship (MSCA-IF 746044). AO was supported by a Marie Sklodowska-Curie individual fellowship (MSCA-IF 747086).</p> </div> <div class="additional-information" title="additional-information"> <div class="title" tagx="title" title="title">
Additional information</div> <div class="fn-group" title="fn-group"> <div class="title" tagx="title" title="title"><b>Competing interests</b></div> <div class="fn-type-COI-statement" title="COI-statement"> <p>No competing interests declared.</p> </div> </div> <div class="fn-group" title="fn-group"> <div class="title" tagx="title" title="title"><b>Author contributions</b></div> <div class="fn-type-con" title="con"> <p>Conceptualization, Data curation, Formal analysis, Funding acquisition, Validation, Investigation, Visualization, Methodology, Writing—original draft.</p> </div> <div class="fn-type-con" title="con"> <p>Conceptualization, Resources, Data curation, Software, Formal analysis, Funding acquisition, Validation, Investigation, Visualization, Methodology, Writing—original draft.</p> </div> <div class="fn-type-con" title="con"> <p>Data curation, Software, Formal analysis, Investigation, Visualization, Methodology, Writing—review and editing.</p> </div> <div class="fn-type-con" title="con"> <p>Data curation, Software, Formal analysis, Investigation, Methodology, Writing—review and editing.</p> </div> <div class="fn-type-con" title="con"> <p>Resources.</p> </div> <div class="fn-type-con" title="con"> <p>Resources, Formal analysis, Methodology, Writing—review and editing.</p> </div> <div class="fn-type-con" title="con"> <p>Resources, Formal analysis, Supervision, Investigation, Visualization, Methodology, Writing—review and editing.</p> </div> <div class="fn-type-con" title="con"> <p>Supervision, Funding acquisition, Writing—review and editing.</p> </div> </div> </div> <div class="supplementary-material" title="supplementary-material"> <div class="title" tagx="title" title="title">
Additional files</div> <div class="local-data" title="local-data"><span class="label" tagx="label" title="label">Transparent reporting form</span><div class="media" title="media"><a href="elife-49801-transrepform.docx">LINK</a></div> </div> </div> <div class="data-availability" title="data-availability"> <div class="title" tagx="title" title="title">
Data availability</div> <p>Sequencing data have been deposited at the following locations: S. arctica genome assembly - BioProject number PRJDB8476; S. arctica transcriptomes - PRJEB32922 (ERP115662) on European Nucleotide Archive.</p> <p>The following datasets were generated:</p> <p><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Suga</span><span class="given-names" tagx="given-names" title="given-names">H</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Toyoda</span><span class="given-names" tagx="given-names" title="given-names">A</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ruiz-Trillo</span><span class="given-names" tagx="given-names" title="given-names">I</span></span></span><span class="year" tagx="year" title="year">2018</span></span></p><div tag="data-title">
Sphaeroforma arctica genome assembly</div><span class="source" tagx="source" title="source">NCBI BioProject</span><span class="pub-id" /><p /> <p><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ondracka</span><span class="given-names" tagx="given-names" title="given-names">A</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Dudin</span><span class="given-names" tagx="given-names" title="given-names">O</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ruiz-Trillo</span><span class="given-names" tagx="given-names" title="given-names">I</span></span></span><span class="year" tagx="year" title="year">2018</span></span></p><div tag="data-title">
Dynamics of transcription during the coenocytic cycle of Sphaeroforma arctica</div><span class="source" tagx="source" title="source">European Nucleotide Archive</span><span class="pub-id" /><p /> </div> <div class="references">
References</div> <div tag="ref-list"> <ul> <div class="title" tagx="title" title="title">
References</div> <li tag="ref"><a name="bib1" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Abu-Jamous</span><span class="given-names" tagx="given-names" title="given-names">B</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kelly</span><span class="given-names" tagx="given-names" title="given-names">S</span></span></span><span class="year" tagx="year" title="year">2018</span><span class="mixed-article-title" title="mixed-article-title">Clust: automatic extraction of optimal co-expressed gene clusters from gene expression data</span><span class="source" tagx="source" title="source">Genome Biology</span><span class="volume" tagx="volume" title="volume">19</span><span class="elocation-id" tagx="elocation-id" title="elocation-id">172</span><span class="pub-id"><a href="https://dx.doi.org/10.1186/s13059-018-1536-8">10.1186/s13059-018-1536-8</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/30359297">30359297</a></span></span></li> <li tag="ref"><a name="bib2" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Adam</span><span class="given-names" tagx="given-names" title="given-names">JC</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Pringle</span><span class="given-names" tagx="given-names" title="given-names">JR</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Peifer</span><span class="given-names" tagx="given-names" title="given-names">M</span></span></span><span class="year" tagx="year" title="year">2000</span><span class="mixed-article-title" title="mixed-article-title">Evidence for functional differentiation among Drosophila septins in cytokinesis and cellularization</span><span class="source" tagx="source" title="source">Molecular Biology of the Cell</span><span class="volume" tagx="volume" title="volume">11</span><span class="fpage" tagx="fpage" title="fpage">3123</span><span class="lpage" tagx="lpage" title="lpage">3135</span><span class="pub-id"><a href="https://dx.doi.org/10.1091/mbc.11.9.3123">10.1091/mbc.11.9.3123</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/10982405">10982405</a></span></span></li> <li tag="ref"><a name="bib3" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Afshar</span><span class="given-names" tagx="given-names" title="given-names">K</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Stuart</span><span class="given-names" tagx="given-names" title="given-names">B</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wasserman</span><span class="given-names" tagx="given-names" title="given-names">SA</span></span></span><span class="year" tagx="year" title="year">2000</span><span class="mixed-article-title" title="mixed-article-title">Functional analysis of the Drosophila diaphanous FH protein in early embryonic development</span><span class="source" tagx="source" title="source">Development</span><span class="volume" tagx="volume" title="volume">127</span><span class="fpage" tagx="fpage" title="fpage">1887</span><span class="lpage" tagx="lpage" title="lpage">1897</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/10751177">10751177</a></span></span></li> <li tag="ref"><a name="bib4" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Alexa</span><span class="given-names" tagx="given-names" title="given-names">A</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Rahnenführer</span><span class="given-names" tagx="given-names" title="given-names">J</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lengauer</span><span class="given-names" tagx="given-names" title="given-names">T</span></span></span><span class="year" tagx="year" title="year">2006</span><span class="mixed-article-title" title="mixed-article-title">Improved scoring of functional groups from gene expression data by decorrelating GO graph structure</span><span class="source" tagx="source" title="source">Bioinformatics</span><span class="volume" tagx="volume" title="volume">22</span><span class="fpage" tagx="fpage" title="fpage">1600</span><span class="lpage" tagx="lpage" title="lpage">1607</span><span class="pub-id"><a href="https://dx.doi.org/10.1093/bioinformatics/btl140">10.1093/bioinformatics/btl140</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16606683">16606683</a></span></span></li> <li tag="ref"><a name="bib5" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Altschul</span><span class="given-names" tagx="given-names" title="given-names">SF</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gish</span><span class="given-names" tagx="given-names" title="given-names">W</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Miller</span><span class="given-names" tagx="given-names" title="given-names">W</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Myers</span><span class="given-names" tagx="given-names" title="given-names">EW</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lipman</span><span class="given-names" tagx="given-names" title="given-names">DJ</span></span></span><span class="year" tagx="year" title="year">1990</span><span class="mixed-article-title" title="mixed-article-title">Basic local alignment search tool</span><span class="source" tagx="source" title="source">Journal of Molecular Biology</span><span class="volume" tagx="volume" title="volume">215</span><span class="fpage" tagx="fpage" title="fpage">403</span><span class="lpage" tagx="lpage" title="lpage">410</span><span class="pub-id"><a href="https://dx.doi.org/10.1016/S0022-2836(05)80360-2">10.1016/S0022-2836(05)80360-2</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/2231712">2231712</a></span></span></li> <li tag="ref"><a name="bib6" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Barter</span><span class="given-names" tagx="given-names" title="given-names">RL</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Yu</span><span class="given-names" tagx="given-names" title="given-names">B</span></span></span><span class="year" tagx="year" title="year">2018</span><span class="mixed-article-title" title="mixed-article-title">Superheat: an R package for creating beautiful and extendable heatmaps for visualizing complex data</span><span class="source" tagx="source" title="source">Journal of Computational and Graphical Statistics</span><span class="volume" tagx="volume" title="volume">27</span><span class="fpage" tagx="fpage" title="fpage">910</span><span class="lpage" tagx="lpage" title="lpage">922</span><span class="pub-id"><a href="https://dx.doi.org/10.1080/10618600.2018.1473780">10.1080/10618600.2018.1473780</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/30911216">30911216</a></span></span></li> <li tag="ref"><a name="bib7" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Betz</span><span class="given-names" tagx="given-names" title="given-names">WJ</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Mao</span><span class="given-names" tagx="given-names" title="given-names">F</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Smith</span><span class="given-names" tagx="given-names" title="given-names">CB</span></span></span><span class="year" tagx="year" title="year">1996</span><span class="mixed-article-title" title="mixed-article-title">Imaging exocytosis and endocytosis</span><span class="source" tagx="source" title="source">Current Opinion in Neurobiology</span><span class="volume" tagx="volume" title="volume">6</span><span class="fpage" tagx="fpage" title="fpage">365</span><span class="lpage" tagx="lpage" title="lpage">371</span><span class="pub-id"><a href="https://dx.doi.org/10.1016/S0959-4388(96)80121-8">10.1016/S0959-4388(96)80121-8</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/8794083">8794083</a></span></span></li> <li tag="ref"><a name="bib8" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bolger</span><span class="given-names" tagx="given-names" title="given-names">AM</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lohse</span><span class="given-names" tagx="given-names" title="given-names">M</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Usadel</span><span class="given-names" tagx="given-names" title="given-names">B</span></span></span><span class="year" tagx="year" title="year">2014</span><span class="mixed-article-title" title="mixed-article-title">Trimmomatic: a flexible trimmer for illumina sequence data</span><span class="source" tagx="source" title="source">Bioinformatics</span><span class="volume" tagx="volume" title="volume">30</span><span class="fpage" tagx="fpage" title="fpage">2114</span><span class="lpage" tagx="lpage" title="lpage">2120</span><span class="pub-id"><a href="https://dx.doi.org/10.1093/bioinformatics/btu170">10.1093/bioinformatics/btu170</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24695404">24695404</a></span></span></li> <li tag="ref"><a name="bib9" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bonner</span><span class="given-names" tagx="given-names" title="given-names">JT</span></span></span><span class="year" tagx="year" title="year">1998</span><span class="mixed-article-title" title="mixed-article-title">The origins of multicellularity</span><span class="source" tagx="source" title="source">Integrative Biology: Issues, News, and Reviews</span><span class="volume" tagx="volume" title="volume">1</span><span class="fpage" tagx="fpage" title="fpage">27</span><span class="lpage" tagx="lpage" title="lpage">36</span><span class="pub-id"><a href="https://dx.doi.org/10.1002/(SICI)1520-6602(1998)1:1&lt;27::AID-INBI4&gt;3.0.CO;2-6">10.1002/(SICI)1520-6602(1998)1:1&amp;lt;27::AID-INBI4&amp;gt;3.0.CO;2-6</a></span></span></li> <li tag="ref"><a name="bib10" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Braet</span><span class="given-names" tagx="given-names" title="given-names">F</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">De Zanger</span><span class="given-names" tagx="given-names" title="given-names">R</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Jans</span><span class="given-names" tagx="given-names" title="given-names">D</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Spector</span><span class="given-names" tagx="given-names" title="given-names">I</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wisse</span><span class="given-names" tagx="given-names" title="given-names">E</span></span></span><span class="year" tagx="year" title="year">1996</span><span class="mixed-article-title" title="mixed-article-title">Microfilament-disrupting agent latrunculin A induces and increased number of fenestrae in rat liver sinusoidal endothelial cells: comparison with cytochalasin B</span><span class="source" tagx="source" title="source">Hepatology</span><span class="volume" tagx="volume" title="volume">24</span><span class="fpage" tagx="fpage" title="fpage">627</span><span class="lpage" tagx="lpage" title="lpage">635</span><span class="pub-id"><a href="https://dx.doi.org/10.1002/hep.510240327">10.1002/hep.510240327</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/8781335">8781335</a></span></span></li> <li tag="ref"><a name="bib11" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bråte</span><span class="given-names" tagx="given-names" title="given-names">J</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Adamski</span><span class="given-names" tagx="given-names" title="given-names">M</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Neumann</span><span class="given-names" tagx="given-names" title="given-names">RS</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Shalchian-Tabrizi</span><span class="given-names" tagx="given-names" title="given-names">K</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Adamska</span><span class="given-names" tagx="given-names" title="given-names">M</span></span></span><span class="year" tagx="year" title="year">2015</span><span class="mixed-article-title" title="mixed-article-title">Regulatory RNA at the root of animals: dynamic expression of developmental lincRNAs in the calcisponge <i>Sycon ciliatum</i></span><span class="source" tagx="source" title="source">Proceedings of the Royal Society B: Biological Sciences</span><span class="volume" tagx="volume" title="volume">282</span><span class="elocation-id" tagx="elocation-id" title="elocation-id">20151746</span><span class="pub-id"><a href="https://dx.doi.org/10.1098/rspb.2015.1746">10.1098/rspb.2015.1746</a></span></span></li> <li tag="ref"><a name="bib12" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bray</span><span class="given-names" tagx="given-names" title="given-names">NL</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Pimentel</span><span class="given-names" tagx="given-names" title="given-names">H</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Melsted</span><span class="given-names" tagx="given-names" title="given-names">P</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Pachter</span><span class="given-names" tagx="given-names" title="given-names">L</span></span></span><span class="year" tagx="year" title="year">2016</span><span class="mixed-article-title" title="mixed-article-title">Near-optimal probabilistic RNA-seq quantification</span><span class="source" tagx="source" title="source">Nature Biotechnology</span><span class="volume" tagx="volume" title="volume">34</span><span class="fpage" tagx="fpage" title="fpage">525</span><span class="lpage" tagx="lpage" title="lpage">527</span><span class="pub-id"><a href="https://dx.doi.org/10.1038/nbt.3519">10.1038/nbt.3519</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/27043002">27043002</a></span></span></li> <li tag="ref"><a name="bib13" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Brunet</span><span class="given-names" tagx="given-names" title="given-names">T</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">King</span><span class="given-names" tagx="given-names" title="given-names">N</span></span></span><span class="year" tagx="year" title="year">2017</span><span class="mixed-article-title" title="mixed-article-title">The origin of animal multicellularity and cell differentiation</span><span class="source" tagx="source" title="source">Developmental Cell</span><span class="volume" tagx="volume" title="volume">43</span><span class="fpage" tagx="fpage" title="fpage">124</span><span class="lpage" tagx="lpage" title="lpage">140</span><span class="pub-id"><a href="https://dx.doi.org/10.1016/j.devcel.2017.09.016">10.1016/j.devcel.2017.09.016</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/29065305">29065305</a></span></span></li> <li tag="ref"><a name="bib14" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bucci</span><span class="given-names" tagx="given-names" title="given-names">C</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Parton</span><span class="given-names" tagx="given-names" title="given-names">RG</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Mather</span><span class="given-names" tagx="given-names" title="given-names">IH</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Stunnenberg</span><span class="given-names" tagx="given-names" title="given-names">H</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Simons</span><span class="given-names" tagx="given-names" title="given-names">K</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hoflack</span><span class="given-names" tagx="given-names" title="given-names">B</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zerial</span><span class="given-names" tagx="given-names" title="given-names">M</span></span></span><span class="year" tagx="year" title="year">1992</span><span class="mixed-article-title" title="mixed-article-title">The small GTPase rab5 functions as a regulatory factor in the early endocytic pathway</span><span class="source" tagx="source" title="source">Cell</span><span class="volume" tagx="volume" title="volume">70</span><span class="fpage" tagx="fpage" title="fpage">715</span><span class="lpage" tagx="lpage" title="lpage">728</span><span class="pub-id"><a href="https://dx.doi.org/10.1016/0092-8674(92)90306-W">10.1016/0092-8674(92)90306-W</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/1516130">1516130</a></span></span></li> <li tag="ref"><a name="bib15" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Castagnetti</span><span class="given-names" tagx="given-names" title="given-names">S</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Novák</span><span class="given-names" tagx="given-names" title="given-names">B</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Nurse</span><span class="given-names" tagx="given-names" title="given-names">P</span></span></span><span class="year" tagx="year" title="year">2007</span><span class="mixed-article-title" title="mixed-article-title">Microtubules offset growth site from the cell centre in fission yeast</span><span class="source" tagx="source" title="source">Journal of Cell Science</span><span class="volume" tagx="volume" title="volume">120</span><span class="fpage" tagx="fpage" title="fpage">2205</span><span class="lpage" tagx="lpage" title="lpage">2213</span><span class="pub-id"><a href="https://dx.doi.org/10.1242/jcs.03464">10.1242/jcs.03464</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17591689">17591689</a></span></span></li> <li tag="ref"><a name="bib16" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Cooper</span><span class="given-names" tagx="given-names" title="given-names">JA</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kiehart</span><span class="given-names" tagx="given-names" title="given-names">DP</span></span></span><span class="year" tagx="year" title="year">1996</span><span class="mixed-article-title" title="mixed-article-title">Septins may form a ubiquitous family of cytoskeletal filaments</span><span class="source" tagx="source" title="source">The Journal of Cell Biology</span><span class="volume" tagx="volume" title="volume">134</span><span class="fpage" tagx="fpage" title="fpage">1345</span><span class="lpage" tagx="lpage" title="lpage">1348</span><span class="pub-id"><a href="https://dx.doi.org/10.1083/jcb.134.6.1345">10.1083/jcb.134.6.1345</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/8830765">8830765</a></span></span></li> <li tag="ref"><a name="bib17" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Csurös</span><span class="given-names" tagx="given-names" title="given-names">M</span></span></span><span class="year" tagx="year" title="year">2010</span><span class="mixed-article-title" title="mixed-article-title">Count: evolutionary analysis of phylogenetic profiles with parsimony and likelihood</span><span class="source" tagx="source" title="source">Bioinformatics</span><span class="volume" tagx="volume" title="volume">26</span><span class="fpage" tagx="fpage" title="fpage">1910</span><span class="lpage" tagx="lpage" title="lpage">1912</span><span class="pub-id"><a href="https://dx.doi.org/10.1093/bioinformatics/btq315">10.1093/bioinformatics/btq315</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20551134">20551134</a></span></span></li> <li tag="ref"><a name="bib18" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">de Mendoza</span><span class="given-names" tagx="given-names" title="given-names">A</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Suga</span><span class="given-names" tagx="given-names" title="given-names">H</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Permanyer</span><span class="given-names" tagx="given-names" title="given-names">J</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Irimia</span><span class="given-names" tagx="given-names" title="given-names">M</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ruiz-Trillo</span><span class="given-names" tagx="given-names" title="given-names">I</span></span></span><span class="year" tagx="year" title="year">2015</span><span class="mixed-article-title" title="mixed-article-title">Complex transcriptional regulation and independent evolution of fungal-like traits in a relative of animals</span><span class="source" tagx="source" title="source">eLife</span><span class="volume" tagx="volume" title="volume">4</span><span class="elocation-id" tagx="elocation-id" title="elocation-id">e08904</span><span class="pub-id"><a href="https://dx.doi.org/10.7554/eLife.08904">10.7554/eLife.08904</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/26465111">26465111</a></span></span></li> <li tag="ref"><a name="bib19" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Dickinson</span><span class="given-names" tagx="given-names" title="given-names">DJ</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Nelson</span><span class="given-names" tagx="given-names" title="given-names">WJ</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Weis</span><span class="given-names" tagx="given-names" title="given-names">WI</span></span></span><span class="year" tagx="year" title="year">2011</span><span class="mixed-article-title" title="mixed-article-title">A polarized epithelium organized by beta- and alpha-catenin predates cadherin and metazoan origins</span><span class="source" tagx="source" title="source">Science</span><span class="volume" tagx="volume" title="volume">331</span><span class="fpage" tagx="fpage" title="fpage">1336</span><span class="lpage" tagx="lpage" title="lpage">1339</span><span class="pub-id"><a href="https://dx.doi.org/10.1126/science.1199633">10.1126/science.1199633</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21393547">21393547</a></span></span></li> <li tag="ref"><a name="bib20" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Dickinson</span><span class="given-names" tagx="given-names" title="given-names">DJ</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Nelson</span><span class="given-names" tagx="given-names" title="given-names">WJ</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Weis</span><span class="given-names" tagx="given-names" title="given-names">WI</span></span></span><span class="year" tagx="year" title="year">2012</span><span class="mixed-article-title" title="mixed-article-title">An epithelial tissue in Dictyostelium challenges the traditional origin of metazoan multicellularity</span><span class="source" tagx="source" title="source">BioEssays</span><span class="volume" tagx="volume" title="volume">34</span><span class="fpage" tagx="fpage" title="fpage">833</span><span class="lpage" tagx="lpage" title="lpage">840</span><span class="pub-id"><a href="https://dx.doi.org/10.1002/bies.201100187">10.1002/bies.201100187</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22930590">22930590</a></span></span></li> <li tag="ref"><a name="bib21" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Dobin</span><span class="given-names" tagx="given-names" title="given-names">A</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Davis</span><span class="given-names" tagx="given-names" title="given-names">CA</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Schlesinger</span><span class="given-names" tagx="given-names" title="given-names">F</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Drenkow</span><span class="given-names" tagx="given-names" title="given-names">J</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zaleski</span><span class="given-names" tagx="given-names" title="given-names">C</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Jha</span><span class="given-names" tagx="given-names" title="given-names">S</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Batut</span><span class="given-names" tagx="given-names" title="given-names">P</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Chaisson</span><span class="given-names" tagx="given-names" title="given-names">M</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gingeras</span><span class="given-names" tagx="given-names" title="given-names">TR</span></span></span><span class="year" tagx="year" title="year">2013</span><span class="mixed-article-title" title="mixed-article-title">STAR: ultrafast universal RNA-seq aligner</span><span class="source" tagx="source" title="source">Bioinformatics</span><span class="volume" tagx="volume" title="volume">29</span><span class="fpage" tagx="fpage" title="fpage">15</span><span class="lpage" tagx="lpage" title="lpage">21</span><span class="pub-id"><a href="https://dx.doi.org/10.1093/bioinformatics/bts635">10.1093/bioinformatics/bts635</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23104886">23104886</a></span></span></li> <li tag="ref"><a name="bib22" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Dollar</span><span class="given-names" tagx="given-names" title="given-names">G</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Struckhoff</span><span class="given-names" tagx="given-names" title="given-names">E</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Michaud</span><span class="given-names" tagx="given-names" title="given-names">J</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Cohen</span><span class="given-names" tagx="given-names" title="given-names">RS</span></span></span><span class="year" tagx="year" title="year">2002</span><span class="mixed-article-title" title="mixed-article-title">Rab11 polarization of the Drosophila oocyte: a novel link between membrane trafficking, microtubule organization, and oskar mRNA localization and translation</span><span class="source" tagx="source" title="source">Development</span><span class="volume" tagx="volume" title="volume">129</span><span class="fpage" tagx="fpage" title="fpage">517</span><span class="lpage" tagx="lpage" title="lpage">526</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11807042">11807042</a></span></span></li> <li tag="ref"><a name="bib23" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Domazet-Lošo</span><span class="given-names" tagx="given-names" title="given-names">T</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Tautz</span><span class="given-names" tagx="given-names" title="given-names">D</span></span></span><span class="year" tagx="year" title="year">2010</span><span class="mixed-article-title" title="mixed-article-title">A phylogenetically based transcriptome age index mirrors ontogenetic divergence patterns</span><span class="source" tagx="source" title="source">Nature</span><span class="volume" tagx="volume" title="volume">468</span><span class="fpage" tagx="fpage" title="fpage">815</span><span class="lpage" tagx="lpage" title="lpage">818</span><span class="pub-id"><a href="https://dx.doi.org/10.1038/nature09632">10.1038/nature09632</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21150997">21150997</a></span></span></li> <li tag="ref"><a name="bib24" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Drost</span><span class="given-names" tagx="given-names" title="given-names">H-G</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gabel</span><span class="given-names" tagx="given-names" title="given-names">A</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Grosse</span><span class="given-names" tagx="given-names" title="given-names">I</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Quint</span><span class="given-names" tagx="given-names" title="given-names">M</span></span></span><span class="year" tagx="year" title="year">2015</span><span class="mixed-article-title" title="mixed-article-title">Evidence for active maintenance of phylotranscriptomic hourglass patterns in animal and plant embryogenesis</span><span class="source" tagx="source" title="source">Molecular Biology and Evolution</span><span class="volume" tagx="volume" title="volume">32</span><span class="fpage" tagx="fpage" title="fpage">1221</span><span class="lpage" tagx="lpage" title="lpage">1231</span><span class="pub-id"><a href="https://dx.doi.org/10.1093/molbev/msv012">10.1093/molbev/msv012</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25631928">25631928</a></span></span></li> <li tag="ref"><a name="bib25" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Eilbeck</span><span class="given-names" tagx="given-names" title="given-names">K</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lewis</span><span class="given-names" tagx="given-names" title="given-names">SE</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Mungall</span><span class="given-names" tagx="given-names" title="given-names">CJ</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Yandell</span><span class="given-names" tagx="given-names" title="given-names">M</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Stein</span><span class="given-names" tagx="given-names" title="given-names">L</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Durbin</span><span class="given-names" tagx="given-names" title="given-names">R</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ashburner</span><span class="given-names" tagx="given-names" title="given-names">M</span></span></span><span class="year" tagx="year" title="year">2005</span><span class="mixed-article-title" title="mixed-article-title">The sequence ontology: a tool for the unification of genome annotations</span><span class="source" tagx="source" title="source">Genome Biology</span><span class="volume" tagx="volume" title="volume">6</span><span class="elocation-id" tagx="elocation-id" title="elocation-id">R44</span><span class="pub-id"><a href="https://dx.doi.org/10.1186/gb-2005-6-5-r44">10.1186/gb-2005-6-5-r44</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15892872">15892872</a></span></span></li> <li tag="ref"><a name="bib26" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Emms</span><span class="given-names" tagx="given-names" title="given-names">DM</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kelly</span><span class="given-names" tagx="given-names" title="given-names">S</span></span></span><span class="year" tagx="year" title="year">2015</span><span class="mixed-article-title" title="mixed-article-title">OrthoFinder: solving fundamental biases in whole genome comparisons dramatically improves orthogroup inference accuracy</span><span class="source" tagx="source" title="source">Genome Biology</span><span class="volume" tagx="volume" title="volume">16</span><span class="fpage" tagx="fpage" title="fpage">1</span><span class="lpage" tagx="lpage" title="lpage">14</span><span class="pub-id"><a href="https://dx.doi.org/10.1186/s13059-015-0721-2">10.1186/s13059-015-0721-2</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25583448">25583448</a></span></span></li> <li tag="ref"><a name="bib27" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Farrell</span><span class="given-names" tagx="given-names" title="given-names">JA</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">O'Farrell</span><span class="given-names" tagx="given-names" title="given-names">PH</span></span></span><span class="year" tagx="year" title="year">2014</span><span class="mixed-article-title" title="mixed-article-title">From egg to gastrula: how the cell cycle is remodeled during the <i>Drosophila</i> mid-blastula transition</span><span class="source" tagx="source" title="source">Annual Review of Genetics</span><span class="volume" tagx="volume" title="volume">48</span><span class="fpage" tagx="fpage" title="fpage">269</span><span class="lpage" tagx="lpage" title="lpage">294</span><span class="pub-id"><a href="https://dx.doi.org/10.1146/annurev-genet-111212-133531">10.1146/annurev-genet-111212-133531</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25195504">25195504</a></span></span></li> <li tag="ref"><a name="bib28" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Figard</span><span class="given-names" tagx="given-names" title="given-names">L</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wang</span><span class="given-names" tagx="given-names" title="given-names">M</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zheng</span><span class="given-names" tagx="given-names" title="given-names">L</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Golding</span><span class="given-names" tagx="given-names" title="given-names">I</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Sokac</span><span class="given-names" tagx="given-names" title="given-names">AM</span></span></span><span class="year" tagx="year" title="year">2016</span><span class="mixed-article-title" title="mixed-article-title">Membrane supply and demand regulates F-Actin in a cell surface reservoir</span><span class="source" tagx="source" title="source">Developmental Cell</span><span class="volume" tagx="volume" title="volume">37</span><span class="fpage" tagx="fpage" title="fpage">267</span><span class="lpage" tagx="lpage" title="lpage">278</span><span class="pub-id"><a href="https://dx.doi.org/10.1016/j.devcel.2016.04.010">10.1016/j.devcel.2016.04.010</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/27165556">27165556</a></span></span></li> <li tag="ref"><a name="bib29" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Francia</span><span class="given-names" tagx="given-names" title="given-names">ME</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Striepen</span><span class="given-names" tagx="given-names" title="given-names">B</span></span></span><span class="year" tagx="year" title="year">2014</span><span class="mixed-article-title" title="mixed-article-title">Cell division in apicomplexan parasites</span><span class="source" tagx="source" title="source">Nature Reviews Microbiology</span><span class="volume" tagx="volume" title="volume">12</span><span class="fpage" tagx="fpage" title="fpage">125</span><span class="lpage" tagx="lpage" title="lpage">136</span><span class="pub-id"><a href="https://dx.doi.org/10.1038/nrmicro3184">10.1038/nrmicro3184</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24384598">24384598</a></span></span></li> <li tag="ref"><a name="bib30" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gaiti</span><span class="given-names" tagx="given-names" title="given-names">F</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Fernandez-Valverde</span><span class="given-names" tagx="given-names" title="given-names">SL</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Nakanishi</span><span class="given-names" tagx="given-names" title="given-names">N</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Calcino</span><span class="given-names" tagx="given-names" title="given-names">AD</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Yanai</span><span class="given-names" tagx="given-names" title="given-names">I</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Tanurdzic</span><span class="given-names" tagx="given-names" title="given-names">M</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Degnan</span><span class="given-names" tagx="given-names" title="given-names">BM</span></span></span><span class="year" tagx="year" title="year">2015</span><span class="mixed-article-title" title="mixed-article-title">Dynamic and widespread lncRNA expression in a sponge and the origin of animal complexity</span><span class="source" tagx="source" title="source">Molecular Biology and Evolution</span><span class="volume" tagx="volume" title="volume">32</span><span class="fpage" tagx="fpage" title="fpage">2367</span><span class="lpage" tagx="lpage" title="lpage">2382</span><span class="pub-id"><a href="https://dx.doi.org/10.1093/molbev/msv117">10.1093/molbev/msv117</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25976353">25976353</a></span></span></li> <li tag="ref"><a name="bib31" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Giansanti</span><span class="given-names" tagx="given-names" title="given-names">MG</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bonaccorsi</span><span class="given-names" tagx="given-names" title="given-names">S</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Williams</span><span class="given-names" tagx="given-names" title="given-names">B</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Williams</span><span class="given-names" tagx="given-names" title="given-names">EV</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Santolamazza</span><span class="given-names" tagx="given-names" title="given-names">C</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Goldberg</span><span class="given-names" tagx="given-names" title="given-names">ML</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gatti</span><span class="given-names" tagx="given-names" title="given-names">M</span></span></span><span class="year" tagx="year" title="year">1998</span><span class="mixed-article-title" title="mixed-article-title">Cooperative interactions between the central spindle and the contractile ring during Drosophila cytokinesis</span><span class="source" tagx="source" title="source">Genes &amp;amp; Development</span><span class="volume" tagx="volume" title="volume">12</span><span class="fpage" tagx="fpage" title="fpage">396</span><span class="lpage" tagx="lpage" title="lpage">410</span><span class="pub-id"><a href="https://dx.doi.org/10.1101/gad.12.3.396">10.1101/gad.12.3.396</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/9450933">9450933</a></span></span></li> <li tag="ref"><a name="bib32" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Grabherr</span><span class="given-names" tagx="given-names" title="given-names">MG</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Haas</span><span class="given-names" tagx="given-names" title="given-names">BJ</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Yassour</span><span class="given-names" tagx="given-names" title="given-names">M</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Levin</span><span class="given-names" tagx="given-names" title="given-names">JZ</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Thompson</span><span class="given-names" tagx="given-names" title="given-names">DA</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Amit</span><span class="given-names" tagx="given-names" title="given-names">I</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Adiconis</span><span class="given-names" tagx="given-names" title="given-names">X</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Fan</span><span class="given-names" tagx="given-names" title="given-names">L</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Raychowdhury</span><span class="given-names" tagx="given-names" title="given-names">R</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zeng</span><span class="given-names" tagx="given-names" title="given-names">Q</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Chen</span><span class="given-names" tagx="given-names" title="given-names">Z</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Mauceli</span><span class="given-names" tagx="given-names" title="given-names">E</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hacohen</span><span class="given-names" tagx="given-names" title="given-names">N</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gnirke</span><span class="given-names" tagx="given-names" title="given-names">A</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Rhind</span><span class="given-names" tagx="given-names" title="given-names">N</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">di Palma</span><span class="given-names" tagx="given-names" title="given-names">F</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Birren</span><span class="given-names" tagx="given-names" title="given-names">BW</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Nusbaum</span><span class="given-names" tagx="given-names" title="given-names">C</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lindblad-Toh</span><span class="given-names" tagx="given-names" title="given-names">K</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Friedman</span><span class="given-names" tagx="given-names" title="given-names">N</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Regev</span><span class="given-names" tagx="given-names" title="given-names">A</span></span></span><span class="year" tagx="year" title="year">2011</span><span class="mixed-article-title" title="mixed-article-title">Full-length transcriptome assembly from RNA-Seq data without a reference genome</span><span class="source" tagx="source" title="source">Nature Biotechnology</span><span class="volume" tagx="volume" title="volume">29</span><span class="fpage" tagx="fpage" title="fpage">644</span><span class="lpage" tagx="lpage" title="lpage">652</span><span class="pub-id"><a href="https://dx.doi.org/10.1038/nbt.1883">10.1038/nbt.1883</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21572440">21572440</a></span></span></li> <li tag="ref"><a name="bib33" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Grau-Bové</span><span class="given-names" tagx="given-names" title="given-names">X</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Torruella</span><span class="given-names" tagx="given-names" title="given-names">G</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Donachie</span><span class="given-names" tagx="given-names" title="given-names">S</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Suga</span><span class="given-names" tagx="given-names" title="given-names">H</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Leonard</span><span class="given-names" tagx="given-names" title="given-names">G</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Richards</span><span class="given-names" tagx="given-names" title="given-names">TA</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ruiz-Trillo</span><span class="given-names" tagx="given-names" title="given-names">I</span></span></span><span class="year" tagx="year" title="year">2017</span><span class="mixed-article-title" title="mixed-article-title">Dynamics of genomic innovation in the unicellular ancestry of animals</span><span class="source" tagx="source" title="source">eLife</span><span class="volume" tagx="volume" title="volume">6</span><span class="elocation-id" tagx="elocation-id" title="elocation-id">e26036</span><span class="pub-id"><a href="https://dx.doi.org/10.7554/eLife.26036">10.7554/eLife.26036</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/28726632">28726632</a></span></span></li> <li tag="ref"><a name="bib34" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Grau-Bové</span><span class="given-names" tagx="given-names" title="given-names">X</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ruiz-Trillo</span><span class="given-names" tagx="given-names" title="given-names">I</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Irimia</span><span class="given-names" tagx="given-names" title="given-names">M</span></span></span><span class="year" tagx="year" title="year">2018</span><span class="mixed-article-title" title="mixed-article-title">Origin of exon skipping-rich transcriptomes in animals driven by evolution of gene architecture</span><span class="source" tagx="source" title="source">Genome Biology</span><span class="volume" tagx="volume" title="volume">19</span><span class="elocation-id" tagx="elocation-id" title="elocation-id">135</span><span class="pub-id"><a href="https://dx.doi.org/10.1186/s13059-018-1499-9">10.1186/s13059-018-1499-9</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/30223879">30223879</a></span></span></li> <li tag="ref"><a name="bib35" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gunsalus</span><span class="given-names" tagx="given-names" title="given-names">KC</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bonaccorsi</span><span class="given-names" tagx="given-names" title="given-names">S</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Williams</span><span class="given-names" tagx="given-names" title="given-names">E</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Verni</span><span class="given-names" tagx="given-names" title="given-names">F</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gatti</span><span class="given-names" tagx="given-names" title="given-names">M</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Goldberg</span><span class="given-names" tagx="given-names" title="given-names">ML</span></span></span><span class="year" tagx="year" title="year">1995</span><span class="mixed-article-title" title="mixed-article-title">Mutations in Twinstar, a Drosophila gene encoding a cofilin/ADF homologue, result in defects in centrosome migration and cytokinesis</span><span class="source" tagx="source" title="source">The Journal of Cell Biology</span><span class="volume" tagx="volume" title="volume">131</span><span class="fpage" tagx="fpage" title="fpage">1243</span><span class="lpage" tagx="lpage" title="lpage">1259</span><span class="pub-id"><a href="https://dx.doi.org/10.1083/jcb.131.5.1243">10.1083/jcb.131.5.1243</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/8522587">8522587</a></span></span></li> <li tag="ref"><a name="bib36" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Haas</span><span class="given-names" tagx="given-names" title="given-names">BJ</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Delcher</span><span class="given-names" tagx="given-names" title="given-names">AL</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Mount</span><span class="given-names" tagx="given-names" title="given-names">SM</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wortman</span><span class="given-names" tagx="given-names" title="given-names">JR</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Smith</span><span class="given-names" tagx="given-names" title="given-names">RK</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hannick</span><span class="given-names" tagx="given-names" title="given-names">LI</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Maiti</span><span class="given-names" tagx="given-names" title="given-names">R</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ronning</span><span class="given-names" tagx="given-names" title="given-names">CM</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Rusch</span><span class="given-names" tagx="given-names" title="given-names">DB</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Town</span><span class="given-names" tagx="given-names" title="given-names">CD</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Salzberg</span><span class="given-names" tagx="given-names" title="given-names">SL</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">White</span><span class="given-names" tagx="given-names" title="given-names">O</span></span></span><span class="year" tagx="year" title="year">2003</span><span class="mixed-article-title" title="mixed-article-title">Improving the Arabidopsis genome annotation using maximal transcript alignment assemblies</span><span class="source" tagx="source" title="source">Nucleic Acids Research</span><span class="volume" tagx="volume" title="volume">31</span><span class="fpage" tagx="fpage" title="fpage">5654</span><span class="lpage" tagx="lpage" title="lpage">5666</span><span class="pub-id"><a href="https://dx.doi.org/10.1093/nar/gkg770">10.1093/nar/gkg770</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/14500829">14500829</a></span></span></li> <li tag="ref"><a name="bib37" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Haas</span><span class="given-names" tagx="given-names" title="given-names">BJ</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Papanicolaou</span><span class="given-names" tagx="given-names" title="given-names">A</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Yassour</span><span class="given-names" tagx="given-names" title="given-names">M</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Grabherr</span><span class="given-names" tagx="given-names" title="given-names">M</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Blood</span><span class="given-names" tagx="given-names" title="given-names">PD</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bowden</span><span class="given-names" tagx="given-names" title="given-names">J</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Couger</span><span class="given-names" tagx="given-names" title="given-names">MB</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Eccles</span><span class="given-names" tagx="given-names" title="given-names">D</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Li</span><span class="given-names" tagx="given-names" title="given-names">B</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lieber</span><span class="given-names" tagx="given-names" title="given-names">M</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">MacManes</span><span class="given-names" tagx="given-names" title="given-names">MD</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ott</span><span class="given-names" tagx="given-names" title="given-names">M</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Orvis</span><span class="given-names" tagx="given-names" title="given-names">J</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Pochet</span><span class="given-names" tagx="given-names" title="given-names">N</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Strozzi</span><span class="given-names" tagx="given-names" title="given-names">F</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Weeks</span><span class="given-names" tagx="given-names" title="given-names">N</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Westerman</span><span class="given-names" tagx="given-names" title="given-names">R</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">William</span><span class="given-names" tagx="given-names" title="given-names">T</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Dewey</span><span class="given-names" tagx="given-names" title="given-names">CN</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Henschel</span><span class="given-names" tagx="given-names" title="given-names">R</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">LeDuc</span><span class="given-names" tagx="given-names" title="given-names">RD</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Friedman</span><span class="given-names" tagx="given-names" title="given-names">N</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Regev</span><span class="given-names" tagx="given-names" title="given-names">A</span></span></span><span class="year" tagx="year" title="year">2013</span><span class="mixed-article-title" title="mixed-article-title">De novo transcript sequence reconstruction from RNA-seq using the trinity platform for reference generation and analysis</span><span class="source" tagx="source" title="source">Nature Protocols</span><span class="volume" tagx="volume" title="volume">8</span><span class="fpage" tagx="fpage" title="fpage">1494</span><span class="lpage" tagx="lpage" title="lpage">1512</span><span class="pub-id"><a href="https://dx.doi.org/10.1038/nprot.2013.084">10.1038/nprot.2013.084</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23845962">23845962</a></span></span></li> <li tag="ref"><a name="bib38" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Haglund</span><span class="given-names" tagx="given-names" title="given-names">K</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Nezis</span><span class="given-names" tagx="given-names" title="given-names">IP</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Stenmark</span><span class="given-names" tagx="given-names" title="given-names">H</span></span></span><span class="year" tagx="year" title="year">2011</span><span class="mixed-article-title" title="mixed-article-title">Structure and functions of stable intercellular bridges formed by incomplete cytokinesis during development</span><span class="source" tagx="source" title="source">Communicative &amp;amp; Integrative Biology</span><span class="volume" tagx="volume" title="volume">4</span><span class="fpage" tagx="fpage" title="fpage">1</span><span class="lpage" tagx="lpage" title="lpage">9</span><span class="pub-id"><a href="https://dx.doi.org/10.4161/cib.13550">10.4161/cib.13550</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21509167">21509167</a></span></span></li> <li tag="ref"><a name="bib39" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hehenberger</span><span class="given-names" tagx="given-names" title="given-names">E</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kradolfer</span><span class="given-names" tagx="given-names" title="given-names">D</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Köhler</span><span class="given-names" tagx="given-names" title="given-names">C</span></span></span><span class="year" tagx="year" title="year">2012</span><span class="mixed-article-title" title="mixed-article-title">Endosperm cellularization defines an important developmental transition for embryo development</span><span class="source" tagx="source" title="source">Development</span><span class="volume" tagx="volume" title="volume">139</span><span class="fpage" tagx="fpage" title="fpage">2031</span><span class="lpage" tagx="lpage" title="lpage">2039</span><span class="pub-id"><a href="https://dx.doi.org/10.1242/dev.077057">10.1242/dev.077057</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22535409">22535409</a></span></span></li> <li tag="ref"><a name="bib40" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hetrick</span><span class="given-names" tagx="given-names" title="given-names">B</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Han</span><span class="given-names" tagx="given-names" title="given-names">MS</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Helgeson</span><span class="given-names" tagx="given-names" title="given-names">LA</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Nolen</span><span class="given-names" tagx="given-names" title="given-names">BJ</span></span></span><span class="year" tagx="year" title="year">2013</span><span class="mixed-article-title" title="mixed-article-title">Small molecules CK-666 and CK-869 inhibit actin-related protein 2/3 complex by blocking an activating conformational change</span><span class="source" tagx="source" title="source">Chemistry &amp;amp; Biology</span><span class="volume" tagx="volume" title="volume">20</span><span class="fpage" tagx="fpage" title="fpage">701</span><span class="lpage" tagx="lpage" title="lpage">712</span><span class="pub-id"><a href="https://dx.doi.org/10.1016/j.chembiol.2013.03.019">10.1016/j.chembiol.2013.03.019</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23623350">23623350</a></span></span></li> <li tag="ref"><a name="bib41" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hezroni</span><span class="given-names" tagx="given-names" title="given-names">H</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Koppstein</span><span class="given-names" tagx="given-names" title="given-names">D</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Schwartz</span><span class="given-names" tagx="given-names" title="given-names">MG</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Avrutin</span><span class="given-names" tagx="given-names" title="given-names">A</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bartel</span><span class="given-names" tagx="given-names" title="given-names">DP</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ulitsky</span><span class="given-names" tagx="given-names" title="given-names">I</span></span></span><span class="year" tagx="year" title="year">2015</span><span class="mixed-article-title" title="mixed-article-title">Principles of long noncoding RNA evolution derived from direct comparison of transcriptomes in 17 species</span><span class="source" tagx="source" title="source">Cell Reports</span><span class="volume" tagx="volume" title="volume">11</span><span class="fpage" tagx="fpage" title="fpage">1110</span><span class="lpage" tagx="lpage" title="lpage">1122</span><span class="pub-id"><a href="https://dx.doi.org/10.1016/j.celrep.2015.04.023">10.1016/j.celrep.2015.04.023</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25959816">25959816</a></span></span></li> <li tag="ref"><a name="bib42" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hoff</span><span class="given-names" tagx="given-names" title="given-names">KJ</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lomsadze</span><span class="given-names" tagx="given-names" title="given-names">A</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Stanke</span><span class="given-names" tagx="given-names" title="given-names">M</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Borodovsky</span><span class="given-names" tagx="given-names" title="given-names">M</span></span></span><span class="year" tagx="year" title="year">2018</span><span class="mixed-article-title" title="mixed-article-title">BRAKER2: incorporating protein homology information into gene prediction with GeneMark-EP and AUGUSTUS</span><span class="conf-name" tagx="conf-name" title="conf-name">Plant and Animal Genomes XXVI, January 14th 2018</span></span></li> <li tag="ref"><a name="bib43" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Huerta-Cepas</span><span class="given-names" tagx="given-names" title="given-names">J</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Forslund</span><span class="given-names" tagx="given-names" title="given-names">K</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Coelho</span><span class="given-names" tagx="given-names" title="given-names">LP</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Szklarczyk</span><span class="given-names" tagx="given-names" title="given-names">D</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Jensen</span><span class="given-names" tagx="given-names" title="given-names">LJ</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">von Mering</span><span class="given-names" tagx="given-names" title="given-names">C</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bork</span><span class="given-names" tagx="given-names" title="given-names">P</span></span></span><span class="year" tagx="year" title="year">2017</span><span class="mixed-article-title" title="mixed-article-title">Fast Genome-Wide functional annotation through orthology assignment by eggNOG-Mapper</span><span class="source" tagx="source" title="source">Molecular Biology and Evolution</span><span class="volume" tagx="volume" title="volume">34</span><span class="fpage" tagx="fpage" title="fpage">2115</span><span class="lpage" tagx="lpage" title="lpage">2122</span><span class="pub-id"><a href="https://dx.doi.org/10.1093/molbev/msx148">10.1093/molbev/msx148</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/28460117">28460117</a></span></span></li> <li tag="ref"><a name="bib44" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hunter</span><span class="given-names" tagx="given-names" title="given-names">C</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wieschaus</span><span class="given-names" tagx="given-names" title="given-names">E</span></span></span><span class="year" tagx="year" title="year">2000</span><span class="mixed-article-title" title="mixed-article-title">Regulated expression of nullo is required for the formation of distinct apical and basal adherens junctions in the Drosophila blastoderm</span><span class="source" tagx="source" title="source">The Journal of Cell Biology</span><span class="volume" tagx="volume" title="volume">150</span><span class="fpage" tagx="fpage" title="fpage">391</span><span class="lpage" tagx="lpage" title="lpage">402</span><span class="pub-id"><a href="https://dx.doi.org/10.1083/jcb.150.2.391">10.1083/jcb.150.2.391</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/10908580">10908580</a></span></span></li> <li tag="ref"><a name="bib45" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kalvari</span><span class="given-names" tagx="given-names" title="given-names">I</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Argasinska</span><span class="given-names" tagx="given-names" title="given-names">J</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Quinones-Olvera</span><span class="given-names" tagx="given-names" title="given-names">N</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Nawrocki</span><span class="given-names" tagx="given-names" title="given-names">EP</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Rivas</span><span class="given-names" tagx="given-names" title="given-names">E</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Eddy</span><span class="given-names" tagx="given-names" title="given-names">SR</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bateman</span><span class="given-names" tagx="given-names" title="given-names">A</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Finn</span><span class="given-names" tagx="given-names" title="given-names">RD</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Petrov</span><span class="given-names" tagx="given-names" title="given-names">AI</span></span></span><span class="year" tagx="year" title="year">2018</span><span class="mixed-article-title" title="mixed-article-title">Rfam 13.0: shifting to a genome-centric resource for non-coding RNA families</span><span class="source" tagx="source" title="source">Nucleic Acids Research</span><span class="volume" tagx="volume" title="volume">46</span><span class="fpage" tagx="fpage" title="fpage">D335</span><span class="lpage" tagx="lpage" title="lpage">D342</span><span class="pub-id"><a href="https://dx.doi.org/10.1093/nar/gkx1038">10.1093/nar/gkx1038</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/29112718">29112718</a></span></span></li> <li tag="ref"><a name="bib46" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kang</span><span class="given-names" tagx="given-names" title="given-names">YJ</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Yang</span><span class="given-names" tagx="given-names" title="given-names">DC</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kong</span><span class="given-names" tagx="given-names" title="given-names">L</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hou</span><span class="given-names" tagx="given-names" title="given-names">M</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Meng</span><span class="given-names" tagx="given-names" title="given-names">YQ</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wei</span><span class="given-names" tagx="given-names" title="given-names">L</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gao</span><span class="given-names" tagx="given-names" title="given-names">G</span></span></span><span class="year" tagx="year" title="year">2017</span><span class="mixed-article-title" title="mixed-article-title">CPC2: a fast and accurate coding potential calculator based on sequence intrinsic features</span><span class="source" tagx="source" title="source">Nucleic Acids Research</span><span class="volume" tagx="volume" title="volume">45</span><span class="fpage" tagx="fpage" title="fpage">W12</span><span class="lpage" tagx="lpage" title="lpage">W16</span><span class="pub-id"><a href="https://dx.doi.org/10.1093/nar/gkx428">10.1093/nar/gkx428</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/28521017">28521017</a></span></span></li> <li tag="ref"><a name="bib47" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Keller</span><span class="given-names" tagx="given-names" title="given-names">O</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kollmar</span><span class="given-names" tagx="given-names" title="given-names">M</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Stanke</span><span class="given-names" tagx="given-names" title="given-names">M</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Waack</span><span class="given-names" tagx="given-names" title="given-names">S</span></span></span><span class="year" tagx="year" title="year">2011</span><span class="mixed-article-title" title="mixed-article-title">A novel hybrid gene prediction method employing protein multiple sequence alignments</span><span class="source" tagx="source" title="source">Bioinformatics</span><span class="volume" tagx="volume" title="volume">27</span><span class="fpage" tagx="fpage" title="fpage">757</span><span class="lpage" tagx="lpage" title="lpage">763</span><span class="pub-id"><a href="https://dx.doi.org/10.1093/bioinformatics/btr010">10.1093/bioinformatics/btr010</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21216780">21216780</a></span></span></li> <li tag="ref"><a name="bib48" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kent</span><span class="given-names" tagx="given-names" title="given-names">WJ</span></span></span><span class="year" tagx="year" title="year">2002</span><span class="mixed-article-title" title="mixed-article-title">BLAT--the BLAST-like alignment tool</span><span class="source" tagx="source" title="source">Genome Research</span><span class="volume" tagx="volume" title="volume">12</span><span class="fpage" tagx="fpage" title="fpage">656</span><span class="lpage" tagx="lpage" title="lpage">664</span><span class="pub-id"><a href="https://dx.doi.org/10.1101/gr.229202">10.1101/gr.229202</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11932250">11932250</a></span></span></li> <li tag="ref"><a name="bib49" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kim</span><span class="given-names" tagx="given-names" title="given-names">D</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Langmead</span><span class="given-names" tagx="given-names" title="given-names">B</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Salzberg</span><span class="given-names" tagx="given-names" title="given-names">SL</span></span></span><span class="year" tagx="year" title="year">2015a</span><span class="mixed-article-title" title="mixed-article-title">HISAT: a fast spliced aligner with low memory requirements</span><span class="source" tagx="source" title="source">Nature Methods</span><span class="volume" tagx="volume" title="volume">12</span><span class="fpage" tagx="fpage" title="fpage">357</span><span class="lpage" tagx="lpage" title="lpage">360</span><span class="pub-id"><a href="https://dx.doi.org/10.1038/nmeth.3317">10.1038/nmeth.3317</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25751142">25751142</a></span></span></li> <li tag="ref"><a name="bib50" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kim</span><span class="given-names" tagx="given-names" title="given-names">HC</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Jo</span><span class="given-names" tagx="given-names" title="given-names">YJ</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kim</span><span class="given-names" tagx="given-names" title="given-names">NH</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Namgoong</span><span class="given-names" tagx="given-names" title="given-names">S</span></span></span><span class="year" tagx="year" title="year">2015b</span><span class="mixed-article-title" title="mixed-article-title">Small molecule inhibitor of formin homology 2 domains (SMIFH2) reveals the roles of the formin family of proteins in spindle assembly and asymmetric division in mouse oocytes</span><span class="source" tagx="source" title="source">PLOS ONE</span><span class="volume" tagx="volume" title="volume">10</span><span class="elocation-id" tagx="elocation-id" title="elocation-id">e0123438</span><span class="pub-id"><a href="https://dx.doi.org/10.1371/journal.pone.0123438">10.1371/journal.pone.0123438</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25837661">25837661</a></span></span></li> <li tag="ref"><a name="bib51" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kovács</span><span class="given-names" tagx="given-names" title="given-names">M</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Tóth</span><span class="given-names" tagx="given-names" title="given-names">J</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hetényi</span><span class="given-names" tagx="given-names" title="given-names">C</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Málnási-Csizmadia</span><span class="given-names" tagx="given-names" title="given-names">A</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Sellers</span><span class="given-names" tagx="given-names" title="given-names">JR</span></span></span><span class="year" tagx="year" title="year">2004</span><span class="mixed-article-title" title="mixed-article-title">Mechanism of blebbistatin inhibition of myosin II</span><span class="source" tagx="source" title="source">Journal of Biological Chemistry</span><span class="volume" tagx="volume" title="volume">279</span><span class="fpage" tagx="fpage" title="fpage">35557</span><span class="lpage" tagx="lpage" title="lpage">35563</span><span class="pub-id"><a href="https://dx.doi.org/10.1074/jbc.M405319200">10.1074/jbc.M405319200</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15205456">15205456</a></span></span></li> <li tag="ref"><a name="bib52" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Langmead</span><span class="given-names" tagx="given-names" title="given-names">B</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Salzberg</span><span class="given-names" tagx="given-names" title="given-names">SL</span></span></span><span class="year" tagx="year" title="year">2012</span><span class="mixed-article-title" title="mixed-article-title">Fast gapped-read alignment with bowtie 2</span><span class="source" tagx="source" title="source">Nature Methods</span><span class="volume" tagx="volume" title="volume">9</span><span class="fpage" tagx="fpage" title="fpage">357</span><span class="lpage" tagx="lpage" title="lpage">359</span><span class="pub-id"><a href="https://dx.doi.org/10.1038/nmeth.1923">10.1038/nmeth.1923</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22388286">22388286</a></span></span></li> <li tag="ref"><a name="bib53" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lawrence</span><span class="given-names" tagx="given-names" title="given-names">M</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Huber</span><span class="given-names" tagx="given-names" title="given-names">W</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Pagès</span><span class="given-names" tagx="given-names" title="given-names">H</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Aboyoun</span><span class="given-names" tagx="given-names" title="given-names">P</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Carlson</span><span class="given-names" tagx="given-names" title="given-names">M</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gentleman</span><span class="given-names" tagx="given-names" title="given-names">R</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Morgan</span><span class="given-names" tagx="given-names" title="given-names">MT</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Carey</span><span class="given-names" tagx="given-names" title="given-names">VJ</span></span></span><span class="year" tagx="year" title="year">2013</span><span class="mixed-article-title" title="mixed-article-title">Software for computing and annotating genomic ranges</span><span class="source" tagx="source" title="source">PLOS Computational Biology</span><span class="volume" tagx="volume" title="volume">9</span><span class="elocation-id" tagx="elocation-id" title="elocation-id">e1003118</span><span class="pub-id"><a href="https://dx.doi.org/10.1371/journal.pcbi.1003118">10.1371/journal.pcbi.1003118</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23950696">23950696</a></span></span></li> <li tag="ref"><a name="bib54" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lecuit</span><span class="given-names" tagx="given-names" title="given-names">T</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Samanta</span><span class="given-names" tagx="given-names" title="given-names">R</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wieschaus</span><span class="given-names" tagx="given-names" title="given-names">E</span></span></span><span class="year" tagx="year" title="year">2002</span><span class="mixed-article-title" title="mixed-article-title">Slam encodes a developmental regulator of polarized membrane growth during cleavage of the Drosophila embryo</span><span class="source" tagx="source" title="source">Developmental Cell</span><span class="volume" tagx="volume" title="volume">2</span><span class="fpage" tagx="fpage" title="fpage">425</span><span class="lpage" tagx="lpage" title="lpage">436</span><span class="pub-id"><a href="https://dx.doi.org/10.1016/S1534-5807(02)00141-7">10.1016/S1534-5807(02)00141-7</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11970893">11970893</a></span></span></li> <li tag="ref"><a name="bib55" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lecuit</span><span class="given-names" tagx="given-names" title="given-names">T</span></span></span><span class="year" tagx="year" title="year">2004</span><span class="mixed-article-title" title="mixed-article-title">Junctions and vesicular trafficking during <i>Drosophila</i> cellularization</span><span class="source" tagx="source" title="source">Journal of Cell Science</span><span class="volume" tagx="volume" title="volume">117</span><span class="fpage" tagx="fpage" title="fpage">3427</span><span class="lpage" tagx="lpage" title="lpage">3433</span><span class="pub-id"><a href="https://dx.doi.org/10.1242/jcs.01312">10.1242/jcs.01312</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15252125">15252125</a></span></span></li> <li tag="ref"><a name="bib56" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lecuit</span><span class="given-names" tagx="given-names" title="given-names">T</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wieschaus</span><span class="given-names" tagx="given-names" title="given-names">E</span></span></span><span class="year" tagx="year" title="year">2000</span><span class="mixed-article-title" title="mixed-article-title">Polarized insertion of new membrane from a cytoplasmic reservoir during cleavage of the Drosophila embryo</span><span class="source" tagx="source" title="source">The Journal of Cell Biology</span><span class="volume" tagx="volume" title="volume">150</span><span class="fpage" tagx="fpage" title="fpage">849</span><span class="lpage" tagx="lpage" title="lpage">860</span><span class="pub-id"><a href="https://dx.doi.org/10.1083/jcb.150.4.849">10.1083/jcb.150.4.849</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/10953008">10953008</a></span></span></li> <li tag="ref"><a name="bib57" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Leys</span><span class="given-names" tagx="given-names" title="given-names">SP</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Nichols</span><span class="given-names" tagx="given-names" title="given-names">SA</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Adams</span><span class="given-names" tagx="given-names" title="given-names">ED</span></span></span><span class="year" tagx="year" title="year">2009</span><span class="mixed-article-title" title="mixed-article-title">Epithelia and integration in sponges</span><span class="source" tagx="source" title="source">Integrative and Comparative Biology</span><span class="volume" tagx="volume" title="volume">49</span><span class="fpage" tagx="fpage" title="fpage">167</span><span class="lpage" tagx="lpage" title="lpage">177</span><span class="pub-id"><a href="https://dx.doi.org/10.1093/icb/icp038">10.1093/icb/icp038</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21669855">21669855</a></span></span></li> <li tag="ref"><a name="bib58" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lomsadze</span><span class="given-names" tagx="given-names" title="given-names">A</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ter-Hovhannisyan</span><span class="given-names" tagx="given-names" title="given-names">V</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Chernoff</span><span class="given-names" tagx="given-names" title="given-names">YO</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Borodovsky</span><span class="given-names" tagx="given-names" title="given-names">M</span></span></span><span class="year" tagx="year" title="year">2005</span><span class="mixed-article-title" title="mixed-article-title">Gene identification in novel eukaryotic genomes by self-training algorithm</span><span class="source" tagx="source" title="source">Nucleic Acids Research</span><span class="volume" tagx="volume" title="volume">33</span><span class="fpage" tagx="fpage" title="fpage">6494</span><span class="lpage" tagx="lpage" title="lpage">6506</span><span class="pub-id"><a href="https://dx.doi.org/10.1093/nar/gki937">10.1093/nar/gki937</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16314312">16314312</a></span></span></li> <li tag="ref"><a name="bib59" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Marshall</span><span class="given-names" tagx="given-names" title="given-names">WL</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Berbee</span><span class="given-names" tagx="given-names" title="given-names">ML</span></span></span><span class="year" tagx="year" title="year">2011</span><span class="mixed-article-title" title="mixed-article-title">Facing unknowns: living cultures (Pirum gemmata gen. nov., sp. nov., and Abeoforma whisleri, gen. nov., sp. nov.) from invertebrate digestive tracts represent an undescribed clade within the unicellular opisthokont lineage ichthyosporea (Mesomycetozoea)</span><span class="source" tagx="source" title="source">Protist</span><span class="volume" tagx="volume" title="volume">162</span><span class="fpage" tagx="fpage" title="fpage">33</span><span class="lpage" tagx="lpage" title="lpage">57</span><span class="pub-id"><a href="https://dx.doi.org/10.1016/j.protis.2010.06.002">10.1016/j.protis.2010.06.002</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20708961">20708961</a></span></span></li> <li tag="ref"><a name="bib60" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Marshall</span><span class="given-names" tagx="given-names" title="given-names">WL</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Berbee</span><span class="given-names" tagx="given-names" title="given-names">ML</span></span></span><span class="year" tagx="year" title="year">2013</span><span class="mixed-article-title" title="mixed-article-title">Comparative morphology and genealogical delimitation of cryptic species of sympatric isolates of Sphaeroforma (Ichthyosporea, opisthokonta)</span><span class="source" tagx="source" title="source">Protist</span><span class="volume" tagx="volume" title="volume">164</span><span class="fpage" tagx="fpage" title="fpage">287</span><span class="lpage" tagx="lpage" title="lpage">311</span><span class="pub-id"><a href="https://dx.doi.org/10.1016/j.protis.2012.12.002">10.1016/j.protis.2012.12.002</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23352078">23352078</a></span></span></li> <li tag="ref"><a name="bib61" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Mazumdar</span><span class="given-names" tagx="given-names" title="given-names">A</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Mazumdar</span><span class="given-names" tagx="given-names" title="given-names">M</span></span></span><span class="year" tagx="year" title="year">2002</span><span class="mixed-article-title" title="mixed-article-title">How one becomes many: blastoderm cellularization in Drosophila Melanogaster</span><span class="source" tagx="source" title="source">BioEssays</span><span class="volume" tagx="volume" title="volume">24</span><span class="fpage" tagx="fpage" title="fpage">1012</span><span class="lpage" tagx="lpage" title="lpage">1022</span><span class="pub-id"><a href="https://dx.doi.org/10.1002/bies.10184">10.1002/bies.10184</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12386932">12386932</a></span></span></li> <li tag="ref"><a name="bib62" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">McLaren</span><span class="given-names" tagx="given-names" title="given-names">W</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gil</span><span class="given-names" tagx="given-names" title="given-names">L</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hunt</span><span class="given-names" tagx="given-names" title="given-names">SE</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Riat</span><span class="given-names" tagx="given-names" title="given-names">HS</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ritchie</span><span class="given-names" tagx="given-names" title="given-names">GR</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Thormann</span><span class="given-names" tagx="given-names" title="given-names">A</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Flicek</span><span class="given-names" tagx="given-names" title="given-names">P</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Cunningham</span><span class="given-names" tagx="given-names" title="given-names">F</span></span></span><span class="year" tagx="year" title="year">2016</span><span class="mixed-article-title" title="mixed-article-title">The ensembl variant effect predictor</span><span class="source" tagx="source" title="source">Genome Biology</span><span class="volume" tagx="volume" title="volume">17</span><span class="fpage" tagx="fpage" title="fpage">1</span><span class="lpage" tagx="lpage" title="lpage">14</span><span class="pub-id"><a href="https://dx.doi.org/10.1186/s13059-016-0974-4">10.1186/s13059-016-0974-4</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/26753840">26753840</a></span></span></li> <li tag="ref"><a name="bib63" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Mendoza</span><span class="given-names" tagx="given-names" title="given-names">L</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Taylor</span><span class="given-names" tagx="given-names" title="given-names">JW</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ajello</span><span class="given-names" tagx="given-names" title="given-names">L</span></span></span><span class="year" tagx="year" title="year">2002</span><span class="mixed-article-title" title="mixed-article-title">The class mesomycetozoea: a heterogeneous group of microorganisms at the Animal-Fungal boundary</span><span class="source" tagx="source" title="source">Annual Review of Microbiology</span><span class="volume" tagx="volume" title="volume">56</span><span class="fpage" tagx="fpage" title="fpage">315</span><span class="lpage" tagx="lpage" title="lpage">344</span><span class="pub-id"><a href="https://dx.doi.org/10.1146/annurev.micro.56.012302.160950">10.1146/annurev.micro.56.012302.160950</a></span></span></li> <li tag="ref"><a name="bib64" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Miller</span><span class="given-names" tagx="given-names" title="given-names">PW</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Clarke</span><span class="given-names" tagx="given-names" title="given-names">DN</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Weis</span><span class="given-names" tagx="given-names" title="given-names">WI</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lowe</span><span class="given-names" tagx="given-names" title="given-names">CJ</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Nelson</span><span class="given-names" tagx="given-names" title="given-names">WJ</span></span></span><span class="year" tagx="year" title="year">2013</span><span class="mixed-article-title" title="mixed-article-title">The evolutionary origin of epithelial cell-cell adhesion mechanisms</span><span class="source" tagx="source" title="source">Current Topics in Membranes</span><span class="volume" tagx="volume" title="volume">72</span><span class="fpage" tagx="fpage" title="fpage">267</span><span class="lpage" tagx="lpage" title="lpage">311</span><span class="pub-id"><a href="https://dx.doi.org/10.1016/B978-0-12-417027-8.00008-8">10.1016/B978-0-12-417027-8.00008-8</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24210433">24210433</a></span></span></li> <li tag="ref"><a name="bib65" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Murray</span><span class="given-names" tagx="given-names" title="given-names">PS</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zaidel-Bar</span><span class="given-names" tagx="given-names" title="given-names">R</span></span></span><span class="year" tagx="year" title="year">2014</span><span class="mixed-article-title" title="mixed-article-title">Pre-metazoan origins and evolution of the cadherin adhesome</span><span class="source" tagx="source" title="source">Biology Open</span><span class="volume" tagx="volume" title="volume">3</span><span class="fpage" tagx="fpage" title="fpage">1183</span><span class="lpage" tagx="lpage" title="lpage">1195</span><span class="pub-id"><a href="https://dx.doi.org/10.1242/bio.20149761">10.1242/bio.20149761</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25395670">25395670</a></span></span></li> <li tag="ref"><a name="bib66" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Nurk</span><span class="given-names" tagx="given-names" title="given-names">S</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bankevich</span><span class="given-names" tagx="given-names" title="given-names">A</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Antipov</span><span class="given-names" tagx="given-names" title="given-names">D</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gurevich</span><span class="given-names" tagx="given-names" title="given-names">AA</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Korobeynikov</span><span class="given-names" tagx="given-names" title="given-names">A</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lapidus</span><span class="given-names" tagx="given-names" title="given-names">A</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Prjibelski</span><span class="given-names" tagx="given-names" title="given-names">AD</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Pyshkin</span><span class="given-names" tagx="given-names" title="given-names">A</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Sirotkin</span><span class="given-names" tagx="given-names" title="given-names">A</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Sirotkin</span><span class="given-names" tagx="given-names" title="given-names">Y</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Stepanauskas</span><span class="given-names" tagx="given-names" title="given-names">R</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Clingenpeel</span><span class="given-names" tagx="given-names" title="given-names">SR</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Woyke</span><span class="given-names" tagx="given-names" title="given-names">T</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">McLean</span><span class="given-names" tagx="given-names" title="given-names">JS</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lasken</span><span class="given-names" tagx="given-names" title="given-names">R</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Tesler</span><span class="given-names" tagx="given-names" title="given-names">G</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Alekseyev</span><span class="given-names" tagx="given-names" title="given-names">MA</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Pevzner</span><span class="given-names" tagx="given-names" title="given-names">PA</span></span></span><span class="year" tagx="year" title="year">2013</span><span class="mixed-article-title" title="mixed-article-title">Assembling single-cell genomes and mini-metagenomes from chimeric MDA products</span><span class="source" tagx="source" title="source">Journal of Computational Biology</span><span class="volume" tagx="volume" title="volume">20</span><span class="fpage" tagx="fpage" title="fpage">714</span><span class="lpage" tagx="lpage" title="lpage">737</span><span class="pub-id"><a href="https://dx.doi.org/10.1089/cmb.2013.0084">10.1089/cmb.2013.0084</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24093227">24093227</a></span></span></li> <li tag="ref"><a name="bib67" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ondracka</span><span class="given-names" tagx="given-names" title="given-names">A</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Dudin</span><span class="given-names" tagx="given-names" title="given-names">O</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ruiz-Trillo</span><span class="given-names" tagx="given-names" title="given-names">I</span></span></span><span class="year" tagx="year" title="year">2018</span><span class="mixed-article-title" title="mixed-article-title">Decoupling of nuclear division cycles and cell size during the coenocytic growth of the ichthyosporean Sphaeroforma arctica</span><span class="source" tagx="source" title="source">Current Biology</span><span class="volume" tagx="volume" title="volume">28</span><span class="fpage" tagx="fpage" title="fpage">1964</span><span class="lpage" tagx="lpage" title="lpage">1969</span><span class="pub-id"><a href="https://dx.doi.org/10.1016/j.cub.2018.04.074">10.1016/j.cub.2018.04.074</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/29887314">29887314</a></span></span></li> <li tag="ref"><a name="bib68" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ondracka</span><span class="given-names" tagx="given-names" title="given-names">A</span></span></span><span class="year" tagx="year" title="year">2019</span><div tag="data-title">
sphaeroforma_transcriptome</div><span class="source" tagx="source" title="source">Github</span><span class="version" tagx="version" title="version">a8771e6</span><a href="https://github.com/andrejondracka/sphaeroforma_transcriptome">https://github.com/andrejondracka/sphaeroforma_transcriptome</a></span></li> <li tag="ref"><a name="bib69" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Parfrey</span><span class="given-names" tagx="given-names" title="given-names">LW</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lahr</span><span class="given-names" tagx="given-names" title="given-names">DJG</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Knoll</span><span class="given-names" tagx="given-names" title="given-names">AH</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Katz</span><span class="given-names" tagx="given-names" title="given-names">LA</span></span></span><span class="year" tagx="year" title="year">2011</span><span class="mixed-article-title" title="mixed-article-title">Estimating the timing of early eukaryotic diversification with multigene molecular clocks</span><span class="source" tagx="source" title="source">PNAS</span><span class="volume" tagx="volume" title="volume">108</span><span class="fpage" tagx="fpage" title="fpage">13624</span><span class="lpage" tagx="lpage" title="lpage">13629</span><span class="pub-id"><a href="https://dx.doi.org/10.1073/pnas.1110633108">10.1073/pnas.1110633108</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21810989">21810989</a></span></span></li> <li tag="ref"><a name="bib70" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Patro</span><span class="given-names" tagx="given-names" title="given-names">R</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Duggal</span><span class="given-names" tagx="given-names" title="given-names">G</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Love</span><span class="given-names" tagx="given-names" title="given-names">MI</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Irizarry</span><span class="given-names" tagx="given-names" title="given-names">RA</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kingsford</span><span class="given-names" tagx="given-names" title="given-names">C</span></span></span><span class="year" tagx="year" title="year">2017</span><span class="mixed-article-title" title="mixed-article-title">Salmon provides fast and bias-aware quantification of transcript expression</span><span class="source" tagx="source" title="source">Nature Methods</span><span class="volume" tagx="volume" title="volume">14</span><span class="fpage" tagx="fpage" title="fpage">417</span><span class="lpage" tagx="lpage" title="lpage">419</span><span class="pub-id"><a href="https://dx.doi.org/10.1038/nmeth.4197">10.1038/nmeth.4197</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/28263959">28263959</a></span></span></li> <li tag="ref"><a name="bib71" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Pelissier</span><span class="given-names" tagx="given-names" title="given-names">A</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Chauvin</span><span class="given-names" tagx="given-names" title="given-names">JP</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lecuit</span><span class="given-names" tagx="given-names" title="given-names">T</span></span></span><span class="year" tagx="year" title="year">2003</span><span class="mixed-article-title" title="mixed-article-title">Trafficking through Rab11 endosomes is required for cellularization during Drosophila embryogenesis</span><span class="source" tagx="source" title="source">Current Biology</span><span class="volume" tagx="volume" title="volume">13</span><span class="fpage" tagx="fpage" title="fpage">1848</span><span class="lpage" tagx="lpage" title="lpage">1857</span><span class="pub-id"><a href="https://dx.doi.org/10.1016/j.cub.2003.10.023">10.1016/j.cub.2003.10.023</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/14588240">14588240</a></span></span></li> <li tag="ref"><a name="bib72" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Pignocchi</span><span class="given-names" tagx="given-names" title="given-names">C</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Minns</span><span class="given-names" tagx="given-names" title="given-names">GE</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Nesi</span><span class="given-names" tagx="given-names" title="given-names">N</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Koumproglou</span><span class="given-names" tagx="given-names" title="given-names">R</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kitsios</span><span class="given-names" tagx="given-names" title="given-names">G</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Benning</span><span class="given-names" tagx="given-names" title="given-names">C</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lloyd</span><span class="given-names" tagx="given-names" title="given-names">CW</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Doonan</span><span class="given-names" tagx="given-names" title="given-names">JH</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hills</span><span class="given-names" tagx="given-names" title="given-names">MJ</span></span></span><span class="year" tagx="year" title="year">2009</span><span class="mixed-article-title" title="mixed-article-title">ENDOSPERM DEFECTIVE1 is a novel Microtubule-Associated protein essential for seed development in Arabidopsis</span><span class="source" tagx="source" title="source">The Plant Cell</span><span class="volume" tagx="volume" title="volume">21</span><span class="fpage" tagx="fpage" title="fpage">90</span><span class="lpage" tagx="lpage" title="lpage">105</span><span class="pub-id"><a href="https://dx.doi.org/10.1105/tpc.108.061812">10.1105/tpc.108.061812</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19151224">19151224</a></span></span></li> <li tag="ref"><a name="bib73" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Postner</span><span class="given-names" tagx="given-names" title="given-names">MA</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wieschaus</span><span class="given-names" tagx="given-names" title="given-names">EF</span></span></span><span class="year" tagx="year" title="year">1994</span><span class="mixed-article-title" title="mixed-article-title">The nullo protein is a component of the actin-myosin network that mediates cellularization in Drosophila Melanogaster embryos</span><span class="source" tagx="source" title="source">Journal of Cell Science</span><span class="volume" tagx="volume" title="volume">107 ( Pt 7</span><span class="fpage" tagx="fpage" title="fpage">1863</span><span class="lpage" tagx="lpage" title="lpage">1873</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/7983153">7983153</a></span></span></li> <li tag="ref"><a name="bib74" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Priti</span><span class="given-names" tagx="given-names" title="given-names">A</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ong</span><span class="given-names" tagx="given-names" title="given-names">HT</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Toyama</span><span class="given-names" tagx="given-names" title="given-names">Y</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Padmanabhan</span><span class="given-names" tagx="given-names" title="given-names">A</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Dasgupta</span><span class="given-names" tagx="given-names" title="given-names">S</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Krajnc</span><span class="given-names" tagx="given-names" title="given-names">M</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zaidel-Bar</span><span class="given-names" tagx="given-names" title="given-names">R</span></span></span><span class="year" tagx="year" title="year">2018</span><span class="mixed-article-title" title="mixed-article-title">Syncytial germline architecture is actively maintained by contraction of an internal actomyosin corset</span><span class="source" tagx="source" title="source">Nature Communications</span><span class="volume" tagx="volume" title="volume">9</span><span class="elocation-id" tagx="elocation-id" title="elocation-id">4694</span><span class="pub-id"><a href="https://dx.doi.org/10.1038/s41467-018-07149-2">10.1038/s41467-018-07149-2</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/30410005">30410005</a></span></span></li> <li tag="ref"><a name="bib75" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Quinlan</span><span class="given-names" tagx="given-names" title="given-names">AR</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hall</span><span class="given-names" tagx="given-names" title="given-names">IM</span></span></span><span class="year" tagx="year" title="year">2010</span><span class="mixed-article-title" title="mixed-article-title">BEDTools: a flexible suite of utilities for comparing genomic features</span><span class="source" tagx="source" title="source">Bioinformatics</span><span class="volume" tagx="volume" title="volume">26</span><span class="fpage" tagx="fpage" title="fpage">841</span><span class="lpage" tagx="lpage" title="lpage">842</span><span class="pub-id"><a href="https://dx.doi.org/10.1093/bioinformatics/btq033">10.1093/bioinformatics/btq033</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20110278">20110278</a></span></span></li> <li tag="ref"><a name="bib76" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Quint</span><span class="given-names" tagx="given-names" title="given-names">M</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Drost</span><span class="given-names" tagx="given-names" title="given-names">HG</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gabel</span><span class="given-names" tagx="given-names" title="given-names">A</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ullrich</span><span class="given-names" tagx="given-names" title="given-names">KK</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bönn</span><span class="given-names" tagx="given-names" title="given-names">M</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Grosse</span><span class="given-names" tagx="given-names" title="given-names">I</span></span></span><span class="year" tagx="year" title="year">2012</span><span class="mixed-article-title" title="mixed-article-title">A transcriptomic hourglass in plant embryogenesis</span><span class="source" tagx="source" title="source">Nature</span><span class="volume" tagx="volume" title="volume">490</span><span class="fpage" tagx="fpage" title="fpage">98</span><span class="lpage" tagx="lpage" title="lpage">101</span><span class="pub-id"><a href="https://dx.doi.org/10.1038/nature11394">10.1038/nature11394</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22951968">22951968</a></span></span></li> <li tag="ref"><a name="bib77" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Richter</span><span class="given-names" tagx="given-names" title="given-names">DJ</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Fozouni</span><span class="given-names" tagx="given-names" title="given-names">P</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Eisen</span><span class="given-names" tagx="given-names" title="given-names">MB</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">King</span><span class="given-names" tagx="given-names" title="given-names">N</span></span></span><span class="year" tagx="year" title="year">2018</span><span class="mixed-article-title" title="mixed-article-title">Gene family innovation, conservation and loss on the animal stem lineage</span><span class="source" tagx="source" title="source">eLife</span><span class="volume" tagx="volume" title="volume">7</span><span class="elocation-id" tagx="elocation-id" title="elocation-id">e34426</span><span class="pub-id"><a href="https://dx.doi.org/10.7554/eLife.34226">10.7554/eLife.34226</a></span></span></li> <li tag="ref"><a name="bib78" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Rodriguez-Boulan</span><span class="given-names" tagx="given-names" title="given-names">E</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Macara</span><span class="given-names" tagx="given-names" title="given-names">IG</span></span></span><span class="year" tagx="year" title="year">2014</span><span class="mixed-article-title" title="mixed-article-title">Organization and execution of the epithelial polarity programme</span><span class="source" tagx="source" title="source">Nature Reviews Molecular Cell Biology</span><span class="volume" tagx="volume" title="volume">15</span><span class="fpage" tagx="fpage" title="fpage">225</span><span class="lpage" tagx="lpage" title="lpage">242</span><span class="pub-id"><a href="https://dx.doi.org/10.1038/nrm3775">10.1038/nrm3775</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24651541">24651541</a></span></span></li> <li tag="ref"><a name="bib79" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Royou</span><span class="given-names" tagx="given-names" title="given-names">A</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Sullivan</span><span class="given-names" tagx="given-names" title="given-names">W</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Karess</span><span class="given-names" tagx="given-names" title="given-names">R</span></span></span><span class="year" tagx="year" title="year">2002</span><span class="mixed-article-title" title="mixed-article-title">Cortical recruitment of nonmuscle myosin II in early syncytial <i>Drosophila</i> embryos</span><span class="source" tagx="source" title="source">The Journal of Cell Biology</span><span class="volume" tagx="volume" title="volume">158</span><span class="fpage" tagx="fpage" title="fpage">127</span><span class="lpage" tagx="lpage" title="lpage">137</span><span class="pub-id"><a href="https://dx.doi.org/10.1083/jcb.200203148">10.1083/jcb.200203148</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12105185">12105185</a></span></span></li> <li tag="ref"><a name="bib80" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Schejter</span><span class="given-names" tagx="given-names" title="given-names">ED</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wieschaus</span><span class="given-names" tagx="given-names" title="given-names">E</span></span></span><span class="year" tagx="year" title="year">1993</span><span class="mixed-article-title" title="mixed-article-title"><i>Bottleneck</i> acts as a regulator of the microfilament network governing cellularization of the Drosophila embryo</span><span class="source" tagx="source" title="source">Cell</span><span class="volume" tagx="volume" title="volume">75</span><span class="fpage" tagx="fpage" title="fpage">373</span><span class="lpage" tagx="lpage" title="lpage">385</span><span class="pub-id"><a href="https://dx.doi.org/10.1016/0092-8674(93)80078-S">10.1016/0092-8674(93)80078-S</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/8402919">8402919</a></span></span></li> <li tag="ref"><a name="bib81" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Schneider</span><span class="given-names" tagx="given-names" title="given-names">CA</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Rasband</span><span class="given-names" tagx="given-names" title="given-names">WS</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Eliceiri</span><span class="given-names" tagx="given-names" title="given-names">KW</span></span></span><span class="year" tagx="year" title="year">2012</span><span class="mixed-article-title" title="mixed-article-title">NIH image to ImageJ: 25 years of image analysis</span><span class="source" tagx="source" title="source">Nature Methods</span><span class="volume" tagx="volume" title="volume">9</span><span class="fpage" tagx="fpage" title="fpage">671</span><span class="lpage" tagx="lpage" title="lpage">675</span><span class="pub-id"><a href="https://dx.doi.org/10.1038/nmeth.2089">10.1038/nmeth.2089</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22930834">22930834</a></span></span></li> <li tag="ref"><a name="bib82" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Schweisguth</span><span class="given-names" tagx="given-names" title="given-names">F</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lepesant</span><span class="given-names" tagx="given-names" title="given-names">JA</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Vincent</span><span class="given-names" tagx="given-names" title="given-names">A</span></span></span><span class="year" tagx="year" title="year">1990</span><span class="mixed-article-title" title="mixed-article-title">The serendipity alpha gene encodes a membrane-associated protein required for the cellularization of the <i>Drosophila</i> embryo</span><span class="source" tagx="source" title="source">Genes &amp;amp; Development</span><span class="volume" tagx="volume" title="volume">4</span><span class="fpage" tagx="fpage" title="fpage">922</span><span class="lpage" tagx="lpage" title="lpage">931</span><span class="pub-id"><a href="https://dx.doi.org/10.1101/gad.4.6.922">10.1101/gad.4.6.922</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/2166703">2166703</a></span></span></li> <li tag="ref"><a name="bib83" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Sebé-Pedrós</span><span class="given-names" tagx="given-names" title="given-names">A</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Irimia</span><span class="given-names" tagx="given-names" title="given-names">M</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Del Campo</span><span class="given-names" tagx="given-names" title="given-names">J</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Parra-Acero</span><span class="given-names" tagx="given-names" title="given-names">H</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Russ</span><span class="given-names" tagx="given-names" title="given-names">C</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Nusbaum</span><span class="given-names" tagx="given-names" title="given-names">C</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Blencowe</span><span class="given-names" tagx="given-names" title="given-names">BJ</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ruiz-Trillo</span><span class="given-names" tagx="given-names" title="given-names">I</span></span></span><span class="year" tagx="year" title="year">2013</span><span class="mixed-article-title" title="mixed-article-title">Regulated aggregative multicellularity in a close unicellular relative of metazoa</span><span class="source" tagx="source" title="source">eLife</span><span class="volume" tagx="volume" title="volume">2</span><span class="elocation-id" tagx="elocation-id" title="elocation-id">e01287</span><span class="pub-id"><a href="https://dx.doi.org/10.7554/eLife.01287">10.7554/eLife.01287</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24368732">24368732</a></span></span></li> <li tag="ref"><a name="bib84" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Sokac</span><span class="given-names" tagx="given-names" title="given-names">AM</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wieschaus</span><span class="given-names" tagx="given-names" title="given-names">E</span></span></span><span class="year" tagx="year" title="year">2008</span><span class="mixed-article-title" title="mixed-article-title">Zygotically controlled F-actin establishes cortical compartments to stabilize furrows during Drosophila cellularization</span><span class="source" tagx="source" title="source">Journal of Cell Science</span><span class="volume" tagx="volume" title="volume">121</span><span class="fpage" tagx="fpage" title="fpage">1815</span><span class="lpage" tagx="lpage" title="lpage">1824</span><span class="pub-id"><a href="https://dx.doi.org/10.1242/jcs.025171">10.1242/jcs.025171</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18460582">18460582</a></span></span></li> <li tag="ref"><a name="bib85" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Sørensen</span><span class="given-names" tagx="given-names" title="given-names">MB</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Mayer</span><span class="given-names" tagx="given-names" title="given-names">U</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lukowitz</span><span class="given-names" tagx="given-names" title="given-names">W</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Robert</span><span class="given-names" tagx="given-names" title="given-names">H</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Chambrier</span><span class="given-names" tagx="given-names" title="given-names">P</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Jürgens</span><span class="given-names" tagx="given-names" title="given-names">G</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Somerville</span><span class="given-names" tagx="given-names" title="given-names">C</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lepiniec</span><span class="given-names" tagx="given-names" title="given-names">L</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Berger</span><span class="given-names" tagx="given-names" title="given-names">F</span></span></span><span class="year" tagx="year" title="year">2002</span><span class="mixed-article-title" title="mixed-article-title">Cellularisation in the endosperm of Arabidopsis thaliana is coupled to mitosis and shares multiple components with cytokinesis</span><span class="source" tagx="source" title="source">Development</span><span class="volume" tagx="volume" title="volume">129</span><span class="fpage" tagx="fpage" title="fpage">5567</span><span class="lpage" tagx="lpage" title="lpage">5576</span><span class="pub-id"><a href="https://dx.doi.org/10.1242/dev.00152">10.1242/dev.00152</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12421698">12421698</a></span></span></li> <li tag="ref"><a name="bib86" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Stevenson</span><span class="given-names" tagx="given-names" title="given-names">V</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hudson</span><span class="given-names" tagx="given-names" title="given-names">A</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Cooley</span><span class="given-names" tagx="given-names" title="given-names">L</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Theurkauf</span><span class="given-names" tagx="given-names" title="given-names">WE</span></span></span><span class="year" tagx="year" title="year">2002</span><span class="mixed-article-title" title="mixed-article-title">Arp2/3-Dependent psuedocleavage furrow assembly in syncytial Drosophila embryos</span><span class="source" tagx="source" title="source">Current Biology</span><span class="volume" tagx="volume" title="volume">12</span><span class="fpage" tagx="fpage" title="fpage">705</span><span class="lpage" tagx="lpage" title="lpage">711</span><span class="pub-id"><a href="https://dx.doi.org/10.1016/S0960-9822(02)00807-2">10.1016/S0960-9822(02)00807-2</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12007413">12007413</a></span></span></li> <li tag="ref"><a name="bib87" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Suga</span><span class="given-names" tagx="given-names" title="given-names">H</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ruiz-Trillo</span><span class="given-names" tagx="given-names" title="given-names">I</span></span></span><span class="year" tagx="year" title="year">2013</span><span class="mixed-article-title" title="mixed-article-title">Development of ichthyosporeans sheds light on the origin of metazoan multicellularity</span><span class="source" tagx="source" title="source">Developmental Biology</span><span class="volume" tagx="volume" title="volume">377</span><span class="fpage" tagx="fpage" title="fpage">284</span><span class="lpage" tagx="lpage" title="lpage">292</span><span class="pub-id"><a href="https://dx.doi.org/10.1016/j.ydbio.2013.01.009">10.1016/j.ydbio.2013.01.009</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23333946">23333946</a></span></span></li> <li tag="ref"><a name="bib88" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Trincado</span><span class="given-names" tagx="given-names" title="given-names">JL</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Entizne</span><span class="given-names" tagx="given-names" title="given-names">JC</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hysenaj</span><span class="given-names" tagx="given-names" title="given-names">G</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Singh</span><span class="given-names" tagx="given-names" title="given-names">B</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Skalic</span><span class="given-names" tagx="given-names" title="given-names">M</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Elliott</span><span class="given-names" tagx="given-names" title="given-names">DJ</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Eyras</span><span class="given-names" tagx="given-names" title="given-names">E</span></span></span><span class="year" tagx="year" title="year">2018</span><span class="mixed-article-title" title="mixed-article-title">SUPPA2: fast, accurate, and uncertainty-aware differential splicing analysis across multiple conditions</span><span class="source" tagx="source" title="source">Genome Biology</span><span class="volume" tagx="volume" title="volume">19</span><span class="fpage" tagx="fpage" title="fpage">1</span><span class="lpage" tagx="lpage" title="lpage">11</span><span class="pub-id"><a href="https://dx.doi.org/10.1186/s13059-018-1417-1">10.1186/s13059-018-1417-1</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/29301551">29301551</a></span></span></li> <li tag="ref"><a name="bib89" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Walker</span><span class="given-names" tagx="given-names" title="given-names">BJ</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Abeel</span><span class="given-names" tagx="given-names" title="given-names">T</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Shea</span><span class="given-names" tagx="given-names" title="given-names">T</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Priest</span><span class="given-names" tagx="given-names" title="given-names">M</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Abouelliel</span><span class="given-names" tagx="given-names" title="given-names">A</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Sakthikumar</span><span class="given-names" tagx="given-names" title="given-names">S</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Cuomo</span><span class="given-names" tagx="given-names" title="given-names">CA</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zeng</span><span class="given-names" tagx="given-names" title="given-names">Q</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wortman</span><span class="given-names" tagx="given-names" title="given-names">J</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Young</span><span class="given-names" tagx="given-names" title="given-names">SK</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Earl</span><span class="given-names" tagx="given-names" title="given-names">AM</span></span></span><span class="year" tagx="year" title="year">2014</span><span class="mixed-article-title" title="mixed-article-title">Pilon: an integrated tool for comprehensive microbial variant detection and genome assembly improvement</span><span class="source" tagx="source" title="source">PLOS ONE</span><span class="volume" tagx="volume" title="volume">9</span><span class="elocation-id" tagx="elocation-id" title="elocation-id">e112963</span><span class="pub-id"><a href="https://dx.doi.org/10.1371/journal.pone.0112963">10.1371/journal.pone.0112963</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25409509">25409509</a></span></span></li> <li tag="ref"><a name="bib90" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wang</span><span class="given-names" tagx="given-names" title="given-names">F</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Dumstrei</span><span class="given-names" tagx="given-names" title="given-names">K</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Haag</span><span class="given-names" tagx="given-names" title="given-names">T</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hartenstein</span><span class="given-names" tagx="given-names" title="given-names">V</span></span></span><span class="year" tagx="year" title="year">2004</span><span class="mixed-article-title" title="mixed-article-title">The role of DE-cadherin during Cellularization, germ layer formation and early neurogenesis in the Drosophila embryo</span><span class="source" tagx="source" title="source">Developmental Biology</span><span class="volume" tagx="volume" title="volume">270</span><span class="fpage" tagx="fpage" title="fpage">350</span><span class="lpage" tagx="lpage" title="lpage">363</span><span class="pub-id"><a href="https://dx.doi.org/10.1016/j.ydbio.2004.03.002">10.1016/j.ydbio.2004.03.002</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15183719">15183719</a></span></span></li> <li tag="ref"><a name="bib91" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Whisler</span><span class="given-names" tagx="given-names" title="given-names">HC</span></span></span><span class="year" tagx="year" title="year">1968</span><span class="mixed-article-title" title="mixed-article-title">Developmental control of Amoebidium parasiticum</span><span class="source" tagx="source" title="source">Developmental Biology</span><span class="volume" tagx="volume" title="volume">17</span><span class="fpage" tagx="fpage" title="fpage">562</span><span class="lpage" tagx="lpage" title="lpage">570</span><span class="pub-id"><a href="https://dx.doi.org/10.1016/0012-1606(68)90005-5">10.1016/0012-1606(68)90005-5</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/5661080">5661080</a></span></span></li> <li tag="ref"><a name="bib92" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wu</span><span class="given-names" tagx="given-names" title="given-names">TD</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Watanabe</span><span class="given-names" tagx="given-names" title="given-names">CK</span></span></span><span class="year" tagx="year" title="year">2005</span><span class="mixed-article-title" title="mixed-article-title">GMAP: a genomic mapping and alignment program for mRNA and EST sequences</span><span class="source" tagx="source" title="source">Bioinformatics</span><span class="volume" tagx="volume" title="volume">21</span><span class="fpage" tagx="fpage" title="fpage">1859</span><span class="lpage" tagx="lpage" title="lpage">1875</span><span class="pub-id"><a href="https://dx.doi.org/10.1093/bioinformatics/bti310">10.1093/bioinformatics/bti310</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15728110">15728110</a></span></span></li> <li tag="ref"><a name="bib93" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Xue</span><span class="given-names" tagx="given-names" title="given-names">W</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Li</span><span class="given-names" tagx="given-names" title="given-names">JT</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zhu</span><span class="given-names" tagx="given-names" title="given-names">YP</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hou</span><span class="given-names" tagx="given-names" title="given-names">GY</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kong</span><span class="given-names" tagx="given-names" title="given-names">XF</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kuang</span><span class="given-names" tagx="given-names" title="given-names">YY</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Sun</span><span class="given-names" tagx="given-names" title="given-names">XW</span></span></span><span class="year" tagx="year" title="year">2013</span><span class="mixed-article-title" title="mixed-article-title">L_RNA_scaffolder: scaffolding genomes with transcripts</span><span class="source" tagx="source" title="source">BMC Genomics</span><span class="volume" tagx="volume" title="volume">14</span><span class="elocation-id" tagx="elocation-id" title="elocation-id">604</span><span class="pub-id"><a href="https://dx.doi.org/10.1186/1471-2164-14-604">10.1186/1471-2164-14-604</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24010822">24010822</a></span></span></li> </ul> </div> </div>  </body></html>