<html xmlns="http://www.w3.org/1999/xhtml"> <head> <style type="text/css">
      	     		 
      	    a                          {background : #ffffff; }
      	    article                    {border-style : dotted; border-width : 2px; }
		 	
		 	div                        {background : #ffffcc;}
		 	div.abstract-title         {font-weight : bold ; font-size : 16pt;}
		 	div.ack                    {border-style : solid ; border-color : red; margin : 2em; }
		 	div.article-meta           {border-style : solid; border-width : 1 pt; margin 1em;}
		 	div.boxed-text             {margin : 5em; border-style : solid;}
		 	div.contrib-group          {margin : 1em; }
		 	div.fig                    {border-style : solid; border-width : 2px; margin : 2em; }
		 	div.funding                {font-weight : bold ; font-size : 16pt;}
		 	div.given-names            {font-style : italic;}
		 	div.intro                  {border-style : inset; margin : 5px;}
		 	div.introduction           {border-style : inset; margin : 5px;}
		 	div.journal-meta           {border-style : solid; border-width : 1 pt; margin 1em;}
		 	div.journal-title-group    {background : #ffeeee;}
		 	div.article-title          {font-weight : bold ; font-size : 18pt;}
		 	div.kwd                    {font-style : italic}
		 	div.meta-name              {font-weight : bold ; font-size : 16pt;}
		 	div.name                   {font-weight : bold;}
		 	div.alt-title              {font-style : italic; font-size : 12px;}
		 	
		 	div.sec                    {border : 2px; margin 5px; padding 2px;}
		 	div.title                  {font-family : courier; font-weight : bold;
		 	                            font-size : 16pt; margin : 5px;}
		 	
		 	div.materials_methods      {border-style : double; margin : 5px; }
		 	div.methods                {border-style : double; margin : 5px; }
		 	
		 	div.results                {border-style : solid; margin : 5px;}
		 	div.background             {border-style : dotted; margin : 5px;}
		 	
		 	div.discussion             {border-style : groove; margin : 5px;}
		 	div.conclusion             {border-style : ridge; margin : 5px;}
		 	div.conclusions             {border-style : ridge; margin : 5px;}
		 	div.supplementary-material {border-style : inset; margin : 5px;}
		 	div.abbreviations          {border-style : double; border-color : red; margin : 5px;}
		 	div.competinginterests     {border-style : double; border-color : blue; margin : 5px;}
		 	div.acknowledgements       {border-style : double; border-color : green; margin : 5px;}
		 	div.authors_contributions   {border-style : double; border-color : purple; margin : 5px;}
		 	
		 	div.publisher              {border-style : outset; margin : 5px;}
		 	div.fn-type-conflict       {background : #f88; }
		 	div.fn-type-con            {background : #ddf; }
		 	div.fn-type-other          {background : #ddd; }
		 	
		 	div.unknown                {background : #ffd;
									 	  border-style : solid;
									 	  border-width : 1px;
									 	  padding : 2 px;}
		 	  
      	    table                      {background : #ffffdd;}
		 	tr                         {background : #ddddff; padding : 1px;}
		 	
		 	span                       {background : #ffcccc;}
		 	
		 	span.citation-author       {font-family : helvetica; background : #ffeeee;}
		 	span.collab                {background : #ddffff; }
		 	span.comment               {font-family : courier; font-size : 6px; background : #ffaaff;}
		 	span.contrib               {background : #ffffff;}
		 	span.corresp               {background : #ddffdd; }
		 	span.doi                   {background : #ffffff;}
		 	span.email                 {font-family : courier; }
		 	span.etal                  {font-style : italic;}
		 	span.fpage                 {font-family : courier;}
		 	span.given-names           {background : #ffffff;}
		 	span.iso-abbrev            {background : #ffffff;}
		 	span.issn-epub             {background : #ffffff;}
		 	span.issn-ppub             {background : #ffffff;}
		 	span.journal-title         {background : #ffffff;}
		 	span.lpage                 {font-family : courier;}
		 	span.mixed-article-title   {font-style : italic ;}
		 	span.nlm-ta                {background : #ffffff;}
		 	span.pmc                   {background : #ffffff;}
		 	span.pmcid                 {background : #ffffff;}
		 	span.pmid                  {background : #ffffff;}
		 	span.publisher             {background : #ffffff;}
		 	span.publisher-id          {background : #ffffff;}
		 	span.publisher-name        {background : #ffffff;}
		 	span.source                {background : #ffffff;}
		 	span.subject               {background : #ffffff;}
		 	span.surname               {background : #ffffff;}
		 	span.volume                {font-family : courier; font-weight : bold;}
		 	span.year                  {font-family : courier ; font-style : italic;}
			</style> </head> <body> <div class="front" title="front"> <div class="journal-meta" tagx="journal-meta" title="journal-meta"><span class="nlm-ta" title="nlm-ta">BMC Evol Biol</span><span class="iso-abbrev" title="iso-abbrev">BMC Evol. Biol</span><div class="journal-title-group" tagx="journal-title-group" title="journal-title-group"><span class="journal-title" tagx="journal-title" title="journal-title">BMC Evolutionary Biology</span></div><span class="issn-epub" tagx="issn" title="issn-epub">1471-2148</span><div class="publisher" tagx="publisher" title="publisher"><span class="publisher-name" tagx="publisher-name" title="publisher-name">BioMed Central</span><span class="publisher-loc" tagx="publisher-loc" title="publisher-loc">London</span></div> </div> <div class="article-meta" tagx="article-meta" title="article-meta"><span class="publisher-id" title="publisher-id">431</span><span class="doi" title="doi"> doi: <a href="https://dx.doi.org/10.1186/s12862-015-0431-0">10.1186/s12862-015-0431-0</a></span><div class="article-categories" title="article-categories">
: <span class="subject" title="subject">Research Article</span></div> <div class="title-group" tagx="title-group" title="title-group"> <div class="article-title" title="article-title">
Evolutionary analysis of selective constraints identifies ameloblastin (AMBN) as a potential candidate for amelogenesis imperfecta</div> </div> <div class="contrib-group" title="contrib-group"><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Delsuc</span><span class="given-names" tagx="given-names" title="given-names">Frédéric</span></span><a href="#Aff1" /></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gasse</span><span class="given-names" tagx="given-names" title="given-names">Barbara</span></span><a href="#Aff2" /></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Sire</span><span class="given-names" tagx="given-names" title="given-names">Jean-Yves</span></span><span class="address" title="address"><span class="email" tagx="email" title="email">jean-yves.sire@upmc.fr</span></span><a href="#Aff2" /></span><span class="citation_author_institution" id="Aff1">[], </span><span class="citation_author_institution" id="Aff2">[], </span></div><span class="pub-date-epub" title="pub-date-epub">epub: <span>2015-7-7</span></span><span class="pub-date-pmc-release" title="pub-date-pmc-release">pmc-release: <span>2015-7-7</span></span><span class="pub-date-collection" title="pub-date-collection">collection: <span>2015</span></span><span class="volume" tagx="volume" title="volume">15</span><span class="elocation-id" tagx="elocation-id" title="elocation-id">148</span><span class="history" title="history"><span class="received" title="received">received: 2015-1-30</span><span class="accepted" title="accepted">accepted: 2015-7-21</span></span><div class="permissions"><span class="copyright" title="copyright">(C) , </span><span class="license" title="license"><span class="license-p" title="license-p">This is an Open Access article distributed under the terms of the Creative Commons Attribution License (<a href="http://creativecommons.org/licenses/by/4.0">http://creativecommons.org/licenses/by/4.0</a>), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly credited. The Creative Commons Public Domain Dedication waiver (<a href="http://creativecommons.org/publicdomain/zero/1.0/">http://creativecommons.org/publicdomain/zero/1.0/</a>) applies to the data made available in this article, unless otherwise stated.</span></span></div> <div class="abstract" title="abstract"> <div class="abstract-title" title="abstract-title">
Abstract</div> <div class="background" title="sec"> <div class="title" tagx="title" title="title">
Background</div> <p>Ameloblastin (AMBN) is a phosphorylated, proline/glutamine-rich protein secreted during enamel formation. Previous studies have revealed that this enamel matrix protein was present early in vertebrate evolution and certainly plays important roles during enamel formation although its precise functions remain unclear. We performed evolutionary analyses of AMBN in order to (i) identify residues and motifs important for the protein function, (ii) predict mutations responsible for genetic diseases, and (iii) understand its molecular evolution in mammals.</p> </div> <div class="results" title="sec"> <div class="title" tagx="title" title="title">
Results</div> <p>In silico searches retrieved 56 complete sequences in public databases that were aligned and analyzed computationally. We showed that <i>AMBN</i> is globally evolving under moderate purifying selection in mammals and contains a strong phylogenetic signal. In addition, our analyses revealed codons evolving under significant positive selection. Evidence for positive selection acting on AMBN was observed in catarrhine primates and the aye-aye. We also found that (i) an additional translation initiation site was recruited in the ancestral placental <i>AMBN</i>, (ii) a short exon was duplicated several times in various species including catarrhine primates, and (iii) several polyadenylation sites are present.</p> </div> <div class="conclusions" title="sec"> <div class="title" tagx="title" title="title">
Conclusions</div> <p>AMBN possesses many positions, which have been subjected to strong selective pressure for 200 million years. These positions correspond to several cleavage sites and hydroxylated, O-glycosylated, and phosphorylated residues. We predict that these conserved positions would be potentially responsible for enamel disorder if substituted. Some motifs that were previously identified as potentially important functionally were confirmed, and we found two, highly conserved, new motifs, the function of which should be tested in the near future. This study illustrates the power of evolutionary analyses for characterizing the functional constraints acting on proteins with yet uncharacterized structure.</p> </div> <div class="electronicsupplementarymaterial" title="sec"> <div class="title" tagx="title" title="title">
Electronic supplementary material</div> <p>The online version of this article (doi:10.1186/s12862-015-0431-0) contains supplementary material, which is available to authorized users.</p> </div> </div> <div class="kwd-group"> <div class="title" tagx="title" title="title">
Keywords</div> <div class="kwd" title="kwd">
Ameloblastin</div> <div class="kwd" title="kwd">
Evolution</div> <div class="kwd" title="kwd">
Enamel</div> <div class="kwd" title="kwd">
Mammals</div> <div class="kwd" title="kwd">
Purifying selection</div> <div class="kwd" title="kwd">
Positive selection</div> <div class="kwd" title="kwd">
Phylomedicine</div> </div> <div class="custom-meta-group" title="custom-meta-group"> <meta name="issue-copyright-statement" value="© The Author(s) 2015" /> </div> </div> </div> <div class="body" title="body"> <div class="introduction" title="introduction"> <div class="title" tagx="title" title="title">
Background</div> <p>Ameloblastin (AMBN), amelogenin (AMEL), enamelin (ENAM), amelotin (AMTN), odontogenic, ameloblast associated (ODAM) and SCPP-PQ1, are ameloblast-secreted proteins involved in various steps of the organization and mineralization of the enamel matrix of mammalian teeth. They belong to the large secretory calcium-binding phosphoprotein (SCPP) family (23 members known in humans), which appeared more than 450 million years ago (Ma) then diversified through gene duplication during vertebrate evolution [<a href="#CR1">1</a>–<a href="#CR4">4</a>]. In mammals, non-AMEL proteins account only for 10–20 % of the total enamel proteins, among which AMBN, also known as sheathlin [<a href="#CR5">5</a>, <a href="#CR6">6</a>] or amelin [<a href="#CR7">7</a>], is the most abundant [<a href="#CR8">8</a>–<a href="#CR10">10</a>]. <i>AMBN</i> mRNA was sequenced in pigs [<a href="#CR6">6</a>], mice [<a href="#CR11">11</a>], humans [<a href="#CR12">12</a>, <a href="#CR13">13</a>], cattles and guinea-pigs (direct submission to NCBI database). In non-mammals, <i>AMBN</i> cDNA was published only in a crocodile [<a href="#CR14">14</a>] and in the clawed toad [<a href="#CR15">15</a>]. Genomic DNA (gDNA) sequences are known in a lizard [<a href="#CR16">16</a>] and in the coelacanth [<a href="#CR17">17</a>]. These findings indicate that <i>AMBN</i> was present early during vertebrate evolution, and at least in the last common ancestor of sarcopterygians (420 million years ago, Ma [<a href="#CR18">18</a>]). However, the large evolutionary distance among the few AMBN sequences available in non mammalian tetrapods does not allow accurate comparison and further analysis, including new data in non mammalian sarcopterygians are needed to better understand AMBN evolution through such a large geological period.</p> <p>AMBN is an intrinsically unstructured, enamel matrix protein (EMP), and its functions remain unclear. It has been suggested either to be a structural component of the enamel matrix playing a role in maintaining the prismatic structure of growing enamel crystals at the rod and interrod boundaries [<a href="#CR6">6</a>, <a href="#CR19">19</a>, <a href="#CR20">20</a>], or to be involved in ameloblast adhesion [<a href="#CR21">21</a>–<a href="#CR23">23</a>], or to function as a growth factor [<a href="#CR24">24</a>–<a href="#CR27">27</a>] or as a signaling molecule [<a href="#CR26">26</a>, <a href="#CR28">28</a>]. AMBN possesses two calcium-binding domains that interact with Ca<sup>2+</sup> ions after being liberated by proteolysis [<a href="#CR29">29</a>–<a href="#CR31">31</a>]. The transcription start site (TSS) and the promoter region of <i>AMBN</i> were analyzed in the mouse [<a href="#CR32">32</a>]. This region contains cis-acting elements that function both to enhance and suppress transcription, some of them regulating transcription activity in mesenchymal cells <i>in vitro</i> [<a href="#CR33">33</a>]. <i>Cbfa1</i>, the essential transcription factor for osteoblast differentiation, was shown to play an important role in <i>AMBN</i> transcription [<a href="#CR34">34</a>].</p> <p>It is worth noting that (i) <i>AMBN</i> expression was reported during craniofacial bone development in rats [<a href="#CR35">35</a>], and (ii) <i>in vitro</i> and <i>in vivo</i> experiments have suggested that this protein could also play a role in bone formation and repair [<a href="#CR27">27</a>, <a href="#CR36">36</a>, <a href="#CR37">37</a>]. These findings could indicate that AMBN plays a role in early bone formation and modeling, but they have been contradicted by a study showing no implication in bone modeling and repair [<a href="#CR38">38</a>]. Moreover, a possible function in bone development remains elusive as (i) <i>AMBN</i><sup>−/−</sup> mice do not exhibit apparent bone phenotype [<a href="#CR21">21</a>], (ii) <i>AMBN</i> is subjected to pseudogenization in birds and in mammalian species, in which the capability to form either enamel or teeth has been lost, e.g., xenarthrans, pangolins and baleen whales, indicating that AMBN is a tooth specific protein [<a href="#CR39">39</a>–<a href="#CR41">41</a>], and (iii) <i>AMBN</i> expression was found to be restricted to teeth in rodents and in a crocodile [<a href="#CR9">9</a>, <a href="#CR14">14</a>]. Finally, <i>AMBN</i><sup>−</sup>/<sup>−</sup> mice develop severe enamel hypoplasia only, a phenotype that indicates a crucial role of AMBN in enamel formation [<a href="#CR21">21</a>].</p> <p>Because of its important role in mammalian enamel formation, for long <i>AMBN</i> has been considered a candidate gene for amelogenesis imperfecta hereditary type 2 (AIH2), a human genetic disease [<a href="#CR42">42</a>–<a href="#CR46">46</a>]. In humans, <i>AMBN</i> is located on the long arm of chromosome 4 (4q13-21), containing the gene locus for the autosomal dominant hypoplastic form of AIH2, that affects enamel formation and is the most prevalent amelogenesis imperfecta (AI) type (85 % of all inherited AI) [<a href="#CR12">12</a>, <a href="#CR47">47</a>]. However, it is only recently that the first case of a disease-associated mutation of <i>AMBN</i> (homozygous exon 6 deletion) was reported in a family having hypoplastic AI [<a href="#CR48">48</a>]. The lack of other cases of <i>AMBN</i>-associated AIH2 strongly contrasts with the important role played by AMBN during amelogenesis. This contradiction could be explained by an autosomal recessive pattern of inheritance as demonstrated in <i>AMBN</i><sup>+/−</sup> mice that do not exhibit an apparent dental phenotype [<a href="#CR49">49</a>].</p> <p>Interestingly, human ameloblastoma, a benign but destructive tumor, expresses <i>AMBN</i> transcripts with tumor-specific mutations [<a href="#CR13">13</a>, <a href="#CR50">50</a>, <a href="#CR51">51</a>]. Such expression suggests that <i>AMBN</i> plays a role in epithelial odontogenic tumors. <i>AMBN</i> is also expressed in osteosarcoma cells [<a href="#CR27">27</a>]. In humans and some mammals, amyloidosis has been associated with calcifying epithelial odontogenic tumors (CEOT) and <i>AMBN</i> was found to be highly expressed in these cells [<a href="#CR52">52</a>]. <i>AMBN</i> deficiency was proposed to be the cause of the odontogenic tumors seen in 20 % of the <i>AMBN</i><sup>−</sup>/<sup>−</sup> mice [<a href="#CR21">21</a>]. Also, tumor suppressor genes <i>P21</i> and <i>P27</i> are upregulated in <i>in vitro</i> transfected ameloblastoma cells [<a href="#CR22">22</a>].</p> <p>In recent years, a series of detailed evolutionary analyses of mammalian SCPPs have been conducted (AMEL [<a href="#CR53">53</a>]; ENAM [<a href="#CR54">54</a>]; MEPE [<a href="#CR55">55</a>]; AMTN [<a href="#CR56">56</a>]; DMP1 [<a href="#CR57">57</a>]) with the aim to (i) reveal regions and residues important for the protein function, (ii) predict or validate mutations responsible for genetic diseases, and (iii) understand their mode of evolution, origin and relationships [<a href="#CR3">3</a>, <a href="#CR4">4</a>]. Here, we perform such an evolutionary analysis of mammalian AMBN sequences in order to predict functionally important sites of the protein and to identify candidate disease-associated mutations responsible for AI in human, an approach called phylomedicine [<a href="#CR58">58</a>].</p> </div> <div class="materials_methods" title="materials|methods"> <div class="title" tagx="title" title="title">
Material and methods</div> <div class="ambnsequencesandalignment" title="sec"> <div class="title" tagx="title" title="title">
AMBN sequences and alignment</div> <p>A total of 56 mammalian full-length <i>AMBN</i> coding sequences, representative of the main lineages, were extracted from the NCBI [<a href="#CR59">59</a>] and Ensembl [<a href="#CR60">60</a>] databases: four published, full-length sequences (human, pig, mouse and rat); two sequences published only in GenBank (cow and guinea-pig), all including the UTRs; 33 computer-predicted sequences, i.e., available from the automatic analysis of sequenced mammalian genomes; and 17 sequences obtained using BLAST from whole genome shotgun data.</p> <p>The <i>AMBN</i> sequences of armadillo and sloth (Xenarthra), and of aardvark (Afrotheria) are particular because the former is enamel-less in adult and the two others lack enamel, even in juveniles [<a href="#CR41">41</a>, <a href="#CR61">61</a>]. AMBN being a tooth-specific protein [<a href="#CR39">39</a>, <a href="#CR40">40</a>] the gene has accumulated mutations after the ability to form enamel was lost, or reduced, in these lineages [<a href="#CR41">41</a>]. Therefore, we performed our evolutionary analyses with two datasets: the complete dataset of 56 sequences and a reduced dataset of 53 functional mammalian AMBN sequences. The published crocodilian sequence (<i>Caiman crocodilus</i>) was also used as outgroup for calculation of the mammalian ancestral sequence. Species names and sequence references are indicated in the Additional file <a href="#MOESM1">1</a>.</p> <p>The coding regions of the <i>AMBN</i> sequences were translated into putative amino acid sequences, aligned to the human sequence using Clustal X 2.0 [<a href="#CR62">62</a>], and checked by hand using Se-Al v.2.0a11 software [<a href="#CR63">63</a>, <a href="#CR64">64</a>]. Each of the computer-predicted sequences was carefully checked against the published sequences, taking into account the intron/exon boundaries. When necessary, they were completed and/or corrected using (i) Blast search against the Ensembl genome database, (ii) the NCBI trace archives, (iii) the whole genome shotgun (WGS) repository sequences, and (iv) resequencing by performing classical PCR on DNA extracted from alcohol-preserved tissues. In our final alignment only a few residues were missing, representing less than 1 % of the data. The positions with missing data were included in our analyses and treated as "unknown states".</p> </div> <div class="5'untranslatedregion(5'utr)andsignalpeptides" title="sec"> <div class="title" tagx="title" title="title">
5' untranslated region (5' UTR) and signal peptides</div> <p>Taking the published mRNA sequences as templates, we obtained the putative 5' UTR (i.e., exon 1 and beginning of exon 2) of <i>AMBN</i> sequences in 49 genomes using BLAST. Then, using Clustal X 2.0 and Se-Al v.2.0a11, we aligned these sequences against the published mammalian sequences of exons 1 and 2 with particular attention paid on the exon-intron boundaries. The transcription start site (5' end of exon 1) was defined by default, but this region was not useful for our analysis. The 49 sequences were analyzed to identify the ATGs that could be correct translation initiation sites (TIS). We used the DNA functional site miner [<a href="#CR65">65</a>] that predicts functional TIS i.e., possessing the highest Kozak consensus score [<a href="#CR66">66</a>, <a href="#CR67">67</a>]. All identified putative signal peptides (SPs) were analyzed using the Signal P 3.0 server [<a href="#CR68">68</a>]. This software predicts the location of the three characteristic regions (n-, h- and c- regions) of a SP, the putative cleavage site of the protein, and calculates the probability for each SP to be functional [<a href="#CR69">69</a>].</p> </div> <div class="phylogeneticreconstruction" title="sec"> <div class="title" tagx="title" title="title">
Phylogenetic reconstruction</div> <p>The 53 functional mammalian <i>AMBN</i> sequences were aligned based on their amino acid translations using MAFFT [<a href="#CR70">70</a>] within Geneious R6.03 [<a href="#CR71">71</a>]. The alignment was then restricted to the length of the human AMBN sequence by excluding all sites containing gaps in non-human sequences and the final stop codon. This procedure resulted in a dataset containing 1341 nucleotide sites (447 codons/amino acids versus 500 when including gaps). Then the three non-functional sequences (aardvark, armadillo and sloth) were added to this dataset using the program MACSE specifically designed to deal with frameshifts and stop codons [<a href="#CR72">72</a>]. Ambiguously aligned codons were then excluded using Gblocks [<a href="#CR73">73</a>] leading to a dataset with 1239 nucleotide sites (413 codons) for 56 taxa. Maximum likelihood (ML) phylogenetic reconstructions were performed on the two nucleotide alignments under the GTR + GAMMA model using RAxML 7.3.2 [<a href="#CR74">74</a>]. The statistical support for the ML tree was evaluated using 100 bootstrap replications of the initial ML search. The platypus (<i>Ornithorhynchus anatinus</i>) sequence was used as outgroup in all analyses.</p> </div> <div class="selectionanalyses" title="sec"> <div class="title" tagx="title" title="title">
Selection analyses</div> <p>We evaluated the selective constraints acting on AMBN by performing a number of statistical tests based on the ratio of non-synonymous (dN) to synonymous (dS) substitutions (dN/dS or ω) using the codeml program of the PAML 4.7a package [<a href="#CR75">75</a>]. In all analyses, we used the mammalian species tree topology as inferred by Meredith et al. [<a href="#CR76">76</a>], except for the position of the placental root, which follows Romiguier et al. [<a href="#CR77">77</a>]. We first tested for significant among-site variation in dN/dS ratio along the AMBN molecule by using site-specific codon models for the detection of positive selection. This was achieved by comparing the fit of a nearly neutral model (M7) versus a model (M8) that allows a fraction of positively selected sites [<a href="#CR78">78</a>, <a href="#CR79">79</a>]. These two models were compared using a hierarchical likelihood ratio test (LRT) [<a href="#CR80">80</a>]. A conservation index (CI) was then calculated for each codon from the Bayes Empirical Bayes (BEB) estimated mean values of ω obtained under the M8 model following the procedure proposed by Burk-Herrick et al. [<a href="#CR81">81</a>]. The CI was defined as 1-ω with values of neutral (ω = 1) and positively selected sites (ω &amp;gt; 1) being set to 0 in order to graphically represent site-specific selective constraints along the AMBN molecule [<a href="#CR82">82</a>].</p> <p>The branch specific variation of dN/dS in AMBN across the mammalian species tree was jointly reconstructed from the complete 56-taxa dataset with divergence times while controlling the effect of life-history traits [<a href="#CR83">83</a>] using the Bayesian framework implemented in the CoEvol program [<a href="#CR84">84</a>]. The dN/dS ratio was estimated under the dsom procedure together with the 24 calibration constraints issued from Benton et al. [<a href="#CR85">85</a>] that were compatible with our taxon sampling to estimate divergence times. The prior on the root node was set to 220 Ma with a standard deviation of 60 Ma following the expert estimate for the split between Monotremata and Placentalia reported on the TimeTree2 website [<a href="#CR86">86</a>]. The values of the three life-history traits incorporated into the analysis (body mass, longevity, and sexual maturity) were extracted from the PanTheria database [<a href="#CR87">87</a>]. Two independent MCMC were run for a total of 2500 cycles sampling points every cycle. The first 500 points of each MCMC were then excluded as the burnin, and inferences were made from the remaining 2000 sampled points.</p> <p>To further test for significant variations in dN/dS along specific branches, we used the branch model implemented in codeml allowing ω to vary among specific branches and/or clades [<a href="#CR88">88</a>]. Hierarchical LRTs were calculated between the one-ratio model (M0) and alternative models (M2ω, M4ω, M5ω, M8ω), in which branches with a mean dN/dS &amp;gt; 0.6 as estimated in the previous Bayesian reconstruction were allowed to have their own ω (see Table <a href="#Tab1">1</a>). <div class="table-wrap_UNKNOWN" id="Tab1"><span class="label" tagx="label" title="label">Table 1</span> </div></p><p>Results of likelihood ratio tests for positive selection in AMBN</p>  <table frame="hsides" rules="groups"> <thead> <tr> <th>Models</th> <th colspan="2">Hypotheses</th> <th colspan="3">LRT</th> </tr> <tr> <th /> <th>Null hypothesis</th> <th>Alternative hypothesis</th> <th>−2ΔLnL (df)</th> <th><i>p</i>-value</th> <th>Sites or branches</th> </tr> </thead> <tbody> <tr> <td>Site Models</td> <td /> <td /> <td /> <td /> <td>Sites under M8 (BEB)</td> </tr> <tr> <td rowspan="5">53-taxa data set</td> <td rowspan="5">M7: beta</td> <td rowspan="5">M8: beta&amp;amp;ω</td> <td rowspan="5">42.77 (2)</td> <td rowspan="5"><i>p</i> &amp;lt; 0.001</td> <td>135 (ω = 1.49 ± 0.06; PP = 0.99)</td> </tr> <tr> <td>328 (ω = 1.50 ± 0.04; PP = 1.00)</td> </tr> <tr> <td>360 (ω = 1.48 ± 0.12; PP = 0.96)</td> </tr> <tr> <td>375 (ω = 1.49 ± 0.06; PP = 0.99)</td> </tr> <tr> <td>394 (ω = 1.48 ± 0.11; PP = 0.97)</td> </tr> <tr> <td>Branch Models</td> <td /> <td /> <td /> <td /> <td>Branches</td> </tr> <tr> <td>53-taxa data set</td> <td>M0: One-ratio model</td> <td>M2ω: 2-ratio model</td> <td>29.89 (1)</td> <td><i>p</i> &amp;lt; 0.001</td> <td>Background (ω = 0.44), Catarrhines (ω = 1.32)</td> </tr> <tr> <td /> <td>M0: One-ratio model</td> <td>M5ω: 5-ratio model</td> <td>52.16 (4)</td> <td><i>p</i> &amp;lt; 0.001</td> <td>Background (ω = 0.43), Catarrhines (ω = 1.32), Elephant (ω = 0.73), Elephant shrew (ω = 0.75), Aye aye (ω = 1.13)</td> </tr> <tr> <td>56-taxa data set</td> <td>M0: One-ratio model</td> <td>M2ω: 2-ratio model</td> <td>18.11 (1)</td> <td><i>p</i> &amp;lt; 0.001</td> <td>Background (ω = 0.45), Catarrhines (ω = 1.11)</td> </tr> <tr> <td /> <td>M0: One-ratio model</td> <td>M4ω: 4-ratio model</td> <td>19.73 (3)</td> <td><i>p</i> &amp;lt; 0.001</td> <td>Background (ω = 0.45), Armadillo (ω = 0.67), Sloth (ω = 1.10), Aardvark (ω = 0.76)</td> </tr> <tr> <td /> <td>M0: One-ratio model</td> <td>M5ω: 5-ratio model</td> <td>39.62 (4)</td> <td><i>p</i> &amp;lt; 0.001</td> <td>Background (ω = 0.44), Catarrhines (ω = 1.11), Armadillo (ω = 0.67), Sloth (ω = 1.10), Aardvark (ω = 0.76)</td> </tr> <tr> <td /> <td>M0: One-ratio model</td> <td>M8ω: 8-ratio model</td> <td>58.14 (7)</td> <td><i>p</i> &amp;lt; 0.001</td> <td>Background (ω = 0.42), Catarrhines (ω = 1.11), Armadillo (ω = 0.68), Sloth (ω = 1.10), Aardvark (ω = 0.76), Elephant (ω = 0.68), Elephant shrew (ω = 0.76), Aye aye (ω = 1.07)</td> </tr> </tbody> </table> <div class="table-wrap-foot_UNKNOWN"> <p><i>BEB</i> Bayes Empirical Bayes, <i>PP</i> Posterior Probability</p> </div>  <p /> </div> <div class="ancestralsequencereconstruction" title="sec"> <div class="title" tagx="title" title="title">
Ancestral sequence reconstruction</div> <p>Ancestral sequence reconstruction of mammalian <i>AMBN</i> was computed with FastML [<a href="#CR89">89</a>] on the FastML server [<a href="#CR90">90</a>]. The <i>AMBN</i> sequence of the caiman (<i>Caiman crocodilus</i>) was added to the 53-taxa alignment using MAFFT in order to serve as outgroup. FastML was used for inferring ancestral amino acid sequences at each node of the nucleotide-derived ML tree under the LG + G8 + F model using the marginal reconstruction procedure including indels.</p> </div> </div> <div class="results" title="results"> <div class="title" tagx="title" title="title">
Results</div> <p>For a better understanding of the results presented below, our current knowledge of <i>AMBN</i> structure (exemplified in humans) and the location of cleavage sites of the protein (identified in pigs) are illustrated in Fig. <a href="#Fig1">1</a>. </p><div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/"><span class="label" tagx="label" title="label">Fig. 1</span></a></div>   <p>Gene structure and proteolytic cleavage products of ameloblastin. <b>a</b>: Human <i>AMBN</i>. Gene structure based on the reference gene sequence (Genbank accession No NC_000004.11). Exons are numbered from 1 to 13. Two translation initiation sites (ATG) are found: in exon 1 and exon 2. The untranslated regions are in light grey. The number of base pairs (bp) is indicated above each exon. The nomenclatural counts of encoded amino acids (p.), cDNA bp (c.) and gDNA bp (g.) are also indicated below each exon. Introns are symbolized by the black lines (not at scale) with the nucleotide number indicated above. The stop codon (TGA) is located in exon 13. <b>b</b>: Porcine AMBN. Diagram showing the cleavage products and their corresponding location on the protein sequence (after [<a href="#CR6">6</a>, <a href="#CR20">20</a>, <a href="#CR29">29</a>, <a href="#CR99">99</a>, <a href="#CR101">101</a>, <a href="#CR116">116</a>]). Top: <i>AMBN</i> exons. Note that exons 8 and 9, which encode 26 residues in humans are lacking in porcine <i>AMBN</i>. Bottom: AMBN protein including either the 26 residues forming the signal peptide-SP (1–421) or not (1–395). Several remarkable residues were identified: two phosphorylated residues (S<sup>17</sup>, T<sup>251</sup>) and two putatively phosphorylated residues (S<sup>209</sup>, S<sup>210</sup>), two O-glycosylated residues (S<sup>86</sup>, T<sup>361</sup>), two hydroxylated prolines (P<sup>11</sup>, P<sup>324</sup>), and the proline-rich region. The secreted protein has an apparent molecular weight of 62 kDa; cleavage sites leading to various AMBN fragments found <i>in vivo</i> are indicated in roman characters, two MMP20 cleavage sites found in <i>in vitro</i> studies are in italics</p>   </div> <p /> <div class="alignmentandsequencecomparisons" title="sec"> <div class="title" tagx="title" title="title">
Alignment and sequence comparisons</div> <p>The 56 AMBN sequences studied here represent 46 families distributed in 19 orders and are representative of the current mammalian phylogenetic diversity (Additional file <a href="#MOESM1">1</a>). The alignment of the 53 functional sequences resulted in a 500 amino acid long sequence, when considering methionine in exon 1 (Additional file <a href="#MOESM2">2</a>), and including gaps required for accurate alignment. For convenience of presentation, the duplicated exons 9c, 9e, and 9d were not shown: they were nearly identical and only found in a few species (see below).</p> <p>Sequence comparison revealed a few insertions, the largest being related to the duplication of small exons in 12 species (see below). These additional exons account for a large part of the length variation observed among the sequences (Additional file <a href="#MOESM2">2</a>). In the 10 rodent AMBN analyzed, an insertion of 4 or 5 residues (QG/NMAP) was identified in the region encoded by exon 3. This event likely occurred in the rodent ancestral branch after the divergence with lagomorphs (rabbit and pika), which do not possess this insertion. The insertion is located close to a remarkably conserved region and probably results from the duplication of the neighboring PGMAS residues (Additional file <a href="#MOESM2">2</a>). In hedgehog, six residues were inserted in the region encoded by exon 13 at positions 365–370, and in marsupials three amino acids (IKQ) were inserted in the region encoded by the 3′ end of exon 2. In addition, a few insertions and deletions (indels) of one to five residues were found distributed in the whole sequences and no sequence repeats were identified, to the exception of the exon duplication reported below.</p> </div> <div class="additionalexons" title="sec"> <div class="title" tagx="title" title="title">
Additional exons</div> <p>Human <i>AMBN</i> possesses two exons, exons 8 and 9, which are absent in rodent sequences. Therefore, we screened the DNA region located between exons 7 and 10 in the mammalian genomic sequences to check whether additional exons were present in this region. We show that a gene structure composed of 11 exons (i.e., exons 1 to 7 and 10 to 13) characterizes 41 mammalian <i>AMBN</i> out of the 53 functional sequences investigated. This structure is also conserved in caiman, which indicates that the ancestral mammalian <i>AMBN</i> most probably possessed 11 exons. In 12 species, however, <i>AMBN</i> displays additional exons in this particular region. They are eight simiiform primates (out of nine), three laurasiatherians (horse, microbat and hedgehog), and one afrotherian (elephant shrew) (Additional file <a href="#MOESM2">2</a>): one exon in orangutan and microbat; two in humans, chimpanzee, gibbon, squirrel monkey, marmoset and hedgehog; three in elephant shrew; four in baboon and macaque; and six in horse. These additional exons have the same length (39 bp) and are nearly identical in sequence to exon 7 (Additional file <a href="#MOESM2">2</a>). These features strongly suggest that these exons originated all from tandem duplications of a short DNA region containing exon 7. This region is the only <i>AMBN</i> location, in which additional exons were identified. The two exons found between exons 7 and 10 in human <i>AMBN</i> were numbered 8 and 9a; therefore, the extra exons found between exon 9a and 10 were numbered 9b, 9c, etc. In the horse, the last duplicated exon was numbered 9e. It is worth noting that closely related species such as shrew <i>vs.</i> hedgehog, megabat <i>vs.</i> microbat, etc. lacked duplicated exons, a finding which strongly suggests that the duplications occurred independently within the different lineages (Additional file <a href="#MOESM2">2</a>). However, in primates, duplication of this short DNA region is found only in the Simiiformes (Platyrrhini + Catarrhini), which could indicate that the first duplication likely occurred in the common ancestor of simiiforms, after this lineage diverged from Tarsiiformes, and was subsequently conserved in all simiiform lineages except in gorilla. The presence of these duplicated exons contributes to a substantial increase in length of the encoded protein: from 421 amino acids (aa) on average in the majority of sequences to e.g., 447 aa in humans, 473 in baboon, and 499 aa in horse, which is the largest AMBN mammalian sequence known to date (Additional file <a href="#MOESM2">2</a>). These exons also encode two to four prolines, which contribute to enlarge the proline-rich region of the protein.</p> <p>Analyses of exon/intron boundaries in representative species indicated that the 21 nucleotides on each side of the exons are rather well conserved (Additional file <a href="#MOESM3">3</a>). The unchanged nucleotides in these intronic regions could be important for intron splicing.</p> </div> <div class="translationinitiationsite(tis)andsignalpeptide(sp)analyses" title="sec"> <div class="title" tagx="title" title="title">
Translation initiation site (TIS) and signal peptide (SP) analyses</div> <p>Alignment of the 5' region of the <i>AMBN</i> sequences revealed that the 3' end of exon 1 and the 5' end of exon 2 are well conserved (Fig. <a href="#Fig2">2</a>). Depending on the species, in exon 1, exon 2 and/or in both, DNAFSMiner identified one to five ATGs in the right context i.e., that does not lead to in-frame downstream stop codons. For each sequence analyzed, the first ATG was always predicted as being the functional TIS. In most placentals the putative TIS was located in exon 1, but with a few exceptions (bushbaby, hedgehog, bats and elephant), in which it is found in exon 2 as in marsupials, and monotremes (Fig. <a href="#Fig2">2</a>). In placentals, the TIS was located at the same position in the 3' region of exon 1, with the exception of Bovinae<i>,</i> in which another TIS, was identified upstream. In humans, for instance, DNAFSMiner predicted that the first ATG (starting at position 76, exon 1) is the correct functional TIS (score = 0.844). However, when exon 1 was not included in the analysis, the first ATG in exon 2 (position 106) was also predicted as the functional TIS (score = 0.627). This means that the second TIS could be used if that encoded by exon 1 was lacking, i.e. by using different promoter and transcription initiation sites. </p><div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/"><span class="label" tagx="label" title="label">Fig. 2</span></a></div>   <p>Location of putative translation initiation sites (TIS) in 30 mammalian and one crocodilian <i>AMBN</i> sequences. The arrows (−3, +4) point to nucleotide positions that play an essential role in the initiation of the translation. Two putatively functional TIS (squared ATGs on grey background in exons 1 and 2) are present in placental sequences, but a few exceptions, while only one TIS is identified in marsupials, platypus and crocodile. An additional, putatively functional TIS is present in cow <i>AMBN</i></p>   </div> <p /> <p>Interestingly, the predicted TIS in exon 1 exhibits a weak Kozak consensus as it lacks a purine (A/G) in position −3 while possessing a pyrimidine in position +4. However, such ATG could serve as the initiator codon [<a href="#CR66">66</a>]. In contrast, the ATG located in exon 2 has a strong Kozak consensus (purines in −3 and +4). As a consequence, translation could occur not only at the first (weak) TIS but also at the next (strong) TIS, resulting in the synthesis of two proteins [<a href="#CR66">66</a>]. The additional TIS identified in cow <i>AMBN</i> exon 1 displays also a strong Kozak consensus (a purine at positions −3 and +4), and DNAFSMiner predicted this TIS as the functional one. This additional TIS is also present in the water buffalo <i>Bubalus bubalis</i> and the American bison <i>Bison bison</i> (Bovinae) but is absent in the sheep, goat and Tibetan antelope <i>Pantholops hodgsonii</i> (Antilopinae) <i>AMBN</i> sequences (not shown), which suggests that the CTG to ATG substitution occurred along the Bovinae lineage after its divergence from Antilopinae some 20 Mya [<a href="#CR91">91</a>].</p> <p>Our alignment indicates that the TIS in exon 2 is conserved in all <i>AMBN</i> analyzed, including that of the caiman, which means an ancient origin for this ATG. In contrast, the ATG in exon 1 was probably recruited in the eutherian ancestor, after an A to T substitution changed the ancestral AAG to ATG (Fig. <a href="#Fig2">2</a>). This ATG appears to have been secondarily lost independently in various lineages or species: ATG to CTG in elephant<i>,</i> ATG to AGG in bats, and ATG to ACA in bushbaby. However, in most primates, glires, most laurasiatherians, and five afrotherians out of six studied, the translated, putative AMBN sequences exhibit two methionines: the methionine encoded by the ATG in exon 1, numbered M<sup>1</sup>, and the one encoded by the ATG in exon 2, M<sup>11</sup>. In cow the first methionine is located five residues upstream M<sup>1</sup>.</p> <p>The analysis of the putative SPs with SignalP revealed a similar, high probability for the SP starting either at M<sup>1</sup> or M<sup>11</sup>. As a consequence, the SP of most placental AMBN can be either large (26 aa), when starting at M<sup>1</sup>, or short (16 aa) when beginning at M<sup>11</sup>. The three typical regions — positively charged n-region, hydrophobic h-region, and polar c-region — were found in both SPs (Additional file <a href="#MOESM4">4</a>). The maximal cleavage site probability was always located between Ala<sup>26</sup> and Val<sup>27</sup>. The corresponding predicted cleavage site is strongly supported by the presence of these two residues in all AMBN sequences analyzed, excepted for the platypus, in which Val is replaced with Ile from the same amino acid group (Additional file <a href="#MOESM2">2</a>).</p> </div> <div class="polyadenylationsites(ps)" title="sec"> <div class="title" tagx="title" title="title">
Polyadenylation sites (PS)</div> <p>Several putative alternative polyadenylation signals (PS) were identified in the 3' UTR of the mammalian <i>AMBN</i> sequences (Table <a href="#Tab2">2</a>). This region is large, ranging from 441 to 557 bp in representative species of the main lineages (493 bp in the caiman). The number of PS ranged from two in the platypus to five in several species, including humans, and were numbered PS1 to PS5, from the stop codon onwards. PS were either the highly conserved hexamer AATAAA found at three locations (PS2, PS3, PS5) or the common variant ATTAAA found at two locations (PS1, PS4) (Table <a href="#Tab2">2</a>). In our dataset the latter was less conserved than the canonical signal, and it was not found in the caiman sequence. When both present, PS1 and PS2 are close to one another and can be considered a single PS; PS3 is found in all sequences but the platypus; PS4 is present in rodents and primates, in some laurasiatherians and in marsupials; PS5 is found in all sequences. The caiman <i>AMBN</i> possesses PS2, PS3 and PS5. In marsupials, PS4 and PS5 are close one to each other. Altogether these findings suggest that (i) three PS (PS2, PS3 and PS5) were present in the 3'UTR of the last common ancestral mammalian <i>AMBN</i> and (ii) PS3 was secondarily lost in the platypus lineage. <div class="table-wrap_UNKNOWN" id="Tab2"><span class="label" tagx="label" title="label">Table 2</span> </div></p><p>Location of putative alternative polyadenylation signals (PS) in the 3' UTR of <i>AMBN</i> cDNA sequences</p>  <table frame="hsides" rules="groups"> <thead> <tr> <th rowspan="2">Species</th> <th>Length</th> <th>PS1</th> <th>PS2</th> <th>PS3</th> <th>PS4</th> <th>PS5</th> </tr> <tr> <th>3'UTR</th> <th>ATTAAA</th> <th>AATAAA</th> <th>AATAAA</th> <th>ATTAAA</th> <th>AATAAA</th> </tr> </thead> <tbody> <tr> <td>Human</td> <td>553</td> <td>106-111</td> <td>129-134</td> <td>189-194</td> <td>415-420</td> <td>536-541</td> </tr> <tr> <td>Bushbaby</td> <td>483</td> <td>77-83</td> <td>100-105</td> <td>157-162</td> <td>344-349</td> <td>466-471</td> </tr> <tr> <td>Tree shrew</td> <td>553</td> <td>106-111</td> <td>129-134</td> <td>189-194</td> <td>412-417</td> <td>536-541</td> </tr> <tr> <td>Mouse</td> <td>543</td> <td>-</td> <td>127-132</td> <td>184-189</td> <td>403-408</td> <td>526-531</td> </tr> <tr> <td>Squirrel</td> <td>539</td> <td>105-110</td> <td>128-133</td> <td>188-193</td> <td>406-411</td> <td>521-526</td> </tr> <tr> <td>Rabbit</td> <td>549</td> <td>106-111</td> <td>129-134</td> <td>189-194</td> <td>410-415</td> <td>532-537</td> </tr> <tr> <td>Cow</td> <td>555</td> <td>106-111</td> <td>129-134</td> <td>189-194</td> <td>419-424</td> <td>538-543</td> </tr> <tr> <td>Horse</td> <td>556</td> <td>106-111</td> <td>129-134</td> <td>189-194</td> <td>-</td> <td>539-544</td> </tr> <tr> <td>Dog</td> <td>557</td> <td>107-112</td> <td>130-135</td> <td>190-195</td> <td>-</td> <td>540-545</td> </tr> <tr> <td>Microbat</td> <td>513</td> <td>-</td> <td>109-114</td> <td>169-174</td> <td>377-382</td> <td>497-502</td> </tr> <tr> <td>Elephant</td> <td>556</td> <td>-</td> <td>126-131</td> <td>186-191</td> <td>-</td> <td>537-542</td> </tr> <tr> <td>Opossum</td> <td>540</td> <td>121-126</td> <td>-</td> <td>209-214</td> <td>495-500</td> <td>520-525</td> </tr> <tr> <td>Platypus</td> <td>441</td> <td>116-121</td> <td>-</td> <td>-</td> <td>-</td> <td>426-431</td> </tr> <tr> <td>Caiman</td> <td>493</td> <td>-</td> <td>95-100</td> <td>298-303</td> <td>-</td> <td>475-480</td> </tr> </tbody> </table> <div class="table-wrap-foot_UNKNOWN"> <p>TGA-------------PS1---PS2--------PS3-----------------------PS4--------------PS5--AAAAAAA</p> <p>The PS are given for 13 representative species of the main mammalian lineages and the crocodile. Bottom: schematical representation of PS locations on the human <i>AMBN</i> sequence from the stop codon (TGA) to the poly(A); an hyphen represents 10 nucleotides</p> </div>  <p /> </div> <div class="aminoacidcomposition" title="sec"> <div class="title" tagx="title" title="title">
Amino acid composition</div> <p>EMPs are characterized by their high amount of proline (P) and glutamine (Q), and are included in the P/Q-rich sub-family of SCPPs [<a href="#CR92">92</a>]. In the seven representative species analyzed for AMBN residue composition, including caiman, proline is well represented (14.09 % in average for the full-length sequence and 23.78 % in average for the proline rich region) compared to the 5.1 % encountered in most proteins (Additional file <a href="#MOESM5">5</a>). Glutamine (8.7 and 15.7 %, respectively) is also well represented compared to the overall value of 4.0 %. It is worth noting that the percentage of proline in both the full-length sequence and the proline-rich domain is relatively similar from caiman to humans, while the glutamine percentage is decreasing from 11.79 to 6.49 % and from 20.93 to 11.20 %, respectively. These two amino acids represent an average of <i>ca</i> 23 % of the residues in the full-length AMBN sequence and <i>ca</i> 40 % in the proline-rich domain. In the ancestral mammalian AMBN (see below) these amino acids represent <i>ca</i> 24 % (<i>P</i> = 14.1; Q = 9.8 %) and <i>ca</i> 42 % (22.2 + 20.2), respectively (Additional file <a href="#MOESM5">5</a>).</p> </div> <div class="ancestralambnandenamel-lessspecies" title="sec"> <div class="title" tagx="title" title="title">
Ancestral AMBN and enamel-less species</div> <p>In addition to its slightly higher P/Q percentage compared to living species, the ancestral mammalian <i>AMBN</i>, calculated from the 53 functional <i>AMBN</i> and using the caiman sequence as outgroup, displays a structure composed of 11 exons (1–7; 10–13), i.e., without duplicated exons between exons 7 and 10. There is a single TIS located in exon 2. Otherwise, the ancestral AMBN sequence is structurally similar to that of most mammalian AMBN and, in particular, possesses all the important positions and domains reported above.</p> <p>Sloth, armadillo and aardvark sequences were aligned with the ancestral AMBN. This alignment revealed that selective pressures on the AMBN sequence of these enamel-less species were more relaxed in sloth and aardvark than in armadillo (Fig. <a href="#Fig3">3</a>). Sloth and aardvark AMBN show (i) numerous residue substitutions, some having occurred in conserved positions, (ii) premature stop codons resulting from nucleotide substitution, (iii) frameshifts in the region encoded by the large exon 13 resulting from nucleotide insertion or deletion, and (iv) nucleotide substitutions in the splicing sites (not shown). In addition, a large deletion has occurred in the DNA region of aardvark <i>AMBN</i> between exons 5 and 7, and exon 6 has disappeared. Taken together these features indicate that <i>AMBN</i> is no longer functional in these two species and was subjected to pseudogenization. In contrast, armadillo <i>AMBN</i> does not exhibit stop codons and frameshifts, and residue substitutions are less numerous. However, several amino acid substitutions have occurred on conserved positions that were suggested above to play an important role for the protein function (Fig. <a href="#Fig3">3</a>). These data indicate that armadillo <i>AMBN</i> either was recently inactivated or is still active but the encoded protein is probably defective. </p><div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/"><span class="label" tagx="label" title="label">Fig. 3</span></a></div>   <p>Alignment of AMBN sequences from three enamel-less mammals with the putative ancestral mammalian AMBN sequence. Exon limits are indicated by vertical lines. The residues conserved during mammalian evolution (see Fig. <a href="#Fig5">5b</a>) are on gray background. In the aardvark genome the region containing exon 6 is deleted. (.): residue identical to the ancestral AMBN residue; (−): indel; (=): unknown amino acid; *: stop codon</p>   </div> <p /> </div> <div class="phylogenetictree" title="sec"> <div class="title" tagx="title" title="title">
Phylogenetic tree</div> <p>The ML phylogram inferred from the complete 56-taxa nucleotide data set under the GTR + GAMMA model is presented in Fig. <a href="#Fig4">4</a>. This tree illustrates the strong phylogenetic signal contained in AMBN for resolving the phylogeny of mammals. Indeed, the ML topology supports the four major placental clades: Xenarthra, Afrotheria, Laurasiatheria and Euarchontoglires, and the grouping of the latter two in Boreoeutheria. Furthermore, almost all nodes are fully resolved and supported by bootstrap values of more than 90 % with the exception of, for instance, the relationships among laurasiatherian orders or the position of tree shrew within Euarchontoglires (Fig. <a href="#Fig4">4</a>). The ML estimate of the gamma shape parameter (α) was 0.92, which underlines the relative homogeneity of the site-specific substitution rates along the AMBN molecule. Among placentals, strong variations in substitution rates are revealed with contrasted branch lengths observed in the ML tree, with fast evolving taxa such as tenrec and elephant-shrew in Afrotheria, shrew and hedgehog in Laurasiatheria, all Glires except pika and squirrel, and bushbaby in Primates within Euarchontoglires. </p><div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/"><span class="label" tagx="label" title="label">Fig. 4</span></a></div>   <p>Maximum likelihood tree (GTR + GAMMA model) using the complete dataset of 56 mammalian <i>AMBN</i> sequences. The platypus (<i>Ornithorhynchus anatinus</i>) was used as outgroup. Bootstrap percentages are indicated. Nodes with more than 90 % bootstrap support are indicated with dots and nodes with no value received less than 70 % bootstrap support. Placentals are colored by main clades: Xenarthra (red), Afrotheria (green), Laurasiatheria (orange), Scandentia (pink), Glires (purple) and Primates (blue). Species possessing various copies of exon 7 are indicated in bold (see also Additional file <a href="#MOESM2">2</a>). The three enamel-less species are underlined. Branch lengths are expressed as substitutions per site</p>   </div> <p /> </div> <div class="site-specificselectionconstraintsandconserveddomains" title="sec"> <div class="title" tagx="title" title="title">
Site-specific selection constraints and conserved domains</div> <p>Our amino acid alignment of the 53 functional AMBN sequences revealed that many residues have been unchanged throughout mammalian evolution, some of them being regrouped into short domains (Additional file <a href="#MOESM2">2</a>). Other conserved positions include amino acids that can be replaced with residues possessing the same characteristics (conservative positions). Unchanged and conservative positions are subjected to strong selective pressure, which indicates that they are functionally important for the protein. Most of them are located in the N-terminal region of the protein (encoded from exon 2 to beginning of exon 6). Conversely, more variable positions are located in the region encoded by exon 6, exon 7, and for a large part in exon 13 (Additional file <a href="#MOESM2">2</a>).</p> <p>Selective forces acting along the 53 <i>AMBN</i> codon sequences were inferred through non-synonymous vs. synonymous substitution rate (Fig. <a href="#Fig5">5a</a>). The comparison between the nearly neutral model (M7) and the model allowing a fraction of positively selected sites (M8) revealed significant among-site variation in dN/dS with the occurrence of positive selection (<i>p</i> &amp;lt; 0.001; Table <a href="#Tab1">1</a>). Under the M8 model, five codons out of 447 were identified as evolving under significant positive selection (PP &amp;gt; 0.95) during mammalian evolution. These sites are located in the proline rich region encoded by exon 6 at position 135, and in the exon 13 encoded region at positions 328, 360, 375, and 394 (Table <a href="#Tab1">1</a>, Fig. <a href="#Fig5">5b</a>). </p><div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/"><span class="label" tagx="label" title="label">Fig. 5</span></a></div>   <p><b>a</b>. Estimation of the codon conservation index along the mammalian AMBN sequences. Positions 1 to 447 refer to the human sequence. Low evolutionary rates i.e., values close to 1, indicate the conserved / important positions, while high evolutionary rates i.e., values close to 0, indicate more variable positions. <b>b</b> Amino acid sequence of human AMBN. Human AMBN is composed of 447 residues when including the 26 residues of the putative leader peptide encoded by exons 1 and 2 (numbers above the residues); otherwise, 421 amino acids constitute the protein (numbers below the residues). Conserved positions are indicated either on black background (unchanged) or on gray background (only replaced with a residue having the same chemical property); variable positions are on white background. Various remarkable domains and remarkable residues are figured in color. The arrow points to the cleavage site of the signal peptide. Arrowheads point to proteolytic sites identified in porcine AMBN. The five sites under significant positive selection are indicated with an asterisk</p>   </div> <p /> <p>The site conservation index calculated from the site-specific dN/dS values obtained under model M8 illustrated the variation of the selection constraints across the AMBN (Fig. <a href="#Fig5">5a</a>). Most of the N-ter region (positions 1 to 123, encoded from exon 1 to the 5' region of exon 6), several small regions dispersed in the AMBN sequence (notably positions 233–265, exons 10–12; 300–320, beginning of exon 13; 379–404, mid exon 13) and a short C-ter sequence (442–447, encoded by exon 13) are subjected to high functional constraints. Most of the remaining sequence is characterized by an alternation of high and low selective pressures (Fig. <a href="#Fig5">5a</a>). We identified 161 positions (out of 447 residues in human AMBN) subjected to purifying selection during more than 200 Ma of mammalian evolution pointing to biologically significant residues (Fig. <a href="#Fig5">5b</a>). These constrained positions have certainly important functions, which explains they were either unchanged (80 positions) or they displayed only few substitutions (81 positions). Among these residues are: the cleavage sites N<sup>31</sup>, G<sup>232</sup>, Y<sup>249</sup> and G<sup>326</sup>/L<sup>327</sup>; the hydroxylated P<sup>11</sup>; the O-glycosylated S<sup>86</sup>; and the phosphorylated S<sup>17</sup>, S<sup>235</sup>, S<sup>236</sup> and T<sup>277</sup>. In contrast, our study identified as variable positions, i.e., having probably no important function, the putative cleavage sites Q<sup>130</sup> and R<sup>196</sup>, hydroxylated P<sup>350</sup>, and O-glycosylated T<sup>387</sup> (Fig. <a href="#Fig5">5b</a>). Out of this dozen of important positions, there are 150 other selectively constrained positions that our evolutionary analysis pointed out as being important for the correct functioning of AMBN. Therefore, we predict that these conserved positions in AMBN, and particularly the unchanged ones, would be potentially responsible for enamel disorder if they were substituted.</p> <p>Three conserved domains (or motifs) were previously identified as possibly playing an important functional or structural role (Fig. <a href="#Fig5">5b</a>). Between the two CD63-interaction domains the position E<sup>364</sup> is conserved, while the RPGF motif is not. Half of the residues of the calcium-binding domain (Y<sup>217</sup>-R<sup>230</sup>) and eight residues out of 28 in the heparin-binding domain are conserved, which confirms that these domains are important. In addition to these domains, several conserved residues are located close one to the other in a same region of the protein, representing new motifs of unknown but important function. They are housed principally in the N-terminal region (residues 1–124) of the protein encoded by exons 3, 4, 5 and beginning of exon 6 (Fig. <a href="#Fig5">5b</a>). This large, conserved AMBN region contains important residues: the two first residues that are involved in the cleavage of the signal peptide, the hydroxylated P<sup>11</sup>, the phosphorylated S<sup>17</sup>, the O-glycosylated S<sup>86</sup> and the cleavage site N<sup>31</sup>. The residues on both sides of these positions are also conserved, but in the same region other highly conserved amino acids constitute motifs of unknown function. We identified notably two highly conserved peptide sequences: the one is encoded by exons 3 and 4 (35 residues, 29 of them being conserved), and the other encoded by the 3' end of exon 5 and 5' extremity of exon 6 (22 residues conserved), that certainly play an important role remaining to be defined. Interestingly, it is known that the 5' end of exon 6 is alternatively spliced in the few human, rodent and porcine <i>AMBN</i> transcripts available. Our alignment revealed that the residue at the splicing site (Q<sup>87</sup> = always encoded by CAG, not shown) is well conserved (Additional file <a href="#MOESM2">2</a>). Similarly, several, highly conserved motifs encoded by exons 5 and 13 are highlighted as being probably important for the functionality of AMBN.</p> <p>In mammals, AMBN does not possess a conserved RGD (integrin-binding) motif; however, we found an RGD sequence in the region encoded by the beginning of exon 13 in the cow and in four primates. This sequence was clearly acquired secondarily during mammalian evolution through the substitution RGG to RGD. The VTKG sequence (heparin-binding motif) is only present in rat, mouse, hamster and kangaroo rat, and was clearly acquired secondarily during rodent evolution through the substitution ATKG to VTKG. However, the TKG sequence is conserved in all eutherians and the lysine, K<sup>288</sup>, is unchanged (Fig. <a href="#Fig5">5b</a>). The DGEA motif previously described in rats and mice as a possible interacting site with integrins is also present in the hamster AMBN but not in other rodents. This motif was acquired secondarily during murid evolution.</p> </div> <div class="lineage-specificselectionconstraints" title="sec"> <div class="title" tagx="title" title="title">
Lineage-specific selection constraints</div> <p>The estimation of the global dN/dS ratio across the whole tree using the one-ratio branch model (M0) resulted in a ω value of 0.46 showing that AMBN is globally evolving under purifying selection in mammals (Table <a href="#Tab1">1</a>). The joint Bayesian reconstruction of the dN/dS ratio along the phylogeny revealed large variations among mammalian lineages with mean ω values of functional sequences ranging from 0.15 in cat to 1.03 in gibbon (Fig. <a href="#Fig6">6</a>). As expected, elevated ω values were found in the non-functional sequences of aardvark (0.65) and sloth (1.04) as a consequence of relaxed selection constraints, whereas the armadillo has a lower value (0.44). Conversely, some lineages with functional AMBN showed high dN/dS values such as elephant (0.74), elephant shrew (0.73), and aye-aye (0.71). However, AMBN has also evolved under strong purifying selection for instance in rodents with low values of ω observed in this group overall (0.19-0.32). </p><div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/"><span class="label" tagx="label" title="label">Fig. 6</span></a></div>   <p>Bayesian reconstruction of dN/dS ratio in AMBN along the mammalian phylogeny. The variation of dN/dS was jointly reconstructed with divergence times while controlling the effect of three life-history traits (body mass, longevity, and maturity). The Bayesian chronogram revealed strong positive selection in catarrihine primates where elevated dN/dS ratios are inferred</p>   </div> <p /> <p>The most striking pattern in this reconstruction is nonetheless the high dN/dS ratios inferred along the whole catarrhine subtree, suggesting the occurrence of positive selection on AMBN in this particular group of Primates (Fig. <a href="#Fig6">6</a>). Hierarchical likelihood ratio tests performed between different branch models allowing ω to vary among branches of the phylogeny revealed that strong positive selection has occurred in Catarrhini (Table <a href="#Tab1">1</a>). Indeed, branch model LRTs for positive selection in the catarrhine subtree were strongly significant (<i>p</i> &amp;lt; 0.001) with both the 53-taxa and 56-taxa data sets, in which the dN/dS ratio in Catarrhini was estimated to be respectively 1.32 and 1.11. Finally, among the candidate branches revealed by the previous Bayesian analysis, two were estimated to have ω values higher than 1: in sloth (ω = 1.10), in which <i>AMBN</i> is pseudogenized, and in aye-aye (ω = 1.13), in which it nevertheless appears fully functional.</p> </div> </div> <div class="discussion" title="discussion"> <div class="title" tagx="title" title="title">
Discussion</div> <div class="asecondtranslationinitiationsite(tis)wasrecruitedinanancestraleutherian" title="sec"> <div class="title" tagx="title" title="title">
A second translation initiation site (TIS) was recruited in an ancestral eutherian</div> <p>Two TIS were clearly identified in our analysis as being conserved through mammalian evolution: one in exon 1, the other in exon 2. The latter is the ancestral TIS in tetrapod <i>AMBN;</i> it is the only TIS present in reptiles [<a href="#CR14">14</a>] and frogs [<a href="#CR15">15</a>]. This TIS is homologous to that of other SCPPs [<a href="#CR1">1</a>]. Interestingly, our analysis indicated also that the former TIS was recruited in the last common eutherian ancestor, then conserved in most lineages during more than 100 Ma of evolution [<a href="#CR76">76</a>]. This finding suggests that this additional TIS plays a role in enamel formation in most eutherian species, and contradicts previous studies proposing that the only functional TIS was that in exon 2 [<a href="#CR9">9</a>, <a href="#CR32">32</a>]. The presence of two (predicted) functional TIS means, however, that they do function through alternative splicing of exon 1 and that this process could play a role in the regulation of the amount of protein synthesized from the mRNA [<a href="#CR66">66</a>, <a href="#CR67">67</a>]. Such alternative splicing may result in the presence of two AMBN isoforms: one with a short, 16 amino acid long, signal peptide (SP) and the other with a long SP (26 residues), the function of which remains to be tested. The presence of two signal peptides (SP) is quite common in proteins, in which two SP could play a role in the export efficiency of the protein [<a href="#CR93">93</a>]. During amelogenesis, the either use of these SP could regulate AMBN secretion in the forming enamel matrix. For further discussion on this topic we refer the readers to our previous study on ENAM, in which two functional TIS were similarly identified in two exons (exons 3 and 4), resulting in a short and a long SP [<a href="#CR54">54</a>]. The second, short SP of AMBN is homologous to the second SP described in ENAM [<a href="#CR1">1</a>], and sequence similarity in the N-terminal region of these two EMPs allowed to propose that <i>AMBN</i> was derived from an ancestral <i>ENAM</i> after duplication [<a href="#CR4">4</a>]. In fact, <i>AMBN</i> exon 2 and <i>ENAM</i> exon 4 are homologous exons, and both possess 15 nucleotides upstream the TIS. In contrast, <i>AMBN</i> exon 1 and <i>ENAM</i> exon 3, in which is located the first TIS, are not homologous. They were recruited separately during evolution. In <i>ENAM</i> the first TIS was recruited in a mammalian ancestor more than 200 Ma [<a href="#CR94">94</a>], while in <i>AMBN</i> we showed here that it was recruited later, in an eutherian ancestor, more than 100 Ma. In mammals, the convergent recruitment of an additional TIS upstream the ancestral one in two related enamel matrix proteins is, however, intriguing. Indeed, one can wonder whether the second event i.e., the recruitment of a large SP in AMBN, was linked to the presence of two SP in ENAM, and what was the benefit of this feature for enamel microstructure. To our knowledge there are no data in the literature showing to what extend the relative amount of the two EMPs, AMBN and ENAM, is important for enamel microstructure, nor whether the two proteins interact. Here, we speculate that the TIS responsible for encoding a larger SP has appeared at random in <i>AMBN</i> exon 1, then was conserved as providing a better fit with the different regulation of ENAM secretion through the presence of its two SP.</p> </div> <div class="ahotspotfordnaduplicationhasappearedinplacentalambn" title="sec"> <div class="title" tagx="title" title="title">
A hotspot for DNA duplication has appeared in placental AMBN</div> <p>As previously reported in the human <i>AMBN</i>, we identified additional exons of same length and similar sequence in the same region of 12 unrelated eutherian <i>AMBNs</i>. We concluded that these additional exons resulted from tandem duplications of the region containing exon 7 and, therefore, are all homologous. Because the duplication of this short DNA region occurred repeatedly and independently in various lineages, we consider that the <i>AMBN</i> region located downstream exon 7 is a hotspot for the occurrence of such duplication. The process of tandem duplication is known as a major mechanism for gene elongation, but is it advantageous for AMBN function or simply neutral? Repeated duplication of exon 7 was found in terminal taxa, except in simiiform primates, in which the first duplication of this region could have occurred in their common ancestor. The duplicated region was conserved (and some copies were added) in all simiiform gDNA studied excepted in the gorilla genome, in which either the duplicated region was secondarily lost or this <i>AMBN</i> region was badly assembled. Given the presence of these duplicated exons in various taxa and their conservation in simiiform lineages one can wonder whether the region they encode could not have some selective advantage. In addition to enlarge the protein with 13 residues, each exon encodes two to four prolines. Because these duplications do not change much the high percentage of proline in this region, this is the increase in length of this proline-rich region that could be advantageous. Indeed, it has been suggested that the length of the proline-rich region of the EMPs could determine the supramolecular enamel matrix assembly [<a href="#CR95">95</a>].</p> </div> <div class="alternativepolyadenylationsignals(ps)wereconservedthroughmammalianevolution" title="sec"> <div class="title" tagx="title" title="title">
Alternative polyadenylation signals (PS) were conserved through mammalian evolution</div> <p>The 3'UTR of a mRNA is important and anomalies in this region of the mRNAs are implicated in a variety of human diseases [<a href="#CR96">96</a>]. Here, we identified two to five PS in the 3' UTR of the sequences analyzed. Three of them were present in the ancestral mammalian <i>AMBN</i> and conserved in all sequences during mammalian evolution. These variant PS could also be selected for regulatory purposes in the concerned sequences. For instance, it is known that mRNAs with multiple PS tend to use noncanonical signals (i.e., ATTAAA) more often than mRNAs with a single PS [<a href="#CR97">97</a>]. The long lasting conservation of the three PS through 200 Ma means that <i>AMBN</i> codes for three mRNAs that differ in their 3' end and play an important role in post-transcriptional regulation. This finding partially confirms previous reports of two <i>AMBN</i> transcripts in rodents and humans: a short sequence ending after PS1 and a large sequence ending after PS5 [<a href="#CR9">9</a>, <a href="#CR11">11</a>, <a href="#CR13">13</a>]. In rodents, one of these polyadenylation sites shortens the sequence at the 3' end, which could explain the presence of two transcripts. Here we show that a third transcript sequence ending at PS3 could also exist. The presence of multiple PS is significantly observed in mammalian mRNAs, and alternative polyadenylation may also influence the stability, translation efficiency, or localization of an mRNA [<a href="#CR98">98</a>, <a href="#CR99">99</a>]. Indeed, mRNAs vary greatly in stability depending on the length of the 3'UTR, which changes a.o., the number of potential binding sites for microRNAs. However, to our knowledge <i>AMBN</i> mRNAs were not found among targets of several miRNAs [<a href="#CR100">100</a>].</p> </div> <div class="selectionanalyseshighlightedresiduesanddomainsimportantforambnfunction" title="sec"> <div class="title" tagx="title" title="title">
Selection analyses highlighted residues and domains important for AMBN function</div> <p>Our evolutionary analysis revealed that most cleavage sites and most hydroxylated, phosphorylated and O-glycosylated residues either identified or predicted in previous studies are included into the 161 conservative positions that were found here subjected to purifying selection (i.e., functionally constrained) during 200 Ma of mammalian evolution. They are for instance, the hydroxylated P<sup>11</sup>, phosphorylated S<sup>17</sup> (SxE motif) and S<sup>86</sup>, and the cleavage site N<sup>31</sup>. These findings, therefore, confirm that these amino acids are important for the AMBN function and, vice versa, given that these positions were previously identified as probably important they support evolutionary analysis as an efficient method for detecting important residues. Similar studies targeting AMEL or ENAM led to the same findings [<a href="#CR53">53</a>, <a href="#CR54">54</a>]. Interestingly, most of these remarkable residues are adjacent to unchanged amino acids, which indicate that the latter certainly play a role in stabilizing the environment of the crucial residue. Also, our analysis raised some doubts about the potential function of a few positions that were previously predicted as being important in studies of rat and pig AMBN, i.e., some cleavage sites [<a href="#CR5">5</a>, <a href="#CR6">6</a>, <a href="#CR101">101</a>, <a href="#CR102">102</a>], and other hydroxylated and O-glycosylated residues [<a href="#CR29">29</a>, <a href="#CR103">103</a>–<a href="#CR105">105</a>]. However, conservation of specific sequences could not be necessary for the cleavages to occur.</p> <p>The proline-rich and calcium-binding domains do not display a large number of unchanged positions but each possesses some conservative positions that probably play a role in the function of the domain. It is known that such domains do not need high sequence conservation but rather a general context appropriate for their function. Proline-rich sequences are known to bind WW and SH3 domains, two small protein modules that mediate protein-protein interactions and are key parts of signaling cascades [<a href="#CR106">106</a>–<a href="#CR108">108</a>]. The calcium-binding site (i.e., an helix-loop-helix structural domain known as EF-hand motif) is supposed either to be involved in mineral nucleation or in calcium-mediated cell binding [<a href="#CR23">23</a>, <a href="#CR29">29</a>, <a href="#CR30">30</a>]. Some motifs reported in rodent AMBN, such as VTKG [<a href="#CR7">7</a>, <a href="#CR49">49</a>, <a href="#CR109">109</a>] and DGEA [<a href="#CR110">110</a>] are not conserved in mammalian evolution. Given that these motifs are known to interact with cell surface proteins (integrins) they are supposed to serve to link the ameloblasts to the forming enamel matrix. However, their absence in most mammalian lineages suggests that these motifs were only acquired in the rodent lineage ancestor and could have a functional role solely in rodents.</p> <p>Remarkably, conserved positions and the adjacent conserved residues are however few compared to the 150 positions that were highlighted in our evolutionary analysis, the function of which remain to be characterized. Two important motifs were identified by their high number of conserved residues in the region encoded by exons 3 and 4, on the one hand, and by the 3' end of exon 5 and the 5' extremity of exon 6, on the other hand. Both motifs contain important residues; the first one houses hydroxylated P<sup>11</sup>, phosphorylated S<sup>17</sup> (SxE motif) and cleavage site N<sup>31</sup>. It remains to be determined whether the other conserved positions are only important in order to keep these residues functional or have other functions. The second motif contains a remarkable residue, a O-glycosylated S<sup>86</sup>. Here also one can wonder what could be the function of the other conserved positions. In addition, previous studies have revealed that the 15 amino acid sequence encoded by the 5' extremity of exon 6 (including the O-glycosylated S<sup>86</sup>) is subjected to alternative splicing in rodents, human and pig <i>AMBN</i> through the presence of an intraexonic splicing site [<a href="#CR6">6</a>, <a href="#CR11">11</a>, <a href="#CR12">12</a>]. Here we showed that the residue at the splicing site (Q<sup>87</sup> = CAG) is conserved in mammalian evolution, which strongly suggests that the two isoforms resulting from the splicing play distinct but crucial roles for the correct functioning of AMBN. In particular, it is worth noting that (i) the short isoform does not contain this O-glycosylated position, suggesting avoiding interaction with the cell membrane is important to be able to fulfill its function, and (ii) deletion of exon 6 is associated with amelogenesis imperfecta [<a href="#CR48">48</a>] with hypoplastic enamel as previously reported in mice lacking <i>AMBN</i> exons 5 and 6 (see below) [<a href="#CR111">111</a>].</p> </div> <div class="identificationofdisease-associatedmutationspotentiallyleadingtoai" title="sec"> <div class="title" tagx="title" title="title">
Identification of disease-associated mutations potentially leading to AI</div> <p>A total of 161 positions were identified in mammalian AMBN as conservative positions, i.e., residues that were unchanged or only substituted with a residue having the same properties during more than 200 Ma of mammalian evolution. This finding leads to the prediction that the amino acids located at these positions play an important role. They cannot be changed otherwise the AMBN function linked to this position will be disturbed and will result in a genetic disease with apparent enamel disorder. In fact, the process of natural selection provides us with tests in nature, and our previous evolutionary analyses performed on various SCPPs, including AMELX, ENAM, MEPE, AMTN, and DMP1, have shown that the patterns of long-term evolutionary conservation are crucial for validating human genetic diseases related to residue substitutions [<a href="#CR53">53</a>–<a href="#CR57">57</a>]. Such patterns were recently used for <i>in silico</i> functional diagnoses of non-synonymous single nucleotide variants found in thousands of disease-associated genes (see review in [<a href="#CR58">58</a>]). Given these various findings, we predict that <i>AMBN</i> is an excellent candidate for enamel genetic disease, amelogenesis imperfecta (AI). This prediction confirmed the conclusions previously reached by many authors because (i) <i>AMBN</i> was located on chromosome 4 in a region containing the locus for AIH2, the autosomal hypoplastic form of AI [<a href="#CR12">12</a>, <a href="#CR47">47</a>], and (ii) the important function of AMBN during amelogenesis, as illustrated by <i>AMBN</i><sup>−/−</sup> mice that exhibit severe enamel hypoplasia [<a href="#CR49">49</a>].</p> <p>During years, mutations of <i>AMBN</i> were never found associated with AI phenotypes until recently when hypoplastic AI was associated with homozygous exon 6 deletion [<a href="#CR48">48</a>]. The reason why other AMBN-associated AI were not identified could be due to an autosomal recessive pattern of inheritance, and this is supported by the fact that <i>AMBN</i><sup>+/−</sup> mice do not display any dental phenotype [<a href="#CR49">49</a>]. Another reason could be that AMELX compensate, at least partially, for AMBN deficiency. The two encoding genes are phylogenetically related, they share some similarities, notably in the 5' region, and both proteins possess a proline-rich region [<a href="#CR4">4</a>, <a href="#CR112">112</a>]. Therefore, in case of homozygous mutation of AMBN compensation by AMELX could contribute to weaken the dental phenotype, which could be hardly detectable.</p> <p>Deletion of exon 6 leads to the lack of 14 highly conserved and 11 conservative residues, among which the O-glycosylated S<sup>86</sup>, and a large part of the proline-rich region of the protein. Given this number of sensitive positions predicted by our evolutionary analysis, the mutation is validated as being responsible for the AI and this region as playing an important role during enamel matrix formation.</p> </div> <div class="lineage-specificpositiveselectionandrelaxationoffunctionalconstraints" title="sec"> <div class="title" tagx="title" title="title">
Lineage-specific positive selection and relaxation of functional constraints</div> <p>Analyses of lineage-specific selective constraints acting on functional AMBN sequences have revealed a large variation in dN/dS ratio across mammals, even though this gene globally evolved under purifying selection. One of our most prominent results is the strong signal for positive selection detected in catarrhine primates, in which the dN/dS ratio was estimated to be greater than 1. This suggests that episodes of positive selection have occurred throughout the evolutionary history of this sub-clade. Interestingly, this elevated dN/dS ratio correlates with the presence of additional <i>AMBN</i> exons in several of these primate species suggesting that these exon duplications might be adaptive. Apart from Catarrhines, we also found evidence for positive selection in AMBN along the aye-aye (<i>Daubentonia madagascariensis</i>) lineage.</p> <p>In Primates a correlation between enamel thickness and diet has been reported [<a href="#CR113">113</a>] and enamel thickness is often used to infer the diet of both extant and fossil primates [<a href="#CR114">114</a>, <a href="#CR115">115</a>]. Adaptive changes related to diet shifts in Primates have been reported in another enamel protein (ENAM) and it has been hypothesized that differences in tooth enamel thickness were correlated with the adaptive evolution of enamelin [<a href="#CR116">116</a>]. Moreover, the aye-aye differs from other lemurs in possessing rodent-like gnawing and ever-growing incisors, and molars with a particularly thick enamel layer [<a href="#CR115">115</a>]. Our results therefore suggest that adaptive changes might have occurred in AMBN of catarrhines and aye-aye in response to selective constraints imposed by dietary adaptation through changes in enamel thickness.</p> <p>Our analyses also detected traces of relaxed selection in enamel-less species: aardvark, armadillo, and sloth. In adult aardvarks (Afrotheria), armadillos and sloths (Xenarthra), teeth indeed lack enamel. In the nine-banded armadillo (<i>Dasypus novemcinctus</i>), however, at birth the teeth are covered with a thin layer of enamel that is no longer present in adults [<a href="#CR61">61</a>, <a href="#CR117">117</a>]. In birds and in various tooth- or enamel-less mammals, selective pressures on enamel-specific proteins were relaxed after the ability to form enamel was lost, and the genes were inactivated, becoming pseudogenes [<a href="#CR39">39</a>–<a href="#CR41">41</a>, <a href="#CR118">118</a>, <a href="#CR119">119</a>]. As expected from these previous studies, we showed that <i>AMBN</i> has accumulated random deleterious mutations resulting in stop codon and frameshifts in sloth and aardvark, which confirms that AMBN is an enamel-specific protein. In the nine-banded armadillo, the <i>AMBN</i> sequence has not drastically changed, a condition that could be expected when considering the presence of a thin enamel cover in the young, which strongly suggests that AMBN could be expressed during enamel formation prior to birth. However, we identified several substitutions at AMBN positions that were considered important for the function of the protein. Such mutations could lead to the expression of a defective protein resulting in the deposition of a thin (hypoplastic?) enamel layer at the tooth surface, which subsequently disappears after birth through abrasion. Further structural and molecular studies of enamel layer formation in this armadillo species are however needed in order to confirm or infirm this hypothesis.</p> </div> </div> <div class="conclusion" title="conclusion"> <div class="title" tagx="title" title="title">
Conclusions</div> <p>By adding 50 new sequences to the six previously published sequences, our study improved considerably our knowledge on the gene structure, protein composition and evolution of AMBN in mammals. In particular, we revealed that (i) an additional TIS was recruited in an ancestral eutherian <i>AMBN</i>, allowing the translation of a second, alternative, large SP, (ii) a short DNA sequence including exon 7 was duplicated several times in various eutherian species, increasing the proline-rich region, (iii) several PS were functional, suggesting a regulation process in the 3'UTR, (iv) numerous residues were conserved during more than 200 Ma of mammalian evolution, which strongly suggests that they are structurally and/or functionally important for the correct function of the protein, (v) several conserved residues constituted new domains of predicted high importance, and (vi) <i>AMBN</i> was invalidated in enamel-less species as previously reported [<a href="#CR40">40</a>, <a href="#CR41">41</a>]. The putative function of some residues identified in previous studies was not confirmed by our analyses, and cannot be generalized to all mammals. Finally, the presence of highly conserved residues indicates that <i>AMBN</i> is a good candidate gene for amelogenesis imperfecta.</p> </div> </div> <div class="back" title="back"> <div class="app-group" title="app-group"> <div class="app" title="app"> <div class="additionalfiles" title="sec"> <div class="title" tagx="title" title="title">
Additional files</div> <p> </p><div class="media" title="media"><a href="12862_2015_431_MOESM1_ESM.pdf">LINK</a><span class="label" tagx="label" title="label">Additional file 1:</span> <p><b>Names of the 56 mammalian species used in this study.</b> Preferred common names, scientific names, families, orders and references in GenBank are listed in alphabetical order of common names. The <i>AMBN</i> sequence of the crocodilian <i>Caiman crocodilus</i> was used as outgroup. (PDF 95 kb)</p>  </div> <div class="media" title="media"><a href="12862_2015_431_MOESM2_ESM.pdf">LINK</a><span class="label" tagx="label" title="label">Additional file 2:</span> <p><b>Amino acid alignment of the 53 mammalian AMBN used in our evolutionary analysis.</b> The sequences were aligned against the human sequence and are ordered following mammalian relationships. Our alignment led to 500 positions, including gaps. For convenience of presentation our alignment does not include the duplicated exons 9c, 9d and 9e (+9c; +9c-9e) that were found in three species only. The specific length of each AMBN in indicated in brackets at the end of the sequence. The signal peptide (underlined) can start at the methionine located either in the region encoded by exon 1 or exon 2 (M, squared). For a better understanding of SP evolution, the sequences in which the first methionine is lacking are highlighted on grey background. Exon limits are indicated by vertical lines; (.): residue identical to human AMBN residue; (−): indel; (?): unknown amino acid; *: stop codon. (PDF 138 kb)</p>  </div> <div class="media" title="media"><a href="12862_2015_431_MOESM3_ESM.pdf">LINK</a><span class="label" tagx="label" title="label">Additional file 3:</span> <p><b>Comparison of intron/exon boundaries and UTR of twelve mammalian</b><b><i>AMBN</i></b><b>sequences.</b> For scientific names and references, see Additional file 1. (//): sequence not shown; (?): unknown nucleotide; (.): nucleotide identical to human <i>AMBN</i> nucleotide; (−): indel. (PDF 143 kb)</p>  </div> <div class="media" title="media"><a href="12862_2015_431_MOESM4_ESM.tif">LINK</a><span class="label" tagx="label" title="label">Additional file 4:</span> <p><b>Prediction of functional signal peptides (SP) in the AMBN sequence using</b><b><i>Signal P</i></b><b>software.</b> Here the human sequence was used as an example. The high probability for the two sequences (score = 0.998) indicates that the SP encoded either by exon 1 + exon 2 (26 residues) (<b>A</b>) or by exon 2 only (16 residues) (<b>B</b>) are functional. The cleavage site (arrow) does not vary in the two sequences analyzed and exhibits a similar, high probability: 0.971 in <b>A</b>, between positions 26 and 27; 0.965 in <b>B</b>, between positions 16 and 17. The three characteristic regions of SP (n-, h- and c-regions) are indicated. (TIFF 488 kb)</p>  </div> <div class="media" title="media"><a href="12862_2015_431_MOESM5_ESM.pdf">LINK</a><span class="label" tagx="label" title="label">Additional file 5:</span> <p><b>Amino acid composition of AMBN.</b> The amino acid percentages are given for the full length sequence and the proline-rich region for six representative mammalian AMBN, the putative ancestral mammalian sequence, and the crocodile, and are compared to average frequency in most proteins. The percentages of proline and glutamine are in bold characters. (PDF 100 kb)</p>  </div> <p /> </div> </div> </div> <div class="fn-group" title="fn-group"> <div class="fn-type-" title=""> <p><b>Competing interests</b></p> <p>The authors declare that they have no competing interests.</p> </div> <div class="fn-type-" title=""> <p><b>Authors’ contributions</b></p> <p>FD performed the evolutionary analyses. BG carried out the molecular biology experiments. JYS performed in silico data recovery and sequence alignment. FD and JYS wrote the manuscript.</p> </div> </div> <div class="ack" title="ack"> <div class="title" tagx="title" title="title">
Acknowledgements</div> <p>We are grateful to Géraldine Véron (Muséum national d’Histoire Naturelle, Paris) and François Catzeflis (Institut des Sciences de l’Evolution de Montpellier) who kindly provided ethanol-preserved tissues from some species. We thank sequencing centers for making available assembled genome sequences. The study was financially supported by grants from the Centre National de la Recherche Scientifique (CNRS), the Scientific Council of the Université de Montpellier, and the Université Pierre &amp;amp; Marie Curie, Paris (UMR 7138). Computational resources were provided by the Montpellier Bioinformatics Biodiversity platform of the Labex CeMEB. This is contribution ISEM 2015-148 of the Institut des Sciences de l’Evolution de Montpellier.</p> </div> <div class="references">
References</div> <div tag="ref-list"> <ul> <div class="title" tagx="title" title="title">
References</div> <li tag="ref"><a name="CR1" /><span class="label" tagx="label" title="label">1.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kawasaki</span><span class="given-names" tagx="given-names" title="given-names">K</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Weiss</span><span class="given-names" tagx="given-names" title="given-names">KM</span></span></span><span class="mixed-article-title" title="mixed-article-title">Mineralized tissue and vertebrate evolution: the secretory calcium-binding phosphoprotein gene cluster</span><span class="source" tagx="source" title="source">Proc Natl Acad Sci U S A</span><span class="year" tagx="year" title="year">2003</span><span class="volume" tagx="volume" title="volume">100</span><span class="fpage" tagx="fpage" title="fpage">4060</span><span class="lpage" tagx="lpage" title="lpage">5</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12646701">12646701</a></span></span></li> <li tag="ref"><a name="CR2" /><span class="label" tagx="label" title="label">2.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kawasaki</span><span class="given-names" tagx="given-names" title="given-names">K</span></span></span><span class="mixed-article-title" title="mixed-article-title">The SCPP gene repertoire in bony vertebrates and graded differences in mineralized tissues</span><span class="source" tagx="source" title="source">Dev Genes Evol</span><span class="year" tagx="year" title="year">2009</span><span class="volume" tagx="volume" title="volume">219</span><span class="fpage" tagx="fpage" title="fpage">147</span><span class="lpage" tagx="lpage" title="lpage">57</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19255778">19255778</a></span></span></li> <li tag="ref"><a name="CR3" /><span class="label" tagx="label" title="label">3.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Sire</span><span class="given-names" tagx="given-names" title="given-names">J-Y</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Delgado</span><span class="given-names" tagx="given-names" title="given-names">S</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Girondot</span><span class="given-names" tagx="given-names" title="given-names">M</span></span></span><span class="mixed-article-title" title="mixed-article-title">The amelogenin story: origin and evolution</span><span class="source" tagx="source" title="source">Eur J Oral Sci</span><span class="year" tagx="year" title="year">2006</span><span class="volume" tagx="volume" title="volume">114 Suppl 1</span><span class="fpage" tagx="fpage" title="fpage">64</span><span class="lpage" tagx="lpage" title="lpage">77</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16674665">16674665</a></span></span></li> <li tag="ref"><a name="CR4" /><span class="label" tagx="label" title="label">4.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Sire</span><span class="given-names" tagx="given-names" title="given-names">J-Y</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Davit-Béal</span><span class="given-names" tagx="given-names" title="given-names">T</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Delgado</span><span class="given-names" tagx="given-names" title="given-names">S</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gu</span><span class="given-names" tagx="given-names" title="given-names">X</span></span></span><span class="mixed-article-title" title="mixed-article-title">The origin and evolution of enamel mineralization genes</span><span class="source" tagx="source" title="source">Cells Tissues Organs</span><span class="year" tagx="year" title="year">2007</span><span class="volume" tagx="volume" title="volume">186</span><span class="fpage" tagx="fpage" title="fpage">25</span><span class="lpage" tagx="lpage" title="lpage">48</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17627117">17627117</a></span></span></li> <li tag="ref"><a name="CR5" /><span class="label" tagx="label" title="label">5.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Uchida</span><span class="given-names" tagx="given-names" title="given-names">T</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Fukae</span><span class="given-names" tagx="given-names" title="given-names">M</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Tanabe</span><span class="given-names" tagx="given-names" title="given-names">T</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Yamakoshi</span><span class="given-names" tagx="given-names" title="given-names">Y</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Satoda</span><span class="given-names" tagx="given-names" title="given-names">T</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Murakami</span><span class="given-names" tagx="given-names" title="given-names">C</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Immunochemical and immunocytochemical study of a 15 kDa non-amelogenin and related proteins in the porcine immature enamel: proposal of a new group of enamel proteins "sheath proteins"</span><span class="source" tagx="source" title="source">Biomed Res</span><span class="year" tagx="year" title="year">1995</span><span class="volume" tagx="volume" title="volume">16</span><span class="fpage" tagx="fpage" title="fpage">131</span><span class="lpage" tagx="lpage" title="lpage">40</span></span></li> <li tag="ref"><a name="CR6" /><span class="label" tagx="label" title="label">6.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hu</span><span class="given-names" tagx="given-names" title="given-names">CC</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Fukae</span><span class="given-names" tagx="given-names" title="given-names">M</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Uchida</span><span class="given-names" tagx="given-names" title="given-names">T</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Qian</span><span class="given-names" tagx="given-names" title="given-names">Q</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zhang</span><span class="given-names" tagx="given-names" title="given-names">CH</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ryu</span><span class="given-names" tagx="given-names" title="given-names">OH</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Sheathlin: cloning, cDNA/polypeptide sequences, and immunolocalization of porcine enamel sheath proteins</span><span class="source" tagx="source" title="source">J Dent Res</span><span class="year" tagx="year" title="year">1997</span><span class="volume" tagx="volume" title="volume">76</span><span class="fpage" tagx="fpage" title="fpage">648</span><span class="lpage" tagx="lpage" title="lpage">57</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/9062558">9062558</a></span></span></li> <li tag="ref"><a name="CR7" /><span class="label" tagx="label" title="label">7.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Černý</span><span class="given-names" tagx="given-names" title="given-names">R</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Slaby</span><span class="given-names" tagx="given-names" title="given-names">I</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hammarström</span><span class="given-names" tagx="given-names" title="given-names">L</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wurtz</span><span class="given-names" tagx="given-names" title="given-names">T</span></span></span><span class="mixed-article-title" title="mixed-article-title">A novel gene expressed in rat ameloblasts codes for proteins with cell binding domains</span><span class="source" tagx="source" title="source">J Bone Miner Res</span><span class="year" tagx="year" title="year">1996</span><span class="volume" tagx="volume" title="volume">11</span><span class="fpage" tagx="fpage" title="fpage">883</span><span class="lpage" tagx="lpage" title="lpage">91</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/8797107">8797107</a></span></span></li> <li tag="ref"><a name="CR8" /><span class="label" tagx="label" title="label">8.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Termine</span><span class="given-names" tagx="given-names" title="given-names">JD</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Belcourt</span><span class="given-names" tagx="given-names" title="given-names">AB</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Christner</span><span class="given-names" tagx="given-names" title="given-names">PJ</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Conn</span><span class="given-names" tagx="given-names" title="given-names">KM</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Nylen</span><span class="given-names" tagx="given-names" title="given-names">MU</span></span></span><span class="mixed-article-title" title="mixed-article-title">Properties of dissociatively extracted fetal tooth matrix proteins. I. Principal molecular species in developing bovine enamel</span><span class="source" tagx="source" title="source">J Biol Chem</span><span class="year" tagx="year" title="year">1980</span><span class="volume" tagx="volume" title="volume">255</span><span class="fpage" tagx="fpage" title="fpage">9760</span><span class="lpage" tagx="lpage" title="lpage">8</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/7430099">7430099</a></span></span></li> <li tag="ref"><a name="CR9" /><span class="label" tagx="label" title="label">9.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Krebsbach</span><span class="given-names" tagx="given-names" title="given-names">PH</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lee</span><span class="given-names" tagx="given-names" title="given-names">SK</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Matsuki</span><span class="given-names" tagx="given-names" title="given-names">Y</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kozak</span><span class="given-names" tagx="given-names" title="given-names">CA</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Yamada</span><span class="given-names" tagx="given-names" title="given-names">KM</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Yamada</span><span class="given-names" tagx="given-names" title="given-names">Y</span></span></span><span class="mixed-article-title" title="mixed-article-title">Full-length sequence, localization, and chromosomal mapping of ameloblastin a novel tooth-specific gene</span><span class="source" tagx="source" title="source">J Biol Chem</span><span class="year" tagx="year" title="year">1996</span><span class="volume" tagx="volume" title="volume">271</span><span class="fpage" tagx="fpage" title="fpage">4431</span><span class="lpage" tagx="lpage" title="lpage">5</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/8626794">8626794</a></span></span></li> <li tag="ref"><a name="CR10" /><span class="label" tagx="label" title="label">10.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Fong</span><span class="given-names" tagx="given-names" title="given-names">CD</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Cerný</span><span class="given-names" tagx="given-names" title="given-names">R</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hammarström</span><span class="given-names" tagx="given-names" title="given-names">L</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Slaby</span><span class="given-names" tagx="given-names" title="given-names">I</span></span></span><span class="mixed-article-title" title="mixed-article-title">Sequential expression of an amelin gene in mesenchymal and epithelial cells during odontogenesis in rats</span><span class="source" tagx="source" title="source">Eur J Oral Sci</span><span class="year" tagx="year" title="year">1998</span><span class="volume" tagx="volume" title="volume">106</span><span class="issue" tagx="issue" title="issue">Suppl 1</span><span class="fpage" tagx="fpage" title="fpage">324</span><span class="lpage" tagx="lpage" title="lpage">30</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/9541243">9541243</a></span></span></li> <li tag="ref"><a name="CR11" /><span class="label" tagx="label" title="label">11.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Simmons</span><span class="given-names" tagx="given-names" title="given-names">D</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gu</span><span class="given-names" tagx="given-names" title="given-names">TT</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Krebsbach</span><span class="given-names" tagx="given-names" title="given-names">PH</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Yamada</span><span class="given-names" tagx="given-names" title="given-names">Y</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">MacDougall</span><span class="given-names" tagx="given-names" title="given-names">M</span></span></span><span class="mixed-article-title" title="mixed-article-title">Identification and characterization of a cDNA for mouse ameloblastin</span><span class="source" tagx="source" title="source">Connect Tissue Res</span><span class="year" tagx="year" title="year">1998</span><span class="volume" tagx="volume" title="volume">39</span><span class="fpage" tagx="fpage" title="fpage">3</span><span class="lpage" tagx="lpage" title="lpage">12</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11062984">11062984</a></span></span></li> <li tag="ref"><a name="CR12" /><span class="label" tagx="label" title="label">12.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">MacDougall</span><span class="given-names" tagx="given-names" title="given-names">M</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Simmons</span><span class="given-names" tagx="given-names" title="given-names">D</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gu</span><span class="given-names" tagx="given-names" title="given-names">TT</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Forsman-Semb</span><span class="given-names" tagx="given-names" title="given-names">K</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kärrman Mårdh</span><span class="given-names" tagx="given-names" title="given-names">C</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Mesbah</span><span class="given-names" tagx="given-names" title="given-names">M</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Cloning, characterization and immunolocalization of human ameloblastin</span><span class="source" tagx="source" title="source">Eur J Oral Sci</span><span class="year" tagx="year" title="year">2000</span><span class="volume" tagx="volume" title="volume">108</span><span class="fpage" tagx="fpage" title="fpage">303</span><span class="lpage" tagx="lpage" title="lpage">10</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/10946765">10946765</a></span></span></li> <li tag="ref"><a name="CR13" /><span class="label" tagx="label" title="label">13.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Toyosawa</span><span class="given-names" tagx="given-names" title="given-names">S</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Fujiwara</span><span class="given-names" tagx="given-names" title="given-names">T</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ooshima</span><span class="given-names" tagx="given-names" title="given-names">T</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Shintani</span><span class="given-names" tagx="given-names" title="given-names">S</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Sato</span><span class="given-names" tagx="given-names" title="given-names">A</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ogawa</span><span class="given-names" tagx="given-names" title="given-names">Y</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Cloning and characterization of the human ameloblastin gene</span><span class="source" tagx="source" title="source">Gene</span><span class="year" tagx="year" title="year">2000</span><span class="volume" tagx="volume" title="volume">256</span><span class="fpage" tagx="fpage" title="fpage">1</span><span class="lpage" tagx="lpage" title="lpage">11</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11054529">11054529</a></span></span></li> <li tag="ref"><a name="CR14" /><span class="label" tagx="label" title="label">14.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Shintani</span><span class="given-names" tagx="given-names" title="given-names">S</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kobata</span><span class="given-names" tagx="given-names" title="given-names">M</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Toyosawa</span><span class="given-names" tagx="given-names" title="given-names">S</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Fujiwara</span><span class="given-names" tagx="given-names" title="given-names">T</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Sato</span><span class="given-names" tagx="given-names" title="given-names">A</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ooshima</span><span class="given-names" tagx="given-names" title="given-names">T</span></span></span><span class="mixed-article-title" title="mixed-article-title">Identification and characterization of ameloblastin gene in a reptile</span><span class="source" tagx="source" title="source">Gene</span><span class="year" tagx="year" title="year">2002</span><span class="volume" tagx="volume" title="volume">283</span><span class="fpage" tagx="fpage" title="fpage">245</span><span class="lpage" tagx="lpage" title="lpage">54</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11867231">11867231</a></span></span></li> <li tag="ref"><a name="CR15" /><span class="label" tagx="label" title="label">15.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Shintani</span><span class="given-names" tagx="given-names" title="given-names">S</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kobata</span><span class="given-names" tagx="given-names" title="given-names">M</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Toyosawa</span><span class="given-names" tagx="given-names" title="given-names">S</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ooshima</span><span class="given-names" tagx="given-names" title="given-names">T</span></span></span><span class="mixed-article-title" title="mixed-article-title">Identification and characterization of ameloblastin gene in an amphibian, <i>Xenopus laevis</i></span><span class="source" tagx="source" title="source">Gene</span><span class="year" tagx="year" title="year">2003</span><span class="volume" tagx="volume" title="volume">318</span><span class="fpage" tagx="fpage" title="fpage">125</span><span class="lpage" tagx="lpage" title="lpage">36</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/14585505">14585505</a></span></span></li> <li tag="ref"><a name="CR16" /><span class="label" tagx="label" title="label">16.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kawasaki</span><span class="given-names" tagx="given-names" title="given-names">K</span></span></span><span class="mixed-article-title" title="mixed-article-title">The SCPP gene family and the complexity of hard tissues in vertebrates</span><span class="source" tagx="source" title="source">Cells Tissues Organs</span><span class="year" tagx="year" title="year">2011</span><span class="volume" tagx="volume" title="volume">194</span><span class="fpage" tagx="fpage" title="fpage">108</span><span class="lpage" tagx="lpage" title="lpage">12</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21576905">21576905</a></span></span></li> <li tag="ref"><a name="CR17" /><span class="label" tagx="label" title="label">17.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kawasaki</span><span class="given-names" tagx="given-names" title="given-names">K</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Amemiya</span><span class="given-names" tagx="given-names" title="given-names">CT</span></span></span><span class="mixed-article-title" title="mixed-article-title">SCPP genes in the coelacanth: tissue mineralization genes shared by sarcopterygians</span><span class="source" tagx="source" title="source">J Exp Zoolog B Mol Dev Evol</span><span class="year" tagx="year" title="year">2014</span><span class="volume" tagx="volume" title="volume">322</span><span class="fpage" tagx="fpage" title="fpage">390</span><span class="lpage" tagx="lpage" title="lpage">402</span></span></li> <li tag="ref"><a name="CR18" /><span class="label" tagx="label" title="label">18.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hedges</span><span class="given-names" tagx="given-names" title="given-names">SB</span></span></span><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hedges</span><span class="given-names" tagx="given-names" title="given-names">SB</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kumar</span><span class="given-names" tagx="given-names" title="given-names">S</span></span></span><span class="mixed-article-title" title="mixed-article-title">Vertebrates (Vertebrata)</span><span class="source" tagx="source" title="source">The TimeTree of Life</span><span class="year" tagx="year" title="year">2009</span><span class="publisher-loc" tagx="publisher-loc" title="publisher-loc">New York</span><span class="publisher-name" tagx="publisher-name" title="publisher-name">Oxford University Press</span><span class="fpage" tagx="fpage" title="fpage">309</span><span class="lpage" tagx="lpage" title="lpage">14</span></span></li> <li tag="ref"><a name="CR19" /><span class="label" tagx="label" title="label">19.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Nanci</span><span class="given-names" tagx="given-names" title="given-names">A</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zalzal</span><span class="given-names" tagx="given-names" title="given-names">S</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lavoie</span><span class="given-names" tagx="given-names" title="given-names">P</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kunikata</span><span class="given-names" tagx="given-names" title="given-names">M</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Chen</span><span class="given-names" tagx="given-names" title="given-names">W-Y</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Krebsbach</span><span class="given-names" tagx="given-names" title="given-names">PH</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Comparative immunochemical analyses of the developmental expression and distribution of ameloblastin and amelogenin in rat incisors</span><span class="source" tagx="source" title="source">J Histochem Cytochem</span><span class="year" tagx="year" title="year">1998</span><span class="volume" tagx="volume" title="volume">46</span><span class="fpage" tagx="fpage" title="fpage">911</span><span class="lpage" tagx="lpage" title="lpage">34</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/9671442">9671442</a></span></span></li> <li tag="ref"><a name="CR20" /><span class="label" tagx="label" title="label">20.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Iwata</span><span class="given-names" tagx="given-names" title="given-names">T</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Yamakoshi</span><span class="given-names" tagx="given-names" title="given-names">Y</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hu</span><span class="given-names" tagx="given-names" title="given-names">JC-C</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ishikawa</span><span class="given-names" tagx="given-names" title="given-names">I</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bartlett</span><span class="given-names" tagx="given-names" title="given-names">JD</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Krebsbach</span><span class="given-names" tagx="given-names" title="given-names">PH</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Processing of ameloblastin by MMP-20</span><span class="source" tagx="source" title="source">J Dent Res</span><span class="year" tagx="year" title="year">2007</span><span class="volume" tagx="volume" title="volume">86</span><span class="fpage" tagx="fpage" title="fpage">153</span><span class="lpage" tagx="lpage" title="lpage">7</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17251515">17251515</a></span></span></li> <li tag="ref"><a name="CR21" /><span class="label" tagx="label" title="label">21.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Fukumoto</span><span class="given-names" tagx="given-names" title="given-names">S</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kiba</span><span class="given-names" tagx="given-names" title="given-names">T</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hall</span><span class="given-names" tagx="given-names" title="given-names">B</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Iehara</span><span class="given-names" tagx="given-names" title="given-names">N</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Nakamura</span><span class="given-names" tagx="given-names" title="given-names">T</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Longenecker</span><span class="given-names" tagx="given-names" title="given-names">G</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Ameloblastin is a cell adhesion molecule required for maintaining the differentiation state of ameloblasts</span><span class="source" tagx="source" title="source">J Cell Biol</span><span class="year" tagx="year" title="year">2004</span><span class="volume" tagx="volume" title="volume">167</span><span class="fpage" tagx="fpage" title="fpage">973</span><span class="lpage" tagx="lpage" title="lpage">83</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15583034">15583034</a></span></span></li> <li tag="ref"><a name="CR22" /><span class="label" tagx="label" title="label">22.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Sonoda</span><span class="given-names" tagx="given-names" title="given-names">A</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Iwamoto</span><span class="given-names" tagx="given-names" title="given-names">T</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Nakamura</span><span class="given-names" tagx="given-names" title="given-names">T</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Fukumoto</span><span class="given-names" tagx="given-names" title="given-names">E</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Yoshizaki</span><span class="given-names" tagx="given-names" title="given-names">K</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Yamada</span><span class="given-names" tagx="given-names" title="given-names">A</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Critical role of heparin binding domains of ameloblastin for dental epithelium cell adhesion and ameloblastoma proliferation</span><span class="source" tagx="source" title="source">J Biol Chem</span><span class="year" tagx="year" title="year">2009</span><span class="volume" tagx="volume" title="volume">284</span><span class="fpage" tagx="fpage" title="fpage">27176</span><span class="lpage" tagx="lpage" title="lpage">84</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19648121">19648121</a></span></span></li> <li tag="ref"><a name="CR23" /><span class="label" tagx="label" title="label">23.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zhang</span><span class="given-names" tagx="given-names" title="given-names">Y</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zhang</span><span class="given-names" tagx="given-names" title="given-names">X</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lu</span><span class="given-names" tagx="given-names" title="given-names">X</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Atsawasuwan</span><span class="given-names" tagx="given-names" title="given-names">P</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Luan</span><span class="given-names" tagx="given-names" title="given-names">X</span></span></span><span class="mixed-article-title" title="mixed-article-title">Ameloblastin regulates cell attachment and proliferation through RhoA and p27</span><span class="source" tagx="source" title="source">Eur J Oral Sci</span><span class="year" tagx="year" title="year">2011</span><span class="volume" tagx="volume" title="volume">119</span><span class="fpage" tagx="fpage" title="fpage">280</span><span class="lpage" tagx="lpage" title="lpage">5</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22243257">22243257</a></span></span></li> <li tag="ref"><a name="CR24" /><span class="label" tagx="label" title="label">24.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zeichner-David</span><span class="given-names" tagx="given-names" title="given-names">M</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Chen</span><span class="given-names" tagx="given-names" title="given-names">L-S</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hsu</span><span class="given-names" tagx="given-names" title="given-names">Z</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Reyna</span><span class="given-names" tagx="given-names" title="given-names">J</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Caton</span><span class="given-names" tagx="given-names" title="given-names">J</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bringas</span><span class="given-names" tagx="given-names" title="given-names">P</span></span></span><span class="mixed-article-title" title="mixed-article-title">Amelogenin and ameloblastin show growth-factor like activity in periodontal ligament cells</span><span class="source" tagx="source" title="source">Eur J Oral Sci</span><span class="year" tagx="year" title="year">2006</span><span class="volume" tagx="volume" title="volume">114</span><span class="fpage" tagx="fpage" title="fpage">244</span><span class="lpage" tagx="lpage" title="lpage">53</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16674693">16674693</a></span></span></li> <li tag="ref"><a name="CR25" /><span class="label" tagx="label" title="label">25.</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation">Bartlett JD, Ganss B, Goldberg M, Moradian-Oldak J, Paine ML, Snead ML, et al. Protein–protein interactions of the developing enamel matrix. In: Schatten GP editor. Current Topics in Developmental Biology. Volume 74. Academic Press, Waltham; 2006:57–115.</span></li> <li tag="ref"><a name="CR26" /><span class="label" tagx="label" title="label">26.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Nakamura</span><span class="given-names" tagx="given-names" title="given-names">Y</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Slaby</span><span class="given-names" tagx="given-names" title="given-names">I</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Spahr</span><span class="given-names" tagx="given-names" title="given-names">A</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Pezeshki</span><span class="given-names" tagx="given-names" title="given-names">G</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Matsumoto</span><span class="given-names" tagx="given-names" title="given-names">K</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lyngstadaas</span><span class="given-names" tagx="given-names" title="given-names">SP</span></span></span><span class="mixed-article-title" title="mixed-article-title">Ameloblastin fusion protein enhances pulpal healing and dentin formation in porcine teeth</span><span class="source" tagx="source" title="source">Calcif Tissue Int</span><span class="year" tagx="year" title="year">2006</span><span class="volume" tagx="volume" title="volume">78</span><span class="fpage" tagx="fpage" title="fpage">278</span><span class="lpage" tagx="lpage" title="lpage">84</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16691493">16691493</a></span></span></li> <li tag="ref"><a name="CR27" /><span class="label" tagx="label" title="label">27.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Iizuka</span><span class="given-names" tagx="given-names" title="given-names">S</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kudo</span><span class="given-names" tagx="given-names" title="given-names">Y</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Yoshida</span><span class="given-names" tagx="given-names" title="given-names">M</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Tsunematsu</span><span class="given-names" tagx="given-names" title="given-names">T</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Yoshiko</span><span class="given-names" tagx="given-names" title="given-names">Y</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Uchida</span><span class="given-names" tagx="given-names" title="given-names">T</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Ameloblastin regulates osteogenic differentiation by inhibiting Src kinase via cross talk between Integrin β1 and CD63</span><span class="source" tagx="source" title="source">Mol Cell Biol</span><span class="year" tagx="year" title="year">2011</span><span class="volume" tagx="volume" title="volume">31</span><span class="fpage" tagx="fpage" title="fpage">783</span><span class="lpage" tagx="lpage" title="lpage">92</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21149578">21149578</a></span></span></li> <li tag="ref"><a name="CR28" /><span class="label" tagx="label" title="label">28.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Spahr</span><span class="given-names" tagx="given-names" title="given-names">A</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lyngstadaas</span><span class="given-names" tagx="given-names" title="given-names">SP</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Slaby</span><span class="given-names" tagx="given-names" title="given-names">I</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Haller</span><span class="given-names" tagx="given-names" title="given-names">B</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Boeckh</span><span class="given-names" tagx="given-names" title="given-names">C</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Tsoulfidou</span><span class="given-names" tagx="given-names" title="given-names">F</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Expression of amelin and trauma-induced dentin formation</span><span class="source" tagx="source" title="source">Clin Oral Investig</span><span class="year" tagx="year" title="year">2002</span><span class="volume" tagx="volume" title="volume">6</span><span class="fpage" tagx="fpage" title="fpage">51</span><span class="lpage" tagx="lpage" title="lpage">7</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11996163">11996163</a></span></span></li> <li tag="ref"><a name="CR29" /><span class="label" tagx="label" title="label">29.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Yamakoshi</span><span class="given-names" tagx="given-names" title="given-names">Y</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Tanabe</span><span class="given-names" tagx="given-names" title="given-names">T</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Oida</span><span class="given-names" tagx="given-names" title="given-names">S</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hu</span><span class="given-names" tagx="given-names" title="given-names">C-C</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Simmer</span><span class="given-names" tagx="given-names" title="given-names">JP</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Fukae</span><span class="given-names" tagx="given-names" title="given-names">M</span></span></span><span class="mixed-article-title" title="mixed-article-title">Calcium binding of enamel proteins and their derivatives with emphasis on the calcium-binding domain of porcine sheathlin</span><span class="source" tagx="source" title="source">Arch Oral Biol</span><span class="year" tagx="year" title="year">2001</span><span class="volume" tagx="volume" title="volume">46</span><span class="fpage" tagx="fpage" title="fpage">1005</span><span class="lpage" tagx="lpage" title="lpage">14</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11543707">11543707</a></span></span></li> <li tag="ref"><a name="CR30" /><span class="label" tagx="label" title="label">30.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Vymětal</span><span class="given-names" tagx="given-names" title="given-names">J</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Slabý</span><span class="given-names" tagx="given-names" title="given-names">I</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Spahr</span><span class="given-names" tagx="given-names" title="given-names">A</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Vondrášek</span><span class="given-names" tagx="given-names" title="given-names">J</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lyngstadaas</span><span class="given-names" tagx="given-names" title="given-names">SP</span></span></span><span class="mixed-article-title" title="mixed-article-title">Bioinformatic analysis and molecular modelling of human ameloblastin suggest a two-domain intrinsically unstructured calcium-binding protein</span><span class="source" tagx="source" title="source">Eur J Oral Sci</span><span class="year" tagx="year" title="year">2008</span><span class="volume" tagx="volume" title="volume">116</span><span class="fpage" tagx="fpage" title="fpage">124</span><span class="lpage" tagx="lpage" title="lpage">34</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18353005">18353005</a></span></span></li> <li tag="ref"><a name="CR31" /><span class="label" tagx="label" title="label">31.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zhang</span><span class="given-names" tagx="given-names" title="given-names">X</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Diekwisch</span><span class="given-names" tagx="given-names" title="given-names">TGH</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Luan</span><span class="given-names" tagx="given-names" title="given-names">X</span></span></span><span class="mixed-article-title" title="mixed-article-title">Structure and function of ameloblastin as an extracellular matrix protein: adhesion, calcium binding, and CD63 interaction in human and mouse</span><span class="source" tagx="source" title="source">Eur J Oral Sci</span><span class="year" tagx="year" title="year">2011</span><span class="volume" tagx="volume" title="volume">119</span><span class="fpage" tagx="fpage" title="fpage">270</span><span class="lpage" tagx="lpage" title="lpage">9</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22243256">22243256</a></span></span></li> <li tag="ref"><a name="CR32" /><span class="label" tagx="label" title="label">32.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Dhamija</span><span class="given-names" tagx="given-names" title="given-names">S</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Liu</span><span class="given-names" tagx="given-names" title="given-names">Y</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Yamada</span><span class="given-names" tagx="given-names" title="given-names">Y</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Snead</span><span class="given-names" tagx="given-names" title="given-names">ML</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Krebsbach</span><span class="given-names" tagx="given-names" title="given-names">PH</span></span></span><span class="mixed-article-title" title="mixed-article-title">Cloning and characterization of the murine ameloblastin promoter</span><span class="source" tagx="source" title="source">J Biol Chem</span><span class="year" tagx="year" title="year">1999</span><span class="volume" tagx="volume" title="volume">274</span><span class="fpage" tagx="fpage" title="fpage">20738</span><span class="lpage" tagx="lpage" title="lpage">43</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/10400709">10400709</a></span></span></li> <li tag="ref"><a name="CR33" /><span class="label" tagx="label" title="label">33.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Tamburstuen</span><span class="given-names" tagx="given-names" title="given-names">MV</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Snead</span><span class="given-names" tagx="given-names" title="given-names">ML</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Reseland</span><span class="given-names" tagx="given-names" title="given-names">JE</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Paine</span><span class="given-names" tagx="given-names" title="given-names">ML</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lyngstadaas</span><span class="given-names" tagx="given-names" title="given-names">SP</span></span></span><span class="mixed-article-title" title="mixed-article-title">Ameloblastin upstream region contains structural elements regulating transcriptional activity in a stromal cell line derived from bone marrow</span><span class="source" tagx="source" title="source">Eur J Oral Sci</span><span class="year" tagx="year" title="year">2011</span><span class="volume" tagx="volume" title="volume">119</span><span class="fpage" tagx="fpage" title="fpage">286</span><span class="lpage" tagx="lpage" title="lpage">92</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22243258">22243258</a></span></span></li> <li tag="ref"><a name="CR34" /><span class="label" tagx="label" title="label">34.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Dhamija</span><span class="given-names" tagx="given-names" title="given-names">S</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Krebsbach</span><span class="given-names" tagx="given-names" title="given-names">PH</span></span></span><span class="mixed-article-title" title="mixed-article-title">Role of Cbfa1 in ameloblastin gene transcription</span><span class="source" tagx="source" title="source">J Biol Chem</span><span class="year" tagx="year" title="year">2001</span><span class="volume" tagx="volume" title="volume">276</span><span class="fpage" tagx="fpage" title="fpage">35159</span><span class="lpage" tagx="lpage" title="lpage">64</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11451945">11451945</a></span></span></li> <li tag="ref"><a name="CR35" /><span class="label" tagx="label" title="label">35.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Spahr</span><span class="given-names" tagx="given-names" title="given-names">A</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lyngstadaas</span><span class="given-names" tagx="given-names" title="given-names">SP</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Slaby</span><span class="given-names" tagx="given-names" title="given-names">I</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Pezeshki</span><span class="given-names" tagx="given-names" title="given-names">G</span></span></span><span class="mixed-article-title" title="mixed-article-title">Ameloblastin expression during craniofacial bone formation in rats</span><span class="source" tagx="source" title="source">Eur J Oral Sci</span><span class="year" tagx="year" title="year">2006</span><span class="volume" tagx="volume" title="volume">114</span><span class="fpage" tagx="fpage" title="fpage">504</span><span class="lpage" tagx="lpage" title="lpage">11</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17184233">17184233</a></span></span></li> <li tag="ref"><a name="CR36" /><span class="label" tagx="label" title="label">36.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Tamburstuen</span><span class="given-names" tagx="given-names" title="given-names">MV</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Reseland</span><span class="given-names" tagx="given-names" title="given-names">JE</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Spahr</span><span class="given-names" tagx="given-names" title="given-names">A</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Brookes</span><span class="given-names" tagx="given-names" title="given-names">SJ</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kvalheim</span><span class="given-names" tagx="given-names" title="given-names">G</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Slaby</span><span class="given-names" tagx="given-names" title="given-names">I</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Ameloblastin expression and putative autoregulation in mesenchymal cells suggest a role in early bone formation and repair</span><span class="source" tagx="source" title="source">Bone</span><span class="year" tagx="year" title="year">2011</span><span class="volume" tagx="volume" title="volume">48</span><span class="fpage" tagx="fpage" title="fpage">406</span><span class="lpage" tagx="lpage" title="lpage">13</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20854943">20854943</a></span></span></li> <li tag="ref"><a name="CR37" /><span class="label" tagx="label" title="label">37.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Tamburstuen</span><span class="given-names" tagx="given-names" title="given-names">MV</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Reppe</span><span class="given-names" tagx="given-names" title="given-names">S</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Spahr</span><span class="given-names" tagx="given-names" title="given-names">A</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Sabetrasekh</span><span class="given-names" tagx="given-names" title="given-names">R</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kvalheim</span><span class="given-names" tagx="given-names" title="given-names">G</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Slaby</span><span class="given-names" tagx="given-names" title="given-names">I</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Ameloblastin promotes bone growth by enhancing proliferation of progenitor cells and by stimulating immunoregulators</span><span class="source" tagx="source" title="source">Eur J Oral Sci</span><span class="year" tagx="year" title="year">2010</span><span class="volume" tagx="volume" title="volume">118</span><span class="fpage" tagx="fpage" title="fpage">451</span><span class="lpage" tagx="lpage" title="lpage">9</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20831578">20831578</a></span></span></li> <li tag="ref"><a name="CR38" /><span class="label" tagx="label" title="label">38.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kuroda</span><span class="given-names" tagx="given-names" title="given-names">S</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wazen</span><span class="given-names" tagx="given-names" title="given-names">R</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Sellin</span><span class="given-names" tagx="given-names" title="given-names">K</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Tanaka</span><span class="given-names" tagx="given-names" title="given-names">E</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Moffatt</span><span class="given-names" tagx="given-names" title="given-names">P</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Nanci</span><span class="given-names" tagx="given-names" title="given-names">A</span></span></span><span class="mixed-article-title" title="mixed-article-title">Ameloblastin is not implicated in bone remodelling and repair</span><span class="source" tagx="source" title="source">Eur Cell Mater</span><span class="year" tagx="year" title="year">2011</span><span class="volume" tagx="volume" title="volume">22</span><span class="fpage" tagx="fpage" title="fpage">56</span><span class="lpage" tagx="lpage" title="lpage">66</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21761392">21761392</a></span></span></li> <li tag="ref"><a name="CR39" /><span class="label" tagx="label" title="label">39.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Sire</span><span class="given-names" tagx="given-names" title="given-names">J-Y</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Delgado</span><span class="given-names" tagx="given-names" title="given-names">SC</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Girondot</span><span class="given-names" tagx="given-names" title="given-names">M</span></span></span><span class="mixed-article-title" title="mixed-article-title">Hen’s teeth with enamel cap: from dream to impossibility</span><span class="source" tagx="source" title="source">BMC Evol Biol</span><span class="year" tagx="year" title="year">2008</span><span class="volume" tagx="volume" title="volume">8</span><span class="fpage" tagx="fpage" title="fpage">246</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18775069">18775069</a></span></span></li> <li tag="ref"><a name="CR40" /><span class="label" tagx="label" title="label">40.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Deméré</span><span class="given-names" tagx="given-names" title="given-names">TA</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">McGowen</span><span class="given-names" tagx="given-names" title="given-names">MR</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Berta</span><span class="given-names" tagx="given-names" title="given-names">A</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gatesy</span><span class="given-names" tagx="given-names" title="given-names">J</span></span></span><span class="mixed-article-title" title="mixed-article-title">Morphological and molecular evidence for a stepwise evolutionary transition from teeth to baleen in mysticete whales</span><span class="source" tagx="source" title="source">Syst Biol</span><span class="year" tagx="year" title="year">2008</span><span class="volume" tagx="volume" title="volume">57</span><span class="fpage" tagx="fpage" title="fpage">15</span><span class="lpage" tagx="lpage" title="lpage">37</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18266181">18266181</a></span></span></li> <li tag="ref"><a name="CR41" /><span class="label" tagx="label" title="label">41.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Meredith</span><span class="given-names" tagx="given-names" title="given-names">RW</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gatesy</span><span class="given-names" tagx="given-names" title="given-names">J</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Springer</span><span class="given-names" tagx="given-names" title="given-names">MS</span></span></span><span class="mixed-article-title" title="mixed-article-title">Molecular decay of enamel matrix protein genes in turtles and other edentulous amniotes</span><span class="source" tagx="source" title="source">BMC Evol Biol</span><span class="year" tagx="year" title="year">2013</span><span class="volume" tagx="volume" title="volume">13</span><span class="fpage" tagx="fpage" title="fpage">20</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23342979">23342979</a></span></span></li> <li tag="ref"><a name="CR42" /><span class="label" tagx="label" title="label">42.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kärrman Mårdh</span><span class="given-names" tagx="given-names" title="given-names">C</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bäckman</span><span class="given-names" tagx="given-names" title="given-names">B</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Simmons</span><span class="given-names" tagx="given-names" title="given-names">D</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Golovleva</span><span class="given-names" tagx="given-names" title="given-names">I</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gu</span><span class="given-names" tagx="given-names" title="given-names">TT</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Holmgren</span><span class="given-names" tagx="given-names" title="given-names">G</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Human ameloblastin gene: genomic organization and mutation analysis in amelogenesis imperfecta patients</span><span class="source" tagx="source" title="source">Eur J Oral Sci</span><span class="year" tagx="year" title="year">2001</span><span class="volume" tagx="volume" title="volume">109</span><span class="fpage" tagx="fpage" title="fpage">8</span><span class="lpage" tagx="lpage" title="lpage">13</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11330937">11330937</a></span></span></li> <li tag="ref"><a name="CR43" /><span class="label" tagx="label" title="label">43.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hart</span><span class="given-names" tagx="given-names" title="given-names">PS</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wright</span><span class="given-names" tagx="given-names" title="given-names">JT</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Savage</span><span class="given-names" tagx="given-names" title="given-names">M</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kang</span><span class="given-names" tagx="given-names" title="given-names">G</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bensen</span><span class="given-names" tagx="given-names" title="given-names">JT</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gorry</span><span class="given-names" tagx="given-names" title="given-names">MC</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Exclusion of candidate genes in two families with autosomal dominant hypocalcified amelogenesis imperfecta</span><span class="source" tagx="source" title="source">Eur J Oral Sci</span><span class="year" tagx="year" title="year">2003</span><span class="volume" tagx="volume" title="volume">111</span><span class="fpage" tagx="fpage" title="fpage">326</span><span class="lpage" tagx="lpage" title="lpage">31</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12887398">12887398</a></span></span></li> <li tag="ref"><a name="CR44" /><span class="label" tagx="label" title="label">44.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Santos</span><span class="given-names" tagx="given-names" title="given-names">MC</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hart</span><span class="given-names" tagx="given-names" title="given-names">PS</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ramaswami</span><span class="given-names" tagx="given-names" title="given-names">M</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kanno</span><span class="given-names" tagx="given-names" title="given-names">CM</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hart</span><span class="given-names" tagx="given-names" title="given-names">TC</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Line</span><span class="given-names" tagx="given-names" title="given-names">SR</span></span></span><span class="mixed-article-title" title="mixed-article-title">Exclusion of known gene for enamel development in two Brazilian families with amelogenesis imperfecta</span><span class="source" tagx="source" title="source">Head Face Med</span><span class="year" tagx="year" title="year">2007</span><span class="volume" tagx="volume" title="volume">3</span><span class="fpage" tagx="fpage" title="fpage">8</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17266769">17266769</a></span></span></li> <li tag="ref"><a name="CR45" /><span class="label" tagx="label" title="label">45.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Chan</span><span class="given-names" tagx="given-names" title="given-names">H-C</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Estrella</span><span class="given-names" tagx="given-names" title="given-names">NMRP</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Milkovich</span><span class="given-names" tagx="given-names" title="given-names">RN</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kim</span><span class="given-names" tagx="given-names" title="given-names">J-W</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Simmer</span><span class="given-names" tagx="given-names" title="given-names">JP</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hu</span><span class="given-names" tagx="given-names" title="given-names">JC-C</span></span></span><span class="mixed-article-title" title="mixed-article-title">Target gene analyses of 39 amelogenesis imperfecta kindreds</span><span class="source" tagx="source" title="source">Eur J Oral Sci</span><span class="year" tagx="year" title="year">2011</span><span class="volume" tagx="volume" title="volume">119</span><span class="fpage" tagx="fpage" title="fpage">311</span><span class="lpage" tagx="lpage" title="lpage">23</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22243262">22243262</a></span></span></li> <li tag="ref"><a name="CR46" /><span class="label" tagx="label" title="label">46.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">MacDougall</span><span class="given-names" tagx="given-names" title="given-names">M</span></span></span><span class="mixed-article-title" title="mixed-article-title">Dental structural diseases mapping to human chromosome 4q21</span><span class="source" tagx="source" title="source">Connect Tissue Res</span><span class="year" tagx="year" title="year">2003</span><span class="volume" tagx="volume" title="volume">44</span><span class="issue" tagx="issue" title="issue">Suppl 1</span><span class="fpage" tagx="fpage" title="fpage">285</span><span class="lpage" tagx="lpage" title="lpage">91</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12952210">12952210</a></span></span></li> <li tag="ref"><a name="CR47" /><span class="label" tagx="label" title="label">47.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">MacDougall</span><span class="given-names" tagx="given-names" title="given-names">M</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">DuPont</span><span class="given-names" tagx="given-names" title="given-names">BR</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Simmons</span><span class="given-names" tagx="given-names" title="given-names">D</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Reus</span><span class="given-names" tagx="given-names" title="given-names">B</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Krebsbach</span><span class="given-names" tagx="given-names" title="given-names">P</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kärrman</span><span class="given-names" tagx="given-names" title="given-names">C</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Ameloblastin gene (<i>AMBN</i>) maps within the critical region for autosomal dominant amelogenesis imperfecta at chromosome 4q21</span><span class="source" tagx="source" title="source">Genomics</span><span class="year" tagx="year" title="year">1997</span><span class="volume" tagx="volume" title="volume">41</span><span class="fpage" tagx="fpage" title="fpage">115</span><span class="lpage" tagx="lpage" title="lpage">8</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/9126491">9126491</a></span></span></li> <li tag="ref"><a name="CR48" /><span class="label" tagx="label" title="label">48.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Poulter</span><span class="given-names" tagx="given-names" title="given-names">JA</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Murillo</span><span class="given-names" tagx="given-names" title="given-names">G</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Brookes</span><span class="given-names" tagx="given-names" title="given-names">SJ</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Smith</span><span class="given-names" tagx="given-names" title="given-names">CEL</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Parry</span><span class="given-names" tagx="given-names" title="given-names">DA</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Silva</span><span class="given-names" tagx="given-names" title="given-names">S</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Deletion of ameloblastin exon 6 is associated with amelogenesis imperfecta</span><span class="source" tagx="source" title="source">Hum Mol Genet</span><span class="year" tagx="year" title="year">2014</span><span class="volume" tagx="volume" title="volume">23</span><span class="fpage" tagx="fpage" title="fpage">5317</span><span class="lpage" tagx="lpage" title="lpage">24</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24858907">24858907</a></span></span></li> <li tag="ref"><a name="CR49" /><span class="label" tagx="label" title="label">49.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Fukumoto</span><span class="given-names" tagx="given-names" title="given-names">S</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Yamada</span><span class="given-names" tagx="given-names" title="given-names">A</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Nonaka</span><span class="given-names" tagx="given-names" title="given-names">K</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Yamada</span><span class="given-names" tagx="given-names" title="given-names">Y</span></span></span><span class="mixed-article-title" title="mixed-article-title">Essential roles of ameloblastin in maintaining ameloblast differentiation and enamel formation</span><span class="source" tagx="source" title="source">Cells Tissues Organs</span><span class="year" tagx="year" title="year">2005</span><span class="volume" tagx="volume" title="volume">181</span><span class="fpage" tagx="fpage" title="fpage">189</span><span class="lpage" tagx="lpage" title="lpage">95</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16612084">16612084</a></span></span></li> <li tag="ref"><a name="CR50" /><span class="label" tagx="label" title="label">50.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Perdigão</span><span class="given-names" tagx="given-names" title="given-names">PF</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gomez</span><span class="given-names" tagx="given-names" title="given-names">RS</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Pimenta</span><span class="given-names" tagx="given-names" title="given-names">FJGS</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">De Marco</span><span class="given-names" tagx="given-names" title="given-names">L</span></span></span><span class="mixed-article-title" title="mixed-article-title">Ameloblastin gene (<i>AMBN</i>) mutations associated with epithelial odontogenic tumors</span><span class="source" tagx="source" title="source">Oral Oncol</span><span class="year" tagx="year" title="year">2004</span><span class="volume" tagx="volume" title="volume">40</span><span class="fpage" tagx="fpage" title="fpage">841</span><span class="lpage" tagx="lpage" title="lpage">6</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15288841">15288841</a></span></span></li> <li tag="ref"><a name="CR51" /><span class="label" tagx="label" title="label">51.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Perdigão</span><span class="given-names" tagx="given-names" title="given-names">PF</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Carvalho</span><span class="given-names" tagx="given-names" title="given-names">VM</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Marco</span><span class="given-names" tagx="given-names" title="given-names">LD</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gomez</span><span class="given-names" tagx="given-names" title="given-names">RS</span></span></span><span class="mixed-article-title" title="mixed-article-title">Mutation of ameloblastin gene in calcifying epithelial odontogenic tumor</span><span class="source" tagx="source" title="source">Anticancer Res</span><span class="year" tagx="year" title="year">2009</span><span class="volume" tagx="volume" title="volume">29</span><span class="fpage" tagx="fpage" title="fpage">3065</span><span class="lpage" tagx="lpage" title="lpage">7</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19661317">19661317</a></span></span></li> <li tag="ref"><a name="CR52" /><span class="label" tagx="label" title="label">52.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hirayama</span><span class="given-names" tagx="given-names" title="given-names">K</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Miyasho</span><span class="given-names" tagx="given-names" title="given-names">T</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ohmachi</span><span class="given-names" tagx="given-names" title="given-names">T</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Watanabe</span><span class="given-names" tagx="given-names" title="given-names">T</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Yokota</span><span class="given-names" tagx="given-names" title="given-names">H</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Taniyama</span><span class="given-names" tagx="given-names" title="given-names">H</span></span></span><span class="mixed-article-title" title="mixed-article-title">Biochemical and immunohistochemical characterization of the amyloid in canine amyloid-producing odontogenic tumor</span><span class="source" tagx="source" title="source">Vet Pathol Online</span><span class="year" tagx="year" title="year">2010</span><span class="volume" tagx="volume" title="volume">47</span><span class="fpage" tagx="fpage" title="fpage">915</span><span class="lpage" tagx="lpage" title="lpage">22</span></span></li> <li tag="ref"><a name="CR53" /><span class="label" tagx="label" title="label">53.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Delgado</span><span class="given-names" tagx="given-names" title="given-names">S</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Girondot</span><span class="given-names" tagx="given-names" title="given-names">M</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Sire</span><span class="given-names" tagx="given-names" title="given-names">J-Y</span></span></span><span class="mixed-article-title" title="mixed-article-title">Molecular evolution of amelogenin in mammals</span><span class="source" tagx="source" title="source">J Mol Evol</span><span class="year" tagx="year" title="year">2005</span><span class="volume" tagx="volume" title="volume">60</span><span class="fpage" tagx="fpage" title="fpage">12</span><span class="lpage" tagx="lpage" title="lpage">30</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15696365">15696365</a></span></span></li> <li tag="ref"><a name="CR54" /><span class="label" tagx="label" title="label">54.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Al-Hashimi</span><span class="given-names" tagx="given-names" title="given-names">N</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Sire</span><span class="given-names" tagx="given-names" title="given-names">J-Y</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Delgado</span><span class="given-names" tagx="given-names" title="given-names">S</span></span></span><span class="mixed-article-title" title="mixed-article-title">Evolutionary analysis of mammalian enamelin, the largest enamel protein, supports a crucial role for the 32-kDa peptide and reveals selective adaptation in rodents and primates</span><span class="source" tagx="source" title="source">J Mol Evol</span><span class="year" tagx="year" title="year">2009</span><span class="volume" tagx="volume" title="volume">69</span><span class="fpage" tagx="fpage" title="fpage">635</span><span class="lpage" tagx="lpage" title="lpage">56</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20012271">20012271</a></span></span></li> <li tag="ref"><a name="CR55" /><span class="label" tagx="label" title="label">55.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bardet</span><span class="given-names" tagx="given-names" title="given-names">C</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Delgado</span><span class="given-names" tagx="given-names" title="given-names">S</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Sire</span><span class="given-names" tagx="given-names" title="given-names">J-Y</span></span></span><span class="mixed-article-title" title="mixed-article-title">MEPE evolution in mammals reveals regions and residues of prime functional importance</span><span class="source" tagx="source" title="source">Cell Mol Life Sci</span><span class="year" tagx="year" title="year">2010</span><span class="volume" tagx="volume" title="volume">67</span><span class="fpage" tagx="fpage" title="fpage">305</span><span class="lpage" tagx="lpage" title="lpage">20</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19924383">19924383</a></span></span></li> <li tag="ref"><a name="CR56" /><span class="label" tagx="label" title="label">56.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gasse</span><span class="given-names" tagx="given-names" title="given-names">B</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Silvent</span><span class="given-names" tagx="given-names" title="given-names">J</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Sire</span><span class="given-names" tagx="given-names" title="given-names">J-Y</span></span></span><span class="mixed-article-title" title="mixed-article-title">Evolutionary analysis suggests that AMTN is enamel-specific and a candidate for AI</span><span class="source" tagx="source" title="source">J Dent Res</span><span class="year" tagx="year" title="year">2012</span><span class="volume" tagx="volume" title="volume">91</span><span class="fpage" tagx="fpage" title="fpage">1085</span><span class="lpage" tagx="lpage" title="lpage">9</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22968158">22968158</a></span></span></li> <li tag="ref"><a name="CR57" /><span class="label" tagx="label" title="label">57.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Silvent</span><span class="given-names" tagx="given-names" title="given-names">J</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Sire</span><span class="given-names" tagx="given-names" title="given-names">J-Y</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Delgado</span><span class="given-names" tagx="given-names" title="given-names">S</span></span></span><span class="mixed-article-title" title="mixed-article-title">The dentin matrix acidic phosphoprotein 1 (DMP1) in the light of mammalian evolution</span><span class="source" tagx="source" title="source">J Mol Evol</span><span class="year" tagx="year" title="year">2013</span><span class="volume" tagx="volume" title="volume">76</span><span class="fpage" tagx="fpage" title="fpage">59</span><span class="lpage" tagx="lpage" title="lpage">70</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23361408">23361408</a></span></span></li> <li tag="ref"><a name="CR58" /><span class="label" tagx="label" title="label">58.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kumar</span><span class="given-names" tagx="given-names" title="given-names">S</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Dudley</span><span class="given-names" tagx="given-names" title="given-names">JT</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Filipski</span><span class="given-names" tagx="given-names" title="given-names">A</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Liu</span><span class="given-names" tagx="given-names" title="given-names">L</span></span></span><span class="mixed-article-title" title="mixed-article-title">Phylomedicine: an evolutionary telescope to explore and diagnose the universe of disease mutations</span><span class="source" tagx="source" title="source">Trends Genet</span><span class="year" tagx="year" title="year">2011</span><span class="volume" tagx="volume" title="volume">27</span><span class="fpage" tagx="fpage" title="fpage">377</span><span class="lpage" tagx="lpage" title="lpage">86</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21764165">21764165</a></span></span></li> <li tag="ref"><a name="CR59" /><span class="label" tagx="label" title="label">59.</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation">National Center for Biotechnology Information. [<a href="http://www.ncbi.nlm.nih.gov/">http://www.ncbi.nlm.nih.gov</a>]</span></li> <li tag="ref"><a name="CR60" /><span class="label" tagx="label" title="label">60.</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation">Ensembl. [<a href="http://www.ensembl.org">http://www.ensembl.org</a>]</span></li> <li tag="ref"><a name="CR61" /><span class="label" tagx="label" title="label">61.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Davit-Béal</span><span class="given-names" tagx="given-names" title="given-names">T</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Tucker</span><span class="given-names" tagx="given-names" title="given-names">AS</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Sire</span><span class="given-names" tagx="given-names" title="given-names">J-Y</span></span></span><span class="mixed-article-title" title="mixed-article-title">Loss of teeth and enamel in tetrapods: fossil record, genetic data and morphological adaptations</span><span class="source" tagx="source" title="source">J Anat</span><span class="year" tagx="year" title="year">2009</span><span class="volume" tagx="volume" title="volume">214</span><span class="fpage" tagx="fpage" title="fpage">477</span><span class="lpage" tagx="lpage" title="lpage">501</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19422426">19422426</a></span></span></li> <li tag="ref"><a name="CR62" /><span class="label" tagx="label" title="label">62.</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation">Higgins DG, Thompson JD, Gibson TJ. Using CLUSTAL for multiple sequence alignments. In: Doolittle RF, editor. Methods in Enzymology. Volume 266. Academic Press, Waltham; 1996:383–402. [Computer Methods for Macromolecular Sequence Analysis]</span></li> <li tag="ref"><a name="CR63" /><span class="label" tagx="label" title="label">63.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Rambaut</span><span class="given-names" tagx="given-names" title="given-names">A</span></span></span><span class="source" tagx="source" title="source">Se-Al: Sequence Alignement Editor</span><span class="year" tagx="year" title="year">1996</span><span class="publisher-loc" tagx="publisher-loc" title="publisher-loc">Oxford</span><span class="publisher-name" tagx="publisher-name" title="publisher-name">Department of Zoology. University of Oxford</span></span></li> <li tag="ref"><a name="CR64" /><span class="label" tagx="label" title="label">64.</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation">Se-Al software. [<a href="http://tree.bio.ed.ac.uk/software/seal/">http://tree.bio.ed.ac.uk/software/seal/</a>]</span></li> <li tag="ref"><a name="CR65" /><span class="label" tagx="label" title="label">65.</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation">DNA Functional Site Miner. [<a href="http://dnafsminer.bic.nus.edu.sg">http://dnafsminer.bic.nus.edu.sg</a>]</span></li> <li tag="ref"><a name="CR66" /><span class="label" tagx="label" title="label">66.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kozak</span><span class="given-names" tagx="given-names" title="given-names">M</span></span></span><span class="mixed-article-title" title="mixed-article-title">Compilation and analysis of sequences upstream from the translational start site in eukaryotic mRNAs</span><span class="source" tagx="source" title="source">Nucleic Acids Res</span><span class="year" tagx="year" title="year">1984</span><span class="volume" tagx="volume" title="volume">12</span><span class="fpage" tagx="fpage" title="fpage">857</span><span class="lpage" tagx="lpage" title="lpage">72</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/6694911">6694911</a></span></span></li> <li tag="ref"><a name="CR67" /><span class="label" tagx="label" title="label">67.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kozak</span><span class="given-names" tagx="given-names" title="given-names">M</span></span></span><span class="mixed-article-title" title="mixed-article-title">Point mutations define a sequence flanking the AUG initiator codon that modulates translation by eukaryotic ribosomes</span><span class="source" tagx="source" title="source">Cell</span><span class="year" tagx="year" title="year">1986</span><span class="volume" tagx="volume" title="volume">44</span><span class="fpage" tagx="fpage" title="fpage">283</span><span class="lpage" tagx="lpage" title="lpage">92</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/3943125">3943125</a></span></span></li> <li tag="ref"><a name="CR68" /><span class="label" tagx="label" title="label">68.</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation">SignalP Server. [<a href="http://www.cbs.dtu.dk/services/SignalP">www.cbs.dtu.dk/services/SignalP</a>]</span></li> <li tag="ref"><a name="CR69" /><span class="label" tagx="label" title="label">69.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Dyrløv Bendtsen</span><span class="given-names" tagx="given-names" title="given-names">J</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Nielsen</span><span class="given-names" tagx="given-names" title="given-names">H</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">von Heijne</span><span class="given-names" tagx="given-names" title="given-names">G</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Brunak</span><span class="given-names" tagx="given-names" title="given-names">S</span></span></span><span class="mixed-article-title" title="mixed-article-title">Improved prediction of signal peptides: SignalP 3.0</span><span class="source" tagx="source" title="source">J Mol Biol</span><span class="year" tagx="year" title="year">2004</span><span class="volume" tagx="volume" title="volume">340</span><span class="fpage" tagx="fpage" title="fpage">783</span><span class="lpage" tagx="lpage" title="lpage">95</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15223320">15223320</a></span></span></li> <li tag="ref"><a name="CR70" /><span class="label" tagx="label" title="label">70.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Katoh</span><span class="given-names" tagx="given-names" title="given-names">K</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Misawa</span><span class="given-names" tagx="given-names" title="given-names">K</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kuma</span><span class="given-names" tagx="given-names" title="given-names">K</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Miyata</span><span class="given-names" tagx="given-names" title="given-names">T</span></span></span><span class="mixed-article-title" title="mixed-article-title">MAFFT: a novel method for rapid multiple sequence alignment based on fast Fourier transform</span><span class="source" tagx="source" title="source">Nucleic Acids Res</span><span class="year" tagx="year" title="year">2002</span><span class="volume" tagx="volume" title="volume">30</span><span class="fpage" tagx="fpage" title="fpage">3059</span><span class="lpage" tagx="lpage" title="lpage">66</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12136088">12136088</a></span></span></li> <li tag="ref"><a name="CR71" /><span class="label" tagx="label" title="label">71.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kearse</span><span class="given-names" tagx="given-names" title="given-names">M</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Moir</span><span class="given-names" tagx="given-names" title="given-names">R</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wilson</span><span class="given-names" tagx="given-names" title="given-names">A</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Stones-Havas</span><span class="given-names" tagx="given-names" title="given-names">S</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Cheung</span><span class="given-names" tagx="given-names" title="given-names">M</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Sturrock</span><span class="given-names" tagx="given-names" title="given-names">S</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Geneious Basic: an integrated and extendable desktop software platform for the organization and analysis of sequence data</span><span class="source" tagx="source" title="source">Bioinformatics</span><span class="year" tagx="year" title="year">2012</span><span class="volume" tagx="volume" title="volume">28</span><span class="fpage" tagx="fpage" title="fpage">1647</span><span class="lpage" tagx="lpage" title="lpage">9</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22543367">22543367</a></span></span></li> <li tag="ref"><a name="CR72" /><span class="label" tagx="label" title="label">72.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ranwez</span><span class="given-names" tagx="given-names" title="given-names">V</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Harispe</span><span class="given-names" tagx="given-names" title="given-names">S</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Delsuc</span><span class="given-names" tagx="given-names" title="given-names">F</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Douzery</span><span class="given-names" tagx="given-names" title="given-names">EJP</span></span></span><span class="mixed-article-title" title="mixed-article-title">MACSE: Multiple Alignment of Coding SEquences accounting for frameshifts and stop codons</span><span class="source" tagx="source" title="source">PLoS ONE</span><span class="year" tagx="year" title="year">2011</span><span class="volume" tagx="volume" title="volume">6</span><span class="fpage" tagx="fpage" title="fpage">e22594</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21949676">21949676</a></span></span></li> <li tag="ref"><a name="CR73" /><span class="label" tagx="label" title="label">73.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Castresana</span><span class="given-names" tagx="given-names" title="given-names">J</span></span></span><span class="mixed-article-title" title="mixed-article-title">Selection of conserved blocks from multiple alignments for their use in phylogenetic analysis</span><span class="source" tagx="source" title="source">Mol Biol Evol</span><span class="year" tagx="year" title="year">2000</span><span class="volume" tagx="volume" title="volume">17</span><span class="fpage" tagx="fpage" title="fpage">540</span><span class="lpage" tagx="lpage" title="lpage">52</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/10742046">10742046</a></span></span></li> <li tag="ref"><a name="CR74" /><span class="label" tagx="label" title="label">74.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Stamatakis</span><span class="given-names" tagx="given-names" title="given-names">A</span></span></span><span class="mixed-article-title" title="mixed-article-title">RAxML-VI-HPC: maximum likelihood-based phylogenetic analyses with thousands of taxa and mixed models</span><span class="source" tagx="source" title="source">Bioinformatics</span><span class="year" tagx="year" title="year">2006</span><span class="volume" tagx="volume" title="volume">22</span><span class="fpage" tagx="fpage" title="fpage">2688</span><span class="lpage" tagx="lpage" title="lpage">90</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16928733">16928733</a></span></span></li> <li tag="ref"><a name="CR75" /><span class="label" tagx="label" title="label">75.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Yang</span><span class="given-names" tagx="given-names" title="given-names">Z</span></span></span><span class="mixed-article-title" title="mixed-article-title">PAML 4: phylogenetic analysis by maximum likelihood</span><span class="source" tagx="source" title="source">Mol Biol Evol</span><span class="year" tagx="year" title="year">2007</span><span class="volume" tagx="volume" title="volume">24</span><span class="fpage" tagx="fpage" title="fpage">1586</span><span class="lpage" tagx="lpage" title="lpage">91</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17483113">17483113</a></span></span></li> <li tag="ref"><a name="CR76" /><span class="label" tagx="label" title="label">76.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Meredith</span><span class="given-names" tagx="given-names" title="given-names">RW</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Janečka</span><span class="given-names" tagx="given-names" title="given-names">JE</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gatesy</span><span class="given-names" tagx="given-names" title="given-names">J</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ryder</span><span class="given-names" tagx="given-names" title="given-names">OA</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Fisher</span><span class="given-names" tagx="given-names" title="given-names">CA</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Teeling</span><span class="given-names" tagx="given-names" title="given-names">EC</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Impacts of the Cretaceous terrestrial revolution and KPg extinction on mammal diversification</span><span class="source" tagx="source" title="source">Science</span><span class="year" tagx="year" title="year">2011</span><span class="volume" tagx="volume" title="volume">334</span><span class="fpage" tagx="fpage" title="fpage">521</span><span class="lpage" tagx="lpage" title="lpage">4</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21940861">21940861</a></span></span></li> <li tag="ref"><a name="CR77" /><span class="label" tagx="label" title="label">77.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Romiguier</span><span class="given-names" tagx="given-names" title="given-names">J</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ranwez</span><span class="given-names" tagx="given-names" title="given-names">V</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Delsuc</span><span class="given-names" tagx="given-names" title="given-names">F</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Galtier</span><span class="given-names" tagx="given-names" title="given-names">N</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Douzery</span><span class="given-names" tagx="given-names" title="given-names">EJP</span></span></span><span class="mixed-article-title" title="mixed-article-title">Less is more in mammalian phylogenomics: AT-rich genes minimize tree conflicts and unravel the root of placental mammals</span><span class="source" tagx="source" title="source">Mol Biol Evol</span><span class="year" tagx="year" title="year">2013</span><span class="volume" tagx="volume" title="volume">30</span><span class="fpage" tagx="fpage" title="fpage">2134</span><span class="lpage" tagx="lpage" title="lpage">44</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23813978">23813978</a></span></span></li> <li tag="ref"><a name="CR78" /><span class="label" tagx="label" title="label">78.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Nielsen</span><span class="given-names" tagx="given-names" title="given-names">R</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Yang</span><span class="given-names" tagx="given-names" title="given-names">Z</span></span></span><span class="mixed-article-title" title="mixed-article-title">Likelihood models for detecting positively selected amino acid sites and applications to the HIV-1 envelope gene</span><span class="source" tagx="source" title="source">Genetics</span><span class="year" tagx="year" title="year">1998</span><span class="volume" tagx="volume" title="volume">148</span><span class="fpage" tagx="fpage" title="fpage">929</span><span class="lpage" tagx="lpage" title="lpage">36</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/9539414">9539414</a></span></span></li> <li tag="ref"><a name="CR79" /><span class="label" tagx="label" title="label">79.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Yang</span><span class="given-names" tagx="given-names" title="given-names">Z</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Nielsen</span><span class="given-names" tagx="given-names" title="given-names">R</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Goldman</span><span class="given-names" tagx="given-names" title="given-names">N</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Pedersen</span><span class="given-names" tagx="given-names" title="given-names">A-MK</span></span></span><span class="mixed-article-title" title="mixed-article-title">Codon-substitution models for heterogeneous selection pressure at amino acid sites</span><span class="source" tagx="source" title="source">Genetics</span><span class="year" tagx="year" title="year">2000</span><span class="volume" tagx="volume" title="volume">155</span><span class="fpage" tagx="fpage" title="fpage">431</span><span class="lpage" tagx="lpage" title="lpage">49</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/10790415">10790415</a></span></span></li> <li tag="ref"><a name="CR80" /><span class="label" tagx="label" title="label">80.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Yang</span><span class="given-names" tagx="given-names" title="given-names">Z</span></span></span><span class="mixed-article-title" title="mixed-article-title">Likelihood ratio tests for detecting positive selection and application to primate lysozyme evolution</span><span class="source" tagx="source" title="source">Mol Biol Evol</span><span class="year" tagx="year" title="year">1998</span><span class="volume" tagx="volume" title="volume">15</span><span class="fpage" tagx="fpage" title="fpage">568</span><span class="lpage" tagx="lpage" title="lpage">73</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/9580986">9580986</a></span></span></li> <li tag="ref"><a name="CR81" /><span class="label" tagx="label" title="label">81.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Burk-Herrick</span><span class="given-names" tagx="given-names" title="given-names">A</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Scally</span><span class="given-names" tagx="given-names" title="given-names">M</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Amrine-Madsen</span><span class="given-names" tagx="given-names" title="given-names">H</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Stanhope</span><span class="given-names" tagx="given-names" title="given-names">MJ</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Springer</span><span class="given-names" tagx="given-names" title="given-names">MS</span></span></span><span class="mixed-article-title" title="mixed-article-title">Natural selection and mammalian BRCA1 sequences: elucidating functionally important sites relevant to breast cancer susceptibility in humans</span><span class="source" tagx="source" title="source">Mamm Genome</span><span class="year" tagx="year" title="year">2006</span><span class="volume" tagx="volume" title="volume">17</span><span class="fpage" tagx="fpage" title="fpage">257</span><span class="lpage" tagx="lpage" title="lpage">70</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16518693">16518693</a></span></span></li> <li tag="ref"><a name="CR82" /><span class="label" tagx="label" title="label">82.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kirwan</span><span class="given-names" tagx="given-names" title="given-names">JD</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bekaert</span><span class="given-names" tagx="given-names" title="given-names">M</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Commins</span><span class="given-names" tagx="given-names" title="given-names">JM</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Davies</span><span class="given-names" tagx="given-names" title="given-names">KTJ</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Rossiter</span><span class="given-names" tagx="given-names" title="given-names">SJ</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Teeling</span><span class="given-names" tagx="given-names" title="given-names">EC</span></span></span><span class="mixed-article-title" title="mixed-article-title">A phylomedicine approach to understanding the evolution of auditory sensory perception and disease in mammals</span><span class="source" tagx="source" title="source">Evol Appl</span><span class="year" tagx="year" title="year">2013</span><span class="volume" tagx="volume" title="volume">6</span><span class="fpage" tagx="fpage" title="fpage">412</span><span class="lpage" tagx="lpage" title="lpage">22</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23745134">23745134</a></span></span></li> <li tag="ref"><a name="CR83" /><span class="label" tagx="label" title="label">83.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lartillot</span><span class="given-names" tagx="given-names" title="given-names">N</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Delsuc</span><span class="given-names" tagx="given-names" title="given-names">F</span></span></span><span class="mixed-article-title" title="mixed-article-title">Joint reconstruction of divergence times and life-history evolution in placental mammals using a phylogenetic covariance model</span><span class="source" tagx="source" title="source">Evolution</span><span class="year" tagx="year" title="year">2012</span><span class="volume" tagx="volume" title="volume">66</span><span class="fpage" tagx="fpage" title="fpage">1773</span><span class="lpage" tagx="lpage" title="lpage">87</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22671546">22671546</a></span></span></li> <li tag="ref"><a name="CR84" /><span class="label" tagx="label" title="label">84.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lartillot</span><span class="given-names" tagx="given-names" title="given-names">N</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Poujol</span><span class="given-names" tagx="given-names" title="given-names">R</span></span></span><span class="mixed-article-title" title="mixed-article-title">A phylogenetic model for investigating correlated evolution of substitution rates and continuous phenotypic characters</span><span class="source" tagx="source" title="source">Mol Biol Evol</span><span class="year" tagx="year" title="year">2011</span><span class="volume" tagx="volume" title="volume">28</span><span class="fpage" tagx="fpage" title="fpage">729</span><span class="lpage" tagx="lpage" title="lpage">44</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20926596">20926596</a></span></span></li> <li tag="ref"><a name="CR85" /><span class="label" tagx="label" title="label">85.</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation">Benton MJ, Donoghue PCJ, Asher RJ. Calibrating and constraining molecular clocks. In: SB Hedges and S Kumar, editors. The Timetree of Life. Oxford University Press, Oxford; 2009.</span></li> <li tag="ref"><a name="CR86" /><span class="label" tagx="label" title="label">86.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kumar</span><span class="given-names" tagx="given-names" title="given-names">S</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hedges</span><span class="given-names" tagx="given-names" title="given-names">SB</span></span></span><span class="mixed-article-title" title="mixed-article-title">TimeTree2: species divergence times on the iPhone</span><span class="source" tagx="source" title="source">Bioinformatics</span><span class="year" tagx="year" title="year">2011</span><span class="volume" tagx="volume" title="volume">27</span><span class="fpage" tagx="fpage" title="fpage">2023</span><span class="lpage" tagx="lpage" title="lpage">4</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21622662">21622662</a></span></span></li> <li tag="ref"><a name="CR87" /><span class="label" tagx="label" title="label">87.</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation">Jones KE, Bielby J, Cardillo M, Fritz SA, O’Dell J, Orme CDL, et al. PanTHERIA: a species-level database of life history, ecology, and geography of extant and recently extinct mammals. Ecology. 2009;90:2648–8.</span></li> <li tag="ref"><a name="CR88" /><span class="label" tagx="label" title="label">88.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Yang</span><span class="given-names" tagx="given-names" title="given-names">Z</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Nielsen</span><span class="given-names" tagx="given-names" title="given-names">R</span></span></span><span class="mixed-article-title" title="mixed-article-title">Synonymous and nonsynonymous rate variation in nuclear genes of mammals</span><span class="source" tagx="source" title="source">J Mol Evol</span><span class="year" tagx="year" title="year">1998</span><span class="volume" tagx="volume" title="volume">46</span><span class="fpage" tagx="fpage" title="fpage">409</span><span class="lpage" tagx="lpage" title="lpage">18</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/9541535">9541535</a></span></span></li> <li tag="ref"><a name="CR89" /><span class="label" tagx="label" title="label">89.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Pupko</span><span class="given-names" tagx="given-names" title="given-names">T</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Pe’er</span><span class="given-names" tagx="given-names" title="given-names">I</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hasegawa</span><span class="given-names" tagx="given-names" title="given-names">M</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Graur</span><span class="given-names" tagx="given-names" title="given-names">D</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Friedman</span><span class="given-names" tagx="given-names" title="given-names">N</span></span></span><span class="mixed-article-title" title="mixed-article-title">A branch-and-bound algorithm for the inference of ancestral amino-acid sequences when the replacement rate varies among sites: Application to the evolution of five gene families</span><span class="source" tagx="source" title="source">Bioinformatics</span><span class="year" tagx="year" title="year">2002</span><span class="volume" tagx="volume" title="volume">18</span><span class="fpage" tagx="fpage" title="fpage">1116</span><span class="lpage" tagx="lpage" title="lpage">23</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12176835">12176835</a></span></span></li> <li tag="ref"><a name="CR90" /><span class="label" tagx="label" title="label">90.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ashkenazy</span><span class="given-names" tagx="given-names" title="given-names">H</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Penn</span><span class="given-names" tagx="given-names" title="given-names">O</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Doron-Faigenboim</span><span class="given-names" tagx="given-names" title="given-names">A</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Cohen</span><span class="given-names" tagx="given-names" title="given-names">O</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Cannarozzi</span><span class="given-names" tagx="given-names" title="given-names">G</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zomer</span><span class="given-names" tagx="given-names" title="given-names">O</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">FastML: a web server for probabilistic reconstruction of ancestral sequences</span><span class="source" tagx="source" title="source">Nucleic Acids Res</span><span class="year" tagx="year" title="year">2012</span><span class="volume" tagx="volume" title="volume">40</span><span class="fpage" tagx="fpage" title="fpage">W580</span><span class="lpage" tagx="lpage" title="lpage">4</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22661579">22661579</a></span></span></li> <li tag="ref"><a name="CR91" /><span class="label" tagx="label" title="label">91.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hassanin</span><span class="given-names" tagx="given-names" title="given-names">A</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Delsuc</span><span class="given-names" tagx="given-names" title="given-names">F</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ropiquet</span><span class="given-names" tagx="given-names" title="given-names">A</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hammer</span><span class="given-names" tagx="given-names" title="given-names">C</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">van Vuuren</span><span class="given-names" tagx="given-names" title="given-names">BJ</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Matthee</span><span class="given-names" tagx="given-names" title="given-names">C</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Pattern and timing of diversification of Cetartiodactyla (Mammalia, Laurasiatheria), as revealed by a comprehensive analysis of mitochondrial genomes</span><span class="source" tagx="source" title="source">C R Biol</span><span class="year" tagx="year" title="year">2012</span><span class="volume" tagx="volume" title="volume">335</span><span class="fpage" tagx="fpage" title="fpage">32</span><span class="lpage" tagx="lpage" title="lpage">50</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22226162">22226162</a></span></span></li> <li tag="ref"><a name="CR92" /><span class="label" tagx="label" title="label">92.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kawasaki</span><span class="given-names" tagx="given-names" title="given-names">K</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Weiss</span><span class="given-names" tagx="given-names" title="given-names">KM</span></span></span><span class="mixed-article-title" title="mixed-article-title">SCPP gene evolution and the dental mineralization continuum</span><span class="source" tagx="source" title="source">J Dent Res</span><span class="year" tagx="year" title="year">2008</span><span class="volume" tagx="volume" title="volume">87</span><span class="fpage" tagx="fpage" title="fpage">520</span><span class="lpage" tagx="lpage" title="lpage">31</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18502959">18502959</a></span></span></li> <li tag="ref"><a name="CR93" /><span class="label" tagx="label" title="label">93.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Davis</span><span class="given-names" tagx="given-names" title="given-names">MJ</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hanson</span><span class="given-names" tagx="given-names" title="given-names">KA</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Clark</span><span class="given-names" tagx="given-names" title="given-names">F</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Fink</span><span class="given-names" tagx="given-names" title="given-names">JL</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zhang</span><span class="given-names" tagx="given-names" title="given-names">F</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kasukawa</span><span class="given-names" tagx="given-names" title="given-names">T</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Differential use of signal peptides and membrane domains is a common occurrence in the protein output of transcriptional units</span><span class="source" tagx="source" title="source">PLoS Genet</span><span class="year" tagx="year" title="year">2006</span><span class="volume" tagx="volume" title="volume">2</span><span class="fpage" tagx="fpage" title="fpage">e46</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16683029">16683029</a></span></span></li> <li tag="ref"><a name="CR94" /><span class="label" tagx="label" title="label">94.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Al-Hashimi</span><span class="given-names" tagx="given-names" title="given-names">N</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lafont</span><span class="given-names" tagx="given-names" title="given-names">A-G</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Delgado</span><span class="given-names" tagx="given-names" title="given-names">S</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kawasaki</span><span class="given-names" tagx="given-names" title="given-names">K</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Sire</span><span class="given-names" tagx="given-names" title="given-names">J-Y</span></span></span><span class="mixed-article-title" title="mixed-article-title">The enamelin genes in lizard, crocodile, and frog and the pseudogene in the chicken provide new insights on enamelin evolution in tetrapods</span><span class="source" tagx="source" title="source">Mol Biol Evol</span><span class="year" tagx="year" title="year">2010</span><span class="volume" tagx="volume" title="volume">27</span><span class="fpage" tagx="fpage" title="fpage">2078</span><span class="lpage" tagx="lpage" title="lpage">94</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20403965">20403965</a></span></span></li> <li tag="ref"><a name="CR95" /><span class="label" tagx="label" title="label">95.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Jin</span><span class="given-names" tagx="given-names" title="given-names">T</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ito</span><span class="given-names" tagx="given-names" title="given-names">Y</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Luan</span><span class="given-names" tagx="given-names" title="given-names">X</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Dangaria</span><span class="given-names" tagx="given-names" title="given-names">S</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Walker</span><span class="given-names" tagx="given-names" title="given-names">C</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Allen</span><span class="given-names" tagx="given-names" title="given-names">M</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Elongated polyproline motifs facilitate enamel evolution through matrix subunit compaction</span><span class="source" tagx="source" title="source">PLoS Biol</span><span class="year" tagx="year" title="year">2009</span><span class="volume" tagx="volume" title="volume">7</span><span class="fpage" tagx="fpage" title="fpage">e1000262</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20027208">20027208</a></span></span></li> <li tag="ref"><a name="CR96" /><span class="label" tagx="label" title="label">96.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Conne</span><span class="given-names" tagx="given-names" title="given-names">B</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Stutz</span><span class="given-names" tagx="given-names" title="given-names">A</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Vassalli</span><span class="given-names" tagx="given-names" title="given-names">JD</span></span></span><span class="mixed-article-title" title="mixed-article-title">The 3′ untranslated region of messenger RNA: a molecular "hotspot" for pathology?</span><span class="source" tagx="source" title="source">Nat Med</span><span class="year" tagx="year" title="year">2000</span><span class="volume" tagx="volume" title="volume">6</span><span class="fpage" tagx="fpage" title="fpage">637</span><span class="lpage" tagx="lpage" title="lpage">41</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/10835679">10835679</a></span></span></li> <li tag="ref"><a name="CR97" /><span class="label" tagx="label" title="label">97.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Beaudoing</span><span class="given-names" tagx="given-names" title="given-names">E</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Freier</span><span class="given-names" tagx="given-names" title="given-names">S</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wyatt</span><span class="given-names" tagx="given-names" title="given-names">JR</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Claverie</span><span class="given-names" tagx="given-names" title="given-names">J-M</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gautheret</span><span class="given-names" tagx="given-names" title="given-names">D</span></span></span><span class="mixed-article-title" title="mixed-article-title">Patterns of variant polyadenylation signal usage in human genes</span><span class="source" tagx="source" title="source">Genome Res</span><span class="year" tagx="year" title="year">2000</span><span class="volume" tagx="volume" title="volume">10</span><span class="fpage" tagx="fpage" title="fpage">1001</span><span class="lpage" tagx="lpage" title="lpage">10</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/10899149">10899149</a></span></span></li> <li tag="ref"><a name="CR98" /><span class="label" tagx="label" title="label">98.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Edwalds-Gilbert</span><span class="given-names" tagx="given-names" title="given-names">G</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Veraldi</span><span class="given-names" tagx="given-names" title="given-names">KL</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Milcarek</span><span class="given-names" tagx="given-names" title="given-names">C</span></span></span><span class="mixed-article-title" title="mixed-article-title">Alternative poly(A) site selection in complex transcription units: means to an end?</span><span class="source" tagx="source" title="source">Nucleic Acids Res</span><span class="year" tagx="year" title="year">1997</span><span class="volume" tagx="volume" title="volume">25</span><span class="fpage" tagx="fpage" title="fpage">2547</span><span class="lpage" tagx="lpage" title="lpage">61</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/9185563">9185563</a></span></span></li> <li tag="ref"><a name="CR99" /><span class="label" tagx="label" title="label">99.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gautheret</span><span class="given-names" tagx="given-names" title="given-names">D</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Poirot</span><span class="given-names" tagx="given-names" title="given-names">O</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lopez</span><span class="given-names" tagx="given-names" title="given-names">F</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Audic</span><span class="given-names" tagx="given-names" title="given-names">S</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Claverie</span><span class="given-names" tagx="given-names" title="given-names">J-M</span></span></span><span class="mixed-article-title" title="mixed-article-title">Alternate polyadenylation in human mRNAs: a large-scale analysis by EST clustering</span><span class="source" tagx="source" title="source">Genome Res</span><span class="year" tagx="year" title="year">1998</span><span class="volume" tagx="volume" title="volume">8</span><span class="fpage" tagx="fpage" title="fpage">524</span><span class="lpage" tagx="lpage" title="lpage">30</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/9582195">9582195</a></span></span></li> <li tag="ref"><a name="CR100" /><span class="label" tagx="label" title="label">100.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Sehic</span><span class="given-names" tagx="given-names" title="given-names">A</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Risnes</span><span class="given-names" tagx="given-names" title="given-names">S</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Khuu</span><span class="given-names" tagx="given-names" title="given-names">C</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Khan</span><span class="given-names" tagx="given-names" title="given-names">Q-E-S</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Osmundsen</span><span class="given-names" tagx="given-names" title="given-names">H</span></span></span><span class="mixed-article-title" title="mixed-article-title">Effects of in vivo transfection with anti-miR-214 on gene expression in murine molar tooth germ</span><span class="source" tagx="source" title="source">Physiol Genomics</span><span class="year" tagx="year" title="year">2011</span><span class="volume" tagx="volume" title="volume">43</span><span class="fpage" tagx="fpage" title="fpage">488</span><span class="lpage" tagx="lpage" title="lpage">98</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21363966">21363966</a></span></span></li> <li tag="ref"><a name="CR101" /><span class="label" tagx="label" title="label">101.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Fukae</span><span class="given-names" tagx="given-names" title="given-names">M</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kanazashi</span><span class="given-names" tagx="given-names" title="given-names">M</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Nagano</span><span class="given-names" tagx="given-names" title="given-names">T</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Tanabe</span><span class="given-names" tagx="given-names" title="given-names">T</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Oida</span><span class="given-names" tagx="given-names" title="given-names">S</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gomi</span><span class="given-names" tagx="given-names" title="given-names">K</span></span></span><span class="mixed-article-title" title="mixed-article-title">Porcine sheath proteins show periodontal ligament regeneration activity</span><span class="source" tagx="source" title="source">Eur J Oral Sci</span><span class="year" tagx="year" title="year">2006</span><span class="volume" tagx="volume" title="volume">114</span><span class="fpage" tagx="fpage" title="fpage">212</span><span class="lpage" tagx="lpage" title="lpage">8</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16674688">16674688</a></span></span></li> <li tag="ref"><a name="CR102" /><span class="label" tagx="label" title="label">102.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Yamakoshi</span><span class="given-names" tagx="given-names" title="given-names">Y</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hu</span><span class="given-names" tagx="given-names" title="given-names">JC-C</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zhang</span><span class="given-names" tagx="given-names" title="given-names">H</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Iwata</span><span class="given-names" tagx="given-names" title="given-names">T</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Yamakoshi</span><span class="given-names" tagx="given-names" title="given-names">F</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Simmer</span><span class="given-names" tagx="given-names" title="given-names">JP</span></span></span><span class="mixed-article-title" title="mixed-article-title">Proteomic analysis of enamel matrix using a two-dimensional protein fractionation system</span><span class="source" tagx="source" title="source">Eur J Oral Sci</span><span class="year" tagx="year" title="year">2006</span><span class="volume" tagx="volume" title="volume">114</span><span class="fpage" tagx="fpage" title="fpage">266</span><span class="lpage" tagx="lpage" title="lpage">71</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16674696">16674696</a></span></span></li> <li tag="ref"><a name="CR103" /><span class="label" tagx="label" title="label">103.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Uchida</span><span class="given-names" tagx="given-names" title="given-names">T</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Murakami</span><span class="given-names" tagx="given-names" title="given-names">C</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wakida</span><span class="given-names" tagx="given-names" title="given-names">K</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Satoda</span><span class="given-names" tagx="given-names" title="given-names">T</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Dohi</span><span class="given-names" tagx="given-names" title="given-names">N</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Takahashi</span><span class="given-names" tagx="given-names" title="given-names">O</span></span></span><span class="mixed-article-title" title="mixed-article-title">Synthesis, secretion, degradation, and fate of ameloblastin during the matrix formation stage of the rat incisor as shown by immunocytochemistry and immunochemistry using region-specific antibodies</span><span class="source" tagx="source" title="source">J Histochem Cytochem</span><span class="year" tagx="year" title="year">1997</span><span class="volume" tagx="volume" title="volume">45</span><span class="fpage" tagx="fpage" title="fpage">1329</span><span class="lpage" tagx="lpage" title="lpage">40</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/9313795">9313795</a></span></span></li> <li tag="ref"><a name="CR104" /><span class="label" tagx="label" title="label">104.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hu</span><span class="given-names" tagx="given-names" title="given-names">JC-C</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Yamakoshi</span><span class="given-names" tagx="given-names" title="given-names">Y</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Yamakoshi</span><span class="given-names" tagx="given-names" title="given-names">F</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Krebsbach</span><span class="given-names" tagx="given-names" title="given-names">PH</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Simmer</span><span class="given-names" tagx="given-names" title="given-names">JP</span></span></span><span class="mixed-article-title" title="mixed-article-title">Proteomics and genetics of dental enamel</span><span class="source" tagx="source" title="source">Cells Tissues Organs</span><span class="year" tagx="year" title="year">2005</span><span class="volume" tagx="volume" title="volume">181</span><span class="fpage" tagx="fpage" title="fpage">219</span><span class="lpage" tagx="lpage" title="lpage">31</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16612087">16612087</a></span></span></li> <li tag="ref"><a name="CR105" /><span class="label" tagx="label" title="label">105.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kobayashi</span><span class="given-names" tagx="given-names" title="given-names">K</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Yamakoshi</span><span class="given-names" tagx="given-names" title="given-names">Y</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hu</span><span class="given-names" tagx="given-names" title="given-names">JC-C</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gomi</span><span class="given-names" tagx="given-names" title="given-names">K</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Arai</span><span class="given-names" tagx="given-names" title="given-names">T</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Fukae</span><span class="given-names" tagx="given-names" title="given-names">M</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Splicing determines the glycosylation state of ameloblastin</span><span class="source" tagx="source" title="source">J Dent Res</span><span class="year" tagx="year" title="year">2007</span><span class="volume" tagx="volume" title="volume">86</span><span class="fpage" tagx="fpage" title="fpage">962</span><span class="lpage" tagx="lpage" title="lpage">7</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17890672">17890672</a></span></span></li> <li tag="ref"><a name="CR106" /><span class="label" tagx="label" title="label">106.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bohley</span><span class="given-names" tagx="given-names" title="given-names">P</span></span></span><span class="mixed-article-title" title="mixed-article-title">Surface hydrophobicity and intracellular degradation of proteins</span><span class="source" tagx="source" title="source">Biol Chem</span><span class="year" tagx="year" title="year">1996</span><span class="volume" tagx="volume" title="volume">377</span><span class="fpage" tagx="fpage" title="fpage">425</span><span class="lpage" tagx="lpage" title="lpage">35</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/8922276">8922276</a></span></span></li> <li tag="ref"><a name="CR107" /><span class="label" tagx="label" title="label">107.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Mayer</span><span class="given-names" tagx="given-names" title="given-names">BJ</span></span></span><span class="mixed-article-title" title="mixed-article-title">SH3 domains: complexity in moderation</span><span class="source" tagx="source" title="source">J Cell Sci</span><span class="year" tagx="year" title="year">2001</span><span class="volume" tagx="volume" title="volume">114</span><span class="fpage" tagx="fpage" title="fpage">1253</span><span class="lpage" tagx="lpage" title="lpage">63</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11256992">11256992</a></span></span></li> <li tag="ref"><a name="CR108" /><span class="label" tagx="label" title="label">108.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ingham</span><span class="given-names" tagx="given-names" title="given-names">RJ</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Colwill</span><span class="given-names" tagx="given-names" title="given-names">K</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Howard</span><span class="given-names" tagx="given-names" title="given-names">C</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Dettwiler</span><span class="given-names" tagx="given-names" title="given-names">S</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lim</span><span class="given-names" tagx="given-names" title="given-names">CSH</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Yu</span><span class="given-names" tagx="given-names" title="given-names">J</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">WW domains provide a platform for the assembly of multiprotein networks</span><span class="source" tagx="source" title="source">Mol Cell Biol</span><span class="year" tagx="year" title="year">2005</span><span class="volume" tagx="volume" title="volume">25</span><span class="fpage" tagx="fpage" title="fpage">7092</span><span class="lpage" tagx="lpage" title="lpage">106</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16055720">16055720</a></span></span></li> <li tag="ref"><a name="CR109" /><span class="label" tagx="label" title="label">109.</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation">Fukumoto S, Yamada A, Iwamoto T, Nakamura T. Dental epithelium proliferation and differentiation regulated by ameloblastin. In: Sasano T et al, editors. Interface Oral Health Science. Springer Japan, Tokyo; 2010:33–8.</span></li> <li tag="ref"><a name="CR110" /><span class="label" tagx="label" title="label">110.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Yoo</span><span class="given-names" tagx="given-names" title="given-names">SY</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kobayashi</span><span class="given-names" tagx="given-names" title="given-names">M</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lee</span><span class="given-names" tagx="given-names" title="given-names">PP</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lee</span><span class="given-names" tagx="given-names" title="given-names">S-W</span></span></span><span class="mixed-article-title" title="mixed-article-title">Early osteogenic differentiation of mouse preosteoblasts induced by collagen-derived DGEA-peptide on nanofibrous phage tissue matrices</span><span class="source" tagx="source" title="source">Biomacromolecules</span><span class="year" tagx="year" title="year">2011</span><span class="volume" tagx="volume" title="volume">12</span><span class="fpage" tagx="fpage" title="fpage">987</span><span class="lpage" tagx="lpage" title="lpage">96</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21344869">21344869</a></span></span></li> <li tag="ref"><a name="CR111" /><span class="label" tagx="label" title="label">111.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wazen</span><span class="given-names" tagx="given-names" title="given-names">RM</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Moffatt</span><span class="given-names" tagx="given-names" title="given-names">P</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zalzal</span><span class="given-names" tagx="given-names" title="given-names">SF</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Yamada</span><span class="given-names" tagx="given-names" title="given-names">Y</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Nanci</span><span class="given-names" tagx="given-names" title="given-names">A</span></span></span><span class="mixed-article-title" title="mixed-article-title">A mouse model expressing a truncated form of ameloblastin exhibits dental and junctional epithelium defects</span><span class="source" tagx="source" title="source">Matrix Biol</span><span class="year" tagx="year" title="year">2009</span><span class="volume" tagx="volume" title="volume">28</span><span class="fpage" tagx="fpage" title="fpage">292</span><span class="lpage" tagx="lpage" title="lpage">303</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19375505">19375505</a></span></span></li> <li tag="ref"><a name="CR112" /><span class="label" tagx="label" title="label">112.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Sire</span><span class="given-names" tagx="given-names" title="given-names">J-Y</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Delgado</span><span class="given-names" tagx="given-names" title="given-names">S</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Fromentin</span><span class="given-names" tagx="given-names" title="given-names">D</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Girondot</span><span class="given-names" tagx="given-names" title="given-names">M</span></span></span><span class="mixed-article-title" title="mixed-article-title">Amelogenin: lessons from evolution</span><span class="source" tagx="source" title="source">Arch Oral Biol</span><span class="year" tagx="year" title="year">2005</span><span class="volume" tagx="volume" title="volume">50</span><span class="fpage" tagx="fpage" title="fpage">205</span><span class="lpage" tagx="lpage" title="lpage">12</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15721151">15721151</a></span></span></li> <li tag="ref"><a name="CR113" /><span class="label" tagx="label" title="label">113.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Shellis</span><span class="given-names" tagx="given-names" title="given-names">RP</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Beynon</span><span class="given-names" tagx="given-names" title="given-names">AD</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Reid</span><span class="given-names" tagx="given-names" title="given-names">DJ</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hiiemae</span><span class="given-names" tagx="given-names" title="given-names">KM</span></span></span><span class="mixed-article-title" title="mixed-article-title">Variations in molar enamel thickness among primates</span><span class="source" tagx="source" title="source">J Hum Evol</span><span class="year" tagx="year" title="year">1998</span><span class="volume" tagx="volume" title="volume">35</span><span class="fpage" tagx="fpage" title="fpage">507</span><span class="lpage" tagx="lpage" title="lpage">22</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/9774508">9774508</a></span></span></li> <li tag="ref"><a name="CR114" /><span class="label" tagx="label" title="label">114.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Dumont</span><span class="given-names" tagx="given-names" title="given-names">ER</span></span></span><span class="mixed-article-title" title="mixed-article-title">Enamel thickness and dietary adaptation among extant primates and chiropterans</span><span class="source" tagx="source" title="source">J Mammal</span><span class="year" tagx="year" title="year">1995</span><span class="volume" tagx="volume" title="volume">76</span><span class="fpage" tagx="fpage" title="fpage">1127</span><span class="lpage" tagx="lpage" title="lpage">36</span></span></li> <li tag="ref"><a name="CR115" /><span class="label" tagx="label" title="label">115.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Pampush</span><span class="given-names" tagx="given-names" title="given-names">JD</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Duque</span><span class="given-names" tagx="given-names" title="given-names">AC</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Burrows</span><span class="given-names" tagx="given-names" title="given-names">BR</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Daegling</span><span class="given-names" tagx="given-names" title="given-names">DJ</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kenney</span><span class="given-names" tagx="given-names" title="given-names">WF</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">McGraw</span><span class="given-names" tagx="given-names" title="given-names">WS</span></span></span><span class="mixed-article-title" title="mixed-article-title">Homoplasy and thick enamel in primates</span><span class="source" tagx="source" title="source">J Hum Evol</span><span class="year" tagx="year" title="year">2013</span><span class="volume" tagx="volume" title="volume">64</span><span class="fpage" tagx="fpage" title="fpage">216</span><span class="lpage" tagx="lpage" title="lpage">24</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23415414">23415414</a></span></span></li> <li tag="ref"><a name="CR116" /><span class="label" tagx="label" title="label">116.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kelley</span><span class="given-names" tagx="given-names" title="given-names">JL</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Swanson</span><span class="given-names" tagx="given-names" title="given-names">WJ</span></span></span><span class="mixed-article-title" title="mixed-article-title">Dietary change and adaptive evolution of enamelin in humans and among primates</span><span class="source" tagx="source" title="source">Genetics</span><span class="year" tagx="year" title="year">2008</span><span class="volume" tagx="volume" title="volume">178</span><span class="fpage" tagx="fpage" title="fpage">1595</span><span class="lpage" tagx="lpage" title="lpage">603</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18245370">18245370</a></span></span></li> <li tag="ref"><a name="CR117" /><span class="label" tagx="label" title="label">117.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Meredith</span><span class="given-names" tagx="given-names" title="given-names">RW</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gatesy</span><span class="given-names" tagx="given-names" title="given-names">J</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Murphy</span><span class="given-names" tagx="given-names" title="given-names">WJ</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ryder</span><span class="given-names" tagx="given-names" title="given-names">OA</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Springer</span><span class="given-names" tagx="given-names" title="given-names">MS</span></span></span><span class="mixed-article-title" title="mixed-article-title">Molecular decay of the tooth gene enamelin (ENAM) mirrors the loss of enamel in the fossil record of placental mammals</span><span class="source" tagx="source" title="source">PLoS Genet</span><span class="year" tagx="year" title="year">2009</span><span class="volume" tagx="volume" title="volume">5</span><span class="fpage" tagx="fpage" title="fpage">e1000634</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19730686">19730686</a></span></span></li> <li tag="ref"><a name="CR118" /><span class="label" tagx="label" title="label">118.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Meredith</span><span class="given-names" tagx="given-names" title="given-names">RW</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zhang</span><span class="given-names" tagx="given-names" title="given-names">G</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gilbert</span><span class="given-names" tagx="given-names" title="given-names">MTP</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Jarvis</span><span class="given-names" tagx="given-names" title="given-names">ED</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Springer</span><span class="given-names" tagx="given-names" title="given-names">MS</span></span></span><span class="mixed-article-title" title="mixed-article-title">Evidence for a single loss of mineralized teeth in the common avian ancestor</span><span class="source" tagx="source" title="source">Science</span><span class="year" tagx="year" title="year">2014</span><span class="volume" tagx="volume" title="volume">346</span><span class="fpage" tagx="fpage" title="fpage">1254390</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25504730">25504730</a></span></span></li> <li tag="ref"><a name="CR119" /><span class="label" tagx="label" title="label">119.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Chun</span><span class="given-names" tagx="given-names" title="given-names">Y-HP</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Yamakoshi</span><span class="given-names" tagx="given-names" title="given-names">Y</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Yamakoshi</span><span class="given-names" tagx="given-names" title="given-names">F</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Fukae</span><span class="given-names" tagx="given-names" title="given-names">M</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hu</span><span class="given-names" tagx="given-names" title="given-names">JC-C</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bartlett</span><span class="given-names" tagx="given-names" title="given-names">JD</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Cleavage site specificity of MMP-20 for secretory-stage ameloblastin</span><span class="source" tagx="source" title="source">J Dent Res</span><span class="year" tagx="year" title="year">2010</span><span class="volume" tagx="volume" title="volume">89</span><span class="fpage" tagx="fpage" title="fpage">785</span><span class="lpage" tagx="lpage" title="lpage">90</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20400724">20400724</a></span></span></li> </ul> </div> </div>  </body></html>