<html xmlns="http://www.w3.org/1999/xhtml"> <head> <style type="text/css">
      	     		 
      	    a                          {background : #ffffff; }
      	    article                    {border-style : dotted; border-width : 2px; }
		 	
		 	div                        {background : #ffffcc;}
		 	div.abstract-title         {font-weight : bold ; font-size : 16pt;}
		 	div.ack                    {border-style : solid ; border-color : red; margin : 2em; }
		 	div.article-meta           {border-style : solid; border-width : 1 pt; margin 1em;}
		 	div.boxed-text             {margin : 5em; border-style : solid;}
		 	div.contrib-group          {margin : 1em; }
		 	div.fig                    {border-style : solid; border-width : 2px; margin : 2em; }
		 	div.funding                {font-weight : bold ; font-size : 16pt;}
		 	div.given-names            {font-style : italic;}
		 	div.intro                  {border-style : inset; margin : 5px;}
		 	div.introduction           {border-style : inset; margin : 5px;}
		 	div.journal-meta           {border-style : solid; border-width : 1 pt; margin 1em;}
		 	div.journal-title-group    {background : #ffeeee;}
		 	div.article-title          {font-weight : bold ; font-size : 18pt;}
		 	div.kwd                    {font-style : italic}
		 	div.meta-name              {font-weight : bold ; font-size : 16pt;}
		 	div.name                   {font-weight : bold;}
		 	div.alt-title              {font-style : italic; font-size : 12px;}
		 	
		 	div.sec                    {border : 2px; margin 5px; padding 2px;}
		 	div.title                  {font-family : courier; font-weight : bold;
		 	                            font-size : 16pt; margin : 5px;}
		 	
		 	div.materials_methods      {border-style : double; margin : 5px; }
		 	div.methods                {border-style : double; margin : 5px; }
		 	
		 	div.results                {border-style : solid; margin : 5px;}
		 	div.background             {border-style : dotted; margin : 5px;}
		 	
		 	div.discussion             {border-style : groove; margin : 5px;}
		 	div.conclusion             {border-style : ridge; margin : 5px;}
		 	div.conclusions             {border-style : ridge; margin : 5px;}
		 	div.supplementary-material {border-style : inset; margin : 5px;}
		 	div.abbreviations          {border-style : double; border-color : red; margin : 5px;}
		 	div.competinginterests     {border-style : double; border-color : blue; margin : 5px;}
		 	div.acknowledgements       {border-style : double; border-color : green; margin : 5px;}
		 	div.authors_contributions   {border-style : double; border-color : purple; margin : 5px;}
		 	
		 	div.publisher              {border-style : outset; margin : 5px;}
		 	div.fn-type-conflict       {background : #f88; }
		 	div.fn-type-con            {background : #ddf; }
		 	div.fn-type-other          {background : #ddd; }
		 	
		 	div.unknown                {background : #ffd;
									 	  border-style : solid;
									 	  border-width : 1px;
									 	  padding : 2 px;}
		 	  
      	    table                      {background : #ffffdd;}
		 	tr                         {background : #ddddff; padding : 1px;}
		 	
		 	span                       {background : #ffcccc;}
		 	
		 	span.citation-author       {font-family : helvetica; background : #ffeeee;}
		 	span.collab                {background : #ddffff; }
		 	span.comment               {font-family : courier; font-size : 6px; background : #ffaaff;}
		 	span.contrib               {background : #ffffff;}
		 	span.corresp               {background : #ddffdd; }
		 	span.doi                   {background : #ffffff;}
		 	span.email                 {font-family : courier; }
		 	span.etal                  {font-style : italic;}
		 	span.fpage                 {font-family : courier;}
		 	span.given-names           {background : #ffffff;}
		 	span.iso-abbrev            {background : #ffffff;}
		 	span.issn-epub             {background : #ffffff;}
		 	span.issn-ppub             {background : #ffffff;}
		 	span.journal-title         {background : #ffffff;}
		 	span.lpage                 {font-family : courier;}
		 	span.mixed-article-title   {font-style : italic ;}
		 	span.nlm-ta                {background : #ffffff;}
		 	span.pmc                   {background : #ffffff;}
		 	span.pmcid                 {background : #ffffff;}
		 	span.pmid                  {background : #ffffff;}
		 	span.publisher             {background : #ffffff;}
		 	span.publisher-id          {background : #ffffff;}
		 	span.publisher-name        {background : #ffffff;}
		 	span.source                {background : #ffffff;}
		 	span.subject               {background : #ffffff;}
		 	span.surname               {background : #ffffff;}
		 	span.volume                {font-family : courier; font-weight : bold;}
		 	span.year                  {font-family : courier ; font-style : italic;}
			</style> </head> <body> <div class="front" title="front"> <div class="journal-meta" tagx="journal-meta" title="journal-meta"><span class="nlm-ta" title="nlm-ta">PLoS One</span><span class="iso-abbrev" title="iso-abbrev">PLoS ONE</span><span class="publisher-id" title="publisher-id">plos</span><span class="pmc" title="pmc">plosone</span><div class="journal-title-group" tagx="journal-title-group" title="journal-title-group"><span class="journal-title" tagx="journal-title" title="journal-title">PLoS ONE</span></div><span class="issn-epub" tagx="issn" title="issn-epub">1932-6203</span><div class="publisher" tagx="publisher" title="publisher"><span class="publisher-name" tagx="publisher-name" title="publisher-name">Public Library of Science</span><span class="publisher-loc" tagx="publisher-loc" title="publisher-loc">San Francisco, CA USA</span></div> </div> <div class="article-meta" tagx="article-meta" title="article-meta"><span class="pmid" title="pmid"> pmid: <a href="http://www.ncbi.nlm.nih.gov/pubmed/26821159">26821159</a></span><span class="doi" title="doi"> doi: <a href="https://dx.doi.org/10.1371/journal.pone.0147647">10.1371/journal.pone.0147647</a></span><span class="publisher-id" title="publisher-id">PONE-D-15-46831</span><div class="article-categories" title="article-categories">
: <span class="subject" title="subject">Research Article</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Biology and Life Sciences</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Genetics</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Cytogenetics</span>: <span class="subject" title="subject">Karyotypes</span></div> </div> </div> <div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Biology and Life Sciences</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Cell Biology</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Chromosome Biology</span>: <span class="subject" title="subject">Chromosomes</span></div> </div> </div> <div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Biology and Life Sciences</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Organisms</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Animals</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Vertebrates</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Mammals</span>: <span class="subject" title="subject">Carnivora</span></div> </div> </div> </div> </div> <div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Biology and Life Sciences</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Organisms</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Animals</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Vertebrates</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Mammals</span>: <span class="subject" title="subject">Dogs</span></div> </div> </div> </div> </div> <div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Biology and Life Sciences</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Molecular Biology</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Molecular Biology Techniques</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Molecular Probe Techniques</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Probe Hybridization</span>: <span class="subject" title="subject">In Situ Hybridization</span></div> </div> </div> </div> </div> <div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Research and Analysis Methods</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Molecular Biology Techniques</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Molecular Probe Techniques</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Probe Hybridization</span>: <span class="subject" title="subject">In Situ Hybridization</span></div> </div> </div> </div> <div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Biology and Life Sciences</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Organisms</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Animals</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Vertebrates</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Mammals</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Marine Mammals</span>: <span class="subject" title="subject">Seals</span></div> </div> </div> </div> </div> </div> <div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Biology and Life Sciences</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Marine Biology</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Marine Mammals</span>: <span class="subject" title="subject">Seals</span></div> </div> </div> <div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Earth Sciences</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Marine and Aquatic Sciences</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Marine Biology</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Marine Mammals</span>: <span class="subject" title="subject">Seals</span></div> </div> </div> </div> <div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Biology and Life Sciences</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Cell Biology</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Chromosome Biology</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Chromosomes</span>: <span class="subject" title="subject">Homologous Chromosomes</span></div> </div> </div> </div> <div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Biology and Life Sciences</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Genetics</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Genomics</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Animal Genomics</span>: <span class="subject" title="subject">Mammalian Genomics</span></div> </div> </div> </div> </div> <div class="title-group" tagx="title-group" title="title-group"> <div class="article-title" title="article-title">
The Ancestral Carnivore Karyotype As Substantiated by Comparative Chromosome Painting of Three Pinnipeds, the Walrus, the Steller Sea Lion and the Baikal Seal (Pinnipedia, Carnivora)</div> <div class="alt-title" title="alt-title">
Chromosome Evolution in Pinnipeds</div> </div> <div class="contrib-group" title="contrib-group"><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Beklemisheva</span><span class="given-names" tagx="given-names" title="given-names">Violetta R.</span></span><a href="#aff001"><sup>1</sup></a><a href="#cor001">*</a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Perelman</span><span class="given-names" tagx="given-names" title="given-names">Polina L.</span></span><a href="#aff001"><sup>1</sup></a><a href="#aff004"><sup>4</sup></a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lemskaya</span><span class="given-names" tagx="given-names" title="given-names">Natalya A.</span></span><a href="#aff001"><sup>1</sup></a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kulemzina</span><span class="given-names" tagx="given-names" title="given-names">Anastasia I.</span></span><a href="#aff001"><sup>1</sup></a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Proskuryakova</span><span class="given-names" tagx="given-names" title="given-names">Anastasia A.</span></span><a href="#aff001"><sup>1</sup></a><a href="#aff004"><sup>4</sup></a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Burkanov</span><span class="given-names" tagx="given-names" title="given-names">Vladimir N.</span></span><a href="#aff002"><sup>2</sup></a><a href="#aff003"><sup>3</sup></a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Graphodatsky</span><span class="given-names" tagx="given-names" title="given-names">Alexander S.</span></span><a href="#aff001"><sup>1</sup></a><a href="#aff004"><sup>4</sup></a></span></div><span class="citation_author_institution" id="aff001">[1], <span class="addr-line" title="addr-line">Department of Comparative Genomics, Institute of Molecular and Cellular Biology, Siberian Branch of Russian Academy of Sciences, Novosibirsk, Russia</span></span><span class="citation_author_institution" id="aff002">[2], <span class="addr-line" title="addr-line">Department of Higher Vertebrates Ecology, Kamchatka Branch of Pacific Geographical Institute of Far East Branch of Russian Academy of Sciences, Petropavlovsk-Kamchatski, Russia</span></span><span class="citation_author_institution" id="aff003">[3], <span class="addr-line" title="addr-line">National Marine Mammal Laboratory, Alaska Fisheries Science Centre, National Marine Fisheries Service, Seattle, Washington, United States of America</span></span><span class="citation_author_institution" id="aff004">[4], <span class="addr-line" title="addr-line">Novosibirsk State University, Novosibirsk, Russia</span></span><div class="contrib-group" title="contrib-group"><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Murphy</span><span class="given-names" tagx="given-names" title="given-names">William J.</span></span><span class="role" tagx="role" title="role">Editor</span><a href="#edit1" /></span></div><span class="citation_author_institution" id="edit1">[], <span class="addr-line" title="addr-line">Texas A&amp;amp;M University, UNITED STATES</span></span><div class="author-notes" title="author-notes"> <div class="fn-type-COI-statement" title="COI-statement"> <p><b>Competing Interests: </b>The authors have declared that no competing interests exist</p> </div> <div class="fn-type-con" title="con"> <p>Conceived and designed the experiments: VRB ASG PLP. Performed the experiments: VRB AIK AAP. Analyzed the data: VRB PLP ASG. Contributed reagents/materials/analysis tools: VRB PLP AIK AAP NAL VNB ASG. Wrote the paper: VRB PLP ASG NAL.</p> </div> <div class="corresp" title="corresp">
* E-mail: <span class="email" tagx="email" title="email">bekl@mcb.nsc.ru</span></div> </div><span class="pub-date-epub" title="pub-date-epub">epub: <span>2016-1-1</span></span><span class="pub-date-collection" title="pub-date-collection">collection: <span>2016</span></span><span class="volume" tagx="volume" title="volume">11</span><span class="issue" tagx="issue" title="issue">1</span><span class="elocation-id" tagx="elocation-id" title="elocation-id">e0147647</span><span class="history" title="history"><span class="received" title="received">received: 2015-10-26</span><span class="accepted" title="accepted">accepted: 2016-1-6</span></span><div class="permissions"><span class="copyright" title="copyright">(C) , </span><span class="license" title="license"><span class="license-p" title="license-p">This is an open access article, free of all copyright, and may be freely reproduced, distributed, transmitted, modified, built upon, or otherwise used by anyone for any lawful purpose. The work is made available under the <a href="https://creativecommons.org/publicdomain/zero/1.0/">Creative Commons CC0</a> public domain dedication.</span></span></div><a class="self-uri" href="pone.0147647.pdf" title="self-uri">pone.0147647.pdf</a><div class="abstract" title="abstract"> <div class="abstract-title" title="abstract-title">
Abstract</div> <p>Karyotype evolution in Carnivora is thoroughly studied by classical and molecular cytogenetics and supplemented by reconstructions of Ancestral Carnivora Karyotype (ACK). However chromosome painting information from two pinniped families (Odobenidae and Otariidae) is noticeably missing. We report on the construction of the comparative chromosome map for species from each of the three pinniped families: the walrus (<i>Odobenus rosmarus</i>, Odobenidae–monotypic family), near threatened Steller sea lion (<i>Eumetopias jubatus</i>, Otariidae) and the endemic Baikal seal (<i>Pusa sibirica</i>, Phocidae) using combination of human, domestic dog and stone marten whole-chromosome painting probes. The earliest karyological studies of Pinnipedia showed that pinnipeds were characterized by a pronounced karyological conservatism that is confirmed here with species from Phocidae, Otariidae and Odobenidae sharing same low number of conserved human autosomal segments (32). Chromosome painting in Pinnipedia and comparison with non-pinniped carnivore karyotypes provide strong support for refined structure of ACK with 2n = 38. Constructed comparative chromosome maps show that pinniped karyotype evolution was characterized by few tandem fusions, seemingly absent inversions and slow rate of genome rearrangements (less then one rearrangement per 10 million years). Integrative comparative analyses with published chromosome painting of <i>Phoca vitulina</i> revealed common cytogenetic signature for Phoca/Pusa branch and supports Phocidae and Otaroidea (Otariidae/Odobenidae) as sister groups. We revealed rearrangements specific for walrus karyotype and found the chromosomal signature linking together families Otariidae and Odobenidae. The Steller sea lion karyotype is the most conserved among three studied species and differs from the ACK by single fusion. The study underlined the strikingly slow karyotype evolution of the Pinnipedia in general and the Otariidae in particular.</p> </div> <div class="funding-group" title="funding-group"> <div class="funding" title="funding">
Funding</div> <div class="funding-statement" title="funding-statement">
This work was supported by Russian Foundation for Basic Research (<a href="https://kias.rfbr.ru">https://kias.rfbr.ru</a>): project 14-04-00467 to VRB; project 15-04-00962 to ASG ; project 15-29-02384 to ASG and the Research grant of the Molecular and Cell Biology of RAS to ASG.</div> </div> <div class="counts" title="counts"><span class="fig-count" title="fig-count">fig-count: </span><span class="table-count" title="table-count">table-count: </span><span class="page-count" title="page-count">page-count: </span></div> <div class="custom-meta-group" title="custom-meta-group"> <meta name="Data Availability" value="All relevant data are within the paper." /> </div> </div> <div class="title" tagx="title" title="title">
Data Availability</div> <p>All relevant data are within the paper.</p> </div> <div class="body" title="body"> <div class="intro" title="intro"> <div class="title" tagx="title" title="title">
Introduction</div> <p>Pinnipeds are remarkable group of marine animals with unique adaptations to semi aquatic life comprising seals, sea lions and walrus. The pinnipeds were one of the first mammalian species group to be thoroughly investigated at each stage of cytogenetics development. Animal cytogenetics made great progress over the past century starting with introduction of techniques for efficient chromosome visualization (like colchicine application, hypotonic treatment, advances in cell culture and chromosome preparation) allowing detailed description of karyotype [<a href="#pone.0147647.ref001">1</a>]. Development of differential staining methods (G-banding in particular) allowed identification of homologous elements and chromosomal rearrangements in karyotypes of different species. This enabled karyotype comparison of different mammalian species and gave rise to comparative mammalian genomics producing vast data about chromosome evolution. The introduction of chromosome specific probes ensured precise delineation of homologous syntenic regions and identification of evolutionary chromosome rearrangements even among distant species.</p> <p>The question about the role of chromosome rearrangements during speciation remains one of the few unsolved and intriguing puzzles of current evolutionary biology. Even more interesting are the genome-shaping evolutionary processes of speciation in marine environment. Fin-footed pinnipeds have split from terrestrial members of Carnivora about 40 MYA [<a href="#pone.0147647.ref002">2</a>]. This group is so distinct that they were once considered to be a separate suborder of Carnivora or separate mammalian order Pinnipedia [<a href="#pone.0147647.ref003">3</a>,<a href="#pone.0147647.ref004">4</a>]. There was even question whether pinnipeds are monophyletic or the true seals had originated from weasels and the other two families from bears [<a href="#pone.0147647.ref005">5</a>]. In recent decades molecular phylogenetic studies have settled many questions showing that Pinnipeds are monophyletic and affine to Caniformia branch of Carnivora with some remaining uncertainty about grouping with musteloids or ursids [<a href="#pone.0147647.ref002">2</a>,<a href="#pone.0147647.ref003">3</a>,<a href="#pone.0147647.ref006">6</a>,<a href="#pone.0147647.ref007">7</a>].</p> <p>The Pinnipedia include three extant families, Odobenidae (walruses, monotypic), Otariidae (sea lions and fur seals) and Phocidae (true or earless seals). Odobenidae and Otariidae constitute the Otaroidea, the sister group of Phocidae. The pioneering karyological studies of Pinnipedia and later comparisons based on differential staining techniques showed that pinnipeds have narrow range of diploid numbers 2n = 32 to 36 and are characterized by a pronounced karyological conservatism [<a href="#pone.0147647.ref008">8</a>–<a href="#pone.0147647.ref012">12</a>]. All Otariidae have strikingly similar karyotypes with the chromosome number 2n = 36. The chromosome number of the walrus is 2n = 32. The Phocidae have two chromosome numbers, 2n = 34 and 2n = 32, separated by a single fusion [<a href="#pone.0147647.ref008">8</a>].</p> <p>Genomes of Carnivora have been investigated in detail both by classic and by molecular cytogenetic methods that showed trends of karyotype evolution in two branches of this order—Caniformia and Feliformia. To date 50 species of Carnivora have been studied by fluorescent <i>in situ</i> hybridization (FISH) with flow-sorted whole-chromosome probes from nine Carnivora species and human [<a href="#pone.0147647.ref013">13</a>,<a href="#pone.0147647.ref014">14</a>]. The conserved syntenic segments and rearrangements of these blocks were established for most families depicting fascinating karyotypic changes during Carnivora radiation. Two major patterns of genome evolution were revealed for this order: fast genome reshuffling during speciation in Canidae, Ursidae and Mephitidae and significant level of conservation among felids, viverrids, and musteloids (Mustelidae, Procyonidae) [<a href="#pone.0147647.ref013">13</a>,<a href="#pone.0147647.ref015">15</a>–<a href="#pone.0147647.ref025">25</a>].</p> <p>Comparative cytogenetics of Carnivora provided the basis for reconstruction of the karyotype of the supposed common ancestor (Ancestral Carnivora Karyotype, ACK) [<a href="#pone.0147647.ref019">19</a>,<a href="#pone.0147647.ref026">26</a>–<a href="#pone.0147647.ref029">29</a>]. In the original attempt to reconstruct ACK based on comparative G-banding analysis pinniped karyotypes played an important role [<a href="#pone.0147647.ref026">26</a>]. The first hypothetical structure of ACK using chromosome painting was based on Pinniped data [<a href="#pone.0147647.ref028">28</a>]. However current versions of reconstructed ACK contain some unsettled questions concerning the state of several ancestral segments that require additional support.</p> <p>To date sequencing data for ten Carnivora species: cat, tiger, lion, giant panda, domestic dog, polar and brown bears, ferret and two pinnipeds—walrus and Weddel’s seal, have been published (<a href="http://www.ensembl.org">http://www.ensembl.org</a>, <a href="http://www.ncbi.nlm.nih.gov/genome">http://www.ncbi.nlm.nih.gov/genome</a>). Sequencing of the Baikal seal genome is in progress (unpublished). Comparative chromosome painting maps provide primary backbone for <i>de novo</i> whole genome assembly. Yet high-resolution G-banded chromosomes and molecular assignment of homologies through chromosome painting is only available for one pinniped species—the harbour seal (<i>Phoca vitulina</i>) that belongs to family Phocidae [<a href="#pone.0147647.ref028">28</a>,<a href="#pone.0147647.ref030">30</a>]. However two other families of pinnipeds (Otariidae and Odobenidae) are lacking the chromosome painting data [<a href="#pone.0147647.ref013">13</a>].</p> <p>Here, we report comparative chromosome maps for representatives from all three pinniped families: the walrus (<i>Odobenus rosmarus</i>, Odobenidae), Steller sea lion (<i>Eumetopias jubatus</i>, Otariidae) and the Baikal seal (<i>Pusa sibirica</i>, Phocidae) established by localization of human, domestic dog and selected set of stone marten chromosome-specific painting probes, compiled with previously published chromosome painting data. We conduct analysis in the context of chromosome evolution in the order Carnivora and in eutherian mammals and refine the ACK based on the new evidence from Pinnipedia chromosome painting.</p> </div> <div class="materials_methods" title="materials|methods"> <div class="title" tagx="title" title="title">
Materials and Methods</div> <div class="speciessampled" title="sec"> <div class="title" tagx="title" title="title">
Species sampled</div> <p>We used samples from wild animals. Tissues from the walrus were collected during aboriginal quota sealing in the coastal waters of Bering Sea (Mechigmen bay, Chukotka Autonomous Okrug, Russian Federation). Flapper and ear skin biopsy samples from Steller sea lions were collected on the Tyuleniy Island (the Sea of Okhotsk, Russia) when pup’s were branded. Isoflurane was used for inhalational anaesthesia. Tissue samples of the Baikal seal (Lake Baikal, Barguzin bay, Russian Federation) were taken from the male killed by fishnets and reported for the scientific research. All samples were collected according to procedures approved by the Committee on the Ethics of Animal Experiments of the Institute of Molecular and Cellular Biology SB RAS. The information about species used in this study is listed in <a href="#pone.0147647.t001">Table 1</a>.</p> <div class="table-wrap_UNKNOWN" id="pone.0147647.t001" orientation="portrait" position="float"><a href="https://dx.doi.org/10.1371/journal.pone.0147647.t001">10.1371/journal.pone.0147647.t001</a><span class="label" tagx="label" title="label">Table 1</span><div class="alternatives" title="alternatives"> <div class="graphic" id="pone.0147647.t001g" /> <table frame="hsides" rules="groups"> <colgroup span="1"> <col align="left" valign="middle" span="1" /></colgroup> <colgroup><col align="left" valign="middle" span="1" /></colgroup> <colgroup><col align="left" valign="middle" span="1" /></colgroup> <colgroup><col align="left" valign="middle" span="1" /></colgroup> <colgroup><col align="left" valign="middle" span="1" /></colgroup> <colgroup><col align="left" valign="middle" span="1" /></colgroup>  <thead> <tr> <th align="center" rowspan="1" colspan="1">No</th> <th align="center" rowspan="1" colspan="1">Latin names</th> <th align="center" rowspan="1" colspan="1">Codes</th> <th align="center" rowspan="1" colspan="1">2N</th> <th align="center" rowspan="1" colspan="1">Sex</th> <th align="center" rowspan="1" colspan="1">Common names</th> </tr> </thead> <tbody> <tr> <td align="center" rowspan="1" colspan="1">1</td> <td align="left" rowspan="1" colspan="1"><i>Odobenus rosmarus</i></td> <td align="center" rowspan="1" colspan="1">OROS</td> <td align="center" rowspan="1" colspan="1">32</td> <td align="center" rowspan="1" colspan="1">M</td> <td align="center" rowspan="1" colspan="1">Walrus</td> </tr> <tr> <td align="center" rowspan="1" colspan="1">2</td> <td align="left" rowspan="1" colspan="1"><i>Eumetopias jubatus</i></td> <td align="center" rowspan="1" colspan="1">EJUB</td> <td align="center" rowspan="1" colspan="1">36</td> <td align="center" rowspan="1" colspan="1">M</td> <td align="center" rowspan="1" colspan="1">Northern sea lion</td> </tr> <tr> <td align="center" rowspan="1" colspan="1">3</td> <td align="left" rowspan="1" colspan="1"><i>Pusa sibirica</i></td> <td align="center" rowspan="1" colspan="1">PSIB</td> <td align="center" rowspan="1" colspan="1">32</td> <td align="center" rowspan="1" colspan="1">M</td> <td align="center" rowspan="1" colspan="1">Baikal seal</td> </tr> <tr> <td align="center" rowspan="1" colspan="1">4</td> <td align="left" rowspan="1" colspan="1"><i>Canis familiaris</i></td> <td align="center" rowspan="1" colspan="1">CFA</td> <td align="center" rowspan="1" colspan="1">78</td> <td align="center" rowspan="1" colspan="1">M</td> <td align="left" rowspan="1" colspan="1">Domestic dog</td> </tr> <tr> <td align="center" rowspan="1" colspan="1">5</td> <td align="left" rowspan="1" colspan="1"><i>Martes foina</i></td> <td align="center" rowspan="1" colspan="1">MFO</td> <td align="center" rowspan="1" colspan="1">38</td> <td align="center" rowspan="1" colspan="1">M</td> <td align="center" rowspan="1" colspan="1">Stone marten</td> </tr> <tr> <td align="center" rowspan="1" colspan="1">6</td> <td align="left" rowspan="1" colspan="1"><i>Homo sapiens</i></td> <td align="center" rowspan="1" colspan="1">HSA</td> <td align="center" rowspan="1" colspan="1">46</td> <td align="center" rowspan="1" colspan="1">M</td> <td align="center" rowspan="1" colspan="1">Human</td> </tr> </tbody> </table> </div> </div> </div> <div class="storageandtransportationoftissuesamples" title="sec"> <div class="title" tagx="title" title="title">
Storage and transportation of tissue samples</div> <p>The lack of cytogenetic data for wild marine mammals and the limited number of available primary cell lines may be explained in part by the remoteness of their habitats and so by difficulties of collecting and delivering aseptic viable tissue samples to the laboratories. While it took 10–14 days to transport specimens to the laboratory from remote areas of collection additional efforts to prevent bacterial and mycotic contamination were undertaken. Upon collection all tissue pieces were thoroughly washed and scrubbed in continuous cool water flow to remove dirt and sand. Then samples were incubated for 20–30 min in growth medium with 5 times excess of antibiotic mixture (gentamicin sulphate+ampicillin+amphotericin b), followed by the incubation in medium with 3 times excess of antibiotics. After the antibiotic/antimycotic treatment all samples were placed into individual 5ml plastic tubes containing sterile transporting medium (growth medium αMEM (Gibco) with 15% bovine calf serum (HyClone), gentamicin sulphate 50mg/L, ampicillin 100 mg/L and amphotericin b 2.5mg/L). Forceps and scissors were washed in soapy water, rinsed in water, then soaked in 96% ethanol and flame-sterilised over the alcohol burner before manipulating each sample. Tubes with such processed samples were kept in the refrigerator at a temperature +4–10°C. Every time (approximately once in two or three day) when the medium in tubes was acidifying and changing the colour samples were moved into new sterile tubes containing fresh transporting medium using aseptic precautions and flame-sterilized tools.</p> </div> <div class="chromosomepreparationandbandingtechniques,chromosomenomenclature" title="sec"> <div class="title" tagx="title" title="title">
Chromosome preparation and banding techniques, chromosome nomenclature</div> <p>The primary fibroblast cell lines were derived from biopsies of ear, lung, sinew or flapper tissues of <i>O</i>. <i>rosmarus</i> (OROS), <i>E</i>. <i>jubatus</i> (EJUB) and <i>P</i>. <i>sibirica</i> (PSIB) using conventional techniques [<a href="#pone.0147647.ref031">31</a>]. Metaphase preparations were made as described earlier [<a href="#pone.0147647.ref022">22</a>,<a href="#pone.0147647.ref032">32</a>,<a href="#pone.0147647.ref033">33</a>]. Standard G-banding staining was performed according to [<a href="#pone.0147647.ref034">34</a>] method. Chromosomes of OROS, EJUB and PSIB were arranged by length.</p> </div> <div class="chromosome-specificpaintingprobespreparationandcharacterization" title="sec"> <div class="title" tagx="title" title="title">
Chromosome-specific painting probes preparation and characterization</div> <p>Sets of human, domestic dog and stone marten chromosome-specific painting probes were described previously [<a href="#pone.0147647.ref022">22</a>,<a href="#pone.0147647.ref035">35</a>,<a href="#pone.0147647.ref036">36</a>]. In this investigation we used the dog chromosomal nomenclatures published by [<a href="#pone.0147647.ref022">22</a>]. Whole-chromosome dog library was used for FISH on <i>O</i>. <i>rosmarus</i> (OROS), <i>E</i>. <i>jubatus</i> (EJUB) and <i>P</i>. <i>sibirica</i> (PSIB) genomes. Painting with human probes was done on EJUB and PSIB chromosomes.</p> </div> <div class="imagecaptureanddataprocessing" title="sec"> <div class="title" tagx="title" title="title">
Image capture and data processing</div> <p>Digital images of the hybridization signals were captured as described by [<a href="#pone.0147647.ref022">22</a>,<a href="#pone.0147647.ref033">33</a>,<a href="#pone.0147647.ref036">36</a>] using the VideoTest system (St.- Petersburg) with a CCD camera (Jenoptic) mounted on a Zeiss microscope Axioscope 2. Metaphase spreads images were edited in Corel Paint Shop Pro Photo X2.</p> </div> </div> <div class="results" title="results"> <div class="title" tagx="title" title="title">
Results</div> <div class="hybridizationofdogprobesontochromosomesofthreepinnipedspecies" title="sec"> <div class="title" tagx="title" title="title">
Hybridization of dog probes onto chromosomes of three pinniped species</div> <p>Hybridization examples of CFA probes are shown in <a href="#pone.0147647.g001">Fig 1A–1C</a>. Each CFA probe highlighted 1 to 5 fragments in the karyotypes of walrus, Steller sea lion and the Baikal seal. The CFA autosomal probes detected 68 homologous chromosomal segments in the genomes of investigated pinniped species. Twelve autosomes and X-chromosome are conserved <i>in toto</i> in the studied karyotypes. Chromosome maps of OROS, EJUB and PSIB are presented in Figs <a href="#pone.0147647.g002">2</a>, <a href="#pone.0147647.g003">3</a> and <a href="#pone.0147647.g004">4</a> respectively. The homology to chromosomal segments of MFO is based on a published comparative chromosome data between MFO and CFA [<a href="#pone.0147647.ref013">13</a>] and shown to the right of pinniped chromosomes.</p> <div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/10.1371/journal.pone.0147647.g001"><span class="label" tagx="label" title="label">Fig 1</span></a></div>   <div class="title" tagx="title" title="title">
Examples of localization of dog (A-C) and human (D-F) painting probes with GTG-banding of the same metaphase to the right.</div> <p>(A) CFA1/26 on EJUB chromosomes; (B) CFA 21+23/18 on OROS chromosomes; (C) CFA16/21+28 on PSIB chromosomes; (D) HSA 19/3 on OROS chromosomes; (E) HSA12/16 on PSIB chromosomes; (F) HSA7/1 on OROS chromosomes.</p>   </div> <div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/10.1371/journal.pone.0147647.g002"><span class="label" tagx="label" title="label">Fig 2</span></a></div>   <div class="title" tagx="title" title="title">
GTG-banded karyotype of walrus (<i>Odobenus rosmarus</i>, 2n = 32) with the assignment of homology to human (HSA), dog (CFA) and stone marten (MFO) chromosomes.</div> <p>The square denotes the centromere position on corresponding chromosome. *—segments were not painted by any dog probe.</p>   </div> <div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/10.1371/journal.pone.0147647.g003"><span class="label" tagx="label" title="label">Fig 3</span></a></div>   <div class="title" tagx="title" title="title">
GTG-banded karyotype of the Steller sea lion (<i>Eumetopias jubatus</i>, 2n = 36) with the assignment of homology to human (HSA), dog (CFA) and stone marten (MFO) chromosomes.</div> <p>The square denotes the centromere position on corresponding chromosome. *—segments were not painted by any dog probe.</p>   </div> <div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/10.1371/journal.pone.0147647.g004"><span class="label" tagx="label" title="label">Fig 4</span></a></div>   <div class="title" tagx="title" title="title">
GTG-banded karyotype of the Baikal seal (<i>Pusa sibirica</i>, 2n = 32) with the assignment of homology to human (HSA), dog (CFA) and stone marten (MFO) chromosomes.</div> <p>The square denotes the centromere position on corresponding chromosome. *—segments were not painted by any dog probe.</p>   </div> </div> <div class="mappingofhumanprobesontochromosomesofthewalrus,stellersealionandthebaikalseal" title="sec"> <div class="title" tagx="title" title="title">
Mapping of human probes onto chromosomes of the walrus, Steller sea lion and the Baikal seal</div> <p>The 22 HSA whole autosomal probes revealed 32 homologous chromosomal segments in the genomes of EJUB, OROS and PSIB. Patterns of chromosome painting with human chromosome-specific painting-probes are demonstrated in <a href="#pone.0147647.g001">Fig 1D–1F</a>. Each HSA probe painted 1 or 2 fragments in the karyotypes of studied species. Human chromosome probes HSA 6, 11 and X each delineated a complete chromosome in the walrus, Steller sea lion and the Baikal seal karyotypes. Probes containing HSA 5 and 16 each detected an entire chromosome in the EJUB chromosomal set, HSA 9 painted a whole chromosome in EJUB and PSIB. 10 chromosome breaks, one inversion and 15 (for EJUB) or 16 (for PSIB and OROS) fusions are required to convert the human karyotype into the pinniped karyotypes.</p> </div> <div class="mappingofselectedstonemartenpaintingprobesontochromosomesofpinnipedspecies" title="sec"> <div class="title" tagx="title" title="title">
Mapping of selected stone marten painting probes onto chromosomes of pinniped species</div> <p>Stone marten chromosomes 15 and 5 were mapped onto metaphases of the Baikal seal, probe combinations of MFO 14 /15 and 12/5 were mapped onto the walrus metaphases to clarify arrangement of ancestral carnivore syntenic elements ambiguous after analyses of human and dog painting probe localizations.</p> </div> </div> <div class="conclusions" title="conclusions"> <div class="title" tagx="title" title="title">
Discussion</div> <p>There were many attempts to shift into aquatic or semi-aquatic lifestyle in different mammalian orders—Cetartiodactyla (whales, dolphins, porpoises), Afrotheria (manatee, dugong), Carnivora (pinnipeds, otters, polar bear), Rodentia (beavers, water vole and others). Pinnipeds hunt, sleep and spend most of their life in the water but come out to bear their pups. The question about genetic impact of the unique position of pinnipeds as marine semi-aquatic animals among mostly terrestrial Carnivora is very intriguing. Surprisingly such drastic change in the mode of life is not accompanied by critical changes at the genetic level. Adaptations to aquatic environment are reflected in amino-acid substitutions of positively selected genomic regions found in pinnipeds and convergent with other marine mammals (whale and manatee) and yet accompanied by remarkably low substitution rate in mutation prone CpG islands [<a href="#pone.0147647.ref037">37</a>]. There is also indication of low level of variation in skull morphology between pinnipeds and fissipeds [<a href="#pone.0147647.ref038">38</a>]. These modern findings reflect on the old hypothesis of chromosomal conservatism in marine mammals [<a href="#pone.0147647.ref039">39</a>] that states "speciation caused by chromosomal rearrangements is rare within mammalian orders characterized by low reproduction, good mobility and continuous range of distribution".</p> <div class="chromosomeevolutioninpinnipedia" title="sec"> <div class="title" tagx="title" title="title">
Chromosome evolution in Pinnipedia</div> <p>The concept of high degree of chromosome conservation in pinnipeds is based on the classical cytogenetic data that have revealed similar diploid numbers (32–36) across this group of semi-aquatic mammals and showed high similarity of G-banding pattern with other mammals. It is supported by painting findings of the low numbers of conservative ancestral elements in harbour seal compared to other mammals [<a href="#pone.0147647.ref008">8</a>–<a href="#pone.0147647.ref011">11</a>,<a href="#pone.0147647.ref028">28</a>]. Our data provide further support documenting remarkable chromosome conservation across all families of pinnipeds. 22 human autosome painting probes delineated 32 conservative autosome segments in the genomes of three studied species. This corresponds to assessment of their genomes as low-rearranged relatively to the ancestral eutherian genome. Similarly low number of autosome segments was detected by human painting probes in the conservative karyotypes of musteloids, felids, hyenas and some viverrids [<a href="#pone.0147647.ref013">13</a>,<a href="#pone.0147647.ref040">40</a>,<a href="#pone.0147647.ref041">41</a>] and conserved karyotypes of other mammals like aardvark and golden mole (30). Much higher numbers are found in other mammals (47 for closely related pangolins, 40–50 in cetartiodactyls and perissodactyls, 42 in bats) or even highly rearranged Carnivora karyotypes (67–73 in canids, 44 in ursids, 40 in skunks) indicating higher divergence from ancestral eutherian karyotype. Chromosome painting in pinnipeds here confirms long-standing hypothesis about karyotype conservatism in marine animals. The underlying reasons for this phenomenon were discussed and include low degree of inbreeding, low reproductivity rate and absence of physical geographic barriers [<a href="#pone.0147647.ref039">39</a>]. Other marine animals with exception of afrotherian manatee (41) also have low number of conservative ancestral units: otter (32), pilot whale (34), finless porpoise (34) and bottlenose dolphin (30). Latter three species belong to the order Cetartiodactyla and at the same time other members of the order have over 45 conservative chromosome segments [<a href="#pone.0147647.ref013">13</a>,<a href="#pone.0147647.ref033">33</a>,<a href="#pone.0147647.ref042">42</a>–<a href="#pone.0147647.ref044">44</a>]. Both groups—Cetacea and Pinnipedia are rather specious (with more than 80 and 32 species) demonstrating that conservative karyotype does not impede the speciation and that the species radiation is not always accompanied by apparent chromosome rearrangements.</p> <p>We revealed in the genomes of three studied pinnipeds conserved syntenic segment associations common for the whole clade Eutheria (HSA 3/21, 4/8, 7/16, 10/12/22, 14/15 and 16/19; [<a href="#pone.0147647.ref024">24</a>,<a href="#pone.0147647.ref029">29</a>] (<a href="#pone.0147647.t002">Table 2</a>). In total pinniped karyotypes differ from ancestral Eutherian karyotype [<a href="#pone.0147647.ref045">45</a>,<a href="#pone.0147647.ref046">46</a>] by 5 fusions (HSA1/8, 2/20, 2/13, 19+3/21, 12/22+18)) and one fission (HSA1). Four syntenic segment associations (HSA 1/8, 2/13, 2/20, 19+3/21) are characteristic for the whole order Carnivora with few exceptions [<a href="#pone.0147647.ref014">14</a>] and represent elements of ancestral Carnivora karyotype with 2n = 38 [<a href="#pone.0147647.ref047">47</a>]. Synteny HSA 12/22/18 was suggested to be a cytogenetic character linking Carnivora and Cetartiodactyla [<a href="#pone.0147647.ref013">13</a>,<a href="#pone.0147647.ref042">42</a>,<a href="#pone.0147647.ref044">44</a>]. It was also suggested that there is an association HSA 1q/10q (with HSA1q being really small size—HSA1q42.1–q43) that is common for all Boreutherian mammals [<a href="#pone.0147647.ref048">48</a>]. However we did not see the segment of 1q on the corresponding pinniped chromosomes (OROS1, EJUB11, PSIB7) and it is likely that the size of this HSA1q segment is at the threshold of chromosome painting resolution. There is no common cytogenetic trait to distinguish all pinnipeds from the rest of Carnivora and to provide cytogenetic link for monophyletic origin of this group versus once suggested diphyly [<a href="#pone.0147647.ref005">5</a>].</p> <div class="table-wrap_UNKNOWN" id="pone.0147647.t002" orientation="portrait" position="float"><a href="https://dx.doi.org/10.1371/journal.pone.0147647.t002">10.1371/journal.pone.0147647.t002</a><span class="label" tagx="label" title="label">Table 2</span> <div class="title" tagx="title" title="title">
Correspondence of pinniped (the Steller sea lion, the walrus, the Baikal seal and the common seal<a href="#t002fn002"><sup>1</sup></a>) chromosomes with dog, human, ancestral carnivore karyotype (ACK<a href="#t002fn003"><sup>2</sup></a>), stone marten<a href="#t002fn004"><sup>3</sup></a> and cat<a href="#t002fn004"><sup>3</sup></a> chromosomes.</div>  <div class="alternatives" title="alternatives"> <div class="graphic" id="pone.0147647.t002g" /> <table frame="hsides" rules="groups"> <colgroup span="1"> <col align="left" valign="middle" span="1" /></colgroup> <colgroup><col align="left" valign="middle" span="1" /></colgroup> <colgroup><col align="left" valign="middle" span="1" /></colgroup> <colgroup><col align="left" valign="middle" span="1" /></colgroup> <colgroup><col align="left" valign="middle" span="1" /></colgroup> <colgroup><col align="left" valign="middle" span="1" /></colgroup> <colgroup><col align="left" valign="middle" span="1" /></colgroup> <colgroup><col align="left" valign="middle" span="1" /></colgroup> <colgroup><col align="left" valign="middle" span="1" /></colgroup>  <thead> <tr> <th align="center" rowspan="1" colspan="1">EJUB</th> <th align="center" rowspan="1" colspan="1">OROS</th> <th align="center" rowspan="1" colspan="1">PSIB</th> <th align="center" rowspan="1" colspan="1">PVIT</th> <th align="center" rowspan="1" colspan="1">CFA</th> <th align="center" rowspan="1" colspan="1">HSA</th> <th align="center" rowspan="1" colspan="1">ACK</th> <th align="center" rowspan="1" colspan="1">MFO</th> <th align="center" rowspan="1" colspan="1">FCA</th> </tr> </thead> <tbody> <tr> <td align="center" rowspan="1" colspan="1">1</td> <td align="center" rowspan="1" colspan="1">2</td> <td align="center" rowspan="1" colspan="1">2</td> <td align="center" rowspan="1" colspan="1">sm1</td> <td align="center" rowspan="1" colspan="1">20/23/35/30/27</td> <td align="center" rowspan="1" colspan="1">19p/3/21</td> <td align="center" rowspan="1" colspan="1">1</td> <td align="center" rowspan="1" colspan="1">2</td> <td align="center" rowspan="1" colspan="1">A2p/C2</td> </tr> <tr> <td align="center" rowspan="1" colspan="1">2</td> <td align="center" rowspan="1" colspan="1">4</td> <td align="center" rowspan="1" colspan="1">4</td> <td align="center" rowspan="1" colspan="1">m1</td> <td align="center" rowspan="1" colspan="1">22/28/19/36/33/28</td> <td align="center" rowspan="1" colspan="1">13/2q</td> <td align="center" rowspan="1" colspan="1">3</td> <td align="center" rowspan="1" colspan="1">3</td> <td align="center" rowspan="1" colspan="1">A1p/C1q</td> </tr> <tr> <td align="center" rowspan="1" colspan="1">3</td> <td align="center" rowspan="1" colspan="1">3</td> <td align="center" rowspan="1" colspan="1">3</td> <td align="center" rowspan="1" colspan="1">sm2</td> <td align="center" rowspan="1" colspan="1">16/28/15/19/32/13/3</td> <td align="center" rowspan="1" colspan="1">4q/8p/4p+q</td> <td align="center" rowspan="1" colspan="1">2</td> <td align="center" rowspan="1" colspan="1">1</td> <td align="center" rowspan="1" colspan="1">B1</td> </tr> <tr> <td align="center" rowspan="1" colspan="1">4</td> <td align="center" rowspan="1" colspan="1">6</td> <td align="center" rowspan="1" colspan="1">5</td> <td align="center" rowspan="1" colspan="1">m2</td> <td align="center" rowspan="1" colspan="1">13/34/17/6/5/15/2/5</td> <td align="center" rowspan="1" colspan="1">8q/1p+q</td> <td align="center" rowspan="1" colspan="1">4</td> <td align="center" rowspan="1" colspan="1">4</td> <td align="center" rowspan="1" colspan="1">F2/C1p</td> </tr> <tr> <td align="center" rowspan="1" colspan="1">5</td> <td align="center" rowspan="1" colspan="1">1q</td> <td align="center" rowspan="1" colspan="1">1q</td> <td align="center" rowspan="1" colspan="1">sq</td> <td align="center" rowspan="1" colspan="1">11/2/3/4/35</td> <td align="center" rowspan="1" colspan="1">5</td> <td align="center" rowspan="1" colspan="1">5</td> <td align="center" rowspan="1" colspan="1">5</td> <td align="center" rowspan="1" colspan="1">A1q</td> </tr> <tr> <td align="center" rowspan="1" colspan="1">6</td> <td align="center" rowspan="1" colspan="1">8</td> <td align="center" rowspan="1" colspan="1">6pprx</td> <td align="center" rowspan="1" colspan="1">m3pprx</td> <td align="center" rowspan="1" colspan="1">2/29/10/15/10</td> <td align="center" rowspan="1" colspan="1">10p/12/22</td> <td align="center" rowspan="1" colspan="1">8</td> <td align="center" rowspan="1" colspan="1">9</td> <td align="center" rowspan="1" colspan="1">B4</td> </tr> <tr> <td align="center" rowspan="1" colspan="1">7</td> <td align="center" rowspan="1" colspan="1">10</td> <td align="center" rowspan="1" colspan="1">10</td> <td align="center" rowspan="1" colspan="1">m5p</td> <td align="center" rowspan="1" colspan="1">28/24/10/17</td> <td align="center" rowspan="1" colspan="1">20/2p</td> <td align="center" rowspan="1" colspan="1">9</td> <td align="center" rowspan="1" colspan="1">8</td> <td align="center" rowspan="1" colspan="1">A3</td> </tr> <tr> <td align="center" rowspan="1" colspan="1">8</td> <td align="center" rowspan="1" colspan="1">7</td> <td align="center" rowspan="1" colspan="1">9</td> <td align="center" rowspan="1" colspan="1">sm4</td> <td align="center" rowspan="1" colspan="1">3/25/8</td> <td align="center" rowspan="1" colspan="1">15/14</td> <td align="center" rowspan="1" colspan="1">7</td> <td align="center" rowspan="1" colspan="1">7</td> <td align="center" rowspan="1" colspan="1">B3</td> </tr> <tr> <td align="center" rowspan="1" colspan="1">9</td> <td align="center" rowspan="1" colspan="1">9</td> <td align="center" rowspan="1" colspan="1">8</td> <td align="center" rowspan="1" colspan="1">sm3</td> <td align="center" rowspan="1" colspan="1">37/12/1</td> <td align="center" rowspan="1" colspan="1">6</td> <td align="center" rowspan="1" colspan="1">6</td> <td align="center" rowspan="1" colspan="1">6</td> <td align="center" rowspan="1" colspan="1">B2</td> </tr> <tr> <td align="center" rowspan="1" colspan="1">10p</td> <td align="center" rowspan="1" colspan="1">11p</td> <td align="center" rowspan="1" colspan="1">6pdist</td> <td align="center" rowspan="1" colspan="1">m3pdist</td> <td align="center" rowspan="1" colspan="1">6</td> <td align="center" rowspan="1" colspan="1">7/16p</td> <td align="center" rowspan="1" colspan="1">18</td> <td align="center" rowspan="1" colspan="1">18</td> <td align="center" rowspan="1" colspan="1">E3</td> </tr> <tr> <td align="center" rowspan="1" colspan="1">10q</td> <td align="center" rowspan="1" colspan="1">11q</td> <td align="center" rowspan="1" colspan="1">7p</td> <td align="center" rowspan="1" colspan="1">m4p</td> <td align="center" rowspan="1" colspan="1">37/8</td> <td align="center" rowspan="1" colspan="1">1q</td> <td align="center" rowspan="1" colspan="1">16</td> <td align="center" rowspan="1" colspan="1">16</td> <td align="center" rowspan="1" colspan="1">F1</td> </tr> <tr> <td align="center" rowspan="1" colspan="1">11</td> <td align="center" rowspan="1" colspan="1">1p</td> <td align="center" rowspan="1" colspan="1">7q</td> <td align="center" rowspan="1" colspan="1">m4q</td> <td align="center" rowspan="1" colspan="1">4/31</td> <td align="center" rowspan="1" colspan="1">10q</td> <td align="center" rowspan="1" colspan="1">12</td> <td align="center" rowspan="1" colspan="1">12</td> <td align="center" rowspan="1" colspan="1">D2</td> </tr> <tr> <td align="center" rowspan="1" colspan="1">12</td> <td align="center" rowspan="1" colspan="1">13</td> <td align="center" rowspan="1" colspan="1">11</td> <td align="center" rowspan="1" colspan="1">m6</td> <td align="center" rowspan="1" colspan="1">5/21/18</td> <td align="center" rowspan="1" colspan="1">11</td> <td align="center" rowspan="1" colspan="1">11</td> <td align="center" rowspan="1" colspan="1">10</td> <td align="center" rowspan="1" colspan="1">D1</td> </tr> <tr> <td align="center" rowspan="1" colspan="1">13</td> <td align="center" rowspan="1" colspan="1">12</td> <td align="center" rowspan="1" colspan="1">12</td> <td align="center" rowspan="1" colspan="1">sm5</td> <td align="center" rowspan="1" colspan="1">18/14/16</td> <td align="center" rowspan="1" colspan="1">7</td> <td align="center" rowspan="1" colspan="1">10</td> <td align="center" rowspan="1" colspan="1">11</td> <td align="center" rowspan="1" colspan="1">A2q</td> </tr> <tr> <td align="center" rowspan="1" colspan="1">14</td> <td align="center" rowspan="1" colspan="1">14</td> <td align="center" rowspan="1" colspan="1">13</td> <td align="center" rowspan="1" colspan="1">m7</td> <td align="center" rowspan="1" colspan="1">26/7/1</td> <td align="center" rowspan="1" colspan="1">12/22/18</td> <td align="center" rowspan="1" colspan="1">13</td> <td align="center" rowspan="1" colspan="1">13</td> <td align="center" rowspan="1" colspan="1">D3</td> </tr> <tr> <td align="center" rowspan="1" colspan="1">15</td> <td align="center" rowspan="1" colspan="1">5q</td> <td align="center" rowspan="1" colspan="1">14</td> <td align="center" rowspan="1" colspan="1">m8</td> <td align="center" rowspan="1" colspan="1">1/11/9</td> <td align="center" rowspan="1" colspan="1">9</td> <td align="center" rowspan="1" colspan="1">14</td> <td align="center" rowspan="1" colspan="1">14</td> <td align="center" rowspan="1" colspan="1">D4</td> </tr> <tr> <td align="center" rowspan="1" colspan="1">16</td> <td align="center" rowspan="1" colspan="1">5p</td> <td align="center" rowspan="1" colspan="1">1p</td> <td align="center" rowspan="1" colspan="1">sp</td> <td align="center" rowspan="1" colspan="1">9/5</td> <td align="center" rowspan="1" colspan="1">17</td> <td align="center" rowspan="1" colspan="1">15</td> <td align="center" rowspan="1" colspan="1">15</td> <td align="center" rowspan="1" colspan="1">E1</td> </tr> <tr> <td align="center" rowspan="1" colspan="1">17</td> <td align="center" rowspan="1" colspan="1">15</td> <td align="center" rowspan="1" colspan="1">15</td> <td align="center" rowspan="1" colspan="1">m9</td> <td align="center" rowspan="1" colspan="1">1/2/5</td> <td align="center" rowspan="1" colspan="1">19q/16q</td> <td align="center" rowspan="1" colspan="1">17</td> <td align="center" rowspan="1" colspan="1">17</td> <td align="center" rowspan="1" colspan="1">E2</td> </tr> <tr> <td align="center" rowspan="1" colspan="1">X</td> <td align="center" rowspan="1" colspan="1">X</td> <td align="center" rowspan="1" colspan="1">X</td> <td align="center" rowspan="1" colspan="1">X</td> <td align="center" rowspan="1" colspan="1">X</td> <td align="center" rowspan="1" colspan="1">X</td> <td align="center" rowspan="1" colspan="1">X</td> <td align="center" rowspan="1" colspan="1">X</td> <td align="center" rowspan="1" colspan="1">X</td> </tr> </tbody> </table> </div> <div class="table-wrap-foot_UNKNOWN"> <div class="fn-type-" title=""> <p>Correspondence with human chromosomes is based on our FISH results, human chromosome segments are designated according to published data:</p> </div> <div class="fn-type-" title=""> <p><sup>1</sup>Frönicke et al. [<a href="#pone.0147647.ref028">28</a>],</p> </div> <div class="fn-type-" title=""> <p><sup>2</sup>Perelman et al. [<a href="#pone.0147647.ref014">14</a>],</p> </div> <div class="fn-type-" title=""> <p><sup>3</sup>Nie et al. [<a href="#pone.0147647.ref013">13</a>]).</p> </div> <div class="fn-type-" title=""> <p>Here we use dog chromosome nomenclature according to Yang et al., [<a href="#pone.0147647.ref022">22</a>]. dist—distal part of chromosome, prx—proximal part of chromosome.</p> </div> </div> </div> <p>Baikal seal is endemic species found only in Lake Baikal with its origin still being under discussion. Mitogenomic data indicate that it has separated from other <i>Pusa</i> species at least 4 MYA [<a href="#pone.0147647.ref049">49</a>]. Previously, only conventionally stained karyotype of <i>Pusa sibirica</i> was reported [<a href="#pone.0147647.ref050">50</a>]. Here we present G-banded karyotype of <i>Pusa sibirica</i> with established homologies to human and dog genomes. Zoo-FISH of harbour seal (<i>Phoca vitulina</i>, PVIT) with human whole-chromosome probes disclosed 30 autosomal conserved syntenic segments[<a href="#pone.0147647.ref028">28</a>] compared to 32 revealed in <i>Pusa sibirica</i>. High similarity of GTG-banding of <i>P</i>. <i>vitulina</i> [<a href="#pone.0147647.ref030">30</a>] and <i>P</i>. <i>sibirica</i> indicates that two conservative fragments were missed in the first analysis of pinniped by comparative chromosome painting: the distal part of p-arm of PVITm3 is homologous to HSA12 and the distal part of p-arm of PVITm7 is homologous to HSA7. It means that the genome of harbor seal contains syntenies homologous to HSA 7/16 and 12/22+18 and consequently there are 32 autosomal segments homologous to human conservative fragments in the genome of PVIT likewise in the genome of PSIB. Otherwise comparison of G-banding and human probes painting patterns in <i>Phoca vitulina</i> and <i>Pusa sibirica</i> karyotypes here did not reveal any differences showing high level of karyotype conservation during speciation in Phocinae. That proves that whole-chromosome probes would not be helpful to uncover the history of speciation in Phocinae to solve unsettled question about <i>Phoca/Pusa</i> division and indicates the necessity for further phylogenetic studies [<a href="#pone.0147647.ref051">51</a>].</p> <p>Karyotype of the walrus, the only living representative of Odobenidae, was described previously [<a href="#pone.0147647.ref010">10</a>]. Attempts to reveal rearrangements between walrus and other pinnipeds did not yield conclusive results[<a href="#pone.0147647.ref010">10</a>,<a href="#pone.0147647.ref052">52</a>]. Using combination of human and domestic dog painting probes of high power of resolution we revealed two fusions characteristic for Odobenidae: HSA 5/10 and 9/17 (ACK 5/12, 14/15) that had remained unresolved in studies of G-banded karyotypes of Otariidae and Odobenidae. Now with links of the walrus chromosomes to well-annotated human and dog genomes the chromosome assignment of existing walrus genome scaffolds may be deducted to provide valuable linkage information and to track evolutionary changes in chromosomes at the fine sequence level (<a href="http://www.ncbi.nlm.nih.gov/genome/14031">http://www.ncbi.nlm.nih.gov/genome/14031</a>).</p> <p>On the base of FISH data with dog painting probes we were able to detect fusions of conservative elements homologous to stone marten autosomes (<i>Martes foina</i>, MFO) [<a href="#pone.0147647.ref013">13</a>]. So we used some whole chromosome probes of MFO to check these assumptions and to reconstruct ancestral carnivore karyotype reorganization during formation of seal’s genomes. Here we apply the ACK chromosome numbers corresponding to [<a href="#pone.0147647.ref047">47</a>] nomenclature. Fusions homologous to HSA 5/17, 7/16+10+12/22 and 10/1 (ACK 5/15, 8/18 and 12/16) found in the Baikal seal genome are common with <i>Phoca vitulina</i> and are likely markers for the whole Phoca/Pusa branch. Fusion HSA 7/16+1 (ACK 16/18) is common for Steller sea lion and the walrus. This rearrangement common for monospecific Odobenidae and representative of Otariidae provides strong support for the growing molecular evidence in solving the long-standing puzzle of affinity of Odobenidae with either Otariidae or Phocidae [<a href="#pone.0147647.ref004">4</a>]. Incidentally this fusion is also found in bears with exception of giant panda [<a href="#pone.0147647.ref019">19</a>,<a href="#pone.0147647.ref023">23</a>,<a href="#pone.0147647.ref024">24</a>]. However the order of dog chromosome probes is different on this syntenic group (CFA7/38/6 in bears and CFA6/7/38 in walrus and sea lion) and most likely indicates hemiplasic nature of this fusion rather than common chromosomal signature for Arctoidea (Ursidae+Pinnipedia +Musteloidea) or affinity of Ursidae and Pinnipedia [<a href="#pone.0147647.ref002">2</a>,<a href="#pone.0147647.ref006">6</a>,<a href="#pone.0147647.ref053">53</a>,<a href="#pone.0147647.ref054">54</a>] (<a href="#pone.0147647.g005">Fig 5</a>). The genome of Steller sea lion retained ancestral state and differs from Ancestral Musteloid Karyotype [<a href="#pone.0147647.ref033">33</a>] by only one fusion ACK 16/18 (HSA 7/16+1). This agrees with conclusion of the comparative G-banding analyses that otariid karyotype retained the most primitive features and strikingly resembles conservative procyonid karyotypes [<a href="#pone.0147647.ref010">10</a>]. Chromosome painting here precisely showed rearrangements separating pinniped karyotypes erasing uncertainties of sole G-banding comparisons and revealed characteristic chromosomal markers for branching order in Pinnipedia.</p> <div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/10.1371/journal.pone.0147647.g005"><span class="label" tagx="label" title="label">Fig 5</span></a></div>   <div class="title" tagx="title" title="title">
Karyotype evolution pathways in Pinnipedia.</div>   </div> </div> <div class="refinementofancestralcarnivorekaryotypebypinnipedpaintingdata" title="sec"> <div class="title" tagx="title" title="title">
Refinement of Ancestral Carnivore Karyotype by pinniped painting data</div> <p>First ancestral carnivore karyotype reconstruction was done by Wurster-Hill and Gray based on comparative chromosome analyses of G-banded karyotypes of large number of carnivore species including available data for pinnipeds [<a href="#pone.0147647.ref009">9</a>,<a href="#pone.0147647.ref026">26</a>]. It was called "hypothetical primitive karyotype" and suggested to contain at least 16 autosomes with minimum fundamental number of 58. It was notable attempt to reconstruct for the first time ancestral karyotype for the whole mammalian order that turned successful owing to remarkably conservative chromosome sets in both carnivore branches of feloids (Felidae, Viverridae) and canoids (Procyonidae, Pinnipedia). Appearance of the gene mapping data for human and cat led to the next variant of ancestral karyotype called CAR that contained 21–22 autosome pairs with FN = 60–68 [<a href="#pone.0147647.ref027">27</a>,<a href="#pone.0147647.ref055">55</a>] and was importantly shown to be very close to the primate ancestral karyotype. Extensive FISH data provided reliable base for new reconstructions of ACK. Yet lack of painting data for several key branches of Carnivora (Otariidae, Odobenidae, Prionodontidae, Nanndiniidae) contributes to some uncertainties in ACK.</p> <p>Pinnipedia occupy unique position among canoids: if families with drastically rearranged karyotypes (Canidae, Ursidae, Mephitidae) are not taken into account then pinnipeds represent the most basal canoid branch and become important lineage for ancestral karyotype reconstruction. Based on chromosome painting data in Carnivora two versions of ACK differing by diploid numbers have been proposed: i.e. 2n = 42 [<a href="#pone.0147647.ref027">27</a>,<a href="#pone.0147647.ref029">29</a>] and 2n = 38 [<a href="#pone.0147647.ref028">28</a>,<a href="#pone.0147647.ref047">47</a>]. The differences are provided by fused or split status of autosome elements homologous to FCA A1p/ C1q and C1p/F2. Our data demonstrate that in karyotypes of three studied pinniped species these fragments are present as a joint element and this is an argument for the ACK diploid number 2n = 38 as it was first suggested in Z-CAR version of ancestral karyotype based on Phocidae painting. Also all pinnipeds have fused state for FCA A2p/C2, which, in contrast, is present as two separate pieces in all feloid species studied by chromosome painting so far. This fusion is ancestral for the whole order Carnivora.</p> <p>Refined version of ACK with 2n = 38 that includes painting data of pinniped karyotypes is presented on <a href="#pone.0147647.g006">Fig 6</a>. The mapping of dog painting probes in three pinniped species provides firm evidence toward plausible order of dog chromosome syntenic segments on ancestral chromosomes. The number of conserved segments of dog autosomes is 68. New painting data bring several changes into our previous version of ACK [<a href="#pone.0147647.ref013">13</a>,<a href="#pone.0147647.ref014">14</a>]. We have found additional fragment corresponding to CFA28 that have not been seen in previous painting studies. In our FISH experiments we used mixed painting probes containing CFA21+28 and CFA21+23. The probe CFA21+23 marked two fragments in the genomes of OROS, EJUB, PSIB, each autosome delineated single segment. The probe containing two dog chromosomes CFA21+28 detected five conservative segments in karyotypes of three studied species: if CFA21 delineates one fragment then CFA28 detects four fragments (<a href="#pone.0147647.g001">Fig 1a</a>). It means that besides three fragments on ACK 2q, ACK 3p and ACK 3q the probe containing CFA28 paints the forth conservative elements on the distal part of OROS 10p, EJUB 7p and PSIB 10p corresponding to ACK 9p dist (CFA28/24/10/17 = HSA20/2p = MFO8 = FCAA3 = ACK9). These chromosomes of the walrus, Steller sea lion and the Baikal seal have very similar banding patterns aligning with homologous chromosomes of cat, ringtail, dwarf mongoose and the Malagasy civet [<a href="#pone.0147647.ref047">47</a>]. It is possible that the fourth conservative fragment of CFA28 was not noticed before due to the small size. There is a gap indicated on the comparative map of <i>H</i>. <i>malayanus</i> (Ursidae) on the chromosome 13 that likely contains this small corresponding fragment homologous to CFA28 [<a href="#pone.0147647.ref023">23</a>]. We have verified the presence of this fragment in <i>Bassariscus astutus</i> (Procyonidae) by painting of corresponding dog probes and confirmed it. Additional experiments are required to confirm the presence of the fragment in other carnivore families. This additional CFA28 segment, however, it is not reflected in Ensemble synteny data.</p> <div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/10.1371/journal.pone.0147647.g006"><span class="label" tagx="label" title="label">Fig 6</span></a></div>   <div class="title" tagx="title" title="title">
Ancestral carnivore karyotype (ACK).</div> <p>Numbers on the left are HSA and CFA chromosomes. Numbers on the right of each ancestral chromosome are ACK chromosomes according to nomenclature of Murphy et al [<a href="#pone.0147647.ref026">26</a>] followed by cat (FCA) homology [<a href="#pone.0147647.ref013">13</a>]. MFO chromosome numbers are indicated right below the ancestral chromosome. Big numerals below correspond to Nash et al. [<a href="#pone.0147647.ref047">47</a>] ACK nomenclature. The proposed ancestral order of syntenic segments corresponding to dog chromosomes is based on parsimony analyses of chromosome painting data of canid painting probes in Carnivora and two outgroup species: pig and human, [<a href="#pone.0147647.ref018">18</a>,<a href="#pone.0147647.ref019">19</a>,<a href="#pone.0147647.ref053">53</a>]. We exchanged ACK11 and ACK12 relative to ACK published in [<a href="#pone.0147647.ref011">11</a>] to match chromosome order [<a href="#pone.0147647.ref047">47</a>] and to be arranged according to the chromosome size in cat and stone marten karyotypes.</p>   </div> <p>The second change in the current ACK structure [<a href="#pone.0147647.ref014">14</a>] concerns conservative segment homologous to CFA26. This conservative segment is present in Ensemble synteny data based on whole-genome sequence alignments for HSA 10 but was not detected by painting earlier (<a href="http://www.ensembl.org/Canis_familiaris/Location/Synteny?r=26:30309989-30409989">http://www.ensembl.org/Canis_familiaris/Location/Synteny?r=26:30309989-30409989</a>). Painting probe containing this autosome detected only one fragment corresponding to ACK13 (HSA 12/22, FCA D3p) in the karyotypes of three studied species and did not reveal the second fragment of CFA26b located on ACK11 (HSA 10, FCA D2p). It remains to be answered why such large fragment (size is about 9 Mb) is not revealed by painting. For now based on the painting data we do not include this segment onto the ACK map.</p> <p>Based on the analysis of G-banded pinniped karyotypes Nash [<a href="#pone.0147647.ref052">52</a>] correctly predicted that fusion FCA F1/D2 (ACK 16/12) is present in Phocidae and absent in Otariidae, and FCA E3 is fused with B4 (ACK 18/8) in Phocidae and with F1 in Otariidae and Odobenidae (ACK 18/16). Our comparison of G-banded chromosomes of OROS, EJUB and PSIB with proposed G-banded ACK [<a href="#pone.0147647.ref047">47</a>] confirms that pinniped karyotypes share conservative banding patterns with other carnivore families that retained in whole ancestral karyotype of the order (<a href="#pone.0147647.g006">Fig 6</a>).</p> <p>Does strict conservation at the level of the chromosome numbers and large chromosome segments in pinnipeds suggest some evolutionary changes at the intrachromosome level? Dog and raccoon dog chromosome painting probes are a good tool for research of intrachromosome structure in Carnivora. Inversions were revealed in many carnivore families studied with canid painting probes [<a href="#pone.0147647.ref013">13</a>]. In contrast we did not find any inversions in studied pinniped species. The order of conservative syntenic segments homologous to dog autosomes on chromosomes of the walrus, Steller sea lion and the Baikal seal is identical and corresponds to the suggested sequence of conservative dog elements in the ACK [<a href="#pone.0147647.ref014">14</a>]. Thus karyotypic conservation in pinnipeds extends to the intrachromosomal level. The question remains whether the same level of conservation will be seen in comparative analyses of pinniped’s whole genome sequences.</p> <p>Genomes of Pinnipedia diverged from other Carnivora about 40 MYA with Odobenidae-Otariidae split happening at roughly 25 MYA [<a href="#pone.0147647.ref049">49</a>]. During this time only few karyotype rearrangements took place suggesting that the rate of genome evolution in pinnipeds is about one rearrangement per 10 million years. This corresponds to ancestral, or slow, pattern of eutherian genome rearrangements [<a href="#pone.0147647.ref029">29</a>].</p> </div> </div> <div class="conclusions" title="conclusions"> <div class="title" tagx="title" title="title">
Conclusions</div> <p>Comparative chromosome maps constructed for species from all three pinniped families (Otariidae, Odobenidae and Phocidae) with human, dog and stone marten painting probes verify G-banding data confirming hypothesis about low rate of karyotype evolution in pinnipeds. Our data demonstrate that only few tandem fusions and apparently no major inversions took place during formation of pinniped karyotypes. Phylogenetically comparative chromosome painting provides chromosomal signature for Odobenidae/Otariidae affinity. These results close a lacuna in painting data for two pinniped families and corroborate knowledge base about genome rearrangements of Carnivora as far as of all Eutheria providing strong support for presented version of Ancestral Carnivore Karyotype with 2n = 38. Still our knowledge about chromosomal and genome evolution in pinnipeds is limited and warrants for further painting experiments for other species from Otariidae and Phocidae subfamilies. The pinniped chromosome maps presented here provide robust platform for future anchoring of sequencing assemblies to chromosomes paving the way to analyses of evolution of unique adaptations to semi-aquatic lifestyle.</p> </div> </div> <div class="back" title="back"> <div class="ack" title="ack"> <p>Authors are thankful to Fengtang Yang, Malcolm A. Ferguson-Smith, Wenhui Nie for access to human, stone marten and dog flow-sorting chromosome libraries. Authors are grateful to Michael Zelensky, Alexey Ottoj and The Community of the Chukotka Autonomous Region indigenous "Lorino" (Russian Federation) for assistance in the walrus tissue sample collection. Authors are also thankful to Maria Chistjaeva for help in collecting biopsy probes from Steller sea lions. Authors express our gratitude to research vessel "Afina" crew for delivering to Tulenij island and to members of KB of Pacific Geographical Institute of FEB RAS and volunteers for assistance in collection of tissue samples from sea lions. Authors are grateful to Michael Grachev, Andrei Varnavsky and Vladimir Peterfeld for support in samples collection from the Baikal seal.</p> <p>The authors thank Ulfur Arnason and the second anonymous reviewer for usefull comments that greatly improved the manuscript.</p> </div> <div class="references">
References</div> <div tag="ref-list"> <ul> <div class="title" tagx="title" title="title">
References</div> <li tag="ref"><a name="pone.0147647.ref001" /><span class="label" tagx="label" title="label">1</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ferguson-Smith</span><span class="given-names" tagx="given-names" title="given-names">MA</span></span>. <span class="mixed-article-title" title="mixed-article-title">History and evolution of cytogenetics</span>. <span class="source" tagx="source" title="source">Mol Cytogenet</span>. <span class="year" tagx="year" title="year">2015</span>;<span class="volume" tagx="volume" title="volume">8</span>: <span class="fpage" tagx="fpage" title="fpage">19</span><span class="pub-id"><a href="https://dx.doi.org/10.1186/s13039-015-0125-8">10.1186/s13039-015-0125-8</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25810762">25810762</a></span></span></li> <li tag="ref"><a name="pone.0147647.ref002" /><span class="label" tagx="label" title="label">2</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Arnason</span><span class="given-names" tagx="given-names" title="given-names">U</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gullberg</span><span class="given-names" tagx="given-names" title="given-names">A</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Janke</span><span class="given-names" tagx="given-names" title="given-names">A</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kullberg</span><span class="given-names" tagx="given-names" title="given-names">M</span></span>. <span class="mixed-article-title" title="mixed-article-title">Mitogenomic analyses of caniform relationships</span>. <span class="source" tagx="source" title="source">Mol Phylogenet Evol</span>. <span class="year" tagx="year" title="year">2007</span>;<span class="volume" tagx="volume" title="volume">45</span>: <span class="fpage" tagx="fpage" title="fpage">863</span>–<span class="lpage" tagx="lpage" title="lpage">874</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1016/j.ympev.2007.06.019">10.1016/j.ympev.2007.06.019</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17919938">17919938</a></span></span></li> <li tag="ref"><a name="pone.0147647.ref003" /><span class="label" tagx="label" title="label">3</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Nowak</span><span class="given-names" tagx="given-names" title="given-names">RM</span></span>. <span class="source" tagx="source" title="source">Walker’s Mammals of the World</span>. <span class="publisher-name" tagx="publisher-name" title="publisher-name">JHU Press</span>; <span class="year" tagx="year" title="year">1999</span>.</span></li> <li tag="ref"><a name="pone.0147647.ref004" /><span class="label" tagx="label" title="label">4</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bininda-Emonds</span><span class="given-names" tagx="given-names" title="given-names">OR</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gittleman</span><span class="given-names" tagx="given-names" title="given-names">JL</span></span>. <span class="mixed-article-title" title="mixed-article-title">Are pinnipeds functionally different from fissiped carnivores? The importance of phylogenetic comparative analyses</span>. <span class="source" tagx="source" title="source">Evol Int J Org Evol</span>. <span class="year" tagx="year" title="year">2000</span>;<span class="volume" tagx="volume" title="volume">54</span>: <span class="fpage" tagx="fpage" title="fpage">1011</span>–<span class="lpage" tagx="lpage" title="lpage">1023</span>.</span></li> <li tag="ref"><a name="pone.0147647.ref005" /><span class="label" tagx="label" title="label">5</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Mitchell</span><span class="given-names" tagx="given-names" title="given-names">E</span></span>. <span class="mixed-article-title" title="mixed-article-title">Controversy over diphyly in Pinnipeds</span>. <span class="source" tagx="source" title="source">Syst Biol</span>. <span class="year" tagx="year" title="year">1967</span>;<span class="volume" tagx="volume" title="volume">1</span>: <span class="fpage" tagx="fpage" title="fpage">350</span>–<span class="lpage" tagx="lpage" title="lpage">351</span>.</span></li> <li tag="ref"><a name="pone.0147647.ref006" /><span class="label" tagx="label" title="label">6</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Yu</span><span class="given-names" tagx="given-names" title="given-names">L</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Luan</span><span class="given-names" tagx="given-names" title="given-names">P-T</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Jin</span><span class="given-names" tagx="given-names" title="given-names">W</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ryder</span><span class="given-names" tagx="given-names" title="given-names">OA</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Chemnick</span><span class="given-names" tagx="given-names" title="given-names">LG</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Davis</span><span class="given-names" tagx="given-names" title="given-names">HA</span></span>, <span class="etal" title="etal"><i>et al.</i></span><span class="mixed-article-title" title="mixed-article-title">Phylogenetic utility of nuclear introns in interfamilial relationships of Caniformia (order Carnivora)</span>. <span class="source" tagx="source" title="source">Syst Biol</span>. <span class="year" tagx="year" title="year">2011</span>;<span class="volume" tagx="volume" title="volume">60</span>: <span class="fpage" tagx="fpage" title="fpage">175</span>–<span class="lpage" tagx="lpage" title="lpage">187</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1093/sysbio/syq090">10.1093/sysbio/syq090</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21252386">21252386</a></span></span></li> <li tag="ref"><a name="pone.0147647.ref007" /><span class="label" tagx="label" title="label">7</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Sato</span><span class="given-names" tagx="given-names" title="given-names">JJ</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wolsan</span><span class="given-names" tagx="given-names" title="given-names">M</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Suzuki</span><span class="given-names" tagx="given-names" title="given-names">H</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hosoda</span><span class="given-names" tagx="given-names" title="given-names">T</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Yamaguchi</span><span class="given-names" tagx="given-names" title="given-names">Y</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hiyama</span><span class="given-names" tagx="given-names" title="given-names">K</span></span>, <span class="etal" title="etal"><i>et al.</i></span><span class="mixed-article-title" title="mixed-article-title">Evidence from Nuclear DNA Sequences Sheds Light on the Phylogenetic Relationships of Pinnipedia: Single Origin with Affinity to Musteloidea</span>. <span class="source" tagx="source" title="source">Zoolog Sci</span>. <span class="year" tagx="year" title="year">2006</span>;<span class="volume" tagx="volume" title="volume">23</span>: <span class="fpage" tagx="fpage" title="fpage">125</span>–<span class="lpage" tagx="lpage" title="lpage">146</span>. <span class="pub-id"><a href="https://dx.doi.org/10.2108/zsj.23.125">10.2108/zsj.23.125</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16603806">16603806</a></span></span></li> <li tag="ref"><a name="pone.0147647.ref008" /><span class="label" tagx="label" title="label">8</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Arnason</span><span class="given-names" tagx="given-names" title="given-names">U</span></span>. <span class="mixed-article-title" title="mixed-article-title">The karyotype of the grey seal (Halichoerus grypus)</span>. <span class="source" tagx="source" title="source">Hereditas</span>. <span class="year" tagx="year" title="year">1970</span>;<span class="volume" tagx="volume" title="volume">64</span>: <span class="fpage" tagx="fpage" title="fpage">237</span>–<span class="lpage" tagx="lpage" title="lpage">242</span>. <span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/5525744">5525744</a></span></span></li> <li tag="ref"><a name="pone.0147647.ref009" /><span class="label" tagx="label" title="label">9</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Arnason</span><span class="given-names" tagx="given-names" title="given-names">U</span></span>. <span class="mixed-article-title" title="mixed-article-title">Comparative chromosome studies in Pinnipedia</span>. <span class="source" tagx="source" title="source">Hereditas</span>. <span class="year" tagx="year" title="year">1974</span>;<span class="volume" tagx="volume" title="volume">76</span>: <span class="fpage" tagx="fpage" title="fpage">179</span>–<span class="lpage" tagx="lpage" title="lpage">226</span>. <span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/4135821">4135821</a></span></span></li> <li tag="ref"><a name="pone.0147647.ref010" /><span class="label" tagx="label" title="label">10</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Arnason</span><span class="given-names" tagx="given-names" title="given-names">U</span></span>. <span class="mixed-article-title" title="mixed-article-title">The relationship between the four principal pinniped karyotypes</span>. <span class="source" tagx="source" title="source">Hereditas</span>. <span class="year" tagx="year" title="year">1977</span>;<span class="volume" tagx="volume" title="volume">87</span>: <span class="fpage" tagx="fpage" title="fpage">227</span>–<span class="lpage" tagx="lpage" title="lpage">242</span>. <span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/608845">608845</a></span></span></li> <li tag="ref"><a name="pone.0147647.ref011" /><span class="label" tagx="label" title="label">11</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Robinson</span><span class="given-names" tagx="given-names" title="given-names">TJ</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Condy</span><span class="given-names" tagx="given-names" title="given-names">PR</span></span>. <span class="mixed-article-title" title="mixed-article-title">The chromosomes of the southern elephant seal, Mirounga leonina (Phocidae: Mammalia)</span>. <span class="source" tagx="source" title="source">Cytogenet Cell Genet</span>. <span class="year" tagx="year" title="year">1979</span>;<span class="volume" tagx="volume" title="volume">23</span>: <span class="fpage" tagx="fpage" title="fpage">157</span>–<span class="lpage" tagx="lpage" title="lpage">162</span>. <span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/436448">436448</a></span></span></li> <li tag="ref"><a name="pone.0147647.ref012" /><span class="label" tagx="label" title="label">12</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Fay</span><span class="given-names" tagx="given-names" title="given-names">FH</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Rausch</span><span class="given-names" tagx="given-names" title="given-names">VR</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Feltz</span><span class="given-names" tagx="given-names" title="given-names">ET</span></span>. <span class="mixed-article-title" title="mixed-article-title">Cytogenetic comparison of some pinnipeds (Mammalia: Eutheria)</span>. <span class="source" tagx="source" title="source">Can J Zool</span>. <span class="year" tagx="year" title="year">1967</span>;<span class="volume" tagx="volume" title="volume">45</span>: <span class="fpage" tagx="fpage" title="fpage">773</span>–<span class="lpage" tagx="lpage" title="lpage">778</span>.</span></li> <li tag="ref"><a name="pone.0147647.ref013" /><span class="label" tagx="label" title="label">13</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Nie</span><span class="given-names" tagx="given-names" title="given-names">W</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wang</span><span class="given-names" tagx="given-names" title="given-names">J</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Su</span><span class="given-names" tagx="given-names" title="given-names">W</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wang</span><span class="given-names" tagx="given-names" title="given-names">D</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Tanomtong</span><span class="given-names" tagx="given-names" title="given-names">A</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Perelman</span><span class="given-names" tagx="given-names" title="given-names">PL</span></span>, <span class="etal" title="etal"><i>et al.</i></span><span class="mixed-article-title" title="mixed-article-title">Chromosomal rearrangements and karyotype evolution in carnivores revealed by chromosome painting</span>. <span class="source" tagx="source" title="source">Heredity</span>. <span class="year" tagx="year" title="year">2012</span>;<span class="volume" tagx="volume" title="volume">108</span>: <span class="fpage" tagx="fpage" title="fpage">17</span>–<span class="lpage" tagx="lpage" title="lpage">27</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1038/hdy.2011.107">10.1038/hdy.2011.107</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22086079">22086079</a></span></span></li> <li tag="ref"><a name="pone.0147647.ref014" /><span class="label" tagx="label" title="label">14</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Perelman</span><span class="given-names" tagx="given-names" title="given-names">PL</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Beklemisheva</span><span class="given-names" tagx="given-names" title="given-names">VR</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Yudkin</span><span class="given-names" tagx="given-names" title="given-names">DV</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Petrina</span><span class="given-names" tagx="given-names" title="given-names">TN</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Rozhnov</span><span class="given-names" tagx="given-names" title="given-names">VV</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Nie</span><span class="given-names" tagx="given-names" title="given-names">W</span></span>, <span class="etal" title="etal"><i>et al.</i></span><span class="mixed-article-title" title="mixed-article-title">Comparative chromosome painting in Carnivora and Pholidota</span>. <span class="source" tagx="source" title="source">Cytogenet Genome Res</span>. <span class="year" tagx="year" title="year">2012</span>;<span class="volume" tagx="volume" title="volume">137</span>: <span class="fpage" tagx="fpage" title="fpage">174</span>–<span class="lpage" tagx="lpage" title="lpage">193</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1159/000341389">10.1159/000341389</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22889959">22889959</a></span></span></li> <li tag="ref"><a name="pone.0147647.ref015" /><span class="label" tagx="label" title="label">15</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wurster-Hill</span><span class="given-names" tagx="given-names" title="given-names">DH</span></span>. <span class="mixed-article-title" title="mixed-article-title">Chromosomes of eight species from five families of Carnivora</span>. <span class="source" tagx="source" title="source">J Mammal</span>. <span class="year" tagx="year" title="year">1973</span>;<span class="volume" tagx="volume" title="volume">54</span>: <span class="fpage" tagx="fpage" title="fpage">753</span>–<span class="lpage" tagx="lpage" title="lpage">760</span>. <span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/4744937">4744937</a></span></span></li> <li tag="ref"><a name="pone.0147647.ref016" /><span class="label" tagx="label" title="label">16</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wurster-Hill</span><span class="given-names" tagx="given-names" title="given-names">DH</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Centerwall</span><span class="given-names" tagx="given-names" title="given-names">WR</span></span>. <span class="mixed-article-title" title="mixed-article-title">The interrelationships of chromosome banding patterns in canids, mustelids, hyena, and felids</span>. <span class="source" tagx="source" title="source">Cytogenet Genome Res</span>. <span class="year" tagx="year" title="year">1982</span>;<span class="volume" tagx="volume" title="volume">34</span>: <span class="fpage" tagx="fpage" title="fpage">178</span>–<span class="lpage" tagx="lpage" title="lpage">192</span>.</span></li> <li tag="ref"><a name="pone.0147647.ref017" /><span class="label" tagx="label" title="label">17</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Graphodatsky</span><span class="given-names" tagx="given-names" title="given-names">AS</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Biltueva</span><span class="given-names" tagx="given-names" title="given-names">LS</span></span>. <span class="mixed-article-title" title="mixed-article-title">Homology of G-banded mammalian chromosomes</span>. <span class="source" tagx="source" title="source">Genetika</span>. <span class="year" tagx="year" title="year">1987</span>;<span class="volume" tagx="volume" title="volume">23</span>: <span class="fpage" tagx="fpage" title="fpage">93</span>–<span class="lpage" tagx="lpage" title="lpage">103</span>. <span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/3469125">3469125</a></span></span></li> <li tag="ref"><a name="pone.0147647.ref018" /><span class="label" tagx="label" title="label">18</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Graphodatsky</span><span class="given-names" tagx="given-names" title="given-names">AS</span></span>. <span class="chapter-title" tagx="chapter-title" title="chapter-title">Conserved and variable elements of mammalian chromosomes</span> In: <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Halnan</span><span class="given-names" tagx="given-names" title="given-names">C</span></span>, editor. <span class="source" tagx="source" title="source">Cytogenetics of Animals</span>. <span class="publisher-loc" tagx="publisher-loc" title="publisher-loc">Oxon, UK</span>: <span class="publisher-name" tagx="publisher-name" title="publisher-name">CAB International Press</span>; <span class="year" tagx="year" title="year">1989</span> pp. <span class="fpage" tagx="fpage" title="fpage">95</span>–<span class="lpage" tagx="lpage" title="lpage">124</span>.</span></li> <li tag="ref"><a name="pone.0147647.ref019" /><span class="label" tagx="label" title="label">19</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Nash</span><span class="given-names" tagx="given-names" title="given-names">WG</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wienberg</span><span class="given-names" tagx="given-names" title="given-names">J</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ferguson-Smith</span><span class="given-names" tagx="given-names" title="given-names">MA</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Menninger</span><span class="given-names" tagx="given-names" title="given-names">JC</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">O’Brien</span><span class="given-names" tagx="given-names" title="given-names">SJ</span></span>. <span class="mixed-article-title" title="mixed-article-title">Comparative genomics: tracking chromosome evolution in the family ursidae using reciprocal chromosome painting</span>. <span class="source" tagx="source" title="source">Cytogenet Cell Genet</span>. <span class="year" tagx="year" title="year">1998</span>;<span class="volume" tagx="volume" title="volume">83</span>: <span class="fpage" tagx="fpage" title="fpage">182</span>–<span class="lpage" tagx="lpage" title="lpage">192</span>. 15176 <span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/10072575">10072575</a></span></span></li> <li tag="ref"><a name="pone.0147647.ref020" /><span class="label" tagx="label" title="label">20</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Nash</span><span class="given-names" tagx="given-names" title="given-names">WG</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Menninger</span><span class="given-names" tagx="given-names" title="given-names">JC</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wienberg</span><span class="given-names" tagx="given-names" title="given-names">J</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Padilla-Nash</span><span class="given-names" tagx="given-names" title="given-names">HM</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">O’Brien</span><span class="given-names" tagx="given-names" title="given-names">SJ</span></span>. <span class="mixed-article-title" title="mixed-article-title">The pattern of phylogenomic evolution of the Canidae</span>. <span class="source" tagx="source" title="source">Cytogenet Cell Genet</span>. <span class="year" tagx="year" title="year">2001</span>;<span class="volume" tagx="volume" title="volume">95</span>: <span class="fpage" tagx="fpage" title="fpage">210</span>–<span class="lpage" tagx="lpage" title="lpage">224</span>. 59348 <span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12063402">12063402</a></span></span></li> <li tag="ref"><a name="pone.0147647.ref021" /><span class="label" tagx="label" title="label">21</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Breen</span><span class="given-names" tagx="given-names" title="given-names">M</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Thomas</span><span class="given-names" tagx="given-names" title="given-names">R</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Binns</span><span class="given-names" tagx="given-names" title="given-names">MM</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Carter</span><span class="given-names" tagx="given-names" title="given-names">NP</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Langford</span><span class="given-names" tagx="given-names" title="given-names">CF</span></span>. <span class="mixed-article-title" title="mixed-article-title">Reciprocal chromosome painting reveals detailed regions of conserved synteny between the karyotypes of the domestic dog (Canis familiaris) and human</span>. <span class="source" tagx="source" title="source">Genomics</span>. <span class="year" tagx="year" title="year">1999</span>;<span class="volume" tagx="volume" title="volume">61</span>: <span class="fpage" tagx="fpage" title="fpage">145</span>–<span class="lpage" tagx="lpage" title="lpage">155</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1006/geno.1999.5947">10.1006/geno.1999.5947</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/10534400">10534400</a></span></span></li> <li tag="ref"><a name="pone.0147647.ref022" /><span class="label" tagx="label" title="label">22</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Yang</span><span class="given-names" tagx="given-names" title="given-names">F</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">O’Brien</span><span class="given-names" tagx="given-names" title="given-names">PCM</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Milne</span><span class="given-names" tagx="given-names" title="given-names">BS</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Graphodatsky</span><span class="given-names" tagx="given-names" title="given-names">AS</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Solanky</span><span class="given-names" tagx="given-names" title="given-names">N</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Trifonov</span><span class="given-names" tagx="given-names" title="given-names">V</span></span>, <span class="etal" title="etal"><i>et al.</i></span><span class="mixed-article-title" title="mixed-article-title">A complete comparative chromosome map for the dog, red fox, and human and its integration with canine genetic maps</span>. <span class="source" tagx="source" title="source">Genomics</span>. <span class="year" tagx="year" title="year">1999</span>;<span class="volume" tagx="volume" title="volume">62</span>: <span class="fpage" tagx="fpage" title="fpage">189</span>–<span class="lpage" tagx="lpage" title="lpage">202</span>. <span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/10610712">10610712</a></span></span></li> <li tag="ref"><a name="pone.0147647.ref023" /><span class="label" tagx="label" title="label">23</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Tian</span><span class="given-names" tagx="given-names" title="given-names">Y</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Nie</span><span class="given-names" tagx="given-names" title="given-names">W</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wang</span><span class="given-names" tagx="given-names" title="given-names">J</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ferguson-Smith</span><span class="given-names" tagx="given-names" title="given-names">MA</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Yang</span><span class="given-names" tagx="given-names" title="given-names">F</span></span>. <span class="mixed-article-title" title="mixed-article-title">Chromosome evolution in bears: reconstructing phylogenetic relationships by cross-species chromosome painting</span>. <span class="source" tagx="source" title="source">Chromosome Res</span>. <span class="year" tagx="year" title="year">2004</span>;<span class="volume" tagx="volume" title="volume">12</span>: <span class="fpage" tagx="fpage" title="fpage">55</span>–<span class="lpage" tagx="lpage" title="lpage">63</span>. <span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/14984102">14984102</a></span></span></li> <li tag="ref"><a name="pone.0147647.ref024" /><span class="label" tagx="label" title="label">24</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Yang</span><span class="given-names" tagx="given-names" title="given-names">F</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Graphodatsky</span><span class="given-names" tagx="given-names" title="given-names">AS</span></span>. <span class="mixed-article-title" title="mixed-article-title">Integrated comparative genome maps and their implications for karyotype evolution of carnivores</span>. <span class="source" tagx="source" title="source">Chromosomes Today. Springer</span>; <span class="year" tagx="year" title="year">2004</span> pp. <span class="fpage" tagx="fpage" title="fpage">215</span>–<span class="lpage" tagx="lpage" title="lpage">224</span>. Available: <a href="http://link.springer.com/chapter/10.1007/978-94-017-1033-6_20">http://link.springer.com/chapter/10.1007/978-94-017-1033-6_20</a></span></li> <li tag="ref"><a name="pone.0147647.ref025" /><span class="label" tagx="label" title="label">25</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Perelman</span><span class="given-names" tagx="given-names" title="given-names">PL</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Graphodatsky</span><span class="given-names" tagx="given-names" title="given-names">AS</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Dragoo</span><span class="given-names" tagx="given-names" title="given-names">JW</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Serdyukova</span><span class="given-names" tagx="given-names" title="given-names">NA</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Stone</span><span class="given-names" tagx="given-names" title="given-names">G</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Cavagna</span><span class="given-names" tagx="given-names" title="given-names">P</span></span>, <span class="etal" title="etal"><i>et al.</i></span><span class="mixed-article-title" title="mixed-article-title">Chromosome painting shows that skunks (Mephitidae, Carnivora) have highly rearranged karyotypes</span>. <span class="source" tagx="source" title="source">Chromosome Res Int J Mol Supramol Evol Asp Chromosome Biol</span>. <span class="year" tagx="year" title="year">2008</span>;<span class="volume" tagx="volume" title="volume">16</span>: <span class="fpage" tagx="fpage" title="fpage">1215</span>–<span class="lpage" tagx="lpage" title="lpage">1231</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1007/s10577-008-1270-2">10.1007/s10577-008-1270-2</a></span></span></li> <li tag="ref"><a name="pone.0147647.ref026" /><span class="label" tagx="label" title="label">26</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wurster-Hill</span><span class="given-names" tagx="given-names" title="given-names">DH</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gray</span><span class="given-names" tagx="given-names" title="given-names">CW</span></span>. <span class="mixed-article-title" title="mixed-article-title">The interrelationships of chromosome banding patterns in procyonids, viverrids, and felids</span>. <span class="source" tagx="source" title="source">Cytogenet Genome Res</span>. <span class="year" tagx="year" title="year">1975</span>;<span class="volume" tagx="volume" title="volume">15</span>: <span class="fpage" tagx="fpage" title="fpage">306</span>–<span class="lpage" tagx="lpage" title="lpage">331</span>.</span></li> <li tag="ref"><a name="pone.0147647.ref027" /><span class="label" tagx="label" title="label">27</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Dutrillaux</span><span class="given-names" tagx="given-names" title="given-names">B</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Couturier</span><span class="given-names" tagx="given-names" title="given-names">J</span></span>. <span class="mixed-article-title" title="mixed-article-title">The ancestral karyotype of Carnivora: comparison with that of platyrrhine monkeys</span>. <span class="source" tagx="source" title="source">Cytogenet Cell Genet</span>. <span class="year" tagx="year" title="year">1983</span>;<span class="volume" tagx="volume" title="volume">35</span>: <span class="fpage" tagx="fpage" title="fpage">200</span>–<span class="lpage" tagx="lpage" title="lpage">208</span>. <span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/6861525">6861525</a></span></span></li> <li tag="ref"><a name="pone.0147647.ref028" /><span class="label" tagx="label" title="label">28</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Frönicke</span><span class="given-names" tagx="given-names" title="given-names">L</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Müller-Navia</span><span class="given-names" tagx="given-names" title="given-names">J</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Romanakis</span><span class="given-names" tagx="given-names" title="given-names">K</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Scherthan</span><span class="given-names" tagx="given-names" title="given-names">H</span></span>. <span class="mixed-article-title" title="mixed-article-title">Chromosomal homeologies between human, harbor seal (Phoca vitulina) and the putative ancestral carnivore karyotype revealed by Zoo-FISH</span>. <span class="source" tagx="source" title="source">Chromosoma</span>. <span class="year" tagx="year" title="year">1997</span>;<span class="volume" tagx="volume" title="volume">106</span>: <span class="fpage" tagx="fpage" title="fpage">108</span>–<span class="lpage" tagx="lpage" title="lpage">113</span>. <span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/9215560">9215560</a></span></span></li> <li tag="ref"><a name="pone.0147647.ref029" /><span class="label" tagx="label" title="label">29</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Murphy</span><span class="given-names" tagx="given-names" title="given-names">WJ</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Stanyon</span><span class="given-names" tagx="given-names" title="given-names">R</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">O’Brien</span><span class="given-names" tagx="given-names" title="given-names">SJ</span></span>. <span class="mixed-article-title" title="mixed-article-title">Evolution of mammalian genome organization inferred from comparative gene mapping</span>. <span class="source" tagx="source" title="source">Genome Biol</span>. <span class="year" tagx="year" title="year">2001</span>;<span class="volume" tagx="volume" title="volume">2</span>: <span class="fpage" tagx="fpage" title="fpage">REVIEWS0005</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11423011">11423011</a></span></span></li> <li tag="ref"><a name="pone.0147647.ref030" /><span class="label" tagx="label" title="label">30</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Nash</span><span class="given-names" tagx="given-names" title="given-names">WG</span></span>. <span class="chapter-title" tagx="chapter-title" title="chapter-title">Common seal (Phoca vitulina)</span> In: <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">O’Brien</span><span class="given-names" tagx="given-names" title="given-names">S</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Nash</span><span class="given-names" tagx="given-names" title="given-names">WG</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Menninger</span><span class="given-names" tagx="given-names" title="given-names">JC</span></span>, editors. <span class="source" tagx="source" title="source">Atlas of mammalian chromosomes</span>. <span class="publisher-loc" tagx="publisher-loc" title="publisher-loc">USA</span>: <span class="publisher-name" tagx="publisher-name" title="publisher-name">John Wiley and Sons Publishers</span>; <span class="year" tagx="year" title="year">2006</span> p. <span class="fpage" tagx="fpage" title="fpage">537</span>.</span></li> <li tag="ref"><a name="pone.0147647.ref031" /><span class="label" tagx="label" title="label">31</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Stanyon</span><span class="given-names" tagx="given-names" title="given-names">R</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Galleni</span><span class="given-names" tagx="given-names" title="given-names">L</span></span>. <span class="mixed-article-title" title="mixed-article-title">A rapid fibroblast culture technique for high resolution karyotypes</span>. <span class="source" tagx="source" title="source">Bolletino Zool</span>. <span class="year" tagx="year" title="year">1991</span>;<span class="volume" tagx="volume" title="volume">58</span>: <span class="fpage" tagx="fpage" title="fpage">81</span>–<span class="lpage" tagx="lpage" title="lpage">83</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1080/11250009109355732">10.1080/11250009109355732</a></span></span></li> <li tag="ref"><a name="pone.0147647.ref032" /><span class="label" tagx="label" title="label">32</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Graphodatsky</span><span class="given-names" tagx="given-names" title="given-names">AS</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Sablina</span><span class="given-names" tagx="given-names" title="given-names">OV</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Meyer</span><span class="given-names" tagx="given-names" title="given-names">MN</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Malikov</span><span class="given-names" tagx="given-names" title="given-names">VG</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Isakova</span><span class="given-names" tagx="given-names" title="given-names">EA</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Trifonov</span><span class="given-names" tagx="given-names" title="given-names">VA</span></span>, <span class="etal" title="etal"><i>et al.</i></span><span class="mixed-article-title" title="mixed-article-title">Comparative cytogenetics of hamsters of the genus Calomyscus</span>. <span class="source" tagx="source" title="source">Cytogenet Cell Genet</span>. <span class="year" tagx="year" title="year">2000</span>;<span class="volume" tagx="volume" title="volume">88</span>: <span class="fpage" tagx="fpage" title="fpage">296</span>–<span class="lpage" tagx="lpage" title="lpage">304</span>. 15513 <span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/10828614">10828614</a></span></span></li> <li tag="ref"><a name="pone.0147647.ref033" /><span class="label" tagx="label" title="label">33</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Graphodatsky</span><span class="given-names" tagx="given-names" title="given-names">AS</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Yang</span><span class="given-names" tagx="given-names" title="given-names">F</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">O’Brien</span><span class="given-names" tagx="given-names" title="given-names">PC</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Perelman</span><span class="given-names" tagx="given-names" title="given-names">P</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Milne</span><span class="given-names" tagx="given-names" title="given-names">BS</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Serdukova</span><span class="given-names" tagx="given-names" title="given-names">N</span></span>, <span class="etal" title="etal"><i>et al.</i></span><span class="mixed-article-title" title="mixed-article-title">Phylogenetic implications of the 38 putative ancestral chromosome segments for four canid species</span>. <span class="source" tagx="source" title="source">Cytogenet Cell Genet</span>. <span class="year" tagx="year" title="year">2001</span>;<span class="volume" tagx="volume" title="volume">92</span>: <span class="fpage" tagx="fpage" title="fpage">243</span>–<span class="lpage" tagx="lpage" title="lpage">247</span>. 56911 <span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11435696">11435696</a></span></span></li> <li tag="ref"><a name="pone.0147647.ref034" /><span class="label" tagx="label" title="label">34</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Seabright</span><span class="given-names" tagx="given-names" title="given-names">M</span></span>. <span class="mixed-article-title" title="mixed-article-title">A rapid banding technique for human chromosomes</span>. <span class="source" tagx="source" title="source">Lancet Lond Engl</span>. <span class="year" tagx="year" title="year">1971</span>;<span class="volume" tagx="volume" title="volume">2</span>: <span class="fpage" tagx="fpage" title="fpage">971</span>–<span class="lpage" tagx="lpage" title="lpage">972</span>.</span></li> <li tag="ref"><a name="pone.0147647.ref035" /><span class="label" tagx="label" title="label">35</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ferguson-Smith</span><span class="given-names" tagx="given-names" title="given-names">MA</span></span>. <span class="mixed-article-title" title="mixed-article-title">Genetic analysis by chromosome sorting and painting: phylogenetic and diagnostic applications</span>. <span class="source" tagx="source" title="source">Eur J Hum Genet EJHG</span>. <span class="year" tagx="year" title="year">1996</span>;<span class="volume" tagx="volume" title="volume">5</span>: <span class="fpage" tagx="fpage" title="fpage">253</span>–<span class="lpage" tagx="lpage" title="lpage">265</span>.</span></li> <li tag="ref"><a name="pone.0147647.ref036" /><span class="label" tagx="label" title="label">36</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Nie</span><span class="given-names" tagx="given-names" title="given-names">W</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wang</span><span class="given-names" tagx="given-names" title="given-names">J</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">O’Brien</span><span class="given-names" tagx="given-names" title="given-names">PCM</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Fu</span><span class="given-names" tagx="given-names" title="given-names">B</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ying</span><span class="given-names" tagx="given-names" title="given-names">T</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ferguson-Smith</span><span class="given-names" tagx="given-names" title="given-names">MA</span></span>, <span class="etal" title="etal"><i>et al.</i></span><span class="mixed-article-title" title="mixed-article-title">The genome phylogeny of domestic cat, red panda and five mustelid species revealed by comparative chromosome painting and G-banding</span>. <span class="source" tagx="source" title="source">Chromosome Res Int J Mol Supramol Evol Asp Chromosome Biol</span>. <span class="year" tagx="year" title="year">2002</span>;<span class="volume" tagx="volume" title="volume">10</span>: <span class="fpage" tagx="fpage" title="fpage">209</span>–<span class="lpage" tagx="lpage" title="lpage">222</span>.</span></li> <li tag="ref"><a name="pone.0147647.ref037" /><span class="label" tagx="label" title="label">37</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Foote</span><span class="given-names" tagx="given-names" title="given-names">AD</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Liu</span><span class="given-names" tagx="given-names" title="given-names">Y</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Thomas</span><span class="given-names" tagx="given-names" title="given-names">GW</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Vinař</span><span class="given-names" tagx="given-names" title="given-names">T</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Alföldi</span><span class="given-names" tagx="given-names" title="given-names">J</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Deng</span><span class="given-names" tagx="given-names" title="given-names">J</span></span>, <span class="etal" title="etal"><i>et al.</i></span><span class="mixed-article-title" title="mixed-article-title">Convergent evolution of the genomes of marine mammals</span>. <span class="source" tagx="source" title="source">Nat Genet</span>. <span class="year" tagx="year" title="year">2015</span>;<span class="volume" tagx="volume" title="volume">47</span>: <span class="fpage" tagx="fpage" title="fpage">272</span>–<span class="lpage" tagx="lpage" title="lpage">275</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1038/ng.3198">10.1038/ng.3198</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25621460">25621460</a></span></span></li> <li tag="ref"><a name="pone.0147647.ref038" /><span class="label" tagx="label" title="label">38</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Jones</span><span class="given-names" tagx="given-names" title="given-names">KE</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Smaers</span><span class="given-names" tagx="given-names" title="given-names">JB</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Goswami</span><span class="given-names" tagx="given-names" title="given-names">A</span></span>. <span class="mixed-article-title" title="mixed-article-title">Impact of the terrestrial-aquatic transition on disparity and rates of evolution in the carnivoran skull</span>. <span class="source" tagx="source" title="source">BMC Evol Biol</span>. <span class="year" tagx="year" title="year">2015</span>;<span class="volume" tagx="volume" title="volume">15</span>: <span class="fpage" tagx="fpage" title="fpage">8</span><span class="pub-id"><a href="https://dx.doi.org/10.1186/s12862-015-0285-5">10.1186/s12862-015-0285-5</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25648618">25648618</a></span></span></li> <li tag="ref"><a name="pone.0147647.ref039" /><span class="label" tagx="label" title="label">39</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Arnason</span><span class="given-names" tagx="given-names" title="given-names">U</span></span>. <span class="mixed-article-title" title="mixed-article-title">The role of chromosomal rearrangement in mammalian speciation with special reference to Cetacea and Pinnipedia</span>. <span class="source" tagx="source" title="source">Hereditas</span>. <span class="year" tagx="year" title="year">1972</span>;<span class="volume" tagx="volume" title="volume">70</span>: <span class="fpage" tagx="fpage" title="fpage">113</span>–<span class="lpage" tagx="lpage" title="lpage">118</span>. <span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/4680630">4680630</a></span></span></li> <li tag="ref"><a name="pone.0147647.ref040" /><span class="label" tagx="label" title="label">40</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hameister</span><span class="given-names" tagx="given-names" title="given-names">H</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Klett</span><span class="given-names" tagx="given-names" title="given-names">C</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bruch</span><span class="given-names" tagx="given-names" title="given-names">J</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Dixkens</span><span class="given-names" tagx="given-names" title="given-names">C</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Vogel</span><span class="given-names" tagx="given-names" title="given-names">W</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Christensen</span><span class="given-names" tagx="given-names" title="given-names">K</span></span>. <span class="mixed-article-title" title="mixed-article-title">Zoo-FISH analysis: the American mink (Mustela vison) closely resembles the cat karyotype</span>. <span class="source" tagx="source" title="source">Chromosome Res Int J Mol Supramol Evol Asp Chromosome Biol</span>. <span class="year" tagx="year" title="year">1997</span>;<span class="volume" tagx="volume" title="volume">5</span>: <span class="fpage" tagx="fpage" title="fpage">5</span>–<span class="lpage" tagx="lpage" title="lpage">11</span>.</span></li> <li tag="ref"><a name="pone.0147647.ref041" /><span class="label" tagx="label" title="label">41</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Cavagna</span><span class="given-names" tagx="given-names" title="given-names">P</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Menotti</span><span class="given-names" tagx="given-names" title="given-names">A</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Stanyon</span><span class="given-names" tagx="given-names" title="given-names">R</span></span>. <span class="mixed-article-title" title="mixed-article-title">Genomic homology of the domestic ferret with cats and humans</span>. <span class="source" tagx="source" title="source">Mamm Genome Off J Int Mamm Genome Soc</span>. <span class="year" tagx="year" title="year">2000</span>;<span class="volume" tagx="volume" title="volume">11</span>: <span class="fpage" tagx="fpage" title="fpage">866</span>–<span class="lpage" tagx="lpage" title="lpage">870</span>.</span></li> <li tag="ref"><a name="pone.0147647.ref042" /><span class="label" tagx="label" title="label">42</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bielec</span><span class="given-names" tagx="given-names" title="given-names">PE</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gallagher</span><span class="given-names" tagx="given-names" title="given-names">DS</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Womack</span><span class="given-names" tagx="given-names" title="given-names">JE</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Busbee</span><span class="given-names" tagx="given-names" title="given-names">DL</span></span>. <span class="mixed-article-title" title="mixed-article-title">Homologies between human and dolphin chromosomes detected by heterologous chromosome painting</span>. <span class="source" tagx="source" title="source">Cytogenet Cell Genet</span>. <span class="year" tagx="year" title="year">1998</span>;<span class="volume" tagx="volume" title="volume">81</span>: <span class="fpage" tagx="fpage" title="fpage">18</span>–<span class="lpage" tagx="lpage" title="lpage">25</span>. <span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/9691170">9691170</a></span></span></li> <li tag="ref"><a name="pone.0147647.ref043" /><span class="label" tagx="label" title="label">43</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kellogg</span><span class="given-names" tagx="given-names" title="given-names">ME</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Burkett</span><span class="given-names" tagx="given-names" title="given-names">S</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Dennis</span><span class="given-names" tagx="given-names" title="given-names">TR</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Stone</span><span class="given-names" tagx="given-names" title="given-names">G</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gray</span><span class="given-names" tagx="given-names" title="given-names">BA</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">McGuire</span><span class="given-names" tagx="given-names" title="given-names">PM</span></span>, <span class="etal" title="etal"><i>et al.</i></span><span class="mixed-article-title" title="mixed-article-title">Chromosome painting in the manatee supports Afrotheria and Paenungulata</span>. <span class="source" tagx="source" title="source">BMC Evol Biol</span>. <span class="year" tagx="year" title="year">2007</span>;<span class="volume" tagx="volume" title="volume">7</span>: <span class="fpage" tagx="fpage" title="fpage">6</span><span class="pub-id"><a href="https://dx.doi.org/10.1186/1471-2148-7-6">10.1186/1471-2148-7-6</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17244368">17244368</a></span></span></li> <li tag="ref"><a name="pone.0147647.ref044" /><span class="label" tagx="label" title="label">44</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kulemzina</span><span class="given-names" tagx="given-names" title="given-names">AI</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Trifonov</span><span class="given-names" tagx="given-names" title="given-names">VA</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Perelman</span><span class="given-names" tagx="given-names" title="given-names">PL</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Rubtsova</span><span class="given-names" tagx="given-names" title="given-names">NV</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Volobuev</span><span class="given-names" tagx="given-names" title="given-names">V</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ferguson-Smith</span><span class="given-names" tagx="given-names" title="given-names">MA</span></span>, <span class="etal" title="etal"><i>et al.</i></span><span class="mixed-article-title" title="mixed-article-title">Cross-species chromosome painting in Cetartiodactyla: reconstructing the karyotype evolution in key phylogenetic lineages</span>. <span class="source" tagx="source" title="source">Chromosome Res Int J Mol Supramol Evol Asp Chromosome Biol</span>. <span class="year" tagx="year" title="year">2009</span>;<span class="volume" tagx="volume" title="volume">17</span>: <span class="fpage" tagx="fpage" title="fpage">419</span>–<span class="lpage" tagx="lpage" title="lpage">436</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1007/s10577-009-9032-3">10.1007/s10577-009-9032-3</a></span></span></li> <li tag="ref"><a name="pone.0147647.ref045" /><span class="label" tagx="label" title="label">45</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ferguson-Smith</span><span class="given-names" tagx="given-names" title="given-names">MA</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Trifonov</span><span class="given-names" tagx="given-names" title="given-names">V</span></span>. <span class="mixed-article-title" title="mixed-article-title">Mammalian karyotype evolution</span>. <span class="source" tagx="source" title="source">Nat Rev Genet</span>. <span class="year" tagx="year" title="year">2007</span>;<span class="volume" tagx="volume" title="volume">8</span>: <span class="fpage" tagx="fpage" title="fpage">950</span>–<span class="lpage" tagx="lpage" title="lpage">962</span>. <span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18007651">18007651</a></span></span></li> <li tag="ref"><a name="pone.0147647.ref046" /><span class="label" tagx="label" title="label">46</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ruiz-Herrera</span><span class="given-names" tagx="given-names" title="given-names">A</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Farré</span><span class="given-names" tagx="given-names" title="given-names">M</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Robinson</span><span class="given-names" tagx="given-names" title="given-names">TJ</span></span>. <span class="mixed-article-title" title="mixed-article-title">Molecular cytogenetic and genomic insights into chromosomal evolution</span>. <span class="source" tagx="source" title="source">Heredity</span>. <span class="year" tagx="year" title="year">2012</span>;<span class="volume" tagx="volume" title="volume">108</span>: <span class="fpage" tagx="fpage" title="fpage">28</span>–<span class="lpage" tagx="lpage" title="lpage">36</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1038/hdy.2011.102">10.1038/hdy.2011.102</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22108627">22108627</a></span></span></li> <li tag="ref"><a name="pone.0147647.ref047" /><span class="label" tagx="label" title="label">47</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Nash</span><span class="given-names" tagx="given-names" title="given-names">WG</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Menninger</span><span class="given-names" tagx="given-names" title="given-names">JC</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Padilla-Nash</span><span class="given-names" tagx="given-names" title="given-names">HM</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Stone</span><span class="given-names" tagx="given-names" title="given-names">G</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Perelman</span><span class="given-names" tagx="given-names" title="given-names">PL</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">O’Brien</span><span class="given-names" tagx="given-names" title="given-names">SJ</span></span>. <span class="mixed-article-title" title="mixed-article-title">The ancestral carnivore karyotype (2n = 38) lives today in ringtails</span>. <span class="source" tagx="source" title="source">J Hered</span>. <span class="year" tagx="year" title="year">2008</span>;<span class="volume" tagx="volume" title="volume">99</span>: <span class="fpage" tagx="fpage" title="fpage">241</span>–<span class="lpage" tagx="lpage" title="lpage">253</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1093/jhered/esm130">10.1093/jhered/esm130</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18339652">18339652</a></span></span></li> <li tag="ref"><a name="pone.0147647.ref048" /><span class="label" tagx="label" title="label">48</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Trifonov</span><span class="given-names" tagx="given-names" title="given-names">VA</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Musilova</span><span class="given-names" tagx="given-names" title="given-names">P</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kulemsina</span><span class="given-names" tagx="given-names" title="given-names">AI</span></span>. <span class="mixed-article-title" title="mixed-article-title">Chromosome evolution in Perissodactyla</span>. <span class="source" tagx="source" title="source">Cytogenet Genome Res</span>. <span class="year" tagx="year" title="year">2012</span>;<span class="volume" tagx="volume" title="volume">137</span>: <span class="fpage" tagx="fpage" title="fpage">208</span>–<span class="lpage" tagx="lpage" title="lpage">217</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1159/000339900">10.1159/000339900</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22813844">22813844</a></span></span></li> <li tag="ref"><a name="pone.0147647.ref049" /><span class="label" tagx="label" title="label">49</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Arnason</span><span class="given-names" tagx="given-names" title="given-names">U</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gullberg</span><span class="given-names" tagx="given-names" title="given-names">A</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Janke</span><span class="given-names" tagx="given-names" title="given-names">A</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kullberg</span><span class="given-names" tagx="given-names" title="given-names">M</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lehman</span><span class="given-names" tagx="given-names" title="given-names">N</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Petrov</span><span class="given-names" tagx="given-names" title="given-names">EA</span></span>, <span class="etal" title="etal"><i>et al.</i></span><span class="mixed-article-title" title="mixed-article-title">Pinniped phylogeny and a new hypothesis for their origin and dispersal</span>. <span class="source" tagx="source" title="source">Mol Phylogenet Evol</span>. <span class="year" tagx="year" title="year">2006</span>;<span class="volume" tagx="volume" title="volume">41</span>: <span class="fpage" tagx="fpage" title="fpage">345</span>–<span class="lpage" tagx="lpage" title="lpage">354</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1016/j.ympev.2006.05.022">10.1016/j.ympev.2006.05.022</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16815048">16815048</a></span></span></li> <li tag="ref"><a name="pone.0147647.ref050" /><span class="label" tagx="label" title="label">50</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Anbinder</span><span class="given-names" tagx="given-names" title="given-names">E</span></span>. <span class="chapter-title" tagx="chapter-title" title="chapter-title">On the problem of systematics and phylogeny of the pinnipeds, family Phocidae</span> In: <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Vorontsov</span><span class="given-names" tagx="given-names" title="given-names">N</span></span>, editor. <span class="source" tagx="source" title="source">The mammals, evolution, karyology, taxonomy, fauna</span>. <span class="publisher-loc" tagx="publisher-loc" title="publisher-loc">Novosibirsk</span>: <span class="publisher-name" tagx="publisher-name" title="publisher-name">Acid.Sci.U.S.S.R., Siberian branch</span>; <span class="year" tagx="year" title="year">1969</span> pp. <span class="fpage" tagx="fpage" title="fpage">23</span>–<span class="lpage" tagx="lpage" title="lpage">25</span>.</span></li> <li tag="ref"><a name="pone.0147647.ref051" /><span class="label" tagx="label" title="label">51</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation">Wilson DE, Reeder DM. Mammal species of the world: a taxonomic and geographic reference [Internet]. JHU Press; 2005. Available: <a href="https://www.google.com/books?hl=ru&amp;lr=&amp;id=YnYED-YG0ZYC&amp;oi=fnd&amp;pg=PR17&amp;dq=44.%09Wilson+DE,+Reeder+DM:+Mammal+Species+of+the+World.+A+Taxonomic+and+Geographic+Reference,+ed+3+(Johns+Hopkins+University+Press,+Baltimore+2005),+p599&amp;ots=1CXx16GV3-&amp;sig=2__Y90I9SO9jQ1lDKqyLx7COms0">https://www.google.com/books?hl=ru&amp;amp;lr=&amp;amp;id=YnYED-YG0ZYC&amp;amp;oi=fnd&amp;amp;pg=PR17&amp;amp;dq=44.%09Wilson+DE,+Reeder+DM:+Mammal+Species+of+the+World.+A+Taxonomic+and+Geographic+Reference,+ed+3+(Johns+Hopkins+University+Press,+Baltimore+2005),+p599&amp;amp;ots=1CXx16GV3-&amp;amp;sig=2__Y90I9SO9jQ1lDKqyLx7COms0</a></span></li> <li tag="ref"><a name="pone.0147647.ref052" /><span class="label" tagx="label" title="label">52</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Nash</span><span class="given-names" tagx="given-names" title="given-names">WG</span></span>. <span class="chapter-title" tagx="chapter-title" title="chapter-title">Order Carnivora</span> In: <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">O’Brien</span><span class="given-names" tagx="given-names" title="given-names">S</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Nash</span><span class="given-names" tagx="given-names" title="given-names">WG</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Menninger</span><span class="given-names" tagx="given-names" title="given-names">J</span></span>, editors. <span class="source" tagx="source" title="source">Atlas of mammalian chromosomes</span>. <span class="publisher-loc" tagx="publisher-loc" title="publisher-loc">USA</span>: <span class="publisher-name" tagx="publisher-name" title="publisher-name">John Wiley and Sons Publishers</span>; <span class="year" tagx="year" title="year">2006</span> pp. <span class="fpage" tagx="fpage" title="fpage">445</span>–<span class="lpage" tagx="lpage" title="lpage">450</span>.</span></li> <li tag="ref"><a name="pone.0147647.ref053" /><span class="label" tagx="label" title="label">53</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Robinson</span><span class="given-names" tagx="given-names" title="given-names">TJ</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ruiz-Herrera</span><span class="given-names" tagx="given-names" title="given-names">A</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Avise</span><span class="given-names" tagx="given-names" title="given-names">JC</span></span>. <span class="mixed-article-title" title="mixed-article-title">Hemiplasy and homoplasy in the karyotypic phylogenies of mammals</span>. <span class="source" tagx="source" title="source">Proc Natl Acad Sci U S A</span>. <span class="year" tagx="year" title="year">2008</span>;<span class="volume" tagx="volume" title="volume">105</span>: <span class="fpage" tagx="fpage" title="fpage">14477</span>–<span class="lpage" tagx="lpage" title="lpage">14481</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1073/pnas.0807433105">10.1073/pnas.0807433105</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18787123">18787123</a></span></span></li> <li tag="ref"><a name="pone.0147647.ref054" /><span class="label" tagx="label" title="label">54</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Robinson</span><span class="given-names" tagx="given-names" title="given-names">TJ</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ropiquet</span><span class="given-names" tagx="given-names" title="given-names">A</span></span>. <span class="mixed-article-title" title="mixed-article-title">Examination of hemiplasy, homoplasy and phylogenetic discordance in chromosomal evolution of the Bovidae</span>. <span class="source" tagx="source" title="source">Syst Biol</span>. <span class="year" tagx="year" title="year">2011</span>;<span class="volume" tagx="volume" title="volume">60</span>: <span class="fpage" tagx="fpage" title="fpage">439</span>–<span class="lpage" tagx="lpage" title="lpage">450</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1093/sysbio/syr045">10.1093/sysbio/syr045</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21527771">21527771</a></span></span></li> <li tag="ref"><a name="pone.0147647.ref055" /><span class="label" tagx="label" title="label">55</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">O’Brien</span><span class="given-names" tagx="given-names" title="given-names">SJ</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Nash</span><span class="given-names" tagx="given-names" title="given-names">WG</span></span>. <span class="mixed-article-title" title="mixed-article-title">Genetic mapping in mammals: chromosome map of domestic cat</span>. <span class="source" tagx="source" title="source">Science</span>. <span class="year" tagx="year" title="year">1982</span>;<span class="volume" tagx="volume" title="volume">216</span>: <span class="fpage" tagx="fpage" title="fpage">257</span>–<span class="lpage" tagx="lpage" title="lpage">265</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1126/science.7063884">10.1126/science.7063884</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/7063884">7063884</a></span></span></li> </ul> </div> </div>  </body></html>