<html xmlns="http://www.w3.org/1999/xhtml"> <head> <style type="text/css">
      	     		 
      	    a                          {background : #ffffff; }
      	    article                    {border-style : dotted; border-width : 2px; }
		 	
		 	div                        {background : #ffffcc;}
		 	div.abstract-title         {font-weight : bold ; font-size : 16pt;}
		 	div.ack                    {border-style : solid ; border-color : red; margin : 2em; }
		 	div.article-meta           {border-style : solid; border-width : 1 pt; margin 1em;}
		 	div.boxed-text             {margin : 5em; border-style : solid;}
		 	div.contrib-group          {margin : 1em; }
		 	div.fig                    {border-style : solid; border-width : 2px; margin : 2em; }
		 	div.funding                {font-weight : bold ; font-size : 16pt;}
		 	div.given-names            {font-style : italic;}
		 	div.intro                  {border-style : inset; margin : 5px;}
		 	div.introduction           {border-style : inset; margin : 5px;}
		 	div.journal-meta           {border-style : solid; border-width : 1 pt; margin 1em;}
		 	div.journal-title-group    {background : #ffeeee;}
		 	div.article-title          {font-weight : bold ; font-size : 18pt;}
		 	div.kwd                    {font-style : italic}
		 	div.meta-name              {font-weight : bold ; font-size : 16pt;}
		 	div.name                   {font-weight : bold;}
		 	div.alt-title              {font-style : italic; font-size : 12px;}
		 	
		 	div.sec                    {border : 2px; margin 5px; padding 2px;}
		 	div.title                  {font-family : courier; font-weight : bold;
		 	                            font-size : 16pt; margin : 5px;}
		 	
		 	div.materials_methods      {border-style : double; margin : 5px; }
		 	div.methods                {border-style : double; margin : 5px; }
		 	
		 	div.results                {border-style : solid; margin : 5px;}
		 	div.background             {border-style : dotted; margin : 5px;}
		 	
		 	div.discussion             {border-style : groove; margin : 5px;}
		 	div.conclusion             {border-style : ridge; margin : 5px;}
		 	div.conclusions             {border-style : ridge; margin : 5px;}
		 	div.supplementary-material {border-style : inset; margin : 5px;}
		 	div.abbreviations          {border-style : double; border-color : red; margin : 5px;}
		 	div.competinginterests     {border-style : double; border-color : blue; margin : 5px;}
		 	div.acknowledgements       {border-style : double; border-color : green; margin : 5px;}
		 	div.authors_contributions   {border-style : double; border-color : purple; margin : 5px;}
		 	
		 	div.publisher              {border-style : outset; margin : 5px;}
		 	div.fn-type-conflict       {background : #f88; }
		 	div.fn-type-con            {background : #ddf; }
		 	div.fn-type-other          {background : #ddd; }
		 	
		 	div.unknown                {background : #ffd;
									 	  border-style : solid;
									 	  border-width : 1px;
									 	  padding : 2 px;}
		 	  
      	    table                      {background : #ffffdd;}
		 	tr                         {background : #ddddff; padding : 1px;}
		 	
		 	span                       {background : #ffcccc;}
		 	
		 	span.citation-author       {font-family : helvetica; background : #ffeeee;}
		 	span.collab                {background : #ddffff; }
		 	span.comment               {font-family : courier; font-size : 6px; background : #ffaaff;}
		 	span.contrib               {background : #ffffff;}
		 	span.corresp               {background : #ddffdd; }
		 	span.doi                   {background : #ffffff;}
		 	span.email                 {font-family : courier; }
		 	span.etal                  {font-style : italic;}
		 	span.fpage                 {font-family : courier;}
		 	span.given-names           {background : #ffffff;}
		 	span.iso-abbrev            {background : #ffffff;}
		 	span.issn-epub             {background : #ffffff;}
		 	span.issn-ppub             {background : #ffffff;}
		 	span.journal-title         {background : #ffffff;}
		 	span.lpage                 {font-family : courier;}
		 	span.mixed-article-title   {font-style : italic ;}
		 	span.nlm-ta                {background : #ffffff;}
		 	span.pmc                   {background : #ffffff;}
		 	span.pmcid                 {background : #ffffff;}
		 	span.pmid                  {background : #ffffff;}
		 	span.publisher             {background : #ffffff;}
		 	span.publisher-id          {background : #ffffff;}
		 	span.publisher-name        {background : #ffffff;}
		 	span.source                {background : #ffffff;}
		 	span.subject               {background : #ffffff;}
		 	span.surname               {background : #ffffff;}
		 	span.volume                {font-family : courier; font-weight : bold;}
		 	span.year                  {font-family : courier ; font-style : italic;}
			</style> </head> <body> <div class="front" title="front"> <div class="journal-meta" tagx="journal-meta" title="journal-meta"><span class="nlm-ta" title="nlm-ta">Brief Bioinform</span><span class="iso-abbrev" title="iso-abbrev">Brief. Bioinformatics</span><span class="publisher-id" title="publisher-id">bib</span><div class="journal-title-group" tagx="journal-title-group" title="journal-title-group"><span class="journal-title" tagx="journal-title" title="journal-title">Briefings in Bioinformatics</span></div><span class="issn-ppub" tagx="issn" title="issn-ppub">1467-5463</span><span class="issn-epub" tagx="issn" title="issn-epub">1477-4054</span><div class="publisher" tagx="publisher" title="publisher"><span class="publisher-name" tagx="publisher-name" title="publisher-name">Oxford University Press</span></div> </div> <div class="article-meta" tagx="article-meta" title="article-meta"><span class="pmcid" title="pmcid"> pmcid: <a href="http://www.ncbi.nlm.nih.gov/pubmed/7454301">7454301</a></span><span class="pmid" title="pmid"> pmid: <a href="http://www.ncbi.nlm.nih.gov/pubmed/32793981">32793981</a></span><span class="doi" title="doi"> doi: <a href="https://dx.doi.org/10.1093/bib/bbaa170">10.1093/bib/bbaa170</a></span><span class="publisher-id" title="publisher-id">bbaa170</span><div class="article-categories" title="article-categories">
: <span class="subject" title="subject">AcademicSubjects/SCI01060</span>: <span class="subject" title="subject">Case Study</span></div> <div class="title-group" tagx="title-group" title="title-group"> <div class="article-title" title="article-title">
A diagnostic genomic signal processing (GSP)-based system for automatic feature analysis and detection of COVID-19</div> </div> <div class="contrib-group" title="contrib-group"><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Naeem</span><span class="given-names" tagx="given-names" title="given-names">Safaa M</span></span><a href="#aff1">1</a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Mabrouk</span><span class="given-names" tagx="given-names" title="given-names">Mai S</span></span><a href="#aff2">2</a><a href="#cor1" /></span><span class="contrib" title="contrib"><span class="contrib-id" tagx="contrib-id" title="contrib-id">http://orcid.org/0000-0001-7958-937X</span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Marzouk</span><span class="given-names" tagx="given-names" title="given-names">Samir Y</span></span><a href="#aff3">3</a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Eldosoky</span><span class="given-names" tagx="given-names" title="given-names">Mohamed A</span></span><a href="#aff4">4</a></span></div><span class="citation_author_institution" id="aff1">[1], <span class="institution" title="institution">Biomedical Engineering Department</span><span class="country" tagx="country" title="country">Egypt</span></span><span class="citation_author_institution" id="aff2">[2], <span class="institution" title="institution">Misr University for Science and Technology</span></span><span class="citation_author_institution" id="aff3">[3], <span class="institution" title="institution">AASTMT</span><span class="country" tagx="country" title="country">Egypt</span></span><span class="citation_author_institution" id="aff4">[4], <span class="institution" title="institution">Helwan University</span><span class="country" tagx="country" title="country">Egypt</span></span><div class="author-notes" title="author-notes"> <div class="corresp" title="corresp">
Corresponding author: Mai S. Mabrouk, Biomedical Engineering Department, Faculty of Engineering, Misr University for Science and Technology (MUST University), Cairo, Egypt. Tel.: +20 100 166 2403; E-mail: <span class="email" tagx="email" title="email">Msm_eng@yahoo.com</span></div> </div><span class="pub-date-epub" title="pub-date-epub">epub: <span>2020-8-8</span></span><span class="elocation-id" tagx="elocation-id" title="elocation-id">bbaa170</span><span class="history" title="history"><span class="received" title="received">received: 2020-5-28</span><span class="rev-recd" title="rev-recd">rev-recd: 2020-6-24</span><span class="accepted" title="accepted">accepted: 2020-7-5</span></span><div class="permissions"><span class="copyright" title="copyright">(C) , <span class="copyright-year" tagx="copyright-year" title="copyright-year">2020</span></span><span class="license" title="license"><span class="license-p" title="license-p">This article is published and distributed under the terms of the Oxford University Press, Standard Journals Publication Model (<a href="https://academic.oup.com/journals/pages/open_access/funder_policies/chorus/standard_publication_model">https://academic.oup.com/journals/pages/open_access/funder_policies/chorus/standard_publication_model</a>)</span></span><span class="license" title="license"><span class="license-p" title="license-p">This article is made available via the PMC Open Access Subset for unrestricted re-use and analyses in any form or by any means with acknowledgement of the original source. These permissions are granted for the duration of the COVID-19 pandemic or until permissions are revoked in writing. Upon expiration of these permissions, PMC is granted a perpetual license to make this article available via PMC and Europe PMC, consistent with existing copyright protections.</span></span></div><a class="self-uri" href="bbaa170.pdf" title="self-uri">bbaa170.pdf</a><div class="abstract" title="abstract"> <div class="abstract-title" title="abstract-title">
Abstract</div> <div class="title" tagx="title" title="title">
Abstract</div> <p>Coronavirus Disease 2019 (COVID-19) is a sudden viral contagion that appeared at the end of last year in Wuhan city, the Chinese province of Hubei, China. The fast spread of COVID-19 has led to a dangerous threat to worldwide health. Also in the last two decades, several viral epidemics have been listed like the severe acute respiratory syndrome coronavirus (SARS-CoV) in 2002/2003, the influenza H1N1 in 2009 and recently the Middle East respiratory syndrome coronavirus (MERS-CoV) which appeared in Saudi Arabia in 2012. In this research, an automated system is created to differentiate between the COVID-19, SARS-CoV and MERS-CoV epidemics by using their genomic sequences recorded in the NCBI <a href="GenBank">GenBank</a> in order to facilitate the diagnosis process and increase the accuracy of disease detection in less time. The selected database contains 76 genes for each epidemic. Then, some features are extracted like a discrete Fourier transform (DFT), discrete cosine transform (DCT) and the seven moment invariants to two different classifiers. These classifiers are the <i>k</i>-nearest neighbor (KNN) algorithm and the trainable cascade-forward back propagation neural network where they give satisfying results to compare. To evaluate the performance of classifiers, there are some effective parameters calculated. They are accuracy (ACC), F1 score, error rate and Matthews correlation coefficient (MCC) that are 100%, 100%, 0 and 1, respectively, for the KNN algorithm and 98.89%, 98.34%, 0.0111 and 0.9754, respectively, for the cascade-forward network.</p> </div> <div class="kwd-group"> <div class="kwd" title="kwd">
COVID-19</div> <div class="kwd" title="kwd">
SARS-CoV</div> <div class="kwd" title="kwd">
MERS-CoV</div> <div class="kwd" title="kwd">
genomic signal processing</div> <div class="kwd" title="kwd">
GenBank</div> </div> <div class="counts" title="counts"><span class="page-count" title="page-count">page-count: </span></div> <div class="custom-meta-group" title="custom-meta-group"> <meta name="article-lifecycle" value="PAP" /> </div> </div> </div> <div class="body" title="body"> <div class="introduction" title="sec"> <div class="title" tagx="title" title="title">
Introduction</div> <p>Coronaviruses (COVs) are single-/positive-stranded RNA (++RNA) viruses that hit humans and animals. It includes four primary proteins which are the envelope, nucleocapsid, membrane and spike proteins [<a href="#ref1">1</a>]. Because of the presence of spike protein, these viruses take the crown-like appearance under the electron microscope. This crown is called Coronam in Latin [<a href="#ref2">2</a>, <a href="#ref3">3</a>].</p> <p>The International Committee on Taxonomy of Viruses (ICTV) considers that the coronaviruses are <i>Nidovirales</i> order members and part of the <i>Cornidovirineae</i> family divided into two subfamilies called <i>Letovirinae</i> and <i>Orthocoronavirinae</i> [<a href="#ref4">4</a>]. <i>Orthocoronavirinae</i> subfamily is classified into four types: AlphaCoV (α-CoV), BetaCoV (β-CoV), DeltaCoV (γ-CoV) and GammaCoV (δ-CoV) [<a href="#ref5">5</a>, <a href="#ref6">6</a>].</p> <p>The α-CoV and the β-CoV are capable to hit mammals; however the γ-CoV and the δ-CoV lead to avian infection. Effective types of AlphaCoV are 229E-CoV and NL63-CoV, while OC43-CoV, HKU1-CoV, MERS-CoV and SARS-CoV are BetaCoV types [<a href="#ref7">7</a>]. Taking into consideration that 229E-CoV, NL63-CoV, OC43-CoV and HKU1-CoV result in simple respiratory marks like that in common cold, SARS-CoV and MERS-CoV cause dangerous and deadly respiratory tract infections [<a href="#ref8">8</a>]. In 2002 and 2012, the flare-up of SARS and MERS happened, respectively, when death raised at huge rates by animal–human infection.</p> <p>By the end of 2019, a new β-CoV coronavirus has been recorded in Wuhan that also can transmit from animal to human [<a href="#ref9">9</a>]. Firstly, the World Health Organization (WHO) named it tentatively as 2019-Novel Coronavirus (2019-nCoV) on 12 January 2020. Secondly, the WHO organization named it formally as Coronavirus Disease 2019 (COVID-19) on 11 February 2020. At the same time, the ICTV committee named it severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2) [<a href="#ref6">6</a>, <a href="#ref10">10</a>]. The danger of COVID-19 started to explode when human–human contagion happened in a quick and terrifying shape. As of 14 May 2020, a total of 4 442 466 COVID-19 cases have been recorded all over the world including 298 322 deaths and 1 668 251 recoveries [<a href="#ref11">11</a>]. By reviewing the previous studies, it is clear that nearly all of them have the following paths.</p> <div class="analyzingthecovid-19virusfromamedicalpointofview" title="sec"> <div class="title" tagx="title" title="title">
Analyzing the COVID-19 virus from a medical point of view</div> <p>In 2020, Tia <i>et al.</i> [<a href="#ref1">1</a>] studied the characteristics of the COVID-19 receptor-binding domain (RBD) by implication for the development of RBD protein as a viral attachment inhibitor and vaccine. Zhou <i>et al.</i> [<a href="#ref3">3</a>] showed that COVID-19 belongs to the species of SARSr-CoV and also uses the same cell entry receptor that is called angiotensin-converting enzyme II (ACE2). Pradhan <i>et al.</i> [<a href="#ref4">4</a>] concentrated light on the development and pathogenicity of the COVID-19 virus with important implications for its diagnosis. Cao <i>et al.</i> [<a href="#ref12">12</a>] introduced a comparative genetic analysis of the COVID-19 receptor ACE2 in various populations.</p> </div> <div class="comparisonbetweendifferenttypesofcoronaviruses" title="sec"> <div class="title" tagx="title" title="title">
Comparison between different types of coronaviruses</div> <ul> <li> <p>Before the COVID-19 outbreak, in 2014, Drexler <i>et al.</i> [<a href="#ref13">13</a>] presented a general view of the obtainable studies on bat CoVs, then conducted a comprehensive phylogenetic analysis of the genera AlphaCov and BetaCov and compared the available data on CoV pathogenesis in bats including SARS-CoV, MERS-CoV and HCoV-229E. In 2015, Lu <i>et al.</i> [<a href="#ref14">14</a>] summarized the advance which has taken place in the past decade in understanding the cross-species transition of SARS- CoV and MERS-CoV by centering on the features of the surface-located spike (S) protein, its receptor obligated characteristics and the cleavage process involved in priming. In 2015, Lee <i>et al.</i> [<a href="#ref15">15</a>] discussed some issues which raised the possibility that inhibitor recognition specificity of MERS-CoV papain-like protease (PLpro) may differ from that of SARS-CoV PLpro. In 2017, Yuan <i>et al.</i> [<a href="#ref16">16</a>] presented high-resolution structures of the trimeric envelope spike (S) proteins of MERS-CoV and SARS-CoV in its pre-fusion conformation by single-particle cryo-electron microscopy.</p> </li> <li> <p>After the COVID-19 outbreak, in 2020, Ashor <i>et al.</i> [<a href="#ref7">7</a>] discussed the structure, genome organization and entry of CoVs into target cells and provide insights into SARS-CoV, MERS-CoV and COVID-19 outbreaks. Cai <i>et al.</i> [<a href="#ref17">17</a>] compared the clinical and pathological features between COVID-19 and SARS patients. Al-Tawfiq <i>et al.</i> [<a href="#ref18">18</a>] discussed the asymptomatic coronavirus infection through a comparison between MERS-CoV and COVID-19. Gorbalenya <i>et al.</i> [<a href="#ref19">19</a>] explained the classifying and naming of the novel COVID-19 epidemic among other coronaviruses. Liu <i>et al.</i> [<a href="#ref20">20</a>] provided a brief introduction to the pathology and pathogenesis of SARS-CoV and MERS-CoV and extrapolate this knowledge to the newly identified COVID-19. Ou <i>et al.</i> [<a href="#ref21">21</a>] studied the spike glycoprotein (S) characteristics of virus entry and its immune cross-reactivity in the case of SARS-CoV and COVID-19.</p> </li> </ul> <div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/"><span class="label" tagx="label" title="label">Figure 1</span></a></div>   <p>The block diagram of the proposed method.</p>   </div> <div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/"><span class="label" tagx="label" title="label">Figure 2</span></a></div>   <p>Cascade-forward back propagation network.</p>   </div> </div> <div class="classificationofcoronavirusesusingimageprocessingalgorithms" title="sec"> <div class="title" tagx="title" title="title">
Classification of coronaviruses using image processing algorithms</div> <p>In 2020, Barstugan <i>et al.</i> [<a href="#ref22">22</a>] introduced a COVID-19 classification method using different types of abdominal computed tomography (CT) images, five feature extraction methods and support vector machine (SVM) so that the best classification accuracy obtained is 99.6%. Basu <i>et al.</i> [<a href="#ref23">23</a>] determined characteristic features from chest X-ray images and used domain extension transfer learning (DETL) for an alternative screening of COVID-19 so that the overall accuracy was measured as 95.3% ± 0.02. Ozturk <i>et al.</i> [<a href="#ref24">24</a>] used raw chest X-ray images and the DarkCovidNet model as a classifier to make two types of classification that were binary classification (COVID versus no findings) and multi-class classification (COVID versus no-findings versus pneumonia), and this obtained accuracy of 98.08% for the binary classification and 87.02% for the multi-class case. Elasnaoui <i>et al.</i> [<a href="#ref25">25</a>] employed both X-ray and CT images for bacterial pneumonia, coronavirus, COVID-19 and normal cases and used different deep learning models so that the best classification accuracy obtained is 92.18%.</p> <p>On the other hand, the diagnosis of COVID-19 is currently based on some analyses performed in different laboratories and scan centers, for example, the reverse transcription-polymerase chain reaction (RT-PCR) test, the one-step real-time RT-PCR (RRT-PCR) test and the computed tomography (CT) scan recently [<a href="#ref26">26</a>, <a href="#ref27">27</a>]. Although these methods are to a large extent accurate, they have many drawbacks. Due to the patient number increase, there is a lack of test kits, in addition to the difficulties of the tests themselves such as the need for suction devices; the experienced operators’ availability; the patient exposure to pain because the sample is extracted from sputum, nasal swab and throat swab; and finally its high cost and long period of time to get results.</p> <p>In this paper, the three most common coronaviruses faced by humanity, in the last two decades, are studied. They are COVID-19, MERS-CoV and SARS-CoV that have made a big argument worldwide and have become dangerous epidemics that resulted in many death cases. This research treated them as genetic diseases, and it proposed a system with high-accuracy results that can detect the type of disease using its genome which leads to a rapid diagnosis of COVID-19 avoiding the disadvantages of the previous traditional diagnostic methods. The proposed system depends on employing the genomic signal processing (GSP) and the classification algorithms, as explained in the next section.</p> </div> </div> <div class="materialsandmethods" title="sec"> <div class="title" tagx="title" title="title">
Materials and methods</div> <p>The following block diagram summarizes the procedures of the research. Firstly, the selected DNA sequences are obtained from a suitable site according to the research need. Secondly, the DNA string values are transformed into its corresponding numerical values for easy use with the GSP methods. Thirdly, the GSP techniques were applied to get the required features. Fourthly, it is the time for the classification step using the selected classifiers. Finally, the obtained results are evaluated using different evaluation methods; see <a href="#f1">Figure 1</a>. Each step will be explained in detail as shown later in this section.</p> <div class="table-wrap_UNKNOWN" id="TB1" orientation="portrait" position="float"><span class="label" tagx="label" title="label">Table 1</span> <p>Resulting values in COVID-19/SARS-CoV and COVID-19/MERS-CoV classifications</p>  <table frame="hsides" rules="groups"> <colgroup span="1"> <col align="left" span="1" /></colgroup> <colgroup><col align="center" span="1" /></colgroup> <colgroup><col align="center" span="1" /></colgroup> <colgroup><col align="center" span="1" /></colgroup> <colgroup><col align="center" span="1" /></colgroup>  <thead> <tr> <th rowspan="1" colspan="1" /> <th colspan="2" align="center" rowspan="1">KNN classifier</th> <th colspan="2" align="center" rowspan="1">Cascade-Forward NN</th> </tr> <tr> <th rowspan="1" colspan="1" /> <th rowspan="1" colspan="1">(COVID-19/SARS-CoV)</th> <th rowspan="1" colspan="1">(COVID-19/MERS-CoV)</th> <th rowspan="1" colspan="1">(COVID-19/SARS-CoV)</th> <th rowspan="1" colspan="1">(COVID-19/MERS-CoV)</th> </tr> </thead> <tbody> <tr> <td rowspan="1" colspan="1">Tp</td> <td rowspan="1" colspan="1">30</td> <td rowspan="1" colspan="1">30</td> <td rowspan="1" colspan="1">30</td> <td rowspan="1" colspan="1">30</td> </tr> <tr> <td rowspan="1" colspan="1">Fp</td> <td rowspan="1" colspan="1">0</td> <td rowspan="1" colspan="1">0</td> <td rowspan="1" colspan="1">0</td> <td rowspan="1" colspan="1">0</td> </tr> <tr> <td rowspan="1" colspan="1">Tn</td> <td rowspan="1" colspan="1">30</td> <td rowspan="1" colspan="1">30</td> <td rowspan="1" colspan="1">30</td> <td rowspan="1" colspan="1">29</td> </tr> <tr> <td rowspan="1" colspan="1">Fn</td> <td rowspan="1" colspan="1">0</td> <td rowspan="1" colspan="1">0</td> <td rowspan="1" colspan="1">0</td> <td rowspan="1" colspan="1">1</td> </tr> </tbody> </table> </div> <div class="table-wrap_UNKNOWN" id="TB2" orientation="portrait" position="float"><span class="label" tagx="label" title="label">Table 2</span> <p>Resulting values in COVID-19/SARS-CoV/MERS-CoV classification</p>  <table frame="hsides" rules="groups"> <colgroup span="1"> <col align="left" span="1" /></colgroup> <colgroup><col align="center" span="1" /></colgroup> <colgroup><col align="center" span="1" /></colgroup>  <thead> <tr> <th rowspan="1" colspan="1" /> <th rowspan="1" colspan="1">KNN classifier</th> <th rowspan="1" colspan="1">Cascade-Forward NN</th> </tr> </thead> <tbody> <tr> <td rowspan="1" colspan="1">T<sub>1</sub></td> <td rowspan="1" colspan="1">30</td> <td rowspan="1" colspan="1">30</td> </tr> <tr> <td rowspan="1" colspan="1">F<sub>1</sub></td> <td rowspan="1" colspan="1">0</td> <td rowspan="1" colspan="1">0</td> </tr> <tr> <td rowspan="1" colspan="1">T<sub>2</sub></td> <td rowspan="1" colspan="1">30</td> <td rowspan="1" colspan="1">30</td> </tr> <tr> <td rowspan="1" colspan="1">F<sub>2</sub></td> <td rowspan="1" colspan="1">0</td> <td rowspan="1" colspan="1">0</td> </tr> <tr> <td rowspan="1" colspan="1">T<sub>3</sub></td> <td rowspan="1" colspan="1">30</td> <td rowspan="1" colspan="1">29</td> </tr> <tr> <td rowspan="1" colspan="1">F<sub>3</sub></td> <td rowspan="1" colspan="1">0</td> <td rowspan="1" colspan="1">1</td> </tr> </tbody> </table> </div> <div class="database" title="sec"> <div class="title" tagx="title" title="title">
Database</div> <p>The National Center for Biotechnology Information (NCBI) at the National Institutes of Health (NIH) is one of the most important sources of nucleotide databases for many different genetic diseases. These databases are called <a href="GenBank">GenBank</a> [<a href="#ref28">28</a>]. In this work, 76 complete genomes for each epidemic (COVID-19, SARS-CoV and MERS-CoV) are used. For each epidemic, each DNA sequence has a length of 30 000 nucleotides. RNA was extracted for whole genome sequencing of the viral isolate. Briefly, RNA was extracted from clarified cell culture supernatant and randomly amplified cDNA prepared by sequence-independent single-primer amplification (SISPA) [<a href="#ref29">29</a>, <a href="#ref30">30</a>]. Sequencing was performed with a combination of Oxford Nanopore Technologies and Illumina short-read sequencing. Genomic assembly of the BetaCoV/Australia/VIC/01/2020 genome was confirmed by parallel de novo and reference-guided methods [<a href="#ref31">31</a>].</p> <div class="table-wrap_UNKNOWN" id="TB3" orientation="portrait" position="float"><span class="label" tagx="label" title="label">Table 3</span> <p>Comparison of calculated parameters for the three systems</p>  <table frame="hsides" rules="groups"> <colgroup span="1"> <col align="left" span="1" /></colgroup> <colgroup><col align="center" span="1" /></colgroup> <colgroup><col align="center" span="1" /></colgroup> <colgroup><col align="center" span="1" /></colgroup> <colgroup><col align="center" span="1" /></colgroup> <colgroup><col align="center" span="1" /></colgroup> <colgroup><col align="center" span="1" /></colgroup>  <thead> <tr> <th rowspan="1" colspan="1" /> <th colspan="3" align="center" rowspan="1">KNN classifier</th> <th colspan="3" align="center" rowspan="1">Cascade-Forward NN</th> </tr> <tr> <th rowspan="1" colspan="1" /> <th rowspan="1" colspan="1">(COVID-19/SARS-CoV)</th> <th rowspan="1" colspan="1">(COVID-19/MERS-CoV)</th> <th rowspan="1" colspan="1">(COVID-19/SARS-CoV/MERS-CoV)</th> <th rowspan="1" colspan="1">(COVID-19/SARS-CoV)</th> <th rowspan="1" colspan="1">(COVID-19/MERS-CoV)</th> <th rowspan="1" colspan="1">(COVID-19/SARS-CoV/MERS-CoV)</th> </tr> </thead> <tbody> <tr> <td rowspan="1" colspan="1">ACC %</td> <td rowspan="1" colspan="1">100</td> <td rowspan="1" colspan="1">100</td> <td rowspan="1" colspan="1">100</td> <td rowspan="1" colspan="1">100</td> <td rowspan="1" colspan="1">98.33</td> <td rowspan="1" colspan="1">98.89</td> </tr> <tr> <td rowspan="1" colspan="1">Error rate</td> <td rowspan="1" colspan="1">0</td> <td rowspan="1" colspan="1">0</td> <td rowspan="1" colspan="1">0</td> <td rowspan="1" colspan="1">0</td> <td rowspan="1" colspan="1">0.0166</td> <td rowspan="1" colspan="1">0.0111</td> </tr> <tr> <td rowspan="1" colspan="1">MCC</td> <td rowspan="1" colspan="1">1</td> <td rowspan="1" colspan="1">1</td> <td rowspan="1" colspan="1">1</td> <td rowspan="1" colspan="1">1</td> <td rowspan="1" colspan="1">0.9672</td> <td rowspan="1" colspan="1">0.9754</td> </tr> <tr> <td rowspan="1" colspan="1">F1%</td> <td rowspan="1" colspan="1">100</td> <td rowspan="1" colspan="1">100</td> <td rowspan="1" colspan="1">100</td> <td rowspan="1" colspan="1">100</td> <td rowspan="1" colspan="1">98.36</td> <td rowspan="1" colspan="1">98.34</td> </tr> </tbody> </table> </div> </div> <div class="numberingmethod" title="sec"> <div class="title" tagx="title" title="title">
Numbering method</div> <p>There are various mapping methods used for converting the nucleotide bases from strings A, T, C and G to numbers, so the GSP techniques can be applied to them. The electron–ion interaction pseudopotential (EIIP) representation is one of these mapping methods [<a href="#ref32">32–35</a>]. The EIIP values symbolize the free electrons’ energy distribution along the nucleotide sequence [<a href="#ref36">36</a>]. These values are 0.1260, 0.1335, 0.1340 and 0.0806 for A, T, C and G, respectively.</p> <div class="table-wrap_UNKNOWN" id="TB4" orientation="portrait" position="float"><span class="label" tagx="label" title="label">Table 4</span> <p>Comparison of different classification studies</p>  <table frame="hsides" rules="groups"> <colgroup span="1"> <col align="left" span="1" /></colgroup> <colgroup><col align="left" span="1" /></colgroup> <colgroup><col align="left" span="1" /></colgroup> <colgroup><col align="left" span="1" /></colgroup> <colgroup><col align="center" span="1" /></colgroup>  <thead> <tr> <th rowspan="1" colspan="1">Study</th> <th rowspan="1" colspan="1">Dataset</th> <th rowspan="1" colspan="1">Method</th> <th rowspan="1" colspan="1">Classifier</th> <th rowspan="1" colspan="1">Accuracy %</th> </tr> </thead> <tbody> <tr> <td rowspan="1" colspan="1">Barstugan <i>et al.</i> [<a href="#ref22">22</a>]</td> <td rowspan="1" colspan="1">Abdominal CT images: 16 × 16</td> <td rowspan="1" colspan="1"> <p>− Grey-level co-occurrence matrix (GLCM)</p> <p>− Local directional pattern (LDP)</p> <p>− Grey-level run length matrix (GLRLM)</p> <p>− Grey-level size zone matrix (GLSZM)</p> <p>− Discrete wavelet transform (DWT)</p> </td> <td rowspan="1" colspan="1">Support vector machine (SVM)</td> <td rowspan="1" colspan="1">99.68</td> </tr> <tr> <td rowspan="1" colspan="1" /> <td rowspan="1" colspan="1">32 × 32</td> <td rowspan="1" colspan="1" /> <td rowspan="1" colspan="1" /> <td rowspan="1" colspan="1">99.37</td> </tr> <tr> <td rowspan="1" colspan="1" /> <td rowspan="1" colspan="1">48 × 48</td> <td rowspan="1" colspan="1" /> <td rowspan="1" colspan="1" /> <td rowspan="1" colspan="1">99.64</td> </tr> <tr> <td rowspan="1" colspan="1" /> <td rowspan="1" colspan="1">64 × 64</td> <td rowspan="1" colspan="1" /> <td rowspan="1" colspan="1" /> <td rowspan="1" colspan="1">97.28</td> </tr> <tr> <td rowspan="1" colspan="1">Basu <i>et al.</i> [<a href="#ref23">23</a>]</td> <td rowspan="1" colspan="1">Chest X-ray images</td> <td rowspan="1" colspan="1">Domain extension transfer learning (DETL)</td> <td rowspan="1" colspan="1">Machine learning (ML)<br /><br />Deep learning (DL)</td> <td rowspan="1" colspan="1">95.3% ± 0.02</td> </tr> <tr> <td rowspan="1" colspan="1">Ozturk <i>et al.</i> [<a href="#ref24">24</a>]</td> <td rowspan="1" colspan="1">Raw chest X-ray images:</td> <td rowspan="1" colspan="1">DarkCovidNet model</td> <td rowspan="1" colspan="1">Deep learning (DL)</td> <td rowspan="1" colspan="1">98.08</td> </tr> <tr> <td rowspan="1" colspan="1" /> <td rowspan="1" colspan="1">− Binary classification (COVID versus no-findings)</td> <td rowspan="1" colspan="1" /> <td rowspan="1" colspan="1" /> <td rowspan="1" colspan="1" /> </tr> <tr> <td rowspan="1" colspan="1" /> <td rowspan="1" colspan="1">− Multi-class classification (COVID versus no-findings versus pneumonia)</td> <td rowspan="1" colspan="1" /> <td rowspan="1" colspan="1" /> <td rowspan="1" colspan="1">87.02</td> </tr> <tr> <td rowspan="1" colspan="1">Elasnaoui <i>et al.</i> [<a href="#ref25">25</a>]</td> <td rowspan="1" colspan="1">X-ray and CT images</td> <td rowspan="1" colspan="1">Models: Inception_ResNet_V2</td> <td rowspan="1" colspan="1">Multilayer perceptron classifier</td> <td rowspan="1" colspan="1">92.18</td> </tr> <tr> <td rowspan="1" colspan="1" /> <td rowspan="1" colspan="1">DenseNet201</td> <td rowspan="1" colspan="1" /> <td rowspan="1" colspan="1" /> <td rowspan="1" colspan="1">88.09</td> </tr> <tr> <td rowspan="1" colspan="1" /> <td rowspan="1" colspan="1">Resnet50</td> <td rowspan="1" colspan="1" /> <td rowspan="1" colspan="1" /> <td rowspan="1" colspan="1">87.54</td> </tr> <tr> <td rowspan="1" colspan="1" /> <td rowspan="1" colspan="1">MobileNet_V2</td> <td rowspan="1" colspan="1" /> <td rowspan="1" colspan="1" /> <td rowspan="1" colspan="1">85.47</td> </tr> <tr> <td rowspan="1" colspan="1" /> <td rowspan="1" colspan="1">Inception_V3</td> <td rowspan="1" colspan="1" /> <td rowspan="1" colspan="1" /> <td rowspan="1" colspan="1">88.03</td> </tr> <tr> <td rowspan="1" colspan="1" /> <td rowspan="1" colspan="1">VGG16</td> <td rowspan="1" colspan="1" /> <td rowspan="1" colspan="1" /> <td rowspan="1" colspan="1">74.84</td> </tr> <tr> <td rowspan="1" colspan="1" /> <td rowspan="1" colspan="1">VGG19</td> <td rowspan="1" colspan="1" /> <td rowspan="1" colspan="1" /> <td rowspan="1" colspan="1">72.52</td> </tr> <tr> <td rowspan="1" colspan="1">Proposed study</td> <td rowspan="1" colspan="1">DNA sequences:</td> <td rowspan="1" colspan="1">− Discrete Fourier transform</td> <td rowspan="1" colspan="1"><i>K</i>-nearest neighbor algorithm</td> <td rowspan="1" colspan="1">100</td> </tr> <tr> <td rowspan="1" colspan="1" /> <td rowspan="1" colspan="1" /> <td rowspan="1" colspan="1">− Discrete cosine transform</td> <td rowspan="1" colspan="1" /> <td rowspan="1" colspan="1" /> </tr> <tr> <td rowspan="1" colspan="1" /> <td rowspan="1" colspan="1">− COVID-19 versus SARS-CoV</td> <td rowspan="1" colspan="1">− Seven moment invariants</td> <td rowspan="1" colspan="1" /> <td rowspan="1" colspan="1" /> </tr> <tr> <td rowspan="1" colspan="1" /> <td rowspan="1" colspan="1">− COVID-19 versus MERS-CoV</td> <td rowspan="1" colspan="1" /> <td rowspan="1" colspan="1" /> <td rowspan="1" colspan="1">100</td> </tr> <tr> <td rowspan="1" colspan="1" /> <td rowspan="1" colspan="1">− COVID-19 versus SARS-CoV versus MERS-CoV</td> <td rowspan="1" colspan="1" /> <td rowspan="1" colspan="1" /> <td rowspan="1" colspan="1">100</td> </tr> <tr> <td rowspan="1" colspan="1" /> <td rowspan="1" colspan="1">− COVID-19 versus SARS-CoV</td> <td rowspan="1" colspan="1" /> <td rowspan="1" colspan="1">Trainable cascade-forward back propagation neural network</td> <td rowspan="1" colspan="1">100</td> </tr> <tr> <td rowspan="1" colspan="1" /> <td rowspan="1" colspan="1">− COVID-19 versus MERS-CoV</td> <td rowspan="1" colspan="1" /> <td rowspan="1" colspan="1" /> <td rowspan="1" colspan="1">98.34</td> </tr> <tr> <td rowspan="1" colspan="1" /> <td rowspan="1" colspan="1">− COVID-19 versus SARS-CoV versus MERS-CoV</td> <td rowspan="1" colspan="1" /> <td rowspan="1" colspan="1" /> <td rowspan="1" colspan="1">98.89</td> </tr> </tbody> </table> </div> </div> <div class="featureextraction" title="sec"> <div class="title" tagx="title" title="title">
Feature extraction</div> <div class="discretefouriertransform(dft)" title="sec"> <div class="title" tagx="title" title="title">
Discrete Fourier transform (DFT)</div> <p>In the discrete Fourier transform, discrete time data sets are converted into a discrete frequency representation [<a href="#ref37">37</a>]. The DFT is a numerical variant of the Fourier transform. Specifically, given a vector of n input amplitudes with length N, the DFT yields a set of frequency magnitudes and is defined as: </p><div class="disp-formula" title="disp-formula"><span class="label" tagx="label" title="label">(1)</span><div class="tex" title="tex"> <div class="tex-math_UNKNOWN" id="M1">
\documentclass[12pt]{minimal} \usepackage{amsmath} \usepackage{wasysym} \usepackage{amsfonts} \usepackage{amssymb} \usepackage{amsbsy} \usepackage{upgreek} \usepackage{mathrsfs} \setlength{\oddsidemargin}{-69pt} \begin{document} }{}\begin{equation*} X\left[k\right]=\sum_{n=0}^{N-1}x\left[n\right]{e}^{\frac{-j2\pi kn}{N}} \end{equation*}\end{document}</div> </div> </div> where <i>k</i> denotes the frequency domain ordinal, <i>n</i> represents the time domain ordinal and <i>N</i> is the length of the sequence to be transformed. To reduce the feature number resulted from using the DFT, the mean of its values is used to get only 1 feature instead of about 30 000 features for each sequence.<p /> </div> <div class="discretecosinetransform(dct)" title="sec"> <div class="title" tagx="title" title="title">
Discrete cosine transform (DCT)</div> <p>The DFT transforms a complex signal into its complex spectrum. If the signal is real (as in most of applications), half of the data is redundant, i.e. half of the computation is wasted. In time domain the imaginary part of the signal is all zero, and in frequency domain, both real and imaginary parts of the spectrum are symmetric [<a href="#ref38">38</a>]. Discrete cosine transform (DCT) generates real spectrum of a real signal and thereby avoids redundant data and computation [<a href="#ref39">39</a>]. The DCT of a real sequence, x (n), with length N is defined as: </p><div class="disp-formula" title="disp-formula"><span class="label" tagx="label" title="label">(2)</span><div class="tex" title="tex"> <div class="tex-math_UNKNOWN" id="M2">
\documentclass[12pt]{minimal} \usepackage{amsmath} \usepackage{wasysym} \usepackage{amsfonts} \usepackage{amssymb} \usepackage{amsbsy} \usepackage{upgreek} \usepackage{mathrsfs} \setlength{\oddsidemargin}{-69pt} \begin{document} }{}\begin{equation*} d(k)=u(k)\sum_{n=1}^Nx(n)\cos \frac{\pi \left(2n-1\right)\left(k-1\right)}{2N},k=1,..,N \end{equation*}\end{document}</div> </div> </div> <div class="disp-formula" title="disp-formula"><span class="label" tagx="label" title="label">(3)</span><div class="tex" title="tex"> <div class="tex-math_UNKNOWN" id="M3">
\documentclass[12pt]{minimal} \usepackage{amsmath} \usepackage{wasysym} \usepackage{amsfonts} \usepackage{amssymb} \usepackage{amsbsy} \usepackage{upgreek} \usepackage{mathrsfs} \setlength{\oddsidemargin}{-69pt} \begin{document} }{}\begin{equation*} u(k)=\left\{\kern-5pt\begin{array}{c}1/\sqrt{N},k=1\ \\{}\sqrt{2/N}, 2\le k\le N\ \end{array}\right. \end{equation*}\end{document}</div> </div> </div> <p /> <p>Similarly in DCT features, the variance of its values is taken to reduce the feature number from about 3000 features to only one for each sequence.</p> </div> <div class="momentinvariants" title="sec"> <div class="title" tagx="title" title="title">
Moment invariants</div> <p>In this step, moment invariants are used as features for the classification. They are first introduced by Hu in [<a href="#ref40">40</a>]; next of that different researches applied them as in [<a href="#ref41">41–44</a>]. The seven moment invariants are defined as six absolute orthogonal invariants and one skew orthogonal invariant. These invariants are constructed using the generalized fundamental theorem of moment invariants (GFTMI) [<a href="#ref45">45</a>].</p> <p>For a function of intensity <i>α</i>(<i>x</i><sub>1</sub>, … ., <i>x<sub>n</sub></i>) = <i>α</i>(<i>y</i>), the <i>n</i>-dimensional moments of order <i>r</i> are calculated in terms of Riemann integral [<a href="#ref46">46</a>] as follows: </p><div class="disp-formula" title="disp-formula"><span class="label" tagx="label" title="label">(4)</span><div class="tex" title="tex"> <div class="tex-math_UNKNOWN" id="M4">
\documentclass[12pt]{minimal} \usepackage{amsmath} \usepackage{wasysym} \usepackage{amsfonts} \usepackage{amssymb} \usepackage{amsbsy} \usepackage{upgreek} \usepackage{mathrsfs} \setlength{\oddsidemargin}{-69pt} \begin{document} }{}\begin{equation*} {v}_{r1\dots rn}=\int \dots \int{x}_1^{r1}\dots \dots{x}_n^{rn}\ \alpha (x)\ d{x}_1\dots .d{x}_n \end{equation*}\end{document}</div> </div> </div> where <i>ri +</i> … <i>rn = r,</i> 0 <i>&amp;lt; r &amp;lt; ∞.</i> Then, the central moment will be: <div class="disp-formula" title="disp-formula"><span class="label" tagx="label" title="label">(5)</span><div class="tex" title="tex"> <div class="tex-math_UNKNOWN" id="M5">
\documentclass[12pt]{minimal} \usepackage{amsmath} \usepackage{wasysym} \usepackage{amsfonts} \usepackage{amssymb} \usepackage{amsbsy} \usepackage{upgreek} \usepackage{mathrsfs} \setlength{\oddsidemargin}{-69pt} \begin{document} }{}\begin{equation*} {\mu}_{r1\dots . rn}=\underset{-\infty }{\overset{\infty }{\int }}\dots \underset{-\infty }{\overset{\infty }{\int }}\dots \dots \Big({x}_n-{\overline{x}}_n\alpha (x)\ d{x}_1\dots d{x}_n \end{equation*}\end{document}</div> </div> </div> where <div class="disp-formula" title="disp-formula"><span class="label" tagx="label" title="label">(6)</span><div class="tex" title="tex"> <div class="tex-math_UNKNOWN" id="M6">
\documentclass[12pt]{minimal} \usepackage{amsmath} \usepackage{wasysym} \usepackage{amsfonts} \usepackage{amssymb} \usepackage{amsbsy} \usepackage{upgreek} \usepackage{mathrsfs} \setlength{\oddsidemargin}{-69pt} \begin{document} }{}\begin{equation*} \overline{x}=\frac{v_{1\dots 0}}{v_{0\dots 0}},\dots \dots, {\overline{x}}_n=\frac{v_{0\dots 1}}{v_{0\dots 0}} \end{equation*}\end{document}</div> </div> </div> <p /> <p>Finally, to get the seven moment invariants, use these equations: </p><div class="disp-formula" title="disp-formula"><span class="label" tagx="label" title="label">(7)</span><div class="tex" title="tex"> <div class="tex-math_UNKNOWN" id="M7">
\documentclass[12pt]{minimal} \usepackage{amsmath} \usepackage{wasysym} \usepackage{amsfonts} \usepackage{amssymb} \usepackage{amsbsy} \usepackage{upgreek} \usepackage{mathrsfs} \setlength{\oddsidemargin}{-69pt} \begin{document} }{}\begin{equation*} {\rho}_1=\frac{1}{\mu^4}\left|\begin{array}{ccc}{\mu}_{2\dots 0}&amp;amp; \dots &amp;amp; {\mu}_{1\dots 1}\\{}\dots &amp;amp; \dots &amp;amp; \dots \\{}{\mu}_{1\dots 1}&amp;amp; \dots &amp;amp; {\mu}_{0\dots 2}\end{array}\right| \end{equation*}\end{document}</div> </div> </div> <div class="disp-formula" title="disp-formula"><span class="label" tagx="label" title="label">(8)</span><div class="tex" title="tex"> <div class="tex-math_UNKNOWN" id="M8">
\documentclass[12pt]{minimal} \usepackage{amsmath} \usepackage{wasysym} \usepackage{amsfonts} \usepackage{amssymb} \usepackage{amsbsy} \usepackage{upgreek} \usepackage{mathrsfs} \setlength{\oddsidemargin}{-69pt} \begin{document} }{}\begin{equation*} {\rho}_2=\frac{1}{\mu^4}\left({\mu}_{20}{\mu}_{02}-{\mu}_{11}^2\right) \end{equation*}\end{document}</div> </div> </div> <div class="disp-formula" title="disp-formula"><span class="label" tagx="label" title="label">(9)</span><div class="tex" title="tex"> <div class="tex-math_UNKNOWN" id="M9">
\documentclass[12pt]{minimal} \usepackage{amsmath} \usepackage{wasysym} \usepackage{amsfonts} \usepackage{amssymb} \usepackage{amsbsy} \usepackage{upgreek} \usepackage{mathrsfs} \setlength{\oddsidemargin}{-69pt} \begin{document} }{}\begin{equation*} {\rho}_3=\frac{1}{\mu^{10}}\left[{\left({\mu}_{30}{\mu}_{03}-{\mu}_{21}{\mu}_{12}\right)}^2-4\left({\mu}_{30}{\mu}_{12}-{\mu}_{21}^2\right)\left({\mu}_{21}{\mu}_{03}-{\mu}_{12}^2\right)\right] \end{equation*}\end{document}</div> </div> </div> <div class="disp-formula" title="disp-formula"><span class="label" tagx="label" title="label">(10)</span><div class="tex" title="tex"> <div class="tex-math_UNKNOWN" id="M10">
\documentclass[12pt]{minimal} \usepackage{amsmath} \usepackage{wasysym} \usepackage{amsfonts} \usepackage{amssymb} \usepackage{amsbsy} \usepackage{upgreek} \usepackage{mathrsfs} \setlength{\oddsidemargin}{-69pt} \begin{document} }{}\begin{equation*} {\rho}_4=\frac{1}{\mu^6}\left({\mu}_{40}{\mu}_{04}-4{\mu}_{31}{\mu}_{13}-3{\mu}_{22}^2\right) \end{equation*}\end{document}</div> </div> </div> <div class="disp-formula" title="disp-formula"><span class="label" tagx="label" title="label">(11)</span><div class="tex" title="tex"> <div class="tex-math_UNKNOWN" id="M11">
\documentclass[12pt]{minimal} \usepackage{amsmath} \usepackage{wasysym} \usepackage{amsfonts} \usepackage{amssymb} \usepackage{amsbsy} \usepackage{upgreek} \usepackage{mathrsfs} \setlength{\oddsidemargin}{-69pt} \begin{document} }{}\begin{equation*} {\rho}_5=\frac{1}{\mu^9}\left({\mu}_{40}{\mu}_{22}{\mu}_{04}+2{\mu}_{31}{\mu}_{22}{\mu}_{13}-{\mu}_{40}{\mu}_{13}^2-{\mu}_{31}^2{\mu}_{04}-{\mu}_{22}^3\right) \end{equation*}\end{document}</div> </div> </div> <div class="disp-formula" title="disp-formula"><span class="label" tagx="label" title="label">(12)</span><div class="tex" title="tex"> <div class="tex-math_UNKNOWN" id="M12">
\documentclass[12pt]{minimal} \usepackage{amsmath} \usepackage{wasysym} \usepackage{amsfonts} \usepackage{amssymb} \usepackage{amsbsy} \usepackage{upgreek} \usepackage{mathrsfs} \setlength{\oddsidemargin}{-69pt} \begin{document} }{}\begin{eqnarray*} &amp;amp;&amp;amp; {\rho}_6=\frac{1}{\mu^5}\left({\mu}_{200}{\mu}_{020}{\mu}_{002}+2{\mu}_{110}{\mu}_{101}{\mu}_{011} \right. \\ &amp;amp;&amp;amp; \quad \left. - {\mu}_{200}{\mu}_{011}^2-{\mu}_{110}^2{\mu}_{002}-{\mu}_{101}^2{\mu}_{020}\right)\nonumber \end{eqnarray*}\end{document}</div> </div> </div> <div class="disp-formula" title="disp-formula"><span class="label" tagx="label" title="label">(13)</span><div class="tex" title="tex"> <div class="tex-math_UNKNOWN" id="M13">
\documentclass[12pt]{minimal} \usepackage{amsmath} \usepackage{wasysym} \usepackage{amsfonts} \usepackage{amssymb} \usepackage{amsbsy} \usepackage{upgreek} \usepackage{mathrsfs} \setlength{\oddsidemargin}{-69pt} \begin{document} }{}\begin{eqnarray*}&amp;amp;&amp;amp; {\rho}_7=\left({\mu}_{20}^2{\mu}_{04}\right)-4{\mu}_{20}{\mu}_{11}{\mu}_{13}+2{\mu}_{20}{\mu}_{02}{\mu}_{22}+4{\mu}_{11}^2{\mu}_{22} \\ &amp;amp;&amp;amp; \quad -4{\mu}_{11}{\mu}_{02}{\mu}_{31}+{\mu}_{02}^2{\mu}_{40}\nonumber \end{eqnarray*}\end{document}</div> </div> </div> where <i>ρ</i><sub>1</sub> to <i>ρ</i><sub>7</sub> represents the seven moment invariants, <i>μ</i> is the central moment gotten from Equation (<a href="#deqn05">5</a>), <i>α</i> is the intensity function and <i>ν<sub>r</sub></i> is the <i>n</i>-dimensional moments of order <i>r</i> obtained from Equation (<a href="#deqn04">4</a>).<p /> </div> </div> <div class="classification" title="sec"> <div class="title" tagx="title" title="title">
Classification</div> <p>The research uses two types of classifiers: the <i>k</i>-nearest neighbor (KNN) algorithm and trainable cascade-forward back propagation neural network for the classification step.</p> <div class="k-nearestneighbor(knn)" title="sec"> <div class="title" tagx="title" title="title"><i>K</i>-nearest neighbor (KNN)</div> <p>The K-nearest algorithm is a learning method based on states, so it doesn’t want a learning stage. It treats with the training pattern that is related to a distance function and the choice of the class function based on the classes of the nearest neighbors [<a href="#ref41">41</a>, <a href="#ref42">42</a>]. When a new pattern is classified, it must be compared to others using a similarity measure taking into consideration the <i>k</i>-neighbors. The distance between the new pattern and the neighbor is used as the weight [<a href="#ref47">47–49</a>]. The most prevalent method to measure this distance is Euclidean. To measure the Euclidean distance between two vectors p<sub>i</sub> and p<sub>j</sub>, the equation mentioned in [<a href="#ref50">50</a>] will be used: </p><div class="disp-formula" title="disp-formula"><span class="label" tagx="label" title="label">(14)</span><div class="tex" title="tex"> <div class="tex-math_UNKNOWN" id="M14">
\documentclass[12pt]{minimal} \usepackage{amsmath} \usepackage{wasysym} \usepackage{amsfonts} \usepackage{amssymb} \usepackage{amsbsy} \usepackage{upgreek} \usepackage{mathrsfs} \setlength{\oddsidemargin}{-69pt} \begin{document} }{}\begin{equation*} D\left({p}_i,{p}_j\right)=\sqrt{\sum_{r=1}^n{\left({p}_{ir}-{p}_{jr}\right)}^2} \end{equation*}\end{document}</div> </div> </div> <p /> </div> <div class="trainablecascade-forwardbackpropagationneuralnetwork" title="sec"> <div class="title" tagx="title" title="title">
Trainable cascade-forward back propagation neural network</div> <p>Despite the cascade-forward NN being similar to feed-forward NN, it contains a weighted link from the input layer to each layer and from each layer to the next layers; see <a href="#f2">Figure 2</a>. For a given three-layer network, there are connections from layer one to layer two, layer two to layer three and layer one to layer three. In addition to this, it includes connections from the input layer to all three layers [<a href="#ref51">51</a>]. The extra links might get better speed at which the network learns the desired relations [<a href="#ref52">52</a>].</p> <p>MATLAB R2017b is used to execute the proposed system. The <i>fitcknn</i> function is used for creating the <i>k</i>-nearest neighbor network with its default number of neighbors <i>k</i> = 1. The <i>cascadeforwardnet</i> function is used for creating the cascade-forward network (supervised learning machine algorithm). It uses one input layer that has nine neurons equal to the number of features, one hidden layer that has the default number of neurons <i>n</i> = 10 and one output layer that has one neuron according to the resulting class.</p> <p>For the classification process, the training data consists of 46 complete genomes, and the testing data consists of 30 complete genomes for each epidemic. In this step, three classification processes are done for every classifier: firstly, creating a system to distinguish between COVID-19 and SARS-CoV epidemics; secondly, generating a system to set apart between COVID-19 and MERS-CoV; thirdly, building a complete system to differentiate between the three pandemics; and finally, comparing the output results from the two used classifiers for all processes.</p> </div> </div> <div class="evaluation" title="sec"> <div class="title" tagx="title" title="title">
Evaluation</div> <p>For the two-class (COVID-19/SARS-CoV and COVID-19/MERS-CoV) classifications, some effective evaluation parameters are calculated. They are accuracy (ACC), error rate, Matthews correlation coefficient (MCC) [<a href="#ref53">53</a>] and F1 score (the coordinated mean of precision and sensitivity) [<a href="#ref54">54</a>, <a href="#ref55">55</a>] as follows: </p><div class="disp-formula" title="disp-formula"><span class="label" tagx="label" title="label">(15)</span><div class="tex" title="tex"> <div class="tex-math_UNKNOWN" id="M15">
\documentclass[12pt]{minimal} \usepackage{amsmath} \usepackage{wasysym} \usepackage{amsfonts} \usepackage{amssymb} \usepackage{amsbsy} \usepackage{upgreek} \usepackage{mathrsfs} \setlength{\oddsidemargin}{-69pt} \begin{document} }{}\begin{equation*} \mathrm{ACC}=\frac{\mathrm{Tp}+\mathrm{Tn}}{\mathrm{Tn}+\mathrm{Tn}+\mathrm{Fp}+\mathrm{Fn}}\times 100\% \end{equation*}\end{document}</div> </div> </div> <p /> <p>(ACC: worst value = 0; best value = 100) </p><div class="disp-formula" title="disp-formula"><span class="label" tagx="label" title="label">(16)</span><div class="tex" title="tex"> <div class="tex-math_UNKNOWN" id="M16">
\documentclass[12pt]{minimal} \usepackage{amsmath} \usepackage{wasysym} \usepackage{amsfonts} \usepackage{amssymb} \usepackage{amsbsy} \usepackage{upgreek} \usepackage{mathrsfs} \setlength{\oddsidemargin}{-69pt} \begin{document} }{}\begin{equation*} \mathrm{Error}\ \mathrm{rate}=\frac{\mathrm{Fp}+\mathrm{Fn}}{\mathrm{Tn}+\mathrm{Tn}+\mathrm{Fp}+\mathrm{Fn}} \end{equation*}\end{document}</div> </div> </div> <p /> <p>(Error rate: worst value = 1; best value = 0) </p><div class="disp-formula" title="disp-formula"><span class="label" tagx="label" title="label">(17)</span><div class="tex" title="tex"> <div class="tex-math_UNKNOWN" id="M17">
\documentclass[12pt]{minimal} \usepackage{amsmath} \usepackage{wasysym} \usepackage{amsfonts} \usepackage{amssymb} \usepackage{amsbsy} \usepackage{upgreek} \usepackage{mathrsfs} \setlength{\oddsidemargin}{-69pt} \begin{document} }{}\begin{equation*} \mathrm{MCC}=\frac{\left(\mathrm{Tp}\times \mathrm{Tn}\right)-\left(\mathrm{Fp}\times \mathrm{Fn}\right)}{\sqrt{\left(\mathrm{Tp}+\mathrm{Fp}\right)\left(\mathrm{Tp}+\mathrm{Fn}\right)\left(\mathrm{Tn}+\mathrm{Fp}\right)\left(\mathrm{Tn}+\mathrm{Fn}\right)}} \end{equation*}\end{document}</div> </div> </div> <p /> <p>(MCC: worst value = −1; best value = +1) </p><div class="disp-formula" title="disp-formula"><span class="label" tagx="label" title="label">(18)</span><div class="tex" title="tex"> <div class="tex-math_UNKNOWN" id="M18">
\documentclass[12pt]{minimal} \usepackage{amsmath} \usepackage{wasysym} \usepackage{amsfonts} \usepackage{amssymb} \usepackage{amsbsy} \usepackage{upgreek} \usepackage{mathrsfs} \setlength{\oddsidemargin}{-69pt} \begin{document} }{}\begin{equation*} {F}_{\mathrm{Score}}=\frac{2\mathrm{Tp}}{2\mathrm{Tp}+\mathrm{Fp}+\mathrm{Fn}}\times 100\% \end{equation*}\end{document}</div> </div> </div> <p /> <p>(F1 score: worst value = 0; best value = 100)</p> <p>where Tp, Tn, Fp and Fn stand for true positive, true negative, false-positive and false-negative values, respectively.</p> <p>For the multi-class (COVID-19/SARS-CoV/MERS-CoV) classification, the same previous parameters are calculated, but in macro-averaging level as mentioned in [<a href="#ref56">56</a>]: </p><div class="disp-formula" title="disp-formula"><span class="label" tagx="label" title="label">(19)</span><div class="tex" title="tex"> <div class="tex-math_UNKNOWN" id="M19">
\documentclass[12pt]{minimal} \usepackage{amsmath} \usepackage{wasysym} \usepackage{amsfonts} \usepackage{amssymb} \usepackage{amsbsy} \usepackage{upgreek} \usepackage{mathrsfs} \setlength{\oddsidemargin}{-69pt} \begin{document} }{}\begin{equation*} \mathrm{Average}\ \mathrm{ACC}=\frac{\sum_{i=1}^m\frac{{\mathrm{Tp}}_i+{\mathrm{Tn}}_i}{{\mathrm{Tn}}_i+{\mathrm{Tn}}_i+{\mathrm{Fp}}_i+{\mathrm{Fn}}_i}}{m}\times 100\% \end{equation*}\end{document}</div> </div> </div> <p /> <p>(Average ACC: worst value = 0; best value = 100) </p><div class="disp-formula" title="disp-formula"><span class="label" tagx="label" title="label">(20)</span><div class="tex" title="tex"> <div class="tex-math_UNKNOWN" id="M20">
\documentclass[12pt]{minimal} \usepackage{amsmath} \usepackage{wasysym} \usepackage{amsfonts} \usepackage{amssymb} \usepackage{amsbsy} \usepackage{upgreek} \usepackage{mathrsfs} \setlength{\oddsidemargin}{-69pt} \begin{document} }{}\begin{equation*} \mathrm{Average}\ \mathrm{error}\ \mathrm{rate}=\frac{\sum_{i=1}^m\frac{{\mathrm{Fp}}_i+{\mathrm{Fn}}_i}{{\mathrm{Tn}}_i+{\mathrm{Tn}}_i+{\mathrm{Fp}}_i+{\mathrm{Fn}}_i}}{m} \end{equation*}\end{document}</div> </div> </div> <p /> <p>(Average error rate: worst value = 1; best value = 0) </p><div class="disp-formula" title="disp-formula"><span class="label" tagx="label" title="label">(21)</span><div class="tex" title="tex"> <div class="tex-math_UNKNOWN" id="M21">
\documentclass[12pt]{minimal} \usepackage{amsmath} \usepackage{wasysym} \usepackage{amsfonts} \usepackage{amssymb} \usepackage{amsbsy} \usepackage{upgreek} \usepackage{mathrsfs} \setlength{\oddsidemargin}{-69pt} \begin{document} }{}\begin{equation*} \mathrm{Average}\ \mathrm{MCC}=\frac{\sum_{i=1}^m\frac{\left({\mathrm{Tp}}_i\times{\mathrm{Tn}}_i\right)-\left({\mathrm{Fp}}_i\times{\mathrm{Fn}}_i\right)}{\sqrt{\left({\mathrm{Tp}}_i+{\mathrm{Fp}}_i\right)\left({\mathrm{Tp}}_i+{\mathrm{Fn}}_i\right)\left({\mathrm{Tn}}_i+{\mathrm{Fp}}_i\right)\left({\mathrm{Tn}}_i+{\mathrm{Fn}}_i\right)}}}{m} \end{equation*}\end{document}</div> </div> </div> <p /> <p>(Average MCC: worst value = −1; best value = +1) </p><div class="disp-formula" title="disp-formula"><span class="label" tagx="label" title="label">(22)</span><div class="tex" title="tex"> <div class="tex-math_UNKNOWN" id="M22">
\documentclass[12pt]{minimal} \usepackage{amsmath} \usepackage{wasysym} \usepackage{amsfonts} \usepackage{amssymb} \usepackage{amsbsy} \usepackage{upgreek} \usepackage{mathrsfs} \setlength{\oddsidemargin}{-69pt} \begin{document} }{}\begin{equation*} \mathrm{Average}\ {F}_{\mathrm{Score}}=\frac{\sum_{i=1}^m\frac{2{\mathrm{Tp}}_i}{2{\mathrm{Tp}}_i+{\mathrm{Fp}}_i+{\mathrm{Fn}}_i}}{m}\times 100\% \end{equation*}\end{document}</div> </div> </div> <p /> <p>(Average F1 score: worst value = 0; best value = 100)</p> <p>where <i>m</i> is the number of classes.</p> </div> </div> <div class="resultsanddiscussions" title="sec"> <div class="title" tagx="title" title="title">
Results and discussions</div> <p>In the COVID-19/SARS-CoV classification process, the cascade-forward NN and the KNN algorithm succeeded to recognize all the testing genes successfully (Tp = 30, Fp = 0, Tn = 30 and Fn = 0). But in the COVID-19/MERS-CoV classification process, the KNN algorithm could recognize all the testing genes successfully (Tp = 30, Fp = 0, Tn = 30 and Fn = 0), while the cascade-forward NN failed to identify one testing gene of MERS-CoV epidemic (Tp = 30, Fp = 0, Tn = 29 and Fn = 1). The COVID-19/SARS-CoV/MERS-CoV classification process showed that the KNN algorithm succeeded to recognize all the testing genes successfully (T1 = 30, F1 = 0, T2 = 30, F2 = 0, T3 = 30 and F3 = 0) while the cascade-forward NN has one false negative in MERS-CoV gene identification (T1 = 30, F1 = 0, T2 = 30, F2 = 0, T3 = 29 and F3 = 1); see <a href="#TB1">Tables 1</a> and <a href="#TB2">2</a>. From these results, it is obvious that the KNN classifier can differentiate the three different types of epidemic in all the classification processes with least error rates.</p> <p><a href="#TB3">Table 3</a> is created using the above values. Note that, the results are perfect in the two cases (two-class and multi-class classifications) for the KNN classifier. That is clear from the calculated parameter values shown in the table. In the three classification systems, the KNN classifier gives 100% accuracy, 0 error rate, 1 MCC and 100% F1 score. But for the cascade-forward NN, it gets these typical results only in the COVID-19/SARS-CoV classification process. In COVID-19/MERS-CoV classification process, it gives 98.33% accuracy, 0.0166 error rates, 0.9672 MCC and 98.36% F1 score besides 98.89% accuracy, 0.0111 error rate, 0.9754 MCC and 98.34% F1 score in the overall classification system.</p> <p>In the comparison case, getting higher ACC, lower error rate, higher F1 score and higher MCC is an evidence that the classification process is more successful and the used classifier is more efficient. These resulting parameters indicate that the classifier can recognize the required target with minimum errors. From the research results, the KNN classifier can achieve these conditions and reach the research purpose to differentiate between the COVID-19, SARS-CoV and MERS-CoV epidemics using the genomic signal processing methods. The proposed system provides a new scope for coronaviruses researches as results illustrate the success of using the GSP methods for the epidemics’ recognition and diagnosis. <a href="#TB4">Table 4</a> provides a simple comparison of results obtained in the proposed work to other existing studies in terms of the used database, extracted features, used classifier and results.</p> <p>From the previous comparison, the best accuracy obtained from the related studies is 99.68% [<a href="#ref22">22</a>], and this research has reached 100% accuracy.</p> </div> <div class="conclusions" title="sec"> <div class="title" tagx="title" title="title">
Conclusions</div> <p>After spreading of COVID-19 epidemic, many researchers all over the world have started medically to study, analyze and plan, hoping to reach medicine for this terrifying disease quickly. As a result, some researchers began to compare the COVID-19 with the other coronaviruses in terms of the origin, symptoms, envelope spike S protein characteristics, infection, CT images shape, etc. Besides, there are many tests like RT-PCR and RRT-PCR that can diagnose the COVID-19 cases but with some drawbacks such as the lack of test kits, need for suction tools, high cost, long period of time to get results and patient pain. Considering the coronaviruses are genetic diseases, using genomic signal processing techniques and choosing suitable classifiers are the basic outlines for the proposed work. This work introduces an automated system for detecting COVID-19 and how to distinguish it from the other coronaviruses like SARS-CoV and MERS-CoV. This system can assist in the rapid diagnosis of COVID-19 with highly acceptable accuracy results as it does not depend on the previously mentioned classical diagnostic methods’ disadvantages. It is executed simply by separating DNA sequences using the DNA centrifuge [<a href="#ref57">57</a>] and continuing with program steps that take just a few minutes. All the results are acceptable and satisfying, but the KNN results are perfect for the created classification system getting the best and most efficient diagnosis method. In the future work, either other features will be used or different classifiers will be selected to achieve new objectives in COVID-19 researches.</p> <div class="boxed-text" tagx="boxed-text" title="boxed-text"> <div class="keypoints" title="sec"> <div class="title" tagx="title" title="title">
Key Points</div> <ul> <li> <p>Coronavirus Disease 2019 (COVID-19) is a sudden viral contagion that appeared at the end of last year in Wuhan city, the Chinese province of Hubei, China.</p> </li> <li> <p>The selected database contains 76 genes for each epidemic.</p> </li> <li> <p>Some features are extracted like a discrete Fourier transform (DFT), discrete cosine transform (DCT) and the seven moment invariants to two different classifiers.</p> </li> <li> <p>The <i>k</i>-nearest neighbor (KNN) algorithm and the trainable cascade-forward back propagation neural network were used, and they gave satisfying results to compare.</p> </li> </ul> </div> </div> </div> </div> <div class="back" title="back"> <p><b>Safaa Moneer</b> is currently working as a teaching assistant in the Biomedical Engineering Department, Faculty of Engineering, Helwan University, Egypt. Her main research interests are signal processing, bioinformatics and programing.</p> <p><b>Mai S. Mabrouk</b> is currently a professor of and department head of biomedical engineering at the Misr University for Science and Technology. She published over 70 peer-reviewed articles in the area of biomedical engineering.</p> <p><b>Samir Y. Marzouk</b> is working as a professor of basic sciences at AASTMT, Egypt. He had authored many research papers in reputed international and national journals.</p> <p><b>Mohamed Eldosoky</b> has completed his PhD in biomedical engineering from Helwan University, Egypt, in 2005. He has more than 80 papers in international journals and conferences.</p> <div class="references">
References</div> <div tag="ref-list"> <ul> <div class="title" tagx="title" title="title">
Reference</div> <li tag="ref"><a name="ref1" /><span class="label" tagx="label" title="label">1.</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Tai</span><span class="given-names" tagx="given-names" title="given-names">W</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">He</span><span class="given-names" tagx="given-names" title="given-names">L</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zhang</span><span class="given-names" tagx="given-names" title="given-names">X</span></span>, <span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Characterization of the receptor-binding domain (RBD) of 2019 novel coronavirus: implication for development of RBD protein as a viral attachment inhibitor and vaccine</span>. <span class="source" tagx="source" title="source">Cell Mol Immunol</span><span class="year" tagx="year" title="year">2020</span><a href="https://www.nature.com/articles/s41423-020-0400-4#citeas">https://www.nature.com/articles/s41423-020-0400-4#citeas</a>.</span></li> <li tag="ref"><a name="ref2" /><span class="label" tagx="label" title="label">2.</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Cascella</span><span class="given-names" tagx="given-names" title="given-names">M</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Rajnik</span><span class="given-names" tagx="given-names" title="given-names">M</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Cuomo</span><span class="given-names" tagx="given-names" title="given-names">A</span></span>, <span class="etal" title="etal"><i>et al.</i></span></span><span class="chapter-title" tagx="chapter-title" title="chapter-title">Features, evaluation and treatment coronavirus (COVID-19)</span> In: <span class="source" tagx="source" title="source">StatPearls [Internet]</span>. <span class="publisher-loc" tagx="publisher-loc" title="publisher-loc">Treasure Island (FL)</span>: <span class="publisher-name" tagx="publisher-name" title="publisher-name">StatPearls Publishing</span>, <a href="https://www.ncbi.nlm.nih.gov/books/NBK554776/">https://www.ncbi.nlm.nih.gov/books/NBK554776/</a><span class="comment">(updated Mar 20, 2020)</span>, <span class="year" tagx="year" title="year">2020</span>.</span></li> <li tag="ref"><a name="ref3" /><span class="label" tagx="label" title="label">3.</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zhou</span><span class="given-names" tagx="given-names" title="given-names">P</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Yang</span><span class="given-names" tagx="given-names" title="given-names">X</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wang</span><span class="given-names" tagx="given-names" title="given-names">X</span></span>, <span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">A pneumonia outbreak associated with a new coronavirus of probable bat origin</span>. <span class="source" tagx="source" title="source">Nature</span><span class="year" tagx="year" title="year">2020</span>;<span class="volume" tagx="volume" title="volume">579</span>:<span class="fpage" tagx="fpage" title="fpage">270</span>–<span class="lpage" tagx="lpage" title="lpage">3</span>. doi: <span class="pub-id"><a href="https://dx.doi.org/10.1038/s41586-020-2012-7">10.1038/s41586-020-2012-7</a></span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/32015507">32015507</a></span></span></li> <li tag="ref"><a name="ref4" /><span class="label" tagx="label" title="label">4.</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Pradhan</span><span class="given-names" tagx="given-names" title="given-names">P</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kumar Pandey</span><span class="given-names" tagx="given-names" title="given-names">A</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Mishra</span><span class="given-names" tagx="given-names" title="given-names">A</span></span>, <span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Uncanny similarity of unique inserts in the 2019-nCoV spike protein to HIV-1 gp120 and gag</span>. <span class="source" tagx="source" title="source">bioRxiv</span><span class="year" tagx="year" title="year">2020</span> preprint. doi: <span class="pub-id"><a href="https://dx.doi.org/10.1101/2020.01.30.927871">10.1101/2020.01.30.927871</a></span>.</span></li> <li tag="ref"><a name="ref5" /><span class="label" tagx="label" title="label">5.</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Vivanco-Lira</span><span class="given-names" tagx="given-names" title="given-names">A</span></span></span><span class="mixed-article-title" title="mixed-article-title">Predicting COVID-19 distribution in Mexico through a discrete and time-dependent Markov chain and an SIR-like model</span>. <span class="source" tagx="source" title="source">arXiv</span> 2020;2003:06758 <span class="comment">preprint</span>.</span></li> <li tag="ref"><a name="ref6" /><span class="label" tagx="label" title="label">6.</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Guo</span><span class="given-names" tagx="given-names" title="given-names">Y</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Cao</span><span class="given-names" tagx="given-names" title="given-names">Q</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hong</span><span class="given-names" tagx="given-names" title="given-names">Z</span></span>, <span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">The origin, transmission and clinical therapies on coronavirus disease 2019 (COVID-19) outbreak—an update on the status</span>. <span class="source" tagx="source" title="source">Military Med Res</span><span class="year" tagx="year" title="year">2020</span>;<span class="volume" tagx="volume" title="volume">7</span>(<span class="issue" tagx="issue" title="issue">11</span>):<span class="fpage" tagx="fpage" title="fpage">1</span>–<span class="lpage" tagx="lpage" title="lpage">10</span>. doi: <span class="pub-id"><a href="https://dx.doi.org/10.1186/s40779-020-00240-0">10.1186/s40779-020-00240-0</a></span>.</span></li> <li tag="ref"><a name="ref7" /><span class="label" tagx="label" title="label">7.</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ashour</span><span class="given-names" tagx="given-names" title="given-names">HM</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Elkhatib</span><span class="given-names" tagx="given-names" title="given-names">WF</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Rahman</span><span class="given-names" tagx="given-names" title="given-names">R</span></span>, <span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Insights into the recent 2019 novel coronavirus (SARS-CoV-2) in light of past human coronavirus outbreaks</span>. <span class="source" tagx="source" title="source">Pathogens</span><span class="year" tagx="year" title="year">2020</span>;<span class="volume" tagx="volume" title="volume">9</span>(<span class="issue" tagx="issue" title="issue">3</span>):<span class="fpage" tagx="fpage" title="fpage">1</span>–<span class="lpage" tagx="lpage" title="lpage">15</span>.</span></li> <li tag="ref"><a name="ref8" /><span class="label" tagx="label" title="label">8.</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Andersen</span><span class="given-names" tagx="given-names" title="given-names">K</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Rambaut</span><span class="given-names" tagx="given-names" title="given-names">A</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lipkin</span><span class="given-names" tagx="given-names" title="given-names">W</span></span>, <span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">The proximal origin of SARS-CoV-2</span>. <span class="source" tagx="source" title="source">Nat Med</span><span class="year" tagx="year" title="year">2020</span>;<span class="volume" tagx="volume" title="volume">26</span>:<span class="fpage" tagx="fpage" title="fpage">450</span>–<span class="lpage" tagx="lpage" title="lpage">2</span>. doi: <span class="pub-id"><a href="https://dx.doi.org/10.1038/s41591-020-0820-9">10.1038/s41591-020-0820-9</a></span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/32284615">32284615</a></span></span></li> <li tag="ref"><a name="ref9" /><span class="label" tagx="label" title="label">9.</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wu</span><span class="given-names" tagx="given-names" title="given-names">F</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zhao</span><span class="given-names" tagx="given-names" title="given-names">S</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Yu</span><span class="given-names" tagx="given-names" title="given-names">B</span></span>, <span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">A new coronavirus associated with human respiratory disease in China</span>. <span class="source" tagx="source" title="source">Nature</span><span class="year" tagx="year" title="year">2020</span>;<span class="volume" tagx="volume" title="volume">579</span>:<span class="fpage" tagx="fpage" title="fpage">265</span>–<span class="lpage" tagx="lpage" title="lpage">9</span>. doi: <span class="pub-id"><a href="https://dx.doi.org/10.1038/s41586-020-2008-3">10.1038/s41586-020-2008-3</a></span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/32015508">32015508</a></span></span></li> <li tag="ref"><a name="ref10" /><span class="label" tagx="label" title="label">10.</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Pan</span><span class="given-names" tagx="given-names" title="given-names">L</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Mu</span><span class="given-names" tagx="given-names" title="given-names">M</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Yang</span><span class="given-names" tagx="given-names" title="given-names">P</span></span>, <span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Clinical characteristics of COVID-19 patients with digestive symptoms in Hubei, China: a descriptive, cross-sectional, multicenter study</span>. <span class="source" tagx="source" title="source">Am J Gastroenterol</span><span class="year" tagx="year" title="year">2020</span>;<span class="volume" tagx="volume" title="volume">115</span>(<span class="issue" tagx="issue" title="issue">5</span>):<span class="fpage" tagx="fpage" title="fpage">766</span>–<span class="lpage" tagx="lpage" title="lpage">73</span>; <span class="comment">(pre-print online)</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/32287140">32287140</a></span></span></li> <li tag="ref"><a name="ref11" /><span class="label" tagx="label" title="label">11.</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="collab'">collab: WorldOmeter</span></span><span class="comment">last visited</span>, <span class="year" tagx="year" title="year">2020</span>, <span class="comment">08:20 GMT</span>, <a href="https://www.worldometers.info/coronavirus/">https://www.worldometers.info/coronavirus/</a>.</span></li> <li tag="ref"><a name="ref12" /><span class="label" tagx="label" title="label">12.</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Cao</span><span class="given-names" tagx="given-names" title="given-names">Y</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Li</span><span class="given-names" tagx="given-names" title="given-names">L</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Feng</span><span class="given-names" tagx="given-names" title="given-names">Z</span></span>, <span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Comparative genetic analysis of the novel coronavirus (2019-nCoV/SARS-CoV-2) receptor ACE2 in different populations</span>. <span class="source" tagx="source" title="source">Cell Discovery</span><span class="year" tagx="year" title="year">2020</span>;<span class="volume" tagx="volume" title="volume">6</span>(<span class="issue" tagx="issue" title="issue">11</span>):<span class="fpage" tagx="fpage" title="fpage">1</span>–<span class="lpage" tagx="lpage" title="lpage">4</span>. doi: <span class="pub-id"><a href="https://dx.doi.org/10.1038/s41421-020-0147-1">10.1038/s41421-020-0147-1</a></span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/31934347">31934347</a></span></span></li> <li tag="ref"><a name="ref13" /><span class="label" tagx="label" title="label">13.</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Drexler</span><span class="given-names" tagx="given-names" title="given-names">JF</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Corman</span><span class="given-names" tagx="given-names" title="given-names">VM</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Drosten</span><span class="given-names" tagx="given-names" title="given-names">C</span></span></span><span class="mixed-article-title" title="mixed-article-title">Ecology, evolution and classification of bat coronaviruses in the aftermath of SARS</span>. <span class="source" tagx="source" title="source">Antivir Res</span><span class="year" tagx="year" title="year">2014</span>;<span class="volume" tagx="volume" title="volume">101</span>:<span class="fpage" tagx="fpage" title="fpage">45</span>–<span class="lpage" tagx="lpage" title="lpage">56</span>. doi: <span class="pub-id"><a href="https://dx.doi.org/10.1016/j.antiviral.2013.10.013">10.1016/j.antiviral.2013.10.013</a></span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24184128">24184128</a></span></span></li> <li tag="ref"><a name="ref14" /><span class="label" tagx="label" title="label">14.</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lu</span><span class="given-names" tagx="given-names" title="given-names">G</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wang</span><span class="given-names" tagx="given-names" title="given-names">Q</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gao</span><span class="given-names" tagx="given-names" title="given-names">GF</span></span></span><span class="mixed-article-title" title="mixed-article-title">Bat-to-human: spike features determining ‘host jump’ of coronaviruses SARS-CoV, MERS-CoV, and beyond</span>. <span class="source" tagx="source" title="source">Trends Microbiol</span><span class="year" tagx="year" title="year">2015</span>;<span class="volume" tagx="volume" title="volume">23</span>(<span class="issue" tagx="issue" title="issue">8</span>):<span class="fpage" tagx="fpage" title="fpage">468</span>–<span class="lpage" tagx="lpage" title="lpage">78</span>. doi: <span class="pub-id"><a href="https://dx.doi.org/10.1016/j.tim.2015.06.003">10.1016/j.tim.2015.06.003</a></span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/26206723">26206723</a></span></span></li> <li tag="ref"><a name="ref15" /><span class="label" tagx="label" title="label">15.</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lee</span><span class="given-names" tagx="given-names" title="given-names">H</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lei</span><span class="given-names" tagx="given-names" title="given-names">H</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Santarsiero</span><span class="given-names" tagx="given-names" title="given-names">BD</span></span>, <span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Inhibitor recognition specificity of MERS-CoV papain-like protease may differ from that of SARS-CoV</span>. <span class="source" tagx="source" title="source">ACS Chem Biol</span><span class="year" tagx="year" title="year">2015</span>;<span class="volume" tagx="volume" title="volume">10</span>(<span class="issue" tagx="issue" title="issue">6</span>):<span class="fpage" tagx="fpage" title="fpage">1456</span>–<span class="lpage" tagx="lpage" title="lpage">65</span>. doi: <span class="pub-id"><a href="https://dx.doi.org/10.1021/cb500917m">10.1021/cb500917m</a></span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25746232">25746232</a></span></span></li> <li tag="ref"><a name="ref16" /><span class="label" tagx="label" title="label">16.</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Yuan</span><span class="given-names" tagx="given-names" title="given-names">Y</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Cao</span><span class="given-names" tagx="given-names" title="given-names">D</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zhang</span><span class="given-names" tagx="given-names" title="given-names">Y</span></span>, <span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Cryo-EM structures of MERS-CoV and SARS-CoV spike glycoproteins reveal the dynamic receptor binding domains</span>. <span class="source" tagx="source" title="source">Nat Commun</span><span class="year" tagx="year" title="year">2017</span>;<span class="volume" tagx="volume" title="volume">8</span>(<span class="issue" tagx="issue" title="issue">15092</span>):<span class="fpage" tagx="fpage" title="fpage">1</span>–<span class="lpage" tagx="lpage" title="lpage">9</span>. doi: <span class="pub-id"><a href="https://dx.doi.org/10.1038/ncomms15092">10.1038/ncomms15092</a></span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/28232747">28232747</a></span></span></li> <li tag="ref"><a name="ref17" /><span class="label" tagx="label" title="label">17.</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Cai</span><span class="given-names" tagx="given-names" title="given-names">X</span></span></span><span class="mixed-article-title" title="mixed-article-title">An insight of comparison between COVID-19 (2019-nCoV disease) and SARS in pathology and pathogenesis</span>. <span class="comment">OSFPREPRINT</span><span class="year" tagx="year" title="year">2020</span>. doi: <span class="pub-id"><a href="https://dx.doi.org/10.31219/osf.io/hw34x">10.31219/osf.io/hw34x</a></span><span class="comment">preprint</span>.</span></li> <li tag="ref"><a name="ref18" /><span class="label" tagx="label" title="label">18.</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Al-Tawfiq</span><span class="given-names" tagx="given-names" title="given-names">JA</span></span></span><span class="mixed-article-title" title="mixed-article-title">Asymptomatic coronavirus infection: MERS-CoV and SARS-CoV-2 (COVID-19)</span>. <span class="source" tagx="source" title="source">Travel Med Infect Dis</span><span class="year" tagx="year" title="year">2020</span>;<span class="volume" tagx="volume" title="volume">35</span>(<span class="issue" tagx="issue" title="issue">101608</span>):<span class="fpage" tagx="fpage" title="fpage">1</span>–<span class="lpage" tagx="lpage" title="lpage">9</span>. doi: <span class="pub-id"><a href="https://dx.doi.org/10.1016/j.tmaid.2020.101608">10.1016/j.tmaid.2020.101608</a></span>.</span></li> <li tag="ref"><a name="ref19" /><span class="label" tagx="label" title="label">19.</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gorbalenya</span><span class="given-names" tagx="given-names" title="given-names">AE</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Baker</span><span class="given-names" tagx="given-names" title="given-names">SC</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Baric</span><span class="given-names" tagx="given-names" title="given-names">RS</span></span>, <span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">The species severe acute respiratory syndromerelated coronavirus: classifying 2019-nCoV and naming it SARS-CoV-2</span>. <span class="source" tagx="source" title="source">Nat Microbiol</span><span class="year" tagx="year" title="year">2020</span>;<span class="volume" tagx="volume" title="volume">5</span>:<span class="fpage" tagx="fpage" title="fpage">536</span>–<span class="lpage" tagx="lpage" title="lpage">44</span>. doi: <span class="pub-id"><a href="https://dx.doi.org/10.1038/s41564-020-0695-z">10.1038/s41564-020-0695-z</a></span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/32123347">32123347</a></span></span></li> <li tag="ref"><a name="ref20" /><span class="label" tagx="label" title="label">20.</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Liu</span><span class="given-names" tagx="given-names" title="given-names">J</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zheng</span><span class="given-names" tagx="given-names" title="given-names">X</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Tong</span><span class="given-names" tagx="given-names" title="given-names">Q</span></span>, <span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Overlapping and discrete aspects of the pathology and pathogenesis of the emerging human pathogenic coronaviruses SARS-CoV, MERS-CoV, and 2019-nCoV</span>. <span class="source" tagx="source" title="source">Med Virol</span><span class="year" tagx="year" title="year">2020</span>;<span class="volume" tagx="volume" title="volume">92</span>(<span class="issue" tagx="issue" title="issue">5</span>):<span class="fpage" tagx="fpage" title="fpage">536</span>–<span class="lpage" tagx="lpage" title="lpage">44</span>. doi: <span class="pub-id"><a href="https://dx.doi.org/10.1002/jmv.25709">10.1002/jmv.25709</a></span>.</span></li> <li tag="ref"><a name="ref21" /><span class="label" tagx="label" title="label">21.</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ou</span><span class="given-names" tagx="given-names" title="given-names">X</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Liu</span><span class="given-names" tagx="given-names" title="given-names">Y</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lei</span><span class="given-names" tagx="given-names" title="given-names">X</span></span>, <span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Characterization of spike glycoprotein of SARS-CoV-2 on virus entry and its immune cross-reactivity with SARS-CoV</span>. <span class="source" tagx="source" title="source">Nat Commun</span><span class="year" tagx="year" title="year">2020</span>. doi: <span class="pub-id"><a href="https://dx.doi.org/10.1038/s41467-020-15562-9">10.1038/s41467-020-15562-9</a></span>.</span></li> <li tag="ref"><a name="ref22" /><span class="label" tagx="label" title="label">22.</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Barstugan</span><span class="given-names" tagx="given-names" title="given-names">M</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ozkaya</span><span class="given-names" tagx="given-names" title="given-names">U</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ozturk</span><span class="given-names" tagx="given-names" title="given-names">S</span></span></span><span class="mixed-article-title" title="mixed-article-title">Coronavirus (COVID-19) classification using CT images by machine learning methods</span>. <span class="source" tagx="source" title="source">arXiv</span>:2020:2003:09424, <span class="comment">preprint</span>.</span></li> <li tag="ref"><a name="ref23" /><span class="label" tagx="label" title="label">23.</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Basu</span><span class="given-names" tagx="given-names" title="given-names">S</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Mitra</span><span class="given-names" tagx="given-names" title="given-names">S</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Saha</span><span class="given-names" tagx="given-names" title="given-names">N</span></span></span><span class="mixed-article-title" title="mixed-article-title">Deep learning for screening COVID-19 using chest X-ray images</span>. <span class="source" tagx="source" title="source">arXiv</span>:2020:2004:10507. <span class="comment">preprint</span>.</span></li> <li tag="ref"><a name="ref24" /><span class="label" tagx="label" title="label">24.</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ozturk</span><span class="given-names" tagx="given-names" title="given-names">T</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Talo</span><span class="given-names" tagx="given-names" title="given-names">M</span></span>, <span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Automated detection of covid-19 cases using deep neural networks with X-ray images</span>. <span class="source" tagx="source" title="source">Comput Biol Med</span><span class="year" tagx="year" title="year">2020</span>;<span class="volume" tagx="volume" title="volume">121</span>(<span class="issue" tagx="issue" title="issue">103792</span>):<span class="fpage" tagx="fpage" title="fpage">1</span>–<span class="lpage" tagx="lpage" title="lpage">12</span>. doi: <span class="pub-id"><a href="https://dx.doi.org/10.1016/j.compbiomed.2020.103792">10.1016/j.compbiomed.2020.103792</a></span>.</span></li> <li tag="ref"><a name="ref25" /><span class="label" tagx="label" title="label">25.</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Elasnaoui</span><span class="given-names" tagx="given-names" title="given-names">K</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Chawki</span><span class="given-names" tagx="given-names" title="given-names">Y</span></span></span><span class="mixed-article-title" title="mixed-article-title">Using X-ray images and deep learning for automated detection of coronavirus disease</span>. <span class="source" tagx="source" title="source">J Biomol Struct Dyn</span><span class="year" tagx="year" title="year">2020</span>;<span class="fpage" tagx="fpage" title="fpage">1</span>–<span class="lpage" tagx="lpage" title="lpage">12</span>. doi: <span class="pub-id"><a href="https://dx.doi.org/10.1080/07391102.2020.1767212">10.1080/07391102.2020.1767212</a></span>.</span></li> <li tag="ref"><a name="ref26" /><span class="label" tagx="label" title="label">26.</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Udugama</span><span class="given-names" tagx="given-names" title="given-names">B</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kadhiresan</span><span class="given-names" tagx="given-names" title="given-names">P</span></span>, <span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Diagnosing COVID-19: the disease and tools for detection</span>. <span class="source" tagx="source" title="source">ACS Nano</span><span class="year" tagx="year" title="year">2020</span>;<span class="volume" tagx="volume" title="volume">4</span>:<span class="fpage" tagx="fpage" title="fpage">3822</span>–<span class="lpage" tagx="lpage" title="lpage">55</span>. doi: <span class="pub-id"><a href="https://dx.doi.org/10.1021/acsnano.0c02624">10.1021/acsnano.0c02624</a></span>.</span></li> <li tag="ref"><a name="ref27" /><span class="label" tagx="label" title="label">27.</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Tahamtan</span><span class="given-names" tagx="given-names" title="given-names">A</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ardebili</span><span class="given-names" tagx="given-names" title="given-names">A</span></span></span><span class="mixed-article-title" title="mixed-article-title">Real-time RT-PCR in COVID-19 detection: issues affecting the results</span>. <span class="source" tagx="source" title="source">Expert Rev Mol Diagn</span><span class="year" tagx="year" title="year">2020</span>;<span class="volume" tagx="volume" title="volume">20</span>(<span class="issue" tagx="issue" title="issue">5</span>):<span class="fpage" tagx="fpage" title="fpage">453</span>–<span class="lpage" tagx="lpage" title="lpage">4</span>. doi: <span class="pub-id"><a href="https://dx.doi.org/10.1080/14737159.2020.1757437">10.1080/14737159.2020.1757437</a></span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/32297805">32297805</a></span></span></li> <li tag="ref"><a name="ref28" /><span class="label" tagx="label" title="label">28.</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Benson</span><span class="given-names" tagx="given-names" title="given-names">DA</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Karsch-Mizrachi</span><span class="given-names" tagx="given-names" title="given-names">I</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lipman</span><span class="given-names" tagx="given-names" title="given-names">DJ</span></span>, <span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">GenBank</span>. <span class="source" tagx="source" title="source">Nucleic Acids Res</span><span class="year" tagx="year" title="year">2008</span>;<span class="volume" tagx="volume" title="volume">36</span>(<span class="issue" tagx="issue" title="issue">Database issue</span>):<span class="fpage" tagx="fpage" title="fpage">D25</span>–<span class="lpage" tagx="lpage" title="lpage">30</span>. doi: <span class="pub-id"><a href="https://dx.doi.org/10.1093/nar/gkl986">10.1093/nar/gkl986</a></span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18073190">18073190</a></span></span></li> <li tag="ref"><a name="ref29" /><span class="label" tagx="label" title="label">29.</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lewandowski</span><span class="given-names" tagx="given-names" title="given-names">K</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Xu</span><span class="given-names" tagx="given-names" title="given-names">Y</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Pullan</span><span class="given-names" tagx="given-names" title="given-names">ST</span></span>, <span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Metagenomic nanopore sequencing of influenza virus direct from clinical respiratory samples</span>. <span class="source" tagx="source" title="source">J Clin Microbiol</span><span class="year" tagx="year" title="year">2019</span>;<span class="volume" tagx="volume" title="volume">58</span>:<span class="fpage" tagx="fpage" title="fpage">e00963</span>–<span class="lpage" tagx="lpage" title="lpage">19</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/31666364">31666364</a></span></span></li> <li tag="ref"><a name="ref30" /><span class="label" tagx="label" title="label">30.</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kafetzopoulou</span><span class="given-names" tagx="given-names" title="given-names">LE</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Efthymiadis</span><span class="given-names" tagx="given-names" title="given-names">K</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lewandowski</span><span class="given-names" tagx="given-names" title="given-names">K</span></span>, <span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Assessment of metagenomic nanopore and illumina sequencing for recovering whole genome sequences of chikungunya and dengue viruses directly from clinical samples</span>. <span class="source" tagx="source" title="source">Euro Surveill</span><span class="year" tagx="year" title="year">2018</span>;<span class="volume" tagx="volume" title="volume">23</span>:<span class="fpage" tagx="fpage" title="fpage">1800228</span>.</span></li> <li tag="ref"><a name="ref31" /><span class="label" tagx="label" title="label">31.</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Caly</span><span class="given-names" tagx="given-names" title="given-names">L</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Druce</span><span class="given-names" tagx="given-names" title="given-names">J</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Roberts</span><span class="given-names" tagx="given-names" title="given-names">J</span></span>, <span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Isolation and rapid sharing of the 2019 novel coronavirus (SARS-CoV-2) from the first patient diagnosed with COVID −19 in Australia</span>. <span class="source" tagx="source" title="source">Med J Aust</span><span class="year" tagx="year" title="year">2020</span>;<span class="volume" tagx="volume" title="volume">212</span>(<span class="issue" tagx="issue" title="issue">10</span>):<span class="fpage" tagx="fpage" title="fpage">459</span>–<span class="lpage" tagx="lpage" title="lpage">62</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/32237278">32237278</a></span></span></li> <li tag="ref"><a name="ref32" /><span class="label" tagx="label" title="label">32.</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Das</span><span class="given-names" tagx="given-names" title="given-names">J</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Barman</span><span class="given-names" tagx="given-names" title="given-names">S</span></span></span><span class="mixed-article-title" title="mixed-article-title">Bayesian fusion in cancer gene prediction</span>. <span class="source" tagx="source" title="source">Int J Comput Appl</span><span class="year" tagx="year" title="year">2014</span>;<span class="volume" tagx="volume" title="volume">1</span>:<span class="fpage" tagx="fpage" title="fpage">5</span>–<span class="lpage" tagx="lpage" title="lpage">10</span>.</span></li> <li tag="ref"><a name="ref33" /><span class="label" tagx="label" title="label">33.</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Trad</span><span class="given-names" tagx="given-names" title="given-names">CH</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Fang</span><span class="given-names" tagx="given-names" title="given-names">Q</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Cosic</span><span class="given-names" tagx="given-names" title="given-names">I</span></span></span><span class="mixed-article-title" title="mixed-article-title">Protein sequence comparison based on the wavelet transform approach</span>. <span class="source" tagx="source" title="source">Protein Eng</span><span class="year" tagx="year" title="year">2002</span>;<span class="volume" tagx="volume" title="volume">15</span>(<span class="issue" tagx="issue" title="issue">3</span>):<span class="fpage" tagx="fpage" title="fpage">193</span>–<span class="lpage" tagx="lpage" title="lpage">203</span>. doi: <span class="pub-id"><a href="https://dx.doi.org/10.1093/protein/15.3.193">10.1093/protein/15.3.193</a></span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11932490">11932490</a></span></span></li> <li tag="ref"><a name="ref34" /><span class="label" tagx="label" title="label">34.</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ghosh</span><span class="given-names" tagx="given-names" title="given-names">A</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Barman</span><span class="given-names" tagx="given-names" title="given-names">S</span></span></span><span class="chapter-title" tagx="chapter-title" title="chapter-title">Prediction of prostate cancer cells based on principal component analysis technique</span> In: <span class="source" tagx="source" title="source">Procedia Technology International Conference Computational Intelligence: Modeling Techniques and Applications (CIMTA)</span>, <span class="year" tagx="year" title="year">2013</span>, doi: <span class="pub-id"><a href="https://dx.doi.org/10.1016/j.protcy.2013.12.334">10.1016/j.protcy.2013.12.334</a></span>.</span></li> <li tag="ref"><a name="ref35" /><span class="label" tagx="label" title="label">35.</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wassfy</span><span class="given-names" tagx="given-names" title="given-names">HM</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Abd Elnaby</span><span class="given-names" tagx="given-names" title="given-names">MM</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Salem</span><span class="given-names" tagx="given-names" title="given-names">ML</span></span>, <span class="etal" title="etal"><i>et al.</i></span></span><span class="chapter-title" tagx="chapter-title" title="chapter-title">Eukaryotic gene prediction using advanced DNA numerical representation schemes</span> In: <span class="source" tagx="source" title="source">Processing of Fifth International Conference Advances in Applied Science and Environmental Engineering (ASEE), Kuala Lumpur, Malaysia</span>, <span class="year" tagx="year" title="year">2016</span>.</span></li> <li tag="ref"><a name="ref36" /><span class="label" tagx="label" title="label">36.</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Nair</span><span class="given-names" tagx="given-names" title="given-names">SA</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Sreenadhan</span><span class="given-names" tagx="given-names" title="given-names">SP</span></span></span><span class="mixed-article-title" title="mixed-article-title">A coding measure scheme employing electron-ion interaction pseudopotential (EIIP)</span>. <span class="source" tagx="source" title="source">Bioinformatics</span><span class="year" tagx="year" title="year">2006</span>;<span class="volume" tagx="volume" title="volume">1</span>(<span class="issue" tagx="issue" title="issue">6</span>):<span class="fpage" tagx="fpage" title="fpage">197</span>–<span class="lpage" tagx="lpage" title="lpage">202</span>.</span></li> <li tag="ref"><a name="ref37" /><span class="label" tagx="label" title="label">37.</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Fessler</span><span class="given-names" tagx="given-names" title="given-names">J</span></span></span><span class="mixed-article-title" title="mixed-article-title">Digital signal processing and analysis</span>. <span class="source" tagx="source" title="source">Lecture Notes</span><span class="year" tagx="year" title="year">2004</span>; <span class="comment">Student Version</span> Available Online: <a href="https://docplayer.net/34200912-Eecs-451-digital-signal-processing-and-analysis-lecture-notes-j-fessler.html">https://docplayer.net/34200912-Eecs-451-digital-signal-processing-and-analysis-lecture-notes-j-fessler.html</a>.</span></li> <li tag="ref"><a name="ref38" /><span class="label" tagx="label" title="label">38.</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ko</span><span class="given-names" tagx="given-names" title="given-names">LT</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Chen</span><span class="given-names" tagx="given-names" title="given-names">JE</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hsin</span><span class="given-names" tagx="given-names" title="given-names">HC</span></span>, <span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">A unified algorithm for subband-based discrete cosine transform</span>. <span class="source" tagx="source" title="source">Math Probl Eng</span><span class="year" tagx="year" title="year">2012</span>. doi: <span class="pub-id"><a href="https://dx.doi.org/10.1155/2012/912194">10.1155/2012/912194</a></span>.</span></li> <li tag="ref"><a name="ref39" /><span class="label" tagx="label" title="label">39.</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Jain</span><span class="given-names" tagx="given-names" title="given-names">AK</span></span></span><span class="chapter-title" tagx="chapter-title" title="chapter-title">Fundamentals of digital image processing, ch. 5</span>, <span class="year" tagx="year" title="year">1989</span>, <span class="fpage" tagx="fpage" title="fpage">150</span>–<span class="lpage" tagx="lpage" title="lpage">4</span>. <span class="publisher-name" tagx="publisher-name" title="publisher-name">Prentice Hall</span>, <span class="publisher-loc" tagx="publisher-loc" title="publisher-loc">Englewood Cliffs, NJ</span>.</span></li> <li tag="ref"><a name="ref40" /><span class="label" tagx="label" title="label">40.</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hu</span><span class="given-names" tagx="given-names" title="given-names">M</span></span></span><span class="mixed-article-title" title="mixed-article-title">Visual pattern recognition by moment invariants</span>. <span class="source" tagx="source" title="source">IRE Trans Inf Theory</span><span class="year" tagx="year" title="year">1962</span>;<span class="volume" tagx="volume" title="volume">8</span>:<span class="fpage" tagx="fpage" title="fpage">179</span>–<span class="lpage" tagx="lpage" title="lpage">87</span>.</span></li> <li tag="ref"><a name="ref41" /><span class="label" tagx="label" title="label">41.</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Mabrouk</span><span class="given-names" tagx="given-names" title="given-names">MS</span></span></span><span class="mixed-article-title" title="mixed-article-title">A nonlinear pattern recognition of pandemic H1N1 using a state space based methods</span>. <span class="source" tagx="source" title="source">Avicenna J Med Biotechnol</span><span class="year" tagx="year" title="year">2011</span>;<span class="volume" tagx="volume" title="volume">3</span>(<span class="issue" tagx="issue" title="issue">1</span>):<span class="fpage" tagx="fpage" title="fpage">25</span>–<span class="lpage" tagx="lpage" title="lpage">9</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23407581">23407581</a></span></span></li> <li tag="ref"><a name="ref42" /><span class="label" tagx="label" title="label">42.</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Mabrouk</span><span class="given-names" tagx="given-names" title="given-names">MS</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Marzouk</span><span class="given-names" tagx="given-names" title="given-names">SY</span></span></span><span class="chapter-title" tagx="chapter-title" title="chapter-title">A chaotic study on pandemic and classical (HINI) using EIIP sequence indicators</span> In: <span class="source" tagx="source" title="source">2nd International Conference on Computer Technology and Development (ICCTD 2010)</span>, <span class="year" tagx="year" title="year">2010</span>.</span></li> <li tag="ref"><a name="ref43" /><span class="label" tagx="label" title="label">43.</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Huang</span><span class="given-names" tagx="given-names" title="given-names">Z</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Leng</span><span class="given-names" tagx="given-names" title="given-names">J</span></span></span><span class="chapter-title" tagx="chapter-title" title="chapter-title">Analysis of Hu’s moment invariants on image scaling and rotation</span> In: <span class="source" tagx="source" title="source">Proceedings of 2010 2nd International Conference on Computer Engineering and Technology (ICCET)</span>, <span class="publisher-loc" tagx="publisher-loc" title="publisher-loc">Chengdu, China</span><span class="publisher-name" tagx="publisher-name" title="publisher-name">IEEE</span>, <span class="year" tagx="year" title="year">2010</span>, DOI: <span class="pub-id"><a href="https://dx.doi.org/10.1109/ICCET.2010.5485542">10.1109/ICCET.2010.5485542</a></span>.</span></li> <li tag="ref"><a name="ref44" /><span class="label" tagx="label" title="label">44.</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Flusser</span><span class="given-names" tagx="given-names" title="given-names">J</span></span></span><span class="mixed-article-title" title="mixed-article-title">Moment invariants in image analysis</span>. <span class="source" tagx="source" title="source">Proc World Acad Sci Eng Technol</span><span class="year" tagx="year" title="year">2006</span>;<span class="volume" tagx="volume" title="volume">1</span>(<span class="issue" tagx="issue" title="issue">11</span>):<span class="fpage" tagx="fpage" title="fpage">3721</span>–<span class="lpage" tagx="lpage" title="lpage">6</span>.</span></li> <li tag="ref"><a name="ref45" /><span class="label" tagx="label" title="label">45.</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Mamistvalov</span><span class="given-names" tagx="given-names" title="given-names">AG</span></span></span><span class="mixed-article-title" title="mixed-article-title">N-dimensional moment invariants and conceptual mathematical theory of recognition n-dimensional solids</span>. <span class="source" tagx="source" title="source">IEEE Trans Pattern Anal Mach Intell</span><span class="year" tagx="year" title="year">1998</span>;<span class="volume" tagx="volume" title="volume">20</span>(<span class="issue" tagx="issue" title="issue">8</span>):<span class="fpage" tagx="fpage" title="fpage">819</span>–<span class="lpage" tagx="lpage" title="lpage">31</span>.</span></li> <li tag="ref"><a name="ref46" /><span class="label" tagx="label" title="label">46.</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Weisstein</span><span class="given-names" tagx="given-names" title="given-names">EW</span></span></span><span class="source" tagx="source" title="source">Riemann Integral</span><span class="year" tagx="year" title="year">2000</span><a href="http://mathworld.wolfram.com/RiemannIntegral.html">http://mathworld.wolfram.com/RiemannIntegral.html</a><span class="comment">(last visited 10 April 2020)</span>.</span></li> <li tag="ref"><a name="ref47" /><span class="label" tagx="label" title="label">47.</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Al Bataineh</span><span class="given-names" tagx="given-names" title="given-names">A</span></span></span><span class="mixed-article-title" title="mixed-article-title">A comparative analysis of nonlinear machine learning algorithms for breast cancer detection</span>. <span class="source" tagx="source" title="source">Int J Mach Learn Comput</span><span class="year" tagx="year" title="year">2019</span>;<span class="volume" tagx="volume" title="volume">9</span>(<span class="issue" tagx="issue" title="issue">3</span>):<span class="fpage" tagx="fpage" title="fpage">248</span>–<span class="lpage" tagx="lpage" title="lpage">54</span>.</span></li> <li tag="ref"><a name="ref48" /><span class="label" tagx="label" title="label">48.</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Fogliatto</span><span class="given-names" tagx="given-names" title="given-names">FS</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Anzanello</span><span class="given-names" tagx="given-names" title="given-names">MJ</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Soares</span><span class="given-names" tagx="given-names" title="given-names">F</span></span>, <span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Decision support for breast cancer detection: classification improvement through feature selection</span>. <span class="source" tagx="source" title="source">Cancer Control</span><span class="year" tagx="year" title="year">2019</span>;<span class="volume" tagx="volume" title="volume">26</span>(<span class="issue" tagx="issue" title="issue">1</span>):<span class="fpage" tagx="fpage" title="fpage">1</span>–<span class="lpage" tagx="lpage" title="lpage">8</span>. doi: <span class="pub-id"><a href="https://dx.doi.org/10.1177/1073274819876598">10.1177/1073274819876598</a></span>.</span></li> <li tag="ref"><a name="ref49" /><span class="label" tagx="label" title="label">49.</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Medjahed</span><span class="given-names" tagx="given-names" title="given-names">SA</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Saadi</span><span class="given-names" tagx="given-names" title="given-names">TA</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Benyettou</span><span class="given-names" tagx="given-names" title="given-names">A</span></span></span><span class="mixed-article-title" title="mixed-article-title">Breast cancer diagnosis by using <i>k</i>-nearest neighbor with different distances and classification rules</span>. <span class="source" tagx="source" title="source">Int J Comput Appl</span><span class="year" tagx="year" title="year">2013</span>;<span class="volume" tagx="volume" title="volume">62</span>(<span class="issue" tagx="issue" title="issue">1</span>):<span class="fpage" tagx="fpage" title="fpage">1</span>–<span class="lpage" tagx="lpage" title="lpage">5</span>.</span></li> <li tag="ref"><a name="ref50" /><span class="label" tagx="label" title="label">50.</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Negnevitsky</span><span class="given-names" tagx="given-names" title="given-names">M</span></span></span><span class="source" tagx="source" title="source">Artificial Intelligence: A Guide to Intelligent Systems</span>. <span class="publisher-name" tagx="publisher-name" title="publisher-name">Pearson</span>, <span class="publisher-loc" tagx="publisher-loc" title="publisher-loc">Edinburgh, England</span><span class="comment">ch. 6</span>, <span class="year" tagx="year" title="year">2005</span>;<span class="volume" tagx="volume" title="volume">3</span>:<span class="fpage" tagx="fpage" title="fpage">175</span>–<span class="lpage" tagx="lpage" title="lpage">9</span>.</span></li> <li tag="ref"><a name="ref51" /><span class="label" tagx="label" title="label">51.</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Goyal</span><span class="given-names" tagx="given-names" title="given-names">S</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Goyal</span><span class="given-names" tagx="given-names" title="given-names">GK</span></span></span><span class="mixed-article-title" title="mixed-article-title">Cascade and feed-forward backpropagation artificial neural network models for prediction of sensory quality of instant coffee flavoured sterilized drink</span>. <span class="source" tagx="source" title="source">Can J Artif Intell Mach Learn Pattern Recogn</span><span class="year" tagx="year" title="year">2011</span>;<span class="volume" tagx="volume" title="volume">2</span>(<span class="issue" tagx="issue" title="issue">6</span>):<span class="fpage" tagx="fpage" title="fpage">78</span>–<span class="lpage" tagx="lpage" title="lpage">82</span>.</span></li> <li tag="ref"><a name="ref52" /><span class="label" tagx="label" title="label">52.</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Demuth</span><span class="given-names" tagx="given-names" title="given-names">H</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Beale</span><span class="given-names" tagx="given-names" title="given-names">M</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hagan</span><span class="given-names" tagx="given-names" title="given-names">M</span></span></span><span class="source" tagx="source" title="source">Neural Network Toolbox User’s Guide</span>. <span class="publisher-loc" tagx="publisher-loc" title="publisher-loc">Natrick, USA</span>: <span class="publisher-name" tagx="publisher-name" title="publisher-name">The MathWorks. Inc.</span>, <span class="year" tagx="year" title="year">2009</span>.</span></li> <li tag="ref"><a name="ref53" /><span class="label" tagx="label" title="label">53.</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Chicco</span><span class="given-names" tagx="given-names" title="given-names">D</span></span></span><span class="mixed-article-title" title="mixed-article-title">Ten quick tips for machine learning in computational biology</span>. <span class="source" tagx="source" title="source">BioData Mining</span><span class="year" tagx="year" title="year">2017</span>;<span class="volume" tagx="volume" title="volume">10</span>(<span class="issue" tagx="issue" title="issue">35</span>):<span class="fpage" tagx="fpage" title="fpage">1</span>–<span class="lpage" tagx="lpage" title="lpage">17</span>. doi: <span class="pub-id"><a href="https://dx.doi.org/10.1186/s13040-017-0155-3">10.1186/s13040-017-0155-3</a></span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/28127402">28127402</a></span></span></li> <li tag="ref"><a name="ref54" /><span class="label" tagx="label" title="label">54.</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Chicco</span><span class="given-names" tagx="given-names" title="given-names">D</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Jurman</span><span class="given-names" tagx="given-names" title="given-names">G</span></span></span><span class="mixed-article-title" title="mixed-article-title">The advantages of the Matthews correlation coefficient (MCC) over F1 score and accuracy in binary classification evaluation</span>. <span class="source" tagx="source" title="source">BMC Genomics</span><span class="year" tagx="year" title="year">2020</span>;<span class="volume" tagx="volume" title="volume">21</span>(<span class="issue" tagx="issue" title="issue">1</span>):<span class="fpage" tagx="fpage" title="fpage">6</span>. doi: <span class="pub-id"><a href="https://dx.doi.org/10.1186/s12864-019-6413-7">10.1186/s12864-019-6413-7</a></span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/31898477">31898477</a></span></span></li> <li tag="ref"><a name="ref55" /><span class="label" tagx="label" title="label">55.</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Sokolova</span><span class="given-names" tagx="given-names" title="given-names">M</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Japkowicz</span><span class="given-names" tagx="given-names" title="given-names">N</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Szpakowicz</span><span class="given-names" tagx="given-names" title="given-names">S</span></span></span><span class="mixed-article-title" title="mixed-article-title">Beyond accuracy, F-score and ROC: a family of discriminant measures for performance evaluation</span>. <span class="source" tagx="source" title="source">Adv Artif Intell</span><span class="year" tagx="year" title="year">2006</span>;<span class="volume" tagx="volume" title="volume">4304</span>:<span class="fpage" tagx="fpage" title="fpage">1015</span>–<span class="lpage" tagx="lpage" title="lpage">1021</span>. doi: <span class="pub-id"><a href="https://dx.doi.org/10.1007/11941439_114">10.1007/11941439_114</a></span>.</span></li> <li tag="ref"><a name="ref56" /><span class="label" tagx="label" title="label">56.</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Sokolova</span><span class="given-names" tagx="given-names" title="given-names">M</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lapalme</span><span class="given-names" tagx="given-names" title="given-names">G</span></span></span><span class="mixed-article-title" title="mixed-article-title">A systematic analysis of performance measures for classification tasks</span>. <span class="source" tagx="source" title="source">Inf Process Manag</span><span class="year" tagx="year" title="year">2009</span>;<span class="volume" tagx="volume" title="volume">45</span>(<span class="issue" tagx="issue" title="issue">4</span>):<span class="fpage" tagx="fpage" title="fpage">427</span>–<span class="lpage" tagx="lpage" title="lpage">37</span>. doi: <span class="pub-id"><a href="https://dx.doi.org/10.1016/j.ipm.2009.03.002">10.1016/j.ipm.2009.03.002</a></span>.</span></li> <li tag="ref"><a name="ref57" /><span class="label" tagx="label" title="label">57.</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Mabrouk</span><span class="given-names" tagx="given-names" title="given-names">MS</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ezz</span><span class="given-names" tagx="given-names" title="given-names">MA</span></span></span><span class="mixed-article-title" title="mixed-article-title">HSLC_FUGE: high speed and low COST LABORATORY centrifuge for genomic DNA purification</span>. <span class="source" tagx="source" title="source">J Mech Med Biol</span><span class="year" tagx="year" title="year">2012</span>:<span class="volume" tagx="volume" title="volume">12</span>(05). doi: <span class="pub-id"><a href="https://dx.doi.org/10.1142/S021951941240026X">10.1142/S021951941240026X</a></span>.</span></li> </ul> </div> </div>  </body></html>