<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<link rel="dns-prefetch" href="//d33xdlntwy0kbs.cloudfront.net" />
<link rel="shortcut icon" href="https://www.biorxiv.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<meta name="type" content="article" />
<meta name="category" content="article" />
<meta name="HW.identifier" content="/biorxiv/early/2017/10/09/200352.atom" />
<meta name="HW.pisa" content="biorxiv;200352v1" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="Landscape genomic prediction for restoration of a Eucalyptus foundation species under climate change" />
<meta name="DC.Identifier" content="10.1101/200352" />
<meta name="DC.Date" content="2017-10-09" />
<meta name="DC.Publisher" content="Cold Spring Harbor Laboratory" />
<meta name="DC.Rights" content="© 2017, Posted by Cold Spring Harbor Laboratory. This pre-print is available under a Creative Commons License (Attribution 4.0 International), CC BY 4.0, as described at http://creativecommons.org/licenses/by/4.0/" />
<meta name="DC.AccessRights" content="restricted" />
<meta name="DC.Description" content="As species face rapid environmental change, we can build resilient populations through restoration projects that incorporate predicted future climates into seed sourcing decisions. Eucalyptus melliodora is a foundation species of a critically endangered community in Australia that is a target for restoration. We examined patterns of genomic and phenotypic variation to make empirical based recommendations for seed sourcing. We examined isolation by distance and isolation by environment, determining gene flow up to 500 km and associations with environmental variables. Climate chamber studies revealed extensive phenotypic variation both within and among sampling sites, but no site-specific differentiation in phenotypic plasticity. Overall our results suggest that seed can be sourced broadly across the landscape, providing ample diversity for adaptation to environmental change. Application of our landscape genomic model to E. melliodora restoration projects can identify genomic variation suitable for predicted future climates, thereby increasing the long term probability of successful restoration." />
<meta name="DC.Contributor" content="Megan A. Supple" />
<meta name="DC.Contributor" content="Jason G. Bragg" />
<meta name="DC.Contributor" content="Linda M. Broadhurst" />
<meta name="DC.Contributor" content="Adrienne B. Nicotra" />
<meta name="DC.Contributor" content="Margaret Byrne" />
<meta name="DC.Contributor" content="Rose L. Andrew" />
<meta name="DC.Contributor" content="Abigail Widdup" />
<meta name="DC.Contributor" content="Nicola C. Aitken" />
<meta name="DC.Contributor" content="Justin O. Borevitz" />
<meta name="article:published_time" content="2017-10-09" />
<meta name="article:section" content="New Results" />
<meta name="citation_title" content="Landscape genomic prediction for restoration of a Eucalyptus foundation species under climate change" />
<meta name="citation_abstract" lang="en" content="&lt;h3&gt;Abstract&lt;/h3&gt;
&lt;p&gt;As species face rapid environmental change, we can build resilient populations through restoration projects that incorporate predicted future climates into seed sourcing decisions. &lt;i&gt;Eucalyptus melliodora&lt;/i&gt; is a foundation species of a critically endangered community in Australia that is a target for restoration. We examined patterns of genomic and phenotypic variation to make empirical based recommendations for seed sourcing. We examined isolation by distance and isolation by environment, determining gene flow up to 500 km and associations with environmental variables. Climate chamber studies revealed extensive phenotypic variation both within and among sampling sites, but no site-specific differentiation in phenotypic plasticity. Overall our results suggest that seed can be sourced broadly across the landscape, providing ample diversity for adaptation to environmental change. Application of our landscape genomic model to &lt;i&gt;E. melliodora&lt;/i&gt; restoration projects can identify genomic variation suitable for predicted future climates, thereby increasing the long term probability of successful restoration.&lt;/p&gt;" />
<meta name="citation_journal_title" content="bioRxiv" />
<meta name="citation_publisher" content="Cold Spring Harbor Laboratory" />
<meta name="citation_publication_date" content="2017/01/01" />
<meta name="citation_mjid" content="biorxiv;200352v1" />
<meta name="citation_id" content="200352v1" />
<meta name="citation_public_url" content="https://www.biorxiv.org/content/10.1101/200352v1" />
<meta name="citation_abstract_html_url" content="https://www.biorxiv.org/content/10.1101/200352v1.abstract" />
<meta name="citation_full_html_url" content="https://www.biorxiv.org/content/10.1101/200352v1.full" />
<meta name="citation_pdf_url" content="https://www.biorxiv.org/content/biorxiv/early/2017/10/09/200352.full.pdf" />
<meta name="citation_doi" content="10.1101/200352" />
<meta name="citation_num_pages" content="46" />
<meta name="citation_article_type" content="Article" />
<meta name="citation_section" content="New Results" />
<meta name="citation_firstpage" content="200352" />
<meta name="citation_author" content="Megan A. Supple" />
<meta name="citation_author_institution" content="Research School of Biology, The Australian National University" />
<meta name="citation_author_institution" content="Department of Ecology and Evolutionary Biology, University of California at Santa Cruz" />
<meta name="citation_author_email" content="megan.a.supple@gmail.com" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0002-0204-7852" />
<meta name="citation_author" content="Jason G. Bragg" />
<meta name="citation_author_institution" content="Research School of Biology, The Australian National University" />
<meta name="citation_author_institution" content="National Herbarium of New South Wales, The Royal Botanic Gardens and Domain Trust" />
<meta name="citation_author" content="Linda M. Broadhurst" />
<meta name="citation_author_institution" content="Centre for Australian National Biodiversity Research, Commonwealth Scientific and Industrial Research Organisation (CSIRO) National Research Collections and Facilities" />
<meta name="citation_author" content="Adrienne B. Nicotra" />
<meta name="citation_author_institution" content="Research School of Biology, The Australian National University" />
<meta name="citation_author" content="Margaret Byrne" />
<meta name="citation_author_institution" content="Science and Conservation Division, Department of Parks and Wildlife Western Australia" />
<meta name="citation_author" content="Rose L. Andrew" />
<meta name="citation_author_institution" content="School of Environmental and Rural Science, University of New England" />
<meta name="citation_author" content="Abigail Widdup" />
<meta name="citation_author_institution" content="Research School of Biology, The Australian National University" />
<meta name="citation_author" content="Nicola C. Aitken" />
<meta name="citation_author_institution" content="Research School of Biology, The Australian National University" />
<meta name="citation_author" content="Justin O. Borevitz" />
<meta name="citation_author_institution" content="Research School of Biology, The Australian National University" />
<meta name="citation_author_institution" content="Centre of Excellence in Plant Energy Biology, The Australian National University" />
<meta name="citation_reference" content="Aitken, S. N., &amp; Whitlock, M. C. (2013). Assisted Gene Flow to Facilitate Local Adaptation to Climate Change. Annual Review of Ecology, Evolution, and Systematics, 44(1), 367–388. https://doi.org/10.1146/annurev-ecolsys-110512-135747" />
<meta name="citation_reference" content="Aitken, S. N., Yeaman, S., Holliday, J. A., Wang, T., &amp; Curtis-McLane, S. (2008). Adaptation, migration or extirpation: climate change outcomes for tree populations. Evolutionary Applications, 1(1), 95–111. https://doi.org/10.1111/j.1752-4571.2007.00013.x" />
<meta name="citation_reference" content="ALA. (2016). Atlas of Living Australia. Retrieved November 24, 2016, from http://www.ala.org.au/" />
<meta name="citation_reference" content="Andrew, R. L., Wallis, I. R., Harwood, C. E., &amp; Foley, W. J. (2010). Genetic and environmental contributions to variation and population divergence in a broad-spectrum foliar defence of Eucalyptus tricarpa. Annals of Botany, 105(5), 707–717. https://doi.org/10.1093/aob/mcq034" />
<meta name="citation_reference" content="Andrews, S. (2012). FastQC. Retrieved from https://www.bioinformatics.babraham.ac.uk/projects/fastqc/" />
<meta name="citation_reference" content="Bartholomé, J., Mandrou, E., Jenkins, J., Nabihoudine, I., Klopp, C., Schmutz, J., Plomion, C., &amp; Gion, J.-M. (2015). High-resolution genetic maps of Eucalyptus improve Eucalyptus grandis genome assembly. New Phytologist, 206, 1283–1296." />
<meta name="citation_reference" content="Bates, D., Mächler, M., Bolker, B., &amp; Walker, S. (2015). Fitting Linear Mixed-Effects Models Using lme4. Journal of Statistical Software, 67(1). https://doi.org/10.18637/jss.v067.i01" />
<meta name="citation_reference" content="Bivand, R., &amp; Rundel, C. (2016). rgeos: Interface to Geometry Engine - Open Source (GEOS)." />
<meta name="citation_reference" content="Brevik, E. (2013). The Potential Impact of Climate Change on Soil Properties and Processes and Corresponding Influence on Food Security. Agriculture, 3(3), 398–417. https://doi.org/10.3390/agriculture3030398" />
<meta name="citation_reference" content="Broadhurst, L. M., Lowe, A., Coates, D. J., Cunningham, S. A., McDonald, M., Vesk, P. A., &amp; Yates, C. (2008). Seed supply for broadscale restoration: Maximizing evolutionary potential. Evolutionary Applications, 1(4), 587–597. https://doi.org/10.1111/j.1752-4571.2008.00045.x" />
<meta name="citation_reference" content="Broadhurst, L. M., Mellick, R., Knerr, N., Li, L., &amp; Supple, M. A. (n.d.). Land availability may be more important than genetic diversity in the range shift response of the widely distributed eucalypt Eucalyptus melliodora (Yellow Box, Myrtaceae)." />
<meta name="citation_reference" content="Byrne, M., Prober, S. M., McLean, E. H., Steane, D. A., Stock, W. D., Potts, B. M., &amp; Vaillancourt, R. E. (2013). Adaptation to climate in widespread eucalypt species. Gold Coast: National Climate Change Adaptation Research Facility." />
<meta name="citation_reference" content="CGIAR-CSI. (2016). SRTM 90m Digital Elevation Data. Retrieved September 5, 2016, from http://srtm.csi.cgiar.org/" />
<meta name="citation_reference" content="CSIRO. (2016). Soil and Landscape Grid of Australia. Retrieved November 29, 2016, from http://www.clw.csiro.au/aclep/soilandlandscapegrid/index.html" />
<meta name="citation_reference" content="Danecek, P., Auton, A., Abecasis, G., Albers, C. A., Banks, E., DePristo, M. A., Handsaker, R. E., Lunter, G., Marth, G. T., Sherry, S. T., McVean, G., &amp; Durbin, R. (2011). The variant call format and VCFtools. Bioinformatics, 27(15), 2156–2158. https://doi.org/10.1093/bioinformatics/btr330" />
<meta name="citation_reference" content="Department of Environment and Climate Change and Water. (2011). National Recovery Plan for White Box - Yellow Box - Blakely’s Red Gum Grassy Woodland and Derived Native Grassland." />
<meta name="citation_reference" content="Department of the Environment and Heritage. (2006). EPBC Act Policy Statements: White Box - Yellow Box - Blakely’s Red Gum grassy woodlands and derived native grasslands." />
<meta name="citation_reference" content="Dray, S., &amp; Dufour, A.-B. (2007). The ade4 Package: Implementing the Duality Diagram for Ecologists. Journal of Statistical Software, 22(4). https://doi.org/10.18637/jss.v022.i04" />
<meta name="citation_reference" content="Elshire, R. J., Glaubitz, J. C., Sun, Q., Poland, J. A., Kawamoto, K., Buckler, E. S., &amp; Mitchell, S. E. (2011). A robust, simple genotyping-by-sequencing (GBS) approach for high diversity species. PloS One, 6(5), e19379. https://doi.org/10.1371/journal.pone.0019379" />
<meta name="citation_reference" content="Ferrier, S., Manion, G., Elith, J., &amp; Richardson, K. (2007). Using generalized dissimilarity modelling to analyse and predict patterns of beta diversity in regional biodiversity assessment. Diversity and Distributions, 13(3), 252–264. https://doi.org/10.1111/j.1472-4642.2007.00341.x" />
<meta name="citation_reference" content="Fitzpatrick, M. C., &amp; Keller, S. R. (2015). Ecological genomics meets community-level modelling of biodiversity: Mapping the genomic landscape of current and future environmental adaptation. Ecology Letters, 18(1), 1–16. https://doi.org/10.1111/ele.12376" />
<meta name="citation_reference" content="González-Orozco, C. E., Pollock, L. J., Thornhill, A. H., Mishler, B. D., Knerr, N., Laffan, S. W., Miller, J. T., Rosauer, D. F., Faith, D. P., Nipperess, D. A., Kujala, H., Linke, S., Butt, N., Külheim, C., Crisp, M. D., &amp; Gruber, B. (2016). Phylogenetic approaches reveal biodiversity threats under climate change. Nature Climate Change, 1(12), 1110–1114. https://doi.org/10.1038/nclimate3126" />
<meta name="citation_reference" content="Goudet, J., &amp; Jombart, T. (2015). hierfstat: Estimation and Tests of Hierarchical F- Statistics." />
<meta name="citation_reference" content="Hoffmann, A., Griffin, P., Dillon, S., Catullo, R., Rane, R., Byrne, M., Jordan, R., Oakeshott, J., Weeks, A., Joseph, L., Lockhart, P., Borevitz, J., &amp; Sgrò, C. (2015). A framework for incorporating evolutionary genomics into biodiversity conservation and management. Climate Change Responses, 2(1). https://doi.org/10.1186/s40665-014-0009-x" />
<meta name="citation_reference" content="JGI. (2015). Eucalyptus grandis v2.0 (Rose gum). Retrieved August 27, 2015, from https://phytozome.jgi.doe.gov/pz/portal.html#!info?alias=Org_Egrandis_er" />
<meta name="citation_reference" content="Jombart, T. (2008). Adegenet: A R package for the multivariate analysis of genetic markers. Bioinformatics, 24(11), 1403–1405. https://doi.org/10.1093/bioinformatics/btn129" />
<meta name="citation_reference" content="Li, H. (2013). Aligning sequence reads, clone sequences and assembly contigs with BWA-MEM. arXiv, 1303.3997v1." />
<meta name="citation_reference" content="Lindenmayer, D. B., Steffen, W., Burbidge, A. A., Hughes, L., Kitching, R. L., Musgrave, W., Stafford Smith, M., &amp; Werner, P. A. (2010). Conservation strategies in response to rapid climate change: Australia as a case study. Biological Conservation, 143(7), 1587–1593. https://doi.org/10.1016/j.biocon.2010.04.014" />
<meta name="citation_reference" content="Manion, G., Lisk, M., Ferrier, S., Nieto-Lugilde, D., &amp; Fitzpatrick, M. C. (2016). gdm: Functions for Generalized Dissimilarity Modeling." />
<meta name="citation_reference" content="McKenna, A., Hanna, M., Banks, E., Sivachenko, A., Cibulskis, K., Kernytsky, A., Garimella, K., Altshuler, D., Gabriel, S., Daly, M., &amp; DePristo, M. A. (2010). The Genome Analysis Toolkit: a MapReduce framework for analyzing next-generation DNA sequencing data. Genome Research, 20(9), 1297–1303. https://doi.org/10.1101/gr.107524.110" />
<meta name="citation_reference" content="McLean, E. H., Prober, S. M., Stock, W. D., Steane, D. A., Potts, B. M., Vaillancourt, R. E., &amp; Byrne, M. (2014). Plasticity of functional traits varies clinally along a rainfall gradient in Eucalyptus tricarpa. Plant, Cell and Environment, 37(6), 1440–1451. https://doi.org/10.1111/pce.12251" />
<meta name="citation_reference" content="Murray, K. D., &amp; Borevitz, J. O. (2017a). Axe: rapid, competitive sequence read demultiplexing using a trie. bioRxiv. https://doi.org/10.1101/160606" />
<meta name="citation_reference" content="Murray, K. D., &amp; Borevitz, J. O. (2017b). libqcpp: A C++14 sequence quality control library. The Journal of Open Source Software, 2(13). https://doi.org/10.21105/joss.00232" />
<meta name="citation_reference" content="Myburg, A. A., Grattapaglia, D., Tuskan, G. A., Hellsten, U., Hayes, R. D., Grimwood, J., Schmutz, J. (2014). The genome of Eucalyptus grandis. Nature, 509(7505), 356–362. https://doi.org/10.1038/nature13308" />
<meta name="citation_reference" content="Nicotra, A. B., Atkin, O. K., Bonser, S. P., Davidson, A. M., Finnegan, E. J., Mathesius, U., Poot, P., Purugganan, M. D., Richards, C. L., Valladares, F., &amp; van Kleunen, M. (2010). Plant phenotypic plasticity in a changing climate. Trends in Plant Science, 15(12), 684–692. https://doi.org/10.1016/j.tplants.2010.09.008" />
<meta name="citation_reference" content="Oksanen, J., Blanchet, F. G., Friendly, M., Kindt, R., Legendre, P., McGlinn, D., Minchin, P. R., O’Hara, R. B., Simpson, G. L., Solymos, P., Stevens, M. H. H., Szoecs, E., &amp; Wagner, H. (2016). vegan: Community Ecology Package." />
<meta name="citation_reference" content="Pateiro-López, B., &amp; Rodríguez-Casal, A. (2010). Generalizing the Convex Hull of a Sample: The R Package Alphahull. Journal of Statistical Software, 34(5). https://doi.org/whttp://dx.doi.org/10.18637/jss.v034.i05" />
<meta name="citation_reference" content="Prober, S. M., Byrne, M., McLean, E. H., Steane, D. A., Potts, B. M., Vaillancourt, R. E., &amp; Stock, W. D. (2015). Climate-adjusted provenancing: a strategy for climate-resilient ecological restoration. Frontiers in Ecology and Evolution, 3(65). https://doi.org/10.3389/fevo.2015.00065" />
<meta name="citation_reference" content="R Core Team. (2015). R: A language and environment for statistical computing. Vienna, Austria: R Foundation for Statistical Computing." />
<meta name="citation_reference" content="Spokas, K., &amp; Forcella, F. (2006). Estimating hourly incoming solar radiation from limited meteorological data. Weed Science, 54(1), 182–189. https://doi.org/10.1614/WS-05-098R.1" />
<meta name="citation_reference" content="Thomassen, H. A., Fuller, T., Buermann, W., Milá, B., Kieswetter, C. M., Jarrín-V., P., … Smith, T. B. (2011). Mapping evolutionary process: A multi-taxa approach to conservation prioritization. Evolutionary Applications, 4(2), 397–413. https://doi.org/10.1111/j.1752-4571.2010.00172.x" />
<meta name="citation_reference" content="Threatened Species Scientific Committee. (2006). Advice to the Minister for the Environment and Heritage." />
<meta name="citation_reference" content="Vavrek, M. J. (2011). Fossil: Palaeoecological and Paelaeogeogrpahical Analysis Tools. Palaeontologia Electronica, 14(1), 1–16." />
<meta name="citation_reference" content="Wang, I. J., &amp; Bradburd, G. S. (2014). Isolation by environment. Molecular Ecology, 23(23), 5649–5662. https://doi.org/10.1111/mec.12938" />
<meta name="citation_reference" content="Weir, B. S., &amp; Cockerham, C. C. (1984). Estimating F-Statistics for the Analysis of Population Structure. Evolution, 38(6), 1358–1370. https://doi.org/10.2307/2409936" />
<meta name="citation_reference" content="Williams, A. V, Nevill, P. G., &amp; Krauss, S. L. (2014). Next generation restoration genetics: Applications and opportunities. Trends in Plant Science, 19(8), 529–537. https://doi.org/10.1016/j.tplants.2014.03.011" />
<meta name="citation_reference" content="WorldClim. (2016a). CMIP5 30-seconds. Retrieved September 30, 2016, from http://www.worldclim.org/cmip5_30s" />
<meta name="citation_reference" content="WorldClim. (2016b). WorldClim 1.4. Retrieved November 28, 2016, from http://www.worldclim.org/current" />
<meta name="twitter:title" content="Landscape genomic prediction for restoration of a Eucalyptus foundation species under climate change" />
<meta name="twitter:site" content="@biorxivpreprint" />
<meta name="twitter:card" content="summary" />
<meta name="twitter:image" content="https://www.biorxiv.org/sites/default/files/images/biorxiv_logo_homepage7-5-small.png" />
<meta name="twitter:description" content="As species face rapid environmental change, we can build resilient populations through restoration projects that incorporate predicted future climates into seed sourcing decisions. Eucalyptus melliodora is a foundation species of a critically endangered community in Australia that is a target for restoration. We examined patterns of genomic and phenotypic variation to make empirical based recommendations for seed sourcing. We examined isolation by distance and isolation by environment, determining gene flow up to 500 km and associations with environmental variables. Climate chamber studies revealed extensive phenotypic variation both within and among sampling sites, but no site-specific differentiation in phenotypic plasticity. Overall our results suggest that seed can be sourced broadly across the landscape, providing ample diversity for adaptation to environmental change. Application of our landscape genomic model to E. melliodora restoration projects can identify genomic variation suitable for predicted future climates, thereby increasing the long term probability of successful restoration." />
<meta name="og-title" property="og:title" content="Landscape genomic prediction for restoration of a Eucalyptus foundation species under climate change" />
<meta name="og-url" property="og:url" content="https://www.biorxiv.org/content/10.1101/200352v1" />
<meta name="og-site-name" property="og:site_name" content="bioRxiv" />
<meta name="og-description" property="og:description" content="As species face rapid environmental change, we can build resilient populations through restoration projects that incorporate predicted future climates into seed sourcing decisions. Eucalyptus melliodora is a foundation species of a critically endangered community in Australia that is a target for restoration. We examined patterns of genomic and phenotypic variation to make empirical based recommendations for seed sourcing. We examined isolation by distance and isolation by environment, determining gene flow up to 500 km and associations with environmental variables. Climate chamber studies revealed extensive phenotypic variation both within and among sampling sites, but no site-specific differentiation in phenotypic plasticity. Overall our results suggest that seed can be sourced broadly across the landscape, providing ample diversity for adaptation to environmental change. Application of our landscape genomic model to E. melliodora restoration projects can identify genomic variation suitable for predicted future climates, thereby increasing the long term probability of successful restoration." />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.biorxiv.org/sites/default/files/images/biorxiv_logo_homepage7-5-small.png" />
<meta name="citation_date" content="2017-10-09" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/10.1101/200352v1.full.pdf" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/10.1101/200352v1.full.txt" />
<link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/10.1101/200352v1.ppt" />
<meta name="description" content="bioRxiv - the preprint server for biology, operated by Cold Spring Harbor Laboratory, a research and educational institution" />
<meta name="generator" content="Drupal 7 (http://drupal.org)" />
<link rel="canonical" href="https://www.biorxiv.org/content/10.1101/200352v1" />
<link rel="shortlink" href="https://www.biorxiv.org/node/60913" />
    <title>Landscape genomic prediction for restoration of a Eucalyptus foundation species under climate change | bioRxiv</title>  
    <link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__jMRAK66KMC1e4TQlwUNn3KiWVDC5AjueUAYEm1xBY_U__KSUjdT4jdcJ4qJz6fKY1K9WkYh1a5EcaWZxt_-zbTis__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q8002p" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__icrP-Y0qw2NP73rJxxkUtrJta0z3VLAABtNjfSdEghA__rlN3rx0wOoswGNHZMeZM96bNn-3cLWHXTztGDoThNlw__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__q48SDxVDWut_cH6dTPyQe7HjADuSLLmJVFztuiJEFrU__J9zYrFgVU9w_9DRO_7M268JyEPb2If_fHKWgrFly-Tw__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<style type="text/css" media="all">
/* <![CDATA[ */
#sliding-popup.sliding-popup-bottom,#sliding-popup.sliding-popup-bottom .eu-cookie-withdraw-banner,.eu-cookie-withdraw-tab{background:gray}#sliding-popup.sliding-popup-bottom.eu-cookie-withdraw-wrapper{background:transparent}#sliding-popup .popup-content #popup-text h1,#sliding-popup .popup-content #popup-text h2,#sliding-popup .popup-content #popup-text h3,#sliding-popup .popup-content #popup-text p,.eu-cookie-compliance-secondary-button,.eu-cookie-withdraw-tab{color:#fff !important}.eu-cookie-withdraw-tab{border-color:#fff}.eu-cookie-compliance-more-button{color:#fff !important}
/* ]]> */
</style>

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__B6tGdsQSt1vT8d8j-19-oS4FKJe83RrSNBkjPs9jfos__6VkE5WcQmmQpSVTQx-xS2-zn_nlwFlDm-EN1HLJNjrA__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__78WRxe8YsWkYp6RcQ_VwTjFG__x9mi_NTPqoxAwFUsA__F3sVKlGyHKCaAPI_WdiHD7l_-05B6b9bLdUwzi1me90__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__dAeSAHswzRXxn-2n4D6zNPmcZDmwaqqLzE-2ab_8jKw__f_9pwrTE8mmshoJRfurwkQ36QonD_Jiwbop2sscnkrE__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://d33xdlntwy0kbs.cloudfront.net/cshl_custom.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/all/libraries/modernizr/modernizr.min.js?q8002p"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__yOhrJUBmMpN-jsuQlJYmS8ovkQU6I6KInr295HrSTb0__c--zHhbuOWmkdWkJYXiNWDmkpDRPesmbBjGlQcyqBOY__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var _prum=[['id', '52e95df3abe53d2e33000000'], ['mark', 'firstbyte',
      (new Date()).getTime()]]; (function() {
      var s=document.getElementsByTagName('script')[0],
      p=document.createElement('script');
      p.async='async'; p.src='//rum-static.pingdom.net/prum.min.js';s.parentNode.insertBefore(p,s);})();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js___KyvuEvfmewWxdOLBXTYsSUo8Pfm_DijV6fWtGvSL_k__HSKkTgIr7nDcdvQYewipf6v4ak6ZLUOQkwPj44wKouA__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__3uCzzYE5jr6ly9nvsM8PRBRkhh4p0NWCMDja4EKBy4M__dmGGVYcB8LhnbR2iAmVW0ZyXI6QuFVYFRORa5XxglXs__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__0zdUME0GLauYYyEch2tGPydIw-IG7Uizip0Qb4wUsig__3RgszrCgk4bOnTzo4vaQ-hrWzaG6oD19l-R9CQ1YQ-o__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__W6wAjopaALytcBam7oT3WhEzllkWaB9R-8hazHyvShM__rK5YBIqBy9qt0di0Sn4DriBx9TJupdmv6uGmLJyZvDs__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-45638026-1", {"cookieDomain":"auto"});ga("set", "page", location.pathname + location.search + location.hash);ga("send", "pageview");ga('create', 'UA-189672-38', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__1PtAOZ8x7l6QAi8D7X6Kj_Y7HcUOFzvWRvvYHUcCT98__U8IB92u_CnnGwJ-X_xaUa9lzsC8OzUfN0WkK2JQfirg__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings,{"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"9-brmDn7hGbmSjMKPvyD9mvQXi36qrhj-oUXcdx2KMM","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"modules\/comment\/comment.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"modules\/forum\/forum.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"sites\/default\/modules\/biorxiv\/css\/biorxiv.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/flexible\/flexible.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_collection\/css\/highwire_subject_collection.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire.highwire-list-expand.css":1,"public:\/\/ctools\/css\/bab59090150caf0c64e27afa1b36eef1.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/contrib\/eu_cookie_compliance\/css\/eu_cookie_compliance.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"0":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_biorxiv_styles\/css\/biorxiv-styles.css.less":1,"https:\/\/d33xdlntwy0kbs.cloudfront.net\/cshl_custom.css":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_figures.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_tables.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"http:\/\/d33xdlntwy0kbs.cloudfront.net\/cshl_custom.js":1,"sites\/all\/modules\/contrib\/eu_cookie_compliance\/js\/eu_cookie_compliance.js":1,"sites\/default\/modules\/jnl_biorxiv_styles\/js\/adv-search-collection-toggle.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/eu_cookie_compliance\/js\/jquery.cookie-1.4.1.min.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/default\/modules\/jnl_biorxiv_styles\/js\/biorxiv-scripts.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_nps_survey\/js\/nps_survey.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"sites\/default\/modules\/contrib\/disqus\/disqus.js":1,"misc\/textarea.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/contrib\/service_links\/js\/twitter_button.js":1,"sites\/all\/modules\/contrib\/service_links\/js\/facebook_like.js":1,"sites\/all\/modules\/contrib\/service_links\/js\/google_plus_one.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.highwire-list-expand.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"60913","apath":"\/biorxiv\/early\/2017\/10\/09\/200352.atom","pisa":"biorxiv;200352v1","processed":["highwire_math"],"markup":[{"requested":"full-text","variant":"full-text","view":"full","pisa":"biorxiv;200352v1"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"disqus":{"domain":"biorxivstage","url":"https:\/\/www.biorxiv.org\/content\/10.1101\/200352v1","title":"Landscape genomic prediction for restoration of a Eucalyptus foundation species under climate change","identifier":"node\/60913"},"panels_ajax_pane":{"new-41":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u00221HYHIO4IBu4gdzwLqPUlWzYydPdiFiWSvwc8y96\\\\\\\/WqBxw0dC4qBc5SjiPaAzIJidf1j9jzUpsGUfNO\\\\\\\/6vbukxFHERP4uHDRuOQTfARhgWUTBQw0inS3yVBuxKbPQHgVv6jmw1dfe5CVQJtiY5KcbZho6bykiLisj+2RGRRNe4\\\\\\\/+bfUgr0Kzb99cGzu17BYH4dedrQWFJApprrlspUMaQf8vsQNeq+FTSvYsHJnauwK7tz7rExVHzgxJIAhwhX6HspxGA1ETKIHs+tIyjebdpPB2c2wBcYwyvZH3tfjxwf3sSDb109jnc5s4+ZD+ckz3RF+v+Pinlbl7thzxezaBWTlTfjJz2aLabU4865xcDan+5Ts5j\\\\\\\/Wf5vkAuGN2Fj3qhP1XPU9CJE+cxMLl4d60J5u\\\\\\\/gjtalvf1l1vH00bajGcuTeSHQnVWC7\\\\\\\/VHIVquu2teb7dN09hjoAwIjv93ZCgnskWRSsVKiwZyC++t\\\\\\\/NOZbNO2W1zlh5qz19Mz2VCJZFecLK\\\\\\\/D\\\\\\\/xmyrTGWx36mOaF5CDVZQvzCECWRAeFX3OVzpsXg1jgTsGThBqIVQfA6HIYSCFEffJzt+Bjoc1QDLJ9qKxPrccOguXNtc8LF7sRyr6cI42B3TQoEteoaDfa5cfOTQq6Qu2e2rQiLBkIVPH49DDJIFwAA+PzrGT39Ld76biG7Rn+DmTWIfH6vzaq6Qx00zaD8hWNJ3vD5GJtf+o\\\\\\\/zcY5sNfPkiFBorPgdnpEy01lN6I6ncWgX3XDJ1Hzco8q52LuSthnt\\\\\\\/fDQUS+57Vny5jQyG\\\\\\\/8+Uq+xpyRmCiTzwUFmw\\\\\\\/2i3rTTB6RtHU8\\\\\\\/xAceTv1D\\\\\\\/Jl3zmwCuFCYhNq1mvUz31P84RaFqkKn6susuDtoaA3oLSqZS2Bau1+qk7kfV9W+sy0Lr+akbKN7k+IVxw5LeEFoK7aRm\\\\\\\/ci+UDY9eTe3XVxXY6IwTVlz36Wk2OFljKqBltbJAo04E4Ztz3dbY5Zxwy9iPfb4T7qIeuEusZc92W8RO\\\\\\\/zHmhyv72K6pH5lP6bUPlZJ\\\\\\\/1BJnFBVfP3mqBBltSzYunwAGScATu80QKZYZZi6LuYWaP\\\\\\\/hnmZQEXQBumTpNaeDjsjYDWdJjiBIgqJAEXwGD3ULyBNO5hLfVa99pXwpRVXEhk6OrSPepHal1TOB7A4wjmxg5Y61rX3LSLao8Pf4pRjxCjaRaIisP+jFtxs16DIi\\\\\\\/xsEGlIuvsHx6PEcJbYABhxE7heRqxKMfb3JQdjx00rjXp68Q0ZueiPxk9\\\\\\\/+C9KiPjsaMK\\\\\\\/OjHjbgMN2apvbcOhenaGbR7y+04A69fQbA6DYB0QfNWnoeo=\\\u0022,\\\u0022iv\\\u0022:\\\u0022Ftar5Pa0MiU7ndwOclhuWQ==\\\u0022,\\\u0022salt\\\u0022:\\\u0022f9d8845712f72c9fee5db7d9ff111eab\\\u0022}\u0022,\u0022hmac\u0022:\u002242e5d438fd57a272e8441f94fe9a15f3305c9a732a2e093cad14e3fe223c1152\u0022}"},"urlIsAjaxTrusted":{"\/content\/10.1101\/200352v1.full":true},"ws_fl":{"width":100,"height":21},"ws_gpo":{"size":"","annotation":"","lang":"","callback":"","width":300},"color":{"logo":"https:\/\/www.biorxiv.org\/sites\/default\/files\/bioRxiv_article.jpg"},"highwire_list_expand":{"is_collapsed":"1"},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"eu_cookie_compliance":{"popup_enabled":1,"popup_agreed_enabled":0,"popup_hide_agreed":0,"popup_clicking_confirmation":1,"popup_scrolling_confirmation":false,"popup_html_info":"\u003Cdiv\u003E\n  \u003Cdiv class =\u0022popup-content info\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n      \u003Cp\u003EWe use cookies on this site to enhance your user experience. By clicking any link on this page you are giving your consent for us to set cookies.\u003C\/p\u003E\n    \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022agree-button eu-cookie-compliance-default-button\u0022\u003EContinue\u003C\/button\u003E\n                    \u003Cbutton type=\u0022button\u0022 class=\u0022find-more-button eu-cookie-compliance-more-button\u0022\u003EFind out more\u003C\/button\u003E\n          \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E","use_mobile_message":false,"mobile_popup_html_info":"\u003Cdiv\u003E\n  \u003Cdiv class =\u0022popup-content info\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n          \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022agree-button eu-cookie-compliance-default-button\u0022\u003EContinue\u003C\/button\u003E\n                    \u003Cbutton type=\u0022button\u0022 class=\u0022find-more-button eu-cookie-compliance-more-button\u0022\u003EFind out more\u003C\/button\u003E\n          \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E\n","mobile_breakpoint":"768","popup_html_agreed":"\u003Cdiv\u003E\n  \u003Cdiv class=\u0022popup-content agreed\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n      \u003Ch2\u003EThank you for accepting cookies\u003C\/h2\u003E\u003Cp\u003EYou can now hide this message or find out more about cookies.\u003C\/p\u003E    \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022hide-popup-button eu-cookie-compliance-hide-button\u0022\u003EHide\u003C\/button\u003E\n              \u003Cbutton type=\u0022button\u0022 class=\u0022find-more-button eu-cookie-compliance-more-button-thank-you\u0022 \u003EMore info\u003C\/button\u003E\n          \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E","popup_use_bare_css":false,"popup_height":"auto","popup_width":"100%","popup_delay":1000,"popup_link":"\/help\/cookie-policy","popup_link_new_window":1,"popup_position":null,"popup_language":"en","store_consent":false,"better_support_for_screen_readers":0,"reload_page":0,"domain":"","popup_eu_only_js":0,"cookie_lifetime":"365","cookie_session":false,"disagree_do_not_show_popup":0,"method":"default","whitelisted_cookies":"","withdraw_markup":"\u003Cbutton type=\u0022button\u0022 class=\u0022eu-cookie-withdraw-tab\u0022\u003EPrivacy settings\u003C\/button\u003E\n\u003Cdiv class=\u0022eu-cookie-withdraw-banner\u0022\u003E\n  \u003Cdiv class=\u0022popup-content info\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n      \u003Cp\u003E\u0026lt;h2\u0026gt;We use cookies on this site to enhance your user experience\u0026lt;\/h2\u0026gt;\u0026lt;p\u0026gt;You have given your consent for us to set cookies.\u0026lt;\/p\u0026gt;\u003C\/p\u003E\n    \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022eu-cookie-withdraw-button\u0022\u003EWithdraw consent\u003C\/button\u003E\n    \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E\n","withdraw_enabled":false},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1,"trackUrlFragments":1},"jnl_biorxiv_styles":{"defaultJCode":"biorxiv"},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-60913 node-type-highwire-article context-content hw-default-jcode-biorxiv hw-article-type-article hw-article-category-new-results">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-M677548" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-M677548');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
        <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
    
  <div id="zone-branding" class="zone zone-branding clearfix print-display-block container-30">
    <div class="grid-15 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="bioRxiv" src="https://www.biorxiv.org/sites/default/files/bioRxiv_article.jpg" /></a>      </div>
                </div>
          </div>
</div><div class="grid-11 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-system block-menu block-main-menu block-system-main-menu odd block-without-title" id="block-system-main-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/" title="" class="" data-icon-position="" data-hide-link-title="0">Home</a></li>
<li class="leaf" role="menuitem"><a href="/about-biorxiv" class="" data-icon-position="" data-hide-link-title="0">About</a></li>
<li class="leaf" role="menuitem"><a href="/submit-a-manuscript" class="" data-icon-position="" data-hide-link-title="0">Submit</a></li>
<li class="last leaf" role="menuitem"><a href="/content/alertsrss" title="" class="" data-icon-position="" data-hide-link-title="0">ALERTS / RSS</a></li>
</ul></nav>    </div>
  </div>
</div><div class="block block-panels-mini block-biorxiv-search-box block-panels-mini-biorxiv-search-box even block-without-title" id="block-panels-mini-biorxiv-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-biorxiv_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/10.1101/200352v1.full" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--2" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-VVnsZgaZwL5PfhvQV-_N0G9fYKkCaQjPx2_zjhCyBdo" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2 advanced-search-link" >
  
      
  
  <div class="pane-content">
    <a href="/search">Advanced Search</a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix container-30">
  	      </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="60913" id="node60913" data-pisa="biorxiv;200352v1" data-pisa-master="biorxiv;200352" data-apath="/biorxiv/early/2017/10/09/200352.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div  class="highwire-cite highwire-cite-highwire-article highwire-citation-biorxiv-article-top clearfix has-author-tooltip" >

      <span class="biorxiv-article-type">
        New Results    </span>
  
        <h1 class="highwire-cite-title" id="page-title">Landscape genomic prediction for restoration of a <em>Eucalyptus</em> foundation species under climate change</h1>  
      <div  class="highwire-cite-authors" ><span  class="highwire-citation-authors"><span class="highwire-citation-author first hw-author-orcid-logo-wrapper" data-delta="0"><a href="http://orcid.org/0000-0002-0204-7852" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a><span class="nlm-given-names">Megan A.</span> <span class="nlm-surname">Supple</span></span>, <span class="highwire-citation-author" data-delta="1"><span class="nlm-given-names">Jason G.</span> <span class="nlm-surname">Bragg</span></span>, <span class="highwire-citation-author" data-delta="2"><span class="nlm-given-names">Linda M.</span> <span class="nlm-surname">Broadhurst</span></span>, <span class="highwire-citation-author" data-delta="3"><span class="nlm-given-names">Adrienne B.</span> <span class="nlm-surname">Nicotra</span></span>, <span class="highwire-citation-author" data-delta="4"><span class="nlm-given-names">Margaret</span> <span class="nlm-surname">Byrne</span></span>, <span class="highwire-citation-author" data-delta="5"><span class="nlm-given-names">Rose L.</span> <span class="nlm-surname">Andrew</span></span>, <span class="highwire-citation-author" data-delta="6"><span class="nlm-given-names">Abigail</span> <span class="nlm-surname">Widdup</span></span>, <span class="highwire-citation-author" data-delta="7"><span class="nlm-given-names">Nicola C.</span> <span class="nlm-surname">Aitken</span></span>, <span class="highwire-citation-author" data-delta="8"><span class="nlm-given-names">Justin O.</span> <span class="nlm-surname">Borevitz</span></span></span></div>
  
      <div  class="highwire-cite-metadata" ><span  class="highwire-cite-metadata-doi highwire-cite-metadata"><span class="label">doi:</span> https://doi.org/10.1101/200352 </span></div>
  
  
  </div>
<div id="hw-article-author-popups-node60913" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Megan A. Supple </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>a</span><span class='nlm-institution'>Research School of Biology, The Australian National University</span>, 134 Linnaeus Way, Canberra, ACT 2601, <span class='nlm-country'>Australia</span>;</div><div class='author-affiliation'><span class='nlm-sup'>b</span><span class='nlm-institution'>Department of Ecology and Evolutionary Biology, University of California at Santa Cruz</span>, 1156 High Street, Santa Cruz, CA 95064, <span class='nlm-country'>USA</span>;</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Megan%2BA.%2BSupple%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Supple%20MA&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AMegan%2BA.%2BSupple%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0002-0204-7852" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Megan A. Supple</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:megan.a.supple@gmail.com" class="" data-icon-position="" data-hide-link-title="0">megan.a.supple@gmail.com</a></span></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Jason G. Bragg </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>a</span><span class='nlm-institution'>Research School of Biology, The Australian National University</span>, 134 Linnaeus Way, Canberra, ACT 2601, <span class='nlm-country'>Australia</span>;</div><div class='author-affiliation'><span class='nlm-sup'>c</span><span class='nlm-institution'>National Herbarium of New South Wales, The Royal Botanic Gardens and Domain Trust</span>, Mrs Macquaries Road, Sydney, NSW 2000, <span class='nlm-country'>Australia</span>;</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jason%2BG.%2BBragg%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Bragg%20JG&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJason%2BG.%2BBragg%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Linda M. Broadhurst </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>d</span><span class='nlm-institution'>Centre for Australian National Biodiversity Research, Commonwealth Scientific and Industrial Research Organisation (CSIRO) National Research Collections and Facilities</span>, PO Box1600, Canberra, ACT 2601, <span class='nlm-country'>Australia</span>;</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Linda%2BM.%2BBroadhurst%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Broadhurst%20LM&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ALinda%2BM.%2BBroadhurst%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Adrienne B. Nicotra </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>a</span><span class='nlm-institution'>Research School of Biology, The Australian National University</span>, 134 Linnaeus Way, Canberra, ACT 2601, <span class='nlm-country'>Australia</span>;</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Adrienne%2BB.%2BNicotra%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Nicotra%20AB&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AAdrienne%2BB.%2BNicotra%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Margaret Byrne </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>e</span><span class='nlm-institution'>Science and Conservation Division, Department of Parks and Wildlife Western Australia</span>, Locked Bag 104, Bentley Delivery Centre, WA, 6983, <span class='nlm-country'>Australia</span>;</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Margaret%2BByrne%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Byrne%20M&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AMargaret%2BByrne%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Rose L. Andrew </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>f</span><span class='nlm-institution'>School of Environmental and Rural Science, University of New England</span>, W55 Geology Rd, Armidale, NSW 2351, <span class='nlm-country'>Australia</span>;</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Rose%2BL.%2BAndrew%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Andrew%20RL&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ARose%2BL.%2BAndrew%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-6"><div class="author-tooltip-name">Abigail Widdup </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>a</span><span class='nlm-institution'>Research School of Biology, The Australian National University</span>, 134 Linnaeus Way, Canberra, ACT 2601, <span class='nlm-country'>Australia</span>;</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Abigail%2BWiddup%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Widdup%20A&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AAbigail%2BWiddup%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-7"><div class="author-tooltip-name">Nicola C. Aitken </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>a</span><span class='nlm-institution'>Research School of Biology, The Australian National University</span>, 134 Linnaeus Way, Canberra, ACT 2601, <span class='nlm-country'>Australia</span>;</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Nicola%2BC.%2BAitken%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Aitken%20NC&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ANicola%2BC.%2BAitken%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-8"><div class="author-tooltip-name">Justin O. Borevitz </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>a</span><span class='nlm-institution'>Research School of Biology, The Australian National University</span>, 134 Linnaeus Way, Canberra, ACT 2601, <span class='nlm-country'>Australia</span>;</div><div class='author-affiliation'><span class='nlm-sup'>g</span><span class='nlm-institution'>Centre of Excellence in Plant Energy Biology, The Australian National University</span>, 134 Linnaeus Way, Canberra, ACT 2601, <span class='nlm-country'>Australia</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Justin%2BO.%2BBorevitz%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Borevitz%20JO&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJustin%2BO.%2BBorevitz%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/10.1101/200352v1" class="panels-ajax-tab-tab" data-panel-name="biorxiv_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:60913" data-trigger="" data-url-enabled="1">Abstract</a><a href="/panels_ajax_tab/biorxiv_tab_art/node:60913/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/10.1101/200352v1.full-text" class="panels-ajax-tab-tab" data-panel-name="article_tab_full_text" data-target-id="highwire_article_tabs" data-entity-context="node:60913" data-trigger="full-text" data-url-enabled="1">Full Text</a><a href="/panels_ajax_tab/article_tab_full_text/node:60913/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/10.1101/200352v1.article-info" class="panels-ajax-tab-tab" data-panel-name="biorxiv_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:60913" data-trigger="article-info" data-url-enabled="1">Info/History</a><a href="/panels_ajax_tab/biorxiv_tab_info/node:60913/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/10.1101/200352v1.article-metrics" class="panels-ajax-tab-tab" data-panel-name="article_tab_metrics" data-target-id="highwire_article_tabs" data-entity-context="node:60913" data-trigger="article-metrics" data-url-enabled="1">Metrics</a><a href="/panels_ajax_tab/article_tab_metrics/node:60913/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/10.1101/200352v1.full.pdf+html" class="panels-ajax-tab-tab" data-panel-name="biorxiv_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:60913" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i> Preview PDF</a><a href="/panels_ajax_tab/biorxiv_tab_pdf/node:60913/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="article_tab_full_text" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.biorxiv.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-article_tab_full_text"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-markup" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2 class="">Abstract</h2><p id="p-2">As species face rapid environmental change, we can build resilient populations through restoration projects that incorporate predicted future climates into seed sourcing decisions. <em>Eucalyptus melliodora</em> is a foundation species of a critically endangered community in Australia that is a target for restoration. We examined patterns of genomic and phenotypic variation to make empirical based recommendations for seed sourcing. We examined isolation by distance and isolation by environment, determining gene flow up to 500 km and associations with environmental variables. Climate chamber studies revealed extensive phenotypic variation both within and among sampling sites, but no site-specific differentiation in phenotypic plasticity. Overall our results suggest that seed can be sourced broadly across the landscape, providing ample diversity for adaptation to environmental change. Application of our landscape genomic model to <em>E. melliodora</em> restoration projects can identify genomic variation suitable for predicted future climates, thereby increasing the long term probability of successful restoration.</p></div><div class="section" id="sec-1"><h2 class="">Introduction</h2><p id="p-3">Species around the globe face rapidly changing environments, often in combination with habitat degradation and fragmentation. These factors are expected to have a negative impact on biodiversity (<a id="xref-ref-28-1" class="xref-bibr" href="#ref-28">Lindenmayer et al., 2010</a>). Three processes enable species survival in the face of altered conditions-migration, adaptation, and phenotypic plasticity (<a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">Aitken &amp; Whitlock, 2013</a>; <a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">Aitken et al., 2008</a>; <a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">Hoffmann et al., 2015</a>; <a id="xref-ref-35-1" class="xref-bibr" href="#ref-35">Nicotra et al., 2010</a>). An important conservation strategy is to assist these natural processes to help build more resilient communities. We can help species shift to regions with their preferred environmental conditions by assisting migration of gene pools across the landscape (<a id="xref-ref-1-2" class="xref-bibr" href="#ref-1">Aitken &amp; Whitlock, 2013</a>; <a id="xref-ref-2-2" class="xref-bibr" href="#ref-2">Aitken et al., 2008</a>). We can aid populations to survive in situ by ensuring that sufficient genomic variation exists for adaptation to changing environments (<a id="xref-ref-24-2" class="xref-bibr" href="#ref-24">Hoffmann et al., 2015</a>). We can enable individuals to respond to a greater range of environments by conserving existing phenotypic plasticity (<a id="xref-ref-35-2" class="xref-bibr" href="#ref-35">Nicotra et al., 2010</a>).</p><p id="p-4">Seed sourcing during landscape restoration provides an ideal opportunity to apply scientific knowledge to enable these key processes and improve conservation outcomes (<a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">Broadhurst et al., 2008</a>; <a id="xref-ref-38-1" class="xref-bibr" href="#ref-38">Prober et al., 2015</a>). For example, seed sources can be selected to restore historical patterns of gene flow across a fragmented landscape, incorporate high genomic diversity, and/or increase phenotypic plasticity. Seed sources can also be matched with current or projected future climates, enabling assisted migration to favorable environments (<a id="xref-ref-1-3" class="xref-bibr" href="#ref-1">Aitken &amp; Whitlock, 2013</a>; <a id="xref-ref-46-1" class="xref-bibr" href="#ref-46">Williams et al., 2014</a>).</p><p id="p-5">Historically, restoration often focused on geographically restricted local sources of seed under the premise that this would improve restoration outcomes by reducing the risk of maladaptation to local conditions and preventing outbreeding depression (<a id="xref-ref-10-2" class="xref-bibr" href="#ref-10">Broadhurst et al., 2008</a>). However, there are several potential drawbacks to this narrow local focus. In a fragmented system, narrow local seed sourcing reduces the number of potential source populations, thereby reducing the pool of available genetic material. This reduced gene pool may result in inbreeding depression in future generations, especially if combined with small population size (<a id="xref-ref-10-3" class="xref-bibr" href="#ref-10">Broadhurst et al., 2008</a>). Obtaining potential seed sources from a wider geographical area can increase genomic and phenotypic diversity, thereby increasing the ability of the species to survive in situ (<a id="xref-ref-10-4" class="xref-bibr" href="#ref-10">Broadhurst et al., 2008</a>). Additionally, the focus on maintaining local adaptation in situ assumes a static environment, not the rapidly changing environment that occurs today. As local conditions change, traits and genes that may have conferred an advantage in the past might not be suitable in the future environment. In recent year, climate adjusted provenancing has been proposed, which is a seed sourcing strategy that incorporates climate variability and focuses on sourcing seed that is predicted to be adapted to future climates (<a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">Byrne et al., 2013</a>; <a id="xref-ref-38-2" class="xref-bibr" href="#ref-38">Prober et al., 2015</a>). This strategic assisted migration of variation across the landscape can aid in the establishment of populations that are more adaptable to future environments (<a id="xref-ref-38-3" class="xref-bibr" href="#ref-38">Prober et al., 2015</a>).</p><p id="p-6">To determine the appropriate seed sourcing strategy and to identify optimal seed sources for a reforestation project, empirical knowledge of genomic variation for the target species can provide valuable information. The technology now exists to assess genomic variation in any target species, enabling determination of patterns of Isolation By Distance (IBD) and Isolation By Environment (IBE). IBD is the association between genomic distance and geographic distance resulting from patterns of dispersal. IBE is the association between genomic distance and environmental distance, while controlling for geographic distance (<a id="xref-ref-44-1" class="xref-bibr" href="#ref-44">Wang &amp; Bradburd, 2014</a>). Landscape genomic models can be generated by fitting geographic and environmental variables to the observed genomic diversity. These predictive models can optimize the genetic material selected for restoration and should improve long term outcomes (<a id="xref-ref-24-3" class="xref-bibr" href="#ref-24">Hoffmann et al., 2015</a>; <a id="xref-ref-46-2" class="xref-bibr" href="#ref-46">Williams et al., 2014</a>).</p><p id="p-7">The extent of phenotypic plasticity in potential seed sources can be measured in growth assays of seedling traits across contrasting environmental conditions. The magnitude of the environmental response can be compared among maternal lines or populations and may identify populations that differ in their response to the environment. Such differing responses have been seen in some species of <em>Eucalyptus</em> (<a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">Andrew et al., 2010</a>; <a id="xref-ref-12-2" class="xref-bibr" href="#ref-12">Byrne et al., 2013</a>; <a id="xref-ref-31-1" class="xref-bibr" href="#ref-31">McLean et al., 2014</a>).</p><p id="p-8"><em>Eucalyptus melliodora</em> (A.Cunn. ex Schauer), commonly called yellow box, is an iconic Australian species that is the subject of extensive restoration efforts across its distribution. It is a foundation species of a critically endangered ecological community: the White Box–Yellow Box–Blakely’s Red Gum Grassy Woodland and Derived Native Grassland (<a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">Department of Environment and Climate Change and Water, 2011</a>; <a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">Department of the Environment and Heritage, 2006</a>; <a id="xref-ref-42-1" class="xref-bibr" href="#ref-42">Threatened Species Scientific Committee, 2006</a>). This woodland community exists in a fragmented landscape, with less than 5% of its original distribution remaining, mostly in small remnant patches (<a id="xref-ref-16-2" class="xref-bibr" href="#ref-16">Department of Environment and Climate Change and Water, 2011</a>; <a id="xref-ref-17-2" class="xref-bibr" href="#ref-17">Department of the Environment and Heritage, 2006</a>; <a id="xref-ref-42-2" class="xref-bibr" href="#ref-42">Threatened Species Scientific Committee, 2006</a>). Efforts to restore this endangered woodland community are ongoing and restoration practitioners are seeking scientific recommendations to improve seed sourcing. Climate change is an important consideration in seed sourcing decisions because species distribution modelling predicts that most eucalypts will need to shift their distributions considerably in response (<a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">González-Orozco et al., 2016</a>). In particular, for <em>E. melliodora</em> ecological niche modelling predicts that by 2090 the species distribution will shift toward the southeast and suitable areas will decrease by 77% as a result of environmental changes (Broadhurst et al., <em>in review</em>).</p><p id="p-9">Here we survey genomic variation in 275 individuals from 37 sites across the present range of <em>E. melliodora</em>. We fit the genotypic data to geographic distance and key environmental variables at the site of origin. We find that effects of genomic isolation by distance begin at approximately 500km. This empirical estimate of "local" is much farther than what is often practiced for local provenancing. We also find that features of the abiotic environment can further explain genomic differentiation after accounting for geographic distance. We also examine seedling growth characteristics under simulated climate conditions and find significant variation in growth traits both within and among sites, but no significant variation in phenotypic plasticity across sites. Our landscape genomic model, which can empirically define local provenances and identify variation suitable for predicted future climates, can help build resilient populations through scientifically based restoration.</p></div><div class="section" id="sec-2"><h2 class="">Results</h2><div id="sec-3" class="subsection"><h3>Genotyping by Sequencing</h3><p id="p-10">We selected leaf material from 39 sites, sampling 3-10 trees per site (<a id="xref-table-wrap-1-1" class="xref-table" href="#T1">Supplemental Table S1</a>). For each sample we Illumina sequenced a Genotyping by Sequencing (GBS) library (<a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">Elshire et al., 2011</a>) and used a reference alignment-based approach to call genotypes. We conducted a preliminary analysis, based on 123,227 SNPs and removed 69 samples due to greater than 60% missing data. Visual examination of a cluster dendrogram of genomic distance between samples showed that technical replicates cluster closely together (<a id="xref-fig-5-1" class="xref-fig" href="#F5">Supplemental Fig. S1</a>). A preliminary principal coordinate analysis (PCA) identified 19 samples that were strong genomic outliers (<a id="xref-fig-6-1" class="xref-fig" href="#F6">Supplemental Fig. S2</a>), likely misidentified samples or recent hybrids. This result is consistent with minor morphological differences noted in these samples, as well as previous microsatellite work (Broadhurst et al., <em>in review</em>). After removal of poor quality and geographic and genomic outlier samples, we reran the genotyping with the remaining 280 samples, resulting in 9,781 SNPs after filtering. A second preliminary PCA identified an additional 5 outlier samples that we considered sufficiently differentiated from the main <em>E. melliodora</em> cluster to merit removal for downstream analyses (<a id="xref-fig-7-1" class="xref-fig" href="#F7">Supplemental Fig. S3</a>). We removed these samples and reran the missing data filter. The final data set included 275 samples from 37 sites (<a id="xref-fig-1-1" class="xref-fig" href="#F1">Fig. 1A</a>), genotyped at 9,378 physically distinct SNPs (&gt;300 bp apart).</p><div id="F5" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2017/10/09/200352/F5.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Technical replicate dendrogram Dendrogram based on genomic distance between samples showing the strong clustering of technical replicates (denoted with an &quot;R&quot; after the sample name and highlighted in yellow). Note that three of the technical replicates failed to pass quality control and are not included in the dendrogram. Additional sample pairs show strong clustering. In each case, the individuals of the pair are from the same sampling site, indicating samples that are closely related." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-864888468" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;span xmlns=&quot;http://www.w3.org/1999/xhtml&quot; class=&quot;caption-title&quot;&gt;Technical replicate dendrogram&lt;/span&gt; Dendrogram based on genomic distance between samples showing the strong clustering of technical replicates (denoted with an &quot;R&quot; after the sample name and highlighted in yellow). Note that three of the technical replicates failed to pass quality control and are not included in the dendrogram. Additional sample pairs show strong clustering. In each case, the individuals of the pair are from the same sampling site, indicating samples that are closely related.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure S1." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2017/10/09/200352/F5.medium.gif" width="440" height="164"/><noscript><img class="highwire-fragment fragment-image" alt="Figure S1." src="https://www.biorxiv.org/content/biorxiv/early/2017/10/09/200352/F5.medium.gif" width="440" height="164"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2017/10/09/200352/F5.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure S1." data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2017/10/09/200352/F5.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml"><span class="fig-label">Figure S1.</span> <span class="caption-title">Technical replicate dendrogram</span><p id="p-58" class="first-child">Dendrogram based on genomic distance between samples showing the strong clustering of technical replicates (denoted with an "R" after the sample name and highlighted in yellow). Note that three of the technical replicates failed to pass quality control and are not included in the dendrogram. Additional sample pairs show strong clustering. In each case, the individuals of the pair are from the same sampling site, indicating samples that are closely related.</p><div class="sb-div caption-clear"></div></div></div><div id="F6" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2017/10/09/200352/F6.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Species identification PCA PCA of genomic distance between samples showing strong outliers that are likely misidentified samples or hybrids. The vertical line at 50 on PCA axis 1 indicates the cutoff, with all samples to the right removed from further analyses." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-864888468" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;span xmlns=&quot;http://www.w3.org/1999/xhtml&quot; class=&quot;caption-title&quot;&gt;Species identification PCA&lt;/span&gt; PCA of genomic distance between samples showing strong outliers that are likely misidentified samples or hybrids. The vertical line at 50 on PCA axis 1 indicates the cutoff, with all samples to the right removed from further analyses.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure S2." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2017/10/09/200352/F6.medium.gif" width="440" height="405"/><noscript><img class="highwire-fragment fragment-image" alt="Figure S2." src="https://www.biorxiv.org/content/biorxiv/early/2017/10/09/200352/F6.medium.gif" width="440" height="405"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2017/10/09/200352/F6.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure S2." data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2017/10/09/200352/F6.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure S2.</span> <span class="caption-title">Species identification PCA</span><p id="p-59" class="first-child">PCA of genomic distance between samples showing strong outliers that are likely misidentified samples or hybrids. The vertical line at 50 on PCA axis 1 indicates the cutoff, with all samples to the right removed from further analyses.</p><div class="sb-div caption-clear"></div></div></div><div id="F7" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2017/10/09/200352/F7.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Outlier PCA PCA of genomic distance between samples for the confirmed E. melliodora samples. The five samples on the left were deemed outliers and removed from further analyses." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-864888468" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;&lt;span class=&quot;caption-title&quot;&gt;Outlier PCA&lt;/span&gt; PCA of genomic distance between samples for the confirmed &lt;em&gt;E. melliodora&lt;/em&gt; samples. The five samples on the left were deemed outliers and removed from further analyses.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure S3." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2017/10/09/200352/F7.medium.gif" width="440" height="412"/><noscript><img class="highwire-fragment fragment-image" alt="Figure S3." src="https://www.biorxiv.org/content/biorxiv/early/2017/10/09/200352/F7.medium.gif" width="440" height="412"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2017/10/09/200352/F7.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure S3." data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2017/10/09/200352/F7.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure S3.</span> <span class="caption-title">Outlier PCA</span><p id="p-60" class="first-child">PCA of genomic distance between samples for the confirmed <em>E. melliodora</em> samples. The five samples on the left were deemed outliers and removed from further analyses.</p><div class="sb-div caption-clear"></div></div></div><div id="T1" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/368682/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup"><a href="/highwire/markup/368682/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/368682" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table S1:</span> <p id="p-67" class="first-child"><em>E. melliodora</em> sampling information</p><div class="sb-div caption-clear"></div></div></div><div id="F1" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2017/10/09/200352/F1.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Map of sampling sites and PCA of genomic distance between samples (A) A map of the geographic locations of the 37 sampling sites in southeastern Australia. Sampling locations are labeled with the site name and color coded in a rainbow gradient based on latitude. Black asterisks indicate 6 sites used for growth chamber experiments. The gray background shading indicates the species distribution polygon. (B) Principal coordinate analysis of the genomic distance between individual samples. Samples are labeled with a sample name that indicates the site name and tree number. Samples are color coded by site to match the map. The percentage on each axis indicates how much of the genomic variation between individuals was explained by the axis. (C) Principal coordinate analysis of Fst between sampling sites. Sampling sites are labeled by name and color coded to match the map. The percentage on each axis indicates how much of the variation in Fst between sampling sites was explained by the axis." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-864888468" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;&lt;span class=&quot;caption-title&quot;&gt;Map of sampling sites and PCA of genomic distance between samples&lt;/span&gt; (A) A map of the geographic locations of the 37 sampling sites in southeastern Australia. Sampling locations are labeled with the site name and color coded in a rainbow gradient based on latitude. Black asterisks indicate 6 sites used for growth chamber experiments. The gray background shading indicates the species distribution polygon. (B) Principal coordinate analysis of the genomic distance between individual samples. Samples are labeled with a sample name that indicates the site name and tree number. Samples are color coded by site to match the map. The percentage on each axis indicates how much of the genomic variation between individuals was explained by the axis. (C) Principal coordinate analysis of F&lt;sub&gt;st&lt;/sub&gt; between sampling sites. Sampling sites are labeled by name and color coded to match the map. The percentage on each axis indicates how much of the variation in F&lt;sub&gt;st&lt;/sub&gt; between sampling sites was explained by the axis.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 1." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2017/10/09/200352/F1.medium.gif" width="440" height="155"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 1." src="https://www.biorxiv.org/content/biorxiv/early/2017/10/09/200352/F1.medium.gif" width="440" height="155"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2017/10/09/200352/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1." data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2017/10/09/200352/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 1.</span> <span class="caption-title">Map of sampling sites and PCA of genomic distance between samples</span><p id="p-11" class="first-child">(A) A map of the geographic locations of the 37 sampling sites in southeastern Australia. Sampling locations are labeled with the site name and color coded in a rainbow gradient based on latitude. Black asterisks indicate 6 sites used for growth chamber experiments. The gray background shading indicates the species distribution polygon. (B) Principal coordinate analysis of the genomic distance between individual samples. Samples are labeled with a sample name that indicates the site name and tree number. Samples are color coded by site to match the map. The percentage on each axis indicates how much of the genomic variation between individuals was explained by the axis. (C) Principal coordinate analysis of F<sub>st</sub> between sampling sites. Sampling sites are labeled by name and color coded to match the map. The percentage on each axis indicates how much of the variation in F<sub>st</sub> between sampling sites was explained by the axis.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-4" class="subsection"><h3>Genomic Analyses</h3><p id="p-12">A PCA of genomic distance among samples showed continuous variation with little suggestion of discrete population structure (<a id="xref-fig-1-2" class="xref-fig" href="#F1">Fig. 1B</a>). The samples largely formed a single cluster, with the first PCA axis corresponding roughly to latitude. Outside of the main cluster, samples from the northernmost site separated out along the first PCA axis (y-axis) and a few samples from two other sites separated out along the second PCA axis (x-axis). Together, the first two PCA axes explained 3.0% of the genomic variation among individuals. The Mantel test estimated that the natural log of the geographic distance between samples explained 2.3% of the variation in individual genomic distance, indicating weak, but statistically significant, isolation by distance (p=0.0001). We summarized genomic diversity between sampling sites using pairwise F<sub>st</sub>. For all comparisons F<sub>st</sub> was low (mean F<sub>st</sub>=0.04, sd=0.02) (<a id="xref-table-wrap-2-1" class="xref-table" href="#T2">Supplemental Table S2</a>). The maximum F<sub>st</sub>=0.10 occurs between sites 3 and 13, which are separated by over 1200 km. Similar to the PCA of genomic distance among samples, the PCA of F<sub>st</sub> between sampling sites corresponded roughly to latitude (<a id="xref-fig-1-3" class="xref-fig" href="#F1">Fig. 1C</a>). In contrast, the first two axes of the PCA of F<sub>st</sub> between sampling sites explained a higher percentage of variation (37.1%). These results highlight the tremendous amount of genomic variation within sampling sites, as well as the ability of thousands of independent genomic markers to distinguish between more distant sampling sites.</p><div id="T2" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/368650/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup"><a href="/highwire/markup/368650/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/368650" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table S2:</span> <p id="p-68" class="first-child">Pairwise population Fst</p><div class="sb-div caption-clear"></div></div></div><p id="p-13">The site by site pairwise F<sub>st</sub> matrix was used to test for geographic and environmental associations using generalized dissimilarity modelling (GDM) (<a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">Ferrier et al., 2007</a>; <a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">Fitzpatrick &amp; Keller, 2015</a>; <a id="xref-ref-41-1" class="xref-bibr" href="#ref-41">Thomassen et al., 2011</a>). Of the 28 environmental variables considered for the model, we removed 12 variables because the single variable model explained less than 5% of the deviance (bioclimatic variables 2, 5, 6, 9, 10, 14, 17, 19; elevation; water at depth; Prescott Index; and MrVBF). We removed an additional 9 variables due to high correlation and lower explanatory power than another remaining variable (bioclimatic variables 1, 4, 7, 12, 13, 15, 18; surface nitrogen; and surface phosphorus) (<a id="xref-table-wrap-3-1" class="xref-table" href="#T3">Supplemental Table S3</a>). We ran permutation testing on a model with the remaining 7 variables and geography. This highlighted an additional 2 variables with low statistical significance and low explanatory power that we removed from the final model (surface water and bioclimatic variable 8). We also removed phosphorus at depth because, although it explained a substantial amount of genomic variation, the sampled sites were not well distributed across the range of values.</p><div id="T3" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/368732/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup"><a href="/highwire/markup/368732/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/368732" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table S3:</span> <p id="p-69" class="first-child">Pearson’s correlation between environmental variables across sampling sites</p><div class="sb-div caption-clear"></div></div></div><p id="p-14">As a result, we included four environmental variables in the final model: isothermality (bioclim 3), mean temperature of the coldest quarter (bioclim 11), precipitation of the wettest quarter (bioclim 16), and total soil nitrogen at 100-200 cm (nitrogen at depth) (<a id="xref-fig-8-1" class="xref-fig" href="#F8">Supplemental Fig. S4</a>). The GDM model with these four variables plus geographic distance explained 40% of the variation in sampling site genomic differentiation (F<sub>st</sub>). The GDM model showed a positive non-linear relationship between environmental distance and genomic distance (<a id="xref-fig-8-2" class="xref-fig" href="#F8">Supplemental Fig. S4A</a>). To test the predictive power of the GDM model, we used a cross validation approach by generating 1000 models with a random 30% of sampling sites removed. GDM proved satisfactory at predicting genomic differences between removed sites (cross validation correlation mean=0.73, standard deviation=0.12) (<a id="xref-fig-8-3" class="xref-fig" href="#F8">Supplemental Fig. S4B</a>).</p><div id="F8" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2017/10/09/200352/F8.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Generalized dissimilarity modelling (GDM) results (A) Non-linear relationship between environmental distance and genomic distance. Points are site pairs; the line is the predicted relationship. (B) Relationship between predicted genomic distance and observed genomic distance. Points are site pairs; the line indicates where observation and prediction match. (C-G) Predicted splines showing the estimated relationship between the environmental variable and genomic distance for (C) total nitrogen content at 100- 200 cm of soil depth, (D) mean temperature of the coldest quarter, (E) precipitation of the wettest quarter, (F) isothermality, and (G) geography. (H) Geographic splines from 100 iterations of sampling 70% of sites. Each solid black line is an iteration; dashed grey line is the full model prediction." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-864888468" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;span xmlns=&quot;http://www.w3.org/1999/xhtml&quot; class=&quot;caption-title&quot;&gt;Generalized dissimilarity modelling (GDM) results&lt;/span&gt; (A) Non-linear relationship between environmental distance and genomic distance. Points are site pairs; the line is the predicted relationship. (B) Relationship between predicted genomic distance and observed genomic distance. Points are site pairs; the line indicates where observation and prediction match. (C-G) Predicted splines showing the estimated relationship between the environmental variable and genomic distance for (C) total nitrogen content at 100- 200 cm of soil depth, (D) mean temperature of the coldest quarter, (E) precipitation of the wettest quarter, (F) isothermality, and (G) geography. (H) Geographic splines from 100 iterations of sampling 70% of sites. Each solid black line is an iteration; dashed grey line is the full model prediction.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure S4." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2017/10/09/200352/F8.medium.gif" width="435" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure S4." src="https://www.biorxiv.org/content/biorxiv/early/2017/10/09/200352/F8.medium.gif" width="435" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2017/10/09/200352/F8.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure S4." data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2017/10/09/200352/F8.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure S4.</span> <span class="caption-title">Generalized dissimilarity modelling (GDM) results</span><p id="p-61" class="first-child">(A) Non-linear relationship between environmental distance and genomic distance. Points are site pairs; the line is the predicted relationship. (B) Relationship between predicted genomic distance and observed genomic distance. Points are site pairs; the line indicates where observation and prediction match. (C-G) Predicted splines showing the estimated relationship between the environmental variable and genomic distance for (C) total nitrogen content at 100- 200 cm of soil depth, (D) mean temperature of the coldest quarter, (E) precipitation of the wettest quarter, (F) isothermality, and (G) geography. (H) Geographic splines from 100 iterations of sampling 70% of sites. Each solid black line is an iteration; dashed grey line is the full model prediction.</p><div class="sb-div caption-clear"></div></div></div><p id="p-15">Of the four environmental variables, nitrogen at depth showed the strongest relationship with genomic distance, with changes in genomic distance predicted across the range of nitrogen values (<a id="xref-fig-8-4" class="xref-fig" href="#F8">Supplemental Fig. S4C</a>). Mean temperature of the coldest quarter was the second strongest predictor, showing a similar pattern as nitrogen (<a id="xref-fig-8-5" class="xref-fig" href="#F8">Supplemental Fig. S4D</a>). Precipitation of the wettest quarter was the third strongest environmental predictor, predicting the largest change in genomic distance between 250 and 400 mm (<a id="xref-fig-8-6" class="xref-fig" href="#F8">Supplemental Fig. S4E</a>). Isothermality (mean diurnal range divided by annual temperature range) was the final predictor, predicting the most change in genomic distance at higher values (<a id="xref-fig-8-7" class="xref-fig" href="#F8">Supplemental Fig. S4F</a>).</p><p id="p-16">Geographic distance showed a non-linear relationship with genomic distance. The geographic spline predicted no genomic differentiation until close to 500 km, at which point an increase in geographic distance predicted an increase in genomic distance (<a id="xref-fig-2-1" class="xref-fig" href="#F2">Fig. 2</a>). Randomly subsampling sites showed that the predicted genomic distance for large geographic distances was quite variable, but for sites less than 500 km apart, all iterations consistently predicted little genomic differentiation between sites (<a id="xref-fig-8-8" class="xref-fig" href="#F8">Supplemental Fig. S4H</a>).</p><div id="F2" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2017/10/09/200352/F2.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Estimated genomic variation as a function of geographic distance The geographic spline estimated from the GDM model showing little predicted genomic change between sites less than 500 km apart and increasing genomic variation as geographic distance increases beyond 500 km." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-864888468" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;span xmlns=&quot;http://www.w3.org/1999/xhtml&quot; class=&quot;caption-title&quot;&gt;Estimated genomic variation as a function of geographic distance&lt;/span&gt; The geographic spline estimated from the GDM model showing little predicted genomic change between sites less than 500 km apart and increasing genomic variation as geographic distance increases beyond 500 km.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 2." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2017/10/09/200352/F2.medium.gif" width="440" height="288"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 2." src="https://www.biorxiv.org/content/biorxiv/early/2017/10/09/200352/F2.medium.gif" width="440" height="288"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2017/10/09/200352/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2." data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2017/10/09/200352/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 2.</span> <span class="caption-title">Estimated genomic variation as a function of geographic distance</span><p id="p-17" class="first-child">The geographic spline estimated from the GDM model showing little predicted genomic change between sites less than 500 km apart and increasing genomic variation as geographic distance increases beyond 500 km.</p><div class="sb-div caption-clear"></div></div></div><p id="p-18">To project the final GDM model onto the current environmental landscape, we first delineated the geographic extent of the analysis by defining an <em>E. melliodora</em> distribution polygon. We then projected the GDM model onto this region using the current values of the environmental variables across the landscape. This analysis partitioned the landscape into a number of regions with different predicted genomic compositions, including northern coastal, northern inland, and southern regions (<a id="xref-fig-3-1" class="xref-fig" href="#F3">Fig.3A</a>). While the biggest differences occurred in regions with few sampling sites, there is a distinction between the northern and southern sites, as well as between sites on opposite sides of the Great Dividing Range in the southern region (e.g. site G versus site A, <a id="xref-fig-3-2" class="xref-fig" href="#F3">Fig. 3A</a>). These projections highlight where environmental filtering of genotypes may have occurred due to different selective pressures.</p><div id="F3" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2017/10/09/200352/F3.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Predicted spatial and temporal variation in genomic composition (A) The spatial distribution of predicted genomic variation based on projecting the GDM model onto geography and current environmental conditions. Regions with similar colors are predicted to have similar genomic composition. (B) The predicted temporal genomic variation based on comparing the GDM model projected onto current environmental conditions and predicted environmental conditions for 2070. The higher the difference (green colors), the more genomic change predicted between current and 2070 conditions. Sampling sites are labeled in black text." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-864888468" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;span xmlns=&quot;http://www.w3.org/1999/xhtml&quot; class=&quot;caption-title&quot;&gt;Predicted spatial and temporal variation in genomic composition&lt;/span&gt; (A) The spatial distribution of predicted genomic variation based on projecting the GDM model onto geography and current environmental conditions. Regions with similar colors are predicted to have similar genomic composition. (B) The predicted temporal genomic variation based on comparing the GDM model projected onto current environmental conditions and predicted environmental conditions for 2070. The higher the difference (green colors), the more genomic change predicted between current and 2070 conditions. Sampling sites are labeled in black text.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 3." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2017/10/09/200352/F3.medium.gif" width="440" height="316"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 3." src="https://www.biorxiv.org/content/biorxiv/early/2017/10/09/200352/F3.medium.gif" width="440" height="316"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2017/10/09/200352/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 3." data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2017/10/09/200352/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 3.</span> <span class="caption-title">Predicted spatial and temporal variation in genomic composition</span><p id="p-19" class="first-child">(A) The spatial distribution of predicted genomic variation based on projecting the GDM model onto geography and current environmental conditions. Regions with similar colors are predicted to have similar genomic composition. (B) The predicted temporal genomic variation based on comparing the GDM model projected onto current environmental conditions and predicted environmental conditions for 2070. The higher the difference (green colors), the more genomic change predicted between current and 2070 conditions. Sampling sites are labeled in black text.</p><div class="sb-div caption-clear"></div></div></div><p id="p-20">We compared the GDM model projected onto current conditions to the GDM model projected onto 2070 climate predictions. This analysis scored each position across the landscape based on how much genomic change was predicted to occur in response to changing environmental conditions (<a id="xref-fig-3-3" class="xref-fig" href="#F3">Fig. 3B</a>). For the middle north region (around sites K2 and 14) and the southern areas towards the coast, the models predicted more intense natural selection in response to climate change. Thus, these areas could be prioritized for assisted migration.</p><p id="p-21">We also used the GDM model to compare the genomic composition under future environmental conditions at a single location to the genomic composition under current climate conditions across the landscape. This comparison is useful for identifying optimal seed sources for restoration sites given climate change scenarios. We demonstrated this utility by selecting two hypothetical reforestation sites and identifying distinct regions that would provide favorable seed sources for each site (<a id="xref-fig-4-1" class="xref-fig" href="#F4">Fig. 4</a>). The analysis for the southern reforestation site identified a large portion of the southern distribution, centered at the reforestation site. For this site it appears that the selected areas are largely driven by the pattern of isolation by distance, in particular the lack of genetic differentiation for long geographical distances. The analysis for the northern reforestation site identified a more limited range of areas across the landscape, possibly driven in part by a decreased power due to lower sampling intensity in the north. In addition to identifying a narrow region in the north that is centered on the reforestation site, a number of more distant areas along the coast were also identified, indicating these selected areas are driven more by patterns of isolation by environment than isolation by distance.</p><div id="F4" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2017/10/09/200352/F4.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Optimal seed sourcing locations for hypothetical reforestation sites The predicted genomic similarity of hypothetical reforestation sites (indicated by black circles) to potential seed sourcing locations under a climate change scenario for 2070. Dark green areas indicate seed sourcing areas predicted to best match future conditions at the hypothetical reforestation sites; white and brown areas indicate areas of potential genomic mismatch." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-864888468" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;span xmlns=&quot;http://www.w3.org/1999/xhtml&quot; class=&quot;caption-title&quot;&gt;Optimal seed sourcing locations for hypothetical reforestation sites&lt;/span&gt; The predicted genomic similarity of hypothetical reforestation sites (indicated by black circles) to potential seed sourcing locations under a climate change scenario for 2070. Dark green areas indicate seed sourcing areas predicted to best match future conditions at the hypothetical reforestation sites; white and brown areas indicate areas of potential genomic mismatch.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 4." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2017/10/09/200352/F4.medium.gif" width="440" height="315"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 4." src="https://www.biorxiv.org/content/biorxiv/early/2017/10/09/200352/F4.medium.gif" width="440" height="315"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2017/10/09/200352/F4.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 4." data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2017/10/09/200352/F4.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 4.</span> <span class="caption-title">Optimal seed sourcing locations for hypothetical reforestation sites</span><p id="p-22" class="first-child">The predicted genomic similarity of hypothetical reforestation sites (indicated by black circles) to potential seed sourcing locations under a climate change scenario for 2070. Dark green areas indicate seed sourcing areas predicted to best match future conditions at the hypothetical reforestation sites; white and brown areas indicate areas of potential genomic mismatch.</p><div class="sb-div caption-clear"></div></div></div><p id="p-23">These genomic analyses suggest that for woodland restoration a geographically wider and environmental model based approach to seed sourcing would allow incorporation of more genetic diversity and enable better matching of the selected genotypes to current and predicted future environmental conditions at the reforestation site.</p></div><div id="sec-5" class="subsection"><h3>Growth Experiments</h3><p id="p-24">We conducted a climate controlled growth experiment to measure variation in seedling growth traits among sampling sites and assay phenotypic plasticity. We grew seedlings from six sites, with six maternal lines per site, at two different climate regimes (average summer conditions and 5°C hotter summer conditions). For analysis of seedling height and total leaf length, we analyzed a total of 291 seedlings (from 32 maternal lines representing six sampling sites) that were determined to be well established at the five week measurement. For analysis of the relative height increment, we analyzed a total of 560 seedlings (from all 36 maternal lines) for which were able to calculate this metric. There were four seedlings that were outliers for the relative growth increment (&gt;0.035). These outliers had little effect on the results of the linear models, so we included them in the final analysis.</p><p id="p-25">The models for all three response variables (seedling height, total leaf length, and relative height increment) showed that all fixed effects (sampling site, maternal line nested within sampling site, and experimental condition) were statistically significant at the p=0.05 level (<a id="xref-table-wrap-4-1" class="xref-table" href="#T4">Supplemental Table S4</a>). Experimental condition explained a small percentage of the variation (1.2-8.1%), as did sampling site (1.8-17.7%) (<a id="xref-fig-9-1" class="xref-fig" href="#F9">Supplemental Fig. S5</a>). Maternal line tended to explain a larger amount of variation (10.6-27.6%).</p><div id="F9" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2017/10/09/200352/F9.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Variation in seedling growth in chamber experiment Box plots showing variation between chamber treatments (left) and sampling sites (right) for three seedling growth traits." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-864888468" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;span xmlns=&quot;http://www.w3.org/1999/xhtml&quot; class=&quot;caption-title&quot;&gt;Variation in seedling growth in chamber experiment&lt;/span&gt; Box plots showing variation between chamber treatments (left) and sampling sites (right) for three seedling growth traits.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure S5." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2017/10/09/200352/F9.medium.gif" width="334" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure S5." src="https://www.biorxiv.org/content/biorxiv/early/2017/10/09/200352/F9.medium.gif" width="334" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2017/10/09/200352/F9.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure S5." data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2017/10/09/200352/F9.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure S5.</span> <span class="caption-title">Variation in seedling growth in chamber experiment</span><p id="p-62" class="first-child">Box plots showing variation between chamber treatments (left) and sampling sites (right) for three seedling growth traits.</p><div class="sb-div caption-clear"></div></div></div><div id="T4" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/368977/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup"><a href="/highwire/markup/368977/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/368977" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table S4:</span> <p id="p-70" class="first-child">Percent of variation explained and p-values for non-interaction linear models for chamber experiment</p><div class="sb-div caption-clear"></div></div></div><p id="p-26">However, most of the variation remained unexplained (56.6-71.5%). None of the three response variables showed significant variation in phenotypic plasticity across sites (all maternal line/sampling site by experimental condition interactions p&gt;0.50) (<a id="xref-fig-10-1" class="xref-fig" href="#F10">Supplemental Fig. S6</a> and <a id="xref-table-wrap-5-1" class="xref-table" href="#T5">Table S5</a>).</p><div id="F10" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2017/10/09/200352/F10.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Interaction plots for chamber experiment Plots showing interactions between three seedling growth traits and the experimental conditions. Each line represents a maternal line, with color indicating the sampling site." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-864888468" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;span xmlns=&quot;http://www.w3.org/1999/xhtml&quot; class=&quot;caption-title&quot;&gt;Interaction plots for chamber experiment&lt;/span&gt; Plots showing interactions between three seedling growth traits and the experimental conditions. Each line represents a maternal line, with color indicating the sampling site.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure S6." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2017/10/09/200352/F10.medium.gif" width="310" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure S6." src="https://www.biorxiv.org/content/biorxiv/early/2017/10/09/200352/F10.medium.gif" width="310" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2017/10/09/200352/F10.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure S6." data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2017/10/09/200352/F10.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure S6.</span> <span class="caption-title">Interaction plots for chamber experiment</span><p id="p-63" class="first-child">Plots showing interactions between three seedling growth traits and the experimental conditions. Each line represents a maternal line, with color indicating the sampling site.</p><div class="sb-div caption-clear"></div></div></div><div id="T5" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/368861/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup"><a href="/highwire/markup/368861/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/368861" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table S5:</span> <p id="p-71" class="first-child">P-values of interaction term in linear model for chamber experiment response variable</p><div class="sb-div caption-clear"></div></div></div><p id="p-27">We then conducted an outdoor drought experiment using a subset of seedlings from the temperature experiment. We analyzed 146 seedlings representing 20 maternal lines from five sampling sites. These seedlings were grouped into 73 pairs, with one of each pair assigned to each treatment—well watered versus drought. We analyzed variation in four response variables: stomatal conductance, leaf length to width ratio, relative chlorophyll content (SPAD index), and specific leaf area (SLA, leaf area divided by dry mass).</p><p id="p-28">The droughted seedlings had significantly lower stomatal conductance rates than the well watered ones, indicating that the seedlings were stressed (p&lt;0.00001) (<a id="xref-table-wrap-6-1" class="xref-table" href="#T6">Supplemental Table S6</a>). Treatment explained most of the variation in stomatal conductance (62.3%), while maternal line and sampling site explained only a small amount of variation (5.8% and 0.9% respectively) (<a id="xref-fig-11-1" class="xref-fig" href="#F11">Supplemental Fig. S7</a>). For the remaining three response variables (leaf length to width ratio, SPAD, and SLA), much of the variation was unexplained (40.5%-70%). Experimental condition was not statistically significant and explained little to no variation (0.0-4.4%) (<a id="xref-fig-11-2" class="xref-fig" href="#F11">Supplemental Fig. S7</a> and <a id="xref-table-wrap-6-2" class="xref-table" href="#T6">Table S6</a>). Sampling site and maternal line were statistically significant in the linear models at the p=0.05 level and explained some variation (6.7-21.2%) (<a id="xref-fig-11-3" class="xref-fig" href="#F11">Supplemental Fig. S7</a> and <a id="xref-table-wrap-6-3" class="xref-table" href="#T6">Table S6</a>). Smaller, thicker leaves, and thus lower SLA values, were expected for droughted seedlings and for seedlings grown from seed collected from drier areas. Consistent with this expectation, the seedlings subjected to drought conditions showed lower SLA values. However, seedlings from drier sampling sites (D and T3) showed higher SLA values than more mesic sites (B, G, and 11), contrary to expectation (<a id="xref-fig-11-4" class="xref-fig" href="#F11">Supplemental Fig. S7</a>). None of the four response variables showed significant variation in phenotypic plasticity across sites (all maternal line/sampling site by experimental condition interactions p&gt;0.13) (<a id="xref-fig-12-1" class="xref-fig" href="#F12">Supplemental Fig. S8</a> and <a id="xref-table-wrap-7-1" class="xref-table" href="#T7">Table S7</a>).</p><div id="F11" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2017/10/09/200352/F11.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Variation in leaf traits in drought experiment Box plots showing variation between water treatments (left) and sampling sites (right) for four leaf traits." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-864888468" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;span xmlns=&quot;http://www.w3.org/1999/xhtml&quot; class=&quot;caption-title&quot;&gt;Variation in leaf traits in drought experiment&lt;/span&gt; Box plots showing variation between water treatments (left) and sampling sites (right) for four leaf traits.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure S7." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2017/10/09/200352/F11.medium.gif" width="249" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure S7." src="https://www.biorxiv.org/content/biorxiv/early/2017/10/09/200352/F11.medium.gif" width="249" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2017/10/09/200352/F11.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure S7." data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2017/10/09/200352/F11.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure S7.</span> <span class="caption-title">Variation in leaf traits in drought experiment</span><p id="p-64" class="first-child">Box plots showing variation between water treatments (left) and sampling sites (right) for four leaf traits.</p><div class="sb-div caption-clear"></div></div></div><div id="F12" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2017/10/09/200352/F12.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Interaction plots for drought experiment Plots showing interactions between the four leaf traits and the water treatment. Each line represents a maternal line, with color indicating the sampling site." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-864888468" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;span xmlns=&quot;http://www.w3.org/1999/xhtml&quot; class=&quot;caption-title&quot;&gt;Interaction plots for drought experiment&lt;/span&gt; Plots showing interactions between the four leaf traits and the water treatment. Each line represents a maternal line, with color indicating the sampling site.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure S8." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2017/10/09/200352/F12.medium.gif" width="258" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure S8." src="https://www.biorxiv.org/content/biorxiv/early/2017/10/09/200352/F12.medium.gif" width="258" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2017/10/09/200352/F12.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure S8." data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2017/10/09/200352/F12.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure S8.</span> <span class="caption-title">Interaction plots for drought experiment</span><p id="p-65" class="first-child">Plots showing interactions between the four leaf traits and the water treatment. Each line represents a maternal line, with color indicating the sampling site.</p><div class="sb-div caption-clear"></div></div></div><div id="T6" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/368770/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup"><a href="/highwire/markup/368770/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/368770" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table S6:</span> <p id="p-72" class="first-child">Percent of variation explained and p-values for non-interaction linear models for drought experiment</p><div class="sb-div caption-clear"></div></div></div><div id="T7" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/368793/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup"><a href="/highwire/markup/368793/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/368793" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table S7:</span> <p id="p-73" class="first-child">P-values of interaction term in linear model for drought experiment</p><div class="sb-div caption-clear"></div></div></div><p id="p-29">In addition to measuring seedling growth traits, we also examined the shape of the leaves from the drought experiment seedlings. We noted substantial variation in leaf shape, both among sites and within (<a id="xref-fig-13-1" class="xref-fig" href="#F13">Supplemental Fig. S9</a>). The remarkable amount of phenotypic variation in the seedlings is consistent with the high levels of genomic variation measured both among sites and within sites.</p><div id="F13" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2017/10/09/200352/F13.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Variation in leaf shape One representative leaf from each maternal line in the drought experiment. Each row shows a single sampling site, identified by site ID and state location (ACT=Australian Capital Territory, VIC=Victoria, NSW=New South Wales). Each leaf is identified by its sampling site, maternal line, and replicate number)." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-864888468" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;span xmlns=&quot;http://www.w3.org/1999/xhtml&quot; class=&quot;caption-title&quot;&gt;Variation in leaf shape&lt;/span&gt; One representative leaf from each maternal line in the drought experiment. Each row shows a single sampling site, identified by site ID and state location (ACT=Australian Capital Territory, VIC=Victoria, NSW=New South Wales). Each leaf is identified by its sampling site, maternal line, and replicate number).&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure S9." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2017/10/09/200352/F13.medium.gif" width="419" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure S9." src="https://www.biorxiv.org/content/biorxiv/early/2017/10/09/200352/F13.medium.gif" width="419" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2017/10/09/200352/F13.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure S9." data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2017/10/09/200352/F13.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure S9.</span> <span class="caption-title">Variation in leaf shape</span><p id="p-66" class="first-child">One representative leaf from each maternal line in the drought experiment. Each row shows a single sampling site, identified by site ID and state location (ACT=Australian Capital Territory, VIC=Victoria, NSW=New South Wales). Each leaf is identified by its sampling site, maternal line, and replicate number).</p><div class="sb-div caption-clear"></div></div></div></div></div><div class="section" id="sec-6"><h2 class="">Discussion</h2><p id="p-30"><em>Eucalyptus melliodora</em> is a foundation species in a critically endangered woodland community that now occupies a fraction of its former distribution and is the subject of restoration projects across its native range. Our examination of the distribution of genomic and phenotypic variation across the range of this species provides valuable information for sourcing seed for restoration, including empirically defining local provenances and matching genotypes to predicted future environmental conditions.</p><p id="p-31">Examining the relationship between genomic and geographic distance, we are able to empirically define "local" in this species to be on the order of 500 km, which is substantially farther than the current practice. This new definition encourages restoration projects to source seed more broadly across the landscape. In a highly fragmented landscape this will increase the number of potential source sites, potentially enabling the collection of higher quality seed with increased genetic diversity (<a id="xref-ref-10-5" class="xref-bibr" href="#ref-10">Broadhurst et al., 2008</a>). Incorporating more naturally occurring genomic variation can increase the adaptive potential of the restored population by providing the substrate for adaptation to rapidly changing environmental conditions.</p><p id="p-32">By modelling genomic variation across the landscape, we can understand the environmental factors that shape patterns of genomic variation and identify variation suitable for predicted future climates. We found several environmental variables that have played a role in the structure of genomic variation across the landscape. Of these, the climate variables are predicted to change rapidly over time. Change in soil nitrogen content might occur over longer time scales, but it is difficult to forecast due to complex biotic feedbacks (<a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">Brevik, 2013</a>). This suggests that optimal seed sourcing will need to balance the tracking of rapidly changing climate variables with the need to account for variables that are more stable due to their dependence on stable features of geology, topography, or hydrology. This also highlights an important concern that key environmental variables may become uncoupled, resulting in less than ideal conditions for this species across the landscape.</p><p id="p-33">Our analyses of phenotypic variation found no site-specific variation in phenotypic plasticity that would enable us to identify provenances better able to cope with rapid environmental change. However, plasticity is trait specific and traits that are hypothesized to be important for establishment and survival should continue to be investigated because they may provide valuable information for restoration projects. Importantly, our growth experiments support the results of the genomic analyses, showing the remarkable extent of variation both among sites and within sites, further supporting our recommendation to source seed to incorporate the high level of variation that occur naturally in <em>E. melliodora</em>.</p><p id="p-34">The results of this study are promising for the future of <em>E. melliodora</em> across its native distribution. We found high genomic and phenotypic diversity within sites, as well as shared across the range. This naturally occurring variation can provide a basis for adaptation to a rapidly changing environment and it should be incorporated into restoration projects through strategic seed sourcing. It is important to note that our genomic analyses were based on mature trees that predate extensive land clearing for agriculture. The same analyses in seedlings or saplings at these sites may show different results, although our phenotypic studies using seedlings produced concordant results. It remains to be determined if human modifications of the landscape have disrupted historical patterns of gene flow, resulting in more fragmented and inbred populations.</p><p id="p-35">Our landscape genomic model can guide seed selection by empirically defining local provenances and identifying variation suitable for predicted future climates. This understanding of the relationship between environmental and genomic variation can be combined with other types of information, such as basic biological knowledge of the ecological community and best agronomic practices in restoration, to establish foundation species and ecosystems with the highest probability of success in a rapidly changing environment.</p></div><div class="section" id="sec-7"><h2 class="">Methods</h2><div id="sec-8" class="subsection"><h3>Sample Collection</h3><p id="p-36">We obtained <em>E. melliodora</em> leaf samples from mature trees at 38 sites across the species’ range through a community science project described in Broadhurst et al. (<em>in review</em>) (<a id="xref-table-wrap-1-2" class="xref-table" href="#T1">Supplemental Table S1</a>). From each site, a citizen scientist collected leaf samples from up to 30 trees, put the samples in silica gel for drying, and shipped them to CSIRO for processing. In addition to leaf material, they also collected seeds from the sampled trees when available. We sampled an additional seven trees planted at a single site in Western Australia, well outside the species’ natural distribution.</p></div><div id="sec-9" class="subsection"><h3>Genotyping by Sequencing</h3><p id="p-37">We selected 3 to 10 trees per sampling site for sequencing and we processed each of the seven trees from Western Australia twice, using different leaves from the same tree, to serve as technical replicates. No power analysis was used to determine sample size during the design of the study. Sample size was determined based on our experience and judgment, with consideration of the availability of samples. We sequenced these 379 samples using a modified Genotyping-By-Sequencing (GBS) protocol (<a id="xref-ref-19-2" class="xref-bibr" href="#ref-19">Elshire et al., 2011</a>). Briefly, we extracted genomic DNA from approximately 50 mg of leaf tissue using the Qiagen DNeasy Plant 96 Kit, digested with PstI for genome complexity reduction, and ligated with a uniquely barcoded sequencing adapter pair. We then individually PCR amplified each sample to avoid sample bias. We pooled samples in equimolar concentrations and extracted library amplicons between 350 and 600 bp from an agarose gel. We sequenced the library pool on an Illumina HiSeq2500 using a 101- bp paired-end protocol at the Biomolecular Resource Facility at the Australian National University, generating almost 260 million read pairs.</p><p id="p-38">We checked the quality of the raw sequencing reads with FastQC (v0.10.1, (<a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">Andrews, 2012</a>)). We used AXE (v0.2.6, (<a id="xref-ref-32-1" class="xref-bibr" href="#ref-32">Murray &amp; Borevitz, 2017a</a>)) to demultiplex the sequencing reads according to each sample’s unique combinatorial barcode and were unable to assign 11% of read pairs to a sample. We used trimit from libqcpp (v0.2.5, (<a id="xref-ref-33-1" class="xref-bibr" href="#ref-33">Murray &amp; Borevitz, 2017b</a>)) to clean the reads for each sample, using default parameters, except q=20. This involved removing adapter contamination due to read-through of small fragments (20% of read pairs) and merging overlapping pairs (49% of read pairs), both steps using algorithms based on a global alignment of read pairs. We also used trimit for sliding window quality trimming (11% of reads). We then used a custom script to remove sequencing reads that did not begin with the expected restriction site sequence (16% of reads). We aligned sequencing reads to the <em>E. grandis</em> reference genome (v2.0, (<a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">Bartholomé et al., 2015</a>; <a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">JGI, 2015</a>; <a id="xref-ref-34-1" class="xref-bibr" href="#ref-34">Myburg et al., 2014</a>)), including all nuclear, chloroplast, mitochondrial, and ribosomal scaffolds, but used only nuclear scaffolds for downstream analyses. We aligned reads using bwa-mem (v0.7.5a-r405, (<a id="xref-ref-27-1" class="xref-bibr" href="#ref-27">Li, 2013</a>)), as paired reads (-p) and treating shorter split hits as secondary alignments (-M), with 88% of reads successfully mapped. We used GATK’s HaplotypeCaller in GVCF mode (v3.6-0-g89b7209, (<a id="xref-ref-30-1" class="xref-bibr" href="#ref-30">McKenna et al., 2010</a>)) to call variants for each sample with heterozygosity (-hets) increased to 0.005, indel heterozygosity (-indelHeterozygosity) increased to 0.0005, and the minimum number of reads sharing the same alignment start (-minReadsPerAlignStart) decreased to 4.</p><p id="p-39">We used GATK’s GenotypeGVCFs (v3.6-0-g89b7209, (<a id="xref-ref-30-2" class="xref-bibr" href="#ref-30">McKenna et al., 2010</a>)) for a preliminary round of joint genotyping across all samples based on the individual variant calls and again increasing the heterozygosity (-hets) to 0.005 and the indel heterozygosity (-indelHeterozygosity) to 0.0005. For basic filtering, we used GATK to remove variants that were indels, had no variation relative to the reference, were non-biallelic SNPs, had QD&lt;2.0 ("variant call confidence normalized by depth of sample reads supporting a variant"), MQ&gt;40.0 ("Root Mean Square of the mapping quality of reads across all samples"), or MQRankSum&lt;-12.5 ("Rank Sum Test for mapping qualities of REF versus ALT reads"). We removed samples with more than 60% missing data and SNPs with more than 80% missing data. We examined the genomic distance between samples to verify that technical replicates clustered closely together. We used a preliminary PCA, based on genomic distance between samples, to identify outlier samples. We removed outlier samples and poorly sequenced samples from the dataset for final genotyping and all downstream analyses.</p><p id="p-40">We reran GATK’s joint genotyping on the final sample set. We again used GATK to remove variants that were indels, SNPs with no variation relative to the reference, and non-biallelic SNPs. We determined final filtering thresholds by examining parameter distributions. A locus was retained for subsequent analysis if ExcessHet&lt;13.0 ("phred-scaled p-value for exact test of excess heterozygosity"),-0.3&lt;InbreedingCoeff&lt;0.3("likelihood-based test for the inbreeding among samples"), MQ&gt;15.0 ("Root Mean Square of the mapping quality of reads across all samples"),-10.0&lt;MQRankSum&lt;10.0 ("Rank Sum Test for mapping qualities of REF versus ALT reads"), and QD&gt;8.0 ("variant call confidence normalized by depth of sample reads supporting a variant"). We ran a second preliminary PCA analysis to identify additional outlier samples. Finally, we used VCFtools (v0.1.12b, (<a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">Danecek et al., 2011</a>)) to remove SNPs with greater than 60% missing data and thin the SNPs so that none were closer than 300 bp.</p></div><div id="sec-10" class="subsection"><h3>Genomic Analyses</h3><p id="p-41">To examine the genomic structure of <em>E. melliodora</em> and how it is influenced by geography, we conducted individual-based analyses. For these analyses, we converted the final genotypic data (a vcf file) to a sample-by-SNP matrix and imported it into a genind object (R adegenet v2.0.1, (<a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">Jombart, 2008</a>)). We calculated the pairwise genomic distances between individuals using a euclidean distance in <em>dist</em> (R stats v3.1.2, (<a id="xref-ref-39-1" class="xref-bibr" href="#ref-39">R Core Team, 2015</a>)). To visualize the genomic distance among samples, we ran a PCA using <em>dudi.pco</em> (R ade4 v1.7-4, (<a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">Dray &amp; Dufour, 2007</a>)). We plotted the first two PCA components, with samples colored in a rainbow gradient based on sample latitude. We calculated the geographic distance between samples based on their GPS coordinates using <em>earth.dist</em> (R fossil v0.3.7, (<a id="xref-ref-43-1" class="xref-bibr" href="#ref-43">Vavrek, 2011</a>)). We used a <em>mantel</em> test (R vegan v2.4-0, (<a id="xref-ref-36-1" class="xref-bibr" href="#ref-36">Oksanen et al., 2016</a>)) to quantify the linear relationship between the genomic distance between individuals and the natural log of geographic distance.</p><p id="p-42">To examine the role that environmental factors play in driving the genomic structure across the landscape, we used Generalized Dissimilarity Modelling (GDM), which uses matrix regression to estimate the non-linear relationship between genomic distance and environmental distance (<a id="xref-ref-20-2" class="xref-bibr" href="#ref-20">Ferrier et al., 2007</a>; <a id="xref-ref-21-2" class="xref-bibr" href="#ref-21">Fitzpatrick &amp; Keller, 2015</a>; <a id="xref-ref-41-2" class="xref-bibr" href="#ref-41">Thomassen et al., 2011</a>). We then used this model to predict the distribution of genomic variation across the landscape under current environmental conditions, as well as predicted future conditions. We obtained environmental data for the GDM from climate, elevation, soil, and landscape raster layers. Climate variables included 19 bioclimatic variables for the current time period (1960-1990), at 30 arc second resolution (<a id="xref-ref-48-1" class="xref-bibr" href="#ref-48">WorldClim, 2016b</a>). Elevation was from a digital elevation model aggregated from 90 m resolution (<a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">CGIAR-CSI, 2016</a>). Soil data included available water capacity, total nitrogen, and total phosphorus sampled at the surface (0-5 cm) and at depth (100-200 cm) (<a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">CSIRO, 2016</a>). Landscape data included the Prescott Index (a measure of water balance) and MrVBF (a topographic index) (<a id="xref-ref-14-2" class="xref-bibr" href="#ref-14">CSIRO, 2016</a>). For future prediction, we used the 19 bioclimatic variables predicted for 2070 at 30 arc second resolution based on GCM MIROC5 for representative concentration pathway 8.5 (<a id="xref-ref-47-1" class="xref-bibr" href="#ref-47">WorldClim, 2016a</a>), which is a greenhouse gas concentration trajectory showing continual increase in emissions over time. We determined the values for each variable at each sampling site based on GPS coordinates and used those values to calculate the environmental distances between sites.</p><p id="p-43">To determine the genomic distances between sampling sites, we used the sample by SNP matrix to calculate pairwise F<sub>st</sub> (<a id="xref-ref-45-1" class="xref-bibr" href="#ref-45">Weir &amp; Cockerham, 1984</a>) using <em>pairwise.WCfst</em> (R hierfstat v0.04-22, (<a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">Goudet &amp; Jombart, 2015</a>)). We ran a sampling site level PCA on the pairwise F<sub>st</sub> matrix using <em>dudi.pco</em> (R ade4 v1.7-4, (<a id="xref-ref-18-2" class="xref-bibr" href="#ref-18">Dray &amp; Dufour, 2007</a>)) and calculated the percent of variation explained for each PCA axis. For the GDM, we scaled the F<sub>st</sub> matrix to between 0 and 1 by subtracting the minimum value and then dividing by the maximum value. We generated the GDM model using <em>gdm</em> (R gdm v1.2.3, (<a id="xref-ref-29-1" class="xref-bibr" href="#ref-29">Manion et al., 2016</a>)) with the scaled F<sub>st</sub> matrix, geographic distance between sites, and environmental distances for the 28 variables for the current time period. Initially, we generated a GDM model for each environmental variable separately and excluded variables from further analysis if the deviance explained by the model was less than 5%. For the remaining variables, we calculated Pearson’s correlation for site values between pairwise sets of variables. If a pair of variables had a correlation greater than 60%, we excluded the variable with the lowest explanatory power from subsequent analysis. We conducted permutation testing using gdm.varImp (R gdm v1.2.3, (<a id="xref-ref-29-2" class="xref-bibr" href="#ref-29">Manion et al., 2016</a>)) with 1000 permutations to determine the explanatory power and statistical significance of the remaining variables and excluded additional inconsequential variables. We generated a final GDM model with the remaining environmental variables.</p><p id="p-44">We cross validated the GDM model using a random 70% of the spatial sampling sites as training data and the remaining 30% of sites as test data and ran 1000 resampled iterations. We used the GDM models from the training sites to predict the genomic dissimilarity between the test sites and used Pearson’s correlation to compare the predicted values to the observed values. To test the robustness of the geographic prediction from the GDM model, we visualized the geographic splines from 100 of these GDM models.</p><p id="p-45">To project the final GDM model onto the current environmental landscape, we first delineated the geographic extent of the analysis by defining an <em>E. melliodora</em> distribution polygon. We downloaded 14,977 <em>E. melliodora</em> occurrence records from the Atlas of Living Australia (<a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">ALA, 2016</a>), of which we removed 189 because they were well outside the expected distribution or were sparse records on the distribution margin. We generated the polygon using ahull (R alphahull v2.1, (<a id="xref-ref-37-1" class="xref-bibr" href="#ref-37">Pateiro-López &amp; Rodríguez-Casal, 2010</a>)), with alpha=15 and gBuffer (R rgeos v0.3-21, (<a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">Bivand &amp; Rundel, 2016</a>)), with a 20 km buffer. We then transformed the environmental rasters based on the model splines (<em>gdm.transform</em>), took a PCA of the transformed layers (<em>prcomp</em> R stats v3.1.2, (<a id="xref-ref-39-2" class="xref-bibr" href="#ref-39">R Core Team, 2015</a>)), and predicted across space (<em>predict</em>). We visualized the result by graphing the first three components of a PCA using a red-green-blue plot (<a id="xref-ref-21-3" class="xref-bibr" href="#ref-21">Fitzpatrick &amp; Keller, 2015</a>). We also projected the model onto a predicted future environmental landscape with the same procedure, except we substituted the current bioclimatic rasters with the future ones for 2070 that were predicted under a high CO2 emission scenario. We calculated the expected change in the distribution of genomic variation over time using the <em>predict</em> function with time=T.</p><p id="p-46">We examined the implications of the GDM model for seed sourcing decisions by selecting two hypothetical reforestation sites. We compared predicted future GDM values at these two hypothetical reforestation sites to current climate GDM values across the landscape of potential seed sources. This enabled us to generate a map of predicted genomic similarity of potential seed sources to the hypothetical reforestation sites under climate change.</p></div><div id="sec-11" class="subsection"><h3>Growth Experiments</h3><p id="p-47">To examine the effect of provenance and environment on phenotype, we conducted experiments in climate controlled growth chambers under two different climate regimes. No power analysis was used to determine sample size during the design of the experiment. Sample size was determined based on our experience and judgment, with consideration of the availability of seed and space in the growth chambers. We selected six sites (11, B, D, G, T1, T3) and six maternal trees per site that had sufficient seed (asterisks in <a id="xref-fig-1-4" class="xref-fig" href="#F1">Fig. 1A</a>). For each chamber, we grew eight or nine replicate seedlings from each maternal tree. To ensure we had a seedling for each intended replicate, four seeds were planted per pot (6.5 cm x 6.5 cm x 20 cm pots with soil that was 80% Martin’s mix and 20% sand). We germinated seeds in climate controlled chambers with 12 hours of light at 25°C and 12 hours of dark at 15°C. We set lights to mimic summer morning light (photosynthetic photon flux 370 nm=82, 400 nm=83, 420 nm=78, 450 nm=37, 530 nm=31, 620 nm=72, 660 nm=28, 735 nm=34, 850 nm=89, 6500 K=94 μmol/m<sup>2</sup>/s). We watered all seeds twice daily to keep the soil moist. We culled to one seedling per pot 12-14 days after planting.</p><p id="p-48">Three weeks after germination, we sorted seedlings into treatment chambers based on a randomized block design. Climate conditions were determined with SolarCalc (<a id="xref-ref-40-1" class="xref-bibr" href="#ref-40">Spokas &amp; Forcella, 2006</a>) to mimic average summer conditions (sampling site 11) and hotter conditions (5°C temperature increase; sampling site T3). We ran the experimental conditions for 12-14 weeks and took phenotypic measurements at five time points (1, 2, 3, 5, and 11 weeks after the experimental treatment began). Measurements included seedling height, number of leaves, and total leaf length.</p><p id="p-49">For the analysis of seedling height and total leaf length, we used the measurements at five weeks after the experimental treatment began and used only seedlings that were determined to be well established at that time. We also calculated a relative height increment for each seedling by determining the last measurement when the seedling had two or fewer leaves and the first measurement with eight or more leaves. The relative height increment is the difference between the natural log of the two height measures, divided by the difference in time.</p><p id="p-50">We investigated phenotypic plasticity by examining interaction plots between maternal line and experimental conditions for three response variables: seedling height, total leaf length, and relative height increment. We statistically tested for an interaction between sampling site/maternal line and experimental condition with linear mixed-effect models using <em>lmer</em> (R lme4 v1.1-10, (<a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">Bates et al., 2015</a>)) for each of the three response variables. Due to a lack of power to consider maternal line nested within sampling site, we ran two models for each response variable—one with maternal line and one with sampling site. These models included the experimental condition, sampling site or maternal line, and and their interactions as fixed effects. We included germination chamber and block as random effects. We identified outliers visually and ran the models with and without outliers to determine if they affected the results.</p><p id="p-51">We visualized the distribution of values for the three response variables across the six sampling sites using box plots. We quantified the distribution of phenotypic variation with linear mixed-effect models using <em>lmer</em> (R lme4 v1.1-10, (<a id="xref-ref-7-2" class="xref-bibr" href="#ref-7">Bates et al., 2015</a>)). For each of the three response variables, the model included maternal line nested within sampling site and experimental condition as main effects, with no interaction term, and germination chamber and block as random effects.</p><p id="p-52">After completion of the chamber experiment, we conducted an outdoor covered drought experiment on the 16 week old seedlings. No power analysis was used to determine sample size during the design of the experiment. Sample size was determined based on our experience and judgment, with consideration of the availability of space in the covered growth facility. We selected 160 seedlings from five sampling sites, with four maternal lines per site. We paired each seedling with a seedling of similar size from the same maternal line and treatment chamber. We randomly assigned each seedling of the pair to a different drought treatment group. We transplanted the seedlings to PVC tubes (9 cm diameter x 50 cm height with sand, perlite, and slow release osmocote) and kept them well watered for seven weeks, allowing them to acclimate to the outdoor conditions. Then we imposed two treatments: well watered and drought. For the well watered treatment, we watered the seedlings to saturation as needed (between three times per week and twice per day, depending on the weather). For the drought treatment, we watered as necessary to reach (but not exceed) 50% saturation.</p><p id="p-53">We measured leaf traits on each seedling three weeks after the initiation of treatment. We measured stomatal conductance with a porometer (SC-1 Leaf Porometer by Decagon Devices) and determined that water stress was induced in the droughted seedlings. We determined the leaf length to width ratio from a scan of the most recent fully expanded leaf from each seedling using image analysis software (WD3 WinDIAS Leaf Image Analysis System by Delta-T Devices). This leaf was initiated prior to the start of treatment, but expanded while under treatment conditions. We took additional measurements two months after the initiation of treatment. We used a chlorophyll meter (SPAD – 502 by Konica Minolta) to determine the SPAD index, which measures relative chlorophyll content; reduction in SPAD index would indicate detrimental effects of water limitation. We calculated specific leaf area (SLA, leaf area divided by dry mass) by scanning a single leaf from each seedling to determine the leaf area (WD3 WinDIAS Leaf Image Analysis System by Delta-T Devices) and weighed oven dried leaves. For analysis, we excluded data for seedlings that died during the experiment. We also excluded the experimental treatment pair of any dead seedlings.</p><p id="p-54">We visualized phenotypic plasticity by examining interaction plots between maternal line and experimental conditions for four response variables: stomatal conductance, leaf length to width ratio, SPAD index, and SLA. We statistically tested for an interaction between sampling site/maternal line and experimental condition with linear mixed-effect models using <em>lmer</em> (R lme4 v1.1-10, (<a id="xref-ref-7-3" class="xref-bibr" href="#ref-7">Bates et al., 2015</a>)) for each of the four response variables. Due to a lack of power to consider maternal line nested within sampling site, we ran two models for each response variable-one with maternal line and one with sampling site. These models included the experimental condition, sampling site or maternal line, and their interactions as fixed effects. We included block and sample pairings as random effects.</p><p id="p-55">We visualized the distribution of values for the four response variables across the five sampling sites using box plots. We quantified the distribution of phenotypic variation with linear mixed-effect models using <em>lmer</em> (R lme4 v1.1-10, (<a id="xref-ref-7-4" class="xref-bibr" href="#ref-7">Bates et al., 2015</a>)). For each of the four response variables, the model included maternal line nested within sampling site and experimental condition as main effects, with no interaction term, and block and sample pairings as random effects. Due to a lack of power, the p-value for the sampling site term was determined from a model without maternal line.</p></div></div><div class="section" id="sec-12"><h2 class="">Data Access</h2><p id="p-57">GBS sequencing reads are available at the NCBI Sequence Read Archive (SRA) (<a href="http://www.ncbi.nlm.nih.gov/sra">http://www.ncbi.nlm.nih.gov/sra</a>) under BioProject PRJNA413429. Growth experiment data and scripts for genomic and phenotypic analyses are available at <a href="https://github.com/LaMariposa/emelliodora">https://github.com/LaMariposa/emelliodora</a>.</p></div><div class="section ack" id="ack-1"><h2 class="">Acknowledgments</h2><p id="p-56">We thank the ANU Bioinformatics Consultancy for computational support and advice, the ANU Statistical Consulting Unit for statistical advice, the Centre for Biodiversity Analysis for advice on GDM modelling, and the ANU NCRIS Plant Growth Facility and RSB Plant Services for assistance with growth experiments.</p></div><div class="section ref-list" id="ref-list-1"><h2 class="">References</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference  in text" id="ref-1">↵</a><div class="cit ref-cit ref-journal" id="cit-200352v1.1" data-doi="10.1146/annurev-ecolsys-110512-135747"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Aitken</span>,  <span class="cit-name-given-names">S. N.</span></span>, &amp; <span class="cit-auth"><span class="cit-name-surname">Whitlock</span>,  <span class="cit-name-given-names">M. C.</span></span> (<span class="cit-pub-date">2013</span>). <span class="cit-article-title">Assisted Gene Flow to Facilitate Local Adaptation to Climate Change</span>. <abbr class="cit-jnl-abbrev">Annual Review of Ecology, Evolution, and Systematics</abbr>, <span class="cit-vol">44</span>(<span class="cit-issue">1</span>), <span class="cit-fpage">367</span>–<span class="cit-lpage">388</span>. <a href="https://doi.org/10.1146/annurev-ecolsys-110512-135747">https://doi.org/10.1146/annurev-ecolsys-110512-135747</a></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAnnual%2BReview%2Bof%2BEcology%252C%2BEvolution%252C%2Band%2BSystematics%26rft.volume%253D44%26rft.spage%253D367%26rft_id%253Dinfo%253Adoi%252F10.1146%252Fannurev-ecolsys-110512-135747%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1146/annurev-ecolsys-110512-135747&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000329821800018&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference  in text" id="ref-2">↵</a><div class="cit ref-cit ref-journal" id="cit-200352v1.2"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Aitken</span>,  <span class="cit-name-given-names">S. N.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Yeaman</span>,  <span class="cit-name-given-names">S.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Holliday</span>,  <span class="cit-name-given-names">J. A.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Wang</span>,  <span class="cit-name-given-names">T.</span></span>, &amp; <span class="cit-auth"><span class="cit-name-surname">Curtis-McLane</span>,  <span class="cit-name-given-names">S.</span></span> (<span class="cit-pub-date">2008</span>). <span class="cit-article-title">Adaptation, migration or extirpation: climate change outcomes for tree populations</span>. <abbr class="cit-jnl-abbrev">Evolutionary Applications</abbr>, <span class="cit-vol">1</span>(<span class="cit-issue">1</span>), <span class="cit-fpage">95</span>–<span class="cit-lpage">111</span>. <a href="https://doi.org/10.1111/j.1752-4571.2007.00013.x">https://doi.org/10.1111/j.1752-4571.2007.00013.x</a></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEvolutionary%2BApplications%26rft.volume%253D1%26rft.spage%253D95%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference  in text" id="ref-3">↵</a><div class="cit ref-cit ref-web" id="cit-200352v1.3"><div class="cit-metadata"><cite><span class="cit-auth cit-collab">ALA</span>. (<span class="cit-pub-date">2016</span>). <span class="cit-source">Atlas of Living Australia</span>. Retrieved November 24, 2016, from <a href="http://www.ala.org.au/">http://www.ala.org.au/</a></cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference  in text" id="ref-4">↵</a><div class="cit ref-cit ref-journal" id="cit-200352v1.4" data-doi="10.1093/aob/mcq034"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Andrew</span>,  <span class="cit-name-given-names">R. L.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Wallis</span>,  <span class="cit-name-given-names">I. R.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Harwood</span>,  <span class="cit-name-given-names">C. E.</span></span>, &amp; <span class="cit-auth"><span class="cit-name-surname">Foley</span>,  <span class="cit-name-given-names">W. J.</span></span> (<span class="cit-pub-date">2010</span>). <span class="cit-article-title">Genetic and environmental contributions to variation and population divergence in a broad-spectrum foliar defence of Eucalyptus tricarpa</span>. <abbr class="cit-jnl-abbrev">Annals of Botany</abbr>, <span class="cit-vol">105</span>(<span class="cit-issue">5</span>), <span class="cit-fpage">707</span>–<span class="cit-lpage">717</span>. <a href="https://doi.org/10.1093/aob/mcq034">https://doi.org/10.1093/aob/mcq034</a></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAnnals%2Bof%2BBotany%26rft_id%253Dinfo%253Adoi%252F10.1093%252Faob%252Fmcq034%26rft_id%253Dinfo%253Apmid%252F20228089%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/aob/mcq034&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20228089&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2017%2F10%2F09%2F200352.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000277729800006&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference  in text" id="ref-5">↵</a><div class="cit ref-cit ref-web" id="cit-200352v1.5"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Andrews</span>,  <span class="cit-name-given-names">S.</span></span> (<span class="cit-pub-date">2012</span>). <span class="cit-source">FastQC</span>. Retrieved from <a href="https://www.bioinformatics.babraham.ac.uk/projects/fastqc/">https://www.bioinformatics.babraham.ac.uk/projects/fastqc/</a></cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference  in text" id="ref-6">↵</a><div class="cit ref-cit ref-journal" id="cit-200352v1.6" data-doi="10.1111/nph.13150"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Bartholomé</span>,  <span class="cit-name-given-names">J.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Mandrou</span>,  <span class="cit-name-given-names">E.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Jenkins</span>,  <span class="cit-name-given-names">J.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Nabihoudine</span>,  <span class="cit-name-given-names">I.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Klopp</span>,  <span class="cit-name-given-names">C.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Schmutz</span>,  <span class="cit-name-given-names">J.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Plomion</span>,  <span class="cit-name-given-names">C.</span></span>, &amp; <span class="cit-auth"><span class="cit-name-surname">Gion</span>,  <span class="cit-name-given-names">J.-M.</span></span> (<span class="cit-pub-date">2015</span>). <span class="cit-article-title">High-resolution genetic maps of Eucalyptus improve Eucalyptus grandis genome assembly</span>. <abbr class="cit-jnl-abbrev">New Phytologist</abbr>, <span class="cit-vol">206</span>, <span class="cit-fpage">1283</span>–<span class="cit-lpage">1296</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNew%2BPhytologist%26rft.volume%253D206%26rft.spage%253D1283%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fnph.13150%26rft_id%253Dinfo%253Apmid%252F25385325%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/nph.13150&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25385325&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2017%2F10%2F09%2F200352.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference  in text" id="ref-7">↵</a><div class="cit ref-cit ref-journal" id="cit-200352v1.7"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Bates</span>,  <span class="cit-name-given-names">D.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Mächler</span>,  <span class="cit-name-given-names">M.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Bolker</span>,  <span class="cit-name-given-names">B.</span></span>, &amp; <span class="cit-auth"><span class="cit-name-surname">Walker</span>,  <span class="cit-name-given-names">S.</span></span> (<span class="cit-pub-date">2015</span>). <span class="cit-article-title">Fitting Linear Mixed-Effects Models Using lme4</span>. <abbr class="cit-jnl-abbrev">Journal of Statistical Software</abbr>, <span class="cit-vol">67</span>(<span class="cit-issue">1</span>). <a href="https://doi.org/10.18637/jss.v067.i01">https://doi.org/10.18637/jss.v067.i01</a></cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference  in text" id="ref-8">↵</a><div class="cit ref-cit ref-journal" id="cit-200352v1.8"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Bivand</span>,  <span class="cit-name-given-names">R.</span></span>, &amp; <span class="cit-auth"><span class="cit-name-surname">Rundel</span>,  <span class="cit-name-given-names">C.</span></span> (<span class="cit-pub-date">2016</span>). <span class="cit-article-title">rgeos: Interface to Geometry Engine - Open Source (GEOS</span>).</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference  in text" id="ref-9">↵</a><div class="cit ref-cit ref-journal" id="cit-200352v1.9"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Brevik</span>,  <span class="cit-name-given-names">E.</span></span> (<span class="cit-pub-date">2013</span>). <span class="cit-article-title">The Potential Impact of Climate Change on Soil Properties and Processes and Corresponding Influence on Food Security</span>. <abbr class="cit-jnl-abbrev">Agriculture</abbr>, <span class="cit-vol">3</span>(<span class="cit-issue">3</span>), <span class="cit-fpage">398</span>–<span class="cit-lpage">417</span>. <a href="https://doi.org/10.3390/agriculture3030398">https://doi.org/10.3390/agriculture3030398</a></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAgriculture%26rft.volume%253D3%26rft.spage%253D398%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference  in text" id="ref-10">↵</a><div class="cit ref-cit ref-journal" id="cit-200352v1.10"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Broadhurst</span>,  <span class="cit-name-given-names">L. M.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Lowe</span>,  <span class="cit-name-given-names">A.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Coates</span>,  <span class="cit-name-given-names">D. J.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Cunningham</span>,  <span class="cit-name-given-names">S. A.</span></span>, <span class="cit-auth"><span class="cit-name-surname">McDonald</span>,  <span class="cit-name-given-names">M.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Vesk</span>,  <span class="cit-name-given-names">P. A.</span></span>, &amp; <span class="cit-auth"><span class="cit-name-surname">Yates</span>,  <span class="cit-name-given-names">C.</span></span> (<span class="cit-pub-date">2008</span>). <span class="cit-article-title">Seed supply for broadscale restoration: Maximizing evolutionary potential</span>. <abbr class="cit-jnl-abbrev">Evolutionary Applications</abbr>, <span class="cit-vol">1</span>(<span class="cit-issue">4</span>), <span class="cit-fpage">587</span>–<span class="cit-lpage">597</span>. <a href="https://doi.org/10.1111/j.1752-4571.2008.00045.x">https://doi.org/10.1111/j.1752-4571.2008.00045.x</a></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEvolutionary%2BApplications%26rft.volume%253D1%26rft.spage%253D587%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-200352v1.11"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Broadhurst</span>,  <span class="cit-name-given-names">L. M.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Mellick</span>,  <span class="cit-name-given-names">R.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Knerr</span>,  <span class="cit-name-given-names">N.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Li</span>,  <span class="cit-name-given-names">L.</span></span>, &amp; <span class="cit-auth"><span class="cit-name-surname">Supple</span>,  <span class="cit-name-given-names">M. A.</span></span> (n.d.). <abbr class="cit-jnl-abbrev">Land availability may be more important than genetic diversity in the range shift response of the widely distributed eucalypt Eucalyptus melliodora (Yellow Box, Myrtaceae)</abbr>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference  in text" id="ref-12">↵</a><div class="cit ref-cit ref-journal" id="cit-200352v1.12"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Byrne</span>,  <span class="cit-name-given-names">M.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Prober</span>,  <span class="cit-name-given-names">S. M.</span></span>, <span class="cit-auth"><span class="cit-name-surname">McLean</span>,  <span class="cit-name-given-names">E. H.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Steane</span>,  <span class="cit-name-given-names">D. A.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Stock</span>,  <span class="cit-name-given-names">W. D.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Potts</span>,  <span class="cit-name-given-names">B. M.</span></span>, &amp; <span class="cit-auth"><span class="cit-name-surname">Vaillancourt</span>,  <span class="cit-name-given-names">R. E.</span></span> (<span class="cit-pub-date">2013</span>). <abbr class="cit-jnl-abbrev">Adaptation to climate in widespread eucalypt species</abbr>. Gold Coast: National Climate Change Adaptation Research Facility.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference  in text" id="ref-13">↵</a><div class="cit ref-cit ref-web" id="cit-200352v1.13"><div class="cit-metadata"><cite><span class="cit-auth cit-collab">CGIAR-CSI</span>. (<span class="cit-pub-date">2016</span>). <span class="cit-source">SRTM 90m Digital Elevation Data</span>. Retrieved September 5, 2016, from <a href="http://srtm.csi.cgiar.org/">http://srtm.csi.cgiar.org/</a></cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference  in text" id="ref-14">↵</a><div class="cit ref-cit ref-web" id="cit-200352v1.14"><div class="cit-metadata"><cite><span class="cit-auth cit-collab">CSIRO</span>. (<span class="cit-pub-date">2016</span>). <span class="cit-source">Soil and Landscape Grid of Australia</span>. Retrieved November 29, 2016, from <a href="http://www.clw.csiro.au/aclep/soilandlandscapegrid/index.html">http://www.clw.csiro.au/aclep/soilandlandscapegrid/index.html</a></cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference  in text" id="ref-15">↵</a><div class="cit ref-cit ref-journal" id="cit-200352v1.15" data-doi="10.1093/bioinformatics/btr330"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Danecek</span>,  <span class="cit-name-given-names">P.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Auton</span>,  <span class="cit-name-given-names">A.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Abecasis</span>,  <span class="cit-name-given-names">G.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Albers</span>,  <span class="cit-name-given-names">C. A.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Banks</span>,  <span class="cit-name-given-names">E.</span></span>, <span class="cit-auth"><span class="cit-name-surname">DePristo</span>,  <span class="cit-name-given-names">M. A.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Handsaker</span>,  <span class="cit-name-given-names">R. E.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Lunter</span>,  <span class="cit-name-given-names">G.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Marth</span>,  <span class="cit-name-given-names">G. T.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Sherry</span>,  <span class="cit-name-given-names">S. T.</span></span>, <span class="cit-auth"><span class="cit-name-surname">McVean</span>,  <span class="cit-name-given-names">G.</span></span>, &amp; <span class="cit-auth"><span class="cit-name-surname">Durbin</span>,  <span class="cit-name-given-names">R.</span></span> (<span class="cit-pub-date">2011</span>). <span class="cit-article-title">The variant call format and VCFtools</span>. <abbr class="cit-jnl-abbrev">Bioinformatics</abbr>, <span class="cit-vol">27</span>(<span class="cit-issue">15</span>), <span class="cit-fpage">2156</span>–<span class="cit-lpage">2158</span>. <a href="https://doi.org/10.1093/bioinformatics/btr330">https://doi.org/10.1093/bioinformatics/btr330</a></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtr330%26rft_id%253Dinfo%253Apmid%252F21653522%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/btr330&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21653522&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2017%2F10%2F09%2F200352.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000292778700023&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference  in text" id="ref-16">↵</a><div class="cit ref-cit ref-journal" id="cit-200352v1.16"><div class="cit-metadata"><cite><span class="cit-auth cit-collab">Department of Environment and Climate Change and Water</span>. (<span class="cit-pub-date">2011</span>). <span class="cit-article-title">National Recovery Plan for White Box - Yellow Box - Blakely’s Red Gum Grassy Woodland and Derived Native Grassland</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference  in text" id="ref-17">↵</a><div class="cit ref-cit ref-journal" id="cit-200352v1.17"><div class="cit-metadata"><cite><span class="cit-auth cit-collab">Department of the Environment and Heritage</span>. (<span class="cit-pub-date">2006</span>). <abbr class="cit-jnl-abbrev">EPBC Act Policy Statements: White Box - Yellow Box - Blakely’s Red Gum grassy woodlands and derived native grasslands.</abbr></cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference  in text" id="ref-18">↵</a><div class="cit ref-cit ref-journal" id="cit-200352v1.18"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Dray</span>,  <span class="cit-name-given-names">S.</span></span>, &amp; <span class="cit-auth"><span class="cit-name-surname">Dufour</span>,  <span class="cit-name-given-names">A.-B.</span></span> (<span class="cit-pub-date">2007</span>). <span class="cit-article-title">The ade4 Package: Implementing the Duality Diagram for Ecologists</span>. <abbr class="cit-jnl-abbrev">Journal of Statistical Software</abbr>, <span class="cit-vol">22</span>(<span class="cit-issue">4</span>). <a href="https://doi.org/10.18637/jss.v022.i04">https://doi.org/10.18637/jss.v022.i04</a></cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference  in text" id="ref-19">↵</a><div class="cit ref-cit ref-journal" id="cit-200352v1.19" data-doi="10.1371/journal.pone.0019379"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Elshire</span>,  <span class="cit-name-given-names">R. J.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Glaubitz</span>,  <span class="cit-name-given-names">J. C.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Sun</span>,  <span class="cit-name-given-names">Q.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Poland</span>,  <span class="cit-name-given-names">J. A.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Kawamoto</span>,  <span class="cit-name-given-names">K.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Buckler</span>,  <span class="cit-name-given-names">E. S.</span></span>, &amp; <span class="cit-auth"><span class="cit-name-surname">Mitchell</span>,  <span class="cit-name-given-names">S. E.</span></span> (<span class="cit-pub-date">2011</span>). <span class="cit-article-title">A robust, simple genotyping-by-sequencing (GBS) approach for high diversity species</span>. <abbr class="cit-jnl-abbrev">PloS One</abbr>, <span class="cit-vol">6</span>(<span class="cit-issue">5</span>), <span class="cit-fpage">e19379</span>. <a href="https://doi.org/10.1371/journal.pone.0019379">https://doi.org/10.1371/journal.pone.0019379</a></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BONE%26rft.aulast%253DElshire%26rft.auinit1%253DR.%2BJ.%26rft.volume%253D6%26rft.issue%253D5%26rft.spage%253De19379%26rft.epage%253De19379%26rft.atitle%253DA%2Brobust%252C%2Bsimple%2Bgenotyping-by-sequencing%2B%2528GBS%2529%2Bapproach%2Bfor%2Bhigh%2Bdiversity%2Bspecies.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0019379%26rft_id%253Dinfo%253Apmid%252F21573248%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0019379&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21573248&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2017%2F10%2F09%2F200352.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference  in text" id="ref-20">↵</a><div class="cit ref-cit ref-journal" id="cit-200352v1.20"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Ferrier</span>,  <span class="cit-name-given-names">S.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Manion</span>,  <span class="cit-name-given-names">G.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Elith</span>,  <span class="cit-name-given-names">J.</span></span>, &amp; <span class="cit-auth"><span class="cit-name-surname">Richardson</span>,  <span class="cit-name-given-names">K.</span></span> (<span class="cit-pub-date">2007</span>). <span class="cit-article-title">Using generalized dissimilarity modelling to analyse and predict patterns of beta diversity in regional biodiversity assessment</span>. <abbr class="cit-jnl-abbrev">Diversity and Distributions</abbr>, <span class="cit-vol">13</span>(<span class="cit-issue">3</span>), <span class="cit-fpage">252</span>–<span class="cit-lpage">264</span>. <a href="https://doi.org/10.1111/j.1472-4642.2007.00341.x">https://doi.org/10.1111/j.1472-4642.2007.00341.x</a></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DDiversity%2Band%2BDistributions%26rft.volume%253D13%26rft.spage%253D252%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference  in text" id="ref-21">↵</a><div class="cit ref-cit ref-journal" id="cit-200352v1.21" data-doi="10.1111/ele.12376"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Fitzpatrick</span>,  <span class="cit-name-given-names">M. C.</span></span>, &amp; <span class="cit-auth"><span class="cit-name-surname">Keller</span>,  <span class="cit-name-given-names">S. R.</span></span> (<span class="cit-pub-date">2015</span>). <span class="cit-article-title">Ecological genomics meets community-level modelling of biodiversity: Mapping the genomic landscape of current and future environmental adaptation</span>. <abbr class="cit-jnl-abbrev">Ecology Letters</abbr>, <span class="cit-vol">18</span>(<span class="cit-issue">1</span>), <span class="cit-fpage">1</span>–<span class="cit-lpage">16</span>. <a href="https://doi.org/10.1111/ele.12376">https://doi.org/10.1111/ele.12376</a></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEcology%2BLetters%26rft.volume%253D18%26rft.spage%253D1%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fele.12376%26rft_id%253Dinfo%253Apmid%252F25270536%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/ele.12376&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25270536&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2017%2F10%2F09%2F200352.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference  in text" id="ref-22">↵</a><div class="cit ref-cit ref-journal" id="cit-200352v1.22"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">González-Orozco</span>,  <span class="cit-name-given-names">C. E.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Pollock</span>,  <span class="cit-name-given-names">L. J.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Thornhill</span>,  <span class="cit-name-given-names">A. H.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Mishler</span>,  <span class="cit-name-given-names">B. D.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Knerr</span>,  <span class="cit-name-given-names">N.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Laffan</span>,  <span class="cit-name-given-names">S. W.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Miller</span>,  <span class="cit-name-given-names">J. T.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Rosauer</span>,  <span class="cit-name-given-names">D. F.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Faith</span>,  <span class="cit-name-given-names">D. P.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Nipperess</span>,  <span class="cit-name-given-names">D. A.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Kujala</span>,  <span class="cit-name-given-names">H.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Linke</span>,  <span class="cit-name-given-names">S.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Butt</span>,  <span class="cit-name-given-names">N.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Külheim</span>,  <span class="cit-name-given-names">C.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Crisp</span>,  <span class="cit-name-given-names">M. D.</span></span>, &amp; <span class="cit-auth"><span class="cit-name-surname">Gruber</span>,  <span class="cit-name-given-names">B.</span></span> (<span class="cit-pub-date">2016</span>). <span class="cit-article-title">Phylogenetic approaches reveal biodiversity threats under climate change</span>. <abbr class="cit-jnl-abbrev">Nature Climate Change</abbr>, <span class="cit-vol">1</span>(<span class="cit-issue">12</span>), <span class="cit-fpage">1110</span>–<span class="cit-lpage">1114</span>. <a href="https://doi.org/10.1038/nclimate3126">https://doi.org/10.1038/nclimate3126</a></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2BClimate%2BChange%26rft.volume%253D1%26rft.spage%253D1110%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference  in text" id="ref-23">↵</a><div class="cit ref-cit ref-journal" id="cit-200352v1.23"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Goudet</span>,  <span class="cit-name-given-names">J.</span></span>, &amp; <span class="cit-auth"><span class="cit-name-surname">Jombart</span>,  <span class="cit-name-given-names">T.</span></span> (<span class="cit-pub-date">2015</span>). <abbr class="cit-jnl-abbrev">hierfstat: Estimation and Tests of Hierarchical F- Statistics.</abbr></cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference  in text" id="ref-24">↵</a><div class="cit ref-cit ref-journal" id="cit-200352v1.24"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Hoffmann</span>,  <span class="cit-name-given-names">A.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Griffin</span>,  <span class="cit-name-given-names">P.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Dillon</span>,  <span class="cit-name-given-names">S.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Catullo</span>,  <span class="cit-name-given-names">R.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Rane</span>,  <span class="cit-name-given-names">R.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Byrne</span>,  <span class="cit-name-given-names">M.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Jordan</span>,  <span class="cit-name-given-names">R.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Oakeshott</span>,  <span class="cit-name-given-names">J.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Weeks</span>,  <span class="cit-name-given-names">A.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Joseph</span>,  <span class="cit-name-given-names">L.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Lockhart</span>,  <span class="cit-name-given-names">P.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Borevitz</span>,  <span class="cit-name-given-names">J.</span></span>, &amp; <span class="cit-auth"><span class="cit-name-surname">Sgrò</span>,  <span class="cit-name-given-names">C.</span></span> (<span class="cit-pub-date">2015</span>). <span class="cit-article-title">A framework for incorporating evolutionary genomics into biodiversity conservation and management</span>. <abbr class="cit-jnl-abbrev">Climate Change Responses</abbr>, <span class="cit-vol">2</span>(<span class="cit-issue">1</span>). <a href="https://doi.org/10.1186/s40665-014-0009-x">https://doi.org/10.1186/s40665-014-0009-x</a></cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference  in text" id="ref-25">↵</a><div class="cit ref-cit ref-web" id="cit-200352v1.25"><div class="cit-metadata"><cite><span class="cit-auth cit-collab">JGI</span>. (<span class="cit-pub-date">2015</span>). <span class="cit-source">Eucalyptus grandis v2.0 (Rose gum)</span>. Retrieved August 27, 2015, from <a href="https://phytozome.jgi.doe.gov/pz/portal.html#!info?alias=Org_Egrandis_er">https://phytozome.jgi.doe.gov/pz/portal.html#!info?alias=Org_Egrandis_er</a></cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference  in text" id="ref-26">↵</a><div class="cit ref-cit ref-journal" id="cit-200352v1.26" data-doi="10.1093/bioinformatics/btn129"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Jombart</span>,  <span class="cit-name-given-names">T.</span></span> (<span class="cit-pub-date">2008</span>). <span class="cit-article-title">Adegenet: A R package for the multivariate analysis of genetic markers</span>. <abbr class="cit-jnl-abbrev">Bioinformatics</abbr>, <span class="cit-vol">24</span>(<span class="cit-issue">11</span>), <span class="cit-fpage">1403</span>–<span class="cit-lpage">1405</span>. <a href="https://doi.org/10.1093/bioinformatics/btn129">https://doi.org/10.1093/bioinformatics/btn129</a></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtn129%26rft_id%253Dinfo%253Apmid%252F18397895%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/btn129&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18397895&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2017%2F10%2F09%2F200352.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000256169300015&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-27-1" title="View reference  in text" id="ref-27">↵</a><div class="cit ref-cit ref-journal" id="cit-200352v1.27"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Li</span>,  <span class="cit-name-given-names">H.</span></span> (<span class="cit-pub-date">2013</span>). <span class="cit-article-title">Aligning sequence reads, clone sequences and assembly contigs with BWA-MEM</span>. <abbr class="cit-jnl-abbrev">arXiv</abbr><span class="cit-pub-id-sep cit-pub-id-arxiv-sep">, </span><span class="cit-pub-id cit-pub-id-arxiv"><span class="cit-pub-id-scheme-arxiv">arxiv:</span>1303.3997v1</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-28-1" title="View reference  in text" id="ref-28">↵</a><div class="cit ref-cit ref-journal" id="cit-200352v1.28" data-doi="10.1016/j.biocon.2010.04.014"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Lindenmayer</span>,  <span class="cit-name-given-names">D. B.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Steffen</span>,  <span class="cit-name-given-names">W.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Burbidge</span>,  <span class="cit-name-given-names">A. A.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Hughes</span>,  <span class="cit-name-given-names">L.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Kitching</span>,  <span class="cit-name-given-names">R. L.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Musgrave</span>,  <span class="cit-name-given-names">W.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Stafford Smith</span>,  <span class="cit-name-given-names">M.</span></span>, &amp; <span class="cit-auth"><span class="cit-name-surname">Werner</span>,  <span class="cit-name-given-names">P. A.</span></span> (<span class="cit-pub-date">2010</span>). <span class="cit-article-title">Conservation strategies in response to rapid climate change: Australia as a case study</span>. <abbr class="cit-jnl-abbrev">Biological Conservation</abbr>, <span class="cit-vol">143</span>(<span class="cit-issue">7</span>), <span class="cit-fpage">1587</span>–<span class="cit-lpage">1593</span>. <a href="https://doi.org/10.1016/j.biocon.2010.04.014">https://doi.org/10.1016/j.biocon.2010.04.014</a></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBiological%2BConservation%26rft.volume%253D143%26rft.spage%253D1587%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.biocon.2010.04.014%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.biocon.2010.04.014&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000279413800003&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-29-1" title="View reference  in text" id="ref-29">↵</a><div class="cit ref-cit ref-journal" id="cit-200352v1.29"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Manion</span>,  <span class="cit-name-given-names">G.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Lisk</span>,  <span class="cit-name-given-names">M.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Ferrier</span>,  <span class="cit-name-given-names">S.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Nieto-Lugilde</span>,  <span class="cit-name-given-names">D.</span></span>, &amp; <span class="cit-auth"><span class="cit-name-surname">Fitzpatrick</span>,  <span class="cit-name-given-names">M. C.</span></span> (<span class="cit-pub-date">2016</span>). <span class="cit-article-title">gdm: Functions for Generalized Dissimilarity Modeling</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-30-1" title="View reference  in text" id="ref-30">↵</a><div class="cit ref-cit ref-journal" id="cit-200352v1.30" data-doi="10.1101/gr.107524.110"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">McKenna</span>,  <span class="cit-name-given-names">A.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Hanna</span>,  <span class="cit-name-given-names">M.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Banks</span>,  <span class="cit-name-given-names">E.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Sivachenko</span>,  <span class="cit-name-given-names">A.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Cibulskis</span>,  <span class="cit-name-given-names">K.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Kernytsky</span>,  <span class="cit-name-given-names">A.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Garimella</span>,  <span class="cit-name-given-names">K.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Altshuler</span>,  <span class="cit-name-given-names">D.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Gabriel</span>,  <span class="cit-name-given-names">S.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Daly</span>,  <span class="cit-name-given-names">M.</span></span>, &amp; <span class="cit-auth"><span class="cit-name-surname">DePristo</span>,  <span class="cit-name-given-names">M. A.</span></span> (<span class="cit-pub-date">2010</span>). <span class="cit-article-title">The Genome Analysis Toolkit: a MapReduce framework for analyzing next-generation DNA sequencing data</span>. <abbr class="cit-jnl-abbrev">Genome Research</abbr>, <span class="cit-vol">20</span>(<span class="cit-issue">9</span>), <span class="cit-fpage">1297</span>–<span class="cit-lpage">1303</span>. <a href="https://doi.org/10.1101/gr.107524.110">https://doi.org/10.1101/gr.107524.110</a></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BResearch%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.107524.110%26rft_id%253Dinfo%253Apmid%252F20644199%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjk6IjIwLzkvMTI5NyI7czo0OiJhdG9tIjtzOjM3OiIvYmlvcnhpdi9lYXJseS8yMDE3LzEwLzA5LzIwMDM1Mi5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-31-1" title="View reference  in text" id="ref-31">↵</a><div class="cit ref-cit ref-journal" id="cit-200352v1.31" data-doi="10.1111/pce.12251"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">McLean</span>,  <span class="cit-name-given-names">E. H.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Prober</span>,  <span class="cit-name-given-names">S. M.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Stock</span>,  <span class="cit-name-given-names">W. D.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Steane</span>,  <span class="cit-name-given-names">D. A.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Potts</span>,  <span class="cit-name-given-names">B. M.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Vaillancourt</span>,  <span class="cit-name-given-names">R. E.</span></span>, &amp; <span class="cit-auth"><span class="cit-name-surname">Byrne</span>,  <span class="cit-name-given-names">M.</span></span> (<span class="cit-pub-date">2014</span>). <span class="cit-article-title">Plasticity of functional traits varies clinally along a rainfall gradient in Eucalyptus tricarpa</span>. <abbr class="cit-jnl-abbrev">Plant, Cell and Environment</abbr>, <span class="cit-vol">37</span>(<span class="cit-issue">6</span>), <span class="cit-fpage">1440</span>–<span class="cit-lpage">1451</span>. <a href="https://doi.org/10.1111/pce.12251">https://doi.org/10.1111/pce.12251</a></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%252C%2BCell%2Band%2BEnvironment%26rft.volume%253D37%26rft.spage%253D1440%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fpce.12251%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/pce.12251&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000335451000016&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-32-1" title="View reference  in text" id="ref-32">↵</a><div class="cit ref-cit ref-journal" id="cit-200352v1.32"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Murray</span>,  <span class="cit-name-given-names">K. D.</span></span>, &amp; <span class="cit-auth"><span class="cit-name-surname">Borevitz</span>,  <span class="cit-name-given-names">J. O.</span></span> (<span class="cit-pub-date">2017a</span>). <span class="cit-article-title">Axe: rapid, competitive sequence read demultiplexing using a trie</span>. <abbr class="cit-jnl-abbrev">bioRxiv.</abbr> <a href="https://doi.org/10.1101/160606">https://doi.org/10.1101/160606</a></cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-33-1" title="View reference  in text" id="ref-33">↵</a><div class="cit ref-cit ref-journal" id="cit-200352v1.33"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Murray</span>,  <span class="cit-name-given-names">K. D.</span></span>, &amp; <span class="cit-auth"><span class="cit-name-surname">Borevitz</span>,  <span class="cit-name-given-names">J. O.</span></span> (<span class="cit-pub-date">2017b</span>). <span class="cit-article-title">libqcpp: A C++14 sequence quality control library</span>. <abbr class="cit-jnl-abbrev">The Journal of Open Source Software</abbr>, <span class="cit-vol">2</span>(<span class="cit-issue">13</span>). <a href="https://doi.org/10.21105/joss.00232">https://doi.org/10.21105/joss.00232</a></cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-34-1" title="View reference  in text" id="ref-34">↵</a><div class="cit ref-cit ref-journal" id="cit-200352v1.34"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Myburg</span>,  <span class="cit-name-given-names">A. A.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Grattapaglia</span>,  <span class="cit-name-given-names">D.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Tuskan</span>,  <span class="cit-name-given-names">G. A.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Hellsten</span>,  <span class="cit-name-given-names">U.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Hayes</span>,  <span class="cit-name-given-names">R. D.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Grimwood</span>,  <span class="cit-name-given-names">J.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Schmutz</span>,  <span class="cit-name-given-names">J.</span></span> (<span class="cit-pub-date">2014</span>). <span class="cit-article-title">The genome of Eucalyptus grandis</span>. <abbr class="cit-jnl-abbrev">Nature</abbr>, <span class="cit-vol">509</span>(<span class="cit-issue">7505</span>), <span class="cit-fpage">356</span>–<span class="cit-lpage">362</span>. <a href="https://doi.org/10.1038/nature13308">https://doi.org/10.1038/nature13308</a></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.volume%253D509%26rft.spage%253D356%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-35-1" title="View reference  in text" id="ref-35">↵</a><div class="cit ref-cit ref-journal" id="cit-200352v1.35" data-doi="10.1016/j.tplants.2010.09.008"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Nicotra</span>,  <span class="cit-name-given-names">A. B.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Atkin</span>,  <span class="cit-name-given-names">O. K.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Bonser</span>,  <span class="cit-name-given-names">S. P.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Davidson</span>,  <span class="cit-name-given-names">A. M.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Finnegan</span>,  <span class="cit-name-given-names">E. J.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Mathesius</span>,  <span class="cit-name-given-names">U.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Poot</span>,  <span class="cit-name-given-names">P.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Purugganan</span>,  <span class="cit-name-given-names">M. D.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Richards</span>,  <span class="cit-name-given-names">C. L.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Valladares</span>,  <span class="cit-name-given-names">F.</span></span>, &amp; <span class="cit-auth"><span class="cit-name-surname">van Kleunen</span>,  <span class="cit-name-given-names">M.</span></span> (<span class="cit-pub-date">2010</span>). <span class="cit-article-title">Plant phenotypic plasticity in a changing climate</span>. <abbr class="cit-jnl-abbrev">Trends in Plant Science</abbr>, <span class="cit-vol">15</span>(<span class="cit-issue">12</span>), <span class="cit-fpage">684</span>–<span class="cit-lpage">692</span>. <a href="https://doi.org/10.1016/j.tplants.2010.09.008">https://doi.org/10.1016/j.tplants.2010.09.008</a></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTrends%2Bin%2Bplant%2Bscience%26rft.stitle%253DTrends%2BPlant%2BSci%26rft.aulast%253DNicotra%26rft.auinit1%253DA.%2BB.%26rft.volume%253D15%26rft.issue%253D12%26rft.spage%253D684%26rft.epage%253D692%26rft.atitle%253DPlant%2Bphenotypic%2Bplasticity%2Bin%2Ba%2Bchanging%2Bclimate.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.tplants.2010.09.008%26rft_id%253Dinfo%253Apmid%252F20970368%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.tplants.2010.09.008&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20970368&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2017%2F10%2F09%2F200352.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000285905400005&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-36-1" title="View reference  in text" id="ref-36">↵</a><div class="cit ref-cit ref-journal" id="cit-200352v1.36"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Oksanen</span>,  <span class="cit-name-given-names">J.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Blanchet</span>,  <span class="cit-name-given-names">F. G.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Friendly</span>,  <span class="cit-name-given-names">M.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Kindt</span>,  <span class="cit-name-given-names">R.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Legendre</span>,  <span class="cit-name-given-names">P.</span></span>, <span class="cit-auth"><span class="cit-name-surname">McGlinn</span>,  <span class="cit-name-given-names">D.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Minchin</span>,  <span class="cit-name-given-names">P. R.</span></span>, <span class="cit-auth"><span class="cit-name-surname">O’Hara</span>,  <span class="cit-name-given-names">R. B.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Simpson</span>,  <span class="cit-name-given-names">G. L.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Solymos</span>,  <span class="cit-name-given-names">P.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Stevens</span>,  <span class="cit-name-given-names">M. H. H.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Szoecs</span>,  <span class="cit-name-given-names">E.</span></span>, &amp; <span class="cit-auth"><span class="cit-name-surname">Wagner</span>,  <span class="cit-name-given-names">H.</span></span> (<span class="cit-pub-date">2016</span>). <abbr class="cit-jnl-abbrev">vegan: Community Ecology Package.</abbr></cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-37-1" title="View reference  in text" id="ref-37">↵</a><div class="cit ref-cit ref-journal" id="cit-200352v1.37"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Pateiro-López</span>,  <span class="cit-name-given-names">B.</span></span>, &amp; <span class="cit-auth"><span class="cit-name-surname">Rodríguez-Casal</span>,  <span class="cit-name-given-names">A.</span></span> (<span class="cit-pub-date">2010</span>). <span class="cit-article-title">Generalizing the Convex Hull of a Sample: The R Package Alphahull</span>. <abbr class="cit-jnl-abbrev">Journal of Statistical Software</abbr>, <span class="cit-vol">34</span>(<span class="cit-issue">5</span>). <a href="https://doi.org/whttp://dx.doi.org/10.18637/jss.v034.i05">https://doi.org/whttp://dx.doi.org/10.18637/jss.v034.i05</a></cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-38-1" title="View reference  in text" id="ref-38">↵</a><div class="cit ref-cit ref-journal" id="cit-200352v1.38"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Prober</span>,  <span class="cit-name-given-names">S. M.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Byrne</span>,  <span class="cit-name-given-names">M.</span></span>, <span class="cit-auth"><span class="cit-name-surname">McLean</span>,  <span class="cit-name-given-names">E. H.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Steane</span>,  <span class="cit-name-given-names">D. A.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Potts</span>,  <span class="cit-name-given-names">B. M.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Vaillancourt</span>,  <span class="cit-name-given-names">R. E.</span></span>, &amp; <span class="cit-auth"><span class="cit-name-surname">Stock</span>,  <span class="cit-name-given-names">W. D.</span></span> (<span class="cit-pub-date">2015</span>). <span class="cit-article-title">Climate-adjusted provenancing: a strategy for climate-resilient ecological restoration</span>. <abbr class="cit-jnl-abbrev">Frontiers in Ecology and Evolution</abbr>, <span class="cit-vol">3</span>(<span class="cit-issue">65</span>). <a href="https://doi.org/10.3389/fevo.2015.00065">https://doi.org/10.3389/fevo.2015.00065</a></cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-39-1" title="View reference  in text" id="ref-39">↵</a><div class="cit ref-cit ref-journal" id="cit-200352v1.39"><div class="cit-metadata"><cite><span class="cit-auth cit-collab">R Core Team</span>. (<span class="cit-pub-date">2015</span>). <span class="cit-article-title">R: A language and environment for statistical computing. Vienna, Austria: R Foundation for Statistical Computing</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-40-1" title="View reference  in text" id="ref-40">↵</a><div class="cit ref-cit ref-journal" id="cit-200352v1.40"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Spokas</span>,  <span class="cit-name-given-names">K.</span></span>, &amp; <span class="cit-auth"><span class="cit-name-surname">Forcella</span>,  <span class="cit-name-given-names">F.</span></span> (<span class="cit-pub-date">2006</span>). <span class="cit-article-title">Estimating hourly incoming solar radiation from limited meteorological data</span>. <abbr class="cit-jnl-abbrev">Weed Science</abbr>, <span class="cit-vol">54</span>(<span class="cit-issue">1</span>), <span class="cit-fpage">182</span>–<span class="cit-lpage">189</span>. <a href="https://doi.org/10.1614/WS-05-098R.1">https://doi.org/10.1614/WS-05-098R.1</a></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DWeed%2BScience%26rft.volume%253D54%26rft.spage%253D182%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-41-1" title="View reference  in text" id="ref-41">↵</a><div class="cit ref-cit ref-journal" id="cit-200352v1.41"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Thomassen</span>,  <span class="cit-name-given-names">H. A.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Fuller</span>,  <span class="cit-name-given-names">T.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Buermann</span>,  <span class="cit-name-given-names">W.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Milá</span>,  <span class="cit-name-given-names">B.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Kieswetter</span>,  <span class="cit-name-given-names">C. M.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Jarrín-V.</span>,  <span class="cit-name-given-names">P.</span></span>, … <span class="cit-auth"><span class="cit-name-surname">Smith</span>,  <span class="cit-name-given-names">T. B.</span></span> (<span class="cit-pub-date">2011</span>). <span class="cit-article-title">Mapping evolutionary process: A multi-taxa approach to conservation prioritization</span>. <abbr class="cit-jnl-abbrev">Evolutionary Applications</abbr>, <span class="cit-vol">4</span>(<span class="cit-issue">2</span>), <span class="cit-fpage">397</span>–<span class="cit-lpage">413</span>. <a href="https://doi.org/10.1111/j.1752-4571.2010.00172.x">https://doi.org/10.1111/j.1752-4571.2010.00172.x</a></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEvolutionary%2BApplications%26rft.volume%253D4%26rft.spage%253D397%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-42-1" title="View reference  in text" id="ref-42">↵</a><div class="cit ref-cit ref-journal" id="cit-200352v1.42"><div class="cit-metadata"><cite><span class="cit-auth cit-collab">Threatened Species Scientific Committee</span>. (<span class="cit-pub-date">2006</span>). <abbr class="cit-jnl-abbrev">Advice to the Minister for the Environment and Heritage</abbr>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-43-1" title="View reference  in text" id="ref-43">↵</a><div class="cit ref-cit ref-journal" id="cit-200352v1.43"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Vavrek</span>,  <span class="cit-name-given-names">M. J.</span></span> (<span class="cit-pub-date">2011</span>). <span class="cit-article-title">Fossil: Palaeoecological and Paelaeogeogrpahical Analysis Tools</span>. <abbr class="cit-jnl-abbrev">Palaeontologia Electronica</abbr>, <span class="cit-vol">14</span>(<span class="cit-issue">1</span>), <span class="cit-fpage">1</span>–<span class="cit-lpage">16</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPalaeontologia%2BElectronica%26rft.volume%253D14%26rft.spage%253D1%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-44-1" title="View reference  in text" id="ref-44">↵</a><div class="cit ref-cit ref-journal" id="cit-200352v1.44" data-doi="10.1111/mec.12938"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Wang</span>,  <span class="cit-name-given-names">I. J.</span></span>, &amp; <span class="cit-auth"><span class="cit-name-surname">Bradburd</span>,  <span class="cit-name-given-names">G. S.</span></span> (<span class="cit-pub-date">2014</span>). <span class="cit-article-title">Isolation by environment</span>. <abbr class="cit-jnl-abbrev">Molecular Ecology</abbr>, <span class="cit-vol">23</span>(<span class="cit-issue">23</span>), <span class="cit-fpage">5649</span>–<span class="cit-lpage">5662</span>. <a href="https://doi.org/10.1111/mec.12938">https://doi.org/10.1111/mec.12938</a></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2BEcology%26rft.volume%253D23%26rft.spage%253D5649%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fmec.12938%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/mec.12938&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-45-1" title="View reference  in text" id="ref-45">↵</a><div class="cit ref-cit ref-journal" id="cit-200352v1.45" data-doi="10.2307/2408641"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Weir</span>,  <span class="cit-name-given-names">B. S.</span></span>, &amp; <span class="cit-auth"><span class="cit-name-surname">Cockerham</span>,  <span class="cit-name-given-names">C. C.</span></span> (<span class="cit-pub-date">1984</span>). <span class="cit-article-title">Estimating F-Statistics for the Analysis of Population Structure</span>. <abbr class="cit-jnl-abbrev">Evolution</abbr>, <span class="cit-vol">38</span>(<span class="cit-issue">6</span>), <span class="cit-fpage">1358</span>–<span class="cit-lpage">1370</span>. <a href="https://doi.org/10.2307/2409936">https://doi.org/10.2307/2409936</a></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEvolution%26rft.volume%253D38%26rft.spage%253D1358%26rft_id%253Dinfo%253Adoi%252F10.2307%252F2408641%26rft_id%253Dinfo%253Apmid%252F28563791%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2307/2408641&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=28563791&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2017%2F10%2F09%2F200352.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1984TY40400017&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-46-1" title="View reference  in text" id="ref-46">↵</a><div class="cit ref-cit ref-journal" id="cit-200352v1.46" data-doi="10.1016/j.tplants.2014.03.011"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Williams</span>,  <span class="cit-name-given-names">A. V</span></span>, <span class="cit-auth"><span class="cit-name-surname">Nevill</span>,  <span class="cit-name-given-names">P. G.</span></span>, &amp; <span class="cit-auth"><span class="cit-name-surname">Krauss</span>,  <span class="cit-name-given-names">S. L.</span></span> (<span class="cit-pub-date">2014</span>). <span class="cit-article-title">Next generation restoration genetics: Applications and opportunities</span>. <abbr class="cit-jnl-abbrev">Trends in Plant Science</abbr>, <span class="cit-vol">19</span>(<span class="cit-issue">8</span>), <span class="cit-fpage">529</span>–<span class="cit-lpage">537</span>. <a href="https://doi.org/10.1016/j.tplants.2014.03.011">https://doi.org/10.1016/j.tplants.2014.03.011</a></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTrends%2Bin%2BPlant%2BScience%26rft.volume%253D19%26rft.spage%253D529%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.tplants.2014.03.011%26rft_id%253Dinfo%253Apmid%252F24767982%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.tplants.2014.03.011&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24767982&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2017%2F10%2F09%2F200352.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000340341500009&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-47-1" title="View reference  in text" id="ref-47">↵</a><div class="cit ref-cit ref-web" id="cit-200352v1.47"><div class="cit-metadata"><cite><span class="cit-auth cit-collab">WorldClim</span>. (<span class="cit-pub-date">2016a</span>). <span class="cit-source">CMIP5 30-seconds</span>. Retrieved September 30, 2016, from <a href="http://www.worldclim.org/cmip5_30s">http://www.worldclim.org/cmip5_30s</a></cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-48-1" title="View reference  in text" id="ref-48">↵</a><div class="cit ref-cit ref-web" id="cit-200352v1.48"><div class="cit-metadata"><cite><span class="cit-auth cit-collab">WorldClim</span>. (<span class="cit-pub-date">2016b</span>). <span class="cit-source">WorldClim 1.4</span>. Retrieved November 28, 2016, from <a href="http://www.worldclim.org/current">http://www.worldclim.org/current</a></cite></div><div class="cit-extra"></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.biorxiv.org/content/10.1101/200352v1.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-disqus-comment" >
  
      
  
  <div class="pane-content">
    <div id="disqus_thread"><noscript><p><a href="http://biorxivstage.disqus.com/?url=https%3A%2F%2Fwww.biorxiv.org%2Fcontent%2F10.1101%2F200352v1" class="" data-icon-position="" data-hide-link-title="0">View the discussion thread.</a></p></noscript></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/10.1101/200303v1" title="Nitrate modulates stem cell dynamics by regulating WUSCHEL expression through cytokinins" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous</span></a></span><span class="pager-next"><a href="/content/10.1101/200360v1" title="Cis-regulatory evolution integrated the Bric-à-brac transcription factors into a novel fruit fly gene regulatory network" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    Posted&nbsp;October 09, 2017.  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-biorxiv-art-tools" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-biorxiv_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/10.1101/200352v1.full.pdf" target="_self" class="article-dl-pdf-link link-icon"><i class="icon-external-link-sign"></i> <span class="title">Download PDF</span></a>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-biorxiv-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="Email this Article" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-biorxiv_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/10.1101/200352v1.full" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for your interest in spreading the word about bioRxiv.</p><p>NOTE: Your email address is requested solely to identify you as the sender of this article.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/10.1101/200352v1" class="active" data-icon-position="" data-hide-link-title="0">Landscape genomic prediction for restoration of a Eucalyptus foundation species under climate change</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from bioRxiv
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would like to see this page from the bioRxiv website.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/60913" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-CF3B-ctRCO_v7LDymLOmTrxHcttnlsZ6HuiyCK_6KjM" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-biorxiv-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-biorxiv_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="60913" id="node60913--2" data-pisa="biorxiv;200352v1" data-pisa-master="biorxiv;200352" data-seqnum="60913" data-apath="/biorxiv/early/2017/10/09/200352.atom"><div  class="highwire-cite highwire-cite-highwire-article highwire-citation-biorxiv-article-pap-list clearfix" >

      <div class="highwire-cite-title" >
      <div class="highwire-cite-title">Landscape genomic prediction for restoration of a <em>Eucalyptus</em> foundation species under climate change</div>    </div>
  
      <div  class="highwire-cite-authors" ><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0"><span class="nlm-given-names">Megan A.</span> <span class="nlm-surname">Supple</span></span>, <span class="highwire-citation-author" data-delta="1"><span class="nlm-given-names">Jason G.</span> <span class="nlm-surname">Bragg</span></span>, <span class="highwire-citation-author" data-delta="2"><span class="nlm-given-names">Linda M.</span> <span class="nlm-surname">Broadhurst</span></span>, <span class="highwire-citation-author" data-delta="3"><span class="nlm-given-names">Adrienne B.</span> <span class="nlm-surname">Nicotra</span></span>, <span class="highwire-citation-author" data-delta="4"><span class="nlm-given-names">Margaret</span> <span class="nlm-surname">Byrne</span></span>, <span class="highwire-citation-author" data-delta="5"><span class="nlm-given-names">Rose L.</span> <span class="nlm-surname">Andrew</span></span>, <span class="highwire-citation-author" data-delta="6"><span class="nlm-given-names">Abigail</span> <span class="nlm-surname">Widdup</span></span>, <span class="highwire-citation-author" data-delta="7"><span class="nlm-given-names">Nicola C.</span> <span class="nlm-surname">Aitken</span></span>, <span class="highwire-citation-author" data-delta="8"><span class="nlm-given-names">Justin O.</span> <span class="nlm-surname">Borevitz</span></span></span></div>
  
      <div  class="highwire-cite-metadata" ><span  class="highwire-cite-metadata-journal highwire-cite-metadata">bioRxiv </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">200352; </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata"><span class="doi_label">doi:</span> https://doi.org/10.1101/200352 </span></div>
  
  
  </div>
</div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.biorxiv.org/content/10.1101/200352v1" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.biorxiv.org/content/10.1101/200352v1&amp;title=Landscape%20genomic%20prediction%20for%20restoration%20of%20a%20Eucalyptus%20foundation%20species%20under%20climate%20change" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.biorxiv.org/content/10.1101/200352v1&amp;title=Landscape%20genomic%20prediction%20for%20restoration%20of%20a%20Eucalyptus%20foundation%20species%20under%20climate%20change" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.biorxiv.org/content/10.1101/200352v1&amp;text=Landscape%20genomic%20prediction%20for%20restoration%20of%20a%20Eucalyptus%20foundation%20species%20under%20climate%20change" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.biorxiv.org/content/10.1101/200352v1&amp;title=Landscape%20genomic%20prediction%20for%20restoration%20of%20a%20Eucalyptus%20foundation%20species%20under%20climate%20change" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.biorxiv.org/content/10.1101/200352v1&amp;t=Landscape%20genomic%20prediction%20for%20restoration%20of%20a%20Eucalyptus%20foundation%20species%20under%20climate%20change" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.biorxiv.org/content/10.1101/200352v1&amp;title=Landscape%20genomic%20prediction%20for%20restoration%20of%20a%20Eucalyptus%20foundation%20species%20under%20climate%20change" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.biorxiv.org/content/10.1101/200352v1&amp;title=Landscape%20genomic%20prediction%20for%20restoration%20of%20a%20Eucalyptus%20foundation%20species%20under%20climate%20change" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-biorxiv-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title="Citation Tools"><i class="icon-globe"></i> <span class="title">Citation Tools</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-biorxiv_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="60913" id="node60913--2" data-pisa="biorxiv;200352v1" data-pisa-master="biorxiv;200352" data-seqnum="60913" data-apath="/biorxiv/early/2017/10/09/200352.atom"><div  class="highwire-cite highwire-cite-highwire-article highwire-citation-biorxiv-article-pap-list clearfix" >

      <div class="highwire-cite-title" >
      <div class="highwire-cite-title">Landscape genomic prediction for restoration of a <em>Eucalyptus</em> foundation species under climate change</div>    </div>
  
      <div  class="highwire-cite-authors" ><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0"><span class="nlm-given-names">Megan A.</span> <span class="nlm-surname">Supple</span></span>, <span class="highwire-citation-author" data-delta="1"><span class="nlm-given-names">Jason G.</span> <span class="nlm-surname">Bragg</span></span>, <span class="highwire-citation-author" data-delta="2"><span class="nlm-given-names">Linda M.</span> <span class="nlm-surname">Broadhurst</span></span>, <span class="highwire-citation-author" data-delta="3"><span class="nlm-given-names">Adrienne B.</span> <span class="nlm-surname">Nicotra</span></span>, <span class="highwire-citation-author" data-delta="4"><span class="nlm-given-names">Margaret</span> <span class="nlm-surname">Byrne</span></span>, <span class="highwire-citation-author" data-delta="5"><span class="nlm-given-names">Rose L.</span> <span class="nlm-surname">Andrew</span></span>, <span class="highwire-citation-author" data-delta="6"><span class="nlm-given-names">Abigail</span> <span class="nlm-surname">Widdup</span></span>, <span class="highwire-citation-author" data-delta="7"><span class="nlm-given-names">Nicola C.</span> <span class="nlm-surname">Aitken</span></span>, <span class="highwire-citation-author" data-delta="8"><span class="nlm-given-names">Justin O.</span> <span class="nlm-surname">Borevitz</span></span></span></div>
  
      <div  class="highwire-cite-metadata" ><span  class="highwire-cite-metadata-journal highwire-cite-metadata">bioRxiv </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">200352; </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata"><span class="doi_label">doi:</span> https://doi.org/10.1101/200352 </span></div>
  
  
  </div>
</div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage&amp;rft.epage&amp;rft.atitle=Landscape%20genomic%20prediction%20for%20restoration%20of%20a%20Eucalyptus%20foundation%20species%20under%20climate%20change&amp;rft.volume&amp;rft.issue&amp;rft.date=2017-01-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=bioRxiv&amp;rft.au=Supple%2C+Megan+A.&amp;rft.au=Bragg%2C+Jason+G.&amp;rft.au=Broadhurst%2C+Linda+M.&amp;rft.au=Nicotra%2C+Adrienne+B.&amp;rft.au=Byrne%2C+Margaret&amp;rft.au=Andrew%2C+Rose+L.&amp;rft.au=Widdup%2C+Abigail&amp;rft.au=Aitken%2C+Nicola+C.&amp;rft.au=Borevitz%2C+Justin+O."></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/60913/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/60913/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/60913/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/60913/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/60913/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/60913/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/60913/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/60913/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/60913/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/60913/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/60913/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/60913/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><div class="item-list"><ul><li class="first"><a href="http://twitter.com/share?url=https%3A//www.biorxiv.org/content/10.1101/200352v1&amp;count=horizontal&amp;via=&amp;text=Landscape%20genomic%20prediction%20for%20restoration%20of%20a%20Eucalyptus%20foundation%20species%20under%20climate%20change&amp;counturl=https%3A//www.biorxiv.org/content/10.1101/200352v1" class="twitter-share-button service-links-twitter-widget" id="twitter_widget" title="Tweet This" rel="nofollow" data-icon-position="" data-hide-link-title="0"><span class="element-invisible">Tweet Widget</span></a></li><li><a href="http://www.facebook.com/plugins/like.php?href=https%3A//www.biorxiv.org/content/10.1101/200352v1&amp;layout=button_count&amp;show_faces=false&amp;action=like&amp;colorscheme=light&amp;width=100&amp;height=21&amp;font=&amp;locale=" id="facebook_like" title="I Like it" class="service-links-facebook-like" rel="nofollow" data-icon-position="" data-hide-link-title="0"><span class="element-invisible">Facebook Like</span></a></li><li class="last"><a href="https://www.biorxiv.org/content/10.1101/200352v1" id="google_plus_one" title="Plus it" class="service-links-google-plus-one" rel="nofollow" data-icon-position="" data-hide-link-title="0"><span class="element-invisible">Google Plus One</span></a></li></ul></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-collections" >
  
        <h2 class="pane-title">Subject Area</h2>
    
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-collections"><div class="highwire-list"><ul class="highwire-article-collection-term-list"><li class="first last odd"><span class="highwire-article-collection-term"><a href="/collection/genomics" class="highlight" data-icon-position="" data-hide-link-title="0">Genomics<i class="icon-caret-right"></i>
</a></span></li></ul></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-biorxiv-subject-collections block-style-col2" >
  
      
  
  <div class="pane-content">
    <div class="panel-flexible panels-flexible-8 clearfix" id="mini-panel-biorxiv_subject_collections">
<div class="panel-flexible-inside panels-flexible-8-inside">
<div class="panels-flexible-region panels-flexible-region-8-center panels-flexible-region-first panels-flexible-region-last">
  <div class="inside panels-flexible-region-inside panels-flexible-region-8-center-inside panels-flexible-region-inside-first panels-flexible-region-inside-last">
<div class="panel-pane pane-snippet" >
  
      
  
  <div class="pane-content">
    <div class="snippet biorxiv-subject-areas-table-title" id="biorxiv-subject-areas-table-title">
  
      
  <div class="snippet-content">
    <b>Subject Areas</b>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-snippet" >
  
      
  
  <div class="pane-content">
    <div class="snippet biorxiv-subject-areas-view-papers" id="biorxiv-subject-areas-view-papers">
  
      
  <div class="snippet-content">
    <a href="/content/early/recent"><strong>All Articles</strong></a>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-subject-collections" >
  
      
  
  <div class="pane-content">
    <ul id="collection" class="collection highwire-list-expand"><li class="outer collection depth-2  child first"><div class = "data-wrapper"><a href="/collection/animal-behavior-and-cognition" class="" data-icon-position="" data-hide-link-title="0">Animal Behavior and Cognition</a> <span class = "article-count">(1781)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/biochemistry" class="" data-icon-position="" data-hide-link-title="0">Biochemistry</a> <span class = "article-count">(3107)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/bioengineering" class="" data-icon-position="" data-hide-link-title="0">Bioengineering</a> <span class = "article-count">(2119)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/bioinformatics" class="" data-icon-position="" data-hide-link-title="0">Bioinformatics</a> <span class = "article-count">(11020)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/biophysics" class="" data-icon-position="" data-hide-link-title="0">Biophysics</a> <span class = "article-count">(4543)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/cancer-biology" class="" data-icon-position="" data-hide-link-title="0">Cancer Biology</a> <span class = "article-count">(3547)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/cell-biology" class="" data-icon-position="" data-hide-link-title="0">Cell Biology</a> <span class = "article-count">(5028)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/clinical-trials" class="" data-icon-position="" data-hide-link-title="0">Clinical Trials</a> <span class = "article-count">(137)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/developmental-biology" class="" data-icon-position="" data-hide-link-title="0">Developmental Biology</a> <span class = "article-count">(3100)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/ecology" class="" data-icon-position="" data-hide-link-title="0">Ecology</a> <span class = "article-count">(4884)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/epidemiology" class="" data-icon-position="" data-hide-link-title="0">Epidemiology</a> <span class = "article-count">(2044)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/evolutionary-biology" class="" data-icon-position="" data-hide-link-title="0">Evolutionary Biology</a> <span class = "article-count">(7665)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/genetics" class="" data-icon-position="" data-hide-link-title="0">Genetics</a> <span class = "article-count">(5808)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/genomics" class="" data-icon-position="" data-hide-link-title="0">Genomics</a> <span class = "article-count">(7279)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/immunology" class="" data-icon-position="" data-hide-link-title="0">Immunology</a> <span class = "article-count">(2698)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/microbiology" class="" data-icon-position="" data-hide-link-title="0">Microbiology</a> <span class = "article-count">(8312)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/molecular-biology" class="" data-icon-position="" data-hide-link-title="0">Molecular Biology</a> <span class = "article-count">(3313)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/neuroscience" class="" data-icon-position="" data-hide-link-title="0">Neuroscience</a> <span class = "article-count">(20149)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/paleontology" class="" data-icon-position="" data-hide-link-title="0">Paleontology</a> <span class = "article-count">(147)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/pathology" class="" data-icon-position="" data-hide-link-title="0">Pathology</a> <span class = "article-count">(522)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/pharmacology-and-toxicology" class="" data-icon-position="" data-hide-link-title="0">Pharmacology and Toxicology</a> <span class = "article-count">(856)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/physiology" class="" data-icon-position="" data-hide-link-title="0">Physiology</a> <span class = "article-count">(1293)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/plant-biology" class="" data-icon-position="" data-hide-link-title="0">Plant Biology</a> <span class = "article-count">(3001)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/scientific-communication-and-education" class="" data-icon-position="" data-hide-link-title="0">Scientific Communication and Education</a> <span class = "article-count">(731)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/synthetic-biology" class="" data-icon-position="" data-hide-link-title="0">Synthetic Biology</a> <span class = "article-count">(971)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/systems-biology" class="" data-icon-position="" data-hide-link-title="0">Systems Biology</a> <span class = "article-count">(3023)</span></div></li>
<li class="outer collection depth-2  child last"><div class = "data-wrapper"><a href="/collection/zoology" class="" data-icon-position="" data-hide-link-title="0">Zoology</a> <span class = "article-count">(502)</span></div></li>
</ul>  </div>

  
  </div>
  </div>
</div>
</div>
</div>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__VNH5GD8Zz7g6_hGCOZjVjdMndxxC6naiExSSJKX3k_A__kzmEdtbqCsJx5Z9yg2Qy0PptB__ufXm-TxUOl0KZzUw__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js" defer="defer"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__N7ERJBYsOWyRnJgyoM125_Aiez2MOJGaUofG1JdWWBg__bLlcwhzZhT-IwoQ_dvm1_xPxU7E7gecFLC6Fyeqtv8s__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
function euCookieComplianceLoadScripts() {}
//--><!]]>
</script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var eu_cookie_compliance_cookie_name = "";
//--><!]]>
</script>
<script type="text/javascript" src="//d33xdlntwy0kbs.cloudfront.net/cshl_custom.js"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__BmqjBnkz3MgYCAoc25s1lDRMEjLhC3mEPVonUFIHi08__Unwv5-ZIuHBfFwytsjEx1niBVJ7n1T4lPws7VrkHXM4__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
  </body>
</html>
