<html xmlns="http://www.w3.org/1999/xhtml"> <head> <style type="text/css">
      	     		 
      	    a                          {background : #ffffff; }
      	    article                    {border-style : dotted; border-width : 2px; }
		 	
		 	div                        {background : #ffffcc;}
		 	div.abstract-title         {font-weight : bold ; font-size : 16pt;}
		 	div.ack                    {border-style : solid ; border-color : red; margin : 2em; }
		 	div.article-meta           {border-style : solid; border-width : 1 pt; margin 1em;}
		 	div.boxed-text             {margin : 5em; border-style : solid;}
		 	div.contrib-group          {margin : 1em; }
		 	div.fig                    {border-style : solid; border-width : 2px; margin : 2em; }
		 	div.funding                {font-weight : bold ; font-size : 16pt;}
		 	div.given-names            {font-style : italic;}
		 	div.intro                  {border-style : inset; margin : 5px;}
		 	div.introduction           {border-style : inset; margin : 5px;}
		 	div.journal-meta           {border-style : solid; border-width : 1 pt; margin 1em;}
		 	div.journal-title-group    {background : #ffeeee;}
		 	div.article-title          {font-weight : bold ; font-size : 18pt;}
		 	div.kwd                    {font-style : italic}
		 	div.meta-name              {font-weight : bold ; font-size : 16pt;}
		 	div.name                   {font-weight : bold;}
		 	div.alt-title              {font-style : italic; font-size : 12px;}
		 	
		 	div.sec                    {border : 2px; margin 5px; padding 2px;}
		 	div.title                  {font-family : courier; font-weight : bold;
		 	                            font-size : 16pt; margin : 5px;}
		 	
		 	div.materials_methods      {border-style : double; margin : 5px; }
		 	div.methods                {border-style : double; margin : 5px; }
		 	
		 	div.results                {border-style : solid; margin : 5px;}
		 	div.background             {border-style : dotted; margin : 5px;}
		 	
		 	div.discussion             {border-style : groove; margin : 5px;}
		 	div.conclusion             {border-style : ridge; margin : 5px;}
		 	div.conclusions             {border-style : ridge; margin : 5px;}
		 	div.supplementary-material {border-style : inset; margin : 5px;}
		 	div.abbreviations          {border-style : double; border-color : red; margin : 5px;}
		 	div.competinginterests     {border-style : double; border-color : blue; margin : 5px;}
		 	div.acknowledgements       {border-style : double; border-color : green; margin : 5px;}
		 	div.authors_contributions   {border-style : double; border-color : purple; margin : 5px;}
		 	
		 	div.publisher              {border-style : outset; margin : 5px;}
		 	div.fn-type-conflict       {background : #f88; }
		 	div.fn-type-con            {background : #ddf; }
		 	div.fn-type-other          {background : #ddd; }
		 	
		 	div.unknown                {background : #ffd;
									 	  border-style : solid;
									 	  border-width : 1px;
									 	  padding : 2 px;}
		 	  
      	    table                      {background : #ffffdd;}
		 	tr                         {background : #ddddff; padding : 1px;}
		 	
		 	span                       {background : #ffcccc;}
		 	
		 	span.citation-author       {font-family : helvetica; background : #ffeeee;}
		 	span.collab                {background : #ddffff; }
		 	span.comment               {font-family : courier; font-size : 6px; background : #ffaaff;}
		 	span.contrib               {background : #ffffff;}
		 	span.corresp               {background : #ddffdd; }
		 	span.doi                   {background : #ffffff;}
		 	span.email                 {font-family : courier; }
		 	span.etal                  {font-style : italic;}
		 	span.fpage                 {font-family : courier;}
		 	span.given-names           {background : #ffffff;}
		 	span.iso-abbrev            {background : #ffffff;}
		 	span.issn-epub             {background : #ffffff;}
		 	span.issn-ppub             {background : #ffffff;}
		 	span.journal-title         {background : #ffffff;}
		 	span.lpage                 {font-family : courier;}
		 	span.mixed-article-title   {font-style : italic ;}
		 	span.nlm-ta                {background : #ffffff;}
		 	span.pmc                   {background : #ffffff;}
		 	span.pmcid                 {background : #ffffff;}
		 	span.pmid                  {background : #ffffff;}
		 	span.publisher             {background : #ffffff;}
		 	span.publisher-id          {background : #ffffff;}
		 	span.publisher-name        {background : #ffffff;}
		 	span.source                {background : #ffffff;}
		 	span.subject               {background : #ffffff;}
		 	span.surname               {background : #ffffff;}
		 	span.volume                {font-family : courier; font-weight : bold;}
		 	span.year                  {font-family : courier ; font-style : italic;}
			</style> </head> <body> <div class="front" title="front"> <div class="journal-meta" tagx="journal-meta" title="journal-meta"><span class="nlm-ta" title="nlm-ta">eLife</span><span class="iso-abbrev" title="iso-abbrev">Elife</span><span class="publisher-id" title="publisher-id">eLife</span><div class="journal-title-group" tagx="journal-title-group" title="journal-title-group"><span class="journal-title" tagx="journal-title" title="journal-title">eLife</span></div><span class="issn-epub" tagx="issn" title="issn-epub">2050-084X</span><div class="publisher" tagx="publisher" title="publisher"><span class="publisher-name" tagx="publisher-name" title="publisher-name">eLife Sciences Publications, Ltd</span></div> </div> <div class="article-meta" tagx="article-meta" title="article-meta"><span class="publisher-id" title="publisher-id">51089</span><span class="doi" title="doi"> doi: <a href="https://dx.doi.org/10.7554/eLife.51089">10.7554/eLife.51089</a></span><div class="article-categories" title="article-categories">
: <span class="subject" title="subject">Research Article</span>: <span class="subject" title="subject">Computational and Systems Biology</span>: <span class="subject" title="subject">Genetics and Genomics</span></div> <div class="title-group" tagx="title-group" title="title-group"> <div class="article-title" title="article-title">
Pan-mammalian analysis of molecular constraints underlying extended lifespan</div> </div> <div class="contrib-group" title="contrib-group"><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kowalczyk</span><span class="given-names" tagx="given-names" title="given-names">Amanda</span></span><span class="contrib-id" tagx="contrib-id" title="contrib-id">https://orcid.org/0000-0002-9061-1336</span><span class="email" tagx="email" title="email">kowaae22@pitt.edu</span><a href="#aff1">1</a><a href="#aff2">2</a><a href="#fund3" /><a href="#con1" /><a href="#conf1" /></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Partha</span><span class="given-names" tagx="given-names" title="given-names">Raghavendran</span></span><span class="contrib-id" tagx="contrib-id" title="contrib-id">http://orcid.org/0000-0002-7900-4375</span><a href="#aff1">1</a><a href="#aff2">2</a><a href="#con2" /><a href="#conf1" /></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Clark</span><span class="given-names" tagx="given-names" title="given-names">Nathan L</span></span><span class="contrib-id" tagx="contrib-id" title="contrib-id">http://orcid.org/0000-0003-0006-8374</span><a href="#aff2">2</a><a href="#aff3">3</a><a href="#aff4">4</a><a href="#fund1" /><a href="#fund2" /><a href="#con3" /><a href="#conf1" /></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Chikina</span><span class="given-names" tagx="given-names" title="given-names">Maria</span></span><span class="email" tagx="email" title="email">mchikina@pitt.edu</span><a href="#aff2">2</a><a href="#fund1" /><a href="#fund2" /><a href="#con4" /><a href="#conf1" /></span><span class="citation_author_institution" id="aff1">[1], <span class="institution" title="institution">Joint Carnegie Mellon University-University of Pittsburgh PhD Program in Computational Biology</span><span class="addr-line" title="addr-line">Pittsburgh</span><span class="country" tagx="country" title="country">United States</span></span><span class="citation_author_institution" id="aff2">[2], <span class="institution" title="institution">Department of Computational and Systems Biology</span><span class="institution" title="institution">University of Pittsburgh</span><span class="addr-line" title="addr-line">Pittsburgh</span><span class="country" tagx="country" title="country">United States</span></span><span class="citation_author_institution" id="aff3">[3], <span class="institution" title="institution">Pittsburgh Center for Evolutionary Biology and Medicine</span><span class="institution" title="institution">University of Pittsburgh</span><span class="addr-line" title="addr-line">Pittsburgh</span><span class="country" tagx="country" title="country">United States</span></span><span class="citation_author_institution" id="aff4">[4], <span class="institution" title="institution">Department of Human Genetics</span><span class="institution" title="institution">University of Utah</span><span class="addr-line" title="addr-line">Salt Lake City</span><span class="country" tagx="country" title="country">United States</span></span></div> <div class="contrib-group" title="contrib-group"><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kaeberlein</span><span class="given-names" tagx="given-names" title="given-names">Matt</span></span><span class="role" tagx="role" title="role">Reviewing Editor</span><span class="citation_author_institution" id="">[], <span class="institution" title="institution">University of Washington</span><span class="country" tagx="country" title="country">United States</span></span></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Tautz</span><span class="given-names" tagx="given-names" title="given-names">Diethard</span></span><span class="role" tagx="role" title="role">Senior Editor</span><span class="citation_author_institution" id="">[], <span class="institution" title="institution">Max-Planck Institute for Evolutionary Biology</span><span class="country" tagx="country" title="country">Germany</span></span></span></div><span class="pub-date-" title="pub-date-">: <span>2020-2-2</span></span><span class="pub-date-collection" title="pub-date-collection">collection: <span>2020</span></span><span class="volume" tagx="volume" title="volume">9</span><span class="elocation-id" tagx="elocation-id" title="elocation-id">e51089</span><span class="history" title="history"><span class="received" title="received">received: 2019-8-14</span><span class="accepted" title="accepted">accepted: 2020-1-14</span></span><div class="permissions"><span class="copyright" title="copyright">(C) <span class="copyright-holder" tagx="copyright-holder" title="copyright-holder">Kowalczyk et al</span>, <span class="copyright-year" tagx="copyright-year" title="copyright-year">2020</span></span><span class="license" title="license"> <div tag="license_ref">
http://creativecommons.org/licenses/by/4.0/</div><span class="license-p" title="license-p">This article is distributed under the terms of the <a href="http://creativecommons.org/licenses/by/4.0/">Creative Commons Attribution License</a>, which permits unrestricted use and redistribution provided that the original author and source are credited.</span></span></div><a class="self-uri" href="elife-51089.pdf" title="self-uri">elife-51089.pdf</a><div class="abstract" title="abstract"> <div class="abstract-title" title="abstract-title">
Abstract</div> <p>Although lifespan in mammals varies over 100-fold, the precise evolutionary mechanisms underlying variation in longevity remain unknown. Species-specific genetic changes have been observed in long-lived species including the naked mole-rat, bats, and the bowhead whale, but these adaptations do not generalize to other mammals. We present a novel method to identify associations between rates of protein evolution and continuous phenotypes across the entire mammalian phylogeny. Unlike previous analyses that focused on individual species, we treat absolute and relative longevity as quantitative traits and demonstrate that these lifespan traits affect the evolutionary constraint on hundreds of genes. Specifically, we find that genes related to cell cycle, DNA repair, cell death, the IGF1 pathway, and immunity are under increased evolutionary constraint in large and long-lived mammals. For mammals exceptionally long-lived for their body size, we find increased constraint in inflammation, DNA repair, and NFKB-related pathways. Strikingly, these pathways have considerable overlap with those that have been previously reported to have potentially adaptive changes in single-species studies, and thus would be expected to show decreased constraint in our analysis. This unexpected finding of increased constraint in many longevity-associated pathways underscores the power of our quantitative approach to detect patterns that generalize across the mammalian phylogeny.</p> </div> <div class="kwd-group"> <div class="kwd" title="kwd">
mammals</div> <div class="kwd" title="kwd">
evolution</div> <div class="kwd" title="kwd">
genomics</div> <div class="kwd" title="kwd">
phylogenomics</div> <div class="kwd" title="kwd">
longevity</div> <div class="kwd" title="kwd">
RERconverge</div> </div> <div class="kwd-group"> <div class="title" tagx="title" title="title">
Research organism</div> <div class="kwd" title="kwd">
Other</div> </div> <div class="funding-group" title="funding-group"> <div class="funding" title="funding">
Funding</div> <div class="award-group" title="award-group"> <div class="funding-source" title="funding-source"> <div class="institution-wrap" title="institution-wrap"><span class="institution-id" title="institution-id">http://dx.doi.org/10.13039/100000002</span><span class="institution" title="institution">National Institutes of Health</span></div> </div><span class="award-id" tagx="award-id" title="award-id">R01HG009299</span><span class="principal-award-recipient" title="principal-award-recipient"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Clark</span><span class="given-names" tagx="given-names" title="given-names">Nathan L</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Chikina</span><span class="given-names" tagx="given-names" title="given-names">Maria</span></span></span></div> <div class="award-group" title="award-group"> <div class="funding-source" title="funding-source"> <div class="institution-wrap" title="institution-wrap"><span class="institution-id" title="institution-id">http://dx.doi.org/10.13039/100000002</span><span class="institution" title="institution">National Institutes of Health</span></div> </div><span class="award-id" tagx="award-id" title="award-id">U54 HG008540</span><span class="principal-award-recipient" title="principal-award-recipient"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Clark</span><span class="given-names" tagx="given-names" title="given-names">Nathan L</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Chikina</span><span class="given-names" tagx="given-names" title="given-names">Maria</span></span></span></div> <div class="award-group" title="award-group"> <div class="funding-source" title="funding-source"> <div class="institution-wrap" title="institution-wrap"><span class="institution-id" title="institution-id">http://dx.doi.org/10.13039/100000011</span><span class="institution" title="institution">Howard Hughes Medical Institute</span></div> </div><span class="award-id" tagx="award-id" title="award-id">T32 EB009403</span><span class="principal-award-recipient" title="principal-award-recipient"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kowalczyk</span><span class="given-names" tagx="given-names" title="given-names">Amanda</span></span></span></div> <div class="funding-statement" title="funding-statement">
The funders had no role in study design, data collection and interpretation, or the decision to submit the work for publication.</div> </div> <div class="custom-meta-group" title="custom-meta-group"> <meta name="Author impact statement" value="Cancer control, DNA repair, and immunity are key functionalities underlying the evolution of extended lifespan in mammals." /> </div> </div> </div> <div class="body" title="body"> <div class="intro" title="intro"> <div class="title" tagx="title" title="title">
Introduction</div> <p>Humans age in the sense that an individual’s probability of dying increases as a function of time lived. Interestingly, this trend is not true of all species but is true of mammals generally (<a href="#bib18">Jones et al., 2014</a>). Numerous hypotheses of mammal-specific aging exist, including the antagonistic pleiotropy hypothesis (<a href="#bib57">Williams, 1957</a>) and the mutation accumulation hypothesis (<a href="#bib30">Medawar, 1952</a>), both of which refer to changes within an individual throughout its lifetime that result in aging. The antagonistic pleiotropy hypothesis postulates that genes that are beneficial to early life become detrimental later in life. Such genes are retained because they increase early-life reproductive output and thus increase fitness. The mutation accumulation hypothesis predicts that a gradual accumulation of errors in DNA sequence as a result of repeated replication during a lifetime’s worth of cell divisions will lead to a gradual breakdown of functionality. Support for both hypotheses has been found in individual species. Recent work has identified disease-related SNPs in age-related genes that are beneficial in early life and detrimental in later life in humans, thus indicating selective pressures associated with gene evolution related to aging and supporting the antagonistic pleiotropy hypothesis (<a href="#bib42">Rodríguez et al., 2017</a>). A study of SNVs in the human brain found that number of mutations was positively correlated with age and that mutations were at loci associated with age-related disease, thus supporting the mutation accumulation hypothesis (<a href="#bib28">Lodato et al., 2018</a>). However, in human populations, variability in the aging phenotype is limited and many confounding biological changes are correlated with aging, making it difficult to pinpoint specific biological processes that are causal and thus amenable to manipulation.</p> <p>On the other hand, lifespan varies dramatically (&amp;gt;100 fold) across mammals (<a href="#bib34">Nowak, 1999</a>), making comparative genomics a fruitful avenue for aging research. Numerous studies have investigated the genomic features of mammals with extreme lifespan such as bats (<a href="#bib10">Foley et al., 2018</a>; <a href="#bib46">Seim et al., 2013</a>), naked mole-rats (<a href="#bib21">Kim et al., 2011</a>), whales (<a href="#bib19">Keane et al., 2015</a>), and elephants (<a href="#bib52">Sulak et al., 2016</a>) to identify potential causative genetic changes. In <i>Myotis</i>, the longest-lived bat genus, species show lack of telomere shortening and corresponding expression changes in telomere maintenance and DNA repair genes (<a href="#bib10">Foley et al., 2018</a>). Comparative genomics studies have also suggested that changes in the insulin growth factor one pathway may enable increased lifespan and cancer resistance in <i>Myotis brandtii</i> (Brandt’s bat) (<a href="#bib46">Seim et al., 2013</a>). Similarly, naked mole-rats show differential regulation of genes associated with macromolecule degradation, mitochondrial function, and TERT, a gene associated with telomere maintenance, as well as changes to genes related to tumor suppression (<a href="#bib21">Kim et al., 2011</a>). In primates, genes associated with cardiovascular function, coagulation, and healing have been demonstrated to show evolutionary correlations with lifespan (<a href="#bib33">Muntané et al., 2018</a>). Sequencing of the bowhead whale genome revealed species-specific changes in DNA repair, cell cycle, and aging genes (<a href="#bib19">Keane et al., 2015</a>). In elephants, a striking increase in TP53 copy number has been linked to increased cancer resistance enabling longer lifespan (<a href="#bib52">Sulak et al., 2016</a>). Despite compelling results, these single- and limited-species studies have limitations. The species studied differ from their nearest sequenced relatives in multiple physiological traits as well as millions of nucleotides. Thus, while single-species studies have yielded some credible candidates for genes associated with increased lifespan, it is difficult to know to what extent these represent insights into the universal mechanisms of lifespan regulation rather than species-specific adaptation or coincidental neutral changes. In this study, we develop new methodology to evaluate the relationship between the evolutionary constraint of genes and pathways and quantitative lifespan traits in an unbiased, genome-wide, pan-mammalian analysis.</p> <p>The wide range of lifespans across the mammalian phylogeny (<a href="#fig1">Figure 1A</a>) provides the ideal dataset to investigate lifespan from a comparative genomics perspective. Because independent changes in lifespan occurred repeatedly in the mammalian species tree, lifespan can be viewed as a convergent trait. Molecular features that correlate with convergent changes in lifespan therefore may also occur repeatedly across a variety of organisms. In our study we use protein evolutionary rates quantified as the number of amino acid substitutions on a phylogenetic branch to infer convergent rate shifts associated with lifespan traits across the mammalian phylogeny.</p> <div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/"><span class="label" tagx="label" title="label">Figure 1.</span></a></div>   <div class="title" tagx="title" title="title">
Many genes have evolutionary rates correlated with longevity phenotypes as demonstrated by analysis with RERconverge.</div> <p>(<b>A</b>) A subset of species used for this analysis alongside their maximum longevity values. Lifespan varies widely across mammals independent of phylogeny. (<b>B</b>) Mammal body size and maximum lifespan values for 61 species. Lines represent the 3L phenotype and the ELL phenotype (also see <a href="#fig1s1">Figure 1—figure supplement 1</a>). (<b>C</b>) RERconverge pipeline to find correlation between relative evolutionary rates of genes and change in lifespan phenotypes. (<b>D</b> and <b>E</b>) Distribution of p-values from correlations between evolutionary rates of genes and change in the 3L and ELL phenotypes indicate an enrichment of significant correlations (also see <a href="#fig1s11">Figure 1—figure supplement 11</a>).</p>   </div> <p>Evolutionary rates are useful for linking phenotypes to genes because they reflect evolutionary constraint experienced by a genetic element (<a href="#bib59">Zhang and Yang, 2015</a>). In the absence of diversifying selection, genetic elements that support a specific trait are expected to be more constrained in species where the trait has a larger contribution to fitness. In agreement with this expectation, multiple studies have shown that a genetic element providing a function less important for a given species is under less constraint and hence exhibits a faster evolutionary rate (<a href="#bib7">Clark et al., 2013</a>; <a href="#bib16">Janiak et al., 2018</a>; <a href="#bib43">Roscito et al., 2018</a>; <a href="#bib55">Wertheim et al., 2015</a>). Reciprocally, when an element becomes relatively more important, its rate is expected to slow. Thus, in cases of phenotypic convergence, rates can be exploited to reveal important genes associated with the phenotype, such as changes to muscle and skin genes associated with the mammalian transition to a marine environment (<a href="#bib6">Chikina et al., 2016</a>) and loss of constraint of vision-related genetic elements in subterranean mammals (<a href="#bib37">Partha et al., 2017</a>; <a href="#bib40">Prudent et al., 2016</a>). Rate shifts can thus provide an evolutionary perspective on the contribution of genes, non-coding elements, or pathways to phenotypes of interest (<a href="#bib13">Hiller et al., 2012</a>). Here we report the genome-wide, pan-mammalian correlations between evolutionary rates of genes and lifespan phenotypes.</p> </div> <div class="results" title="results"> <div class="title" tagx="title" title="title">
Results</div> <p>In mammals, lifespan is strongly positively correlated with adult body size such that the largest mammals (whales) are longest-lived and the smallest mammals (small rodents) are shortest lived (<a href="#fig1">Figure 1B</a>). However, if lifespan is corrected for body size, species including bats, the naked mole-rat, and some primates are clearly exceptionally long-lived given their body sizes. The term ‘longevity’ is applied to both phenotypes and previous studies have focused on small numbers of species at both phenotypic extremes to find genetic and physiological explanations for the extended phenotypes (<a href="#bib10">Foley et al., 2018</a>; <a href="#bib19">Keane et al., 2015</a>; <a href="#bib21">Kim et al., 2011</a>; <a href="#bib46">Seim et al., 2013</a>; <a href="#bib52">Sulak et al., 2016</a>). In our study we explicitly distinguish two different extended longevity traits: the ‘long-lived large-bodied’ trait (3L) and the ‘exceptionally long-lived given body size’ trait (ELL). Using maximum lifespan and body size data (<a href="#bib54">Tacutu et al., 2018</a>) we define the 3L and ELL phenotypes (<a href="#supp1">Supplementary file 1</a>) to be the first and second principal components of body size and maximum lifespan (<a href="#fig1">Figure 1B</a>, <a href="#fig1s1">Figure 1—figure supplement 1</a>). The resulting trait values are orthogonal with respect to each other, ensuring that they can be analyzed independently.</p> <p>Having defined the 3L and ELL phenotypes, we compute the association between these phenotypes and protein-specific relative evolutionary rates (RERs) using the RERconverge method (<a href="#bib24">Kowalczyk et al., 2018</a>; <a href="#bib38">Partha et al., 2018</a>) (<a href="#fig1">Figure 1C</a> and Materials and methods). Relative evolutionary rates quantify the deviation in evolutionary rate of a protein along a specific phylogenetic branch from proteome-wide expectations. Negative RERs indicate fewer substitutions than expected due to increased constraint. Positive RERs correspond to more substitutions than expected, which could arise due to relaxation of constraint or positive selection.</p> <p>After computing correlations between all protein relative evolutionary rates and the 3L and ELL phenotypes, we find an excess of low p-values (<a href="#fig1">Figure 1D</a> and <a href="#fig1">Figure 1E</a>). In order to evaluate how much of the signal is due to true association with the phenotypes, we used a phylogenetically restricted permutation strategy (termed ‘permulations’, see Materials and methods) to generate null correlation statistics. We find that the real and permuted p-value distributions are indeed different, indicating that a large fraction of genes are correlated with lifespan phenotypes. We quantify the fraction of non-null genes using the π<sub>1</sub> method (<a href="#bib49">Storey, 2003</a>). Using our permulation p-values as the null distribution, the fraction of true positives was inferred to be ~15% (π<sub>1</sub> = 0.153) for the 3L phenotype and ~7.5% (π<sub>1</sub> = 0.075) for the ELL phenotype. Using the theoretical uniform null distribution, the corresponding values are π<sub>1</sub>=0.108 and π<sub>1</sub> = 0.021, respectively. Regardless of the null distribution choice, our analysis clearly demonstrates a significant molecular signal for gene evolutionary rates correlated with lifespan phenotypes, with a considerably higher number of associations for the 3L phenotype.</p> <p>Our analysis investigates both positive and negative correlations between evolutionary rates of genes and changes in lifespan phenotypes. Positive correlations represent genes with faster evolutionary rates in species with high 3L and ELL phenotype values relative to species with low phenotype values. Conversely, negative correlations represent genes with slower evolutionary rates in species with high 3L and ELL phenotype values relative to species with low phenotype values. Note that correlation directionality is relative – faster evolution in low phenotype values corresponds to slower evolution in high phenotype values and vice versa. However, the choice has important consequences for biological interpretation and necessitates an in-depth discussion.</p> <p>RERs reflect the amount of constraint on the genetic element. A decrease in RER implies greater evolutionary constraint and a greater contribution of that gene to fitness. An increase in RER, on the other hand, has two nearly opposite interpretations. An increased RER could arise because of a relaxation of evolutionary constraint driven by a reduced contribution to fitness. Alternatively, an increased RER could arise due to positive (also termed directional) selection, which implies that that the gene is actively undergoing directed evolution and thus could be contributing to trait-related innovation. However, since there is no default evolutionary rate for protein coding sequence, genes evolving <i>slower</i> in <i>long-lived</i> species could just as easily be interpreted as evolving <i>faster</i> in <i>short-lived</i> species. In our study, we have chosen to interpret the rate changes with reference to the effect in long-lived species (<a href="#fig2">Figure 2</a>). The ancestral mammal is believed to have been small and short-lived, and thus large values of 3L and ELL are derived traits. Consequently, our interpretation reduces to assuming that change in phenotype and shifts in evolutionary rate coincide.</p> <div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/"><span class="label" tagx="label" title="label">Figure 2.</span></a></div>   <div class="title" tagx="title" title="title">
During selective pressure shifts that drive phenotypic changes, the genetic evolutionary rate landscape shifts for many genes.</div> <p>Imagine a simplified case where Gene 1 and Gene 2 start at the same evolutionary rate (solid red and blue lines, respectively). Trait-driving genes like Gene 1 enter a transient state of increased evolutionary rates to drive trait change. Complementary genes that support the trait like Gene 2 experience increased purifying selection to allow the trait to persist as it is established. True rates (solid red and blue lines) are not measurable because they represent rates at inaccessible ancestral evolutionary timepoints – only accessible extant sequences can be used to access average rates over time (dashed red and blue lines, which represent positive and negative RERs). Therefore, transient periods of positive selection are less readily able to be captured by RERconverge than sustained purifying selection due to the greater magnitude of their captured rate shifts.</p>   </div> <p>With this interpretation, negative correlations imply that a gene is more important in species with large values of 3L and ELL, while positive correlations imply either a relaxation of constraint or positive selection in species with large values of 3L and ELL. While such positively correlated genes would be of great interest if they indeed represent molecular innovations underlying evolution of extended lifespan, we find relatively fewer such genes, no evidence of positive selection among them (see <a href="#supp6">Supplementary file 6</a>), and no enriched pathways associated with them.</p> <p>We thus focus our analysis on the negatively correlated genes, which we interpret as being under increased purifying selection in species with high longevity (3L or ELL) values. It is theoretically possible for negative correlations to be caused by accelerated evolution in small and short-lived species because of directional selection associated with development of low longevity values. However, this possibility can be rules out via branch-site models for positive selection using the low longevity species as foreground, as these show little evidence for positive selection (see <a href="#supp6">Supplementary file 6</a>). Together, these analyses support a single interpretation of the main rate convergence signal as a <i>decrease</i> in evolutionary rate, and thus an <i>increase</i> in purifying selection, experienced by species with large 3L and ELL values. It is important to emphasize that these genes are unlikely to have contributed to molecular innovation that lead to the establishment of 3L and ELL traits, but rather these represent existing biological systems that become especially important after the traits are established (see Discussion).</p> <p>While we observe a clear excess of genes at low p-values, we focus on pathway enrichment analysis which both demonstrates a stronger signal and facilitates interpreting our results in the context of existing knowledge. We investigate enriched pathways for both 3L and ELL phenotypes using a rank-based method (<a href="#supp2">Supplementary file 2</a>). After performing standard multiple-hypothesis testing corrections on the empirical p-values from permulations, there remains considerable pathway-level signal underlying the 3L and ELL traits. Both the gene-level and pathway-level results were highly robust to species removal, which indicates the biological pathways revealed here are important for longevity across mammals and are not restricted to specific species (see Materials and methods, <a href="#fig1s2">Figure 1—figure supplement 2</a>, <a href="#supp4">Supplementary file 4</a>, and <a href="#supp5">Supplementary file 5</a>).</p> <p>Among pathways under increased constraint in 3L species, we find a striking abundance of pathways related to cancer control. Those pathways can be organized into the broad categories of ‘cell cycle control’, ‘cell death’, and ‘innate and adaptive immunity’, and they also include other cancer-related pathways such as p53 regulation and telomere maintenance (<a href="#fig3">Figure 3A</a>). We likewise see a significant enrichment in cancer-related genes more broadly. We compared correlation statistics between known cancer genes from <a href="#bib1">Bailey et al. (2018)</a> and found that tumor suppressor genes had significantly lower correlation statistics (rho) than all genes (Wilcoxon rank-sum p-value=4.114e-8) while oncogenes had no significant difference in correlation statistics compared to all genes (Wilcoxon rank-sum p-value=0.3745) (<a href="#fig3s1">Figure 3—figure supplement 1</a>). This may indicate preferential purifying selection on tumor suppressor genes in large, long-lived species for lower cancer incidence. Considering that these findings, our 3L results can be naturally interpreted in the context of Peto’s Paradox (<a href="#bib39">Peto, 2016</a>). The paradox reasons as follows: if all cells have a similar probability of undergoing a malignant transformation, organisms with more cells should have a greater risk of developing cancer. However, empirical cancer rates do not vary with body size (<a href="#bib39">Peto, 2016</a>), which implies that larger animals harbor mechanisms to suppress cancer rates. Top 3L constrained pathways are associated with multiple cancer control mechanisms, including DNA repair, cell cycle control, cell death, and immune function (<a href="#fig3">Figure 3B</a> and <a href="#fig3">Figure 3C</a>). A normal cell’s transformation to malignancy involves failure of all these processes, and our analysis suggests that 3L animals are invested in the maintenance of each of their associated pathways through increased purifying selection. Based on enrichment and permulation results, we can infer that cell cycle fidelity, an early step in cancer development, is most important over evolutionary time scales for 3L species. Further, there is no evidence for enrichment of pathways associated with metastasis and angiogenesis, later steps in cancer development. This finding suggests that large, long-lived species have experienced increased selective pressure to protect pathways involved in early cancer stages but not later stages, perhaps because the most severely negative fitness impacts of cancer are felt earlier in its development. Species-specific cancer control mechanisms have been identified in individual species, such as increased <i>TP53</i> copy number in elephants (<a href="#bib52">Sulak et al., 2016</a>), but we show here that investment is cancer control is key to longevity across the entire mammalian phylogeny because top enriched pathways for the 3L phenotype do not depend on a handful of species (<a href="#fig1s3">Figure 1—figure supplement 3</a> and <a href="#fig1s2">Figure 1—figure supplement 2</a>).</p> <div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/"><span class="label" tagx="label" title="label">Figure 3.</span></a></div>   <div class="title" tagx="title" title="title">
Pathways that evolve slower in long-lived, large-bodied mammals are related to control of cancer.</div> <p>(<b>A</b>) Significantly enriched pathways under increased constraint in species with larger values of the 3L phenotype. Each dot represents a pathway, and the size and color of the dot represents the negative log of the rank-sum enrichment statistic. Width of lines connecting pathways represent the number of genes the pathways have in common. (<b>B</b> and <b>C</b>) Pathways under increased constraint in 3L species play various roles in cancer control. Pathways associated with early stages of cancer development (DNA repair, cell cycle control, cell death, and immune functions) are significantly enriched, while pathways for later stages of cancer development (angiogenesis and metastasis) are not enriched. In C), each dot represents a pathway. Yellow dots have significant permulation p-values while black dots do not. Note that dots for ‘All’ pathways are excluded for the sake of clarity.</p>   </div> <p>An additional pathway that shows a strong signal of increased constraint with the 3L phenotype is the insulin-like growth factor (IGF1) signaling pathway (<a href="#fig4">Figure 4</a>), which deserves special consideration. Perturbations of IGF1 signaling result in changes in lifespan and body size in diverse organisms (<a href="#bib17">Johnson et al., 2013</a>; <a href="#bib22">Kimura et al., 1997</a>; <a href="#bib50">Stout et al., 2013</a>), which suggests that the IGF1 pathway may be a source of innovation underlying the evolution of the 3L trait. However, we find that IGF1 pathway genes in fact evolve more slowly and are thus under increased <i>purifying</i> selection in large, long-lived species. The magnitude of this signal is quite striking; IGF1 and IGF1R are ranked 1014 and 94 respectively for increased constraint and several other pathway members are near the top (see <a href="#fig3">Figure 3A</a>). A possible explanation is that the IGF1 pathway plays an important role in cancer control (<a href="#bib26">Larsson et al., 2005</a>), and this cancer-related signal of constraint dominates any adaptive signal related to lifespan. There are also further reasons to believe that the IGF1 pathway is not the main source of the 3L trait. Across the mammalian phylogeny, lifespan is strongly correlated with body size, but genetic perturbations in the IGF pathway result in longer-lived individuals that are of smaller size (<a href="#bib14">Holzenberger et al., 2003</a>; <a href="#bib53">Sutter et al., 2007</a>) thus decoupling the two traits. This strongly suggests that changes in the IGF1 pathway are unlikely to drive the natural evolution of the large, long-lived phenotype, which is established through a different, yet-unknown mechanism.</p> <div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/"><span class="label" tagx="label" title="label">Figure 4.</span></a></div>   <div class="title" tagx="title" title="title">
The IGF1 signaling pathway is under increased evolutionary constraint in long-lived, large-bodied mammals.</div> <p>(<b>A</b>) IGF1 signaling pathway genes are significantly correlated with change in the 3L phenotype. (<b>B</b>) The IGF1 signaling pathway is significantly enriched for increased evolutionary constraint in large, long-lived species. The barcode indicates ranks of genes in the pathway within the list of all pathway-annotated genes. The worm indicates enrichment as calculated by a tricube moving average, a type of moving average in which values near the end of the sliding window are down weighted to reduce the effect of extreme values in any given window. The dashed horizontal line marks the null value indicating no enrichment. (<b>C</b>) The IGF1 signaling pathway contains many genes whose evolutionary rates are negatively correlated with the 3L phenotype. Shading indicates the Rho-signed negative log p-value for the correlation. Genes in white are not included in the IGF1 pathway annotation used to calculate pathway enrichment statistics, but they are included in the diagram for sake of completeness. The GNB2L1 gene (gray) is in the IGF1 pathway annotation, but correlation statistics were not calculated for that gene because too few branches in the gene tree met the minimum branch length cut-off.</p>   </div> <p>For the ELL phenotype, we find a smaller, more focused set of enriched genes and pathways. Of those significantly enriched constrained pathways for ELL, we see some overlap with functional groups represented in results from the 3L phenotype, notably immune-related and DNA repair pathways (<a href="#fig5">Figure 5A</a>). However, although the functional groups are the same, the pathways contained within them differ between the two phenotypes (File S2, File S3). In particular, the only significantly constrained DNA repair pathways for the 3L phenotype involve Fanconi’s anemia, while the ELL phenotype shows significantly constrained DNA repair pathways for a variety of repair functions (<a href="#fig5">Figure 5</a>). Such pathways stand out not only because of the connection between DNA repair and cancer control, but also because of the observed relationship between DNA repair and aging independent of cancer incidence. This relationship can be demonstrated experimentally by creating double-stranded DNA breaks in laboratory mice to induce an aging phenotype (<a href="#bib56">White et al., 2015</a>). There is also evidence that DNA damage causes dysregulation of the cellular chromatin state and thus can contribute to aging even in post-mitotic cells (<a href="#bib35">Oberdoerffer et al., 2008</a>; <a href="#bib47">Shi and Oberdoerffer, 2012</a>).</p> <div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/"><span class="label" tagx="label" title="label">Figure 5.</span></a></div>   <div class="title" tagx="title" title="title">
DNA repair pathways are under increased evolutionary constraint in mammals that are exceptionally long-lived given their body size.</div> <p>(<b>A</b>) Significantly enriched pathways under increased constraint in species with larger values of the ELL phenotype. Bar height indicates the negative log permulation p-value for each pathway, and the color of bars indicates the pathway enrichment statistic. (<b>B</b>) DNA repair pathways are more significantly enriched for increased evolutionary constraint in species with large values of the ELL phenotype than species with large values of the 3L phenotype. The barcodes indicate ranks of genes in the pathways within the list of all pathway-annotated genes. The worms indicate enrichment as calculated by a tricube moving average, a type of moving average in which values near the end of the sliding window are down weighted to reduce the effect of extreme values in any given window. The dashed horizontal lines mark the null value indicating no enrichment.</p>   </div> <p>In addition to DNA repair-related pathways, we also noted pathways related to NFKB signaling for which overexpression in downstream targets has been associated with aging. Experimental evidence suggests a connection between NFKB signaling and DNA repair through sirtuins, a chromatin regulator family that has already been implicated in lifespan control (<a href="#bib15">Howitz et al., 2003</a>; <a href="#bib29">Mao et al., 2012</a>). Sirtuins mediate DNA damage-induced dysregulation and are also responsible for silencing NFKB-regulated genes, thus connecting the two processes (<a href="#bib45">Salminen et al., 2008</a>). Overall, our analysis strongly suggests that fidelity in DNA repair and NFKB signaling contributes to the fitness of ELL species, indicating that these pathways may be a fruitful avenue for aging research and intervention.</p> </div> <div class="discussion" title="discussion"> <div class="title" tagx="title" title="title">
Discussion</div> <p>We employed an evolutionary rates-based method as an unbiased, genome-wide, pan-mammalian scan to identify genes and pathways that evolve significantly slower in long-lived large-bodied species (3L) and species that are exceptionally long-lived given their body size (ELL). Pathways related to cancer control, including cell cycle, DNA repair, cell death, and immunity, evolve significantly slower in 3L species, which suggests that cancer resistance is an important functionality the enable to evolution of large and long-lived species. Alternatively, a broader set of DNA repair genes and a more focused set of immune genes related to NFKB signaling evolve significantly slower in ELL species, both of which may be linked to effective DNA repair in order to preserve chromatin state.</p> <p>Our analysis differs from previous efforts in both methodology and results. Firstly, we do not consider individual amino acid changes within a protein as the unit of convergence, but rather we calculate the overall evolutionary rate of a protein on each branch of a phylogeny and consider these evolutionary rates as the unit of convergence. Secondly, unlike previous studies that focused on lineage- or species-specific changes, we look for correlations between evolutionary rates and quantitative life history traits across the entire phylogeny. This pan-mammalian approach allows us to generalize our findings to describe evolutionary trends throughout all mammals. We also draw a careful distinction between absolute lifespan and relative lifespan, which allows us to make district conclusions about the 3L and the ELL phenotypes.</p> <p>The most important distinction between our work and previous work is that prior studies began with the assumption that some genes must be under positive selection in association with evolution of extended lifespan. From our unbiased analysis, we find that in fact the strongest signal of evolutionary convergence is that of <i>increased</i> constraint on certain genes and gene families in long-lived species (see <a href="#fig3s1">Figure 3—figure supplement 1</a>). While some of the pathways have been reported previously (such as cell cycle control, DNA repair, telomerase repair, and IGF1 signaling) our finding is actually the <i>opposite</i> of (though not necessarily contradictory to) the positive selection signal that has been emphasized in previous work, which from the perspective of evolutionary rate is decreased constraint. In fact, we find comparatively few genes and no significant pathway enrichments for the opposite trend – faster relative rates in longer lived species – which would correspond to genes potentially under positive selection in longer-lived species. We therefore focus our discussion on genes and pathways evolving slower in species with large values for the 3L and ELL traits. We considered these genetic elements to be important to the evolution of the 3L and ELL traits because they have been protected from accumulating deleterious mutations and therefore evolve slower.</p> <p>Despite considerable previous work on genes under positive selection in species with extreme lifespan traits, our analysis did not recover any positively selected pathways in the pan-mammalian analysis. However, for any new trait to arise, some corresponding genetic changes must occur. It is possible that many of the molecular innovations that lead to increased lifespan are species-specific and thus would not be detected by our method. However, it is also true that our relative evolutionary rates method is much more suited to detecting the kind of long-term convergent increases in purifying selection that we observe. <a href="#fig2">Figure 2</a> represents a simple schematic history of the evolution of a trait and corresponding rate changes. During the establishment of a new trait, some genes experience a brief period of positive selection which generates the molecular innovation to create the new trait, after which the derived sequence will again be subject to purifying selection. Using extant species as a genomic reference for evolutionary history, we can only infer the average rate over the entire history and the brief period of positive selection results in a small, potentially undetectable signal. On the other hand, we can hypothesize another class of genes which become more important as a trait is established and thus experience continuously increased purifying selection after the trait exists. These are the trait ‘enabling’ as opposed to the trait ‘establishing’ genes. Because the rate change in the case of trait-enabling genes is permanent, it has a greater impact on the inferred average rate observable from extant data. Our analyses strongly support that in the case of extended longevity, such ‘enabling’ genes not only exist but are also convergent across independent trait change events.</p> <p>We have interpreted the rate correlation from the prospective of species with large phenotype-values, thereby assuming phenotype changes and rate changes coincide. The implicit choice of default rate does not affect the final gene ranking on which our analysis is based, but it does have one important consequence. While decreases in rate are easily interpretable as increased constraint, rate increase can be due to relaxation of constraint or positive selection. We address this asymmetry simply by testing both possibilities. We test the relative accelerated genes for positive selection in the long-lived species and the relatively decelerated genes for positive selection in the short-lived species. We find no evidence of systematic positive selection in either direction.</p> <p>We interpret our strongest pathway enrichment signals as indicating that cancer control is important for enabling evolution of the 3L phenotype, which is in agreement with previous work that has found similar changes in expression levels of cell cycle and immune function genes in both cancerous and aging cells (<a href="#bib5">Chatsirisupachai et al., 2019</a>). However, there may be alternative explanations. Specifically, differential pathogen pressure is expected to affect some of the same pathways. The connection to immune pathways is clear, and viruses are known to co-opt the cell cycle. TP53 and cell death are likewise important mechanisms by which cells control viral infection. All of these pathways may thus be under increased constraint in species more likely to experience viral infections. However, we believe these explanations are unlikely for several reasons. Firstly, we do not see an enrichment of genes that are specialized for virus control, such as MX1, OAS1, DHX58, and genes activated by type I interferon. Secondly, the interaction between pathogen pressure and evolutionary rates has been studied extensively and it is found that it typically drives adaptive changes in pathogen interacting protein as pathogens are often unique to a species (<a href="#bib23">Kosiol et al., 2008</a>; <a href="#bib44">Sackton et al., 2007</a>; <a href="#bib48">Shultz and Sackton, 2019</a>). This is at odds with the increased constraint of immune genes that we observe. Finally, the probability of cancer increases with age and there is theoretical and empirical evidence that importance of cancer resistance increases with body-size and lifespan. On the other hand, for humans the probability of dying from many infections decreases sharply in early life (though increases again at post-reproductive ages) (<a href="#bib36">Palmer et al., 2018</a>) suggesting that infection resistance should not be preferentially selected for in long-lived species.</p> <p>Overall, the genes and pathways we identified whose constraint is negatively correlated with the 3L and ELL traits encompass functionalities important for the evolution of extended lifespan, and they therefore represent candidate genes and pathways for further experimental exploration. Importantly, such genes were uncovered using an unbiased, genome-wide pan-mammalian scan. As such, these results point to keys to exceptional longevity that are not specific to one or a handful of species, but that are universal across mammals.</p> </div> <div class="materials_methods" title="materials|methods"> <div class="title" tagx="title" title="title">
Materials and methods</div> <p>RERconverge measures relationships between relative evolutionary rates of orthologous genes and phenotype values based on a set of gene trees whose branch lengths represent protein evolutionary rates and a set of phenotype values, either binary or continuous. The RERconverge package is freely-available for use at <a href="https://github.com/nclark-lab/RERconverge">https://github.com/nclark-lab/RERconverge</a> along with walkthroughs for beginner users (<a href="#bib24">Kowalczyk et al., 2018</a>). The original RERconverge method was successfully used to find genes undergoing convergent evolutionary rate shifts in marine mammals (<a href="#bib6">Chikina et al., 2016</a>) and subterranean mammals (<a href="#bib37">Partha et al., 2017</a>), and recent statistical improvements have made RERconverge more robust and have given it more power to detect such rate shifts (<a href="#bib38">Partha et al., 2018</a>). Here, the RERconverge methods were extended to use on continuous traits.</p> <p>To perform the RERconverge analysis, we first used PAML (<a href="#bib58">Yang, 2007</a>) to create maximum likelihood gene trees whose branch lengths represent evolutionary rates by means of number of amino acid substitutions. Gene trees were generated for 19,149 amino acid alignments of orthologous genes from the 62 mammal species from the UCSC 100-way alignment (<a href="#bib4">Blanchette et al., 2004</a>; <a href="#bib12">Harris, 2007</a>; <a href="#bib20">Kent et al., 2002</a>) using the hg19 genome, all of which are available at <a href="http://genome.ucsc.edu/">http://genome.ucsc.edu/</a>. Trees were read into R using the readTrees() function from RERconverge. This step also estimated a master tree with branch lengths that represented the average branch lengths across many gene trees. For all further steps, only 61 species were used as listed in <a href="#supp1">Supplementary file 1</a>; the cape golden mole was excluded because longevity data were not available for it. Relative evolutionary rates corrected for genome-wide evolutionary rates for each species as well as corrected for branch length heteroskedasticity using a weighted regression approach (<a href="#bib38">Partha et al., 2018</a>) were calculated using getAllResiduals() with weight = T, scale = T, cutoff = 0.001, and useSpecies = species names for the 61 mammals species.</p> <p>Relative evolutionary rates (RERs) quantify the deviation in evolutionary rate of a protein along a specific phylogenetic branch from proteome-wide expectations. Negative RERs indicate fewer substitutions than expected due to increased constraint. Positive RERs correspond to more substitutions than expected, which could arise due to relaxation of constraint or positive selection. To correlate RERs with lifespan phenotypes, we use phenotypic change along phylogenetic branches computed from maximum likelihood ancestral state reconstruction (<a href="#bib41">Revell, 2012</a>). This transformation is equivalent to phylogenetically independent contrasts (<a href="#bib9">Felsenstein, 1985</a>) and thus removes phylogenetic dependence from the phenotype values.</p> <p>We used the char2Paths() function on each phenotype with trees read in using readTrees() to create a phenotype vector that represented the predicted difference in phenotype values between each species and its ancestor. The char2Paths() function first uses maximum likelihood estimation through the fastAnc() function from phytools (<a href="#bib41">Revell, 2012</a>) and then subtracts the values between connected pairs of species to calculate a phylogenetically-independent measure of the change in the phenotype over evolutionary time. Finally, the getAllCor() function was run with each phenotype and the RER matrix with method = ‘p’, min.pos = 0, and winsorizeRER = 3 to perform Pearson correlations with RERs winsorized such that the two most extreme RER values are set to the value of the third most extreme RER. See <a href="#fig1s4">Figure 1—figure supplement 4</a> for evolutionary rate-phenotype scatterplots with high correlation values.</p> <p>The gene lists produced from the RERconverge correlation analyses were used to calculate pathway enrichments for all canonical pathways from mSigDB (<a href="#bib27">Liberzon et al., 2011</a>) and Mouse Genome Informatics (MGI) functional annotations (<a href="#bib3">Blake et al., 2003</a>; <a href="#bib8">Eppig et al., 2015</a>). For each pathway, a Wilcoxon Rank-Sum statistic was calculated to compare the sign of Rho times the negative log p-value for correlations of genes in the pathway to the same measure for all genes included in a pathway annotation using getStat() from RERconverge to calculate the sign of Rho times the negative log p-value for correlations for each gene and fastwilcoxGMTall() to quickly calculate an approximation of the Wilcoxon Rank-Sum statistic.</p> <div class="permulationanalysis(phylogenetically-restrictedpermutations)" title="sec"> <div class="title" tagx="title" title="title">
Permulation analysis (Phylogenetically-Restricted Permutations)</div> <p>In addition to calculating theoretical enrichment statistics, we developed a novel phylogenetically-restricted permutation strategy dubbed ‘permulations’ to calculate empirical p-values for each pathway enrichment statistic. Permulations are a combination of permutations and simulations in reference to the strategy for generating null phenotype values to use to generate null statistics. To perform permulations, first phenotype values (3L and ELL) were simulated for each species using phylogenetic simulations, and then original phenotype values were reassigned to the species based on the rank of the simulated values (see <a href="#fig1s5">Figure 1—figure supplement 5</a>). Simulations were performed using the geiger library in R using a Brownian motion approach and the average evolutionary rate tree created by RERconverge (<a href="#bib11">Harmon et al., 2008</a>). After creating new phenotype values, RERconverge analyses were performed to calculate correlations between evolutionary rates of genes and phenotype values, and enrichment statistics were calculated using these gene results. The process of creating new phenotype values and calculating enrichment statistics was repeated 1000 times for each phenotype and the empirical p-value for pathway enrichment was calculated as the proportion of times the permulation enrichment statistic was greater than the enrichment statistic calculated using real phenotype values.</p> <p>Permulations are an attractive option to calculate empirical p-values because they use fabricated phenotypes that are independent of RERs but respect the underlying phylogenetic relationships between species (due to use of phylogenetic simulations) while maintaining the original range of the data (due to rank-based assignment of true phenotype values). These types of analyses are a common tool in genomics because they allow for control of the dependence across tests caused by intercorrelations among genes. Permulations are highly analogous to permutations commonly performed in differential expression analysis, and we use them for the same purpose, namely to calculate empirical pathway-level statistics (<a href="#bib51">Subramanian et al., 2005</a>). This step is critical for evaluating pathway enrichment results because many pathways have elevated RER correlations even when conditioned on the phenotype of interest. A pathway with high RER correlation among its genes is more likely to show up as enriched when using a test that assumes gene independence, such as the Wilcoxon Rank-Sum test. Permulations allow us to generate an empirical null distribution for pathway enrichment statistics to correct for interdependence among gene ranks. Indeed, we find multiple pathways that show significant enrichment using a Wilcoxon Rank-Sum test but insignificant empirical p-values using permulations. We also show that permulation p-values, which take phylogeny into account, are more conservative than permutation p-values, which ignore phylogenetic dependence, and equally as conservative as the phylogenetic simulation p-values (see <a href="#fig1s6">Figure 1—figure supplement 6</a>).</p> </div> <div class="positiveselectiontests" title="sec"> <div class="title" tagx="title" title="title">
Positive selection tests</div> <p>In genes significantly correlated with the 3L and the ELL phenotypes, we investigated evidence for relaxation of constraint and positive selection on trait-defining foreground branches using phylogenetic models of codon evolution. We did so using a representative subset of the full mammalian phylogeny (see <a href="#fig1s7">Figure 1—figure supplement 7</a>). Trait-defining foreground branches were specified independently for the two phenotypes along both axes of trait values – positive and negative. In total we have four sets of trait-defining foreground branches, namely positive 3L, negative 3L, positive ELL, and negative ELL. <a href="#fig1s8">Figure 1—figure supplement 8</a> shows the phylogeny of species used for this analysis and the four sets of trait-defining foreground branches. Species were selected for the representative subset based on three criteria: 1) all species were placental mammals (the monotreme and marsupials were excluded), 2) species with highest and lowest 3L and ELL phenotype values were included and used as foreground species in their respective tests, and 3) closely-related non-foreground species were included. Only placental mammals were used because the inference of positive selection can be confounded in the non-placental clade due to the long divergence time between the clades and hence saturation of synonymous sites for many genes. Foreground species were selected based on 3L and ELL values. A subset of non-foreground species was selected to correct for over- and under-sampling in particular clades while maintaining valid ‘outgroups’ for foreground species. This subset also eliminated species that were essentially duplicates in terms of tests for positive selection. For example, the primate clade is sampled much more than other clades, that is more primate genomes are available. Additionally, because most of these species are non-foreground species, their presence has little impact on positive selection tests. To correct for the oversampling, we used five out of twelve primates in our subtree, three of which were foreground species (human, marmoset, and squirrel monkey) and two of which were outgroups (rhesus as an outgroup to human and bushbaby as an outgroup to the whole clade). We used similar logic to select outgroup species throughout the rest of the phylogeny.</p> <p>We inferred the significance of relaxation of constraint on each foreground branch set using likelihood ratio tests (LRT) between Branch-site Neutral (BS Neutral) and its nested null model M1 (sites neutral model) in PAML (<a href="#bib58">Yang, 2007</a>). Similarly, we performed LRTs between branch-site selection model (BS Alt Mod) and its null BS Neutral were used to infer positive selection on the foreground branches. Probabilities for each of these two LRTs were estimated using the chi-square distribution with 1 degree of freedom. We additionally inferred significance of mammal-wide relaxation of constraint and positive selection using the LRTs between M8A (Neutral model) vs M0 (null model) and M8 (positive selection model) vs M8A respectively. Prior to performing the mammal-wide tests in genes corresponding to each of the four foreground branch sets, we removed the corresponding foreground branches, allowing us to obtain unbiased estimates for significance of relaxation of constraint and positive selection from only the background mammalian branches.</p> </div> <div class="speciesrobustnessthroughsubtreeanalysis" title="sec"> <div class="title" tagx="title" title="title">
Species robustness through subtree analysis</div> <p>We were interested in assessing the sensitivity of our results to the choice of mammalian species used in our analyses. The mammalian genomes available represent a subset of not only all extant mammals, but all mammal species that have ever existed. Therefore, the genomes used in this analysis represent an incomplete, and perhaps even a biased representation of mammal species. Since we would like to extend our conclusions to pertain to mammals in general, we sought to quantify the effects of our incomplete data on our gene and pathway results.</p> <p>To do this, we created subsets of our data that contained fewer species. These subsets had 10, 20, 30, 40, 50, 60, 70, and 80 percent of species randomly removed (6, 12, 18, 24, 30, 36, 42, and 48 species removed out of 61 total, respectively) with ten random subsets created for each species removal level. We then ran the standard RERconverge analysis on these subsets to acquire correlation statistics representing the relationship between evolutionary rate of each gene across species and longevity phenotypes in those species. We also ran enrichment analyses on the subset gene results to acquire pathway enrichment statistics. After performing these analyses, we calculated correlations between results from our full dataset and results from the subsets, where results were quantified as the negative log p-value times the sign of the statistic.</p> <p>We further tested the sensitivity of our results to specific species presence/absence by performing targeted species removal of species groups with which we expected to have other phenotypes confounded with body size, namely marine mammals, and species that may have non-convergent genetic mechanisms for lifespan extension, namely bats and the naked mole rat. We created two new data subsets that only contained non-marine species (dolphin, manatee, killer whale, walrus, and Weddell seal were removed) and only non-bat and non-naked-mole-rat species (megabat, black flying-fox, microbat, David’s myotis bat, big brown bat, and naked mole-rat were removed). We then performed the standard RERconverge analysis to find correlations between the evolutionary rates of genes across species and longevity phenotypes in those species. We also performed pathway enrichment analyses on the gene results.</p> <p>We found good correlation of both enrichment and correlation statistics between subset results and full dataset results based on negative log p-values times the sign of the statistic (<a href="#fig1s3">Figure 1—figure supplement 3</a>). For data subsets in which a proportion of species were randomly eliminated, there was upwards of a median 60% correlation between results even with half all species removed from our analyses, which indicates that randomly removing species did not significantly change our results. These findings are quantified numerically as an among-subtree variance in <a href="#supp4">Supplementary file 4</a> and <a href="#supp5">Supplementary file 5</a>. We also compared full dataset results to results from targeted data subsets without marine mammals and without bats and the naked mole-rat by quantifying correlations between their negative log p-values times the sign of the statistic for gene correlations and pathway enrichment statistics (<a href="#fig1s8">Figure 1—figure supplement 8</a>). From these comparisons, we found a strong relationship between full dataset results and targeted subset results, which indicates that presence or absence of potentially problematic species such as marine mammals, bats, and the naked mole-rat is not strongly impacting our results. Quantitative representations of these findings are available in <a href="#supp4">Supplementary file 4</a> and <a href="#supp5">Supplementary file 5</a> Together, these findings indicate that our results are not species, clade, or species subgroup-specific, but instead represent trends across all mammals.</p> </div> <div class="alternatetreetopologyanalysis" title="sec"> <div class="title" tagx="title" title="title">
Alternate tree topology analysis</div> <p>In addition to determining whether individual or groups of species were disproportionately driving our conclusions, we were also interested in verifying that the fixed tree topology used for analyses was not affecting results. To do so, we reran analyses using alternate plausible topologies representing different ancestral relationships among species (trees shown in <a href="#fig1s9">Figure 1—figure supplement 9</a>). Differences in the alternate tree topologies represent points of potential incomplete lineage sorting that lead to uncertainty, they and are thus reasonable alternatives compared to the original tree used for analyses.</p> <p>RERconverge was run using gene trees generated using the alternate topologies and results were compared to results using the original tree. As shown in <a href="#fig1s10">Figure 1—figure supplement 10</a>, there is a strong correlation between all sets of results, which indicates that uncertainties in tree topology do not strongly affect results. This is true for very similar topologies (Robinson-Foulds distance 6) and fairly different topologies (Robinson-Foulds distance 22).</p> </div> <div class="phylogenetictrees" title="sec"> <div class="title" tagx="title" title="title">
Phylogenetic trees</div> <p>Full Tree (61 mammals):</p> <p>((((((((((((Human:0.005957477577,Chimp:0.006721826689):0.001382639829,Gorilla:0.007765177171):0.005572327638,Orangutan:0.0164503644):0.002187630666,Gibbon:0.01770384793):0.007043113559,(Green_monkey:0.007693724903,((Crab-eating_macaque:0.001292320552,Rhesus:0.00713015786):0.002951690224,Baboon:0.005199240711):0.002049749893):0.01566263562):0.0135408115,(Marmoset:0.02474184521,Squirrel_monkey:0.02096868307):0.02784675729):0.04299750653,Bushbaby:0.108738222):0.01379370868,((((((Guinea_pig:0.09048639907,(Chinchilla:0.05332953299,Brush-tailed_rat:0.08476954109):0.01287861561):0.02118937782,Naked_mole-rat:0.08588673524):0.07432515556,Squirrel:0.08896424642):0.006291577528,((((Chinese_hamster:0.04084640027,Golden_hamster:0.04456203524):0.02314125062,Prairie_vole:0.06932402649):0.01947113467,(Mouse:0.05273642272,Rat:0.05576007402):0.04435347588):0.08380065137,Lesser_Egyptian_jerboa:0.1438649666):0.04270536633):0.01663675397,(Pika:0.1256544445,Rabbit:0.07131655591):0.06535533418):0.009050428462,Chinese_tree_shrew:0.1191189141):0.003894252213):0.01425600689,(((((Panda:0.03854019703,((Weddell_seal:0.02002160645,Pacific_walrus:0.02064385875):0.01734764946,Ferret:0.04613997497):0.002879093616):0.009005888384,Dog:0.05339127565):0.01185166857,Cat:0.05020331605):0.03285617057,((((((Cow:0.02168740723,((Domestic_goat:0.01157093136,Sheep:0.01246322594):0.0049716126,Tibetan_antelope:0.01522587482):0.01465511149):0.0662523666,(Killer_whale:0.006371664911,Dolphin:0.01086552617):0.06014682602):0.01216198069,Pig:0.0796745271):0.006785823323,(Bactrian_camel:0.01240650215,Alpaca:0.01096629635):0.06374554586):0.02551888691,(White_rhinoceros:0.04977357056,Horse:0.061454379):0.02510111297):0.00331214686,((Big_brown_bat:0.03248546656,(Davids_Myotis_bat:0.02344332842,Microbat:0.01567729315):0.02193849809):0.09455328094,(Black_flying-fox:0.005833353548,Megabat:0.01611220178):0.07567400302):0.02385546003):0.002057771224):0.004845253848,(Star-nosed_mole:0.1239823369,(Hedgehog:0.1696142244,Shrew:0.1934205791):0.02079474546):0.0235875333):0.01477733374):0.01316436193,(((((Cape_golden_mole:0.1017903453,Tenrec:0.1749615473):0.01592632003,Cape_elephant_shrew:0.1516860647):0.006610995228,Aardvark:0.08326528894):0.008243787904,(Elephant:0.06812658238,Manatee:0.06198982615):0.0224994529):0.03384011363,Armadillo:0.1342602666):0.005989703247):0.2206952867,((Wallaby:0.1270943532,Tasmanian_devil:0.09944141622):0.02717055443,Opossum:0.1181200712):0.1802966572):0,Platypus:0.4322118716);</p> <p>Branch Site Tree (34 placental mammals):</p> <p>((((((Human:0.02214318927,Rhesus:0.0277942336):0.0135408115,(Marmoset:0.02474184521,Squirrel_monkey:0.02096868307):0.02784675729):0.04299750653,Bushbaby:0.108738222):0.01379370868,(((Chinchilla:0.08739752642,Naked_mole-rat:0.08588673524):0.07432515556,Squirrel:0.08896424642):0.006291577528,((((Chinese_hamster:0.04084640027,Golden_hamster:0.04456203524):0.02314125062,Prairie_vole:0.06932402649):0.01947113467,(Mouse:0.05273642272,Rat:0.05576007402):0.04435347588):0.08380065137,Lesser_Egyptian_jerboa:0.1438649666):0.04270536633):0.02958143464):0.01425600689,((((Weddell_seal:0.02002160645,Pacific_walrus:0.02064385875):0.04108430003,Cat:0.05020331605):0.03285617057,(((((Cow:0.02168740723,(Sheep:0.01743483854,Tibetan_antelope:0.01522587482):0.01465511149):0.0662523666,Killer_whale:0.06651849094):0.01216198069,Pig:0.0796745271):0.03230471024,Horse:0.08655549197):0.00331214686,((Big_brown_bat:0.03248546656,(Davids_Myotis_bat:0.02344332842,Microbat:0.01567729315):0.02193849809):0.09455328094,Black_flying-fox:0.08150735657):0.02385546003):0.002057771224):0.004845253848,(Star-nosed_mole:0.1239823369,(Hedgehog:0.1696142244,Shrew:0.1934205791):0.02079474546):0.0235875333):0.01477733374):0.01316436193,((Cape_elephant_shrew:0.1665408478,(Elephant:0.06812658238,Manatee:0.06198982615):0.0224994529):0.03384011363,Armadillo:0.1342602666):0.005989703247);</p> </div> <div class="dataandmaterialsavailability" title="sec"> <div class="title" tagx="title" title="title">
Data and materials availability</div> <p>All data and code are publicly available, provided in the main text or supplementary materials, or available through the RERconverge package on GitHub at <a href="https://github.com/nclark-lab/RERconverge">https://github.com/nclark-lab/RERconverge</a>. (<a href="#bib25">Kowalczyk et al., 2019</a>; copy archived at <a href="https://github.com/elifesciences-publications/RERconverge">https://github.com/elifesciences-publications/RERconverge</a>).</p> </div> </div> </div> <div class="back" title="back"> <div class="funding-information" title="funding-information"> <div class="title" tagx="title" title="title">
Funding Information</div> <p>This paper was supported by the following grants:</p> <ul> <li> <p> </p><div class="funding-source" title="funding-source"> <div class="institution-wrap" title="institution-wrap"><span class="institution-id" title="institution-id">http://dx.doi.org/10.13039/100000002</span><span class="institution" title="institution">National Institutes of Health</span></div> </div><span class="award-id" tagx="award-id" title="award-id">R01HG009299</span> to Nathan L Clark, Maria Chikina.<p /> </li> <li> <p> </p><div class="funding-source" title="funding-source"> <div class="institution-wrap" title="institution-wrap"><span class="institution-id" title="institution-id">http://dx.doi.org/10.13039/100000002</span><span class="institution" title="institution">National Institutes of Health</span></div> </div><span class="award-id" tagx="award-id" title="award-id">U54 HG008540</span> to Nathan L Clark, Maria Chikina.<p /> </li> <li> <p> </p><div class="funding-source" title="funding-source"> <div class="institution-wrap" title="institution-wrap"><span class="institution-id" title="institution-id">http://dx.doi.org/10.13039/100000011</span><span class="institution" title="institution">Howard Hughes Medical Institute</span></div> </div><span class="award-id" tagx="award-id" title="award-id">T32 EB009403</span> to Amanda Kowalczyk.<p /> </li> </ul> </div> <div class="ack" title="ack"> <div class="title" tagx="title" title="title">
Acknowledgements</div> <p>We would like to thank Dr. Andreas Pfenning and Dr. Dennis Kostka for helpful discussion and feedback, as well as all members of the Clark and Chikina labs. </p> </div> <div class="additional-information" title="additional-information"> <div class="title" tagx="title" title="title">
Additional information</div> <div class="fn-group" title="fn-group"> <div class="title" tagx="title" title="title"><b>Competing interests</b></div> <div class="fn-type-COI-statement" title="COI-statement"> <p>No competing interests declared.</p> </div> </div> <div class="fn-group" title="fn-group"> <div class="title" tagx="title" title="title"><b>Author contributions</b></div> <div class="fn-type-con" title="con"> <p>Conceptualization, Data curation, Software, Formal analysis, Investigation, Visualization, Methodology.</p> </div> <div class="fn-type-con" title="con"> <p>Conceptualization, Resources, Data curation, Software, Formal analysis, Supervision, Funding acquisition, Visualization, Methodology, Project administration.</p> </div> <div class="fn-type-con" title="con"> <p>Conceptualization, Resources, Supervision, Funding acquisition, Project administration.</p> </div> <div class="fn-type-con" title="con"> <p>Conceptualization, Resources, Data curation, Software, Supervision, Funding acquisition, Visualization, Methodology, Project administration.</p> </div> </div> </div> <div class="supplementary-material" title="supplementary-material"> <div class="title" tagx="title" title="title">
Additional files</div> <div class="local-data" title="local-data"><span class="label" tagx="label" title="label">Supplementary file 1.</span> <div class="title" tagx="title" title="title">
Phenotype values.</div> <p>Includes longevity, body size, 3L, and ELL values for the 61 mammal species used for analyses.</p>  <div class="media" title="media"><a href="elife-51089-supp1.xlsx">LINK</a></div> </div> <div class="local-data" title="local-data"><span class="label" tagx="label" title="label">Supplementary file 2.</span> <div class="title" tagx="title" title="title">
Pathway enrichment analysis results.</div> <p>Pathway enrichment results for both the 3L and ELL phenotypes using MSigDB canonical pathway annotations.</p>  <div class="media" title="media"><a href="elife-51089-supp2.xlsx">LINK</a></div> </div> <div class="local-data" title="local-data"><span class="label" tagx="label" title="label">Supplementary file 3.</span> <div class="title" tagx="title" title="title">
Pathway clustering for 3L enriched pathways.</div> <p>Pathway group membership for the 3L phenotype as depicted in <a href="#fig2">Figure 2</a>.</p>  <div class="media" title="media"><a href="elife-51089-supp3.xlsx">LINK</a></div> </div> <div class="local-data" title="local-data"><span class="label" tagx="label" title="label">Supplementary file 4.</span> <div class="title" tagx="title" title="title">
Gene-phenotype evolutionary rate correlation and subtree analysis gene-phenotype correlation results.</div> <p>Correlation statistics between gene evolutionary rates and phenotype values as obtained from RERconverge for both the 3L and ELL phenotypes separated by correlation direction. Also included are statistics from species removal analyses.</p>  <div class="media" title="media"><a href="elife-51089-supp4.xlsx">LINK</a></div> </div> <div class="local-data" title="local-data"><span class="label" tagx="label" title="label">Supplementary file 5.</span> <div class="title" tagx="title" title="title">
Subtree analysis pathway enrichment results.</div> <p>Pathway enrichment statistics from species removal analyses for both the 3L and ELL phenotypes using MSigDB canonical pathway annotations.</p>  <div class="media" title="media"><a href="elife-51089-supp5.xlsx">LINK</a></div> </div> <div class="local-data" title="local-data"><span class="label" tagx="label" title="label">Supplementary file 6.</span> <div class="title" tagx="title" title="title">
Branch-site model tests for positive selection results.</div> <p>Results from branch-site model tests on genes to detect genes under positive selection in species at both extremes of the 3L and ELL phenotypes.</p>  <div class="media" title="media"><a href="elife-51089-supp6.xlsx">LINK</a></div> </div> <div class="local-data" title="local-data"><span class="label" tagx="label" title="label">Transparent reporting form</span><div class="media" title="media"><a href="elife-51089-transrepform.docx">LINK</a></div> </div> </div> <div class="data-availability" title="data-availability"> <div class="title" tagx="title" title="title">
Data availability</div> <p>Genome alignments are publicly available through the UCSC genome browser - processed alignments in the form of phylogenetic trees that were used for this study are available via the RERconverge package on GitHub. Code for the RERconverge package and vignettes instructing proper use are publicly available on GitHub (<a href="https://github.com/nclark-lab/RERconverge">https://github.com/nclark-lab/RERconverge</a>; copy archived at <a href="https://github.com/elifesciences-publications/RERconverge">https://github.com/elifesciences-publications/RERconverge</a>). Longevity phenotype data is available as supplementary material.</p> <p>The following previously published dataset was used:</p> <p><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Blanchette</span><span class="given-names" tagx="given-names" title="given-names">M</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kent</span><span class="given-names" tagx="given-names" title="given-names">WJ</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Riemer</span><span class="given-names" tagx="given-names" title="given-names">C</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Elnitski</span><span class="given-names" tagx="given-names" title="given-names">L</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Smit</span><span class="given-names" tagx="given-names" title="given-names">AFA</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Roskin</span><span class="given-names" tagx="given-names" title="given-names">KM</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Baertsch</span><span class="given-names" tagx="given-names" title="given-names">R</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Rosenbloom</span><span class="given-names" tagx="given-names" title="given-names">K</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Clawson</span><span class="given-names" tagx="given-names" title="given-names">H</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Green</span><span class="given-names" tagx="given-names" title="given-names">ED</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Haussler</span><span class="given-names" tagx="given-names" title="given-names">D</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Miller</span><span class="given-names" tagx="given-names" title="given-names">W</span></span></span><span class="year" tagx="year" title="year">2003</span></span></p><div tag="data-title">
UCSC 100-way alignment</div><span class="source" tagx="source" title="source">UCSC genome browser</span><span class="pub-id" /><p /> </div> <div class="references">
References</div> <div tag="ref-list"> <ul> <div class="title" tagx="title" title="title">
References</div> <li tag="ref"><a name="bib1" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bailey</span><span class="given-names" tagx="given-names" title="given-names">MH</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Tokheim</span><span class="given-names" tagx="given-names" title="given-names">C</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Porta-Pardo</span><span class="given-names" tagx="given-names" title="given-names">E</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Sengupta</span><span class="given-names" tagx="given-names" title="given-names">S</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bertrand</span><span class="given-names" tagx="given-names" title="given-names">D</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Weerasinghe</span><span class="given-names" tagx="given-names" title="given-names">A</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Colaprico</span><span class="given-names" tagx="given-names" title="given-names">A</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wendl</span><span class="given-names" tagx="given-names" title="given-names">MC</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kim</span><span class="given-names" tagx="given-names" title="given-names">J</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Reardon</span><span class="given-names" tagx="given-names" title="given-names">B</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kwok-Shing Ng</span><span class="given-names" tagx="given-names" title="given-names">P</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Jeong</span><span class="given-names" tagx="given-names" title="given-names">KJ</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Cao</span><span class="given-names" tagx="given-names" title="given-names">S</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wang</span><span class="given-names" tagx="given-names" title="given-names">Z</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gao</span><span class="given-names" tagx="given-names" title="given-names">J</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gao</span><span class="given-names" tagx="given-names" title="given-names">Q</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wang</span><span class="given-names" tagx="given-names" title="given-names">F</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Liu</span><span class="given-names" tagx="given-names" title="given-names">EM</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Mularoni</span><span class="given-names" tagx="given-names" title="given-names">L</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Rubio-Perez</span><span class="given-names" tagx="given-names" title="given-names">C</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Nagarajan</span><span class="given-names" tagx="given-names" title="given-names">N</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Cortés-Ciriano</span><span class="given-names" tagx="given-names" title="given-names">I</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zhou</span><span class="given-names" tagx="given-names" title="given-names">DC</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Liang</span><span class="given-names" tagx="given-names" title="given-names">WW</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hess</span><span class="given-names" tagx="given-names" title="given-names">JM</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Yellapantula</span><span class="given-names" tagx="given-names" title="given-names">VD</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Tamborero</span><span class="given-names" tagx="given-names" title="given-names">D</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gonzalez-Perez</span><span class="given-names" tagx="given-names" title="given-names">A</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Suphavilai</span><span class="given-names" tagx="given-names" title="given-names">C</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ko</span><span class="given-names" tagx="given-names" title="given-names">JY</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Khurana</span><span class="given-names" tagx="given-names" title="given-names">E</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Park</span><span class="given-names" tagx="given-names" title="given-names">PJ</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Van Allen</span><span class="given-names" tagx="given-names" title="given-names">EM</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Liang</span><span class="given-names" tagx="given-names" title="given-names">H</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lawrence</span><span class="given-names" tagx="given-names" title="given-names">MS</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Godzik</span><span class="given-names" tagx="given-names" title="given-names">A</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lopez-Bigas</span><span class="given-names" tagx="given-names" title="given-names">N</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Stuart</span><span class="given-names" tagx="given-names" title="given-names">J</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wheeler</span><span class="given-names" tagx="given-names" title="given-names">D</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Getz</span><span class="given-names" tagx="given-names" title="given-names">G</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Chen</span><span class="given-names" tagx="given-names" title="given-names">K</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lazar</span><span class="given-names" tagx="given-names" title="given-names">AJ</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Mills</span><span class="given-names" tagx="given-names" title="given-names">GB</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Karchin</span><span class="given-names" tagx="given-names" title="given-names">R</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ding</span><span class="given-names" tagx="given-names" title="given-names">L</span></span><span class="collab'">collab: MC3 Working Group, Cancer Genome Atlas Research Network</span></span><span class="year" tagx="year" title="year">2018</span><span class="mixed-article-title" title="mixed-article-title">Comprehensive characterization of Cancer driver genes and mutations</span><span class="source" tagx="source" title="source">Cell</span><span class="volume" tagx="volume" title="volume">174</span><span class="fpage" tagx="fpage" title="fpage">1034</span><span class="lpage" tagx="lpage" title="lpage">1035</span><span class="pub-id"><a href="https://dx.doi.org/10.1016/j.cell.2018.07.034">10.1016/j.cell.2018.07.034</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/30096302">30096302</a></span></span></li> <li tag="ref"><a name="bib2" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bininda-Emonds</span><span class="given-names" tagx="given-names" title="given-names">OR</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Cardillo</span><span class="given-names" tagx="given-names" title="given-names">M</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Jones</span><span class="given-names" tagx="given-names" title="given-names">KE</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">MacPhee</span><span class="given-names" tagx="given-names" title="given-names">RD</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Beck</span><span class="given-names" tagx="given-names" title="given-names">RM</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Grenyer</span><span class="given-names" tagx="given-names" title="given-names">R</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Price</span><span class="given-names" tagx="given-names" title="given-names">SA</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Vos</span><span class="given-names" tagx="given-names" title="given-names">RA</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gittleman</span><span class="given-names" tagx="given-names" title="given-names">JL</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Purvis</span><span class="given-names" tagx="given-names" title="given-names">A</span></span></span><span class="year" tagx="year" title="year">2007</span><span class="mixed-article-title" title="mixed-article-title">The delayed rise of present-day mammals</span><span class="source" tagx="source" title="source">Nature</span><span class="volume" tagx="volume" title="volume">446</span><span class="fpage" tagx="fpage" title="fpage">507</span><span class="lpage" tagx="lpage" title="lpage">512</span><span class="pub-id"><a href="https://dx.doi.org/10.1038/nature05634">10.1038/nature05634</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17392779">17392779</a></span></span></li> <li tag="ref"><a name="bib3" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Blake</span><span class="given-names" tagx="given-names" title="given-names">JA</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Richardson</span><span class="given-names" tagx="given-names" title="given-names">JE</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bult</span><span class="given-names" tagx="given-names" title="given-names">CJ</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kadin</span><span class="given-names" tagx="given-names" title="given-names">JA</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Eppig</span><span class="given-names" tagx="given-names" title="given-names">JT</span></span><span class="collab'">collab: Mouse Genome Database Group</span></span><span class="year" tagx="year" title="year">2003</span><span class="mixed-article-title" title="mixed-article-title">MGD: the mouse genome database</span><span class="source" tagx="source" title="source">Nucleic Acids Research</span><span class="volume" tagx="volume" title="volume">31</span><span class="fpage" tagx="fpage" title="fpage">193</span><span class="lpage" tagx="lpage" title="lpage">195</span><span class="pub-id"><a href="https://dx.doi.org/10.1093/nar/gkg047">10.1093/nar/gkg047</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12519980">12519980</a></span></span></li> <li tag="ref"><a name="bib4" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Blanchette</span><span class="given-names" tagx="given-names" title="given-names">M</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kent</span><span class="given-names" tagx="given-names" title="given-names">WJ</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Riemer</span><span class="given-names" tagx="given-names" title="given-names">C</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Elnitski</span><span class="given-names" tagx="given-names" title="given-names">L</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Smit</span><span class="given-names" tagx="given-names" title="given-names">AF</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Roskin</span><span class="given-names" tagx="given-names" title="given-names">KM</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Baertsch</span><span class="given-names" tagx="given-names" title="given-names">R</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Rosenbloom</span><span class="given-names" tagx="given-names" title="given-names">K</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Clawson</span><span class="given-names" tagx="given-names" title="given-names">H</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Green</span><span class="given-names" tagx="given-names" title="given-names">ED</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Haussler</span><span class="given-names" tagx="given-names" title="given-names">D</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Miller</span><span class="given-names" tagx="given-names" title="given-names">W</span></span></span><span class="year" tagx="year" title="year">2004</span><span class="mixed-article-title" title="mixed-article-title">Aligning multiple genomic sequences with the threaded blockset aligner</span><span class="source" tagx="source" title="source">Genome Research</span><span class="volume" tagx="volume" title="volume">14</span><span class="fpage" tagx="fpage" title="fpage">708</span><span class="lpage" tagx="lpage" title="lpage">715</span><span class="pub-id"><a href="https://dx.doi.org/10.1101/gr.1933104">10.1101/gr.1933104</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15060014">15060014</a></span></span></li> <li tag="ref"><a name="bib5" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Chatsirisupachai</span><span class="given-names" tagx="given-names" title="given-names">K</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Palmer</span><span class="given-names" tagx="given-names" title="given-names">D</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ferreira</span><span class="given-names" tagx="given-names" title="given-names">S</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">de Magalhães</span><span class="given-names" tagx="given-names" title="given-names">JP</span></span></span><span class="year" tagx="year" title="year">2019</span><span class="mixed-article-title" title="mixed-article-title">A human tissue-specific transcriptomic analysis reveals a complex relationship between aging, Cancer, and cellular senescence</span><span class="source" tagx="source" title="source">Aging Cell</span><span class="volume" tagx="volume" title="volume">18</span><span class="elocation-id" tagx="elocation-id" title="elocation-id">e13041</span><span class="pub-id"><a href="https://dx.doi.org/10.1111/acel.13041">10.1111/acel.13041</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/31560156">31560156</a></span></span></li> <li tag="ref"><a name="bib6" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Chikina</span><span class="given-names" tagx="given-names" title="given-names">M</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Robinson</span><span class="given-names" tagx="given-names" title="given-names">JD</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Clark</span><span class="given-names" tagx="given-names" title="given-names">NL</span></span></span><span class="year" tagx="year" title="year">2016</span><span class="mixed-article-title" title="mixed-article-title">Hundreds of genes experienced convergent shifts in selective pressure in marine mammals</span><span class="source" tagx="source" title="source">Molecular Biology and Evolution</span><span class="volume" tagx="volume" title="volume">33</span><span class="fpage" tagx="fpage" title="fpage">2182</span><span class="lpage" tagx="lpage" title="lpage">2192</span><span class="pub-id"><a href="https://dx.doi.org/10.1093/molbev/msw112">10.1093/molbev/msw112</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/27329977">27329977</a></span></span></li> <li tag="ref"><a name="bib7" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Clark</span><span class="given-names" tagx="given-names" title="given-names">NL</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Alani</span><span class="given-names" tagx="given-names" title="given-names">E</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Aquadro</span><span class="given-names" tagx="given-names" title="given-names">CF</span></span></span><span class="year" tagx="year" title="year">2013</span><span class="mixed-article-title" title="mixed-article-title">Evolutionary rate covariation in meiotic proteins results from fluctuating evolutionary pressure in yeasts and mammals</span><span class="source" tagx="source" title="source">Genetics</span><span class="volume" tagx="volume" title="volume">193</span><span class="fpage" tagx="fpage" title="fpage">529</span><span class="lpage" tagx="lpage" title="lpage">538</span><span class="pub-id"><a href="https://dx.doi.org/10.1534/genetics.112.145979">10.1534/genetics.112.145979</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23183665">23183665</a></span></span></li> <li tag="ref"><a name="bib8" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Eppig</span><span class="given-names" tagx="given-names" title="given-names">JT</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Blake</span><span class="given-names" tagx="given-names" title="given-names">JA</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bult</span><span class="given-names" tagx="given-names" title="given-names">CJ</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kadin</span><span class="given-names" tagx="given-names" title="given-names">JA</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Richardson</span><span class="given-names" tagx="given-names" title="given-names">JE</span></span><span class="collab'">collab: Mouse Genome Database Group</span></span><span class="year" tagx="year" title="year">2015</span><span class="mixed-article-title" title="mixed-article-title">The mouse genome database (MGD): facilitating mouse as a model for human biology and disease</span><span class="source" tagx="source" title="source">Nucleic Acids Research</span><span class="volume" tagx="volume" title="volume">43</span><span class="fpage" tagx="fpage" title="fpage">D726</span><span class="lpage" tagx="lpage" title="lpage">D736</span><span class="pub-id"><a href="https://dx.doi.org/10.1093/nar/gku967">10.1093/nar/gku967</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25348401">25348401</a></span></span></li> <li tag="ref"><a name="bib9" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Felsenstein</span><span class="given-names" tagx="given-names" title="given-names">J</span></span></span><span class="year" tagx="year" title="year">1985</span><span class="mixed-article-title" title="mixed-article-title">Phylogenies and the comparative method</span><span class="source" tagx="source" title="source">The American Naturalist</span><span class="volume" tagx="volume" title="volume">125</span><span class="fpage" tagx="fpage" title="fpage">1</span><span class="lpage" tagx="lpage" title="lpage">15</span><span class="pub-id"><a href="https://dx.doi.org/10.1086/284325">10.1086/284325</a></span></span></li> <li tag="ref"><a name="bib10" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Foley</span><span class="given-names" tagx="given-names" title="given-names">NM</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hughes</span><span class="given-names" tagx="given-names" title="given-names">GM</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Huang</span><span class="given-names" tagx="given-names" title="given-names">Z</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Clarke</span><span class="given-names" tagx="given-names" title="given-names">M</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Jebb</span><span class="given-names" tagx="given-names" title="given-names">D</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Whelan</span><span class="given-names" tagx="given-names" title="given-names">CV</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Petit</span><span class="given-names" tagx="given-names" title="given-names">EJ</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Touzalin</span><span class="given-names" tagx="given-names" title="given-names">F</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Farcy</span><span class="given-names" tagx="given-names" title="given-names">O</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Jones</span><span class="given-names" tagx="given-names" title="given-names">G</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ransome</span><span class="given-names" tagx="given-names" title="given-names">RD</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kacprzyk</span><span class="given-names" tagx="given-names" title="given-names">J</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">O'Connell</span><span class="given-names" tagx="given-names" title="given-names">MJ</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kerth</span><span class="given-names" tagx="given-names" title="given-names">G</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Rebelo</span><span class="given-names" tagx="given-names" title="given-names">H</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Rodrigues</span><span class="given-names" tagx="given-names" title="given-names">L</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Puechmaille</span><span class="given-names" tagx="given-names" title="given-names">SJ</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Teeling</span><span class="given-names" tagx="given-names" title="given-names">EC</span></span></span><span class="year" tagx="year" title="year">2018</span><span class="mixed-article-title" title="mixed-article-title">Growing old, yet staying young: the role of telomeres in bats' exceptional longevity</span><span class="source" tagx="source" title="source">Science Advances</span><span class="volume" tagx="volume" title="volume">4</span><span class="elocation-id" tagx="elocation-id" title="elocation-id">eaao0926</span><span class="pub-id"><a href="https://dx.doi.org/10.1126/sciadv.aao0926">10.1126/sciadv.aao0926</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/29441358">29441358</a></span></span></li> <li tag="ref"><a name="bib11" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Harmon</span><span class="given-names" tagx="given-names" title="given-names">LJ</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Weir</span><span class="given-names" tagx="given-names" title="given-names">JT</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Brock</span><span class="given-names" tagx="given-names" title="given-names">CD</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Glor</span><span class="given-names" tagx="given-names" title="given-names">RE</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Challenger</span><span class="given-names" tagx="given-names" title="given-names">W</span></span></span><span class="year" tagx="year" title="year">2008</span><span class="mixed-article-title" title="mixed-article-title">GEIGER: investigating evolutionary radiations</span><span class="source" tagx="source" title="source">Bioinformatics</span><span class="volume" tagx="volume" title="volume">24</span><span class="fpage" tagx="fpage" title="fpage">129</span><span class="lpage" tagx="lpage" title="lpage">131</span><span class="pub-id"><a href="https://dx.doi.org/10.1093/bioinformatics/btm538">10.1093/bioinformatics/btm538</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18006550">18006550</a></span></span></li> <li tag="ref"><a name="bib12" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Harris</span><span class="given-names" tagx="given-names" title="given-names">RS</span></span></span><span class="year" tagx="year" title="year">2007</span><span class="mixed-article-title" title="mixed-article-title">Improved Pairwise Alignment of Genomic DNA</span><span class="publisher-name" tagx="publisher-name" title="publisher-name">Pennsylvania State University</span></span></li> <li tag="ref"><a name="bib13" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hiller</span><span class="given-names" tagx="given-names" title="given-names">M</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Schaar</span><span class="given-names" tagx="given-names" title="given-names">BT</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Indjeian</span><span class="given-names" tagx="given-names" title="given-names">VB</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kingsley</span><span class="given-names" tagx="given-names" title="given-names">DM</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hagey</span><span class="given-names" tagx="given-names" title="given-names">LR</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bejerano</span><span class="given-names" tagx="given-names" title="given-names">G</span></span></span><span class="year" tagx="year" title="year">2012</span><span class="mixed-article-title" title="mixed-article-title">A "forward genomics" approach links genotype to phenotype using independent phenotypic losses among related species</span><span class="source" tagx="source" title="source">Cell Reports</span><span class="volume" tagx="volume" title="volume">2</span><span class="fpage" tagx="fpage" title="fpage">817</span><span class="lpage" tagx="lpage" title="lpage">823</span><span class="pub-id"><a href="https://dx.doi.org/10.1016/j.celrep.2012.08.032">10.1016/j.celrep.2012.08.032</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23022484">23022484</a></span></span></li> <li tag="ref"><a name="bib14" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Holzenberger</span><span class="given-names" tagx="given-names" title="given-names">M</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Dupont</span><span class="given-names" tagx="given-names" title="given-names">J</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ducos</span><span class="given-names" tagx="given-names" title="given-names">B</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Leneuve</span><span class="given-names" tagx="given-names" title="given-names">P</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Géloën</span><span class="given-names" tagx="given-names" title="given-names">A</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Even</span><span class="given-names" tagx="given-names" title="given-names">PC</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Cervera</span><span class="given-names" tagx="given-names" title="given-names">P</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Le Bouc</span><span class="given-names" tagx="given-names" title="given-names">Y</span></span></span><span class="year" tagx="year" title="year">2003</span><span class="mixed-article-title" title="mixed-article-title">IGF-1 receptor regulates lifespan and resistance to oxidative stress in mice</span><span class="source" tagx="source" title="source">Nature</span><span class="volume" tagx="volume" title="volume">421</span><span class="fpage" tagx="fpage" title="fpage">182</span><span class="lpage" tagx="lpage" title="lpage">187</span><span class="pub-id"><a href="https://dx.doi.org/10.1038/nature01298">10.1038/nature01298</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12483226">12483226</a></span></span></li> <li tag="ref"><a name="bib15" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Howitz</span><span class="given-names" tagx="given-names" title="given-names">KT</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bitterman</span><span class="given-names" tagx="given-names" title="given-names">KJ</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Cohen</span><span class="given-names" tagx="given-names" title="given-names">HY</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lamming</span><span class="given-names" tagx="given-names" title="given-names">DW</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lavu</span><span class="given-names" tagx="given-names" title="given-names">S</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wood</span><span class="given-names" tagx="given-names" title="given-names">JG</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zipkin</span><span class="given-names" tagx="given-names" title="given-names">RE</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Chung</span><span class="given-names" tagx="given-names" title="given-names">P</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kisielewski</span><span class="given-names" tagx="given-names" title="given-names">A</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zhang</span><span class="given-names" tagx="given-names" title="given-names">LL</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Scherer</span><span class="given-names" tagx="given-names" title="given-names">B</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Sinclair</span><span class="given-names" tagx="given-names" title="given-names">DA</span></span></span><span class="year" tagx="year" title="year">2003</span><span class="mixed-article-title" title="mixed-article-title">Small molecule activators of sirtuins extend Saccharomyces cerevisiae lifespan</span><span class="source" tagx="source" title="source">Nature</span><span class="volume" tagx="volume" title="volume">425</span><span class="fpage" tagx="fpage" title="fpage">191</span><span class="lpage" tagx="lpage" title="lpage">196</span><span class="pub-id"><a href="https://dx.doi.org/10.1038/nature01960">10.1038/nature01960</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12939617">12939617</a></span></span></li> <li tag="ref"><a name="bib16" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Janiak</span><span class="given-names" tagx="given-names" title="given-names">MC</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Chaney</span><span class="given-names" tagx="given-names" title="given-names">ME</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Tosi</span><span class="given-names" tagx="given-names" title="given-names">AJ</span></span></span><span class="year" tagx="year" title="year">2018</span><span class="mixed-article-title" title="mixed-article-title">Evolution of acidic mammalian chitinase genes (CHIA) Is related to body mass and insectivory in primates</span><span class="source" tagx="source" title="source">Molecular Biology and Evolution</span><span class="volume" tagx="volume" title="volume">35</span><span class="fpage" tagx="fpage" title="fpage">607</span><span class="lpage" tagx="lpage" title="lpage">622</span><span class="pub-id"><a href="https://dx.doi.org/10.1093/molbev/msx312">10.1093/molbev/msx312</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/29216399">29216399</a></span></span></li> <li tag="ref"><a name="bib17" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Johnson</span><span class="given-names" tagx="given-names" title="given-names">SC</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Rabinovitch</span><span class="given-names" tagx="given-names" title="given-names">PS</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kaeberlein</span><span class="given-names" tagx="given-names" title="given-names">M</span></span></span><span class="year" tagx="year" title="year">2013</span><span class="mixed-article-title" title="mixed-article-title">mTOR is a key modulator of ageing and age-related disease</span><span class="source" tagx="source" title="source">Nature</span><span class="volume" tagx="volume" title="volume">493</span><span class="fpage" tagx="fpage" title="fpage">338</span><span class="lpage" tagx="lpage" title="lpage">345</span><span class="pub-id"><a href="https://dx.doi.org/10.1038/nature11861">10.1038/nature11861</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23325216">23325216</a></span></span></li> <li tag="ref"><a name="bib18" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Jones</span><span class="given-names" tagx="given-names" title="given-names">OR</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Scheuerlein</span><span class="given-names" tagx="given-names" title="given-names">A</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Salguero-Gómez</span><span class="given-names" tagx="given-names" title="given-names">R</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Camarda</span><span class="given-names" tagx="given-names" title="given-names">CG</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Schaible</span><span class="given-names" tagx="given-names" title="given-names">R</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Casper</span><span class="given-names" tagx="given-names" title="given-names">BB</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Dahlgren</span><span class="given-names" tagx="given-names" title="given-names">JP</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ehrlén</span><span class="given-names" tagx="given-names" title="given-names">J</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">García</span><span class="given-names" tagx="given-names" title="given-names">MB</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Menges</span><span class="given-names" tagx="given-names" title="given-names">ES</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Quintana-Ascencio</span><span class="given-names" tagx="given-names" title="given-names">PF</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Caswell</span><span class="given-names" tagx="given-names" title="given-names">H</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Baudisch</span><span class="given-names" tagx="given-names" title="given-names">A</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Vaupel</span><span class="given-names" tagx="given-names" title="given-names">JW</span></span></span><span class="year" tagx="year" title="year">2014</span><span class="mixed-article-title" title="mixed-article-title">Diversity of ageing across the tree of life</span><span class="source" tagx="source" title="source">Nature</span><span class="volume" tagx="volume" title="volume">505</span><span class="fpage" tagx="fpage" title="fpage">169</span><span class="lpage" tagx="lpage" title="lpage">173</span><span class="pub-id"><a href="https://dx.doi.org/10.1038/nature12789">10.1038/nature12789</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24317695">24317695</a></span></span></li> <li tag="ref"><a name="bib19" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Keane</span><span class="given-names" tagx="given-names" title="given-names">M</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Semeiks</span><span class="given-names" tagx="given-names" title="given-names">J</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Webb</span><span class="given-names" tagx="given-names" title="given-names">AE</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Li</span><span class="given-names" tagx="given-names" title="given-names">YI</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Quesada</span><span class="given-names" tagx="given-names" title="given-names">V</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Craig</span><span class="given-names" tagx="given-names" title="given-names">T</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Madsen</span><span class="given-names" tagx="given-names" title="given-names">LB</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">van Dam</span><span class="given-names" tagx="given-names" title="given-names">S</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Brawand</span><span class="given-names" tagx="given-names" title="given-names">D</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Marques</span><span class="given-names" tagx="given-names" title="given-names">PI</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Michalak</span><span class="given-names" tagx="given-names" title="given-names">P</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kang</span><span class="given-names" tagx="given-names" title="given-names">L</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bhak</span><span class="given-names" tagx="given-names" title="given-names">J</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Yim</span><span class="given-names" tagx="given-names" title="given-names">H-S</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Grishin</span><span class="given-names" tagx="given-names" title="given-names">NV</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Nielsen</span><span class="given-names" tagx="given-names" title="given-names">NH</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Heide-Jørgensen</span><span class="given-names" tagx="given-names" title="given-names">MP</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Oziolor</span><span class="given-names" tagx="given-names" title="given-names">EM</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Matson</span><span class="given-names" tagx="given-names" title="given-names">CW</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Church</span><span class="given-names" tagx="given-names" title="given-names">GM</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Stuart</span><span class="given-names" tagx="given-names" title="given-names">GW</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Patton</span><span class="given-names" tagx="given-names" title="given-names">JC</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">George</span><span class="given-names" tagx="given-names" title="given-names">JC</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Suydam</span><span class="given-names" tagx="given-names" title="given-names">R</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Larsen</span><span class="given-names" tagx="given-names" title="given-names">K</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">López-Otín</span><span class="given-names" tagx="given-names" title="given-names">C</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">O’Connell</span><span class="given-names" tagx="given-names" title="given-names">MJ</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bickham</span><span class="given-names" tagx="given-names" title="given-names">JW</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Thomsen</span><span class="given-names" tagx="given-names" title="given-names">B</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">de Magalhães</span><span class="given-names" tagx="given-names" title="given-names">JP</span></span></span><span class="year" tagx="year" title="year">2015</span><span class="mixed-article-title" title="mixed-article-title">Insights into the evolution of longevity from the bowhead whale genome</span><span class="source" tagx="source" title="source">Cell Reports</span><span class="volume" tagx="volume" title="volume">10</span><span class="fpage" tagx="fpage" title="fpage">112</span><span class="lpage" tagx="lpage" title="lpage">122</span><span class="pub-id"><a href="https://dx.doi.org/10.1016/j.celrep.2014.12.008">10.1016/j.celrep.2014.12.008</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25565328">25565328</a></span></span></li> <li tag="ref"><a name="bib20" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kent</span><span class="given-names" tagx="given-names" title="given-names">WJ</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Sugnet</span><span class="given-names" tagx="given-names" title="given-names">CW</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Furey</span><span class="given-names" tagx="given-names" title="given-names">TS</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Roskin</span><span class="given-names" tagx="given-names" title="given-names">KM</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Pringle</span><span class="given-names" tagx="given-names" title="given-names">TH</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zahler</span><span class="given-names" tagx="given-names" title="given-names">AM</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Haussler</span><span class="given-names" tagx="given-names" title="given-names">D</span></span></span><span class="year" tagx="year" title="year">2002</span><span class="mixed-article-title" title="mixed-article-title">The human genome browser at UCSC</span><span class="source" tagx="source" title="source">Genome Research</span><span class="volume" tagx="volume" title="volume">12</span><span class="fpage" tagx="fpage" title="fpage">996</span><span class="lpage" tagx="lpage" title="lpage">1006</span><span class="pub-id"><a href="https://dx.doi.org/10.1101/gr.229102">10.1101/gr.229102</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12045153">12045153</a></span></span></li> <li tag="ref"><a name="bib21" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kim</span><span class="given-names" tagx="given-names" title="given-names">EB</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Fang</span><span class="given-names" tagx="given-names" title="given-names">X</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Fushan</span><span class="given-names" tagx="given-names" title="given-names">AA</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Huang</span><span class="given-names" tagx="given-names" title="given-names">Z</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lobanov</span><span class="given-names" tagx="given-names" title="given-names">AV</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Han</span><span class="given-names" tagx="given-names" title="given-names">L</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Marino</span><span class="given-names" tagx="given-names" title="given-names">SM</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Sun</span><span class="given-names" tagx="given-names" title="given-names">X</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Turanov</span><span class="given-names" tagx="given-names" title="given-names">AA</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Yang</span><span class="given-names" tagx="given-names" title="given-names">P</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Yim</span><span class="given-names" tagx="given-names" title="given-names">SH</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zhao</span><span class="given-names" tagx="given-names" title="given-names">X</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kasaikina</span><span class="given-names" tagx="given-names" title="given-names">MV</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Stoletzki</span><span class="given-names" tagx="given-names" title="given-names">N</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Peng</span><span class="given-names" tagx="given-names" title="given-names">C</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Polak</span><span class="given-names" tagx="given-names" title="given-names">P</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Xiong</span><span class="given-names" tagx="given-names" title="given-names">Z</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kiezun</span><span class="given-names" tagx="given-names" title="given-names">A</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zhu</span><span class="given-names" tagx="given-names" title="given-names">Y</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Chen</span><span class="given-names" tagx="given-names" title="given-names">Y</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kryukov</span><span class="given-names" tagx="given-names" title="given-names">GV</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zhang</span><span class="given-names" tagx="given-names" title="given-names">Q</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Peshkin</span><span class="given-names" tagx="given-names" title="given-names">L</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Yang</span><span class="given-names" tagx="given-names" title="given-names">L</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bronson</span><span class="given-names" tagx="given-names" title="given-names">RT</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Buffenstein</span><span class="given-names" tagx="given-names" title="given-names">R</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wang</span><span class="given-names" tagx="given-names" title="given-names">B</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Han</span><span class="given-names" tagx="given-names" title="given-names">C</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Li</span><span class="given-names" tagx="given-names" title="given-names">Q</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Chen</span><span class="given-names" tagx="given-names" title="given-names">L</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zhao</span><span class="given-names" tagx="given-names" title="given-names">W</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Sunyaev</span><span class="given-names" tagx="given-names" title="given-names">SR</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Park</span><span class="given-names" tagx="given-names" title="given-names">TJ</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zhang</span><span class="given-names" tagx="given-names" title="given-names">G</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wang</span><span class="given-names" tagx="given-names" title="given-names">J</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gladyshev</span><span class="given-names" tagx="given-names" title="given-names">VN</span></span></span><span class="year" tagx="year" title="year">2011</span><span class="mixed-article-title" title="mixed-article-title">Genome sequencing reveals insights into physiology and longevity of the naked mole rat</span><span class="source" tagx="source" title="source">Nature</span><span class="volume" tagx="volume" title="volume">479</span><span class="fpage" tagx="fpage" title="fpage">223</span><span class="lpage" tagx="lpage" title="lpage">227</span><span class="pub-id"><a href="https://dx.doi.org/10.1038/nature10533">10.1038/nature10533</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21993625">21993625</a></span></span></li> <li tag="ref"><a name="bib22" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kimura</span><span class="given-names" tagx="given-names" title="given-names">KD</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Tissenbaum</span><span class="given-names" tagx="given-names" title="given-names">HA</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Liu</span><span class="given-names" tagx="given-names" title="given-names">Y</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ruvkun</span><span class="given-names" tagx="given-names" title="given-names">G</span></span></span><span class="year" tagx="year" title="year">1997</span><span class="mixed-article-title" title="mixed-article-title">daf-2, an insulin receptor-like gene that regulates longevity and diapause in Caenorhabditis elegans</span><span class="source" tagx="source" title="source">Science</span><span class="volume" tagx="volume" title="volume">277</span><span class="fpage" tagx="fpage" title="fpage">942</span><span class="lpage" tagx="lpage" title="lpage">946</span><span class="pub-id"><a href="https://dx.doi.org/10.1126/science.277.5328.942">10.1126/science.277.5328.942</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/9252323">9252323</a></span></span></li> <li tag="ref"><a name="bib23" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kosiol</span><span class="given-names" tagx="given-names" title="given-names">C</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Vinar</span><span class="given-names" tagx="given-names" title="given-names">T</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">da Fonseca</span><span class="given-names" tagx="given-names" title="given-names">RR</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hubisz</span><span class="given-names" tagx="given-names" title="given-names">MJ</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bustamante</span><span class="given-names" tagx="given-names" title="given-names">CD</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Nielsen</span><span class="given-names" tagx="given-names" title="given-names">R</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Siepel</span><span class="given-names" tagx="given-names" title="given-names">A</span></span></span><span class="year" tagx="year" title="year">2008</span><span class="mixed-article-title" title="mixed-article-title">Patterns of positive selection in six mammalian genomes</span><span class="source" tagx="source" title="source">PLOS Genetics</span><span class="volume" tagx="volume" title="volume">4</span><span class="elocation-id" tagx="elocation-id" title="elocation-id">e1000144</span><span class="pub-id"><a href="https://dx.doi.org/10.1371/journal.pgen.1000144">10.1371/journal.pgen.1000144</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18670650">18670650</a></span></span></li> <li tag="ref"><a name="bib24" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kowalczyk</span><span class="given-names" tagx="given-names" title="given-names">A</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Meyer</span><span class="given-names" tagx="given-names" title="given-names">WK</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Partha</span><span class="given-names" tagx="given-names" title="given-names">R</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Mao</span><span class="given-names" tagx="given-names" title="given-names">W</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Clark</span><span class="given-names" tagx="given-names" title="given-names">NL</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Chikina</span><span class="given-names" tagx="given-names" title="given-names">M</span></span></span><span class="year" tagx="year" title="year">2018</span><span class="mixed-article-title" title="mixed-article-title">RERconverge: an R package for associating evolutionary rates with convergent traits</span><span class="source" tagx="source" title="source">Bioinformatics</span><span class="volume" tagx="volume" title="volume">35</span><span class="fpage" tagx="fpage" title="fpage">4815</span><span class="lpage" tagx="lpage" title="lpage">4817</span><span class="pub-id"><a href="https://dx.doi.org/10.1093/bioinformatics/btz468">10.1093/bioinformatics/btz468</a></span></span></li> <li tag="ref"><a name="bib25" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kowalczyk</span><span class="given-names" tagx="given-names" title="given-names">A</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Partha</span><span class="given-names" tagx="given-names" title="given-names">R</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Clark</span><span class="given-names" tagx="given-names" title="given-names">NL</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Chikina</span><span class="given-names" tagx="given-names" title="given-names">M</span></span></span><span class="year" tagx="year" title="year">2019</span><div tag="data-title">
RERconverge</div><span class="source" tagx="source" title="source">GitHub</span><span class="version" tagx="version" title="version">c1a9883</span><a href="https://github.com/nclark-lab/RERconverge">https://github.com/nclark-lab/RERconverge</a></span></li> <li tag="ref"><a name="bib26" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Larsson</span><span class="given-names" tagx="given-names" title="given-names">O</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Girnita</span><span class="given-names" tagx="given-names" title="given-names">A</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Girnita</span><span class="given-names" tagx="given-names" title="given-names">L</span></span></span><span class="year" tagx="year" title="year">2005</span><span class="mixed-article-title" title="mixed-article-title">Role of insulin-like growth factor 1 receptor signalling in Cancer</span><span class="source" tagx="source" title="source">British Journal of Cancer</span><span class="volume" tagx="volume" title="volume">92</span><span class="fpage" tagx="fpage" title="fpage">2097</span><span class="lpage" tagx="lpage" title="lpage">2101</span><span class="pub-id"><a href="https://dx.doi.org/10.1038/sj.bjc.6602627">10.1038/sj.bjc.6602627</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15956962">15956962</a></span></span></li> <li tag="ref"><a name="bib27" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Liberzon</span><span class="given-names" tagx="given-names" title="given-names">A</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Subramanian</span><span class="given-names" tagx="given-names" title="given-names">A</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Pinchback</span><span class="given-names" tagx="given-names" title="given-names">R</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Thorvaldsdóttir</span><span class="given-names" tagx="given-names" title="given-names">H</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Tamayo</span><span class="given-names" tagx="given-names" title="given-names">P</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Mesirov</span><span class="given-names" tagx="given-names" title="given-names">JP</span></span></span><span class="year" tagx="year" title="year">2011</span><span class="mixed-article-title" title="mixed-article-title">Molecular signatures database (MSigDB) 3.0</span><span class="source" tagx="source" title="source">Bioinformatics</span><span class="volume" tagx="volume" title="volume">27</span><span class="fpage" tagx="fpage" title="fpage">1739</span><span class="lpage" tagx="lpage" title="lpage">1740</span><span class="pub-id"><a href="https://dx.doi.org/10.1093/bioinformatics/btr260">10.1093/bioinformatics/btr260</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21546393">21546393</a></span></span></li> <li tag="ref"><a name="bib28" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lodato</span><span class="given-names" tagx="given-names" title="given-names">MA</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Rodin</span><span class="given-names" tagx="given-names" title="given-names">RE</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bohrson</span><span class="given-names" tagx="given-names" title="given-names">CL</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Coulter</span><span class="given-names" tagx="given-names" title="given-names">ME</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Barton</span><span class="given-names" tagx="given-names" title="given-names">AR</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kwon</span><span class="given-names" tagx="given-names" title="given-names">M</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Sherman</span><span class="given-names" tagx="given-names" title="given-names">MA</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Vitzthum</span><span class="given-names" tagx="given-names" title="given-names">CM</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Luquette</span><span class="given-names" tagx="given-names" title="given-names">LJ</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Yandava</span><span class="given-names" tagx="given-names" title="given-names">CN</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Yang</span><span class="given-names" tagx="given-names" title="given-names">P</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Chittenden</span><span class="given-names" tagx="given-names" title="given-names">TW</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hatem</span><span class="given-names" tagx="given-names" title="given-names">NE</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ryu</span><span class="given-names" tagx="given-names" title="given-names">SC</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Woodworth</span><span class="given-names" tagx="given-names" title="given-names">MB</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Park</span><span class="given-names" tagx="given-names" title="given-names">PJ</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Walsh</span><span class="given-names" tagx="given-names" title="given-names">CA</span></span></span><span class="year" tagx="year" title="year">2018</span><span class="mixed-article-title" title="mixed-article-title">Aging and neurodegeneration are associated with increased mutations in single human neurons</span><span class="source" tagx="source" title="source">Science</span><span class="volume" tagx="volume" title="volume">359</span><span class="fpage" tagx="fpage" title="fpage">555</span><span class="lpage" tagx="lpage" title="lpage">559</span><span class="pub-id"><a href="https://dx.doi.org/10.1126/science.aao4426">10.1126/science.aao4426</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/29217584">29217584</a></span></span></li> <li tag="ref"><a name="bib29" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Mao</span><span class="given-names" tagx="given-names" title="given-names">Z</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Tian</span><span class="given-names" tagx="given-names" title="given-names">X</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Van Meter</span><span class="given-names" tagx="given-names" title="given-names">M</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ke</span><span class="given-names" tagx="given-names" title="given-names">Z</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gorbunova</span><span class="given-names" tagx="given-names" title="given-names">V</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Seluanov</span><span class="given-names" tagx="given-names" title="given-names">A</span></span></span><span class="year" tagx="year" title="year">2012</span><span class="mixed-article-title" title="mixed-article-title">Sirtuin 6 (SIRT6) rescues the decline of homologous recombination repair during replicative senescence</span><span class="source" tagx="source" title="source">PNAS</span><span class="volume" tagx="volume" title="volume">109</span><span class="fpage" tagx="fpage" title="fpage">11800</span><span class="lpage" tagx="lpage" title="lpage">11805</span><span class="pub-id"><a href="https://dx.doi.org/10.1073/pnas.1200583109">10.1073/pnas.1200583109</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22753495">22753495</a></span></span></li> <li tag="ref"><a name="bib30" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Medawar</span><span class="given-names" tagx="given-names" title="given-names">PB</span></span></span><span class="year" tagx="year" title="year">1952</span><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lewis</span><span class="given-names" tagx="given-names" title="given-names">HK</span></span></span><span class="source" tagx="source" title="source">An Unsolved Problem in Biology</span><span class="publisher-loc" tagx="publisher-loc" title="publisher-loc">London</span><span class="publisher-name" tagx="publisher-name" title="publisher-name">Scientific Research</span></span></li> <li tag="ref"><a name="bib31" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Meredith</span><span class="given-names" tagx="given-names" title="given-names">RW</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Janečka</span><span class="given-names" tagx="given-names" title="given-names">JE</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gatesy</span><span class="given-names" tagx="given-names" title="given-names">J</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ryder</span><span class="given-names" tagx="given-names" title="given-names">OA</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Fisher</span><span class="given-names" tagx="given-names" title="given-names">CA</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Teeling</span><span class="given-names" tagx="given-names" title="given-names">EC</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Goodbla</span><span class="given-names" tagx="given-names" title="given-names">A</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Eizirik</span><span class="given-names" tagx="given-names" title="given-names">E</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Simão</span><span class="given-names" tagx="given-names" title="given-names">TL</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Stadler</span><span class="given-names" tagx="given-names" title="given-names">T</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Rabosky</span><span class="given-names" tagx="given-names" title="given-names">DL</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Honeycutt</span><span class="given-names" tagx="given-names" title="given-names">RL</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Flynn</span><span class="given-names" tagx="given-names" title="given-names">JJ</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ingram</span><span class="given-names" tagx="given-names" title="given-names">CM</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Steiner</span><span class="given-names" tagx="given-names" title="given-names">C</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Williams</span><span class="given-names" tagx="given-names" title="given-names">TL</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Robinson</span><span class="given-names" tagx="given-names" title="given-names">TJ</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Burk-Herrick</span><span class="given-names" tagx="given-names" title="given-names">A</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Westerman</span><span class="given-names" tagx="given-names" title="given-names">M</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ayoub</span><span class="given-names" tagx="given-names" title="given-names">NA</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Springer</span><span class="given-names" tagx="given-names" title="given-names">MS</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Murphy</span><span class="given-names" tagx="given-names" title="given-names">WJ</span></span></span><span class="year" tagx="year" title="year">2011</span><span class="mixed-article-title" title="mixed-article-title">Impacts of the cretaceous terrestrial revolution and KPg extinction on mammal diversification</span><span class="source" tagx="source" title="source">Science</span><span class="volume" tagx="volume" title="volume">334</span><span class="fpage" tagx="fpage" title="fpage">521</span><span class="lpage" tagx="lpage" title="lpage">524</span><span class="pub-id"><a href="https://dx.doi.org/10.1126/science.1211028">10.1126/science.1211028</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21940861">21940861</a></span></span></li> <li tag="ref"><a name="bib32" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Meyer</span><span class="given-names" tagx="given-names" title="given-names">WK</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Jamison</span><span class="given-names" tagx="given-names" title="given-names">J</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Richter</span><span class="given-names" tagx="given-names" title="given-names">R</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Woods</span><span class="given-names" tagx="given-names" title="given-names">SE</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Partha</span><span class="given-names" tagx="given-names" title="given-names">R</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kowalczyk</span><span class="given-names" tagx="given-names" title="given-names">A</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kronk</span><span class="given-names" tagx="given-names" title="given-names">C</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Chikina</span><span class="given-names" tagx="given-names" title="given-names">M</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bonde</span><span class="given-names" tagx="given-names" title="given-names">RK</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Crocker</span><span class="given-names" tagx="given-names" title="given-names">DE</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gaspard</span><span class="given-names" tagx="given-names" title="given-names">J</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lanyon</span><span class="given-names" tagx="given-names" title="given-names">JM</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Marsillach</span><span class="given-names" tagx="given-names" title="given-names">J</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Furlong</span><span class="given-names" tagx="given-names" title="given-names">CE</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Clark</span><span class="given-names" tagx="given-names" title="given-names">NL</span></span></span><span class="year" tagx="year" title="year">2018</span><span class="mixed-article-title" title="mixed-article-title">Ancient convergent losses of <i>paraoxonase 1</i> yield potential risks for modern marine mammals</span><span class="source" tagx="source" title="source">Science</span><span class="volume" tagx="volume" title="volume">361</span><span class="fpage" tagx="fpage" title="fpage">591</span><span class="lpage" tagx="lpage" title="lpage">594</span><span class="pub-id"><a href="https://dx.doi.org/10.1126/science.aap7714">10.1126/science.aap7714</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/30093596">30093596</a></span></span></li> <li tag="ref"><a name="bib33" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Muntané</span><span class="given-names" tagx="given-names" title="given-names">G</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Farré</span><span class="given-names" tagx="given-names" title="given-names">X</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Rodríguez</span><span class="given-names" tagx="given-names" title="given-names">JA</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Pegueroles</span><span class="given-names" tagx="given-names" title="given-names">C</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hughes</span><span class="given-names" tagx="given-names" title="given-names">DA</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">de Magalhães</span><span class="given-names" tagx="given-names" title="given-names">JP</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gabaldón</span><span class="given-names" tagx="given-names" title="given-names">T</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Navarro</span><span class="given-names" tagx="given-names" title="given-names">A</span></span></span><span class="year" tagx="year" title="year">2018</span><span class="mixed-article-title" title="mixed-article-title">Biological processes modulating longevity across primates: a phylogenetic Genome-Phenome analysis</span><span class="source" tagx="source" title="source">Molecular Biology and Evolution</span><span class="volume" tagx="volume" title="volume">35</span><span class="fpage" tagx="fpage" title="fpage">1990</span><span class="lpage" tagx="lpage" title="lpage">2004</span><span class="pub-id"><a href="https://dx.doi.org/10.1093/molbev/msy105">10.1093/molbev/msy105</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/29788292">29788292</a></span></span></li> <li tag="ref"><a name="bib34" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Nowak</span><span class="given-names" tagx="given-names" title="given-names">RM</span></span></span><span class="year" tagx="year" title="year">1999</span><span class="source" tagx="source" title="source">Walker’s Mammals of the World</span><span class="edition" tagx="edition" title="edition">6th ed</span><span class="publisher-loc" tagx="publisher-loc" title="publisher-loc">Baltimore</span><span class="publisher-name" tagx="publisher-name" title="publisher-name">The Johns Hopkins University Press</span></span></li> <li tag="ref"><a name="bib35" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Oberdoerffer</span><span class="given-names" tagx="given-names" title="given-names">P</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Michan</span><span class="given-names" tagx="given-names" title="given-names">S</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">McVay</span><span class="given-names" tagx="given-names" title="given-names">M</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Mostoslavsky</span><span class="given-names" tagx="given-names" title="given-names">R</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Vann</span><span class="given-names" tagx="given-names" title="given-names">J</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Park</span><span class="given-names" tagx="given-names" title="given-names">SK</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hartlerode</span><span class="given-names" tagx="given-names" title="given-names">A</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Stegmuller</span><span class="given-names" tagx="given-names" title="given-names">J</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hafner</span><span class="given-names" tagx="given-names" title="given-names">A</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Loerch</span><span class="given-names" tagx="given-names" title="given-names">P</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wright</span><span class="given-names" tagx="given-names" title="given-names">SM</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Mills</span><span class="given-names" tagx="given-names" title="given-names">KD</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bonni</span><span class="given-names" tagx="given-names" title="given-names">A</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Yankner</span><span class="given-names" tagx="given-names" title="given-names">BA</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Scully</span><span class="given-names" tagx="given-names" title="given-names">R</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Prolla</span><span class="given-names" tagx="given-names" title="given-names">TA</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Alt</span><span class="given-names" tagx="given-names" title="given-names">FW</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Sinclair</span><span class="given-names" tagx="given-names" title="given-names">DA</span></span></span><span class="year" tagx="year" title="year">2008</span><span class="mixed-article-title" title="mixed-article-title">SIRT1 redistribution on chromatin promotes genomic stability but alters gene expression during aging</span><span class="source" tagx="source" title="source">Cell</span><span class="volume" tagx="volume" title="volume">135</span><span class="fpage" tagx="fpage" title="fpage">907</span><span class="lpage" tagx="lpage" title="lpage">918</span><span class="pub-id"><a href="https://dx.doi.org/10.1016/j.cell.2008.10.025">10.1016/j.cell.2008.10.025</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19041753">19041753</a></span></span></li> <li tag="ref"><a name="bib36" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Palmer</span><span class="given-names" tagx="given-names" title="given-names">S</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Albergante</span><span class="given-names" tagx="given-names" title="given-names">L</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Blackburn</span><span class="given-names" tagx="given-names" title="given-names">CC</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Newman</span><span class="given-names" tagx="given-names" title="given-names">TJ</span></span></span><span class="year" tagx="year" title="year">2018</span><span class="mixed-article-title" title="mixed-article-title">Thymic involution and rising disease incidence with age</span><span class="source" tagx="source" title="source">PNAS</span><span class="volume" tagx="volume" title="volume">115</span><span class="fpage" tagx="fpage" title="fpage">1883</span><span class="lpage" tagx="lpage" title="lpage">1888</span><span class="pub-id"><a href="https://dx.doi.org/10.1073/pnas.1714478115">10.1073/pnas.1714478115</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/29432166">29432166</a></span></span></li> <li tag="ref"><a name="bib37" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Partha</span><span class="given-names" tagx="given-names" title="given-names">R</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Chauhan</span><span class="given-names" tagx="given-names" title="given-names">BK</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ferreira</span><span class="given-names" tagx="given-names" title="given-names">Z</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Robinson</span><span class="given-names" tagx="given-names" title="given-names">JD</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lathrop</span><span class="given-names" tagx="given-names" title="given-names">K</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Nischal</span><span class="given-names" tagx="given-names" title="given-names">KK</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Chikina</span><span class="given-names" tagx="given-names" title="given-names">M</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Clark</span><span class="given-names" tagx="given-names" title="given-names">NL</span></span></span><span class="year" tagx="year" title="year">2017</span><span class="mixed-article-title" title="mixed-article-title">Subterranean mammals show convergent regression in ocular genes and enhancers, along with adaptation to tunneling</span><span class="source" tagx="source" title="source">eLife</span><span class="volume" tagx="volume" title="volume">6</span><span class="elocation-id" tagx="elocation-id" title="elocation-id">e25884</span><span class="pub-id"><a href="https://dx.doi.org/10.7554/eLife.25884">10.7554/eLife.25884</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/29035697">29035697</a></span></span></li> <li tag="ref"><a name="bib38" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Partha</span><span class="given-names" tagx="given-names" title="given-names">R</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kowalczyk</span><span class="given-names" tagx="given-names" title="given-names">A</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Clark</span><span class="given-names" tagx="given-names" title="given-names">NL</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Chikina</span><span class="given-names" tagx="given-names" title="given-names">M</span></span></span><span class="year" tagx="year" title="year">2018</span><span class="mixed-article-title" title="mixed-article-title">Robust methods for detecting convergent shifts in evolutionary rates</span><span class="source" tagx="source" title="source">bioRxiv</span><span class="pub-id"><a href="https://dx.doi.org/10.1101/457309">10.1101/457309</a></span></span></li> <li tag="ref"><a name="bib39" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Peto</span><span class="given-names" tagx="given-names" title="given-names">R</span></span></span><span class="year" tagx="year" title="year">2016</span><span class="mixed-article-title" title="mixed-article-title">Epidemiology, multistage models, and short-term mutagenicity tests</span><span class="source" tagx="source" title="source">International Journal of Epidemiology</span><span class="volume" tagx="volume" title="volume">45</span><span class="fpage" tagx="fpage" title="fpage">621</span><span class="lpage" tagx="lpage" title="lpage">637</span><span class="pub-id"><a href="https://dx.doi.org/10.1093/ije/dyv199">10.1093/ije/dyv199</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/27582437">27582437</a></span></span></li> <li tag="ref"><a name="bib40" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Prudent</span><span class="given-names" tagx="given-names" title="given-names">X</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Parra</span><span class="given-names" tagx="given-names" title="given-names">G</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Schwede</span><span class="given-names" tagx="given-names" title="given-names">P</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Roscito</span><span class="given-names" tagx="given-names" title="given-names">JG</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hiller</span><span class="given-names" tagx="given-names" title="given-names">M</span></span></span><span class="year" tagx="year" title="year">2016</span><span class="mixed-article-title" title="mixed-article-title">Controlling for phylogenetic relatedness and evolutionary rates improves the discovery of associations between species' Phenotypic and genomic differences</span><span class="source" tagx="source" title="source">Molecular Biology and Evolution</span><span class="volume" tagx="volume" title="volume">33</span><span class="fpage" tagx="fpage" title="fpage">2135</span><span class="lpage" tagx="lpage" title="lpage">2150</span><span class="pub-id"><a href="https://dx.doi.org/10.1093/molbev/msw098">10.1093/molbev/msw098</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/27222536">27222536</a></span></span></li> <li tag="ref"><a name="bib41" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Revell</span><span class="given-names" tagx="given-names" title="given-names">LJ</span></span></span><span class="year" tagx="year" title="year">2012</span><span class="mixed-article-title" title="mixed-article-title">Phytools: an R package for phylogenetic comparative biology (and other things)</span><span class="source" tagx="source" title="source">Methods in Ecology and Evolution</span><span class="volume" tagx="volume" title="volume">3</span><span class="fpage" tagx="fpage" title="fpage">217</span><span class="lpage" tagx="lpage" title="lpage">223</span><span class="pub-id"><a href="https://dx.doi.org/10.1111/j.2041-210X.2011.00169.x">10.1111/j.2041-210X.2011.00169.x</a></span></span></li> <li tag="ref"><a name="bib42" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Rodríguez</span><span class="given-names" tagx="given-names" title="given-names">JA</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Marigorta</span><span class="given-names" tagx="given-names" title="given-names">UM</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hughes</span><span class="given-names" tagx="given-names" title="given-names">DA</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Spataro</span><span class="given-names" tagx="given-names" title="given-names">N</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bosch</span><span class="given-names" tagx="given-names" title="given-names">E</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Navarro</span><span class="given-names" tagx="given-names" title="given-names">A</span></span></span><span class="year" tagx="year" title="year">2017</span><span class="mixed-article-title" title="mixed-article-title">Antagonistic pleiotropy and mutation accumulation influence human senescence and disease</span><span class="source" tagx="source" title="source">Nature Ecology &amp;amp; Evolution</span><span class="volume" tagx="volume" title="volume">1</span><span class="elocation-id" tagx="elocation-id" title="elocation-id">0055</span><span class="pub-id"><a href="https://dx.doi.org/10.1038/s41559-016-0055">10.1038/s41559-016-0055</a></span></span></li> <li tag="ref"><a name="bib43" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Roscito</span><span class="given-names" tagx="given-names" title="given-names">JG</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Sameith</span><span class="given-names" tagx="given-names" title="given-names">K</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Parra</span><span class="given-names" tagx="given-names" title="given-names">G</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Langer</span><span class="given-names" tagx="given-names" title="given-names">BE</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Petzold</span><span class="given-names" tagx="given-names" title="given-names">A</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Moebius</span><span class="given-names" tagx="given-names" title="given-names">C</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bickle</span><span class="given-names" tagx="given-names" title="given-names">M</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Rodrigues</span><span class="given-names" tagx="given-names" title="given-names">MT</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hiller</span><span class="given-names" tagx="given-names" title="given-names">M</span></span></span><span class="year" tagx="year" title="year">2018</span><span class="mixed-article-title" title="mixed-article-title">Phenotype loss is associated with widespread divergence of the gene regulatory landscape in evolution</span><span class="source" tagx="source" title="source">Nature Communications</span><span class="volume" tagx="volume" title="volume">9</span><span class="elocation-id" tagx="elocation-id" title="elocation-id">07122</span><span class="pub-id"><a href="https://dx.doi.org/10.1038/s41467-018-07122-z">10.1038/s41467-018-07122-z</a></span></span></li> <li tag="ref"><a name="bib44" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Sackton</span><span class="given-names" tagx="given-names" title="given-names">TB</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lazzaro</span><span class="given-names" tagx="given-names" title="given-names">BP</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Schlenke</span><span class="given-names" tagx="given-names" title="given-names">TA</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Evans</span><span class="given-names" tagx="given-names" title="given-names">JD</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hultmark</span><span class="given-names" tagx="given-names" title="given-names">D</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Clark</span><span class="given-names" tagx="given-names" title="given-names">AG</span></span></span><span class="year" tagx="year" title="year">2007</span><span class="mixed-article-title" title="mixed-article-title">Dynamic evolution of the innate immune system in Drosophila</span><span class="source" tagx="source" title="source">Nature Genetics</span><span class="volume" tagx="volume" title="volume">39</span><span class="fpage" tagx="fpage" title="fpage">1461</span><span class="lpage" tagx="lpage" title="lpage">1468</span><span class="pub-id"><a href="https://dx.doi.org/10.1038/ng.2007.60">10.1038/ng.2007.60</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17987029">17987029</a></span></span></li> <li tag="ref"><a name="bib45" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Salminen</span><span class="given-names" tagx="given-names" title="given-names">A</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Huuskonen</span><span class="given-names" tagx="given-names" title="given-names">J</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ojala</span><span class="given-names" tagx="given-names" title="given-names">J</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kauppinen</span><span class="given-names" tagx="given-names" title="given-names">A</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kaarniranta</span><span class="given-names" tagx="given-names" title="given-names">K</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Suuronen</span><span class="given-names" tagx="given-names" title="given-names">T</span></span></span><span class="year" tagx="year" title="year">2008</span><span class="mixed-article-title" title="mixed-article-title">Activation of innate immunity system during aging: nf-kb signaling is the molecular culprit of inflamm-aging</span><span class="source" tagx="source" title="source">Ageing Research Reviews</span><span class="volume" tagx="volume" title="volume">7</span><span class="fpage" tagx="fpage" title="fpage">83</span><span class="lpage" tagx="lpage" title="lpage">105</span><span class="pub-id"><a href="https://dx.doi.org/10.1016/j.arr.2007.09.002">10.1016/j.arr.2007.09.002</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17964225">17964225</a></span></span></li> <li tag="ref"><a name="bib46" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Seim</span><span class="given-names" tagx="given-names" title="given-names">I</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Fang</span><span class="given-names" tagx="given-names" title="given-names">X</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Xiong</span><span class="given-names" tagx="given-names" title="given-names">Z</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lobanov</span><span class="given-names" tagx="given-names" title="given-names">AV</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Huang</span><span class="given-names" tagx="given-names" title="given-names">Z</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ma</span><span class="given-names" tagx="given-names" title="given-names">S</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Feng</span><span class="given-names" tagx="given-names" title="given-names">Y</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Turanov</span><span class="given-names" tagx="given-names" title="given-names">AA</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zhu</span><span class="given-names" tagx="given-names" title="given-names">Y</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lenz</span><span class="given-names" tagx="given-names" title="given-names">TL</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gerashchenko</span><span class="given-names" tagx="given-names" title="given-names">MV</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Fan</span><span class="given-names" tagx="given-names" title="given-names">D</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hee Yim</span><span class="given-names" tagx="given-names" title="given-names">S</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Yao</span><span class="given-names" tagx="given-names" title="given-names">X</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Jordan</span><span class="given-names" tagx="given-names" title="given-names">D</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Xiong</span><span class="given-names" tagx="given-names" title="given-names">Y</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ma</span><span class="given-names" tagx="given-names" title="given-names">Y</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lyapunov</span><span class="given-names" tagx="given-names" title="given-names">AN</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Chen</span><span class="given-names" tagx="given-names" title="given-names">G</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kulakova</span><span class="given-names" tagx="given-names" title="given-names">OI</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Sun</span><span class="given-names" tagx="given-names" title="given-names">Y</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lee</span><span class="given-names" tagx="given-names" title="given-names">SG</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bronson</span><span class="given-names" tagx="given-names" title="given-names">RT</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Moskalev</span><span class="given-names" tagx="given-names" title="given-names">AA</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Sunyaev</span><span class="given-names" tagx="given-names" title="given-names">SR</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zhang</span><span class="given-names" tagx="given-names" title="given-names">G</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Krogh</span><span class="given-names" tagx="given-names" title="given-names">A</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wang</span><span class="given-names" tagx="given-names" title="given-names">J</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gladyshev</span><span class="given-names" tagx="given-names" title="given-names">VN</span></span></span><span class="year" tagx="year" title="year">2013</span><span class="mixed-article-title" title="mixed-article-title">Genome analysis reveals insights into physiology and longevity of the Brandt's bat Myotis brandtii</span><span class="source" tagx="source" title="source">Nature Communications</span><span class="volume" tagx="volume" title="volume">4</span><span class="elocation-id" tagx="elocation-id" title="elocation-id">2212</span><span class="pub-id"><a href="https://dx.doi.org/10.1038/ncomms3212">10.1038/ncomms3212</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23962925">23962925</a></span></span></li> <li tag="ref"><a name="bib47" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Shi</span><span class="given-names" tagx="given-names" title="given-names">L</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Oberdoerffer</span><span class="given-names" tagx="given-names" title="given-names">P</span></span></span><span class="year" tagx="year" title="year">2012</span><span class="mixed-article-title" title="mixed-article-title">Chromatin dynamics in DNA double-strand break repair</span><span class="source" tagx="source" title="source">Biochimica Et Biophysica Acta (BBA) - Gene Regulatory Mechanisms</span><span class="volume" tagx="volume" title="volume">1819</span><span class="fpage" tagx="fpage" title="fpage">811</span><span class="lpage" tagx="lpage" title="lpage">819</span><span class="pub-id"><a href="https://dx.doi.org/10.1016/j.bbagrm.2012.01.002">10.1016/j.bbagrm.2012.01.002</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22285574">22285574</a></span></span></li> <li tag="ref"><a name="bib48" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Shultz</span><span class="given-names" tagx="given-names" title="given-names">AJ</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Sackton</span><span class="given-names" tagx="given-names" title="given-names">TB</span></span></span><span class="year" tagx="year" title="year">2019</span><span class="mixed-article-title" title="mixed-article-title">Immune genes are hotspots of shared positive selection across birds and mammals</span><span class="source" tagx="source" title="source">eLife</span><span class="volume" tagx="volume" title="volume">8</span><span class="elocation-id" tagx="elocation-id" title="elocation-id">e41815</span><span class="pub-id"><a href="https://dx.doi.org/10.7554/eLife.41815">10.7554/eLife.41815</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/30620335">30620335</a></span></span></li> <li tag="ref"><a name="bib49" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Storey</span><span class="given-names" tagx="given-names" title="given-names">JD</span></span></span><span class="year" tagx="year" title="year">2003</span><span class="mixed-article-title" title="mixed-article-title">The positive false discovery rate: a bayesian interpretation and the q -value</span><span class="source" tagx="source" title="source">The Annals of Statistics</span><span class="volume" tagx="volume" title="volume">31</span><span class="fpage" tagx="fpage" title="fpage">2013</span><span class="lpage" tagx="lpage" title="lpage">2035</span><span class="pub-id"><a href="https://dx.doi.org/10.1214/aos/1074290335">10.1214/aos/1074290335</a></span></span></li> <li tag="ref"><a name="bib50" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Stout</span><span class="given-names" tagx="given-names" title="given-names">GJ</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Stigter</span><span class="given-names" tagx="given-names" title="given-names">EC</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Essers</span><span class="given-names" tagx="given-names" title="given-names">PB</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Mulder</span><span class="given-names" tagx="given-names" title="given-names">KW</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kolkman</span><span class="given-names" tagx="given-names" title="given-names">A</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Snijders</span><span class="given-names" tagx="given-names" title="given-names">DS</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">van den Broek</span><span class="given-names" tagx="given-names" title="given-names">NJ</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Betist</span><span class="given-names" tagx="given-names" title="given-names">MC</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Korswagen</span><span class="given-names" tagx="given-names" title="given-names">HC</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Macinnes</span><span class="given-names" tagx="given-names" title="given-names">AW</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Brenkman</span><span class="given-names" tagx="given-names" title="given-names">AB</span></span></span><span class="year" tagx="year" title="year">2013</span><span class="mixed-article-title" title="mixed-article-title">Insulin/IGF-1-mediated longevity is marked by reduced protein metabolism</span><span class="source" tagx="source" title="source">Molecular Systems Biology</span><span class="volume" tagx="volume" title="volume">9</span><span class="elocation-id" tagx="elocation-id" title="elocation-id">679</span><span class="pub-id"><a href="https://dx.doi.org/10.1038/msb.2013.35">10.1038/msb.2013.35</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23820781">23820781</a></span></span></li> <li tag="ref"><a name="bib51" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Subramanian</span><span class="given-names" tagx="given-names" title="given-names">A</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Tamayo</span><span class="given-names" tagx="given-names" title="given-names">P</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Mootha</span><span class="given-names" tagx="given-names" title="given-names">VK</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Mukherjee</span><span class="given-names" tagx="given-names" title="given-names">S</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ebert</span><span class="given-names" tagx="given-names" title="given-names">BL</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gillette</span><span class="given-names" tagx="given-names" title="given-names">MA</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Paulovich</span><span class="given-names" tagx="given-names" title="given-names">A</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Pomeroy</span><span class="given-names" tagx="given-names" title="given-names">SL</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Golub</span><span class="given-names" tagx="given-names" title="given-names">TR</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lander</span><span class="given-names" tagx="given-names" title="given-names">ES</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Mesirov</span><span class="given-names" tagx="given-names" title="given-names">JP</span></span></span><span class="year" tagx="year" title="year">2005</span><span class="mixed-article-title" title="mixed-article-title">Gene set enrichment analysis: a knowledge-based approach for interpreting genome-wide expression profiles</span><span class="source" tagx="source" title="source">PNAS</span><span class="volume" tagx="volume" title="volume">102</span><span class="fpage" tagx="fpage" title="fpage">15545</span><span class="lpage" tagx="lpage" title="lpage">15550</span><span class="pub-id"><a href="https://dx.doi.org/10.1073/pnas.0506580102">10.1073/pnas.0506580102</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16199517">16199517</a></span></span></li> <li tag="ref"><a name="bib52" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Sulak</span><span class="given-names" tagx="given-names" title="given-names">M</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Fong</span><span class="given-names" tagx="given-names" title="given-names">L</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Mika</span><span class="given-names" tagx="given-names" title="given-names">K</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Chigurupati</span><span class="given-names" tagx="given-names" title="given-names">S</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Yon</span><span class="given-names" tagx="given-names" title="given-names">L</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Mongan</span><span class="given-names" tagx="given-names" title="given-names">NP</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Emes</span><span class="given-names" tagx="given-names" title="given-names">RD</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lynch</span><span class="given-names" tagx="given-names" title="given-names">VJ</span></span></span><span class="year" tagx="year" title="year">2016</span><span class="mixed-article-title" title="mixed-article-title"><i>TP53</i> copy number expansion is associated with the evolution of increased body size and an enhanced DNA damage response in elephants</span><span class="source" tagx="source" title="source">eLife</span><span class="volume" tagx="volume" title="volume">5</span><span class="elocation-id" tagx="elocation-id" title="elocation-id">e11994</span><span class="pub-id"><a href="https://dx.doi.org/10.7554/eLife.11994">10.7554/eLife.11994</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/27642012">27642012</a></span></span></li> <li tag="ref"><a name="bib53" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Sutter</span><span class="given-names" tagx="given-names" title="given-names">NB</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bustamante</span><span class="given-names" tagx="given-names" title="given-names">CD</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Chase</span><span class="given-names" tagx="given-names" title="given-names">K</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gray</span><span class="given-names" tagx="given-names" title="given-names">MM</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zhao</span><span class="given-names" tagx="given-names" title="given-names">K</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zhu</span><span class="given-names" tagx="given-names" title="given-names">L</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Padhukasahasram</span><span class="given-names" tagx="given-names" title="given-names">B</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Karlins</span><span class="given-names" tagx="given-names" title="given-names">E</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Davis</span><span class="given-names" tagx="given-names" title="given-names">S</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Jones</span><span class="given-names" tagx="given-names" title="given-names">PG</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Quignon</span><span class="given-names" tagx="given-names" title="given-names">P</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Johnson</span><span class="given-names" tagx="given-names" title="given-names">GS</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Parker</span><span class="given-names" tagx="given-names" title="given-names">HG</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Fretwell</span><span class="given-names" tagx="given-names" title="given-names">N</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Mosher</span><span class="given-names" tagx="given-names" title="given-names">DS</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lawler</span><span class="given-names" tagx="given-names" title="given-names">DF</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Satyaraj</span><span class="given-names" tagx="given-names" title="given-names">E</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Nordborg</span><span class="given-names" tagx="given-names" title="given-names">M</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lark</span><span class="given-names" tagx="given-names" title="given-names">KG</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wayne</span><span class="given-names" tagx="given-names" title="given-names">RK</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ostrander</span><span class="given-names" tagx="given-names" title="given-names">EA</span></span></span><span class="year" tagx="year" title="year">2007</span><span class="mixed-article-title" title="mixed-article-title">A single IGF1 allele is a major determinant of small size in dogs</span><span class="source" tagx="source" title="source">Science</span><span class="volume" tagx="volume" title="volume">316</span><span class="fpage" tagx="fpage" title="fpage">112</span><span class="lpage" tagx="lpage" title="lpage">115</span><span class="pub-id"><a href="https://dx.doi.org/10.1126/science.1137045">10.1126/science.1137045</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17412960">17412960</a></span></span></li> <li tag="ref"><a name="bib54" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Tacutu</span><span class="given-names" tagx="given-names" title="given-names">R</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Thornton</span><span class="given-names" tagx="given-names" title="given-names">D</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Johnson</span><span class="given-names" tagx="given-names" title="given-names">E</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Budovsky</span><span class="given-names" tagx="given-names" title="given-names">A</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Barardo</span><span class="given-names" tagx="given-names" title="given-names">D</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Craig</span><span class="given-names" tagx="given-names" title="given-names">T</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Diana</span><span class="given-names" tagx="given-names" title="given-names">E</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lehmann</span><span class="given-names" tagx="given-names" title="given-names">G</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Toren</span><span class="given-names" tagx="given-names" title="given-names">D</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wang</span><span class="given-names" tagx="given-names" title="given-names">J</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Fraifeld</span><span class="given-names" tagx="given-names" title="given-names">VE</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">de Magalhães</span><span class="given-names" tagx="given-names" title="given-names">JP</span></span></span><span class="year" tagx="year" title="year">2018</span><span class="mixed-article-title" title="mixed-article-title">Human ageing genomic resources: new and updated databases</span><span class="source" tagx="source" title="source">Nucleic Acids Research</span><span class="volume" tagx="volume" title="volume">46</span><span class="fpage" tagx="fpage" title="fpage">D1083</span><span class="lpage" tagx="lpage" title="lpage">D1090</span><span class="pub-id"><a href="https://dx.doi.org/10.1093/nar/gkx1042">10.1093/nar/gkx1042</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/29121237">29121237</a></span></span></li> <li tag="ref"><a name="bib55" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wertheim</span><span class="given-names" tagx="given-names" title="given-names">JO</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Murrell</span><span class="given-names" tagx="given-names" title="given-names">B</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Smith</span><span class="given-names" tagx="given-names" title="given-names">MD</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kosakovsky Pond</span><span class="given-names" tagx="given-names" title="given-names">SL</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Scheffler</span><span class="given-names" tagx="given-names" title="given-names">K</span></span></span><span class="year" tagx="year" title="year">2015</span><span class="mixed-article-title" title="mixed-article-title">RELAX: detecting relaxed selection in a phylogenetic framework</span><span class="source" tagx="source" title="source">Molecular Biology and Evolution</span><span class="volume" tagx="volume" title="volume">32</span><span class="fpage" tagx="fpage" title="fpage">820</span><span class="lpage" tagx="lpage" title="lpage">832</span><span class="pub-id"><a href="https://dx.doi.org/10.1093/molbev/msu400">10.1093/molbev/msu400</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25540451">25540451</a></span></span></li> <li tag="ref"><a name="bib56" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">White</span><span class="given-names" tagx="given-names" title="given-names">RR</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Milholland</span><span class="given-names" tagx="given-names" title="given-names">B</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">de Bruin</span><span class="given-names" tagx="given-names" title="given-names">A</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Curran</span><span class="given-names" tagx="given-names" title="given-names">S</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Laberge</span><span class="given-names" tagx="given-names" title="given-names">RM</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">van Steeg</span><span class="given-names" tagx="given-names" title="given-names">H</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Campisi</span><span class="given-names" tagx="given-names" title="given-names">J</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Maslov</span><span class="given-names" tagx="given-names" title="given-names">AY</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Vijg</span><span class="given-names" tagx="given-names" title="given-names">J</span></span></span><span class="year" tagx="year" title="year">2015</span><span class="mixed-article-title" title="mixed-article-title">Controlled induction of DNA double-strand breaks in the mouse liver induces features of tissue ageing</span><span class="source" tagx="source" title="source">Nature Communications</span><span class="volume" tagx="volume" title="volume">6</span><span class="elocation-id" tagx="elocation-id" title="elocation-id">6790</span><span class="pub-id"><a href="https://dx.doi.org/10.1038/ncomms7790">10.1038/ncomms7790</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25858675">25858675</a></span></span></li> <li tag="ref"><a name="bib57" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Williams</span><span class="given-names" tagx="given-names" title="given-names">GC</span></span></span><span class="year" tagx="year" title="year">1957</span><span class="mixed-article-title" title="mixed-article-title">Pleiotropy, natural selection, and the evolution of senescence</span><span class="source" tagx="source" title="source">Evolution</span><span class="volume" tagx="volume" title="volume">11</span><span class="elocation-id" tagx="elocation-id" title="elocation-id">398</span><span class="pub-id"><a href="https://dx.doi.org/10.2307/2406060">10.2307/2406060</a></span></span></li> <li tag="ref"><a name="bib58" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Yang</span><span class="given-names" tagx="given-names" title="given-names">Z</span></span></span><span class="year" tagx="year" title="year">2007</span><span class="mixed-article-title" title="mixed-article-title">PAML 4: phylogenetic analysis by maximum likelihood</span><span class="source" tagx="source" title="source">Molecular Biology and Evolution</span><span class="volume" tagx="volume" title="volume">24</span><span class="fpage" tagx="fpage" title="fpage">1586</span><span class="lpage" tagx="lpage" title="lpage">1591</span><span class="pub-id"><a href="https://dx.doi.org/10.1093/molbev/msm088">10.1093/molbev/msm088</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17483113">17483113</a></span></span></li> <li tag="ref"><a name="bib59" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zhang</span><span class="given-names" tagx="given-names" title="given-names">J</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Yang</span><span class="given-names" tagx="given-names" title="given-names">JR</span></span></span><span class="year" tagx="year" title="year">2015</span><span class="mixed-article-title" title="mixed-article-title">Determinants of the rate of protein sequence evolution</span><span class="source" tagx="source" title="source">Nature Reviews Genetics</span><span class="volume" tagx="volume" title="volume">16</span><span class="fpage" tagx="fpage" title="fpage">409</span><span class="lpage" tagx="lpage" title="lpage">420</span><span class="pub-id"><a href="https://dx.doi.org/10.1038/nrg3950">10.1038/nrg3950</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/26055156">26055156</a></span></span></li> </ul> </div> </div>  </body></html>