<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<link rel="dns-prefetch" href="//d33xdlntwy0kbs.cloudfront.net" />
<link rel="shortcut icon" href="https://www.biorxiv.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<meta name="article_thumbnail" content="https://www.biorxiv.org/content/biorxiv/early/2019/12/30/2019.12.29.890608/embed/graphic-1.gif" />
<meta name="type" content="article" />
<meta name="category" content="article" />
<meta name="HW.identifier" content="/biorxiv/early/2019/12/30/2019.12.29.890608.atom" />
<meta name="HW.pisa" content="biorxiv;2019.12.29.890608v1" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="Tracking niche change through time: simultaneous inference of environmental niche evolution and estimation of contemporary niches" />
<meta name="DC.Identifier" content="10.1101/2019.12.29.890608" />
<meta name="DC.Date" content="2019-12-30" />
<meta name="DC.Publisher" content="Cold Spring Harbor Laboratory" />
<meta name="DC.Rights" content="© 2019, Posted by Cold Spring Harbor Laboratory. The copyright holder for this pre-print is the author. All rights reserved. The material may not be redistributed, re-used or adapted without the author's permission." />
<meta name="DC.AccessRights" content="restricted" />
<meta name="DC.Description" content="The environmental niche of a species is often studied from two different perspectives. Niche modelling correlates a species’ niche to known observations from contemporary species distribution. Niche evolution applies phylogenetic methods on summary statistics of species distribution to infer the patterns of changing niche over time. Even though the two areas of research are fundamentally linked and are typically both based on the same species distribution data, modelling the contemporary niche and inferring niche history are never combined in the same analytical framework. Here we provide a new way to combine both niche modelling and niche evolution, using distribution data and phylogenies. Doing so allows us to simultaneously infer the history of niche evolution and estimate the environmental niche of each tip species on the phylogeny. It also avoids some limitations of the separate methods, such as the failure of environmental niche models to account for the role of history in shaping current species distribution, and the use of unrealistic models of change in phylogenetic comparative methods. The novelty of our method is that it explicitly models three fundamental processes of niche evolution − adaptation, speciation, and dispersal − applying a reversible jump algorithm to infer the occurrences of these processes on phylogeny. Simulations demonstrate high accuracy of our method for estimating the environmental niche of tip species and reasonable power to identify the occurrences of the three processes on phylogeny. We also prove the efficacy of our approach using real-world data. We show that our method of combining niche modelling with niche evolution is far more effective at predicting salt tolerance in Australian Acacia species than using environmental niche modelling alone. We also show that we can correctly infer periods of evolutionary change in response to changing climate: our method identified two peak times in drought adaptation events in the evolutionary history of Australian Acacia , which correspond to the two drying periods in Australia during the Cenozoic. Therefore, we believe our method provides a promising approach to integrate different research areas in understanding species distribution." />
<meta name="DC.Contributor" content="Xia Hua" />
<meta name="DC.Contributor" content="Marcel Cardillo" />
<meta name="DC.Contributor" content="Lindell Bromham" />
<meta name="article:published_time" content="2019-12-30" />
<meta name="article:section" content="New Results" />
<meta name="citation_title" content="Tracking niche change through time: simultaneous inference of environmental niche evolution and estimation of contemporary niches" />
<meta name="citation_abstract" lang="en" content="&lt;h3&gt;Abstract&lt;/h3&gt;
&lt;p&gt;The environmental niche of a species is often studied from two different perspectives. Niche modelling correlates a species’ niche to known observations from contemporary species distribution. Niche evolution applies phylogenetic methods on summary statistics of species distribution to infer the patterns of changing niche over time. Even though the two areas of research are fundamentally linked and are typically both based on the same species distribution data, modelling the contemporary niche and inferring niche history are never combined in the same analytical framework. Here we provide a new way to combine both niche modelling and niche evolution, using distribution data and phylogenies. Doing so allows us to simultaneously infer the history of niche evolution and estimate the environmental niche of each tip species on the phylogeny. It also avoids some limitations of the separate methods, such as the failure of environmental niche models to account for the role of history in shaping current species distribution, and the use of unrealistic models of change in phylogenetic comparative methods. The novelty of our method is that it explicitly models three fundamental processes of niche evolution − adaptation, speciation, and dispersal − applying a reversible jump algorithm to infer the occurrences of these processes on phylogeny. Simulations demonstrate high accuracy of our method for estimating the environmental niche of tip species and reasonable power to identify the occurrences of the three processes on phylogeny. We also prove the efficacy of our approach using real-world data. We show that our method of combining niche modelling with niche evolution is far more effective at predicting salt tolerance in Australian &lt;i&gt;Acacia&lt;/i&gt; species than using environmental niche modelling alone. We also show that we can correctly infer periods of evolutionary change in response to changing climate: our method identified two peak times in drought adaptation events in the evolutionary history of Australian &lt;i&gt;Acacia&lt;/i&gt;, which correspond to the two drying periods in Australia during the Cenozoic. Therefore, we believe our method provides a promising approach to integrate different research areas in understanding species distribution.&lt;/p&gt;" />
<meta name="citation_journal_title" content="bioRxiv" />
<meta name="citation_publisher" content="Cold Spring Harbor Laboratory" />
<meta name="citation_publication_date" content="2019/01/01" />
<meta name="citation_mjid" content="biorxiv;2019.12.29.890608v1" />
<meta name="citation_id" content="2019.12.29.890608v1" />
<meta name="citation_public_url" content="https://www.biorxiv.org/content/10.1101/2019.12.29.890608v1" />
<meta name="citation_abstract_html_url" content="https://www.biorxiv.org/content/10.1101/2019.12.29.890608v1.abstract" />
<meta name="citation_full_html_url" content="https://www.biorxiv.org/content/10.1101/2019.12.29.890608v1.full" />
<meta name="citation_pdf_url" content="https://www.biorxiv.org/content/biorxiv/early/2019/12/30/2019.12.29.890608.full.pdf" />
<meta name="citation_doi" content="10.1101/2019.12.29.890608" />
<meta name="citation_num_pages" content="53" />
<meta name="citation_article_type" content="Article" />
<meta name="citation_section" content="New Results" />
<meta name="citation_firstpage" content="2019.12.29.890608" />
<meta name="citation_author" content="Xia Hua" />
<meta name="citation_author_institution" content="Macroevolution and Macroecology, Research School of Biology, Australian National University" />
<meta name="citation_author_institution" content="Mathematical Sciences Institute, Australian National University" />
<meta name="citation_author_email" content="xia.hua@anu.edu.au" />
<meta name="citation_author" content="Marcel Cardillo" />
<meta name="citation_author_institution" content="Macroevolution and Macroecology, Research School of Biology, Australian National University" />
<meta name="citation_author" content="Lindell Bromham" />
<meta name="citation_author_institution" content="Macroevolution and Macroecology, Research School of Biology, Australian National University" />
<meta name="citation_reference" content="Bennett T.H., Flowers T.J., Bromham L. 2013. Repeated evolution of salt-tolerance in grasses. Biol. Lett. 9:20130029." />
<meta name="citation_reference" content="Bromham L., Hua X., Cardillo M. 2016. Detecting macroevolutionary self-destruction from phylogenies. Syst. Biol. 65:109–127." />
<meta name="citation_reference" content="Bui E., Wilford J., de Caritat P. 2017. Predictive spatial modelling for mapping soil salinity at continental scale, EGU General Assembly, Vienna, Austria. doi: 10.13140/RG.2.2.25552.20483." />
<meta name="citation_reference" content="Byrne M., Yeates D.K., Joseph L., Kearney M., Bowler J., Williams M.A., Cooper S., Donnellan S.C., Keogh J.S., Leys R., Melville J., Murphy D.J., Porch N., Wyrwoll K.H. 2008. Birth of a biome: insights into the assembly and maintenance of the Australian arid zone biota. Mol. Ecol. 17:4398–4417." />
<meta name="citation_reference" content="Cang F.A., Wilson A.A., Wiens J.J. 2016. Climate change is projected to outpace rates of niche change in grasses. Biol. Lett. 12:20160368." />
<meta name="citation_reference" content="Castro-Insua A., Gómez-Rodríguez C., Wiens J.J., Baselga A. 2018. Climatic niche divergence drives patterns of diversification and richness among mammal families. Sci. Rep. 8:8781." />
<meta name="citation_reference" content="Catullo R.A., Ferrier S., Hoffmann A.A. 2015. Extending spatial modelling of climate change responses beyond the realized niche: estimating, and accommodating, physiological limits and adaptive evolution. Glob. Ecol. Biogeogr. 24:1192–1202." />
<meta name="citation_reference" content="Crisp M.D., Cook L.G. 2013. How was the Australian flora assembled over the last 65 million years? A molecular phylogenetic perspective. Annu. Rev. Ecol. Evol. Syst. 44:303–324." />
<meta name="citation_reference" content="Delzon S. 2015. New insight into leaf drought tolerance. Funct. Ecol. 29:1247–1249." />
<meta name="citation_reference" content="Dormann C.F., Schymanski S.J., Cabral J., Chuine I., Graham C., Hartig F., Kearney M., Morin X., Römermann C., Schröder B., Singer A. 2012. Correlation and process in species distribution models: bridging a dichotomy. J. Biogeogr. 39:2119–2131." />
<meta name="citation_reference" content="Eastman J.M., Alfaro M.E., Joyce P., Hipp A.L., Harmon L.J. 2011. A novel comparative method for identifying shifts in the rate of character evolution on trees. Evolution 65:3578–3589." />
<meta name="citation_reference" content="Felsenstein J. 1985. Phylogenies and the comparative method. Am. Nat. 12:1." />
<meta name="citation_reference" content="Fithian W., Elith J., Hastie T., Keith D.A. 2015. Bias correction in species distribution models: pooling survey and collection data for multiple species. Methods Ecol. Evol. 6:424–438." />
<meta name="citation_reference" content="FitzJohn R.G. 2012. Diversitree: comparative phylogenetic analyses of diversification in R. Methods Ecol. Evol. 3:1084–1092." />
<meta name="citation_reference" content="González-Orozco C.E., Pollock L.J., Thornhill A.H., Mishler B.D., Knerr N., Laffan S.W., Miller J.T., Rosauer D.F., Faith P.D., Nipperess D.A., Kujala H., Linke S., Butt N., Külheim C., Crisp M.D., Bruber B. 2016. Phylogenetic approaches reveal biodiversity threats under climate change. Nat. Clim. Change 6:1110–1115." />
<meta name="citation_reference" content="Green P.J. 1995. Reversible jump Markov Chain Monte Carlo computation and Bayesian model determination. Biometrika 82:711–732." />
<meta name="citation_reference" content="Hardy C.R. 2006. Reconstructing ancestral ecologies: challenges and possible solutions. Diversity Distrib. 12:7–19." />
<meta name="citation_reference" content="Hutchinson G.E. 1957. Concluding remarks. Cold Spring Harb. Symp. 22:415–427." />
<meta name="citation_reference" content="Holt R.D. 2009. Bringing the Hutchinsonian niche into the 21st century: ecological and evolutionary perspectives. Proc. Natl Acad. Sci. USA 106:19659–19665." />
<meta name="citation_reference" content="Kearney M., Porter W. 2009. Mechanistic niche modelling: combining physiological and spatial data to predict species’ ranges. Ecol. Lett. 12:334–350." />
<meta name="citation_reference" content="Kendall D.G. 1948. On the generalized ‘‘Birth-and-Death’’ process. Ann. Math. Statist. 19:1–15." />
<meta name="citation_reference" content="Kozak K., Wiens J. 2010. Accelerated rates of climatic-niche evolution underlie rapid species diversification. Ecol. Lett. 13:1378–1389." />
<meta name="citation_reference" content="Losos J.B. 2001. Seeing the forest for the trees: the limitations of phylogenies in comparative biology. Am. Nat. 177:709–727." />
<meta name="citation_reference" content="Maguire K.C., Shinneman D.J., Potter K.M., Hipkins V.D. 2018. Intraspecific niche models for ponderosa pine (Pinus ponderosa) suggest potential variability in population-level response to climate change. Syst. Biol. 67:965–978." />
<meta name="citation_reference" content="Meseguer A.S., Lobo J.M., Ree R., Beerling D.J., Sanmartin I. 2015. Integrating fossils, phylogenies, and niche models into biogeography to reveal ancient evolutionary history: the case of Hypericum (Hypericaceae). Syst. Biol. 64:215–232." />
<meta name="citation_reference" content="Miller J.T., Murphy, D.J., Brown G.K., Richardson D.M., González-Orozco C.E. 2011. The evolution and phylogenetic placement of invasive Australian Acacia species. Diversity Distrib. 17:848–860." />
<meta name="citation_reference" content="Miller J.T., Murphy D.J., Ho S.Y.W., Cantrill D.J., Seigler D. 2013. Comparative dating of Acacia: combining fossils and multiple phylogenies to infer ages of clades with poor fossil records. Aust. J. Bot. 61:436−445." />
<meta name="citation_reference" content="Mishler B.D., Knerr N., González-Orozco C.E., Thornhill A.H., Laffan S.W., Miller J.T. 2014. Phylogenetic measures of biodiversity and neo- and paleo-endemism in Australian Acacia. Nat. Comm. 5:4473." />
<meta name="citation_reference" content="Moore G.M. 2013. Adaptations of Australian tree species relevant to water scarcity in the urban forest. Arboric. Urban. For. 39:109−115." />
<meta name="citation_reference" content="Moore B.R., Ho□hna S., May M.R., Rannala B., Huelsenbeck J.P. 2016. Critically evaluating the theory and performance of Bayesian analysis of macroevolutionary mixtures. Proc. Natl. Acad. Sci. USA 113:9569−9574." />
<meta name="citation_reference" content="Morales-Castilla I., Davies T.J., Pearse W.D., Peres-Neto P. 2017. Combining phylogeny and co-occurrence to improve single species distribution models. Glob. Ecol. Biogeogr. 26:740–752." />
<meta name="citation_reference" content="Moray C., Hua X., Bromham L. 2015. Salt tolerance is evolutionarily labile in a diverse set of angiosperm families. BMC Evol. Biol. 15:90." />
<meta name="citation_reference" content="Moray C., Goolsby E., Bromham L. 2016. The phylogenetic association between salt tolerance and heavy metal hyperaccumulation in angiosperms. Evol. Biol. 43:119–130." />
<meta name="citation_reference" content="Mu□nkemu□ller T., Boucher F.C., Thuiller W., Lavergne S. 2015. Phylogenetic niche conservatism – common pitfalls and ways forward. Funct. Ecol. 29:627−639." />
<meta name="citation_reference" content="Owen C.L., Marshall D.C., Hill K.B.R., Simon C. 2017. How the aridification of Australia structured the biogeography and influenced the diversification of a large lineage of Australian cicadas. Syst. Biol. 66:569−589." />
<meta name="citation_reference" content="Paradis E., Schliep K. 2018. ape 5.0: an environment for modern phylogenetics and evolutionary analyses in R. Bioinformatics, bty633." />
<meta name="citation_reference" content="Peterson A.T. 2001. Predicting species’ geographic distributions based on ecological niche modeling. Condor 103:599–605." />
<meta name="citation_reference" content="Quintero I., Keil R., Jetz W., Crawford F.W. 2015. Historical biogeography using species geographic ranges. Syst. Biol. 64:1059–1073." />
<meta name="citation_reference" content="Rabosky D. L. 2014. Automatic detection of key innovations, rate shifts, and density-dependence on phylogenetic trees. PLOS ONE 9:e89543." />
<meta name="citation_reference" content="Rannala B. 2002. Identifiability of parameters in MCMC Bayesian inference of phylogeny. Syst. Biol. 51:754–760." />
<meta name="citation_reference" content="Renner I.W., Elith J., Baddeley A., Fithian W., Hastie T., Phillips S.J., Popovic G., Warton D.I. 2015. Point process models for presence-only analysis. Methods Ecol. Evol. 6:366– 379." />
<meta name="citation_reference" content="Revell L.J., Harmon L.J., Collar D.C. 2008. Phylogenetic signal, evolutionary process, and rate. Syst. Biol. 57:591–601." />
<meta name="citation_reference" content="Ronquist F., Klopfstein S., Vilhelmsen L., Schulmeister S., Murray D.L., Rasnitsyn A.P. 2012. A total-evidence approach to dating with fossils, applied to the early radiation of the Hymenoptera. Syst. Biol. 61:973–999." />
<meta name="citation_reference" content="Shi J.J., Rabosky D.L. 2015. Speciation dynamics during the global radiation of extant bats. Evolution 69:1528–1545." />
<meta name="citation_reference" content="Smith A.B., Godsoe W., Rodríguez-Sánchez F., Wang H., Warren D. 2018. Niche estimation above and below the species level. Trends Ecol. Evol. 34:260–273." />
<meta name="citation_reference" content="Sturges H.A. 1926. The choice of a class interval. J. Am. Stat. Assoc. 21:65–66." />
<meta name="citation_reference" content="Uyeda J.C., Harmon L.J. 2014. A novel Bayesian method for inferring and interpreting the dynamics of adaptive landscapes from phylogenetic comparative data. Syst. Biol. 63:902– 918." />
<meta name="citation_reference" content="Weber M.M., Stevens R.D., Diniz-Filho J.A., Grelle C.E. 2017. Is there a correlation between abundance and environmental suitability derived from ecological niche modelling? A meta-analysis. Ecography 40: 817–828." />
<meta name="citation_reference" content="Williams K.J., Belbin L., Austin M.P., Stein J.L., Ferrier S. 2012. Which environmental variables should I use in my biodiversity model? Int. J. Geogr. Inf. Sci. 26:2009–2047." />
<meta name="citation_reference" content="Theerawitaya C., Tisarum R., Samphumphuang T., Singh H.P., Cha-Um S., Kirdmanee C., Takabe T. 2015. Physio-biochemical and morphological characters of halophyte legume shrub, Acacia ampliceps seedlings in response to salt stress under greenhouse. Front. Plant Sci. 6:630." />
<meta name="citation_reference" content="Miah M.A. 2013. Salt tolerances of some mainland tree species select as through nursery screening. Pak. J. Biol. Sci. 16:945−949." />
<meta name="citation_reference" content="Patel A.D., Jadeja H., Pandey A.N. 2010. Effect of salinization of soil on growth, water status and nutrient accumulation in seedlings of Acacia auriculiformis (Fabaceae). J. Plant Nutrition 33: 914−932." />
<meta name="citation_reference" content="Craig G.F., Bell D.T., Atkins C.A., 1990. Response to salt and waterlogging stress of ten taxa of Acacia selected from naturally saline areas of Western Australia. Aust. J. Bot. 38:619−630." />
<meta name="citation_reference" content="Reichman S.M., Bellairs S.M., Mulligan D.R. 2006. The effects of temperature and salinity on Acacia harpophylla (brigalow) (Mimosaceae) germination. Rangeland J. 28:175−178." />
<meta name="citation_reference" content="Aswathappa N., Marcar N.E., Thompson L.A., 1987. Salt tolerance of Australian tropical and subtropical Acacias. In: Turnbull J. (Ed.), Australian Acacias in Developing Countries. ACIAR Proceedings, No. 16, pp. 70−73." />
<meta name="twitter:title" content="Tracking niche change through time: simultaneous inference of environmental niche evolution and estimation of contemporary niches" />
<meta name="twitter:site" content="@biorxivpreprint" />
<meta name="twitter:card" content="summary" />
<meta name="twitter:image" content="https://www.biorxiv.org/sites/default/files/images/biorxiv_logo_homepage7-5-small.png" />
<meta name="twitter:description" content="The environmental niche of a species is often studied from two different perspectives. Niche modelling correlates a species’ niche to known observations from contemporary species distribution. Niche evolution applies phylogenetic methods on summary statistics of species distribution to infer the patterns of changing niche over time. Even though the two areas of research are fundamentally linked and are typically both based on the same species distribution data, modelling the contemporary niche and inferring niche history are never combined in the same analytical framework. Here we provide a new way to combine both niche modelling and niche evolution, using distribution data and phylogenies. Doing so allows us to simultaneously infer the history of niche evolution and estimate the environmental niche of each tip species on the phylogeny. It also avoids some limitations of the separate methods, such as the failure of environmental niche models to account for the role of history in shaping current species distribution, and the use of unrealistic models of change in phylogenetic comparative methods. The novelty of our method is that it explicitly models three fundamental processes of niche evolution − adaptation, speciation, and dispersal − applying a reversible jump algorithm to infer the occurrences of these processes on phylogeny. Simulations demonstrate high accuracy of our method for estimating the environmental niche of tip species and reasonable power to identify the occurrences of the three processes on phylogeny. We also prove the efficacy of our approach using real-world data. We show that our method of combining niche modelling with niche evolution is far more effective at predicting salt tolerance in Australian Acacia species than using environmental niche modelling alone. We also show that we can correctly infer periods of evolutionary change in response to changing climate: our method identified two peak times in drought adaptation events in the evolutionary history of Australian Acacia , which correspond to the two drying periods in Australia during the Cenozoic. Therefore, we believe our method provides a promising approach to integrate different research areas in understanding species distribution." />
<meta name="og-title" property="og:title" content="Tracking niche change through time: simultaneous inference of environmental niche evolution and estimation of contemporary niches" />
<meta name="og-url" property="og:url" content="https://www.biorxiv.org/content/10.1101/2019.12.29.890608v1" />
<meta name="og-site-name" property="og:site_name" content="bioRxiv" />
<meta name="og-description" property="og:description" content="The environmental niche of a species is often studied from two different perspectives. Niche modelling correlates a species’ niche to known observations from contemporary species distribution. Niche evolution applies phylogenetic methods on summary statistics of species distribution to infer the patterns of changing niche over time. Even though the two areas of research are fundamentally linked and are typically both based on the same species distribution data, modelling the contemporary niche and inferring niche history are never combined in the same analytical framework. Here we provide a new way to combine both niche modelling and niche evolution, using distribution data and phylogenies. Doing so allows us to simultaneously infer the history of niche evolution and estimate the environmental niche of each tip species on the phylogeny. It also avoids some limitations of the separate methods, such as the failure of environmental niche models to account for the role of history in shaping current species distribution, and the use of unrealistic models of change in phylogenetic comparative methods. The novelty of our method is that it explicitly models three fundamental processes of niche evolution − adaptation, speciation, and dispersal − applying a reversible jump algorithm to infer the occurrences of these processes on phylogeny. Simulations demonstrate high accuracy of our method for estimating the environmental niche of tip species and reasonable power to identify the occurrences of the three processes on phylogeny. We also prove the efficacy of our approach using real-world data. We show that our method of combining niche modelling with niche evolution is far more effective at predicting salt tolerance in Australian Acacia species than using environmental niche modelling alone. We also show that we can correctly infer periods of evolutionary change in response to changing climate: our method identified two peak times in drought adaptation events in the evolutionary history of Australian Acacia , which correspond to the two drying periods in Australia during the Cenozoic. Therefore, we believe our method provides a promising approach to integrate different research areas in understanding species distribution." />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.biorxiv.org/sites/default/files/images/biorxiv_logo_homepage7-5-small.png" />
<meta name="citation_date" content="2019-12-30" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/10.1101/2019.12.29.890608v1.full.pdf" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/10.1101/2019.12.29.890608v1.full.txt" />
<link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/10.1101/2019.12.29.890608v1.ppt" />
<meta name="description" content="bioRxiv - the preprint server for biology, operated by Cold Spring Harbor Laboratory, a research and educational institution" />
<meta name="generator" content="Drupal 7 (http://drupal.org)" />
<link rel="canonical" href="https://www.biorxiv.org/content/10.1101/2019.12.29.890608v1" />
<link rel="shortlink" href="https://www.biorxiv.org/node/1089946" />
    <title>Tracking niche change through time: simultaneous inference of environmental niche evolution and estimation of contemporary niches | bioRxiv</title>  
    <link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__jMRAK66KMC1e4TQlwUNn3KiWVDC5AjueUAYEm1xBY_U__KSUjdT4jdcJ4qJz6fKY1K9WkYh1a5EcaWZxt_-zbTis__QJacVmveFyLIBSPkp21xL0ZvuDW9DMBrD2HB1P4Ry5I.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q5p1rr" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__icrP-Y0qw2NP73rJxxkUtrJta0z3VLAABtNjfSdEghA__rlN3rx0wOoswGNHZMeZM96bNn-3cLWHXTztGDoThNlw__QJacVmveFyLIBSPkp21xL0ZvuDW9DMBrD2HB1P4Ry5I.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__q48SDxVDWut_cH6dTPyQe7HjADuSLLmJVFztuiJEFrU__DJqddQGjm5dXCHsaiwNriGytKN5eevIQqyjdq65TZtE__QJacVmveFyLIBSPkp21xL0ZvuDW9DMBrD2HB1P4Ry5I.css" media="all" />
<style type="text/css" media="all">
/* <![CDATA[ */
#sliding-popup.sliding-popup-bottom,#sliding-popup.sliding-popup-bottom .eu-cookie-withdraw-banner,.eu-cookie-withdraw-tab{background:gray}#sliding-popup.sliding-popup-bottom.eu-cookie-withdraw-wrapper{background:transparent}#sliding-popup .popup-content #popup-text h1,#sliding-popup .popup-content #popup-text h2,#sliding-popup .popup-content #popup-text h3,#sliding-popup .popup-content #popup-text p,.eu-cookie-compliance-secondary-button,.eu-cookie-withdraw-tab{color:#fff !important}.eu-cookie-withdraw-tab{border-color:#fff}.eu-cookie-compliance-more-button{color:#fff !important}
/* ]]> */
</style>

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__QJacVmveFyLIBSPkp21xL0ZvuDW9DMBrD2HB1P4Ry5I.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__B6tGdsQSt1vT8d8j-19-oS4FKJe83RrSNBkjPs9jfos__6VkE5WcQmmQpSVTQx-xS2-zn_nlwFlDm-EN1HLJNjrA__QJacVmveFyLIBSPkp21xL0ZvuDW9DMBrD2HB1P4Ry5I.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__I6pBwZrmO51afTZVOjN83FdGH1QRb9t15lSLso0JVV4__F3sVKlGyHKCaAPI_WdiHD7l_-05B6b9bLdUwzi1me90__QJacVmveFyLIBSPkp21xL0ZvuDW9DMBrD2HB1P4Ry5I.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__QJacVmveFyLIBSPkp21xL0ZvuDW9DMBrD2HB1P4Ry5I.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__QJacVmveFyLIBSPkp21xL0ZvuDW9DMBrD2HB1P4Ry5I.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__CRdjIL0HKDL2T9slYvkTSTLypBBPjyoDhwBezBuDy7c__cHebPmPNdg2jyPmpyQpZWssJQ4cPrqfaabp5Or9KlkU__QJacVmveFyLIBSPkp21xL0ZvuDW9DMBrD2HB1P4Ry5I.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://d33xdlntwy0kbs.cloudfront.net/cshl_custom.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/all/libraries/modernizr/modernizr.min.js?q5p1rr"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__yOhrJUBmMpN-jsuQlJYmS8ovkQU6I6KInr295HrSTb0__c--zHhbuOWmkdWkJYXiNWDmkpDRPesmbBjGlQcyqBOY__QJacVmveFyLIBSPkp21xL0ZvuDW9DMBrD2HB1P4Ry5I.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var _prum=[['id', '52e95df3abe53d2e33000000'], ['mark', 'firstbyte',
      (new Date()).getTime()]]; (function() {
      var s=document.getElementsByTagName('script')[0],
      p=document.createElement('script');
      p.async='async'; p.src='//rum-static.pingdom.net/prum.min.js';s.parentNode.insertBefore(p,s);})();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js___KyvuEvfmewWxdOLBXTYsSUo8Pfm_DijV6fWtGvSL_k__HSKkTgIr7nDcdvQYewipf6v4ak6ZLUOQkwPj44wKouA__QJacVmveFyLIBSPkp21xL0ZvuDW9DMBrD2HB1P4Ry5I.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__3uCzzYE5jr6ly9nvsM8PRBRkhh4p0NWCMDja4EKBy4M__dmGGVYcB8LhnbR2iAmVW0ZyXI6QuFVYFRORa5XxglXs__QJacVmveFyLIBSPkp21xL0ZvuDW9DMBrD2HB1P4Ry5I.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__0zdUME0GLauYYyEch2tGPydIw-IG7Uizip0Qb4wUsig__3RgszrCgk4bOnTzo4vaQ-hrWzaG6oD19l-R9CQ1YQ-o__QJacVmveFyLIBSPkp21xL0ZvuDW9DMBrD2HB1P4Ry5I.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__W6wAjopaALytcBam7oT3WhEzllkWaB9R-8hazHyvShM__rK5YBIqBy9qt0di0Sn4DriBx9TJupdmv6uGmLJyZvDs__QJacVmveFyLIBSPkp21xL0ZvuDW9DMBrD2HB1P4Ry5I.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-45638026-1", {"cookieDomain":"auto"});ga("set", "page", location.pathname + location.search + location.hash);ga("send", "pageview");ga('create', 'UA-189672-38', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__1PtAOZ8x7l6QAi8D7X6Kj_Y7HcUOFzvWRvvYHUcCT98__U8IB92u_CnnGwJ-X_xaUa9lzsC8OzUfN0WkK2JQfirg__QJacVmveFyLIBSPkp21xL0ZvuDW9DMBrD2HB1P4Ry5I.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings,{"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"caYfBtsyt91B3OLypsa_9U5QtFbt_eEH8LWf-x5gP_Q","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"modules\/comment\/comment.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"modules\/forum\/forum.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"sites\/default\/modules\/biorxiv\/css\/biorxiv.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/flexible\/flexible.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_collection\/css\/highwire_subject_collection.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire.highwire-list-expand.css":1,"public:\/\/ctools\/css\/bab59090150caf0c64e27afa1b36eef1.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/contrib\/eu_cookie_compliance\/css\/eu_cookie_compliance.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"0":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_biorxiv_styles\/css\/biorxiv-styles.css.less":1,"https:\/\/d33xdlntwy0kbs.cloudfront.net\/cshl_custom.css":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_figures.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_tables.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"http:\/\/d33xdlntwy0kbs.cloudfront.net\/cshl_custom.js":1,"sites\/all\/modules\/contrib\/eu_cookie_compliance\/js\/eu_cookie_compliance.js":1,"sites\/default\/modules\/jnl_biorxiv_styles\/js\/adv-search-collection-toggle.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/eu_cookie_compliance\/js\/jquery.cookie-1.4.1.min.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/default\/modules\/jnl_biorxiv_styles\/js\/biorxiv-scripts.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_nps_survey\/js\/nps_survey.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"sites\/default\/modules\/contrib\/disqus\/disqus.js":1,"misc\/textarea.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/contrib\/service_links\/js\/twitter_button.js":1,"sites\/all\/modules\/contrib\/service_links\/js\/facebook_like.js":1,"sites\/all\/modules\/contrib\/service_links\/js\/google_plus_one.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.highwire-list-expand.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"1089946","apath":"\/biorxiv\/early\/2019\/12\/30\/2019.12.29.890608.atom","pisa":"biorxiv;2019.12.29.890608v1","processed":["highwire_math"],"markup":[{"requested":"full-text","variant":"full-text","view":"full","pisa":"biorxiv;2019.12.29.890608v1"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"disqus":{"domain":"biorxivstage","url":"https:\/\/www.biorxiv.org\/content\/10.1101\/2019.12.29.890608v1","title":"Tracking niche change through time: simultaneous inference of environmental niche evolution and estimation of contemporary niches","identifier":"node\/1089946"},"panels_ajax_pane":{"new-41":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022PqN+EBe4bRYv7Vqz0l1WbbFKfx899fpfp0PBuEOThzRxKntG9D8uLxpowawoHNpsoQWpxy1uvS0ZJBNswGl3Z\\\\\\\/TxJeieTCSy7fJ9rsmfLbeO6zlHyDorfNjVy3eRl977aB4bcGM1qEMaczCbDqSpbM\\\\\\\/AUeZ1yVTe4ynFe9+rC8ZNhusFRor0wzv\\\\\\\/kWCIEVS0JF1b1obhSk7AED2vTbA3izcX6ReLHQq4htNr5Rt4+liIVNaJ+tV4QLwA0L+vuNWpuhWB76\\\\\\\/ymPyjATpTVBNBx9521iccINwOE4yeZ\\\\\\\/FefK8XT4\\\\\\\/zIDgyVirUhrABbwayoilxylX\\\\\\\/Ot6cvttXD6HpUAK4ZIk9GQhBCuAXvAnQqdH\\\\\\\/wzW5IMHXFtjlYHNyCHbk4tbuIqx56kbUcp8+nq7wJQoSVo1Aqtj\\\\\\\/+Q7biW9ENA3RkQJrxlmJkbJgqdoRtstzZAPy7DRIzEs4KcaoC0zXpJc2rM67odMYUliTqXqYsNCrlZni1f8KN0DH1q5At9m10rJIk0JDM4mydatc+nNwUsb982B\\\\\\\/uB1OcYDuUIMrp7G6uJ17htcT5NbMAUZuue6Lpy9cC6addlPAhYIXp\\\\\\\/JZvSMZS41FKc7kZJrBL0K6mahChmG+GkTuAFifl2G3PY6HCzhQHAjb4HIFkzC5Y0z+DzQPz8RzN+7GDGledyulTcBPnwMsyBBR+QithLZvb8XS6HrsjIWBlaCctI5SCte+uAg4UqG+rjbFCL7MGCoeclqZTnGjClyhes9CVIH45mPHyjDMTODasZLKYyNLW7mQbLbbZr6sDmMa7z6AvRd8ocVFMDOOBY0HTrtdQ1roPAPrIORMN2f5kMzlJymcXoKpykZOEOHe65puyHyxyrqXnt98ZzYkMlCnU60SS3xWHvvkHRg0xIPyU6YPofrmhkWD6wVl8jfzntlqhP1jsDts7l0sBNRZZU1C3xKq8TxCSq1cw0LVXCo8S8QfUbfyfo97kTkpmBQmok2MYJsTu5mGXpbN3rT+bPGIYNg2GDFcwGfkNQeqPBeyaI7GPN4GC3lfrCc6kdDCiiAt2DFe\\\\\\\/FUTGqexqw7ix5hNmyIBcaxAfcsk4jfsU22S6GeUfnr+9axKOyaH9B\\\\\\\/EPTckeHJJZyAe1fKAZeLjkwrbZ9tKguX8+553lVtEKcFgcxzhOoxqlgfV6lPRZCVtEJ7zPSltaH2TTSe3LRNd\\\\\\\/MpCU+GaUs3+RHBi36HV7wgU511iedXAWJDUZLSBXMKcVF1v9QG6RnDRsTtzLhjzYzVp58ZbG0a4hr+q9aZ+GbnLxZ\\\\\\\/BY3mxKOUMUzc2iUh9I4S46ZT3V9Rpm24aGb1Y9xkLkf2PKvMdaTTC\\\u0022,\\\u0022iv\\\u0022:\\\u0022cW+8BrZgBI61KaaK8k7i4g==\\\u0022,\\\u0022salt\\\u0022:\\\u0022f9d8845712f72c9fee5db7d9ff111eab\\\u0022}\u0022,\u0022hmac\u0022:\u00229b44af312f1b7d050af3cbf46a5ea7ebc11d22d2081adb2985a238feed8035ff\u0022}"},"urlIsAjaxTrusted":{"\/content\/10.1101\/2019.12.29.890608v1.full":true},"ws_fl":{"width":100,"height":21},"ws_gpo":{"size":"","annotation":"","lang":"","callback":"","width":300},"color":{"logo":"https:\/\/www.biorxiv.org\/sites\/default\/files\/bioRxiv_article.jpg"},"highwire_list_expand":{"is_collapsed":"1"},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"eu_cookie_compliance":{"popup_enabled":1,"popup_agreed_enabled":0,"popup_hide_agreed":0,"popup_clicking_confirmation":1,"popup_scrolling_confirmation":false,"popup_html_info":"\u003Cdiv\u003E\n  \u003Cdiv class =\u0022popup-content info\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n      \u003Cp\u003EWe use cookies on this site to enhance your user experience. By clicking any link on this page you are giving your consent for us to set cookies.\u003C\/p\u003E\n    \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022agree-button eu-cookie-compliance-default-button\u0022\u003EContinue\u003C\/button\u003E\n                    \u003Cbutton type=\u0022button\u0022 class=\u0022find-more-button eu-cookie-compliance-more-button\u0022\u003EFind out more\u003C\/button\u003E\n          \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E","use_mobile_message":false,"mobile_popup_html_info":"\u003Cdiv\u003E\n  \u003Cdiv class =\u0022popup-content info\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n          \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022agree-button eu-cookie-compliance-default-button\u0022\u003EContinue\u003C\/button\u003E\n                    \u003Cbutton type=\u0022button\u0022 class=\u0022find-more-button eu-cookie-compliance-more-button\u0022\u003EFind out more\u003C\/button\u003E\n          \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E\n","mobile_breakpoint":"768","popup_html_agreed":"\u003Cdiv\u003E\n  \u003Cdiv class=\u0022popup-content agreed\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n      \u003Ch2\u003EThank you for accepting cookies\u003C\/h2\u003E\u003Cp\u003EYou can now hide this message or find out more about cookies.\u003C\/p\u003E    \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022hide-popup-button eu-cookie-compliance-hide-button\u0022\u003EHide\u003C\/button\u003E\n              \u003Cbutton type=\u0022button\u0022 class=\u0022find-more-button eu-cookie-compliance-more-button-thank-you\u0022 \u003EMore info\u003C\/button\u003E\n          \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E","popup_use_bare_css":false,"popup_height":"auto","popup_width":"100%","popup_delay":1000,"popup_link":"\/help\/cookie-policy","popup_link_new_window":1,"popup_position":null,"popup_language":"en","store_consent":false,"better_support_for_screen_readers":0,"reload_page":0,"domain":"","popup_eu_only_js":0,"cookie_lifetime":"365","cookie_session":false,"disagree_do_not_show_popup":0,"method":"default","whitelisted_cookies":"","withdraw_markup":"\u003Cbutton type=\u0022button\u0022 class=\u0022eu-cookie-withdraw-tab\u0022\u003EPrivacy settings\u003C\/button\u003E\n\u003Cdiv class=\u0022eu-cookie-withdraw-banner\u0022\u003E\n  \u003Cdiv class=\u0022popup-content info\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n      \u003Cp\u003E\u0026lt;h2\u0026gt;We use cookies on this site to enhance your user experience\u0026lt;\/h2\u0026gt;\u0026lt;p\u0026gt;You have given your consent for us to set cookies.\u0026lt;\/p\u0026gt;\u003C\/p\u003E\n    \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022eu-cookie-withdraw-button\u0022\u003EWithdraw consent\u003C\/button\u003E\n    \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E\n","withdraw_enabled":false},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1,"trackUrlFragments":1},"jnl_biorxiv_styles":{"defaultJCode":"biorxiv"},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-1089946 node-type-highwire-article context-content hw-default-jcode-biorxiv hw-article-type-article hw-article-category-new-results">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-M677548" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-M677548');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
        <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
    
  <div id="zone-branding" class="zone zone-branding clearfix print-display-block container-30">
    <div class="grid-15 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="bioRxiv" src="https://www.biorxiv.org/sites/default/files/bioRxiv_article.jpg" /></a>      </div>
                </div>
          </div>
</div><div class="grid-11 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-system block-menu block-main-menu block-system-main-menu odd block-without-title" id="block-system-main-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/" title="" class="" data-icon-position="" data-hide-link-title="0">Home</a></li>
<li class="leaf" role="menuitem"><a href="/about-biorxiv" class="" data-icon-position="" data-hide-link-title="0">About</a></li>
<li class="leaf" role="menuitem"><a href="/submit-a-manuscript" class="" data-icon-position="" data-hide-link-title="0">Submit</a></li>
<li class="last leaf" role="menuitem"><a href="/content/alertsrss" title="" class="" data-icon-position="" data-hide-link-title="0">ALERTS / RSS</a></li>
</ul></nav>    </div>
  </div>
</div><div class="block block-panels-mini block-biorxiv-search-box block-panels-mini-biorxiv-search-box even block-without-title" id="block-panels-mini-biorxiv-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-biorxiv_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/10.1101/2019.12.29.890608v1.full" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--2" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-y14WMGVjamcBPw-n3Qa49RzVTPWOH7BXkq1aRBCegDU" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2 advanced-search-link" >
  
      
  
  <div class="pane-content">
    <a href="/search">Advanced Search</a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix container-30">
  	      </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="1089946" id="node1089946" data-pisa="biorxiv;2019.12.29.890608v1" data-pisa-master="biorxiv;2019.12.29.890608" data-apath="/biorxiv/early/2019/12/30/2019.12.29.890608.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div  class="highwire-cite highwire-cite-highwire-article highwire-citation-biorxiv-article-top clearfix has-author-tooltip" >

      <span class="biorxiv-article-type">
        New Results    </span>
  
        <h1 class="highwire-cite-title" id="page-title">Tracking niche change through time: simultaneous inference of environmental niche evolution and estimation of contemporary niches</h1>  
      <div  class="highwire-cite-authors" ><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0"><span class="nlm-given-names">Xia</span> <span class="nlm-surname">Hua</span></span>, <span class="highwire-citation-author" data-delta="1"><span class="nlm-given-names">Marcel</span> <span class="nlm-surname">Cardillo</span></span>, <span class="highwire-citation-author" data-delta="2"><span class="nlm-given-names">Lindell</span> <span class="nlm-surname">Bromham</span></span></span></div>
  
      <div  class="highwire-cite-metadata" ><span  class="highwire-cite-metadata-doi highwire-cite-metadata"><span class="label">doi:</span> https://doi.org/10.1101/2019.12.29.890608 </span></div>
  
  
  </div>
<div id="hw-article-author-popups-node1089946" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Xia Hua </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-institution'>Macroevolution and Macroecology, Research School of Biology, Australian National University</span>, Canberra ACT 0200 <span class='nlm-country'>Australia</span></div><div class='author-affiliation'><span class='nlm-institution'>Mathematical Sciences Institute, Australian National University</span>, Canberra ACT 0200 <span class='nlm-country'>Australia</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Xia%2BHua%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Hua%20X&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AXia%2BHua%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:xia.hua@anu.edu.au" class="" data-icon-position="" data-hide-link-title="0">xia.hua@anu.edu.au</a></span></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Marcel Cardillo </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-institution'>Macroevolution and Macroecology, Research School of Biology, Australian National University</span>, Canberra ACT 0200 <span class='nlm-country'>Australia</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Marcel%2BCardillo%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Cardillo%20M&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AMarcel%2BCardillo%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Lindell Bromham </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-institution'>Macroevolution and Macroecology, Research School of Biology, Australian National University</span>, Canberra ACT 0200 <span class='nlm-country'>Australia</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Lindell%2BBromham%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Bromham%20L&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ALindell%2BBromham%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/10.1101/2019.12.29.890608v1" class="panels-ajax-tab-tab" data-panel-name="biorxiv_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:1089946" data-trigger="" data-url-enabled="1">Abstract</a><a href="/panels_ajax_tab/biorxiv_tab_art/node:1089946/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/10.1101/2019.12.29.890608v1.full-text" class="panels-ajax-tab-tab" data-panel-name="article_tab_full_text" data-target-id="highwire_article_tabs" data-entity-context="node:1089946" data-trigger="full-text" data-url-enabled="1">Full Text</a><a href="/panels_ajax_tab/article_tab_full_text/node:1089946/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/10.1101/2019.12.29.890608v1.article-info" class="panels-ajax-tab-tab" data-panel-name="biorxiv_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:1089946" data-trigger="article-info" data-url-enabled="1">Info/History</a><a href="/panels_ajax_tab/biorxiv_tab_info/node:1089946/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/10.1101/2019.12.29.890608v1.article-metrics" class="panels-ajax-tab-tab" data-panel-name="article_tab_metrics" data-target-id="highwire_article_tabs" data-entity-context="node:1089946" data-trigger="article-metrics" data-url-enabled="1">Metrics</a><a href="/panels_ajax_tab/article_tab_metrics/node:1089946/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/10.1101/2019.12.29.890608v1.full.pdf+html" class="panels-ajax-tab-tab" data-panel-name="biorxiv_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:1089946" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i> Preview PDF</a><a href="/panels_ajax_tab/biorxiv_tab_pdf/node:1089946/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="article_tab_full_text" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.biorxiv.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-article_tab_full_text"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-markup" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2 class="">Abstract</h2><p id="p-2">The environmental niche of a species is often studied from two different perspectives. Niche modelling correlates a species’ niche to known observations from contemporary species distribution. Niche evolution applies phylogenetic methods on summary statistics of species distribution to infer the patterns of changing niche over time. Even though the two areas of research are fundamentally linked and are typically both based on the same species distribution data, modelling the contemporary niche and inferring niche history are never combined in the same analytical framework. Here we provide a new way to combine both niche modelling and niche evolution, using distribution data and phylogenies. Doing so allows us to simultaneously infer the history of niche evolution and estimate the environmental niche of each tip species on the phylogeny. It also avoids some limitations of the separate methods, such as the failure of environmental niche models to account for the role of history in shaping current species distribution, and the use of unrealistic models of change in phylogenetic comparative methods. The novelty of our method is that it explicitly models three fundamental processes of niche evolution − adaptation, speciation, and dispersal − applying a reversible jump algorithm to infer the occurrences of these processes on phylogeny. Simulations demonstrate high accuracy of our method for estimating the environmental niche of tip species and reasonable power to identify the occurrences of the three processes on phylogeny. We also prove the efficacy of our approach using real-world data. We show that our method of combining niche modelling with niche evolution is far more effective at predicting salt tolerance in Australian <em>Acacia</em> species than using environmental niche modelling alone. We also show that we can correctly infer periods of evolutionary change in response to changing climate: our method identified two peak times in drought adaptation events in the evolutionary history of Australian <em>Acacia</em>, which correspond to the two drying periods in Australia during the Cenozoic. Therefore, we believe our method provides a promising approach to integrate different research areas in understanding species distribution.</p></div><p id="p-3">A species’ environmental niche describes the environmental conditions under which it is able to maintain positive population growth (the ‘fundamental niche’ of <a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">Hutchinson 1957</a>; <a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">Holt 2009</a>). Ecologists often have strong motivations for describing a species’ environmental niche, given observations made about its current distribution. One is to define the range of conditions under which the species can persist: this information is useful for current conservation planning (e.g. <a id="xref-ref-48-1" class="xref-bibr" href="#ref-48">Weber et al. 2017</a>) and for predicting future distributions given a change in environmental conditions (e.g. <a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">Catullo et al. 2015</a>; <a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">González-Orozco et al. 2016</a>). Prediction of the present-day environmental niche from data on species occurrence is referred to as environmental niche modelling (ENM) or species distribution modelling (SDM). In addition, we may be interested in reconstructing the history of change in a species niche over time, in order to describe the way lineages respond to environmental change (e.g. <a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">Cang et al. 2016</a>) and diversify to occupy a range of conditions (e.g.<a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">Castro-Insua et al. 2018</a>). Typically, these two areas of inference – modelling species present-day niches and reconstructing niche evolution over time – have been conducted separately. But these two areas are tightly linked, because the current niches of species are the product of evolutionary history, and reconstructions of niche evolution usually proceed from data on current species niches. Therefore, it is likely that a powerful approach for estimating current niches and reconstructing their underlying history is to do both simultaneously, in a single analytical framework (<a id="xref-ref-45-1" class="xref-bibr" href="#ref-45">Smith et al. 2018</a>).</p><p id="p-4">While there have been advances in incorporating evolutionary information to improve current niche estimation (<a id="xref-ref-45-2" class="xref-bibr" href="#ref-45">Smith et al. 2018</a>), such as estimating a species’ niche by aggregating its subspecies’ niches (<a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">Maguire et al. 2018</a>) or modelling niches of multiple species while accounting for their phylogenetic non-independence (<a id="xref-ref-31-1" class="xref-bibr" href="#ref-31">Morales-Castilla et al. 2017</a>), we still lack a method that can simultaneously infer the history of niche evolution and estimate current niches. In this study, we present a method that combines, in a single statistical framework, an environmental niche model (ENM) and a phylogenetic comparative method (PCM) to reconstruct niche evolution.</p><p id="p-5">ENM estimates a species environmental niche as a mathematical representation of the environmental conditions in the places where the species is known to occur or not occur (<a id="xref-ref-37-1" class="xref-bibr" href="#ref-37">Peterson 2001</a>). ENM methods make the strong assumption that a species current distribution is determined by its current environmental niche. But the observed distribution of a species may not be an exact reflection of its environmental niche. Firstly, the available occurrence data are typically an incomplete sample of the species true occurrences. A sparse sampling of occurrence records is used to estimate a continuous range (extent of occurrence), and those records may not be randomly sampled from the species distribution. ENM methods have been developed to model and account for sampling biases, for example most species are more likely to be recorded from areas close to population centres or roads (<a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">Fithian et al. 2015</a>). In addition, while many studies reliably record the presence of a species in a given sample location, explicit information on the absence of the species from sampling locations is lacking, so we must model the probability that the lack of occurrence data represents true absence or failure to sample. Secondly, the current distribution reflects not only the environmental tolerances of the species, but also reflects other ecological processes such as interspecific competition, as well as evolutionary history. The species may not fully occupy its potential range if it has not had the time or opportunity to disperse to all parts of the range, or if competitive exclusion has confined it to a suboptimal part of the environmental niche. ENM methods typically use observed species occurrences without accounting for the potential role of history and chance in determining current distributions of species. Combining ENM with niche evolution models allows us to explicitly model how history has shaped a species niche, leading to its current distribution.</p><p id="p-6">The evolution of niches has previously been modelled in the framework of phylogenetic comparative methods (PCM), which model the evolution of biological traits in a group of species based on their current trait values and phylogenetic relationships (<a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">Felsenstein 1985</a>). PCMs have been applied to analyses of niche evolution by considering the environmental niche to be a trait (<a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">Kozak and Wiens 2010</a>). However, modelling niches in this way raises two important problems. First, a species niche is often quantified as a distribution of values along one or more environmental axes, but PCMs were designed to work with point estimates rather than distributions (<a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">Hardy 2006</a>). Second, niche evolution involves complex (and still incompletely understood) biological processes, which may not be adequately characterized by the kinds of models (usually Brownian Motion or one of its variants) used to characterize trait evolution in PCMs. Models for trait evolution used in PCMs have been criticized as lacking biological realism (<a id="xref-ref-34-1" class="xref-bibr" href="#ref-34">Mu□nkemu□ller et al. 2015</a>) and being hard to interpret, because a range of different biological processes can lead to the identical outcomes as expected by these evolutionary models (<a id="xref-ref-42-1" class="xref-bibr" href="#ref-42">Revell et al. 2008</a>; <a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">Losos 2011</a>).</p><p id="p-7">We can overcome many of the limitations of these two approaches by combining the strengths of both. To combine ENM with niche evolution models and to address limitations in using PCM to study niche evolution, we introduce a novel process-based model for niche evolution. Our approach involves modelling explicitly the occurrence on phylogenies of three biological processes that are expected to drive niche evolution and influence species distribution: <em>in situ</em> adaptation, gradient speciation, and dispersal. By <em>in situ</em> adaptation, we refer to a process where the environmental niche of the species shifts to encompass a new range of conditions, for example, through the evolution of greater tolerance to drought. Gradient speciation is a process where different populations of an ancestral species diverged along an environmental axis and eventually evolve into distinct species. Dispersal is a process where a species extends its current distribution to areas where conditions would allow it to persist.</p><p id="p-8">Where these processes occur on the phylogeny, and how they influence the environmental niche of a species, define the species’ history of niche evolution. We take a Bayesian approach to simultaneously infer the history of niche evolution and the environmental niche of each tip species on the phylogeny. First, we propose a history of niche evolution. We then let the environmental niche of each tip species to evolve from the root to the tip following the history of niche evolution. The inferred environmental niche of each tip species allows us to assess the fit of the history of niche evolution by calculating the likelihood of the observed distribution of each species given its inferred niche using ENM. Last, we decide whether to accept the proposed history of niche evolution based on its fit to the distribution data. These steps repeat until we get a good approximation of the posterior of the history of niche evolution. The logic of our approach is similar to PCMs, but it extends PCM application to traits that are not point estimates and allows more biologically informative model of niche evolution.</p><p id="p-9">We use simulations to demonstrate the power of our method to correctly infer the history of niche evolution, and the accuracy of our method to estimate the environmental niche of each tip species. We also demonstrate how the method can be used to test hypotheses of niche evolution in real datasets, using the example of drought tolerance and salt tolerance in Australian members of the plant genus <em>Acacia</em>. Australian <em>Acacia</em> is a good case study for our method because they are widely distributed across Australia, often a dominant part of the flora, and include specialists that are adapted to many different environmental conditions, from desert taxa to alpine specialists to coastal dune species. Australian <em>Acacia</em> is also a data-rich case study, because it is a large (∼800 species) and widely-distributed species group, with geographic distribution data for nearly all species and a well-sampled phylogeny (<a id="xref-ref-28-1" class="xref-bibr" href="#ref-28">Mishler et al. 2014</a>). We focus on the evolution of drought tolerance in Australian <em>Acacia</em> to test the power of our method to identify processes of niche evolution. This allows us to test our inferred patterns of niche evolution against independent data on changing climate. Specifically, there have been three significant drying periods in Australia during the Cenozoic Era: around 28-23 Million years ago (Ma), 14-5 Ma, and shortly after 5 Ma (<a id="xref-ref-35-1" class="xref-bibr" href="#ref-35">Owen et al. 2017</a>). Our method uses only contemporary species distribution data, plus a phylogeny, so comparison of our results to this climate change history provides an independent test of the usefulness of our method. If our method can correctly infer the history of niche evolution, we would expect to see rapid evolution of climatic niches during these periods as <em>Acacia</em> species developed adaptations to drier climates. In addition, we use adaptation to high salinity in some <em>Acacia</em> species to evaluate the performance of our method in estimating the environmental niche of each <em>Acacia</em> species. Salt tolerance for several <em>Acacia</em> species has been well studied, so we can assemble a list of known salt-tolerant species. We can then test if our method identifies these salt-tolerant <em>Acacia</em> species as having significantly higher fitness under high salinity than the other <em>Acacia</em> species, according to their estimated niche along a salinity axis.</p><div class="section" id="sec-1"><h2 class="">M<span class="sc">ethods</span></h2><div id="sec-2" class="subsection"><h3>Basic model structure</h3><p id="p-10"><a id="xref-fig-1-1" class="xref-fig" href="#F1">Figure 1</a> summarizes the basic model structure of our approach. There are three major components of our method: the phylogenetic comparative method (PCM), the model of niche evolution, and the environmental niche model (ENM). We can express these three components in terms of conditional probability as follows. PCM estimates <em>p</em>(model|phylogeny), which is the probability of a certain history of niche evolution along a given time-scaled phylogeny. The model of niche evolution estimates <em>p</em>(niche|model,phylogeny), which is the probability of evolving the inferred contemporary niche of each tip species given the history of niche evolution and the phylogeny. ENM estimates <em>p</em>(presence and absence|niche), which is the probability of observing the presence and/or absence of a species given our inferred niche of the species – this information allows us to assess the fit of the inferred niche to the presence and/or absence locations of the species. The product of these probabilities gives the probability of observing the presence and/or absence of a species (the data) given its contemporary niche inferred from a history of niche evolution along a phylogeny (the inference). Given the prior for each parameter in the three components, we can make Bayesian inferences about both contemporary (tip) niches and the history of niche evolution, as:
<span class="disp-formula" id="disp-formula-1"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2019/12/30/2019.12.29.890608/embed/graphic-1.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2019/12/30/2019.12.29.890608/embed/graphic-1.gif"/></noscript></span>

</span>
</p><div id="F1" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/12/30/2019.12.29.890608/F1.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Illustration of the basic model structure. Our method uses a Bayesian framework that includes three major components: the phylogenetic comparative method (PCM), the model of niche evolution, and the environmental niche model (ENM). The PCM component estimates the location (t) of each in situ adaptation event and gradient speciation event on the phylogeny, by assuming that these events occur across a phylogeny under an inhomogeneous Poisson process with rate rA and rS. The distribution of these events on the phylogeny gives a history of niche evolution that we want to infer. The niche evolution component defines how three aspects of species niches (fundamental niche, realized niche, available niche) evolve through all the events (in situ adaptation, gradient speciation, dispersal) from the root to each tip along the phylogeny. This is done by estimating both local parameters (n, p, x) and global parameters (, d, root niches) for all the events that describe how niches evolve. The model infers the three niches of each tip species. The ENM component assesses the fit of the inferred niches to the presence and/or absence locations of the tip species (species distribution data). It also accounts for sampling bias of these locations by estimating how included factors affect sampling bias (). The fit of the inferred niches to the data then feedbacks to our inferences of the history of niche evolution and the niches of tip species." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-377686284" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Illustration of the basic model structure. Our method uses a Bayesian framework that includes three major components: the phylogenetic comparative method (PCM), the model of niche evolution, and the environmental niche model (ENM). The PCM component estimates the location (&lt;em&gt;t&lt;/em&gt;) of each &lt;em&gt;in situ&lt;/em&gt; adaptation event and gradient speciation event on the phylogeny, by assuming that these events occur across a phylogeny under an inhomogeneous Poisson process with rate &lt;em&gt;r&lt;/em&gt;&lt;sub&gt;A&lt;/sub&gt; and &lt;em&gt;r&lt;/em&gt;&lt;sub&gt;S&lt;/sub&gt;. The distribution of these events on the phylogeny gives a history of niche evolution that we want to infer. The niche evolution component defines how three aspects of species niches (fundamental niche, realized niche, available niche) evolve through all the events (&lt;em&gt;in situ&lt;/em&gt; adaptation, gradient speciation, dispersal) from the root to each tip along the phylogeny. This is done by estimating both local parameters (&lt;em&gt;n&lt;/em&gt;, &lt;em&gt;p&lt;/em&gt;, &lt;em&gt;x&lt;/em&gt;) and global parameters (, &lt;em&gt;d&lt;/em&gt;, root niches) for all the events that describe how niches evolve. The model infers the three niches of each tip species. The ENM component assesses the fit of the inferred niches to the presence and/or absence locations of the tip species (species distribution data). It also accounts for sampling bias of these locations by estimating how included factors affect sampling bias (). The fit of the inferred niches to the data then feedbacks to our inferences of the history of niche evolution and the niches of tip species.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 1." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2019/12/30/2019.12.29.890608/F1.medium.gif" width="440" height="112"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 1." src="https://www.biorxiv.org/content/biorxiv/early/2019/12/30/2019.12.29.890608/F1.medium.gif" width="440" height="112"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/12/30/2019.12.29.890608/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1." data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/12/30/2019.12.29.890608/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml"><span class="fig-label">Figure 1.</span> <p id="p-11" class="first-child">Illustration of the basic model structure. Our method uses a Bayesian framework that includes three major components: the phylogenetic comparative method (PCM), the model of niche evolution, and the environmental niche model (ENM). The PCM component estimates the location (<em>t</em>) of each <em>in situ</em> adaptation event and gradient speciation event on the phylogeny, by assuming that these events occur across a phylogeny under an inhomogeneous Poisson process with rate <em>r</em><sub>A</sub> and <em>r</em><sub>S</sub>. The distribution of these events on the phylogeny gives a history of niche evolution that we want to infer. The niche evolution component defines how three aspects of species niches (fundamental niche, realized niche, available niche) evolve through all the events (<em>in situ</em> adaptation, gradient speciation, dispersal) from the root to each tip along the phylogeny. This is done by estimating both local parameters (<em>n</em>, <em>p</em>, <em>x</em>) and global parameters (, <em>d</em>, root niches) for all the events that describe how niches evolve. The model infers the three niches of each tip species. The ENM component assesses the fit of the inferred niches to the presence and/or absence locations of the tip species (species distribution data). It also accounts for sampling bias of these locations by estimating how included factors affect sampling bias (). The fit of the inferred niches to the data then feedbacks to our inferences of the history of niche evolution and the niches of tip species.</p><div class="sb-div caption-clear"></div></div></div><p id="p-12">This gives the general framework of our method. The inputs are (presence and absence,phylogeny): distribution data with environmental conditions, such as the degree of aridity or salinity, at each presence location, and in some cases, reports of definite absence of the species from other locations, plus a time-scaled phylogeny of the taxa. The outputs are (niche,model): the posterior distribution of the environmental niche of each tip species and the posterior distribution of the history of niche evolution. <a id="xref-fig-1-2" class="xref-fig" href="#F1">Figure 1</a> summarizes the parameters to estimate in the Bayesian inference. We describe these parameters in each of the three components of our approach in the next three sections.</p></div><div id="sec-3" class="subsection"><h3>The model of niche evolution: process-based model</h3><p id="p-13">For the model of niche evolution, we developed a process-based model, in which three kinds of biological processes are involved in niche evolution and are modelled as events that occur on the phylogeny. The three processes are (1) <em>in situ</em> adaptation, (2) gradient speciation, and (3) dispersal. Each event has distinct properties modelled by local and global parameters. Local parameters are those that are estimated for each event, such as the location of the event on the phylogeny (see the PCM section) and the way that a species niche changes during the event (see details below). Global parameters are those that are shared among events, such as adaptation rate and dispersal rate (<a id="xref-fig-1-3" class="xref-fig" href="#F1">Fig. 1</a>). To make the model of niche evolution tractable, we assume that a species environmental niche evolves in a deterministic way according to the order and the type of events that occur along the path from the root to the tip, so <em>p</em>(niche|model,phylogeny)=1. Before we describe how a niche evolves in each type of event, it is important to note that in these processes, fitness is expressed as probability of a species persisting under particular environmental conditions. While described at the species level, it is assumed to result from differential survival and reproduction of individuals. For example, if the environment shifts, only those individuals of a species with tolerance to the new environments may be able to reproduce, and in this way the range of environmental conditions a species can tolerate can shift over time in response to environmental change.</p><p id="p-14">We define three aspects of a species niche (<a id="xref-fig-1-4" class="xref-fig" href="#F1">Fig. 1</a>) that change during the processes of niche evolution. One is the fundamental niche of the species, the range of conditions under which the species can persist. One is the available niche of the species, the range of conditions in the potential areas that the species could expand into, if it had the ability to persist under those conditions. Another is the realised niche of the species, the actual distribution of the species or the environmental conditions under which it is currently found. All these niches are specified with respect to a particular environmental factor (e.g. drought or salt), but can be extended to include multiple environmental factors. In our current model, we focus on the evolution of tolerance to extreme conditions, so environmental conditions are assumed to follow a continuous gradient on an environmental axis, becoming more extreme towards the right of the axis (<a id="xref-fig-2-1" class="xref-fig" href="#F2">Fig. 2</a>).</p><div id="F2" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/12/30/2019.12.29.890608/F2.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Illustration of the process-based model for niche evolution. There are three types of events: in situ adaptation (a), gradient speciation (b), and dispersal (c). Each type of event may change three aspects of niches along an environmental axis (with more extreme environmental conditions towards the right of the axis). In each plot, the solid curve is the niche before the event and the dashed curve is the niche after the event if any change happens (realized niche after the event is also shaded in grey). Bold letters are parameters to be estimated for the type of event. Each type of event is described by two parameters: dispersal rate (d) and the location of barrier on the environmental axis (x) for a dispersal event; adaptation rate () and the number of steps of environmental change (n) for an in situ adaptation event; adaptation rate ( ) and the location of species split on the environmental axis (p) for a gradient speciation event. n&#x2019; is the difference in the mean of the realized niche before and after an event. Together with adaptation rate ( ), they determine change in fundamental niche through and, where increase in leads to higher fitness under extreme condition and decrease in leads to less abrupt decline in fitness towards extreme condition." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-377686284" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Illustration of the process-based model for niche evolution. There are three types of events: &lt;em&gt;in situ&lt;/em&gt; adaptation (a), gradient speciation (b), and dispersal (c). Each type of event may change three aspects of niches along an environmental axis (with more extreme environmental conditions towards the right of the axis). In each plot, the solid curve is the niche before the event and the dashed curve is the niche after the event if any change happens (realized niche after the event is also shaded in grey). Bold letters are parameters to be estimated for the type of event. Each type of event is described by two parameters: dispersal rate (&lt;em&gt;d&lt;/em&gt;) and the location of barrier on the environmental axis (&lt;em&gt;x&lt;/em&gt;) for a dispersal event; adaptation rate () and the number of steps of environmental change (&lt;em&gt;n&lt;/em&gt;) for an &lt;em&gt;in situ&lt;/em&gt; adaptation event; adaptation rate ( ) and the location of species split on the environmental axis (&lt;em&gt;p&lt;/em&gt;) for a gradient speciation event. &lt;em&gt;n&#x2019;&lt;/em&gt; is the difference in the mean of the realized niche before and after an event. Together with adaptation rate ( ), they determine change in fundamental niche through and, where increase in leads to higher fitness under extreme condition and decrease in leads to less abrupt decline in fitness towards extreme condition.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 2." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2019/12/30/2019.12.29.890608/F2.medium.gif" width="440" height="259"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 2." src="https://www.biorxiv.org/content/biorxiv/early/2019/12/30/2019.12.29.890608/F2.medium.gif" width="440" height="259"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/12/30/2019.12.29.890608/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2." data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/12/30/2019.12.29.890608/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 2.</span> <p id="p-15" class="first-child">Illustration of the process-based model for niche evolution. There are three types of events: <em>in situ</em> adaptation (a), gradient speciation (b), and dispersal (c). Each type of event may change three aspects of niches along an environmental axis (with more extreme environmental conditions towards the right of the axis). In each plot, the solid curve is the niche before the event and the dashed curve is the niche after the event if any change happens (realized niche after the event is also shaded in grey). Bold letters are parameters to be estimated for the type of event. Each type of event is described by two parameters: dispersal rate (<em>d</em>) and the location of barrier on the environmental axis (<em>x</em>) for a dispersal event; adaptation rate () and the number of steps of environmental change (<em>n</em>) for an <em>in situ</em> adaptation event; adaptation rate ( ) and the location of species split on the environmental axis (<em>p</em>) for a gradient speciation event. <em>n’</em> is the difference in the mean of the realized niche before and after an event. Together with adaptation rate ( ), they determine change in fundamental niche through and, where increase in leads to higher fitness under extreme condition and decrease in leads to less abrupt decline in fitness towards extreme condition.</p><div class="sb-div caption-clear"></div></div></div><p id="p-16">Our model of niche evolution starts from the root of the phylogeny, so we need to parameterize the three aspects of the ancestral species niche at the root. The fundamental niche of the root species is assumed to follow <span class="inline-formula" id="inline-formula-1"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2019/12/30/2019.12.29.890608/embed/inline-graphic-1.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2019/12/30/2019.12.29.890608/embed/inline-graphic-1.gif"/></noscript></span></span> on the environmental axis, so that under the least extreme condition, fitness <em>f</em>(0) =1. Parameter <em>α</em> and <em>β</em> control the scale and the shape of the niche: the larger the <em>α</em>, the more spread out the niche; the smaller the <em>β</em>, the slower the fitness declines towards extreme conditions (<a id="xref-fig-2-2" class="xref-fig" href="#F2">Fig. 2</a>). The available niche of the root species is assumed to follow a normal distribution on the environmental axis. The realized niche of the root species equals the product of its fundamental niche and its available niche: that is, the distribution of the root species is at equilibrium, including all the areas of suitable habitat that the species is able to disperse to and occupy. Without niche evolution, the three niches are inherited by all the descendent lineages with no changes.</p><p id="p-17">Now we can describe how the three aspects of niches change during <em>in situ</em> adaptation, gradient speciation, or dispersal events. An <em>in situ</em> adaptation event (<a id="xref-fig-2-3" class="xref-fig" href="#F2">Fig. 2a</a>) is triggered by a change in the environmental conditions in the species distribution range. With the environmental change, the species no longer has high fitness in its distribution area. Some individuals die, some survive. This differential survival of individuals leads to shift in the species realized niche and fundamental niche along the environmental axis. To model this process, we first model the environmental change as a shift by <em>n</em> towards extreme conditions in both the species’ realized niche and available niche (<a id="xref-fig-2-4" class="xref-fig" href="#F2">Fig. 2a</a>). In other words, the environmental change influences both the species’ distribution area and the areas it is able to disperse to. Then we model the species new realized niche as the product of the species fundamental niche and the shifted realized niche, because the species fundamental niche describes differential survival of individuals under different environmental conditions and the shifted realized niche describes the new environmental conditions in the species distribution area before the species disperses to other areas (dispersal is modelled as a separate event). As a result, the final amount of shift from the old realized niche to the new realized niche is not <em>n</em> and we denote it as <em>n</em>′ in <a id="xref-fig-2-5" class="xref-fig" href="#F2">Figure 2a</a>. Along with the change in the realized niche, the species fundamental niche also evolves into higher fitness in extreme conditions. According to our parameterization of the fundamental niche, higher <em>α</em> leads to higher fitness in extreme conditions and lower <em>β</em> leads to less abrupt decline in fitness towards extreme conditions (<a id="xref-fig-2-6" class="xref-fig" href="#F2">Fig. 2a</a>), so increasing fitness in extreme conditions is reflected in increasing <em>α</em> values and decreasing <em>β</em> values. So, we model changes in the fundamental niche as the amount of change in <em>α</em> and <em>β</em>, which is <em>γn</em>′ and <em>γn</em>′ on a logarithmic scale, where <em>γ</em> is adaptation rate. A species with higher adaptation rate evolves its fundamental niche faster in response to environmental deterioration, by evolving higher <em>α</em> and lower <em>β</em>.</p><p id="p-18">When a gradient speciation event occurs (<a id="xref-fig-2-7" class="xref-fig" href="#F2">Fig. 2b</a>), two populations of an ancestral species become distinct species, each occupying different conditions along an environmental axis. In <a id="xref-fig-2-8" class="xref-fig" href="#F2">Figure 2b</a>, where the ancestral species splits on the axis is determined by parameter <em>p</em>, such that a proportion <em>p</em> of individuals on the right of the ancestral species realized niche become a species distinct from the remaining 1-<em>p</em> individuals on the left of the realized niche. One new species consists of individuals with higher fitness under the extreme end of the conditions that the ancestral species could tolerate. The other new species will have, on average, lower fitness under these extreme conditions. For both species, the amount of change in <em>α</em> and <em>β</em> during gradient speciation is <em>γn</em>′ and –<em>γn</em>′ on the logarithmic scale, where <em>n</em>′ is difference in the mean of realized niche between the new species and the ancestral species. A gradient speciation event can occur at any node in the phylogeny. When this happens, the two daughter species are randomly assigned to the two descendant lineages in the phylogeny, based on the sign of <em>p</em>. With a non-zero extinction rate, a gradient speciation event can also occur along a branch in the phylogeny with one of the two daughter species going extinct. When this happens, the gradient speciation effectively leads to a punctuated change in the realized niche and the fundamental niche of the ancestral species. The sign of parameter <em>p</em> decides which species remains and so the other species is regarded as extinct.</p><p id="p-19">Between any two events, either <em>in situ</em> adaptation or gradient speciation, dispersal occurs (<a id="xref-fig-2-9" class="xref-fig" href="#F2">Fig. 2c</a>), which is a continuous process of a species to extend its realized niche along the environmental axis until it reaches equilibrium. The species is at equilibrium when its realized niche equals the product of its fundamental niche and its available niche. Dispersal rate (<em>d</em>) defines how fast a species reaches the equilibrium. We consider dispersal after each <em>in situ</em> adaptation event or each gradient speciation event, because these events cause a lag between the species realized niche and the available niche during which the species distribution is not at equilibrium. Since we parameterize the available niche as a continuous distribution, which does not allow any discrete barrier for dispersal, we introduce another parameter <em>x</em> to model a dispersal barrier by truncating the available niche under non-extreme conditions (truncating the side left to <em>x</em> in <a id="xref-fig-2-10" class="xref-fig" href="#F2">Fig. 2c</a>). We only model dispersal barriers under non-extreme conditions, because a species distribution under extreme conditions is constrained by the fundamental niche and so little signal for the presence of dispersal barrier under extreme conditions can be detected. The mechanism for a barrier is not important for our model, instead we treat <em>x</em> as a nuisance parameter. For example, we model <em>x</em> and <em>p</em> as two independent parameters, but this does not exclude the possibility that the location of the barrier (<em>x</em>) may coincide with the location where species splits on the environmental axis (<em>p</em>) during a gradient speciation event, which may suggest competitive exclusion during the speciation event.</p><p id="p-20">In summary, our process-based model includes three types of processes: dispersal, <em>in situ</em> adaptation, and gradient speciation. Each type of process is described by two parameters: dispersal rate (<em>d</em>) and the location of barriers on the environmental axis (<em>x</em>) for a dispersal event; adaptation rate (<em>γ</em>) and the amount of environmental change (<em>n</em>) for an <em>in situ</em> adaptation event; adaptation rate (<em>γ</em>) and the location of species split on the environmental axis (<em>p</em>) for a gradient speciation event (<a id="xref-fig-1-5" class="xref-fig" href="#F1">Fig. 1</a> and <a id="xref-fig-2-11" class="xref-fig" href="#F2">Fig. 2</a>). To reduce the risk of over-parameterization, dispersal rate and adaptation rate are assumed to be constant over time and across lineages, so they are global parameters (<a id="xref-fig-1-6" class="xref-fig" href="#F1">Fig. 1</a>). Each <em>in situ</em> adaptation event or gradient speciation event is assumed to be a discrete event, occurring at a specific time point on the phylogeny. Dispersal event is a continuous process, happening in-between two discrete events. So, for each <em>in situ</em> adaptation event and gradient speciation event, we have an additional parameter (<em>t</em>) to describe the location on the phylogeny when the event occurs (<a id="xref-fig-1-7" class="xref-fig" href="#F1">Fig. 1</a>). Last, we have two parameters (mean and variance of a normal distribution) to define the available niche and two parameters (<em>α</em> and <em>β</em>) to define the fundamental niche of the ancestral species at the root of the phylogeny (<a id="xref-fig-1-8" class="xref-fig" href="#F1">Fig. 1</a>).</p></div><div id="sec-4" class="subsection"><h3>ENM: inhomogeneous Poisson point process</h3><p id="p-21">The model described above shows how we model the evolution of the three aspects of niche forwards in time along the phylogeny, starting from the niches at the root, evolving through a series of events of the three different biological processes, resulting in the niches at the tips. The three niches of each tip are then used to calculate <em>p</em>(presence and absence|niche) using ENM. In principle, any niche modelling method (ENM) can be used to calculate <em>p</em>(presence and absence|niche). Here we use an inhomogeneous Poisson point process (<a id="xref-ref-41-1" class="xref-bibr" href="#ref-41">Renner et al. 2015</a>), because this process can account for sampling bias, and so is applicable to both presence/absence data or presence-only data without generating pseudo-absence locations (<a id="xref-ref-13-2" class="xref-bibr" href="#ref-13">Fithian et al. 2015</a>). The process models the number of recorded presences of a species per unit area as a Poisson random variable. The inferred rate of recording a presence of the species in the area depends on the abundance of the species in that area and the sampling intensity in that area (see details in <a id="xref-ref-13-3" class="xref-bibr" href="#ref-13">Fithian et al. 2015</a>). Below, we describe how the three aspects of niches are linked to parameters in the ENM, assuming equal sampling intensity across the area (unequal sampling intensity is accounted for later in the Case study section).</p><p id="p-22">We can start by considering a dataset where we have presence-only data on distribution: that is, where our knowledge of species niches are informed by records of that species at particular sampling locations for which we can derive relevant environmental data. In brief, the log-likelihood of the presence-only data of each tip species is
<span class="disp-formula" id="disp-formula-2"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2019/12/30/2019.12.29.890608/embed/graphic-4.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2019/12/30/2019.12.29.890608/embed/graphic-4.gif"/></noscript></span>

</span>
, where <em>I<sub>PO</sub></em> is the set of all the species presence-only locations. In the first term, <span class="inline-formula" id="inline-formula-2"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2019/12/30/2019.12.29.890608/embed/inline-graphic-2.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2019/12/30/2019.12.29.890608/embed/inline-graphic-2.gif"/></noscript></span></span> is the realized niche, or the rate of recording a presence of the species under the environmental condition at the <em>i</em><sup>th</sup> presence location. The second term is the integral of the product between the fundamental niche (<em>λ<sub>f</sub></em>) and the available niche (<em>λ<sub>a</sub></em>), and 𝒟 is the geographic area of the available niche. This term describes the overall probability of recording a presence of the species in the species available niche. Often, 𝒟 cannot be measured accurately without knowing the dispersal ability of a species. However, it can be used to adjust how informative presence-only locations are for inferring the species fundamental niche. For example, a species with widely scattered presence records has a higher 𝒟 value than a species with more spatially concentrated records, so in the species fundamental niche, the same amount of decrease in fitness in conditions not in the presence locations will increase the model likelihood more for the species with higher 𝒟 value.</p><p id="p-23">Now we can consider the case where in addition to presence data (indicating points where the species is known to occur), we also have absence data (locations where the species was actively sought but not found). Absence of a species is often inferred from surveys within a local area, such as quadrat sampling. The log-likelihood of the presence-absence data of each tip species is
<span class="disp-formula" id="disp-formula-3"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2019/12/30/2019.12.29.890608/embed/graphic-5.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2019/12/30/2019.12.29.890608/embed/graphic-5.gif"/></noscript></span>

</span>
, where <em>I<sub>PA</sub></em> is the set of all the presence and absence locations of the species, with y<em><sub>i</sub></em> = 1 if the species is present at the <em>i</em><sup>th</sup> location and y<em><sub>i</sub></em> = 0 if the species is absent at the <em>i</em><sup>th</sup> location. We don’t need to account for the available niche here, because the survey is complete and so wherever the species does not occur, it is assumed to be absent.</p><p id="p-24">To make our model analytically tractable, we converted the continuous environmental axis into bins. The minimum and maximum bins are determined by the minimum and maximum values of environmental conditions in the study area. The three aspects of niche were converted into discrete distributions using the probability that values from these niche distributions fall in the range of each of the bins. For the bin containing the minimum or the maximum value, it is the probability that values from these niche distributions are lower or higher than the minimum or the maximum value.</p></div><div id="sec-5" class="subsection"><h3>PCM: reversible-jump model</h3><p id="p-25">Our model of niche evolution includes three biological processes: <em>in situ</em> adaptation, gradient speciation, and dispersal. A phylogenetic comparative method (PCM) is needed to estimate the number and the location of each type of event on the phylogeny. Here we assume events occur across a phylogeny under an inhomogeneous Poisson process and use reversible jump Markov Chain Monte Carlo (rjMCMC; <a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">Green 1995</a>) to implement the process in a Bayesian framework. Several PCM studies have used rjMCMC to study jumps in evolutionary processes of phenotypic evolution (e.g., <a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">Eastman et al. 2011</a>; <a id="xref-ref-47-1" class="xref-bibr" href="#ref-47">Uyeda and Harmon 2014</a>) and diversification (<a id="xref-ref-39-1" class="xref-bibr" href="#ref-39">Rabosky et al. 2014</a>). In our implementation, each jump is an occurrence of a discrete event: an <em>in situ</em> adaptation event or a gradient speciation event. As in these previous studies, our implementation involves several steps: 1) adding a new event to the model, 2) removing an existing event from the model, 3) updating the location of an existing event on the phylogeny, 4) updating parameters specific to that event, 5) updating parameters shared among events, 6) updating the occurrence rate of an event. Details on these steps are given in the Appendices.</p><p id="p-26">To calculate <em>p</em>(model|phylogeny), we assume <em>in situ</em> adaptation events occur randomly according to a homogeneous Poisson process with rate <em>r<sub>A</sub></em>, and gradient speciation events occur randomly according to an inhomogeneous Poisson process with overall rate <em>r<sub>S</sub></em> (<a id="xref-fig-1-9" class="xref-fig" href="#F1">Fig. 1</a>). The occurrence of gradient speciation is inhomogeneous because a speciation event occurring along a branch must be accompanied by an extinction event and the occurrence probability of the extinction event depends on the time (<em>t</em>) when the speciation event occurs, which is <span class="inline-formula" id="inline-formula-3"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2019/12/30/2019.12.29.890608/embed/inline-graphic-3.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2019/12/30/2019.12.29.890608/embed/inline-graphic-3.gif"/></noscript></span></span> (<a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">Kendall 1948</a>), where <em>λ</em> is speciation rate and <em>μ</em> is extinction rate. Given that a gradient speciation event occurs, the probability of the event occurring at a node relative to the probability of the event occurring along a branch is the number of nodes in the phylogeny divided by the integral of <em>λp<sub>E</sub>(t)</em> over all the branch lengths of the phylogeny. When a gradient speciation event occurs at a node, it has equal probability to occur at any node in the phylogeny. When a gradient speciation event occurs on a branch, the relative probability that the event occurs on a specific branch depends on the integral of <em>p<sub>E</sub>(t)</em> from the starting and the ending times of that branch, and the relative probability of the event to occur at time <em>t</em> in that branch is <em>p<sub>E</sub>(t)</em>.</p></div><div id="sec-6" class="subsection"><h3>Simulation study</h3><p id="p-27">We performed a simulation study to assess the performance of our method given the assumptions that species niches evolve according to our process-based model, and processes occur across phylogenies under the inhomogeneous Poisson process. First, we generated simulated datasets by evolving niches along phylogenies, then sampling locations from the resulting realized niche of each tip species to give species presence and absence data. Phylogenies were simulated under a birth-death process using the R package diversitree (<a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">FitzJohn 2012</a>) with 100 tips, a birth rate of 0.1, and a death rate of 0.03. All the simulated phylogenies had the same available niche along an arbitrary environmental axis at the root, with mean of 1 and standard deviation of 2, and the same fundamental niche at the root, with <em>α</em> of 3 and <em>β</em> of 2. On each simulated phylogeny, we randomly placed 20 <em>in situ</em> adaptation events with equal probability of placement at any location on the phylogeny, and we placed 20 gradient speciation events with different probabilities of different locations, as described above for <em>p</em>(model|phylogeny). Parameters specific to each event (<em>x, n, p</em>) were randomly drawn from their corresponding prior distributions (see Appendices). Species niches were then evolved through the randomly placed events along the path from the root to each tip according to our process-based model. This produced the available niche, the realized niche, and the fundamental niche of each tip species. During this process, the environmental axis was discretised into 10 bins and dispersal rate was fixed to 1 across the phylogeny. Two simulation schemes were applied to adaption rate (γ). In one scheme, <em>γ</em> was fixed to 0.03 across the phylogeny. In the other scheme, <em>γ</em> on each branch was randomly drawn from an exponential distribution with mean of 0.03, so that different lineages may have different adaptation rates. We generate two sets of sample data for the analysis. One dataset is presence-only data, in which presence-only locations were randomly drawn for each tip species from the species realized niche (<em>λ<sub>r</sub></em>). The number of presence-only locations is drawn from a Poisson distribution with mean equal to the overall probability of recording a presence of the species in the species available niche. The other dataset is presence and absence data, in which 1000 presence-absence locations were randomly drawn for each tip species with equal probability from each bin of the environmental axis, of which the <em>i</em><sup>th</sup> location has <span class="inline-formula" id="inline-formula-4"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2019/12/30/2019.12.29.890608/embed/inline-graphic-4.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2019/12/30/2019.12.29.890608/embed/inline-graphic-4.gif"/></noscript></span></span> probability to be an absence location.</p><p id="p-28">For each simulated phylogeny and the sampled presence-only and presence-absence data of the tip species on the phylogeny, we ran two independent MCMCs for 10<sup>6</sup> generations, with a thinning interval of 1000 and the first 600,000 generations discarded as burnin. We tested two aspects of the performance of our method: (1) accuracy of estimating the fundamental niche of each tip species, and (2) power to correctly identify the occurrences of <em>in situ</em> adaptation events and gradient speciation events on the phylogeny. To assess the accuracy of our method to estimate the fundamental niche of each tip species, we first estimated, for each simulation, the fundamental niche of each tip species as the average of the values of parameters <em>α</em> and <em>β</em> of each tip species in the posterior samples. We then compared these estimated values of parameters <em>α</em> and <em>β</em> of each tip species to their corresponding values in the simulation, using linear regression. Last, we summarized the regression results over all the simulations by the slope estimate and the R squared of the regression model. Slope values close to 1 and R squared values close to 1 indicate a high level of accuracy in our estimation of the fundamental niche of tip species.</p><p id="p-29">To assess the power of our method to identify <em>in situ</em> adaptation events, we first calculated for each branch the posterior probability of at least one <em>in situ</em> adaptation event occurring on each branch, by counting the proportion of posterior samples with <em>in situ</em> adaptation events on that branch. We then identified branches that had significantly higher posterior probability than expected if all the branches had equal probability of having adaptation events. Following <a id="xref-ref-44-1" class="xref-bibr" href="#ref-44">Shi and Raboksy (2015)</a>, a branch had significantly higher posterior probability than expected, if its marginal odds ratio was larger than a threshold arbitrarily set to 5. This is essentially hypothesis testing based on Bayes factor for a model with adaptation events on the branch (against a model without adaptation events on the branch), and Bayes factor larger than 5 is generally considered as substantial evidence for the model. Next, we calculated the power of our method as the proportion of branches with true adaptation events that passed the test: in other words, how many branches with true adaptation events we have identified. We also calculated the false positive rate of our method as the proportion of branches that passed the test but with no true adaptation events: in other words, how many inferred adaptation events were not true adaptation events.</p><p id="p-30">Last, we wished to find out what kind of branch is more likely to pass the test. We expected that events that occurred more recently, or events with larger influence on niche evolution, are more likely to be correctly identified by our method, so we measured two properties of each branch. For a branch with true adaptation events, one property is the average of the ages of all the <em>in situ</em> adaptation events placed on the branch during simulation. The other property is the sum of the value of parameter <em>n</em> over all the <em>in situ</em> adaptation events placed on the branch during simulation, with a larger value suggesting more niche evolution due to <em>in situ</em> adaptation on that branch. For a branch with no true adaptation events, the same two properties were measured, but they were measured by the events included in the posterior samples instead of the events placed on the branch during simulation. We divided all the branches into groups according to the two properties. For each group, we calculated the ratio of the sum of the posterior probabilities of all the branches with true adaptation events against the sum of the posterior probabilities of all the branches with no true adaptation events. A ratio larger than 1 suggests that, in the group, a branch that passed the test is more likely to contain true adaptation events. We applied the same steps to assess the power of our method to identify gradient speciation events, except that the second property was measured by the average of the absolute value of parameter <em>p</em> over all the gradient speciation events placed on the branch during simulation.</p></div><div id="sec-7" class="subsection"><h3>Case study: drought and salt tolerance in Acacia</h3><p id="p-31">For the <em>Acacia</em> case study, we used a published dataset that consists of 132,295 presence locations of 508 <em>Acacia</em> species (∼80% described <em>Acacia</em> species in Australia) as well as a published phylogenetic tree that includes the same 508 <em>Acacia</em> species (see <a id="xref-ref-28-2" class="xref-bibr" href="#ref-28">Mishler et al. 2014</a>). We dated the phylogenetic tree by the relaxed clock model using R package ape (<a id="xref-ref-36-1" class="xref-bibr" href="#ref-36">Paradis and Schliep 2018</a>) with the root date constrained between 21 and 23.9 million years ago, a range of crown divergence dates previously estimated for <em>Acacia</em> (<a id="xref-ref-27-1" class="xref-bibr" href="#ref-27">Miller et al. 2013</a>). We assembled species presence-absence survey data from New South Wales (<a href="http://www.environment.nsw.gov.au">www.environment.nsw.gov.au</a>), Victoria (<a href="http://www.data.vic.gov.au">www.data.vic.gov.au</a>), and West Australia (naturemap.dpaw.wa.gov.au) government databases. We only included data from complete floristic surveys that were conducted later than 2005 to ensure consistency in taxonomy, and under government custodianship to ensure survey quality. As a result, our presence-absence survey data included a total of 27,850 survey sites across three states of Australia. These presence-absence data were analysed together with the presence locations. We extracted the degree of aridity and salinity at each presence location of each species and each survey site from the geographic layer of maximum monthly aridity index (<a id="xref-ref-49-1" class="xref-bibr" href="#ref-49">Williams et al. 2012</a>) for the drought tolerance analysis, and from the geographic layer of root zone soil electrical conductivity (<a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">Bui et al. 2017</a>) for the salt tolerance analysis.</p><p id="p-32">For both analyses, we modelled sampling bias in the presence locations. This bias results from the fact that the majority of presence locations for most species are in the form of presence-only records that are typically collected opportunistically, so their spatial distributions tend to be biased towards regions that are more easily accessible to researchers (Fithian et al.2015). To correct for this bias, we included two factors that may influence sampling effort at a location: distance to the nearest road (using data from <a href="http://www.ga.gov.au/geoscience/national">www.ga.gov.au/geoscience/national</a>) and whether it is in a protected area, such as a conservation reserve or national park (from <a href="http://www.environment.gov.au/land/nrs/science/capad">www.environment.gov.au/land/nrs/science/capad</a>).</p><p id="p-33">Following <a id="xref-ref-13-4" class="xref-bibr" href="#ref-13">Fithian et al. (2015)</a>, the log-likelihood of the presence-only data of each tip species is modified as,
<span class="disp-formula" id="disp-formula-4"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2019/12/30/2019.12.29.890608/embed/graphic-6.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2019/12/30/2019.12.29.890608/embed/graphic-6.gif"/></noscript></span>

</span>
, where <em>z<sub>i</sub></em> are values of the factors that influence sampling effort at the <em>i</em><sup>th</sup> location and are the coefficients of these factors, which are assumed to be constant across tip species and were updated in the same way as the other global parameters in each MCMC (see <a id="xref-fig-1-10" class="xref-fig" href="#F1">Fig. 1</a> and Appendices). Note that the second term is no longer a direct integral of <em>λ<sub>f</sub>λ<sub>a</sub></em>, because we also need to account for sampling bias. We can only approximate the integral by randomly sampling <em>n</em> locations from the surroundings of the species presence locations to construct a set of background locations <em>I<sub>BG</sub></em> and averaging the term to integrate over all the background locations. For each <em>Acacia</em> species, background locations were sampled randomly from areas within an arbitrary large radius (20 <em>km</em>) of each presence location of the species. The total number of background locations was ten times the number of the species presence locations, with a minimum of 1000 for species with few presence locations. The area (𝒟) is the total area that falls within the 20 <em>km</em> radius of each species presence location. Note that although the radius is arbitrarily defined and background locations were sampled randomly from the area, whether the species is potentially present at <em>i</em><sup>th</sup> background location depends on both its fundamental niche <span class="inline-formula" id="inline-formula-5"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2019/12/30/2019.12.29.890608/embed/inline-graphic-5.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2019/12/30/2019.12.29.890608/embed/inline-graphic-5.gif"/></noscript></span></span> and its available niche <span class="inline-formula" id="inline-formula-6"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2019/12/30/2019.12.29.890608/embed/inline-graphic-6.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2019/12/30/2019.12.29.890608/embed/inline-graphic-6.gif"/></noscript></span></span>. Based on the environmental conditions over all the background locations, we used Sturges’ formula (<a id="xref-ref-46-1" class="xref-bibr" href="#ref-46">Sturges 1926</a>) to decide the bin size to discretise the aridity index and the electrical conductivity.</p><p id="p-34">The starting values for the available niche and the fundamental niche at the root, and the coefficients for the two factors that may influence sampling effort, were set to their maximum likelihood estimates given the likelihood of each tip species’ presence-only data and presence-absence data, assuming that no events of niche evolution occurred on the phylogeny. The starting value for dispersal rate (<em>d</em>) was set to 1 and for adaptation rate (<em>γ</em>) to 0.03. Dispersal rate and the coefficients for factors on sampling effort had rather flat priors, an exponential distribution with rate of 0.1. The parameters for the niches at the root and the adaptation rate (<em>γ</em>) had relatively narrow priors, a lognormal distribution with logarithm mean of their initial values and logarithm standard deviation of 0.5. The narrow priors for the root niches were used to ensure that the root niches do not exceed the maximum and minimum observed values for aridity and salinity in Australia too often during MCMCs. The narrow prior for <em>γ</em> was used to ensure a finite likelihood value, as parameters of the fundamental niche change proportionally to <em>γ</em> on a log scale and so there is a narrow range of <em>γ</em> that gives finite likelihood values. The starting value for the occurrence rate of <em>in situ</em> adaptation events and the occurrence rate of gradient speciation events was set to 0.1. Because the simulation study suggests a tendency to overestimate the number of <em>in situ</em> adaptation events and gradient speciation events (see Results), we used a narrow prior for the occurrence rates of both events (<em>r<sub>A</sub></em> and <em>r<sub>S</sub></em>), an exponential distribution with rate of 10, in order to reduce the degree of overestimation. For both studies, we ran 20 independent MCMCs for 1×10<sup>6</sup> generations, with a thinning interval of 1000 and the first 600,000 generations discarded as burnin.</p><p id="p-35">We used the climatic history of Australia to generate <em>a priori</em> hypotheses which we then tested against the data. <a id="xref-ref-27-2" class="xref-bibr" href="#ref-27">Miller et al (2013)</a> estimated the age of the common ancestor of extant <em>Acacia</em> as 23Ma at the earliest, and since then there have been two key periods of declining temperatures and increasing aridity: a long period from 14-5Ma, followed by a brief warmer and wetter period at the start of the Pliocene, then a second period of aridification through the late Pliocene and through the Pleistocene (<a id="xref-ref-35-2" class="xref-bibr" href="#ref-35">Owen et al. 2017</a>). So we would expect our method to infer more environmental change for the <em>in situ</em> adaptation events, along the axis of drought tolerance, between 14-5 Ma and shortly after 5 Ma, compared to other periods. To test this, we plotted the histogram of the dates of <em>in situ</em> adaptation events in the posterior samples, with the date of each event weighted by the amount of environmental change (<em>n</em>) for that event. We then corrected the relative frequency in each bin or date interval of the histogram by the number of lineages on the phylogeny during that interval, so that the frequency gives the relative amount of environmental change that each lineage had experienced during the date interval in history, as inferred by our method. Using the same method, we calculated the frequency for the salt tolerance analysis as a comparison to the draught tolerance analysis.</p><p id="p-36">In order to test the accuracy of our method for predicting current niches, we used independent information on the salinity tolerance of <em>Acacia</em> species from the literature to compile an <em>a priori</em> list of known salt-tolerant species. The list of known salt-tolerators included 12 <em>Acacia</em> species in the phylogeny (see <a id="xref-table-wrap-1-1" class="xref-table" href="#T1">Table S1</a> in Appendices), allowing us to ask whether our method correctly identifies these species as salt-tolerators. Note that this is an incomplete list as we only included studies that did physiological experiments on the species over a range of salinity conditions, so that we were confident that each species on the list was salt tolerant. We then calculated the mean of parameter <em>α</em> and <em>β</em> of the fundamental niche of each tip species from the posterior samples. If our method gives reasonable estimates for the fundamental niche of each tip species, then the estimated <em>α</em> of the 12 known salt-tolerant <em>Acacia</em> species should be at the high end of the distribution of values of other <em>Acacia</em> species and the estimated <em>β</em> of these salt-tolerant species should be at the low end of the distribution of values of other <em>Acacia</em> species. A Welch’s <em>t</em>-test was used to test for the significance in these differences. As a comparison to standard ENM that do not incorporate niche evolution, we also estimated the environmental niche along the salinity axis of each <em>Acacia</em> species from only ENM, using the MultispeciesPP R package (<a id="xref-ref-13-5" class="xref-bibr" href="#ref-13">Fithian et al. 2015</a>) which implements the same ENM model as used by our method and corrects sampling bias across species. This ENM analysis was applied to the same dataset as our method, i.e., the same species distribution data, the same data on soil electrical conductivity, and the same variables to correct for sampling bias. We used three customised natural spline bases of the soil electrical conductivity to predict a species’ environmental niche along salinity axis. We then tested if the 12 known salt-tolerance <em>Acacia</em> species have higher predicted fitness, i.e., larger area under their predicted niche curve, under high salinity than the other <em>Acacia</em> species, using Welch’s <em>t</em>-test. We used different threshold to define high salinity and reported the lowest <em>p</em> value of the Welch’s <em>t</em>-test.</p></div></div><div class="section" id="sec-8"><h2 class="">R<span class="sc">esults</span></h2><div id="sec-9" class="subsection"><h3>Simulation study</h3><p id="p-37">Overall, simulations suggested that our method performed well under the assumptions of the method, with high accuracy in estimating the fundamental niches of tip species and reasonably high power to identify the occurrence of processes of niche evolution on the phylogeny. Over all the simulations, the average R squared is above 0.8 and the average slope is around 1, suggesting that the estimates of both parameters <em>α</em> and <em>β</em> of the fundamental niches of tip species are highly correlated with their true values in the simulations (<a id="xref-fig-3-1" class="xref-fig" href="#F3">Fig. 3</a>). The power of our method to correctly identify branches with <em>in situ</em> adaptation events (the true positive adaptation) is on average 0.8 over simulations, and the power to correctly identify branches with gradient speciation events (the true positive speciation) is on average 0.5 over all simulations (<a id="xref-fig-3-2" class="xref-fig" href="#F3">Fig. 3</a>). However, our method has a high false positive rate for both <em>in situ</em> adaptation events and gradient speciation events, which is on average 0.6 over all simulations (<a id="xref-fig-3-3" class="xref-fig" href="#F3">Fig. 3</a>). For <em>in situ</em> adaptation events, these false positives tend to be events with low amount of environmental change (or low influence on niche evolution; <a id="xref-fig-4-1" class="xref-fig" href="#F4">Fig. 4b</a>). In other words, inferred <em>in situ</em> adaptation events with lower amount of environmental change are more likely to be ‘false’ events. For gradient speciation events, the false positives tend to be old events closer to the root (<a id="xref-fig-4-2" class="xref-fig" href="#F4">Fig. 4a</a>). This suggests that our method does not reliably identify gradient speciation events that occurred in the early diversification history of a species group, at least within this parameter space. This is not surprising given that these gradient speciation events do not influence the available niche and so their influence on species distribution can be quickly erased by dispersal.</p><div id="F3" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/12/30/2019.12.29.890608/F3.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Performance of our method on simulated data. Eight statistics were used to assess the performance. To assess the accuracy of our method for predicting tip species niches, we used the R2 and the slope of regressions of estimated &#x3B1; and &#x3B2; values of each tip species against their true (simulated) values. To assess the power of our method to correctly identify events of niche evolution, we used the proportion of branches with true speciation events (True positive speciation) or true adaptation event (True positive adaptation) that were identified as having significantly higher posterior probability of having speciation or adaptation events than expected. To assess the false positive rate of our method, we used the proportion of identified branches (with significantly higher posterior probability of having speciation or adaptation events than expected) that did not have a true speciation event (False positive speciation) or true adaptation event (False positive adaptation). The top row shows the performance of our method under the exact assumptions of the method. The bottom row shows the performance of our method when the assumption of constant adaptation rate is violated." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-377686284" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Performance of our method on simulated data. Eight statistics were used to assess the performance. To assess the accuracy of our method for predicting tip species niches, we used the R&lt;sup&gt;2&lt;/sup&gt; and the slope of regressions of estimated &lt;em&gt;&#x3B1;&lt;/em&gt; and &lt;em&gt;&#x3B2;&lt;/em&gt; values of each tip species against their true (simulated) values. To assess the power of our method to correctly identify events of niche evolution, we used the proportion of branches with true speciation events (True positive speciation) or true adaptation event (True positive adaptation) that were identified as having significantly higher posterior probability of having speciation or adaptation events than expected. To assess the false positive rate of our method, we used the proportion of identified branches (with significantly higher posterior probability of having speciation or adaptation events than expected) that did not have a true speciation event (False positive speciation) or true adaptation event (False positive adaptation). The top row shows the performance of our method under the exact assumptions of the method. The bottom row shows the performance of our method when the assumption of constant adaptation rate is violated.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 3." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2019/12/30/2019.12.29.890608/F3.medium.gif" width="440" height="157"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 3." src="https://www.biorxiv.org/content/biorxiv/early/2019/12/30/2019.12.29.890608/F3.medium.gif" width="440" height="157"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/12/30/2019.12.29.890608/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 3." data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/12/30/2019.12.29.890608/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 3.</span> <p id="p-38" class="first-child">Performance of our method on simulated data. Eight statistics were used to assess the performance. To assess the accuracy of our method for predicting tip species niches, we used the R<sup>2</sup> and the slope of regressions of estimated <em>α</em> and <em>β</em> values of each tip species against their true (simulated) values. To assess the power of our method to correctly identify events of niche evolution, we used the proportion of branches with true speciation events (True positive speciation) or true adaptation event (True positive adaptation) that were identified as having significantly higher posterior probability of having speciation or adaptation events than expected. To assess the false positive rate of our method, we used the proportion of identified branches (with significantly higher posterior probability of having speciation or adaptation events than expected) that did not have a true speciation event (False positive speciation) or true adaptation event (False positive adaptation). The top row shows the performance of our method under the exact assumptions of the method. The bottom row shows the performance of our method when the assumption of constant adaptation rate is violated.</p><div class="sb-div caption-clear"></div></div></div><div id="F4" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/12/30/2019.12.29.890608/F4.large.jpg?width=800&amp;height=600&amp;carousel=1" title="The confidence that a branch is correctly identified to contain true speciation events (a) and true adaptation events (b). The confidence is measured by the logarithm of the ratio in the posterior probability of an identified branch with true events to an identified branch with no true events. A positive value (red) suggests that the branch is more likely to be correctly identified, that is the identified branch contains true events. A negative value (blue) suggests that the branch is more likely to be falsely identified, that is the identified branch does not contain true events. The confidence depends on two properties of the branch: how much niche evolution occurred on that branch (x-axis) and how old the events are on the branch (y-axis)." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-377686284" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;The confidence that a branch is correctly identified to contain true speciation events (a) and true adaptation events (b). The confidence is measured by the logarithm of the ratio in the posterior probability of an identified branch with true events to an identified branch with no true events. A positive value (red) suggests that the branch is more likely to be correctly identified, that is the identified branch contains true events. A negative value (blue) suggests that the branch is more likely to be falsely identified, that is the identified branch does not contain true events. The confidence depends on two properties of the branch: how much niche evolution occurred on that branch (x-axis) and how old the events are on the branch (y-axis).&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 4." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2019/12/30/2019.12.29.890608/F4.medium.gif" width="440" height="185"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 4." src="https://www.biorxiv.org/content/biorxiv/early/2019/12/30/2019.12.29.890608/F4.medium.gif" width="440" height="185"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/12/30/2019.12.29.890608/F4.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 4." data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/12/30/2019.12.29.890608/F4.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 4.</span> <p id="p-39" class="first-child">The confidence that a branch is correctly identified to contain true speciation events (a) and true adaptation events (b). The confidence is measured by the logarithm of the ratio in the posterior probability of an identified branch with true events to an identified branch with no true events. A positive value (red) suggests that the branch is more likely to be correctly identified, that is the identified branch contains true events. A negative value (blue) suggests that the branch is more likely to be falsely identified, that is the identified branch does not contain true events. The confidence depends on two properties of the branch: how much niche evolution occurred on that branch (x-axis) and how old the events are on the branch (y-axis).</p><div class="sb-div caption-clear"></div></div></div><p id="p-40">Note that the sum of true positive rate and false positive rate for both <em>in situ</em> adaptation and gradient speciation is over one (<a id="xref-fig-3-4" class="xref-fig" href="#F3">Fig. 3</a>), suggesting that our method overestimates the number of both types of events. However, our method still accurately estimates the fundamental niche of each tip species (<a id="xref-fig-3-5" class="xref-fig" href="#F3">Fig. 3</a>). So falsely inferred events overfit the history of niche evolution to species presence-only and presence-absence data, but they have minimal influence on the accuracy of the inferred amount of niche evolution from the root to each tip species. This is also suggested by the result that inferred gradient speciation events with older dates and inferred <em>in situ</em> adaptation events with smaller amounts of environmental change are more likely to be false events (<a id="xref-fig-4-3" class="xref-fig" href="#F4">Fig. 4</a>). In other words, species distribution data are less informative for events with less influence on niche evolution, so the number of falsely inferred events in the posterior samples is more sensitive to the prior than the number of true events. Therefore, we suggest using narrow priors on the event occurrence rates (<em>r<sub>A</sub></em> and <em>r<sub>S</sub></em>) in any empirical application of our method, in order to reduce the number false events.</p><p id="p-41">An unrealistic assumption of our method is constant adaptation rates across lineages, which is unlikely to be true in the real world, as species’ responses to natural selection depend on many factors. To assess the performance of our method when this assumption is violated, we simulated the adaptation rate of each branch on each simulated phylogeny as a random variable. Note that the actual amount of niche evolution is on the exponential scale of the adaptation rate, so little variation in the adaptation rate will cause large differences in the fundamental niche. As expected, our method has reduced accuracy in estimating the fundamental niche of each tip species under the violation, with the average R squared reduced to 0.5 and the average slope reduced to 0.5 (<a id="xref-fig-3-6" class="xref-fig" href="#F3">Fig. 3</a>). However, the violation does not affect the power of our method to identify the occurrence of niche evolution on the phylogeny. The true positive and false positive rates of both <em>in situ</em> adaptation events and gradient speciation events do not differ significantly from those under the simulation scenario of constant adaptation rate (<a id="xref-fig-3-7" class="xref-fig" href="#F3">Fig. 3</a>).</p></div><div id="sec-10" class="subsection"><h3>Drought and salt tolerance in Acacia</h3><p id="p-42">Applying our method to drought tolerance and salt tolerance in <em>Acacia</em> species gives results that are consistent with expectations, suggesting that our method provides useful insights into niche evolution in the real world. First, for both drought tolerance and salt tolerance, <em>in situ</em> adaptation and gradient speciation events tend to have high occurrence in clades that are distributed predominantly in semiarid and arid areas of Australia (mostly Central and Western Australia, <a id="xref-fig-5-1" class="xref-fig" href="#F5">Fig. 5</a> and <a id="xref-fig-6-1" class="xref-fig" href="#F6">Fig. 6</a>), such as the <em>victoriae</em>/<em>pyrifolia</em> clade (<a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">Miller et al. 2011</a>), while few events occur in the large clade of a recent radiation in relatively mesic areas (mostly Northern and Eastern Australia, <a id="xref-fig-5-2" class="xref-fig" href="#F5">Fig. 5</a> and <a id="xref-fig-6-2" class="xref-fig" href="#F6">Fig. 6</a>).</p><div id="F5" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/12/30/2019.12.29.890608/F5.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Phylogenetic distribution of inferred adaptation events (a) and speciation events (b) in the evolutionary history of Acacia drought tolerance. (a) The grey scale gives the average total amount of environmental change (parameter n) in the inferred adaptation events on each branch over the posterior samples. Each tip species is color coded according to its current distribution in Australia. The map shows the spatial variation in the aridity index in Australia, with lower values in more arid area. (b) The color scale gives the average amount of niche evolution (parameter p) of inferred speciation events on each branch over the posterior samples. Negative values suggest the lineage has lower fitness under extreme conditions than its ancestral lineage. Positive value suggests the lineage has higher fitness under extreme conditions than its ancestral lineage. Short branch lengths are elongated to help view the speciation events near the tips." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-377686284" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Phylogenetic distribution of inferred adaptation events (a) and speciation events (b) in the evolutionary history of &lt;em&gt;Acacia&lt;/em&gt; drought tolerance. (a) The grey scale gives the average total amount of environmental change (parameter &lt;em&gt;n&lt;/em&gt;) in the inferred adaptation events on each branch over the posterior samples. Each tip species is color coded according to its current distribution in Australia. The map shows the spatial variation in the aridity index in Australia, with lower values in more arid area. (b) The color scale gives the average amount of niche evolution (parameter &lt;em&gt;p&lt;/em&gt;) of inferred speciation events on each branch over the posterior samples. Negative values suggest the lineage has lower fitness under extreme conditions than its ancestral lineage. Positive value suggests the lineage has higher fitness under extreme conditions than its ancestral lineage. Short branch lengths are elongated to help view the speciation events near the tips.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 5." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2019/12/30/2019.12.29.890608/F5.medium.gif" width="440" height="222"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 5." src="https://www.biorxiv.org/content/biorxiv/early/2019/12/30/2019.12.29.890608/F5.medium.gif" width="440" height="222"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/12/30/2019.12.29.890608/F5.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 5." data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/12/30/2019.12.29.890608/F5.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 5.</span> <p id="p-43" class="first-child">Phylogenetic distribution of inferred adaptation events (a) and speciation events (b) in the evolutionary history of <em>Acacia</em> drought tolerance. (a) The grey scale gives the average total amount of environmental change (parameter <em>n</em>) in the inferred adaptation events on each branch over the posterior samples. Each tip species is color coded according to its current distribution in Australia. The map shows the spatial variation in the aridity index in Australia, with lower values in more arid area. (b) The color scale gives the average amount of niche evolution (parameter <em>p</em>) of inferred speciation events on each branch over the posterior samples. Negative values suggest the lineage has lower fitness under extreme conditions than its ancestral lineage. Positive value suggests the lineage has higher fitness under extreme conditions than its ancestral lineage. Short branch lengths are elongated to help view the speciation events near the tips.</p><div class="sb-div caption-clear"></div></div></div><div id="F6" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/12/30/2019.12.29.890608/F6.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Phylogenetic distribution of inferred adaptation events (a) and speciation events (b) in the evolutionary history of Acacia salt tolerance. (a) The grey scale gives the average total amount of environmental change (parameter n) in the inferred adaptation events on each branch over the posterior samples. Each tip species is color coded according to its current distribution in Australia. The map shows the spatial variation in the soil electrical conductivity index in Australia, with higher values in more salty area. The 12 known salt-tolerant species are marked by asterisks. (b) The color scale gives the average amount of niche evolution (parameter p) of inferred speciation events on each branch over the posterior samples. Negative value suggests the lineage has lower fitness under extreme conditions than its ancestral lineage. Positive value suggests the lineage has higher fitness under extreme conditions than its ancestral lineage. Short branch length is elongated to help view the speciation events near the tips." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-377686284" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Phylogenetic distribution of inferred adaptation events (a) and speciation events (b) in the evolutionary history of &lt;em&gt;Acacia&lt;/em&gt; salt tolerance. (a) The grey scale gives the average total amount of environmental change (parameter &lt;em&gt;n&lt;/em&gt;) in the inferred adaptation events on each branch over the posterior samples. Each tip species is color coded according to its current distribution in Australia. The map shows the spatial variation in the soil electrical conductivity index in Australia, with higher values in more salty area. The 12 known salt-tolerant species are marked by asterisks. (b) The color scale gives the average amount of niche evolution (parameter &lt;em&gt;p&lt;/em&gt;) of inferred speciation events on each branch over the posterior samples. Negative value suggests the lineage has lower fitness under extreme conditions than its ancestral lineage. Positive value suggests the lineage has higher fitness under extreme conditions than its ancestral lineage. Short branch length is elongated to help view the speciation events near the tips.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 6." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2019/12/30/2019.12.29.890608/F6.medium.gif" width="440" height="222"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 6." src="https://www.biorxiv.org/content/biorxiv/early/2019/12/30/2019.12.29.890608/F6.medium.gif" width="440" height="222"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/12/30/2019.12.29.890608/F6.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 6." data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/12/30/2019.12.29.890608/F6.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 6.</span> <p id="p-44" class="first-child">Phylogenetic distribution of inferred adaptation events (a) and speciation events (b) in the evolutionary history of <em>Acacia</em> salt tolerance. (a) The grey scale gives the average total amount of environmental change (parameter <em>n</em>) in the inferred adaptation events on each branch over the posterior samples. Each tip species is color coded according to its current distribution in Australia. The map shows the spatial variation in the soil electrical conductivity index in Australia, with higher values in more salty area. The 12 known salt-tolerant species are marked by asterisks. (b) The color scale gives the average amount of niche evolution (parameter <em>p</em>) of inferred speciation events on each branch over the posterior samples. Negative value suggests the lineage has lower fitness under extreme conditions than its ancestral lineage. Positive value suggests the lineage has higher fitness under extreme conditions than its ancestral lineage. Short branch length is elongated to help view the speciation events near the tips.</p><div class="sb-div caption-clear"></div></div></div><p id="p-45">Second, for drought tolerance, the inferred amount of niche evolution per lineage due to <em>in situ</em> adaptation events peaks at 13 Ma and near the present, corresponding to the two drying periods of Australia at 14-5 Ma and after 5 Ma (<a id="xref-fig-7-1" class="xref-fig" href="#F7">Fig. 7</a>). For salt tolerance, the inferred amount of niche evolution per lineage due to <em>in situ</em> adaptation events peaks near the present, with two additional lower peaks around 10 Ma and 17 Ma (<a id="xref-fig-7-2" class="xref-fig" href="#F7">Fig. 7</a>). As a result, the inferred amount of niche evolution due to <em>in situ</em> adaptation events does not correlate between drought tolerance and salt tolerance (Kendall’s rank correlation coefficient=0.04, <em>p</em>=0.13; compare <a id="xref-fig-5-3" class="xref-fig" href="#F5">Fig. 5a</a> and <a id="xref-fig-6-3" class="xref-fig" href="#F6">Fig. 6a</a>). However, the inferred amount of niche evolution due to gradient speciation events significantly correlates between drought tolerance and salt tolerance (compare <a id="xref-fig-5-4" class="xref-fig" href="#F5">Fig. 5b</a> and <a id="xref-fig-6-4" class="xref-fig" href="#F6">Fig. 6b</a>): the average value of parameter <em>p</em> of the gradient speciation events inferred on a branch is correlated between the analysis on drought tolerance and the analysis on salt tolerance (Kendall’s rank correlation coefficient=0.12, <em>p</em>&lt;0.001).</p><div id="F7" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/12/30/2019.12.29.890608/F7.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Histogram of the inferred dates of in situ adaptation events in Acacia drought tolerance (solid) and salt tolerance (dashed). The relative frequency of the histogram is weighted by the amount environmental change in each event and is corrected by the number of lineages in the phylogeny during each time interval." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-377686284" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Histogram of the inferred dates of &lt;em&gt;in situ&lt;/em&gt; adaptation events in &lt;em&gt;Acacia&lt;/em&gt; drought tolerance (solid) and salt tolerance (dashed). The relative frequency of the histogram is weighted by the amount environmental change in each event and is corrected by the number of lineages in the phylogeny during each time interval.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 7." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2019/12/30/2019.12.29.890608/F7.medium.gif" width="440" height="392"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 7." src="https://www.biorxiv.org/content/biorxiv/early/2019/12/30/2019.12.29.890608/F7.medium.gif" width="440" height="392"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/12/30/2019.12.29.890608/F7.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 7." data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/12/30/2019.12.29.890608/F7.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 7.</span> <p id="p-46" class="first-child">Histogram of the inferred dates of <em>in situ</em> adaptation events in <em>Acacia</em> drought tolerance (solid) and salt tolerance (dashed). The relative frequency of the histogram is weighted by the amount environmental change in each event and is corrected by the number of lineages in the phylogeny during each time interval.</p><div class="sb-div caption-clear"></div></div></div><p id="p-47">Third, for salt tolerance, the estimated values of parameter <em>α</em> and <em>β</em> of known salt-tolerant species are significantly higher than the estimated values of the other <em>Acacia</em> species (Welch’s <em>t</em>=2.48, <em>p</em>=0.01). Since species with higher <em>α</em> have higher fitness under extreme condition and species with lower <em>β</em> have less abrupt decline in fitness towards extreme condition, these results suggest that our methods successfully predict higher fitness under high salinity of the known salt-tolerant species compared to remaining species. In contrast, using only ENM without accounting for shared history of niche evolution among species, we cannot identify the known salt-tolerant species. Their estimated niches do not have higher fitness under high salinity compared to the other <em>Acacia</em> species (Welch’s <em>t</em>=0.85, <em>p</em>=0.21).</p></div></div><div class="section" id="sec-11"><h2 class="">D<span class="sc">iscussion</span></h2><p id="p-48">Environmental niche models (ENMs) range from correlative models that infer species fundamental niches or potential geographic distributions from observed occurrence-environment correlations, to mechanistic models that incorporate processes based on the physiological or functional traits of species (<a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">Kearney and Porter 2009</a>; <a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">Dormann et al. 2012</a>). However, no current approach accounts for the historical role of speciation and adaptation in determining species present-day niches. While there have been several recent developments of ENMs that incorporate a phylogenetic dimension (<a id="xref-ref-45-3" class="xref-bibr" href="#ref-45">Smith et al. 2018</a>; <a id="xref-ref-31-2" class="xref-bibr" href="#ref-31">Morales-Castilla et al. 2017</a>; <a id="xref-ref-24-2" class="xref-bibr" href="#ref-24">Maguire et al. 2018</a>), these have mostly aimed to account for the statistical problem of non-independence due to relatedness among contemporary species, rather than to model the historical development of niches through evolutionary time. By combining ENMs and niche evolution models in the same statistical framework, the approach we present in this paper allows for species present-day niches to be shaped by evolutionary history and allows us to test historical hypotheses of niche evolution. Adding historical processes to ENMs results in better predictions of present-day niches from contemporary data. Simulations show that the method accurately estimates the fundamental niche of species and has reasonable power to identify the occurrences of <em>in situ</em> adaptation events and gradient speciation events on the phylogeny. Using the method, we were able to infer temporal patterns in drought tolerance and salt tolerance in <em>Acacia</em> species that are consistent with predictions based on major trends in Australia’s climatic history over the past 23Myr. We were also able to identify known salt-tolerant <em>Acacia</em> species with this method, while the standard ENM could not.</p><div id="sec-12" class="subsection"><h3>Drought and salt tolerance in Acacia</h3><p id="p-49">The climatic history of Australia through the second half of the Cenozoic has been characterized by increasing aridification punctuated by several brief periods of more mesic climates (<a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">Byrne et al. 2008</a>). The contraction of humid forests to coastal areas, and their replacement by the more open, sclerophyll vegetation that now covers much of the continent, began around 33Ma following the establishment of the Antarctic Circumpolar Current and a global drop in temperatures at the Eocene-Oligocene boundary. During this period Australia’s flora became dominated by arid-adapted lineages, especially in the large families of Myrtaceae, Fabaceae and Proteaceae (<a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">Crisp and Cook 2013</a>). To survive drought, species in these families have evolved a range of strategies to cope with water stress. These strategies have been broadly classified into escape, avoidance, and tolerance (<a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">Delzon 2015</a>). In <em>Acacia</em>, all three strategies are seen, for example phyllode shedding (escape), scleromorphic structures such as sunken stomata or needle-shaped phyllodes (avoidance), and physiological adaptations such as low leaf water potential (<a id="xref-ref-29-1" class="xref-bibr" href="#ref-29">Moore 2013</a>). The wide range of mechanisms adopted by <em>Acacia</em> species to cope with water stress are summarized in their geographic distributions, and hence their climatic niches.</p><p id="p-50">Because aridification of Australia has occurred at varying rates and was more rapid in the periods 28-23Ma, 14-5Ma and after 5Ma and because <em>Acacia</em> originated around 23 Ma at the earliest, we hypothesized that bursts of niche evolution along the drought axis (aridity index) would correspond with 14-5Ma and after 5Ma periods. Our method was able to reconstruct both temporal and spatial variation in the degree of niche evolution consistent with these expectations. We inferred peak times in adaptation to drought that correspond to these two major drying periods, and we reconstructed a concentration of niche evolution events in both drought and salinity tolerance in clades distributed predominantly in arid and semiarid regions.</p><p id="p-51">The amount of niche evolution in the aridity index and root zone electrical conductivity attributable to gradient speciation events is highly correlated across branches of the <em>Acacia</em> phylogeny. This result is consistent with the observation that saline soils are frequently associated with aridity, so many species are adapted to both high salinity and high aridity. However, we found no correlation in the occurrence of adaptation events between drought tolerance and salt tolerance: while adaptation events in drought tolerance occurred primarily in the major drying periods, adaptation events in salt tolerance occurred primarily near the tips (<a id="xref-fig-6-5" class="xref-fig" href="#F6">Fig. 6</a>). This kind of “tippy” macroevolutionary pattern is consistent with the finding in grasses that salt tolerance evolves frequently but does not give rise to a large clade of salt-tolerant descendants (<a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">Bennett et al. 2013</a>; <a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">Bromham et al. 2016</a>), suggesting that the evolution of salt tolerance may be a macroevolutionary “dead-end” (<a id="xref-ref-2-2" class="xref-bibr" href="#ref-2">Bromham et al 2016</a>). Consequently, even if salt tolerance has evolved many times through old adaptation events, these events rarely gave rise to persistently salt tolerant lineages of large clades of related salt tolerant taxa, leaving less phylogenetic signal for us to reconstruct adaptation events in the deep part of the phylogeny than adaptation events near the tip of the phylogeny.</p><p id="p-52">We also found that, in <em>Acacia</em> salt tolerance, branches with high occurrence of gradient speciation events tend to also have high occurrence of <em>in situ</em> adaptation (Kendall’s rank correlation coefficient=0.35, <em>p</em>&lt;0.001). This result suggests an important role of gradient speciation induced by environmental change in the evolution of tolerance to high salinity in <em>Acacia</em>. The prevalence of gradient speciation in <em>Acacia</em> species along the salinity axis may provide a mechanism for the high lability of salt tolerance (<a id="xref-ref-2-3" class="xref-bibr" href="#ref-2">Bromham et al. 2016</a>).</p><p id="p-53">Another contribution of our study to understanding the macroevolutionary pattern of tolerance is that it provides a continuous estimate of the degree of tolerance for tip species. Previous studies on macroevolutionary patterns of tolerance have required species to be categorized as either tolerant or sensitive to an environmental stress, based on existing evidence or expert opinions (e.g., <a id="xref-ref-1-2" class="xref-bibr" href="#ref-1">Bennett et al. 2013</a>; <a id="xref-ref-32-1" class="xref-bibr" href="#ref-32">Moray et al. 2015</a>; <a id="xref-ref-33-1" class="xref-bibr" href="#ref-33">Moray et al. 2016</a>; <a id="xref-ref-2-4" class="xref-bibr" href="#ref-2">Bromham et al. 2016</a>). These studies had a common methodological issue that the cutoff for distinguishing tolerant from sensitive species is arbitrary, and many species cannot be assigned definitively to one category or the other. Our method was able to identify known salt-tolerant <em>Acacia</em> species by their values of parameters <em>α</em> and <em>β</em>, suggesting that these parameters can give informative estimates of the degree of salt tolerance from real data. Our method therefore provides a more flexible approach to analyzing macroevolutionary patterns of tolerance by estimating the degree of tolerance from current species distribution data, which are available in some form for large numbers of species.</p></div><div id="sec-13" class="subsection"><h3>Process-based model of niche evolution</h3><p id="p-54">Unlike previous methods that incorporate evolutionary history into niche models (<a id="xref-ref-45-4" class="xref-bibr" href="#ref-45">Smith et al. 2018</a>), our method does not use phylogenetic distance as a measure of how much information on a species’ environmental niche can be derived from the distribution data of other species, an approach that assumes strong phylogenetic niche conservatism (<a id="xref-ref-34-2" class="xref-bibr" href="#ref-34">Mu□nkemu□ller et al. 2015</a>). Instead, we explicitly inferred events of niche evolution on the phylogeny from species distribution data and let the data tell us the amount of niche evolution occurred on each branch of a phylogeny. As a result, the amount of inferred niche evolution along the ancestral lineage of any two species describes the relatedness in their niches. But as with all methods, our method has its own assumptions. One major assumption is that the environmental niche of a species has evolved according to our process-based model. Our model includes three fundamental processes of niche evolution – <em>in situ</em> adaptation, gradient speciation, and dispersal - but the modelling of each process is highly simplified in order to reduce the number of parameters and limit calculation time. For example, we use one parameter (<em>p</em>) to model the process of gradient speciation by splitting the realized niche at a particular point, irrespective of the cause of the speciation event, but different causes may result in different ways of splitting the realized niche. For example, incipient species under divergent selection may actively move apart from each other to occupy their optimal environments rather than simply cut their realized niche at a particular point. However, even with highly simplified processes of niche evolution, our model is already much more biologically informative than a pure mathematical model, such as the Brownian-motion related models used by most previous PCMs (e.g. <a id="xref-ref-42-2" class="xref-bibr" href="#ref-42">Revell et al. 2008</a>; <a id="xref-ref-23-2" class="xref-bibr" href="#ref-23">Losos 2011</a>).</p><p id="p-55">The other major assumption is that the occurrence of events of niche evolution follows a compound Poisson process. Like other applications of the compound Poisson process on molecular evolution and trait evolution, this assumption makes our model of niche evolution weakly identifiable (<a id="xref-ref-30-1" class="xref-bibr" href="#ref-30">Moore et al. 2016</a>); that is, the model can explain variation in the data equally well either by including lots of events each with a small influence on niche evolution, or by inferring few events with a large influence on niche evolution. This caused the large number of falsely inferred events from the simulation data using our method, where gradient speciation events deep in the phylogeny, and <em>in situ</em> adaptation events with small amounts of environmental change, were inferred on branches along which no such events occurred. However, simulations also suggested that inference of true events with large influence on niche evolution were less sensitive to the assumption, and so our method still gives accurate estimates on the environmental niche of tip species and reliable inferences on the occurrence of major events of niche evolution. Despite possible violations of these assumptions in the real world, the <em>Acacia</em> case study demonstrated that our model was able to capture occurrences of important events in niche evolution and give sensible estimation of the environmental niche of tip species.</p><p id="p-56">Two additional assumptions of our method are a constant dispersal rate and a constant adaptation rate across the phylogeny. Simulations showed that variation in adaptation rate does not influence the power of our method to infer the history of niche evolution, but it reduces the accuracy of our method to estimate the fundamental niche of tip species. These assumptions can be relaxed, for example, by sampling these rates as random variables in MCMCs. However, whether treating adaptation rate as a random variable can improve the performance of our method requires further investigation. In particular, when both adaptation rate and the number of events of niche evolution are free to change on each branch, the model becomes unidentified because the amount of niche evolution depends on the product of adaptation rate and the number of events. This problem is similar to allowing the molecular evolution rate to change freely in molecular dating (<a id="xref-ref-40-1" class="xref-bibr" href="#ref-40">Rannala 2002</a>). In molecular dating the solution is to add calibrations on divergence dates (<a id="xref-ref-43-1" class="xref-bibr" href="#ref-43">Ronquist et al. 2012</a>). Similarly, we can add other types of input data in addition to species distribution data to our combined analysis of niche modelling and niche evolution.</p><p id="p-57">Four types of data may be used to ‘calibrate’ the analysis. First, information on paleoclimate is useful for adjusting priors for dates of <em>in situ</em> adaptation events. For example, we know there were three major drying periods in Australia during the Cenozoic – in this paper, we used this as an independent source of historical data against which to test the inferences from our model. However, for studies aiming to make inferences about the way species adapt over time, this kind of <em>a priori</em> information on changing climate could be used to inform the analysis, for example by increasing the prior probability of the occurrence of <em>in situ</em> adaptation events during the evolution of drought tolerance in these periods. Second, evidence on the tolerance range of tip species can be used as another source of data to inform the fundamental niche of the species. For example, if we know the absolute tolerance limits of a species, we can add another term in the likelihood function to calculate the probability that the species cannot survive under conditions that are more extreme than the known tolerance limits. Third, species presence/absence not only records the environmental conditions where the species can or cannot live, it also gives co-distribution patterns between different species and the existence of potential barriers to dispersal, for example if a species is present only on one side of a mountain range (<a id="xref-ref-31-3" class="xref-bibr" href="#ref-31">Morales-Castilla et al. 2017</a>). This information can be used to inform the occurrence of gradient speciation and parameter <em>x</em> in the dispersal event. It is also possible to combine our method with existing methods in historical biogeography to reconstruct ancestral geographic distributions. For example, in each MCMC, we sample a history of niche evolution and reconstruct the ancestral fundamental niches. These ancestral niches can be used with a method in historical biogeography (e.g., <a id="xref-ref-38-1" class="xref-bibr" href="#ref-38">Quintero et al. 2015</a>) to reconstruct ancestral realized niches under reconstructed paleoclimates. A fourth type of data, fossil occurrence records, can then calibrate both ancestral fundamental niches and ancestral realized niches. Methods are available to calibrate ancestral geographic distribution with fossil records and use the reconstructed distribution and past climate data to reconstruct the climatic niche of ancestral lineages (<a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">Meseguer et al. 2015</a>).</p></div></div><div class="section" id="sec-14"><h2 class="">C<span class="sc">onclusion</span></h2><p id="p-58">Paleoclimate, species tolerance range, species distributions, fossil records and phylogeny are a group of tightly linked data, where each type of data carries information on the other types of data. Our method makes it possible to use all these types of data to study niche evolution and estimate species niche. We have shown that combining niche evolution models with environmental niche modelling led to greater predictive power of the environmental tolerances in contemporary species from distribution data. We have also demonstrated that this method could reconstruct past history of niche evolution. The logical next step is to combine our method with methods in historical biogeography towards an integrated approach to understanding species’ geographic distribution through the lens of both niche evolution and geographic range evolution. With the exponential growth of data and the fast development of methods in these highly linked areas of macroecology and macroevolution, integrating data and methods in these areas is very much needed to relax biologically unrealistic assumptions in studying complex biological problems. A critical step towards an integrated approach is to identify common biological processes in these areas. Speciation, adaptation, and dispersal are the three fundamental processes underlying both niche evolution and geographic range evolution. By explicitly modelling the three processes, our method paves a promising path to a holistic understanding of species distribution.</p></div><div class="section" id="sec-15"><h2 class="">S<span class="sc">oftware</span> A<span class="sc">vailability</span></h2><p id="p-59">Source code for the method and for the simulation is available at <a href="https://github.com/huaxia1985/Phyloniches">https://github.com/huaxia1985/Phyloniches</a></p></div><div class="section" id="sec-16"><h2 class="">F<span class="sc">unding</span></h2><p id="p-60">This work was supported by an Australian Research Council Discovery Project (DP160103915) and a Centre for Biodiversity Analysis Ignition Grant.</p></div><div class="section ack" id="ack-1"><h2 class="">A<span class="sc">cknowledgements</span></h2><p id="p-61">This paper is dedicated to the memory of John le Salle: we are grateful for his support and encouragement in this and other projects, and we will miss his infectious enthusiasm for biodiversity data and research. We thank Joe Miller, Elisabeth Bui, John le Salle, Rebecca Pirzl, and Lee Belbin for useful discussions. We thank Joe Miller for providing the <em>Acacia</em> distribution data. We thank Caroline Chong for her contribution to some of the preliminary groundwork for this project.</p></div><h2 class="">A<span class="sc">ppendices</span></h2><div class="section app" id="app-1"><h2 class="">Reversible-jump model</h2><p id="p-62">The implementation involves six steps: 1) adding a new event to the model, 2) removing an existing event from the model, 3) updating the location of an existing event on the phylogeny, 4) updating parameters specific to that event, 5) updating parameters shared among events, 6) updating the occurrence rate of an event.</p><p id="p-63">In steps 1-2, the acceptance probability of the addition or the removal of an event, or of a proposed model that has one more or one fewer events than the current model, is <em>min</em>(1, <em>LikelihoodRatio× PriorRatio×ProposalRatio×Jacobian</em>). <em>LikelihoodRatio</em> is the ratio of the likelihood of the proposed model to the current model. <em>PriorRatio</em> is the ratio of the probability, <em>p</em>(model|phylogeny), of the proposed model to the current model<em>. ProposalRatio</em> is the relative probability of adding or deleting a move to the current model and we followed arguments in Rabosky (2014, equation 6-7) to set <em>ProposalRatio</em>. <em>Jacobian</em> is included to account for parameter transformation. We sampled values of each parameter for each event from the same prior distribution as all the other events, so there is no parameter transformation in our implementation and so <em>Jacobian</em>=1. For an <em>in situ</em> adaptation event, parameter <em>n</em> (the number of steps of environmental change) has a discrete prior of equal probability for any integer between one and the number of bins on the environmental axis. For a gradient speciation event, parameter <em>p</em> (the location on the environmental axis where species split) has a discrete prior with equal probability for a sequence from −0.9 to 0.9 by 0.2 increment. For the dispersal process following an <em>in situ</em> adaptation event or a gradient speciation event, parameter <em>x</em> (the location of dispersal barrier on the environmental axis) has a discrete prior with equal probability for any integer between 0 and the number of bins on the environmental axis.</p><p id="p-64">In steps 3-4, an existing event was randomly chosen to update. The location of the chosen event on the phylogeny was updated by either shifting the location by a small random quantity that was sampled from a uniform distribution or by moving the event up and down one branch. The frequency of the two ways of proposing a new location is arbitrarily set to 10:1. The proposed location is accepted with probability <em>min□</em>(1,<em>LikelihoodRatio×PriorRatio</em>), where <em>PriorRatio</em> is the ratio of the prior probability of the proposed location to the current location of the chosen event. Parameter <em>x</em> and <em>n</em> or <em>p</em> for the chosen event is updated with equal probability of moving the current value to its adjacent lower or higher value in its discrete prior. For example, if the current value for <em>x</em> is 1, the proposed value can be either 0 or 2. The proposed value is accepted with probability <em>min□</em>(1, <em>LikelihoodRatio×ProposalRatio</em>), where <em>ProposalRatio</em> is included because the proposal probability is not symmetric: the proposed value can only be higher or lower than the current value, if the current value is at the minimum or maximum bound of the discrete prior.</p><p id="p-65">In steps 5-6, each global parameter was updated, including the mean and the variance of the root available niche, parameter <em>α</em> and <em>β</em> of the root fundamental niche, dispersal rate (<em>d</em>), adaptation rate (γ), the occurrence rate of <em>in situ</em> adaptation events (<em>r<sub>A</sub></em>), and the overall occurrence rate of gradient speciation events (<em>r<sub>S</sub></em>). The mean of the root available niche is updated by adding its current value with a uniform random variable between -ε and ε, where ε is a tuning parameter, and the accepted probability is <em>min□</em>(1,<em>LikelihoodRatio</em>). All the other parameters are strictly positive, so they are updated by multiplying the current value with <em>e<sup>η(U−0.5)</sup></em>, where <em>U</em> is a uniform random variable between 0 and 1, and η is a tuning parameter. The proposed value is accepted with probability <em>min□</em>(1, <em>LikelihoodRatio×e<sup>η(U−0.5)</sup></em>)).</p><div id="T1" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/1091453/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup"><a href="/highwire/markup/1091453/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/1091453" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table S1.</span> <p id="p-66" class="first-child"><em>Known salt-tolerant Acacia species.</em> We identified salt-tolerant <em>Acacia</em> species from studies that did experiments on <em>Acacia</em> species over a range of salinity conditions. The reported salinity conditions are the highest salinity that a species was treated with in the studies and was able to tolerate.</p><div class="sb-div caption-clear"></div></div></div></div><div class="section ref-list" id="ref-list-1"><h2 class="">R<span class="sc">eferences</span></h2><ol class="cit-list ref-use-labels"><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference  in text" id="ref-1">↵</a><div class="cit ref-cit ref-journal" id="cit-2019.12.29.890608v1.1" data-doi="10.1098/rsbl.2013.0029"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Bennett</span>  <span class="cit-name-given-names">T.H.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Flowers</span>  <span class="cit-name-given-names">T.J.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Bromham</span>  <span class="cit-name-given-names">L</span></span>. <span class="cit-pub-date">2013</span>. <span class="cit-article-title">Repeated evolution of salt-tolerance in grasses</span>. <abbr class="cit-jnl-abbrev">Biol. Lett</abbr>. <span class="cit-vol">9</span>:<span class="cit-fpage">20130029</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBiol.%2BLett%26rft_id%253Dinfo%253Adoi%252F10.1098%252Frsbl.2013.0029%26rft_id%253Dinfo%253Apmid%252F23445947%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1098/rsbl.2013.0029&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23445947&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F12%2F30%2F2019.12.29.890608.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference  in text" id="ref-2">↵</a><div class="cit ref-cit ref-journal" id="cit-2019.12.29.890608v1.2" data-doi="10.1093/sysbio/syv062"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Bromham</span>  <span class="cit-name-given-names">L.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Hua</span>  <span class="cit-name-given-names">X.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Cardillo</span>  <span class="cit-name-given-names">M</span></span>. <span class="cit-pub-date">2016</span>. <span class="cit-article-title">Detecting macroevolutionary self-destruction from phylogenies</span>. <abbr class="cit-jnl-abbrev">Syst. Biol</abbr>. <span class="cit-vol">65</span>:<span class="cit-fpage">109</span>–<span class="cit-lpage">127</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DSyst.%2BBiol%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fsysbio%252Fsyv062%26rft_id%253Dinfo%253Apmid%252F26454872%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/sysbio/syv062&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=26454872&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F12%2F30%2F2019.12.29.890608.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference  in text" id="ref-3">↵</a><div class="cit ref-cit ref-journal" id="cit-2019.12.29.890608v1.3" data-doi="10.13140/RG.2.2.25552.20483"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Bui</span>  <span class="cit-name-given-names">E.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Wilford</span>  <span class="cit-name-given-names">J.</span></span>, <span class="cit-auth"><span class="cit-name-surname">de Caritat</span>  <span class="cit-name-given-names">P.</span></span> <span class="cit-pub-date">2017</span>. <span class="cit-article-title">Predictive spatial modelling for mapping soil salinity at continental scale, EGU General Assembly, Vienna, Austria</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">doi: </span><span class="cit-pub-id cit-pub-id-doi">10.13140/RG.2.2.25552.20483</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft_id%253Dinfo%253Adoi%252F10.13140%252FRG.2.2.25552.20483%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.13140/RG.2.2.25552.20483&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference  in text" id="ref-4">↵</a><div class="cit ref-cit ref-journal" id="cit-2019.12.29.890608v1.4" data-doi="doi:10.1111/j.1365-294X.2008.03899.x"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Byrne</span>  <span class="cit-name-given-names">M.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Yeates</span>  <span class="cit-name-given-names">D.K.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Joseph</span>  <span class="cit-name-given-names">L.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Kearney</span>  <span class="cit-name-given-names">M.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Bowler</span>  <span class="cit-name-given-names">J.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Williams</span>  <span class="cit-name-given-names">M.A.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Cooper</span>  <span class="cit-name-given-names">S.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Donnellan</span>  <span class="cit-name-given-names">S.C.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Keogh</span>  <span class="cit-name-given-names">J.S.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Leys</span>  <span class="cit-name-given-names">R.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Melville</span>  <span class="cit-name-given-names">J.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Murphy</span>  <span class="cit-name-given-names">D.J.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Porch</span>  <span class="cit-name-given-names">N.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Wyrwoll</span>  <span class="cit-name-given-names">K.H</span></span>. <span class="cit-pub-date">2008</span>. <span class="cit-article-title">Birth of a biome: insights into the assembly and maintenance of the Australian arid zone biota</span>. <abbr class="cit-jnl-abbrev">Mol. Ecol</abbr>. <span class="cit-vol">17</span>:<span class="cit-fpage">4398</span>–<span class="cit-lpage">4417</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2BEcology%2B%2528Print%2529%26rft.stitle%253DMolecular%2BEcology%2B%2528Print%2529%26rft.aulast%253DByrne%26rft.auinit1%253DM.%26rft.volume%253D17%26rft.issue%253D20%26rft.spage%253D4398%26rft.epage%253D4417%26rft.atitle%253DBirth%2Bof%2Ba%2Bbiome%253A%2Binsights%2Binto%2Bthe%2Bassembly%2Band%2Bmaintenance%2Bof%2Bthe%2BAustralian%2Barid%2Bzone%2Bbiota.%26rft_id%253Dinfo%253Adoi%252Fdoi%253A10.1111%252Fj.1365-294X.2008.03899.x%26rft_id%253Dinfo%253Apmid%252F18761619%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=doi:10.1111/j.1365-294X.2008.03899.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18761619&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F12%2F30%2F2019.12.29.890608.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000259953500002&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference  in text" id="ref-5">↵</a><div class="cit ref-cit ref-journal" id="cit-2019.12.29.890608v1.5" data-doi="10.1098/rsbl.2016.0368"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Cang</span>  <span class="cit-name-given-names">F.A.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Wilson</span>  <span class="cit-name-given-names">A.A.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Wiens</span>  <span class="cit-name-given-names">J.J</span></span>. <span class="cit-pub-date">2016</span>. <span class="cit-article-title">Climate change is projected to outpace rates of niche change in grasses</span>. <abbr class="cit-jnl-abbrev">Biol. Lett</abbr>. <span class="cit-vol">12</span>:<span class="cit-fpage">20160368</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBiol.%2BLett%26rft_id%253Dinfo%253Adoi%252F10.1098%252Frsbl.2016.0368%26rft_id%253Dinfo%253Apmid%252F27677813%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1098/rsbl.2016.0368&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=27677813&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F12%2F30%2F2019.12.29.890608.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference  in text" id="ref-6">↵</a><div class="cit ref-cit ref-journal" id="cit-2019.12.29.890608v1.6"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Castro-Insua</span>  <span class="cit-name-given-names">A.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Gómez-Rodríguez</span>  <span class="cit-name-given-names">C.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Wiens</span>  <span class="cit-name-given-names">J.J.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Baselga</span>  <span class="cit-name-given-names">A</span></span>. <span class="cit-pub-date">2018</span>. <span class="cit-article-title">Climatic niche divergence drives patterns of diversification and richness among mammal families</span>. <abbr class="cit-jnl-abbrev">Sci. Rep</abbr>. <span class="cit-vol">8</span>:<span class="cit-fpage">8781</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DSci.%2BRep%26rft.volume%253D8%26rft.spage%253D8781%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference  in text" id="ref-7">↵</a><div class="cit ref-cit ref-journal" id="cit-2019.12.29.890608v1.7"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Catullo</span>  <span class="cit-name-given-names">R.A.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Ferrier</span>  <span class="cit-name-given-names">S.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Hoffmann</span>  <span class="cit-name-given-names">A.A</span></span>. <span class="cit-pub-date">2015</span>. <span class="cit-article-title">Extending spatial modelling of climate change responses beyond the realized niche: estimating, and accommodating, physiological limits and adaptive evolution</span>. <abbr class="cit-jnl-abbrev">Glob. Ecol. Biogeogr</abbr>. <span class="cit-vol">24</span>:<span class="cit-fpage">1192</span>–<span class="cit-lpage">1202</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGlob.%2BEcol.%2BBiogeogr%26rft.volume%253D24%26rft.spage%253D1192%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference  in text" id="ref-8">↵</a><div class="cit ref-cit ref-journal" id="cit-2019.12.29.890608v1.8" data-doi="10.1146/annurev-ecolsys-110512-135910"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Crisp</span>  <span class="cit-name-given-names">M.D.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Cook</span>  <span class="cit-name-given-names">L.G</span></span>. <span class="cit-pub-date">2013</span>. <span class="cit-article-title">How was the Australian flora assembled over the last 65 million years? A molecular phylogenetic perspective</span>. <abbr class="cit-jnl-abbrev">Annu. Rev. Ecol. Evol. Syst</abbr>. <span class="cit-vol">44</span>:<span class="cit-fpage">303</span>–<span class="cit-lpage">324</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAnnu.%2BRev.%2BEcol.%2BEvol.%2BSyst%26rft.volume%253D44%26rft.spage%253D303%26rft_id%253Dinfo%253Adoi%252F10.1146%252Fannurev-ecolsys-110512-135910%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1146/annurev-ecolsys-110512-135910&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference  in text" id="ref-9">↵</a><div class="cit ref-cit ref-journal" id="cit-2019.12.29.890608v1.9"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Delzon</span>  <span class="cit-name-given-names">S</span></span>. <span class="cit-pub-date">2015</span>. <span class="cit-article-title">New insight into leaf drought tolerance</span>. <abbr class="cit-jnl-abbrev">Funct. Ecol</abbr>. <span class="cit-vol">29</span>:<span class="cit-fpage">1247</span>–<span class="cit-lpage">1249</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DFunct.%2BEcol%26rft.volume%253D29%26rft.spage%253D1247%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference  in text" id="ref-10">↵</a><div class="cit ref-cit ref-journal" id="cit-2019.12.29.890608v1.10" data-doi="10.1111/j.1365-2699.2011.02659.x"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Dormann</span>  <span class="cit-name-given-names">C.F.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Schymanski</span>  <span class="cit-name-given-names">S.J.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Cabral</span>  <span class="cit-name-given-names">J.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Chuine</span>  <span class="cit-name-given-names">I.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Graham</span>  <span class="cit-name-given-names">C.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Hartig</span>  <span class="cit-name-given-names">F.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Kearney</span>  <span class="cit-name-given-names">M.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Morin</span>  <span class="cit-name-given-names">X.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Römermann</span>  <span class="cit-name-given-names">C.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Schröder</span>  <span class="cit-name-given-names">B.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Singer</span>  <span class="cit-name-given-names">A.</span></span> <span class="cit-pub-date">2012</span>. <span class="cit-article-title">Correlation and process in species distribution models: bridging a dichotomy</span>. <abbr class="cit-jnl-abbrev">J. Biogeogr</abbr>. <span class="cit-vol">39</span>:<span class="cit-fpage">2119</span>–<span class="cit-lpage">2131</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BBiogeogr%26rft.volume%253D39%26rft.spage%253D2119%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1365-2699.2011.02659.x%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1365-2699.2011.02659.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000311384100006&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference  in text" id="ref-11">↵</a><div class="cit ref-cit ref-journal" id="cit-2019.12.29.890608v1.11" data-doi="10.1111/j.1558-5646.2011.01401.x"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Eastman</span>  <span class="cit-name-given-names">J.M.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Alfaro</span>  <span class="cit-name-given-names">M.E.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Joyce</span>  <span class="cit-name-given-names">P.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Hipp</span>  <span class="cit-name-given-names">A.L.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Harmon</span>  <span class="cit-name-given-names">L.J</span></span>. <span class="cit-pub-date">2011</span>. <span class="cit-article-title">A novel comparative method for identifying shifts in the rate of character evolution on trees</span>. <abbr class="cit-jnl-abbrev">Evolution</abbr> <span class="cit-vol">65</span>:<span class="cit-fpage">3578</span>–<span class="cit-lpage">3589</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEvolution%26rft.stitle%253DEvolution%26rft.aulast%253DEastman%26rft.auinit1%253DJ.%2BM.%26rft.volume%253D65%26rft.issue%253D12%26rft.spage%253D3578%26rft.epage%253D3589%26rft.atitle%253DA%2Bnovel%2Bcomparative%2Bmethod%2Bfor%2Bidentifying%2Bshifts%2Bin%2Bthe%2Brate%2Bof%2Bcharacter%2Bevolution%2Bon%2Btrees.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1558-5646.2011.01401.x%26rft_id%253Dinfo%253Apmid%252F22133227%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1558-5646.2011.01401.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22133227&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F12%2F30%2F2019.12.29.890608.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000297732000018&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference  in text" id="ref-12">↵</a><div class="cit ref-cit ref-journal" id="cit-2019.12.29.890608v1.12"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Felsenstein</span>  <span class="cit-name-given-names">J</span></span>. <span class="cit-pub-date">1985</span>. <span class="cit-article-title">Phylogenies and the comparative method</span>. <abbr class="cit-jnl-abbrev">Am. Nat</abbr>. <span class="cit-vol">12</span>:<span class="cit-fpage">1</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAm.%2BNat%26rft.volume%253D12%26rft.spage%253D1%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference  in text" id="ref-13">↵</a><div class="cit ref-cit ref-journal" id="cit-2019.12.29.890608v1.13"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Fithian</span>  <span class="cit-name-given-names">W.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Elith</span>  <span class="cit-name-given-names">J.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Hastie</span>  <span class="cit-name-given-names">T.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Keith</span>  <span class="cit-name-given-names">D.A</span></span>. <span class="cit-pub-date">2015</span>. <span class="cit-article-title">Bias correction in species distribution models: pooling survey and collection data for multiple species</span>. <abbr class="cit-jnl-abbrev">Methods Ecol. Evol</abbr>. <span class="cit-vol">6</span>:<span class="cit-fpage">424</span>–<span class="cit-lpage">438</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMethods%2BEcol.%2BEvol%26rft.volume%253D6%26rft.spage%253D424%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference  in text" id="ref-14">↵</a><div class="cit ref-cit ref-journal" id="cit-2019.12.29.890608v1.14" data-doi="10.1111/j.2041-210X.2012.00234.x"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">FitzJohn</span>  <span class="cit-name-given-names">R.G</span></span>. <span class="cit-pub-date">2012</span>. <span class="cit-article-title">Diversitree: comparative phylogenetic analyses of diversification in R</span>. <abbr class="cit-jnl-abbrev">Methods Ecol. Evol</abbr>. <span class="cit-vol">3</span>:<span class="cit-fpage">1084</span>–<span class="cit-lpage">1092</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMethods%2BEcol.%2BEvol%26rft.volume%253D3%26rft.spage%253D1084%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.2041-210X.2012.00234.x%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.2041-210X.2012.00234.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference  in text" id="ref-15">↵</a><div class="cit ref-cit ref-journal" id="cit-2019.12.29.890608v1.15"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">González-Orozco</span>  <span class="cit-name-given-names">C.E.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Pollock</span>  <span class="cit-name-given-names">L.J.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Thornhill</span>  <span class="cit-name-given-names">A.H.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Mishler</span>  <span class="cit-name-given-names">B.D.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Knerr</span>  <span class="cit-name-given-names">N.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Laffan</span>  <span class="cit-name-given-names">S.W.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Miller</span>  <span class="cit-name-given-names">J.T.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Rosauer</span>  <span class="cit-name-given-names">D.F.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Faith</span>  <span class="cit-name-given-names">P.D.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Nipperess</span>  <span class="cit-name-given-names">D.A.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Kujala</span>  <span class="cit-name-given-names">H.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Linke</span>  <span class="cit-name-given-names">S.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Butt</span>  <span class="cit-name-given-names">N.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Külheim</span>  <span class="cit-name-given-names">C.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Crisp</span>  <span class="cit-name-given-names">M.D.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Bruber</span>  <span class="cit-name-given-names">B</span></span>. <span class="cit-pub-date">2016</span>. <span class="cit-article-title">Phylogenetic approaches reveal biodiversity threats under climate change. Nat</span>. <abbr class="cit-jnl-abbrev">Clim. Change</abbr> <span class="cit-vol">6</span>:<span class="cit-fpage">1110</span>–<span class="cit-lpage">1115</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DClim.%2BChange%26rft.volume%253D6%26rft.spage%253D1110%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference  in text" id="ref-16">↵</a><div class="cit ref-cit ref-journal" id="cit-2019.12.29.890608v1.16" data-doi="10.1093/biomet/82.4.711"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Green</span>  <span class="cit-name-given-names">P.J</span></span>. <span class="cit-pub-date">1995</span>. <span class="cit-article-title">Reversible jump Markov Chain Monte Carlo computation and Bayesian model determination</span>. <abbr class="cit-jnl-abbrev">Biometrika</abbr> <span class="cit-vol">82</span>:<span class="cit-fpage">711</span>–<span class="cit-lpage">732</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBiometrika%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbiomet%252F82.4.711%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/biomet/82.4.711&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=A1995TT22800012&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference  in text" id="ref-17">↵</a><div class="cit ref-cit ref-journal" id="cit-2019.12.29.890608v1.17"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Hardy</span>  <span class="cit-name-given-names">C.R</span></span>. <span class="cit-pub-date">2006</span>. <span class="cit-article-title">Reconstructing ancestral ecologies: challenges and possible solutions</span>. <abbr class="cit-jnl-abbrev">Diversity Distrib</abbr>. <span class="cit-vol">12</span>:<span class="cit-fpage">7</span>–<span class="cit-lpage">19</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DDiversity%2BDistrib%26rft.volume%253D12%26rft.spage%253D7%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference  in text" id="ref-18">↵</a><div class="cit ref-cit ref-journal" id="cit-2019.12.29.890608v1.18" data-doi="10.1101/SQB.1957.022.01.039"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Hutchinson</span>  <span class="cit-name-given-names">G.E</span></span>. <span class="cit-pub-date">1957</span>. <span class="cit-article-title">Concluding remarks</span>. <abbr class="cit-jnl-abbrev">Cold Spring Harb. Symp</abbr>. <span class="cit-vol">22</span>:<span class="cit-fpage">415</span>–<span class="cit-lpage">427</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCold%2BSpring%2BHarb.%2BSymp%26rft.volume%253D22%26rft.spage%253D415%26rft_id%253Dinfo%253Adoi%252F10.1101%252FSQB.1957.022.01.039%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1101/SQB.1957.022.01.039&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference  in text" id="ref-19">↵</a><div class="cit ref-cit ref-journal" id="cit-2019.12.29.890608v1.19" data-doi="10.1073/pnas.0905137106"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Holt</span>  <span class="cit-name-given-names">R.D</span></span>. <span class="cit-pub-date">2009</span>. <span class="cit-article-title">Bringing the Hutchinsonian niche into the 21st century: ecological and evolutionary perspectives</span>. <abbr class="cit-jnl-abbrev">Proc. Natl Acad. Sci. USA</abbr> <span class="cit-vol">106</span>:<span class="cit-fpage">19659</span>–<span class="cit-lpage">19665</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.0905137106%26rft_id%253Dinfo%253Apmid%252F19903876%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoyMjoiMTA2L1N1cHBsZW1lbnRfMi8xOTY1OSI7czo0OiJhdG9tIjtzOjQ4OiIvYmlvcnhpdi9lYXJseS8yMDE5LzEyLzMwLzIwMTkuMTIuMjkuODkwNjA4LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference  in text" id="ref-20">↵</a><div class="cit ref-cit ref-journal" id="cit-2019.12.29.890608v1.20" data-doi="10.1111/j.1461-0248.2008.01277.x"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Kearney</span>  <span class="cit-name-given-names">M.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Porter</span>  <span class="cit-name-given-names">W</span></span>. <span class="cit-pub-date">2009</span>. <span class="cit-article-title">Mechanistic niche modelling: combining physiological and spatial data to predict species’ ranges</span>. <abbr class="cit-jnl-abbrev">Ecol. Lett</abbr>. <span class="cit-vol">12</span>:<span class="cit-fpage">334</span>–<span class="cit-lpage">350</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DEcol%2BLett%26rft.aulast%253DKearney%26rft.auinit1%253DM.%26rft.volume%253D12%26rft.issue%253D4%26rft.spage%253D334%26rft.epage%253D350%26rft.atitle%253DMechanistic%2Bniche%2Bmodelling%253A%2Bcombining%2Bphysiological%2Band%2Bspatial%2Bdata%2Bto%2Bpredict%2Bspecies%2527%2Branges.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1461-0248.2008.01277.x%26rft_id%253Dinfo%253Apmid%252F19292794%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1461-0248.2008.01277.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19292794&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F12%2F30%2F2019.12.29.890608.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000264067600007&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference  in text" id="ref-21">↵</a><div class="cit ref-cit ref-journal" id="cit-2019.12.29.890608v1.21" data-doi="10.1214/aoms/1177730285"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Kendall</span>  <span class="cit-name-given-names">D.G</span></span>. <span class="cit-pub-date">1948</span>. <span class="cit-article-title">On the generalized ‘‘Birth-and-Death’’ process</span>. <abbr class="cit-jnl-abbrev">Ann. Math. Statist</abbr>. <span class="cit-vol">19</span>:<span class="cit-fpage">1</span>–<span class="cit-lpage">15</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAnn.%2BMath.%2BStatist%26rft.volume%253D19%26rft.spage%253D1%26rft_id%253Dinfo%253Adoi%252F10.1214%252Faoms%252F1177730285%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1214/aoms/1177730285&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference  in text" id="ref-22">↵</a><div class="cit ref-cit ref-journal" id="cit-2019.12.29.890608v1.22" data-doi="10.1111/j.1461-0248.2010.01530.x"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Kozak</span>  <span class="cit-name-given-names">K.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Wiens</span>  <span class="cit-name-given-names">J</span></span>. <span class="cit-pub-date">2010</span>. <span class="cit-article-title">Accelerated rates of climatic-niche evolution underlie rapid species diversification</span>. <abbr class="cit-jnl-abbrev">Ecol. Lett</abbr>. <span class="cit-vol">13</span>:<span class="cit-fpage">1378</span>–<span class="cit-lpage">1389</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DEcol%2BLett%26rft.aulast%253DKozak%26rft.auinit1%253DK.%2BH.%26rft.volume%253D13%26rft.issue%253D11%26rft.spage%253D1378%26rft.epage%253D1389%26rft.atitle%253DAccelerated%2Brates%2Bof%2Bclimatic-niche%2Bevolution%2Bunderlie%2Brapid%2Bspecies%2Bdiversification.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1461-0248.2010.01530.x%26rft_id%253Dinfo%253Apmid%252F20875038%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1461-0248.2010.01530.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20875038&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F12%2F30%2F2019.12.29.890608.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000283157500006&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference  in text" id="ref-23">↵</a><div class="cit ref-cit ref-journal" id="cit-2019.12.29.890608v1.23"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Losos</span>  <span class="cit-name-given-names">J.B</span></span>. <span class="cit-pub-date">2001</span>. <span class="cit-article-title">Seeing the forest for the trees: the limitations of phylogenies in comparative biology</span>. <abbr class="cit-jnl-abbrev">Am. Nat</abbr>. <span class="cit-vol">177</span>:<span class="cit-fpage">709</span>–<span class="cit-lpage">727</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAm.%2BNat%26rft.volume%253D177%26rft.spage%253D709%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference  in text" id="ref-24">↵</a><div class="cit ref-cit ref-journal" id="cit-2019.12.29.890608v1.24"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Maguire</span>  <span class="cit-name-given-names">K.C.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Shinneman</span>  <span class="cit-name-given-names">D.J.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Potter</span>  <span class="cit-name-given-names">K.M.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Hipkins</span>  <span class="cit-name-given-names">V.D</span></span>. <span class="cit-pub-date">2018</span>. <span class="cit-article-title">Intraspecific niche models for ponderosa pine (<em>Pinus ponderosa</em>) suggest potential variability in population-level response to climate change</span>. <abbr class="cit-jnl-abbrev">Syst. Biol</abbr>. <span class="cit-vol">67</span>:<span class="cit-fpage">965</span>–<span class="cit-lpage">978</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DSyst.%2BBiol%26rft.volume%253D67%26rft.spage%253D965%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference  in text" id="ref-25">↵</a><div class="cit ref-cit ref-journal" id="cit-2019.12.29.890608v1.25" data-doi="10.1093/sysbio/syu088"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Meseguer</span>  <span class="cit-name-given-names">A.S.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Lobo</span>  <span class="cit-name-given-names">J.M.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Ree</span>  <span class="cit-name-given-names">R.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Beerling</span>  <span class="cit-name-given-names">D.J.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Sanmartin</span>  <span class="cit-name-given-names">I</span></span>. <span class="cit-pub-date">2015</span>. <span class="cit-article-title">Integrating fossils, phylogenies, and niche models into biogeography to reveal ancient evolutionary history: the case of <em>Hypericum</em> (Hypericaceae)</span>. <abbr class="cit-jnl-abbrev">Syst. Biol</abbr>. <span class="cit-vol">64</span>:<span class="cit-fpage">215</span>–<span class="cit-lpage">232</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DSyst.%2BBiol%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fsysbio%252Fsyu088%26rft_id%253Dinfo%253Apmid%252F25398444%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/sysbio/syu088&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25398444&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F12%2F30%2F2019.12.29.890608.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference  in text" id="ref-26">↵</a><div class="cit ref-cit ref-journal" id="cit-2019.12.29.890608v1.26"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Miller</span>  <span class="cit-name-given-names">J.T.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Murphy</span>,  <span class="cit-name-given-names">D.J.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Brown</span>  <span class="cit-name-given-names">G.K.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Richardson</span>  <span class="cit-name-given-names">D.M.</span></span>, <span class="cit-auth"><span class="cit-name-surname">González-Orozco</span>  <span class="cit-name-given-names">C.E.</span></span> <span class="cit-pub-date">2011</span>. <span class="cit-article-title">The evolution and phylogenetic placement of invasive Australian <em>Acacia</em> species</span>. <abbr class="cit-jnl-abbrev">Diversity Distrib</abbr>. <span class="cit-vol">17</span>:<span class="cit-fpage">848</span>–<span class="cit-lpage">860</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DDiversity%2BDistrib%26rft.volume%253D17%26rft.spage%253D848%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-27-1" title="View reference  in text" id="ref-27">↵</a><div class="cit ref-cit ref-journal" id="cit-2019.12.29.890608v1.27"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Miller</span>  <span class="cit-name-given-names">J.T.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Murphy</span>  <span class="cit-name-given-names">D.J.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Ho</span>  <span class="cit-name-given-names">S.Y.W.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Cantrill</span>  <span class="cit-name-given-names">D.J.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Seigler</span>  <span class="cit-name-given-names">D</span></span>. <span class="cit-pub-date">2013</span>. <span class="cit-article-title">Comparative dating of <em>Acacia</em>: combining fossils and multiple phylogenies to infer ages of clades with poor fossil records</span>. <abbr class="cit-jnl-abbrev">Aust. J. Bot</abbr>. <span class="cit-vol">61</span>:<span class="cit-fpage">436</span>−<span class="cit-lpage">445</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAust.%2BJ.%2BBot%26rft.volume%253D61%26rft.spage%253D436%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-28-1" title="View reference  in text" id="ref-28">↵</a><div class="cit ref-cit ref-journal" id="cit-2019.12.29.890608v1.28"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Mishler</span>  <span class="cit-name-given-names">B.D.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Knerr</span>  <span class="cit-name-given-names">N.</span></span>, <span class="cit-auth"><span class="cit-name-surname">González-Orozco</span>  <span class="cit-name-given-names">C.E.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Thornhill</span>  <span class="cit-name-given-names">A.H.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Laffan</span>  <span class="cit-name-given-names">S.W.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Miller</span>  <span class="cit-name-given-names">J.T.</span></span> <span class="cit-pub-date">2014</span>. <span class="cit-article-title">Phylogenetic measures of biodiversity and neo- and paleo-endemism in Australian <em>Acacia</em></span>. <abbr class="cit-jnl-abbrev">Nat. Comm</abbr>. <span class="cit-vol">5</span>:<span class="cit-fpage">4473</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat.%2BComm%26rft.volume%253D5%26rft.spage%253D4473%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-29-1" title="View reference  in text" id="ref-29">↵</a><div class="cit ref-cit ref-journal" id="cit-2019.12.29.890608v1.29"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Moore</span>  <span class="cit-name-given-names">G.M</span></span>. <span class="cit-pub-date">2013</span>. <span class="cit-article-title">Adaptations of Australian tree species relevant to water scarcity in the urban forest</span>. <abbr class="cit-jnl-abbrev">Arboric. Urban. For</abbr>. <span class="cit-vol">39</span>:<span class="cit-fpage">109</span>−<span class="cit-lpage">115</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DArboric.%2BUrban.%2BFor%26rft.volume%253D39%26rft.spage%253D109%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-30-1" title="View reference  in text" id="ref-30">↵</a><div class="cit ref-cit ref-journal" id="cit-2019.12.29.890608v1.30" data-doi="10.1073/pnas.1518659113"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Moore</span>  <span class="cit-name-given-names">B.R.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Ho□hna</span>  <span class="cit-name-given-names">S.</span></span>, <span class="cit-auth"><span class="cit-name-surname">May</span>  <span class="cit-name-given-names">M.R.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Rannala</span>  <span class="cit-name-given-names">B.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Huelsenbeck</span>  <span class="cit-name-given-names">J.P.</span></span> <span class="cit-pub-date">2016</span>. <span class="cit-article-title">Critically evaluating the theory and performance of Bayesian analysis of macroevolutionary mixtures</span>. <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">113</span>:<span class="cit-fpage">9569</span>−<span class="cit-lpage">9574</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.1518659113%26rft_id%253Dinfo%253Apmid%252F27512038%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMToiMTEzLzM0Lzk1NjkiO3M6NDoiYXRvbSI7czo0ODoiL2Jpb3J4aXYvZWFybHkvMjAxOS8xMi8zMC8yMDE5LjEyLjI5Ljg5MDYwOC5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-31-1" title="View reference  in text" id="ref-31">↵</a><div class="cit ref-cit ref-journal" id="cit-2019.12.29.890608v1.31" data-doi="doi.org/10.1111/geb.12580"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Morales-Castilla</span>  <span class="cit-name-given-names">I.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Davies</span>  <span class="cit-name-given-names">T.J.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Pearse</span>  <span class="cit-name-given-names">W.D.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Peres-Neto</span>  <span class="cit-name-given-names">P</span></span>. <span class="cit-pub-date">2017</span>. <span class="cit-article-title">Combining phylogeny and co-occurrence to improve single species distribution models</span>. <abbr class="cit-jnl-abbrev">Glob. Ecol. Biogeogr</abbr>. <span class="cit-vol">26</span>:<span class="cit-fpage">740</span>–<span class="cit-lpage">752</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGlob.%2BEcol.%2BBiogeogr%26rft.volume%253D26%26rft.spage%253D740%26rft_id%253Dinfo%253Adoi%252Fdoi.org%252F10.1111%252Fgeb.12580%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=doi.org/10.1111/geb.12580&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-32-1" title="View reference  in text" id="ref-32">↵</a><div class="cit ref-cit ref-journal" id="cit-2019.12.29.890608v1.32"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Moray</span>  <span class="cit-name-given-names">C.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Hua</span>  <span class="cit-name-given-names">X.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Bromham</span>  <span class="cit-name-given-names">L</span></span>. <span class="cit-pub-date">2015</span>. <span class="cit-article-title">Salt tolerance is evolutionarily labile in a diverse set of angiosperm families</span>. <abbr class="cit-jnl-abbrev">BMC Evol. Biol</abbr>. <span class="cit-vol">15</span>:<span class="cit-fpage">90</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2BEvol.%2BBiol%26rft.volume%253D15%26rft.spage%253D90%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-33-1" title="View reference  in text" id="ref-33">↵</a><div class="cit ref-cit ref-journal" id="cit-2019.12.29.890608v1.33"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Moray</span>  <span class="cit-name-given-names">C.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Goolsby</span>  <span class="cit-name-given-names">E.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Bromham</span>  <span class="cit-name-given-names">L</span></span>. <span class="cit-pub-date">2016</span>. <span class="cit-article-title">The phylogenetic association between salt tolerance and heavy metal hyperaccumulation in angiosperms</span>. <abbr class="cit-jnl-abbrev">Evol. Biol</abbr>. <span class="cit-vol">43</span>:<span class="cit-fpage">119</span>–<span class="cit-lpage">130</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEvol.%2BBiol%26rft.volume%253D43%26rft.spage%253D119%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-34-1" title="View reference  in text" id="ref-34">↵</a><div class="cit ref-cit ref-journal" id="cit-2019.12.29.890608v1.34"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Mu□nkemu□ller</span>  <span class="cit-name-given-names">T.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Boucher</span>  <span class="cit-name-given-names">F.C.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Thuiller</span>  <span class="cit-name-given-names">W.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Lavergne</span>  <span class="cit-name-given-names">S.</span></span> <span class="cit-pub-date">2015</span>. <span class="cit-article-title">Phylogenetic niche conservatism – common pitfalls and ways forward</span>. <abbr class="cit-jnl-abbrev">Funct. Ecol</abbr>. <span class="cit-vol">29</span>:<span class="cit-fpage">627</span>−<span class="cit-lpage">639</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DFunct.%2BEcol%26rft.volume%253D29%26rft.spage%253D627%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-35-1" title="View reference  in text" id="ref-35">↵</a><div class="cit ref-cit ref-journal" id="cit-2019.12.29.890608v1.35"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Owen</span>  <span class="cit-name-given-names">C.L.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Marshall</span>  <span class="cit-name-given-names">D.C.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Hill</span>  <span class="cit-name-given-names">K.B.R.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Simon</span>  <span class="cit-name-given-names">C</span></span>. <span class="cit-pub-date">2017</span>. <span class="cit-article-title">How the aridification of Australia structured the biogeography and influenced the diversification of a large lineage of Australian cicadas</span>. <abbr class="cit-jnl-abbrev">Syst. Biol</abbr>. <span class="cit-vol">66</span>:<span class="cit-fpage">569</span>−<span class="cit-lpage">589</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DSyst.%2BBiol%26rft.volume%253D66%26rft.spage%253D569%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-36-1" title="View reference  in text" id="ref-36">↵</a><div class="cit ref-cit ref-journal" id="cit-2019.12.29.890608v1.36"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Paradis</span>  <span class="cit-name-given-names">E.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Schliep</span>  <span class="cit-name-given-names">K</span></span>. <span class="cit-pub-date">2018</span>. <abbr class="cit-jnl-abbrev">ape 5.0: an environment for modern phylogenetics and evolutionary analyses in R. Bioinformatics, bty633</abbr>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-37-1" title="View reference  in text" id="ref-37">↵</a><div class="cit ref-cit ref-journal" id="cit-2019.12.29.890608v1.37" data-doi="10.1650/0010-5422(2001)103[0599:PSGDBO]2.0.CO;2"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Peterson</span>  <span class="cit-name-given-names">A.T</span></span>. <span class="cit-pub-date">2001</span>. <span class="cit-article-title">Predicting species’ geographic distributions based on ecological niche modeling</span>. <abbr class="cit-jnl-abbrev">Condor</abbr> <span class="cit-vol">103</span>:<span class="cit-fpage">599</span>–<span class="cit-lpage">605</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCondor%26rft.volume%253D103%26rft.spage%253D599%26rft_id%253Dinfo%253Adoi%252F10.1650%252F0010-5422%25282001%2529103%255B0599%253APSGDBO%255D2.0.CO%253B2%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1650/0010-5422(2001)103[0599:PSGDBO]2.0.CO;2&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-38-1" title="View reference  in text" id="ref-38">↵</a><div class="cit ref-cit ref-journal" id="cit-2019.12.29.890608v1.38" data-doi="10.1093/sysbio/syv057"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Quintero</span>  <span class="cit-name-given-names">I.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Keil</span>  <span class="cit-name-given-names">R.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Jetz</span>  <span class="cit-name-given-names">W.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Crawford</span>  <span class="cit-name-given-names">F.W</span></span>. <span class="cit-pub-date">2015</span>. <span class="cit-article-title">Historical biogeography using species geographic ranges</span>. <abbr class="cit-jnl-abbrev">Syst. Biol</abbr>. <span class="cit-vol">64</span>:<span class="cit-fpage">1059</span>–<span class="cit-lpage">1073</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DSyst.%2BBiol%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fsysbio%252Fsyv057%26rft_id%253Dinfo%253Apmid%252F26254671%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/sysbio/syv057&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=26254671&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F12%2F30%2F2019.12.29.890608.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-39-1" title="View reference  in text" id="ref-39">↵</a><div class="cit ref-cit ref-journal" id="cit-2019.12.29.890608v1.39" data-doi="10.1371/journal.pone.0089543"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Rabosky</span>  <span class="cit-name-given-names">D. L</span></span>. <span class="cit-pub-date">2014</span>. <span class="cit-article-title">Automatic detection of key innovations, rate shifts, and density-dependence on phylogenetic trees</span>. <abbr class="cit-jnl-abbrev">PLOS ONE</abbr> <span class="cit-vol">9</span>:<span class="cit-fpage">e89543</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLOS%2BONE%26rft.volume%253D9%26rft.spage%253De89543%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0089543%26rft_id%253Dinfo%253Apmid%252F24586858%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0089543&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24586858&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F12%2F30%2F2019.12.29.890608.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-40-1" title="View reference  in text" id="ref-40">↵</a><div class="cit ref-cit ref-journal" id="cit-2019.12.29.890608v1.40" data-doi="10.1080/10635150290102429"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Rannala</span>  <span class="cit-name-given-names">B</span></span>. <span class="cit-pub-date">2002</span>. <span class="cit-article-title">Identifiability of parameters in MCMC Bayesian inference of phylogeny</span>. <abbr class="cit-jnl-abbrev">Syst. Biol</abbr>. <span class="cit-vol">51</span>:<span class="cit-fpage">754</span>–<span class="cit-lpage">760</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DSystematic%2BBiology%26rft.stitle%253DSyst%2BBiol%26rft.aulast%253DRannala%26rft.auinit1%253DB.%26rft.volume%253D51%26rft.issue%253D5%26rft.spage%253D754%26rft.epage%253D760%26rft.atitle%253DIdentifiability%2Bof%2Bparameters%2Bin%2BMCMC%2BBayesian%2Binference%2Bof%2Bphylogeny.%26rft_id%253Dinfo%253Adoi%252F10.1080%252F10635150290102429%26rft_id%253Dinfo%253Apmid%252F12396589%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1080/10635150290102429&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=12396589&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F12%2F30%2F2019.12.29.890608.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000178079500007&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-41-1" title="View reference  in text" id="ref-41">↵</a><div class="cit ref-cit ref-journal" id="cit-2019.12.29.890608v1.41"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Renner</span>  <span class="cit-name-given-names">I.W.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Elith</span>  <span class="cit-name-given-names">J.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Baddeley</span>  <span class="cit-name-given-names">A.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Fithian</span>  <span class="cit-name-given-names">W.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Hastie</span>  <span class="cit-name-given-names">T.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Phillips</span>  <span class="cit-name-given-names">S.J.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Popovic</span>  <span class="cit-name-given-names">G.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Warton</span>  <span class="cit-name-given-names">D.I</span></span>. <span class="cit-pub-date">2015</span>. <span class="cit-article-title">Point process models for presence-only analysis</span>. <abbr class="cit-jnl-abbrev">Methods Ecol. Evol</abbr>. <span class="cit-vol">6</span>:<span class="cit-fpage">366</span>– <span class="cit-lpage">379</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMethods%2BEcol.%2BEvol%26rft.volume%253D6%26rft.spage%253D366%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-42-1" title="View reference  in text" id="ref-42">↵</a><div class="cit ref-cit ref-journal" id="cit-2019.12.29.890608v1.42" data-doi="10.1080/10635150802302427"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Revell</span>  <span class="cit-name-given-names">L.J.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Harmon</span>  <span class="cit-name-given-names">L.J.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Collar</span>  <span class="cit-name-given-names">D.C</span></span>. <span class="cit-pub-date">2008</span>. <span class="cit-article-title">Phylogenetic signal, evolutionary process, and rate</span>. <abbr class="cit-jnl-abbrev">Syst. Biol</abbr>. <span class="cit-vol">57</span>:<span class="cit-fpage">591</span>–<span class="cit-lpage">601</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DSyst.%2BBiol%26rft_id%253Dinfo%253Adoi%252F10.1080%252F10635150802302427%26rft_id%253Dinfo%253Apmid%252F18709597%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1080/10635150802302427&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18709597&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F12%2F30%2F2019.12.29.890608.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000263702800005&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-43-1" title="View reference  in text" id="ref-43">↵</a><div class="cit ref-cit ref-journal" id="cit-2019.12.29.890608v1.43" data-doi="10.1093/sysbio/sys058"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Ronquist</span>  <span class="cit-name-given-names">F.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Klopfstein</span>  <span class="cit-name-given-names">S.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Vilhelmsen</span>  <span class="cit-name-given-names">L.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Schulmeister</span>  <span class="cit-name-given-names">S.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Murray</span>  <span class="cit-name-given-names">D.L.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Rasnitsyn</span>  <span class="cit-name-given-names">A.P</span></span>. <span class="cit-pub-date">2012</span>. <span class="cit-article-title">A total-evidence approach to dating with fossils, applied to the early radiation of the Hymenoptera</span>. <abbr class="cit-jnl-abbrev">Syst. Biol</abbr>. <span class="cit-vol">61</span>:<span class="cit-fpage">973</span>–<span class="cit-lpage">999</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DSyst.%2BBiol%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fsysbio%252Fsys058%26rft_id%253Dinfo%253Apmid%252F22723471%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/sysbio/sys058&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22723471&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F12%2F30%2F2019.12.29.890608.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-44-1" title="View reference  in text" id="ref-44">↵</a><div class="cit ref-cit ref-journal" id="cit-2019.12.29.890608v1.44" data-doi="10.1111/evo.12681"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Shi</span>  <span class="cit-name-given-names">J.J.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Rabosky</span>  <span class="cit-name-given-names">D.L</span></span>. <span class="cit-pub-date">2015</span>. <span class="cit-article-title">Speciation dynamics during the global radiation of extant bats</span>. <abbr class="cit-jnl-abbrev">Evolution</abbr> <span class="cit-vol">69</span>:<span class="cit-fpage">1528</span>–<span class="cit-lpage">1545</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEvolution%26rft.volume%253D69%26rft.spage%253D1528%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fevo.12681%26rft_id%253Dinfo%253Apmid%252F25958922%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/evo.12681&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25958922&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F12%2F30%2F2019.12.29.890608.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-45-1" title="View reference  in text" id="ref-45">↵</a><div class="cit ref-cit ref-journal" id="cit-2019.12.29.890608v1.45"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Smith</span>  <span class="cit-name-given-names">A.B.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Godsoe</span>  <span class="cit-name-given-names">W.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Rodríguez-Sánchez</span>  <span class="cit-name-given-names">F.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">H.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Warren</span>  <span class="cit-name-given-names">D</span></span>. <span class="cit-pub-date">2018</span>. <span class="cit-article-title">Niche estimation above and below the species level</span>. <abbr class="cit-jnl-abbrev">Trends Ecol. Evol</abbr>. <span class="cit-vol">34</span>:<span class="cit-fpage">260</span>–<span class="cit-lpage">273</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTrends%2BEcol.%2BEvol%26rft.volume%253D34%26rft.spage%253D260%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-46-1" title="View reference  in text" id="ref-46">↵</a><div class="cit ref-cit ref-journal" id="cit-2019.12.29.890608v1.46" data-doi="10.1080/01621459.1926.10502161"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Sturges</span>  <span class="cit-name-given-names">H.A</span></span>. <span class="cit-pub-date">1926</span>. <span class="cit-article-title">The choice of a class interval</span>. <abbr class="cit-jnl-abbrev">J. Am. Stat. Assoc</abbr>. <span class="cit-vol">21</span>:<span class="cit-fpage">65</span>–<span class="cit-lpage">66</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BAm.%2BStat.%2BAssoc%26rft.volume%253D21%26rft.spage%253D65%26rft_id%253Dinfo%253Adoi%252F10.1080%252F01621459.1926.10502161%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1080/01621459.1926.10502161&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000188338800009&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-47-1" title="View reference  in text" id="ref-47">↵</a><div class="cit ref-cit ref-journal" id="cit-2019.12.29.890608v1.47" data-doi="10.1093/sysbio/syu057"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Uyeda</span>  <span class="cit-name-given-names">J.C.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Harmon</span>  <span class="cit-name-given-names">L.J</span></span>. <span class="cit-pub-date">2014</span>. <span class="cit-article-title">A novel Bayesian method for inferring and interpreting the dynamics of adaptive landscapes from phylogenetic comparative data</span>. <abbr class="cit-jnl-abbrev">Syst. Biol</abbr>. <span class="cit-vol">63</span>:<span class="cit-fpage">902</span>– <span class="cit-lpage">918</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DSyst.%2BBiol%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fsysbio%252Fsyu057%26rft_id%253Dinfo%253Apmid%252F25077513%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/sysbio/syu057&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25077513&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F12%2F30%2F2019.12.29.890608.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-48-1" title="View reference  in text" id="ref-48">↵</a><div class="cit ref-cit ref-journal" id="cit-2019.12.29.890608v1.48"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Weber</span>  <span class="cit-name-given-names">M.M.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Stevens</span>  <span class="cit-name-given-names">R.D.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Diniz-Filho</span>  <span class="cit-name-given-names">J.A.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Grelle</span>  <span class="cit-name-given-names">C.E</span></span>. <span class="cit-pub-date">2017</span>. <span class="cit-article-title">Is there a correlation between abundance and environmental suitability derived from ecological niche modelling? A meta-analysis</span>. <abbr class="cit-jnl-abbrev">Ecography</abbr> <span class="cit-vol">40</span>: <span class="cit-fpage">817</span>–<span class="cit-lpage">828</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEcography%26rft.volume%253D40%26rft.spage%253D817%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-49-1" title="View reference  in text" id="ref-49">↵</a><div class="cit ref-cit ref-journal" id="cit-2019.12.29.890608v1.49"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Williams</span>  <span class="cit-name-given-names">K.J.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Belbin</span>  <span class="cit-name-given-names">L.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Austin</span>  <span class="cit-name-given-names">M.P.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Stein</span>  <span class="cit-name-given-names">J.L.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Ferrier</span>  <span class="cit-name-given-names">S</span></span>. <span class="cit-pub-date">2012</span>. <span class="cit-article-title">Which environmental variables should I use in my biodiversity model?</span> <abbr class="cit-jnl-abbrev">Int. J. Geogr. Inf. Sci</abbr>. <span class="cit-vol">26</span>:<span class="cit-fpage">2009</span>–<span class="cit-lpage">2047</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DInt.%2BJ.%2BGeogr.%2BInf.%2BSci%26rft.volume%253D26%26rft.spage%253D2009%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li></ol></div><div class="section ref-list" id="ref-list-2"><h2 class="">Reference</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label">1.</span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-2019.12.29.890608v1.50"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Theerawitaya</span>  <span class="cit-name-given-names">C.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Tisarum</span>  <span class="cit-name-given-names">R.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Samphumphuang</span>  <span class="cit-name-given-names">T.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Singh</span>  <span class="cit-name-given-names">H.P.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Cha-Um</span>  <span class="cit-name-given-names">S.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Kirdmanee</span>  <span class="cit-name-given-names">C.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Takabe</span>  <span class="cit-name-given-names">T</span></span>. <span class="cit-pub-date">2015</span>. <span class="cit-article-title">Physio-biochemical and morphological characters of halophyte legume shrub, <em>Acacia ampliceps</em> seedlings in response to salt stress under greenhouse</span>. <abbr class="cit-jnl-abbrev">Front. Plant Sci</abbr>. <span class="cit-vol">6</span>:<span class="cit-fpage">630</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DFront.%2BPlant%2BSci%26rft.volume%253D6%26rft.spage%253D630%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">2.</span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-2019.12.29.890608v1.51"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Miah</span>  <span class="cit-name-given-names">M.A</span></span>. <span class="cit-pub-date">2013</span>. <span class="cit-article-title">Salt tolerances of some mainland tree species select as through nursery screening</span>. <abbr class="cit-jnl-abbrev">Pak. J. Biol. Sci</abbr>. <span class="cit-vol">16</span>:<span class="cit-fpage">945</span>−<span class="cit-lpage">949</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPak.%2BJ.%2BBiol.%2BSci%26rft.volume%253D16%26rft.spage%253D945%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">3.</span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-2019.12.29.890608v1.52"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Patel</span>  <span class="cit-name-given-names">A.D.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Jadeja</span>  <span class="cit-name-given-names">H.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Pandey</span>  <span class="cit-name-given-names">A.N</span></span>. <span class="cit-pub-date">2010</span>. <span class="cit-article-title">Effect of salinization of soil on growth, water status and nutrient accumulation in seedlings of <em>Acacia auriculiformis</em> (Fabaceae)</span>. <abbr class="cit-jnl-abbrev">J. Plant Nutrition</abbr> <span class="cit-vol">33</span>: <span class="cit-fpage">914</span>−<span class="cit-lpage">932</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BPlant%2BNutrition%26rft.volume%253D33%26rft.spage%253D914%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">4.</span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-2019.12.29.890608v1.53"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Craig</span>  <span class="cit-name-given-names">G.F.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Bell</span>  <span class="cit-name-given-names">D.T.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Atkins</span>  <span class="cit-name-given-names">C.A</span></span>., <span class="cit-pub-date">1990</span>. <span class="cit-article-title">Response to salt and waterlogging stress of ten taxa of Acacia selected from naturally saline areas of Western Australia</span>. <abbr class="cit-jnl-abbrev">Aust. J. Bot</abbr>. <span class="cit-vol">38</span>:<span class="cit-fpage">619</span>−<span class="cit-lpage">630</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAust.%2BJ.%2BBot%26rft.volume%253D38%26rft.spage%253D619%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">5.</span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-2019.12.29.890608v1.54" data-doi="10.1071/RJ06027"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Reichman</span>  <span class="cit-name-given-names">S.M.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Bellairs</span>  <span class="cit-name-given-names">S.M.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Mulligan</span>  <span class="cit-name-given-names">D.R</span></span>. <span class="cit-pub-date">2006</span>. <span class="cit-article-title">The effects of temperature and salinity on <em>Acacia harpophylla</em> (brigalow) (Mimosaceae) germination</span>. <abbr class="cit-jnl-abbrev">Rangeland J</abbr>. <span class="cit-vol">28</span>:<span class="cit-fpage">175</span>−<span class="cit-lpage">178</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DRangeland%2BJ%26rft.volume%253D28%26rft.spage%253D175%26rft_id%253Dinfo%253Adoi%252F10.1071%252FRJ06027%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1071/RJ06027&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000241880400009&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">6.</span><div class="cit ref-cit ref-book no-rev-xref" id="cit-2019.12.29.890608v1.55"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Aswathappa</span>  <span class="cit-name-given-names">N.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Marcar</span>  <span class="cit-name-given-names">N.E.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Thompson</span>  <span class="cit-name-given-names">L.A</span></span>., <span class="cit-pub-date">1987</span>. <span class="cit-source">Salt tolerance of Australian tropical and subtropical Acacias</span>. In: <span class="cit-auth"><span class="cit-name-surname">Turnbull</span>  <span class="cit-name-given-names">J.</span></span> (Ed.), <span class="cit-publ-name">Australian Acacias in Developing Countries</span>. ACIAR Proceedings, No. 16, pp. <span class="cit-fpage">70</span>−<span class="cit-lpage">73</span>.</cite></div><div class="cit-extra"></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.biorxiv.org/content/10.1101/2019.12.29.890608v1.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-disqus-comment" >
  
      
  
  <div class="pane-content">
    <div id="disqus_thread"><noscript><p><a href="http://biorxivstage.disqus.com/?url=https%3A%2F%2Fwww.biorxiv.org%2Fcontent%2F10.1101%2F2019.12.29.890608v1" class="" data-icon-position="" data-hide-link-title="0">View the discussion thread.</a></p></noscript></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/10.1101/2019.12.29.885731v1" title="Conglomeration of highly antigenic nucleoproteins to inaugurate a heterosubtypic next generation vaccine candidate against Arenaviridae family" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous</span></a></span><span class="pager-next"><a href="/content/10.1101/569319v4" title="Neonatal morphometric similarity mapping for predicting brain age and characterizing neuroanatomic variation associated with preterm birth" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    Posted&nbsp;December 30, 2019.  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-biorxiv-art-tools" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-biorxiv_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/10.1101/2019.12.29.890608v1.full.pdf" target="_self" class="article-dl-pdf-link link-icon"><i class="icon-external-link-sign"></i> <span class="title">Download PDF</span></a>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-biorxiv-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="Email this Article" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-biorxiv_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/10.1101/2019.12.29.890608v1.full" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for your interest in spreading the word about bioRxiv.</p><p>NOTE: Your email address is requested solely to identify you as the sender of this article.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/10.1101/2019.12.29.890608v1" class="active" data-icon-position="" data-hide-link-title="0">Tracking niche change through time: simultaneous inference of environmental niche evolution and estimation of contemporary niches</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from bioRxiv
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would like to see this page from the bioRxiv website.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/1089946" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-467Nf6-djjcMyHX6AtIIvtWbdjtEuISKS15duyAZ8BE" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-biorxiv-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-biorxiv_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="1089946" id="node1089946--2" data-pisa="biorxiv;2019.12.29.890608v1" data-pisa-master="biorxiv;2019.12.29.890608" data-seqnum="1089946" data-apath="/biorxiv/early/2019/12/30/2019.12.29.890608.atom"><div  class="highwire-cite highwire-cite-highwire-article highwire-citation-biorxiv-article-pap-list clearfix" >

      <div class="highwire-cite-title" >
      <div class="highwire-cite-title">Tracking niche change through time: simultaneous inference of environmental niche evolution and estimation of contemporary niches</div>    </div>
  
      <div  class="highwire-cite-authors" ><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0"><span class="nlm-given-names">Xia</span> <span class="nlm-surname">Hua</span></span>, <span class="highwire-citation-author" data-delta="1"><span class="nlm-given-names">Marcel</span> <span class="nlm-surname">Cardillo</span></span>, <span class="highwire-citation-author" data-delta="2"><span class="nlm-given-names">Lindell</span> <span class="nlm-surname">Bromham</span></span></span></div>
  
      <div  class="highwire-cite-metadata" ><span  class="highwire-cite-metadata-journal highwire-cite-metadata">bioRxiv </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">2019.12.29.890608; </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata"><span class="doi_label">doi:</span> https://doi.org/10.1101/2019.12.29.890608 </span></div>
  
  
  </div>
</div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.biorxiv.org/content/10.1101/2019.12.29.890608v1" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.biorxiv.org/content/10.1101/2019.12.29.890608v1&amp;title=Tracking%20niche%20change%20through%20time%3A%20simultaneous%20inference%20of%20environmental%20niche%20evolution%20and%20estimation%20of%20contemporary%20niches" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.biorxiv.org/content/10.1101/2019.12.29.890608v1&amp;title=Tracking%20niche%20change%20through%20time%3A%20simultaneous%20inference%20of%20environmental%20niche%20evolution%20and%20estimation%20of%20contemporary%20niches" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.biorxiv.org/content/10.1101/2019.12.29.890608v1&amp;text=Tracking%20niche%20change%20through%20time%3A%20simultaneous%20inference%20of%20environmental%20niche%20evolution%20and%20estimation%20of%20contemporary%20niches" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.biorxiv.org/content/10.1101/2019.12.29.890608v1&amp;title=Tracking%20niche%20change%20through%20time%3A%20simultaneous%20inference%20of%20environmental%20niche%20evolution%20and%20estimation%20of%20contemporary%20niches" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.biorxiv.org/content/10.1101/2019.12.29.890608v1&amp;t=Tracking%20niche%20change%20through%20time%3A%20simultaneous%20inference%20of%20environmental%20niche%20evolution%20and%20estimation%20of%20contemporary%20niches" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.biorxiv.org/content/10.1101/2019.12.29.890608v1&amp;title=Tracking%20niche%20change%20through%20time%3A%20simultaneous%20inference%20of%20environmental%20niche%20evolution%20and%20estimation%20of%20contemporary%20niches" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.biorxiv.org/content/10.1101/2019.12.29.890608v1&amp;title=Tracking%20niche%20change%20through%20time%3A%20simultaneous%20inference%20of%20environmental%20niche%20evolution%20and%20estimation%20of%20contemporary%20niches" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-biorxiv-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title="Citation Tools"><i class="icon-globe"></i> <span class="title">Citation Tools</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-biorxiv_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="1089946" id="node1089946--3" data-pisa="biorxiv;2019.12.29.890608v1" data-pisa-master="biorxiv;2019.12.29.890608" data-seqnum="1089946" data-apath="/biorxiv/early/2019/12/30/2019.12.29.890608.atom"><div  class="highwire-cite highwire-cite-highwire-article highwire-citation-biorxiv-article-pap-list clearfix" >

      <div class="highwire-cite-title" >
      <div class="highwire-cite-title">Tracking niche change through time: simultaneous inference of environmental niche evolution and estimation of contemporary niches</div>    </div>
  
      <div  class="highwire-cite-authors" ><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0"><span class="nlm-given-names">Xia</span> <span class="nlm-surname">Hua</span></span>, <span class="highwire-citation-author" data-delta="1"><span class="nlm-given-names">Marcel</span> <span class="nlm-surname">Cardillo</span></span>, <span class="highwire-citation-author" data-delta="2"><span class="nlm-given-names">Lindell</span> <span class="nlm-surname">Bromham</span></span></span></div>
  
      <div  class="highwire-cite-metadata" ><span  class="highwire-cite-metadata-journal highwire-cite-metadata">bioRxiv </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">2019.12.29.890608; </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata"><span class="doi_label">doi:</span> https://doi.org/10.1101/2019.12.29.890608 </span></div>
  
  
  </div>
</div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage&amp;rft.epage&amp;rft.atitle=Tracking%20niche%20change%20through%20time%3A%20simultaneous%20inference%20of%20environmental%20niche%20evolution%20and%20estimation%20of%20contemporary%20niches&amp;rft.volume&amp;rft.issue&amp;rft.date=2019-01-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=bioRxiv&amp;rft.au=Hua%2C+Xia&amp;rft.au=Cardillo%2C+Marcel&amp;rft.au=Bromham%2C+Lindell"></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/1089946/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/1089946/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/1089946/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/1089946/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/1089946/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/1089946/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/1089946/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/1089946/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/1089946/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/1089946/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/1089946/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/1089946/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><div class="item-list"><ul><li class="first"><a href="http://twitter.com/share?url=https%3A//www.biorxiv.org/content/10.1101/2019.12.29.890608v1&amp;count=horizontal&amp;via=&amp;text=Tracking%20niche%20change%20through%20time%3A%20simultaneous%20inference%20of%20environmental%20niche%20evolution%20and%20estimation%20of%20contemporary%20niches&amp;counturl=https%3A//www.biorxiv.org/content/10.1101/2019.12.29.890608v1" class="twitter-share-button service-links-twitter-widget" id="twitter_widget" title="Tweet This" rel="nofollow" data-icon-position="" data-hide-link-title="0"><span class="element-invisible">Tweet Widget</span></a></li><li><a href="http://www.facebook.com/plugins/like.php?href=https%3A//www.biorxiv.org/content/10.1101/2019.12.29.890608v1&amp;layout=button_count&amp;show_faces=false&amp;action=like&amp;colorscheme=light&amp;width=100&amp;height=21&amp;font=&amp;locale=" id="facebook_like" title="I Like it" class="service-links-facebook-like" rel="nofollow" data-icon-position="" data-hide-link-title="0"><span class="element-invisible">Facebook Like</span></a></li><li class="last"><a href="https://www.biorxiv.org/content/10.1101/2019.12.29.890608v1" id="google_plus_one" title="Plus it" class="service-links-google-plus-one" rel="nofollow" data-icon-position="" data-hide-link-title="0"><span class="element-invisible">Google Plus One</span></a></li></ul></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-collections" >
  
        <h2 class="pane-title">Subject Area</h2>
    
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-collections"><div class="highwire-list"><ul class="highwire-article-collection-term-list"><li class="first last odd"><span class="highwire-article-collection-term"><a href="/collection/evolutionary-biology" class="highlight" data-icon-position="" data-hide-link-title="0">Evolutionary Biology<i class="icon-caret-right"></i>
</a></span></li></ul></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-biorxiv-subject-collections block-style-col2" >
  
      
  
  <div class="pane-content">
    <div class="panel-flexible panels-flexible-8 clearfix" id="mini-panel-biorxiv_subject_collections">
<div class="panel-flexible-inside panels-flexible-8-inside">
<div class="panels-flexible-region panels-flexible-region-8-center panels-flexible-region-first panels-flexible-region-last">
  <div class="inside panels-flexible-region-inside panels-flexible-region-8-center-inside panels-flexible-region-inside-first panels-flexible-region-inside-last">
<div class="panel-pane pane-snippet" >
  
      
  
  <div class="pane-content">
    <div class="snippet biorxiv-subject-areas-table-title" id="biorxiv-subject-areas-table-title">
  
      
  <div class="snippet-content">
    <b>Subject Areas</b>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-snippet" >
  
      
  
  <div class="pane-content">
    <div class="snippet biorxiv-subject-areas-view-papers" id="biorxiv-subject-areas-view-papers">
  
      
  <div class="snippet-content">
    <a href="/content/early/recent"><strong>All Articles</strong></a>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-subject-collections" >
  
      
  
  <div class="pane-content">
    <ul id="collection" class="collection highwire-list-expand"><li class="outer collection depth-2  child first"><div class = "data-wrapper"><a href="/collection/animal-behavior-and-cognition" class="" data-icon-position="" data-hide-link-title="0">Animal Behavior and Cognition</a> <span class = "article-count">(1662)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/biochemistry" class="" data-icon-position="" data-hide-link-title="0">Biochemistry</a> <span class = "article-count">(2789)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/bioengineering" class="" data-icon-position="" data-hide-link-title="0">Bioengineering</a> <span class = "article-count">(1946)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/bioinformatics" class="" data-icon-position="" data-hide-link-title="0">Bioinformatics</a> <span class = "article-count">(10400)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/biophysics" class="" data-icon-position="" data-hide-link-title="0">Biophysics</a> <span class = "article-count">(4243)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/cancer-biology" class="" data-icon-position="" data-hide-link-title="0">Cancer Biology</a> <span class = "article-count">(3272)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/cell-biology" class="" data-icon-position="" data-hide-link-title="0">Cell Biology</a> <span class = "article-count">(4641)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/clinical-trials" class="" data-icon-position="" data-hide-link-title="0">Clinical Trials</a> <span class = "article-count">(135)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/developmental-biology" class="" data-icon-position="" data-hide-link-title="0">Developmental Biology</a> <span class = "article-count">(2889)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/ecology" class="" data-icon-position="" data-hide-link-title="0">Ecology</a> <span class = "article-count">(4544)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/epidemiology" class="" data-icon-position="" data-hide-link-title="0">Epidemiology</a> <span class = "article-count">(2041)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/evolutionary-biology" class="" data-icon-position="" data-hide-link-title="0">Evolutionary Biology</a> <span class = "article-count">(7288)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/genetics" class="" data-icon-position="" data-hide-link-title="0">Genetics</a> <span class = "article-count">(5540)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/genomics" class="" data-icon-position="" data-hide-link-title="0">Genomics</a> <span class = "article-count">(6893)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/immunology" class="" data-icon-position="" data-hide-link-title="0">Immunology</a> <span class = "article-count">(2434)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/microbiology" class="" data-icon-position="" data-hide-link-title="0">Microbiology</a> <span class = "article-count">(7616)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/molecular-biology" class="" data-icon-position="" data-hide-link-title="0">Molecular Biology</a> <span class = "article-count">(3051)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/neuroscience" class="" data-icon-position="" data-hide-link-title="0">Neuroscience</a> <span class = "article-count">(18885)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/paleontology" class="" data-icon-position="" data-hide-link-title="0">Paleontology</a> <span class = "article-count">(142)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/pathology" class="" data-icon-position="" data-hide-link-title="0">Pathology</a> <span class = "article-count">(477)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/pharmacology-and-toxicology" class="" data-icon-position="" data-hide-link-title="0">Pharmacology and Toxicology</a> <span class = "article-count">(791)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/physiology" class="" data-icon-position="" data-hide-link-title="0">Physiology</a> <span class = "article-count">(1178)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/plant-biology" class="" data-icon-position="" data-hide-link-title="0">Plant Biology</a> <span class = "article-count">(2771)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/scientific-communication-and-education" class="" data-icon-position="" data-hide-link-title="0">Scientific Communication and Education</a> <span class = "article-count">(686)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/synthetic-biology" class="" data-icon-position="" data-hide-link-title="0">Synthetic Biology</a> <span class = "article-count">(905)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/systems-biology" class="" data-icon-position="" data-hide-link-title="0">Systems Biology</a> <span class = "article-count">(2881)</span></div></li>
<li class="outer collection depth-2  child last"><div class = "data-wrapper"><a href="/collection/zoology" class="" data-icon-position="" data-hide-link-title="0">Zoology</a> <span class = "article-count">(472)</span></div></li>
</ul>  </div>

  
  </div>
  </div>
</div>
</div>
</div>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__VNH5GD8Zz7g6_hGCOZjVjdMndxxC6naiExSSJKX3k_A__kzmEdtbqCsJx5Z9yg2Qy0PptB__ufXm-TxUOl0KZzUw__QJacVmveFyLIBSPkp21xL0ZvuDW9DMBrD2HB1P4Ry5I.js"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__QJacVmveFyLIBSPkp21xL0ZvuDW9DMBrD2HB1P4Ry5I.js" defer="defer"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__N7ERJBYsOWyRnJgyoM125_Aiez2MOJGaUofG1JdWWBg__bLlcwhzZhT-IwoQ_dvm1_xPxU7E7gecFLC6Fyeqtv8s__QJacVmveFyLIBSPkp21xL0ZvuDW9DMBrD2HB1P4Ry5I.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
function euCookieComplianceLoadScripts() {}
//--><!]]>
</script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var eu_cookie_compliance_cookie_name = "";
//--><!]]>
</script>
<script type="text/javascript" src="//d33xdlntwy0kbs.cloudfront.net/cshl_custom.js"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__BmqjBnkz3MgYCAoc25s1lDRMEjLhC3mEPVonUFIHi08__Unwv5-ZIuHBfFwytsjEx1niBVJ7n1T4lPws7VrkHXM4__QJacVmveFyLIBSPkp21xL0ZvuDW9DMBrD2HB1P4Ry5I.js"></script>
  </body>
</html>
