<html xmlns="http://www.w3.org/1999/xhtml"> <head> <style type="text/css">
      	     		 
      	    a                          {background : #ffffff; }
      	    article                    {border-style : dotted; border-width : 2px; }
		 	
		 	div                        {background : #ffffcc;}
		 	div.abstract-title         {font-weight : bold ; font-size : 16pt;}
		 	div.ack                    {border-style : solid ; border-color : red; margin : 2em; }
		 	div.article-meta           {border-style : solid; border-width : 1 pt; margin 1em;}
		 	div.boxed-text             {margin : 5em; border-style : solid;}
		 	div.contrib-group          {margin : 1em; }
		 	div.fig                    {border-style : solid; border-width : 2px; margin : 2em; }
		 	div.funding                {font-weight : bold ; font-size : 16pt;}
		 	div.given-names            {font-style : italic;}
		 	div.intro                  {border-style : inset; margin : 5px;}
		 	div.introduction           {border-style : inset; margin : 5px;}
		 	div.journal-meta           {border-style : solid; border-width : 1 pt; margin 1em;}
		 	div.journal-title-group    {background : #ffeeee;}
		 	div.article-title          {font-weight : bold ; font-size : 18pt;}
		 	div.kwd                    {font-style : italic}
		 	div.meta-name              {font-weight : bold ; font-size : 16pt;}
		 	div.name                   {font-weight : bold;}
		 	div.alt-title              {font-style : italic; font-size : 12px;}
		 	
		 	div.sec                    {border : 2px; margin 5px; padding 2px;}
		 	div.title                  {font-family : courier; font-weight : bold;
		 	                            font-size : 16pt; margin : 5px;}
		 	
		 	div.materials_methods      {border-style : double; margin : 5px; }
		 	div.methods                {border-style : double; margin : 5px; }
		 	
		 	div.results                {border-style : solid; margin : 5px;}
		 	div.background             {border-style : dotted; margin : 5px;}
		 	
		 	div.discussion             {border-style : groove; margin : 5px;}
		 	div.conclusion             {border-style : ridge; margin : 5px;}
		 	div.conclusions             {border-style : ridge; margin : 5px;}
		 	div.supplementary-material {border-style : inset; margin : 5px;}
		 	div.abbreviations          {border-style : double; border-color : red; margin : 5px;}
		 	div.competinginterests     {border-style : double; border-color : blue; margin : 5px;}
		 	div.acknowledgements       {border-style : double; border-color : green; margin : 5px;}
		 	div.authors_contributions   {border-style : double; border-color : purple; margin : 5px;}
		 	
		 	div.publisher              {border-style : outset; margin : 5px;}
		 	div.fn-type-conflict       {background : #f88; }
		 	div.fn-type-con            {background : #ddf; }
		 	div.fn-type-other          {background : #ddd; }
		 	
		 	div.unknown                {background : #ffd;
									 	  border-style : solid;
									 	  border-width : 1px;
									 	  padding : 2 px;}
		 	  
      	    table                      {background : #ffffdd;}
		 	tr                         {background : #ddddff; padding : 1px;}
		 	
		 	span                       {background : #ffcccc;}
		 	
		 	span.citation-author       {font-family : helvetica; background : #ffeeee;}
		 	span.collab                {background : #ddffff; }
		 	span.comment               {font-family : courier; font-size : 6px; background : #ffaaff;}
		 	span.contrib               {background : #ffffff;}
		 	span.corresp               {background : #ddffdd; }
		 	span.doi                   {background : #ffffff;}
		 	span.email                 {font-family : courier; }
		 	span.etal                  {font-style : italic;}
		 	span.fpage                 {font-family : courier;}
		 	span.given-names           {background : #ffffff;}
		 	span.iso-abbrev            {background : #ffffff;}
		 	span.issn-epub             {background : #ffffff;}
		 	span.issn-ppub             {background : #ffffff;}
		 	span.journal-title         {background : #ffffff;}
		 	span.lpage                 {font-family : courier;}
		 	span.mixed-article-title   {font-style : italic ;}
		 	span.nlm-ta                {background : #ffffff;}
		 	span.pmc                   {background : #ffffff;}
		 	span.pmcid                 {background : #ffffff;}
		 	span.pmid                  {background : #ffffff;}
		 	span.publisher             {background : #ffffff;}
		 	span.publisher-id          {background : #ffffff;}
		 	span.publisher-name        {background : #ffffff;}
		 	span.source                {background : #ffffff;}
		 	span.subject               {background : #ffffff;}
		 	span.surname               {background : #ffffff;}
		 	span.volume                {font-family : courier; font-weight : bold;}
		 	span.year                  {font-family : courier ; font-style : italic;}
			</style> </head> <body> <div class="front" title="front"> <div class="journal-meta" tagx="journal-meta" title="journal-meta"><span class="nlm-ta" title="nlm-ta">Mol Biol Evol</span><span class="iso-abbrev" title="iso-abbrev">Mol. Biol. Evol</span><span class="publisher-id" title="publisher-id">molbev</span><div class="journal-title-group" tagx="journal-title-group" title="journal-title-group"><span class="journal-title" tagx="journal-title" title="journal-title">Molecular Biology and Evolution</span></div><span class="issn-ppub" tagx="issn" title="issn-ppub">0737-4038</span><span class="issn-epub" tagx="issn" title="issn-epub">1537-1719</span><div class="publisher" tagx="publisher" title="publisher"><span class="publisher-name" tagx="publisher-name" title="publisher-name">Oxford University Press</span></div> </div> <div class="article-meta" tagx="article-meta" title="article-meta"><span class="pmid" title="pmid"> pmid: <a href="http://www.ncbi.nlm.nih.gov/pubmed/29562344">29562344</a></span><span class="doi" title="doi"> doi: <a href="https://dx.doi.org/10.1093/molbev/msy028">10.1093/molbev/msy028</a></span><span class="publisher-id" title="publisher-id">msy028</span><div class="article-categories" title="article-categories">
: <span class="subject" title="subject">Discoveries</span></div> <div class="title-group" tagx="title-group" title="title-group"> <div class="article-title" title="article-title">
The Birth and Death of Olfactory Receptor Gene Families in Mammalian Niche Adaptation</div> </div> <div class="contrib-group" title="contrib-group"><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hughes</span><span class="given-names" tagx="given-names" title="given-names">Graham M</span></span><a href="#msy028-aff1">1</a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Boston</span><span class="given-names" tagx="given-names" title="given-names">Emma S M</span></span><a href="#msy028-aff2">2</a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Finarelli</span><span class="given-names" tagx="given-names" title="given-names">John A</span></span><a href="#msy028-aff1">1</a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Murphy</span><span class="given-names" tagx="given-names" title="given-names">William J</span></span><a href="#msy028-aff3">3</a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Higgins</span><span class="given-names" tagx="given-names" title="given-names">Desmond G</span></span><a href="#msy028-aff4">4</a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Teeling</span><span class="given-names" tagx="given-names" title="given-names">Emma C</span></span><a href="#msy028-cor1" /><a href="#msy028-aff1">1</a></span></div> <div class="contrib-group" title="contrib-group"><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Satta</span><span class="given-names" tagx="given-names" title="given-names">Yoko</span></span><span class="role" tagx="role" title="role">Associate editor</span></span></div><span class="citation_author_institution" id="msy028-aff1">[1], </span><span class="citation_author_institution" id="msy028-aff2">[2], </span><span class="citation_author_institution" id="msy028-aff3">[3], </span><span class="citation_author_institution" id="msy028-aff4">[4], </span><div class="author-notes" title="author-notes"> <div class="corresp" title="corresp"><b>Corresponding author:</b> E-mail: <span class="email" tagx="email" title="email">emma.teeling@ucd.ie</span>.</div> <div class="fn-type-" title=""> <p>Sequence data generated in this study can be accessed using Genbank accession numbers KU833289–KU836505.</p> </div> </div><span class="pub-date-ppub" title="pub-date-ppub">ppub: <span>2018-6</span></span><span class="pub-date-epub" title="pub-date-epub">epub: <span>2018-3-3</span></span><span class="pub-date-pmc-release" title="pub-date-pmc-release">pmc-release: <span>2018-3-3</span></span><span class="volume" tagx="volume" title="volume">35</span><span class="issue" tagx="issue" title="issue">6</span><span class="fpage" tagx="fpage" title="fpage">1390</span><span class="lpage" tagx="lpage" title="lpage">1406</span><div class="permissions"><span class="copyright" title="copyright">(C) , <span class="copyright-year" tagx="copyright-year" title="copyright-year">2018</span></span><span class="license" title="license"><span class="license-p" title="license-p">This is an Open Access article distributed under the terms of the Creative Commons Attribution Non-Commercial License (<a href="http://creativecommons.org/licenses/by-nc/4.0/">http://creativecommons.org/licenses/by-nc/4.0/</a>), which permits non-commercial re-use, distribution, and reproduction in any medium, provided the original work is properly cited. For commercial re-use, please contact journals.permissions@oup.com</span></span></div><a class="self-uri" href="msy028.pdf" title="self-uri">msy028.pdf</a><div class="abstract" title="abstract"> <div class="abstract-title" title="abstract-title">
Abstract</div> <div class="title" tagx="title" title="title">
Abstract</div> <p>The olfactory receptor (OR) gene families, which govern mammalian olfaction, have undergone extensive expansion and contraction through duplication and pseudogenization. Previous studies have shown that broadly defined environmental adaptations (e.g., terrestrial vs. aquatic) are correlated with the number of functional and non-functional OR genes retained. However, to date, no study has examined species-specific gene duplications in multiple phylogenetically divergent mammals to elucidate OR evolution and adaptation. Here, we identify the OR gene families driving adaptation to different ecological niches by mapping the fate of species-specific gene duplications in the OR repertoire of 94 diverse mammalian taxa, using molecular phylogenomic methods. We analyze &amp;gt;70,000 OR gene sequences mined from whole genomes, generated from novel amplicon sequencing data, and collated with data from previous studies, comprising one of the largest OR studies to date. For the first time, we demonstrate statistically significant patterns of OR species-specific gene duplications associated with the presence of a functioning vomeronasal organ. With respect to dietary niche, we uncover a novel link between a large number of duplications in OR family 5/8/9 and herbivory. Our results also highlight differences between social and solitary niches, indicating that a greater OR repertoire expansion may be associated with a solitary lifestyle. This study demonstrates the utility of species-specific duplications in elucidating gene family evolution, revealing how the OR repertoire has undergone expansion and contraction with respect to a number of ecological adaptations in mammals.</p> </div> <div class="kwd-group"> <div class="kwd" title="kwd">
olfactory receptors</div> <div class="kwd" title="kwd">
mammalian evolution</div> <div class="kwd" title="kwd">
ecological niche</div> <div class="kwd" title="kwd">
adaptation</div> <div class="kwd" title="kwd">
comparative genomics</div> </div> <div class="funding-group" title="funding-group"> <div class="funding" title="funding">
Funding</div> <div class="award-group" title="award-group"><span class="award-id" tagx="award-id" title="award-id">Graduate Research Education Programme (GREP)</span></div> <div class="award-group" title="award-group"> <div class="funding-source" title="funding-source"><span class="named-content" title="named-content">Science Foundation Ireland</span><span class="named-content" title="named-content">10.13039/501100001602</span></div><span class="award-id" tagx="award-id" title="award-id">PIYRA 06/YI3/B932</span><span class="award-id" tagx="award-id" title="award-id">EF0629849</span></div> </div> <div class="counts" title="counts"><span class="page-count" title="page-count">page-count: </span></div> </div> </div> <div class="body" title="body"> <div class="intro" title="intro"> <div class="title" tagx="title" title="title">
Introduction</div> <p>More than 5,400 species of extant mammals have been described to date, with a shared evolutionary history dating back at least 200 My (<a href="#msy028-B70">Luo 2007</a>; <a href="#msy028-B74">Meredith et al. 2011</a>; <a href="#msy028-B22">Dos Reis et al. 2014</a>). Mammals have successfully exploited a wide range of ecological niches, and have varied habitats, diets, social structures, and rhythmic activity phases. These adaptations involve the accumulation of different biological characteristics (<a href="#msy028-B94">Peterson et al. 1999</a>), involving changes in physiology, feeding ecology, and sensory perception. An example of this is diet, which has undergone numerous shifts to meet the energetic demands of novel environments (<a href="#msy028-B68">Luca et al. 2010</a>). Adaptation to a new energy source requires changes at the genetic, metabolic and morphological levels. As feeding is multi-sensorial, dietary shifts also involve a number of changes to visual, olfactory and taste perception (<a href="#msy028-B68">Luca et al. 2010</a>).</p> <p>The Olfactory Receptor (OR) repertoire is the set of G-protein coupled receptor (GPCR) genes responsible for the perception of chemosensory information, found mainly in the cilia of the olfactory epithelium of the nasal passage in vertebrates. It is the largest multigene family in vertebrates, typically accounting for ∼4–5% of protein coding genes in mammals (<a href="#msy028-B46">Hayden et al. 2010</a>). OR genes are intron-less and roughly 1 kb in size, with each mammalian species having an average of 1,259 genes (including both functional and non-functional genes; <a href="#msy028-B46">Hayden et al. 2010</a>). The OR repertoire can be split into Class I receptors (binds water-borne odorants) and Class II receptors (binds mainly volatile odorants). These classes are split into four families (OR 51, OR52, OR55, and OR56) and nine OR gene families (OR 1/3/7, OR 2/13, OR 4, OR 5/8/9, OR 6, OR 10, OR 11, OR 12, and OR 14) respectively, with each family also having a range of smaller subfamilies. The OR gene repertoire follows a "birth and death" model of gene evolution, expanding through processes such as tandem gene duplication (<a href="#msy028-B80">Nei and Rooney 2005</a>). ORs are connected to the main olfactory bulb (MOB) in the forebrain via OR neuron axons (<a href="#msy028-B27">Farbiszewski and Kranc 2013</a>). The posterior region of the MOB also contains the accessory olfactory bulb, projecting to the vomeronasal organ (VNO). The VNO is involved in binding pheromones and contains its own multigene family receptors, the vomeronasal receptors (V1R, V2R, and formyl peptide receptors; <a href="#msy028-B131">Young et al. 2010</a>), enabling an additional form of chemosensory perception.</p> <p>The mammalian OR repertoire consists of functional and non-functional genes. A number of these genes may have been present in the most recent common ancestor of a target species and its closest living sister taxon, possibly with the same or similar functionality. However, with adaptation to new or changing environments and gene duplications giving rise to new ORs, the fixation or loss of novel genes can be specific to an individual species and/or that species’ ecological niche. Previous research has focused on total number of OR genes in an attempt to characterize how the distribution of genes across OR gene families differs as a function of ecological niche adaptation. In 2010, Hayden et al., using normalized total gene counts, showed that mammals can be categorized into different ecological habitats based on the distribution of OR genes per gene family, highlighting differences between aquatic, volant, and terrestrial mammalian OR repertoires. A further association between the evolution of OR family 1/3/7 with frugivory among bat species has also been demonstrated (<a href="#msy028-B47">Hayden et al. 2014</a>). Similar associations between niche and OR gene families have also been documented in birds (<a href="#msy028-B59">Khan et al. 2015</a>). Yet total counts cannot enable a full understanding of how the OR repertoire has evolved. To truly elucidate the role of OR gene families in mammalian evolution, paralogous OR genes born through duplication events specific to a target species must be uncovered, and the subsequent fate of these OR genes (e.g., retained, lost, or gained new function) must be assessed. Retention of functionality in the new gene after duplication suggests either a critical role of the original gene or "neofunctionalization", thus a novel role for that gene, perhaps in some form of environmental adaptation.</p> <p>Here, we explored which OR gene families are potentially driving adaptation to different ecological niches by uncovering the fate of paralogous OR genes resulting from species-specific gene duplications. We identified these genes that have been retained in the genomes of 94 diverse mammalian taxa using molecular phylogenomic methods. We analyzed &amp;gt;70,000 OR gene sequences to investigate if such OR gene duplication and subsequent fate of the daughter OR correlates with the presence or absence of a functional VNO, dietary niche, habitat, sociality, and rhythmic activity phase. These data were mined from publicly available whole-genome sequences, generated from de novo OR amplicon next generation sequencing (NGS) data (14 species) and collated with OR genes from previous studies. This data set represents one of the largest mammalian OR studies to date, in terms of taxonomic and genic representation. Our results show that OR gene expansion correlates with the presence versus absence of a functioning VNO, emphasizing the link between odorant and pheromone chemosensory detection mechanisms. We show that there are statistically significant patterns of gene duplications with respect to OR gene families and dietary niche. Specifically, OR gene family 5/8/9 shows a large number of gene duplications and retention of function in postduplication paralogs across a wide range of mammalian species with an herbivorous diet. We find significant differences in the number of OR gene duplication events across ecological niches with respect to habitat and sociality but show that rhythmic activity phase (time when a species is most active) does not correlate with OR repertoire expansion. Our results shed light on how the olfactory gene repertoire has undergone expansion and contraction with respect to a variety of ecological niche adaptations in mammals and how the analysis of species-specific gene duplications can be used to elucidate the evolution of gene families.</p> </div> <div class="results" title="results"> <div class="title" tagx="title" title="title">
Results</div> <div class="numberofcontigsrecoveredfrom454-ngsampliconsequencing" title="sec"> <div class="title" tagx="title" title="title">
Number of Contigs Recovered from 454-NGS Amplicon Sequencing</div> <p>A total of 744,432 reads were generated for 14 new species using 454 NGS amplicon data (<a href="#sup1">supplementary table S1</a>, <a href="#sup1">Supplementary Material</a> online). Mean read length was 736 bp, with a modal length of 744 bp. The number of contigs generated using the de Bruijn graph assembly methods SOAPdenovo, ABySS, and our clustering method using CD-HIT 454 are displayed in <a href="#sup1">supplementary table S2</a>, <a href="#sup1">Supplementary Material</a> online. Contigs from each different assembly method were compared to determine the optimum method of gene reconstruction given the read data. In the cluster method, a number of various clustering identities were tested. Ultimately, 97% identity was determined as the optimal threshold based on comparisons between reference genomes (dog, cat, little brown bat, and greater horseshoe bat) and read data, for the clustering assembly method. The number of ORs per family, the average number of reads per cluster, the interquartile range across clusters and the maximum cluster sizes are displayed in <a href="#sup1">supplementary tables S2 and S3</a>. The assembly using the SOAPdenovo program recovered the most genes for dog (528 ORs), while clustering recovered the highest number of genes for the cat, little brown bat and greater horseshoe bat (521, 378, and 218, respectively, <a href="#sup1">supplementary table S2</a>, <a href="#sup1">Supplementary Material</a> online).</p> </div> <div class="ngsampliconcomparisons" title="sec"> <div class="title" tagx="title" title="title">
NGS Amplicon Comparisons</div> <p>Each species showed on average 99% sequence identity to their respective target gene, except for contigs generated through clustering in the little brown bat (<a href="#sup1">supplementary table S3</a>, <a href="#sup1">Supplementary Material</a> online). The ABySS assembly had the largest average sequence lengths at 595, 617, 616, and 599 bp for dog, cat, little brown bat and greater horseshoe bat, respectively (<a href="#sup1">supplementary table S3</a>, <a href="#sup1">Supplementary Material</a> online). SOAPdenovo had the lowest number of redundant sequences for the cat, little brown bat and greater horseshoe bat at 12%, 14%, and 13%, respectively (<a href="#sup1">supplementary table S3</a>, <a href="#sup1">Supplementary Material</a> online). The distribution of assembled ORs for the cat, little brown bat and greater horseshoe bat were not significantly different from their respective genomic distributions for all methods of contig assembly (<a href="#sup1">supplementary table S3</a>, <a href="#sup1">Supplementary Material</a> online). For the dog data, the number of OR genes represented by the sequenced amplicons in OR gene family 6 appeared underrepresented, considering the size of this gene family (118 ORs) mined from the <i>C. familiaris</i> genome (<a href="#msy028-B3">Aloni et al. 2006</a>), compared to the number of contigs assembled (27, 36, and 21 ORs; <a href="#sup1">supplementary table S2</a> and <a href="#sup1">fig. S1</a>, <a href="#sup1">Supplementary Material</a> online). The underrepresentation of this family has previously been observed in the dog (<a href="#msy028-B46">Hayden et al. 2010</a>). This is perhaps due to the high number of diverse OR genes in OR family 6 causing problems with amplification using degenerate primers, or problems in the primer design. Comparisons to full repertoires conducted both with and without OR family 6 (<a href="#sup1">supplementary table S3</a>, <a href="#sup1">Supplementary Material</a> online) showed no significant differences. Based on different metrics used to compare assembly versus cluster methods, it was determined that SOAPdenovo was the optimal method to reconstruct the 454 OR amplicon sequence data. The final number of contigs generated for each species using SOAPdenovo is displayed in <a href="#sup1">supplementary table S1</a>, <a href="#sup1">Supplementary Material</a> online (5,114 contigs in total) and were used for further analyses. The species with the smallest number of ORs assembled using NGS methods was <i>Megaptera novaeangliae</i> (humpback whale) at 192 ORs. This low number reflects the small OR repertoire size previously documented in cetaceans (<a href="#msy028-B46">Hayden et al. 2010</a>).</p> </div> <div class="analysisoforrepertoiresfromwgs-basedgenomeassemblies" title="sec"> <div class="title" tagx="title" title="title">
Analysis of OR Repertoires from WGS-Based Genome Assemblies</div> <p>Of the 58 mammals with whole genome sequence (WGS; "Genomic") based data, the species with the highest and lowest number of OR genes were <i>Loxodonta africana</i> (African elephant) and <i>Tursiops truncatus</i> (common bottlenose dolphin) with 4,230 and 156 ORs, respectively (<a href="#sup1">supplementary table S4</a>, <a href="#sup1">Supplementary Material</a> online). Using fully sequenced genomes, an average of 1,213 ORs were found per mammal. The number of OR genes that we considered putatively functional (coding sequence of 650 bp or more with no in-frame stop codons) ranged from 58 (dolphin) to 2,514 (elephant; <a href="#sup1">supplementary table S4</a>, <a href="#sup1">Supplementary Material</a> online), with an average of 794 functional ORs (65% of repertoire) and 419 (35%) ORs considered non-functional per mammal (coding sequence &amp;lt;650 bp or contained in frame stop codon). The full repertoire of functional and nonfunctional ORs in the "Genomic" data set is displayed in <a href="#msy028-F1">figure 1</a>. The total number of genes for each mammalian OR repertoire differs slightly from previous studies such as <a href="#msy028-B81">Niimura and Nei (2007)</a>, <a href="#msy028-B46">Hayden et al. (2010)</a>, and <a href="#msy028-B82">Niimura et al. (2014)</a>, most likely as a consequence of using different methods, workflows and genome assemblies to mine and annotate the OR data. Such discrepancies have also been observed in OR gene studies in the class Aves (<a href="#msy028-B67">Lu et al. 2016</a>). A total of 9,108 OR genes were added from an additional 36 mammals sequenced from OR amplicons (NGS data set; <a href="#sup1">supplementary table S4</a>, <a href="#sup1">Supplementary Material</a> online). The species with the most and fewest assembled ORs are <i>Ursus arctos</i> (brown bear; Illumina) and <i>Globicephala</i> sp. (pilot whale, Sanger sequencing) at 496 ORs and 42 ORs, respectively. </p> <div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/"><span class="label" tagx="label" title="label"> <div class="sc_UNKNOWN">
Fig</div>. 1.</span></a></div>   <p>The full OR repertoire of 58 mammalian genomes. The sequenced genomes of 58 ecologically and phylogenetically diverse mammals were mined for OR gene sequences. The number of functional ORs (yellow bars) and non-functional ORs (grey bars) in the full OR gene repertoire varies between different species. Species are ranked based on the size of their functional repertoire. The coverage of the genomes from which these data were mined, as noted in their assembly builds online, are also included with "C" representing "complete" for human and mouse.</p>   </div> </div> <div class="orspecies-specificgeneduplicationevents" title="sec"> <div class="title" tagx="title" title="title">
OR Species-Specific Gene Duplication Events</div> <p>For the "Genomic" data set, the number of species-specific duplication (SSD) events was determined for all OR gene families using a tree parsing methodology (<a href="#msy028-F2">fig. 2</a>). A total of 22,826 duplications across 58 mammals were detected. These SSD events have given rise to 31,595 paralogous OR genes, 18,907 of which remain functional (postduplication functional) while 12,688 have since lost their function (postduplication non-functional) through pseudogenisation (<a href="#msy028-T1">table 1</a>). OR gene families 5/8/9, 1/3/7, 2/13, and 4 have on average the largest number of SSDs (<a href="#msy028-T1">table 1</a> and <a href="#msy028-F3">fig. 3</a>) and the most functional and non-functional ORs after duplication (<a href="#msy028-T1">table 1</a>). When both daughter ORs have retained function after a duplication event, this may suggest subfunctionalization or neofunctionalization, and differ across taxa (<a href="#msy028-F4">fig. 4</a>). Family 55 had the smallest average number of SSDs suggesting very few gene births, or alternatively, that duplication events have occurred but subsequent loss of function and degradation has made them undetectable. The subsequent fates of OR genes after a duplication for the "Genomic" data are displayed in <a href="#sup1">supplementary figure S2</a>, <a href="#sup1">Supplementary Material</a> online. <div class="table-wrap_UNKNOWN" id="msy028-T1" orientation="portrait" position="float"><span class="label" tagx="label" title="label">Table 1.</span> </div></p><p>The Number of Species-Specific Duplication Events, Paralogous ORs Retaining Function and Paralogous ORs Having Lost Function After Duplication, in Addition to the Average Per OR Family for Each Data Set Are Displayed.</p>  <table frame="hsides" rules="groups"> <colgroup span="1"> <col valign="top" align="left" span="1" /></colgroup> <colgroup><col valign="top" align="char" char="." span="1" /></colgroup> <colgroup><col valign="top" align="char" char="." span="1" /></colgroup> <colgroup><col valign="top" align="char" char="." span="1" /></colgroup> <colgroup><col valign="top" align="char" char="." span="1" /></colgroup> <colgroup><col valign="top" align="char" char="." span="1" /></colgroup> <colgroup><col valign="top" align="char" char="." span="1" /></colgroup>  <thead> <tr> <th rowspan="1" colspan="1" /> <th rowspan="1" colspan="1">Total Duplications</th> <th rowspan="1" colspan="1">Mean</th> <th rowspan="1" colspan="1">Total Functional</th> <th rowspan="1" colspan="1">Mean</th> <th rowspan="1" colspan="1">Total non-functional</th> <th rowspan="1" colspan="1">Mean</th> </tr> </thead> <tbody> <tr> <td colspan="7" rowspan="1">Genomic data (<i>N</i> = 58 mammals)</td> </tr> <tr> <td rowspan="1" colspan="1">OR 1/3/7</td> <td rowspan="1" colspan="1">3,948</td> <td rowspan="1" colspan="1">68.07</td> <td rowspan="1" colspan="1">2,680</td> <td rowspan="1" colspan="1">44.66</td> <td rowspan="1" colspan="1">2,444</td> <td rowspan="1" colspan="1">40.73</td> </tr> <tr> <td rowspan="1" colspan="1">OR 2/13</td> <td rowspan="1" colspan="1">3,518</td> <td rowspan="1" colspan="1">60.55</td> <td rowspan="1" colspan="1">2,820</td> <td rowspan="1" colspan="1">47</td> <td rowspan="1" colspan="1">1,991</td> <td rowspan="1" colspan="1">33.18</td> </tr> <tr> <td rowspan="1" colspan="1">OR 4</td> <td rowspan="1" colspan="1">3,138</td> <td rowspan="1" colspan="1">54.10</td> <td rowspan="1" colspan="1">2,505</td> <td rowspan="1" colspan="1">41.75</td> <td rowspan="1" colspan="1">1,967</td> <td rowspan="1" colspan="1">32.78</td> </tr> <tr> <td rowspan="1" colspan="1">OR 5/8/9</td> <td rowspan="1" colspan="1">5,616</td> <td rowspan="1" colspan="1">96.81</td> <td rowspan="1" colspan="1">4,716</td> <td rowspan="1" colspan="1">78.6</td> <td rowspan="1" colspan="1">959</td> <td rowspan="1" colspan="1">49.31</td> </tr> <tr> <td rowspan="1" colspan="1">OR6</td> <td rowspan="1" colspan="1">1,630</td> <td rowspan="1" colspan="1">28.10</td> <td rowspan="1" colspan="1">1,479</td> <td rowspan="1" colspan="1">24.65</td> <td rowspan="1" colspan="1">861</td> <td rowspan="1" colspan="1">14.35</td> </tr> <tr> <td rowspan="1" colspan="1">OR 10</td> <td rowspan="1" colspan="1">1,336</td> <td rowspan="1" colspan="1">23.03</td> <td rowspan="1" colspan="1">1,415</td> <td rowspan="1" colspan="1">23.58</td> <td rowspan="1" colspan="1">539</td> <td rowspan="1" colspan="1">8.98</td> </tr> <tr> <td rowspan="1" colspan="1">OR 11</td> <td rowspan="1" colspan="1">330</td> <td rowspan="1" colspan="1">5.69</td> <td rowspan="1" colspan="1">316</td> <td rowspan="1" colspan="1">5.26</td> <td rowspan="1" colspan="1">185</td> <td rowspan="1" colspan="1">3.08</td> </tr> <tr> <td rowspan="1" colspan="1">OR 12</td> <td rowspan="1" colspan="1">149</td> <td rowspan="1" colspan="1">2.57</td> <td rowspan="1" colspan="1">128</td> <td rowspan="1" colspan="1">2.13</td> <td rowspan="1" colspan="1">74</td> <td rowspan="1" colspan="1">1.23</td> </tr> <tr> <td rowspan="1" colspan="1">OR 14</td> <td rowspan="1" colspan="1">1,295</td> <td rowspan="1" colspan="1">22.32</td> <td rowspan="1" colspan="1">825</td> <td rowspan="1" colspan="1">13.75</td> <td rowspan="1" colspan="1">37</td> <td rowspan="1" colspan="1">12.28</td> </tr> <tr> <td rowspan="1" colspan="1">OR 51</td> <td rowspan="1" colspan="1">659</td> <td rowspan="1" colspan="1">11.36</td> <td rowspan="1" colspan="1">709</td> <td rowspan="1" colspan="1">11.81</td> <td rowspan="1" colspan="1">341</td> <td rowspan="1" colspan="1">5.68</td> </tr> <tr> <td rowspan="1" colspan="1">OR 52</td> <td rowspan="1" colspan="1">958</td> <td rowspan="1" colspan="1">16.52</td> <td rowspan="1" colspan="1">1,052</td> <td rowspan="1" colspan="1">17.53</td> <td rowspan="1" colspan="1">472</td> <td rowspan="1" colspan="1">7.86</td> </tr> <tr> <td rowspan="1" colspan="1">OR 55</td> <td rowspan="1" colspan="1">11</td> <td rowspan="1" colspan="1">0.19</td> <td rowspan="1" colspan="1">11</td> <td rowspan="1" colspan="1">0.18</td> <td rowspan="1" colspan="1">9</td> <td rowspan="1" colspan="1">0.15</td> </tr> <tr> <td rowspan="1" colspan="1">OR 56</td> <td rowspan="1" colspan="1">239</td> <td rowspan="1" colspan="1">4.12</td> <td rowspan="1" colspan="1">251</td> <td rowspan="1" colspan="1">4.18</td> <td rowspan="1" colspan="1">109</td> <td rowspan="1" colspan="1">1.81</td> </tr> <tr> <td rowspan="1" colspan="1">Total</td> <td rowspan="1" colspan="1">22,826</td> <td rowspan="1" colspan="1">393.55</td> <td rowspan="1" colspan="1">18,907</td> <td rowspan="1" colspan="1">315.11</td> <td rowspan="1" colspan="1">12,688</td> <td rowspan="1" colspan="1">211.46</td> </tr> <tr> <td colspan="7" rowspan="1">Genomic + NGS data (<i>N</i> = 94 mammals)</td> </tr> <tr> <td rowspan="1" colspan="1">OR 1/3/7</td> <td rowspan="1" colspan="1">4,383</td> <td rowspan="1" colspan="1">47.13</td> <td rowspan="1" colspan="1">3,045</td> <td rowspan="1" colspan="1">31.71</td> <td rowspan="1" colspan="1">2,723</td> <td rowspan="1" colspan="1">28.36</td> </tr> <tr> <td rowspan="1" colspan="1">OR 2/13</td> <td rowspan="1" colspan="1">3,809</td> <td rowspan="1" colspan="1">40.96</td> <td rowspan="1" colspan="1">3,110</td> <td rowspan="1" colspan="1">32.39</td> <td rowspan="1" colspan="1">2,175</td> <td rowspan="1" colspan="1">22.65</td> </tr> <tr> <td rowspan="1" colspan="1">OR 4</td> <td rowspan="1" colspan="1">3,238</td> <td rowspan="1" colspan="1">34.82</td> <td rowspan="1" colspan="1">609</td> <td rowspan="1" colspan="1">27.17</td> <td rowspan="1" colspan="1">1,992</td> <td rowspan="1" colspan="1">20.75</td> </tr> <tr> <td rowspan="1" colspan="1">OR 5/8/9</td> <td rowspan="1" colspan="1">6,096</td> <td rowspan="1" colspan="1">65.55</td> <td rowspan="1" colspan="1">5,250</td> <td rowspan="1" colspan="1">54.68</td> <td rowspan="1" colspan="1">3,187</td> <td rowspan="1" colspan="1">33.19</td> </tr> <tr> <td rowspan="1" colspan="1">OR6</td> <td rowspan="1" colspan="1">1,635</td> <td rowspan="1" colspan="1">17.58</td> <td rowspan="1" colspan="1">1,502</td> <td rowspan="1" colspan="1">15.64</td> <td rowspan="1" colspan="1">853</td> <td rowspan="1" colspan="1">8.88</td> </tr> <tr> <td rowspan="1" colspan="1">OR 10</td> <td rowspan="1" colspan="1">1,393</td> <td rowspan="1" colspan="1">14.98</td> <td rowspan="1" colspan="1">1,449</td> <td rowspan="1" colspan="1">15.09</td> <td rowspan="1" colspan="1">599</td> <td rowspan="1" colspan="1">6.23</td> </tr> <tr> <td rowspan="1" colspan="1">OR 11</td> <td rowspan="1" colspan="1">352</td> <td rowspan="1" colspan="1">3.78</td> <td rowspan="1" colspan="1">345</td> <td rowspan="1" colspan="1">3.59</td> <td rowspan="1" colspan="1">192</td> <td rowspan="1" colspan="1">2</td> </tr> <tr> <td rowspan="1" colspan="1">OR 12</td> <td rowspan="1" colspan="1">148</td> <td rowspan="1" colspan="1">1.59</td> <td rowspan="1" colspan="1">124</td> <td rowspan="1" colspan="1">1.29</td> <td rowspan="1" colspan="1">80</td> <td rowspan="1" colspan="1">0.83</td> </tr> <tr> <td rowspan="1" colspan="1">OR 14</td> <td rowspan="1" colspan="1">1,307</td> <td rowspan="1" colspan="1">14.05</td> <td rowspan="1" colspan="1">37</td> <td rowspan="1" colspan="1">8.71</td> <td rowspan="1" colspan="1">737</td> <td rowspan="1" colspan="1">7.67</td> </tr> <tr> <td rowspan="1" colspan="1">OR 51</td> <td rowspan="1" colspan="1">790</td> <td rowspan="1" colspan="1">8.49</td> <td rowspan="1" colspan="1">834</td> <td rowspan="1" colspan="1">8.68</td> <td rowspan="1" colspan="1">414</td> <td rowspan="1" colspan="1">4.31</td> </tr> <tr> <td rowspan="1" colspan="1">OR 52</td> <td rowspan="1" colspan="1">1,003</td> <td rowspan="1" colspan="1">10.78</td> <td rowspan="1" colspan="1">1,108</td> <td rowspan="1" colspan="1">11.54</td> <td rowspan="1" colspan="1">499</td> <td rowspan="1" colspan="1">5.19</td> </tr> <tr> <td rowspan="1" colspan="1">OR 55</td> <td rowspan="1" colspan="1">10</td> <td rowspan="1" colspan="1">0.11</td> <td rowspan="1" colspan="1">10</td> <td rowspan="1" colspan="1">0.10</td> <td rowspan="1" colspan="1">8</td> <td rowspan="1" colspan="1">0.08</td> </tr> <tr> <td rowspan="1" colspan="1">OR 56</td> <td rowspan="1" colspan="1">244</td> <td rowspan="1" colspan="1">2.62</td> <td rowspan="1" colspan="1">255</td> <td rowspan="1" colspan="1">2.65</td> <td rowspan="1" colspan="1">118</td> <td rowspan="1" colspan="1">1.22</td> </tr> <tr> <td rowspan="1" colspan="1">Total</td> <td rowspan="1" colspan="1">24,408</td> <td rowspan="1" colspan="1">262.45</td> <td rowspan="1" colspan="1">20,478</td> <td rowspan="1" colspan="1">213.31</td> <td rowspan="1" colspan="1">13,577</td> <td rowspan="1" colspan="1">141.42</td> </tr> </tbody> </table> <div class="table-wrap-foot_UNKNOWN"> <div class="fn-type-" title=""> <p> <div class="sc_UNKNOWN">
Note</div>.—The nomenclature as described in <a href="#msy028-B46">Hayden et al. (2010)</a> is used to classify and describe OR families and clusters.</p> </div> </div>  <p /> <div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/"><span class="label" tagx="label" title="label"> <div class="sc_UNKNOWN">
Fig</div>. 2.</span></a></div>   <p>Method for identifying gene "birth" and "death" events. (<i>a</i>) Evolutionary relationships between ORs are established based on a maximum likelihood tree (JTT + Γ + F). Receptors born through gene duplication appear as paralogs. (<i>b</i>) Chained subtrees composed entirely of paralogs for one species are counted as multiple duplications (four duplication events in this example). (<i>c</i>) ORs undergoing duplication may retain function or become pseudogenes (death).</p>   </div> <div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/"><span class="label" tagx="label" title="label"> <div class="sc_UNKNOWN">
Fig</div>. 3.</span></a></div>   <p>Average number of gene duplication events for the "Genomic" and "Genomic + NGS" data sets. The number of species-specific gene duplication (SSD) events were counted for each OR family based on a gene tree established using maximum likelihood.</p>   </div> <div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/"><span class="label" tagx="label" title="label"> <div class="sc_UNKNOWN">
Fig</div>. 4.</span></a></div>   <p>The number of functional ORs after species-specific duplications (SSDs) in "Genomic" data. The 58 mammalian species that are in the "Genomic" data are ranked based on the number of OR paralogs born through duplication that are putatively functional. The top line (black, blue, and green) shows the highest ranked species. As not all mammals can be represented on a single line, the different numbers of functional ORs after duplication are expanded upon for visualization via the second and third line.</p>   </div> <p>With the inclusion of the NGS data set (<a href="#sup1">supplementary table S5</a>, <a href="#sup1">Supplementary Material</a> online), the total number of SSDs was 24,408 leading to a total of 34,055 ORs from duplication for the "Genomic + NGS" data set. Of these, 20,478 are functional while 13,577 have since lost their function (<a href="#msy028-T1">table 1</a> and <a href="#sup1">supplementary tables S5 and S6</a>, <a href="#sup1">Supplementary Material</a> online). The OR families with the highest average number of SSDs were 5/8/9, 1/3/7, 2/13, and 4 (<a href="#msy028-T1">table 1</a> and <a href="#msy028-F3">fig. 3</a>). Family 55 still had the smallest number of SSDs when the NGS data was added. The overall average duplications for the "Genomic + NGS" data set was systematically lower than the "Genomic" data, due to the addition 36 species representing a subset of OR genes affecting internal arrangement of the OR gene trees, and was therefore used to investigate robustness of duplication rates observed relative to the "Genomic" data set. Only one species, <i>Aonyx cinerea</i> (oriental small-clawed otter) did not show any identifiable SSDs. The varying number of node types (based on fate of the daughter ORs) for the NGS data were expressed as a percentage of the total duplication nodes, as they did not represent the full OR subgenome (<a href="#sup1">supplementary fig. S3</a>, <a href="#sup1">Supplementary Material</a> online).</p> </div> <div class="ecologicalnicheandspecies-specificduplications" title="sec"> <div class="title" tagx="title" title="title">
Ecological Niche and Species-Specific Duplications</div> <p>The number of subfamilies within each OR gene family showing retention of functionality in genes born through SSD ranged from 1 (OR Family 55) to 78 (OR family 5/8/9) subfamilies, suggesting OR family diversification (<a href="#msy028-T2">table 2</a>). Nucleotide and inferred translated amino acid sequence identity ranged from an average of 89% to 94.51% and 86% to 91.98% across all OR genes after an SSD, respectively (<a href="#msy028-T2">table 2</a>), highlighting differences between sister ORs, indicating sequence diversification. The average number of OR gene SSDs in each niche type in the "Genomic" data is displayed in <a href="#msy028-T3">table 3</a> and <a href="#sup1">supplementary table S7</a>, <a href="#sup1">Supplementary Material</a> online. <div class="table-wrap_UNKNOWN" id="msy028-T2" orientation="portrait" position="float"><span class="label" tagx="label" title="label">Table 2.</span> </div></p><p>The Number of Different Subfamilies Showing Species-Specific Duplication Events for Each or Gene Family Is Displayed, Including the Maximum Number of Subfamilies in a Single Taxon and the Mean Percentage Identity between Putative Functional Postduplication ORs within These Subfamilies.</p>  <table frame="hsides" rules="groups"> <colgroup span="1"> <col valign="top" align="left" span="1" /></colgroup> <colgroup><col valign="top" align="char" char="." span="1" /></colgroup> <colgroup><col valign="top" align="char" char="." span="1" /></colgroup> <colgroup><col valign="top" align="char" char="." span="1" /></colgroup> <colgroup><col valign="top" align="char" char="." span="1" /></colgroup> <colgroup><col valign="top" align="center" span="1" /></colgroup>  <thead> <tr> <th rowspan="1" colspan="1">OR Family</th> <th rowspan="1" colspan="1">Total Subfamilies Represented</th> <th rowspan="1" colspan="1">Max Subfamilies in a Single Taxon Showing Gene Duplication</th> <th rowspan="1" colspan="1">IQR of Subfamilies Showing Duplications Across Taxa</th> <th rowspan="1" colspan="1">Mean Percentage Identity (DNA) (%)</th> <th rowspan="1" colspan="1">Mean Percentage Identity (Protein) (%)</th> </tr> </thead> <tbody> <tr> <td rowspan="1" colspan="1">1/3/7</td> <td rowspan="1" colspan="1">44</td> <td rowspan="1" colspan="1">24</td> <td rowspan="1" colspan="1">8.75</td> <td rowspan="1" colspan="1">92.5</td> <td rowspan="1" colspan="1">88.69</td> </tr> <tr> <td rowspan="1" colspan="1">2/13</td> <td rowspan="1" colspan="1">73</td> <td rowspan="1" colspan="1">50</td> <td rowspan="1" colspan="1">17.75</td> <td rowspan="1" colspan="1">93.59</td> <td rowspan="1" colspan="1">90.81</td> </tr> <tr> <td rowspan="1" colspan="1">4</td> <td rowspan="1" colspan="1">51</td> <td rowspan="1" colspan="1">40</td> <td rowspan="1" colspan="1">15.75</td> <td rowspan="1" colspan="1">93.53</td> <td rowspan="1" colspan="1">90.64</td> </tr> <tr> <td rowspan="1" colspan="1">5/8/9</td> <td rowspan="1" colspan="1">78</td> <td rowspan="1" colspan="1">52</td> <td rowspan="1" colspan="1">21</td> <td rowspan="1" colspan="1">93.05</td> <td rowspan="1" colspan="1">89.69</td> </tr> <tr> <td rowspan="1" colspan="1">6</td> <td rowspan="1" colspan="1">28</td> <td rowspan="1" colspan="1">17</td> <td rowspan="1" colspan="1">6.75</td> <td rowspan="1" colspan="1">94.01</td> <td rowspan="1" colspan="1">91.43</td> </tr> <tr> <td rowspan="1" colspan="1">10</td> <td rowspan="1" colspan="1">40</td> <td rowspan="1" colspan="1">18</td> <td rowspan="1" colspan="1">8.75</td> <td rowspan="1" colspan="1">93.71</td> <td rowspan="1" colspan="1">91.13</td> </tr> <tr> <td rowspan="1" colspan="1">11</td> <td rowspan="1" colspan="1">9</td> <td rowspan="1" colspan="1">8</td> <td rowspan="1" colspan="1">3</td> <td rowspan="1" colspan="1">93.62</td> <td rowspan="1" colspan="1">90.57</td> </tr> <tr> <td rowspan="1" colspan="1">12</td> <td rowspan="1" colspan="1">2</td> <td rowspan="1" colspan="1">2</td> <td rowspan="1" colspan="1">1</td> <td rowspan="1" colspan="1">93.49</td> <td rowspan="1" colspan="1">90.25</td> </tr> <tr> <td rowspan="1" colspan="1">14</td> <td rowspan="1" colspan="1">7</td> <td rowspan="1" colspan="1">6</td> <td rowspan="1" colspan="1">3</td> <td rowspan="1" colspan="1">92.47</td> <td rowspan="1" colspan="1">88.18</td> </tr> <tr> <td rowspan="1" colspan="1">51</td> <td rowspan="1" colspan="1">22</td> <td rowspan="1" colspan="1">19</td> <td rowspan="1" colspan="1">5</td> <td rowspan="1" colspan="1">93.69</td> <td rowspan="1" colspan="1">91.32</td> </tr> <tr> <td rowspan="1" colspan="1">52</td> <td rowspan="1" colspan="1">28</td> <td rowspan="1" colspan="1">20</td> <td rowspan="1" colspan="1">10</td> <td rowspan="1" colspan="1">93.96</td> <td rowspan="1" colspan="1">91.64</td> </tr> <tr> <td rowspan="1" colspan="1">55</td> <td rowspan="1" colspan="1">1</td> <td rowspan="1" colspan="1">1</td> <td rowspan="1" colspan="1">0</td> <td rowspan="1" colspan="1">89.45</td> <td rowspan="1" colspan="1">86</td> </tr> <tr> <td rowspan="1" colspan="1">56</td> <td rowspan="1" colspan="1">7</td> <td rowspan="1" colspan="1">6</td> <td rowspan="1" colspan="1">2</td> <td rowspan="1" colspan="1">94.51</td> <td rowspan="1" colspan="1">91.98</td> </tr> <tr> <td rowspan="1" colspan="1">Total</td> <td rowspan="1" colspan="1">389</td> <td rowspan="1" colspan="1">258</td> <td rowspan="1" colspan="1">92.75</td> <td rowspan="1" colspan="1">93.32</td> <td rowspan="1" colspan="1">90.23</td> </tr> </tbody> </table>  <div class="table-wrap_UNKNOWN" id="msy028-T3" orientation="portrait" position="float"><span class="label" tagx="label" title="label">Table 3.</span> <p>The Total and Average Number of a Species-Specific Duplication Events per Niche Is Displayed for the "Genomic" Data.</p>  <table frame="hsides" rules="groups"> <colgroup span="1"> <col valign="top" align="left" span="1" /></colgroup> <colgroup><col valign="top" align="char" char="(" span="1" /></colgroup> <colgroup><col valign="top" align="char" char="." span="1" /></colgroup> <colgroup><col valign="top" align="char" char="." span="1" /></colgroup> <colgroup><col valign="top" align="char" char="." span="1" /></colgroup> <colgroup><col valign="top" align="char" char="." span="1" /></colgroup>  <thead align="center"> <tr> <th colspan="2" align="left" rowspan="1">Niche</th> <th rowspan="1" colspan="1">Average % of Total OR Repertoire that is Functional</th> <th rowspan="1" colspan="1">Total Duplications</th> <th rowspan="1" colspan="1">Average Duplications</th> <th rowspan="1" colspan="1">Average % OR Repertoire from Gene Duplications</th> </tr> </thead> <tbody> <tr> <td rowspan="5" colspan="1">Diet</td> <td rowspan="1" colspan="1">Carnivore (<i>n</i> = 8)</td> <td rowspan="1" colspan="1">68.54</td> <td rowspan="1" colspan="1">1410</td> <td rowspan="1" colspan="1">176.25</td> <td rowspan="1" colspan="1">27.47</td> </tr> <tr> <td rowspan="1" colspan="1">Frugivore (<i>n</i> = 5)</td> <td rowspan="1" colspan="1">59.91</td> <td rowspan="1" colspan="1">271</td> <td rowspan="1" colspan="1">54.2</td> <td rowspan="1" colspan="1">12.84</td> </tr> <tr> <td rowspan="1" colspan="1">Insectivore (<i>n</i> = 12)</td> <td rowspan="1" colspan="1">70.26</td> <td rowspan="1" colspan="1">5861</td> <td rowspan="1" colspan="1">488.42</td> <td rowspan="1" colspan="1">38.85</td> </tr> <tr> <td rowspan="1" colspan="1">Herbivore (<i>n</i> = 19)</td> <td rowspan="1" colspan="1">60.96</td> <td rowspan="1" colspan="1">11,200</td> <td rowspan="1" colspan="1">589.47</td> <td rowspan="1" colspan="1">42.25</td> </tr> <tr> <td rowspan="1" colspan="1">Omnivore (<i>n</i> = 14)</td> <td rowspan="1" colspan="1">70.89</td> <td rowspan="1" colspan="1">4084</td> <td rowspan="1" colspan="1">291.71</td> <td rowspan="1" colspan="1">31.88</td> </tr> <tr> <td rowspan="2" colspan="1">Sociality</td> <td rowspan="1" colspan="1">Social (<i>n</i> = 36)</td> <td rowspan="1" colspan="1">65.57</td> <td rowspan="1" colspan="1">9350</td> <td rowspan="1" colspan="1">259.72</td> <td rowspan="1" colspan="1">25.91</td> </tr> <tr> <td rowspan="1" colspan="1">Solitary (<i>n</i> = 22)</td> <td rowspan="1" colspan="1">67.33</td> <td rowspan="1" colspan="1">13,476</td> <td rowspan="1" colspan="1">612.55</td> <td rowspan="1" colspan="1">48.47</td> </tr> <tr> <td rowspan="3" colspan="1">Rhythmic activity phase</td> <td rowspan="1" colspan="1">Crepuscular (<i>n</i> = 3)</td> <td rowspan="1" colspan="1">70.46</td> <td rowspan="1" colspan="1">842</td> <td rowspan="1" colspan="1">280.67</td> <td rowspan="1" colspan="1">39.11</td> </tr> <tr> <td rowspan="1" colspan="1">Diurnal (<i>n</i> = 29)</td> <td rowspan="1" colspan="1">60.07</td> <td rowspan="1" colspan="1">12,201</td> <td rowspan="1" colspan="1">420.72</td> <td rowspan="1" colspan="1">32.91</td> </tr> <tr> <td rowspan="1" colspan="1">Nocturnal (<i>n</i> = 26)</td> <td rowspan="1" colspan="1">72.63</td> <td rowspan="1" colspan="1">9783</td> <td rowspan="1" colspan="1">376.27</td> <td rowspan="1" colspan="1">35.67</td> </tr> <tr> <td rowspan="3" colspan="1">Habitat</td> <td rowspan="1" colspan="1">Volant (<i>n</i> = 9)</td> <td rowspan="1" colspan="1">67.52</td> <td rowspan="1" colspan="1">359</td> <td rowspan="1" colspan="1">39.88</td> <td rowspan="1" colspan="1">11.37</td> </tr> <tr> <td rowspan="1" colspan="1">Terrestrial (<i>n</i> = 46)</td> <td rowspan="1" colspan="1">67.11</td> <td rowspan="1" colspan="1">21,937</td> <td rowspan="1" colspan="1">476.89</td> <td rowspan="1" colspan="1">38.95</td> </tr> <tr> <td rowspan="1" colspan="1">Aquatic (<i>n</i> = 2)</td> <td rowspan="1" colspan="1">42.2</td> <td rowspan="1" colspan="1">160</td> <td rowspan="1" colspan="1">80</td> <td rowspan="1" colspan="1">13.13</td> </tr> <tr> <td rowspan="2" colspan="1">Vomeronasal organ</td> <td rowspan="1" colspan="1">VNO Present (<i>n</i> = 41)</td> <td rowspan="1" colspan="1">68.33</td> <td rowspan="1" colspan="1">22,071</td> <td rowspan="1" colspan="1">525.5</td> <td rowspan="1" colspan="1">43.31</td> </tr> <tr> <td rowspan="1" colspan="1">VNO Absent (<i>n</i> = 15)</td> <td rowspan="1" colspan="1">60.16</td> <td rowspan="1" colspan="1">755</td> <td rowspan="1" colspan="1">47.19</td> <td rowspan="1" colspan="1">11.24</td> </tr> </tbody> </table> <div class="table-wrap-foot_UNKNOWN"> <div class="fn-type-" title=""> <p> <div class="sc_UNKNOWN">
Note</div>.—The average percentage of the entire OR repertoire that is functional in mammals per niche is displayed, as is the percentage of each repertoire that has come from gene duplication events after diverging from a common ancestor.</p> </div> </div> </div> <p /> <p>When OR families were ranked from most to least functional ORs born through SSD, family 5/8/9 in herbivores had the highest quantity in the majority of species (<a href="#msy028-F5">fig. 5</a>). Family 1/3/7 showed some retention of OR genes after duplication in frugivores and insectivores (specifically in bats), however, no family specific functional retention was apparent in carnivores or omnivores (<a href="#sup1">supplementary figs. S4–S7</a>, <a href="#sup1">Supplementary Material</a> online). Fisher’s exact test did not detect any significant differences in the distributions of SSD events across OR gene families as estimated based on: 1) "Genomic" and (2) "Genomic + NGS" for the 58 species present in both. This suggests that inclusion of the additional 36 species does not significantly affect the estimation of SSDs across the entire repertoire, indicating robustness of these data. </p> <div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/"><span class="label" tagx="label" title="label"> <div class="sc_UNKNOWN">
Fig</div>. 5.</span></a></div>   <p>Herbivore OR families ranked by functional OR genes after a duplication event. OR gene families for each herbivorous species from the "Genomic" data were ranked by the number of OR genes born through species-specific gene duplication (SSD) that have retained function. Herbivores from the "Genomic + NGS" data (delimited by a black star) are also included.</p>   </div> </div> <div class="phylogeneticgeneralizedleastsquaresanalyses" title="sec"> <div class="title" tagx="title" title="title">
Phylogenetic Generalized Least Squares Analyses</div> <p>Using Phylogenetic Generalized Least Squares (PGLS) analyses, significant differences were found for OR gene families across multiple niches with respect to the number of SSDs detected and the functional status of their daughter OR genes (<a href="#msy028-T4">table 4</a>). Only OR families and ecological niches showing significant differences in both data sets ("Genomic", "Genomic + NGS") are reported here (see <a href="#sup1">supplementary tables S8–S10</a>, <a href="#sup1">Supplementary Material</a> online for OR gene families specific to each data set). For dietary niche, family 5/8/9 showed significant differences for herbivory compared with all other diets with respect to number of SSDs and functional/non-functional daughter ORs. Family 1/3/7 showed a significant difference in the number of SSDs in insectivores and in the number of functional daughter ORs for habitat. Family 5/8/9 had significantly more SSDs and functional daughters in taxa with a functional VNO. Rhythmic activity phase and sociality showed significant differences in the number of non-functional daughters for OR families 4 and 11, respectively (more pseudogenization in diurnal and crepuscular mammals versus nocturnal for OR family 4 and for solitary mammals in OR family 11). <div class="table-wrap_UNKNOWN" id="msy028-T4" orientation="portrait" position="float"><span class="label" tagx="label" title="label">Table 4.</span> </div></p><p>OR Families Were Compared Across Ecological Niches Using PGLS to Identify Families Showing Significantly Differences with Respect To Species-Specific Duplication Events, Paralogous ORs Retaining Function and Paralogous ORs Having Lost Function After Duplication.</p>  <table frame="hsides" rules="groups"> <colgroup span="1"> <col valign="top" align="left" span="1" /></colgroup> <colgroup><col valign="top" align="center" span="1" /></colgroup> <colgroup><col valign="top" align="center" span="1" /></colgroup> <colgroup><col valign="top" align="center" span="1" /></colgroup> <colgroup><col valign="top" align="center" span="1" /></colgroup> <colgroup><col valign="top" align="center" span="1" /></colgroup>  <thead> <tr> <th rowspan="1" colspan="1">Family</th> <th rowspan="1" colspan="1">Niche</th> <th rowspan="1" colspan="1">Data Set</th> <th rowspan="1" colspan="1">PGLS <i>P</i> Value</th> <th rowspan="1" colspan="1">Estimated λ</th> <th rowspan="1" colspan="1">Significant Niches</th> </tr> </thead> <tbody> <tr> <td rowspan="1" colspan="1">Duplications</td> <td rowspan="1" colspan="1" /> <td rowspan="1" colspan="1" /> <td rowspan="1" colspan="1" /> <td rowspan="1" colspan="1" /> <td rowspan="1" colspan="1" /> </tr> <tr> <td rowspan="2" colspan="1">OR 1/3/7</td> <td rowspan="2" colspan="1">Diet</td> <td rowspan="1" colspan="1">Genomic</td> <td rowspan="1" colspan="1"><i>P</i> = 0.011</td> <td rowspan="1" colspan="1">0.28</td> <td rowspan="2" colspan="1">Insectivory</td> </tr> <tr> <td rowspan="1" colspan="1">Genomic+NGS</td> <td rowspan="1" colspan="1"><i>P</i> = 0.005</td> <td rowspan="1" colspan="1">0</td> </tr> <tr> <td rowspan="4" colspan="1">OR 5/8/9</td> <td rowspan="2" colspan="1">Diet</td> <td rowspan="1" colspan="1">Genomic</td> <td rowspan="1" colspan="1"><i>P</i> = 0.004</td> <td rowspan="1" colspan="1">0.43</td> <td rowspan="2" colspan="1">Herbivory</td> </tr> <tr> <td rowspan="1" colspan="1">Genomic+NGS</td> <td rowspan="1" colspan="1"><i>P</i> = 0.005</td> <td rowspan="1" colspan="1">0.13</td> </tr> <tr> <td rowspan="2" colspan="1">VNO</td> <td rowspan="1" colspan="1">Genomic</td> <td rowspan="1" colspan="1"><i>P</i> = 0.000</td> <td rowspan="1" colspan="1">0.28</td> <td rowspan="2" colspan="1">VNO</td> </tr> <tr> <td rowspan="1" colspan="1">Genomic+NGS</td> <td rowspan="1" colspan="1"><i>P</i> = 0.002</td> <td rowspan="1" colspan="1">0.30</td> </tr> <tr> <td rowspan="4" colspan="1">OR 52</td> <td rowspan="4" colspan="1">Habitat</td> <td rowspan="1" colspan="1">Genomic</td> <td rowspan="1" colspan="1"><i>P</i> = 0.009</td> <td rowspan="4" colspan="1">0</td> <td rowspan="2" colspan="1">Aquatic</td> </tr> <tr> <td rowspan="1" colspan="1">Genomic+NGS</td> <td rowspan="1" colspan="1"><i>P</i> = 0.034</td> </tr> <tr> <td rowspan="1" colspan="1">Genomic</td> <td rowspan="1" colspan="1"><i>P</i> = 0.0003</td> <td rowspan="2" colspan="1">Terrestrial</td> </tr> <tr> <td rowspan="1" colspan="1">Genomic+NGS</td> <td rowspan="1" colspan="1"><i>P</i> = 0.02</td> </tr> <tr> <td colspan="6" rowspan="1">Postduplication functional</td> </tr> <tr> <td rowspan="4" colspan="1">OR 1/3/7</td> <td rowspan="4" colspan="1">Habitat</td> <td rowspan="1" colspan="1">Genomic</td> <td rowspan="1" colspan="1"><i>P</i> = 0.0009</td> <td rowspan="4" colspan="1">0</td> <td rowspan="2" colspan="1">Aquatic</td> </tr> <tr> <td rowspan="1" colspan="1">Genomic+NGS</td> <td rowspan="1" colspan="1"><i>P</i> = 0.009</td> </tr> <tr> <td rowspan="1" colspan="1">Genomic</td> <td rowspan="1" colspan="1"><i>P</i> = 0</td> <td rowspan="2" colspan="1">Terrestrial</td> </tr> <tr> <td rowspan="1" colspan="1">Genomic+NGS</td> <td rowspan="1" colspan="1"><i>P</i> = 0</td> </tr> <tr> <td rowspan="4" colspan="1">OR 5/8/9</td> <td rowspan="2" colspan="1">Diet</td> <td rowspan="1" colspan="1">Genomic</td> <td rowspan="1" colspan="1"><i>P</i> = 0.029</td> <td rowspan="1" colspan="1">0.23</td> <td rowspan="2" colspan="1">Herbivory</td> </tr> <tr> <td rowspan="1" colspan="1">Genomic+NGS</td> <td rowspan="1" colspan="1"><i>P</i> = 0.014</td> <td rowspan="1" colspan="1">0.29</td> </tr> <tr> <td rowspan="2" colspan="1">VNO</td> <td rowspan="1" colspan="1">Genomic</td> <td rowspan="1" colspan="1"><i>P</i> = 0.0004</td> <td rowspan="1" colspan="1">0.07</td> <td rowspan="2" colspan="1">VNO</td> </tr> <tr> <td rowspan="1" colspan="1">Genomic+NGS</td> <td rowspan="1" colspan="1"><i>P</i> = 0.001</td> <td rowspan="1" colspan="1">0.33</td> </tr> <tr> <td rowspan="2" colspan="1">OR 52</td> <td rowspan="2" colspan="1">Habitat</td> <td rowspan="1" colspan="1">Genomic</td> <td rowspan="1" colspan="1"><i>P</i> = 0.023</td> <td rowspan="2" colspan="1">0</td> <td rowspan="2" colspan="1">Aquatic</td> </tr> <tr> <td rowspan="1" colspan="1">Genomic+NGS</td> <td rowspan="1" colspan="1"><i>P</i> = 0.044</td> </tr> <tr> <td colspan="6" rowspan="1">Postduplication non-functional</td> </tr> <tr> <td rowspan="4" colspan="1">OR 4</td> <td rowspan="4" colspan="1">Rhythmic Activity Phase</td> <td rowspan="1" colspan="1">Genomic</td> <td rowspan="1" colspan="1"><i>P</i> = 0.02</td> <td rowspan="4" colspan="1">0</td> <td rowspan="2" colspan="1">Diurnal</td> </tr> <tr> <td rowspan="1" colspan="1">Genomic+NGS</td> <td rowspan="1" colspan="1"><i>P</i> = 0.01</td> </tr> <tr> <td rowspan="1" colspan="1">Genomic</td> <td rowspan="1" colspan="1"><i>P</i> = 0.001</td> <td rowspan="2" colspan="1">Nocturnal</td> </tr> <tr> <td rowspan="1" colspan="1">Genomic+NGS</td> <td rowspan="1" colspan="1"><i>P</i> = 0.002</td> </tr> <tr> <td rowspan="4" colspan="1">OR 5/8/9</td> <td rowspan="2" colspan="1">Diet</td> <td rowspan="1" colspan="1">Genomic</td> <td rowspan="1" colspan="1"><i>P</i> = 0.001</td> <td rowspan="1" colspan="1">0.55</td> <td rowspan="2" colspan="1">Herbivory</td> </tr> <tr> <td rowspan="1" colspan="1">Genomic+NGS</td> <td rowspan="1" colspan="1"><i>P</i> = 0.002</td> <td rowspan="1" colspan="1">0.03</td> </tr> <tr> <td rowspan="2" colspan="1">VNO</td> <td rowspan="1" colspan="1">Genomic</td> <td rowspan="1" colspan="1"><i>P</i> = 0.001</td> <td rowspan="1" colspan="1">0.59</td> <td rowspan="2" colspan="1">VNO</td> </tr> <tr> <td rowspan="1" colspan="1">Genomic+NGS</td> <td rowspan="1" colspan="1"><i>P</i> = 0.02</td> <td rowspan="1" colspan="1">0.29</td> </tr> <tr> <td rowspan="4" colspan="1">OR 11</td> <td rowspan="2" colspan="1">Diet</td> <td rowspan="1" colspan="1">Genomic</td> <td rowspan="1" colspan="1"><i>P</i> = 0.02</td> <td rowspan="2" colspan="1">0</td> <td rowspan="2" colspan="1">Frugivory</td> </tr> <tr> <td rowspan="1" colspan="1">Genomic+NGS</td> <td rowspan="1" colspan="1"><i>P</i> = 0.008</td> </tr> <tr> <td rowspan="2" colspan="1">Sociality</td> <td rowspan="1" colspan="1">Genomic</td> <td rowspan="1" colspan="1"><i>P</i> = 0.02</td> <td rowspan="1" colspan="1">0.15</td> <td rowspan="2" colspan="1">Sociality</td> </tr> <tr> <td rowspan="1" colspan="1">Genomic+NGS</td> <td rowspan="1" colspan="1"><i>P</i> = 0.01</td> <td rowspan="1" colspan="1">0</td> </tr> <tr> <td rowspan="2" colspan="1">OR 56</td> <td rowspan="2" colspan="1">Diet</td> <td rowspan="1" colspan="1">Genomic</td> <td rowspan="1" colspan="1"><i>P</i> = 0.001</td> <td rowspan="2" colspan="1">0</td> <td rowspan="2" colspan="1">Herbivory</td> </tr> <tr> <td rowspan="1" colspan="1">Genomic+NGS</td> <td rowspan="1" colspan="1"><i>P</i> = 0.01</td> </tr> </tbody> </table> <div class="table-wrap-foot_UNKNOWN"> <div class="fn-type-" title=""> <p> <div class="sc_UNKNOWN">
Note</div>.—OR gene families significant in both "Genomic" and "Genomic + NGS" data sets are displayed (<i>P</i> &amp;lt; 0.05, λ&amp;lt;0.9).</p> </div> </div>  <p /> </div> <div class="poissonmodelingofecologicalnichedata" title="sec"> <div class="title" tagx="title" title="title">
Poisson Modeling of Ecological Niche Data</div> <p>Results for the AICc model selection for each data set (SSDs; functional daughters; non-functional daughters) are given in <a href="#sup1">supplementary table S11</a>, <a href="#sup1">Supplementary Material</a> online. With the exception of non-functional OR daughters, the best supported model in all data sets modeled SSD rates as a function of the presence or absence of a functional VNO, with a posterior probability of 1.00 (<a href="#sup1">supplementary table S11</a>, <a href="#sup1">Supplementary Material</a> online), with individual gene families having separate rates of duplication. In all cases, the estimated species-specific rates were substantially higher for taxa with a functional VNO (<a href="#sup1">supplementary table S11</a>, <a href="#sup1">Supplementary Material</a> online). For non-functional daughters, dietary niche separated by individual OR gene families, was the most supported model. Dietary niche, with all diets as independent categories, ranked second for the rate of SSDs in both data sets.</p> </div> </div> <div class="discussion" title="discussion"> <div class="title" tagx="title" title="title">
Discussion</div> <p>Gene duplication through processes such as tandem gene duplication, segmental duplication or whole-genome duplication is an important means by which novel diversifying genotypes and phenotypes can arise (<a href="#msy028-B18">Cotton 2008</a>; <a href="#msy028-B13">Chang and Duda 2012</a>). Expansion or reduction of gene families in this way can be random or a consequence of selective forces. However, demonstrating evidence of selection on expanding gene families can be a difficult task (<a href="#msy028-B44">Harris and Hofmann 2015</a>).</p> <p>The elephant has the largest OR repertoire among mammals (<a href="#msy028-B46">Hayden et al. 2010</a>; <a href="#msy028-B82">Niimura et al. 2014</a>), with African elephants being able to distinguish between family members using olfaction (<a href="#msy028-B6">Bates et al. 2008</a>) and Asian elephants being able to distinguish between enantiomer odorant pairs (<a href="#msy028-B101">Rizvanovic et al. 2013</a>). This exceptional olfactory capability is reflected in the number of detectable species-specific duplications (SSD) observed in the African elephant genome and its subsequent ORs that have putatively retained function. In contrast, the dolphin has the smallest number of SSDs, reflecting the diminished role of olfaction after the transition from land to water in the evolutionary history of cetaceans (<a href="#msy028-B118">Springer and Gatesy 2017</a>). To explore how gene expansion has occurred in the OR gene family, with respect to ecological niche adaptation, we investigated SSD events and the fate of the genes such events give rise to in 94 mammals (58 sequenced genomes and 36 from NGS amplicon sequence data). These data included species representatives from all superordinal groups.</p> <div class="orgeneevolutionindietaryniche" title="sec"> <div class="title" tagx="title" title="title">
OR Gene Evolution in Dietary Niche</div> <p>Our findings show that in herbivores, OR gene family 5/8/9 had the highest number of SSDs and retained more functional OR genes born through duplication than in nonherbivores. These SSD events were found in a diverse number of subfamilies within the OR 5/8/9 gene family, suggesting diversification through duplication. The sampled mammalian herbivores span clades that are distantly related: Laurasiatheria (Cetartiodactyla, Perissodactyla, and Carnivora), Euarchontoglires (Rodentia), Afrotheria (Paenungulata), Xenarthra (Pilosa), and Metatheria (Diprodontia) (<a href="#msy028-B74">Meredith et al. 2011</a>), with herbivory evolving independently from omnivorous or carnivorous diets (<a href="#msy028-B96">Price et al. 2012</a>), yet showing convergent functional expansion of OR 5/8/9. For the giant panda (<i>Ailuropoda melanoleuca</i>), an herbivorous mammal that still maintains the genetic requirements for a carnivorous diet (<a href="#msy028-B64">Li et al. 2009</a>), family 5/8/9 had slightly more functional ORs after SSDs than any other OR family highlighting a potential role in the evolution of herbivory. In humans and mice, family 5/8/9 is linked to binding odorants whose "descriptive aroma" has been associated with oils and plants in addition to fruity or floral aromas (coumarin, β-ionone, menthol, prenyl acetate, heptanone, acetophenone, eugenol, vanillin; <a href="#msy028-B24">Dunkel et al. 2014</a>) suggesting its importance in a herbivorous diet. However, family 5/8/9 can bind odorants that are not related to plant matter (nonanoic acid; <a href="#msy028-B24">Dunkel et al. 2014</a>), highlighting the need for further research into the role of receptors and diet.</p> <p>For frugivores, OR family 1/3/7 shows a higher number of SSDs in bats compared to others species and families. Frugivores sampled in this study come from the orders Chiroptera and Primates. <a href="#msy028-B47">Hayden et al. (2014)</a> established an association between frugivory and functional ORs in family 1/3/7 in bats, and this is corroborated by the SSD data observed here. We also note high numbers of SSDs for family 1/3/7 in the orangutan and white-cheeked gibbon, whose diets are highly frugivorous (<a href="#msy028-B76">Mitani 1989</a>; <a href="#msy028-B61">Kissling et al. 2014</a>; <a href="#msy028-B78">Muchlinski and Deane 2014</a>). The ancestor to all modern bats is inferred to have been insectivorous (<a href="#msy028-B107">Schondube et al. 2001</a>) but dietary shifts to frugivory occurred, independently, in <i>Phyllostomidae</i> and in <i>Pteropodidae</i> (<a href="#msy028-B47">Hayden et al. 2014</a>). We find that OR 1/3/7 also shows significant duplication rates associated with insectivory across a wide range of mammalian orders, with mammalian dietary shifts potentially converging on the same gene family. In humans, family 1/3/7 ORs have been shown to bind to different molecules that are associated with fruity or floral aromas, similar to OR 5/8/9 (citronellal, benzyl acetate, heptanal, γ-decalactone, lilial, ethyl-2-methyl proponoate; <a href="#msy028-B24">Dunkel et al. 2014</a>). Future deorphaning of OR and odorant molecules may allow a more comprehensive study of mammalian olfactory space, and the role of diet in olfactory evolution. Duplication and retention of function in both frugivorous and insectivorous species across major placental supraordinal clades suggests family 1/3/7, like family 5/8/9, has undergone multiple expansions in phylogenetically diverse mammals with respect to dietary niche.</p> </div> <div class="theevolutionoforsandthevno" title="sec"> <div class="title" tagx="title" title="title">
The Evolution of ORs and the VNO</div> <p>The Vomeronasal or Jacobson organ (VNO) is used to detect pheromones: water-soluble chemical signals that trigger a social response in a number of mammals (<a href="#msy028-B133">Zhang and Webb 2003</a>). When looking at whole genome and NGS amplicon data, significant differences between the number of putatively functional ORs born through SSD suggests a link exists between the expansion of the OR repertoire and the presence/absence of a functional VNO. This pattern was observed across all statistical analyses and was the best-supported niche to explain observed duplication events. The VNO has previously shown the ability to express OR genes (<a href="#msy028-B57">Keller and Vosshall 2008</a>), and it has been demonstrated that pheromones can stimulate olfactory neurons, which suggests a complex interaction between the two sensory modes. A transcriptome analysis of the olfactory system in mice (<a href="#msy028-B52">Ibarra-Soria et al. 2014</a>) highlighted at least 17 OR genes expressed on the VNO. The OR gene with the highest gene expression in the mouse VNO, <i>olfr124</i>, was one of a pair of OR genes born through a SSD event that we have identified in mice. Additionally, <i>olfr533</i>, <i>olfr741</i>, and <i>olfr536</i> were also found to be expressed in the mouse VNO, and are postduplication functional OR genes in mice.</p> <p>Our results indicate that the OR repertoire has not evolved to compensate for the complete loss of a VNO, but rather is augmented by the presence of a functioning one in extant taxa. Such a relationship may explain the highly developed OR and VNO systems of the Asian elephant (<a href="#msy028-B101">Rizvanovic et al. 2013</a>) and the fact that a number of mammals without a functional VNO (16 bat species, six primates, and two cetaceans across both data sets; <a href="#msy028-F6">fig. 6</a>) demonstrate a stronger reliance on alternative means of sensory perception, namely echolocation, and vision. Further studies into the differential expression of VNO receptor genes in olfactory epithelia, the expression of OR genes in VNO tissue and their potential correlations with OR subgenome evolution may help elucidate the patterns observed here. </p> <div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/"><span class="label" tagx="label" title="label"> <div class="sc_UNKNOWN">
Fig</div>. 6.</span></a></div>   <p>Ecological niches of 94 mammals. The ecological niche data for 94 mammals used in this study are displayed.</p>   </div> </div> <div class="orgeneevolutioninsocialversussolitarymammals" title="sec"> <div class="title" tagx="title" title="title">
OR Gene Evolution in Social versus Solitary Mammals</div> <p>Solitary species have on average twice the number of SSDs compared with social species, with no family in particular showing significant differences to others. This suggests that individual OR gene families may have expanded at similar high rates with respect to sociality, with broader repertoires being adaptive to a solitary lifestyle. It is possible that mammals living in large groups may rely less on olfaction and the detection of odorant molecules as a whole, due to the greater protection and cooperation in finding resources conferred by a social group membership (<a href="#msy028-B14">Chesser et al. 1993</a>; <a href="#msy028-B112">Silk 2007</a>). In this case, vocalization or display may be the dominant form of communication. Solitary mammals may need to be more vigilant with respect to prey, predators and mating and as such, an expanded OR repertoire could potentially allow a finer tuned sense of olfaction. An exception to this observation is the African elephant, which shows more OR gene duplication events than any other taxon, despite predominantly living in social groups. (<a href="#msy028-B129">Wittemyer et al. 2005</a>). Interestingly, the OR gene families that are associated with diet (1/3/7 and 5/8/9) do not show a significant difference between social and solitary taxa, but also suggests that the increases in SSD events in solitary mammals are contained largely in OR subfamilies associated with surveying the physical environment around the individual.</p> </div> <div class="orevolutionandrhythmicactivityphase" title="sec"> <div class="title" tagx="title" title="title">
OR Evolution and Rhythmic Activity Phase</div> <p>Given the importance of light in maintaining circadian rhythm (<a href="#msy028-B12">Challet 2007</a>), activity phase in mammals has previously been linked to adaptations in visual perception. Eye morphology has previously shown to be a discriminator between nocturnal and diurnal species (<a href="#msy028-B106">Schmitz and Motani 2010</a>), and a loss of function in opsin color vision genes has been reported in nocturnal mammals (<a href="#msy028-B135">Zhao et al. 2009</a>; see section below). In reduced light, olfaction likely plays an important role in hunting, identifying potential threats and communication with conspecifics for both nocturnal and crepuscular species. Bats have maintained the ancestral mammalian OR repertoire with little loss of function associated with gain of echolocation suggesting an important role for olfaction in bat survival, potentially driven by their nocturnality (<a href="#msy028-B46">Hayden et al. 2010</a>). Odorant detection has previously been used to elicit some circadian response in mice (<a href="#msy028-B41">Granados-Fuentes et al. 2006</a>) and nocturnal mammals contain more intact vomeronasal type 1 receptor (V1R) genes for pheromone perception compared to diurnal taxa (Wang et al. 2010). While nocturnal mammals showed a lower number of non-functional daughters after SSD in OR family 4 compared with crepuscular and diurnal species, the lack of an observed significant correlation between activity phase and OR gene family expansion through duplication in our data is surprising. It is possible that selective forces are acting on the preservation of OR gene function in nocturnal mammals after duplication, similar to that of the VNO (Wang et al. 2010). This could be due to different taxa occupying the same ecological niches but at different times. As rhythmic activity phase is affected by a number of internal and external cues, including geographical distribution (<a href="#msy028-B7">Bennie et al. 2014</a>), a more focused, in-depth analysis of rhythmic activity phase and its regulation with respect to chemosensory perception is needed to truly elucidate any patterns of olfaction and activity phase.</p> </div> <div class="theevolutionoforgenesandhabitat" title="sec"> <div class="title" tagx="title" title="title">
The Evolution of OR Genes and Habitat</div> <p>In a previous study carried out by <a href="#msy028-B46">Hayden et al. (2010)</a>, differences in the number of functional ORs between terrestrial and volant mammals suggested a correlation between the evolution of the OR repertoire and adaptation to various habitat types. However, when accounting for the percentage of the OR repertoire attributable to SSD, terrestrial mammal OR repertoires have undergone almost three times as much OR gene duplication as volant or aquatic mammals. This reflects previous conclusions regarding OR repertoire composition and ecological adaptation (<a href="#msy028-B46">Hayden et al. 2010</a>). Terrestrial mammals occupy a variety of different niches, which may be associated with such high rates of gene duplications. The expansion of OR gene families for terrestrial adaptation can also be observed in the high rate of duplication in Class I OR genes (which typically bind water odorants) compared to aquatic mammals, suggesting a potential novel usage or cooptimization for this class of gene families in adaptation to a nonaquatic environment, as previously observed in the dog (<a href="#msy028-B85">Olender et al. 2004</a>). The relatively small number of OR genes in aquatic mammals would imply relaxed selective pressures on olfactory mechanisms for an aquatic lifestyle. Nonetheless, we observed some duplication of OR genes in aquatic taxa, with a higher rate in the manatee compared to cetacean species, suggesting some putatively retained OR functionality in an aquatic environment.</p> </div> </div> <div class="conclusion" title="conclusion"> <div class="title" tagx="title" title="title">
Conclusion</div> <p>Our findings indicate that the evolution and diversification of the mammalian OR repertoire through SSD (large expansions within a specific OR family, or of the OR repertoire as a whole) and the retention of function in duplicated OR genes is associated with adaptations to a range of ecological niches. We have identified specific OR families that appear to be associated with adaptation to different dietary niches (herbivory: 5/8/9, insectivory and frugivory: 1/3/7). Furthermore, our findings suggest that the OR repertoire and the VNO are intimately linked, with a functional VNO associated with rapid expansion of the OR gene repertoire. In addition, solitary mammals have undergone nearly twice as many SSD events as social mammals. However, there were no significant differences observed between rhythmic activity phases with respect to OR gene evolution through SSD. The wide range of different terrestrial ecological niches to which mammals have successfully adapted is reflected in the fact that three times as many OR genes are born through gene duplication in terrestrial mammals compared to volant or aquatic mammals.</p> <p>In this study, we have shown the utility of using phylogenetically independent SSDs as a means of elucidating how environmental niche adaptation is associated with the evolution of sensory perception. As amplicon sequencing and whole genome sequencing become cheaper and faster, future studies will allow the possibility of further exploring these results. Higher quality genomes will allow further exploration of the associations between OR gene evolution and ecological niche adaptation, possibly uncovering more functional genes or resolving any instances of incorrectly identified pseudogenes. The Genome 10 K sequencing project (G10K; <a href="#msy028-B45">Haussler et al. 2009</a>), which aims to sequence the full genomes of &amp;gt;10,000 vertebrates, includes a large set of mammalian taxa. Such a large set of genomic data will play a huge role in the investigation and understanding of mammalian sensory evolution, shedding more light on the relationship between sensory evolution and niche adaptation. Additionally, the modeling of 3D structures of ORs, and deorphaning of receptor and ligand may also help further explore associations between the role of OR repertoire and ecological niche.</p> </div> <div class="materialsandmethods" title="sec"> <div class="title" tagx="title" title="title">
Materials and Methods</div> <div class="sequencingandassembly" title="sec"><span class="label" tagx="label" title="label">454</span><div class="title" tagx="title" title="title">
Sequencing and Assembly</div> <p>A representative subset of OR genes from 14 species, ten of which are new to this study, (<a href="#sup1">supplementary table S1</a>, <a href="#sup1">Supplementary Material</a> online) were amplified using primer pairs GPC1 and GPC2, (degenerate primers designed to amplify a representative subsample of mammalian OR genes), using protocols described by <a href="#msy028-B46">Hayden et al. (2010)</a>, modified from <a href="#msy028-B35">Gilad et al. (2004)</a>. PCRs were performed in a total volume of 25 µl containing: 2.5 µl 10× PCR Buffer, 2 mM MgCl<sub>2</sub>, 1 U Platinum Taq (Invitrogen Corporation), 0.2 µM dNTP’s, 0.2 µM of each primer, and approximately 10 ng of DNA. Conditions for initial PCR were modified from <a href="#msy028-B35">Gilad et al. (2004)</a>. A first step of denaturation for 10 min at 94°C followed by 35 cycles of denaturation for 15 s, annealing for 30 s at a temperature gradient of 38–50°C, and an extension for 1 min at 72°C, with a final extension for 10 min at 72°C. PCR products were visualized on a 1% agarose gel using SYBR Safe DNA gel stain (Invitrogen Corporation). Samples were quantified using Qubit<sup>®</sup> 3.0 Fluorometer (Life Technologies). The two PCR products per species were pooled then purified and concentrated using Millipore Centrifugal filter units. Samples were sequenced on the Roche 454 FLX+ Titanium sequencer using Lib-L sequencing, with an estimated average amplicon size of 700–750 bp.</p> </div> <div class="sequenceassembly" title="sec"> <div class="title" tagx="title" title="title">
Sequence Assembly</div> <p>The OR gene amplicons were sequenced and assembled for all 14 species (<a href="#sup1">supplementary table S1</a>, <a href="#sup1">Supplementary Material</a> online). For each read of the 14 species, tailing nucleotides with a Q-score of ≤30 were removed using the FastX toolkit (<a href="#msy028-B93">Pearson et al. 1997</a>). Two de Bruijn graph based assemblers, SOAPdenovo (<a href="#msy028-B69">Luo et al. 2012</a>), and ABySS (<a href="#msy028-B113">Simpson et al. 2009</a>) were used to assemble the ORs contained in the read data, collapsing multiple amplicons into their target genes for each species. Such assembly algorithms use a parameter <i>k</i> to control the assembly of reads into contiguous sequences (contigs). Each read was split into smaller fragments of length "<i>k</i>", termed <i>k</i>-mers. Low values for <i>k</i> increase the chances of successful overlaps, and are more sensitive regarding the assembly of fragments compared to larger values. Larger <i>k</i> values however reduce the risk of incorrect overlaps, and are thus more specific. The relatively large average read lengths (700 bp) in these read data allowed a wide range of <i>k</i>-values to be investigated. Two independent <i>k</i> ranges were tested: a low <i>k</i>-value range (20–127) using SOAPdenovo, which allows a maximum <i>k</i> of 127, and high <i>k-</i>value ranges using ABySS. As the number of assembled contigs dropped dramatically for higher <i>k</i> values, a range of <i>k</i>-values between 200 and 500 was applied.</p> <p>Assembly parameters and approaches were tested using four species with previously sequenced genomes: <i>Canis familiaris</i> (dog), <i>Felis catus</i> (domestic cat), <i>Myotis lucifugus</i> (little brown bat), and <i>Rhinolophus ferrumequinum</i> (greater horseshoe bat) to ensure that the NGS data generated was a true representation of the OR subgenome. OR sequences were mined from these genomes using the Olfactory Receptor Assigner (ORA; <a href="#msy028-B46">Hayden et al. 2010</a>) and TBLASTX (<a href="#msy028-B4">Altschul et al. 1990</a>) using previously annotated ORs and served as a benchmark for comparing assembly methods. ORA uses profile Hidden Markov Models (HMMs), based on alignments of mammalian ORs, to scan through a given contig or sequence and identify putative OR genes present. It was used to assign each assembled OR to the correct gene family (as defined by <a href="#msy028-B46">Hayden et al. 2010</a>).</p> </div> <div class="sequenceclustering" title="sec"> <div class="title" tagx="title" title="title">
Sequence Clustering</div> <p>As each read represented a potential amplified OR gene sequence, a clustering method was applied in addition to de Bruijn graph assemblers to collapse amplicon data. Reads for the four test species were sorted into clusters using the CD-HIT 454 clustering package (<a href="#msy028-B33">Fu et al. 2012</a>). With an error rate of 1% per read for 454 sequencing (<a href="#msy028-B36">Glenn 2011</a>), at least seven erroneous bases per read were expected (based on a mean read length of 736 bp), hence clusters were created based on ≥97% sequence identity. This allowed reads to join clusters despite potential sequence errors and undetermined nucleotide positions. Consensus sequences were then made for each cluster. Undetermined nucleotide positions represented by "N" in consensus sequences may lead to multiple sequences representing the same gene. To account for this, all consensus sequences showing ≥98% sequence identity were merged together, considered as one gene and assigned to their appropriate OR family.</p> </div> <div class="clusteringversusassembly" title="sec"> <div class="title" tagx="title" title="title">
Clustering versus Assembly</div> <p>To determine which method of sequence reconstruction (assembly vs. clustering) produced the best representation of the OR repertoire, cluster-consensus and assembled OR sequences were mapped to the genomes of the four test species using BLASTN (<a href="#msy028-B4">Altschul et al. 1990</a>). The clustering and assembly methods were compared based on: </p><ol> <li> <p>Number of unique ORs present in the data;</p> </li> <li> <p>Data redundancy (number of unique ORs vs. number of contigs);</p> </li> <li> <p>Comparison of gene distributions between assembled and genomic ORs;</p> </li> <li> <p>Average percentage identity between sequencing and genomic data;</p> </li> <li> <p>Average length of sequenced genes.</p> </li> </ol> <p /> <p>The distributions of generated ORs and genomic ORs for each species were compared in <i>R</i> using both Pearson’s chi-square test and Fisher’s exact test.</p> </div> <div class="insilicowholegenomeassemblyordata" title="sec"> <div class="title" tagx="title" title="title">
In Silico Whole Genome Assembly OR Data</div> <p>Olfactory Receptor gene sequences were mined from 58 different sequenced mammalian genomes, covering 17 mammalian orders (<a href="#sup1">supplementary table S12</a>, <a href="#sup1">Supplementary Material</a> online and <a href="#msy028-F1">fig. 1</a>). Genes were detected using TBLASTX (<a href="#msy028-B4">Altschul et al. 1990</a>) in addition to gene mining using the ORA (<a href="#msy028-B46">Hayden et al. 2010</a>), as above. While OR annotations of a number of genomes in this study have already been characterized (<a href="#msy028-B38">Glusman et al. 2001</a>; <a href="#msy028-B85">Olender et al. 2004</a>; <a href="#msy028-B63">Lee et al. 2013</a>), we nonetheless applied our mining methods to the raw genome assemblies to ensure a ubiquitous application of the same threshold parameters (such as determining functionality) and methods of data collection across all taxa. Due to this, and differences in genome assembly versions in different studies, we expect to see potential differences in the size of the OR repertoires we have mined compared to previously characterized repertoires (<a href="#msy028-B81">Niimura and Nei 2007</a>; <a href="#msy028-B46">Hayden et al. 2010</a>; <a href="#msy028-B82">Niimura et al. 2014</a>). As some of the genomes used in this study have not yet been annotated, it is possible that a number of identical redundant contigs were present in the whole genome assemblies, either due to different alleles or multiple copies of the same contiguous sequence, leading to an overestimation of the number of ORs per genome. Such potential duplicate sequences were identified through searching for genes with three or fewer nucleotide differences compared to others (≥99% sequence identity), and removed. ORs were grouped into clusters according to their families based on <a href="#msy028-B46">Hayden et al. 2010</a> (OR 1/3/7, OR 2/13, OR 4, OR 5/8/9, OR 6, OR 10, OR 11, OR 12, OR 14, OR 51, OR 52, OR 55, OR 56), were further assigned to a subfamily based on homology to reference OR genes, for example, OR51A1 indicates OR gene family 51, gene 1 of subfamily A, (<a href="#msy028-B37">Glusman et al. 2000</a>) and translated into amino acid sequences. OR genes were considered "putatively functional" (referred to throughout simply as "functional") if they were &amp;gt;650 bp, having the potential to encode the 7-transmembrane GPCR protein structure, and did not contain an in-frame stop codon, as in <a href="#msy028-B46">Hayden et al. (2010)</a> and <a href="#msy028-B59">Khan et al. (2015)</a>, rather than functionality inferred from expression studies. This length threshold allowed us to identify and include functional ORs from fragmented whole genome assemblies or genomes with long, unresolved sequence regions that did not contain a complete ORF. Such an approach allowed an upper estimate of the functional OR repertoire size for species with lower quality genome assemblies, but differs slightly from more conservative methods such as <a href="#msy028-B77">Montague et al. (2014)</a>. An OR gene was treated as non-functional if it contained an in-frame stop codon, insertion or deletion frame-shift mutation. Our computational methods detected functional OR genes, OR genes with a premature stop codon, and truncated or degraded pseudogenes. However, given that relaxed selection after a pseudogenization event can change a gene to the point that it can no longer be detected using sequence homology, we will always underestimate their occurrence with these methods (<a href="#msy028-B5">Balasubramanian et al. 2009</a>). Therefore, our counts of OR pseudogenes represent a minimum number of detectable pseudogenes within the genome.</p> <p>The genomic data set, consisting of ORs from 58 fully sequenced mammals, was termed "Genomic" and contained 70,369 OR genes. These OR genes showed on average 95.91% nucleotide identity when mapped to a subset of annotated RefSeq genes (mean of 98.81% with pseudogenes excluded), confirming they represented true OR sequences.</p> </div> <div class="combinedgenomicandngsdata" title="sec"> <div class="title" tagx="title" title="title">
Combined Genomic and NGS Data</div> <p>A second data set containing 94 taxa, termed "Genomic + NGS", was considered. This data set consisted of OR data from the 58 fully sequenced mammalian genomes coupled with the assembled 454 gene data from the ten species (had no genome sequenced) described above (3,338 OR sequences). In addition, using the GPC1 and GPC2 primer pairs, similar representative subsets of OR genes were sequenced in five additional mammalian species (<i>Ailurus fulgens</i> [red panda], <i>Orycteropus afer</i> [aardvark], <i>Panthera uncia</i> [snow leopard], <i>Trachops cirrohsus</i> [fringe-lipped bat], and <i>Ursus arctos</i> [brown bear]; 2,933 total contigs) using the Illumina sequencing platform, laboratory protocols, and bioinformatic pipelines described by <a href="#msy028-B50">Hughes et al. (2013)</a>. These genes, as well as OR genes from 21 species amplified in previous studies (<a href="#msy028-B46">Hayden et al. 2010</a>, <a href="#msy028-B47">2014</a>; <a href="#msy028-B50">Hughes et al. 2013</a>), were added to the "Genomic + NGS" data set. This resulted in 10,536 OR sequences for 36 taxa (10,454 generated, five Illumina generated, all new to this study; 21 from <a href="#msy028-B46">Hayden et al. (2010)</a> using Sanger sequencing). OR sequences were classified and converted to amino acid sequences as described above. A length of 80 amino acids (240 bp) was used as an operational minimum cutoff in the NGS data to differentiate between short read/assembly fragments and definitive OR sequences (<a href="#msy028-B50">Hughes et al. 2013</a>), allowing for the identification of putative degraded pseudogenes. This reduced the NGS data down to 9,083 sequences, equating to 79,452 ORs in the "Genomic + NGS" data set. As the amplified data represent only a subset rather than the full OR repertoire, analyses were performed with and without the additional 36 taxa (i.e., "Genomic" data set), to investigate potential biases in our results. This allowed us test the robustness of our gene data with respect to additional taxa, niches and sequences.</p> </div> <div class="treebuilding" title="sec"> <div class="title" tagx="title" title="title">
Tree Building</div> <p>The two data sets, "Genomic" and "Genomic + NGS" contained 58 and 94 mammalian species, respectively. OR families were aligned individually using the full distance matrix for guide tree calculation in Clustal Omega (<a href="#msy028-B111">Sievers et al. 2011</a>). Functional and non-functional OR genes were included in each alignment. As both data sets contained 13 OR families, a total of 26 protein alignments were generated. Each alignment was then used to generate phylogenetic trees using RAxML-LITE (<a href="#msy028-B119">Stamatakis et al. 2012</a>), with the Jones–Taylor–Thornton model of sequence evolution, gamma model of rate heterogeneity, and observed amino acid frequencies (JTT+Γ+F). This model was considered the best-fit model of sequence evolution using ProtTest 3 (<a href="#msy028-B19">Darriba et al. 2011</a>) on small alignments (OR families 10, 11, 12, 14, 51, 55, and 56) and PartitionFinder (<a href="#msy028-B62">Lanfear et al. 2012</a>) on the larger ones (OR families 1/3/7, 2/13, 4, 5/8/9, 6, 52).</p> </div> <div class="countingorgeneduplicationevents" title="sec"> <div class="title" tagx="title" title="title">
Counting OR Gene Duplication Events</div> <p>We counted species-specific OR gene duplication (SSD) events, whereby OR genes have undergone duplication leading to two or more paralogous genes (daughters) in a single species alone, that are retained in the genome. To do this, we parsed gene trees for each OR family to identify nodes indicating a SSD or "birth" event. A node was considered to represent an SSD if it contained two or more leaf nodes, all from the same species (<a href="#msy028-F2">fig. 2a</a>). Chained sub trees consisting of a single target species were counted as multiple SSD events (<a href="#msy028-F2">fig. 2b</a>). Under this method of counting duplications, we cannot rule out the possibility of a duplication event in which one daughter gene retains function whilst the other degrades beyond recognition in the genome, and therefore cannot be documented as a duplication event. Although we expect the frequency of such events in extant taxa to be low, the estimates here refer to SSDs resulting in paralogous OR genes (daughters) that are retained and detectable in the genome, and therefore represent minimum levels of observable OR duplication. Additionally, this method will not recover shared ancestral duplication events.</p> <p>The R package "<i>ape</i>" was used to decompose each gene tree into all possible subtrees. A perl script was used to parse these trees, highlighting instances of SSD while relaying specific information about each gene. We identified three types of duplication nodes: duplication events where all daughter paralogous genes are functional, duplication events where all daughter genes have since lost their function and duplication events showing a combination of both.</p> <p>With respect to non-functional OR genes that are one of a duplicated pair, it is unknown if function was lost soon after duplication or at a later point in the evolutionary history of that species. There also exists the possibility that a stop codon is a consequence of sequencing error, however the majority of genome assemblies had a read coverage between 5× and 222×, reducing the likelihood of an incorrect pseudogene assignment. The number of SSDs, the number of OR functional and non functional daughter genes per species (<a href="#msy028-F2">fig. 2c</a>) were counted, with tables generated for each data type. To investigate if duplication was diversifying the OR repertoire, the number of different subfamilies within each OR gene family showing SSD events was determined.</p> </div> <div class="comparisonofmammalianniches" title="sec"> <div class="title" tagx="title" title="title">
Comparison of Mammalian Niches</div> <p>To investigate the potential link between OR gene duplication and ecological adaptation, we considered the following niches: dietary (carnivore, frugivore, insectivore, herbivore, or omnivore), functional VNO(present/absent, see <a href="#sup1">supplementary table S12</a>, <a href="#sup1">Supplementary Material</a> online), sociality (social and solitary), rhythmic activity phase (crepuscular, nocturnal, and diurnal), and habitat (volant, terrestrial, and aquatic). These niches imply many different ecological modalities, environmental conditions, and sensory perception repertoires. Species were assigned to each niche according to data from the literature (<a href="#sup1">supplementary table S12</a>, <a href="#sup1">Supplementary Material</a> online and <a href="#msy028-F6">fig. 6</a>). Data was normalized to allow comparisons across niches and comparisons between genes mined from whole genomes and NGS data. The distribution of SSDs detected in each family was normalized by dividing the number of observed SSDs per family by the total SSDs observed for that species (<a href="#sup1">supplementary table S13</a>, <a href="#sup1">Supplementary Material</a> online, see worked example), and this normalization was also applied to the counts of functional and non-functional daughters. The percentage of the OR repertoire that has undergone SSD in a given taxon was determined as the ratio of total SSD daughters to the total number of OR genes (functional and non-functional) in the full repertoire (e.g., see <a href="#sup1">supplementary table S13</a>, <a href="#sup1">Supplementary Material</a> online). This was used to investigate if the OR repertoire expansion differs between various niche types. It was only estimated from the "Genomic" data set, as this data set was considered a "complete" catalogue of all OR genes present in the whole genome assemblies for species included.</p> </div> <div class="statisticalcomparisonsofspecies-specificorduplicationdata" title="sec"> <div class="title" tagx="title" title="title">
Statistical Comparisons of Species-Specific OR Duplication Data</div> <p>We analyzed the pattern of OR gene evolution relative to niche adaptation using several statistical comparisons. First, OR families within each species were ranked from highest to lowest based on the number of OR SSD functional daughters. To investigate if the addition of taxa from the NGS data set affected observed rates of OR gene family evolution significantly, we compared the distributions of SSDs in the 58 "Genomic" species with the combined "Genomic" and additional 36 NGS species, using Fisher’s exact test.</p> <p>We analyzed mammalian OR gene duplication data relative to the phylogenetic tree using two complimentary techniques. First, we performed a PGLS (<a href="#msy028-B56">Kamilar and Cooper 2013</a>). Second, we fitted a Poisson model to the SSD data, comparing rates of duplication across ecological niches.</p> <p>Phylogenetic generalized least squares is a general linear model, which accounts for phylogenetic history by allowing correlations between predictor and response variables while controlling for the nonindependence due to phylogeny. Analysis of the data in this manner considers OR SSDs explicitly in the context of the phylogenetic tree. PGLS can accommodate different models of character evolution and branch scaling parameters (<a href="#msy028-B34">Garland and Ives 2000</a>). Here, we used Pagel’s λ (<a href="#msy028-B89">Pagel 1999</a>) as a scaling factor, estimated using maximum likelihood. The λ value can range from 0 (no phylogenetic signal) to 1 (trait data matched a Brownian model of evolution). For this study, the trait data consisted of the total number of SSD functional and non-functional daughter ORs across taxa with known niche data. We used a composite mammalian phylogeny based on <a href="#msy028-B74">Meredith et al. (2011)</a> and <a href="#msy028-B30">Foley et al. (2016)</a> for interordinal relationships, with species relationships added based on <a href="#msy028-B103">Ruedi et al. (2013)</a>, <a href="#msy028-B2">Almeida et al. (2014)</a>, and <a href="#msy028-B47">Hayden et al. (2014</a>; <a href="#sup1">supplementary table S12</a>, <a href="#sup1">Supplementary Material</a> online). We estimated branch lengths using the method devised by <a href="#msy028-B39">Grafen (1989)</a>, implemented in the "<i>ape</i>" package in R (<a href="#msy028-B92">Paradis et al. 2004</a>). We used the R packages "<i>ape</i>" (<a href="#msy028-B92">Paradis et al. 2004</a>), "<i>geiger</i>" (<a href="#msy028-B43">Harmon et al. 2008</a>), "<i>nlme</i>" (<a href="#msy028-B95">Pinheiro et al. 2015</a>), and "<i>phytools</i>" (<a href="#msy028-B100">Revell 2012</a>) to read phylogenetic trees and run the PGLS. Instances of λ &amp;gt; 0.90 were not considered evidence of ecological adaptation.</p> <p>We modeled the rate of SSDs detected in each Class II OR gene family using a fitted Poisson model (<a href="#msy028-B25">Edwards 1992</a>; <a href="#msy028-B9">Burnham and Anderson 2002</a>; <a href="#sup1">supplementary table S11</a>, <a href="#sup1">Supplementary Material</a> online). This model relates the rate of SSDs to different ecological factors. Analysis of the data in this manner considers OR gene duplications in the context of different classes or partitions of the ecological variables. As such, rates of SSDs are analyzed with respect to these variables in a manner similar to "treatment classes" (<a href="#msy028-B28">Finarelli and Goswami 2013</a>, <a href="#msy028-B29">Goswami and Finarelli 2016</a>; <a href="#msy028-B124">Vartia et al. 2016</a>). Taking the sociality variable as an example, we compared rates of SSDs across the two sociality "treatments": social versus solitary. It should be noted that this model is being used to identify the relative strengths of association between ecological variables and the SSD counts observed in mammalian species, not to construct predictive model for the purpose of estimation of SSD counts.</p> <p>We considered a total of 23 models. Duplication rate estimates were modeled for each variable as: 1) a single rate for all gene families for each ecological niche category, 2) as a distinct estimated rate for each individual OR gene family, and 3) as a distinct rate for each of the two major groups apparent in the data: (OR1/3/7, OR2/13, OR4, and OR5/8/9) versus (OR6, OR10, OR11, OR12, and OR14). Taking sociality again for the purpose of example, we modeled a single rate for all gene families for social and for solitary taxa (two rate parameters), distinct rates for social and solitary taxa for each of the gene families (18 rate parameters), and rates for social and solitary taxa for each of the two major OR family groups (four rate parameters). We employed two coding schemes for dietary niche: a five-state condition with carnivory, frugivory, insectivory, herbivory, and omnivory, and a three-state partition of "animalivorous" (carnivory and insectivory), "herbivory" (frugivorous and herbivory), and omnivorous. Model selection was carried out using the Akaike Information Criterion (<a href="#msy028-B1">Akaike 1973</a>; <a href="#msy028-B9">Burnham and Anderson 2002</a>) using the finite sample corrected AIC (AICc; <a href="#msy028-B51">Hurvich and Tsai 1989</a>) converting AICc scores to posterior probabilities across the set of examined models. This model selection procedure was performed using the same set of models for SSDs; functional daughters; non-functional daughters in both data sets ("Genomic" and "Genomic + NGS").</p> </div> </div> <div class="supplementary-material" title="supplementary-material"> <div class="title" tagx="title" title="title">
Supplementary Material</div> <p><a href="#sup1">Supplementary data</a> are available at <i>Molecular Biology and Evolution</i> online.</p> </div> <div class="supplementary-material" title="supplementary-material"> <div class="title" tagx="title" title="title">
Supplementary Material</div> <div class="local-data" title="local-data"><span class="label" tagx="label" title="label">Supplementary Data</span><div class="media" title="media"><a href="msy028_supp.zip">LINK</a> <p>Click here for additional data file.</p>  </div> </div> </div> </div> <div class="back" title="back"> <div class="ack" title="ack"> <div class="title" tagx="title" title="title">
Acknowledgments</div> <p>This project was funded: by the Irish Research Council (IRC) Graduate Research Education Programme (GREP) awarded to G.H., E.C.T., and D.H.; a Science Foundation Ireland PIYRA 06/YI3/B932; a Short Term Travel Fellowship 06/Y13/B932STTF08 awarded to E.C.T. Currently, E.C.T. is funded by a European Research Council Starting grant ERC-2012-StG311000. W.J.M. acknowledges funding from NSF (EF0629849). We thank Michael Bekaert for ORA and Nicole Foley and all of Batlab for their input. Animal silhouettes used in figures were downloaded from phylopic.org under Creative Commons Public Domain Dedication (CC0 1.0) and Creative Commons Attribution 3.0 Unported, unaltered images (CC BY 3.0, Sarah Werning, T. Michael Keesey, after Mauricio Antón). We would like to thank Amanda Melin and our other anonymous reviewers for taking the time to read our work and providing constructive suggestions.</p> </div> <div class="references">
References</div> <div tag="ref-list"> <ul> <div class="title" tagx="title" title="title">
References</div> <li tag="ref"><a name="msy028-B1" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Akaike</span><span class="given-names" tagx="given-names" title="given-names">H.</span></span></span><span class="year" tagx="year" title="year">1973</span><span class="chapter-title" tagx="chapter-title" title="chapter-title">Information theory as an extension of the maximum likelihood principle</span> In: <span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Petrov</span><span class="given-names" tagx="given-names" title="given-names">BN</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Casaki</span><span class="given-names" tagx="given-names" title="given-names">F</span></span></span>, editors. <span class="source" tagx="source" title="source">Second international symposium on information theory</span>. <span class="publisher-loc" tagx="publisher-loc" title="publisher-loc">Budapest</span>: <span class="publisher-name" tagx="publisher-name" title="publisher-name">Akademiai Kiado</span>, p. <span class="fpage" tagx="fpage" title="fpage">267</span>–<span class="lpage" tagx="lpage" title="lpage">281</span>.</span></li> <li tag="ref"><a name="msy028-B2" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Almeida</span><span class="given-names" tagx="given-names" title="given-names">FC</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Giannini</span><span class="given-names" tagx="given-names" title="given-names">NP</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Simmons</span><span class="given-names" tagx="given-names" title="given-names">NB</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Helgen</span><span class="given-names" tagx="given-names" title="given-names">KM.</span></span></span><span class="year" tagx="year" title="year">2014</span><span class="mixed-article-title" title="mixed-article-title">Each flying fox on its own branch: a phylogenetic tree for <i>Pteropus</i> and related genera (Chiroptera: Pteropodidae)</span>. <span class="source" tagx="source" title="source">Mol Phylogenet Evol.</span><span class="volume" tagx="volume" title="volume">77</span>:<span class="fpage" tagx="fpage" title="fpage">83</span>–<span class="lpage" tagx="lpage" title="lpage">95</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24662680">24662680</a></span></span></li> <li tag="ref"><a name="msy028-B3" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Aloni</span><span class="given-names" tagx="given-names" title="given-names">R</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Olender</span><span class="given-names" tagx="given-names" title="given-names">T</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lancet</span><span class="given-names" tagx="given-names" title="given-names">D.</span></span></span><span class="year" tagx="year" title="year">2006</span><span class="mixed-article-title" title="mixed-article-title">Ancient genomic architecture for mammalian olfactory receptor clusters</span>. <span class="source" tagx="source" title="source">Genome Biol.</span><span class="volume" tagx="volume" title="volume">7</span>(<span class="issue" tagx="issue" title="issue">10</span>): <span class="fpage" tagx="fpage" title="fpage">R88.</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17010214">17010214</a></span></span></li> <li tag="ref"><a name="msy028-B4" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Altschul</span><span class="given-names" tagx="given-names" title="given-names">SF</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gish</span><span class="given-names" tagx="given-names" title="given-names">W</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Miller</span><span class="given-names" tagx="given-names" title="given-names">W</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Myers</span><span class="given-names" tagx="given-names" title="given-names">EW</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lipman</span><span class="given-names" tagx="given-names" title="given-names">DJ.</span></span></span><span class="year" tagx="year" title="year">1990</span><span class="mixed-article-title" title="mixed-article-title">Basic local alignment search tool</span>. <span class="source" tagx="source" title="source">J Mol Biol.</span><span class="volume" tagx="volume" title="volume">215</span>(<span class="issue" tagx="issue" title="issue">3</span>): <span class="fpage" tagx="fpage" title="fpage">403</span>–<span class="lpage" tagx="lpage" title="lpage">410</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/2231712">2231712</a></span></span></li> <li tag="ref"><a name="msy028-B5" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Balasubramanian</span><span class="given-names" tagx="given-names" title="given-names">S</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zheng</span><span class="given-names" tagx="given-names" title="given-names">D</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Liu</span><span class="given-names" tagx="given-names" title="given-names">Y-J</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Fang</span><span class="given-names" tagx="given-names" title="given-names">G</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Frankish</span><span class="given-names" tagx="given-names" title="given-names">A</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Carriero</span><span class="given-names" tagx="given-names" title="given-names">N</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Robilotto</span><span class="given-names" tagx="given-names" title="given-names">R</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Cayting</span><span class="given-names" tagx="given-names" title="given-names">P</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gerstein</span><span class="given-names" tagx="given-names" title="given-names">M.</span></span></span><span class="year" tagx="year" title="year">2009</span><span class="mixed-article-title" title="mixed-article-title">Comparative analysis of processed ribosomal protein pseudogenes in four mammalian genomes</span>. <span class="source" tagx="source" title="source">Genome Biol.</span><span class="volume" tagx="volume" title="volume">10</span>(<span class="issue" tagx="issue" title="issue">1</span>): <span class="fpage" tagx="fpage" title="fpage">R2.</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19123937">19123937</a></span></span></li> <li tag="ref"><a name="msy028-B6" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bates</span><span class="given-names" tagx="given-names" title="given-names">LA</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Sayialel</span><span class="given-names" tagx="given-names" title="given-names">KN</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Njiraini</span><span class="given-names" tagx="given-names" title="given-names">NW</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Poole</span><span class="given-names" tagx="given-names" title="given-names">JH</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Moss</span><span class="given-names" tagx="given-names" title="given-names">CJ</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Byrne</span><span class="given-names" tagx="given-names" title="given-names">RW.</span></span></span><span class="year" tagx="year" title="year">2008</span><span class="mixed-article-title" title="mixed-article-title">African elephants have expectations about the locations of out-of-sight family members</span>. <span class="source" tagx="source" title="source">Biol Lett.</span><span class="volume" tagx="volume" title="volume">4</span>(<span class="issue" tagx="issue" title="issue">1</span>): <span class="fpage" tagx="fpage" title="fpage">34</span>–<span class="lpage" tagx="lpage" title="lpage">36</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18055407">18055407</a></span></span></li> <li tag="ref"><a name="msy028-B7" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bennie</span><span class="given-names" tagx="given-names" title="given-names">JJ</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Duffy</span><span class="given-names" tagx="given-names" title="given-names">JP</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Inger</span><span class="given-names" tagx="given-names" title="given-names">R</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gaston</span><span class="given-names" tagx="given-names" title="given-names">KJ.</span></span></span><span class="year" tagx="year" title="year">2014</span><span class="mixed-article-title" title="mixed-article-title">Biogeography of time partitioning in mammals</span>. <span class="source" tagx="source" title="source">Proc Natl Acad Sci U S A.</span><span class="volume" tagx="volume" title="volume">111</span> (<span class="issue" tagx="issue" title="issue">38</span>): <span class="fpage" tagx="fpage" title="fpage">13727</span>–<span class="lpage" tagx="lpage" title="lpage">13732</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25225371">25225371</a></span></span></li> <li tag="ref"><a name="msy028-B9" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Burnham</span><span class="given-names" tagx="given-names" title="given-names">KP</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Anderson</span><span class="given-names" tagx="given-names" title="given-names">DR.</span></span></span><span class="year" tagx="year" title="year">2002</span><span class="source" tagx="source" title="source">Model selection and multimodel inference: a practical information–theoretic approach</span>. <span class="publisher-loc" tagx="publisher-loc" title="publisher-loc">New York</span>: <span class="publisher-name" tagx="publisher-name" title="publisher-name">Springer</span>.</span></li> <li tag="ref"><a name="msy028-B12" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Challet</span><span class="given-names" tagx="given-names" title="given-names">E.</span></span></span><span class="year" tagx="year" title="year">2007</span><span class="mixed-article-title" title="mixed-article-title">Minireview: entrainment of the suprachiasmatic clockwork in diurnal and nocturnal mammals</span>. <span class="source" tagx="source" title="source">Endocrinology</span><span class="volume" tagx="volume" title="volume">148</span> (<span class="issue" tagx="issue" title="issue">12</span>): <span class="fpage" tagx="fpage" title="fpage">5648</span>–<span class="lpage" tagx="lpage" title="lpage">5655</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17901231">17901231</a></span></span></li> <li tag="ref"><a name="msy028-B13" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Chang</span><span class="given-names" tagx="given-names" title="given-names">D</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Duda</span><span class="given-names" tagx="given-names" title="given-names">TF</span><span class="suffix" tagx="suffix" title="suffix">Jr.</span></span></span><span class="year" tagx="year" title="year">2012</span><span class="mixed-article-title" title="mixed-article-title">Extensive and continuous duplication facilitates rapid evolution and diversification of gene families</span>. <span class="source" tagx="source" title="source">Mol Biol Evol.</span><span class="volume" tagx="volume" title="volume">29</span>(<span class="issue" tagx="issue" title="issue">8</span>): <span class="fpage" tagx="fpage" title="fpage">2019</span>–<span class="lpage" tagx="lpage" title="lpage">2029</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22337864">22337864</a></span></span></li> <li tag="ref"><a name="msy028-B14" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Chesser</span><span class="given-names" tagx="given-names" title="given-names">RK</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Sugg</span><span class="given-names" tagx="given-names" title="given-names">DW</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Rhodes</span><span class="given-names" tagx="given-names" title="given-names">OE</span><span class="suffix" tagx="suffix" title="suffix">Jr</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Novak</span><span class="given-names" tagx="given-names" title="given-names">JM</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Smith</span><span class="given-names" tagx="given-names" title="given-names">MH.</span></span></span><span class="year" tagx="year" title="year">1993</span><span class="mixed-article-title" title="mixed-article-title">Evolution of mammalian social structure</span>. <span class="source" tagx="source" title="source">Acta Theoriol</span>. <span class="volume" tagx="volume" title="volume">38</span>:<span class="fpage" tagx="fpage" title="fpage">163</span>–<span class="lpage" tagx="lpage" title="lpage">174</span>.</span></li> <li tag="ref"><a name="msy028-B18" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Cotton</span><span class="given-names" tagx="given-names" title="given-names">JA.</span></span></span><span class="year" tagx="year" title="year">2008</span><span class="chapter-title" tagx="chapter-title" title="chapter-title">The impact of gene duplication on human genome evolution</span><span class="source" tagx="source" title="source">Encyclopedia of life sciences</span>. <span class="publisher-loc" tagx="publisher-loc" title="publisher-loc">Chichester</span>: <span class="publisher-name" tagx="publisher-name" title="publisher-name">John Wiley and Sons</span>.</span></li> <li tag="ref"><a name="msy028-B19" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Darriba</span><span class="given-names" tagx="given-names" title="given-names">D</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Taboada</span><span class="given-names" tagx="given-names" title="given-names">GL</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Doallo</span><span class="given-names" tagx="given-names" title="given-names">R</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Posada</span><span class="given-names" tagx="given-names" title="given-names">D.</span></span></span><span class="year" tagx="year" title="year">2011</span><span class="mixed-article-title" title="mixed-article-title">ProtTest 3: fast selection of best-fit models of protein evolution</span>. <span class="source" tagx="source" title="source">Bioinformatics</span><span class="volume" tagx="volume" title="volume">27</span>(<span class="issue" tagx="issue" title="issue">8</span>): <span class="fpage" tagx="fpage" title="fpage">1164</span>–<span class="lpage" tagx="lpage" title="lpage">1165</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21335321">21335321</a></span></span></li> <li tag="ref"><a name="msy028-B22" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Dos Reis</span><span class="given-names" tagx="given-names" title="given-names">M</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Donoghue</span><span class="given-names" tagx="given-names" title="given-names">PCJ</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Yang</span><span class="given-names" tagx="given-names" title="given-names">Z.</span></span></span><span class="year" tagx="year" title="year">2014</span><span class="mixed-article-title" title="mixed-article-title">Neither phylogenomic nor paleontological data support a paleogene origin of placental mammals</span>. <span class="source" tagx="source" title="source">Biol Lett.</span><span class="volume" tagx="volume" title="volume">10</span> (<span class="issue" tagx="issue" title="issue">1</span>): <span class="fpage" tagx="fpage" title="fpage">20131003.</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24429684">24429684</a></span></span></li> <li tag="ref"><a name="msy028-B24" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Dunkel</span><span class="given-names" tagx="given-names" title="given-names">A</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Steinhaus</span><span class="given-names" tagx="given-names" title="given-names">M</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kotthoff</span><span class="given-names" tagx="given-names" title="given-names">M</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Nowak</span><span class="given-names" tagx="given-names" title="given-names">B</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Krautwurst</span><span class="given-names" tagx="given-names" title="given-names">D</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Schieberle</span><span class="given-names" tagx="given-names" title="given-names">P</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hofmann</span><span class="given-names" tagx="given-names" title="given-names">T.</span></span></span><span class="year" tagx="year" title="year">2014</span><span class="mixed-article-title" title="mixed-article-title">Nature’s chemical signatures in human olfaction: a foodborne perspective for future biotechnology</span>. <span class="source" tagx="source" title="source">Angew Chem.</span><span class="volume" tagx="volume" title="volume">53</span> (<span class="issue" tagx="issue" title="issue">28</span>): <span class="fpage" tagx="fpage" title="fpage">7124</span>–<span class="lpage" tagx="lpage" title="lpage">7143</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24939725">24939725</a></span></span></li> <li tag="ref"><a name="msy028-B25" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Edwards</span><span class="given-names" tagx="given-names" title="given-names">AWF.</span></span></span><span class="year" tagx="year" title="year">1992</span><span class="source" tagx="source" title="source">Likelihood: expanded edition</span>. <span class="publisher-loc" tagx="publisher-loc" title="publisher-loc">Baltimore</span>: <span class="publisher-name" tagx="publisher-name" title="publisher-name">The Johns Hopkins University Press</span>.</span></li> <li tag="ref"><a name="msy028-B27" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Farbiszewski</span><span class="given-names" tagx="given-names" title="given-names">R</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kranc</span><span class="given-names" tagx="given-names" title="given-names">R.</span></span></span><span class="year" tagx="year" title="year">2013</span><span class="mixed-article-title" title="mixed-article-title">Olfactory receptors and the mechanism of odor perception</span>. <span class="source" tagx="source" title="source">Pol Ann Med.</span><span class="volume" tagx="volume" title="volume">20</span>(<span class="issue" tagx="issue" title="issue">1</span>): <span class="fpage" tagx="fpage" title="fpage">51</span>–<span class="lpage" tagx="lpage" title="lpage">55</span>.</span></li> <li tag="ref"><a name="msy028-B28" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Finarelli</span><span class="given-names" tagx="given-names" title="given-names">JA</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Goswami</span><span class="given-names" tagx="given-names" title="given-names">A.</span></span></span><span class="year" tagx="year" title="year">2013</span><span class="mixed-article-title" title="mixed-article-title">Potential pitfalls of reconstructing deep time evolutionary history with only extant data, a case study using the Canidae (Mammalia, Carnivora)</span>. <span class="source" tagx="source" title="source">Evolution</span><span class="volume" tagx="volume" title="volume">67</span> (<span class="issue" tagx="issue" title="issue">12</span>): <span class="fpage" tagx="fpage" title="fpage">3678</span>–<span class="lpage" tagx="lpage" title="lpage">3685</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24106995">24106995</a></span></span></li> <li tag="ref"><a name="msy028-B30" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Foley</span><span class="given-names" tagx="given-names" title="given-names">NM</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Springer</span><span class="given-names" tagx="given-names" title="given-names">MS</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Teeling</span><span class="given-names" tagx="given-names" title="given-names">EC.</span></span></span><span class="year" tagx="year" title="year">2016</span><span class="mixed-article-title" title="mixed-article-title">Mammal madness: is the mammal tree of life not yet resolved?</span>. <span class="source" tagx="source" title="source">Philos Trans R Soc Lond B Biol Sci.</span><span class="volume" tagx="volume" title="volume">371</span>:<span class="fpage" tagx="fpage" title="fpage">20150140</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/27325836">27325836</a></span></span></li> <li tag="ref"><a name="msy028-B33" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Fu</span><span class="given-names" tagx="given-names" title="given-names">L</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Niu</span><span class="given-names" tagx="given-names" title="given-names">B</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zhu</span><span class="given-names" tagx="given-names" title="given-names">Z</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wu</span><span class="given-names" tagx="given-names" title="given-names">S</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Li</span><span class="given-names" tagx="given-names" title="given-names">W.</span></span></span><span class="year" tagx="year" title="year">2012</span><span class="mixed-article-title" title="mixed-article-title">CD-HIT: accelerated for clustering the next generation sequencing data</span>. <span class="source" tagx="source" title="source">Bioinformatics</span><span class="volume" tagx="volume" title="volume">28</span>(<span class="issue" tagx="issue" title="issue">23</span>): <span class="fpage" tagx="fpage" title="fpage">3150</span>–<span class="lpage" tagx="lpage" title="lpage">3152</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23060610">23060610</a></span></span></li> <li tag="ref"><a name="msy028-B34" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Garland</span><span class="given-names" tagx="given-names" title="given-names">T</span><span class="suffix" tagx="suffix" title="suffix">Jr</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ives</span><span class="given-names" tagx="given-names" title="given-names">AR.</span></span></span><span class="year" tagx="year" title="year">2000</span><span class="mixed-article-title" title="mixed-article-title">Using the past to predict the present: confidence intervals for regression equations in phylogenetic comparative methods</span>. <span class="source" tagx="source" title="source">Am Nat.</span><span class="volume" tagx="volume" title="volume">155</span>(<span class="issue" tagx="issue" title="issue">3</span>): <span class="fpage" tagx="fpage" title="fpage">346</span>–<span class="lpage" tagx="lpage" title="lpage">364</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/10718731">10718731</a></span></span></li> <li tag="ref"><a name="msy028-B35" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gilad</span><span class="given-names" tagx="given-names" title="given-names">Y</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wiebe</span><span class="given-names" tagx="given-names" title="given-names">V</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Przeworski</span><span class="given-names" tagx="given-names" title="given-names">M</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lancet</span><span class="given-names" tagx="given-names" title="given-names">D</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Paabo</span><span class="given-names" tagx="given-names" title="given-names">S.</span></span></span><span class="year" tagx="year" title="year">2004</span><span class="mixed-article-title" title="mixed-article-title">Loss of olfactory receptor genes coincides with the acquisition of full trichromatic vision in primates</span>. <span class="source" tagx="source" title="source">PLoS Biol.</span><span class="volume" tagx="volume" title="volume">2</span>(<span class="issue" tagx="issue" title="issue">1</span>): <span class="fpage" tagx="fpage" title="fpage">E5.</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/14737185">14737185</a></span></span></li> <li tag="ref"><a name="msy028-B36" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Glenn</span><span class="given-names" tagx="given-names" title="given-names">T.</span></span></span><span class="year" tagx="year" title="year">2011</span><span class="mixed-article-title" title="mixed-article-title">Field guide to next generation DNA sequencers</span>. <span class="source" tagx="source" title="source">Mol Ecol Res.</span><span class="volume" tagx="volume" title="volume">11</span> (<span class="issue" tagx="issue" title="issue">5</span>): <span class="fpage" tagx="fpage" title="fpage">759</span>–<span class="lpage" tagx="lpage" title="lpage">769</span>.</span></li> <li tag="ref"><a name="msy028-B37" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Glusman</span><span class="given-names" tagx="given-names" title="given-names">G</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bahar</span><span class="given-names" tagx="given-names" title="given-names">A</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Sharon</span><span class="given-names" tagx="given-names" title="given-names">D</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Pilpel</span><span class="given-names" tagx="given-names" title="given-names">Y</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">White</span><span class="given-names" tagx="given-names" title="given-names">J</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lancet</span><span class="given-names" tagx="given-names" title="given-names">D.</span></span></span><span class="year" tagx="year" title="year">2000</span><span class="mixed-article-title" title="mixed-article-title">The olfactory receptor gene superfamily: data mining, classification, and nomenclature</span>. <span class="source" tagx="source" title="source">Mamm Genome</span><span class="volume" tagx="volume" title="volume">11</span> (<span class="issue" tagx="issue" title="issue">11</span>): <span class="fpage" tagx="fpage" title="fpage">1016</span>–<span class="lpage" tagx="lpage" title="lpage">1023</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11063259">11063259</a></span></span></li> <li tag="ref"><a name="msy028-B38" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Glusman</span><span class="given-names" tagx="given-names" title="given-names">G</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Yanai</span><span class="given-names" tagx="given-names" title="given-names">I</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Rubin</span><span class="given-names" tagx="given-names" title="given-names">I</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lancet</span><span class="given-names" tagx="given-names" title="given-names">D.</span></span></span><span class="year" tagx="year" title="year">2001</span><span class="mixed-article-title" title="mixed-article-title">The complete human olfactory subgenome</span>. <span class="source" tagx="source" title="source">Genome Res.</span><span class="volume" tagx="volume" title="volume">11</span>(<span class="issue" tagx="issue" title="issue">5</span>): <span class="fpage" tagx="fpage" title="fpage">685</span>–<span class="lpage" tagx="lpage" title="lpage">702</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11337468">11337468</a></span></span></li> <li tag="ref"><a name="msy028-B29" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Goswami</span><span class="given-names" tagx="given-names" title="given-names">A</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Finarelli</span><span class="given-names" tagx="given-names" title="given-names">JA.</span></span></span><span class="year" tagx="year" title="year">2016</span><span class="mixed-article-title" title="mixed-article-title">EMMLi: a maximum likelihood approach to the analysis of modularity</span>. <span class="source" tagx="source" title="source">Evolution</span><span class="volume" tagx="volume" title="volume">70</span> (<span class="issue" tagx="issue" title="issue">7</span>): <span class="fpage" tagx="fpage" title="fpage">1622</span>–<span class="lpage" tagx="lpage" title="lpage">1637</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/27188434">27188434</a></span></span></li> <li tag="ref"><a name="msy028-B39" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Grafen</span><span class="given-names" tagx="given-names" title="given-names">A.</span></span></span><span class="year" tagx="year" title="year">1989</span><span class="mixed-article-title" title="mixed-article-title">The phylogenetic regression</span>. <span class="source" tagx="source" title="source">Philos Trans R Soc Lond B Biol Sci.</span><span class="volume" tagx="volume" title="volume">326</span>(<span class="issue" tagx="issue" title="issue">1233</span>): <span class="fpage" tagx="fpage" title="fpage">119</span>–<span class="lpage" tagx="lpage" title="lpage">157</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/2575770">2575770</a></span></span></li> <li tag="ref"><a name="msy028-B41" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Granados-Fuentes</span><span class="given-names" tagx="given-names" title="given-names">D</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Tseng</span><span class="given-names" tagx="given-names" title="given-names">A</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Herzog</span><span class="given-names" tagx="given-names" title="given-names">ED.</span></span></span><span class="year" tagx="year" title="year">2006</span><span class="mixed-article-title" title="mixed-article-title">A circadian clock in the olfactory bulb controls olfactory responsivity</span>. <span class="source" tagx="source" title="source">J Neurosci.</span><span class="volume" tagx="volume" title="volume">26</span> (<span class="issue" tagx="issue" title="issue">47</span>): <span class="fpage" tagx="fpage" title="fpage">12219</span>–<span class="lpage" tagx="lpage" title="lpage">12225</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17122046">17122046</a></span></span></li> <li tag="ref"><a name="msy028-B43" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Harmon</span><span class="given-names" tagx="given-names" title="given-names">LJ</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Weir</span><span class="given-names" tagx="given-names" title="given-names">JT</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Brock</span><span class="given-names" tagx="given-names" title="given-names">CD</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Glor</span><span class="given-names" tagx="given-names" title="given-names">RE</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Challenger</span><span class="given-names" tagx="given-names" title="given-names">W.</span></span></span><span class="year" tagx="year" title="year">2008</span><span class="mixed-article-title" title="mixed-article-title">GEIGER: investigating evolutionary radiations</span>. <span class="source" tagx="source" title="source">Bioinformatics</span><span class="volume" tagx="volume" title="volume">24</span> (<span class="issue" tagx="issue" title="issue">1</span>): <span class="fpage" tagx="fpage" title="fpage">129</span>–<span class="lpage" tagx="lpage" title="lpage">131</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18006550">18006550</a></span></span></li> <li tag="ref"><a name="msy028-B44" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Harris</span><span class="given-names" tagx="given-names" title="given-names">RM</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hofmann</span><span class="given-names" tagx="given-names" title="given-names">HA.</span></span></span><span class="year" tagx="year" title="year">2015</span><span class="mixed-article-title" title="mixed-article-title">Seeing is believing: dynamic evolution of gene families</span>. <span class="source" tagx="source" title="source">Proc Natl Acad Sci U S A.</span><span class="volume" tagx="volume" title="volume">112</span> (<span class="issue" tagx="issue" title="issue">5</span>): <span class="fpage" tagx="fpage" title="fpage">1252</span>–<span class="lpage" tagx="lpage" title="lpage">1253</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25624486">25624486</a></span></span></li> <li tag="ref"><a name="msy028-B45" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Haussler</span><span class="given-names" tagx="given-names" title="given-names">D</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">O’Brien</span><span class="given-names" tagx="given-names" title="given-names">SJ</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ryder</span><span class="given-names" tagx="given-names" title="given-names">O</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Barker</span><span class="given-names" tagx="given-names" title="given-names">FK</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Clamp</span><span class="given-names" tagx="given-names" title="given-names">M</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Crawford</span><span class="given-names" tagx="given-names" title="given-names">AJ</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hanner</span><span class="given-names" tagx="given-names" title="given-names">R</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hanotte</span><span class="given-names" tagx="given-names" title="given-names">O</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Johnson</span><span class="given-names" tagx="given-names" title="given-names">WE</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">McGuire</span><span class="given-names" tagx="given-names" title="given-names">JA.</span></span></span><span class="year" tagx="year" title="year">2009</span><span class="mixed-article-title" title="mixed-article-title">Genome 10K: a proposal to obtain whole-genome sequence for 10,000 vertebrate species</span>. <span class="source" tagx="source" title="source">J Hered.</span><span class="volume" tagx="volume" title="volume">100</span> (<span class="issue" tagx="issue" title="issue">6</span>): <span class="fpage" tagx="fpage" title="fpage">659</span>–<span class="lpage" tagx="lpage" title="lpage">674</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19892720">19892720</a></span></span></li> <li tag="ref"><a name="msy028-B46" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hayden</span><span class="given-names" tagx="given-names" title="given-names">S</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bekaert</span><span class="given-names" tagx="given-names" title="given-names">M</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Crider</span><span class="given-names" tagx="given-names" title="given-names">TA</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Mariani</span><span class="given-names" tagx="given-names" title="given-names">S</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Murphy</span><span class="given-names" tagx="given-names" title="given-names">WJ</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Teeling</span><span class="given-names" tagx="given-names" title="given-names">EC.</span></span></span><span class="year" tagx="year" title="year">2010</span><span class="mixed-article-title" title="mixed-article-title">Ecological adaptation determined functional mammalian olfactory subgenomes</span>. <span class="source" tagx="source" title="source">Genome Res.</span><span class="volume" tagx="volume" title="volume">20</span>(<span class="issue" tagx="issue" title="issue">1</span>): <span class="fpage" tagx="fpage" title="fpage">1</span>–<span class="lpage" tagx="lpage" title="lpage">9</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19952139">19952139</a></span></span></li> <li tag="ref"><a name="msy028-B47" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hayden</span><span class="given-names" tagx="given-names" title="given-names">S</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bekaert</span><span class="given-names" tagx="given-names" title="given-names">M</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Goodbla</span><span class="given-names" tagx="given-names" title="given-names">A</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Murphy</span><span class="given-names" tagx="given-names" title="given-names">WJ</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Dávalos</span><span class="given-names" tagx="given-names" title="given-names">LM</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Teeling</span><span class="given-names" tagx="given-names" title="given-names">EC.</span></span></span><span class="year" tagx="year" title="year">2014</span><span class="mixed-article-title" title="mixed-article-title">A cluster of olfactory receptor genes linked to frugivory in bats</span>. <span class="source" tagx="source" title="source">Mol Biol Evol.</span><span class="volume" tagx="volume" title="volume">31</span> (<span class="issue" tagx="issue" title="issue">4</span>): <span class="fpage" tagx="fpage" title="fpage">917</span>–<span class="lpage" tagx="lpage" title="lpage">927</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24441035">24441035</a></span></span></li> <li tag="ref"><a name="msy028-B50" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hughes</span><span class="given-names" tagx="given-names" title="given-names">GM</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gang</span><span class="given-names" tagx="given-names" title="given-names">L</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Murphy</span><span class="given-names" tagx="given-names" title="given-names">WJ</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Higgins</span><span class="given-names" tagx="given-names" title="given-names">DG</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Teeling</span><span class="given-names" tagx="given-names" title="given-names">EC.</span></span></span><span class="year" tagx="year" title="year">2013</span><span class="mixed-article-title" title="mixed-article-title">Using illumina next generation sequencing technologies to sequence multigene families in <i>de novo</i> Species</span>. <span class="source" tagx="source" title="source">Mol Ecol Res.</span><span class="volume" tagx="volume" title="volume">13</span>(<span class="issue" tagx="issue" title="issue">3</span>): <span class="fpage" tagx="fpage" title="fpage">510</span>–<span class="lpage" tagx="lpage" title="lpage">533</span>.</span></li> <li tag="ref"><a name="msy028-B51" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hurvich</span><span class="given-names" tagx="given-names" title="given-names">CM</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Tsai</span><span class="given-names" tagx="given-names" title="given-names">C–L.</span></span></span><span class="year" tagx="year" title="year">1989</span><span class="mixed-article-title" title="mixed-article-title">Regression and time series model selection in small samples</span>. <span class="source" tagx="source" title="source">Biometrika</span><span class="volume" tagx="volume" title="volume">76</span> (<span class="issue" tagx="issue" title="issue">2</span>): <span class="fpage" tagx="fpage" title="fpage">297</span>–<span class="lpage" tagx="lpage" title="lpage">307</span>.</span></li> <li tag="ref"><a name="msy028-B52" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ibarra-Soria</span><span class="given-names" tagx="given-names" title="given-names">X</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Levitin</span><span class="given-names" tagx="given-names" title="given-names">MO</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Saraiva</span><span class="given-names" tagx="given-names" title="given-names">LR</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Logan</span><span class="given-names" tagx="given-names" title="given-names">DW.</span></span></span><span class="year" tagx="year" title="year">2014</span><span class="mixed-article-title" title="mixed-article-title">The olfactory transcriptome of mice</span>. <span class="source" tagx="source" title="source">PLoS Genet.</span><span class="volume" tagx="volume" title="volume">10</span>(<span class="issue" tagx="issue" title="issue">9</span>): <span class="fpage" tagx="fpage" title="fpage">e1004593.</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25187969">25187969</a></span></span></li> <li tag="ref"><a name="msy028-B56" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kamilar</span><span class="given-names" tagx="given-names" title="given-names">JM</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Cooper</span><span class="given-names" tagx="given-names" title="given-names">N.</span></span></span><span class="year" tagx="year" title="year">2013</span><span class="mixed-article-title" title="mixed-article-title">Phylogenetic signal in primate behaviour, ecology and life history</span>. <span class="source" tagx="source" title="source">Philos Trans R Soc Lond B Biol Sci.</span><span class="volume" tagx="volume" title="volume">368</span>(<span class="issue" tagx="issue" title="issue">1618</span>): <span class="fpage" tagx="fpage" title="fpage">20120341.</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23569289">23569289</a></span></span></li> <li tag="ref"><a name="msy028-B57" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Keller</span><span class="given-names" tagx="given-names" title="given-names">A</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Vosshall</span><span class="given-names" tagx="given-names" title="given-names">LB.</span></span></span><span class="year" tagx="year" title="year">2008</span><span class="mixed-article-title" title="mixed-article-title">Better smelling through genetics: mammalian odor perception</span>. <span class="source" tagx="source" title="source">Curr Opin Neurobiol</span>, <span class="volume" tagx="volume" title="volume">18</span> (<span class="issue" tagx="issue" title="issue">4</span>): <span class="fpage" tagx="fpage" title="fpage">364</span>–<span class="lpage" tagx="lpage" title="lpage">369</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18938244">18938244</a></span></span></li> <li tag="ref"><a name="msy028-B59" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Khan</span><span class="given-names" tagx="given-names" title="given-names">I</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Yang</span><span class="given-names" tagx="given-names" title="given-names">Z</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Maldonado</span><span class="given-names" tagx="given-names" title="given-names">E</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Li</span><span class="given-names" tagx="given-names" title="given-names">C</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zhang</span><span class="given-names" tagx="given-names" title="given-names">G</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gilbert</span><span class="given-names" tagx="given-names" title="given-names">MTP</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Jarvis</span><span class="given-names" tagx="given-names" title="given-names">ED</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">O’Brien</span><span class="given-names" tagx="given-names" title="given-names">SJ</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Johnson</span><span class="given-names" tagx="given-names" title="given-names">WE</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Antunes</span><span class="given-names" tagx="given-names" title="given-names">A.</span></span></span><span class="year" tagx="year" title="year">2015</span><span class="mixed-article-title" title="mixed-article-title">Olfactory receptor subgenomes linked with broad ecological adaptations in Sauropsida</span>. <span class="source" tagx="source" title="source">Mol Biol Evol.</span><span class="volume" tagx="volume" title="volume">32</span> (<span class="issue" tagx="issue" title="issue">11</span>): <span class="fpage" tagx="fpage" title="fpage">2832</span>–<span class="lpage" tagx="lpage" title="lpage">2843</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/26219582">26219582</a></span></span></li> <li tag="ref"><a name="msy028-B61" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kissling</span><span class="given-names" tagx="given-names" title="given-names">WD</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Dalby</span><span class="given-names" tagx="given-names" title="given-names">L</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Fløjgaard</span><span class="given-names" tagx="given-names" title="given-names">C</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lenoir</span><span class="given-names" tagx="given-names" title="given-names">J</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Sandel</span><span class="given-names" tagx="given-names" title="given-names">B</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Sandom</span><span class="given-names" tagx="given-names" title="given-names">C</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Trøjelsgaard</span><span class="given-names" tagx="given-names" title="given-names">K</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Svenning</span><span class="given-names" tagx="given-names" title="given-names">J-C.</span></span></span><span class="year" tagx="year" title="year">2014</span><span class="mixed-article-title" title="mixed-article-title">Establishing macroecological trait datasets: digitization, extrapolation, and validation of diet preferences in terrestrial mammals worldwide</span>. <span class="source" tagx="source" title="source">Ecol. Evol</span>. <span class="volume" tagx="volume" title="volume">4</span> (<span class="issue" tagx="issue" title="issue">14</span>): <span class="fpage" tagx="fpage" title="fpage">2913</span>–<span class="lpage" tagx="lpage" title="lpage">2930</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25165528">25165528</a></span></span></li> <li tag="ref"><a name="msy028-B62" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lanfear</span><span class="given-names" tagx="given-names" title="given-names">R</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Calcott</span><span class="given-names" tagx="given-names" title="given-names">B</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ho</span><span class="given-names" tagx="given-names" title="given-names">SYW</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Guindon</span><span class="given-names" tagx="given-names" title="given-names">S.</span></span></span><span class="year" tagx="year" title="year">2012</span><span class="mixed-article-title" title="mixed-article-title">PartitionFinder: combined selection of partitioning schemes and substitution models for phylogenetic analyses</span>. <span class="source" tagx="source" title="source">Mol Biol Evol.</span><span class="volume" tagx="volume" title="volume">29</span> (<span class="issue" tagx="issue" title="issue">6</span>): <span class="fpage" tagx="fpage" title="fpage">1695</span>–<span class="lpage" tagx="lpage" title="lpage">1701</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22319168">22319168</a></span></span></li> <li tag="ref"><a name="msy028-B63" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lee</span><span class="given-names" tagx="given-names" title="given-names">K</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Nguyen</span><span class="given-names" tagx="given-names" title="given-names">DT</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Choi</span><span class="given-names" tagx="given-names" title="given-names">M</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Cha</span><span class="given-names" tagx="given-names" title="given-names">S-Y</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kim</span><span class="given-names" tagx="given-names" title="given-names">J-H</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Dadi</span><span class="given-names" tagx="given-names" title="given-names">H</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Seo</span><span class="given-names" tagx="given-names" title="given-names">HG</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Seo</span><span class="given-names" tagx="given-names" title="given-names">K</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Chun</span><span class="given-names" tagx="given-names" title="given-names">T</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Park</span><span class="given-names" tagx="given-names" title="given-names">C.</span></span></span><span class="year" tagx="year" title="year">2013</span><span class="mixed-article-title" title="mixed-article-title">Analysis of cattle olfactory subgenome: the first detailed study on the characteristics of the complete olfactory receptor repertoire of a ruminant</span>. <span class="source" tagx="source" title="source">BMC Genomics</span><span class="volume" tagx="volume" title="volume">14</span>(<span class="issue" tagx="issue" title="issue">1</span>): <span class="fpage" tagx="fpage" title="fpage">596.</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24004971">24004971</a></span></span></li> <li tag="ref"><a name="msy028-B64" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Li</span><span class="given-names" tagx="given-names" title="given-names">R</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Fan</span><span class="given-names" tagx="given-names" title="given-names">W</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Tian</span><span class="given-names" tagx="given-names" title="given-names">G</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zhu</span><span class="given-names" tagx="given-names" title="given-names">H</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">He</span><span class="given-names" tagx="given-names" title="given-names">L</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Cai</span><span class="given-names" tagx="given-names" title="given-names">J</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Huang</span><span class="given-names" tagx="given-names" title="given-names">Q</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Cai</span><span class="given-names" tagx="given-names" title="given-names">Q</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Li</span><span class="given-names" tagx="given-names" title="given-names">B</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bai</span><span class="given-names" tagx="given-names" title="given-names">Y</span></span>, <span class="etal" title="etal"><i>et al.</i></span></span><span class="year" tagx="year" title="year">2009</span><span class="mixed-article-title" title="mixed-article-title">The sequence and de novo assembly of the giant panda</span>. <span class="source" tagx="source" title="source">Nature</span><span class="volume" tagx="volume" title="volume">463</span>:<span class="fpage" tagx="fpage" title="fpage">311</span>–<span class="lpage" tagx="lpage" title="lpage">317</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20010809">20010809</a></span></span></li> <li tag="ref"><a name="msy028-B67" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lu</span><span class="given-names" tagx="given-names" title="given-names">Q</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wang</span><span class="given-names" tagx="given-names" title="given-names">K</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lei</span><span class="given-names" tagx="given-names" title="given-names">F</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Yu</span><span class="given-names" tagx="given-names" title="given-names">D</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zhao</span><span class="given-names" tagx="given-names" title="given-names">H.</span></span></span><span class="year" tagx="year" title="year">2016</span><span class="mixed-article-title" title="mixed-article-title">Penguins reduced olfactory receptor genes common to other waterbirds</span>. <span class="source" tagx="source" title="source">Sci Rep.</span><span class="volume" tagx="volume" title="volume">6</span>:<span class="fpage" tagx="fpage" title="fpage">31671.</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/27527385">27527385</a></span></span></li> <li tag="ref"><a name="msy028-B68" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Luca</span><span class="given-names" tagx="given-names" title="given-names">F</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Perry</span><span class="given-names" tagx="given-names" title="given-names">GH</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Di Rienzo</span><span class="given-names" tagx="given-names" title="given-names">A.</span></span></span><span class="year" tagx="year" title="year">2010</span><span class="mixed-article-title" title="mixed-article-title">Evolutionary adaptations to dietary changes</span>. <span class="source" tagx="source" title="source">Annu Rev Nutr.</span><span class="volume" tagx="volume" title="volume">30</span>:<span class="fpage" tagx="fpage" title="fpage">291</span>–<span class="lpage" tagx="lpage" title="lpage">314</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20420525">20420525</a></span></span></li> <li tag="ref"><a name="msy028-B69" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Luo</span><span class="given-names" tagx="given-names" title="given-names">R</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Liu</span><span class="given-names" tagx="given-names" title="given-names">B</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Yinlong</span><span class="given-names" tagx="given-names" title="given-names">X</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Li</span><span class="given-names" tagx="given-names" title="given-names">Z</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Huang</span><span class="given-names" tagx="given-names" title="given-names">W</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Yuan</span><span class="given-names" tagx="given-names" title="given-names">J</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">He</span><span class="given-names" tagx="given-names" title="given-names">G</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Chen</span><span class="given-names" tagx="given-names" title="given-names">Y</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Pan</span><span class="given-names" tagx="given-names" title="given-names">Q</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Liu</span><span class="given-names" tagx="given-names" title="given-names">Y</span></span>, <span class="etal" title="etal"><i>et al.</i></span></span><span class="year" tagx="year" title="year">2012</span><span class="mixed-article-title" title="mixed-article-title">SOAPdenovo2: an empiricaly improved memory-efficient short-read de novo assembler</span>. <span class="source" tagx="source" title="source">GigaScience</span><span class="volume" tagx="volume" title="volume">1</span>(<span class="issue" tagx="issue" title="issue">1</span>):.</span></li> <li tag="ref"><a name="msy028-B70" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Luo</span><span class="given-names" tagx="given-names" title="given-names">Z-X.</span></span></span><span class="year" tagx="year" title="year">2007</span><span class="mixed-article-title" title="mixed-article-title">Transformation and diversification in early mammal evolution</span>. <span class="source" tagx="source" title="source">Nature</span><span class="volume" tagx="volume" title="volume">450</span>(<span class="issue" tagx="issue" title="issue">7172</span>): <span class="fpage" tagx="fpage" title="fpage">1011</span>–<span class="lpage" tagx="lpage" title="lpage">1019</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18075580">18075580</a></span></span></li> <li tag="ref"><a name="msy028-B74" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Meredith</span><span class="given-names" tagx="given-names" title="given-names">RW</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Janecka</span><span class="given-names" tagx="given-names" title="given-names">JE</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gatesy</span><span class="given-names" tagx="given-names" title="given-names">J</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ryder</span><span class="given-names" tagx="given-names" title="given-names">OA</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Fisher</span><span class="given-names" tagx="given-names" title="given-names">CA</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Teeling</span><span class="given-names" tagx="given-names" title="given-names">EC</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Goodbla</span><span class="given-names" tagx="given-names" title="given-names">A</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Eizirik</span><span class="given-names" tagx="given-names" title="given-names">E</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Simao</span><span class="given-names" tagx="given-names" title="given-names">TLL</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Stadler</span><span class="given-names" tagx="given-names" title="given-names">T</span></span>, <span class="etal" title="etal"><i>et al.</i></span></span><span class="year" tagx="year" title="year">2011</span><span class="mixed-article-title" title="mixed-article-title">Impacts of the cretaceous terrestrial revolution and KPg extinction on mammal diversification</span>. <span class="source" tagx="source" title="source">Science</span><span class="volume" tagx="volume" title="volume">334</span>(<span class="issue" tagx="issue" title="issue">6055</span>): <span class="fpage" tagx="fpage" title="fpage">521</span>–<span class="lpage" tagx="lpage" title="lpage">524</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21940861">21940861</a></span></span></li> <li tag="ref"><a name="msy028-B76" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Mitani</span><span class="given-names" tagx="given-names" title="given-names">JC.</span></span></span><span class="year" tagx="year" title="year">1989</span><span class="mixed-article-title" title="mixed-article-title">Orangutan activity budgets: monthly variations and the effects of body size, partruition and sociality</span>. <span class="source" tagx="source" title="source">Am J Primatol.</span><span class="volume" tagx="volume" title="volume">18</span>:<span class="fpage" tagx="fpage" title="fpage">87</span>–<span class="lpage" tagx="lpage" title="lpage">100</span>.</span></li> <li tag="ref"><a name="msy028-B77" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Montague</span><span class="given-names" tagx="given-names" title="given-names">MJ</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Li</span><span class="given-names" tagx="given-names" title="given-names">G</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gandolfi</span><span class="given-names" tagx="given-names" title="given-names">B</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Khan</span><span class="given-names" tagx="given-names" title="given-names">R</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Aken</span><span class="given-names" tagx="given-names" title="given-names">BL</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Searle</span><span class="given-names" tagx="given-names" title="given-names">SMJ</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Minx</span><span class="given-names" tagx="given-names" title="given-names">P</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hillier</span><span class="given-names" tagx="given-names" title="given-names">LW</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Koboldt</span><span class="given-names" tagx="given-names" title="given-names">DC</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Davis</span><span class="given-names" tagx="given-names" title="given-names">BW</span></span>, <span class="etal" title="etal"><i>et al.</i></span></span><span class="year" tagx="year" title="year">2014</span><span class="mixed-article-title" title="mixed-article-title">Comparative analysis of the domesticated cat genome reveals genetic signatures underlying feline biology and domestication</span>. <span class="source" tagx="source" title="source">Proc Natl Acad Sci U S A.</span><span class="volume" tagx="volume" title="volume">111</span> (<span class="issue" tagx="issue" title="issue">48</span>): <span class="fpage" tagx="fpage" title="fpage">17230</span>–<span class="lpage" tagx="lpage" title="lpage">17235</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25385592">25385592</a></span></span></li> <li tag="ref"><a name="msy028-B78" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Muchlinski</span><span class="given-names" tagx="given-names" title="given-names">MN</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Deane</span><span class="given-names" tagx="given-names" title="given-names">AS.</span></span></span><span class="year" tagx="year" title="year">2014</span><span class="mixed-article-title" title="mixed-article-title">The interpretive power of infraorbital foramen area in making dietary inferences in extant apes</span>. <span class="source" tagx="source" title="source">Anat Rec.</span><span class="volume" tagx="volume" title="volume">297</span>(<span class="issue" tagx="issue" title="issue">8</span>): <span class="fpage" tagx="fpage" title="fpage">1377</span>–<span class="lpage" tagx="lpage" title="lpage">1384</span>.</span></li> <li tag="ref"><a name="msy028-B80" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Nei</span><span class="given-names" tagx="given-names" title="given-names">M</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Rooney</span><span class="given-names" tagx="given-names" title="given-names">AP.</span></span></span><span class="year" tagx="year" title="year">2005</span><span class="mixed-article-title" title="mixed-article-title">Concerted and birth-and-death evolution of multigene families</span>. <span class="source" tagx="source" title="source">Annu Rev Genet.</span><span class="volume" tagx="volume" title="volume">39</span>:<span class="fpage" tagx="fpage" title="fpage">121</span>–<span class="lpage" tagx="lpage" title="lpage">152</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16285855">16285855</a></span></span></li> <li tag="ref"><a name="msy028-B81" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Niimura</span><span class="given-names" tagx="given-names" title="given-names">Y</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Nei</span><span class="given-names" tagx="given-names" title="given-names">M.</span></span></span><span class="year" tagx="year" title="year">2007</span><span class="mixed-article-title" title="mixed-article-title">Extensive gains and losses of olfactory receptor genes in mammalian evolution</span>. <span class="source" tagx="source" title="source">PLoS One</span><span class="volume" tagx="volume" title="volume">2</span> (<span class="issue" tagx="issue" title="issue">8</span>): <span class="fpage" tagx="fpage" title="fpage">e708</span>–<span class="lpage" tagx="lpage" title="lpage">e710</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17684554">17684554</a></span></span></li> <li tag="ref"><a name="msy028-B82" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Niimura</span><span class="given-names" tagx="given-names" title="given-names">Y</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Matsui</span><span class="given-names" tagx="given-names" title="given-names">A</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Touhara</span><span class="given-names" tagx="given-names" title="given-names">K.</span></span></span><span class="year" tagx="year" title="year">2014</span><span class="mixed-article-title" title="mixed-article-title">Extreme expansion of the olfactory receptor gene repertoire in African elephants and evolutionary dynamics of orthologous gene groups in 13 placental mammals</span>. <span class="source" tagx="source" title="source">Genome Res.</span><span class="volume" tagx="volume" title="volume">24</span>(<span class="issue" tagx="issue" title="issue">9</span>): <span class="fpage" tagx="fpage" title="fpage">1485</span>–<span class="lpage" tagx="lpage" title="lpage">1496</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25053675">25053675</a></span></span></li> <li tag="ref"><a name="msy028-B85" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Olender</span><span class="given-names" tagx="given-names" title="given-names">T</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Fuchs</span><span class="given-names" tagx="given-names" title="given-names">T</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Linhart</span><span class="given-names" tagx="given-names" title="given-names">C</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Shamir</span><span class="given-names" tagx="given-names" title="given-names">R</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Adams</span><span class="given-names" tagx="given-names" title="given-names">M</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kalush</span><span class="given-names" tagx="given-names" title="given-names">F</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Khen</span><span class="given-names" tagx="given-names" title="given-names">M</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lancet</span><span class="given-names" tagx="given-names" title="given-names">D.</span></span></span><span class="year" tagx="year" title="year">2004</span><span class="mixed-article-title" title="mixed-article-title">The canine olfactory subgenome</span>. <span class="source" tagx="source" title="source">Genomics</span><span class="volume" tagx="volume" title="volume">83</span> (<span class="issue" tagx="issue" title="issue">3</span>): <span class="fpage" tagx="fpage" title="fpage">361</span>–<span class="lpage" tagx="lpage" title="lpage">372</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/14962662">14962662</a></span></span></li> <li tag="ref"><a name="msy028-B89" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Pagel</span><span class="given-names" tagx="given-names" title="given-names">M.</span></span></span><span class="year" tagx="year" title="year">1999</span><span class="mixed-article-title" title="mixed-article-title">Inferring the historical patterns of biological evolution</span>. <span class="source" tagx="source" title="source">Nature</span><span class="volume" tagx="volume" title="volume">401</span>(<span class="issue" tagx="issue" title="issue">6756</span>): <span class="fpage" tagx="fpage" title="fpage">877</span>–<span class="lpage" tagx="lpage" title="lpage">884</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/10553904">10553904</a></span></span></li> <li tag="ref"><a name="msy028-B92" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Paradis</span><span class="given-names" tagx="given-names" title="given-names">E</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Claude</span><span class="given-names" tagx="given-names" title="given-names">J</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Strimmer</span><span class="given-names" tagx="given-names" title="given-names">K.</span></span></span><span class="year" tagx="year" title="year">2004</span><span class="mixed-article-title" title="mixed-article-title">APE: analysis of phylogenetics and evolution in R language</span>. <span class="source" tagx="source" title="source">Bioinformatics</span><span class="volume" tagx="volume" title="volume">20</span>(<span class="issue" tagx="issue" title="issue">2</span>): <span class="fpage" tagx="fpage" title="fpage">289</span>–<span class="lpage" tagx="lpage" title="lpage">290</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/14734327">14734327</a></span></span></li> <li tag="ref"><a name="msy028-B93" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Pearson</span><span class="given-names" tagx="given-names" title="given-names">WR</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wood</span><span class="given-names" tagx="given-names" title="given-names">T</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zhang</span><span class="given-names" tagx="given-names" title="given-names">Z</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Miller</span><span class="given-names" tagx="given-names" title="given-names">W.</span></span></span><span class="year" tagx="year" title="year">1997</span><span class="mixed-article-title" title="mixed-article-title">Comparison of DNA sequences with protein sequences</span>. <span class="source" tagx="source" title="source">Genomics</span><span class="volume" tagx="volume" title="volume">46</span> (<span class="issue" tagx="issue" title="issue">1</span>): <span class="fpage" tagx="fpage" title="fpage">24</span>–<span class="lpage" tagx="lpage" title="lpage">36</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/9403055">9403055</a></span></span></li> <li tag="ref"><a name="msy028-B94" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Peterson</span><span class="given-names" tagx="given-names" title="given-names">AT</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Soberon</span><span class="given-names" tagx="given-names" title="given-names">J</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Sanchez-Cordero</span><span class="given-names" tagx="given-names" title="given-names">V.</span></span></span><span class="year" tagx="year" title="year">1999</span><span class="mixed-article-title" title="mixed-article-title">Conservation of ecological niches in evolutionary time</span>. <span class="source" tagx="source" title="source">Science</span><span class="volume" tagx="volume" title="volume">285</span>:<span class="fpage" tagx="fpage" title="fpage">1265</span>–<span class="lpage" tagx="lpage" title="lpage">1267</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/10455053">10455053</a></span></span></li> <li tag="ref"><a name="msy028-B95" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Pinheiro</span><span class="given-names" tagx="given-names" title="given-names">J</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bates</span><span class="given-names" tagx="given-names" title="given-names">D</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">DebRoy</span><span class="given-names" tagx="given-names" title="given-names">S</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Sarkar</span><span class="given-names" tagx="given-names" title="given-names">D.</span></span></span><span class="collab'">collab: R Core Team</span>. <span class="year" tagx="year" title="year">2015</span><span class="source" tagx="source" title="source">Nlme: linear and nonlinear mixed effects models</span> [Internet]. R package version 3.1–128. Available from: <a href="https://cran.r-project.org/web/packages/nlme/nlme.pdf">https://cran.r-project.org/web/packages/nlme/nlme.pdf</a>.</span></li> <li tag="ref"><a name="msy028-B96" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Price</span><span class="given-names" tagx="given-names" title="given-names">SA</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hopkins</span><span class="given-names" tagx="given-names" title="given-names">SSB</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Smith</span><span class="given-names" tagx="given-names" title="given-names">KK</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Roth</span><span class="given-names" tagx="given-names" title="given-names">VL.</span></span></span><span class="year" tagx="year" title="year">2012</span><span class="mixed-article-title" title="mixed-article-title">Tempo of trophic evolution and its impact on mammalian diversification</span>. <span class="source" tagx="source" title="source">Proc Natl Acad Sci U S A</span>. <span class="volume" tagx="volume" title="volume">109</span> (<span class="issue" tagx="issue" title="issue">18</span>): <span class="fpage" tagx="fpage" title="fpage">7008</span>–<span class="lpage" tagx="lpage" title="lpage">7012</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22509033">22509033</a></span></span></li> <li tag="ref"><a name="msy028-B100" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Revell</span><span class="given-names" tagx="given-names" title="given-names">LJ.</span></span></span><span class="year" tagx="year" title="year">2012</span><span class="mixed-article-title" title="mixed-article-title">Phytools: an R package for phylogenetic comparative biology (and other things)</span>. <span class="source" tagx="source" title="source">Methods Ecol Evol.</span><span class="volume" tagx="volume" title="volume">3</span> (<span class="issue" tagx="issue" title="issue">2</span>): <span class="fpage" tagx="fpage" title="fpage">217</span>–<span class="lpage" tagx="lpage" title="lpage">223</span>.</span></li> <li tag="ref"><a name="msy028-B101" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Rizvanovic</span><span class="given-names" tagx="given-names" title="given-names">A</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Amundin</span><span class="given-names" tagx="given-names" title="given-names">M</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Laska</span><span class="given-names" tagx="given-names" title="given-names">M.</span></span></span><span class="year" tagx="year" title="year">2013</span><span class="mixed-article-title" title="mixed-article-title">Olfactory discrimination ability of Asian elephants (<i>Elephas maximus</i>) for structurally related odorants</span>. <span class="source" tagx="source" title="source">Chem Senses</span><span class="volume" tagx="volume" title="volume">38</span>(<span class="issue" tagx="issue" title="issue">2</span>): <span class="fpage" tagx="fpage" title="fpage">107</span>–<span class="lpage" tagx="lpage" title="lpage">118</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23246848">23246848</a></span></span></li> <li tag="ref"><a name="msy028-B103" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ruedi</span><span class="given-names" tagx="given-names" title="given-names">M</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Stadelmann</span><span class="given-names" tagx="given-names" title="given-names">B</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gager</span><span class="given-names" tagx="given-names" title="given-names">Y</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Douzery</span><span class="given-names" tagx="given-names" title="given-names">EJP</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Francis</span><span class="given-names" tagx="given-names" title="given-names">CM</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lin</span><span class="given-names" tagx="given-names" title="given-names">L-K</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Guillén-Servent</span><span class="given-names" tagx="given-names" title="given-names">A</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Cibois</span><span class="given-names" tagx="given-names" title="given-names">A.</span></span></span><span class="year" tagx="year" title="year">2013</span><span class="mixed-article-title" title="mixed-article-title">Molecular phylogenetic reconstructions identify east Asia as the cradle for the evolution of the cosmopolitan genus myotis (Mammalia, Chiroptera)</span>. <span class="source" tagx="source" title="source">Mol Phylogenet Evol.</span><span class="volume" tagx="volume" title="volume">69</span>(<span class="issue" tagx="issue" title="issue">3</span>): <span class="fpage" tagx="fpage" title="fpage">437</span>–<span class="lpage" tagx="lpage" title="lpage">449</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23988307">23988307</a></span></span></li> <li tag="ref"><a name="msy028-B106" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Schmitz</span><span class="given-names" tagx="given-names" title="given-names">L</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Motani</span><span class="given-names" tagx="given-names" title="given-names">R.</span></span></span><span class="year" tagx="year" title="year">2010</span><span class="mixed-article-title" title="mixed-article-title">Morphological differences between the eyeballs of nocturnal and diurnal amniotes revisited from optical perspectives of visual environments</span>. <span class="source" tagx="source" title="source">Vision Res.</span><span class="volume" tagx="volume" title="volume">50</span>(<span class="issue" tagx="issue" title="issue">10</span>): <span class="fpage" tagx="fpage" title="fpage">936</span>–<span class="lpage" tagx="lpage" title="lpage">946</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20304001">20304001</a></span></span></li> <li tag="ref"><a name="msy028-B107" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Schondube</span><span class="given-names" tagx="given-names" title="given-names">JE</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Herrera–M</span><span class="given-names" tagx="given-names" title="given-names">LG</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Del Rio</span><span class="given-names" tagx="given-names" title="given-names">C.</span></span></span><span class="year" tagx="year" title="year">2001</span><span class="mixed-article-title" title="mixed-article-title">Diet and the evolution of digestion and renal function in phyllostomid bats</span>. <span class="source" tagx="source" title="source">Zoology</span><span class="volume" tagx="volume" title="volume">104</span>(<span class="issue" tagx="issue" title="issue">1</span>): <span class="fpage" tagx="fpage" title="fpage">59</span>–<span class="lpage" tagx="lpage" title="lpage">73</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16351819">16351819</a></span></span></li> <li tag="ref"><a name="msy028-B111" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Sievers</span><span class="given-names" tagx="given-names" title="given-names">F</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wilm</span><span class="given-names" tagx="given-names" title="given-names">A</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Dineen</span><span class="given-names" tagx="given-names" title="given-names">D</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gibson</span><span class="given-names" tagx="given-names" title="given-names">TJ</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Karplus</span><span class="given-names" tagx="given-names" title="given-names">K</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Li</span><span class="given-names" tagx="given-names" title="given-names">W</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lopez</span><span class="given-names" tagx="given-names" title="given-names">R</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">McWilliam</span><span class="given-names" tagx="given-names" title="given-names">H</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Remmert</span><span class="given-names" tagx="given-names" title="given-names">M</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Soding</span><span class="given-names" tagx="given-names" title="given-names">J</span></span>, <span class="etal" title="etal"><i>et al.</i></span></span><span class="year" tagx="year" title="year">2011</span><span class="mixed-article-title" title="mixed-article-title">Fast, scalable generation of high-quality protein multiple sequence alignments using clustal omega</span>. <span class="source" tagx="source" title="source">Mol Syst Biol.</span><span class="volume" tagx="volume" title="volume">7</span>:<span class="fpage" tagx="fpage" title="fpage">539.</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21988835">21988835</a></span></span></li> <li tag="ref"><a name="msy028-B112" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Silk</span><span class="given-names" tagx="given-names" title="given-names">JB.</span></span></span><span class="year" tagx="year" title="year">2007</span><span class="mixed-article-title" title="mixed-article-title">The adaptive value of sociality in mammalian groups</span>. <span class="source" tagx="source" title="source">Philos Trans R Soc Lond B Biol Sci.</span><span class="volume" tagx="volume" title="volume">362</span>(<span class="issue" tagx="issue" title="issue">1480</span>): <span class="fpage" tagx="fpage" title="fpage">539</span>–<span class="lpage" tagx="lpage" title="lpage">559</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17363359">17363359</a></span></span></li> <li tag="ref"><a name="msy028-B113" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Simpson</span><span class="given-names" tagx="given-names" title="given-names">JT</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wong</span><span class="given-names" tagx="given-names" title="given-names">K</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Jackman</span><span class="given-names" tagx="given-names" title="given-names">SD</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Schein</span><span class="given-names" tagx="given-names" title="given-names">JE</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Jones</span><span class="given-names" tagx="given-names" title="given-names">SJM</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Birol</span><span class="given-names" tagx="given-names" title="given-names">I.</span></span></span><span class="year" tagx="year" title="year">2009</span><span class="mixed-article-title" title="mixed-article-title">ABySS: a parallel assembler for short read sequence data</span>. <span class="source" tagx="source" title="source">Genome Res.</span><span class="volume" tagx="volume" title="volume">19</span> (<span class="issue" tagx="issue" title="issue">6</span>): <span class="fpage" tagx="fpage" title="fpage">1117</span>–<span class="lpage" tagx="lpage" title="lpage">1123</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19251739">19251739</a></span></span></li> <li tag="ref"><a name="msy028-B118" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Springer</span><span class="given-names" tagx="given-names" title="given-names">MS</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gatesy</span><span class="given-names" tagx="given-names" title="given-names">J.</span></span></span><span class="year" tagx="year" title="year">2017</span><span class="mixed-article-title" title="mixed-article-title">Inactivation of the olfactory marker protein (OMP) gene in river dolphins and other odontocete cetaceans</span>. <span class="source" tagx="source" title="source">Mol Phylogenet Evol.</span><span class="volume" tagx="volume" title="volume">109</span>:<span class="fpage" tagx="fpage" title="fpage">375</span>–<span class="lpage" tagx="lpage" title="lpage">387</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/28193458">28193458</a></span></span></li> <li tag="ref"><a name="msy028-B119" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Stamatakis</span><span class="given-names" tagx="given-names" title="given-names">A</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Abere</span><span class="given-names" tagx="given-names" title="given-names">AJ</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Goll</span><span class="given-names" tagx="given-names" title="given-names">C</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Smith</span><span class="given-names" tagx="given-names" title="given-names">SA</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Berger</span><span class="given-names" tagx="given-names" title="given-names">SA</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Izquierdo-Carrasco</span><span class="given-names" tagx="given-names" title="given-names">F.</span></span></span><span class="year" tagx="year" title="year">2012</span><span class="mixed-article-title" title="mixed-article-title">RaxML-Light: a tool for computing terabyte phylogenies</span>. <span class="source" tagx="source" title="source">Bioinformatics</span><span class="volume" tagx="volume" title="volume">28</span> (<span class="issue" tagx="issue" title="issue">15</span>): <span class="fpage" tagx="fpage" title="fpage">2064</span>–<span class="lpage" tagx="lpage" title="lpage">2066</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22628519">22628519</a></span></span></li> <li tag="ref"><a name="msy028-B124" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Vartia</span><span class="given-names" tagx="given-names" title="given-names">S</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Villanueva-Cañas</span><span class="given-names" tagx="given-names" title="given-names">JL</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Finarelli</span><span class="given-names" tagx="given-names" title="given-names">J</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Farrell</span><span class="given-names" tagx="given-names" title="given-names">ED</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Collins</span><span class="given-names" tagx="given-names" title="given-names">PC</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hughes</span><span class="given-names" tagx="given-names" title="given-names">GM</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Carlsson</span><span class="given-names" tagx="given-names" title="given-names">JEL</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gauthier</span><span class="given-names" tagx="given-names" title="given-names">DT</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">McGinnity</span><span class="given-names" tagx="given-names" title="given-names">P</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Cross</span><span class="given-names" tagx="given-names" title="given-names">TF</span></span>, <span class="etal" title="etal"><i>et al.</i></span></span><span class="year" tagx="year" title="year">2016</span><span class="mixed-article-title" title="mixed-article-title">A novel method of microsatellite genotyping-by-sequencing using individual combinatorial barcoding</span>. <span class="source" tagx="source" title="source">R Soc Open Sci.</span><span class="volume" tagx="volume" title="volume">3</span>(<span class="issue" tagx="issue" title="issue">1</span>): <span class="fpage" tagx="fpage" title="fpage">150565.</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/26909185">26909185</a></span></span></li> <li tag="ref"><a name="msy028-B129" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wittemyer</span><span class="given-names" tagx="given-names" title="given-names">G</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Douglas-Hamilton</span><span class="given-names" tagx="given-names" title="given-names">I</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Getz</span><span class="given-names" tagx="given-names" title="given-names">WM.</span></span></span><span class="year" tagx="year" title="year">2005</span><span class="mixed-article-title" title="mixed-article-title">The socioecology of elephants: analysis of the processes creating multitiered social structures</span>. <span class="source" tagx="source" title="source">Anim Behav.</span><span class="volume" tagx="volume" title="volume">69</span> (<span class="issue" tagx="issue" title="issue">6</span>): <span class="fpage" tagx="fpage" title="fpage">1357</span>–<span class="lpage" tagx="lpage" title="lpage">1371</span>.</span></li> <li tag="ref"><a name="msy028-B131" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Young</span><span class="given-names" tagx="given-names" title="given-names">JM</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Massa</span><span class="given-names" tagx="given-names" title="given-names">HF</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hsu</span><span class="given-names" tagx="given-names" title="given-names">L</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Trask</span><span class="given-names" tagx="given-names" title="given-names">BJ.</span></span></span><span class="year" tagx="year" title="year">2010</span><span class="mixed-article-title" title="mixed-article-title">Extreme variability among mammalian V1R gene families</span>. <span class="source" tagx="source" title="source">Genome Res.</span><span class="volume" tagx="volume" title="volume">20</span> (<span class="issue" tagx="issue" title="issue">1</span>): <span class="fpage" tagx="fpage" title="fpage">10</span>–<span class="lpage" tagx="lpage" title="lpage">18</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19952141">19952141</a></span></span></li> <li tag="ref"><a name="msy028-B133" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zhang</span><span class="given-names" tagx="given-names" title="given-names">J</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Webb</span><span class="given-names" tagx="given-names" title="given-names">DM.</span></span></span><span class="year" tagx="year" title="year">2003</span><span class="mixed-article-title" title="mixed-article-title">Evolutionary deterioration of the vomeronasal pheromone signalling transduction pathway in catarrhine primates</span>. <span class="source" tagx="source" title="source">Proc Natl Acad Sci U S A</span>. <span class="volume" tagx="volume" title="volume">100</span> (<span class="issue" tagx="issue" title="issue">14</span>): <span class="fpage" tagx="fpage" title="fpage">8337</span>–<span class="lpage" tagx="lpage" title="lpage">8341</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12826614">12826614</a></span></span></li> <li tag="ref"><a name="msy028-B135" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zhao</span><span class="given-names" tagx="given-names" title="given-names">H</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Rossiter</span><span class="given-names" tagx="given-names" title="given-names">SJ</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Teeling</span><span class="given-names" tagx="given-names" title="given-names">EC</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Li</span><span class="given-names" tagx="given-names" title="given-names">C</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Cotton</span><span class="given-names" tagx="given-names" title="given-names">JA</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zhang</span><span class="given-names" tagx="given-names" title="given-names">S.</span></span></span><span class="year" tagx="year" title="year">2009</span><span class="mixed-article-title" title="mixed-article-title">The evolution of color vision in nocturnal mammals</span>. <span class="source" tagx="source" title="source">Proc Natl Acad Sci U S A</span>. <span class="volume" tagx="volume" title="volume">106</span> (<span class="issue" tagx="issue" title="issue">22</span>): <span class="fpage" tagx="fpage" title="fpage">8980</span>–<span class="lpage" tagx="lpage" title="lpage">8985</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19470491">19470491</a></span></span></li> <li tag="ref"><a name="msy028-B136" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zhao</span><span class="given-names" tagx="given-names" title="given-names">H</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Xu</span><span class="given-names" tagx="given-names" title="given-names">D</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zhang</span><span class="given-names" tagx="given-names" title="given-names">S</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zhang</span><span class="given-names" tagx="given-names" title="given-names">J.</span></span></span><span class="year" tagx="year" title="year">2011</span><span class="mixed-article-title" title="mixed-article-title">Widespread losses of vomeronasal signal transduction in bats</span>. <span class="source" tagx="source" title="source">Mol Biol Evol.</span><span class="volume" tagx="volume" title="volume">28</span>(<span class="issue" tagx="issue" title="issue">1</span>): <span class="fpage" tagx="fpage" title="fpage">7</span>–<span class="lpage" tagx="lpage" title="lpage">12</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20693241">20693241</a></span></span></li> <li tag="ref"><a name="msy028-B137" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zimin</span><span class="given-names" tagx="given-names" title="given-names">AV</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Delcher</span><span class="given-names" tagx="given-names" title="given-names">AL</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Florea</span><span class="given-names" tagx="given-names" title="given-names">L</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kelley</span><span class="given-names" tagx="given-names" title="given-names">DR</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Schatz</span><span class="given-names" tagx="given-names" title="given-names">MC</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Puiu</span><span class="given-names" tagx="given-names" title="given-names">D</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hanrahan</span><span class="given-names" tagx="given-names" title="given-names">F</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Pertea</span><span class="given-names" tagx="given-names" title="given-names">G</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Van Tassell</span><span class="given-names" tagx="given-names" title="given-names">CP</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Sonstegard</span><span class="given-names" tagx="given-names" title="given-names">TS</span></span>, <span class="etal" title="etal"><i>et al.</i></span></span><span class="year" tagx="year" title="year">2009</span><span class="mixed-article-title" title="mixed-article-title">A whole-genome assembly of the domestic cow, <i>Bos taurus</i></span>. <span class="source" tagx="source" title="source">Genome Biol.</span><span class="volume" tagx="volume" title="volume">10</span>(<span class="issue" tagx="issue" title="issue">4</span>): <span class="fpage" tagx="fpage" title="fpage">R42</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19393038">19393038</a></span></span></li> </ul> </div> </div>  </body></html>