<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<link rel="dns-prefetch" href="//d33xdlntwy0kbs.cloudfront.net" />
<link rel="shortcut icon" href="https://www.biorxiv.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<meta name="type" content="article" />
<meta name="category" content="article" />
<meta name="HW.identifier" content="/biorxiv/early/2020/01/30/2020.01.29.925396.atom" />
<meta name="HW.pisa" content="biorxiv;2020.01.29.925396v1" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="Global change adaptation limits future resilience" />
<meta name="DC.Identifier" content="10.1101/2020.01.29.925396" />
<meta name="DC.Date" content="2020-01-30" />
<meta name="DC.Publisher" content="Cold Spring Harbor Laboratory" />
<meta name="DC.Rights" content="© 2020, Posted by Cold Spring Harbor Laboratory. This pre-print is available under a Creative Commons License (Attribution-NonCommercial-NoDerivs 4.0 International), CC BY-NC-ND 4.0, as described at http://creativecommons.org/licenses/by-nc-nd/4.0/" />
<meta name="DC.AccessRights" content="restricted" />
<meta name="DC.Description" content="Global conditions are changing at unprecedented rates ([1][1]), challenging species resilience ([2][2]). Populations can respond to these changes through genetic adaptation and physiological plasticity ([3][3], [4][4]), and it is well accepted that these processes interact ([5][5]). However, the relative role plasticity and adaptation play in promoting resilience to global change and the potential impacts of rapid adaptation on plasticity during this process are not well understood ([3][3], [5][5]–[10][6]). Here, using experimental evolution and reciprocal transplantation of the marine coastal copepod, Acartia tonsa , we show rapid adaptation to global change conditions carries costs, notably a loss of physiological plasticity. Twenty generations of selection resulted in rapid, highly parallel adaptation to greenhouse conditions. However, this adaptation reduced plasticity and fitness, particularly when returned to ambient conditions and under food limitation. Due to the loss of plasticity, greenhouse adapted lines could no longer tolerate ambient conditions and compensated by additional adaptive evolution, a process that eroded nucleotide diversity in genes containing adaptive genetic variation for global change conditions. These results show that adaptation can negatively affect plasticity, limiting resilience to new stressors and previously benign environments. Our findings challenge the common assertion that the presence of adaptive genetic potential or physiological plasticity alone will enable population resilience under global change conditions.

Significance statement Understanding how species will respond to changing environmental conditions is an essential step in mitigating the potential negative consequences of human induced global change conditions. Typically, it is assumed that a species with sufficient plasticity or adaptive potential will be resilient in the face of these changes. Here, we experimentally evolve a copepod to simulated future acidic and warm conditions. We show that rapid adaptation reduced phenotypic plasticity and eroded the adaptive genetic variation necessary to tolerate additional environmental stress. These findings demonstrate that the presence of plasticity or adaptive potential do not necessarily indicate a resilient population, illustrating the importance of considering the interaction between these factors when predicting species tolerance to global change.

 [1]: #ref-1
 [2]: #ref-2
 [3]: #ref-3
 [4]: #ref-4
 [5]: #ref-5
 [6]: #ref-10" />
<meta name="DC.Contributor" content="Reid S. Brennan" />
<meta name="DC.Contributor" content="James A. deMayo" />
<meta name="DC.Contributor" content="Hans G. Dam" />
<meta name="DC.Contributor" content="Michael Finiguerra" />
<meta name="DC.Contributor" content="Hannes Baumann" />
<meta name="DC.Contributor" content="Melissa H. Pespeni" />
<meta name="article:published_time" content="2020-01-30" />
<meta name="article:section" content="New Results" />
<meta name="citation_title" content="Global change adaptation limits future resilience" />
<meta name="citation_abstract" lang="en" content="&lt;p&gt;Global conditions are changing at unprecedented rates (1), challenging species resilience (2). Populations can respond to these changes through genetic adaptation and physiological plasticity (3, 4), and it is well accepted that these processes interact (5). However, the relative role plasticity and adaptation play in promoting resilience to global change and the potential impacts of rapid adaptation on plasticity during this process are not well understood (3, 5-10). Here, using experimental evolution and reciprocal transplantation of the marine coastal copepod, Acartia tonsa, we show rapid adaptation to global change conditions carries costs, notably a loss of physiological plasticity. Twenty generations of selection resulted in rapid, highly parallel adaptation to greenhouse conditions. However, this adaptation reduced plasticity and fitness, particularly when returned to ambient conditions and under food limitation. Due to the loss of plasticity, greenhouse adapted lines could no longer tolerate ambient conditions and compensated by additional adaptive evolution, a process that eroded nucleotide diversity in genes containing adaptive genetic variation for global change conditions. These results show that adaptation can negatively affect plasticity, limiting resilience to new stressors and previously benign environments. Our findings challenge the common assertion that the presence of adaptive genetic potential or physiological plasticity alone will enable population resilience under global change conditions.&lt;/p&gt;" />
<meta name="citation_journal_title" content="bioRxiv" />
<meta name="citation_publisher" content="Cold Spring Harbor Laboratory" />
<meta name="citation_publication_date" content="2020/01/01" />
<meta name="citation_mjid" content="biorxiv;2020.01.29.925396v1" />
<meta name="citation_id" content="2020.01.29.925396v1" />
<meta name="citation_public_url" content="https://www.biorxiv.org/content/10.1101/2020.01.29.925396v1" />
<meta name="citation_abstract_html_url" content="https://www.biorxiv.org/content/10.1101/2020.01.29.925396v1.abstract" />
<meta name="citation_full_html_url" content="https://www.biorxiv.org/content/10.1101/2020.01.29.925396v1.full" />
<meta name="citation_pdf_url" content="https://www.biorxiv.org/content/biorxiv/early/2020/01/30/2020.01.29.925396.full.pdf" />
<meta name="citation_doi" content="10.1101/2020.01.29.925396" />
<meta name="citation_num_pages" content="28" />
<meta name="citation_article_type" content="Article" />
<meta name="citation_section" content="New Results" />
<meta name="citation_firstpage" content="2020.01.29.925396" />
<meta name="citation_author" content="Reid S. Brennan" />
<meta name="citation_author_institution" content="Department of Biology, University of Vermont" />
<meta name="citation_author_email" content="Reid.Brennan@uvm.edu" />
<meta name="citation_author_email" content="mpespeni@uvm.edu" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0001-7678-564X" />
<meta name="citation_author" content="James A. deMayo" />
<meta name="citation_author_institution" content="Department of Marine Sciences, University of Connecticut" />
<meta name="citation_author" content="Hans G. Dam" />
<meta name="citation_author_institution" content="Department of Marine Sciences, University of Connecticut" />
<meta name="citation_author" content="Michael Finiguerra" />
<meta name="citation_author_institution" content="Department of Ecology and Evolutionary Biology, University of Connecticut" />
<meta name="citation_author" content="Hannes Baumann" />
<meta name="citation_author_institution" content="Department of Marine Sciences, University of Connecticut" />
<meta name="citation_author" content="Melissa H. Pespeni" />
<meta name="citation_author_institution" content="Department of Biology, University of Vermont" />
<meta name="citation_author_email" content="Reid.Brennan@uvm.edu" />
<meta name="citation_author_email" content="mpespeni@uvm.edu" />
<meta name="citation_reference" content="N. S. Diffenbaugh, C. B. Field, Changes in ecologically critical terrestrial climate conditions. Science 341, 486–492 (2013)." />
<meta name="citation_reference" content="C. D. Thomas, et al., Extinction risk from climate change. Nature 427, 145–148 (2004)." />
<meta name="citation_reference" content="L.-M. Chevin, R. Lande, G. M. Mace, Adaptation, plasticity, and extinction in a changing environment: towards a predictive theory. PLoS Biol. 8, e1000357 (2010)." />
<meta name="citation_reference" content="J. Merilä, A. P. Hendry, Climate change, adaptation, and phenotypic plasticity: the problem and the evidence. Evol. Appl. 7, 1–14 (2014)." />
<meta name="citation_reference" content="M. J. West-Eberhard, Developmental Plasticity and Evolution (Oxford University Press, 2003)." />
<meta name="citation_reference" content="L. W. Ancel, Undermining the Baldwin expediting effect: does phenotypic plasticity accelerate evolution? Theor. Popul. Biol. 58, 307–319 (2000)." />
<meta name="citation_reference" content="T. D. Price, A. Qvarnström, D. E. Irwin, The role of phenotypic plasticity in driving genetic evolution. Proc. Biol. Sci. 270, 1433–1440 (2003)." />
<meta name="citation_reference" content="C. K. Ghalambor, J. K. McKAY, S. P. Carroll, D. N. Reznick, Adaptive versus non-adaptive phenotypic plasticity and the potential for contemporary adaptation in new environments. Funct. Ecol. 21, 394–407 (2007)." />
<meta name="citation_reference" content="C. K. Ghalambor, et al., Non-adaptive plasticity potentiates rapid adaptive evolution of gene expression in nature. Nature 525, 372–375 (2015)." />
<meta name="citation_reference" content="R. Lande, Adaptation to an extraordinary environment by evolution of phenotypic plasticity and genetic assimilation. J. Evol. Biol. 22, 1435–1446 (2009)." />
<meta name="citation_reference" content="S. Rahmstorf, D. Coumou, Increase of extreme events in a warming world. Proc. Natl. Acad. Sci. U. S. A. 108, 17905–17909 (2011)." />
<meta name="citation_reference" content="G. N. Somero, The physiology of climate change: how potentials for acclimatization and genetic adaptation will determine “winners” and “losers.” J. Exp. Biol. (2010)." />
<meta name="citation_reference" content="A. A. Hoffmann, C. M. Sgrò, Climate change and evolutionary adaptation. Nature 470, 479–485 (2011)." />
<meta name="citation_reference" content="S. C. Campbell-Staton, et al., Winter storms drive rapid phenotypic, regulatory, and genomic shifts in the green anole lizard. Science 357, 495–498 (2017)." />
<meta name="citation_reference" content="N. M. Reid, et al., The genomic landscape of rapid repeated evolutionary adaptation to toxic pollution in wild fish. Science 354, 1305–1308 (2016)." />
<meta name="citation_reference" content="R. D. H. Barrett, et al., Linking a mutation to survival in wild mice. Science 363, 499–504 (2019)." />
<meta name="citation_reference" content="H. C. Bumpus, The Elimination of the Unfit as Illustrated by the Introduced Sparrow, Passer Domesticus:(a Fourth Contribution to the Study of Variation) (Gin, 1899)." />
<meta name="citation_reference" content="N. O. Therkildsen, et al., Contrasting genomic shifts underlie parallel phenotypic evolution in response to fishing. Science 365, 487–490 (2019)." />
<meta name="citation_reference" content="R. S. Brennan, A. D. Garrett, K. E. Huber, H. Hargarten, M. H. Pespeni, Rare genetic variation and balanced polymorphisms are important for survival in global change conditions. Proceedings of the Royal Society B: Biological Sciences 286, 20190943 (2019)." />
<meta name="citation_reference" content="S. Via, R. Lande, Genotype-Environment Interaction and the Evolution of Phenotypic Plasticity. Evolution 39, 505–522 (1985)." />
<meta name="citation_reference" content="L.-M. Chevin, R. Lande, When do adaptive plasticity and genetic evolution prevent extinction of a density-regulated population? Evolution 64, 1143–1150 (2010)." />
<meta name="citation_reference" content="S. E. Williams, L. P. Shoo, J. L. Isaac, A. A. Hoffmann, G. Langham, Towards an integrated framework for assessing the vulnerability of species to climate change. PLoS Biol. 6, 2621–2626 (2008)." />
<meta name="citation_reference" content="B. Helmuth, et al., Long-term, high frequency in situ measurements of intertidal mussel bed temperatures using biomimetic sensors. Sci Data 3, 160087 (2016)." />
<meta name="citation_reference" content="R. A. Feely, C. L. Sabine, J. M. Hernandez-Ayon, D. Ianson, B. Hales, Evidence for upwelling of corrosive “acidified” water onto the continental shelf. Science 320, 1490–1492 (2008)." />
<meta name="citation_reference" content="A. G. Humes, How many copepods? in Ecology and Morphology of Copepods, (Springer Netherlands, 1994), pp. 1–7." />
<meta name="citation_reference" content="J. Mauchline, The Biology of Calanoid Copepods (Academic Press, 1998)." />
<meta name="citation_reference" content="J. A. F. Langer, et al., Acclimation and adaptation of the coastal calanoid copepod Acartia tonsa to ocean acidification: a long-term laboratory investigation. Mar. Ecol. Prog. Ser. 619, 35–51 (2019)." />
<meta name="citation_reference" content="M. W. Kelly, M. S. Pankey, M. B. DeBiasse, D. C. Plachetzki, Adaptation to heat stress reduces phenotypic and transcriptional plasticity in a marine copepod. Funct. Ecol. 31, 398–406 (2017)." />
<meta name="citation_reference" content="H. G. Dam, Evolutionary adaptation of marine zooplankton to global change. Ann. Rev. Mar. Sci. 5, 349–370 (2013)." />
<meta name="citation_reference" content="C. J. Gobler, H. Baumann, Hypoxia and acidification in ocean ecosystems: coupled dynamics and effects on marine life. Biol. Lett. 12 (2016)." />
<meta name="citation_reference" content="E. Rice, H. G. Dam, G. Stewart, Impact of Climate Change on Estuarine Zooplankton: Surface Water Warming in Long Island Sound Is Associated with Changes in Copepod Size and Community Structure. Estuaries Coasts 38, 13–23 (2015)." />
<meta name="citation_reference" content="citation_author=IPCC, “Climate Change 2014: Mitigation of Climate Change;citation_year=2014" />
<meta name="citation_reference" content="K. Caldeira, M. E. Wickett, Oceanography: anthropogenic carbon and ocean pH. Nature 425, 365 (2003)." />
<meta name="citation_reference" content="M. J. Behrenfeld, et al., Climate-driven trends in contemporary ocean productivity. Nature 444, 752–755 (2006)." />
<meta name="citation_reference" content="D. Zoccola, et al., Coral Carbonic Anhydrases: Regulation by Ocean Acidification. Mar. Drugs 14 (2016)." />
<meta name="citation_reference" content="S. P. Hergeth, R. Schneider, The H1 linker histones: multifunctional proteins beyond the nucleosomal core particle. EMBO Rep. 16, 1439–1453 (2015)." />
<meta name="citation_reference" content="M. F. Decristofaro, et al., Characterization of SWI/SNF protein expression in human breast cancer cell lines and other malignancies. J. Cell. Physiol. 186, 136–145 (2001)." />
<meta name="citation_reference" content="H. Caswell, Matrix population models. Encyclopedia of Environmetrics 3 (2006)." />
<meta name="citation_reference" content="A. Bailey, et al., Regulation of gene expression is associated with tolerance of the Arctic copepod Calanus glacialis to CO2-acidified sea water. Ecol. Evol. 7, 7145–7160 (2017)." />
<meta name="citation_reference" content="R. A. Bay, et al., Genomic signals of selection predict climate-driven population declines in a migratory bird. Science 359, 83–86 (2018)." />
<meta name="citation_reference" content="R. A. Bay, et al., Predicting Responses to Contemporary Environmental Change Using Evolutionary Response Architectures. Am. Nat. 189, 463–473 (2017)." />
<meta name="citation_reference" content="A. Bush, et al., Incorporating evolutionary adaptation in species distribution modelling reduces projected vulnerability to climate change. Ecol. Lett. 19, 1468–1478 (2016)." />
<meta name="citation_reference" content="A. Vehmaa, A. Brutemark, J. Engström-Öst, Maternal effects may act as an adaptation mechanism for copepods facing pH and temperature changes. PLoS One 7, e48538 (2012)." />
<meta name="citation_reference" content="L. R. Feinberg, H. G. Dam, Effects of diet on dimensions, density and sinking rates of fecal pellets of the copepod Acartia tonsa. Mar. Ecol. Prog. Ser. 175, 87–96 (1998)." />
<meta name="citation_reference" content="T. M. Therneau, P. M. Grambsch, Modeling Survival Data: Extending the Cox Model (Springer, 2013)." />
<meta name="citation_reference" content="T. Therneau, A Package for Survival Analysis in S. version 2.38 (2015)." />
<meta name="citation_reference" content="A. Kassambara, M. Kosinski, P. Biecek, survminer: Drawing Survival Curves using ggplot2. 1 (2017)." />
<meta name="citation_reference" content="S. E. L. Houde, M. R. Roman, Effects of food quality on the functional ingestion response of the copepod Acartia tonsa. Mar. Ecol. Prog. Ser. 40, 69–77 (1987)." />
<meta name="citation_reference" content="R Core Team, R: A Language and Environment for Statistical Computing (2019)." />
<meta name="citation_reference" content="A. M. Bolger, M. Lohse, B. Usadel, Trimmomatic: a flexible trimmer for Illumina sequence data. Bioinformatics 30, 2114–2120 (2014)." />
<meta name="citation_reference" content="T. S. Jørgensen, et al., The genome and mRNA transcriptome of the cosmopolitan calanoid copepod Acartia tonsa Dana improve the understanding of copepod genome size evolution. Genome Biol. Evol. (2019) https:/doi.org/10.1093/gbe/evz067." />
<meta name="citation_reference" content="B. J. Haas, et al., De novo transcript sequence reconstruction from RNA-seq using the Trinity platform for reference generation and analysis. Nat. Protoc. 8, 1494–1512 (2013)." />
<meta name="citation_reference" content="N. M. Davidson, A. D. K. Hawkins, A. Oshlack, SuperTranscripts: a data driven reference for analysis and visualisation of transcriptomes. Genome Biol. 18, 148 (2017)." />
<meta name="citation_reference" content="D. C. Koboldt, et al., VarScan 2: somatic mutation and copy number alteration discovery in cancer by exome sequencing. Genome Res. 22, 568–576 (2012)." />
<meta name="citation_reference" content="C. Soneson, M. I. Love, M. D. Robinson, Differential analyses for RNA-seq: transcriptlevel estimates improve gene-level inferences. F1000Res. 4 (2016)." />
<meta name="citation_reference" content="R. Patro, G. Duggal, M. I. Love, R. A. Irizarry, C. Kingsford, Salmon provides fast and bias-aware quantification of transcript expression. Nat. Methods 14, 417–419 (2017)." />
<meta name="citation_reference" content="M. I. Love, W. Huber, S. Anders, Moderated estimation of fold change and dispersion for RNA-seq data with DESeq2. Genome Biol. 15, 550 (2014)." />
<meta name="citation_reference" content="T. Jombart, I. Ahmed, adegenet 1.3-1: new tools for the analysis of genome-wide SNP data. Bioinformatics (2011) https:/doi.org/10.1093/bioinformatics/btr521." />
<meta name="citation_reference" content="J. D. Hadfield, MCMC Methods for Multi-Response Generalized Linear Mixed Models: The MCMCglmm R Package. Journal of Statistical Software 33, 1–22 (2010)." />
<meta name="citation_reference" content="C. Schlötterer, R. Kofler, E. Versace, R. Tobler, S. U. Franssen, Combining experimental evolution with next-generation sequencing: a powerful tool to study adaptation from standing genetic variation. Heredity 114, 431–440 (2015)." />
<meta name="citation_reference" content="P. Orozco-terWengel, et al., Adaptation of Drosophila to a novel laboratory environment reveals temporally heterogeneous trajectories of selected alleles. Mol. Ecol. 21, 4931–4941 (2012)." />
<meta name="citation_reference" content="R. Kofler, et al., PoPoolation: a toolbox for population genetic analysis of next generation sequencing data from pooled individuals. PLoS One 6, e15925 (2011)." />
<meta name="citation_reference" content="R. M. Wright, G. V. Aglyamova, E. Meyer, M. V. Matz, Gene expression associated with white syndromes in a reef building coral, Acropora hyacinthus. BMC Genomics 16, 371 (2015)." />
<meta name="twitter:title" content="Global change adaptation limits future resilience" />
<meta name="twitter:site" content="@biorxivpreprint" />
<meta name="twitter:card" content="summary" />
<meta name="twitter:image" content="https://www.biorxiv.org/sites/default/files/images/biorxiv_logo_homepage7-5-small.png" />
<meta name="twitter:description" content="Global conditions are changing at unprecedented rates ([1][1]), challenging species resilience ([2][2]). Populations can respond to these changes through genetic adaptation and physiological plasticity ([3][3], [4][4]), and it is well accepted that these processes interact ([5][5]). However, the relative role plasticity and adaptation play in promoting resilience to global change and the potential impacts of rapid adaptation on plasticity during this process are not well understood ([3][3], [5][5]–[10][6]). Here, using experimental evolution and reciprocal transplantation of the marine coastal copepod, Acartia tonsa , we show rapid adaptation to global change conditions carries costs, notably a loss of physiological plasticity. Twenty generations of selection resulted in rapid, highly parallel adaptation to greenhouse conditions. However, this adaptation reduced plasticity and fitness, particularly when returned to ambient conditions and under food limitation. Due to the loss of plasticity, greenhouse adapted lines could no longer tolerate ambient conditions and compensated by additional adaptive evolution, a process that eroded nucleotide diversity in genes containing adaptive genetic variation for global change conditions. These results show that adaptation can negatively affect plasticity, limiting resilience to new stressors and previously benign environments. Our findings challenge the common assertion that the presence of adaptive genetic potential or physiological plasticity alone will enable population resilience under global change conditions.

Significance statement Understanding how species will respond to changing environmental conditions is an essential step in mitigating the potential negative consequences of human induced global change conditions. Typically, it is assumed that a species with sufficient plasticity or adaptive potential will be resilient in the face of these changes. Here, we experimentally evolve a copepod to simulated future acidic and warm conditions. We show that rapid adaptation reduced phenotypic plasticity and eroded the adaptive genetic variation necessary to tolerate additional environmental stress. These findings demonstrate that the presence of plasticity or adaptive potential do not necessarily indicate a resilient population, illustrating the importance of considering the interaction between these factors when predicting species tolerance to global change.

 [1]: #ref-1
 [2]: #ref-2
 [3]: #ref-3
 [4]: #ref-4
 [5]: #ref-5
 [6]: #ref-10" />
<meta name="og-title" property="og:title" content="Global change adaptation limits future resilience" />
<meta name="og-url" property="og:url" content="https://www.biorxiv.org/content/10.1101/2020.01.29.925396v1" />
<meta name="og-site-name" property="og:site_name" content="bioRxiv" />
<meta name="og-description" property="og:description" content="Global conditions are changing at unprecedented rates ([1][1]), challenging species resilience ([2][2]). Populations can respond to these changes through genetic adaptation and physiological plasticity ([3][3], [4][4]), and it is well accepted that these processes interact ([5][5]). However, the relative role plasticity and adaptation play in promoting resilience to global change and the potential impacts of rapid adaptation on plasticity during this process are not well understood ([3][3], [5][5]–[10][6]). Here, using experimental evolution and reciprocal transplantation of the marine coastal copepod, Acartia tonsa , we show rapid adaptation to global change conditions carries costs, notably a loss of physiological plasticity. Twenty generations of selection resulted in rapid, highly parallel adaptation to greenhouse conditions. However, this adaptation reduced plasticity and fitness, particularly when returned to ambient conditions and under food limitation. Due to the loss of plasticity, greenhouse adapted lines could no longer tolerate ambient conditions and compensated by additional adaptive evolution, a process that eroded nucleotide diversity in genes containing adaptive genetic variation for global change conditions. These results show that adaptation can negatively affect plasticity, limiting resilience to new stressors and previously benign environments. Our findings challenge the common assertion that the presence of adaptive genetic potential or physiological plasticity alone will enable population resilience under global change conditions.

Significance statement Understanding how species will respond to changing environmental conditions is an essential step in mitigating the potential negative consequences of human induced global change conditions. Typically, it is assumed that a species with sufficient plasticity or adaptive potential will be resilient in the face of these changes. Here, we experimentally evolve a copepod to simulated future acidic and warm conditions. We show that rapid adaptation reduced phenotypic plasticity and eroded the adaptive genetic variation necessary to tolerate additional environmental stress. These findings demonstrate that the presence of plasticity or adaptive potential do not necessarily indicate a resilient population, illustrating the importance of considering the interaction between these factors when predicting species tolerance to global change.

 [1]: #ref-1
 [2]: #ref-2
 [3]: #ref-3
 [4]: #ref-4
 [5]: #ref-5
 [6]: #ref-10" />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.biorxiv.org/sites/default/files/images/biorxiv_logo_homepage7-5-small.png" />
<meta name="citation_date" content="2020-01-30" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/10.1101/2020.01.29.925396v1.full.pdf" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/10.1101/2020.01.29.925396v1.full.txt" />
<link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/10.1101/2020.01.29.925396v1.ppt" />
<meta name="description" content="bioRxiv - the preprint server for biology, operated by Cold Spring Harbor Laboratory, a research and educational institution" />
<meta name="generator" content="Drupal 7 (http://drupal.org)" />
<link rel="canonical" href="https://www.biorxiv.org/content/10.1101/2020.01.29.925396v1" />
<link rel="shortlink" href="https://www.biorxiv.org/node/1127478" />
    <title>Global change adaptation limits future resilience | bioRxiv</title>  
    <link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__jMRAK66KMC1e4TQlwUNn3KiWVDC5AjueUAYEm1xBY_U__KSUjdT4jdcJ4qJz6fKY1K9WkYh1a5EcaWZxt_-zbTis__QJacVmveFyLIBSPkp21xL0ZvuDW9DMBrD2HB1P4Ry5I.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q5p1rr" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__icrP-Y0qw2NP73rJxxkUtrJta0z3VLAABtNjfSdEghA__rlN3rx0wOoswGNHZMeZM96bNn-3cLWHXTztGDoThNlw__QJacVmveFyLIBSPkp21xL0ZvuDW9DMBrD2HB1P4Ry5I.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__q48SDxVDWut_cH6dTPyQe7HjADuSLLmJVFztuiJEFrU__DJqddQGjm5dXCHsaiwNriGytKN5eevIQqyjdq65TZtE__QJacVmveFyLIBSPkp21xL0ZvuDW9DMBrD2HB1P4Ry5I.css" media="all" />
<style type="text/css" media="all">
/* <![CDATA[ */
#sliding-popup.sliding-popup-bottom,#sliding-popup.sliding-popup-bottom .eu-cookie-withdraw-banner,.eu-cookie-withdraw-tab{background:gray}#sliding-popup.sliding-popup-bottom.eu-cookie-withdraw-wrapper{background:transparent}#sliding-popup .popup-content #popup-text h1,#sliding-popup .popup-content #popup-text h2,#sliding-popup .popup-content #popup-text h3,#sliding-popup .popup-content #popup-text p,.eu-cookie-compliance-secondary-button,.eu-cookie-withdraw-tab{color:#fff !important}.eu-cookie-withdraw-tab{border-color:#fff}.eu-cookie-compliance-more-button{color:#fff !important}
/* ]]> */
</style>

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__QJacVmveFyLIBSPkp21xL0ZvuDW9DMBrD2HB1P4Ry5I.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__B6tGdsQSt1vT8d8j-19-oS4FKJe83RrSNBkjPs9jfos__6VkE5WcQmmQpSVTQx-xS2-zn_nlwFlDm-EN1HLJNjrA__QJacVmveFyLIBSPkp21xL0ZvuDW9DMBrD2HB1P4Ry5I.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__I6pBwZrmO51afTZVOjN83FdGH1QRb9t15lSLso0JVV4__F3sVKlGyHKCaAPI_WdiHD7l_-05B6b9bLdUwzi1me90__QJacVmveFyLIBSPkp21xL0ZvuDW9DMBrD2HB1P4Ry5I.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__QJacVmveFyLIBSPkp21xL0ZvuDW9DMBrD2HB1P4Ry5I.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__QJacVmveFyLIBSPkp21xL0ZvuDW9DMBrD2HB1P4Ry5I.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__CRdjIL0HKDL2T9slYvkTSTLypBBPjyoDhwBezBuDy7c__cHebPmPNdg2jyPmpyQpZWssJQ4cPrqfaabp5Or9KlkU__QJacVmveFyLIBSPkp21xL0ZvuDW9DMBrD2HB1P4Ry5I.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://d33xdlntwy0kbs.cloudfront.net/cshl_custom.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/all/libraries/modernizr/modernizr.min.js?q5p1rr"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__yOhrJUBmMpN-jsuQlJYmS8ovkQU6I6KInr295HrSTb0__c--zHhbuOWmkdWkJYXiNWDmkpDRPesmbBjGlQcyqBOY__QJacVmveFyLIBSPkp21xL0ZvuDW9DMBrD2HB1P4Ry5I.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var _prum=[['id', '52e95df3abe53d2e33000000'], ['mark', 'firstbyte',
      (new Date()).getTime()]]; (function() {
      var s=document.getElementsByTagName('script')[0],
      p=document.createElement('script');
      p.async='async'; p.src='//rum-static.pingdom.net/prum.min.js';s.parentNode.insertBefore(p,s);})();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js___KyvuEvfmewWxdOLBXTYsSUo8Pfm_DijV6fWtGvSL_k__HSKkTgIr7nDcdvQYewipf6v4ak6ZLUOQkwPj44wKouA__QJacVmveFyLIBSPkp21xL0ZvuDW9DMBrD2HB1P4Ry5I.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__3uCzzYE5jr6ly9nvsM8PRBRkhh4p0NWCMDja4EKBy4M__dmGGVYcB8LhnbR2iAmVW0ZyXI6QuFVYFRORa5XxglXs__QJacVmveFyLIBSPkp21xL0ZvuDW9DMBrD2HB1P4Ry5I.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__0zdUME0GLauYYyEch2tGPydIw-IG7Uizip0Qb4wUsig__3RgszrCgk4bOnTzo4vaQ-hrWzaG6oD19l-R9CQ1YQ-o__QJacVmveFyLIBSPkp21xL0ZvuDW9DMBrD2HB1P4Ry5I.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__W6wAjopaALytcBam7oT3WhEzllkWaB9R-8hazHyvShM__rK5YBIqBy9qt0di0Sn4DriBx9TJupdmv6uGmLJyZvDs__QJacVmveFyLIBSPkp21xL0ZvuDW9DMBrD2HB1P4Ry5I.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-45638026-1", {"cookieDomain":"auto"});ga("set", "page", location.pathname + location.search + location.hash);ga("send", "pageview");ga('create', 'UA-189672-38', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__1PtAOZ8x7l6QAi8D7X6Kj_Y7HcUOFzvWRvvYHUcCT98__U8IB92u_CnnGwJ-X_xaUa9lzsC8OzUfN0WkK2JQfirg__QJacVmveFyLIBSPkp21xL0ZvuDW9DMBrD2HB1P4Ry5I.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings,{"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"zhHsAgc3RGslRkn1G_EUGp2UUsf5N8YRL3Mx6Yg7X80","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"modules\/comment\/comment.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"modules\/forum\/forum.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"sites\/default\/modules\/biorxiv\/css\/biorxiv.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/flexible\/flexible.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_collection\/css\/highwire_subject_collection.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire.highwire-list-expand.css":1,"public:\/\/ctools\/css\/bab59090150caf0c64e27afa1b36eef1.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/contrib\/eu_cookie_compliance\/css\/eu_cookie_compliance.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"0":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_biorxiv_styles\/css\/biorxiv-styles.css.less":1,"https:\/\/d33xdlntwy0kbs.cloudfront.net\/cshl_custom.css":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_figures.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"http:\/\/d33xdlntwy0kbs.cloudfront.net\/cshl_custom.js":1,"sites\/all\/modules\/contrib\/eu_cookie_compliance\/js\/eu_cookie_compliance.js":1,"sites\/default\/modules\/jnl_biorxiv_styles\/js\/adv-search-collection-toggle.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/eu_cookie_compliance\/js\/jquery.cookie-1.4.1.min.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/default\/modules\/jnl_biorxiv_styles\/js\/biorxiv-scripts.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_nps_survey\/js\/nps_survey.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"sites\/default\/modules\/contrib\/disqus\/disqus.js":1,"misc\/textarea.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/contrib\/service_links\/js\/twitter_button.js":1,"sites\/all\/modules\/contrib\/service_links\/js\/facebook_like.js":1,"sites\/all\/modules\/contrib\/service_links\/js\/google_plus_one.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.highwire-list-expand.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"1127478","apath":"\/biorxiv\/early\/2020\/01\/30\/2020.01.29.925396.atom","pisa":"biorxiv;2020.01.29.925396v1","processed":["highwire_math"],"markup":[{"requested":"full-text","variant":"full-text","view":"full","pisa":"biorxiv;2020.01.29.925396v1"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"disqus":{"domain":"biorxivstage","url":"https:\/\/www.biorxiv.org\/content\/10.1101\/2020.01.29.925396v1","title":"Global change adaptation limits future resilience","identifier":"node\/1127478"},"panels_ajax_pane":{"new-41":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u00222X4fKfQrJh3r0biNEHmlu8yXrymMughqCGrbJ0qDEKfCSi9ee4o4ogt2qcwVObAnqZSq+LgWMqpoAW60KwRyl7+65FTNE1EO4145MQkJFtX8VqqTmZw9RZirNH5wd\\\\\\\/Bo8dOtrXRajCohcmBEPtW5\\\\\\\/ndjVQDRcxxCbNZlAtv0ys4Axy+EbwgBRLQjcxdyyI6Z0klmnizhucmTW5su1mH3gST08H\\\\\\\/Haof58Bn8RTNudUfLR02F90SKFD17McVrhmakDHGPVurRkWxP84bpgNx0+oqluVcVkbhNe74q\\\\\\\/61UMkx8Om5TqKRNtDg1P8po299Nj7Sx2Supd+RoQHBo4IlAqu2jsWMGLpKT2GTrKwiUlJif5fFhE1fJ5seXfxepT57Oulqa9pQEB4mWElJSp0zG7ePFOAUAIVA2cSS3fPEMTZ4uMiyJC+TP1WwXxksqsPbCRdXoZ6CZJSW9b3+t0suHqeSa1HI5JQkTRyPyJVqffVoggrILJKcWlnr7Jy9AJNDAsd+ndk9zO1qEHS37SpOcrropunh028p6UzemnlynqC1UbhphPlMypMFLgnE+E+cQS\\\\\\\/u179pwoxtz9VqwWi\\\\\\\/ktR0D\\\\\\\/JRzOdiDX0DkH07jtv3FjEyJfJ1q3eII6yLa1xpIHOREOFlUBVMCANsjfUXHLinkP6\\\\\\\/CRXTaOO6I7v5GzFrKGUNnQXw8dSDclB+kbqKXDLG30GL6UYKh30fCK9CFx2bL0rmToh3R4QfH+iUgaMlsfdW5mQomCpQdg0GeXT6wNuP6qByvAZZXn4O\\\\\\\/PhzE35538sk2hKUq3bBoFgY2Bjgf5ZeRGBJMUWgqOBA7FxMWCfFgK3DhegUyty040oeSL0fi51rlL6dZkJMl\\\\\\\/utsNBWiQkIQemAf\\\\\\\/BHm0sKTO+Ny2Ra91KqIjv+mS5jvaW3lMO0dmhf61+q+9ELtuaxBachIh\\\\\\\/SJUmvlg8tV\\\\\\\/+FsyBd6qcFmir4pdmVi6YCohCFmq1jctJ7PG2flAilYYpoKBb7+45Ir3ZNKnRD86op7bQxeg0SDcEineU7zAo991FMw84Z+wgmltiaax6S7nnpQhT4r\\\\\\\/MH+JwTueiClPvhbpNJDqoT41xhgC2DCU4JHk31XhDEiMwXJqxYbRj+KgJFg\\\\\\\/KoZh4EYbIT9M7A\\\\\\\/qIHJr925cRmw59fSYnwiKsPcy7tzYl3tb7A0+lGr3a\\\\\\\/t7GffYKvs8WXJdTnyh35HqGvBsI\\\\\\\/zbf+CVni2vRxyx9gnWOtIWU63fW4SCZ5sgK3X9Vj9t\\\\\\\/qSfIdK8Mf+1B+D+zstW137Bpy7kn+v0teBCOernJ3aIXiRuthtJR97r1SWuBHgag4Pd8gkp1HfXzzLr35jcQPq\\\u0022,\\\u0022iv\\\u0022:\\\u00228nk\\\\\\\/FUz2LEU+G47p15TEYw==\\\u0022,\\\u0022salt\\\u0022:\\\u0022f9d8845712f72c9fee5db7d9ff111eab\\\u0022}\u0022,\u0022hmac\u0022:\u002292bc707e1566d49ece6aa93636b8a93792c94be05dd832c9810fccdb10b8ba56\u0022}"},"urlIsAjaxTrusted":{"\/content\/10.1101\/2020.01.29.925396v1.full":true},"ws_fl":{"width":100,"height":21},"ws_gpo":{"size":"","annotation":"","lang":"","callback":"","width":300},"color":{"logo":"https:\/\/www.biorxiv.org\/sites\/default\/files\/bioRxiv_article.jpg"},"highwire_list_expand":{"is_collapsed":"1"},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"eu_cookie_compliance":{"popup_enabled":1,"popup_agreed_enabled":0,"popup_hide_agreed":0,"popup_clicking_confirmation":1,"popup_scrolling_confirmation":false,"popup_html_info":"\u003Cdiv\u003E\n  \u003Cdiv class =\u0022popup-content info\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n      \u003Cp\u003EWe use cookies on this site to enhance your user experience. By clicking any link on this page you are giving your consent for us to set cookies.\u003C\/p\u003E\n    \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022agree-button eu-cookie-compliance-default-button\u0022\u003EContinue\u003C\/button\u003E\n                    \u003Cbutton type=\u0022button\u0022 class=\u0022find-more-button eu-cookie-compliance-more-button\u0022\u003EFind out more\u003C\/button\u003E\n          \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E","use_mobile_message":false,"mobile_popup_html_info":"\u003Cdiv\u003E\n  \u003Cdiv class =\u0022popup-content info\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n          \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022agree-button eu-cookie-compliance-default-button\u0022\u003EContinue\u003C\/button\u003E\n                    \u003Cbutton type=\u0022button\u0022 class=\u0022find-more-button eu-cookie-compliance-more-button\u0022\u003EFind out more\u003C\/button\u003E\n          \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E\n","mobile_breakpoint":"768","popup_html_agreed":"\u003Cdiv\u003E\n  \u003Cdiv class=\u0022popup-content agreed\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n      \u003Ch2\u003EThank you for accepting cookies\u003C\/h2\u003E\u003Cp\u003EYou can now hide this message or find out more about cookies.\u003C\/p\u003E    \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022hide-popup-button eu-cookie-compliance-hide-button\u0022\u003EHide\u003C\/button\u003E\n              \u003Cbutton type=\u0022button\u0022 class=\u0022find-more-button eu-cookie-compliance-more-button-thank-you\u0022 \u003EMore info\u003C\/button\u003E\n          \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E","popup_use_bare_css":false,"popup_height":"auto","popup_width":"100%","popup_delay":1000,"popup_link":"\/help\/cookie-policy","popup_link_new_window":1,"popup_position":null,"popup_language":"en","store_consent":false,"better_support_for_screen_readers":0,"reload_page":0,"domain":"","popup_eu_only_js":0,"cookie_lifetime":"365","cookie_session":false,"disagree_do_not_show_popup":0,"method":"default","whitelisted_cookies":"","withdraw_markup":"\u003Cbutton type=\u0022button\u0022 class=\u0022eu-cookie-withdraw-tab\u0022\u003EPrivacy settings\u003C\/button\u003E\n\u003Cdiv class=\u0022eu-cookie-withdraw-banner\u0022\u003E\n  \u003Cdiv class=\u0022popup-content info\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n      \u003Cp\u003E\u0026lt;h2\u0026gt;We use cookies on this site to enhance your user experience\u0026lt;\/h2\u0026gt;\u0026lt;p\u0026gt;You have given your consent for us to set cookies.\u0026lt;\/p\u0026gt;\u003C\/p\u003E\n    \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022eu-cookie-withdraw-button\u0022\u003EWithdraw consent\u003C\/button\u003E\n    \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E\n","withdraw_enabled":false},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1,"trackUrlFragments":1},"jnl_biorxiv_styles":{"defaultJCode":"biorxiv"},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-1127478 node-type-highwire-article context-content hw-default-jcode-biorxiv hw-article-type-article hw-article-category-new-results">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-M677548" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-M677548');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
        <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
    
  <div id="zone-branding" class="zone zone-branding clearfix print-display-block container-30">
    <div class="grid-15 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="bioRxiv" src="https://www.biorxiv.org/sites/default/files/bioRxiv_article.jpg" /></a>      </div>
                </div>
          </div>
</div><div class="grid-11 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-system block-menu block-main-menu block-system-main-menu odd block-without-title" id="block-system-main-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/" title="" class="" data-icon-position="" data-hide-link-title="0">Home</a></li>
<li class="leaf" role="menuitem"><a href="/about-biorxiv" class="" data-icon-position="" data-hide-link-title="0">About</a></li>
<li class="leaf" role="menuitem"><a href="/submit-a-manuscript" class="" data-icon-position="" data-hide-link-title="0">Submit</a></li>
<li class="last leaf" role="menuitem"><a href="/content/alertsrss" title="" class="" data-icon-position="" data-hide-link-title="0">ALERTS / RSS</a></li>
</ul></nav>    </div>
  </div>
</div><div class="block block-panels-mini block-biorxiv-search-box block-panels-mini-biorxiv-search-box even block-without-title" id="block-panels-mini-biorxiv-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-biorxiv_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/10.1101/2020.01.29.925396v1.full" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--2" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-4UpvEzZLfNGHQ5OVBjC0V3k_GiX6iZ9bzZPGu6ippEo" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2 advanced-search-link" >
  
      
  
  <div class="pane-content">
    <a href="/search">Advanced Search</a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix container-30">
  	      </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="1127478" id="node1127478" data-pisa="biorxiv;2020.01.29.925396v1" data-pisa-master="biorxiv;2020.01.29.925396" data-apath="/biorxiv/early/2020/01/30/2020.01.29.925396.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div  class="highwire-cite highwire-cite-highwire-article highwire-citation-biorxiv-article-top clearfix has-author-tooltip" >

      <span class="biorxiv-article-type">
        New Results    </span>
  
        <h1 class="highwire-cite-title" id="page-title">Global change adaptation limits future resilience</h1>  
      <div  class="highwire-cite-authors" ><span  class="highwire-citation-authors"><span class="highwire-citation-author first hw-author-orcid-logo-wrapper" data-delta="0"><a href="http://orcid.org/0000-0001-7678-564X" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a><span class="nlm-given-names">Reid S.</span> <span class="nlm-surname">Brennan</span></span>, <span class="highwire-citation-author" data-delta="1"><span class="nlm-given-names">James A.</span> <span class="nlm-surname">deMayo</span></span>, <span class="highwire-citation-author" data-delta="2"><span class="nlm-given-names">Hans G.</span> <span class="nlm-surname">Dam</span></span>, <span class="highwire-citation-author" data-delta="3"><span class="nlm-given-names">Michael</span> <span class="nlm-surname">Finiguerra</span></span>, <span class="highwire-citation-author" data-delta="4"><span class="nlm-given-names">Hannes</span> <span class="nlm-surname">Baumann</span></span>, <span class="highwire-citation-author" data-delta="5"><span class="nlm-given-names">Melissa H.</span> <span class="nlm-surname">Pespeni</span></span></span></div>
  
      <div  class="highwire-cite-metadata" ><span  class="highwire-cite-metadata-doi highwire-cite-metadata"><span class="label">doi:</span> https://doi.org/10.1101/2020.01.29.925396 </span></div>
  
  
  </div>
<div id="hw-article-author-popups-node1127478" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Reid S. Brennan </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-institution'>Department of Biology, University of Vermont</span>, 109 Carrigan Burlington, VT, <span class='nlm-country'>USA</span>, 05405</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Reid%2BS.%2BBrennan%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Brennan%20RS&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AReid%2BS.%2BBrennan%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0001-7678-564X" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Reid S. Brennan</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:Reid.Brennan@uvm.edu" class="" data-icon-position="" data-hide-link-title="0">Reid.Brennan@uvm.edu</a>
<a href="mailto:mpespeni@uvm.edu" class="" data-icon-position="" data-hide-link-title="0">mpespeni@uvm.edu</a></span></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">James A. deMayo </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-institution'>Department of Marine Sciences, University of Connecticut</span>, 1080 Shennecossett Rd, Groton, CT, <span class='nlm-country'>USA</span>, 06340</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=James%2BA.%2BdeMayo%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=deMayo%20JA&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJames%2BA.%2BdeMayo%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Hans G. Dam </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-institution'>Department of Marine Sciences, University of Connecticut</span>, 1080 Shennecossett Rd, Groton, CT, <span class='nlm-country'>USA</span>, 06340</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Hans%2BG.%2BDam%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Dam%20HG&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AHans%2BG.%2BDam%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Michael Finiguerra </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-institution'>Department of Ecology and Evolutionary Biology, University of Connecticut</span>, 1080 Shennecossett Rd, Groton, CT, <span class='nlm-country'>USA</span>, 06340</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Michael%2BFiniguerra%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Finiguerra%20M&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AMichael%2BFiniguerra%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Hannes Baumann </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-institution'>Department of Marine Sciences, University of Connecticut</span>, 1080 Shennecossett Rd, Groton, CT, <span class='nlm-country'>USA</span>, 06340</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Hannes%2BBaumann%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Baumann%20H&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AHannes%2BBaumann%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Melissa H. Pespeni </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-institution'>Department of Biology, University of Vermont</span>, 109 Carrigan Burlington, VT, <span class='nlm-country'>USA</span>, 05405</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Melissa%2BH.%2BPespeni%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Pespeni%20MH&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AMelissa%2BH.%2BPespeni%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:Reid.Brennan@uvm.edu" class="" data-icon-position="" data-hide-link-title="0">Reid.Brennan@uvm.edu</a>
<a href="mailto:mpespeni@uvm.edu" class="" data-icon-position="" data-hide-link-title="0">mpespeni@uvm.edu</a></span></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/10.1101/2020.01.29.925396v1" class="panels-ajax-tab-tab" data-panel-name="biorxiv_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:1127478" data-trigger="" data-url-enabled="1">Abstract</a><a href="/panels_ajax_tab/biorxiv_tab_art/node:1127478/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/10.1101/2020.01.29.925396v1.full-text" class="panels-ajax-tab-tab" data-panel-name="article_tab_full_text" data-target-id="highwire_article_tabs" data-entity-context="node:1127478" data-trigger="full-text" data-url-enabled="1">Full Text</a><a href="/panels_ajax_tab/article_tab_full_text/node:1127478/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/10.1101/2020.01.29.925396v1.article-info" class="panels-ajax-tab-tab" data-panel-name="biorxiv_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:1127478" data-trigger="article-info" data-url-enabled="1">Info/History</a><a href="/panels_ajax_tab/biorxiv_tab_info/node:1127478/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/10.1101/2020.01.29.925396v1.article-metrics" class="panels-ajax-tab-tab" data-panel-name="article_tab_metrics" data-target-id="highwire_article_tabs" data-entity-context="node:1127478" data-trigger="article-metrics" data-url-enabled="1">Metrics</a><a href="/panels_ajax_tab/article_tab_metrics/node:1127478/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/10.1101/2020.01.29.925396v1.full.pdf+html" class="panels-ajax-tab-tab" data-panel-name="biorxiv_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:1127478" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i> Preview PDF</a><a href="/panels_ajax_tab/biorxiv_tab_pdf/node:1127478/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="article_tab_full_text" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.biorxiv.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-article_tab_full_text"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-markup" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2 class="">Abstract</h2><p id="p-3">Global conditions are changing at unprecedented rates (<a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">1</a>), challenging species resilience (<a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">2</a>). Populations can respond to these changes through genetic adaptation and physiological plasticity (<a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">3</a>, <a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">4</a>), and it is well accepted that these processes interact (<a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">5</a>). However, the relative role plasticity and adaptation play in promoting resilience to global change and the potential impacts of rapid adaptation on plasticity during this process are not well understood (<a id="xref-ref-3-2" class="xref-bibr" href="#ref-3">3</a>, <a id="xref-ref-5-2" class="xref-bibr" href="#ref-5">5</a>–<a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">10</a>). Here, using experimental evolution and reciprocal transplantation of the marine coastal copepod, <em>Acartia tonsa</em>, we show rapid adaptation to global change conditions carries costs, notably a loss of physiological plasticity. Twenty generations of selection resulted in rapid, highly parallel adaptation to greenhouse conditions. However, this adaptation reduced plasticity and fitness, particularly when returned to ambient conditions and under food limitation. Due to the loss of plasticity, greenhouse adapted lines could no longer tolerate ambient conditions and compensated by additional adaptive evolution, a process that eroded nucleotide diversity in genes containing adaptive genetic variation for global change conditions. These results show that adaptation can negatively affect plasticity, limiting resilience to new stressors and previously benign environments. Our findings challenge the common assertion that the presence of adaptive genetic potential or physiological plasticity alone will enable population resilience under global change conditions.</p><div id="sec-1" class="subsection"><p id="p-4"><strong>Significance statement</strong> Understanding how species will respond to changing environmental conditions is an essential step in mitigating the potential negative consequences of human induced global change conditions. Typically, it is assumed that a species with sufficient plasticity or adaptive potential will be resilient in the face of these changes. Here, we experimentally evolve a copepod to simulated future acidic and warm conditions. We show that rapid adaptation reduced phenotypic plasticity and eroded the adaptive genetic variation necessary to tolerate additional environmental stress. These findings demonstrate that the presence of plasticity or adaptive potential do not necessarily indicate a resilient population, illustrating the importance of considering the interaction between these factors when predicting species tolerance to global change.</p></div></div><div class="section" id="sec-2"><h2 class="">Main Text</h2><p id="p-5">Though global conditions are changing at a geologically unprecedented rate and the frequency of extreme events is increasing (<a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">11</a>), species can use genetic adaptation or physiological plasticity to persist (<a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">12</a>, <a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">13</a>). Even over remarkably short time scales, one to four generations, selection can shift populations towards adaptive phenotypes (<a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">14</a>–<a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">19</a>). However, we still know little about the genome-wide underpinnings of complex physiological phenotypes, the relative importance of genetic adaptation versus plasticity for responding to environmental changes on different time scales, or the impacts of rapid adaptation on plasticity.</p><p id="p-6">In theory, the species best equipped to respond to rapid environmental change are those that live in dynamic environments, have broad distributions, and high dispersal (<a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">20</a>, <a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">21</a>). These characteristics support the maintenance of both standing genetic variation and physiological plasticity that can enable population persistence as conditions change (<a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">8</a>, <a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">22</a>). Many marine organisms fit this description (<a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">23</a>, <a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">24</a>), particularly copepods, the most abundant marine metazoan (<a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">25</a>) and the main link between primary producers and upper trophic levels (<a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">26</a>). Copepods have been shown to have the capacity for plasticity and genetic adaptation to global change conditions (<a id="xref-ref-27-1" class="xref-bibr" href="#ref-27">27</a>–<a id="xref-ref-29-1" class="xref-bibr" href="#ref-29">29</a>). However, empirical evidence of adaptive capacity to global change conditions over longer timescales in natural populations of any metazoan is limited. Here, we use 20 generations of experimental evolution to greenhouse conditions and three generations of reciprocal transplantation to understand the interplay of rapid adaptation and plasticity and demonstrate associated costs of adaptation.</p><p id="p-7">We experimentally evolved the copepod <em>Acartia tonsa</em>, a globally distributed, foundational species (<a id="xref-ref-26-2" class="xref-bibr" href="#ref-26">26</a>), for 20 generations in combined high CO<sub>2</sub> × temperature (greenhouse, GH: 2000 ppm <em>p</em>CO<sub>2</sub> and 22°C) versus ambient (AM: 400 ppm <em>p</em>CO<sub>2</sub> and 18°C) conditions followed by three generations of reciprocal transplantation of both lines to opposite conditions. These conditions were chosen as they represent a worst case, yet realistic scenario (<a id="xref-ref-30-1" class="xref-bibr" href="#ref-30">30</a>, <a id="xref-ref-31-1" class="xref-bibr" href="#ref-31">31</a>); in the coming century, global mean ocean surface temperatures will increase by 2-4 °C (<a id="xref-ref-32-1" class="xref-bibr" href="#ref-32">32</a>) and oceanic CO<sub>2</sub> concentrations will potentially reach 2000 ppm by the year 2300 (<a id="xref-ref-33-1" class="xref-bibr" href="#ref-33">33</a>). We denote evolved lines with their abbreviation and the treatment condition as a subscript (i.e., AM<sub>GH</sub> = ambient line transplanted to greenhouse conditions). RNA was collected from pools of 20 adults from each of four replicate cultures at the first, second, and third generation after transplant for both treatment and transplanted lines. We estimated allele frequencies at 322,595 variant sites with at least 50× coverage across all samples and quantified transcript abundance for an average 24,112 genes. For organismal evidence of the presence and costs of adaptation, we measured fecundity and survival for all cultures and their transplants. Because costs may only be evident in the face of additional stressors, we quantified fitness under limited food availability, which is a potential consequence of climate change for marine environments (<a id="xref-ref-34-1" class="xref-bibr" href="#ref-34">34</a>). We integrated these genomic, physiological, and organismal data to: 1) quantify the degree and mechanisms of adaptation, and 2) test for potential interactions of adaptation and plasticity.</p><p id="p-8">Replicate cultures showed consistent signatures of genetic divergence after 20 generations in greenhouse versus ambient conditions with PC1 explaining 14.32% of the genome-wide variance (<a id="xref-fig-1-1" class="xref-fig" href="#F1">Fig. 1A</a>). Low variance among replicates within a treatment and strong, directional divergence between experimental lines suggest that divergence in allele frequencies was due to selection. As additional evidence that genetic divergence was adaptive, differences in allele frequencies were non-randomly concentrated in genes related to cytoskeleton organization, morphogenesis, responses to stress, and regulation of gene expression (Table S1). Greenhouse lines genetically diverged from each other after transplant; by the third generation GH<sub>GH</sub> and GH<sub>AM</sub> diverged in allele frequencies while AM<sub>AM</sub> and AM<sub>GH</sub> did not (<a id="xref-fig-1-2" class="xref-fig" href="#F1">Fig. 1C</a>, PC2). Considering gene expression patterns, variance across all transcripts showed strong effects of selection line at the end of the first transplant generation (<a id="xref-fig-1-3" class="xref-fig" href="#F1">Fig. 1D</a>). However, by the end of the third transplant generation, populations clustered by treatment rather than selection line, demonstrating transgenerational convergence in gene expression (<a id="xref-fig-1-4" class="xref-fig" href="#F1">Fig. 1D-F</a>). Evolved expression differences between GH<sub>GH</sub> and AM<sub>AM</sub> were enriched for functions related to responses to stimuli, development, and homeostasis (Table S2). Among these were Carbonic Anhydrase, SWIB domain-containing and Histone H1-like. Carbonic Anhydrase catalyzes the hydration of CO<sub>2</sub> to bicarbonate and is regularly responsive to experimental low pH (<a id="xref-ref-35-1" class="xref-bibr" href="#ref-35">35</a>) while SWIB domain and Histone H1-like are involved in chromatin structure and transcriptional regulation (<a id="xref-ref-36-1" class="xref-bibr" href="#ref-36">36</a>, <a id="xref-ref-37-1" class="xref-bibr" href="#ref-37">37</a>).</p><div id="F1" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/01/30/2020.01.29.925396/F1.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Principal component analysis of genome-wide genetic variation (panels A-C) and gene expression patterns (panels D-F) during three generations of reciprocal transplant following 20 generations of selection for ambient (AM) and greenhouse (GH) conditions. Shape indicates selection line and color indicates environmental condition at that generation. Both gene expression and genetic variation clustered by line, rather than condition at F1. By F3, a line effect was evident for both lines&#x2019; gene expression, but only GH lines began to diverge in allele frequency." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1553945654" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Principal component analysis of genome-wide genetic variation (panels A-C) and gene expression patterns (panels D-F) during three generations of reciprocal transplant following 20 generations of selection for ambient (AM) and greenhouse (GH) conditions. Shape indicates selection line and color indicates environmental condition at that generation. Both gene expression and genetic variation clustered by line, rather than condition at F1. By F3, a line effect was evident for both lines&#x2019; gene expression, but only GH lines began to diverge in allele frequency.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 1:" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2020/01/30/2020.01.29.925396/F1.medium.gif" width="440" height="307"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 1:" src="https://www.biorxiv.org/content/biorxiv/early/2020/01/30/2020.01.29.925396/F1.medium.gif" width="440" height="307"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/01/30/2020.01.29.925396/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1:" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/01/30/2020.01.29.925396/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml"><span class="fig-label">Figure 1:</span> <p id="p-9" class="first-child">Principal component analysis of genome-wide genetic variation (panels A-C) and gene expression patterns (panels D-F) during three generations of reciprocal transplant following 20 generations of selection for ambient (AM) and greenhouse (GH) conditions. Shape indicates selection line and color indicates environmental condition at that generation. Both gene expression and genetic variation clustered by line, rather than condition at F1. By F3, a line effect was evident for both lines’ gene expression, but only GH lines began to diverge in allele frequency.</p><div class="sb-div caption-clear"></div></div></div><p id="p-10">Measures of fitness also indicate that lines adaptively evolved; fecundity was equally high for lines in their home environments (<a id="xref-fig-2-1" class="xref-fig" href="#F2">Fig. 2A</a>). AM<sub>AM</sub> and AM<sub>GH</sub> showed similarly high egg production, indicating plasticity to tolerate both conditions (<a id="xref-fig-2-2" class="xref-fig" href="#F2">Fig. 2A</a>). In contrast, transplanting greenhouse lines to ambient conditions (GH<sub>AM</sub>) resulted in a 48% reduction in egg production relative to GH<sub>GH</sub> (<em>P</em> &lt; 0.02; <a id="xref-fig-2-3" class="xref-fig" href="#F2">Fig. 2A</a>), demonstrating a loss of plasticity to tolerate previously benign conditions. Exposure to food limitation revealed further adaptation costs in greenhouse lines regardless of environment with a 67% reduction in egg production rates relative to ambient lines (<em>P</em> &lt; 0.001; <a id="xref-fig-2-4" class="xref-fig" href="#F2">Fig 2A</a>). In addition, survivorship from nauplii to reproductively mature adulthood was 60% lower for greenhouse lineages compared to ambient lineages regardless of the environment or food regime (<em>P</em> &lt; 0.001; <a id="xref-fig-2-5" class="xref-fig" href="#F2">Fig. 2B</a>). To explore this further, we estimated the population growth rate and fitness by calculating the Malthusian parameter (see supplemental methods) for each line and condition (<a id="xref-ref-38-1" class="xref-bibr" href="#ref-38">38</a>). This measure, which integrates survival and fecundity, showed ambient lines, transplanted or not, at high food conditions had the highest fitness while GH<sub>AM</sub> performed the worst (<em>P</em> &lt; 0.03; <a id="xref-fig-2-6" class="xref-fig" href="#F2">Fig 2C</a>). GH<sub>GH</sub> lines were intermediate in fitness relative to the other groups while greenhouse lines in low food conditions had the lowest fitness (<a id="xref-fig-2-7" class="xref-fig" href="#F2">Fig. 2C</a>). The reductions in performance suggest that while directional selection in greenhouse conditions resulted in adaptation and the maintenance of high egg production, it came at the cost of maintaining plasticity to tolerate other environments and reduced survival and fitness overall.</p><div id="F2" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/01/30/2020.01.29.925396/F2.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Fitness costs under replete and low food conditions. A) Egg production under ad-libitum and lowered food. B) Survivorship under ad-libitum and lowered food. C) Malthusian parameter calculated from combined fecundity and survivorship data. Error bars indicate 95% confidence intervals." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1553945654" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Fitness costs under replete and low food conditions. A) Egg production under ad-libitum and lowered food. B) Survivorship under ad-libitum and lowered food. C) Malthusian parameter calculated from combined fecundity and survivorship data. Error bars indicate 95% confidence intervals.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 2." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2020/01/30/2020.01.29.925396/F2.medium.gif" width="440" height="159"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 2." src="https://www.biorxiv.org/content/biorxiv/early/2020/01/30/2020.01.29.925396/F2.medium.gif" width="440" height="159"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/01/30/2020.01.29.925396/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2." data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/01/30/2020.01.29.925396/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 2.</span> <p id="p-11" class="first-child">Fitness costs under replete and low food conditions. A) Egg production under ad-libitum and lowered food. B) Survivorship under ad-libitum and lowered food. C) Malthusian parameter calculated from combined fecundity and survivorship data. Error bars indicate 95% confidence intervals.</p><div class="sb-div caption-clear"></div></div></div><p id="p-12">To further characterize the genomic underpinnings of the adaptive divergence, we tested for the degree of overlap between evolved allelic and expression differences. Adaptive allele frequencies were identified as a set of 17,720 single nucleotide polymorphisms (SNPs; 5.5% of all SNPs) in 4,270 genes (53% of genes with at least 1 SNP) that were consistently different between GH<sub>GH</sub> and AM<sub>AM</sub> lines in all three generations. Evolved differences in expression were defined as transcripts differentially expressed (identified using DESeq2, <em>P</em><sub>adj</sub> &lt; 0.05) between GH<sub>GH</sub> and AM<sub>AM</sub> lines. There was little overlap in the specific genetic loci that had significant expression and allelic differences (16%; Fisher’s exact test; <em>P</em> &gt; 0.05) and no correlation between expression and allelic responses (<a id="xref-fig-5-1" class="xref-fig" href="#F5">Fig. S1</a>; R<sup>2</sup> = 0.002). Importantly, the lack of overlap suggests that estimates of allele frequencies from pools of sequenced RNA were not impacted by allele-biased expression. Though there was no overlap at the gene level, there was substantial overlap between the functional classes of genes, indicating that adaptive divergence in protein function versus expression targeted different, but functionally related genes. Notable were those related to the response and detection of stress, and actin organization and regulation. Actin regulation is linked to cytoskeleton maintenance and has been identified as responsive to low pH stress in copepods (<a id="xref-ref-39-1" class="xref-bibr" href="#ref-39">39</a>). In addition, we found strong allelic differences between lines in regulators of expression (Table S1), suggesting that trans-acting regulators may have been under selection and responsible for the adaptive changes in gene expression.</p><p id="p-13">Measuring expression patterns across three consecutive transplant generations allowed the unique opportunity to assess the degree to which changes in gene expression across generations were adaptive. We characterized the relationship between evolved and transplant responses for each transplant generation. Transplant responses were defined as the change in gene expression of a line in transplanted versus selected conditions (y-axes of <a id="xref-fig-3-1" class="xref-fig" href="#F3">Fig. 3A</a>) and were compared to the evolved differences in expression between non-transplanted lines (x-axes of <a id="xref-fig-3-2" class="xref-fig" href="#F3">Fig. 3A</a>). A correlation between evolved and transplant responses indicates the degree of adaptive gene expression, where a positive correlation represents adaptive changes in expression (<a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">9</a>). After one generation in transplant conditions, transcriptional expression was adaptive and positively correlated with the evolved response for both lines (ρ<sub>AM</sub> = 0.57, ρ<sub>GH</sub> = 0.64, <em>P</em> &lt; 0.001). This correlation increased with each successive generation (<em>P</em> &lt; 0.001; <a id="xref-fig-2-8" class="xref-fig" href="#F2">Fig. 2A</a>). In accord, the proportion of positively correlated, adaptively expressed genes (dark points; <a id="xref-fig-2-9" class="xref-fig" href="#F2">Fig. 2A</a>) increased across generations for both ambient and greenhouse lines (<em>P</em> &lt; 0.05; Proportions: F1<sub>AM</sub> = 0.85, F2<sub>AM</sub> = 0.87, F3<sub>AM</sub> = 0.90, F1<sub>GH</sub> = 0.66, F2<sub>GH</sub> = 0.94, F3<sub>GH</sub> = 0.95) with the greatest response in greenhouse lines, 29% versus 5% increased change in proportion adaptive. This increase in adaptive gene expression across generations indicates that copepods, even after 20 generations of experimental evolution in ambient or greenhouse conditions, were able to match adaptive gene expression profiles increasingly with each successive generation.</p><div id="F3" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/01/30/2020.01.29.925396/F3.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Convergence on evolved differences following transplant. For all plots, the dashed black line is a 1:1 relationship, the solid red line is the observed relationship between the x and y axes. The x-axis is the evolved and adaptive differences between the lines in their home environment. The y-axis compares each line in its transplanted environment to its home environment. Plots illustrate whether transplanted lines converge on the adaptive pattern for a given treatment condition. Dark points show the same pattern as the adaptive difference, light points are in the opposite direction. Pearson&#x2019;s correlation (&#x3C1;) is presented for each plot at each generation; generations organized F1 to F3, top to bottom. (A) Evolved changes in gene expression between GH and AM in their home environments (GHGH and AMAM; x-axis) versus the transplant changes in expression (GHAM and AMGH1; y-axis) for the same genes. (B) Evolved differences in allele frequency between GH and AM in their home environment (x-axis) versus the change in allele frequency after transplant (y-axis) for the same genes." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1553945654" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Convergence on evolved differences following transplant. For all plots, the dashed black line is a 1:1 relationship, the solid red line is the observed relationship between the x and y axes. The x-axis is the evolved and adaptive differences between the lines in their home environment. The y-axis compares each line in its transplanted environment to its home environment. Plots illustrate whether transplanted lines converge on the adaptive pattern for a given treatment condition. Dark points show the same pattern as the adaptive difference, light points are in the opposite direction. Pearson&#x2019;s correlation (&#x3C1;) is presented for each plot at each generation; generations organized F1 to F3, top to bottom. (A) Evolved changes in gene expression between GH and AM in their home environments (GH&lt;sub&gt;GH&lt;/sub&gt; and AM&lt;sub&gt;AM&lt;/sub&gt;; x-axis) versus the transplant changes in expression (GH&lt;sub&gt;AM&lt;/sub&gt; and AM&lt;sub&gt;GH&lt;/sub&gt;1; y-axis) for the same genes. (B) Evolved differences in allele frequency between GH and AM in their home environment (x-axis) versus the change in allele frequency after transplant (y-axis) for the same genes.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 3." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2020/01/30/2020.01.29.925396/F3.medium.gif" width="440" height="252"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 3." src="https://www.biorxiv.org/content/biorxiv/early/2020/01/30/2020.01.29.925396/F3.medium.gif" width="440" height="252"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/01/30/2020.01.29.925396/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 3." data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/01/30/2020.01.29.925396/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 3.</span> <p id="p-14" class="first-child">Convergence on evolved differences following transplant. For all plots, the dashed black line is a 1:1 relationship, the solid red line is the observed relationship between the x and y axes. The x-axis is the evolved and adaptive differences between the lines in their home environment. The y-axis compares each line in its transplanted environment to its home environment. Plots illustrate whether transplanted lines converge on the adaptive pattern for a given treatment condition. Dark points show the same pattern as the adaptive difference, light points are in the opposite direction. Pearson’s correlation (ρ) is presented for each plot at each generation; generations organized F1 to F3, top to bottom. (A) Evolved changes in gene expression between GH and AM in their home environments (GH<sub>GH</sub> and AM<sub>AM</sub>; x-axis) versus the transplant changes in expression (GH<sub>AM</sub> and AM<sub>GH</sub>1; y-axis) for the same genes. (B) Evolved differences in allele frequency between GH and AM in their home environment (x-axis) versus the change in allele frequency after transplant (y-axis) for the same genes.</p><div class="sb-div caption-clear"></div></div></div><p id="p-15">To test if changes in gene expression were due to plasticity or rapid adaptive evolution, we took a similar approach as above and quantified the shift in allele frequency in response to transplant (y-axes of <a id="xref-fig-3-3" class="xref-fig" href="#F3">Fig. 3B</a>) and compared this to the evolved divergence in allele frequency, calculatedas the mean difference in allele frequency between GH<sub>GH</sub> and AM<sub>AM</sub> (x-axes of <a id="xref-fig-3-4" class="xref-fig" href="#F3">Fig. 3B</a>). This quantified the degree to which transplanted lines utilized the same adaptively evolved alleles when transplanted into that environment. GH<sub>AM</sub> copepods successively evolved to match allele frequencies of the ambient adaptive alleles with increasing changes in frequencies across each generation (<a id="xref-fig-3-5" class="xref-fig" href="#F3">Fig. 3B</a>). In contrast, AM<sub>GH</sub> lines did not undergo the same rapid adaptation but maintained allele frequencies (<a id="xref-fig-3-6" class="xref-fig" href="#F3">Fig. 3B</a>), even though their transcriptional profiles shifted to match greenhouse adaptive expression (<a id="xref-fig-3-7" class="xref-fig" href="#F3">Fig. 3A</a>). Likewise, the proportion of adaptive alleles (dark points) increased across transplant generations for the greenhouse lines, but not the ambient lines (<a id="xref-fig-3-8" class="xref-fig" href="#F3">Fig. 3B</a>; <em>P</em> &lt; 0.001; Proportions: F1<sub>AM</sub> = 0.68, F2<sub>AM</sub> = 0.64, F3<sub>AM</sub> = 0.65; F1<sub>GH</sub> = 0.68, F2<sub>GH</sub> = 0.75, F3<sub>GH</sub> = 0.76). Together, these results show that the GH<sub>AM</sub> animals matched the adaptive transcriptional profile through selected changes in allele frequencies across generations while AM<sub>GH</sub> animals were able to match the greenhouse adaptive transcriptional profiles without changes in allele frequencies.</p><p id="p-16">Combining experimental evolution and reciprocal transplantation, we observe response mechanisms across different time scales. Transplanted populations responded to greenhouse conditions using plasticity across three generations whereas 20 generations of greenhouse conditions led to genetic adaptation and loss of physiological plasticity. Taken together, these results suggest that while an initial response to greenhouse conditions can be mediated by plasticity, adaptive transcriptional profiles become canalized through genetic adaptation over longer periods of time. This canalization, which could happen over a single year of greenhouse conditions, in turn can limit organismal plasticity and reduce fitness when conditions change, even a return to a previously benign state.</p><p id="p-17">Rapid adaptation can result in a selective bottleneck which could drive a loss of genetic diversity. To test this hypothesis, we quantified nucleotide diversity (π) for each treatment. Remarkably, we found no significant loss of genetic diversity after long-term adaptation to greenhouse conditions (GH<sub>GH</sub>, <a id="xref-fig-4-1" class="xref-fig" href="#F4">Fig. 4A</a>, <em>P</em> &gt; 0.05). Similarly, ambient lines, both transplanted and not, maintained high levels of genetic diversity (<em>P</em> &gt; 0.05). However, when transplanted back to ambient conditions, GH<sub>AM</sub> showed a significant drop in diversity by F3 (<em>P</em> &lt; 0.05; <a id="xref-fig-4-2" class="xref-fig" href="#F4">Fig. 4A</a>), indicating a global loss of genetic diversity This suggests that the rapid changes in allele frequency after transplant of GH to AM (GH<sub>AM</sub>; <a id="xref-fig-3-9" class="xref-fig" href="#F3">Fig. 3</a>) resulted in a loss of genetic diversity. To test if this global loss of genetic diversity was concentrated in specific regions of the genome, we compared changes in nucleotide diversity in loci identified as adaptively diverged between greenhouse and ambient lines versus not. The loss of nucleotide diversity was concentrated in genomic regions containing adaptive loci for GH<sub>AM</sub> lines (<a id="xref-fig-4-3" class="xref-fig" href="#F4">Fig. 4B</a>; <em>P</em> = 0.005). More broadly, the loss of nucleotide diversity was concentrated in genes with functions related to sequestration of actin monomers, cytokinesis, and response to stress (<a id="xref-fig-4-4" class="xref-fig" href="#F4">Fig. 4C</a>; Table S3), similar to those underlying adaptive genetic divergence between AM<sub>AM</sub> and GH<sub>GH</sub> lines (Table S1). In contrast, AM<sub>GH</sub> lines did not show a loss of nucleotide diversity (<em>P</em> = 0.11) or functional enrichment, indicating no targeted loss of diversity, which is consistent with the sustained ability for a plastic response in these lines. These results also indicate that the loss of genetic diversity for GH<sub>AM</sub> was due to selection following their loss of physiological plasticity and suggests a conflict between greenhouse adaptive and ambient adaptive alleles.</p><div id="F4" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/01/30/2020.01.29.925396/F4.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Costs of adaptation. (A) Median nucleotide diversity (&#x3C0;) for all treatments and generations following transplant. (B) Mean change in &#x3C0; from F1 to F3 after transplant for adaptive and non-adaptive SNPs. Letters above each point (for A and B) show significance; points sharing letters are not significantly different. (C) Gene ontology enrichment for the loss of genetic diversity of GHAM at transplant F3; AMGH showed no enrichment." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1553945654" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Costs of adaptation. (A) Median nucleotide diversity (&#x3C0;) for all treatments and generations following transplant. (B) Mean change in &lt;em&gt;&#x3C0;&lt;/em&gt; from F1 to F3 after transplant for adaptive and non-adaptive SNPs. Letters above each point (for A and B) show significance; points sharing letters are not significantly different. (C) Gene ontology enrichment for the loss of genetic diversity of GH&lt;sub&gt;AM&lt;/sub&gt; at transplant F3; AM&lt;sub&gt;GH&lt;/sub&gt; showed no enrichment.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 4." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2020/01/30/2020.01.29.925396/F4.medium.gif" width="440" height="133"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 4." src="https://www.biorxiv.org/content/biorxiv/early/2020/01/30/2020.01.29.925396/F4.medium.gif" width="440" height="133"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/01/30/2020.01.29.925396/F4.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 4." data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/01/30/2020.01.29.925396/F4.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 4.</span> <p id="p-18" class="first-child">Costs of adaptation. (A) Median nucleotide diversity (π) for all treatments and generations following transplant. (B) Mean change in <em>π</em> from F1 to F3 after transplant for adaptive and non-adaptive SNPs. Letters above each point (for A and B) show significance; points sharing letters are not significantly different. (C) Gene ontology enrichment for the loss of genetic diversity of GH<sub>AM</sub> at transplant F3; AM<sub>GH</sub> showed no enrichment.</p><div class="sb-div caption-clear"></div></div></div><p id="p-19">Even though our experimental conditions represent worst case greenhouse scenarios, our results demonstrate that copepods can adapt to global change conditions in as little as 20 generations, approximately one year, using standing genetic variation. Despite the high predicted adaptive capacity for this species, reciprocal transplantation and food limitation revealed costs associated with rapid evolution. The reduction in plasticity, genetic diversity, and tolerance to a novel stressor suggests that future adaptive potential and tolerance to other global climate change conditions may be limited, particularly for species with less standing genetic diversity and lower physiological plasticity. We demonstrate the power of experimental evolution from natural populations to reveal the mechanisms, time-scales of responses, consequences, and reversibility of complex, physiological adaptation. As we begin to incorporate adaptive potential into species persistence models (<a id="xref-ref-40-1" class="xref-bibr" href="#ref-40">40</a>–<a id="xref-ref-42-1" class="xref-bibr" href="#ref-42">42</a>), these results caution that, even for species predicted to be resilient, there may be hidden costs to rapid adaptive evolution.</p></div><div class="section" id="sec-3"><h2 class="">Methods</h2><div id="sec-4" class="subsection"><h3>Experimental setup</h3><p id="p-20">Copepods were collected in June of 2016 from Esker Point Beach in Groton, CT, USA (41.320725, −72.001643) and raised for at least three generations prior to the start of transgenerational adaptation to limit maternal effects(<a id="xref-ref-43-1" class="xref-bibr" href="#ref-43">43</a>). Parental individuals were acclimated to one of two experimental conditions: 1) AM – Ambient temperature (18 °C), Ambient CO<sub>2</sub> (~400 ppmv CO<sub>2</sub>, pH ~8.2), 2) GH – High temperature (22 °C), high CO<sub>2</sub> (~2000 ppmv CO<sub>2</sub>, pH ~7.5). Cultures were fed every 48-72 hours at food-replete concentrations of carbon (&gt;800 μg C/L) split equally between three species of prey phytoplankton. Prey phytoplankton included <em>Tetraselmis</em> spp., <em>Rhodomonas</em> spp., and <em>Thalassiosira weissflogii</em>, which is a common diet combination used for rearing copepods (<a id="xref-ref-44-1" class="xref-bibr" href="#ref-44">44</a>). About 400 females and 200 males were used to seed each treatment, yielding an estimated 15,000 eggs and newly hatched nauplii for the initial F0 generation. Each treatment was kept in separate temperature-controlled incubators and separated into four replicate 10L culture containers. Elevated CO<sub>2</sub> levels were achieved with two-gas proportioners (Cole-Parmer, Vernon Hills, IL, USA) combining air with 100% dry CO<sub>2</sub> that was delivered continuously to the bottom of each replicate culture. Target pH values were monitored daily using a handheld pH probe (Orion Ross Ultra pH/ATC Triode with Orion Star A121 pH Portable Meter (Thermo FisherScientific®, Waltham, MA, USA). Continuous bubbling maintained higher than necessary dissolved oxygen levels (&gt;8 mg/L). To assess functional lifehistory traits, smaller volume experiments were housed in the same temperature-controlled incubators in custom plexiglass enclosures with the atmosphere continuously flooded with CO<sub>2</sub> infused air at the appropriate concentration which allows for passive diffusion of CO<sub>2</sub> into the experiments. Copepods were raised in the respective four experimental treatments for 20 generations before the reciprocal transplant.</p></div><div id="sec-5" class="subsection"><h3>Reciprocal transplant</h3><p id="p-21">At the 21st generation, we performed a reciprocal transplant between AM and GH. Each of the four replicates from each treatment was split to yield four additional replicates for each of two new transplant treatments: AM<sub>GH</sub> and GH<sub>AM</sub> (as well as sham transfers: AM<sub>AM</sub> and GH<sub>GH</sub>). Copepods were raised for three subsequent generations and were fed every 48-72 hours as described above. This design lead to 48 total cultures: 2 treatments × 2 transplant/non-transplant × 4 replicates × 3 generations.</p></div><div id="sec-6" class="subsection"><h3>Life-history traits</h3><p id="p-22">Day-specific survivorship was measured every 48-72 hours. Food media was acclimated to the appropriate temperature and CO<sub>2</sub> concentration and replaced on monitoring days at the appropriate concentration (&gt;800 μg C/L for food-replete and 250 μg C/L for food-limited) divided in equal carbon proportions between the three afore-mentioned prey species. Survivorship was assessed among eight 250-mL beakers per treatment (2 food concentrations × 4 replicates = 8 beakers per treatment) containing 25 individual N1 nauplii and monitored until sexual maturity (adulthood) Log rank analysis of survivorship was assessed using the survival (<a id="xref-ref-45-1" class="xref-bibr" href="#ref-45">45</a>, <a id="xref-ref-46-1" class="xref-bibr" href="#ref-46">46</a>) and survminer (<a id="xref-ref-47-1" class="xref-bibr" href="#ref-47">47</a>) packages in R.</p><p id="p-23">Egg production rate (EPR) and hatching frequency (HF) were assessed with 36 individual mate pairs of newly matured adults per treatment (12 pairs per food concentration × 2 food concentrations = 24 pairs per treatment) divided evenly between four replicates within a treatment. Adults were incubated in 25 mL petri dishes (Fisher Scientific, Waltham, MA, USA) over three days in the same incubators and plexiglass enclosures described above. After the initial three-day incubation, adults were assessed for survival and removed to avoid cannibalization of eggs. Eggs were allowed to hatch over a subsequent three-day incubation. Food media was prepared as described for survivorship and replaced daily during egg laying to ensure accurate food concentrations were near saturation and not reduced due to daily grazing (<a id="xref-ref-48-1" class="xref-bibr" href="#ref-48">48</a>). Lids of petri dishes were left off-center to allow for full contact with the atmosphere and diffusion of CO<sub>2</sub>. Plates with dead males were still evaluated for EPR, but not HF. Plates with dead females were not evaluated. After the hatching period, plates were preserved with non-acid Lugol’s solution and quantified. Per capita EPR was calculated as (E<sub>u</sub>+E<sub>h</sub>)/t where E<sub>u</sub> represents unhatched eggs, E<sub>h</sub> represents hatched eggs (nauplii), and t represents egg laying time. Hatching frequency was calculated as E<sub>h</sub>/(E<sub>u</sub>+E<sub>h</sub>). All statistical pairwise comparisons were completed using the stats package in R (<a id="xref-ref-49-1" class="xref-bibr" href="#ref-49">49</a>).</p><p id="p-24">The Malthusian parameter was calculated as the log of the dominant eigenvalue of an assembled projected age-structured Leslie Matrix constructed from survivorship and fecundity data. Leslie Matrices are used to estimate population growth. Briefly, day-specific probabilities of survivorship are calculated from day-specific survivorship as where <em>l<sub>x</sub></em> represents the individuals on day <em>x</em> and <em>l<sub>x-1</sub></em> represents the number of individuals on day <em>x-1</em>. Probabilities of survivorship on day 1 are assumed to be 100% or a proportion of 1. Per capita EPR and HF are calculated as described above, with fecundity rates equaling the product of EPR and HF. Fecundity rates are assigned to the last day of survivorship as all individuals have reached sexual maturity by this point. Matrices were constructed for each mate pair assessed for fecundity for each replicate within each treatment using the corresponding survivorship data for the respective replicate (3 mate pairs × 4 replicates = 12 Matrices per treatment per food concentration). For example, survivorship data from food-replete GH Replicate 1 was paired with three corresponding fecundity values to yield three separate matrices of which the Malthusian parameters were calculated. Statistical pairwise comparisons were completed using the stats package in R with a two-way analysis of variance model (Malthusian ~ treatment * food) to compare treatment and food interaction effects (<a id="xref-ref-49-2" class="xref-bibr" href="#ref-49">49</a>).</p></div><div id="sec-7" class="subsection"><h3>Genomics</h3><p id="p-25">RNA was extracted using TRIzol reagent (Invitrogen, Carlsbad, CA, USA) and purified with Qiagen RNeasy spin columns (Qiagen, Germantown, MD, USA). RNAseq libraries were prepared by Novogene (Sacramento, CA, USA) and sequenced with 150 bp paired end reads on an Illumina NovaSeq 600, generating 1.29 billion reads. Raw reads were trimmed for quality and adapter contamination using Trimmomatic V0.36 (<a id="xref-ref-50-1" class="xref-bibr" href="#ref-50">50</a>) where leading and trailing quality was set at 2, sliding window length was 4 with a quality of 2, and minimum length was 31.</p><p id="p-26">To quantify genetic variation, SuperTranscripts were generated from the reference transcriptome (<a id="xref-ref-51-1" class="xref-bibr" href="#ref-51">51</a>) using the Trinity_gene_splice_modeler.py command in the Trinity pipeline (<a id="xref-ref-52-1" class="xref-bibr" href="#ref-52">52</a>). SuperTranscripts act as a reference by collapsing isoforms into a single transcript, which enables variant detection without a reference genome (<a id="xref-ref-53-1" class="xref-bibr" href="#ref-53">53</a>). Trimmed reads were aligned to the SuperTranscripts using bwa mem (<a id="xref-ref-54-1" class="xref-bibr" href="#ref-54">54</a>). After filtering, we identify 322,595 variant sites among all replicates.</p><p id="p-27">Variants were called using VarScan v2.3 (<a id="xref-ref-55-1" class="xref-bibr" href="#ref-55">55</a>) (2 alternate reads to call a variant, p-value of 0.1, minimum coverage of 30x, minimum variant allele frequency of 0.01). Following this, variants were stringently filtered to include no missing data, only sites where at least 4 samples were called as variable and each with a minor allele frequency of at least 0.025 (minimum of 1 heterozygous individual), any sites with depth greater than 3x the median (median=132x), and sites with a minimum coverage of 40x per sample (2 reads/diploid individual). This reduced called variants from an initial 5,547,802 to 322,595.</p><p id="p-28">Gene expression was quantified at the gene level as recommended (<a id="xref-ref-56-1" class="xref-bibr" href="#ref-56">56</a>) using Salmon v0.10.2(<a id="xref-ref-57-1" class="xref-bibr" href="#ref-57">57</a>) and transcript abundances were converted to gene-level counts using the tximport package in R (<a id="xref-ref-49-3" class="xref-bibr" href="#ref-49">49</a>). The reference transcriptome was indexed using quasi mapping with a k-mer length of 31 and transcripts were quantified while correcting for sequence specific bias and GC bias. We were interested in assessing expression patterns between treatments with each generation and how these patterns changed through time and, therefore, analyzed each generation separately. Genes with low expression (fewer than 10 counts in more than 90% of the samples) were removed from the dataset. This left 23 324, 24 882, and 24 132 genes for F1, F2, and F3, respectively. Counts were normalized and log transformed using the rlog command in DESeq2 (<a id="xref-ref-58-1" class="xref-bibr" href="#ref-58">58</a>).</p></div><div id="sec-8" class="subsection"><h3>Gene expression change across generations</h3><p id="p-29">Using the filtered, normalized, and transformed expression data from DeSeq2, we used Discriminant analysis of principal components (DAPC) to quantify the shifts in gene expression in transplanted lines across generations using the adegenet package in R (<a id="xref-ref-59-1" class="xref-bibr" href="#ref-59">59</a>). Discriminant functions for each generation were first generated using the non-transplanted lines to identify genes consistently differentially expressed. We fit transplanted lines to this discriminant function space and used MCMCglmm models in R (<a id="xref-ref-60-1" class="xref-bibr" href="#ref-60">60</a>) to model the effect of line origin and transplant on movement in discriminant function space. We generated 2500 posterior estimates and quantified the difference in the transplant effect for each line by subtracting the absolute difference between these estimates. This can be viewed as the difference between the lengths of the lines representing average shifts in <a id="xref-fig-1-5" class="xref-fig" href="#F1">Fig. 1.</a> These differences were used to generate a 95% credible interval and the proportion of positive or negative values were considered a p-value for the difference in magnitude of the effect of transplant on each line in discriminant function space. DESeq2 was used to identify the specific genes that were adaptive differentially expressed between non-transplanted AM and GH lines at each generation. For each generation, we again required at least 10 counts in 90% of samples. Differentially expressed genes were identified using the model ~Line + Treatment + Line:Treatment and any contrasts between AM and GH in their home environment were considered significant when adjusted p-values were &lt; 0.05. For each of these significantly differentially expressed genes, we assessed the degree to which transplanted lines converged on the adaptive transcriptional profile of each environment by plotting the log<sub>2</sub> fold change in expression between lines in their home environment versus the change following transplant and calculating the correlation between the differences in expression.</p></div><div id="sec-9" class="subsection"><h3>Allele frequency changes</h3><p id="p-30">We used DAPC to identify divergence in allele frequency between the AM and GH lines. The same approach as for gene expression (above) was taken where we generated the DAPC with lines in their home environment and then fit transplanted lines to discriminant function space. We then used Cochran-Mantel-Haenszel (CMH) tests to identify specific SNPs that were consistently diverged between the AM and GH lines and represent the most likely targets of adaptation. This approach looks for consistent changes in allele frequencies across all replicates and is a common and powerful technique in experimental evolution (<a id="xref-ref-61-1" class="xref-bibr" href="#ref-61">61</a>, <a id="xref-ref-62-1" class="xref-bibr" href="#ref-62">62</a>). We calculate CMH p-values for each set of AM and GH lines in their home conditions at each generation (separate tests for each generation). Significance thresholds were defined with Bonferroni corrections using the total number of SNPs multiplied by three, representing the total number of tests conducted. The SNPs that were identified in all three generations were considered adaptive. This approach assumes that AM and GH lines have reached their phenotypic optimum and any additional evolution in the F2 or F3 generations is a product of drift, an assumption that is reasonable given the preceding 20 generations of selection.</p><p id="p-31">We assessed the degree of adaptive evolution for each transplanted line across the generations by quantifying the change in allele frequencies of adaptive loci in transplanted lines. We consider the adaptive difference in allele frequencies as the difference between mean frequencies of AM and GH lines (for loci identified with the CMH test) and assess the convergence of transplanted lines on this adaptive difference (i.e., do AM in GH frequencies converge on the GH allele frequency?).</p><p id="p-32">We estimate genetic diversity (π) for each replicate using Popoolation (<a id="xref-ref-63-1" class="xref-bibr" href="#ref-63">63</a>) with 100 bp nonoverlapping sliding windows. To quantify if <em>π</em> was differentially lost in any treatment, come compare difference in median <em>π</em> were using an Anova with a Tukey post-hoc test. We next asked if regions containing adaptively divergent loci between non-transplanted GH and AM lines lost diversity at a different rate than regions containing only neutral variants. We calculated the change in <em>π</em> for each replicate and compare the change in <em>π</em> between GH in AM and AM in GH conditions using Wilcoxon Rank Sum tests with bonferroni corrections. Gene ontology enrichment was performed with GO Mann-Whitney U (<a id="xref-ref-64-1" class="xref-bibr" href="#ref-64">64</a>), which requires no significance threshold, but takes the change in <em>π</em> for each group and asks if any functional category falls towards the tails of this distribution (in our case, one-sided to identify low values). We use this same approach for gene expression and allele frequencies, but with contribution scores from the DAPC as input. For allele frequencies, contribution scores were averaged for each transcript.</p><p id="p-33">Finally, we tested for overlap between the allelic and gene expression results to determine if the genes in each were significantly correlated. We correlated the relationship between the −log<sub>10</sub> of the CMH p-values for the allele frequencies and the DAPC contribution score for gene expression. This analysis showed the two sets were distinct and the variation explained by each was less than 1% at each generation. This indicates that there was minimal bias in estimating allele frequencies due to differentially expressed genes.</p></div></div><div class="section" id="sec-10"><h2 class="">Funding</h2><p id="p-35">This work was funded by the National Science Foundation grants to M.H.P. (OCE 1559075) and H.G.D, H.B. and M.F. (OCE 1559180);</p></div><div class="section" id="sec-11"><h2 class="">Author contributions</h2><p id="p-36">Conceptualization and experimental design: all authors; Experimental execution: JAD; Data collection and analysis: RSB and JAD; Writing-original draft: RSB and MHP; Writing-review and editing: all authors;</p></div><div class="section" id="sec-12"><h2 class="">Competing interests</h2><p id="p-37">Authors declare no competing interests;</p></div><div class="section" id="sec-13"><h2 class="">Data and materials availability</h2><p id="p-38">Raw sequence data is available at NCBI BioProject PRJNA555881. Life history data are available as a supplemental file and allele frequency and gene expression table is available on Figshare (<a href="https://doi.org/10.6084/m9.figshare.10301690">https://doi.org/10.6084/m9.figshare.10301690</a>). Correspondence and requests for materials should be addressed to RSB (<span class="em-link"><span class="em-addr">reid.brennan{at}uvm.edu</span></span>) or MHP (<span class="em-link"><span class="em-addr">mpespeni{at}uvm.edu</span></span>). Code to reproduce all analyses can be found at: <a href="https://github.com/PespeniLab/tonsa_reciprocal_transplant">https://github.com/PespeniLab/tonsa_reciprocal_transplant</a></p></div><div class="section" id="sec-14"><h2 class="">Supplemental Materials</h2><p id="p-39"><strong>Table S1:</strong> Gene ontology enrichment results for gene expression.</p><p id="p-40"><strong>Table S2:</strong> Gene ontology enrichment results for allele frequency data</p><p id="p-41"><strong>Table S3:</strong> Gene ontology enrichment for the change in genetic diversity following transplant of GH to AM conditions.</p><p id="p-42"><strong>Table S4</strong>: Summary table of gene expression. AAAA = ambient line in ambient conditions; HHHH = greenhouse in greenhouse conditions; HHAA = ambient in greenhouse conditions; AAHH = greenhouse in ambient conditions. DAPC results are the contribution score for each gene for the DAPC loading. P-values and fold change were calculated with DESeq2.</p><p id="p-43"><strong>Table S5</strong>: Raw survivorship data from <a id="xref-fig-2-10" class="xref-fig" href="#F2">figure 2</a>.</p><p id="p-44"><strong>Table S6</strong>: Raw fecundity data from <a id="xref-fig-2-11" class="xref-fig" href="#F2">figure 2</a>.</p><div id="F5" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/01/30/2020.01.29.925396/F5.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Relationship between gene expression and allele frequency divergence. Y-axis is the &#x2212;log10 of p-values from the CMH between AM and GH in their home conditions. X-axis corresponds to the contribution scores from the DAPC separating AM and GH in home conditions. For all cases, the variance explained is low, indicating that gene expression and allele frequency divergence is largely distinct." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1553945654" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Relationship between gene expression and allele frequency divergence. Y-axis is the &#x2212;log10 of p-values from the CMH between AM and GH in their home conditions. X-axis corresponds to the contribution scores from the DAPC separating AM and GH in home conditions. For all cases, the variance explained is low, indicating that gene expression and allele frequency divergence is largely distinct.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure S1:" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2020/01/30/2020.01.29.925396/F5.medium.gif" width="440" height="150"/><noscript><img class="highwire-fragment fragment-image" alt="Figure S1:" src="https://www.biorxiv.org/content/biorxiv/early/2020/01/30/2020.01.29.925396/F5.medium.gif" width="440" height="150"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/01/30/2020.01.29.925396/F5.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure S1:" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/01/30/2020.01.29.925396/F5.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure S1:</span> <p id="p-45" class="first-child">Relationship between gene expression and allele frequency divergence. Y-axis is the −log10 of p-values from the CMH between AM and GH in their home conditions. X-axis corresponds to the contribution scores from the DAPC separating AM and GH in home conditions. For all cases, the variance explained is low, indicating that gene expression and allele frequency divergence is largely distinct.</p><div class="sb-div caption-clear"></div></div></div><div id="F6" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/01/30/2020.01.29.925396/F6.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Genome-wide variation in (A) gene expression and (B) allele frequencies across generations. The x-axis shows the discriminant function space that maximizes differences between lines in their home environments. Shape indicates selection line, color indicates treatment condition. The arrows above the points show the mean change for each line after transplant. Gene expression profiles show the same degree of movement for each line following transplant (P &gt; 0.05). The GH line shifts allele frequency to a greater degree than the AM line by generation F3 (P = 0.02). These results show that the GHAM lines matched the adaptive ambient transcriptional profile with concurrent shifts in allele frequencies. Conversely AMGH lines matched the adaptive greenhouse transcriptional profile with plasticity and no changes in allele frequency. These results reinforce the conclusions that over 20 generations of experimental evolution, ambient lines had maintained physiological plasticity to transcriptionally respond to greenhouse conditions, while greenhouse lines lost ancestral plasticity that was compensated for by adaptive evolution that likely enabled the shift of transcriptional profiles to match the environment." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1553945654" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Genome-wide variation in (A) gene expression and (B) allele frequencies across generations. The x-axis shows the discriminant function space that maximizes differences between lines in their home environments. Shape indicates selection line, color indicates treatment condition. The arrows above the points show the mean change for each line after transplant. Gene expression profiles show the same degree of movement for each line following transplant (P &gt; 0.05). The GH line shifts allele frequency to a greater degree than the AM line by generation F3 (P = 0.02). These results show that the GH&lt;sub&gt;AM&lt;/sub&gt; lines matched the adaptive ambient transcriptional profile with concurrent shifts in allele frequencies. Conversely AM&lt;sub&gt;GH&lt;/sub&gt; lines matched the adaptive greenhouse transcriptional profile with plasticity and no changes in allele frequency. These results reinforce the conclusions that over 20 generations of experimental evolution, ambient lines had maintained physiological plasticity to transcriptionally respond to greenhouse conditions, while greenhouse lines lost ancestral plasticity that was compensated for by adaptive evolution that likely enabled the shift of transcriptional profiles to match the environment.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure S2:" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2020/01/30/2020.01.29.925396/F6.medium.gif" width="440" height="270"/><noscript><img class="highwire-fragment fragment-image" alt="Figure S2:" src="https://www.biorxiv.org/content/biorxiv/early/2020/01/30/2020.01.29.925396/F6.medium.gif" width="440" height="270"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/01/30/2020.01.29.925396/F6.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure S2:" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/01/30/2020.01.29.925396/F6.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure S2:</span> <p id="p-46" class="first-child">Genome-wide variation in (A) gene expression and (B) allele frequencies across generations. The x-axis shows the discriminant function space that maximizes differences between lines in their home environments. Shape indicates selection line, color indicates treatment condition. The arrows above the points show the mean change for each line after transplant. Gene expression profiles show the same degree of movement for each line following transplant (P &gt; 0.05). The GH line shifts allele frequency to a greater degree than the AM line by generation F3 (P = 0.02). These results show that the GH<sub>AM</sub> lines matched the adaptive ambient transcriptional profile with concurrent shifts in allele frequencies. Conversely AM<sub>GH</sub> lines matched the adaptive greenhouse transcriptional profile with plasticity and no changes in allele frequency. These results reinforce the conclusions that over 20 generations of experimental evolution, ambient lines had maintained physiological plasticity to transcriptionally respond to greenhouse conditions, while greenhouse lines lost ancestral plasticity that was compensated for by adaptive evolution that likely enabled the shift of transcriptional profiles to match the environment.</p><div class="sb-div caption-clear"></div></div></div></div><div class="section ack" id="ack-1"><h2 class="">Acknowledgments</h2><p id="p-34">We thank Lydia Norton and Gihong Park for culture maintenance and experimental help. RSB appreciates the sponsorship from Maile Neel as a Visiting Research Scientist in the Department of Plant Science &amp; Landscape Architecture at the University of Maryland College Park.</p></div><div class="section ref-list" id="ref-list-1"><h2 class="">References</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label">1.</span><a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference 1. in text" id="ref-1">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.01.29.925396v1.1" data-doi="10.1126/science.1237123"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">N. S.</span> <span class="cit-name-surname">Diffenbaugh</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">C. B.</span> <span class="cit-name-surname">Field</span></span>, <span class="cit-article-title">Changes in ecologically critical terrestrial climate conditions</span>. <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">341</span>, <span class="cit-fpage">486</span>–<span class="cit-lpage">492</span> (<span class="cit-pub-date">2013</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DDiffenbaugh%26rft.auinit1%253DN.%2BS.%26rft.volume%253D341%26rft.issue%253D6145%26rft.spage%253D486%26rft.epage%253D492%26rft.atitle%253DChanges%2Bin%2BEcologically%2BCritical%2BTerrestrial%2BClimate%2BConditions%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1237123%26rft_id%253Dinfo%253Apmid%252F23908225%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEyOiIzNDEvNjE0NS80ODYiO3M6NDoiYXRvbSI7czo0ODoiL2Jpb3J4aXYvZWFybHkvMjAyMC8wMS8zMC8yMDIwLjAxLjI5LjkyNTM5Ni5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">2.</span><a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference 2. in text" id="ref-2">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.01.29.925396v1.2" data-doi="10.1038/nature02121"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">C. D.</span> <span class="cit-name-surname">Thomas</span></span>, <span class="cit-etal">et al.</span>, <span class="cit-article-title">Extinction risk from climate change</span>. <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">427</span>, <span class="cit-fpage">145</span>–<span class="cit-lpage">148</span> (<span class="cit-pub-date">2004</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DThomas%26rft.auinit1%253DC.%2BD.%26rft.volume%253D427%26rft.issue%253D6970%26rft.spage%253D145%26rft.epage%253D148%26rft.atitle%253DExtinction%2Brisk%2Bfrom%2Bclimate%2Bchange.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature02121%26rft_id%253Dinfo%253Apmid%252F14712274%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature02121&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=14712274&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F01%2F30%2F2020.01.29.925396.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000187863900033&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">3.</span><a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference 3. in text" id="ref-3">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.01.29.925396v1.3" data-doi="10.1371/journal.pbio.1000357"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">L.-M.</span> <span class="cit-name-surname">Chevin</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">R.</span> <span class="cit-name-surname">Lande</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">G. M.</span> <span class="cit-name-surname">Mace</span></span>, <span class="cit-article-title">Adaptation, plasticity, and extinction in a changing environment: towards a predictive theory</span>. <abbr class="cit-jnl-abbrev">PLoS Biol.</abbr> <span class="cit-vol">8</span>, <span class="cit-fpage">e1000357</span> (<span class="cit-pub-date">2010</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2Bbiology%26rft.stitle%253DPLoS%2BBiol%26rft.aulast%253DChevin%26rft.auinit1%253DL.%2BM.%26rft.volume%253D8%26rft.issue%253D4%26rft.spage%253De1000357%26rft.epage%253De1000357%26rft.atitle%253DAdaptation%252C%2Bplasticity%252C%2Band%2Bextinction%2Bin%2Ba%2Bchanging%2Benvironment%253A%2Btowards%2Ba%2Bpredictive%2Btheory.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pbio.1000357%26rft_id%253Dinfo%253Apmid%252F20463950%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pbio.1000357&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20463950&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F01%2F30%2F2020.01.29.925396.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">4.</span><a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference 4. in text" id="ref-4">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.01.29.925396v1.4" data-doi="10.1111/eva.12137"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Merilä</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">A. P.</span> <span class="cit-name-surname">Hendry</span></span>, <span class="cit-article-title">Climate change, adaptation, and phenotypic plasticity: the problem and the evidence</span>. <abbr class="cit-jnl-abbrev">Evol. Appl.</abbr> <span class="cit-vol">7</span>, <span class="cit-fpage">1</span>–<span class="cit-lpage">14</span> (<span class="cit-pub-date">2014</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEvol.%2BAppl.%26rft.volume%253D7%26rft.spage%253D1%26rft_id%253Dinfo%253Adoi%252F10.1111%252Feva.12137%26rft_id%253Dinfo%253Apmid%252F24454544%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/eva.12137&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24454544&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F01%2F30%2F2020.01.29.925396.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">5.</span><a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference 5. in text" id="ref-5">↵</a><div class="cit ref-cit ref-book" id="cit-2020.01.29.925396v1.5"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">M. J.</span> <span class="cit-name-surname">West-Eberhard</span></span>, <span class="cit-source">Developmental Plasticity and Evolution</span> (<span class="cit-publ-name">Oxford University Press</span>, <span class="cit-pub-date">2003</span>).</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">6.</span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-2020.01.29.925396v1.6" data-doi="10.1006/tpbi.2000.1484"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">L. W.</span> <span class="cit-name-surname">Ancel</span></span>, <span class="cit-article-title">Undermining the Baldwin expediting effect: does phenotypic plasticity accelerate evolution?</span> <abbr class="cit-jnl-abbrev">Theor. Popul. Biol.</abbr> <span class="cit-vol">58</span>, <span class="cit-fpage">307</span>–<span class="cit-lpage">319</span> (<span class="cit-pub-date">2000</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTheoretical%2Bpopulation%2Bbiology%26rft.stitle%253DTheor%2BPopul%2BBiol%26rft.aulast%253DAncel%26rft.auinit1%253DL.%2BW.%26rft.volume%253D58%26rft.issue%253D4%26rft.spage%253D307%26rft.epage%253D319%26rft.atitle%253DUndermining%2Bthe%2BBaldwin%2Bexpediting%2Beffect%253A%2Bdoes%2Bphenotypic%2Bplasticity%2Baccelerate%2Bevolution%253F%26rft_id%253Dinfo%253Adoi%252F10.1006%252Ftpbi.2000.1484%26rft_id%253Dinfo%253Apmid%252F11162789%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1006/tpbi.2000.1484&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=11162789&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F01%2F30%2F2020.01.29.925396.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000166772300004&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">7.</span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-2020.01.29.925396v1.7" data-doi="10.1098/rspb.2003.2372"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">T. D.</span> <span class="cit-name-surname">Price</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Qvarnström</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">D. E.</span> <span class="cit-name-surname">Irwin</span></span>, <span class="cit-article-title">The role of phenotypic plasticity in driving genetic evolution</span>. <abbr class="cit-jnl-abbrev">Proc. Biol. Sci.</abbr> <span class="cit-vol">270</span>, <span class="cit-fpage">1433</span>–<span class="cit-lpage">1440</span> (<span class="cit-pub-date">2003</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProceedings%2Bof%2Bthe%2BRoyal%2BSociety%2BB%253A%2BBiological%2BSciences%26rft.stitle%253DProc%2BR%2BSoc%2BB%26rft.aulast%253DPrice%26rft.auinit1%253DT.%2BD.%26rft.volume%253D270%26rft.issue%253D1523%26rft.spage%253D1433%26rft.epage%253D1440%26rft.atitle%253DThe%2Brole%2Bof%2Bphenotypic%2Bplasticity%2Bin%2Bdriving%2Bgenetic%2Bevolution%26rft_id%253Dinfo%253Adoi%252F10.1098%252Frspb.2003.2372%26rft_id%253Dinfo%253Apmid%252F12965006%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1098/rspb.2003.2372&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=12965006&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F01%2F30%2F2020.01.29.925396.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000184272500001&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">8.</span><a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference 8. in text" id="ref-8">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.01.29.925396v1.8"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">C. K.</span> <span class="cit-name-surname">Ghalambor</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">J. K.</span> <span class="cit-name-surname">McKAY</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">S. P.</span> <span class="cit-name-surname">Carroll</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">D. N.</span> <span class="cit-name-surname">Reznick</span></span>, <span class="cit-article-title">Adaptive versus non-adaptive phenotypic plasticity and the potential for contemporary adaptation in new environments</span>. <abbr class="cit-jnl-abbrev">Funct. Ecol.</abbr> <span class="cit-vol">21</span>, <span class="cit-fpage">394</span>–<span class="cit-lpage">407</span> (<span class="cit-pub-date">2007</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DFunct.%2BEcol.%26rft.volume%253D21%26rft.spage%253D394%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">9.</span><a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference 9. in text" id="ref-9">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.01.29.925396v1.9" data-doi="10.1038/nature15256"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">C. K.</span> <span class="cit-name-surname">Ghalambor</span></span>, <span class="cit-etal">et al.</span>, <span class="cit-article-title">Non-adaptive plasticity potentiates rapid adaptive evolution of gene expression in nature</span>. <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">525</span>, <span class="cit-fpage">372</span>–<span class="cit-lpage">375</span> (<span class="cit-pub-date">2015</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.volume%253D525%26rft.spage%253D372%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature15256%26rft_id%253Dinfo%253Apmid%252F26331546%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature15256&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=26331546&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F01%2F30%2F2020.01.29.925396.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">10.</span><a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference 10. in text" id="ref-10">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.01.29.925396v1.10" data-doi="10.1111/j.1420-9101.2009.01754.x"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">R.</span> <span class="cit-name-surname">Lande</span></span>, <span class="cit-article-title">Adaptation to an extraordinary environment by evolution of phenotypic plasticity and genetic assimilation</span>. <abbr class="cit-jnl-abbrev">J. Evol. Biol.</abbr> <span class="cit-vol">22</span>, <span class="cit-fpage">1435</span>–<span class="cit-lpage">1446</span> (<span class="cit-pub-date">2009</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2Bevolutionary%2Bbiology%26rft.stitle%253DJ%2BEvol%2BBiol%26rft.aulast%253DLande%26rft.auinit1%253DR.%26rft.volume%253D22%26rft.issue%253D7%26rft.spage%253D1435%26rft.epage%253D1446%26rft.atitle%253DAdaptation%2Bto%2Ban%2Bextraordinary%2Benvironment%2Bby%2Bevolution%2Bof%2Bphenotypic%2Bplasticity%2Band%2Bgenetic%2Bassimilation.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1420-9101.2009.01754.x%26rft_id%253Dinfo%253Apmid%252F19467134%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1420-9101.2009.01754.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19467134&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F01%2F30%2F2020.01.29.925396.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000267129500008&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">11.</span><a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference 11. in text" id="ref-11">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.01.29.925396v1.11" data-doi="10.1073/pnas.1101766108"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">Rahmstorf</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">D.</span> <span class="cit-name-surname">Coumou</span></span>, <span class="cit-article-title">Increase of extreme events in a warming world</span>. <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. U. S. A.</abbr> <span class="cit-vol">108</span>, <span class="cit-fpage">17905</span>–<span class="cit-lpage">17909</span> (<span class="cit-pub-date">2011</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BU.%2BS.%2BA.%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.1101766108%26rft_id%253Dinfo%253Apmid%252F22025683%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMjoiMTA4LzQ0LzE3OTA1IjtzOjQ6ImF0b20iO3M6NDg6Ii9iaW9yeGl2L2Vhcmx5LzIwMjAvMDEvMzAvMjAyMC4wMS4yOS45MjUzOTYuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">12.</span><a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference 12. in text" id="ref-12">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.01.29.925396v1.12"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">G. N.</span> <span class="cit-name-surname">Somero</span></span>, <span class="cit-article-title">The physiology of climate change: how potentials for acclimatization and genetic adaptation will determine “winners” and “losers.”</span> <abbr class="cit-jnl-abbrev">J. Exp. Biol</abbr>. (<span class="cit-pub-date">2010</span>).</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">13.</span><a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference 13. in text" id="ref-13">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.01.29.925396v1.13" data-doi="10.1038/nature09670"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">A. A.</span> <span class="cit-name-surname">Hoffmann</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">C. M.</span> <span class="cit-name-surname">Sgrò</span></span>, <span class="cit-article-title">Climate change and evolutionary adaptation</span>. <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">470</span>, <span class="cit-fpage">479</span>–<span class="cit-lpage">485</span> (<span class="cit-pub-date">2011</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DHoffmann%26rft.auinit1%253DA.%2BA.%26rft.volume%253D470%26rft.issue%253D7335%26rft.spage%253D479%26rft.epage%253D485%26rft.atitle%253DClimate%2Bchange%2Band%2Bevolutionary%2Badaptation.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature09670%26rft_id%253Dinfo%253Apmid%252F21350480%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature09670&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21350480&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F01%2F30%2F2020.01.29.925396.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000287652900031&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">14.</span><a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference 14. in text" id="ref-14">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.01.29.925396v1.14" data-doi="10.1126/science.aam5512"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">S. C.</span> <span class="cit-name-surname">Campbell-Staton</span></span>, <span class="cit-etal">et al.</span>, <span class="cit-article-title">Winter storms drive rapid phenotypic, regulatory, and genomic shifts in the green anole lizard</span>. <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">357</span>, <span class="cit-fpage">495</span>–<span class="cit-lpage">498</span> (<span class="cit-pub-date">2017</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DCampbell-Staton%26rft.auinit1%253DS.%2BC.%26rft.volume%253D357%26rft.issue%253D6350%26rft.spage%253D495%26rft.epage%253D498%26rft.atitle%253DWinter%2Bstorms%2Bdrive%2Brapid%2Bphenotypic%252C%2Bregulatory%252C%2Band%2Bgenomic%2Bshifts%2Bin%2Bthe%2Bgreen%2Banole%2Blizard%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.aam5512%26rft_id%253Dinfo%253Apmid%252F28774927%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEyOiIzNTcvNjM1MC80OTUiO3M6NDoiYXRvbSI7czo0ODoiL2Jpb3J4aXYvZWFybHkvMjAyMC8wMS8zMC8yMDIwLjAxLjI5LjkyNTM5Ni5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">15.</span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-2020.01.29.925396v1.15" data-doi="10.1126/science.aah4993"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">N. M.</span> <span class="cit-name-surname">Reid</span></span>, <span class="cit-etal">et al.</span>, <span class="cit-article-title">The genomic landscape of rapid repeated evolutionary adaptation to toxic pollution in wild fish</span>. <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">354</span>, <span class="cit-fpage">1305</span>–<span class="cit-lpage">1308</span> (<span class="cit-pub-date">2016</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DReid%26rft.auinit1%253DN.%2BM.%26rft.volume%253D354%26rft.issue%253D6317%26rft.spage%253D1305%26rft.epage%253D1308%26rft.atitle%253DThe%2Bgenomic%2Blandscape%2Bof%2Brapid%2Brepeated%2Bevolutionary%2Badaptation%2Bto%2Btoxic%2Bpollution%2Bin%2Bwild%2Bfish%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.aah4993%26rft_id%253Dinfo%253Apmid%252F27940876%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEzOiIzNTQvNjMxNy8xMzA1IjtzOjQ6ImF0b20iO3M6NDg6Ii9iaW9yeGl2L2Vhcmx5LzIwMjAvMDEvMzAvMjAyMC4wMS4yOS45MjUzOTYuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">16.</span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-2020.01.29.925396v1.16" data-doi="10.1126/science.aav3824"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">R. D. H.</span> <span class="cit-name-surname">Barrett</span></span>, <span class="cit-etal">et al.</span>, <span class="cit-article-title">Linking a mutation to survival in wild mice</span>. <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">363</span>, <span class="cit-fpage">499</span>–<span class="cit-lpage">504</span> (<span class="cit-pub-date">2019</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DBarrett%26rft.auinit1%253DR.%2BD.%2BH.%26rft.volume%253D363%26rft.issue%253D6426%26rft.spage%253D499%26rft.epage%253D504%26rft.atitle%253DLinking%2Ba%2Bmutation%2Bto%2Bsurvival%2Bin%2Bwild%2Bmice%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.aav3824%26rft_id%253Dinfo%253Apmid%252F30705186%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEyOiIzNjMvNjQyNi80OTkiO3M6NDoiYXRvbSI7czo0ODoiL2Jpb3J4aXYvZWFybHkvMjAyMC8wMS8zMC8yMDIwLjAxLjI5LjkyNTM5Ni5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">17.</span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-2020.01.29.925396v1.17"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">H. C.</span> <span class="cit-name-surname">Bumpus</span></span>, <abbr class="cit-jnl-abbrev">The Elimination of the Unfit as Illustrated by the Introduced Sparrow, Passer Domesticus:(a Fourth Contribution to the Study of Variation)</abbr> (<span class="cit-auth cit-collab">Gin</span>, <span class="cit-pub-date">1899</span>).</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">18.</span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-2020.01.29.925396v1.18" data-doi="10.1126/science.aaw7271"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">N. O.</span> <span class="cit-name-surname">Therkildsen</span></span>, <span class="cit-etal">et al.</span>, <span class="cit-article-title">Contrasting genomic shifts underlie parallel phenotypic evolution in response to fishing</span>. <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">365</span>, <span class="cit-fpage">487</span>–<span class="cit-lpage">490</span> (<span class="cit-pub-date">2019</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DTherkildsen%26rft.auinit1%253DN.%2BO.%26rft.volume%253D365%26rft.issue%253D6452%26rft.spage%253D487%26rft.epage%253D490%26rft.atitle%253DContrasting%2Bgenomic%2Bshifts%2Bunderlie%2Bparallel%2Bphenotypic%2Bevolution%2Bin%2Bresponse%2Bto%2Bfishing%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.aaw7271%26rft_id%253Dinfo%253Apmid%252F31371613%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEyOiIzNjUvNjQ1Mi80ODciO3M6NDoiYXRvbSI7czo0ODoiL2Jpb3J4aXYvZWFybHkvMjAyMC8wMS8zMC8yMDIwLjAxLjI5LjkyNTM5Ni5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">19.</span><a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference 19. in text" id="ref-19">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.01.29.925396v1.19"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">R. S.</span> <span class="cit-name-surname">Brennan</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">A. D.</span> <span class="cit-name-surname">Garrett</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">K. E.</span> <span class="cit-name-surname">Huber</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">H.</span> <span class="cit-name-surname">Hargarten</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">M. H.</span> <span class="cit-name-surname">Pespeni</span></span>, <span class="cit-article-title">Rare genetic variation and balanced polymorphisms are important for survival in global change conditions</span>. <abbr class="cit-jnl-abbrev">Proceedings of the Royal Society B: Biological Sciences</abbr> <span class="cit-vol">286</span>, <span class="cit-fpage">20190943</span> (<span class="cit-pub-date">2019</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProceedings%2Bof%2Bthe%2BRoyal%2BSociety%2BB%253A%2BBiological%2BSciences%26rft.volume%253D286%26rft.spage%253D20190943%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">20.</span><a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference 20. in text" id="ref-20">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.01.29.925396v1.20" data-doi="10.2307/2408649"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">Via</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">R.</span> <span class="cit-name-surname">Lande</span></span>, <span class="cit-article-title">Genotype-Environment Interaction and the Evolution of Phenotypic Plasticity</span>. <abbr class="cit-jnl-abbrev">Evolution</abbr> <span class="cit-vol">39</span>, <span class="cit-fpage">505</span>–<span class="cit-lpage">522</span> (<span class="cit-pub-date">1985</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEvolution%26rft.volume%253D39%26rft.spage%253D505%26rft_id%253Dinfo%253Adoi%252F10.2307%252F2408649%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2307/2408649&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=A1985AJY2600003&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">21.</span><a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference 21. in text" id="ref-21">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.01.29.925396v1.21" data-doi="10.1111/j.1558-5646.2009.00875.x"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">L.-M.</span> <span class="cit-name-surname">Chevin</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">R.</span> <span class="cit-name-surname">Lande</span></span>, <span class="cit-article-title">When do adaptive plasticity and genetic evolution prevent extinction of a density-regulated population?</span> <abbr class="cit-jnl-abbrev">Evolution</abbr> <span class="cit-vol">64</span>, <span class="cit-fpage">1143</span>–<span class="cit-lpage">1150</span> (<span class="cit-pub-date">2010</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEvolution%26rft.stitle%253DEvolution%26rft.aulast%253DChevin%26rft.auinit1%253DL.%2BM.%26rft.volume%253D64%26rft.issue%253D4%26rft.spage%253D1143%26rft.epage%253D1150%26rft.atitle%253DWhen%2Bdo%2Badaptive%2Bplasticity%2Band%2Bgenetic%2Bevolution%2Bprevent%2Bextinction%2Bof%2Ba%2Bdensity-regulated%2Bpopulation%253F%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1558-5646.2009.00875.x%26rft_id%253Dinfo%253Apmid%252F19863583%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1558-5646.2009.00875.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19863583&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F01%2F30%2F2020.01.29.925396.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000276035000023&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">22.</span><a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference 22. in text" id="ref-22">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.01.29.925396v1.22" data-doi="10.1371/journal.pbio.0060325"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">S. E.</span> <span class="cit-name-surname">Williams</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">L. P.</span> <span class="cit-name-surname">Shoo</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">J. L.</span> <span class="cit-name-surname">Isaac</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">A. A.</span> <span class="cit-name-surname">Hoffmann</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">G.</span> <span class="cit-name-surname">Langham</span></span>, <span class="cit-article-title">Towards an integrated framework for assessing the vulnerability of species to climate change</span>. <abbr class="cit-jnl-abbrev">PLoS Biol.</abbr> <span class="cit-vol">6</span>, <span class="cit-fpage">2621</span>–<span class="cit-lpage">2626</span> (<span class="cit-pub-date">2008</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2Bbiology%26rft.stitle%253DPLoS%2BBiol%26rft.aulast%253DWilliams%26rft.auinit1%253DS.%2BE.%26rft.volume%253D6%26rft.issue%253D12%26rft.spage%253D2621%26rft.epage%253D2626%26rft.atitle%253DTowards%2Ban%2Bintegrated%2Bframework%2Bfor%2Bassessing%2Bthe%2Bvulnerability%2Bof%2Bspecies%2Bto%2Bclimate%2Bchange.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pbio.0060325%26rft_id%253Dinfo%253Apmid%252F19108608%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pbio.0060325&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19108608&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F01%2F30%2F2020.01.29.925396.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000261913700002&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">23.</span><a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference 23. in text" id="ref-23">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.01.29.925396v1.23"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">B.</span> <span class="cit-name-surname">Helmuth</span></span>, <span class="cit-etal">et al.</span>, <span class="cit-article-title">Long-term, high frequency in situ measurements of intertidal mussel bed temperatures using biomimetic sensors</span>. <abbr class="cit-jnl-abbrev">Sci Data</abbr> <span class="cit-vol">3</span>, <span class="cit-fpage">160087</span> (<span class="cit-pub-date">2016</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DSci%2BData%26rft.volume%253D3%26rft.spage%253D160087%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">24.</span><a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference 24. in text" id="ref-24">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.01.29.925396v1.24" data-doi="10.1126/science.1155676"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">R. A.</span> <span class="cit-name-surname">Feely</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">C. L.</span> <span class="cit-name-surname">Sabine</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">J. M.</span> <span class="cit-name-surname">Hernandez-Ayon</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">D.</span> <span class="cit-name-surname">Ianson</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">B.</span> <span class="cit-name-surname">Hales</span></span>, <span class="cit-article-title">Evidence for upwelling of corrosive “acidified” water onto the continental shelf</span>. <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">320</span>, <span class="cit-fpage">1490</span>–<span class="cit-lpage">1492</span> (<span class="cit-pub-date">2008</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1155676%26rft_id%253Dinfo%253Apmid%252F18497259%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEzOiIzMjAvNTg4Mi8xNDkwIjtzOjQ6ImF0b20iO3M6NDg6Ii9iaW9yeGl2L2Vhcmx5LzIwMjAvMDEvMzAvMjAyMC4wMS4yOS45MjUzOTYuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">25.</span><a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference 25. in text" id="ref-25">↵</a><div class="cit ref-cit ref-book" id="cit-2020.01.29.925396v1.25"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">A. G.</span> <span class="cit-name-surname">Humes</span></span>, <span class="cit-chapter-title">How many copepods?</span> in <span class="cit-source">Ecology and Morphology of Copepods</span>, (<span class="cit-publ-name">Springer</span> <span class="cit-publ-loc">Netherlands</span>, <span class="cit-pub-date">1994</span>), pp. <span class="cit-fpage">1</span>–<span class="cit-lpage">7</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">26.</span><a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference 26. in text" id="ref-26">↵</a><div class="cit ref-cit ref-book" id="cit-2020.01.29.925396v1.26"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Mauchline</span></span>, <span class="cit-source">The Biology of Calanoid Copepods</span> (<span class="cit-publ-name">Academic Press</span>, <span class="cit-pub-date">1998</span>).</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">27.</span><a class="rev-xref-ref" href="#xref-ref-27-1" title="View reference 27. in text" id="ref-27">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.01.29.925396v1.27"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">J. A. F.</span> <span class="cit-name-surname">Langer</span></span>, <span class="cit-etal">et al.</span>, <span class="cit-article-title">Acclimation and adaptation of the coastal calanoid copepod Acartia tonsa to ocean acidification: a long-term laboratory investigation</span>. <abbr class="cit-jnl-abbrev">Mar. Ecol. Prog. Ser.</abbr> <span class="cit-vol">619</span>, <span class="cit-fpage">35</span>–<span class="cit-lpage">51</span> (<span class="cit-pub-date">2019</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMar.%2BEcol.%2BProg.%2BSer.%26rft.volume%253D619%26rft.spage%253D35%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">28.</span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-2020.01.29.925396v1.28"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">M. W.</span> <span class="cit-name-surname">Kelly</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">M. S.</span> <span class="cit-name-surname">Pankey</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">M. B.</span> <span class="cit-name-surname">DeBiasse</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">D. C.</span> <span class="cit-name-surname">Plachetzki</span></span>, <span class="cit-article-title">Adaptation to heat stress reduces phenotypic and transcriptional plasticity in a marine copepod</span>. <abbr class="cit-jnl-abbrev">Funct. Ecol.</abbr> <span class="cit-vol">31</span>, <span class="cit-fpage">398</span>–<span class="cit-lpage">406</span> (<span class="cit-pub-date">2017</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DFunct.%2BEcol.%26rft.volume%253D31%26rft.spage%253D398%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">29.</span><a class="rev-xref-ref" href="#xref-ref-29-1" title="View reference 29. in text" id="ref-29">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.01.29.925396v1.29" data-doi="10.1146/annurev-marine-121211-172229"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">H. G.</span> <span class="cit-name-surname">Dam</span></span>, <span class="cit-article-title">Evolutionary adaptation of marine zooplankton to global change</span>. <abbr class="cit-jnl-abbrev">Ann. Rev. Mar. Sci.</abbr> <span class="cit-vol">5</span>, <span class="cit-fpage">349</span>–<span class="cit-lpage">370</span> (<span class="cit-pub-date">2013</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAnn.%2BRev.%2BMar.%2BSci.%26rft.volume%253D5%26rft.spage%253D349%26rft_id%253Dinfo%253Adoi%252F10.1146%252Fannurev-marine-121211-172229%26rft_id%253Dinfo%253Apmid%252F22809192%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1146/annurev-marine-121211-172229&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22809192&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F01%2F30%2F2020.01.29.925396.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">30.</span><a class="rev-xref-ref" href="#xref-ref-30-1" title="View reference 30. in text" id="ref-30">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.01.29.925396v1.30"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">C. J.</span> <span class="cit-name-surname">Gobler</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">H.</span> <span class="cit-name-surname">Baumann</span></span>, <span class="cit-article-title">Hypoxia and acidification in ocean ecosystems: coupled dynamics and effects on marine life</span>. <abbr class="cit-jnl-abbrev">Biol. Lett</abbr>. <span class="cit-vol">12</span> (<span class="cit-pub-date">2016</span>).</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">31.</span><a class="rev-xref-ref" href="#xref-ref-31-1" title="View reference 31. in text" id="ref-31">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.01.29.925396v1.31" data-doi="10.1007/s12237-014-9770-0"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">E.</span> <span class="cit-name-surname">Rice</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">H. G.</span> <span class="cit-name-surname">Dam</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">G.</span> <span class="cit-name-surname">Stewart</span></span>, <span class="cit-article-title">Impact of Climate Change on Estuarine Zooplankton: Surface Water Warming in Long Island Sound Is Associated with Changes in Copepod Size and Community Structure</span>. <abbr class="cit-jnl-abbrev">Estuaries Coasts</abbr> <span class="cit-vol">38</span>, <span class="cit-fpage">13</span>–<span class="cit-lpage">23</span> (<span class="cit-pub-date">2015</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEstuaries%2BCoasts%26rft.volume%253D38%26rft.spage%253D13%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs12237-014-9770-0%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s12237-014-9770-0&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label">32.</span><a class="rev-xref-ref" href="#xref-ref-32-1" title="View reference 32. in text" id="ref-32">↵</a><div class="cit ref-cit ref-book" id="cit-2020.01.29.925396v1.32"><div class="cit-metadata"><ol class="duplicate"><li><span class="cit-ed"> <span class="cit-name-given-names">R. K.</span> <span class="cit-name-surname">Pachauri</span></span> and </li><li><span class="cit-ed"> <span class="cit-name-given-names">L. A.</span> <span class="cit-name-surname">Meyer</span></span></li></ol><cite><span class="cit-auth cit-collab">IPCC, “Climate Change 2014: Mitigation of Climate Change</span>. <span class="cit-chapter-title">Contribution of Working Group III to the Fifth Assessment Report of the Intergovernmental Panel on Climate Change</span>,” <span class="cit-auth cit-collab">Core Writing Team</span>, <span class="cit-ed"> <span class="cit-name-given-names">R. K.</span> <span class="cit-name-surname">Pachauri</span></span> and <span class="cit-ed"> <span class="cit-name-given-names">L. A.</span> <span class="cit-name-surname">Meyer</span></span>, Ed. (<span class="cit-publ-name">IPCC</span>, <span class="cit-pub-date">2014</span>).</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">33.</span><a class="rev-xref-ref" href="#xref-ref-33-1" title="View reference 33. in text" id="ref-33">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.01.29.925396v1.33" data-doi="10.1038/425365a"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">K.</span> <span class="cit-name-surname">Caldeira</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">M. E.</span> <span class="cit-name-surname">Wickett</span></span>, <span class="cit-article-title">Oceanography: anthropogenic carbon and ocean pH</span>. <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">425</span>, <span class="cit-fpage">365</span> (<span class="cit-pub-date">2003</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.volume%253D425%26rft.issue%253D6956%26rft.spage%253D365%26rft.atitle%253DAnthropogenic%2Bcarbon%2Band%2Bocean%2BpH%26rft_id%253Dinfo%253Adoi%252F10.1038%252F425365a%26rft_id%253Dinfo%253Apmid%252F14508477%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/425365a&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=2004082942&amp;link_type=GEOREF" class="cit-ref-sprinkles cit-ref-sprinkles-georef"><span>GeoRef</span></a><a href="/lookup/external-ref?access_num=14508477&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F01%2F30%2F2020.01.29.925396.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000185502300029&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">34.</span><a class="rev-xref-ref" href="#xref-ref-34-1" title="View reference 34. in text" id="ref-34">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.01.29.925396v1.34" data-doi="10.1038/nature05317"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">M. J.</span> <span class="cit-name-surname">Behrenfeld</span></span>, <span class="cit-etal">et al.</span>, <span class="cit-article-title">Climate-driven trends in contemporary ocean productivity</span>. <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">444</span>, <span class="cit-fpage">752</span>–<span class="cit-lpage">755</span> (<span class="cit-pub-date">2006</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DBehrenfeld%26rft.auinit1%253DM.%2BJ.%26rft.volume%253D444%26rft.issue%253D7120%26rft.spage%253D752%26rft.epage%253D755%26rft.atitle%253DClimate-driven%2Btrends%2Bin%2Bcontemporary%2Bocean%2Bproductivity.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature05317%26rft_id%253Dinfo%253Apmid%252F17151666%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature05317&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17151666&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F01%2F30%2F2020.01.29.925396.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000242581100059&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">35.</span><a class="rev-xref-ref" href="#xref-ref-35-1" title="View reference 35. in text" id="ref-35">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.01.29.925396v1.35"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">D.</span> <span class="cit-name-surname">Zoccola</span></span>, <span class="cit-etal">et al.</span>, <span class="cit-article-title">Coral Carbonic Anhydrases: Regulation by Ocean Acidification</span>. <abbr class="cit-jnl-abbrev">Mar. Drugs</abbr> <span class="cit-vol">14</span> (<span class="cit-pub-date">2016</span>).</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">36.</span><a class="rev-xref-ref" href="#xref-ref-36-1" title="View reference 36. in text" id="ref-36">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.01.29.925396v1.36" data-doi="10.15252/embr.201540749"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">S. P.</span> <span class="cit-name-surname">Hergeth</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">R.</span> <span class="cit-name-surname">Schneider</span></span>, <span class="cit-article-title">The H1 linker histones: multifunctional proteins beyond the nucleosomal core particle</span>. <abbr class="cit-jnl-abbrev">EMBO Rep.</abbr> <span class="cit-vol">16</span>, <span class="cit-fpage">1439</span>–<span class="cit-lpage">1453</span> (<span class="cit-pub-date">2015</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEMBO%2BRep.%26rft_id%253Dinfo%253Adoi%252F10.15252%252Fembr.201540749%26rft_id%253Dinfo%253Apmid%252F26474902%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NToiZW1ib3IiO3M6NToicmVzaWQiO3M6MTA6IjE2LzExLzE0MzkiO3M6NDoiYXRvbSI7czo0ODoiL2Jpb3J4aXYvZWFybHkvMjAyMC8wMS8zMC8yMDIwLjAxLjI5LjkyNTM5Ni5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">37.</span><a class="rev-xref-ref" href="#xref-ref-37-1" title="View reference 37. in text" id="ref-37">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.01.29.925396v1.37" data-doi="10.1002/1097-4652(200101)186:1&lt;136::AID-JCP1010&gt;3.0.CO;2-4"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">M. F.</span> <span class="cit-name-surname">Decristofaro</span></span>, <span class="cit-etal">et al.</span>, <span class="cit-article-title">Characterization of SWI/SNF protein expression in human breast cancer cell lines and other malignancies</span>. <abbr class="cit-jnl-abbrev">J. Cell. Physiol.</abbr> <span class="cit-vol">186</span>, <span class="cit-fpage">136</span>–<span class="cit-lpage">145</span> (<span class="cit-pub-date">2001</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2Bcellular%2Bphysiology%26rft.stitle%253DJ%2BCell%2BPhysiol%26rft.aulast%253DDecristofaro%26rft.auinit1%253DM.%2BF.%26rft.volume%253D186%26rft.issue%253D1%26rft.spage%253D136%26rft.epage%253D145%26rft.atitle%253DCharacterization%2Bof%2BSWI%252FSNF%2Bprotein%2Bexpression%2Bin%2Bhuman%2Bbreast%2Bcancer%2Bcell%2Blines%2Band%2Bother%2Bmalignancies.%26rft_id%253Dinfo%253Adoi%252F10.1002%252F1097-4652%2528200101%2529186%253A1%253C136%253A%253AAID-JCP1010%253E3.0.CO%253B2-4%26rft_id%253Dinfo%253Apmid%252F11147808%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1002/1097-4652(200101)186:1&lt;136::AID-JCP1010&gt;3.0.CO;2-4&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=11147808&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F01%2F30%2F2020.01.29.925396.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000165645300015&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">38.</span><a class="rev-xref-ref" href="#xref-ref-38-1" title="View reference 38. in text" id="ref-38">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.01.29.925396v1.38"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">H.</span> <span class="cit-name-surname">Caswell</span></span>, <span class="cit-article-title">Matrix population models</span>. <abbr class="cit-jnl-abbrev">Encyclopedia of Environmetrics</abbr> <span class="cit-vol">3</span> (<span class="cit-pub-date">2006</span>).</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">39.</span><a class="rev-xref-ref" href="#xref-ref-39-1" title="View reference 39. in text" id="ref-39">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.01.29.925396v1.39" data-doi="10.1002/ece3.3063"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Bailey</span></span>, <span class="cit-etal">et al.</span>, <span class="cit-article-title">Regulation of gene expression is associated with tolerance of the Arctic copepod <em>Calanus glacialis</em> to CO<sub>2</sub>-acidified sea water</span>. <abbr class="cit-jnl-abbrev">Ecol. Evol.</abbr> <span class="cit-vol">7</span>, <span class="cit-fpage">7145</span>–<span class="cit-lpage">7160</span> (<span class="cit-pub-date">2017</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEcol.%2BEvol.%26rft.volume%253D7%26rft.spage%253D7145%26rft_id%253Dinfo%253Adoi%252F10.1002%252Fece3.3063%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1002/ece3.3063&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label">40.</span><a class="rev-xref-ref" href="#xref-ref-40-1" title="View reference 40. in text" id="ref-40">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.01.29.925396v1.40" data-doi="10.1126/science.aan4380"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">R. A.</span> <span class="cit-name-surname">Bay</span></span>, <span class="cit-etal">et al.</span>, <span class="cit-article-title">Genomic signals of selection predict climate-driven population declines in a migratory bird</span>. <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">359</span>, <span class="cit-fpage">83</span>–<span class="cit-lpage">86</span> (<span class="cit-pub-date">2018</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DBay%26rft.auinit1%253DR.%2BA.%26rft.volume%253D359%26rft.issue%253D6371%26rft.spage%253D83%26rft.epage%253D86%26rft.atitle%253DGenomic%2Bsignals%2Bof%2Bselection%2Bpredict%2Bclimate-driven%2Bpopulation%2Bdeclines%2Bin%2Ba%2Bmigratory%2Bbird%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.aan4380%26rft_id%253Dinfo%253Apmid%252F29302012%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjExOiIzNTkvNjM3MS84MyI7czo0OiJhdG9tIjtzOjQ4OiIvYmlvcnhpdi9lYXJseS8yMDIwLzAxLzMwLzIwMjAuMDEuMjkuOTI1Mzk2LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">41.</span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-2020.01.29.925396v1.41" data-doi="10.1086/691233"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">R. A.</span> <span class="cit-name-surname">Bay</span></span>, <span class="cit-etal">et al.</span>, <span class="cit-article-title">Predicting Responses to Contemporary Environmental Change Using Evolutionary Response Architectures</span>. <abbr class="cit-jnl-abbrev">Am. Nat.</abbr> <span class="cit-vol">189</span>, <span class="cit-fpage">463</span>–<span class="cit-lpage">473</span> (<span class="cit-pub-date">2017</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAm.%2BNat.%26rft.volume%253D189%26rft.spage%253D463%26rft_id%253Dinfo%253Adoi%252F10.1086%252F691233%26rft_id%253Dinfo%253Apmid%252F28410032%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1086/691233&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=28410032&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F01%2F30%2F2020.01.29.925396.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">42.</span><a class="rev-xref-ref" href="#xref-ref-42-1" title="View reference 42. in text" id="ref-42">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.01.29.925396v1.42"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Bush</span></span>, <span class="cit-etal">et al.</span>, <span class="cit-article-title">Incorporating evolutionary adaptation in species distribution modelling reduces projected vulnerability to climate change</span>. <abbr class="cit-jnl-abbrev">Ecol. Lett.</abbr> <span class="cit-vol">19</span>, <span class="cit-fpage">1468</span>–<span class="cit-lpage">1478</span> (<span class="cit-pub-date">2016</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEcol.%2BLett.%26rft.volume%253D19%26rft.spage%253D1468%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">43.</span><a class="rev-xref-ref" href="#xref-ref-43-1" title="View reference 43. in text" id="ref-43">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.01.29.925396v1.43" data-doi="10.1371/journal.pone.0048538"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Vehmaa</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Brutemark</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Engström-Öst</span></span>, <span class="cit-article-title">Maternal effects may act as an adaptation mechanism for copepods facing pH and temperature changes</span>. <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">7</span>, <span class="cit-fpage">e48538</span> (<span class="cit-pub-date">2012</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BONE%26rft.aulast%253DVehmaa%26rft.auinit1%253DA.%26rft.volume%253D7%26rft.issue%253D10%26rft.spage%253De48538%26rft.epage%253De48538%26rft.atitle%253DMaternal%2BEffects%2BMay%2BAct%2Bas%2Ban%2BAdaptation%2BMechanism%2Bfor%2BCopepods%2BFacing%2BpH%2Band%2BTemperature%2BChanges.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0048538%26rft_id%253Dinfo%253Apmid%252F23119052%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0048538&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23119052&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F01%2F30%2F2020.01.29.925396.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">44.</span><a class="rev-xref-ref" href="#xref-ref-44-1" title="View reference 44. in text" id="ref-44">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.01.29.925396v1.44" data-doi="10.3354/meps175087"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">L. R.</span> <span class="cit-name-surname">Feinberg</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">H. G.</span> <span class="cit-name-surname">Dam</span></span>, <span class="cit-article-title">Effects of diet on dimensions, density and sinking rates of fecal pellets of the copepod Acartia tonsa</span>. <abbr class="cit-jnl-abbrev">Mar. Ecol. Prog. Ser.</abbr> <span class="cit-vol">175</span>, <span class="cit-fpage">87</span>–<span class="cit-lpage">96</span> (<span class="cit-pub-date">1998</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMar.%2BEcol.%2BProg.%2BSer.%26rft.volume%253D175%26rft.spage%253D87%26rft_id%253Dinfo%253Adoi%252F10.3354%252Fmeps175087%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3354/meps175087&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label">45.</span><a class="rev-xref-ref" href="#xref-ref-45-1" title="View reference 45. in text" id="ref-45">↵</a><div class="cit ref-cit ref-book" id="cit-2020.01.29.925396v1.45"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">T. M.</span> <span class="cit-name-surname">Therneau</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">P. M.</span> <span class="cit-name-surname">Grambsch</span></span>, <span class="cit-source">Modeling Survival Data: Extending the Cox Model</span> (<span class="cit-publ-name">Springer</span>, <span class="cit-pub-date">2013</span>).</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">46.</span><a class="rev-xref-ref" href="#xref-ref-46-1" title="View reference 46. in text" id="ref-46">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.01.29.925396v1.46"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">T.</span> <span class="cit-name-surname">Therneau</span></span>, <span class="cit-article-title">A Package for Survival Analysis in S</span>. <abbr class="cit-jnl-abbrev">version 2.38</abbr> (<span class="cit-pub-date">2015</span>).</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">47.</span><a class="rev-xref-ref" href="#xref-ref-47-1" title="View reference 47. in text" id="ref-47">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.01.29.925396v1.47"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Kassambara</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Kosinski</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">P.</span> <span class="cit-name-surname">Biecek</span></span>, <abbr class="cit-jnl-abbrev">survminer: Drawing Survival Curves using ggplot2</abbr>. <span class="cit-vol">1</span> (<span class="cit-pub-date">2017</span>).</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">48.</span><a class="rev-xref-ref" href="#xref-ref-48-1" title="View reference 48. in text" id="ref-48">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.01.29.925396v1.48" data-doi="10.3354/meps040069"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">S. E. L.</span> <span class="cit-name-surname">Houde</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">M. R.</span> <span class="cit-name-surname">Roman</span></span>, <span class="cit-article-title">Effects of food quality on the functional ingestion response of the copepod Acartia tonsa</span>. <abbr class="cit-jnl-abbrev">Mar. Ecol. Prog. Ser.</abbr> <span class="cit-vol">40</span>, <span class="cit-fpage">69</span>–<span class="cit-lpage">77</span> (<span class="cit-pub-date">1987</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMar.%2BEcol.%2BProg.%2BSer.%26rft.volume%253D40%26rft.spage%253D69%26rft_id%253Dinfo%253Adoi%252F10.3354%252Fmeps040069%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3354/meps040069&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=A1987K620000007&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">49.</span><a class="rev-xref-ref" href="#xref-ref-49-1" title="View reference 49. in text" id="ref-49">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.01.29.925396v1.49"><div class="cit-metadata"><cite><span class="cit-auth cit-collab">R Core Team</span>, <span class="cit-article-title">R: A Language and Environment for Statistical Computing</span> (<span class="cit-pub-date">2019</span>).</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">50.</span><a class="rev-xref-ref" href="#xref-ref-50-1" title="View reference 50. in text" id="ref-50">↵</a><div class="cit ref-cit ref-web" id="cit-2020.01.29.925396v1.50" data-doi="10.1093/bioinformatics/btu170"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">A. M.</span> <span class="cit-name-surname">Bolger</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Lohse</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">B.</span> <span class="cit-name-surname">Usadel</span></span>, <span class="cit-article-title">Trimmomatic: a flexible trimmer for Illumina sequence data</span>. <span class="cit-source">Bioinformatics</span> <span class="cit-vol">30</span>, <span class="cit-fpage">2114</span>–<span class="cit-lpage">2120</span> (<span class="cit-pub-date">2014</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtu170%26rft_id%253Dinfo%253Apmid%252F24695404%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/btu170&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24695404&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F01%2F30%2F2020.01.29.925396.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000340049100004&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">51.</span><a class="rev-xref-ref" href="#xref-ref-51-1" title="View reference 51. in text" id="ref-51">↵</a><div class="cit ref-cit ref-web" id="cit-2020.01.29.925396v1.51"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">T. S.</span> <span class="cit-name-surname">Jørgensen</span></span>, <span class="cit-etal">et al.</span>, <span class="cit-article-title">The genome and mRNA transcriptome of the cosmopolitan calanoid copepod Acartia tonsa Dana improve the understanding of copepod genome size evolution</span>. <span class="cit-source">Genome Biol. Evol</span>. (<span class="cit-pub-date">2019</span>) https:/doi.org/10.1093/gbe/evz067.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">52.</span><a class="rev-xref-ref" href="#xref-ref-52-1" title="View reference 52. in text" id="ref-52">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.01.29.925396v1.52" data-doi="10.1038/nprot.2013.084"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">B. J.</span> <span class="cit-name-surname">Haas</span></span>, <span class="cit-etal">et al.</span>, <span class="cit-article-title">De novo transcript sequence reconstruction from RNA-seq using the Trinity platform for reference generation and analysis</span>. <abbr class="cit-jnl-abbrev">Nat. Protoc.</abbr> <span class="cit-vol">8</span>, <span class="cit-fpage">1494</span>–<span class="cit-lpage">1512</span> (<span class="cit-pub-date">2013</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat.%2BProtoc.%26rft.volume%253D8%26rft.spage%253D1494%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnprot.2013.084%26rft_id%253Dinfo%253Apmid%252F23845962%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nprot.2013.084&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23845962&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F01%2F30%2F2020.01.29.925396.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">53.</span><a class="rev-xref-ref" href="#xref-ref-53-1" title="View reference 53. in text" id="ref-53">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.01.29.925396v1.53" data-doi="10.1186/s13059-017-1284-1"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">N. M.</span> <span class="cit-name-surname">Davidson</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">A. D. K.</span> <span class="cit-name-surname">Hawkins</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Oshlack</span></span>, <span class="cit-article-title">SuperTranscripts: a data driven reference for analysis and visualisation of transcriptomes</span>. <abbr class="cit-jnl-abbrev">Genome Biol.</abbr> <span class="cit-vol">18</span>, <span class="cit-fpage">148</span> (<span class="cit-pub-date">2017</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BBiol.%26rft.volume%253D18%26rft.spage%253D148%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fs13059-017-1284-1%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/s13059-017-1284-1&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label">54.</span><a class="rev-xref-ref" href="#xref-ref-54-1" title="View reference 54. in text" id="ref-54">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.01.29.925396v1.54"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">H.</span> <span class="cit-name-surname">Li</span></span>, <span class="cit-article-title">Aligning sequence reads, clone sequences and assembly contigs with BWA-MEM</span>. <abbr class="cit-jnl-abbrev">arXiv preprint</abbr><span class="cit-pub-id-sep cit-pub-id-arxiv-sep"> </span><span class="cit-pub-id-scheme">arXiv:</span><span class="cit-pub-id cit-pub-id-arxiv">1303.3997</span> (<span class="cit-pub-date">2013</span>).</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">55.</span><a class="rev-xref-ref" href="#xref-ref-55-1" title="View reference 55. in text" id="ref-55">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.01.29.925396v1.55" data-doi="10.1101/gr.129684.111"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">D. C.</span> <span class="cit-name-surname">Koboldt</span></span>, <span class="cit-etal">et al.</span>, <span class="cit-article-title">VarScan 2: somatic mutation and copy number alteration discovery in cancer by exome sequencing</span>. <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">22</span>, <span class="cit-fpage">568</span>–<span class="cit-lpage">576</span> (<span class="cit-pub-date">2012</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.129684.111%26rft_id%253Dinfo%253Apmid%252F22300766%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjg6IjIyLzMvNTY4IjtzOjQ6ImF0b20iO3M6NDg6Ii9iaW9yeGl2L2Vhcmx5LzIwMjAvMDEvMzAvMjAyMC4wMS4yOS45MjUzOTYuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">56.</span><a class="rev-xref-ref" href="#xref-ref-56-1" title="View reference 56. in text" id="ref-56">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.01.29.925396v1.56"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">C.</span> <span class="cit-name-surname">Soneson</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">M. I.</span> <span class="cit-name-surname">Love</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">M. D.</span> <span class="cit-name-surname">Robinson</span></span>, <span class="cit-article-title">Differential analyses for RNA-seq: transcriptlevel estimates improve gene-level inferences</span>. <abbr class="cit-jnl-abbrev">F1000Res</abbr>. <span class="cit-fpage">4</span> (<span class="cit-pub-date">2016</span>).</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">57.</span><a class="rev-xref-ref" href="#xref-ref-57-1" title="View reference 57. in text" id="ref-57">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.01.29.925396v1.57" data-doi="10.1038/nmeth.4197"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">R.</span> <span class="cit-name-surname">Patro</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">G.</span> <span class="cit-name-surname">Duggal</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">M. I.</span> <span class="cit-name-surname">Love</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">R. A.</span> <span class="cit-name-surname">Irizarry</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">C.</span> <span class="cit-name-surname">Kingsford</span></span>, <span class="cit-article-title">Salmon provides fast and bias-aware quantification of transcript expression</span>. <abbr class="cit-jnl-abbrev">Nat. Methods</abbr> <span class="cit-vol">14</span>, <span class="cit-fpage">417</span>–<span class="cit-lpage">419</span> (<span class="cit-pub-date">2017</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat.%2BMethods%26rft.volume%253D14%26rft.spage%253D417%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnmeth.4197%26rft_id%253Dinfo%253Apmid%252F28263959%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nmeth.4197&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=28263959&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F01%2F30%2F2020.01.29.925396.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">58.</span><a class="rev-xref-ref" href="#xref-ref-58-1" title="View reference 58. in text" id="ref-58">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.01.29.925396v1.58" data-doi="10.1186/s13059-014-0550-8"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">M. I.</span> <span class="cit-name-surname">Love</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">W.</span> <span class="cit-name-surname">Huber</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">Anders</span></span>, <span class="cit-article-title">Moderated estimation of fold change and dispersion for RNA-seq data with DESeq2</span>. <abbr class="cit-jnl-abbrev">Genome Biol.</abbr> <span class="cit-vol">15</span>, <span class="cit-fpage">550</span> (<span class="cit-pub-date">2014</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BBiol.%26rft.volume%253D15%26rft.spage%253D550%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fs13059-014-0550-8%26rft_id%253Dinfo%253Apmid%252F25516281%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/s13059-014-0550-8&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25516281&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F01%2F30%2F2020.01.29.925396.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">59.</span><a class="rev-xref-ref" href="#xref-ref-59-1" title="View reference 59. in text" id="ref-59">↵</a><div class="cit ref-cit ref-web" id="cit-2020.01.29.925396v1.59"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">T.</span> <span class="cit-name-surname">Jombart</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">I.</span> <span class="cit-name-surname">Ahmed</span></span>, <span class="cit-article-title">adegenet 1.3-1: new tools for the analysis of genome-wide SNP data</span>. <span class="cit-source">Bioinformatics</span> (<span class="cit-pub-date">2011</span>) https:/doi.org/10.1093/bioinformatics/btr521.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">60.</span><a class="rev-xref-ref" href="#xref-ref-60-1" title="View reference 60. in text" id="ref-60">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.01.29.925396v1.60" data-doi="10.18637/jss.v033.i01"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">J. D.</span> <span class="cit-name-surname">Hadfield</span></span>, <span class="cit-article-title">MCMC Methods for Multi-Response Generalized Linear Mixed Models: The MCMCglmm R Package</span>. <abbr class="cit-jnl-abbrev">Journal of Statistical Software</abbr> <span class="cit-vol">33</span>, <span class="cit-fpage">1</span>–<span class="cit-lpage">22</span> (<span class="cit-pub-date">2010</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BStatistical%2BSoftware%26rft.volume%253D33%26rft.spage%253D1%26rft_id%253Dinfo%253Adoi%252F10.18637%252Fjss.v033.i01%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.18637/jss.v033.i01&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label">61.</span><a class="rev-xref-ref" href="#xref-ref-61-1" title="View reference 61. in text" id="ref-61">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.01.29.925396v1.61" data-doi="10.1038/hdy.2014.86"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">C.</span> <span class="cit-name-surname">Schlötterer</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">R.</span> <span class="cit-name-surname">Kofler</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">E.</span> <span class="cit-name-surname">Versace</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">R.</span> <span class="cit-name-surname">Tobler</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">S. U.</span> <span class="cit-name-surname">Franssen</span></span>, <span class="cit-article-title">Combining experimental evolution with next-generation sequencing: a powerful tool to study adaptation from standing genetic variation</span>. <abbr class="cit-jnl-abbrev">Heredity</abbr> <span class="cit-vol">114</span>, <span class="cit-fpage">431</span>–<span class="cit-lpage">440</span> (<span class="cit-pub-date">2015</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DHeredity%26rft.volume%253D114%26rft.spage%253D431%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fhdy.2014.86%26rft_id%253Dinfo%253Apmid%252F25269380%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/hdy.2014.86&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25269380&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F01%2F30%2F2020.01.29.925396.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">62.</span><a class="rev-xref-ref" href="#xref-ref-62-1" title="View reference 62. in text" id="ref-62">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.01.29.925396v1.62" data-doi="10.1111/j.1365-294X.2012.05673.x"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">P.</span> <span class="cit-name-surname">Orozco-terWengel</span></span>, <span class="cit-etal">et al.</span>, <span class="cit-article-title">Adaptation of <em>Drosophila</em> to a novel laboratory environment reveals temporally heterogeneous trajectories of selected alleles</span>. <abbr class="cit-jnl-abbrev">Mol. Ecol.</abbr> <span class="cit-vol">21</span>, <span class="cit-fpage">4931</span>–<span class="cit-lpage">4941</span> (<span class="cit-pub-date">2012</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2BEcology%2B%2528Print%2529%26rft.stitle%253DMolecular%2BEcology%2B%2528Print%2529%26rft.aulast%253DOrozco-Terwengel%26rft.auinit1%253DP.%26rft.volume%253D21%26rft.issue%253D20%26rft.spage%253D4931%26rft.epage%253D4941%26rft.atitle%253DAdaptation%2Bof%2BDrosophila%2Bto%2Ba%2Bnovel%2Blaboratory%2Benvironment%2Breveals%2Btemporally%2Bheterogeneous%2Btrajectories%2Bof%2Bselected%2Balleles.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1365-294X.2012.05673.x%26rft_id%253Dinfo%253Apmid%252F22726122%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1365-294X.2012.05673.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22726122&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F01%2F30%2F2020.01.29.925396.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000309888200005&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">63.</span><a class="rev-xref-ref" href="#xref-ref-63-1" title="View reference 63. in text" id="ref-63">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.01.29.925396v1.63" data-doi="10.1371/journal.pone.0015925"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">R.</span> <span class="cit-name-surname">Kofler</span></span>, <span class="cit-etal">et al.</span>, <span class="cit-article-title">PoPoolation: a toolbox for population genetic analysis of next generation sequencing data from pooled individuals</span>. <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">6</span>, <span class="cit-fpage">e15925</span> (<span class="cit-pub-date">2011</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BONE%26rft.aulast%253DKofler%26rft.auinit1%253DR.%26rft.volume%253D6%26rft.issue%253D1%26rft.spage%253De15925%26rft.epage%253De15925%26rft.atitle%253DPoPoolation%253A%2Ba%2Btoolbox%2Bfor%2Bpopulation%2Bgenetic%2Banalysis%2Bof%2Bnext%2Bgeneration%2Bsequencing%2Bdata%2Bfrom%2Bpooled%2Bindividuals.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0015925%26rft_id%253Dinfo%253Apmid%252F21253599%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0015925&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21253599&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F01%2F30%2F2020.01.29.925396.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">64.</span><a class="rev-xref-ref" href="#xref-ref-64-1" title="View reference 64. in text" id="ref-64">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.01.29.925396v1.64" data-doi="10.1186/s12864-015-1540-2"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">R. M.</span> <span class="cit-name-surname">Wright</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">G. V.</span> <span class="cit-name-surname">Aglyamova</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">E.</span> <span class="cit-name-surname">Meyer</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">M. V.</span> <span class="cit-name-surname">Matz</span></span>, <span class="cit-article-title">Gene expression associated with white syndromes in a reef building coral, Acropora hyacinthus</span>. <abbr class="cit-jnl-abbrev">BMC Genomics</abbr> <span class="cit-vol">16</span>, <span class="cit-fpage">371</span> (<span class="cit-pub-date">2015</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2BGenomics%26rft.volume%253D16%26rft.spage%253D371%26rft.atitle%253DGENE%2BEXPRESSION%2BASSOCIATED%2BWITH%2BWHITE%2BSYNDROMES%2BIN%2BA%2BREEF%2BBUILDING%2BCORAL%252C%2BACROPORA%2BHYACINTHUS%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fs12864-015-1540-2%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/s12864-015-1540-2&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.biorxiv.org/content/10.1101/2020.01.29.925396v1.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-disqus-comment" >
  
      
  
  <div class="pane-content">
    <div id="disqus_thread"><noscript><p><a href="http://biorxivstage.disqus.com/?url=https%3A%2F%2Fwww.biorxiv.org%2Fcontent%2F10.1101%2F2020.01.29.925396v1" class="" data-icon-position="" data-hide-link-title="0">View the discussion thread.</a></p></noscript></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/10.1101/2020.01.29.925479v1" title="Germinal center B cells separate signals for selection and division like compact discs" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous</span></a></span><span class="pager-next"><a href="/content/10.1101/2020.01.29.923862v1" title="Tailored Design of Protein Nanoparticle Scaffolds for Multivalent Presentation of Viral Glycoprotein Antigens" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    Posted&nbsp;January 30, 2020.  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-biorxiv-art-tools" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-biorxiv_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/10.1101/2020.01.29.925396v1.full.pdf" target="_self" class="article-dl-pdf-link link-icon"><i class="icon-external-link-sign"></i> <span class="title">Download PDF</span></a>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-biorxiv-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="Email this Article" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-biorxiv_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/10.1101/2020.01.29.925396v1.full" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for your interest in spreading the word about bioRxiv.</p><p>NOTE: Your email address is requested solely to identify you as the sender of this article.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/10.1101/2020.01.29.925396v1" class="active" data-icon-position="" data-hide-link-title="0">Global change adaptation limits future resilience</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from bioRxiv
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would like to see this page from the bioRxiv website.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/1127478" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-wyjavGzjUqd5ozc9bdeNfWF_BXbbiCXJNMrdELEWb5Y" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-biorxiv-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-biorxiv_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="1127478" id="node1127478--2" data-pisa="biorxiv;2020.01.29.925396v1" data-pisa-master="biorxiv;2020.01.29.925396" data-seqnum="1127478" data-apath="/biorxiv/early/2020/01/30/2020.01.29.925396.atom"><div  class="highwire-cite highwire-cite-highwire-article highwire-citation-biorxiv-article-pap-list clearfix" >

      <div class="highwire-cite-title" >
      <div class="highwire-cite-title">Global change adaptation limits future resilience</div>    </div>
  
      <div  class="highwire-cite-authors" ><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0"><span class="nlm-given-names">Reid S.</span> <span class="nlm-surname">Brennan</span></span>, <span class="highwire-citation-author" data-delta="1"><span class="nlm-given-names">James A.</span> <span class="nlm-surname">deMayo</span></span>, <span class="highwire-citation-author" data-delta="2"><span class="nlm-given-names">Hans G.</span> <span class="nlm-surname">Dam</span></span>, <span class="highwire-citation-author" data-delta="3"><span class="nlm-given-names">Michael</span> <span class="nlm-surname">Finiguerra</span></span>, <span class="highwire-citation-author" data-delta="4"><span class="nlm-given-names">Hannes</span> <span class="nlm-surname">Baumann</span></span>, <span class="highwire-citation-author" data-delta="5"><span class="nlm-given-names">Melissa H.</span> <span class="nlm-surname">Pespeni</span></span></span></div>
  
      <div  class="highwire-cite-metadata" ><span  class="highwire-cite-metadata-journal highwire-cite-metadata">bioRxiv </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">2020.01.29.925396; </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata"><span class="doi_label">doi:</span> https://doi.org/10.1101/2020.01.29.925396 </span></div>
  
  
  </div>
</div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.biorxiv.org/content/10.1101/2020.01.29.925396v1" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.biorxiv.org/content/10.1101/2020.01.29.925396v1&amp;title=Global%20change%20adaptation%20limits%20future%20resilience" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.biorxiv.org/content/10.1101/2020.01.29.925396v1&amp;title=Global%20change%20adaptation%20limits%20future%20resilience" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.biorxiv.org/content/10.1101/2020.01.29.925396v1&amp;text=Global%20change%20adaptation%20limits%20future%20resilience" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.biorxiv.org/content/10.1101/2020.01.29.925396v1&amp;title=Global%20change%20adaptation%20limits%20future%20resilience" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.biorxiv.org/content/10.1101/2020.01.29.925396v1&amp;t=Global%20change%20adaptation%20limits%20future%20resilience" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.biorxiv.org/content/10.1101/2020.01.29.925396v1&amp;title=Global%20change%20adaptation%20limits%20future%20resilience" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.biorxiv.org/content/10.1101/2020.01.29.925396v1&amp;title=Global%20change%20adaptation%20limits%20future%20resilience" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-biorxiv-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title="Citation Tools"><i class="icon-globe"></i> <span class="title">Citation Tools</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-biorxiv_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="1127478" id="node1127478--3" data-pisa="biorxiv;2020.01.29.925396v1" data-pisa-master="biorxiv;2020.01.29.925396" data-seqnum="1127478" data-apath="/biorxiv/early/2020/01/30/2020.01.29.925396.atom"><div  class="highwire-cite highwire-cite-highwire-article highwire-citation-biorxiv-article-pap-list clearfix" >

      <div class="highwire-cite-title" >
      <div class="highwire-cite-title">Global change adaptation limits future resilience</div>    </div>
  
      <div  class="highwire-cite-authors" ><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0"><span class="nlm-given-names">Reid S.</span> <span class="nlm-surname">Brennan</span></span>, <span class="highwire-citation-author" data-delta="1"><span class="nlm-given-names">James A.</span> <span class="nlm-surname">deMayo</span></span>, <span class="highwire-citation-author" data-delta="2"><span class="nlm-given-names">Hans G.</span> <span class="nlm-surname">Dam</span></span>, <span class="highwire-citation-author" data-delta="3"><span class="nlm-given-names">Michael</span> <span class="nlm-surname">Finiguerra</span></span>, <span class="highwire-citation-author" data-delta="4"><span class="nlm-given-names">Hannes</span> <span class="nlm-surname">Baumann</span></span>, <span class="highwire-citation-author" data-delta="5"><span class="nlm-given-names">Melissa H.</span> <span class="nlm-surname">Pespeni</span></span></span></div>
  
      <div  class="highwire-cite-metadata" ><span  class="highwire-cite-metadata-journal highwire-cite-metadata">bioRxiv </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">2020.01.29.925396; </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata"><span class="doi_label">doi:</span> https://doi.org/10.1101/2020.01.29.925396 </span></div>
  
  
  </div>
</div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage&amp;rft.epage&amp;rft.atitle=Global%20change%20adaptation%20limits%20future%20resilience&amp;rft.volume&amp;rft.issue&amp;rft.date=2020-01-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=bioRxiv&amp;rft.au=Brennan%2C+Reid+S.&amp;rft.au=deMayo%2C+James+A.&amp;rft.au=Dam%2C+Hans+G.&amp;rft.au=Finiguerra%2C+Michael&amp;rft.au=Baumann%2C+Hannes&amp;rft.au=Pespeni%2C+Melissa+H."></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/1127478/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/1127478/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/1127478/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/1127478/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/1127478/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/1127478/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/1127478/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/1127478/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/1127478/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/1127478/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/1127478/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/1127478/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><div class="item-list"><ul><li class="first"><a href="http://twitter.com/share?url=https%3A//www.biorxiv.org/content/10.1101/2020.01.29.925396v1&amp;count=horizontal&amp;via=&amp;text=Global%20change%20adaptation%20limits%20future%20resilience&amp;counturl=https%3A//www.biorxiv.org/content/10.1101/2020.01.29.925396v1" class="twitter-share-button service-links-twitter-widget" id="twitter_widget" title="Tweet This" rel="nofollow" data-icon-position="" data-hide-link-title="0"><span class="element-invisible">Tweet Widget</span></a></li><li><a href="http://www.facebook.com/plugins/like.php?href=https%3A//www.biorxiv.org/content/10.1101/2020.01.29.925396v1&amp;layout=button_count&amp;show_faces=false&amp;action=like&amp;colorscheme=light&amp;width=100&amp;height=21&amp;font=&amp;locale=" id="facebook_like" title="I Like it" class="service-links-facebook-like" rel="nofollow" data-icon-position="" data-hide-link-title="0"><span class="element-invisible">Facebook Like</span></a></li><li class="last"><a href="https://www.biorxiv.org/content/10.1101/2020.01.29.925396v1" id="google_plus_one" title="Plus it" class="service-links-google-plus-one" rel="nofollow" data-icon-position="" data-hide-link-title="0"><span class="element-invisible">Google Plus One</span></a></li></ul></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-collections" >
  
        <h2 class="pane-title">Subject Area</h2>
    
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-collections"><div class="highwire-list"><ul class="highwire-article-collection-term-list"><li class="first last odd"><span class="highwire-article-collection-term"><a href="/collection/evolutionary-biology" class="highlight" data-icon-position="" data-hide-link-title="0">Evolutionary Biology<i class="icon-caret-right"></i>
</a></span></li></ul></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-biorxiv-subject-collections block-style-col2" >
  
      
  
  <div class="pane-content">
    <div class="panel-flexible panels-flexible-8 clearfix" id="mini-panel-biorxiv_subject_collections">
<div class="panel-flexible-inside panels-flexible-8-inside">
<div class="panels-flexible-region panels-flexible-region-8-center panels-flexible-region-first panels-flexible-region-last">
  <div class="inside panels-flexible-region-inside panels-flexible-region-8-center-inside panels-flexible-region-inside-first panels-flexible-region-inside-last">
<div class="panel-pane pane-snippet" >
  
      
  
  <div class="pane-content">
    <div class="snippet biorxiv-subject-areas-table-title" id="biorxiv-subject-areas-table-title">
  
      
  <div class="snippet-content">
    <b>Subject Areas</b>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-snippet" >
  
      
  
  <div class="pane-content">
    <div class="snippet biorxiv-subject-areas-view-papers" id="biorxiv-subject-areas-view-papers">
  
      
  <div class="snippet-content">
    <a href="/content/early/recent"><strong>All Articles</strong></a>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-subject-collections" >
  
      
  
  <div class="pane-content">
    <ul id="collection" class="collection highwire-list-expand"><li class="outer collection depth-2  child first"><div class = "data-wrapper"><a href="/collection/animal-behavior-and-cognition" class="" data-icon-position="" data-hide-link-title="0">Animal Behavior and Cognition</a> <span class = "article-count">(1662)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/biochemistry" class="" data-icon-position="" data-hide-link-title="0">Biochemistry</a> <span class = "article-count">(2789)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/bioengineering" class="" data-icon-position="" data-hide-link-title="0">Bioengineering</a> <span class = "article-count">(1946)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/bioinformatics" class="" data-icon-position="" data-hide-link-title="0">Bioinformatics</a> <span class = "article-count">(10400)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/biophysics" class="" data-icon-position="" data-hide-link-title="0">Biophysics</a> <span class = "article-count">(4243)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/cancer-biology" class="" data-icon-position="" data-hide-link-title="0">Cancer Biology</a> <span class = "article-count">(3272)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/cell-biology" class="" data-icon-position="" data-hide-link-title="0">Cell Biology</a> <span class = "article-count">(4641)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/clinical-trials" class="" data-icon-position="" data-hide-link-title="0">Clinical Trials</a> <span class = "article-count">(135)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/developmental-biology" class="" data-icon-position="" data-hide-link-title="0">Developmental Biology</a> <span class = "article-count">(2889)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/ecology" class="" data-icon-position="" data-hide-link-title="0">Ecology</a> <span class = "article-count">(4544)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/epidemiology" class="" data-icon-position="" data-hide-link-title="0">Epidemiology</a> <span class = "article-count">(2041)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/evolutionary-biology" class="" data-icon-position="" data-hide-link-title="0">Evolutionary Biology</a> <span class = "article-count">(7288)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/genetics" class="" data-icon-position="" data-hide-link-title="0">Genetics</a> <span class = "article-count">(5540)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/genomics" class="" data-icon-position="" data-hide-link-title="0">Genomics</a> <span class = "article-count">(6893)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/immunology" class="" data-icon-position="" data-hide-link-title="0">Immunology</a> <span class = "article-count">(2434)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/microbiology" class="" data-icon-position="" data-hide-link-title="0">Microbiology</a> <span class = "article-count">(7616)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/molecular-biology" class="" data-icon-position="" data-hide-link-title="0">Molecular Biology</a> <span class = "article-count">(3051)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/neuroscience" class="" data-icon-position="" data-hide-link-title="0">Neuroscience</a> <span class = "article-count">(18885)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/paleontology" class="" data-icon-position="" data-hide-link-title="0">Paleontology</a> <span class = "article-count">(142)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/pathology" class="" data-icon-position="" data-hide-link-title="0">Pathology</a> <span class = "article-count">(477)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/pharmacology-and-toxicology" class="" data-icon-position="" data-hide-link-title="0">Pharmacology and Toxicology</a> <span class = "article-count">(791)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/physiology" class="" data-icon-position="" data-hide-link-title="0">Physiology</a> <span class = "article-count">(1178)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/plant-biology" class="" data-icon-position="" data-hide-link-title="0">Plant Biology</a> <span class = "article-count">(2771)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/scientific-communication-and-education" class="" data-icon-position="" data-hide-link-title="0">Scientific Communication and Education</a> <span class = "article-count">(686)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/synthetic-biology" class="" data-icon-position="" data-hide-link-title="0">Synthetic Biology</a> <span class = "article-count">(905)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/systems-biology" class="" data-icon-position="" data-hide-link-title="0">Systems Biology</a> <span class = "article-count">(2881)</span></div></li>
<li class="outer collection depth-2  child last"><div class = "data-wrapper"><a href="/collection/zoology" class="" data-icon-position="" data-hide-link-title="0">Zoology</a> <span class = "article-count">(472)</span></div></li>
</ul>  </div>

  
  </div>
  </div>
</div>
</div>
</div>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__VNH5GD8Zz7g6_hGCOZjVjdMndxxC6naiExSSJKX3k_A__kzmEdtbqCsJx5Z9yg2Qy0PptB__ufXm-TxUOl0KZzUw__QJacVmveFyLIBSPkp21xL0ZvuDW9DMBrD2HB1P4Ry5I.js"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__QJacVmveFyLIBSPkp21xL0ZvuDW9DMBrD2HB1P4Ry5I.js" defer="defer"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__llgGUcz2-k3Jmyyyog-_4yI1wJfGWwx0eQ2lbnM9IIo__t6UjIjIKIAD6nKc6BAU9qmlVV2VR4CBTNQbxlbJPgN0__QJacVmveFyLIBSPkp21xL0ZvuDW9DMBrD2HB1P4Ry5I.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
function euCookieComplianceLoadScripts() {}
//--><!]]>
</script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var eu_cookie_compliance_cookie_name = "";
//--><!]]>
</script>
<script type="text/javascript" src="//d33xdlntwy0kbs.cloudfront.net/cshl_custom.js"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__BmqjBnkz3MgYCAoc25s1lDRMEjLhC3mEPVonUFIHi08__Unwv5-ZIuHBfFwytsjEx1niBVJ7n1T4lPws7VrkHXM4__QJacVmveFyLIBSPkp21xL0ZvuDW9DMBrD2HB1P4Ry5I.js"></script>
  </body>
</html>
