<html xmlns="http://www.w3.org/1999/xhtml"> <head> <style type="text/css">
      	     		 
      	    a                          {background : #ffffff; }
      	    article                    {border-style : dotted; border-width : 2px; }
		 	
		 	div                        {background : #ffffcc;}
		 	div.abstract-title         {font-weight : bold ; font-size : 16pt;}
		 	div.ack                    {border-style : solid ; border-color : red; margin : 2em; }
		 	div.article-meta           {border-style : solid; border-width : 1 pt; margin 1em;}
		 	div.boxed-text             {margin : 5em; border-style : solid;}
		 	div.contrib-group          {margin : 1em; }
		 	div.fig                    {border-style : solid; border-width : 2px; margin : 2em; }
		 	div.funding                {font-weight : bold ; font-size : 16pt;}
		 	div.given-names            {font-style : italic;}
		 	div.intro                  {border-style : inset; margin : 5px;}
		 	div.introduction           {border-style : inset; margin : 5px;}
		 	div.journal-meta           {border-style : solid; border-width : 1 pt; margin 1em;}
		 	div.journal-title-group    {background : #ffeeee;}
		 	div.article-title          {font-weight : bold ; font-size : 18pt;}
		 	div.kwd                    {font-style : italic}
		 	div.meta-name              {font-weight : bold ; font-size : 16pt;}
		 	div.name                   {font-weight : bold;}
		 	div.alt-title              {font-style : italic; font-size : 12px;}
		 	
		 	div.sec                    {border : 2px; margin 5px; padding 2px;}
		 	div.title                  {font-family : courier; font-weight : bold;
		 	                            font-size : 16pt; margin : 5px;}
		 	
		 	div.materials_methods      {border-style : double; margin : 5px; }
		 	div.methods                {border-style : double; margin : 5px; }
		 	
		 	div.results                {border-style : solid; margin : 5px;}
		 	div.background             {border-style : dotted; margin : 5px;}
		 	
		 	div.discussion             {border-style : groove; margin : 5px;}
		 	div.conclusion             {border-style : ridge; margin : 5px;}
		 	div.conclusions             {border-style : ridge; margin : 5px;}
		 	div.supplementary-material {border-style : inset; margin : 5px;}
		 	div.abbreviations          {border-style : double; border-color : red; margin : 5px;}
		 	div.competinginterests     {border-style : double; border-color : blue; margin : 5px;}
		 	div.acknowledgements       {border-style : double; border-color : green; margin : 5px;}
		 	div.authors_contributions   {border-style : double; border-color : purple; margin : 5px;}
		 	
		 	div.publisher              {border-style : outset; margin : 5px;}
		 	div.fn-type-conflict       {background : #f88; }
		 	div.fn-type-con            {background : #ddf; }
		 	div.fn-type-other          {background : #ddd; }
		 	
		 	div.unknown                {background : #ffd;
									 	  border-style : solid;
									 	  border-width : 1px;
									 	  padding : 2 px;}
		 	  
      	    table                      {background : #ffffdd;}
		 	tr                         {background : #ddddff; padding : 1px;}
		 	
		 	span                       {background : #ffcccc;}
		 	
		 	span.citation-author       {font-family : helvetica; background : #ffeeee;}
		 	span.collab                {background : #ddffff; }
		 	span.comment               {font-family : courier; font-size : 6px; background : #ffaaff;}
		 	span.contrib               {background : #ffffff;}
		 	span.corresp               {background : #ddffdd; }
		 	span.doi                   {background : #ffffff;}
		 	span.email                 {font-family : courier; }
		 	span.etal                  {font-style : italic;}
		 	span.fpage                 {font-family : courier;}
		 	span.given-names           {background : #ffffff;}
		 	span.iso-abbrev            {background : #ffffff;}
		 	span.issn-epub             {background : #ffffff;}
		 	span.issn-ppub             {background : #ffffff;}
		 	span.journal-title         {background : #ffffff;}
		 	span.lpage                 {font-family : courier;}
		 	span.mixed-article-title   {font-style : italic ;}
		 	span.nlm-ta                {background : #ffffff;}
		 	span.pmc                   {background : #ffffff;}
		 	span.pmcid                 {background : #ffffff;}
		 	span.pmid                  {background : #ffffff;}
		 	span.publisher             {background : #ffffff;}
		 	span.publisher-id          {background : #ffffff;}
		 	span.publisher-name        {background : #ffffff;}
		 	span.source                {background : #ffffff;}
		 	span.subject               {background : #ffffff;}
		 	span.surname               {background : #ffffff;}
		 	span.volume                {font-family : courier; font-weight : bold;}
		 	span.year                  {font-family : courier ; font-style : italic;}
			</style> </head> <body> <div class="front" title="front"> <div class="journal-meta" tagx="journal-meta" title="journal-meta"><span class="nlm-ta" title="nlm-ta">PLoS Genet</span><span class="iso-abbrev" title="iso-abbrev">PLoS Genet</span><span class="publisher-id" title="publisher-id">pgen</span><span class="publisher-id" title="publisher-id">plge</span><span class="pmc" title="pmc">plosgen</span><div class="journal-title-group" tagx="journal-title-group" title="journal-title-group"><span class="journal-title" tagx="journal-title" title="journal-title">PLoS Genetics</span></div><span class="issn-ppub" tagx="issn" title="issn-ppub">1553-7390</span><span class="issn-epub" tagx="issn" title="issn-epub">1553-7404</span><div class="publisher" tagx="publisher" title="publisher"><span class="publisher-name" tagx="publisher-name" title="publisher-name">Public Library of Science</span><span class="publisher-loc" tagx="publisher-loc" title="publisher-loc">San Francisco, USA</span></div> </div> <div class="article-meta" tagx="article-meta" title="article-meta"><span class="pmid" title="pmid"> pmid: <a href="http://www.ncbi.nlm.nih.gov/pubmed/16848642">16848642</a></span><span class="doi" title="doi"> doi: <a href="https://dx.doi.org/10.1371/journal.pgen.0020109">10.1371/journal.pgen.0020109</a></span><span class="publisher-id" title="publisher-id">06-PLGE-RA-0016R2</span><span class="sici" title="sici">plge-02-07-12</span><div class="article-categories" title="article-categories">
: <span class="subject" title="subject">Research Article</span>: <span class="subject" title="subject">Evolution</span><span class="subject" title="subject">Genetics/Comparative Genomics</span><span class="subject" title="subject">Genetics/Chromosome Biology</span>: <span class="subject" title="subject">Vertebrates</span><span class="subject" title="subject">Mammals</span><span class="subject" title="subject">Xenarthra</span></div> <div class="title-group" tagx="title-group" title="title-group"> <div class="article-title" title="article-title">
The Ancestral Eutherian Karyotype Is Present in Xenarthra</div> <div class="alt-title" title="alt-title">
Ancestral Eutherian Karyotype in Xenarthra</div> </div> <div class="contrib-group" title="contrib-group"><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Svartman</span><span class="given-names" tagx="given-names" title="given-names">Marta</span></span><a href="#cor1">*</a><a href="#aff1" /></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Stone</span><span class="given-names" tagx="given-names" title="given-names">Gary</span></span><a href="#aff1" /></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Stanyon</span><span class="given-names" tagx="given-names" title="given-names">Roscoe</span></span><a href="#aff1" /></span></div><span class="citation_author_institution" id="aff1">[], </span><div class="contrib-group" title="contrib-group"><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Trask</span><span class="given-names" tagx="given-names" title="given-names">Barbara</span></span><span class="role" tagx="role" title="role">Editor</span><a href="#edit1" /></span></div><span class="citation_author_institution" id="edit1">[], </span><div class="author-notes" title="author-notes"> <div class="corresp" title="corresp">
* To whom correspondence should be addressed. E-mail: <span class="email" tagx="email" title="email">svartmanm@hotmail.com</span></div> </div><span class="pub-date-ppub" title="pub-date-ppub">ppub: <span>2006-7</span></span><span class="pub-date-epub" title="pub-date-epub">epub: <span>2006-7-7</span></span><span class="volume" tagx="volume" title="volume">2</span><span class="issue" tagx="issue" title="issue">7</span><span class="elocation-id" tagx="elocation-id" title="elocation-id">e109</span><span class="history" title="history"><span class="received" title="received">received: 2006-1-20</span><span class="accepted" title="accepted">accepted: 2006-5-31</span></span><div class="permissions"><span class="copyright" title="copyright">(C) , <span class="copyright-year" tagx="copyright-year" title="copyright-year">2006</span></span><span class="license" title="license"><span class="license-p" title="license-p">This is an open-access article distributed under the terms of the Creative Commons Public Domain declaration, which stipulates that, once placed in the public domain, this work may be freely reproduced, distributed, transmitted, modified, built upon, or otherwise used by anyone for any lawful purpose.</span></span></div> <div class="abstract" title="abstract"> <div class="abstract-title" title="abstract-title">
Abstract</div> <p>Molecular studies have led recently to the proposal of a new super-ordinal arrangement of the 18 extant Eutherian orders. From the four proposed super-orders, Afrotheria and Xenarthra were considered the most basal. Chromosome-painting studies with human probes in these two mammalian groups are thus key in the quest to establish the ancestral Eutherian karyotype. Although a reasonable amount of chromosome-painting data with human probes have already been obtained for Afrotheria, no Xenarthra species has been thoroughly analyzed with this approach. We hybridized human chromosome probes to metaphases of species <i>(Dasypus novemcinctus, Tamandua tetradactyla,</i> and <i>Choloepus hoffmanii)</i> representing three of the four Xenarthra families<i>.</i> Our data allowed us to review the current hypotheses for the ancestral Eutherian karyotype, which range from 2<i>n</i> = 44 to 2<i>n</i> = 48. One of the species studied, the two-toed sloth <span class="named-content" title="named-content">C. hoffmanii</span> (2<i>n</i> = 50), showed a chromosome complement strikingly similar to the proposed 2<i>n</i> = 48 ancestral Eutherian karyotype, strongly reinforcing it.</p> </div> <div class="abstract" title="abstract"> <div class="title" tagx="title" title="title">
Synopsis</div> <p>Living mammals are classified into three major groups: monotremes, marsupials, and placental mammals or Eutherians, composed of 18 orders. Phylogenetic studies point to Afrotheria (a clade of six endemic African orders) or Xenarthra (armadillos, anteaters, and sloths, mostly found in Central and South America) as the most basal Eutherian group. One of the most daring aims of molecular cytogenetics in the past decade has been to establish the karyotype present in a common ancestor of all Eutherians. The approach used, cross-species chromosome painting, involves the use of probes from specific chromosomes or chromosome segments from one species, which are hybridized to metaphases of another species, highlighting regions of homology between both genomes. More than 60 species have already been analyzed with this method, and it is believed that the ancestral karyotype had 44, 46, or 48 chromosomes. The authors, using human chromosomes as probes to study three Xenarthra species, found that one of them, the two-toed sloth <span class="named-content" title="named-content">Choloepus hoffmannii</span> (2<i>n</i> = 50), has a karyotype strikingly similar to the proposed 2<i>n</i> = 48 ancestral Eutherian complement. This observation, besides reinforcing the hypothesized karyotype, suggests that Xenarthra may be at the root of the Eutherian tree.</p> </div> <div class="counts" title="counts"><span class="page-count" title="page-count">page-count: </span></div> <div class="custom-meta-group" title="custom-meta-group"> <meta name="citation" value="Svartman M, Stone G, Stanyon R (2006) The ancestral Eutherian karyotype is present in Xenarthra. PLoS Genet 2(7): e109. DOI: 10.1371/journal.pgen.0020109" /> </div> </div> </div> <div class="body" title="body"> <div class="introduction" title="sec"> <div class="title" tagx="title" title="title">
Introduction</div> <p>Extensive molecular data on mammalian genomes have led recently to the proposal of a new phylogenetic tree for Eutherians, which encompasses four super-orders: Afrotheria (elephants, manatees, hyraxes, tenrecs, aardvark, and elephant shrews); Xenarthra (sloths, anteaters, and armadillos); Euarchontoglires (rodents and lagomorphs as a sister taxon to primates, flying lemurs, and tree shrews); and Laurasiatheria (cetaceans, artiodactyls, perissodactyls, carnivores, pangolins, bats, and insectivores) [<a href="#pgen-0020109-b001">1</a>,<a href="#pgen-0020109-b002">2</a>]. At first very debated, the grouping of the 18 living placental mammalian orders into these four clades is rapidly approaching a consensus [<a href="#pgen-0020109-b003">3</a>]. One of the pivotal questions still remaining relates to the root of the placental tree. Either Xenarthra or Afrotheria, or the combination Xenarthra plus Afrotheria, could represent the first split from a common Eutherian ancestor (reviewed in [<a href="#pgen-0020109-b003">3</a>]). Although Afrotheria has been favored as the first offshoot in the Eutherian tree [<a href="#pgen-0020109-b002">2</a>], both Xenarthra and Afrotheria emerged around 100 million years ago (mya), within a short interval from each other [<a href="#pgen-0020109-b004">4</a>,<a href="#pgen-0020109-b005">5</a>]. This is also the time of the separation between South America, where Xenarthrans are endemic, and Africa, home to the Afrotheria. Therefore, a causal connection between the plate-tectonic events and the diversification of mammals has been proposed [<a href="#pgen-0020109-b001">1</a>,<a href="#pgen-0020109-b003">3</a>].</p> <p>Extant Xenarthra (sloths, anteaters, and armadillos) are restricted to the Americas, mostly Central and South America. The 30 known living species are relicts of the once highly diversified Xenarthra that dominated the South American fauna from the late Cretaceous (80–65 mya) until the late Tertiary (3–2.5 mya), while South America was an isolated landmass. Xenarthrans were abundant until the Pleistocene (10,000 years ago), and more than 200 fossil genera have been reported [<a href="#pgen-0020109-b005">5</a>].</p> <p>Morphological and molecular studies support the monophyly of the group, which also extends to each of the four recognized extant families: Myrmecophagidae (four species of anteater), Bradypodidae (four species of three-toed sloth), Megalonychidae (two species of two-toed sloth), and Dasypodidae (about 20 species of armadillo) [<a href="#pgen-0020109-b006">6</a>–<a href="#pgen-0020109-b009">9</a>]. According to molecular data estimates, the radiation of Xenarthra occurred around 65 mya, during the Cretaceous/Tertiary boundary. Armadillos, the most basal group of Xenarthra, diverged around 65 mya, followed by the divergence between anteaters and sloths around 55 mya [<a href="#pgen-0020109-b005">5</a>].</p> <p>Molecular data on Xenarthra are rapidly accumulating in the literature [<a href="#pgen-0020109-b005">5</a>–<a href="#pgen-0020109-b009">9</a>], but only scarce information is available on their chromosomes. Nineteen species had their karyotypes described [<a href="#pgen-0020109-b010">10</a>], but only a few with banding techniques [<a href="#pgen-0020109-b011">11</a>–<a href="#pgen-0020109-b013">13</a>]. The reported diploid numbers range from 2<i>n</i> = 38 in <span class="named-content" title="named-content">Tolypeutes matacus</span> to 2<i>n</i> = 64 in <i>Dasypus novemcinctus, D</i>. <i>hybridus,</i> and <span class="named-content" title="named-content">Cyclopes didactyla</span> [<a href="#pgen-0020109-b011">11</a>]. The first comparative study between karyotypes of different Xenarthra species using molecular cytogenetics has recently been published, and the rate of chromosome repatterning in Xenarthra was considered low [<a href="#pgen-0020109-b013">13</a>].</p> <p>Chromosome painting to establish the karyotype of a common ancestral Eutherian mammal has already been applied in 12 out of the 18 orders in the group. The proposed diploid numbers range from 2<i>n</i> = 44 to 2<i>n</i> = 50 [<a href="#pgen-0020109-b014">14</a>–<a href="#pgen-0020109-b021">21</a>]. The most recent studies suggest 2<i>n</i> = 44 [<a href="#pgen-0020109-b019">19</a>], 2<i>n</i> = 46 [<a href="#pgen-0020109-b020">20</a>], and 2<i>n</i> = 48 [<a href="#pgen-0020109-b017">17</a>,<a href="#pgen-0020109-b021">21</a>]. Chromosome-painting studies in Afrotheria and Xenarthra, the most basal of the living Eutherians, are essential to establish the putative ancestral Eutherian complement. Some Afrotheria have already had their genomes analyzed by chromosome painting with human probes [<a href="#pgen-0020109-b019">19</a>–<a href="#pgen-0020109-b022">22</a>]. Data of comparative molecular cytogenetics in Xenarthra using human chromosomes as painting probes are restricted to very limited data for two species [<a href="#pgen-0020109-b017">17</a>,<a href="#pgen-0020109-b018">18</a>,<a href="#pgen-0020109-b023">23</a>].</p> <p>In this study, we hybridized paints of all human chromosomes in species representing three of the four families of Xenarthra, and reevaluated the current hypotheses for the ancestral Eutherian karyotype based on our results.</p> </div> <div class="results" title="sec"> <div class="title" tagx="title" title="title">
Results</div> <p>Some examples of human chromosome paints hybridized to the three Xenarthra species are shown in <a href="#pgen-0020109-g001">Figure 1</a>. The results of the chromosome painting with human probes in the three species are summarized in <a href="#pgen-0020109-t001">Table 1</a>. The G-banded karyotype of the three species studied were mounted according to Jorge et al. [<a href="#pgen-0020109-b011">11</a>] and are presented with the corresponding human chromosome segments in <a href="#pgen-0020109-g002">Figure 2</a><i>(D. novemcinctus</i> and <i>Tamandua tetradactyla)</i> and <a href="#pgen-0020109-g003">Figure 3</a>A <i>(Choloepus hoffmanii).</i></p> <div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/"><span class="label" tagx="label" title="label">Figure 1</span></a></div>   <div class="title" tagx="title" title="title">
Partial Metaphases of <i>D. novemcinctus, T. tetradactyla,</i> and <span class="named-content" title="named-content">C. hoffmanii</span> after In Situ Hybridization with Human Chromosome–Specific Probes</div> <p>(A–C) <i>D. novemcinctus,</i> (D–F) <i>T. tetradactyla,</i> and (G–I) <span class="named-content" title="named-content">C. hoffmanii</span>. Two probes were used in each experiment, and the hybridizations were detected with avidin–FITC (green) and antidigoxigenin–rhodamine (red). The chromosomes were counterstained with DAPI, and the human chromosome probes used are indicated.</p>   </div> <div class="table-wrap_UNKNOWN" id="pgen-0020109-t001" content-type="2col" position="float"><span class="label" tagx="label" title="label">Table 1</span> <p>Results of Chromosome Painting with Human Probes in Xenarthra</p>  <div class="graphic" /> </div> <p>The X chromosome of the three Xenarthra species hybridized with human chromosome X and showed a highly conserved G-banding pattern when compared to the human X chromosome, reinforcing its known conservation. No signals were obtained with human chromosome Y.</p> <p>In the nine-banded armadillo <span class="named-content" title="named-content">D. novemcinctus</span> (2<i>n</i> = 64; fundamental number [<i>FN</i>] = 78), chromosome painting resulted in 41 segments, and seven associations, or combinations of segments from two human chromosomes, were observed (<a href="#pgen-0020109-g002">Figure 2</a>A).</p> <div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/"><span class="label" tagx="label" title="label">Figure 2</span></a></div>   <div class="title" tagx="title" title="title">
G-Banded Karyotypes of Female Nine-Banded Armadillo and Female Lesser Anteater</div> <p>(A) Female nine-banded armadillo <span class="named-content" title="named-content">D. novemcinctus</span> (2<i>n</i> = 64) and (B) female lesser anteater <span class="named-content" title="named-content">T. tetradactyla</span> (2<i>n</i> = 54).</p> <p>The correspondence to human chromosomes as revealed by chromosome painting is shown on the left of each chromosome. The question mark indicates questionable results. Some regions were not painted by any human probe and are probably composed of repetitive sequences (see text).</p>   </div> <p>The karyotype of the lesser anteater <span class="named-content" title="named-content">T. tetradactyla</span> (2<i>n</i> = 54; <i>FN</i> = 104) consists exclusively of biarmed chromosomes. Painting with human probes resulted in a total of 44 segments, and 15 associations were observed (<a href="#pgen-0020109-g002">Figure 2</a>B).</p> <p>The two-toed sloth <span class="named-content" title="named-content">C. hoffmanii</span> is the species with the lowest diploid number (2<i>n</i> = 50; <i>FN</i> = 61). Owing to a heterozygous translocation between pairs 2 and 24, pair 2 was heteromorphic in the cells analyzed; this individual is thus trisomic for Chromosome 24. A total of 32 segments and six associations were detected with the human probes (<a href="#pgen-0020109-g003">Figure 3</a>A).</p> <div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/"><span class="label" tagx="label" title="label">Figure 3</span></a></div>   <div class="title" tagx="title" title="title">
The Karyotype of Hoffmann's Two-Toed Sloth (2<i>n</i> = 50) Closely Resembles the Proposed Eutherian Ancestral Karyotype (2<i>n</i> = 48)</div> <p>(A) G-banded karyotype of a male <span class="named-content" title="named-content">C. hoffmannii</span> (2<i>n</i> = 50) with the corresponding human chromosomes indicated to the left of each chromosome. (B) Diagram of <span class="named-content" title="named-content">C. hoffmannii</span> chromosomes. (C) The proposed ancestral Eutherian karyotype (2<i>n</i> = 48) [<a href="#pgen-0020109-b017">17</a>,<a href="#pgen-0020109-b021">21</a>]. In (B) and (C), the corresponding human chromosomes are each represented by a different color, and their numbers are indicated at the bottom of each chromosome. The chromosomes are drawn only roughly to scale. In (C), the corresponding <span class="named-content" title="named-content">C. hoffmannii</span> chromosomes are indicated to the left of each ancestral chromosome. The asterisks and lines show the chromosomes that differ between the karyotypes of <span class="named-content" title="named-content">C. hoffmanii</span> (CHO) and those proposed as ancestral (ANC) for Eutheria (CHO 14 and ANC 10q and 10p [black asterisks]; CHO 13 and 19 and ANC 8q [green asterisks]; and CHO 20 and 21 and ANC 7b/16p [red asterisks]).</p>   </div> <p>It is interesting to note that some chromosome regions of the three Xenarthra species were not labeled by any of the human probes (<a href="#pgen-0020109-g002">Figures 2</a> and <a href="#pgen-0020109-g003">3</a>A). These regions, which were either proximal (for instance <span class="named-content" title="named-content">D. novemcinctus</span> Chromosomes 25 and 30), distal (<span class="named-content" title="named-content">T. tetradactyla</span> Chromosome 1 and <span class="named-content" title="named-content">C. hoffmanii</span> Chromosome 7), or encompassed whole short arms (as with <span class="named-content" title="named-content">D. novemcinctus</span> Chromosomes 2 and 4, <span class="named-content" title="named-content">T. tetradactyla</span> Chromosome 12, and <span class="named-content" title="named-content">C. hoffmanii</span> Chromosomes 3, 4, and 5), are very likely to represent repetitive sequences that are absent from the human genome.</p> </div> <div class="discussion" title="sec"> <div class="title" tagx="title" title="title">
Discussion</div> <p>The effort to establish the ancestral Eutherian karyotype relies on the concept of parsimony, according to which likely ancestral chromosomes are present in species of divergent Eutherian orders. There is a conspicuous lack of data from Xenarthra, one of the most basal mammalian groups, in the quest to establish the ancestral karyotype. Aiming to fill this gap, we conducted painting experiments with human chromosome–specific probes in species representing three of the four major Xenarthra groups.</p> <p>Richard et al. [<a href="#pgen-0020109-b018">18</a>,<a href="#pgen-0020109-b023">23</a>] previously reported results for ten human chromosomes (3, 7, 12, 14, 15, 16, 18, 21, 22, and the X chromosome) in <span class="named-content" title="named-content">D. novemcinctus</span> metaphases. The armadillo chromosomes labeled were not specified, and three of the probes (human Chromosomes 7, 12, and 22) gave results that were questioned by Richard et al. In the three cases, only one signal was observed, while the same probes labeled two (human Chromosomes 7 and 22) or three (human Chromosome 12) autosomes in our sample. We obtained the same number of signals already reported for human Chromosomes 3, 14, 15, 16, 18, and the X chromosome [<a href="#pgen-0020109-b018">18</a>], but we detected two signals for human Chromosome 21, instead of the single labeling described by Richard et al. [<a href="#pgen-0020109-b018">18</a>]. Human Chromosome 1 was previously shown to be conserved in <span class="named-content" title="named-content">C. hoffmanii</span> [<a href="#pgen-0020109-b017">17</a>], and this was confirmed in our experiments.</p> <p>Six human chromosome associations (human Chromosomes 3/21, 4/8, 7/16, 12/22 twice, 14/15, and 16/19) are agreed, by most authors, as having been present in the ancestral Eutherian karyotype. All of these syntenies were found in the three species of Xenarthra that we studied (<a href="#pgen-0020109-t001">Table 1</a>). The associations human Chromosomes 3/21 and 14/15 were each disrupted in, respectively, <span class="named-content" title="named-content">D. novemcinctus</span> and <span class="named-content" title="named-content">T. tetradactyla</span> (<a href="#pgen-0020109-g001">Figure 1</a>F), suggesting that they underwent rearrangements in these species after the split from a common ancestor. In <i>C. hoffmanii,</i> no other syntenies besides the six already mentioned were found, and in <i>D. novemcinctus,</i> a single extra combination (human Chromosome 10/12) was identified. Human Chromosomes 9, 13, 17, 18, and 20 were found to be conserved in toto in the three Xenarthra species.</p> <p>Although Xenarthra is widely accepted as a monophyletic group on morphological and molecular grounds, no common chromosome syntenies besides those considered present in the common Eutherian ancestral karyotype were found in the three species studied. Thus, there is an unusual absence of chromosome synapomorphic traits linking the three families that we studied. The only exception might be human Chromosome 8, believed to be disrupted into two blocks in a common Eutherian ancestral and found as three segments in both <span class="named-content" title="named-content">D. novemcinctus</span> and <span class="named-content" title="named-content">T. tetradactyla</span> and possibly also in <span class="named-content" title="named-content">C. hoffmannii</span> (<a href="#pgen-0020109-t001">Table 1</a>). Additional experiments are needed to confirm this observation.</p> <p>On the other hand, we observed a striking resemblance of Xenarthran complements to the proposed 2<i>n</i> = 48 ancestral Eutherian karyotype [<a href="#pgen-0020109-b017">17</a>,<a href="#pgen-0020109-b021">21</a>], particularly in the case of <span class="named-content" title="named-content">C. hoffmannii</span> (2<i>n</i> = 50) (<a href="#pgen-0020109-g003">Figure 3</a>B). In this species, 20 autosome pairs appear to be the same as the most recent hypothesis for the putative Eutherian ancestral karyotype with 2<i>n</i> = 48 (human Chromosomes 1, 2p, 2q, 3/21, 4/8p, 5, 6, 7a, 7b/16p, 9, 11, 12/22 twice, 13, 14/15, 16q/19q, 17, 18, 19p, and 20). The only differences are with regard to human Chromosome 10, considered as two blocks in the ancestral karyotype and kept intact in this species; human Chromosome 8q, which might be further disrupted in <span class="named-content" title="named-content">C. hoffmannii</span> resulting in a third signal for this chromosome; and an additional signal for human Chromosome 16, represented by <span class="named-content" title="named-content">C. hoffmanii</span> Chromosome 20 (<a href="#pgen-0020109-g003">Figure 3</a>).</p> <p>In <i>D. novemcinctus,</i> some of the putative ancestral Eutherian chromosomes are disrupted into further blocks (for instance, human Chromosomes 1, 3/21, 6, 7, and 11), which would account for its higher diploid number (2<i>n</i> = 64) in relation to an ancestral karyotype, but otherwise this chromosome complement is also very close to that considered to be the ancestral one.</p> <p><span class="named-content" title="named-content">T. tetradactyla</span> presents the most rearranged complement in relation to the ancestral karyotype, and was also reported as such when compared to other Xenarthra taxa [<a href="#pgen-0020109-b013">13</a>]. In this case, several rearrangements seem to have occurred, resulting in chromosome disruptions and new associations in relation to a common ancestral complement. This species presented two associations previously reported in Afrotheria (human Chromosomes 1/19 and 2/8). One of them, human Chromosome 1/19, was considered as a synapomorphic trait of Afrotheria [<a href="#pgen-0020109-b021">21</a>] and was postulated as ancestral to all Eutheria by Yang et al. [<a href="#pgen-0020109-b019">19</a>]. Nevertheless, its occurrence in only one highly rearranged primate species [<a href="#pgen-0020109-b024">24</a>] outside Afrotheria was considered insufficient to accept it as ancestral [<a href="#pgen-0020109-b021">21</a>]. As human Chromosome 1/19 was found only in one of the Xenarthrans herein analyzed, and it is likely to be a different association regarding the segments involved, we do not consider it as a probable ancestral Eutherian combination.</p> <p>Human Chromosome 2/8 was absent in elephants, but was found in four other species of Afrotheria that were analyzed with chromosome painting [<a href="#pgen-0020109-b019">19</a>–<a href="#pgen-0020109-b022">22</a>]. It was thus considered a phylogenetic link between those species after the divergence of the elephant [<a href="#pgen-0020109-b021">21</a>]. Human Chromosome 2/8 was observed only in one species of Xenarthra, and it is not clear whether the segments involved are the same as those detected in Afrotheria, which rules out this association as a common ancestral one.</p> <p>An alternative putative ancestral karyotype with 2<i>n</i> = 46 was proposed, in which the combination human Chromosome 10/12/22 would be present [<a href="#pgen-0020109-b020">20</a>,<a href="#pgen-0020109-b025">25</a>]. Although we observed an association human Chromosome 10/12 in <i>D. novemcinctus,</i> we could not detect it in the other two species. As previously discussed [<a href="#pgen-0020109-b021">21</a>], the ancestral status of human Chromosome 10/12/22, which seems to be widespread in Afrotheria and Carnivores, remains doubtful. Our new data on Xenarthra do not lend support to this association as an ancestral Eutherian karyotype feature, thus reinforcing the 2<i>n</i> = 48 ancestral karyotype hypothesis.</p> <p>The status of Xenarthra as a basal group in the Eutherian phylogenetic tree is still disputed on molecular grounds [<a href="#pgen-0020109-b003">3</a>]. Based on the chromosome-painting results, it is clear that no Afrotheria has a complement so close to the putative ancestral Eutherian karyotype as the one we found herein in Xenarthra. Thus, Xenarthra cannot yet be ruled out as the most basal Eutheria, and more data should be gathered on the two basal groups before a more definitive answer is reached.</p> <p>A final conclusion on the ancestral Eutherian karyotype will ultimately depend on comparisons with the most suitable outgroup, represented by marsupials. Technical constraints currently prevent cross-species chromosome painting between such distantly related groups as Eutherians and marsupials.</p> <p>It is important to point out that the resolution limit of interspecies chromosome painting is about 5 Mbp. Another constraint of this approach is that it cannot detect intrachromosomal rearrangements, as inversions, that result in gene-order changes. Recent comparative studies encompassing results obtained from chromosome painting, radiation hybrid maps, and whole-genome sequencing of various mammalian groups show the potential of this broad kind of study to reveal conserved genome segments in mammals as well as the breakpoints involved in chromosome repatterning at a much more detailed level [<a href="#pgen-0020109-b025">25</a>,<a href="#pgen-0020109-b026">26</a>]. This approach promises ultimately to allow the reconstruction of the ancestral Eutherian karyotype, as whole-genome sequencing data become available for more mammalian species, including marsupials and monotremes, in the not too distant future.</p> </div> <div class="materialsandmethods" title="sec"> <div class="title" tagx="title" title="title">
Materials and Methods</div> <p>Chromosome preparations were obtained from cultured fibroblasts of a female nine-banded armadillo <span class="named-content" title="named-content">D. novemcinctus</span> (2<i>n</i> = 64), a male two-toed sloth <span class="named-content" title="named-content">C. hoffmanii</span> (2<i>n</i> = 50), and a female lesser anteater <span class="named-content" title="named-content">T. tetradactyla</span> (2<i>n</i> = 54). The cells from the two former species were kindly supplied by the Center for Reproduction of Endangered Species (CRES) of the San Diego Zoo, California, United States. The lesser anteater cells were purchased from ATCC (Manassas, Virginia, United States). Cells were cultivated in αDMEM with 10% fetal bovine serum at 35 °C. Routine procedures were used for chromosome preparations. GTG banding was performed as described [<a href="#pgen-0020109-b027">27</a>] with modifications.</p> <p>Human chromosome–specific probes were obtained by flow sorting, and DOP-PCR as well as the interspecific in situ hybridization experiments were performed as already described [<a href="#pgen-0020109-b021">21</a>].</p> </div> </div> <div class="back" title="back"> <div class="ack" title="ack"> <p><b>Author contributions.</b> RS conceived and designed the experiments. MS and RS performed the experiments and analyzed the data. GS and RS contributed reagents/materials/analysis tools. MS wrote the paper.</p> </div> <div class="fn-group" title="fn-group"> <div class="fn-type-COI-statement" title="COI-statement"> <p><b>Competing interests.</b> The authors have declared that no competing interests exist.</p> </div> <div class="fn-type-financial-disclosure" title="financial-disclosure"> <p><b>Funding.</b> This research was supported by the Intramural Research Program of the National Institutes of Health, National Cancer Institute, Center for Cancer Research.</p> </div> </div> <div class="glossary" title="glossary"> <div class="title" tagx="title" title="title">
Abbreviations</div> <div class="def-list" title="def-list"> <div class="def-item" title="def-item"> <div class="term" title="term">
CRES</div> <div class="def" title="def"> <p>Center for Reproduction of Endangered Species</p> </div> </div> <div class="def-item" title="def-item"> <div class="term" title="term"><i>FN</i></div> <div class="def" title="def"> <p>fundamental number (number of autosomal chromosome arms)</p> </div> </div> <div class="def-item" title="def-item"> <div class="term" title="term">
mya</div> <div class="def" title="def"> <p>million years ago</p> </div> </div> </div> </div> <div class="references">
References</div> <div tag="ref-list"> <ul> <div class="title" tagx="title" title="title">
References</div> <li tag="ref"><a name="pgen-0020109-b001" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Murphy</span><span class="given-names" tagx="given-names" title="given-names">WJ</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Eizirik</span><span class="given-names" tagx="given-names" title="given-names">E</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Johnson</span><span class="given-names" tagx="given-names" title="given-names">WE</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zhang</span><span class="given-names" tagx="given-names" title="given-names">YP</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ryder</span><span class="given-names" tagx="given-names" title="given-names">OA</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="year" tagx="year" title="year">2001</span><span class="mixed-article-title" title="mixed-article-title">Molecular phylogenetics and the origins of placental mammals</span><span class="source" tagx="source" title="source">Nature</span><span class="volume" tagx="volume" title="volume">409</span><span class="fpage" tagx="fpage" title="fpage">614</span><span class="lpage" tagx="lpage" title="lpage">618</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11214319">11214319</a></span></span></li> <li tag="ref"><a name="pgen-0020109-b002" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Murphy</span><span class="given-names" tagx="given-names" title="given-names">WJ</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Eizirik</span><span class="given-names" tagx="given-names" title="given-names">E</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">O'Brien</span><span class="given-names" tagx="given-names" title="given-names">SJ</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Madsen</span><span class="given-names" tagx="given-names" title="given-names">O</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Scally</span><span class="given-names" tagx="given-names" title="given-names">M</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="year" tagx="year" title="year">2001</span><span class="mixed-article-title" title="mixed-article-title">Resolution of the early placental mammal radiation using Bayesian phylogenetics</span><span class="source" tagx="source" title="source">Science</span><span class="volume" tagx="volume" title="volume">294</span><span class="fpage" tagx="fpage" title="fpage">2348</span><span class="lpage" tagx="lpage" title="lpage">2351</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11743200">11743200</a></span></span></li> <li tag="ref"><a name="pgen-0020109-b003" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Springer</span><span class="given-names" tagx="given-names" title="given-names">MS</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Stanhope</span><span class="given-names" tagx="given-names" title="given-names">MJ</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Madsen</span><span class="given-names" tagx="given-names" title="given-names">O</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">de Jong</span><span class="given-names" tagx="given-names" title="given-names">WW</span></span></span><span class="year" tagx="year" title="year">2004</span><span class="mixed-article-title" title="mixed-article-title">Molecules consolidate the placental mammal tree</span><span class="source" tagx="source" title="source">Trends Ecol Evol</span><span class="volume" tagx="volume" title="volume">19</span><span class="fpage" tagx="fpage" title="fpage">430</span><span class="lpage" tagx="lpage" title="lpage">438</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16701301">16701301</a></span></span></li> <li tag="ref"><a name="pgen-0020109-b004" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Eizirik</span><span class="given-names" tagx="given-names" title="given-names">E</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Murphy</span><span class="given-names" tagx="given-names" title="given-names">WJ</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">O'Brien</span><span class="given-names" tagx="given-names" title="given-names">SJ</span></span></span><span class="year" tagx="year" title="year">2001</span><span class="mixed-article-title" title="mixed-article-title">Molecular dating and biogeography of the early placental mammal radiation</span><span class="source" tagx="source" title="source">J Hered</span><span class="volume" tagx="volume" title="volume">92</span><span class="fpage" tagx="fpage" title="fpage">212</span><span class="lpage" tagx="lpage" title="lpage">219</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11396581">11396581</a></span></span></li> <li tag="ref"><a name="pgen-0020109-b005" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Delsuc</span><span class="given-names" tagx="given-names" title="given-names">F</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Vizcaíno</span><span class="given-names" tagx="given-names" title="given-names">SF</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Douzery</span><span class="given-names" tagx="given-names" title="given-names">EJP</span></span></span><span class="year" tagx="year" title="year">2004</span><span class="mixed-article-title" title="mixed-article-title">Influence of Tertiary paleoenvironmental changes on the diversification of South American mammals: A relaxed molecular clock study within Xenarthrans</span><span class="source" tagx="source" title="source">BMC Evol Biol</span><span class="volume" tagx="volume" title="volume">4</span><span class="fpage" tagx="fpage" title="fpage">11</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15115541">15115541</a></span></span></li> <li tag="ref"><a name="pgen-0020109-b006" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Delsuc</span><span class="given-names" tagx="given-names" title="given-names">F</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Catzeflis</span><span class="given-names" tagx="given-names" title="given-names">FM</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Stanhope</span><span class="given-names" tagx="given-names" title="given-names">MJ</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Douzery</span><span class="given-names" tagx="given-names" title="given-names">EJ</span></span></span><span class="year" tagx="year" title="year">2001</span><span class="mixed-article-title" title="mixed-article-title">The evolution of armadillos, anteaters and sloths depicted by nuclear and mitochondrial phylogenies: Implications for the status of the enigmatic fossil Eurotamandua</span><span class="source" tagx="source" title="source">Proc R Soc Lond B Biol Sci</span><span class="volume" tagx="volume" title="volume">268</span><span class="fpage" tagx="fpage" title="fpage">1605</span><span class="lpage" tagx="lpage" title="lpage">1615</span></span></li> <li tag="ref"><a name="pgen-0020109-b007" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Delsuc</span><span class="given-names" tagx="given-names" title="given-names">F</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Scally</span><span class="given-names" tagx="given-names" title="given-names">M</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Madsen</span><span class="given-names" tagx="given-names" title="given-names">O</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Stanhope</span><span class="given-names" tagx="given-names" title="given-names">MJ</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">de Jong</span><span class="given-names" tagx="given-names" title="given-names">WW</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="year" tagx="year" title="year">2002</span><span class="mixed-article-title" title="mixed-article-title">Molecular phylogeny of living Xenarthrans and the impact of character and taxon sampling on the placental tree rooting</span><span class="source" tagx="source" title="source">Mol Biol Evol</span><span class="volume" tagx="volume" title="volume">19</span><span class="fpage" tagx="fpage" title="fpage">1656</span><span class="lpage" tagx="lpage" title="lpage">1671</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12270893">12270893</a></span></span></li> <li tag="ref"><a name="pgen-0020109-b008" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Delsuc</span><span class="given-names" tagx="given-names" title="given-names">F</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Stanhope</span><span class="given-names" tagx="given-names" title="given-names">MJ</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Douzery</span><span class="given-names" tagx="given-names" title="given-names">EJ</span></span></span><span class="year" tagx="year" title="year">2003</span><span class="mixed-article-title" title="mixed-article-title">Molecular systematics of armadillos (Xenarthra, Dasypodidae): Contribution of maximum likelihood and Bayesian analyses of mitochondrial and nuclear genes</span><span class="source" tagx="source" title="source">Mol Phylogenet Evol</span><span class="volume" tagx="volume" title="volume">28</span><span class="fpage" tagx="fpage" title="fpage">261</span><span class="lpage" tagx="lpage" title="lpage">275</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12878463">12878463</a></span></span></li> <li tag="ref"><a name="pgen-0020109-b009" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Barros</span><span class="given-names" tagx="given-names" title="given-names">MC</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Sampaio</span><span class="given-names" tagx="given-names" title="given-names">I</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Schneider</span><span class="given-names" tagx="given-names" title="given-names">H</span></span></span><span class="year" tagx="year" title="year">2003</span><span class="mixed-article-title" title="mixed-article-title">Phylogenetic analysis of 16S mitochrondrial DNA data in sloths and anteaters</span><span class="source" tagx="source" title="source">Genet Mol Biol</span><span class="volume" tagx="volume" title="volume">26</span><span class="fpage" tagx="fpage" title="fpage">5</span><span class="lpage" tagx="lpage" title="lpage">11</span></span></li> <li tag="ref"><a name="pgen-0020109-b010" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Pereira</span><span class="given-names" tagx="given-names" title="given-names">HRJ</span><span class="suffix" tagx="suffix" title="suffix">Jr</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Jorge</span><span class="given-names" tagx="given-names" title="given-names">W</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Costa</span><span class="given-names" tagx="given-names" title="given-names">MELT</span></span></span><span class="year" tagx="year" title="year">2004</span><span class="mixed-article-title" title="mixed-article-title">Chromosome study of anteaters (Myrmecophagidae, Xenarthra)—A preliminary report</span><span class="source" tagx="source" title="source">Genet Mol Biol</span><span class="volume" tagx="volume" title="volume">27</span><span class="fpage" tagx="fpage" title="fpage">391</span><span class="lpage" tagx="lpage" title="lpage">394</span></span></li> <li tag="ref"><a name="pgen-0020109-b011" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Jorge</span><span class="given-names" tagx="given-names" title="given-names">W</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Meritt</span><span class="given-names" tagx="given-names" title="given-names">DA</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Benirschke</span><span class="given-names" tagx="given-names" title="given-names">K</span></span></span><span class="year" tagx="year" title="year">1978</span><span class="mixed-article-title" title="mixed-article-title">Chromosome studies in Edentata</span><span class="source" tagx="source" title="source">Cytobios</span><span class="volume" tagx="volume" title="volume">18</span><span class="fpage" tagx="fpage" title="fpage">157</span><span class="lpage" tagx="lpage" title="lpage">172</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/679723">679723</a></span></span></li> <li tag="ref"><a name="pgen-0020109-b012" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Barroso</span><span class="given-names" tagx="given-names" title="given-names">CM</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Seuánez</span><span class="given-names" tagx="given-names" title="given-names">H</span></span></span><span class="year" tagx="year" title="year">1991</span><span class="mixed-article-title" title="mixed-article-title">Chromosome studies on <i>Dasypus, Euphractus</i> and <i>Cabassous</i> genera (Edentata: Dasypodidae)</span><span class="source" tagx="source" title="source">Cytobios</span><span class="volume" tagx="volume" title="volume">68</span><span class="fpage" tagx="fpage" title="fpage">179</span><span class="lpage" tagx="lpage" title="lpage">196</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/1813217">1813217</a></span></span></li> <li tag="ref"><a name="pgen-0020109-b013" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Dobigny</span><span class="given-names" tagx="given-names" title="given-names">G</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Yang</span><span class="given-names" tagx="given-names" title="given-names">F</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">O'Brien</span><span class="given-names" tagx="given-names" title="given-names">PC</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Volobouev</span><span class="given-names" tagx="given-names" title="given-names">V</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kovács</span><span class="given-names" tagx="given-names" title="given-names">A</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="year" tagx="year" title="year">2005</span><span class="mixed-article-title" title="mixed-article-title">Low rate of genomic repatterning in Xenarthra inferred from chromosome painting data</span><span class="source" tagx="source" title="source">Chromosome Res</span><span class="volume" tagx="volume" title="volume">13</span><span class="fpage" tagx="fpage" title="fpage">651</span><span class="lpage" tagx="lpage" title="lpage">663</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16235115">16235115</a></span></span></li> <li tag="ref"><a name="pgen-0020109-b014" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Chowdhary</span><span class="given-names" tagx="given-names" title="given-names">BP</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Raudsepp</span><span class="given-names" tagx="given-names" title="given-names">T</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Fronicke</span><span class="given-names" tagx="given-names" title="given-names">L</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Scherthan</span><span class="given-names" tagx="given-names" title="given-names">H</span></span></span><span class="year" tagx="year" title="year">1998</span><span class="mixed-article-title" title="mixed-article-title">Emerging patterns of comparative genome organization in some mammalian species as revealed by Zoo-FISH</span><span class="source" tagx="source" title="source">Genome Res</span><span class="volume" tagx="volume" title="volume">8</span><span class="fpage" tagx="fpage" title="fpage">577</span><span class="lpage" tagx="lpage" title="lpage">589</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/9647633">9647633</a></span></span></li> <li tag="ref"><a name="pgen-0020109-b015" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wienberg</span><span class="given-names" tagx="given-names" title="given-names">J</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Froenicke</span><span class="given-names" tagx="given-names" title="given-names">L</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Stanyon</span><span class="given-names" tagx="given-names" title="given-names">R</span></span></span><span class="year" tagx="year" title="year">2000</span><span class="mixed-article-title" title="mixed-article-title">Insights into mammalian genome organization and evolution by molecular cytogenetics</span><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Clark</span><span class="given-names" tagx="given-names" title="given-names">M</span></span></span><span class="source" tagx="source" title="source">Comparative genomics</span><span class="publisher-loc" tagx="publisher-loc" title="publisher-loc">Boston/Dordrecht/London</span><span class="publisher-name" tagx="publisher-name" title="publisher-name">Kluwer Academic Publishers</span><span class="fpage" tagx="fpage" title="fpage">207</span><span class="lpage" tagx="lpage" title="lpage">244</span></span></li> <li tag="ref"><a name="pgen-0020109-b016" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Murphy</span><span class="given-names" tagx="given-names" title="given-names">WJ</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Stanyon</span><span class="given-names" tagx="given-names" title="given-names">R</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">O'Brien</span><span class="given-names" tagx="given-names" title="given-names">SJ</span></span></span><span class="year" tagx="year" title="year">2001</span><span class="mixed-article-title" title="mixed-article-title">Evolution of mammalian genome organization inferred from comparative gene mapping</span><span class="source" tagx="source" title="source">Genome Biol</span><span class="volume" tagx="volume" title="volume">2</span><span class="comment">REVIEWS0005. E-pub 2001 Jun 5.</span></span></li> <li tag="ref"><a name="pgen-0020109-b017" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Murphy</span><span class="given-names" tagx="given-names" title="given-names">WJ</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Froenicke</span><span class="given-names" tagx="given-names" title="given-names">L</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">O'Brien</span><span class="given-names" tagx="given-names" title="given-names">SJ</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Stanyon</span><span class="given-names" tagx="given-names" title="given-names">R</span></span></span><span class="year" tagx="year" title="year">2003</span><span class="mixed-article-title" title="mixed-article-title">The origin of human Chromosome 1 and its homologs in placental mammals</span><span class="source" tagx="source" title="source">Genome Res</span><span class="volume" tagx="volume" title="volume">13</span><span class="fpage" tagx="fpage" title="fpage">1880</span><span class="lpage" tagx="lpage" title="lpage">1888</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12869576">12869576</a></span></span></li> <li tag="ref"><a name="pgen-0020109-b018" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Richard</span><span class="given-names" tagx="given-names" title="given-names">F</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lombard</span><span class="given-names" tagx="given-names" title="given-names">M</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Dutrillaux</span><span class="given-names" tagx="given-names" title="given-names">B</span></span></span><span class="year" tagx="year" title="year">2003</span><span class="mixed-article-title" title="mixed-article-title">Reconstruction of the ancestral karyotype of Eutherian mammals</span><span class="source" tagx="source" title="source">Chromosome Res</span><span class="volume" tagx="volume" title="volume">11</span><span class="fpage" tagx="fpage" title="fpage">605</span><span class="lpage" tagx="lpage" title="lpage">618</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/14516069">14516069</a></span></span></li> <li tag="ref"><a name="pgen-0020109-b019" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Yang</span><span class="given-names" tagx="given-names" title="given-names">F</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Alkalaeva</span><span class="given-names" tagx="given-names" title="given-names">EZ</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Perelman</span><span class="given-names" tagx="given-names" title="given-names">PL</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Pardini</span><span class="given-names" tagx="given-names" title="given-names">AT</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Harrison</span><span class="given-names" tagx="given-names" title="given-names">WR</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="year" tagx="year" title="year">2003</span><span class="mixed-article-title" title="mixed-article-title">Reciprocal chromosome painting among human, aardvark, and elephant (superorder Afrotheria) reveals the likely Eutherian ancestral karyotype</span><span class="source" tagx="source" title="source">Proc Natl Acad Sci U S A</span><span class="volume" tagx="volume" title="volume">100</span><span class="fpage" tagx="fpage" title="fpage">1062</span><span class="lpage" tagx="lpage" title="lpage">1066</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12552116">12552116</a></span></span></li> <li tag="ref"><a name="pgen-0020109-b020" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Froenicke</span><span class="given-names" tagx="given-names" title="given-names">L</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wienberg</span><span class="given-names" tagx="given-names" title="given-names">J</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Stone</span><span class="given-names" tagx="given-names" title="given-names">G</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Adams</span><span class="given-names" tagx="given-names" title="given-names">L</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Stanyon</span><span class="given-names" tagx="given-names" title="given-names">R</span></span></span><span class="year" tagx="year" title="year">2003</span><span class="mixed-article-title" title="mixed-article-title">Towards the delineation of the ancestral Eutherian genome organization: Comparative genome maps of human and the African elephant <i>(Loxodonta africana)</i> generated by chromosome painting</span><span class="source" tagx="source" title="source">Proc R Soc Lond B Biol Sci</span><span class="volume" tagx="volume" title="volume">270</span><span class="fpage" tagx="fpage" title="fpage">1331</span><span class="lpage" tagx="lpage" title="lpage">1340</span></span></li> <li tag="ref"><a name="pgen-0020109-b021" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Svartman</span><span class="given-names" tagx="given-names" title="given-names">M</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Stone</span><span class="given-names" tagx="given-names" title="given-names">G</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Page</span><span class="given-names" tagx="given-names" title="given-names">JE</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Stanyon</span><span class="given-names" tagx="given-names" title="given-names">R</span></span></span><span class="year" tagx="year" title="year">2004</span><span class="mixed-article-title" title="mixed-article-title">A chromosome painting test of the basal Eutherian karyotype</span><span class="source" tagx="source" title="source">Chromosome Res</span><span class="volume" tagx="volume" title="volume">12</span><span class="fpage" tagx="fpage" title="fpage">45</span><span class="lpage" tagx="lpage" title="lpage">53</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/14984101">14984101</a></span></span></li> <li tag="ref"><a name="pgen-0020109-b022" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Robinson</span><span class="given-names" tagx="given-names" title="given-names">TJ</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Fu</span><span class="given-names" tagx="given-names" title="given-names">B</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ferguson-Smith</span><span class="given-names" tagx="given-names" title="given-names">MA</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Yang</span><span class="given-names" tagx="given-names" title="given-names">F</span></span></span><span class="year" tagx="year" title="year">2004</span><span class="mixed-article-title" title="mixed-article-title">Cross-species chromosome painting in the golden mole and elephant-shrew: Support for the mammalian clades Afrotheria and Afroinsectiphillia but not Afroinsectivora</span><span class="source" tagx="source" title="source">Proc R Soc Lond B Biol Sci</span><span class="volume" tagx="volume" title="volume">271</span><span class="fpage" tagx="fpage" title="fpage">1477</span><span class="lpage" tagx="lpage" title="lpage">1484</span></span></li> <li tag="ref"><a name="pgen-0020109-b023" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Richard</span><span class="given-names" tagx="given-names" title="given-names">F</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lombard</span><span class="given-names" tagx="given-names" title="given-names">M</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Dutrillaux</span><span class="given-names" tagx="given-names" title="given-names">B</span></span></span><span class="year" tagx="year" title="year">2000</span><span class="mixed-article-title" title="mixed-article-title">Phylogenetic origin of human Chromosomes 7, 16, and 19 and their homologs in placental mammals</span><span class="source" tagx="source" title="source">Genome Res</span><span class="volume" tagx="volume" title="volume">10</span><span class="fpage" tagx="fpage" title="fpage">644</span><span class="lpage" tagx="lpage" title="lpage">651</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/10810086">10810086</a></span></span></li> <li tag="ref"><a name="pgen-0020109-b024" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Stanyon</span><span class="given-names" tagx="given-names" title="given-names">R</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Koehler</span><span class="given-names" tagx="given-names" title="given-names">U</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Consigliere</span><span class="given-names" tagx="given-names" title="given-names">S</span></span></span><span class="year" tagx="year" title="year">2002</span><span class="mixed-article-title" title="mixed-article-title">Chromosome painting reveals that galagos have highly derived karyotypes</span><span class="source" tagx="source" title="source">Am J Phys Anthropol</span><span class="volume" tagx="volume" title="volume">117</span><span class="fpage" tagx="fpage" title="fpage">319</span><span class="lpage" tagx="lpage" title="lpage">326</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11920367">11920367</a></span></span></li> <li tag="ref"><a name="pgen-0020109-b025" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Froenicke</span><span class="given-names" tagx="given-names" title="given-names">L</span></span></span><span class="year" tagx="year" title="year">2005</span><span class="mixed-article-title" title="mixed-article-title">Origins of primate chromosomes—As delineated by Zoo-FISH and alignments of human and mouse draft genome sequences</span><span class="source" tagx="source" title="source">Cytogenet Genome Res</span><span class="volume" tagx="volume" title="volume">108</span><span class="fpage" tagx="fpage" title="fpage">122</span><span class="lpage" tagx="lpage" title="lpage">138</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15545724">15545724</a></span></span></li> <li tag="ref"><a name="pgen-0020109-b026" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Murphy</span><span class="given-names" tagx="given-names" title="given-names">WJ</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Larkin</span><span class="given-names" tagx="given-names" title="given-names">DM</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Everts-van der Wind</span><span class="given-names" tagx="given-names" title="given-names">A</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bourque</span><span class="given-names" tagx="given-names" title="given-names">G</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Tesler</span><span class="given-names" tagx="given-names" title="given-names">G</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="year" tagx="year" title="year">2005</span><span class="mixed-article-title" title="mixed-article-title">Dynamics of mammalian chromosome evolution inferred from multispecies comparative maps</span><span class="source" tagx="source" title="source">Science</span><span class="volume" tagx="volume" title="volume">309</span><span class="fpage" tagx="fpage" title="fpage">613</span><span class="lpage" tagx="lpage" title="lpage">617</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16040707">16040707</a></span></span></li> <li tag="ref"><a name="pgen-0020109-b027" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Seabright</span><span class="given-names" tagx="given-names" title="given-names">M</span></span></span><span class="year" tagx="year" title="year">1971</span><span class="mixed-article-title" title="mixed-article-title">A rapid banding technique for human chromosomes</span><span class="source" tagx="source" title="source">Lancet</span><span class="volume" tagx="volume" title="volume">11</span><span class="fpage" tagx="fpage" title="fpage">971</span><span class="lpage" tagx="lpage" title="lpage">972</span></span></li> </ul> </div> </div>  </body></html>