<?xml version="1.0" encoding="UTF-8"?>
<html lang="en" dir="ltr" xmlns="http://www.w3.org/1999/xhtml">
 <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#"> 
  <meta http-equiv="Content-Type" content="text/html; charset=utf-8"/> 
  <meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes"/>
  <meta name="article_thumbnail" content="https://www.biorxiv.org/content/biorxiv/early/2020/01/15/2020.01.15.907709/embed/graphic-7.gif"/>
  <meta name="type" content="article"/>
  <meta name="category" content="article"/>
  <meta name="HW.identifier" content="/biorxiv/early/2020/01/15/2020.01.15.907709.atom"/>
  <meta name="HW.pisa" content="biorxiv;2020.01.15.907709v1"/>
  <meta name="DC.Format" content="text/html"/>
  <meta name="DC.Language" content="en"/>
  <meta name="DC.Title" content="Genomic analyses provide insights into peach local adaptation and responses to climate change"/>
  <meta name="DC.Identifier" content="10.1101/2020.01.15.907709"/>
  <meta name="DC.Date" content="2020-01-15"/>
  <meta name="DC.Publisher" content="Cold Spring Harbor Laboratory"/>
  <meta name="DC.Rights" content="© 2020, Posted by Cold Spring Harbor Laboratory. The copyright holder for this pre-print is the author. All rights reserved. The material may not be redistributed, re-used or adapted without the author's permission."/>
  <meta name="DC.AccessRights" content="restricted"/>
  <meta name="DC.Description" content="The environment has constantly shaped plant genomes, but the genetic bases underlying how plants adapt to environmental influences remain largely unknown. We constructed a high-density genomic variation map by re-sequencing genomes of 263 geographically representative peach landraces and wild relatives. A combination of whole-genome selection scans and genome-wide environmental association studies (GWEAS) was performed to reveal the genomic bases of peach local adaptation to diverse climates comprehensively. A total of 2,092 selective sweeps that underlie local adaptation to both mild and extreme climates were identified, including 339 sweeps conferring genomic pattern of adaptation to high altitudes. Using GWEAS, a total of 3,496 genomic loci strongly associated with 51 specific environmental variables were detected. The molecular mechanism underlying adaptive evolution of high drought, strong UV-B, cold hardiness, sugar content, flesh color, and bloom date were revealed. Finally, based on 30 years of observation, a candidate gene associated with bloom date advance, representing peach responses to global warming, was identified. Collectively, our study provides insights into molecular bases of how environments have shaped peach genomes by natural selection and adds valuable genome resources and candidate genes for future studies on evolutionary genetics, adaptation to climate changes, and future breeding."/>
  <meta name="DC.Contributor" content="Yong Li"/>
  <meta name="DC.Contributor" content="Ke Cao"/>
  <meta name="DC.Contributor" content="Nan Li"/>
  <meta name="DC.Contributor" content="Gengrui Zhu"/>
  <meta name="DC.Contributor" content="Weichao Fang"/>
  <meta name="DC.Contributor" content="Changwen Chen"/>
  <meta name="DC.Contributor" content="Xinwei Wang"/>
  <meta name="DC.Contributor" content="Xiuli Zeng"/>
  <meta name="DC.Contributor" content="Jian Guo"/>
  <meta name="DC.Contributor" content="Shanshan Zhang"/>
  <meta name="DC.Contributor" content="Qi Wang"/>
  <meta name="DC.Contributor" content="Tiyu Ding"/>
  <meta name="DC.Contributor" content="Jiao Wang"/>
  <meta name="DC.Contributor" content="Liping Guan"/>
  <meta name="DC.Contributor" content="Junxiu Wang"/>
  <meta name="DC.Contributor" content="Kuozhan Liu"/>
  <meta name="DC.Contributor" content="Wenwu Guo"/>
  <meta name="DC.Contributor" content="Pere Arús"/>
  <meta name="DC.Contributor" content="Sanwen Huang"/>
  <meta name="DC.Contributor" content="Zhangjun Fei"/>
  <meta name="DC.Contributor" content="Lirong Wang"/>
  <meta name="article:published_time" content="2020-01-15"/>
  <meta name="article:section" content="New Results"/>
  <meta name="citation_title" content="Genomic analyses provide insights into peach local adaptation and responses to climate change"/>
  <meta name="citation_abstract" lang="en" content="&lt;;p&gt;;The environment has constantly shaped plant genomes, but the genetic bases underlying how plants adapt to environmental influences remain largely unknown. We constructed a high-density genomic variation map by re-sequencing genomes of 263 geographically representative peach landraces and wild relatives. A combination of whole-genome selection scans and genome-wide environmental association studies (GWEAS) was performed to reveal the genomic bases of peach local adaptation to diverse climates comprehensively. A total of 2,092 selective sweeps that underlie local adaptation to both mild and extreme climates were identified, including 339 sweeps conferring genomic pattern of adaptation to high altitudes. Using GWEAS, a total of 3,496 genomic loci strongly associated with 51 specific environmental variables were detected. The molecular mechanism underlying adaptive evolution of high drought, strong UV-B, cold hardiness, sugar content, flesh color, and bloom date were revealed. Finally, based on 30 years of observation, a candidate gene associated with bloom date advance, representing peach responses to global warming, was identified. Collectively, our study provides insights into molecular bases of how environments have shaped peach genomes by natural selection and adds valuable genome resources and candidate genes for future studies on evolutionary genetics, adaptation to climate changes, and future breeding.&lt;;/p&gt;;"/>
  <meta name="citation_journal_title" content="bioRxiv"/>
  <meta name="citation_publisher" content="Cold Spring Harbor Laboratory"/>
  <meta name="citation_publication_date" content="2020/01/01"/>
  <meta name="citation_mjid" content="biorxiv;2020.01.15.907709v1"/>
  <meta name="citation_id" content="2020.01.15.907709v1"/>
  <meta name="citation_public_url" content="https://www.biorxiv.org/content/10.1101/2020.01.15.907709v1"/>
  <meta name="citation_abstract_html_url" content="https://www.biorxiv.org/content/10.1101/2020.01.15.907709v1.abstract"/>
  <meta name="citation_full_html_url" content="https://www.biorxiv.org/content/10.1101/2020.01.15.907709v1.full"/>
  <meta name="citation_pdf_url" content="https://www.biorxiv.org/content/biorxiv/early/2020/01/15/2020.01.15.907709.full.pdf"/>
  <meta name="citation_doi" content="10.1101/2020.01.15.907709"/>
  <meta name="citation_num_pages" content="32"/>
  <meta name="citation_article_type" content="Article"/>
  <meta name="citation_section" content="New Results"/>
  <meta name="citation_firstpage" content="2020.01.15.907709"/>
  <meta name="citation_author" content="Yong Li"/>
  <meta name="citation_author_institution" content="Zhengzhou Fruit Research Institute, Chinese Academy of Agricultural Sciences"/>
  <meta name="citation_author_institution" content="Key Laboratory of Horticultural Plant Biology (Ministry of Education), College of Horticulture &amp;; Forestry Sciences, Huazhong Agricultural University"/>
  <meta name="citation_author" content="Ke Cao"/>
  <meta name="citation_author_institution" content="Zhengzhou Fruit Research Institute, Chinese Academy of Agricultural Sciences"/>
  <meta name="citation_author" content="Nan Li"/>
  <meta name="citation_author_institution" content="Agricultural Genome Institute at Shenzhen, Chinese Academy of Agricultural Sciences"/>
  <meta name="citation_author" content="Gengrui Zhu"/>
  <meta name="citation_author_institution" content="Zhengzhou Fruit Research Institute, Chinese Academy of Agricultural Sciences"/>
  <meta name="citation_author" content="Weichao Fang"/>
  <meta name="citation_author_institution" content="Zhengzhou Fruit Research Institute, Chinese Academy of Agricultural Sciences"/>
  <meta name="citation_author" content="Changwen Chen"/>
  <meta name="citation_author_institution" content="Zhengzhou Fruit Research Institute, Chinese Academy of Agricultural Sciences"/>
  <meta name="citation_author" content="Xinwei Wang"/>
  <meta name="citation_author_institution" content="Zhengzhou Fruit Research Institute, Chinese Academy of Agricultural Sciences"/>
  <meta name="citation_author" content="Xiuli Zeng"/>
  <meta name="citation_author_institution" content="Tibet Academy of Agricultural and Animal Husbandry Sciences"/>
  <meta name="citation_author" content="Jian Guo"/>
  <meta name="citation_author_institution" content="Zhengzhou Fruit Research Institute, Chinese Academy of Agricultural Sciences"/>
  <meta name="citation_author" content="Shanshan Zhang"/>
  <meta name="citation_author_institution" content="Tibet Academy of Agricultural and Animal Husbandry Sciences"/>
  <meta name="citation_author" content="Qi Wang"/>
  <meta name="citation_author_institution" content="Zhengzhou Fruit Research Institute, Chinese Academy of Agricultural Sciences"/>
  <meta name="citation_author" content="Tiyu Ding"/>
  <meta name="citation_author_institution" content="Zhengzhou Fruit Research Institute, Chinese Academy of Agricultural Sciences"/>
  <meta name="citation_author" content="Jiao Wang"/>
  <meta name="citation_author_institution" content="Zhengzhou Fruit Research Institute, Chinese Academy of Agricultural Sciences"/>
  <meta name="citation_author" content="Liping Guan"/>
  <meta name="citation_author_institution" content="Zhengzhou Fruit Research Institute, Chinese Academy of Agricultural Sciences"/>
  <meta name="citation_author" content="Junxiu Wang"/>
  <meta name="citation_author_institution" content="Zhengzhou Fruit Research Institute, Chinese Academy of Agricultural Sciences"/>
  <meta name="citation_author" content="Kuozhan Liu"/>
  <meta name="citation_author_institution" content="Zhengzhou Fruit Research Institute, Chinese Academy of Agricultural Sciences"/>
  <meta name="citation_author" content="Wenwu Guo"/>
  <meta name="citation_author_institution" content="Key Laboratory of Horticultural Plant Biology (Ministry of Education), College of Horticulture &amp;; Forestry Sciences, Huazhong Agricultural University"/>
  <meta name="citation_author" content="Pere Arús"/>
  <meta name="citation_author_institution" content="IRTA–Centre de Recerca en Agrigenòmica (CSIC-IRTA-UAB-UB)"/>
  <meta name="citation_author" content="Sanwen Huang"/>
  <meta name="citation_author_institution" content="Agricultural Genome Institute at Shenzhen, Chinese Academy of Agricultural Sciences"/>
  <meta name="citation_author" content="Zhangjun Fei"/>
  <meta name="citation_author_institution" content="Boyce Thompson Institute for Plant Research, Cornell University"/>
  <meta name="citation_author_institution" content="U.S. Department of Agriculture-Agricultural Research Service, Robert W. Holley Center for Agriculture and Health"/>
  <meta name="citation_author_email" content="wanglirong@caas.cn"/>
  <meta name="citation_author_email" content="zf25@cornell.edu"/>
  <meta name="citation_author" content="Lirong Wang"/>
  <meta name="citation_author_institution" content="Zhengzhou Fruit Research Institute, Chinese Academy of Agricultural Sciences"/>
  <meta name="citation_author_email" content="wanglirong@caas.cn"/>
  <meta name="citation_author_email" content="zf25@cornell.edu"/>
  <meta name="citation_reference" content="Alachiotis N, Pavlidis P. 2018. RAiSD detects positive selection based on multiple signatures of a selective sweep and SNP vectors. Commun Biol 1: 79."/>
  <meta name="citation_reference" content="Blanquart F, Kaltz O, Nuismer SL, Gandon S. 2013. A practical guide to measuring local adaptation. Ecol Lett 16: 1195–1205."/>
  <meta name="citation_reference" content="Bolger A, Scossa F, Bolger ME, Lanz C, Maumus F, Tohge T, Quesneville H, Alseekh S, Sørensen I, Lichtenstein G, et al. 2014. The genome of the stress-tolerant wild tomato species Solanum pennellii. Nat Genet 46: 1034–1038."/>
  <meta name="citation_reference" content="Bolger AM, Lohse M, Usadel B. 2014. Trimmomatic: a flexible trimmer for Illumina sequence data. Bioinformatics 30: 2114–2120."/>
  <meta name="citation_reference" content="Cao K, Zheng Z, Wang L, Liu X, Zhu G, Fang W, Cheng S, Zeng P, Chen C, Wang X, et al. 2014. Comparative population genomics reveals the domestication history of the peach, Prunus persica, and human influences on perennial fruit crops. Genome Biol 15: 415."/>
  <meta name="citation_reference" content="Cao K, Zhou Z, Wang Q, Guo J, Zhao P, Zhu G, Fang W, Chen C, Wang X, Wang X, et al. 2016. Genome-wide association study of 12 agronomic traits in peach. Nat Commu 7: 13246."/>
  <meta name="citation_reference" content="Chen X, Schulz-Trieglaff O, Shaw R, Barnes B, Schlesinger F, Källberg M, Cox AJ, Kruglyak S, Saunders CT. 2016. Manta: rapid detection of structural variants and indels for germline and cancer sequencing applications. Bioinformatics 32: 1220–1222."/>
  <meta name="citation_reference" content="Chiang C, Layer RM, Faust GG, Lindberg MR, Rose DB, Garrison EP, Marth GT, Quinlan AR, Hall IM. 2015. SpeedSeq: ultra-fast personal genome analysis and interpretation. Nat Methods 12: 966–968."/>
  <meta name="citation_reference" content="Cingolani P, Platts A, Wang L, Coon M, Nguyen T, Wang L, Land SJ, Lu X, Ruden DM. 2012. A program for annotating and predicting the effects of single nucleotide polymorphisms, SnpEff. Fly 6: 80."/>
  <meta name="citation_reference" content="Danecek P, Auton A, Abecasis G, Albers CA, Banks E, DePristo MA, Handsaker RE, Lunter G, Marth GT, Sherry ST. 2011. The variant call format and VCFtools. Bioinformatics 27: 2156–2158."/>
  <meta name="citation_reference" content="Depristo MA, Banks E, Poplin R, Garimella KV, Maguire JR, Hartl C, Philippakis AA, del Angel G, Rivas MA, Hanna M, et al. 2011. A framework for variation discovery and genotyping using next-generation DNA sequencing data. Nat Genet 43: 491–498."/>
  <meta name="citation_reference" content="Falchi R, Vendramin E, Zanon L, Scalabrin S, Cipriani G, Verde I, Vizzotto G, Morgante M. 2013. Three distinct mutational mechanisms acting on a single gene underpin the origin of yellow flesh in peach. Plant J 76: 75–87."/>
  <meta name="citation_reference" content="Fan S, Bielenberg DG, Zhebentyayeva TN, Reighard GL, Okie WR, Holland D, Abbott AG. 2010. Mapping quantitative trait loci associated with chilling requirement, heat requirement and bloom date in peach (Prunus persica). New Phytol 185: 917–930."/>
  <meta name="citation_reference" content="Felsenstein J. 1989. PHYLIP-phylogeny inference package (version 3.2). Cladistics 5: 164–166."/>
  <meta name="citation_reference" content="Ferrer JL, Jez JM, Bowman ME, Dixon RA, Noel JP. 1999. Structure of chalcone synthase and the molecular basis of plant polyketide pathway. Nat Struct Biol 6: 775–784."/>
  <meta name="citation_reference" content="Fournier-Level A, Korte A, Cooper MD, Nordborg M, Schmitt J, Wilczek AM. 2011. A map of local adaptation in Arabidopsis thaliana. Science 334: 83–86."/>
  <meta name="citation_reference" content="Frazee AC, Pertea G, Jaffe AE, Langmead B, Salzberg SL, Leek JT. 2015. Ballgown bridges the gap between transcriptome assembly and expression analysis. Nat Biotech 33: 243–246."/>
  <meta name="citation_reference" content="Hancock AM, Brachi B, Faure N, Horton MW, Jarymowycz LB, Sperone FG, Toomajian C, Roux F, Bergelson J. 2011. Adaptation to climate across the Arabidopsis thaliana genome. Science 334: 83–86."/>
  <meta name="citation_reference" content="Hara K, Yokoo T, Kajita R, Onishi T, Yahata S, Peterson KM, Torii KU, Kakimoto T. 2009. Epidermal cell density is autoregulated via a secretory peptide, EPIDERMAL PATTERNING FACTOR 2 in Arabidopsis leaves. Plant Cell Physiol 50: 1019–1031."/>
  <meta name="citation_reference" content="Harter RD. 1983. Effect of soil pH on adsorption of lead, copper, zinc, and nickel. Soil Sci Soc Am J 47: 47–51."/>
  <meta name="citation_reference" content="Hu Y, Jiang L, Wang W, Yu D. 2013. Jasmonate regulates the INDUCER OF CBF EXPRESSION–C-REPEAT BINDING FACTOR/DRE BINDING FACTOR1 Cascade and freezing tolerance in Arabidopsis. Plant Cell 25: 2907–2924."/>
  <meta name="citation_reference" content="Huang DW, Sherman BT, Lempicki RA. 2009. Systematic and integrative analysis of large gene lists using DAVID Bioinformatics Resources. Nat Protoc 4: 44–57."/>
  <meta name="citation_reference" content="Jeffares DC, Jolly C, Hoti M, Speed D, Shaw L, Rallis C, Balloux F, Dessimoz C, Bähler J, Sedlazeck FJ. 2017. Transient structural variations have strong effects on quantitative traits and reproductive isolation in fission yeast. Nat Commun 8: 14061."/>
  <meta name="citation_reference" content="Jeon J, Kim J. 2013. Arabidopsis response regulator1 and Arabidopsis histidine phosphotransfer protein2 (AHP2), AHP3, and AHP5 function in cold signaling. Plant Physiol 161: 408–424."/>
  <meta name="citation_reference" content="Kim D, Langmead B, Salzberg SL. 2015. HISAT: a fast spliced aligner with low memory requirements. Nat Methods 12: 357–360."/>
  <meta name="citation_reference" content="Kobashi K, Gemma H, Iwahori S. 2000. Abscisic acid content and sugar metabolism of peaches grown under water stress. J Amer Soc Hort Sci 125: 425–428."/>
  <meta name="citation_reference" content="Kobashi K, Sugaya S, Gemma H, Iwahori S. 2001. Effect of abscisic acid (ABA) on sugar accumulation in the flesh tissue of peach fruit at the start of the maturation stage. Plant Growth Regul 35: 215–223."/>
  <meta name="citation_reference" content="Lasky JR, Upadhyaya HD, Ramu P, Deshpande S, Hash CT, Bonnette J, Juenger TE, Hyma K, Acharya C, Mitchell SE. 2015. Genome-environment associations in sorghum landraces predict adaptive traits. Sci Adv 1: e1400218."/>
  <meta name="citation_reference" content="Lastdrager J, Hanson J, Smeekens S. 2014. Sugar signals and the control of plant growth and development. J Exp Bot 65: 799–807."/>
  <meta name="citation_reference" content="Li H, Durbin R. 2009. Fast and accurate short read alignment with Burrows-Wheeler transform. Bioinformatics 25: 1754–1760."/>
  <meta name="citation_reference" content="Li H, Durbin R. 2011. Inference of human population history from individual whole-genome sequences. Nature 475: 493–496."/>
  <meta name="citation_reference" content="Li H, Handsaker B, Wysoker A, Fennell T, Ruan J, Homer N, Marth G, Abecasis G, Durbin R, 1000 Genome Project Data Processing Subgroup. 2009. The sequence alignment/map format and SAMtools. Bioinformatics 25: 2078–2079."/>
  <meta name="citation_reference" content="Li J, Oulee TM, Raba R, Amundson RG, Last RL. 1993. Arabidopsis Flavonoid Mutants Are Hypersensitive to UV-B Irradiation. Plant Cell 5: 71–179."/>
  <meta name="citation_reference" content="Li Y, Cao K, Zhu G, Fang W, Chen C, Wang X, Zhao P, Guo J, Ding T, Guan L, et al. 2019. Genomic analyses of an extensive collection of wild and cultivated accessions provide new insights into peach breeding history. Genome Biol 20(1): 36."/>
  <meta name="citation_reference" content="Li Y, Wang L, Zhu G, Fang W, Cao K, Chen C, Wang X, Wang, X. (2016). Phenological response of peach to climate change exhibits a relatively dramatic trend in China, 1983-2012. Sci Hortic-Amsterda 209:192–200."/>
  <meta name="citation_reference" content="Li Z, Reighard GL, Abbott AG, Bielenberg DG. 2009. Dormancy-associated MADS genes from the EVG locus of peach [Prunus persica (L.) Batsch] have distinct seasonal and photoperiodic expression patterns. J Exp Bot 60: 3521–3530."/>
  <meta name="citation_reference" content="Ma Q, Sun M, Lu J, Liu Y, Hu D, Hao Y. 2017. Transcription factor AREB2 is involved in soluble sugar accumulation by activating sugar transporter and amylase genes. Plant Physio 174: 2348–2362."/>
  <meta name="citation_reference" content="Martínez-García PJ, Parfitt DE, Ogundiwin EA, Fass J, Chan HM, Ahmad R, Lurie S, Dandekar A, Gradziel TM, Crisosto CH. 2013. High density SNP mapping and QTL analysis for fruit quality characteristics in peach (Prunus persica L.). Tree Genet &amp;; Genomes 9: 9–36."/>
  <meta name="citation_reference" content="McKenna A, Hanna M, Banks E, Sivachenko A, Cibulskis K, Kernytsky A, Garimella K, Altshuler D, Gabriel S, Daly M, et al. 2010. The Genome Analysis Toolkit: a MapReduce framework for analyzing next-generation DNA sequencing data. Genome Research 20: 1297–1303."/>
  <meta name="citation_reference" content="Menzel A, Sparks TH, Estrella N, Koch E, Aasa A, Ahas P, Alm-Kubler K, Bissolli P, Braslavska O, Briede A, et al. 2006. European phenological response to climate change matches the warming pattern. Glob Chang Biol 12: 1969–1976."/>
  <meta name="citation_reference" content="Mikkelsen MD, Thomashow MF. 2009. A role for circadian evening elements in cold-regulated gene expression in Arabidopsis. Plant J 60: 328–339."/>
  <meta name="citation_reference" content="Monihan SM, Magness CA, Yadegari R, Smith SE, Schumaker KS. 2016. Arabidopsis CALCINEURIN B-LIKE10 functions independently of the SOS pathway during reproductive development in saline conditions. Plant Physio 171: 369–379."/>
  <meta name="citation_reference" content="Murray M, Thompson WF. 1980. Rapid isolation of high molecular weight plant DNA. Nucleic Acids Res 8: 4321–4326."/>
  <meta name="citation_reference" content="Pertea M, Kim D, Pertea G, Leek JT, Salzberg SL. 2016. Transcript-level expression analysis of RNA-seq experiments with HISAT, StringTie and Ballgown. Nat Protoc 11: 1650–1667."/>
  <meta name="citation_reference" content="Pertea ML, Pertea GM, Antonescu CM, Chang TC, Mendell JT, Salzberg SL. 2015. StringTie enables improved reconstruction of a transcriptome from RNA-seq reads. Nat Biotech 33: 290–295."/>
  <meta name="citation_reference" content="Price AL, Patterson NJ, Plenge RM, Weinblatt ME, Shadick NA, Reich D. 2006. Principal components analysis corrects for stratification in genome-wide association studies. Na Genet 38: 904–909."/>
  <meta name="citation_reference" content="Pritchard J, Di Rienzo A. 2010. Adaptation-not by sweeps only. Nat Rev Genet 11: 665–667."/>
  <meta name="citation_reference" content="Rugnone ML, Faigón Soverna A, Sanchez SE, Schlaen RG, Hernando CE, Seymour DK, Mancini E, Chernomoretz A, Weigel D, Más P, et al. 2013. LNK genes integrate light and clock signaling networks at the core of the Arabidopsis oscillator. Proc Natl Acad Sci U S A 110: 12120–12125."/>
  <meta name="citation_reference" content="Sasaki R, Yamane H, Ooka T, Jotatsu H, Kitamura Y, Akagi T, Tao R. 2011. Functional and expressional analyses of PmDAM genes associated with endodormancy in Japanese apricot. Plant Physiol 157: 485–497."/>
  <meta name="citation_reference" content="Seguel A, Cumming JR, Klugh-Stewart K, Cornejo P, Borie F. 2013. The role of arbuscular mycorrhizas in decreasing aluminium phytotoxicity in acidic soils: a review. Mycorrhiza 23: 167–183."/>
  <meta name="citation_reference" content="Tang H, Peng J, Wang P, Risch N. 2005. Estimation of individual admixture: analytical and study design considerations. Genet Epidemiol 28: 289–301."/>
  <meta name="citation_reference" content="Tim W, Braun JV. 2013. Climate change impacts on global food security. Science 341: 508–513."/>
  <meta name="citation_reference" content="Tobias R, Zichner T, Schlattl A, Stütz AM, Benes V, Korbel JO. 2012. Delly: structural variant discovery by integrated paired-end and split-read analysis. Bioinformatics 28: i333–i339."/>
  <meta name="citation_reference" content="Verde I, Abbott AG, Scalabrin S, Jung S, Shu S, Marroni F, Zhebentyayeva T, Dettori MT, Grimwood J, Cattonaro F, et al. 2013. The high-quality draft genome of peach (Prunus persica) identifies unique patterns of genetic diversity, domestication and genome evolution. Nat Genet 45: 487–494."/>
  <meta name="citation_reference" content="Wang J, Ding J, Tan B, Robinson KM, Michelson IH, Johansson A, Nystedt B, Scofield DG, Nilsson O, Jansson S, Street NR, et al. 2018. A major locus controls local adaptation and adaptive life history variation in a perennial plant. Genome Bio 19: 72."/>
  <meta name="citation_reference" content="Wang L, Zhu GR, Fang WC. 2012. Peach genetic resource in China. China Agriculture Press."/>
  <meta name="citation_reference" content="Yan W, Liu H, Zhou X, Li Q, Zhang J, Lu L, Liu T, Liu H, Zhang C, Zhang Z, et al. 2013. Natural variation in Ghd7.1 plays an important role in grain yield and adaptation in rice. Cell Res 23(7): 969–971."/>
  <meta name="citation_reference" content="Zhang Q, Chen W, Sun L, Zhao F, Huang B, Yang W, Tao Y, Wang J, Yuan Z, Fan G, et al. (2012). The genome of Prunus mume. Nat Commun 3: 1318."/>
  <meta name="citation_reference" content="Zheng Y, Crawford GW, Chen X. 2014. Archaeological evidence for peach (Prunus persica) cultivation and domestication in China. PLoS ONE 9: e106595."/>
  <meta name="citation_reference" content="Zhou X, Stephens M. 2012. Genome-wide efficient mixed-model analysis for association studies. Nat Genet 44: 821–824."/>
  <meta name="twitter:title" content="Genomic analyses provide insights into peach local adaptation and responses to climate change"/>
  <meta name="twitter:site" content="@biorxivpreprint"/>
  <meta name="twitter:card" content="summary"/>
  <meta name="twitter:image" content="https://www.biorxiv.org/sites/default/files/images/biorxiv_logo_homepage7-5-small.png"/>
  <meta name="twitter:description" content="The environment has constantly shaped plant genomes, but the genetic bases underlying how plants adapt to environmental influences remain largely unknown. We constructed a high-density genomic variation map by re-sequencing genomes of 263 geographically representative peach landraces and wild relatives. A combination of whole-genome selection scans and genome-wide environmental association studies (GWEAS) was performed to reveal the genomic bases of peach local adaptation to diverse climates comprehensively. A total of 2,092 selective sweeps that underlie local adaptation to both mild and extreme climates were identified, including 339 sweeps conferring genomic pattern of adaptation to high altitudes. Using GWEAS, a total of 3,496 genomic loci strongly associated with 51 specific environmental variables were detected. The molecular mechanism underlying adaptive evolution of high drought, strong UV-B, cold hardiness, sugar content, flesh color, and bloom date were revealed. Finally, based on 30 years of observation, a candidate gene associated with bloom date advance, representing peach responses to global warming, was identified. Collectively, our study provides insights into molecular bases of how environments have shaped peach genomes by natural selection and adds valuable genome resources and candidate genes for future studies on evolutionary genetics, adaptation to climate changes, and future breeding."/>
  <meta name="og-title" property="og:title" content="Genomic analyses provide insights into peach local adaptation and responses to climate change"/>
  <meta name="og-url" property="og:url" content="https://www.biorxiv.org/content/10.1101/2020.01.15.907709v1"/>
  <meta name="og-site-name" property="og:site_name" content="bioRxiv"/>
  <meta name="og-description" property="og:description" content="The environment has constantly shaped plant genomes, but the genetic bases underlying how plants adapt to environmental influences remain largely unknown. We constructed a high-density genomic variation map by re-sequencing genomes of 263 geographically representative peach landraces and wild relatives. A combination of whole-genome selection scans and genome-wide environmental association studies (GWEAS) was performed to reveal the genomic bases of peach local adaptation to diverse climates comprehensively. A total of 2,092 selective sweeps that underlie local adaptation to both mild and extreme climates were identified, including 339 sweeps conferring genomic pattern of adaptation to high altitudes. Using GWEAS, a total of 3,496 genomic loci strongly associated with 51 specific environmental variables were detected. The molecular mechanism underlying adaptive evolution of high drought, strong UV-B, cold hardiness, sugar content, flesh color, and bloom date were revealed. Finally, based on 30 years of observation, a candidate gene associated with bloom date advance, representing peach responses to global warming, was identified. Collectively, our study provides insights into molecular bases of how environments have shaped peach genomes by natural selection and adds valuable genome resources and candidate genes for future studies on evolutionary genetics, adaptation to climate changes, and future breeding."/>
  <meta name="og-type" property="og:type" content="article"/>
  <meta name="og-image" property="og:image" content="https://www.biorxiv.org/sites/default/files/images/biorxiv_logo_homepage7-5-small.png"/>
  <meta name="citation_date" content="2020-01-15"/> 
  <meta name="description" content="bioRxiv - the preprint server for biology, operated by Cold Spring Harbor Laboratory, a research and educational institution"/>
  <meta name="generator" content="Drupal 7 (http://drupal.org)"/> 
  <title>Genomic analyses provide insights into peach local adaptation and responses to climate change | bioRxiv</title> 
 </head>
 <body class="html not-front not-logged-in page-node page-node- page-node-1106353 node-type-highwire-article context-content hw-default-jcode-biorxiv hw-article-type-article hw-article-category-new-results"> 
  <noscript>
   <iframe src="//www.googletagmanager.com/ns.html?id=GTM-M677548" height="0" width="0" style="display:none]]visibility:hidden"/>
  </noscript> 
  <div id="skip-link">
   <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
  </div>
  <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
   <div id="section-header" class="section section-header">
    <div id="zone-branding" class="zone zone-branding clearfix print-display-block container-30">
     <div class="grid-15 prefix-1 region region-branding print-display-block" id="region-branding">
      <div class="region-inner region-branding-inner">
       <div class="branding-data clearfix">
        <div class="logo-img">
         <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0">
          <img alt="bioRxiv" src="https://www.biorxiv.org/sites/default/files/bioRxiv_article.jpg"/>
         </a>
        </div>
       </div>
      </div>
     </div>
     <div class="grid-11 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
      <div class="region-inner region-branding-second-inner">
       <div class="block block-system block-menu block-main-menu block-system-main-menu odd block-without-title" id="block-system-main-menu">
        <div class="block-inner clearfix">
         <div class="content clearfix">
          <div class="menubar-nav">
           <ul class="menu" role="menu">
            <li class="first leaf" role="menuitem">
             <a href="/" title="" class="" data-icon-position="" data-hide-link-title="0">Home</a>
            </li>
            <li class="leaf" role="menuitem">
             <a href="/about-biorxiv" class="" data-icon-position="" data-hide-link-title="0">About</a>
            </li>
            <li class="leaf" role="menuitem">
             <a href="/submit-a-manuscript" class="" data-icon-position="" data-hide-link-title="0">Submit</a>
            </li>
            <li class="last leaf" role="menuitem">
             <a href="/content/alertsrss" title="" class="" data-icon-position="" data-hide-link-title="0">ALERTS / RSS</a>
            </li>
           </ul>
          </div>
         </div>
        </div>
       </div>
       <div class="block block-panels-mini block-biorxiv-search-box block-panels-mini-biorxiv-search-box even block-without-title" id="block-panels-mini-biorxiv-search-box">
        <div class="block-inner clearfix">
         <div class="content clearfix">
          <div class="panel-display panel-1col clearfix" id="mini-panel-biorxiv_search_box">
           <div class="panel-panel panel-col">
            <div>
             <div class="panel-pane pane-highwire-seach-quicksearch">
              <div class="pane-content"> </div>
             </div>
             <div class="panel-separator"/>
             <div class="panel-pane pane-custom pane-2 advanced-search-link">
              <div class="pane-content">
               <a href="/search">Advanced Search</a>
              </div>
             </div>
            </div>
           </div>
          </div>
         </div>
        </div>
       </div>
      </div>
     </div>
    </div>
    <div id="zone-header" class="zone zone-header clearfix container-30"> </div>
   </div>
   <div id="section-content" class="section section-content">
    <div id="zone-content" class="zone zone-content clearfix container-30">
     <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
      <div class="region-inner region-content-inner">
       <a id="main-content"/>
       <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
        <div class="block-inner clearfix">
         <div class="content clearfix">
          <div class="panel-display panels-960-layout jcore-2col-layout">
           <div class="panel-row-wrapper clearfix">
            <div class="main-content-wrapper grid-17 suffix-1 alpha">
             <div class="panel-panel panel-region-content">
              <div class="inside">
               <div class="panel-pane pane-highwire-article-citation">
                <div class="pane-content">
                 <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="1106353" id="node1106353" data-pisa="biorxiv;2020.01.15.907709v1" data-pisa-master="biorxiv;2020.01.15.907709" data-apath="/biorxiv/early/2020/01/15/2020.01.15.907709.atom" data-hw-author-tooltip-instance="highwire_author_tooltip">
                  <div class="highwire-cite highwire-cite-highwire-article highwire-citation-biorxiv-article-top clearfix has-author-tooltip">
                   <span class="biorxiv-article-type"> New Results </span>
                   <h1 class="highwire-cite-title" id="page-title">Genomic analyses provide insights into peach local adaptation and responses to climate change</h1>
                   <div class="highwire-cite-authors">
                    <span class="highwire-citation-authors">
                     <span class="highwire-citation-author first" data-delta="0">
                      <span class="nlm-given-names">Yong</span>
                      <span class="nlm-surname">Li</span>
                     </span>, 
                     <span class="highwire-citation-author" data-delta="1">
                      <span class="nlm-given-names">Ke</span>
                      <span class="nlm-surname">Cao</span>
                     </span>, 
                     <span class="highwire-citation-author" data-delta="2">
                      <span class="nlm-given-names">Nan</span>
                      <span class="nlm-surname">Li</span>
                     </span>, 
                     <span class="highwire-citation-author" data-delta="3">
                      <span class="nlm-given-names">Gengrui</span>
                      <span class="nlm-surname">Zhu</span>
                     </span>, 
                     <span class="highwire-citation-author" data-delta="4">
                      <span class="nlm-given-names">Weichao</span>
                      <span class="nlm-surname">Fang</span>
                     </span>, 
                     <span class="highwire-citation-author" data-delta="5">
                      <span class="nlm-given-names">Changwen</span>
                      <span class="nlm-surname">Chen</span>
                     </span>, 
                     <span class="highwire-citation-author" data-delta="6">
                      <span class="nlm-given-names">Xinwei</span>
                      <span class="nlm-surname">Wang</span>
                     </span>, 
                     <span class="highwire-citation-author" data-delta="7">
                      <span class="nlm-given-names">Xiuli</span>
                      <span class="nlm-surname">Zeng</span>
                     </span>, 
                     <span class="highwire-citation-author" data-delta="8">
                      <span class="nlm-given-names">Jian</span>
                      <span class="nlm-surname">Guo</span>
                     </span>, 
                     <span class="highwire-citation-author" data-delta="9">
                      <span class="nlm-given-names">Shanshan</span>
                      <span class="nlm-surname">Zhang</span>
                     </span>, 
                     <span class="highwire-citation-author" data-delta="10">
                      <span class="nlm-given-names">Qi</span>
                      <span class="nlm-surname">Wang</span>
                     </span>, 
                     <span class="highwire-citation-author" data-delta="11">
                      <span class="nlm-given-names">Tiyu</span>
                      <span class="nlm-surname">Ding</span>
                     </span>, 
                     <span class="highwire-citation-author" data-delta="12">
                      <span class="nlm-given-names">Jiao</span>
                      <span class="nlm-surname">Wang</span>
                     </span>, 
                     <span class="highwire-citation-author" data-delta="13">
                      <span class="nlm-given-names">Liping</span>
                      <span class="nlm-surname">Guan</span>
                     </span>, 
                     <span class="highwire-citation-author" data-delta="14">
                      <span class="nlm-given-names">Junxiu</span>
                      <span class="nlm-surname">Wang</span>
                     </span>, 
                     <span class="highwire-citation-author" data-delta="15">
                      <span class="nlm-given-names">Kuozhan</span>
                      <span class="nlm-surname">Liu</span>
                     </span>, 
                     <span class="highwire-citation-author" data-delta="16">
                      <span class="nlm-given-names">Wenwu</span>
                      <span class="nlm-surname">Guo</span>
                     </span>, 
                     <span class="highwire-citation-author" data-delta="17">
                      <span class="nlm-given-names">Pere</span>
                      <span class="nlm-surname">Arús</span>
                     </span>, 
                     <span class="highwire-citation-author" data-delta="18">
                      <span class="nlm-given-names">Sanwen</span>
                      <span class="nlm-surname">Huang</span>
                     </span>, 
                     <span class="highwire-citation-author" data-delta="19">
                      <span class="nlm-given-names">Zhangjun</span>
                      <span class="nlm-surname">Fei</span>
                     </span>, 
                     <span class="highwire-citation-author" data-delta="20">
                      <span class="nlm-given-names">Lirong</span>
                      <span class="nlm-surname">Wang</span>
                     </span>
                    </span>
                   </div>
                   <div class="highwire-cite-metadata">
                    <span class="highwire-cite-metadata-doi highwire-cite-metadata">
                     <span class="label">doi:</span> https://doi.org/10.1101/2020.01.15.907709 
                    </span>
                   </div>
                  </div>
                  <div id="hw-article-author-popups-node1106353" style="display: none;">
                   <div class="author-tooltip-0">
                    <div class="author-tooltip-name">Yong Li </div>
                    <div class="author-tooltip-affiliation">
                     <span class="author-tooltip-text">
                      <div class="author-affiliation">
                       <span class="nlm-institution">Zhengzhou Fruit Research Institute, Chinese Academy of Agricultural Sciences</span>, Zhengzhou, 
                       <span class="nlm-country">China</span>
                      </div>
                      <div class="author-affiliation">
                       <span class="nlm-institution">Key Laboratory of Horticultural Plant Biology (Ministry of Education), College of Horticulture [[#38]] Forestry Sciences, Huazhong Agricultural University</span>, Wuhan, 
                       <span class="nlm-country">China</span>
                      </div>
                     </span>
                    </div>
                    <ul class="author-tooltip-find-more">
                     <li class="author-tooltip-gs-link first">
                      <a href="/lookup/google-scholar?link_type=googlescholar&amp;;gs_type=author&amp;;author%5B0%5D=Yong%2BLi%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a>
                     </li>
                     <li class="author-tooltip-pubmed-link">
                      <a href="/lookup/external-ref?access_num=Li%20Y&amp;;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a>
                     </li>
                     <li class="author-site-search-link last">
                      <a href="/search/author1%3AYong%2BLi%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a>
                     </li>
                    </ul>
                   </div>
                   <div class="author-tooltip-1">
                    <div class="author-tooltip-name">Ke Cao </div>
                    <div class="author-tooltip-affiliation">
                     <span class="author-tooltip-text">
                      <div class="author-affiliation">
                       <span class="nlm-institution">Zhengzhou Fruit Research Institute, Chinese Academy of Agricultural Sciences</span>, Zhengzhou, 
                       <span class="nlm-country">China</span>
                      </div>
                     </span>
                    </div>
                    <ul class="author-tooltip-find-more">
                     <li class="author-tooltip-gs-link first">
                      <a href="/lookup/google-scholar?link_type=googlescholar&amp;;gs_type=author&amp;;author%5B0%5D=Ke%2BCao%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a>
                     </li>
                     <li class="author-tooltip-pubmed-link">
                      <a href="/lookup/external-ref?access_num=Cao%20K&amp;;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a>
                     </li>
                     <li class="author-site-search-link last">
                      <a href="/search/author1%3AKe%2BCao%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a>
                     </li>
                    </ul>
                   </div>
                   <div class="author-tooltip-2">
                    <div class="author-tooltip-name">Nan Li </div>
                    <div class="author-tooltip-affiliation">
                     <span class="author-tooltip-text">
                      <div class="author-affiliation">
                       <span class="nlm-institution">Agricultural Genome Institute at Shenzhen, Chinese Academy of Agricultural Sciences</span>, Shenzhen, 
                       <span class="nlm-country">China</span>
                      </div>
                     </span>
                    </div>
                    <ul class="author-tooltip-find-more">
                     <li class="author-tooltip-gs-link first">
                      <a href="/lookup/google-scholar?link_type=googlescholar&amp;;gs_type=author&amp;;author%5B0%5D=Nan%2BLi%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a>
                     </li>
                     <li class="author-tooltip-pubmed-link">
                      <a href="/lookup/external-ref?access_num=Li%20N&amp;;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a>
                     </li>
                     <li class="author-site-search-link last">
                      <a href="/search/author1%3ANan%2BLi%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a>
                     </li>
                    </ul>
                   </div>
                   <div class="author-tooltip-3">
                    <div class="author-tooltip-name">Gengrui Zhu </div>
                    <div class="author-tooltip-affiliation">
                     <span class="author-tooltip-text">
                      <div class="author-affiliation">
                       <span class="nlm-institution">Zhengzhou Fruit Research Institute, Chinese Academy of Agricultural Sciences</span>, Zhengzhou, 
                       <span class="nlm-country">China</span>
                      </div>
                     </span>
                    </div>
                    <ul class="author-tooltip-find-more">
                     <li class="author-tooltip-gs-link first">
                      <a href="/lookup/google-scholar?link_type=googlescholar&amp;;gs_type=author&amp;;author%5B0%5D=Gengrui%2BZhu%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a>
                     </li>
                     <li class="author-tooltip-pubmed-link">
                      <a href="/lookup/external-ref?access_num=Zhu%20G&amp;;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a>
                     </li>
                     <li class="author-site-search-link last">
                      <a href="/search/author1%3AGengrui%2BZhu%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a>
                     </li>
                    </ul>
                   </div>
                   <div class="author-tooltip-4">
                    <div class="author-tooltip-name">Weichao Fang </div>
                    <div class="author-tooltip-affiliation">
                     <span class="author-tooltip-text">
                      <div class="author-affiliation">
                       <span class="nlm-institution">Zhengzhou Fruit Research Institute, Chinese Academy of Agricultural Sciences</span>, Zhengzhou, 
                       <span class="nlm-country">China</span>
                      </div>
                     </span>
                    </div>
                    <ul class="author-tooltip-find-more">
                     <li class="author-tooltip-gs-link first">
                      <a href="/lookup/google-scholar?link_type=googlescholar&amp;;gs_type=author&amp;;author%5B0%5D=Weichao%2BFang%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a>
                     </li>
                     <li class="author-tooltip-pubmed-link">
                      <a href="/lookup/external-ref?access_num=Fang%20W&amp;;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a>
                     </li>
                     <li class="author-site-search-link last">
                      <a href="/search/author1%3AWeichao%2BFang%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a>
                     </li>
                    </ul>
                   </div>
                   <div class="author-tooltip-5">
                    <div class="author-tooltip-name">Changwen Chen </div>
                    <div class="author-tooltip-affiliation">
                     <span class="author-tooltip-text">
                      <div class="author-affiliation">
                       <span class="nlm-institution">Zhengzhou Fruit Research Institute, Chinese Academy of Agricultural Sciences</span>, Zhengzhou, 
                       <span class="nlm-country">China</span>
                      </div>
                     </span>
                    </div>
                    <ul class="author-tooltip-find-more">
                     <li class="author-tooltip-gs-link first">
                      <a href="/lookup/google-scholar?link_type=googlescholar&amp;;gs_type=author&amp;;author%5B0%5D=Changwen%2BChen%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a>
                     </li>
                     <li class="author-tooltip-pubmed-link">
                      <a href="/lookup/external-ref?access_num=Chen%20C&amp;;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a>
                     </li>
                     <li class="author-site-search-link last">
                      <a href="/search/author1%3AChangwen%2BChen%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a>
                     </li>
                    </ul>
                   </div>
                   <div class="author-tooltip-6">
                    <div class="author-tooltip-name">Xinwei Wang </div>
                    <div class="author-tooltip-affiliation">
                     <span class="author-tooltip-text">
                      <div class="author-affiliation">
                       <span class="nlm-institution">Zhengzhou Fruit Research Institute, Chinese Academy of Agricultural Sciences</span>, Zhengzhou, 
                       <span class="nlm-country">China</span>
                      </div>
                     </span>
                    </div>
                    <ul class="author-tooltip-find-more">
                     <li class="author-tooltip-gs-link first">
                      <a href="/lookup/google-scholar?link_type=googlescholar&amp;;gs_type=author&amp;;author%5B0%5D=Xinwei%2BWang%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a>
                     </li>
                     <li class="author-tooltip-pubmed-link">
                      <a href="/lookup/external-ref?access_num=Wang%20X&amp;;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a>
                     </li>
                     <li class="author-site-search-link last">
                      <a href="/search/author1%3AXinwei%2BWang%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a>
                     </li>
                    </ul>
                   </div>
                   <div class="author-tooltip-7">
                    <div class="author-tooltip-name">Xiuli Zeng </div>
                    <div class="author-tooltip-affiliation">
                     <span class="author-tooltip-text">
                      <div class="author-affiliation">
                       <span class="nlm-institution">Tibet Academy of Agricultural and Animal Husbandry Sciences</span>, Lhasa, 
                       <span class="nlm-country">China</span>
                      </div>
                     </span>
                    </div>
                    <ul class="author-tooltip-find-more">
                     <li class="author-tooltip-gs-link first">
                      <a href="/lookup/google-scholar?link_type=googlescholar&amp;;gs_type=author&amp;;author%5B0%5D=Xiuli%2BZeng%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a>
                     </li>
                     <li class="author-tooltip-pubmed-link">
                      <a href="/lookup/external-ref?access_num=Zeng%20X&amp;;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a>
                     </li>
                     <li class="author-site-search-link last">
                      <a href="/search/author1%3AXiuli%2BZeng%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a>
                     </li>
                    </ul>
                   </div>
                   <div class="author-tooltip-8">
                    <div class="author-tooltip-name">Jian Guo </div>
                    <div class="author-tooltip-affiliation">
                     <span class="author-tooltip-text">
                      <div class="author-affiliation">
                       <span class="nlm-institution">Zhengzhou Fruit Research Institute, Chinese Academy of Agricultural Sciences</span>, Zhengzhou, 
                       <span class="nlm-country">China</span>
                      </div>
                     </span>
                    </div>
                    <ul class="author-tooltip-find-more">
                     <li class="author-tooltip-gs-link first">
                      <a href="/lookup/google-scholar?link_type=googlescholar&amp;;gs_type=author&amp;;author%5B0%5D=Jian%2BGuo%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a>
                     </li>
                     <li class="author-tooltip-pubmed-link">
                      <a href="/lookup/external-ref?access_num=Guo%20J&amp;;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a>
                     </li>
                     <li class="author-site-search-link last">
                      <a href="/search/author1%3AJian%2BGuo%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a>
                     </li>
                    </ul>
                   </div>
                   <div class="author-tooltip-9">
                    <div class="author-tooltip-name">Shanshan Zhang </div>
                    <div class="author-tooltip-affiliation">
                     <span class="author-tooltip-text">
                      <div class="author-affiliation">
                       <span class="nlm-institution">Tibet Academy of Agricultural and Animal Husbandry Sciences</span>, Lhasa, 
                       <span class="nlm-country">China</span>
                      </div>
                     </span>
                    </div>
                    <ul class="author-tooltip-find-more">
                     <li class="author-tooltip-gs-link first">
                      <a href="/lookup/google-scholar?link_type=googlescholar&amp;;gs_type=author&amp;;author%5B0%5D=Shanshan%2BZhang%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a>
                     </li>
                     <li class="author-tooltip-pubmed-link">
                      <a href="/lookup/external-ref?access_num=Zhang%20S&amp;;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a>
                     </li>
                     <li class="author-site-search-link last">
                      <a href="/search/author1%3AShanshan%2BZhang%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a>
                     </li>
                    </ul>
                   </div>
                   <div class="author-tooltip-10">
                    <div class="author-tooltip-name">Qi Wang </div>
                    <div class="author-tooltip-affiliation">
                     <span class="author-tooltip-text">
                      <div class="author-affiliation">
                       <span class="nlm-institution">Zhengzhou Fruit Research Institute, Chinese Academy of Agricultural Sciences</span>, Zhengzhou, 
                       <span class="nlm-country">China</span>
                      </div>
                     </span>
                    </div>
                    <ul class="author-tooltip-find-more">
                     <li class="author-tooltip-gs-link first">
                      <a href="/lookup/google-scholar?link_type=googlescholar&amp;;gs_type=author&amp;;author%5B0%5D=Qi%2BWang%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a>
                     </li>
                     <li class="author-tooltip-pubmed-link">
                      <a href="/lookup/external-ref?access_num=Wang%20Q&amp;;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a>
                     </li>
                     <li class="author-site-search-link last">
                      <a href="/search/author1%3AQi%2BWang%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a>
                     </li>
                    </ul>
                   </div>
                   <div class="author-tooltip-11">
                    <div class="author-tooltip-name">Tiyu Ding </div>
                    <div class="author-tooltip-affiliation">
                     <span class="author-tooltip-text">
                      <div class="author-affiliation">
                       <span class="nlm-institution">Zhengzhou Fruit Research Institute, Chinese Academy of Agricultural Sciences</span>, Zhengzhou, 
                       <span class="nlm-country">China</span>
                      </div>
                     </span>
                    </div>
                    <ul class="author-tooltip-find-more">
                     <li class="author-tooltip-gs-link first">
                      <a href="/lookup/google-scholar?link_type=googlescholar&amp;;gs_type=author&amp;;author%5B0%5D=Tiyu%2BDing%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a>
                     </li>
                     <li class="author-tooltip-pubmed-link">
                      <a href="/lookup/external-ref?access_num=Ding%20T&amp;;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a>
                     </li>
                     <li class="author-site-search-link last">
                      <a href="/search/author1%3ATiyu%2BDing%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a>
                     </li>
                    </ul>
                   </div>
                   <div class="author-tooltip-12">
                    <div class="author-tooltip-name">Jiao Wang </div>
                    <div class="author-tooltip-affiliation">
                     <span class="author-tooltip-text">
                      <div class="author-affiliation">
                       <span class="nlm-institution">Zhengzhou Fruit Research Institute, Chinese Academy of Agricultural Sciences</span>, Zhengzhou, 
                       <span class="nlm-country">China</span>
                      </div>
                     </span>
                    </div>
                    <ul class="author-tooltip-find-more">
                     <li class="author-tooltip-gs-link first">
                      <a href="/lookup/google-scholar?link_type=googlescholar&amp;;gs_type=author&amp;;author%5B0%5D=Jiao%2BWang%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a>
                     </li>
                     <li class="author-tooltip-pubmed-link">
                      <a href="/lookup/external-ref?access_num=Wang%20J&amp;;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a>
                     </li>
                     <li class="author-site-search-link last">
                      <a href="/search/author1%3AJiao%2BWang%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a>
                     </li>
                    </ul>
                   </div>
                   <div class="author-tooltip-13">
                    <div class="author-tooltip-name">Liping Guan </div>
                    <div class="author-tooltip-affiliation">
                     <span class="author-tooltip-text">
                      <div class="author-affiliation">
                       <span class="nlm-institution">Zhengzhou Fruit Research Institute, Chinese Academy of Agricultural Sciences</span>, Zhengzhou, 
                       <span class="nlm-country">China</span>
                      </div>
                     </span>
                    </div>
                    <ul class="author-tooltip-find-more">
                     <li class="author-tooltip-gs-link first">
                      <a href="/lookup/google-scholar?link_type=googlescholar&amp;;gs_type=author&amp;;author%5B0%5D=Liping%2BGuan%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a>
                     </li>
                     <li class="author-tooltip-pubmed-link">
                      <a href="/lookup/external-ref?access_num=Guan%20L&amp;;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a>
                     </li>
                     <li class="author-site-search-link last">
                      <a href="/search/author1%3ALiping%2BGuan%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a>
                     </li>
                    </ul>
                   </div>
                   <div class="author-tooltip-14">
                    <div class="author-tooltip-name">Junxiu Wang </div>
                    <div class="author-tooltip-affiliation">
                     <span class="author-tooltip-text">
                      <div class="author-affiliation">
                       <span class="nlm-institution">Zhengzhou Fruit Research Institute, Chinese Academy of Agricultural Sciences</span>, Zhengzhou, 
                       <span class="nlm-country">China</span>
                      </div>
                     </span>
                    </div>
                    <ul class="author-tooltip-find-more">
                     <li class="author-tooltip-gs-link first">
                      <a href="/lookup/google-scholar?link_type=googlescholar&amp;;gs_type=author&amp;;author%5B0%5D=Junxiu%2BWang%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a>
                     </li>
                     <li class="author-tooltip-pubmed-link">
                      <a href="/lookup/external-ref?access_num=Wang%20J&amp;;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a>
                     </li>
                     <li class="author-site-search-link last">
                      <a href="/search/author1%3AJunxiu%2BWang%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a>
                     </li>
                    </ul>
                   </div>
                   <div class="author-tooltip-15">
                    <div class="author-tooltip-name">Kuozhan Liu </div>
                    <div class="author-tooltip-affiliation">
                     <span class="author-tooltip-text">
                      <div class="author-affiliation">
                       <span class="nlm-institution">Zhengzhou Fruit Research Institute, Chinese Academy of Agricultural Sciences</span>, Zhengzhou, 
                       <span class="nlm-country">China</span>
                      </div>
                     </span>
                    </div>
                    <ul class="author-tooltip-find-more">
                     <li class="author-tooltip-gs-link first">
                      <a href="/lookup/google-scholar?link_type=googlescholar&amp;;gs_type=author&amp;;author%5B0%5D=Kuozhan%2BLiu%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a>
                     </li>
                     <li class="author-tooltip-pubmed-link">
                      <a href="/lookup/external-ref?access_num=Liu%20K&amp;;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a>
                     </li>
                     <li class="author-site-search-link last">
                      <a href="/search/author1%3AKuozhan%2BLiu%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a>
                     </li>
                    </ul>
                   </div>
                   <div class="author-tooltip-16">
                    <div class="author-tooltip-name">Wenwu Guo </div>
                    <div class="author-tooltip-affiliation">
                     <span class="author-tooltip-text">
                      <div class="author-affiliation">
                       <span class="nlm-institution">Key Laboratory of Horticultural Plant Biology (Ministry of Education), College of Horticulture [[#38]] Forestry Sciences, Huazhong Agricultural University</span>, Wuhan, 
                       <span class="nlm-country">China</span>
                      </div>
                     </span>
                    </div>
                    <ul class="author-tooltip-find-more">
                     <li class="author-tooltip-gs-link first">
                      <a href="/lookup/google-scholar?link_type=googlescholar&amp;;gs_type=author&amp;;author%5B0%5D=Wenwu%2BGuo%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a>
                     </li>
                     <li class="author-tooltip-pubmed-link">
                      <a href="/lookup/external-ref?access_num=Guo%20W&amp;;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a>
                     </li>
                     <li class="author-site-search-link last">
                      <a href="/search/author1%3AWenwu%2BGuo%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a>
                     </li>
                    </ul>
                   </div>
                   <div class="author-tooltip-17">
                    <div class="author-tooltip-name">Pere Arús </div>
                    <div class="author-tooltip-affiliation">
                     <span class="author-tooltip-text">
                      <div class="author-affiliation">
                       <span class="nlm-institution">IRTA–Centre de Recerca en Agrigenòmica (CSIC-IRTA-UAB-UB)</span>, Barcelona, 
                       <span class="nlm-country">Spain</span>
                      </div>
                     </span>
                    </div>
                    <ul class="author-tooltip-find-more">
                     <li class="author-tooltip-gs-link first">
                      <a href="/lookup/google-scholar?link_type=googlescholar&amp;;gs_type=author&amp;;author%5B0%5D=Pere%2BAr%C3%BAs%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a>
                     </li>
                     <li class="author-tooltip-pubmed-link">
                      <a href="/lookup/external-ref?access_num=Ar%C3%BAs%20P&amp;;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a>
                     </li>
                     <li class="author-site-search-link last">
                      <a href="/search/author1%3APere%2BAr%25C3%25BAs%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a>
                     </li>
                    </ul>
                   </div>
                   <div class="author-tooltip-18">
                    <div class="author-tooltip-name">Sanwen Huang </div>
                    <div class="author-tooltip-affiliation">
                     <span class="author-tooltip-text">
                      <div class="author-affiliation">
                       <span class="nlm-institution">Agricultural Genome Institute at Shenzhen, Chinese Academy of Agricultural Sciences</span>, Shenzhen, 
                       <span class="nlm-country">China</span>
                      </div>
                     </span>
                    </div>
                    <ul class="author-tooltip-find-more">
                     <li class="author-tooltip-gs-link first">
                      <a href="/lookup/google-scholar?link_type=googlescholar&amp;;gs_type=author&amp;;author%5B0%5D=Sanwen%2BHuang%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a>
                     </li>
                     <li class="author-tooltip-pubmed-link">
                      <a href="/lookup/external-ref?access_num=Huang%20S&amp;;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a>
                     </li>
                     <li class="author-site-search-link last">
                      <a href="/search/author1%3ASanwen%2BHuang%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a>
                     </li>
                    </ul>
                   </div>
                   <div class="author-tooltip-19">
                    <div class="author-tooltip-name">Zhangjun Fei </div>
                    <div class="author-tooltip-affiliation">
                     <span class="author-tooltip-text">
                      <div class="author-affiliation">
                       <span class="nlm-institution">Boyce Thompson Institute for Plant Research, Cornell University</span>, Ithaca, New York, 
                       <span class="nlm-country">USA</span>
                      </div>
                      <div class="author-affiliation">
                       <span class="nlm-institution">U.S. Department of Agriculture-Agricultural Research Service, Robert W. Holley Center for Agriculture and Health</span>, Ithaca, New York, 
                       <span class="nlm-country">USA</span>
                      </div>
                     </span>
                    </div>
                    <ul class="author-tooltip-find-more">
                     <li class="author-tooltip-gs-link first">
                      <a href="/lookup/google-scholar?link_type=googlescholar&amp;;gs_type=author&amp;;author%5B0%5D=Zhangjun%2BFei%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a>
                     </li>
                     <li class="author-tooltip-pubmed-link">
                      <a href="/lookup/external-ref?access_num=Fei%20Z&amp;;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a>
                     </li>
                     <li class="author-site-search-link">
                      <a href="/search/author1%3AZhangjun%2BFei%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a>
                     </li>
                     <li class="author-corresp-email-link last">
                      <span>For correspondence: 
                       <a href="mailto:wanglirong@caas.cn" class="" data-icon-position="" data-hide-link-title="0">wanglirong@caas.cn</a>
                       <a href="mailto:zf25@cornell.edu" class="" data-icon-position="" data-hide-link-title="0">zf25@cornell.edu</a>
                      </span>
                     </li>
                    </ul>
                   </div>
                   <div class="author-tooltip-20">
                    <div class="author-tooltip-name">Lirong Wang </div>
                    <div class="author-tooltip-affiliation">
                     <span class="author-tooltip-text">
                      <div class="author-affiliation">
                       <span class="nlm-institution">Zhengzhou Fruit Research Institute, Chinese Academy of Agricultural Sciences</span>, Zhengzhou, 
                       <span class="nlm-country">China</span>
                      </div>
                     </span>
                    </div>
                    <ul class="author-tooltip-find-more">
                     <li class="author-tooltip-gs-link first">
                      <a href="/lookup/google-scholar?link_type=googlescholar&amp;;gs_type=author&amp;;author%5B0%5D=Lirong%2BWang%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a>
                     </li>
                     <li class="author-tooltip-pubmed-link">
                      <a href="/lookup/external-ref?access_num=Wang%20L&amp;;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a>
                     </li>
                     <li class="author-site-search-link">
                      <a href="/search/author1%3ALirong%2BWang%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a>
                     </li>
                     <li class="author-corresp-email-link last">
                      <span>For correspondence: 
                       <a href="mailto:wanglirong@caas.cn" class="" data-icon-position="" data-hide-link-title="0">wanglirong@caas.cn</a>
                       <a href="mailto:zf25@cornell.edu" class="" data-icon-position="" data-hide-link-title="0">zf25@cornell.edu</a>
                      </span>
                     </li>
                    </ul>
                   </div>
                  </div>
                 </div>
                </div>
               </div>
               <div class="panel-separator"/>
               <div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs">
                <div class="pane-content">
                 <div class="item-list">
                  <ul class="tabs inline panels-ajax-tab">
                   <li class="first">
                    <a href="/content/10.1101/2020.01.15.907709v1" class="panels-ajax-tab-tab" data-panel-name="biorxiv_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:1106353" data-trigger="" data-url-enabled="1">Abstract</a>
                    <a href="/panels_ajax_tab/biorxiv_tab_art/node:1106353/1" rel="nofollow" style="display:none" class="js-crawler-link"/>
                   </li>
                   <li>
                    <a href="/content/10.1101/2020.01.15.907709v1.full-text" class="panels-ajax-tab-tab" data-panel-name="article_tab_full_text" data-target-id="highwire_article_tabs" data-entity-context="node:1106353" data-trigger="full-text" data-url-enabled="1">Full Text</a>
                    <a href="/panels_ajax_tab/article_tab_full_text/node:1106353/1" rel="nofollow" style="display:none" class="js-crawler-link"/>
                   </li>
                   <li>
                    <a href="/content/10.1101/2020.01.15.907709v1.article-info" class="panels-ajax-tab-tab" data-panel-name="biorxiv_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:1106353" data-trigger="article-info" data-url-enabled="1">Info/History</a>
                    <a href="/panels_ajax_tab/biorxiv_tab_info/node:1106353/1" rel="nofollow" style="display:none" class="js-crawler-link"/>
                   </li>
                   <li>
                    <a href="/content/10.1101/2020.01.15.907709v1.article-metrics" class="panels-ajax-tab-tab" data-panel-name="article_tab_metrics" data-target-id="highwire_article_tabs" data-entity-context="node:1106353" data-trigger="article-metrics" data-url-enabled="1">Metrics</a>
                    <a href="/panels_ajax_tab/article_tab_metrics/node:1106353/1" rel="nofollow" style="display:none" class="js-crawler-link"/>
                   </li>
                   <li class="last">
                    <a href="/content/10.1101/2020.01.15.907709v1.full.pdf+html" class="panels-ajax-tab-tab" data-panel-name="biorxiv_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:1106353" data-trigger="full.pdf+html" data-url-enabled="1"> Preview PDF</a>
                    <a href="/panels_ajax_tab/biorxiv_tab_pdf/node:1106353/1" rel="nofollow" style="display:none" class="js-crawler-link"/>
                   </li>
                  </ul>
                 </div>
                </div>
               </div>
               <div class="panel-separator"/>
               <div class="panel-pane pane-highwire-panel-tabs-container">
                <div class="pane-content">
                 <div data-panels-ajax-tab-preloaded="article_tab_full_text" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container">
                  <div class="panels-ajax-tab-loading" style="display:none">
                   <img class="loading" src="https://www.biorxiv.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading"/>
                  </div>
                  <div class="panels-ajax-tab-wrap-article_tab_full_text">
                   <div class="panel-display panel-1col clearfix">
                    <div class="panel-panel panel-col">
                     <div>
                      <div class="panel-pane pane-highwire-markup">
                       <div class="pane-content">
                        <div class="highwire-markup">
                         <div id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip">
                          <span id="related-urls"/>
                         </div>
                         <a href="https://www.biorxiv.org/content/10.1101/2020.01.15.907709v1.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a>
                        </div>
                       </div>
                      </div>
                     </div>
                    </div>
                   </div>
                  </div>
                 </div>
                </div>
               </div>
               <div class="panel-separator"/>
               <div class="panel-pane pane-disqus-comment">
                <div class="pane-content">
                 <div id="disqus_thread">
                  <noscript>
                   <p>
                    <a href="http://biorxivstage.disqus.com/?url=https%3A%2F%2Fwww.biorxiv.org%2Fcontent%2F10.1101%2F2020.01.15.907709v1" class="" data-icon-position="" data-hide-link-title="0">View the discussion thread.</a>
                   </p>
                  </noscript>
                 </div>
                </div>
               </div>
               <div class="panel-separator"/>
               <div class="panel-pane pane-highwire-back-to-top">
                <div class="pane-content">
                 <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"> Back to top</a>
                </div>
               </div>
              </div>
             </div>
            </div>
            <div class="sidebar-right-wrapper grid-10 omega">
             <div class="panel-panel panel-region-sidebar-right">
              <div class="inside">
               <div class="panel-pane pane-highwire-node-pager">
                <div class="pane-content">
                 <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager">
                  <span class="pager-prev">
                   <a href="/content/10.1101/2020.01.15.907881v1" title="Statistical analysis of no observed effect concentrations or levels in eco-toxicological assays with overdispersed count endpoints" rel="prev" class="pager-link-prev link-icon">
                    <span class="title">Previous</span>
                   </a>
                  </span>
                  <span class="pager-next">
                   <a href="/content/10.1101/569319v4" title="Neonatal morphometric similarity mapping for predicting brain age and characterizing neuroanatomic variation associated with preterm birth" rel="next" class="pager-link-next link-icon-right link-icon">
                    <span class="title">Next</span>
                   </a>
                  </span>
                 </div>
                </div>
               </div>
               <div class="panel-separator"/>
               <div class="panel-pane pane-custom pane-1">
                <div class="pane-content"> Posted[[nbsp]]January 15, 2020. </div>
               </div>
               <div class="panel-separator"/>
               <div class="panel-pane pane-panels-mini pane-biorxiv-art-tools">
                <div class="pane-content">
                 <div id="mini-panel-biorxiv_art_tools" class="highwire-2col-stacked panel-display">
                  <div class="panel-row-wrapper clearfix">
                   <div class="content-left-wrapper content-column">
                    <div class="panel-panel panel-region-content-left">
                     <div class="inside">
                      <div class="panel-pane pane-highwire-variant-link">
                       <div class="pane-content">
                        <a href="/content/10.1101/2020.01.15.907709v1.full.pdf" target="_self" class="article-dl-pdf-link link-icon">
                         <span class="title">Download PDF</span>
                        </a>
                       </div>
                      </div>
                     </div>
                    </div>
                   </div>
                   <div class="content-right-wrapper content-column">
                    <div class="panel-panel panel-region-content-right">
                     <div class="inside">
                      <div class="panel-pane pane-minipanel-dialog-link pane-biorxiv-art-email">
                       <div class="pane-content">
                        <div class="minipanel-dialog-wrapper">
                         <div class="minipanel-dialog-link-link">
                          <a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="Email this Article" data-icon-position="" data-hide-link-title="0"> Email</a>
                         </div>
                         <div class="minipanel-dialog-link-mini" style="display:none">
                          <div class="panel-display panel-1col clearfix" id="mini-panel-biorxiv_art_email">
                           <div class="panel-panel panel-col">
                            <div>
                             <div class="panel-pane pane-block pane-forward-form pane-forward">
                              <div class="pane-content"> </div>
                             </div>
                            </div>
                           </div>
                          </div>
                         </div>
                        </div>
                       </div>
                      </div>
                      <div class="panel-separator"/>
                      <div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax" id="shareit">
                       <div class="pane-content">
                        <a href="/" class="link-icon">
                         <span class="title">Share</span>
                        </a>
                       </div>
                      </div>
                      <div class="panel-separator"/>
                      <div class="panel-pane pane-panels-mini pane-biorxiv-share highwire_clipboard_form_ajax_shareit">
                       <div class="pane-content">
                        <div class="panel-display omega-12-onecol" id="mini-panel-biorxiv_share">
                         <div class="panel-panel grid-12 panel-region-preface">
                          <div class="inside">
                           <div class="panel-pane pane-highwire-article-citation">
                            <div class="pane-content">
                             <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="1106353" id="node1106353--2" data-pisa="biorxiv;2020.01.15.907709v1" data-pisa-master="biorxiv;2020.01.15.907709" data-seqnum="1106353" data-apath="/biorxiv/early/2020/01/15/2020.01.15.907709.atom">
                              <div class="highwire-cite highwire-cite-highwire-article highwire-citation-biorxiv-article-pap-list clearfix">
                               <div class="highwire-cite-title">
                                <div class="highwire-cite-title">Genomic analyses provide insights into peach local adaptation and responses to climate change</div>
                               </div>
                               <div class="highwire-cite-authors">
                                <span class="highwire-citation-authors">
                                 <span class="highwire-citation-author first" data-delta="0">
                                  <span class="nlm-given-names">Yong</span>
                                  <span class="nlm-surname">Li</span>
                                 </span>, 
                                 <span class="highwire-citation-author" data-delta="1">
                                  <span class="nlm-given-names">Ke</span>
                                  <span class="nlm-surname">Cao</span>
                                 </span>, 
                                 <span class="highwire-citation-author" data-delta="2">
                                  <span class="nlm-given-names">Nan</span>
                                  <span class="nlm-surname">Li</span>
                                 </span>, 
                                 <span class="highwire-citation-author" data-delta="3">
                                  <span class="nlm-given-names">Gengrui</span>
                                  <span class="nlm-surname">Zhu</span>
                                 </span>, 
                                 <span class="highwire-citation-author" data-delta="4">
                                  <span class="nlm-given-names">Weichao</span>
                                  <span class="nlm-surname">Fang</span>
                                 </span>, 
                                 <span class="highwire-citation-author" data-delta="5">
                                  <span class="nlm-given-names">Changwen</span>
                                  <span class="nlm-surname">Chen</span>
                                 </span>, 
                                 <span class="highwire-citation-author" data-delta="6">
                                  <span class="nlm-given-names">Xinwei</span>
                                  <span class="nlm-surname">Wang</span>
                                 </span>, 
                                 <span class="highwire-citation-author" data-delta="7">
                                  <span class="nlm-given-names">Xiuli</span>
                                  <span class="nlm-surname">Zeng</span>
                                 </span>, 
                                 <span class="highwire-citation-author" data-delta="8">
                                  <span class="nlm-given-names">Jian</span>
                                  <span class="nlm-surname">Guo</span>
                                 </span>, 
                                 <span class="highwire-citation-author" data-delta="9">
                                  <span class="nlm-given-names">Shanshan</span>
                                  <span class="nlm-surname">Zhang</span>
                                 </span>, 
                                 <span class="highwire-citation-author" data-delta="10">
                                  <span class="nlm-given-names">Qi</span>
                                  <span class="nlm-surname">Wang</span>
                                 </span>, 
                                 <span class="highwire-citation-author" data-delta="11">
                                  <span class="nlm-given-names">Tiyu</span>
                                  <span class="nlm-surname">Ding</span>
                                 </span>, 
                                 <span class="highwire-citation-author" data-delta="12">
                                  <span class="nlm-given-names">Jiao</span>
                                  <span class="nlm-surname">Wang</span>
                                 </span>, 
                                 <span class="highwire-citation-author" data-delta="13">
                                  <span class="nlm-given-names">Liping</span>
                                  <span class="nlm-surname">Guan</span>
                                 </span>, 
                                 <span class="highwire-citation-author" data-delta="14">
                                  <span class="nlm-given-names">Junxiu</span>
                                  <span class="nlm-surname">Wang</span>
                                 </span>, 
                                 <span class="highwire-citation-author" data-delta="15">
                                  <span class="nlm-given-names">Kuozhan</span>
                                  <span class="nlm-surname">Liu</span>
                                 </span>, 
                                 <span class="highwire-citation-author" data-delta="16">
                                  <span class="nlm-given-names">Wenwu</span>
                                  <span class="nlm-surname">Guo</span>
                                 </span>, 
                                 <span class="highwire-citation-author" data-delta="17">
                                  <span class="nlm-given-names">Pere</span>
                                  <span class="nlm-surname">Arús</span>
                                 </span>, 
                                 <span class="highwire-citation-author" data-delta="18">
                                  <span class="nlm-given-names">Sanwen</span>
                                  <span class="nlm-surname">Huang</span>
                                 </span>, 
                                 <span class="highwire-citation-author" data-delta="19">
                                  <span class="nlm-given-names">Zhangjun</span>
                                  <span class="nlm-surname">Fei</span>
                                 </span>, 
                                 <span class="highwire-citation-author" data-delta="20">
                                  <span class="nlm-given-names">Lirong</span>
                                  <span class="nlm-surname">Wang</span>
                                 </span>
                                </span>
                               </div>
                               <div class="highwire-cite-metadata">
                                <span class="highwire-cite-metadata-journal highwire-cite-metadata">bioRxiv </span>
                                <span class="highwire-cite-metadata-pages highwire-cite-metadata">2020.01.15.907709; </span>
                                <span class="highwire-cite-metadata-doi highwire-cite-metadata">
                                 <span class="doi_label">doi:</span> https://doi.org/10.1101/2020.01.15.907709 
                                </span>
                               </div>
                              </div>
                             </div>
                            </div>
                           </div>
                          </div>
                         </div>
                         <div class="panel-panel grid-12 panel-region-content">
                          <div class="inside">
                           <div class="panel-pane pane-highwire-article-clipboard-copy">
                            <div class="pane-content">
                             <div class="clipboard-copy">
                              <span class="label-url">
                               <label for="dynamic">Share This Article:</label>
                              </span>
                              <span class="input-text-url">
                               <input type="text" id="dynamic" value="https://www.biorxiv.org/content/10.1101/2020.01.15.907709v1" size="50"/>
                              </span>
                              <span class="copy-button button">
                               <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
                              </span>
                             </div>
                            </div>
                           </div>
                          </div>
                         </div>
                         <div class="panel-panel grid-12 panel-region-postscript">
                          <div class="inside">
                           <div class="panel-pane pane-service-links">
                            <div class="pane-content">
                             <div class="service-links">
                              <a href="http://digg.com/submit?phase=2&amp;;url=https%3A//www.biorxiv.org/content/10.1101/2020.01.15.907709v1&amp;;title=Genomic%20analyses%20provide%20insights%20into%20peach%20local%20adaptation%20and%20responses%20to%20climate%20change" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0">
                               <img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo"/>
                              </a>
                              <a href="http://reddit.com/submit?url=https%3A//www.biorxiv.org/content/10.1101/2020.01.15.907709v1&amp;;title=Genomic%20analyses%20provide%20insights%20into%20peach%20local%20adaptation%20and%20responses%20to%20climate%20change" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0">
                               <img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo"/>
                              </a>
                              <a href="http://twitter.com/share?url=https%3A//www.biorxiv.org/content/10.1101/2020.01.15.907709v1&amp;;text=Genomic%20analyses%20provide%20insights%20into%20peach%20local%20adaptation%20and%20responses%20to%20climate%20change" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0">
                               <img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo"/>
                              </a>
                              <a href="http://www.citeulike.org/posturl?url=https%3A//www.biorxiv.org/content/10.1101/2020.01.15.907709v1&amp;;title=Genomic%20analyses%20provide%20insights%20into%20peach%20local%20adaptation%20and%20responses%20to%20climate%20change" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0">
                               <img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo"/>
                              </a>
                              <a href="http://www.facebook.com/sharer.php?u=https%3A//www.biorxiv.org/content/10.1101/2020.01.15.907709v1&amp;;t=Genomic%20analyses%20provide%20insights%20into%20peach%20local%20adaptation%20and%20responses%20to%20climate%20change" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0">
                               <img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo"/>
                              </a>
                              <a href="http://www.google.com/bookmarks/mark?op=add&amp;;bkmk=https%3A//www.biorxiv.org/content/10.1101/2020.01.15.907709v1&amp;;title=Genomic%20analyses%20provide%20insights%20into%20peach%20local%20adaptation%20and%20responses%20to%20climate%20change" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0">
                               <img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo"/>
                              </a>
                              <a href="http://www.mendeley.com/import/?url=https%3A//www.biorxiv.org/content/10.1101/2020.01.15.907709v1&amp;;title=Genomic%20analyses%20provide%20insights%20into%20peach%20local%20adaptation%20and%20responses%20to%20climate%20change" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0">
                               <img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo"/>
                              </a>
                             </div>
                            </div>
                           </div>
                          </div>
                         </div>
                        </div>
                       </div>
                      </div>
                      <div class="panel-separator"/>
                      <div class="panel-pane pane-minipanel-dialog-link pane-biorxiv-cite-tool">
                       <div class="pane-content">
                        <div class="minipanel-dialog-wrapper">
                         <div class="minipanel-dialog-link-link">
                          <a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title="Citation Tools">
                           <span class="title">Citation Tools</span>
                          </a>
                         </div>
                         <div class="minipanel-dialog-link-mini" style="display:none">
                          <div class="panel-display panel-1col clearfix" id="mini-panel-biorxiv_cite_tool">
                           <div class="panel-panel panel-col">
                            <div>
                             <div class="panel-pane pane-highwire-citation-export">
                              <div class="pane-content">
                               <div class="highwire-citation-export">
                                <div class="highwire-citation-info">
                                 <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="1106353" id="node1106353--3" data-pisa="biorxiv;2020.01.15.907709v1" data-pisa-master="biorxiv;2020.01.15.907709" data-seqnum="1106353" data-apath="/biorxiv/early/2020/01/15/2020.01.15.907709.atom">
                                  <div class="highwire-cite highwire-cite-highwire-article highwire-citation-biorxiv-article-pap-list clearfix">
                                   <div class="highwire-cite-title">
                                    <div class="highwire-cite-title">Genomic analyses provide insights into peach local adaptation and responses to climate change</div>
                                   </div>
                                   <div class="highwire-cite-authors">
                                    <span class="highwire-citation-authors">
                                     <span class="highwire-citation-author first" data-delta="0">
                                      <span class="nlm-given-names">Yong</span>
                                      <span class="nlm-surname">Li</span>
                                     </span>, 
                                     <span class="highwire-citation-author" data-delta="1">
                                      <span class="nlm-given-names">Ke</span>
                                      <span class="nlm-surname">Cao</span>
                                     </span>, 
                                     <span class="highwire-citation-author" data-delta="2">
                                      <span class="nlm-given-names">Nan</span>
                                      <span class="nlm-surname">Li</span>
                                     </span>, 
                                     <span class="highwire-citation-author" data-delta="3">
                                      <span class="nlm-given-names">Gengrui</span>
                                      <span class="nlm-surname">Zhu</span>
                                     </span>, 
                                     <span class="highwire-citation-author" data-delta="4">
                                      <span class="nlm-given-names">Weichao</span>
                                      <span class="nlm-surname">Fang</span>
                                     </span>, 
                                     <span class="highwire-citation-author" data-delta="5">
                                      <span class="nlm-given-names">Changwen</span>
                                      <span class="nlm-surname">Chen</span>
                                     </span>, 
                                     <span class="highwire-citation-author" data-delta="6">
                                      <span class="nlm-given-names">Xinwei</span>
                                      <span class="nlm-surname">Wang</span>
                                     </span>, 
                                     <span class="highwire-citation-author" data-delta="7">
                                      <span class="nlm-given-names">Xiuli</span>
                                      <span class="nlm-surname">Zeng</span>
                                     </span>, 
                                     <span class="highwire-citation-author" data-delta="8">
                                      <span class="nlm-given-names">Jian</span>
                                      <span class="nlm-surname">Guo</span>
                                     </span>, 
                                     <span class="highwire-citation-author" data-delta="9">
                                      <span class="nlm-given-names">Shanshan</span>
                                      <span class="nlm-surname">Zhang</span>
                                     </span>, 
                                     <span class="highwire-citation-author" data-delta="10">
                                      <span class="nlm-given-names">Qi</span>
                                      <span class="nlm-surname">Wang</span>
                                     </span>, 
                                     <span class="highwire-citation-author" data-delta="11">
                                      <span class="nlm-given-names">Tiyu</span>
                                      <span class="nlm-surname">Ding</span>
                                     </span>, 
                                     <span class="highwire-citation-author" data-delta="12">
                                      <span class="nlm-given-names">Jiao</span>
                                      <span class="nlm-surname">Wang</span>
                                     </span>, 
                                     <span class="highwire-citation-author" data-delta="13">
                                      <span class="nlm-given-names">Liping</span>
                                      <span class="nlm-surname">Guan</span>
                                     </span>, 
                                     <span class="highwire-citation-author" data-delta="14">
                                      <span class="nlm-given-names">Junxiu</span>
                                      <span class="nlm-surname">Wang</span>
                                     </span>, 
                                     <span class="highwire-citation-author" data-delta="15">
                                      <span class="nlm-given-names">Kuozhan</span>
                                      <span class="nlm-surname">Liu</span>
                                     </span>, 
                                     <span class="highwire-citation-author" data-delta="16">
                                      <span class="nlm-given-names">Wenwu</span>
                                      <span class="nlm-surname">Guo</span>
                                     </span>, 
                                     <span class="highwire-citation-author" data-delta="17">
                                      <span class="nlm-given-names">Pere</span>
                                      <span class="nlm-surname">Arús</span>
                                     </span>, 
                                     <span class="highwire-citation-author" data-delta="18">
                                      <span class="nlm-given-names">Sanwen</span>
                                      <span class="nlm-surname">Huang</span>
                                     </span>, 
                                     <span class="highwire-citation-author" data-delta="19">
                                      <span class="nlm-given-names">Zhangjun</span>
                                      <span class="nlm-surname">Fei</span>
                                     </span>, 
                                     <span class="highwire-citation-author" data-delta="20">
                                      <span class="nlm-given-names">Lirong</span>
                                      <span class="nlm-surname">Wang</span>
                                     </span>
                                    </span>
                                   </div>
                                   <div class="highwire-cite-metadata">
                                    <span class="highwire-cite-metadata-journal highwire-cite-metadata">bioRxiv </span>
                                    <span class="highwire-cite-metadata-pages highwire-cite-metadata">2020.01.15.907709; </span>
                                    <span class="highwire-cite-metadata-doi highwire-cite-metadata">
                                     <span class="doi_label">doi:</span> https://doi.org/10.1101/2020.01.15.907709 
                                    </span>
                                   </div>
                                  </div>
                                 </div>
                                </div>
                                <div class="highwire-citation-formats">
                                 <h2>Citation Manager Formats</h2>
                                 <div class="highwire-citation-formats-links">
                                  <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;;rft.spage&amp;;rft.epage&amp;;rft.atitle=Genomic%20analyses%20provide%20insights%20into%20peach%20local%20adaptation%20and%20responses%20to%20climate%20change&amp;;rft.volume&amp;;rft.issue&amp;;rft.date=2020-01-01%2000%3A00%3A00&amp;;rft.stitle&amp;;rft.jtitle=bioRxiv&amp;;rft.au=Li%2C+Yong&amp;;rft.au=Cao%2C+Ke&amp;;rft.au=Li%2C+Nan&amp;;rft.au=Zhu%2C+Gengrui&amp;;rft.au=Fang%2C+Weichao&amp;;rft.au=Chen%2C+Changwen&amp;;rft.au=Wang%2C+Xinwei&amp;;rft.au=Zeng%2C+Xiuli&amp;;rft.au=Guo%2C+Jian&amp;;rft.au=Zhang%2C+Shanshan&amp;;rft.au=Wang%2C+Qi&amp;;rft.au=Ding%2C+Tiyu&amp;;rft.au=Wang%2C+Jiao&amp;;rft.au=Guan%2C+Liping&amp;;rft.au=Wang%2C+Junxiu&amp;;rft.au=Liu%2C+Kuozhan&amp;;rft.au=Guo%2C+Wenwu&amp;;rft.au=Ar%C3%BAs%2C+Pere&amp;;rft.au=Huang%2C+Sanwen&amp;;rft.au=Fei%2C+Zhangjun&amp;;rft.au=Wang%2C+Lirong"/>
                                  <ul class="hw-citation-links inline button button-alt button-grid clearfix">
                                   <li class="bibtext first">
                                    <a href="/highwire/citation/1106353/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a>
                                   </li>
                                   <li class="bookends">
                                    <a href="/highwire/citation/1106353/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a>
                                   </li>
                                   <li class="easybib">
                                    <a href="/highwire/citation/1106353/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a>
                                   </li>
                                   <li class="endnote-tagged">
                                    <a href="/highwire/citation/1106353/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a>
                                   </li>
                                   <li class="endnote-8-xml">
                                    <a href="/highwire/citation/1106353/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a>
                                   </li>
                                   <li class="medlars">
                                    <a href="/highwire/citation/1106353/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a>
                                   </li>
                                   <li class="mendeley">
                                    <a href="/highwire/citation/1106353/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a>
                                   </li>
                                   <li class="papers">
                                    <a href="/highwire/citation/1106353/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a>
                                   </li>
                                   <li class="refworks-tagged">
                                    <a href="/highwire/citation/1106353/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a>
                                   </li>
                                   <li class="reference-manager">
                                    <a href="/highwire/citation/1106353/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a>
                                   </li>
                                   <li class="ris">
                                    <a href="/highwire/citation/1106353/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a>
                                   </li>
                                   <li class="zotero last">
                                    <a href="/highwire/citation/1106353/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a>
                                   </li>
                                  </ul>
                                 </div>
                                </div>
                               </div>
                              </div>
                             </div>
                            </div>
                           </div>
                          </div>
                         </div>
                        </div>
                       </div>
                      </div>
                     </div>
                    </div>
                   </div>
                  </div> 
                 </div>
                </div>
               </div>
               <div class="panel-separator"/>
               <div class="panel-pane pane-service-links">
                <div class="pane-content">
                 <div class="service-links">
                  <div class="item-list">
                   <ul>
                    <li class="first">
                     <a href="http://twitter.com/share?url=https%3A//www.biorxiv.org/content/10.1101/2020.01.15.907709v1&amp;;count=horizontal&amp;;via=&amp;;text=Genomic%20analyses%20provide%20insights%20into%20peach%20local%20adaptation%20and%20responses%20to%20climate%20change&amp;;counturl=https%3A//www.biorxiv.org/content/10.1101/2020.01.15.907709v1" class="twitter-share-button service-links-twitter-widget" id="twitter_widget" title="Tweet This" rel="nofollow" data-icon-position="" data-hide-link-title="0">
                      <span class="element-invisible">Tweet Widget</span>
                     </a>
                    </li>
                    <li>
                     <a href="http://www.facebook.com/plugins/like.php?href=https%3A//www.biorxiv.org/content/10.1101/2020.01.15.907709v1&amp;;layout=button_count&amp;;show_faces=false&amp;;action=like&amp;;colorscheme=light&amp;;width=100&amp;;height=21&amp;;font=&amp;;locale=" id="facebook_like" title="I Like it" class="service-links-facebook-like" rel="nofollow" data-icon-position="" data-hide-link-title="0">
                      <span class="element-invisible">Facebook Like</span>
                     </a>
                    </li>
                    <li class="last">
                     <a href="https://www.biorxiv.org/content/10.1101/2020.01.15.907709v1" id="google_plus_one" title="Plus it" class="service-links-google-plus-one" rel="nofollow" data-icon-position="" data-hide-link-title="0">
                      <span class="element-invisible">Google Plus One</span>
                     </a>
                    </li>
                   </ul>
                  </div>
                 </div>
                </div>
               </div>
               <div class="panel-separator"/>
               <div class="panel-pane pane-highwire-article-collections">
                <h2 class="pane-title">Subject Area</h2>
                <div class="pane-content">
                 <div class="highwire-list-wrapper highwire-article-collections">
                  <div class="highwire-list">
                   <ul class="highwire-article-collection-term-list">
                    <li class="first last odd">
                     <span class="highwire-article-collection-term">
                      <a href="/collection/genomics" class="highlight" data-icon-position="" data-hide-link-title="0">Genomics </a>
                     </span>
                    </li>
                   </ul>
                  </div>
                 </div>
                </div>
               </div>
               <div class="panel-separator"/>
               <div class="panel-pane pane-panels-mini pane-biorxiv-subject-collections block-style-col2">
                <div class="pane-content">
                 <div class="panel-flexible panels-flexible-8 clearfix" id="mini-panel-biorxiv_subject_collections">
                  <div class="panel-flexible-inside panels-flexible-8-inside">
                   <div class="panels-flexible-region panels-flexible-region-8-center panels-flexible-region-first panels-flexible-region-last">
                    <div class="inside panels-flexible-region-inside panels-flexible-region-8-center-inside panels-flexible-region-inside-first panels-flexible-region-inside-last">
                     <div class="panel-pane pane-snippet">
                      <div class="pane-content">
                       <div class="snippet biorxiv-subject-areas-table-title" id="biorxiv-subject-areas-table-title">
                        <div class="snippet-content"> </div>
                       </div>
                      </div>
                     </div>
                     <div class="panel-separator"/>
                     <div class="panel-pane pane-snippet">
                      <div class="pane-content">
                       <div class="snippet biorxiv-subject-areas-view-papers" id="biorxiv-subject-areas-view-papers">
                        <div class="snippet-content">
                         <a href="/content/early/recent">
                          <strong>All Articles</strong>
                         </a>
                        </div>
                       </div>
                      </div>
                     </div>
                     <div class="panel-separator"/>
                     <div class="panel-pane pane-highwire-subject-collections">
                      <div class="pane-content">
                       <ul id="collection" class="collection highwire-list-expand">
                        <li class="outer collection depth-2 child first">
                         <div class="data-wrapper">
                          <a href="/collection/animal-behavior-and-cognition" class="" data-icon-position="" data-hide-link-title="0">Animal Behavior and Cognition</a>
                          <span class="article-count">(1674)</span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/biochemistry" class="" data-icon-position="" data-hide-link-title="0">Biochemistry</a>
                          <span class="article-count">(2825)</span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/bioengineering" class="" data-icon-position="" data-hide-link-title="0">Bioengineering</a>
                          <span class="article-count">(1963)</span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/bioinformatics" class="" data-icon-position="" data-hide-link-title="0">Bioinformatics</a>
                          <span class="article-count">(10456)</span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/biophysics" class="" data-icon-position="" data-hide-link-title="0">Biophysics</a>
                          <span class="article-count">(4275)</span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/cancer-biology" class="" data-icon-position="" data-hide-link-title="0">Cancer Biology</a>
                          <span class="article-count">(3301)</span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/cell-biology" class="" data-icon-position="" data-hide-link-title="0">Cell Biology</a>
                          <span class="article-count">(4686)</span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/clinical-trials" class="" data-icon-position="" data-hide-link-title="0">Clinical Trials</a>
                          <span class="article-count">(136)</span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/developmental-biology" class="" data-icon-position="" data-hide-link-title="0">Developmental Biology</a>
                          <span class="article-count">(2911)</span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/ecology" class="" data-icon-position="" data-hide-link-title="0">Ecology</a>
                          <span class="article-count">(4575)</span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/epidemiology" class="" data-icon-position="" data-hide-link-title="0">Epidemiology</a>
                          <span class="article-count">(2041)</span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/evolutionary-biology" class="" data-icon-position="" data-hide-link-title="0">Evolutionary Biology</a>
                          <span class="article-count">(7337)</span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/genetics" class="" data-icon-position="" data-hide-link-title="0">Genetics</a>
                          <span class="article-count">(5574)</span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/genomics" class="" data-icon-position="" data-hide-link-title="0">Genomics</a>
                          <span class="article-count">(6938)</span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/immunology" class="" data-icon-position="" data-hide-link-title="0">Immunology</a>
                          <span class="article-count">(2463)</span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/microbiology" class="" data-icon-position="" data-hide-link-title="0">Microbiology</a>
                          <span class="article-count">(7696)</span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/molecular-biology" class="" data-icon-position="" data-hide-link-title="0">Molecular Biology</a>
                          <span class="article-count">(3078)</span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/neuroscience" class="" data-icon-position="" data-hide-link-title="0">Neuroscience</a>
                          <span class="article-count">(19031)</span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/paleontology" class="" data-icon-position="" data-hide-link-title="0">Paleontology</a>
                          <span class="article-count">(142)</span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/pathology" class="" data-icon-position="" data-hide-link-title="0">Pathology</a>
                          <span class="article-count">(481)</span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/pharmacology-and-toxicology" class="" data-icon-position="" data-hide-link-title="0">Pharmacology and Toxicology</a>
                          <span class="article-count">(796)</span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/physiology" class="" data-icon-position="" data-hide-link-title="0">Physiology</a>
                          <span class="article-count">(1188)</span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/plant-biology" class="" data-icon-position="" data-hide-link-title="0">Plant Biology</a>
                          <span class="article-count">(2805)</span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/scientific-communication-and-education" class="" data-icon-position="" data-hide-link-title="0">Scientific Communication and Education</a>
                          <span class="article-count">(699)</span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/synthetic-biology" class="" data-icon-position="" data-hide-link-title="0">Synthetic Biology</a>
                          <span class="article-count">(919)</span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/systems-biology" class="" data-icon-position="" data-hide-link-title="0">Systems Biology</a>
                          <span class="article-count">(2898)</span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child last">
                         <div class="data-wrapper">
                          <a href="/collection/zoology" class="" data-icon-position="" data-hide-link-title="0">Zoology</a>
                          <span class="article-count">(473)</span>
                         </div>
                        </li>
                       </ul>
                      </div>
                     </div>
                    </div>
                   </div>
                  </div>
                 </div>
                </div>
               </div>
              </div>
             </div>
            </div>
           </div> 
          </div>
         </div>
        </div>
       </div>
      </div>
     </div>
    </div>
   </div>
  </div>
  <div class="region region-page-bottom" id="region-page-bottom">
   <div class="region-inner region-page-bottom-inner"> </div>
  </div>
  <div class="article fulltext-view ">
   <span class="highwire-journal-article-marker-start"/>
   <div class="section abstract" id="abstract-1">
    <h2>Abstract</h2>
    <p id="p-3">The environment has constantly shaped plant genomes, but the genetic bases underlying how plants adapt to environmental influences remain largely unknown. We constructed a high-density genomic variation map by re-sequencing genomes of 263 geographically representative peach landraces and wild relatives. A combination of whole-genome selection scans and genome-wide environmental association studies (GWEAS) was performed to reveal the genomic bases of peach local adaptation to diverse climates comprehensively. A total of 2,092 selective sweeps that underlie local adaptation to both mild and extreme climates were identified, including 339 sweeps conferring genomic pattern of adaptation to high altitudes. Using GWEAS, a total of 3,496 genomic loci strongly associated with 51 specific environmental variables were detected. The molecular mechanism underlying adaptive evolution of high drought, strong UV-B, cold hardiness, sugar content, flesh color, and bloom date were revealed. Finally, based on 30 years of observation, a candidate gene associated with bloom date advance, representing peach responses to global warming, was identified. Collectively, our study provides insights into molecular bases of how environments have shaped peach genomes by natural selection and adds valuable genome resources and candidate genes for future studies on evolutionary genetics, adaptation to climate changes, and future breeding.</p>
   </div>
   <p id="p-4">Environmental adaptation is fundamental to species survival and conservation of biodiversity, especially under threats of climate change (
    <a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">Blanquart et al. 2013</a>). Unlike animals, which can escape from hostile environments, plants are sessile and have to adapt by shaping and/or fixing genetic variants that are conducive for survival. Generally, climate is the major selective pressure driving adaptive evolution, resulting in different ecotypes within a single species (
    <a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">Hancock et al. 2011</a>; 
    <a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">Fournier-Level et al. 2011</a>). However, the mechanisms underlying how climate shapes plant genomes remain largely unclear. Recently, identifying adaptive variants and understanding molecular mechanism of adaptation across a genome have become tractable due to the advances of sequencing technologies. Recent studies have sought to elucidate genetic bases of adaptation through genome-wide identification of regions under positive selection and/or loci that control adaptive traits in 
    <em>Arabidopsis thaliana</em> (
    <a id="xref-ref-16-2" class="xref-bibr" href="#ref-16">Fournier-Level et al. 2011</a>), rice (
    <a id="xref-ref-57-1" class="xref-bibr" href="#ref-57">Yan et al. 2013</a>), sorghum (
    <a id="xref-ref-28-1" class="xref-bibr" href="#ref-28">Lasky et al. 2015</a>), and poplar (
    <a id="xref-ref-55-1" class="xref-bibr" href="#ref-55">Wang et al. 2018</a>). However, no study has focused on genetic bases of adaptation in domesticated perennial fruit crops. Domesticated crops have adapted to diverse climates during domestication and subsequent spread, and show local adaptation through long-term natural selection. Landraces and wild relatives harbor great genetic diversity and an abundance of resistance genes, which provide excellent resources for breeding initiatives. This is especially the case with accessions originating from stressful environments that may have specific stress-resistance genes (Bolger et al. 2014a). However, a cost of domestication is that many resistance related genes have been lost. In addition, global climate change is driving decreases in productivity and distribution changes in several crop species (
    <a id="xref-ref-52-1" class="xref-bibr" href="#ref-52">Tim and Braun, 2013</a>). Therefore, it is of great importance to identify adaptive genes that can contribute to crop improvement, species survival, and global food security in the face of environmental deterioration.
   </p>
   <p id="p-5">Peach is an important temperate fruit species, with a global yield of 24.7 million tons in 2017 (FAOSTAT; 
    <a href="http://www.fao.org/faostat">http://www.fao.org/faostat</a>). It is also an important model system for the Rosaceae family, members of which provide one of world’s main resources of fruits. Peach originated in southwestern China, and its landraces and wild relatives are widespread in both temperate and sub-tropical regions, as well as in wet and dry climates (
    <a id="xref-ref-56-1" class="xref-bibr" href="#ref-56">Wang et al. 2012</a>). On the grounds of wide distributions, peach can be regarded as an excellent material for studying adaptation genetics. Peach has a relatively small genome size (∼227.4 Mb) (
    <a id="xref-ref-54-1" class="xref-bibr" href="#ref-54">Verde et al. 2013</a>) and genomic analyses have identified a number of loci and candidate genes associated with human selection and agronomically important traits (
    <a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">Cao et al. 2014</a>; 
    <a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">Cao et al. 2016</a>; 
    <a id="xref-ref-34-1" class="xref-bibr" href="#ref-34">Li et al. 2019</a>). However, there have been few studies describing genomic loci associated with environmental adaptation and natural selection.
   </p>
   <p id="p-6">To investigate the genetic basis of local adaptation, we sequenced a wide collection of 263 peach accessions from a broad range of geographical origins and associated with diverse climates, spanning mild and extreme environments. Using the sequencing data, we deciphered adaptive patterns across peach genome by combining the identification of signatures of selective sweeps with genome-wide association studies of environmental variables and adaptive traits. Finally, we identified a candidate gene associated with peach responses to global warming, based on observations over a 30-year period.</p>
   <div class="section" id="sec-1">
    <h2 class="">Results and discussion</h2>
    <div id="sec-2" class="subsection">
     <h3>Genomic variation map and population structure</h3>
     <p id="p-7">We first constructed a genome variation map for peach using a collection of 263 diverse accessions (
      <a id="xref-fig-1-1" class="xref-fig" href="#F1">Fig. 1A</a>), consisting of 52 wild relatives and 211 landraces (Supplementary Table S1), which collectively capture more than 95% of geographic diversity of native distribution of peach landrace and wild relatives. A total of 342.7 Gb of sequence was generated, with a median depth of 5.3 × and coverage of 91.7% of reference peach ‘Lovell’ genome (release v2.0) (
      <a id="xref-ref-54-2" class="xref-bibr" href="#ref-54">Verde et al. 2013</a>) (Supplementary Table S1). We identified a final set of 4,611,842 high-quality single-nucleotide polymorphisms (SNPs) (Supplementary Fig. S1A), of which 1,931,310 were intronic (∼11.33%) and 848,638 (∼4.98%) were exonic. Among SNPs in coding sequence, we found that 7,853 SNPs present in 5,512 peach genes (∼20.5% of total genes) are likely to have a major impact on gene function. The accuracy of identified SNPs was found to be ∼95.6%, based on genotyping of 18 randomly selected SNPs in 130 accessions using a Sequenom MassARRAY platform (Supplementary Table S2). In addition, we identified 1,049,266 small insertions and deletions (INDELs) (shorter than or equal to 6 bp) and 106,388 large structural variations (SVs) (&gt;; 30 bp) (Supplementary Fig. S1A).
     </p>
     <div id="F1" class="fig pos-float type-figure odd">
      <div class="highwire-figure">
       <div class="fig-inline-img-wrapper">
        <div class="fig-inline-img">
         <a href="https://www.biorxiv.org/content/biorxiv/early/2020/01/15/2020.01.15.907709/F1.large.jpg?width=800&amp;;height=600&amp;;carousel=1" title="Summary of 263 samples and genes under selection for seven peach groups. (A) Geographic distribution of 263 peach accessions used in this study. Each accession is represented by a dot on the world map. Seven ecotypes are highlighted using rings with different colors. (B) Demographic history of the seven peach groups. Ancestral population size was inferred using the PSMC model. Three periods of the last glacial maximum (LGM, [[#x223C]]20 KYA), Naynayxungla Glaciation (NG, 0.5[[#x223C]]0.78 MYA), and Xixiabangma Glaciation (XG, 0.8[[#x223C]]0.17 MYA) are shaded in green, red, and blue, respectively. (C) Venn diagram showing the number of genes under selection in the seven groups. (D) Over-represented gene ontology (GO) terms in overall selection regions. Only the top 20 most over-represented terms are shown. YG, Yun-gui Plateau. NW, Northwest China. NP, North Plain China. YT, Yangtze River Middle and Backward. NE, Northeast China. TB, Tibet plateau. ST, South China Sub-tropical." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-709642862" data-figure-caption="&lt;;div class=&quot;;highwire-markup&quot;;&gt;;&lt;;div xmlns=&quot;;http://www.w3.org/1999/xhtml&quot;;&gt;;&lt;;span class=&quot;;caption-title&quot;;&gt;;Summary of 263 samples and genes under selection for seven peach groups.&lt;;/span&gt;; (&lt;;strong&gt;;A&lt;;/strong&gt;;) Geographic distribution of 263 peach accessions used in this study. Each accession is represented by a dot on the world map. Seven ecotypes are highlighted using rings with different colors. (&lt;;strong&gt;;B&lt;;/strong&gt;;) Demographic history of the seven peach groups. Ancestral population size was inferred using the PSMC model. Three periods of the last glacial maximum (LGM, [[#x223C]]20 KYA), Naynayxungla Glaciation (NG, 0.5[[#x223C]]0.78 MYA), and Xixiabangma Glaciation (XG, 0.8[[#x223C]]0.17 MYA) are shaded in green, red, and blue, respectively. (&lt;;strong&gt;;C&lt;;/strong&gt;;) Venn diagram showing the number of genes under selection in the seven groups. (&lt;;strong&gt;;D&lt;;/strong&gt;;) Over-represented gene ontology (GO) terms in overall selection regions. Only the top 20 most over-represented terms are shown. YG, Yun-gui Plateau. NW, Northwest China. NP, North Plain China. YT, Yangtze River Middle and Backward. NE, Northeast China. TB, Tibet plateau. ST, South China Sub-tropical.&lt;;/div&gt;;&lt;;/div&gt;;" data-icon-position="" data-hide-link-title="0">
          <span class="hw-responsive-img">
           <img class="highwire-fragment fragment-image lazyload" alt="Fig. 1" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2020/01/15/2020.01.15.907709/F1.medium.gif" width="281" height="440"/>
           <noscript>
            <img class="highwire-fragment fragment-image" alt="Fig. 1" src="https://www.biorxiv.org/content/biorxiv/early/2020/01/15/2020.01.15.907709/F1.medium.gif" width="281" height="440"/>
           </noscript>
          </span>
         </a>
        </div>
       </div>
       <ul class="highwire-figure-links inline">
        <li class="download-fig first">
         <a href="https://www.biorxiv.org/content/biorxiv/early/2020/01/15/2020.01.15.907709/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Fig. 1" data-icon-position="" data-hide-link-title="0">Download figure</a>
        </li>
        <li class="new-tab last">
         <a href="https://www.biorxiv.org/content/biorxiv/early/2020/01/15/2020.01.15.907709/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a>
        </li>
       </ul>
      </div>
      <div class="fig-caption">
       <span class="fig-label">Fig. 1</span>
       <span class="caption-title">Summary of 263 samples and genes under selection for seven peach groups.</span>
       <p id="p-8" class="first-child">(
        <strong>A</strong>) Geographic distribution of 263 peach accessions used in this study. Each accession is represented by a dot on the world map. Seven ecotypes are highlighted using rings with different colors. (
        <strong>B</strong>) Demographic history of the seven peach groups. Ancestral population size was inferred using the PSMC model. Three periods of the last glacial maximum (LGM, ∼20 KYA), Naynayxungla Glaciation (NG, 0.5∼0.78 MYA), and Xixiabangma Glaciation (XG, 0.8∼0.17 MYA) are shaded in green, red, and blue, respectively. (
        <strong>C</strong>) Venn diagram showing the number of genes under selection in the seven groups. (
        <strong>D</strong>) Over-represented gene ontology (GO) terms in overall selection regions. Only the top 20 most over-represented terms are shown. YG, Yun-gui Plateau. NW, Northwest China. NP, North Plain China. YT, Yangtze River Middle and Backward. NE, Northeast China. TB, Tibet plateau. ST, South China Sub-tropical.
       </p>
       <div class="sb-div caption-clear"/>
      </div>
     </div>
     <p id="p-9">We explored the genetic relationships among 263 accessions using 2,468,307 SNPs with minor allele frequency (MAF) greater than 0.05. Based on the neighbor-joining tree and population structure analysis, the 263 peach accessions could be divided into seven major groups, which were largely congruent with ecotypes classified according to their geographic information, including YG (Yun-gui Plateau), NW (Northwest China), NP (North Plain China), YT (Yangtze River Middle and Backward), NE (Northeast China), TB (Tibet plateau), and ST (South China Sub-tropical) groups (Supplementary Fig. S1B, Supplementary Fig. S2, and Supplementary Table S1). Although the neighbor-joining tree largely supported the division of seven major groups, there were some discrepancies between geographical characterization and phylogenetic clustering (Supplementary Fig. S2D), indicating shared ancestral variation and historical gene flow among landraces in closely related groups. Moreover, principal component analysis (PCA) and model-based clustering analyses also supported the extensive admixture and possible gene flow among landrace groups (Supplementary Fig. S2E and S2F). Furthermore, we found the small pair-wise genetic differentiation (
      <em>F</em>
      <sub>ST</sub>) values between different landrace groups, again consistent with population admixture (Supplementary Fig. S2G).
     </p>
     <p id="p-10">Using the demographic analysis with the pairwise sequential Markovian coalescent (PSMC) model (
      <a id="xref-ref-31-1" class="xref-bibr" href="#ref-31">Li and Durbin 2011</a>), we found the sharply decline of effective population size (
      <em>Ne</em>) during the two largest Pleistocene glaciations: the Xixiabangma glaciation (1.17-0.8 MYA) and Naynayxungla glaciation (0.78-0.50 MYA), and a slight decline of 
      <em>Ne</em> during the last glacial maximum (∼20,000 years ago) (
      <a id="xref-fig-1-2" class="xref-fig" href="#F1">Fig. 1B</a>).
     </p>
    </div>
    <div id="sec-3" class="subsection">
     <h3>Selective sweeps related to adaptation to diverse environments</h3>
     <p id="p-11">Peach accessions of each group have adapted locally through long-term selection under local environments (Supplementary Table S3). To identify genomic loci that favor local adaptation for seven groups, we detected signatures of selective sweeps for each group. This revealed a total of 2,092 genomic regions (19.1 Mb, ∼8.4%; 189, 387, 301, 235, 280, 339, and 378 regions for the YG, NW, NP, YT, NE, TB, and ST groups, respectively) (Supplementary Fig. S3), which were termed candidate selection regions (CSRs) (Supplementary Table S4). The overall CSRs harbored 4,198 genes (∼17.5%), including 506, 1,197, 835, 530, 747, 920, and 869 genes for the YG, NW, NP, YT, NE, TB, and ST groups, respectively (
      <a id="xref-fig-1-3" class="xref-fig" href="#F1">Fig. 1C</a>). Selections on these genes may underlie the adaptation to different climates. Notably, we found that few genes were shared among different groups (
      <a id="xref-fig-1-4" class="xref-fig" href="#F1">Fig. 1C</a>), suggesting the unique adaptive patterns for each group and that different climates may shape distinct genomic regions.
     </p>
     <p id="p-12">We found that genes related to response to different types of stimuli and stress, including temperature, radiation, salt, DNA damage, osmotic, toxin, were overrepresented (
      <em>P</em> &lt;; 0.05), suggesting that stress-related genes have participated in adaptive evolution (
      <a id="xref-fig-1-5" class="xref-fig" href="#F1">Fig. 1D</a>, Supplementary Table S5). For instance, two cation/H
      <sup>+</sup> exchanger family genes (
      <em>CHX</em>) (
      <em>Prupe</em>.
      <em>6G251600</em> and 
      <em>Prupe</em>.
      <em>6G251700</em>) and one 
      <em>salt overly sensitive 3</em> (
      <em>SOS3</em>) (
      <em>Prupe</em>.
      <em>2G188700</em>) gene showed high reduction of diversity (ROD) and 
      <em>F</em>
      <sub>ST</sub> values in the NW group. Homologs of these genes are involved in salt resistance in 
      <em>A. thaliana</em> (
      <a id="xref-ref-42-1" class="xref-bibr" href="#ref-42">Monihan et al. 2016</a>), suggesting their potential contributions to adaptation to saline soils in northwestern China. The resistance-related LRR (leucine-rich repeat) domain and PPR (pentatricopeptide repeat) gene family were highly enriched in CSRs (
      <em>P</em> &lt;; 0.05) (Supplementary Table S5). The LRR domain, which is considered to be one of the most important domains for plant resistance genes, was also enriched (
      <em>P</em> &lt;; 0.05), with 121 of 612 members (19.8%) in CSRs. PPR proteins form one of the largest protein families in land plants that are related to environmental responses, with 286 members in peach genome, of which 79 (∼27.6%) were in CSRs.
     </p>
     <p id="p-13">The known genes or biological pathways involved in adaptation to the environment in the habitat of each group were determined. For instance, the YG group was distributed on the Yun-gui plateau (Southwest China), a low-latitude and high-altitude (∼2000 m) region with high annual precipitation (&gt;; 1100 mm) and acidic soil (pH 4.5∼5.5) (Supplementary Table S3). Genes related to metal ion (including potassium, iron, and zinc) binding and transport, cell membrane function, and response to toxins were overrepresented in this group (107 genes, 
      <em>P</em> &lt;; 0.05) (Supplementary Table S5), consistent with functions in overcoming cation deficiency and aluminum toxicity that are common in acidic soils (
      <a id="xref-ref-50-1" class="xref-bibr" href="#ref-50">Seguel et al. 2013</a>). For the YT group, we observed enrichments of the LRR domain (24 genes), NB-ARC domain (8 genes), and other genes related to stress responses (32 genes) (
      <em>P</em> &lt;; 0.05) (Supplementary Table S5), in comparison to other groups. This suggests that the YT group has accumulated more abiotic and biotic stress-resistance variants due to strong selective pressures in high temperature and high humidity areas (Supplementary Table S3). These results indicate that accessions from the YT group may exhibit higher adaptability than other landrace accessions.
     </p>
    </div>
    <div id="sec-4" class="subsection">
     <h3>Genome-wide environmental association studies of 51 environmental variables</h3>
     <p id="p-14">Although we obtained candidate genes underlying adaptation by identifying selective sweeps, many adaptive events in natural populations may occur by polygenic adaptation, which would be largely undetected by conventional methods for detecting selection (
      <a id="xref-ref-47-1" class="xref-bibr" href="#ref-47">Pritchard and Di Rienzo 2010</a>). However, local adaptation can generate correlations between environmental variables (EVs) and genomic loci which can be used to detect polygenic adaptation. We investigated a total of 51 EVs of the geographic origin of each accession that are important for plant adaptation (Supplementary Table S6 and S7), representing extremes and seasonality of temperature and precipitation, latitude, altitude, relative air humidity, water vapor pressure, growing season lengths, and radiations. Using a Mantel test, we found a significant correlation between geographic and genetic distances (Pearson’s 
      <em>r</em> = 0.73, 
      <em>P</em> = 0.000999), with most associations being driven by altitude. To obtain loci associated with EVs, we performed GWEAS on 51 EVs. A total of 9393 association SNPs (Supplementary Table S8), involving 3807 genes, were identified (
      <a id="xref-fig-2-1" class="xref-fig" href="#F2">Fig. 2A</a>). Notably, we found an EV association hotspot regions at the top of chromosome 2 that was enriched with genes encoding NB-LRR proteins in peach genome (
      <a id="xref-ref-54-3" class="xref-bibr" href="#ref-54">Verde et al. 2013</a>). Consistent with the high correlations among some climate variables (Supplementary Fig. S4), only 3496 association SNPs were unique, and ∼62.8% of the associations were shared across different types of EVs, suggesting that different EVs may shape same genomic regions. Notably, a total of 82 genomic loci associated with more than 10 EVs were identified.
     </p>
     <div id="F2" class="fig pos-float type-figure odd">
      <div class="highwire-figure">
       <div class="fig-inline-img-wrapper">
        <div class="fig-inline-img">
         <a href="https://www.biorxiv.org/content/biorxiv/early/2020/01/15/2020.01.15.907709/F2.large.jpg?width=800&amp;;height=600&amp;;carousel=1" title="Genome-wide environmental association studies of 51 environmental variables and genomic loci associated with winter cold adaptation. (A) SNPs associated with environmental variables (EVs). Only the top 10 association signals for each EV are shown. All signals were included if the total number of signals was &lt;; 10. (B) The PpAHP locus involved in adaptation to winter low temperature in peach. Manhattan plots for a GWAS study of cold hardiness (CH) and winter lowest temperature (MIN), and selection signals of the NE group (ROD) were detailed. The dashed line represents the significance threshold for each test. The candidate genomic region is highlighted between two dashed blue vertical lines. (C) Distribution of FST values between NE and ST groups in the candidate region. The green bar indicates the PpAHP locus. (D) Close-up view of the FST values in a region corresponding to the green bar in (C). This region contains six PpAHP homologs (orange) and one other gene (light blue). The candidate SNP is highlighted using an orange dot. (E) Relative expression changes of PpAHP5 after cold treatment ([[#x2212]]28[[#xB0]]C) in resistant and sensitive cultivars. (F) Association between genotypes and cold hardiness (lethal temperature of 50%, LT50). (G) Allele frequencies of association locus (Pp06: 9,187,362) in PpAHP5 across seven groups." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-709642862" data-figure-caption="&lt;;div class=&quot;;highwire-markup&quot;;&gt;;&lt;;div xmlns=&quot;;http://www.w3.org/1999/xhtml&quot;;&gt;;&lt;;span class=&quot;;caption-title&quot;;&gt;;Genome-wide environmental association studies of 51 environmental variables and genomic loci associated with winter cold adaptation.&lt;;/span&gt;; (&lt;;strong&gt;;A&lt;;/strong&gt;;) SNPs associated with environmental variables (EVs). Only the top 10 association signals for each EV are shown. All signals were included if the total number of signals was &lt;; 10. (&lt;;strong&gt;;B&lt;;/strong&gt;;) The &lt;;em&gt;;PpAHP&lt;;/em&gt;; locus involved in adaptation to winter low temperature in peach. Manhattan plots for a GWAS study of cold hardiness (CH) and winter lowest temperature (MIN), and selection signals of the NE group (ROD) were detailed. The dashed line represents the significance threshold for each test. The candidate genomic region is highlighted between two dashed blue vertical lines. (&lt;;strong&gt;;C&lt;;/strong&gt;;) Distribution of &lt;;em&gt;;F&lt;;/em&gt;;&lt;;sub&gt;;ST&lt;;/sub&gt;; values between NE and ST groups in the candidate region. The green bar indicates the &lt;;em&gt;;PpAHP&lt;;/em&gt;; locus. (&lt;;strong&gt;;D&lt;;/strong&gt;;) Close-up view of the &lt;;em&gt;;F&lt;;/em&gt;;&lt;;sub&gt;;ST&lt;;/sub&gt;; values in a region corresponding to the green bar in (&lt;;strong&gt;;C&lt;;/strong&gt;;). This region contains six &lt;;em&gt;;PpAHP&lt;;/em&gt;; homologs (orange) and one other gene (light blue). The candidate SNP is highlighted using an orange dot. (&lt;;strong&gt;;E&lt;;/strong&gt;;) Relative expression changes of &lt;;em&gt;;PpAHP5&lt;;/em&gt;; after cold treatment ([[#x2212]]28[[#xB0]]C) in resistant and sensitive cultivars. (&lt;;strong&gt;;F&lt;;/strong&gt;;) Association between genotypes and cold hardiness (lethal temperature of 50%, LT50). (&lt;;strong&gt;;G&lt;;/strong&gt;;) Allele frequencies of association locus (Pp06: 9,187,362) in &lt;;em&gt;;PpAHP5&lt;;/em&gt;; across seven groups.&lt;;/div&gt;;&lt;;/div&gt;;" data-icon-position="" data-hide-link-title="0">
          <span class="hw-responsive-img">
           <img class="highwire-fragment fragment-image lazyload" alt="Fig. 2" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2020/01/15/2020.01.15.907709/F2.medium.gif" width="440" height="231"/>
           <noscript>
            <img class="highwire-fragment fragment-image" alt="Fig. 2" src="https://www.biorxiv.org/content/biorxiv/early/2020/01/15/2020.01.15.907709/F2.medium.gif" width="440" height="231"/>
           </noscript>
          </span>
         </a>
        </div>
       </div>
       <ul class="highwire-figure-links inline">
        <li class="download-fig first">
         <a href="https://www.biorxiv.org/content/biorxiv/early/2020/01/15/2020.01.15.907709/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Fig. 2" data-icon-position="" data-hide-link-title="0">Download figure</a>
        </li>
        <li class="new-tab last">
         <a href="https://www.biorxiv.org/content/biorxiv/early/2020/01/15/2020.01.15.907709/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a>
        </li>
       </ul>
      </div>
      <div class="fig-caption">
       <span class="fig-label">Fig. 2</span>
       <span class="caption-title">Genome-wide environmental association studies of 51 environmental variables and genomic loci associated with winter cold adaptation.</span>
       <p id="p-15" class="first-child">(
        <strong>A</strong>) SNPs associated with environmental variables (EVs). Only the top 10 association signals for each EV are shown. All signals were included if the total number of signals was &lt;; 10. (
        <strong>B</strong>) The 
        <em>PpAHP</em> locus involved in adaptation to winter low temperature in peach. Manhattan plots for a GWAS study of cold hardiness (CH) and winter lowest temperature (MIN), and selection signals of the NE group (ROD) were detailed. The dashed line represents the significance threshold for each test. The candidate genomic region is highlighted between two dashed blue vertical lines. (
        <strong>C</strong>) Distribution of 
        <em>F</em>
        <sub>ST</sub> values between NE and ST groups in the candidate region. The green bar indicates the 
        <em>PpAHP</em> locus. (
        <strong>D</strong>) Close-up view of the 
        <em>F</em>
        <sub>ST</sub> values in a region corresponding to the green bar in (
        <strong>C</strong>). This region contains six 
        <em>PpAHP</em> homologs (orange) and one other gene (light blue). The candidate SNP is highlighted using an orange dot. (
        <strong>E</strong>) Relative expression changes of 
        <em>PpAHP5</em> after cold treatment (−28°C) in resistant and sensitive cultivars. (
        <strong>F</strong>) Association between genotypes and cold hardiness (lethal temperature of 50%, LT50). (
        <strong>G</strong>) Allele frequencies of association locus (Pp06: 9,187,362) in 
        <em>PpAHP5</em> across seven groups.
       </p>
       <div class="sb-div caption-clear"/>
      </div>
     </div>
     <p id="p-16">Next, we identified known biological processes that were overrepresented among associations for each EV and for overall EVs (Supplementary Table S9). Functional categories related to response to a series of abiotic or biotic stimuli, “programmed cell death (PCD)”, “innate immune response”, and “LRR domain” were highly overrepresented (
      <em>P</em> &lt;; 0.05), suggesting that EVs mainly shaped genomic regions related to stress responses. Notably, a series of processes involved in secondary metabolism, including “flavonoid metabolic process”, “jasmonic acid (JA) biosynthetic process”, and “biosynthesis of plant hormones and terpenoids”, were significantly overrepresented (
      <em>P</em> &lt;; 0.05) (Supplementary Table S9). We found that genes related to JA biosynthesis were enriched in altitude associations (
      <em>P</em> &lt;; 0.05). Previous studies have shown that JA treatment contributes to enhanced cold resistance by promoting expression of the ICE-CBF/DREB1 transcriptional pathway, while a mutation in a key JA biosynthesis gene, 
      <em>LOX1</em> (
      <em>Prupe</em>.
      <em>6G324400</em>, an altitude association gene in this study), leads to cold hypersensitive phenotypes (
      <a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">Hu et al. 2013</a>). For each EV, several known biological processes were overrepresented (Supplementary Table S9). For instance, genes involved in ion transport were highly enriched in those associated with soil pH (
      <em>P</em> &lt;; 0.05), as soil pH affects absorption of metal ions in plants (
      <a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">Harter 1983</a>).
     </p>
     <p id="p-17">Temperature and precipitation are two of the most important EVs that affect plant distribution and survival. We identified temperature associated SNPs, distributed across all eight peach chromosomes, and five association hotspots on chromosome 1, 2, 5, 6, and 8 were detected in GWEAS for more than nine temperature-related EVs and altitude (Supplementary Fig. S5A and 5B). Tolerance to low temperature in winter is a major factor that restricts the spread of peach to extremely cold regions (north of 40 °N). To characterize genetic loci underlying adaptation to extremely cold climates in peach, we performed a GWAS analysis of cold hardiness and identified four association peaks, on chromosomes 2, 4, 6, and 7 (
      <a id="xref-fig-2-2" class="xref-fig" href="#F2">Fig. 2B</a>). Of these, the peak on chromosome 6 showed a strong selection signal, with sharp ROD in the NE group that experienced an extreme cold winter (lowest temperature &lt;; −30 
      <sup>o</sup>C) (
      <a id="xref-fig-2-3" class="xref-fig" href="#F2">Fig. 2B</a>). Moreover, this peak overlapped with the temperature association hotspot on chromosome 6 and association peaks of annual lowest temperature (
      <a id="xref-fig-2-4" class="xref-fig" href="#F2">Fig. 2B</a>). The NE group (
      <em>n</em> = 19) inhabits areas north of 40 °N that have extremely low winter temperatures, while the ST group (
      <em>n</em> = 14) grows in a contrasting climate, south of 25 °N in areas with a warm winter (lowest temperature &gt;; 10°C). We searched for genomic regions and SNPs with extremely high differentiation between ST and NE groups. One (Pp06: 9,187,362) of these SNPs (
      <em>F</em>
      <sub>ST</sub> = 1) resided within the overlapping intervals between annual lowest temperature and cold hardiness associations (
      <a id="xref-fig-2-5" class="xref-fig" href="#F2">Fig. 2C</a>). This SNP was located in the gene 
      <em>PpAHP5</em> (
      <em>Prupe</em>.
      <em>6G123100</em>), belonging to a gene cluster encoding six histidine phosphotransfer proteins (AHP) (
      <a id="xref-fig-2-6" class="xref-fig" href="#F2">Fig. 2D</a>), which have been reported to be involved in mediating cold signaling in 
      <em>A. thaliana</em> (
      <a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">Jeon and Kim 2013</a>). Using cold treatment, we found this gene was up-regulated by cold and resistance cultivars harbored significantly high expression level than sensitive one (
      <a id="xref-fig-2-7" class="xref-fig" href="#F2">Fig. 2E</a>). At this SNP locus, all representative accessions in the NE group showed a distinct genotype (TT) compared with the ST group (CC) (
      <a id="xref-fig-2-8" class="xref-fig" href="#F2">Fig. 2F</a>), indicating that the TT genotype in 
      <em>PpAHP5</em> is favored in high-altitude cold regions (
      <a id="xref-fig-2-9" class="xref-fig" href="#F2">Fig. 2G</a>), and that 
      <em>PpAHP5</em> is a candidate for conferring cold resistance in peach. We also detected six strong association regions for precipitation-related EVs, including annual and seasonal precipitation, length of growing season, aridity, and relative air humidity (Supplementary Fig. S5C and 5D). An extremely strong association hotspot on Pp02 (4.8∼7.2 Mb) was identified, exhibiting enrichments of 
      <em>R</em> genes (
      <a id="xref-ref-54-4" class="xref-bibr" href="#ref-54">Verde et al. 2013</a>), 
      <em>RLKs</em> super family genes, NB-ARC domains, and other stress response-related genes, suggesting a genetic basis for precipitation adaptation.
     </p>
     <p id="p-18">To further elucidate the pattern of adaptation, we detected overlaps between selective sweeps and GWEAS. A total of 888 genes (∼23.3% of GWEAS genes) were shared between selective sweeps and GWEAS (Supplementary Fig. S6). This revealed that although selective sweeps are important, adaptations from standing variation or polygenic adaptation are also likely an important mode of adaptation in peach, which may be related to its shortly spread history after domestication (
      <a id="xref-ref-34-2" class="xref-bibr" href="#ref-34">Li et al. 2019</a>). These findings suggest that domesticated fruit species, such as peach, are generating and enhancing adaptation by standing selection on existing multiple sites. This situation is different from 
      <em>A. thaliana</em>, which may have reached its adaptive limits owing to the constraints imposed by the limited generation of new mutations (
      <a id="xref-ref-18-2" class="xref-bibr" href="#ref-18">Hancock et al. 2011</a>). Collectively, these results indicate that both selective sweeps and GWEAS are central factors in the adaptive genetics of domesticated species.
     </p>
    </div>
    <div id="sec-5" class="subsection">
     <h3>Adaptation to highly drought regions</h3>
     <p id="p-19">The NW group is from northwestern China, which has an extreme climate, characterized by severe aridity (&lt;; 150 mm annual rainfall) (
      <a id="xref-fig-3-1" class="xref-fig" href="#F3">Fig. 3A</a>) and extreme high or low temperatures in the summer (&gt;; 40 °C) or winter (&lt;; −30 °C) (Supplementary Table S3). Peach accessions from this region are frequently challenged by high drought stress. We found that genes overrepresented in this group included those involved in abscisic acid (ABA) biosynthesis and signal transduction (
      <em>P</em> &lt;; 0.05) (Supplementary Table S5), which are well known to regulate drought stress responses. Transcriptome analyses of peach accessions grown under drought stress conditions revealed that genes involved in the ABA pathway were highly enriched among differentially expressed genes (DEGs), including 
      <em>NCED, PYR, ABA2, PP2C</em>, and 
      <em>ABRE</em> genes that showed selective signals in the NW group (
      <a id="xref-fig-3-2" class="xref-fig" href="#F3">Fig. 3B</a>), further suggesting a key role of ABA pathway in peach drought responses.
     </p>
     <div id="F3" class="fig pos-float type-figure odd">
      <div class="highwire-figure">
       <div class="fig-inline-img-wrapper">
        <div class="fig-inline-img">
         <a href="https://www.biorxiv.org/content/biorxiv/early/2020/01/15/2020.01.15.907709/F3.large.jpg?width=800&amp;;height=600&amp;;carousel=1" title="Genetic basis of drought resistance and high sugar content in the NW group. (A) Annual precipitation among the seven groups. (B) Relationship between the ABA pathway, drought stress and evolution of flesh color. Heat map in orange indicate gene expression levels (FPKM) under drought stress (0h, 6h, 12h, 24h, 3d, 6d, 12d). Heat maps in pink indicate gene expression levels (FPKM) during peach fruit development (10, 50, and 90 days post bloom date (dpb) for PpCCD4; 20, 40, 60, 80, 100, 120 dpb for PpSPS1). Genes under selection in the NW group are highlighted in red. Red arrows indicate the increase in levels of ABA and sugars. (C) Soluble solid content (SSC) among the seven groups. (D) ABRE cis-acting elements in the promoters of PpSPS1, PpBAM1, and PpAMY1. Orange boxes indicate ABRE elements in the promoter of each gene. The number around each ABRE represents the position from the ATG. The distribution of ABRE elements and nucleotide diversity in the promoter of PpSPS1 in the NW and other groups are shown in a dashed box. (E) Distribution of ROD around PpSPS1 on chromosome 1. Black arrow points to PpSPS1. (F) Distribution of CLR values around PpSPS1 on chromosome 1. Black arrow points to PpSPS1. (G) Verification of the interaction between PpAREB (Prupe.1G434500) and the promoter of PpSPS1 (Prupe.1G483200) using a yeast one-hybrid assay." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-709642862" data-figure-caption="&lt;;div class=&quot;;highwire-markup&quot;;&gt;;&lt;;div xmlns=&quot;;http://www.w3.org/1999/xhtml&quot;;&gt;;&lt;;span class=&quot;;caption-title&quot;;&gt;;Genetic basis of drought resistance and high sugar content in the NW group.&lt;;/span&gt;; (&lt;;strong&gt;;A&lt;;/strong&gt;;) Annual precipitation among the seven groups. (&lt;;strong&gt;;B&lt;;/strong&gt;;) Relationship between the ABA pathway, drought stress and evolution of flesh color. Heat map in orange indicate gene expression levels (FPKM) under drought stress (0h, 6h, 12h, 24h, 3d, 6d, 12d). Heat maps in pink indicate gene expression levels (FPKM) during peach fruit development (10, 50, and 90 days post bloom date (dpb) for &lt;;em&gt;;PpCCD4&lt;;/em&gt;;; 20, 40, 60, 80, 100, 120 dpb for &lt;;em&gt;;PpSPS1&lt;;/em&gt;;). Genes under selection in the NW group are highlighted in red. Red arrows indicate the increase in levels of ABA and sugars. (&lt;;strong&gt;;C&lt;;/strong&gt;;) Soluble solid content (SSC) among the seven groups. (&lt;;strong&gt;;D&lt;;/strong&gt;;) ABRE &lt;;em&gt;;cis&lt;;/em&gt;;-acting elements in the promoters of &lt;;em&gt;;PpSPS1, PpBAM1&lt;;/em&gt;;, and &lt;;em&gt;;PpAMY1&lt;;/em&gt;;. Orange boxes indicate ABRE elements in the promoter of each gene. The number around each ABRE represents the position from the ATG. The distribution of ABRE elements and nucleotide diversity in the promoter of &lt;;em&gt;;PpSPS1&lt;;/em&gt;; in the NW and other groups are shown in a dashed box. (&lt;;strong&gt;;E&lt;;/strong&gt;;) Distribution of ROD around &lt;;em&gt;;PpSPS1&lt;;/em&gt;; on chromosome 1. Black arrow points to &lt;;em&gt;;PpSPS1&lt;;/em&gt;;. (&lt;;strong&gt;;F&lt;;/strong&gt;;) Distribution of CLR values around &lt;;em&gt;;PpSPS1&lt;;/em&gt;; on chromosome 1. Black arrow points to &lt;;em&gt;;PpSPS1&lt;;/em&gt;;. (&lt;;strong&gt;;G&lt;;/strong&gt;;) Verification of the interaction between &lt;;em&gt;;PpAREB&lt;;/em&gt;; (&lt;;em&gt;;Prupe&lt;;/em&gt;;.&lt;;em&gt;;1G434500&lt;;/em&gt;;) and the promoter of &lt;;em&gt;;PpSPS1&lt;;/em&gt;; (&lt;;em&gt;;Prupe&lt;;/em&gt;;.&lt;;em&gt;;1G483200&lt;;/em&gt;;) using a yeast one-hybrid assay.&lt;;/div&gt;;&lt;;/div&gt;;" data-icon-position="" data-hide-link-title="0">
          <span class="hw-responsive-img">
           <img class="highwire-fragment fragment-image lazyload" alt="Fig. 3" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2020/01/15/2020.01.15.907709/F3.medium.gif" width="440" height="262"/>
           <noscript>
            <img class="highwire-fragment fragment-image" alt="Fig. 3" src="https://www.biorxiv.org/content/biorxiv/early/2020/01/15/2020.01.15.907709/F3.medium.gif" width="440" height="262"/>
           </noscript>
          </span>
         </a>
        </div>
       </div>
       <ul class="highwire-figure-links inline">
        <li class="download-fig first">
         <a href="https://www.biorxiv.org/content/biorxiv/early/2020/01/15/2020.01.15.907709/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Fig. 3" data-icon-position="" data-hide-link-title="0">Download figure</a>
        </li>
        <li class="new-tab last">
         <a href="https://www.biorxiv.org/content/biorxiv/early/2020/01/15/2020.01.15.907709/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a>
        </li>
       </ul>
      </div>
      <div class="fig-caption">
       <span class="fig-label">Fig. 3</span>
       <span class="caption-title">Genetic basis of drought resistance and high sugar content in the NW group.</span>
       <p id="p-20" class="first-child">(
        <strong>A</strong>) Annual precipitation among the seven groups. (
        <strong>B</strong>) Relationship between the ABA pathway, drought stress and evolution of flesh color. Heat map in orange indicate gene expression levels (FPKM) under drought stress (0h, 6h, 12h, 24h, 3d, 6d, 12d). Heat maps in pink indicate gene expression levels (FPKM) during peach fruit development (10, 50, and 90 days post bloom date (dpb) for 
        <em>PpCCD4</em>; 20, 40, 60, 80, 100, 120 dpb for 
        <em>PpSPS1</em>). Genes under selection in the NW group are highlighted in red. Red arrows indicate the increase in levels of ABA and sugars. (
        <strong>C</strong>) Soluble solid content (SSC) among the seven groups. (
        <strong>D</strong>) ABRE 
        <em>cis</em>-acting elements in the promoters of 
        <em>PpSPS1, PpBAM1</em>, and 
        <em>PpAMY1</em>. Orange boxes indicate ABRE elements in the promoter of each gene. The number around each ABRE represents the position from the ATG. The distribution of ABRE elements and nucleotide diversity in the promoter of 
        <em>PpSPS1</em> in the NW and other groups are shown in a dashed box. (
        <strong>E</strong>) Distribution of ROD around 
        <em>PpSPS1</em> on chromosome 1. Black arrow points to 
        <em>PpSPS1</em>. (
        <strong>F</strong>) Distribution of CLR values around 
        <em>PpSPS1</em> on chromosome 1. Black arrow points to 
        <em>PpSPS1</em>. (
        <strong>G</strong>) Verification of the interaction between 
        <em>PpAREB</em> (
        <em>Prupe</em>.
        <em>1G434500</em>) and the promoter of 
        <em>PpSPS1</em> (
        <em>Prupe</em>.
        <em>1G483200</em>) using a yeast one-hybrid assay.
       </p>
       <div class="sb-div caption-clear"/>
      </div>
     </div>
     <p id="p-21">Sugars function as the important signaling molecules in response to a range of abiotic and biotic stresses in plants (
      <a id="xref-ref-29-1" class="xref-bibr" href="#ref-29">Lastdrager et al. 2014</a>). We found that peach fruits produced by accessions from the NW group, especially accessions from Xinjiang province (
      <a id="xref-ref-56-2" class="xref-bibr" href="#ref-56">Wang et al. 2012</a>), consistently had higher soluble sugar contents than those from other groups (
      <a id="xref-fig-3-3" class="xref-fig" href="#F3">Fig. 3C</a>). Associated long-term natural selection pressures contributing to greater accumulation of soluble sugars likely include aridity, high diurnal temperature variation, and long sunshine duration. Moreover, the starch and sucrose metabolism pathways were overrepresented in both DEGs under drought stress treatment (35 genes) and genes under selection in the NW group (12 genes) (
      <em>P</em> &lt;; 0.05), congruent with roles of sugars in drought stress. Furthermore, all the 12 genes in the selective sweeps were differentially expressed following the drought stress treatment. We conclude that higher soluble sugar contents in accessions from northwestern China represent an adaptive trait driven by the local drought environment.
     </p>
     <p id="p-22">Previous studies of apple have demonstrated that drought stress and ABA contributed to soluble sugar accumulation through the activation of sugar transporter and amylase genes by the ABA-responsive transcription factor, 
      <em>AREB2</em> (
      <a id="xref-ref-37-1" class="xref-bibr" href="#ref-37">Ma et al. 2017</a>). Similarly, both drought stress and exogenous ABA induce an increase in soluble sugar accumulation in peach fruit (
      <a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">Kobashi et al. 2000</a>; 
      <a id="xref-ref-27-1" class="xref-bibr" href="#ref-27">Kobashi et al. 2001</a>). Here we found that two putative gene targets of AREB2 (
      <a id="xref-fig-3-4" class="xref-fig" href="#F3">Fig. 3B</a> and 
      <a id="xref-fig-3-5" class="xref-fig" href="#F3">3D</a>), 
      <em>PpAMY1</em> (
      <em>Prupe</em>.
      <em>1G142400</em>) and 
      <em>PpBAM1</em> (
      <em>Prupe</em>.
      <em>1G053800</em>), were up-regulated by drought treatment; however, neither exhibited a significant selection signal. To identify additional target genes in drought mediated sugar accumulation, we searched for genes harboring the putative binding domain of AREB2 among genes under selection in the NW group. This revealed a sucrose phosphate synthase gene (
      <em>PpSPS1, Prupe</em>.
      <em>1G483200</em>), with six ABA-responsive elements (ABREs) in the promoter region (
      <a id="xref-fig-3-6" class="xref-fig" href="#F3">Fig. 3D</a>), showing a strong selection signal, with high ROD and CLR values (
      <a id="xref-fig-3-7" class="xref-fig" href="#F3">Fig. 3E</a> and 
      <a id="xref-fig-3-8" class="xref-fig" href="#F3">3F</a>). 
      <em>PpSPS1</em>, which is involved in the biosynthesis of sucrose, the predominant soluble sugar in mature peach fruit and the key factor conferring sweetness, was up-regulated by drought treatment (
      <a id="xref-fig-3-9" class="xref-fig" href="#F3">Fig. 3B</a>), suggesting its roles in drought stress response. The expression of 
      <em>PpSPS1</em> increased by ∼500-fold during fruit maturity (
      <a id="xref-fig-3-10" class="xref-fig" href="#F3">Fig. 3B</a>), implying its roles in fruit ripening and sugar accumulation. Using a yeast one-hybrid experiment, we verified the interactions between AREB/ABF and the promoter of 
      <em>PpSPS1</em> (
      <a id="xref-fig-3-11" class="xref-fig" href="#F3">Fig. 3G</a>), providing new insight into ABA-mediated enhanced sugar accumulation under drought stress. The selection on sugar related genes may mediate adaptation to drought stress in the NW group, accompanied by the increases in fruit sugar content. In addition, we found that the top of chromosome 5 and the middle of chromosome 4, which have been reported to harbor major SSC-and sugar content-associated quantitative trait loci (QTLs) and SSC candidate gene 
      <em>PpNCED3</em> (
      <a id="xref-ref-38-1" class="xref-bibr" href="#ref-38">Martínez-García et al. 2013</a>; 
      <a id="xref-ref-34-3" class="xref-bibr" href="#ref-34">Li et al. 2019</a>), also showed strong selection signals in the NW group. Selections on these genes may underlie the genetic basis of high sugar levels in peach accessions grown in areas with high drought stress. Moreover, such genes represent excellent candidates for high-sugar breeding.
     </p>
     <p id="p-23">Intriguingly, we found that flesh color of peach showed strong geographic pattern, with ∼80% of yellow-fleshed peach landraces originating from northwestern China (NW group). Yellow flesh of peach mainly depends on the content of carotenoids at maturity, including β-cryptoxanthin and β-carotene, and carotenoids are believed to be the major precursors for ABA biosynthesis (
      <a id="xref-fig-3-12" class="xref-fig" href="#F3">Fig. 3B</a>). A previous study has identified three loss-of-function variants involved in a carotenoid cleavage dioxygenase gene (
      <em>PpCCD4, Prupe</em>.
      <em>1G255500</em>), leading to the abnormal carotenoid degradation and yellow flesh (
      <a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">Falchi et al. 2013</a>). The disturbed function of 
      <em>PpCCD4</em> in yellow-fleshed peach resulted in the retention of carotenoids, which can provide more precursors for ABA biosynthesis (
      <a id="xref-fig-3-13" class="xref-fig" href="#F3">Fig. 3B</a>), and may contribute to higher ABA levels and subsequent enhanced drought tolerance. Moreover, using transcriptional analyses, we found that 
      <em>PpCCD4</em> was down-regulated by drought treatments (
      <a id="xref-fig-3-14" class="xref-fig" href="#F3">Fig. 3B</a>), suggesting its response to drought stress. Furthermore, the carotenoid biosynthetic pathway was highly overrepresented with genes under selection in the NW group (
      <em>P</em> &lt;; 0.05). Therefore, we conclude that yellow peach flesh has undergone long-term adaptive selection, driven by drought stress, and that modern yellow-fleshed peach cultivars may originate from northwestern China.
     </p>
     <p id="p-24">Collectively, we constructed a joint pathway for drought adaptation evolution in peach, driven by the complicated interactions between carotenoids, ABA, and sugar, of which ABA may be the central controller and play the key roles.</p>
    </div>
    <div id="sec-6" class="subsection">
     <h3>Adaptation to high altitudes</h3>
     <p id="p-25">Members of the TB group (
      <em>n</em>=45) are from ‘the roof of the world’, Tibet plateau, which is the highest plateau on the earth, with an average elevation of 4500 m. This area is inhospitable to many organisms because of its strong ultraviolet radiation, hypoxia, and severe cold (Supplementary Table S3). At high altitudes, genome integrity is continuously challenged by intensive solar ultraviolet radiation (UV-B, 280-315 nm)-induced DNA damage. Peach accessions in the TB group tolerate these conditions using several adaptation-related phenotypes, such as a dark branch color, epigeal germination, and red-colored new shoots (Supplementary Fig. S7). We identified 339 genomic regions, harboring 920 genes, showing signals of natural selection in the TB group (Supplementary Table S4). Of which, we found a significant enrichment of genes involved in ‘response to UV-B’ category (
      <em>P</em> = 0.0004) (Supplementary Table S5), which is consistent with the adaptation to high-altitude origin of the TB group. Flavonoids are a group of plant secondary metabolites, which play important roles in UV-B protection (
      <a id="xref-ref-33-1" class="xref-bibr" href="#ref-33">Li et al. 1993</a>), and we found two genes in the flavonoid biosynthetic pathway in the ‘response to UV-B’ category (
      <a id="xref-fig-4-1" class="xref-fig" href="#F4">Fig. 4A</a>): chalcone synthase (
      <em>PpCHS2, Prupe</em>.
      <em>4G252100</em>) and phenylalanine ammonia-lyase (
      <em>PpPAL, Prupe</em>.
      <em>6G235400</em>), both of which showed strong selection signals in the TB group, with high μ and Tajima’s 
      <em>D</em> values (
      <a id="xref-fig-4-2" class="xref-fig" href="#F4">Fig. 4B</a> and 
      <a id="xref-fig-4-3" class="xref-fig" href="#F4">4C</a>). Chalcone synthase catalyzes the first committed step in flavonoid biosynthesis and previous studies showed that functional perturbation of an 
      <em>A. thaliana</em> homolog, 
      <em>AtCHS</em>, resulted in UV-hypersensitive phenotypes, while in a UV-B-tolerant mutant 
      <em>Atchs</em> was up-regulated (Birza et al. 2001). We found that 
      <em>PpCHS2</em> was highly and specifically expressed in the phloem of new shoots in the TB group (
      <a id="xref-fig-4-4" class="xref-fig" href="#F4">Fig. 4D</a>), consistent with the red new shoot phenotype. By scanning genomic variants in or around 
      <em>PpCHS2</em>, we found that a SNP (Pp04: 16,896,126, A&gt;;T) causing the introduction of a premature termination codon (
      <a id="xref-fig-4-5" class="xref-fig" href="#F4">Fig. 4E</a>) showed a high frequency in low altitude accessions (76.3%), but extreme low frequency of substitution allele in the TB group (2.0%). Intriguingly, this SNP was located at the key active region for protein function, CoA-binding motif (
      <a id="xref-fig-4-6" class="xref-fig" href="#F4">Fig. 4F</a>), leading to an incomplete binding motif that may result in the loss of function. Moreover, the premature termination resulted in the loss of one conserved catalytic residue which is also crucial for catalytic activity (
      <a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">Ferrer et al. 1999</a>). Therefore, this SNP was designated as a candidate causative variant for the red new shoot phenotype involved in flavonoid-mediated UV-B adaptation. Collectively, our results suggest that selection on 
      <em>CHS</em> gene and the regulation of anthocyanins may be one of important mechanisms to confer avoiding damage from UV irradiation for peach at high altitudes.
     </p>
     <div id="F4" class="fig pos-float type-figure odd">
      <div class="highwire-figure">
       <div class="fig-inline-img-wrapper">
        <div class="fig-inline-img">
         <a href="https://www.biorxiv.org/content/biorxiv/early/2020/01/15/2020.01.15.907709/F4.large.jpg?width=800&amp;;height=600&amp;;carousel=1" title="Genomic regions and candidate genes related to high-altitude adaptation in Tibet. (A) Pathway related to plant response to UV-B. Genes under selection are highlighted in red. (B-C) Distribution of Tajima[[#x2019]]s D (B) and [[#x3BC]] values (C) in the region around PpCHS2 (Prupe.4G252100) on chromosome 4 (15.5-19.0 Mb). The dashed horizontal lines indicate a threshold of top 5% for Tajima[[#x2019]]s D ([[#x2265]] 0.36) and [[#x3BC]] test ([[#x2265]] 1.07). Arrows point to PpCHS2. (D) Heatmap of expression profiles of PpCHS2 in different tissues in low- and high-altitude accessions. (E) A candidate stop-gained SNP in PpCHS2 that is associated with high altitude adaption and new shoot colors in accessions from low- and high-altitudes. (F) Effects of stop-gained SNP on protein structure of CHS. 3D structure of CHS protein was obtained from Swiss-prot. The red shadow represents the CoA-binding motif. The green shadow represents one of the conserved enzyme active site. (G) Scanning electron microscopy (SEM) of stomata from the leaves of high- and low-altitude accessions. The magnification is 800[[#xD7]]. (H-I) Stomatal length (H) and stomatal density (i) in high- and low-altitude accessions. ** indicates P &lt;; 0.01. (J) Heatmap of expression profiles of PpEPF1 in different tissues in accessions from low- and high-altitudes. (K-L) Distribution of Tajima[[#x2019]]s D (K) and [[#x3BC]] values (L) in a region around PpEPF1 (Prupe.3G235800) on chromosome 3 (21.0-25.0 Mb). The dashed horizontal lines indicate a threshold of top 5% for Tajima[[#x2019]]s D ([[#x2265]] 0.36) and [[#x3BC]] test ([[#x2265]] 1.07). Arrows point to PpEPF1. (M) Structure of PpEPF1 and the position of the 207-bp deletion. The presence and absence of the 207-bp deletion in the seven groups are given." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-709642862" data-figure-caption="&lt;;div class=&quot;;highwire-markup&quot;;&gt;;&lt;;div xmlns=&quot;;http://www.w3.org/1999/xhtml&quot;;&gt;;&lt;;span class=&quot;;caption-title&quot;;&gt;;Genomic regions and candidate genes related to high-altitude adaptation in Tibet.&lt;;/span&gt;; (&lt;;strong&gt;;A&lt;;/strong&gt;;) Pathway related to plant response to UV-B. Genes under selection are highlighted in red. (&lt;;strong&gt;;B-C&lt;;/strong&gt;;) Distribution of Tajima[[#x2019]]s &lt;;em&gt;;D&lt;;/em&gt;; (&lt;;strong&gt;;B&lt;;/strong&gt;;) and [[#x3BC]] values (&lt;;strong&gt;;C&lt;;/strong&gt;;) in the region around &lt;;em&gt;;PpCHS2&lt;;/em&gt;; (&lt;;em&gt;;Prupe&lt;;/em&gt;;.&lt;;em&gt;;4G252100&lt;;/em&gt;;) on chromosome 4 (15.5-19.0 Mb). The dashed horizontal lines indicate a threshold of top 5% for Tajima[[#x2019]]s &lt;;em&gt;;D&lt;;/em&gt;; ([[#x2265]] 0.36) and [[#x3BC]] test ([[#x2265]] 1.07). Arrows point to &lt;;em&gt;;PpCHS2&lt;;/em&gt;;. (&lt;;strong&gt;;D&lt;;/strong&gt;;) Heatmap of expression profiles of &lt;;em&gt;;PpCHS2&lt;;/em&gt;; in different tissues in low- and high-altitude accessions. (&lt;;strong&gt;;E&lt;;/strong&gt;;) A candidate stop-gained SNP in &lt;;em&gt;;PpCHS2&lt;;/em&gt;; that is associated with high altitude adaption and new shoot colors in accessions from low- and high-altitudes. (&lt;;strong&gt;;F&lt;;/strong&gt;;) Effects of stop-gained SNP on protein structure of CHS. 3D structure of CHS protein was obtained from Swiss-prot. The red shadow represents the CoA-binding motif. The green shadow represents one of the conserved enzyme active site. (&lt;;strong&gt;;G&lt;;/strong&gt;;) Scanning electron microscopy (SEM) of stomata from the leaves of high- and low-altitude accessions. The magnification is 800[[#xD7]]. (&lt;;strong&gt;;H-I&lt;;/strong&gt;;) Stomatal length (&lt;;strong&gt;;H&lt;;/strong&gt;;) and stomatal density (&lt;;strong&gt;;i&lt;;/strong&gt;;) in high- and low-altitude accessions. ** indicates &lt;;em&gt;;P&lt;;/em&gt;; &lt;; 0.01. (&lt;;strong&gt;;J&lt;;/strong&gt;;) Heatmap of expression profiles of &lt;;em&gt;;PpEPF1&lt;;/em&gt;; in different tissues in accessions from low- and high-altitudes. (&lt;;strong&gt;;K-L&lt;;/strong&gt;;) Distribution of Tajima[[#x2019]]s &lt;;em&gt;;D&lt;;/em&gt;; (&lt;;strong&gt;;K&lt;;/strong&gt;;) and [[#x3BC]] values (&lt;;strong&gt;;L&lt;;/strong&gt;;) in a region around &lt;;em&gt;;PpEPF1&lt;;/em&gt;; (&lt;;em&gt;;Prupe&lt;;/em&gt;;.&lt;;em&gt;;3G235800&lt;;/em&gt;;) on chromosome 3 (21.0-25.0 Mb). The dashed horizontal lines indicate a threshold of top 5% for Tajima[[#x2019]]s &lt;;em&gt;;D&lt;;/em&gt;; ([[#x2265]] 0.36) and [[#x3BC]] test ([[#x2265]] 1.07). Arrows point to &lt;;em&gt;;PpEPF1&lt;;/em&gt;;. (&lt;;strong&gt;;M&lt;;/strong&gt;;) Structure of &lt;;em&gt;;PpEPF1&lt;;/em&gt;; and the position of the 207-bp deletion. The presence and absence of the 207-bp deletion in the seven groups are given.&lt;;/div&gt;;&lt;;/div&gt;;" data-icon-position="" data-hide-link-title="0">
          <span class="hw-responsive-img">
           <img class="highwire-fragment fragment-image lazyload" alt="Fig. 4" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2020/01/15/2020.01.15.907709/F4.medium.gif" width="440" height="241"/>
           <noscript>
            <img class="highwire-fragment fragment-image" alt="Fig. 4" src="https://www.biorxiv.org/content/biorxiv/early/2020/01/15/2020.01.15.907709/F4.medium.gif" width="440" height="241"/>
           </noscript>
          </span>
         </a>
        </div>
       </div>
       <ul class="highwire-figure-links inline">
        <li class="download-fig first">
         <a href="https://www.biorxiv.org/content/biorxiv/early/2020/01/15/2020.01.15.907709/F4.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Fig. 4" data-icon-position="" data-hide-link-title="0">Download figure</a>
        </li>
        <li class="new-tab last">
         <a href="https://www.biorxiv.org/content/biorxiv/early/2020/01/15/2020.01.15.907709/F4.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a>
        </li>
       </ul>
      </div>
      <div class="fig-caption">
       <span class="fig-label">Fig. 4</span>
       <span class="caption-title">Genomic regions and candidate genes related to high-altitude adaptation in Tibet.</span>
       <p id="p-26" class="first-child">(
        <strong>A</strong>) Pathway related to plant response to UV-B. Genes under selection are highlighted in red. (
        <strong>B-C</strong>) Distribution of Tajima’s 
        <em>D</em> (
        <strong>B</strong>) and μ values (
        <strong>C</strong>) in the region around 
        <em>PpCHS2</em> (
        <em>Prupe</em>.
        <em>4G252100</em>) on chromosome 4 (15.5-19.0 Mb). The dashed horizontal lines indicate a threshold of top 5% for Tajima’s 
        <em>D</em> (≥ 0.36) and μ test (≥ 1.07). Arrows point to 
        <em>PpCHS2</em>. (
        <strong>D</strong>) Heatmap of expression profiles of 
        <em>PpCHS2</em> in different tissues in low- and high-altitude accessions. (
        <strong>E</strong>) A candidate stop-gained SNP in 
        <em>PpCHS2</em> that is associated with high altitude adaption and new shoot colors in accessions from low- and high-altitudes. (
        <strong>F</strong>) Effects of stop-gained SNP on protein structure of CHS. 3D structure of CHS protein was obtained from Swiss-prot. The red shadow represents the CoA-binding motif. The green shadow represents one of the conserved enzyme active site. (
        <strong>G</strong>) Scanning electron microscopy (SEM) of stomata from the leaves of high- and low-altitude accessions. The magnification is 800×. (
        <strong>H-I</strong>) Stomatal length (
        <strong>H</strong>) and stomatal density (
        <strong>i</strong>) in high- and low-altitude accessions. ** indicates 
        <em>P</em> &lt;; 0.01. (
        <strong>J</strong>) Heatmap of expression profiles of 
        <em>PpEPF1</em> in different tissues in accessions from low- and high-altitudes. (
        <strong>K-L</strong>) Distribution of Tajima’s 
        <em>D</em> (
        <strong>K</strong>) and μ values (
        <strong>L</strong>) in a region around 
        <em>PpEPF1</em> (
        <em>Prupe</em>.
        <em>3G235800</em>) on chromosome 3 (21.0-25.0 Mb). The dashed horizontal lines indicate a threshold of top 5% for Tajima’s 
        <em>D</em> (≥ 0.36) and μ test (≥ 1.07). Arrows point to 
        <em>PpEPF1</em>. (
        <strong>M</strong>) Structure of 
        <em>PpEPF1</em> and the position of the 207-bp deletion. The presence and absence of the 207-bp deletion in the seven groups are given.
       </p>
       <div class="sb-div caption-clear"/>
      </div>
     </div>
     <p id="p-27">We observed that, compared with low-altitude accessions, those from high-altitudes had a lower density of stomata and larger stomata size (
      <a id="xref-fig-4-7" class="xref-fig" href="#F4">Fig. 4G-4I</a>). This may represent an adaptive evolution to hypoxia at high altitudes. Interestingly, we found that the biological category ‘stomatal complex patterning’ was significantly enriched in the gene set under selection (
      <em>P</em> = 0.008). By transcriptional analyses of these genes, we found one of them, 
      <em>Prupe</em>.
      <em>3G235800</em>, was highly and specifically expressed in leaves, showing an altitudinal pattern with higher expression levels in the TB group than in the low-altitude group (
      <a id="xref-fig-4-8" class="xref-fig" href="#F4">Fig. 4J</a>). Notably, 
      <em>Prupe</em>.
      <em>3G235800</em>, which encodes the epidermal patterning factor 1 (PpEPF1) involved in stomatal development (
      <a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">Hara et al. 2009</a>), showed strong selection signals, based on the high Tajima’s 
      <em>D</em> and μ values (
      <a id="xref-fig-4-9" class="xref-fig" href="#F4">Fig. 4K</a> and 
      <a id="xref-fig-4-10" class="xref-fig" href="#F4">4L</a>). Previous studies have shown that the mutation of a homolog of 
      <em>PpEPF1</em> in 
      <em>A. thaliana</em> results in increased stomatal density (
      <a id="xref-ref-19-2" class="xref-bibr" href="#ref-19">Hara et al. 2009</a>). By scanning the variants in 
      <em>PpEPF1</em>, we found that SNPs with functional significance were absent. Through further scanning variants at the upstream or downstream of 
      <em>PpEPF1</em>, we identified a TB group specific 207-bp deletion in the promoter region (−758 bp from the start codon) of 
      <em>PpEPF1</em> (
      <a id="xref-fig-4-11" class="xref-fig" href="#F4">Fig. 4M</a>), suggesting that the adaptive evolution controlled by 
      <em>PpEPF1</em> may be mediated by regulation of its expression. Furthermore, over-expression of 
      <em>PpEPF1</em> in 
      <em>A. thaliana</em> resulted in a decrease in stomatal density (Supplementary Fig. S8). These findings suggest that selection on 
      <em>PpEPF1</em> may be closely related to adaptation to hypoxia in high-altitudes through the regulation of stomatal density.
     </p>
    </div>
    <div id="sec-7" class="subsection">
     <h3>A major 
      <em>SVP</em> locus involved in adaptive evolution of bloom date
     </h3>
     <p id="p-28">Bloom date (BD) is crucial for local adaptation in peach, and is controlled by multiple genes (
      <a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">Fan et al. 2010</a>). To explore the genetic basis of adaptation of BD, we performed GWAS of BD using 174 accessions that were phenotyped. This revealed 399 associated SNPs and 12 association peaks (
      <a id="xref-fig-5-1" class="xref-fig" href="#F5">Fig. 5A</a>), of which six overlapped with previously reported QTLs (
      <a id="xref-ref-13-2" class="xref-bibr" href="#ref-13">Fan et al. 2010</a>). Next, we identified candidates involved in local adaptation by detecting SNPs showing associations with EVs using a latent factor mixed-effect model (LFMM), resulting in a final set of 23 association peaks (
      <a id="xref-fig-5-2" class="xref-fig" href="#F5">Fig. 5A</a>). By overlapping BD GWAS and LFMM analyses, we found four regions on chromosomes 3, 5, 6, and 8 that may underlie the local adaptation of BD during spread of peach to different climates (
      <a id="xref-fig-5-3" class="xref-fig" href="#F5">Fig. 5A</a>).
     </p>
     <div id="F5" class="fig pos-float type-figure odd">
      <div class="highwire-figure">
       <div class="fig-inline-img-wrapper">
        <div class="fig-inline-img">
         <a href="https://www.biorxiv.org/content/biorxiv/early/2020/01/15/2020.01.15.907709/F5.large.jpg?width=800&amp;;height=600&amp;;carousel=1" title="A major PpSVP locus involved in local adaptation of bloom date in peach. (A) Manhattan plots of SNPs associated with EVs (LFMM), bloom date (BD), and chilling requirement (CR). Dashed lines represent the significance thresholds for the tests. The overlapped regions between GWAS for BD and LFMM are highlighted using green shaded rectangles. The major QTL for CR and BD overlapping with local selection signals on chromosome 8 surrounding PpSVP is indicated by a blue triangle. The EVG locus is highlighted using a gray shaded rectangle. (B) Neighbor-joining tree of PpSVP and MIKC-type MADS family genes. The clade containing PpSVP is highlighted in red. (C) Temporal and spatial expression patterns of PpSVP. Error bars represent standard deviation of three biological replicates. (D) Patterns of normalized iHS scores across the [[#x223C]]4 Mb genomic region around PpSVP. The dashed horizontal lines represent the threshold of positive selection signal (|iHS| &gt;; 2.5). The blue dot indicates the SNP (Pp08: 10,173,576) that showed high iHS score in PpSVP. (E) FST around PpSVP among different groups. The associated SNP in PpSVP is indicated using vertical black line. (F) Allelic frequencies of the associated SNP (Pp08: 10,173,576) in PpSVP across seven groups. (G) Relationship between genotypes of associated SNP (Pp08: 10,173,576) and bloom date." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-709642862" data-figure-caption="&lt;;div class=&quot;;highwire-markup&quot;;&gt;;&lt;;div xmlns=&quot;;http://www.w3.org/1999/xhtml&quot;;&gt;;&lt;;span class=&quot;;caption-title&quot;;&gt;;A major &lt;;em&gt;;PpSVP&lt;;/em&gt;; locus involved in local adaptation of bloom date in peach.&lt;;/span&gt;; (&lt;;strong&gt;;A&lt;;/strong&gt;;) Manhattan plots of SNPs associated with EVs (LFMM), bloom date (BD), and chilling requirement (CR). Dashed lines represent the significance thresholds for the tests. The overlapped regions between GWAS for BD and LFMM are highlighted using green shaded rectangles. The major QTL for CR and BD overlapping with local selection signals on chromosome 8 surrounding &lt;;em&gt;;PpSVP&lt;;/em&gt;; is indicated by a blue triangle. The &lt;;em&gt;;EVG&lt;;/em&gt;; locus is highlighted using a gray shaded rectangle. (&lt;;strong&gt;;B&lt;;/strong&gt;;) Neighbor-joining tree of &lt;;em&gt;;PpSVP&lt;;/em&gt;; and MIKC-type MADS family genes. The clade containing &lt;;em&gt;;PpSVP&lt;;/em&gt;; is highlighted in red. (&lt;;strong&gt;;C&lt;;/strong&gt;;) Temporal and spatial expression patterns of &lt;;em&gt;;PpSVP&lt;;/em&gt;;. Error bars represent standard deviation of three biological replicates. (&lt;;strong&gt;;D&lt;;/strong&gt;;) Patterns of normalized iHS scores across the [[#x223C]]4 Mb genomic region around &lt;;em&gt;;PpSVP&lt;;/em&gt;;. The dashed horizontal lines represent the threshold of positive selection signal (|iHS| &gt;; 2.5). The blue dot indicates the SNP (Pp08: 10,173,576) that showed high iHS score in &lt;;em&gt;;PpSVP&lt;;/em&gt;;. (&lt;;strong&gt;;E&lt;;/strong&gt;;) &lt;;em&gt;;F&lt;;/em&gt;;&lt;;sub&gt;;ST&lt;;/sub&gt;; around &lt;;em&gt;;PpSVP&lt;;/em&gt;; among different groups. The associated SNP in &lt;;em&gt;;PpSVP&lt;;/em&gt;; is indicated using vertical black line. (&lt;;strong&gt;;F&lt;;/strong&gt;;) Allelic frequencies of the associated SNP (Pp08: 10,173,576) in &lt;;em&gt;;PpSVP&lt;;/em&gt;; across seven groups. (&lt;;strong&gt;;G&lt;;/strong&gt;;) Relationship between genotypes of associated SNP (Pp08: 10,173,576) and bloom date.&lt;;/div&gt;;&lt;;/div&gt;;" data-icon-position="" data-hide-link-title="0">
          <span class="hw-responsive-img">
           <img class="highwire-fragment fragment-image lazyload" alt="Fig. 5" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2020/01/15/2020.01.15.907709/F5.medium.gif" width="440" height="217"/>
           <noscript>
            <img class="highwire-fragment fragment-image" alt="Fig. 5" src="https://www.biorxiv.org/content/biorxiv/early/2020/01/15/2020.01.15.907709/F5.medium.gif" width="440" height="217"/>
           </noscript>
          </span>
         </a>
        </div>
       </div>
       <ul class="highwire-figure-links inline">
        <li class="download-fig first">
         <a href="https://www.biorxiv.org/content/biorxiv/early/2020/01/15/2020.01.15.907709/F5.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Fig. 5" data-icon-position="" data-hide-link-title="0">Download figure</a>
        </li>
        <li class="new-tab last">
         <a href="https://www.biorxiv.org/content/biorxiv/early/2020/01/15/2020.01.15.907709/F5.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a>
        </li>
       </ul>
      </div>
      <div class="fig-caption">
       <span class="fig-label">Fig. 5</span>
       <span class="caption-title">A major 
        <em>PpSVP</em> locus involved in local adaptation of bloom date in peach.
       </span>
       <p id="p-29" class="first-child">(
        <strong>A</strong>) Manhattan plots of SNPs associated with EVs (LFMM), bloom date (BD), and chilling requirement (CR). Dashed lines represent the significance thresholds for the tests. The overlapped regions between GWAS for BD and LFMM are highlighted using green shaded rectangles. The major QTL for CR and BD overlapping with local selection signals on chromosome 8 surrounding 
        <em>PpSVP</em> is indicated by a blue triangle. The 
        <em>EVG</em> locus is highlighted using a gray shaded rectangle. (
        <strong>B</strong>) Neighbor-joining tree of 
        <em>PpSVP</em> and MIKC-type MADS family genes. The clade containing 
        <em>PpSVP</em> is highlighted in red. (
        <strong>C</strong>) Temporal and spatial expression patterns of 
        <em>PpSVP</em>. Error bars represent standard deviation of three biological replicates. (
        <strong>D</strong>) Patterns of normalized iHS scores across the ∼4 Mb genomic region around 
        <em>PpSVP</em>. The dashed horizontal lines represent the threshold of positive selection signal (|iHS| &gt;; 2.5). The blue dot indicates the SNP (Pp08: 10,173,576) that showed high iHS score in 
        <em>PpSVP</em>. (
        <strong>E</strong>) 
        <em>F</em>
        <sub>ST</sub> around 
        <em>PpSVP</em> among different groups. The associated SNP in 
        <em>PpSVP</em> is indicated using vertical black line. (
        <strong>F</strong>) Allelic frequencies of the associated SNP (Pp08: 10,173,576) in 
        <em>PpSVP</em> across seven groups. (
        <strong>G</strong>) Relationship between genotypes of associated SNP (Pp08: 10,173,576) and bloom date.
       </p>
       <div class="sb-div caption-clear"/>
      </div>
     </div>
     <p id="p-30">Chilling requirement (CR) is another important adaptive trait and is significantly correlated with BD. We re-performed the GWAS for CR based on our previous study (
      <a id="xref-ref-34-4" class="xref-bibr" href="#ref-34">Li et al. 2019</a>) using 174 landrace accessions and identified six association peaks, of which three (chromosome 1, 7, and 8) were shared with BD (
      <a id="xref-fig-5-4" class="xref-fig" href="#F5">Fig. 5A</a>), including the major QTL for CR harboring the 
      <em>EVG</em> locus conferring dormancy mutation in peach (
      <a id="xref-ref-32-1" class="xref-bibr" href="#ref-32">Li et al. 2009</a>). After overlapping GWAS of CR and BD with the LFMM analysis, we found a strong overlap spanning ∼4-Mb on chromosome 8, which may be important for local adaptation of BD in peach (
      <a id="xref-fig-5-5" class="xref-fig" href="#F5">Fig. 5A</a>). Interestingly, the major QTL for CR and BD on chromosome 1 showed no local adaptation signal in the LFMM analysis (
      <a id="xref-fig-5-6" class="xref-fig" href="#F5">Fig. 5A</a>), suggesting that climates may drive the evolution of BD and CR by shaping QTLs with small effects.
     </p>
     <p id="p-31">The 4-Mb region encompasses 275 genes, including a putative ortholog of 
      <em>A. thaliana SHORT VEGETATIVE PHASE</em> (
      <em>PpSVP, Prupe</em>.
      <em>8G069300</em>). 
      <em>SVP</em> is involved in controlling flowering time and has previously been implicated in regulating dormancy in 
      <em>Prunus</em> (
      <a id="xref-ref-32-2" class="xref-bibr" href="#ref-32">Li et al. 2009</a>; 
      <a id="xref-ref-49-1" class="xref-bibr" href="#ref-49">Sasaki et al. 2011</a>; 
      <a id="xref-ref-58-1" class="xref-bibr" href="#ref-58">Zhang et al. 2012</a>). Phylogenetic analysis confirmed that 
      <em>PpSVP</em> belongs to a MADS-box family and is closely related to the 
      <em>AGL22</em> subfamily (
      <a id="xref-fig-5-7" class="xref-fig" href="#F5">Fig. 5B</a>). 
      <em>PpSVP</em> showed strong tissue-specific expression, with high expression only in vegetative organs. Moreover, expression of 
      <em>PpSVP</em> was up-regulated during dormancy induction and down-regulated by winter chill (0-7.2°C) and by forcing temperature (heat) in floral buds in spring (
      <a id="xref-fig-5-8" class="xref-fig" href="#F5">Fig. 5C</a>), suggesting its potential role in regulating BD and CR. Moreover, through calculating the standardized integrated haplotype score (iHS) for SNPs located in this overlap region, we found a strong positive selection signal around the 
      <em>PpSVP</em> locus (
      <a id="xref-fig-5-9" class="xref-fig" href="#F5">Fig. 5D</a>). Additionally, an exceptionally high 
      <em>F</em>
      <sub>ST</sub> value was identified in this region, especially between the ST and NE groups and between the ST and YT groups (
      <a id="xref-fig-5-10" class="xref-fig" href="#F5">Fig. 5E</a>) that harbor distinct bloom date. The 
      <em>PpSVP</em> locus thus represents a strong candidate gene for local adaptation of BD and CR. We propose that spatially varying selection has driven latitudinal differentiation at this locus. Positive selection signals, revealed by a CLR test, were also detected in the NE and ST groups (
      <a id="xref-fig-5-11" class="xref-fig" href="#F5">Fig. 5F</a>). Overall, all these results provide compelling evidence of local selection on the 
      <em>PpSVP</em> locus during adaptive evolution to different climates after domestication.
     </p>
     <p id="p-32">To identify the causal variants underlying adaptation of BD, we screened for SNPs with high 
      <em>F</em>
      <sub>ST</sub> between the NE (late bloom) and ST (early bloom) groups at the 
      <em>PpSVP</em> locus. No SNP with high differentiation was identified that caused an amino acid change. However, a SNP located at 5’-untranslated regions (5’-UTR) with high 
      <em>F</em>
      <sub>ST</sub> value (
      <em>F</em>
      <sub>ST</sub>=0.9) was identified, suggesting that the BD and CR may adapt to different climates through shaping the expression of the controlled gene. Allele frequencies of this SNP showed strong geographical pattern and the early bloom alleles (CC) mainly occurred in low altitude regions (ST and YG groups) and wild group (TB group) (
      <a id="xref-fig-5-12" class="xref-fig" href="#F5">Fig. 5G</a> and 
      <a id="xref-fig-5-13" class="xref-fig" href="#F5">5H</a>), consistent with phenotype. This also provides insights into two distinct evolutionary routes of BD and CR to low and high chill regions. Moreover, overexpression of the low-altitude favored genotype of 
      <em>PpSVP</em> (CC) in 
      <em>A. thaliana</em> resulted in plants with strong vegetative growth and delayed flowering time (
      <a id="xref-ref-34-5" class="xref-bibr" href="#ref-34">Li et al. 2019</a>).
     </p>
    </div>
    <div id="sec-8" class="subsection">
     <h3>Genomic locus associated with response to climate change</h3>
     <p id="p-33">Adaptation to accelerating rates of climate change is increasingly important for species survival. The advance in bloom date (ABD), as a consequence of global warming over recent decades, has been observed in many temperate species, including peach (
      <a id="xref-ref-40-1" class="xref-bibr" href="#ref-40">Menzel et al. 2006</a>; 
      <a id="xref-ref-35-1" class="xref-bibr" href="#ref-35">Li et al. 2016</a>). However, the genetic mechanism underlying ABD have not been characterized. We performed a long-term observation of BD with 89 peach accessions spanning three decades, from the 1980s to 2010s (Supplementary Fig. S9A). We observed a significant ABD (
      <em>P</em> &lt;; 0.001), based on an additive main effects and multiplicative interaction (AMMI) analysis (Annicchiarico, 1997), and the main driver was found to be a temperature rise in the spring (explained 61.3% variation, 
      <em>P</em> &lt;; 0.001) (
      <a id="xref-fig-6-1" class="xref-fig" href="#F6">Fig. 6A</a>). Using a linear regression analysis, we quantified ABD and found that BD has advanced by approximately 10 days on average over last 30 years (
      <a id="xref-fig-6-2" class="xref-fig" href="#F6">Fig. 6B</a>).
     </p>
     <div id="F6" class="fig pos-float type-figure odd">
      <div class="highwire-figure">
       <div class="fig-inline-img-wrapper">
        <div class="fig-inline-img">
         <a href="https://www.biorxiv.org/content/biorxiv/early/2020/01/15/2020.01.15.907709/F6.large.jpg?width=800&amp;;height=600&amp;;carousel=1" title="Genotype-environment interaction analysis and genome-wide association study of advance in bloom date. (A) Genotype-environment interaction analysis of bloom date from 1983 to 2011 using the AMMI analysis. (B) Scatter plots of relative bloom date of 89 peach accessions from 1983 to 2011 and temperature change in the spring. The blue and orange lines represent the trend of bloom date changes and temperature changes in the spring, respectively, based on the linear regression analyses. [[#x394]]T24 indicates anomalies in the mean temperature from February to April compared to those from 1983-2011. (C) Regional Manhattan plot of GWAS for ABD on chromosome 8 of the 7.0-14.0 Mb region. The gray dashed line indicates significance threshold (P &lt;; 7.28[[#xD7]]10[[#x2212]]8 or [[#x2212]]log10(P) &gt;; 7.08) using a Bonferroni test (0.05). (D) Most significant SNP associated with ABD and its location relative to gene PpLNK1 (Prupe.8G062200). (E) Association between genotypes of the most significant SNP and ABD. (F) Changes in PpLNK1 expression in three cultivars in a climate warming simulation experiment. dpt, days post treatment. (G) Comparison of PpLNK1 expression between accessions sensitive and insensitive to global warming at blooming. ** represents P &lt;; 0.01. (H) Comparison of BD between wild type (WT) and PpLNK1 over-expression (OE) A. thaliana lines. * indicates P &lt;; 0.05." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-709642862" data-figure-caption="&lt;;div class=&quot;;highwire-markup&quot;;&gt;;&lt;;div xmlns=&quot;;http://www.w3.org/1999/xhtml&quot;;&gt;;&lt;;span class=&quot;;caption-title&quot;;&gt;;Genotype-environment interaction analysis and genome-wide association study of advance in bloom date.&lt;;/span&gt;; (&lt;;strong&gt;;A&lt;;/strong&gt;;) Genotype-environment interaction analysis of bloom date from 1983 to 2011 using the AMMI analysis. (&lt;;strong&gt;;B&lt;;/strong&gt;;) Scatter plots of relative bloom date of 89 peach accessions from 1983 to 2011 and temperature change in the spring. The blue and orange lines represent the trend of bloom date changes and temperature changes in the spring, respectively, based on the linear regression analyses. [[#x394]]T24 indicates anomalies in the mean temperature from February to April compared to those from 1983-2011. (&lt;;strong&gt;;C&lt;;/strong&gt;;) Regional Manhattan plot of GWAS for ABD on chromosome 8 of the 7.0-14.0 Mb region. The gray dashed line indicates significance threshold (&lt;;em&gt;;P&lt;;/em&gt;; &lt;; 7.28[[#xD7]]10&lt;;sup&gt;;[[#x2212]]8&lt;;/sup&gt;; or [[#x2212]]log10(&lt;;em&gt;;P&lt;;/em&gt;;) &gt;; 7.08) using a Bonferroni test (0.05). (&lt;;strong&gt;;D&lt;;/strong&gt;;) Most significant SNP associated with ABD and its location relative to gene &lt;;em&gt;;PpLNK1&lt;;/em&gt;; (&lt;;em&gt;;Prupe&lt;;/em&gt;;.&lt;;em&gt;;8G062200&lt;;/em&gt;;). (&lt;;strong&gt;;E&lt;;/strong&gt;;) Association between genotypes of the most significant SNP and ABD. (&lt;;strong&gt;;F&lt;;/strong&gt;;) Changes in &lt;;em&gt;;PpLNK1&lt;;/em&gt;; expression in three cultivars in a climate warming simulation experiment. dpt, days post treatment. (&lt;;strong&gt;;G&lt;;/strong&gt;;) Comparison of &lt;;em&gt;;PpLNK1&lt;;/em&gt;; expression between accessions sensitive and insensitive to global warming at blooming. ** represents &lt;;em&gt;;P&lt;;/em&gt;; &lt;; 0.01. (&lt;;strong&gt;;H&lt;;/strong&gt;;) Comparison of BD between wild type (WT) and &lt;;em&gt;;PpLNK1&lt;;/em&gt;; over-expression (OE) &lt;;em&gt;;A. thaliana&lt;;/em&gt;; lines. * indicates &lt;;em&gt;;P&lt;;/em&gt;; &lt;; 0.05.&lt;;/div&gt;;&lt;;/div&gt;;" data-icon-position="" data-hide-link-title="0">
          <span class="hw-responsive-img">
           <img class="highwire-fragment fragment-image lazyload" alt="Fig. 6" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2020/01/15/2020.01.15.907709/F6.medium.gif" width="440" height="245"/>
           <noscript>
            <img class="highwire-fragment fragment-image" alt="Fig. 6" src="https://www.biorxiv.org/content/biorxiv/early/2020/01/15/2020.01.15.907709/F6.medium.gif" width="440" height="245"/>
           </noscript>
          </span>
         </a>
        </div>
       </div>
       <ul class="highwire-figure-links inline">
        <li class="download-fig first">
         <a href="https://www.biorxiv.org/content/biorxiv/early/2020/01/15/2020.01.15.907709/F6.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Fig. 6" data-icon-position="" data-hide-link-title="0">Download figure</a>
        </li>
        <li class="new-tab last">
         <a href="https://www.biorxiv.org/content/biorxiv/early/2020/01/15/2020.01.15.907709/F6.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a>
        </li>
       </ul>
      </div>
      <div class="fig-caption">
       <span class="fig-label">Fig. 6</span>
       <span class="caption-title">Genotype-environment interaction analysis and genome-wide association study of advance in bloom date.</span>
       <p id="p-34" class="first-child">(
        <strong>A</strong>) Genotype-environment interaction analysis of bloom date from 1983 to 2011 using the AMMI analysis. (
        <strong>B</strong>) Scatter plots of relative bloom date of 89 peach accessions from 1983 to 2011 and temperature change in the spring. The blue and orange lines represent the trend of bloom date changes and temperature changes in the spring, respectively, based on the linear regression analyses. ΔT24 indicates anomalies in the mean temperature from February to April compared to those from 1983-2011. (
        <strong>C</strong>) Regional Manhattan plot of GWAS for ABD on chromosome 8 of the 7.0-14.0 Mb region. The gray dashed line indicates significance threshold (
        <em>P</em> &lt;; 7.28×10
        <sup>−8</sup> or −log10(
        <em>P</em>) &gt;; 7.08) using a Bonferroni test (0.05). (
        <strong>D</strong>) Most significant SNP associated with ABD and its location relative to gene 
        <em>PpLNK1</em> (
        <em>Prupe</em>.
        <em>8G062200</em>). (
        <strong>E</strong>) Association between genotypes of the most significant SNP and ABD. (
        <strong>F</strong>) Changes in 
        <em>PpLNK1</em> expression in three cultivars in a climate warming simulation experiment. dpt, days post treatment. (
        <strong>G</strong>) Comparison of 
        <em>PpLNK1</em> expression between accessions sensitive and insensitive to global warming at blooming. ** represents 
        <em>P</em> &lt;; 0.01. (
        <strong>H</strong>) Comparison of BD between wild type (WT) and 
        <em>PpLNK1</em> over-expression (OE) 
        <em>A. thaliana</em> lines. * indicates 
        <em>P</em> &lt;; 0.05.
       </p>
       <div class="sb-div caption-clear"/>
      </div>
     </div>
     <p id="p-35">Next, we performed GWAS for ABD to identify genetic loci associated with responses to global warming (Supplementary Fig. S9B). This revealed a strong association peak on chromosome 8 (
      <em>P</em> &lt;; 7.28 × 10
      <sup>−8</sup>) (
      <a id="xref-fig-6-3" class="xref-fig" href="#F6">Fig. 6C</a>) in an area harboring 14 candidate genes around peak association. This association was also located at overlap among GWAS signals of CR, BD and LFMM analysis. The most significant SNP was located in a region upstream of 
      <em>Prupe</em>.
      <em>8G062200</em>, with genotype of TT showing sensitive to global warming and CC insensitive (
      <a id="xref-fig-6-4" class="xref-fig" href="#F6">Fig. 6D</a> and 
      <a id="xref-fig-6-5" class="xref-fig" href="#F6">6E</a>). 
      <em>Prupe</em>.
      <em>8G062200</em> encodes a putative night light-inducible and clock-regulated 1 (LNK1) protein, and showed high expression levels at blooming. A homolog of this gene in 
      <em>A. thaliana</em> is involved in regulation of the circadian clock, which regulates 
      <em>COL1</em> genes at warm temperatures, and thus a potential regulator of flowering time (
      <a id="xref-ref-41-1" class="xref-bibr" href="#ref-41">Mikkelsen and Thomashow 2009</a>; 
      <a id="xref-ref-48-1" class="xref-bibr" href="#ref-48">Rugnone et al. 2013</a>). A simulation experiment showed that 
      <em>PpLNK1</em> was up-regulated by rising temperatures during heat accumulation, suggesting that 
      <em>PpLNK1</em> may be up-regulated by temperature rise in spring (
      <a id="xref-fig-6-6" class="xref-fig" href="#F6">Fig. 6F</a>). In addition, expression of 
      <em>PpLNK1</em> in peach accessions that are sensitive to global warming was significantly higher than in those that are insensitive (
      <a id="xref-fig-6-7" class="xref-fig" href="#F6">Fig. 6G</a>). Notably, over-expression of 
      <em>PpLNK1</em> in model plant, 
      <em>A. thaliana</em>, led to the early flowering (
      <a id="xref-fig-6-8" class="xref-fig" href="#F6">Fig. 6H</a>). Moreover, several cis-elements associated with temperature and light responsiveness was identified (Supplementary Table S10). Therefore, we conclude that 
      <em>PpLNK1</em> may play important roles in regulating annual circadian clock of flowering time as influenced by rising temperature in peach. 
      <em>PpLNK1</em> is thus a plausible candidate gene for responses to global warming, but further work will be necessary to provide more direct evidence of its roles. Collectively, our comprehensive analyses detected genomic loci associated with responses to global warming, which can improve our understanding of the genetic architecture of plant adaptation to global climate change.
     </p>
     <p id="p-36">Long-term observation of BD enabled multi-year GWAS. We identified a total of 713 SNPs associated with BD (
      <em>P</em> &lt;; 7.28 × 10
      <sup>−8</sup>), including 483 temporary associations that were identified only in one year, 214 associations in at least two years, and 16 stable associations in more than five years, of which several overlapped with previous reported QTLs (
      <a id="xref-ref-13-3" class="xref-bibr" href="#ref-13">Fan et al. 2010</a>) (Supplementary Table S11). Among stable associations, a strong association peak within a small intergenic region (Pp06: 15,327,714∼15,354,080) on chromosome 6 was identified in eight years of GWAS, which can be further developed for marker-assisted selection.
     </p>
    </div>
   </div>
   <div class="section" id="sec-9">
    <h2 class="">Conclusions</h2>
    <p id="p-37">Plant genomes have been shaped by natural selection during the local adaptation to diverse environmental conditions. Peach provides an excellent model to investigate the genetic basis and mode of adaptation to climate change, thanks to its relatively small genome size (∼227.4Mb) and extensive climatic variation across its native range. We generated a large variation map for peach through sequencing of a climate-extensive panel of 263 peach landraces and wild relatives. Notably, we first detected the genetic basis of adaptation to high altitudes for fruit species, 
     <em>P. mira</em> (TB group), and we found that genes involved in the biosynthesis of flavonoids 
     <em>(PpCHS2</em>) and stomatal development (
     <em>PpEPF1</em>) may play important roles in overcoming strong UV-B radiation and hypoxia, respectively, on the Tibet Plateau. We discovered that high sugar content and yellow flesh of peach in drought regions were drought-induced adaptive evolution mediated by interactions between the abscisic acid pathway, 
     <em>PpSPS1</em> and carotenoids. More than nine thousand genomic loci, associated with 51 specific climate variables, were identified. These included several hotspots associated with temperature and precipitation, as well as a SNP associated with cold hardiness. Integrative analyses of selective sweeps and GWEAS suggest that peach adaptation was generated and enhanced by standing selection on multi sites. Genomic loci underlying the local adaption of BD and CR were found to be two evolutionary adaptations to low and high latitude regions. In addition, through data collected over a 30-year period, we identified a candidate genetic locus associated with responses to global warming in plant species.
    </p>
    <p id="p-38">This study provides new insights into peach adaptation to its habits and how climate has shaped the genome of a perennial tree plant through natural selection. These results also provide a new resource for studies of peach evolutionary biology and breeding, especially with regard to enhancing stress-resistance.</p>
   </div>
   <div class="section" id="sec-10">
    <h2 class="">Methods</h2>
    <div id="sec-11" class="subsection">
     <h3>Plant materials and sequencing</h3>
     <p id="p-39">A total of 263 peach accessions were sampled from the NPGRC (National Peach Germplasm Repository of China), except the 45 
      <em>P. mira</em> accessions, which were sampled from the Tibet plateau. These accessions, collected from almost all the distribution regions of peach landraces and wild relatives, including seven major ecotypes. These accessions included 45 of 
      <em>P. mira</em> Koehne, 4 of 
      <em>P. davidiana</em> (Carr.) Franch., 2 of 
      <em>P. kansuensis</em> Rehd., a single 
      <em>P. potaninii</em> Batal., 205 of 
      <em>P. persica</em> L., and 6 of 
      <em>P. ferganensis</em> Kost. et Riab (Supplemental Table 1). Of these, 
      <em>P. persica</em> L. and 
      <em>P. ferganensis</em> Kost. et Riab accessions belong to landraces, while the others are wild relatives. Total genomic DNA was extracted from young leaves using the cetyltriethylammnonium bromide (CTAB) method (
      <a id="xref-ref-43-1" class="xref-bibr" href="#ref-43">Murray and Thompson 1980</a>). At least 4 μg of genomic DNA from each accession was used to construct pair-end sequencing libraries with insert sizes of approximately 300-bp or 500-bp following the manufacturer’s instructions (Illumina Inc.) (Supplemental Table 1). A total of &gt;;1 Gb of sequence data was generated for each accession from 49-bp, 90-bp, or 125-bp paired-end reads, using the Illumina GA or HiSeq 2500 platform (Illumina, San Diego, USA) (Supplemental Table S1).
     </p>
    </div>
    <div id="sec-12" class="subsection">
     <h3>Read mapping and variation calling</h3>
     <p id="p-40">Pair-end reads from each accession were mapped to the peach Lovell genome (release v2.0) using BWA (
      <a id="xref-ref-30-1" class="xref-bibr" href="#ref-30">Li and Durbin 2009</a>) (Version: 0.7.12) with the following parameters: bwa mem -t 4 - M -R. Read alignments were converted into the BAM format, sorted according mapping coordinates, and PCR duplicates removed using the Picard package (
      <a href="http://broadinstitute.github.io/picard/">http://broadinstitute.github.io/picard/</a>; Version: 1.136) with default parameters. The coverage and depth of sequence alignments were computed using the Genome Analysis Toolkit (GATK, version: 3.4-46; see URLs) DepthOfCoverage program (
      <a id="xref-ref-39-1" class="xref-bibr" href="#ref-39">McKenna et al. 2010</a>). The coverage and depth of each accession are detailed in Supplemental Table S1.
     </p>
     <p id="p-41">To accurately identify SNPs, the low-quality alignments (a mapping quality score &lt;;20) were filtered using SAMtools (
      <a id="xref-ref-32-3" class="xref-bibr" href="#ref-32">Li et al. 2009</a>). SNP detection was performed using GATK HaplotypeCaller, which identifies SNPs by local 
      <em>de novo</em> assembly of haplotypes in an active region (
      <a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">Depristo et al. 2011</a>). The detailed processes were as follows: (1) After filtering the low-quality alignments, the reads around the INDELs were realigned through two steps, including identifying regions where realignment was needed using the GATK RealignerTargetCreator package, and realigning the regions found in the first step GATK IndelRealigner package. Next, a realigned BAM file for each accession, which was used for SNP detection, was generated using GATK PrintReads packages. (2) SNPs were detected at a population level using the realigned BAM file with GATK HaplotypeCaller. To reduce the number of false positives, a high SNP confidence score was set with the following parameters: -stand_call_conf 30 -stand_emit_conf 40. (3) To ensure the quality of variant calling, a hard filter was applied for the raw SNPs with SNP quality &gt;; 40 and the number of supporting reads &gt;; 2, using GTAK VariantFiltration, with the following parameters: QUAL &lt;; 40, QD &lt;; 2.0, FS &gt;; 60.0, MQ &lt;; 40.0, MQRankSum &lt;; −12.5, ReadPosRankSum &lt;; −8.0, -cluster 3, -window 10.
     </p>
     <p id="p-42">The accuracy of SNPs was assessed using a Sequenom MassARRAY platform (Sequenom, San Diego, USA), following the manufacturer’s protocol. A total of 18 randomly selected SNPs was investigated in 130 accessions. The list of accessions is provided in Supplemental Table S2.</p>
     <p id="p-43">INDEL calling was performed using the same pipeline as the SNP calling since the GATK is capable of calling SNPs and INDELs simultaneously. To reduce the number of false positives, we also applied a harder filter for raw INDELs using GTAK VariantFiltration with the following parameters: QD &lt;; 2.0, FS &gt;; 200.0, ReadPosRankSum &lt;; −20.0. Insertions and deletions ≤6 bp were defined as the small INDELs.</p>
     <p id="p-44">SV calling was performed using the SpeedSeq (
      <a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">Chiang et al. 2015</a>), DELLY (
      <a id="xref-ref-53-1" class="xref-bibr" href="#ref-53">Tobias et al. 2012</a>), and manta (
      <a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">Chen et al. 2016</a>) programs. For SpeedSeq calling, paired-end reads were mapped to the reference genome using the ‘align’ module in SpeedSeq and the following parameters: speedseq align -R -t 4. Three BAM files were generated, including a full, duplicate-marked, sorted BAM, a BAM file containing split reads, and a BAM file containing discordant read-pairs. SVs were identified using the ‘sv’ module in SpeedSeq, using the following settings: speedseq sv -o -x -t 25 -R -B -D -S -g -P. For DELLY calling, mapped pair-end reads in BAM format, generated by BWA-MEM (
      <a id="xref-ref-30-2" class="xref-bibr" href="#ref-30">Li and Durbin 2009</a>) after sorting and marking PCR duplicates, were used as input files. SVs were identified using the call module in DELLY with default parameters. SV files in VCF format for all of 263 samples were merged into a population level VCF file using BCFtools (
      <a id="xref-ref-32-4" class="xref-bibr" href="#ref-32">Li et al. 2009</a>). For SV calling with manta, the same BAM files with DELLY were used to detect SVs, with default parameters. SV files for 263 accessions were then merged using SURIVAR (
      <a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">Jeffares et al 2017</a>) and genotyped using SVtyper (
      <a id="xref-ref-8-2" class="xref-bibr" href="#ref-8">Chiang et al. 2015</a>) with default parameters. Finally, SVs identified by at least two callers were designated as the final set of SVs.
     </p>
    </div>
    <div id="sec-13" class="subsection">
     <h3>SNP annotation</h3>
     <p id="p-45">SNP annotation was performed based on genomic locations and predicted coding effects, according to the peach genome annotation (release annotation v2.1, see URLs), using the snpEff (
      <a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">Cingolani et al. 2012</a>) (Version: 4.1g). The final SNPs were categorized in exonic regions, intronic regions, splicing sites, 5’ UTRs and 3’ UTRs, upstream and downstream regions, and intergenic regions, based on the peach genome annotation. SNPs in coding sequence were further grouped into synonymous SNPs (no amino acid changes) and nonsynonymous SNPs (amino acid changes). SNP effects were further divided into four types according to their impacts on gene function, including HIGH, MODERATE, LOW, and MODIFIER.
     </p>
    </div>
    <div id="sec-14" class="subsection">
     <h3>Population genetics analysis</h3>
     <p id="p-46">To build a phylogenetic tree, we selected a subset of 2,468,307 SNPs with minor allele frequency (MAF) &gt;;0.05 in all 263 accessions from the final SNP data set (4,611,842). A neighbor-joining tree was constructed using PHYLIP (Felsentein 1989) (Version:3.696) on the basis of the distance matrix with 1,000 bootstrap replicates. The software FigTree (
      <a href="http://tree.bio.ed.ac.uk/software/figtree/">http://tree.bio.ed.ac.uk/software/figtree/</a>; version: 1.4.2) was used to visualize the neighbor-joining tree. The principal component analysis (PCA) was performed based on the same SNPs data set (2,468,307 SNPs with MAF &gt;; 0.05) using the smartpca program in the EIGENSFOT
      <sup>70</sup> software (Version: 6.0.1) with default settings (
      <a id="xref-ref-46-1" class="xref-bibr" href="#ref-46">Price et al. 2006</a>). The first three eigenvectors were used to plot the data in two and three dimensions. The population structure was also investigated using the same SNP data set (2,468,307 SNPs with MAF&gt;;0.05) with the FRAPPE (Version: 1.1) software (
      <a id="xref-ref-51-1" class="xref-bibr" href="#ref-51">Tang et al. 2005</a>), which is based on a maximum likelihood method. We ran 10,000 iterations, and the numbers of clusters (
      <em>K</em>) were set from 2 to 8.
     </p>
    </div>
    <div id="sec-15" class="subsection">
     <h3>Identification of select sweeps</h3>
     <p id="p-47">To detect signals of selective sweeps, we selected three distinct genome-wide selection metrics for each group (excluding the TB group), including the reduction of nucleotide diversity (π), Tajima’s 
      <em>D</em>, and genetic differentiation (
      <em>F</em>
      <sub>ST</sub>). We calculated these three selection metrics based on all SNPs (4,611,842) using VCFtools (
      <a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">Danecek et al. 2011</a>) (Version: 0.1.13), with a 10-kb window and a step size of 1 kb. We defined the empirical top 5% of windows or regions as candidate selective outliers for each selection scan metric. The adjacent selective outliers were merged. For each population, selection outliers detected in at least two of the selection scan metrics were designated as the candidate selection regions (CSRs). The TB group consisted of wild relatives (
      <em>P. mira</em>) and three other methods were used to detect selective sweeps: Tajima’s 
      <em>D</em>, RAiSD (Alachiotis et al. 2018), and CLR (Pavlidis et al. 2013). Similarly, the top 5% of windows or regions identified in at least two metrics were designated as candidate selective sweeps.
     </p>
    </div>
    <div id="sec-16" class="subsection">
     <h3>Collection of climate variables</h3>
     <p id="p-48">A total of 51 environmental variables were selected as being essential for peach growth and survival (Supplemental Table S6), representing extremes and seasonality of temperature and precipitation, altitude, latitude, relative air humidity, water vapor pressure, growing season lengths, and aridity. Of these, 39 datasets of climate variables were downloaded from WorldClim (
      <a href="http://www.worldclim.org">http://www.worldclim.org</a>; version: 1.4), with a resolution of 2.5 minutes, and climate variables for each accession were extracted using DIVA-GIS (
      <a href="http://www.diva-gis.org">http://www.diva-gis.org</a>; version: 7.5) (Supplemental Table S6). Six climate variables were downloaded from CDMC (
      <a href="http://data.cma.cn/en/?r=site/index">http://data.cma.cn/en/?r=site/index</a>) and climate variables for each accession were extracted using ArcGIS (
      <a href="http://www.arcgis.com">http://www.arcgis.com</a>; version: 10.3) (Supplemental Table S6). Four climate variables were downloaded from the FAO (
      <a href="http://www.fao.org/geonetwork/srv/en/main.home">http://www.fao.org/geonetwork/srv/en/main.home</a>), with a resolution of 5 minutes or 10 minutes and climate variables for each accession were extracted using ArcGIS (Supplemental Table S7). Altitude and latitude for each accession were recorded using a GPS (Magellangps triton 300E; 
      <a href="http://www.magellangps.com">http://www.magellangps.com</a>) when the accessions were collected.
     </p>
    </div>
    <div id="sec-17" class="subsection">
     <h3>Genome-wide environmental association study (GWEAS)</h3>
     <p id="p-49">GWEAS was performed for 51 climate variables using 4,611,842 high-quality SNPs. The association analyses were performed using the mixed linear model (MLM) with Efficient Mixed-Model Association eXpedited (EMMAX) software (
      <a id="xref-ref-60-1" class="xref-bibr" href="#ref-60">Zhou and Stephens 2012</a>). To minimize the number of false positives and increase statistical power, population structure was corrected using a kinship matrix, which was estimated with EMMAX emmax-kin program (
      <a id="xref-ref-60-2" class="xref-bibr" href="#ref-60">Zhou and Stephens 2012</a>). The genome-wide significance thresholds of the GWEAS were determined using the Bonferroni test. Based on a nominal level of 0.05, the threshold was set as 0.05/total SNPs (log10(
      <em>P</em>) = −7.13).
     </p>
    </div>
    <div id="sec-18" class="subsection">
     <h3>Functional enrichment and pathway analysis</h3>
     <p id="p-50">To test whether candidate genes were overrepresented among lists from known biological processes, gene families and pathways, a functional enrichment and pathway analysis was performed based on Fisher exact tests (
      <em>P</em> &lt;; 0.05), using the Database for Annotation, Visualization and Integrated Discovery (DAVID) (
      <a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">Huang et al. 2009</a>) (Version: 6.7). To obtain the comprehensive functional annotations, a list of annotation categories was selected, including GO terms and KEGG pathway. The annotation analysis was performed for genes that were in selective sweeps and GWEAS associations.
     </p>
    </div>
    <div id="sec-19" class="subsection">
     <h3>Phenotyping and genome-wide association study (GWAS)</h3>
     <p id="p-51">The first bloom date (BD) was measured at the National Peach Germplasm Repository of China (NPGRC) (N34.71°, E113.70°, A.S.L. 74 m), located in Zhengzhou, Henan Province, China. The first bloom date data used span February 25 to April 25 from 1983 to 2011 as this period captured the majority of diversity of BD. A total of 89 accessions, with each represented by two replicates, were used to investigate BD (Supplemental Fig. S9A). The first bloom date was defined as the day when approximately 5% of the flowers have completely opened. The advance in bloom date (ABD) for each accession was estimated using a linear regression analysis, based on the BD from 1983 to 2011. The ABD information for each accession is detailed in Supplemental Fig. S9B. To identify genetic loci associated with ABD, GWAS was performed for ABD based a set of 873,895 SNPs, identified after removing SNPs with low-frequency (MAF &lt;; 0.05) and a high missing rate (missing rate &gt;; 0.2) using the EMMAX program (
      <a id="xref-ref-60-3" class="xref-bibr" href="#ref-60">Zhou and Stephens 2012</a>). To minimize the number of false positives and to increase the statistical power, population structure was corrected using a kinship matrix, which was calculated with EMMAX emmax-kin program (
      <a id="xref-ref-60-4" class="xref-bibr" href="#ref-60">Zhou and Stephens 2012</a>). The genome-wide significance threshold of the GWAS was determined using the Bonferroni test. Based on a nominal level of 0.05, the threshold was set as 0.05/total SNPs (log10(
      <em>P</em>) = −7.08). GWAS was also performed for yearly BD data from 1983 to 2011 based on the same SNP data set, using the same method as above.
     </p>
     <p id="p-52">For CR, phenotyping analyses were performed in 2011 and 2012 as in 
      <a id="xref-ref-13-4" class="xref-bibr" href="#ref-13">Fan et al (2010)</a>. A 0-7.2°C model was chosen to evaluate CR and the number of hours in this range (chilling hours; CHs) was counted, starting when the daily average air temperature dropped to below 7.2°C. Starting at 50 CHs, the branches of each accession were cut every 50 CHs until 1,300 CHs. For each accession, two clones were sampled, and three branches longer than 40 cm with floral buds were taken from each clone. Branch cuttings were placed in water in a greenhouse at 25°C and a 16 h/8 h photoperiod to force floral bud break. The frequency of floral bud break was evaluated after two weeks. The CR of an accession was defined as being sufficient at a specific sampling time if 50% of floral buds on the branch cuttings opened. GWAS for CR was also performed using MLM in EMMAX.
     </p>
     <p id="p-53">Cold hardiness was evaluated using a conductance-based semi-lethal temperature method in December-January of 2013-2014 on 143 accessions. Six annual branches longer than 20 cm were sampled for each accession. A total of six cold treatments were used: −10, −15, −20, −25, −30, and −35°C. Branch cuttings were incubated in freezer with the six treatments for 24 h. After cold treatments, the cuttings were placed at 0°C for 8 h. Branch cuttings were then cut into 2 mm segments. A total of 2 g of segments was used to measure the conductance, with three biological replicates. The initial conductance (C1) was measured after a 12 h steep in 10 ml water. The final conductance (C2) was measured after boiling the samples for 20 min and leaving them to cool to room temperature for a subsequent 2 h period. The relative conductance (RC) was calculated using following formula: 
      <span class="disp-formula" id="disp-formula-1">
       <span class="highwire-responsive-lazyload">
        <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/01/15/2020.01.15.907709/embed/graphic-7.gif"/>
        <noscript>
         <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/01/15/2020.01.15.907709/embed/graphic-7.gif"/>
        </noscript>
       </span>
      </span>
     </p>
     <p id="p-54">Finally, the semi-lethal temperature (LT50) was calculated using a logistic function based on RC.</p>
    </div>
    <div id="sec-20" class="subsection">
     <h3>Yeast one-hybrid assay</h3>
     <p id="p-55">Yeast one-hybridization assay was performed using the Matchmaker
      <sup>®</sup> Gold Yeast One-Hybrid System (Clontech, Palo Alto, CA, USA). The promoter sequence (upstream 2kb from the start codon) of the sucrose phosphate synthase, 
      <em>PpSPS1</em> (
      <em>Prupe</em>.
      <em>1G483200</em>), was cloned into the pAbAi vector. Similarly, the full-length of ABA-responsive element binding 1, 
      <em>PpAREB1</em> (
      <em>Prupe</em>.
      <em>1G434500</em>), was subcloned into the pGADT7 AD vector. The auto-activation and TF– protein interaction analyses were conducted according to manufacturer’s protocol.
     </p>
    </div>
    <div id="sec-21" class="subsection">
     <h3>Scanning electron microscopy (SEM)</h3>
     <p id="p-56">Stomata were examined by SEM in young leaves from three accessions from the TB group and three accessions from the NP group, representing high-altitude and low-altitude accession, respectively. Three replicates were sampled from each accession. Samples were fixed in 2.5% glutaraldehyde (pH = 7.4) for 24 h at 4
      <strong>°C</strong>. Subsequently, fixed samples were dehydrated using an ethanol series (30% ethanol, 20 min; 50% ethanol, 20 min; 70% ethanol, 20 min; 100% ethanol, 30 min (twice)). The dehydrated samples were then dried in a critical-point drying apparatus (Quorum K850; England). Dried samples were mounted on stubs and sputter-coated with gold (FEI; America) and observed under a scanning electron microscopy (SEM) (FEI Quanta 250; America).
     </p>
    </div>
    <div id="sec-22" class="subsection">
     <h3>RNA-Seq analysis</h3>
     <p id="p-57">For drought stress treatment, four-year-old potted peach seedlings from peach cultivar “Dong Xue Mi Tao” were used. Fruit flesh were taken at six drought stress treatment time points, including 6 hours, 12 hours, 24 hours, 3 days, 6 days, and 12 days. For expression profiles in different tissues, roots, leaves, fruit, flowers, phloem, and seeds were sampled from “Aba Guang He Tao” (high-altitude) and “B-4” (low-altitude). For the expression of 
      <em>PpCCD4</em>, fruit fleshes were sampled from “Zao Huang Pan Tao” (yellow-fleshed) and “Zhong Tao Hong Yu” (white-fleshed) at 10, 50, and 90 days post bloom date (dpb). For the expression of 
      <em>PpSPS1</em>, fruit fleshes were sampled from “Chinese cling” at 20, 40, 60, 80, 100, 120 dpb. Three biological replicates were collected for each sample. The tissues were immediately frozen in liquid nitrogen and then ground to fine powder. Total RNA was extracted using a quick extraction kit (Aidlab, Beijing, China). First and second strand complementary DNA (cDNA) was synthesized using a cDNA Synthesis System kit (TOYOBO, Osaka, Japan), following the manufacturer’s protocol. Double-strand cDNAs were then purified and adapters were ligated to the short fragments. The constructed RNA-Seq libraries were sequenced using the Illumina HiSeq 2000 platform (Illumina, San Diego, USA) in paired-end 150-bp mode. Low-quality reads were filtered from the raw reads using Trimmomatic (
      <a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">Bolger et al. 2014</a>). Data analysis followed the protocol proposed by 
      <a id="xref-ref-44-1" class="xref-bibr" href="#ref-44">Pertea et al (2016)</a>. Cleaned reads were mapped to the peach reference genome using Hisat2 (Version 2.0.5) (
      <a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">Kim et al. 2015</a>) with default parameters. Transcript abundances were calculated and transcript assembly was performed using Stringtie (
      <a id="xref-ref-45-1" class="xref-bibr" href="#ref-45">Pertea et al. 2015</a>). DEG analysis was carried out using the R package ballgown (
      <a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">Frazee et al. 2015</a>).
     </p>
    </div>
    <div id="sec-23" class="subsection">
     <h3>Over-expression of candidate genes in 
      <em>A. thaliana</em>
     </h3>
     <p id="p-58">The full-length open reading frames of three peach genes, 
      <em>PpEPF1</em> (
      <em>Prupe</em>.
      <em>3G235800</em>
      <strong>)</strong>, 
      <em>PpSVP</em> (
      <em>Prupe</em>.
      <em>8G069300</em>), and 
      <em>PpLNK1</em> (
      <em>Prupe</em>.
      <em>8G062200</em>
      <strong>)</strong>, were amplified by PCR using cDNAs derived from young leaves of “Aba Guang He Tao”, ‘Nanshan Tian Tao’ (CR=200h), and “Nanshan Tian Tao”, respectively. The PCR products were cloned into the pBI121 vector driven by the cauliflower mosaic virus (CaMV) 35S promoter at Sangon Biotech (Sangon, Shanghai, China). The resulting constructs were then transformed into 
      <em>A. thaliana</em> Columbia type using 
      <em>Agrobacterium tumefaciens</em> GV3101 and positive transformants selected with kanamycin. Ten transgenic lines for each gene were used to evaluate the flowering time. The stomata size and density were observed under a light microscope (Olympus BX51, Tokyo, Japan) with a 400 × objective lens.
     </p>
    </div>
    <div id="sec-24" class="subsection">
     <h3>RNA extraction and expression analysis using qRT-PCR</h3>
     <p id="p-59">For 
      <em>PpSVP</em> expression analysis, floral buds from ‘Nanshan Tian Tao’ were sampled on October 15, November 15, December 15, January 15, February 15, March 15 in 2016-2017. 
      <em>PpLNK1</em> expression was measured in floral buds (blooming soon) from three global warming-sensitive accessions (‘Wu Yue Xian’, ‘Nanshan Tian Tao’, and ‘Li He Pan Tao’) and three global warming-insensitive accessions (‘Xinjiang Pan Tao’, ‘Wuhan 2’, and ‘Kashi 2’) at 2016 and 2017. For 
      <em>PpAHP5</em>, the phloem (including cambium) was collected from two cultivars ‘Hunchun Tao’ (cold resistant) and ‘Nanshan Tian Tao’ (cold sensitive) after 24 hours treatment under −28 °C refrigerator and following 21 °C incubation in water. For each sample, three biological replicates were used. Total RNA was extracted using an extraction kit (Aidlab, Beijing, China) and first-strand cDNA was synthesized with 1µg RNA using a FastQuant RT Kit (with gDNase) (TIANGEN, Beijing, China). Gene-specific primers were designed using Primer-BLAST software (National Center for Biotechnology Information, Maryland, USA). qRT-PCR was performed using a SYBR green I master kit (Roche Diagnostics, Indianapolis, USA) with the LightCycler System (Roche LightCycler 480, Indianapolis, USA), following the manufacturer’s protocol. Relative expression levels were calculated using the 2
      <sup>-ΔΔCT</sup> method. A β-actin was used as the reference gene.
     </p>
    </div>
    <div id="sec-25" class="subsection">
     <h3>Global warming simulation experiment</h3>
     <p id="p-60">The global warming simulation experiment was performed in 2016-2017. Three peach cultivars (Nanshan Tian Tao, Hu Jing Mi Lu, and Maravila), each with two clones, were used as plant materials. For each cultivar, ∼30 annual branches longer than 40 cm with floral buds were taken from each clone when the winter chill accumulation was ∼900 chilling hours (0∼7.2°C, excluding 0°C). Branch cuttings were placed in water in greenhouse at 25°C and with a 16 h/8 h photoperiod, to simulate climate warming. The ratio of bud break was investigated daily, starting from the day that the branch cuttings were placed in the greenhouse. The floral buds, excluding the tegmentum, were collected weekly and frozen in liquid nitrogen. The sampled floral buds were used for qRT-PCR analyses following the protocol described above.</p>
    </div>
    <div id="sec-26" class="subsection">
     <h3>Data access</h3>
     <p id="p-61">Raw sequence data have been deposited in the NCBI Short Read Archive (SRA) under accession SRP108113. SNPs and SVs in Variant Call Format (VCF) have been deposited into the Figshare database (SNPs: 
      <a href="https://figshare.com/articles/SNPs_for_263_peach_accessions/7636715">https://figshare.com/articles/SNPs_for_263_peach_accessions/7636715</a>, SVs: 
      <a href="https://figshare.com/articles/SVs_for_peach_sequencing/7636721">https://figshare.com/articles/SVs_for_peach_sequencing/7636721</a>). All other relevant data are contained within the paper and available in supplementary files.
     </p>
    </div>
   </div>
   <div class="section" id="sec-27">
    <h2 class="">Author contributions</h2>
    <p id="p-63">L.W., S.H., Z.F. and W.G. designed and managed the project; Y.L., G.Z., X.Z., S.Z. and C.C. collected materials; Y.L., P.Z., J.G., X.W., and Q.Z. prepared and purified DNA samples; Y.L., K.C., and N.L. performed the data analyses; Y.L., T.D., J.W., L.G., Q.H., and W.F. performed phenotyping. Y.L. performed the molecular experiment. Y.L. and K. C. wrote the paper; L.W., Z.F., W.G., and S.H. revised the paper. All authors read and approved the final manuscript.</p>
   </div>
   <div class="section ack" id="ack-1">
    <h2 class="">Acknowledgements</h2>
    <p id="p-62">This work was supported by grants from the Agricultural Science and Technology Innovation Program (CAAS-ASTIP-2020-ZFRI-01), the National Natural Science Foundation of China (31572094), the Crop Germplasm Resources Conservation Project (2016NWB041), and the US National Science Foundation (IOS-1339287 and IOS-1539831). We thank Prof. Jialong Yao from The Plant and Food Research Institute of New Zealand and Dr. Amandine Cornille from Université Paris-Sud for helpful suggestions in paper writing. We thank Dr. Yanling Wen from Beijing Institute of Genomics, Chinese Academy of Sciences for assistance in data visualization.</p>
   </div>
   <div class="section ref-list" id="ref-list-1">
    <h2 class="">References</h2>
    <ol class="cit-list ref-use-labels">
     <li>
      <span class="ref-label ref-label-empty"/>
      <div class="cit ref-cit ref-journal no-rev-xref" id="cit-2020.01.15.907709v1.1">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Alachiotis</span>
          <span class="cit-name-given-names">N</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Pavlidis</span>
          <span class="cit-name-given-names">P.</span>
         </span>
         <span class="cit-pub-date">2018</span>. 
         <span class="cit-article-title">RAiSD detects positive selection based on multiple signatures of a selective sweep and SNP vectors</span>. 
         <abbr class="cit-jnl-abbrev">Commun Biol</abbr>
         <span class="cit-vol">1</span>: 
         <span class="cit-fpage">79</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DCommun%2BBiol%26rft.volume%253D1%26rft.spage%253D79%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label ref-label-empty"/>
      <a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference in text" id="ref-2">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.01.15.907709v1.2" data-doi="10.1111/ele.12150">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Blanquart</span>
          <span class="cit-name-given-names">F</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Kaltz</span>
          <span class="cit-name-given-names">O</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Nuismer</span>
          <span class="cit-name-given-names">SL</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Gandon</span>
          <span class="cit-name-given-names">S.</span>
         </span>
         <span class="cit-pub-date">2013</span>. 
         <span class="cit-article-title">A practical guide to measuring local adaptation</span>. 
         <abbr class="cit-jnl-abbrev">Ecol Lett</abbr>
         <span class="cit-vol">16</span>: 
         <span class="cit-fpage">1195</span>–
         <span class="cit-lpage">1205</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DEcol%2BLett%26rft.volume%253D16%26rft.spage%253D1195%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fele.12150%26rft_id%253Dinfo%253Apmid%252F23848550%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1111/ele.12150&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
        <a href="/lookup/external-ref?access_num=23848550&amp;;link_type=MED&amp;;atom=%2Fbiorxiv%2Fearly%2F2020%2F01%2F15%2F2020.01.15.907709.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
         <span>PubMed</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label ref-label-empty"/>
      <a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference in text" id="ref-3">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.01.15.907709v1.3" data-doi="10.1038/ng.3046">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Bolger</span>
          <span class="cit-name-given-names">A</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Scossa</span>
          <span class="cit-name-given-names">F</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Bolger</span>
          <span class="cit-name-given-names">ME</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Lanz</span>
          <span class="cit-name-given-names">C</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Maumus</span>
          <span class="cit-name-given-names">F</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Tohge</span>
          <span class="cit-name-given-names">T</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Quesneville</span>
          <span class="cit-name-given-names">H</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Alseekh</span>
          <span class="cit-name-given-names">S</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Sørensen</span>
          <span class="cit-name-given-names">I</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Lichtenstein</span>
          <span class="cit-name-given-names">G</span>
         </span>, 
         <span class="cit-etal">et al.</span>
         <span class="cit-pub-date">2014</span>. 
         <span class="cit-article-title">The genome of the stress-tolerant wild tomato species Solanum pennellii</span>. 
         <abbr class="cit-jnl-abbrev">Nat Genet</abbr>
         <span class="cit-vol">46</span>: 
         <span class="cit-fpage">1034</span>–
         <span class="cit-lpage">1038</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DNat%2BGenet%26rft.volume%253D46%26rft.spage%253D1034%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fng.3046%26rft_id%253Dinfo%253Apmid%252F25064008%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1038/ng.3046&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
        <a href="/lookup/external-ref?access_num=25064008&amp;;link_type=MED&amp;;atom=%2Fbiorxiv%2Fearly%2F2020%2F01%2F15%2F2020.01.15.907709.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
         <span>PubMed</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label ref-label-empty"/>
      <div class="cit ref-cit ref-journal no-rev-xref" id="cit-2020.01.15.907709v1.4" data-doi="10.1093/bioinformatics/btu170">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Bolger</span>
          <span class="cit-name-given-names">AM</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Lohse</span>
          <span class="cit-name-given-names">M</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Usadel</span>
          <span class="cit-name-given-names">B.</span>
         </span>
         <span class="cit-pub-date">2014</span>. 
         <span class="cit-article-title">Trimmomatic: a flexible trimmer for Illumina sequence data</span>. 
         <abbr class="cit-jnl-abbrev">Bioinformatics</abbr>
         <span class="cit-vol">30</span>: 
         <span class="cit-fpage">2114</span>–
         <span class="cit-lpage">2120</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtu170%26rft_id%253Dinfo%253Apmid%252F24695404%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1093/bioinformatics/btu170&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
        <a href="/lookup/external-ref?access_num=24695404&amp;;link_type=MED&amp;;atom=%2Fbiorxiv%2Fearly%2F2020%2F01%2F15%2F2020.01.15.907709.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
         <span>PubMed</span>
        </a>
        <a href="/lookup/external-ref?access_num=000340049100004&amp;;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience">
         <span>Web of Science</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label ref-label-empty"/>
      <a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference in text" id="ref-5">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.01.15.907709v1.5">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Cao</span>
          <span class="cit-name-given-names">K</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Zheng</span>
          <span class="cit-name-given-names">Z</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Wang</span>
          <span class="cit-name-given-names">L</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Liu</span>
          <span class="cit-name-given-names">X</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Zhu</span>
          <span class="cit-name-given-names">G</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Fang</span>
          <span class="cit-name-given-names">W</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Cheng</span>
          <span class="cit-name-given-names">S</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Zeng</span>
          <span class="cit-name-given-names">P</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Chen</span>
          <span class="cit-name-given-names">C</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Wang</span>
          <span class="cit-name-given-names">X</span>
         </span>, 
         <span class="cit-etal">et al.</span>
         <span class="cit-pub-date">2014</span>. 
         <span class="cit-article-title">Comparative population genomics reveals the domestication history of the peach, 
          <em>Prunus persica</em>, and human influences on perennial fruit crops
         </span>. 
         <abbr class="cit-jnl-abbrev">Genome Biol</abbr>
         <span class="cit-vol">15</span>: 
         <span class="cit-fpage">415</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DGenome%2BBiol%26rft.volume%253D15%26rft.spage%253D415%26rft_id%253Dinfo%253Apmid%252F25079967%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=25079967&amp;;link_type=MED&amp;;atom=%2Fbiorxiv%2Fearly%2F2020%2F01%2F15%2F2020.01.15.907709.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
         <span>PubMed</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label ref-label-empty"/>
      <a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference in text" id="ref-6">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.01.15.907709v1.6">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Cao</span>
          <span class="cit-name-given-names">K</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Zhou</span>
          <span class="cit-name-given-names">Z</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Wang</span>
          <span class="cit-name-given-names">Q</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Guo</span>
          <span class="cit-name-given-names">J</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Zhao</span>
          <span class="cit-name-given-names">P</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Zhu</span>
          <span class="cit-name-given-names">G</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Fang</span>
          <span class="cit-name-given-names">W</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Chen</span>
          <span class="cit-name-given-names">C</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Wang</span>
          <span class="cit-name-given-names">X</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Wang</span>
          <span class="cit-name-given-names">X</span>
         </span>, 
         <span class="cit-etal">et al.</span>
         <span class="cit-pub-date">2016</span>. 
         <span class="cit-article-title">Genome-wide association study of 12 agronomic traits in peach</span>. 
         <abbr class="cit-jnl-abbrev">Nat Commu</abbr>
         <span class="cit-vol">7</span>: 
         <span class="cit-fpage">13246</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DNat%2BCommu%26rft.volume%253D7%26rft.spage%253D13246%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label ref-label-empty"/>
      <a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference in text" id="ref-7">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.01.15.907709v1.7" data-doi="10.1093/bioinformatics/btv710">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Chen</span>
          <span class="cit-name-given-names">X</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Schulz-Trieglaff</span>
          <span class="cit-name-given-names">O</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Shaw</span>
          <span class="cit-name-given-names">R</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Barnes</span>
          <span class="cit-name-given-names">B</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Schlesinger</span>
          <span class="cit-name-given-names">F</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Källberg</span>
          <span class="cit-name-given-names">M</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Cox</span>
          <span class="cit-name-given-names">AJ</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Kruglyak</span>
          <span class="cit-name-given-names">S</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Saunders</span>
          <span class="cit-name-given-names">CT</span>
         </span>. 
         <span class="cit-pub-date">2016</span>. 
         <span class="cit-article-title">Manta: rapid detection of structural variants and indels for germline and cancer sequencing applications</span>. 
         <abbr class="cit-jnl-abbrev">Bioinformatics</abbr>
         <span class="cit-vol">32</span>: 
         <span class="cit-fpage">1220</span>–
         <span class="cit-lpage">1222</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtv710%26rft_id%253Dinfo%253Apmid%252F26647377%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1093/bioinformatics/btv710&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
        <a href="/lookup/external-ref?access_num=26647377&amp;;link_type=MED&amp;;atom=%2Fbiorxiv%2Fearly%2F2020%2F01%2F15%2F2020.01.15.907709.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
         <span>PubMed</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label ref-label-empty"/>
      <a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference in text" id="ref-8">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.01.15.907709v1.8" data-doi="10.1038/nmeth.3505">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Chiang</span>
          <span class="cit-name-given-names">C</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Layer</span>
          <span class="cit-name-given-names">RM</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Faust</span>
          <span class="cit-name-given-names">GG</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Lindberg</span>
          <span class="cit-name-given-names">MR</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Rose</span>
          <span class="cit-name-given-names">DB</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Garrison</span>
          <span class="cit-name-given-names">EP</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Marth</span>
          <span class="cit-name-given-names">GT</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Quinlan</span>
          <span class="cit-name-given-names">AR</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Hall</span>
          <span class="cit-name-given-names">IM</span>
         </span>. 
         <span class="cit-pub-date">2015</span>. 
         <span class="cit-article-title">SpeedSeq: ultra-fast personal genome analysis and interpretation</span>. 
         <abbr class="cit-jnl-abbrev">Nat Methods</abbr>
         <span class="cit-vol">12</span>: 
         <span class="cit-fpage">966</span>–
         <span class="cit-lpage">968</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DNat%2BMethods%26rft.volume%253D12%26rft.spage%253D966%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnmeth.3505%26rft_id%253Dinfo%253Apmid%252F26258291%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1038/nmeth.3505&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
        <a href="/lookup/external-ref?access_num=26258291&amp;;link_type=MED&amp;;atom=%2Fbiorxiv%2Fearly%2F2020%2F01%2F15%2F2020.01.15.907709.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
         <span>PubMed</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label ref-label-empty"/>
      <a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference in text" id="ref-9">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.01.15.907709v1.9" data-doi="10.4161/fly.19695">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Cingolani</span>
          <span class="cit-name-given-names">P</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Platts</span>
          <span class="cit-name-given-names">A</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Wang</span>
          <span class="cit-name-given-names">L</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Coon</span>
          <span class="cit-name-given-names">M</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Nguyen</span>
          <span class="cit-name-given-names">T</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Wang</span>
          <span class="cit-name-given-names">L</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Land</span>
          <span class="cit-name-given-names">SJ</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Lu</span>
          <span class="cit-name-given-names">X</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Ruden</span>
          <span class="cit-name-given-names">DM</span>
         </span>. 
         <span class="cit-pub-date">2012</span>. 
         <span class="cit-article-title">A program for annotating and predicting the effects of single nucleotide polymorphisms, SnpEff</span>. 
         <abbr class="cit-jnl-abbrev">Fly</abbr>
         <span class="cit-vol">6</span>: 
         <span class="cit-fpage">80</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DFly%26rft.volume%253D6%26rft.spage%253D80%26rft_id%253Dinfo%253Adoi%252F10.4161%252Ffly.19695%26rft_id%253Dinfo%253Apmid%252F22728672%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.4161/fly.19695&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
        <a href="/lookup/external-ref?access_num=22728672&amp;;link_type=MED&amp;;atom=%2Fbiorxiv%2Fearly%2F2020%2F01%2F15%2F2020.01.15.907709.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
         <span>PubMed</span>
        </a>
        <a href="/lookup/external-ref?access_num=000305965500003&amp;;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience">
         <span>Web of Science</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label ref-label-empty"/>
      <a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference in text" id="ref-10">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.01.15.907709v1.10" data-doi="10.1093/bioinformatics/btr330">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Danecek</span>
          <span class="cit-name-given-names">P</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Auton</span>
          <span class="cit-name-given-names">A</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Abecasis</span>
          <span class="cit-name-given-names">G</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Albers</span>
          <span class="cit-name-given-names">CA</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Banks</span>
          <span class="cit-name-given-names">E</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">DePristo</span>
          <span class="cit-name-given-names">MA</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Handsaker</span>
          <span class="cit-name-given-names">RE</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Lunter</span>
          <span class="cit-name-given-names">G</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Marth</span>
          <span class="cit-name-given-names">GT</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Sherry</span>
          <span class="cit-name-given-names">ST</span>
         </span>. 
         <span class="cit-pub-date">2011</span>. 
         <span class="cit-article-title">The variant call format and VCFtools</span>. 
         <abbr class="cit-jnl-abbrev">Bioinformatics</abbr>
         <span class="cit-vol">27</span>: 
         <span class="cit-fpage">2156</span>–
         <span class="cit-lpage">2158</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtr330%26rft_id%253Dinfo%253Apmid%252F21653522%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1093/bioinformatics/btr330&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
        <a href="/lookup/external-ref?access_num=21653522&amp;;link_type=MED&amp;;atom=%2Fbiorxiv%2Fearly%2F2020%2F01%2F15%2F2020.01.15.907709.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
         <span>PubMed</span>
        </a>
        <a href="/lookup/external-ref?access_num=000292778700023&amp;;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience">
         <span>Web of Science</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label ref-label-empty"/>
      <a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference in text" id="ref-11">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.01.15.907709v1.11" data-doi="10.1038/ng.806">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Depristo</span>
          <span class="cit-name-given-names">MA</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Banks</span>
          <span class="cit-name-given-names">E</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Poplin</span>
          <span class="cit-name-given-names">R</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Garimella</span>
          <span class="cit-name-given-names">KV</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Maguire</span>
          <span class="cit-name-given-names">JR</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Hartl</span>
          <span class="cit-name-given-names">C</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Philippakis</span>
          <span class="cit-name-given-names">AA</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">del Angel</span>
          <span class="cit-name-given-names">G</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Rivas</span>
          <span class="cit-name-given-names">MA</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Hanna</span>
          <span class="cit-name-given-names">M</span>
         </span>, 
         <span class="cit-etal">et al.</span>
         <span class="cit-pub-date">2011</span>. 
         <span class="cit-article-title">A framework for variation discovery and genotyping using next-generation DNA sequencing data</span>. 
         <abbr class="cit-jnl-abbrev">Nat Genet</abbr>
         <span class="cit-vol">43</span>: 
         <span class="cit-fpage">491</span>–
         <span class="cit-lpage">498</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DNature%2Bgenetics%26rft.stitle%253DNat%2BGenet%26rft.aulast%253DDepristo%26rft.auinit1%253DM.%2BA.%26rft.volume%253D43%26rft.issue%253D5%26rft.spage%253D491%26rft.epage%253D498%26rft.atitle%253DA%2Bframework%2Bfor%2Bvariation%2Bdiscovery%2Band%2Bgenotyping%2Busing%2Bnext-generation%2BDNA%2Bsequencing%2Bdata.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fng.806%26rft_id%253Dinfo%253Apmid%252F21478889%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1038/ng.806&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
        <a href="/lookup/external-ref?access_num=21478889&amp;;link_type=MED&amp;;atom=%2Fbiorxiv%2Fearly%2F2020%2F01%2F15%2F2020.01.15.907709.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
         <span>PubMed</span>
        </a>
        <a href="/lookup/external-ref?access_num=000289972600023&amp;;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience">
         <span>Web of Science</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label ref-label-empty"/>
      <a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference in text" id="ref-12">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.01.15.907709v1.12">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Falchi</span>
          <span class="cit-name-given-names">R</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Vendramin</span>
          <span class="cit-name-given-names">E</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Zanon</span>
          <span class="cit-name-given-names">L</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Scalabrin</span>
          <span class="cit-name-given-names">S</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Cipriani</span>
          <span class="cit-name-given-names">G</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Verde</span>
          <span class="cit-name-given-names">I</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Vizzotto</span>
          <span class="cit-name-given-names">G</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Morgante</span>
          <span class="cit-name-given-names">M.</span>
         </span>
         <span class="cit-pub-date">2013</span>. 
         <span class="cit-article-title">Three distinct mutational mechanisms acting on a single gene underpin the origin of yellow flesh in peach</span>. 
         <abbr class="cit-jnl-abbrev">Plant J</abbr>
         <span class="cit-vol">76</span>: 
         <span class="cit-fpage">75</span>–
         <span class="cit-lpage">87</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DPlant%2BJ%26rft.volume%253D76%26rft.spage%253D75%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label ref-label-empty"/>
      <a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference in text" id="ref-13">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.01.15.907709v1.13" data-doi="10.1111/j.1469-8137.2009.03119.x">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Fan</span>
          <span class="cit-name-given-names">S</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Bielenberg</span>
          <span class="cit-name-given-names">DG</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Zhebentyayeva</span>
          <span class="cit-name-given-names">TN</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Reighard</span>
          <span class="cit-name-given-names">GL</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Okie</span>
          <span class="cit-name-given-names">WR</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Holland</span>
          <span class="cit-name-given-names">D</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Abbott</span>
          <span class="cit-name-given-names">AG</span>
         </span>. 
         <span class="cit-pub-date">2010</span>. 
         <span class="cit-article-title">Mapping quantitative trait loci associated with chilling requirement, heat requirement and bloom date in peach (
          <em>Prunus persica</em>)
         </span>. 
         <abbr class="cit-jnl-abbrev">New Phytol</abbr>
         <span class="cit-vol">185</span>: 
         <span class="cit-fpage">917</span>–
         <span class="cit-lpage">930</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DNew%2BPhytologist%26rft.stitle%253DNew%2BPhytologist%26rft.aulast%253DFan%26rft.auinit1%253DS.%26rft.volume%253D185%26rft.issue%253D4%26rft.spage%253D917%26rft.epage%253D930%26rft.atitle%253DMapping%2Bquantitative%2Btrait%2Bloci%2Bassociated%2Bwith%2Bchilling%2Brequirement%252C%2Bheat%2Brequirement%2Band%2Bbloom%2Bdate%2Bin%2Bpeach%2B%2528Prunus%2Bpersica%2529.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1469-8137.2009.03119.x%26rft_id%253Dinfo%253Apmid%252F20028471%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1111/j.1469-8137.2009.03119.x&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
        <a href="/lookup/external-ref?access_num=20028471&amp;;link_type=MED&amp;;atom=%2Fbiorxiv%2Fearly%2F2020%2F01%2F15%2F2020.01.15.907709.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
         <span>PubMed</span>
        </a>
        <a href="/lookup/external-ref?access_num=000274412400012&amp;;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience">
         <span>Web of Science</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label ref-label-empty"/>
      <div class="cit ref-cit ref-journal no-rev-xref" id="cit-2020.01.15.907709v1.14" data-doi="10.1111/j.l096-0031.1989.tb00562.x">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Felsenstein</span>
          <span class="cit-name-given-names">J.</span>
         </span>
         <span class="cit-pub-date">1989</span>. 
         <span class="cit-article-title">PHYLIP-phylogeny inference package (version 3.2)</span>. 
         <abbr class="cit-jnl-abbrev">Cladistics</abbr>
         <span class="cit-vol">5</span>: 
         <span class="cit-fpage">164</span>–
         <span class="cit-lpage">166</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DCladistics%26rft.volume%253D5%26rft.spage%253D164%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.l096-0031.1989.tb00562.x%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1111/j.l096-0031.1989.tb00562.x&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label ref-label-empty"/>
      <a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference in text" id="ref-15">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.01.15.907709v1.15" data-doi="10.1038/11553">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Ferrer</span>
          <span class="cit-name-given-names">JL</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Jez</span>
          <span class="cit-name-given-names">JM</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Bowman</span>
          <span class="cit-name-given-names">ME</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Dixon</span>
          <span class="cit-name-given-names">RA</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Noel</span>
          <span class="cit-name-given-names">JP</span>
         </span>. 
         <span class="cit-pub-date">1999</span>. 
         <span class="cit-article-title">Structure of chalcone synthase and the molecular basis of plant polyketide pathway</span>. 
         <abbr class="cit-jnl-abbrev">Nat Struct Biol</abbr>
         <span class="cit-vol">6</span>: 
         <span class="cit-fpage">775</span>–
         <span class="cit-lpage">784</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DNature%2Bstructural%2Bbiology%26rft.stitle%253DNat%2BStruct%2BBiol%26rft.aulast%253DFerrer%26rft.auinit1%253DJ.%2BL.%26rft.volume%253D6%26rft.issue%253D8%26rft.spage%253D775%26rft.epage%253D784%26rft.atitle%253DStructure%2Bof%2Bchalcone%2Bsynthase%2Band%2Bthe%2Bmolecular%2Bbasis%2Bof%2Bplant%2Bpolyketide%2Bbiosynthesis.%26rft_id%253Dinfo%253Adoi%252F10.1038%252F11553%26rft_id%253Dinfo%253Apmid%252F10426957%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1038/11553&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
        <a href="/lookup/external-ref?access_num=10426957&amp;;link_type=MED&amp;;atom=%2Fbiorxiv%2Fearly%2F2020%2F01%2F15%2F2020.01.15.907709.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
         <span>PubMed</span>
        </a>
        <a href="/lookup/external-ref?access_num=000081683800019&amp;;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience">
         <span>Web of Science</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label ref-label-empty"/>
      <a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference in text" id="ref-16">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.01.15.907709v1.16" data-doi="10.1126/science.1209244">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Fournier-Level</span>
          <span class="cit-name-given-names">A</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Korte</span>
          <span class="cit-name-given-names">A</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Cooper</span>
          <span class="cit-name-given-names">MD</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Nordborg</span>
          <span class="cit-name-given-names">M</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Schmitt</span>
          <span class="cit-name-given-names">J</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Wilczek</span>
          <span class="cit-name-given-names">AM</span>
         </span>. 
         <span class="cit-pub-date">2011</span>. 
         <span class="cit-article-title">A map of local adaptation in 
          <em>Arabidopsis thaliana</em>
         </span>. 
         <abbr class="cit-jnl-abbrev">Science</abbr>
         <span class="cit-vol">334</span>: 
         <span class="cit-fpage">83</span>–
         <span class="cit-lpage">86</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DHancock%26rft.auinit1%253DA.%2BM.%26rft.volume%253D334%26rft.issue%253D6052%26rft.spage%253D83%26rft.epage%253D86%26rft.atitle%253DAdaptation%2Bto%2BClimate%2BAcross%2Bthe%2BArabidopsis%2Bthaliana%2BGenome%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1209244%26rft_id%253Dinfo%253Apmid%252F21980108%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjExOiIzMzQvNjA1Mi84MyI7czo0OiJhdG9tIjtzOjQ4OiIvYmlvcnhpdi9lYXJseS8yMDIwLzAxLzE1LzIwMjAuMDEuMTUuOTA3NzA5LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink">
         <span>
          <span class="cit-reflinks-abstract">Abstract</span>
          <span class="cit-sep cit-reflinks-variant-name-sep">/</span>
          <span class="cit-reflinks-full-text">
           <span class="free-full-text">FREE </span>Full Text
          </span>
         </span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label ref-label-empty"/>
      <a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference in text" id="ref-17">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.01.15.907709v1.17" data-doi="10.1038/nbt.3172">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Frazee</span>
          <span class="cit-name-given-names">AC</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Pertea</span>
          <span class="cit-name-given-names">G</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Jaffe</span>
          <span class="cit-name-given-names">AE</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Langmead</span>
          <span class="cit-name-given-names">B</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Salzberg</span>
          <span class="cit-name-given-names">SL</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Leek</span>
          <span class="cit-name-given-names">JT</span>
         </span>. 
         <span class="cit-pub-date">2015</span>. 
         <span class="cit-article-title">Ballgown bridges the gap between transcriptome assembly and expression analysis</span>. 
         <abbr class="cit-jnl-abbrev">Nat Biotech</abbr>
         <span class="cit-vol">33</span>: 
         <span class="cit-fpage">243</span>–
         <span class="cit-lpage">246</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DNat%2BBiotech%26rft.volume%253D33%26rft.spage%253D243%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnbt.3172%26rft_id%253Dinfo%253Apmid%252F25748911%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1038/nbt.3172&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
        <a href="/lookup/external-ref?access_num=25748911&amp;;link_type=MED&amp;;atom=%2Fbiorxiv%2Fearly%2F2020%2F01%2F15%2F2020.01.15.907709.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
         <span>PubMed</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label ref-label-empty"/>
      <a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference in text" id="ref-18">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.01.15.907709v1.18" data-doi="10.1126/science.1209244">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Hancock</span>
          <span class="cit-name-given-names">AM</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Brachi</span>
          <span class="cit-name-given-names">B</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Faure</span>
          <span class="cit-name-given-names">N</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Horton</span>
          <span class="cit-name-given-names">MW</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Jarymowycz</span>
          <span class="cit-name-given-names">LB</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Sperone</span>
          <span class="cit-name-given-names">FG</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Toomajian</span>
          <span class="cit-name-given-names">C</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Roux</span>
          <span class="cit-name-given-names">F</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Bergelson</span>
          <span class="cit-name-given-names">J.</span>
         </span>
         <span class="cit-pub-date">2011</span>. 
         <span class="cit-article-title">Adaptation to climate across the 
          <em>Arabidopsis thaliana</em> genome
         </span>. 
         <abbr class="cit-jnl-abbrev">Science</abbr>
         <span class="cit-vol">334</span>: 
         <span class="cit-fpage">83</span>–
         <span class="cit-lpage">86</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DHancock%26rft.auinit1%253DA.%2BM.%26rft.volume%253D334%26rft.issue%253D6052%26rft.spage%253D83%26rft.epage%253D86%26rft.atitle%253DAdaptation%2Bto%2BClimate%2BAcross%2Bthe%2BArabidopsis%2Bthaliana%2BGenome%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1209244%26rft_id%253Dinfo%253Apmid%252F21980108%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjExOiIzMzQvNjA1Mi84MyI7czo0OiJhdG9tIjtzOjQ4OiIvYmlvcnhpdi9lYXJseS8yMDIwLzAxLzE1LzIwMjAuMDEuMTUuOTA3NzA5LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink">
         <span>
          <span class="cit-reflinks-abstract">Abstract</span>
          <span class="cit-sep cit-reflinks-variant-name-sep">/</span>
          <span class="cit-reflinks-full-text">
           <span class="free-full-text">FREE </span>Full Text
          </span>
         </span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label ref-label-empty"/>
      <a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference in text" id="ref-19">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.01.15.907709v1.19" data-doi="10.1093/pcp/pcp068">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Hara</span>
          <span class="cit-name-given-names">K</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Yokoo</span>
          <span class="cit-name-given-names">T</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Kajita</span>
          <span class="cit-name-given-names">R</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Onishi</span>
          <span class="cit-name-given-names">T</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Yahata</span>
          <span class="cit-name-given-names">S</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Peterson</span>
          <span class="cit-name-given-names">KM</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Torii</span>
          <span class="cit-name-given-names">KU</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Kakimoto</span>
          <span class="cit-name-given-names">T.</span>
         </span>
         <span class="cit-pub-date">2009</span>. 
         <span class="cit-article-title">Epidermal cell density is autoregulated via a secretory peptide, EPIDERMAL PATTERNING FACTOR 2 in 
          <em>Arabidopsis</em> leaves
         </span>. 
         <abbr class="cit-jnl-abbrev">Plant Cell Physiol</abbr>
         <span class="cit-vol">50</span>: 
         <span class="cit-fpage">1019</span>–
         <span class="cit-lpage">1031</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DPlant%2BCell%2BPhysiol%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fpcp%252Fpcp068%26rft_id%253Dinfo%253Apmid%252F19435754%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1093/pcp/pcp068&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
        <a href="/lookup/external-ref?access_num=19435754&amp;;link_type=MED&amp;;atom=%2Fbiorxiv%2Fearly%2F2020%2F01%2F15%2F2020.01.15.907709.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
         <span>PubMed</span>
        </a>
        <a href="/lookup/external-ref?access_num=000266967600002&amp;;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience">
         <span>Web of Science</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label ref-label-empty"/>
      <a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference in text" id="ref-20">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.01.15.907709v1.20">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Harter</span>
          <span class="cit-name-given-names">RD</span>
         </span>. 
         <span class="cit-pub-date">1983</span>. 
         <span class="cit-article-title">Effect of soil pH on adsorption of lead, copper, zinc, and nickel</span>. 
         <abbr class="cit-jnl-abbrev">Soil Sci Soc Am J</abbr>
         <span class="cit-vol">47</span>: 
         <span class="cit-fpage">47</span>–
         <span class="cit-lpage">51</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DSoil%2BScience%2BSociety%2Bof%2BAmerica%2BJournal%26rft.stitle%253DSoil%2BSci.%26rft.volume%253D47%26rft.issue%253D1%26rft.spage%253D47%26rft.epage%253D51%26rft.atitle%253DEffect%2Bof%2Bsoil%2BpH%2Bon%2Badsorption%2Bof%2Blead%252C%2Bcopper%252C%2Bzinc%252C%2Band%2Bnickel%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=1983048929&amp;;link_type=GEOREF" class="cit-ref-sprinkles cit-ref-sprinkles-georef">
         <span>GeoRef</span>
        </a>
        <a href="/lookup/external-ref?access_num=A1983QF69200009&amp;;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience">
         <span>Web of Science</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label ref-label-empty"/>
      <a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference in text" id="ref-21">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.01.15.907709v1.21" data-doi="10.1105/tpc.113.112631">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Hu</span>
          <span class="cit-name-given-names">Y</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Jiang</span>
          <span class="cit-name-given-names">L</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Wang</span>
          <span class="cit-name-given-names">W</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Yu</span>
          <span class="cit-name-given-names">D.</span>
         </span>
         <span class="cit-pub-date">2013</span>. 
         <span class="cit-article-title">Jasmonate regulates the INDUCER OF CBF EXPRESSION–C-REPEAT BINDING FACTOR/DRE BINDING FACTOR1 Cascade and freezing tolerance in 
          <em>Arabidopsis</em>
         </span>. 
         <abbr class="cit-jnl-abbrev">Plant Cell</abbr>
         <span class="cit-vol">25</span>: 
         <span class="cit-fpage">2907</span>–
         <span class="cit-lpage">2924</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DPlant%2BCell%26rft_id%253Dinfo%253Adoi%252F10.1105%252Ftpc.113.112631%26rft_id%253Dinfo%253Apmid%252F23933884%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6OToicGxhbnRjZWxsIjtzOjU6InJlc2lkIjtzOjk6IjI1LzgvMjkwNyI7czo0OiJhdG9tIjtzOjQ4OiIvYmlvcnhpdi9lYXJseS8yMDIwLzAxLzE1LzIwMjAuMDEuMTUuOTA3NzA5LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink">
         <span>
          <span class="cit-reflinks-abstract">Abstract</span>
          <span class="cit-sep cit-reflinks-variant-name-sep">/</span>
          <span class="cit-reflinks-full-text">
           <span class="free-full-text">FREE </span>Full Text
          </span>
         </span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label ref-label-empty"/>
      <a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference in text" id="ref-22">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.01.15.907709v1.22" data-doi="10.1038/nprot.2008.211">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Huang</span>
          <span class="cit-name-given-names">DW</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Sherman</span>
          <span class="cit-name-given-names">BT</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Lempicki</span>
          <span class="cit-name-given-names">RA</span>
         </span>. 
         <span class="cit-pub-date">2009</span>. 
         <span class="cit-article-title">Systematic and integrative analysis of large gene lists using DAVID Bioinformatics Resources</span>. 
         <abbr class="cit-jnl-abbrev">Nat Protoc</abbr>
         <span class="cit-vol">4</span>: 
         <span class="cit-fpage">44</span>–
         <span class="cit-lpage">57</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.stitle%253DNat%2BProtoc%26rft.aulast%253DHuang%26rft.auinit1%253Dd.%2Ba.%2B%2B.%2BW.%26rft.volume%253D4%26rft.issue%253D1%26rft.spage%253D44%26rft.epage%253D57%26rft.atitle%253DSystematic%2Band%2Bintegrative%2Banalysis%2Bof%2Blarge%2Bgene%2Blists%2Busing%2BDAVID%2Bbioinformatics%2Bresources.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnprot.2008.211%26rft_id%253Dinfo%253Apmid%252F19131956%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1038/nprot.2008.211&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
        <a href="/lookup/external-ref?access_num=19131956&amp;;link_type=MED&amp;;atom=%2Fbiorxiv%2Fearly%2F2020%2F01%2F15%2F2020.01.15.907709.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
         <span>PubMed</span>
        </a>
        <a href="/lookup/external-ref?access_num=000265781800006&amp;;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience">
         <span>Web of Science</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label ref-label-empty"/>
      <a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference in text" id="ref-23">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.01.15.907709v1.23" data-doi="10.1038/ncomms14061">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Jeffares</span>
          <span class="cit-name-given-names">DC</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Jolly</span>
          <span class="cit-name-given-names">C</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Hoti</span>
          <span class="cit-name-given-names">M</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Speed</span>
          <span class="cit-name-given-names">D</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Shaw</span>
          <span class="cit-name-given-names">L</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Rallis</span>
          <span class="cit-name-given-names">C</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Balloux</span>
          <span class="cit-name-given-names">F</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Dessimoz</span>
          <span class="cit-name-given-names">C</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Bähler</span>
          <span class="cit-name-given-names">J</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Sedlazeck</span>
          <span class="cit-name-given-names">FJ</span>
         </span>. 
         <span class="cit-pub-date">2017</span>. 
         <span class="cit-article-title">Transient structural variations have strong effects on quantitative traits and reproductive isolation in fission yeast</span>. 
         <abbr class="cit-jnl-abbrev">Nat Commun</abbr>
         <span class="cit-vol">8</span>: 
         <span class="cit-fpage">14061</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DNat%2BCommun%26rft.volume%253D8%26rft.spage%253D14061%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fncomms14061%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1038/ncomms14061&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label ref-label-empty"/>
      <a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference in text" id="ref-24">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.01.15.907709v1.24" data-doi="10.1104/pp.112.207621">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Jeon</span>
          <span class="cit-name-given-names">J</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Kim</span>
          <span class="cit-name-given-names">J.</span>
         </span>
         <span class="cit-pub-date">2013</span>. 
         <span class="cit-article-title">
          <em>Arabidopsis</em> response regulator1 and Arabidopsis histidine phosphotransfer protein2 (AHP2), AHP3, and AHP5 function in cold signaling
         </span>. 
         <abbr class="cit-jnl-abbrev">Plant Physiol</abbr>
         <span class="cit-vol">161</span>: 
         <span class="cit-fpage">408</span>–
         <span class="cit-lpage">424</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DPlant%2BPhysiol%26rft_id%253Dinfo%253Adoi%252F10.1104%252Fpp.112.207621%26rft_id%253Dinfo%253Apmid%252F23124324%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MTI6InBsYW50cGh5c2lvbCI7czo1OiJyZXNpZCI7czo5OiIxNjEvMS80MDgiO3M6NDoiYXRvbSI7czo0ODoiL2Jpb3J4aXYvZWFybHkvMjAyMC8wMS8xNS8yMDIwLjAxLjE1LjkwNzcwOS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink">
         <span>
          <span class="cit-reflinks-abstract">Abstract</span>
          <span class="cit-sep cit-reflinks-variant-name-sep">/</span>
          <span class="cit-reflinks-full-text">
           <span class="free-full-text">FREE </span>Full Text
          </span>
         </span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label ref-label-empty"/>
      <a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference in text" id="ref-25">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.01.15.907709v1.25" data-doi="10.1038/nmeth.3317">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Kim</span>
          <span class="cit-name-given-names">D</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Langmead</span>
          <span class="cit-name-given-names">B</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Salzberg</span>
          <span class="cit-name-given-names">SL</span>
         </span>. 
         <span class="cit-pub-date">2015</span>. 
         <span class="cit-article-title">HISAT: a fast spliced aligner with low memory requirements</span>. 
         <abbr class="cit-jnl-abbrev">Nat Methods</abbr>
         <span class="cit-vol">12</span>: 
         <span class="cit-fpage">357</span>–
         <span class="cit-lpage">360</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DNat%2BMethods%26rft.volume%253D12%26rft.spage%253D357%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnmeth.3317%26rft_id%253Dinfo%253Apmid%252F25751142%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1038/nmeth.3317&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
        <a href="/lookup/external-ref?access_num=25751142&amp;;link_type=MED&amp;;atom=%2Fbiorxiv%2Fearly%2F2020%2F01%2F15%2F2020.01.15.907709.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
         <span>PubMed</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label ref-label-empty"/>
      <a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference in text" id="ref-26">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.01.15.907709v1.26">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Kobashi</span>
          <span class="cit-name-given-names">K</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Gemma</span>
          <span class="cit-name-given-names">H</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Iwahori</span>
          <span class="cit-name-given-names">S.</span>
         </span>
         <span class="cit-pub-date">2000</span>. 
         <span class="cit-article-title">Abscisic acid content and sugar metabolism of peaches grown under water stress</span>. 
         <abbr class="cit-jnl-abbrev">J Amer Soc Hort Sci</abbr>
         <span class="cit-vol">125</span>: 
         <span class="cit-fpage">425</span>–
         <span class="cit-lpage">428</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DJ%2BAmer%2BSoc%2BHort%2BSci%26rft.volume%253D125%26rft.spage%253D425%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label ref-label-empty"/>
      <a class="rev-xref-ref" href="#xref-ref-27-1" title="View reference in text" id="ref-27">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.01.15.907709v1.27" data-doi="10.1023/A:1014421712254">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Kobashi</span>
          <span class="cit-name-given-names">K</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Sugaya</span>
          <span class="cit-name-given-names">S</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Gemma</span>
          <span class="cit-name-given-names">H</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Iwahori</span>
          <span class="cit-name-given-names">S.</span>
         </span>
         <span class="cit-pub-date">2001</span>. 
         <span class="cit-article-title">Effect of abscisic acid (ABA) on sugar accumulation in the flesh tissue of peach fruit at the start of the maturation stage</span>. 
         <abbr class="cit-jnl-abbrev">Plant Growth Regul</abbr>
         <span class="cit-vol">35</span>: 
         <span class="cit-fpage">215</span>–
         <span class="cit-lpage">223</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DPlant%2BGrowth%2BRegul%26rft.volume%253D35%26rft.spage%253D215%26rft_id%253Dinfo%253Adoi%252F10.1023%252FA%253A1014421712254%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1023/A:1014421712254&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
        <a href="/lookup/external-ref?access_num=000174574700003&amp;;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience">
         <span>Web of Science</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label ref-label-empty"/>
      <a class="rev-xref-ref" href="#xref-ref-28-1" title="View reference in text" id="ref-28">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.01.15.907709v1.28" data-doi="10.1126/sciadv.1400218">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Lasky</span>
          <span class="cit-name-given-names">JR</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Upadhyaya</span>
          <span class="cit-name-given-names">HD</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Ramu</span>
          <span class="cit-name-given-names">P</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Deshpande</span>
          <span class="cit-name-given-names">S</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Hash</span>
          <span class="cit-name-given-names">CT</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Bonnette</span>
          <span class="cit-name-given-names">J</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Juenger</span>
          <span class="cit-name-given-names">TE</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Hyma</span>
          <span class="cit-name-given-names">K</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Acharya</span>
          <span class="cit-name-given-names">C</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Mitchell</span>
          <span class="cit-name-given-names">SE</span>
         </span>. 
         <span class="cit-pub-date">2015</span>. 
         <span class="cit-article-title">Genome-environment associations in sorghum landraces predict adaptive traits</span>. 
         <abbr class="cit-jnl-abbrev">Sci Adv</abbr>
         <span class="cit-vol">1</span>: 
         <span class="cit-fpage">e1400218</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DScience%2BAdvances%26rft.stitle%253DSci%2BAdv%26rft.aulast%253DLasky%26rft.auinit1%253DJ.%2BR.%26rft.volume%253D1%26rft.issue%253D6%26rft.spage%253De1400218%26rft.epage%253De1400218%26rft.atitle%253DGenome-environment%2Bassociations%2Bin%2Bsorghum%2Blandraces%2Bpredict%2Badaptive%2Btraits%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fsciadv.1400218%26rft_id%253Dinfo%253Apmid%252F26601206%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6MzoiUERGIjtzOjExOiJqb3VybmFsQ29kZSI7czo4OiJhZHZhbmNlcyI7czo1OiJyZXNpZCI7czoxMjoiMS82L2UxNDAwMjE4IjtzOjQ6ImF0b20iO3M6NDg6Ii9iaW9yeGl2L2Vhcmx5LzIwMjAvMDEvMTUvMjAyMC4wMS4xNS45MDc3MDkuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink">
         <span>
          <span class="cit-reflinks-full-text">
           <span class="free-full-text">FREE </span>Full Text
          </span>
         </span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label ref-label-empty"/>
      <a class="rev-xref-ref" href="#xref-ref-29-1" title="View reference in text" id="ref-29">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.01.15.907709v1.29" data-doi="10.1093/jxb/ert474">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Lastdrager</span>
          <span class="cit-name-given-names">J</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Hanson</span>
          <span class="cit-name-given-names">J</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Smeekens</span>
          <span class="cit-name-given-names">S.</span>
         </span>
         <span class="cit-pub-date">2014</span>. 
         <span class="cit-article-title">Sugar signals and the control of plant growth and development</span>. 
         <abbr class="cit-jnl-abbrev">J Exp Bot</abbr>
         <span class="cit-vol">65</span>: 
         <span class="cit-fpage">799</span>–
         <span class="cit-lpage">807</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DJ%2BExp%2BBot%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fjxb%252Fert474%26rft_id%253Dinfo%253Apmid%252F24453229%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1093/jxb/ert474&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
        <a href="/lookup/external-ref?access_num=24453229&amp;;link_type=MED&amp;;atom=%2Fbiorxiv%2Fearly%2F2020%2F01%2F15%2F2020.01.15.907709.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
         <span>PubMed</span>
        </a>
        <a href="/lookup/external-ref?access_num=000331815300005&amp;;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience">
         <span>Web of Science</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label ref-label-empty"/>
      <a class="rev-xref-ref" href="#xref-ref-30-1" title="View reference in text" id="ref-30">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.01.15.907709v1.30" data-doi="10.1093/bioinformatics/btp324">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Li</span>
          <span class="cit-name-given-names">H</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Durbin</span>
          <span class="cit-name-given-names">R.</span>
         </span>
         <span class="cit-pub-date">2009</span>. 
         <span class="cit-article-title">Fast and accurate short read alignment with Burrows-Wheeler transform</span>. 
         <abbr class="cit-jnl-abbrev">Bioinformatics</abbr>
         <span class="cit-vol">25</span>: 
         <span class="cit-fpage">1754</span>–
         <span class="cit-lpage">1760</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtp324%26rft_id%253Dinfo%253Apmid%252F19451168%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1093/bioinformatics/btp324&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
        <a href="/lookup/external-ref?access_num=19451168&amp;;link_type=MED&amp;;atom=%2Fbiorxiv%2Fearly%2F2020%2F01%2F15%2F2020.01.15.907709.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
         <span>PubMed</span>
        </a>
        <a href="/lookup/external-ref?access_num=000267665900006&amp;;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience">
         <span>Web of Science</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label ref-label-empty"/>
      <a class="rev-xref-ref" href="#xref-ref-31-1" title="View reference in text" id="ref-31">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.01.15.907709v1.31" data-doi="10.1038/nature10231">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Li</span>
          <span class="cit-name-given-names">H</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Durbin</span>
          <span class="cit-name-given-names">R.</span>
         </span>
         <span class="cit-pub-date">2011</span>. 
         <span class="cit-article-title">Inference of human population history from individual whole-genome sequences</span>. 
         <abbr class="cit-jnl-abbrev">Nature</abbr>
         <span class="cit-vol">475</span>: 
         <span class="cit-fpage">493</span>–
         <span class="cit-lpage">496</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DLi%26rft.auinit1%253DH.%26rft.volume%253D475%26rft.issue%253D7357%26rft.spage%253D493%26rft.epage%253D496%26rft.atitle%253DInference%2Bof%2Bhuman%2Bpopulation%2Bhistory%2Bfrom%2Bindividual%2Bwhole-genome%2Bsequences.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature10231%26rft_id%253Dinfo%253Apmid%252F21753753%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1038/nature10231&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
        <a href="/lookup/external-ref?access_num=21753753&amp;;link_type=MED&amp;;atom=%2Fbiorxiv%2Fearly%2F2020%2F01%2F15%2F2020.01.15.907709.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
         <span>PubMed</span>
        </a>
        <a href="/lookup/external-ref?access_num=000293167900039&amp;;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience">
         <span>Web of Science</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label ref-label-empty"/>
      <a class="rev-xref-ref" href="#xref-ref-32-1" title="View reference in text" id="ref-32">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.01.15.907709v1.32" data-doi="10.1093/bioinformatics/btp352">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Li</span>
          <span class="cit-name-given-names">H</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Handsaker</span>
          <span class="cit-name-given-names">B</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Wysoker</span>
          <span class="cit-name-given-names">A</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Fennell</span>
          <span class="cit-name-given-names">T</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Ruan</span>
          <span class="cit-name-given-names">J</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Homer</span>
          <span class="cit-name-given-names">N</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Marth</span>
          <span class="cit-name-given-names">G</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Abecasis</span>
          <span class="cit-name-given-names">G</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Durbin</span>
          <span class="cit-name-given-names">R</span>
         </span>, 
         <span class="cit-auth cit-collab">1000 Genome Project Data Processing Subgroup</span>. 
         <span class="cit-pub-date">2009</span>. 
         <span class="cit-article-title">The sequence alignment/map format and SAMtools</span>. 
         <abbr class="cit-jnl-abbrev">Bioinformatics</abbr>
         <span class="cit-vol">25</span>: 
         <span class="cit-fpage">2078</span>–
         <span class="cit-lpage">2079</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtp352%26rft_id%253Dinfo%253Apmid%252F19505943%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1093/bioinformatics/btp352&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
        <a href="/lookup/external-ref?access_num=19505943&amp;;link_type=MED&amp;;atom=%2Fbiorxiv%2Fearly%2F2020%2F01%2F15%2F2020.01.15.907709.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
         <span>PubMed</span>
        </a>
        <a href="/lookup/external-ref?access_num=000268808600014&amp;;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience">
         <span>Web of Science</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label ref-label-empty"/>
      <a class="rev-xref-ref" href="#xref-ref-33-1" title="View reference in text" id="ref-33">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.01.15.907709v1.33">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Li</span>
          <span class="cit-name-given-names">J</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Oulee</span>
          <span class="cit-name-given-names">TM</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Raba</span>
          <span class="cit-name-given-names">R</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Amundson</span>
          <span class="cit-name-given-names">RG</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Last</span>
          <span class="cit-name-given-names">RL</span>
         </span>. 
         <span class="cit-pub-date">1993</span>. 
         <span class="cit-article-title">
          <em>Arabidopsis</em> Flavonoid Mutants Are Hypersensitive to UV-B Irradiation
         </span>. 
         <abbr class="cit-jnl-abbrev">Plant Cell</abbr>
         <span class="cit-vol">5</span>: 
         <span class="cit-fpage">71</span>–
         <span class="cit-lpage">179</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DPlant%2BCell%26rft.volume%253D5%26rft.spage%253D71%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label ref-label-empty"/>
      <a class="rev-xref-ref" href="#xref-ref-34-1" title="View reference in text" id="ref-34">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.01.15.907709v1.34">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Li</span>
          <span class="cit-name-given-names">Y</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Cao</span>
          <span class="cit-name-given-names">K</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Zhu</span>
          <span class="cit-name-given-names">G</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Fang</span>
          <span class="cit-name-given-names">W</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Chen</span>
          <span class="cit-name-given-names">C</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Wang</span>
          <span class="cit-name-given-names">X</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Zhao</span>
          <span class="cit-name-given-names">P</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Guo</span>
          <span class="cit-name-given-names">J</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Ding</span>
          <span class="cit-name-given-names">T</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Guan</span>
          <span class="cit-name-given-names">L</span>
         </span>, 
         <span class="cit-etal">et al.</span>
         <span class="cit-pub-date">2019</span>. 
         <span class="cit-article-title">Genomic analyses of an extensive collection of wild and cultivated accessions provide new insights into peach breeding history</span>. 
         <abbr class="cit-jnl-abbrev">Genome Biol</abbr>
         <span class="cit-vol">20</span>(
         <span class="cit-issue">1</span>): 
         <span class="cit-fpage">36</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DGenome%2BBiol%26rft.volume%253D20%26rft.spage%253D36%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label ref-label-empty"/>
      <a class="rev-xref-ref" href="#xref-ref-35-1" title="View reference in text" id="ref-35">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.01.15.907709v1.35">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Li</span>
          <span class="cit-name-given-names">Y</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Wang</span>
          <span class="cit-name-given-names">L</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Zhu</span>
          <span class="cit-name-given-names">G</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Fang</span>
          <span class="cit-name-given-names">W</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Cao</span>
          <span class="cit-name-given-names">K</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Chen</span>
          <span class="cit-name-given-names">C</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Wang</span>
          <span class="cit-name-given-names">X</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Wang</span>, 
          <span class="cit-name-given-names">X.</span>
         </span> (
         <span class="cit-pub-date">2016</span>). 
         <span class="cit-article-title">Phenological response of peach to climate change exhibits a relatively dramatic trend in China, 1983-2012</span>. 
         <abbr class="cit-jnl-abbrev">Sci Hortic-Amsterda</abbr>
         <span class="cit-vol">209</span>:
         <span class="cit-fpage">192</span>–
         <span class="cit-lpage">200</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DSci%2BHortic-Amsterda%26rft.volume%253D209%26rft.spage%253D192%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label ref-label-empty"/>
      <div class="cit ref-cit ref-journal no-rev-xref" id="cit-2020.01.15.907709v1.36" data-doi="10.1093/jxb/erp195">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Li</span>
          <span class="cit-name-given-names">Z</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Reighard</span>
          <span class="cit-name-given-names">GL</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Abbott</span>
          <span class="cit-name-given-names">AG</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Bielenberg</span>
          <span class="cit-name-given-names">DG</span>
         </span>. 
         <span class="cit-pub-date">2009</span>. 
         <span class="cit-article-title">Dormancy-associated MADS genes from the 
          <em>EVG</em> locus of peach [
          <em>Prunus persica</em> (L.) Batsch] have distinct seasonal and photoperiodic expression patterns
         </span>. 
         <abbr class="cit-jnl-abbrev">J Exp Bot</abbr>
         <span class="cit-vol">60</span>: 
         <span class="cit-fpage">3521</span>–
         <span class="cit-lpage">3530</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DJ%2BExp%2BBot%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fjxb%252Ferp195%26rft_id%253Dinfo%253Apmid%252F19553369%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1093/jxb/erp195&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
        <a href="/lookup/external-ref?access_num=19553369&amp;;link_type=MED&amp;;atom=%2Fbiorxiv%2Fearly%2F2020%2F01%2F15%2F2020.01.15.907709.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
         <span>PubMed</span>
        </a>
        <a href="/lookup/external-ref?access_num=000269014600020&amp;;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience">
         <span>Web of Science</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label ref-label-empty"/>
      <a class="rev-xref-ref" href="#xref-ref-37-1" title="View reference in text" id="ref-37">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.01.15.907709v1.37">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Ma</span>
          <span class="cit-name-given-names">Q</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Sun</span>
          <span class="cit-name-given-names">M</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Lu</span>
          <span class="cit-name-given-names">J</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Liu</span>
          <span class="cit-name-given-names">Y</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Hu</span>
          <span class="cit-name-given-names">D</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Hao</span>
          <span class="cit-name-given-names">Y.</span>
         </span>
         <span class="cit-pub-date">2017</span>. 
         <span class="cit-article-title">Transcription factor AREB2 is involved in soluble sugar accumulation by activating sugar transporter and amylase genes</span>. 
         <abbr class="cit-jnl-abbrev">Plant Physio</abbr>
         <span class="cit-vol">174</span>: 
         <span class="cit-fpage">2348</span>–
         <span class="cit-lpage">2362</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DPlant%2BPhysio%26rft.volume%253D174%26rft.spage%253D2348%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label ref-label-empty"/>
      <a class="rev-xref-ref" href="#xref-ref-38-1" title="View reference in text" id="ref-38">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.01.15.907709v1.38">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Martínez-García</span>
          <span class="cit-name-given-names">PJ</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Parfitt</span>
          <span class="cit-name-given-names">DE</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Ogundiwin</span>
          <span class="cit-name-given-names">EA</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Fass</span>
          <span class="cit-name-given-names">J</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Chan</span>
          <span class="cit-name-given-names">HM</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Ahmad</span>
          <span class="cit-name-given-names">R</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Lurie</span>
          <span class="cit-name-given-names">S</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Dandekar</span>
          <span class="cit-name-given-names">A</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Gradziel</span>
          <span class="cit-name-given-names">TM</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Crisosto</span>
          <span class="cit-name-given-names">CH</span>
         </span>. 
         <span class="cit-pub-date">2013</span>. 
         <span class="cit-article-title">High density SNP mapping and QTL analysis for fruit quality characteristics in peach (
          <em>Prunus persica</em> L
         </span>.). 
         <abbr class="cit-jnl-abbrev">Tree Genet &amp;; Genomes</abbr>
         <span class="cit-vol">9</span>: 
         <span class="cit-fpage">9</span>–
         <span class="cit-lpage">36</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DTree%2BGenet%2B%2526%2BGenomes%26rft.volume%253D9%26rft.spage%253D9%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label ref-label-empty"/>
      <a class="rev-xref-ref" href="#xref-ref-39-1" title="View reference in text" id="ref-39">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.01.15.907709v1.39" data-doi="10.1101/gr.107524.110">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">McKenna</span>
          <span class="cit-name-given-names">A</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Hanna</span>
          <span class="cit-name-given-names">M</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Banks</span>
          <span class="cit-name-given-names">E</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Sivachenko</span>
          <span class="cit-name-given-names">A</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Cibulskis</span>
          <span class="cit-name-given-names">K</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Kernytsky</span>
          <span class="cit-name-given-names">A</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Garimella</span>
          <span class="cit-name-given-names">K</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Altshuler</span>
          <span class="cit-name-given-names">D</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Gabriel</span>
          <span class="cit-name-given-names">S</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Daly</span>
          <span class="cit-name-given-names">M</span>
         </span>, 
         <span class="cit-etal">et al.</span>
         <span class="cit-pub-date">2010</span>. 
         <span class="cit-article-title">The Genome Analysis Toolkit: a MapReduce framework for analyzing next-generation DNA sequencing data</span>. 
         <abbr class="cit-jnl-abbrev">Genome Research</abbr>
         <span class="cit-vol">20</span>: 
         <span class="cit-fpage">1297</span>–
         <span class="cit-lpage">1303</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DGenome%2BResearch%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.107524.110%26rft_id%253Dinfo%253Apmid%252F20644199%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjk6IjIwLzkvMTI5NyI7czo0OiJhdG9tIjtzOjQ4OiIvYmlvcnhpdi9lYXJseS8yMDIwLzAxLzE1LzIwMjAuMDEuMTUuOTA3NzA5LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink">
         <span>
          <span class="cit-reflinks-abstract">Abstract</span>
          <span class="cit-sep cit-reflinks-variant-name-sep">/</span>
          <span class="cit-reflinks-full-text">
           <span class="free-full-text">FREE </span>Full Text
          </span>
         </span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label ref-label-empty"/>
      <a class="rev-xref-ref" href="#xref-ref-40-1" title="View reference in text" id="ref-40">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.01.15.907709v1.40">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Menzel</span>
          <span class="cit-name-given-names">A</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Sparks</span>
          <span class="cit-name-given-names">TH</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Estrella</span>
          <span class="cit-name-given-names">N</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Koch</span>
          <span class="cit-name-given-names">E</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Aasa</span>
          <span class="cit-name-given-names">A</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Ahas</span>
          <span class="cit-name-given-names">P</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Alm-Kubler</span>
          <span class="cit-name-given-names">K</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Bissolli</span>
          <span class="cit-name-given-names">P</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Braslavska</span>
          <span class="cit-name-given-names">O</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Briede</span>
          <span class="cit-name-given-names">A</span>
         </span>, 
         <span class="cit-etal">et al.</span>
         <span class="cit-pub-date">2006</span>. 
         <span class="cit-article-title">European phenological response to climate change matches the warming pattern</span>. 
         <abbr class="cit-jnl-abbrev">Glob Chang Biol</abbr>
         <span class="cit-vol">12</span>: 
         <span class="cit-fpage">1969</span>–
         <span class="cit-lpage">1976</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DGlob%2BChang%2BBiol%26rft.volume%253D12%26rft.spage%253D1969%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label ref-label-empty"/>
      <a class="rev-xref-ref" href="#xref-ref-41-1" title="View reference in text" id="ref-41">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.01.15.907709v1.41" data-doi="10.1111/j.1365-313X.2009.03957.x">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Mikkelsen</span>
          <span class="cit-name-given-names">MD</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Thomashow</span>
          <span class="cit-name-given-names">MF</span>
         </span>. 
         <span class="cit-pub-date">2009</span>. 
         <span class="cit-article-title">A role for circadian evening elements in cold-regulated gene expression in 
          <em>Arabidopsis</em>
         </span>. 
         <abbr class="cit-jnl-abbrev">Plant J</abbr>
         <span class="cit-vol">60</span>: 
         <span class="cit-fpage">328</span>–
         <span class="cit-lpage">339</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DThe%2BPlant%2Bjournal%2B%253A%2B%2Bfor%2Bcell%2Band%2Bmolecular%2Bbiology%26rft.stitle%253DPlant%2BJ%26rft.aulast%253DMikkelsen%26rft.auinit1%253DM.%2BD.%26rft.volume%253D60%26rft.issue%253D2%26rft.spage%253D328%26rft.epage%253D339%26rft.atitle%253DA%2Brole%2Bfor%2Bcircadian%2Bevening%2Belements%2Bin%2Bcold-regulated%2Bgene%2Bexpression%2Bin%2BArabidopsis.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1365-313X.2009.03957.x%26rft_id%253Dinfo%253Apmid%252F19566593%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1111/j.1365-313X.2009.03957.x&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
        <a href="/lookup/external-ref?access_num=19566593&amp;;link_type=MED&amp;;atom=%2Fbiorxiv%2Fearly%2F2020%2F01%2F15%2F2020.01.15.907709.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
         <span>PubMed</span>
        </a>
        <a href="/lookup/external-ref?access_num=000270664900012&amp;;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience">
         <span>Web of Science</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label ref-label-empty"/>
      <a class="rev-xref-ref" href="#xref-ref-42-1" title="View reference in text" id="ref-42">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.01.15.907709v1.42">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Monihan</span>
          <span class="cit-name-given-names">SM</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Magness</span>
          <span class="cit-name-given-names">CA</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Yadegari</span>
          <span class="cit-name-given-names">R</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Smith</span>
          <span class="cit-name-given-names">SE</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Schumaker</span>
          <span class="cit-name-given-names">KS</span>
         </span>. 
         <span class="cit-pub-date">2016</span>. 
         <abbr class="cit-jnl-abbrev">Arabidopsis CALCINEURIN B-LIKE10 functions independently of the SOS pathway during reproductive development in saline conditions. Plant Physio</abbr>
         <span class="cit-vol">171</span>: 
         <span class="cit-fpage">369</span>–
         <span class="cit-lpage">379</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DArabidopsis%2BCALCINEURIN%2BB-LIKE10%2Bfunctions%2Bindependently%2Bof%2Bthe%2BSOS%2Bpathway%2Bduring%2Breproductive%2Bdevelopment%2Bin%2Bsaline%2Bconditions.%2BPlant%2BPhysio%26rft.volume%253D171%26rft.spage%253D369%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label ref-label-empty"/>
      <a class="rev-xref-ref" href="#xref-ref-43-1" title="View reference in text" id="ref-43">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.01.15.907709v1.43" data-doi="10.1093/nar/8.19.4321">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Murray</span>
          <span class="cit-name-given-names">M</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Thompson</span>
          <span class="cit-name-given-names">WF</span>
         </span>. 
         <span class="cit-pub-date">1980</span>. 
         <span class="cit-article-title">Rapid isolation of high molecular weight plant DNA</span>. 
         <abbr class="cit-jnl-abbrev">Nucleic Acids Res</abbr>
         <span class="cit-vol">8</span>: 
         <span class="cit-fpage">4321</span>–
         <span class="cit-lpage">4326</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BResearch%26rft.stitle%253DNucleic%2BAcids%2BRes%26rft.aulast%253DMurray%26rft.auinit1%253DM.G.%26rft.volume%253D8%26rft.issue%253D19%26rft.spage%253D4321%26rft.epage%253D4326%26rft.atitle%253DRapid%2Bisolation%2Bof%2Bhigh%2Bmolecular%2Bweight%2Bplant%2BDNA%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252F8.19.4321%26rft_id%253Dinfo%253Apmid%252F7433111%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1093/nar/8.19.4321&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
        <a href="/lookup/external-ref?access_num=7433111&amp;;link_type=MED&amp;;atom=%2Fbiorxiv%2Fearly%2F2020%2F01%2F15%2F2020.01.15.907709.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
         <span>PubMed</span>
        </a>
        <a href="/lookup/external-ref?access_num=A1980KL92600001&amp;;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience">
         <span>Web of Science</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label ref-label-empty"/>
      <a class="rev-xref-ref" href="#xref-ref-44-1" title="View reference in text" id="ref-44">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.01.15.907709v1.44" data-doi="10.1038/nprot.2016.095">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Pertea</span>
          <span class="cit-name-given-names">M</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Kim</span>
          <span class="cit-name-given-names">D</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Pertea</span>
          <span class="cit-name-given-names">G</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Leek</span>
          <span class="cit-name-given-names">JT</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Salzberg</span>
          <span class="cit-name-given-names">SL</span>
         </span>. 
         <span class="cit-pub-date">2016</span>. 
         <span class="cit-article-title">Transcript-level expression analysis of RNA-seq experiments with HISAT, StringTie and Ballgown</span>. 
         <abbr class="cit-jnl-abbrev">Nat Protoc</abbr>
         <span class="cit-vol">11</span>: 
         <span class="cit-fpage">1650</span>–
         <span class="cit-lpage">1667</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DNat%2BProtoc%26rft.volume%253D11%26rft.spage%253D1650%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnprot.2016.095%26rft_id%253Dinfo%253Apmid%252F27560171%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1038/nprot.2016.095&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
        <a href="/lookup/external-ref?access_num=27560171&amp;;link_type=MED&amp;;atom=%2Fbiorxiv%2Fearly%2F2020%2F01%2F15%2F2020.01.15.907709.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
         <span>PubMed</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label ref-label-empty"/>
      <a class="rev-xref-ref" href="#xref-ref-45-1" title="View reference in text" id="ref-45">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.01.15.907709v1.45" data-doi="10.1038/nbt.3122">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Pertea</span>
          <span class="cit-name-given-names">ML</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Pertea</span>
          <span class="cit-name-given-names">GM</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Antonescu</span>
          <span class="cit-name-given-names">CM</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Chang</span>
          <span class="cit-name-given-names">TC</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Mendell</span>
          <span class="cit-name-given-names">JT</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Salzberg</span>
          <span class="cit-name-given-names">SL</span>
         </span>. 
         <span class="cit-pub-date">2015</span>. 
         <span class="cit-article-title">StringTie enables improved reconstruction of a transcriptome from RNA-seq reads</span>. 
         <abbr class="cit-jnl-abbrev">Nat Biotech</abbr>
         <span class="cit-vol">33</span>: 
         <span class="cit-fpage">290</span>–
         <span class="cit-lpage">295</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DNat%2BBiotech%26rft.volume%253D33%26rft.spage%253D290%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnbt.3122%26rft_id%253Dinfo%253Apmid%252F25690850%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1038/nbt.3122&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
        <a href="/lookup/external-ref?access_num=25690850&amp;;link_type=MED&amp;;atom=%2Fbiorxiv%2Fearly%2F2020%2F01%2F15%2F2020.01.15.907709.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
         <span>PubMed</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label ref-label-empty"/>
      <a class="rev-xref-ref" href="#xref-ref-46-1" title="View reference in text" id="ref-46">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.01.15.907709v1.46">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Price</span>
          <span class="cit-name-given-names">AL</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Patterson</span>
          <span class="cit-name-given-names">NJ</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Plenge</span>
          <span class="cit-name-given-names">RM</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Weinblatt</span>
          <span class="cit-name-given-names">ME</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Shadick</span>
          <span class="cit-name-given-names">NA</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Reich</span>
          <span class="cit-name-given-names">D.</span>
         </span>
         <span class="cit-pub-date">2006</span>. 
         <span class="cit-article-title">Principal components analysis corrects for stratification in genome-wide association studies</span>. 
         <abbr class="cit-jnl-abbrev">Na Genet</abbr>
         <span class="cit-vol">38</span>: 
         <span class="cit-fpage">904</span>–
         <span class="cit-lpage">909</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DNa%2BGenet%26rft.volume%253D38%26rft.spage%253D904%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label ref-label-empty"/>
      <a class="rev-xref-ref" href="#xref-ref-47-1" title="View reference in text" id="ref-47">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.01.15.907709v1.47" data-doi="10.1038/nrg2880">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Pritchard</span>
          <span class="cit-name-given-names">J</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Di Rienzo</span>
          <span class="cit-name-given-names">A.</span>
         </span>
         <span class="cit-pub-date">2010</span>. 
         <span class="cit-article-title">Adaptation-not by sweeps only</span>. 
         <abbr class="cit-jnl-abbrev">Nat Rev Genet</abbr>
         <span class="cit-vol">11</span>: 
         <span class="cit-fpage">665</span>–
         <span class="cit-lpage">667</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DNature%2Breviews.%2BGenetics%26rft.stitle%253DNat%2BRev%2BGenet%26rft.aulast%253DPritchard%26rft.auinit1%253DJ.%2BK.%26rft.volume%253D11%26rft.issue%253D10%26rft.spage%253D665%26rft.epage%253D667%26rft.atitle%253DAdaptation%2B-%2Bnot%2Bby%2Bsweeps%2Balone.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnrg2880%26rft_id%253Dinfo%253Apmid%252F20838407%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1038/nrg2880&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
        <a href="/lookup/external-ref?access_num=20838407&amp;;link_type=MED&amp;;atom=%2Fbiorxiv%2Fearly%2F2020%2F01%2F15%2F2020.01.15.907709.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
         <span>PubMed</span>
        </a>
        <a href="/lookup/external-ref?access_num=000281911300014&amp;;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience">
         <span>Web of Science</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label ref-label-empty"/>
      <a class="rev-xref-ref" href="#xref-ref-48-1" title="View reference in text" id="ref-48">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.01.15.907709v1.48" data-doi="10.1073/pnas.1302170110">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Rugnone</span>
          <span class="cit-name-given-names">ML</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Faigón Soverna</span>
          <span class="cit-name-given-names">A</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Sanchez</span>
          <span class="cit-name-given-names">SE</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Schlaen</span>
          <span class="cit-name-given-names">RG</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Hernando</span>
          <span class="cit-name-given-names">CE</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Seymour</span>
          <span class="cit-name-given-names">DK</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Mancini</span>
          <span class="cit-name-given-names">E</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Chernomoretz</span>
          <span class="cit-name-given-names">A</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Weigel</span>
          <span class="cit-name-given-names">D</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Más</span>
          <span class="cit-name-given-names">P</span>
         </span>, 
         <span class="cit-etal">et al.</span>
         <span class="cit-pub-date">2013</span>. 
         <span class="cit-article-title">
          <em>LNK</em> genes integrate light and clock signaling networks at the core of the 
          <em>Arabidopsis</em> oscillator
         </span>. 
         <abbr class="cit-jnl-abbrev">Proc Natl Acad Sci U S A</abbr>
         <span class="cit-vol">110</span>: 
         <span class="cit-fpage">12120</span>–
         <span class="cit-lpage">12125</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DProc%2BNatl%2BAcad%2BSci%2BU%2BS%2BA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.1302170110%26rft_id%253Dinfo%253Apmid%252F23818596%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMjoiMTEwLzI5LzEyMTIwIjtzOjQ6ImF0b20iO3M6NDg6Ii9iaW9yeGl2L2Vhcmx5LzIwMjAvMDEvMTUvMjAyMC4wMS4xNS45MDc3MDkuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink">
         <span>
          <span class="cit-reflinks-abstract">Abstract</span>
          <span class="cit-sep cit-reflinks-variant-name-sep">/</span>
          <span class="cit-reflinks-full-text">
           <span class="free-full-text">FREE </span>Full Text
          </span>
         </span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label ref-label-empty"/>
      <a class="rev-xref-ref" href="#xref-ref-49-1" title="View reference in text" id="ref-49">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.01.15.907709v1.49" data-doi="10.1104/pp.111.181982">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Sasaki</span>
          <span class="cit-name-given-names">R</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Yamane</span>
          <span class="cit-name-given-names">H</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Ooka</span>
          <span class="cit-name-given-names">T</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Jotatsu</span>
          <span class="cit-name-given-names">H</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Kitamura</span>
          <span class="cit-name-given-names">Y</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Akagi</span>
          <span class="cit-name-given-names">T</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Tao</span>
          <span class="cit-name-given-names">R.</span>
         </span>
         <span class="cit-pub-date">2011</span>. 
         <span class="cit-article-title">Functional and expressional analyses of 
          <em>PmDAM</em> genes associated with endodormancy in Japanese apricot
         </span>. 
         <abbr class="cit-jnl-abbrev">Plant Physiol</abbr>
         <span class="cit-vol">157</span>: 
         <span class="cit-fpage">485</span>–
         <span class="cit-lpage">497</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DPlant%2BPhysiol%26rft_id%253Dinfo%253Adoi%252F10.1104%252Fpp.111.181982%26rft_id%253Dinfo%253Apmid%252F21795580%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MTI6InBsYW50cGh5c2lvbCI7czo1OiJyZXNpZCI7czo5OiIxNTcvMS80ODUiO3M6NDoiYXRvbSI7czo0ODoiL2Jpb3J4aXYvZWFybHkvMjAyMC8wMS8xNS8yMDIwLjAxLjE1LjkwNzcwOS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink">
         <span>
          <span class="cit-reflinks-abstract">Abstract</span>
          <span class="cit-sep cit-reflinks-variant-name-sep">/</span>
          <span class="cit-reflinks-full-text">
           <span class="free-full-text">FREE </span>Full Text
          </span>
         </span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label ref-label-empty"/>
      <a class="rev-xref-ref" href="#xref-ref-50-1" title="View reference in text" id="ref-50">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.01.15.907709v1.50">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Seguel</span>
          <span class="cit-name-given-names">A</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Cumming</span>
          <span class="cit-name-given-names">JR</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Klugh-Stewart</span>
          <span class="cit-name-given-names">K</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Cornejo</span>
          <span class="cit-name-given-names">P</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Borie</span>
          <span class="cit-name-given-names">F.</span>
         </span>
         <span class="cit-pub-date">2013</span>. 
         <span class="cit-article-title">The role of arbuscular mycorrhizas in decreasing aluminium phytotoxicity in acidic soils: a review</span>. 
         <abbr class="cit-jnl-abbrev">Mycorrhiza</abbr>
         <span class="cit-vol">23</span>: 
         <span class="cit-fpage">167</span>–
         <span class="cit-lpage">183</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DMycorrhiza%26rft.volume%253D23%26rft.spage%253D167%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label ref-label-empty"/>
      <a class="rev-xref-ref" href="#xref-ref-51-1" title="View reference in text" id="ref-51">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.01.15.907709v1.51" data-doi="10.1002/gepi.20064">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Tang</span>
          <span class="cit-name-given-names">H</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Peng</span>
          <span class="cit-name-given-names">J</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Wang</span>
          <span class="cit-name-given-names">P</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Risch</span>
          <span class="cit-name-given-names">N.</span>
         </span>
         <span class="cit-pub-date">2005</span>. 
         <span class="cit-article-title">Estimation of individual admixture: analytical and study design considerations</span>. 
         <abbr class="cit-jnl-abbrev">Genet Epidemiol</abbr>
         <span class="cit-vol">28</span>: 
         <span class="cit-fpage">289</span>–
         <span class="cit-lpage">301</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DGenetic%2Bepidemiology%26rft.stitle%253DGenet%2BEpidemiol%26rft.aulast%253DTang%26rft.auinit1%253DH.%26rft.volume%253D28%26rft.issue%253D4%26rft.spage%253D289%26rft.epage%253D301%26rft.atitle%253DEstimation%2Bof%2Bindividual%2Badmixture%253A%2Banalytical%2Band%2Bstudy%2Bdesign%2Bconsiderations.%26rft_id%253Dinfo%253Adoi%252F10.1002%252Fgepi.20064%26rft_id%253Dinfo%253Apmid%252F15712363%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1002/gepi.20064&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
        <a href="/lookup/external-ref?access_num=15712363&amp;;link_type=MED&amp;;atom=%2Fbiorxiv%2Fearly%2F2020%2F01%2F15%2F2020.01.15.907709.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
         <span>PubMed</span>
        </a>
        <a href="/lookup/external-ref?access_num=000228573700001&amp;;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience">
         <span>Web of Science</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label ref-label-empty"/>
      <a class="rev-xref-ref" href="#xref-ref-52-1" title="View reference in text" id="ref-52">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.01.15.907709v1.52" data-doi="10.1126/science.1239402">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Tim</span>
          <span class="cit-name-given-names">W</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Braun</span>
          <span class="cit-name-given-names">JV</span>
         </span>. 
         <span class="cit-pub-date">2013</span>. 
         <span class="cit-article-title">Climate change impacts on global food security</span>. 
         <abbr class="cit-jnl-abbrev">Science</abbr>
         <span class="cit-vol">341</span>: 
         <span class="cit-fpage">508</span>–
         <span class="cit-lpage">513</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DWheeler%26rft.auinit1%253DT.%26rft.volume%253D341%26rft.issue%253D6145%26rft.spage%253D508%26rft.epage%253D513%26rft.atitle%253DClimate%2BChange%2BImpacts%2Bon%2BGlobal%2BFood%2BSecurity%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1239402%26rft_id%253Dinfo%253Apmid%252F23908229%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEyOiIzNDEvNjE0NS81MDgiO3M6NDoiYXRvbSI7czo0ODoiL2Jpb3J4aXYvZWFybHkvMjAyMC8wMS8xNS8yMDIwLjAxLjE1LjkwNzcwOS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink">
         <span>
          <span class="cit-reflinks-abstract">Abstract</span>
          <span class="cit-sep cit-reflinks-variant-name-sep">/</span>
          <span class="cit-reflinks-full-text">
           <span class="free-full-text">FREE </span>Full Text
          </span>
         </span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label ref-label-empty"/>
      <a class="rev-xref-ref" href="#xref-ref-53-1" title="View reference in text" id="ref-53">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.01.15.907709v1.53" data-doi="10.1093/bioinformatics/bts378">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Tobias</span>
          <span class="cit-name-given-names">R</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Zichner</span>
          <span class="cit-name-given-names">T</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Schlattl</span>
          <span class="cit-name-given-names">A</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Stütz</span>
          <span class="cit-name-given-names">AM</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Benes</span>
          <span class="cit-name-given-names">V</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Korbel</span>
          <span class="cit-name-given-names">JO</span>
         </span>. 
         <span class="cit-pub-date">2012</span>. 
         <span class="cit-article-title">Delly: structural variant discovery by integrated paired-end and split-read analysis</span>. 
         <abbr class="cit-jnl-abbrev">Bioinformatics</abbr>
         <span class="cit-vol">28</span>: 
         <span class="cit-fpage">i333</span>–
         <span class="cit-lpage">i339</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbts378%26rft_id%253Dinfo%253Apmid%252F22962449%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1093/bioinformatics/bts378&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
        <a href="/lookup/external-ref?access_num=22962449&amp;;link_type=MED&amp;;atom=%2Fbiorxiv%2Fearly%2F2020%2F01%2F15%2F2020.01.15.907709.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
         <span>PubMed</span>
        </a>
        <a href="/lookup/external-ref?access_num=000308532300005&amp;;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience">
         <span>Web of Science</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label ref-label-empty"/>
      <a class="rev-xref-ref" href="#xref-ref-54-1" title="View reference in text" id="ref-54">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.01.15.907709v1.54" data-doi="10.1038/ng.2586">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Verde</span>
          <span class="cit-name-given-names">I</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Abbott</span>
          <span class="cit-name-given-names">AG</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Scalabrin</span>
          <span class="cit-name-given-names">S</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Jung</span>
          <span class="cit-name-given-names">S</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Shu</span>
          <span class="cit-name-given-names">S</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Marroni</span>
          <span class="cit-name-given-names">F</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Zhebentyayeva</span>
          <span class="cit-name-given-names">T</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Dettori</span>
          <span class="cit-name-given-names">MT</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Grimwood</span>
          <span class="cit-name-given-names">J</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Cattonaro</span>
          <span class="cit-name-given-names">F</span>
         </span>, 
         <span class="cit-etal">et al.</span>
         <span class="cit-pub-date">2013</span>. 
         <span class="cit-article-title">The high-quality draft genome of peach (
          <em>Prunus persica</em>) identifies unique patterns of genetic diversity, domestication and genome evolution
         </span>. 
         <abbr class="cit-jnl-abbrev">Nat Genet</abbr>
         <span class="cit-vol">45</span>: 
         <span class="cit-fpage">487</span>–
         <span class="cit-lpage">494</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DNat%2BGenet%26rft.volume%253D45%26rft.spage%253D487%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fng.2586%26rft_id%253Dinfo%253Apmid%252F23525075%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1038/ng.2586&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
        <a href="/lookup/external-ref?access_num=23525075&amp;;link_type=MED&amp;;atom=%2Fbiorxiv%2Fearly%2F2020%2F01%2F15%2F2020.01.15.907709.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
         <span>PubMed</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label ref-label-empty"/>
      <a class="rev-xref-ref" href="#xref-ref-55-1" title="View reference in text" id="ref-55">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.01.15.907709v1.55">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Wang</span>
          <span class="cit-name-given-names">J</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Ding</span>
          <span class="cit-name-given-names">J</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Tan</span>
          <span class="cit-name-given-names">B</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Robinson</span>
          <span class="cit-name-given-names">KM</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Michelson</span>
          <span class="cit-name-given-names">IH</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Johansson</span>
          <span class="cit-name-given-names">A</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Nystedt</span>
          <span class="cit-name-given-names">B</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Scofield</span>
          <span class="cit-name-given-names">DG</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Nilsson</span>
          <span class="cit-name-given-names">O</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Jansson</span>
          <span class="cit-name-given-names">S</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Street</span>
          <span class="cit-name-given-names">NR</span>
         </span>, 
         <span class="cit-etal">et al.</span>
         <span class="cit-pub-date">2018</span>. 
         <span class="cit-article-title">A major locus controls local adaptation and adaptive life history variation in a perennial plant</span>. 
         <abbr class="cit-jnl-abbrev">Genome Bio</abbr>
         <span class="cit-vol">19</span>: 
         <span class="cit-fpage">72</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DGenome%2BBio%26rft.volume%253D19%26rft.spage%253D72%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label ref-label-empty"/>
      <a class="rev-xref-ref" href="#xref-ref-56-1" title="View reference in text" id="ref-56">↵</a>
      <div class="cit ref-cit ref-book" id="cit-2020.01.15.907709v1.56">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Wang</span>
          <span class="cit-name-given-names">L</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Zhu</span>
          <span class="cit-name-given-names">GR</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Fang</span>
          <span class="cit-name-given-names">WC</span>
         </span>. 
         <span class="cit-pub-date">2012</span>. 
         <span class="cit-source">Peach genetic resource in China</span>. 
         <span class="cit-publ-name">China Agriculture Press</span>.
        </cite>
       </div>
       <div class="cit-extra"/>
      </div>
     </li>
     <li>
      <span class="ref-label ref-label-empty"/>
      <a class="rev-xref-ref" href="#xref-ref-57-1" title="View reference in text" id="ref-57">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.01.15.907709v1.57" data-doi="10.1038/cr.2013.43">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Yan</span>
          <span class="cit-name-given-names">W</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Liu</span>
          <span class="cit-name-given-names">H</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Zhou</span>
          <span class="cit-name-given-names">X</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Li</span>
          <span class="cit-name-given-names">Q</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Zhang</span>
          <span class="cit-name-given-names">J</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Lu</span>
          <span class="cit-name-given-names">L</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Liu</span>
          <span class="cit-name-given-names">T</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Liu</span>
          <span class="cit-name-given-names">H</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Zhang</span>
          <span class="cit-name-given-names">C</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Zhang</span>
          <span class="cit-name-given-names">Z</span>
         </span>, 
         <span class="cit-etal">et al.</span>
         <span class="cit-pub-date">2013</span>. 
         <span class="cit-article-title">Natural variation in 
          <em>Ghd7.1</em> plays an important role in grain yield and adaptation in rice
         </span>. 
         <abbr class="cit-jnl-abbrev">Cell Res</abbr>
         <span class="cit-vol">23</span>(
         <span class="cit-issue">7</span>): 
         <span class="cit-fpage">969</span>–
         <span class="cit-lpage">971</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DCell%2BRes%26rft.volume%253D23%26rft.spage%253D969%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fcr.2013.43%26rft_id%253Dinfo%253Apmid%252F23507971%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1038/cr.2013.43&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
        <a href="/lookup/external-ref?access_num=23507971&amp;;link_type=MED&amp;;atom=%2Fbiorxiv%2Fearly%2F2020%2F01%2F15%2F2020.01.15.907709.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
         <span>PubMed</span>
        </a>
        <a href="/lookup/external-ref?access_num=000321113700015&amp;;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience">
         <span>Web of Science</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label ref-label-empty"/>
      <a class="rev-xref-ref" href="#xref-ref-58-1" title="View reference in text" id="ref-58">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.01.15.907709v1.58" data-doi="10.1038/ncomms2290">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Zhang</span>
          <span class="cit-name-given-names">Q</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Chen</span>
          <span class="cit-name-given-names">W</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Sun</span>
          <span class="cit-name-given-names">L</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Zhao</span>
          <span class="cit-name-given-names">F</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Huang</span>
          <span class="cit-name-given-names">B</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Yang</span>
          <span class="cit-name-given-names">W</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Tao</span>
          <span class="cit-name-given-names">Y</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Wang</span>
          <span class="cit-name-given-names">J</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Yuan</span>
          <span class="cit-name-given-names">Z</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Fan</span>
          <span class="cit-name-given-names">G</span>
         </span>, 
         <span class="cit-etal">et al.</span> (
         <span class="cit-pub-date">2012</span>). 
         <span class="cit-article-title">The genome of 
          <em>Prunus mume</em>
         </span>. 
         <abbr class="cit-jnl-abbrev">Nat Commun</abbr>
         <span class="cit-vol">3</span>: 
         <span class="cit-fpage">1318</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DNat%2BCommun%26rft.volume%253D3%26rft.spage%253D1318%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fncomms2290%26rft_id%253Dinfo%253Apmid%252F23271652%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1038/ncomms2290&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
        <a href="/lookup/external-ref?access_num=23271652&amp;;link_type=MED&amp;;atom=%2Fbiorxiv%2Fearly%2F2020%2F01%2F15%2F2020.01.15.907709.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
         <span>PubMed</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label ref-label-empty"/>
      <div class="cit ref-cit ref-journal no-rev-xref" id="cit-2020.01.15.907709v1.59" data-doi="10.1371/journal.pone.0106595">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Zheng</span>
          <span class="cit-name-given-names">Y</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Crawford</span>
          <span class="cit-name-given-names">GW</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Chen</span>
          <span class="cit-name-given-names">X.</span>
         </span>
         <span class="cit-pub-date">2014</span>. 
         <span class="cit-article-title">Archaeological evidence for peach (
          <em>Prunus persica</em>) cultivation and domestication in China
         </span>. 
         <abbr class="cit-jnl-abbrev">PLoS ONE</abbr>
         <span class="cit-vol">9</span>: 
         <span class="cit-fpage">e106595</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DPLoS%2BONE%26rft.volume%253D9%26rft.spage%253De106595%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0106595%26rft_id%253Dinfo%253Apmid%252F25192436%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1371/journal.pone.0106595&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
        <a href="/lookup/external-ref?access_num=25192436&amp;;link_type=MED&amp;;atom=%2Fbiorxiv%2Fearly%2F2020%2F01%2F15%2F2020.01.15.907709.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
         <span>PubMed</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label ref-label-empty"/>
      <a class="rev-xref-ref" href="#xref-ref-60-1" title="View reference in text" id="ref-60">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.01.15.907709v1.60" data-doi="10.1038/ng.2310">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Zhou</span>
          <span class="cit-name-given-names">X</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Stephens</span>
          <span class="cit-name-given-names">M.</span>
         </span>
         <span class="cit-pub-date">2012</span>. 
         <span class="cit-article-title">Genome-wide efficient mixed-model analysis for association studies</span>. 
         <abbr class="cit-jnl-abbrev">Nat Genet</abbr>
         <span class="cit-vol">44</span>: 
         <span class="cit-fpage">821</span>–
         <span class="cit-lpage">824</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DNature%2Bgenetics%26rft.stitle%253DNat%2BGenet%26rft.aulast%253DZhou%26rft.auinit1%253DX.%26rft.volume%253D44%26rft.issue%253D7%26rft.spage%253D821%26rft.epage%253D824%26rft.atitle%253DGenome-wide%2Befficient%2Bmixed-model%2Banalysis%2Bfor%2Bassociation%2Bstudies.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fng.2310%26rft_id%253Dinfo%253Apmid%252F22706312%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1038/ng.2310&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
        <a href="/lookup/external-ref?access_num=22706312&amp;;link_type=MED&amp;;atom=%2Fbiorxiv%2Fearly%2F2020%2F01%2F15%2F2020.01.15.907709.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
         <span>PubMed</span>
        </a>
       </div>
      </div>
     </li>
    </ol>
   </div>
   <span class="highwire-journal-article-marker-end"/>
  </div>
 </body>
</html>
