<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<link rel="dns-prefetch" href="//d33xdlntwy0kbs.cloudfront.net" />
<link rel="shortcut icon" href="https://www.biorxiv.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<meta name="article_thumbnail" content="https://www.biorxiv.org/content/biorxiv/early/2016/03/15/043729/embed/graphic-4.gif" />
<meta name="type" content="article" />
<meta name="category" content="article" />
<meta name="HW.identifier" content="/biorxiv/early/2016/03/15/043729.atom" />
<meta name="HW.pisa" content="biorxiv;043729v1" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="Common species link global ecosystems to climate change" />
<meta name="DC.Identifier" content="10.1101/043729" />
<meta name="DC.Date" content="2016-03-15" />
<meta name="DC.Publisher" content="Cold Spring Harbor Laboratory" />
<meta name="DC.Rights" content="© 2016, Posted by Cold Spring Harbor Laboratory. The copyright holder for this pre-print is the author. All rights reserved. The material may not be redistributed, re-used or adapted without the author's permission." />
<meta name="DC.AccessRights" content="restricted" />
<meta name="DC.Description" content="Common species shape the world around us, and changes in their commonness signify large-scale shifts in ecosystem structure and function[1][1]-[4][2]. Dominant taxa drive productivity and biogeochemical cycling, in direct interaction with abiotic components of the Earth system[3][3],[4][2]. However, our understanding of the dynamic response of ecosystems to global environmental changes in the past is limited by our ability to robustly estimate fossil taxonomic richness[5][4],[6][5], and by our neglect of the importance of common species. To rectify this, we use observations of the most common and widespread species to track global changes in their distribution in the deep geological past. Our simple approach is robust to factors that bias richness estimators, including widely used sampling-standardization methods[5][4], which we show are highly sensitive to variability in the species-abundance distribution. Causal analyses of common species frequency in the deep-sea sedimentary record detect a lagged response in the ecological prominence of planktonic foraminifera to oceanographic changes captured by deep-ocean temperature records over the last 65 million years, encompassing one of Earth's major climate transitions. Our results demonstrate that common species can act as tracers of a past global ecosystem and its response to physical changes in Earth's dynamic history.

 [1]: #ref-1
 [2]: #ref-4
 [3]: #ref-3
 [4]: #ref-5
 [5]: #ref-6" />
<meta name="DC.Contributor" content="Bjarte Hannisdal" />
<meta name="DC.Contributor" content="Kristian Agasøster Haaga" />
<meta name="DC.Contributor" content="Trond Reitan" />
<meta name="DC.Contributor" content="David Diego" />
<meta name="DC.Contributor" content="Lee Hsiang Liow" />
<meta name="article:published_time" content="2016-03-15" />
<meta name="article:section" content="New Results" />
<meta name="citation_title" content="Common species link global ecosystems to climate change" />
<meta name="citation_abstract" lang="en" content="&lt;p&gt;Common species shape the world around us, and changes in their commonness signify large-scale shifts in ecosystem structure and function&lt;sup&gt;1-4&lt;/sup&gt;. Dominant taxa drive productivity and biogeochemical cycling, in direct interaction with abiotic components of the Earth system&lt;sup&gt;3,4&lt;/sup&gt;. However, our understanding of the dynamic response of ecosystems to global environmental changes in the past is limited by our ability to robustly estimate fossil taxonomic richness&lt;sup&gt;5,6&lt;/sup&gt;, and by our neglect of the importance of common species. To rectify this, we use observations of the most common and widespread species to track global changes in their distribution in the deep geological past. Our simple approach is robust to factors that bias richness estimators, including widely used sampling-standardization methods&lt;sup&gt;5&lt;/sup&gt;, which we show are highly sensitive to variability in the species-abundance distribution. Causal analyses of common species frequency in the deep-sea sedimentary record detect a lagged response in the ecological prominence of planktonic foraminifera to oceanographic changes captured by deep-ocean temperature records over the last 65 million years, encompassing one of Earth9s major climate transitions. Our results demonstrate that common species can act as tracers of a past global ecosystem and its response to physical changes in Earth9s dynamic history.&lt;/p&gt;" />
<meta name="citation_journal_title" content="bioRxiv" />
<meta name="citation_publisher" content="Cold Spring Harbor Laboratory" />
<meta name="citation_publication_date" content="2016/01/01" />
<meta name="citation_mjid" content="biorxiv;043729v1" />
<meta name="citation_id" content="043729v1" />
<meta name="citation_public_url" content="https://www.biorxiv.org/content/10.1101/043729v1" />
<meta name="citation_abstract_html_url" content="https://www.biorxiv.org/content/10.1101/043729v1.abstract" />
<meta name="citation_full_html_url" content="https://www.biorxiv.org/content/10.1101/043729v1.full" />
<meta name="citation_pdf_url" content="https://www.biorxiv.org/content/biorxiv/early/2016/03/15/043729.full.pdf" />
<meta name="citation_doi" content="10.1101/043729" />
<meta name="citation_num_pages" content="31" />
<meta name="citation_article_type" content="Article" />
<meta name="citation_section" content="New Results" />
<meta name="citation_firstpage" content="043729" />
<meta name="citation_author" content="Bjarte Hannisdal" />
<meta name="citation_author_institution" content="Department of Earth Science, Centre for Geobiology, University of Bergen" />
<meta name="citation_author_email" content="bjarte.hannisdal@uib.no" />
<meta name="citation_author" content="Kristian Agasøster Haaga" />
<meta name="citation_author_institution" content="Department of Earth Science, Centre for Geobiology, University of Bergen" />
<meta name="citation_author" content="Trond Reitan" />
<meta name="citation_author_institution" content="Centre for Ecological and Evolutionary Synthesis, Department of Biosciences, University of Oslo" />
<meta name="citation_author" content="David Diego" />
<meta name="citation_author_institution" content="Department of Earth Science, Centre for Geobiology, University of Bergen" />
<meta name="citation_author" content="Lee Hsiang Liow" />
<meta name="citation_author_institution" content="Centre for Ecological and Evolutionary Synthesis, Department of Biosciences, University of Oslo" />
<meta name="citation_author_institution" content="Natural History Museum, University of Oslo" />
<meta name="citation_reference" content="Preston, F. W. The commonness, and rarity, of species. Ecology 29, 254–283 (1948)." />
<meta name="citation_reference" content="Connolly, S. R. et al. Commonness and rarity in the marine biosphere. Proceedings of the National Academy of Sciences 111, 8524–8529 (2014)." />
<meta name="citation_reference" content="ter Steege, H. et al. Hyperdominance in the Amazonian Tree Flora. Science 342, 1243092 (2013)." />
<meta name="citation_reference" content="de Vargas, C. et al. Eukaryotic plankton diversity in the sunlit ocean. Science 348, 1261605 (2015)." />
<meta name="citation_reference" content="Alroy, J. The Shifting Balance of Diversity Among Major Marine Animal Groups. Science 329, 1191–1194 (2010)." />
<meta name="citation_reference" content="Ezard, T. H. G., Aze, T., Pearson, P. N. &amp; Purvis, A. Interplay between changing climate and species’ ecology drives macroevolutionary dynamics. Science 332, 349–351 (2011)." />
<meta name="citation_reference" content="Gaston, K. J. et al. Abundance–occupancy relationships. Journal of Applied Ecology 37, 39–59(2000)." />
<meta name="citation_reference" content="Foote, M. Symmetric waxing and waning of marine invertebrate genera. Paleobiology 33, 517–529(2007)." />
<meta name="citation_reference" content="Liow, L. H. &amp; Stenseth, N. C. The rise and fall of species: implications for macroevolutionary and macroecological studies. Proceedings of the Royal Society B: Biological Sciences 274, 2745–2752 (2007)." />
<meta name="citation_reference" content="Liow, L. H., Skaug, H. J., Ergon, T. &amp; Schweder, T. Global occurrence trajectories of microfossils: environmental volatility and the rise and fall of individual species. Paleobiology 36, 224–252 (2010)." />
<meta name="citation_reference" content="Hannisdal, B., Henderiks, J. &amp; Liow, L. H. Long-term evolutionary and ecological responses of calcifying phytoplankton to changes in atmospheric CO2. Glob Change Biol 18, 3504–3516 (2012)." />
<meta name="citation_reference" content="Alroy, J. New methods for quantifying macroevolutionary patterns and processes. Paleobiology 26, 707–733 (2000)." />
<meta name="citation_reference" content="Spencer-Cervato, C. The Cenozoic deep sea microfossil record: explorations of the DSDP/ODP sample set using the Neptune database. Palaeontologia Electronica 42, 1268 (1999)." />
<meta name="citation_reference" content="Lazarus, D. Neptune: A marine micropaleontology database. Math Geol 26, 817–832 (1994)." />
<meta name="citation_reference" content="Rivero-Calle, S., Gnanadesikan, A., Del Castillo, C. E., Balch, W. M. &amp; Guikema, S. D. Multidecadal increase in North Atlantic coccolithophores and the potential role of rising CO2. Science 350, 1533–1537 (2015)." />
<meta name="citation_reference" content="Ridgwell, A. &amp; Zeebe, R. E. The role of the global carbonate cycle in the regulation and evolution of the Earth system. Earth and Planetary Science Letters 234, 299–315 (2005)." />
<meta name="citation_reference" content="Palike, H. et al. A Cenozoic record of the equatorial Pacific carbonate compensation depth. Nature 488, 609–614 (2012)." />
<meta name="citation_reference" content="Cramer, B. S., Miller, K. G., Barrett, P. J. &amp; Wright, J. D. Late Cretaceous–Neogene trends in deep ocean temperature and continental ice volume: Reconciling records of benthic foraminiferal geochemistry ( δ18O and Mg/Ca) with sea level history. J. Geophys. Res. 116, C12023 (2011)." />
<meta name="citation_reference" content="Cifelli, R. Radiation of Cenozoic Planktonic Foraminifera. Syst Biol 18, 154–168 (1969)." />
<meta name="citation_reference" content="Schmidt, D. N., Thierstein, H. R., Bollmann, J. &amp; Schiebel, R. Abiotic forcing of plankton evolution in the Cenozoic. Science 303, 207–210 (2004)." />
<meta name="citation_reference" content="Peters, S. E., Kelly, D. C. &amp; Fraass, A. J. Oceanographic controls on the diversity and extinction of planktonic foraminifera. Nature 493, 398–401 (2013)." />
<meta name="citation_reference" content="Sugihara, G. et al. Detecting Causality in Complex Ecosystems. Science 338, 496–500 (2012)." />
<meta name="citation_reference" content="Verdes, P. Assessing causality from multivariate time series. Phys. Rev. E 72, 026222 (2005)." />
<meta name="citation_reference" content="Hannisdal, B. Non-parametric inference of causal interactions from geological records. American Journal of Science 311, 315–334 (2011)." />
<meta name="citation_reference" content="Schreiber, T. Measuring information transfer. Phys. Rev. Lett. 85, 461–464 (2000)." />
<meta name="citation_reference" content="Reitan, T., Schweder, T. &amp; Henderiks, J. Phenotypic evolution studied by layered stochastic differential equations. The Annals of Applied Statistics 6, 1531–1551 (2012)." />
<meta name="citation_reference" content="Liow, L. H., Reitan, T. &amp; Harnik, P. G. Ecological interactions on macroevolutionary time scales: clams and brachiopods are more than ships that pass in the night. Ecol Letters 18, 1030–1039 (2015)." />
<meta name="citation_reference" content="Ye, H., Deyle, E. R., Gilarranz, L. J. &amp; Sugihara, G. Distinguishing time-delayed causal interactions using convergent cross mapping. Sci. Rep. 5, 14750 (2015)." />
<meta name="citation_reference" content="Jernvall, J. &amp; Fortelius, M. Common mammals drive the evolutionary increase of hypsodonty in the Neogene. Nature 417, 538–540 (2002)." />
<meta name="citation_reference" content="Stewart, D. &amp; Pearson, P. N. PLANKRANGE: a database of planktonic foraminiferal ranges. (2000). at &lt;http://palaeo.gly.bris.ac.uk/Data/plankrange.html&gt;" />
<meta name="citation_reference" content="Aze, T. et al. A phylogeny of Cenozoic macroperforate planktonic foraminifera from fossil data. Biological Reviews 86, 900–927 (2011)." />
<meta name="citation_reference" content="Gradstein, F. M., Ogg, J. G. &amp; Smith, A. G. (eds.) A Geologic Time Scale 2004. (Cambridge University Press 2004)." />
<meta name="citation_reference" content="Casella, G. &amp; Berger, R. L. Statistical inference. (Duxbury, 2002)." />
<meta name="citation_reference" content="Bush, A. M., Markey, M. J. &amp; Marshall, C. R. Removing bias from diversitycurves: the effects of spatially organized biodiversity on sampling-standardization. Paleobiology 30, 666–686 (2004)" />
<meta name="citation_reference" content="Alroy, J. Accurate and precise estimates of origination and extinction rates. Paleobiology 40, 374–397 (2014)." />
<meta name="citation_reference" content="Alroy, J. Fair sampling of taxonomic richness and unbiased estimation of origination and extinction rates. The Paleontological Society Papers 16, 55–80 (2010)." />
<meta name="citation_reference" content="Alroy, J. Geographical, environmental and intrinsic biotic controls on Phanerozoic marine diversification. Palaeontology 53, 1211–1235 (2010)." />
<meta name="citation_reference" content="Kwiatkowski, D., Phillips, P. C. B., Schmidt, P. &amp; Shin, Y. Testing the null hypothesis of stationarity against the alternative of a unit root. Journal of Econometrics 54, 159178 (1992)." />
<meta name="citation_reference" content="Takens, F. in Dynamical Systems and Turbulence (eds. Rand, D. A. &amp; Young, L. S.) 898, 366–381 (Springer, 1981)." />
<meta name="citation_reference" content="Sugihara, G. &amp; May, R. M. Nonlinear forecasting as a way of distinguishing chaos from measurement error in time series. Nature 344, 734–741 (1990)." />
<meta name="citation_reference" content="Ye, H. et al. Reply to Luo et al.: Robustness of causal effects of galactic cosmic rays on interannual variation in global temperature. Proceedings of the National Academy of Sciences of the United States of America 112, E4640–E4641 (2015)." />
<meta name="citation_reference" content="Luo, M., Kantz, H., Lau, N.-C., Huang, W. &amp; Zhou, Y. Questionable dynamical evidence for causality between galactic cosmic rays and interannual variation in global temperature. Proceedings of the National Academy of Sciences of the United States of America 112, E4638–E4639 (2015)." />
<meta name="citation_reference" content="Theiler, J., Eubank, S., Longtin, A. &amp; Galdrikian, B. Testing for nonlinearity in time series: the method of surrogate data. Physica D: Nonlinear Phenomena 58, 77–94 (1992)." />
<meta name="citation_reference" content="Ebisuzaki, W. A method to estimate the statistical significance of a correlation when the data are serially correlated. Journal of Climate 10, 2147–2153 (1997)." />
<meta name="citation_reference" content="Palus, M. &amp; Vejmelka, M. Directionality of coupling from bivariate time series: How to avoid false causalities and missed connections. Phys. Rev. E 75, 056211 (2007)." />
<meta name="citation_reference" content="Granger, C. W. J. Investigating Causal Relations by Econometric Models and Cross-spectral Methods. Econometrica: Journal of the Econometric Society 37, 424–438 (1969)." />
<meta name="citation_reference" content="Barnett, L., Barrett, A. B. &amp; Seth, A. K. Granger Causality and Transfer Entropy Are Equivalent for Gaussian Variables. Phys. Rev. Lett. 103, 238701 (2009)." />
<meta name="citation_reference" content="Hannisdal, B. &amp; Peters, S. E. Phanerozoic Earth system evolution and marine biodiversity. Science 334, 1122–1124 (2011)." />
<meta name="citation_reference" content="Dunhill, A. M., Hannisdal, B. &amp; Benton, M. J. Disentangling rock record bias and common-cause from redundancy in the British fossil record. Nat Comms 5, 5818 (2014)." />
<meta name="citation_reference" content="Jeffreys, H. Theory of Probability. (Oxford University Press, 1961)." />
<meta name="twitter:title" content="Common species link global ecosystems to climate change" />
<meta name="twitter:site" content="@biorxivpreprint" />
<meta name="twitter:card" content="summary" />
<meta name="twitter:image" content="https://www.biorxiv.org/sites/default/files/images/biorxiv_logo_homepage7-5-small.png" />
<meta name="twitter:description" content="Common species shape the world around us, and changes in their commonness signify large-scale shifts in ecosystem structure and function[1][1]-[4][2]. Dominant taxa drive productivity and biogeochemical cycling, in direct interaction with abiotic components of the Earth system[3][3],[4][2]. However, our understanding of the dynamic response of ecosystems to global environmental changes in the past is limited by our ability to robustly estimate fossil taxonomic richness[5][4],[6][5], and by our neglect of the importance of common species. To rectify this, we use observations of the most common and widespread species to track global changes in their distribution in the deep geological past. Our simple approach is robust to factors that bias richness estimators, including widely used sampling-standardization methods[5][4], which we show are highly sensitive to variability in the species-abundance distribution. Causal analyses of common species frequency in the deep-sea sedimentary record detect a lagged response in the ecological prominence of planktonic foraminifera to oceanographic changes captured by deep-ocean temperature records over the last 65 million years, encompassing one of Earth's major climate transitions. Our results demonstrate that common species can act as tracers of a past global ecosystem and its response to physical changes in Earth's dynamic history.

 [1]: #ref-1
 [2]: #ref-4
 [3]: #ref-3
 [4]: #ref-5
 [5]: #ref-6" />
<meta name="og-title" property="og:title" content="Common species link global ecosystems to climate change" />
<meta name="og-url" property="og:url" content="https://www.biorxiv.org/content/10.1101/043729v1" />
<meta name="og-site-name" property="og:site_name" content="bioRxiv" />
<meta name="og-description" property="og:description" content="Common species shape the world around us, and changes in their commonness signify large-scale shifts in ecosystem structure and function[1][1]-[4][2]. Dominant taxa drive productivity and biogeochemical cycling, in direct interaction with abiotic components of the Earth system[3][3],[4][2]. However, our understanding of the dynamic response of ecosystems to global environmental changes in the past is limited by our ability to robustly estimate fossil taxonomic richness[5][4],[6][5], and by our neglect of the importance of common species. To rectify this, we use observations of the most common and widespread species to track global changes in their distribution in the deep geological past. Our simple approach is robust to factors that bias richness estimators, including widely used sampling-standardization methods[5][4], which we show are highly sensitive to variability in the species-abundance distribution. Causal analyses of common species frequency in the deep-sea sedimentary record detect a lagged response in the ecological prominence of planktonic foraminifera to oceanographic changes captured by deep-ocean temperature records over the last 65 million years, encompassing one of Earth's major climate transitions. Our results demonstrate that common species can act as tracers of a past global ecosystem and its response to physical changes in Earth's dynamic history.

 [1]: #ref-1
 [2]: #ref-4
 [3]: #ref-3
 [4]: #ref-5
 [5]: #ref-6" />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.biorxiv.org/sites/default/files/images/biorxiv_logo_homepage7-5-small.png" />
<meta name="citation_date" content="2016-03-15" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/10.1101/043729v1.full.pdf" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/10.1101/043729v1.full.txt" />
<link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/10.1101/043729v1.ppt" />
<meta name="description" content="bioRxiv - the preprint server for biology, operated by Cold Spring Harbor Laboratory, a research and educational institution" />
<meta name="generator" content="Drupal 7 (http://drupal.org)" />
<link rel="canonical" href="https://www.biorxiv.org/content/10.1101/043729v1" />
<link rel="shortlink" href="https://www.biorxiv.org/node/11918" />
    <title>Common species link global ecosystems to climate change | bioRxiv</title>  
    <link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__jMRAK66KMC1e4TQlwUNn3KiWVDC5AjueUAYEm1xBY_U__KSUjdT4jdcJ4qJz6fKY1K9WkYh1a5EcaWZxt_-zbTis__dPhHATyREQrnntSIgxU8XNoYny4rjInxP7lLKSNxQyY.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q5yzns" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__icrP-Y0qw2NP73rJxxkUtrJta0z3VLAABtNjfSdEghA__rlN3rx0wOoswGNHZMeZM96bNn-3cLWHXTztGDoThNlw__dPhHATyREQrnntSIgxU8XNoYny4rjInxP7lLKSNxQyY.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__e8NuQlin4bcnMtI2zOEVRblohOk3xZwfnFAFdibS6pw__XVRsOpcO2N5B6_yYKvNFz_CihNSGO6Yy3SOy-5iyzfE__dPhHATyREQrnntSIgxU8XNoYny4rjInxP7lLKSNxQyY.css" media="all" />
<style type="text/css" media="all">
/* <![CDATA[ */
#sliding-popup.sliding-popup-bottom,#sliding-popup.sliding-popup-bottom .eu-cookie-withdraw-banner,.eu-cookie-withdraw-tab{background:gray}#sliding-popup.sliding-popup-bottom.eu-cookie-withdraw-wrapper{background:transparent}#sliding-popup .popup-content #popup-text h1,#sliding-popup .popup-content #popup-text h2,#sliding-popup .popup-content #popup-text h3,#sliding-popup .popup-content #popup-text p,.eu-cookie-compliance-secondary-button,.eu-cookie-withdraw-tab{color:#fff !important}.eu-cookie-withdraw-tab{border-color:#fff}.eu-cookie-compliance-more-button{color:#fff !important}
/* ]]> */
</style>

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__dPhHATyREQrnntSIgxU8XNoYny4rjInxP7lLKSNxQyY.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__B6tGdsQSt1vT8d8j-19-oS4FKJe83RrSNBkjPs9jfos__6VkE5WcQmmQpSVTQx-xS2-zn_nlwFlDm-EN1HLJNjrA__dPhHATyREQrnntSIgxU8XNoYny4rjInxP7lLKSNxQyY.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__Cm80tkaB8yhCBXUaefYJe7GLLFWYsVxjke4GKdtw7XY__F3sVKlGyHKCaAPI_WdiHD7l_-05B6b9bLdUwzi1me90__dPhHATyREQrnntSIgxU8XNoYny4rjInxP7lLKSNxQyY.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__dPhHATyREQrnntSIgxU8XNoYny4rjInxP7lLKSNxQyY.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__dPhHATyREQrnntSIgxU8XNoYny4rjInxP7lLKSNxQyY.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__O9AieMsHBlZYeQh-sAOhdoS_nAEDhkj0jDZjGc_kUug__f_9pwrTE8mmshoJRfurwkQ36QonD_Jiwbop2sscnkrE__dPhHATyREQrnntSIgxU8XNoYny4rjInxP7lLKSNxQyY.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://d33xdlntwy0kbs.cloudfront.net/cshl_custom.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/all/libraries/modernizr/modernizr.min.js?q5yzns"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__yOhrJUBmMpN-jsuQlJYmS8ovkQU6I6KInr295HrSTb0__c--zHhbuOWmkdWkJYXiNWDmkpDRPesmbBjGlQcyqBOY__dPhHATyREQrnntSIgxU8XNoYny4rjInxP7lLKSNxQyY.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var _prum=[['id', '52e95df3abe53d2e33000000'], ['mark', 'firstbyte',
      (new Date()).getTime()]]; (function() {
      var s=document.getElementsByTagName('script')[0],
      p=document.createElement('script');
      p.async='async'; p.src='//rum-static.pingdom.net/prum.min.js';s.parentNode.insertBefore(p,s);})();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js___KyvuEvfmewWxdOLBXTYsSUo8Pfm_DijV6fWtGvSL_k__HSKkTgIr7nDcdvQYewipf6v4ak6ZLUOQkwPj44wKouA__dPhHATyREQrnntSIgxU8XNoYny4rjInxP7lLKSNxQyY.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__3uCzzYE5jr6ly9nvsM8PRBRkhh4p0NWCMDja4EKBy4M__dmGGVYcB8LhnbR2iAmVW0ZyXI6QuFVYFRORa5XxglXs__dPhHATyREQrnntSIgxU8XNoYny4rjInxP7lLKSNxQyY.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__0zdUME0GLauYYyEch2tGPydIw-IG7Uizip0Qb4wUsig__3RgszrCgk4bOnTzo4vaQ-hrWzaG6oD19l-R9CQ1YQ-o__dPhHATyREQrnntSIgxU8XNoYny4rjInxP7lLKSNxQyY.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__W6wAjopaALytcBam7oT3WhEzllkWaB9R-8hazHyvShM__rK5YBIqBy9qt0di0Sn4DriBx9TJupdmv6uGmLJyZvDs__dPhHATyREQrnntSIgxU8XNoYny4rjInxP7lLKSNxQyY.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-45638026-1", {"cookieDomain":"auto"});ga("set", "page", location.pathname + location.search + location.hash);ga("send", "pageview");ga('create', 'UA-189672-38', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__1PtAOZ8x7l6QAi8D7X6Kj_Y7HcUOFzvWRvvYHUcCT98__U8IB92u_CnnGwJ-X_xaUa9lzsC8OzUfN0WkK2JQfirg__dPhHATyREQrnntSIgxU8XNoYny4rjInxP7lLKSNxQyY.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings,{"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"zxpQznE-iEsnnnWjzyVCKLIH5MdGHRcOm5AKG0fUemk","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"modules\/comment\/comment.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"modules\/forum\/forum.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"sites\/default\/modules\/biorxiv\/css\/biorxiv.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/flexible\/flexible.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_collection\/css\/highwire_subject_collection.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire.highwire-list-expand.css":1,"public:\/\/ctools\/css\/bab59090150caf0c64e27afa1b36eef1.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/contrib\/eu_cookie_compliance\/css\/eu_cookie_compliance.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"0":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_biorxiv_styles\/css\/biorxiv-styles.css.less":1,"https:\/\/d33xdlntwy0kbs.cloudfront.net\/cshl_custom.css":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_figures.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_tables.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"http:\/\/d33xdlntwy0kbs.cloudfront.net\/cshl_custom.js":1,"sites\/all\/modules\/contrib\/eu_cookie_compliance\/js\/eu_cookie_compliance.js":1,"sites\/default\/modules\/jnl_biorxiv_styles\/js\/adv-search-collection-toggle.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/eu_cookie_compliance\/js\/jquery.cookie-1.4.1.min.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/default\/modules\/jnl_biorxiv_styles\/js\/biorxiv-scripts.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_nps_survey\/js\/nps_survey.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"sites\/default\/modules\/contrib\/disqus\/disqus.js":1,"misc\/textarea.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/contrib\/service_links\/js\/twitter_button.js":1,"sites\/all\/modules\/contrib\/service_links\/js\/facebook_like.js":1,"sites\/all\/modules\/contrib\/service_links\/js\/google_plus_one.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.highwire-list-expand.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"11918","apath":"\/biorxiv\/early\/2016\/03\/15\/043729.atom","pisa":"biorxiv;043729v1","markup":[{"requested":"index","variant":"index","view":"full","pisa":"biorxiv;043729\/DC1"},{"requested":"index","variant":"index","view":"full","pisa":"biorxiv;043729\/DC1"},{"requested":"full-text","variant":"full-text","view":"full","pisa":"biorxiv;043729v1"},{"requested":"index","variant":"index","view":"full","pisa":"biorxiv;043729\/DC1"},{"requested":"index","variant":"index","view":"full","pisa":"biorxiv;043729\/DC1"}],"processed":["highwire_math"],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"disqus":{"domain":"biorxivstage","url":"https:\/\/www.biorxiv.org\/content\/10.1101\/043729v1","title":"Common species link global ecosystems to climate change","identifier":"node\/11918"},"panels_ajax_pane":{"new-41":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u002268WbW7dYdY7ZevoEuE9MW8\\\\\\\/iw6wubi0UUoS7HEE4BvaHCVdAmlyrS8ixu33PHbFHgyyJL0QyWb3JmlGkjxffkZIX9k1ZzY3sigXGU9mG61n9+6ShKyc8RK9kWyaDxEiW\\\\\\\/B1Hnvnfb33LDe9MuDhpOjg3wnOj0klhR71BFdTovWvM+MYXVER92y043r+IAIsspRemTwt6a\\\\\\\/xKQoHnkajuSLZirevSihhFJzRW0Jx0KAAQyrJc3Ys1v0JH1gFQPKAc2DzJiN2DxJa2UR48CFtUrt2UveNftAE3hswH8QTcDFc8nuDqyqnw1D91QuMYouy4rbT5+DOQOWFO9AhSNL1LARUpSyXoApAHwVL9KaXt5Lp2jjI+moJnno7ykD3ZdP6go73BbmW2gSAHjvfTYF1FtDMRuXltJyB\\\\\\\/J4\\\\\\\/Sx2N+xuVdjPbfI7vlWjZvB5cUhhFyL3Zn1PUeAAelkMLvaIa34mxd\\\\\\\/wOUPm2wMbeybH\\\\\\\/w8ahioITGYItiicGKv3rL5mAuSCPCPwUNtM2zqydoLBfZz4G1o0BeFPpTsCeT4avITTJgdhZ0\\\\\\\/qVABd0dlxL6LQSH9wpQ4f+4yB54bNyx\\\\\\\/ps\\\\\\\/7SF8ug3Y+fqKmzzZESsPDUttm5jTNFhPeOTJk0YDI102YXkpnSHWkM1jdlRfJj9UEpP2LEbOdumZG0H6+CvMkrKDEPcBme7uxp0VgnHaeaRHXYY34ifopBzEuf7E\\\\\\\/9ecQ+pnSJlhKyqvho8W1xJoi4fb8yN0ooEZQa0qOwY3N01RLE3J0kYYIC+iwdEQWoALBMCa6m2pC\\\\\\\/2h6ytmLCg31UtuilVch6M2rJgA+lJaGeMCJ70pytl1fevWaSiSQeNBMSYlhIVdwmeUe182yY1AUuRCmHTIU7nOrreyG49gCRh7xk6y\\\\\\\/Wj0ehf5\\\\\\\/vD5zUxogKE3WAczyqPjw01m9Wf4IehLgWeT4pgRpgo1ZLD3DUCVm3lHGO2gQx7TRTDJQmWbwQr0IO5\\\\\\\/3YbrDqXx7T4Q2Jub4LNd7EKaulu6TmZ3KmXfPuK1cY52P4cJi+LE4Men4+4MScxQqs68OcEEOVOcwi25hNg4apTL+Y5vb7SQ3I9gq4othT7Y+ytberEokuKBwoXbzdoDk4rEwuErt9NMCgaeWvEujnL4lB2byEJmYN7aZNX4r3yGV8e3vSTNHLt2QfQjFDnsIB+4XGhSt3dmueL0fey89SfV9SspvIJOXo6MbN6qhPnUHVgfEQaFb3pJE8Nn7fl7LowpLlOyzLcIvbSACQq2fqrcYKYAdVzU9pJzNVY1SHPFPwAprvl3Gn8LVjeElNESXSvcuUW\\\\\\\/6iYb1RA=\\\u0022,\\\u0022iv\\\u0022:\\\u00224RRXJP\\\\\\\/t\\\\\\\/II64JOgy625yw==\\\u0022,\\\u0022salt\\\u0022:\\\u0022f9d8845712f72c9fee5db7d9ff111eab\\\u0022}\u0022,\u0022hmac\u0022:\u002244f76b61d2b1dc576f8b6fa0983423513fe56802e03327f07f8edfaa889545e6\u0022}"},"urlIsAjaxTrusted":{"\/content\/10.1101\/043729v1.full":true},"ws_fl":{"width":100,"height":21},"ws_gpo":{"size":"","annotation":"","lang":"","callback":"","width":300},"color":{"logo":"https:\/\/www.biorxiv.org\/sites\/default\/files\/bioRxiv_article.jpg"},"highwire_list_expand":{"is_collapsed":"1"},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"eu_cookie_compliance":{"popup_enabled":1,"popup_agreed_enabled":0,"popup_hide_agreed":0,"popup_clicking_confirmation":1,"popup_scrolling_confirmation":false,"popup_html_info":"\u003Cdiv\u003E\n  \u003Cdiv class =\u0022popup-content info\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n      \u003Cp\u003EWe use cookies on this site to enhance your user experience. By clicking any link on this page you are giving your consent for us to set cookies.\u003C\/p\u003E\n    \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022agree-button eu-cookie-compliance-default-button\u0022\u003EContinue\u003C\/button\u003E\n                    \u003Cbutton type=\u0022button\u0022 class=\u0022find-more-button eu-cookie-compliance-more-button\u0022\u003EFind out more\u003C\/button\u003E\n          \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E","use_mobile_message":false,"mobile_popup_html_info":"\u003Cdiv\u003E\n  \u003Cdiv class =\u0022popup-content info\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n          \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022agree-button eu-cookie-compliance-default-button\u0022\u003EContinue\u003C\/button\u003E\n                    \u003Cbutton type=\u0022button\u0022 class=\u0022find-more-button eu-cookie-compliance-more-button\u0022\u003EFind out more\u003C\/button\u003E\n          \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E\n","mobile_breakpoint":"768","popup_html_agreed":"\u003Cdiv\u003E\n  \u003Cdiv class=\u0022popup-content agreed\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n      \u003Ch2\u003EThank you for accepting cookies\u003C\/h2\u003E\u003Cp\u003EYou can now hide this message or find out more about cookies.\u003C\/p\u003E    \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022hide-popup-button eu-cookie-compliance-hide-button\u0022\u003EHide\u003C\/button\u003E\n              \u003Cbutton type=\u0022button\u0022 class=\u0022find-more-button eu-cookie-compliance-more-button-thank-you\u0022 \u003EMore info\u003C\/button\u003E\n          \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E","popup_use_bare_css":false,"popup_height":"auto","popup_width":"100%","popup_delay":1000,"popup_link":"\/help\/cookie-policy","popup_link_new_window":1,"popup_position":null,"popup_language":"en","store_consent":false,"better_support_for_screen_readers":0,"reload_page":0,"domain":"","popup_eu_only_js":0,"cookie_lifetime":"365","cookie_session":false,"disagree_do_not_show_popup":0,"method":"default","whitelisted_cookies":"","withdraw_markup":"\u003Cbutton type=\u0022button\u0022 class=\u0022eu-cookie-withdraw-tab\u0022\u003EPrivacy settings\u003C\/button\u003E\n\u003Cdiv class=\u0022eu-cookie-withdraw-banner\u0022\u003E\n  \u003Cdiv class=\u0022popup-content info\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n      \u003Cp\u003E\u0026lt;h2\u0026gt;We use cookies on this site to enhance your user experience\u0026lt;\/h2\u0026gt;\u0026lt;p\u0026gt;You have given your consent for us to set cookies.\u0026lt;\/p\u0026gt;\u003C\/p\u003E\n    \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022eu-cookie-withdraw-button\u0022\u003EWithdraw consent\u003C\/button\u003E\n    \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E\n","withdraw_enabled":false},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1,"trackUrlFragments":1},"jnl_biorxiv_styles":{"defaultJCode":"biorxiv"},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-11918 node-type-highwire-article context-content hw-default-jcode-biorxiv hw-article-type-article hw-article-category-new-results">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-M677548" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-M677548');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
        <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
    
  <div id="zone-branding" class="zone zone-branding clearfix print-display-block container-30">
    <div class="grid-15 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="bioRxiv" src="https://www.biorxiv.org/sites/default/files/bioRxiv_article.jpg" /></a>      </div>
                </div>
          </div>
</div><div class="grid-11 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-system block-menu block-main-menu block-system-main-menu odd block-without-title" id="block-system-main-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/" title="" class="" data-icon-position="" data-hide-link-title="0">Home</a></li>
<li class="leaf" role="menuitem"><a href="/about-biorxiv" class="" data-icon-position="" data-hide-link-title="0">About</a></li>
<li class="leaf" role="menuitem"><a href="/submit-a-manuscript" class="" data-icon-position="" data-hide-link-title="0">Submit</a></li>
<li class="last leaf" role="menuitem"><a href="/content/alertsrss" title="" class="" data-icon-position="" data-hide-link-title="0">ALERTS / RSS</a></li>
</ul></nav>    </div>
  </div>
</div><div class="block block-panels-mini block-biorxiv-search-box block-panels-mini-biorxiv-search-box even block-without-title" id="block-panels-mini-biorxiv-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-biorxiv_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/10.1101/043729v1.full" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--2" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-aHSFXYGk5Bpz_88xXpHCJVMqPNDYT-vqdwkkGkp_xGI" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2 advanced-search-link" >
  
      
  
  <div class="pane-content">
    <a href="/search">Advanced Search</a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix container-30">
  	      </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="11918" id="node11918" data-pisa="biorxiv;043729v1" data-pisa-master="biorxiv;043729" data-apath="/biorxiv/early/2016/03/15/043729.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div  class="highwire-cite highwire-cite-highwire-article highwire-citation-biorxiv-article-top clearfix has-author-tooltip" >

      <span class="biorxiv-article-type">
        New Results    </span>
  
        <h1 class="highwire-cite-title" id="page-title">Common species link global ecosystems to climate change</h1>  
      <div  class="highwire-cite-authors" ><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0"><span class="nlm-given-names">Bjarte</span> <span class="nlm-surname">Hannisdal</span></span>, <span class="highwire-citation-author" data-delta="1"><span class="nlm-given-names">Kristian Agasøster</span> <span class="nlm-surname">Haaga</span></span>, <span class="highwire-citation-author" data-delta="2"><span class="nlm-given-names">Trond</span> <span class="nlm-surname">Reitan</span></span>, <span class="highwire-citation-author" data-delta="3"><span class="nlm-given-names">David</span> <span class="nlm-surname">Diego</span></span>, <span class="highwire-citation-author" data-delta="4"><span class="nlm-given-names">Lee Hsiang</span> <span class="nlm-surname">Liow</span></span></span></div>
  
      <div  class="highwire-cite-metadata" ><span  class="highwire-cite-metadata-doi highwire-cite-metadata"><span class="label">doi:</span> https://doi.org/10.1101/043729 </span></div>
  
  
  </div>
<div id="hw-article-author-popups-node11918" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Bjarte Hannisdal </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-institution'>Department of Earth Science, Centre for Geobiology, University of Bergen</span>, PO Box 7803, N-5020 Bergen, <span class='nlm-country'>Norway</span>.</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Bjarte%2BHannisdal%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Hannisdal%20B&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ABjarte%2BHannisdal%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:bjarte.hannisdal@uib.no" class="" data-icon-position="" data-hide-link-title="0">bjarte.hannisdal@uib.no</a></span></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Kristian Agasøster Haaga </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-institution'>Department of Earth Science, Centre for Geobiology, University of Bergen</span>, PO Box 7803, N-5020 Bergen, <span class='nlm-country'>Norway</span>.</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Kristian%2BAgas%C3%B8ster%2BHaaga%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Haaga%20KA&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AKristian%2BAgas%25C3%25B8ster%2BHaaga%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Trond Reitan </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-institution'>Centre for Ecological and Evolutionary Synthesis, Department of Biosciences, University of Oslo</span>, PO Box 1066 Blindern, N-0316 Oslo, <span class='nlm-country'>Norway</span>.</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Trond%2BReitan%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Reitan%20T&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ATrond%2BReitan%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">David Diego </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-institution'>Department of Earth Science, Centre for Geobiology, University of Bergen</span>, PO Box 7803, N-5020 Bergen, <span class='nlm-country'>Norway</span>.</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=David%2BDiego%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Diego%20D&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ADavid%2BDiego%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Lee Hsiang Liow </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-institution'>Centre for Ecological and Evolutionary Synthesis, Department of Biosciences, University of Oslo</span>, PO Box 1066 Blindern, N-0316 Oslo, <span class='nlm-country'>Norway</span>.</div><div class='author-affiliation'><span class='nlm-institution'>Natural History Museum, University of Oslo</span>, PO Box 1172 Blindern, N-0318 Oslo, <span class='nlm-country'>Norway</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Lee%2BHsiang%2BLiow%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Liow%20LH&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ALee%2BHsiang%2BLiow%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/10.1101/043729v1" class="panels-ajax-tab-tab" data-panel-name="biorxiv_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:11918" data-trigger="" data-url-enabled="1">Abstract</a><a href="/panels_ajax_tab/biorxiv_tab_art/node:11918/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/10.1101/043729v1.full-text" class="panels-ajax-tab-tab" data-panel-name="article_tab_full_text" data-target-id="highwire_article_tabs" data-entity-context="node:11918" data-trigger="full-text" data-url-enabled="1">Full Text</a><a href="/panels_ajax_tab/article_tab_full_text/node:11918/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/10.1101/043729v1.article-info" class="panels-ajax-tab-tab" data-panel-name="biorxiv_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:11918" data-trigger="article-info" data-url-enabled="1">Info/History</a><a href="/panels_ajax_tab/biorxiv_tab_info/node:11918/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/10.1101/043729v1.article-metrics" class="panels-ajax-tab-tab" data-panel-name="article_tab_metrics" data-target-id="highwire_article_tabs" data-entity-context="node:11918" data-trigger="article-metrics" data-url-enabled="1">Metrics</a><a href="/panels_ajax_tab/article_tab_metrics/node:11918/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/10.1101/043729v1.supplementary-material" class="panels-ajax-tab-tab" data-panel-name="biorxiv_tab_data" data-target-id="highwire_article_tabs" data-entity-context="node:11918" data-trigger="supplementary-material" data-url-enabled="1">Supplementary material</a><a href="/panels_ajax_tab/biorxiv_tab_data/node:11918/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/10.1101/043729v1.full.pdf+html" class="panels-ajax-tab-tab" data-panel-name="biorxiv_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:11918" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i> Preview PDF</a><a href="/panels_ajax_tab/biorxiv_tab_pdf/node:11918/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="article_tab_full_text" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.biorxiv.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-article_tab_full_text"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-markup" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2>Abstract</h2><p id="p-2">Common species shape the world around us, and changes in their commonness signify large-scale shifts in ecosystem structure and function<sup><a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">1</a>-<a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">4</a></sup>. Dominant taxa drive productivity and biogeochemical cycling, in direct interaction with abiotic components of the Earth system<sup><a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">3</a>,<a id="xref-ref-4-2" class="xref-bibr" href="#ref-4">4</a></sup>. However, our understanding of the dynamic response of ecosystems to global environmental changes in the past is limited by our ability to robustly estimate fossil taxonomic richness<sup><a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">5</a>,<a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">6</a></sup>, and by our neglect of the importance of common species. To rectify this, we use observations of the most common and widespread species to track global changes in their distribution in the deep geological past. Our simple approach is robust to factors that bias richness estimators, including widely used sampling-standardization methods<sup><a id="xref-ref-5-2" class="xref-bibr" href="#ref-5">5</a></sup>, which we show are highly sensitive to variability in the species-abundance distribution. Causal analyses of common species frequency in the deep-sea sedimentary record detect a lagged response in the ecological prominence of planktonic foraminifera to oceanographic changes captured by deep-ocean temperature records over the last 65 million years, encompassing one of Earth's major climate transitions. Our results demonstrate that common species can act as tracers of a past global ecosystem and its response to physical changes in Earth's dynamic history.</p></div><p id="p-3">True species richness can be elusive even in well-studied ecosystems, because most species are very rare, and relatively few species account for most of the total abundance<sup><a id="xref-ref-1-2" class="xref-bibr" href="#ref-1">1</a>,<a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">2</a></sup>. For example, only ~1.4 <em>%</em> of the estimated tree species account for half of the biomass and control the cycling of water, carbon, and nutrients in the Amazon forest<sup><a id="xref-ref-3-2" class="xref-bibr" href="#ref-3">3</a></sup>. Similarly, a recent survey of eukaryotic diversity in the oceans found that ~0.24 % of the taxa accounted for half of the total number of rDNA reads<sup><a id="xref-ref-4-3" class="xref-bibr" href="#ref-4">4</a></sup>.</p><p id="p-4">Abundance and occupancy are typically positively correlated, with the more abundant species being the more widespread<sup><a id="xref-ref-4-4" class="xref-bibr" href="#ref-4">4</a>,<a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">7</a></sup>. In the fossil record, species and higher taxa generally have a humped temporal distribution of occurrences, being rare in the early and late stages of their known stratigraphic range<sup><a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">8</a>-<a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">10</a></sup>.</p><p id="p-5">Here we accommodate these ecological features by focussing only on species that are common and widespread at any given time, using the Summed Common species Occurrence Rate (SCOR), a very simple occurrence-based quantity that is sensitive to changes in total abundance (Methods)<sup><a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">11</a></sup>. We apply SCOR to deep-sea sedimentary records of calcifying plankton (coccolithophores and foraminifera) over the last 65 years to demonstrate how relative changes in the distribution of common and widespread species were linked to climate change on geological time scales.</p><p id="p-6">First, we evaluate the sensitivity of SCOR and commonly used richness estimators to potential biases using Poseidon, a simulation model of planktonic microfossil occurrences (<a id="xref-fig-1-1" class="xref-fig" href="#F1">Fig 1</a>; Methods and Supplementary Code). We target methods currently popular in palaeobiology, and highlight the effects of two main factors: variability in the spatial sampling completeness (<a id="xref-fig-1-2" class="xref-fig" href="#F1">Fig. 1b</a>), and variability in the shape of the species rank-abundance distribution (RAD; <a id="xref-fig-1-3" class="xref-fig" href="#F1">Fig. 1c</a>).</p><div id="F1" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2016/03/15/043729/F1.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Performance of SCOR and richness estimators in Poseidon model experiments. a, Simulated species richness and total abundance are decoupled. b, Sampled species occurrences reflect abundance distorted by the trend and short-term variability in sampled sites (in this example, variability = 0.1, corresponding to the standard deviation around the mean trend). c, Sampled species evenness (Pielou's J) captures changes in the shape parameter &#x3C3; of the RAD (in this example, variability = 2, corresponding to the range of &#x3C3;), superimposed on richness fluctuations and a net decrease caused by the sampling trend. d, Sensitivity to sampling variability and RAD shape variability. Values are median goodness-of-fit (R2) of 50 model runs, comparing SCOR to true abundance, and richness estimates to true richness. See text for abbreviations." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1598411233" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;&lt;span class=&quot;caption-title&quot;&gt;Performance of SCOR and richness estimators in Poseidon model experiments.&lt;/span&gt; &lt;strong&gt;a&lt;/strong&gt;, Simulated species richness and total abundance are decoupled. &lt;strong&gt;b&lt;/strong&gt;, Sampled species occurrences reflect abundance distorted by the trend and short-term variability in sampled sites (in this example, variability = 0.1, corresponding to the standard deviation around the mean trend). &lt;strong&gt;c&lt;/strong&gt;, Sampled species evenness (Pielou's J) captures changes in the shape parameter &#x3C3; of the RAD (in this example, variability = 2, corresponding to the range of &#x3C3;), superimposed on richness fluctuations and a net decrease caused by the sampling trend. &lt;strong&gt;d&lt;/strong&gt;, Sensitivity to sampling variability and RAD shape variability. Values are median goodness-of-fit (R&lt;sup&gt;2&lt;/sup&gt;) of 50 model runs, comparing SCOR to true abundance, and richness estimates to true richness. See text for abbreviations.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 1|" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2016/03/15/043729/F1.medium.gif" width="440" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 1|" src="https://www.biorxiv.org/content/biorxiv/early/2016/03/15/043729/F1.medium.gif" width="440" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2016/03/15/043729/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1|" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2016/03/15/043729/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml"><span class="fig-label">Figure 1|</span> <span class="caption-title">Performance of SCOR and richness estimators in Poseidon model experiments.</span><p id="p-7" class="first-child"><strong>a</strong>, Simulated species richness and total abundance are decoupled. <strong>b</strong>, Sampled species occurrences reflect abundance distorted by the trend and short-term variability in sampled sites (in this example, variability = 0.1, corresponding to the standard deviation around the mean trend). <strong>c</strong>, Sampled species evenness (Pielou's J) captures changes in the shape parameter σ of the RAD (in this example, variability = 2, corresponding to the range of σ), superimposed on richness fluctuations and a net decrease caused by the sampling trend. <strong>d</strong>, Sensitivity to sampling variability and RAD shape variability. Values are median goodness-of-fit (R<sup><a id="xref-ref-2-2" class="xref-bibr" href="#ref-2">2</a></sup>) of 50 model runs, comparing SCOR to true abundance, and richness estimates to true richness. See text for abbreviations.</p><div class="sb-div caption-clear"></div></div></div><p id="p-8">Our simulations with Poseidon show that the SCOR estimate of relative changes in total abundance is highly robust to variability in both spatial sampling and RAD shape (<a id="xref-fig-1-4" class="xref-fig" href="#F1">Fig. 1d</a>). By definition, SCOR is immune to the loss of rare species, and decoupled from changes in richness. As expected, the fidelity of raw sampled richness (S) decays rapidly with increasing sampling variability, but shows little sensitivity to changes in the shape of the RAD. Simple range-through richness (RT; assuming a species existed in all time bins between its first and last occurrence) is relatively robust to both factors, indicating that the level of sampling in Poseidon is sufficient to avoid severe edge effects. The Shannon entropy H,which reflects both richness and evenness, is very sensitive to RAD shape variability, ultimately tracking changes in evenness at the expense of changes in richness. Classical rarefaction (CR) and shareholder quorum subsampling (SQS)<sup><a id="xref-ref-5-3" class="xref-bibr" href="#ref-5">5</a></sup>, being sampling-standardization methods, are robust to the effect of spatial sampling variability on richness, all else being equal. However, both CR and SQS are highly sensitive to changes in RAD shape. As with Shannon H, increasing RAD variability causes CR and SQS to lose track of richness and respond to changes in the shape parameter σ of the RAD instead. Note that the σ values used in Poseidon generally correspond to high, moderately variable species evenness (<a id="xref-fig-4-1" class="xref-fig" href="#F4">Supplementary Fig. 1</a>). A third subsampling method, occurrences-squared weighted (O2W)<sup><a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">12</a></sup>, shows overall poor agreement with true richness.</p><div id="F4" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2016/03/15/043729/F4.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Distribution of sampled evenness across all Poseidon experiments. Shaded histogram represents the model runs testing the sensitivity to variability in the proportion of species randomly removed, and variability in RAD shape parameter &#x3C3; (Supplementary Fig. 3). Un-shaded histogram (note transparency in overlap) represents the model runs testing the sensitivity to variability in the proportion of sites sampled, and variability in &#x3C3; (Fig. 1). The median evenness is 0.76, and 95 % of the values are in the range 0.65 &#x2013; 0.90." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1598411233" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;&lt;span class=&quot;caption-title&quot;&gt;Distribution of sampled evenness across all Poseidon experiments.&lt;/span&gt; Shaded histogram represents the model runs testing the sensitivity to variability in the proportion of species randomly removed, and variability in RAD shape parameter &#x3C3; (Supplementary Fig. 3). Un-shaded histogram (note transparency in overlap) represents the model runs testing the sensitivity to variability in the proportion of sites sampled, and variability in &#x3C3; (Fig. 1). The median evenness is 0.76, and 95 &lt;em&gt;%&lt;/em&gt; of the values are in the range 0.65 &#x2013; 0.90.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Supplementary Figure 1|" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2016/03/15/043729/F4.medium.gif" width="440" height="365"/><noscript><img class="highwire-fragment fragment-image" alt="Supplementary Figure 1|" src="https://www.biorxiv.org/content/biorxiv/early/2016/03/15/043729/F4.medium.gif" width="440" height="365"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2016/03/15/043729/F4.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Supplementary Figure 1|" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2016/03/15/043729/F4.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Supplementary Figure 1|</span> <span class="caption-title">Distribution of sampled evenness across all Poseidon experiments.</span><p id="p-48" class="first-child">Shaded histogram represents the model runs testing the sensitivity to variability in the proportion of species randomly removed, and variability in RAD shape parameter σ (<a id="xref-fig-6-2" class="xref-fig" href="#F6">Supplementary Fig. 3</a>). Un-shaded histogram (note transparency in overlap) represents the model runs testing the sensitivity to variability in the proportion of sites sampled, and variability in σ (<a id="xref-fig-1-8" class="xref-fig" href="#F1">Fig. 1</a>). The median evenness is 0.76, and 95 <em>%</em> of the values are in the range 0.65 – 0.90.</p><div class="sb-div caption-clear"></div></div></div><p id="p-9">Turning to the rich deep-sea sedimentary record of the Cenozoic Era (0-65 million years ago), we analysed global occurrences of the two most prominent groups of calcifying plankton, coccolithophores and foraminifera, from the Neptune Sandbox Berlin (NSB) database<sup><a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">13</a>,<a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">14</a></sup> (Methods). In both groups, raw S generally increases along with the number of boreholes representing the spatial sampling, while sampled evenness (J) decreases (<a id="xref-fig-5-1" class="xref-fig" href="#F5">Supplementary Fig. 2a,b</a>), as expected if improved sampling enhances the detection of rare species (<a id="xref-fig-1-5" class="xref-fig" href="#F1">Fig. 1b,c</a>). Sampling-standardized richness estimates (SQS) seem to remove the sampling trend, but given the sensitivity of subsampling methods to RAD shape found in Poseidon, we suspected an evenness signal in the SQS estimates. Indeed, SQS richness can be reproduced by simply adding together the raw S and J curves (<a id="xref-fig-5-2" class="xref-fig" href="#F5">Supplementary Fig. 2c,d</a>), a relationship that emerges across NSB data and simulation runs (<a id="xref-fig-2-1" class="xref-fig" href="#F2">Fig. 2</a>). This result implies that changes in evenness are a major confounding factor for current sampling-standardized richness estimators.</p><div id="F5" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2016/03/15/043729/F5.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Sampling-standardized richness can be reproduced by the sum of raw richness and evenness. a,b, Raw sampled richness (S) and evenness (Pielou's J) of Cenozoic coccolithophores (a) and planktonic foraminifera (b) species from the NSB database. c, d, The sum of raw S and raw J superimposed on shareholder quorum subsampling (SQS) estimates of richness for coccolithophores (c) and foraminifera (d), all normalized to zero mean and unit standard deviation. SQS was calculated with a quorum level of 0.4 (higher quorum levels give nearly identical results but are less complete for the older part of the record). Ma, million years before present; Paleoc., Paleocene; Oligoc., Oligocene; P, Pliocene; Pt, Pleistocene." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1598411233" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;&lt;span class=&quot;caption-title&quot;&gt;Sampling-standardized richness can be reproduced by the sum of raw richness and evenness.&lt;/span&gt; &lt;strong&gt;a&lt;/strong&gt;,&lt;strong&gt;b&lt;/strong&gt;, Raw sampled richness (S) and evenness (Pielou's J) of Cenozoic coccolithophores (&lt;strong&gt;a&lt;/strong&gt;) and planktonic foraminifera (&lt;strong&gt;b&lt;/strong&gt;) species from the NSB database. &lt;strong&gt;c&lt;/strong&gt;, &lt;strong&gt;d&lt;/strong&gt;, The sum of raw S and raw J superimposed on shareholder quorum subsampling (SQS) estimates of richness for coccolithophores (&lt;strong&gt;c&lt;/strong&gt;) and foraminifera (&lt;strong&gt;d&lt;/strong&gt;), all normalized to zero mean and unit standard deviation. SQS was calculated with a quorum level of 0.4 (higher quorum levels give nearly identical results but are less complete for the older part of the record). Ma, million years before present; Paleoc., Paleocene; Oligoc., Oligocene; P, Pliocene; Pt, Pleistocene.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Supplementary Figure 2|" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2016/03/15/043729/F5.medium.gif" width="440" height="314"/><noscript><img class="highwire-fragment fragment-image" alt="Supplementary Figure 2|" src="https://www.biorxiv.org/content/biorxiv/early/2016/03/15/043729/F5.medium.gif" width="440" height="314"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2016/03/15/043729/F5.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Supplementary Figure 2|" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2016/03/15/043729/F5.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Supplementary Figure 2|</span> <span class="caption-title">Sampling-standardized richness can be reproduced by the sum of raw richness and evenness.</span><p id="p-49" class="first-child"><strong>a</strong>,<strong>b</strong>, Raw sampled richness (S) and evenness (Pielou's J) of Cenozoic coccolithophores (<strong>a</strong>) and planktonic foraminifera (<strong>b</strong>) species from the NSB database. <strong>c</strong>, <strong>d</strong>, The sum of raw S and raw J superimposed on shareholder quorum subsampling (SQS) estimates of richness for coccolithophores (<strong>c</strong>) and foraminifera (<strong>d</strong>), all normalized to zero mean and unit standard deviation. SQS was calculated with a quorum level of 0.4 (higher quorum levels give nearly identical results but are less complete for the older part of the record). Ma, million years before present; Paleoc., Paleocene; Oligoc., Oligocene; P, Pliocene; Pt, Pleistocene.</p><div class="sb-div caption-clear"></div></div></div><div id="F2" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2016/03/15/043729/F2.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Empirical relationship between sampling-standardized richness and the sum of raw richness and evenness. Values are first differences of normalized time series of SQS richness and of the sum of normalized raw richness (S) and evenness (Pielou's J). Data include Cenozoic coccolithophores (black) and planktonic foraminifera (red) from the NSB database (Supplementary Fig. 2), and all Poseidon model experiments (blue; N = 262,500). Stippled line marks the 1:1 relationship." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1598411233" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;&lt;span class=&quot;caption-title&quot;&gt;Empirical relationship between sampling-standardized richness and the sum of raw richness and evenness.&lt;/span&gt; Values are first differences of normalized time series of SQS richness and of the sum of normalized raw richness (S) and evenness (Pielou's J). Data include Cenozoic coccolithophores (black) and planktonic foraminifera (red) from the NSB database (Supplementary Fig. 2), and all Poseidon model experiments (blue; &lt;em&gt;N&lt;/em&gt; = 262,500). Stippled line marks the 1:1 relationship.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 2|" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2016/03/15/043729/F2.medium.gif" width="440" height="439"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 2|" src="https://www.biorxiv.org/content/biorxiv/early/2016/03/15/043729/F2.medium.gif" width="440" height="439"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2016/03/15/043729/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2|" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2016/03/15/043729/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 2|</span> <span class="caption-title">Empirical relationship between sampling-standardized richness and the sum of raw richness and evenness.</span><p id="p-10" class="first-child">Values are first differences of normalized time series of SQS richness and of the sum of normalized raw richness (S) and evenness (Pielou's J). Data include Cenozoic coccolithophores (black) and planktonic foraminifera (red) from the NSB database (<a id="xref-fig-5-3" class="xref-fig" href="#F5">Supplementary Fig. 2</a>), and all Poseidon model experiments (blue; <em>N</em> = 262,500). Stippled line marks the 1:1 relationship.</p><div class="sb-div caption-clear"></div></div></div><p id="p-11">Focussing instead on common species, coccolithophores and foraminifera have markedly different SCOR trajectories through the Cenozoic. On average, coccolithophores have their highest SCOR values in the Eocene, followed by a decline in the Oligocene and a resurgence in the late Miocene and Pliocene. Aspects of the coccolithophore SCOR pattern have been linked to Cenozoic proxy records of atmospheric CO<sub>2</sub>, suggesting that coccolithophores could thrive in a high-CO<sub>2</sub> world<sup><a id="xref-ref-11-2" class="xref-bibr" href="#ref-11">11</a>,<a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">15</a></sup>. Since their rise in the Mesozoic, coccolithophores shifted the dominant locus of carbonate burial from continental shelves to the deep sea, providing a new mechanism for buffering ocean chemistry and atmospheric CO<sub>2</sub> through carbonate compensation<sup><a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">16</a></sup>. Oligocene cooling and CO<sub>2</sub> decline was accompanied by a lowering of the carbonate compensation depth, which has been attributed to changes in the supply of weathering products to the ocean<sup><a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">17</a></sup>. The Oligocene reduction of coccolithophore SCOR is opposite to that expected if SCOR were biased upward by enhanced deep-sea preservation<sup><a id="xref-ref-11-3" class="xref-bibr" href="#ref-11">11</a></sup>, and carbonate preservation trends cannot explain the independent SCOR patterns in the two calcifying groups. Selective dissolution or taxonomic preferences in sample processing may cause short-term volatility in SCOR, but only if species presence or absence is random with respect to commonness (<a id="xref-fig-6-1" class="xref-fig" href="#F6">Supplementary Fig. 3</a>).</p><div id="F6" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2016/03/15/043729/F6.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Effect of random species loss in Poseidon model experiments. a, Simulated richness and abundance as in Fig. 1a. b, Site sampling increases smoothly in all experiments. Instead, a proportion of the species is randomly removed in each time step, causing volatility in occurrences. No variability in the proportion lost means that 50 % are always removed. In this example, variability = 0.4, meaning that between 30 % and 70 % of species are lost. c, Even with a constant original RAD shape, random species loss, and variability in the proportion lost, generates volatility in sampled evenness (this example is an extreme case, see Supplementary Fig. 1). d, Sensitivity to variability in RAD shape and in the proportion of species lost. Values are median goodness-of-fit (R2) of 50 model runs, comparing SCOR to true abundance, and richness estimates to true richness." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1598411233" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;&lt;span class=&quot;caption-title&quot;&gt;Effect of random species loss in Poseidon model experiments.&lt;/span&gt; &lt;strong&gt;a&lt;/strong&gt;, Simulated richness and abundance as in Fig. 1a. &lt;strong&gt;b&lt;/strong&gt;, Site sampling increases smoothly in all experiments. Instead, a proportion of the species is randomly removed in each time step, causing volatility in occurrences. No variability in the proportion lost means that 50 &lt;em&gt;%&lt;/em&gt; are always removed. In this example, variability = 0.4, meaning that between 30 % and 70 % of species are lost. &lt;strong&gt;c&lt;/strong&gt;, Even with a constant original RAD shape, random species loss, and variability in the proportion lost, generates volatility in sampled evenness (&lt;em&gt;t&lt;/em&gt;his example is an extreme case, see Supplementary Fig. 1). &lt;strong&gt;d&lt;/strong&gt;, Sensitivity to variability in RAD shape and in the proportion of species lost. Values are median goodness-of-fit (R&lt;sup&gt;2&lt;/sup&gt;) of 50 model runs, comparing SCOR to true abundance, and richness estimates to true richness.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Supplementary Figure 3|" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2016/03/15/043729/F6.medium.gif" width="437" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Supplementary Figure 3|" src="https://www.biorxiv.org/content/biorxiv/early/2016/03/15/043729/F6.medium.gif" width="437" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2016/03/15/043729/F6.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Supplementary Figure 3|" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2016/03/15/043729/F6.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Supplementary Figure 3|</span> <span class="caption-title">Effect of random species loss in Poseidon model experiments.</span><p id="p-50" class="first-child"><strong>a</strong>, Simulated richness and abundance as in <a id="xref-fig-1-9" class="xref-fig" href="#F1">Fig. 1a</a>. <strong>b</strong>, Site sampling increases smoothly in all experiments. Instead, a proportion of the species is randomly removed in each time step, causing volatility in occurrences. No variability in the proportion lost means that 50 <em>%</em> are always removed. In this example, variability = 0.4, meaning that between 30 % and 70 % of species are lost. <strong>c</strong>, Even with a constant original RAD shape, random species loss, and variability in the proportion lost, generates volatility in sampled evenness (<em>t</em>his example is an extreme case, see <a id="xref-fig-4-2" class="xref-fig" href="#F4">Supplementary Fig. 1</a>). <strong>d</strong>, Sensitivity to variability in RAD shape and in the proportion of species lost. Values are median goodness-of-fit (R<sup>2</sup>) of 50 model runs, comparing SCOR to true abundance, and richness estimates to true richness.</p><div class="sb-div caption-clear"></div></div></div><p id="p-12">Planktonic foraminifera SCOR was compared to Cenozoic deep-ocean temperature (DOT) records<sup><a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">18</a></sup> (Methods; Supplementary Data Set). Although the net trends are inversely related (foraminifera flourish as the world cools), shorter-term changes suggest positive covariation, including the Early Eocene climate optimum, Eocene cooling, as well as Miocene and Pliocene optima (<a id="xref-fig-3-1" class="xref-fig" href="#F3">Fig. 3a</a>). Geological proxy records are generally noisy mixtures of signals representing multiple processes, derived from a sedimentary record that is itself an active component of the Earth system. Any causal connection detected between proxy records would necessarily be indirect with respect to the underlying processes of interest. Nonetheless, the DOT record reflects a set of climate-related variables, including changes in ocean thermohaline circulation, water mass structure, and nutrient dynamics, all considered to be important abiotic controls on the long-term evolution of planktonic foraminifera<sup><a id="xref-ref-6-2" class="xref-bibr" href="#ref-6">6</a>,<a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">19</a>-<a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">21</a></sup>. Here we tested this drive-response hypothesis using three conceptually very different methods for causal detection in time series (Methods): (1) Convergent cross mapping<sup><a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">22</a></sup>, based on the concept of state space reconstruction from time-delay embedding; (2) Information transfer analysis <sup><a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">23</a>,<a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">24</a></sup>, based on the concept of transfer entropy<sup><a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">25</a></sup>; and (3) Bayesian inference of causal models based on linear stochastic differential equations<sup><a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">26</a>,<a id="xref-ref-27-1" class="xref-bibr" href="#ref-27">27</a></sup>.</p><div id="F3" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2016/03/15/043729/F3.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Testing a causal link between planktonic foraminifera SCOR and Cenozoic climate changes. a, SCOR of planktonic foraminifera from the NSB database at 0.5 Myr resolution, and DOT estimates18 at 0.1 Myr resolution. b, c, CCM skill (b) and IT (c) between SCOR and DOT as a function of time lag. If past DOT drives SCOR, then SCOR xmap DOT, while information flows DOT &#x2192; SCOR, at negative lags. Values are medians (dots) and 95% ranges (whiskers) for 500 random subsamples of length 100, dashed lines are 95th percentiles of 1,000 surrogates. All values are normalized to a surrogate mean of zero. Ma, million years before present; Paleoc., Paleocene; Oligoc., Oligocene; P, Pliocene; Pt, Pleistocene." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1598411233" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;&lt;span class=&quot;caption-title&quot;&gt;Testing a causal link between planktonic foraminifera SCOR and Cenozoic climate changes.&lt;/span&gt; &lt;strong&gt;a&lt;/strong&gt;, SCOR of planktonic foraminifera from the NSB database at 0.5 Myr resolution, and DOT estimates&lt;sup&gt;18&lt;/sup&gt; at 0.1 Myr resolution. &lt;strong&gt;b&lt;/strong&gt;, &lt;strong&gt;c&lt;/strong&gt;, CCM skill (&lt;strong&gt;b&lt;/strong&gt;) and IT (&lt;strong&gt;c&lt;/strong&gt;) between SCOR and DOT as a function of time lag. If past DOT drives SCOR, then SCOR &lt;em&gt;xmap&lt;/em&gt; DOT, while information flows DOT &#x2192; SCOR, at negative lags. Values are medians (dots) and 95% ranges (whiskers) for 500 random subsamples of length 100, dashed lines are 95&lt;sup&gt;th&lt;/sup&gt; percentiles of 1,000 surrogates. All values are normalized to a surrogate mean of zero. Ma, million years before present; Paleoc., Paleocene; Oligoc., Oligocene; P, Pliocene; Pt, Pleistocene.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 3|" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2016/03/15/043729/F3.medium.gif" width="257" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 3|" src="https://www.biorxiv.org/content/biorxiv/early/2016/03/15/043729/F3.medium.gif" width="257" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2016/03/15/043729/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 3|" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2016/03/15/043729/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 3|</span> <span class="caption-title">Testing a causal link between planktonic foraminifera SCOR and Cenozoic climate changes.</span><p id="p-13" class="first-child"><strong>a</strong>, SCOR of planktonic foraminifera from the NSB database at 0.5 Myr resolution, and DOT estimates<sup><a id="xref-ref-18-2" class="xref-bibr" href="#ref-18">18</a></sup> at 0.1 Myr resolution. <strong>b</strong>, <strong>c</strong>, CCM skill (<strong>b</strong>) and IT (<strong>c</strong>) between SCOR and DOT as a function of time lag. If past DOT drives SCOR, then SCOR <em>xmap</em> DOT, while information flows DOT → SCOR, at negative lags. Values are medians (dots) and 95% ranges (whiskers) for 500 random subsamples of length 100, dashed lines are 95<sup>th</sup> percentiles of 1,000 surrogates. All values are normalized to a surrogate mean of zero. Ma, million years before present; Paleoc., Paleocene; Oligoc., Oligocene; P, Pliocene; Pt, Pleistocene.</p><div class="sb-div caption-clear"></div></div></div><p id="p-14">Convergent cross mapping (CCM) from foraminifera SCOR to DOT peaks at a negative lag, indicating that the SCOR signal carries a response to past changes captured in the DOT record (<a id="xref-fig-3-2" class="xref-fig" href="#F3">Fig. 3b</a>). The optimum lag is a single time bin, implying a causal delay of 0.5 million years (Myr) or less. CCM is also significant in the opposite direction but this is stronger at positive lags (<a id="xref-fig-3-3" class="xref-fig" href="#F3">Fig. 3b</a>), which are non-causal (future "drives" past). This result is consistent with a unidirectional forcing where the dynamics of the response variable (SCOR) is dominated by the driving variable (DOT), such that predictability flows both ways<sup><a id="xref-ref-28-1" class="xref-bibr" href="#ref-28">28</a></sup>. Information transfer (IT) analysis supports this inference: predictive information flow is significant from past DOT to SCOR, although the optimal lag is shifted backward by one time bin, implying a more protracted causal delay (<a id="xref-fig-3-4" class="xref-fig" href="#F3">Fig. 3c</a>). In the opposite direction, IT peaks at the corresponding positive (non-causal) lags, but is significantly weaker than in the causal direction. Using a series of linear Stochastic Differential Equations (SDEs) to model correlation and causality between the two records (<a id="xref-fig-7-1" class="xref-fig" href="#F7">Supplementary Fig. 4</a>), we recover relatively strong evidence that SCOR responds to changes in DOT, with a time lag of 0.3-31.1 Myr, comparable to the CCM and IT analyses (<a id="xref-table-wrap-1-1" class="xref-table" href="#T1">Supplementary Tables 1</a>, <a id="xref-table-wrap-2-1" class="xref-table" href="#T2">2</a>), although the detailed nature of the causal relationship cannot be clearly resolved (Methods).</p><div id="F7" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2016/03/15/043729/F7.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Schematic of all connection models between SCOR and DOT in linear SDE analysis. The best model for SCOR in isolation is a one-layered OUR The best model for DOT in isolation is a three-layered model with a WP as the bottom layer (DOT3). All models possible between these two best models are shown. Note that SCOR cannot drive DOT3 because DOT3 is a WP. Percentage values represent posterior model probabilities. Solid arrows represent casual connections pointing from driver to response. Dotted lines represent correlative relationships. See Methods for details." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1598411233" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;span xmlns=&quot;http://www.w3.org/1999/xhtml&quot; class=&quot;caption-title&quot;&gt;Schematic of all connection models between SCOR and DOT in linear SDE analysis.&lt;/span&gt; The best model for SCOR in isolation is a one-layered OUR The best model for DOT in isolation is a three-layered model with a WP as the bottom layer (DOT3). All models possible between these two best models are shown. Note that SCOR cannot drive DOT3 because DOT3 is a WP. Percentage values represent posterior model probabilities. Solid arrows represent casual connections pointing from driver to response. Dotted lines represent correlative relationships. See Methods for details.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Supplementary Figure 4|" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2016/03/15/043729/F7.medium.gif" width="440" height="343"/><noscript><img class="highwire-fragment fragment-image" alt="Supplementary Figure 4|" src="https://www.biorxiv.org/content/biorxiv/early/2016/03/15/043729/F7.medium.gif" width="440" height="343"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2016/03/15/043729/F7.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Supplementary Figure 4|" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2016/03/15/043729/F7.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Supplementary Figure 4|</span> <span class="caption-title">Schematic of all connection models between SCOR and DOT in linear SDE analysis.</span><p id="p-51" class="first-child">The best model for SCOR in isolation is a one-layered OUR The best model for DOT in isolation is a three-layered model with a WP as the bottom layer (DOT3). All models possible between these two best models are shown. Note that SCOR cannot drive DOT3 because DOT3 is a WP. Percentage values represent posterior model probabilities. Solid arrows represent casual connections pointing from driver to response. Dotted lines represent correlative relationships. See Methods for details.</p><div class="sb-div caption-clear"></div></div></div><div id="T1" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/200241/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup"><a href="/highwire/markup/200241/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/200241" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div></div><div class="table-caption"><span class="table-label">Supplementary Table 1|</span> <span class="caption-title">Parameter estimates for the most probable connection model between SCOR and DOT</span><div class="sb-div caption-clear"></div></div></div><div id="T2" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/200057/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup"><a href="/highwire/markup/200057/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/200057" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div></div><div class="table-caption"><span class="table-label">Supplementary Table 2|</span> <span class="caption-title">Parameter estimates for the most probable connection model between uSCOR and uDOT</span><div class="sb-div caption-clear"></div></div></div><p id="p-15">The congruence of these results strongly suggests that the ecological prominence of planktonic foraminifera has evolved in response to past climatic and oceanographic changes captured in the deep-ocean temperature proxy record. Furthermore, the inferred time delay implies that the causal connection is highly indirect, involving climate changes propagating through the Earth system to influence the commonness of foraminifera in the global plankton on evolutionary time scales. In the modern global ocean, eukaryotic plankton richness involves a vast number of parasite and symbiont species, highlighting the importance of biotic interactions in driving diversification through trophic connectivity and complexity<sup><a id="xref-ref-4-5" class="xref-bibr" href="#ref-4">4</a></sup>. Abiotic factors, such as differences in nutrient level among ocean basins, are more clearly reflected in the relative abundance of the dominant species. A restructuring of water masses and nutrient distributions is likely to cause a dramatic and discernible shift in the distribution and abundance of many species, yet have a far less predictable impact on richness. Our results imply that if such a fundamental ecosystem response were to leave a signature in the fossil record, it would be far more evident in the robustly detectable distribution of the most common species than in the indeterminate richness of rare species. Dominant groups also reveal macroevolutionary trends in functional morphology otherwise obscured by rare taxa<sup><a id="xref-ref-29-1" class="xref-bibr" href="#ref-29">29</a></sup>. Given their critical importance to ecosystem functioning, common species provide a nexus for understanding the role of an evolving biota in global environmental changes of the past.</p><div class="section" id="sec-1"><h2 class="">Author Contributions</h2><p id="p-17">B.H. and L.H.L designed the study. B.H. ran the Poseidon simulations, analysed data, performed the IT analysis, and wrote the paper. L.H.L. retrieved the NSB data, calculated SCOR, and wrote the Poseidon code. K.A.H. and D.D. performed the CCM analysis. T.R. performed the SDE analysis. All authors discussed the results and commented on the manuscript.</p></div><div class="section" id="sec-3"><h2 class="">METHODS</h2><div class="section" id="sec-4"><h3>Data</h3><p id="p-19">Microfossil occurrences were retrieved from the NSB database<sup><a id="xref-ref-13-2" class="xref-bibr" href="#ref-13">13</a>,<a id="xref-ref-14-2" class="xref-bibr" href="#ref-14">14</a></sup> (accessed April 22, 2015). SCOR and richness estimates were calculated using 0.5 Myr time bins. For the planktonic foraminifera, we compared the inferred times of species rise and fall in NSB<sup><a id="xref-ref-10-2" class="xref-bibr" href="#ref-10">10</a></sup>, which encompass any period of potential commonness, to the species ranges in the PlankRange database<sup><a id="xref-ref-30-1" class="xref-bibr" href="#ref-30">30</a></sup> (<a href="http://palaeo.gly.bris.ac.uk/Data/plankrange.html">http://palaeo.gly.bris.ac.uk/Data/plankrange.html</a>, accessed Aug. 24, 2014). After resolving most of the taxonomic discrepancies, ~82 % of the species have a rise-fall interval that fits within the proposed range or is offset by &lt; 2 Myr (a single time bin in ref. 10). Of the remaining species, which could either not be matched taxonomically or have a significant range offset, ~73 % are rare (peak occurrence frequency, as a proportion of all sites with at least one species sampled, &lt; 0.2), and none have a peak occurrence frequency &gt; 0.4. Taxonomic or range errors in NSB are therefore unlikely to have a significant impact on SCOR, which is sensitive only to the most widespread species. Quantification of this impact awaits the public release of the updated PlankRange database<sup><a id="xref-ref-31-1" class="xref-bibr" href="#ref-31">31</a></sup>.</p><p id="p-20">DOT estimates were obtained from <a id="xref-ref-18-3" class="xref-bibr" href="#ref-18">ref. 18</a>, using their &gt; 9 Ma T<sub>d-SL</sub> record (based on subtracting New Jersey sea level records from a benthic δ<sup><a id="xref-ref-18-4" class="xref-bibr" href="#ref-18">18</a></sup>O stack) and scaled δ<sup><a id="xref-ref-18-5" class="xref-bibr" href="#ref-18">18</a></sup>O record for the interval &lt; 9 Ma, with their interpolation at 0.1 Myr resolution. Both SCOR and DOT were tied to the GTS2004 time scale<sup><a id="xref-ref-32-1" class="xref-bibr" href="#ref-32">32</a></sup>.</p></div><div class="section" id="sec-5"><h3>SCOR</h3><p id="p-21">We treat the observation of a specific number of individuals as a Poisson-distributed variable with parameter <em>λ</em> in each time bin. The probability of finding an individual of species <em>i</em> in time bin <em>j</em> is then <em>p</em><sub><em>ij</em></sub> = 1 − exp(−<em>λ<sub>ij</sub></em>), and thus <em>λ</em><sub><em>ij</em></sub> = − ln(1 − <em>p</em><sub><em>ij</em></sub>). In practice, <em>p</em><sub><em>ij</em></sub> is estimated as <em>y</em><sub><em>ij</em></sub>/<em>n</em><sub><em>j</em></sub>, where <em>y</em><sub><em>ij</em></sub> is the number of sites in which species <em>i</em> is observed at time bin <em>j</em> and <em>n</em><sub><em>j</em></sub> is the number of sites in that time bin where at least one of the species included in the analysis is observed. SCOR is the total density of a given set of <em>m</em><sub><em>j</em></sub> species in a time bin:
<span class="disp-formula" id="disp-formula-1"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2016/03/15/043729/embed/graphic-4.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2016/03/15/043729/embed/graphic-4.gif"/></noscript></span></span>
and we estimate its variance by the delta method<sup><a id="xref-ref-33-1" class="xref-bibr" href="#ref-33">33</a></sup>:
<span class="disp-formula" id="disp-formula-2"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2016/03/15/043729/embed/graphic-5.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2016/03/15/043729/embed/graphic-5.gif"/></noscript></span></span></p><p id="p-22">SCOR is based on the observation that the more globally abundant a species is, the more likely it is to occur at a greater number of sites<sup><a id="xref-ref-4-6" class="xref-bibr" href="#ref-4">4</a></sup>. As <em>p</em> approaches 1, the rate of increase in <em>λ</em> grows rapidly, such that very widespread species have a much greater influence on SCOR than restricted species. If a species occurs at all sites in a time bin, its <em>λ</em> for that time bin is undefined.</p><p id="p-23">SCOR is decoupled from species richness and relative abundance. If a species becomes globally more abundant and widespread in a time interval, then its <em>λ</em>, and thus SCOR, will increase. Even if all species became exactly equally more common in absolute terms, with no change in relative abundance, their individual <em>λ</em> values will be higher and SCOR will capture the proportional change in absolute abundance of the total species set.</p></div><div class="section" id="sec-6"><h3>Poseidon simulations</h3><p id="p-24">To evaluate the performance of SCOR relative to commonly used diversity metrics, we designed a set of numerical experiments on the effects of temporal variability in three factors: (1) spatial sampling completeness; (2) the shape of the species rank-abundance distribution (RAD); and (3) the proportion of species lost randomly (with respect to abundance).</p><p id="p-25">Poseidon consists of 1,000 spatial grid cells and 105 time steps, where true species richness and abundance are allowed to vary independently (<a id="xref-fig-1-6" class="xref-fig" href="#F1">Fig. 1a</a>). In each time step, we randomly assign an abundance value to each species, such that the entire community has a log-normal rank-abundance distribution (RAD), the shape of which can be fixed or time-varying. Species ranks are randomly reshuffled between time steps. We then randomly assign a spatial grid cell (site) to an individual of a species, where it can potentially be preserved and sampled.</p><p id="p-26">Next, we sample only a proportion of the sites (spatial cells) such that this proportion increases linearly from 0.1 to 0.4, representing a declining sampling coverage with age, typical of deep-sea sedimentary records. Any short-term variability is thus superimposed on this trend (e.g. <a id="xref-fig-1-7" class="xref-fig" href="#F1">Fig. 1b</a>). Furthermore, a proportion of the remaining species can be randomly removed (representing dissolution, selective picking, or other processes causing a species to be absent in a time bin, regardless of its original abundance).</p><p id="p-27">We then calculate raw S, RT, Shannon H, and three sampling-standardized richness metrics widely used in palaeobiology (CR, O2W, and SQS). Although a generalized OXW has been recommended for paleontological datasets<sup><a id="xref-ref-34-1" class="xref-bibr" href="#ref-34">34</a></sup>, we use O2W here because our data meet the assumptions of the latter<sup><a id="xref-ref-12-2" class="xref-bibr" href="#ref-12">12</a></sup>. We used two versions of the shareholder quorum subsampling (SQS) method<sup><a id="xref-ref-5-4" class="xref-bibr" href="#ref-5">5</a>,<a id="xref-ref-35-1" class="xref-bibr" href="#ref-35">35</a>-<a id="xref-ref-37-1" class="xref-bibr" href="#ref-37">37</a></sup>: the SQS3.3 R script (<a href="http://bio.mq.edu.au/~jalroy/SQS.html">http://bio.mq.edu.au/~jalroy/SQS.html</a>, downloaded Aug. 26, 2014), and the SQS4.3 perl script, kindly provided by J. Alroy (Sept. 3, 2014). We did not modify the SQS codes but wrote a function to format Poseidon output to species occurrence data for SQS4.3. For SQS3.3, we input the list of sampled individuals (abundances). A quorum level of 0.6 was used in all runs. Both SQS versions yielded very similar results when given the same type of data (abundances or occurrences). For all subsampling methods (CR, O2W, and SQS), 100 iterations were used in obtaining each estimated richness value. Increasing the number of iterations offered no discernible improvement. Shannon H was output from the SQS 3.3 R script and used to calculate Pielou's J evenness. The goodness-of-fit between true and estimated richness, and between true abundance and SCOR, was assessed by the coefficient of determination (R<sup><a id="xref-ref-2-3" class="xref-bibr" href="#ref-2">2</a></sup>) between time series. Poseidon R scripts are provided as Supplementary Code.</p></div><div class="section" id="sec-7"><h3>Time Series Analysis</h3><p id="p-28">We used three different time series analysis methods to test for a causal relationship between SCOR and DOT. Two of the methods are non-parametric, while the third is based on linear models. To implement a time-displacement (lag) analysis for the non-parametric methods, missing SCOR values were linearly interpolated, and the 0.1-Myr resolution DOT record was bin-averaged on the SCOR time bins (0.5 Myr). Furthermore, the non-parametric methods use surrogate time series to assess significance, which requires detrending of the original time series. To avoid any bias from differences in non-stationarity that are not reproduced by the surrogates, both records were detrended using a third-order polynomial to satisfy a stationarity criterion<sup><a id="xref-ref-38-1" class="xref-bibr" href="#ref-38">38</a></sup>, then normalized to zero mean and unit standard deviation. For consistency, the model-based analysis was also performed on the preprocessed data. However, the detrending may remove components of the variation relevant to uncovering the parameters of underlying processes. We therefore repeated the model-based analysis on untransformed data.</p></div><div class="section" id="sec-8"><h3>CCM analysis</h3><p id="p-29">CCM is a method for causal inference in nonlinear dynamical systems based on the theory of state-space reconstruction<sup><a id="xref-ref-22-2" class="xref-bibr" href="#ref-22">22</a></sup>. Consider two time series <em>X</em> and <em>Y</em> consisting of scalar observations of variables in a dynamical system. According to Takens's theorem<sup><a id="xref-ref-39-1" class="xref-bibr" href="#ref-39">39</a></sup>, we can construct a delay-coordinate embedding of the state space of the dynamical system into an m-dimensional real space, by constructing the vectors <em>E</em><sub><em>X</em></sub> = { ⟨(<em>x</em>(<em>t</em><sub>i</sub>), <em>x</em>(<em>t</em><sub>i</sub> – τ), <em>x</em>(<em>t</em><sub>i</sub> – 2τ), …, <em>x</em>(<em>t</em><sub>i</sub> – (<em>mτ</em>) ⟩}, where <em>x</em>(<em>t</em><sub>i</sub>) is the scalar value of the time series <em>X</em>at time <em>t</em><sub>i</sub>. That is, the vectors in <em>E</em><sub><em>X</em></sub> are in one-to-one correspondence with the states of the dynamical system. If <em>X</em> and <em>Y</em> are coupled variables of the same dynamical system (i.e. they are causally influencing each other), this correspondence is also true for time series <em>Y</em>, and therefore <em>E</em><sub><em>X</em></sub> and <em>E</em><sub><em>Y</em></sub> are in one-to-one correspondence with each other. CCM uses this result to predict scalar values of <em>Y</em> from the coordinate-delay embedding of <em>X</em> and vice versa.</p><p id="p-30">The CCM algorithm locates, for each scalar point <em>P</em><sub>i</sub> in the prediction set (subset of time series <em>Y</em>), the contemporaneous state vector <em>L</em><sub>i</sub> in the library set (subset of state vectors in the time-delay embedding constructed from time series <em>X</em>). Next, it finds <em>L</em><sub>i</sub>'s closest neighbours and estimates a value for the predictee <em>P</em><sub>i</sub>* using simplex projection<sup><a id="xref-ref-40-1" class="xref-bibr" href="#ref-40">40</a></sup>. CCM skill is determined by the correlation (Pearson's ρ) between estimated <em>P</em><sub>i</sub>* and actual values of <em>P</em><sub>i</sub>.</p><p id="p-31">With increasing library size, CCM skill is expected to increase and converge if the variables are causally related. The notation “<em>X</em> xmap <em>Y</em>” refers to estimating <em>y</em>(<em>t</em><sub>i</sub>) from corresponding lagged-coordinate state versions of <em>x</em>(<em>t</em><sub>i</sub>), which in a causal context is read as “<em>Y</em> is causally influencing <em>X</em>”.</p><p id="p-32">CCM analysis was performed using the rEDM software package<sup><a id="xref-ref-41-1" class="xref-bibr" href="#ref-41">41</a></sup>. We constructed time-delay embeddings using embedding dimension <em>m</em> = 2 and delay time step <em>τ</em> = 1. Cross mapping was then performed using the entire time series as both prediction and library sets. To avoid biased results, we used leave-one-out cross validation (i.e. the predictee <em>P</em><sub>i</sub> itself and points in a time radius of <em>E</em> around <em>P</em><sub>i</sub> were excluded from the libraries, such that no points sharing coordinates with <em>P</em><sub>i</sub> were used in the predictions; see refs. 41, 42).</p><p id="p-33">If unidirectional forcing is sufficiently strong, the dynamics of the response variable can become dominated by the driving variable. In this case, CCM may be significant in both directions, and thus unable to distinguish unidirectional forcing from bidirectional causality. To address this, we used the extended CCM approach <sup><a id="xref-ref-28-2" class="xref-bibr" href="#ref-28">28</a></sup>, which repeats the cross mapping using different time-displacements of the original time series. If there is a discernable lag between cause and effect, then optimal cross map skill is expected to occur for negative time lags in the direction(s) of true causality (past drives future). If true causality is unidirectional, then any CCM skill in the non-causal direction is expected to peak for positive lags (future "drives" past).</p><p id="p-34">Extended CCM analysis of SCOR and DOT is reported as median cross map skill and 95 % ranges at different lags after drawing 500 samples with replacement from libraries of size 100. Statistical significance is evaluated against a null distribution of CCM results for 1000 surrogate time series. For each lag, CCM skill is considered significant if it exceeds the 95<sup>th</sup> percentile of the surrogates. We verified the results using three different methods for generating surrogate data: phase-randomized and amplitude-adjusted Fourier transform (AAFT)<sup><a id="xref-ref-43-1" class="xref-bibr" href="#ref-43">43</a></sup>, phase-randomized Fourier transform<sup><a id="xref-ref-44-1" class="xref-bibr" href="#ref-44">44</a></sup>, and randomly shuffled surrogates. All three methods indicate significant causality from DOT to SCOR, and we limit our results to the AAFT method, which gave the most conservative significance estimates.</p></div><div class="section" id="sec-9"><h3>IT analysis</h3><p id="p-35">If two processes <em>X</em> and <em>Y</em> are independent, then a general Markov property will hold<sup><a id="xref-ref-25-2" class="xref-bibr" href="#ref-25">25</a></sup>:
<span class="disp-formula" id="disp-formula-3"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2016/03/15/043729/embed/graphic-6.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2016/03/15/043729/embed/graphic-6.gif"/></noscript></span></span>
where <em>p(<em>x</em></em><sub><em>n</em>~1</sub>) is the transition probability to state <em>n~</em>1, and indices <em>k</em> and <em>l</em> are the dimensions of vectors of past states. In the absence of information flow from <em>Y</em> to <em>X</em>, knowing the past <em>l</em> states of <em>Y</em> has no influence on the transition probability of <em>X</em> beyond knowing the past <em>k</em> states of <em>X</em> alone. Transfer entropy<sup><a id="xref-ref-25-3" class="xref-bibr" href="#ref-25">25</a></sup> quantifies the incorrectness of assuming independence by means of a Kullback-Leibler divergence, a non-symmetric measure of the information lost when the right hand side is used to approximate the left hand side of equation (3):
<span class="disp-formula" id="disp-formula-4"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2016/03/15/043729/embed/graphic-7.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2016/03/15/043729/embed/graphic-7.gif"/></noscript></span></span></p><p id="p-36">Transfer entropy is thus a non-symmetric measure of information flow, which has been shown to be equivalent to a conditional mutual information<sup><a id="xref-ref-45-1" class="xref-bibr" href="#ref-45">45</a></sup>, and equivalent to Granger causality<sup><a id="xref-ref-46-1" class="xref-bibr" href="#ref-46">46</a></sup> for linear, Gaussian systems<sup><a id="xref-ref-47-1" class="xref-bibr" href="#ref-47">47</a></sup>. We implement it in the modified IT form proposed by Verdes<sup><a id="xref-ref-23-2" class="xref-bibr" href="#ref-23">23</a></sup>, which has previously been applied to the analysis of geological records<sup><a id="xref-ref-11-4" class="xref-bibr" href="#ref-11">11</a>,<a id="xref-ref-24-2" class="xref-bibr" href="#ref-24">24</a>,<a id="xref-ref-48-1" class="xref-bibr" href="#ref-48">48</a>,<a id="xref-ref-49-1" class="xref-bibr" href="#ref-49">49</a></sup>.</p><p id="p-37">Here we expand on these earlier applications by repeating the IT analysis for different time-displacements of the original time series, analogous to the extended CCM analysis described above. Similar to CCM, predictive information flow may become symmetric if unidirectional forcing and/or linear correlation is sufficiently strong. However, if there is a discernable lag between cause and effect, then optimal information transfer is expected to occur for negative time lags in the direction(s) of true causality (past → future). If true causality is unidirectional, then any information flow in the non-causal direction is expected to peak for positive lags (future → past). IT is a coarse-grained relative entropy measure, which varies as a function of the data gridding resolution, summarized in a single IT value as the area under the resulting curve<sup><a id="xref-ref-23-3" class="xref-bibr" href="#ref-23">23</a></sup>. Lagged IT analysis of SCOR and DOT is reported as median IT and 95 % ranges at different lags after drawing 500 random subsamples of size 100. IT is considered significant if it exceeds the 95<sup>th</sup> percentile of a null distribution of IT results for 1,000 AAFT surrogate time series. Unlike CCM, this IT implementation does not use time-delay embedding. Combined with coarse-graining of the dat<strong>a</strong>, this may help explain the difference in optimal lag between IT and CCM (<a id="xref-fig-3-5" class="xref-fig" href="#F3">Fig. 3b,c</a>), although more work is needed to clarify this.</p></div><div class="section" id="sec-10"><h3>Linear SDE analysis</h3><p id="p-38">Given two time series representing two measured processes, linear SDEs can be used to distinguish between correlation and Granger causality. Uni-and bidirectional causation as well as hidden (unmeasured) processes can be modelled in the SDE framework, expanding the space of possible connections <sup><a id="xref-ref-26-2" class="xref-bibr" href="#ref-26">26</a>,<a id="xref-ref-27-2" class="xref-bibr" href="#ref-27">27</a></sup> A basic linear SDE can be written as
<span class="disp-formula" id="disp-formula-5"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2016/03/15/043729/embed/graphic-8.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2016/03/15/043729/embed/graphic-8.gif"/></noscript></span></span></p><p id="p-39">This describes a mean-reverting Ornstein-Uhlenbeck process (OUP) <em>X</em>, which contains a systematic part (the <em>dt</em> term) and a stochastic part (<em>t</em>he <em>dB</em> term). If the systematic part is dropped (<em>α</em><sub><em>X</em></sub> = 0), then equation (5) describes a Wiener process (WP, or random walk). The OUP has expectation <em>μ</em><sub><em>X</em></sub>, stationary standard deviation <span class="inline-formula" id="inline-formula-1"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2016/03/15/043729/embed/inline-graphic-1.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2016/03/15/043729/embed/inline-graphic-1.gif"/></noscript></span></span> and half-life <em>t</em><sub>1/2,<em>X</em></sub> = log(2)/ <em>α</em><sub><em>X</em></sub>. To model a hidden process, we can write
<span class="disp-formula" id="disp-formula-6"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2016/03/15/043729/embed/graphic-9.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2016/03/15/043729/embed/graphic-9.gif"/></noscript></span></span></p><p id="p-40">Here, the measured process <em>Y</em><sub><em>1</em></sub> has a hidden process (or layer) <em>Y</em><sub><em>2</em></sub> folded into its systematic part, such that <em>Y</em><sub><em>1</em></sub> tracks <em>Y</em><sub>2</sub>. <em>Y</em><sub><em>1</em></sub> is similar to an OUP, but instead of fluctuating around a fixed expected value it fluctuates with a lagged response to the OUP <em>Y</em><sub>2</sub>.</p><p id="p-41">When modelling connections between processes, we use vector notation. A pure correlation between <em>X</em> and <em>Y</em> entails that the covariance matrix in front of the stochastic term <em>d<span class="underline">B</span></em> will have off-diagonal elements. If there is a causal connection from <em>Y</em><sub><em>2</em></sub> to <em>X</em>, for instance, the system takes the following form
<span class="disp-formula" id="disp-formula-7"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2016/03/15/043729/embed/graphic-10.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2016/03/15/043729/embed/graphic-10.gif"/></noscript></span></span>
where <em>β</em><sub><em>Y</em><sub>2</sub></sub> → <em>X</em> describes the connection strength from <em>Y</em><sub><em>2</em></sub> to <em>X.</em> Equation (7) describes a “common cause” situation, where <em>Y</em><sub><em>2</em></sub> drives both <em>X</em> and <em>Y</em><sub><em>1</em></sub>.</p><p id="p-42">To analyse the SCOR and DOT records, we first characterized each time series separately, examining models with up to three layers (two hidden). In each model, the layers could be WP or OUP (including fully deterministic layers where <em>σ</em><sub><em>i</em></sub> = 0), excluding a one-layered WP, which prohibits incoming causal links. We also excluded internal feedback loops in multi-layer models because of numerical intractability. For both time series, <em>μ</em> was assigned a prior distribution <em>μ</em><sub><em>i</em></sub> ~ <em>N</em>(0,1), where <em>i</em> denotes the layer. All model parameters were assigned normal priors, with 95 % prior probability ranges of <em>σ</em><sub><em>i</em></sub> ∈ (0.01,1.0) for the stochastic term, <em>t</em><sub>1/2</sub> ∈ (0.1<em>My</em>,50<em>My</em>) for the half life, <em>β</em>∈ (-2,2) for the causal connections, and <em>ρ</em>∈ (-0.96,0.96), for the logit-transformed correlation coefficients. We used MCMC importance sampling to estimate Bayesian model likelihoods and calculate model probabilities.</p><p id="p-43">The best model for SCOR in isolation was a one-layered OUP, while a three-layered model with a WP as the bottom driver was the best model for DOT in isolation. We then investigated all 15 connection models between these two best models, including the null hypothesis of no relationship (<a id="xref-fig-7-2" class="xref-fig" href="#F7">Supplementary Fig. 4</a>). We allowed for causality from SCOR to DOT because both proxy records ultimately derive from deep-sea carbonate sediments, hence SCOR could in principle contain a signal of processes that have influenced DOT. The null hypothesis was assigned 50 % prior probability, while 50 % was distributed evenly among the 14 connection models. For model comparison, we used Jeffreys’s scale to assess the strength of evidence represented by the Bayes factor B, where <span class="inline-formula" id="inline-formula-2"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2016/03/15/043729/embed/inline-graphic-2.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2016/03/15/043729/embed/inline-graphic-2.gif"/></noscript></span></span> is evidence “barely worth mentioning”, <span class="inline-formula" id="inline-formula-3"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2016/03/15/043729/embed/inline-graphic-3.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2016/03/15/043729/embed/inline-graphic-3.gif"/></noscript></span></span> is “substantial evidence”, 10 &lt; <em>B</em> &lt; 10<sup>3/2</sup> is “strong” evidence and <em>B</em> &lt; 10<sup>3/2</sup> is “very strong” evidence<sup><a id="xref-ref-50-1" class="xref-bibr" href="#ref-50">50</a></sup>.</p><p id="p-44">The posterior probability of the null hypothesis was 11.2 % (Model 1; <a id="xref-fig-7-3" class="xref-fig" href="#F7">Supplementary Fig. 4</a>), hence the Bayes factor favouring a connection between SCOR and DOT is 7.9 (substantial evidence). The most probable model (Model 5; <a id="xref-fig-7-4" class="xref-fig" href="#F7">Supplementary Fig. 4</a>) involves a feedback loop, where the upper DOT process (DOT1) affects SCOR positively while SCOR affects DOT1 negatively. All parameter estimates with credible intervals for the best model are presented in <a id="xref-table-wrap-1-2" class="xref-table" href="#T1">Supplementary Table 1</a>. In the second most (and almost equally) probable model, (Model 12; <a id="xref-fig-7-5" class="xref-fig" href="#F7">Supplementary Fig. 4</a>), SCOR affects the second DOT layer (DOT2) instead.</p><p id="p-45">With a half-life of ~0.5 Myr, SCOR responds to DOT processes on time scales comparable to those inferred from the other analyses (<a id="xref-fig-3-6" class="xref-fig" href="#F3">Fig. 3b,c</a>). In contrast, DOT processes react very slowly to changes in SCOR (<a id="xref-table-wrap-1-3" class="xref-table" href="#T1">Supplementary Table 1</a>), and because SCOR changes rapidly, the response in DOT will be smoothed out. From equation 5, the effect of SCOR on DOT1 is 0.07 Myr<sup>-1</sup>, while the effect of DOT1 on the SCOR process is 0.6 Myr<sup>-1</sup>. Thus, DOT influences SCOR much more strongly per time unit than vice versa.</p><p id="p-46">The third most likely model (Model 4; <a id="xref-fig-7-6" class="xref-fig" href="#F7">Supplementary Fig. 4</a>) only had a causal connection from DOT1 to SCOR, consistent with CCM and IT inferences. This model had a posterior probability of 17.5 %, hence the evidence for a feedback loop is less than “substantial" (B = 2.6). In summary, we find evidence for there being at least one connection (B = 7.9); for the connections to be causal rather than correlative given that there are connections (B = 12.5); and specifically for a causal connection from DOT to SCOR given that there are connections (B = 15.4).</p><p id="p-47">We then repeated the analysis on untransformed data (not detrended or normalized), denoted uDOT and uSCOR. In this case, the best isolated model for both time series is a three-layer model with a WP at the bottom. The Bayes factor favouring a connection over no connection is 73, which is deemed “very strong evidence”. The best connection model involves a feedback loop between the top layers uDOT1 and uSCOR1. However, there is a very high probability for parameters enforcing cyclical behaviour, with a period of 1.5 Myr, which is consistent with an internal feedback loop model as the best isolated model for uDOT. Parameter estimates are shown in <a id="xref-table-wrap-2-2" class="xref-table" href="#T2">Supplementary Table 2</a>.</p></div></div><div class="section ack" id="ack-1"><h2 class="">Acknowledgments</h2><p id="p-16">We thank the contributors to the Neptune Sandbox Berlin database. Thanks to J. Alroy for sharing his SQS 4.3 script. This project used DSDP/ODP samples provided by the Integrated Ocean Drilling Program (IODP). IODP is sponsored by the U.S. National Science Foundation (NSF) and participating countries under the management of Joint Oceanographic Institutions (JOI), Inc. The authors were funded by the Norwegian Research Council (231259 to B.H. and 235073 to L.H.L.) and the Bergen Research Foundation (B.H. and K.A.H.).</p></div><div class="section ref-list" id="ref-list-1"><h2 class="">References</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label">1.</span><a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference 1. in text" id="ref-1">↵</a><div class="cit ref-cit ref-journal" id="cit-043729v1.1" data-doi="10.2307/1930989"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Preston</span>,  <span class="cit-name-given-names">F. W.</span></span> <span class="cit-article-title">The commonness, and rarity, of species</span>. <abbr class="cit-jnl-abbrev">Ecology</abbr> <span class="cit-vol">29</span>, <span class="cit-fpage">254</span>–<span class="cit-lpage">283</span> (<span class="cit-pub-date">1948</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEcology%26rft.volume%253D29%26rft.spage%253D254%26rft_id%253Dinfo%253Adoi%252F10.2307%252F1930989%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2307/1930989&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=A1948XW59500003&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">2.</span><a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference 2. in text" id="ref-2">↵</a><div class="cit ref-cit ref-journal" id="cit-043729v1.2" data-doi="10.1073/pnas.1406664111"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Connolly</span>,  <span class="cit-name-given-names">S. R.</span></span> <span class="cit-etal">et al.</span> <span class="cit-article-title">Commonness and rarity in the marine biosphere</span>. <abbr class="cit-jnl-abbrev">Proceedings of the National Academy of Sciences</abbr> <span class="cit-vol">111</span>, <span class="cit-fpage">8524</span>–<span class="cit-lpage">8529</span> (<span class="cit-pub-date">2014</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProceedings%2Bof%2Bthe%2BNational%2BAcademy%2Bof%2BSciences%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.1406664111%26rft_id%253Dinfo%253Apmid%252F24912168%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMToiMTExLzIzLzg1MjQiO3M6NDoiYXRvbSI7czozNzoiL2Jpb3J4aXYvZWFybHkvMjAxNi8wMy8xNS8wNDM3MjkuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">3.</span><a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference 3. in text" id="ref-3">↵</a><div class="cit ref-cit ref-journal" id="cit-043729v1.3" data-doi="10.1126/science.1243092"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">ter Steege</span>,  <span class="cit-name-given-names">H.</span></span> <span class="cit-etal">et al.</span> <span class="cit-article-title">Hyperdominance in the Amazonian Tree Flora</span>. <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">342</span>, <span class="cit-fpage">1243092</span> (<span class="cit-pub-date">2013</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253Dter%2BSteege%26rft.auinit1%253DH.%26rft.volume%253D342%26rft.issue%253D6156%26rft.spage%253D1243092%26rft.epage%253D1243092%26rft.atitle%253DHyperdominance%2Bin%2Bthe%2BAmazonian%2BTree%2BFlora%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1243092%26rft_id%253Dinfo%253Apmid%252F24136971%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjE2OiIzNDIvNjE1Ni8xMjQzMDkyIjtzOjQ6ImF0b20iO3M6Mzc6Ii9iaW9yeGl2L2Vhcmx5LzIwMTYvMDMvMTUvMDQzNzI5LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">4.</span><a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference 4. in text" id="ref-4">↵</a><div class="cit ref-cit ref-journal" id="cit-043729v1.4" data-doi="10.1126/science.1261605"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">de Vargas</span>,  <span class="cit-name-given-names">C.</span></span> <span class="cit-etal">et al.</span> <span class="cit-article-title">Eukaryotic plankton diversity in the sunlit ocean</span>. <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">348</span>, <span class="cit-fpage">1261605</span> (<span class="cit-pub-date">2015</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253Dde%2BVargas%26rft.auinit1%253DC.%26rft.volume%253D348%26rft.issue%253D6237%26rft.spage%253D1261605%26rft.epage%253D1261605%26rft.atitle%253DEukaryotic%2Bplankton%2Bdiversity%2Bin%2Bthe%2Bsunlit%2Bocean%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1261605%26rft_id%253Dinfo%253Apmid%252F25999516%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjE2OiIzNDgvNjIzNy8xMjYxNjA1IjtzOjQ6ImF0b20iO3M6Mzc6Ii9iaW9yeGl2L2Vhcmx5LzIwMTYvMDMvMTUvMDQzNzI5LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">5.</span><a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference 5. in text" id="ref-5">↵</a><div class="cit ref-cit ref-journal" id="cit-043729v1.5" data-doi="10.1126/science.1189910"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Alroy</span>,  <span class="cit-name-given-names">J.</span></span> <span class="cit-article-title">The Shifting Balance of Diversity Among Major Marine Animal Groups</span>. <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">329</span>, <span class="cit-fpage">1191</span>–<span class="cit-lpage">1194</span> (<span class="cit-pub-date">2010</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DAlroy%26rft.auinit1%253DJ.%26rft.volume%253D329%26rft.issue%253D5996%26rft.spage%253D1191%26rft.epage%253D1194%26rft.atitle%253DThe%2BShifting%2BBalance%2Bof%2BDiversity%2BAmong%2BMajor%2BMarine%2BAnimal%2BGroups%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1189910%26rft_id%253Dinfo%253Apmid%252F20813951%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEzOiIzMjkvNTk5Ni8xMTkxIjtzOjQ6ImF0b20iO3M6Mzc6Ii9iaW9yeGl2L2Vhcmx5LzIwMTYvMDMvMTUvMDQzNzI5LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">6.</span><a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference 6. in text" id="ref-6">↵</a><div class="cit ref-cit ref-journal" id="cit-043729v1.6" data-doi="10.1126/science.1203060"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Ezard</span>,  <span class="cit-name-given-names">T. H. G.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Aze</span>,  <span class="cit-name-given-names">T.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Pearson</span>,  <span class="cit-name-given-names">P. N.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Purvis</span>,  <span class="cit-name-given-names">A.</span></span> <span class="cit-article-title">Interplay between changing climate and species’ ecology drives macroevolutionary dynamics</span>. <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">332</span>, <span class="cit-fpage">349</span>–<span class="cit-lpage">351</span> (<span class="cit-pub-date">2011</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DEzard%26rft.auinit1%253DT.%2BH.%2BG.%26rft.volume%253D332%26rft.issue%253D6027%26rft.spage%253D349%26rft.epage%253D351%26rft.atitle%253DInterplay%2BBetween%2BChanging%2BClimate%2Band%2BSpecies%2527%2BEcology%2BDrives%2BMacroevolutionary%2BDynamics%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1203060%26rft_id%253Dinfo%253Apmid%252F21493859%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEyOiIzMzIvNjAyNy8zNDkiO3M6NDoiYXRvbSI7czozNzoiL2Jpb3J4aXYvZWFybHkvMjAxNi8wMy8xNS8wNDM3MjkuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">7.</span><a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference 7. in text" id="ref-7">↵</a><div class="cit ref-cit ref-journal" id="cit-043729v1.7" data-doi="10.1046/j.1365-2664.2000.00485.x"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Gaston</span>,  <span class="cit-name-given-names">K. J.</span></span> <span class="cit-etal">et al.</span> <span class="cit-article-title">Abundance–occupancy relationships</span>. <abbr class="cit-jnl-abbrev">Journal of Applied Ecology</abbr> <span class="cit-vol">37</span>, <span class="cit-fpage">39</span>–<span class="cit-lpage">59</span>(<span class="cit-pub-date">2000</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BApplied%2BEcology%26rft.volume%253D37%26rft.spage%253D39%26rft_id%253Dinfo%253Adoi%252F10.1046%252Fj.1365-2664.2000.00485.x%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1046/j.1365-2664.2000.00485.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000089917500005&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">8.</span><a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference 8. in text" id="ref-8">↵</a><div class="cit ref-cit ref-journal" id="cit-043729v1.8" data-doi="10.1666/06084.1"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Foote</span>,  <span class="cit-name-given-names">M.</span></span> <span class="cit-article-title">Symmetric waxing and waning of marine invertebrate genera</span>. <abbr class="cit-jnl-abbrev">Paleobiology</abbr> <span class="cit-vol">33</span>, <span class="cit-fpage">517</span>–<span class="cit-lpage">529</span>(<span class="cit-pub-date">2007</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPaleobiology%26rft.stitle%253DPaleobiology%26rft.aulast%253DFoote%26rft.auinit1%253DM.%26rft.volume%253D33%26rft.issue%253D4%26rft.spage%253D517%26rft.epage%253D529%26rft.atitle%253DSymmetric%2Bwaxing%2Band%2Bwaning%2Bof%2Bmarine%2Binvertebrate%2Bgenera%26rft_id%253Dinfo%253Adoi%252F10.1666%252F06084.1%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MTA6ImdzcGFsZW9iaW8iO3M6NToicmVzaWQiO3M6ODoiMzMvNC81MTciO3M6NDoiYXRvbSI7czozNzoiL2Jpb3J4aXYvZWFybHkvMjAxNi8wMy8xNS8wNDM3MjkuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">9.</span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-043729v1.9" data-doi="10.1098/rspb.2007.1006"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Liow</span>,  <span class="cit-name-given-names">L. H.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Stenseth</span>,  <span class="cit-name-given-names">N. C.</span></span> <span class="cit-article-title">The rise and fall of species: implications for macroevolutionary and macroecological studies</span>. <abbr class="cit-jnl-abbrev">Proceedings of the Royal Society B: Biological Sciences</abbr> <span class="cit-vol">274</span>, <span class="cit-fpage">2745</span>–<span class="cit-lpage">2752</span> (<span class="cit-pub-date">2007</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProceedings%2Bof%2Bthe%2BRoyal%2BSociety%2BB%253A%2BBiological%2BSciences%26rft.stitle%253DProc%2BR%2BSoc%2BB%26rft.aulast%253DLiow%26rft.auinit1%253DL.%2BH.%26rft.volume%253D274%26rft.issue%253D1626%26rft.spage%253D2745%26rft.epage%253D2752%26rft.atitle%253DThe%2Brise%2Band%2Bfall%2Bof%2Bspecies%253A%2Bimplications%2Bfor%2Bmacroevolutionary%2Band%2Bmacroecological%2Bstudies%26rft_id%253Dinfo%253Adoi%252F10.1098%252Frspb.2007.1006%26rft_id%253Dinfo%253Apmid%252F17711843%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1098/rspb.2007.1006&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=2013054829&amp;link_type=GEOREF" class="cit-ref-sprinkles cit-ref-sprinkles-georef"><span>GeoRef</span></a><a href="/lookup/external-ref?access_num=17711843&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2016%2F03%2F15%2F043729.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000249557900014&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">10.</span><a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference 10. in text" id="ref-10">↵</a><div class="cit ref-cit ref-journal" id="cit-043729v1.10" data-doi="10.1666/08080.1"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Liow</span>,  <span class="cit-name-given-names">L. H.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Skaug</span>,  <span class="cit-name-given-names">H. J.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Ergon</span>,  <span class="cit-name-given-names">T.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Schweder</span>,  <span class="cit-name-given-names">T.</span></span> <span class="cit-article-title">Global occurrence trajectories of microfossils: environmental volatility and the rise and fall of individual species</span>. <abbr class="cit-jnl-abbrev">Paleobiology</abbr> <span class="cit-vol">36</span>, <span class="cit-fpage">224</span>–<span class="cit-lpage">252</span> (<span class="cit-pub-date">2010</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPaleobiology%26rft.stitle%253DPaleobiology%26rft.aulast%253DLiow%26rft.auinit1%253DL.%2BH.%26rft.volume%253D36%26rft.issue%253D2%26rft.spage%253D224%26rft.epage%253D252%26rft.atitle%253DGlobal%2Boccurrence%2Btrajectories%2Bof%2Bmicrofossils%253A%2Benvironmental%2Bvolatility%2Band%2Bthe%2Brise%2Band%2Bfall%2Bof%2Bindividual%2Bspecies%26rft_id%253Dinfo%253Adoi%252F10.1666%252F08080.1%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MTA6ImdzcGFsZW9iaW8iO3M6NToicmVzaWQiO3M6ODoiMzYvMi8yMjQiO3M6NDoiYXRvbSI7czozNzoiL2Jpb3J4aXYvZWFybHkvMjAxNi8wMy8xNS8wNDM3MjkuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">11.</span><a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference 11. in text" id="ref-11">↵</a><div class="cit ref-cit ref-journal" id="cit-043729v1.11"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Hannisdal</span>,  <span class="cit-name-given-names">B.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Henderiks</span>,  <span class="cit-name-given-names">J.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Liow</span>,  <span class="cit-name-given-names">L. H.</span></span> <span class="cit-article-title">Long-term evolutionary and ecological responses of calcifying phytoplankton to changes in atmospheric CO<sub>2</sub></span>. <abbr class="cit-jnl-abbrev">Glob Change Biol</abbr> <span class="cit-vol">18</span>, <span class="cit-fpage">3504</span>–<span class="cit-lpage">3516</span> (<span class="cit-pub-date">2012</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGlob%2BChange%2BBiol%26rft.volume%253D18%26rft.spage%253D3504%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">12.</span><a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference 12. in text" id="ref-12">↵</a><div class="cit ref-cit ref-journal" id="cit-043729v1.12" data-doi="10.1666/0094-8373(2000)026&lt;0707:NMFQMP&gt;2.0.CO;2"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Alroy</span>,  <span class="cit-name-given-names">J.</span></span> <span class="cit-article-title">New methods for quantifying macroevolutionary patterns and processes</span>. <abbr class="cit-jnl-abbrev">Paleobiology</abbr> <span class="cit-vol">26</span>, <span class="cit-fpage">707</span>–<span class="cit-lpage">733</span> (<span class="cit-pub-date">2000</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPaleobiology%26rft.stitle%253DPaleobiology%26rft.volume%253D26%26rft.issue%253D4%26rft.spage%253D707%26rft.epage%253D733%26rft.atitle%253DNew%2Bmethods%2Bfor%2Bquantifying%2Bmacroevolutionary%2Bpatterns%2Band%2Bprocesses%26rft_id%253Dinfo%253Adoi%252F10.1666%252F0094-8373%25282000%2529026%253C0707%253ANMFQMP%253E2.0.CO%253B2%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MTA6ImdzcGFsZW9iaW8iO3M6NToicmVzaWQiO3M6ODoiMjYvNC83MDciO3M6NDoiYXRvbSI7czozNzoiL2Jpb3J4aXYvZWFybHkvMjAxNi8wMy8xNS8wNDM3MjkuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">13.</span><a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference 13. in text" id="ref-13">↵</a><div class="cit ref-cit ref-journal" id="cit-043729v1.13"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Spencer-Cervato</span>,  <span class="cit-name-given-names">C.</span></span> <span class="cit-article-title">The Cenozoic deep sea microfossil record: explorations of the DSDP/ODP sample set using the Neptune database</span>. <abbr class="cit-jnl-abbrev">Palaeontologia Electronica</abbr> <span class="cit-vol">42</span>, <span class="cit-fpage">1268</span> (<span class="cit-pub-date">1999</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPalaeontologia%2BElectronica%26rft.volume%253D42%26rft.spage%253D1268%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">14.</span><a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference 14. in text" id="ref-14">↵</a><div class="cit ref-cit ref-journal" id="cit-043729v1.14" data-doi="10.1007/BF02083119"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Lazarus</span>,  <span class="cit-name-given-names">D.</span></span> <span class="cit-article-title">Neptune: A marine micropaleontology database</span>. <abbr class="cit-jnl-abbrev">Math Geol</abbr> <span class="cit-vol">26</span>, <span class="cit-fpage">817</span>–<span class="cit-lpage">832</span> (<span class="cit-pub-date">1994</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMath%2BGeol%26rft.volume%253D26%26rft.spage%253D817%26rft_id%253Dinfo%253Adoi%252F10.1007%252FBF02083119%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/BF02083119&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label">15.</span><a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference 15. in text" id="ref-15">↵</a><div class="cit ref-cit ref-journal" id="cit-043729v1.15" data-doi="10.1126/science.aaa8026"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Rivero-Calle</span>,  <span class="cit-name-given-names">S.</span></span>., <span class="cit-auth"><span class="cit-name-surname">Gnanadesikan</span>,  <span class="cit-name-given-names">A.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Del Castillo</span>,  <span class="cit-name-given-names">C. E.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Balch</span>,  <span class="cit-name-given-names">W. M.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Guikema</span>,  <span class="cit-name-given-names">S. D.</span></span> <span class="cit-article-title">Multidecadal increase in North Atlantic coccolithophores and the potential role of rising CO<sub>2</sub></span>. <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">350</span>, <span class="cit-fpage">1533</span>–<span class="cit-lpage">1537</span> (<span class="cit-pub-date">2015</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.aaa8026%26rft_id%253Dinfo%253Apmid%252F26612836%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEzOiIzNTAvNjI2Ny8xNTMzIjtzOjQ6ImF0b20iO3M6Mzc6Ii9iaW9yeGl2L2Vhcmx5LzIwMTYvMDMvMTUvMDQzNzI5LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">16.</span><a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference 16. in text" id="ref-16">↵</a><div class="cit ref-cit ref-journal" id="cit-043729v1.16" data-doi="10.1016/j.epsl.2005.03.006"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Ridgwell</span>,  <span class="cit-name-given-names">A.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Zeebe</span>,  <span class="cit-name-given-names">R. E.</span></span> <span class="cit-article-title">The role of the global carbonate cycle in the regulation and evolution of the Earth system</span>. <abbr class="cit-jnl-abbrev">Earth and Planetary Science Letters</abbr> <span class="cit-vol">234</span>, <span class="cit-fpage">299</span>–<span class="cit-lpage">315</span> (<span class="cit-pub-date">2005</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEarth%2Band%2BPlanetary%2BScience%2BLetters%26rft.stitle%253DEarth%2Band%2BPlanetary%2BScience%2BLetters%26rft.volume%253D234%26rft.issue%253D3-4%26rft.spage%253D299%26rft.epage%253D315%26rft.atitle%253DThe%2Brole%2Bof%2Bthe%2Bglobal%2Bcarbonate%2Bcycle%2Bin%2Bthe%2Bregulation%2Band%2Bevolution%2Bof%2Bthe%2BEarth%2Bsystem%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.epsl.2005.03.006%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.epsl.2005.03.006&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=2006034279&amp;link_type=GEOREF" class="cit-ref-sprinkles cit-ref-sprinkles-georef"><span>GeoRef</span></a><a href="/lookup/external-ref?access_num=000230071200001&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">17.</span><a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference 17. in text" id="ref-17">↵</a><div class="cit ref-cit ref-journal" id="cit-043729v1.17" data-doi="10.1038/nature11360"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Palike</span>,  <span class="cit-name-given-names">H.</span></span> <span class="cit-etal">et al.</span> <span class="cit-article-title">A Cenozoic record of the equatorial Pacific carbonate compensation depth</span>. <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">488</span>, <span class="cit-fpage">609</span>–<span class="cit-lpage">614</span> (<span class="cit-pub-date">2012</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DPalike%26rft.auinit1%253DH.%26rft.volume%253D488%26rft.issue%253D7413%26rft.spage%253D609%26rft.epage%253D614%26rft.atitle%253DA%2BCenozoic%2Brecord%2Bof%2Bthe%2BEquatorial%2BPacific%2Bcarbonate%2Bcompensation%2Bdepth%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature11360%26rft_id%253Dinfo%253Apmid%252F22932385%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature11360&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=2012086252&amp;link_type=GEOREF" class="cit-ref-sprinkles cit-ref-sprinkles-georef"><span>GeoRef</span></a><a href="/lookup/external-ref?access_num=22932385&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2016%2F03%2F15%2F043729.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000308095100047&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">18.</span><a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference 18. in text" id="ref-18">↵</a><div class="cit ref-cit ref-journal" id="cit-043729v1.18"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Cramer</span>,  <span class="cit-name-given-names">B. S.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Miller</span>,  <span class="cit-name-given-names">K. G.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Barrett</span>,  <span class="cit-name-given-names">P. J.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Wright</span>,  <span class="cit-name-given-names">J. D.</span></span> <span class="cit-article-title">Late Cretaceous–Neogene trends in deep ocean temperature and continental ice volume: Reconciling records of benthic foraminiferal geochemistry ( δ18O and Mg/Ca) with sea level history</span>. <abbr class="cit-jnl-abbrev">J. Geophys. Res.</abbr> <span class="cit-vol">116</span>, <span class="cit-fpage">C12023</span> (<span class="cit-pub-date">2011</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BGeophys.%2BRes.%26rft.volume%253D116%26rft.spage%253D12023C%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">19.</span><a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference 19. in text" id="ref-19">↵</a><div class="cit ref-cit ref-journal" id="cit-043729v1.19" data-doi="10.2307/2412601"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Cifelli</span>,  <span class="cit-name-given-names">R.</span></span> <span class="cit-article-title">Radiation of Cenozoic Planktonic Foraminifera</span>. <abbr class="cit-jnl-abbrev">Syst Biol</abbr> <span class="cit-vol">18</span>, <span class="cit-fpage">154</span>–<span class="cit-lpage">168</span> (<span class="cit-pub-date">1969</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DSyst%2BBiol%26rft_id%253Dinfo%253Adoi%252F10.2307%252F2412601%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2307/2412601&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=1969032263&amp;link_type=GEOREF" class="cit-ref-sprinkles cit-ref-sprinkles-georef"><span>GeoRef</span></a></div></div></li><li><span class="ref-label">20.</span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-043729v1.20" data-doi="10.1126/science.1090592"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Schmidt</span>,  <span class="cit-name-given-names">D. N.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Thierstein</span>,  <span class="cit-name-given-names">H. R.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Bollmann</span>,  <span class="cit-name-given-names">J.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Schiebel</span>,  <span class="cit-name-given-names">R.</span></span> <span class="cit-article-title">Abiotic forcing of plankton evolution in the Cenozoic</span>. <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">303</span>, <span class="cit-fpage">207</span>–<span class="cit-lpage">210</span> (<span class="cit-pub-date">2004</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DSchmidt%26rft.auinit1%253DD.%2BN.%26rft.volume%253D303%26rft.issue%253D5655%26rft.spage%253D207%26rft.epage%253D210%26rft.atitle%253DAbiotic%2BForcing%2Bof%2BPlankton%2BEvolution%2Bin%2Bthe%2BCenozoic%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1090592%26rft_id%253Dinfo%253Apmid%252F14716007%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEyOiIzMDMvNTY1NS8yMDciO3M6NDoiYXRvbSI7czozNzoiL2Jpb3J4aXYvZWFybHkvMjAxNi8wMy8xNS8wNDM3MjkuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">21.</span><a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference 21. in text" id="ref-21">↵</a><div class="cit ref-cit ref-journal" id="cit-043729v1.21" data-doi="10.1038/nature11815"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Peters</span>,  <span class="cit-name-given-names">S. E.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Kelly</span>,  <span class="cit-name-given-names">D. C.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Fraass</span>,  <span class="cit-name-given-names">A. J.</span></span> <span class="cit-article-title">Oceanographic controls on the diversity and extinction of planktonic foraminifera</span>. <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">493</span>, <span class="cit-fpage">398</span>–<span class="cit-lpage">401</span> (<span class="cit-pub-date">2013</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DPeters%26rft.auinit1%253DS.%2BE.%26rft.volume%253D493%26rft.issue%253D7432%26rft.spage%253D398%26rft.epage%253D401%26rft.atitle%253DOceanographic%2Bcontrols%2Bon%2Bthe%2Bdiversity%2Band%2Bextinction%2Bof%2Bplanktonic%2BForaminifera%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature11815%26rft_id%253Dinfo%253Apmid%252F23302802%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature11815&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=2013028513&amp;link_type=GEOREF" class="cit-ref-sprinkles cit-ref-sprinkles-georef"><span>GeoRef</span></a><a href="/lookup/external-ref?access_num=23302802&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2016%2F03%2F15%2F043729.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000313615900050&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">22.</span><a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference 22. in text" id="ref-22">↵</a><div class="cit ref-cit ref-journal" id="cit-043729v1.22" data-doi="10.1126/science.1227079"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Sugihara</span>,  <span class="cit-name-given-names">G.</span></span> <span class="cit-etal">et al.</span> <span class="cit-article-title">Detecting Causality in Complex Ecosystems</span>. <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">338</span>, <span class="cit-fpage">496</span>–<span class="cit-lpage">500</span> (<span class="cit-pub-date">2012</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1227079%26rft_id%253Dinfo%253Apmid%252F22997134%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEyOiIzMzgvNjEwNi80OTYiO3M6NDoiYXRvbSI7czozNzoiL2Jpb3J4aXYvZWFybHkvMjAxNi8wMy8xNS8wNDM3MjkuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">23.</span><a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference 23. in text" id="ref-23">↵</a><div class="cit ref-cit ref-journal" id="cit-043729v1.23"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Verdes</span>,  <span class="cit-name-given-names">P.</span></span> <span class="cit-article-title">Assessing causality from multivariate time series</span>. <abbr class="cit-jnl-abbrev">Phys. Rev. E</abbr> <span class="cit-vol">72</span>, <span class="cit-fpage">026222</span> (<span class="cit-pub-date">2005</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPhys.%2BRev.%2BE%26rft.volume%253D72%26rft.spage%253D026222%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">24.</span><a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference 24. in text" id="ref-24">↵</a><div class="cit ref-cit ref-journal" id="cit-043729v1.24" data-doi="10.2475/04.2011.02"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Hannisdal</span>,  <span class="cit-name-given-names">B.</span></span> <span class="cit-article-title">Non-parametric inference of causal interactions from geological records</span>. <abbr class="cit-jnl-abbrev">American Journal of Science</abbr> <span class="cit-vol">311</span>, <span class="cit-fpage">315</span>–<span class="cit-lpage">334</span> (<span class="cit-pub-date">2011</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAmerican%2BJournal%2Bof%2BScience%26rft.stitle%253Dajs%26rft.aulast%253DHannisdal%26rft.auinit1%253DB.%26rft.volume%253D311%26rft.issue%253D4%26rft.spage%253D315%26rft.epage%253D334%26rft.atitle%253DNon-parametric%2Binference%2Bof%2Bcausal%2Binteractions%2Bfrom%2Bgeological%2Brecords%26rft_id%253Dinfo%253Adoi%252F10.2475%252F04.2011.02%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiYWpzIjtzOjU6InJlc2lkIjtzOjk6IjMxMS80LzMxNSI7czo0OiJhdG9tIjtzOjM3OiIvYmlvcnhpdi9lYXJseS8yMDE2LzAzLzE1LzA0MzcyOS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">25.</span><a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference 25. in text" id="ref-25">↵</a><div class="cit ref-cit ref-journal" id="cit-043729v1.25" data-doi="10.1103/PhysRevLett.85.461"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Schreiber</span>,  <span class="cit-name-given-names">T.</span></span> <span class="cit-article-title">Measuring information transfer</span>. <abbr class="cit-jnl-abbrev">Phys. Rev. Lett.</abbr> <span class="cit-vol">85</span>, <span class="cit-fpage">461</span>–<span class="cit-lpage">464</span> (<span class="cit-pub-date">2000</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPhysical%2BReview%2BLetters%26rft.stitle%253DPhysical%2BReview%2BLetters%26rft.aulast%253DSchreiber%26rft.auinit1%253DT.%26rft.volume%253D85%26rft.issue%253D2%26rft.spage%253D461%26rft.epage%253D464%26rft.atitle%253DMeasuring%2Binformation%2Btransfer%26rft_id%253Dinfo%253Adoi%252F10.1103%252FPhysRevLett.85.461%26rft_id%253Dinfo%253Apmid%252F10991308%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1103/PhysRevLett.85.461&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=10991308&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2016%2F03%2F15%2F043729.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000088048700059&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">26.</span><a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference 26. in text" id="ref-26">↵</a><div class="cit ref-cit ref-journal" id="cit-043729v1.26"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Reitan</span>,  <span class="cit-name-given-names">T.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Schweder</span>,  <span class="cit-name-given-names">T.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Henderiks</span>,  <span class="cit-name-given-names">J.</span></span> <span class="cit-article-title">Phenotypic evolution studied by layered stochastic differential equations</span>. <abbr class="cit-jnl-abbrev">The Annals of Applied Statistics</abbr> <span class="cit-vol">6</span>, <span class="cit-fpage">1531</span>–<span class="cit-lpage">1551</span> (<span class="cit-pub-date">2012</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DThe%2BAnnals%2Bof%2BApplied%2BStatistics%26rft.volume%253D6%26rft.spage%253D1531%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">27.</span><a class="rev-xref-ref" href="#xref-ref-27-1" title="View reference 27. in text" id="ref-27">↵</a><div class="cit ref-cit ref-journal" id="cit-043729v1.27"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Liow</span>,  <span class="cit-name-given-names">L. H.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Reitan</span>,  <span class="cit-name-given-names">T.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Harnik</span>,  <span class="cit-name-given-names">P. G.</span></span> <span class="cit-article-title">Ecological interactions on macroevolutionary time scales: clams and brachiopods are more than ships that pass in the night</span>. <abbr class="cit-jnl-abbrev">Ecol Letters</abbr> <span class="cit-vol">18</span>, <span class="cit-fpage">1030</span>–<span class="cit-lpage">1039</span> (<span class="cit-pub-date">2015</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEcol%2BLetters%26rft.volume%253D18%26rft.spage%253D1030%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">28.</span><a class="rev-xref-ref" href="#xref-ref-28-1" title="View reference 28. in text" id="ref-28">↵</a><div class="cit ref-cit ref-journal" id="cit-043729v1.28"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Ye</span>,  <span class="cit-name-given-names">H.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Deyle</span>,  <span class="cit-name-given-names">E. R.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Gilarranz</span>,  <span class="cit-name-given-names">L. J.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Sugihara</span>,  <span class="cit-name-given-names">G.</span></span> <span class="cit-article-title">Distinguishing time-delayed causal interactions using convergent cross mapping</span>. <abbr class="cit-jnl-abbrev">Sci. Rep.</abbr> <span class="cit-vol">5</span>, <span class="cit-fpage">14750</span> (<span class="cit-pub-date">2015</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DSci.%2BRep.%26rft.volume%253D5%26rft.spage%253D14750%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">29.</span><a class="rev-xref-ref" href="#xref-ref-29-1" title="View reference 29. in text" id="ref-29">↵</a><div class="cit ref-cit ref-journal" id="cit-043729v1.29" data-doi="10.1038/417538a"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Jernvall</span>,  <span class="cit-name-given-names">J.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Fortelius</span>,  <span class="cit-name-given-names">M.</span></span> <span class="cit-article-title">Common mammals drive the evolutionary increase of hypsodonty in the Neogene</span>. <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">417</span>, <span class="cit-fpage">538</span>–<span class="cit-lpage">540</span> (<span class="cit-pub-date">2002</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.volume%253D417%26rft.issue%253D6888%26rft.spage%253D538%26rft.epage%253D540%26rft.atitle%253DCommon%2Bmammals%2Bdrive%2Bthe%2Bevolutionary%2Bincrease%2Bof%2Bhypsodonty%2Bin%2Bthe%2BNeogene%26rft_id%253Dinfo%253Adoi%252F10.1038%252F417538a%26rft_id%253Dinfo%253Apmid%252F12037565%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/417538a&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=2002047796&amp;link_type=GEOREF" class="cit-ref-sprinkles cit-ref-sprinkles-georef"><span>GeoRef</span></a><a href="/lookup/external-ref?access_num=12037565&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2016%2F03%2F15%2F043729.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li></ol></div><div class="section ref-list" id="ref-list-2"><ol class="cit-list ref-use-labels"><li><span class="ref-label">30.</span><a class="rev-xref-ref" href="#xref-ref-30-1" title="View reference 30. in text" id="ref-30">↵</a><div class="cit ref-cit ref-web" id="cit-043729v1.30"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Stewart</span>,  <span class="cit-name-given-names">D.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Pearson</span>,  <span class="cit-name-given-names">P. N.</span></span> <span class="cit-source">PLANKRANGE: a database of planktonic foraminiferal ranges</span>. (<span class="cit-pub-date">2000</span>). at &lt;<a href="http://palaeo.gly.bris.ac.uk/Data/plankrange.html">http://palaeo.gly.bris.ac.uk/Data/plankrange.html</a>&gt;</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">31.</span><a class="rev-xref-ref" href="#xref-ref-31-1" title="View reference 31. in text" id="ref-31">↵</a><div class="cit ref-cit ref-journal" id="cit-043729v1.31" data-doi="10.1111/j.1469-185X.2011.00178.x"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Aze</span>,  <span class="cit-name-given-names">T.</span></span> <span class="cit-etal">et al.</span> <span class="cit-article-title">A phylogeny of Cenozoic macroperforate planktonic foraminifera from fossil data</span>. <abbr class="cit-jnl-abbrev">Biological Reviews</abbr> <span class="cit-vol">86</span>, <span class="cit-fpage">900</span>–<span class="cit-lpage">927</span> (<span class="cit-pub-date">2011</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBiological%2BReviews%2B%2528Cambridge%253B%2BPrint%2529%26rft.stitle%253DBiological%2BReviews%2B%2528Cambridge%253B%2BPrint%2529%26rft.aulast%253DAze%26rft.auinit1%253DT.%26rft.volume%253D86%26rft.issue%253D4%26rft.spage%253D900%26rft.epage%253D927%26rft.atitle%253DA%2Bphylogeny%2Bof%2BCenozoic%2Bmacroperforate%2Bplanktonic%2Bforaminifera%2Bfrom%2Bfossil%2Bdata.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1469-185X.2011.00178.x%26rft_id%253Dinfo%253Apmid%252F21492379%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1469-185X.2011.00178.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=2012039450&amp;link_type=GEOREF" class="cit-ref-sprinkles cit-ref-sprinkles-georef"><span>GeoRef</span></a><a href="/lookup/external-ref?access_num=21492379&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2016%2F03%2F15%2F043729.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">32.</span><a class="rev-xref-ref" href="#xref-ref-32-1" title="View reference 32. in text" id="ref-32">↵</a><div class="cit ref-cit ref-book" id="cit-043729v1.32"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Gradstein</span>,  <span class="cit-name-given-names">F. M.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Ogg</span>,  <span class="cit-name-given-names">J. G.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Smith</span>,  <span class="cit-name-given-names">A. G.</span></span> (eds.) <span class="cit-source">A Geologic Time Scale 2004</span>. (<span class="cit-publ-name">Cambridge University Press</span> <span class="cit-pub-date">2004</span>).</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">33.</span><a class="rev-xref-ref" href="#xref-ref-33-1" title="View reference 33. in text" id="ref-33">↵</a><div class="cit ref-cit ref-book" id="cit-043729v1.33"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Casella</span>,  <span class="cit-name-given-names">G.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Berger</span>,  <span class="cit-name-given-names">R. L.</span></span> <span class="cit-source">Statistical inference</span>. (<span class="cit-publ-name">Duxbury</span>, <span class="cit-pub-date">2002</span>).</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">34.</span><a class="rev-xref-ref" href="#xref-ref-34-1" title="View reference 34. in text" id="ref-34">↵</a><div class="cit ref-cit ref-journal" id="cit-043729v1.34" data-doi="10.1666/0094-8373(2004)030&lt;0666:RBFDCT&gt;2.0.CO;2"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Bush</span>,  <span class="cit-name-given-names">A. M.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Markey</span>,  <span class="cit-name-given-names">M. J.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Marshall</span>,  <span class="cit-name-given-names">C. R.</span></span> <span class="cit-article-title">Removing bias from diversitycurves: the effects of spatially organized biodiversity on sampling-standardization</span>. <abbr class="cit-jnl-abbrev">Paleobiology</abbr> <span class="cit-vol">30</span>, <span class="cit-fpage">666</span>–<span class="cit-lpage">686</span> (<span class="cit-pub-date">2004</span>)</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPaleobiology%26rft.stitle%253DPaleobiology%26rft.volume%253D30%26rft.issue%253D4%26rft.spage%253D666%26rft.epage%253D686%26rft.atitle%253DRemoving%2Bbias%2Bfrom%2Bdiversity%2Bcurves%253A%2Bthe%2Beffects%2Bof%2Bspatially%2Borganized%2Bbiodiversity%2Bon%2Bsampling-standardization%26rft_id%253Dinfo%253Adoi%252F10.1666%252F0094-8373%25282004%2529030%253C0666%253ARBFDCT%253E2.0.CO%253B2%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MTA6ImdzcGFsZW9iaW8iO3M6NToicmVzaWQiO3M6ODoiMzAvNC82NjYiO3M6NDoiYXRvbSI7czozNzoiL2Jpb3J4aXYvZWFybHkvMjAxNi8wMy8xNS8wNDM3MjkuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">35.</span><a class="rev-xref-ref" href="#xref-ref-35-1" title="View reference 35. in text" id="ref-35">↵</a><div class="cit ref-cit ref-journal" id="cit-043729v1.35" data-doi="10.1666/13036"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Alroy</span>,  <span class="cit-name-given-names">J.</span></span> <span class="cit-article-title">Accurate and precise estimates of origination and extinction rates</span>. <abbr class="cit-jnl-abbrev">Paleobiology</abbr> <span class="cit-vol">40</span>, <span class="cit-fpage">374</span>–<span class="cit-lpage">397</span> (<span class="cit-pub-date">2014</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPaleobiology%26rft.stitle%253DPaleobiology%26rft.aulast%253DAlroy%26rft.auinit1%253DJ.%26rft.volume%253D40%26rft.issue%253D3%26rft.spage%253D374%26rft.epage%253D397%26rft.atitle%253DAccurate%2Band%2Bprecise%2Bestimates%2Bof%2Borigination%2Band%2Bextinction%2Brates%26rft_id%253Dinfo%253Adoi%252F10.1666%252F13036%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MTA6ImdzcGFsZW9iaW8iO3M6NToicmVzaWQiO3M6ODoiNDAvMy8zNzQiO3M6NDoiYXRvbSI7czozNzoiL2Jpb3J4aXYvZWFybHkvMjAxNi8wMy8xNS8wNDM3MjkuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">36.</span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-043729v1.36"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Alroy</span>,  <span class="cit-name-given-names">J.</span></span> <span class="cit-article-title">Fair sampling of taxonomic richness and unbiased estimation of origination and extinction rates</span>. <abbr class="cit-jnl-abbrev">The Paleontological Society Papers</abbr> <span class="cit-vol">16</span>, <span class="cit-fpage">55</span>–<span class="cit-lpage">80</span> (<span class="cit-pub-date">2010</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DThe%2BPaleontological%2BSociety%2BPapers%26rft.volume%253D16%26rft.spage%253D55%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">37.</span><a class="rev-xref-ref" href="#xref-ref-37-1" title="View reference 37. in text" id="ref-37">↵</a><div class="cit ref-cit ref-journal" id="cit-043729v1.37" data-doi="10.1111/j.1475-4983.2010.01011.x"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Alroy</span>,  <span class="cit-name-given-names">J.</span></span> <span class="cit-auth"><span class="cit-name-surname">Geographical</span></span>, <span class="cit-article-title">environmental and intrinsic biotic controls on Phanerozoic marine diversification</span>. <abbr class="cit-jnl-abbrev">Palaeontology</abbr> <span class="cit-vol">53</span>, <span class="cit-fpage">1211</span>–<span class="cit-lpage">1235</span> (<span class="cit-pub-date">2010</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPalaeontology%26rft.stitle%253DPalaeontology%26rft.volume%253D53%26rft.issue%253D6%26rft.spage%253D1211%26rft.epage%253D1235%26rft.atitle%253DGeographical%252C%2Benvironmental%2Band%2Bintrinsic%2Bbiotic%2Bcontrols%2Bon%2BPhanerozoic%2Bmarine%2Bdiversification%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1475-4983.2010.01011.x%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1475-4983.2010.01011.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=2011022510&amp;link_type=GEOREF" class="cit-ref-sprinkles cit-ref-sprinkles-georef"><span>GeoRef</span></a><a href="/lookup/external-ref?access_num=000284320600003&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">38.</span><a class="rev-xref-ref" href="#xref-ref-38-1" title="View reference 38. in text" id="ref-38">↵</a><div class="cit ref-cit ref-journal" id="cit-043729v1.38"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Kwiatkowski</span>,  <span class="cit-name-given-names">D.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Phillips</span>,  <span class="cit-name-given-names">P. C. B.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Schmidt</span>,  <span class="cit-name-given-names">P.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Shin</span>,  <span class="cit-name-given-names">Y.</span></span> <span class="cit-article-title">Testing the null hypothesis of stationarity against the alternative of a unit root</span>. <abbr class="cit-jnl-abbrev">Journal of Econometrics</abbr> <span class="cit-vol">54</span>, <span class="cit-fpage">159178</span> (<span class="cit-pub-date">1992</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BEconometrics%26rft.volume%253D54%26rft.spage%253D159178%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">39.</span><a class="rev-xref-ref" href="#xref-ref-39-1" title="View reference 39. in text" id="ref-39">↵</a><div class="cit ref-cit ref-book" id="cit-043729v1.39" data-doi="10.1007/BFb0091924"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Takens</span>,  <span class="cit-name-given-names">F.</span></span> in <span class="cit-source">Dynamical Systems and Turbulence</span> (eds. <span class="cit-auth"><span class="cit-name-surname">Rand</span>,  <span class="cit-name-given-names">D. A.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Young</span>,  <span class="cit-name-given-names">L. S.</span></span>) <span class="cit-vol">898</span>, <span class="cit-fpage">366</span>–<span class="cit-lpage">381</span> (<span class="cit-publ-name">Springer</span>, <span class="cit-pub-date">1981</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DDynamical%2BSystems%2Band%2BTurbulence%26rft.volume%253D898%26rft.spage%253D366%26rft_id%253Dinfo%253Adoi%252F10.1007%252FBFb0091924%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/BFb0091924&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label">40.</span><a class="rev-xref-ref" href="#xref-ref-40-1" title="View reference 40. in text" id="ref-40">↵</a><div class="cit ref-cit ref-journal" id="cit-043729v1.40" data-doi="10.1038/344734a0"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Sugihara</span>,  <span class="cit-name-given-names">G.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">May</span>,  <span class="cit-name-given-names">R. M.</span></span> <span class="cit-article-title">Nonlinear forecasting as a way of distinguishing chaos from measurement error in time series</span>. <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">344</span>, <span class="cit-fpage">734</span>–<span class="cit-lpage">741</span> (<span class="cit-pub-date">1990</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DSugihara%26rft.auinit1%253DG.%26rft.volume%253D344%26rft.issue%253D6268%26rft.spage%253D734%26rft.epage%253D741%26rft.atitle%253DNonlinear%2Bforecasting%2Bas%2Ba%2Bway%2Bof%2Bdistinguishing%2Bchaos%2Bfrom%2Bmeasurement%2Berror%2Bin%2Btime%2Bseries.%26rft_id%253Dinfo%253Adoi%252F10.1038%252F344734a0%26rft_id%253Dinfo%253Apmid%252F2330029%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/344734a0&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=2330029&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2016%2F03%2F15%2F043729.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1990CZ67800045&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">41.</span><a class="rev-xref-ref" href="#xref-ref-41-1" title="View reference 41. in text" id="ref-41">↵</a><div class="cit ref-cit ref-journal" id="cit-043729v1.41" data-doi="10.1073/pnas.1511080112"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Ye</span>,  <span class="cit-name-given-names">H.</span></span> <span class="cit-etal">et al.</span> <span class="cit-article-title">Reply to Luo et al.: Robustness of causal effects of galactic cosmic rays on interannual variation in global temperature</span>. <abbr class="cit-jnl-abbrev">Proceedings of the National Academy of Sciences of the United States of America</abbr> <span class="cit-vol">112</span>, <span class="cit-fpage">E4640</span>–<span class="cit-lpage">E4641</span> (<span class="cit-pub-date">2015</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProceedings%2Bof%2Bthe%2BNational%2BAcademy%2Bof%2BSciences%2Bof%2Bthe%2BUnited%2BStates%2Bof%2BAmerica%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.1511080112%26rft_id%253Dinfo%253Apmid%252F26283406%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiRlVMTCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMjoiMTEyLzM0L0U0NjQwIjtzOjQ6ImF0b20iO3M6Mzc6Ii9iaW9yeGl2L2Vhcmx5LzIwMTYvMDMvMTUvMDQzNzI5LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">42.</span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-043729v1.42" data-doi="10.1073/pnas.1510571112"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Luo</span>,  <span class="cit-name-given-names">M.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Kantz</span>,  <span class="cit-name-given-names">H.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Lau</span>,  <span class="cit-name-given-names">N.-C.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Huang</span>,  <span class="cit-name-given-names">W.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Zhou</span>,  <span class="cit-name-given-names">Y.</span></span> <span class="cit-article-title">Questionable dynamical evidence for causality between galactic cosmic rays and interannual variation in global temperature</span>. <abbr class="cit-jnl-abbrev">Proceedings of the National Academy of Sciences of the United States of America</abbr> <span class="cit-vol">112</span>, <span class="cit-fpage">E4638</span>–<span class="cit-lpage">E4639</span> (<span class="cit-pub-date">2015</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProceedings%2Bof%2Bthe%2BNational%2BAcademy%2Bof%2BSciences%2Bof%2Bthe%2BUnited%2BStates%2Bof%2BAmerica%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.1510571112%26rft_id%253Dinfo%253Apmid%252F26283407%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiRlVMTCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMjoiMTEyLzM0L0U0NjM4IjtzOjQ6ImF0b20iO3M6Mzc6Ii9iaW9yeGl2L2Vhcmx5LzIwMTYvMDMvMTUvMDQzNzI5LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">43.</span><a class="rev-xref-ref" href="#xref-ref-43-1" title="View reference 43. in text" id="ref-43">↵</a><div class="cit ref-cit ref-journal" id="cit-043729v1.43" data-doi="10.1016/0167-2789(92)90102-S"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Theiler</span>,  <span class="cit-name-given-names">J.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Eubank</span>,  <span class="cit-name-given-names">S.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Longtin</span>,  <span class="cit-name-given-names">A.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Galdrikian</span>,  <span class="cit-name-given-names">B.</span></span> <span class="cit-article-title">Testing for nonlinearity in time series: the method of surrogate data</span>. <abbr class="cit-jnl-abbrev">Physica D: Nonlinear Phenomena</abbr> <span class="cit-vol">58</span>, <span class="cit-fpage">77</span>–<span class="cit-lpage">94</span> (<span class="cit-pub-date">1992</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPhysica%2BD%253A%2BNonlinear%2BPhenomena%26rft.volume%253D58%26rft.spage%253D77%26rft_id%253Dinfo%253Adoi%252F10.1016%252F0167-2789%252892%252990102-S%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/0167-2789(92)90102-S&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label">44.</span><a class="rev-xref-ref" href="#xref-ref-44-1" title="View reference 44. in text" id="ref-44">↵</a><div class="cit ref-cit ref-journal" id="cit-043729v1.44" data-doi="10.1175/1520-0442(1997)010&lt;2147:AMTETS&gt;2.0.CO;2"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Ebisuzaki</span>,  <span class="cit-name-given-names">W.</span></span> <span class="cit-article-title">A method to estimate the statistical significance of a correlation when the data are serially correlated</span>. <abbr class="cit-jnl-abbrev">Journal of Climate</abbr> <span class="cit-vol">10</span>, <span class="cit-fpage">2147</span>–<span class="cit-lpage">2153</span> (<span class="cit-pub-date">1997</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BClimate%26rft.volume%253D10%26rft.spage%253D2147%26rft_id%253Dinfo%253Adoi%252F10.1175%252F1520-0442%25281997%2529010%253C2147%253AAMTETS%253E2.0.CO%253B2%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1175/1520-0442(1997)010&lt;2147:AMTETS&gt;2.0.CO;2&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=A1997XX48200002&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">45.</span><a class="rev-xref-ref" href="#xref-ref-45-1" title="View reference 45. in text" id="ref-45">↵</a><div class="cit ref-cit ref-journal" id="cit-043729v1.45"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Palus</span>,  <span class="cit-name-given-names">M.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Vejmelka</span>,  <span class="cit-name-given-names">M.</span></span> <span class="cit-article-title">Directionality of coupling from bivariate time series: How to avoid false causalities and missed connections</span>. <abbr class="cit-jnl-abbrev">Phys. Rev. E</abbr> <span class="cit-vol">75</span>, <span class="cit-fpage">056211</span> (<span class="cit-pub-date">2007</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPhys.%2BRev.%2BE%26rft.volume%253D75%26rft.spage%253D056211%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">46.</span><a class="rev-xref-ref" href="#xref-ref-46-1" title="View reference 46. in text" id="ref-46">↵</a><div class="cit ref-cit ref-journal" id="cit-043729v1.46" data-doi="10.2307/1912791"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Granger</span>,  <span class="cit-name-given-names">C. W. J.</span></span> <span class="cit-article-title">Investigating Causal Relations by Econometric Models and Cross-spectral Methods</span>. <abbr class="cit-jnl-abbrev">Econometrica: Journal of the Econometric Society</abbr> <span class="cit-vol">37</span>, <span class="cit-fpage">424</span>–<span class="cit-lpage">438</span> (<span class="cit-pub-date">1969</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEconometrica%253A%2BJournal%2Bof%2Bthe%2BEconometric%2BSociety%26rft.volume%253D37%26rft.spage%253D424%26rft_id%253Dinfo%253Adoi%252F10.2307%252F1912791%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2307/1912791&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label">47.</span><a class="rev-xref-ref" href="#xref-ref-47-1" title="View reference 47. in text" id="ref-47">↵</a><div class="cit ref-cit ref-journal" id="cit-043729v1.47" data-doi="10.1103/PhysRevLett.103.238701"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Barnett</span>,  <span class="cit-name-given-names">L.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Barrett</span>,  <span class="cit-name-given-names">A. B.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Seth</span>,  <span class="cit-name-given-names">A. K.</span></span> <span class="cit-article-title">Granger Causality and Transfer Entropy Are Equivalent for Gaussian Variables</span>. <abbr class="cit-jnl-abbrev">Phys. Rev. Lett.</abbr> <span class="cit-vol">103</span>, <span class="cit-fpage">238701</span> (<span class="cit-pub-date">2009</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPhysical%2BReview%2BLetters%26rft.stitle%253DPhysical%2BReview%2BLetters%26rft.aulast%253DBarnett%26rft.auinit1%253DL.%26rft.volume%253D103%26rft.issue%253D23%26rft.spage%253D238701%26rft.epage%253D238701%26rft.atitle%253DGranger%2Bcausality%2Band%2Btransfer%2Bentropy%2Bare%2Bequivalent%2Bfor%2BGaussian%2Bvariables.%26rft_id%253Dinfo%253Adoi%252F10.1103%252FPhysRevLett.103.238701%26rft_id%253Dinfo%253Apmid%252F20366183%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1103/PhysRevLett.103.238701&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20366183&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2016%2F03%2F15%2F043729.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">48.</span><a class="rev-xref-ref" href="#xref-ref-48-1" title="View reference 48. in text" id="ref-48">↵</a><div class="cit ref-cit ref-journal" id="cit-043729v1.48"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Hannisdal</span>,  <span class="cit-name-given-names">B.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Peters</span>,  <span class="cit-name-given-names">S. E.</span></span> <span class="cit-article-title">Phanerozoic Earth system evolution and marine biodiversity</span>. <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">334</span>, <span class="cit-fpage">1122</span>–<span class="cit-lpage">1124</span> (<span class="cit-pub-date">2011</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.volume%253D334%26rft.spage%253D1122%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">49.</span><a class="rev-xref-ref" href="#xref-ref-49-1" title="View reference 49. in text" id="ref-49">↵</a><div class="cit ref-cit ref-journal" id="cit-043729v1.49"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Dunhill</span>,  <span class="cit-name-given-names">A. M.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Hannisdal</span>,  <span class="cit-name-given-names">B.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Benton</span>,  <span class="cit-name-given-names">M. J.</span></span> <span class="cit-article-title">Disentangling rock record bias and common-cause from redundancy in the British fossil record</span>. <abbr class="cit-jnl-abbrev">Nat Comms</abbr> <span class="cit-vol">5</span>, <span class="cit-fpage">5818</span> (<span class="cit-pub-date">2014</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat%2BComms%26rft.volume%253D5%26rft.spage%253D5818%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">50.</span><a class="rev-xref-ref" href="#xref-ref-50-1" title="View reference 50. in text" id="ref-50">↵</a><div class="cit ref-cit ref-book" id="cit-043729v1.50"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Jeffreys</span>,  <span class="cit-name-given-names">H.</span></span> <span class="cit-source">Theory of Probability</span>. (<span class="cit-publ-name">Oxford University Press</span>, <span class="cit-pub-date">1961</span>).</cite></div><div class="cit-extra"></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.biorxiv.org/content/10.1101/043729v1.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-disqus-comment" >
  
      
  
  <div class="pane-content">
    <div id="disqus_thread"><noscript><p><a href="http://biorxivstage.disqus.com/?url=https%3A%2F%2Fwww.biorxiv.org%2Fcontent%2F10.1101%2F043729v1" class="" data-icon-position="" data-hide-link-title="0">View the discussion thread.</a></p></noscript></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/10.1101/043570v1" title="Synlet: an R package for systemically analyzing synthetic lethal RNA interference screen data" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous</span></a></span><span class="pager-next"><a href="/content/10.1101/043901v1" title="Transdermal electrical neuromodulation of the trigeminal sensory nuclear complex improves sleep quality and mood" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    Posted&nbsp;March 15, 2016.  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-biorxiv-art-tools" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-biorxiv_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/10.1101/043729v1.full.pdf" target="_self" class="article-dl-pdf-link link-icon"><i class="icon-external-link-sign"></i> <span class="title">Download PDF</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-biorxiv-supplementary-fragment" >
  
      
  
  <div class="pane-content">
    <p><a href="/content/10.1101/043729v1.supplementary-material"><i class="icon-file"></i>Supplementary Material </a></p>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-biorxiv-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="Email this Article" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-biorxiv_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/10.1101/043729v1.full" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for your interest in spreading the word about bioRxiv.</p><p>NOTE: Your email address is requested solely to identify you as the sender of this article.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/10.1101/043729v1" class="active" data-icon-position="" data-hide-link-title="0">Common species link global ecosystems to climate change</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from bioRxiv
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would like to see this page from the bioRxiv website.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/11918" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-3saUWaxKmL9zGv8HJOzJalsVVKQIAS_p_8K3fkovlRQ" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-biorxiv-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-biorxiv_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="11918" id="node11918--2" data-pisa="biorxiv;043729v1" data-pisa-master="biorxiv;043729" data-seqnum="11918" data-apath="/biorxiv/early/2016/03/15/043729.atom"><div  class="highwire-cite highwire-cite-highwire-article highwire-citation-biorxiv-article-pap-list clearfix" >

      <div class="highwire-cite-title" >
      <div class="highwire-cite-title">Common species link global ecosystems to climate change</div>    </div>
  
      <div  class="highwire-cite-authors" ><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0"><span class="nlm-given-names">Bjarte</span> <span class="nlm-surname">Hannisdal</span></span>, <span class="highwire-citation-author" data-delta="1"><span class="nlm-given-names">Kristian Agasøster</span> <span class="nlm-surname">Haaga</span></span>, <span class="highwire-citation-author" data-delta="2"><span class="nlm-given-names">Trond</span> <span class="nlm-surname">Reitan</span></span>, <span class="highwire-citation-author" data-delta="3"><span class="nlm-given-names">David</span> <span class="nlm-surname">Diego</span></span>, <span class="highwire-citation-author" data-delta="4"><span class="nlm-given-names">Lee Hsiang</span> <span class="nlm-surname">Liow</span></span></span></div>
  
      <div  class="highwire-cite-metadata" ><span  class="highwire-cite-metadata-journal highwire-cite-metadata">bioRxiv </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">043729; </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata"><span class="doi_label">doi:</span> https://doi.org/10.1101/043729 </span></div>
  
  
  </div>
</div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.biorxiv.org/content/10.1101/043729v1" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.biorxiv.org/content/10.1101/043729v1&amp;title=Common%20species%20link%20global%20ecosystems%20to%20climate%20change" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.biorxiv.org/content/10.1101/043729v1&amp;title=Common%20species%20link%20global%20ecosystems%20to%20climate%20change" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.biorxiv.org/content/10.1101/043729v1&amp;text=Common%20species%20link%20global%20ecosystems%20to%20climate%20change" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.biorxiv.org/content/10.1101/043729v1&amp;title=Common%20species%20link%20global%20ecosystems%20to%20climate%20change" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.biorxiv.org/content/10.1101/043729v1&amp;t=Common%20species%20link%20global%20ecosystems%20to%20climate%20change" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.biorxiv.org/content/10.1101/043729v1&amp;title=Common%20species%20link%20global%20ecosystems%20to%20climate%20change" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.biorxiv.org/content/10.1101/043729v1&amp;title=Common%20species%20link%20global%20ecosystems%20to%20climate%20change" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-biorxiv-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title="Citation Tools"><i class="icon-globe"></i> <span class="title">Citation Tools</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-biorxiv_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="11918" id="node11918--3" data-pisa="biorxiv;043729v1" data-pisa-master="biorxiv;043729" data-seqnum="11918" data-apath="/biorxiv/early/2016/03/15/043729.atom"><div  class="highwire-cite highwire-cite-highwire-article highwire-citation-biorxiv-article-pap-list clearfix" >

      <div class="highwire-cite-title" >
      <div class="highwire-cite-title">Common species link global ecosystems to climate change</div>    </div>
  
      <div  class="highwire-cite-authors" ><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0"><span class="nlm-given-names">Bjarte</span> <span class="nlm-surname">Hannisdal</span></span>, <span class="highwire-citation-author" data-delta="1"><span class="nlm-given-names">Kristian Agasøster</span> <span class="nlm-surname">Haaga</span></span>, <span class="highwire-citation-author" data-delta="2"><span class="nlm-given-names">Trond</span> <span class="nlm-surname">Reitan</span></span>, <span class="highwire-citation-author" data-delta="3"><span class="nlm-given-names">David</span> <span class="nlm-surname">Diego</span></span>, <span class="highwire-citation-author" data-delta="4"><span class="nlm-given-names">Lee Hsiang</span> <span class="nlm-surname">Liow</span></span></span></div>
  
      <div  class="highwire-cite-metadata" ><span  class="highwire-cite-metadata-journal highwire-cite-metadata">bioRxiv </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">043729; </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata"><span class="doi_label">doi:</span> https://doi.org/10.1101/043729 </span></div>
  
  
  </div>
</div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage&amp;rft.epage&amp;rft.atitle=Common%20species%20link%20global%20ecosystems%20to%20climate%20change&amp;rft.volume&amp;rft.issue&amp;rft.date=2016-01-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=bioRxiv&amp;rft.au=Hannisdal%2C+Bjarte&amp;rft.au=Haaga%2C+Kristian+Agas%C3%B8ster&amp;rft.au=Reitan%2C+Trond&amp;rft.au=Diego%2C+David&amp;rft.au=Liow%2C+Lee+Hsiang"></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/11918/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/11918/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/11918/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/11918/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/11918/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/11918/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/11918/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/11918/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/11918/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/11918/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/11918/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/11918/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><div class="item-list"><ul><li class="first"><a href="http://twitter.com/share?url=https%3A//www.biorxiv.org/content/10.1101/043729v1&amp;count=horizontal&amp;via=&amp;text=Common%20species%20link%20global%20ecosystems%20to%20climate%20change&amp;counturl=https%3A//www.biorxiv.org/content/10.1101/043729v1" class="twitter-share-button service-links-twitter-widget" id="twitter_widget" title="Tweet This" rel="nofollow" data-icon-position="" data-hide-link-title="0"><span class="element-invisible">Tweet Widget</span></a></li><li><a href="http://www.facebook.com/plugins/like.php?href=https%3A//www.biorxiv.org/content/10.1101/043729v1&amp;layout=button_count&amp;show_faces=false&amp;action=like&amp;colorscheme=light&amp;width=100&amp;height=21&amp;font=&amp;locale=" id="facebook_like" title="I Like it" class="service-links-facebook-like" rel="nofollow" data-icon-position="" data-hide-link-title="0"><span class="element-invisible">Facebook Like</span></a></li><li class="last"><a href="https://www.biorxiv.org/content/10.1101/043729v1" id="google_plus_one" title="Plus it" class="service-links-google-plus-one" rel="nofollow" data-icon-position="" data-hide-link-title="0"><span class="element-invisible">Google Plus One</span></a></li></ul></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-collections" >
  
        <h2 class="pane-title">Subject Area</h2>
    
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-collections"><div class="highwire-list"><ul class="highwire-article-collection-term-list"><li class="first last odd"><span class="highwire-article-collection-term"><a href="/collection/paleontology" class="highlight" data-icon-position="" data-hide-link-title="0">Paleontology<i class="icon-caret-right"></i>
</a></span></li></ul></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-biorxiv-subject-collections block-style-col2" >
  
      
  
  <div class="pane-content">
    <div class="panel-flexible panels-flexible-8 clearfix" id="mini-panel-biorxiv_subject_collections">
<div class="panel-flexible-inside panels-flexible-8-inside">
<div class="panels-flexible-region panels-flexible-region-8-center panels-flexible-region-first panels-flexible-region-last">
  <div class="inside panels-flexible-region-inside panels-flexible-region-8-center-inside panels-flexible-region-inside-first panels-flexible-region-inside-last">
<div class="panel-pane pane-snippet" >
  
      
  
  <div class="pane-content">
    <div class="snippet biorxiv-subject-areas-table-title" id="biorxiv-subject-areas-table-title">
  
      
  <div class="snippet-content">
    <b>Subject Areas</b>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-snippet" >
  
      
  
  <div class="pane-content">
    <div class="snippet biorxiv-subject-areas-view-papers" id="biorxiv-subject-areas-view-papers">
  
      
  <div class="snippet-content">
    <a href="/content/early/recent"><strong>All Articles</strong></a>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-subject-collections" >
  
      
  
  <div class="pane-content">
    <ul id="collection" class="collection highwire-list-expand"><li class="outer collection depth-2  child first"><div class = "data-wrapper"><a href="/collection/animal-behavior-and-cognition" class="" data-icon-position="" data-hide-link-title="0">Animal Behavior and Cognition</a> <span class = "article-count">(1674)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/biochemistry" class="" data-icon-position="" data-hide-link-title="0">Biochemistry</a> <span class = "article-count">(2825)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/bioengineering" class="" data-icon-position="" data-hide-link-title="0">Bioengineering</a> <span class = "article-count">(1963)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/bioinformatics" class="" data-icon-position="" data-hide-link-title="0">Bioinformatics</a> <span class = "article-count">(10456)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/biophysics" class="" data-icon-position="" data-hide-link-title="0">Biophysics</a> <span class = "article-count">(4275)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/cancer-biology" class="" data-icon-position="" data-hide-link-title="0">Cancer Biology</a> <span class = "article-count">(3301)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/cell-biology" class="" data-icon-position="" data-hide-link-title="0">Cell Biology</a> <span class = "article-count">(4686)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/clinical-trials" class="" data-icon-position="" data-hide-link-title="0">Clinical Trials</a> <span class = "article-count">(136)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/developmental-biology" class="" data-icon-position="" data-hide-link-title="0">Developmental Biology</a> <span class = "article-count">(2911)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/ecology" class="" data-icon-position="" data-hide-link-title="0">Ecology</a> <span class = "article-count">(4575)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/epidemiology" class="" data-icon-position="" data-hide-link-title="0">Epidemiology</a> <span class = "article-count">(2041)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/evolutionary-biology" class="" data-icon-position="" data-hide-link-title="0">Evolutionary Biology</a> <span class = "article-count">(7337)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/genetics" class="" data-icon-position="" data-hide-link-title="0">Genetics</a> <span class = "article-count">(5574)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/genomics" class="" data-icon-position="" data-hide-link-title="0">Genomics</a> <span class = "article-count">(6939)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/immunology" class="" data-icon-position="" data-hide-link-title="0">Immunology</a> <span class = "article-count">(2463)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/microbiology" class="" data-icon-position="" data-hide-link-title="0">Microbiology</a> <span class = "article-count">(7697)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/molecular-biology" class="" data-icon-position="" data-hide-link-title="0">Molecular Biology</a> <span class = "article-count">(3078)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/neuroscience" class="" data-icon-position="" data-hide-link-title="0">Neuroscience</a> <span class = "article-count">(19031)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/paleontology" class="" data-icon-position="" data-hide-link-title="0">Paleontology</a> <span class = "article-count">(142)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/pathology" class="" data-icon-position="" data-hide-link-title="0">Pathology</a> <span class = "article-count">(481)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/pharmacology-and-toxicology" class="" data-icon-position="" data-hide-link-title="0">Pharmacology and Toxicology</a> <span class = "article-count">(796)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/physiology" class="" data-icon-position="" data-hide-link-title="0">Physiology</a> <span class = "article-count">(1188)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/plant-biology" class="" data-icon-position="" data-hide-link-title="0">Plant Biology</a> <span class = "article-count">(2805)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/scientific-communication-and-education" class="" data-icon-position="" data-hide-link-title="0">Scientific Communication and Education</a> <span class = "article-count">(699)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/synthetic-biology" class="" data-icon-position="" data-hide-link-title="0">Synthetic Biology</a> <span class = "article-count">(919)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/systems-biology" class="" data-icon-position="" data-hide-link-title="0">Systems Biology</a> <span class = "article-count">(2898)</span></div></li>
<li class="outer collection depth-2  child last"><div class = "data-wrapper"><a href="/collection/zoology" class="" data-icon-position="" data-hide-link-title="0">Zoology</a> <span class = "article-count">(473)</span></div></li>
</ul>  </div>

  
  </div>
  </div>
</div>
</div>
</div>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__VNH5GD8Zz7g6_hGCOZjVjdMndxxC6naiExSSJKX3k_A__kzmEdtbqCsJx5Z9yg2Qy0PptB__ufXm-TxUOl0KZzUw__dPhHATyREQrnntSIgxU8XNoYny4rjInxP7lLKSNxQyY.js"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__dPhHATyREQrnntSIgxU8XNoYny4rjInxP7lLKSNxQyY.js" defer="defer"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__N7ERJBYsOWyRnJgyoM125_Aiez2MOJGaUofG1JdWWBg__bLlcwhzZhT-IwoQ_dvm1_xPxU7E7gecFLC6Fyeqtv8s__dPhHATyREQrnntSIgxU8XNoYny4rjInxP7lLKSNxQyY.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
function euCookieComplianceLoadScripts() {}
//--><!]]>
</script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var eu_cookie_compliance_cookie_name = "";
//--><!]]>
</script>
<script type="text/javascript" src="//d33xdlntwy0kbs.cloudfront.net/cshl_custom.js"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__BmqjBnkz3MgYCAoc25s1lDRMEjLhC3mEPVonUFIHi08__Unwv5-ZIuHBfFwytsjEx1niBVJ7n1T4lPws7VrkHXM4__dPhHATyREQrnntSIgxU8XNoYny4rjInxP7lLKSNxQyY.js"></script>
  </body>
</html>
