<html xmlns="http://www.w3.org/1999/xhtml"> <head> <style type="text/css">
      	     		 
      	    a                          {background : #ffffff; }
      	    article                    {border-style : dotted; border-width : 2px; }
		 	
		 	div                        {background : #ffffcc;}
		 	div.abstract-title         {font-weight : bold ; font-size : 16pt;}
		 	div.ack                    {border-style : solid ; border-color : red; margin : 2em; }
		 	div.article-meta           {border-style : solid; border-width : 1 pt; margin 1em;}
		 	div.boxed-text             {margin : 5em; border-style : solid;}
		 	div.contrib-group          {margin : 1em; }
		 	div.fig                    {border-style : solid; border-width : 2px; margin : 2em; }
		 	div.funding                {font-weight : bold ; font-size : 16pt;}
		 	div.given-names            {font-style : italic;}
		 	div.intro                  {border-style : inset; margin : 5px;}
		 	div.introduction           {border-style : inset; margin : 5px;}
		 	div.journal-meta           {border-style : solid; border-width : 1 pt; margin 1em;}
		 	div.journal-title-group    {background : #ffeeee;}
		 	div.article-title          {font-weight : bold ; font-size : 18pt;}
		 	div.kwd                    {font-style : italic}
		 	div.meta-name              {font-weight : bold ; font-size : 16pt;}
		 	div.name                   {font-weight : bold;}
		 	div.alt-title              {font-style : italic; font-size : 12px;}
		 	
		 	div.sec                    {border : 2px; margin 5px; padding 2px;}
		 	div.title                  {font-family : courier; font-weight : bold;
		 	                            font-size : 16pt; margin : 5px;}
		 	
		 	div.materials_methods      {border-style : double; margin : 5px; }
		 	div.methods                {border-style : double; margin : 5px; }
		 	
		 	div.results                {border-style : solid; margin : 5px;}
		 	div.background             {border-style : dotted; margin : 5px;}
		 	
		 	div.discussion             {border-style : groove; margin : 5px;}
		 	div.conclusion             {border-style : ridge; margin : 5px;}
		 	div.conclusions             {border-style : ridge; margin : 5px;}
		 	div.supplementary-material {border-style : inset; margin : 5px;}
		 	div.abbreviations          {border-style : double; border-color : red; margin : 5px;}
		 	div.competinginterests     {border-style : double; border-color : blue; margin : 5px;}
		 	div.acknowledgements       {border-style : double; border-color : green; margin : 5px;}
		 	div.authors_contributions   {border-style : double; border-color : purple; margin : 5px;}
		 	
		 	div.publisher              {border-style : outset; margin : 5px;}
		 	div.fn-type-conflict       {background : #f88; }
		 	div.fn-type-con            {background : #ddf; }
		 	div.fn-type-other          {background : #ddd; }
		 	
		 	div.unknown                {background : #ffd;
									 	  border-style : solid;
									 	  border-width : 1px;
									 	  padding : 2 px;}
		 	  
      	    table                      {background : #ffffdd;}
		 	tr                         {background : #ddddff; padding : 1px;}
		 	
		 	span                       {background : #ffcccc;}
		 	
		 	span.citation-author       {font-family : helvetica; background : #ffeeee;}
		 	span.collab                {background : #ddffff; }
		 	span.comment               {font-family : courier; font-size : 6px; background : #ffaaff;}
		 	span.contrib               {background : #ffffff;}
		 	span.corresp               {background : #ddffdd; }
		 	span.doi                   {background : #ffffff;}
		 	span.email                 {font-family : courier; }
		 	span.etal                  {font-style : italic;}
		 	span.fpage                 {font-family : courier;}
		 	span.given-names           {background : #ffffff;}
		 	span.iso-abbrev            {background : #ffffff;}
		 	span.issn-epub             {background : #ffffff;}
		 	span.issn-ppub             {background : #ffffff;}
		 	span.journal-title         {background : #ffffff;}
		 	span.lpage                 {font-family : courier;}
		 	span.mixed-article-title   {font-style : italic ;}
		 	span.nlm-ta                {background : #ffffff;}
		 	span.pmc                   {background : #ffffff;}
		 	span.pmcid                 {background : #ffffff;}
		 	span.pmid                  {background : #ffffff;}
		 	span.publisher             {background : #ffffff;}
		 	span.publisher-id          {background : #ffffff;}
		 	span.publisher-name        {background : #ffffff;}
		 	span.source                {background : #ffffff;}
		 	span.subject               {background : #ffffff;}
		 	span.surname               {background : #ffffff;}
		 	span.volume                {font-family : courier; font-weight : bold;}
		 	span.year                  {font-family : courier ; font-style : italic;}
			</style> </head> <body> <div class="front" title="front"> <div class="journal-meta" tagx="journal-meta" title="journal-meta"><span class="nlm-ta" title="nlm-ta">PLoS One</span><span class="iso-abbrev" title="iso-abbrev">PLoS ONE</span><span class="publisher-id" title="publisher-id">plos</span><span class="pmc" title="pmc">plosone</span><div class="journal-title-group" tagx="journal-title-group" title="journal-title-group"><span class="journal-title" tagx="journal-title" title="journal-title">PLoS ONE</span></div><span class="issn-epub" tagx="issn" title="issn-epub">1932-6203</span><div class="publisher" tagx="publisher" title="publisher"><span class="publisher-name" tagx="publisher-name" title="publisher-name">Public Library of Science</span><span class="publisher-loc" tagx="publisher-loc" title="publisher-loc">San Francisco, USA</span></div> </div> <div class="article-meta" tagx="article-meta" title="article-meta"><span class="pmid" title="pmid"> pmid: <a href="http://www.ncbi.nlm.nih.gov/pubmed/24015252">24015252</a></span><span class="publisher-id" title="publisher-id">PONE-D-13-13372</span><span class="doi" title="doi"> doi: <a href="https://dx.doi.org/10.1371/journal.pone.0072506">10.1371/journal.pone.0072506</a></span><div class="article-categories" title="article-categories">
: <span class="subject" title="subject">Research Article</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Agriculture</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Animal Management</span>: <span class="subject" title="subject">Animal Genetics</span></div> </div> <div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Biology</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Ecology</span>: <span class="subject" title="subject">Conservation Science</span><span class="subject" title="subject">Spatial and Landscape Ecology</span></div> <div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Evolutionary Biology</span>: <span class="subject" title="subject">Comparative Genomics</span><span class="subject" title="subject">Evolutionary Genetics</span><span class="subject" title="subject">Evolutionary Systematics</span></div> <div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Genetics</span>: <span class="subject" title="subject">Animal Genetics</span></div> <div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Zoology</span>: <span class="subject" title="subject">Mammalogy</span></div> </div> <div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Veterinary Science</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Animal Management</span>: <span class="subject" title="subject">Animal Genetics</span></div> </div> </div> <div class="title-group" tagx="title-group" title="title-group"> <div class="article-title" title="article-title">
Evolutionary History of the Grey-Faced Sengi, <i>Rhynchocyon udzungwensis</i>, from Tanzania: A Molecular and Species Distribution Modelling Approach</div> <div class="alt-title" title="alt-title">
Ancient Introgression in African Elephant-Shrews</div> </div> <div class="contrib-group" title="contrib-group"><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lawson</span><span class="given-names" tagx="given-names" title="given-names">Lucinda P.</span></span><a href="#aff1"><sup>1</sup></a><a href="#cor1"><sup>*</sup></a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Vernesi</span><span class="given-names" tagx="given-names" title="given-names">Cristiano</span></span><a href="#aff2"><sup>2</sup></a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ricci</span><span class="given-names" tagx="given-names" title="given-names">Silvia</span></span><a href="#aff3"><sup>3</sup></a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Rovero</span><span class="given-names" tagx="given-names" title="given-names">Francesco</span></span><a href="#aff3"><sup>3</sup></a></span></div><span class="citation_author_institution" id="aff1">[1], <span class="addr-line" title="addr-line">Department of Biological Sciences, University of Cincinnati, Cincinnati, Ohio, United States of America</span></span><span class="citation_author_institution" id="aff2">[2], <span class="addr-line" title="addr-line">Department of Biodiversity and Molecular Ecology, Centre for Research and Innovation, Michele all’Adige, Italy</span></span><span class="citation_author_institution" id="aff3">[3], <span class="addr-line" title="addr-line">Tropical Biodiversity Section, Museo delle Scienze, Trento, Italy</span></span><div class="contrib-group" title="contrib-group"><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Murphy</span><span class="given-names" tagx="given-names" title="given-names">William J.</span></span><span class="role" tagx="role" title="role">Editor</span><a href="#edit1" /></span></div><span class="citation_author_institution" id="edit1">[], <span class="addr-line" title="addr-line">Texas A&amp;amp;M University, United States of America</span></span><div class="author-notes" title="author-notes"> <div class="corresp" title="corresp">
* E-mail: <span class="email" tagx="email" title="email">lucinda.lawson@uc.edu</span></div> <div class="fn-type-COI-statement" title="COI-statement"> <p><b>Competing Interests: </b>The authors have declared that no competing interests exist.</p> </div> <div class="fn-type-con" title="con"> <p>Conceived and designed the experiments: CV FR. Performed the experiments: SR FR. Analyzed the data: CV LPL. Contributed reagents/materials/analysis tools: SR FR CV. Wrote the paper: LPL FR.</p> </div> </div><span class="pub-date-collection" title="pub-date-collection">collection: <span>2013</span></span><span class="pub-date-epub" title="pub-date-epub">epub: <span>2013-8-8</span></span><span class="volume" tagx="volume" title="volume">8</span><span class="issue" tagx="issue" title="issue">8</span><span class="elocation-id" tagx="elocation-id" title="elocation-id">e72506</span><span class="history" title="history"><span class="received" title="received">received: 2013-3-25</span><span class="accepted" title="accepted">accepted: 2013-7-10</span></span><div class="permissions"><span class="copyright" title="copyright">(C) <span class="copyright-holder" tagx="copyright-holder" title="copyright-holder">Lawson et al</span>, <span class="copyright-year" tagx="copyright-year" title="copyright-year">2013</span></span><span class="license" title="license"><span class="license-p" title="license-p">This is an open-access article distributed under the terms of the Creative Commons Attribution License, which permits unrestricted use, distribution, and reproduction in any medium, provided the original author and source are properly credited.</span></span></div> <div class="abstract" title="abstract"> <div class="abstract-title" title="abstract-title">
Abstract</div> <p><i>Rhynchocyon udzungwensis</i> is a recently described and poorly understood sengi (giant elephant-shrew) endemic to two small montane forests in Southern Tanzania, and surrounded in lower forests by <i>R. cirnei reichardi</i>. In this study, we investigate the molecular genetic relationship between <i>R. udzungwensis</i> and <i>R. c. reichardi</i>, and the possible role that shifting species distributions in response to climate fluctuations may have played in shaping their evolutionary history. <i>Rhynchocyon udzungwensis</i> and <i>R. c. reichardi</i> individuals were sampled from five localities for genetic analyses. Three mitochondrial and two nuclear loci were used to construct species trees for delimitation and to determine whether introgression was detectable either from ancient or ongoing hybridization. All species-tree results show <i>R. udzungwensis</i> and <i>R. c. reichardi</i> as distinct lineages, though mtDNA shows evidence of introgression in some populations. Nuclear loci of each species were monophyletic, implying introgression is exclusively historical. Because we found evidence of introgression, we used distribution data and species distribution modelling for present, glacial, and interglacial climate cycles to predict how shifting species distributions may have facilitated hybridization in some populations. Though interpretations are affected by the limited range of these species, a likely scenario is that the mtDNA introgression found in eastern mid-elevation populations was facilitated by low numbers of <i>R. udzungwensis</i> that expanded into lowland heavily occupied <i>R. c. reichardi</i> areas during interglacial climate cycles. These results imply that relationships within the genus <i>Rhynchocyon</i> may be confounded by porous species boundaries and introgression, even if species are not currently sympatric.</p> </div> <div class="funding-group" title="funding-group"> <div class="funding" title="funding">
Funding</div> <div class="funding-statement" title="funding-statement">
This research was supported by grants from National Geographic’s Committee for Research and Exploration (grant 8497-08 to F.R.), Trento Autonomous Province, Trento Science Museum and the Critical Ecosystem Partnership Fund. The funders had no role in study design, data collection and analysis, decision to publish, or preparation of the manuscript.</div> </div> <div class="counts" title="counts"><span class="page-count" title="page-count">page-count: </span></div> </div> </div> <div class="body" title="body"> <div class="introduction" title="sec"> <div class="title" tagx="title" title="title">
Introduction</div> <p>When a new species is described to science, understanding its genetic and morphological distinctness, relationship to congeneric taxa, the permeability of species boundaries, and overall evolutionary history are necessary to place the lineage in the grand scheme of evolution. While many initial investigations document morphological and sometimes molecular relationships to related lineages, considering broader aspects of evolution including non-linear species divergence and interspecific interactions may provide a more accurate estimate of evolutionary history and uniqueness. The prevalence of introgression is non-trivial, as ∼10% of animal species appear to have hybridization and introgression in their evolutionary past <a href="#pone.0072506-Mallet1">[1]</a>. In this study, we investigate the evolutionary history and genetic distinctness of the newly described grey-faced sengi or elephant-shrews (<i>Rhynchocyon udzungwensis</i> Rathbun and Rovero 2008) and its relationship to the parapatric chequered sengi (<i>Rhynchocyon cirnei reichardi</i> Peters 1847) in the Udzungwa Mountains of Tanzania. Evolutionary relationships within the genus <i>Rhynchocyon</i> are poorly understood, and the relationship of the newest member to other East African species and subspecies is of great interest to conservation efforts for preserving unique lineages in the face of small population sizes and human degradation of habitat.</p> <p>The grey-faced sengi, <i>R. udzungwensis,</i> was described in 2008 <a href="#pone.0072506-Rovero1">[2]</a> as the largest member of a poorly known group of African mammals of ancient and intriguing evolutionary history. Sengis are an ancient monophyletic order (Macroscelidea) of 18 extant species, endemic to Africa <a href="#pone.0072506-Rathbun1">[3]</a>, <a href="#pone.0072506-Dumbacher1">[4]</a>. Based largely on molecular data, sengis belonging to the super-cohort Afrotheria, which also includes elephants, hyraxes, sea cows (Paenungulata), tenrecs and golden moles (Afrosoricida), and the aardvark (Tubulidentata) <a href="#pone.0072506-Springer1">[5]</a>–<a href="#pone.0072506-Seiffert1">[7]</a>. The sengi order consists of two subfamilies: the forest-dwelling giant sengis (subfamily Rhynchocyoninae, single genus <i>Rhynchocyon,</i> species = 4), and the soft-furred, arid adapted sengis (subfamily Macroscelidinae, three genera: <i>Macroscelides</i> (species = 2), <i>Petrodromus</i> (species = 1), and <i>Elephantulus</i> (species = 11)). While a large-scale phylogeny of most sengis has been completed <a href="#pone.0072506-Smit1">[8]</a>, little is known about the taxonomy and phylogenetic relationships of the giant sengis (<i>Rhynchocyon)</i><a href="#pone.0072506-Rathbun1">[3]</a>, <a href="#pone.0072506-Coals1">[9]</a>. Corbet &amp;amp; Hanks <a href="#pone.0072506-Corbet1">[10]</a> recognized three <i>Rhynchocyon</i> species based on morphological differences and allopatric distribution (four are now recognized with the discovery of <i>R. udzungwensis</i>). They also described six subspecies within <i>R</i>. <i>cirnei</i>, where <i>R</i>. <i>cirnei reichardi</i> occurs in the Udzungwa Mountains and the highlands of southwestern Tanzania and northern Malawi. This taxonomy, however, is based nearly exclusively on pelage patterns and is debatable in some cases <a href="#pone.0072506-Rathbun1">[3]</a>, <a href="#pone.0072506-Kingdon1">[11]</a>. While sufficient sampling for a robust phylogeny of <i>Rhynchocyon</i> is currently beyond our grasp because giant sengis are elusive and sufficient samples are not available, we now have adequate material to investigate the relationship between two parapatric lineages: the newly described <i>R. udzungwensis</i> and the more widespread <i>R. cirnei reichardi.</i></p> <p><i>Rhynchocyon udzungwensis</i> has an extremely small range (ca. 390 km<sup>2</sup>) limited to two forests in the northern Udzungwa Mountains in Tanzania: the western Ndundulu-Luhomero forest and the eastern Mwanihana forest, which are separated by a ∼25 km gap of lower elevation and drier woodland <a href="#pone.0072506-Rovero1">[2]</a>. The range of <i>R. udzungwensis</i> is nearly encompassed by the range of <i>R. cirnei reichardi</i>, and the two taxa present partial overlap in elevation (<i>R</i>. <i>udzungwensis</i> occurring between 350 and 2300 meters elevation and <i>R</i>. <i>c</i>. <i>reichardi</i> between 290 and 1800 meters in the Udzungwa Mountains <a href="#pone.0072506-Rovero1">[2]</a>, <a href="#pone.0072506-Corbet1">[10]</a>). The distributions of the two taxa meet only along the central part of the eastern Mwanihana forest (<a href="#pone.0072506-Rovero2">[12]</a>; <a href="#pone-0072506-g001">Fig. 1</a>), with no indication of physical barriers, abrupt habitat changes, or steep rises. While there is no evidence of intermediate pelage colouration between these two taxa that might suggest hybridization (based on camera-trapping images, sightings, and voucher specimens), the physical proximity of these congeneric lineages, coupled with small population sizes estimated for <i>R. udzungwensis</i>, led us to include the relationship with <i>R. c. reichardi</i> when investigating <i>R. udzungwensis’</i> evolutionary history.</p> <div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/10.1371/journal.pone.0072506.g001"><span class="label" tagx="label" title="label">Figure 1</span></a></div>   <div class="title" tagx="title" title="title">
Map of the north-eastern portion of the Udzungwa Mountains, south-central Tanzania with sampling localities.</div> <p>Distribution records of the grey-faced sengi, <i>Rhynchocyon udzungwensis,</i> shown as white dots. The occurrence of the checkered sengi, <i>R. cirnei reichardi,</i> shown as black triangles. The western portion of the distribution of <i>R. udzungwensis</i> is in Ndundulu/Luhomero forest, and the eastern portion is in Mwanihana forest. <i>Rhynchocyon c. reichardi’s</i> distribution extends into the Magombera forest on the eastern plain. The five sampling sites for molecular analysis of the two species are also shown. The background layer is a topographic map (dark is lower elevation). The inset shows the location of the Udzungwa Mountains in Tanzania (left) and of the mapped area within the Udzungwa Mountains (right). Insert reprinted from <a href="#pone.0072506-Marshall1">[75]</a> under a CC BY license, with permission from A. Marshall, original copyright 2005.</p>   </div> <p>We began this study with a simple investigation of genetic distinction between the two morphological lineages, and then added additional molecular data, species tree analyses, and current and historical species distribution modelling (SDM; see <a href="#pone.0072506-Guisan1">[13]</a> for a review) in order to provide a scenario that could accommodate non-bifurcating molecular data. In this way, we can determine how historical scenarios of contact, isolation, and introgression have shaped evolution for rare and isolated montane fauna, and better understand evolutionary dynamics in a biodiversity hotspot. Species with stable, allopatric distributions are not expected to be prone to hybridization, but species with historic range shifts tracking fluctuating climatic conditions may have experienced temporary sympatry and associated genetic introgression <a href="#pone.0072506-Parmesan1">[14]</a>–<a href="#pone.0072506-Garroway1">[17]</a>. While climate-driven range shifts have been documented in a wide variety of settings, the most pronounced shifts are expected in montane species, as climatic similarity may correlate with distant montane areas rather than nearby lowlands <a href="#pone.0072506-Hewitt1">[18]</a>, <a href="#pone.0072506-Guralnick1">[19]</a>. In particular, by using SDM through time, we are able to assess the potential of overlap between sengis during both current and past time periods to accommodate the possibility of low-level historical and/or current introgression <a href="#pone.0072506-Nyari1">[20]</a>.</p> </div> <div class="materials_methods" title="materials|methods"> <div class="title" tagx="title" title="title">
Materials and Methods</div> <div class="ethicsstatement" title="sec"> <div class="title" tagx="title" title="title">
Ethics Statement</div> <p>All field procedures involving live animals met the standards for the ethical and humane treatment of animals of the American Society of Mammalogists and the 2000 American Veterinary Medical Association guidelines <a href="#pone.0072506-Sikes1">[21]</a>, <a href="#pone.0072506-AVMA1">[22]</a>, and all precautions to minimize pain, distress or suffering by trapped animals were taken. Vouchered animals were euthanized using chloroform, as approved by the Guidelines of the American Society of Mammalogists for the use of wild mammals in research <a href="#pone.0072506-Sikes1">[21]</a>. All fieldwork was performed under research/collecting permit number 2008-311-ER-2008-150 to SR and permit number 2010-270-ER-2009-49 to FR, issued by the Tanzania Wildlife Research Institute. Research and collection permits were also issued by the Tanzania Commission for Science and Technology and by National Parks. Materials were exported under Tanzania Wildlife Division/CITES Office export permits #57189 and #58267 to SR.</p> </div> <div class="tissuesampling" title="sec"> <div class="title" tagx="title" title="title">
Tissue Sampling</div> <p>We collected tissues for molecular analysis from 31 specimens in Tanzania from 2006–2009: <i>R. udzungwensis</i> (n = 22, localities = 3), <i>R</i>. <i>cirnei reichardi</i> (n = 8, localities = 2), and one outgroup sengi, black-and-rufous sengi (<i>Rhynchocyon petersi,</i> n = 1, locality = riparian forest on the north side of the Rufiji river, 150 km east of the Udzungwa Mountains) (<a href="#pone-0072506-g001">Fig. 1</a>). For live animals to be released, tissue collection consisted of a small amount of hair along with a tiny piece of auricle. The ear was then disinfected with betadine and the animal was monitored for a few minutes to ensure no adverse reaction. Dead specimens and specimens destined for voucher had fresh muscle, liver, or kidney tissues preserved in 95% ethanol. Tissue samples for genetic analysis are stored at the Foundation Edmund Mach in Trento, Italy. A complete list of sampled specimens and sampling localities can be found in <a href="#pone.0072506.s004">Table S1</a>.</p> <p>We trapped sengis with nylon twin snares <a href="#pone.0072506-Rovero1">[2]</a> and double-door live traps (Tomahawk Live Traps, 24×6×6 inch) that were set where sengi were likely to travel. Capture and handling of specimens followed the Animal Care and Use Committee of the American Society of Mammalogists Guidelines. Voucher specimens were collected (<i>R. udzungwensis</i>: n = 8 (of which 4 were specimens taken for the description and of these, 3 were given to other museums), <i>R. cirnei reichardi</i>: n = 5 (including one given to California Academy of Sciences) and were prepared as study skins and skulls.</p> </div> <div class="geneticanalyses" title="sec"> <div class="title" tagx="title" title="title">
Genetic Analyses</div> <p>Genomic DNA was extracted with the DNeasy Tissue kit (Qiagen, Valencia, USA) following manufacturer’s instructions. We used PCR to amplify fragments of three mitochondrial loci: NADH dehydrogenase 2 (ND2: Met-1 and Trp-2 <a href="#pone.0072506-Olson1">[23]</a>), the hypervariable 5′ end of the control region (D-loop: L0-F <a href="#pone.0072506-Douzery1">[24]</a> and E3 <a href="#pone.0072506-Huchon1">[25]</a>) and the ribosomal gene 12S (12S –F &amp;amp; 12S-R, <a href="#pone.0072506-Kocher1">[26]</a>) totalling 1756 bp. For the mtDNA markers we used 10 ng of DNA and PCR conditions were as follows: 94°C for 10 min, followed by 35 cycles of 94°C for 1 min, 55°C for 45 sec, and 72°C for 1 min, with a final extension of 72°C for 10 min. To avoid comparing paralogs, we tested for the presence of numts (for details see <a href="#pone.0072506.s005">Text S1</a>). Starting from 50 ng of extracted DNA, we also amplified portions of two nuclear protein-coding genes, von Willebrand factor (vWF; 1100 bp) and Enamelin (ENAM; 2598 bp), using primers and PCR conditions as in <a href="#pone.0072506-Douady1">[27]</a>. During both DNA extraction and amplification, blank samples were inserted to check for contamination. All sequence data are deposited in GenBank (Accession Nos. KF202138–KF202289).</p> <p>Sequencing was performed with the same set of primers used for PCR, using Big Dye terminator cycle sequencing with an ABI 3730×l. Sequences were edited and assembled with Sequencher 4.8 (Gene Codes Corporation). All sequences aligned unambiguously, but we additionally evaluated alignments in MUSCLE (<a href="http://www.ebi.ac.uk/Tools/msa/muscle">http://www.ebi.ac.uk/Tools/msa/muscle</a>; <a href="#pone.0072506-Edgar1">[28]</a>. Heterozygous genotypes in the nuclear gene data were resolved using PHASE 2.1.1 <a href="#pone.0072506-Stephens1">[29]</a>, <a href="#pone.0072506-Stephens2">[30]</a>, under the default options of 100 iterations, 1 thinning interval, 100 burn-in iterations, and confidence probability thresholds of 0.90. One nuclear allele from each individual was chosen at random to represent the individual in species-tree analyses.</p> <p>We concatenated all three mitochondrial loci to form a single mitochondrial dataset for phylogenetic analyses except where specified otherwise. Models of sequence evolution for each locus were selected using the corrected Akaike information criterion (AICc) as implemented in jModelTest <a href="#pone.0072506-Posada1">[31]</a> (<a href="#pone.0072506.s005">Text S1</a>). We assessed the performance of all Bayesian analyses (convergence and stationarity) with the program Tracer v. 1.5 <a href="#pone.0072506-Drummond1">[32]</a>. Only runs with adequate mixing and ESS above 200 were considered for final analyses. <i>Rhynchocyon petersi</i> was included as an outgroup in initial analyses and removed when inclusion failed to alter relationships between <i>R. c. reichardi</i> and <i>R. udzungwensis</i>.</p> <div class="singlegenetreeanalysis" title="sec"> <div class="title" tagx="title" title="title">
Single gene tree analysis</div> <p>Each locus was individually analysed using Bayesian inference in MrBayes, <a href="#pone.0072506-Ronquist1">[33]</a>. All analyses were run for 200 million Markov chain Monte Carlo (MCMC) generations, sampling every 1000th tree. Half of the resulting trees were discarded as burnin. Maximum clade credibility trees were calculated in TreeAnnotator from Beast v. 1.6.2 <a href="#pone.0072506-Drummond1">[32]</a>.</p> </div> <div class="speciestreeconstruction" title="sec"> <div class="title" tagx="title" title="title">
Species tree construction</div> <p>We employed several methods for estimating species trees with different assumptions on ILS (Incomplete Lineage Sorting) and horizontal gene transfer HGT to accommodate the possibility that introgression and HGT may be a substantial part of the evolutionary lineages of at least some populations within these species. Three species tree methods, BEST (Bayesian Estimation of Species Trees: <a href="#pone.0072506-Liu1">[34]</a>, <a href="#pone.0072506-Liu2">[35]</a>, *BEAST <b>(</b>Bayesian Evolutionary Analysis Sampling Trees: <a href="#pone.0072506-Heled1">[36]</a>, and BP&amp;amp;P (Bayesian Phylogenetics &amp;amp; Phylogeography: <a href="#pone.0072506-Rannala1">[37]</a>, <a href="#pone.0072506-Yang1">[38]</a> have been shown to accurately estimate phylogenetic (species tree) relationships in spite of gene tree heterogeneity. They assume, however, that ILS is the only cause of gene tree-species tree discordance. To complement these assumptions with a model that is agnostic to the causes of gene tree heterogeneity, we also analysed our dataset in BUCKy (Bayesian Untangling of Concordance Knots, Bayesian concordance analysis: <a href="#pone.0072506-An1">[39]</a>–<a href="#pone.0072506-Larget1">[41]</a>, which can be used to reject ILS as sufficient to explain gene tree discordance. If ILS cannot be rejected as the cause of mismatch between our datasets, then the species-trees produced from *BEAST and BEST should be good estimates of the true species tree without invoking HGT to explain discordance of genetrees. If BUCKy rejects ILS as sufficient to explain genetree discordance, the individual gene trees can be assessed to determine likely transgenic events <a href="#pone.0072506-Chung1">[42]</a>.</p> <p>BEST v2.3.1 is a hierarchical coalescent model implemented in a two-step MCMC algorithm. The first step estimates the posterior probability distributions of gene trees for each locus in MrBayes, and the second step uses these probabilities to estimate the posterior probability distribution of species trees. In our BEST analysis, we obtained Bayesian posterior probabilities from 100 million MCMC cycles with a sample frequency of 1,000 and a burn-in period of 25 million generations, with a relatively flat prior for θ (α = 3, β = 0.03) as employed in <a href="#pone.0072506-Leach1">[43]</a>, <a href="#pone.0072506-Lee1">[44]</a>. Models of molecular evolution for each locus were defined from the highest scoring models in jModeltest available in BEST (mtDNA: nst = 2, haploid, Invgamma; ENAM: nst = 2, diploid, Invgamma; vWF: nst = 2, diploid, Equal).</p> <p>The program *BEAST v. 1.6.2 was used to jointly estimate the posterior distributions of species trees and contained gene trees using coalescent models. The *BEAST analysis was conducted under a strict molecular clock model (no loci violated a strict clock assumption, data not shown) using the SRD06 model of sequence evolution <a href="#pone.0072506-Shapiro1">[45]</a> and a random starting tree. As no estimated mutation rates or calibrated phylogenies are available for this clade, relative evolutionary rates were estimated by setting the mean clock rate of mtDNA to 1.0 and allowing all other loci to be estimated in relation to this rate. The final analysis was run for 100 million generations, sampling every 1,000th generation. A maximum clade credibility tree was generated using the program Tree Annotator v.1.6.2 provided in the BEAST package, with a burn-in of 1,000 (10%).</p> <p>We used the hyperresolved (population level) species tree generated in *BEAST as the user-specified "guide tree" for species tree estimation in the program BPP, v. 2.0. Population size parameters (qs) and the age of the root of the species tree (t0) were assigned gamma priors (α = 2, β = 1000), while all other divergence time parameters are assigned the Dirichlet prior <a href="#pone.0072506-Yang1">[38]</a>. Each analysis was run twice to confirm consistency between runs. Analyses were run for 150,000 generations, with the first 50,000 discarded as burn-in and a sampling frequency of 5. The species delimitation algorithms 0 and 1 were both run for this dataset (species delimitation: 1 0 5, reported). A variety of algorithms, <i>a</i>’s, <i>e</i>’s, and <i>m</i>’s were explored (algorithm 0, e = 5 &amp;amp; 20. algorithm 1, α = 1 &amp;amp; 2, <i>m = </i>0.5, 1, 2) with similar results (<a href="#pone.0072506.s005">Text S1</a>). All five loci (3 mtDNA and 2 nuDNA) were allowed to vary independently in this analysis. Finetune variables were adjusted in the control file for each run so that the acceptance proportions were contained in the interval (0.15, 0.7).</p> <p>BUCKy v.1.4.0 was used to estimate a primary concordance tree and a population tree from genetrees estimated in MrBayes. Analyses were run for 2 million generations with four different heating chains after a 200,000 generation burn-in. Two extremes of α were explored (α = 0.5, α = 10) with no change to our results. To determine whether HGT appears to play a major role within this region, we compare the Population Tree (PT) and the Primary Concordance Tree (PCT). If ILS is the sole source of gene tree conflict, the two trees should be in tight concordance <a href="#pone.0072506-Degnan1">[46]</a>. If, however the Population Tree and Primary Concordance Tree are in disagreement, this is evidence of hybridization <a href="#pone.0072506-Chung1">[42]</a>.</p> </div> </div> <div class="speciesdistributionmodels(sdms)" title="sec"> <div class="title" tagx="title" title="title">
Species Distribution Models (SDMs)</div> <div class="occurrencedata" title="sec"> <div class="title" tagx="title" title="title">
Occurrence data</div> <p>Data on the presence of <i>R. udzungwensis</i> and <i>R. cirnei reichardi</i> were collected from live traps and camera traps <a href="#pone.0072506-Rovero2">[12]</a>, museum records, and personal communications (<a href="#pone-0072506-g001">Fig. 1</a> and <a href="#pone.0072506.s005">Text S1</a>) for a total of 32 and 172 localities for grey-faced and chequered sengi, respectively. To reduce taxonomic misidentification of sengis used in the modelling effort, only data from Tanzania were used to avoid uncertainty of identification and distribution of <i>R. c. hendersoni</i> in relation to <i>R. c. reichardi</i><a href="#pone.0072506-Kingdon1">[11]</a>. Similarly, samples of <i>R. c. reichardi</i> from the western border of Lake Tanganyika were eliminated from our analyses due to uncertainty regarding the affinity with southern Tanzanian samples. Though the recently discovered <i>Rhynchocyon</i> in the northern coastal forests of Kenya <a href="#pone.0072506-Andanje1">[47]</a> has pelage colouration and patterning very similar to <i>R. udzungwensis</i>, we have not included it in our analyses because it has not been adequately sampled or described.</p> </div> <div class="environmentaldata" title="sec"> <div class="title" tagx="title" title="title">
Environmental data</div> <p>Climate layers were acquired from the WorldClim data set <a href="#pone.0072506-Hijmans1">[48]</a>. Three time points representing the recent climate extremes of glacial and interglacial cycles were assessed: current, Last Glacial Maximum (LGM –21 kya, Community Climate System Model: <a href="#pone.0072506-Collins1">[49]</a>, and Last Interglacial (LIG –120 kya: <a href="#pone.0072506-OttoBliesner1">[50]</a>. While these specific time points (21 kya and 120 kya) may not be directly related to range shifts or genetic events concerning these species, by representing well documented dry and wet extremes known to have impacted forest habitat and extent in East Africa, we can approximate the range of distributions experienced by these species. We ran an initial correlation assessment of Bioclim layers at each time point (calculated using ENMtools, v 1.3, <a href="#pone.0072506-Warren1">[51]</a> to attempt to reduce the number of auto-correlated variables which have the potential of over-fitting models of distribution predictions. As these correlations were not stable between time points or between species, all 19 variables were retained for analyses to avoid under-fitting the model. All layers were clipped to both the extent of Tanzanian boundaries and to a rectangle encompassing the area of species overlap in the Udzungwa Mountains (Latitude: −6° to −9.25°, Longitude: 35° to 37.25°) in ArcGIS (ESRI v.10.1).</p> </div> <div class="speciesdistributionmodels(sdms)" title="sec"> <div class="title" tagx="title" title="title">
Species Distribution Models (SDMs)</div> <p>Species distribution models were created with MaxEnt version 3.3.3 <a href="#pone.0072506-Phillips1">[52]</a>. MaxEnt was chosen because of its ability to make accurate predictions of species distributions on small and spatially limited datasets <a href="#pone.0072506-Hernandez1">[53]</a>, <a href="#pone.0072506-Hernandez2">[54]</a>. All MaxEnt runs were trained on the current climate conditions with a convergence threshold of 0.00001, 1,000 iterations, 25% model testing, logistic output, and the 10% training presence threshold for outputs to create binary models of habitat suitability. Outputs were then projected onto climate models of the LGM and LIG. ENMtools was used to evaluate the similarity of the environmental niche occupied by the two lineages using the Niche Overlap, Niche Identity, and Background functions (200 randomized pseudoreplicates). Background comparisons were completed on using both the distributions envelope of each species (minus the other’s range) from which to generate random points, and the larger shared distribution of the limited area rectangle. The hypothesis of identical climatic niche is rejected when the empirically observed values of identity (D and I statistic: <a href="#pone.0072506-Schoener1">[55]</a>, <a href="#pone.0072506-Warren2">[56]</a> respectively) are significantly lower than the range of pseudoreplicate values <a href="#pone.0072506-Warren1">[51]</a>. Range overlap through time was calculated in ArcMap.</p> </div> </div> </div> <div class="results" title="sec"> <div class="title" tagx="title" title="title">
Results</div> <div class="molecularanalysis" title="sec"> <div class="title" tagx="title" title="title">
Molecular Analysis</div> <div class="genetrees" title="sec"> <div class="title" tagx="title" title="title">
Gene trees</div> <p>Nuclear loci were in agreement concerning the monophyly of <i>R. udzungwensis</i>. <i>Rhynchocyon c. reichardi</i> has strong support of monophyly for vWF, but an uncertain relationship to the <i>R. udzungwensis</i> clade in ENAM. Neither species showed intraspecific population structure for nuclear loci (<a href="#pone-0072506-g002">Fig. 2</a>, top). The concatenated mitochondrial loci, however, showed substantial paraphyly of the two species, particularly in regards to <i>R. c. reichardi</i> from Magombera and <i>R. udzungwensis</i> from Mwanihana forest, which both possessed mtDNA haplotypes from both major clades (<a href="#pone-0072506-g002">Fig. 2</a>, bottom). Dating divergence events between these lineages is currently impossible without a full phylogeny and fossil representation, but future work will help establish the timing of separation of these species and the approximate timing of genetic transfer between species.</p> <div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/10.1371/journal.pone.0072506.g002"><span class="label" tagx="label" title="label">Figure 2</span></a></div>   <div class="title" tagx="title" title="title">
Gene Trees and Haplotype networks.</div> <p>Bayesian Gene trees of two nuclear loci (top: ENAM and vWF) and concatenated mtDNA (bottom left: ND2, <i>D-loop</i> region, and the ribosomal gene 12S) constructed in MrBayes. Posterior probabilities displayed for major nodes. Branch lengths proportional to genetic distance. (Bottom Right) Haplotype network of concatenated mtDNA. Each node represents one base pair change.</p>   </div> </div> <div class="speciestrees" title="sec"> <div class="title" tagx="title" title="title">
Species trees</div> <p>Species tree methods were in agreement concerning the monophyly of the species and phylogeographic relationships within species, with strong support (posterior = 1.00 for all three methods) for the basal split between the two species (<a href="#pone-0072506-g003">Fig. 3</a>). The BUCKy analysis, which was run with individuals as the terminals instead of populations as in the other species trees (e.g., <a href="#pone.0072506-Lee1">[44]</a>), showed monophyly in the Population Tree, yet extreme paraphyly in the Primary Concordance Tree (<a href="#pone-0072506-g004">Fig. 4</a>). As the CFs (concordance factors, which measure the genomic support of each clade) of the PCT were low, it is likely that many alternative topologies are possible for each node, consistent with Horizontal Gene Transfer.</p> <div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/10.1371/journal.pone.0072506.g003"><span class="label" tagx="label" title="label">Figure 3</span></a></div>   <div class="title" tagx="title" title="title">
Highest probability species tree selected by all three methods (*BEAST, BPP, and BEST).</div> <p>Posterior probabilities shown for each node. Both species are monophyletic for all species tree reconstructions. Branch lengths proportional to genetic distance.</p>   </div> <div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/10.1371/journal.pone.0072506.g004"><span class="label" tagx="label" title="label">Figure 4</span></a></div>   <div class="title" tagx="title" title="title">
Bayesian concordance analysis in BUCKy: Population Tree (top), Primary concordance tree (bottom).</div> <p>Posterior mean concordance factors at major nodes are displayed above branches. Branches with concordance factors below 0.2 are not shown except in major nodes. The population tree shows monophyly for the two species, while the primary concordance tree shows admixture between the species and poorly supported relationships implying some level of admixture.</p>   </div> </div> </div> <div class="speciesdistributionmodels" title="sec"> <div class="title" tagx="title" title="title">
Species Distribution Models</div> <div class="nicheoverlapbetweenspecies" title="sec"> <div class="title" tagx="title" title="title">
Niche overlap between species</div> <p>Geographic scale did not significantly impact the current distribution predictions or niche similarity estimates for these species, as models based on the full extent of Tanzania and models generated from just the rectangular region around the Udzungwa Mountains and Southern Highlands yielded similar results (<a href="#pone.0072506.s001">Fig. S1</a>). Projections into past climate conditions were less stable between the two scales, so results are interpreted from the limited distribution rectangle with insights from the larger scale. The AUC (Area Under the Curve) in the rectangle encompassing the Udzungwa Mountains was 0.901 for <i>R. c. reichardi</i>, and 0.994 for <i>R. udzungwensis</i>. The niches of the two species (<i>R. udzungwensis</i> and <i>R. c. reichardi</i>) are significantly more dissimilar than chance by all similarity measures (P&amp;lt;0.001; <a href="#pone.0072506.s002">Fig. S2</a>, <a href="#pone.0072506.s003">S3</a>). As expected given their separate distributions and elevation separation, all models confirmed that they currently occupy different niches despite their close spatial distribution.</p> </div> <div class="distributionsandshifts" title="sec"> <div class="title" tagx="title" title="title">
Distributions and shifts</div> <p>Under current climate conditions, <i>R. c. reichardi</i> has a much larger and more continuous distribution in the region of interest than does <i>R. udzungwensis</i> (<a href="#pone-0072506-g005">Fig. 5</a>, left), which appears to be limited to the two extant forest blocks. The distributions during past wet climate cycles (exemplified by the LIG) show increased suitability throughout the region for <i>R. c. reichardi</i>. Instead of expanding in a similar manner to <i>R. c. reichardi</i>, <i>R. udzungwensis</i> appears to experience a shift in range towards eastern, lower-elevation forests. Instead of the current distribution with one population in a western high elevation forest (Ndundulu-Luhomero) and one population in a central mid-elevation forest (Mwanihana), the LIG distribution consists of populations in the central mid-elevation (Mwanihana) and the eastern lower-elevation forest of Magombera (currently only occupied by <i>R. c. reichardi</i>) (<a href="#pone-0072506-g005">Fig. 5</a>, right). The distribution of both species during dry climate cycles (exemplified by the LGM) was likely greatly reduced, as our climate models fail to predict any substantial areas of suitable habitat within this region for either species.</p> <div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/10.1371/journal.pone.0072506.g005"><span class="label" tagx="label" title="label">Figure 5</span></a></div>   <div class="title" tagx="title" title="title">
Species Distribution Models during current climate scenario (left) and wetter Last Interglacial (right; LIG).</div> <p>Current sampling points used in model construction are shown. The limited distribution of <i>R. udzungwensis</i> is entirely contained within the suitable area for <i>R. c. reichardi</i> at both time points. During the LIG, the ranges for both species were predicted to occur at lower elevations in the eastern areas of the Udzungwa Mountains foothills. During this time, <i>R. udzungwensis</i> shifted its entire distribution eastward resulting in a loss occupancy in the western Ndundulu/Luhomero forest, a reduced range in the Mwanihana forest, and a relatively extensive distribution east of Mwanihana in the Kilombero Valley including the Magombera forest (see Fig. 1 for place names).</p>   </div> </div> </div> </div> <div class="discussion" title="sec"> <div class="title" tagx="title" title="title">
Discussion</div> <p>Species-tree methods confirm the genetic distinctness of the two species with very high support, validating the morphological description of <i>R. udzungwensis</i><a href="#pone.0072506-Rovero1">[2]</a>. This is a first, important result given the complex phylogeny of the genus. However, the discrepancy between the BUCKy population tree and primary concordance tree, and the clear introgression of mtDNA haplotypes despite nuclear monophyly imply that ancient hybridization occurred between these two taxa. The absence of current admixture inferred from monophyletic nuclear loci is supported by the fact that despite the close proximity of <i>R. c. reichardi</i> and <i>R. udzungwensis</i> populations in the Udzungwa Mountains, no morphologically intermediate individuals (presumed hybrids) have been found. Given the spatial proximity of these species, apparent mitochondrial introgression, and the narrow habitat requirements of <i>R. udzungwensis</i>, four related questions arise: (1) Why is there no evidence of nuclear gene introgression; (2) Why do some populations of <i>R. udzungwensis</i> exhibit ancient hybridization while others do not; (3) Why is there no evidence of current hybridization between the two species; and (4) Why do SDM models predict a near complete lack of suitable habitat in this area during the LGM?</p> <div class="mitochondrialintrogression" title="sec"> <div class="title" tagx="title" title="title">
Mitochondrial Introgression</div> <p>The phenomenon of extensive mitochondrial introgression without a comparable nuclear signal is relatively common in phylogenetic studies (e.g., <a href="#pone.0072506-Funk1">[57]</a>–<a href="#pone.0072506-Spinks1">[60]</a>, exemplified by a stark disagreement between resolved nuclear genes and admixed mitochondrial loci. Given the ∼4-fold increase in mutation rate for mtDNA vs. nuclear <a href="#pone.0072506-Moore1">[61]</a>, <a href="#pone.0072506-Palumbi1">[62]</a>, these patterns are unlikely to be a result of incomplete lineage sorting <a href="#pone.0072506-Hudson1">[63]</a>, particularly when two morphologically and genetically distinct lineages share exact mitochondrial haplotypes. Though data are extremely limited within this sengis, no examples of incomplete lineage sorting of mtDNA have been found <a href="#pone.0072506-Dumbacher1">[4]</a>, <a href="#pone.0072506-Smit2">[64]</a>.</p> <p>The signal of mitochondrial introgression can persist between species, despite loss of a nuclear signal when two potentially hybridizing species meet in circumstances of disparate abundance. In these uneven encounters, normally choosy females of the less abundant species may accept heterospecific mates if conspecifics mates are rare <a href="#pone.0072506-Chan1">[58]</a>. In these instances, though both nuclear and mitochondrial genes from the rare species introgress into the common species, mitochondrial introgression invariably exceeds nuclear. This is particularly apparent when the proportion of immigrants is small <a href="#pone.0072506-Chan1">[58]</a>.</p> </div> <div class="introgressedandpurepopulations" title="sec"> <div class="title" tagx="title" title="title">
Introgressed and Pure Populations</div> <p>Historical uneven encounters could be expected in some populations of <i>R. udzungwensis</i> and <i>R. c reichardi</i>, while others may have remained isolated from introgression throughout history. Evidence from this study suggests that some populations have undergone introgression while others have not: (1) mtDNA of <i>R. udzungwensis</i> from the Luhomero forests appears introgressed into <i>R. c. reichardi</i> individuals from Magombera forest, (2) apparent mtDNA haplotypes from <i>R. c. reichardi</i> appear in <i>R. udzungwensis</i> individuals from Mwanihana, and (3) <i>R. udzungwensis</i> from Ndundulu appear isolated and free from admixture despite the fact that the Ndundulu forests are equally surrounded by <i>R. c. reichardi</i> as the other fragments occupied by <i>R. udzungwensis</i>.</p> <p>The spatial distribution of habitat during both current and LIG climate conditions, and the unique nature of the Ndundulu forests offer possible explanations for spatially structured introgression. The Ndundulu forest appears to act as an ark of diversity for a number of forest-restricted species, providing stable habitat through time for its unusually divergent occupants (e.g., Udzungwa forest partridge (<i>Xenoperdix udzungwensis</i>, <a href="#pone.0072506-Dinesen1">[65]</a>–<a href="#pone.0072506-Bowie1">[67]</a>), kipunji (<i>Rungwecebus kipunji,</i><a href="#pone.0072506-Davenport1">[68]</a>, <a href="#pone.0072506-Roberts1">[69]</a>, and rufous-winged sunbird (<i>Cinnyris rufipennis</i>, <a href="#pone.0072506-Dinesen2">[70]</a>). The reason that distinct and relictual species are preserved in the Ndundulu forest is not understood, but perhaps various environmental aspects filter generalist species from invading and displacing or intermingling with local lineages.</p> <p>Together, the current parapatric distribution, population-specific mtDNA introgression, and the predicted historical ranges of these species during interglacial cycles support a scenario of hybridization with <i>R. c. reichardi</i> through uneven encounters. As species ranges shifted in response to glacial climate cycles, rare colonists of displaced <i>R. udzungwensis</i> may have encountered abundant and established <i>R. c. reichardi</i> in the newly accessible flood plain forest of which Magombera is today a remnant (<a href="#pone-0072506-g005">Fig. 5</a>) leading to a transfer of mtDNA haplotypes into <i>R. c. reichardi</i> populations <a href="#pone.0072506-McPeek1">[71]</a>. Likewise, <i>R. c. reichardi</i> may have expanded into the northern Mwanihana forests as <i>R. udzungwensis’</i> range contracted, leading to the mtDNA introgression from <i>R. c. reichardi</i> into <i>R. udzungwensis</i>. These scenarios of predicted relative abundance (i.e., "rare colonists" and "abundant established") reflect stable range distributions of one species (based on current and LIG models) vs. newly occupied areas for the other. With the current data, much of this interpretation is speculative, but fits with known models of interactions. Future studies incorporating larger sampling of the diversity within <i>R. c. reichardi</i> and <i>R. cirnei</i> as a whole will help to clarify these hypotheses.</p> </div> <div class="currentgeneticisolation" title="sec"> <div class="title" tagx="title" title="title">
Current Genetic Isolation</div> <p>SDM show that the current <i>R. udzungwensis</i> range is completely within the predicted, and much broader, range of <i>R. c. reichardi</i>, yet no on-going hybridization appears to be taking place. In situations such as this, where one species is predicted to occupy the range of another yet is distinctly absent from that area, competition coupled with reinforcement for potentially hybridizing species is typically invoked. It is likely that the larger-bodied <i>R. udzungwensis</i> is able to outcompete <i>R. c. reichardi</i> at the higher/harsher elevations that it currently occupies, but <i>R. c. reichardi</i> is better able to utilize resources in the available landscape surrounding the <i>R. udzungwensis</i> refugia. In this case, gradual historic climate change may explain the trend towards reduced introgression at the present by reinforcing the concentration of <i>R. udzungwensis</i> in its wetter, highland refugia if it cannot compete in the drier mid-elevation areas occupied by <i>R. c. reichardi</i>. Though East Africa experienced many climate fluctuations in the past 5 my, oscillating from wet to dry climate conditions with associated forest expansions and contractions, there has been a general drying trend from 1.86 (+/−0.44 Ma) to present (ODP 721/722 dust flux record, <a href="#pone.0072506-Trauth1">[72]</a>. From this point on, the range of <i>R. udzungwensis</i> would have concentrated in the highland forests of the Udzungwa Mountains tops instead of the expanded eastern range with <i>R. c. reichardi</i>. This scenario is difficult to verify without absolute dating of the larger phylogeny and the proposed introgression event (an ongoing effort of these authors and others), but is likely the levels of divergence and other phylogenetic and phylogeographic patterns in East Africa (discussed in <a href="#pone.0072506-Lawson1">[73]</a>).</p> </div> <div class="sdmmodelpredictions" title="sec"> <div class="title" tagx="title" title="title">
SDM Model Predictions</div> <p>Species distribution modelling is an invaluable tool for many aspects of evolutionary and conservation biology, but there are limitations for range-restricted species that tend to have little latitude in their environmental envelope <a href="#pone.0072506-Graham1">[74]</a>. The modelling approach in this study, MaxEnt based on bioclimatic variables, did not predict any areas of stable habitat (using a threshold cut-off of suitability) within the Udzungwa Mountains region for either sengi in this study during the LGM, a result that seems unlikely given our molecular findings. Though this may imply that populations in this area were nearly extinct, as should be investigated with a larger population genetics dataset, it is also possible that the distribution requirements of sengis are either not adequately predicted by these models or that the reconstructions of climate during the LGM may be inadequate for this region. Future improvements in both models and sengi distribution requirements will aid in our understanding of these dynamics: (1) Models of past and future climate continue to evolve, particularly in terms of sensitivity to the under-represented African tropics. These advancements will greatly enhance the understanding of climatic effects on diversity within African lineages as a whole. (2) As more ecological information becomes available for sengis including microhabitat requirements, physiological limits, and potential competitors, we may be able to provide a more sensitive model of distributions. These traits, however, have limited ability to be predicted into past and future climates.</p> </div> <div class="advancingourunderstandingofsengis" title="sec"> <div class="title" tagx="title" title="title">
Advancing our Understanding of Sengis</div> <p>This study is the first known case of introgression in sengis, a result that was only identifiable using multi-locus data from multiple populations. Previous investigations into relationships of co-distributed sengis failed to detect introgression, but lacked multi-locus data <a href="#pone.0072506-Dumbacher1">[4]</a>, <a href="#pone.0072506-Smit1">[8]</a>. A multi-locus perspective will be necessary in studies of sengis, including phylogenetic analyses, to adequately interpret interspecific relationships.</p> <p>In the current study, we are able to see a clear signal of introgression using multiple individuals from multiple localities. In future studies, increased sample sizes sufficient to investigate demographic events (such as expansions and contractions) and estimate population diversity across ranges would both verify proposed demographic processes and ground our estimates of the extent of introgression between species.</p> <p>This study demonstrates that multi-locus, population-level sampling including introgression analyses is imperative for any attempts at investigating the broader relationships and evolutionary history of <i>Rhynchocyon</i>. Within this, all subspecies, pelage variants, and allopatric populations should be considered for the fullest understanding of evolution and diversification within this poorly understood genus. It also highlights how comparisons with neighboring and closely related species highlight the uniqueness and complex history of a new and vulnerable giant sengi.</p> </div> </div> <div class="supplementary-material" title="supplementary-material"> <div class="title" tagx="title" title="title">
Supporting Information</div> <div class="local-data" title="local-data"><span class="label" tagx="label" title="label">Figure S1</span> <p>ENM predictions of Tanzania vs. reduced rectangle. Current predictions for <i>R. c. reichardi</i> (left) and <i>R. udzungwensis</i> (right) with Udzungwa NP and Kilombero Forest reserve outlined in black for reference.</p> <p>(TIF)</p>  <div class="media" title="media"><a href="pone.0072506.s001.tif">LINK</a> <p>Click here for additional data file.</p>  </div> </div> <div class="local-data" title="local-data"><span class="label" tagx="label" title="label">Figure S2</span> <p>Niche Identity comparisons between <i>R. udzungwensis</i> and <i>R. c. reichardi.</i></p> <p>(TIF)</p>  <div class="media" title="media"><a href="pone.0072506.s002.tif">LINK</a> <p>Click here for additional data file.</p>  </div> </div> <div class="local-data" title="local-data"><span class="label" tagx="label" title="label">Figure S3</span> <p>Similarity of <i>R. udzungwensis</i> and <i>R. c. reichardi</i> niche to backgrounds. Top of each set: Shared background of entire evaluated rectangle. Bottom of each set: compared to random points in the threshold occurrence envelope of the other species. Arrows represent actual similarity.</p> <p>(TIF)</p>  <div class="media" title="media"><a href="pone.0072506.s003.tif">LINK</a> <p>Click here for additional data file.</p>  </div> </div> <div class="local-data" title="local-data"><span class="label" tagx="label" title="label">Table S1</span> <p>Collection and identification of specimens used in this study.</p> <p>(XLS)</p>  <div class="media" title="media"><a href="pone.0072506.s004.xls">LINK</a> <p>Click here for additional data file.</p>  </div> </div> <div class="local-data" title="local-data"><span class="label" tagx="label" title="label">Text S1</span> <p>Supplementary information regarding jModelTest, BPP, exclusion of 12S numts, niche comparisons between <i>R. udzungwensis</i> and <i>R. c. reichardi.</i></p> <p>(DOCX)</p>  <div class="media" title="media"><a href="pone.0072506.s005.docx">LINK</a> <p>Click here for additional data file.</p>  </div> </div> </div> </div> <div class="back" title="back"> <div class="ack" title="ack"> <p>Several field assistants, especially Ruben Mwakisoma, helped data collection invaluably. Barbara Crestanello provided sequencing assistance and Matteo Girardi undertook a great deal of the lab work. Galen Rathbun was an advisor during initiation of the project, provided access to his sengi distributional database, and greatly improved the clarity of the manuscript. Aaron Iemma helped making the map in Fig. 1. We are grateful to the Tanzania Wildlife Research Institute, Tanzania Commission for Science and Technology, Forestry and Beekeeping Division and Tanzania National Parks for research permits.</p> </div> <div class="references">
References</div> <div tag="ref-list"> <ul> <div class="title" tagx="title" title="title">
References</div> <li tag="ref"><a name="pone.0072506-Mallet1" /><span class="label" tagx="label" title="label">1</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Mallet</span><span class="given-names" tagx="given-names" title="given-names">J</span></span> (<span class="year" tagx="year" title="year">2005</span>) <span class="mixed-article-title" title="mixed-article-title">Hybridization as an invasion of the genome</span>. <span class="source" tagx="source" title="source">Trends Ecol Evol</span><span class="volume" tagx="volume" title="volume">20</span>: <span class="fpage" tagx="fpage" title="fpage">229</span>–<span class="lpage" tagx="lpage" title="lpage">237</span><span class="comment">doi:<a href="http://dx.doi.org/10.1016/j.tree.2005.02.010">10.1016/j.tree.2005.02.010</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16701374">16701374</a></span></span></li> <li tag="ref"><a name="pone.0072506-Rovero1" /><span class="label" tagx="label" title="label">2</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Rovero</span><span class="given-names" tagx="given-names" title="given-names">F</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Rathbun</span><span class="given-names" tagx="given-names" title="given-names">GB</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Perkin</span><span class="given-names" tagx="given-names" title="given-names">A</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Jones</span><span class="given-names" tagx="given-names" title="given-names">T</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ribble</span><span class="given-names" tagx="given-names" title="given-names">DO</span></span>, <span class="etal" title="etal"><i>et al.</i></span> (<span class="year" tagx="year" title="year">2008</span>) <span class="mixed-article-title" title="mixed-article-title">A new species of giant sengi or elephant-shrew (genus <i>Rhynchocyon</i>) highlights the exceptional biodiversity of the Udzungwa Mountains of Tanzania</span>. <span class="source" tagx="source" title="source">J Zool</span><span class="volume" tagx="volume" title="volume">274</span>: <span class="fpage" tagx="fpage" title="fpage">126</span>–<span class="lpage" tagx="lpage" title="lpage">133</span><span class="comment">doi:<a href="http://dx.doi.org/10.1111/j.1469-7998.2007.00363.x">10.1111/j.1469-7998.2007.00363.x</a></span></span></li> <li tag="ref"><a name="pone.0072506-Rathbun1" /><span class="label" tagx="label" title="label">3</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Rathbun</span><span class="given-names" tagx="given-names" title="given-names">GB</span></span> (<span class="year" tagx="year" title="year">2009</span>) <span class="mixed-article-title" title="mixed-article-title">Why is there discordant diversity in sengi (Mammalia: Afrotheria: Macroscelidea) taxonomy and ecology?</span><span class="source" tagx="source" title="source">Afr J Ecol</span><span class="volume" tagx="volume" title="volume">47</span>: <span class="fpage" tagx="fpage" title="fpage">1</span>–<span class="lpage" tagx="lpage" title="lpage">13</span><span class="comment">doi:<a href="http://dx.doi.org/10.1111/j.1365-2028.2009.01102.x">10.1111/j.1365-2028.2009.01102.x</a></span></span></li> <li tag="ref"><a name="pone.0072506-Dumbacher1" /><span class="label" tagx="label" title="label">4</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Dumbacher</span><span class="given-names" tagx="given-names" title="given-names">JP</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Rathbun</span><span class="given-names" tagx="given-names" title="given-names">GB</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Smit</span><span class="given-names" tagx="given-names" title="given-names">HA</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Eiseb</span><span class="given-names" tagx="given-names" title="given-names">SJ</span></span> (<span class="year" tagx="year" title="year">2012</span>) <span class="mixed-article-title" title="mixed-article-title">Phylogeny and Taxonomy of the Round-Eared Sengis or Elephant-Shrews, Genus <i>Macroscelides</i> (Mammalia, Afrotheria, Macroscelidea)</span>. <span class="source" tagx="source" title="source">PLoS ONE</span><span class="volume" tagx="volume" title="volume">7</span>: <span class="fpage" tagx="fpage" title="fpage">e32410</span><span class="comment">doi:<a href="http://dx.doi.org/10.1371/journal.pone.0032410">10.1371/journal.pone.0032410</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22479325">22479325</a></span></span></li> <li tag="ref"><a name="pone.0072506-Springer1" /><span class="label" tagx="label" title="label">5</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Springer</span><span class="given-names" tagx="given-names" title="given-names">MS</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Cleven</span><span class="given-names" tagx="given-names" title="given-names">GC</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Madsen</span><span class="given-names" tagx="given-names" title="given-names">O</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">de Jong</span><span class="given-names" tagx="given-names" title="given-names">WW</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Waddell</span><span class="given-names" tagx="given-names" title="given-names">VG</span></span>, <span class="etal" title="etal"><i>et al.</i></span> (<span class="year" tagx="year" title="year">1997</span>) <span class="mixed-article-title" title="mixed-article-title">Endemic African mammals shake the phylogenetic tree</span>. <span class="source" tagx="source" title="source">Nature</span><span class="volume" tagx="volume" title="volume">388</span>: <span class="fpage" tagx="fpage" title="fpage">61</span>–<span class="lpage" tagx="lpage" title="lpage">64</span><span class="comment">doi:<a href="http://dx.doi.org/10.1038/40386">10.1038/40386</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/9214502">9214502</a></span></span></li> <li tag="ref"><a name="pone.0072506-Stanhope1" /><span class="label" tagx="label" title="label">6</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Stanhope</span><span class="given-names" tagx="given-names" title="given-names">MJ</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Waddell</span><span class="given-names" tagx="given-names" title="given-names">VG</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Madsen</span><span class="given-names" tagx="given-names" title="given-names">O</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">de Jong</span><span class="given-names" tagx="given-names" title="given-names">W</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hedges</span><span class="given-names" tagx="given-names" title="given-names">SB</span></span>, <span class="etal" title="etal"><i>et al.</i></span> (<span class="year" tagx="year" title="year">1998</span>) <span class="mixed-article-title" title="mixed-article-title">Molecular evidence for multiple origins of Insectivora and for a new order of endemic African insectivore mammals</span>. <span class="source" tagx="source" title="source">PNAS</span><span class="volume" tagx="volume" title="volume">95</span>: <span class="fpage" tagx="fpage" title="fpage">9967</span>–<span class="lpage" tagx="lpage" title="lpage">9972</span><span class="comment">doi:<a href="http://dx.doi.org/10.1073/pnas.95.17.9967">10.1073/pnas.95.17.9967</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/9707584">9707584</a></span></span></li> <li tag="ref"><a name="pone.0072506-Seiffert1" /><span class="label" tagx="label" title="label">7</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Seiffert</span><span class="given-names" tagx="given-names" title="given-names">ER</span></span> (<span class="year" tagx="year" title="year">2007</span>) <span class="mixed-article-title" title="mixed-article-title">A new estimate of afrotherian phylogeny based on simultaneous analysis of genomic, morphological, and fossil evidence</span>. <span class="source" tagx="source" title="source">BMC Evolutionary Biology</span><span class="volume" tagx="volume" title="volume">7</span>: <span class="fpage" tagx="fpage" title="fpage">224</span><span class="comment">doi:<a href="http://dx.doi.org/10.1186/1471-2148-7-224">10.1186/1471-2148-7-224</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17999766">17999766</a></span></span></li> <li tag="ref"><a name="pone.0072506-Smit1" /><span class="label" tagx="label" title="label">8</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Smit</span><span class="given-names" tagx="given-names" title="given-names">HA</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Jansen van Vuuren</span><span class="given-names" tagx="given-names" title="given-names">B</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">O’Brien</span><span class="given-names" tagx="given-names" title="given-names">PCM</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ferguson-Smith</span><span class="given-names" tagx="given-names" title="given-names">M</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Yang</span><span class="given-names" tagx="given-names" title="given-names">F</span></span>, <span class="etal" title="etal"><i>et al.</i></span> (<span class="year" tagx="year" title="year">2011</span>) <span class="mixed-article-title" title="mixed-article-title">Phylogenetic relationships of elephant-shrews (Afrotheria, Macroscelididae)</span>. <span class="source" tagx="source" title="source">J Zool</span><span class="volume" tagx="volume" title="volume">284</span>: <span class="fpage" tagx="fpage" title="fpage">133</span>–<span class="lpage" tagx="lpage" title="lpage">143</span><span class="comment">doi:<a href="http://dx.doi.org/10.1111/j.1469-7998.2011.00790.x">10.1111/j.1469-7998.2011.00790.x</a></span></span></li> <li tag="ref"><a name="pone.0072506-Coals1" /><span class="label" tagx="label" title="label">9</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation">Coals PGR, Rathbun GB (in press) The taxonomic status of giant sengis (genus <i>Rhynchocyon</i>) in Mozambique. JE Afr Nat Hist.</span></li> <li tag="ref"><a name="pone.0072506-Corbet1" /><span class="label" tagx="label" title="label">10</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Corbet</span><span class="given-names" tagx="given-names" title="given-names">GB</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hanks</span><span class="given-names" tagx="given-names" title="given-names">J</span></span> (<span class="year" tagx="year" title="year">1968</span>) <span class="mixed-article-title" title="mixed-article-title">A revision of the elephant-shrews, Family Macroscelididae</span>. <span class="source" tagx="source" title="source">Bull Br Mus Nat Hist Zool</span><span class="volume" tagx="volume" title="volume">16</span>: <span class="fpage" tagx="fpage" title="fpage">47</span>–<span class="lpage" tagx="lpage" title="lpage">111</span>.</span></li> <li tag="ref"><a name="pone.0072506-Kingdon1" /><span class="label" tagx="label" title="label">11</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation">Kingdon J (1974) East African mammals; An atlas of evolution in Africa. New York, USA: Academic Press.</span></li> <li tag="ref"><a name="pone.0072506-Rovero2" /><span class="label" tagx="label" title="label">12</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation">Rovero F, Collett C, Ricci S, Martin E, Spitale D (in press) Distribution, occupancy and habitat associations of the grey-faced sengi (<i>Rhynchocyon udzungwensis</i>) as revealed by camera traps. J Mammal.</span></li> <li tag="ref"><a name="pone.0072506-Guisan1" /><span class="label" tagx="label" title="label">13</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Guisan</span><span class="given-names" tagx="given-names" title="given-names">A</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zimmermann</span><span class="given-names" tagx="given-names" title="given-names">NE</span></span> (<span class="year" tagx="year" title="year">2000</span>) <span class="mixed-article-title" title="mixed-article-title">Predictive habitat distribution models in ecology</span>. <span class="source" tagx="source" title="source">Ecol Model</span><span class="volume" tagx="volume" title="volume">135</span>: <span class="fpage" tagx="fpage" title="fpage">147</span>–<span class="lpage" tagx="lpage" title="lpage">186</span><span class="comment">doi:<a href="http://dx.doi.org/10.1016/S0304-3800(00)00354-9">10.1016/S0304-3800(00)00354-9</a></span></span></li> <li tag="ref"><a name="pone.0072506-Parmesan1" /><span class="label" tagx="label" title="label">14</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Parmesan</span><span class="given-names" tagx="given-names" title="given-names">C</span></span> (<span class="year" tagx="year" title="year">2006</span>) <span class="mixed-article-title" title="mixed-article-title">Ecological and Evolutionary Responses to Recent Climate Change</span>. <span class="source" tagx="source" title="source">Annu Rev Ecol Evol Syst</span><span class="volume" tagx="volume" title="volume">37</span>: <span class="fpage" tagx="fpage" title="fpage">637</span>–<span class="lpage" tagx="lpage" title="lpage">669</span>.</span></li> <li tag="ref"><a name="pone.0072506-Alves1" /><span class="label" tagx="label" title="label">15</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Alves</span><span class="given-names" tagx="given-names" title="given-names">PC</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Melo-Ferreira</span><span class="given-names" tagx="given-names" title="given-names">J</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Freitas</span><span class="given-names" tagx="given-names" title="given-names">H</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Boursot</span><span class="given-names" tagx="given-names" title="given-names">P</span></span> (<span class="year" tagx="year" title="year">2008</span>) <span class="mixed-article-title" title="mixed-article-title">The Ubiquitous Mountain Hare Mitochondria: Multiple Introgressive Hybridization in Hares, Genus <i>Lepus</i></span>. <span class="source" tagx="source" title="source">Phil Trans R Soc B</span><span class="volume" tagx="volume" title="volume">363</span>: <span class="fpage" tagx="fpage" title="fpage">2831</span>–<span class="lpage" tagx="lpage" title="lpage">2839</span><span class="comment">doi:<a href="http://dx.doi.org/10.1098/rstb.2008.0053">10.1098/rstb.2008.0053</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18508749">18508749</a></span></span></li> <li tag="ref"><a name="pone.0072506-Mallet2" /><span class="label" tagx="label" title="label">16</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Mallet</span><span class="given-names" tagx="given-names" title="given-names">J</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Meyer</span><span class="given-names" tagx="given-names" title="given-names">A</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Nosil</span><span class="given-names" tagx="given-names" title="given-names">P</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Feder</span><span class="given-names" tagx="given-names" title="given-names">JL</span></span> (<span class="year" tagx="year" title="year">2009</span>) <span class="mixed-article-title" title="mixed-article-title">Space, sympatry and speciation</span>. <span class="source" tagx="source" title="source">J Evol Biol</span><span class="volume" tagx="volume" title="volume">22</span>: <span class="fpage" tagx="fpage" title="fpage">2332</span>–<span class="lpage" tagx="lpage" title="lpage">2341</span><span class="comment">doi:<a href="http://dx.doi.org/10.1111/j.1420-9101.2009.01816.x">10.1111/j.1420-9101.2009.01816.x</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19732264">19732264</a></span></span></li> <li tag="ref"><a name="pone.0072506-Garroway1" /><span class="label" tagx="label" title="label">17</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Garroway</span><span class="given-names" tagx="given-names" title="given-names">CJ</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bowman</span><span class="given-names" tagx="given-names" title="given-names">J</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Cascaden</span><span class="given-names" tagx="given-names" title="given-names">TJ</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Holloway</span><span class="given-names" tagx="given-names" title="given-names">GL</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Mahan</span><span class="given-names" tagx="given-names" title="given-names">CG</span></span>, <span class="etal" title="etal"><i>et al.</i></span> (<span class="year" tagx="year" title="year">2010</span>) <span class="mixed-article-title" title="mixed-article-title">Climate change induced hybridization in flying squirrels</span>. <span class="source" tagx="source" title="source">Glob Change Biol</span><span class="volume" tagx="volume" title="volume">16</span>: <span class="fpage" tagx="fpage" title="fpage">113</span>–<span class="lpage" tagx="lpage" title="lpage">121</span><span class="comment">doi:<a href="http://dx.doi.org/10.1111/j.1365-2486.2009.01948.x">10.1111/j.1365-2486.2009.01948.x</a></span></span></li> <li tag="ref"><a name="pone.0072506-Hewitt1" /><span class="label" tagx="label" title="label">18</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hewitt</span><span class="given-names" tagx="given-names" title="given-names">GM</span></span> (<span class="year" tagx="year" title="year">1996</span>) <span class="mixed-article-title" title="mixed-article-title">Some genetic consequences of ice ages, and their role in divergence and speciation</span>. <span class="source" tagx="source" title="source">Biol J Linn Soc</span><span class="volume" tagx="volume" title="volume">58</span>: <span class="fpage" tagx="fpage" title="fpage">247</span>–<span class="lpage" tagx="lpage" title="lpage">276</span><span class="comment">doi:<a href="http://dx.doi.org/10.1006/bijl.1996.0035">10.1006/bijl.1996.0035</a></span></span></li> <li tag="ref"><a name="pone.0072506-Guralnick1" /><span class="label" tagx="label" title="label">19</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Guralnick</span><span class="given-names" tagx="given-names" title="given-names">R</span></span> (<span class="year" tagx="year" title="year">2007</span>) <span class="mixed-article-title" title="mixed-article-title">Differential effects of past climate warming on mountain and flatland species distributions: a multispecies North American mammal assessment</span>. <span class="source" tagx="source" title="source">Global Ecol Biogeogr</span><span class="volume" tagx="volume" title="volume">16</span>: <span class="fpage" tagx="fpage" title="fpage">14</span>–<span class="lpage" tagx="lpage" title="lpage">23</span><span class="comment">doi:<a href="http://dx.doi.org/10.1111/j.1466-8238.2006.00260.x">10.1111/j.1466-8238.2006.00260.x</a></span></span></li> <li tag="ref"><a name="pone.0072506-Nyari1" /><span class="label" tagx="label" title="label">20</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Nyari</span><span class="given-names" tagx="given-names" title="given-names">AS</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Peterson</span><span class="given-names" tagx="given-names" title="given-names">AT</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Rathbun</span><span class="given-names" tagx="given-names" title="given-names">GB</span></span> (<span class="year" tagx="year" title="year">2010</span>) <span class="mixed-article-title" title="mixed-article-title">Late Pleistocene potential distribution of the North African sengi or elephant-shrew <i>Elephantulus rozeti</i> (Mammalia: Macroscelidea)</span>. <span class="source" tagx="source" title="source">Afr Zool</span><span class="volume" tagx="volume" title="volume">45</span>: <span class="fpage" tagx="fpage" title="fpage">330</span>–<span class="lpage" tagx="lpage" title="lpage">339</span><span class="comment">doi:<a href="http://dx.doi.org/10.3377/004.045.0212">10.3377/004.045.0212</a></span></span></li> <li tag="ref"><a name="pone.0072506-Sikes1" /><span class="label" tagx="label" title="label">21</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Sikes</span><span class="given-names" tagx="given-names" title="given-names">RS</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gannon</span><span class="given-names" tagx="given-names" title="given-names">WL</span></span> (<span class="year" tagx="year" title="year">2011</span>) <span class="mixed-article-title" title="mixed-article-title">Guidelines of the American Society of Mammalogists for the use of wild mammals in research</span>. <span class="source" tagx="source" title="source">J Mammal</span><span class="volume" tagx="volume" title="volume">92</span>: <span class="fpage" tagx="fpage" title="fpage">235</span>–<span class="lpage" tagx="lpage" title="lpage">253</span><span class="comment">doi:<a href="http://dx.doi.org/10.1644/10-MAMM-F-355.1">10.1644/10-MAMM-F-355.1</a></span></span></li> <li tag="ref"><a name="pone.0072506-AVMA1" /><span class="label" tagx="label" title="label">22</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation">AVMA (2000) AVMA guidelines on euthanasia. American Veterinary Medical Association. Available: <a href="http://www.avma.org/issues/animal_welfare/euthanasia.pdf">http://www.avma.org/issues/animal_welfare/euthanasia.pdf</a>.</span></li> <li tag="ref"><a name="pone.0072506-Olson1" /><span class="label" tagx="label" title="label">23</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Olson</span><span class="given-names" tagx="given-names" title="given-names">LE</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Goodman</span><span class="given-names" tagx="given-names" title="given-names">SM</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Yoder</span><span class="given-names" tagx="given-names" title="given-names">AD</span></span> (<span class="year" tagx="year" title="year">2004</span>) <span class="mixed-article-title" title="mixed-article-title">Illumination of cryptic species boundaries in long-tailed shrew tenrecs (Mammalia: Tenrecidae; Microgale), with new insights into geographic variation and distributional constraints</span>. <span class="source" tagx="source" title="source">Biol J Linn Soc</span><span class="volume" tagx="volume" title="volume">83</span>: <span class="fpage" tagx="fpage" title="fpage">1</span>–<span class="lpage" tagx="lpage" title="lpage">22</span><span class="comment">doi:<a href="http://dx.doi.org/10.1111/j.1095-8312.2004.00366.x">10.1111/j.1095-8312.2004.00366.x</a></span></span></li> <li tag="ref"><a name="pone.0072506-Douzery1" /><span class="label" tagx="label" title="label">24</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Douzery</span><span class="given-names" tagx="given-names" title="given-names">E</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Randi</span><span class="given-names" tagx="given-names" title="given-names">E</span></span> (<span class="year" tagx="year" title="year">1997</span>) <span class="mixed-article-title" title="mixed-article-title">The mitochondrial control region of Cervidae: evolutionary patterns and phylogenetic content</span>. <span class="source" tagx="source" title="source">Mol Biol Evol</span><span class="volume" tagx="volume" title="volume">14</span>: <span class="fpage" tagx="fpage" title="fpage">1154</span>–<span class="lpage" tagx="lpage" title="lpage">1166</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/9364773">9364773</a></span></span></li> <li tag="ref"><a name="pone.0072506-Huchon1" /><span class="label" tagx="label" title="label">25</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Huchon</span><span class="given-names" tagx="given-names" title="given-names">D</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Delsuc</span><span class="given-names" tagx="given-names" title="given-names">F</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Catzeflis</span><span class="given-names" tagx="given-names" title="given-names">FM</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Douzery</span><span class="given-names" tagx="given-names" title="given-names">EJP</span></span> (<span class="year" tagx="year" title="year">1999</span>) <span class="mixed-article-title" title="mixed-article-title">Armadillos exhibit less genetic polymorphism in North America than in South America: nuclear and mitochondrial data confirm a founder effect in <i>Dasypus novemcinctus</i> (Xenarthra)</span>. <span class="source" tagx="source" title="source">Mol Ecol</span><span class="volume" tagx="volume" title="volume">8</span>: <span class="fpage" tagx="fpage" title="fpage">1743</span>–<span class="lpage" tagx="lpage" title="lpage">1748</span><span class="comment">doi:<a href="http://dx.doi.org/10.1046/j.1365-294x.1999.00768.x">10.1046/j.1365-294x.1999.00768.x</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/10583836">10583836</a></span></span></li> <li tag="ref"><a name="pone.0072506-Kocher1" /><span class="label" tagx="label" title="label">26</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kocher</span><span class="given-names" tagx="given-names" title="given-names">TD</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Thomas</span><span class="given-names" tagx="given-names" title="given-names">WK</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Meyer</span><span class="given-names" tagx="given-names" title="given-names">A</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Edwards</span><span class="given-names" tagx="given-names" title="given-names">SV</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Pääbo</span><span class="given-names" tagx="given-names" title="given-names">S</span></span>, <span class="etal" title="etal"><i>et al.</i></span> (<span class="year" tagx="year" title="year">1989</span>) <span class="mixed-article-title" title="mixed-article-title">Dynamics of mitochondrial DNA evolution in animals: amplification and sequencing with conserved primers</span>. <span class="source" tagx="source" title="source">PNAS</span><span class="volume" tagx="volume" title="volume">86</span>: <span class="fpage" tagx="fpage" title="fpage">6196</span>–<span class="lpage" tagx="lpage" title="lpage">6200</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/2762322">2762322</a></span></span></li> <li tag="ref"><a name="pone.0072506-Douady1" /><span class="label" tagx="label" title="label">27</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Douady</span><span class="given-names" tagx="given-names" title="given-names">CJ</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Catzeflis</span><span class="given-names" tagx="given-names" title="given-names">F</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Raman</span><span class="given-names" tagx="given-names" title="given-names">J</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Springer</span><span class="given-names" tagx="given-names" title="given-names">MS</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Stanhope</span><span class="given-names" tagx="given-names" title="given-names">MJ</span></span> (<span class="year" tagx="year" title="year">2003</span>) <span class="mixed-article-title" title="mixed-article-title">The Sahara as a vicariant agent, and the role of Miocene climatic events, in the diversification of the mammalian order Macroscelidea (elephant-shrews)</span>. <span class="source" tagx="source" title="source">PNAS</span><span class="volume" tagx="volume" title="volume">100</span>: <span class="fpage" tagx="fpage" title="fpage">8325</span>–<span class="lpage" tagx="lpage" title="lpage">8330</span><span class="comment">doi:<a href="http://dx.doi.org/10.1073/pnas.0832467100">10.1073/pnas.0832467100</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12821774">12821774</a></span></span></li> <li tag="ref"><a name="pone.0072506-Edgar1" /><span class="label" tagx="label" title="label">28</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Edgar</span><span class="given-names" tagx="given-names" title="given-names">RC</span></span> (<span class="year" tagx="year" title="year">2004</span>) <span class="mixed-article-title" title="mixed-article-title">MUSCLE: Multiple Sequence Alignment with High Accuracy and High Throughput</span>. <span class="source" tagx="source" title="source">Nucl Acids Res</span><span class="volume" tagx="volume" title="volume">32</span>: <span class="fpage" tagx="fpage" title="fpage">1792</span>–<span class="lpage" tagx="lpage" title="lpage">1797</span><span class="comment">doi:<a href="http://dx.doi.org/10.1093/nar/gkh340">10.1093/nar/gkh340</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15034147">15034147</a></span></span></li> <li tag="ref"><a name="pone.0072506-Stephens1" /><span class="label" tagx="label" title="label">29</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Stephens</span><span class="given-names" tagx="given-names" title="given-names">M</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Smith</span><span class="given-names" tagx="given-names" title="given-names">NJ</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Donnelly</span><span class="given-names" tagx="given-names" title="given-names">P</span></span> (<span class="year" tagx="year" title="year">2001</span>) <span class="mixed-article-title" title="mixed-article-title">A new statistical method for haplotype reconstruction from population data</span>. <span class="source" tagx="source" title="source">Am J Hum Genet</span><span class="volume" tagx="volume" title="volume">68</span>: <span class="fpage" tagx="fpage" title="fpage">978</span>–<span class="lpage" tagx="lpage" title="lpage">989</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11254454">11254454</a></span></span></li> <li tag="ref"><a name="pone.0072506-Stephens2" /><span class="label" tagx="label" title="label">30</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Stephens</span><span class="given-names" tagx="given-names" title="given-names">M</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Scheet</span><span class="given-names" tagx="given-names" title="given-names">P</span></span> (<span class="year" tagx="year" title="year">2005</span>) <span class="mixed-article-title" title="mixed-article-title">Accounting for decay of linkage disequilibrium in haplotype inference and missing-data imputation</span>. <span class="source" tagx="source" title="source">Am J Hum Genet</span><span class="volume" tagx="volume" title="volume">76</span>: <span class="fpage" tagx="fpage" title="fpage">449</span>–<span class="lpage" tagx="lpage" title="lpage">462</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15700229">15700229</a></span></span></li> <li tag="ref"><a name="pone.0072506-Posada1" /><span class="label" tagx="label" title="label">31</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Posada</span><span class="given-names" tagx="given-names" title="given-names">D</span></span> (<span class="year" tagx="year" title="year">2008</span>) <span class="mixed-article-title" title="mixed-article-title">jModelTest: phylogenetic model averaging</span>. <span class="source" tagx="source" title="source">Mol Biol Evol</span><span class="volume" tagx="volume" title="volume">25</span>: <span class="fpage" tagx="fpage" title="fpage">1253</span>–<span class="lpage" tagx="lpage" title="lpage">1256</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18397919">18397919</a></span></span></li> <li tag="ref"><a name="pone.0072506-Drummond1" /><span class="label" tagx="label" title="label">32</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Drummond</span><span class="given-names" tagx="given-names" title="given-names">AJ</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Rambaut</span><span class="given-names" tagx="given-names" title="given-names">A</span></span> (<span class="year" tagx="year" title="year">2007</span>) <span class="mixed-article-title" title="mixed-article-title">BEAST: Bayesian evolutionary analysis by sampling trees</span>. <span class="source" tagx="source" title="source">BMC Evol Biol</span><span class="volume" tagx="volume" title="volume">7</span>: <span class="fpage" tagx="fpage" title="fpage">1</span>–<span class="lpage" tagx="lpage" title="lpage">8</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17214884">17214884</a></span></span></li> <li tag="ref"><a name="pone.0072506-Ronquist1" /><span class="label" tagx="label" title="label">33</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ronquist</span><span class="given-names" tagx="given-names" title="given-names">F</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Huelsenbeck</span><span class="given-names" tagx="given-names" title="given-names">JP</span></span> (<span class="year" tagx="year" title="year">2003</span>) <span class="mixed-article-title" title="mixed-article-title">MrBayes 3: Bayesian phylogenetic inference under mixed models</span>. <span class="source" tagx="source" title="source">Bioinformatics</span><span class="volume" tagx="volume" title="volume">19</span>: <span class="fpage" tagx="fpage" title="fpage">1572</span>–<span class="lpage" tagx="lpage" title="lpage">1574</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12912839">12912839</a></span></span></li> <li tag="ref"><a name="pone.0072506-Liu1" /><span class="label" tagx="label" title="label">34</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Liu</span><span class="given-names" tagx="given-names" title="given-names">L</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Pearl</span><span class="given-names" tagx="given-names" title="given-names">DK</span></span> (<span class="year" tagx="year" title="year">2007</span>) <span class="mixed-article-title" title="mixed-article-title">Species trees from gene trees: reconstructing Bayesian posterior distributions of a species phylogeny using estimated gene tree distributions</span>. <span class="source" tagx="source" title="source">Syst Biol</span><span class="volume" tagx="volume" title="volume">56</span>: <span class="fpage" tagx="fpage" title="fpage">504</span>–<span class="lpage" tagx="lpage" title="lpage">514</span><span class="comment">doi:<a href="http://dx.doi.org/10.1080/10635150701429982">10.1080/10635150701429982</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17562474">17562474</a></span></span></li> <li tag="ref"><a name="pone.0072506-Liu2" /><span class="label" tagx="label" title="label">35</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Liu</span><span class="given-names" tagx="given-names" title="given-names">L</span></span> (<span class="year" tagx="year" title="year">2008</span>) <span class="mixed-article-title" title="mixed-article-title">BEST: Bayesian estimation of species trees under the coalescent model</span>. <span class="source" tagx="source" title="source">Bioinformatics</span><span class="volume" tagx="volume" title="volume">24</span>: <span class="fpage" tagx="fpage" title="fpage">2542</span>–<span class="lpage" tagx="lpage" title="lpage">2543</span><span class="comment">doi:<a href="http://dx.doi.org/10.1093/bioinformatics/btn484">10.1093/bioinformatics/btn484</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18799483">18799483</a></span></span></li> <li tag="ref"><a name="pone.0072506-Heled1" /><span class="label" tagx="label" title="label">36</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Heled</span><span class="given-names" tagx="given-names" title="given-names">J</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Drummond</span><span class="given-names" tagx="given-names" title="given-names">AJ</span></span> (<span class="year" tagx="year" title="year">2010</span>) <span class="mixed-article-title" title="mixed-article-title">Bayesian inference of species trees from multilocus data</span>. <span class="source" tagx="source" title="source">Mol Biol Evol</span><span class="volume" tagx="volume" title="volume">27</span>: <span class="fpage" tagx="fpage" title="fpage">570</span>–<span class="lpage" tagx="lpage" title="lpage">580</span><span class="comment">doi:<a href="http://dx.doi.org/10.1093/molbev/msp274">10.1093/molbev/msp274</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19906793">19906793</a></span></span></li> <li tag="ref"><a name="pone.0072506-Rannala1" /><span class="label" tagx="label" title="label">37</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Rannala</span><span class="given-names" tagx="given-names" title="given-names">B</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Yang</span><span class="given-names" tagx="given-names" title="given-names">Z</span></span> (<span class="year" tagx="year" title="year">2003</span>) <span class="mixed-article-title" title="mixed-article-title">Bayes estimation of species divergence times and ancestral population sizes using DNA sequences from multiple loci</span>. <span class="source" tagx="source" title="source">Genetics</span><span class="volume" tagx="volume" title="volume">164</span>: <span class="fpage" tagx="fpage" title="fpage">1645</span>–<span class="lpage" tagx="lpage" title="lpage">1656</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12930768">12930768</a></span></span></li> <li tag="ref"><a name="pone.0072506-Yang1" /><span class="label" tagx="label" title="label">38</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Yang</span><span class="given-names" tagx="given-names" title="given-names">Z</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Rannala</span><span class="given-names" tagx="given-names" title="given-names">B</span></span> (<span class="year" tagx="year" title="year">2010</span>) <span class="mixed-article-title" title="mixed-article-title">Bayesian species delimitation using multilocus sequence data</span>. <span class="source" tagx="source" title="source">PNAS</span><span class="volume" tagx="volume" title="volume">107</span>: <span class="fpage" tagx="fpage" title="fpage">9264</span>–<span class="lpage" tagx="lpage" title="lpage">9269</span><span class="comment">doi:<a href="http://dx.doi.org/10.1073/pnas.0913022107">10.1073/pnas.0913022107</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20439743">20439743</a></span></span></li> <li tag="ref"><a name="pone.0072506-An1" /><span class="label" tagx="label" title="label">39</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ané</span><span class="given-names" tagx="given-names" title="given-names">C</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Larget</span><span class="given-names" tagx="given-names" title="given-names">B</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Baum</span><span class="given-names" tagx="given-names" title="given-names">DA</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Smith</span><span class="given-names" tagx="given-names" title="given-names">SD</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Rokas</span><span class="given-names" tagx="given-names" title="given-names">A</span></span> (<span class="year" tagx="year" title="year">2007</span>) <span class="mixed-article-title" title="mixed-article-title">Bayesian estimation of concordance among gene trees</span>. <span class="source" tagx="source" title="source">Mol Biol Evol</span><span class="volume" tagx="volume" title="volume">24</span>: <span class="fpage" tagx="fpage" title="fpage">412</span>–<span class="lpage" tagx="lpage" title="lpage">426</span><span class="comment">doi:<a href="http://dx.doi.org/10.1093/molbev/msl170">10.1093/molbev/msl170</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17095535">17095535</a></span></span></li> <li tag="ref"><a name="pone.0072506-An2" /><span class="label" tagx="label" title="label">40</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ané</span><span class="given-names" tagx="given-names" title="given-names">C</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Dewey</span><span class="given-names" tagx="given-names" title="given-names">CN</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kotha</span><span class="given-names" tagx="given-names" title="given-names">SK</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Larget</span><span class="given-names" tagx="given-names" title="given-names">BR</span></span> (<span class="year" tagx="year" title="year">2010</span>) <span class="mixed-article-title" title="mixed-article-title">Gene tree reconciliation: new developments in Bayesian concordance analysis with BUCKy</span>. <span class="source" tagx="source" title="source">Bioinformatics</span><span class="volume" tagx="volume" title="volume">26</span>: <span class="fpage" tagx="fpage" title="fpage">2910</span>–<span class="lpage" tagx="lpage" title="lpage">2911</span><span class="comment">doi:<a href="http://dx.doi.org/10.1093/bioinformatics/btq539">10.1093/bioinformatics/btq539</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20861028">20861028</a></span></span></li> <li tag="ref"><a name="pone.0072506-Larget1" /><span class="label" tagx="label" title="label">41</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Larget</span><span class="given-names" tagx="given-names" title="given-names">BR</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kotha</span><span class="given-names" tagx="given-names" title="given-names">SK</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Dewey</span><span class="given-names" tagx="given-names" title="given-names">CN</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ané</span><span class="given-names" tagx="given-names" title="given-names">C</span></span> (<span class="year" tagx="year" title="year">2010</span>) <span class="mixed-article-title" title="mixed-article-title">BUCKy: Gene tree/species tree reconciliation with Bayesian concordance analysis</span>. <span class="source" tagx="source" title="source">Bioinformatics</span><span class="volume" tagx="volume" title="volume">26</span>: <span class="fpage" tagx="fpage" title="fpage">2910</span>–<span class="lpage" tagx="lpage" title="lpage">2911</span><span class="comment">doi:<a href="http://dx.doi.org/10.1093/bioinformatics/btq539">10.1093/bioinformatics/btq539</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20861028">20861028</a></span></span></li> <li tag="ref"><a name="pone.0072506-Chung1" /><span class="label" tagx="label" title="label">42</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Chung</span><span class="given-names" tagx="given-names" title="given-names">Y</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ané</span><span class="given-names" tagx="given-names" title="given-names">C</span></span> (<span class="year" tagx="year" title="year">2011</span>) <span class="mixed-article-title" title="mixed-article-title">Comparing two Bayesian methods for gene tree/species tree reconstruction: simulations with incomplete lineage sorting and horizontal gene transfer</span>. <span class="source" tagx="source" title="source">Syst Biol</span><span class="volume" tagx="volume" title="volume">60</span>: <span class="fpage" tagx="fpage" title="fpage">261</span>–<span class="lpage" tagx="lpage" title="lpage">275</span><span class="comment">doi:<a href="http://dx.doi.org/10.1093/sysbio/syr003">10.1093/sysbio/syr003</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21368324">21368324</a></span></span></li> <li tag="ref"><a name="pone.0072506-Leach1" /><span class="label" tagx="label" title="label">43</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Leaché</span><span class="given-names" tagx="given-names" title="given-names">AD</span></span> (<span class="year" tagx="year" title="year">2009</span>) <span class="mixed-article-title" title="mixed-article-title">Species tree discordance traces to phylogeographic clade boundaries in North American fence lizards (<i>Sceloporus</i>)</span>. <span class="source" tagx="source" title="source">Syst Biol</span><span class="volume" tagx="volume" title="volume">58</span>: <span class="fpage" tagx="fpage" title="fpage">547</span>–<span class="lpage" tagx="lpage" title="lpage">559</span><span class="comment">doi:<a href="http://dx.doi.org/10.1093/sysbio/syp057">10.1093/sysbio/syp057</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20525608">20525608</a></span></span></li> <li tag="ref"><a name="pone.0072506-Lee1" /><span class="label" tagx="label" title="label">44</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lee</span><span class="given-names" tagx="given-names" title="given-names">JY</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Joseph</span><span class="given-names" tagx="given-names" title="given-names">L</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Edwards</span><span class="given-names" tagx="given-names" title="given-names">SV</span></span> (<span class="year" tagx="year" title="year">2012</span>) <span class="mixed-article-title" title="mixed-article-title">A species tree for the Australo-Papuan Fairy-wrens and Allies (Aves: Maluridae)</span>. <span class="source" tagx="source" title="source">Syst Biol</span><span class="volume" tagx="volume" title="volume">61</span>: <span class="fpage" tagx="fpage" title="fpage">253</span>–<span class="lpage" tagx="lpage" title="lpage">271</span><span class="comment">doi:<a href="http://dx.doi.org/10.1093/sysbio/syr101">10.1093/sysbio/syr101</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21978990">21978990</a></span></span></li> <li tag="ref"><a name="pone.0072506-Shapiro1" /><span class="label" tagx="label" title="label">45</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Shapiro</span><span class="given-names" tagx="given-names" title="given-names">B</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Rambaut</span><span class="given-names" tagx="given-names" title="given-names">A</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Drummond</span><span class="given-names" tagx="given-names" title="given-names">AJ</span></span> (<span class="year" tagx="year" title="year">2006</span>) <span class="mixed-article-title" title="mixed-article-title">Choosing appropriate substitution models for the phylogenetic analysis of protein-coding sequences</span>. <span class="source" tagx="source" title="source">Mol Biol Evol</span><span class="volume" tagx="volume" title="volume">23</span>: <span class="fpage" tagx="fpage" title="fpage">7</span>–<span class="lpage" tagx="lpage" title="lpage">9</span><span class="comment">doi:<a href="http://dx.doi.org/10.1093/molbev/msj021">10.1093/molbev/msj021</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16177232">16177232</a></span></span></li> <li tag="ref"><a name="pone.0072506-Degnan1" /><span class="label" tagx="label" title="label">46</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Degnan</span><span class="given-names" tagx="given-names" title="given-names">JH</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">DeGiorgio</span><span class="given-names" tagx="given-names" title="given-names">M</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bryant</span><span class="given-names" tagx="given-names" title="given-names">D</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Rosenberg</span><span class="given-names" tagx="given-names" title="given-names">NA</span></span> (<span class="year" tagx="year" title="year">2009</span>) <span class="mixed-article-title" title="mixed-article-title">Properties of consensus methods for inferring species trees from gene trees</span>. <span class="source" tagx="source" title="source">Syst Biol</span><span class="volume" tagx="volume" title="volume">58</span>: <span class="fpage" tagx="fpage" title="fpage">35</span>–<span class="lpage" tagx="lpage" title="lpage">54</span><span class="comment">doi:<a href="http://dx.doi.org/10.1093/sysbio/syp008">10.1093/sysbio/syp008</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20525567">20525567</a></span></span></li> <li tag="ref"><a name="pone.0072506-Andanje1" /><span class="label" tagx="label" title="label">47</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Andanje</span><span class="given-names" tagx="given-names" title="given-names">S</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Agwanda</span><span class="given-names" tagx="given-names" title="given-names">BR</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ngaruiya</span><span class="given-names" tagx="given-names" title="given-names">GW</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Amin</span><span class="given-names" tagx="given-names" title="given-names">R</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Rathbun</span><span class="given-names" tagx="given-names" title="given-names">GB</span></span> (<span class="year" tagx="year" title="year">2010</span>) <span class="mixed-article-title" title="mixed-article-title">Sengi (elephant-shrew) observations from northern coastal Kenya</span>. <span class="source" tagx="source" title="source">JE Afr Nat Hist</span><span class="volume" tagx="volume" title="volume">99</span>: <span class="fpage" tagx="fpage" title="fpage">1</span>–<span class="lpage" tagx="lpage" title="lpage">8</span><span class="comment">doi:<a href="http://dx.doi.org/10.2982/028.099.0101">10.2982/028.099.0101</a></span></span></li> <li tag="ref"><a name="pone.0072506-Hijmans1" /><span class="label" tagx="label" title="label">48</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hijmans</span><span class="given-names" tagx="given-names" title="given-names">RJ</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Cameron</span><span class="given-names" tagx="given-names" title="given-names">SE</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Parra</span><span class="given-names" tagx="given-names" title="given-names">JL</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Jones</span><span class="given-names" tagx="given-names" title="given-names">PG</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Jarvis</span><span class="given-names" tagx="given-names" title="given-names">A</span></span> (<span class="year" tagx="year" title="year">2005</span>) <span class="mixed-article-title" title="mixed-article-title">Very high resolution interpolated climate surfaces for global land areas</span>. <span class="source" tagx="source" title="source">Int J Climatol</span><span class="volume" tagx="volume" title="volume">25</span>: <span class="fpage" tagx="fpage" title="fpage">1965</span>–<span class="lpage" tagx="lpage" title="lpage">1978</span><span class="comment">doi:<a href="http://dx.doi.org/10.1002/joc.1276">10.1002/joc.1276</a></span></span></li> <li tag="ref"><a name="pone.0072506-Collins1" /><span class="label" tagx="label" title="label">49</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Collins</span><span class="given-names" tagx="given-names" title="given-names">WD</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bitz</span><span class="given-names" tagx="given-names" title="given-names">CM</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Blackmon</span><span class="given-names" tagx="given-names" title="given-names">ML</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bonan</span><span class="given-names" tagx="given-names" title="given-names">GB</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bretherton</span><span class="given-names" tagx="given-names" title="given-names">CS</span></span>, <span class="etal" title="etal"><i>et al.</i></span> (<span class="year" tagx="year" title="year">2006</span>) <span class="mixed-article-title" title="mixed-article-title">The community climate system model version 3 (CCSM3)</span>. <span class="source" tagx="source" title="source">J Climate</span><span class="volume" tagx="volume" title="volume">19</span>: <span class="fpage" tagx="fpage" title="fpage">2122</span>–<span class="lpage" tagx="lpage" title="lpage">2143</span>.</span></li> <li tag="ref"><a name="pone.0072506-OttoBliesner1" /><span class="label" tagx="label" title="label">50</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation">Otto-Bliesner BL, Marshall SJ, Overpeck JT, Miller GH, Hu A, <span class="etal" title="etal"><i>et al.</i></span>.. (2006) Simulating Warm-Arctic Climate and Ice Sheet Sensitivity for the Last Interglaciation. Science: 1751–1753.</span></li> <li tag="ref"><a name="pone.0072506-Warren1" /><span class="label" tagx="label" title="label">51</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Warren</span><span class="given-names" tagx="given-names" title="given-names">DL</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Glor</span><span class="given-names" tagx="given-names" title="given-names">RE</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Turelli</span><span class="given-names" tagx="given-names" title="given-names">M</span></span> (<span class="year" tagx="year" title="year">2010</span>) <span class="mixed-article-title" title="mixed-article-title">ENMTools: a toolbox for comparative studies of environmental niche models</span>. <span class="source" tagx="source" title="source">Ecography</span><span class="volume" tagx="volume" title="volume">33</span>: <span class="fpage" tagx="fpage" title="fpage">607</span>–<span class="lpage" tagx="lpage" title="lpage">611</span><span class="comment">doi:<a href="http://dx.doi.org/10.1111/j.1600-0587.2009.06142.x">10.1111/j.1600-0587.2009.06142.x</a></span></span></li> <li tag="ref"><a name="pone.0072506-Phillips1" /><span class="label" tagx="label" title="label">52</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation">Phillips SJ, Dudík M, Schapire RE (2004) A maximum entropy approach to species distribution modeling. Proceedings of the twenty-first international conference on Machine learning. ACM New York, NY, USA. 472–486.</span></li> <li tag="ref"><a name="pone.0072506-Hernandez1" /><span class="label" tagx="label" title="label">53</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hernandez</span><span class="given-names" tagx="given-names" title="given-names">PA</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Graham</span><span class="given-names" tagx="given-names" title="given-names">CH</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Master</span><span class="given-names" tagx="given-names" title="given-names">LL</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Albert</span><span class="given-names" tagx="given-names" title="given-names">DL</span></span> (<span class="year" tagx="year" title="year">2006</span>) <span class="mixed-article-title" title="mixed-article-title">The effect of sample size and species characteristics on performance of different species distribution modeling methods</span>. <span class="source" tagx="source" title="source">Ecography</span><span class="volume" tagx="volume" title="volume">29</span>: <span class="fpage" tagx="fpage" title="fpage">773</span>–<span class="lpage" tagx="lpage" title="lpage">785</span><span class="comment">doi:<a href="http://dx.doi.org/10.1111/j.0906-7590.2006.04700.x">10.1111/j.0906-7590.2006.04700.x</a></span></span></li> <li tag="ref"><a name="pone.0072506-Hernandez2" /><span class="label" tagx="label" title="label">54</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hernandez</span><span class="given-names" tagx="given-names" title="given-names">PA</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Franke</span><span class="given-names" tagx="given-names" title="given-names">I</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Herzog</span><span class="given-names" tagx="given-names" title="given-names">SK</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Pacheco</span><span class="given-names" tagx="given-names" title="given-names">V</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Paniagua</span><span class="given-names" tagx="given-names" title="given-names">L</span></span>, <span class="etal" title="etal"><i>et al.</i></span> (<span class="year" tagx="year" title="year">2008</span>) <span class="mixed-article-title" title="mixed-article-title">Predicting species distributions in poorly-studied landscapes</span>. <span class="source" tagx="source" title="source">Biodivers Conserv</span><span class="volume" tagx="volume" title="volume">17</span>: <span class="fpage" tagx="fpage" title="fpage">1353</span>–<span class="lpage" tagx="lpage" title="lpage">1366</span><span class="comment">doi:<a href="http://dx.doi.org/10.1007/s10531-007-9314-z">10.1007/s10531-007-9314-z</a></span></span></li> <li tag="ref"><a name="pone.0072506-Schoener1" /><span class="label" tagx="label" title="label">55</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Schoener</span><span class="given-names" tagx="given-names" title="given-names">TW</span></span> (<span class="year" tagx="year" title="year">1968</span>) <span class="mixed-article-title" title="mixed-article-title">The Anolis lizards of Bimini: resource partitioning in a complex fauna</span>. <span class="source" tagx="source" title="source">Ecology</span><span class="volume" tagx="volume" title="volume">49</span>: <span class="fpage" tagx="fpage" title="fpage">704</span>–<span class="lpage" tagx="lpage" title="lpage">726</span>.</span></li> <li tag="ref"><a name="pone.0072506-Warren2" /><span class="label" tagx="label" title="label">56</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Warren</span><span class="given-names" tagx="given-names" title="given-names">DL</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Glor</span><span class="given-names" tagx="given-names" title="given-names">RE</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Turelli</span><span class="given-names" tagx="given-names" title="given-names">M</span></span> (<span class="year" tagx="year" title="year">2008</span>) <span class="mixed-article-title" title="mixed-article-title">Environmental niche equivalency versus conservatism: quantitative approaches to niche evolution</span>. <span class="source" tagx="source" title="source">Evolution</span><span class="volume" tagx="volume" title="volume">62</span>: <span class="fpage" tagx="fpage" title="fpage">2868</span>–<span class="lpage" tagx="lpage" title="lpage">2883</span><span class="comment">doi:<a href="http://dx.doi.org/10.1111/j.1558-5646.2008.00482.x">10.1111/j.1558-5646.2008.00482.x</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18752605">18752605</a></span></span></li> <li tag="ref"><a name="pone.0072506-Funk1" /><span class="label" tagx="label" title="label">57</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Funk</span><span class="given-names" tagx="given-names" title="given-names">DJ</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Omland</span><span class="given-names" tagx="given-names" title="given-names">KE</span></span> (<span class="year" tagx="year" title="year">2003</span>) <span class="mixed-article-title" title="mixed-article-title">Species-Level Paraphyly and Polyphyly: Frequency, Causes, and Consequences, with Insights from Animal Mitochondrial DNA</span>. <span class="source" tagx="source" title="source">Annu Rev Ecol Evol Syst</span><span class="volume" tagx="volume" title="volume">34</span>: <span class="fpage" tagx="fpage" title="fpage">397</span>–<span class="lpage" tagx="lpage" title="lpage">423</span><span class="comment">doi:<a href="http://dx.doi.org/10.1146/annurev.ecolsys.34.011802.132421">10.1146/annurev.ecolsys.34.011802.132421</a></span></span></li> <li tag="ref"><a name="pone.0072506-Chan1" /><span class="label" tagx="label" title="label">58</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Chan</span><span class="given-names" tagx="given-names" title="given-names">KMA</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Levin</span><span class="given-names" tagx="given-names" title="given-names">SA</span></span> (<span class="year" tagx="year" title="year">2005</span>) <span class="mixed-article-title" title="mixed-article-title">Leaky prezygotic isolation and porous genomes: rapid introgression of maternally inherited DNA</span>. <span class="source" tagx="source" title="source">Evolution</span><span class="volume" tagx="volume" title="volume">59</span>: <span class="fpage" tagx="fpage" title="fpage">720</span>–<span class="lpage" tagx="lpage" title="lpage">729</span><span class="comment">doi:<a href="http://dx.doi.org/10.1554/04-534">10.1554/04-534</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15926684">15926684</a></span></span></li> <li tag="ref"><a name="pone.0072506-Linnen1" /><span class="label" tagx="label" title="label">59</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Linnen</span><span class="given-names" tagx="given-names" title="given-names">CR</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Farrell</span><span class="given-names" tagx="given-names" title="given-names">BD</span></span> (<span class="year" tagx="year" title="year">2008</span>) <span class="mixed-article-title" title="mixed-article-title">Comparison of methods for species-tree inference in the sawfly genus <i>Neodiprion</i> (Hymenoptera: Diprionidae)</span>. <span class="source" tagx="source" title="source">Syst Biol</span><span class="volume" tagx="volume" title="volume">57</span>: <span class="fpage" tagx="fpage" title="fpage">876</span>–<span class="lpage" tagx="lpage" title="lpage">890</span><span class="comment">doi:<a href="http://dx.doi.org/10.1080/10635150802580949">10.1080/10635150802580949</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19085330">19085330</a></span></span></li> <li tag="ref"><a name="pone.0072506-Spinks1" /><span class="label" tagx="label" title="label">60</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Spinks</span><span class="given-names" tagx="given-names" title="given-names">PQ</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Shaffer</span><span class="given-names" tagx="given-names" title="given-names">HB</span></span> (<span class="year" tagx="year" title="year">2009</span>) <span class="mixed-article-title" title="mixed-article-title">Conflicting mitochondrial and nuclear phylogenies for the widely disjunct <i>Emys</i> (Testudines: Emydidae) species complex, and what they tell us about biogeography and hybridization</span>. <span class="source" tagx="source" title="source">Syst Biol</span><span class="volume" tagx="volume" title="volume">58</span>: <span class="fpage" tagx="fpage" title="fpage">1</span>–<span class="lpage" tagx="lpage" title="lpage">20</span><span class="comment">doi:<a href="http://dx.doi.org/10.1093/sysbio/syp005">10.1093/sysbio/syp005</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20525565">20525565</a></span></span></li> <li tag="ref"><a name="pone.0072506-Moore1" /><span class="label" tagx="label" title="label">61</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Moore</span><span class="given-names" tagx="given-names" title="given-names">WS</span></span> (<span class="year" tagx="year" title="year">1995</span>) <span class="mixed-article-title" title="mixed-article-title">Inferring phylogenies from mtDNA variation: mitochondrial-gene trees versus nuclear-gene trees</span>. <span class="source" tagx="source" title="source">Evolution</span><span class="volume" tagx="volume" title="volume">49</span>: <span class="fpage" tagx="fpage" title="fpage">718</span>–<span class="lpage" tagx="lpage" title="lpage">726</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/28565131">28565131</a></span></span></li> <li tag="ref"><a name="pone.0072506-Palumbi1" /><span class="label" tagx="label" title="label">62</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Palumbi</span><span class="given-names" tagx="given-names" title="given-names">SR</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Cipriano</span><span class="given-names" tagx="given-names" title="given-names">F</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hare</span><span class="given-names" tagx="given-names" title="given-names">MP</span></span> (<span class="year" tagx="year" title="year">2001</span>) <span class="mixed-article-title" title="mixed-article-title">Predicting nuclear gene coalescence from mitochondrial data: the three-times rule</span>. <span class="source" tagx="source" title="source">Evolution</span><span class="volume" tagx="volume" title="volume">55</span>: <span class="fpage" tagx="fpage" title="fpage">859</span>–<span class="lpage" tagx="lpage" title="lpage">868</span><span class="comment">doi:[];<a href="http://dx.doi.org/10.1554/0014-3820(2001)0550859:PNGCFM2.0.CO2">10.1554/0014-3820(2001)055[0859:PNGCFM]2.0.CO;2</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11430646">11430646</a></span></span></li> <li tag="ref"><a name="pone.0072506-Hudson1" /><span class="label" tagx="label" title="label">63</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hudson</span><span class="given-names" tagx="given-names" title="given-names">RR</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Coyne</span><span class="given-names" tagx="given-names" title="given-names">JA</span></span> (<span class="year" tagx="year" title="year">2002</span>) <span class="mixed-article-title" title="mixed-article-title">Mathematical consequences of the genealogical species concept</span>. <span class="source" tagx="source" title="source">Evolution</span><span class="volume" tagx="volume" title="volume">56</span>: <span class="fpage" tagx="fpage" title="fpage">1557</span>–<span class="lpage" tagx="lpage" title="lpage">1565</span><span class="comment">doi:[];<a href="http://dx.doi.org/10.1554/0014-3820(2002)0561557:MCOTGS2.0.CO2">10.1554/0014-3820(2002)056[1557:MCOTGS]2.0.CO;2</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12353748">12353748</a></span></span></li> <li tag="ref"><a name="pone.0072506-Smit2" /><span class="label" tagx="label" title="label">64</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Smit</span><span class="given-names" tagx="given-names" title="given-names">HA</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Robinson</span><span class="given-names" tagx="given-names" title="given-names">TJ</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Van Vuuren</span><span class="given-names" tagx="given-names" title="given-names">BJ</span></span> (<span class="year" tagx="year" title="year">2007</span>) <span class="mixed-article-title" title="mixed-article-title">Coalescence methods reveal the impact of vicariance on the spatial genetic structure of <i>Elephantulus edwardii</i> (Afrotheria, Macroscelidea)</span>. <span class="source" tagx="source" title="source">Mol Ecol</span><span class="volume" tagx="volume" title="volume">16</span>: <span class="fpage" tagx="fpage" title="fpage">2680</span>–<span class="lpage" tagx="lpage" title="lpage">2692</span><span class="comment">doi:<a href="http://dx.doi.org/10.1111/j.1365-294X.2007.03334.x">10.1111/j.1365-294X.2007.03334.x</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17594439">17594439</a></span></span></li> <li tag="ref"><a name="pone.0072506-Dinesen1" /><span class="label" tagx="label" title="label">65</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Dinesen</span><span class="given-names" tagx="given-names" title="given-names">L</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lehmberg</span><span class="given-names" tagx="given-names" title="given-names">T</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Svendsen</span><span class="given-names" tagx="given-names" title="given-names">TO</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hansen</span><span class="given-names" tagx="given-names" title="given-names">LA</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Fjeldsaå</span><span class="given-names" tagx="given-names" title="given-names">J</span></span> (<span class="year" tagx="year" title="year">1994</span>) <span class="mixed-article-title" title="mixed-article-title">A new genus and species of perdicine bird (Phasianidae, Perdicini) from Tanzania; a relict form with Indo-Malayan affinities</span>. <span class="source" tagx="source" title="source">Ibis</span><span class="volume" tagx="volume" title="volume">136</span>: <span class="fpage" tagx="fpage" title="fpage">3</span>–<span class="lpage" tagx="lpage" title="lpage">11</span>.</span></li> <li tag="ref"><a name="pone.0072506-Fjelds1" /><span class="label" tagx="label" title="label">66</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Fjeldså</span><span class="given-names" tagx="given-names" title="given-names">J</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kiure</span><span class="given-names" tagx="given-names" title="given-names">J</span></span> (<span class="year" tagx="year" title="year">2003</span>) <span class="mixed-article-title" title="mixed-article-title">A new population of the Udzungwa forest partridge</span>. <span class="source" tagx="source" title="source">Bull Br Ornithol Club</span><span class="volume" tagx="volume" title="volume">123</span>: <span class="fpage" tagx="fpage" title="fpage">52</span>–<span class="lpage" tagx="lpage" title="lpage">56</span>.</span></li> <li tag="ref"><a name="pone.0072506-Bowie1" /><span class="label" tagx="label" title="label">67</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bowie</span><span class="given-names" tagx="given-names" title="given-names">RCK</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Fjeldså</span><span class="given-names" tagx="given-names" title="given-names">J</span></span> (<span class="year" tagx="year" title="year">2005</span>) <span class="mixed-article-title" title="mixed-article-title">Genetic and morphological evidence for two species in the Udzungwa Forest Partridge <i>Xenoperdix udzungwensis</i></span>. <span class="source" tagx="source" title="source">JE Afr Nat Hist</span><span class="volume" tagx="volume" title="volume">94</span>: <span class="fpage" tagx="fpage" title="fpage">191</span>–<span class="lpage" tagx="lpage" title="lpage">201</span>.</span></li> <li tag="ref"><a name="pone.0072506-Davenport1" /><span class="label" tagx="label" title="label">68</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Davenport</span><span class="given-names" tagx="given-names" title="given-names">TRB</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Stanley</span><span class="given-names" tagx="given-names" title="given-names">WT</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Sargis</span><span class="given-names" tagx="given-names" title="given-names">EJ</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">De Luca</span><span class="given-names" tagx="given-names" title="given-names">DW</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Mpunga</span><span class="given-names" tagx="given-names" title="given-names">NE</span></span>, <span class="etal" title="etal"><i>et al.</i></span> (<span class="year" tagx="year" title="year">2006</span>) <span class="mixed-article-title" title="mixed-article-title">A new genus of African monkey, <i>Rungwecebus</i>: morphology, ecology, and molecular phylogenetics</span>. <span class="source" tagx="source" title="source">Science</span><span class="volume" tagx="volume" title="volume">312</span>: <span class="fpage" tagx="fpage" title="fpage">1378</span>–<span class="lpage" tagx="lpage" title="lpage">1381</span><span class="comment">doi:<a href="http://dx.doi.org/10.1126/science.1125631">10.1126/science.1125631</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16690815">16690815</a></span></span></li> <li tag="ref"><a name="pone.0072506-Roberts1" /><span class="label" tagx="label" title="label">69</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Roberts</span><span class="given-names" tagx="given-names" title="given-names">TE</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Davenport</span><span class="given-names" tagx="given-names" title="given-names">TRB</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hildebrandt</span><span class="given-names" tagx="given-names" title="given-names">KBP</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Jones</span><span class="given-names" tagx="given-names" title="given-names">T</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Stanley</span><span class="given-names" tagx="given-names" title="given-names">WT</span></span>, <span class="etal" title="etal"><i>et al.</i></span> (<span class="year" tagx="year" title="year">2010</span>) <span class="mixed-article-title" title="mixed-article-title">The biogeography of introgression in the critically endangered African monkey <i>Rungwecebus kipunji</i></span>. <span class="source" tagx="source" title="source">Biol Lett</span><span class="volume" tagx="volume" title="volume">6</span>: <span class="fpage" tagx="fpage" title="fpage">233</span>–<span class="lpage" tagx="lpage" title="lpage">237</span><span class="comment">doi:<a href="http://dx.doi.org/10.1098/rsbl.2009.0741">10.1098/rsbl.2009.0741</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19906681">19906681</a></span></span></li> <li tag="ref"><a name="pone.0072506-Dinesen2" /><span class="label" tagx="label" title="label">70</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Dinesen</span><span class="given-names" tagx="given-names" title="given-names">L</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lehmberg</span><span class="given-names" tagx="given-names" title="given-names">T</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Svendsen</span><span class="given-names" tagx="given-names" title="given-names">JO</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hansen</span><span class="given-names" tagx="given-names" title="given-names">LA</span></span> (<span class="year" tagx="year" title="year">1993</span>) <span class="mixed-article-title" title="mixed-article-title">Range extension and other notes on some restricted-range forest birds from West Kilombero in the Udzungwa Mountains, Tanzania</span>. <span class="source" tagx="source" title="source">Scopus</span><span class="volume" tagx="volume" title="volume">17</span>: <span class="fpage" tagx="fpage" title="fpage">48</span>–<span class="lpage" tagx="lpage" title="lpage">59</span>.</span></li> <li tag="ref"><a name="pone.0072506-McPeek1" /><span class="label" tagx="label" title="label">71</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">McPeek</span><span class="given-names" tagx="given-names" title="given-names">MA</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gavrilets</span><span class="given-names" tagx="given-names" title="given-names">S</span></span> (<span class="year" tagx="year" title="year">2006</span>) <span class="mixed-article-title" title="mixed-article-title">The evolution of female mating preferences: differentiation from species with promiscuous males can promote speciation</span>. <span class="source" tagx="source" title="source">Evolution</span><span class="volume" tagx="volume" title="volume">60</span>: <span class="fpage" tagx="fpage" title="fpage">1967</span>–<span class="lpage" tagx="lpage" title="lpage">1980</span><span class="comment">doi:<a href="http://dx.doi.org/10.1554/06-184.1">10.1554/06-184.1</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17133854">17133854</a></span></span></li> <li tag="ref"><a name="pone.0072506-Trauth1" /><span class="label" tagx="label" title="label">72</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Trauth</span><span class="given-names" tagx="given-names" title="given-names">MH</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Larrasoaña</span><span class="given-names" tagx="given-names" title="given-names">JC</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Mudelsee</span><span class="given-names" tagx="given-names" title="given-names">M</span></span> (<span class="year" tagx="year" title="year">2009</span>) <span class="mixed-article-title" title="mixed-article-title">Trends, rhythms and events in Plio-Pleistocene African climate</span>. <span class="source" tagx="source" title="source">Quat Sci Rev</span><span class="volume" tagx="volume" title="volume">28</span>: <span class="fpage" tagx="fpage" title="fpage">399</span>–<span class="lpage" tagx="lpage" title="lpage">411</span><span class="comment">doi:<a href="http://dx.doi.org/10.1016/j.quascirev.2008.11.003">10.1016/j.quascirev.2008.11.003</a></span></span></li> <li tag="ref"><a name="pone.0072506-Lawson1" /><span class="label" tagx="label" title="label">73</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation">Lawson LP (2010) The discordance of diversification: evolution in the tropical-montane frogs of the Eastern Arc Mountains of Tanzania. Mol Ecol 19, 4046–4060. doi: 10.1111/j.1365-294X.2010.04788.x.</span></li> <li tag="ref"><a name="pone.0072506-Graham1" /><span class="label" tagx="label" title="label">74</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Graham</span><span class="given-names" tagx="given-names" title="given-names">CH</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">VanDerWal</span><span class="given-names" tagx="given-names" title="given-names">J</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Phillips</span><span class="given-names" tagx="given-names" title="given-names">SJ</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Moritz</span><span class="given-names" tagx="given-names" title="given-names">C</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Williams</span><span class="given-names" tagx="given-names" title="given-names">SE</span></span> (<span class="year" tagx="year" title="year">2010</span>) <span class="mixed-article-title" title="mixed-article-title">Dynamic refugia and species persistence: tracking spatial shifts in habitat through time</span>. <span class="source" tagx="source" title="source">Ecography</span><span class="volume" tagx="volume" title="volume">33</span>: <span class="fpage" tagx="fpage" title="fpage">1062</span>–<span class="lpage" tagx="lpage" title="lpage">1069</span><span class="comment">doi:<a href="http://dx.doi.org/10.1111/j.1600-0587.2010.06430.x">10.1111/j.1600-0587.2010.06430.x</a></span></span></li> <li tag="ref"><a name="pone.0072506-Marshall1" /><span class="label" tagx="label" title="label">75</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation">Marshall AR, Topp-Jørgensen JE, Brink H, Fanning E (2005) Monkey abundance and social structure in two high elevation forest reserves in the Udzungwa Mountains of Tanzania. Int J Primatol 26(1), 127–145. doi: 10.1007/s10764-005-0011-z.</span></li> </ul> </div> </div>  </body></html>