<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<link rel="dns-prefetch" href="//d33xdlntwy0kbs.cloudfront.net" />
<link rel="shortcut icon" href="https://www.biorxiv.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<meta name="type" content="article" />
<meta name="category" content="article" />
<meta name="HW.identifier" content="/biorxiv/early/2019/05/13/635862.atom" />
<meta name="HW.pisa" content="biorxiv;635862v1" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="Climate adaptation shaped by subtle to moderate allele frequency shifts in loblolly pine" />
<meta name="DC.Identifier" content="10.1101/635862" />
<meta name="DC.Date" content="2019-05-13" />
<meta name="DC.Publisher" content="Cold Spring Harbor Laboratory" />
<meta name="DC.Rights" content="© 2019, Posted by Cold Spring Harbor Laboratory. The copyright holder for this pre-print is the author. All rights reserved. The material may not be redistributed, re-used or adapted without the author's permission." />
<meta name="DC.AccessRights" content="restricted" />
<meta name="DC.Description" content="Understanding the genomic basis of local adaptation is crucial to determine the potential of long-lived woody species to withstand changes in their natural environment. In the past, efforts to dissect the genomic architecture in gymnosperms species have been limited due to the absence of reference genomes. Recently, the genomes of some commercially important conifers, such as loblolly pine, have become available, allowing whole-genome studies of these species. In this study, we test for associations between 87k SNPs, obtained from whole-genome re-sequencing of loblolly pine individuals, and 270 environmental variables and combinations of them. We determine the geographic location of significant alleles and identify their genomic location using our newly constructed ultra-dense 26k SNP linkage map. We found that water availability is the main climatic variable shaping local adaptation of the species, and found 492 SNPs showing significant associations with climatic variables or combinations of them. Our results suggest that adaptation to climate in the species might have occurred by many changes in the allele frequency of alleles with moderate to small effect sizes, and by the smaller contribution of large effect alleles in genes related to moisture deficit, temperature and precipitation. Genomic regions of low recombination and high population differentiation harbored SNPs associated with principal components but not with individual climatic variables, suggesting climate adaptation might have evolved as a result of different selection pressures acting on groups of genes associated with an aspect of climate rather than on individual climatic variables." />
<meta name="DC.Contributor" content="Amanda R. De La Torre" />
<meta name="DC.Contributor" content="David B Neale" />
<meta name="article:published_time" content="2019-05-13" />
<meta name="article:section" content="New Results" />
<meta name="citation_title" content="Climate adaptation shaped by subtle to moderate allele frequency shifts in loblolly pine" />
<meta name="citation_abstract" lang="en" content="&lt;h3&gt;ABSTRACT&lt;/h3&gt;
&lt;p&gt;Understanding the genomic basis of local adaptation is crucial to determine the potential of long-lived woody species to withstand changes in their natural environment. In the past, efforts to dissect the genomic architecture in gymnosperms species have been limited due to the absence of reference genomes. Recently, the genomes of some commercially important conifers, such as loblolly pine, have become available, allowing whole-genome studies of these species. In this study, we test for associations between 87k SNPs, obtained from whole-genome re-sequencing of loblolly pine individuals, and 270 environmental variables and combinations of them. We determine the geographic location of significant alleles and identify their genomic location using our newly constructed ultra-dense 26k SNP linkage map. We found that water availability is the main climatic variable shaping local adaptation of the species, and found 492 SNPs showing significant associations with climatic variables or combinations of them. Our results suggest that adaptation to climate in the species might have occurred by many changes in the allele frequency of alleles with moderate to small effect sizes, and by the smaller contribution of large effect alleles in genes related to moisture deficit, temperature and precipitation. Genomic regions of low recombination and high population differentiation harbored SNPs associated with principal components but not with individual climatic variables, suggesting climate adaptation might have evolved as a result of different selection pressures acting on groups of genes associated with an aspect of climate rather than on individual climatic variables.&lt;/p&gt;" />
<meta name="citation_journal_title" content="bioRxiv" />
<meta name="citation_publisher" content="Cold Spring Harbor Laboratory" />
<meta name="citation_publication_date" content="2019/01/01" />
<meta name="citation_mjid" content="biorxiv;635862v1" />
<meta name="citation_id" content="635862v1" />
<meta name="citation_public_url" content="https://www.biorxiv.org/content/10.1101/635862v1" />
<meta name="citation_abstract_html_url" content="https://www.biorxiv.org/content/10.1101/635862v1.abstract" />
<meta name="citation_full_html_url" content="https://www.biorxiv.org/content/10.1101/635862v1.full" />
<meta name="citation_pdf_url" content="https://www.biorxiv.org/content/biorxiv/early/2019/05/13/635862.full.pdf" />
<meta name="citation_doi" content="10.1101/635862" />
<meta name="citation_num_pages" content="41" />
<meta name="citation_article_type" content="Article" />
<meta name="citation_section" content="New Results" />
<meta name="citation_firstpage" content="635862" />
<meta name="citation_author" content="Amanda R. De La Torre" />
<meta name="citation_author_institution" content="School of Forestry, Northern Arizona University" />
<meta name="citation_author_email" content="Amanda.de-la-torre@nau.edu" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0001-6647-723X" />
<meta name="citation_author" content="David B Neale" />
<meta name="citation_author_institution" content="Department of Plant Sciences, University of California-Davis" />
<meta name="citation_reference" content="Anderson JT, Lee CR, Rushworth C, Colautti R, Mitchell-Olds (2013) Genetic tradeoff and conditional neutrality contribute to local adaptation. Molecular Ecology, 22(3), 699–708." />
<meta name="citation_reference" content="Broman KW, Wu H, Sen S, Churchill GA. 2003. R/qtl: QTL mapping in experimental crosses. Bioinformatics, 19, 889–890." />
<meta name="citation_reference" content="Berg JJ, Coop G (2014) A population genetic signal of polygenic adaptation. Plos Genetics, 10(8), e1004412." />
<meta name="citation_reference" content="Bernhardsson C, Vidalis A, Wang X, Scofield DG, Schiffthaler B, Baison J, Street NR, Garcia-Gil MR, Ingvarsson PK (2019) An ultra-dense haploid genetic map for evaluating the highly fragmented genome assembly of Norway spruce (Picea abies). Genes, Genomes and Genetics https://dpi.org/10.1534/g3.118.200840." />
<meta name="citation_reference" content="Cavalli-Sforza LL (1966) Population structure and human evolution. Proceedings of the Royal Society B: Biological Sciences, 164, 362–379." />
<meta name="citation_reference" content="Csillery K, Lalague H, Vendramin GG, Gonzalez-Martinez SC, Fady B, Oddou-Muratorio S (2014) Detecting short spatial scale local adaptation and epistatic selection in climate-related candidate genes in European beech (Fagus sylvatica) populations. Molecular Ecology, 23, 4696–4708." />
<meta name="citation_reference" content="De La Torre AR, Birol I, Bousquet J, Ingvarsson PK, Jansson S, Jones SJM, Keeling CI, MacKay J, Nilsson O, Ritland K, Street N, Yanchuk A, Zerbe P, Bohlmann J (2014a) Insights into Conifer Giga-genomes. Plant Physiology, 166, 1–9." />
<meta name="citation_reference" content="De La Torre AR, Roberts DR, Aitken SN (2014b) Genome-wide admixture and ecological niche modelling reveal the maintenance of species boundaries despite long history of interspecific gene flow. Molecular Ecology, 23, 2046–2059." />
<meta name="citation_reference" content="De La Torre AR, Li Z, Van de Peer Y, Ingvarsson PK (2017) Contrasting rates of molecular evolution and patterns of selection among gymnosperms and flowering plants. Molecular Biology and Evolution, 34(6), 1363–1377." />
<meta name="citation_reference" content="De La Torre AR, Puiu D, Crepeau MW, Stevens K, Salzberg SL, Langley CH, Neale DB (2018) Genomic architecture of complex traits in loblolly pine. New Phytologist, 221:1789–1801." />
<meta name="citation_reference" content="Eckert AJ, Bower AD, Gonzalez-Martinez SC, Wegrzyn JL, Coop G, Neale DB (2010a) Back to nature: ecological genomics of loblolly pine (Pinus taeda, Pinaceae). Molecular Ecology, 19, 3789–3805." />
<meta name="citation_reference" content="Eckert AJ, van Heerwaarden J, Wegrzyn JL, Nelson CD, Ross-Ibarra J, Gonzalez-Martinez SC, Neale DB (2010b) Patterns of population structure and environmental association to aridity across the range of loblolly pine (Pinus taeda L, Pinaceae). Genetics, 185, 962–982." />
<meta name="citation_reference" content="Endelman JB, Plomion C (2014) LPmerge: An R package for merging genetic maps by linear programming. Bioinformatics, 30(11), 1623–1624." />
<meta name="citation_reference" content="Fournier-Level A, Korte A, Cooper MD, Nordborg M, Schmitt J, Wilczek AM (2011) A Map of Local Adaptation in Arabidopsis thaliana. Science, 334, 86–89." />
<meta name="citation_reference" content="Forester BR, Lasky JR, Wagner HH, Urban DL (2018). Comparing methods for detecting multilocus adaptation with multivariate genotype-environment associations. Molecular Ecology, 27, 2215–2233." />
<meta name="citation_reference" content="Frichot E, Schoville SD, Bouchard G, Francois O (2013) Testing for Associations between loci and environmental gradients using latent factor mixed models. Molecular Biology and Evolution, 30(7), 1687–1699." />
<meta name="citation_reference" content="Frichot E, Francois O (2015) LEA: An R package for landscape and ecological association studies. Methods in Ecology and Evolution, 6, 925–929." />
<meta name="citation_reference" content="Gonzalez-Martinez SC, Ersoz E, Brown GR, Wheeler NC, Neale DB (2006) DNA sequence variation and selection of tag SNP at candidate genes for drought-stress response in Pinus taeda L. Genetics, 172,1915–1926." />
<meta name="citation_reference" content="Hancock AM, Alkorta-Aranburu G, Witonsky DB, Di Rienzo A (2010) Adaptations to new environments in humans: the role of subtle allele frequency shifts. Phil trans R Soc B, 365, 2459–2468." />
<meta name="citation_reference" content="Hancock AM, Brachi B, Faure N, Horton MW, Jarymowycz LB, Sperone FG, Toomajian, Roux F, Bergelson J (2011) Adaptation to climate across the Arabidopsis thaliana genome. Science, 334(7), 83–86." />
<meta name="citation_reference" content="Harrisson KA, Amish SJ, Pavlova A, Narum SR, Telonis-Scott M, Rourke ML, Lyon J, Tonkin Z, Gilligan DM, Ingram BA, Lintermans M, Gan HM, Austin CM, Luikart G, Sunnucks P (2017) Signatures of polygenic adaptation associated with climate across the range of a threatened fish species with high connectivity. Molecular Ecology, 26, 6253–6269." />
<meta name="citation_reference" content="Hermisson J, Pennings PS (2005) Soft sweeps - molecular population genetics of adaptation from standing genetic variation. Genetics, 169, 2335–2352." />
<meta name="citation_reference" content="Hohenlohe PA, Phillips PC, Cresko W (2010) Using population genomics to detect selection in natural populations: key concepts and methodological considerations. International Journal of Plant Science, 171, 1059–1071." />
<meta name="citation_reference" content="Holliday JA, Zhou L, Bawa R, Zhang M, Oubida RW (2016) Evidence for extensive parallelism but divergent genomic architecture of adaptation along altitudinal and latitudinal gradients in Populus trichocarpa. New Phytologist, 209, 1240–1251." />
<meta name="citation_reference" content="Hornoy B, Pavy N, Gerardi S, Beaulieu J, Bousquet J (2015) Genetic adaptation to climate in white spruce involves small to moderate allele frequency shifts in functionally diverse genes. Genome Biology and Evolution, 7(12), 3269–3285." />
<meta name="citation_reference" content="Jombart, T (2008) Adegenet: An R package for the multivariate analysis of genetic markers. Bioinformatics 24, 1403–1405." />
<meta name="citation_reference" content="Jombart T. and Ahmed I (2011) Adegenet 1.3-1: new tools for the analysis of genome-wide SNP data. Bioinformatics." />
<meta name="citation_reference" content="Kopelman NM, Mayzel J, Jakobsson M, Rosenberg NA, Mayrose I (2015) CLUMPAK: a program for identifying clustering modes and packaging population structure inferences across K. Molecular Ecology Resources, 15(5), 1179–1191." />
<meta name="citation_reference" content="Krzywinski MI, Schein JE, Birol I, Connors J, Gascoyne R, Horsman D, Jones SJ, Marra MA (2009) Circos: An information aesthetic for comparative genomics. Genome Research, doi:10.1101/gr.092759.109." />
<meta name="citation_reference" content="Latta RG (1998) Differentiation of allelic frequencies at quantitative trait loci affecting locally adaptive traits. The American Naturalist, 151, 283–292." />
<meta name="citation_reference" content="Le Corre V, Kremer (2003) Genetic variability at neutral markers, quantitative trait loci, and trait in a subdivided population under selection. Genetics, 164, 1205–1219." />
<meta name="citation_reference" content="Le Corre V, Kremer A (2012) The genetic differentiation at quantitative trait loci under local adaptation. Molecular Ecology, 12, 1548–1566." />
<meta name="citation_reference" content="Lipka AE, Tian F, Wang Q, Peiffer J, Li M, Bradbury PJ, Gore MA, Buckler ES, Zhang Z (2012) GAPIT: genome association and prediction integrated tool. Bioinformatics, 28(18), 2397–2399." />
<meta name="citation_reference" content="Liu H, Cao F, Yin T, Chen Y (2017) A highly dense genetic map for Ginkgo biloba constructed using sequence-based markers. Frontiers in Plant Science, 8, 1041." />
<meta name="citation_reference" content="Lotterhos KE, Yeaman S, Degner J, Aitken S, Hodgins KA (2018) Modularity of genes involved in local adaptation to climate despite physical linkage. Genome Biology, 19,157." />
<meta name="citation_reference" content="Lu M, Krutovsky KV, Nelson CD, West JB, Reilly NA, Loopstra C (2017) Association genetics of growth and adaptive traits in loblolly pine (Pinus taeda L.) using whole-exome-discovered polymorphisms. Tree Genetics &amp; Genomes, 13, 57." />
<meta name="citation_reference" content="Martinez-Garcia PJ, Stevens KA, Wegrzyn JL, Liechty J, Crepeau M, Langley CH, Neale DB (2013) Combination of multipoint maximum likelihood (MML) and regression mapping algorithms to construct a high-density genetic linkage map for loblolly pine (Pinus taeda L.). Tree Genetics &amp; Genomes, 9, 1529–1535." />
<meta name="citation_reference" content="Neale DB, Martinez-Garcia PJ, De La Torre AR, Montanari S, Wei X (2017) Novel insights into tree biology and genome evolution as revealed through genomics. Annual Review of Plant Biology, 68, 13.1–13.27." />
<meta name="citation_reference" content="Neale DB, Kremer A (2011) Forest tree genomics: growing resources and applications. Nature Review Genetics, 12, 111–121." />
<meta name="citation_reference" content="Pfeifer B, Wittelsburger U, Ramos-Onsins SE, Lercher MJ (2014) PopGenome: an efficient Swiss army knife for populations genomic analyses in R. Molecular Biology and Evolution, 31(7), 1929–36." />
<meta name="citation_reference" content="Pritchard JK, Pickrell JK, Coop G (2010a) The genetics of human adaptation: hard sweeps, soft sweeps, and polygenic adaptation. Current Biology, 20, R208–R215." />
<meta name="citation_reference" content="Pritchard JK, Di Rienzo A (2010b) Adaptation-not by sweeps alone. Nature Review Genetics, 11(10), 665–667." />
<meta name="citation_reference" content="Prunier J, Laroche J, Beaulieu J, Bousquet J (2011) Scanning the genome for gene SNPs related to climate adaptation and estimating selection at the molecular level in boreal black spruce. Molecular Ecology, 20(8), 1702–1716." />
<meta name="citation_reference" content="RStudio Team (2016) RStudio: Integrated development for R. RStudio, Inc., Boston, MA. URL http://www.rstudio.com/" />
<meta name="citation_reference" content="Raj A, Stephens M, and JK. Pritchard (2014) fastSTRUCTURE: Variational Inference of Population Structure in Large SNP Data Sets. Genetics, 197, 573–589." />
<meta name="citation_reference" content="Savolainen O, Pyhajarvi T, Knurr T (2007) Gene flow and local adaptation in trees. Annual Review of Ecology, Evolution and Systematics, 38, 595–619." />
<meta name="citation_reference" content="Sewell MM, Sherman BK, Neale DB (1999) A Consensus Map for loblolly Pine (Pinus taeda L.). I. Construction and Integration of Individual Linkage Maps from Two outbred three-generation pedigrees. Genetics, 151, 321–330." />
<meta name="citation_reference" content="Schmidtling R (2001) Southern pine seed sources. In: Gen. Tech. Rep. SRS-44. U.S. Department of Agriculture, Forest Service, Southern Research Station, Asheville, NC, p 25." />
<meta name="citation_reference" content="Schmidtling RC (2003) The southern pines during the Pleistocene. ISHS Acta Horticulturae, 615, 203–209." />
<meta name="citation_reference" content="Slotte T (2014) The impact of linked selection on plant genomic variation. Briefings in Functional Genomics, 13(4), 268–275." />
<meta name="citation_reference" content="Smith JM, Haigh J (1974) The hitch-hiking of a favourable gene. Genetics Research, 23(1), 23–35." />
<meta name="citation_reference" content="Soltis DE, Morris AB, Mclachlan JS, Manos PS, Soltis PS (2006) Comparative phylogeography of unglaciated eastern North America. Molecular Ecology, 15, 4261–4293." />
<meta name="citation_reference" content="Taylor J, Butler D (2017) R package ASMap: Efficient Genetic Linkage Map Construction and Diagnosis. DOI: 10.18637/jss.v079.i06. arXiv:1705.06916." />
<meta name="citation_reference" content="Teskey RO, Bongarten BC, Cregg BM, Dougherty PM, Hennessey TC (1987) Physiology and genetics of tree growth response to moisture and temperature stress: an examination of the characteristics of loblolly pine (Pinus taeda L.). Tree Physiology, 3, 41–61." />
<meta name="citation_reference" content="Turner SD (2014) Qqman: an R package for visualizing GWAS results using Q-Q and manhattan plots. biorXiv DOI: 10.1101/005165." />
<meta name="citation_reference" content="Van Ooijen JW (2017) JoinMap5, software for the calculation of genetic linkage maps in experimental populations of diploid species. Kyazma B.V, Wageningen, Netherlands." />
<meta name="citation_reference" content="Via S (2012) Divergence hitchhiking and the spread of genomic isolation during ecological speciation-with-gene-flow. Phil. Trans R Soc B, 367, 451–460." />
<meta name="citation_reference" content="Wells OO, Wakeley PC (1966) Geographic variation in survival, growth and fusiform infection of planted loblolly pine. Forest Science Monographs, 11, 1–40." />
<meta name="citation_reference" content="Wang T, Hamann A, Spittlehouse D, Carroll C (2016) Locally downscaled and spatially customizable climate data for historical and future periods for North America. Plos One, 11(6), e0156720.https://doi.org/10.1371/journal.pone.0156720." />
<meta name="citation_reference" content="Wang J, Street NR, Scofield DG, Ingvarsson PK (2016) Variation in linked selection and recombination drive genomic divergence during allopatric speciation of European and American aspens. Molecular Biology and Evolution, 33(7), 1754–1767." />
<meta name="citation_reference" content="Westbrook JW, Chhatre VE, Wu LS, Chamala S, Gomide Neves L, Munoz P, Martinez-Garcia P, Neale D, Kirst M, Mockaitis K, Nelson CD, Peter GF, Davis JM, Echt CS (2015) A consensus genetic map for Pinus taeda and Pinus elliottii and extent of linkage disequilibrium in two genotype-phenotype discovery populations of Pinus taeda. Genes, genomes and genetics, 5, 1685–1694." />
<meta name="citation_reference" content="Whitlock MC, Lotterhos KE (2015) Reliable detection of loci responsible for local adaptation: inference of a null model through trimming the distribution of Fst. The American Naturalist, 186, S24–S36." />
<meta name="citation_reference" content="Williamson RJ, Josephs EB, Platts AE, Hazzouri KM, Haudry A, Blanchette M, Wright SI (2014) Evidence for widespread positive and negative selection in coding and conserved noncoding regions of Capsella. PLos Genetics, 10(9), e1004622." />
<meta name="citation_reference" content="Yeaman S, Whitlock MC (2011) The genetic architecture of adaptation under migration-selection balance. Evolution, 65, 1897–1911." />
<meta name="citation_reference" content="Yeaman S (2013) Genomic rearrangements and the evolution of clusters of locally adapted loci. PNAS, e1743–e1751." />
<meta name="citation_reference" content="Yeaman S (2015) Local adaptation by alleles of small effect. The American Naturalist, 186, S74–S89." />
<meta name="citation_reference" content="Yoder JB, Stanton-Geddes J, Zhou P, Briskine R, Young ND, Tiffin P (2014) Genomic signature of adaptation in Medicago truncatula. Genetics, 196, 1263–1275." />
<meta name="citation_reference" content="Zhang Z, Ersoz E, Lai CQ, Todhunter RJ, Tiwari HK, Gore MA, Bradbury PJ, Yu J, Arnett DK, Ordovas JM, Buckler ES (2010) Mixed linear model approach adapted for genome wide association studies. Nature Genetics, 42, 355–360." />
<meta name="citation_reference" content="Zheng Z, Schwartz S, Wagner L, Miller W (2000) A greedy algorithm for aligning DNA sequences. Journal of Computational Biology, 7(1-2), 203–14." />
<meta name="twitter:title" content="Climate adaptation shaped by subtle to moderate allele frequency shifts in loblolly pine" />
<meta name="twitter:site" content="@biorxivpreprint" />
<meta name="twitter:card" content="summary" />
<meta name="twitter:image" content="https://www.biorxiv.org/sites/default/files/images/biorxiv_logo_homepage7-5-small.png" />
<meta name="twitter:description" content="Understanding the genomic basis of local adaptation is crucial to determine the potential of long-lived woody species to withstand changes in their natural environment. In the past, efforts to dissect the genomic architecture in gymnosperms species have been limited due to the absence of reference genomes. Recently, the genomes of some commercially important conifers, such as loblolly pine, have become available, allowing whole-genome studies of these species. In this study, we test for associations between 87k SNPs, obtained from whole-genome re-sequencing of loblolly pine individuals, and 270 environmental variables and combinations of them. We determine the geographic location of significant alleles and identify their genomic location using our newly constructed ultra-dense 26k SNP linkage map. We found that water availability is the main climatic variable shaping local adaptation of the species, and found 492 SNPs showing significant associations with climatic variables or combinations of them. Our results suggest that adaptation to climate in the species might have occurred by many changes in the allele frequency of alleles with moderate to small effect sizes, and by the smaller contribution of large effect alleles in genes related to moisture deficit, temperature and precipitation. Genomic regions of low recombination and high population differentiation harbored SNPs associated with principal components but not with individual climatic variables, suggesting climate adaptation might have evolved as a result of different selection pressures acting on groups of genes associated with an aspect of climate rather than on individual climatic variables." />
<meta name="og-title" property="og:title" content="Climate adaptation shaped by subtle to moderate allele frequency shifts in loblolly pine" />
<meta name="og-url" property="og:url" content="https://www.biorxiv.org/content/10.1101/635862v1" />
<meta name="og-site-name" property="og:site_name" content="bioRxiv" />
<meta name="og-description" property="og:description" content="Understanding the genomic basis of local adaptation is crucial to determine the potential of long-lived woody species to withstand changes in their natural environment. In the past, efforts to dissect the genomic architecture in gymnosperms species have been limited due to the absence of reference genomes. Recently, the genomes of some commercially important conifers, such as loblolly pine, have become available, allowing whole-genome studies of these species. In this study, we test for associations between 87k SNPs, obtained from whole-genome re-sequencing of loblolly pine individuals, and 270 environmental variables and combinations of them. We determine the geographic location of significant alleles and identify their genomic location using our newly constructed ultra-dense 26k SNP linkage map. We found that water availability is the main climatic variable shaping local adaptation of the species, and found 492 SNPs showing significant associations with climatic variables or combinations of them. Our results suggest that adaptation to climate in the species might have occurred by many changes in the allele frequency of alleles with moderate to small effect sizes, and by the smaller contribution of large effect alleles in genes related to moisture deficit, temperature and precipitation. Genomic regions of low recombination and high population differentiation harbored SNPs associated with principal components but not with individual climatic variables, suggesting climate adaptation might have evolved as a result of different selection pressures acting on groups of genes associated with an aspect of climate rather than on individual climatic variables." />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.biorxiv.org/sites/default/files/images/biorxiv_logo_homepage7-5-small.png" />
<meta name="citation_date" content="2019-05-13" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/10.1101/635862v1.full.pdf" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/10.1101/635862v1.full.txt" />
<link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/10.1101/635862v1.ppt" />
<meta name="description" content="bioRxiv - the preprint server for biology, operated by Cold Spring Harbor Laboratory, a research and educational institution" />
<meta name="generator" content="Drupal 7 (http://drupal.org)" />
<link rel="canonical" href="https://www.biorxiv.org/content/10.1101/635862v1" />
<link rel="shortlink" href="https://www.biorxiv.org/node/732092" />
    <title>Climate adaptation shaped by subtle to moderate allele frequency shifts in loblolly pine | bioRxiv</title>  
    <link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__jMRAK66KMC1e4TQlwUNn3KiWVDC5AjueUAYEm1xBY_U__KSUjdT4jdcJ4qJz6fKY1K9WkYh1a5EcaWZxt_-zbTis__QJacVmveFyLIBSPkp21xL0ZvuDW9DMBrD2HB1P4Ry5I.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q5p1rr" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__icrP-Y0qw2NP73rJxxkUtrJta0z3VLAABtNjfSdEghA__rlN3rx0wOoswGNHZMeZM96bNn-3cLWHXTztGDoThNlw__QJacVmveFyLIBSPkp21xL0ZvuDW9DMBrD2HB1P4Ry5I.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__q48SDxVDWut_cH6dTPyQe7HjADuSLLmJVFztuiJEFrU__DJqddQGjm5dXCHsaiwNriGytKN5eevIQqyjdq65TZtE__QJacVmveFyLIBSPkp21xL0ZvuDW9DMBrD2HB1P4Ry5I.css" media="all" />
<style type="text/css" media="all">
/* <![CDATA[ */
#sliding-popup.sliding-popup-bottom,#sliding-popup.sliding-popup-bottom .eu-cookie-withdraw-banner,.eu-cookie-withdraw-tab{background:gray}#sliding-popup.sliding-popup-bottom.eu-cookie-withdraw-wrapper{background:transparent}#sliding-popup .popup-content #popup-text h1,#sliding-popup .popup-content #popup-text h2,#sliding-popup .popup-content #popup-text h3,#sliding-popup .popup-content #popup-text p,.eu-cookie-compliance-secondary-button,.eu-cookie-withdraw-tab{color:#fff !important}.eu-cookie-withdraw-tab{border-color:#fff}.eu-cookie-compliance-more-button{color:#fff !important}
/* ]]> */
</style>

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__QJacVmveFyLIBSPkp21xL0ZvuDW9DMBrD2HB1P4Ry5I.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__B6tGdsQSt1vT8d8j-19-oS4FKJe83RrSNBkjPs9jfos__6VkE5WcQmmQpSVTQx-xS2-zn_nlwFlDm-EN1HLJNjrA__QJacVmveFyLIBSPkp21xL0ZvuDW9DMBrD2HB1P4Ry5I.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__I6pBwZrmO51afTZVOjN83FdGH1QRb9t15lSLso0JVV4__F3sVKlGyHKCaAPI_WdiHD7l_-05B6b9bLdUwzi1me90__QJacVmveFyLIBSPkp21xL0ZvuDW9DMBrD2HB1P4Ry5I.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__QJacVmveFyLIBSPkp21xL0ZvuDW9DMBrD2HB1P4Ry5I.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__QJacVmveFyLIBSPkp21xL0ZvuDW9DMBrD2HB1P4Ry5I.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__CRdjIL0HKDL2T9slYvkTSTLypBBPjyoDhwBezBuDy7c__cHebPmPNdg2jyPmpyQpZWssJQ4cPrqfaabp5Or9KlkU__QJacVmveFyLIBSPkp21xL0ZvuDW9DMBrD2HB1P4Ry5I.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://d33xdlntwy0kbs.cloudfront.net/cshl_custom.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/all/libraries/modernizr/modernizr.min.js?q5p1rr"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__yOhrJUBmMpN-jsuQlJYmS8ovkQU6I6KInr295HrSTb0__c--zHhbuOWmkdWkJYXiNWDmkpDRPesmbBjGlQcyqBOY__QJacVmveFyLIBSPkp21xL0ZvuDW9DMBrD2HB1P4Ry5I.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var _prum=[['id', '52e95df3abe53d2e33000000'], ['mark', 'firstbyte',
      (new Date()).getTime()]]; (function() {
      var s=document.getElementsByTagName('script')[0],
      p=document.createElement('script');
      p.async='async'; p.src='//rum-static.pingdom.net/prum.min.js';s.parentNode.insertBefore(p,s);})();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js___KyvuEvfmewWxdOLBXTYsSUo8Pfm_DijV6fWtGvSL_k__HSKkTgIr7nDcdvQYewipf6v4ak6ZLUOQkwPj44wKouA__QJacVmveFyLIBSPkp21xL0ZvuDW9DMBrD2HB1P4Ry5I.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__3uCzzYE5jr6ly9nvsM8PRBRkhh4p0NWCMDja4EKBy4M__dmGGVYcB8LhnbR2iAmVW0ZyXI6QuFVYFRORa5XxglXs__QJacVmveFyLIBSPkp21xL0ZvuDW9DMBrD2HB1P4Ry5I.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__0zdUME0GLauYYyEch2tGPydIw-IG7Uizip0Qb4wUsig__3RgszrCgk4bOnTzo4vaQ-hrWzaG6oD19l-R9CQ1YQ-o__QJacVmveFyLIBSPkp21xL0ZvuDW9DMBrD2HB1P4Ry5I.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__W6wAjopaALytcBam7oT3WhEzllkWaB9R-8hazHyvShM__rK5YBIqBy9qt0di0Sn4DriBx9TJupdmv6uGmLJyZvDs__QJacVmveFyLIBSPkp21xL0ZvuDW9DMBrD2HB1P4Ry5I.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-45638026-1", {"cookieDomain":"auto"});ga("set", "page", location.pathname + location.search + location.hash);ga("send", "pageview");ga('create', 'UA-189672-38', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__1PtAOZ8x7l6QAi8D7X6Kj_Y7HcUOFzvWRvvYHUcCT98__U8IB92u_CnnGwJ-X_xaUa9lzsC8OzUfN0WkK2JQfirg__QJacVmveFyLIBSPkp21xL0ZvuDW9DMBrD2HB1P4Ry5I.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings,{"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"eiOrfHZYXRtrkfE0irB_H973pgoenu-mxyHurgPTXcg","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"modules\/comment\/comment.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"modules\/forum\/forum.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"sites\/default\/modules\/biorxiv\/css\/biorxiv.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/flexible\/flexible.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_collection\/css\/highwire_subject_collection.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire.highwire-list-expand.css":1,"public:\/\/ctools\/css\/bab59090150caf0c64e27afa1b36eef1.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/contrib\/eu_cookie_compliance\/css\/eu_cookie_compliance.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"0":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_biorxiv_styles\/css\/biorxiv-styles.css.less":1,"https:\/\/d33xdlntwy0kbs.cloudfront.net\/cshl_custom.css":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_figures.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_tables.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"http:\/\/d33xdlntwy0kbs.cloudfront.net\/cshl_custom.js":1,"sites\/all\/modules\/contrib\/eu_cookie_compliance\/js\/eu_cookie_compliance.js":1,"sites\/default\/modules\/jnl_biorxiv_styles\/js\/adv-search-collection-toggle.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/eu_cookie_compliance\/js\/jquery.cookie-1.4.1.min.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/default\/modules\/jnl_biorxiv_styles\/js\/biorxiv-scripts.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_nps_survey\/js\/nps_survey.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"sites\/default\/modules\/contrib\/disqus\/disqus.js":1,"misc\/textarea.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/contrib\/service_links\/js\/twitter_button.js":1,"sites\/all\/modules\/contrib\/service_links\/js\/facebook_like.js":1,"sites\/all\/modules\/contrib\/service_links\/js\/google_plus_one.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.highwire-list-expand.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"732092","apath":"\/biorxiv\/early\/2019\/05\/13\/635862.atom","pisa":"biorxiv;635862v1","processed":["highwire_math"],"markup":[{"requested":"full-text","variant":"full-text","view":"full","pisa":"biorxiv;635862v1"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"disqus":{"domain":"biorxivstage","url":"https:\/\/www.biorxiv.org\/content\/10.1101\/635862v1","title":"Climate adaptation shaped by subtle to moderate allele frequency shifts in loblolly pine","identifier":"node\/732092"},"panels_ajax_pane":{"new-41":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022JwxDiQ1QFDnYs4Afu7+8Xw3VzWgdYeV4ffRisLvY4AHpXvGh61asiDjSBrXRlYHBjbMnvZw4GEODwDSCYEOZn0H\\\\\\\/ADeLPnV1I0SRACEoBt9ZUQDrahB\\\\\\\/EZj9cHb3JSH8DzNmuChvEwiyWgHkbR9moN8n2yFGwWjdFriQqBYeQsNpD66ItPWOZjtdPZxrzbd1MxfKYmWoWdF0LofA+T\\\\\\\/o9uec45y+L+FLmiOmRh+LqQBiCbX0ntU5fOfhywLf9JdTjZAEHbiujB38n7nPPKE1n\\\\\\\/S\\\\\\\/XIT7CExrRwBKRXJdYIgcunAFGZugMe4vBYyLyLSd5r4qiCKxGdm0I4l0dM8lJJLq3tp9NudG4a7KEvdot3ABWAGj0+yl+97DEMttCkUMDlREdj35MCunyH4kYtXIWfLydHDecBIFYEDwkLnrnxUXU\\\\\\\/zks4Jt55+zjml9WNqkntX446bqQuzAPV\\\\\\\/rrj8qda0OXwUUAm3vpmaWUUfffV7AcXpCSPt0uOzzpS9m36MlKI0pRnPw0N70HsC3XPXCXASqkRZXaqBBFsPHt83md\\\\\\\/no+\\\\\\\/pQfOjXlomPQwQ6cBK9bz26jlE\\\\\\\/32rnp\\\\\\\/3cOiYQPeYFaHg75Yal9YvQjO1NtBnhtjAn1qlZOFVaaG2kLLiPsI73pQ54uj2b+f+PYrN6H+Xko6TdIFCMKcajzcJo37AX1sX\\\\\\\/b2k8lSjPU+\\\\\\\/CLj6lKSkPBMqoNeCp6GYYu3DnLBUB4EmE4P+iPdDhfYPSYB3Y5q3vz+y6WtnpJczVM7vXUwQGCPPblqzTTw70XjflIio75PkSRThbeBKXjyXVgjvVzwl+Xtq7zx7FAFFd\\\\\\\/FryaeBK\\\\\\\/Jq3YRGYo520265Oyno7tgK\\\\\\\/QNjjhRpdcXtQltugPIkhQxFjmNyGYsO2HZRSiJ8AYZzgmFmCDxaYPxSHR2MCQyxmbZr3UPDJ1bS0noLTRJ+23VV\\\\\\\/rrYjofphxr8uxOvRlSncBLIhM9ZXpO5Wzs4GTy0fd7UDe2Cg\\\\\\\/w9FrMDIVD2jNt5af4wokbdY0TjziHIGaPG85BPmV4uqrroF\\\\\\\/OVOHUT\\\\\\\/tJVTHYToAOr2GxQEvHu6kr9UDwIQF1m7YxM9UkvE6I1bZIM8fLD0woDwlegD6niPXnNrH14jqKs4e0azjAdXvEx\\\\\\\/ufGuK0sdTAOFhA+fZy7eKOgqsNEQQc0OFWlbKB94p84FaP121ykSMQsttk9BEZ7K757hMOjKJfRZDwlGd40O4J63yDBSylN7v31Wkg+lwXDNmjaFaL\\\\\\\/u2NQTuA5qQHdxLuV0iPCM7LpuFsEZV+S1E\\\\\\\/YbL7BHZ2xO5gxVv\\\\\\\/7UrXZlEX0ZdozmoMA=\\\u0022,\\\u0022iv\\\u0022:\\\u0022Yeba49PyowJE2q7E+qwftw==\\\u0022,\\\u0022salt\\\u0022:\\\u0022f9d8845712f72c9fee5db7d9ff111eab\\\u0022}\u0022,\u0022hmac\u0022:\u00222943440b01ae568c9a52edf3ea52729aa1e61043159223b11de9d13e2681731c\u0022}"},"urlIsAjaxTrusted":{"\/content\/10.1101\/635862v1.full":true},"ws_fl":{"width":100,"height":21},"ws_gpo":{"size":"","annotation":"","lang":"","callback":"","width":300},"color":{"logo":"https:\/\/www.biorxiv.org\/sites\/default\/files\/bioRxiv_article.jpg"},"highwire_list_expand":{"is_collapsed":"1"},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"eu_cookie_compliance":{"popup_enabled":1,"popup_agreed_enabled":0,"popup_hide_agreed":0,"popup_clicking_confirmation":1,"popup_scrolling_confirmation":false,"popup_html_info":"\u003Cdiv\u003E\n  \u003Cdiv class =\u0022popup-content info\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n      \u003Cp\u003EWe use cookies on this site to enhance your user experience. By clicking any link on this page you are giving your consent for us to set cookies.\u003C\/p\u003E\n    \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022agree-button eu-cookie-compliance-default-button\u0022\u003EContinue\u003C\/button\u003E\n                    \u003Cbutton type=\u0022button\u0022 class=\u0022find-more-button eu-cookie-compliance-more-button\u0022\u003EFind out more\u003C\/button\u003E\n          \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E","use_mobile_message":false,"mobile_popup_html_info":"\u003Cdiv\u003E\n  \u003Cdiv class =\u0022popup-content info\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n          \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022agree-button eu-cookie-compliance-default-button\u0022\u003EContinue\u003C\/button\u003E\n                    \u003Cbutton type=\u0022button\u0022 class=\u0022find-more-button eu-cookie-compliance-more-button\u0022\u003EFind out more\u003C\/button\u003E\n          \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E\n","mobile_breakpoint":"768","popup_html_agreed":"\u003Cdiv\u003E\n  \u003Cdiv class=\u0022popup-content agreed\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n      \u003Ch2\u003EThank you for accepting cookies\u003C\/h2\u003E\u003Cp\u003EYou can now hide this message or find out more about cookies.\u003C\/p\u003E    \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022hide-popup-button eu-cookie-compliance-hide-button\u0022\u003EHide\u003C\/button\u003E\n              \u003Cbutton type=\u0022button\u0022 class=\u0022find-more-button eu-cookie-compliance-more-button-thank-you\u0022 \u003EMore info\u003C\/button\u003E\n          \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E","popup_use_bare_css":false,"popup_height":"auto","popup_width":"100%","popup_delay":1000,"popup_link":"\/help\/cookie-policy","popup_link_new_window":1,"popup_position":null,"popup_language":"en","store_consent":false,"better_support_for_screen_readers":0,"reload_page":0,"domain":"","popup_eu_only_js":0,"cookie_lifetime":"365","cookie_session":false,"disagree_do_not_show_popup":0,"method":"default","whitelisted_cookies":"","withdraw_markup":"\u003Cbutton type=\u0022button\u0022 class=\u0022eu-cookie-withdraw-tab\u0022\u003EPrivacy settings\u003C\/button\u003E\n\u003Cdiv class=\u0022eu-cookie-withdraw-banner\u0022\u003E\n  \u003Cdiv class=\u0022popup-content info\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n      \u003Cp\u003E\u0026lt;h2\u0026gt;We use cookies on this site to enhance your user experience\u0026lt;\/h2\u0026gt;\u0026lt;p\u0026gt;You have given your consent for us to set cookies.\u0026lt;\/p\u0026gt;\u003C\/p\u003E\n    \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022eu-cookie-withdraw-button\u0022\u003EWithdraw consent\u003C\/button\u003E\n    \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E\n","withdraw_enabled":false},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1,"trackUrlFragments":1},"jnl_biorxiv_styles":{"defaultJCode":"biorxiv"},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-732092 node-type-highwire-article context-content hw-default-jcode-biorxiv hw-article-type-article hw-article-category-new-results">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-M677548" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-M677548');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
        <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
    
  <div id="zone-branding" class="zone zone-branding clearfix print-display-block container-30">
    <div class="grid-15 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="bioRxiv" src="https://www.biorxiv.org/sites/default/files/bioRxiv_article.jpg" /></a>      </div>
                </div>
          </div>
</div><div class="grid-11 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-system block-menu block-main-menu block-system-main-menu odd block-without-title" id="block-system-main-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/" title="" class="" data-icon-position="" data-hide-link-title="0">Home</a></li>
<li class="leaf" role="menuitem"><a href="/about-biorxiv" class="" data-icon-position="" data-hide-link-title="0">About</a></li>
<li class="leaf" role="menuitem"><a href="/submit-a-manuscript" class="" data-icon-position="" data-hide-link-title="0">Submit</a></li>
<li class="last leaf" role="menuitem"><a href="/content/alertsrss" title="" class="" data-icon-position="" data-hide-link-title="0">ALERTS / RSS</a></li>
</ul></nav>    </div>
  </div>
</div><div class="block block-panels-mini block-biorxiv-search-box block-panels-mini-biorxiv-search-box even block-without-title" id="block-panels-mini-biorxiv-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-biorxiv_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/10.1101/635862v1.full" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--2" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-RlRozmwlMSW9jMPbDAbwo3gmBNmH8HPSKrjHGljkCgs" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2 advanced-search-link" >
  
      
  
  <div class="pane-content">
    <a href="/search">Advanced Search</a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix container-30">
  	      </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="732092" id="node732092" data-pisa="biorxiv;635862v1" data-pisa-master="biorxiv;635862" data-apath="/biorxiv/early/2019/05/13/635862.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div  class="highwire-cite highwire-cite-highwire-article highwire-citation-biorxiv-article-top clearfix has-author-tooltip" >

      <span class="biorxiv-article-type">
        New Results    </span>
  
        <h1 class="highwire-cite-title" id="page-title">Climate adaptation shaped by subtle to moderate allele frequency shifts in loblolly pine</h1>  
      <div  class="highwire-cite-authors" ><span  class="highwire-citation-authors"><span class="highwire-citation-author first hw-author-orcid-logo-wrapper" data-delta="0"><a href="http://orcid.org/0000-0001-6647-723X" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a><span class="nlm-given-names">Amanda R.</span> <span class="nlm-surname">De La Torre</span></span>, <span class="highwire-citation-author" data-delta="1"><span class="nlm-given-names">David B</span> <span class="nlm-surname">Neale</span></span></span></div>
  
      <div  class="highwire-cite-metadata" ><span  class="highwire-cite-metadata-doi highwire-cite-metadata"><span class="label">doi:</span> https://doi.org/10.1101/635862 </span></div>
  
  
  </div>
<div id="hw-article-author-popups-node732092" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Amanda R. De La Torre </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-institution'>School of Forestry, Northern Arizona University</span>, AZ86011, <span class='nlm-country'>USA</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Amanda%2BR.%2BDe%2BLa%2BTorre%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=De%20La%20Torre%20AR&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AAmanda%2BR.%2BDe%2BLa%2BTorre%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0001-6647-723X" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Amanda R. De La Torre</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:Amanda.de-la-torre@nau.edu" class="" data-icon-position="" data-hide-link-title="0">Amanda.de-la-torre@nau.edu</a></span></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">David B Neale </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-institution'>Department of Plant Sciences, University of California-Davis</span>, One Shields Avenue CA95616, California, <span class='nlm-country'>USA</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=David%2BB%2BNeale%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Neale%20DB&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ADavid%2BB%2BNeale%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/10.1101/635862v1" class="panels-ajax-tab-tab" data-panel-name="biorxiv_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:732092" data-trigger="" data-url-enabled="1">Abstract</a><a href="/panels_ajax_tab/biorxiv_tab_art/node:732092/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/10.1101/635862v1.full-text" class="panels-ajax-tab-tab" data-panel-name="article_tab_full_text" data-target-id="highwire_article_tabs" data-entity-context="node:732092" data-trigger="full-text" data-url-enabled="1">Full Text</a><a href="/panels_ajax_tab/article_tab_full_text/node:732092/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/10.1101/635862v1.article-info" class="panels-ajax-tab-tab" data-panel-name="biorxiv_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:732092" data-trigger="article-info" data-url-enabled="1">Info/History</a><a href="/panels_ajax_tab/biorxiv_tab_info/node:732092/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/10.1101/635862v1.article-metrics" class="panels-ajax-tab-tab" data-panel-name="article_tab_metrics" data-target-id="highwire_article_tabs" data-entity-context="node:732092" data-trigger="article-metrics" data-url-enabled="1">Metrics</a><a href="/panels_ajax_tab/article_tab_metrics/node:732092/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/10.1101/635862v1.full.pdf+html" class="panels-ajax-tab-tab" data-panel-name="biorxiv_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:732092" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i> Preview PDF</a><a href="/panels_ajax_tab/biorxiv_tab_pdf/node:732092/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="article_tab_full_text" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.biorxiv.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-article_tab_full_text"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-markup" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2 class="">ABSTRACT</h2><p id="p-2">Understanding the genomic basis of local adaptation is crucial to determine the potential of long-lived woody species to withstand changes in their natural environment. In the past, efforts to dissect the genomic architecture in gymnosperms species have been limited due to the absence of reference genomes. Recently, the genomes of some commercially important conifers, such as loblolly pine, have become available, allowing whole-genome studies of these species. In this study, we test for associations between 87k SNPs, obtained from whole-genome re-sequencing of loblolly pine individuals, and 270 environmental variables and combinations of them. We determine the geographic location of significant alleles and identify their genomic location using our newly constructed ultra-dense 26k SNP linkage map. We found that water availability is the main climatic variable shaping local adaptation of the species, and found 492 SNPs showing significant associations with climatic variables or combinations of them. Our results suggest that adaptation to climate in the species might have occurred by many changes in the allele frequency of alleles with moderate to small effect sizes, and by the smaller contribution of large effect alleles in genes related to moisture deficit, temperature and precipitation. Genomic regions of low recombination and high population differentiation harbored SNPs associated with principal components but not with individual climatic variables, suggesting climate adaptation might have evolved as a result of different selection pressures acting on groups of genes associated with an aspect of climate rather than on individual climatic variables.</p></div><div class="section" id="sec-1"><h2 class="">INTRODUCTION</h2><p id="p-3">Local adaptation may arise by differential selection pressures across heterogeneous environments leading to increased fitness in the local environment compared to the non-local environment. Although of great interest by population geneticists, the genomic architecture of local adaptation remains largely unsolved in natural populations of non-model species (<a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">Anderson et al., 2013</a>). A majority of studies aiming to dissect the genetic architecture of local adaptation have focused at detecting signals of selection in which a new advantageous mutation in a single gene is rapidly driven to fixation, also known as the “hard sweep” model (<a id="xref-ref-51-1" class="xref-bibr" href="#ref-51">Smith &amp; Haigh, 1974</a>). In contrast, recent genome-wide association studies in humans and forest trees species have suggested a largely polygenic basis of local adaptation (<a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">Hancock et al., 2010</a>; <a id="xref-ref-41-1" class="xref-bibr" href="#ref-41">Pritchard et al., 2010a</a>; <a id="xref-ref-39-1" class="xref-bibr" href="#ref-39">Neale &amp; Kremer, 2011</a>; <a id="xref-ref-32-1" class="xref-bibr" href="#ref-32">Le Corre &amp; Kremer, 2012</a>). If a population that is well adapted to a geographic location moves to a new environment, natural selection will increase the frequency of certain alleles until the typical phenotype in the population matches the phenotype optimum in the new environment (<a id="xref-ref-41-2" class="xref-bibr" href="#ref-41">Pritchard et al., 2010a</a>). This type of adaptation also called “polygenic adaptation” is characterized by subtle to moderate shifts in allele frequencies and may be frequent in traits that have standing genetic variation for selection to act on, are highly heritable and controlled by many loci of small effect (<a id="xref-ref-41-3" class="xref-bibr" href="#ref-41">Pritchard et al., 2010a</a>, 2010b; <a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">Berg &amp; Coop, 2014</a>).</p><p id="p-4">A common approach to find genes contributing to local adaptation has been based on the idea that genes under selection should be more genetically differentiated among populations than a neutral locus, and will therefore have high Fst values (<a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">Cavalli-Sforza, 1966</a>; <a id="xref-ref-62-1" class="xref-bibr" href="#ref-62">Whitlock &amp; Lotterhos, 2015</a>). Loci mostly affected by spatially heterogeneous selection will have high Fst values, whereas the ones under spatially uniform balancing selection will show lower than neutral Fst values (<a id="xref-ref-62-2" class="xref-bibr" href="#ref-62">Whitlock &amp; Lotterhos, 2015</a>). Due to a wide distribution of Fst values in neutral markers, only advantageous alleles with high frequencies can be detected under the Fst outlier approach (Pritchard et al., 2010). This problem is exacerbated by the lack of power of individual tests when correcting for multiple comparisons over thousands of loci. As a result, weakly selected loci, characteristic of polygenic adaptation, will unlikely to be detected (<a id="xref-ref-32-2" class="xref-bibr" href="#ref-32">Le Corre &amp; Kremer, 2012</a>; <a id="xref-ref-66-1" class="xref-bibr" href="#ref-66">Yeaman, 2015</a>; <a id="xref-ref-62-3" class="xref-bibr" href="#ref-62">Whitlock &amp; Lotterhos, 2015</a>). In contrast, genotype by environment association (GEA) studies are more likely to detect signals associated with smaller allele frequency shifts (<a id="xref-ref-19-2" class="xref-bibr" href="#ref-19">Hancock et al., 2010</a>; <a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">Forester et al., 2018</a>).</p><p id="p-5">Lately, increasing numbers of genome-wide markers have enabled the study of the genomic architecture of local adaptation in natural populations. The geographic and genomic distribution of adaptive alleles can give us insights into the evolutionary forces that had shaped adaptation in a species. For example, alleles associated with day length in natural populations of <em>Arabidopsis thaliana</em> showed narrow geographic distribution in which one allele had rapidly driven to high frequency in the population as a result of a hard-selective sweep, whereas SNPs associated with relative humidity had widespread distributions (<a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">Hancock et al., 2011</a>). Also, the clustering of adaptive alleles in genomic regions of low recombination due to linkage or divergence hitchhiking can give us insights into the maintenance of population differentiation and local adaptation in the face of gene flow (<a id="xref-ref-57-1" class="xref-bibr" href="#ref-57">Via, 2012</a>; <a id="xref-ref-65-1" class="xref-bibr" href="#ref-65">Yeaman, 2013</a>).</p><p id="p-6">In the past, efforts to dissect the genomic architecture of local adaptation in gymnosperms species have been limited due to the absence of reference genomes. Recently, the genomes of some commercially important conifers have become available, allowing whole-genome studies of these species (<a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">De La Torre et al., 2014a</a>; <a id="xref-ref-38-1" class="xref-bibr" href="#ref-38">Neale et al., 2017</a>). Current assemblies of reference genomes are still quite fragmented and do not allow the location of genes in the genomes unless high-density linkage maps are available. Loblolly pine (<em>Pinus taeda</em>) is a widely distributed species in the southeastern United States, characterized by its outcrossing mating system, large population sizes, weak population structure and rapid decay of linkage disequilibrium (<a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">Eckert et al., 2010b</a>). Phylogeographic studies of unglaciated North America suggested loblolly pine follows the Mississippi River discontinuity, which is consistent with a dual Pleistocene refugial model, and has been used to explain differences in growth, disease resistance, drought tolerance and genetic differentiation between eastern and western populations (<a id="xref-ref-52-1" class="xref-bibr" href="#ref-52">Soltis et al., 2006</a>; <a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">Eckert et al., 2010a</a>, <a id="xref-ref-12-2" class="xref-bibr" href="#ref-12">2010b</a>; <a id="xref-ref-54-1" class="xref-bibr" href="#ref-54">Teskey et al., 1987</a>; <a id="xref-ref-48-1" class="xref-bibr" href="#ref-48">Schmidtling, 2001</a>). In this study, we aimed to dissect the genomic architecture of climate adaptation in loblolly pine. We tested for associations between 270 environmental variables and 87k SNPs obtained from widely distributed coding and non-coding regions across the 22-Gb genome of the species. We then determined the geographic and genomic location of significant alleles using our newly constructed, ultra-dense, 26k SNP linkage map. In addition to identify the main climatic variables driving the adaptation of the species, we were also interested in the following questions: a) Are adaptive alleles globally occurring alleles with varying frequencies or localized ones? b) Do adaptive alleles have narrow or widespread genomic distributions? c) Does local adaptation occur by large or subtle shifts in allele frequencies?.</p></div><div class="section" id="sec-2"><h2 class="">MATERIAL AND METHODS</h2><div id="sec-3" class="subsection"><h3>Sample collection and SNP genotyping</h3><p id="p-7">Needle tissue from 377 outcrossing, unrelated individuals distributed across the species’ natural range were collected from the ADEPT2 common garden located in Mississippi, southeast United States (<a id="xref-fig-1-1" class="xref-fig" href="#F1">Figure 1A</a>). In addition, two three-generation full-cross outbred pedigrees constructed and maintained by the Weyerhauser Company were used to collect 192 needle samples (<a id="xref-ref-47-1" class="xref-bibr" href="#ref-47">Sewell et al. 1999</a>). From these, 92 full-sib progeny samples came from the <em>qtl</em> pedigree, and 100 full-sibs came from the <em>base</em> pedigree. DNA was extracted using a protocol that included one day of tissue lysis and incubation at 96°C, followed by several steps of precipitation and filtering using the Qiagen DNeasy mini-prep Plant kit with an Eppendorf automated pipetting workstation. DNA concentration and quality were evaluated using picogreen on a Qubit Fluorometer. Raw reads from whole-genome re-sequencing data for ten individuals were used to call a large number of SNPs (455M) that were later scored, filtered and included in an Affymetrix Axiom myDesign species-specific and customized SNP array comprising 635k SNP markers (full description of this procedure can be found in <a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">De La Torre et al., 2018</a>). After removing SNPs that did not pass the genotyping quality control criteria and those that were monomorphic, we kept 84,738 high-resolution SNPs. In addition, 3,087 gene-based SNPs previously reported in Eckert et al. 2010 were added to the dataset, resulting in a total of 87,825 SNPs. From these SNPs, 20,367 matched genes, exons, transcripts or a combination of those.</p><div id="F1" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/05/13/635862/F1.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Population structure and nucleotide diversity of loblolly pine based on 87k genome&#xAD;wide SNP markers. Results of PCA and Structure suggest the presence of three different clusters along longitude (Fig.1A). Manhattan plots show pairwise Fst distributions between west and center populations (Fig. 1B), west and east populations (Fig.1C), and center and east populations (Fig. 1D). Dotted horizontal white line represents the mean pairwise Fst for each pair of populations. Average nucleotide diversity, Fst one vs all populations, and pairwise Fst among populations are shown in Figures 1E, 1F, 1G. Colors in all figures match genetic clusters (populations) in Fig.1A." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1639511907" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Population structure and nucleotide diversity of loblolly pine based on 87k genome&#xAD;wide SNP markers. Results of PCA and Structure suggest the presence of three different clusters along longitude (Fig.1A). Manhattan plots show pairwise Fst distributions between west and center populations (Fig. 1B), west and east populations (Fig.1C), and center and east populations (Fig. 1D). Dotted horizontal white line represents the mean pairwise Fst for each pair of populations. Average nucleotide diversity, Fst one vs all populations, and pairwise Fst among populations are shown in Figures 1E, 1F, 1G. Colors in all figures match genetic clusters (populations) in Fig.1A.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 1." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2019/05/13/635862/F1.medium.gif" width="440" height="226"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 1." src="https://www.biorxiv.org/content/biorxiv/early/2019/05/13/635862/F1.medium.gif" width="440" height="226"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/05/13/635862/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1." data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/05/13/635862/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml"><span class="fig-label">Figure 1.</span> <p id="p-8" class="first-child">Population structure and nucleotide diversity of loblolly pine based on 87k genome­wide SNP markers. Results of PCA and Structure suggest the presence of three different clusters along longitude (<a id="xref-fig-1-2" class="xref-fig" href="#F1">Fig.1A</a>). Manhattan plots show pairwise Fst distributions between west and center populations (<a id="xref-fig-1-3" class="xref-fig" href="#F1">Fig. 1B</a>), west and east populations (<a id="xref-fig-1-4" class="xref-fig" href="#F1">Fig.1C</a>), and center and east populations (<a id="xref-fig-1-5" class="xref-fig" href="#F1">Fig. 1D</a>). Dotted horizontal white line represents the mean pairwise Fst for each pair of populations. Average nucleotide diversity, Fst one vs all populations, and pairwise Fst among populations are shown in <a id="xref-fig-1-6" class="xref-fig" href="#F1">Figures 1E, 1F, 1G</a>. Colors in all figures match genetic clusters (populations) in <a id="xref-fig-1-7" class="xref-fig" href="#F1">Fig.1A</a>.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-4" class="subsection"><h3>Population structure, diversity estimates and Fst outlier test</h3><p id="p-9">Within populations’ nucleotide diversity for each SNP in each of the 12 linkage groups was estimated with the R Package PopGenome v.2.2.4 (<a id="xref-ref-40-1" class="xref-bibr" href="#ref-40">Pfeifer et al., 2014</a>). Population pairwise fixation index (Fst) was estimated by comparing all possible pairs of populations, and by comparing each population against all other individuals (Fst one vs. all). Fst values for each SNP in each linkage group were plotted in <a id="xref-fig-1-8" class="xref-fig" href="#F1">Figures 1B, 1C</a> and <a id="xref-fig-1-9" class="xref-fig" href="#F1">1D</a>; and average Fst values were displayed in <a id="xref-fig-1-10" class="xref-fig" href="#F1">Figures 1E, 1F, 1G</a>. Population structure in the SNP dataset was evaluated using the Python2.x fastStructure algortithm based on a variational framework for posterior inference of K clusters (<a id="xref-ref-45-1" class="xref-bibr" href="#ref-45">Raj et al., 2014</a>). Models in fastStructure were replicated 10 times with K from 1 to 10 using the default prior; seeds for random number generators were modified for each run. The chooseK.py python script in fastStructure was used to estimate the model complexity that maximizes marginal likelihood and the model components were used to explain structure in the data. Main pipeline and Distruct package in CLUMPAK (<a id="xref-ref-28-1" class="xref-bibr" href="#ref-28">Kopelman et al., 2015</a>) were used for the summation and graphical representation of FastStructure results. In addition, we did a PCA analysis using the Adegenet v2.0.1 R package (<a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">Jombart, 2008</a>; <a id="xref-ref-27-1" class="xref-bibr" href="#ref-27">Jombart &amp; Ahmed, 2011</a>). Outlier SNPs showing higher Fst than neutral loci were identified using the OutFLANK R package. OutFLANK infers the null distribution by removing loci in the top and bottom 5% of the distribution, and it is suggested to be robust to demographic history (<a id="xref-ref-62-4" class="xref-bibr" href="#ref-62">Whitlock &amp; Lotterhos, 2015</a>).</p></div><div id="sec-5" class="subsection"><h3>Genotype-environment associations (GEA)</h3><p id="p-10">We used a combination of 248 monthly, seasonal, and annual variables obtained from climate normal data from 1961-1990 in ClimateNA v5.41 (<a id="xref-ref-59-1" class="xref-bibr" href="#ref-59">Wang et al., 2016</a>). In addition, we used 19 GIS-derived bioclimatic variables from WorldClim 2.5-min (<a href="http://www.worldclim.org">www.worldclim.org</a>), and aridity index by quarter (every 4 months starting January), as previously calculated by <a id="xref-ref-12-3" class="xref-bibr" href="#ref-12">Eckert et al. (2010b)</a>. Geographical variables for each individual tree (latitude and longitude), and combinations of environmental data in the form of principal components, were also added to the analysis. Genotype by environment associations were identified for each of the 270 climatic variables with 87,859 SNPs with compressed mixed linear model (<a id="xref-ref-68-1" class="xref-bibr" href="#ref-68">Zhang et al., 2010</a>) implemented in the GAPIT R package (<a id="xref-ref-33-1" class="xref-bibr" href="#ref-33">Lipka et al., 2012</a>). To reduce the chance of identifying false positive associations as a result of population structure, we conducted the association analysis with only those SNPs having a minor allele frequency (maf) higher than 10% and used principal components of genetic data as co-variants. Manhattan plots were built using the SNP locations in our newly constructed ultra­dense linkage map for loblolly pine with the R package qqman (<a id="xref-ref-55-1" class="xref-bibr" href="#ref-55">Turner et al., 2014</a>). A subset of environmental traits showing significant associations with at least ten SNPs, were used to test associations between effect size (proportion of the variance explained) and minor allele frequency. SNP functional annotations were obtained from the annotated genome of loblolly pine v2.01 in TreeGenes (<a href="https://treegenesdb.org/Drupal">https://treegenesdb.org/Drupal</a>). For the SNPs matching transcripts, we aligned them against the non-redundant protein sequences database using BLASTX 2.8.0 (evalue &lt;1e-10) (<a id="xref-ref-69-1" class="xref-bibr" href="#ref-69">Zheng et al., 2000</a>).</p></div><div id="sec-6" class="subsection"><h3>Latent Factor Mixed Models (LFMM) analysis</h3><p id="p-11">In addition, we used a Bayesian bootstrap approach implemented in LFMM command line version 1.5 (<a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">Frichot et al., 2013</a>). LFMM accounts for random effects due to population structure and spatial autocorrelation with the use of latent factors (k) (<a id="xref-ref-16-2" class="xref-bibr" href="#ref-16">Frichot et al., 2013</a>). Each run was repeated five times with random seeds using the following parameters: k=3, 10,000 iterations, 5,000 burning length. Correction for multiple testing was done by adjusting p-values with the genomic inflation factor (l= median(Z-score<sup>2</sup>)/0.456) after combining z-scores obtained from multiple runs with the LEA R package (<a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">Frichot &amp; Francois, 2015</a>). Runs were repeated with k=2 and k=5, to check for sensitivity of results to the number of latent factors in the model. To address the potential correlations among environmental variables, we run a PCA analysis using the prcomp function in R package. The first 3 Principal Components resulting from this analysis were tested for associations with all the 87k SNPs in LFMM. Only SNPs with a minor allele frequency &gt;10% were included. We also tested for the presence of genomic clusters (more than 10 SNPs within a 1cM window) for all SNPs significantly associated with any climatic variable or principal component of them. Top 20% SNPs with the highest −log10(P) values for each principal component were retained for further analysis and considered as candidates for adaptation, following <a id="xref-ref-67-1" class="xref-bibr" href="#ref-67">Yoder et al. 2014</a>.</p></div><div id="sec-7" class="subsection"><h3>Environmental co-association network analysis</h3><p id="p-12">In addition to the univariate GEA analysis implemented in LFMM and Gapit, we also tested for the multivariate response of groups of genes associated with the environment, using the environmental co-association network analysis as described in <a id="xref-ref-35-1" class="xref-bibr" href="#ref-35">Lotterhos et al. 2018</a>. SNPs showing univariate associations in the GEA Gapit analysis were used to construct a network analysis using a hierarchical clustering of the associations between SNP allele frequencies and environmental variables using the reshape2 and gplots R packages in RStudio (<a id="xref-ref-44-1" class="xref-bibr" href="#ref-44">RStudio Team, 2016</a>).</p></div><div id="sec-8" class="subsection"><h3>Construction of individual pedigree linkage maps</h3><p id="p-13"><em>Qtl</em> and <em>base</em> pedigrees’ pseudo-testcrosses were used as R/qtl (<a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">Broman et al., 2003</a>) objects to construct ultra-dense linkage maps using the MSTmap algorithm implemented in the ASMap v.0.4 R package with default p-value (<a id="xref-ref-53-1" class="xref-bibr" href="#ref-53">Taylor &amp; Butler, 2017</a>). Pairwise recombination (r) and LOD scores (obtained from linkage disequilibrium test) were estimated for each pair of SNP markers. Markers showing an r&lt;0.5 were considered located in the same linkage group. SNP markers with pairwise recombination frequency estimated as zero (co-locating markers), were placed into a recombination bin with ASMap. Several rounds of stringent filtering included the removal of markers with 30% or more missing data, duplicated individuals, double crossovers, markers with distorted segregation patterns, and markers not mapping well in any of the linkage groups. The presence of switched alleles (wrong phase) was also accounted for. JointMap v5.0 (Van Ooijen et al., 2006) was used for fine mapping and ordering of bins obtained from ASMap.</p></div><div id="sec-9" class="subsection"><h3>Construction of average-sex and consensus maps</h3><p id="p-14">To allow the construction of averaged-sex maps for each pedigree, a set of anchor markers composed by 131 fragment-based markers (RAPD, RFLPs, ESTs, and SSRs) and 2804 SNPs (<a id="xref-ref-37-1" class="xref-bibr" href="#ref-37">Martinez-Garcia et al., 2013</a>; <a id="xref-ref-12-4" class="xref-bibr" href="#ref-12">Eckert et al., 2010b</a>) were added to our dataset. SNPs with suspect linkages as defined by a recombination frequency higher than 0.6 and a LOD score higher than 1 were excluded in further analyses. Forty-eight individual maps were merged to create twenty-four averaged-sex maps for each pedigree. Averaged-sex maps were merged using maximum intervals (K) from 1 to 8, generating eight consensus maps for each of the 12 linkage groups, with the R package LPmerge (<a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">Endelman &amp; Plomion, 2014</a>). Consensus maps with the lowest root mean-squared error (RMSE) standard deviation (mapping conflicts between individuals maps and consensus) were selected for each linkage group (<a id="xref-ref-13-2" class="xref-bibr" href="#ref-13">Endelman &amp; Plomion, 2014</a>). Graphical display was done with Circos (<a id="xref-ref-29-1" class="xref-bibr" href="#ref-29">Krzywinski et al., 2009</a>). All markers were anchored to the reference genome of loblolly pine v2.01 (<a href="https://treegenesdb.org/Drupal">https://treegenesdb.org/Drupal</a>) with the BWA-MEM algorithm in BWA (<a href="http://bio-bwa.sourceforge.net">http://bio-bwa.sourceforge.net</a>). Convergence and map accuracy was evaluated by comparing genome physical location (scaffold ID) and linkage group; and by comparing our maps with previously published maps in the species (<a id="xref-ref-37-2" class="xref-bibr" href="#ref-37">Martinez-Garcia et al., 2013</a>; <a id="xref-ref-61-1" class="xref-bibr" href="#ref-61">Westbrook et al., 2015</a>).</p></div></div><div class="section" id="sec-10"><h2 class="">RESULTS</h2><div id="sec-11" class="subsection"><h3>Population structure and genetic diversity levels</h3><p id="p-15">Results of the PCA analysis with 87k SNPs implemented in the Adegenet and Gapit R packages suggest the presence of three major genetic clusters (east, center, west) that extend longitudinally across the species’ natural range (<a id="xref-fig-1-11" class="xref-fig" href="#F1">Figure 1A</a>, Supporting Information Figure S1). When using the posterior inference of clusters based on variational Bayesian framework implemented in fastStructure, we found two major clusters when K varies from 2 to 4. When K=5, there is a third smaller cluster (Supporting Information Figure S2). However, our fastStructure test of the model complexity that maximizes marginal likelihood suggested K=2 better explains the genetic structure of the species. When K=2, the center and eastern populations are differentiated from the western populations, suggesting the Mississippi river as the major barrier for gene flow, as previously observed in <a id="xref-ref-10-2" class="xref-bibr" href="#ref-10">De La Torre et al. 2018</a>. We found that eastern and western populations present higher pairwise Fst estimates and were therefore more genetically distant than eastern and center, and center and western populations (p-value&lt;0.001; <a id="xref-table-wrap-1-1" class="xref-table" href="#T1">Table 1</a>; <a id="xref-fig-1-12" class="xref-fig" href="#F1">Figures 1B, 1C</a> and <a id="xref-fig-1-13" class="xref-fig" href="#F1">1D</a>). For example, from the 622 SNPs with a pairwise Fst &gt;0.3, 394 SNPs differentiated the western and eastern populations. Pairwise Fst between pairs of populations was found to significantly vary in all linkage groups with the exception of LG 7 and 10 (p-value&lt;0.001; <a id="xref-table-wrap-1-2" class="xref-table" href="#T1">Table 1</a>; <a id="xref-fig-1-14" class="xref-fig" href="#F1">Figures 1F, 1G</a>). Nucleotide diversity across all linkage groups was significantly different among populations (p-value&lt;0.01), and also differed between the center and west, and the east and west in linkage groups 3, 4, 7, 9 and 12 (p-value&lt;0.001; <a id="xref-table-wrap-1-3" class="xref-table" href="#T1">Table 1</a>; <a id="xref-fig-1-15" class="xref-fig" href="#F1">Figure 1E</a>). Average nucleotide diversity for all populations, based on SNP data, was 0.296, average Fst values (one population vs all others) was 0.029, and average pairwise Fst among all pairs of populations was 0.027 (<a id="xref-table-wrap-1-4" class="xref-table" href="#T1">Table 1</a>).</p><div id="T1" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/733985/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup"><a href="/highwire/markup/733985/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/733985" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 1.</span> <p id="p-16" class="first-child">Population differentiation (Fst comparison between one vs. all populations, and pairwise Fst calculations) and nucleotide diversity based on SNPs with known linkage group location in loblolly pine. Population labels (pop 1-3) represent genetic clusters from east to west, as described in <a id="xref-fig-1-16" class="xref-fig" href="#F1">Figure 1</a>.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-12" class="subsection"><h3>Fst outlier analysis</h3><p id="p-17">We screened out all SNP loci having low heterozygosity due to their confounding effects on Fst distribution, and we ended up with a dataset of 68k SNPs. Our results identified 152 SNPs with p-values less than 0.001, however none of them passed the threshold after correction for multiple testing (q value &lt;0.01). When increasing the RightTrimFraction (parameter that removes the loci mostly affected by selection before estimating the shape of the Fst distribution through likelihood), we found that 392 SNPs passed the q value threshold, however the fit of the null distribution model was reduced. OutFLANK identified a large number of SNPs with very high Fst values of up to 0.6, however the wide distribution of Fst values did not allow a clear distinction between putatively neutral and putatively under selection SNPs, as it is expected in a Fst outlier distribution.</p></div><div id="sec-13" class="subsection"><h3>Genotype-environment associations (GEA) and environmental co-association network analysis</h3><p id="p-18">A total of 2432 significant associations were found among 193 climate variables and 630 SNPs (FDR-Adjusted p-values&lt;0.01). After filtering SNPs with a minor allele frequency lower than 10%, 1279 associations were found among 186 climate variables and 113 SNPs (Supporting Information Tables S1 and S2). From the 113 SNPs, 31 SNPs came from coding regions, and 50 were located in linkage groups. No significant associations between maf and effect size [0.1-0.6, mean=0.36] was found in the dataset, suggesting a low contribution of rare alleles (Supporting Information Table S2). In fact, SNPs associated with principal components of climatic variables had an average maf of 0.365, and the ones associated with climate variables had an average maf of 0.278.</p><p id="p-19">From the climate variables, CMD_wt (winter Hargreaves Climatic Moisture Deficit, which is the difference between a reference evaporation and precipitation during winter) and CMD02 (Hargreaves Climatic Moisture Deficit in February) showed the highest number of associated markers, with 72 SNPs associated to each of them. All other climatic variables had a range of 3 to 13 SNPs associated to each climatic variable. Only a small group of SNPs matching transcripts aligned to known proteins at the NCBI non-redundant protein sequences database. Associated SNPs were mainly found to be involved in transport, transcriptional activity and enzymatic functions (Supporting Information Table S2). SNPs associated with any of the climatic variables were widely distributed across all 12 linkage groups in the genome of loblolly pine and some of them were associated with several climatic variables.</p><p id="p-20">Hierarchical clustering of significant associations between SNP allele frequencies and environmental variables suggests the presence of two main modules, one related to aridity (temperature variables, radiation and degree-days above 18°C) and another mainly associated with humidity and cold (climate moisture deficit during winter CMDwt, NFFD, Annual Heat-Moisture index AHM) (Supporting Information Figure S4). In the first module, six SNPs (AX-172791235, AX-173011888, AX-172909447, AX-173250534, AX-173348038, and AX-173361850) showed strong associations with a large number of temperature-related environmental variables (Supporting Information Table S2). These SNPs were also found to be associated with PC1 in both the Gapit and LFMM analysis. Interestingly, even though we were not able to map any of these SNPs in our linkage map, we know that three of them are located in the same scaffold (super 3645) based on the latest genome assembly of the species. The second module is composed by several sub-modules or sub-groups. In one of them, all 72 SNPs associated with CMDwt and CMD02 cluster together. Smaller sub-modules are associated with NFFD during april (9SNPs), Eref during the summer (4 SNPs); and Annual heat moisture index (4SNPs). Although our results suggest pleiotropic effects of the some of the SNPs, our conclusions are limited by the confounding effects of environmental correlations (Supporting Information Figure S5).</p><p id="p-21">PCA analysis of environmental variables showed PC1 explains 66.56% of the variation in the dataset, PC2 explains 14.49%, and PC3 6.51%. PC1 negatively correlates with Temperature-derived variables, Relative Humidity (RH) and Radiation (Rad); and positively with Degree-days below 0c (DD_0), Degree-days below 18c (DD_18), Precipitation as snow (PAS), Frost-Free period (FFP) and Temperature seasonality variables such as Isothermality (BIO3); Temperature seasonality (BIO4); Annual Temperature Range (BIO7), and Continentality (TD). PC2 positively correlates with Relative Humidity (RH), and negatively with Temperature seasonality variables (TS) (<a id="xref-fig-2-1" class="xref-fig" href="#F2">Figure 2A</a>, Supporting Information Figure S3). LFMM results showed 716 SNPs associated with PC1, 444 SNPs with PC2, and 741 SNPs with PC3 (Bonferroni-Holmes-Adjusted p-value&lt;0.01, maf&gt;0.1). Changing the number of latent factors (k) that account for population structure mainly identified the same group of associated SNPs (data not shown). We selected the top 20% significant SNPs with highest −log<sub>10</sub>(p-value) for each PC as potential candidates for local adaptation, resulting in 143 SNPs associated with PC1, 91 SNPs associated with PC2, and 149 SNPs associated with PC3 (Supporting Information Table S3). SNPs associated with any of the PCs were widely distributed across all 12 linkage groups in the genome (<a id="xref-fig-2-2" class="xref-fig" href="#F2">Figure 2B</a>).</p><div id="F2" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/05/13/635862/F2.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Principal component analysis of environmental variables. Principal components 1 and 2 are shown (Fig.2A). Climate variables full names are shown in Supporting Information Table S1. Figure 2B shows the genomic location of SNPs associated with Principal components of climate variables." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1639511907" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Principal component analysis of environmental variables. Principal components 1 and 2 are shown (Fig.2A). Climate variables full names are shown in Supporting Information Table S1. Figure 2B shows the genomic location of SNPs associated with Principal components of climate variables.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 2." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2019/05/13/635862/F2.medium.gif" width="440" height="260"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 2." src="https://www.biorxiv.org/content/biorxiv/early/2019/05/13/635862/F2.medium.gif" width="440" height="260"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/05/13/635862/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2." data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/05/13/635862/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 2.</span> <p id="p-22" class="first-child">Principal component analysis of environmental variables. Principal components 1 and 2 are shown (<a id="xref-fig-2-3" class="xref-fig" href="#F2">Fig.2A</a>). Climate variables full names are shown in Supporting Information Table S1. <a id="xref-fig-2-4" class="xref-fig" href="#F2">Figure 2B</a> shows the genomic location of SNPs associated with Principal components of climate variables.</p><div class="sb-div caption-clear"></div></div></div><p id="p-23">From the 492 individual SNPs showing associations with any climatic variable or Principal Component, 156 SNPs showed clinal patterns with increased or decreased subtle to moderate allele frequency shifts (mean=0.14 ± 0.07, median=0.125) along the longitudinal species’ natural range. Allele frequency of the minor allele increased from east to west in SNPs associated with Climatic Moisture Deficit during winter (CMD_wt), February (CMD02) and March (CMD03), and with Annual Heat Moisture Index (AHM) (<a id="xref-fig-3-1" class="xref-fig" href="#F3">Figure 3</a>). SNPs associated with Precipitation as snow (PAS01), Precipitation in September (PPT09), Degree-days above 18 °C, and Number of frost free days during autumn and spring showed an increase in the frequency of the minor allele from west to east. In SNPs associated with PC1 we found 36 SNPs showing increased or decreased allele frequency of the minor allele along longitude; 34 SNPs with PC2; and 59 SNPs with PC3 (Supporting Information Table S4).</p><div id="F3" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/05/13/635862/F3.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Genetic clines along longitude in which allele frequency of the minor allele increases from east to west in SNPs associated with Annual Heat Moisture Index (AHM) and Climatic Moisture Deficit (CMD)." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1639511907" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Genetic clines along longitude in which allele frequency of the minor allele increases from east to west in SNPs associated with Annual Heat Moisture Index (AHM) and Climatic Moisture Deficit (CMD).&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 3." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2019/05/13/635862/F3.medium.gif" width="440" height="313"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 3." src="https://www.biorxiv.org/content/biorxiv/early/2019/05/13/635862/F3.medium.gif" width="440" height="313"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/05/13/635862/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 3." data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/05/13/635862/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 3.</span> <p id="p-24" class="first-child">Genetic clines along longitude in which allele frequency of the minor allele increases from east to west in SNPs associated with Annual Heat Moisture Index (AHM) and Climatic Moisture Deficit (CMD).</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-14" class="subsection"><h3>Linkage maps</h3><p id="p-25">As a result of mapping with ASMap and JoinMap, 17,924 SNPs from the <em>base</em> pedigree and 10,995 SNPs from the <em>qtl</em> pedigree were mapped in 12 linkage groups in loblolly pine. Pairwise recombination and LOD scores for the <em>Qtl</em> and <em>base</em> pedigrees’ pseudo-testcrosses can be found in Supporting Information Figure S6. Anchors allowed the construction of average-sex maps using JoinMap. Total lengths were 2158.662 cM for the <em>base</em> linkage map and 2141.44 cM for the <em>qtl</em> linkage map. The consensus map had a length of 2270.41 cM across 12 linkage groups, and was built with 26,360 SNPs (<a id="xref-table-wrap-2-1" class="xref-table" href="#T2">Table 2</a>, Figure 6, Supporting Information Table S7). These results represent the most complete and dense map ever built for the species (previous map had 3856 markers in <a id="xref-ref-61-2" class="xref-bibr" href="#ref-61">Westbrook et al. 2015</a>), and one of the few ultra-dense linkage maps available to date in gymnosperms (others include Norway spruce, <a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">Bernhardsson et al., 2019</a>; and <em>Ginkgo biloba</em>, <a id="xref-ref-34-1" class="xref-bibr" href="#ref-34">Liu et al., 2017</a>). The mapped SNPs were distributed in 18,163 scaffolds in the genome of the species (loblolly pine v2.01 in TreeGenes, treegenesdb.org). Consensus maps had variable measurements of the lowest root mean-squared error (RMSE) standard deviation (mapping conflicts between individuals maps and consensus), ranging from 0.09 to 20.78. Scaffolds’ information of co-located SNPs and assuming all SNPs in the same scaffold were also in the same linkage group, we identified the location of 18,362 more SNPs, resulting in 44,722 SNPs with known positions in linkage groups (17,486 scaffolds) (Supporting Information Table S6). Average number of SNPs among linkage groups was 3726, with the number of SNPs in each linkage group ranged from 3179 to 4148 markers (Supporting Information Table S8). Convergence and map accuracy was evaluated by comparing genome physical location (scaffold ID) and linkage group and by comparing our maps with previously published maps in the species (<a id="xref-ref-37-3" class="xref-bibr" href="#ref-37">Martinez-Garcia et al., 2013</a>; <a id="xref-ref-61-3" class="xref-bibr" href="#ref-61">Westbrook et al., 2015</a>). Our results indicate that 4% of SNPs had conflicting positions between the physical and linkage positions, suggesting a very small number of SNPs within the same scaffolds were assigned to different linkage groups. These SNPs locations were not considered in further analysis. Comparison between 715 common SNPs in our map and previous maps revealed a convergence of 95.3% of linkage group assignment with <a id="xref-ref-37-4" class="xref-bibr" href="#ref-37">Martinez-Garcia et al. 2013</a> map and 94.2% with <a id="xref-ref-61-4" class="xref-bibr" href="#ref-61">Westbrook et al. 2015</a>. Linkage group numbers were chosen to match those at the <a id="xref-ref-37-5" class="xref-bibr" href="#ref-37">Martinez-Garcia et al. 2013</a> linkage map.</p><div id="T2" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/733981/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup"><a href="/highwire/markup/733981/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/733981" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 2.</span> <p id="p-26" class="first-child">Results of the linkage mapping in loblolly pine showing the length and number of SNPs in each linkage group for individual maps (<em>base</em> and <em>qtl</em> pedigrees) and consensus map. The ultra­dense consensus map has 26, 021 SNP markers and a length of 2270.21 cM. The root mean-squared error (RMSE) standard deviation gives information about potential mapping conflicts between individuals maps and the consensus map.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-15" class="subsection"><h3>Genomic clusters</h3><p id="p-27">We looked for the presence of genomic clusters at four different levels: a) SNPs associated with individual climatic variables; b) SNPs associated with principal components of climate variables; c) SNPs showing high pairwise Fst among populations; and d) SNPs located in the same environmental co-association modules. When evaluating SNPs associated with individual climatic variables, we found that clusters were non-existent (with the exception of a few SNPs associated with Climate Moisture Deficit that co-located at linkage groups 2 and 9; Supporting Information Table S2). In contrast, when evaluating SNPs associated with principal components, we found that larger numbers of co-located SNPs were found to be associated with the same or different groups of environmental variables (PCs) as it was observed in linkage groups 2 (10 SNPs at 190.9 cM), 5 (15 SNPs at 48.76 cM), and 10 (30 SNPs at 19.28 cM). SNPs in LG10 genomic cluster were also included among the top 20% most significant SNPs in the dataset (<a id="xref-fig-4-1" class="xref-fig" href="#F4">Figure 4</a>).</p><div id="F4" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/05/13/635862/F4.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Consensus linkage map containing 26,360 SNPs for loblolly pine. Blue region in the concentric inner circle represent the location of the largest cluster of SNPs associated with principal components of climatic variables. Size of the region was enlarged for easier visualization." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1639511907" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Consensus linkage map containing 26,360 SNPs for loblolly pine. Blue region in the concentric inner circle represent the location of the largest cluster of SNPs associated with principal components of climatic variables. Size of the region was enlarged for easier visualization.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 4." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2019/05/13/635862/F4.medium.gif" width="440" height="437"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 4." src="https://www.biorxiv.org/content/biorxiv/early/2019/05/13/635862/F4.medium.gif" width="440" height="437"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/05/13/635862/F4.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 4." data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/05/13/635862/F4.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 4.</span> <p id="p-28" class="first-child">Consensus linkage map containing 26,360 SNPs for loblolly pine. Blue region in the concentric inner circle represent the location of the largest cluster of SNPs associated with principal components of climatic variables. Size of the region was enlarged for easier visualization.</p><div class="sb-div caption-clear"></div></div></div><p id="p-29">Finally, the evaluation of SNPs showing high population differentiation suggested that many of the SNPs were located in groups with low recombination (co-located or within close proximity) (Supporting Information Table S5). Interestingly, even though many of these SNPs were not found to be associated with any principal component or individual climatic variables, they were located in some of same genomic clusters with SNPs showing associations (Supporting Information Table S5). In the case of SNPs located in the same environmental co-association groups, we found that SNPs located in the aridity module seem to be located in the same scaffold of the species, whereas the SNPs within the humidity module were located in different linkage groups (Supporting Information Figure S4)</p></div></div><div class="section" id="sec-16"><h2 class="">DISCUSSION</h2><p id="p-30">Previous studies suggest that population structure of loblolly pine has been mainly shaped by a dual-Pleistocene refugia that separated populations located east and west of the Mississippi river (<a id="xref-ref-58-1" class="xref-bibr" href="#ref-58">Wells &amp; Wakeley, 1966</a>; <a id="xref-ref-49-1" class="xref-bibr" href="#ref-49">Schmidtling, 2003</a>). In addition to that initial isolation, and in spite of continuous gene flow, east and west populations continued to differentiate as they became adapted to their new distinct environments. Our results suggest that adaptation to climate in the species might have occurred by many changes in the allele frequency of alleles with moderate to small effect sizes, and by the smaller contribution of large effect alleles in genes related to moisture deficit, temperature and precipitation.</p><div id="sec-17" class="subsection"><h3>Population structure</h3><p id="p-31">Our results suggest the presence of two major genetic clusters (east, west) and a smaller third cluster (center) that extend longitudinally across the species’ natural range (<a id="xref-fig-1-17" class="xref-fig" href="#F1">Figure 1A</a>, Supporting Information Figure S1). When K=2, the center and eastern populations are differentiated from the western populations, suggesting the Mississippi river as the major barrier for gene flow. It is being suggested that the Mississippi discontinuity is consistent with a dual­Pleistocene refugial model in which populations in southern Florida and southern Texas later migrated north and expanded their distribution to the current natural distribution of the species (<a id="xref-ref-58-2" class="xref-bibr" href="#ref-58">Wells &amp; Wakeley, 1966</a>; <a id="xref-ref-49-2" class="xref-bibr" href="#ref-49">Schmidtling, 2003</a>). We found that eastern and western populations present higher pairwise Fst estimates and were therefore more genetically distant than eastern and center, and center and western populations (<a id="xref-fig-1-18" class="xref-fig" href="#F1">Figures 1B, 1C</a> and <a id="xref-fig-1-19" class="xref-fig" href="#F1">1D</a>). Despite this differentiation, it is clear that populations were and currently are exchanging gene flow, as general genetic differentiation levels are low suggesting a low population structure. Our results are broadly consistent with previous, smaller scale studies regarding patterns of population structure in loblolly pine (Schimidtling, 2003; <a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">Gonzalez-Martinez et al., 2006</a>; <a id="xref-ref-11-2" class="xref-bibr" href="#ref-11">Eckert et al., 2010a</a>; <a id="xref-ref-12-5" class="xref-bibr" href="#ref-12">Eckert et al., 2010b</a>); and with studies in outcrossing, widespread forest tree species with large population sizes (<a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">De La Torre et al., 2014b</a>; <a id="xref-ref-46-1" class="xref-bibr" href="#ref-46">Savolainen et al., 2007</a>).</p></div><div id="sec-18" class="subsection"><h3>Water availability is the highest determinant for adaptation of the species</h3><p id="p-32">Water stress and temperature variation impose limitations in the survival, growth and productivity of many forest tree species. Loblolly pine is not the exception. Previous physiological and genetic studies have suggested differential responses to temperature and moisture across geographically distant populations of the species (<a id="xref-ref-54-2" class="xref-bibr" href="#ref-54">Teskey et al., 1987</a>; <a id="xref-ref-11-3" class="xref-bibr" href="#ref-11">Eckert et al., 2010a</a>; <a id="xref-ref-12-6" class="xref-bibr" href="#ref-12">Eckert et al., 2010b</a>; <a id="xref-ref-36-1" class="xref-bibr" href="#ref-36">Lu et al., 2017</a>). Populations east from the Mississippi river grow faster and taller and are less drought-tolerant than population in the west side (<a id="xref-ref-54-3" class="xref-bibr" href="#ref-54">Teskey et al., 1987</a>; <a id="xref-ref-48-2" class="xref-bibr" href="#ref-48">Schmidtling, 2001</a>). Although, <a id="xref-ref-36-2" class="xref-bibr" href="#ref-36">Lu et al, 2017</a> found a higher water use efficiency in eastern populations but attributed those results to the young age of trees growing in common gardens, which might be different to mature trees growing in natural populations. It is being suggested that average yearly minimum temperatures of the place of origin affect growth and survival in plantations of the species (<a id="xref-ref-48-3" class="xref-bibr" href="#ref-48">Schmidtling, 2001</a>).</p><p id="p-33">In our study, we found that moisture deficit and the relationship between precipitation and temperature seem to be driving the adaptation of the species. Our LFMM results showed that principal component 1, which explained 66.56% of the climatic variation in the dataset, was negatively correlated with Temperature-derived variables, Relative Humidity (RH) and Radiation (Rad); and positively with Degree-days below 0c (DD_0), Degree-days below 18c (DD_18), Precipitation as snow (PAS), Frost-Free period (FFP) and Temperature seasonality variables such as Isothermality (BIO3), Temperature seasonality (BIO4), Annual Temperature Range (BIO7), and Continentality (TD). A total of 143 SNPs were significantly associated with PC1 and many of them showed increased or decreased frequencies of the minor allele along the longitudinal range of the species. Functional annotation of these SNPs included ion transport, fatty acid metabolism, response to auxins, photosynthesis, salt-induced kinases, stress sensing and signal transduction (Supporting Information Table S3). In addition, our GEA results indicate a large number of SNPs associated with Climate Moisture Deficit (CMD_wt, CMD02, CMD03), especially during the winter months; Annual Heat-Moisture Index (AHM) and aridity during the third quarter of the year (AI_Q3); Radiation, Precipitation and Temperature. These SNPs were involved in transport, enzymatic and transcriptional activity (Supporting Information Table S2).</p><p id="p-34">These results are consistent with our previous study in the species that estimate the relationship between expression of xylem development genes and environmental variables. In that study, higher expression levels of MADS box protein, a transcription factor putatively acting as a heat shock protein binding were found with increased levels of climatic moisture deficit in May, and increased radiation in Spring. Also, higher expression levels of Xyloglucan endotransglycosylase 2 (XET-2), an enzyme involved in xylem development (and associated with Laccase 3, Laccase 7 and Phenylalanine ammonia lyase-1), were found with increased radiation during the Spring and decreased precipitation and moisture deficit during the summer (<a id="xref-ref-10-3" class="xref-bibr" href="#ref-10">De la Torre et al., 2018</a>).</p></div><div id="sec-19" class="subsection"><h3>Genomic distribution of adaptive alleles</h3><p id="p-35">Theory predicts that due to the decrease in fitness with increasing recombination rates, clusters of alleles contributing to adaptive trait variation are frequently located in genomic regions with low recombination (<a id="xref-ref-65-2" class="xref-bibr" href="#ref-65">Yeaman, 2013</a>). In our study, however, we didn’t find evidence for genomic clustering or “genomic Islands” in SNPs associated with individual climatic variables, which is consistent with the long-standing view that in conifers linkage disequilibrium decreases rapidly due to their high outcrossing rates leading to high recombination (<a id="xref-ref-39-2" class="xref-bibr" href="#ref-39">Neale &amp; Kremer, 2011</a>). Most SNPs showing significant associations with climatic variables or principal components of them had a wide genomic distribution within and among linkage groups, and were present in all the 12 linkage groups of the species. Widespread genomic distributions of adaptive alleles were also found in <em>Picea mariana</em> and <em>Medicago truncatula</em> (<a id="xref-ref-43-1" class="xref-bibr" href="#ref-43">Prunier et al., 2011</a>; <a id="xref-ref-67-2" class="xref-bibr" href="#ref-67">Yoder et al., 2014</a>). We only found small haplotype blocks (SNPs closely located with recombination equal or close to zero) in linkage groups 2 and 9 in SNPs associated with Climate Moisture Deficit during winter (CMD_wt) and February (CMD02). Other genomic clustering was not found in SNPs associated with any other climatic variable. However, it is important to mention that although our newly constructed map contains the largest number of SNPs (26k) ever mapped in the species, many of the SNPs showing significant associations with climate locations could not be located within linkage groups. An even higher density linkage map or a chromosome-scale reference genome would be required to confidently locate most or all of the SNPs associated with climatic variables.</p><p id="p-36">Interestingly, when evaluating SNPs associated with Principal Components and environmental co­association modules, we found larger numbers of co-located SNPs. This suggests that adaptation to climate in loblolly pine may have occur as a complex process in which different selection pressures are more likely to act on groups of genes associated with an aspect of climate rather than on individual climatic variables. These “co-adapted” complexes of genes may buffer against gene flow coming from maladaptive alleles from geographically proximal but climatically different locations, maintaining polymorphisms across the species’ natural range (<a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">Holliday et al., 2016</a>). Increased clustering of outlier loci was found across altitudinal gradients with high gene flow between populations of <em>Populus trichocarpa</em>, suggesting adaptation with gene flow might have occurred by divergence hitchhiking of physically proximate alleles in the species (<a id="xref-ref-24-2" class="xref-bibr" href="#ref-24">Holliday et al., 2016</a>). Evidence for recurrent hitchhiking was also found in <em>Capsella grandiflora</em>, an outcrossing species with large effective population size and low levels of linkage disequilibrium (<a id="xref-ref-63-1" class="xref-bibr" href="#ref-63">Williamson et al., 2014</a>).</p><p id="p-37">In addition to SNPs adapting to the same aspect of climate or PC, we also found a smaller group of co-located SNPs associated with different groups of environmental variables (PCs) as it was observed in linkage groups 2, 5 and 10 (Figure 6). In addition, a large cluster of SNPs was found at position 19.28 cM in linkage group 10, where 30 different SNPs associated with PC1, PC2 and PC3 were co-located. It is important to mention, however, that map accuracy of linkage group 10 might have been lowered by some discrepancies between individuals maps and the consensus map, as it was suggested by a higher RMSE (root mean-squared error) than in other linkage groups. Our results suggest that SNPs located in the aridity co-association module seem to be located in the same scaffold of the species, whereas the SNPs within the humidity module were located in different linkage groups. Physical linkage among loci adapting to different aspects of climate was also found in <em>Pinus contorta</em>, while studying modules of co-associated SNPs (<a id="xref-ref-35-2" class="xref-bibr" href="#ref-35">Lotterhos et al., 2018</a>). Both <a id="xref-ref-35-3" class="xref-bibr" href="#ref-35">Lotterhos et al. 2018</a> and this study suggest a complex genomic architecture of local adaptation in conifer species, in which the extent of physical linkage among loci is just one of the factors contributing to the species’ evolutionary response to changes in the climate.</p><p id="p-38">Perhaps our most interesting finding in relation to the genomic architecture in loblolly pine was that in spite of the rapid decay of LD, we found genomic regions of low recombination in three of the twelve linkage groups. These genomic regions showed a variety of SNPs associated with either PC1, PC2, PC3, or Climate Moisture Deficit and showed high population differentiation. Increased linkage disequilibrium between these climate-associated alleles of small effect may have prevented their swamping by gene flow and may have promoted their contribution to adaptive trait divergence (<a id="xref-ref-64-1" class="xref-bibr" href="#ref-64">Yeaman &amp; Whitlock, 2011</a>; <a id="xref-ref-66-2" class="xref-bibr" href="#ref-66">Yeaman, 2015</a>). In the case of the SNPs located in the genomic cluster in linkage group 5 (48.04-48.84 cM), we found a correlation between increased population differentiation and decreased diversity (R<sup>2</sup>= 0.3, P&lt;0.05). A negative relationship between recombination rate and genetic differentiation is a common signature of linked selection and has been observed in several plant species including <em>Populus tremula</em> and <em>P. tremuloides</em> (<a id="xref-ref-50-1" class="xref-bibr" href="#ref-50">Slotte et al., 2014</a>; <a id="xref-ref-60-1" class="xref-bibr" href="#ref-60">Wang et al., 2016</a>). In addition, two of these genomic clusters (groups 2 at 189.62-191.79 cM and 5 at 48.04-48.84 cM) were previously considered as “metabolic hotspots” because they harbor SNPs associated with several metabolites such as pelargonic acid, threonine, and other metabolites of unknown origin (<a id="xref-ref-10-4" class="xref-bibr" href="#ref-10">De La Torre et al., 2018</a>).</p></div><div id="sec-20" class="subsection"><h3>Adaptive alleles- globally occurring or localized?</h3><p id="p-39">An important question in population genetics is whether alleles conferring adaptation are globally occurring alleles or localized ones. If natural selection favors specific alleles in specific locations, it is expected that these would be common in these geographic locations but rare in others. On the contrary, if natural selection removes alleles that are deleterious in one location but neutral in others, we would expect to find high-frequency alleles across the species’ range (<a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">Fournier-Level et al., 2011</a>). Localized or private alleles with narrow geographic distribution in which one allele has rapidly driven to high frequency in the population may be a result of a hard-selective sweep, as it is being observed in natural populations of <em>Arabidopsis thaliana</em> (<a id="xref-ref-20-2" class="xref-bibr" href="#ref-20">Hancock et al., 2011</a>). Alternate alleles might be favored in different environments leading to antagonistic pleiotropy that can result in local adaptation and the maintenance of genetic polymorphisms by natural selection. In a different scenario (conditional neutrality), alleles may be under positive selection in one environment but neutral in others (<a id="xref-ref-1-2" class="xref-bibr" href="#ref-1">Anderson et al., 2013</a>).</p><p id="p-40">Our study found that putatively adaptive alleles in loblolly pine were widely distributed across the species’ natural range rather than localized ones. In fact, the presence of private alleles (only present in one population) was not observed in any of the SNPs showing associations with climate variables. Globally occurring alleles had varying frequencies in which the frequency of the minor allele increased from east to west in SNPs associated with Climatic Moisture Deficit during winter (CMD_wt), February (CMD02) and March (CMD03), and with Annual Heat Moisture Index (AHM); whereas SNPs associated with Precipitation as snow in January (PAS01), Degree-days above 18 °C during winter, and Number of frost free days during autumn and spring, showed an increase in the frequency of the minor allele from west to east. In <em>Arabidopsis thaliana</em> populations, SNPs associated with relative humidity also had widespread distributions (<a id="xref-ref-20-3" class="xref-bibr" href="#ref-20">Hancock et al., 2011</a>). With the absence of fitness measurements, we cannot tell if loblolly pine individuals carrying these alleles are fitter in one environment or the other. However, the fact that the direction of the increase of the minor allele is coincident with increasing levels of the associated climatic variable suggests that these individuals may be locally adapted in that environment.</p></div><div id="sec-21" class="subsection"><h3>Allele frequency shifts at many adaptive loci</h3><p id="p-41">Local adaptation in natural populations may arise by differential selection pressures across heterogeneous environments in which the targets of selection may change from one environment to another. As a consequence of this, different combinations of alleles might be favored in different environments and maintained as stable polymorphisms, or experience “partial” or “soft” sweeps due to selection acting on standing variation (<a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">Hermisson &amp; Pennings, 2005</a>; <a id="xref-ref-67-3" class="xref-bibr" href="#ref-67">Yoder et al., 2014</a>). Recent studies have found a largely polygenic basis of adaptation in natural populations, in which trait variation is controlled by many loci of small effect and adaptation is characterized by subtle to moderate shifts in allele frequencies (<a id="xref-ref-41-4" class="xref-bibr" href="#ref-41">Pritchard et al., 2010a</a>, <a id="xref-ref-42-1" class="xref-bibr" href="#ref-42">2010b</a>; <a id="xref-ref-3-2" class="xref-bibr" href="#ref-3">Berg &amp; Coop, 2014</a>). With strong diversifying selection and high gene flow, considerable trait divergence may evolve with small allele frequency changes at individual loci (low Fst) but high between-population covariance in allele effect sizes (<a id="xref-ref-30-1" class="xref-bibr" href="#ref-30">Latta, 1998</a>; <a id="xref-ref-31-1" class="xref-bibr" href="#ref-31">Le Corre &amp; Kremer, 2003</a>, 2012). Adaptation to climate variation via selection on polygenic traits and/or small allele frequency shifts has been observed in <em>Medicago truncatula</em>, <em>Picea glauca</em>, <em>Fagus sylvatica</em> and <em>Maccullochella peelii</em> (<a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">Csillery et al., 2014</a>; <a id="xref-ref-67-4" class="xref-bibr" href="#ref-67">Yoder et al., 2014</a>; <a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">Hornoy et al., 2015</a>; Harrison et al., 2017). In <em>Picea glauca</em>, small to moderate shifts in the allele frequency of putatively climate-adaptive genes was found in response to recent selection and high gene flow among populations (<a id="xref-ref-25-2" class="xref-bibr" href="#ref-25">Hornoy et al., 2015</a>).</p><p id="p-42">In our study, we found a large number of SNPs with small to moderate effect sizes associated with climatic variables or combinations of them. In most of these SNPs, we found subtle to moderate shifts in allele frequencies across different environments, in which in many cases the increase in the frequency of the minor allele mirrored an increase of the climate variable along longitudinal gradients. These SNPs were largely captured by our GEA study, which, in contrast to Fst outlier tests has been suggested to detect signals associated with smaller allele frequency shifts (<a id="xref-ref-19-3" class="xref-bibr" href="#ref-19">Hancock et al., 2010</a>). Coincidently, our Fst outlier test failed to detect the signal of selection in these weakly selected loci involved in local adaptation of loblolly pine. In polygenic traits, most loci involved in local adaptation will experience weak selection, therefore they will not be substantially more differentiated than expected of neutral loci (<a id="xref-ref-62-5" class="xref-bibr" href="#ref-62">Whitlock &amp; Lotterhos, 2015</a>; <a id="xref-ref-32-3" class="xref-bibr" href="#ref-32">Le Corre &amp; Kremer, 2012</a>). Loci highly associated with expression and disease resistance and previously suggested to be under balancing selection (<a id="xref-ref-10-5" class="xref-bibr" href="#ref-10">De La Torre et al., 2018</a>) were also not identified by the Fst outlier analysis, probably because the OutFLANK procedure is not accurate in the left tail of the Fst distribution (<a id="xref-ref-62-6" class="xref-bibr" href="#ref-62">Whitlock &amp; Lotterhos, 2015</a>). The little difference in Fst across associated versus randomly chosen SNPs may also suggest that natural selection is not driving large-scale adaptive differences among lineages of loblolly pine. Instead, genotypes are being favored by natural selection across different environments regardless of their ancestry (<a id="xref-ref-12-7" class="xref-bibr" href="#ref-12">Eckert et al., 2010b</a>).</p><p id="p-43">Responses to selection that arise from standing genetic variation rather than new mutations or that are relatively recent for fixation to have occurred leave a fainter molecular signature (<a id="xref-ref-22-2" class="xref-bibr" href="#ref-22">Hermisson &amp; Pennings, 2005</a>; <a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">Hohenlohe et al., 2010</a>). These partial, soft sweeps often leave a signature of reduced haplotype or nucleotide diversity and extended linkage, as we found in our LG 5. Considering the low mutation rates in conifers (<a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">De La Torre et al., 2017</a>), it is likely that many of these changes in allele frequencies might have been facilitated by the great levels of standing genetic variation rather than by de novo mutations in loblolly pine. Simulations studies have suggested that good levels of standing genetic variation are required when local adaptation occurs by alleles of small effect (<a id="xref-ref-66-3" class="xref-bibr" href="#ref-66">Yeaman, 2015</a>). In addition, the long generation times in conifers and relatively recent migration from refugia of the species, might have contributed to the genetic architecture we see today, in which most beneficial alleles are still segregating in the populations and have not reached fixation. We therefore conclude that local adaptation to climate in loblolly pine might have occurred by many changes in the allele frequency of alleles with moderate to small effect sizes, and by the smaller contribution of large effect alleles in genes related to moisture deficit, temperature and precipitation.</p></div></div><div class="section" id="sec-22"><h2 class="">DATA ACCESIBILITY</h2><p id="p-45">Consensus linkage maps and genotyping data will be available in public database repositories such as TreeGenes and Dryad after the acceptance of the paper.</p></div><div class="section" id="sec-23"><h2 class="">AUTHOR CONTRIBUTIONS</h2><p id="p-46">D.N and A.D.L.T designed the research study, A.D.L.T did all the data analyses and wrote the manuscript, D.N reviewed and commented the final version of the manuscript.</p></div><div class="section COI-statement" id="sec-24"><h2 class="">CONFLICT OF INTEREST</h2><p id="p-47">The authors declare no conflict of interest.</p></div><div class="section" id="sec-25"><h2 class="">SUPPORTING INFORMATION</h2><p id="p-48"><strong>Table S1</strong>. List of climatic variables showing significant associations with one or more environmental variables tested in this study.</p><p id="p-49"><strong>Table S2</strong>. Results of the GEA analysis based on 87k genome-wide SNPs using GAPIT R package. Variables include: climate variable, SNP marker, physical location (scaffold id), physical position (scaffold position), consensus linkage group (LG) and position (cM), consensus_physical_linkage (linkage group is known but not position), maf (minor allele frequency), R<sup>2</sup> (percent of variation explained), p-value (p-value before correction for multiple testing), adjusted_p-values (p-value after correction for multiple testing), annotation and annotation id, functional annotation, and species (reference for functional annotation).</p><p id="p-50"><strong>Table S3</strong>. Results of the GEA analysis using the program LFMM. Only associated SNPs within the top 20% with the highest −log<sub>10</sub>(p-value) are shown. The first three principal components of environmental data were tested for associations with 87k genome-wide SNPs. Variables include: climate variable, SNP marker, physical location (scaffold id), consensus linkage group (LG) and position (cM), consensus_physical_linkage (linkage group is known but not position), maf (minor allele frequency), R<sup>2</sup> (percent of variation explained), p-value (p-value before correction for multiple testing), adjusted_p-values (p-value after correction for multiple testing), annotation and annotation id, functional annotation, and species (reference for functional annotation).</p><p id="p-51"><strong>Table S4</strong>. Genetic clines along longitude for SNPs showing significant associations with environmental variables or principal components of them. Variables include: climate variable, SNP, allele frequencies in the east, center and west populations, allele frequency differential, direction of increase in allele frequency, maf (minor allele frequency), p-value (p-value before correction for multiple testing), adjusted_p-values (p-value after correction for multiple testing), annotation and annotation id, functional annotation, species (reference for functional annotation),</p><p id="p-52"><strong>Table S5</strong>. Genomic clusters of SNPs showing significant pairwise population differentiation. Variables included are: SNP, physical location (scaffold id), physical position (scaffold position), consensus linkage group (LG) and position (cM), Pairwise Fst between pairs of populations, nucleotide diversity for each population, annotation and annotation id.</p><p id="p-53"><strong>Table S6</strong>. Genomic location for all the 87k SNPs in this study taken from our newly constructed ultra-dense linkage maps in loblolly pine. Variables indicate: physical location in the genome (scaffold id), consensus linkage group (LG) and position (cM), and consensus_physical_linkage (linkage group is known but not position),</p><p id="p-54"><strong>Table S7</strong>. Number of SNPs identified in each linkage group after construction of the consensus ultra-dense linkage map and incorporation of physical (scaffold) location.</p><p id="p-55"><strong>Figure S1</strong>. Principal component analysis of genotyping data for 87k SNPs suggest the presence of three major genetic clusters (from left to right: east, center and west) that extend longitudinally across the natural distribution of loblolly pine.</p><p id="p-56"><strong>Figure S2</strong>. FastStructure cluster plots from K=2 to K=5 show two major genetic clusters (east and center in blue, and west in orange) in the natural distribution of loblolly pine.</p><p id="p-57"><strong>Figure S3</strong>. Significant correlations between individual environmental variables and Principal Component 1 (PC1).</p><p id="p-58"><strong>Figure S4</strong>. Results of environmental co-association network analysis. Hierarchical clustering of top-significant (R2&gt;0.4) associations between SNP allele frequencies and environmental variables shows two main clusters, right cluster: related to aridity (temperature variables, radiation and degree-days above 18°C) and a left cluster: mainly associated with humidity and cold (precipitation during winter).</p><p id="p-59"><strong>Figure S5</strong>. Heatmap showing correlations among individual environmental variables. Large cluster in blue (top right) is composed by associated variables NFFD, Radiation (Rad and MAR), Temperature-related variables, Degree-days above 18°C during the summer, Evaporation (Eref), and Degree-days above 5°C.</p><p id="p-60"><strong>Figure S6</strong>. Pairwise recombination and Linkage disequilibrium in loblolly pine; (A) recombination (left side) and LOD scores (right side) in <em>Qtl</em> pedigree pseudo-testcross 1; (B) recombination (left side) and LOD scores (right side) in <em>Qtl</em> pedigree pseudo-testcross 2; (C) Map recombination fraction, map distance, LOD scores, estimated recombination fraction and number of recombination events in each linkage group in the <em>Qtl</em> pedigree pseudo-testcross 1.</p></div><div class="section ack" id="ack-1"><h2 class="">ACKNOWLEDGEMENTS</h2><p id="p-44">This project was supported by the U.S. Department of Agriculture/National Institute of Food and Agriculture (“PineRefSeq”; 2011-67009-30030) awarded to D.B.N at the University of California, Davis. The authors would like to thank Chuck Burdine, Patrick Cumbie and Dana Nelson for sample collection; and members of the Neale’s Lab Annarita Marrano, Sara Montanari and Pedro Martinez for suggestions on linkage mapping.</p></div><div class="section ref-list" id="ref-list-1"><h2 class="">REFERENCES</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference  in text" id="ref-1">↵</a><div class="cit ref-cit ref-journal" id="cit-635862v1.1" data-doi="10.1111/j.1365-294X.2012.05522.x"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Anderson</span>  <span class="cit-name-given-names">JT</span></span>, <span class="cit-auth"><span class="cit-name-surname">Lee</span>  <span class="cit-name-given-names">CR</span></span>, <span class="cit-auth"><span class="cit-name-surname">Rushworth</span>  <span class="cit-name-given-names">C</span></span>, <span class="cit-auth"><span class="cit-name-surname">Colautti</span>  <span class="cit-name-given-names">R</span></span>, <span class="cit-auth"><span class="cit-name-surname">Mitchell-Olds</span></span> (<span class="cit-pub-date">2013</span>) <span class="cit-article-title">Genetic tradeoff and conditional neutrality contribute to local adaptation</span>. <abbr class="cit-jnl-abbrev">Molecular Ecology</abbr>, <span class="cit-vol">22</span>(<span class="cit-issue">3</span>), <span class="cit-fpage">699</span>–<span class="cit-lpage">708</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2BEcology%26rft.volume%253D22%26rft.spage%253D699%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1365-294X.2012.05522.x%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1365-294X.2012.05522.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000313726300012&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference  in text" id="ref-2">↵</a><div class="cit ref-cit ref-journal" id="cit-635862v1.2" data-doi="10.1093/bioinformatics/btg112"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Broman</span>  <span class="cit-name-given-names">KW</span></span>, <span class="cit-auth"><span class="cit-name-surname">Wu</span>  <span class="cit-name-given-names">H</span></span>, <span class="cit-auth"><span class="cit-name-surname">Sen</span>  <span class="cit-name-given-names">S</span></span>, <span class="cit-auth"><span class="cit-name-surname">Churchill</span>  <span class="cit-name-given-names">GA</span></span>. <span class="cit-pub-date">2003</span>. <span class="cit-article-title">R/qtl: QTL mapping in experimental crosses</span>. <abbr class="cit-jnl-abbrev">Bioinformatics</abbr>, <span class="cit-vol">19</span>, <span class="cit-fpage">889</span>–<span class="cit-lpage">890</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft.stitle%253DBioinformatics%26rft.aulast%253DBroman%26rft.auinit1%253DK.%2BW.%26rft.volume%253D19%26rft.issue%253D7%26rft.spage%253D889%26rft.epage%253D890%26rft.atitle%253DR%252Fqtl%253A%2BQTL%2Bmapping%2Bin%2Bexperimental%2Bcrosses%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtg112%26rft_id%253Dinfo%253Apmid%252F12724300%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/btg112&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=12724300&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F05%2F13%2F635862.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000182634800014&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference  in text" id="ref-3">↵</a><div class="cit ref-cit ref-journal" id="cit-635862v1.3"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Berg</span>  <span class="cit-name-given-names">JJ</span></span>, <span class="cit-auth"><span class="cit-name-surname">Coop</span>  <span class="cit-name-given-names">G</span></span> (<span class="cit-pub-date">2014</span>) <span class="cit-article-title">A population genetic signal of polygenic adaptation</span>. <abbr class="cit-jnl-abbrev">Plos Genetics</abbr>, <span class="cit-vol">10</span>(<span class="cit-issue">8</span>), <span class="cit-fpage">e1004412</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlos%2BGenetics%26rft.volume%253D10%26rft.spage%253D1004412e%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference  in text" id="ref-4">↵</a><div class="cit ref-cit ref-journal" id="cit-635862v1.4"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Bernhardsson</span>  <span class="cit-name-given-names">C</span></span>, <span class="cit-auth"><span class="cit-name-surname">Vidalis</span>  <span class="cit-name-given-names">A</span></span>, <span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">X</span></span>, <span class="cit-auth"><span class="cit-name-surname">Scofield</span>  <span class="cit-name-given-names">DG</span></span>, <span class="cit-auth"><span class="cit-name-surname">Schiffthaler</span>  <span class="cit-name-given-names">B</span></span>, <span class="cit-auth"><span class="cit-name-surname">Baison</span>  <span class="cit-name-given-names">J</span></span>, <span class="cit-auth"><span class="cit-name-surname">Street</span>  <span class="cit-name-given-names">NR</span></span>, <span class="cit-auth"><span class="cit-name-surname">Garcia-Gil</span>  <span class="cit-name-given-names">MR</span></span>, <span class="cit-auth"><span class="cit-name-surname">Ingvarsson</span>  <span class="cit-name-given-names">PK</span></span> (<span class="cit-pub-date">2019</span>) <span class="cit-article-title">An ultra-dense haploid genetic map for evaluating the highly fragmented genome assembly of Norway spruce (<em>Picea abies</em>). Genes</span>, <abbr class="cit-jnl-abbrev">Genomes and Genetics</abbr> <a href="https://doi.org/10.1534/g3.118.200840">https://dpi.org/10.1534/g3.118.200840</a>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference  in text" id="ref-5">↵</a><div class="cit ref-cit ref-journal" id="cit-635862v1.5" data-doi="10.1098/rspb.1966.0038"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Cavalli-Sforza</span>  <span class="cit-name-given-names">LL</span></span> (<span class="cit-pub-date">1966</span>) <span class="cit-article-title">Population structure and human evolution</span>. <abbr class="cit-jnl-abbrev">Proceedings of the Royal Society B: Biological Sciences</abbr>, <span class="cit-vol">164</span>, <span class="cit-fpage">362</span>–<span class="cit-lpage">379</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProceedings%2Bof%2Bthe%2BRoyal%2BSociety%2BB%253A%2BBiological%2BSciences%26rft_id%253Dinfo%253Adoi%252F10.1098%252Frspb.1966.0038%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1098/rspb.1966.0038&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference  in text" id="ref-6">↵</a><div class="cit ref-cit ref-journal" id="cit-635862v1.6"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Csillery</span>  <span class="cit-name-given-names">K</span></span>, <span class="cit-auth"><span class="cit-name-surname">Lalague</span>  <span class="cit-name-given-names">H</span></span>, <span class="cit-auth"><span class="cit-name-surname">Vendramin</span>  <span class="cit-name-given-names">GG</span></span>, <span class="cit-auth"><span class="cit-name-surname">Gonzalez-Martinez</span>  <span class="cit-name-given-names">SC</span></span>, <span class="cit-auth"><span class="cit-name-surname">Fady</span>  <span class="cit-name-given-names">B</span></span>, <span class="cit-auth"><span class="cit-name-surname">Oddou-Muratorio</span>  <span class="cit-name-given-names">S</span></span> (<span class="cit-pub-date">2014</span>) <span class="cit-article-title">Detecting short spatial scale local adaptation and epistatic selection in climate-related candidate genes in European beech (<em>Fagus sylvatica</em>) populations</span>. <abbr class="cit-jnl-abbrev">Molecular Ecology</abbr>, <span class="cit-vol">23</span>, <span class="cit-fpage">4696</span>–<span class="cit-lpage">4708</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2BEcology%26rft.volume%253D23%26rft.spage%253D4696%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference  in text" id="ref-7">↵</a><div class="cit ref-cit ref-journal" id="cit-635862v1.7" data-doi="10.1104/pp.114.900492"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">De La Torre</span>  <span class="cit-name-given-names">AR</span></span>, <span class="cit-auth"><span class="cit-name-surname">Birol</span>  <span class="cit-name-given-names">I</span></span>, <span class="cit-auth"><span class="cit-name-surname">Bousquet</span>  <span class="cit-name-given-names">J</span></span>, <span class="cit-auth"><span class="cit-name-surname">Ingvarsson</span>  <span class="cit-name-given-names">PK</span></span>, <span class="cit-auth"><span class="cit-name-surname">Jansson</span>  <span class="cit-name-given-names">S</span></span>, <span class="cit-auth"><span class="cit-name-surname">Jones</span>  <span class="cit-name-given-names">SJM</span></span>, <span class="cit-auth"><span class="cit-name-surname">Keeling</span>  <span class="cit-name-given-names">CI</span></span>, <span class="cit-auth"><span class="cit-name-surname">MacKay</span>  <span class="cit-name-given-names">J</span></span>, <span class="cit-auth"><span class="cit-name-surname">Nilsson</span>  <span class="cit-name-given-names">O</span></span>, <span class="cit-auth"><span class="cit-name-surname">Ritland</span>  <span class="cit-name-given-names">K</span></span>, <span class="cit-auth"><span class="cit-name-surname">Street</span>  <span class="cit-name-given-names">N</span></span>, <span class="cit-auth"><span class="cit-name-surname">Yanchuk</span>  <span class="cit-name-given-names">A</span></span>, <span class="cit-auth"><span class="cit-name-surname">Zerbe</span>  <span class="cit-name-given-names">P</span></span>, <span class="cit-auth"><span class="cit-name-surname">Bohlmann</span>  <span class="cit-name-given-names">J</span></span> (<span class="cit-pub-date">2014a</span>) <span class="cit-article-title">Insights into Conifer Giga-genomes</span>. <abbr class="cit-jnl-abbrev">Plant Physiology</abbr>, <span class="cit-vol">166</span>, <span class="cit-fpage">1</span>–<span class="cit-lpage">9</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BPhysiology%26rft.stitle%253DPlant%2BPhysiol.%26rft.volume%253D166%26rft.issue%253D1%26rft.spage%253D1%26rft.epage%253D2%26rft.atitle%253DOn%2Bthe%2BInside%26rft_id%253Dinfo%253Adoi%252F10.1104%252Fpp.114.900492%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiRlVMTCI7czoxMToiam91cm5hbENvZGUiO3M6MTI6InBsYW50cGh5c2lvbCI7czo1OiJyZXNpZCI7czo3OiIxNjYvMS8xIjtzOjQ6ImF0b20iO3M6Mzc6Ii9iaW9yeGl2L2Vhcmx5LzIwMTkvMDUvMTMvNjM1ODYyLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference  in text" id="ref-8">↵</a><div class="cit ref-cit ref-journal" id="cit-635862v1.8" data-doi="10.1111/mec.12710"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">De La Torre</span>  <span class="cit-name-given-names">AR</span></span>, <span class="cit-auth"><span class="cit-name-surname">Roberts</span>  <span class="cit-name-given-names">DR</span></span>, <span class="cit-auth"><span class="cit-name-surname">Aitken</span>  <span class="cit-name-given-names">SN</span></span> (<span class="cit-pub-date">2014b</span>) <span class="cit-article-title">Genome-wide admixture and ecological niche modelling reveal the maintenance of species boundaries despite long history of interspecific gene flow</span>. <abbr class="cit-jnl-abbrev">Molecular Ecology</abbr>, <span class="cit-vol">23</span>, <span class="cit-fpage">2046</span>–<span class="cit-lpage">2059</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2BEcology%26rft.volume%253D23%26rft.spage%253D2046%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fmec.12710%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/mec.12710&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000333858200013&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference  in text" id="ref-9">↵</a><div class="cit ref-cit ref-journal" id="cit-635862v1.9" data-doi="10.1093/molbev/msx069"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">De La Torre</span>  <span class="cit-name-given-names">AR</span></span>, <span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">Z</span></span>, <span class="cit-auth"><span class="cit-name-surname">Van de Peer</span>  <span class="cit-name-given-names">Y</span></span>, <span class="cit-auth"><span class="cit-name-surname">Ingvarsson</span>  <span class="cit-name-given-names">PK</span></span> (<span class="cit-pub-date">2017</span>) <span class="cit-article-title">Contrasting rates of molecular evolution and patterns of selection among gymnosperms and flowering plants</span>. <abbr class="cit-jnl-abbrev">Molecular Biology and Evolution</abbr>, <span class="cit-vol">34</span>(<span class="cit-issue">6</span>), <span class="cit-fpage">1363</span>–<span class="cit-lpage">1377</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2BBiology%2Band%2BEvolution%26rft.volume%253D34%26rft.spage%253D1363%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fmolbev%252Fmsx069%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/molbev/msx069&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference  in text" id="ref-10">↵</a><div class="cit ref-cit ref-journal" id="cit-635862v1.10"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">De La Torre</span>  <span class="cit-name-given-names">AR</span></span>, <span class="cit-auth"><span class="cit-name-surname">Puiu</span>  <span class="cit-name-given-names">D</span></span>, <span class="cit-auth"><span class="cit-name-surname">Crepeau</span>  <span class="cit-name-given-names">MW</span></span>, <span class="cit-auth"><span class="cit-name-surname">Stevens</span>  <span class="cit-name-given-names">K</span></span>, <span class="cit-auth"><span class="cit-name-surname">Salzberg</span>  <span class="cit-name-given-names">SL</span></span>, <span class="cit-auth"><span class="cit-name-surname">Langley</span>  <span class="cit-name-given-names">CH</span></span>, <span class="cit-auth"><span class="cit-name-surname">Neale</span>  <span class="cit-name-given-names">DB</span></span> (<span class="cit-pub-date">2018</span>) <span class="cit-article-title">Genomic architecture of complex traits in loblolly pine</span>. <abbr class="cit-jnl-abbrev">New Phytologist</abbr>, <span class="cit-vol">221</span>:<span class="cit-fpage">1789</span>–<span class="cit-lpage">1801</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNew%2BPhytologist%26rft.volume%253D221%26rft.spage%253D1789%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference  in text" id="ref-11">↵</a><div class="cit ref-cit ref-journal" id="cit-635862v1.11"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Eckert</span>  <span class="cit-name-given-names">AJ</span></span>, <span class="cit-auth"><span class="cit-name-surname">Bower</span>  <span class="cit-name-given-names">AD</span></span>, <span class="cit-auth"><span class="cit-name-surname">Gonzalez-Martinez</span>  <span class="cit-name-given-names">SC</span></span>, <span class="cit-auth"><span class="cit-name-surname">Wegrzyn</span>  <span class="cit-name-given-names">JL</span></span>, <span class="cit-auth"><span class="cit-name-surname">Coop</span>  <span class="cit-name-given-names">G</span></span>, <span class="cit-auth"><span class="cit-name-surname">Neale</span>  <span class="cit-name-given-names">DB</span></span> (<span class="cit-pub-date">2010a</span>) <span class="cit-article-title">Back to nature: ecological genomics of loblolly pine (<em>Pinus taeda</em></span>, <abbr class="cit-jnl-abbrev">Pinaceae). Molecular Ecology</abbr>, <span class="cit-vol">19</span>, <span class="cit-fpage">3789</span>–<span class="cit-lpage">3805</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPinaceae%2529.%2BMolecular%2BEcology%26rft.volume%253D19%26rft.spage%253D3789%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference  in text" id="ref-12">↵</a><div class="cit ref-cit ref-journal" id="cit-635862v1.12"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Eckert</span>  <span class="cit-name-given-names">AJ</span></span>, <span class="cit-auth"><span class="cit-name-surname">van Heerwaarden</span>  <span class="cit-name-given-names">J</span></span>, <span class="cit-auth"><span class="cit-name-surname">Wegrzyn</span>  <span class="cit-name-given-names">JL</span></span>, <span class="cit-auth"><span class="cit-name-surname">Nelson</span>  <span class="cit-name-given-names">CD</span></span>, <span class="cit-auth"><span class="cit-name-surname">Ross-Ibarra</span>  <span class="cit-name-given-names">J</span></span>, <span class="cit-auth"><span class="cit-name-surname">Gonzalez-Martinez</span>  <span class="cit-name-given-names">SC</span></span>, <span class="cit-auth"><span class="cit-name-surname">Neale</span>  <span class="cit-name-given-names">DB</span></span> (<span class="cit-pub-date">2010b</span>) <span class="cit-article-title">Patterns of population structure and environmental association to aridity across the range of loblolly pine (<em>Pinus taeda</em> L, Pinaceae)</span>. <abbr class="cit-jnl-abbrev">Genetics</abbr>, <span class="cit-vol">185</span>, <span class="cit-fpage">962</span>–<span class="cit-lpage">982</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.volume%253D185%26rft.spage%253D962%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference  in text" id="ref-13">↵</a><div class="cit ref-cit ref-journal" id="cit-635862v1.13" data-doi="10.1093/bioinformatics/btu091"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Endelman</span>  <span class="cit-name-given-names">JB</span></span>, <span class="cit-auth"><span class="cit-name-surname">Plomion</span>  <span class="cit-name-given-names">C</span></span> (<span class="cit-pub-date">2014</span>) <span class="cit-article-title">LPmerge: An R package for merging genetic maps by linear programming</span>. <abbr class="cit-jnl-abbrev">Bioinformatics</abbr>, <span class="cit-vol">30</span>(<span class="cit-issue">11</span>), <span class="cit-fpage">1623</span>–<span class="cit-lpage">1624</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtu091%26rft_id%253Dinfo%253Apmid%252F24532720%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/btu091&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24532720&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F05%2F13%2F635862.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference  in text" id="ref-14">↵</a><div class="cit ref-cit ref-journal" id="cit-635862v1.14" data-doi="10.1126/science.1209271"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Fournier-Level</span>  <span class="cit-name-given-names">A</span></span>, <span class="cit-auth"><span class="cit-name-surname">Korte</span>  <span class="cit-name-given-names">A</span></span>, <span class="cit-auth"><span class="cit-name-surname">Cooper</span>  <span class="cit-name-given-names">MD</span></span>, <span class="cit-auth"><span class="cit-name-surname">Nordborg</span>  <span class="cit-name-given-names">M</span></span>, <span class="cit-auth"><span class="cit-name-surname">Schmitt</span>  <span class="cit-name-given-names">J</span></span>, <span class="cit-auth"><span class="cit-name-surname">Wilczek</span>  <span class="cit-name-given-names">AM</span></span> (<span class="cit-pub-date">2011</span>) <span class="cit-article-title">A Map of Local Adaptation in <em>Arabidopsis thaliana</em></span>. <abbr class="cit-jnl-abbrev">Science</abbr>, <span class="cit-vol">334</span>, <span class="cit-fpage">86</span>–<span class="cit-lpage">89</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DFournier-Level%26rft.auinit1%253DA.%26rft.volume%253D334%26rft.issue%253D6052%26rft.spage%253D86%26rft.epage%253D89%26rft.atitle%253DA%2BMap%2Bof%2BLocal%2BAdaptation%2Bin%2BArabidopsis%2Bthaliana%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1209271%26rft_id%253Dinfo%253Apmid%252F21980109%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjExOiIzMzQvNjA1Mi84NiI7czo0OiJhdG9tIjtzOjM3OiIvYmlvcnhpdi9lYXJseS8yMDE5LzA1LzEzLzYzNTg2Mi5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference  in text" id="ref-15">↵</a><div class="cit ref-cit ref-journal" id="cit-635862v1.15" data-doi="10.1111 /mec. 14584"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Forester</span>  <span class="cit-name-given-names">BR</span></span>, <span class="cit-auth"><span class="cit-name-surname">Lasky</span>  <span class="cit-name-given-names">JR</span></span>, <span class="cit-auth"><span class="cit-name-surname">Wagner</span>  <span class="cit-name-given-names">HH</span></span>, <span class="cit-auth"><span class="cit-name-surname">Urban</span>  <span class="cit-name-given-names">DL</span></span> (<span class="cit-pub-date">2018</span>). <span class="cit-article-title">Comparing methods for detecting multilocus adaptation with multivariate genotype-environment associations</span>. <abbr class="cit-jnl-abbrev">Molecular Ecology</abbr>, <span class="cit-vol">27</span>, <span class="cit-fpage">2215</span>–<span class="cit-lpage">2233</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2BEcology%26rft.volume%253D27%26rft.spage%253D2215%26rft_id%253Dinfo%253Adoi%252F10.1111%2B%252Fmec.%2B14584%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111 /mec. 14584&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference  in text" id="ref-16">↵</a><div class="cit ref-cit ref-journal" id="cit-635862v1.16" data-doi="10.1093/molbev/mst063"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Frichot</span>  <span class="cit-name-given-names">E</span></span>, <span class="cit-auth"><span class="cit-name-surname">Schoville</span>  <span class="cit-name-given-names">SD</span></span>, <span class="cit-auth"><span class="cit-name-surname">Bouchard</span>  <span class="cit-name-given-names">G</span></span>, <span class="cit-auth"><span class="cit-name-surname">Francois</span>  <span class="cit-name-given-names">O</span></span> (<span class="cit-pub-date">2013</span>) <span class="cit-article-title">Testing for Associations between loci and environmental gradients using latent factor mixed models</span>. <abbr class="cit-jnl-abbrev">Molecular Biology and Evolution</abbr>, <span class="cit-vol">30</span>(<span class="cit-issue">7</span>), <span class="cit-fpage">1687</span>–<span class="cit-lpage">1699</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2BBiology%2Band%2BEvolution%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fmolbev%252Fmst063%26rft_id%253Dinfo%253Apmid%252F23543094%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/molbev/mst063&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23543094&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F05%2F13%2F635862.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000321056200018&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference  in text" id="ref-17">↵</a><div class="cit ref-cit ref-journal" id="cit-635862v1.17"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Frichot</span>  <span class="cit-name-given-names">E</span></span>, <span class="cit-auth"><span class="cit-name-surname">Francois</span>  <span class="cit-name-given-names">O</span></span> (<span class="cit-pub-date">2015</span>) <span class="cit-article-title">LEA: An R package for landscape and ecological association studies</span>. <abbr class="cit-jnl-abbrev">Methods in Ecology and Evolution</abbr>, <span class="cit-vol">6</span>, <span class="cit-fpage">925</span>–<span class="cit-lpage">929</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMethods%2Bin%2BEcology%2Band%2BEvolution%26rft.volume%253D6%26rft.spage%253D925%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference  in text" id="ref-18">↵</a><div class="cit ref-cit ref-journal" id="cit-635862v1.18" data-doi="10.1534/genetics.105.047126"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Gonzalez-Martinez</span>  <span class="cit-name-given-names">SC</span></span>, <span class="cit-auth"><span class="cit-name-surname">Ersoz</span>  <span class="cit-name-given-names">E</span></span>, <span class="cit-auth"><span class="cit-name-surname">Brown</span>  <span class="cit-name-given-names">GR</span></span>, <span class="cit-auth"><span class="cit-name-surname">Wheeler</span>  <span class="cit-name-given-names">NC</span></span>, <span class="cit-auth"><span class="cit-name-surname">Neale</span>  <span class="cit-name-given-names">DB</span></span> (<span class="cit-pub-date">2006</span>) <span class="cit-article-title">DNA sequence variation and selection of tag SNP at candidate genes for drought-stress response in <em>Pinus taeda</em> L</span>. <abbr class="cit-jnl-abbrev">Genetics</abbr>, <span class="cit-vol">172</span>,<span class="cit-fpage">1915</span>–<span class="cit-lpage">1926</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.105.047126%26rft_id%253Dinfo%253Apmid%252F16387885%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE3Mi8zLzE5MTUiO3M6NDoiYXRvbSI7czozNzoiL2Jpb3J4aXYvZWFybHkvMjAxOS8wNS8xMy82MzU4NjIuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference  in text" id="ref-19">↵</a><div class="cit ref-cit ref-journal" id="cit-635862v1.19" data-doi="10.1098/rstb.2010.0032"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Hancock</span>  <span class="cit-name-given-names">AM</span></span>, <span class="cit-auth"><span class="cit-name-surname">Alkorta-Aranburu</span>  <span class="cit-name-given-names">G</span></span>, <span class="cit-auth"><span class="cit-name-surname">Witonsky</span>  <span class="cit-name-given-names">DB</span></span>, <span class="cit-auth"><span class="cit-name-surname">Di Rienzo</span>  <span class="cit-name-given-names">A</span></span> (<span class="cit-pub-date">2010</span>) <span class="cit-article-title">Adaptations to new environments in humans: the role of subtle allele frequency shifts</span>. <abbr class="cit-jnl-abbrev">Phil trans R Soc B</abbr>, <span class="cit-vol">365</span>, <span class="cit-fpage">2459</span>–<span class="cit-lpage">2468</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPhilosophical%2BTransactions%2Bof%2Bthe%2BRoyal%2BSociety%2BB%253A%2BBiological%2BSciences%26rft.stitle%253DPhil%2BTrans%2BR%2BSoc%2BB%26rft.aulast%253DHancock%26rft.auinit1%253DA.%2BM.%26rft.volume%253D365%26rft.issue%253D1552%26rft.spage%253D2459%26rft.epage%253D2468%26rft.atitle%253DAdaptations%2Bto%2Bnew%2Benvironments%2Bin%2Bhumans%253A%2Bthe%2Brole%2Bof%2Bsubtle%2Ballele%2Bfrequency%2Bshifts%26rft_id%253Dinfo%253Adoi%252F10.1098%252Frstb.2010.0032%26rft_id%253Dinfo%253Apmid%252F20643735%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1098/rstb.2010.0032&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20643735&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F05%2F13%2F635862.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference  in text" id="ref-20">↵</a><div class="cit ref-cit ref-journal" id="cit-635862v1.20" data-doi="10.1126/science.1209244"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Hancock</span>  <span class="cit-name-given-names">AM</span></span>, <span class="cit-auth"><span class="cit-name-surname">Brachi</span>  <span class="cit-name-given-names">B</span></span>, <span class="cit-auth"><span class="cit-name-surname">Faure</span>  <span class="cit-name-given-names">N</span></span>, <span class="cit-auth"><span class="cit-name-surname">Horton</span>  <span class="cit-name-given-names">MW</span></span>, <span class="cit-auth"><span class="cit-name-surname">Jarymowycz</span>  <span class="cit-name-given-names">LB</span></span>, <span class="cit-auth"><span class="cit-name-surname">Sperone</span>  <span class="cit-name-given-names">FG</span></span>, <span class="cit-auth"><span class="cit-name-surname">Toomajian</span>,  <span class="cit-name-given-names">Roux F</span></span>, <span class="cit-auth"><span class="cit-name-surname">Bergelson</span>  <span class="cit-name-given-names">J</span></span> (<span class="cit-pub-date">2011</span>) <span class="cit-article-title">Adaptation to climate across the <em>Arabidopsis thaliana</em> genome</span>. <abbr class="cit-jnl-abbrev">Science</abbr>, <span class="cit-vol">334</span>(<span class="cit-issue">7</span>), <span class="cit-fpage">83</span>–<span class="cit-lpage">86</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DHancock%26rft.auinit1%253DA.%2BM.%26rft.volume%253D334%26rft.issue%253D6052%26rft.spage%253D83%26rft.epage%253D86%26rft.atitle%253DAdaptation%2Bto%2BClimate%2BAcross%2Bthe%2BArabidopsis%2Bthaliana%2BGenome%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1209244%26rft_id%253Dinfo%253Apmid%252F21980108%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjExOiIzMzQvNjA1Mi84MyI7czo0OiJhdG9tIjtzOjM3OiIvYmlvcnhpdi9lYXJseS8yMDE5LzA1LzEzLzYzNTg2Mi5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-635862v1.21" data-doi="10.1111/mec.14368"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Harrisson</span>  <span class="cit-name-given-names">KA</span></span>, <span class="cit-auth"><span class="cit-name-surname">Amish</span>  <span class="cit-name-given-names">SJ</span></span>, <span class="cit-auth"><span class="cit-name-surname">Pavlova</span>  <span class="cit-name-given-names">A</span></span>, <span class="cit-auth"><span class="cit-name-surname">Narum</span>  <span class="cit-name-given-names">SR</span></span>, <span class="cit-auth"><span class="cit-name-surname">Telonis-Scott</span>  <span class="cit-name-given-names">M</span></span>, <span class="cit-auth"><span class="cit-name-surname">Rourke</span>  <span class="cit-name-given-names">ML</span></span>, <span class="cit-auth"><span class="cit-name-surname">Lyon</span>  <span class="cit-name-given-names">J</span></span>, <span class="cit-auth"><span class="cit-name-surname">Tonkin</span>  <span class="cit-name-given-names">Z</span></span>, <span class="cit-auth"><span class="cit-name-surname">Gilligan</span>  <span class="cit-name-given-names">DM</span></span>, <span class="cit-auth"><span class="cit-name-surname">Ingram</span>  <span class="cit-name-given-names">BA</span></span>, <span class="cit-auth"><span class="cit-name-surname">Lintermans</span>  <span class="cit-name-given-names">M</span></span>, <span class="cit-auth"><span class="cit-name-surname">Gan</span>  <span class="cit-name-given-names">HM</span></span>, <span class="cit-auth"><span class="cit-name-surname">Austin</span>  <span class="cit-name-given-names">CM</span></span>, <span class="cit-auth"><span class="cit-name-surname">Luikart</span>  <span class="cit-name-given-names">G</span></span>, <span class="cit-auth"><span class="cit-name-surname">Sunnucks</span>  <span class="cit-name-given-names">P</span></span> (<span class="cit-pub-date">2017</span>) <span class="cit-article-title">Signatures of polygenic adaptation associated with climate across the range of a threatened fish species with high connectivity</span>. <abbr class="cit-jnl-abbrev">Molecular Ecology</abbr>, <span class="cit-vol">26</span>, <span class="cit-fpage">6253</span>–<span class="cit-lpage">6269</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2BEcology%26rft.volume%253D26%26rft.spage%253D6253%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fmec.14368%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/mec.14368&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference  in text" id="ref-22">↵</a><div class="cit ref-cit ref-journal" id="cit-635862v1.22" data-doi="10.1534/genetics.104.036947"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Hermisson</span>  <span class="cit-name-given-names">J</span></span>, <span class="cit-auth"><span class="cit-name-surname">Pennings</span>  <span class="cit-name-given-names">PS</span></span> (<span class="cit-pub-date">2005</span>) <span class="cit-article-title">Soft sweeps - molecular population genetics of adaptation from standing genetic variation</span>. <abbr class="cit-jnl-abbrev">Genetics</abbr>, <span class="cit-vol">169</span>, <span class="cit-fpage">2335</span>–<span class="cit-lpage">2352</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.104.036947%26rft_id%253Dinfo%253Apmid%252F15716498%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE2OS80LzIzMzUiO3M6NDoiYXRvbSI7czozNzoiL2Jpb3J4aXYvZWFybHkvMjAxOS8wNS8xMy82MzU4NjIuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference  in text" id="ref-23">↵</a><div class="cit ref-cit ref-journal" id="cit-635862v1.23" data-doi="10.1086/656306"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Hohenlohe</span>  <span class="cit-name-given-names">PA</span></span>, <span class="cit-auth"><span class="cit-name-surname">Phillips</span>  <span class="cit-name-given-names">PC</span></span>, <span class="cit-auth"><span class="cit-name-surname">Cresko</span>  <span class="cit-name-given-names">W</span></span> (<span class="cit-pub-date">2010</span>) <span class="cit-article-title">Using population genomics to detect selection in natural populations: key concepts and methodological considerations</span>. <abbr class="cit-jnl-abbrev">International Journal of Plant Science</abbr>, <span class="cit-vol">171</span>, <span class="cit-fpage">1059</span>–<span class="cit-lpage">1071</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DInternational%2BJournal%2Bof%2BPlant%2BSciences%26rft.stitle%253DInternational%2BJournal%2Bof%2BPlant%2BSciences%26rft.aulast%253DHohenlohe%26rft.auinit1%253DP.%2BA.%26rft.volume%253D171%26rft.issue%253D9%26rft.spage%253D1059%26rft.epage%253D1071%26rft.atitle%253DUSING%2BPOPULATION%2BGENOMICS%2BTO%2BDETECT%2BSELECTION%2BIN%2BNATURAL%2BPOPULATIONS%253A%2BKEY%2BCONCEPTS%2BAND%2BMETHODOLOGICAL%2BCONSIDERATIONS.%26rft_id%253Dinfo%253Adoi%252F10.1086%252F656306%26rft_id%253Dinfo%253Apmid%252F21218185%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1086/656306&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21218185&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F05%2F13%2F635862.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000284000800012&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference  in text" id="ref-24">↵</a><div class="cit ref-cit ref-journal" id="cit-635862v1.24" data-doi="10.1111/nph.13643"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Holliday</span>  <span class="cit-name-given-names">JA</span></span>, <span class="cit-auth"><span class="cit-name-surname">Zhou</span>  <span class="cit-name-given-names">L</span></span>, <span class="cit-auth"><span class="cit-name-surname">Bawa</span>  <span class="cit-name-given-names">R</span></span>, <span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">M</span></span>, <span class="cit-auth"><span class="cit-name-surname">Oubida</span>  <span class="cit-name-given-names">RW</span></span> (<span class="cit-pub-date">2016</span>) <span class="cit-article-title">Evidence for extensive parallelism but divergent genomic architecture of adaptation along altitudinal and latitudinal gradients in <em>Populus trichocarpa</em></span>. <abbr class="cit-jnl-abbrev">New Phytologist</abbr>, <span class="cit-vol">209</span>, <span class="cit-fpage">1240</span>–<span class="cit-lpage">1251</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNew%2BPhytologist%26rft.volume%253D209%26rft.spage%253D1240%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fnph.13643%26rft_id%253Dinfo%253Apmid%252F26372471%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/nph.13643&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=26372471&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F05%2F13%2F635862.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference  in text" id="ref-25">↵</a><div class="cit ref-cit ref-journal" id="cit-635862v1.25" data-doi="10.1093/gbe/evv218"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Hornoy</span>  <span class="cit-name-given-names">B</span></span>, <span class="cit-auth"><span class="cit-name-surname">Pavy</span>  <span class="cit-name-given-names">N</span></span>, <span class="cit-auth"><span class="cit-name-surname">Gerardi</span>  <span class="cit-name-given-names">S</span></span>, <span class="cit-auth"><span class="cit-name-surname">Beaulieu</span>  <span class="cit-name-given-names">J</span></span>, <span class="cit-auth"><span class="cit-name-surname">Bousquet</span>  <span class="cit-name-given-names">J</span></span> (<span class="cit-pub-date">2015</span>) <span class="cit-article-title">Genetic adaptation to climate in white spruce involves small to moderate allele frequency shifts in functionally diverse genes</span>. <abbr class="cit-jnl-abbrev">Genome Biology and Evolution</abbr>, <span class="cit-vol">7</span>(<span class="cit-issue">12</span>), <span class="cit-fpage">3269</span>–<span class="cit-lpage">3285</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BBiology%2Band%2BEvolution%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fgbe%252Fevv218%26rft_id%253Dinfo%253Apmid%252F26560341%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/gbe/evv218&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=26560341&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F05%2F13%2F635862.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference  in text" id="ref-26">↵</a><div class="cit ref-cit ref-journal" id="cit-635862v1.26" data-doi="10.1093/bioinformatics/btn129"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Jombart</span>,  <span class="cit-name-given-names">T</span></span> (<span class="cit-pub-date">2008</span>) <span class="cit-article-title">Adegenet: An R package for the multivariate analysis of genetic markers</span>. <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">24</span>, <span class="cit-fpage">1403</span>–<span class="cit-lpage">1405</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtn129%26rft_id%253Dinfo%253Apmid%252F18397895%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/btn129&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18397895&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F05%2F13%2F635862.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000256169300015&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-27-1" title="View reference  in text" id="ref-27">↵</a><div class="cit ref-cit ref-journal" id="cit-635862v1.27"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Jombart</span>  <span class="cit-name-given-names">T.</span></span> and <span class="cit-auth"><span class="cit-name-surname">Ahmed</span>  <span class="cit-name-given-names">I</span></span> (<span class="cit-pub-date">2011</span>) <span class="cit-article-title">Adegenet 1.3-1: new tools for the analysis of genome-wide SNP data</span>. <abbr class="cit-jnl-abbrev">Bioinformatics</abbr>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-28-1" title="View reference  in text" id="ref-28">↵</a><div class="cit ref-cit ref-journal" id="cit-635862v1.28"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Kopelman</span>  <span class="cit-name-given-names">NM</span></span>, <span class="cit-auth"><span class="cit-name-surname">Mayzel</span>  <span class="cit-name-given-names">J</span></span>, <span class="cit-auth"><span class="cit-name-surname">Jakobsson</span>  <span class="cit-name-given-names">M</span></span>, <span class="cit-auth"><span class="cit-name-surname">Rosenberg</span>  <span class="cit-name-given-names">NA</span></span>, <span class="cit-auth"><span class="cit-name-surname">Mayrose</span>  <span class="cit-name-given-names">I</span></span> (<span class="cit-pub-date">2015</span>) <span class="cit-article-title">CLUMPAK: a program for identifying clustering modes and packaging population structure inferences across K</span>. <abbr class="cit-jnl-abbrev">Molecular Ecology Resources</abbr>, <span class="cit-vol">15</span>(<span class="cit-issue">5</span>), <span class="cit-fpage">1179</span>–<span class="cit-lpage">1191</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2BEcology%2BResources%26rft.volume%253D15%26rft.spage%253D1179%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-29-1" title="View reference  in text" id="ref-29">↵</a><div class="cit ref-cit ref-journal" id="cit-635862v1.29" data-doi="10.1101/gr.092759.109"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Krzywinski</span>  <span class="cit-name-given-names">MI</span></span>, <span class="cit-auth"><span class="cit-name-surname">Schein</span>  <span class="cit-name-given-names">JE</span></span>, <span class="cit-auth"><span class="cit-name-surname">Birol</span>  <span class="cit-name-given-names">I</span></span>, <span class="cit-auth"><span class="cit-name-surname">Connors</span>  <span class="cit-name-given-names">J</span></span>, <span class="cit-auth"><span class="cit-name-surname">Gascoyne</span>  <span class="cit-name-given-names">R</span></span>, <span class="cit-auth"><span class="cit-name-surname">Horsman</span>  <span class="cit-name-given-names">D</span></span>, <span class="cit-auth"><span class="cit-name-surname">Jones</span>  <span class="cit-name-given-names">SJ</span></span>, <span class="cit-auth"><span class="cit-name-surname">Marra</span>  <span class="cit-name-given-names">MA</span></span> (<span class="cit-pub-date">2009</span>) <span class="cit-article-title">Circos: An information aesthetic for comparative genomics</span>. <abbr class="cit-jnl-abbrev">Genome Research</abbr><span class="cit-pub-id-sep cit-pub-id-doi-sep">, </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1101/gr.092759.109</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BResearch%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.092759.109%26rft_id%253Dinfo%253Apmid%252F19541911%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjk6IjE5LzkvMTYzOSI7czo0OiJhdG9tIjtzOjM3OiIvYmlvcnhpdi9lYXJseS8yMDE5LzA1LzEzLzYzNTg2Mi5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-30-1" title="View reference  in text" id="ref-30">↵</a><div class="cit ref-cit ref-journal" id="cit-635862v1.30" data-doi="10.1086/286119"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Latta</span>  <span class="cit-name-given-names">RG</span></span> (<span class="cit-pub-date">1998</span>) <span class="cit-article-title">Differentiation of allelic frequencies at quantitative trait loci affecting locally adaptive traits</span>. <abbr class="cit-jnl-abbrev">The American Naturalist</abbr>, <span class="cit-vol">151</span>, <span class="cit-fpage">283</span>–<span class="cit-lpage">292</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DThe%2BAmerican%2BNaturalist%26rft.volume%253D151%26rft.spage%253D283%26rft_id%253Dinfo%253Adoi%252F10.1086%252F286119%26rft_id%253Dinfo%253Apmid%252F18811359%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1086/286119&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18811359&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F05%2F13%2F635862.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000072128600008&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-31-1" title="View reference  in text" id="ref-31">↵</a><div class="cit ref-cit ref-journal" id="cit-635862v1.31"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">Le Corre V</span>, <span class="cit-name-surname">Kremer</span></span> (<span class="cit-pub-date">2003</span>) <span class="cit-article-title">Genetic variability at neutral markers, quantitative trait loci, and trait in a subdivided population under selection</span>. <abbr class="cit-jnl-abbrev">Genetics</abbr>, <span class="cit-vol">164</span>, <span class="cit-fpage">1205</span>–<span class="cit-lpage">1219</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DLe%2BCorre%26rft.auinit1%253DV.%26rft.volume%253D164%26rft.issue%253D3%26rft.spage%253D1205%26rft.epage%253D1219%26rft.atitle%253DGenetic%2BVariability%2Bat%2BNeutral%2BMarkers%252C%2BQuantitative%2BTrait%2BLoci%2Band%2BTrait%2Bin%2Ba%2BSubdivided%2BPopulation%2BUnder%2BSelection%26rft_id%253Dinfo%253Apmid%252F12871925%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE2NC8zLzEyMDUiO3M6NDoiYXRvbSI7czozNzoiL2Jpb3J4aXYvZWFybHkvMjAxOS8wNS8xMy82MzU4NjIuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-32-1" title="View reference  in text" id="ref-32">↵</a><div class="cit ref-cit ref-journal" id="cit-635862v1.32"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Le Corre</span>  <span class="cit-name-given-names">V</span></span>, <span class="cit-auth"><span class="cit-name-surname">Kremer</span>  <span class="cit-name-given-names">A</span></span> (<span class="cit-pub-date">2012</span>) <span class="cit-article-title">The genetic differentiation at quantitative trait loci under local adaptation</span>. <abbr class="cit-jnl-abbrev">Molecular Ecology</abbr>, <span class="cit-vol">12</span>, <span class="cit-fpage">1548</span>–<span class="cit-lpage">1566</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2BEcology%26rft.volume%253D12%26rft.spage%253D1548%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-33-1" title="View reference  in text" id="ref-33">↵</a><div class="cit ref-cit ref-journal" id="cit-635862v1.33" data-doi="10.1093/bioinformatics/bts444"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Lipka</span>  <span class="cit-name-given-names">AE</span></span>, <span class="cit-auth"><span class="cit-name-surname">Tian</span>  <span class="cit-name-given-names">F</span></span>, <span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">Q</span></span>, <span class="cit-auth"><span class="cit-name-surname">Peiffer</span>  <span class="cit-name-given-names">J</span></span>, <span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">M</span></span>, <span class="cit-auth"><span class="cit-name-surname">Bradbury</span>  <span class="cit-name-given-names">PJ</span></span>, <span class="cit-auth"><span class="cit-name-surname">Gore</span>  <span class="cit-name-given-names">MA</span></span>, <span class="cit-auth"><span class="cit-name-surname">Buckler</span>  <span class="cit-name-given-names">ES</span></span>, <span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">Z</span></span> (<span class="cit-pub-date">2012</span>) <span class="cit-article-title">GAPIT: genome association and prediction integrated tool</span>. <abbr class="cit-jnl-abbrev">Bioinformatics</abbr>, <span class="cit-vol">28</span>(<span class="cit-issue">18</span>), <span class="cit-fpage">2397</span>–<span class="cit-lpage">2399</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbts444%26rft_id%253Dinfo%253Apmid%252F22796960%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/bts444&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22796960&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F05%2F13%2F635862.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000308532300066&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-34-1" title="View reference  in text" id="ref-34">↵</a><div class="cit ref-cit ref-journal" id="cit-635862v1.34"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Liu</span>  <span class="cit-name-given-names">H</span></span>, <span class="cit-auth"><span class="cit-name-surname">Cao</span>  <span class="cit-name-given-names">F</span></span>, <span class="cit-auth"><span class="cit-name-surname">Yin</span>  <span class="cit-name-given-names">T</span></span>, <span class="cit-auth"><span class="cit-name-surname">Chen</span>  <span class="cit-name-given-names">Y</span></span> (<span class="cit-pub-date">2017</span>) <span class="cit-article-title">A highly dense genetic map for <em>Ginkgo biloba</em> constructed using sequence-based markers</span>. <abbr class="cit-jnl-abbrev">Frontiers in Plant Science</abbr>, <span class="cit-vol">8</span>, <span class="cit-fpage">1041</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DFrontiers%2Bin%2BPlant%2BScience%26rft.volume%253D8%26rft.spage%253D1041%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-35-1" title="View reference  in text" id="ref-35">↵</a><div class="cit ref-cit ref-journal" id="cit-635862v1.35"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Lotterhos</span>  <span class="cit-name-given-names">KE</span></span>, <span class="cit-auth"><span class="cit-name-surname">Yeaman</span>  <span class="cit-name-given-names">S</span></span>, <span class="cit-auth"><span class="cit-name-surname">Degner</span>  <span class="cit-name-given-names">J</span></span>, <span class="cit-auth"><span class="cit-name-surname">Aitken</span>  <span class="cit-name-given-names">S</span></span>, <span class="cit-auth"><span class="cit-name-surname">Hodgins</span>  <span class="cit-name-given-names">KA</span></span> (<span class="cit-pub-date">2018</span>) <span class="cit-article-title">Modularity of genes involved in local adaptation to climate despite physical linkage</span>. <abbr class="cit-jnl-abbrev">Genome Biology</abbr>, <span class="cit-vol">19</span>,<span class="cit-fpage">157</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BBiology%26rft.volume%253D19%26rft.spage%253D157%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-36-1" title="View reference  in text" id="ref-36">↵</a><div class="cit ref-cit ref-journal" id="cit-635862v1.36"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Lu</span>  <span class="cit-name-given-names">M</span></span>, <span class="cit-auth"><span class="cit-name-surname">Krutovsky</span>  <span class="cit-name-given-names">KV</span></span>, <span class="cit-auth"><span class="cit-name-surname">Nelson</span>  <span class="cit-name-given-names">CD</span></span>, <span class="cit-auth"><span class="cit-name-surname">West</span>  <span class="cit-name-given-names">JB</span></span>, <span class="cit-auth"><span class="cit-name-surname">Reilly</span>  <span class="cit-name-given-names">NA</span></span>, <span class="cit-auth"><span class="cit-name-surname">Loopstra</span>  <span class="cit-name-given-names">C</span></span> (<span class="cit-pub-date">2017</span>) <span class="cit-article-title">Association genetics of growth and adaptive traits in loblolly pine (<em>Pinus taeda</em> L</span>.) <abbr class="cit-jnl-abbrev">using whole-exome-discovered polymorphisms. Tree Genetics &amp; Genomes</abbr>, <span class="cit-vol">13</span>, <span class="cit-fpage">57</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253Dusing%2Bwhole-exome-discovered%2Bpolymorphisms.%2BTree%2BGenetics%2B%2526%2BGenomes%26rft.volume%253D13%26rft.spage%253D57%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-37-1" title="View reference  in text" id="ref-37">↵</a><div class="cit ref-cit ref-journal" id="cit-635862v1.37"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Martinez-Garcia</span>  <span class="cit-name-given-names">PJ</span></span>, <span class="cit-auth"><span class="cit-name-surname">Stevens</span>  <span class="cit-name-given-names">KA</span></span>, <span class="cit-auth"><span class="cit-name-surname">Wegrzyn</span>  <span class="cit-name-given-names">JL</span></span>, <span class="cit-auth"><span class="cit-name-surname">Liechty</span>  <span class="cit-name-given-names">J</span></span>, <span class="cit-auth"><span class="cit-name-surname">Crepeau</span>  <span class="cit-name-given-names">M</span></span>, <span class="cit-auth"><span class="cit-name-surname">Langley</span>  <span class="cit-name-given-names">CH</span></span>, <span class="cit-auth"><span class="cit-name-surname">Neale</span>  <span class="cit-name-given-names">DB</span></span> (<span class="cit-pub-date">2013</span>) <span class="cit-article-title">Combination of multipoint maximum likelihood (MML) and regression mapping algorithms to construct a high-density genetic linkage map for loblolly pine (<em>Pinus taeda</em> L</span>.). <abbr class="cit-jnl-abbrev">Tree Genetics &amp; Genomes</abbr>, <span class="cit-vol">9</span>, <span class="cit-fpage">1529</span>–<span class="cit-lpage">1535</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTree%2BGenetics%2B%2526%2BGenomes%26rft.volume%253D9%26rft.spage%253D1529%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-38-1" title="View reference  in text" id="ref-38">↵</a><div class="cit ref-cit ref-journal" id="cit-635862v1.38"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Neale</span>  <span class="cit-name-given-names">DB</span></span>, <span class="cit-auth"><span class="cit-name-surname">Martinez-Garcia</span>  <span class="cit-name-given-names">PJ</span></span>, <span class="cit-auth"><span class="cit-name-surname">De La Torre</span>  <span class="cit-name-given-names">AR</span></span>, <span class="cit-auth"><span class="cit-name-surname">Montanari</span>  <span class="cit-name-given-names">S</span></span>, <span class="cit-auth"><span class="cit-name-surname">Wei</span>  <span class="cit-name-given-names">X</span></span> (<span class="cit-pub-date">2017</span>) <span class="cit-article-title">Novel insights into tree biology and genome evolution as revealed through genomics</span>. <abbr class="cit-jnl-abbrev">Annual Review of Plant Biology</abbr>, <span class="cit-vol">68</span>, <span class="cit-fpage">13.1</span>–<span class="cit-lpage">13.27</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAnnual%2BReview%2Bof%2BPlant%2BBiology%26rft.volume%253D68%26rft.spage%253D13.1%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-39-1" title="View reference  in text" id="ref-39">↵</a><div class="cit ref-cit ref-journal" id="cit-635862v1.39" data-doi="10.1038/nrg2931"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Neale</span>  <span class="cit-name-given-names">DB</span></span>, <span class="cit-auth"><span class="cit-name-surname">Kremer</span>  <span class="cit-name-given-names">A</span></span> (<span class="cit-pub-date">2011</span>) <span class="cit-article-title">Forest tree genomics: growing resources and applications</span>. <abbr class="cit-jnl-abbrev">Nature Review Genetics</abbr>, <span class="cit-vol">12</span>, <span class="cit-fpage">111</span>–<span class="cit-lpage">121</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Breviews.%2BGenetics%26rft.stitle%253DNat%2BRev%2BGenet%26rft.aulast%253DNeale%26rft.auinit1%253DD.%2BB.%26rft.volume%253D12%26rft.issue%253D2%26rft.spage%253D111%26rft.epage%253D122%26rft.atitle%253DForest%2Btree%2Bgenomics%253A%2Bgrowing%2Bresources%2Band%2Bapplications.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnrg2931%26rft_id%253Dinfo%253Apmid%252F21245829%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nrg2931&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21245829&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F05%2F13%2F635862.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-40-1" title="View reference  in text" id="ref-40">↵</a><div class="cit ref-cit ref-journal" id="cit-635862v1.40" data-doi="10.1093/molbev/msu136"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Pfeifer</span>  <span class="cit-name-given-names">B</span></span>, <span class="cit-auth"><span class="cit-name-surname">Wittelsburger</span>  <span class="cit-name-given-names">U</span></span>, <span class="cit-auth"><span class="cit-name-surname">Ramos-Onsins</span>  <span class="cit-name-given-names">SE</span></span>, <span class="cit-auth"><span class="cit-name-surname">Lercher</span>  <span class="cit-name-given-names">MJ</span></span> (<span class="cit-pub-date">2014</span>) <span class="cit-article-title">PopGenome: an efficient Swiss army knife for populations genomic analyses in R</span>. <abbr class="cit-jnl-abbrev">Molecular Biology and Evolution</abbr>, <span class="cit-vol">31</span>(<span class="cit-issue">7</span>), <span class="cit-fpage">1929</span>–<span class="cit-lpage">36</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2BBiology%2Band%2BEvolution%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fmolbev%252Fmsu136%26rft_id%253Dinfo%253Apmid%252F24739305%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/molbev/msu136&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24739305&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F05%2F13%2F635862.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000339423800026&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-41-1" title="View reference  in text" id="ref-41">↵</a><div class="cit ref-cit ref-journal" id="cit-635862v1.41" data-doi="10.1016/j.cub.2009.11.055"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Pritchard</span>  <span class="cit-name-given-names">JK</span></span>, <span class="cit-auth"><span class="cit-name-surname">Pickrell</span>  <span class="cit-name-given-names">JK</span></span>, <span class="cit-auth"><span class="cit-name-surname">Coop</span>  <span class="cit-name-given-names">G</span></span> (<span class="cit-pub-date">2010a</span>) <span class="cit-article-title">The genetics of human adaptation: hard sweeps, soft sweeps, and polygenic adaptation</span>. <abbr class="cit-jnl-abbrev">Current Biology</abbr>, <span class="cit-vol">20</span>, <span class="cit-fpage">R208</span>–<span class="cit-lpage">R215</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCurrent%2Bbiology%2B%253A%2B%2BCB%26rft.stitle%253DCurr%2BBiol%26rft.aulast%253DPritchard%26rft.auinit1%253DJ.%2BK.%26rft.volume%253D20%26rft.issue%253D4%26rft.spage%253DR208%26rft.epage%253DR215%26rft.atitle%253DThe%2Bgenetics%2Bof%2Bhuman%2Badaptation%253A%2Bhard%2Bsweeps%252C%2Bsoft%2Bsweeps%252C%2Band%2Bpolygenic%2Badaptation.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.cub.2009.11.055%26rft_id%253Dinfo%253Apmid%252F20178769%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.cub.2009.11.055&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20178769&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F05%2F13%2F635862.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000275149200026&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-42-1" title="View reference  in text" id="ref-42">↵</a><div class="cit ref-cit ref-journal" id="cit-635862v1.42" data-doi="10.1038/nrg2880"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Pritchard</span>  <span class="cit-name-given-names">JK</span></span>, <span class="cit-auth"><span class="cit-name-surname">Di Rienzo</span>  <span class="cit-name-given-names">A</span></span> (<span class="cit-pub-date">2010b</span>) <span class="cit-article-title">Adaptation-not by sweeps alone</span>. <abbr class="cit-jnl-abbrev">Nature Review Genetics</abbr>, <span class="cit-vol">11</span>(<span class="cit-issue">10</span>), <span class="cit-fpage">665</span>–<span class="cit-lpage">667</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Breviews.%2BGenetics%26rft.stitle%253DNat%2BRev%2BGenet%26rft.aulast%253DPritchard%26rft.auinit1%253DJ.%2BK.%26rft.volume%253D11%26rft.issue%253D10%26rft.spage%253D665%26rft.epage%253D667%26rft.atitle%253DAdaptation%2B-%2Bnot%2Bby%2Bsweeps%2Balone.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnrg2880%26rft_id%253Dinfo%253Apmid%252F20838407%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nrg2880&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20838407&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F05%2F13%2F635862.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000281911300014&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-43-1" title="View reference  in text" id="ref-43">↵</a><div class="cit ref-cit ref-journal" id="cit-635862v1.43" data-doi="10.1111/j.1365-294X.2011.05045.x"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Prunier</span>  <span class="cit-name-given-names">J</span></span>, <span class="cit-auth"><span class="cit-name-surname">Laroche</span>  <span class="cit-name-given-names">J</span></span>, <span class="cit-auth"><span class="cit-name-surname">Beaulieu</span>  <span class="cit-name-given-names">J</span></span>, <span class="cit-auth"><span class="cit-name-surname">Bousquet</span>  <span class="cit-name-given-names">J</span></span> (<span class="cit-pub-date">2011</span>) <span class="cit-article-title">Scanning the genome for gene SNPs related to climate adaptation and estimating selection at the molecular level in boreal black spruce</span>. <abbr class="cit-jnl-abbrev">Molecular Ecology</abbr>, <span class="cit-vol">20</span>(<span class="cit-issue">8</span>), <span class="cit-fpage">1702</span>–<span class="cit-lpage">1716</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2BEcology%2B%2528Print%2529%26rft.stitle%253DMolecular%2BEcology%2B%2528Print%2529%26rft.aulast%253DPrunier%26rft.auinit1%253DJ.%26rft.volume%253D20%26rft.issue%253D8%26rft.spage%253D1702%26rft.epage%253D1716%26rft.atitle%253DScanning%2Bthe%2Bgenome%2Bfor%2Bgene%2BSNPs%2Brelated%2Bto%2Bclimate%2Badaptation%2Band%2Bestimating%2Bselection%2Bat%2Bthe%2Bmolecular%2Blevel%2Bin%2Bboreal%2Bblack%2Bspruce.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1365-294X.2011.05045.x%26rft_id%253Dinfo%253Apmid%252F21375634%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1365-294X.2011.05045.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21375634&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F05%2F13%2F635862.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-44-1" title="View reference  in text" id="ref-44">↵</a><div class="cit ref-cit ref-journal" id="cit-635862v1.44"><div class="cit-metadata"><cite><span class="cit-auth cit-collab">RStudio Team</span> (<span class="cit-pub-date">2016</span>) <span class="cit-article-title">RStudio: Integrated development for R. RStudio, Inc., Boston, MA</span>. URL <a href="http://www.rstudio.com/">http://www.rstudio.com/</a></cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-45-1" title="View reference  in text" id="ref-45">↵</a><div class="cit ref-cit ref-journal" id="cit-635862v1.45" data-doi="10.1534/genetics.114.164350"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Raj</span>  <span class="cit-name-given-names">A</span></span>, <span class="cit-auth"><span class="cit-name-surname">Stephens</span>  <span class="cit-name-given-names">M</span></span>, and <span class="cit-auth"> <span class="cit-name-given-names">JK.</span> <span class="cit-name-surname">Pritchard</span></span> (<span class="cit-pub-date">2014</span>) <span class="cit-article-title">fastSTRUCTURE: Variational Inference of Population Structure in Large SNP Data Sets</span>. <abbr class="cit-jnl-abbrev">Genetics</abbr>, <span class="cit-vol">197</span>, <span class="cit-fpage">573</span>–<span class="cit-lpage">589</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.114.164350%26rft_id%253Dinfo%253Apmid%252F24700103%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTk3LzIvNTczIjtzOjQ6ImF0b20iO3M6Mzc6Ii9iaW9yeGl2L2Vhcmx5LzIwMTkvMDUvMTMvNjM1ODYyLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-46-1" title="View reference  in text" id="ref-46">↵</a><div class="cit ref-cit ref-journal" id="cit-635862v1.46"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Savolainen</span>  <span class="cit-name-given-names">O</span></span>, <span class="cit-auth"><span class="cit-name-surname">Pyhajarvi</span>  <span class="cit-name-given-names">T</span></span>, <span class="cit-auth"><span class="cit-name-surname">Knurr</span>  <span class="cit-name-given-names">T</span></span> (<span class="cit-pub-date">2007</span>) <span class="cit-article-title">Gene flow and local adaptation in trees. <em>Annual Review of Ecology</em></span>, <abbr class="cit-jnl-abbrev">Evolution and Systematics</abbr>, <span class="cit-vol">38</span>, <span class="cit-fpage">595</span>–<span class="cit-lpage">619</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEvolution%2Band%2BSystematics%26rft.volume%253D38%26rft.spage%253D595%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-47-1" title="View reference  in text" id="ref-47">↵</a><div class="cit ref-cit ref-journal" id="cit-635862v1.47"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Sewell</span>  <span class="cit-name-given-names">MM</span></span>, <span class="cit-auth"><span class="cit-name-surname">Sherman</span>  <span class="cit-name-given-names">BK</span></span>, <span class="cit-auth"><span class="cit-name-surname">Neale</span>  <span class="cit-name-given-names">DB</span></span> (<span class="cit-pub-date">1999</span>) <span class="cit-article-title">A Consensus Map for loblolly Pine (<em>Pinus taeda</em> L.). I. Construction and Integration of Individual Linkage Maps from Two outbred three-generation pedigrees</span>. <abbr class="cit-jnl-abbrev">Genetics</abbr>, <span class="cit-vol">151</span>, <span class="cit-fpage">321</span>–<span class="cit-lpage">330</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DSewell%26rft.auinit1%253DM.%2BM.%26rft.volume%253D151%26rft.issue%253D1%26rft.spage%253D321%26rft.epage%253D330%26rft.atitle%253DA%2BConsensus%2BMap%2Bfor%2BLoblolly%2BPine%2B%2528Pinus%2Btaeda%2BL.%2529.%2BI.%2BConstruction%2Band%2BIntegration%2Bof%2BIndividual%2BLinkage%2BMaps%2BFrom%2BTwo%2BOutbred%2BThree-Generation%2BPedigrees%26rft_id%253Dinfo%253Apmid%252F9872970%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTUxLzEvMzIxIjtzOjQ6ImF0b20iO3M6Mzc6Ii9iaW9yeGl2L2Vhcmx5LzIwMTkvMDUvMTMvNjM1ODYyLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-48-1" title="View reference  in text" id="ref-48">↵</a><div class="cit ref-cit ref-journal" id="cit-635862v1.48"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Schmidtling</span>  <span class="cit-name-given-names">R</span></span> (<span class="cit-pub-date">2001</span>) <span class="cit-article-title">Southern pine seed sources</span>. In: <abbr class="cit-jnl-abbrev">Gen. Tech. Rep</abbr>. SRS-44. <span class="cit-auth cit-collab">U.S. Department of Agriculture, Forest Service, Southern Research Station</span>, Asheville, NC, p <span class="cit-fpage">25</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-49-1" title="View reference  in text" id="ref-49">↵</a><div class="cit ref-cit ref-journal" id="cit-635862v1.49"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Schmidtling</span>  <span class="cit-name-given-names">RC</span></span> (<span class="cit-pub-date">2003</span>) <span class="cit-article-title">The southern pines during the Pleistocene</span>. <abbr class="cit-jnl-abbrev">ISHS Acta Horticulturae</abbr>, <span class="cit-vol">615</span>, <span class="cit-fpage">203</span>–<span class="cit-lpage">209</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DISHS%2BActa%2BHorticulturae%26rft.volume%253D615%26rft.spage%253D203%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-50-1" title="View reference  in text" id="ref-50">↵</a><div class="cit ref-cit ref-journal" id="cit-635862v1.50" data-doi="10.1093/bfgp/elu009"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Slotte</span>  <span class="cit-name-given-names">T</span></span> (<span class="cit-pub-date">2014</span>) <span class="cit-article-title">The impact of linked selection on plant genomic variation</span>. <abbr class="cit-jnl-abbrev">Briefings in Functional Genomics</abbr>, <span class="cit-vol">13</span>(<span class="cit-issue">4</span>), <span class="cit-fpage">268</span>–<span class="cit-lpage">275</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBriefings%2Bin%2BFunctional%2BGenomics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbfgp%252Felu009%26rft_id%253Dinfo%253Apmid%252F24759704%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bfgp/elu009&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24759704&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F05%2F13%2F635862.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-51-1" title="View reference  in text" id="ref-51">↵</a><div class="cit ref-cit ref-journal" id="cit-635862v1.51"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Smith</span>  <span class="cit-name-given-names">JM</span></span>, <span class="cit-auth"><span class="cit-name-surname">Haigh</span>  <span class="cit-name-given-names">J</span></span> (<span class="cit-pub-date">1974</span>) <span class="cit-article-title">The hitch-hiking of a favourable gene</span>. <abbr class="cit-jnl-abbrev">Genetics Research</abbr>, <span class="cit-vol">23</span>(<span class="cit-issue">1</span>), <span class="cit-fpage">23</span>–<span class="cit-lpage">35</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%2BResearch%26rft.volume%253D23%26rft.spage%253D23%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-52-1" title="View reference  in text" id="ref-52">↵</a><div class="cit ref-cit ref-journal" id="cit-635862v1.52" data-doi="10.1111/j.1365-294X.2006.03061.x"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Soltis</span>  <span class="cit-name-given-names">DE</span></span>, <span class="cit-auth"><span class="cit-name-surname">Morris</span>  <span class="cit-name-given-names">AB</span></span>, <span class="cit-auth"><span class="cit-name-surname">Mclachlan</span>  <span class="cit-name-given-names">JS</span></span>, <span class="cit-auth"><span class="cit-name-surname">Manos</span>  <span class="cit-name-given-names">PS</span></span>, <span class="cit-auth"><span class="cit-name-surname">Soltis</span>  <span class="cit-name-given-names">PS</span></span> (<span class="cit-pub-date">2006</span>) <span class="cit-article-title">Comparative phylogeography of unglaciated eastern North America</span>. <abbr class="cit-jnl-abbrev">Molecular Ecology</abbr>, <span class="cit-vol">15</span>, <span class="cit-fpage">4261</span>–<span class="cit-lpage">4293</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2BEcology%2B%2528Print%2529%26rft.stitle%253DMolecular%2BEcology%2B%2528Print%2529%26rft.aulast%253DSoltis%26rft.auinit1%253DD.%2BE.%26rft.volume%253D15%26rft.issue%253D14%26rft.spage%253D4261%26rft.epage%253D4293%26rft.atitle%253DComparative%2Bphylogeography%2Bof%2Bunglaciated%2Beastern%2BNorth%2BAmerica.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1365-294X.2006.03061.x%26rft_id%253Dinfo%253Apmid%252F17107465%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1365-294X.2006.03061.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17107465&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F05%2F13%2F635862.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000242042800001&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-53-1" title="View reference  in text" id="ref-53">↵</a><div class="cit ref-cit ref-journal" id="cit-635862v1.53" data-doi="10.18637/jss.v079.i06"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Taylor</span>  <span class="cit-name-given-names">J</span></span>, <span class="cit-auth"><span class="cit-name-surname">Butler</span>  <span class="cit-name-given-names">D</span></span> (<span class="cit-pub-date">2017</span>) <span class="cit-article-title">R package ASMap: Efficient Genetic Linkage Map Construction and Diagnosis</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">DOI: </span><span class="cit-pub-id cit-pub-id-doi">10.18637/jss.v079.i06</span><span class="cit-pub-id-sep cit-pub-id-doi-sep">.</span><span class="cit-pub-id-sep cit-pub-id-arxiv-sep"> </span><span class="cit-pub-id-scheme">arXiv:</span><span class="cit-pub-id cit-pub-id-arxiv">1705.06916</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.volume%253D79%26rft.spage%253D1%26rft_id%253Dinfo%253Adoi%252F10.18637%252Fjss.v079.i06%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.18637/jss.v079.i06&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-54-1" title="View reference  in text" id="ref-54">↵</a><div class="cit ref-cit ref-journal" id="cit-635862v1.54" data-doi="10.1093/treephys/3.1.41"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Teskey</span>  <span class="cit-name-given-names">RO</span></span>, <span class="cit-auth"><span class="cit-name-surname">Bongarten</span>  <span class="cit-name-given-names">BC</span></span>, <span class="cit-auth"><span class="cit-name-surname">Cregg</span>  <span class="cit-name-given-names">BM</span></span>, <span class="cit-auth"><span class="cit-name-surname">Dougherty</span>  <span class="cit-name-given-names">PM</span></span>, <span class="cit-auth"><span class="cit-name-surname">Hennessey</span>  <span class="cit-name-given-names">TC</span></span> (<span class="cit-pub-date">1987</span>) <span class="cit-article-title">Physiology and genetics of tree growth response to moisture and temperature stress: an examination of the characteristics of loblolly pine (<em>Pinus taeda</em> L</span>.). <abbr class="cit-jnl-abbrev">Tree Physiology</abbr>, <span class="cit-vol">3</span>, <span class="cit-fpage">41</span>–<span class="cit-lpage">61</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTree%2BPhysiology%26rft_id%253Dinfo%253Adoi%252F10.1093%252Ftreephys%252F3.1.41%26rft_id%253Dinfo%253Apmid%252F14975834%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/treephys/3.1.41&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=14975834&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F05%2F13%2F635862.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000207043800005&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-55-1" title="View reference  in text" id="ref-55">↵</a><div class="cit ref-cit ref-journal" id="cit-635862v1.55" data-doi="10.1101/005165"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Turner</span>  <span class="cit-name-given-names">SD</span></span> (<span class="cit-pub-date">2014</span>) <span class="cit-article-title">Qqman: an R package for visualizing GWAS results using Q-Q and manhattan plots</span>. <abbr class="cit-jnl-abbrev">biorXiv</abbr><span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">DOI: </span><span class="cit-pub-id cit-pub-id-doi">10.1101/005165</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DbiorXiv%26rft_id%253Dinfo%253Adoi%252F10.1101%252F005165%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiYmlvcnhpdiI7czo1OiJyZXNpZCI7czo4OiIwMDUxNjV2MSI7czo0OiJhdG9tIjtzOjM3OiIvYmlvcnhpdi9lYXJseS8yMDE5LzA1LzEzLzYzNTg2Mi5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><div class="cit ref-cit ref-book no-rev-xref" id="cit-635862v1.56"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Van Ooijen</span>  <span class="cit-name-given-names">JW</span></span> (<span class="cit-pub-date">2017</span>) <span class="cit-source">JoinMap5, software for the calculation of genetic linkage maps in experimental populations of diploid species</span>. <span class="cit-publ-name">Kyazma B.V</span>, <span class="cit-publ-loc">Wageningen, Netherlands</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-57-1" title="View reference  in text" id="ref-57">↵</a><div class="cit ref-cit ref-journal" id="cit-635862v1.57" data-doi="10.1098/rstb.2011.0260"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Via</span>  <span class="cit-name-given-names">S</span></span> (<span class="cit-pub-date">2012</span>) <span class="cit-article-title">Divergence hitchhiking and the spread of genomic isolation during ecological speciation-with-gene-flow</span>. <abbr class="cit-jnl-abbrev">Phil. Trans R Soc B</abbr>, <span class="cit-vol">367</span>, <span class="cit-fpage">451</span>–<span class="cit-lpage">460</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPhilosophical%2BTransactions%2Bof%2Bthe%2BRoyal%2BSociety%2BB%253A%2BBiological%2BSciences%26rft.stitle%253DPhil%2BTrans%2BR%2BSoc%2BB%26rft.aulast%253DVia%26rft.auinit1%253DS.%26rft.volume%253D367%26rft.issue%253D1587%26rft.spage%253D451%26rft.epage%253D460%26rft.atitle%253DDivergence%2Bhitchhiking%2Band%2Bthe%2Bspread%2Bof%2Bgenomic%2Bisolation%2Bduring%2Becological%2Bspeciation-with-gene-flow%26rft_id%253Dinfo%253Adoi%252F10.1098%252Frstb.2011.0260%26rft_id%253Dinfo%253Apmid%252F22201174%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1098/rstb.2011.0260&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22201174&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F05%2F13%2F635862.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-58-1" title="View reference  in text" id="ref-58">↵</a><div class="cit ref-cit ref-journal" id="cit-635862v1.58"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Wells</span>  <span class="cit-name-given-names">OO</span></span>, <span class="cit-auth"><span class="cit-name-surname">Wakeley</span>  <span class="cit-name-given-names">PC</span></span> (<span class="cit-pub-date">1966</span>) <span class="cit-article-title">Geographic variation in survival, growth and fusiform infection of planted loblolly pine</span>. <abbr class="cit-jnl-abbrev">Forest Science Monographs</abbr>, <span class="cit-vol">11</span>, <span class="cit-fpage">1</span>–<span class="cit-lpage">40</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DForest%2BScience%2BMonographs%26rft.volume%253D11%26rft.spage%253D1%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-59-1" title="View reference  in text" id="ref-59">↵</a><div class="cit ref-cit ref-journal" id="cit-635862v1.59" data-doi="10.1371/journal.pone.0156720"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">T</span></span>, <span class="cit-auth"><span class="cit-name-surname">Hamann</span>  <span class="cit-name-given-names">A</span></span>, <span class="cit-auth"><span class="cit-name-surname">Spittlehouse</span>  <span class="cit-name-given-names">D</span></span>, <span class="cit-auth"><span class="cit-name-surname">Carroll</span>  <span class="cit-name-given-names">C</span></span> (<span class="cit-pub-date">2016</span>) <span class="cit-article-title">Locally downscaled and spatially customizable climate data for historical and future periods for North America</span>. <abbr class="cit-jnl-abbrev">Plos One</abbr>, <span class="cit-vol">11</span>(<span class="cit-issue">6</span>), <span class="cit-fpage">e0156720</span>.<a href="https://doi.org/10.1371/journal.pone.0156720">https://doi.org/10.1371/journal.pone.0156720</a>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlos%2BOne%26rft.volume%253D11%26rft.spage%253De0156720%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0156720%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0156720&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-60-1" title="View reference  in text" id="ref-60">↵</a><div class="cit ref-cit ref-journal" id="cit-635862v1.60" data-doi="10.1093/molbev/msw051"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">J</span></span>, <span class="cit-auth"><span class="cit-name-surname">Street</span>  <span class="cit-name-given-names">NR</span></span>, <span class="cit-auth"><span class="cit-name-surname">Scofield</span>  <span class="cit-name-given-names">DG</span></span>, <span class="cit-auth"><span class="cit-name-surname">Ingvarsson</span>  <span class="cit-name-given-names">PK</span></span> (<span class="cit-pub-date">2016</span>) <span class="cit-article-title">Variation in linked selection and recombination drive genomic divergence during allopatric speciation of European and American aspens</span>. <abbr class="cit-jnl-abbrev">Molecular Biology and Evolution</abbr>, <span class="cit-vol">33</span>(<span class="cit-issue">7</span>), <span class="cit-fpage">1754</span>–<span class="cit-lpage">1767</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2BBiology%2Band%2BEvolution%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fmolbev%252Fmsw051%26rft_id%253Dinfo%253Apmid%252F26983554%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/molbev/msw051&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=26983554&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F05%2F13%2F635862.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-61-1" title="View reference  in text" id="ref-61">↵</a><div class="cit ref-cit ref-journal" id="cit-635862v1.61"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Westbrook</span>  <span class="cit-name-given-names">JW</span></span>, <span class="cit-auth"><span class="cit-name-surname">Chhatre</span>  <span class="cit-name-given-names">VE</span></span>, <span class="cit-auth"><span class="cit-name-surname">Wu</span>  <span class="cit-name-given-names">LS</span></span>, <span class="cit-auth"><span class="cit-name-surname">Chamala</span>  <span class="cit-name-given-names">S</span></span>, <span class="cit-auth"><span class="cit-name-surname">Gomide Neves</span>  <span class="cit-name-given-names">L</span></span>, <span class="cit-auth"><span class="cit-name-surname">Munoz</span>  <span class="cit-name-given-names">P</span></span>, <span class="cit-auth"><span class="cit-name-surname">Martinez-Garcia</span>  <span class="cit-name-given-names">P</span></span>, <span class="cit-auth"><span class="cit-name-surname">Neale</span>  <span class="cit-name-given-names">D</span></span>, <span class="cit-auth"><span class="cit-name-surname">Kirst</span>  <span class="cit-name-given-names">M</span></span>, <span class="cit-auth"><span class="cit-name-surname">Mockaitis</span>  <span class="cit-name-given-names">K</span></span>, <span class="cit-auth"><span class="cit-name-surname">Nelson</span>  <span class="cit-name-given-names">CD</span></span>, <span class="cit-auth"><span class="cit-name-surname">Peter</span>  <span class="cit-name-given-names">GF</span></span>, <span class="cit-auth"><span class="cit-name-surname">Davis</span>  <span class="cit-name-given-names">JM</span></span>, <span class="cit-auth"><span class="cit-name-surname">Echt</span>  <span class="cit-name-given-names">CS</span></span> (<span class="cit-pub-date">2015</span>) <span class="cit-article-title">A consensus genetic map for <em>Pinus taeda</em> and <em>Pinus elliottii</em> and extent of linkage disequilibrium in two genotype-phenotype discovery populations of <em>Pinus taeda</em></span>. <abbr class="cit-jnl-abbrev">Genes, genomes and genetics</abbr>, <span class="cit-vol">5</span>, <span class="cit-fpage">1685</span>–<span class="cit-lpage">1694</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenes%252C%2Bgenomes%2Band%2Bgenetics%26rft.volume%253D5%26rft.spage%253D1685%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-62-1" title="View reference  in text" id="ref-62">↵</a><div class="cit ref-cit ref-journal" id="cit-635862v1.62" data-doi="10.1086/682949"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Whitlock</span>  <span class="cit-name-given-names">MC</span></span>, <span class="cit-auth"><span class="cit-name-surname">Lotterhos</span>  <span class="cit-name-given-names">KE</span></span> (<span class="cit-pub-date">2015</span>) <span class="cit-article-title">Reliable detection of loci responsible for local adaptation: inference of a null model through trimming the distribution of Fst</span>. <abbr class="cit-jnl-abbrev">The American Naturalist</abbr>, <span class="cit-vol">186</span>, <span class="cit-fpage">S24</span>–<span class="cit-lpage">S36</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DThe%2BAmerican%2BNaturalist%26rft.volume%253D186%26rft.spage%253DS24%26rft_id%253Dinfo%253Adoi%252F10.1086%252F682949%26rft_id%253Dinfo%253Apmid%252F26656214%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1086/682949&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=26656214&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F05%2F13%2F635862.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-63-1" title="View reference  in text" id="ref-63">↵</a><div class="cit ref-cit ref-journal" id="cit-635862v1.63"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Williamson</span>  <span class="cit-name-given-names">RJ</span></span>, <span class="cit-auth"><span class="cit-name-surname">Josephs</span>  <span class="cit-name-given-names">EB</span></span>, <span class="cit-auth"><span class="cit-name-surname">Platts</span>  <span class="cit-name-given-names">AE</span></span>, <span class="cit-auth"><span class="cit-name-surname">Hazzouri</span>  <span class="cit-name-given-names">KM</span></span>, <span class="cit-auth"><span class="cit-name-surname">Haudry</span>  <span class="cit-name-given-names">A</span></span>, <span class="cit-auth"><span class="cit-name-surname">Blanchette</span>  <span class="cit-name-given-names">M</span></span>, <span class="cit-auth"><span class="cit-name-surname">Wright</span>  <span class="cit-name-given-names">SI</span></span> (<span class="cit-pub-date">2014</span>) <span class="cit-article-title">Evidence for widespread positive and negative selection in coding and conserved noncoding regions of <em>Capsella</em></span>. <abbr class="cit-jnl-abbrev">PLos Genetics</abbr>, <span class="cit-vol">10</span>(<span class="cit-issue">9</span>), <span class="cit-fpage">e1004622</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLos%2BGenetics%26rft.volume%253D10%26rft.spage%253De1004622%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-64-1" title="View reference  in text" id="ref-64">↵</a><div class="cit ref-cit ref-journal" id="cit-635862v1.64" data-doi="10.1111/j.1558-5646.2011.01269.x"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Yeaman</span>  <span class="cit-name-given-names">S</span></span>, <span class="cit-auth"><span class="cit-name-surname">Whitlock</span>  <span class="cit-name-given-names">MC</span></span> (<span class="cit-pub-date">2011</span>) <span class="cit-article-title">The genetic architecture of adaptation under migration-selection balance</span>. <abbr class="cit-jnl-abbrev">Evolution</abbr>, <span class="cit-vol">65</span>, <span class="cit-fpage">1897</span>–<span class="cit-lpage">1911</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEvolution%26rft.stitle%253DEvolution%26rft.aulast%253DYeaman%26rft.auinit1%253DS.%26rft.volume%253D65%26rft.issue%253D7%26rft.spage%253D1897%26rft.epage%253D1911%26rft.atitle%253DThe%2Bgenetic%2Barchitecture%2Bof%2Badaptation%2Bunder%2Bmigration-selection%2Bbalance.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1558-5646.2011.01269.x%26rft_id%253Dinfo%253Apmid%252F21729046%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1558-5646.2011.01269.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21729046&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F05%2F13%2F635862.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000292447000006&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-65-1" title="View reference  in text" id="ref-65">↵</a><div class="cit ref-cit ref-journal" id="cit-635862v1.65"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Yeaman</span>  <span class="cit-name-given-names">S</span></span> (<span class="cit-pub-date">2013</span>) <span class="cit-article-title">Genomic rearrangements and the evolution of clusters of locally adapted loci</span>. <abbr class="cit-jnl-abbrev">PNAS</abbr>, <span class="cit-fpage">e1743</span>–<span class="cit-lpage">e1751</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-66-1" title="View reference  in text" id="ref-66">↵</a><div class="cit ref-cit ref-journal" id="cit-635862v1.66" data-doi="10.1086/682405"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Yeaman</span>  <span class="cit-name-given-names">S</span></span> (<span class="cit-pub-date">2015</span>) <span class="cit-article-title">Local adaptation by alleles of small effect</span>. <abbr class="cit-jnl-abbrev">The American Naturalist</abbr>, <span class="cit-vol">186</span>, <span class="cit-fpage">S74</span>–<span class="cit-lpage">S89</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DThe%2BAmerican%2BNaturalist%26rft.volume%253D186%26rft.spage%253DS74%26rft_id%253Dinfo%253Adoi%252F10.1086%252F682405%26rft_id%253Dinfo%253Apmid%252F26656219%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1086/682405&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=26656219&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F05%2F13%2F635862.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-67-1" title="View reference  in text" id="ref-67">↵</a><div class="cit ref-cit ref-journal" id="cit-635862v1.67" data-doi="10.1534/genetics.113.159319"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Yoder</span>  <span class="cit-name-given-names">JB</span></span>, <span class="cit-auth"><span class="cit-name-surname">Stanton-Geddes</span>  <span class="cit-name-given-names">J</span></span>, <span class="cit-auth"><span class="cit-name-surname">Zhou</span>  <span class="cit-name-given-names">P</span></span>, <span class="cit-auth"><span class="cit-name-surname">Briskine</span>  <span class="cit-name-given-names">R</span></span>, <span class="cit-auth"><span class="cit-name-surname">Young</span>  <span class="cit-name-given-names">ND</span></span>, <span class="cit-auth"><span class="cit-name-surname">Tiffin</span>  <span class="cit-name-given-names">P</span></span> (<span class="cit-pub-date">2014</span>) <span class="cit-article-title">Genomic signature of adaptation in <em>Medicago truncatula</em></span>. <abbr class="cit-jnl-abbrev">Genetics</abbr>, <span class="cit-vol">196</span>, <span class="cit-fpage">1263</span>–<span class="cit-lpage">1275</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.113.159319%26rft_id%253Dinfo%253Apmid%252F24443444%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE5Ni80LzEyNjMiO3M6NDoiYXRvbSI7czozNzoiL2Jpb3J4aXYvZWFybHkvMjAxOS8wNS8xMy82MzU4NjIuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-68-1" title="View reference  in text" id="ref-68">↵</a><div class="cit ref-cit ref-journal" id="cit-635862v1.68" data-doi="10.1038/ng.546"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">Z</span></span>, <span class="cit-auth"><span class="cit-name-surname">Ersoz</span>  <span class="cit-name-given-names">E</span></span>, <span class="cit-auth"><span class="cit-name-surname">Lai</span>  <span class="cit-name-given-names">CQ</span></span>, <span class="cit-auth"><span class="cit-name-surname">Todhunter</span>  <span class="cit-name-given-names">RJ</span></span>, <span class="cit-auth"><span class="cit-name-surname">Tiwari</span>  <span class="cit-name-given-names">HK</span></span>, <span class="cit-auth"><span class="cit-name-surname">Gore</span>  <span class="cit-name-given-names">MA</span></span>, <span class="cit-auth"><span class="cit-name-surname">Bradbury</span>  <span class="cit-name-given-names">PJ</span></span>, <span class="cit-auth"><span class="cit-name-surname">Yu</span>  <span class="cit-name-given-names">J</span></span>, <span class="cit-auth"><span class="cit-name-surname">Arnett</span>  <span class="cit-name-given-names">DK</span></span>, <span class="cit-auth"><span class="cit-name-surname">Ordovas</span>  <span class="cit-name-given-names">JM</span></span>, <span class="cit-auth"><span class="cit-name-surname">Buckler</span>  <span class="cit-name-given-names">ES</span></span> (<span class="cit-pub-date">2010</span>) <span class="cit-article-title">Mixed linear model approach adapted for genome wide association studies</span>. <abbr class="cit-jnl-abbrev">Nature Genetics</abbr>, <span class="cit-vol">42</span>, <span class="cit-fpage">355</span>–<span class="cit-lpage">360</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bgenetics%26rft.stitle%253DNat%2BGenet%26rft.aulast%253DZhang%26rft.auinit1%253DZ.%26rft.volume%253D42%26rft.issue%253D4%26rft.spage%253D355%26rft.epage%253D360%26rft.atitle%253DMixed%2Blinear%2Bmodel%2Bapproach%2Badapted%2Bfor%2Bgenome-wide%2Bassociation%2Bstudies.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fng.546%26rft_id%253Dinfo%253Apmid%252F20208535%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ng.546&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20208535&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F05%2F13%2F635862.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000276150500017&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-69-1" title="View reference  in text" id="ref-69">↵</a><div class="cit ref-cit ref-journal" id="cit-635862v1.69" data-doi="10.1089/10665270050081478"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Zheng</span>  <span class="cit-name-given-names">Z</span></span>, <span class="cit-auth"><span class="cit-name-surname">Schwartz</span>  <span class="cit-name-given-names">S</span></span>, <span class="cit-auth"><span class="cit-name-surname">Wagner</span>  <span class="cit-name-given-names">L</span></span>, <span class="cit-auth"><span class="cit-name-surname">Miller</span>  <span class="cit-name-given-names">W</span></span> (<span class="cit-pub-date">2000</span>) <span class="cit-article-title">A greedy algorithm for aligning DNA sequences</span>. <abbr class="cit-jnl-abbrev">Journal of Computational Biology</abbr>, <span class="cit-vol">7</span>(<span class="cit-issue">1-2</span>), <span class="cit-fpage">203</span>–<span class="cit-lpage">14</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2Bcomputational%2Bbiology%2B%253A%2B%2Ba%2Bjournal%2Bof%2Bcomputational%2Bmolecular%2Bcell%2Bbiology%26rft.stitle%253DJ%2BComput%2BBiol%26rft.aulast%253DZhang%26rft.auinit1%253DZ.%26rft.volume%253D7%26rft.issue%253D1-2%26rft.spage%253D203%26rft.epage%253D214%26rft.atitle%253DA%2Bgreedy%2Balgorithm%2Bfor%2Baligning%2BDNA%2Bsequences.%26rft_id%253Dinfo%253Adoi%252F10.1089%252F10665270050081478%26rft_id%253Dinfo%253Apmid%252F10890397%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1089/10665270050081478&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=10890397&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F05%2F13%2F635862.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000087833300012&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.biorxiv.org/content/10.1101/635862v1.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-disqus-comment" >
  
      
  
  <div class="pane-content">
    <div id="disqus_thread"><noscript><p><a href="http://biorxivstage.disqus.com/?url=https%3A%2F%2Fwww.biorxiv.org%2Fcontent%2F10.1101%2F635862v1" class="" data-icon-position="" data-hide-link-title="0">View the discussion thread.</a></p></noscript></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/10.1101/632984v1" title="Entecavir resistance mutations rtL180M/T184L/M204V combined with rtA200V lead to tenofovir resistance" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous</span></a></span><span class="pager-next"><a href="/content/10.1101/283929v2-0" title="Cell-type heterogeneity in adipose tissue is associated with complex traits and reveals disease-relevant cell-specific eQTLs" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    Posted&nbsp;May 13, 2019.  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-biorxiv-art-tools" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-biorxiv_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/10.1101/635862v1.full.pdf" target="_self" class="article-dl-pdf-link link-icon"><i class="icon-external-link-sign"></i> <span class="title">Download PDF</span></a>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-biorxiv-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="Email this Article" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-biorxiv_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/10.1101/635862v1.full" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for your interest in spreading the word about bioRxiv.</p><p>NOTE: Your email address is requested solely to identify you as the sender of this article.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/10.1101/635862v1" class="active" data-icon-position="" data-hide-link-title="0">Climate adaptation shaped by subtle to moderate allele frequency shifts in loblolly pine</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from bioRxiv
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would like to see this page from the bioRxiv website.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/732092" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-0gwE4p5WTsN9BAYcxv8gb56-H2X4ltnc4SVYMvRwHoY" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-biorxiv-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-biorxiv_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="732092" id="node732092--2" data-pisa="biorxiv;635862v1" data-pisa-master="biorxiv;635862" data-seqnum="732092" data-apath="/biorxiv/early/2019/05/13/635862.atom"><div  class="highwire-cite highwire-cite-highwire-article highwire-citation-biorxiv-article-pap-list clearfix" >

      <div class="highwire-cite-title" >
      <div class="highwire-cite-title">Climate adaptation shaped by subtle to moderate allele frequency shifts in loblolly pine</div>    </div>
  
      <div  class="highwire-cite-authors" ><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0"><span class="nlm-given-names">Amanda R.</span> <span class="nlm-surname">De La Torre</span></span>, <span class="highwire-citation-author" data-delta="1"><span class="nlm-given-names">David B</span> <span class="nlm-surname">Neale</span></span></span></div>
  
      <div  class="highwire-cite-metadata" ><span  class="highwire-cite-metadata-journal highwire-cite-metadata">bioRxiv </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">635862; </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata"><span class="doi_label">doi:</span> https://doi.org/10.1101/635862 </span></div>
  
  
  </div>
</div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.biorxiv.org/content/10.1101/635862v1" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.biorxiv.org/content/10.1101/635862v1&amp;title=Climate%20adaptation%20shaped%20by%20subtle%20to%20moderate%20allele%20frequency%20shifts%20in%20loblolly%20pine" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.biorxiv.org/content/10.1101/635862v1&amp;title=Climate%20adaptation%20shaped%20by%20subtle%20to%20moderate%20allele%20frequency%20shifts%20in%20loblolly%20pine" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.biorxiv.org/content/10.1101/635862v1&amp;text=Climate%20adaptation%20shaped%20by%20subtle%20to%20moderate%20allele%20frequency%20shifts%20in%20loblolly%20pine" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.biorxiv.org/content/10.1101/635862v1&amp;title=Climate%20adaptation%20shaped%20by%20subtle%20to%20moderate%20allele%20frequency%20shifts%20in%20loblolly%20pine" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.biorxiv.org/content/10.1101/635862v1&amp;t=Climate%20adaptation%20shaped%20by%20subtle%20to%20moderate%20allele%20frequency%20shifts%20in%20loblolly%20pine" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.biorxiv.org/content/10.1101/635862v1&amp;title=Climate%20adaptation%20shaped%20by%20subtle%20to%20moderate%20allele%20frequency%20shifts%20in%20loblolly%20pine" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.biorxiv.org/content/10.1101/635862v1&amp;title=Climate%20adaptation%20shaped%20by%20subtle%20to%20moderate%20allele%20frequency%20shifts%20in%20loblolly%20pine" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-biorxiv-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title="Citation Tools"><i class="icon-globe"></i> <span class="title">Citation Tools</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-biorxiv_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="732092" id="node732092--3" data-pisa="biorxiv;635862v1" data-pisa-master="biorxiv;635862" data-seqnum="732092" data-apath="/biorxiv/early/2019/05/13/635862.atom"><div  class="highwire-cite highwire-cite-highwire-article highwire-citation-biorxiv-article-pap-list clearfix" >

      <div class="highwire-cite-title" >
      <div class="highwire-cite-title">Climate adaptation shaped by subtle to moderate allele frequency shifts in loblolly pine</div>    </div>
  
      <div  class="highwire-cite-authors" ><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0"><span class="nlm-given-names">Amanda R.</span> <span class="nlm-surname">De La Torre</span></span>, <span class="highwire-citation-author" data-delta="1"><span class="nlm-given-names">David B</span> <span class="nlm-surname">Neale</span></span></span></div>
  
      <div  class="highwire-cite-metadata" ><span  class="highwire-cite-metadata-journal highwire-cite-metadata">bioRxiv </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">635862; </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata"><span class="doi_label">doi:</span> https://doi.org/10.1101/635862 </span></div>
  
  
  </div>
</div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage&amp;rft.epage&amp;rft.atitle=Climate%20adaptation%20shaped%20by%20subtle%20to%20moderate%20allele%20frequency%20shifts%20in%20loblolly%20pine&amp;rft.volume&amp;rft.issue&amp;rft.date=2019-01-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=bioRxiv&amp;rft.au=De+La+Torre%2C+Amanda+R.&amp;rft.au=Neale%2C+David+B"></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/732092/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/732092/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/732092/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/732092/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/732092/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/732092/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/732092/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/732092/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/732092/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/732092/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/732092/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/732092/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><div class="item-list"><ul><li class="first"><a href="http://twitter.com/share?url=https%3A//www.biorxiv.org/content/10.1101/635862v1&amp;count=horizontal&amp;via=&amp;text=Climate%20adaptation%20shaped%20by%20subtle%20to%20moderate%20allele%20frequency%20shifts%20in%20loblolly%20pine&amp;counturl=https%3A//www.biorxiv.org/content/10.1101/635862v1" class="twitter-share-button service-links-twitter-widget" id="twitter_widget" title="Tweet This" rel="nofollow" data-icon-position="" data-hide-link-title="0"><span class="element-invisible">Tweet Widget</span></a></li><li><a href="http://www.facebook.com/plugins/like.php?href=https%3A//www.biorxiv.org/content/10.1101/635862v1&amp;layout=button_count&amp;show_faces=false&amp;action=like&amp;colorscheme=light&amp;width=100&amp;height=21&amp;font=&amp;locale=" id="facebook_like" title="I Like it" class="service-links-facebook-like" rel="nofollow" data-icon-position="" data-hide-link-title="0"><span class="element-invisible">Facebook Like</span></a></li><li class="last"><a href="https://www.biorxiv.org/content/10.1101/635862v1" id="google_plus_one" title="Plus it" class="service-links-google-plus-one" rel="nofollow" data-icon-position="" data-hide-link-title="0"><span class="element-invisible">Google Plus One</span></a></li></ul></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-collections" >
  
        <h2 class="pane-title">Subject Area</h2>
    
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-collections"><div class="highwire-list"><ul class="highwire-article-collection-term-list"><li class="first last odd"><span class="highwire-article-collection-term"><a href="/collection/genomics" class="highlight" data-icon-position="" data-hide-link-title="0">Genomics<i class="icon-caret-right"></i>
</a></span></li></ul></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-biorxiv-subject-collections block-style-col2" >
  
      
  
  <div class="pane-content">
    <div class="panel-flexible panels-flexible-8 clearfix" id="mini-panel-biorxiv_subject_collections">
<div class="panel-flexible-inside panels-flexible-8-inside">
<div class="panels-flexible-region panels-flexible-region-8-center panels-flexible-region-first panels-flexible-region-last">
  <div class="inside panels-flexible-region-inside panels-flexible-region-8-center-inside panels-flexible-region-inside-first panels-flexible-region-inside-last">
<div class="panel-pane pane-snippet" >
  
      
  
  <div class="pane-content">
    <div class="snippet biorxiv-subject-areas-table-title" id="biorxiv-subject-areas-table-title">
  
      
  <div class="snippet-content">
    <b>Subject Areas</b>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-snippet" >
  
      
  
  <div class="pane-content">
    <div class="snippet biorxiv-subject-areas-view-papers" id="biorxiv-subject-areas-view-papers">
  
      
  <div class="snippet-content">
    <a href="/content/early/recent"><strong>All Articles</strong></a>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-subject-collections" >
  
      
  
  <div class="pane-content">
    <ul id="collection" class="collection highwire-list-expand"><li class="outer collection depth-2  child first"><div class = "data-wrapper"><a href="/collection/animal-behavior-and-cognition" class="" data-icon-position="" data-hide-link-title="0">Animal Behavior and Cognition</a> <span class = "article-count">(1662)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/biochemistry" class="" data-icon-position="" data-hide-link-title="0">Biochemistry</a> <span class = "article-count">(2789)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/bioengineering" class="" data-icon-position="" data-hide-link-title="0">Bioengineering</a> <span class = "article-count">(1946)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/bioinformatics" class="" data-icon-position="" data-hide-link-title="0">Bioinformatics</a> <span class = "article-count">(10400)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/biophysics" class="" data-icon-position="" data-hide-link-title="0">Biophysics</a> <span class = "article-count">(4243)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/cancer-biology" class="" data-icon-position="" data-hide-link-title="0">Cancer Biology</a> <span class = "article-count">(3272)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/cell-biology" class="" data-icon-position="" data-hide-link-title="0">Cell Biology</a> <span class = "article-count">(4641)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/clinical-trials" class="" data-icon-position="" data-hide-link-title="0">Clinical Trials</a> <span class = "article-count">(135)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/developmental-biology" class="" data-icon-position="" data-hide-link-title="0">Developmental Biology</a> <span class = "article-count">(2889)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/ecology" class="" data-icon-position="" data-hide-link-title="0">Ecology</a> <span class = "article-count">(4544)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/epidemiology" class="" data-icon-position="" data-hide-link-title="0">Epidemiology</a> <span class = "article-count">(2041)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/evolutionary-biology" class="" data-icon-position="" data-hide-link-title="0">Evolutionary Biology</a> <span class = "article-count">(7288)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/genetics" class="" data-icon-position="" data-hide-link-title="0">Genetics</a> <span class = "article-count">(5540)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/genomics" class="" data-icon-position="" data-hide-link-title="0">Genomics</a> <span class = "article-count">(6893)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/immunology" class="" data-icon-position="" data-hide-link-title="0">Immunology</a> <span class = "article-count">(2434)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/microbiology" class="" data-icon-position="" data-hide-link-title="0">Microbiology</a> <span class = "article-count">(7616)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/molecular-biology" class="" data-icon-position="" data-hide-link-title="0">Molecular Biology</a> <span class = "article-count">(3051)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/neuroscience" class="" data-icon-position="" data-hide-link-title="0">Neuroscience</a> <span class = "article-count">(18885)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/paleontology" class="" data-icon-position="" data-hide-link-title="0">Paleontology</a> <span class = "article-count">(142)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/pathology" class="" data-icon-position="" data-hide-link-title="0">Pathology</a> <span class = "article-count">(477)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/pharmacology-and-toxicology" class="" data-icon-position="" data-hide-link-title="0">Pharmacology and Toxicology</a> <span class = "article-count">(791)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/physiology" class="" data-icon-position="" data-hide-link-title="0">Physiology</a> <span class = "article-count">(1178)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/plant-biology" class="" data-icon-position="" data-hide-link-title="0">Plant Biology</a> <span class = "article-count">(2771)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/scientific-communication-and-education" class="" data-icon-position="" data-hide-link-title="0">Scientific Communication and Education</a> <span class = "article-count">(686)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/synthetic-biology" class="" data-icon-position="" data-hide-link-title="0">Synthetic Biology</a> <span class = "article-count">(905)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/systems-biology" class="" data-icon-position="" data-hide-link-title="0">Systems Biology</a> <span class = "article-count">(2881)</span></div></li>
<li class="outer collection depth-2  child last"><div class = "data-wrapper"><a href="/collection/zoology" class="" data-icon-position="" data-hide-link-title="0">Zoology</a> <span class = "article-count">(472)</span></div></li>
</ul>  </div>

  
  </div>
  </div>
</div>
</div>
</div>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__VNH5GD8Zz7g6_hGCOZjVjdMndxxC6naiExSSJKX3k_A__kzmEdtbqCsJx5Z9yg2Qy0PptB__ufXm-TxUOl0KZzUw__QJacVmveFyLIBSPkp21xL0ZvuDW9DMBrD2HB1P4Ry5I.js"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__QJacVmveFyLIBSPkp21xL0ZvuDW9DMBrD2HB1P4Ry5I.js" defer="defer"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__N7ERJBYsOWyRnJgyoM125_Aiez2MOJGaUofG1JdWWBg__bLlcwhzZhT-IwoQ_dvm1_xPxU7E7gecFLC6Fyeqtv8s__QJacVmveFyLIBSPkp21xL0ZvuDW9DMBrD2HB1P4Ry5I.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
function euCookieComplianceLoadScripts() {}
//--><!]]>
</script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var eu_cookie_compliance_cookie_name = "";
//--><!]]>
</script>
<script type="text/javascript" src="//d33xdlntwy0kbs.cloudfront.net/cshl_custom.js"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__BmqjBnkz3MgYCAoc25s1lDRMEjLhC3mEPVonUFIHi08__Unwv5-ZIuHBfFwytsjEx1niBVJ7n1T4lPws7VrkHXM4__QJacVmveFyLIBSPkp21xL0ZvuDW9DMBrD2HB1P4Ry5I.js"></script>
  </body>
</html>
