<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<link rel="dns-prefetch" href="//d33xdlntwy0kbs.cloudfront.net" />
<link rel="shortcut icon" href="https://www.biorxiv.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<meta name="article_thumbnail" content="https://www.biorxiv.org/content/biorxiv/early/2019/10/09/798595/embed/inline-graphic-1.gif" />
<meta name="type" content="article" />
<meta name="category" content="article" />
<meta name="HW.identifier" content="/biorxiv/early/2019/10/09/798595.atom" />
<meta name="HW.pisa" content="biorxiv;798595v1" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="Estimating the genome-wide contribution of selection to temporal allele frequency change" />
<meta name="DC.Identifier" content="10.1101/798595" />
<meta name="DC.Date" content="2019-10-09" />
<meta name="DC.Publisher" content="Cold Spring Harbor Laboratory" />
<meta name="DC.Rights" content="© 2019, Posted by Cold Spring Harbor Laboratory. This pre-print is available under a Creative Commons License (Attribution-NoDerivs 4.0 International), CC BY-ND 4.0, as described at http://creativecommons.org/licenses/by-nd/4.0/" />
<meta name="DC.AccessRights" content="restricted" />
<meta name="DC.Description" content="Rapid phenotypic adaptation is often observed in natural populations and selection experiments. However, detecting the genome-wide impact of this selection is difficult, since adaptation often proceeds from standing variation and selection on highly polygenic traits, both of which may leave faint genomic signals indistinguishable from a noisy background of genetic drift. One promising signal comes from the genome-wide covariance between allele frequency changes observable from temporal genomic data, e.g. evolve-and-resequence studies. These temporal covariances reflect how the change in neutral allele frequency at one timepoint is predictive of the changes at later timepoints when there is heritable fitness variation in the population, as neutral alleles can remain associated with selected alleles over time. Since genetic drift does not lead to temporal covariance, we can use these covariances to estimate what fraction of the variation in allele frequency change through time is driven by linked selection. Here, we reanalyze two Drosophila simulans evolve-and-resequence studies, and one artificial selection experiment in mice, to quantify the effects of linked selection over short timescales using covariance among time-points and across replicates. We estimate that at least 17% to 37% of allele frequency change is driven by selection in these experiments. Against this background of positive genome-wide temporal covariances we also identify signals of negative temporal covariance corresponding to reversals in the direction of selection for a reasonable proportion of loci over the time course of a selection experiment. Overall, we find that in the three studies we analyzed, linked selection has a large impact on short-term allele frequency dynamics that is readily distinguishable from genetic drift." />
<meta name="DC.Contributor" content="Vince Buffalo" />
<meta name="DC.Contributor" content="Graham Coop" />
<meta name="article:published_time" content="2019-10-09" />
<meta name="article:section" content="New Results" />
<meta name="citation_title" content="Estimating the genome-wide contribution of selection to temporal allele frequency change" />
<meta name="citation_abstract" lang="en" content="&lt;h3&gt;Abstract&lt;/h3&gt;
&lt;p&gt;Rapid phenotypic adaptation is often observed in natural populations and selection experiments. However, detecting the genome-wide impact of this selection is difficult, since adaptation often proceeds from standing variation and selection on highly polygenic traits, both of which may leave faint genomic signals indistinguishable from a noisy background of genetic drift. One promising signal comes from the genome-wide covariance between allele frequency changes observable from temporal genomic data, e.g. evolve-and-resequence studies. These temporal covariances reflect how the change in neutral allele frequency at one timepoint is predictive of the changes at later timepoints when there is heritable fitness variation in the population, as neutral alleles can remain associated with selected alleles over time. Since genetic drift does not lead to temporal covariance, we can use these covariances to estimate what fraction of the variation in allele frequency change through time is driven by linked selection. Here, we reanalyze two &lt;i&gt;Drosophila simulans&lt;/i&gt; evolve-and-resequence studies, and one artificial selection experiment in mice, to quantify the effects of linked selection over short timescales using covariance among time-points and across replicates. We estimate that at least 17% to 37% of allele frequency change is driven by selection in these experiments. Against this background of positive genome-wide temporal covariances we also identify signals of negative temporal covariance corresponding to reversals in the direction of selection for a reasonable proportion of loci over the time course of a selection experiment. Overall, we find that in the three studies we analyzed, linked selection has a large impact on short-term allele frequency dynamics that is readily distinguishable from genetic drift.&lt;/p&gt;" />
<meta name="citation_journal_title" content="bioRxiv" />
<meta name="citation_publisher" content="Cold Spring Harbor Laboratory" />
<meta name="citation_publication_date" content="2019/01/01" />
<meta name="citation_mjid" content="biorxiv;798595v1" />
<meta name="citation_id" content="798595v1" />
<meta name="citation_public_url" content="https://www.biorxiv.org/content/10.1101/798595v1" />
<meta name="citation_abstract_html_url" content="https://www.biorxiv.org/content/10.1101/798595v1.abstract" />
<meta name="citation_full_html_url" content="https://www.biorxiv.org/content/10.1101/798595v1.full" />
<meta name="citation_pdf_url" content="https://www.biorxiv.org/content/biorxiv/early/2019/10/09/798595.full.pdf" />
<meta name="citation_doi" content="10.1101/798595" />
<meta name="citation_num_pages" content="32" />
<meta name="citation_article_type" content="Article" />
<meta name="citation_section" content="New Results" />
<meta name="citation_firstpage" content="798595" />
<meta name="citation_author" content="Vince Buffalo" />
<meta name="citation_author_institution" content="Population Biology Graduate Group" />
<meta name="citation_author_institution" content="Center for Population Biology, Department of Evolution and Ecology, University of California" />
<meta name="citation_author_email" content="vsbuffalo@ucdavis.edu" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0003-4510-1609" />
<meta name="citation_author" content="Graham Coop" />
<meta name="citation_author_institution" content="Center for Population Biology, Department of Evolution and Ecology, University of California" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0001-8431-0302" />
<meta name="citation_reference" content="Andersen, Erik C et al. (2012). “Chromosome-scale selective sweeps shape Caenorhabditis elegans genomic diversity”. en. In: Nat. Genet.s 44.3, pp. 285–290." />
<meta name="citation_reference" content="Andolfatto, Peter (2007). “Hitchhiking effects of recurrent beneficial amino acid substitutions in the Drosophila melanogaster genome”. en. In: Genome Res. 17.12, pp. 1755–1762." />
<meta name="citation_reference" content="Baldwin-Brown, J G, A D Long, and K R Thornton (2014). “The Power to Detect Quantitative Trait Loci Using Resequenced, Experimentally Evolved Populations of Diploid, Sexual Organisms”. In: Mol. Biol. Evol. 31.4, pp. 1040–1055." />
<meta name="citation_reference" content="Barghi, Neda et al. (2019). “Genetic redundancy fuels polygenic adaptation in Drosophila”. en. In: PLoS Biol. 17.2, e3000128." />
<meta name="citation_reference" content="Barton, N H (2000). “Genetic hitchhiking”. In: Proceedings of the Royal Society of London B: Biological Sciences 355.1403, pp. 1553–1562." />
<meta name="citation_reference" content="Baym, Michael et al. (2016). “Spatiotemporal microbial evolution on antibiotic landscapes”. en. In: Science 353.6304, pp. 1147–1151." />
<meta name="citation_reference" content="Begun, David J et al. (2007). “Population genomics: whole-genome analysis of polymorphism and divergence in Drosophila simulans”. en. In: PLoS Biol. 5.11, e310." />
<meta name="citation_reference" content="Beissinger, Timothy M et al. (2016). “Recent demography drives changes in linked selection across the maize genome”. en. In: Nat Plants 2, p. 16084." />
<meta name="citation_reference" content="Benjamini, Yoav and Yosef Hochberg (1995). “Controlling the False Discovery Rate: A Practical and Powerful Approach to Multiple Testing”. In: J. R. Stat. Soc. Series B Stat. Methodol. 57.1, pp. 289–300." />
<meta name="citation_reference" content="Bennett, A F, K M Dao, and R E Lenski (1990). “Rapid evolution in response to high-temperature selection”. en. In: Nature 346.6279, pp. 79–81." />
<meta name="citation_reference" content="Bergland, Alan O, Emily L Behrman, Katherine R O’Brien, Paul S Schmidt, and Dmitri A Petrov (2014). “Genomic Evidence of Rapid and Stable Adaptive Oscillations over Seasonal Time Scales in Drosophila”. In: PLoS Genet. 10.11, e1004775–19." />
<meta name="citation_reference" content="Bhatia, Gaurav, Nick Patterson, Sriram Sankararaman, and Alkes L Price (2013). “Estimating and interpreting FST: the impact of rare variants”. en. In: Genome Res. 23.9, pp. 1514–1521." />
<meta name="citation_reference" content="Buffalo, Vince and Graham Coop (2019). “The Linked Selection Signature of Rapid Adaptation in Temporal Genomic Data”. en." />
<meta name="citation_reference" content="Burke, Molly K et al. (2010). “Genome-wide analysis of a long-term evolution experiment with Drosophila”. In: Nature 467.7315, pp. 587–590." />
<meta name="citation_reference" content="Castro, João Pl et al. (2019). “An integrative genomic analysis of the Longshanks selection experiment for longer limbs in mice”. en. In: Elife 8." />
<meta name="citation_reference" content="Charlesworth, Brian, M T Morgan, and D Charlesworth (1993). “The effect of deleterious mutations on neutral molecular variation”. In: Genetics 134.4, pp. 1289–1303." />
<meta name="citation_reference" content="Comeron, Josep M (2014). “Background selection as baseline for nucleotide variation across the Drosophila genome”. en. In: PLoS Genet. 10.6, e1004434." />
<meta name="citation_reference" content="Coop, Graham (2016). Does linked selection explain the narrow range of genetic diversity across species? Tech. rep. Cold Spring Harbor Labs Journals." />
<meta name="citation_reference" content="Corbett-Detig, Russell B, Daniel L Hartl, and Timothy B Sackton (2015). “Natural Selection Constrains Neutral Diversity across A Wide Range of Species”. In: PLoS Biol. 13.4, e1002112–25." />
<meta name="citation_reference" content="Cutter, A D and J Y Choi (2010). “Natural selection shapes nucleotide polymorphism across the genome of the nematode Caenorhabditis briggsae”. In: Genome Res. 20.8, pp. 1103–1111." />
<meta name="citation_reference" content="Cutter, Asher D and Bret A Payseur (2013). “Genomic signatures of selection at linked sites: unifying the disparity among species”. In: Nature Publishing Group 14.4, pp. 262–274." />
<meta name="citation_reference" content="Davison, A C and D V Hinkley (2013). Bootstrap Methods and Their Application. New York, NY, USA: Cambridge University Press." />
<meta name="citation_reference" content="Elyashiv, Eyal et al. (2016). “A Genomic Map of the Effects of Linked Selection in Drosophila”. In: PLoS Genet. 12.8, e1006130–24." />
<meta name="citation_reference" content="Franks, Steven J, Sheina Sim, and Arthur E Weis (2007). “Rapid evolution of flowering time by an annual plant in response to a climate fluctuation”. In: Proceedings of the National Academy of Sciences 104.4, pp. 1278–1282." />
<meta name="citation_reference" content="Good, Benjamin H, Michael J McDonald, Jeffrey E Barrick, Richard E Lenski, and Michael M Desai (2017). “The dynamics of molecular evolution over 60,000 generations”. en. In: Nature 551.7678, pp. 45–50." />
<meta name="citation_reference" content="Grant, Peter R and B Rosemary Grant (2006). “Evolution of Character Displacement in Darwin’s Finches”. In: Science 313.5784, pp. 224–226." />
<meta name="citation_reference" content="Grant, Peter R and B Rosemary Grant (2011). “Causes of lifetime fitness of Darwin’s finches in a fluctuating environment”. en. In: Proc. Natl. Acad. Sci. U. S. A. 108.2, pp. 674–679." />
<meta name="citation_reference" content="Hairston Jr, Nelson G, Stephen P Ellner, Monica A Geber, Takehito Yoshida, and Jennifer A Fox (2005). “Rapid evolution and the convergence of ecological and evolutionary time”. In: Ecol. Lett. 8.10, pp. 1114–1127." />
<meta name="citation_reference" content="Hendry, A P and M T Kinnison (1999). “Perspective: the pace of modern life: measuring rates of contemporary microevolution”. In: Evolution 53.6, p. 1637." />
<meta name="citation_reference" content="Hernandez, R D, J L Kelley, E Elyashiv, and S C Melton (2011). “Classic selective sweeps were rare in recent human evolution”. In: Science." />
<meta name="citation_reference" content="Howie, James Malcolm, Rupert Mazzucco, Thomas Taus, Viola Nolte, and Christian Schlotterer (2018). “DNA motifs are not general predictors of recombination in two Drosophila sister species”. en." />
<meta name="citation_reference" content="Jónás, Á, T Taus, C Kosiol, and C Schlötterer (2016). “Estimating the Effective Population Size from Temporal Allele Frequency Changes in Experimental Evolution”. In: Genetics." />
<meta name="citation_reference" content="Kelly, John K and Kimberly A Hughes (2019). “Pervasive Linked Selection and Intermediate-Frequency Alleles Are Implicated in an Evolve-and-Resequencing Experiment of Drosophila simulans”. en. In: Genetics 211.3, pp. 943–961." />
<meta name="citation_reference" content="Kolaczkowski, Bryan, Andrew D Kern, Alisha K Holloway, and David J Begun (2011). “Genomic differentiation between temperate and tropical Australian populations of Drosophila melanogaster”. en. In: Genetics 187.1, pp. 245–260." />
<meta name="citation_reference" content="Machado, Heather E et al. (2018). “Broad geographic sampling reveals predictable and pervasive seasonal adaptation in Drosophila”. en." />
<meta name="citation_reference" content="Macpherson, J Michael, Guy Sella, Jerel C Davis, and Dmitri A Petrov (2007). “Genomewide spatial correspondence between nonsynonymous divergence and neutral polymorphism reveals extensive adaptation in Drosophila”. en. In: Genetics 177.4, pp. 2083–2099." />
<meta name="citation_reference" content="Marchini, Marta et al. (2014). “Impacts of genetic correlation on the independent evolution of body mass and skeletal size in mammals”. en. In: BMC Evol. Biol. 14, p. 258." />
<meta name="citation_reference" content="Maynard Smith, John and John Haigh (1974). “The hitch-hiking effect of a favourable gene”. In: Genet. Res. 23.01, pp. 23–35." />
<meta name="citation_reference" content="McVicker, Graham, David Gordon, Colleen Davis, and Phil Green (2009). “Widespread Genomic Signatures of Natural Selection in Hominid Evolution”. In: PLoS Genet. 5.5, e1000471–16." />
<meta name="citation_reference" content="Nei, Masatoshi and Fumio Tajima (1981). “Genetic Drift And Estimation Of Effective Population Size”. In: Genetics 98.3, pp. 625–640." />
<meta name="citation_reference" content="Nordborg, Magnus, Brian Charlesworth, and Deborah Charlesworth (1996). “The effect of recombination on background selection*”. In: Genet. Res. 67.02, pp. 159–174." />
<meta name="citation_reference" content="Nordborg, Magnus et al. (2005). “The Pattern of Polymorphism in Arabidopsis thaliana”. In: PLoS Biol. 3.7, e196–11." />
<meta name="citation_reference" content="Nuzhdin, Sergey V and Thomas L Turner (2013). “Promises and limitations of hitchhiking mapping”. en. In: Curr. Opin. Genet. Dev. 23.6, pp. 694–699." />
<meta name="citation_reference" content="Reznick, David N, Frank H Shaw, F Helen Rodd, and Ruth G Shaw (1997). “Evaluation of the Rate of Evolution in Natural Populations of Guppies (Poecilia reticulata)”. In: Science 275.5308, pp. 1934–1937." />
<meta name="citation_reference" content="Robertson, Alan (1961). “Inbreeding in artificial selection programmes”. In: Genet. Res. 2.02, pp. 189–194." />
<meta name="citation_reference" content="Santiago, Enrique and Armando Caballero (1995). “Effective size of populations under selection”. In: Genetics 139.2, pp. 1013–1030." />
<meta name="citation_reference" content="Santiago, Enrique and Armando Caballero (1998). “Effective Size and Polymorphism of Linked Neutral Loci in Populations Under Directional Selection”. In: Genetics 149.4, pp. 2105–2117." />
<meta name="citation_reference" content="Sattath, Shmuel, Eyal Elyashiv, Oren Kolodny, Yosef Rinott, and Guy Sella (2011). “Pervasive adaptive protein evolution apparent in diversity patterns around amino acid substitutions in Drosophila simulans”. en. In: PLoS Genet. 7.2, e1001302." />
<meta name="citation_reference" content="Sella, Guy, Dmitri A Petrov, Molly Przeworski, and Peter Andolfatto (2009). “Pervasive natural selection in the Drosophila genome?” en. In: PLoS Genet. 5.6, e1000495." />
<meta name="citation_reference" content="Signor, Sarah A, Felicia N New, and Sergey Nuzhdin (2018). “A Large Panel of Drosophila simulans Reveals an Abundance of Common Variants”. en. In: Genome Biol. Evol. 10.1, pp. 189–206." />
<meta name="citation_reference" content="Therkildsen, Nina O et al. (2019). “Contrasting genomic shifts underlie parallel phenotypic evolution in response to fishing”. en. In: Science 365.6452, pp. 487–490." />
<meta name="citation_reference" content="Turner, T L and P M Miller (2012). “Investigating Natural Variation in Drosophila Courtship Song by the Evolve and Resequence Approach”. In: Genetics 191.2, pp. 633–642." />
<meta name="citation_reference" content="Turner, Thomas L, Andrew D Stewart, Andrew T Fields, William R Rice, and Aaron M Tarone (2011). “Population-Based Resequencing of Experimentally Evolved Populations Reveals the Genetic Basis of Body Size Variation in Drosophila melanogaster”. In: PLoS Genet. 7.3, e1001336–10." />
<meta name="citation_reference" content="Waples, R S (1989). “A generalized approach for estimating effective population size from temporal changes in allele frequency”. In: Genetics 121.2, pp. 379–391." />
<meta name="citation_reference" content="Wasserman, Larry (2006). All of Nonparametric Statistics. en. Springer Science &amp; Business Media." />
<meta name="citation_reference" content="Williamson, Robert J et al. (2014). “Evidence for widespread positive and negative selection in coding and conserved noncoding regions of Capsella grandiflora”. en. In: PLoS Genet. 10.9, e1004622." />
<meta name="citation_reference" content="Woolliams, J A, N R Wray, and R Thompson (1993). “Prediction of long-term contributions and inbreeding in populations undergoing mass selection”. In: Genet. Res. 62.3, pp. 231–242." />
<meta name="citation_reference" content="Wray, Naomi R and Robin Thompson (1990). “Prediction of rates of inbreeding in selected populations”. In: Genet. Res. 55.01, pp. 41–54." />
<meta name="twitter:title" content="Estimating the genome-wide contribution of selection to temporal allele frequency change" />
<meta name="twitter:site" content="@biorxivpreprint" />
<meta name="twitter:card" content="summary" />
<meta name="twitter:image" content="https://www.biorxiv.org/sites/default/files/images/biorxiv_logo_homepage7-5-small.png" />
<meta name="twitter:description" content="Rapid phenotypic adaptation is often observed in natural populations and selection experiments. However, detecting the genome-wide impact of this selection is difficult, since adaptation often proceeds from standing variation and selection on highly polygenic traits, both of which may leave faint genomic signals indistinguishable from a noisy background of genetic drift. One promising signal comes from the genome-wide covariance between allele frequency changes observable from temporal genomic data, e.g. evolve-and-resequence studies. These temporal covariances reflect how the change in neutral allele frequency at one timepoint is predictive of the changes at later timepoints when there is heritable fitness variation in the population, as neutral alleles can remain associated with selected alleles over time. Since genetic drift does not lead to temporal covariance, we can use these covariances to estimate what fraction of the variation in allele frequency change through time is driven by linked selection. Here, we reanalyze two Drosophila simulans evolve-and-resequence studies, and one artificial selection experiment in mice, to quantify the effects of linked selection over short timescales using covariance among time-points and across replicates. We estimate that at least 17% to 37% of allele frequency change is driven by selection in these experiments. Against this background of positive genome-wide temporal covariances we also identify signals of negative temporal covariance corresponding to reversals in the direction of selection for a reasonable proportion of loci over the time course of a selection experiment. Overall, we find that in the three studies we analyzed, linked selection has a large impact on short-term allele frequency dynamics that is readily distinguishable from genetic drift." />
<meta name="og-title" property="og:title" content="Estimating the genome-wide contribution of selection to temporal allele frequency change" />
<meta name="og-url" property="og:url" content="https://www.biorxiv.org/content/10.1101/798595v1" />
<meta name="og-site-name" property="og:site_name" content="bioRxiv" />
<meta name="og-description" property="og:description" content="Rapid phenotypic adaptation is often observed in natural populations and selection experiments. However, detecting the genome-wide impact of this selection is difficult, since adaptation often proceeds from standing variation and selection on highly polygenic traits, both of which may leave faint genomic signals indistinguishable from a noisy background of genetic drift. One promising signal comes from the genome-wide covariance between allele frequency changes observable from temporal genomic data, e.g. evolve-and-resequence studies. These temporal covariances reflect how the change in neutral allele frequency at one timepoint is predictive of the changes at later timepoints when there is heritable fitness variation in the population, as neutral alleles can remain associated with selected alleles over time. Since genetic drift does not lead to temporal covariance, we can use these covariances to estimate what fraction of the variation in allele frequency change through time is driven by linked selection. Here, we reanalyze two Drosophila simulans evolve-and-resequence studies, and one artificial selection experiment in mice, to quantify the effects of linked selection over short timescales using covariance among time-points and across replicates. We estimate that at least 17% to 37% of allele frequency change is driven by selection in these experiments. Against this background of positive genome-wide temporal covariances we also identify signals of negative temporal covariance corresponding to reversals in the direction of selection for a reasonable proportion of loci over the time course of a selection experiment. Overall, we find that in the three studies we analyzed, linked selection has a large impact on short-term allele frequency dynamics that is readily distinguishable from genetic drift." />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.biorxiv.org/sites/default/files/images/biorxiv_logo_homepage7-5-small.png" />
<meta name="citation_date" content="2019-10-09" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/10.1101/798595v1.full.pdf" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/10.1101/798595v1.full.txt" />
<link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/10.1101/798595v1.ppt" />
<meta name="description" content="bioRxiv - the preprint server for biology, operated by Cold Spring Harbor Laboratory, a research and educational institution" />
<meta name="generator" content="Drupal 7 (http://drupal.org)" />
<link rel="canonical" href="https://www.biorxiv.org/content/10.1101/798595v1" />
<link rel="shortlink" href="https://www.biorxiv.org/node/942517" />
    <title>Estimating the genome-wide contribution of selection to temporal allele frequency change | bioRxiv</title>  
    <link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__jMRAK66KMC1e4TQlwUNn3KiWVDC5AjueUAYEm1xBY_U__KSUjdT4jdcJ4qJz6fKY1K9WkYh1a5EcaWZxt_-zbTis__QJacVmveFyLIBSPkp21xL0ZvuDW9DMBrD2HB1P4Ry5I.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q5p1rr" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__icrP-Y0qw2NP73rJxxkUtrJta0z3VLAABtNjfSdEghA__rlN3rx0wOoswGNHZMeZM96bNn-3cLWHXTztGDoThNlw__QJacVmveFyLIBSPkp21xL0ZvuDW9DMBrD2HB1P4Ry5I.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__q48SDxVDWut_cH6dTPyQe7HjADuSLLmJVFztuiJEFrU__DJqddQGjm5dXCHsaiwNriGytKN5eevIQqyjdq65TZtE__QJacVmveFyLIBSPkp21xL0ZvuDW9DMBrD2HB1P4Ry5I.css" media="all" />
<style type="text/css" media="all">
/* <![CDATA[ */
#sliding-popup.sliding-popup-bottom,#sliding-popup.sliding-popup-bottom .eu-cookie-withdraw-banner,.eu-cookie-withdraw-tab{background:gray}#sliding-popup.sliding-popup-bottom.eu-cookie-withdraw-wrapper{background:transparent}#sliding-popup .popup-content #popup-text h1,#sliding-popup .popup-content #popup-text h2,#sliding-popup .popup-content #popup-text h3,#sliding-popup .popup-content #popup-text p,.eu-cookie-compliance-secondary-button,.eu-cookie-withdraw-tab{color:#fff !important}.eu-cookie-withdraw-tab{border-color:#fff}.eu-cookie-compliance-more-button{color:#fff !important}
/* ]]> */
</style>

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__QJacVmveFyLIBSPkp21xL0ZvuDW9DMBrD2HB1P4Ry5I.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__B6tGdsQSt1vT8d8j-19-oS4FKJe83RrSNBkjPs9jfos__6VkE5WcQmmQpSVTQx-xS2-zn_nlwFlDm-EN1HLJNjrA__QJacVmveFyLIBSPkp21xL0ZvuDW9DMBrD2HB1P4Ry5I.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__I6pBwZrmO51afTZVOjN83FdGH1QRb9t15lSLso0JVV4__F3sVKlGyHKCaAPI_WdiHD7l_-05B6b9bLdUwzi1me90__QJacVmveFyLIBSPkp21xL0ZvuDW9DMBrD2HB1P4Ry5I.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__QJacVmveFyLIBSPkp21xL0ZvuDW9DMBrD2HB1P4Ry5I.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__QJacVmveFyLIBSPkp21xL0ZvuDW9DMBrD2HB1P4Ry5I.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__CRdjIL0HKDL2T9slYvkTSTLypBBPjyoDhwBezBuDy7c__cHebPmPNdg2jyPmpyQpZWssJQ4cPrqfaabp5Or9KlkU__QJacVmveFyLIBSPkp21xL0ZvuDW9DMBrD2HB1P4Ry5I.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://d33xdlntwy0kbs.cloudfront.net/cshl_custom.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/all/libraries/modernizr/modernizr.min.js?q5p1rr"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__yOhrJUBmMpN-jsuQlJYmS8ovkQU6I6KInr295HrSTb0__c--zHhbuOWmkdWkJYXiNWDmkpDRPesmbBjGlQcyqBOY__QJacVmveFyLIBSPkp21xL0ZvuDW9DMBrD2HB1P4Ry5I.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var _prum=[['id', '52e95df3abe53d2e33000000'], ['mark', 'firstbyte',
      (new Date()).getTime()]]; (function() {
      var s=document.getElementsByTagName('script')[0],
      p=document.createElement('script');
      p.async='async'; p.src='//rum-static.pingdom.net/prum.min.js';s.parentNode.insertBefore(p,s);})();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js___KyvuEvfmewWxdOLBXTYsSUo8Pfm_DijV6fWtGvSL_k__HSKkTgIr7nDcdvQYewipf6v4ak6ZLUOQkwPj44wKouA__QJacVmveFyLIBSPkp21xL0ZvuDW9DMBrD2HB1P4Ry5I.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__3uCzzYE5jr6ly9nvsM8PRBRkhh4p0NWCMDja4EKBy4M__dmGGVYcB8LhnbR2iAmVW0ZyXI6QuFVYFRORa5XxglXs__QJacVmveFyLIBSPkp21xL0ZvuDW9DMBrD2HB1P4Ry5I.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__0zdUME0GLauYYyEch2tGPydIw-IG7Uizip0Qb4wUsig__3RgszrCgk4bOnTzo4vaQ-hrWzaG6oD19l-R9CQ1YQ-o__QJacVmveFyLIBSPkp21xL0ZvuDW9DMBrD2HB1P4Ry5I.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__W6wAjopaALytcBam7oT3WhEzllkWaB9R-8hazHyvShM__rK5YBIqBy9qt0di0Sn4DriBx9TJupdmv6uGmLJyZvDs__QJacVmveFyLIBSPkp21xL0ZvuDW9DMBrD2HB1P4Ry5I.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-45638026-1", {"cookieDomain":"auto"});ga("set", "page", location.pathname + location.search + location.hash);ga("send", "pageview");ga('create', 'UA-189672-38', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__1PtAOZ8x7l6QAi8D7X6Kj_Y7HcUOFzvWRvvYHUcCT98__U8IB92u_CnnGwJ-X_xaUa9lzsC8OzUfN0WkK2JQfirg__QJacVmveFyLIBSPkp21xL0ZvuDW9DMBrD2HB1P4Ry5I.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings,{"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"hi2Tq9VMD34lIGfxDqTMZ_Jz-jjsG4e97mw2ohi9r7g","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"modules\/comment\/comment.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"modules\/forum\/forum.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"sites\/default\/modules\/biorxiv\/css\/biorxiv.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/flexible\/flexible.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_collection\/css\/highwire_subject_collection.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire.highwire-list-expand.css":1,"public:\/\/ctools\/css\/bab59090150caf0c64e27afa1b36eef1.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/contrib\/eu_cookie_compliance\/css\/eu_cookie_compliance.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"0":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_biorxiv_styles\/css\/biorxiv-styles.css.less":1,"https:\/\/d33xdlntwy0kbs.cloudfront.net\/cshl_custom.css":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_figures.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_tables.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"http:\/\/d33xdlntwy0kbs.cloudfront.net\/cshl_custom.js":1,"sites\/all\/modules\/contrib\/eu_cookie_compliance\/js\/eu_cookie_compliance.js":1,"sites\/default\/modules\/jnl_biorxiv_styles\/js\/adv-search-collection-toggle.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/eu_cookie_compliance\/js\/jquery.cookie-1.4.1.min.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/default\/modules\/jnl_biorxiv_styles\/js\/biorxiv-scripts.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_nps_survey\/js\/nps_survey.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"sites\/default\/modules\/contrib\/disqus\/disqus.js":1,"misc\/textarea.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/contrib\/service_links\/js\/twitter_button.js":1,"sites\/all\/modules\/contrib\/service_links\/js\/facebook_like.js":1,"sites\/all\/modules\/contrib\/service_links\/js\/google_plus_one.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.highwire-list-expand.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"942517","apath":"\/biorxiv\/early\/2019\/10\/09\/798595.atom","pisa":"biorxiv;798595v1","processed":["highwire_math"],"markup":[{"requested":"full-text","variant":"full-text","view":"full","pisa":"biorxiv;798595v1"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"disqus":{"domain":"biorxivstage","url":"https:\/\/www.biorxiv.org\/content\/10.1101\/798595v1","title":"Estimating the genome-wide contribution of selection to temporal allele frequency change","identifier":"node\/942517"},"panels_ajax_pane":{"new-41":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022lWawaCt3fFq6JMp8MWrj2hEI\\\\\\\/BxYdRSG4EWz7dTH6E\\\\\\\/EYlOhCx6W+gxwQ4f83qMlu8KSrzqISJTApmdpXzC1dqoIaMVuNaXWFmXR4cARhELyzSEJrIK78QcZkjeJgqFl9cYyB4lmJ0JpeGlTwVr\\\\\\\/Pz3jHYeTzRTaC8ktmNKPZTdhIRA6kyP32JR2Poae1LHarRE\\\\\\\/5ShLb+FLWucvfi1TrXXYXIy6DKPSpYLGaX+u9ErD+BSVNi8T9KF10LVab\\\\\\\/ZXhIvRDAFEcWttNJ8Uo0dM2Ej9JqdcMZ4hc+OwL1hjBQWFAi4r7moLk8lMnvFyczNX96DzZNDSO6UGwQaJenBf7qHYMbQyyqoKK80AtTNA1d9RUemmg9liVnFQKe0OV25HzFGNy\\\\\\\/S62rockCRdHEuys+FdqP8eW5k95PWOun6iFu7mbQVjd+mWt2MjIegvIy0vS\\\\\\\/aTo7rMeLV1RRq6imJTv5vsqBe0cv3a87DLmzoXB4D8uE7NkxzyCQrtb0A\\\\\\\/2r4V9mCToI7qkQZpm1mCvkrHtzmLlxEoR7O7g9UiOhVy9lUXULtPpGTnzH45QB0w2+slONT3uRljXUoyGZzi\\\\\\\/bniheMvnwgRLXmXJ1uOa314dvIARb8BbTczCguujYLqfQxVHhvf57bFuh49e62qjO5lGzwTD32RUiCDqdwCkjh691fir+yol2kd47WENnfUKIcOb5Ng\\\\\\\/EjYfrcd\\\\\\\/mjZKG6sluJZlXVGjT+49RzX98p\\\\\\\/QjL7P7NuU588P2QHY2zHASjMd3dTtL+zrC272j115dRrJxDZGzUJHMQLG5J+F1nCGhg7fbNvP7ZRpqlyYLFJb+cQBEk0ffv45g9Pc11CPr18ylJ8h0gMiR7x+Uhlp\\\\\\\/QZoK2svEHRrur5sWMKEFrA+x\\\\\\\/jbhqfI1M5N3j1MjM55mRWhiu+Ni+TAVJjyvjCsnZcUafl4K76dTn+9C9nISJ3B6WyHpgyjDGrRIbj2V+uvlUJykeAE6EXLBEjsE9p4UyO+kH1pM6zz4EOrZ\\\\\\\/2AxoogVuGJ+4m10u\\\\\\\/hbhyE9C9EKx5I2sBxSC6IJVYy70NsdgWnvxINBN99PPRKmZMsbOufpFihVhIvh3r\\\\\\\/2Y9+W\\\\\\\/URUKZ3xw\\\\\\\/ABXgwORe1xBYmEvt6QWJKRPgRIh7uvEafl00f3xgfYSsSt\\\\\\\/fq4PBSLTXhgbAz0KMOmb0hUlnwTFRi53lE4hoSrfr4UNRlt1HIz5e3SjUhG2bac\\\\\\\/\\\\\\\/WeVkrkdtCrOVoqBDAv9VLGbevTyRwjnLySgTa00Y2iGb3u9aM97+WBBHjPjGsqpamGXvMMskTht25zqIBkufJMS+XBbS\\\\\\\/tYmgLY=\\\u0022,\\\u0022iv\\\u0022:\\\u0022gmCbMs0KzENkUVCKulRRpQ==\\\u0022,\\\u0022salt\\\u0022:\\\u0022f9d8845712f72c9fee5db7d9ff111eab\\\u0022}\u0022,\u0022hmac\u0022:\u00227251866d3ad2b9ac068b2857d1414ad497177457581a15fcf80c75f20002f2ea\u0022}"},"urlIsAjaxTrusted":{"\/content\/10.1101\/798595v1.full":true},"ws_fl":{"width":100,"height":21},"ws_gpo":{"size":"","annotation":"","lang":"","callback":"","width":300},"color":{"logo":"https:\/\/www.biorxiv.org\/sites\/default\/files\/bioRxiv_article.jpg"},"highwire_list_expand":{"is_collapsed":"1"},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"eu_cookie_compliance":{"popup_enabled":1,"popup_agreed_enabled":0,"popup_hide_agreed":0,"popup_clicking_confirmation":1,"popup_scrolling_confirmation":false,"popup_html_info":"\u003Cdiv\u003E\n  \u003Cdiv class =\u0022popup-content info\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n      \u003Cp\u003EWe use cookies on this site to enhance your user experience. By clicking any link on this page you are giving your consent for us to set cookies.\u003C\/p\u003E\n    \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022agree-button eu-cookie-compliance-default-button\u0022\u003EContinue\u003C\/button\u003E\n                    \u003Cbutton type=\u0022button\u0022 class=\u0022find-more-button eu-cookie-compliance-more-button\u0022\u003EFind out more\u003C\/button\u003E\n          \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E","use_mobile_message":false,"mobile_popup_html_info":"\u003Cdiv\u003E\n  \u003Cdiv class =\u0022popup-content info\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n          \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022agree-button eu-cookie-compliance-default-button\u0022\u003EContinue\u003C\/button\u003E\n                    \u003Cbutton type=\u0022button\u0022 class=\u0022find-more-button eu-cookie-compliance-more-button\u0022\u003EFind out more\u003C\/button\u003E\n          \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E\n","mobile_breakpoint":"768","popup_html_agreed":"\u003Cdiv\u003E\n  \u003Cdiv class=\u0022popup-content agreed\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n      \u003Ch2\u003EThank you for accepting cookies\u003C\/h2\u003E\u003Cp\u003EYou can now hide this message or find out more about cookies.\u003C\/p\u003E    \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022hide-popup-button eu-cookie-compliance-hide-button\u0022\u003EHide\u003C\/button\u003E\n              \u003Cbutton type=\u0022button\u0022 class=\u0022find-more-button eu-cookie-compliance-more-button-thank-you\u0022 \u003EMore info\u003C\/button\u003E\n          \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E","popup_use_bare_css":false,"popup_height":"auto","popup_width":"100%","popup_delay":1000,"popup_link":"\/help\/cookie-policy","popup_link_new_window":1,"popup_position":null,"popup_language":"en","store_consent":false,"better_support_for_screen_readers":0,"reload_page":0,"domain":"","popup_eu_only_js":0,"cookie_lifetime":"365","cookie_session":false,"disagree_do_not_show_popup":0,"method":"default","whitelisted_cookies":"","withdraw_markup":"\u003Cbutton type=\u0022button\u0022 class=\u0022eu-cookie-withdraw-tab\u0022\u003EPrivacy settings\u003C\/button\u003E\n\u003Cdiv class=\u0022eu-cookie-withdraw-banner\u0022\u003E\n  \u003Cdiv class=\u0022popup-content info\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n      \u003Cp\u003E\u0026lt;h2\u0026gt;We use cookies on this site to enhance your user experience\u0026lt;\/h2\u0026gt;\u0026lt;p\u0026gt;You have given your consent for us to set cookies.\u0026lt;\/p\u0026gt;\u003C\/p\u003E\n    \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022eu-cookie-withdraw-button\u0022\u003EWithdraw consent\u003C\/button\u003E\n    \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E\n","withdraw_enabled":false},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1,"trackUrlFragments":1},"jnl_biorxiv_styles":{"defaultJCode":"biorxiv"},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-942517 node-type-highwire-article context-content hw-default-jcode-biorxiv hw-article-type-article hw-article-category-new-results">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-M677548" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-M677548');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
        <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
    
  <div id="zone-branding" class="zone zone-branding clearfix print-display-block container-30">
    <div class="grid-15 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="bioRxiv" src="https://www.biorxiv.org/sites/default/files/bioRxiv_article.jpg" /></a>      </div>
                </div>
          </div>
</div><div class="grid-11 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-system block-menu block-main-menu block-system-main-menu odd block-without-title" id="block-system-main-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/" title="" class="" data-icon-position="" data-hide-link-title="0">Home</a></li>
<li class="leaf" role="menuitem"><a href="/about-biorxiv" class="" data-icon-position="" data-hide-link-title="0">About</a></li>
<li class="leaf" role="menuitem"><a href="/submit-a-manuscript" class="" data-icon-position="" data-hide-link-title="0">Submit</a></li>
<li class="last leaf" role="menuitem"><a href="/content/alertsrss" title="" class="" data-icon-position="" data-hide-link-title="0">ALERTS / RSS</a></li>
</ul></nav>    </div>
  </div>
</div><div class="block block-panels-mini block-biorxiv-search-box block-panels-mini-biorxiv-search-box even block-without-title" id="block-panels-mini-biorxiv-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-biorxiv_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/10.1101/798595v1.full" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--2" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-bsRGP550dJaVeZCfCiZqFyw1JhUPHD57xOKB0emPIfY" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2 advanced-search-link" >
  
      
  
  <div class="pane-content">
    <a href="/search">Advanced Search</a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix container-30">
  	      </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="942517" id="node942517" data-pisa="biorxiv;798595v1" data-pisa-master="biorxiv;798595" data-apath="/biorxiv/early/2019/10/09/798595.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div  class="highwire-cite highwire-cite-highwire-article highwire-citation-biorxiv-article-top clearfix has-author-tooltip" >

      <span class="biorxiv-article-type">
        New Results    </span>
  
        <h1 class="highwire-cite-title" id="page-title">Estimating the genome-wide contribution of selection to temporal allele frequency change</h1>  
      <div  class="highwire-cite-authors" ><span  class="highwire-citation-authors"><span class="highwire-citation-author first hw-author-orcid-logo-wrapper" data-delta="0"><a href="http://orcid.org/0000-0003-4510-1609" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a><span class="nlm-given-names">Vince</span> <span class="nlm-surname">Buffalo</span></span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="1"><a href="http://orcid.org/0000-0001-8431-0302" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a><span class="nlm-given-names">Graham</span> <span class="nlm-surname">Coop</span></span></span></div>
  
      <div  class="highwire-cite-metadata" ><span  class="highwire-cite-metadata-doi highwire-cite-metadata"><span class="label">doi:</span> https://doi.org/10.1101/798595 </span></div>
  
  
  </div>
<div id="hw-article-author-popups-node942517" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Vince Buffalo </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-institution'>Population Biology Graduate Group</span></div><div class='author-affiliation'><span class='nlm-institution'>Center for Population Biology, Department of Evolution and Ecology, University of California</span>, Davis, CA 95616</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Vince%2BBuffalo%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Buffalo%20V&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AVince%2BBuffalo%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0003-4510-1609" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Vince Buffalo</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:vsbuffalo@ucdavis.edu" class="" data-icon-position="" data-hide-link-title="0">vsbuffalo@ucdavis.edu</a></span></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Graham Coop </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-institution'>Center for Population Biology, Department of Evolution and Ecology, University of California</span>, Davis, CA 95616</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Graham%2BCoop%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Coop%20G&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AGraham%2BCoop%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0001-8431-0302" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Graham Coop</a></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/10.1101/798595v1" class="panels-ajax-tab-tab" data-panel-name="biorxiv_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:942517" data-trigger="" data-url-enabled="1">Abstract</a><a href="/panels_ajax_tab/biorxiv_tab_art/node:942517/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/10.1101/798595v1.full-text" class="panels-ajax-tab-tab" data-panel-name="article_tab_full_text" data-target-id="highwire_article_tabs" data-entity-context="node:942517" data-trigger="full-text" data-url-enabled="1">Full Text</a><a href="/panels_ajax_tab/article_tab_full_text/node:942517/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/10.1101/798595v1.article-info" class="panels-ajax-tab-tab" data-panel-name="biorxiv_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:942517" data-trigger="article-info" data-url-enabled="1">Info/History</a><a href="/panels_ajax_tab/biorxiv_tab_info/node:942517/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/10.1101/798595v1.article-metrics" class="panels-ajax-tab-tab" data-panel-name="article_tab_metrics" data-target-id="highwire_article_tabs" data-entity-context="node:942517" data-trigger="article-metrics" data-url-enabled="1">Metrics</a><a href="/panels_ajax_tab/article_tab_metrics/node:942517/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/10.1101/798595v1.full.pdf+html" class="panels-ajax-tab-tab" data-panel-name="biorxiv_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:942517" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i> Preview PDF</a><a href="/panels_ajax_tab/biorxiv_tab_pdf/node:942517/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="article_tab_full_text" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.biorxiv.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-article_tab_full_text"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-markup" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2 class="">Abstract</h2><p id="p-2">Rapid phenotypic adaptation is often observed in natural populations and selection experiments. However, detecting the genome-wide impact of this selection is difficult, since adaptation often proceeds from standing variation and selection on highly polygenic traits, both of which may leave faint genomic signals indistinguishable from a noisy background of genetic drift. One promising signal comes from the genome-wide covariance between allele frequency changes observable from temporal genomic data, e.g. evolve-and-resequence studies. These temporal covariances reflect how the change in neutral allele frequency at one timepoint is predictive of the changes at later timepoints when there is heritable fitness variation in the population, as neutral alleles can remain associated with selected alleles over time. Since genetic drift does not lead to temporal covariance, we can use these covariances to estimate what fraction of the variation in allele frequency change through time is driven by linked selection. Here, we reanalyze two <em>Drosophila simulans</em> evolve-and-resequence studies, and one artificial selection experiment in mice, to quantify the effects of linked selection over short timescales using covariance among time-points and across replicates. We estimate that at least 17% to 37% of allele frequency change is driven by selection in these experiments. Against this background of positive genome-wide temporal covariances we also identify signals of negative temporal covariance corresponding to reversals in the direction of selection for a reasonable proportion of loci over the time course of a selection experiment. Overall, we find that in the three studies we analyzed, linked selection has a large impact on short-term allele frequency dynamics that is readily distinguishable from genetic drift.</p></div><div class="section" id="sec-1"><h2 class="">1 Introduction</h2><p id="p-3">A long-standing problem in evolutionary genetics is quantifying the roles of genetic drift and selection in shaping genome-wide allele frequency changes. Selection can both directly and indirectly affect allele frequencies, with the indirect effect coming from the action of selection on correlated loci elsewhere in genome e.g. linked selection (Maynard Smith and Haigh 1974, <a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">Charlesworth et al. 1993</a>; <a id="xref-ref-41-1" class="xref-bibr" href="#ref-41">Nordborg et al. 1996</a>; see <a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">Barton 2000</a> for a review). Previous work on this question has mostly focused on teasing apart the impacts of drift and selection on genome-wide diversity using population samples from a single contemporary timepoint, often by modeling the correlation between regional recombination rate, gene density, and diversity created in the presence of linked selection (<a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">Cutter and Payseur 2013</a>; <a id="xref-ref-49-1" class="xref-bibr" href="#ref-49">Sella et al. 2009</a>). This approach has shown linked selection has a major role in shaping patterns of genome-wide diversity across the genomes of a range of sexual species (<a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">Andersen et al. 2012</a>; <a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">Andolfatto 2007</a>; <a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">Begun et al. 2007</a>; <a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">Beissinger et al. 2016</a>; <a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">Cutter and Choi 2010</a>; <a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">Elyashiv et al. 2016</a>; <a id="xref-ref-36-1" class="xref-bibr" href="#ref-36">Macpherson et al. 2007</a>; <a id="xref-ref-48-1" class="xref-bibr" href="#ref-48">Sattath et al. 2011</a>; <a id="xref-ref-56-1" class="xref-bibr" href="#ref-56">Williamson et al. 2014</a>), and has allowed us to quantify the relative influence of positive selection (hitchhiking) and negative selection (background selection; <a id="xref-ref-2-2" class="xref-bibr" href="#ref-2">Andolfatto 2007</a>; <a id="xref-ref-23-2" class="xref-bibr" href="#ref-23">Elyashiv et al. 2016</a>; <a id="xref-ref-30-1" class="xref-bibr" href="#ref-30">Hernandez et al. 2011</a>; <a id="xref-ref-36-2" class="xref-bibr" href="#ref-36">Macpherson et al. 2007</a>; <a id="xref-ref-39-1" class="xref-bibr" href="#ref-39">McVicker et al. 2009</a>; <a id="xref-ref-42-1" class="xref-bibr" href="#ref-42">Nordborg et al. 2005</a>). However, we lack an understanding of how genome-wide linked selection acts over time.</p><p id="p-4">There are numerous examples of rapid phenotypic adaptation (<a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">Franks et al. 2007</a>; <a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">Grant and Grant 2006</a>, <a id="xref-ref-27-1" class="xref-bibr" href="#ref-27">2011</a>; <a id="xref-ref-44-1" class="xref-bibr" href="#ref-44">Reznick et al. 1997</a>) and rapid, selection-driven genomic evolution in asexual populations (<a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">Baym et al. 2016</a>; <a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">Bennett et al. 1990</a>; <a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">Good et al. 2017</a>). Yet the polygenic nature of fitness makes detecting the impact of selection on genome-wide variation over short timescales in sexual populations remarkably difficult. This is because the effect of selection on a polygenic trait (such as fitness) is distributed across loci in proportion to their effect sizes. This can lead to subtle allele frequency shifts on standing variation that are difficult to distinguish from background levels of genetic drift and sampling variance. However, increasingly genomic experimental evolution studies with multiple timepoints, and in some cases multiple replicate populations, are being used to detect large effect selected loci (<a id="xref-ref-52-1" class="xref-bibr" href="#ref-52">Turner and Miller 2012</a>; <a id="xref-ref-53-1" class="xref-bibr" href="#ref-53">Turner et al. 2011</a>) and differentiate modes of selection (<a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">Barghi et al. 2019</a>; <a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">Burke et al. 2010</a>; <a id="xref-ref-51-1" class="xref-bibr" href="#ref-51">Therkildsen et al. 2019</a>). In addition these temporal-genomic studies have begun in wild populations, some with the goal of finding variants that exhibit frequency changes consistent with fluctuating selection (<a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">Bergland et al. 2014</a>; <a id="xref-ref-35-1" class="xref-bibr" href="#ref-35">Machado et al. 2018</a>). In a previous paper, we proposed that one useful signal for understanding the genome-wide impact of polygenic linked selection detectable from genomic studies with multiple timepoints is the temporal autocovariance in allele frequency changes (<a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">Buffalo and Coop 2019</a>). These covariances are directly estimable from temporal genomic data and are created when the loci that underly heritable fitness variation perturb the frequencies of linked neutral alleles; in contrast, when genetic drift acts alone in a closed population, these covariances are zero in expectation. Mathematically, temporal covariances are useful because it is natural to decompose the total variance in allele frequency change across a set of time intervals into the variances and covariances in allele frequency change among time intervals. Furthermore, biologically, these covariances reflect the extent to which neutral allele frequency changes in one generation predict changes in another due to a shared selection pressures and associations to selected loci.</p><p id="p-5">Here, we provide the first empirical analyses to quantify the impact of linked selection acting over short timescales (tens of generations) across two evolve and re-sequence studies (<a id="xref-ref-4-2" class="xref-bibr" href="#ref-4">Barghi et al. 2019</a>; <a id="xref-ref-33-1" class="xref-bibr" href="#ref-33">Kelly and Hughes 2019</a>), and an artificial selection experiment (<a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">Castro et al. 2019</a>). We repeatedly find a signal of temporal covariance, consistent with linked selection acting to significantly perturb genome-wide allele frequency changes across the genome in a manner that other approaches would not be able differentiate from genetic drift. We estimate the lower bound on the proportion of total variation in allele frequency change caused by selection, and the correlation between allele frequency changes between replicate populations caused by the response to convergent selection pressures. Overall, we demonstrate that linked selection has a powerful role in shaping genome-wide allele frequency changes over very short timescales.</p><div id="T1" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/945677/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup"><a href="/highwire/markup/945677/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/945677" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 1:</span> <p id="p-6" class="first-child">A summary of the main selection studies we analyzed.</p><div class="sb-div caption-clear"></div></div></div></div><div class="section" id="sec-2"><h2 class="">2 Results</h2><p id="p-7">We first analyzed <a id="xref-ref-4-3" class="xref-bibr" href="#ref-4">Barghi et al. (2019)</a>, an evolve-and-resequence study with ten replicate populations exposed to a high temperature lab environment and evolved for 60 generations, and sequenced every ten generations. Using the seven timepoints and ten replicate populations, we estimated the genome-wide 6 × 6 temporal covariance matrix <strong>Q</strong> for each of the ten replicates. Each row of these matrices represent the temporal covariance Cov(Δ<sub>10</sub><em>p<sub>s</sub></em>, Δ<sub>10</sub><em>p<sub>t</sub></em>), between the allele frequency change (in ten-generation intervals, denoted Δ<sub>10</sub><em>p<sub>t</sub></em>) in some initial reference generation <em>s</em> (the row of the matrix), and some later timepoint <em>t</em> (the column of the matrix). We corrected these matrices for biases created due to sampling noise, and normalize the entries for heterozygosity (see Supplementary Materials <a id="xref-sec-7-1" class="xref-sec" href="#sec-7">Sections 1.1.2</a> and <a id="xref-sec-9-1" class="xref-sec" href="#sec-9">1.1.4</a>). These covariances are expected to be zero when only drift is acting, as only heritable variation for fitness can create covariance between allele frequency changes in a closed population (<a id="xref-ref-13-2" class="xref-bibr" href="#ref-13">Buffalo and Coop 2019</a>). Averaging across the ten replicate temporal covariances matrices, we find temporal covariances that are statistically significant (95% block bootstraps CIs do not contain zero), consistent with linked selection perturbing genome-wide allele frequency changes over very short time periods. The covariances between all adjacent time intervals are positive and then decay towards zero as we look at more distant time intervals (<a id="xref-fig-1-1" class="xref-fig" href="#F1">Figure 1 A</a>), as expected when directional selection affects linked variants’ frequency trajectories until ultimately linkage disequilibrium and the additive genetic variance for fitness associated with neutral alleles decays (<a id="xref-ref-13-3" class="xref-bibr" href="#ref-13">Buffalo and Coop 2019</a>). The temporal covariances per replicate are noisier but this general pattern holds; see <a id="xref-fig-9-1" class="xref-fig" href="#F9">Supplementary Figure S6</a>. <a id="xref-ref-4-4" class="xref-bibr" href="#ref-4">Barghi et al. (2019)</a>’s design means that the covariances we see in adjacent time intervals are on average ten generations apart, and given the temporal decay in covariance we see, the covariances on shorter time-scales (e.g. if adjacent generations had been sequenced) may well be higher yet (see Supplementary Material <a id="xref-sec-10-1" class="xref-sec" href="#sec-10">Section 1.1.5</a> for more details).</p><div id="F1" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/10/09/798595/F1.large.jpg?width=800&amp;height=600&amp;carousel=1" title="A: Temporal covariance, averaged across all ten replicate populations, through time from the Barghi et al. (2019) study. Each line depicts the temporal covariance Cov(&#x394;ps, &#x394;pt) from some reference generation s to a later time t which varies along the x-axis; each line corresponds to a row of the upper-triangle of the temporal covariance matrix with the same color (upper right). The ranges around each point are 95% block-bootstrap confidence intervals. B: The proportion of the total variance in allele frequency change explained by linked selection, G(t), as it varies through time t along the x-axis. The black line is the G(t) averaged across replicates, with the 95% block-bootstrap confidence interval. The other lines are the G(t) for each individual replicate, with colors indicating what subset of the temporal-covariance matrix to the right is being included in the calculation of G(t)." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-290610267" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;A: Temporal covariance, averaged across all ten replicate populations, through time from the Barghi et al. (2019) study. Each line depicts the temporal covariance Cov(&#x394;&lt;em&gt;p&lt;/em&gt;&lt;sub&gt;&lt;em&gt;s&lt;/em&gt;&lt;/sub&gt;, &#x394;&lt;em&gt;p&lt;/em&gt;&lt;sub&gt;&lt;em&gt;t&lt;/em&gt;&lt;/sub&gt;) from some reference generation &lt;em&gt;s&lt;/em&gt; to a later time &lt;em&gt;t&lt;/em&gt; which varies along the x-axis; each line corresponds to a row of the upper-triangle of the temporal covariance matrix with the same color (upper right). The ranges around each point are 95% block-bootstrap confidence intervals. B: The proportion of the total variance in allele frequency change explained by linked selection, &lt;em&gt;G&lt;/em&gt;(&lt;em&gt;t&lt;/em&gt;), as it varies through time &lt;em&gt;t&lt;/em&gt; along the x-axis. The black line is the &lt;em&gt;G&lt;/em&gt;(&lt;em&gt;t&lt;/em&gt;) averaged across replicates, with the 95% block-bootstrap confidence interval. The other lines are the &lt;em&gt;G&lt;/em&gt;(&lt;em&gt;t&lt;/em&gt;) for each individual replicate, with colors indicating what subset of the temporal-covariance matrix to the right is being included in the calculation of &lt;em&gt;G&lt;/em&gt;(&lt;em&gt;t&lt;/em&gt;).&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 1:" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2019/10/09/798595/F1.medium.gif" width="440" height="314"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 1:" src="https://www.biorxiv.org/content/biorxiv/early/2019/10/09/798595/F1.medium.gif" width="440" height="314"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/10/09/798595/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1:" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/10/09/798595/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml"><span class="fig-label">Figure 1:</span> <p id="p-8" class="first-child">A: Temporal covariance, averaged across all ten replicate populations, through time from the <a id="xref-ref-4-5" class="xref-bibr" href="#ref-4">Barghi et al. (2019)</a> study. Each line depicts the temporal covariance Cov(Δ<em>p</em><sub><em>s</em></sub>, Δ<em>p</em><sub><em>t</em></sub>) from some reference generation <em>s</em> to a later time <em>t</em> which varies along the x-axis; each line corresponds to a row of the upper-triangle of the temporal covariance matrix with the same color (upper right). The ranges around each point are 95% block-bootstrap confidence intervals. B: The proportion of the total variance in allele frequency change explained by linked selection, <em>G</em>(<em>t</em>), as it varies through time <em>t</em> along the x-axis. The black line is the <em>G</em>(<em>t</em>) averaged across replicates, with the 95% block-bootstrap confidence interval. The other lines are the <em>G</em>(<em>t</em>) for each individual replicate, with colors indicating what subset of the temporal-covariance matrix to the right is being included in the calculation of <em>G</em>(<em>t</em>).</p><div class="sb-div caption-clear"></div></div></div><p id="p-9">One concern is that these covariances reflect the localized impact of a few large-effect loci rather than selection on a polygenic trait. Since our covariances are essentially averages over loci, the covariance estimate can be strongly affected by a single outlier region. To test whether large outlier regions drive the genome-wide signal we see in the <a id="xref-ref-4-6" class="xref-bibr" href="#ref-4">Barghi et al. (2019)</a> data, we calculate the covariances in 100kb windows along the genome (we refer to these as windowed covariances throughout) and take the median windowed covariance, and trimmed-mean windowed covariance, as a measure of the genome-wide covariance robust to large-effect loci. These robust estimates (<a id="xref-table-wrap-2-1" class="xref-table" href="#T2">Supplementary Table S1</a> and <a id="xref-fig-10-1" class="xref-fig" href="#F10">Supplementary Figure S7</a>) confirm the patterns we see using the mean covariance, confirming that genomic temporal covariances are non-zero due to the impact of selection acting across many genomic windows.</p><p id="p-10">While the presence of positive temporal covariances is consistent with selection affecting allele frequencies over time, this measure is not easily interpretable. We can calculate a more intuitive measure from the temporal covariances to quantify the impact of selection on allele frequency change: the ratio of total covariance in allele frequency change to the total variance in allele frequency change. We denote the change in allele frequency as Δ<em>p</em><sub><em>t</em></sub> = <em>p</em><sub><em>t</em>+1</sub> − <em>p</em><sub><em>t</em></sub>, where <em>p</em><sub><em>t</em></sub> is the allele frequency in generation <em>t</em>. Since the total variation in allele frequency change can be partitioned into variance and covariance components, <span class="inline-formula" id="inline-formula-1"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2019/10/09/798595/embed/inline-graphic-1.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2019/10/09/798595/embed/inline-graphic-1.gif"/></noscript></span></span> (we bias correct these for sequencing depth), and the covariances are zero when drift acts alone, this is a lower bound on how much of the variance in allele frequency change is caused by linked selection (<a id="xref-ref-13-4" class="xref-bibr" href="#ref-13">Buffalo and Coop 2019</a>). We call this measure <em>G</em>(<em>t</em>), defined as
<span class="disp-formula" id="disp-formula-1"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2019/10/09/798595/embed/graphic-3.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2019/10/09/798595/embed/graphic-3.gif"/></noscript></span>

</span>
which estimates the effect of selection on allele frequency change between the initial generation 0 and some later generation <em>t</em>, which can be varied to see how this quantity grows through time. Since <a id="xref-ref-4-7" class="xref-bibr" href="#ref-4">Barghi et al. (2019)</a> experiment is sequenced every ten generations, in the numerator for the covariance we use the allele frequency changes between adjacent timepoints, which are ten generations apart. Consequently, this leads our measure <em>G</em>(<em>t</em>) to be strongly conservative, since the temporal covariances within each ten-generation block are not directly observable, and thus are not included in the numerator of <em>G</em>(<em>t</em>). Still, we find a remarkably strong signal. Greater than 20% of total, genome-wide allele frequency change over 60 generations is the result of selection (<a id="xref-fig-1-2" class="xref-fig" href="#F1">Figure 1 B</a>).</p><p id="p-11">Additionally, we looked for a signal of temporal autocovariance in <a id="xref-ref-11-2" class="xref-bibr" href="#ref-11">Bergland et al. (2014)</a>, a study that collected <em>Drosophila melanogaster</em> through Spring-Fall season pairs across three years. If there was a strong pattern of genome-wide fluctuating selection, we might expect a pattern of positive covariances between similar seasonal changes, e.g. Spring-Fall in two adjacent years, and negative covariances between dissimilar seasonal changes, e.g. Spring-Fall and Fall-Spring in two adjacent years. However, we find no such signal over years; we discuss this in more depth in Supplementary Materials <a id="xref-sec-14-1" class="xref-sec" href="#sec-14">Section 1.5</a>.</p><p id="p-12">The replicate design of <a id="xref-ref-4-8" class="xref-bibr" href="#ref-4">Barghi et al. (2019)</a> allows us to quantify another covariance: the covariance in allele frequency change between replicate populations experiencing convergent selection pressures. These between-replicate covariances are created in the same way as temporal covariances: neutral alleles linked to a particular fitness background are expected to have allele frequency changes in the same direction if the selection pressures are similar. Intuitively, where temporal covariances reflect that neutral alleles associated with heritable fitness backgrounds are predictive of frequency changes between generations, replicate covariances reflect that heritable fitness back-grounds common to each replicate predict (under the same selection pressures) frequency changes between replicates. We measure this through a statistic similar to a correlation, which we call the convergent correlation: the ratio of average between-replicate covariance across all pairs to the average standard deviation across all pairs of replicates,
<span class="disp-formula" id="disp-formula-2"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2019/10/09/798595/embed/graphic-4.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2019/10/09/798595/embed/graphic-4.gif"/></noscript></span>

</span>
where <em>A</em> and <em>B</em> here are two replicate labels, and for the <a id="xref-ref-4-9" class="xref-bibr" href="#ref-4">Barghi et al. (2019)</a> data, we use Δ<sub>10</sub> <em>p</em><sub><em>t</em></sub>.</p><p id="p-13">We’ve calculated the convergent correlation for all rows of the replicate covariance matrices. Like temporal covariances, we visualize these through time (<a id="xref-fig-2-1" class="xref-fig" href="#F2">Figure 2 A</a>), with each line representing the convergent correlation from a particular reference generation <em>s</em> as it varies with <em>t</em> (shown on the x-axis). In other words, each of the colored lines corresponds to the like-colored row of the convergence correlation matrix (upper left in <a id="xref-fig-2-2" class="xref-fig" href="#F2">Figure 2 A</a>). We find these convergent covariances are relatively weak, and decay very quickly from an initial value of about 0.1 (95% block bootstrap confidence intervals [0.094, 0.11]) to around 0.01 (95% CIs [0.0087, 0.015]) within 20 generations. This suggests that while a reasonable fraction of the initial response is shared over the replicates, this is followed by a rapid decay, a result consistent with the primary finding of the original <a id="xref-ref-4-10" class="xref-bibr" href="#ref-4">Barghi et al. (2019)</a> study: that alternative loci contribute to longer term adaptation across the different replicates.</p><div id="F2" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/10/09/798595/F2.large.jpg?width=800&amp;height=600&amp;carousel=1" title="A: The convergence correlation, averaged across Barghi et al. (2019) replicate pairs, through time. Each line represents the convergence correlation cor(&#x394;ps, &#x394;ps) from a starting reference generation s to a later time t, which varies along the x-axis; each line corresponds to a row of the temporal convergence correlation matrix depicted to the right. We note that convergent correlation for the last timepoint is an outlier; we are unsure as to the cause of this, e.g. it does not appear to be driven by a single pair of replicates. B: The convergence covariance between individual pairs of replicates in the Kelly and Hughes (2019) data. C: The convergence covariance between individual pairs of replicates in (Castro et al. 2019) data, for the two selection lines (LS1 and LS2) and the control (Ctrl); gray CIs are those using the complete dataset, blue CIs exclude chromosomes 5 and 10 which harbor the two regions Castro et al. (2019) found to have signals of parallel selection between LS1 and LS2." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-290610267" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;&lt;strong&gt;A&lt;/strong&gt;: The convergence correlation, averaged across Barghi et al. (2019) replicate pairs, through time. Each line represents the convergence correlation cor(&#x394;&lt;em&gt;p&lt;/em&gt;&lt;sub&gt;&lt;em&gt;s&lt;/em&gt;&lt;/sub&gt;, &#x394;&lt;em&gt;p&lt;/em&gt;&lt;sub&gt;&lt;em&gt;s&lt;/em&gt;&lt;/sub&gt;) from a starting reference generation &lt;em&gt;s&lt;/em&gt; to a later time &lt;em&gt;t&lt;/em&gt;, which varies along the x-axis; each line corresponds to a row of the temporal convergence correlation matrix depicted to the right. We note that convergent correlation for the last timepoint is an outlier; we are unsure as to the cause of this, e.g. it does not appear to be driven by a single pair of replicates. &lt;strong&gt;B&lt;/strong&gt;: The convergence covariance between individual pairs of replicates in the Kelly and Hughes (2019) data. &lt;strong&gt;C&lt;/strong&gt;: The convergence covariance between individual pairs of replicates in (Castro et al. 2019) data, for the two selection lines (LS1 and LS2) and the control (Ctrl); gray CIs are those using the complete dataset, blue CIs exclude chromosomes 5 and 10 which harbor the two regions Castro et al. (2019) found to have signals of parallel selection between LS1 and LS2.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 2:" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2019/10/09/798595/F2.medium.gif" width="440" height="312"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 2:" src="https://www.biorxiv.org/content/biorxiv/early/2019/10/09/798595/F2.medium.gif" width="440" height="312"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/10/09/798595/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2:" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/10/09/798595/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 2:</span> <p id="p-14" class="first-child"><strong>A</strong>: The convergence correlation, averaged across <a id="xref-ref-4-11" class="xref-bibr" href="#ref-4">Barghi et al. (2019)</a> replicate pairs, through time. Each line represents the convergence correlation cor(Δ<em>p</em><sub><em>s</em></sub>, Δ<em>p</em><sub><em>s</em></sub>) from a starting reference generation <em>s</em> to a later time <em>t</em>, which varies along the x-axis; each line corresponds to a row of the temporal convergence correlation matrix depicted to the right. We note that convergent correlation for the last timepoint is an outlier; we are unsure as to the cause of this, e.g. it does not appear to be driven by a single pair of replicates. <strong>B</strong>: The convergence covariance between individual pairs of replicates in the <a id="xref-ref-33-2" class="xref-bibr" href="#ref-33">Kelly and Hughes (2019)</a> data. <strong>C</strong>: The convergence covariance between individual pairs of replicates in (<a id="xref-ref-15-2" class="xref-bibr" href="#ref-15">Castro et al. 2019</a>) data, for the two selection lines (LS1 and LS2) and the control (Ctrl); gray CIs are those using the complete dataset, blue CIs exclude chromosomes 5 and 10 which harbor the two regions <a id="xref-ref-15-3" class="xref-bibr" href="#ref-15">Castro et al. (2019)</a> found to have signals of parallel selection between LS1 and LS2.</p><div class="sb-div caption-clear"></div></div></div><p id="p-15">A benefit of between-replicate covariances is that unlike temporal covariances, these can be calculated with only two sequenced timepoints and a replicated study design. This allowed us to assess the impact of linked selection in driving convergent patterns of allele frequency change across replicate populations in two other studies. First, we reanalyzed the selection experiment of <a id="xref-ref-33-3" class="xref-bibr" href="#ref-33">Kelly and Hughes (2019)</a>, which evolved three replicate wild populations of <em>Drosophila simulans</em> for 14 generations adapting to a novel laboratory environment. Since each replicate was exposed to the same selection pressure and share linkage disequilibria common to the original natural founding population, we expected each of the three replicate populations to have positive between-replicate covariances. We find all three pairwise between-replicate covariances are positive and statistically significant (<a id="xref-fig-2-3" class="xref-fig" href="#F2">Figure 2 B</a>. We estimate the convergent correlation coefficient across these replicates as 0.36 (95% CI [0.31, 0.40]). Similarly, we can calculate the proportion of the total variance in allele frequency change from convergent selection pressure analogous to <em>G</em> where the numerator is is the convergent covariance and the denominator is the total variance (see Supplementary Material 1.3). We find that 37% of the total variance is due to shared allele frequency changes caused by selection (95% CI [29%, 41%]; these are similar to the convergence correlation, since the variance is relatively constant across the replicates.</p><p id="p-16">Next, we reanalyzed the Longshanks selection experiment, which selected for longer tibiae length relative to body size in mice, leading to a response to selection of about 5 standard deviations over the course of twenty generations (<a id="xref-ref-15-4" class="xref-bibr" href="#ref-15">Castro et al. 2019</a>; <a id="xref-ref-37-1" class="xref-bibr" href="#ref-37">Marchini et al. 2014</a>). This study includes two independent selection lines, Longshanks 1 and 2 (LS1 and LS2), and an unselected control line (Ctrl). Consequently, this selection experiment offers a useful control to test our between-replicate covariances: we expect to see positive between-replicate covariance in the comparison between the two Longshanks selection lines, but not between the two pairwise comparisons between the control line and each of the two Longshanks lines. We find that this is the case (gray confidence intervals in <a id="xref-fig-2-4" class="xref-fig" href="#F2">Figure 2 C</a>), with the two Longshanks comparisons to the control line not being significantly different from zero, while the comparison between the two Longshanks lines is statistically significantly different from zero (CIs [0.0129, 0.0400]).</p><p id="p-17">One finding in the Longshanks study was that two major-effect loci showed parallel frequency shifts between the two selection lines: a region harboring the gene Nkx3-2 known to be involved in limb development, and another region containing six other candidate genes. We were curious to what extent our genome-wide covariances were being driven by these two outlier large-effect loci, so we excluded them from the analysis. Since we do not know the extent to which linkage disequilibrium around these large-effect loci affects neighboring loci, we took the conservative precaution of excluding the entire chromosomes these loci reside on (chromosomes 5 and 10), and re-calculating the temporal covariances. We find excluding these large effect loci has little impact on the confidence intervals (blue confidence intervals in <a id="xref-fig-2-5" class="xref-fig" href="#F2">Figure 2 C</a>), indicating that these across-replicate covariances are indeed driven by a large number of loci. This is consistent with a signal of selection on a polygenic trait driving genome-wide change, although we note that large-effect loci can contribute to the indirect change at unlinked loci (<a id="xref-ref-45-1" class="xref-bibr" href="#ref-45">Robertson 1961</a>; <a id="xref-ref-46-1" class="xref-bibr" href="#ref-46">Santiago and Caballero 1995</a>).</p><p id="p-18">The presence of an unselected control line provides an alternative way to partition the effects of linked selection and genetic drift: we can compare the total variance in allele frequency change of the control line (which excludes the effect of artificial selection on allele frequencies) to the total variance in frequency change of the Longshanks selection lines. We can partition the allele frequency change between the two timepoints (20 generations apart) for a Longshanks line as Δ<em>p</em><sub><em>t</em></sub>,LS1 = Δ<sub><em>D</em></sub><em>p</em><sub><em>t</em></sub>,LS1 + Δ<sub><em>U</em></sub><em>p</em><sub><em>t</em></sub>,LS1 + Δ<sub><em>S</em></sub><em>p</em><sub><em>t</em></sub>,LS where these terms are the decomposition in the allele frequency change due to drift in Longshanks replicate 1 (Δ<sub><em>D</em></sub><em>p</em><sub><em>t</em></sub>,LS1), selection unique to the LS1 replicate (Δ<sub><em>U</em></sub><em>p</em><sub><em>t</em></sub>,LS1), and selection response shared between the two Longshanks replicates (Δ<sub><em>S</em></sub><em>p</em><sub><em>t</em></sub>,LS) respectively (and similarly for the Longshanks two line, LS2). By construction we will assume that each of these terms are uncorrelated within replicates, and that only the shared term covaries between the replicates. Assuming that we can approximate the contribution of genetic drift in the LS lines as the variance in allele frequency change observed in the control, i.e. Var(Δ<em>p</em><sub><em>t</em></sub>,Ctrl) = Var(Δ<sub><em>D</em></sub><em>p</em><sub><em>t</em></sub>,LS2) = Var(Δ<sub><em>D</em></sub><em>p</em><sub><em>t</em></sub>,LS2), then we can estimate the increase in variance in allele frequency change due to selection as (Var(Δ<em>p</em><sub><em>t</em></sub>,LS1) + Var(Δ<em>p</em><sub><em>t</em></sub>,LS2))/2 − Var(Δ<em>p</em><sub><em>t</em></sub>,Ctrl) and the shared effect of selection across selected lines as Cov(Δ<em>p</em><sub><em>t</em></sub>,LS1, Δ<em>p</em><sub><em>t</em></sub>,LS2) (see Supplementary Material <a id="xref-sec-12-1" class="xref-sec" href="#sec-12">Section 1.3</a> for more details). We estimate at least 32% (95% CI [21%, 48%]) of the variance in allele frequency change is driven by the effects of selection, of which 14% (95% CI [3%, 33%]) is estimated to be unique to a selection line, and 17% (95% CI [9%, 23%]) is the effect of shared selection between the two Longshanks selection lines (and the value of the convergence correlation between the Longshanks lines, a related statistic, is 0.18, 95% CI [0.0743, 0.254]).</p><p id="p-19">Finally, we observed that in the longest study we analyzed (<a id="xref-ref-4-12" class="xref-bibr" href="#ref-4">Barghi et al. 2019</a>), some genome-wide temporal covariances become negative at future timepoints (see the first two rows in <a id="xref-fig-1-3" class="xref-fig" href="#F1">Figure 1 A</a>). This shows that alleles that were on average going up initially are later going down in frequency, i.e. that the average direction of selection experienced by alleles has flipped. This must reflect either a change in the environment or the genetic background, due to epistatic relationships among alleles altered by frequency changes or recombination breaking up selective alleles. Such reversals of selective dynamics could be occurring at other timepoints but the signal of a change in the direction of selection at particular loci may be washed out when we calculate our genome-wide average temporal covariances. To address this limitation, we calculated the distribution of the temporal covariances over 100kb windowed covariances (<a id="xref-fig-3-1" class="xref-fig" href="#F3">Figure 3</a> shows these distributions pooling across all replicates; see <a id="xref-fig-12-1" class="xref-fig" href="#F12">Supplementary Figure S9</a> for individuals replicates). The covariance estimate of each genomic window will be noisy, due to sampling and genetic drift, and the neutral distribution of the covariance is complicated due to linkage disequilibria (which can occur over long physical distances in E&amp;R and selection studies, <a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">Baldwin-Brown et al. 2014</a>; <a id="xref-ref-43-1" class="xref-bibr" href="#ref-43">Nuzhdin and Turner 2013</a>). To address this, we have developed a permutation-based procedure that constructs an empirical null distribution by randomly flipping the signs of the allele frequency changes pergenomic window. This destroys the systematic covariances created by linked selection and creates a sampling distribution of the covariances spuriously created by neutral genetic drift while preserving the complex dependencies between adjacent loci created by linkage disequilibrium. This empirical neutral null distribution is conservative in the sense that the variances of the covariances are wider than expected under drift alone as they include the effect of selection on the allele frequency change within a time-interval, just not between time-intervals. We see (<a id="xref-fig-3-2" class="xref-fig" href="#F3">Figure 3 A</a> and <a id="xref-fig-3-3" class="xref-fig" href="#F3">B</a>) that windowed temporal covariances between close timepoints are skewed positive (a heavy right tail), while between more distant timepoints these windowed temporal covariances tend to shift to become more negative (a heavy left tail). We quantified the degree to which the left and right tails are inflated compared to the null distribution as a function of time, and see excesses in both tails in <a id="xref-fig-3-4" class="xref-fig" href="#F3">Figure 3 C</a>. This finding is also robust to sign-permuting allele frequency changes on a chromosome-level, the longest extent that gametic linkage disequilibria can extend (<a id="xref-fig-15-1" class="xref-fig" href="#F15">Supplementary Figure S12</a>). We see a striking pattern that the windowed covariances not only decay towards zero, but in fact become negative through time, consistent with many regions in the genome having had a reversed fitness effect at later timepoints.</p><div id="F3" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/10/09/798595/F3.large.jpg?width=800&amp;height=600&amp;carousel=1" title="A, B: The distribution of temporal covariances calculated in 100kb genomic windows from the Barghi et al. (2019) study, plotted alongside an empirical neutral null distribution created by recalculating the windowed covariances on 1,000 sign permutations of allele frequency changes within tiles. The histogram bin number is 88, chosen by cross validation (Supplementary Materials S8). In subfigure A, windowed covariances Cov(&#x394;pt, &#x394;pt+k) are separated by k = 2 &#xD7; 10 generations and in subfigure A the covariances are separated by k = 4 10 generations; each k is an off-diagonal from the variance diagonal of the temporal covariance matrix (see cartoon of upper-triangle of covariance matrix in subfigures A and B, where the first diagonal is the variance, and the dark gray indicates which off-diagonal of the covariance matrix is plotted in the histograms). C: The lower and upper tail probabilities of the observed windowed covariances, at 20% and 80% quintiles of the empirical neutral null distribution, for varying time between allele frequency changes (i.e. which off-diagonal k). The confidence intervals are 95% block-bootstrap confidence intervals, and the light gray dashed line indicates the 20% tail probability expected under the neutral null. Similar figures for different values of k are in Supplementary Figures S10." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-290610267" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;&lt;strong&gt;A&lt;/strong&gt;, &lt;strong&gt;B&lt;/strong&gt;: The distribution of temporal covariances calculated in 100kb genomic windows from the Barghi et al. (2019) study, plotted alongside an empirical neutral null distribution created by recalculating the windowed covariances on 1,000 sign permutations of allele frequency changes within tiles. The histogram bin number is 88, chosen by cross validation (Supplementary Materials S8). In subfigure &lt;strong&gt;A&lt;/strong&gt;, windowed covariances Cov(&#x394;&lt;em&gt;p&lt;/em&gt;&lt;sub&gt;&lt;em&gt;t&lt;/em&gt;&lt;/sub&gt;, &#x394;&lt;em&gt;p&lt;/em&gt;&lt;sub&gt;&lt;em&gt;t&lt;/em&gt;+&lt;em&gt;k&lt;/em&gt;&lt;/sub&gt;) are separated by &lt;em&gt;k&lt;/em&gt; = 2 &#xD7; 10 generations and in subfigure &lt;strong&gt;A&lt;/strong&gt; the covariances are separated by &lt;em&gt;k&lt;/em&gt; = 4 10 generations; each &lt;em&gt;k&lt;/em&gt; is an off-diagonal from the variance diagonal of the temporal covariance matrix (see cartoon of upper-triangle of covariance matrix in subfigures &lt;strong&gt;A&lt;/strong&gt; and &lt;strong&gt;B&lt;/strong&gt;, where the first diagonal is the variance, and the dark gray indicates which off-diagonal of the covariance matrix is plotted in the histograms). &lt;strong&gt;C&lt;/strong&gt;: The lower and upper tail probabilities of the observed windowed covariances, at 20% and 80% quintiles of the empirical neutral null distribution, for varying time between allele frequency changes (i.e. which off-diagonal &lt;em&gt;k&lt;/em&gt;). The confidence intervals are 95% block-bootstrap confidence intervals, and the light gray dashed line indicates the 20% tail probability expected under the neutral null. Similar figures for different values of &lt;em&gt;k&lt;/em&gt; are in Supplementary Figures S10.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 3:" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2019/10/09/798595/F3.medium.gif" width="233" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 3:" src="https://www.biorxiv.org/content/biorxiv/early/2019/10/09/798595/F3.medium.gif" width="233" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/10/09/798595/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 3:" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/10/09/798595/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 3:</span> <p id="p-20" class="first-child"><strong>A</strong>, <strong>B</strong>: The distribution of temporal covariances calculated in 100kb genomic windows from the <a id="xref-ref-4-13" class="xref-bibr" href="#ref-4">Barghi et al. (2019)</a> study, plotted alongside an empirical neutral null distribution created by recalculating the windowed covariances on 1,000 sign permutations of allele frequency changes within tiles. The histogram bin number is 88, chosen by cross validation (Supplementary Materials S8). In subfigure <strong>A</strong>, windowed covariances Cov(Δ<em>p</em><sub><em>t</em></sub>, Δ<em>p</em><sub><em>t</em>+<em>k</em></sub>) are separated by <em>k</em> = 2 × 10 generations and in subfigure <strong>A</strong> the covariances are separated by <em>k</em> = 4 10 generations; each <em>k</em> is an off-diagonal from the variance diagonal of the temporal covariance matrix (see cartoon of upper-triangle of covariance matrix in subfigures <strong>A</strong> and <strong>B</strong>, where the first diagonal is the variance, and the dark gray indicates which off-diagonal of the covariance matrix is plotted in the histograms). <strong>C</strong>: The lower and upper tail probabilities of the observed windowed covariances, at 20% and 80% quintiles of the empirical neutral null distribution, for varying time between allele frequency changes (i.e. which off-diagonal <em>k</em>). The confidence intervals are 95% block-bootstrap confidence intervals, and the light gray dashed line indicates the 20% tail probability expected under the neutral null. Similar figures for different values of <em>k</em> are in Supplementary Figures S10.</p><div class="sb-div caption-clear"></div></div></div></div><div class="section" id="sec-3"><h2 class="">3 Discussion</h2><p id="p-21">Since the seminal analysis of <a id="xref-ref-38-1" class="xref-bibr" href="#ref-38">Maynard Smith and Haigh (1974)</a> demonstrating that linked neutral diversity is reduced as an advantageous polymorphism arises and sweeps to fixation, over four decades of theoretical and empirical research has enriched our understanding of linked selection. One under-used approach to understand the genome-wide effects of selection on standing variation, e.g. selection on an infinitesimal polygenic trait, stems from an early quantitative genetic model of linked selection (<a id="xref-ref-45-2" class="xref-bibr" href="#ref-45">Robertson 1961</a>) and its later developments (<a id="xref-ref-46-2" class="xref-bibr" href="#ref-46">Santiago and Caballero 1995</a>, <a id="xref-ref-47-1" class="xref-bibr" href="#ref-47">1998</a>; <a id="xref-ref-57-1" class="xref-bibr" href="#ref-57">Woolliams et al. 1993</a>; <a id="xref-ref-58-1" class="xref-bibr" href="#ref-58">Wray and Thompson 1990</a>; see also <a id="xref-ref-5-2" class="xref-bibr" href="#ref-5">Barton 2000</a> for a comparison of these models with classic hitchhiking models). Implicit in these models is that autocovariance between allele frequency change is created when there is heritable fitness variation in the population, a signal that may be readily detected from temporal genomic data (<a id="xref-ref-13-5" class="xref-bibr" href="#ref-13">Buffalo and Coop 2019</a>). Depending on how many loci affect fitness, such a strong effect of linked selection may not be differentiable from genetic drift using only single contemporary population samples or looking at at temporal allele frequency change at each locus in isolation. In this way, averaging summaries of temporal data allows us to sidestep the key problem of detecting selection from standing variation: that the genomic footprint leaves too soft of a signature to differentiate from a background of genetic drift. In fact we find that the temporal covariance signal is detectable even in the most extremely difficult to detect soft sweep case: polygenic selection on highly polygenic traits (<a id="xref-ref-13-6" class="xref-bibr" href="#ref-13">Buffalo and Coop 2019</a>).</p><p id="p-22">It is worth building some intuition why temporal covariance allows us to detect such faint signals of polygenic linked selection from temporal genomic data. Each variant is subject to both variance in allele frequency due to drift and sampling noise, which at any locus may swamp the temporal covariance signal and creates spurious covariances. However, these spurious covariances do not share a directional signal whereas the covariances created by linked selection do; consequently, averaging across the entire genome, the temporal signal exceeds sampling noise.</p><p id="p-23">Our analyses reveal that a sizable proportion of allele frequency change in these populations is due to the (indirect) action of selection. Capitalizing on replicated designs, we characterized the extent to which convergent selection pressures lead to parallel changes in allele frequencies across replicate populations, and found that a reasonable proportion of the response is shared across short timescales. These likely represent substantial under-estimates of the contribution of linked selection because the studies we have reanalyzed do not sequence the population each generation, preventing us from including the effects of stronger correlations between adjacent generations. Furthermore, our estimation methods are intentionally conservative, for example they exclude the contribution of selection that does not persist across generations and selection that reverses sign; thus they can be seen as a strong lower bound of the effects of selection.</p><p id="p-24">These estimates of the contribution of selection could be refined by using patterns of LD and recombination which would allow us to more fully parameterize a linked-selection model of temporal allele frequency change (<a id="xref-ref-13-7" class="xref-bibr" href="#ref-13">Buffalo and Coop 2019</a>). The basic prediction is that regions of higher linkage disequilibrium and lower recombination should have greater temporal autocovariance than regions with lower LD and higher recombination. However, one limitation of these pooled sequence datasets is that none of the studies we reanalyzed estimated linkage disequilibria data for the evolved populations. While there are LD data for a natural population of <em>D. simulans</em> (<a id="xref-ref-31-1" class="xref-bibr" href="#ref-31">Howie et al. 2018</a>; <a id="xref-ref-50-1" class="xref-bibr" href="#ref-50">Signor et al. 2018</a>), we did not find a relationship between temporal covariance and LD. We believe this is driven by the idiosyncratic nature of LD in evolve-and-resequence populations, which often extends over large genomic distances (<a id="xref-ref-33-4" class="xref-bibr" href="#ref-33">Kelly and Hughes 2019</a>; <a id="xref-ref-43-2" class="xref-bibr" href="#ref-43">Nuzhdin and Turner 2013</a>). Future studies complete with LD data and recombination maps would allow one to disentangle the influence of closely linked sites from more distant sites in causing temporal autocovariance, and allow the fitting of more parametric models to estimate population parameters such as the additive genetic variation for fitness directly from temporal genomic data alone (<a id="xref-ref-13-8" class="xref-bibr" href="#ref-13">Buffalo and Coop 2019</a>).</p><p id="p-25">Our primary focus here has been on evolution in laboratory populations. It is unclear whether we should expect a similar impact of selection in natural populations. In some of these experiments, selection pressures may have been stronger or more sustained that in natural populations (<a id="xref-ref-28-1" class="xref-bibr" href="#ref-28">Hairston et al. 2005</a>; <a id="xref-ref-29-1" class="xref-bibr" href="#ref-29">Hendry and Kinnison 1999</a>). Conversely, these lab populations were maintained at very small effective population sizes, estimated at 300, 450, and 45 for the <a id="xref-ref-4-14" class="xref-bibr" href="#ref-4">Barghi et al. (2019)</a>, <a id="xref-ref-33-5" class="xref-bibr" href="#ref-33">Kelly and Hughes (2019)</a>, and <a id="xref-ref-15-5" class="xref-bibr" href="#ref-15">Castro et al. (2019)</a> studies respectively, which will amplify the role of genetic drift. The advantage of lab experiments is that they are closed populations, in natural populations temporal covariance could also arise from the systematic migration of alleles from differentiated populations. Adapting these methods to natural populations will require either populations that are reasonably closed to migration, or for the effect of migration to be accounted for possibly either by knowledge of allele frequencies in source populations or the identification of migrant individuals.</p><p id="p-26">While it challenging to apply temporal methods to natural populations there is a lot of promise for these approaches (<a id="xref-ref-11-3" class="xref-bibr" href="#ref-11">Bergland et al. 2014</a>; <a id="xref-ref-35-2" class="xref-bibr" href="#ref-35">Machado et al. 2018</a>). Efforts to quantify the impact of linked selection have found obligately sexual organisms have up to an 89% reduction in genome-wide diversity over long time periods (<a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">Comeron 2014</a>; <a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">Coop 2016</a>; <a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">Corbett-Detig et al. 2015</a>; <a id="xref-ref-23-3" class="xref-bibr" href="#ref-23">Elyashiv et al. 2016</a>; <a id="xref-ref-39-2" class="xref-bibr" href="#ref-39">McVicker et al. 2009</a>) Thus linked selection makes a sizeable contribution to long-term allele frequency change in some species, and there is reason to be hopeful that we could detect this from temporal data, which would help to resolve the timescales that linked selection act over. In our reanalysis of the <a id="xref-ref-4-15" class="xref-bibr" href="#ref-4">Barghi et al. (2019)</a> study, we find evidence of complex linked selection dynamics, with selection pressures flipping over time due to either environmental change, the breakup of epistatic combinations or advantageous haplotypes. Such patterns would be completely obscured in samples only from contemporary populations. Thus, we can hope to have a much richer picture of the impact of selection as temporal sequencing becomes more common, allowing us to observe the effects of ecological dynamics in genomic data (<a id="xref-ref-28-2" class="xref-bibr" href="#ref-28">Hairston et al. 2005</a>).</p><p id="p-27">Furthermore, understanding the dynamics of linked selection over short timescales will help to unite phenotypic studies of rapid adaptation with a detectable genomic signature, to address long-standing questions concerning linked selection, evolutionary quantitative genetics, and the overall impact selection has on genetic variation.</p></div><div class="section" id="sec-4"><h2 class="">Supplementary Material</h2><div class="section" id="sec-5"><h3>1.1 Estimator Bias Correction</h3><div class="section" id="sec-6"><h4>1.1.1 Correcting variance bias with a single depth sampling process</h4><p id="p-29">Following <a id="xref-ref-54-1" class="xref-bibr" href="#ref-54">Waples (1989)</a>, we have that the variance in allele frequency change at a locus in the initial generation, which is entirely due to the binomial sampling process, is Var(<em>p</em><sub>0</sub>) = <em>p</em><sub>0</sub>(1<em>−p</em><sub>0</sub>)<em>/d</em><sub>0</sub> where <em>d</em><sub>0</sub> is the number of binomial draws (e.g. read depth). At a later timepoint, the variance in allele frequency is a result of both the binomial sampling process at time <em>t</em> and the evolutionary process. Using the law of total variation we can partition the variation from each process,
<span class="disp-formula" id="disp-formula-3"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2019/10/09/798595/embed/graphic-7.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2019/10/09/798595/embed/graphic-7.gif"/></noscript></span>

</span>
<span class="disp-formula" id="disp-formula-4"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2019/10/09/798595/embed/graphic-8.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2019/10/09/798595/embed/graphic-8.gif"/></noscript></span>

</span></p><p id="p-30">Under a drift-only process, <span class="inline-formula" id="inline-formula-2"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2019/10/09/798595/embed/inline-graphic-2.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2019/10/09/798595/embed/inline-graphic-2.gif"/></noscript></span></span>. However, with heritable variation in fitness, we need to consider the covariance in allele frequency changes across generations (<a id="xref-ref-13-9" class="xref-bibr" href="#ref-13">Buffalo and Coop 2019</a>). We can write
<span class="disp-formula" id="disp-formula-5"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2019/10/09/798595/embed/graphic-9.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2019/10/09/798595/embed/graphic-9.gif"/></noscript></span>

</span>
<span class="disp-formula" id="disp-formula-6"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2019/10/09/798595/embed/graphic-10.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2019/10/09/798595/embed/graphic-10.gif"/></noscript></span>

</span>
<span class="disp-formula" id="disp-formula-7"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2019/10/09/798595/embed/graphic-11.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2019/10/09/798595/embed/graphic-11.gif"/></noscript></span>

</span></p><p id="p-31">Each allele frequency change is equally like to be positive as it is to be negative; thus by symmetry this second term is zero. Additionally Var(<em>p</em><sub>0</sub>) = 0, as we treat <em>p</em><sub>0</sub> as a fixed initial frequency. We can write,
<span class="disp-formula" id="disp-formula-8"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2019/10/09/798595/embed/graphic-12.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2019/10/09/798595/embed/graphic-12.gif"/></noscript></span>

</span></p><p id="p-32">The second term, the cumulative impact of variance in allele frequency change can be partitioned into heritable fitness and drift components (<a id="xref-ref-13-10" class="xref-bibr" href="#ref-13">Buffalo and Coop 2019</a>; <a id="xref-ref-46-3" class="xref-bibr" href="#ref-46">Santiago and Caballero 1995</a>)
<span class="disp-formula" id="disp-formula-9"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2019/10/09/798595/embed/graphic-13.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2019/10/09/798595/embed/graphic-13.gif"/></noscript></span>

</span>
where Δ<sub><em>H</em></sub><em>p</em><sub><em>t</em></sub> and Δ<sub><em>D</em></sub><em>p</em><sub><em>t</em></sub> indicate the allele frequency change due to heritable fitness variation and drift respectively. Then, sum of drift variances in allele frequency change is
<span class="disp-formula" id="disp-formula-10"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2019/10/09/798595/embed/graphic-14.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2019/10/09/798595/embed/graphic-14.gif"/></noscript></span>

</span>
replacing the heterozygosity in generation <em>i</em> with its expectation, we have
<span class="disp-formula" id="disp-formula-11"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2019/10/09/798595/embed/graphic-15.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2019/10/09/798595/embed/graphic-15.gif"/></noscript></span>

</span>
<span class="disp-formula" id="disp-formula-12"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2019/10/09/798595/embed/graphic-16.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2019/10/09/798595/embed/graphic-16.gif"/></noscript></span>

</span>
which is the usual variance in allele frequency change due to drift. Then, the total allele frequency change from generations 0 to <em>t</em> is <span class="inline-formula" id="inline-formula-3"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2019/10/09/798595/embed/inline-graphic-3.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2019/10/09/798595/embed/inline-graphic-3.gif"/></noscript></span></span>, where the covariance depends on the nature of the-sampling plan (see <a id="xref-ref-40-1" class="xref-bibr" href="#ref-40">Nei and Tajima 1981</a>; <a id="xref-ref-54-2" class="xref-bibr" href="#ref-54">Waples 1989</a>). In the case where there is heritable variation for fitness, and using the fact that <span class="inline-formula" id="inline-formula-4"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2019/10/09/798595/embed/inline-graphic-4.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2019/10/09/798595/embed/inline-graphic-4.gif"/></noscript></span></span> <span class="inline-formula" id="inline-formula-5"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2019/10/09/798595/embed/inline-graphic-5.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2019/10/09/798595/embed/inline-graphic-5.gif"/></noscript></span></span> for Plan I sampling procedures (<a id="xref-ref-54-3" class="xref-bibr" href="#ref-54">Waples 1989</a>), we write,
<span class="disp-formula" id="disp-formula-13"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2019/10/09/798595/embed/graphic-17.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2019/10/09/798595/embed/graphic-17.gif"/></noscript></span>

</span>
<span class="disp-formula" id="disp-formula-14"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2019/10/09/798595/embed/graphic-18.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2019/10/09/798595/embed/graphic-18.gif"/></noscript></span>

</span>
<span class="disp-formula" id="disp-formula-15"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2019/10/09/798595/embed/graphic-19.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2019/10/09/798595/embed/graphic-19.gif"/></noscript></span>

</span>
<span class="disp-formula" id="disp-formula-16"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2019/10/09/798595/embed/graphic-20.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2019/10/09/798595/embed/graphic-20.gif"/></noscript></span>

</span>
<span class="disp-formula" id="disp-formula-17"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2019/10/09/798595/embed/graphic-21.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2019/10/09/798595/embed/graphic-21.gif"/></noscript></span>

</span>
where <em>C</em> = 1 if Plan I is used, and <em>C</em> = 0 if Plan II is used (see <a id="xref-ref-54-4" class="xref-bibr" href="#ref-54">Waples 1989</a>, p. 380 and <a id="xref-fig-1-4" class="xref-fig" href="#F1">Figure 1</a> for a description of these sampling procedures; throughout the paper we use sampling Plan II). Rearranging, we can create a bias-corrected estimator for the population variance in allele frequency change, and replace all population heterozygosity terms with the unbiased sample estimators, e.g. <span class="inline-formula" id="inline-formula-6"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2019/10/09/798595/embed/inline-graphic-6.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2019/10/09/798595/embed/inline-graphic-6.gif"/></noscript></span></span>,
<span class="disp-formula" id="disp-formula-18"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2019/10/09/798595/embed/graphic-22.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2019/10/09/798595/embed/graphic-22.gif"/></noscript></span>

</span></p></div><div class="section" id="sec-7"><h4>1.1.2 Correcting variance bias with individual and depth sampling processes</h4><p id="p-33">Here, we extend the sampling bias correction described above to handle two binomial sampling processes: one as individuals are binomially sampled from the population, and another as reads are binomially sampled during sequencing. (see also <a id="xref-ref-32-1" class="xref-bibr" href="#ref-32">Jónás et al. 2016</a>). Let <em>X</em><sub><em>t</em></sub> ~ Binom(<em>n</em><sub><em>t</em></sub>, <em>p</em><sub><em>t</em></sub>) where <em>X</em><sub><em>t</em></sub> is the count of alleles and <em>n</em><sub><em>t</em></sub> is the number of diploids sampled at time <em>t</em>. Then, these individuals are sequenced at a depth of <em>d</em><sub><em>t</em></sub>, and <em>Y</em><sub><em>t</em></sub> ~ Binom(<em>d</em><sub><em>t</em></sub>, <em>X</em><sub><em>t</em></sub>/<em>n</em><sub><em>t</em></sub>) reads have the tracked allele. We let <span class="inline-formula" id="inline-formula-7"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2019/10/09/798595/embed/inline-graphic-7.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2019/10/09/798595/embed/inline-graphic-7.gif"/></noscript></span></span> be the observed sample allele frequency. Then, the sampling noise is
<span class="disp-formula" id="disp-formula-19"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2019/10/09/798595/embed/graphic-23.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2019/10/09/798595/embed/graphic-23.gif"/></noscript></span>

</span>
<span class="disp-formula" id="disp-formula-20"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2019/10/09/798595/embed/graphic-24.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2019/10/09/798595/embed/graphic-24.gif"/></noscript></span>

</span>
<span class="disp-formula" id="disp-formula-21"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2019/10/09/798595/embed/graphic-25.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2019/10/09/798595/embed/graphic-25.gif"/></noscript></span>

</span>
<span class="disp-formula" id="disp-formula-22"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2019/10/09/798595/embed/graphic-26.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2019/10/09/798595/embed/graphic-26.gif"/></noscript></span>

</span>
<span class="disp-formula" id="disp-formula-23"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2019/10/09/798595/embed/graphic-27.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2019/10/09/798595/embed/graphic-27.gif"/></noscript></span>

</span></p><p id="p-34">Through the law of total expectation (see <a id="xref-ref-34-1" class="xref-bibr" href="#ref-34">Kolaczkowski et al. 2011</a> Supplementary File 1 for a sample proof), one can find that an unbiased estimator of the half the heterozygosity is
<span class="disp-formula" id="disp-formula-24"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2019/10/09/798595/embed/graphic-28.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2019/10/09/798595/embed/graphic-28.gif"/></noscript></span>

</span></p><p id="p-35">Replacing this unbiased estimator for half of the heterozygosity into our expression above, the total sample variance is
<span class="disp-formula" id="disp-formula-25"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2019/10/09/798595/embed/graphic-29.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2019/10/09/798595/embed/graphic-29.gif"/></noscript></span>

</span>
</p><p id="p-36">As with equation (18), we can rearrange this to get a biased-corrected estimate of the variance in allele frequency change between adjacent generations, Var(Δ<em>p</em><sub><em>t</em></sub>).</p></div><div class="section" id="sec-8"><h4>1.1.3 Covariance Correction</h4><p id="p-37">We also need to apply a bias correction to the temporal covariances (and possibly the replicate covariances if the initial sample frequencies are all shared). The basic issue is that <span class="inline-formula" id="inline-formula-8"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2019/10/09/798595/embed/inline-graphic-8.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2019/10/09/798595/embed/inline-graphic-8.gif"/></noscript></span></span> <span class="inline-formula" id="inline-formula-9"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2019/10/09/798595/embed/inline-graphic-9.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2019/10/09/798595/embed/inline-graphic-9.gif"/></noscript></span></span>, and thus shares the sampling noise of timepoint <em>t</em> + 1. Thus acts to bias the covariance by subtracting off the noise variance term of <span class="inline-formula" id="inline-formula-10"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2019/10/09/798595/embed/inline-graphic-10.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2019/10/09/798595/embed/inline-graphic-10.gif"/></noscript></span></span>, so we add the expectation of this bias, derived above, back in. We discuss this in more detail below in deriving the bias correction for the temporal-replicate variance covariance matrix.</p></div><div class="section" id="sec-9"><h4>1.1.4 Temporal-Replicate Covariance Matrix Correction</h4><p id="p-38">In practice, we simultaneously estimate the temporal and replicate covariance matrices for each replicate, which we call the temporal-replicate covariance matrix. This needs a bias correction; we extend the bias corrections for single locus variance and covariance described in Supplementary Material <a id="xref-sec-6-1" class="xref-sec" href="#sec-6">Sections 1.1.1</a>, <a id="xref-sec-7-2" class="xref-sec" href="#sec-7">1.1.2</a>, and <a id="xref-sec-8-1" class="xref-sec" href="#sec-8">1.1.3</a> to multiple sampled loci and the temporal-replicate covariance matrix here. With frequency data collected at <em>T</em> + 1 timepoints across <em>R</em> replicate populations at <em>L</em> loci, we have multidimensional arrays <strong>F</strong> of allele frequencies, <strong>D</strong> of sequencing depths, and <strong>N</strong> of the number of individuals sequenced, each of dimension <em>R</em> × (<em>T</em> + 1) × <em>L</em>. We calculate the array <strong>ΔF</strong> which contains the allele frequency changes between adjacent generations, and has dimension <em>R</em> × <em>T</em> × <em>L</em>. The operation flat(<strong>ΔF</strong>) flattens this array to a (<em>R</em> · <em>T</em>) × <em>L</em> matrix, such that rows are grouped by replicate, e.g. for timepoint <em>t</em>, replicate <em>r</em>, and locus <em>l</em> such that for allele frequencies <em>p</em><sub><em>t</em>,<em>r</em>,<em>l</em></sub>, the frequency change entries are
<span class="disp-formula" id="disp-formula-26"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2019/10/09/798595/embed/graphic-30.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2019/10/09/798595/embed/graphic-30.gif"/></noscript></span>

</span>
where each Δ<em>p</em><sub><em>t</em>,<em>r</em>,<em>l</em></sub> = <em>p</em><sub><em>t</em> +1,<em>r</em>,<em>l</em></sub> − <em>p</em><sub><em>t</em>,<em>r</em>,<em>l</em></sub>. Then, the sample temporal-replicate covariance matrix <strong>Q</strong>′ calculated on flat(<strong>ΔF</strong>) is a (<em>R</em> · <em>T</em>) (<em>R</em> · <em>T</em>) matrix, with the <em>R</em> temporal-covariance block submatrices along the diagonal, and the <em>R</em>(<em>R</em> − 1) replicate-covariance submatrices matrices in the upper and lower triangles of the matrix,
<span class="disp-formula" id="disp-formula-27"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2019/10/09/798595/embed/graphic-31.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2019/10/09/798595/embed/graphic-31.gif"/></noscript></span>

</span>
where each submatrix <strong>Q</strong>′<sub><em>i</em>,<em>j</em></sub> (<em>i</em> ≠ <em>j</em>) is the <em>T</em> × <em>T</em> sample replicate covariance matrix for replicates <em>i</em> and <em>j</em>, and the submatrices along the diagonal <strong>Q</strong>′<sub><em>r</em>,<em>r</em></sub> are the temporal covariance matrices for replicate <em>r</em>.</p><p id="p-39">Given the bias of the sample covariance of allele frequency changes, we calculated an expected bias matrix <strong>B</strong>, averaging over loci,
<span class="disp-formula" id="disp-formula-28"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2019/10/09/798595/embed/graphic-32.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2019/10/09/798595/embed/graphic-32.gif"/></noscript></span>

</span>
where ○ denotes elementwise product, and <strong>h</strong><sub><em>l</em></sub>, <strong>d</strong><sub><em>l</em></sub>, and <strong>n</strong><sub><em>l</em></sub>, are rows corresponding to locus <em>l</em> of the unbiased heterozygosity arrays <strong>H</strong>, depth matrix <strong>D</strong>, and number of diploids matrix <strong>N</strong>. The unbiased <em>R</em> × (<em>T</em> + 1) × <em>L</em> heterozygosity array can be calculated as
<span class="disp-formula" id="disp-formula-29"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2019/10/09/798595/embed/graphic-33.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2019/10/09/798595/embed/graphic-33.gif"/></noscript></span>

</span>
where division here is elementwise. Thus, <strong>B</strong> is a <em>R</em> × (<em>T</em> +1) matrix. As explained in Supplementary Material <a id="xref-sec-7-3" class="xref-sec" href="#sec-7">Section 1.1.2</a> and <a id="xref-sec-8-2" class="xref-sec" href="#sec-8">1.1.3</a>, the temporal variances and covariances require bias corrections, meaning each temporal covariance submatrix <strong>Q</strong><sub><em>r</em>,<em>r</em></sub> requires two corrections. For an element <em>Q</em><sub><em>r</em>,<em>t</em>,<em>s</em></sub> = Cov(Δ<em>p</em><sub><em>t</em></sub>, Δ<em>p</em><sub><em>s</em></sub>) of the temporal covariance submatrix for replicate <em>r</em>, <strong>Q</strong><sub><em>r</em>,<em>r</em></sub>, we apply the following correction
<span class="disp-formula" id="disp-formula-30"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2019/10/09/798595/embed/graphic-34.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2019/10/09/798595/embed/graphic-34.gif"/></noscript></span>

</span>
where <em>b</em><sub><em>r</em>,<em>t</em></sub> is element in row <em>r</em> and column <em>t</em> of <strong>B</strong>.</p></div><div class="section" id="sec-10"><h4>1.1.5 <a id="xref-ref-4-16" class="xref-bibr" href="#ref-4">Barghi et al. (2019)</a> Temporal Covariances</h4><p id="p-40">Since each replicate population was sequenced every ten generations, the timepoints <em>t</em><sub>0</sub> = 0 generations, <em>t</em><sub>1</sub> = 10 generations, <em>t</em><sub>2</sub> = 20 generations, etc., lead to observed allele frequency changes across ten generation blocks, <span class="inline-formula" id="inline-formula-11"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2019/10/09/798595/embed/inline-graphic-11.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2019/10/09/798595/embed/inline-graphic-11.gif"/></noscript></span></span>. Consequently, the ten temporal covariance matrices for each of the ten replicate populations have off-diagonal elements of the form <span class="inline-formula" id="inline-formula-12"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2019/10/09/798595/embed/inline-graphic-12.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2019/10/09/798595/embed/inline-graphic-12.gif"/></noscript></span></span> <span class="inline-formula" id="inline-formula-13"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2019/10/09/798595/embed/inline-graphic-13.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2019/10/09/798595/embed/inline-graphic-13.gif"/></noscript></span></span>. Each diagonal element has the form <span class="inline-formula" id="inline-formula-14"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2019/10/09/798595/embed/inline-graphic-14.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2019/10/09/798595/embed/inline-graphic-14.gif"/></noscript></span></span> <span class="inline-formula" id="inline-formula-15"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2019/10/09/798595/embed/inline-graphic-15.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2019/10/09/798595/embed/inline-graphic-15.gif"/></noscript></span></span>, and is thus a combination of the effects of drift and selection, as both the variance in allele frequency changes and cumulative temporal autocovariances terms increase the variance in allele frequency. With sampling each generation, one could more accurately partition the total variance in allele frequency change (<a id="xref-ref-13-11" class="xref-bibr" href="#ref-13">Buffalo and Coop 2019</a>); while we cannot directly estimate the contribution of linked selection to the variance in allele frequency change here, the presence of a positive observed covariance between allele frequency change can only be caused linked selection.</p></div></div><div class="section" id="sec-11"><h3>1.2 Block Bootstrap Procedure</h3><p id="p-41">To infer the uncertainty of covariance, convergence correlation, and <em>G</em>(<em>t</em>) estimates, we used a block bootstrap procedure. This is a version of the bootstrap that resamples blocks of data points, rather than individual data points, to infer the uncertainty of an statistic in the presence of unknown correlation structure between data. With genome-wide data, linkage disequilibria between sites creates complex and unknown dependencies between variants. The estimators used in this paper are predominantly ratios, e.g. temporal-replicate covariance standardized by half the heterozygosity, <em>G</em>(<em>t</em>) which is the ratio of covariance to total variance, and the convergence correlation (<a id="xref-disp-formula-2-1" class="xref-disp-formula" href="#disp-formula-2">equation (2)</a>). In these cases, we can exploit the linearity of the expectation to make the bootstrap procedure more computationally efficient, by pre-calculating the statistics of the ratio’s numerator and denominator, <em>N</em> (<strong>x</strong><sub><em>i</em></sub>) and <em>D</em>(<strong>x</strong><sub><em>i</em></sub>), on the data <strong>x</strong><sub><em>i</em></sub> for all blocks <em>i</em> ∈ {1, 2, …, <em>W</em>} in the genome. Then we draw <em>W</em> bootstrap samples with replacement, and compute the estimate for bootstrap sample <em>b</em> with an average weighted by the fraction <em>w</em><sub><em>i</em></sub> of total loci contained in each block,
<span class="disp-formula" id="disp-formula-31"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2019/10/09/798595/embed/graphic-35.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2019/10/09/798595/embed/graphic-35.gif"/></noscript></span>

</span></p><p id="p-42">Note that computing the ratio of averages rather than the average of a ratio is a practice common for population genetic statistics like <em>F</em><sub><em>ST</em></sub> (<a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">Bhatia et al. 2013</a>). With these <em>B</em> bootstrap estimates, we calculate the <em>α</em>/2 and 1 − <em>α</em>/2 quantiles, which we use to estimate the 1 − <em>α</em> = 95% pivot confidence intervals (p. 33 <a id="xref-ref-55-1" class="xref-bibr" href="#ref-55">Wasserman 2006</a>, p. 194 <a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">Davison and Hinkley 2013</a>) throughout the paper,
<span class="disp-formula" id="disp-formula-32"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2019/10/09/798595/embed/graphic-36.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2019/10/09/798595/embed/graphic-36.gif"/></noscript></span>

</span>
where <span class="inline-formula" id="inline-formula-16"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2019/10/09/798595/embed/inline-graphic-16.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2019/10/09/798595/embed/inline-graphic-16.gif"/></noscript></span></span> is the estimate, and <em>q</em><sub><em>x</em></sub> is bootstrap quantile for probability <em>x</em>.</p></div><div class="section" id="sec-12"><h3>1.3 Replicate <em>G</em> and Partitioning the Variance in Allele Frequency</h3><p id="p-43">We define a statistic similar to <em>G</em> for estimating the proportion of allele frequency change common between two replicate populations due to linked selection. Covariance in allele frequency change between two replicate populations is due to convergent selection pressure selecting haplotypes shared between the two replicate populations, which acts to perturb linked neutral variation in parallel way.
<span class="disp-formula" id="disp-formula-33"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2019/10/09/798595/embed/graphic-37.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2019/10/09/798595/embed/graphic-37.gif"/></noscript></span>

</span>
where <span class="inline-formula" id="inline-formula-17"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2019/10/09/798595/embed/inline-graphic-17.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2019/10/09/798595/embed/inline-graphic-17.gif"/></noscript></span></span> indicates that the expectation is taken over all ordered pairs of replicates (e.g. summing all off-diagonal elements replicate covariances), and <span class="inline-formula" id="inline-formula-18"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2019/10/09/798595/embed/inline-graphic-18.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2019/10/09/798595/embed/inline-graphic-18.gif"/></noscript></span></span> indicates taking expectation over all replicates. This measures the fraction of variance in allele frequency change (averaged across replicates) due to shared selection pressure.</p><p id="p-44">Extending our theoretic work in <a id="xref-ref-13-12" class="xref-bibr" href="#ref-13">Buffalo and Coop (2019)</a>, we can partition the allele frequency change in two replicates into drift, and shared selection and replicate-specific selection components of allele frequency change. For two replicates, <em>A</em> and <em>B</em>,
<span class="disp-formula" id="disp-formula-34"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2019/10/09/798595/embed/graphic-38.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2019/10/09/798595/embed/graphic-38.gif"/></noscript></span>

</span>
<span class="disp-formula" id="disp-formula-35"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2019/10/09/798595/embed/graphic-39.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2019/10/09/798595/embed/graphic-39.gif"/></noscript></span>

</span>
where Δ<sub><em>D</em></sub><em>p</em><sub><em>t</em>,<em>A</em></sub> is allele frequency change due to drift (this is specific to a replicate, and equal to Δ<sub><em>N</em></sub><em>p</em><sub><em>t</em>,<em>A</em></sub> + Δ<sub><em>M</em></sub><em>p</em><sub><em>t</em>,<em>A</em></sub> in the notation of <a id="xref-ref-13-13" class="xref-bibr" href="#ref-13">Buffalo and Coop 2019</a>), Δ<sub><em>U</em></sub><em>p<sub><em>t</em>,<em>A</em></sub></em> is the allele frequency change from indirect selection specific to replicate <em>A</em> (and likewise with Δ<sub><em>U</em></sub><em>p<sub><em>t</em>,<em>A</em></sub></em> for replicate <em>B</em>), and Δ<sub><em>S</em></sub><em>p</em><sub><em>t</em></sub> is the allele frequency change from indirect selection shared across the replicates <em>A</em> and <em>B</em> (this term lacks a replicate subscript since by construction it is identical between replicates). By construction, each of these terms is uncorrelated, so the variance and be written as:
<span class="disp-formula" id="disp-formula-36"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2019/10/09/798595/embed/graphic-40.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2019/10/09/798595/embed/graphic-40.gif"/></noscript></span>

</span>
</p><p id="p-45">The shared effects of indirect selection can be quantified from the observed allele frequency changes, since the covariance in allele frequency change across replicates is the covariance of the shared term by construction,
<span class="disp-formula" id="disp-formula-37"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2019/10/09/798595/embed/graphic-41.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2019/10/09/798595/embed/graphic-41.gif"/></noscript></span>

</span></p><p id="p-46">In artificial selection studies with a control (non-selected) line, such as the <a id="xref-ref-15-6" class="xref-bibr" href="#ref-15">Castro et al. (2019)</a> study, this allows us to estimate the contribution of the effects of shared and unique indirect selection. In the case of this study, we can estimate the drift, unique selection effect, and shared selection effect terms using the fact that,
<span class="disp-formula" id="disp-formula-38"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2019/10/09/798595/embed/graphic-42.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2019/10/09/798595/embed/graphic-42.gif"/></noscript></span>

</span>
<span class="disp-formula" id="disp-formula-39"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2019/10/09/798595/embed/graphic-43.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2019/10/09/798595/embed/graphic-43.gif"/></noscript></span>

</span>
<span class="disp-formula" id="disp-formula-40"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2019/10/09/798595/embed/graphic-44.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2019/10/09/798595/embed/graphic-44.gif"/></noscript></span>

</span></p><p id="p-47">Note that since the control replicate does not undergo artificial selection, we assume that its allele frequency changes are determined entirely by genetic drift. With free mating individuals (such as in a cage population), this may not be the case, and sequencing adjacent generations would allow one to differentiate the effects of selection and drift.</p><p id="p-48">We assume that we can approximate the contribution of genetic drift in the Longshanks selection lines with the observed variance in the control line, or <span class="inline-formula" id="inline-formula-19"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2019/10/09/798595/embed/inline-graphic-19.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2019/10/09/798595/embed/inline-graphic-19.gif"/></noscript></span></span> <span class="inline-formula" id="inline-formula-20"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2019/10/09/798595/embed/inline-graphic-20.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2019/10/09/798595/embed/inline-graphic-20.gif"/></noscript></span></span>. Then, the combined effects of selection can be estimated by averaging the variances of the two Longshanks selection lines, and subtracting the variance in allele frequency change in the control line, which we treat as driven by drift alone (since matings are random). Note that each variance is bias-corrected according to the methods described in Supplementary Materials <a id="xref-sec-9-2" class="xref-sec" href="#sec-9">1.1.4</a>, and the average sequencing depths between lines are nearly identical. Thus, we have
<span class="disp-formula" id="disp-formula-41"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2019/10/09/798595/embed/graphic-45.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2019/10/09/798595/embed/graphic-45.gif"/></noscript></span>

</span>
where the bar indicates values averaged both Longshanks selection lines. Additionally, use the fact that
<span class="disp-formula" id="disp-formula-42"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2019/10/09/798595/embed/graphic-46.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2019/10/09/798595/embed/graphic-46.gif"/></noscript></span>

</span>
we can also separate out the unique and shared components by subtracting off this covariance,
<span class="disp-formula" id="disp-formula-43"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2019/10/09/798595/embed/graphic-47.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2019/10/09/798595/embed/graphic-47.gif"/></noscript></span>

</span></p><p id="p-49">Finally, we can divide each of these values by the total variance to get the proportion of total variance drift, and unique and shared effects of selection contribute towards the total. To derive confidence intervals for the estimates of unique and shared effects of selection, we use a block bootstrap procedure as described in Supplementary Materials <a id="xref-sec-11-1" class="xref-sec" href="#sec-11">Section 1.2</a>.</p></div><div class="section" id="sec-13"><h3>1.4 The Empirical Neutral Null Windowed Covariance Distribution</h3><p id="p-50">To detect an excess of genomic regions with unusually high or low covariances, we need to compare the distribution of observed windowed covariances to a null distribution of windowed covariances that we would expect under no selection. While we could construct a theoretic sampling distribution of the spurious covariances created by neutral genetic drift at particular site, the unknown linkage disequilibrium between sites would mean that this is not an adequate null model for the distribution of windowed covariances in our data.</p><p id="p-51">To address this limitation, we construct a neutral null model by sign-permuting the observed allele frequency changes. This destroys the covariances built up by selection, mimicking a neutral allele’s frequency trajectory. This approach is conservative, since selection also acts to increase the magnitude of allele frequency changes (see <a id="xref-disp-formula-1-1" class="xref-disp-formula" href="#disp-formula-1">equation 1</a> of <a id="xref-ref-13-14" class="xref-bibr" href="#ref-13">Buffalo and Coop 2019</a>), but this magnitude is not affected by the sign-permutation procedure. Consequently, the resulting empirical null distribution has higher variance than would be expected under neutrality alone.</p><p id="p-52">Still, we wanted to ensure that LD between sign-permuted blocks, which will affect the variance of the empirical null distribution, does not impact our primary finding that the distribution of temporal covariances becomes increasingly negative in the <a id="xref-ref-4-17" class="xref-bibr" href="#ref-4">Barghi et al. (2019)</a> dataset through time. To address this, we also sign-permuted at the whole chromosome level finding we recapitulate the same pattern (<a id="xref-fig-15-2" class="xref-fig" href="#F15">Supplementary Figure S12</a>).</p></div><div class="section" id="sec-14"><h3>1.5 <a id="xref-ref-11-4" class="xref-bibr" href="#ref-11">Bergland et al. (2014)</a> Re-Analysis</h3><p id="p-53">We also applied our temporal covariance approach to <a id="xref-ref-11-5" class="xref-bibr" href="#ref-11">Bergland et al. (2014)</a>, which found evidence of genome-wide fluctuating selection between Spring and Fall seasons across three years <em>Drosophila melanogaster</em>. As described in <a id="xref-ref-13-15" class="xref-bibr" href="#ref-13">Buffalo and Coop (2019)</a>, if fluctuating selection pressure among time-periods are the dominant genome-wide pattern, we might expect positive covariances between like seasons changes (e.g. Spring 2010 to Fall 2010 and Spring 2011 to Fall 2011), and negative covariances between dislike seasonal changes (e.g. Fall 2009 to Spring 2010 and Fall 2010 to Spring 2011). However, while we find temporal covariances that are non-zero, we find only weak support for a seasonal fluctuating model driving these covariances. In <a id="xref-fig-4-1" class="xref-fig" href="#F4">Supplementary Figure S1</a>, we show the temporal covariances from varying reference generations, across seasonal transitions that are alike (e.g. the covariance between the allele frequency changes between Fall 2009 and Spring 2009, and frequency changes between Fall 2010 and Spring 2010), and dislike (e.g. the covariance between the allele frequency change between Fall 2009 and Spring 2009, and the frequency changes between Spring 2010 and Fall 2009). The first row of temporal covariance matrix is consistent with fluctuating selection operating for two timepoints, as the first covariance is negative, and the second is positive, and later covariances are not statistically differentiable from zero (which could occur if LD and additive genetic variance decay). However, all other temporal covariances do not fit the pattern we would expect under genome-wide fluctuating selection.</p><p id="p-54">We wanted to establish that our temporal-covariance matrix bias correction was working correctly. We find that it corrects the relationship between depth and both variance and covariance (<a id="xref-fig-7-1" class="xref-fig" href="#F7">Supplementary Figure S4</a>) as expected.</p><p id="p-55">It is unclear how strong the fluctuations would have to be to generate a genome-wide average signal of fluctuating selection from temporal covariances. For example, many loci could still show a signal of fluctuating selection, but the average signal could be overwhelmed by other signals of other selection. To investigate whether there was a genome-wide excess of loci showing evidence of fluctuating selection we reanalyzed the data of (<a id="xref-ref-11-6" class="xref-bibr" href="#ref-11">Bergland et al. 2014</a>) using the same seasonal fluctuating model as the original paper. This model is a Binomial logit-linked GLM fit per-locus, where the frequencies are regressed on the Spring/Fall seasons are encoded as a dummy variable. We use the same binomial weighting procedure as <a id="xref-ref-11-7" class="xref-bibr" href="#ref-11">Bergland et al. (2014)</a>, where the weights are determined by the effective number of chromosomes, <em>N</em><sub><em>eff</em></sub> = (2<em>n</em><sub><em>t</em></sub><em>d</em><sub><em>t</em></sub> − 1)/(2<em>n</em><sub><em>t</em></sub> + <em>d</em><sub><em>t</em></sub>) (<em>n</em><sub><em>t</em></sub> and <em>d</em><sub><em>t</em></sub> are the number of diploid individuals and the read depth at timepoint <em>t</em>, respectively). We fit this model on all loci marked as used in the VCF provided with the <a id="xref-ref-11-8" class="xref-bibr" href="#ref-11">Bergland et al. (2014)</a> study (doi:10.5061/dryad.v883p). Overall, our p-values for the Wald test for each locus closely match those of the original paper (Pearson correlation coefficient 0.98, p-value &lt; 2.2 × 10<sup>−16</sup>; see <a id="xref-fig-5-1" class="xref-fig" href="#F5">Supplementary Figure S2 A</a>), and the histograms of the p-values are nearly identical (<a id="xref-fig-5-2" class="xref-fig" href="#F5">Supplementary Figure S2 B</a>). <a id="xref-ref-11-9" class="xref-bibr" href="#ref-11">Bergland et al. (2014)</a> find loci with a significant association with season after a Benjamini and Hochberg FDR p-value adjustment (<a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">Benjamini and Hochberg 1995</a>), however, the null hypothesis of the Wald test does not give us an idea of the expected number of variants that may spuriously fit the pattern of seasonal fluctuating selection as it does not account for genetic drift or other forms of hitchhiking.</p><div id="F4" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/10/09/798595/F4.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Temporal covariances from the Bergland et al. (2014) study, from varying reference generations (e.g. rows along the temporal covariance matrix). Each covariance is labeled indicating whether the covariance is between two like seasonal transitions (e.g. the covariance between allele frequency changes from fall to spring in one year, and fall to spring in another) or two dislike seasons (e.g. the covariance between fall to spring in one year, and spring to fall in another year). Covariances between like transitions are expected to be positive when there is a genome-wide effect of fluctuating selection (and these labels are colored blue), while covariances between dislike transitions are expected to be negative (and these labels are colored red). 95% confidence intervals were constructed by a block-bootstrapping procedure where the blocks are megabase tiles." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-290610267" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Temporal covariances from the Bergland et al. (2014) study, from varying reference generations (e.g. rows along the temporal covariance matrix). Each covariance is labeled indicating whether the covariance is between two like seasonal transitions (e.g. the covariance between allele frequency changes from fall to spring in one year, and fall to spring in another) or two dislike seasons (e.g. the covariance between fall to spring in one year, and spring to fall in another year). Covariances between like transitions are expected to be positive when there is a genome-wide effect of fluctuating selection (and these labels are colored blue), while covariances between dislike transitions are expected to be negative (and these labels are colored red). 95% confidence intervals were constructed by a block-bootstrapping procedure where the blocks are megabase tiles.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure S1:" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2019/10/09/798595/F4.medium.gif" width="440" height="214"/><noscript><img class="highwire-fragment fragment-image" alt="Figure S1:" src="https://www.biorxiv.org/content/biorxiv/early/2019/10/09/798595/F4.medium.gif" width="440" height="214"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/10/09/798595/F4.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure S1:" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/10/09/798595/F4.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure S1:</span> <p id="p-56" class="first-child">Temporal covariances from the <a id="xref-ref-11-10" class="xref-bibr" href="#ref-11">Bergland et al. (2014)</a> study, from varying reference generations (e.g. rows along the temporal covariance matrix). Each covariance is labeled indicating whether the covariance is between two like seasonal transitions (e.g. the covariance between allele frequency changes from fall to spring in one year, and fall to spring in another) or two dislike seasons (e.g. the covariance between fall to spring in one year, and spring to fall in another year). Covariances between like transitions are expected to be positive when there is a genome-wide effect of fluctuating selection (and these labels are colored blue), while covariances between dislike transitions are expected to be negative (and these labels are colored red). 95% confidence intervals were constructed by a block-bootstrapping procedure where the blocks are megabase tiles.</p><div class="sb-div caption-clear"></div></div></div><div id="F5" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/10/09/798595/F5.large.jpg?width=800&amp;height=600&amp;carousel=1" title="A: Scatterplot of the original unadjusted p-values from Bergland et al. (2014) and the p-values from our reanalysis of the same data using the same statistical methods; the minor discrepancy is likely due to software version differences. B: The histograms of the p-values of our reanalysis and the original Bergland et al. (2014) data; again the minor discrepancy is likely due to software differences. Overall, our implementation of Bergland et al.&#x2019;s statistical methods produces results very close to the original analysis." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-290610267" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;&lt;strong&gt;A&lt;/strong&gt;: Scatterplot of the original unadjusted p-values from Bergland et al. (2014) and the p-values from our reanalysis of the same data using the same statistical methods; the minor discrepancy is likely due to software version differences. &lt;strong&gt;B&lt;/strong&gt;: The histograms of the p-values of our reanalysis and the original Bergland et al. (2014) data; again the minor discrepancy is likely due to software differences. Overall, our implementation of Bergland et al.&#x2019;s statistical methods produces results very close to the original analysis.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure S2:" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2019/10/09/798595/F5.medium.gif" width="440" height="167"/><noscript><img class="highwire-fragment fragment-image" alt="Figure S2:" src="https://www.biorxiv.org/content/biorxiv/early/2019/10/09/798595/F5.medium.gif" width="440" height="167"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/10/09/798595/F5.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure S2:" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/10/09/798595/F5.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure S2:</span> <p id="p-57" class="first-child"><strong>A</strong>: Scatterplot of the original unadjusted p-values from <a id="xref-ref-11-11" class="xref-bibr" href="#ref-11">Bergland et al. (2014)</a> and the p-values from our reanalysis of the same data using the same statistical methods; the minor discrepancy is likely due to software version differences. <strong>B</strong>: The histograms of the p-values of our reanalysis and the original <a id="xref-ref-11-12" class="xref-bibr" href="#ref-11">Bergland et al. (2014)</a> data; again the minor discrepancy is likely due to software differences. Overall, our implementation of Bergland et al.’s statistical methods produces results very close to the original analysis.</p><div class="sb-div caption-clear"></div></div></div><div id="F6" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/10/09/798595/F6.large.jpg?width=800&amp;height=600&amp;carousel=1" title="A: Histogram of original Bergland et al. (2014) seasonal p-values and p-values creating by randomly permuting the seasons at each locus. B: Histogram of original Bergland et al. (2014) p-values alongside all unique permutations (ignoring symmetries that lead to identical p-values)." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-290610267" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;&lt;strong&gt;A&lt;/strong&gt;: Histogram of original Bergland et al. (2014) seasonal p-values and p-values creating by randomly permuting the seasons at each locus. &lt;strong&gt;B&lt;/strong&gt;: Histogram of original Bergland et al. (2014) p-values alongside all unique permutations (ignoring symmetries that lead to identical p-values).&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure S3:" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2019/10/09/798595/F6.medium.gif" width="440" height="168"/><noscript><img class="highwire-fragment fragment-image" alt="Figure S3:" src="https://www.biorxiv.org/content/biorxiv/early/2019/10/09/798595/F6.medium.gif" width="440" height="168"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/10/09/798595/F6.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure S3:" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/10/09/798595/F6.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure S3:</span> <p id="p-58" class="first-child"><strong>A</strong>: Histogram of original <a id="xref-ref-11-13" class="xref-bibr" href="#ref-11">Bergland et al. (2014)</a> seasonal p-values and p-values creating by randomly permuting the seasons at each locus. <strong>B</strong>: Histogram of original <a id="xref-ref-11-14" class="xref-bibr" href="#ref-11">Bergland et al. (2014)</a> p-values alongside all unique permutations (ignoring symmetries that lead to identical p-values).</p><div class="sb-div caption-clear"></div></div></div><p id="p-59">To investigate whether there is a genome-wide evidence of an enrichment of fluctuating selection we created an empirical null distribution by randomly permuting the season labels and re-running the per-locus seasonal GLM model, as proposed by Machado et al. (2018). We find, regardless of whether we permute at the locus-level or the permutation replicate-level, that the observed seasonal p-value distribution <a id="xref-ref-11-15" class="xref-bibr" href="#ref-11">Bergland et al. (2014)</a> is not enriched for significant p-values beyond what we would expect from the permutation null. In fact, there appears there is more enrichment for low p-values when seasonal labels are randomly permuted (<a id="xref-fig-6-1" class="xref-fig" href="#F6">Supplementary Figure S3</a>, suggesting by random chance we might expect more variants with a seasonal fluctuating pattern than found in the original <a id="xref-ref-11-16" class="xref-bibr" href="#ref-11">Bergland et al. (2014)</a> study. While surprising, this could be explained by the presence of temporal structure across the samples not consistent with seasonal fluctuating selection. Some fraction of the permutations happen to fit this structure well, leading to an enrichment of small p-values. This non-seasonal temporal structure is also evident in our temporal covariances (<a id="xref-fig-4-2" class="xref-fig" href="#F4">Supplementary Figure S1</a>), where we see strong evidence of selection (non-zero temporal covariances), yet the pattern does not follow that of seasonal fluctuating selection.</p><div id="F7" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/10/09/798595/F7.large.jpg?width=800&amp;height=600&amp;carousel=1" title="The variance and covariances from the Bergland et al. (2014) study, calculated in 100kb genomic windows plotted against average depth in a window before and after bias correction. Each panel has a least-squares estimate between the variance and covariance, and the average depth. The bias correction procedure is correcting sampling bias in both the variance and covariance such that the relationship with depth is constant. Colors indicate the different chromosomes of D. melanogaster; we have excluded the X chromosome (yellow points; chromosome 4 was not in the original study) from the regression due to large differences in average coverage." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-290610267" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;The variance and covariances from the Bergland et al. (2014) study, calculated in 100kb genomic windows plotted against average depth in a window before and after bias correction. Each panel has a least-squares estimate between the variance and covariance, and the average depth. The bias correction procedure is correcting sampling bias in both the variance and covariance such that the relationship with depth is constant. Colors indicate the different chromosomes of &lt;em&gt;D. melanogaster&lt;/em&gt;; we have excluded the X chromosome (yellow points; chromosome 4 was not in the original study) from the regression due to large differences in average coverage.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure S4:" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2019/10/09/798595/F7.medium.gif" width="440" height="252"/><noscript><img class="highwire-fragment fragment-image" alt="Figure S4:" src="https://www.biorxiv.org/content/biorxiv/early/2019/10/09/798595/F7.medium.gif" width="440" height="252"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/10/09/798595/F7.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure S4:" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/10/09/798595/F7.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure S4:</span> <p id="p-60" class="first-child">The variance and covariances from the <a id="xref-ref-11-17" class="xref-bibr" href="#ref-11">Bergland et al. (2014)</a> study, calculated in 100kb genomic windows plotted against average depth in a window before and after bias correction. Each panel has a least-squares estimate between the variance and covariance, and the average depth. The bias correction procedure is correcting sampling bias in both the variance and covariance such that the relationship with depth is constant. Colors indicate the different chromosomes of <em>D. melanogaster</em>; we have excluded the X chromosome (yellow points; chromosome 4 was not in the original study) from the regression due to large differences in average coverage.</p><div class="sb-div caption-clear"></div></div></div></div><div class="section" id="sec-15"><h3>1.6 Bias Correction for <a id="xref-ref-4-18" class="xref-bibr" href="#ref-4">Barghi et al. (2019)</a></h3><p id="p-61">We have investigated the effectiveness of our correction on real data by exploiting the relationship between sampling depth and the magnitude of the variance and covariance biases, and comparing the observed variances and covariances before and after correction. We plot the variance and covariance (between adjacent timepoints) before and after the bias correction against the average sample depth in 100kb genomic windows in <a id="xref-fig-8-1" class="xref-fig" href="#F8">Figure S5</a>. Overall, we find the biased-correction procedure removes the relationship between variance and covariance and depth, indicating it is working adequately.</p><div id="F8" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/10/09/798595/F8.large.jpg?width=800&amp;height=600&amp;carousel=1" title="The variance and covariances from the Barghi et al. (2019) study, calculated in 100kb genomic windows plotted against average depth in a window before and after bias correction. Each panel has a leastsquares estimate between the variance and covariance, and the average depth. Overall, the bias correction corrects sampling bias in both the variance and covariance such that the relationship with depth is constant. Colors indicate the different chromosomes of D. simulans; we have excluded the X chromosome (yellow points) and chromosome 4 points (green points to far right) from the regression due to large differences in average coverage." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-290610267" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;The variance and covariances from the Barghi et al. (2019) study, calculated in 100kb genomic windows plotted against average depth in a window before and after bias correction. Each panel has a leastsquares estimate between the variance and covariance, and the average depth. Overall, the bias correction corrects sampling bias in both the variance and covariance such that the relationship with depth is constant. Colors indicate the different chromosomes of &lt;em&gt;D. simulans&lt;/em&gt;; we have excluded the X chromosome (yellow points) and chromosome 4 points (green points to far right) from the regression due to large differences in average coverage.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure S5:" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2019/10/09/798595/F8.medium.gif" width="440" height="253"/><noscript><img class="highwire-fragment fragment-image" alt="Figure S5:" src="https://www.biorxiv.org/content/biorxiv/early/2019/10/09/798595/F8.medium.gif" width="440" height="253"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/10/09/798595/F8.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure S5:" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/10/09/798595/F8.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure S5:</span> <p id="p-62" class="first-child">The variance and covariances from the <a id="xref-ref-4-19" class="xref-bibr" href="#ref-4">Barghi et al. (2019)</a> study, calculated in 100kb genomic windows plotted against average depth in a window before and after bias correction. Each panel has a leastsquares estimate between the variance and covariance, and the average depth. Overall, the bias correction corrects sampling bias in both the variance and covariance such that the relationship with depth is constant. Colors indicate the different chromosomes of <em>D. simulans</em>; we have excluded the X chromosome (yellow points) and chromosome 4 points (green points to far right) from the regression due to large differences in average coverage.</p><div class="sb-div caption-clear"></div></div></div><div class="section" id="sec-16"><h4>1.6.1 <a id="xref-ref-4-20" class="xref-bibr" href="#ref-4">Barghi et al. (2019)</a> Temporal Covariances Per Replicate</h4><div id="F9" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/10/09/798595/F9.large.jpg?width=800&amp;height=600&amp;carousel=1" title="The temporal covariances from the Barghi et al. (2019) study, for each replicate individually. As in Figure 1, each line follows the temporal covariances from some initial reference generation through time, which represent the rows of temporal covariance matrix." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-290610267" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;The temporal covariances from the Barghi et al. (2019) study, for each replicate individually. As in Figure 1, each line follows the temporal covariances from some initial reference generation through time, which represent the rows of temporal covariance matrix.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure S6:" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2019/10/09/798595/F9.medium.gif" width="440" height="173"/><noscript><img class="highwire-fragment fragment-image" alt="Figure S6:" src="https://www.biorxiv.org/content/biorxiv/early/2019/10/09/798595/F9.medium.gif" width="440" height="173"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/10/09/798595/F9.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure S6:" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/10/09/798595/F9.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure S6:</span> <p id="p-63" class="first-child">The temporal covariances from the <a id="xref-ref-4-21" class="xref-bibr" href="#ref-4">Barghi et al. (2019)</a> study, for each replicate individually. As in <a id="xref-fig-1-5" class="xref-fig" href="#F1">Figure 1</a>, each line follows the temporal covariances from some initial reference generation through time, which represent the rows of temporal covariance matrix.</p><div class="sb-div caption-clear"></div></div></div><div id="T2" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/945675/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup"><a href="/highwire/markup/945675/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/945675" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table S1:</span> <p id="p-64" class="first-child">Table of median of windowed covariance estimates (Cov(Δ<em>p</em><sub><em>s</em></sub>, Δ<em>p</em><sub><em>t</em></sub>) × 100) between generations <em>t</em> and <em>s</em> and the trimmed mean windowed covariance which excludes the lower and upper 5% windows with the highest covariance.</p><div class="sb-div caption-clear"></div></div></div></div></div><div class="section" id="sec-17"><h3>1.7 <a id="xref-ref-4-22" class="xref-bibr" href="#ref-4">Barghi et al. (2019)</a> Trimmed Window Covariances</h3><p id="p-65">Here we report median and trimmed mean of the windowed covariances (<a id="xref-table-wrap-2-2" class="xref-table" href="#T2">Supplementary Table S1</a>). We note that the median covariance is also limiting result of a trimmed mean that symmetrically excludes the upper and lower <em>α</em> tails to calculate the trimmed average windowed covariance. As <em>α</em> increases to 0.5, the trimmed covariance converges to the median windowed covariance (by the definition of the median; see <a id="xref-fig-10-2" class="xref-fig" href="#F10">Supplementary Figure S7</a>). Thus our genomic temporal covariances are non-zero due to the impact of selection on many genomic windows.</p><div id="F10" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/10/09/798595/F10.large.jpg?width=800&amp;height=600&amp;carousel=1" title="The genome-wide covariance (Cov(&#x394;p0, &#x394;p10) pooling all replicates) averaged (red line) and the median windowed covariance (blue) for the Barghi et al. (2019) dataset. The trimmed average window covariance, excluding the &#x3B1; lower and upper tails, converges to the median windowed covariance. This indicates that genome-wide covariance are not being overly dominated by a large-effect loci in few windows." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-290610267" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;The genome-wide covariance (Cov(&#x394;&lt;em&gt;p&lt;/em&gt;&lt;sub&gt;0&lt;/sub&gt;, &#x394;&lt;em&gt;p&lt;/em&gt;&lt;sub&gt;1&lt;/sub&gt;0) pooling all replicates) averaged (red line) and the median windowed covariance (blue) for the Barghi et al. (2019) dataset. The trimmed average window covariance, excluding the &lt;em&gt;&#x3B1;&lt;/em&gt; lower and upper tails, converges to the median windowed covariance. This indicates that genome-wide covariance are not being overly dominated by a large-effect loci in few windows.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure S7:" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2019/10/09/798595/F10.medium.gif" width="440" height="294"/><noscript><img class="highwire-fragment fragment-image" alt="Figure S7:" src="https://www.biorxiv.org/content/biorxiv/early/2019/10/09/798595/F10.medium.gif" width="440" height="294"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/10/09/798595/F10.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure S7:" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/10/09/798595/F10.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure S7:</span> <p id="p-66" class="first-child">The genome-wide covariance (Cov(Δ<em>p</em><sub>0</sub>, Δ<em>p</em><sub>1</sub>0) pooling all replicates) averaged (red line) and the median windowed covariance (blue) for the <a id="xref-ref-4-23" class="xref-bibr" href="#ref-4">Barghi et al. (2019)</a> dataset. The trimmed average window covariance, excluding the <em>α</em> lower and upper tails, converges to the median windowed covariance. This indicates that genome-wide covariance are not being overly dominated by a large-effect loci in few windows.</p><div class="sb-div caption-clear"></div></div></div></div><div class="section" id="sec-18"><h3>1.8 <a id="xref-ref-4-24" class="xref-bibr" href="#ref-4">Barghi et al. (2019)</a> Empirical Null and Windowed Covariance Distributions</h3><div id="F11" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/10/09/798595/F11.large.jpg?width=800&amp;height=600&amp;carousel=1" title="We chose number of bins used in the histograms of Figure 3 via an analytic expression for the cross-validation risk, based on the equation 6.16 of (Wasserman 2006, p. 129). Above, we plot the cross-validation risk for various numbers of bins, for each of the four off-diagonals of the temporal covariance matrix that we analyze. Overall, because the number of data points is large, oversmoothing is less of a problem, leading the cross-validation risk to be relatively flat across a large number of bins. Each gray point indicates the minimal risk for a particular off-diagonal, and the dashed line indicates the best average binwidth across off-diagonals." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-290610267" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;We chose number of bins used in the histograms of Figure 3 via an analytic expression for the cross-validation risk, based on the equation 6.16 of (Wasserman 2006, p. 129). Above, we plot the cross-validation risk for various numbers of bins, for each of the four off-diagonals of the temporal covariance matrix that we analyze. Overall, because the number of data points is large, oversmoothing is less of a problem, leading the cross-validation risk to be relatively flat across a large number of bins. Each gray point indicates the minimal risk for a particular off-diagonal, and the dashed line indicates the best average binwidth across off-diagonals.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure S8:" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2019/10/09/798595/F11.medium.gif" width="440" height="417"/><noscript><img class="highwire-fragment fragment-image" alt="Figure S8:" src="https://www.biorxiv.org/content/biorxiv/early/2019/10/09/798595/F11.medium.gif" width="440" height="417"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/10/09/798595/F11.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure S8:" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/10/09/798595/F11.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure S8:</span> <p id="p-67" class="first-child">We chose number of bins used in the histograms of <a id="xref-fig-3-5" class="xref-fig" href="#F3">Figure 3</a> via an analytic expression for the cross-validation risk, based on the <a id="xref-disp-formula-6-1" class="xref-disp-formula" href="#disp-formula-6">equation 6</a>.16 of (<a id="xref-ref-55-2" class="xref-bibr" href="#ref-55">Wasserman 2006</a>, p. 129). Above, we plot the cross-validation risk for various numbers of bins, for each of the four off-diagonals of the temporal covariance matrix that we analyze. Overall, because the number of data points is large, oversmoothing is less of a problem, leading the cross-validation risk to be relatively flat across a large number of bins. Each gray point indicates the minimal risk for a particular off-diagonal, and the dashed line indicates the best average binwidth across off-diagonals.</p><div class="sb-div caption-clear"></div></div></div><div id="F12" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/10/09/798595/F12.large.jpg?width=800&amp;height=600&amp;carousel=1" title="The distribution of windowed temporal covariances alongside the empirical neutral null for five randomly sampled replicates (columns), for k = 2 (first row) and k = 5 (second row). The main figure of the paper pools all replicate window and empirical neutral null covariances; we show here the windowed temporal covariances tend to shift from being positive (a heavier right tail) to become more negative (a heavier left tail) through time within particular replicates." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-290610267" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;The distribution of windowed temporal covariances alongside the empirical neutral null for five randomly sampled replicates (columns), for &lt;em&gt;k&lt;/em&gt; = 2 (first row) and &lt;em&gt;k&lt;/em&gt; = 5 (second row). The main figure of the paper pools all replicate window and empirical neutral null covariances; we show here the windowed temporal covariances tend to shift from being positive (a heavier right tail) to become more negative (a heavier left tail) through time within particular replicates.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure S9:" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2019/10/09/798595/F12.medium.gif" width="440" height="184"/><noscript><img class="highwire-fragment fragment-image" alt="Figure S9:" src="https://www.biorxiv.org/content/biorxiv/early/2019/10/09/798595/F12.medium.gif" width="440" height="184"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/10/09/798595/F12.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure S9:" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/10/09/798595/F12.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure S9:</span> <p id="p-68" class="first-child">The distribution of windowed temporal covariances alongside the empirical neutral null for five randomly sampled replicates (columns), for <em>k</em> = 2 (first row) and <em>k</em> = 5 (second row). The main figure of the paper pools all replicate window and empirical neutral null covariances; we show here the windowed temporal covariances tend to shift from being positive (a heavier right tail) to become more negative (a heavier left tail) through time within particular replicates.</p><div class="sb-div caption-clear"></div></div></div><div id="F13" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/10/09/798595/F13.large.jpg?width=800&amp;height=600&amp;carousel=1" title="The distribution of temporal covariances calculated across 100kb genomic windows from Barghi et al. (2019)&#x2019;s study (orange) and the block sign permuted empirical neutral null distribution of the windowed covariances (blue). Each panel shows these windowed covariances and the empirical null distribution for covariances Cov(&#x394;pt, &#x394;pt+k), k is the number of generations between allele frequency changes." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-290610267" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;The distribution of temporal covariances calculated across 100kb genomic windows from Barghi et al. (2019)&#x2019;s study (orange) and the block sign permuted empirical neutral null distribution of the windowed covariances (blue). Each panel shows these windowed covariances and the empirical null distribution for covariances Cov(&#x394;&lt;em&gt;p&lt;/em&gt;&lt;sub&gt;&lt;em&gt;t&lt;/em&gt;&lt;/sub&gt;, &#x394;&lt;em&gt;p&lt;/em&gt;&lt;sub&gt;&lt;em&gt;t&lt;/em&gt;+&lt;em&gt;k&lt;/em&gt;&lt;/sub&gt;), &lt;em&gt;k&lt;/em&gt; is the number of generations between allele frequency changes.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure S10:" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2019/10/09/798595/F13.medium.gif" width="440" height="285"/><noscript><img class="highwire-fragment fragment-image" alt="Figure S10:" src="https://www.biorxiv.org/content/biorxiv/early/2019/10/09/798595/F13.medium.gif" width="440" height="285"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/10/09/798595/F13.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure S10:" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/10/09/798595/F13.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure S10:</span> <p id="p-69" class="first-child">The distribution of temporal covariances calculated across 100kb genomic windows from <a id="xref-ref-4-25" class="xref-bibr" href="#ref-4">Barghi et al. (2019)</a>’s study (orange) and the block sign permuted empirical neutral null distribution of the windowed covariances (blue). Each panel shows these windowed covariances and the empirical null distribution for covariances Cov(Δ<em>p</em><sub><em>t</em></sub>, Δ<em>p</em><sub><em>t</em>+<em>k</em></sub>), <em>k</em> is the number of generations between allele frequency changes.</p><div class="sb-div caption-clear"></div></div></div></div><div class="section" id="sec-19"><h3>1.9 <a id="xref-ref-4-26" class="xref-bibr" href="#ref-4">Barghi et al. (2019)</a> Tail Probabilities for Windowed Covariances Distributions</h3><div id="F14" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/10/09/798595/F14.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Barghi et al. (2019) tail probabilities compared to sign-permuted empirical null distribution for various &#x3B1; levels." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-290610267" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Barghi et al. (2019) tail probabilities compared to sign-permuted empirical null distribution for various &lt;em&gt;&#x3B1;&lt;/em&gt; levels.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure S11:" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2019/10/09/798595/F14.medium.gif" width="440" height="218"/><noscript><img class="highwire-fragment fragment-image" alt="Figure S11:" src="https://www.biorxiv.org/content/biorxiv/early/2019/10/09/798595/F14.medium.gif" width="440" height="218"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/10/09/798595/F14.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure S11:" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/10/09/798595/F14.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure S11:</span> <p id="p-70" class="first-child"><a id="xref-ref-4-27" class="xref-bibr" href="#ref-4">Barghi et al. (2019)</a> tail probabilities compared to sign-permuted empirical null distribution for various <em>α</em> levels.</p><div class="sb-div caption-clear"></div></div></div><div id="F15" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/10/09/798595/F15.large.jpg?width=800&amp;height=600&amp;carousel=1" title="The 20% lower and upper tail probabilities for the observed windowed covariances from the Barghi et al. (2019) study, based on sign-permuting at the chromosome level. This permutation empirical null is robust to long-range linkage disequilibrium acting over entire chromosomes (see Supplementary Material section 1.4)." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-290610267" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;The 20% lower and upper tail probabilities for the observed windowed covariances from the Barghi et al. (2019) study, based on sign-permuting at the chromosome level. This permutation empirical null is robust to long-range linkage disequilibrium acting over entire chromosomes (see Supplementary Material section 1.4).&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure S12:" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2019/10/09/798595/F15.medium.gif" width="440" height="205"/><noscript><img class="highwire-fragment fragment-image" alt="Figure S12:" src="https://www.biorxiv.org/content/biorxiv/early/2019/10/09/798595/F15.medium.gif" width="440" height="205"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/10/09/798595/F15.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure S12:" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/10/09/798595/F15.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure S12:</span> <p id="p-71" class="first-child">The 20% lower and upper tail probabilities for the observed windowed covariances from the <a id="xref-ref-4-28" class="xref-bibr" href="#ref-4">Barghi et al. (2019)</a> study, based on sign-permuting at the chromosome level. This permutation empirical null is robust to long-range linkage disequilibrium acting over entire chromosomes (see Supplementary Material <a id="xref-sec-13-1" class="xref-sec" href="#sec-13">section 1.4</a>).</p><div class="sb-div caption-clear"></div></div></div></div></div><div class="section ack" id="ack-1"><h2 class="">4 Acknowledgments</h2><p id="p-28">We would like to thank the authors of the original studies we’ve analyzed, including Neda Barghi, Christian Schlötterer, John Kelly, Kimberly Hughes, Frank Chan, Campbell Rolian, Nick Barton, Alan Bergland, and Dmitri Petrov. We would also like to thank Doc Edge for helpful statistical advice, and Matt Osmond, Erin Calfee, and Sivan Yair for helpful discussions. This research was supported by an NSF Graduate Research Fellowship grant awarded to VB (1650042), and NIH (R01-GM108779) and NSF (1353380) awarded to GC.</p></div><div class="section ref-list" id="ref-list-1"><h2 class="">References</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference  in text" id="ref-1">↵</a><div class="cit ref-cit ref-journal" id="cit-798595v1.1"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Andersen</span>,  <span class="cit-name-given-names">Erik C</span></span> <span class="cit-etal">et al.</span> (<span class="cit-pub-date">2012</span>). “<span class="cit-article-title">Chromosome-scale selective sweeps shape Caenorhabditis elegans genomic diversity</span>”. en. In: <abbr class="cit-jnl-abbrev">Nat. Genet.s</abbr> <span class="cit-vol">44</span>.<span class="cit-issue">3</span>, pp. <span class="cit-fpage">285</span>–<span class="cit-lpage">290</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat.%2BGenet.s%26rft.volume%253D44%26rft.spage%253D285%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference  in text" id="ref-2">↵</a><div class="cit ref-cit ref-journal" id="cit-798595v1.2" data-doi="10.1101/gr.6691007"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Andolfatto</span>,  <span class="cit-name-given-names">Peter</span></span> (<span class="cit-pub-date">2007</span>). “<span class="cit-article-title">Hitchhiking effects of recurrent beneficial amino acid substitutions in the Drosophila melanogaster genome</span>”. en. In: <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">17</span>.<span class="cit-issue">12</span>, pp. <span class="cit-fpage">1755</span>–<span class="cit-lpage">1762</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.6691007%26rft_id%253Dinfo%253Apmid%252F17989248%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjEwOiIxNy8xMi8xNzU1IjtzOjQ6ImF0b20iO3M6Mzc6Ii9iaW9yeGl2L2Vhcmx5LzIwMTkvMTAvMDkvNzk4NTk1LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference  in text" id="ref-3">↵</a><div class="cit ref-cit ref-journal" id="cit-798595v1.3" data-doi="10.1093/molbev/msu048"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Baldwin-Brown</span>,  <span class="cit-name-given-names">J G</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">A D</span> <span class="cit-name-surname">Long</span></span>, and <span class="cit-auth"> <span class="cit-name-given-names">K R</span> <span class="cit-name-surname">Thornton</span></span> (<span class="cit-pub-date">2014</span>). “<span class="cit-article-title">The Power to Detect Quantitative Trait Loci Using Resequenced, Experimentally Evolved Populations of Diploid, Sexual Organisms</span>”. In: <abbr class="cit-jnl-abbrev">Mol. Biol. Evol.</abbr> <span class="cit-vol">31</span>.<span class="cit-issue">4</span>, pp. <span class="cit-fpage">1040</span>–<span class="cit-lpage">1055</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BBiol.%2BEvol.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fmolbev%252Fmsu048%26rft_id%253Dinfo%253Apmid%252F24441104%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/molbev/msu048&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24441104&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F10%2F09%2F798595.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000333283200021&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference  in text" id="ref-4">↵</a><div class="cit ref-cit ref-journal" id="cit-798595v1.4" data-doi="10.1371/journal.pbio.3000128"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Barghi</span>,  <span class="cit-name-given-names">Neda</span></span> <span class="cit-etal">et al.</span> (<span class="cit-pub-date">2019</span>). “<span class="cit-article-title">Genetic redundancy fuels polygenic adaptation in Drosophila</span>”. en. In: <abbr class="cit-jnl-abbrev">PLoS Biol.</abbr> <span class="cit-vol">17</span>.<span class="cit-issue">2</span>, <span class="cit-fpage">e3000128</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BBiol.%26rft.volume%253D17%26rft.spage%253De3000128%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pbio.3000128%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pbio.3000128&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference  in text" id="ref-5">↵</a><div class="cit ref-cit ref-journal" id="cit-798595v1.5"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Barton</span>,  <span class="cit-name-given-names">N H</span></span> (<span class="cit-pub-date">2000</span>). “<span class="cit-article-title">Genetic hitchhiking</span>”. In: <abbr class="cit-jnl-abbrev">Proceedings of the Royal Society of London B: Biological Sciences</abbr> <span class="cit-vol">355</span>.<span class="cit-issue">1403</span>, pp. <span class="cit-fpage">1553</span>–<span class="cit-lpage">1562</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProceedings%2Bof%2Bthe%2BRoyal%2BSociety%2Bof%2BLondon%2BB%253A%2BBiological%2BSciences%26rft.volume%253D355%26rft.spage%253D1553%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference  in text" id="ref-6">↵</a><div class="cit ref-cit ref-journal" id="cit-798595v1.6" data-doi="10.1126/science.aag0822"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Baym</span>,  <span class="cit-name-given-names">Michael</span></span> <span class="cit-etal">et al.</span> (<span class="cit-pub-date">2016</span>). “<span class="cit-article-title">Spatiotemporal microbial evolution on antibiotic landscapes</span>”. en. In: <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">353</span>.<span class="cit-issue">6304</span>, pp. <span class="cit-fpage">1147</span>–<span class="cit-lpage">1151</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DBaym%26rft.auinit1%253DM.%26rft.volume%253D353%26rft.issue%253D6304%26rft.spage%253D1147%26rft.epage%253D1151%26rft.atitle%253DSpatiotemporal%2Bmicrobial%2Bevolution%2Bon%2Bantibiotic%2Blandscapes%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.aag0822%26rft_id%253Dinfo%253Apmid%252F27609891%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEzOiIzNTMvNjMwNC8xMTQ3IjtzOjQ6ImF0b20iO3M6Mzc6Ii9iaW9yeGl2L2Vhcmx5LzIwMTkvMTAvMDkvNzk4NTk1LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference  in text" id="ref-7">↵</a><div class="cit ref-cit ref-journal" id="cit-798595v1.7" data-doi="10.1371/journal.pbio.0050310"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Begun</span>,  <span class="cit-name-given-names">David J</span></span> <span class="cit-etal">et al.</span> (<span class="cit-pub-date">2007</span>). “<span class="cit-article-title">Population genomics: whole-genome analysis of polymorphism and divergence in Drosophila simulans</span>”. en. In: <abbr class="cit-jnl-abbrev">PLoS Biol.</abbr> <span class="cit-vol">5</span>.<span class="cit-issue">11</span>, <span class="cit-fpage">e310</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2Bbiology%26rft.stitle%253DPLoS%2BBiol%26rft.aulast%253DBegun%26rft.auinit1%253DD.%2BJ.%26rft.volume%253D5%26rft.issue%253D11%26rft.spage%253De310%26rft.epage%253De310%26rft.atitle%253DPopulation%2Bgenomics%253A%2Bwhole-genome%2Banalysis%2Bof%2Bpolymorphism%2Band%2Bdivergence%2Bin%2BDrosophila%2Bsimulans.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pbio.0050310%26rft_id%253Dinfo%253Apmid%252F17988176%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pbio.0050310&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17988176&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F10%2F09%2F798595.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference  in text" id="ref-8">↵</a><div class="cit ref-cit ref-journal" id="cit-798595v1.8"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Beissinger</span>,  <span class="cit-name-given-names">Timothy M</span></span> <span class="cit-etal">et al.</span> (<span class="cit-pub-date">2016</span>). “<span class="cit-article-title">Recent demography drives changes in linked selection across the maize genome</span>”. en. In: <abbr class="cit-jnl-abbrev">Nat Plants</abbr> <span class="cit-vol">2</span>, p. <span class="cit-fpage">16084</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat%2BPlants%26rft.volume%253D2%26rft.spage%253D16084%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference  in text" id="ref-9">↵</a><div class="cit ref-cit ref-journal" id="cit-798595v1.9"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Benjamini</span>,  <span class="cit-name-given-names">Yoav</span></span> and <span class="cit-auth"> <span class="cit-name-given-names">Yosef</span> <span class="cit-name-surname">Hochberg</span></span> (<span class="cit-pub-date">1995</span>). “<span class="cit-article-title">Controlling the False Discovery Rate: A Practical and Powerful Approach to Multiple Testing</span>”. In: <abbr class="cit-jnl-abbrev">J. R. Stat. Soc. Series B Stat. Methodol.</abbr> <span class="cit-vol">57</span>.<span class="cit-issue">1</span>, pp. <span class="cit-fpage">289</span>–<span class="cit-lpage">300</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BR.%2BStat.%2BSoc.%2BSeries%2BB%2BStat.%2BMethodol.%26rft.volume%253D57%26rft.spage%253D289%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference  in text" id="ref-10">↵</a><div class="cit ref-cit ref-journal" id="cit-798595v1.10" data-doi="10.1038/346079a0"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Bennett</span>,  <span class="cit-name-given-names">A F</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">K M</span> <span class="cit-name-surname">Dao</span></span>, and <span class="cit-auth"> <span class="cit-name-given-names">R E</span> <span class="cit-name-surname">Lenski</span></span> (<span class="cit-pub-date">1990</span>). “<span class="cit-article-title">Rapid evolution in response to high-temperature selection</span>”. en. In: <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">346</span>.<span class="cit-issue">6279</span>, pp. <span class="cit-fpage">79</span>–<span class="cit-lpage">81</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DBennett%26rft.auinit1%253DA.%2BF.%26rft.volume%253D346%26rft.issue%253D6279%26rft.spage%253D79%26rft.epage%253D81%26rft.atitle%253DRapid%2Bevolution%2Bin%2Bresponse%2Bto%2Bhigh-temperature%2Bselection.%26rft_id%253Dinfo%253Adoi%252F10.1038%252F346079a0%26rft_id%253Dinfo%253Apmid%252F2195353%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/346079a0&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=2195353&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F10%2F09%2F798595.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1990DM39600067&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference  in text" id="ref-11">↵</a><div class="cit ref-cit ref-journal" id="cit-798595v1.11" data-doi="10.1371/journal.pgen.1004775"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Bergland</span>,  <span class="cit-name-given-names">Alan O</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">Emily L</span> <span class="cit-name-surname">Behrman</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">Katherine R</span> <span class="cit-name-surname">O’Brien</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">Paul S</span> <span class="cit-name-surname">Schmidt</span></span>, and <span class="cit-auth"> <span class="cit-name-given-names">Dmitri A</span> <span class="cit-name-surname">Petrov</span></span> (<span class="cit-pub-date">2014</span>). “<span class="cit-article-title">Genomic Evidence of Rapid and Stable Adaptive Oscillations over Seasonal Time Scales in Drosophila</span>”. In: <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr> <span class="cit-vol">10</span>.<span class="cit-issue">11</span>, <span class="cit-fpage">e1004775</span>–<span class="cit-lpage">19</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BGenet.%26rft.volume%253D10%26rft.spage%253De1004775%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.1004775%26rft_id%253Dinfo%253Apmid%252F25375361%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.1004775&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25375361&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F10%2F09%2F798595.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference  in text" id="ref-12">↵</a><div class="cit ref-cit ref-journal" id="cit-798595v1.12" data-doi="10.1101/gr.154831.113"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Bhatia</span>,  <span class="cit-name-given-names">Gaurav</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">Nick</span> <span class="cit-name-surname">Patterson</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">Sriram</span> <span class="cit-name-surname">Sankararaman</span></span>, and <span class="cit-auth"> <span class="cit-name-given-names">Alkes L</span> <span class="cit-name-surname">Price</span></span> (<span class="cit-pub-date">2013</span>). “<span class="cit-article-title">Estimating and interpreting FST: the impact of rare variants</span>”. en. In: <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">23</span>.<span class="cit-issue">9</span>, pp. <span class="cit-fpage">1514</span>–<span class="cit-lpage">1521</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.154831.113%26rft_id%253Dinfo%253Apmid%252F23861382%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjk6IjIzLzkvMTUxNCI7czo0OiJhdG9tIjtzOjM3OiIvYmlvcnhpdi9lYXJseS8yMDE5LzEwLzA5Lzc5ODU5NS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference  in text" id="ref-13">↵</a><div class="cit ref-cit ref-journal" id="cit-798595v1.13"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Buffalo</span>,  <span class="cit-name-given-names">Vince</span></span> and <span class="cit-auth"> <span class="cit-name-given-names">Graham</span> <span class="cit-name-surname">Coop</span></span> (<span class="cit-pub-date">2019</span>). “<span class="cit-article-title">The Linked Selection Signature of Rapid Adaptation in Temporal Genomic Data</span>”. en.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference  in text" id="ref-14">↵</a><div class="cit ref-cit ref-journal" id="cit-798595v1.14" data-doi="10.1038/nature09352"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Burke</span>,  <span class="cit-name-given-names">Molly K</span></span> <span class="cit-etal">et al.</span> (<span class="cit-pub-date">2010</span>). “<span class="cit-article-title">Genome-wide analysis of a long-term evolution experiment with Drosophila</span>”. In: <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">467</span>.<span class="cit-issue">7315</span>, pp. <span class="cit-fpage">587</span>–<span class="cit-lpage">590</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DBurke%26rft.auinit1%253DM.%2BK.%26rft.volume%253D467%26rft.issue%253D7315%26rft.spage%253D587%26rft.epage%253D590%26rft.atitle%253DGenome-wide%2Banalysis%2Bof%2Ba%2Blong-term%2Bevolution%2Bexperiment%2Bwith%2BDrosophila.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature09352%26rft_id%253Dinfo%253Apmid%252F20844486%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature09352&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20844486&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F10%2F09%2F798595.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000282273100038&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference  in text" id="ref-15">↵</a><div class="cit ref-cit ref-journal" id="cit-798595v1.15"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Castro</span>,  <span class="cit-name-given-names">João Pl</span></span> <span class="cit-etal">et al.</span> (<span class="cit-pub-date">2019</span>). “<span class="cit-article-title">An integrative genomic analysis of the Longshanks selection experiment for longer limbs in mice</span>”. en. In: <abbr class="cit-jnl-abbrev">Elife</abbr> <span class="cit-fpage">8</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference  in text" id="ref-16">↵</a><div class="cit ref-cit ref-journal" id="cit-798595v1.16" data-doi="10.1111/j.0014-3820.2002.tb00188.x"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Charlesworth</span>,  <span class="cit-name-given-names">Brian</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">M T</span> <span class="cit-name-surname">Morgan</span></span>, and <span class="cit-auth"> <span class="cit-name-given-names">D</span> <span class="cit-name-surname">Charlesworth</span></span> (<span class="cit-pub-date">1993</span>). “<span class="cit-article-title">The effect of deleterious mutations on neutral molecular variation</span>”. In: <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">134</span>.<span class="cit-issue">4</span>, pp. <span class="cit-fpage">1289</span>–<span class="cit-lpage">1303</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DCharlesworth%26rft.auinit1%253DB.%26rft.volume%253D134%26rft.issue%253D4%26rft.spage%253D1289%26rft.epage%253D1303%26rft.atitle%253DThe%2Beffect%2Bof%2Bdeleterious%2Bmutations%2Bon%2Bneutral%2Bmolecular%2Bvariation.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.0014-3820.2002.tb00188.x%26rft_id%253Dinfo%253Apmid%252F8375663%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjEzNC80LzEyODkiO3M6NDoiYXRvbSI7czozNzoiL2Jpb3J4aXYvZWFybHkvMjAxOS8xMC8wOS83OTg1OTUuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference  in text" id="ref-17">↵</a><div class="cit ref-cit ref-journal" id="cit-798595v1.17" data-doi="10.1371/journal.pgen.1004434"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Comeron</span>,  <span class="cit-name-given-names">Josep M</span></span> (<span class="cit-pub-date">2014</span>). “<span class="cit-article-title">Background selection as baseline for nucleotide variation across the Drosophila genome</span>”. en. In: <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr> <span class="cit-vol">10</span>.<span class="cit-issue">6</span>, <span class="cit-fpage">e1004434</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BGenet.%26rft.volume%253D10%26rft.spage%253De1004434%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.1004434%26rft_id%253Dinfo%253Apmid%252F24968283%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.1004434&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24968283&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F10%2F09%2F798595.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference  in text" id="ref-18">↵</a><div class="cit ref-cit ref-journal" id="cit-798595v1.18"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Coop</span>,  <span class="cit-name-given-names">Graham</span></span> (<span class="cit-pub-date">2016</span>). <span class="cit-article-title">Does linked selection explain the narrow range of genetic diversity across species?</span> <abbr class="cit-jnl-abbrev">Tech. rep. Cold Spring Harbor Labs Journals</abbr>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference  in text" id="ref-19">↵</a><div class="cit ref-cit ref-journal" id="cit-798595v1.19" data-doi="10.1371/journal.pbio.1002112"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Corbett-Detig</span>,  <span class="cit-name-given-names">Russell B</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">Daniel L</span> <span class="cit-name-surname">Hartl</span></span>, and <span class="cit-auth"> <span class="cit-name-given-names">Timothy B</span> <span class="cit-name-surname">Sackton</span></span> (<span class="cit-pub-date">2015</span>). “<span class="cit-article-title">Natural Selection Constrains Neutral Diversity across A Wide Range of Species</span>”. In: <abbr class="cit-jnl-abbrev">PLoS Biol.</abbr> <span class="cit-vol">13</span>.<span class="cit-issue">4</span>, <span class="cit-fpage">e1002112</span>–<span class="cit-lpage">25</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BBiol.%26rft.volume%253D13%26rft.spage%253De1002112%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pbio.1002112%26rft_id%253Dinfo%253Apmid%252F25859758%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pbio.1002112&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25859758&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F10%2F09%2F798595.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference  in text" id="ref-20">↵</a><div class="cit ref-cit ref-journal" id="cit-798595v1.20" data-doi="10.1101/gr.104331.109"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Cutter</span>,  <span class="cit-name-given-names">A D</span></span> and <span class="cit-auth"> <span class="cit-name-given-names">J Y</span> <span class="cit-name-surname">Choi</span></span> (<span class="cit-pub-date">2010</span>). “<span class="cit-article-title">Natural selection shapes nucleotide polymorphism across the genome of the nematode Caenorhabditis briggsae</span>”. In: <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">20</span>.<span class="cit-issue">8</span>, pp. <span class="cit-fpage">1103</span>–<span class="cit-lpage">1111</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.104331.109%26rft_id%253Dinfo%253Apmid%252F20508143%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjk6IjIwLzgvMTEwMyI7czo0OiJhdG9tIjtzOjM3OiIvYmlvcnhpdi9lYXJseS8yMDE5LzEwLzA5Lzc5ODU5NS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference  in text" id="ref-21">↵</a><div class="cit ref-cit ref-journal" id="cit-798595v1.21"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Cutter</span>,  <span class="cit-name-given-names">Asher D</span></span> and <span class="cit-auth"> <span class="cit-name-given-names">Bret A</span> <span class="cit-name-surname">Payseur</span></span> (<span class="cit-pub-date">2013</span>). “<span class="cit-article-title">Genomic signatures of selection at linked sites: unifying the disparity among species</span>”. In: <abbr class="cit-jnl-abbrev">Nature Publishing Group</abbr> <span class="cit-vol">14</span>.<span class="cit-issue">4</span>, pp. <span class="cit-fpage">262</span>–<span class="cit-lpage">274</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2BPublishing%2BGroup%26rft.volume%253D14%26rft.spage%253D262%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference  in text" id="ref-22">↵</a><div class="cit ref-cit ref-book" id="cit-798595v1.22"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Davison</span>,  <span class="cit-name-given-names">A C</span></span> and <span class="cit-auth"> <span class="cit-name-given-names">D V</span> <span class="cit-name-surname">Hinkley</span></span> (<span class="cit-pub-date">2013</span>). <span class="cit-chapter-title">Bootstrap Methods and Their Application</span>. <span class="cit-publ-loc">New York, NY, USA</span>: <span class="cit-publ-name">Cambridge University Press</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference  in text" id="ref-23">↵</a><div class="cit ref-cit ref-journal" id="cit-798595v1.23" data-doi="10.1371/journal.pgen.1006130"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Elyashiv</span>,  <span class="cit-name-given-names">Eyal</span></span> <span class="cit-etal">et al.</span> (<span class="cit-pub-date">2016</span>). “<span class="cit-article-title">A Genomic Map of the Effects of Linked Selection in Drosophila</span>”. In: <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr> <span class="cit-vol">12</span>.<span class="cit-issue">8</span>, <span class="cit-fpage">e1006130</span>–<span class="cit-lpage">24</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BGenet.%26rft.volume%253D12%26rft.spage%253De1006130%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.1006130%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.1006130&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference  in text" id="ref-24">↵</a><div class="cit ref-cit ref-journal" id="cit-798595v1.24" data-doi="10.1073/pnas.0608379104"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Franks</span>,  <span class="cit-name-given-names">Steven J</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">Sheina</span> <span class="cit-name-surname">Sim</span></span>, and <span class="cit-auth"> <span class="cit-name-given-names">Arthur E</span> <span class="cit-name-surname">Weis</span></span> (<span class="cit-pub-date">2007</span>). “<span class="cit-article-title">Rapid evolution of flowering time by an annual plant in response to a climate fluctuation</span>”. In: <abbr class="cit-jnl-abbrev">Proceedings of the National Academy of Sciences</abbr> <span class="cit-vol">104</span>.<span class="cit-issue">4</span>, pp. <span class="cit-fpage">1278</span>–<span class="cit-lpage">1282</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProceedings%2Bof%2Bthe%2BNational%2BAcademy%2Bof%2BSciences%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.0608379104%26rft_id%253Dinfo%253Apmid%252F17220273%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMDoiMTA0LzQvMTI3OCI7czo0OiJhdG9tIjtzOjM3OiIvYmlvcnhpdi9lYXJseS8yMDE5LzEwLzA5Lzc5ODU5NS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference  in text" id="ref-25">↵</a><div class="cit ref-cit ref-journal" id="cit-798595v1.25" data-doi="10.1038/nature24287"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Good</span>,  <span class="cit-name-given-names">Benjamin H</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">Michael J</span> <span class="cit-name-surname">McDonald</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">Jeffrey E</span> <span class="cit-name-surname">Barrick</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">Richard E</span> <span class="cit-name-surname">Lenski</span></span>, and <span class="cit-auth"> <span class="cit-name-given-names">Michael M</span> <span class="cit-name-surname">Desai</span></span> (<span class="cit-pub-date">2017</span>). “<span class="cit-article-title">The dynamics of molecular evolution over 60,000 generations</span>”. en. In: <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">551</span>.<span class="cit-issue">7678</span>, pp. <span class="cit-fpage">45</span>–<span class="cit-lpage">50</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.volume%253D551%26rft.spage%253D45%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature24287%26rft_id%253Dinfo%253Apmid%252F29045390%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature24287&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=29045390&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F10%2F09%2F798595.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference  in text" id="ref-26">↵</a><div class="cit ref-cit ref-journal" id="cit-798595v1.26" data-doi="10.1126/science.1128374"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Grant</span>,  <span class="cit-name-given-names">Peter R</span></span> and <span class="cit-auth"> <span class="cit-name-given-names">B Rosemary</span> <span class="cit-name-surname">Grant</span></span> (<span class="cit-pub-date">2006</span>). “<span class="cit-article-title">Evolution of Character Displacement in Darwin’s Finches</span>”. In: <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">313</span>.<span class="cit-issue">5784</span>, pp. <span class="cit-fpage">224</span>–<span class="cit-lpage">226</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DGrant%26rft.auinit1%253DP.%2BR.%26rft.volume%253D313%26rft.issue%253D5784%26rft.spage%253D224%26rft.epage%253D226%26rft.atitle%253DEvolution%2Bof%2Bcharacter%2Bdisplacement%2Bin%2BDarwin%2527s%2Bfinches.%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1128374%26rft_id%253Dinfo%253Apmid%252F16840700%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEyOiIzMTMvNTc4NC8yMjQiO3M6NDoiYXRvbSI7czozNzoiL2Jpb3J4aXYvZWFybHkvMjAxOS8xMC8wOS83OTg1OTUuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-27-1" title="View reference  in text" id="ref-27">↵</a><div class="cit ref-cit ref-journal" id="cit-798595v1.27" data-doi="10.1073/pnas.1018080108"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Grant</span>,  <span class="cit-name-given-names">Peter R</span></span> and <span class="cit-auth"> <span class="cit-name-given-names">B Rosemary</span> <span class="cit-name-surname">Grant</span></span> (<span class="cit-pub-date">2011</span>). “<span class="cit-article-title">Causes of lifetime fitness of Darwin’s finches in a fluctuating environment</span>”. en. In: <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. U. S. A.</abbr> <span class="cit-vol">108</span>.<span class="cit-issue">2</span>, pp. <span class="cit-fpage">674</span>–<span class="cit-lpage">679</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BU.%2BS.%2BA.%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.1018080108%26rft_id%253Dinfo%253Apmid%252F21199941%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czo5OiIxMDgvMi82NzQiO3M6NDoiYXRvbSI7czozNzoiL2Jpb3J4aXYvZWFybHkvMjAxOS8xMC8wOS83OTg1OTUuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-28-1" title="View reference  in text" id="ref-28">↵</a><div class="cit ref-cit ref-journal" id="cit-798595v1.28" data-doi="10.1111/j.1461-0248.2005.00812.x"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Hairston</span>   <span class="cit-name-suffix">Jr</span>,  <span class="cit-name-given-names">Nelson G</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">Stephen P</span> <span class="cit-name-surname">Ellner</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">Monica A</span> <span class="cit-name-surname">Geber</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">Takehito</span> <span class="cit-name-surname">Yoshida</span></span>, and <span class="cit-auth"> <span class="cit-name-given-names">Jennifer A</span> <span class="cit-name-surname">Fox</span></span> (<span class="cit-pub-date">2005</span>). “<span class="cit-article-title">Rapid evolution and the convergence of ecological and evolutionary time</span>”. In: <abbr class="cit-jnl-abbrev">Ecol. Lett.</abbr> <span class="cit-vol">8</span>.<span class="cit-issue">10</span>, pp. <span class="cit-fpage">1114</span>–<span class="cit-lpage">1127</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEcol.%2BLett.%26rft.volume%253D8%26rft.spage%253D1114%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1461-0248.2005.00812.x%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1461-0248.2005.00812.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000231737200011&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-29-1" title="View reference  in text" id="ref-29">↵</a><div class="cit ref-cit ref-journal" id="cit-798595v1.29" data-doi="10.2307/2640428"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Hendry</span>,  <span class="cit-name-given-names">A P</span></span> and <span class="cit-auth"> <span class="cit-name-given-names">M T</span> <span class="cit-name-surname">Kinnison</span></span> (<span class="cit-pub-date">1999</span>). “<span class="cit-article-title">Perspective: the pace of modern life: measuring rates of contemporary microevolution</span>”. In: <abbr class="cit-jnl-abbrev">Evolution</abbr> <span class="cit-vol">53</span>.<span class="cit-issue">6</span>, p. <span class="cit-fpage">1637</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEvolution%26rft.volume%253D53%26rft.spage%253D1637%26rft_id%253Dinfo%253Adoi%252F10.2307%252F2640428%26rft_id%253Dinfo%253Apmid%252FWOS%253A00008470%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2307/2640428&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=WOS:00008470&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F10%2F09%2F798595.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000084703100001&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-30-1" title="View reference  in text" id="ref-30">↵</a><div class="cit ref-cit ref-journal" id="cit-798595v1.30"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Hernandez</span>,  <span class="cit-name-given-names">R D</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">J L</span> <span class="cit-name-surname">Kelley</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">E</span> <span class="cit-name-surname">Elyashiv</span></span>, and <span class="cit-auth"> <span class="cit-name-given-names">S C</span> <span class="cit-name-surname">Melton</span></span> (<span class="cit-pub-date">2011</span>). “<span class="cit-article-title">Classic selective sweeps were rare in recent human evolution</span>”. In: <abbr class="cit-jnl-abbrev">Science</abbr>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-31-1" title="View reference  in text" id="ref-31">↵</a><div class="cit ref-cit ref-journal" id="cit-798595v1.31"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Howie</span>,  <span class="cit-name-given-names">James Malcolm</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">Rupert</span> <span class="cit-name-surname">Mazzucco</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">Thomas</span> <span class="cit-name-surname">Taus</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">Viola</span> <span class="cit-name-surname">Nolte</span></span>, and <span class="cit-auth"> <span class="cit-name-given-names">Christian</span> <span class="cit-name-surname">Schlotterer</span></span> (<span class="cit-pub-date">2018</span>). “<span class="cit-article-title">DNA motifs are not general predictors of recombination in two Drosophila sister species</span>”. en.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-32-1" title="View reference  in text" id="ref-32">↵</a><div class="cit ref-cit ref-journal" id="cit-798595v1.32"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Jónás</span>,  <span class="cit-name-given-names">Á</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">T</span> <span class="cit-name-surname">Taus</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">C</span> <span class="cit-name-surname">Kosiol</span></span>, and <span class="cit-auth"> <span class="cit-name-given-names">C</span> <span class="cit-name-surname">Schlötterer</span></span> (<span class="cit-pub-date">2016</span>). “<span class="cit-article-title">Estimating the Effective Population Size from Temporal Allele Frequency Changes in Experimental Evolution</span>”. In: <abbr class="cit-jnl-abbrev">Genetics</abbr>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-33-1" title="View reference  in text" id="ref-33">↵</a><div class="cit ref-cit ref-journal" id="cit-798595v1.33" data-doi="10.1534/genetics.118.301824"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Kelly</span>,  <span class="cit-name-given-names">John K</span></span> and <span class="cit-auth"> <span class="cit-name-given-names">Kimberly A</span> <span class="cit-name-surname">Hughes</span></span> (<span class="cit-pub-date">2019</span>). “<span class="cit-article-title">Pervasive Linked Selection and Intermediate-Frequency Alleles Are Implicated in an Evolve-and-Resequencing Experiment of Drosophila simulans</span>”. en. In: <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">211</span>.<span class="cit-issue">3</span>, pp. <span class="cit-fpage">943</span>–<span class="cit-lpage">961</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.118.301824%26rft_id%253Dinfo%253Apmid%252F30593495%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMjExLzMvOTQzIjtzOjQ6ImF0b20iO3M6Mzc6Ii9iaW9yeGl2L2Vhcmx5LzIwMTkvMTAvMDkvNzk4NTk1LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-34-1" title="View reference  in text" id="ref-34">↵</a><div class="cit ref-cit ref-journal" id="cit-798595v1.34" data-doi="10.1534/genetics.110.123059"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Kolaczkowski</span>,  <span class="cit-name-given-names">Bryan</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">Andrew D</span> <span class="cit-name-surname">Kern</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">Alisha K</span> <span class="cit-name-surname">Holloway</span></span>, and <span class="cit-auth"> <span class="cit-name-given-names">David J</span> <span class="cit-name-surname">Begun</span></span> (<span class="cit-pub-date">2011</span>). “<span class="cit-article-title">Genomic differentiation between temperate and tropical Australian populations of Drosophila melanogaster</span>”. en. In: <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">187</span>.<span class="cit-issue">1</span>, pp. <span class="cit-fpage">245</span>–<span class="cit-lpage">260</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.110.123059%26rft_id%253Dinfo%253Apmid%252F21059887%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTg3LzEvMjQ1IjtzOjQ6ImF0b20iO3M6Mzc6Ii9iaW9yeGl2L2Vhcmx5LzIwMTkvMTAvMDkvNzk4NTk1LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-35-1" title="View reference  in text" id="ref-35">↵</a><div class="cit ref-cit ref-journal" id="cit-798595v1.35"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Machado</span>,  <span class="cit-name-given-names">Heather E</span></span> <span class="cit-etal">et al.</span> (<span class="cit-pub-date">2018</span>). “<span class="cit-article-title">Broad geographic sampling reveals predictable and pervasive seasonal adaptation in Drosophila</span>”. en.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-36-1" title="View reference  in text" id="ref-36">↵</a><div class="cit ref-cit ref-journal" id="cit-798595v1.36" data-doi="10.1534/genetics.107.080226"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Macpherson</span>,  <span class="cit-name-given-names">J Michael</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">Guy</span> <span class="cit-name-surname">Sella</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">Jerel C</span> <span class="cit-name-surname">Davis</span></span>, and <span class="cit-auth"> <span class="cit-name-given-names">Dmitri A</span> <span class="cit-name-surname">Petrov</span></span> (<span class="cit-pub-date">2007</span>). “<span class="cit-article-title">Genomewide spatial correspondence between nonsynonymous divergence and neutral polymorphism reveals extensive adaptation in Drosophila</span>”. en. In: <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">177</span>.<span class="cit-issue">4</span>, pp. <span class="cit-fpage">2083</span>–<span class="cit-lpage">2099</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DMacpherson%26rft.auinit1%253DJ.%2BM.%26rft.volume%253D177%26rft.issue%253D4%26rft.spage%253D2083%26rft.epage%253D2099%26rft.atitle%253DGenomewide%2BSpatial%2BCorrespondence%2BBetween%2BNonsynonymous%2BDivergence%2Band%2BNeutral%2BPolymorphism%2BReveals%2BExtensive%2BAdaptation%2Bin%2BDrosophila%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.107.080226%26rft_id%253Dinfo%253Apmid%252F18073425%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE3Ny80LzIwODMiO3M6NDoiYXRvbSI7czozNzoiL2Jpb3J4aXYvZWFybHkvMjAxOS8xMC8wOS83OTg1OTUuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-37-1" title="View reference  in text" id="ref-37">↵</a><div class="cit ref-cit ref-journal" id="cit-798595v1.37" data-doi="10.1186/s12862-014-0258-0"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Marchini</span>,  <span class="cit-name-given-names">Marta</span></span> <span class="cit-etal">et al.</span> (<span class="cit-pub-date">2014</span>). “<span class="cit-article-title">Impacts of genetic correlation on the independent evolution of body mass and skeletal size in mammals</span>”. en. In: <abbr class="cit-jnl-abbrev">BMC Evol. Biol.</abbr> <span class="cit-vol">14</span>, p. <span class="cit-fpage">258</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2BEvol.%2BBiol.%26rft.volume%253D14%26rft.spage%253D258%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fs12862-014-0258-0%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/s12862-014-0258-0&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-38-1" title="View reference  in text" id="ref-38">↵</a><div class="cit ref-cit ref-journal" id="cit-798595v1.38" data-doi="10.1017/S0016672300014634"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Maynard</span>  <span class="cit-name-given-names">Smith</span></span>, <span class="cit-auth"><span class="cit-name-surname">John</span></span> and <span class="cit-auth"> <span class="cit-name-given-names">John</span> <span class="cit-name-surname">Haigh</span></span> (<span class="cit-pub-date">1974</span>). “<span class="cit-article-title">The hitch-hiking effect of a favourable gene</span>”. In: <abbr class="cit-jnl-abbrev">Genet. Res.</abbr> <span class="cit-vol">23</span>.<span class="cit-issue">01</span>, pp. <span class="cit-fpage">23</span>–<span class="cit-lpage">35</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetical%2Bresearch%26rft.stitle%253DGenet%2BRes%26rft.aulast%253DSmith%26rft.auinit1%253DJ.%2BM.%26rft.volume%253D23%26rft.issue%253D1%26rft.spage%253D23%26rft.epage%253D35%26rft.atitle%253DThe%2Bhitch-hiking%2Beffect%2Bof%2Ba%2Bfavourable%2Bgene.%26rft_id%253Dinfo%253Adoi%252F10.1017%252FS0016672300014634%26rft_id%253Dinfo%253Apmid%252F4407212%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1017/S0016672300014634&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=4407212&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F10%2F09%2F798595.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1974T216900003&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-39-1" title="View reference  in text" id="ref-39">↵</a><div class="cit ref-cit ref-journal" id="cit-798595v1.39" data-doi="10.1371/journal.pgen.1000471"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">McVicker</span>,  <span class="cit-name-given-names">Graham</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">David</span> <span class="cit-name-surname">Gordon</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">Colleen</span> <span class="cit-name-surname">Davis</span></span>, and <span class="cit-auth"> <span class="cit-name-given-names">Phil</span> <span class="cit-name-surname">Green</span></span> (<span class="cit-pub-date">2009</span>). “<span class="cit-article-title">Widespread Genomic Signatures of Natural Selection in Hominid Evolution</span>”. In: <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr> <span class="cit-vol">5</span>.<span class="cit-issue">5</span>, <span class="cit-fpage">e1000471</span>–<span class="cit-lpage">16</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BGenet%26rft.aulast%253DMcVicker%26rft.auinit1%253DG.%26rft.volume%253D5%26rft.issue%253D5%26rft.spage%253De1000471%26rft.epage%253De1000471%26rft.atitle%253DWidespread%2Bgenomic%2Bsignatures%2Bof%2Bnatural%2Bselection%2Bin%2Bhominid%2Bevolution.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.1000471%26rft_id%253Dinfo%253Apmid%252F19424416%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.1000471&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19424416&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F10%2F09%2F798595.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-40-1" title="View reference  in text" id="ref-40">↵</a><div class="cit ref-cit ref-journal" id="cit-798595v1.40"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Nei</span>,  <span class="cit-name-given-names">Masatoshi</span></span> and <span class="cit-auth"> <span class="cit-name-given-names">Fumio</span> <span class="cit-name-surname">Tajima</span></span> (<span class="cit-pub-date">1981</span>). “<span class="cit-article-title">Genetic Drift And Estimation Of Effective Population Size</span>”. In: <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">98</span>.<span class="cit-issue">3</span>, pp. <span class="cit-fpage">625</span>–<span class="cit-lpage">640</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DNei%26rft.auinit1%253DM.%26rft.volume%253D98%26rft.issue%253D3%26rft.spage%253D625%26rft.epage%253D640%26rft.atitle%253DGENETIC%2BDRIFT%2BAND%2BESTIMATION%2BOF%2BEFFECTIVE%2BPOPULATION%2BSIZE%26rft_id%253Dinfo%253Apmid%252F17249104%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6ODoiOTgvMy82MjUiO3M6NDoiYXRvbSI7czozNzoiL2Jpb3J4aXYvZWFybHkvMjAxOS8xMC8wOS83OTg1OTUuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-41-1" title="View reference  in text" id="ref-41">↵</a><div class="cit ref-cit ref-journal" id="cit-798595v1.41" data-doi="10.1017/S0016672300033619"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Nordborg</span>,  <span class="cit-name-given-names">Magnus</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">Brian</span> <span class="cit-name-surname">Charlesworth</span></span>, and <span class="cit-auth"> <span class="cit-name-given-names">Deborah</span> <span class="cit-name-surname">Charlesworth</span></span> (<span class="cit-pub-date">1996</span>). “<span class="cit-article-title">The effect of recombination on background selection*</span>”. In: <abbr class="cit-jnl-abbrev">Genet. Res.</abbr> <span class="cit-vol">67</span>.<span class="cit-issue">02</span>, pp. <span class="cit-fpage">159</span>–<span class="cit-lpage">174</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetical%2Bresearch%26rft.stitle%253DGenet%2BRes%26rft.aulast%253DNordborg%26rft.auinit1%253DM.%26rft.volume%253D67%26rft.issue%253D2%26rft.spage%253D159%26rft.epage%253D174%26rft.atitle%253DThe%2Beffect%2Bof%2Brecombination%2Bon%2Bbackground%2Bselection.%26rft_id%253Dinfo%253Adoi%252F10.1017%252FS0016672300033619%26rft_id%253Dinfo%253Apmid%252F8801188%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1017/S0016672300033619&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=8801188&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F10%2F09%2F798595.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1996UK49900007&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-42-1" title="View reference  in text" id="ref-42">↵</a><div class="cit ref-cit ref-journal" id="cit-798595v1.42" data-doi="10.1371/journal.pbio.0030196"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Nordborg</span>,  <span class="cit-name-given-names">Magnus</span></span> <span class="cit-etal">et al.</span> (<span class="cit-pub-date">2005</span>). “<span class="cit-article-title">The Pattern of Polymorphism in Arabidopsis thaliana</span>”. In: <abbr class="cit-jnl-abbrev">PLoS Biol.</abbr> <span class="cit-vol">3</span>.<span class="cit-issue">7</span>, <span class="cit-fpage">e196</span>–<span class="cit-lpage">11</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2Bbiology%26rft.stitle%253DPLoS%2BBiol%26rft.aulast%253DNordborg%26rft.auinit1%253DM.%26rft.volume%253D3%26rft.issue%253D7%26rft.spage%253De196%26rft.epage%253De196%26rft.atitle%253DThe%2Bpattern%2Bof%2Bpolymorphism%2Bin%2BArabidopsis%2Bthaliana.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pbio.0030196%26rft_id%253Dinfo%253Apmid%252F15907155%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pbio.0030196&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15907155&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F10%2F09%2F798595.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-43-1" title="View reference  in text" id="ref-43">↵</a><div class="cit ref-cit ref-journal" id="cit-798595v1.43" data-doi="10.1016/j.gde.2013.10.002"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Nuzhdin</span>,  <span class="cit-name-given-names">Sergey V</span></span> and <span class="cit-auth"> <span class="cit-name-given-names">Thomas L</span> <span class="cit-name-surname">Turner</span></span> (<span class="cit-pub-date">2013</span>). “<span class="cit-article-title">Promises and limitations of hitchhiking mapping</span>”. en. In: <abbr class="cit-jnl-abbrev">Curr. Opin. Genet. Dev.</abbr> <span class="cit-vol">23</span>.<span class="cit-issue">6</span>, pp. <span class="cit-fpage">694</span>–<span class="cit-lpage">699</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCurr.%2BOpin.%2BGenet.%2BDev.%26rft.volume%253D23%26rft.spage%253D694%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.gde.2013.10.002%26rft_id%253Dinfo%253Apmid%252F24239053%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.gde.2013.10.002&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24239053&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F10%2F09%2F798595.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-44-1" title="View reference  in text" id="ref-44">↵</a><div class="cit ref-cit ref-journal" id="cit-798595v1.44" data-doi="10.1126/science.275.5308.1934"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Reznick</span>,  <span class="cit-name-given-names">David N</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">Frank H</span> <span class="cit-name-surname">Shaw</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">F Helen</span> <span class="cit-name-surname">Rodd</span></span>, and <span class="cit-auth"> <span class="cit-name-given-names">Ruth G</span> <span class="cit-name-surname">Shaw</span></span> (<span class="cit-pub-date">1997</span>). “<span class="cit-article-title">Evaluation of the Rate of Evolution in Natural Populations of Guppies (Poecilia reticulata)</span>”. In: <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">275</span>.<span class="cit-issue">5308</span>, pp. <span class="cit-fpage">1934</span>–<span class="cit-lpage">1937</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DReznick%26rft.auinit1%253DD.%2BN.%26rft.volume%253D275%26rft.issue%253D5308%26rft.spage%253D1934%26rft.epage%253D1937%26rft.atitle%253DEvaluation%2Bof%2Bthe%2BRate%2Bof%2BEvolution%2Bin%2BNatural%2BPopulations%2Bof%2BGuppies%2B%2528Poecilia%2Breticulata%2529%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.275.5308.1934%26rft_id%253Dinfo%253Apmid%252F9072971%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEzOiIyNzUvNTMwOC8xOTM0IjtzOjQ6ImF0b20iO3M6Mzc6Ii9iaW9yeGl2L2Vhcmx5LzIwMTkvMTAvMDkvNzk4NTk1LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-45-1" title="View reference  in text" id="ref-45">↵</a><div class="cit ref-cit ref-journal" id="cit-798595v1.45" data-doi="10.1017/S0016672300000690"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Robertson</span>,  <span class="cit-name-given-names">Alan</span></span> (<span class="cit-pub-date">1961</span>). “<span class="cit-article-title">Inbreeding in artificial selection programmes</span>”. In: <abbr class="cit-jnl-abbrev">Genet. Res.</abbr> <span class="cit-vol">2</span>.<span class="cit-issue">02</span>, pp. <span class="cit-fpage">189</span>–<span class="cit-lpage">194</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenet.%2BRes.%26rft.volume%253D2%26rft.spage%253D189%26rft_id%253Dinfo%253Adoi%252F10.1017%252FS0016672300000690%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1017/S0016672300000690&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=A19611852B00011&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-46-1" title="View reference  in text" id="ref-46">↵</a><div class="cit ref-cit ref-journal" id="cit-798595v1.46"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Santiago</span>,  <span class="cit-name-given-names">Enrique</span></span> and <span class="cit-auth"> <span class="cit-name-given-names">Armando</span> <span class="cit-name-surname">Caballero</span></span> (<span class="cit-pub-date">1995</span>). “<span class="cit-article-title">Effective size of populations under selection</span>”. In: <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">139</span>.<span class="cit-issue">2</span>, pp. <span class="cit-fpage">1013</span>–<span class="cit-lpage">1030</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DSantiago%26rft.auinit1%253DE.%26rft.volume%253D139%26rft.issue%253D2%26rft.spage%253D1013%26rft.epage%253D1030%26rft.atitle%253DEffective%2Bsize%2Bof%2Bpopulations%2Bunder%2Bselection.%26rft_id%253Dinfo%253Apmid%252F7713405%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjEzOS8yLzEwMTMiO3M6NDoiYXRvbSI7czozNzoiL2Jpb3J4aXYvZWFybHkvMjAxOS8xMC8wOS83OTg1OTUuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-47-1" title="View reference  in text" id="ref-47">↵</a><div class="cit ref-cit ref-journal" id="cit-798595v1.47"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Santiago</span>,  <span class="cit-name-given-names">Enrique</span></span> and <span class="cit-auth"> <span class="cit-name-given-names">Armando</span> <span class="cit-name-surname">Caballero</span></span> (<span class="cit-pub-date">1998</span>). “<span class="cit-article-title">Effective Size and Polymorphism of Linked Neutral Loci in Populations Under Directional Selection</span>”. In: <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">149</span>.<span class="cit-issue">4</span>, pp. <span class="cit-fpage">2105</span>–<span class="cit-lpage">2117</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DSantiago%26rft.auinit1%253DE.%26rft.volume%253D149%26rft.issue%253D4%26rft.spage%253D2105%26rft.epage%253D2117%26rft.atitle%253DEffective%2BSize%2Band%2BPolymorphism%2Bof%2BLinked%2BNeutral%2BLoci%2Bin%2BPopulations%2BUnder%2BDirectional%2BSelection%26rft_id%253Dinfo%253Apmid%252F9691062%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE0OS80LzIxMDUiO3M6NDoiYXRvbSI7czozNzoiL2Jpb3J4aXYvZWFybHkvMjAxOS8xMC8wOS83OTg1OTUuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-48-1" title="View reference  in text" id="ref-48">↵</a><div class="cit ref-cit ref-journal" id="cit-798595v1.48" data-doi="10.1371/journal.pgen.1001302"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Sattath</span>,  <span class="cit-name-given-names">Shmuel</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">Eyal</span> <span class="cit-name-surname">Elyashiv</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">Oren</span> <span class="cit-name-surname">Kolodny</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">Yosef</span> <span class="cit-name-surname">Rinott</span></span>, and <span class="cit-auth"> <span class="cit-name-given-names">Guy</span> <span class="cit-name-surname">Sella</span></span> (<span class="cit-pub-date">2011</span>). “<span class="cit-article-title">Pervasive adaptive protein evolution apparent in diversity patterns around amino acid substitutions in Drosophila simulans</span>”. en. In: <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr> <span class="cit-vol">7</span>.<span class="cit-issue">2</span>, <span class="cit-fpage">e1001302</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BGenet%26rft.aulast%253DSattath%26rft.auinit1%253DS.%26rft.volume%253D7%26rft.issue%253D2%26rft.spage%253De1001302%26rft.epage%253De1001302%26rft.atitle%253DPervasive%2Badaptive%2Bprotein%2Bevolution%2Bapparent%2Bin%2Bdiversity%2Bpatterns%2Baround%2Bamino%2Bacid%2Bsubstitutions%2Bin%2BDrosophila%2Bsimulans.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.1001302%26rft_id%253Dinfo%253Apmid%252F21347283%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.1001302&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21347283&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F10%2F09%2F798595.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-49-1" title="View reference  in text" id="ref-49">↵</a><div class="cit ref-cit ref-journal" id="cit-798595v1.49" data-doi="10.1371/journal.pgen.1000495"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Sella</span>,  <span class="cit-name-given-names">Guy</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">Dmitri A</span> <span class="cit-name-surname">Petrov</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">Molly</span> <span class="cit-name-surname">Przeworski</span></span>, and <span class="cit-auth"> <span class="cit-name-given-names">Peter</span> <span class="cit-name-surname">Andolfatto</span></span> (<span class="cit-pub-date">2009</span>). “<span class="cit-article-title">Pervasive natural selection in the Drosophila genome?</span>” en. In: <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr> <span class="cit-vol">5</span>.<span class="cit-issue">6</span>, <span class="cit-fpage">e1000495</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BGenet%26rft.aulast%253DSella%26rft.auinit1%253DG.%26rft.volume%253D5%26rft.issue%253D6%26rft.spage%253De1000495%26rft.epage%253De1000495%26rft.atitle%253DPervasive%2Bnatural%2Bselection%2Bin%2Bthe%2BDrosophila%2Bgenome%253F%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.1000495%26rft_id%253Dinfo%253Apmid%252F19503600%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.1000495&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19503600&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F10%2F09%2F798595.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-50-1" title="View reference  in text" id="ref-50">↵</a><div class="cit ref-cit ref-journal" id="cit-798595v1.50" data-doi="10.1093/gbe/evx262"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Signor</span>,  <span class="cit-name-given-names">Sarah A</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">Felicia N</span> <span class="cit-name-surname">New</span></span>, and <span class="cit-auth"> <span class="cit-name-given-names">Sergey</span> <span class="cit-name-surname">Nuzhdin</span></span> (<span class="cit-pub-date">2018</span>). “<span class="cit-article-title">A Large Panel of Drosophila simulans Reveals an Abundance of Common Variants</span>”. en. In: <abbr class="cit-jnl-abbrev">Genome Biol. Evol.</abbr> <span class="cit-vol">10</span>.<span class="cit-issue">1</span>, pp. <span class="cit-fpage">189</span>–<span class="cit-lpage">206</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BBiol.%2BEvol.%26rft.volume%253D10%26rft.spage%253D189%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fgbe%252Fevx262%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/gbe/evx262&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-51-1" title="View reference  in text" id="ref-51">↵</a><div class="cit ref-cit ref-journal" id="cit-798595v1.51" data-doi="10.1126/science.aaw7271"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Therkildsen</span>,  <span class="cit-name-given-names">Nina O</span></span> <span class="cit-etal">et al.</span> (<span class="cit-pub-date">2019</span>). “<span class="cit-article-title">Contrasting genomic shifts underlie parallel phenotypic evolution in response to fishing</span>”. en. In: <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">365</span>.<span class="cit-issue">6452</span>, pp. <span class="cit-fpage">487</span>–<span class="cit-lpage">490</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DTherkildsen%26rft.auinit1%253DN.%2BO.%26rft.volume%253D365%26rft.issue%253D6452%26rft.spage%253D487%26rft.epage%253D490%26rft.atitle%253DContrasting%2Bgenomic%2Bshifts%2Bunderlie%2Bparallel%2Bphenotypic%2Bevolution%2Bin%2Bresponse%2Bto%2Bfishing%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.aaw7271%26rft_id%253Dinfo%253Apmid%252F31371613%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEyOiIzNjUvNjQ1Mi80ODciO3M6NDoiYXRvbSI7czozNzoiL2Jpb3J4aXYvZWFybHkvMjAxOS8xMC8wOS83OTg1OTUuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-52-1" title="View reference  in text" id="ref-52">↵</a><div class="cit ref-cit ref-journal" id="cit-798595v1.52" data-doi="10.1534/genetics.112.139337"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Turner</span>,  <span class="cit-name-given-names">T L</span></span> and <span class="cit-auth"> <span class="cit-name-given-names">P M</span> <span class="cit-name-surname">Miller</span></span> (<span class="cit-pub-date">2012</span>). “<span class="cit-article-title">Investigating Natural Variation in Drosophila Courtship Song by the Evolve and Resequence Approach</span>”. In: <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">191</span>.<span class="cit-issue">2</span>, pp. <span class="cit-fpage">633</span>–<span class="cit-lpage">642</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.112.139337%26rft_id%253Dinfo%253Apmid%252F22466043%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTkxLzIvNjMzIjtzOjQ6ImF0b20iO3M6Mzc6Ii9iaW9yeGl2L2Vhcmx5LzIwMTkvMTAvMDkvNzk4NTk1LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-53-1" title="View reference  in text" id="ref-53">↵</a><div class="cit ref-cit ref-journal" id="cit-798595v1.53" data-doi="10.1371/journal.pgen.1001336"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Turner</span>,  <span class="cit-name-given-names">Thomas L</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">Andrew D</span> <span class="cit-name-surname">Stewart</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">Andrew T</span> <span class="cit-name-surname">Fields</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">William R</span> <span class="cit-name-surname">Rice</span></span>, and <span class="cit-auth"> <span class="cit-name-given-names">Aaron M</span> <span class="cit-name-surname">Tarone</span></span> (<span class="cit-pub-date">2011</span>). “<span class="cit-article-title">Population-Based Resequencing of Experimentally Evolved Populations Reveals the Genetic Basis of Body Size Variation in Drosophila melanogaster</span>”. In: <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr> <span class="cit-vol">7</span>.<span class="cit-issue">3</span>, <span class="cit-fpage">e1001336</span>–<span class="cit-lpage">10</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BGenet%26rft.aulast%253DTurner%26rft.auinit1%253DT.%2BL.%26rft.volume%253D7%26rft.issue%253D3%26rft.spage%253De1001336%26rft.epage%253De1001336%26rft.atitle%253DPopulation-based%2Bresequencing%2Bof%2Bexperimentally%2Bevolved%2Bpopulations%2Breveals%2Bthe%2Bgenetic%2Bbasis%2Bof%2Bbody%2Bsize%2Bvariation%2Bin%2BDrosophila%2Bmelanogaster.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.1001336%26rft_id%253Dinfo%253Apmid%252F21437274%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.1001336&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21437274&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F10%2F09%2F798595.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-54-1" title="View reference  in text" id="ref-54">↵</a><div class="cit ref-cit ref-journal" id="cit-798595v1.54"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Waples</span>,  <span class="cit-name-given-names">R S</span></span> (<span class="cit-pub-date">1989</span>). “<span class="cit-article-title">A generalized approach for estimating effective population size from temporal changes in allele frequency</span>”. In: <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">121</span>.<span class="cit-issue">2</span>, pp. <span class="cit-fpage">379</span>–<span class="cit-lpage">391</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DWaples%26rft.auinit1%253DR.%2BS.%26rft.volume%253D121%26rft.issue%253D2%26rft.spage%253D379%26rft.epage%253D391%26rft.atitle%253DA%2Bgeneralized%2Bapproach%2Bfor%2Bestimating%2Beffective%2Bpopulation%2Bsize%2Bfrom%2Btemporal%2Bchanges%2Bin%2Ballele%2Bfrequency.%26rft_id%253Dinfo%253Apmid%252F2731727%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTIxLzIvMzc5IjtzOjQ6ImF0b20iO3M6Mzc6Ii9iaW9yeGl2L2Vhcmx5LzIwMTkvMTAvMDkvNzk4NTk1LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-55-1" title="View reference  in text" id="ref-55">↵</a><div class="cit ref-cit ref-book" id="cit-798595v1.55"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Wasserman</span>,  <span class="cit-name-given-names">Larry</span></span> (<span class="cit-pub-date">2006</span>). <span class="cit-chapter-title">All of Nonparametric Statistics</span>. en. <span class="cit-publ-name">Springer Science &amp; Business Media</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-56-1" title="View reference  in text" id="ref-56">↵</a><div class="cit ref-cit ref-journal" id="cit-798595v1.56" data-doi="10.1371/journal.pgen.1004622"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Williamson</span>,  <span class="cit-name-given-names">Robert J</span></span> <span class="cit-etal">et al.</span> (<span class="cit-pub-date">2014</span>). “<span class="cit-article-title">Evidence for widespread positive and negative selection in coding and conserved noncoding regions of Capsella grandiflora</span>”. en. In: <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr> <span class="cit-vol">10</span>.<span class="cit-issue">9</span>, <span class="cit-fpage">e1004622</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BGenet.%26rft.volume%253D10%26rft.spage%253De1004622%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.1004622%26rft_id%253Dinfo%253Apmid%252F25255320%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.1004622&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25255320&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F10%2F09%2F798595.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-57-1" title="View reference  in text" id="ref-57">↵</a><div class="cit ref-cit ref-journal" id="cit-798595v1.57"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Woolliams</span>,  <span class="cit-name-given-names">J A</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">N R</span> <span class="cit-name-surname">Wray</span></span>, and <span class="cit-auth"> <span class="cit-name-given-names">R</span> <span class="cit-name-surname">Thompson</span></span> (<span class="cit-pub-date">1993</span>). “<span class="cit-article-title">Prediction of long-term contributions and inbreeding in populations undergoing mass selection</span>”. In: <abbr class="cit-jnl-abbrev">Genet. Res.</abbr> <span class="cit-vol">62</span>.<span class="cit-issue">3</span>, pp. <span class="cit-fpage">231</span>–<span class="cit-lpage">242</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenet.%2BRes.%26rft.volume%253D62%26rft.spage%253D231%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-58-1" title="View reference  in text" id="ref-58">↵</a><div class="cit ref-cit ref-journal" id="cit-798595v1.58" data-doi="10.1017/S0016672300025180"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Wray</span>,  <span class="cit-name-given-names">Naomi R</span></span> and <span class="cit-auth"> <span class="cit-name-given-names">Robin</span> <span class="cit-name-surname">Thompson</span></span> (<span class="cit-pub-date">1990</span>). “<span class="cit-article-title">Prediction of rates of inbreeding in selected populations</span>”. In: <abbr class="cit-jnl-abbrev">Genet. Res.</abbr> <span class="cit-vol">55</span>.<span class="cit-issue">01</span>, pp. <span class="cit-fpage">41</span>–<span class="cit-lpage">54</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetical%2Bresearch%26rft.stitle%253DGenet%2BRes%26rft.aulast%253DWray%26rft.auinit1%253DN.%2BR.%26rft.volume%253D55%26rft.issue%253D1%26rft.spage%253D41%26rft.epage%253D54%26rft.atitle%253DPrediction%2Bof%2Brates%2Bof%2Binbreeding%2Bin%2Bselected%2Bpopulations.%26rft_id%253Dinfo%253Adoi%252F10.1017%252FS0016672300025180%26rft_id%253Dinfo%253Apmid%252F2318415%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1017/S0016672300025180&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=2318415&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F10%2F09%2F798595.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1990CW88900007&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.biorxiv.org/content/10.1101/798595v1.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-disqus-comment" >
  
      
  
  <div class="pane-content">
    <div id="disqus_thread"><noscript><p><a href="http://biorxivstage.disqus.com/?url=https%3A%2F%2Fwww.biorxiv.org%2Fcontent%2F10.1101%2F798595v1" class="" data-icon-position="" data-hide-link-title="0">View the discussion thread.</a></p></noscript></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/10.1101/798207v1" title="Two-stage electro-mechanical coupling of a KV channel in voltage-dependent activation" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous</span></a></span><span class="pager-next"><a href="/content/10.1101/569319v4" title="Neonatal morphometric similarity mapping for predicting brain age and characterizing neuroanatomic variation associated with preterm birth" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    Posted&nbsp;October 09, 2019.  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-biorxiv-art-tools" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-biorxiv_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/10.1101/798595v1.full.pdf" target="_self" class="article-dl-pdf-link link-icon"><i class="icon-external-link-sign"></i> <span class="title">Download PDF</span></a>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-biorxiv-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="Email this Article" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-biorxiv_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/10.1101/798595v1.full" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for your interest in spreading the word about bioRxiv.</p><p>NOTE: Your email address is requested solely to identify you as the sender of this article.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/10.1101/798595v1" class="active" data-icon-position="" data-hide-link-title="0">Estimating the genome-wide contribution of selection to temporal allele frequency change</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from bioRxiv
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would like to see this page from the bioRxiv website.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/942517" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-NpdFySEmih1SpfhrOzbNo1MQq1OozgvmtXI2Trn4uHY" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-biorxiv-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-biorxiv_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="942517" id="node942517--2" data-pisa="biorxiv;798595v1" data-pisa-master="biorxiv;798595" data-seqnum="942517" data-apath="/biorxiv/early/2019/10/09/798595.atom"><div  class="highwire-cite highwire-cite-highwire-article highwire-citation-biorxiv-article-pap-list clearfix" >

      <div class="highwire-cite-title" >
      <div class="highwire-cite-title">Estimating the genome-wide contribution of selection to temporal allele frequency change</div>    </div>
  
      <div  class="highwire-cite-authors" ><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0"><span class="nlm-given-names">Vince</span> <span class="nlm-surname">Buffalo</span></span>, <span class="highwire-citation-author" data-delta="1"><span class="nlm-given-names">Graham</span> <span class="nlm-surname">Coop</span></span></span></div>
  
      <div  class="highwire-cite-metadata" ><span  class="highwire-cite-metadata-journal highwire-cite-metadata">bioRxiv </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">798595; </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata"><span class="doi_label">doi:</span> https://doi.org/10.1101/798595 </span></div>
  
  
  </div>
</div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.biorxiv.org/content/10.1101/798595v1" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.biorxiv.org/content/10.1101/798595v1&amp;title=Estimating%20the%20genome-wide%20contribution%20of%20selection%20to%20temporal%20allele%20frequency%20change" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.biorxiv.org/content/10.1101/798595v1&amp;title=Estimating%20the%20genome-wide%20contribution%20of%20selection%20to%20temporal%20allele%20frequency%20change" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.biorxiv.org/content/10.1101/798595v1&amp;text=Estimating%20the%20genome-wide%20contribution%20of%20selection%20to%20temporal%20allele%20frequency%20change" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.biorxiv.org/content/10.1101/798595v1&amp;title=Estimating%20the%20genome-wide%20contribution%20of%20selection%20to%20temporal%20allele%20frequency%20change" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.biorxiv.org/content/10.1101/798595v1&amp;t=Estimating%20the%20genome-wide%20contribution%20of%20selection%20to%20temporal%20allele%20frequency%20change" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.biorxiv.org/content/10.1101/798595v1&amp;title=Estimating%20the%20genome-wide%20contribution%20of%20selection%20to%20temporal%20allele%20frequency%20change" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.biorxiv.org/content/10.1101/798595v1&amp;title=Estimating%20the%20genome-wide%20contribution%20of%20selection%20to%20temporal%20allele%20frequency%20change" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-biorxiv-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title="Citation Tools"><i class="icon-globe"></i> <span class="title">Citation Tools</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-biorxiv_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="942517" id="node942517--3" data-pisa="biorxiv;798595v1" data-pisa-master="biorxiv;798595" data-seqnum="942517" data-apath="/biorxiv/early/2019/10/09/798595.atom"><div  class="highwire-cite highwire-cite-highwire-article highwire-citation-biorxiv-article-pap-list clearfix" >

      <div class="highwire-cite-title" >
      <div class="highwire-cite-title">Estimating the genome-wide contribution of selection to temporal allele frequency change</div>    </div>
  
      <div  class="highwire-cite-authors" ><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0"><span class="nlm-given-names">Vince</span> <span class="nlm-surname">Buffalo</span></span>, <span class="highwire-citation-author" data-delta="1"><span class="nlm-given-names">Graham</span> <span class="nlm-surname">Coop</span></span></span></div>
  
      <div  class="highwire-cite-metadata" ><span  class="highwire-cite-metadata-journal highwire-cite-metadata">bioRxiv </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">798595; </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata"><span class="doi_label">doi:</span> https://doi.org/10.1101/798595 </span></div>
  
  
  </div>
</div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage&amp;rft.epage&amp;rft.atitle=Estimating%20the%20genome-wide%20contribution%20of%20selection%20to%20temporal%20allele%20frequency%20change&amp;rft.volume&amp;rft.issue&amp;rft.date=2019-01-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=bioRxiv&amp;rft.au=Buffalo%2C+Vince&amp;rft.au=Coop%2C+Graham"></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/942517/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/942517/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/942517/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/942517/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/942517/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/942517/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/942517/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/942517/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/942517/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/942517/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/942517/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/942517/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><div class="item-list"><ul><li class="first"><a href="http://twitter.com/share?url=https%3A//www.biorxiv.org/content/10.1101/798595v1&amp;count=horizontal&amp;via=&amp;text=Estimating%20the%20genome-wide%20contribution%20of%20selection%20to%20temporal%20allele%20frequency%20change&amp;counturl=https%3A//www.biorxiv.org/content/10.1101/798595v1" class="twitter-share-button service-links-twitter-widget" id="twitter_widget" title="Tweet This" rel="nofollow" data-icon-position="" data-hide-link-title="0"><span class="element-invisible">Tweet Widget</span></a></li><li><a href="http://www.facebook.com/plugins/like.php?href=https%3A//www.biorxiv.org/content/10.1101/798595v1&amp;layout=button_count&amp;show_faces=false&amp;action=like&amp;colorscheme=light&amp;width=100&amp;height=21&amp;font=&amp;locale=" id="facebook_like" title="I Like it" class="service-links-facebook-like" rel="nofollow" data-icon-position="" data-hide-link-title="0"><span class="element-invisible">Facebook Like</span></a></li><li class="last"><a href="https://www.biorxiv.org/content/10.1101/798595v1" id="google_plus_one" title="Plus it" class="service-links-google-plus-one" rel="nofollow" data-icon-position="" data-hide-link-title="0"><span class="element-invisible">Google Plus One</span></a></li></ul></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-collections" >
  
        <h2 class="pane-title">Subject Area</h2>
    
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-collections"><div class="highwire-list"><ul class="highwire-article-collection-term-list"><li class="first last odd"><span class="highwire-article-collection-term"><a href="/collection/evolutionary-biology" class="highlight" data-icon-position="" data-hide-link-title="0">Evolutionary Biology<i class="icon-caret-right"></i>
</a></span></li></ul></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-biorxiv-subject-collections block-style-col2" >
  
      
  
  <div class="pane-content">
    <div class="panel-flexible panels-flexible-8 clearfix" id="mini-panel-biorxiv_subject_collections">
<div class="panel-flexible-inside panels-flexible-8-inside">
<div class="panels-flexible-region panels-flexible-region-8-center panels-flexible-region-first panels-flexible-region-last">
  <div class="inside panels-flexible-region-inside panels-flexible-region-8-center-inside panels-flexible-region-inside-first panels-flexible-region-inside-last">
<div class="panel-pane pane-snippet" >
  
      
  
  <div class="pane-content">
    <div class="snippet biorxiv-subject-areas-table-title" id="biorxiv-subject-areas-table-title">
  
      
  <div class="snippet-content">
    <b>Subject Areas</b>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-snippet" >
  
      
  
  <div class="pane-content">
    <div class="snippet biorxiv-subject-areas-view-papers" id="biorxiv-subject-areas-view-papers">
  
      
  <div class="snippet-content">
    <a href="/content/early/recent"><strong>All Articles</strong></a>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-subject-collections" >
  
      
  
  <div class="pane-content">
    <ul id="collection" class="collection highwire-list-expand"><li class="outer collection depth-2  child first"><div class = "data-wrapper"><a href="/collection/animal-behavior-and-cognition" class="" data-icon-position="" data-hide-link-title="0">Animal Behavior and Cognition</a> <span class = "article-count">(1662)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/biochemistry" class="" data-icon-position="" data-hide-link-title="0">Biochemistry</a> <span class = "article-count">(2789)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/bioengineering" class="" data-icon-position="" data-hide-link-title="0">Bioengineering</a> <span class = "article-count">(1946)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/bioinformatics" class="" data-icon-position="" data-hide-link-title="0">Bioinformatics</a> <span class = "article-count">(10400)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/biophysics" class="" data-icon-position="" data-hide-link-title="0">Biophysics</a> <span class = "article-count">(4243)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/cancer-biology" class="" data-icon-position="" data-hide-link-title="0">Cancer Biology</a> <span class = "article-count">(3272)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/cell-biology" class="" data-icon-position="" data-hide-link-title="0">Cell Biology</a> <span class = "article-count">(4641)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/clinical-trials" class="" data-icon-position="" data-hide-link-title="0">Clinical Trials</a> <span class = "article-count">(135)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/developmental-biology" class="" data-icon-position="" data-hide-link-title="0">Developmental Biology</a> <span class = "article-count">(2889)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/ecology" class="" data-icon-position="" data-hide-link-title="0">Ecology</a> <span class = "article-count">(4544)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/epidemiology" class="" data-icon-position="" data-hide-link-title="0">Epidemiology</a> <span class = "article-count">(2041)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/evolutionary-biology" class="" data-icon-position="" data-hide-link-title="0">Evolutionary Biology</a> <span class = "article-count">(7288)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/genetics" class="" data-icon-position="" data-hide-link-title="0">Genetics</a> <span class = "article-count">(5540)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/genomics" class="" data-icon-position="" data-hide-link-title="0">Genomics</a> <span class = "article-count">(6893)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/immunology" class="" data-icon-position="" data-hide-link-title="0">Immunology</a> <span class = "article-count">(2434)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/microbiology" class="" data-icon-position="" data-hide-link-title="0">Microbiology</a> <span class = "article-count">(7616)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/molecular-biology" class="" data-icon-position="" data-hide-link-title="0">Molecular Biology</a> <span class = "article-count">(3051)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/neuroscience" class="" data-icon-position="" data-hide-link-title="0">Neuroscience</a> <span class = "article-count">(18885)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/paleontology" class="" data-icon-position="" data-hide-link-title="0">Paleontology</a> <span class = "article-count">(142)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/pathology" class="" data-icon-position="" data-hide-link-title="0">Pathology</a> <span class = "article-count">(477)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/pharmacology-and-toxicology" class="" data-icon-position="" data-hide-link-title="0">Pharmacology and Toxicology</a> <span class = "article-count">(791)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/physiology" class="" data-icon-position="" data-hide-link-title="0">Physiology</a> <span class = "article-count">(1178)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/plant-biology" class="" data-icon-position="" data-hide-link-title="0">Plant Biology</a> <span class = "article-count">(2771)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/scientific-communication-and-education" class="" data-icon-position="" data-hide-link-title="0">Scientific Communication and Education</a> <span class = "article-count">(686)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/synthetic-biology" class="" data-icon-position="" data-hide-link-title="0">Synthetic Biology</a> <span class = "article-count">(905)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/systems-biology" class="" data-icon-position="" data-hide-link-title="0">Systems Biology</a> <span class = "article-count">(2881)</span></div></li>
<li class="outer collection depth-2  child last"><div class = "data-wrapper"><a href="/collection/zoology" class="" data-icon-position="" data-hide-link-title="0">Zoology</a> <span class = "article-count">(472)</span></div></li>
</ul>  </div>

  
  </div>
  </div>
</div>
</div>
</div>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__VNH5GD8Zz7g6_hGCOZjVjdMndxxC6naiExSSJKX3k_A__kzmEdtbqCsJx5Z9yg2Qy0PptB__ufXm-TxUOl0KZzUw__QJacVmveFyLIBSPkp21xL0ZvuDW9DMBrD2HB1P4Ry5I.js"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__QJacVmveFyLIBSPkp21xL0ZvuDW9DMBrD2HB1P4Ry5I.js" defer="defer"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__N7ERJBYsOWyRnJgyoM125_Aiez2MOJGaUofG1JdWWBg__bLlcwhzZhT-IwoQ_dvm1_xPxU7E7gecFLC6Fyeqtv8s__QJacVmveFyLIBSPkp21xL0ZvuDW9DMBrD2HB1P4Ry5I.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
function euCookieComplianceLoadScripts() {}
//--><!]]>
</script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var eu_cookie_compliance_cookie_name = "";
//--><!]]>
</script>
<script type="text/javascript" src="//d33xdlntwy0kbs.cloudfront.net/cshl_custom.js"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__BmqjBnkz3MgYCAoc25s1lDRMEjLhC3mEPVonUFIHi08__Unwv5-ZIuHBfFwytsjEx1niBVJ7n1T4lPws7VrkHXM4__QJacVmveFyLIBSPkp21xL0ZvuDW9DMBrD2HB1P4Ry5I.js"></script>
  </body>
</html>
