<html xmlns="http://www.w3.org/1999/xhtml"> <head> <style type="text/css">
      	     		 
      	    a                          {background : #ffffff; }
      	    article                    {border-style : dotted; border-width : 2px; }
		 	
		 	div                        {background : #ffffcc;}
		 	div.abstract-title         {font-weight : bold ; font-size : 16pt;}
		 	div.ack                    {border-style : solid ; border-color : red; margin : 2em; }
		 	div.article-meta           {border-style : solid; border-width : 1 pt; margin 1em;}
		 	div.boxed-text             {margin : 5em; border-style : solid;}
		 	div.contrib-group          {margin : 1em; }
		 	div.fig                    {border-style : solid; border-width : 2px; margin : 2em; }
		 	div.funding                {font-weight : bold ; font-size : 16pt;}
		 	div.given-names            {font-style : italic;}
		 	div.intro                  {border-style : inset; margin : 5px;}
		 	div.introduction           {border-style : inset; margin : 5px;}
		 	div.journal-meta           {border-style : solid; border-width : 1 pt; margin 1em;}
		 	div.journal-title-group    {background : #ffeeee;}
		 	div.article-title          {font-weight : bold ; font-size : 18pt;}
		 	div.kwd                    {font-style : italic}
		 	div.meta-name              {font-weight : bold ; font-size : 16pt;}
		 	div.name                   {font-weight : bold;}
		 	div.alt-title              {font-style : italic; font-size : 12px;}
		 	
		 	div.sec                    {border : 2px; margin 5px; padding 2px;}
		 	div.title                  {font-family : courier; font-weight : bold;
		 	                            font-size : 16pt; margin : 5px;}
		 	
		 	div.materials_methods      {border-style : double; margin : 5px; }
		 	div.methods                {border-style : double; margin : 5px; }
		 	
		 	div.results                {border-style : solid; margin : 5px;}
		 	div.background             {border-style : dotted; margin : 5px;}
		 	
		 	div.discussion             {border-style : groove; margin : 5px;}
		 	div.conclusion             {border-style : ridge; margin : 5px;}
		 	div.conclusions             {border-style : ridge; margin : 5px;}
		 	div.supplementary-material {border-style : inset; margin : 5px;}
		 	div.abbreviations          {border-style : double; border-color : red; margin : 5px;}
		 	div.competinginterests     {border-style : double; border-color : blue; margin : 5px;}
		 	div.acknowledgements       {border-style : double; border-color : green; margin : 5px;}
		 	div.authors_contributions   {border-style : double; border-color : purple; margin : 5px;}
		 	
		 	div.publisher              {border-style : outset; margin : 5px;}
		 	div.fn-type-conflict       {background : #f88; }
		 	div.fn-type-con            {background : #ddf; }
		 	div.fn-type-other          {background : #ddd; }
		 	
		 	div.unknown                {background : #ffd;
									 	  border-style : solid;
									 	  border-width : 1px;
									 	  padding : 2 px;}
		 	  
      	    table                      {background : #ffffdd;}
		 	tr                         {background : #ddddff; padding : 1px;}
		 	
		 	span                       {background : #ffcccc;}
		 	
		 	span.citation-author       {font-family : helvetica; background : #ffeeee;}
		 	span.collab                {background : #ddffff; }
		 	span.comment               {font-family : courier; font-size : 6px; background : #ffaaff;}
		 	span.contrib               {background : #ffffff;}
		 	span.corresp               {background : #ddffdd; }
		 	span.doi                   {background : #ffffff;}
		 	span.email                 {font-family : courier; }
		 	span.etal                  {font-style : italic;}
		 	span.fpage                 {font-family : courier;}
		 	span.given-names           {background : #ffffff;}
		 	span.iso-abbrev            {background : #ffffff;}
		 	span.issn-epub             {background : #ffffff;}
		 	span.issn-ppub             {background : #ffffff;}
		 	span.journal-title         {background : #ffffff;}
		 	span.lpage                 {font-family : courier;}
		 	span.mixed-article-title   {font-style : italic ;}
		 	span.nlm-ta                {background : #ffffff;}
		 	span.pmc                   {background : #ffffff;}
		 	span.pmcid                 {background : #ffffff;}
		 	span.pmid                  {background : #ffffff;}
		 	span.publisher             {background : #ffffff;}
		 	span.publisher-id          {background : #ffffff;}
		 	span.publisher-name        {background : #ffffff;}
		 	span.source                {background : #ffffff;}
		 	span.subject               {background : #ffffff;}
		 	span.surname               {background : #ffffff;}
		 	span.volume                {font-family : courier; font-weight : bold;}
		 	span.year                  {font-family : courier ; font-style : italic;}
			</style> </head> <body> <div class="front" title="front"> <div class="journal-meta" tagx="journal-meta" title="journal-meta"><span class="nlm-ta" title="nlm-ta">BMC Evol Biol</span><span class="journal-title" tagx="journal-title" title="journal-title">BMC Evolutionary Biology</span><span class="issn-epub" tagx="issn" title="issn-epub">1471-2148</span><div class="publisher" tagx="publisher" title="publisher"><span class="publisher-name" tagx="publisher-name" title="publisher-name">BioMed Central</span></div> </div> <div class="article-meta" tagx="article-meta" title="article-meta"><span class="publisher-id" title="publisher-id">1471-2148-7-199</span><span class="pmid" title="pmid"> pmid: <a href="http://www.ncbi.nlm.nih.gov/pubmed/17958882">17958882</a></span><span class="doi" title="doi"> doi: <a href="https://dx.doi.org/10.1186/1471-2148-7-199">10.1186/1471-2148-7-199</a></span><div class="article-categories" title="article-categories">
: <span class="subject" title="subject">Research Article</span></div> <div class="title-group" tagx="title-group" title="title-group"> <div class="article-title" title="article-title">
Chromosomal instability in Afrotheria: fragile sites, evolutionary breakpoints and phylogenetic inference from genome sequence assemblies</div> </div> <div class="contrib-group" title="contrib-group"><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ruiz-Herrera</span><span class="given-names" tagx="given-names" title="given-names">Aurora</span></span><a href="#I1">1</a><a href="#I2">2</a><span class="email" tagx="email" title="email">moreno@ipvgen.unipv.it</span></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Robinson</span><span class="given-names" tagx="given-names" title="given-names">Terence J</span></span><a href="#I3">3</a><span class="email" tagx="email" title="email">tjr@sun.ac.za</span></span></div><span class="citation_author_institution" id="I1">[1], </span><span class="citation_author_institution" id="I2">[2], </span><span class="citation_author_institution" id="I3">[3], </span><span class="pub-date-collection" title="pub-date-collection">collection: <span>2007</span></span><span class="pub-date-epub" title="pub-date-epub">epub: <span>2007-10-10</span></span><span class="volume" tagx="volume" title="volume">7</span><span class="fpage" tagx="fpage" title="fpage">199</span><span class="lpage" tagx="lpage" title="lpage">199</span><a href="http://www.biomedcentral.com/1471-2148/7/199" /><span class="history" title="history"><span class="received" title="received">received: 2007-4-25</span><span class="accepted" title="accepted">accepted: 2007-10-24</span></span><div class="permissions"><span class="copyright" title="copyright">(C) <span class="copyright-holder" tagx="copyright-holder" title="copyright-holder">Ruiz-Herrera and Robinson; licensee BioMed Central Ltd.</span>, <span class="copyright-year" tagx="copyright-year" title="copyright-year">2007</span></span><span class="license" title="license"> <p>This is an Open Access article distributed under the terms of the Creative Commons Attribution License (<a href="http://creativecommons.org/licenses/by/2.0" />), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited.</p></span></div> <div class="abstract" title="abstract"> <div class="abstract-title" title="abstract-title">
Abstract</div> <div class="background" title="sec"> <div class="title" tagx="title" title="title">
Background</div> <p>Extant placental mammals are divided into four major clades (Laurasiatheria, Supraprimates, Xenarthra and Afrotheria). Given that Afrotheria is generally thought to root the eutherian tree in phylogenetic analysis of large nuclear gene data sets, the study of the organization of the genomes of afrotherian species provides new insights into the dynamics of mammalian chromosomal evolution. Here we test if there are chromosomal bands with a high tendency to break and reorganize in Afrotheria, and by analyzing the expression of aphidicolin-induced common fragile sites in three afrotherian species, whether these are coincidental with recognized evolutionary breakpoints.</p> </div> <div class="results" title="sec"> <div class="title" tagx="title" title="title">
Results</div> <p>We described 29 fragile sites in the aardvark (OAF) genome, 27 in the golden mole (CAS), and 35 in the elephant-shrew (EED) genome. We show that fragile sites are conserved among afrotherian species and these are correlated with evolutionary breakpoints when compared to the human (HSA) genome. Inddition, by computationally scanning the newly released opossum (<i>Monodelphis domestica</i>) and chicken sequence assemblies for use as outgroups to Placentalia, we validate the HSA 3/21/5 chromosomal synteny as a rare genomic change that defines the monophyly of this ancient African clade of mammals. On the other hand, support for HSA 1/19p, which is also thought to underpin Afrotheria, is currently ambiguous.</p> </div> <div class="conclusion" title="sec"> <div class="title" tagx="title" title="title">
Conclusion</div> <p>We provide evidence that (i) the evolutionary breakpoints that characterise human syntenies detected in the basal Afrotheria correspond at the chromosomal band level with fragile sites, (ii) that HSA 3p/21 was in the amniote ancestor (i.e., common to turtles, lepidosaurs, crocodilians, birds and mammals) and was subsequently disrupted in the lineage leading to marsupials. Its expansion to include HSA 5 in Afrotheria is unique and (iii) that its fragmentation to HSA 3p/21 + HSA 5/21 in elephant and manatee was due to a fission within HSA 21 that is probably shared by all Paenungulata.</p> </div> </div> </div> </div> <div class="body" title="body"> <div class="background" title="sec"> <div class="title" tagx="title" title="title">
Background</div> <p>Analyzing how mammalian genomes are organized and how chromosomal rearrangements are involved in speciation and macroevolution are fundamental to understanding the dynamics of mammalian chromosomal evolution. Phylogenetic analysis of both nuclear and mitochondrial DNA [<a href="#B1">1</a>-<a href="#B6">6</a>], among others, as well as the insertion sites of multiple long interspersed elements (LINE, [<a href="#B7">7</a>]) and long terminal repeats (LTRs, [<a href="#B8">8</a>]), all support the division of extant placental mammals into four major clades. These are Laurasiatheria and Supraprimates that together form Boreoeutheria (with a northern hemisphere origin), and Xenarthra and Afrotheria which have a Gondwanan (southern hemisphere) genesis, although this biogeographic hypothesis is not without detractors [<a href="#B9">9</a>]. Although Afrotheria is usually depicted as basal in sequence based phylogenies, the first divergence in the placental tree has been a matter of concern for some time. For example, some argue [<a href="#B8">8</a>] for a basal Xenarthra (the so-called Epitheria hypothesis, [<a href="#B10">10</a>]) on insertion sites of retroposed elements (but see [<a href="#B11">11</a>] for a contrary view), while data from 218 genes encompassing 205 kb of sequence resulted in a highly supported phylogeny that places the root between Afrotheria and other Placentalia [<a href="#B6">6</a>], consistent with the basal Afrotheria or Exafroplacentalia hypothesis [<a href="#B4">4</a>]. Most recently, however, Xenarthra and Afrotheria have been placed on a common basal branch (the Atlantogenata or Xenafrotheria hypothesis), to the exclusion of Boreoeutheria [<a href="#B12">12</a>].</p> <p>Afrotheria, the focus of our study, includes six mammalian orders all with an Afro-Arabian origin that exhibit extreme morphogical diversity and niche preference thought to result from the long period of isolation when Africa was an island continent 105-25 mya [<a href="#B13">13</a>]. The six orders are Proboscidea (elephant), Sirenia (manatees and dugongs), Hyracoidea (hyrax), Tubulidentata (aardvark), Macroscelidea (elephant shrews) and Afrosoricida (golden moles and tenrecs). In cases where the analysis of primary sequences generates ambiguous phylogenetic results, "rare genomic changes" (RGCs, [<a href="#B14">14</a>]) such as indels, LINEs, SINEs and chromosomal rearrangements have been widely viewed as markers that could, given their low levels of homoplasy, provide additional resolution to seemingly intractable phylogenetic problems (see [<a href="#B15">15</a>] for application of SINEs in vertebrate phylogenetics). So far, afrotherian monophyly is supported by a suite of sequence-based characters that include a 9 bp deletion in exon 11 of the BRCA1 gene [<a href="#B16">16</a>], 5' and 3' deletions in exon 26 of apolipoprotein B [<a href="#B5">5</a>], the presence of a unique family of SINEs (AfroSINEs, [<a href="#B17">17</a>-<a href="#B19">19</a>]), long interspersed elements (LINE 1, [<a href="#B7">7</a>]) and long terminal repeat (LTR) elements [<a href="#B8">8</a>]. Consistent with the view that chromosomal rearrangements are similarly rarely homoplasious, and therefore robust indicators of evolutionary change (a default rate of 1–2 changes per 10 million years of mammalian evolution has been suggested, [<a href="#B20">20</a>-<a href="#B22">22</a>]), it comes as no surprise that in addition to providing evidence in support of the uniqueness of Afrotheria, chromosomal syntenies have also proved useful for clarifying the phylogenetic relationships within the group [<a href="#B23">23</a>,<a href="#B24">24</a>].</p> <p>One of the most important features of mammalian chromosomal evolution is the suggestion of a non-random distribution of regions implicated in evolutionary chromosomal reorganization, the so-called "fragile-breakage" hypothesis [<a href="#B25">25</a>]. Related to this, recent experimental data have demonstrated a correlation between the location of fragile sites and evolutionary breakpoints [<a href="#B26">26</a>-<a href="#B28">28</a>] suggesting that these unstable regions could be one of many factors implicated in the evolutionary process. At the cytogenetic level, fragile sites are expressed as non-stained gaps and breaks when cells are cultured under specific conditions [<a href="#B29">29</a>]. In general, fragile sites can be expressed by agents such as aphidicolin, BrdU and 5-azacytidine among others, which delay or inhibit DNA replication or repair [<a href="#B30">30</a>]. According to their frequency in the human population, as well as their mechanisms of expression, fragile sites have been conventionally classified into two groups: common and rare [<a href="#B31">31</a>]. Common fragile sites in particular have been studied in different mammalian species confirming the initial hypothesis that they are structural characteristics of mammalian chromosomes [<a href="#B31">31</a>]. Common fragile sites have been expressed in rodents [<a href="#B32">32</a>-<a href="#B37">37</a>], pig and cow [<a href="#B38">38</a>-<a href="#B41">41</a>], horse [<a href="#B42">42</a>], cat [<a href="#B43">43</a>-<a href="#B45">45</a>], dog [<a href="#B46">46</a>,<a href="#B47">47</a>] and different primate species [<a href="#B26">26</a>,<a href="#B48">48</a>-<a href="#B51">51</a>]. However, in all instances the species studied group within Laurasiatheria and Supraprimates, the most distant relatives of Xenarthra and Afrotheria that are thought to have diverged ~93 mya [<a href="#B52">52</a>]. There is, at this point, no comparable data for the deeper divergences such as the Afrotheria whose separation from Boreoeutheria and Xenarthra is estimated at ~105 mya [<a href="#B52">52</a>].</p> <p>In an attempt to test if there are loci with a high tendency to break and reorganize in the Afrotheria, and whether these are coincidental with evolutionary breakpoints, we have analyzed the expression of aphidicolin-induced common fragile sites in fibroblast cultures from different specimens of three afrotherian species. These are the aardvark (<i>Orycteropus afer</i>, OAF, Tubulidentata), golden mole (<i>Chrysochloris asiatica</i>, CAS, Afrosoricida) and elephant-shrew (<i>Elephantulus edwardii</i>, EED, Macroscelidea). Given the position of Afrotheria near the root of Placentalia (species on the so-called "eutherian" side of the "metatherian-eutherian" dichotomy), the analysis of chromosomal instability in these species provides a unique opportunity to further our understanding of the mechanisms underpinning mammalian chromosomal evolution.</p> </div> <div class="results" title="sec"> <div class="title" tagx="title" title="title">
Results</div> <div class="fragilesiteexpression" title="sec"> <div class="title" tagx="title" title="title">
Fragile site expression</div> <div class="(i)orycteropusafer" title="sec"> <div class="title" tagx="title" title="title">
(i) <i>Orycteropus afer</i></div> <p>A total of 652 metaphase spreads were analysed in two specimens, 374 from cultures treated with 0.2 μM APC, and 278 from control cultures (table <a href="#T1">1</a>). As expected, cells treated with aphidicolin presented the highest number of chromosomal aberrations (51.34% of the total metaphases), a 13-fold increase with respect to the control cultures. A total of 287 chromosomal abnormalities were detected of which the most common aberrations were chromatid breaks (61–94% of all aberrations detected, see table <a href="#T2">2</a>).</p> <div class="table-wrap_UNKNOWN" position="float" id="T1"><span class="label" tagx="label" title="label">Table 1</span> <p>Numbers of metaphase cells analysed, chromosomal breaks (gaps included) and chromatid breaks (gaps included) per metaphase observed in aphidicolin-treated and control cultures from the all afrotherian species and specimens studied.</p>  <table frame="hsides" rules="groups"> <thead> <tr> <td /> <td align="center" colspan="4"><b>0.2 μM APC</b></td> <td align="center" colspan="4"><b>CONTROL</b></td> </tr> </thead> <tbody> <tr> <td align="center"><b>Specimen</b></td> <td align="center"><b>Metaphase cells analysed</b></td> <td align="center"><b>Normal (%)</b></td> <td align="center"><b>Aberrant (%)</b></td> <td align="center"><b>Breaks/Metaphase</b></td> <td align="center"><b>Metaphase cells analysed</b></td> <td align="center"><b>Normal (%)</b></td> <td align="center"><b>Aberrant (%)</b></td> <td align="center"><b>Breaks/Metaphase</b></td> </tr> <tr> <td colspan="9"> <hr /> </td> </tr> <tr> <td align="center"><b>OAF1</b></td> <td align="center">130</td> <td align="center">48 (36.92%)</td> <td align="center">82 (63.08%)</td> <td align="center">1.64</td> <td align="center">132</td> <td align="center">127 (96.21%)</td> <td align="center">5 (3.78%)</td> <td align="center">0.045</td> </tr> <tr> <td align="center"><b>OAF3</b></td> <td align="center">244</td> <td align="center">134 (54.92%)</td> <td align="center">110 (46.08%)</td> <td align="center">0.59</td> <td align="center">146</td> <td align="center">140 (95.89%)</td> <td align="center">6 (4.11%)</td> <td align="center">0.041</td> </tr> <tr> <td colspan="9"> <hr /> </td> </tr> <tr> <td align="center"><i>total</i></td> <td align="center"><i>374</i></td> <td align="center"><i>182 (48.66%)</i></td> <td align="center"><i>192 (51.34%)</i></td> <td align="center"><i>1.12</i></td> <td align="center"><i>278</i></td> <td align="center"><i>267 (96.04%)</i></td> <td align="center"><i>11 (3.96%)</i></td> <td align="center"><i>0.043</i></td> </tr> <tr> <td colspan="9"> <hr /> </td> </tr> <tr> <td align="center"><b>CAS1</b></td> <td align="center">134</td> <td align="center">33 (24.66%)</td> <td align="center">101 (75.34%)</td> <td align="center">1.42</td> <td align="center">223</td> <td align="center">204 (91.48%)</td> <td align="center">19 (8.52%)</td> <td align="center">0.10</td> </tr> <tr> <td align="center"><b>CAS2</b></td> <td align="center">165</td> <td align="center">51 (30.91%)</td> <td align="center">114 (69.09%)</td> <td align="center">1.60</td> <td align="center">209</td> <td align="center">197 (94.26%)</td> <td align="center">12 (5.74%)</td> <td align="center">0.05</td> </tr> <tr> <td align="center"><b>CAS3</b></td> <td align="center">183</td> <td align="center">67 (36.61%)</td> <td align="center">116 (63.39%)</td> <td align="center">0.95</td> <td align="center">133</td> <td align="center">124 (93.23%)</td> <td align="center">9 (6.77%)</td> <td align="center">0.07</td> </tr> <tr> <td colspan="9"> <hr /> </td> </tr> <tr> <td align="center"><i>total</i></td> <td align="center"><i>482</i></td> <td align="center"><i>151 (31.33%)</i></td> <td align="center"><i>331 (68.67%)</i></td> <td align="center"><i>0.99</i></td> <td align="center"><i>565</i></td> <td align="center"><i>525 (92.92%)</i></td> <td align="center"><i>40 (7.08%)</i></td> <td align="center"><i>0.07</i></td> </tr> <tr> <td colspan="9"> <hr /> </td> </tr> <tr> <td align="center"><b>EED1</b></td> <td align="center">188</td> <td align="center">49 (26.06%)</td> <td align="center">139 (73.94%)</td> <td align="center">1.29</td> <td align="center">233</td> <td align="center">134 (90.54%)</td> <td align="center">14 (9.46%)</td> <td align="center">0.10</td> </tr> <tr> <td align="center"><b>EED2</b></td> <td align="center">170</td> <td align="center">29 (17.06%)</td> <td align="center">141 (82.94%)</td> <td align="center">1.44</td> <td align="center">164</td> <td align="center">150 (89.82%)</td> <td align="center">17 (10.18%)</td> <td align="center">0.12</td> </tr> <tr> <td align="center"><b>EED3</b></td> <td align="center">162</td> <td align="center">46 (28.40%)</td> <td align="center">116 (71.60%)</td> <td align="center">1.23</td> <td align="center">193</td> <td align="center">177 (91.71%)</td> <td align="center">16 (8.29%)</td> <td align="center">0.07</td> </tr> <tr> <td align="center"><b>EED4</b></td> <td align="center">100</td> <td align="center">24 (24%)</td> <td align="center">76 (76%)</td> <td align="center">1.36</td> <td align="center">111</td> <td align="center">104 (93.69%)</td> <td align="center">7 (6.31%)</td> <td align="center">0.09</td> </tr> <tr> <td colspan="9"> <hr /> </td> </tr> <tr> <td align="center"><i>total</i></td> <td align="center"><i>620</i></td> <td align="center"><i>148 (23.87%)</i></td> <td align="center"><i>280 (78.21%)</i></td> <td align="center"><i>1.33</i></td> <td align="center"><i>619</i></td> <td align="center"><i>565 (91.32%)</i></td> <td align="center"><i>54 (8.77%)</i></td> <td align="center"><i>0.09</i></td> </tr> </tbody> </table> <div class="table-wrap-foot_UNKNOWN"> <p>Abbreviations – OAF: <i>O. afer</i>, CAS: <i>C. asiatica </i>and EED: <i>E. edwardii</i>.</p> </div> </div> <div class="table-wrap_UNKNOWN" position="float" id="T2"><span class="label" tagx="label" title="label">Table 2</span> <p>Number of chromosomal aberrations observed in cells analysed from aphidicolin-treated cultures of three afrotherian species.</p>  <table frame="hsides" rules="groups"> <thead> <tr> <td /> <td align="center" colspan="5"><b>Chromosomal aberrations</b></td> </tr> </thead> <tbody> <tr> <td align="center"><b>Specimens</b></td> <td align="center"><b>chr breaks</b></td> <td align="center"><b>chr gaps</b></td> <td align="center"><b>cht breaks</b></td> <td align="center"><b>cht gaps</b></td> <td align="center"><b>total</b></td> </tr> <tr> <td colspan="6"> <hr /> </td> </tr> <tr> <td align="center"><b>OAF1</b></td> <td align="center">8 (5.59%)</td> <td align="center">46 (32.17%)</td> <td align="center">87 (60.84%)</td> <td align="center">2 (1.40%)</td> <td align="center">143</td> </tr> <tr> <td align="center"><b>OAF3</b></td> <td align="center">1 (0.69%)</td> <td align="center">5 (3.47%)</td> <td align="center">135 (93.75%)</td> <td align="center">3 (2.08%)</td> <td align="center">144</td> </tr> <tr> <td colspan="6"> <hr /> </td> </tr> <tr> <td align="center"><b>CAS1</b></td> <td align="center">1 (0.69%)</td> <td align="center">5 (3.47%)</td> <td align="center">135 (93.75%)</td> <td align="center">3 (2.08%)</td> <td align="center">144</td> </tr> <tr> <td align="center"><b>CAS2</b></td> <td align="center">2 (1.10%)</td> <td align="center">10 (5.49%)</td> <td align="center">152 (83.52%)</td> <td align="center">18 (9.89%)</td> <td align="center">182</td> </tr> <tr> <td align="center"><b>CAS3</b></td> <td align="center">3 (1.71%)</td> <td align="center">38 (21.71%)</td> <td align="center">124 (70.86%)</td> <td align="center">10 (5.71%)</td> <td align="center">175</td> </tr> <tr> <td colspan="6"> <hr /> </td> </tr> <tr> <td align="center"><b>EED1</b></td> <td align="center">0 (0%)</td> <td align="center">13 (5.35%)</td> <td align="center">224 (92.18%)</td> <td align="center">6 (2.47%)</td> <td align="center">243</td> </tr> <tr> <td align="center"><b>EED2</b></td> <td align="center">6 (2.45%)</td> <td align="center">46 (18.78%)</td> <td align="center">183 (74.69%)</td> <td align="center">10 (4.08%)</td> <td align="center">245</td> </tr> <tr> <td align="center"><b>EED3</b></td> <td align="center">0 (0%)</td> <td align="center">50 (25%)</td> <td align="center">135 (67.65%)</td> <td align="center">15 (7.5%)</td> <td align="center">200</td> </tr> <tr> <td align="center"><b>EED4</b></td> <td align="center">1 (0.74%)</td> <td align="center">35 (25.74%)</td> <td align="center">92 (67.65%)</td> <td align="center">8 (5.88%)</td> <td align="center">136</td> </tr> </tbody> </table> <div class="table-wrap-foot_UNKNOWN"> <p>Abbreviations – OAF: <i>O. afer</i>, CAS: <i>C. asiatica</i>, EED: <i>E. edwardii</i>, <i>chr breaks</i>: chromosome breaks, <i>chr gaps</i>: chromosome gaps, <i>cht breaks</i>: chromatid breaks and <i>cht gaps</i>: chromatid gaps.</p> </div> </div> <p>For each data set, the FSM program used in our analysis of fragile sites provides a critical value (C<sub>α</sub>) that indicates the minimum number of breaks needed for a chromosomal band to be considered fragile. This value was ≥3 for aardvark cultures treated with 0.2 μM aphidicolin. The number of fragile sites detected ranged from 23 in OAF specimen 1 to 12 in OAF specimen 2. Given the intraspecific variability of fragile site expression in mammalian species, we combined all expressed sites into a single species-specific analysis (this was also done for the golden mole and elephant shrew, see below). On this basis, a total of 29 sites were considered fragile in the aardvark (figure <a href="#F1">1a</a> and table <a href="#T3">3</a>) showing that there are regions in this species' genome that are prone to breakage under specific culture conditions; of these, six were expressed in both aardvark specimens (1p11, 1q18, 1q44, 2q13, 2q15 and 3p11, table <a href="#T3">3</a>).</p> <div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/"><span class="label" tagx="label" title="label">Figure 1</span></a></div>   <p><b>Ideograms representative of <i>O. afer </i>(A), <i>C. asiatica </i>(B) and <i>E. edwardii </i>(C) chromosomes</b>. The regions of homology with human chromosomes [see 24, 53, 54] are depicted in red and indicated by the numbers to the left of each chromosomal schematic. In aardvark, homology with the African elephant chromosomes is shown in blue. The location of aphidicolin-induced fragile sites (fs) found in all specimens studied is indicated to the right of each chromosomal schematic in the three afrotherian species. Asterisks mark fragile sites conserved in at least two of the three species studied. Heterochromatic regions are marked by diagonal lines.</p>   </div> <div class="table-wrap_UNKNOWN" position="float" id="T3"><span class="label" tagx="label" title="label">Table 3</span> <p>Chromosomal bands which are considered to contain fragile sites in the two aardvark (OAF) specimens studied.</p>  <table frame="hsides" rules="groups"> <thead> <tr> <td /> <td align="left" colspan="2"><b>No. aberrations</b></td> </tr> </thead> <tbody> <tr> <td align="left"><b>band</b></td> <td align="left"><b>OAF 1</b></td> <td align="left"><b>OAF3</b></td> </tr> <tr> <td colspan="3"> <hr /> </td> </tr> <tr> <td align="left">1p11</td> <td align="left">4</td> <td align="left">6</td> </tr> <tr> <td align="left">1q13</td> <td align="left">0</td> <td align="left">4</td> </tr> <tr> <td align="left">1q18</td> <td align="left">4</td> <td align="left">5</td> </tr> <tr> <td align="left">1q22</td> <td align="left">7</td> <td align="left">0</td> </tr> <tr> <td align="left">1q28</td> <td align="left">6</td> <td align="left">0</td> </tr> <tr> <td align="left">1q38</td> <td align="left">4</td> <td align="left">0</td> </tr> <tr> <td align="left">1q42</td> <td align="left">6</td> <td align="left">0</td> </tr> <tr> <td align="left">1q44</td> <td align="left">7</td> <td align="left">7</td> </tr> <tr> <td align="left">2q13</td> <td align="left">13</td> <td align="left">5</td> </tr> <tr> <td align="left">2q25</td> <td align="left">3</td> <td align="left">4</td> </tr> <tr> <td align="left">2q31</td> <td align="left">0</td> <td align="left">6</td> </tr> <tr> <td align="left">2q42</td> <td align="left">0</td> <td align="left">4</td> </tr> <tr> <td align="left">2q46</td> <td align="left">4</td> <td align="left">0</td> </tr> <tr> <td align="left">2q48</td> <td align="left">4</td> <td align="left">0</td> </tr> <tr> <td align="left">3p11</td> <td align="left">7</td> <td align="left">5</td> </tr> <tr> <td align="left">3q12</td> <td align="left">3</td> <td align="left">0</td> </tr> <tr> <td align="left">3q24</td> <td align="left">0</td> <td align="left">4</td> </tr> <tr> <td align="left">3q26</td> <td align="left">0</td> <td align="left">5</td> </tr> <tr> <td align="left">3q29</td> <td align="left">3</td> <td align="left">0</td> </tr> <tr> <td align="left">4p13</td> <td align="left">3</td> <td align="left">0</td> </tr> <tr> <td align="left">4q19</td> <td align="left">4</td> <td align="left">0</td> </tr> <tr> <td align="left">4q24</td> <td align="left">3</td> <td align="left">0</td> </tr> <tr> <td align="left">5p13</td> <td align="left">3</td> <td align="left">0</td> </tr> <tr> <td align="left">5p15</td> <td align="left">0</td> <td align="left">6</td> </tr> <tr> <td align="left">6p11</td> <td align="left">6</td> <td align="left">0</td> </tr> <tr> <td align="left">6q23</td> <td align="left">10</td> <td align="left">0</td> </tr> <tr> <td align="left">7q11</td> <td align="left">3</td> <td align="left">0</td> </tr> <tr> <td align="left">7q13</td> <td align="left">4</td> <td align="left">0</td> </tr> <tr> <td align="left">Xp13</td> <td align="left">3</td> <td align="left">0</td> </tr> <tr> <td colspan="3"> <hr /> </td> </tr> <tr> <td align="left">total</td> <td align="left">114</td> <td align="left">66</td> </tr> </tbody> </table> <div class="table-wrap-foot_UNKNOWN"> <p>The number of chromosomal aberrations (chromosomal breaks, chromosomal gaps, chromatid breaks and chromatid gaps) observed in each specimen (OAF1 and OAF3) at each fragile band is provided.</p> </div> </div> </div> <div class="(ii)chrysochlorisasiatica" title="sec"> <div class="title" tagx="title" title="title">
(ii) <i>Chrysochloris asiatica</i></div> <p>A total of 1047 metaphase spreads were analysed from the three specimens included in our investigation: 482 cells from cultures treated with 0.2 μM APC, and 565 from control cultures (table <a href="#T1">1</a>). Cells treated with aphidicolin presented the highest number of chromosomal aberrations (68.67% of the total metaphases scored) reflecting a 10-fold increase with respect to the control cultures. A total of 501 chromosomal abnormalities were detected of which chromatid breaks were the most frequent class of aberration encountered in this species (71–94%, table <a href="#T2">2</a>).</p> <p>Analysis of the aphidicolin induced aberrations using the FSM program indicate that chromosomal bands characterised by ≥3 or ≥4 abnormalities per band (depending on the specimen analysed) could be considered fragile. Using these values, a total of 27 fragile sites were detected in the golden mole genome (figure <a href="#F1">1b</a> and table <a href="#T4">4</a>), with the number of fragile sites ranging from 12 in CAS specimen 1, to 21 in CAS specimen 3 (table <a href="#T4">4</a>). Eight of the fragile sites (1p13, 1q13, 1q32, 2p13, 2q22, 2q31, 6q23 and 8q17) were found to be expressed in all three specimens examined (table <a href="#T4">4</a>).</p> <div class="table-wrap_UNKNOWN" position="float" id="T4"><span class="label" tagx="label" title="label">Table 4</span> <p>Chromosomal bands that are considered to contain fragile sites in the three golden moles (CAS) specimens studied.</p>  <table frame="hsides" rules="groups"> <thead> <tr> <td /> <td align="left" colspan="3"><b>No. aberrations</b></td> </tr> </thead> <tbody> <tr> <td align="left"><b>band</b></td> <td align="left"><b>CAS1</b></td> <td align="left"><b>CAS2</b></td> <td align="left"><b>CAS3</b></td> </tr> <tr> <td colspan="4"> <hr /> </td> </tr> <tr> <td align="left">1p15</td> <td align="left">0</td> <td align="left">0</td> <td align="left">8</td> </tr> <tr> <td align="left">1p13</td> <td align="left">8</td> <td align="left">8</td> <td align="left">5</td> </tr> <tr> <td align="left">1p11</td> <td align="left">0</td> <td align="left">0</td> <td align="left">3</td> </tr> <tr> <td align="left">1q13</td> <td align="left">11</td> <td align="left">8</td> <td align="left">5</td> </tr> <tr> <td align="left">1q18</td> <td align="left">3</td> <td align="left">0</td> <td align="left">0</td> </tr> <tr> <td align="left">1q21</td> <td align="left">0</td> <td align="left">8</td> <td align="left">3</td> </tr> <tr> <td align="left">1q32</td> <td align="left">6</td> <td align="left">7</td> <td align="left">4</td> </tr> <tr> <td align="left">2p15</td> <td align="left">0</td> <td align="left">0</td> <td align="left">8</td> </tr> <tr> <td align="left">2p13</td> <td align="left">6</td> <td align="left">6</td> <td align="left">3</td> </tr> <tr> <td align="left">2q17</td> <td align="left">0</td> <td align="left">0</td> <td align="left">3</td> </tr> <tr> <td align="left">2q22</td> <td align="left">7</td> <td align="left">7</td> <td align="left">5</td> </tr> <tr> <td align="left">2q31</td> <td align="left">18</td> <td align="left">26</td> <td align="left">25</td> </tr> <tr> <td align="left">3p11</td> <td align="left">0</td> <td align="left">4</td> <td align="left">0</td> </tr> <tr> <td align="left">3q15</td> <td align="left">0</td> <td align="left">0</td> <td align="left">3</td> </tr> <tr> <td align="left">3q24</td> <td align="left">0</td> <td align="left">0</td> <td align="left">4</td> </tr> <tr> <td align="left">3q25</td> <td align="left">3</td> <td align="left">0</td> <td align="left">0</td> </tr> <tr> <td align="left">4q24</td> <td align="left">0</td> <td align="left">0</td> <td align="left">3</td> </tr> <tr> <td align="left">5q24</td> <td align="left">3</td> <td align="left">0</td> <td align="left">0</td> </tr> <tr> <td align="left">6q23</td> <td align="left">3</td> <td align="left">6</td> <td align="left">6</td> </tr> <tr> <td align="left">7q13</td> <td align="left">0</td> <td align="left">4</td> <td align="left">3</td> </tr> <tr> <td align="left">7q25</td> <td align="left">0</td> <td align="left">0</td> <td align="left">4</td> </tr> <tr> <td align="left">8p15</td> <td align="left">0</td> <td align="left">0</td> <td align="left">6</td> </tr> <tr> <td align="left">8q17</td> <td align="left">5</td> <td align="left">5</td> <td align="left">15</td> </tr> <tr> <td align="left">9p13</td> <td align="left">0</td> <td align="left">0</td> <td align="left">3</td> </tr> <tr> <td align="left">9q14</td> <td align="left">0</td> <td align="left">4</td> <td align="left">6</td> </tr> <tr> <td align="left">10q23</td> <td align="left">0</td> <td align="left">6</td> <td align="left">0</td> </tr> <tr> <td align="left">12q12</td> <td align="left">3</td> <td align="left">0</td> <td align="left">0</td> </tr> <tr> <td colspan="4"> <hr /> </td> </tr> <tr> <td align="left">Total</td> <td align="left">76</td> <td align="left">99</td> <td align="left">125</td> </tr> </tbody> </table> <div class="table-wrap-foot_UNKNOWN"> <p>The number of chromosomal aberrations (chromosomal breaks, chromosomal gaps, chromatid breaks and chromatid gaps) observed in each specimen studied (CAS1, CAS2 and CAS3) at each fragile band is provided.</p> </div> </div> </div> <div class="(iii)elephantulusedwardii" title="sec"> <div class="title" tagx="title" title="title">
(iii) <i>Elephantulus edwardii</i></div> <p>A total of 1239 metaphase spreads were analysed in the four specimens of this species of which 620 were from cultures treated with 0.2 μM APC, and 619 from control cultures (table <a href="#T1">1</a>). We detected 824 chromosomal abnormalities of which chromatid breaks were the most frequent class of aberration scored (67–92%, table <a href="#T2">2</a>).</p> <p>Mirroring the results in the golden mole, the critical value generated by the FSM program for the elephant shrew was ≥3 or, depending on the specimen studied, ≥4 chromosomal abnormalities per band. The number of fragile sites detected ranged from 14 in EED specimen 4 to 20 in EED specimen 3 (table <a href="#T5">5</a>); in total 35 sites were considered fragile in the elephant-shrew genome (figure <a href="#F1">1c</a> and table <a href="#T5">5</a>). Only three of these (2p16, 3q26 and 5q25) are expressed in all four specimens studied (table <a href="#T5">5</a>).</p> <div class="table-wrap_UNKNOWN" position="float" id="T5"><span class="label" tagx="label" title="label">Table 5</span> <p>Chromosomal bands which are considered to contain fragile sites in the three elephant-shrew (EED) specimens studied.</p>  <table frame="hsides" rules="groups"> <thead> <tr> <td /> <td align="left" colspan="4"><b>No. aberrations</b></td> </tr> </thead> <tbody> <tr> <td align="left"><b>Band</b></td> <td align="left"><b>EED1</b></td> <td align="left"><b>EED2</b></td> <td align="left"><b>EED3</b></td> <td align="left"><b>EED4</b></td> </tr> <tr> <td colspan="5"> <hr /> </td> </tr> <tr> <td align="left">1p22</td> <td align="left">8</td> <td align="left">0</td> <td align="left">0</td> <td align="left">0</td> </tr> <tr> <td align="left">1q11</td> <td align="left">4</td> <td align="left">0</td> <td align="left">0</td> <td align="left">0</td> </tr> <tr> <td align="left">1q15</td> <td align="left">0</td> <td align="left">0</td> <td align="left">5</td> <td align="left">0</td> </tr> <tr> <td align="left">1q16</td> <td align="left">0</td> <td align="left">0</td> <td align="left">3</td> <td align="left">7</td> </tr> <tr> <td align="left">1q21</td> <td align="left">0</td> <td align="left">7</td> <td align="left">16</td> <td align="left">5</td> </tr> <tr> <td align="left">1q31</td> <td align="left">0</td> <td align="left">0</td> <td align="left">8</td> <td align="left">5</td> </tr> <tr> <td align="left">1q33</td> <td align="left">0</td> <td align="left">0</td> <td align="left">16</td> <td align="left">3</td> </tr> <tr> <td align="left">2p16</td> <td align="left">12</td> <td align="left">8</td> <td align="left">7</td> <td align="left">3</td> </tr> <tr> <td align="left">2p13</td> <td align="left">5</td> <td align="left">6</td> <td align="left">0</td> <td align="left">0</td> </tr> <tr> <td align="left">2q13</td> <td align="left">5</td> <td align="left">0</td> <td align="left">3</td> <td align="left">0</td> </tr> <tr> <td align="left">2q26</td> <td align="left">0</td> <td align="left">4</td> <td align="left">3</td> <td align="left">3</td> </tr> <tr> <td align="left">2q31</td> <td align="left">0</td> <td align="left">0</td> <td align="left">3</td> <td align="left">3</td> </tr> <tr> <td align="left">3p26</td> <td align="left">9</td> <td align="left">0</td> <td align="left">4</td> <td align="left">0</td> </tr> <tr> <td align="left">3q26</td> <td align="left">43</td> <td align="left">27</td> <td align="left">11</td> <td align="left">19</td> </tr> <tr> <td align="left">3q28</td> <td align="left">0</td> <td align="left">0</td> <td align="left">17</td> <td align="left">0</td> </tr> <tr> <td align="left">4p11</td> <td align="left">6</td> <td align="left">5</td> <td align="left">0</td> <td align="left">0</td> </tr> <tr> <td align="left">4q11</td> <td align="left">4</td> <td align="left">0</td> <td align="left">0</td> <td align="left">0</td> </tr> <tr> <td align="left">4q18</td> <td align="left">6</td> <td align="left">0</td> <td align="left">5</td> <td align="left">5</td> </tr> <tr> <td align="left">4q24</td> <td align="left">6</td> <td align="left">0</td> <td align="left">0</td> <td align="left">0</td> </tr> <tr> <td align="left">4q26</td> <td align="left">5</td> <td align="left">5</td> <td align="left">0</td> <td align="left">3</td> </tr> <tr> <td align="left">5p12</td> <td align="left">0</td> <td align="left">6</td> <td align="left">4</td> <td align="left">0</td> </tr> <tr> <td align="left">5q25</td> <td align="left">41</td> <td align="left">60</td> <td align="left">18</td> <td align="left">20</td> </tr> <tr> <td align="left">6q16</td> <td align="left">5</td> <td align="left">0</td> <td align="left">0</td> <td align="left">0</td> </tr> <tr> <td align="left">7q12</td> <td align="left">0</td> <td align="left">26</td> <td align="left">4</td> <td align="left">0</td> </tr> <tr> <td align="left">7q14</td> <td align="left">4</td> <td align="left">5</td> <td align="left">0</td> <td align="left">0</td> </tr> <tr> <td align="left">8q11</td> <td align="left">0</td> <td align="left">0</td> <td align="left">0</td> <td align="left">4</td> </tr> <tr> <td align="left">9p12</td> <td align="left">0</td> <td align="left">0</td> <td align="left">0</td> <td align="left">9</td> </tr> <tr> <td align="left">9p11</td> <td align="left">0</td> <td align="left">5</td> <td align="left">3</td> <td align="left">0</td> </tr> <tr> <td align="left">9q14</td> <td align="left">0</td> <td align="left">0</td> <td align="left">0</td> <td align="left">3</td> </tr> <tr> <td align="left">10q12</td> <td align="left">0</td> <td align="left">0</td> <td align="left">5</td> <td align="left">0</td> </tr> <tr> <td align="left">11p12</td> <td align="left">4</td> <td align="left">4</td> <td align="left">0</td> <td align="left">0</td> </tr> <tr> <td align="left">Xp13</td> <td align="left">5</td> <td align="left">6</td> <td align="left">4</td> <td align="left">0</td> </tr> <tr> <td align="left">Xp11</td> <td align="left">0</td> <td align="left">6</td> <td align="left">12</td> <td align="left">0</td> </tr> <tr> <td align="left">Xq13</td> <td align="left">10</td> <td align="left">7</td> <td align="left">0</td> <td align="left">0</td> </tr> <tr> <td align="left">Xq15</td> <td align="left">0</td> <td align="left">12</td> <td align="left">0</td> <td align="left">0</td> </tr> <tr> <td colspan="5"> <hr /> </td> </tr> <tr> <td align="left">total</td> <td align="left">163</td> <td align="left">159</td> <td align="left">127</td> <td align="left">92</td> </tr> </tbody> </table> <div class="table-wrap-foot_UNKNOWN"> <p>The number of chromosomal aberrations (chromosomal breaks, chromosomal gaps, chromatid breaks and chromatid gaps) observed in each specimen (EED1, EED2, EED3 and EED4) at each fragile band is provided.</p> </div> </div> </div> </div> <div class="distributionofevolutionarybreakpointsandconservationoffragilesites" title="sec"> <div class="title" tagx="title" title="title">
Distribution of evolutionary breakpoints and conservation of fragile sites</div> <p>We plotted all human chromosomal homologies previously described in [<a href="#B53">53</a>] and [<a href="#B24">24</a>] onto the ideogram of each of the afrotherian species studied so as to identify bands that delimit human syntenic blocks. Using this approach we identified a set of evolutionary chromosomal bands that correspond to junctions defining human chromosomal syntenies in the three afrotherian species studied herein. These are: (a) Aardvark – 1p11, 1q18, 1q28, 1q35, 2q31, 2q34, 3q29, 4q15, 4q24, 5q21, 6q22 and 6q23, (b) Golden mole – 1q18, 1q21, 2q21, 2q22, 3q28, 4p14, 4q12, 4q22, 6q12, 9q14, 11p13 and 11p12 and (c) Cape rock elephant shrew – 1p26, 1q15, 1q21, 2q17, 2q19, 4q14, 4q24, 5q12, 5q22, 7q12, 10q12, 11p12 and 11q12 (see figure <a href="#F1">1</a>). Additionally, we were able to plot the African elephant/aardvark chromosomal syntenies described in [<a href="#B54">54</a>] that are based on reciprocal painting of these two species thereby providing insights into the association between evolutionary breakpoints and fragile sites among these two species compared to the older, phylogenetically more distant human/Afrotheria evolutionary comparison. Nineteen evolutionary chromosomal bands were detected when comparing the aardvark and African elephant genomes: 1p11, 1q15, 1q18, 1q24, 1q26, 1q28, 1q35, 1q38, 2q13, 2q15, 2q21, 2q25, 2q27, 2q31, 3q12, 3q15, 4p11, 5q14 and 6q23 (figure <a href="#F1">1a</a>). On combining these data (human and elephant chromosomal syntenies) 25 distinct evolutionary breakpoints could be defined in the aardvark; of these, six are common to both data sets (1p11, 1q18, 1q28, 1q35, 2q31 and 6q23).</p> <p>Given these findings, we proceeded to determine if there is a correlation between the position of evolutionary breakpoints and the location of fragile sites in each of the afrotherian species studied using contingency analysis. Of the 12 evolutionary breakpoints identified in the aardvark by reciprocal painting with human painting probes [<a href="#B53">53</a>], seven are coincidental with regions of fragility as defined by fragile site location (1.8 bands expected if the distribution was random, p = 0.0004) (figure <a href="#F1">1a</a>). Of the 19 evolutionary breakpoints identified in the aardvark by reciprocal painting with the African elephant painting probes [<a href="#B54">54</a>], eight are coincidental with regions of fragility (three bands are expected if the distribution was random, p = 0.0032) (figure <a href="#F1">1a</a>). It is noteworthy that of the six bands (1p11, 1q18, 1q28, 1q35, 2q31 and 6q23) that delimit human and elephant chromosomal syntenic blocks in the aardvark genome (see above), all but one (1q35) express fragile sites. Additionally, 27 fragile sites were expressed in the golden mole of which four (1.6 expected, p = 0.07) show correspondence with the 12 evolutionary breakpoints detected by chromosome painting (figure <a href="#F1">1b</a>). Of the 35 aphidicolin induced fragile sites in the elephant shrew, six (2.5 expected, p = 0.02) (figure <a href="#F1">1c</a>) were coincidental with the 13 evolutionary breakpoints previously identified in this species using human chromosome painting probes [<a href="#B24">24</a>].</p> <p>We conducted a more refined analysis of the afrotherian fragile sites by comparing those that are (i) expressed in a single species (i.e., species-specific fragile sites), and (ii) those fragile sites conserved between two or more species (i.e., conserved fragile sites). As above, we assessed each category of fragile site for correspondence with evolutionary breakpoints. Our aim was to test if conserved fragile sites, which are more likely to be ancient fragile sites, might show an enrichment of evolutionary breakpoints. This was borne out by the data which show that of the 12 evolutionary breakpoints identified in the aardvark by reciprocal painting with human painting probes [<a href="#B53">53</a>], three are coincidental with aardvark-specific fragile sites (1.15 bands are expected if the distribution was random, p = 0.001) and four are coincidental with conserved fragile sites (0.7 bands are expected if the distribution was random, p = 0.001). Similarly, a significant association was found in the elephant shrew (p = 0.03). However, in the case of the golden mole, the tendency was not significant. These data suggest, therefore, that evolutionary breakpoints tend to concentrate more frequently in conserved fragile sites than in those that are species-specific, although only significantly so in two of the three species analysed. Finally, three conserved fragile sites were shared between all three afrotherian species (located in bands 1q28, 1q44 and 3q24 in the aardvark, bands 1q28, 1q32 and 3q24 in the golden mole, and bands 1p22, 1q15 and 1q32 in the elephant shrew). One of these (corresponding to band 1q28 in the aardvark, 1q18 in the golden mole and 1q15 in the elephant shrew) was coincidental with the site of an evolutionary breakpoint in all three species – that corresponding to HSA 2/8, the only chromosomal synteny which strongly supports the Afroinsectiphillia (golden moles, elephant shrews and aardvark) to the exclusion of the elephant [<a href="#B24">24</a>]).</p> </div> </div> <div class="discussion" title="sec"> <div class="title" tagx="title" title="title">
Discussion</div> <div class="fragilesitesandchromosomalevolution" title="sec"> <div class="title" tagx="title" title="title">
Fragile sites and chromosomal evolution</div> <p>This investigation confirms and extends earlier observations that fragile sites form part of the chromosomal structure in mammals, and that the characteristics underlying their susceptibility to breakage have been conserved during evolution [<a href="#B26">26</a>,<a href="#B36">36</a>,<a href="#B37">37</a>,<a href="#B51">51</a>,<a href="#B55">55</a>,<a href="#B56">56</a>]. Using data from fragile site expression, G-banding analysis, and cross-species chromosome painting, we have identified fragile sites in aardvark, golden mole and elephant-shrew (Afroinsectiphillia) that are located in homologous chromosomal positions in these species. We detected 11 conserved fragile sites in aardvark genome, eight in golden mole, and 10 fragile sites in the elephant-shrew (figure <a href="#F1">1</a>). Fragile sites detected in more than one species were regarded as "conserved fragile sites" in order to distinguish them from those that were species-specific.</p> <p>Although fragile sites have been considered "hot spots" for evolutionary reorganization in a variety of mammalian species (i.e. are regions where chromosomal rearrangements such as fusion/fissions and inversions can repeatedly occur), the data are limited to a single clade, Boreoeutheria [<a href="#B26">26</a>-<a href="#B28">28</a>]. This begs the question whether this fragility is a more general phenomenon in mammals, and whether the evolutionary breakpoints that characterise human syntenies detected in the basal Afrotheria similarly correspond at the chromosomal band level with fragile sites detected in other species when using a chemical challenge. Twenty nine fragile sites were detected in aardvark, 27 in golden mole and 35 in the elephant-shrew (Figure <a href="#F1">1</a>). A contingency analysis shows that there is a significant association for bands that contain evolutionary breakpoints to accumulate fragile sites in the aardvark (p = 0.0004), as well as in the Cape rock elephant shrew (p = 0.02) genomes. The association was not statistically significant in the golden mole, but there is, nonetheless, a tendency for bands that contain evolutionary breakpoints to accumulate fragile sites in this species (p = 0.07).</p> <p>The inclusion of the elephant/aardvark chromosome painting data into the analysis offered an opportunity to compare old (aardvark vs. human) and phylogenetically younger evolutionary breakpoints (aardvark vs. elephant) and their correlation with chromosomal fragility. We reasoned that younger breakpoints may show a greater correlation with afrotherian fragile sites than their more ancient counterparts identified in the human vs. afrotherian comparisons. The analysis, however, did not reveal marked differences between the two types of evolutionary breakpoints. When the human chromosomal homologies are plotted against the aardvark genome, seven of the 12 evolutionary breakpoints co-localize with fragile sites (1.8 expected, p = 0.0004). Plotting the elephant chromosomal homologies to the aardvark genome revealed that eight of 19 evolutionary breakpoints co-localize with fragile sites (3 expected, p = 0.0032) indicating that the mechanism causing the fragility is conserved. Given these findings, we then proceeded to determine whether the data inform previous conclusions on chromosomal syntenies thought to underpin the recognition of Afrotheria as one of the four major supraordinal clades of placental mammals (Placentalia).</p> </div> <div class="chromosomalsignaturesinafrotheria" title="sec"> <div class="title" tagx="title" title="title">
Chromosomal signatures in Afrotheria</div> <p>The recognition of a monophyletic afrotherian clade was initially based on DNA sequence comparisons [<a href="#B13">13</a>,<a href="#B57">57</a>] and subsequently supported by the analysis of large concatenations of nuclear and mitochondrial genes [<a href="#B1">1</a>,<a href="#B2">2</a>,<a href="#B4">4</a>,<a href="#B5">5</a>,<a href="#B16">16</a>,<a href="#B58">58</a>,<a href="#B59">59</a>], unique insertion and deletion events (indels) [<a href="#B8">8</a>,<a href="#B18">18</a>,<a href="#B19">19</a>], comparative cytogenetic studies [<a href="#B23">23</a>,<a href="#B24">24</a>,<a href="#B60">60</a>], morphology [<a href="#B61">61</a>,<a href="#B62">62</a>], placentation [<a href="#B63">63</a>] and, most recently, by whole genome assemblies [<a href="#B11">11</a>]. Given that Afrotheria is thought to be near the root of the eutherian tree, the organization of their genomes could provide unique insights into the dynamics of mammalian chromosomal evolution.</p> <p>Two human syntenies have been proposed by chromosome painting studies to link afrotherians to the exclusion of other placental mammals (1/19p [<a href="#B23">23</a>,<a href="#B53">53</a>] and 3/21/5 [<a href="#B24">24</a>]). In terms of the former, it is noteworthy that HSA 1/19 has been reported in a xenarthran species (<i>Tamandua tetradactyla</i>, [<a href="#B64">64</a>,<a href="#B65">65</a>]), the insectivore shrew-hedgehog (<i>Neotetracus sinensis</i>, [<a href="#B66">66</a>]), the pig [<a href="#B67">67</a>], as well as in the prosimians <i>Galago moholi, Otolemur crassicaudatus </i>and <i>Nycticebus coucang </i>[<a href="#B68">68</a>,<a href="#B69">69</a>], the New World monkeys <i>Saimiri sciureus </i>[<a href="#B70">70</a>] and <i>Callicebus lugens </i>[<a href="#B71">71</a>], and the Old World monkeys <i>Presbytis cristata </i>[<a href="#B72">72</a>], <i>Pygathrix nemaeus </i>[<a href="#B73">73</a>], <i>Nasalis larvatus </i>[<a href="#B74">74</a>], <i>Trachypithecus francoisi </i>[<a href="#B74">74</a>] and <i>T. phayrei </i>[<a href="#B74">74</a>]. However, only three of these studies relied on reciprocal chromosomal painting [<a href="#B23">23</a>,<a href="#B53">53</a>,<a href="#B69">69</a>] with a fourth [<a href="#B67">67</a>] based on unidirectional painting, but complemented by comparative gene mapping (figure <a href="#F2">2</a>). These data are a prerequisite for allowing unequivocal identification of the chromosomal arms (either 19p or 19q) involved in the 1/19 syntenies. Of these, <i>G. moholi </i>and <i>N. coucang </i>[<a href="#B69">69</a>] show an HSA 1q/19q association and the pig HSA 1p/19q [<a href="#B67">67</a>]. In contrast, the African elephant and the aardvark share HSA 1/19p [<a href="#B23">23</a>,<a href="#B53">53</a>] begging more detailed analysis of whether 1/19p is truly an afrotherian specific chromosomal signature (figure <a href="#F2">2</a>).</p> <div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/"><span class="label" tagx="label" title="label">Figure 2</span></a></div>   <p><b>Phylogenetic tree showing the HSA 1/19 chromosomal syntenies in different mammalian species</b>. In this scheme we place Afrotheria at the root. There are other competing hypotheses for the basal resolution of Placentalia (see text for details). The ancestral chromosomal forms corresponding to HSA 1, HSA 19p and HSA 19q/16q are represented as single conserved entities in both the therian (Marsupialia + Placentalia) and boreoeutherian ancestors. Data based on chromosomal painting in the African elephant (Afrotheria, [23]), the anteater (Xenarthra, [64, 65]), loris (Primates, [69]), the shrew-hedgehog (Eulipotyphla, [66]) and the pig (Cetartiodactyla, [67]) are represented. The human/opossum homologies are determined from ENSEMBL genome sequence alignments [82]. Question marks indicate instances of ambiguity where reciprocal chromosomal painting has not been performed and therefore unequivocal identification of chromosomal arms is not possible. Inset shows G-banding comparisons between elephant chromosome 2 (LAF 2) and human chromosomes 1 and 19 (HSA 1 and HSA 19); syntenic boundaries are derived from reciprocal chromosome painting data [23]. Note that HSA 1 is inverted to facilitate comparisons with LAF 2. Centromeres are marked by asterisks.</p>   </div> <p>By computationally scanning the genomic assemblies of human and opossum (a marsupial outgroup to Afrotheria and other Placentalia) we sought to validate the HSA 1/19p synteny as a lineage specific, rare genomic change underpinning the monophyly of Afrotheria. In attempting to address this it is important to point out that the defining character in a conserved segmental association is the presence of the breakpoint (i.e. the junctions 1/19p and 3/21/5) since, as has been noted elsewhere [<a href="#B28">28</a>,<a href="#B60">60</a>], the size of segments may be altered by subsequent translocations to other regions in the genome, and FISH provides no insight to gene order within the syntenic block which may be altered by intrachromosomal rearrangement. Furthermore, in deciding the most parsimonious pathway to derive a specific chromosomal rearrangement we follow [<a href="#B22">22</a>] in viewing the independent disruption of a syntenic group to be more likely than the same adjacent synteny being independently reassembled in different lineages. While the opossum genome shows a HSA 1p/19q/1p/19q/1p/19q/1p association on its chromosome 4 (figure <a href="#F3">3</a>), this adjacent synteny is different to the HSA 1q/19q found in the prosimian species <i>G. moholi </i>and <i>N. coucang </i>([<a href="#B69">69</a>] and figure <a href="#F2">2</a>) further reinforcing the finding that HSA 1, reportedly the largest physical unit in the eutherian ancestral genome, has suffered multiple independent fissions [<a href="#B75">75</a>].</p> <div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/"><span class="label" tagx="label" title="label">Figure 3</span></a></div>   <p><b>Representation of the syntenies HSA 3/21/X and HSA 1/19 in the chicken and opossum genomes</b>. Different colours show contiguous synteny blocks between (a) chicken and human, and (b) opossum and human as determined from ENSEMBL genome sequence alignments [82].</p>   </div> <p>Importantly, however, and of substantial phylogenetic significance, the human chromosomal segment involved in the HSA 1/19p afrotherian synteny is currently ambiguous since the painting data do not allow inference on whether the junction is between HSA 1q or HSA 1p in those species for which there are reciprocal painting data (i.e., elephant and aardvark [<a href="#B23">23</a>,<a href="#B53">53</a>]). G-banding homology on the other hand favours HSA 1p/19p (see insert figure <a href="#F2">2</a>). If correct, this association would support the monophyly of Afrotheria [<a href="#B23">23</a>,<a href="#B53">53</a>] on current information. However, should further analysis reveal its presence in Xenarthra, this would give credence to the recognition of Atlantogenata [<a href="#B76">76</a>], a clade containing Afrotheria and Xenarthra to the exclusion of Boreoeutheria [<a href="#B11">11</a>]. Both outcomes underscore the importance of resolving this critical chromosomal synteny for clarifying deep divergences in the eutherian tree although recent strong support for a sister group relationship for Afrotheria and Xenarthra (based on a ~2.2 mega-base data set of protein coding sequences), clearly tips the odds in favour of HSA 1(p?)/19p being a shared synteny for Atlantogenata [<a href="#B12">12</a>].</p> <p>Moreover, it is of interest that the human syntenies HSA 1, HSA 16q/19q and HSA 19p have all previously been proposed for the boreoeutherian ancestor [<a href="#B22">22</a>,<a href="#B28">28</a>,<a href="#B77">77</a>] which, if present in the eutherian ancestor, would require a fusion (possibly promoted by an ancient fragile site retained in aardvark OAF 3q29 – figure <a href="#F1">1a</a>) to derive the HSA 1/19p synteny that possibly unites Afrotheria to the exclusion of other Placentalia (figure <a href="#F2">2</a>). If the same syntenies were present at the therian root all that would be required is a breakpoint in HSA19q with a fusion to HSA1 to give the opossum HSA 1p/19q and HSA1 6q/19q combinations (figure <a href="#F2">2</a>).</p> <p>Using a similar approach we examined the second synteny thought to underpin Afrotheria monophyly, HSA 3/21/5. It was previously argued [<a href="#B24">24</a>] that the ancestral association HSA 3/21 [<a href="#B53">53</a>] should be expanded to include segments homologous to human chromosome 5 forming an HSA 3/21/5 segmental combination defining Afrotheria. The authors posit that the most parsimonious explanation for the observed patterns is that HSA 21 appears to have fissioned within Paenungulata; in this regard it is noteworthy that 2q31 is expressed as a fragile site in aardvark (figure <a href="#F1">1a</a>), perhaps indicating an ancestral locus. Their reasoning was that the HSA 3/21/5 configuration is present in aardvark, golden mole and elephant-shrew with all three chromosomes retained as intact, conserved entities in the two former species. In the case of the elephant, the fissioning of HSA 21 gave rise to HSA 5/21 (LAF 3) and HSA 1/3/21/3 on LAF 21 (see [<a href="#B23">23</a>] and inset in figure <a href="#F4">4</a>). This rearrangement (the modification of 3/21/5 to HSA 3/21 + HSA 5/21) was recently confirmed [<a href="#B54">54</a>] in manatee and elephant (data on the hyrax are incomplete) by reciprocal painting with paenungulate species-specific painting probes, and through inferences made from human and aardvark [<a href="#B53">53</a>]. New information on the Florida manatee (<i>Trichechus manatus latirostris</i>, TMA, [<a href="#B78">78</a>]) resulting from unidirectional painting experiments with human probes similarly show HSA 5/21 on TMA 1, and HSA 2/3/21 on TMA 15.</p> <div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/"><span class="label" tagx="label" title="label">Figure 4</span></a></div>   <p><b>Phylogenetic tree showing the HSA 3/21 chromosomal syntenies in different mammalian species</b>. In this scheme we place Afrotheria at the root. There are other competing hypotheses for the basal resolution of Placentalia (see text for details). The ancestral chromosomal forms corresponding to HSA 3/21 are represented as single conserved entities in the amniote, therian, eutherian and boreoeutherian ancestors. Source references for the species shown in the tree are: African elephant [23], domestic pig [84], rabbit [85], tree shrew [86], grey squirrel [87], domestic cat [88], aardvark [53], the xenarthran taxa the amadillo and lesser anteater [64], and the ENSEMBL genome database [82] for the rat, mouse, cattle, dog, opossum and chicken sequence alignments. The asterisks indicate the unequivocal identification of the HSA 3p/21 synteny based on reciprocal chromosome painting or data from the ENSEMBL genome database [82]. The estimated divergence dates follow [11]. The opossum and the chicken are included as representative outgroups species. Inset shows G-banding comparisons between elephant chromosomes 3 (LAF 3) and 21 (LAF 21) and aardvark chromosome 2 (OAF 2) showing the regions of homology to human chromosomes 3, 5 and 21 (HSA 3, HSA 5 and HSA 21); syntenic boundaries are derived from reciprocal chromosome painting data [23].</p>   </div> <p>The widely accepted boreutherian ancestral syntenies (HSA 4/8/4, 7a/16p, 10/12a/22a, 12b/22b, 14/15 and 16q/19q, [<a href="#B22">22</a>,<a href="#B77">77</a>]) are all in the opossum genome suggesting their presence in a therian ancestor, and retention for &amp;gt;180 mya (divergence based on stem branches between crown placentals and crown marsupials, [<a href="#B79">79</a>-<a href="#B81">81</a>]). One further ancestral synteny (HSA 3/21) deserves special comment, especially with respect to its importance for Afrotheria. Froenicke [<a href="#B22">22</a>] provided evidence that neighbouring segments homologous to HSA 3 and HSA 21 have been found in all eutherian orders for which there is information, and that the combined analysis of reciprocal chromosome painting data in conjunction with draft genome sequence information for mouse and human showed that the breakpoint is located in HSA 3p, the region closest to the centromere of this chromosome. While there is no evidence of this synteny in opossum, electronic screening of the chicken genome assembly indicates its retention in this species (the HSA 3p segment extends from 76Mb-90 Mb, [<a href="#B82">82</a>], see figure <a href="#F3">3</a>). Opossum has a 21/Xp/3q/Xp/3q synteny in chromosome 4 and, importantly, 3p/3q/Xp in chromosome 7 indicating different breakpoints in Marsupialia (figure <a href="#F4">4</a>). In summary therefore, it would seem that HSA 3p/21 was present in the common ancestor of Amniota (i.e. common to turtles, lepidosaurs, crocodilians, birds and mammals) supporting its identification as an ancestral synteny that was present &amp;gt;310 MYA [<a href="#B79">79</a>], but which was disrupted in the lineage leading to the marsupials. The expansion to include HSA 5 in the afrotherian ancestor is unique [<a href="#B24">24</a>] and defines the monophyly of this ancient African clade of mammals.</p> </div> </div> <div class="conclusion" title="sec"> <div class="title" tagx="title" title="title">
Conclusion</div> <p>Using data from fragile site expression, G-banding analysis, and cross-species chromosome painting, we have described a suite of afrotherian common fragile sites that are correlated with evolutionary breakpoints when compared to the human genome. By computationally scanning the newly released opossum and chicken genomes as outgroups to Placentalia, we have shown that the primitive HSA 3p/21 synteny was present in the amniote ancestor, and its expansion to include HSA 5 validates the HSA 3/21/5 synteny as a robust cytogenetic signature that defines the monophyly of Afrotheria. Its fission into two segments (HSA 3p/21 + HSA 5/21) is probably shared by all Paenungulata and may have been facilitated by an ancient fragile site that is still expressed in aardvark. Further, if the human syntenies HSA 1, HSA 16q/19q and HSA 19p (all previously proposed for the boreoeutherian ancestor) were present at the eutherian root, a single fusion (the breakpoint junction being coincidental with a fragile site retained in aardvark at OAF 3q29) would be required to derive the HSA 1/19p synteny that may, with further refined analysis, be found to unite Afrotheria to the exclusion of other Placentalia.</p> </div> <div class="methods" title="methods"> <div class="title" tagx="title" title="title">
Methods</div> <div class="cellcultureandfragilesiteexpression" title="sec"> <div class="title" tagx="title" title="title">
Cell culture and fragile site expression</div> <p>Fibroblast cell cultures were established from two aardvark (one male and one female), three golden mole (two males and one female) and four elephant-shrew specimens (two females and two males). Twenty four hours before harvesting 50 μl of aphidicolin (APC, 2 mM) dissolved in DMSO was added to cell cultures at a final concentration of 0.2 μM. Concurrent control cultures were established for each experiment. Cells were harvested and chromosomal preparations obtained using standard protocols.</p> <p>Ideograms were constructed for each of the species according to the standardised karyotype for <i>O. afer </i>(2n = 20, OAF [<a href="#B53">53</a>]), <i>C. asiatica </i>(2n = 30, CAS [<a href="#B24">24</a>]) and <i>E. edwardii </i>(2n = 28, EED [<a href="#B24">24</a>]). (Note that the karyotype presented in [<a href="#B24">24</a>] was originally incorrectly identified as <i>E. rupestris</i>; DNA from the same specimen was subsequently extracted and sequenced, verifying its status as <i>E. edwardii</i>; unpublished data). Our banding data show that 177 chromosomal bands define the <i>O. afer </i>ideogram, whereas the <i>C. asiatica </i>and <i>E. edwardii </i>ideograms have 200 and 182 bands, respectively.</p> <p>All metaphases were sequentially solid-stained and then G-banded to establish the location of the breakpoints. Digital images were captured with a BX60 Olympus microscope utilizing the GENUS imaging System (version 2.75) (Applied Imaging Corporation).</p> </div> <div class="analysisoffragilesites" title="sec"> <div class="title" tagx="title" title="title">
Analysis of fragile sites</div> <p>In order to identify which chromosomal bands could be considered fragile regions, a statistical analysis of the distribution of chromosomal abnormalities detected in each specimen analysed was undertaken using the programme FSM (version 995, [<a href="#B83">83</a>]). In this program, the standardized χ<sup>2 </sup>and G<sup>2 </sup>test statistics are used for all chromosomal bands that express non-random breaks or gaps. The hypothesis tested by the programme is that the probability of breakage is equal for all chromosomal bands in a given karyotype using a 0.05 level of significance (α). The FSM analysis gives a critical value for each data set analysed, abbreviated to C<sub>α</sub>, which indicates the lowest frequency of breakage per band that exceeds the level of significance [<a href="#B35">35</a>,<a href="#B83">83</a>]. Any chromosomal band with a number of breaks greater than the critical value is considered a fragile site. This value ranged from 3 to 4 depending on the number of chromosomal bands determined in each karyotype (177 bands for the aardvark, 200 bands for the golden mole and 182 bands for the elephant shrew), and the number of breaks detected in each species' data set. In our analyses, chromatid and chromosomal breaks and gaps were treated equally as representing single chromosomal events, and these were mapped to the respective ideogram of each afrotherian species.</p> </div> <div class="computationalanalysisofafrotherianandboreoeutherianlineage-specificsyntenies" title="sec"> <div class="title" tagx="title" title="title">
Computational analysis of afrotherian and boreoeutherian lineage-specific syntenies</div> <p>Data from different studies [<a href="#B24">24</a>,<a href="#B53">53</a>,<a href="#B54">54</a>,<a href="#B78">78</a>] were used as sources for determining homologies between the human genome and those of the aardvark, golden mole and Cape rock elephant shrew. Evolutionary breakpoints were defined as the limit between each adjacent human homologous region. Evolutionary bands are chromosomal bands that contain evolutionary breakpoints (see [<a href="#B27">27</a>] for further clarification of evolutionary breakpoints and evolutionary bands). We omitted the centromeres from the analysis and plotted all fragile sites and evolutionary breakpoints onto the ideograms of each of the afrotherian species analysed. A contingency analysis was used (JMP package version 5.1.2; SAS Institute Inc.) to evaluate if evolutionary breakpoints concentrate significantly (p ≤ 0.05) in chromosomal bands containing fragile sites. This is based on the assumption that the chromosomal bands of the genomes of each afrotherian species analysed have the same probability to be affected, independently of the size of the band involved.</p> <p>The Ensembl genome browser of the Sanger Center and EMBL data base [<a href="#B82">82</a>] were used for determining homologies between the human genome and those of the opossum and chicken. We used the completed human/chicken (WASHUC 1) and human/opossum (MonDom 4.0) whole-genome sequence assemblies that are available on the Ensembl genome browser [<a href="#B82">82</a>] to determine syntenic regions between the human genome (NCBI Build 36) and those of opossum and chicken</p> </div> </div> <div class="competinginterests" title="sec"> <div class="title" tagx="title" title="title">
Competing interests</div> <p>The author(s) declares that there are no competing interests.</p> </div> <div class="authors'contributions" title="sec"> <div class="title" tagx="title" title="title">
Authors' contributions</div> <p>ARH and TR conceived and designed the experiments. ARH performed the experiments and analysed the fragile site data. Both authors scrutinised evolutionary breakpoint for use in a phylogenetic context, and contributed equally to writing the manuscript.</p> </div> </div> <div class="back" title="back"> <div class="ack" title="ack"> <div class="acknowledgements" title="sec"> <div class="title" tagx="title" title="title">
Acknowledgements</div> <p>Financial support to TJR from the National Research Foundation, South Africa and a postdoctoral fellowship from the Spanish Ministry of Education and Science (MEC) to ARH are gratefully acknowledged. The authors thank Lutz Froenicke for providing insightful comment on an earlier draft of this manuscript.</p> </div> </div> <div class="references">
References</div> <div tag="ref-list"> <ul> <li tag="ref"><a name="B1" /><div class="citation" tagx="citation" title="citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Murphy</span><span class="given-names" tagx="given-names" title="given-names">WJ</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Eizirik</span><span class="given-names" tagx="given-names" title="given-names">E</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Jonson</span><span class="given-names" tagx="given-names" title="given-names">WE</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zhang</span><span class="given-names" tagx="given-names" title="given-names">YP</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ryder</span><span class="given-names" tagx="given-names" title="given-names">OA</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">O'Brien</span><span class="given-names" tagx="given-names" title="given-names">SJ</span></span></span><span class="article-title" tagx="article-title" title="article-title">Molecular phylogenetics and the origins of placental mammals</span><span class="source" tagx="source" title="source">Nature</span><span class="year" tagx="year" title="year">2001</span><span class="volume" tagx="volume" title="volume">409</span><span class="fpage" tagx="fpage" title="fpage">614</span><span class="lpage" tagx="lpage" title="lpage">618</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11214319">11214319</a></span><span class="pub-id"><a href="https://dx.doi.org/10.1038/35054550">10.1038/35054550</a></span></div> </li> <li tag="ref"><a name="B2" /><div class="citation" tagx="citation" title="citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Murphy</span><span class="given-names" tagx="given-names" title="given-names">WJ</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Eizirik</span><span class="given-names" tagx="given-names" title="given-names">E</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">O'Brien</span><span class="given-names" tagx="given-names" title="given-names">SJ</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Madsen</span><span class="given-names" tagx="given-names" title="given-names">O</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Scally</span><span class="given-names" tagx="given-names" title="given-names">M</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Douady</span><span class="given-names" tagx="given-names" title="given-names">CJ</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Teeling</span><span class="given-names" tagx="given-names" title="given-names">E</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ryder</span><span class="given-names" tagx="given-names" title="given-names">OA</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Stanhope</span><span class="given-names" tagx="given-names" title="given-names">MJ</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">de Jong</span><span class="given-names" tagx="given-names" title="given-names">WW</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Springer</span><span class="given-names" tagx="given-names" title="given-names">MS</span></span></span><span class="article-title" tagx="article-title" title="article-title">Resolution of the early placental mammal radiation using Bayesian phylogenetics</span><span class="source" tagx="source" title="source">Science</span><span class="year" tagx="year" title="year">2001</span><span class="volume" tagx="volume" title="volume">294</span><span class="fpage" tagx="fpage" title="fpage">2348</span><span class="lpage" tagx="lpage" title="lpage">2351</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11743200">11743200</a></span><span class="pub-id"><a href="https://dx.doi.org/10.1126/science.1067179">10.1126/science.1067179</a></span></div> </li> <li tag="ref"><a name="B3" /><div class="citation" tagx="citation" title="citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Waddell</span><span class="given-names" tagx="given-names" title="given-names">PJ</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kishino</span><span class="given-names" tagx="given-names" title="given-names">H</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ota</span><span class="given-names" tagx="given-names" title="given-names">R</span></span></span><span class="article-title" tagx="article-title" title="article-title">A phylogenetic foundation for comparative mammalian genomics</span><span class="source" tagx="source" title="source">Genome Inform</span><span class="year" tagx="year" title="year">2001</span><span class="volume" tagx="volume" title="volume">12</span><span class="fpage" tagx="fpage" title="fpage">141</span><span class="lpage" tagx="lpage" title="lpage">154</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11791233">11791233</a></span></div> </li> <li tag="ref"><a name="B4" /><div class="citation" tagx="citation" title="citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Waddell</span><span class="given-names" tagx="given-names" title="given-names">PJ</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Shelley</span><span class="given-names" tagx="given-names" title="given-names">S</span></span></span><span class="article-title" tagx="article-title" title="article-title">Evaluating placental inter-ordinal phylogenies with novel sequences including RAG1, gamma-fibrinogen, ND6, and mt-tRNA, plus MCMC-driven nucleotide, amino acid, and codon models</span><span class="source" tagx="source" title="source">Mol Phylogenet Evol</span><span class="year" tagx="year" title="year">2003</span><span class="volume" tagx="volume" title="volume">28</span><span class="fpage" tagx="fpage" title="fpage">197</span><span class="lpage" tagx="lpage" title="lpage">224</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12878459">12878459</a></span><span class="pub-id"><a href="https://dx.doi.org/10.1016/S1055-7903(03)00115-5">10.1016/S1055-7903(03)00115-5</a></span></div> </li> <li tag="ref"><a name="B5" /><div class="citation" tagx="citation" title="citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Amrine-Madsen</span><span class="given-names" tagx="given-names" title="given-names">H</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Koepfli</span><span class="given-names" tagx="given-names" title="given-names">K-P</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wayne</span><span class="given-names" tagx="given-names" title="given-names">RK</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Springer</span><span class="given-names" tagx="given-names" title="given-names">MS</span></span></span><span class="article-title" tagx="article-title" title="article-title">A new phylogenetic marker, apolipoprotein B, provides compelling evidence for eutherian relationships</span><span class="source" tagx="source" title="source">Mol Phylogenet Evol</span><span class="year" tagx="year" title="year">2003</span><span class="volume" tagx="volume" title="volume">28</span><span class="fpage" tagx="fpage" title="fpage">225</span><span class="lpage" tagx="lpage" title="lpage">240</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12878460">12878460</a></span><span class="pub-id"><a href="https://dx.doi.org/10.1016/S1055-7903(03)00118-0">10.1016/S1055-7903(03)00118-0</a></span></div> </li> <li tag="ref"><a name="B6" /><div class="citation" tagx="citation" title="citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Nikolaev</span><span class="given-names" tagx="given-names" title="given-names">S</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Montoya-Burgos</span><span class="given-names" tagx="given-names" title="given-names">JI</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Margulies</span><span class="given-names" tagx="given-names" title="given-names">EH</span></span><span class="collab'">collab: NISC Comparative Sequencing Program</span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Rougemont</span><span class="given-names" tagx="given-names" title="given-names">J</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Nyffeler</span><span class="given-names" tagx="given-names" title="given-names">B</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Antonarakis</span><span class="given-names" tagx="given-names" title="given-names">SE</span></span></span><span class="article-title" tagx="article-title" title="article-title">Early History of Mammals Is Elucidated with the ENCODE Multiple Species Sequencing Data</span><span class="source" tagx="source" title="source">PLoS Genet</span><span class="year" tagx="year" title="year">2007</span><span class="volume" tagx="volume" title="volume">3</span><span class="fpage" tagx="fpage" title="fpage">e2</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17206863">17206863</a></span><span class="pub-id"><a href="https://dx.doi.org/10.1371/journal.pgen.0030002">10.1371/journal.pgen.0030002</a></span></div> </li> <li tag="ref"><a name="B7" /><div class="citation" tagx="citation" title="citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Waters</span><span class="given-names" tagx="given-names" title="given-names">PD</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Dobigny</span><span class="given-names" tagx="given-names" title="given-names">G</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Waddell</span><span class="given-names" tagx="given-names" title="given-names">PJ</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Robinson</span><span class="given-names" tagx="given-names" title="given-names">TJ</span></span></span><span class="article-title" tagx="article-title" title="article-title">Evolutionary History of Line-1 in the Major Clades of Placental Mammals. </span><span class="source" tagx="source" title="source">PLoS One</span><span class="year" tagx="year" title="year">2007</span><span class="volume" tagx="volume" title="volume">2</span><span class="fpage" tagx="fpage" title="fpage">e158</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17225861">17225861</a></span><span class="pub-id"><a href="https://dx.doi.org/10.1371/journal.pone.0000158">10.1371/journal.pone.0000158</a></span></div> </li> <li tag="ref"><a name="B8" /><div class="citation" tagx="citation" title="citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kriegs</span><span class="given-names" tagx="given-names" title="given-names">JO</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Churakov</span><span class="given-names" tagx="given-names" title="given-names">G</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kiefmann</span><span class="given-names" tagx="given-names" title="given-names">M</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Jordan</span><span class="given-names" tagx="given-names" title="given-names">U</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Brosius</span><span class="given-names" tagx="given-names" title="given-names">J</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Schmitz</span><span class="given-names" tagx="given-names" title="given-names">J</span></span></span><span class="article-title" tagx="article-title" title="article-title">Retroposed elements as archives for the evolutionary history of placental mammals</span><span class="source" tagx="source" title="source">PLoS Biol</span><span class="year" tagx="year" title="year">2006</span><span class="volume" tagx="volume" title="volume">4</span><span class="fpage" tagx="fpage" title="fpage">e91</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16515367">16515367</a></span><span class="pub-id"><a href="https://dx.doi.org/10.1371/journal.pbio.0040091">10.1371/journal.pbio.0040091</a></span></div> </li> <li tag="ref"><a name="B9" /><div class="citation" tagx="citation" title="citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hunter</span><span class="given-names" tagx="given-names" title="given-names">JP</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Janis</span><span class="given-names" tagx="given-names" title="given-names">CM</span></span></span><span class="article-title" tagx="article-title" title="article-title">Spiny Norman in the Garden of Eden? Dispersal and early biogeography of Placentalia</span><span class="source" tagx="source" title="source">J Mammal Evol</span><span class="year" tagx="year" title="year">2006</span><span class="volume" tagx="volume" title="volume">13</span><span class="fpage" tagx="fpage" title="fpage">89</span><span class="lpage" tagx="lpage" title="lpage">123</span><span class="pub-id"><a href="https://dx.doi.org/10.1007/s10914-006-9006-6">10.1007/s10914-006-9006-6</a></span></div> </li> <li tag="ref"><a name="B10" /><div class="citation" tagx="citation" title="citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">McKenna</span><span class="given-names" tagx="given-names" title="given-names">MC</span></span></span><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Luckett WP, Szalay FS</span></span></span><span class="article-title" tagx="article-title" title="article-title">Toward a phylogenetic classification of the Mammalia</span><span class="source" tagx="source" title="source">Phylogeny of the Primates</span><span class="year" tagx="year" title="year">1975</span><span class="publisher-name" tagx="publisher-name" title="publisher-name">New York: Plenum Press</span><span class="fpage" tagx="fpage" title="fpage">21</span><span class="lpage" tagx="lpage" title="lpage">46</span></div> </li> <li tag="ref"><a name="B11" /><div class="citation" tagx="citation" title="citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Murphy</span><span class="given-names" tagx="given-names" title="given-names">WJ</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Pringle</span><span class="given-names" tagx="given-names" title="given-names">TH</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Crider</span><span class="given-names" tagx="given-names" title="given-names">TA</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Springer</span><span class="given-names" tagx="given-names" title="given-names">MS</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Miller</span><span class="given-names" tagx="given-names" title="given-names">W</span></span></span><span class="article-title" tagx="article-title" title="article-title">Using genomic data to unravel the root of the placental mammal phylogeny</span><span class="source" tagx="source" title="source">Genome Res</span><span class="year" tagx="year" title="year">2007</span><span class="volume" tagx="volume" title="volume">17</span><span class="fpage" tagx="fpage" title="fpage">413</span><span class="lpage" tagx="lpage" title="lpage">421</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17322288">17322288</a></span><span class="pub-id"><a href="https://dx.doi.org/10.1101/gr.5918807">10.1101/gr.5918807</a></span></div> </li> <li tag="ref"><a name="B12" /><div class="citation" tagx="citation" title="citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hallström</span><span class="given-names" tagx="given-names" title="given-names">B</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kullberg</span><span class="given-names" tagx="given-names" title="given-names">M</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Nilsson</span><span class="given-names" tagx="given-names" title="given-names">M</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Janke</span><span class="given-names" tagx="given-names" title="given-names">A</span></span></span><span class="article-title" tagx="article-title" title="article-title">Phylogenomic data analyses provide evidence that Xenarthra and Afrotheria are sistergroups</span><span class="source" tagx="source" title="source">Mol Biol Evol</span><span class="volume" tagx="volume" title="volume">24</span><span class="fpage" tagx="fpage" title="fpage">2059</span><span class="lpage" tagx="lpage" title="lpage">2068</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17630282">17630282</a></span><span class="pub-id"><a href="https://dx.doi.org/10.1093/molbev/msm136">10.1093/molbev/msm136</a></span></div> </li> <li tag="ref"><a name="B13" /><div class="citation" tagx="citation" title="citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Springer</span><span class="given-names" tagx="given-names" title="given-names">MS</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Cleven</span><span class="given-names" tagx="given-names" title="given-names">GC</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Madsen</span><span class="given-names" tagx="given-names" title="given-names">O</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">de Jong</span><span class="given-names" tagx="given-names" title="given-names">WW</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Waddell</span><span class="given-names" tagx="given-names" title="given-names">VG</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Amrine</span><span class="given-names" tagx="given-names" title="given-names">HM</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Stanhope</span><span class="given-names" tagx="given-names" title="given-names">MJ</span></span></span><span class="article-title" tagx="article-title" title="article-title">Endemic African mammals shake the phylogenetic tree</span><span class="source" tagx="source" title="source">Nature</span><span class="year" tagx="year" title="year">1997</span><span class="volume" tagx="volume" title="volume">388</span><span class="fpage" tagx="fpage" title="fpage">61</span><span class="lpage" tagx="lpage" title="lpage">64</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/9214502">9214502</a></span><span class="pub-id"><a href="https://dx.doi.org/10.1038/40386">10.1038/40386</a></span></div> </li> <li tag="ref"><a name="B14" /><div class="citation" tagx="citation" title="citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Rokas</span><span class="given-names" tagx="given-names" title="given-names">A</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Holland</span><span class="given-names" tagx="given-names" title="given-names">PW</span></span></span><span class="article-title" tagx="article-title" title="article-title">Rare genomic changes as a tool for phylogenetics</span><span class="source" tagx="source" title="source">Trends Ecol Evol</span><span class="year" tagx="year" title="year">2000</span><span class="volume" tagx="volume" title="volume">15</span><span class="fpage" tagx="fpage" title="fpage">454</span><span class="lpage" tagx="lpage" title="lpage">459</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11050348">11050348</a></span><span class="pub-id"><a href="https://dx.doi.org/10.1016/S0169-5347(00)01967-4">10.1016/S0169-5347(00)01967-4</a></span></div> </li> <li tag="ref"><a name="B15" /><div class="citation" tagx="citation" title="citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Shedlock</span><span class="given-names" tagx="given-names" title="given-names">AM</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Okada</span><span class="given-names" tagx="given-names" title="given-names">N</span></span></span><span class="article-title" tagx="article-title" title="article-title">SINE insertions: a powerful tools for molecular systematics</span><span class="source" tagx="source" title="source">BioEssays</span><span class="year" tagx="year" title="year">2000</span><span class="volume" tagx="volume" title="volume">22</span><span class="fpage" tagx="fpage" title="fpage">148</span><span class="lpage" tagx="lpage" title="lpage">160</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/10655034">10655034</a></span><span class="pub-id"><a href="https://dx.doi.org/10.1002/(SICI)1521-1878(200002)22:2&lt;148::AID-BIES6&gt;3.0.CO;2-Z">10.1002/(SICI)1521-1878(200002)22:2&amp;lt;148::AID-BIES6&amp;gt;3.0.CO;2-Z</a></span></div> </li> <li tag="ref"><a name="B16" /><div class="citation" tagx="citation" title="citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Madsen</span><span class="given-names" tagx="given-names" title="given-names">O</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Scally</span><span class="given-names" tagx="given-names" title="given-names">M</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Douady</span><span class="given-names" tagx="given-names" title="given-names">CJ</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kao</span><span class="given-names" tagx="given-names" title="given-names">DJ</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">DeBry</span><span class="given-names" tagx="given-names" title="given-names">RW</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Adkins</span><span class="given-names" tagx="given-names" title="given-names">R</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Amrine</span><span class="given-names" tagx="given-names" title="given-names">HM</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Stanhope</span><span class="given-names" tagx="given-names" title="given-names">MJ</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">de Jong</span><span class="given-names" tagx="given-names" title="given-names">WW</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Springer</span><span class="given-names" tagx="given-names" title="given-names">MS</span></span></span><span class="article-title" tagx="article-title" title="article-title">Parallel adaptive radiations in two major clades of placental mammals</span><span class="source" tagx="source" title="source">Nature</span><span class="year" tagx="year" title="year">2001</span><span class="volume" tagx="volume" title="volume">409</span><span class="fpage" tagx="fpage" title="fpage">610</span><span class="lpage" tagx="lpage" title="lpage">614</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11214318">11214318</a></span><span class="pub-id"><a href="https://dx.doi.org/10.1038/35054544">10.1038/35054544</a></span></div> </li> <li tag="ref"><a name="B17" /><div class="citation" tagx="citation" title="citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Nikaido</span><span class="given-names" tagx="given-names" title="given-names">M</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Nishihara</span><span class="given-names" tagx="given-names" title="given-names">H</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hukumoto</span><span class="given-names" tagx="given-names" title="given-names">Y</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Okada</span><span class="given-names" tagx="given-names" title="given-names">N</span></span></span><span class="article-title" tagx="article-title" title="article-title">Ancient SINEs from African endemic mammals</span><span class="source" tagx="source" title="source">Mol Biol Evol</span><span class="year" tagx="year" title="year">2003</span><span class="volume" tagx="volume" title="volume">20</span><span class="fpage" tagx="fpage" title="fpage">522</span><span class="lpage" tagx="lpage" title="lpage">527</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12654932">12654932</a></span><span class="pub-id"><a href="https://dx.doi.org/10.1093/molbev/msg052">10.1093/molbev/msg052</a></span></div> </li> <li tag="ref"><a name="B18" /><div class="citation" tagx="citation" title="citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Nishihara</span><span class="given-names" tagx="given-names" title="given-names">H</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Satta</span><span class="given-names" tagx="given-names" title="given-names">Y</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Nikaido</span><span class="given-names" tagx="given-names" title="given-names">M</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Thewissen</span><span class="given-names" tagx="given-names" title="given-names">JGM</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Stanhope</span><span class="given-names" tagx="given-names" title="given-names">MJ</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Okada</span><span class="given-names" tagx="given-names" title="given-names">N</span></span></span><span class="article-title" tagx="article-title" title="article-title">A Retroposon Analysis of Afrotherian Phylogeny</span><span class="source" tagx="source" title="source">Mol Biol Evol</span><span class="year" tagx="year" title="year">2005</span><span class="volume" tagx="volume" title="volume">22</span><span class="fpage" tagx="fpage" title="fpage">1823</span><span class="lpage" tagx="lpage" title="lpage">1833</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15930154">15930154</a></span><span class="pub-id"><a href="https://dx.doi.org/10.1093/molbev/msi179">10.1093/molbev/msi179</a></span></div> </li> <li tag="ref"><a name="B19" /><div class="citation" tagx="citation" title="citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Nishihara</span><span class="given-names" tagx="given-names" title="given-names">H</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hasegawa</span><span class="given-names" tagx="given-names" title="given-names">M</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Okada</span><span class="given-names" tagx="given-names" title="given-names">N</span></span></span><span class="article-title" tagx="article-title" title="article-title">Pegasoferae, an unexpected mammalian clade revealed by tracking ancient retroposon insertions</span><span class="source" tagx="source" title="source">Proc Natl Acad Sci USA</span><span class="year" tagx="year" title="year">2006</span><span class="volume" tagx="volume" title="volume">103</span><span class="fpage" tagx="fpage" title="fpage">9929</span><span class="lpage" tagx="lpage" title="lpage">9934</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16785431">16785431</a></span><span class="pub-id"><a href="https://dx.doi.org/10.1073/pnas.0603797103">10.1073/pnas.0603797103</a></span></div> </li> <li tag="ref"><a name="B20" /><div class="citation" tagx="citation" title="citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">O'Brien</span><span class="given-names" tagx="given-names" title="given-names">SJ</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Stanyon</span><span class="given-names" tagx="given-names" title="given-names">R</span></span></span><span class="article-title" tagx="article-title" title="article-title">Ancestral primate viewed</span><span class="source" tagx="source" title="source">Nature</span><span class="year" tagx="year" title="year">1999</span><span class="volume" tagx="volume" title="volume">402</span><span class="fpage" tagx="fpage" title="fpage">365</span><span class="lpage" tagx="lpage" title="lpage">366</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/10586870">10586870</a></span><span class="pub-id"><a href="https://dx.doi.org/10.1038/46450">10.1038/46450</a></span></div> </li> <li tag="ref"><a name="B21" /><div class="citation" tagx="citation" title="citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wienberg</span><span class="given-names" tagx="given-names" title="given-names">J</span></span></span><span class="article-title" tagx="article-title" title="article-title">The evolution of eutherian chromosomes</span><span class="source" tagx="source" title="source">Curr Opin Genet Dev</span><span class="year" tagx="year" title="year">2004</span><span class="volume" tagx="volume" title="volume">14</span><span class="fpage" tagx="fpage" title="fpage">657</span><span class="lpage" tagx="lpage" title="lpage">666</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15531161">15531161</a></span><span class="pub-id"><a href="https://dx.doi.org/10.1016/j.gde.2004.10.001">10.1016/j.gde.2004.10.001</a></span></div> </li> <li tag="ref"><a name="B22" /><div class="citation" tagx="citation" title="citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Froenicke</span><span class="given-names" tagx="given-names" title="given-names">L</span></span></span><span class="article-title" tagx="article-title" title="article-title">Origins of primate chromosomes – as delineated by Zoo-FISH and alignments of human and mouse draft genome sequences</span><span class="source" tagx="source" title="source">Cytogenet Genome Res</span><span class="year" tagx="year" title="year">2005</span><span class="volume" tagx="volume" title="volume">108</span><span class="fpage" tagx="fpage" title="fpage">122</span><span class="lpage" tagx="lpage" title="lpage">138</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15545724">15545724</a></span><span class="pub-id"><a href="https://dx.doi.org/10.1159/000080810">10.1159/000080810</a></span></div> </li> <li tag="ref"><a name="B23" /><div class="citation" tagx="citation" title="citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Froenicke</span><span class="given-names" tagx="given-names" title="given-names">L</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wienberg</span><span class="given-names" tagx="given-names" title="given-names">J</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Stone</span><span class="given-names" tagx="given-names" title="given-names">G</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Adams</span><span class="given-names" tagx="given-names" title="given-names">L</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Stanyon</span><span class="given-names" tagx="given-names" title="given-names">R</span></span></span><span class="article-title" tagx="article-title" title="article-title">Towards the delineation of the ancestral eutherian genome organization: comparative genome maps of human and the African elephant (<i>Loxodonta africana</i>) generated by chromosome painting</span><span class="source" tagx="source" title="source">Proc R Soc Lond B Biol Sci</span><span class="year" tagx="year" title="year">2003</span><span class="volume" tagx="volume" title="volume">270</span><span class="fpage" tagx="fpage" title="fpage">1331</span><span class="lpage" tagx="lpage" title="lpage">1340</span><span class="pub-id"><a href="https://dx.doi.org/10.1098/rspb.2003.2383">10.1098/rspb.2003.2383</a></span></div> </li> <li tag="ref"><a name="B24" /><div class="citation" tagx="citation" title="citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Robinson</span><span class="given-names" tagx="given-names" title="given-names">TJ</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Fu</span><span class="given-names" tagx="given-names" title="given-names">B</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ferguson-Smith</span><span class="given-names" tagx="given-names" title="given-names">MA</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Yang</span><span class="given-names" tagx="given-names" title="given-names">F</span></span></span><span class="article-title" tagx="article-title" title="article-title">Cross-species chromosome painting in the golden mole and elephant-shrew: support for the mammalian clades Afrotheria and Afroinsectiphillia but not Afroinsectivora</span><span class="source" tagx="source" title="source">Proc R Soc Lond B Biol Sci</span><span class="year" tagx="year" title="year">2004</span><span class="volume" tagx="volume" title="volume">271</span><span class="fpage" tagx="fpage" title="fpage">1477</span><span class="lpage" tagx="lpage" title="lpage">1484</span><span class="pub-id"><a href="https://dx.doi.org/10.1098/rspb.2004.2754">10.1098/rspb.2004.2754</a></span></div> </li> <li tag="ref"><a name="B25" /><div class="citation" tagx="citation" title="citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Pevzner</span><span class="given-names" tagx="given-names" title="given-names">P</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Tesler</span><span class="given-names" tagx="given-names" title="given-names">G</span></span></span><span class="article-title" tagx="article-title" title="article-title">Human and mouse genomic sequences reveal extensive breakpoint reuse in mammalian evolution</span><span class="source" tagx="source" title="source">Proc Natl Acad Sci USA</span><span class="year" tagx="year" title="year">2003</span><span class="volume" tagx="volume" title="volume">100</span><span class="fpage" tagx="fpage" title="fpage">7672</span><span class="lpage" tagx="lpage" title="lpage">7677</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12810957">12810957</a></span><span class="pub-id"><a href="https://dx.doi.org/10.1073/pnas.1330369100">10.1073/pnas.1330369100</a></span></div> </li> <li tag="ref"><a name="B26" /><div class="citation" tagx="citation" title="citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ruiz-Herrera</span><span class="given-names" tagx="given-names" title="given-names">A</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ponsà</span><span class="given-names" tagx="given-names" title="given-names">M</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">García</span><span class="given-names" tagx="given-names" title="given-names">F</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Egozcue</span><span class="given-names" tagx="given-names" title="given-names">J</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Garcia</span><span class="given-names" tagx="given-names" title="given-names">M</span></span></span><span class="article-title" tagx="article-title" title="article-title">Fragile sites in human and <i>Macaca fascicularis </i>are breakpoints in chromosome evolution</span><span class="source" tagx="source" title="source">Chromosome Res</span><span class="year" tagx="year" title="year">2002</span><span class="volume" tagx="volume" title="volume">10</span><span class="fpage" tagx="fpage" title="fpage">33</span><span class="lpage" tagx="lpage" title="lpage">44</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11863068">11863068</a></span><span class="pub-id"><a href="https://dx.doi.org/10.1023/A:1014261909613">10.1023/A:1014261909613</a></span></div> </li> <li tag="ref"><a name="B27" /><div class="citation" tagx="citation" title="citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ruiz-Herrera</span><span class="given-names" tagx="given-names" title="given-names">A</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Castresana</span><span class="given-names" tagx="given-names" title="given-names">J</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Robinson</span><span class="given-names" tagx="given-names" title="given-names">TJ</span></span></span><span class="article-title" tagx="article-title" title="article-title">Is mammalian evolution driven by regions of genome fragility?</span><span class="source" tagx="source" title="source">Genome Biol</span><span class="year" tagx="year" title="year">2006</span><span class="volume" tagx="volume" title="volume">7</span><span class="fpage" tagx="fpage" title="fpage">R115</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17156441">17156441</a></span><span class="pub-id"><a href="https://dx.doi.org/10.1186/gb-2006-7-12-r115">10.1186/gb-2006-7-12-r115</a></span></div> </li> <li tag="ref"><a name="B28" /><div class="citation" tagx="citation" title="citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Robinson</span><span class="given-names" tagx="given-names" title="given-names">TJ</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ruiz-Herrera</span><span class="given-names" tagx="given-names" title="given-names">A</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Froenicke</span><span class="given-names" tagx="given-names" title="given-names">L</span></span></span><span class="article-title" tagx="article-title" title="article-title">Dissecting the mammalian genome – new insights into chromosomal evolution</span><span class="source" tagx="source" title="source">Trends Genet</span><span class="year" tagx="year" title="year">2006</span><span class="volume" tagx="volume" title="volume">22</span><span class="fpage" tagx="fpage" title="fpage">297</span><span class="lpage" tagx="lpage" title="lpage">301</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16678302">16678302</a></span><span class="pub-id"><a href="https://dx.doi.org/10.1016/j.tig.2006.04.002">10.1016/j.tig.2006.04.002</a></span></div> </li> <li tag="ref"><a name="B29" /><div class="citation" tagx="citation" title="citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Sutherland</span><span class="given-names" tagx="given-names" title="given-names">GR</span></span></span><span class="article-title" tagx="article-title" title="article-title">Fragile sites on human chromosomes: demonstration of their dependence on the type of tissue culture medium</span><span class="source" tagx="source" title="source">Science</span><span class="year" tagx="year" title="year">1977</span><span class="volume" tagx="volume" title="volume">197</span><span class="fpage" tagx="fpage" title="fpage">265</span><span class="lpage" tagx="lpage" title="lpage">266</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/877551">877551</a></span><span class="pub-id"><a href="https://dx.doi.org/10.1126/science.877551">10.1126/science.877551</a></span></div> </li> <li tag="ref"><a name="B30" /><div class="citation" tagx="citation" title="citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Sutherland</span><span class="given-names" tagx="given-names" title="given-names">GR</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Baker</span><span class="given-names" tagx="given-names" title="given-names">E</span></span></span><span class="article-title" tagx="article-title" title="article-title">The clinical significance of fragile sites on human chromosomes</span><span class="source" tagx="source" title="source">Clin Genet</span><span class="year" tagx="year" title="year">2000</span><span class="volume" tagx="volume" title="volume">58</span><span class="fpage" tagx="fpage" title="fpage">157</span><span class="lpage" tagx="lpage" title="lpage">161</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11076037">11076037</a></span><span class="pub-id"><a href="https://dx.doi.org/10.1034/j.1399-0004.2000.580301.x">10.1034/j.1399-0004.2000.580301.x</a></span></div> </li> <li tag="ref"><a name="B31" /><div class="citation" tagx="citation" title="citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Sutherland</span><span class="given-names" tagx="given-names" title="given-names">GR</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Richards</span><span class="given-names" tagx="given-names" title="given-names">RI</span></span></span><span class="article-title" tagx="article-title" title="article-title">Fragile sites-cytogenetic similarity with molecular diversity</span><span class="source" tagx="source" title="source">Am J Hum Genet</span><span class="year" tagx="year" title="year">1999</span><span class="volume" tagx="volume" title="volume">64</span><span class="fpage" tagx="fpage" title="fpage">354</span><span class="lpage" tagx="lpage" title="lpage">359</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/9973272">9973272</a></span><span class="pub-id"><a href="https://dx.doi.org/10.1086/302267">10.1086/302267</a></span></div> </li> <li tag="ref"><a name="B32" /><div class="citation" tagx="citation" title="citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Robinson</span><span class="given-names" tagx="given-names" title="given-names">TJ</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Elder</span><span class="given-names" tagx="given-names" title="given-names">FF</span></span></span><span class="article-title" tagx="article-title" title="article-title">Multiple common fragile sites are expressed in the genome of the laboratory rat</span><span class="source" tagx="source" title="source">Chromosoma</span><span class="year" tagx="year" title="year">1987</span><span class="volume" tagx="volume" title="volume">96</span><span class="fpage" tagx="fpage" title="fpage">45</span><span class="lpage" tagx="lpage" title="lpage">49</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/2963735">2963735</a></span><span class="pub-id"><a href="https://dx.doi.org/10.1007/BF00285882">10.1007/BF00285882</a></span></div> </li> <li tag="ref"><a name="B33" /><div class="citation" tagx="citation" title="citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Tewari</span><span class="given-names" tagx="given-names" title="given-names">R</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Juyal</span><span class="given-names" tagx="given-names" title="given-names">RC</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Thelma</span><span class="given-names" tagx="given-names" title="given-names">BK</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Das</span><span class="given-names" tagx="given-names" title="given-names">BC</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Rao</span><span class="given-names" tagx="given-names" title="given-names">SRV</span></span></span><span class="article-title" tagx="article-title" title="article-title">Folate-sensitive fragile sites on the X-chromosomes heterochromatin of the Indian mole rat <i>Nesokia indica</i></span><span class="source" tagx="source" title="source">Cytogenet Cell Genet</span><span class="year" tagx="year" title="year">1987</span><span class="volume" tagx="volume" title="volume">44</span><span class="fpage" tagx="fpage" title="fpage">11</span><span class="lpage" tagx="lpage" title="lpage">17</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/3816302">3816302</a></span></div> </li> <li tag="ref"><a name="B34" /><div class="citation" tagx="citation" title="citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Elder</span><span class="given-names" tagx="given-names" title="given-names">FFB</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Robinson</span><span class="given-names" tagx="given-names" title="given-names">TJ</span></span></span><span class="article-title" tagx="article-title" title="article-title">Rodent common fragile sites, are they conserved? Evidence from mouse and rat</span><span class="source" tagx="source" title="source">Chromosoma</span><span class="year" tagx="year" title="year">1989</span><span class="volume" tagx="volume" title="volume">97</span><span class="fpage" tagx="fpage" title="fpage">459</span><span class="lpage" tagx="lpage" title="lpage">464</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/2501072">2501072</a></span><span class="pub-id"><a href="https://dx.doi.org/10.1007/BF00295030">10.1007/BF00295030</a></span></div> </li> <li tag="ref"><a name="B35" /><div class="citation" tagx="citation" title="citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">McAllister</span><span class="given-names" tagx="given-names" title="given-names">B</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Greenbaum</span><span class="given-names" tagx="given-names" title="given-names">IF</span></span></span><span class="article-title" tagx="article-title" title="article-title">How common are common fragile sites: variation of aphidicolin-induced chromosomal fragile sites in a population of the deer mouse (<i>Peromyscus maniculatus</i>)</span><span class="source" tagx="source" title="source">Hum Genet</span><span class="year" tagx="year" title="year">1997</span><span class="volume" tagx="volume" title="volume">100</span><span class="fpage" tagx="fpage" title="fpage">182</span><span class="lpage" tagx="lpage" title="lpage">188</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/9254846">9254846</a></span><span class="pub-id"><a href="https://dx.doi.org/10.1007/s004390050487">10.1007/s004390050487</a></span></div> </li> <li tag="ref"><a name="B36" /><div class="citation" tagx="citation" title="citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Rozier</span><span class="given-names" tagx="given-names" title="given-names">L</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">El-Achkar</span><span class="given-names" tagx="given-names" title="given-names">E</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Apiou</span><span class="given-names" tagx="given-names" title="given-names">F</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Debatisse</span><span class="given-names" tagx="given-names" title="given-names">M</span></span></span><span class="article-title" tagx="article-title" title="article-title">Characterization of a conserved aphidicolin-sensitive common fragile site at human 4q22 and mouse 6C1: possible association with an inherited disease and cancer</span><span class="source" tagx="source" title="source">Oncogene</span><span class="year" tagx="year" title="year">2004</span><span class="volume" tagx="volume" title="volume">23</span><span class="fpage" tagx="fpage" title="fpage">6872</span><span class="lpage" tagx="lpage" title="lpage">6880</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15286716">15286716</a></span><span class="pub-id"><a href="https://dx.doi.org/10.1038/sj.onc.1207809">10.1038/sj.onc.1207809</a></span></div> </li> <li tag="ref"><a name="B37" /><div class="citation" tagx="citation" title="citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Helmrich</span><span class="given-names" tagx="given-names" title="given-names">A</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Stout-Weider</span><span class="given-names" tagx="given-names" title="given-names">K</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hermann</span><span class="given-names" tagx="given-names" title="given-names">K</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Schrock</span><span class="given-names" tagx="given-names" title="given-names">E</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Heiden</span><span class="given-names" tagx="given-names" title="given-names">T</span></span></span><span class="article-title" tagx="article-title" title="article-title">Common fragile sites are conserved features of human and mouse chromosomes and relate to large active genes</span><span class="source" tagx="source" title="source">Genome Res</span><span class="year" tagx="year" title="year">2006</span><span class="volume" tagx="volume" title="volume">16</span><span class="fpage" tagx="fpage" title="fpage">1222</span><span class="lpage" tagx="lpage" title="lpage">1230</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16954539">16954539</a></span><span class="pub-id"><a href="https://dx.doi.org/10.1101/gr.5335506">10.1101/gr.5335506</a></span></div> </li> <li tag="ref"><a name="B38" /><div class="citation" tagx="citation" title="citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Riggs</span><span class="given-names" tagx="given-names" title="given-names">PK</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kuczek</span><span class="given-names" tagx="given-names" title="given-names">T</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Chrisman</span><span class="given-names" tagx="given-names" title="given-names">CL</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bidwell</span><span class="given-names" tagx="given-names" title="given-names">CA</span></span></span><span class="article-title" tagx="article-title" title="article-title">Analysis of aphidicolin-induced chromosome fragility in the domestic pig (<i>Sus scrofa</i>)</span><span class="source" tagx="source" title="source">Cytogenet Cell Genet</span><span class="year" tagx="year" title="year">1993</span><span class="volume" tagx="volume" title="volume">62</span><span class="fpage" tagx="fpage" title="fpage">110</span><span class="lpage" tagx="lpage" title="lpage">116</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/8428508">8428508</a></span></div> </li> <li tag="ref"><a name="B39" /><div class="citation" tagx="citation" title="citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Yang</span><span class="given-names" tagx="given-names" title="given-names">MY</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Long</span><span class="given-names" tagx="given-names" title="given-names">SE</span></span></span><span class="article-title" tagx="article-title" title="article-title">Folate sensitive common fragile sites in chromosomes of the domestic pig (<i>Sus scrofa</i>)</span><span class="source" tagx="source" title="source">Res Vet Sci</span><span class="year" tagx="year" title="year">1993</span><span class="volume" tagx="volume" title="volume">55</span><span class="fpage" tagx="fpage" title="fpage">231</span><span class="lpage" tagx="lpage" title="lpage">235</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/8235092">8235092</a></span></div> </li> <li tag="ref"><a name="B40" /><div class="citation" tagx="citation" title="citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ronne</span><span class="given-names" tagx="given-names" title="given-names">M</span></span></span><span class="article-title" tagx="article-title" title="article-title">Localization of fragile sites in the karyotype of <i>Sus scrofa domestica</i>: present status</span><span class="source" tagx="source" title="source">Hereditas</span><span class="year" tagx="year" title="year">1995</span><span class="volume" tagx="volume" title="volume">122</span><span class="fpage" tagx="fpage" title="fpage">153</span><span class="lpage" tagx="lpage" title="lpage">162</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/7558883">7558883</a></span><span class="pub-id"><a href="https://dx.doi.org/10.1111/j.1601-5223.1995.00153.x">10.1111/j.1601-5223.1995.00153.x</a></span></div> </li> <li tag="ref"><a name="B41" /><div class="citation" tagx="citation" title="citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Rodriguez</span><span class="given-names" tagx="given-names" title="given-names">V</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Llambi</span><span class="given-names" tagx="given-names" title="given-names">S</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Postiglioni</span><span class="given-names" tagx="given-names" title="given-names">A</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Guevara</span><span class="given-names" tagx="given-names" title="given-names">K</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Rincon</span><span class="given-names" tagx="given-names" title="given-names">G</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="article-title" tagx="article-title" title="article-title">Localisation of aphidicolin-induced break points in Holstein-Friesian cattle (<i>Bos taurus</i>) using RBG-banding</span><span class="source" tagx="source" title="source">Genet Sel Evol</span><span class="year" tagx="year" title="year">2002</span><span class="volume" tagx="volume" title="volume">34</span><span class="fpage" tagx="fpage" title="fpage">649</span><span class="lpage" tagx="lpage" title="lpage">656</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12486396">12486396</a></span><span class="pub-id"><a href="https://dx.doi.org/10.1051/gse:2002029">10.1051/gse:2002029</a></span></div> </li> <li tag="ref"><a name="B42" /><div class="citation" tagx="citation" title="citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ronne</span><span class="given-names" tagx="given-names" title="given-names">M</span></span></span><span class="article-title" tagx="article-title" title="article-title">Putative fragile sites in the horse karyotype</span><span class="source" tagx="source" title="source">Hereditas</span><span class="year" tagx="year" title="year">1992</span><span class="volume" tagx="volume" title="volume">117</span><span class="fpage" tagx="fpage" title="fpage">127</span><span class="lpage" tagx="lpage" title="lpage">136</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/1459856">1459856</a></span></div> </li> <li tag="ref"><a name="B43" /><div class="citation" tagx="citation" title="citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Stone</span><span class="given-names" tagx="given-names" title="given-names">DM</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Stephens</span><span class="given-names" tagx="given-names" title="given-names">KE</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Doles</span><span class="given-names" tagx="given-names" title="given-names">J</span></span></span><span class="article-title" tagx="article-title" title="article-title">Folate-sensitive and aphidicolin-inducible fragile sites are expressed in the genome of the domestic cat</span><span class="source" tagx="source" title="source">Cancer Genet Cytogenet</span><span class="year" tagx="year" title="year">1993</span><span class="volume" tagx="volume" title="volume">65</span><span class="fpage" tagx="fpage" title="fpage">130</span><span class="lpage" tagx="lpage" title="lpage">134</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/8453598">8453598</a></span><span class="pub-id"><a href="https://dx.doi.org/10.1016/0165-4608(93)90221-7">10.1016/0165-4608(93)90221-7</a></span></div> </li> <li tag="ref"><a name="B44" /><div class="citation" tagx="citation" title="citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ronne</span><span class="given-names" tagx="given-names" title="given-names">M</span></span></span><span class="article-title" tagx="article-title" title="article-title">Localization of fragile sites in the karyotypeof Felis catus</span><span class="source" tagx="source" title="source">Hereditas</span><span class="year" tagx="year" title="year">1995</span><span class="volume" tagx="volume" title="volume">122</span><span class="fpage" tagx="fpage" title="fpage">279</span><span class="lpage" tagx="lpage" title="lpage">283</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/8537243">8537243</a></span><span class="pub-id"><a href="https://dx.doi.org/10.1111/j.1601-5223.1995.00279.x">10.1111/j.1601-5223.1995.00279.x</a></span></div> </li> <li tag="ref"><a name="B45" /><div class="citation" tagx="citation" title="citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kubo</span><span class="given-names" tagx="given-names" title="given-names">K</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Matsuyama</span><span class="given-names" tagx="given-names" title="given-names">S</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Sato</span><span class="given-names" tagx="given-names" title="given-names">K</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Shiomi</span><span class="given-names" tagx="given-names" title="given-names">A</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ono</span><span class="given-names" tagx="given-names" title="given-names">K</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ito</span><span class="given-names" tagx="given-names" title="given-names">Y</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ohashi</span><span class="given-names" tagx="given-names" title="given-names">F</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Takamori</span><span class="given-names" tagx="given-names" title="given-names">Y</span></span></span><span class="article-title" tagx="article-title" title="article-title">Novel putative fragile sites observed in feline fibroblasts treated with aphidicolin and fluorodeoxyuridine</span><span class="source" tagx="source" title="source">J Vet Med Sci</span><span class="year" tagx="year" title="year">1998</span><span class="volume" tagx="volume" title="volume">60</span><span class="fpage" tagx="fpage" title="fpage">809</span><span class="lpage" tagx="lpage" title="lpage">813</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/9713808">9713808</a></span><span class="pub-id"><a href="https://dx.doi.org/10.1292/jvms.60.809">10.1292/jvms.60.809</a></span></div> </li> <li tag="ref"><a name="B46" /><div class="citation" tagx="citation" title="citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Stone</span><span class="given-names" tagx="given-names" title="given-names">DM</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Jacky</span><span class="given-names" tagx="given-names" title="given-names">PB</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hancock</span><span class="given-names" tagx="given-names" title="given-names">DD</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Prieur</span><span class="given-names" tagx="given-names" title="given-names">DJ</span></span></span><span class="article-title" tagx="article-title" title="article-title">Chromosomal fragile site expression in dogs: I. Breed specific differences</span><span class="source" tagx="source" title="source">Am J Med Genet</span><span class="year" tagx="year" title="year">1991</span><span class="volume" tagx="volume" title="volume">40</span><span class="fpage" tagx="fpage" title="fpage">214</span><span class="lpage" tagx="lpage" title="lpage">22</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/1910263">1910263</a></span><span class="pub-id"><a href="https://dx.doi.org/10.1002/ajmg.1320400219">10.1002/ajmg.1320400219</a></span></div> </li> <li tag="ref"><a name="B47" /><div class="citation" tagx="citation" title="citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Stone</span><span class="given-names" tagx="given-names" title="given-names">DM</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Stephens</span><span class="given-names" tagx="given-names" title="given-names">KE</span></span></span><span class="article-title" tagx="article-title" title="article-title">Bromodeoxyuridine induces chromosomal fragile sites in the canine genome</span><span class="source" tagx="source" title="source">Am J Med Genet</span><span class="year" tagx="year" title="year">1993</span><span class="volume" tagx="volume" title="volume">46</span><span class="fpage" tagx="fpage" title="fpage">198</span><span class="lpage" tagx="lpage" title="lpage">202</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/8484410">8484410</a></span><span class="pub-id"><a href="https://dx.doi.org/10.1002/ajmg.1320460220">10.1002/ajmg.1320460220</a></span></div> </li> <li tag="ref"><a name="B48" /><div class="citation" tagx="citation" title="citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Schmid</span><span class="given-names" tagx="given-names" title="given-names">M</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ott</span><span class="given-names" tagx="given-names" title="given-names">G</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Haaf</span><span class="given-names" tagx="given-names" title="given-names">T</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Scheres</span><span class="given-names" tagx="given-names" title="given-names">JMJC</span></span></span><span class="article-title" tagx="article-title" title="article-title">Evolutionary conservation of fragile sites induced by 5-azacytidine and 5-azadeoxycytidine in man, gorilla and chimpanzee</span><span class="source" tagx="source" title="source">Hum Genet</span><span class="year" tagx="year" title="year">1985</span><span class="volume" tagx="volume" title="volume">71</span><span class="fpage" tagx="fpage" title="fpage">342</span><span class="lpage" tagx="lpage" title="lpage">350</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/4077049">4077049</a></span><span class="pub-id"><a href="https://dx.doi.org/10.1007/BF00388461">10.1007/BF00388461</a></span></div> </li> <li tag="ref"><a name="B49" /><div class="citation" tagx="citation" title="citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Smeets</span><span class="given-names" tagx="given-names" title="given-names">DF</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Van de Klundert</span><span class="given-names" tagx="given-names" title="given-names">FA</span></span></span><span class="article-title" tagx="article-title" title="article-title">Common fragile sites in man and three closely related primate species</span><span class="source" tagx="source" title="source">Cytogenet Cell Genet</span><span class="year" tagx="year" title="year">1990</span><span class="volume" tagx="volume" title="volume">53</span><span class="fpage" tagx="fpage" title="fpage">8</span><span class="lpage" tagx="lpage" title="lpage">14</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/2323227">2323227</a></span></div> </li> <li tag="ref"><a name="B50" /><div class="citation" tagx="citation" title="citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Fundia</span><span class="given-names" tagx="given-names" title="given-names">A</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gorostiaga</span><span class="given-names" tagx="given-names" title="given-names">M</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Mudry</span><span class="given-names" tagx="given-names" title="given-names">M</span></span></span><span class="article-title" tagx="article-title" title="article-title">Expression of common fragile sites in two Ceboidea species: <i>Saimiri boliviensis </i>and <i>Alouatta caraya </i>(Primates: Platyrrhini)</span><span class="source" tagx="source" title="source">Genet Sel Evol</span><span class="year" tagx="year" title="year">2000</span><span class="volume" tagx="volume" title="volume">32</span><span class="fpage" tagx="fpage" title="fpage">87</span><span class="lpage" tagx="lpage" title="lpage">97</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/14736409">14736409</a></span><span class="pub-id"><a href="https://dx.doi.org/10.1051/gse:2000108">10.1051/gse:2000108</a></span></div> </li> <li tag="ref"><a name="B51" /><div class="citation" tagx="citation" title="citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ruiz-Herrera</span><span class="given-names" tagx="given-names" title="given-names">A</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Garcia</span><span class="given-names" tagx="given-names" title="given-names">F</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Giulotto</span><span class="given-names" tagx="given-names" title="given-names">E</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Attolini</span><span class="given-names" tagx="given-names" title="given-names">C</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Egozcue</span><span class="given-names" tagx="given-names" title="given-names">J</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ponsà</span><span class="given-names" tagx="given-names" title="given-names">M</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Garcia</span><span class="given-names" tagx="given-names" title="given-names">M</span></span></span><span class="article-title" tagx="article-title" title="article-title">Evolutionary breakpoints are co-localized with fragile sites and intrachromosomal telomeric sequences in primates</span><span class="source" tagx="source" title="source">Cytogenet GenomeRes</span><span class="year" tagx="year" title="year">2005</span><span class="volume" tagx="volume" title="volume">108</span><span class="fpage" tagx="fpage" title="fpage">234</span><span class="lpage" tagx="lpage" title="lpage">247</span><span class="pub-id"><a href="https://dx.doi.org/10.1159/000080822">10.1159/000080822</a></span></div> </li> <li tag="ref"><a name="B52" /><div class="citation" tagx="citation" title="citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Springer</span><span class="given-names" tagx="given-names" title="given-names">MS</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Murphy</span><span class="given-names" tagx="given-names" title="given-names">WJ</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Eizirik</span><span class="given-names" tagx="given-names" title="given-names">E</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">O'Brien</span><span class="given-names" tagx="given-names" title="given-names">SJ</span></span></span><span class="article-title" tagx="article-title" title="article-title">Placental mammal diversification and the Cretaceous-Tertiary boundary</span><span class="source" tagx="source" title="source">Proc Natl Acad Sci USA</span><span class="year" tagx="year" title="year">2003</span><span class="volume" tagx="volume" title="volume">100</span><span class="fpage" tagx="fpage" title="fpage">1056</span><span class="lpage" tagx="lpage" title="lpage">1061</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12552136">12552136</a></span><span class="pub-id"><a href="https://dx.doi.org/10.1073/pnas.0334222100">10.1073/pnas.0334222100</a></span></div> </li> <li tag="ref"><a name="B53" /><div class="citation" tagx="citation" title="citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Yang</span><span class="given-names" tagx="given-names" title="given-names">F</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Alkalaeva</span><span class="given-names" tagx="given-names" title="given-names">EZ</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Perelman</span><span class="given-names" tagx="given-names" title="given-names">PL</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Pardini</span><span class="given-names" tagx="given-names" title="given-names">AT</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Harrison</span><span class="given-names" tagx="given-names" title="given-names">WR</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">O'Brien</span><span class="given-names" tagx="given-names" title="given-names">PC</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Fu</span><span class="given-names" tagx="given-names" title="given-names">B</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Graphodatsky</span><span class="given-names" tagx="given-names" title="given-names">AS</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ferguson-Smith</span><span class="given-names" tagx="given-names" title="given-names">MA</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Robinson</span><span class="given-names" tagx="given-names" title="given-names">TJ</span></span></span><span class="article-title" tagx="article-title" title="article-title">Reciprocal chromosome painting among human, aardvark, and elephant (superorder Afrotheria) reveals the likely eutherian ancestral karyotype</span><span class="source" tagx="source" title="source">Proc Natl Acad Sci USA</span><span class="year" tagx="year" title="year">2003</span><span class="volume" tagx="volume" title="volume">100</span><span class="fpage" tagx="fpage" title="fpage">1062</span><span class="lpage" tagx="lpage" title="lpage">1066</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12552116">12552116</a></span><span class="pub-id"><a href="https://dx.doi.org/10.1073/pnas.0335540100">10.1073/pnas.0335540100</a></span></div> </li> <li tag="ref"><a name="B54" /><div class="citation" tagx="citation" title="citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Pardini</span><span class="given-names" tagx="given-names" title="given-names">AT</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">O'Brien</span><span class="given-names" tagx="given-names" title="given-names">PC</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Fu</span><span class="given-names" tagx="given-names" title="given-names">B</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bonde</span><span class="given-names" tagx="given-names" title="given-names">RK</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Elder</span><span class="given-names" tagx="given-names" title="given-names">FFB</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ferguson-Smith</span><span class="given-names" tagx="given-names" title="given-names">MA</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Yang</span><span class="given-names" tagx="given-names" title="given-names">F</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Robinson</span><span class="given-names" tagx="given-names" title="given-names">TJ</span></span></span><span class="article-title" tagx="article-title" title="article-title">Chromosome painting among Proboscidea, Hyracoidea and Sirenia: support for Paenungulata (Afrotheria, Mammalia) but not Tethytheria</span><span class="source" tagx="source" title="source">Proc R Soc Lond B Biol Sci</span><span class="year" tagx="year" title="year">2007</span><span class="volume" tagx="volume" title="volume">274</span><span class="fpage" tagx="fpage" title="fpage">1333</span><span class="lpage" tagx="lpage" title="lpage">1340</span><span class="pub-id"><a href="https://dx.doi.org/10.1098/rspb.2007.0088">10.1098/rspb.2007.0088</a></span></div> </li> <li tag="ref"><a name="B55" /><div class="citation" tagx="citation" title="citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Glover</span><span class="given-names" tagx="given-names" title="given-names">TW</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hoge</span><span class="given-names" tagx="given-names" title="given-names">AW</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Miller</span><span class="given-names" tagx="given-names" title="given-names">DE</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ascara-Wilke</span><span class="given-names" tagx="given-names" title="given-names">JE</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Adam</span><span class="given-names" tagx="given-names" title="given-names">AN</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Dagenais</span><span class="given-names" tagx="given-names" title="given-names">SL</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wilke</span><span class="given-names" tagx="given-names" title="given-names">CM</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Dierick</span><span class="given-names" tagx="given-names" title="given-names">HA</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Beer</span><span class="given-names" tagx="given-names" title="given-names">DG</span></span></span><span class="article-title" tagx="article-title" title="article-title">The murine Fhit gene is highly similar to its human orthologue and maps to a common fragile site region</span><span class="source" tagx="source" title="source">Cancer Res</span><span class="year" tagx="year" title="year">1998</span><span class="volume" tagx="volume" title="volume">58</span><span class="fpage" tagx="fpage" title="fpage">3409</span><span class="lpage" tagx="lpage" title="lpage">3414</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/9699673">9699673</a></span></div> </li> <li tag="ref"><a name="B56" /><div class="citation" tagx="citation" title="citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Uboldi</span><span class="given-names" tagx="given-names" title="given-names">C</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Guidi</span><span class="given-names" tagx="given-names" title="given-names">E</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Roperto</span><span class="given-names" tagx="given-names" title="given-names">S</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Russo</span><span class="given-names" tagx="given-names" title="given-names">V</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Roperto</span><span class="given-names" tagx="given-names" title="given-names">F</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Di Meo</span><span class="given-names" tagx="given-names" title="given-names">GP</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Iannuzzi</span><span class="given-names" tagx="given-names" title="given-names">L</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Floriot</span><span class="given-names" tagx="given-names" title="given-names">S</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Boussaha</span><span class="given-names" tagx="given-names" title="given-names">M</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Eggen</span><span class="given-names" tagx="given-names" title="given-names">A</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ferretti</span><span class="given-names" tagx="given-names" title="given-names">L</span></span></span><span class="article-title" tagx="article-title" title="article-title">Comparative genomic mapping of the bovine Fragile Histidine Triad (FHIT) tumour suppressor gene: characterization of a 2 Mb BAC contig covering the locus, complete annotation of the gene, analysis of cDNA and of physiological expression profiles</span><span class="source" tagx="source" title="source">BMC Genomics</span><span class="year" tagx="year" title="year">2006</span><span class="volume" tagx="volume" title="volume">7</span><span class="fpage" tagx="fpage" title="fpage">123</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16719907">16719907</a></span><span class="pub-id"><a href="https://dx.doi.org/10.1186/1471-2164-7-123">10.1186/1471-2164-7-123</a></span></div> </li> <li tag="ref"><a name="B57" /><div class="citation" tagx="citation" title="citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Stanhope</span><span class="given-names" tagx="given-names" title="given-names">MJ</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Waddell</span><span class="given-names" tagx="given-names" title="given-names">VG</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Madsen</span><span class="given-names" tagx="given-names" title="given-names">O</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">de Jong</span><span class="given-names" tagx="given-names" title="given-names">W</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hedges</span><span class="given-names" tagx="given-names" title="given-names">SB</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Cleven</span><span class="given-names" tagx="given-names" title="given-names">GC</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kao</span><span class="given-names" tagx="given-names" title="given-names">D</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Springer</span><span class="given-names" tagx="given-names" title="given-names">MS</span></span></span><span class="article-title" tagx="article-title" title="article-title">Molecular evidence for multiple origins of Insectivora and for a new order of endemic African insectivore mammals</span><span class="source" tagx="source" title="source">Proc Natl Acad Sci USA</span><span class="year" tagx="year" title="year">1998</span><span class="volume" tagx="volume" title="volume">95</span><span class="fpage" tagx="fpage" title="fpage">9967</span><span class="lpage" tagx="lpage" title="lpage">9972</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/9707584">9707584</a></span><span class="pub-id"><a href="https://dx.doi.org/10.1073/pnas.95.17.9967">10.1073/pnas.95.17.9967</a></span></div> </li> <li tag="ref"><a name="B58" /><div class="citation" tagx="citation" title="citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Malia</span><span class="given-names" tagx="given-names" title="given-names">MJ</span><span class="suffix" tagx="suffix" title="suffix">Jr</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Adkins</span><span class="given-names" tagx="given-names" title="given-names">RM</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Allard</span><span class="given-names" tagx="given-names" title="given-names">MW</span></span></span><span class="article-title" tagx="article-title" title="article-title">Molecular support for Afrotheria and the polyphyly of Lipotyphla based on analyses of the growth hormone receptor gene</span><span class="source" tagx="source" title="source">Mol Phylogenet Evol</span><span class="year" tagx="year" title="year">2002</span><span class="volume" tagx="volume" title="volume">24</span><span class="fpage" tagx="fpage" title="fpage">91</span><span class="lpage" tagx="lpage" title="lpage">101</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12128031">12128031</a></span><span class="pub-id"><a href="https://dx.doi.org/10.1016/S1055-7903(02)00219-1">10.1016/S1055-7903(02)00219-1</a></span></div> </li> <li tag="ref"><a name="B59" /><div class="citation" tagx="citation" title="citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Murata</span><span class="given-names" tagx="given-names" title="given-names">Y</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Nikaido</span><span class="given-names" tagx="given-names" title="given-names">M</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Sasaki</span><span class="given-names" tagx="given-names" title="given-names">T</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Cao</span><span class="given-names" tagx="given-names" title="given-names">Y</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Fukumoto</span><span class="given-names" tagx="given-names" title="given-names">Y</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hasegawa</span><span class="given-names" tagx="given-names" title="given-names">M</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Okada</span><span class="given-names" tagx="given-names" title="given-names">N</span></span></span><span class="article-title" tagx="article-title" title="article-title">Afrotherian phylogeny as inferred from complete mitochondrial genomes</span><span class="source" tagx="source" title="source">Mol Phylogenet Evol</span><span class="year" tagx="year" title="year">2003</span><span class="volume" tagx="volume" title="volume">28</span><span class="fpage" tagx="fpage" title="fpage">253</span><span class="lpage" tagx="lpage" title="lpage">260</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12878462">12878462</a></span><span class="pub-id"><a href="https://dx.doi.org/10.1016/S1055-7903(03)00035-6">10.1016/S1055-7903(03)00035-6</a></span></div> </li> <li tag="ref"><a name="B60" /><div class="citation" tagx="citation" title="citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Robinson</span><span class="given-names" tagx="given-names" title="given-names">TJ</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Seiffert</span><span class="given-names" tagx="given-names" title="given-names">E</span></span></span><span class="article-title" tagx="article-title" title="article-title">Afrotherian origins and interrelationships: new views and future prospects</span><span class="source" tagx="source" title="source">Curr Topics Dev Biol</span><span class="year" tagx="year" title="year">2004</span><span class="volume" tagx="volume" title="volume">63</span><span class="fpage" tagx="fpage" title="fpage">37</span><span class="lpage" tagx="lpage" title="lpage">60</span></div> </li> <li tag="ref"><a name="B61" /><div class="citation" tagx="citation" title="citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Sanchez-Villagra</span><span class="given-names" tagx="given-names" title="given-names">MR</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Narita</span><span class="given-names" tagx="given-names" title="given-names">Y</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kuratani</span><span class="given-names" tagx="given-names" title="given-names">S</span></span></span><span class="article-title" tagx="article-title" title="article-title">Thoracolumbar vertebral number: The first skeletal synapomorphy for afrotherian mammals</span><span class="source" tagx="source" title="source">Syst Biodiversity</span><span class="year" tagx="year" title="year">2007</span><span class="volume" tagx="volume" title="volume">5</span><span class="fpage" tagx="fpage" title="fpage">1</span><span class="lpage" tagx="lpage" title="lpage">7</span><span class="pub-id"><a href="https://dx.doi.org/10.1017/S1477200006002258">10.1017/S1477200006002258</a></span></div> </li> <li tag="ref"><a name="B62" /><div class="citation" tagx="citation" title="citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Tabuce</span><span class="given-names" tagx="given-names" title="given-names">R</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Marivaux</span><span class="given-names" tagx="given-names" title="given-names">L</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Adaci</span><span class="given-names" tagx="given-names" title="given-names">M</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bensalah</span><span class="given-names" tagx="given-names" title="given-names">M</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hartenberger</span><span class="given-names" tagx="given-names" title="given-names">J-L</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Mahboubi</span><span class="given-names" tagx="given-names" title="given-names">M</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Mebrouk</span><span class="given-names" tagx="given-names" title="given-names">F</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Tafforeau</span><span class="given-names" tagx="given-names" title="given-names">P</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Jaeger</span><span class="given-names" tagx="given-names" title="given-names">JJ</span></span></span><span class="article-title" tagx="article-title" title="article-title">Early Tertiary mammals from North Africa reinforce the molecular Afrotheria clade</span><span class="source" tagx="source" title="source">Proc R Soc Lond B Biol Sci</span><span class="year" tagx="year" title="year">2007</span><span class="volume" tagx="volume" title="volume">274</span><span class="fpage" tagx="fpage" title="fpage">1159</span><span class="lpage" tagx="lpage" title="lpage">1166</span><span class="pub-id"><a href="https://dx.doi.org/10.1098/rspb.2006.0229">10.1098/rspb.2006.0229</a></span></div> </li> <li tag="ref"><a name="B63" /><div class="citation" tagx="citation" title="citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Carter</span><span class="given-names" tagx="given-names" title="given-names">AM</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Blankenship</span><span class="given-names" tagx="given-names" title="given-names">TN</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Enders</span><span class="given-names" tagx="given-names" title="given-names">AC</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Vogel</span><span class="given-names" tagx="given-names" title="given-names">P</span></span></span><span class="article-title" tagx="article-title" title="article-title">The fetal membranes of the otter shrews and a synapomorphy for Afrotheria</span><span class="source" tagx="source" title="source">Placenta</span><span class="year" tagx="year" title="year">2006</span><span class="volume" tagx="volume" title="volume">27</span><span class="fpage" tagx="fpage" title="fpage">e268</span></div> </li> <li tag="ref"><a name="B64" /><div class="citation" tagx="citation" title="citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Svartman</span><span class="given-names" tagx="given-names" title="given-names">M</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Stone</span><span class="given-names" tagx="given-names" title="given-names">G</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Stanyon</span><span class="given-names" tagx="given-names" title="given-names">R</span></span></span><span class="article-title" tagx="article-title" title="article-title">The ancestral Eutherian karyotype is present in Xenarthra</span><span class="source" tagx="source" title="source">PLoS Genet</span><span class="year" tagx="year" title="year">2006</span><span class="volume" tagx="volume" title="volume">2</span><span class="fpage" tagx="fpage" title="fpage">e109</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16848642">16848642</a></span><span class="pub-id"><a href="https://dx.doi.org/10.1371/journal.pgen.0020109">10.1371/journal.pgen.0020109</a></span></div> </li> <li tag="ref"><a name="B65" /><div class="citation" tagx="citation" title="citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Yang</span><span class="given-names" tagx="given-names" title="given-names">F</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Graphodatsky</span><span class="given-names" tagx="given-names" title="given-names">AS</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Li</span><span class="given-names" tagx="given-names" title="given-names">T</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Fu</span><span class="given-names" tagx="given-names" title="given-names">B</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Dobigny</span><span class="given-names" tagx="given-names" title="given-names">G</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wang</span><span class="given-names" tagx="given-names" title="given-names">J</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Perelman</span><span class="given-names" tagx="given-names" title="given-names">PL</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Serdukova</span><span class="given-names" tagx="given-names" title="given-names">NA</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Su</span><span class="given-names" tagx="given-names" title="given-names">W</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">O'Brien</span><span class="given-names" tagx="given-names" title="given-names">PC</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wang</span><span class="given-names" tagx="given-names" title="given-names">Y</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ferguson-Smith</span><span class="given-names" tagx="given-names" title="given-names">MA</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Volobouev</span><span class="given-names" tagx="given-names" title="given-names">V</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Nie</span><span class="given-names" tagx="given-names" title="given-names">W</span></span></span><span class="article-title" tagx="article-title" title="article-title">Comparative genome maps of the pangolin, hedgehog, sloth, anteater and human revealed by cross-species chromosome painting: further insight into the ancestral karyotype and genome evolution of eutherian mammals</span><span class="source" tagx="source" title="source">Chromosome Res</span><span class="year" tagx="year" title="year">2006</span><span class="volume" tagx="volume" title="volume">14</span><span class="fpage" tagx="fpage" title="fpage">283</span><span class="lpage" tagx="lpage" title="lpage">296</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16628499">16628499</a></span><span class="pub-id"><a href="https://dx.doi.org/10.1007/s10577-006-1045-6">10.1007/s10577-006-1045-6</a></span></div> </li> <li tag="ref"><a name="B66" /><div class="citation" tagx="citation" title="citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ye</span><span class="given-names" tagx="given-names" title="given-names">J</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Biltueva</span><span class="given-names" tagx="given-names" title="given-names">L</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Huang</span><span class="given-names" tagx="given-names" title="given-names">L</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Nie</span><span class="given-names" tagx="given-names" title="given-names">W</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wang</span><span class="given-names" tagx="given-names" title="given-names">J</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Jing</span><span class="given-names" tagx="given-names" title="given-names">M</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Su</span><span class="given-names" tagx="given-names" title="given-names">W</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Vorobieva</span><span class="given-names" tagx="given-names" title="given-names">NV</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Jiang</span><span class="given-names" tagx="given-names" title="given-names">X</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Graphodatsky</span><span class="given-names" tagx="given-names" title="given-names">AS</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Yang</span><span class="given-names" tagx="given-names" title="given-names">F</span></span></span><span class="article-title" tagx="article-title" title="article-title">Cross-species chromosome painting unveils cytogenetic signatures for the Eulipotyphla and evidence for the polyphyly of Insectivora</span><span class="source" tagx="source" title="source">Chromosome Res</span><span class="year" tagx="year" title="year">2006</span><span class="volume" tagx="volume" title="volume">14</span><span class="fpage" tagx="fpage" title="fpage">151</span><span class="lpage" tagx="lpage" title="lpage">159</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16544189">16544189</a></span><span class="pub-id"><a href="https://dx.doi.org/10.1007/s10577-006-1032-y">10.1007/s10577-006-1032-y</a></span></div> </li> <li tag="ref"><a name="B67" /><div class="citation" tagx="citation" title="citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Biltueva</span><span class="given-names" tagx="given-names" title="given-names">LS</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Yang</span><span class="given-names" tagx="given-names" title="given-names">F</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Vorobieva</span><span class="given-names" tagx="given-names" title="given-names">NV</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Graphodatsky</span><span class="given-names" tagx="given-names" title="given-names">AS</span></span></span><span class="article-title" tagx="article-title" title="article-title">Comparative maps between the domestic dog and pig</span><span class="source" tagx="source" title="source">Mamm Genome</span><span class="year" tagx="year" title="year">2004</span><span class="volume" tagx="volume" title="volume">15</span><span class="fpage" tagx="fpage" title="fpage">809</span><span class="lpage" tagx="lpage" title="lpage">818</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15520883">15520883</a></span><span class="pub-id"><a href="https://dx.doi.org/10.1007/s00335-004-2391-3">10.1007/s00335-004-2391-3</a></span></div> </li> <li tag="ref"><a name="B68" /><div class="citation" tagx="citation" title="citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Stanyon</span><span class="given-names" tagx="given-names" title="given-names">R</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Koehler</span><span class="given-names" tagx="given-names" title="given-names">U</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Consigliere</span><span class="given-names" tagx="given-names" title="given-names">S</span></span></span><span class="article-title" tagx="article-title" title="article-title">Chromosome painting reveals that galagos have highly derived karyotypes</span><span class="source" tagx="source" title="source">Am J Phys Anthropol</span><span class="year" tagx="year" title="year">2002</span><span class="volume" tagx="volume" title="volume">117</span><span class="fpage" tagx="fpage" title="fpage">319</span><span class="lpage" tagx="lpage" title="lpage">326</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11920367">11920367</a></span><span class="pub-id"><a href="https://dx.doi.org/10.1002/ajpa.10047">10.1002/ajpa.10047</a></span></div> </li> <li tag="ref"><a name="B69" /><div class="citation" tagx="citation" title="citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Nie</span><span class="given-names" tagx="given-names" title="given-names">W</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">O'Brien</span><span class="given-names" tagx="given-names" title="given-names">PCM</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Fu</span><span class="given-names" tagx="given-names" title="given-names">B</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wang</span><span class="given-names" tagx="given-names" title="given-names">J</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Su</span><span class="given-names" tagx="given-names" title="given-names">W</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ferguson-Smith</span><span class="given-names" tagx="given-names" title="given-names">MA</span></span></span><span class="article-title" tagx="article-title" title="article-title">Chromosome painting between human and lorisform prosimians: evidence for the HSA 7/16 synteny in the primate ancestral karyotype</span><span class="source" tagx="source" title="source">Am J Phys Anthropol</span><span class="year" tagx="year" title="year">2006</span><span class="volume" tagx="volume" title="volume">129</span><span class="fpage" tagx="fpage" title="fpage">250</span><span class="lpage" tagx="lpage" title="lpage">259</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16323198">16323198</a></span><span class="pub-id"><a href="https://dx.doi.org/10.1002/ajpa.20299">10.1002/ajpa.20299</a></span></div> </li> <li tag="ref"><a name="B70" /><div class="citation" tagx="citation" title="citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Stanyon</span><span class="given-names" tagx="given-names" title="given-names">R</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Consigliere</span><span class="given-names" tagx="given-names" title="given-names">S</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Müller</span><span class="given-names" tagx="given-names" title="given-names">S</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Morescalchi</span><span class="given-names" tagx="given-names" title="given-names">A</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Neusser</span><span class="given-names" tagx="given-names" title="given-names">M</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wienberg</span><span class="given-names" tagx="given-names" title="given-names">J</span></span></span><span class="article-title" tagx="article-title" title="article-title">Fluorescent in situ hybridization (FISH) maps chromosomal homologies between the dusty titi and squirrel monkey</span><span class="source" tagx="source" title="source">Am J Primatol</span><span class="year" tagx="year" title="year">2000</span><span class="volume" tagx="volume" title="volume">50</span><span class="fpage" tagx="fpage" title="fpage">95</span><span class="lpage" tagx="lpage" title="lpage">107</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/10676707">10676707</a></span><span class="pub-id"><a href="https://dx.doi.org/10.1002/(SICI)1098-2345(200002)50:2&lt;95::AID-AJP1&gt;3.0.CO;2-8">10.1002/(SICI)1098-2345(200002)50:2&amp;lt;95::AID-AJP1&amp;gt;3.0.CO;2-8</a></span></div> </li> <li tag="ref"><a name="B71" /><div class="citation" tagx="citation" title="citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Stanyon</span><span class="given-names" tagx="given-names" title="given-names">R</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bonvicino</span><span class="given-names" tagx="given-names" title="given-names">CR</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Svartman</span><span class="given-names" tagx="given-names" title="given-names">M</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Seuanez</span><span class="given-names" tagx="given-names" title="given-names">HN</span></span></span><span class="article-title" tagx="article-title" title="article-title">Chromosome painting in <i>Callicebus lugens</i>, the species with the lowest diploid number (2n = 16) known in primates</span><span class="source" tagx="source" title="source">Chromosoma</span><span class="year" tagx="year" title="year">2004</span><span class="volume" tagx="volume" title="volume">112</span><span class="fpage" tagx="fpage" title="fpage">201</span><span class="lpage" tagx="lpage" title="lpage">206</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/14608465">14608465</a></span><span class="pub-id"><a href="https://dx.doi.org/10.1007/s00412-003-0261-5">10.1007/s00412-003-0261-5</a></span></div> </li> <li tag="ref"><a name="B72" /><div class="citation" tagx="citation" title="citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bigoni</span><span class="given-names" tagx="given-names" title="given-names">F</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Stanyon</span><span class="given-names" tagx="given-names" title="given-names">R</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Koehler</span><span class="given-names" tagx="given-names" title="given-names">U</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Morescalchi</span><span class="given-names" tagx="given-names" title="given-names">AM</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wienberg</span><span class="given-names" tagx="given-names" title="given-names">J</span></span></span><span class="article-title" tagx="article-title" title="article-title">Mapping homology between human and black and white colobine monkey chromosomes by fluorescent in situ hybridization</span><span class="source" tagx="source" title="source">Am J Primatol</span><span class="year" tagx="year" title="year">1997</span><span class="volume" tagx="volume" title="volume">42</span><span class="fpage" tagx="fpage" title="fpage">289</span><span class="lpage" tagx="lpage" title="lpage">298</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/9261510">9261510</a></span><span class="pub-id"><a href="https://dx.doi.org/10.1002/(SICI)1098-2345(1997)42:4&lt;289::AID-AJP4&gt;3.0.CO;2-T">10.1002/(SICI)1098-2345(1997)42:4&amp;lt;289::AID-AJP4&amp;gt;3.0.CO;2-T</a></span></div> </li> <li tag="ref"><a name="B73" /><div class="citation" tagx="citation" title="citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bigoni</span><span class="given-names" tagx="given-names" title="given-names">F</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Houck</span><span class="given-names" tagx="given-names" title="given-names">ML</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ryder</span><span class="given-names" tagx="given-names" title="given-names">OA</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wienberg</span><span class="given-names" tagx="given-names" title="given-names">J</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Stanyon</span><span class="given-names" tagx="given-names" title="given-names">R</span></span></span><span class="article-title" tagx="article-title" title="article-title">Chromosome painting shows that <i>Pygathrix nemaeus </i>has the most basalkaryotype among Asia Colobinae</span><span class="source" tagx="source" title="source">Int J Primatol</span><span class="year" tagx="year" title="year">2004</span><span class="volume" tagx="volume" title="volume">25</span><span class="fpage" tagx="fpage" title="fpage">679</span><span class="lpage" tagx="lpage" title="lpage">688</span><span class="pub-id"><a href="https://dx.doi.org/10.1023/B:IJOP.0000023580.46185.9c">10.1023/B:IJOP.0000023580.46185.9c</a></span></div> </li> <li tag="ref"><a name="B74" /><div class="citation" tagx="citation" title="citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bigoni</span><span class="given-names" tagx="given-names" title="given-names">F</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Stanyon</span><span class="given-names" tagx="given-names" title="given-names">R</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wimmer</span><span class="given-names" tagx="given-names" title="given-names">R</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Schempp</span><span class="given-names" tagx="given-names" title="given-names">W</span></span></span><span class="article-title" tagx="article-title" title="article-title">Chromosome painting shows that the proboscis monkey (<i>Nasalis larvatus</i>) has a derived karyotype and is phylogenetically nested within Asian Colobines</span><span class="source" tagx="source" title="source">Am J Primatol</span><span class="year" tagx="year" title="year">2003</span><span class="volume" tagx="volume" title="volume">60</span><span class="fpage" tagx="fpage" title="fpage">85</span><span class="lpage" tagx="lpage" title="lpage">93</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12874840">12874840</a></span><span class="pub-id"><a href="https://dx.doi.org/10.1002/ajp.10095">10.1002/ajp.10095</a></span></div> </li> <li tag="ref"><a name="B75" /><div class="citation" tagx="citation" title="citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Murphy</span><span class="given-names" tagx="given-names" title="given-names">WJ</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Froenicke</span><span class="given-names" tagx="given-names" title="given-names">L</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">O'Brien</span><span class="given-names" tagx="given-names" title="given-names">SJ</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Stanyon</span><span class="given-names" tagx="given-names" title="given-names">R</span></span></span><span class="article-title" tagx="article-title" title="article-title">The origin of human chromosome 1 and its homologs in placental mammals</span><span class="source" tagx="source" title="source">Genome Res</span><span class="year" tagx="year" title="year">2003</span><span class="volume" tagx="volume" title="volume">13</span><span class="fpage" tagx="fpage" title="fpage">1880</span><span class="lpage" tagx="lpage" title="lpage">1888</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12869576">12869576</a></span></div> </li> <li tag="ref"><a name="B76" /><div class="citation" tagx="citation" title="citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Waddell</span><span class="given-names" tagx="given-names" title="given-names">PJ</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Cao</span><span class="given-names" tagx="given-names" title="given-names">Y</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hasegawa</span><span class="given-names" tagx="given-names" title="given-names">M</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Mindell</span><span class="given-names" tagx="given-names" title="given-names">DP</span></span></span><span class="article-title" tagx="article-title" title="article-title">Assessing the Cretaceous superordinal divergence times within birds and placental mammals by using whole mitochondrial protein sequences and an extended statistical framework</span><span class="source" tagx="source" title="source">Syst Biol</span><span class="year" tagx="year" title="year">1999</span><span class="volume" tagx="volume" title="volume">48</span><span class="fpage" tagx="fpage" title="fpage">119</span><span class="lpage" tagx="lpage" title="lpage">137</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12078636">12078636</a></span><span class="pub-id"><a href="https://dx.doi.org/10.1080/106351599260481">10.1080/106351599260481</a></span></div> </li> <li tag="ref"><a name="B77" /><div class="citation" tagx="citation" title="citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Froenicke</span><span class="given-names" tagx="given-names" title="given-names">L</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Caldes</span><span class="given-names" tagx="given-names" title="given-names">MG</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Graphodatsky</span><span class="given-names" tagx="given-names" title="given-names">A</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Müller</span><span class="given-names" tagx="given-names" title="given-names">S</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lyons</span><span class="given-names" tagx="given-names" title="given-names">LA</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Robinson</span><span class="given-names" tagx="given-names" title="given-names">TJ</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Volleth</span><span class="given-names" tagx="given-names" title="given-names">M</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Yang</span><span class="given-names" tagx="given-names" title="given-names">F</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wienberg</span><span class="given-names" tagx="given-names" title="given-names">J</span></span></span><span class="article-title" tagx="article-title" title="article-title">Are molecular cytogenetics and bioinformatics suggesting contradictory models of ancestral mammalian genomes?</span><span class="source" tagx="source" title="source">Genome Res</span><span class="year" tagx="year" title="year">2006</span><span class="volume" tagx="volume" title="volume">16</span><span class="fpage" tagx="fpage" title="fpage">306</span><span class="lpage" tagx="lpage" title="lpage">310</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16510895">16510895</a></span><span class="pub-id"><a href="https://dx.doi.org/10.1101/gr.3955206">10.1101/gr.3955206</a></span></div> </li> <li tag="ref"><a name="B78" /><div class="citation" tagx="citation" title="citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kellogg</span><span class="given-names" tagx="given-names" title="given-names">ME</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Burkett</span><span class="given-names" tagx="given-names" title="given-names">S</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Dennis</span><span class="given-names" tagx="given-names" title="given-names">TR</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Stone</span><span class="given-names" tagx="given-names" title="given-names">G</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gray</span><span class="given-names" tagx="given-names" title="given-names">BA</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">McGuire</span><span class="given-names" tagx="given-names" title="given-names">PM</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zori</span><span class="given-names" tagx="given-names" title="given-names">RT</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Stanyon</span><span class="given-names" tagx="given-names" title="given-names">R</span></span></span><span class="article-title" tagx="article-title" title="article-title">Chromosome painting in the manatee supports Afrotheria and Paenungulata</span><span class="source" tagx="source" title="source">BMC Evol Biol</span><span class="year" tagx="year" title="year">2007</span><span class="volume" tagx="volume" title="volume">7</span><span class="fpage" tagx="fpage" title="fpage">6</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17244368">17244368</a></span><span class="pub-id"><a href="https://dx.doi.org/10.1186/1471-2148-7-6">10.1186/1471-2148-7-6</a></span></div> </li> <li tag="ref"><a name="B79" /><div class="citation" tagx="citation" title="citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kumar</span><span class="given-names" tagx="given-names" title="given-names">S</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hedges</span><span class="given-names" tagx="given-names" title="given-names">SB</span></span></span><span class="article-title" tagx="article-title" title="article-title">A molecular timescale for vertebrate evolution</span><span class="source" tagx="source" title="source">Nature</span><span class="year" tagx="year" title="year">1998</span><span class="volume" tagx="volume" title="volume">392</span><span class="fpage" tagx="fpage" title="fpage">917</span><span class="lpage" tagx="lpage" title="lpage">920</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/9582070">9582070</a></span><span class="pub-id"><a href="https://dx.doi.org/10.1038/31927">10.1038/31927</a></span></div> </li> <li tag="ref"><a name="B80" /><div class="citation" tagx="citation" title="citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Woodburne</span><span class="given-names" tagx="given-names" title="given-names">MO</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Rich</span><span class="given-names" tagx="given-names" title="given-names">TH</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Springer</span><span class="given-names" tagx="given-names" title="given-names">MS</span></span></span><span class="article-title" tagx="article-title" title="article-title">The evolution of tribospheny and the antiquity of mammalian clades</span><span class="source" tagx="source" title="source">Mol Phylogenet Evol</span><span class="year" tagx="year" title="year">2003</span><span class="volume" tagx="volume" title="volume">28</span><span class="fpage" tagx="fpage" title="fpage">360</span><span class="lpage" tagx="lpage" title="lpage">385</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12878472">12878472</a></span><span class="pub-id"><a href="https://dx.doi.org/10.1016/S1055-7903(03)00113-1">10.1016/S1055-7903(03)00113-1</a></span></div> </li> <li tag="ref"><a name="B81" /><div class="citation" tagx="citation" title="citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">van Rheede</span><span class="given-names" tagx="given-names" title="given-names">T</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bastiaans</span><span class="given-names" tagx="given-names" title="given-names">T</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Boone</span><span class="given-names" tagx="given-names" title="given-names">DN</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hedges</span><span class="given-names" tagx="given-names" title="given-names">SB</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">de Jong</span><span class="given-names" tagx="given-names" title="given-names">WW</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Madsen</span><span class="given-names" tagx="given-names" title="given-names">O</span></span></span><span class="article-title" tagx="article-title" title="article-title">The platypus is in its place: nuclear genes and indels confirm the sister group relation of monotremes and therians</span><span class="source" tagx="source" title="source">Mol Biol Evol</span><span class="year" tagx="year" title="year">2006</span><span class="volume" tagx="volume" title="volume">23</span><span class="fpage" tagx="fpage" title="fpage">587</span><span class="lpage" tagx="lpage" title="lpage">597</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16291999">16291999</a></span><span class="pub-id"><a href="https://dx.doi.org/10.1093/molbev/msj064">10.1093/molbev/msj064</a></span></div> </li> <li tag="ref"><a name="B82" /><div class="citation" tagx="citation" title="citation"><span class="article-title" tagx="article-title" title="article-title">The Ensembl genome browser of Sanger Center and EMBL database</span><a href="http://www.ensembl.org" /></div> </li> <li tag="ref"><a name="B83" /><div class="citation" tagx="citation" title="citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Böhm</span><span class="given-names" tagx="given-names" title="given-names">U</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Dahm</span><span class="given-names" tagx="given-names" title="given-names">PF</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">McAllister</span><span class="given-names" tagx="given-names" title="given-names">BF</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Greenbaum</span><span class="given-names" tagx="given-names" title="given-names">IF</span></span></span><span class="article-title" tagx="article-title" title="article-title">Identifying chromosomal fragile sites from individuals: a multinomial statistical model</span><span class="source" tagx="source" title="source">Hum Genet</span><span class="year" tagx="year" title="year">1995</span><span class="volume" tagx="volume" title="volume">95</span><span class="fpage" tagx="fpage" title="fpage">249</span><span class="lpage" tagx="lpage" title="lpage">256</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/7868115">7868115</a></span><span class="pub-id"><a href="https://dx.doi.org/10.1007/BF00225189">10.1007/BF00225189</a></span></div> </li> <li tag="ref"><a name="B84" /><div class="citation" tagx="citation" title="citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Goureau</span><span class="given-names" tagx="given-names" title="given-names">A</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Yerle</span><span class="given-names" tagx="given-names" title="given-names">M</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Schmitz</span><span class="given-names" tagx="given-names" title="given-names">A</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Riquet</span><span class="given-names" tagx="given-names" title="given-names">J</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Milan</span><span class="given-names" tagx="given-names" title="given-names">D</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Pinton</span><span class="given-names" tagx="given-names" title="given-names">P</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Frelat</span><span class="given-names" tagx="given-names" title="given-names">G</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gellin</span><span class="given-names" tagx="given-names" title="given-names">J</span></span></span><span class="article-title" tagx="article-title" title="article-title">Human and porcine correspondence of chromosome segments using bidirectional chromosome painting</span><span class="source" tagx="source" title="source">Genomics</span><span class="year" tagx="year" title="year">1996</span><span class="volume" tagx="volume" title="volume">36</span><span class="fpage" tagx="fpage" title="fpage">252</span><span class="lpage" tagx="lpage" title="lpage">62</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/8812451">8812451</a></span><span class="pub-id"><a href="https://dx.doi.org/10.1006/geno.1996.0460">10.1006/geno.1996.0460</a></span></div> </li> <li tag="ref"><a name="B85" /><div class="citation" tagx="citation" title="citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Korstanje</span><span class="given-names" tagx="given-names" title="given-names">R</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">O'Brien</span><span class="given-names" tagx="given-names" title="given-names">PCM</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Yang</span><span class="given-names" tagx="given-names" title="given-names">F</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Rens</span><span class="given-names" tagx="given-names" title="given-names">W</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bosma</span><span class="given-names" tagx="given-names" title="given-names">AA</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">van Lith</span><span class="given-names" tagx="given-names" title="given-names">AH</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">van Zutphen</span><span class="given-names" tagx="given-names" title="given-names">LFM</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ferguson-Smith</span><span class="given-names" tagx="given-names" title="given-names">MA</span></span></span><span class="article-title" tagx="article-title" title="article-title">Complete homology maps of the rabbit (<i>Oryctolagus cuniculus</i>) and human by reciprocal chromosome painting</span><span class="source" tagx="source" title="source">Cytogenet Cell Genet</span><span class="year" tagx="year" title="year">1999</span><span class="volume" tagx="volume" title="volume">86</span><span class="fpage" tagx="fpage" title="fpage">317</span><span class="lpage" tagx="lpage" title="lpage">322</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/10575232">10575232</a></span><span class="pub-id"><a href="https://dx.doi.org/10.1159/000015325">10.1159/000015325</a></span></div> </li> <li tag="ref"><a name="B86" /><div class="citation" tagx="citation" title="citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Müller</span><span class="given-names" tagx="given-names" title="given-names">S</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Stanyon</span><span class="given-names" tagx="given-names" title="given-names">R</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">O'Brien</span><span class="given-names" tagx="given-names" title="given-names">PCM</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ferguson-Smith</span><span class="given-names" tagx="given-names" title="given-names">MA</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Plesker</span><span class="given-names" tagx="given-names" title="given-names">R</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wienberg</span><span class="given-names" tagx="given-names" title="given-names">J</span></span></span><span class="article-title" tagx="article-title" title="article-title">Defining the ancestral karyotype of all primates by multidirectional chromosome painting between tree shrews, lemurs and humans</span><span class="source" tagx="source" title="source">Chromosoma</span><span class="year" tagx="year" title="year">1999</span><span class="volume" tagx="volume" title="volume">108</span><span class="fpage" tagx="fpage" title="fpage">393</span><span class="lpage" tagx="lpage" title="lpage">400</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/10591999">10591999</a></span><span class="pub-id"><a href="https://dx.doi.org/10.1007/s004120050391">10.1007/s004120050391</a></span></div> </li> <li tag="ref"><a name="B87" /><div class="citation" tagx="citation" title="citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Stanyon</span><span class="given-names" tagx="given-names" title="given-names">R</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Stone</span><span class="given-names" tagx="given-names" title="given-names">G</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Garcia</span><span class="given-names" tagx="given-names" title="given-names">M</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Froenicke</span><span class="given-names" tagx="given-names" title="given-names">L</span></span></span><span class="article-title" tagx="article-title" title="article-title">Reciprocal chromosome painting shows that squirrels, unlike murid rodents, have a highly conserved genome organization</span><span class="source" tagx="source" title="source">Genomics</span><span class="year" tagx="year" title="year">2003</span><span class="volume" tagx="volume" title="volume">82</span><span class="fpage" tagx="fpage" title="fpage">245</span><span class="lpage" tagx="lpage" title="lpage">249</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12837274">12837274</a></span><span class="pub-id"><a href="https://dx.doi.org/10.1016/S0888-7543(03)00109-5">10.1016/S0888-7543(03)00109-5</a></span></div> </li> <li tag="ref"><a name="B88" /><div class="citation" tagx="citation" title="citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wienberg</span><span class="given-names" tagx="given-names" title="given-names">J</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Stanyon</span><span class="given-names" tagx="given-names" title="given-names">R</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Nash</span><span class="given-names" tagx="given-names" title="given-names">WG</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">O'Brien</span><span class="given-names" tagx="given-names" title="given-names">PC</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Yang</span><span class="given-names" tagx="given-names" title="given-names">F</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">O'Brien</span><span class="given-names" tagx="given-names" title="given-names">SJ</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ferguson-Smith</span><span class="given-names" tagx="given-names" title="given-names">MA</span></span></span><span class="article-title" tagx="article-title" title="article-title">Conservation of human vs. feline genome organization revealed by reciprocal chromosome painting</span><span class="source" tagx="source" title="source">Cytogenet Cell Genet</span><span class="year" tagx="year" title="year">1997</span><span class="volume" tagx="volume" title="volume">77</span><span class="fpage" tagx="fpage" title="fpage">211</span><span class="lpage" tagx="lpage" title="lpage">217</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/9284919">9284919</a></span></div> </li> </ul> </div> </div>  </body></html>