<html xmlns="http://www.w3.org/1999/xhtml"><head> <style type="text/css">
         	     		 
         	    a                          {background : #ffffff; }
         	    article                    {border-style : dotted; border-width : 2px; }
         		 	
         		 	div                        {background : #ffffcc;}
         		 	div.abstract-title         {font-weight : bold ; font-size : 16pt;}
         		 	div.ack                    {border-style : solid ; border-color : red; margin
         : 2em; }
         		 	div.article-meta           {border-style : solid; border-width : 1 pt; margin
         1em;}
         		 	div.boxed-text             {margin : 5em; border-style : solid;}
         		 	div.contrib-group          {margin : 1em; }
         		 	div.fig                    {border-style : solid; border-width : 2px; margin :
         2em; }
         		 	div.funding                {font-weight : bold ; font-size : 16pt;}
         		 	div.given-names            {font-style : italic;}
         		 	div.intro                  {border-style : inset; margin : 5px;}
         		 	div.introduction           {border-style : inset; margin : 5px;}
         		 	div.journal-meta           {border-style : solid; border-width : 1 pt; margin
         1em;}
         		 	div.journal-title-group    {background : #ffeeee;}
         		 	div.article-title          {font-weight : bold ; font-size : 18pt;}
         		 	div.kwd                    {font-style : italic}
         		 	div.meta-name              {font-weight : bold ; font-size : 16pt;}
         		 	div.name                   {font-weight : bold;}
         		 	div.alt-title              {font-style : italic; font-size : 12px;}
         		 	
         		 	div.sec                    {border : 2px; margin 5px; padding 2px;}
         		 	div.title                  {font-family : courier; font-weight : bold;
         		 	                            font-size : 16pt; margin : 5px;}
         		 	
         		 	div.materials_methods      {border-style : double; margin : 5px; }
         		 	div.methods                {border-style : double; margin : 5px; }
         		 	
         		 	div.results                {border-style : solid; margin : 5px;}
         		 	div.background             {border-style : dotted; margin : 5px;}
         		 	
         		 	div.discussion             {border-style : groove; margin : 5px;}
         		 	div.conclusion             {border-style : ridge; margin : 5px;}
         		 	div.conclusions             {border-style : ridge; margin : 5px;}
         		 	div.supplementary-material {border-style : inset; margin : 5px;}
         		 	div.abbreviations          {border-style : double; border-color : red; margin
         : 5px;}
         		 	div.competinginterests     {border-style : double; border-color : blue; margin
         : 5px;}
         		 	div.acknowledgements       {border-style : double; border-color : green; margin
         : 5px;}
         		 	div.authors_contributions   {border-style : double; border-color : purple; margin
         : 5px;}
         		 	
         		 	div.publisher              {border-style : outset; margin : 5px;}
         		 	div.fn-type-conflict       {background : #f88; }
         		 	div.fn-type-con            {background : #ddf; }
         		 	div.fn-type-other          {background : #ddd; }
         		 	
         		 	div.unknown                {background : #ffd;
         									 	  border-style : solid;
         									 	  border-width : 1px;
         									 	  padding : 2 px;}
         		 	  
         	    table                      {background : #ffffdd;}
         		 	tr                         {background : #ddddff; padding : 1px;}
         		 	
         		 	span                       {background : #ffcccc;}
         		 	
         		 	span.citation-author       {font-family : helvetica; background : #ffeeee;}
         		 	span.collab                {background : #ddffff; }
         		 	span.comment               {font-family : courier; font-size : 6px; background
         : #ffaaff;}
         		 	span.contrib               {background : #ffffff;}
         		 	span.corresp               {background : #ddffdd; }
         		 	span.doi                   {background : #ffffff;}
         		 	span.email                 {font-family : courier; }
         		 	span.etal                  {font-style : italic;}
         		 	span.fpage                 {font-family : courier;}
         		 	span.given-names           {background : #ffffff;}
         		 	span.iso-abbrev            {background : #ffffff;}
         		 	span.issn-epub             {background : #ffffff;}
         		 	span.issn-ppub             {background : #ffffff;}
         		 	span.journal-title         {background : #ffffff;}
         		 	span.lpage                 {font-family : courier;}
         		 	span.mixed-article-title   {font-style : italic ;}
         		 	span.nlm-ta                {background : #ffffff;}
         		 	span.pmc                   {background : #ffffff;}
         		 	span.pmcid                 {background : #ffffff;}
         		 	span.pmid                  {background : #ffffff;}
         		 	span.publisher             {background : #ffffff;}
         		 	span.publisher-id          {background : #ffffff;}
         		 	span.publisher-name        {background : #ffffff;}
         		 	span.source                {background : #ffffff;}
         		 	span.subject               {background : #ffffff;}
         		 	span.surname               {background : #ffffff;}
         		 	span.volume                {font-family : courier; font-weight : bold;}
         		 	span.year                  {font-family : courier ; font-style : italic;}
         			
      </style> </head> <body> <div class="front" title="front"> <div class="journal-meta" tagx="journal-meta" title="journal-meta"> <span class="nlm-ta" title="nlm-ta">Nucleic Acids Res</span> <span class="iso-abbrev" title="iso-abbrev">Nucleic Acids Res</span> <span class="publisher-id" title="publisher-id">nar</span> <div class="journal-title-group" tagx="journal-title-group" title="journal-title-group"> <span class="journal-title" tagx="journal-title" title="journal-title">Nucleic Acids Research</span> </div> <span class="issn-ppub" tagx="issn" title="issn-ppub">0305-1048</span> <span class="issn-epub" tagx="issn" title="issn-epub">1362-4962</span> <div class="publisher" tagx="publisher" title="publisher"> <span class="publisher-name" tagx="publisher-name" title="publisher-name">Oxford University Press</span> </div> </div> <div class="article-meta" tagx="article-meta" title="article-meta"> <span class="pmcid" title="pmcid"> pmcid: <a href="http://www.ncbi.nlm.nih.gov/pubmed/5737085">5737085</a> </span> <span class="pmid" title="pmid"> pmid: <a href="http://www.ncbi.nlm.nih.gov/pubmed/28977480">28977480</a> </span> <span class="doi" title="doi"> doi: <a href="https://dx.doi.org/10.1093/nar/gkx745">10.1093/nar/gkx745</a> </span> <span class="publisher-id" title="publisher-id">gkx745</span> <div class="article-categories" title="article-categories">
: <span class="subject" title="subject">Nucleic Acid Enzymes</span> </div> <div class="title-group"> <div class="article-title" title="article-title">
Plant organellar DNA primase-helicase synthesizes RNA primers for organellar DNA polymerases using a unique recognition sequence </div> </div> <div class="contrib-group" title="contrib-group"> <span class="contrib" title="contrib"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Peralta-Castro</span> <span class="given-names" tagx="given-names" title="given-names">Antolín</span> </span> <a href="#AFF1" /> </span> <span class="contrib" title="contrib"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Baruch-Torres</span> <span class="given-names" tagx="given-names" title="given-names">Noe</span> </span> <a href="#AFF1" /> </span> <span class="contrib" title="contrib"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Brieba</span> <span class="given-names" tagx="given-names" title="given-names">Luis G.</span> </span> <a href="#AFF1" /> <a href="#COR1" /> </span> </div> <span class="citation_author_institution" id="AFF1">[], </span> <div class="author-notes" title="author-notes"> <div class="corresp" title="corresp"> <span class="label" tagx="label" title="label">*</span>To whom correspondence should be addressed. Tel: +52 462 166 3007; Fax: +52 462 624 5846; Email: <span class="email" tagx="email" title="email">luis.brieba@cinvestav.mx</span> </div> </div> <span class="pub-date-ppub" title="pub-date-ppub">ppub: <span>2017-10-10</span> </span> <span class="pub-date-epub" title="pub-date-epub">epub: <span>2017-8-8</span> </span> <span class="pub-date-pmc-release" title="pub-date-pmc-release">pmc-release: <span>2017-8-8</span> </span> <span class="volume" tagx="volume" title="volume">45</span> <span class="issue" tagx="issue" title="issue">18</span> <span class="fpage" tagx="fpage" title="fpage">10764</span> <span class="lpage" tagx="lpage" title="lpage">10774</span> <span class="history" title="history"> <span class="accepted" title="accepted">accepted: 2017-8-24</span> <span class="rev-recd" title="rev-recd">rev-recd: 2017-8-09</span> <span class="received" title="received">received: 2017-6-01</span> </span> <div class="permissions"> <span class="copyright" title="copyright">(C) , <span class="copyright-year" tagx="copyright-year" title="copyright-year">2017</span> </span> <span class="license" title="license"> <span class="license-p" title="license-p">This is an Open Access article distributed under the terms of the Creative Commons Attribution License ( <span class="uri" title="uri"> <a href="http://creativecommons.org/licenses/by-nc/4.0/">http://creativecommons.org/licenses/by-nc/4.0/</a> </span>), which permits non-commercial re-use, distribution, and reproduction in any medium, provided the original work is properly cited. For commercial re-use, please contact <span class="email" tagx="email" title="email">journals.permissions@oup.com</span> </span> </span> </div> <a class="self-uri" href="gkx745.pdf" title="self-uri">gkx745.pdf</a> <span class="related-article" tagx="related-article" title="related-article">10.1093/nar/gkx744</span> <div class="abstract" title="abstract"> <div class="abstract-title" title="abstract-title">
Abstract</div> <div class="title" tagx="title" title="title">
Abstract</div> <p>DNA primases recognize single-stranded DNA (ssDNA) sequences to synthesize RNA primers during lagging-strand replication. <i>Arabidopsis thaliana</i> encodes an ortholog of the DNA primase-helicase from bacteriophage T7, dubbed AtTwinkle, that localizes in chloroplasts and mitochondria. Herein, we report that AtTwinkle synthesizes RNA primers from a 5′-(G/C)G <div class="u_UNKNOWN">GA</div>-3′ template sequence. Within this sequence, the underlined nucleotides are cryptic, meaning that they are essential for template recognition but are not instructional during RNA synthesis. Thus, in contrast to all primases characterized to date, the sequence recognized by AtTwinkle requires two nucleotides (5′- <div class="u_UNKNOWN">GA</div>-3′) as a cryptic element. The divergent zinc finger binding domain (ZBD) of the primase module of AtTwinkle may be responsible for template sequence recognition. During oligoribonucleotide synthesis, AtTwinkle shows a strong preference for rCTP as its initial ribonucleotide and a moderate preference for rGMP or rCMP incorporation during elongation. RNA products synthetized by AtTwinkle are efficiently used as primers for plant organellar DNA polymerases. In sum, our data strongly suggest that AtTwinkle primes organellar DNA polymerases during lagging strand synthesis in plant mitochondria and chloroplast following a primase-mediated mechanism. This mechanism contrasts to lagging-strand DNA replication in metazoan mitochondria, in which transcripts synthesized by mitochondrial RNA polymerase prime mitochondrial DNA polymerase γ. </p> </div> <div class="counts" title="counts"> <span class="page-count" title="page-count">page-count: </span> </div> </div> </div> <div class="body" title="body"> <div class="intro" title="intro"> <div class="title" tagx="title" title="title">
INTRODUCTION</div> <p>Organelles are products of early bacterial endosymbiotic events. Phylogenetic analyses, however, reveal that the gene products responsible for transcription and replication in human and yeast mitochondria share an evolutionary origin with the replication apparatus of T-odd bacteriophages ( <a href="#B1">1</a>– <a href="#B3">3</a>). In the T7 replisome, three bacteriophage-encoded proteins and one host protein coordinate DNA synthesis: T7 DNA polymerase (gp5), T7 primase-helicase (T7 primase-helicase or gp4), single-stranded DNA binding protein (T7 SSB or gp2.5) ( <a href="#B4">4</a>, <a href="#B5">5</a>) and <i>Escherichia coli</i>′s thioredoxin. T7 primase-helicase is a modular protein that includes a primase module that synthesizes primers for the lagging-strand T7 DNA polymerase and a helicase module that processively unwinds dsDNA ( <a href="#B4">4</a>, <a href="#B6">6</a>, <a href="#B7">7</a>). T7 primase belongs to the family of prokaryotic primases that is composed of a RNA polymerization domain (RPD) that catalyzes ribonucleotide addition and a zinc-binding domain (ZBD) involved in ssDNA template recognition ( <a href="#B8">8</a>, <a href="#B9">9</a>). Organellar proteins derived from T7-odd bacteriophages have undergone evolutionary changes with respect to the ancestral bacteriophage proteins ( <a href="#B10">10</a>– <a href="#B12">12</a>). An example of these changes occurs in the metazoan replicative mitochondrial DNA helicase TWINKLE (T7 gp4-like protein with intra-mitochondrial nucleoid localization). Metazoan TWINKLE lacks the cysteines necessary for zinc coordination in the region homologous to the ZBD of T7 primase and the conserved residues in the RPD-like region required for RNA synthesis ( <a href="#B13">13</a>, <a href="#B14">14</a>). In metazoans, transcripts generated by mitochondrial RNA polymerase are used as primers during leader and lagging strand-synthesis ( <a href="#B14">14</a>). Furthermore, fungi are devoid of a TWINKLE homologue, and short RNA transcripts synthesized by mitochondrial RNA polymerases may be used as primers ( <a href="#B10">10</a>, <a href="#B15">15</a>– <a href="#B18">18</a>). </p> <p>Prokaryotic primases recognize a variety of template sequences. For example, bacterial primases from <i>Staphylococcus aeurus, Aquifex aeolicus, E. coli</i>, and primases-helicases from bacteriophages T4, T7 and SP6 recognize 5′-CTA-3′, 5′-CCC-3′, 5′-CTG-3′, 5′-GTT-3′, 5′-GTC-3′ and 5′-GCA-3′, respectively, as ssDNA template sequences ( <a href="#B19">19</a>– <a href="#B25">25</a>). In a primase, the identity of the middle base of a minimal trinucleotide recognition sequence determines the identity of the initial or 5′ NTP, and the 5′ base determines 3′ NMP incorporation specificity ( <a href="#B9">9</a>, <a href="#B26">26</a>). Despite the diversity in template recognition sequences, in all primases studied to date the 3′-nucleotide of the initiation triplet sequence is not transcribed. It is required, however, for template recognition, and for this reason it is called cryptic. </p> <p>Seminal work predicted that TWINKLE harbors both primase and helicase activities in most eukaryotes, except in metazoans where this enzyme is predicted only have helicase acitivity ( <a href="#B2">2</a>, <a href="#B27">27</a>). The nuclear-encoded TWINKLE homolog from the flowering plant <i>Arabidopsis thaliana</i> (dubbed AtTwinkle) is a 709-residue protein that is translocated into mitochondria and chloroplasts owing to its dual N-terminal targeting sequence ( <a href="#B28">28</a>– <a href="#B32">32</a>). The module with a predicted primase activity contains the six conserved regions present in prokaryotic primases ( <a href="#B2">2</a>, <a href="#B27">27</a>). Motif I corresponds to the ZBD necessary for template recognition ( <a href="#B26">26</a>), whereas regions II to VI are located in the RNAP domain. <i>In vitro</i> characterization of AtTwinkle demonstrated that this enzyme has both primase and helicase activities ( <a href="#B28">28</a>, <a href="#B29">29</a>, <a href="#B32">32</a>). However, its preferred ssDNA template sequence, its putative role in organellar replication, and its interactions with plant organellar DNA polymerases remain unclear. </p> <p>Herein, we found that AtTwinkle requires for template recognition a ssDNA template sequence that consists of two cryptic nucleotides and that RNA products generated by AtTwinkle are used as primers for plant organellar DNA polymerases. Our data suggests that the plant organellar replisome functionally resembles the replisome of T-odd bacteriophages. </p> </div> <div class="materials_methods" title="materials|methods"> <div class="title" tagx="title" title="title">
MATERIALS AND METHODS</div> <div class="expressionandpurificationofrecombinantproteins" title="sec"> <div class="title" tagx="title" title="title">
Expression and purification of recombinant proteins</div> <p>The open reading frame corresponding to the predicted processed AtTwinkle (residues 92–709) was codon optimized for bacterial expression and cloned into a pUC19 vector. Open reading frames of AtPrimase-Helicase and AtPrimase were PCR amplified using the cloning vector as a template. The primase domain of bacteriophage T7 primase-helicase (residues 1–262) was PCR amplified from genomic T7 DNA. AtPrimase-Helicase (residues 92–709) and AtPrimase (residues 92–410) gene products were subcloned into a modified pET19 vector between NdeI and BamHI restriction sites, whereas T7 primase was cloned into a pET11 vector. The modified pET19 vector adds a nine-histidine tag at the N-terminus of each construct before the cleavage site of the human rhinovirus 3C protease (PreScission protease, GE Healthcare) allowing the histidine tag to be removed by specific proteolysis. Protein expression of AtPrimase-helicase, AtPrimase, and T7 primase were carried out in an <i>E. coli</i> BL21(DE3) strain at 21°C in Luria Bertani medium supplemented with 0.1 mM ZnCl <sub>2</sub>. After cell cultures reached an OD <sub>600</sub> between 0.6 to 0.8, protein expression was induced by adding 1 mM IPTG and the cell cultures were incubated for 12 h. Cell cultures were harvested by centrifugation and resuspended in 30 ml of buffer A (20 mM Tris–HCl pH 7.5 and 150 mM NaCl), and freeze-thawed three times in the presence of lysozyme (0.2 mg/ml). Clear bacterial lysate was collected by centrifugation and loaded onto a Ni-NTA column by gravity. Proteins were subsequently eluted from the column with buffer A containing 0, 50, 100, 250 or 500 mM imidazole. Protein containing fractions were dialyzed in 20 mM Tris–HCl pH 7.5, 25 mM NaCl and 10% glycerol. Proteins then were loaded in a HiTrap HP heparin column and washed with a linear gradient from 0 to 1000 mM of NaCl. Fractions containing pure protein were pooled and concentrated. Protein samples were further purified by gel filtration, using a Superdex 75 10/300 for AtPrimase and a Superdex 200 10/300 for the AtPrimase-Helicase. Both gel filtration columns were equilibrated in 20 mM Tris–HCl pH 7.5, 25 mM NaCl and 10% glycerol. T7 primase was expressed and purified as previously described ( <a href="#B33">33</a>). </p> <p>To determine the molar amount of oligoribonucleotide synthesis product by T7 and Arabidopsis primase, the radioactive [α- <sup>32</sup>P]-ATP substrate and the oligonucleotide RNAs were quantified with ImageQuant. The processed intensity was subjected to the following equation, previously derived by the Patel group ( <a href="#B34">34</a>): </p><div class="disp-formula" title="disp-formula"> <div class="tex" title="tex"> <div class="tex-math_UNKNOWN" id="M1">
\documentclass[12pt]{minimal} \usepackage{amsmath} \usepackage{wasysym} \usepackage{amsfonts} \usepackage{amssymb} \usepackage{amsbsy} \usepackage{upgreek} \usepackage{mathrsfs} \setlength{\oddsidemargin}{-69pt} \begin{document} }{}\begin{equation*}[{\rm{RNA}}\;{\rm{ribonucleotide}}(\mu {\rm{M}})] = ({\rm{R}}/({\rm{R}} + {\rm{A}})*([{\rm{ATP}}])(\mu {\rm{M}})\end{equation*}\end{document} </div> </div> </div>where R refers to the band intensity observed for each ribonucleotide, A is the intensity of the radioactively label ATP substrate and [ATP] refers to the total molar concentration of ATP (labeled and unlabeled). The molar amount of RNA products is obtained by dividing the number of ATPs in the RNA chain. <p /> </div> <div class="template-directedoligoribonucleotidesynthesis" title="sec"> <div class="title" tagx="title" title="title">
Template-directed oligoribonucleotide synthesis</div> <p>Primase reactions were assayed in a buffer containing 40 mM Tris–HCl pH 7.5, 50 mM potassium glutamate, 10 mM MgCl <sub>2</sub> and 10 mM DTT. Primase reactions contained 100 μM of the indicated NTPs and 10 μCi of [α- <sup>32</sup>P]-NTP (3,000 Ci/mM). After incubation at 30°C for 60 min, reactions were terminated by 15 mM of EDTA. Loading buffer (95% formamide, 0.1% xylene cyanol) was immediately added to the reaction and the products were separated on a 27% denaturing polyacrylamide gel containing 3 M urea. Each primase reaction contained varied amounts of recombinant protein or single-stranded DNA template as indicated in the figure legends. The ribonucleotide products were analyzed by phosphorimaging on a Storm 860 PhosphorImager. </p> </div> <div class="phylogeneticanalysisandmolecularmodeling" title="sec"> <div class="title" tagx="title" title="title">
Phylogenetic analysis and molecular modeling</div> <p>The phylogenetic tree was constructed using the amino acid residue sequences of enzymes related to T7 primase-helicase. The amino acid residue sequences were aligned in Clustal Omega. This sequence alignment was used to construct a dendogram with the Neighbor-Joining method of the Molecular Evolutionary Genetic Analysis (MEGA) software. The robustness of the dendogram was assessed by a bootstrap analysis of 1000 replicates. To build the structural model of AtPrimase, its amino acid sequence (residues 98 to 402) was aligned with the corresponding amino acid sequence present in the bacteriophage T7 primase crystal structure (PDB ID: 1NUI) ( <a href="#B8">8</a>) and a homology model was constructed based on this alignment. A structural alignment was carried out using the align module of the Molecular Operating Environment (MOE). Ten models were generated and each was minimized using the CHARMM27 force field. </p> </div> </div> <div class="results" title="results"> <div class="title" tagx="title" title="title">
RESULTS</div> <div class="attwinkleisanevolutionarilyconservedprokaryoticprimase-helicase" title="sec"> <div class="title" tagx="title" title="title">
AtTwinkle is an evolutionarily conserved prokaryotic primase-helicase</div> <p>Computational analysis predicts that the nuclear-encoded AtTwinkle is processed at residue 91 during its translocation into organelles ( <a href="#B28">28</a>). The residues corresponding to the predicted processed AtTwinkle have only 22% amino acid identity with T7 DNA primase-helicase. Despite this limited amino acid identity, seminal work by Shutt and Gray ( <a href="#B1">1</a>, <a href="#B2">2</a>) identified that green plants and algae contain the four conserved cysteines predicted to coordinate a Zn <sup>2+</sup> atom and the catalytic amino acids for RNA polymerase and DNA helicase activities (Figure <a href="#F1">1A</a> and <a href="#sup1">Supplementary Figures S1 and S2</a>). A phylogenetic analysis using primase-helicases from bacteriophage, plants, and metazoans indicates that plant primase-helicases are closely related to T-odd primase-helicases (Figure <a href="#F1">1B</a>). Furthermore, metazoan TWINKLEs lost the cysteine residues involved in metal coordination and present amino acid substitutions of the putative catalytic residues of the RPD domain (Figure <a href="#F1">1A</a> and <a href="#F1">C</a>) ( <a href="#B2">2</a>). The main differences between the conserved motifs of green plants and T-odd bacteriophage primase-helicases reside in the length and sequence of their ZBD. In green plants, the ZBD is on average 10 amino acids longer than primase-helicases from T-odd bacteriophages (Figure <a href="#F1">1C</a>). A homology model of the primase module of AtTwinkle, in comparison to the crystal structure of T7 primase, which includes the ZBD and RPD domains ( <a href="#B8">8</a>, <a href="#B35">35</a>, <a href="#B36">36</a>), predicts that the primase domain of AtTwinkle would follow similar structural arrangements to T7 primase-helicase and illustrates the possible coordination of the Zn <sup>2+</sup> atom by the conserved cysteines in its ZBD (Figure <a href="#F1">1D</a>). </p> <div class="fig" title="fig"> <div class="figure" title="figure"> <a href="http://doi.org/"> <span class="label" tagx="label" title="label">Figure 1.</span> </a> </div>   <p>AtTwinkle is a homolog of bacteriophage T7 primase-helicase and mitochondrial Twinkle. ( <b>A</b>) Schematic representation of the bifunctional T7 primase-helicase in comparison to <i>Arabidopsis</i> and human Twinkles. T7 primase-helicase contains the six conserved motifs necessary for primase activity. Motif I is located at the ZBD and motifs II to VI in the RNAP domain. AtTwinkle conserves these motifs and contain an extra N-terminal amino acid sequence (residues 1–91) that harbors a dual organellar localization signal. Human Twinkle contains an N-terminal mitochondrial targeting sequence but lacks functional ZBD and RNAP domains. ( <b>B</b>) Phylogenetic analysis between phage, plant and metazoan primase-helicases. The phylogenetic tree shows that plant and phage primase-helicases share a closer evolutionary relationship than plant and metazoan Twinkles. ( <b>C</b>) Logo sequence of the amino acid region corresponding to the ZBD. The main differences between T7 Primase and AtTwinkle are the length of the ZBD that in AtTwinkle is ∼15 amino acids longer. ( <b>D</b>) Computational homology model of the primase domain of AtPrimase-Helicase. The zinc finger is colored in blue and the RNAP domain in red. The conserved cysteines that coordinate the zinc atom are in a ball-stick representation. </p>   </div> </div> <div class="recombinantattwinklepresentslowactivityonat7templaterecognitionsequence" title="sec"> <div class="title" tagx="title" title="title">
Recombinant AtTwinkle presents low activity on a T7 template recognition sequence</div> <p>In order to biochemically elucidate the ssDNA template recognition sequence of AtTwinkle, we codon-optimized its open reading frame for bacterial protein expression. We designed a construct that encodes for the primase and helicase modules after the predicted organellar targeting sequence (residues 92 to 709) ( <a href="#B28">28</a>) and a construct containing only the primase module (residues 92–410). Herein, these protein fragments of AtTwinkle are dubbed AtPrimase-Helicase and AtPrimase respectively. Both recombinant proteins were purified to homogeneity after three chromatographic steps, with similar purity as a recombinant primase module (residues 1–262) from the T7 primase-helicase gene used as a control during this study (Figure <a href="#F2">2A</a>). Previous biochemical characterizations of AtTwinkle indicated that this protein is active on single-stranded M13 templates ( <a href="#B28">28</a>, <a href="#B29">29</a>, <a href="#B32">32</a>). To assess if our purified proteins exhibit oligoribonucleotide synthesis activity we used a random heptaoligonucleotide as a template and assayed AtPrimase and T7 primase fragments for ribonucleotide synthesis. We used a random oligonucleotide to avoid bias in the different combinations of template sequences that may occur in the fixed nucleotide sequence of the M13 template. The most abundant T7 primase product from its minimal recognition sequence 5′-GTC-3′ is the 5′-pppAC diribonucleotide ( <a href="#B26">26</a>). In this minimal recognition template, the 3′ cytidine is cryptic, the middle thymidine drives the initial binding of the 5′ ATP, and the 5′ guanosine templates for 3′ CMP incorporation ( <a href="#B26">26</a>). T7 primase generated a product that migrates with the same molecular mass when the reactions were labeled with [α- <b> <sup>32</sup> </b>P] <b>-</b>CTP or [α- <b> <sup>32</sup> </b>P]-ATP, suggesting that this product corresponds to the 5′-pppAC dinucleotide (Figure <a href="#F2">2B</a>, lanes 7–10). On the basis of a control experiment using a synthetic template containing the 5′-GTC-3′ sequence we inferred that the main T7 primase product present in the random heptamer is indeed the 5′-pppAC dinucleotide (data not shown). A similar experiment was executed with recombinant AtPrimase but with the addition of [α- <b> <sup>32</sup> </b>P] <b>-</b>CTP, [α- <b> <sup>32</sup> </b>P]-ATP or [α- <b> <sup>32</sup> </b>P]-GTP as labeled ribonucleotides. In the presence of radioactive ATP only a faint product band is observed in comparison to the control reaction without added AtPrimase (Figure <a href="#F2">2B</a>, lanes 1 and 2). However, in the presence of radioactively labeled CTP or GTP, AtPrimase catalyzed the synthesis of oligoribonucleotides longer than five ribonucleotides. A strong signal corresponding to the migration of a diribonucleotide is observed only when CTP is used as the radioactive label, suggesting that the identity of this product is 5′-pppCC (Figure <a href="#F2">2B</a>, lanes 4–8). As ATP is not efficiently incorporated by AtPrimase, a template thymidine may not be part of the template sequence recognized by AtTwinkle. On the other hand, CTP and GTP are incorporated into the RNA products. In order to directly identify the initial ribonucleotides synthesized by AtPrimase-Helicase we characterized the 5′ and 3′ ribonucleotides synthesized on a single-stranded M13 template using strategies previously developed in the study of the T7 replisome ( <a href="#B37">37</a>, <a href="#B38">38</a>). We identified the 3′-ribonucleotide synthesized by AtPrimase-Helicase by measuring the incorporation of radioactive ribonucleotide products in individual reactions using each of the four [α- <sup>32</sup>P]-NTPs (ATP, UTP, CTP and GTP) in the presence of the other three non-radioactive NTPs and posterior phosphatase treatment. Since phosphatase removes the 5′-triphosphate the only radioactive product would correspond to the second or third phosphodiester bond. In this experiment, strong product bands were observed only in the presence of [α- <b> <sup>32</sup> </b>P]-CTP, indicating that CMP is the main ribonucleotide in the 3′ position (Figure <a href="#F2">2C</a>, lane 3). The elucidation of CMP as the preferred nucleotide in the 3′ position allows the identification of the 5′ ribonucleotide by a primer extension reaction in the presence of each of the four non-radioactively labeled NTP in the presence of [α- <b> <sup>32</sup> </b>P]-CTP. To our surprise, radioactively labeled products were observed in the presence of each of the four NTPs (Figure <a href="#F2">2D</a>), suggesting that when CTP is incorporated in the 3′ position AtTwinkle is non-selective for the identity of the 5′ ribonucleotide. </p> <div class="fig" title="fig"> <div class="figure" title="figure"> <a href="http://doi.org/"> <span class="label" tagx="label" title="label">Figure 2.</span> </a> </div>   <p>Recombinant AtTwinkle is an active primase. ( <b>A</b>) Heterologous purification of recombinant primases showing the purified proteins after three chromatographic steps. The expected theoretical molecular masses for AtPrimase, AtPrimase-Helicase and T7 Primase are 36, 70, and 28 kDa, respectively. Purified protein samples were run onto a 12.5% SDS-polyacrylamide gel and stained with Coomassie Brilliant Blue. The proteins migrate near their expected theoretical molecular mass. ( <b>B</b>) Oligoribonucleotide synthesis in a random heptameric template using [α- <sup>32</sup>P]-ATP, [α- <sup>32</sup>P]-CTP and [α- <sup>32</sup>P]-GTP by AtPrimase in comparison to a synthesis reaction without added primase. The appearance of ribonucleotide products (lanes 4 and 6) in comparison to a T7 primase control (lanes 8 and 10) indicate that AtTwinkle is an active primase. ( <b>C</b>) Identification of the 3′ nucleotide synthesized by AtPrimase-Helicase in M13 ssDNA template. The identity of each of the four labeled [α- <sup>32</sup>P]-NTPs is indicated. In this experiment the rest of the unlabeled rNTPs were included in the reaction. After an incubation of 30 min the products were treated with alkaline phosphatase to eliminate the label corresponding to the 5′ nucleotide. ( <b>D</b>) Identification of the 5′ nucleotide synthesized by AtPrimase-Helicase when CTP is the preferred nucleotide in the 3′ position. In this experiment individual unlabeled NTPs were incubated in the presence of [α- <sup>32</sup>P]-CTP. The products were treated with alkaline phosphatase and run on a 27% sequencing gel. </p>   </div> </div> <div class="atprimase-helicasesynthetizesribonucleotideslongerthan4ntsin4outof64trinucleotidesequences" title="sec"> <div class="title" tagx="title" title="title">
AtPrimase-Helicase synthetizes ribonucleotides longer than 4 nts in 4 out of 64 trinucleotide sequences </div> <p>To precisely elucidate the trinucleotide recognition sequence of AtTwinkle, we used an array of 64 oligonucleotides corresponding to all possible trinucleotide combinations on primer extension reactions. We assayed oligoribonucleotide synthesis reactions labeled with either [α- <b> <sup>32</sup> </b>P]-ATP, [α- <b> <sup>32</sup> </b>P]-CTP, [α- <b> <sup>32</sup> </b>P]-GTP or [α- <b> <sup>32</sup> </b>P]-UTP using the AtPrimase-Helicase construct ( <a href="#sup1">Supplementary Figures S3–S6</a>). These experiments show that AtPrimase-Helicase efficiently uses 5′-GGA-3′ as the preferred recognition sequence during oligoribonucleotide synthesis ( <a href="#sup1">Supplementary Figures S3 and S4</a>, lane 11). A condensed summary of these experiments is shown in Figure <a href="#F3">3</a>. In this experiment, the base in middle of the random trinucleotide sequence serves as a template for the labeled [α- <b> <sup>32</sup> </b>P]-NTP (Figure <a href="#F3">3</a>, <a href="#sup1">Supplementary Figure S7</a>). This experiment shows that AtPrimase-Helicase abundantly synthesizes 2-mers in 42 out of 64 possible trinucleotide combinations. However AtPrimase-Helicase predominantly uses the template sequence 5′-GGA-3′ to synthesize products longer than four oligoribonucleotides (Figure <a href="#F3">3B</a>, lane 3) and to a lesser extent the 5′-GTA-3′ and 5′-CCC-3′ sequences (Figure <a href="#F3">3A</a>, lane 3 and Figure <a href="#F3">3C</a>, lane 6, Figure <a href="#F3">3E</a> and <a href="#sup1">Supplementary Figure S7</a>). Reaction products corresponding to 5′-pppAC (Figure <a href="#F3">3A</a>, lanes 3 and 7), pppCC (Figure <a href="#F3">3B</a>, lane 7), pppGC (Figure <a href="#F3">3C</a>, lane 3, 7, 11 and 15) and pppUC (Figure <a href="#F3">3D</a>, lane 7) diribonucleotides are abundantly observed, however these ribonucleotides are not converted into longer oligomers in the majority of the substrates (Figure <a href="#F3">3A</a>, lane 2, lane 8, lanes 12–14, 16). </p> <div class="fig" title="fig"> <div class="figure" title="figure"> <a href="http://doi.org/"> <span class="label" tagx="label" title="label">Figure 3.</span> </a> </div>   <p>AtPrimase-Helicase is active in 4 out 64 trinucleotide sequences. Oligoribonucleotide synthesis reactions on an array of 64 possible trinucleotide combinations. The ribonucleotide complementary to the middle nucleotide dictates the identity of the radioactively labeled products. Reactions labeled with [α- <sup>32</sup>P]-ATP, [α- <sup>32</sup>P]-CTP, [α- <sup>32</sup>P]-GTP and [α- <sup>32</sup>P]-UTP, were run in panels ( <b>A</b>), ( <b>B</b>), ( <b>C</b>) and ( <b>D</b>) respectively. ( <b>E</b>) Relative intensity of RNA products synthetized by each of the 64 trinucleotide templates by AtTwinkle. The Y-axes indicates the identity of the trinucleotide substrate and the X-ayes present the relative intensity of each product. Graphical representation of RNA synthesis indicating the percentage of each RNA product (2-mer to 8-mer) for each individual template. If the case of empty bars the intensity for the background and the reactions were equal. Oligonucleotide templates contained seven thymidines after the assayed trinucleotide sequence. Incorporation opposite those thymidines produce ribonucleotide products longer than three nucleotides. Reactions contained 5 mM of the synthetic template and 2 mM AtPrimase-Helicase. </p>   </div> <p>A comparison of the migration of the RNA products observed in the 5′-GTA-3′ and 5′-GGA-3′ templates (Figure <a href="#F3">3A</a>, lane 3 and Figure <a href="#F3">3B</a>, lane 3) shows that their ribonucleotide products migrate with the same molecular mass (data not shown). This observation suggests that the 5′-GGA-3′ oligonucleotide directs the synthesis of a 5′-pppCA diribonucleotide that is further extended with adenines and that the 5′-GGA-3′ sequence contains two cryptic bases. A direct comparison between the RNA products synthesized by AtPrimase and T7 primase on templates containing 5′-GGA-3′ and 5′-GTA-3′ sequences shows that the synthesized ribonucleotides present the same migration pattern. This comparison also shows that AtTwinkle is an efficient DNA primase with comparable catalytic activity to T7 primase (Figure <a href="#F4">4</a>). The migration pattern of the synthesized ribonucleotides is independent of the identity of the labeled [α- <b> <sup>32</sup> </b>P]-NTP. This experiment further reveals that AtTwinkle recognizes two nucleotides, 5′-GA-3′, as a cryptic element and starts oligoribonucleotide synthesis with a 5′ CTP as its initial NTP. </p> <div class="fig" title="fig"> <div class="figure" title="figure"> <a href="http://doi.org/"> <span class="label" tagx="label" title="label">Figure 4.</span> </a> </div>   <p>Direct comparison of the ribonucleotide synthesis activity between AtPrimase and T7 Primase. A sequencing gel showing the migration of the synthesized products by AtPrimase and T7 Primase in 5′-GGA'-3 and 5′-GTC'-3 single-stranded DNA sequences. In the case of T7 primase the lower RNA migrating product corresponds to a 5′-pppAC diribonucleotide (lanes 3 and 4) that is extended up to nine ribonucleotides according to the template thymidines. In At-Primase (lanes 1 and 2) the lower migrating product corresponds to a 5′-pppCA diribonucleotide that is extended following incorporation of AMP using the template thymidines. Reaction products were specifically labeled with [α- <sup>32</sup>P]-ATP (lanes 1 and 3) or [α- <sup>32</sup>P]-CTP (lanes 2 and 4). Primase reactions were carried out using 5 μM of the synthetic template and 1 μM of AtPrimase and 0.2 μM of T7 Primase. </p>   </div> </div> <div class="attwinklerecognizestwocryptic5′-ga-3′bases" title="sec"> <div class="title" tagx="title" title="title">
AtTwinkle recognizes two cryptic 5′-GA-3′ bases</div> <p>To further corroborate the unique character of the template recognized by AtTwinkle, we compared ribonucleotide synthesis by AtPrimase-Helicase using the 5′-GGA-3′ sequence and well characterized primase recognition sequences using independently labeled ribonucleotides (Figure <a href="#F5">5</a>, lanes 5 and 6). This experiment confirms that 5′-pppCA is the initial ribonucleotide synthesized by AtPrimase-Helicase and that it is extended up to nine ribonucleotides following insertion opposite template thymidines (Figure <a href="#F5">5</a>, lanes 5–8). Similar results were observed with the AtPrimase construct (data not shown). This experiment also shows that at the T7 primase recognition sequence, AtPrimase-Helicase synthesizes a 5′-AC-3′ ribonucleotide that is not further extended (Figure <a href="#F5">5</a>, lanes 17–20). Interestingly, oligoribonucleotide synthesis using the primase recognition sequence of <i>A. aeolicus</i> (5′-CCC-3′) is only observed when [α- <b> <sup>32</sup> </b>P]-GTP is used as a label (Figure <a href="#F5">5</a>, lanes 13–16). As the 5′-CCC-3′ template sequence contains seven thymidines that are instructional for adenines, the absence of a labeled RNA product in the presence of radioactive ATP indicates that poly-rG is synthesized by an iterative process. This process resembles the iterative abortive RNA synthesis by bacterial RNAPs ( <a href="#B39">39</a>, <a href="#B40">40</a>). </p> <div class="fig" title="fig"> <div class="figure" title="figure"> <a href="http://doi.org/"> <span class="label" tagx="label" title="label">Figure 5.</span> </a> </div>   <p>AtPrimase does not recognize bacterial and bacteriophage derived template sequences. Oligoribonucleotide synthesis by AtPrimase-Helicase on synthetic templates with canonical primase recognition sequences ( <i>S. aeurus, A. aeolicus</i>, bacteriophage T7, <i>E. coli</i>) in comparison to the 5′-GGA'-3 sequence. RNA products were independently labeled with [α- <sup>32</sup>P]-ATP, [α- <sup>32</sup>P]-CTP, [α- <sup>32</sup>P]-GTP or [α- <sup>32</sup>P]-CTP. The identity of the RNA products from the 5′-pppCA'-3 diribonucleotide to the 5′-pppCAAAAAAA'-3 octaribonucleotide are indicated in the sequencing gel. Reactions contained 5 μM of the synthetic template and 2 μM of AtPrimase. </p>   </div> <p>To further corroborate that the 5′-GGA-3′ sequence contains two cryptic bases, we generated oligonucleotides in which we systematically modified the 3′ adenine and middle guanosine and performed ribonucleotide synthesis experiments with all four labeled NTPs (Figure <a href="#F6">6</a>). Substitution of either ‘cryptic’ adenine or guanosine for any other base resulted in great decrease of ribonucleotide synthesis (Figure <a href="#F6">6</a>, lanes 1–12). In oligoribonucleotide synthesis experiments in which the 3′ adenine is modified, faint radioactive bands corresponding to products longer than 4 ribonucleotides are observed in the 5′-GGG-3′ sequence solely when CTP is used as radioactive label (Figure <a href="#F6">6</a>, lane 6). The absence of radioactively labeled products in reactions labeled with ATP, indicates that AtTwinkle interacts with homopolymeric 5′-CCC-3′ or 5′-GGG-3′ sequences in a process that drives iterative poly-rG or poly-rC synthesis. This experiment implies that the identity of the 3′ adenine and middle guanosine are critical for efficient ribonucleotide synthesis. </p> <div class="fig" title="fig"> <div class="figure" title="figure"> <a href="http://doi.org/"> <span class="label" tagx="label" title="label">Figure 6.</span> </a> </div>   <p>Template recognition requires the cryptic 5′-GA'-3 pair. The identity of the ‘cryptic’ guanosine or adenine in the 5′-GA'-3 pair is necessary for oligoribonucleotide synthesis. Oligoribonucleotide synthesis decreases when the cryptic adenine (lanes 1–12) or guanosine (lanes 17–28) of the 5′GA'3 pair are modified with any other deoxynucleotide. Products were independently labeled with all four radioactive ribonucleotides. A control experiment with the canonical 5′-GA'-3 pair (lanes 13–16) is present and the identity of the oligoribonucleotide products are indicated. </p>   </div> </div> <div class="attwinkledisplaylowselectivityfor3′ribonucleotideincorporationonthe5′gga3′sequence" title="sec"> <div class="title" tagx="title" title="title">
AtTwinkle display low selectivity for 3′ ribonucleotide incorporation on the 5′ GGA 3′ sequence </div> <p>Modifying the identity of the 5′-G base in the 5′-GGA-3′ sequence results in a ssDNA sequence that is a template for primer synthesis by AtTwinkle ( <a href="#sup1">Supplementary Figures S3 and S4</a>, lanes 9–12). Altogether, these results argue that the identity of the 5′ rNTP during oligoribonucleotide synthesis by AtTwinkle is CTP and that two cryptic bases are required for template recognition. To test the preference for 3′ NMP incorporation we use a 5′-XGGA-3′ synthetic template sequence and systematically changed the identity of the 5′ base to all deoxynucleotides. An examination of the extension products by AtPrimase when dCMP or dGMP are in the 5′ position of the template oligonucleotide shows that the extent of ribonucleotide synthesis is approximately the same (Figure <a href="#F7">7A</a> and <a href="#F7">B</a>). In comparison, ribonucleotide synthesis decreases when dAMP or dTMP are in 5′ position of the templating oligonucleotide. Thus, when ‘G’ is the base 5′ to the cryptic GA dinucleotide, AtTwinkle uses 5′-GGGA-3′, 5′-CGGA-3′3′, and to a lesser extent 5′-TGGA-3′ and 5′-AGGA-3′ as templates. This lack of selectivity for the 3′-incorporated ribonucleotide contrasts with the selectivity of T7 primase that preferentially binds CTP in the RNAP binding site ( <a href="#B26">26</a>). </p> <div class="fig" title="fig"> <div class="figure" title="figure"> <a href="http://doi.org/"> <span class="label" tagx="label" title="label">Figure 7.</span> </a> </div>   <p>AtTwinkle is an efficient enzyme with non-selective incorporation for the 3′ rNTP base. AtTwinkle presents a preference for CTP as the 5′-NTP or initiating nucleotide. To assess the specificity for the 3′-rNTP base, the identify to a 5′-XG <div class="u_UNKNOWN">GA</div>-‘-3 template sequence was systematically changed to all deoxynucleotides. The underlined <div class="u_UNKNOWN">GA</div> corresponds to the cryptic element and X corresponds to the identity of the deoxynucleotide that templates for the incorporation of the 3′-rNMP. Oligoribonucleotides synthesis reactions where labeled with [α- <sup>32</sup>P]-ATP ( <b>A</b>) or [α- <sup>32</sup>P]-CTP ( <b>B</b>). The identity of the template base was either adenine (lane 1), citosine (lane 2), guanosine (lane 3) or thymidine (lane 4). The identity of the ribonucleotide products is indicated in the gel </p>   </div> </div> <div class="attwinkleperformsefficientribonucleotidesynthesis" title="sec"> <div class="title" tagx="title" title="title">
AtTwinkle performs efficient ribonucleotide synthesis</div> <p>In order to directly compare the enzymatic activities of AtPrimase and T7 primase, we measured their ribonucleotide synthesis. The concentrations of AtPrimase varied from 100 to 800 nM whereas T7 primase was present at protein concentrations ranking from 6.25 to 400 nM. At a protein concentration of 6.25 nM, T7 primase synthesized approximately the same amount of a nonaribonucleotide products than AtPrimase present at a protein concentration of 800 nM on the 5′-GG <div class="u_UNKNOWN">GA</div>-3′ template (Figure <a href="#F8">8A</a>, lanes 2 and 17). This comparison establishes that T7 primase is more active that AtPrimase. However, it is interesting to note that AtPrimase, at a concentration of 100 nM, synthesizes similar amounts of a tetraribonucleotide as 200 nM of T7 Primase (Figure <a href="#F8">8A</a>, lanes 3 and15). A quantification of the synthesized oligoribonucleotides indicates that AtPrimase is a highly efficient primase for the synthesis of RNA products between three to five ribonucleotides. </p> <div class="fig" title="fig"> <div class="figure" title="figure"> <a href="http://doi.org/"> <span class="label" tagx="label" title="label">Figure 8.</span> </a> </div>   <p>AtPrimase efficiently synthesizes RNA primers for organellar DNA polymerases. ( <b>A</b>) Oligoribonucleotide synthesis by the AtPrimase in 5′-GG <div class="u_UNKNOWN">GA</div>-3′, 5′-G <div class="u_UNKNOWN">GA</div>-3′, 5′-GT <div class="u_UNKNOWN">A</div>-3′, 5′-GT <div class="u_UNKNOWN">C</div>-3′ synthetic templates (lanes 6–21) in comparison to the synthesis present by T7 Primase (lanes 1–5) in a 5′-GGGT <div class="u_UNKNOWN">C</div>'-3 recognition sequence. AtPrimase was present from 100 to 800 nM, whereas T7 primase was present at lower protein concentrations (from 6.25 to 400 nM). Each reaction was labeled with [α- <b> <sup>32</sup> </b>P]-ATP. The identity of the oligoribonucleotides is indicated in the gel. The bottom part of the figure illustrates the molar amount of oligoribonucleotide synthesis products by 100 nM of T7 and <i>Arabidopsis</i> primases in their preferred 5′-GGG <div class="u_UNKNOWN">C</div>-3′ and 5′-GG <div class="u_UNKNOWN">GA</div>-3′ ssDNA templates. To determine the molar amount of RNA, the intensity of each radioactive band was corrected with the amount of AMPs present in each ribonucleotide. ( <b>B</b>) AtPrimase primes organellar DNA polymerases. Coupled primase-DNA polymerase reactions labeled with [α- <b> <sup>32</sup> </b>P]-dATP. RNA ladder generated by T7 Primase labeled with [α- <b> <sup>32</sup> </b>P]-ATP in the absence of DNA polymerases used as molecular weight markers. (lane 1). AtPolIA and AtPolIB, generate DNA:RNA products that correspond to the expected migration of a 19-mer in the presence of AtPrimase and unlabeled NTPs (lanes 2–4 and 5–7). In contrast the Klenow fragment of <i>E. coli</i> DNA polymerase I (lanes 8–10) and T7 DNA polymerase (lanes 11–13) are unable to perform coupled DNA synthesis. Coupled primase-DNA polymerase reactions were labeled with [α- <b> <sup>32</sup> </b>P]-dATP. AtPrim was present at 150 nM, whereas DNA polymerases were used to 75 and 150 nM. The template sequence used in the coupled primase-polymerase assay was 3′(T) <sub>10</sub> <b> <div class="u_UNKNOWN">AGGG</div> </b>GGT( <sub>9</sub>)A(T <sub>5</sub>). Within this sequence, the AtPrimase recognition sequence is underlined </p>   </div> </div> <div class="atprimaseprimesplantorganellardnapolymerases" title="sec"> <div class="title" tagx="title" title="title">
AtPrimase primes Plant Organellar DNA polymerases</div> <p> <i>Arabidopsis thaliana</i> encodes two organellar DNA polymerases able to bypass an Apurinic/apyrimidinic (AP) site (( <a href="#B41">41</a>) accompanying manuscript). These polymerases, AtPolIA and AtPolIB, are the only DNA polymerases located in plant organelles and therefore are the logical candidates to extend RNA primers synthesized by AtTwinkle at the lagging strand of the organellar replication fork ( <a href="#B42">42</a>). In order to elucidate the capabilities of AtTwinkle to synthesize primers for AtPolI's, we evaluated the efficiencies of AtPolIA and AtPolIB in comparison to <i>E. coli</i>′s Klenow fragment and T7 DNA polymerase to extend RNA primers synthesized by AtTwinkle. In this experiment, a single-stranded DNA template containing the 5′-(T) <sub>7</sub>A-(T) <sub>6</sub>GGG <div class="u_UNKNOWN">GA</div>(T) <sub>9</sub>-3′ sequence was incubated with increased concentrations of AtTwinkle and the assayed DNA polymerases in the presence of [α- <sup>32</sup>P]-dATP for 1 h. A control experiment shows that at the lowest concentrations all DNA polymerases completely extend an annealed double stranded primer-template substrate in 10 min and that the single stranded oligonucleotide is not used as a template by those polymerases (data not shown). However, in the coupled primase-polymerase assay, products were observed in reactions incubated with AtPolIA and AtPolIB (Figure <a href="#F8">8B</a>, lanes 2–7) and not with equimolar amounts of Klenow Fragment of <i>E. coli</i> DNA polymerase I or T7 DNA polymerase (Figure <a href="#F8">8B</a>, lanes–13). Our results contrast with the primer extension observed in a coupled primase-polymerase reaction with <i>E. coli</i> DNA polymerase I and AtPrimase-Helicase using single-stranded M13 as a substrate ( <a href="#B28">28</a>). Although the precise concentration of <i>E. coli</i> DNA polymerase I was not provided in that study, RNA transcripts may be used as primers if DNA polymerases are present at high concentrations. </p> </div> </div> <div class="discussion" title="discussion"> <div class="title" tagx="title" title="title">
DISCUSSION</div> <p>The organellar DNA Primase-Helicase from the plant model <i>A. thaliana</i>, AtTwinkle, recognizes a ssDNA sequence in which two cryptic nucleotides are necessary for efficient RNA synthesis. AtTwinkle effectively synthesizes long ribonucleotides on a trinucleotide 5′-GGA-3′ sequence (Figure <a href="#F3">3</a>). In which the 5′-GA-3′ pair functions as a cryptic element. Substitutions of either the cryptic 5′-G or cryptic A-3′ nucleotides and substitutions of the initial 5′-G, in the trinucleotide recognition sequence abolishes ribonucleotide synthesis ( <a href="#sup1">Supplementary Figures S3–S6</a>). </p> <p>In synthetic templates AtTwinkle efficiently synthesizes pppAC, pppUC, pppGC, and pppCC di-ribonucleotides, however these are not extended to longer RNA products (Figures <a href="#F2">2C</a> and <a href="#F3">3</a>). The apparent lack of discrimination for the 5′ position observed in a phosphodiesterase assay using a M13 ssDNA template may be due to the synthesis of these di-ribunucleotides (Figure <a href="#F2">2D</a>). In synthetic templates containing the 5′-GA-3′ cryptic sequence, AtTwinkle exhibits a strong selectivity for rCTP as the 5′ ribonucleotide. This selectivity contrasts with the relaxed selectivity for the 3′ ribonucleotide in which the identity at the 3′ elongation nucleotide can be any NTP, however CMP or GMP are preferentially incorporated (Figure <a href="#F7">7</a>). Our data reveals that the preferred ssDNA recognition sequences by AtTwinkle are 5′-GG <div class="u_UNKNOWN">GA</div>-3′ and 5′-CG <div class="u_UNKNOWN">GA</div>-3′, within these sequences the two underlined bases are cryptic nucleotides. We show that the preferred ribonucleotide products by AtTwinkle are preferentially composed of cytosines and guanines, suggesting that plant primases selectively incorporate these ribonucleotides from the A/T-rich genome of organelles ( <a href="#B43">43</a>). The preferred ribonucleotides synthesized by AtTwinkle resemble those synthesized by <i>Aquifex aeolicus</i> ( <a href="#B24">24</a>) suggesting that plants have selected thermodynamically stable base pairing during primer DNA synthesis. Although the stability conferred by these sequences that may be needed during primer hand-off to the replicating DNA polymerase, the selectivity for a G/C-rich recognition sequences also may play a role in determining the length of the Okazaki fragments during organellar replication. </p> <p>The precise mechanisms by which a ssDNA template sequence is recognized by a primase are not fully elucidated. Biochemical and structural evidence strongly suggest that an interaction between the ZBD and the RNAP domain drives ssDNA template recognition directing the synthesis of the initial di-ribonucleotide ( <a href="#B26">26</a>, <a href="#B38">38</a>, <a href="#B44">44</a>– <a href="#B46">46</a>). The ZBD residues involved in metal coordination are only composed of cysteines (Cys4 motif) in bacteriophage primases, whereas in most bacteria a histidine replaces one of these cysteines (Cys-His-Cys2 motif). The highly conserved CXXC(H)(X <sub>15–18</sub>)CXXC motif of the ZBD in prokaryotic primases folds into a zinc ribbon ( <a href="#B47">47</a>) that can be divided in two conserved CXXC(H) repeats. Within these repeats, the cysteines or histidine residues involved in Zn <sup>2+</sup> coordination are separated by two non-conserved amino acids. The most striking difference between plant and T-odd bacteriophage primases is the variation in the residues near the conserved cysteines that coordinate the Zn <sup>2+</sup> atom. In land plant primases, the first CXXC repeat is conserved, whereas the second CXXC repeat is substituted by a CX <div class="u_UNKNOWN">R</div>X <div class="u_UNKNOWN">K</div>C sequence (Figure <a href="#F9">9A</a>). Biochemical studies indicate that residues His33 and Lys70 located before the second CXXC repeat in T7 and <i>Clostridium difficile</i> primases drive ssDNA template recognition ( <a href="#B26">26</a>, <a href="#B48">48</a>). However, the corresponding amino acids in AtTwinkle (Thr161 and Asn163) are not conserved in land plants suggesting that these residues may not be involved in ssDNA template recognition (Figure <a href="#F9">9A</a> and <a href="#sup1">Supplementary Figure S2</a>). A computational model of the ZBD of AtTwinkle shows that the side chains of Arg166 and Lys168 and a conserved tryptophan residue (Trp 162) (Figure <a href="#F9">9B</a> and <a href="#sup1">Supplementary Figure S2</a>) are in similar orientations than His33 of T7 primase and Lys70 of <i>C. difficile</i> primase. This substrate-accessible orientation suggests that Arg166, Lys168, and Trp162 may play a role in ssDNA template recognition. The CXRXKC sequence is also present in primases from <i>C. botulinum</i> and bacteriophage VpV262 ( <a href="#B49">49</a>) (Figure <a href="#F9">9A</a>) suggesting that if this element plays a role in template recognition, the need for two cryptic bases would not be unique to plants. Furthermore, the nuclear genome of <i>Arabidopsis</i> encodes for an organellar protein that only contains the primase module of AtTwinkle (At1g30660 or AtDnaG). This primase also contains a sequence that resembles the CXRXKC element ( <a href="#B28">28</a>). Although structure-function studies are needed to elucidate the mechanisms that account for the need of two cryptic bases in the ssDNA template recognition sequence by AtTwinkle, it is tempting to speculate that the amino acid differences at the ZBD with prokaryotic primase-helicases may be relevant for template selectivity. </p> <div class="fig" title="fig"> <div class="figure" title="figure"> <a href="http://doi.org/"> <span class="label" tagx="label" title="label">Figure 9.</span> </a> </div>   <p>Structural suggestion for the altered template recognition by AtTwinkle. ( <b>A</b>) Multiple sequence alignment between the ZBD of T7 primase-helicase, bacterial primases, and plant Twinkles. The fours cysteines that coordinate the Zn <sup>2+</sup> atom in T7 primase can be divided in two CXXC elements. In land plant primases, the first CXXC elements is conserved, whereas the second one is substituted by a CXRXKC element. Protein are designated with abbreviations as follows: Mpo, <i>Marchantia polymorpha</i>; Nta, <i>Nicotiana tabacum</i>; Zma, <i>Zea mays</i>; Atri, <i>Amborella trichopoda</i>; Cbo, <i>Clostridium botulinum</i>; Cdi, <i>Clostridium difficile</i>; Gst, <i>geobacillus stearothermophilus</i>. ( <b>B</b>) Computational model of the zinc finger of AtTwinkle in comparison to the crystal structure of T7-Primase. The four β-strands that correspond to the zinc finger are colored in blue. The cysteine residues CXXC element are colored in magenta and red. The side chains of the amino acids in the second CXXC element are in a ball-stick representation. The Zn <sup>2+</sup> atom is colored in black. His33 implicated in template recognition in T7 primase is in a ball-stick representation. The computational model of AtTwinkle shows the proposed structural localization of residues R166 and K168. </p>   </div> <p>Furthermore, understanding the recognition sequence of AtTwinkle is a first step toward our biochemical understanding of the plant organellar replisome. We found that RNA products synthesized by AtTwinkle are efficiently used as primers for plant organellar DNA polymerases. In the analogous bacteriophage T7 replisome, T7 DNA polymerase uses short tetra-ribonucleotide synthesized by T7 primase-helicase as primers via a ‘handoff’ mechanism. This mechanism implicates a precise interaction between primase and polymerase ( <a href="#B50">50</a>, <a href="#B51">51</a>). It is interesting to note that AtTwinkle synthesizes abundant RNA products from three to five ribonucleotides, as these short oligoribonucleotides are easily dissociated from a DNA template. It is possible that organellar DNA primase and polymerase physically interact during primer extension. Although the precise length and identity of the oligo-ribonucleotide primers synthesized by AtTwinkle remain to be identified, our work demonstrates that AtTwinkle is a functional primase that selectively primes AtPolIs. The efficient synthesis of product of tetra-ribonucleotide or penta-ribonucleotides by AtTwinkle may be correlated with an RNA of four or five ribonucleotides that is used as a primer. It is possible that the extra recognition cryptic base provides a mechanism to generate specificity during primer synthesis and coordinate leader and lagging DNA synthesis. </p> <p>Our studies strongly suggest that organellar DNA replication mechanistically resembles phage T7-replication. This process contrasts with DNA replication in metazoan mitochondria in which human mitochondrial RNAP primes human mitochondrial DNA polymerase γ. In this work, we demonstrate that a plant organellar primase-helicase, phylogenetically related to bacteriophage T7 primase-helicase, efficiently primes plant organellar DNA polymerases that are phylogenetically related to bacteria, illustrating that two enzymes with different evolutionary origins coevolved to assemble the plant organellar replisome. </p> </div> <div class="supplementary-material" title="supplementary-material"> <div class="title" tagx="title" title="title">
Supplementary Material</div> <div class="local-data" title="local-data"> <span class="label" tagx="label" title="label">Supplementary Data</span> <div class="media" title="media"> <a href="gkx745_supp.pdf">LINK</a>  <p>Click here for additional data file.</p>  </div> </div> </div> </div> <div class="back" title="back"> <div class="ack" title="ack"> <div class="title" tagx="title" title="title">
ACKNOWLEDGEMENTS</div> <p>We thank Corina Diaz-Quezada for her technical support. We thank Oleg Tsodikov, Alfredo Hernandez, Teri Markow, Seung-Joo Lee and Charles Richardson for critical reading. </p> </div> <div class="supplementarydata" title="sec"> <div class="title" tagx="title" title="title">
SUPPLEMENTARY DATA</div> <p> <a href="#sup1">Supplementary Data</a> are available at NAR Online. </p> </div> <div class="funding" title="sec"> <div class="title" tagx="title" title="title">
FUNDING</div> <p>CONACYT (Ciencia Basica 2015) [253757 to L.G.B.]. Funding for open access charge: CONACYT [253757]; Cinvestav-IPN. </p> <p> <i>Conflict of interest statement</i>. None declared. </p> </div> <div class="references">
References</div> <div tag="ref-list"> <ul> <div class="title" tagx="title" title="title">
REFERENCES</div> <li tag="ref"> <a name="B1" /> <span class="label" tagx="label" title="label">1.</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Shutt</span> <span class="given-names" tagx="given-names" title="given-names">T.E.</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Gray</span> <span class="given-names" tagx="given-names" title="given-names">M.W.</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Bacteriophage origins of mitochondrial replication and transcription proteins</span>. <span class="source" tagx="source" title="source">Trends Genet.</span> <span class="year" tagx="year" title="year">2006</span>; <span class="volume" tagx="volume" title="volume">22</span>: <span class="fpage" tagx="fpage" title="fpage">90</span>– <span class="lpage" tagx="lpage" title="lpage">95</span>. <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/16364493">16364493</a> </span> </span> </li> <li tag="ref"> <a name="B2" /> <span class="label" tagx="label" title="label">2.</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Shutt</span> <span class="given-names" tagx="given-names" title="given-names">T.E.</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Gray</span> <span class="given-names" tagx="given-names" title="given-names">M.W.</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Twinkle, the mitochondrial replicative DNA helicase, is widespread in the eukaryotic radiation and may also be the mitochondrial DNA primase in most eukaryotes </span>. <span class="source" tagx="source" title="source">J. Mol. Evol.</span> <span class="year" tagx="year" title="year">2006</span>; <span class="volume" tagx="volume" title="volume">62</span>: <span class="fpage" tagx="fpage" title="fpage">588</span>– <span class="lpage" tagx="lpage" title="lpage">599</span>. <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/16612544">16612544</a> </span> </span> </li> <li tag="ref"> <a name="B3" /> <span class="label" tagx="label" title="label">3.</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Masters</span> <span class="given-names" tagx="given-names" title="given-names">B.S.</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Stohl</span> <span class="given-names" tagx="given-names" title="given-names">L.L.</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Clayton</span> <span class="given-names" tagx="given-names" title="given-names">D.A.</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Yeast mitochondrial RNA polymerase is homologous to those encoded by bacteriophages T3 and T7 </span>. <span class="source" tagx="source" title="source">Cell</span>. <span class="year" tagx="year" title="year">1987</span>; <span class="volume" tagx="volume" title="volume">51</span>: <span class="fpage" tagx="fpage" title="fpage">89</span>– <span class="lpage" tagx="lpage" title="lpage">99</span>. <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/3308116">3308116</a> </span> </span> </li> <li tag="ref"> <a name="B4" /> <span class="label" tagx="label" title="label">4.</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Hamdan</span> <span class="given-names" tagx="given-names" title="given-names">S.M.</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Richardson</span> <span class="given-names" tagx="given-names" title="given-names">C.C.</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Motors, switches, and contacts in the replisome</span>. <span class="source" tagx="source" title="source">Annu. Rev. Biochem.</span> <span class="year" tagx="year" title="year">2009</span>; <span class="volume" tagx="volume" title="volume">78</span>: <span class="fpage" tagx="fpage" title="fpage">205</span>– <span class="lpage" tagx="lpage" title="lpage">243</span>. <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/19298182">19298182</a> </span> </span> </li> <li tag="ref"> <a name="B5" /> <span class="label" tagx="label" title="label">5.</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Lee</span> <span class="given-names" tagx="given-names" title="given-names">S.J.</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Richardson</span> <span class="given-names" tagx="given-names" title="given-names">C.C.</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Choreography of bacteriophage T7 DNA replication</span>. <span class="source" tagx="source" title="source">Curr. Opin. Chem. Biol.</span> <span class="year" tagx="year" title="year">2011</span>; <span class="volume" tagx="volume" title="volume">15</span>: <span class="fpage" tagx="fpage" title="fpage">580</span>– <span class="lpage" tagx="lpage" title="lpage">586</span>. <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/21907611">21907611</a> </span> </span> </li> <li tag="ref"> <a name="B6" /> <span class="label" tagx="label" title="label">6.</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Kulczyk</span> <span class="given-names" tagx="given-names" title="given-names">A.W.</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Akabayov</span> <span class="given-names" tagx="given-names" title="given-names">B.</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Lee</span> <span class="given-names" tagx="given-names" title="given-names">S.J.</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Bostina</span> <span class="given-names" tagx="given-names" title="given-names">M.</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Berkowitz</span> <span class="given-names" tagx="given-names" title="given-names">S.A.</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Richardson</span> <span class="given-names" tagx="given-names" title="given-names">C.C.</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">An interaction between DNA polymerase and helicase is essential for the high processivity of the bacteriophage T7 replisome </span>. <span class="source" tagx="source" title="source">J. Biol. Chem.</span> <span class="year" tagx="year" title="year">2012</span>; <span class="volume" tagx="volume" title="volume">287</span>: <span class="fpage" tagx="fpage" title="fpage">39050</span>– <span class="lpage" tagx="lpage" title="lpage">39060</span>. <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/22977246">22977246</a> </span> </span> </li> <li tag="ref"> <a name="B7" /> <span class="label" tagx="label" title="label">7.</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Zhang</span> <span class="given-names" tagx="given-names" title="given-names">H.</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Lee</span> <span class="given-names" tagx="given-names" title="given-names">S.J.</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Richardson</span> <span class="given-names" tagx="given-names" title="given-names">C.C.</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Essential protein interactions within the replisome regulate DNA replication</span>. <span class="source" tagx="source" title="source">Cell Cycle</span>. <span class="year" tagx="year" title="year">2011</span>; <span class="volume" tagx="volume" title="volume">10</span>: <span class="fpage" tagx="fpage" title="fpage">3413</span>– <span class="lpage" tagx="lpage" title="lpage">3414</span>. <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/22067709">22067709</a> </span> </span> </li> <li tag="ref"> <a name="B8" /> <span class="label" tagx="label" title="label">8.</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Kato</span> <span class="given-names" tagx="given-names" title="given-names">M.</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Ito</span> <span class="given-names" tagx="given-names" title="given-names">T.</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Wagner</span> <span class="given-names" tagx="given-names" title="given-names">G.</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Richardson</span> <span class="given-names" tagx="given-names" title="given-names">C.C.</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Ellenberger</span> <span class="given-names" tagx="given-names" title="given-names">T.</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Modular architecture of the bacteriophage T7 primase couples RNA primer synthesis to DNA synthesis </span>. <span class="source" tagx="source" title="source">Mol. Cell</span>. <span class="year" tagx="year" title="year">2003</span>; <span class="volume" tagx="volume" title="volume">11</span>: <span class="fpage" tagx="fpage" title="fpage">1349</span>– <span class="lpage" tagx="lpage" title="lpage">1360</span>. <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/12769857">12769857</a> </span> </span> </li> <li tag="ref"> <a name="B9" /> <span class="label" tagx="label" title="label">9.</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Frick</span> <span class="given-names" tagx="given-names" title="given-names">D.N.</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Richardson</span> <span class="given-names" tagx="given-names" title="given-names">C.C.</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">DNA primases</span>. <span class="source" tagx="source" title="source">Annu. Rev. Biochem.</span> <span class="year" tagx="year" title="year">2001</span>; <span class="volume" tagx="volume" title="volume">70</span>: <span class="fpage" tagx="fpage" title="fpage">39</span>– <span class="lpage" tagx="lpage" title="lpage">80</span>. <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/11395402">11395402</a> </span> </span> </li> <li tag="ref"> <a name="B10" /> <span class="label" tagx="label" title="label">10.</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Sanchez-Sandoval</span> <span class="given-names" tagx="given-names" title="given-names">E.</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Diaz-Quezada</span> <span class="given-names" tagx="given-names" title="given-names">C.</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Velazquez</span> <span class="given-names" tagx="given-names" title="given-names">G.</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Arroyo-Navarro</span> <span class="given-names" tagx="given-names" title="given-names">L.F.</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Almanza-Martinez</span> <span class="given-names" tagx="given-names" title="given-names">N.</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Trasvina-Arenas</span> <span class="given-names" tagx="given-names" title="given-names">C.H.</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Brieba</span> <span class="given-names" tagx="given-names" title="given-names">L.G.</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Yeast mitochondrial RNA polymerase primes mitochondrial DNA polymerase at origins of replication and promoter sequences </span>. <span class="source" tagx="source" title="source">Mitochondrion</span>. <span class="year" tagx="year" title="year">2015</span>; <span class="volume" tagx="volume" title="volume">24</span>: <span class="fpage" tagx="fpage" title="fpage">22</span>– <span class="lpage" tagx="lpage" title="lpage">31</span>. <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/26184436">26184436</a> </span> </span> </li> <li tag="ref"> <a name="B11" /> <span class="label" tagx="label" title="label">11.</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Cupp</span> <span class="given-names" tagx="given-names" title="given-names">J.D.</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Nielsen</span> <span class="given-names" tagx="given-names" title="given-names">B.L.</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Minireview: DNA replication in plant mitochondria</span>. <span class="source" tagx="source" title="source">Mitochondrion</span>. <span class="year" tagx="year" title="year">2014</span>; <span class="volume" tagx="volume" title="volume">19</span>: <span class="fpage" tagx="fpage" title="fpage">231</span>– <span class="lpage" tagx="lpage" title="lpage">237</span>. <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/24681310">24681310</a> </span> </span> </li> <li tag="ref"> <a name="B12" /> <span class="label" tagx="label" title="label">12.</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Korhonen</span> <span class="given-names" tagx="given-names" title="given-names">J.A.</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Pham</span> <span class="given-names" tagx="given-names" title="given-names">X.H.</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Pellegrini</span> <span class="given-names" tagx="given-names" title="given-names">M.</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Falkenberg</span> <span class="given-names" tagx="given-names" title="given-names">M.</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Reconstitution of a minimal mtDNA replisome in vitro</span>. <span class="source" tagx="source" title="source">EMBO J.</span> <span class="year" tagx="year" title="year">2004</span>; <span class="volume" tagx="volume" title="volume">23</span>: <span class="fpage" tagx="fpage" title="fpage">2423</span>– <span class="lpage" tagx="lpage" title="lpage">2429</span>. <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/15167897">15167897</a> </span> </span> </li> <li tag="ref"> <a name="B13" /> <span class="label" tagx="label" title="label">13.</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Spelbrink</span> <span class="given-names" tagx="given-names" title="given-names">J.N.</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Li</span> <span class="given-names" tagx="given-names" title="given-names">F.Y.</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Tiranti</span> <span class="given-names" tagx="given-names" title="given-names">V.</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Nikali</span> <span class="given-names" tagx="given-names" title="given-names">K.</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Yuan</span> <span class="given-names" tagx="given-names" title="given-names">Q.P.</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Tariq</span> <span class="given-names" tagx="given-names" title="given-names">M.</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Wanrooij</span> <span class="given-names" tagx="given-names" title="given-names">S.</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Garrido</span> <span class="given-names" tagx="given-names" title="given-names">N.</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Comi</span> <span class="given-names" tagx="given-names" title="given-names">G.</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Morandi</span> <span class="given-names" tagx="given-names" title="given-names">L.</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Human mitochondrial DNA deletions associated with mutations in the gene encoding Twinkle, a phage T7 gene 4-like protein localized in mitochondria </span>. <span class="source" tagx="source" title="source">Nat. Genet.</span> <span class="year" tagx="year" title="year">2001</span>; <span class="volume" tagx="volume" title="volume">28</span>: <span class="fpage" tagx="fpage" title="fpage">223</span>– <span class="lpage" tagx="lpage" title="lpage">231</span>. <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/11431692">11431692</a> </span> </span> </li> <li tag="ref"> <a name="B14" /> <span class="label" tagx="label" title="label">14.</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Wanrooij</span> <span class="given-names" tagx="given-names" title="given-names">S.</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Fuste</span> <span class="given-names" tagx="given-names" title="given-names">J.M.</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Farge</span> <span class="given-names" tagx="given-names" title="given-names">G.</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Shi</span> <span class="given-names" tagx="given-names" title="given-names">Y.</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Gustafsson</span> <span class="given-names" tagx="given-names" title="given-names">C.M.</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Falkenberg</span> <span class="given-names" tagx="given-names" title="given-names">M.</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Human mitochondrial RNA polymerase primes lagging-strand DNA synthesis in vitro</span>. <span class="source" tagx="source" title="source">Proc. Natl. Acad. Sci. U.S.A.</span> <span class="year" tagx="year" title="year">2008</span>; <span class="volume" tagx="volume" title="volume">105</span>: <span class="fpage" tagx="fpage" title="fpage">11122</span>– <span class="lpage" tagx="lpage" title="lpage">11127</span>. <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/18685103">18685103</a> </span> </span> </li> <li tag="ref"> <a name="B15" /> <span class="label" tagx="label" title="label">15.</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Seow</span> <span class="given-names" tagx="given-names" title="given-names">F.</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Sato</span> <span class="given-names" tagx="given-names" title="given-names">S.</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Janssen</span> <span class="given-names" tagx="given-names" title="given-names">C.S.</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Riehle</span> <span class="given-names" tagx="given-names" title="given-names">M.O.</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Mukhopadhyay</span> <span class="given-names" tagx="given-names" title="given-names">A.</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Phillips</span> <span class="given-names" tagx="given-names" title="given-names">R.S.</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Wilson</span> <span class="given-names" tagx="given-names" title="given-names">R.J.</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Barrett</span> <span class="given-names" tagx="given-names" title="given-names">M.P.</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">The plastidic DNA replication enzyme complex of Plasmodium falciparum</span>. <span class="source" tagx="source" title="source">Mol. Biochem. Parasitol.</span> <span class="year" tagx="year" title="year">2005</span>; <span class="volume" tagx="volume" title="volume">141</span>: <span class="fpage" tagx="fpage" title="fpage">145</span>– <span class="lpage" tagx="lpage" title="lpage">153</span>. <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/15926203">15926203</a> </span> </span> </li> <li tag="ref"> <a name="B16" /> <span class="label" tagx="label" title="label">16.</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">McKinney</span> <span class="given-names" tagx="given-names" title="given-names">E.A.</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Oliveira</span> <span class="given-names" tagx="given-names" title="given-names">M.T.</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Replicating animal mitochondrial DNA</span>. <span class="source" tagx="source" title="source">Genet. Mol. Biol.</span> <span class="year" tagx="year" title="year">2013</span>; <span class="volume" tagx="volume" title="volume">36</span>: <span class="fpage" tagx="fpage" title="fpage">308</span>– <span class="lpage" tagx="lpage" title="lpage">315</span>. <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/24130435">24130435</a> </span> </span> </li> <li tag="ref"> <a name="B17" /> <span class="label" tagx="label" title="label">17.</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Ramachandran</span> <span class="given-names" tagx="given-names" title="given-names">A.</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Nandakumar</span> <span class="given-names" tagx="given-names" title="given-names">D.</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Deshpande</span> <span class="given-names" tagx="given-names" title="given-names">A.P.</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Lucas</span> <span class="given-names" tagx="given-names" title="given-names">T.P.</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">R</span> <span class="given-names" tagx="given-names" title="given-names">R.B.</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Tang</span> <span class="given-names" tagx="given-names" title="given-names">G.Q.</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Raney</span> <span class="given-names" tagx="given-names" title="given-names">K.</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Yin</span> <span class="given-names" tagx="given-names" title="given-names">Y.W.</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Patel</span> <span class="given-names" tagx="given-names" title="given-names">S.S.</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">The yeast mitochondrial RNA polymerase and transcription factor complex catalyzes efficient priming of DNA synthesis on single-stranded DNA </span>. <span class="source" tagx="source" title="source">J. Biol. Chem.</span> <span class="year" tagx="year" title="year">2016</span>; <span class="volume" tagx="volume" title="volume">291</span>: <span class="fpage" tagx="fpage" title="fpage">16828</span>– <span class="lpage" tagx="lpage" title="lpage">16839</span>. <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/27311715">27311715</a> </span> </span> </li> <li tag="ref"> <a name="B18" /> <span class="label" tagx="label" title="label">18.</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Ling</span> <span class="given-names" tagx="given-names" title="given-names">F.</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Hori</span> <span class="given-names" tagx="given-names" title="given-names">A.</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Yoshitani</span> <span class="given-names" tagx="given-names" title="given-names">A.</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Niu</span> <span class="given-names" tagx="given-names" title="given-names">R.</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Yoshida</span> <span class="given-names" tagx="given-names" title="given-names">M.</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Shibata</span> <span class="given-names" tagx="given-names" title="given-names">T.</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Din7 and Mhr1 expression levels regulate double-strand-break-induced replication and recombination of mtDNA at ori5 in yeast </span>. <span class="source" tagx="source" title="source">Nucleic Acids Res.</span> <span class="year" tagx="year" title="year">2013</span>; <span class="volume" tagx="volume" title="volume">41</span>: <span class="fpage" tagx="fpage" title="fpage">5799</span>– <span class="lpage" tagx="lpage" title="lpage">5816</span>. <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/23598996">23598996</a> </span> </span> </li> <li tag="ref"> <a name="B19" /> <span class="label" tagx="label" title="label">19.</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Tseng</span> <span class="given-names" tagx="given-names" title="given-names">T.Y.</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Frick</span> <span class="given-names" tagx="given-names" title="given-names">D.N.</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Richardson</span> <span class="given-names" tagx="given-names" title="given-names">C.C.</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Characterization of a novel DNA primase from the Salmonella typhimurium bacteriophage SP6 </span>. <span class="source" tagx="source" title="source">Biochemistry</span>. <span class="year" tagx="year" title="year">2000</span>; <span class="volume" tagx="volume" title="volume">39</span>: <span class="fpage" tagx="fpage" title="fpage">1643</span>– <span class="lpage" tagx="lpage" title="lpage">1654</span>. <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/10677213">10677213</a> </span> </span> </li> <li tag="ref"> <a name="B20" /> <span class="label" tagx="label" title="label">20.</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Mendelman</span> <span class="given-names" tagx="given-names" title="given-names">L.V.</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Richardson</span> <span class="given-names" tagx="given-names" title="given-names">C.C.</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Requirements for primer synthesis by bacteriophage T7 63-kDa gene 4 protein. Roles of template sequence and T7 56-kDa gene 4 protein </span>. <span class="source" tagx="source" title="source">J. Biol. Chem.</span> <span class="year" tagx="year" title="year">1991</span>; <span class="volume" tagx="volume" title="volume">266</span>: <span class="fpage" tagx="fpage" title="fpage">23240</span>– <span class="lpage" tagx="lpage" title="lpage">23250</span>. <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/1744119">1744119</a> </span> </span> </li> <li tag="ref"> <a name="B21" /> <span class="label" tagx="label" title="label">21.</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Swart</span> <span class="given-names" tagx="given-names" title="given-names">J.R.</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Griep</span> <span class="given-names" tagx="given-names" title="given-names">M.A.</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Primase from Escherichia coli primes single-stranded templates in the absence of single-stranded DNA-binding protein or other auxiliary proteins. Template sequence requirements based on the bacteriophage G4 complementary strand origin and Okazaki fragment initiation sites </span>. <span class="source" tagx="source" title="source">J. Biol. Chem.</span> <span class="year" tagx="year" title="year">1993</span>; <span class="volume" tagx="volume" title="volume">268</span>: <span class="fpage" tagx="fpage" title="fpage">12970</span>– <span class="lpage" tagx="lpage" title="lpage">12976</span>. <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/8509429">8509429</a> </span> </span> </li> <li tag="ref"> <a name="B22" /> <span class="label" tagx="label" title="label">22.</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Koepsell</span> <span class="given-names" tagx="given-names" title="given-names">S.A.</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Larson</span> <span class="given-names" tagx="given-names" title="given-names">M.A.</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Griep</span> <span class="given-names" tagx="given-names" title="given-names">M.A.</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Hinrichs</span> <span class="given-names" tagx="given-names" title="given-names">S.H.</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Staphylococcus aureus helicase but not Escherichia coli helicase stimulates S. aureus primase activity and maintains initiation specificity </span>. <span class="source" tagx="source" title="source">J. Bacteriol.</span> <span class="year" tagx="year" title="year">2006</span>; <span class="volume" tagx="volume" title="volume">188</span>: <span class="fpage" tagx="fpage" title="fpage">4673</span>– <span class="lpage" tagx="lpage" title="lpage">4680</span>. <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/16788176">16788176</a> </span> </span> </li> <li tag="ref"> <a name="B23" /> <span class="label" tagx="label" title="label">23.</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Koepsell</span> <span class="given-names" tagx="given-names" title="given-names">S.A.</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Larson</span> <span class="given-names" tagx="given-names" title="given-names">M.A.</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Frey</span> <span class="given-names" tagx="given-names" title="given-names">C.A.</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Hinrichs</span> <span class="given-names" tagx="given-names" title="given-names">S.H.</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Griep</span> <span class="given-names" tagx="given-names" title="given-names">M.A.</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Staphylococcus aureus primase has higher initiation specificity, interacts with single-stranded DNA stronger, but is less stimulated by its helicase than Escherichia coli primase </span>. <span class="source" tagx="source" title="source">Mol. Microbiol.</span> <span class="year" tagx="year" title="year">2008</span>; <span class="volume" tagx="volume" title="volume">68</span>: <span class="fpage" tagx="fpage" title="fpage">1570</span>– <span class="lpage" tagx="lpage" title="lpage">1582</span>. <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/18452509">18452509</a> </span> </span> </li> <li tag="ref"> <a name="B24" /> <span class="label" tagx="label" title="label">24.</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Larson</span> <span class="given-names" tagx="given-names" title="given-names">M.A.</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Bressani</span> <span class="given-names" tagx="given-names" title="given-names">R.</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Sayood</span> <span class="given-names" tagx="given-names" title="given-names">K.</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Corn</span> <span class="given-names" tagx="given-names" title="given-names">J.E.</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Berger</span> <span class="given-names" tagx="given-names" title="given-names">J.M.</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Griep</span> <span class="given-names" tagx="given-names" title="given-names">M.A.</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Hinrichs</span> <span class="given-names" tagx="given-names" title="given-names">S.H.</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Hyperthermophilic Aquifex aeolicus initiates primer synthesis on a limited set of trinucleotides comprised of cytosines and guanines </span>. <span class="source" tagx="source" title="source">Nucleic Acids Res.</span> <span class="year" tagx="year" title="year">2008</span>; <span class="volume" tagx="volume" title="volume">36</span>: <span class="fpage" tagx="fpage" title="fpage">5260</span>– <span class="lpage" tagx="lpage" title="lpage">5269</span>. <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/18684998">18684998</a> </span> </span> </li> <li tag="ref"> <a name="B25" /> <span class="label" tagx="label" title="label">25.</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Cha</span> <span class="given-names" tagx="given-names" title="given-names">T.A.</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Alberts</span> <span class="given-names" tagx="given-names" title="given-names">B.M.</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Studies of the DNA helicase-RNA primase unit from bacteriophage T4. A trinucleotide sequence on the DNA template starts RNA primer synthesis </span>. <span class="source" tagx="source" title="source">J. Biol. Chem.</span> <span class="year" tagx="year" title="year">1986</span>; <span class="volume" tagx="volume" title="volume">261</span>: <span class="fpage" tagx="fpage" title="fpage">7001</span>– <span class="lpage" tagx="lpage" title="lpage">7010</span>. <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/2422175">2422175</a> </span> </span> </li> <li tag="ref"> <a name="B26" /> <span class="label" tagx="label" title="label">26.</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Kusakabe</span> <span class="given-names" tagx="given-names" title="given-names">T.</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Hine</span> <span class="given-names" tagx="given-names" title="given-names">A.V.</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Hyberts</span> <span class="given-names" tagx="given-names" title="given-names">S.G.</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Richardson</span> <span class="given-names" tagx="given-names" title="given-names">C.C.</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">The Cys4 zinc finger of bacteriophage T7 primase in sequence-specific single-stranded DNA recognition </span>. <span class="source" tagx="source" title="source">Proc. Natl. Acad. Sci. U.S.A.</span> <span class="year" tagx="year" title="year">1999</span>; <span class="volume" tagx="volume" title="volume">96</span>: <span class="fpage" tagx="fpage" title="fpage">4295</span>– <span class="lpage" tagx="lpage" title="lpage">4300</span>. <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/10200256">10200256</a> </span> </span> </li> <li tag="ref"> <a name="B27" /> <span class="label" tagx="label" title="label">27.</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Kaguni</span> <span class="given-names" tagx="given-names" title="given-names">L.S.</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Oliveira</span> <span class="given-names" tagx="given-names" title="given-names">M.T.</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Structure, function and evolution of the animal mitochondrial replicative DNA helicase</span>. <span class="source" tagx="source" title="source">Crit. Rev. Biochem. Mol. Biol.</span> <span class="year" tagx="year" title="year">2016</span>; <span class="volume" tagx="volume" title="volume">51</span>: <span class="fpage" tagx="fpage" title="fpage">53</span>– <span class="lpage" tagx="lpage" title="lpage">64</span>. <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/26615986">26615986</a> </span> </span> </li> <li tag="ref"> <a name="B28" /> <span class="label" tagx="label" title="label">28.</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Diray-Arce</span> <span class="given-names" tagx="given-names" title="given-names">J.</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Liu</span> <span class="given-names" tagx="given-names" title="given-names">B.</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Cupp</span> <span class="given-names" tagx="given-names" title="given-names">J.D.</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Hunt</span> <span class="given-names" tagx="given-names" title="given-names">T.</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Nielsen</span> <span class="given-names" tagx="given-names" title="given-names">B.L.</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">The Arabidopsis At1g30680 gene encodes a homologue to the phage T7 gp4 protein that has both DNA primase and DNA helicase activities </span>. <span class="source" tagx="source" title="source">BMC Plant Biol.</span> <span class="year" tagx="year" title="year">2013</span>; <span class="volume" tagx="volume" title="volume">13</span>: <span class="fpage" tagx="fpage" title="fpage">36</span>. <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/23452619">23452619</a> </span> </span> </li> <li tag="ref"> <a name="B29" /> <span class="label" tagx="label" title="label">29.</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Towle-Weicksel</span> <span class="given-names" tagx="given-names" title="given-names">J.B.</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Cao</span> <span class="given-names" tagx="given-names" title="given-names">Y.</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Crislip</span> <span class="given-names" tagx="given-names" title="given-names">L.J.</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Thurlow</span> <span class="given-names" tagx="given-names" title="given-names">D.L.</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Crampton</span> <span class="given-names" tagx="given-names" title="given-names">D.J.</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Chimeric proteins constructed from bacteriophage T7 gp4 and a putative primase-helicase from Arabidopsis thaliana </span>. <span class="source" tagx="source" title="source">Mol. Biol. Rep.</span> <span class="year" tagx="year" title="year">2014</span>; <span class="volume" tagx="volume" title="volume">41</span>: <span class="fpage" tagx="fpage" title="fpage">7783</span>– <span class="lpage" tagx="lpage" title="lpage">7795</span>. <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/25098604">25098604</a> </span> </span> </li> <li tag="ref"> <a name="B30" /> <span class="label" tagx="label" title="label">30.</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Carrie</span> <span class="given-names" tagx="given-names" title="given-names">C.</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Kuhn</span> <span class="given-names" tagx="given-names" title="given-names">K.</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Murcha</span> <span class="given-names" tagx="given-names" title="given-names">M.W.</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Duncan</span> <span class="given-names" tagx="given-names" title="given-names">O.</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Small</span> <span class="given-names" tagx="given-names" title="given-names">I.D.</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">O’Toole</span> <span class="given-names" tagx="given-names" title="given-names">N.</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Whelan</span> <span class="given-names" tagx="given-names" title="given-names">J.</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Approaches to defining dual-targeted proteins in Arabidopsis</span>. <span class="source" tagx="source" title="source">Plant J.</span> <span class="year" tagx="year" title="year">2009</span>; <span class="volume" tagx="volume" title="volume">57</span>: <span class="fpage" tagx="fpage" title="fpage">1128</span>– <span class="lpage" tagx="lpage" title="lpage">1139</span>. <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/19036033">19036033</a> </span> </span> </li> <li tag="ref"> <a name="B31" /> <span class="label" tagx="label" title="label">31.</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Leipe</span> <span class="given-names" tagx="given-names" title="given-names">D.D.</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Aravind</span> <span class="given-names" tagx="given-names" title="given-names">L.</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Grishin</span> <span class="given-names" tagx="given-names" title="given-names">N.V.</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Koonin</span> <span class="given-names" tagx="given-names" title="given-names">E.V.</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">The bacterial replicative helicase DnaB evolved from a RecA duplication</span>. <span class="source" tagx="source" title="source">Genome Res.</span> <span class="year" tagx="year" title="year">2000</span>; <span class="volume" tagx="volume" title="volume">10</span>: <span class="fpage" tagx="fpage" title="fpage">5</span>– <span class="lpage" tagx="lpage" title="lpage">16</span>. <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/10645945">10645945</a> </span> </span> </li> <li tag="ref"> <a name="B32" /> <span class="label" tagx="label" title="label">32.</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Cao</span> <span class="given-names" tagx="given-names" title="given-names">Y.</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">A bifunctional mitochondrial DNA primase/helicase (AMH) identified in Arabidopsis thaliana: Preliminary biochemical characterization of its functions </span>. <span class="source" tagx="source" title="source">Department of Chemistry and Biochemistry</span>. <span class="year" tagx="year" title="year">2012</span>; <span class="publisher-name" tagx="publisher-name" title="publisher-name">Clark University</span>. </span> </li> <li tag="ref"> <a name="B33" /> <span class="label" tagx="label" title="label">33.</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Frick</span> <span class="given-names" tagx="given-names" title="given-names">D.N.</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Baradaran</span> <span class="given-names" tagx="given-names" title="given-names">K.</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Richardson</span> <span class="given-names" tagx="given-names" title="given-names">C.C.</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">An N-terminal fragment of the gene 4 helicase/primase of bacteriophage T7 retains primase activity in the absence of helicase activity </span>. <span class="source" tagx="source" title="source">Proc. Natl. Acad. Sci. U.S.A.</span> <span class="year" tagx="year" title="year">1998</span>; <span class="volume" tagx="volume" title="volume">95</span>: <span class="fpage" tagx="fpage" title="fpage">7957</span>– <span class="lpage" tagx="lpage" title="lpage">7962</span>. <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/9653122">9653122</a> </span> </span> </li> <li tag="ref"> <a name="B34" /> <span class="label" tagx="label" title="label">34.</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Deshpande</span> <span class="given-names" tagx="given-names" title="given-names">A.P.</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Patel</span> <span class="given-names" tagx="given-names" title="given-names">S.S.</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Interactions of the yeast mitochondrial RNA polymerase with the +1 and +2 promoter bases dictate transcription initiation efficiency </span>. <span class="source" tagx="source" title="source">Nucleic Acids Res.</span> <span class="year" tagx="year" title="year">2014</span>; <span class="volume" tagx="volume" title="volume">42</span>: <span class="fpage" tagx="fpage" title="fpage">11721</span>– <span class="lpage" tagx="lpage" title="lpage">11732</span>. <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/25249624">25249624</a> </span> </span> </li> <li tag="ref"> <a name="B35" /> <span class="label" tagx="label" title="label">35.</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Toth</span> <span class="given-names" tagx="given-names" title="given-names">E.A.</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Li</span> <span class="given-names" tagx="given-names" title="given-names">Y.</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Sawaya</span> <span class="given-names" tagx="given-names" title="given-names">M.R.</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Cheng</span> <span class="given-names" tagx="given-names" title="given-names">Y.</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Ellenberger</span> <span class="given-names" tagx="given-names" title="given-names">T.</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">The crystal structure of the bifunctional primase-helicase of bacteriophage T7</span>. <span class="source" tagx="source" title="source">Mol. Cell</span>. <span class="year" tagx="year" title="year">2003</span>; <span class="volume" tagx="volume" title="volume">12</span>: <span class="fpage" tagx="fpage" title="fpage">1113</span>– <span class="lpage" tagx="lpage" title="lpage">1123</span>. <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/14636571">14636571</a> </span> </span> </li> <li tag="ref"> <a name="B36" /> <span class="label" tagx="label" title="label">36.</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Sawaya</span> <span class="given-names" tagx="given-names" title="given-names">M.R.</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Guo</span> <span class="given-names" tagx="given-names" title="given-names">S.</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Tabor</span> <span class="given-names" tagx="given-names" title="given-names">S.</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Richardson</span> <span class="given-names" tagx="given-names" title="given-names">C.C.</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Ellenberger</span> <span class="given-names" tagx="given-names" title="given-names">T.</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Crystal structure of the helicase domain from the replicative helicase-primase of bacteriophage T7 </span>. <span class="source" tagx="source" title="source">Cell</span>. <span class="year" tagx="year" title="year">1999</span>; <span class="volume" tagx="volume" title="volume">99</span>: <span class="fpage" tagx="fpage" title="fpage">167</span>– <span class="lpage" tagx="lpage" title="lpage">177</span>. <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/10535735">10535735</a> </span> </span> </li> <li tag="ref"> <a name="B37" /> <span class="label" tagx="label" title="label">37.</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Kusakabe</span> <span class="given-names" tagx="given-names" title="given-names">T.</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Richardson</span> <span class="given-names" tagx="given-names" title="given-names">C.C.</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Template recognition and ribonucleotide specificity of the DNA primase of bacteriophage T7 </span>. <span class="source" tagx="source" title="source">J. Biol. Chem.</span> <span class="year" tagx="year" title="year">1997</span>; <span class="volume" tagx="volume" title="volume">272</span>: <span class="fpage" tagx="fpage" title="fpage">5943</span>– <span class="lpage" tagx="lpage" title="lpage">5951</span>. <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/9038214">9038214</a> </span> </span> </li> <li tag="ref"> <a name="B38" /> <span class="label" tagx="label" title="label">38.</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Kusakabe</span> <span class="given-names" tagx="given-names" title="given-names">T.</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Richardson</span> <span class="given-names" tagx="given-names" title="given-names">C.C.</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">The role of the zinc motif in sequence recognition by DNA primases</span>. <span class="source" tagx="source" title="source">J. Biol. Chem.</span> <span class="year" tagx="year" title="year">1996</span>; <span class="volume" tagx="volume" title="volume">271</span>: <span class="fpage" tagx="fpage" title="fpage">19563</span>– <span class="lpage" tagx="lpage" title="lpage">19570</span>. <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/8702650">8702650</a> </span> </span> </li> <li tag="ref"> <a name="B39" /> <span class="label" tagx="label" title="label">39.</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Guajardo</span> <span class="given-names" tagx="given-names" title="given-names">R.</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Lopez</span> <span class="given-names" tagx="given-names" title="given-names">P.</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Dreyfus</span> <span class="given-names" tagx="given-names" title="given-names">M.</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Sousa</span> <span class="given-names" tagx="given-names" title="given-names">R.</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">NTP concentration effects on initial transcription by T7 RNAP indicate that translocation occurs through passive sliding and reveal that divergent promoters have distinct NTP concentration requirements for productive initiation </span>. <span class="source" tagx="source" title="source">J. Mol. Biol.</span> <span class="year" tagx="year" title="year">1998</span>; <span class="volume" tagx="volume" title="volume">281</span>: <span class="fpage" tagx="fpage" title="fpage">777</span>– <span class="lpage" tagx="lpage" title="lpage">792</span>. <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/9719634">9719634</a> </span> </span> </li> <li tag="ref"> <a name="B40" /> <span class="label" tagx="label" title="label">40.</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Pal</span> <span class="given-names" tagx="given-names" title="given-names">M.</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Luse</span> <span class="given-names" tagx="given-names" title="given-names">D.S.</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">The initiation-elongation transition: lateral mobility of RNA in RNA polymerase II complexes is greatly reduced at +8/+9 and absent by +23 </span>. <span class="source" tagx="source" title="source">Proc. Natl. Acad. Sci. U.S.A.</span> <span class="year" tagx="year" title="year">2003</span>; <span class="volume" tagx="volume" title="volume">100</span>: <span class="fpage" tagx="fpage" title="fpage">5700</span>– <span class="lpage" tagx="lpage" title="lpage">5705</span>. <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/12719526">12719526</a> </span> </span> </li> <li tag="ref"> <a name="B41" /> <span class="label" tagx="label" title="label">41.</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Baruch-Torres</span> <span class="given-names" tagx="given-names" title="given-names">N.</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Brieba</span> <span class="given-names" tagx="given-names" title="given-names">L.G.</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Plant organellar DNA polymerases are replicative and translesion DNA synthesis polymerases</span>. <span class="source" tagx="source" title="source">Nucleic Acids Res.</span> <span class="year" tagx="year" title="year">2017</span>; <span class="comment">doi:10.1093/nar/gkx744</span>. </span> </li> <li tag="ref"> <a name="B42" /> <span class="label" tagx="label" title="label">42.</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Moriyama</span> <span class="given-names" tagx="given-names" title="given-names">T.</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Terasawa</span> <span class="given-names" tagx="given-names" title="given-names">K.</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Sato</span> <span class="given-names" tagx="given-names" title="given-names">N.</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Conservation of POPs, the plant organellar DNA polymerases, in eukaryotes</span>. <span class="source" tagx="source" title="source">Protist</span>. <span class="year" tagx="year" title="year">2011</span>; <span class="volume" tagx="volume" title="volume">162</span>: <span class="fpage" tagx="fpage" title="fpage">177</span>– <span class="lpage" tagx="lpage" title="lpage">187</span>. <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/20663713">20663713</a> </span> </span> </li> <li tag="ref"> <a name="B43" /> <span class="label" tagx="label" title="label">43.</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Smith</span> <span class="given-names" tagx="given-names" title="given-names">D.R.</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Updating our view of organelle genome nucleotide landscape</span>. <span class="source" tagx="source" title="source">Front Genet.</span> <span class="year" tagx="year" title="year">2012</span>; <span class="volume" tagx="volume" title="volume">3</span>: <span class="fpage" tagx="fpage" title="fpage">175</span>. <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/22973299">22973299</a> </span> </span> </li> <li tag="ref"> <a name="B44" /> <span class="label" tagx="label" title="label">44.</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Corn</span> <span class="given-names" tagx="given-names" title="given-names">J.E.</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Pease</span> <span class="given-names" tagx="given-names" title="given-names">P.J.</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Hura</span> <span class="given-names" tagx="given-names" title="given-names">G.L.</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Berger</span> <span class="given-names" tagx="given-names" title="given-names">J.M.</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Crosstalk between primase subunits can act to regulate primer synthesis in trans</span>. <span class="source" tagx="source" title="source">Mol. Cell</span>. <span class="year" tagx="year" title="year">2005</span>; <span class="volume" tagx="volume" title="volume">20</span>: <span class="fpage" tagx="fpage" title="fpage">391</span>– <span class="lpage" tagx="lpage" title="lpage">401</span>. <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/16285921">16285921</a> </span> </span> </li> <li tag="ref"> <a name="B45" /> <span class="label" tagx="label" title="label">45.</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Zhou</span> <span class="given-names" tagx="given-names" title="given-names">Y.</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Luo</span> <span class="given-names" tagx="given-names" title="given-names">H.</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Liu</span> <span class="given-names" tagx="given-names" title="given-names">Z.</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Yang</span> <span class="given-names" tagx="given-names" title="given-names">M.</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Pang</span> <span class="given-names" tagx="given-names" title="given-names">X.</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Sun</span> <span class="given-names" tagx="given-names" title="given-names">F.</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Wang</span> <span class="given-names" tagx="given-names" title="given-names">G.</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Structural Insight into the specific DNA template binding to DnaG primase in bacteria</span>. <span class="source" tagx="source" title="source">Sci. Rep.</span> <span class="year" tagx="year" title="year">2017</span>; <span class="volume" tagx="volume" title="volume">7</span>: <span class="fpage" tagx="fpage" title="fpage">659</span>. <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/28386108">28386108</a> </span> </span> </li> <li tag="ref"> <a name="B46" /> <span class="label" tagx="label" title="label">46.</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Lee</span> <span class="given-names" tagx="given-names" title="given-names">S.J.</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Zhu</span> <span class="given-names" tagx="given-names" title="given-names">B.</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Hamdan</span> <span class="given-names" tagx="given-names" title="given-names">S.M.</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Richardson</span> <span class="given-names" tagx="given-names" title="given-names">C.C.</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Mechanism of sequence-specific template binding by the DNA primase of bacteriophage T7 </span>. <span class="source" tagx="source" title="source">Nucleic Acids Res.</span> <span class="year" tagx="year" title="year">2010</span>; <span class="volume" tagx="volume" title="volume">38</span>: <span class="fpage" tagx="fpage" title="fpage">4372</span>– <span class="lpage" tagx="lpage" title="lpage">4383</span>. <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/20350931">20350931</a> </span> </span> </li> <li tag="ref"> <a name="B47" /> <span class="label" tagx="label" title="label">47.</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Krishna</span> <span class="given-names" tagx="given-names" title="given-names">S.S.</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Majumdar</span> <span class="given-names" tagx="given-names" title="given-names">I.</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Grishin</span> <span class="given-names" tagx="given-names" title="given-names">N.V.</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Structural classification of zinc fingers: survey and summary</span>. <span class="source" tagx="source" title="source">Nucleic Acids Res.</span> <span class="year" tagx="year" title="year">2003</span>; <span class="volume" tagx="volume" title="volume">31</span>: <span class="fpage" tagx="fpage" title="fpage">532</span>– <span class="lpage" tagx="lpage" title="lpage">550</span>. <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/12527760">12527760</a> </span> </span> </li> <li tag="ref"> <a name="B48" /> <span class="label" tagx="label" title="label">48.</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">van Eijk</span> <span class="given-names" tagx="given-names" title="given-names">E.</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Paschalis</span> <span class="given-names" tagx="given-names" title="given-names">V.</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Green</span> <span class="given-names" tagx="given-names" title="given-names">M.</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Friggen</span> <span class="given-names" tagx="given-names" title="given-names">A.H.</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Larson</span> <span class="given-names" tagx="given-names" title="given-names">M.A.</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Spriggs</span> <span class="given-names" tagx="given-names" title="given-names">K.</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Briggs</span> <span class="given-names" tagx="given-names" title="given-names">G.S.</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Soultanas</span> <span class="given-names" tagx="given-names" title="given-names">P.</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Smits</span> <span class="given-names" tagx="given-names" title="given-names">W.K.</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Primase is required for helicase activity and helicase alters the specificity of primase in the enteropathogen Clostridium difficile </span>. <span class="source" tagx="source" title="source">Open Biol.</span> <span class="year" tagx="year" title="year">2016</span>; <span class="volume" tagx="volume" title="volume">6</span>: <span class="fpage" tagx="fpage" title="fpage">160272</span>. <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/28003473">28003473</a> </span> </span> </li> <li tag="ref"> <a name="B49" /> <span class="label" tagx="label" title="label">49.</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Hardies</span> <span class="given-names" tagx="given-names" title="given-names">S.C.</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Comeau</span> <span class="given-names" tagx="given-names" title="given-names">A.M.</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Serwer</span> <span class="given-names" tagx="given-names" title="given-names">P.</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Suttle</span> <span class="given-names" tagx="given-names" title="given-names">C.A.</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">The complete sequence of marine bacteriophage VpV262 infecting vibrio parahaemolyticus indicates that an ancestral component of a T7 viral supergroup is widespread in the marine environment </span>. <span class="source" tagx="source" title="source">Virology</span>. <span class="year" tagx="year" title="year">2003</span>; <span class="volume" tagx="volume" title="volume">310</span>: <span class="fpage" tagx="fpage" title="fpage">359</span>– <span class="lpage" tagx="lpage" title="lpage">371</span>. <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/12781722">12781722</a> </span> </span> </li> <li tag="ref"> <a name="B50" /> <span class="label" tagx="label" title="label">50.</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Kato</span> <span class="given-names" tagx="given-names" title="given-names">M.</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Frick</span> <span class="given-names" tagx="given-names" title="given-names">D.N.</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Lee</span> <span class="given-names" tagx="given-names" title="given-names">J.</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Tabor</span> <span class="given-names" tagx="given-names" title="given-names">S.</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Richardson</span> <span class="given-names" tagx="given-names" title="given-names">C.C.</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Ellenberger</span> <span class="given-names" tagx="given-names" title="given-names">T.</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">A complex of the bacteriophage T7 primase-helicase and DNA polymerase directs primer utilization </span>. <span class="source" tagx="source" title="source">J. Biol. Chem.</span> <span class="year" tagx="year" title="year">2001</span>; <span class="volume" tagx="volume" title="volume">276</span>: <span class="fpage" tagx="fpage" title="fpage">21809</span>– <span class="lpage" tagx="lpage" title="lpage">21820</span>. <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/11279245">11279245</a> </span> </span> </li> <li tag="ref"> <a name="B51" /> <span class="label" tagx="label" title="label">51.</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Kato</span> <span class="given-names" tagx="given-names" title="given-names">M.</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Ito</span> <span class="given-names" tagx="given-names" title="given-names">T.</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Wagner</span> <span class="given-names" tagx="given-names" title="given-names">G.</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Ellenberger</span> <span class="given-names" tagx="given-names" title="given-names">T.</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">A molecular handoff between bacteriophage T7 DNA primase and T7 DNA polymerase initiates DNA synthesis </span>. <span class="source" tagx="source" title="source">J. Biol. Chem.</span> <span class="year" tagx="year" title="year">2004</span>; <span class="volume" tagx="volume" title="volume">279</span>: <span class="fpage" tagx="fpage" title="fpage">30554</span>– <span class="lpage" tagx="lpage" title="lpage">30562</span>. <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/15133047">15133047</a> </span> </span> </li> </ul> </div> </div>  </body></html>