<html xmlns="http://www.w3.org/1999/xhtml"><head> <style type="text/css">
         	     		 
         	    a                          {background : #ffffff; }
         	    article                    {border-style : dotted; border-width : 2px; }
         		 	
         		 	div                        {background : #ffffcc;}
         		 	div.abstract-title         {font-weight : bold ; font-size : 16pt;}
         		 	div.ack                    {border-style : solid ; border-color : red; margin
         : 2em; }
         		 	div.article-meta           {border-style : solid; border-width : 1 pt; margin
         1em;}
         		 	div.boxed-text             {margin : 5em; border-style : solid;}
         		 	div.contrib-group          {margin : 1em; }
         		 	div.fig                    {border-style : solid; border-width : 2px; margin :
         2em; }
         		 	div.funding                {font-weight : bold ; font-size : 16pt;}
         		 	div.given-names            {font-style : italic;}
         		 	div.intro                  {border-style : inset; margin : 5px;}
         		 	div.introduction           {border-style : inset; margin : 5px;}
         		 	div.journal-meta           {border-style : solid; border-width : 1 pt; margin
         1em;}
         		 	div.journal-title-group    {background : #ffeeee;}
         		 	div.article-title          {font-weight : bold ; font-size : 18pt;}
         		 	div.kwd                    {font-style : italic}
         		 	div.meta-name              {font-weight : bold ; font-size : 16pt;}
         		 	div.name                   {font-weight : bold;}
         		 	div.alt-title              {font-style : italic; font-size : 12px;}
         		 	
         		 	div.sec                    {border : 2px; margin 5px; padding 2px;}
         		 	div.title                  {font-family : courier; font-weight : bold;
         		 	                            font-size : 16pt; margin : 5px;}
         		 	
         		 	div.materials_methods      {border-style : double; margin : 5px; }
         		 	div.methods                {border-style : double; margin : 5px; }
         		 	
         		 	div.results                {border-style : solid; margin : 5px;}
         		 	div.background             {border-style : dotted; margin : 5px;}
         		 	
         		 	div.discussion             {border-style : groove; margin : 5px;}
         		 	div.conclusion             {border-style : ridge; margin : 5px;}
         		 	div.conclusions             {border-style : ridge; margin : 5px;}
         		 	div.supplementary-material {border-style : inset; margin : 5px;}
         		 	div.abbreviations          {border-style : double; border-color : red; margin
         : 5px;}
         		 	div.competinginterests     {border-style : double; border-color : blue; margin
         : 5px;}
         		 	div.acknowledgements       {border-style : double; border-color : green; margin
         : 5px;}
         		 	div.authors_contributions   {border-style : double; border-color : purple; margin
         : 5px;}
         		 	
         		 	div.publisher              {border-style : outset; margin : 5px;}
         		 	div.fn-type-conflict       {background : #f88; }
         		 	div.fn-type-con            {background : #ddf; }
         		 	div.fn-type-other          {background : #ddd; }
         		 	
         		 	div.unknown                {background : #ffd;
         									 	  border-style : solid;
         									 	  border-width : 1px;
         									 	  padding : 2 px;}
         		 	  
         	    table                      {background : #ffffdd;}
         		 	tr                         {background : #ddddff; padding : 1px;}
         		 	
         		 	span                       {background : #ffcccc;}
         		 	
         		 	span.citation-author       {font-family : helvetica; background : #ffeeee;}
         		 	span.collab                {background : #ddffff; }
         		 	span.comment               {font-family : courier; font-size : 6px; background
         : #ffaaff;}
         		 	span.contrib               {background : #ffffff;}
         		 	span.corresp               {background : #ddffdd; }
         		 	span.doi                   {background : #ffffff;}
         		 	span.email                 {font-family : courier; }
         		 	span.etal                  {font-style : italic;}
         		 	span.fpage                 {font-family : courier;}
         		 	span.given-names           {background : #ffffff;}
         		 	span.iso-abbrev            {background : #ffffff;}
         		 	span.issn-epub             {background : #ffffff;}
         		 	span.issn-ppub             {background : #ffffff;}
         		 	span.journal-title         {background : #ffffff;}
         		 	span.lpage                 {font-family : courier;}
         		 	span.mixed-article-title   {font-style : italic ;}
         		 	span.nlm-ta                {background : #ffffff;}
         		 	span.pmc                   {background : #ffffff;}
         		 	span.pmcid                 {background : #ffffff;}
         		 	span.pmid                  {background : #ffffff;}
         		 	span.publisher             {background : #ffffff;}
         		 	span.publisher-id          {background : #ffffff;}
         		 	span.publisher-name        {background : #ffffff;}
         		 	span.source                {background : #ffffff;}
         		 	span.subject               {background : #ffffff;}
         		 	span.surname               {background : #ffffff;}
         		 	span.volume                {font-family : courier; font-weight : bold;}
         		 	span.year                  {font-family : courier ; font-style : italic;}
         			
      </style> </head> <body> <div class="front" title="front"> <div class="journal-meta" tagx="journal-meta" title="journal-meta"> <span class="nlm-ta" title="nlm-ta">BMC Genomics</span> <span class="journal-title" tagx="journal-title" title="journal-title">BMC Genomics</span> <span class="issn-epub" tagx="issn" title="issn-epub">1471-2164</span> <div class="publisher" tagx="publisher" title="publisher"> <span class="publisher-name" tagx="publisher-name" title="publisher-name">BioMed Central</span> <span class="publisher-loc" tagx="publisher-loc" title="publisher-loc">London</span> </div> </div> <div class="article-meta" tagx="article-meta" title="article-meta"> <span class="pmcid" title="pmcid"> pmcid: <a href="http://www.ncbi.nlm.nih.gov/pubmed/1925096">1925096</a> </span> <span class="publisher-id" title="publisher-id">1471-2164-8-174</span> <span class="pmid" title="pmid"> pmid: <a href="http://www.ncbi.nlm.nih.gov/pubmed/17573971">17573971</a> </span> <span class="doi" title="doi"> doi: <a href="https://doi.org/10.1186/1471-2164-8-174">10.1186/1471-2164-8-174</a> </span> <div class="article-categories" title="article-categories">
: <span class="subject" title="subject">Research Article</span> </div> <div class="title-group"> <div class="article-title" title="article-title">
Comparative chloroplast genomics: analyses including new sequences from the angiosperms <i>Nuphar advena </i>and <i>Ranunculus macranthus</i> </div> </div> <div class="contrib-group" title="contrib-group"> <span class="contrib" title="contrib"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Raubeson</span> <span class="given-names" tagx="given-names" title="given-names">Linda A</span> </span> <a href="#I1">1</a> <span class="email" tagx="email" title="email">raubeson@cwu.edu</span> </span> <span class="contrib" title="contrib"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Peery</span> <span class="given-names" tagx="given-names" title="given-names">Rhiannon</span> </span> <a href="#I1">1</a> <span class="email" tagx="email" title="email">peeryr@cwu.edu</span> </span> <span class="contrib" title="contrib"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Chumley</span> <span class="given-names" tagx="given-names" title="given-names">Timothy W</span> </span> <a href="#I2">2</a> <span class="email" tagx="email" title="email">tchumley@mail.utexas.edu</span> </span> <span class="contrib" title="contrib"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Dziubek</span> <span class="given-names" tagx="given-names" title="given-names">Chris</span> </span> <a href="#I1">1</a> <span class="email" tagx="email" title="email">cdz5@hotmail.com</span> </span> <span class="contrib" title="contrib"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Fourcade</span> <span class="given-names" tagx="given-names" title="given-names">H Matthew</span> </span> <a href="#I3">3</a> <span class="email" tagx="email" title="email">fourcade@bu.edu</span> </span> <span class="contrib" title="contrib"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Boore</span> <span class="given-names" tagx="given-names" title="given-names">Jeffrey L</span> </span> <a href="#I3">3</a> <span class="email" tagx="email" title="email">JLBoore@Berkeley.edu</span> </span> <span class="contrib" title="contrib"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Jansen</span> <span class="given-names" tagx="given-names" title="given-names">Robert K</span> </span> <a href="#I2">2</a> <span class="email" tagx="email" title="email">jansen@mail.utexas.edu</span> </span> </div> <span class="citation_author_institution" id="I1">[1], </span> <span class="citation_author_institution" id="I2">[2], </span> <span class="citation_author_institution" id="I3">[3], </span> <span class="pub-date-collection" title="pub-date-collection">collection: <span>2007</span> </span> <span class="pub-date-epub" title="pub-date-epub">epub: <span>2007-6-6</span> </span> <span class="volume" tagx="volume" title="volume">8</span> <span class="fpage" tagx="fpage" title="fpage">174</span> <span class="lpage" tagx="lpage" title="lpage">174</span> <a href="http://www.biomedcentral.com/1471-2164/8/174" /> <span class="history" title="history"> <span class="received" title="received">received: 2006-12-15</span> <span class="accepted" title="accepted">accepted: 2007-6-15</span> </span> <div class="permissions"> <span class="copyright" title="copyright">(C) <span class="copyright-holder" tagx="copyright-holder" title="copyright-holder">Raubeson et al; licensee BioMed Central Ltd.</span>, <span class="copyright-year" tagx="copyright-year" title="copyright-year">2007</span> </span> <span class="license" title="license"> <p>This is an Open Access article distributed under the terms of the Creative Commons Attribution License ( <a href="http://creativecommons.org/licenses/by/2.0" />), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited. </p> </span> </div> <div class="abstract" title="abstract"> <div class="abstract-title" title="abstract-title">
Abstract</div> <div class="background" title="sec"> <div class="title" tagx="title" title="title">
Background</div> <p>The number of completely sequenced plastid genomes available is growing rapidly. This array of sequences presents new opportunities to perform comparative analyses. In comparative studies, it is often useful to compare across wide phylogenetic spans and, within angiosperms, to include representatives from basally diverging lineages such as the genomes reported here: <i>Nuphar advena </i>(from a basal-most lineage) and <i>Ranunculus macranthus </i>(a basal eudicot). We report these two new plastid genome sequences and make comparisons (within angiosperms, seed plants, or all photosynthetic lineages) to evaluate features such as the status of <i>ycf15 </i>and <i>ycf68 </i>as protein coding genes, the distribution of simple sequence repeats (SSRs) and longer dispersed repeats (SDR), and patterns of nucleotide composition. </p> </div> <div class="results" title="sec"> <div class="title" tagx="title" title="title">
Results</div> <p>The <i>Nuphar </i>[GenBank: <a href="NC_008788">NC_008788</a>] and <i>Ranunculus </i>[GenBank: <a href="NC_008796">NC_008796</a>] plastid genomes share characteristics of gene content and organization with many other chloroplast genomes. Like other plastid genomes, these genomes are A+T-rich, except for rRNA and tRNA genes. Detailed comparisons of <i>Nuphar </i>with <i>Nymphaea</i>, another Nymphaeaceae, show that more than two-thirds of these genomes exhibit at least 95% sequence identity and that most SSRs are shared. In broader comparisons, SSRs vary among genomes in terms of abundance and length and most contain repeat motifs based on A and T nucleotides. </p> </div> <div class="conclusion" title="sec"> <div class="title" tagx="title" title="title">
Conclusion</div> <p>SSR and SDR abundance varies by genome and, for SSRs, is proportional to genome size. Long SDRs are rare in the genomes assessed. SSRs occur less frequently than predicted and, although the majority of the repeat motifs do include A and T nucleotides, the A+T bias in SSRs is less than that predicted from the underlying genomic nucleotide composition. In codon usage third positions show an A+T bias, however variation in codon usage does not correlate with differences in A+T-richness. Thus, although plastome nucleotide composition shows "A+T richness", an A+T bias is not apparent upon more in-depth analysis, at least in these aspects. The pattern of evolution in the sequences identified as <i>ycf15 </i>and <i>ycf68 </i>is not consistent with them being protein-coding genes. In fact, these regions show no evidence of sequence conservation beyond what is normal for non-coding regions of the IR. </p> </div> </div> </div> </div> <div class="body" title="body"> <div class="background" title="sec"> <div class="title" tagx="title" title="title">
Background</div> <p>In this paper we report the complete chloroplast genome sequences of the angiosperms <i>Nuphar advena </i>(Nymphaeaceae) and <i>Ranunculus macranthus </i>(Ranunculaceae). The Nymphaeaceae is placed very near or even at the base of extant angiosperms [ <a href="#B1">1</a>- <a href="#B6">6</a>], whereas the <i>Ranunculus </i>chloroplast genome represents the basal-most eudicot characterized to date [ <a href="#B4">4</a>, <a href="#B5">5</a>]. Thus we add to the small number of genomes not representing monocots or crown eudicots, increasing our ability to compare genomes across all angiosperms and determine general characteristics. Most chloroplast genomes in angiosperms (reviewed in: [ <a href="#B7">7</a>- <a href="#B9">9</a>]) range from 135 to 160 kb and exist, at least in part [ <a href="#B10">10</a>] as single genome circles. In the majority of angiosperm chloroplast genomes two copies of a large inverted repeat (IR) of about 25 kb separate the remainder of the genome into two regions of unique DNA, the large (about 90 kb) and small (about 20 kb) single copy regions (LSC and SSC, respectively). Tobacco ( <i>Nicotiana tabacum</i>), the first completely sequenced chloroplast genome [ <a href="#B11">11</a>], is most frequently used to contrast features of newly studied angiosperm cpDNAs and is parsimoniously inferred to represent the ancestral (or at least consensus) angiosperm chloroplast genome in terms of gene content and organization [ <a href="#B7">7</a>, <a href="#B8">8</a>]. This is reinforced by the similarity of the tobacco cpDNA to the basal angiosperms, <i>Amborella </i>[ <a href="#B12">12</a>] and <i>Nymphaea </i>[ <a href="#B13">13</a>], and the magnolids, <i>Calycanthus </i>[ <a href="#B14">14</a>] <i>Drimys, Liriodendron</i>, and <i>Piper </i>[ <a href="#B6">6</a>]. In some derived angiosperm lineages, this ancestral condition has been somewhat or highly modified via inversions, gene losses, presence or absence of ORFs and minor ycfs, and changes in IR extent [ <a href="#B8">8</a>, <a href="#B9">9</a>, <a href="#B15">15</a>, <a href="#B16">16</a>]. However, the tobacco-like pattern is widely distributed in crown eudicots, e.g., <i>Panax </i>[ <a href="#B17">17</a>], <i>Eucalyptus </i>[ <a href="#B18">18</a>], and <i>Gossypium </i>[ <a href="#B19">19</a>]. </p> <p>Comparative chloroplast genomics as well as detailed characterizations of individual chloroplast genomes serve as the basis for functional genomic studies [ <a href="#B20">20</a>] and can direct attempts at chloroplast transformation for genetic engineering [ <a href="#B21">21</a>]. In addition the chloroplast genome is an important source of genetic markers for phylogenetic analysis, population-level studies, genotyping and mapping that can be further exploited with additional genomic characterization and comparison. Detailed comparisons of genomic sequence have the potential, for example, to identify functional sequence outside of coding regions (promoters, terminators, replication origins, etc.), test the reality of hypothetical protein coding regions, make inferences about mutational rates and mechanisms, and detect selective signatures in gene sequences. Many fundamental aspects of the chloroplast genome are poorly understood and incompletely described. Here we use genomic comparisons to investigate the likelihood that <i>ycf15 </i>and <i>ycf68 </i>are not protein-coding genes, the occurrence of microsatellites or simple sequence repeats (SSRs), the presence of somewhat larger more complex repeats or small disperse repeats (SDR) and how nucleotide composition contributes to patterns of genome organization such as codon usage and repeat structure. </p> </div> <div class="resultsanddiscussion" title="sec"> <div class="title" tagx="title" title="title">
Results and Discussion</div> <div class="genomiccharacteristics,includingirextent" title="sec"> <div class="title" tagx="title" title="title">
Genomic characteristics, including IR extent</div> <p>Both the <i>Nuphar </i>[GenBank: <a href="NC_008788">NC_008788</a>] and <i>Ranunculus </i>[GenBank: <a href="NC_008796">NC_008796</a>] genomes exhibit the quadripartite structure common to most land plant genomes with large and small single copy regions (LSC and SSC, respectively) separated by two copies of the IR. The <i>Nuphar </i>chloroplast genome (Figure <a href="#F1">1</a>) is 160,866 bp in total length; the LSC is 90,379 bp, the SSC 18,817 bp and the two IR copies each 25,835 bp in length. In <i>Ranunculus </i>(Figure <a href="#F2">2</a>), the overall length is 155,129 bp with a LSC of 84,638 bp, a SSC of 18,909 bp, and two IR regions each of 25,791 bp. As is common to chloroplast genomes in general [ <a href="#B5">5</a>, <a href="#B6">6</a>, <a href="#B9">9</a>], the nucleotide composition of both of these genomes are biased towards A and T nucleotides, i.e., they are "A+T-rich". Overall the <i>Nuphar </i>genome is 60.9% A+T and <i>Ranunculus </i>62.1% A+T. Different regions of the genome vary in A+T content, but all partitions are A+T rich with the exception of the two classes of RNA genes (Table <a href="#T1">1</a>). </p> <div class="fig" title="fig"> <div class="figure" title="figure"> <a href="https://doi.org/"> <span class="label" tagx="label" title="label">Figure 1</span> </a> </div>   <p>Linearized <i>Nuphar advena </i>plastome map. Genes are represented by boxes extending above or below the base line depending on the direction of transcription. The color of the gene boxes and the intergenic regions indicates the level of similarity of the region between the <i>Nuphar </i>and <i>Nymphaea </i>plastomes. </p>   </div> <div class="fig" title="fig"> <div class="figure" title="figure"> <a href="https://doi.org/"> <span class="label" tagx="label" title="label">Figure 2</span> </a> </div>   <p>Circular <i>Ranunculus macranthus </i>plastome map. Genes are represented by boxes inside or outside the circle to indicate the direction of transcription, clockwise or counterclockwise, respectively. The color of the gene boxes indicates the functional group to which the gene belongs. </p>   </div> <div class="table-wrap_UNKNOWN"> <span class="label" tagx="label" title="label">Table 1</span>  <p>AT richness (%A+T) of the <i>Nuphar </i>and <i>Ranunculus </i>plastomes, overall and in various partitions. </p>  <table> <thead> <tr> <td /> <td>Genome</td> <td>LSC</td> <td>SSC</td> <td>IR</td> <td>Non-coding (IGS)</td> <td>Protein- coding genes (CDS)</td> </tr> </thead> <tbody> <tr> <td> <i>Nuphar</i> </td> <td>60.9</td> <td>62.9</td> <td>65.3</td> <td>56.7</td> <td>65.5</td> <td>60.6</td> </tr> <tr> <td> <i>Ranunculus</i> </td> <td>62.1</td> <td>65.5</td> <td>69.0</td> <td>56.5</td> <td>68.3</td> <td>62.0</td> </tr> <tr> <td> <hr /> </td> </tr> <tr> <td /> <td>introns</td> <td>1st position</td> <td>2nd position</td> <td>3rd position</td> <td>rRNA genes</td> <td>tRNA genes</td> </tr> <tr> <td> <hr /> </td> </tr> <tr> <td> <i>Nuphar</i> </td> <td>63.0</td> <td>53.1</td> <td>60.8</td> <td>68.0</td> <td>44.4</td> <td>46.5</td> </tr> <tr> <td> <i>Ranunculus</i> </td> <td>63.5</td> <td>54.2</td> <td>61.7</td> <td>70.1</td> <td>44.5</td> <td>46.8</td> </tr> </tbody> </table> </div> <p>Although the two copies of the IR were not sequenced independently, the identity of the two copies could still be assessed. All random reads generated from the two IR regions falsely assemble in one location, but presumably about half of the reads are sequenced from one copy of the repeat and half from the other. If the two copies varied in sequence, one would expect, on average, half the reads to reflect one variant and half the other, but all reads that assembled into this region for both <i>Nuphar </i>and <i>Ranunculus </i>were identical in sequence. In fact, throughout the entire cpDNA of <i>Nuphar</i>, only one case of potential heteroplasmy was observed, where four reads indicated a run of 10 As and four others indicated 11. In <i>Ranunculus </i>no cases of high quality mismatch involving more than a single read were detected, even though multiple individuals contributed to the sequencing template. Nucleotide polymorphisms in chloroplast DNA sequences have been detected in several other groups. In the completely sequenced <i>Pelargonium </i>genome [ <a href="#B16">16</a>], 11 polymorphisms were detected and nine of these were located in the LSC. Unfortunately, it was not possible to determine if the differences in <i>Nuphar </i>or <i>Pelargonium </i>represent heteroplasmy because multiple individuals were used in both studies. Heteroplasmy in the chloroplast genome has been detected in several other groups, including rice [ <a href="#B22">22</a>], <i>Medicago </i>[ <a href="#B23">23</a>], and <i>Senecio </i>[ <a href="#B24">24</a>]. </p> <p>The IR extent in <i>Nuphar </i>and <i>Ranunculus</i>, as well as those of the basal angiosperms sequenced by Goremykin et al [ <a href="#B12">12</a>- <a href="#B14">14</a>], were confirmed independently of the primary sequencing effort by PCR amplification and sequencing of the boundary regions. Although all these genomes contain IRs similar in extent to those of <i>Nicotiana </i>and many other angiosperms, some minor modifications were detected (Figure <a href="#F3">3</a>). As is common among angiosperms, a complete copy of <i>ycf1 </i>spans the SSC/IR <sub>A </sub>junction and the 5'end of the gene is duplicated at the SSC end of IRb. Among the comparisons shown in Figure <a href="#F3">3</a>, the amount of <i>ycf1 </i>that is duplicated ranges from 156 bp in <i>Nymphaea </i>to 1,583 bp in <i>Amborella</i>. At the LSC end of the IR, there is variation in these taxa over whether and how much of the gene <i>trnH </i>is duplicated. No part of <i>trnH </i>is duplicated in <i>Calycanthus </i>or <i>Nicotiana </i>but amounts ranging from one bp in <i>Nymphaea </i>to the entire gene (plus 140 bp of IGS) in <i>Nuphar </i>have been incorporated into the IR. The <i>trnH </i>gene has also been incorporated in the IR of <i>Drimys </i>but not in the other two recently sequenced magnoliid genomes, <i>Liriodendron</i>, and <i>Piper </i>[ <a href="#B6">6</a>]. Large or complex changes in the extent of the IR should make distinctive markers of evolutionary lineages [ <a href="#B25">25</a>, <a href="#B26">26</a>]. However, given the small size of the changes discussed here and the relative ease of small (~100 bp) movements of the IR boundaries [ <a href="#B27">27</a>], the amount of <i>ycf1 </i>duplicated and the migration of <i>trnH </i>relative to the IR would not make very reliable phylogenetic markers. </p> <div class="fig" title="fig"> <div class="figure" title="figure"> <a href="https://doi.org/"> <span class="label" tagx="label" title="label">Figure 3</span> </a> </div>   <p>Comparison of inverted repeat-single copy boundaries in six representative angiosperms. Variation occurs at each of the four junctions. In <i>Calycanthus rpl2 </i>is not in the IR. J <sub>SB </sub>occurs within <i>ycf1 </i>in all of the genomes but the amount of the 5' end of <i>ycf1 </i>that is duplicated ranges from 156 bp in <i>Nymphaea </i>to 1583 bp in <i>Amborella</i>. Eleven bp of <i>ndhF </i>is duplicated in <i>Nuphar </i>but none of the other genomes shown have any duplication of the gene. J <sub>LA </sub>varies from including 5 bp of spacer downstream of <i>trnH </i>in <i>Nicotiana </i>to the inclusion of <i>trnH </i>and an additional 140 bp upstream sequence in the IR in <i>Nuphar</i>. </p>   </div> </div> <div class="genecontent,includingycf15andycf68" title="sec"> <div class="title" tagx="title" title="title">
Gene content, including ycf15 and ycf68</div> <p>The gene content and arrangement of <i>Nuphar </i>and <i>Ranunculus </i>plastid genomes (Figs. <a href="#F1">1</a>, <a href="#F2">2</a>) are identical with each other and with <i>Nymphaea</i>, <i>Calycanthus</i>, <i>Amborella </i>and <i>Nicotiana </i>(among others) except for details of the IR extent (described above), whether or not <i>infA </i>occurs as a pseudogene (in <i>Ranunculus </i>and <i>Nicotiana</i>) or a functional copy (the others listed), and the nature of <i>ycf15 </i>and <i>ycf68 </i>(see below). Seventy-nine different protein-coding genes (including, in this count, four hypothetical genes, <i>ycf1</i>, <i>ycf2</i>, <i>ycf3</i>, and <i>ycf4</i>), four rRNA genes and 30 tRNA genes occur in these genomes. Eighteen of these genes contain introns including two genes, <i>clpP </i>and <i>ycf3</i>, each with two introns, and one gene, <i>rps12</i>, also composed of three exons, but with the 5' exon separated from the two 3' exons. These features are common characteristics of land plant chloroplast genomes [ <a href="#B9">9</a>, <a href="#B28">28</a>]. </p> <p>The hypothetical gene <i>ycf15 </i>was first identified as ORF87 in <i>Nicotiana </i>[ <a href="#B11">11</a>] and has been included in the annotation of a subset of the completed land plant genomes. However, the validity of <i>ycf15 </i>as a protein-coding gene has been questioned [ <a href="#B14">14</a>, <a href="#B18">18</a>, <a href="#B29">29</a>]. Schmitz-Linneweber et al [ <a href="#B29">29</a>] found that the plastomes of <i>Nicotiana</i>, <i>Epifagus </i>and <i>Cuscuta </i>contain intact copies of <i>ycf15</i>, whereas those of <i>Spinacia </i>and <i>Arabidopsis </i>contain <i>ycf15 </i>as two pieces, with the 5' and 3' sections separated by 250–300 bp of 'intervening sequence'. They reasoned that if <i>ycf15 </i>is a functional protein in spinach, then the intervening sequence would need to be removed and the 5' and 3' sections spliced in order to make a functional <i>ycf15 </i>mature transcript. If the intervening sequence were not removed, numerous in-frame stop codons would lead to a truncated protein (Figure <a href="#F4">4</a>). Reverse transcription experiments in spinach determined that <i>ycf15 </i>was not spliced (although it was transcribed) and so presumably was not translated in spinach [ <a href="#B29">29</a>]. They concluded that the <i>ycf15 </i>sequence, since it is highly conserved, probably has functional significance but probably does not code for a protein. Using an alternative approach, Goremykin et al [ <a href="#B14">14</a>] compared nucleotide substitution rates (dN/dS) in <i>ycf15 </i>and found a ratio that suggests <i>ycf15 </i>is not a protein evolving under evolutionary constraint. </p> <div class="fig" title="fig"> <div class="figure" title="figure"> <a href="https://doi.org/"> <span class="label" tagx="label" title="label">Figure 4</span> </a> </div>   <p>Alignment of the <i>ycf15 </i>region in six representative angiosperms. <i>Atropa </i>and <i>Nicotiana </i>represent the uninterrupted form. Codons highlighted in green represent start codons as annotated in the published genomes <i>Atropa </i>and <i>Nicotiana</i>. Codons highlighted in red represent stop codons in frame with those start codons. Although the sequence is highly conserved, it is not an open reading frame in most taxa. </p>   </div> <p>We examined the <i>ycf2:trnL </i>spacer region, where <i>ycf15 </i>is located, in several chloroplast genomes, including <i>Nuphar </i>and <i>Ranunculus</i>, to determine the distribution and nature of <i>ycf15 </i>(Table <a href="#T2">2</a>). Schmitz-Linneweber <i>et al</i>. [ <a href="#B29">29</a>], based on their small sample of taxa, suggested that the distribution of the interrupted versus uninterrupted <i>ycf15 </i>sequence, whether or not the sequence codes for a protein, might have phylogenetic significance. That indeed appears to be the case; all taxa, among those we examined, that contain an uninterrupted <i>ycf15 </i>motif are asterids and no asterid has the interrupted form (although some lack the 5' portion of the motif). We assume, based on the sequence similarity of the intervening sequence, that the Schmitz-Linneweber <i>et al</i>. [ <a href="#B29">29</a>] finding of lack of splicing in <i>Spinacia </i>is likely to hold for other interrupted forms of the motif. The existence of the widely distributed interrupted form suggests that this is not a protein-coding gene in any of these genomes. If this truly is a protein-coding gene in the asterids, we would have to imagine that pseudogenes (as the interrupted forms would be, see Figure <a href="#F4">4</a>) are being retained, conserved over broad evolutionary distances, and either that the active form was inactivated multiple times by the insertion of the same intervening sequence at the same location or that an inactivated gene was re-activated in the asterids through the excision of the intervening sequence. Thus, we concur that <i>ycf15 </i>is unlikely to represent a protein-coding gene and so we did not annotate the sequence in either genome. </p> <div class="table-wrap_UNKNOWN"> <span class="label" tagx="label" title="label">Table 2</span>  <p>Extent of <i>ycf15 </i>motif (as defined by the tobacco sequence) in published angiosperm chloroplast genomes. </p>  <table> <thead> <tr> <td /> <td>Species <sup>a</sup> </td> <td>Accession Number</td> <td>5' portion</td> <td>3' portion</td> <td>Length of intervening sequence <sup>b </sup>(bp) </td> </tr> </thead> <tbody> <tr> <td>"basal" angiosperms</td> <td> <i>Amborella trichopoda</i> </td> <td> <a href="NC_005086">NC_005086</a> </td> <td>154</td> <td>92</td> <td>295</td> </tr> <tr> <td /> <td> <i>Nuphar advena</i> </td> <td> <a href="NC_008788">NC_008788</a> </td> <td>154</td> <td>70</td> <td>299</td> </tr> <tr> <td /> <td> <i>Nymphaea alba</i> </td> <td> <a href="NC_006050">NC_006050</a> </td> <td>154</td> <td>74</td> <td>299</td> </tr> <tr> <td /> <td> <i>Calycanthus floridus</i> </td> <td> <a href="NC_004993">NC_004993</a> </td> <td>152</td> <td>92</td> <td>291</td> </tr> <tr> <td>Monocots</td> <td> <i>Acorus calamus</i> </td> <td> <a href="NC_007407">NC_007407</a> </td> <td>154</td> <td>0</td> <td>N/A</td> </tr> <tr> <td /> <td> <i>Phalaenopsis aphrodite</i> </td> <td> <a href="NC_007499">NC_007499</a> </td> <td>136</td> <td>45</td> <td>295</td> </tr> <tr> <td /> <td> <i>Saccharum officinarum</i> </td> <td> <a href="NC_006084">NC_006084</a> </td> <td>139</td> <td>79</td> <td>305</td> </tr> <tr> <td /> <td> <i>Zea mays</i> </td> <td> <a href="NC_001666">NC_001666</a> </td> <td>139</td> <td>79</td> <td>305</td> </tr> <tr> <td>Eudicots Non-asterids</td> <td> <i>Arabidopsis thaliana</i> </td> <td> <a href="NC_000932">NC_000932</a> </td> <td>120</td> <td>105</td> <td>285</td> </tr> <tr> <td /> <td> <i>Eucalyptus globulus</i> </td> <td> <a href="NC_008115">NC_008115</a> </td> <td>154</td> <td>106</td> <td>296</td> </tr> <tr> <td /> <td> <i>Gossypium hirsutum</i> </td> <td> <a href="NC_007944">NC_007944</a> </td> <td>154</td> <td>107</td> <td>307</td> </tr> <tr> <td /> <td> <i>Lotus japonicus</i> </td> <td> <a href="NC_002694">NC_002694</a> </td> <td>0</td> <td>98</td> <td>N/A</td> </tr> <tr> <td /> <td> <i>Oenothera elata</i> </td> <td> <a href="NC_002693">NC_002693</a> </td> <td>154</td> <td>110</td> <td>284</td> </tr> <tr> <td /> <td> <i>Ranunculus macranthus</i> </td> <td> <a href="NC_008796">NC_008796</a> </td> <td>0</td> <td>78</td> <td>N/A</td> </tr> <tr> <td /> <td> <i>Spinacia oleracea</i> </td> <td> <a href="NC_002202">NC_002202</a> </td> <td>153</td> <td>92</td> <td>254</td> </tr> <tr> <td /> <td> <i>Vitis vinifera</i> </td> <td> <a href="NC_007957">NC_007957</a> </td> <td>154</td> <td>107</td> <td>292</td> </tr> <tr> <td>Eudicots Asterids</td> <td> <i>Atropa belladonna</i> </td> <td> <a href="NC_004561">NC_004561</a> </td> <td>154</td> <td>110</td> <td>0</td> </tr> <tr> <td /> <td> <i>Epifagus virginiana</i> </td> <td> <a href="NC_001568">NC_001568</a> </td> <td>150</td> <td>107</td> <td>0</td> </tr> <tr> <td /> <td> <i>Helianthus annuus</i> </td> <td> <a href="NC_007977">NC_007977</a> </td> <td>0</td> <td>53</td> <td>N/A</td> </tr> <tr> <td /> <td> <i>Lactuca sativa</i> </td> <td> <a href="DQ383816">DQ383816</a> </td> <td>0</td> <td>53</td> <td>N/A</td> </tr> <tr> <td /> <td> <i>Lycopersicon esculentum</i> </td> <td> <a href="DQ347959">DQ347959</a> </td> <td>154</td> <td>110</td> <td>0</td> </tr> <tr> <td /> <td> <i>Nicotiana sylvestris</i> </td> <td> <a href="NC_007500">NC_007500</a> </td> <td>154</td> <td>110</td> <td>0</td> </tr> <tr> <td /> <td> <i>Nicotiana tobaccum</i> </td> <td> <a href="NC_001879">NC_001879</a> </td> <td>154</td> <td>110</td> <td>0</td> </tr> <tr> <td /> <td> <i>Nicotiana tomentosiformis</i> </td> <td> <a href="NC_007602">NC_007602</a> </td> <td>154</td> <td>110</td> <td>0</td> </tr> <tr> <td /> <td> <i>Panax ginseng</i> </td> <td> <a href="NC_006290">NC_006290</a> </td> <td>154</td> <td>106</td> <td>0</td> </tr> <tr> <td /> <td> <i>Solanum bulbocastanum</i> </td> <td> <a href="NC_007943">NC_007943</a> </td> <td>154</td> <td>110</td> <td>0</td> </tr> <tr> <td /> <td> <i>Solanum tuberosum</i> </td> <td> <a href="NC_008096">NC_008096</a> </td> <td>154</td> <td>110</td> <td>0</td> </tr> </tbody> </table> <div class="table-wrap-foot_UNKNOWN"> <p> <sup>a </sup> <i>ycf15 </i>is 264 bp in the original annotation of <i>Nicotiana</i>, 147 bp in <i>Amborella</i>, 243 bp in <i>Nymphaea </i>and <i>Calycanthus</i>, 300 bp in 234 bp in <i>Arabidopsis</i>, 138 bp in <i>Gossypium</i>, 234 bp in <i>Vitus</i>, 213 bp in <i>Atropa</i>, 264 bp in <i>Nicotiana sp</i>. and <i>Solanum sp</i>., 264 bp in <i>Lycopersicon</i>, 162 bp in <i>Helianthus</i>, and 303 bp in <i>Panax</i>. </p> <p> <sup>b </sup>N/A = not applicable (cases where either 5' or 3' portion is missing) </p> </div> </div> <p>Similarly a second hypothetical protein-coding gene, <i>ycf68</i>, also may not code for a protein. This conserved motif has been reported in the <i>trnI</i>-GAU intron of rice (ORF133), corn (ORF133), <i>Pinus </i>(ORF75a), <i>Eucalyptus </i>(ORF113) and <i>Nymphaea</i>. Wheat and sugarcane also contain an ORF in this region apparently homologous to that of rice and corn [ <a href="#B30">30</a>]. We did not find an equivalent sequence in the chloroplast genome of any alga, which all lack an intron in the <i>trnI</i>-GAU gene, or in <i>Selaginella</i>, which lacks the <i>trnI</i>-GAU gene (Table <a href="#T3">3</a>). In the plastid genomes of all other vascular plant taxa examined, a similar sequence occurs in the <i>trnI </i>intron but in the majority of cases it contains numerous frameshifts and stop codons (Figure <a href="#F5">5</a>). Based only on its length and lack of internal stop codons, the <i>ycf68 </i>sequence could represent a functional protein-coding gene in the grasses, Nymphaeales, and in the gymnosperms <i>Pinus thunbergii </i>and <i>P. koraiensis</i>. However, in the others it can only be, at most, a pseudogene. Again, it seems unlikely that a non-functional gene could remain as conserved as the motif seen here over such vast evolutionary distances. If the sequence has any functional significance it must be other than coding for a protein, for example, in intron excision or in gene regulation. We did attempt to detect relationships between the conserved <i>ycf68 </i>motif and folding of the intron without success (see methods). We did not include <i>ycf68 </i>in the annotation of either <i>Nuphar </i>or <i>Ranunculus</i>. </p> <div class="table-wrap_UNKNOWN"> <span class="label" tagx="label" title="label">Table 3</span>  <p>Distribution of the <i>ycf68 </i>motif in completely sequenced chloroplast genomes. </p>  <table> <thead> <tr> <td>Species</td> <td>Accession number</td> <td>Absent <sup>a</sup> </td> <td>Present but cannot be functional <sup>b</sup> </td> <td>Present without internal stops <sup>c</sup> </td> </tr> </thead> <tbody> <tr> <td> <i>Nuphar advena</i> </td> <td> <a href="NC_008788">NC_008788</a> </td> <td /> <td /> <td>102 aa</td> </tr> <tr> <td> <i>Nymphaea alba</i> </td> <td> <a href="NC_006050">NC_006050</a> </td> <td /> <td /> <td>102 aa</td> </tr> <tr> <td> <i>Oryza sativa</i> </td> <td> <a href="NC_001320">NC_001320</a> </td> <td /> <td /> <td>134 aa</td> </tr> <tr> <td> <i>Pinus thunbergii</i> </td> <td> <a href="NC_001631">NC_001631</a> </td> <td /> <td /> <td>75 aa</td> </tr> <tr> <td> <i>Saccharum officinarum</i> </td> <td> <a href="NC_006084">NC_006084</a> </td> <td /> <td /> <td>134 aa</td> </tr> <tr> <td> <i>Triticum aestivum</i> </td> <td> <a href="NC_002762">NC_002762</a> </td> <td /> <td /> <td>144 aa</td> </tr> <tr> <td> <i>Zea mays</i> </td> <td> <a href="NC_001666">NC_001666</a> </td> <td /> <td /> <td>134 aa</td> </tr> <tr> <td> <i>Acorus calamus</i> </td> <td> <a href="NC_007407">NC_007407</a> </td> <td /> <td>X</td> <td /> </tr> <tr> <td> <i>Adiantum capillus-veneris</i> </td> <td> <a href="NC_004766">NC_004766</a> </td> <td /> <td>X</td> <td /> </tr> <tr> <td> <i>Amborella trichopoda</i> </td> <td> <a href="NC_005086">NC_005086</a> </td> <td /> <td>X</td> <td /> </tr> <tr> <td> <i>Anthocerus formosae</i> </td> <td> <a href="NC_004543">NC_004543</a> </td> <td /> <td>X</td> <td /> </tr> <tr> <td> <i>Arabidopsis thaliana</i> </td> <td> <a href="NC_000932">NC_000932</a> </td> <td /> <td>X</td> <td /> </tr> <tr> <td> <i>Atropa belladonna</i> </td> <td> <a href="NC_004561">NC_004561</a> </td> <td /> <td>X</td> <td /> </tr> <tr> <td> <i>Calycanthus floridus</i> </td> <td> <a href="NC_004993">NC_004993</a> </td> <td /> <td>X</td> <td /> </tr> <tr> <td> <i>Cucumis sativus</i> </td> <td> <a href="NC_007144">NC_007144</a> </td> <td /> <td>X</td> <td /> </tr> <tr> <td> <i>Epifagus virginiana</i> </td> <td> <a href="NC_001568">NC_001568</a> </td> <td /> <td>X</td> <td /> </tr> <tr> <td> <i>Eucalyptus globulus</i> <sup>d</sup> </td> <td> <a href="NC_008115">NC_008115</a> </td> <td /> <td>X</td> <td /> </tr> <tr> <td> <i>Huperzia lucidula</i> </td> <td> <a href="NC_006861">NC_006861</a> </td> <td /> <td>X</td> <td /> </tr> <tr> <td> <i>Lactuca sativa</i> </td> <td> <a href="NC_007578">NC_007578</a> </td> <td /> <td>X</td> <td /> </tr> <tr> <td> <i>Lotus japonicus</i> </td> <td> <a href="NC_002694">NC_002694</a> </td> <td /> <td>X</td> <td /> </tr> <tr> <td> <i>Marchantia polymorpha</i> </td> <td> <a href="NC_001319">NC_001319</a> </td> <td /> <td>X</td> <td /> </tr> <tr> <td> <i>Medicago polymorpha</i> </td> <td> <a href="NC_003119">NC_003119</a> </td> <td /> <td>X</td> <td /> </tr> <tr> <td> <i>Nicotiana tabacum</i> </td> <td> <a href="NC_001879">NC_001879</a> </td> <td /> <td>X</td> <td /> </tr> <tr> <td> <i>Oenothera elata</i> </td> <td> <a href="NC_002693">NC_002693</a> </td> <td /> <td>X</td> <td /> </tr> <tr> <td> <i>Panax ginseng</i> </td> <td> <a href="NC_006290">NC_006290</a> </td> <td /> <td>X</td> <td /> </tr> <tr> <td> <i>Phalaenopsis aphrodite</i> </td> <td> <a href="NC_007499">NC_007499</a> </td> <td /> <td>X</td> <td /> </tr> <tr> <td> <i>Physcomitrella patens</i> </td> <td> <a href="NC_005087">NC_005087</a> </td> <td /> <td>X</td> <td /> </tr> <tr> <td> <i>Psilotum nudum</i> </td> <td> <a href="NC_003386">NC_003386</a> </td> <td /> <td>X</td> <td /> </tr> <tr> <td> <i>Ranunculus macranthus</i> </td> <td> <a href="NC_008796">NC_008796</a> </td> <td /> <td>X</td> <td /> </tr> <tr> <td> <i>Spinacia oleracea</i> </td> <td> <a href="NC_002202">NC_002202</a> </td> <td /> <td>X</td> <td /> </tr> <tr> <td> <i>Chaetosphaeridium globosum</i> </td> <td> <a href="NC_004115">NC_004115</a> </td> <td>X <sup>e</sup> </td> <td /> <td /> </tr> <tr> <td> <i>Chlamydomonas reinhardtii</i> </td> <td> <a href="NC_005353">NC_005353</a> </td> <td>X <sup>e</sup> </td> <td /> <td /> </tr> <tr> <td> <i>Chlorella vulgaris</i> </td> <td> <a href="NC_001865">NC_001865</a> </td> <td>X <sup>e</sup> </td> <td /> <td /> </tr> <tr> <td> <i>Cyanidioschyzon merolae</i> </td> <td> <a href="NC_004799">NC_004799</a> </td> <td>X <sup>e</sup> </td> <td /> <td /> </tr> <tr> <td> <i>Cyanidium caldarium</i> </td> <td> <a href="NC_001840">NC_001840</a> </td> <td>X <sup>e</sup> </td> <td /> <td /> </tr> <tr> <td> <i>Cyanophora paradoxa</i> </td> <td> <a href="NC_001675">NC_001675</a> </td> <td>X <sup>e</sup> </td> <td /> <td /> </tr> <tr> <td> <i>Eimeria tenella</i> </td> <td> <a href="NC_004823">NC_004823</a> </td> <td>X <sup>e</sup> </td> <td /> <td /> </tr> <tr> <td> <i>Emiliania huxleyi</i> </td> <td> <a href="NC_007288">NC_007288</a> </td> <td>X <sup>e</sup> </td> <td /> <td /> </tr> <tr> <td> <i>Euglena gracilis</i> </td> <td> <a href="NC_001603">NC_001603</a> </td> <td>X <sup>e</sup> </td> <td /> <td /> </tr> <tr> <td> <i>Gracilaria tenuistipitata</i> </td> <td> <a href="NC_006137">NC_006137</a> </td> <td>X <sup>e</sup> </td> <td /> <td /> </tr> <tr> <td> <i>Guilardia theta</i> </td> <td> <a href="NC_000926">NC_000926</a> </td> <td>X <sup>e</sup> </td> <td /> <td /> </tr> <tr> <td> <i>Mesostigma viride</i> </td> <td> <a href="NC_002186">NC_002186</a> </td> <td>X <sup>e</sup> </td> <td /> <td /> </tr> <tr> <td> <i>Nephroselmis olivacea</i> </td> <td> <a href="NC_000927">NC_000927</a> </td> <td>X <sup>e</sup> </td> <td /> <td /> </tr> <tr> <td> <i>Odontella sinensis</i> </td> <td> <a href="NC_001713">NC_001713</a> </td> <td>X <sup>e</sup> </td> <td /> <td /> </tr> <tr> <td> <i>Porphyra purpurea</i> </td> <td> <a href="NC_000925">NC_000925</a> </td> <td>X <sup>e</sup> </td> <td /> <td /> </tr> <tr> <td> <i>Selaginella uncinata</i> </td> <td> <a href="NC_007625">NC_007625</a> </td> <td>X <sup>f</sup> </td> <td /> <td /> </tr> <tr> <td> <i>Toxoplasma gondii</i> </td> <td> <a href="NC_001799">NC_001799</a> </td> <td>X <sup>e</sup> </td> <td /> <td /> </tr> </tbody> </table> <div class="table-wrap-foot_UNKNOWN"> <p> <sup>a </sup>No significant similarity in blast search to published ycf68 regions anywhere in genome </p> <p> <sup>b </sup>A complete copy of the motif is present but there are internal stops making it "non-functional" </p> <p> <sup>c </sup> <i>ycf68 </i>motif occurs as ORF; Recognized as CDS in the genome annotation </p> <p> <sup>d </sup>Published <i>Eucalyptus </i>genome map [18] includes <i>ycf68</i>, but <i>ycf68 </i>region includes internal stops </p> <p> <sup>e</sup>no intron in <i>tRNA</i>-Ile </p> <p> <sup>f</sup>no <i>tRNA</i>-Ile </p> </div> </div> <div class="fig" title="fig"> <div class="figure" title="figure"> <a href="https://doi.org/"> <span class="label" tagx="label" title="label">Figure 5</span> </a> </div>   <p>Alignment of the <i>ycf68 </i>region in 14 representative angiosperms. <i>Amborella, Nuphar, Zea</i>, and <i>Spinacia </i>represent the form that includes intervening sequence. Codons highlighted in green represent start codons as annotated in the published grass genomes ( <i>Zea</i>, <i>Saccharum</i>, <i>Oryza </i>and <i>Triticum</i>) and <i>Nymphaea</i>. Gorymekin et al identified a later start codon in their annotation of the Nymphaea <i>ycf68 </i>in order to maintain an open reading frame. Codons highlighted in red represent in frame stop codons (in frame with the grass start codon in the initial part of the alignment and in frame with the <i>Nymphaea </i>start codon once that point is reached). In either frame, these sequences, although largely conserved at the nucleotide level, are not open in most taxa. </p>   </div> <p>To further investigate the properties of these two " <i>ycf</i>s" we applied two additional approaches: 1) graphing codon usage patterns, following Echols <i>et al</i>. [ <a href="#B31">31</a>], of these putative genes relative to that of known genes and non-coding regions; and 2) comparing the level of conservation in the <i>ycf15</i>- and <i>ycf68</i>-containing regions with other similar regions using Mulan [ <a href="#B32">32</a>]. Codon usage results were ambiguous (data not shown, see Methods). The Mulan results, though intuitive, were instructive (Figure <a href="#F6">6</a>). We compared the <i>ycf15</i>-containing intergenic spacer as well as five other similarly sized IGS (two from the IR regions and three from the LSC) to the homologous regions in <i>Nicotiana</i>, for each of 14 genomes (Figure <a href="#F6">6</a> top). For <i>ycf68</i>, we did similar comparisons, using <i>Zea </i>as the reference taxon, of the <i>trnI</i>-GAU ( <i>ycf68</i>-containing) intron, two other introns from the IR and three introns from the LSC (Figure <a href="#F6">6</a> bottom). In both cases, but especially for the intron ( <i>ycf68</i>) comparison, it can be seen that other IGS or intron sequences are as (or even more) conserved as the <i>ycf </i>regions and that non-coding sequences (introns or IGS) are much more conserved in the IR than in the LSC. Thus, it is seems likely that these regions ( <i>ycf15 </i>and <i>ycf68</i>) are conserved, not because they serve some function, but simply because they are in the inverted repeat; they simply appear to code for a polypeptide chain of suggestive length due to chance and are conserved across large evolutionary distances because of the especially low rates of change within the IR [ <a href="#B33">33</a>]. </p> <div class="fig" title="fig"> <div class="figure" title="figure"> <a href="https://doi.org/"> <span class="label" tagx="label" title="label">Figure 6</span> </a> </div>   <p>Sequence similarity comparisons of IGS and introns within the IR and the LSC. In both the top and bottom section, each 14 pairwise Mulan alignments is displayed as a histogram showing the similiarity (ranging from 50% to 100%) between each taxon (A-N) and the reference ( <i>Nicotiana </i>top or <i>Zea </i>bottom). The height of the blue histogram topped by the horizontal black lines indicates the degree of similarity; similarity histograms are blue except where we have re-colored yellow the regions equivalent to <i>ycf15 </i>(top) and <i>ycf68 </i>(bottom) to highlight those regions. [The black horizontal lines without blue bars subtending them indicate short regions of similarity, basically SDRs. Red bars above the histogram indicate evolutionary conserved regions as determined in Mulan.] In interpreting the diagram, essentially the more blue (or yellow) in a region, the more similar are the two sequences. (Top) Comparisons, relevant to the conservation of <i>ycf15</i>, of six IGS regions from <i>Nicotiana tobaccum </i>were made to <i>Calycanthus floridus </i>(A), <i>Amborella trichopoda </i>(B), <i>Zea mays </i>(C), <i>Saccharum officinarum </i>(D), <i>Phalaenopsis aphrodite </i>(E), <i>Lotus japonicus </i>(F), <i>Acorus calamus </i>(G), <i>Arabidopsis thaliana </i>(H), <i>Spinacia oleracea </i>(I), <i>Oenothera elata </i>(J), <i>Eucalyptus globulus </i>(K), <i>Nymphaea alba </i>(L), <i>Nuphar advena </i>(M), and <i>Ranunculus macranthus </i>(N). (Bottom) Comparisons, relevant to the conservation of <i>ycf68</i>, of introns from <i>Zea mays </i>were made to those of <i>Ranunculus macranthus </i>(A), <i>Calycanthus floridus </i>(B), <i>Eucalyptus globulus </i>(C), <i>Lotus japonicus </i>(D), <i>Spinacia oleracea </i>(E), <i>Phalaenopsis aphrodite </i>(F), <i>Nuphar advena </i>(G), <i>Nymphaea alba </i>(H), <i>Arabidopsis thaliana </i>(I), <i>Nicotiana tobaccum </i>(J), <i>Oenothera elata </i>(K), <i>Amborella trichopoda </i>(L), <i>Acorus calamus </i>(M), and <i>Saccharum officinarum </i>(N). </p>   </div> </div> <div class="codonusage" title="sec"> <div class="title" tagx="title" title="title">
Codon Usage</div> <p>We examined codon usage patterns in <i>Nuphar </i>and <i>Ranunculus </i>for the 79 protein-coding genes (i.e., not including the hypothetical genes, <i>ycf15 </i>and <i>ycf68</i>). We compared start codon usage in ten representative genomes being careful to compare homologous positions as much as possible. [This was straightforward among seed plants but sometimes problematic with <i>Huperzia</i>.] For five genes in each organism (with some but not all genes held in common), either ACG or GTG appear to be used as an alternative to ATG as the start codon, as is common for a variety of genes in the plastid genomes of seed plants (Table <a href="#T4">4</a>). In the pteridophytes, <i>Huperzia </i>(Table <a href="#T4">4</a>) and <i>Adiantum </i>[ <a href="#B34">34</a>], even more genes (10 and 26, respectively) use alternative start codons, and the pteridophyte repertoire includes GCG and ATT, in addition to ACG and GTG. </p> <div class="table-wrap_UNKNOWN"> <span class="label" tagx="label" title="label">Table 4</span>  <p>Alternative start codon usage in selected land plant genomes.</p>  <table> <thead> <tr> <td>Gene Species</td> <td> <i>Huperzia</i> </td> <td> <i>Pinus</i> </td> <td> <i>Amborella</i> </td> <td> <i>Nymphaea</i> </td> <td> <i>Nuphar</i> </td> <td> <i>Calycanthus</i> </td> <td> <i>Ranaunculus</i> </td> <td> <i>Arabidopsis</i> </td> <td> <i>Nicotiana</i> </td> <td> <i>Triticum</i> </td> </tr> </thead> <tbody> <tr> <td> <i>atpI</i> </td> <td>ACG</td> <td /> <td /> <td /> <td /> <td /> <td /> <td /> <td /> <td /> </tr> <tr> <td> <i>cemA</i> </td> <td /> <td /> <td>GTG</td> <td>GTG</td> <td>GTG</td> <td>GTG</td> <td /> <td /> <td /> <td /> </tr> <tr> <td> <i>chlL</i> </td> <td /> <td>GTG</td> <td>-----</td> <td>-----</td> <td>-----</td> <td>-----</td> <td>-----</td> <td>-----</td> <td>-----</td> <td>-----</td> </tr> <tr> <td> <i>matK</i> </td> <td>ACG</td> <td /> <td /> <td /> <td /> <td /> <td /> <td /> <td /> <td /> </tr> <tr> <td> <i>ndhB</i> </td> <td /> <td>-----</td> <td /> <td /> <td /> <td /> <td>ACG</td> <td /> <td /> <td /> </tr> <tr> <td> <i>ndhD</i> </td> <td>ACG</td> <td>-----</td> <td>ACG</td> <td /> <td>ACG</td> <td>ACG</td> <td>ACG</td> <td /> <td>GTG</td> <td /> </tr> <tr> <td> <i>ndhG</i> </td> <td>ATT</td> <td>-----</td> <td /> <td /> <td /> <td /> <td /> <td /> <td /> <td /> </tr> <tr> <td> <i>petN</i> </td> <td>GTG</td> <td /> <td /> <td /> <td /> <td /> <td /> <td /> <td /> <td /> </tr> <tr> <td> <i>psaJ</i> </td> <td>GTG</td> <td /> <td /> <td /> <td /> <td /> <td /> <td /> <td /> <td /> </tr> <tr> <td> <i>psbL</i> </td> <td /> <td /> <td>ACG</td> <td>ACG</td> <td>ACG</td> <td>ACG</td> <td /> <td /> <td>ACG</td> <td /> </tr> <tr> <td> <i>rpl2</i> </td> <td /> <td /> <td>ACG</td> <td /> <td /> <td /> <td>ACG</td> <td /> <td /> <td>ACG</td> </tr> <tr> <td> <i>rpl36</i> </td> <td>ACG</td> <td /> <td /> <td /> <td /> <td /> <td /> <td /> <td /> <td /> </tr> <tr> <td> <i>rpoC2</i> </td> <td>ACG</td> <td /> <td /> <td /> <td /> <td /> <td /> <td /> <td /> <td /> </tr> <tr> <td> <i>rps19</i> </td> <td /> <td /> <td /> <td>GTG</td> <td>GTG</td> <td>GTG</td> <td>GTG</td> <td>GTG</td> <td>GTG</td> <td>GTG</td> </tr> <tr> <td> <i>ycf1</i> </td> <td>ACG</td> <td /> <td /> <td /> <td /> <td /> <td /> <td /> <td /> <td /> </tr> <tr> <td> <i>ycf2</i> </td> <td>ATT</td> <td /> <td /> <td /> <td /> <td /> <td>GTG</td> <td /> <td /> <td /> </tr> <tr> <td> <hr /> </td> </tr> <tr> <td>Total Number</td> <td>10</td> <td>1</td> <td>4</td> <td>3</td> <td>4</td> <td>4</td> <td>5</td> <td>1</td> <td>3</td> <td>2</td> </tr> </tbody> </table> <div class="table-wrap-foot_UNKNOWN"> <p>Pseudogenes and ORFs were not included. Dashes in a cell indicate that that gene is absent from that genome. </p> </div> </div> <p>Overall codon usage in the <i>Nuphar </i>and <i>Ranunculus </i>genomes (Table <a href="#T5">5</a>) is generally similar to that reported from other genomes such as <i>Panax </i>[ <a href="#B17">17</a>], <i>Lotus </i>[ <a href="#B35">35</a>] and <i>Nicotiana </i>[ <a href="#B36">36</a>]. As in these and other genomes [ <a href="#B5">5</a>, <a href="#B6">6</a>, <a href="#B36">36</a>- <a href="#B38">38</a>] where the genetic code is redundant, codons with a third position nucleotide of A or T(U) are used more frequently than those terminating in G or C (Table <a href="#T5">5</a>). The base composition at each of the three codon positions varies, with the first position having the lowest proportion of A+T and the third position the highest (Table <a href="#T1">1</a>). It has been suggested that codon usage patterns are driven by this composition bias [ <a href="#B39">39</a>, <a href="#B40">40</a>]. However, when we apply methods to assess the impact of nucleotide composition on codon usage within <i>Nuphar </i>and <i>Ranunculus</i>, it appears that the A+T-richness of the third position is at most a partial influence on codon usage. We used CodonW [ <a href="#B41">41</a>] to calculate a variety of codon usage statistics that we then contrasted graphically (Figure <a href="#F7">7</a>). COrrespondence Analysis of codon usage (COA) was calculated based on codon usage as well as Relative Synonymous Codon Usage (RSCU). In each case the first two axes together explained only a modest amount of the variation (15–20%) and plotting each of the 56 degenerate codons on the first and second axes did not produce a pattern related to the A+T-richness of the third position in the codon sequence (Figure <a href="#F7">7</a> top), except that codons ending in A or T are more tightly clustered than the codons ending in G or C. These results suggest that A+T-richness is not the most important factor in explaining variation in codon use. In addition we graphed each gene on the two axes ENc (the effective number of codons) and GC3 (the G+C percentage at the third position) (Figure <a href="#F7">7</a> middle). If codon usage is random with respect to factors other than A+T-richness (i.e., GC content) of the third position, each gene is expected to fall on the prediction line for ENc based solely on its GC3 value [ <a href="#B42">42</a>, <a href="#B43">43</a>]. Again our results reinforce the finding that factors other than nucleotide composition are operating in the plastid genome to select among synonymous codons across genes, in that most points do not fall on the prediction line. Other studies, for example Wall and Herbeck's study of codon bias in the plastid gene <i>rbcL </i>[ <a href="#B44">44</a>], also have found that codon usage patterns are not explained by G+C patterns. Finally, we calculated COA eigen values for each gene for the <i>Nuphar </i>or <i>Ranunculus </i>genome (based on codon usage, RSCU or amino acid usage, the results are comparable although more obvious for codon or amino acid usage). The eigen values for the primary axes are higher (explain a higher proportion of the variation) than those seen for codons (about 25% in the case of the codon usage analyses) and plotting each gene on the first two axes produces patterns (Figure <a href="#F7">7</a> bottom) suggesting that different functional groups of genes have different codon usage and amino acid usage patterns as has been found in broader comparisons [ <a href="#B6">6</a>, <a href="#B36">36</a>]. Overall, although A+T-richness of the third position is the most obvious pattern observable in plastome codon usage, other factors are important in determining codon usage patterns in particular genes (and perhaps genomes). </p> <div class="table-wrap_UNKNOWN"> <span class="label" tagx="label" title="label">Table 5</span>  <p>Codon usage (codon frequency relative to each amino acid) for <i>Nuphar </i>(Nuad) and <i>Ranunculus </i>(Rama). </p>  <table> <thead> <tr> <td /> <td /> <td>Nuad</td> <td>Rama</td> <td /> <td /> <td>Nuad</td> <td>Rama</td> <td /> <td /> <td>Nuad</td> <td>Rama</td> <td /> <td /> <td>Nuad</td> <td>Rama</td> </tr> </thead> <tbody> <tr> <td>F</td> <td>TTT</td> <td> <b>59.0</b> </td> <td> <b>67.0</b> </td> <td>S</td> <td>TCT</td> <td> <b>27.0</b> </td> <td> <b>27.3</b> </td> <td>Y</td> <td>TAT</td> <td> <b>77.6</b> </td> <td> <b>80.1</b> </td> <td>C</td> <td>TGT</td> <td> <b>66.2</b> </td> <td> <b>76.7</b> </td> </tr> <tr> <td>F</td> <td> <b>TTC</b> </td> <td>41.0</td> <td>33.0</td> <td>S</td> <td> <b>TCC</b> </td> <td>17.0</td> <td>16.6</td> <td>Y</td> <td> <b>TAC</b> </td> <td>22.4</td> <td>19.9</td> <td>C</td> <td> <b>TGC</b> </td> <td>33.8</td> <td>23.3</td> </tr> <tr> <td>L</td> <td> <b>TTA</b> </td> <td> <b>28.6</b> </td> <td> <b>32.6</b> </td> <td>S</td> <td> <b>TCA</b> </td> <td>21.2</td> <td>19.4</td> <td>*</td> <td>TAA</td> <td> <b>40.5</b> </td> <td> <b>48.4</b> </td> <td>*</td> <td>TGA</td> <td>30.4</td> <td>25.3</td> </tr> <tr> <td>L</td> <td> <b>TTG</b> </td> <td>21.3</td> <td>21.0</td> <td>S</td> <td>TCG</td> <td>8.1</td> <td>10.1</td> <td>*</td> <td>TAG</td> <td>29.1</td> <td>26.4</td> <td>W</td> <td> <b>TGG</b> </td> <td>100.0</td> <td>100.0</td> </tr> <tr> <td>L</td> <td>CTT</td> <td>19.7</td> <td>21.4</td> <td>P</td> <td>CCT</td> <td> <b>37.4</b> </td> <td> <b>37.0 </b> </td> <td>H</td> <td>CAT</td> <td> <b>75.4</b> </td> <td> <b>75.0</b> </td> <td>R</td> <td> <b>CGT</b> </td> <td>21.1</td> <td>22.8</td> </tr> <tr> <td>L</td> <td>CTC</td> <td>7.4</td> <td>6.2</td> <td>P</td> <td>CCC</td> <td>21.7</td> <td>21.1</td> <td>H</td> <td> <b>CAC</b> </td> <td>24.6</td> <td>25.0</td> <td>R</td> <td>CGC</td> <td>7.8</td> <td>7.5</td> </tr> <tr> <td>L</td> <td> <b>CTA</b> </td> <td>14.7</td> <td>12.7</td> <td>P</td> <td> <b>CCA</b> </td> <td>28.9</td> <td>28.0</td> <td>Q</td> <td> <b>CAA</b> </td> <td> <b>71.7</b> </td> <td> <b>75.5</b> </td> <td>R</td> <td>CGA</td> <td>21.5</td> <td>22.5</td> </tr> <tr> <td>L</td> <td>CTG</td> <td>8.3</td> <td>6.0</td> <td>P</td> <td>CCG</td> <td>12.1</td> <td>13.8</td> <td>Q</td> <td>CAG</td> <td>28.3</td> <td>24.5</td> <td>R</td> <td>CGG</td> <td>7.3</td> <td>7.1</td> </tr> <tr> <td>I</td> <td>ATT</td> <td> <b>47.1</b> </td> <td> <b>51.9</b> </td> <td>T</td> <td>ACT</td> <td> <b>39.7</b> </td> <td> <b>37.8</b> </td> <td>N</td> <td>AAT</td> <td> <b>77.4</b> </td> <td> <b>77.4</b> </td> <td>S</td> <td>AGT</td> <td>20.6</td> <td>20.7</td> </tr> <tr> <td>I</td> <td> <b>ATC</b> </td> <td>22.1</td> <td>15.9</td> <td>T</td> <td> <b>ACC</b> </td> <td>20.6</td> <td>18.1</td> <td>N</td> <td> <b>AAC</b> </td> <td>22.6</td> <td>22.6</td> <td>S</td> <td> <b>AGC</b> </td> <td>6.1</td> <td>6.0</td> </tr> <tr> <td>I</td> <td>ATA</td> <td>30.8</td> <td>32.2</td> <td>T</td> <td> <b>ACA</b> </td> <td>28.3</td> <td>31.6</td> <td>K</td> <td> <b>AAA</b> </td> <td> <b>71.3</b> </td> <td> <b>75.2</b> </td> <td>R</td> <td> <b>AGA</b> </td> <td> <b>30.1</b> </td> <td> <b>29.0</b> </td> </tr> <tr> <td>M</td> <td> <b>ATG</b> </td> <td>100.0</td> <td>100.0</td> <td>T</td> <td>ACG</td> <td>11.3</td> <td>12.5</td> <td>K</td> <td>AAG</td> <td>28.7</td> <td>24.8</td> <td>R</td> <td>AGG</td> <td>12.3</td> <td>11.1</td> </tr> <tr> <td>V</td> <td>GTT</td> <td>35.1</td> <td>37.9</td> <td>A</td> <td>GCT</td> <td> <b>45.9</b> </td> <td> <b>41.5</b> </td> <td>D</td> <td>GAT</td> <td> <b>78.7</b> </td> <td> <b>78.7</b> </td> <td>G</td> <td>GGT</td> <td>34.5</td> <td>34.2</td> </tr> <tr> <td>V</td> <td> <b>GTC</b> </td> <td>12.7</td> <td>10.7</td> <td>A</td> <td>GCC</td> <td>15.5</td> <td>16.1</td> <td>D</td> <td> <b>GAC</b> </td> <td>21.3</td> <td>21.3</td> <td>G</td> <td> <b>GGC</b> </td> <td>10.3</td> <td>9.2</td> </tr> <tr> <td>V</td> <td> <b>GTA</b> </td> <td> <b>36.1</b> </td> <td> <b>38.5 </b> </td> <td>A</td> <td> <b>GCA</b> </td> <td>27.4</td> <td>29.0</td> <td>E</td> <td> <b>GAA</b> </td> <td> <b>72.8</b> </td> <td> <b>71.9</b> </td> <td>G</td> <td> <b>GGA</b> </td> <td> <b>38.0</b> </td> <td> <b>38.6</b> </td> </tr> <tr> <td>V</td> <td>GTG</td> <td>16.1</td> <td>12.9</td> <td>A</td> <td>GCG</td> <td>11.2</td> <td>13.4</td> <td>E</td> <td>GAG</td> <td>27.2</td> <td>28.1</td> <td>G</td> <td>GGG</td> <td>17.1</td> <td>17.9</td> </tr> </tbody> </table> <div class="table-wrap-foot_UNKNOWN"> <p>Codons shown in bold complement the anticodons of the tRNAs encoded in the chloroplast genome. Frequencies shown in bold indicate the most common codon (where synonymous codons exist for that amino acid or termination). </p> </div> </div> <div class="fig" title="fig"> <div class="figure" title="figure"> <a href="https://doi.org/"> <span class="label" tagx="label" title="label">Figure 7</span> </a> </div>   <p>Graphical analyses of codon usage patterns. (top) Plots of the two most significant axes generated by the COA of RSCU values for <i>Nuphar </i>(top left) and <i>Ranunculus </i>(top right). Each point represents one of the 59 degenerate codons. The points are coded S (black circle) if the 3 <sup>rd </sup>position nucleotide is G or C, and W (red square) if the 3 <sup>rd </sup>position nucleotide is A or T. (middle) Plots of ENc (effective number of codons) by GC3 (the percentage G + C at the 3 <sup>rd </sup>position) for each of the 79 protein-coding genes in <i>Nuphar </i>(middle left) and <i>Ranunculus </i>(middle right). The line in each graph (middle left and right) indicates the relationship predicted if codon usage was determined solely by 3 <sup>rd </sup>position composition. (bottom) Plots of the two most significant axes generated by COA on CU (codon usage) for genes in <i>Nuphar </i>(bottom left) and <i>Ranunculus </i>(bottom right). Each gene is categorized as related to photosynthesis (green diamonds), gene expression (black circles) or other (red squares). </p>   </div> </div> <div class="comparisonsofnupharandnymphaea" title="sec"> <div class="title" tagx="title" title="title">
Comparisons of Nuphar and Nymphaea</div> <p>Between the <i>Nuphar </i>and <i>Nymphaea </i>chloroplast genomes, we compared each individual gene, intron or intergenic spacer (IGS) region and calculated percent similarity (Table <a href="#T6">6</a>, Figure <a href="#F1">1</a>). For summary calculations, only one copy of each region in the IR was included and identities for multiple introns within a single gene were calculated separately. More than two-thirds of <i>Nuphar </i>regions match the homologous region of <i>Nymphaea </i>at a similarity level of 95% or higher; only 3% of the genomes fell below a threshold of 70% identity. As expected, coding regions are more highly conserved than IGS on average, although not in all cases. All four genes for rRNAs are 100% identical and those for tRNAs have at least 95% identity. Three protein-coding genes are 90–94% identical and all others are at least 95% identical. In some cases IGS and introns are more similar than coding regions – 60% of introns and 41% of IGS are at least 95% identical in sequence between the two genomes. Interestingly, the distribution patterns of the numbers of IGS and the numbers of introns in the different percent identity categories appeared quite similar, suggesting that similar forces may impact both types of non-coding sequences. </p> <div class="table-wrap_UNKNOWN"> <span class="label" tagx="label" title="label">Table 6</span>  <p>Number of <i>Nuphar </i>plastome regions attaining different sequence identities relative to homologous <i>Nymphaea </i>plastome regions. </p>  <table> <thead> <tr> <td> <b>Region</b> </td> <td> <b>99–100%</b> </td> <td> <b>95–98%</b> </td> <td> <b>90–94%</b> </td> <td> <b>80–89%</b> </td> <td> <b>70–79%</b> </td> <td> <b>0–69%</b> </td> <td> <b>Total</b> </td> </tr> </thead> <tbody> <tr> <td> <b>Protein</b> </td> <td>17</td> <td>63</td> <td>3</td> <td>0</td> <td>0</td> <td>0</td> <td>83</td> </tr> <tr> <td> <b>Intergenic</b> </td> <td>16</td> <td>29</td> <td>27</td> <td>23</td> <td>8</td> <td>7</td> <td>110</td> </tr> <tr> <td> <b>Introns</b> </td> <td>4</td> <td>8</td> <td>5</td> <td>2</td> <td>1</td> <td>0</td> <td>20</td> </tr> <tr> <td> <b>rRNA</b> </td> <td>4</td> <td>0</td> <td>0</td> <td>0</td> <td>0</td> <td>0</td> <td>4</td> </tr> <tr> <td> <b>tRNA</b> </td> <td>19</td> <td>11</td> <td>0</td> <td>0</td> <td>0</td> <td>0</td> <td>30</td> </tr> <tr> <td> <hr /> </td> </tr> <tr> <td> <b>Total</b> </td> <td>60</td> <td>111</td> <td>35</td> <td>25</td> <td>9</td> <td>7</td> <td>247</td> </tr> </tbody> </table> </div> <p>Detailed percent identity comparisons are rarely reported for individual regions of entire chloroplast genomes. One study compared sugar cane with rice, maize and wheat and reported comparisons as one of three categories: 0–30%, 31–79% and 80–100% [ <a href="#B45">45</a>]. As one might expect, most of the regions fell into the latter category. Timme <i>et al</i>. [ <a href="#B46">46</a>] compared sequence divergence in both coding and non-coding regions between two completely sequenced chloroplast genomes from representatives of two of the subfamilies of Asteraceae. Their results showed that intergenic spacer regions were nearly two times as divergent as introns, and that the 10 most divergent coding sequences represent several different functional groups, including photosynthetic genes, ribosomal proteins, and <i>ndh </i>genes. Another study compared divergence within functional groups across four different species in three genera of Solanaceae and found that RNA and photosynthesis genes are the most conserved [ <a href="#B47">47</a>], consistent with the results from a second Solanceae comparison based on individual coding regions [ <a href="#B48">48</a>]. Kim and Lee [ <a href="#B17">17</a>], in their comparison of coding regions (omitting the tRNA genes) from 16 fully sequenced vascular plant chloroplast genomes, found the four rRNA genes to be the least divergent, followed <i>by psbA, psbD, rps12, psbE, psbL </i>and <i>petB</i>. The most conserved coding regions in <i>Nuphar-Nymphaea </i>are similar in that the four rRNA genes, <i>rps12, psbL </i>and <i>petB </i>are also among the most conservative genes but differ in including <i>petN, psbM</i>, and <i>rpl23 </i>among the 10 most conserved coding regions. </p> <p>We also examined indels (insertions and deletions) between <i>Nuphar </i>and <i>Nymphaea </i>which, of course, are much more likely to occur in IGS than in coding regions. In our study, 88.6% of insertions and 89.9% of deletions occurred in IGS (data not shown), comparable to results from a similar comparison between sugarcane and maize where 84.9% of insertions and 74.2% of deletions occurred in IGS [ <a href="#B45">45</a>]. In another study, the comparison of two varieties of rice ( <i>Oryza sativa</i>) found 110 indels between those two plastid genomes [ <a href="#B49">49</a>], whereas we found almost four times as many (413), but, of course, our taxa are less closely related. However a study comparing <i>Atropa belladonna </i>to <i>Nicotiana tabaccum</i>, a comparison perhaps more comparable to ours, found 65 insertions and 60 deletions equal to or larger than five bp in intergenic regions and introns [ <a href="#B50">50</a>], whereas we found 163 insertions and 206 deletions in <i>Nuphar </i>relative to <i>Nymphaea</i>. Together the combined lengths of the indels account for 0.08% of the total genome length of <i>Atropa </i>and 0.08% of the genome length in the rice comparison, but 0.23% of the genomic length in the <i>Nuphar-Nymphaea </i>comparison. </p> <p>Only recently have plastome sequences been available for closely related taxa, allowing for detailed comparisons [ <a href="#B46">46</a>- <a href="#B48">48</a>]. These comparisons suggest that non-coding sequences within the genome evolve more rapidly in terms of both substitution and indel mutations, although this is not universally so (some non-coding sequences are quite highly conserved). Comparisons across studies show that some regions are consistently slow to evolve and others commonly evolve at a higher rate but at least minor differences are seen across these studies. Before general patterns and processes can be identified more genomic data allowing for such comparisons will need to become available. </p> </div> <div class="repeatanalysis–simplesequencerepeats(ssrs)" title="sec"> <div class="title" tagx="title" title="title">
Repeat Analysis – Simple Sequence Repeats (SSRs)</div> <p>We screened for perfect SSRs in <i>Nuphar</i>, <i>Ranunculus </i>and 24 other chloroplast genomes (Table <a href="#T7">7</a>). We report the number of mononucleotide repeats ≥ 8 nt, dinucleotide repeats ≥ 8 nt (i.e., four repeat units), and trinucleotide repeats ≥ 9 nt (i.e., three repeat units); hereafter referred to collectively as the 8,8,9 SSRs. We also report the number of longer repeats ≥ 10 nt/repeats for mononucleotide runs, ≥ 10 nt (or five copies of the repeat unit) for dinucleotide repeats, and ≥ 12 nt (i.e., four copies of the repeat unit) for trinucleotide repeats; referred to collectively as the 10,10,12 SSRs. Of course, any particular threshold (e.g., 8,8,9) is rather arbitrary and no consensus has developed on what nucleotide length or repeat unit number is significant [ <a href="#B51">51</a>]. However, it has been suggested that SSRs of length 8 nt or more (regardless of repeat motif) are prone to slip-strand mispairing (SSM, thought to be the primary mutational mechanism to affect SSRs), whereas those of lesser length are not [ <a href="#B52">52</a>]. Elsewhere the critical threshold is estimated at 7–10 bp [ <a href="#B53">53</a>]. Other workers have chosen similar thresholds, of 8 or 10 nt, in their reports [e.g., [ <a href="#B54">54</a>]]. The number of 8,8,9 SSRs vary from 152 in <i>Pinus thunbergii </i>to 393 in <i>Lotus </i>and comprise between 1 and 2.5% of the chloroplast genome. The number of 10,10,12 SSRs range from 16 in one of the rice genomes to 113 in <i>Medicago</i>. </p> <div class="table-wrap_UNKNOWN"> <span class="label" tagx="label" title="label">Table 7</span>  <p>Number and maximum length of SSRs present in twenty-six land plant chloroplast genomes.</p>  <table> <thead> <tr> <td /> <td /> <td /> <td>8,8,9 SSRs (number)</td> <td>10, 10,12 SSRs (number)</td> <td /> <td /> <td /> </tr> <tr> <td /> <td /> <td /> <td> <hr /> </td> <td /> <td /> <td /> </tr> <tr> <td>Taxon</td> <td>Accession Number</td> <td>Genome size</td> <td>mono</td> <td>di</td> <td>tri</td> <td> <b>total</b> </td> <td>mono</td> <td>di</td> <td>tri</td> <td> <b>total</b> </td> <td>max mono (units)</td> <td>max di (units)</td> <td>max tri (units)</td> </tr> </thead> <tbody> <tr> <td> <i>Huperzia lucidula</i> </td> <td> <a href="NC_006861">NC_006861</a> </td> <td>154,373</td> <td>147</td> <td>34</td> <td>80</td> <td> <b>261</b> </td> <td>25</td> <td>10</td> <td>7</td> <td> <b>42</b> </td> <td>15</td> <td>9</td> <td>10</td> </tr> <tr> <td> <i>Psilotum nudum</i> </td> <td> <a href="NC_003386">NC_003386</a> </td> <td>138,829</td> <td>111</td> <td>35</td> <td>58</td> <td> <b>204</b> </td> <td>28</td> <td>12</td> <td>3</td> <td> <b>43</b> </td> <td>17</td> <td>6</td> <td>5</td> </tr> <tr> <td> <i>Adiantum capillus-veneris</i> </td> <td> <a href="NC_004766">NC_004766</a> </td> <td>150,568</td> <td>146</td> <td>32</td> <td>39</td> <td> <b>217</b> </td> <td>94</td> <td>4</td> <td>2</td> <td> <b>100</b> </td> <td>19</td> <td>6</td> <td>4</td> </tr> <tr> <td> <i>Pinus thunbergii</i> </td> <td> <a href="NC_001631">NC_001631</a> </td> <td>119,707</td> <td>74</td> <td>36</td> <td>42</td> <td> <b>152</b> </td> <td>20</td> <td>5</td> <td>0</td> <td> <b>25</b> </td> <td>17</td> <td>7</td> <td>3</td> </tr> <tr> <td> <i>Pinus koraiensis</i> </td> <td> <a href="NC_004677">NC_004677</a> </td> <td>116,866</td> <td>77</td> <td>38</td> <td>39</td> <td> <b>154</b> </td> <td>36</td> <td>6</td> <td>0</td> <td> <b>42</b> </td> <td>23</td> <td>9</td> <td>3</td> </tr> <tr> <td> <i>Oryza nivara</i> </td> <td> <a href="NC_005973">NC_005973</a> </td> <td>134,494</td> <td>91</td> <td>20</td> <td>42</td> <td> <b>153</b> </td> <td>10</td> <td>4</td> <td>3</td> <td> <b>17</b> </td> <td>13</td> <td>5</td> <td>4</td> </tr> <tr> <td> <i>Oryza sativa indica 93-11</i> </td> <td> <a href="NC_008155">NC_008155</a> </td> <td>134,496</td> <td>94</td> <td>20</td> <td>42</td> <td> <b>156</b> </td> <td>9</td> <td>4</td> <td>3</td> <td> <b>16</b> </td> <td>16</td> <td>5</td> <td>4</td> </tr> <tr> <td> <i>O. sativa japonica Nipponbare</i> </td> <td> <a href="AY522330">AY522330</a> </td> <td>134,551</td> <td>94</td> <td>20</td> <td>42</td> <td> <b>156</b> </td> <td>11</td> <td>4</td> <td>3</td> <td> <b>18</b> </td> <td>17</td> <td>5</td> <td>4</td> </tr> <tr> <td> <i>O. sativa japonica PA64S</i> </td> <td> <a href="AY522331">AY522331</a> </td> <td>134,551</td> <td>94</td> <td>20</td> <td>42</td> <td> <b>156</b> </td> <td>11</td> <td>4</td> <td>3</td> <td> <b>18</b> </td> <td>17</td> <td>5</td> <td>4</td> </tr> <tr> <td> <i>Saccharum officinarum</i> </td> <td> <a href="NC_006084">NC_006084</a> </td> <td>141,182</td> <td>128</td> <td>26</td> <td>47</td> <td> <b>201</b> </td> <td>32</td> <td>5</td> <td>1</td> <td> <b>38</b> </td> <td>15</td> <td>5</td> <td>4</td> </tr> <tr> <td> <i>Zea mays</i> </td> <td> <a href="NC_001666">NC_001666</a> </td> <td>140,384</td> <td>121</td> <td>29</td> <td>52</td> <td> <b>202</b> </td> <td>34</td> <td>6</td> <td>2</td> <td> <b>42</b> </td> <td>18</td> <td>6</td> <td>5</td> </tr> <tr> <td> <i>Triticum aestivum</i> </td> <td> <a href="NC_002762">NC_002762</a> </td> <td>134,545</td> <td>116</td> <td>33</td> <td>43</td> <td> <b>192</b> </td> <td>24</td> <td>8</td> <td>3</td> <td> <b>35</b> </td> <td>15</td> <td>6</td> <td>5</td> </tr> <tr> <td> <i>Nuphar advena</i> </td> <td> <a href="NC_008788">NC_008788</a> </td> <td>160,866</td> <td>71</td> <td>65</td> <td>84</td> <td> <b>220</b> </td> <td>19</td> <td>23</td> <td>10</td> <td> <b>52</b> </td> <td>16</td> <td>11</td> <td>5</td> </tr> <tr> <td> <i>Nymphaea alba</i> </td> <td> <a href="NC_006050">NC_006050</a> </td> <td>159,939</td> <td>63</td> <td>60</td> <td>73</td> <td> <b>196</b> </td> <td>15</td> <td>4</td> <td>4</td> <td> <b>23</b> </td> <td>16</td> <td>5</td> <td>4</td> </tr> <tr> <td> <i>Amborella trichopoda</i> </td> <td> <a href="NC_005086">NC_005086</a> </td> <td>162,686</td> <td>101</td> <td>47</td> <td>57</td> <td> <b>205</b> </td> <td>35</td> <td>8</td> <td>6</td> <td> <b>49</b> </td> <td>15</td> <td>9</td> <td>4</td> </tr> <tr> <td> <i>Calycanthus fertilis</i> </td> <td> <a href="NC_004993">NC_004993</a> </td> <td>153,337</td> <td>105</td> <td>35</td> <td>93</td> <td> <b>233</b> </td> <td>14</td> <td>8</td> <td>4</td> <td> <b>26</b> </td> <td>13</td> <td>8</td> <td>4</td> </tr> <tr> <td> <i>Ranunculus macranthus</i> </td> <td> <a href="NC_008796">NC_008796</a> </td> <td>155,158</td> <td>146</td> <td>60</td> <td>55</td> <td> <b>261</b> </td> <td>28</td> <td>9</td> <td>3</td> <td> <b>40</b> </td> <td>16</td> <td>8</td> <td>5</td> </tr> <tr> <td> <i>Arabidopsis thaliana</i> </td> <td> <a href="NC_000932">NC_000932</a> </td> <td>154,478</td> <td>234</td> <td>83</td> <td>61</td> <td> <b>378</b> </td> <td>69</td> <td>18</td> <td>6</td> <td> <b>93</b> </td> <td>17</td> <td>8</td> <td>5</td> </tr> <tr> <td> <i>Oenothera elata</i> </td> <td> <a href="NC_002693">NC_002693</a> </td> <td>163,935</td> <td>155</td> <td>48</td> <td>68</td> <td> <b>271</b> </td> <td>56</td> <td>6</td> <td>8</td> <td> <b>70</b> </td> <td>24</td> <td>6</td> <td>4</td> </tr> <tr> <td> <i>Panax ginseng</i> </td> <td> <a href="NC_006290">NC_006290</a> </td> <td>156,318</td> <td>92</td> <td>39</td> <td>60</td> <td> <b>191</b> </td> <td>18</td> <td>5</td> <td>3</td> <td> <b>26</b> </td> <td>13</td> <td>7</td> <td>4</td> </tr> <tr> <td> <i>Nicotiana tabacum</i> </td> <td> <a href="NC_001879">NC_001879</a> </td> <td>155,939</td> <td>116</td> <td>41</td> <td>73</td> <td> <b>230</b> </td> <td>38</td> <td>7</td> <td>5</td> <td> <b>50</b> </td> <td>17</td> <td>5</td> <td>4</td> </tr> <tr> <td> <i>Atropa belladonna</i> </td> <td> <a href="NC_004561">NC_004561</a> </td> <td>156,687</td> <td>116</td> <td>46</td> <td>74</td> <td> <b>236</b> </td> <td>39</td> <td>10</td> <td>2</td> <td> <b>51</b> </td> <td>17</td> <td>6</td> <td>4</td> </tr> <tr> <td> <i>Spinacia oleracea</i> </td> <td> <a href="NC_002202">NC_002202</a> </td> <td>150,725</td> <td>146</td> <td>55</td> <td>64</td> <td> <b>265</b> </td> <td>40</td> <td>9</td> <td>4</td> <td> <b>53</b> </td> <td>12</td> <td>7</td> <td>4</td> </tr> <tr> <td> <i>Epifagus virginiana</i> </td> <td> <a href="NC_001568">NC_001568</a> </td> <td>70,028</td> <td>106</td> <td>36</td> <td>50</td> <td> <b>192</b> </td> <td>25</td> <td>16</td> <td>4</td> <td> <b>45</b> </td> <td>15</td> <td>10</td> <td>5</td> </tr> <tr> <td> <i>Lotus japonicus</i> </td> <td> <a href="NC_002694">NC_002694</a> </td> <td>150,519</td> <td>236</td> <td>80</td> <td>77</td> <td> <b>393</b> </td> <td>76</td> <td>27</td> <td>6</td> <td> <b>109</b> </td> <td>16</td> <td>11</td> <td>5</td> </tr> <tr> <td> <i>Medicago truncatula</i> </td> <td> <a href="NC_003119">NC_003119</a> </td> <td>124,033</td> <td>190</td> <td>63</td> <td>93</td> <td> <b>346</b> </td> <td>76</td> <td>28</td> <td>9</td> <td> <b>113</b> </td> <td>18</td> <td>7</td> <td>6</td> </tr> </tbody> </table> </div> <p>We calculated, based on the data presented in Table <a href="#T7">7</a>, Spearman Rank Correlation statistics [using [ <a href="#B55">55</a>]] to look for relationships between the number of short and long SSRs and between genome size and the total number of SSRs (Figure <a href="#F8">8</a>). If some genomes are generally more likely to contain SSRs (due to differences in mutational biases or selection pressures or other factors) and a common mechanism (or suite of mechanisms) controlled the likelihood of SSR presence, then a correlation between short and long SSRs would be predicted. Alternatively if some genomes were predisposed to long SSRs whereas others were less likely for SSRs to attain greater length, then no correlation might be seen or even a negative correlation might be observed. Genome size and total number of SSRs should be correlated if SSRs occur randomly. Both these comparisons showed a small but significant positive relationship. Genomes with a higher number of "short" SSRs (from the first to the second threshold) were more likely to have a higher number of "long" SSRs (at or above the second threshold) – Figure <a href="#F8">8</a> (top), r <sub>s </sub>= 0.534, p = 0.009. Larger genomes were more likely to contain more SSRs than smaller genomes – Figure <a href="#F8">8</a> (middle), r <sub>s </sub>= 0.542, p = 0.008 (r <sub>s </sub>= 0.524, p = 0.012 with <i>Epifagus </i>excluded). Thus we can infer that the larger the genome the more SSRs are to be expected and that "long" and "short" SSRs most likely are simply points on a continuum evolving under similar mechanisms. However these two factors explain only a portion of the variance in SSR number. </p> <div class="fig" title="fig"> <div class="figure" title="figure"> <a href="https://doi.org/"> <span class="label" tagx="label" title="label">Figure 8</span> </a> </div>   <p>Scatter plots showing relationships between aspects of SSR frequency and other characteristics. (top) The relationship of "short" SSRs and "long" SSRs. "Long" SSRs are the 10,10.12 repeats. "Short" SSRs are the 8,8,9 repeats with the 10,10,12 repeats excluded. These are shown for the 24 taxa in Table 7. (middle) The relationship between total SSR number and genome size (in nucleotides) for the 24 taxa. (bottom) The relationship of A+T-richness (the overall A+T percentage of the genome) and the frequency of A and T mononucleotide repeats for the 10 taxa involved in the more detailed comparison. No other SSR category showed a relationship to any aspect of nucleotide composition. </p>   </div> <p>For a subset of the chloroplast genomes, we conducted a more detailed comparison of the SSRs (data shown in additional file <a href="#S1">1</a>), including all of the then available basal angiosperms and basal eudicots, plus a pteridophyte, a conifer, two derived eudicots and one monocot. In these comparisons we characterized repeats based on nucleotide composition in addition to length and number. We then compared (data not shown) the predicted and observed number of repeats of various types and the predicted and observed ratios between repeats of various compositions. For all types of repeats, of all lengths and all compositions, in all genomes, fewer SSRs were observed than were predicted. In many cases the difference between the number of SSRs predicted and the number observed was more than an order of magnitude. For mononucleotide repeats in most angiosperms, the ratio of A or T (W) containing runs to G or C (S) containing runs was not significantly different from the ratio predicted from the genomic base composition. However, in the two Nymphaeaceae ( <i>Nuphar </i>and <i>Nymphaea</i>), <i>Pinus </i>and <i>Huperzia</i>, the observed ratio of W to S mononucleotide SSRs was significantly different from the predicted ratio (Chi-Square test, p &amp;lt; 0.001). Interestingly the observed ratio was skewed in favor of S (G and C) containing repeats. When predicted ratios of dinucleotide repeats were calculated based on the overall nucleotide composition, all genomes showed a significantly biased ratio (Chi-square test, p &amp;lt; 0.001). In this case more WS SSRs were observed than expected and fewer WW SSRs. When the dinucleotide calculations were repeated based on the genomic dinucleotide frequencies, the deviation of the observed from the expected was less and for <i>Arabidopsis </i>and <i>Triticum </i>was not significant. Again where there is a deviation from expected, the WS category is elevated and the WW category is reduced in the observed. Finally, in calculations related to trinucleotide SSR compositions and based on overall trinucleotide composition, all but <i>Arabidopsis </i>and <i>Triticum </i>showed biased compositional ratios. Bias was due primarily to fewer WWW SSRs than expected. Thus, surprisingly, whenever the observed compositional ratio of SSRs was different from expectation (the majority of cases), the deviation was due to a deficiency of W (A and T) containing repeats. </p> <p>Other (non-plastid) genomes that are A+T rich have been found to exhibit a strong overrepresentation of A or T mononucleotide SSRs, e.g., <i>Caenorhabditis elegans </i>[ <a href="#B56">56</a>] and <i>Plasmodium falciparum </i>[ <a href="#B53">53</a>] and various explanations for this have been offered. Because A-T base pairs are held together by two hydrogen bonds rather than three, A-T containing repeats are easier to denature and therefore perhaps more prone to slip-strand mispairing (SSM). Other explanations include mutational bias (i.e., G-C to A-T mutations being more likely than the reverse [ <a href="#B56">56</a>]), involvement of A-T runs in gene regulation [ <a href="#B53">53</a>], and regions being more mutagenic due to A-T runs (this increased mutability being selected for in regions where higher mutations rates would be advantageous [ <a href="#B56">56</a>, <a href="#B57">57</a>]). Although these proposals seem reasonable, SSRs (and specifically A and T containing SSRs) are underrepresented in the plastid genomes in contrast also to the work of Dieringer and Schlotterer [ <a href="#B58">58</a>] who found SSRs overrepresented when comparing nine nuclear genomes. They predicted that the more biased from equal nucleotide composition the genome the greater the density of microsatellites (either mono- or di- nucleotide SSRs in their case). For the genomes considered in our detailed comparison, we tested for correlation between various aspects of nucleotide composition and SSR frequency. Only A and T mononucleotide repeat density correlated with genomic nucleotide composition (Figure <a href="#F8">8</a> bottom, r <sub>s </sub>= 0.81, p = 0.015). So this correlation can be seen even when the W repeats are not over represented. G and C mononucleotide repeat density did not correlate with nucleotide composition and di- and tri-nucleotide repeat densities correlated neither with genomic frequencies nor with di- or tri-nucleotide frequencies calculated over the entire genome (data not shown). We note that GpC and CpG based (SS) repeats are completely absent from all the plastid genomes examined. This may be the result of selection against these motifs in addition to simple nucleotide composition (these would be predicted to be very rare; about one SS repeat of length 8 nt per genome on average) as SS-based dinucleotide repeats are absent or strongly underrepresented in other genomes as well [ <a href="#B51">51</a>, <a href="#B58">58</a>, <a href="#B59">59</a>]. Presumably some mechanism (mutational or selectional) in the chloroplast genome is acting on the plastid genomes such that many fewer SSRs are observed than would be expected and that mechanism is acting more strongly on W-containing repeats than S-containing ones. </p> <p>In an alternative attempt to clarify underlying patterns and mechanisms of SSR evolution, we compared the 10,10,12 SSRs between <i>Nuphar </i>and <i>Nymphaea </i>to determine how many of the SSRs were shared (determined by identity of flanking sequence and repeat position as well as repeat motif) between these two genomes from relatively closely related plants (additional file <a href="#S2">2</a>). The <i>Nuphar </i>genome contains more than twice as many 10,10,12 SSRs as the <i>Nymphaea </i>genome – 52 in <i>Nuphar</i>, 23 in <i>Nymphaea</i>. Sixty-six different 10,10,12 SSRs occur in the combined set. The vast majority, but not all, of the SSRs occur in non-coding DNA (55 in IGS, six in introns, and five in coding regions). The majority (50 of 66) of these repeats are shared between the two genomes; most are simply lower than the reporting threshold in one genome or the other. Of the 50 shared SSRs, only seven are the same length in both genomes. Where the shared SSRs are of different lengths, in 18 cases length differences were due to SSM, in 14 cases due to a nucleotide substitution, in four cases due to an indel (a length mutation other than a change in repeat unit number) and in eight cases due to a combination of mechanisms. Of the 16 SSRs not shared, 14 were found only in <i>Nuphar </i>and only two in <i>Nymphaea</i>. Thus, it appears that <i>Nuphar </i>is more likely to have these SSRs and for the SSRs to be longer when they occur; in 30 of the 43 cases where shared repeats differ in length the <i>Nuphar </i>SSR is longer. </p> <p>It is thought that SSRs begin as random runs of nucleotides [ <a href="#B51">51</a>, <a href="#B60">60</a>]. Any bias in mutation patterns or nucleotide composition would make certain runs more likely. Then, once present in a location, the repeat would grow via SSM [ <a href="#B51">51</a>, <a href="#B57">57</a>, <a href="#B58">58</a>, <a href="#B60">60</a>]. Longer SSRs lead to more stable heteroduplex intermediates, making SSM more likely [ <a href="#B57">57</a>]. However longer SSRs also have higher mutations rates [ <a href="#B51">51</a>]. One model of SSR evolution posits that the distribution of repeat lengths in a genome represents an equilibrium between SSM and point mutation [ <a href="#B51">51</a>]. In the <i>Nuphar-Nymphaea </i>comparison SSM and point mutation occur with about equal frequency, consistent with this hypothesis. In terms of phylogenetic utility of SSR variation, the <i>Nuphar-Nymphaea </i>comparison suggests that individual SSRs are stable at least over relatively short periods of evolutionary time and that they do commonly vary in repeat number. However the small size of most repeats probably limits their utility and more needs to be known about the specifics of SSR evolution before any phylogenetic utility can be fully realized. Understanding more about the processes of SSR evolution will also help us investigate possible selective or functional roles for these motifs. </p> </div> <div class="repeatanalysis–smalldispersedrepeats(sdr)" title="sec"> <div class="title" tagx="title" title="title">
Repeat Analysis – Small Dispersed Repeats (SDR)</div> <p>We also searched for SDRs in the plastomes of representative angiosperms. These repeats are based on a more complex motif and are longer than SSRs. Our SDR analysis, within each of the eight genomes, identified 114–350 direct and inverted repeats 30 bp or longer with a sequence identity of at least 80% (Figure <a href="#F9">9</a>). The number of repeats was lowest in <i>Nymphaea </i>(114) and highest in <i>Arabidopsis </i>(350). In most cases, the number of direct repeats (62–208) was substantially higher than the number of inverted repeats (32–142). The vast majority (84–97%) of the repeats were only 30–40 bp in length and the longest repeat was 193 bp in <i>Triticum</i>. </p> <div class="fig" title="fig"> <div class="figure" title="figure"> <a href="https://doi.org/"> <span class="label" tagx="label" title="label">Figure 9</span> </a> </div>   <p>The number of SDRs of different length classes found in eight different angiosperm plastid genomes. The majority of repeats are 40 nt or less in length, but some genomes so have repeats that are longer. <i>Triticum</i>, the only genome to have repeats over 100 nt in length, is also the only genome to exhibit inversions changing aspects of gene order from the angiosperm consensus order exhibited by <i>Nicotiana </i>(and the other genomes included.) </p>   </div> <p>Blast comparisons of the repeats identified in each of the genomes were performed against all other genomes to locate shared repeats with an e-value of 2. Although these comparisons were performed using the repeats in each of the eight genomes as the reference, we only present the results (Table <a href="#T8">8</a>) using <i>Nymphaea </i>as the reference genome because it had the fewest number of repeats and thus would contain any repeats shared throughout all these genomes. Overall, the analyses identified 83 groups of shared repeats among these eight angiosperm chloroplast genomes, ranging in length from 30 to 49 bp. The majority of the shared repeats were located within intergenic spacer regions and introns. However repeats were sometimes found within genes and some cases represented inter-tRNA similarities within different families of tRNA genes. The largest shared repeat (49 bp) represents shared sequence between <i>psaA </i>and <i>psaB</i>. In general, the longer shared SDRs (36 – 49 bp) tend to have lower A+T-richness (at the level of the genome or lower). However, most (68 of the 83) of the shared SDRs are short, between 30 and 32 nt in length. This class of SDRs shows the widest range in A+T-richness (29% to 97%) but most exhibit A+T-richness of greater than 50%. Most shared SDRs are present in only a small number of copies. Only two repeats occur more than four times in the genomes (both are found in introns and IGS and exhibit A+T richness less than that of the genomes overall); one is 42 nt long (51% AT) and the other, occurring about 10 times, is 40 nt long (54% AT). Overall, no trends were detected between repeat location, number, length, and A+T-richness. </p> <div class="table-wrap_UNKNOWN"> <span class="label" tagx="label" title="label">Table 8</span>  <p>Shared repeats among eight angiosperm chloroplast genomes.</p>  <table> <thead> <tr> <td> <b>Repeat Size (AT%)</b> </td> <td> <b> <i>Nymphaea </i>(reference genome) </b> </td> <td> <b> <i>Amborella</i> </b> </td> <td> <b> <i>Nuphar</i> </b> </td> <td> <b> <i>Calycanthus</i> </b> </td> <td> <b> <i>Triticum</i> </b> </td> <td> <b> <i>Ranunculus</i> </b> </td> <td> <b> <i>Arabidopsis</i> </b> </td> <td> <b> <i>Nicotiana</i> </b> </td> <td> <b>Location</b> </td> </tr> </thead> <tbody> <tr> <td>49(64)</td> <td>2</td> <td>2</td> <td>2</td> <td>2</td> <td>2</td> <td>2</td> <td>2</td> <td>2</td> <td> <i>psaA</i>, <i>psaB</i> </td> </tr> <tr> <td>43(61)</td> <td>4</td> <td>4</td> <td>4</td> <td>4</td> <td>0</td> <td>4</td> <td>2</td> <td>2</td> <td> <i>ycf2</i> </td> </tr> <tr> <td>42(51)</td> <td>5</td> <td>6</td> <td>7</td> <td>7</td> <td>5</td> <td>5</td> <td>6</td> <td>7</td> <td>intron <i>ycf3</i>, intron <i>rpoC1</i>, IGS <i>psbH</i>: <i>petB</i>, IGS <i>petD</i>: <i>petB</i>, IGS <i>rpl16</i>: <i>rps3</i>, IGS <i>trnV</i>: <i>rps12 </i>3' </td> </tr> <tr> <td>40(54)</td> <td>11</td> <td>10</td> <td>11</td> <td>11</td> <td>11</td> <td>7</td> <td>9</td> <td>10</td> <td>intron <i>ycf3</i>, intron <i>ndhB</i>, IGS <i>trnV</i>: <i>rps12 </i>3', IGS <i>rpl16</i>: <i>rps3</i>, IGS <i>psbH</i>: <i>petB</i>, IGS <i>rps12 </i>3': <i>rps7</i> </td> </tr> <tr> <td>37(39)</td> <td>2</td> <td>2</td> <td>2</td> <td>2</td> <td>1</td> <td>2</td> <td>2</td> <td>2</td> <td>IGS <i>trnS</i>: <i>trnR</i> </td> </tr> <tr> <td>36(51)</td> <td>3</td> <td>2</td> <td>3</td> <td>2</td> <td>2</td> <td>3</td> <td>2</td> <td>3</td> <td> <i>trnV</i>, <i>trnA</i> </td> </tr> <tr> <td>35(75)</td> <td>2</td> <td>0</td> <td>1</td> <td>0</td> <td>0</td> <td>0</td> <td>0</td> <td>0</td> <td>IGS <i>rps12 </i>5': <i>clpP</i> </td> </tr> <tr> <td>35(47)</td> <td>3</td> <td>3</td> <td>3</td> <td>3</td> <td>3</td> <td>3</td> <td>3</td> <td>3</td> <td> <i>trnS</i> </td> </tr> <tr> <td>35(100)</td> <td>3</td> <td>2</td> <td>4</td> <td>1</td> <td>0</td> <td>2</td> <td>4</td> <td>1</td> <td>intron <i>trnL</i>, IGS <i>ndhC</i>: <i>trnV</i> </td> </tr> <tr> <td>34(91)</td> <td>2</td> <td>2</td> <td>2</td> <td>2</td> <td>2</td> <td>1</td> <td>2</td> <td>2</td> <td> <i>trnfM</i>, <i>trnP</i> </td> </tr> <tr> <td>34(86)</td> <td>2</td> <td>0</td> <td>2</td> <td>0</td> <td>0</td> <td>0</td> <td>0</td> <td>0</td> <td>IGS <i>atpH</i>: <i>atpI</i>, IGS <i>trnS</i>: <i>trnR</i> </td> </tr> <tr> <td>34(54)</td> <td>4</td> <td>1</td> <td>1</td> <td>0</td> <td>1</td> <td>0</td> <td>4</td> <td>0</td> <td>intron <i>trnL</i>, IGS <i>trnE</i>: <i>trnT</i> </td> </tr> <tr> <td>33(59)</td> <td>2</td> <td>3</td> <td>2</td> <td>2</td> <td>3</td> <td>2</td> <td>3</td> <td>2</td> <td>intron <i>atpF</i>, IGS <i>trnS</i>: <i>trnR</i> </td> </tr> <tr> <td>33(85)</td> <td>2</td> <td>2</td> <td>1</td> <td>1</td> <td>0</td> <td>1</td> <td>1</td> <td>2</td> <td>intron <i>clpP</i>, <i>rpoC2</i> </td> </tr> <tr> <td>33(47)</td> <td>4</td> <td>4</td> <td>4</td> <td>2</td> <td>4</td> <td>4</td> <td>4</td> <td>4</td> <td> <i>trnI</i>, <i>trnN</i> </td> </tr> <tr> <td>32(79)</td> <td>2</td> <td>0</td> <td>1</td> <td>1</td> <td>0</td> <td>0</td> <td>0</td> <td>0</td> <td>IGS <i>psbA</i>: <i>matK</i> </td> </tr> <tr> <td>32(94)</td> <td>2</td> <td>0</td> <td>1</td> <td>0</td> <td>0</td> <td>0</td> <td>0</td> <td>0</td> <td>IGS <i>trnP</i>: <i>psaJ</i> </td> </tr> <tr> <td>32(58)</td> <td>3</td> <td>3</td> <td>3</td> <td>3</td> <td>4</td> <td>3</td> <td>3</td> <td>3</td> <td> <i>trnT</i>, <i>trnM</i> </td> </tr> <tr> <td>32(64)</td> <td>2</td> <td>2</td> <td>2</td> <td>2</td> <td>1</td> <td>2</td> <td>0</td> <td>2</td> <td> <i>psaA</i>, <i>psaB</i> </td> </tr> <tr> <td>32(97)</td> <td>3</td> <td>1</td> <td>0</td> <td>1</td> <td>1</td> <td>1</td> <td>4</td> <td>1</td> <td>intron <i>trnL</i>, IGS <i>ndhC</i>: <i>trnV</i> </td> </tr> <tr> <td>32(69)</td> <td>2</td> <td>0</td> <td>2</td> <td>1</td> <td>1</td> <td>1</td> <td>1</td> <td>2</td> <td>intron <i>ndhA</i>, <i>rpoA</i> </td> </tr> <tr> <td>31(66)</td> <td>4</td> <td>2</td> <td>4</td> <td>4</td> <td>0</td> <td>6</td> <td>2</td> <td>4</td> <td> <i>ycf2</i> </td> </tr> <tr> <td>31(66)</td> <td>2</td> <td>2</td> <td>2</td> <td>2</td> <td>2</td> <td>2</td> <td>2</td> <td>2</td> <td>IGS <i>trnS</i>: <i>trnR</i>, <i>trnG</i> </td> </tr> <tr> <td>31(56)</td> <td>2</td> <td>1</td> <td>2</td> <td>2</td> <td>1</td> <td>2</td> <td>2</td> <td>2</td> <td>Intron <i>ndhA</i>, IGS <i>rpl16</i>: <i>rps3</i> </td> </tr> <tr> <td>31(59)</td> <td>2</td> <td>2</td> <td>2</td> <td>2</td> <td>2</td> <td>2</td> <td>2</td> <td>2</td> <td> <i>trnT</i> </td> </tr> <tr> <td>31(53)</td> <td>2</td> <td>1</td> <td>2</td> <td>1</td> <td>3</td> <td>1</td> <td>1</td> <td>3</td> <td>intron <i>ycf3</i>, intron <i>ndhA</i> </td> </tr> <tr> <td>31(66)</td> <td>4</td> <td>4</td> <td>4</td> <td>4</td> <td>0</td> <td>2</td> <td>4</td> <td>4</td> <td> <i>ycf2</i> </td> </tr> <tr> <td>31(75)</td> <td>2</td> <td>2</td> <td>2</td> <td>2</td> <td>0</td> <td>2</td> <td>1</td> <td>2</td> <td>intron <i>ndhB</i> </td> </tr> <tr> <td>31(88)</td> <td>2</td> <td>0</td> <td>1</td> <td>0</td> <td>0</td> <td>0</td> <td>0</td> <td>0</td> <td>IGS <i>rps16</i>: <i>trnQ</i>, <i>cemA</i> </td> </tr> <tr> <td>31(84)</td> <td>2</td> <td>1</td> <td>2</td> <td>1</td> <td>0</td> <td>1</td> <td>0</td> <td>0</td> <td>intron <i>ndhB</i>, IGS <i>trnT</i>: <i>psbD</i> </td> </tr> <tr> <td>31(34)</td> <td>2</td> <td>2</td> <td>2</td> <td>1</td> <td>0</td> <td>2</td> <td>1</td> <td>1</td> <td> <i>psaB</i>, <i>psaA</i> </td> </tr> <tr> <td>31(56)</td> <td>2</td> <td>2</td> <td>2</td> <td>1</td> <td>1</td> <td>1</td> <td>1</td> <td>1</td> <td> <i>psaB</i>, <i>psaA</i> </td> </tr> <tr> <td>31(59)</td> <td>2</td> <td>0</td> <td>2</td> <td>0</td> <td>0</td> <td>0</td> <td>0</td> <td>0</td> <td>IGS <i>rps12 </i>5': <i>clpP</i> </td> </tr> <tr> <td>31(75)</td> <td>2</td> <td>0</td> <td>2</td> <td>0</td> <td>0</td> <td>0</td> <td>0</td> <td>0</td> <td>intron <i>clpP</i>, intron <i>rps16</i> </td> </tr> <tr> <td>31(84)</td> <td>2</td> <td>0</td> <td>1</td> <td>0</td> <td>0</td> <td>0</td> <td>0</td> <td>0</td> <td>IGS <i>rps15</i>: <i>trnN</i> </td> </tr> <tr> <td>30(77)</td> <td>2</td> <td>1</td> <td>0</td> <td>0</td> <td>0</td> <td>0</td> <td>0</td> <td>0</td> <td>intron <i>ycf3</i> </td> </tr> <tr> <td>30(52)</td> <td>2</td> <td>2</td> <td>2</td> <td>2</td> <td>2</td> <td>2</td> <td>2</td> <td>2</td> <td> <i>trnS</i> </td> </tr> <tr> <td>30(71)</td> <td>3</td> <td>3</td> <td>3</td> <td>3</td> <td>4</td> <td>4</td> <td>3</td> <td>3</td> <td>intron <i>ndhB</i>, IGS <i>trnS</i>: <i>trnR</i> </td> </tr> <tr> <td>30(52)</td> <td>3</td> <td>3</td> <td>3</td> <td>3</td> <td>3</td> <td>3</td> <td>3</td> <td>4</td> <td>intron <i>ycf3</i>, IGS <i>trnV</i>: <i>rps12 </i>3' </td> </tr> <tr> <td>30(55)</td> <td>2</td> <td>1</td> <td>2</td> <td>0</td> <td>0</td> <td>0</td> <td>0</td> <td>0</td> <td>IGS <i>petA</i>: <i>psbJ</i> </td> </tr> <tr> <td>30(42)</td> <td>4</td> <td>2</td> <td>4</td> <td>2</td> <td>2</td> <td>2</td> <td>2</td> <td>2</td> <td>IGS <i>trnV</i>: <i>rps12 </i>3', <i>rrn23</i> </td> </tr> <tr> <td>30(39)</td> <td>4</td> <td>2</td> <td>4</td> <td>2</td> <td>2</td> <td>2</td> <td>2</td> <td>2</td> <td>IGS <i>trnV</i>: <i>rps12 </i>3', <i>rrn23</i> </td> </tr> <tr> <td>30(68)</td> <td>2</td> <td>2</td> <td>2</td> <td>1</td> <td>2</td> <td>2</td> <td>2</td> <td>1</td> <td>intron <i>ndhA</i>, IGS <i>trnS</i>: <i>trnR</i> </td> </tr> <tr> <td>30(84)</td> <td>3</td> <td>0</td> <td>2</td> <td>0</td> <td>0</td> <td>0</td> <td>0</td> <td>0</td> <td> <i>rpl23</i> </td> </tr> <tr> <td>30(68)</td> <td>4</td> <td>4</td> <td>4</td> <td>4</td> <td>0</td> <td>4</td> <td>6</td> <td>6</td> <td> <i>ycf2</i> </td> </tr> <tr> <td>30(74)</td> <td>4</td> <td>4</td> <td>4</td> <td>6</td> <td>0</td> <td>2</td> <td>8</td> <td>4</td> <td> <i>ycf2</i> </td> </tr> <tr> <td>30(55)</td> <td>2</td> <td>1</td> <td>2</td> <td>1</td> <td>0</td> <td>0</td> <td>0</td> <td>0</td> <td>IGS <i>trnH</i>: <i>psbA</i>, <i>rpoB</i> </td> </tr> <tr> <td>30(74)</td> <td>2</td> <td>0</td> <td>2</td> <td>0</td> <td>0</td> <td>0</td> <td>0</td> <td>0</td> <td>IGS <i>psbK</i>: <i>psbI</i>, IGS <i>psbF</i>: <i>psbE</i> </td> </tr> <tr> <td>30(97)</td> <td>2</td> <td>0</td> <td>1</td> <td>0</td> <td>0</td> <td>0</td> <td>0</td> <td>0</td> <td>IGS <i>atpH</i>: <i>atpI</i> </td> </tr> <tr> <td>30(90)</td> <td>2</td> <td>1</td> <td>1</td> <td>0</td> <td>0</td> <td>0</td> <td>1</td> <td>0</td> <td>IGS <i>atpH</i>: <i>atpI</i>, IGS <i>psaI</i>: <i>ycf4</i> </td> </tr> <tr> <td>30(68)</td> <td>2</td> <td>1</td> <td>2</td> <td>1</td> <td>0</td> <td>1</td> <td>1</td> <td>0</td> <td>IGS <i>accD</i>: <i>psaI</i>, <i>rpoB</i> </td> </tr> <tr> <td>30(74)</td> <td>2</td> <td>0</td> <td>2</td> <td>0</td> <td>0</td> <td>0</td> <td>0</td> <td>0</td> <td>IGS <i>ycf6</i>: <i>psbM</i>, IGS <i>psaI</i>: <i>ycf4</i> </td> </tr> <tr> <td>30(68)</td> <td>3</td> <td>3</td> <td>3</td> <td>3</td> <td>0</td> <td>3</td> <td>3</td> <td>3</td> <td>IGS <i>psbD</i>: <i>psbC</i>, IGS <i>rpl14</i>: <i>rpl16</i>, <i>ycf2</i> </td> </tr> <tr> <td>30(71)</td> <td>2</td> <td>2</td> <td>2</td> <td>2</td> <td>2</td> <td>2</td> <td>2</td> <td>1</td> <td> <i>psaB</i>, <i>psaA</i> </td> </tr> <tr> <td>30(55)</td> <td>3</td> <td>1</td> <td>3</td> <td>0</td> <td>2</td> <td>3</td> <td>1</td> <td>1</td> <td>intron <i>ycf3</i>, IGS <i>trnV</i>: <i>rps12 </i>3' </td> </tr> <tr> <td>30(65)</td> <td>2</td> <td>0</td> <td>1</td> <td>0</td> <td>0</td> <td>0</td> <td>0</td> <td>0</td> <td>IGS <i>trnS</i>: <i>rps4</i>, IGS <i>rps15</i>: <i>trnN</i> </td> </tr> <tr> <td>30(74)</td> <td>2</td> <td>2</td> <td>2</td> <td>2</td> <td>1</td> <td>1</td> <td>2</td> <td>1</td> <td> <i>rps4</i>, <i>ndhC</i> </td> </tr> <tr> <td>30(65)</td> <td>2</td> <td>1</td> <td>2</td> <td>2</td> <td>0</td> <td>1</td> <td>0</td> <td>2</td> <td> <i>ndhK</i>, <i>ndhD</i> </td> </tr> <tr> <td>30(61)</td> <td>3</td> <td>1</td> <td>3</td> <td>1</td> <td>1</td> <td>1</td> <td>1</td> <td>1</td> <td>intron <i>trnV</i>, IGS <i>trnV</i>: <i>rps12 </i>3' </td> </tr> <tr> <td>30(58)</td> <td>2</td> <td>2</td> <td>1</td> <td>0</td> <td>0</td> <td>1</td> <td>1</td> <td>1</td> <td> <i>atpE</i>, <i>ndhD</i> </td> </tr> <tr> <td>30(74)</td> <td>2</td> <td>1</td> <td>1</td> <td>1</td> <td>1</td> <td>1</td> <td>1</td> <td>0</td> <td>IGS <i>petB</i>: <i>petD</i>, IGS <i>rps15</i>: <i>trnN</i> </td> </tr> <tr> <td>30(71)</td> <td>4</td> <td>2</td> <td>4</td> <td>1</td> <td>2</td> <td>2</td> <td>4</td> <td>4</td> <td> <i>rpl2</i>, <i>ycf2</i> </td> </tr> <tr> <td>30(52)</td> <td>4</td> <td>2</td> <td>4</td> <td>1</td> <td>2</td> <td>2</td> <td>2</td> <td>2</td> <td> <i>rpl2</i>, <i>ycf2</i> </td> </tr> <tr> <td>30(29)</td> <td>2</td> <td>2</td> <td>2</td> <td>0</td> <td>0</td> <td>0</td> <td>0</td> <td>0</td> <td>IGS <i>trnI</i>: <i>rrn16</i> </td> </tr> </tbody> </table> <div class="table-wrap-foot_UNKNOWN"> <p>Repeats size is for <i>Nymphaea</i>, which was used the reference genome in Blast comparisons. Numbers in parentheses are percentage AT in each repeat. Numbers listed under each taxon indicate the number of copies of repeat shared with <i>Nymphaea</i>. The location indicates position of shared repeats. IGS = intergenic spacer. </p> </div> </div> <p>Repeated sequences are generally considered to be uncommon in chloroplast genomes with the notable exception of the large IR present in most land plants [ <a href="#B7">7</a>]. Dispersed repeats are found more commonly in genomes that have experienced changes in genome organization [ <a href="#B16">16</a>, <a href="#B61">61</a>, <a href="#B62">62</a>], especially in highly rearranged algal genomes [ <a href="#B63">63</a>, <a href="#B64">64</a>]. A correlation between the number of repeats and the extent of gene order change has been suggested for algal genomes [ <a href="#B64">64</a>]. However, comparisons of completely sequenced chloroplast genomes of the angiosperm families Fabaceae [ <a href="#B65">65</a>], Solanaceae [ <a href="#B47">47</a>], and Asteraceae [ <a href="#B46">46</a>] have revealed the presence of numerous small repeats scattered throughout these genomes even though these genomes have few if any rearrangements. In each of these families assessed by others, most of the repeats are 20 – 40 bp in length and they are located mostly in intergenic spacer regions and introns, although several are located in the protein-coding genes <i>psaA</i>, <i>psaB</i>, and <i>ycf2</i>. Our examination of repeats here in eight angiosperm chloroplast genomes, representing a wider phylogenetic diversity than that of earlier studies, identified numerous repeats in each genome of a nature and pattern similar to those reported by others based on narrower comparisons (Figure <a href="#F9">9</a>). Again here, some of the shared SDR repeats, such as those located in tRNA genes and those shared between <i>psaA </i>and <i>psaB</i>, result from conserved sequence similarity of related genes. The vast majority of the plastome SDRs are restricted to intergenic spacer regions and introns and are small in size. Earlier work suggesting that larger repeats (of a size detectable via Southern Hybridization) are rare in unrearranged plastid genomes is supported. However, the pattern that seems to be emerging from all of these analyses is that small SDRs in angiosperm genomes are quite common and they tend to be located in the same regions. The role of these conserved repeats is not known but given that many of them are shared broadly and are located in the same regions suggests that some may be functional. </p> </div> </div> <div class="conclusion" title="sec"> <div class="title" tagx="title" title="title">
Conclusion</div> <p>As additional chloroplast genomes from less-derived angiosperm taxa are characterized we obtain a clearer picture of the ancestral plastid genome organization for angiosperms. In large part these additional genomes (reported here and by others) confirm that the <i>Nicotiana </i>plastome is reasonably inferred to represent the ancestral angiosperm for gene content and organization, although perhaps not for exact IR boundaries. With notable exceptions, these features, seen in <i>Nicotiana</i>, are remarkably conserved in most angiosperm lineages. The <i>Nuphar advena </i>and <i>Ranuculus macranthus </i>chloroplast genome sequences add to this growing body of data for reconstructing the evolution of plastid genomes. The <i>Nuphar </i>genome, in addition, provided the opportunity for comparison with the relatively closely related <i>Nymphaea alba </i>plastome sequence. This comparison confirmed views on the conservative nature of the genome, with even some IGS regions showing very high levels of nucleotide similarity. The <i>Nuphar-Nymphaea </i>comparison also supported the view that SSR frequencies represent a balance between two mutation types: SSM and substitutions. Detailed comparisons among these and other genomes reveal many differences and unexplained conservation of features that both remain to be understood. However, we are able to suggest that the widely conserved sequences designated <i>ycf15 </i>and <i>ycf68 </i>are not protein-coding genes. </p> <p>As has been noted many times elsewhere, chloroplast genomes are biased towards A and T nucleotides, i.e., are "A+T-rich", except for the RNA genes. Howe <i>et al</i>. [ <a href="#B66">66</a>] suggest that the A+T-richness of plastid genomes is the result of endosymbiosis (or at least enhanced due to endosymbiosis). They argue that there might be a selective advantage for a particular protein-coding gene to be either A+T-rich or G+C-rich and that each class of genes can be maintained through compartmentalization in the different genomes [ <a href="#B66">66</a>]. However, many other genomes, prokaryotic or eukaryotic, are as A+T-rich as (or even more biased than) plastid genomes [ <a href="#B31">31</a>, <a href="#B53">53</a>]. We suggest that the plastome A+T bias is relatively modest and results from a slight mutation and/or error checking bias of the plastid DNA polymerases or perhaps some selection for A and T in otherwise neutral positions to increase ease of denaturation during replication or transcription. In any event, a bias can be seen in overall composition, in the composition of the 3 <sup>rd </sup>position of codons, and in which SSR motifs are most abundant, among other aspects of the genomic sequence. However on closer inspection A+T richness does not correlate with variation in codon usage or details of SSR abundance. We speculate that the interesting patterns are those that fail to track patterns of A+T-richness; for example, repeat sequences in non-coding regions enriched for G and C are the ones more likely to be functional and understanding the components of codon bias that is <div class="u_UNKNOWN">not</div> related to A+T-richness is more likely to be significant. </p> </div> <div class="methods" title="methods"> <div class="title" tagx="title" title="title">
Methods</div> <div class="dnasources" title="sec"> <div class="title" tagx="title" title="title">
DNA Sources</div> <p>Leaf material of <i>Nuphar advena </i>(Aiton) W. T. Aiton was obtained from a cultivated plant (vegetatively propagated from wild material from Lake Moshanon, Center County, PA) in the Biology Greenhouse at Pennsylvania State University. A voucher made from this same plant (collection Claude W. dePamphilis 2001.301) was deposited at PAC. </p> <p> <i>Ranunculus macranthus </i>Scheele leaf material was collected from a wild population in Austin, TX. Leaves from multiple individual plants were combined to provide enough material for the cpDNA isolations. A voucher was made from a plant from this population and deposited in TEX. </p> </div> <div class="isolationmethods" title="sec"> <div class="title" tagx="title" title="title">
Isolation Methods</div> <p> <i>Nuphar advena </i>chloroplast DNA was prepared by the sucrose gradient method [ <a href="#B67">67</a>]. However, the sucrose-gradient method did not yield pure enough cpDNA from <i>Ranunculus macranthus</i>. Therefore, for <i>Ranunculus macranthus </i>the NaCl method [ <a href="#B68">68</a>] was used and yielded concentrated, sufficiently pure cpDNA and was used as the sequencing template. Our isolation methods are described in detail elsewhere [ <a href="#B69">69</a>]. </p> </div> <div class="shotgunsequencingandfinishing" title="sec"> <div class="title" tagx="title" title="title">
Shotgun Sequencing and Finishing</div> <p>Details of our methodology for producing finished genomic sequences from the sequencing template (cpDNA in this case) are provided in Jansen et al [ <a href="#B69">69</a>]. We will provide an overview of our approach here. </p> <p>Draft genomic sequence was prepared from the cpDNA preparations at the DOE Joint Genome Institute. To do so, the DNA was sheared by passage through a narrow aperture and then fragments averaging 3 kb were selected from an agarose gel and cloned into plasmids, which were then used to transform <i>E. coli</i>. Clones were randomly selected from these libraries and placed into 384-well plates and templates amplified for sequencing using rolling circle amplification. When sequenced, forward and reverse plasmid sequencing primers produced 500–750 bp of sequence data (reads) from each end of the inserts. The individual reads were then processed and assembled into contigs using Phred and Phrap [ <a href="#B70">70</a>, <a href="#B71">71</a>]. Four or five plates (generating 768 reads each) provide 8–10× coverage if the library is 60–80% cpDNA. </p> <p>In the case of <i>Nuphar</i>, one library was constructed. The sequencing reads from five plates assembled into a draft genome composed of a single contig. In the case of <i>Ranunculus</i>, three libraries were constructed and fourteen plates of sequence data were generated. These data assembled into a draft genome composed of a single contig. The draft sequences were then assessed visually using Consed [ <a href="#B72">72</a>] to determine the level of quality of each nucleotide. Each nucleotide in each read was assigned a quality score using Phred [ <a href="#B70">70</a>]. Our minimum criterion was two reads with a quality score of ≥ 20 for each position. At the vast majority of positions this criterion was greatly exceeded, but occasionally areas of low coverage occurred and there were some instances where the minimum criterion was not met. In those cases, we designed primers to flank the regions of these "quality gaps", PCR amplified a product that contained the questionable nucleotide or nucleotides, and sequenced the PCR product until the criterion was met. Five regions of low quality were confirmed in this manner in <i>Nuphar </i>and two in <i>Ranunculus</i>. </p> <p>We also confirmed the extent of the IR with a PCR and sequencing strategy. In the shotgun sequencing approach, the two copies of the IR are not sequenced separately. Like the remainder of the genome, the sequence for the IR region is built up via the overlap and assembly of the 500–700 bp sequencing reads. Sequences derived from templates representing both copies of the IR assemble together. The IR can be recognized generally in Consed as the depth of sequencing reads doubles in that region and the boundaries can be inferred where the two reads from a single clone assemble far from each other. However to precisely define the boundaries we designed primers to amplify across each of the four IR-single copy junctions, sequenced the PCR products and compared those sequences to one another and to the draft genome. In this manner we were able to confirm the precise location of the IR-LSC and IR-SSC boundaries. </p> </div> <div class="annotationandrelatedstudies" title="sec"> <div class="title" tagx="title" title="title">
Annotation and related studies</div> <p>We used DOGMA [ <a href="#B73">73</a>] as our primary tool for annotating these two genomes. DOGMA uses BLASTX [ <a href="#B74">74</a>] to compare the genomic sequence against a custom database of genes constructed using corrected annotations of 17 completely sequenced chloroplast genomes. This produces a draft annotation that is then inspected using DOGMA's tools for accurate assessment of the start and codon of each gene and any contained exon-intron boundaries. Because of the limitations of BLAST searches, small exons (6–9 nucleotides) that occur in three chloroplast genes cannot be found by DOGMA or by using other versions of Blast searches, so these were located manually. Putative gene and exon boundaries are determined by detailed comparison with other annotated genomes and individual gene sequences; no expression or protein studies were conducted to confirm the assignments. </p> </div> <div class="investigationsofycf15andycf68" title="sec"> <div class="title" tagx="title" title="title">
Investigations of ycf15 and ycf68</div> <p>To investigate the distribution and nature of <i>ycf15</i>, we extracted the <i>ycf15 </i>sequence from the <i>Nicotiana tabaccum </i>genome ( <a href="NC_001879">NC_001879</a>) and conducted pairwise BLAST searches between this and each of the 63 then-available complete chloroplast genome sequences. If any portion of the <i>ycf15 </i>sequence was not detected in comparisons against whole genome sequences, the <i>ycf2:trnL </i>spacer region was extracted and just that portion of the genome was compared to the <i>Nicotiana ycf15 </i>sequence. ClustalW [ <a href="#B75">75</a>] alignments were also conducted to assess levels of conservation in a subset of genomes. </p> <p>Pairwise BLAST (bl2seq [ <a href="#B76">76</a>]) was also used to assess <i>ycf68 </i>distributional patterns, here using the <i>ycf68 </i>sequence from <i>Zea </i>and <i>Nymphaea </i>against all 63 then-published plastid genomes. Also, since <i>ycf68 </i>is found in the <i>trnI</i>-GAU intron, it seemed possible that the conserved sequence could be related to folding during excision. The <i>trnI</i>-GAU intron was extracted using NCBI or DOGMA from all the species listed in Table <a href="#T3">3</a> that contain an intron in <i>trnI</i>-GAU and folded using the web-based programs mfold [ <a href="#B77">77</a>] and DINAmelt [ <a href="#B78">78</a>] using default settings. We then examined the folding patterns to look for regions with conserved folding domains. We saw no obvious correlation between the position of the region of sequence similarity and the folding structure hypothesized by either program. </p> <p>Another method for investigating the functionality of an ORF was designed by Echols <i>et al</i>. [ <a href="#B31">31</a>]. They looked at codon usage in pseudogenes, genes, and intergenic regions to determine how pseudogenes were evolving and if amino acid frequency could be an indicator of the functionality of a conserved region of DNA [ <a href="#B31">31</a>]. They separately calculated the frequency of amino acids in coding, noncoding, and known pseudogene sequences and then graphed the results to display trends of usage in the three sequence types. They found that the amino acid frequency in pseudogenes was an exact intermediate between amino acid frequency in known genes and intergenic DNA, and argued that this method is a valid way to determine whether or not a gene is functional [ <a href="#B31">31</a>]. We used this method to investigate whether or not <i>ycf15 </i>and <i>ycf68 </i>had amino acid frequencies similar to coding, noncoding, or an intermediate to differentiate among the hypotheses that the ycfs are genes, pseudogenes, or simply non-coding DNA. Algorithms from web-based suite of tools BABEL [ <a href="#B79">79</a>] were used to calculate the frequency of amino acids. For <i>Ranunculus </i>and <i>Nuphar</i>, we input all coding DNA sequences and recorded the amino acid frequency. We determined codon usage in the noncoding sequence by averaging the codon usage for each reading frame (the difference in frequency among the six reading frames is minimal) to get a frequency estimate. BABEL tools [ <a href="#B79">79</a>] were also used to calculate amino acid frequency in the intron sequence of <i>trnI</i>-GAU and the <i>ycf2:trnL</i>-CAA intergenic region. These results were then graphed. In the Echols <i>et al</i>. [ <a href="#B31">31</a>] analysis the amino acid frequency for each amino acid was graphed in order of decreasing levels of variation. We tried multiple methods for arranging the amino acids including that of Echols <i>et al</i>., alphabetical, high and low frequency, standard deviation of the frequency across all types of DNA, etc. We found that manipulation of the order of amino acids on the X-axis changed the results so much that we could display evidence for any of the hypotheses under consideration. Therefore, we do not believe that this type of analysis is valid, at least not with our data. Perhaps with the larger sample sizes, from nuclear genomes, available to Echols <i>et al</i>. the approach is more consistent. </p> <p>We further investigated the characteristics of <i>ycf15 </i>and <i>ycf68 </i>by comparing sequence similarity of the <i>trnI</i>-GAU intron and the intergenic spacer between <i>ycf2 </i>and <i>trnL</i>-CAA to other introns and intergenic spacers throughout the chloroplast genome to see if the intergenic/intron regions containing <i>ycf15 </i>and <i>ycf68 </i>are more conserved than that of other noncoding DNA in the chloroplast genome. We used the web-based program Mulan [ <a href="#B32">32</a>] to compare noncoding DNA in our sequences to a reference sequence – <i>Nicotiana tabacum </i>for <i>ycf15 </i>and <i>Zea mays </i>for <i>ycf68</i>. Mulan performs pairwise sequence comparisons of the input sequence to the chosen reference sequence. We used the default setting of 100 for the ECR (evolutionary conserved region) length, the minimum amount of base pairs that have to align for similarity to register on the histogram, and 50 percent for the ECR similarity. The algorithm then returns graphical information about the likeness of the input sequences to the reference sequence; output is in the form of a histogram showing the similarity of the sequences from 50 to 100 percent. To see if introns were commonly conserved at a level equivalent to that represented by the <i>ycf68 </i>motif, we compared the intron in <i>trnI</i>-GAU with a sample of other introns within the inverted repeat ( <i>trnA</i>-UGC and <i>ndhB</i>), and in the large single copy region ( <i>trnG</i>-UCC, <i>trnL</i>-UAA, and <i>trnV-</i>UAC). To investigate conservation of the nucleotide sequence in the <i>ycf2</i>: <i>trnL </i>intergenic spacer, relative to the <i>ycf15 </i>question, we compared it to intergenic DNA that is outside of known operons in the inverted repeat (3' <i>rps12</i>: <i>trnV</i>-GAC and <i>trnN-</i>GUU: <i>ycf1</i>) and the large single copy region ( <i>rpoB</i>: <i>trnC</i>-GCA, <i>trnT</i>-GGU: <i>psbD</i>, and <i>psaA</i>: <i>ycf3</i>). </p> </div> <div class="codonusage" title="sec"> <div class="title" tagx="title" title="title">
Codon Usage</div> <p>Alternative start codon usage was determined based on published annotations with additional comparisons. All genes annotated with an alternative start were aligned to determine whether or not the start was homologous with non-alternative start codons in other taxa. If the annotated alternative start was the result of a longer or shorter annotation, the region homologous to the start codon of other taxa was examined to determine whether or not the normal start codon was present. </p> <p>Codon usage patterns for each genome were determined using FREQSQ from BABEL [ <a href="#B79">79</a>] and CodonW [ <a href="#B41">41</a>]. The predicted relationship of ENc and GC3 was determined using Wright's equation as given by Novembre [ <a href="#B43">43</a>]. The equation as printed in the Wright 1990 paper [ <a href="#B42">42</a>] contains a typographical error. </p> </div> <div class="nuphar-nymphaeacomparison" title="sec"> <div class="title" tagx="title" title="title">
Nuphar-Nymphaea comparison</div> <p>The complete chloroplast genome of <i>Nuphar </i>was sub-divided into individual coding and non-coding regions, and each region was compared against the entire <i>Nymphaea </i>genome using the BLAST2 algorithm [ <a href="#B76">76</a>] available on the National Center for Biotechnology Information website. The output files from BLAST2 were used to determine the percent similarity of each region. BLAST2 presents errors when one sequence under comparison contains mononucleotide repeats with runs greater than six nucleotides; n is indicated in place of each repeating nucleotide at those positions and considered "non-matching" in similarity calculations. These errors were checked against the genome sequences and corrected before calculations were made. Nucleotide sequence fragments that BLAST2 did not align (and did not show) were scored as zeros and negatively affected the % identity value. For example if only 425 of the known 450 nucleotides in a gene were aligned by BLAST2, and it was otherwise a perfect match, then the % identity would be 425/450 or 94.4%. BLAST2 does not display insertions or deletion events (indels) of greater than 10 nucleotides; such a disparity between the sequences would lead to the two adjoining regions of similarity being displayed as separate regions of identity. </p> <p>We looked at indels as well as nucleotide similarity. We characterized indels as insertions or deletions relative to <i>Nuphar</i>. If <i>Nuphar </i>contained sequence for which there was no equivalent in <i>Nymphaea </i>is was considered an insertion, whereas if nucleotides in the <i>Nymphaea </i>sequence lacked a <i>Nuphar </i>equivalent it was a deletion. We used BLAST2 [ <a href="#B76">76</a>] to quantify indels in coding regions only. MULAN [ <a href="#B32">32</a>] was used to produce a total genome alignment for <i>Nuphar </i>and <i>Nymphaea</i>. Indels in the entire genome were quantified using this output. Simple subtraction of the coding region indels from the total number of indels provided the number found in intergenic regions. We quantified indels of 5–9 bp, 10–19 bp and 20 or greater bp. </p> </div> <div class="ssranalysis" title="sec"> <div class="title" tagx="title" title="title">
SSR Analysis</div> <p>To locate and characterize SSRs in these genomes, we used the Simple Sequence Repeat (SSR) Extractor Utility [ <a href="#B80">80</a>]. We screened for all perfect mono-, di- and tri-nucleotide repeats of length at least 8,8, or 9 nucleotides, respectively. We also determined the number and nature of mono-, di- and tri- nucleotide repeats of length at least 10, 10 or 12 nucleotides. In addition to analyzing SSR content of <i>Nuphar </i>and <i>Ranunculus</i>, we examined the chloroplast genomes of the other completely sequenced land plant genomes available at the time of the analysis. These chloroplast genome sequences were obtained from GenBank (accession numbers given in Table <a href="#T7">7</a>). To assess possible relationships between the number of longer SSRs and the number of shorter SSRs as well as the number of SSRs and genome size and A+T-richness and frequencies of particular SSR motifs, we conducted Spearman rank correlation tests using an online calculator [ <a href="#B55">55</a>]. Predicted numbers of SSRs of particular composition were calculated based on the underlying compositional nucleotide frequencies. For example, if A+T frequency in the genome overall is 0.638 then the frequency of a mononucleotide repeat of length 8 nt would be (0.638) <sup>8 </sup>and the number of length 8nt A+T SSRs expected would be the frequency* (genome length/SSR length). To determine whether the composition of SSRs deviated from expectation based on overall nucleotide, dinucleotide and trinucleotide frequencies of the genomes (determined using FREQSQ [ <a href="#B79">79</a>]), we used a Chi-square test. Predicted number of each class of repeat was calculated as the ((expected frequency of a particular type of repeat, calculated as just stated)/(total of expected frequencies of all categories)) * observed total number of repeats. For example, in <i>Huperzia</i>, which has an A+T frequency of 0.638 and a G+C frequency of 0.362, the expected frequency of A+T mononucleotide repeats of length 8 is (0.638) <sup>8 </sup>and of G+C 8 nt mononucleotide SSRs is (0.362) <sup>8</sup>. The expected number of A+T mononucleotide runs, given that 147 mononucleotide repeats of 8nt and longer are observed in that genome, is (0.638) <sup>8</sup>/((0.638) <sup>8</sup>*(0.362) <sup>8</sup>)*147 = 145.44. We did simplify the calculations by basing our predictions on repeats of minimum length, which was also the mean and median length (i.e., we considered all the repeats that we observed to be of minimum length for these calculations). </p> </div> <div class="sdranalysis" title="sec"> <div class="title" tagx="title" title="title">
SDR analysis</div> <p>Shared and unique direct and inverted repeats were identified for eight angiosperm chloroplast genomes using the Comparative Repeat Analysis program [ <a href="#B81">81</a>]. This program uses REPuter [ <a href="#B82">82</a>] but it has two additional features: it filters out repeats that are contained entirely within other repeats, and it identifies shared repeats among the input genomes by Blasting the repeats from each genome against all other genomes. For repeat identification, the following settings were used: (i) minimum repeat size of 30 bp; (ii) 90% or greater sequence identity, based on Hamming distance equal to 3; and (iii) an e-value of 2 for Blast comparisons against the other genomes. </p> </div> </div> <div class="listofabbreviations" title="sec"> <div class="title" tagx="title" title="title">
List of abbreviations</div> <p>bp – base pair</p> <p>COA – Correspondence Analysis of Codon Usage [ <a href="#B41">41</a>] </p> <p>CpG (or GpC or ApT, etc) – to nucleotides adjacent on one DNA strand and thus linked by a phosphodiester bond (symbolized by the p) </p> <p>DOGMA – Dual Organellar Genome Annotator [ <a href="#B73">73</a>] </p> <p>ENc – Effective number of codons [ <a href="#B42">42</a>] </p> <p>GC3 – percent G+C at the 3 <sup>rd </sup>position of codons </p> <p>IGS – intergenic spacer</p> <p>IR – inverted repeat</p> <p>LSC – large single copy region</p> <p>nt – nucleotide</p> <p>RSCU – relative synonymous codon usage</p> <p>S – "strong" nucleotide (G or C)</p> <p>SDR – small dispersed repeat</p> <p>SSC – small single copy region</p> <p>SSM – slip strand mispairing</p> <p>SSR – simple sequence repeat</p> <p>W – "weak" nucleotide (A or T)</p> </div> <div class="authors'contributions" title="sec"> <div class="title" tagx="title" title="title">
Authors' contributions</div> <p>LAR conceived of the project, drafted the manuscript, conducted the SSR, nucleotide composition and codon usage analyses and helped with and coordinated other aspects of the project; RP finished the <i>Ranunculus </i>sequence, completed the annotations of the two genomes, prepared the GenBank submissions, made the comparisons related to <i>ycf15 </i>and <i>ycf68</i>, compared SSRs of <i>Nuphar </i>and <i>Nymphaea</i>, compared <i>Nuphar </i>and <i>Ranunuclus </i>with other genomes for gene content, gene order and start codon usage, and drafted manuscript sections related to her work; TWC performed the finishing and initial annotation work on <i>Nuphar</i>, conducted experiments to confirm IR boundaries, and helped generate genome map figures; CD performed comparisons between <i>Nuphar </i>and <i>Nymphaea</i>, the initial COA analyses of codon usage in <i>Nuphar </i>and <i>Ranunculus</i>, and drafted the manuscript section related to the <i>Nuphar-Nymphaea </i>comparisons; HMF and JLB generated the draft sequence for the two genomes; RKJ assisted in the preparation of the sequencing templates, helped with the annotation of <i>Nuphar</i>, performed the SDR analyses, drafted the SDR manuscript section, and contributed to the design of the project. All authors assisted with manuscript preparation and read and approved the final draft. </p> </div> <div class="supplementary-material" title="supplementary-material"> <div class="title" tagx="title" title="title">
Supplementary Material</div> <div class="local-data" title="local-data">  <div class="title" tagx="title" title="title">
Additional file 1</div> <p>Detailed comparison of SSRs. This table (Table S1) provides a more detailed comparison of SSRs among the plastid genomes of ten vascular plants (eight angiosperms, one gymnosperm and one pteridophyte). SSRs are enumerated by composition as well as length. </p>  <div class="media" title="media"> <a href="1471-2164-8-174-S1.doc">LINK</a>  <p>Click here for file</p>  </div> </div> <div class="local-data" title="local-data">  <div class="title" tagx="title" title="title">
Additional file 2</div> <p>Comparison of SSRs between the plastomes of <i>Nuphar </i>and <i>Nymphaea</i>. This table (Table S2) shows the results of a comparison of each 10,10,12 SSR in the plastid genomes of <i>Nuphar </i>and <i>Nymphaea</i>. The nature of any difference in the SSR between the two genomes is given. </p>  <div class="media" title="media"> <a href="1471-2164-8-174-S2.doc">LINK</a>  <p>Click here for file</p>  </div> </div> </div> </div> <div class="back" title="back"> <div class="ack" title="ack"> <div class="acknowledgements" title="sec"> <div class="title" tagx="title" title="title">
Acknowledgements</div> <p>We thank Gwen Gage for help with Figure <a href="#F3">3</a>, Claude dePamphilis for <i>Nuphar </i>tissue, Romey Haberle for collection of <i>Ranunculus </i>tissue and help with DNA isolations, and Tom Cottrell and Kris Ernest for consultations regarding statistics. The comments of two anonymous reviewers helped improve the manuscript. The National Science Foundation (DEB0075700 to LAR, and DEB0120709 to LAR, JLB and RKJ) funded this work. Part of this work was performed under the auspices of the U. S. Department of Energy's Office of Science, Biological and Environmental Research Program, and by the University of California, Lawrence Berkeley National Laboratory under Contract No. DE-AC02-05CH11231. </p> </div> </div> <div class="references">
References</div> <div tag="ref-list"> <ul> <li tag="ref"> <a name="B1" /> <div class="citation" tagx="citation" title="citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Zanis</span> <span class="given-names" tagx="given-names" title="given-names">MJ</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Soltis</span> <span class="given-names" tagx="given-names" title="given-names">DE</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Soltis</span> <span class="given-names" tagx="given-names" title="given-names">PE</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Mathews</span> <span class="given-names" tagx="given-names" title="given-names">S</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Donoghue</span> <span class="given-names" tagx="given-names" title="given-names">MJ</span> </span> </span> <div class="article-title" title="article-title">
The root of the angiosperms revisited</div> <span class="source" tagx="source" title="source">Proc Nat Acad Sci USA</span> <span class="year" tagx="year" title="year">2002</span> <span class="volume" tagx="volume" title="volume">99</span> <span class="fpage" tagx="fpage" title="fpage">6848</span> <span class="lpage" tagx="lpage" title="lpage">6853</span> <span class="comment">[RMP15]</span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/12011443">12011443</a> </span> <span class="pub-id"> <a href="https://doi.org/10.1073/pnas.092136399">10.1073/pnas.092136399</a> </span> </div> </li> <li tag="ref"> <a name="B2" /> <div class="citation" tagx="citation" title="citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Stefanovic</span> <span class="given-names" tagx="given-names" title="given-names">S</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Rice</span> <span class="given-names" tagx="given-names" title="given-names">DW</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Palmer</span> <span class="given-names" tagx="given-names" title="given-names">JD</span> </span> </span> <div class="article-title" title="article-title">
Long branch attraction, taxon sampling, and the earliest angiosperm: <i>Amborella </i>or monocots? </div> <span class="source" tagx="source" title="source">BMC Evol Biol</span> <span class="year" tagx="year" title="year">2004</span> <span class="volume" tagx="volume" title="volume">4</span> <span class="fpage" tagx="fpage" title="fpage">35</span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/15453916">15453916</a> </span> <span class="pub-id"> <a href="https://doi.org/10.1186/1471-2148-4-35">10.1186/1471-2148-4-35</a> </span> </div> </li> <li tag="ref"> <a name="B3" /> <div class="citation" tagx="citation" title="citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Goremykin</span> <span class="given-names" tagx="given-names" title="given-names">VV</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Holland</span> <span class="given-names" tagx="given-names" title="given-names">B</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Hirsch-Ernst</span> <span class="given-names" tagx="given-names" title="given-names">KI</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Hellwig</span> <span class="given-names" tagx="given-names" title="given-names">FH</span> </span> </span> <div class="article-title" title="article-title">
Analysis of <i>Acorus calamus </i>chloroplast genome and its phylogenetic implications </div> <span class="source" tagx="source" title="source">Mol Biol Evol</span> <span class="year" tagx="year" title="year">2005</span> <span class="volume" tagx="volume" title="volume">22</span> <span class="fpage" tagx="fpage" title="fpage">1813</span> <span class="lpage" tagx="lpage" title="lpage">1822</span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/15930156">15930156</a> </span> <span class="pub-id"> <a href="https://doi.org/10.1093/molbev/msi173">10.1093/molbev/msi173</a> </span> </div> </li> <li tag="ref"> <a name="B4" /> <div class="citation" tagx="citation" title="citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Leebens-Mack</span> <span class="given-names" tagx="given-names" title="given-names">J</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Raubeson</span> <span class="given-names" tagx="given-names" title="given-names">LA</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Cui</span> <span class="given-names" tagx="given-names" title="given-names">L</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Kuehl</span> <span class="given-names" tagx="given-names" title="given-names">J</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Fourcade</span> <span class="given-names" tagx="given-names" title="given-names">M</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Chumley</span> <span class="given-names" tagx="given-names" title="given-names">T</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Boore</span> <span class="given-names" tagx="given-names" title="given-names">JL</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Jansen</span> <span class="given-names" tagx="given-names" title="given-names">RK</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">dePamphilis</span> <span class="given-names" tagx="given-names" title="given-names">CW</span> </span> </span> <div class="article-title" title="article-title">
Identifying the basal angiosperms in chloroplast genome phylogenies: Sampling one's way out of the Felsenstein zone </div> <span class="source" tagx="source" title="source">Mol Biol Evol</span> <span class="year" tagx="year" title="year">2005</span> <span class="volume" tagx="volume" title="volume">22</span> <span class="fpage" tagx="fpage" title="fpage">1948</span> <span class="lpage" tagx="lpage" title="lpage">1963</span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/15944438">15944438</a> </span> <span class="pub-id"> <a href="https://doi.org/10.1093/molbev/msi191">10.1093/molbev/msi191</a> </span> </div> </li> <li tag="ref"> <a name="B5" /> <div class="citation" tagx="citation" title="citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Chang</span> <span class="given-names" tagx="given-names" title="given-names">C-C</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Lin</span> <span class="given-names" tagx="given-names" title="given-names">H-C</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Lin</span> <span class="given-names" tagx="given-names" title="given-names">I-P</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Chow</span> <span class="given-names" tagx="given-names" title="given-names">T-Y</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Chen</span> <span class="given-names" tagx="given-names" title="given-names">H-H</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Chen</span> <span class="given-names" tagx="given-names" title="given-names">W-H</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Cheng</span> <span class="given-names" tagx="given-names" title="given-names">C-H</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Lin</span> <span class="given-names" tagx="given-names" title="given-names">C-Y</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Liu</span> <span class="given-names" tagx="given-names" title="given-names">S-M</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Chang</span> <span class="given-names" tagx="given-names" title="given-names">C-C</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Chaw</span> <span class="given-names" tagx="given-names" title="given-names">S-M</span> </span> </span> <div class="article-title" title="article-title">
The chloroplast genome of <i>Phalaenopsis aphrodite </i>(Orchidaceae): Comparative analysis of evolutionary rate with that of grasses and its phylogenetic implications </div> <span class="source" tagx="source" title="source">Mol Biol Evol</span> <span class="year" tagx="year" title="year">2006</span> <span class="volume" tagx="volume" title="volume">23</span> <span class="fpage" tagx="fpage" title="fpage">279</span> <span class="lpage" tagx="lpage" title="lpage">291</span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/16207935">16207935</a> </span> <span class="pub-id"> <a href="https://doi.org/10.1093/molbev/msj029">10.1093/molbev/msj029</a> </span> </div> </li> <li tag="ref"> <a name="B6" /> <div class="citation" tagx="citation" title="citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Cai</span> <span class="given-names" tagx="given-names" title="given-names">Z</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Penaflor</span> <span class="given-names" tagx="given-names" title="given-names">C</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Kuehl</span> <span class="given-names" tagx="given-names" title="given-names">JV</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Leebens-Mack</span> <span class="given-names" tagx="given-names" title="given-names">J</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Carlson</span> <span class="given-names" tagx="given-names" title="given-names">JE</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">dePamphilis</span> <span class="given-names" tagx="given-names" title="given-names">CW</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Boore</span> <span class="given-names" tagx="given-names" title="given-names">JL</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Jansen</span> <span class="given-names" tagx="given-names" title="given-names">JK</span> </span> </span> <div class="article-title" title="article-title">
Complete chloroplast genome sequences of <i>Drimys, Liriodendron </i>and <i>Piper</i>: Implications for the phylogeny of magnoliids </div> <span class="source" tagx="source" title="source">BMC Evol Biol</span> <span class="year" tagx="year" title="year">2006</span> <span class="volume" tagx="volume" title="volume">6</span> <span class="fpage" tagx="fpage" title="fpage">77</span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/17020608">17020608</a> </span> <span class="pub-id"> <a href="https://doi.org/10.1186/1471-2148-6-77">10.1186/1471-2148-6-77</a> </span> </div> </li> <li tag="ref"> <a name="B7" /> <div class="citation" tagx="citation" title="citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Palmer</span> <span class="given-names" tagx="given-names" title="given-names">JD</span> </span> </span> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Bogorad L, Vasil IK</span> </span> </span> <div class="article-title" title="article-title">
Plastid chromosomes: structure and evolution</div> <span class="source" tagx="source" title="source">The Molecular Biology of Plastids</span> <span class="year" tagx="year" title="year">1991</span> <span class="publisher-name" tagx="publisher-name" title="publisher-name">New York: Academic Press</span> <span class="fpage" tagx="fpage" title="fpage">5</span> <span class="lpage" tagx="lpage" title="lpage">53</span> </div> </li> <li tag="ref"> <a name="B8" /> <div class="citation" tagx="citation" title="citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Downie</span> <span class="given-names" tagx="given-names" title="given-names">SR</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Palmer</span> <span class="given-names" tagx="given-names" title="given-names">JD</span> </span> </span> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Soltis P, Soltis D, Doyle JJ</span> </span> </span> <div class="article-title" title="article-title">
Use of chloroplast DNA rearrangements in reconstructing plant phylogeny</div> <span class="source" tagx="source" title="source">Plant Molecular Systematics</span> <span class="year" tagx="year" title="year">1992</span> <span class="publisher-name" tagx="publisher-name" title="publisher-name">New York: Chapman and Hall</span> <span class="fpage" tagx="fpage" title="fpage">14</span> <span class="lpage" tagx="lpage" title="lpage">35</span> <span class="comment">RMP17</span> </div> </li> <li tag="ref"> <a name="B9" /> <div class="citation" tagx="citation" title="citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Raubeson</span> <span class="given-names" tagx="given-names" title="given-names">LA</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Jansen</span> <span class="given-names" tagx="given-names" title="given-names">RK</span> </span> </span> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Henry R</span> </span> </span> <div class="article-title" title="article-title">
Chloroplast genomes of plants</div> <span class="source" tagx="source" title="source">Diversity and Evolution of Plants; Genotypic and Phenotypic Variation in Higher Plants</span> <span class="year" tagx="year" title="year">2005</span> <span class="publisher-name" tagx="publisher-name" title="publisher-name">London: CABI Publishing</span> <span class="fpage" tagx="fpage" title="fpage">45</span> <span class="lpage" tagx="lpage" title="lpage">68</span> </div> </li> <li tag="ref"> <a name="B10" /> <div class="citation" tagx="citation" title="citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Bendich</span> <span class="given-names" tagx="given-names" title="given-names">AJ</span> </span> </span> <div class="article-title" title="article-title">
Circular chloroplast chromosomes: the grand illusion</div> <span class="source" tagx="source" title="source">Plant Cell</span> <span class="year" tagx="year" title="year">2004</span> <span class="volume" tagx="volume" title="volume">16</span> <span class="fpage" tagx="fpage" title="fpage">1661</span> <span class="lpage" tagx="lpage" title="lpage">1666</span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/15235123">15235123</a> </span> <span class="pub-id"> <a href="https://doi.org/10.1105/tpc.160771">10.1105/tpc.160771</a> </span> </div> </li> <li tag="ref"> <a name="B11" /> <div class="citation" tagx="citation" title="citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Shinozaki</span> <span class="given-names" tagx="given-names" title="given-names">K</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Ohme</span> <span class="given-names" tagx="given-names" title="given-names">M</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Tanaka</span> <span class="given-names" tagx="given-names" title="given-names">M</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Wakasugi</span> <span class="given-names" tagx="given-names" title="given-names">T</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Hayashida</span> <span class="given-names" tagx="given-names" title="given-names">N</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Matsubayashi</span> <span class="given-names" tagx="given-names" title="given-names">T</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Zaita</span> <span class="given-names" tagx="given-names" title="given-names">N</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Chunwongse</span> <span class="given-names" tagx="given-names" title="given-names">J</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Obokata</span> <span class="given-names" tagx="given-names" title="given-names">J</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Yamaguchi-Shinozaki</span> <span class="given-names" tagx="given-names" title="given-names">K</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Ohto</span> <span class="given-names" tagx="given-names" title="given-names">C</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Torazawa</span> <span class="given-names" tagx="given-names" title="given-names">K</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Meng</span> <span class="given-names" tagx="given-names" title="given-names">BY</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Sugita</span> <span class="given-names" tagx="given-names" title="given-names">M</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Deno</span> <span class="given-names" tagx="given-names" title="given-names">H</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Kamogashira</span> <span class="given-names" tagx="given-names" title="given-names">T</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Yamada</span> <span class="given-names" tagx="given-names" title="given-names">K</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Kusuda</span> <span class="given-names" tagx="given-names" title="given-names">J</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Takaiwa</span> <span class="given-names" tagx="given-names" title="given-names">F</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Kato</span> <span class="given-names" tagx="given-names" title="given-names">A</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Tohdoh</span> <span class="given-names" tagx="given-names" title="given-names">N</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Shimada</span> <span class="given-names" tagx="given-names" title="given-names">H</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Sugiura</span> <span class="given-names" tagx="given-names" title="given-names">M</span> </span> </span> <div class="article-title" title="article-title">
The complete nucleotide sequence of the tobacco chloroplast genome: its gene organization and expression </div> <span class="source" tagx="source" title="source">EMBO J</span> <span class="year" tagx="year" title="year">1986</span> <span class="volume" tagx="volume" title="volume">5</span> <span class="fpage" tagx="fpage" title="fpage">2043</span> <span class="lpage" tagx="lpage" title="lpage">2049</span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/16453699">16453699</a> </span> </div> </li> <li tag="ref"> <a name="B12" /> <div class="citation" tagx="citation" title="citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Goremykin</span> <span class="given-names" tagx="given-names" title="given-names">VV</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Hirsch-Ernst</span> <span class="given-names" tagx="given-names" title="given-names">KI</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Wolfl</span> <span class="given-names" tagx="given-names" title="given-names">S</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Hellwig</span> <span class="given-names" tagx="given-names" title="given-names">FH</span> </span> </span> <div class="article-title" title="article-title">
Analysis of the <i>Amborella trichopoda </i>chloroplast genome sequence suggests that <i>Amborella </i>is not a basal angiosperm </div> <span class="source" tagx="source" title="source">Mol Biol Evol</span> <span class="year" tagx="year" title="year">2003</span> <span class="volume" tagx="volume" title="volume">20</span> <span class="fpage" tagx="fpage" title="fpage">1499</span> <span class="lpage" tagx="lpage" title="lpage">1505</span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/12832641">12832641</a> </span> <span class="pub-id"> <a href="https://doi.org/10.1093/molbev/msg159">10.1093/molbev/msg159</a> </span> </div> </li> <li tag="ref"> <a name="B13" /> <div class="citation" tagx="citation" title="citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Goremykin</span> <span class="given-names" tagx="given-names" title="given-names">VV</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Hirsch-Ernst</span> <span class="given-names" tagx="given-names" title="given-names">KI</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Wolfl</span> <span class="given-names" tagx="given-names" title="given-names">S</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Hellwig</span> <span class="given-names" tagx="given-names" title="given-names">FH</span> </span> </span> <div class="article-title" title="article-title">
The chloroplast genome of <i>Nymphaea alba </i>: whole-genome analyses and the problem of identifying the most basal angiosperm </div> <span class="source" tagx="source" title="source">Mol Biol Evol</span> <span class="year" tagx="year" title="year">2004</span> <span class="volume" tagx="volume" title="volume">21</span> <span class="fpage" tagx="fpage" title="fpage">1445</span> <span class="lpage" tagx="lpage" title="lpage">1454</span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/15084683">15084683</a> </span> <span class="pub-id"> <a href="https://doi.org/10.1093/molbev/msh147">10.1093/molbev/msh147</a> </span> </div> </li> <li tag="ref"> <a name="B14" /> <div class="citation" tagx="citation" title="citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Goremykin</span> <span class="given-names" tagx="given-names" title="given-names">VV</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Hirsch-Ernst</span> <span class="given-names" tagx="given-names" title="given-names">KI</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Wolfl</span> <span class="given-names" tagx="given-names" title="given-names">S</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Hellwig</span> <span class="given-names" tagx="given-names" title="given-names">FH</span> </span> </span> <div class="article-title" title="article-title">
The chloroplast genome of the "basal" angiosperm <i>Calycanthus fertilis </i>– structural and phylogenetic analyses </div> <span class="source" tagx="source" title="source">Plt Syst Evol</span> <span class="year" tagx="year" title="year">2003</span> <span class="volume" tagx="volume" title="volume">242</span> <span class="fpage" tagx="fpage" title="fpage">119</span> <span class="lpage" tagx="lpage" title="lpage">135</span> <span class="pub-id"> <a href="https://doi.org/10.1007/s00606-003-0056-4">10.1007/s00606-003-0056-4</a> </span> </div> </li> <li tag="ref"> <a name="B15" /> <div class="citation" tagx="citation" title="citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Cosner</span> <span class="given-names" tagx="given-names" title="given-names">ME</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Raubeson</span> <span class="given-names" tagx="given-names" title="given-names">LA</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Jansen</span> <span class="given-names" tagx="given-names" title="given-names">RK</span> </span> </span> <div class="article-title" title="article-title">
Chloroplast DNA rearrangments in Campanulaceae: phylogenetic utility of highly rearranged genomes </div> <span class="source" tagx="source" title="source">BMC Evol Biol</span> <span class="year" tagx="year" title="year">2004</span> <span class="volume" tagx="volume" title="volume">4</span> <span class="fpage" tagx="fpage" title="fpage">27</span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/15324459">15324459</a> </span> <span class="pub-id"> <a href="https://doi.org/10.1186/1471-2148-4-27">10.1186/1471-2148-4-27</a> </span> </div> </li> <li tag="ref"> <a name="B16" /> <div class="citation" tagx="citation" title="citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Chumley</span> <span class="given-names" tagx="given-names" title="given-names">TW</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Palmer</span> <span class="given-names" tagx="given-names" title="given-names">JD</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Mower</span> <span class="given-names" tagx="given-names" title="given-names">JP</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Boore</span> <span class="given-names" tagx="given-names" title="given-names">JL</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Fourcade</span> <span class="given-names" tagx="given-names" title="given-names">HM</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Calie</span> <span class="given-names" tagx="given-names" title="given-names">PJ</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Jansen</span> <span class="given-names" tagx="given-names" title="given-names">RK</span> </span> </span> <div class="article-title" title="article-title">
The complete chloroplast genome sequence of <i>Pelargonium </i>x <i>hortorum</i>: Organization and evolution of the largest and most highly rearranged chloroplast genome of land plants </div> <span class="source" tagx="source" title="source">Mol Biol Evol</span> <span class="year" tagx="year" title="year">2006</span> <span class="volume" tagx="volume" title="volume">23</span> <span class="fpage" tagx="fpage" title="fpage">2175</span> <span class="lpage" tagx="lpage" title="lpage">2190</span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/16916942">16916942</a> </span> <span class="pub-id"> <a href="https://doi.org/10.1093/molbev/msl089">10.1093/molbev/msl089</a> </span> </div> </li> <li tag="ref"> <a name="B17" /> <div class="citation" tagx="citation" title="citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Kim</span> <span class="given-names" tagx="given-names" title="given-names">K-J</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Lee</span> <span class="given-names" tagx="given-names" title="given-names">H-L</span> </span> </span> <div class="article-title" title="article-title">
Complete chloroplast genome sequence from Korean Ginseng ( <i>Panax schiseng </i>Nees) and comparative analysis of sequence evolution among 17 vascular plants </div> <span class="source" tagx="source" title="source">DNA Res</span> <span class="year" tagx="year" title="year">2004</span> <span class="volume" tagx="volume" title="volume">11</span> <span class="fpage" tagx="fpage" title="fpage">247</span> <span class="lpage" tagx="lpage" title="lpage">261</span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/15500250">15500250</a> </span> <span class="pub-id"> <a href="https://doi.org/10.1093/dnares/11.4.247">10.1093/dnares/11.4.247</a> </span> </div> </li> <li tag="ref"> <a name="B18" /> <div class="citation" tagx="citation" title="citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Steane</span> <span class="given-names" tagx="given-names" title="given-names">DA</span> </span> </span> <div class="article-title" title="article-title">
Complete nucleotide sequence of the chloroplast genome from the Tasmanian blue gum, <i>Eucalyptus globulus </i>(Myrtaceae) </div> <span class="source" tagx="source" title="source">DNA Res</span> <span class="year" tagx="year" title="year">2005</span> <span class="volume" tagx="volume" title="volume">12</span> <span class="fpage" tagx="fpage" title="fpage">215</span> <span class="lpage" tagx="lpage" title="lpage">220</span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/16303753">16303753</a> </span> <span class="pub-id"> <a href="https://doi.org/10.1093/dnares/dsi006">10.1093/dnares/dsi006</a> </span> </div> </li> <li tag="ref"> <a name="B19" /> <div class="citation" tagx="citation" title="citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Lee</span> <span class="given-names" tagx="given-names" title="given-names">SB</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Kaittanis</span> <span class="given-names" tagx="given-names" title="given-names">C</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Jansen</span> <span class="given-names" tagx="given-names" title="given-names">RK</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Hostetler</span> <span class="given-names" tagx="given-names" title="given-names">JB</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Tallon</span> <span class="given-names" tagx="given-names" title="given-names">LJ</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Town</span> <span class="given-names" tagx="given-names" title="given-names">CD</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Daniell</span> <span class="given-names" tagx="given-names" title="given-names">H</span> </span> </span> <div class="article-title" title="article-title">
The complete chloroplast genome sequence of <i>Gossypium hirsutum </i>: organization and phylogenetic relationships to other angiosperms </div> <span class="source" tagx="source" title="source">BMC Genomics</span> <span class="year" tagx="year" title="year">2006</span> <span class="volume" tagx="volume" title="volume">7</span> <span class="fpage" tagx="fpage" title="fpage">61</span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/16553962">16553962</a> </span> <span class="pub-id"> <a href="https://doi.org/10.1186/1471-2164-7-61">10.1186/1471-2164-7-61</a> </span> </div> </li> <li tag="ref"> <a name="B20" /> <div class="citation" tagx="citation" title="citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Sugiura</span> <span class="given-names" tagx="given-names" title="given-names">M</span> </span> </span> <div class="article-title" title="article-title">
The history of chloroplast genomics</div> <span class="source" tagx="source" title="source">Photosynth Res</span> <span class="year" tagx="year" title="year">2003</span> <span class="volume" tagx="volume" title="volume">76</span> <span class="fpage" tagx="fpage" title="fpage">371</span> <span class="lpage" tagx="lpage" title="lpage">377</span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/16228593">16228593</a> </span> <span class="pub-id"> <a href="https://doi.org/10.1023/A:1024913304263">10.1023/A:1024913304263</a> </span> </div> </li> <li tag="ref"> <a name="B21" /> <div class="citation" tagx="citation" title="citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Daniell</span> <span class="given-names" tagx="given-names" title="given-names">H</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Cohill</span> <span class="given-names" tagx="given-names" title="given-names">PR</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Kumar</span> <span class="given-names" tagx="given-names" title="given-names">S</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Dufourmantel</span> <span class="given-names" tagx="given-names" title="given-names">N</span> </span> </span> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Daniell H, Chase CD</span> </span> </span> <div class="article-title" title="article-title">
Chloroplast genetic engineering</div> <span class="source" tagx="source" title="source">Molecular Biology and Biotechnology of Plant Organelles</span> <span class="year" tagx="year" title="year">2004</span> <span class="publisher-name" tagx="publisher-name" title="publisher-name">Netherlands: Springer Publishers</span> <span class="fpage" tagx="fpage" title="fpage">443</span> <span class="lpage" tagx="lpage" title="lpage">490</span> </div> </li> <li tag="ref"> <a name="B22" /> <div class="citation" tagx="citation" title="citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Moon</span> <span class="given-names" tagx="given-names" title="given-names">E</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Kao</span> <span class="given-names" tagx="given-names" title="given-names">T-H</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Wu</span> <span class="given-names" tagx="given-names" title="given-names">R</span> </span> </span> <div class="article-title" title="article-title">
Rice chloroplast DNA molecules are heterogeneous as revealed by DNA sequences of a cluster of genes </div> <span class="source" tagx="source" title="source">Nucl Acids Res</span> <span class="year" tagx="year" title="year">1987</span> <span class="volume" tagx="volume" title="volume">15</span> <span class="fpage" tagx="fpage" title="fpage">611</span> <span class="lpage" tagx="lpage" title="lpage">630</span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/3029686">3029686</a> </span> <span class="pub-id"> <a href="https://doi.org/10.1093/nar/15.2.611">10.1093/nar/15.2.611</a> </span> </div> </li> <li tag="ref"> <a name="B23" /> <div class="citation" tagx="citation" title="citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Johnson</span> <span class="given-names" tagx="given-names" title="given-names">LB</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Palmer</span> <span class="given-names" tagx="given-names" title="given-names">JD</span> </span> </span> <div class="article-title" title="article-title">
Heteroplasmy of chloroplast DNA in <i>Medicago </i> </div> <span class="source" tagx="source" title="source">Plt Mol Biol</span> <span class="year" tagx="year" title="year">1989</span> <span class="volume" tagx="volume" title="volume">12</span> <span class="fpage" tagx="fpage" title="fpage">3</span> <span class="lpage" tagx="lpage" title="lpage">11</span> <span class="pub-id"> <a href="https://doi.org/10.1007/BF00017442">10.1007/BF00017442</a> </span> </div> </li> <li tag="ref"> <a name="B24" /> <div class="citation" tagx="citation" title="citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Frey</span> <span class="given-names" tagx="given-names" title="given-names">JE</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Frey</span> <span class="given-names" tagx="given-names" title="given-names">B</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Forcioli</span> <span class="given-names" tagx="given-names" title="given-names">D</span> </span> </span> <div class="article-title" title="article-title">
Quantitative assessment of heteroplasmy levels in <i>Senecio vulgaris </i>chloroplast DNA </div> <span class="source" tagx="source" title="source">Genetica</span> <span class="year" tagx="year" title="year">2005</span> <span class="volume" tagx="volume" title="volume">123</span> <span class="fpage" tagx="fpage" title="fpage">255</span> <span class="lpage" tagx="lpage" title="lpage">261</span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/15954496">15954496</a> </span> <span class="pub-id"> <a href="https://doi.org/10.1007/s10709-004-3711-y">10.1007/s10709-004-3711-y</a> </span> </div> </li> <li tag="ref"> <a name="B25" /> <div class="citation" tagx="citation" title="citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Raubeson</span> <span class="given-names" tagx="given-names" title="given-names">LA</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Jansen</span> <span class="given-names" tagx="given-names" title="given-names">RK</span> </span> </span> <div class="article-title" title="article-title">
A rare chloroplast DNA structural mutation is shared by all conifers</div> <span class="source" tagx="source" title="source">Biochem Syst Ecol</span> <span class="year" tagx="year" title="year">1992</span> <span class="volume" tagx="volume" title="volume">20</span> <span class="fpage" tagx="fpage" title="fpage">17</span> <span class="lpage" tagx="lpage" title="lpage">24</span> <span class="pub-id"> <a href="https://doi.org/10.1016/0305-1978(92)90067-N">10.1016/0305-1978(92)90067-N</a> </span> </div> </li> <li tag="ref"> <a name="B26" /> <div class="citation" tagx="citation" title="citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Rokas</span> <span class="given-names" tagx="given-names" title="given-names">A</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Holland</span> <span class="given-names" tagx="given-names" title="given-names">PWH</span> </span> </span> <div class="article-title" title="article-title">
Rare genomic changes as a tool for phylogenetics</div> <span class="source" tagx="source" title="source">Trends Ecol Evol</span> <span class="volume" tagx="volume" title="volume">15</span> <span class="fpage" tagx="fpage" title="fpage">454</span> <span class="lpage" tagx="lpage" title="lpage">459</span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/11050348">11050348</a> </span> <span class="pub-id"> <a href="https://doi.org/10.1016/S0169-5347(00)01967-4">10.1016/S0169-5347(00)01967-4</a> </span> </div> </li> <li tag="ref"> <a name="B27" /> <div class="citation" tagx="citation" title="citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Goulding</span> <span class="given-names" tagx="given-names" title="given-names">SE</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Olmstead</span> <span class="given-names" tagx="given-names" title="given-names">RG</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Morden</span> <span class="given-names" tagx="given-names" title="given-names">CW</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Wolfe</span> <span class="given-names" tagx="given-names" title="given-names">KH</span> </span> </span> <div class="article-title" title="article-title">
Ebb and flow of the chloroplast inverted repeat</div> <span class="source" tagx="source" title="source">Mol Gen Genet</span> <span class="year" tagx="year" title="year">1996</span> <span class="volume" tagx="volume" title="volume">252</span> <span class="fpage" tagx="fpage" title="fpage">195</span> <span class="lpage" tagx="lpage" title="lpage">206</span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/8804393">8804393</a> </span> <span class="pub-id"> <a href="https://doi.org/10.1007/BF02173220">10.1007/BF02173220</a> </span> </div> </li> <li tag="ref"> <a name="B28" /> <div class="citation" tagx="citation" title="citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Sugiura</span> <span class="given-names" tagx="given-names" title="given-names">M</span> </span> </span> <div class="article-title" title="article-title">
The chloroplast genome</div> <span class="source" tagx="source" title="source">Plant Mol Biol</span> <span class="year" tagx="year" title="year">1992</span> <span class="volume" tagx="volume" title="volume">19</span> <span class="fpage" tagx="fpage" title="fpage">149</span> <span class="lpage" tagx="lpage" title="lpage">168</span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/1600166">1600166</a> </span> <span class="pub-id"> <a href="https://doi.org/10.1007/BF00015612">10.1007/BF00015612</a> </span> </div> </li> <li tag="ref"> <a name="B29" /> <div class="citation" tagx="citation" title="citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Schmitz-Linneweber</span> <span class="given-names" tagx="given-names" title="given-names">C</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Maier</span> <span class="given-names" tagx="given-names" title="given-names">RM</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Alcaraz</span> <span class="given-names" tagx="given-names" title="given-names">JP</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Cottet</span> <span class="given-names" tagx="given-names" title="given-names">A</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Herrmann</span> <span class="given-names" tagx="given-names" title="given-names">RG</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Mache</span> <span class="given-names" tagx="given-names" title="given-names">R</span> </span> </span> <div class="article-title" title="article-title">
The plastid chromosome of spinach ( <i>Spinacia oleracea</i>): complete nucleotide sequence and gene organization </div> <span class="source" tagx="source" title="source">Plt Mol Biol</span> <span class="year" tagx="year" title="year">2001</span> <span class="volume" tagx="volume" title="volume">45</span> <span class="fpage" tagx="fpage" title="fpage">307</span> <span class="lpage" tagx="lpage" title="lpage">315</span> <span class="pub-id"> <a href="https://doi.org/10.1023/A:1006478403810">10.1023/A:1006478403810</a> </span> </div> </li> <li tag="ref"> <a name="B30" /> <div class="citation" tagx="citation" title="citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Chaw</span> <span class="given-names" tagx="given-names" title="given-names">SM</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Chang</span> <span class="given-names" tagx="given-names" title="given-names">CC</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Chen</span> <span class="given-names" tagx="given-names" title="given-names">HL</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Li</span> <span class="given-names" tagx="given-names" title="given-names">WH</span> </span> </span> <div class="article-title" title="article-title">
Dating the monocot-dicot divergence and the origin of the core eudicots using whole chloroplast genomes </div> <span class="source" tagx="source" title="source">J Mol Evol</span> <span class="year" tagx="year" title="year">2004</span> <span class="volume" tagx="volume" title="volume">58</span> <span class="fpage" tagx="fpage" title="fpage">424</span> <span class="lpage" tagx="lpage" title="lpage">441</span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/15114421">15114421</a> </span> <span class="pub-id"> <a href="https://doi.org/10.1007/s00239-003-2564-9">10.1007/s00239-003-2564-9</a> </span> </div> </li> <li tag="ref"> <a name="B31" /> <div class="citation" tagx="citation" title="citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Echols</span> <span class="given-names" tagx="given-names" title="given-names">N</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Harrison</span> <span class="given-names" tagx="given-names" title="given-names">P</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Balasubramanian</span> <span class="given-names" tagx="given-names" title="given-names">S</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Luscombe</span> <span class="given-names" tagx="given-names" title="given-names">NM</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Bertone</span> <span class="given-names" tagx="given-names" title="given-names">P</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Zhang</span> <span class="given-names" tagx="given-names" title="given-names">Z</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Gerstein</span> <span class="given-names" tagx="given-names" title="given-names">M</span> </span> </span> <div class="article-title" title="article-title">
Comprehensive analysis of amino acid and nuclear composition in eukaryotic genomes, comparing genes and pseudogenes </div> <span class="source" tagx="source" title="source">Nucl Acids Res</span> <span class="year" tagx="year" title="year">2002</span> <span class="volume" tagx="volume" title="volume">30</span> <span class="fpage" tagx="fpage" title="fpage">2515</span> <span class="lpage" tagx="lpage" title="lpage">2523</span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/12034841">12034841</a> </span> <span class="pub-id"> <a href="https://doi.org/10.1093/nar/30.11.2515">10.1093/nar/30.11.2515</a> </span> </div> </li> <li tag="ref"> <a name="B32" /> <div class="citation" tagx="citation" title="citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Ovcharenko</span> <span class="given-names" tagx="given-names" title="given-names">I</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Loots</span> <span class="given-names" tagx="given-names" title="given-names">GG</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Giardine</span> <span class="given-names" tagx="given-names" title="given-names">BM</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Hou</span> <span class="given-names" tagx="given-names" title="given-names">M</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Ma</span> <span class="given-names" tagx="given-names" title="given-names">J</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Hardison</span> <span class="given-names" tagx="given-names" title="given-names">RC</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Stubbs</span> <span class="given-names" tagx="given-names" title="given-names">L</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Miller</span> <span class="given-names" tagx="given-names" title="given-names">W</span> </span> </span> <div class="article-title" title="article-title">
Mulan: Multiple-sequence local alignment and visualization for studying function and evolution </div> <span class="source" tagx="source" title="source">Genet Res</span> <span class="year" tagx="year" title="year">2005</span> <span class="volume" tagx="volume" title="volume">15</span> <span class="fpage" tagx="fpage" title="fpage">184</span> <span class="lpage" tagx="lpage" title="lpage">194</span> </div> </li> <li tag="ref"> <a name="B33" /> <div class="citation" tagx="citation" title="citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Perry</span> <span class="given-names" tagx="given-names" title="given-names">AS</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Wolfe</span> <span class="given-names" tagx="given-names" title="given-names">KH</span> </span> </span> <div class="article-title" title="article-title">
Nucleotide substitution rates in legume chloroplast DNA depend on the presence of the inverted repeat </div> <span class="source" tagx="source" title="source">J Mol Evol</span> <span class="year" tagx="year" title="year">2002</span> <span class="volume" tagx="volume" title="volume">55</span> <span class="fpage" tagx="fpage" title="fpage">501</span> <span class="lpage" tagx="lpage" title="lpage">508</span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/12399924">12399924</a> </span> <span class="pub-id"> <a href="https://doi.org/10.1007/s00239-002-2333-y">10.1007/s00239-002-2333-y</a> </span> </div> </li> <li tag="ref"> <a name="B34" /> <div class="citation" tagx="citation" title="citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Wolf</span> <span class="given-names" tagx="given-names" title="given-names">PG</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Rowe</span> <span class="given-names" tagx="given-names" title="given-names">CA</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Sinclair</span> <span class="given-names" tagx="given-names" title="given-names">RG</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Hasebe</span> <span class="given-names" tagx="given-names" title="given-names">M</span> </span> </span> <div class="article-title" title="article-title">
Complete nucleotide sequence of the chloroplast genome from a leptosporangiate fern, <i>Adiantum capillus-veneris </i>L </div> <span class="source" tagx="source" title="source">DNA Research</span> <span class="year" tagx="year" title="year">2003</span> <span class="volume" tagx="volume" title="volume">10</span> <span class="fpage" tagx="fpage" title="fpage">59</span> <span class="lpage" tagx="lpage" title="lpage">65</span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/12755170">12755170</a> </span> <span class="pub-id"> <a href="https://doi.org/10.1093/dnares/10.2.59">10.1093/dnares/10.2.59</a> </span> </div> </li> <li tag="ref"> <a name="B35" /> <div class="citation" tagx="citation" title="citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Kato</span> <span class="given-names" tagx="given-names" title="given-names">T</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Kaneko</span> <span class="given-names" tagx="given-names" title="given-names">T</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Sato</span> <span class="given-names" tagx="given-names" title="given-names">S</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Nakamura</span> <span class="given-names" tagx="given-names" title="given-names">Y</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Tabata</span> <span class="given-names" tagx="given-names" title="given-names">S</span> </span> </span> <div class="article-title" title="article-title">
Complete structure of the chloroplast genome of a legume, <i>Lotus japonicus</i> </div> <span class="source" tagx="source" title="source">DNA Res</span> <span class="year" tagx="year" title="year">2000</span> <span class="volume" tagx="volume" title="volume">7</span> <span class="fpage" tagx="fpage" title="fpage">323</span> <span class="lpage" tagx="lpage" title="lpage">330</span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/11214967">11214967</a> </span> <span class="pub-id"> <a href="https://doi.org/10.1093/dnares/7.6.323">10.1093/dnares/7.6.323</a> </span> </div> </li> <li tag="ref"> <a name="B36" /> <div class="citation" tagx="citation" title="citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Shimada</span> <span class="given-names" tagx="given-names" title="given-names">H</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Sugiura</span> <span class="given-names" tagx="given-names" title="given-names">M</span> </span> </span> <div class="article-title" title="article-title">
Fine structural features of the chloroplast genome: comparison of the sequenced chloroplast genomes </div> <span class="source" tagx="source" title="source">Nucl Acids Res</span> <span class="year" tagx="year" title="year">1991</span> <span class="volume" tagx="volume" title="volume">19</span> <span class="fpage" tagx="fpage" title="fpage">983</span> <span class="lpage" tagx="lpage" title="lpage">995</span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/1708498">1708498</a> </span> <span class="pub-id"> <a href="https://doi.org/10.1093/nar/19.5.983">10.1093/nar/19.5.983</a> </span> </div> </li> <li tag="ref"> <a name="B37" /> <div class="citation" tagx="citation" title="citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Martin</span> <span class="given-names" tagx="given-names" title="given-names">W</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Rujan</span> <span class="given-names" tagx="given-names" title="given-names">T</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Richly</span> <span class="given-names" tagx="given-names" title="given-names">E</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Hansen</span> <span class="given-names" tagx="given-names" title="given-names">A</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Cornelsen</span> <span class="given-names" tagx="given-names" title="given-names">S</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Lins</span> <span class="given-names" tagx="given-names" title="given-names">T</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Leister</span> <span class="given-names" tagx="given-names" title="given-names">D</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Stoebe</span> <span class="given-names" tagx="given-names" title="given-names">B</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Hasegawa</span> <span class="given-names" tagx="given-names" title="given-names">M</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Penny</span> <span class="given-names" tagx="given-names" title="given-names">D</span> </span> </span> <div class="article-title" title="article-title">
Evolutionary analysis of <i>Arabidopsis</i>, cyanobacterial, and chloroplast genomes reveals plastid phylogeny and thousands of cyanobacterial genes in the nucleus </div> <span class="source" tagx="source" title="source">Proc Natl Acad Sci USA</span> <span class="year" tagx="year" title="year">2002</span> <span class="volume" tagx="volume" title="volume">99</span> <span class="fpage" tagx="fpage" title="fpage">12246</span> <span class="lpage" tagx="lpage" title="lpage">12251</span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/12218172">12218172</a> </span> <span class="pub-id"> <a href="https://doi.org/10.1073/pnas.182432999">10.1073/pnas.182432999</a> </span> </div> </li> <li tag="ref"> <a name="B38" /> <div class="citation" tagx="citation" title="citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Liu</span> <span class="given-names" tagx="given-names" title="given-names">Q</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Xue</span> <span class="given-names" tagx="given-names" title="given-names">Q</span> </span> </span> <div class="article-title" title="article-title">
Comparative studies on codon usage patterns of chloroplasts and their host nuclear genes in four plant species </div> <span class="source" tagx="source" title="source">J Genet</span> <span class="year" tagx="year" title="year">2005</span> <span class="volume" tagx="volume" title="volume">84</span> <span class="fpage" tagx="fpage" title="fpage">55</span> <span class="lpage" tagx="lpage" title="lpage">62</span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/15876584">15876584</a> </span> </div> </li> <li tag="ref"> <a name="B39" /> <div class="citation" tagx="citation" title="citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Morton</span> <span class="given-names" tagx="given-names" title="given-names">BR</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Levin</span> <span class="given-names" tagx="given-names" title="given-names">JA</span> </span> </span> <div class="article-title" title="article-title">
The atypical codon usage of the plant <i>psbA </i>gene may be the remnant of an ancestral bias </div> <span class="source" tagx="source" title="source">Proc Natl Acad Sci USA</span> <span class="year" tagx="year" title="year">1997</span> <span class="volume" tagx="volume" title="volume">94</span> <span class="fpage" tagx="fpage" title="fpage">11434</span> <span class="lpage" tagx="lpage" title="lpage">11438</span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/9326627">9326627</a> </span> <span class="pub-id"> <a href="https://doi.org/10.1073/pnas.94.21.11434">10.1073/pnas.94.21.11434</a> </span> </div> </li> <li tag="ref"> <a name="B40" /> <div class="citation" tagx="citation" title="citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Knight</span> <span class="given-names" tagx="given-names" title="given-names">RD</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Freeland</span> <span class="given-names" tagx="given-names" title="given-names">SJ</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Landweber</span> <span class="given-names" tagx="given-names" title="given-names">LF</span> </span> </span> <div class="article-title" title="article-title">
A simple model based on mutation and selection explains trends in codon and amino-acid usage and GC composition within and across genomes </div> <span class="source" tagx="source" title="source">Genome Biol</span> <span class="year" tagx="year" title="year">2001</span> <span class="volume" tagx="volume" title="volume">2</span> <span class="fpage" tagx="fpage" title="fpage">research0010.1</span> <span class="lpage" tagx="lpage" title="lpage">0010.13</span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/11305938">11305938</a> </span> <span class="pub-id"> <a href="https://doi.org/10.1186/gb-2001-2-4-research0010">10.1186/gb-2001-2-4-research0010</a> </span> </div> </li> <li tag="ref"> <a name="B41" /> <div class="citation" tagx="citation" title="citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Codon</span> <span class="given-names" tagx="given-names" title="given-names">W</span> </span> </span> <a href="http://bioweb.pasteur.fr/seqanal/interfaces/codonw.html" /> </div> </li> <li tag="ref"> <a name="B42" /> <div class="citation" tagx="citation" title="citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Wright</span> <span class="given-names" tagx="given-names" title="given-names">F</span> </span> </span> <div class="article-title" title="article-title">
The 'effective number of codons' used in a gene</div> <span class="source" tagx="source" title="source">Gene</span> <span class="year" tagx="year" title="year">1990</span> <span class="volume" tagx="volume" title="volume">87</span> <span class="fpage" tagx="fpage" title="fpage">23</span> <span class="lpage" tagx="lpage" title="lpage">29</span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/2110097">2110097</a> </span> <span class="pub-id"> <a href="https://doi.org/10.1016/0378-1119(90)90491-9">10.1016/0378-1119(90)90491-9</a> </span> </div> </li> <li tag="ref"> <a name="B43" /> <div class="citation" tagx="citation" title="citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Novembre</span> <span class="given-names" tagx="given-names" title="given-names">J</span> </span> </span> <div class="article-title" title="article-title">
Accounting for background nucleotide composition when measuring codon usage bias</div> <span class="source" tagx="source" title="source">Mol Biol Evol</span> <span class="year" tagx="year" title="year">2002</span> <span class="volume" tagx="volume" title="volume">19</span> <span class="fpage" tagx="fpage" title="fpage">1390</span> <span class="lpage" tagx="lpage" title="lpage">1394</span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/12140252">12140252</a> </span> </div> </li> <li tag="ref"> <a name="B44" /> <div class="citation" tagx="citation" title="citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Wall</span> <span class="given-names" tagx="given-names" title="given-names">DP</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Herbeck</span> <span class="given-names" tagx="given-names" title="given-names">JT</span> </span> </span> <div class="article-title" title="article-title">
Evolutionary patterns of codon usage in the chloroplast gene <i>rbcL </i> </div> <span class="source" tagx="source" title="source">J Mol Evol</span> <span class="year" tagx="year" title="year">2003</span> <span class="volume" tagx="volume" title="volume">56</span> <span class="fpage" tagx="fpage" title="fpage">673</span> <span class="lpage" tagx="lpage" title="lpage">688</span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/12911031">12911031</a> </span> <span class="pub-id"> <a href="https://doi.org/10.1007/s00239-002-2436-8">10.1007/s00239-002-2436-8</a> </span> </div> </li> <li tag="ref"> <a name="B45" /> <div class="citation" tagx="citation" title="citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Asano</span> <span class="given-names" tagx="given-names" title="given-names">T</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Tsudzuki</span> <span class="given-names" tagx="given-names" title="given-names">T</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Takahashi</span> <span class="given-names" tagx="given-names" title="given-names">S</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Shimada</span> <span class="given-names" tagx="given-names" title="given-names">H</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Kadowaki</span> <span class="given-names" tagx="given-names" title="given-names">K</span> </span> </span> <div class="article-title" title="article-title">
Complete nucleotide sequence of the sugarcane ( <i>Saccharum officinarum</i>) chloroplast genome: a comparative analysis of four monocot chloroplast genomes </div> <span class="source" tagx="source" title="source">DNA Res</span> <span class="year" tagx="year" title="year">2004</span> <span class="volume" tagx="volume" title="volume">11</span> <span class="fpage" tagx="fpage" title="fpage">93</span> <span class="lpage" tagx="lpage" title="lpage">99</span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/15449542">15449542</a> </span> <span class="pub-id"> <a href="https://doi.org/10.1093/dnares/11.2.93">10.1093/dnares/11.2.93</a> </span> </div> </li> <li tag="ref"> <a name="B46" /> <div class="citation" tagx="citation" title="citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Timme</span> <span class="given-names" tagx="given-names" title="given-names">RE</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Kuehl</span> <span class="given-names" tagx="given-names" title="given-names">JV</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Boore</span> <span class="given-names" tagx="given-names" title="given-names">JL</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Jansen</span> <span class="given-names" tagx="given-names" title="given-names">RK</span> </span> </span> <div class="article-title" title="article-title">
A comparative analysis of the <i>Lactuca </i>and <i>Helianthus </i>(Asteraceae) plastid genomes: Identification of divergent regions and categorization of shared repeats </div> <span class="source" tagx="source" title="source">Amer J Bot</span> <span class="year" tagx="year" title="year">2007</span> <span class="volume" tagx="volume" title="volume">94</span> <span class="fpage" tagx="fpage" title="fpage">302</span> <span class="lpage" tagx="lpage" title="lpage">312</span> </div> </li> <li tag="ref"> <a name="B47" /> <div class="citation" tagx="citation" title="citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Daniell</span> <span class="given-names" tagx="given-names" title="given-names">H</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Lee</span> <span class="given-names" tagx="given-names" title="given-names">S-B</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Grevich</span> <span class="given-names" tagx="given-names" title="given-names">J</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Saski</span> <span class="given-names" tagx="given-names" title="given-names">C</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Quesada-Vargas</span> <span class="given-names" tagx="given-names" title="given-names">T</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Guda</span> <span class="given-names" tagx="given-names" title="given-names">C</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Tomkins</span> <span class="given-names" tagx="given-names" title="given-names">J</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Jansen</span> <span class="given-names" tagx="given-names" title="given-names">RK</span> </span> </span> <div class="article-title" title="article-title">
Complete chloroplast genome sequences of <i>Solanum bulbocastanum</i>, <i>Solanum lycopersicum </i>and comparative analyses with other Solanaceae genomes </div> <span class="source" tagx="source" title="source">Theor Appl Genet</span> <span class="year" tagx="year" title="year">2006</span> <span class="volume" tagx="volume" title="volume">112</span> <span class="fpage" tagx="fpage" title="fpage">1503</span> <span class="lpage" tagx="lpage" title="lpage">1518</span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/16575560">16575560</a> </span> <span class="pub-id"> <a href="https://doi.org/10.1007/s00122-006-0254-x">10.1007/s00122-006-0254-x</a> </span> </div> </li> <li tag="ref"> <a name="B48" /> <div class="citation" tagx="citation" title="citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Kahlau</span> <span class="given-names" tagx="given-names" title="given-names">S</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Aspinall</span> <span class="given-names" tagx="given-names" title="given-names">S</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Gray</span> <span class="given-names" tagx="given-names" title="given-names">JC</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Bock</span> <span class="given-names" tagx="given-names" title="given-names">R</span> </span> </span> <div class="article-title" title="article-title">
Sequence of the tomato chloroplast DNA and evolutionary comparisons of solanaceous plastid genomes </div> <span class="source" tagx="source" title="source">J Mol Evol</span> <span class="year" tagx="year" title="year">2006</span> <span class="volume" tagx="volume" title="volume">63</span> <span class="fpage" tagx="fpage" title="fpage">194</span> <span class="lpage" tagx="lpage" title="lpage">207</span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/16830097">16830097</a> </span> <span class="pub-id"> <a href="https://doi.org/10.1007/s00239-005-0254-5">10.1007/s00239-005-0254-5</a> </span> </div> </li> <li tag="ref"> <a name="B49" /> <div class="citation" tagx="citation" title="citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Tang</span> <span class="given-names" tagx="given-names" title="given-names">J</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Xia</span> <span class="given-names" tagx="given-names" title="given-names">H</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Cao</span> <span class="given-names" tagx="given-names" title="given-names">M</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Zhang</span> <span class="given-names" tagx="given-names" title="given-names">X</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Zeng</span> <span class="given-names" tagx="given-names" title="given-names">W</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Hu</span> <span class="given-names" tagx="given-names" title="given-names">S</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Tong</span> <span class="given-names" tagx="given-names" title="given-names">W</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Wang</span> <span class="given-names" tagx="given-names" title="given-names">J</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Yu</span> <span class="given-names" tagx="given-names" title="given-names">J</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Yang</span> <span class="given-names" tagx="given-names" title="given-names">H</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Zhu</span> <span class="given-names" tagx="given-names" title="given-names">L</span> </span> </span> <div class="article-title" title="article-title">
A comparison of rice chloroplast genomes</div> <span class="source" tagx="source" title="source">Pl Phys</span> <span class="year" tagx="year" title="year">2004</span> <span class="volume" tagx="volume" title="volume">135</span> <span class="fpage" tagx="fpage" title="fpage">412</span> <span class="lpage" tagx="lpage" title="lpage">420</span> <span class="pub-id"> <a href="https://doi.org/10.1104/pp.103.031245">10.1104/pp.103.031245</a> </span> </div> </li> <li tag="ref"> <a name="B50" /> <div class="citation" tagx="citation" title="citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Schmitz-Linneweber</span> <span class="given-names" tagx="given-names" title="given-names">C</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Regel</span> <span class="given-names" tagx="given-names" title="given-names">R</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Du</span> <span class="given-names" tagx="given-names" title="given-names">TG</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Hupfer</span> <span class="given-names" tagx="given-names" title="given-names">H</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Herrmann</span> <span class="given-names" tagx="given-names" title="given-names">RG</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Maier</span> <span class="given-names" tagx="given-names" title="given-names">RM</span> </span> </span> <div class="article-title" title="article-title">
The plastid chromosome of <i>Atropa belladonna </i>and its comparison with that of <i>Nicotiana tabacum</i>: the role of RNA editing in generating divergence in the process of plant speciation </div> <span class="source" tagx="source" title="source">Mol Biol Evol</span> <span class="year" tagx="year" title="year">2002</span> <span class="volume" tagx="volume" title="volume">19</span> <span class="fpage" tagx="fpage" title="fpage">1602</span> <span class="lpage" tagx="lpage" title="lpage">1612</span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/12200487">12200487</a> </span> </div> </li> <li tag="ref"> <a name="B51" /> <div class="citation" tagx="citation" title="citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Ellegren</span> <span class="given-names" tagx="given-names" title="given-names">H</span> </span> </span> <div class="article-title" title="article-title">
Microsatellites: simple sequences with complex evolution</div> <span class="source" tagx="source" title="source">Nature Reviews Genetics</span> <span class="year" tagx="year" title="year">2004</span> <span class="volume" tagx="volume" title="volume">5</span> <span class="fpage" tagx="fpage" title="fpage">435</span> <span class="lpage" tagx="lpage" title="lpage">445</span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/15153996">15153996</a> </span> <span class="pub-id"> <a href="https://doi.org/10.1038/nrg1348">10.1038/nrg1348</a> </span> </div> </li> <li tag="ref"> <a name="B52" /> <div class="citation" tagx="citation" title="citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Rose</span> <span class="given-names" tagx="given-names" title="given-names">O</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Falush</span> <span class="given-names" tagx="given-names" title="given-names">D</span> </span> </span> <div class="article-title" title="article-title">
A threshold size for microsatellite expansion</div> <span class="source" tagx="source" title="source">Mol Biol Evol</span> <span class="year" tagx="year" title="year">1998</span> <span class="volume" tagx="volume" title="volume">15</span> <span class="fpage" tagx="fpage" title="fpage">613</span> <span class="lpage" tagx="lpage" title="lpage">615</span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/9580993">9580993</a> </span> </div> </li> <li tag="ref"> <a name="B53" /> <div class="citation" tagx="citation" title="citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Dechering</span> <span class="given-names" tagx="given-names" title="given-names">KJ</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Cuelenaere</span> <span class="given-names" tagx="given-names" title="given-names">K</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Konings</span> <span class="given-names" tagx="given-names" title="given-names">RN</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Leunissen</span> <span class="given-names" tagx="given-names" title="given-names">JA</span> </span> </span> <div class="article-title" title="article-title">
Distinct frequency-distributions of homopolymeric DNA tracts in different genomes</div> <span class="source" tagx="source" title="source">Nucl Acids Res</span> <span class="year" tagx="year" title="year">1998</span> <span class="volume" tagx="volume" title="volume">26</span> <span class="fpage" tagx="fpage" title="fpage">4056</span> <span class="lpage" tagx="lpage" title="lpage">4062</span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/9705519">9705519</a> </span> <span class="pub-id"> <a href="https://doi.org/10.1093/nar/26.17.4056">10.1093/nar/26.17.4056</a> </span> </div> </li> <li tag="ref"> <a name="B54" /> <div class="citation" tagx="citation" title="citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Provan</span> <span class="given-names" tagx="given-names" title="given-names">J</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Powell</span> <span class="given-names" tagx="given-names" title="given-names">W</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Hollingsworth</span> <span class="given-names" tagx="given-names" title="given-names">PM</span> </span> </span> <div class="article-title" title="article-title">
Chloroplast microsatellites: new tools for studies in plant ecology and evolution</div> <span class="source" tagx="source" title="source">TREE</span> <span class="year" tagx="year" title="year">2001</span> <span class="volume" tagx="volume" title="volume">16</span> <span class="fpage" tagx="fpage" title="fpage">142</span> <span class="lpage" tagx="lpage" title="lpage">147</span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/11179578">11179578</a> </span> </div> </li> <li tag="ref"> <a name="B55" /> <div class="citation" tagx="citation" title="citation"> <div class="article-title" title="article-title">
Wessa Free Statistics Software</div> <a href="http://www.wessa.net/" /> </div> </li> <li tag="ref"> <a name="B56" /> <div class="citation" tagx="citation" title="citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Denver</span> <span class="given-names" tagx="given-names" title="given-names">DR</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Morris</span> <span class="given-names" tagx="given-names" title="given-names">K</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Kewalramani</span> <span class="given-names" tagx="given-names" title="given-names">A</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Harris</span> <span class="given-names" tagx="given-names" title="given-names">KE</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Chow</span> <span class="given-names" tagx="given-names" title="given-names">A</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Estes</span> <span class="given-names" tagx="given-names" title="given-names">S</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Lynch</span> <span class="given-names" tagx="given-names" title="given-names">M</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Thomas</span> <span class="given-names" tagx="given-names" title="given-names">WK</span> </span> </span> <div class="article-title" title="article-title">
Abundance, distribution, and mutation rates of homopolymeric nucleotide runs in the genome of <i>Caenorhabditis elegans </i> </div> <span class="source" tagx="source" title="source">J Mol Evol</span> <span class="year" tagx="year" title="year">2004</span> <span class="volume" tagx="volume" title="volume">58</span> <span class="fpage" tagx="fpage" title="fpage">584</span> <span class="lpage" tagx="lpage" title="lpage">595</span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/15170261">15170261</a> </span> <span class="pub-id"> <a href="https://doi.org/10.1007/s00239-004-2580-4">10.1007/s00239-004-2580-4</a> </span> </div> </li> <li tag="ref"> <a name="B57" /> <div class="citation" tagx="citation" title="citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Lovett</span> <span class="given-names" tagx="given-names" title="given-names">ST</span> </span> </span> <div class="article-title" title="article-title">
Encoded errors: mutations and rearrangements mediated by misalignment at repetitive DNA sequences </div> <span class="source" tagx="source" title="source">Mol Microbiol</span> <span class="year" tagx="year" title="year">2004</span> <span class="volume" tagx="volume" title="volume">52</span> <span class="fpage" tagx="fpage" title="fpage">1243</span> <span class="lpage" tagx="lpage" title="lpage">1253</span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/15165229">15165229</a> </span> <span class="pub-id"> <a href="https://doi.org/10.1111/j.1365-2958.2004.04076.x">10.1111/j.1365-2958.2004.04076.x</a> </span> </div> </li> <li tag="ref"> <a name="B58" /> <div class="citation" tagx="citation" title="citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Dieringer</span> <span class="given-names" tagx="given-names" title="given-names">D</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Schlötterer</span> <span class="given-names" tagx="given-names" title="given-names">C</span> </span> </span> <div class="article-title" title="article-title">
Two distinct modes of microsatellite mutation processes: evidence from the complete genomic sequences of nine species </div> <span class="source" tagx="source" title="source">Genome Res</span> <span class="year" tagx="year" title="year">2003</span> <span class="volume" tagx="volume" title="volume">13</span> <span class="fpage" tagx="fpage" title="fpage">2242</span> <span class="lpage" tagx="lpage" title="lpage">2251</span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/14525926">14525926</a> </span> <span class="pub-id"> <a href="https://doi.org/10.1101/gr.1416703">10.1101/gr.1416703</a> </span> </div> </li> <li tag="ref"> <a name="B59" /> <div class="citation" tagx="citation" title="citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Almeida</span> <span class="given-names" tagx="given-names" title="given-names">P</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Penha-Gonçalves</span> <span class="given-names" tagx="given-names" title="given-names">C</span> </span> </span> <div class="article-title" title="article-title">
Long perfect dinucleotide repeats are typical of vertebrates, show motif preferences and size convergence </div> <span class="source" tagx="source" title="source">Mol Biol Evol</span> <span class="year" tagx="year" title="year">2004</span> <span class="volume" tagx="volume" title="volume">21</span> <span class="fpage" tagx="fpage" title="fpage">1226</span> <span class="lpage" tagx="lpage" title="lpage">1233</span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/15014141">15014141</a> </span> <span class="pub-id"> <a href="https://doi.org/10.1093/molbev/msh108">10.1093/molbev/msh108</a> </span> </div> </li> <li tag="ref"> <a name="B60" /> <div class="citation" tagx="citation" title="citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Levinson</span> <span class="given-names" tagx="given-names" title="given-names">G</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Gutman</span> <span class="given-names" tagx="given-names" title="given-names" /> </span> </span> <div class="article-title" title="article-title">
Slipped-strand mispairing: a major mechanism for DNA sequence evolution</div> <span class="source" tagx="source" title="source">Mol Biol Evol</span> <span class="year" tagx="year" title="year">1987</span> <span class="volume" tagx="volume" title="volume">4</span> <span class="fpage" tagx="fpage" title="fpage">203</span> <span class="lpage" tagx="lpage" title="lpage">221</span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/3328815">3328815</a> </span> </div> </li> <li tag="ref"> <a name="B61" /> <div class="citation" tagx="citation" title="citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Milligan</span> <span class="given-names" tagx="given-names" title="given-names">BG</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Hampton</span> <span class="given-names" tagx="given-names" title="given-names">JN</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Palmer</span> <span class="given-names" tagx="given-names" title="given-names">JD</span> </span> </span> <div class="article-title" title="article-title">
Dispersed repeats and structural reorganization in subclover chloroplast DNA</div> <span class="source" tagx="source" title="source">Mol Biol Evol</span> <span class="year" tagx="year" title="year">1989</span> <span class="volume" tagx="volume" title="volume">6</span> <span class="fpage" tagx="fpage" title="fpage">355</span> <span class="lpage" tagx="lpage" title="lpage">368</span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/2615639">2615639</a> </span> </div> </li> <li tag="ref"> <a name="B62" /> <div class="citation" tagx="citation" title="citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Cosner</span> <span class="given-names" tagx="given-names" title="given-names">ME</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Jansen</span> <span class="given-names" tagx="given-names" title="given-names">RK</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Palmer</span> <span class="given-names" tagx="given-names" title="given-names">JD</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Downie</span> <span class="given-names" tagx="given-names" title="given-names">SR</span> </span> </span> <div class="article-title" title="article-title">
The highly rearranged chloroplast genome of <i>Trachelium caeruleum </i>(Campanulaceae): Multiple inversions, inverted repeat expansion and contraction, transposition, insertions/deletions, and several repeat families </div> <span class="source" tagx="source" title="source">Curr Genet</span> <span class="year" tagx="year" title="year">1997</span> <span class="volume" tagx="volume" title="volume">31</span> <span class="fpage" tagx="fpage" title="fpage">419</span> <span class="lpage" tagx="lpage" title="lpage">429</span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/9162114">9162114</a> </span> <span class="pub-id"> <a href="https://doi.org/10.1007/s002940050225">10.1007/s002940050225</a> </span> </div> </li> <li tag="ref"> <a name="B63" /> <div class="citation" tagx="citation" title="citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Maul</span> <span class="given-names" tagx="given-names" title="given-names">JE</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Lilly</span> <span class="given-names" tagx="given-names" title="given-names">JW</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Cui</span> <span class="given-names" tagx="given-names" title="given-names">L</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">dePamphilis</span> <span class="given-names" tagx="given-names" title="given-names">CW</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Miller</span> <span class="given-names" tagx="given-names" title="given-names">W</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Harris</span> <span class="given-names" tagx="given-names" title="given-names">EH</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Stern</span> <span class="given-names" tagx="given-names" title="given-names">DB</span> </span> </span> <div class="article-title" title="article-title">
The <i>Chlamydomonas reinhardtii </i>plastid chromosome: Islands of genes in a sea of repeats </div> <span class="source" tagx="source" title="source">Plant Cell</span> <span class="year" tagx="year" title="year">2002</span> <span class="volume" tagx="volume" title="volume">14</span> <span class="fpage" tagx="fpage" title="fpage">2659</span> <span class="lpage" tagx="lpage" title="lpage">2679</span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/12417694">12417694</a> </span> <span class="pub-id"> <a href="https://doi.org/10.1105/tpc.006155">10.1105/tpc.006155</a> </span> </div> </li> <li tag="ref"> <a name="B64" /> <div class="citation" tagx="citation" title="citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Pombert</span> <span class="given-names" tagx="given-names" title="given-names">J-F</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Otis</span> <span class="given-names" tagx="given-names" title="given-names">C</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Lemieux</span> <span class="given-names" tagx="given-names" title="given-names">C</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Turmel</span> <span class="given-names" tagx="given-names" title="given-names">M</span> </span> </span> <div class="article-title" title="article-title">
The chloroplast genome sequence of the green alga <i>Pseudendoclonium akinetum </i>(Ulvophyceae) reveals unusual structural features and new insights into the branching order of chlorophyte lineages </div> <span class="source" tagx="source" title="source">Mol Biol Evol</span> <span class="year" tagx="year" title="year">2005</span> <span class="volume" tagx="volume" title="volume">22</span> <span class="fpage" tagx="fpage" title="fpage">1903</span> <span class="lpage" tagx="lpage" title="lpage">1918</span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/15930151">15930151</a> </span> <span class="pub-id"> <a href="https://doi.org/10.1093/molbev/msi182">10.1093/molbev/msi182</a> </span> </div> </li> <li tag="ref"> <a name="B65" /> <div class="citation" tagx="citation" title="citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Saski</span> <span class="given-names" tagx="given-names" title="given-names">C</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Lee</span> <span class="given-names" tagx="given-names" title="given-names">S</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Daniell</span> <span class="given-names" tagx="given-names" title="given-names">H</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Wood</span> <span class="given-names" tagx="given-names" title="given-names">T</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Tomkins</span> <span class="given-names" tagx="given-names" title="given-names">J</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Kim</span> <span class="given-names" tagx="given-names" title="given-names">H-G</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Jansen</span> <span class="given-names" tagx="given-names" title="given-names">RK</span> </span> </span> <div class="article-title" title="article-title">
Complete chloroplast genome sequence of <i>Glycine max </i>and comparative analyses with other legume genomes </div> <span class="source" tagx="source" title="source">Plt Mol Biol</span> <span class="year" tagx="year" title="year">2005</span> <span class="volume" tagx="volume" title="volume">59</span> <span class="fpage" tagx="fpage" title="fpage">309</span> <span class="lpage" tagx="lpage" title="lpage">322</span> <span class="pub-id"> <a href="https://doi.org/10.1007/s11103-005-8882-0">10.1007/s11103-005-8882-0</a> </span> </div> </li> <li tag="ref"> <a name="B66" /> <div class="citation" tagx="citation" title="citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Howe</span> <span class="given-names" tagx="given-names" title="given-names">CJ</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Barbrook</span> <span class="given-names" tagx="given-names" title="given-names">AC</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Koumandou</span> <span class="given-names" tagx="given-names" title="given-names">VL</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Nisbet</span> <span class="given-names" tagx="given-names" title="given-names">RER</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Symington</span> <span class="given-names" tagx="given-names" title="given-names">HA</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Wightman</span> <span class="given-names" tagx="given-names" title="given-names">TF</span> </span> </span> <div class="article-title" title="article-title">
Evolution of the chloroplast genome</div> <span class="source" tagx="source" title="source">Phil Trans R Soc Lond B</span> <span class="year" tagx="year" title="year">2003</span> <span class="volume" tagx="volume" title="volume">358</span> <span class="fpage" tagx="fpage" title="fpage">99</span> <span class="lpage" tagx="lpage" title="lpage">107</span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/12594920">12594920</a> </span> <span class="pub-id"> <a href="https://doi.org/10.1098/rstb.2002.1176">10.1098/rstb.2002.1176</a> </span> </div> </li> <li tag="ref"> <a name="B67" /> <div class="citation" tagx="citation" title="citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Palmer</span> <span class="given-names" tagx="given-names" title="given-names">JD</span> </span> </span> <div class="article-title" title="article-title">
Isolation and structural analysis of chloroplast DNA</div> <span class="source" tagx="source" title="source">Meth Enzymol</span> <span class="year" tagx="year" title="year">1986</span> <span class="volume" tagx="volume" title="volume">118</span> <span class="fpage" tagx="fpage" title="fpage">167</span> <span class="lpage" tagx="lpage" title="lpage">186</span> </div> </li> <li tag="ref"> <a name="B68" /> <div class="citation" tagx="citation" title="citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Bookjans</span> <span class="given-names" tagx="given-names" title="given-names">G</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Stummann</span> <span class="given-names" tagx="given-names" title="given-names">BM</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Henningse</span> <span class="given-names" tagx="given-names" title="given-names">KW</span> </span> </span> <div class="article-title" title="article-title">
Preparation of chloroplast DNA from pea plastids isolated in a medium of high ionic-strength</div> <span class="source" tagx="source" title="source">Anal Biochem</span> <span class="year" tagx="year" title="year">1984</span> <span class="volume" tagx="volume" title="volume">141</span> <span class="fpage" tagx="fpage" title="fpage">244</span> <span class="lpage" tagx="lpage" title="lpage">247</span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/6496932">6496932</a> </span> <span class="pub-id"> <a href="https://doi.org/10.1016/0003-2697(84)90452-4">10.1016/0003-2697(84)90452-4</a> </span> </div> </li> <li tag="ref"> <a name="B69" /> <div class="citation" tagx="citation" title="citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Jansen</span> <span class="given-names" tagx="given-names" title="given-names">RK</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Raubeson</span> <span class="given-names" tagx="given-names" title="given-names">LA</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Boore</span> <span class="given-names" tagx="given-names" title="given-names">JL</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">dePamphilis</span> <span class="given-names" tagx="given-names" title="given-names">CW</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Chumley</span> <span class="given-names" tagx="given-names" title="given-names">TW</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Haberle</span> <span class="given-names" tagx="given-names" title="given-names">RC</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Wyman</span> <span class="given-names" tagx="given-names" title="given-names">SK</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Alverson</span> <span class="given-names" tagx="given-names" title="given-names">AJ</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Peery</span> <span class="given-names" tagx="given-names" title="given-names">R</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Herman</span> <span class="given-names" tagx="given-names" title="given-names">SJ</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Fourcade</span> <span class="given-names" tagx="given-names" title="given-names">HM</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Kuehl</span> <span class="given-names" tagx="given-names" title="given-names">JV</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">McNeal</span> <span class="given-names" tagx="given-names" title="given-names">JR</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Leebens-Mack</span> <span class="given-names" tagx="given-names" title="given-names">J</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Cui</span> <span class="given-names" tagx="given-names" title="given-names">L</span> </span> </span> <div class="article-title" title="article-title">
Methods for obtaining and analyzing chloroplast genome sequences</div> <span class="source" tagx="source" title="source">Meth Enzym</span> <span class="year" tagx="year" title="year">2005</span> <span class="volume" tagx="volume" title="volume">395</span> <span class="fpage" tagx="fpage" title="fpage">348</span> <span class="lpage" tagx="lpage" title="lpage">384</span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/15865976">15865976</a> </span> <span class="pub-id"> <a href="https://doi.org/10.1016/S0076-6879(05)95020-9">10.1016/S0076-6879(05)95020-9</a> </span> </div> </li> <li tag="ref"> <a name="B70" /> <div class="citation" tagx="citation" title="citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Ewing</span> <span class="given-names" tagx="given-names" title="given-names">B</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Hillier</span> <span class="given-names" tagx="given-names" title="given-names">L</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Wendl</span> <span class="given-names" tagx="given-names" title="given-names">MC</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Green</span> <span class="given-names" tagx="given-names" title="given-names">P</span> </span> </span> <div class="article-title" title="article-title">
Base-calling of automated sequencer traces using phred. I. Accuracy assessment</div> <span class="source" tagx="source" title="source">Genome Res</span> <span class="year" tagx="year" title="year">1998</span> <span class="volume" tagx="volume" title="volume">8</span> <span class="fpage" tagx="fpage" title="fpage">175</span> <span class="lpage" tagx="lpage" title="lpage">185</span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/9521921">9521921</a> </span> </div> </li> <li tag="ref"> <a name="B71" /> <div class="citation" tagx="citation" title="citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Ewing</span> <span class="given-names" tagx="given-names" title="given-names">B</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Green</span> <span class="given-names" tagx="given-names" title="given-names">P</span> </span> </span> <div class="article-title" title="article-title">
Base-calling of automated sequencer traces using phred. II Error probabilities</div> <span class="source" tagx="source" title="source">Genome Res</span> <span class="year" tagx="year" title="year">1998</span> <span class="volume" tagx="volume" title="volume">8</span> <span class="fpage" tagx="fpage" title="fpage">186</span> <span class="lpage" tagx="lpage" title="lpage">194</span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/9521922">9521922</a> </span> </div> </li> <li tag="ref"> <a name="B72" /> <div class="citation" tagx="citation" title="citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Gordon</span> <span class="given-names" tagx="given-names" title="given-names">D</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Abajian</span> <span class="given-names" tagx="given-names" title="given-names">C</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Green</span> <span class="given-names" tagx="given-names" title="given-names">P</span> </span> </span> <div class="article-title" title="article-title"> <i>Consed </i>: A graphical tool for sequence finishing </div> <span class="source" tagx="source" title="source">Genome Res</span> <span class="year" tagx="year" title="year">1998</span> <span class="volume" tagx="volume" title="volume">8</span> <span class="fpage" tagx="fpage" title="fpage">195</span> <span class="lpage" tagx="lpage" title="lpage">202</span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/9521923">9521923</a> </span> </div> </li> <li tag="ref"> <a name="B73" /> <div class="citation" tagx="citation" title="citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Wyman</span> <span class="given-names" tagx="given-names" title="given-names">SK</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Boore</span> <span class="given-names" tagx="given-names" title="given-names">JL</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Jansen</span> <span class="given-names" tagx="given-names" title="given-names">RK</span> </span> </span> <div class="article-title" title="article-title">
Automatic annotation of organellar genomes with DOGMA</div> <span class="source" tagx="source" title="source">Bioinformatics</span> <span class="year" tagx="year" title="year">2004</span> <span class="volume" tagx="volume" title="volume">20</span> <span class="fpage" tagx="fpage" title="fpage">3252</span> <span class="lpage" tagx="lpage" title="lpage">3255</span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/15180927">15180927</a> </span> <span class="pub-id"> <a href="https://doi.org/10.1093/bioinformatics/bth352">10.1093/bioinformatics/bth352</a> </span> </div> </li> <li tag="ref"> <a name="B74" /> <div class="citation" tagx="citation" title="citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Altschul</span> <span class="given-names" tagx="given-names" title="given-names">SF</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Gish</span> <span class="given-names" tagx="given-names" title="given-names">W</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Miller</span> <span class="given-names" tagx="given-names" title="given-names">W</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Myers</span> <span class="given-names" tagx="given-names" title="given-names">EW</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Lipman</span> <span class="given-names" tagx="given-names" title="given-names">DJ</span> </span> </span> <div class="article-title" title="article-title">
Basic local alignment search tool</div> <span class="source" tagx="source" title="source">J Mol Biol</span> <span class="year" tagx="year" title="year">1990</span> <span class="volume" tagx="volume" title="volume">215</span> <span class="fpage" tagx="fpage" title="fpage">403</span> <span class="lpage" tagx="lpage" title="lpage">10</span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/2231712">2231712</a> </span> </div> </li> <li tag="ref"> <a name="B75" /> <div class="citation" tagx="citation" title="citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Higgins</span> <span class="given-names" tagx="given-names" title="given-names">DG</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Thompson</span> <span class="given-names" tagx="given-names" title="given-names">JD</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Gibson</span> <span class="given-names" tagx="given-names" title="given-names">TJ</span> </span> </span> <div class="article-title" title="article-title">
Using CLUSTAL for multiple sequence alignments</div> <span class="source" tagx="source" title="source">Meth Enzy</span> <span class="year" tagx="year" title="year">1996</span> <span class="volume" tagx="volume" title="volume">266</span> <span class="fpage" tagx="fpage" title="fpage">383</span> <span class="lpage" tagx="lpage" title="lpage">402</span> </div> </li> <li tag="ref"> <a name="B76" /> <div class="citation" tagx="citation" title="citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Tatusova</span> <span class="given-names" tagx="given-names" title="given-names">TA</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Madden</span> <span class="given-names" tagx="given-names" title="given-names">TL</span> </span> </span> <div class="article-title" title="article-title">
Blast 2 sequences – a new tool for comparing protein and nucleotide sequences</div> <span class="source" tagx="source" title="source">FEMS Microbiol Lett</span> <span class="year" tagx="year" title="year">1999</span> <span class="volume" tagx="volume" title="volume">174</span> <span class="fpage" tagx="fpage" title="fpage">247</span> <span class="lpage" tagx="lpage" title="lpage">250</span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/10339815">10339815</a> </span> <span class="pub-id"> <a href="https://doi.org/10.1111/j.1574-6968.1999.tb13575.x">10.1111/j.1574-6968.1999.tb13575.x</a> </span> </div> </li> <li tag="ref"> <a name="B77" /> <div class="citation" tagx="citation" title="citation"> <div class="article-title" title="article-title">
mfold</div> <a href="http://www.bioinfo.rpi.edu/applications/mfold/" /> </div> </li> <li tag="ref"> <a name="B78" /> <div class="citation" tagx="citation" title="citation"> <div class="article-title" title="article-title">
DINAmelt</div> <a href="http://www.bioinfo.rpi.edu/applications/hybrid/twostate-fold.php" /> </div> </li> <li tag="ref"> <a name="B79" /> <div class="citation" tagx="citation" title="citation"> <div class="article-title" title="article-title">
BABEL</div> <a href="http://bioinfo.hku.hk/services/analyseq/cgi-bin/freqsq_in.pl" /> <span class="comment">no longer available as of July 2006</span> </div> </li> <li tag="ref"> <a name="B80" /> <div class="citation" tagx="citation" title="citation"> <div class="article-title" title="article-title">
Simple Sequence Repeat (SSR) Extractor Utility</div> <a href="http://www.aridolan.com/ssr/ssr.aspx" /> </div> </li> <li tag="ref"> <a name="B81" /> <div class="citation" tagx="citation" title="citation"> <div class="article-title" title="article-title">
Comparative Repeat Analysis</div> <a href="http://bugmaster.jgi-psf.org/repeats/" /> </div> </li> <li tag="ref"> <a name="B82" /> <div class="citation" tagx="citation" title="citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Kurtz</span> <span class="given-names" tagx="given-names" title="given-names">S</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Choudhuri</span> <span class="given-names" tagx="given-names" title="given-names">JV</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Ohlebusch</span> <span class="given-names" tagx="given-names" title="given-names">E</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Schleiermacher</span> <span class="given-names" tagx="given-names" title="given-names">C</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Stoye</span> <span class="given-names" tagx="given-names" title="given-names">J</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Giegerich</span> <span class="given-names" tagx="given-names" title="given-names">R</span> </span> </span> <div class="article-title" title="article-title">
REPuter: the manifold applications of repeat analysis on a genomic scale</div> <span class="source" tagx="source" title="source">Nucl Acids Res</span> <span class="year" tagx="year" title="year">2001</span> <span class="volume" tagx="volume" title="volume">29</span> <span class="fpage" tagx="fpage" title="fpage">4633</span> <span class="lpage" tagx="lpage" title="lpage">4642</span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/11713313">11713313</a> </span> <span class="pub-id"> <a href="https://doi.org/10.1093/nar/29.22.4633">10.1093/nar/29.22.4633</a> </span> </div> </li> </ul> </div> </div>  </body></html>