<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<link rel="dns-prefetch" href="//d33xdlntwy0kbs.cloudfront.net" />
<link rel="shortcut icon" href="https://www.biorxiv.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<meta name="type" content="article" />
<meta name="category" content="article" />
<meta name="HW.identifier" content="/biorxiv/early/2016/09/19/075960.atom" />
<meta name="HW.pisa" content="biorxiv;075960v1" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="Correlations of genotype with climate parameters suggest Caenorhabditis elegans niche preferences" />
<meta name="DC.Identifier" content="10.1101/075960" />
<meta name="DC.Date" content="2016-09-19" />
<meta name="DC.Publisher" content="Cold Spring Harbor Laboratory" />
<meta name="DC.Rights" content="© 2016, Posted by Cold Spring Harbor Laboratory. This pre-print is available under a Creative Commons License (Attribution-NonCommercial 4.0 International), CC BY-NC 4.0, as described at http://creativecommons.org/licenses/by-nc/4.0/" />
<meta name="DC.AccessRights" content="restricted" />
<meta name="DC.Description" content="Species inhabit a variety of environmental niches, and the adaptation to a particular niche is often controlled by genetic factors, including gene-by-environment interactions. The genes that vary in order to regulate the ability to colonize a niche are often difficult to identify, especially in the context of complex ecological systems and in experimentally uncontrolled natural environments. Quantitative genetic approaches provide an opportunity to investigate correlations between genetic factors and environmental parameters that might define a niche. Previously, we have shown how a collection of 208 whole-genome sequenced wild Caenorhabditis elegans can facilitate association mapping approaches. To correlate climate parameters with the variation found in this collection of wild strains, we used geographic data to exhaustively curate daily weather measurements in short-term (three month), middle-term (one year), and long-term (three year) durations surrounding the data of strain isolation. These climate parameters were then used as quantitative traits in the mapping approaches. We identified 10 QTL underlying variation in three traits: elevation, relative humidity, and average temperature. We then performed statistical analyses to further narrow the genomic interval of interest to identify gene candidates with variants potentially underlying phenotypic differences. Additionally, we performed two-strain competition assays at high and low temperatures to validate a QTL for temperature preference and found suggestive evidence that genotypes might be adapted to particular temperatures.

100-word summary for G3 Quantitative genetic approaches provide an opportunity to investigate correlations between genetic factors and environmental parameters that might define a niche, but these genes are difficult to identify, especially in the context of complex ecological systems. Here, we used a collection of 152 sequenced wild Caenorhabditis elegans to correlate climate parameters with the variation found in this collection of wild strains. We identified 10 QTL in five traits, including elevation, relative humidity, and temperature. Additionally, we performed competition assays to validate a QTL for temperature preference and found suggestive evidence that genotypes might be adapted to particular temperatures." />
<meta name="DC.Contributor" content="Kathryn S. Evans" />
<meta name="DC.Contributor" content="Yuehui Zhao" />
<meta name="DC.Contributor" content="Shannon C. Brady" />
<meta name="DC.Contributor" content="Lijiang Long" />
<meta name="DC.Contributor" content="Patrick T. McGrath" />
<meta name="DC.Contributor" content="Erik C. Andersen" />
<meta name="article:published_time" content="2016-09-19" />
<meta name="article:section" content="New Results" />
<meta name="citation_title" content="Correlations of genotype with climate parameters suggest Caenorhabditis elegans niche preferences" />
<meta name="citation_abstract" lang="en" content="&lt;h3&gt;Abstract&lt;/h3&gt;
&lt;p&gt;Species inhabit a variety of environmental niches, and the adaptation to a particular niche is often controlled by genetic factors, including gene-by-environment interactions. The genes that vary in order to regulate the ability to colonize a niche are often difficult to identify, especially in the context of complex ecological systems and in experimentally uncontrolled natural environments. Quantitative genetic approaches provide an opportunity to investigate correlations between genetic factors and environmental parameters that might define a niche. Previously, we have shown how a collection of 208 whole-genome sequenced wild &lt;i&gt;Caenorhabditis elegans&lt;/i&gt; can facilitate association mapping approaches. To correlate climate parameters with the variation found in this collection of wild strains, we used geographic data to exhaustively curate daily weather measurements in short-term (three month), middle-term (one year), and long-term (three year) durations surrounding the data of strain isolation. These climate parameters were then used as quantitative traits in the mapping approaches. We identified 10 QTL underlying variation in three traits: elevation, relative humidity, and average temperature. We then performed statistical analyses to further narrow the genomic interval of interest to identify gene candidates with variants potentially underlying phenotypic differences. Additionally, we performed two-strain competition assays at high and low temperatures to validate a QTL for temperature preference and found suggestive evidence that genotypes might be adapted to particular temperatures.&lt;/p&gt;&lt;h3&gt;100-word summary for &lt;i&gt;G3&lt;/i&gt;&lt;/h3&gt;
&lt;p&gt;Quantitative genetic approaches provide an opportunity to investigate correlations between genetic factors and environmental parameters that might define a niche, but these genes are difficult to identify, especially in the context of complex ecological systems. Here, we used a collection of 152 sequenced wild &lt;i&gt;Caenorhabditis elegans&lt;/i&gt; to correlate climate parameters with the variation found in this collection of wild strains. We identified 10 QTL in five traits, including elevation, relative humidity, and temperature. Additionally, we performed competition assays to validate a QTL for temperature preference and found suggestive evidence that genotypes might be adapted to particular temperatures.&lt;/p&gt;" />
<meta name="citation_journal_title" content="bioRxiv" />
<meta name="citation_publisher" content="Cold Spring Harbor Laboratory" />
<meta name="citation_publication_date" content="2016/01/01" />
<meta name="citation_mjid" content="biorxiv;075960v1" />
<meta name="citation_id" content="075960v1" />
<meta name="citation_public_url" content="https://www.biorxiv.org/content/10.1101/075960v1" />
<meta name="citation_abstract_html_url" content="https://www.biorxiv.org/content/10.1101/075960v1.abstract" />
<meta name="citation_full_html_url" content="https://www.biorxiv.org/content/10.1101/075960v1.full" />
<meta name="citation_pdf_url" content="https://www.biorxiv.org/content/biorxiv/early/2016/09/19/075960.full.pdf" />
<meta name="citation_doi" content="10.1101/075960" />
<meta name="citation_num_pages" content="34" />
<meta name="citation_article_type" content="Article" />
<meta name="citation_section" content="New Results" />
<meta name="citation_firstpage" content="075960" />
<meta name="citation_author" content="Kathryn S. Evans" />
<meta name="citation_author_institution" content="Department of Molecular Biosciences, Northwestern University" />
<meta name="citation_author_institution" content="Interdisciplinary Biological Sciences Graduate Program, Northwestern University" />
<meta name="citation_author" content="Yuehui Zhao" />
<meta name="citation_author_institution" content="School of Biology, Georgia Institute of Technology" />
<meta name="citation_author" content="Shannon C. Brady" />
<meta name="citation_author_institution" content="Department of Molecular Biosciences, Northwestern University" />
<meta name="citation_author_institution" content="Interdisciplinary Biological Sciences Graduate Program, Northwestern University" />
<meta name="citation_author" content="Lijiang Long" />
<meta name="citation_author_institution" content="School of Biology, Georgia Institute of Technology" />
<meta name="citation_author" content="Patrick T. McGrath" />
<meta name="citation_author_institution" content="School of Biology, Georgia Institute of Technology" />
<meta name="citation_author" content="Erik C. Andersen" />
<meta name="citation_author_institution" content="Department of Molecular Biosciences, Northwestern University" />
<meta name="citation_author_institution" content="Chemistry of Life Processes Institute, Northwestern University" />
<meta name="citation_author_institution" content="Northwestern Institute on Complex Systems, Northwestern University" />
<meta name="citation_author_email" content="erik.andersen@northwestern.edu" />
<meta name="citation_reference" content="Andersen, Erik C., Justin P. Gerke, Joshua A. Shapiro, Jonathan R. Crissman, Rajarshi Ghosh, Joshua S. Bloom, Marie-Anne Félix, and Leonid Kruglyak. 2012. “Chromosome-Scale Selective Sweeps Shape Caenorhabditis Elegans Genomic Diversity.” Nature Genetics 44 (3): 285–90." />
<meta name="citation_reference" content="Bozinovic, Francisco, Nadia R. Medina, José M. Alruiz, Grisel Cavieres, and Pablo Sabat. 2016. “Thermal Tolerance and Survival Responses to Scenarios of Experimental Climatic Change: Changing Thermal Variability Reduces the Heat and Cold Tolerance in a Fly.” Journal of Comparative Physiology. B, Biochemical, Systemic, and Environmental Physiology 186 (5): 581–87." />
<meta name="citation_reference" content="Cingolani, Pablo, Adrian Platts, Le Lily Wang, Melissa Coon, Tung Nguyen, Luan Wang, Susan J. Land, Xiangyi Lu, and Douglas M. Ruden. 2012. “A Program for Annotating and Predicting the Effects of Single Nucleotide Polymorphisms, SnpEff: SNPs in the Genome of Drosophila Melanogaster Strain w1118; Iso-2; Iso-3.” Fly 6 (2): 80–92." />
<meta name="citation_reference" content="Cook, Daniel E., Stefan Zdraljevic, Robyn E. Tanny, Beomseok Seo, David D. Riccardi, Luke M. Noble, Matthew V. Rockman, et al. 2016. “The Genetic Basis of Natural Variation in Caenorhabditis Elegans Telomere Length.” Genetics, July. doi:10.1534/genetics.116.191148." />
<meta name="citation_reference" content="Endelman, J. B. 2011. “Ridge Regression and Other Kernels for Genomic Selection with R Package rrBLUP.” The Plant Genome. dl.sciencesocieties.org. https://dl.sciencesocieties.org/publications/tpg/abstracts/4/3/250." />
<meta name="citation_reference" content="Félix, Marie-Anne, and Christian Braendle. 2010. “The Natural History of Caenorhabditis Elegans.” Current Biology: CB 20 (22): R965–69." />
<meta name="citation_reference" content="Frézal, Lise, and Marie-Anne Félix. 2015. “C. Elegans Outside the Petri Dish.” eLife 4 (March). doi:10.7554/eLife.05849." />
<meta name="citation_reference" content="Gerken, Alison R., Olivia C. Eller, Daniel A. Hahn, and Theodore J. Morgan. 2015. “Constraints, Independence, and Evolution of Thermal Plasticity: Probing Genetic Architecture of Long- and Short-Term Thermal Acclimation.” Proceedings of the National Academy of Sciences of the United States of America 112 (14): 4399–4404." />
<meta name="citation_reference" content="Hangartner, Sandra B., Ary A. Hoffmann, Ailie Smith, and Philippa C. Griffin. 2015. “A Collection of Australian Drosophila Datasets on Climate Adaptation and Species Distributions.” Scientific Data 2 (November): 150067." />
<meta name="citation_reference" content="Hijmans, R. J., E. Williams, and C. Vennes. 2012. “Geosphere: Spherical Trigonometry. R Package Version 1.2–28.” CRAN. R-Project. Org/package= Geosphere." />
<meta name="citation_reference" content="Hirschhorn, Joel N., and Mark J. Daly. 2005. “Genome-Wide Association Studies for Common Diseases and Complex Traits.” Nature Reviews. Genetics 6 (2): 95–108." />
<meta name="citation_reference" content="Hodgkin, J., and T. Doniach. 1997. “Natural Variation and Copulatory Plug Formation in Caenorhabditis Elegans.” Genetics 146 (1): 149–64." />
<meta name="citation_reference" content="Hutchinson, G. E. 1957. “Cold Spring Harbor Symposium on Quantitative Biology.” Concluding Remarks." />
<meta name="citation_reference" content="Iannone, Richard. 2015. “stationaRy: Get Hourly Meteorological Data from Global Stations. R Package Version 0.4.1.” CRAN. R-Project. Org/package= stationaRy, October. Comprehensive R Archive Network (CRAN)." />
<meta name="citation_reference" content="“Integrated Surface Global Hourly Data - NOAA Data Catalog.” 2016. Accessed September 2. https://data.noaa.gov/dataset/integrated-surface-global-hourly-data." />
<meta name="citation_reference" content="Kang, Hyun Min, Noah A. Zaitlen, Claire M. Wade, Andrew Kirby, David Heckerman, Mark J. Daly, and Eleazar Eskin. 2008. “Efficient Control of Population Structure in Model Organism Association Mapping.” Genetics 178 (3): 1709–23." />
<meta name="citation_reference" content="Li, Heng, and Richard Durbin. 2009. “Fast and Accurate Short Read Alignment with Burrows–Wheeler Transform.” Bioinformatics 25 (14): 1754–60." />
<meta name="citation_reference" content="Li, Heng, Bob Handsaker, Alec Wysoker, Tim Fennell, Jue Ruan, Nils Homer, Gabor Marth, Goncalo Abecasis, Richard Durbin, and 1000 Genome Project Data Processing Subgroup. 2009. “The Sequence Alignment/Map Format and SAMtools.” Bioinformatics 25 (16): 2078–79." />
<meta name="citation_reference" content="Machado, Heather E., Alan O. Bergland, Katherine R. O’ Brien, Emily L. Behrman, Paul S. Schmidt, and Dmitri A. Petrov. 2016. “Comparative Population Genomics of Latitudinal Variation in Drosophila Simulans and Drosophila Melanogaster.” Molecular Ecology 25 (3): 723–40." />
<meta name="citation_reference" content="Saltz, Julia B., and Sergey V. Nuzhdin. 2014. “Genetic Variation in Niche Construction: Implications for Development and Evolutionary Genetics.” Trends in Ecology &amp; Evolution 29 (1): 8–14." />
<meta name="citation_reference" content="Sterken, Mark G., L. Basten Snoek, Jan E. Kammenga, and Erik C. Andersen. 2015. “The Laboratory Domestication of Caenorhabditis Elegans.” Trends in Genetics: TIG 31 (5): 224–31." />
<meta name="citation_reference" content="Taylor, Michelle L., Alison Skeats, Alastair J. Wilson, Tom A. R. Price, and Nina Wedell. 2015. “Opposite Environmental and Genetic Influences on Body Size in North American Drosophila Pseudoobscura.” BMC Evolutionary Biology 15 (March): 51." />
<meta name="citation_reference" content="Tyukmaeva, Venera I., Paris Veltsos, Jon Slate, Emma Gregson, Hannele Kauranen, Maaria Kankare, Michael G. Ritchie, Roger K. Butlin, and Anneli Hoikkala. 2015. “Localization of Quantitative Trait Loci for Diapause and Other Photoperiodically Regulated Life History Traits Important in Adaptation to Seasonally Varying Environments.” Molecular Ecology 24 (11): 2809–19." />
<meta name="twitter:title" content="Correlations of genotype with climate parameters suggest Caenorhabditis elegans niche preferences" />
<meta name="twitter:site" content="@biorxivpreprint" />
<meta name="twitter:card" content="summary" />
<meta name="twitter:image" content="https://www.biorxiv.org/sites/default/files/images/biorxiv_logo_homepage7-5-small.png" />
<meta name="twitter:description" content="Species inhabit a variety of environmental niches, and the adaptation to a particular niche is often controlled by genetic factors, including gene-by-environment interactions. The genes that vary in order to regulate the ability to colonize a niche are often difficult to identify, especially in the context of complex ecological systems and in experimentally uncontrolled natural environments. Quantitative genetic approaches provide an opportunity to investigate correlations between genetic factors and environmental parameters that might define a niche. Previously, we have shown how a collection of 208 whole-genome sequenced wild Caenorhabditis elegans can facilitate association mapping approaches. To correlate climate parameters with the variation found in this collection of wild strains, we used geographic data to exhaustively curate daily weather measurements in short-term (three month), middle-term (one year), and long-term (three year) durations surrounding the data of strain isolation. These climate parameters were then used as quantitative traits in the mapping approaches. We identified 10 QTL underlying variation in three traits: elevation, relative humidity, and average temperature. We then performed statistical analyses to further narrow the genomic interval of interest to identify gene candidates with variants potentially underlying phenotypic differences. Additionally, we performed two-strain competition assays at high and low temperatures to validate a QTL for temperature preference and found suggestive evidence that genotypes might be adapted to particular temperatures.

100-word summary for G3 Quantitative genetic approaches provide an opportunity to investigate correlations between genetic factors and environmental parameters that might define a niche, but these genes are difficult to identify, especially in the context of complex ecological systems. Here, we used a collection of 152 sequenced wild Caenorhabditis elegans to correlate climate parameters with the variation found in this collection of wild strains. We identified 10 QTL in five traits, including elevation, relative humidity, and temperature. Additionally, we performed competition assays to validate a QTL for temperature preference and found suggestive evidence that genotypes might be adapted to particular temperatures." />
<meta name="og-title" property="og:title" content="Correlations of genotype with climate parameters suggest Caenorhabditis elegans niche preferences" />
<meta name="og-url" property="og:url" content="https://www.biorxiv.org/content/10.1101/075960v1" />
<meta name="og-site-name" property="og:site_name" content="bioRxiv" />
<meta name="og-description" property="og:description" content="Species inhabit a variety of environmental niches, and the adaptation to a particular niche is often controlled by genetic factors, including gene-by-environment interactions. The genes that vary in order to regulate the ability to colonize a niche are often difficult to identify, especially in the context of complex ecological systems and in experimentally uncontrolled natural environments. Quantitative genetic approaches provide an opportunity to investigate correlations between genetic factors and environmental parameters that might define a niche. Previously, we have shown how a collection of 208 whole-genome sequenced wild Caenorhabditis elegans can facilitate association mapping approaches. To correlate climate parameters with the variation found in this collection of wild strains, we used geographic data to exhaustively curate daily weather measurements in short-term (three month), middle-term (one year), and long-term (three year) durations surrounding the data of strain isolation. These climate parameters were then used as quantitative traits in the mapping approaches. We identified 10 QTL underlying variation in three traits: elevation, relative humidity, and average temperature. We then performed statistical analyses to further narrow the genomic interval of interest to identify gene candidates with variants potentially underlying phenotypic differences. Additionally, we performed two-strain competition assays at high and low temperatures to validate a QTL for temperature preference and found suggestive evidence that genotypes might be adapted to particular temperatures.

100-word summary for G3 Quantitative genetic approaches provide an opportunity to investigate correlations between genetic factors and environmental parameters that might define a niche, but these genes are difficult to identify, especially in the context of complex ecological systems. Here, we used a collection of 152 sequenced wild Caenorhabditis elegans to correlate climate parameters with the variation found in this collection of wild strains. We identified 10 QTL in five traits, including elevation, relative humidity, and temperature. Additionally, we performed competition assays to validate a QTL for temperature preference and found suggestive evidence that genotypes might be adapted to particular temperatures." />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.biorxiv.org/sites/default/files/images/biorxiv_logo_homepage7-5-small.png" />
<meta name="citation_date" content="2016-09-19" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/10.1101/075960v1.full.pdf" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/10.1101/075960v1.full.txt" />
<link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/10.1101/075960v1.ppt" />
<meta name="description" content="bioRxiv - the preprint server for biology, operated by Cold Spring Harbor Laboratory, a research and educational institution" />
<meta name="generator" content="Drupal 7 (http://drupal.org)" />
<link rel="canonical" href="https://www.biorxiv.org/content/10.1101/075960v1" />
<link rel="shortlink" href="https://www.biorxiv.org/node/20978" />
    <title>Correlations of genotype with climate parameters suggest Caenorhabditis elegans niche preferences | bioRxiv</title>  
    <link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__jMRAK66KMC1e4TQlwUNn3KiWVDC5AjueUAYEm1xBY_U__KSUjdT4jdcJ4qJz6fKY1K9WkYh1a5EcaWZxt_-zbTis__QJacVmveFyLIBSPkp21xL0ZvuDW9DMBrD2HB1P4Ry5I.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q5p1rr" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__icrP-Y0qw2NP73rJxxkUtrJta0z3VLAABtNjfSdEghA__rlN3rx0wOoswGNHZMeZM96bNn-3cLWHXTztGDoThNlw__QJacVmveFyLIBSPkp21xL0ZvuDW9DMBrD2HB1P4Ry5I.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__q48SDxVDWut_cH6dTPyQe7HjADuSLLmJVFztuiJEFrU__DJqddQGjm5dXCHsaiwNriGytKN5eevIQqyjdq65TZtE__QJacVmveFyLIBSPkp21xL0ZvuDW9DMBrD2HB1P4Ry5I.css" media="all" />
<style type="text/css" media="all">
/* <![CDATA[ */
#sliding-popup.sliding-popup-bottom,#sliding-popup.sliding-popup-bottom .eu-cookie-withdraw-banner,.eu-cookie-withdraw-tab{background:gray}#sliding-popup.sliding-popup-bottom.eu-cookie-withdraw-wrapper{background:transparent}#sliding-popup .popup-content #popup-text h1,#sliding-popup .popup-content #popup-text h2,#sliding-popup .popup-content #popup-text h3,#sliding-popup .popup-content #popup-text p,.eu-cookie-compliance-secondary-button,.eu-cookie-withdraw-tab{color:#fff !important}.eu-cookie-withdraw-tab{border-color:#fff}.eu-cookie-compliance-more-button{color:#fff !important}
/* ]]> */
</style>

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__QJacVmveFyLIBSPkp21xL0ZvuDW9DMBrD2HB1P4Ry5I.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__B6tGdsQSt1vT8d8j-19-oS4FKJe83RrSNBkjPs9jfos__6VkE5WcQmmQpSVTQx-xS2-zn_nlwFlDm-EN1HLJNjrA__QJacVmveFyLIBSPkp21xL0ZvuDW9DMBrD2HB1P4Ry5I.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__I6pBwZrmO51afTZVOjN83FdGH1QRb9t15lSLso0JVV4__F3sVKlGyHKCaAPI_WdiHD7l_-05B6b9bLdUwzi1me90__QJacVmveFyLIBSPkp21xL0ZvuDW9DMBrD2HB1P4Ry5I.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__QJacVmveFyLIBSPkp21xL0ZvuDW9DMBrD2HB1P4Ry5I.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__QJacVmveFyLIBSPkp21xL0ZvuDW9DMBrD2HB1P4Ry5I.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__CRdjIL0HKDL2T9slYvkTSTLypBBPjyoDhwBezBuDy7c__cHebPmPNdg2jyPmpyQpZWssJQ4cPrqfaabp5Or9KlkU__QJacVmveFyLIBSPkp21xL0ZvuDW9DMBrD2HB1P4Ry5I.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://d33xdlntwy0kbs.cloudfront.net/cshl_custom.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/all/libraries/modernizr/modernizr.min.js?q5p1rr"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__yOhrJUBmMpN-jsuQlJYmS8ovkQU6I6KInr295HrSTb0__c--zHhbuOWmkdWkJYXiNWDmkpDRPesmbBjGlQcyqBOY__QJacVmveFyLIBSPkp21xL0ZvuDW9DMBrD2HB1P4Ry5I.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var _prum=[['id', '52e95df3abe53d2e33000000'], ['mark', 'firstbyte',
      (new Date()).getTime()]]; (function() {
      var s=document.getElementsByTagName('script')[0],
      p=document.createElement('script');
      p.async='async'; p.src='//rum-static.pingdom.net/prum.min.js';s.parentNode.insertBefore(p,s);})();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js___KyvuEvfmewWxdOLBXTYsSUo8Pfm_DijV6fWtGvSL_k__HSKkTgIr7nDcdvQYewipf6v4ak6ZLUOQkwPj44wKouA__QJacVmveFyLIBSPkp21xL0ZvuDW9DMBrD2HB1P4Ry5I.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__3uCzzYE5jr6ly9nvsM8PRBRkhh4p0NWCMDja4EKBy4M__dmGGVYcB8LhnbR2iAmVW0ZyXI6QuFVYFRORa5XxglXs__QJacVmveFyLIBSPkp21xL0ZvuDW9DMBrD2HB1P4Ry5I.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__0zdUME0GLauYYyEch2tGPydIw-IG7Uizip0Qb4wUsig__3RgszrCgk4bOnTzo4vaQ-hrWzaG6oD19l-R9CQ1YQ-o__QJacVmveFyLIBSPkp21xL0ZvuDW9DMBrD2HB1P4Ry5I.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__W6wAjopaALytcBam7oT3WhEzllkWaB9R-8hazHyvShM__rK5YBIqBy9qt0di0Sn4DriBx9TJupdmv6uGmLJyZvDs__QJacVmveFyLIBSPkp21xL0ZvuDW9DMBrD2HB1P4Ry5I.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-45638026-1", {"cookieDomain":"auto"});ga("set", "page", location.pathname + location.search + location.hash);ga("send", "pageview");ga('create', 'UA-189672-38', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__1PtAOZ8x7l6QAi8D7X6Kj_Y7HcUOFzvWRvvYHUcCT98__U8IB92u_CnnGwJ-X_xaUa9lzsC8OzUfN0WkK2JQfirg__QJacVmveFyLIBSPkp21xL0ZvuDW9DMBrD2HB1P4Ry5I.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings,{"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"jsbJiuQCAF3a4EGe5TplTO-FdBHKReG901W8QCbjm_M","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"modules\/comment\/comment.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"modules\/forum\/forum.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"sites\/default\/modules\/biorxiv\/css\/biorxiv.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/flexible\/flexible.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_collection\/css\/highwire_subject_collection.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire.highwire-list-expand.css":1,"public:\/\/ctools\/css\/bab59090150caf0c64e27afa1b36eef1.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/contrib\/eu_cookie_compliance\/css\/eu_cookie_compliance.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"0":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_biorxiv_styles\/css\/biorxiv-styles.css.less":1,"https:\/\/d33xdlntwy0kbs.cloudfront.net\/cshl_custom.css":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_figures.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_tables.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"http:\/\/d33xdlntwy0kbs.cloudfront.net\/cshl_custom.js":1,"sites\/all\/modules\/contrib\/eu_cookie_compliance\/js\/eu_cookie_compliance.js":1,"sites\/default\/modules\/jnl_biorxiv_styles\/js\/adv-search-collection-toggle.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/eu_cookie_compliance\/js\/jquery.cookie-1.4.1.min.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/default\/modules\/jnl_biorxiv_styles\/js\/biorxiv-scripts.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_nps_survey\/js\/nps_survey.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"sites\/default\/modules\/contrib\/disqus\/disqus.js":1,"misc\/textarea.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/contrib\/service_links\/js\/twitter_button.js":1,"sites\/all\/modules\/contrib\/service_links\/js\/facebook_like.js":1,"sites\/all\/modules\/contrib\/service_links\/js\/google_plus_one.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.highwire-list-expand.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"20978","apath":"\/biorxiv\/early\/2016\/09\/19\/075960.atom","pisa":"biorxiv;075960v1","processed":["highwire_math"],"markup":[{"requested":"full-text","variant":"full-text","view":"full","pisa":"biorxiv;075960v1"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"disqus":{"domain":"biorxivstage","url":"https:\/\/www.biorxiv.org\/content\/10.1101\/075960v1","title":"Correlations of genotype with climate parameters suggest Caenorhabditis elegans niche preferences","identifier":"node\/20978"},"panels_ajax_pane":{"new-41":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022mlcv1fvwNUoXLLycoIeKKxEcza3pxwx+DIyOjva5t7oMxmZdWfUB\\\\\\\/2BjzLLa72OQamCQe1XK\\\\\\\/ljO\\\\\\\/5NYRx9p0fAZzQ3LBP7JOGtaLtWsc\\\\\\\/MthFwkBb65qsiDK6WcJOAxVVnJcCoWe1Qnnesjczz1fwmdeglNIpzKBKFlU61TMti+bTJg8L6F8PZ6JCPzu+DbC0HCRqlPHKAtzixLjG\\\\\\\/hDJg7UzKOYsAZz5sdSSZEMQxLEk4C67jRp0yzTkzdADhgLv2pIsvVJKFF7Vc7Kru7H6BqKD\\\\\\\/4S2I\\\\\\\/1zqt6aWxM42jmVMdW2InVkBNaSMQGr2HvPby54VXrJN1XhxHa8Z1a0vmxK6jRw9b5W6hbr2I1SKxIryH8IDvzVqbLRGtvpIr1gdSiU++1xdXgigXTmvNsan9IyFWw3XekLO7dif4Fv1ofsCOVUiW2c+5QPLpvqC\\\\\\\/J3R\\\\\\\/aFfqeqYm5mV6QGOgx+zksy1+pPRhP469J8enbfoWFdD3+EVrBtnsBzPHhD72aHigGJ3b3FBKBseo8HUQb811v2dDR9IPOiUTTVw+NnzRbCEpvSLTAobkDP1s24Fdgf22WzRWgsW0Fdmw8go6YmUPykpzePnkyYC8bfdEYwTPnm7y+u3M63CnZRRLlLU8iS2Y5aNZDTS2+sCpH8RRpCeYTn7Yj\\\\\\\/mQJEdA9RwuEvUExTnX0skI1PPqALQ46wofluWbURHUZ+XMUtsVRquFGjsvzcSTvPrs\\\\\\\/OjG5C0TcwP8MczMXUEq6NLLJDssFv5nI3y+knvQVJefNXM876HuoUH\\\\\\\/mPHvuxLECnqUnlklCNnbDC2m3Yf2F6MGFP\\\\\\\/uB1ldWB3nHXMyLCRAk\\\\\\\/VDxvWfkSDlo\\\\\\\/0ymx15nWq6bXFiMnqilwlIYRXlPC+iZ0Fp\\\\\\\/4XnG7KR1mbh8xHmvmbNqIAeJHJRlMfsINJX5761fTY\\\\\\\/8UPPs2ktNe2qdc\\\\\\\/A13ojNQKvRgAcn+fHWeFp7GEcAFjAgl6ATxwColdlD6VT83ZCKPBrF1qPPMeU2uQYJoH2bez6b5lvbhNdwAG9cf2G82\\\\\\\/XOqH\\\\\\\/55nLBM3dNtuTPDqrB4PwsEQuRNbzLX6bCYJM3m6dNXVzDZilMHZn8ZM11Syct\\\\\\\/w2vKblivKyrX4cgQI9N4kvYqOG5T9OZA2raIAc39CP2q2LX\\\\\\\/mOGlNjjs5p2cUgDhmA+ZAgG+6GjOdeAU8K7VcowPEaWM8ITRhRHm5Ux+rf2nvfzyMYvT03Pgj9uexSVc+yAuj1gF8t6so5yss+hFNiOAhVGngDYX7coEUvjpbP6frMvyF1ADuawdTSPezyfmDWGx0QnSAH5zo452WBNdM=\\\u0022,\\\u0022iv\\\u0022:\\\u0022rwuyPkINbA9H84kCp4LrfA==\\\u0022,\\\u0022salt\\\u0022:\\\u0022f9d8845712f72c9fee5db7d9ff111eab\\\u0022}\u0022,\u0022hmac\u0022:\u00229a61161e0a1cdcc76660bf9a85bc9442c1aac4b3d32f9ea7f61f49a0e87e8309\u0022}"},"urlIsAjaxTrusted":{"\/content\/10.1101\/075960v1.full":true},"ws_fl":{"width":100,"height":21},"ws_gpo":{"size":"","annotation":"","lang":"","callback":"","width":300},"color":{"logo":"https:\/\/www.biorxiv.org\/sites\/default\/files\/bioRxiv_article.jpg"},"highwire_list_expand":{"is_collapsed":"1"},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"eu_cookie_compliance":{"popup_enabled":1,"popup_agreed_enabled":0,"popup_hide_agreed":0,"popup_clicking_confirmation":1,"popup_scrolling_confirmation":false,"popup_html_info":"\u003Cdiv\u003E\n  \u003Cdiv class =\u0022popup-content info\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n      \u003Cp\u003EWe use cookies on this site to enhance your user experience. By clicking any link on this page you are giving your consent for us to set cookies.\u003C\/p\u003E\n    \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022agree-button eu-cookie-compliance-default-button\u0022\u003EContinue\u003C\/button\u003E\n                    \u003Cbutton type=\u0022button\u0022 class=\u0022find-more-button eu-cookie-compliance-more-button\u0022\u003EFind out more\u003C\/button\u003E\n          \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E","use_mobile_message":false,"mobile_popup_html_info":"\u003Cdiv\u003E\n  \u003Cdiv class =\u0022popup-content info\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n          \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022agree-button eu-cookie-compliance-default-button\u0022\u003EContinue\u003C\/button\u003E\n                    \u003Cbutton type=\u0022button\u0022 class=\u0022find-more-button eu-cookie-compliance-more-button\u0022\u003EFind out more\u003C\/button\u003E\n          \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E\n","mobile_breakpoint":"768","popup_html_agreed":"\u003Cdiv\u003E\n  \u003Cdiv class=\u0022popup-content agreed\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n      \u003Ch2\u003EThank you for accepting cookies\u003C\/h2\u003E\u003Cp\u003EYou can now hide this message or find out more about cookies.\u003C\/p\u003E    \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022hide-popup-button eu-cookie-compliance-hide-button\u0022\u003EHide\u003C\/button\u003E\n              \u003Cbutton type=\u0022button\u0022 class=\u0022find-more-button eu-cookie-compliance-more-button-thank-you\u0022 \u003EMore info\u003C\/button\u003E\n          \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E","popup_use_bare_css":false,"popup_height":"auto","popup_width":"100%","popup_delay":1000,"popup_link":"\/help\/cookie-policy","popup_link_new_window":1,"popup_position":null,"popup_language":"en","store_consent":false,"better_support_for_screen_readers":0,"reload_page":0,"domain":"","popup_eu_only_js":0,"cookie_lifetime":"365","cookie_session":false,"disagree_do_not_show_popup":0,"method":"default","whitelisted_cookies":"","withdraw_markup":"\u003Cbutton type=\u0022button\u0022 class=\u0022eu-cookie-withdraw-tab\u0022\u003EPrivacy settings\u003C\/button\u003E\n\u003Cdiv class=\u0022eu-cookie-withdraw-banner\u0022\u003E\n  \u003Cdiv class=\u0022popup-content info\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n      \u003Cp\u003E\u0026lt;h2\u0026gt;We use cookies on this site to enhance your user experience\u0026lt;\/h2\u0026gt;\u0026lt;p\u0026gt;You have given your consent for us to set cookies.\u0026lt;\/p\u0026gt;\u003C\/p\u003E\n    \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022eu-cookie-withdraw-button\u0022\u003EWithdraw consent\u003C\/button\u003E\n    \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E\n","withdraw_enabled":false},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1,"trackUrlFragments":1},"jnl_biorxiv_styles":{"defaultJCode":"biorxiv"},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-20978 node-type-highwire-article context-content hw-default-jcode-biorxiv hw-article-type-article hw-article-category-new-results">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-M677548" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-M677548');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
        <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
    
  <div id="zone-branding" class="zone zone-branding clearfix print-display-block container-30">
    <div class="grid-15 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="bioRxiv" src="https://www.biorxiv.org/sites/default/files/bioRxiv_article.jpg" /></a>      </div>
                </div>
          </div>
</div><div class="grid-11 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-system block-menu block-main-menu block-system-main-menu odd block-without-title" id="block-system-main-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/" title="" class="" data-icon-position="" data-hide-link-title="0">Home</a></li>
<li class="leaf" role="menuitem"><a href="/about-biorxiv" class="" data-icon-position="" data-hide-link-title="0">About</a></li>
<li class="leaf" role="menuitem"><a href="/submit-a-manuscript" class="" data-icon-position="" data-hide-link-title="0">Submit</a></li>
<li class="last leaf" role="menuitem"><a href="/content/alertsrss" title="" class="" data-icon-position="" data-hide-link-title="0">ALERTS / RSS</a></li>
</ul></nav>    </div>
  </div>
</div><div class="block block-panels-mini block-biorxiv-search-box block-panels-mini-biorxiv-search-box even block-without-title" id="block-panels-mini-biorxiv-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-biorxiv_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/10.1101/075960v1.full" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--2" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-4nwYMhDSDFimTWQ5ej9kTEhuXbqMrfEMTvdAAsr07ZA" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2 advanced-search-link" >
  
      
  
  <div class="pane-content">
    <a href="/search">Advanced Search</a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix container-30">
  	      </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="20978" id="node20978" data-pisa="biorxiv;075960v1" data-pisa-master="biorxiv;075960" data-apath="/biorxiv/early/2016/09/19/075960.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div  class="highwire-cite highwire-cite-highwire-article highwire-citation-biorxiv-article-top clearfix has-author-tooltip" >

      <span class="biorxiv-article-type">
        New Results    </span>
  
        <h1 class="highwire-cite-title" id="page-title">Correlations of genotype with climate parameters suggest <em>Caenorhabditis elegans</em> niche preferences</h1>  
      <div  class="highwire-cite-authors" ><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0"><span class="nlm-given-names">Kathryn S.</span> <span class="nlm-surname">Evans</span></span>, <span class="highwire-citation-author" data-delta="1"><span class="nlm-given-names">Yuehui</span> <span class="nlm-surname">Zhao</span></span>, <span class="highwire-citation-author" data-delta="2"><span class="nlm-given-names">Shannon C.</span> <span class="nlm-surname">Brady</span></span>, <span class="highwire-citation-author" data-delta="3"><span class="nlm-given-names">Lijiang</span> <span class="nlm-surname">Long</span></span>, <span class="highwire-citation-author" data-delta="4"><span class="nlm-given-names">Patrick T.</span> <span class="nlm-surname">McGrath</span></span>, <span class="highwire-citation-author" data-delta="5"><span class="nlm-given-names">Erik C.</span> <span class="nlm-surname">Andersen</span></span></span></div>
  
      <div  class="highwire-cite-metadata" ><span  class="highwire-cite-metadata-doi highwire-cite-metadata"><span class="label">doi:</span> https://doi.org/10.1101/075960 </span></div>
  
  
  </div>
<div id="hw-article-author-popups-node20978" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Kathryn S. Evans </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-institution'>Department of Molecular Biosciences, Northwestern University</span>, Evanston, IL 60208,</div><div class='author-affiliation'><span class='nlm-institution'>Interdisciplinary Biological Sciences Graduate Program, Northwestern University</span>, Evanston, IL 60208</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Kathryn%2BS.%2BEvans%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Evans%20KS&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AKathryn%2BS.%2BEvans%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Yuehui Zhao </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-institution'>School of Biology, Georgia Institute of Technology</span>, Atlanta, GA 30332</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Yuehui%2BZhao%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Zhao%20Y&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AYuehui%2BZhao%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Shannon C. Brady </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-institution'>Department of Molecular Biosciences, Northwestern University</span>, Evanston, IL 60208,</div><div class='author-affiliation'><span class='nlm-institution'>Interdisciplinary Biological Sciences Graduate Program, Northwestern University</span>, Evanston, IL 60208</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Shannon%2BC.%2BBrady%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Brady%20SC&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AShannon%2BC.%2BBrady%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Lijiang Long </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-institution'>School of Biology, Georgia Institute of Technology</span>, Atlanta, GA 30332</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Lijiang%2BLong%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Long%20L&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ALijiang%2BLong%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Patrick T. McGrath </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-institution'>School of Biology, Georgia Institute of Technology</span>, Atlanta, GA 30332</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Patrick%2BT.%2BMcGrath%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=McGrath%20PT&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3APatrick%2BT.%2BMcGrath%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Erik C. Andersen </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-institution'>Department of Molecular Biosciences, Northwestern University</span>, Evanston, IL 60208,</div><div class='author-affiliation'><span class='nlm-institution'>Chemistry of Life Processes Institute, Northwestern University</span>, Evanston, IL 60208</div><div class='author-affiliation'><span class='nlm-institution'>Northwestern Institute on Complex Systems, Northwestern University</span>, Evanston, IL 60208</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Erik%2BC.%2BAndersen%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Andersen%20EC&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AErik%2BC.%2BAndersen%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:erik.andersen@northwestern.edu" class="" data-icon-position="" data-hide-link-title="0">erik.andersen@northwestern.edu</a></span></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/10.1101/075960v1" class="panels-ajax-tab-tab" data-panel-name="biorxiv_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:20978" data-trigger="" data-url-enabled="1">Abstract</a><a href="/panels_ajax_tab/biorxiv_tab_art/node:20978/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/10.1101/075960v1.full-text" class="panels-ajax-tab-tab" data-panel-name="article_tab_full_text" data-target-id="highwire_article_tabs" data-entity-context="node:20978" data-trigger="full-text" data-url-enabled="1">Full Text</a><a href="/panels_ajax_tab/article_tab_full_text/node:20978/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/10.1101/075960v1.article-info" class="panels-ajax-tab-tab" data-panel-name="biorxiv_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:20978" data-trigger="article-info" data-url-enabled="1">Info/History</a><a href="/panels_ajax_tab/biorxiv_tab_info/node:20978/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/10.1101/075960v1.article-metrics" class="panels-ajax-tab-tab" data-panel-name="article_tab_metrics" data-target-id="highwire_article_tabs" data-entity-context="node:20978" data-trigger="article-metrics" data-url-enabled="1">Metrics</a><a href="/panels_ajax_tab/article_tab_metrics/node:20978/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/10.1101/075960v1.full.pdf+html" class="panels-ajax-tab-tab" data-panel-name="biorxiv_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:20978" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i> Preview PDF</a><a href="/panels_ajax_tab/biorxiv_tab_pdf/node:20978/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="article_tab_full_text" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.biorxiv.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-article_tab_full_text"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-markup" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2 class="">Abstract</h2><p id="p-2">Species inhabit a variety of environmental niches, and the adaptation to a particular niche is often controlled by genetic factors, including gene-by-environment interactions. The genes that vary in order to regulate the ability to colonize a niche are often difficult to identify, especially in the context of complex ecological systems and in experimentally uncontrolled natural environments. Quantitative genetic approaches provide an opportunity to investigate correlations between genetic factors and environmental parameters that might define a niche. Previously, we have shown how a collection of 208 whole-genome sequenced wild <em>Caenorhabditis elegans</em> can facilitate association mapping approaches. To correlate climate parameters with the variation found in this collection of wild strains, we used geographic data to exhaustively curate daily weather measurements in short-term (three month), middle-term (one year), and long-term (three year) durations surrounding the data of strain isolation. These climate parameters were then used as quantitative traits in the mapping approaches. We identified 10 QTL underlying variation in three traits: elevation, relative humidity, and average temperature. We then performed statistical analyses to further narrow the genomic interval of interest to identify gene candidates with variants potentially underlying phenotypic differences. Additionally, we performed two-strain competition assays at high and low temperatures to validate a QTL for temperature preference and found suggestive evidence that genotypes might be adapted to particular temperatures.</p><div id="sec-1" class="subsection"><p id="p-3"><strong>100-word summary for <em>G3</em></strong> Quantitative genetic approaches provide an opportunity to investigate correlations between genetic factors and environmental parameters that might define a niche, but these genes are difficult to identify, especially in the context of complex ecological systems. Here, we used a collection of 152 sequenced wild <em>Caenorhabditis elegans</em> to correlate climate parameters with the variation found in this collection of wild strains. We identified 10 QTL in five traits, including elevation, relative humidity, and temperature. Additionally, we performed competition assays to validate a QTL for temperature preference and found suggestive evidence that genotypes might be adapted to particular temperatures.</p></div></div><div class="section" id="sec-2"><h2 class="">INTRODUCTION</h2><p id="p-4">Ecological niches describe how individuals of a species respond to and alter the distribution of resources and competitors within their environment (<a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">Hutchinson 1957</a>). These resources could include food availability, soil type, short-term weather conditions, and long-term climate. Often, a species can be found in multiple distinct geographic areas that all share a common set of environmental factors and resources. For example, a plant that thrives at high temperature might grow equally well anywhere along the equator. An organism’s ability to survive in a specific niche is driven by both environmental and genetic factors. Genetic variation between species, and among individuals within a species, contributes to the wide variety of niches observed (<a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">Saltz and Nuzhdin 2014</a>). A genetic variant could result in an increased affinity for an individual to its environment. This individual will be selected and, over time, evolution will favor the successful variant. This phenomenon, known as gene-by-environment interactions, refers to when different genotypes respond to environmental variation in diverse ways.</p><p id="p-5">Previous studies in model organisms, particularly <em>Drosophila</em>, have investigated gene-by-environment interactions with clinal variation. Selection on body size is correlated with temperature (<a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">Taylor <em>et al.</em> 2015</a>), and survival is affected by climate change (<a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">Bozinovic <em>et al.</em> 2016</a>). Machado <em>et al.</em> performed a longitudinal study of <em>Drosophila</em> collected at differing latitudes during a two-year time span and compared physiological traits of two different species – <em>D. melanogaster</em> and <em>D. simulans</em> (<a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">Machado <em>et al.</em> 2016</a>). Other studies have gone further by identifying quantitative trait loci (QTL) for body size and cold tolerance traits involved in adaptation to seasonally varying environments (<a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">Tyukmaeva <em>et al.</em> 2015</a>; <a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">Hangartner <em>et al.</em> 2015</a>). Gerken <em>et al.</em> found substantial heritable variation in both short-term and long-term acclimation (<a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">Gerken <em>et al.</em> 2015</a>). They then performed genome-wide association mappings on these traits and found the QTL for short-term and long-term adaptations did not overlap, but each resulted in a set of gene candidates sharing similar functions of apoptosis, autophagy, cytoskeletal and membrane structural components, and ion binding and transport. Missing from these studies is genome-wide association mappings for other environmental traits besides temperature and latitude. Furthermore, little has been studied for other model organisms, such as <em>Caenorhabditis elegans</em>.</p><p id="p-6"><em>C. elegans</em> is a free-living nematode often found in microorganism-rich organic material such as rotting fruits and compost heaps in temperate and humid environments (<a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">Frézal and Félix 2015</a>; <a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">Félix and Braendle 2010</a>). The first studied <em>C. elegans</em> strain, N2, was isolated from mushroom farm compost in Bristol, England in 1951 (<a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">Hodgkin and Doniach 1997</a>). Since that time, N2 has been used as the wild-type strain for <em>C. elegans</em> laboratory research. This strain was cultured for many years in the laboratory, potentially resulting in selection for alleles favorable in that environment (<a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">Sterken <em>et al.</em> 2015</a>). To study natural variation and the role of niche specification on this species, we require a worldwide collection of wild strains. Our research group acquired a large collection of 208 wild strains and sequenced the whole genomes of these strains (<a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">Cook <em>et al.</em> 2016</a>). By comparing the genomes of the 208 strains, we found that some strains from similar geographic locations have nearly identical genome sequences. This analysis results in 152 unique genome-wide haplotypes or isotypes. This large pool of genetic information provides us with the statistical power to make connections between genotype and phenotype using genome-wide association (GWA) studies. GWA studies use common genetic variation across a population of individuals to discover variants that are correlated with a specific trait of interest (<a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">Hirschhorn and Daly 2005</a>).</p><p id="p-7">In this study, we correlate natural genetic variation among 152 wild <em>C. elegans</em> strains with climate measurements of their environmental niches as quantitative traits. We mapped traits that describe the niche of the isolation location for each strain, including geographic parameters, seasonal weather patterns, and climate variables. We find significant associations for elevation, relative humidity, and temperature. These findings suggest genetic control of niche specification. Additionally, we tested the quantitative trait locus (QTL) associated with temperature and found possible evidence of a specific preference for lower temperatures based on genetic background.</p></div><div class="section" id="sec-3"><h2 class="">RESULTS</h2><div id="sec-4" class="subsection"><h3>Genome-wide association of geographic traits</h3><p id="p-8">The location where a <em>C. elegans</em> strain was identified could reflect the process of selection for a particular genotype in a specific niche. For this reason, we investigated correlations between genetic variation in the <em>C. elegans</em> population and parameters describing the geographic locations of isolation as quantitative traits. Previous work with a smaller set of strains (97 wild isolates) detected a significant QTL on the left arm of chromosome II associated with the latitude where strains were isolated (<a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">Andersen <em>et al.</em> 2012</a>). To evaluate this trait and other parameters describing the location where each strain was isolated in our larger strain set (152 strains), we curated the isolation location information and defined several traits based on geographic data for each strain (<a id="xref-table-wrap-1-1" class="xref-table" href="#T1">Table 1</a>), namely latitude, longitude, elevation, the absolute value of latitude, and the absolute value of longitude (File S1). We performed genome-wide association mappings with 149 wild strains with known isolation locations (<a id="xref-fig-1-1" class="xref-fig" href="#F1">Figure 1A</a>) to correlate these trait values with common genetic variation (File S2; <em>see Methods</em>). Using this strain set, only the mapping of the elevation of the isolation location identified a significant QTL on the left arm of chromosome III (<a id="xref-fig-2-1" class="xref-fig" href="#F2">Figure 2A</a>). When we divided the population by the genotype at the peak marker, we found that the elevation values for these two sets of strains were similar with a few outliers (<a id="xref-fig-2-2" class="xref-fig" href="#F2">Figure 2B</a>), suggesting that the outliers were causing the detection of a QTL. It is possible that the association is spurious and driven by outliers or that the outliers are strains harboring rare alleles in the <em>C. elegans</em> species that impact this trait. The outlier strains in our mapping could share some genetic similarities as they were all collected within the last 15 years, and most were collected in France or elsewhere in Northern Europe. We did not recapitulate the QTL for latitude observed in the previous study (<a id="xref-ref-1-2" class="xref-bibr" href="#ref-1">Andersen <em>et al.</em> 2012</a>) likely because it also appears to be driven by strains with extreme latitude values. Again, these outlier strains could be highly related as seven of the 15 strains with the alternate genotype at the peak marker position originated from South Africa or Kenya. Our larger strain set reduces the effect of these outliers on the GWA mapping, and the previously detected QTL is no longer significant. These results suggest that common variation in the <em>C. elegans</em> species does not correlate with geographic parameters describing the location of strain isolation. However, rare variants might control whether strains can colonize and/or proliferate in specific geographic locations.</p><div id="F1" class="fig pos-float type-fig  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2016/09/19/075960/F1.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Global distribution of wild isolates and NOAA weather stations. A) Map of 27,447 ISD NOAA weather stations (blue) and 149 C. elegans wild isolate locations (red). Three isotypes are not depicted as they have no known location of isolation. B) Histogram of station distance from wild isolate, measured in kilometers (km). Data from the three-year weather dataset are shown." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-118663972" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;&lt;span class=&quot;caption-title&quot;&gt;Global distribution of wild isolates and NOAA weather stations.&lt;/span&gt; A) Map of 27,447 ISD NOAA weather stations (blue) and 149 &lt;em&gt;C. elegans&lt;/em&gt; wild isolate locations (red). Three isotypes are not depicted as they have no known location of isolation. B) Histogram of station distance from wild isolate, measured in kilometers (km). Data from the three-year weather dataset are shown.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 1." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2016/09/19/075960/F1.medium.gif" width="440" height="248"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 1." src="https://www.biorxiv.org/content/biorxiv/early/2016/09/19/075960/F1.medium.gif" width="440" height="248"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2016/09/19/075960/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1." data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2016/09/19/075960/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml"><span class="fig-label">Figure 1.</span> <span class="caption-title">Global distribution of wild isolates and NOAA weather stations.</span><p id="p-9" class="first-child">A) Map of 27,447 ISD NOAA weather stations (blue) and 149 <em>C. elegans</em> wild isolate locations (red). Three isotypes are not depicted as they have no known location of isolation. B) Histogram of station distance from wild isolate, measured in kilometers (km). Data from the three-year weather dataset are shown.</p><div class="sb-div caption-clear"></div></div></div><div id="F2" class="fig pos-float type-fig  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2016/09/19/075960/F2.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Genome-wide association of elevation. A) Genome-wide association of elevation of strain isolation shown as Manhattan plots. Genomic position is plotted on the x-axis against the negative log-transformed p-value on the y-axis. Single nucleotide variants (SNVs) that are above the Bonferroni-corrected significance threshold, indicated by the dotted grey line, are shown in red and SNVs below the Bonferroni threshold are shown in black. Confidence intervals are represented by the pink bars. B) Box plots show strain isolation elevations, measured in meters (m), separated by the genotype at the peak marker location. Each point represents one strain. The reference genotype (REF) refers to strains that share the genotype of the reference strain, N2. The alternative genotype (ALT) refers to all other strains that do not have the reference genotype at the peak marker position." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-118663972" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;&lt;span class=&quot;caption-title&quot;&gt;Genome-wide association of elevation.&lt;/span&gt; A) Genome-wide association of elevation of strain isolation shown as Manhattan plots. Genomic position is plotted on the x-axis against the negative log-transformed &lt;em&gt;p&lt;/em&gt;-value on the y-axis. Single nucleotide variants (SNVs) that are above the Bonferroni-corrected significance threshold, indicated by the dotted grey line, are shown in red and SNVs below the Bonferroni threshold are shown in black. Confidence intervals are represented by the pink bars. B) Box plots show strain isolation elevations, measured in meters (m), separated by the genotype at the peak marker location. Each point represents one strain. The reference genotype (REF) refers to strains that share the genotype of the reference strain, N2. The alternative genotype (ALT) refers to all other strains that do not have the reference genotype at the peak marker position.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 2." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2016/09/19/075960/F2.medium.gif" width="440" height="153"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 2." src="https://www.biorxiv.org/content/biorxiv/early/2016/09/19/075960/F2.medium.gif" width="440" height="153"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2016/09/19/075960/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2." data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2016/09/19/075960/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 2.</span> <span class="caption-title">Genome-wide association of elevation.</span><p id="p-10" class="first-child">A) Genome-wide association of elevation of strain isolation shown as Manhattan plots. Genomic position is plotted on the x-axis against the negative log-transformed <em>p</em>-value on the y-axis. Single nucleotide variants (SNVs) that are above the Bonferroni-corrected significance threshold, indicated by the dotted grey line, are shown in red and SNVs below the Bonferroni threshold are shown in black. Confidence intervals are represented by the pink bars. B) Box plots show strain isolation elevations, measured in meters (m), separated by the genotype at the peak marker location. Each point represents one strain. The reference genotype (REF) refers to strains that share the genotype of the reference strain, N2.</p><p id="p-11">The alternative genotype (ALT) refers to all other strains that do not have the reference genotype at the peak marker position.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-5" class="subsection"><h3>Weather conditions and climate parameters can be determined using the geographic location of the site of strain isolation</h3><p id="p-12">It is likely that the possible genetic association we observed between the elevation of strain isolation and a region on chromosome III is correlated with weather patterns and/or climate variables at specific geographic locations. Strain latitude and longitude can be used to determine the weather or climate at the time and location from which each wild <em>C. elegans</em> strain was isolated. The short-term weather surrounding the day of isolation as well as the long-term climate of the geographic location for each strain could improve our understanding of possible preferred niches for specific strains or for the species as a whole. Furthermore, combining these climate parameters with whole-genome sequence data could identify potential alleles that underlie preferences for certain environmental factors.</p><p id="p-13">The National Oceanic and Atmospheric Administration (NOAA) collects and provides multiple datasets related to weather and climate information, including the Integrated Surfaces Data (ISD). The ISD dataset is archived at the National Climatic Data Center (NCDC) and is composed of worldwide surface weather observations from over 27,000 stations managed by different global institutions (“Integrated Surface Global Hourly Data - NOAA Data Catalog” 2016). First, we manually curated the isolation information for the 152 strains, including the date of isolation, location, and sampling information (File S1). Then, we overlaid the locations of the 27,447 ISD weather stations with the isolation locations of the 149 <em>C. elegans</em> wild strains with complete sampling data (<a id="xref-fig-1-2" class="xref-fig" href="#F1">Figure 1A</a>). Using the date and isolation location for each wild strain, we identified the closest weather station with available data and collected weather observations in three time periods surrounding the date of nematode isolation: three months, one year, and three years. Most strains were found less than 60 km from a weather station (<a id="xref-fig-1-3" class="xref-fig" href="#F1">Figure 1B</a>). Of the 149 strains with known isolation locations, we knew at least the year of isolation for 145 strains, the month for 138 strains, and the day for 122 strains. For the three-month period, we analyzed weather station data only for strains with known days of isolation to provide a precise account of the daily weather experienced immediately surrounding the date of isolation for each strain. However, for the one-year period we used data from strains with known day or month of isolation. For the three-year period, we used data from strains where either the day, month, or year of isolation is known to provide an estimated overall climate of the strain isolation location. Not every weather station sampled contained data for each weather parameter. Additionally, only quantitative weather parameters that were measured at locations shared in a majority of the wild-isolate population (more than 90% of the strains) were considered for further analysis (<a id="xref-table-wrap-1-2" class="xref-table" href="#T1">Table 1</a>). All observations for each weather parameter were averaged over the given time period, and this average value was used as the trait measurement for each strain (File S2, <em>see Methods</em>).</p><div id="T1" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/466098/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup"><a href="/highwire/markup/466098/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/466098" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 1.</span> <p id="p-14" class="first-child">Definition of geographic and weather traits <sup>a</sup></p><div class="sb-div caption-clear"></div></div></div><p id="p-15">We evaluated all weather observations over the three months, one year, or three years surrounding the date of nematode isolation for each of the 149 wild strains to define the weather or climate experienced by each strain. These data were mapped using GWA to define nine QTL (Figure S1) for two distinct traits – relative humidity and temperature.</p></div><div id="sec-6" class="subsection"><h3>Genome-wide association of average relative humidity</h3><p id="p-16"><em>C. elegans</em> is found at various relative humidities ranging from 36% to 92%, with an average of 71.8% (File S2). This estimate of the average relative humidity of the <em>C. elegans</em> niche is in concordance with previous studies that show <em>C. elegans</em> is often found in humid environments (<a id="xref-ref-7-2" class="xref-bibr" href="#ref-7">Frézal and Félix 2015</a>). To determine if variation in relative humidity at isolation location is correlated with genetic variation, we performed GWA mapping for relative humidity of isolation location for three months, one year, and three years surrounding the date of isolation (<em>see Methods</em>). We found eight significant QTL in three distinct regions of the <em>C. elegans</em> genome: the left arm of chromosome II, the right arm of chromosome III, and the right arm of chromosome V (<a id="xref-fig-3-1" class="xref-fig" href="#F3">Figure 3</a>, File S2). The mapping for average relative humidity over three months surrounding the date of isolation (<a id="xref-fig-3-2" class="xref-fig" href="#F3">Figure 3A</a>) resulted in two linked QTL on the left arm of chromosome II (LD r<sup>2</sup> = 0.999, Figure S1) and a QTL on the right arm of chromosome III that is also in high linkage disequilibrium with the QTL on chromosome II (LD r<sup>2</sup> = 0.634, 0.812; Figure S1). The mapping for average relative humidity over one year surrounding the date of isolation (<a id="xref-fig-3-3" class="xref-fig" href="#F3">Figure 3B</a>) resulted in one QTL on the left arm of chromosome II and two highly linked QTL on the right of chromosome V (LD r<sup>2</sup> = 0.536; Figure S1). The position of the chromosome II QTL is the same as that observed for the mapping of three-month humidity. Finally, the mapping of average relative humidity over the three years surrounding the date of isolation (<a id="xref-fig-3-4" class="xref-fig" href="#F3">Figure 3C</a>) resulted in the same two QTL on chromosome V as found for the mapping of one-year relative humidity (Figure S1). For each QTL, strains with the reference allele at the peak marker tend to be isolated at higher relative humidities, and strains with an alternative allele tend to be isolated at lower relative humidities (Figure S1). This evidence of a phenotypic split dependent on genotype of the peak marker suggests that at least one variant could underlie a preference for relative humidity.</p><div id="F3" class="fig pos-float type-fig  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2016/09/19/075960/F3.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Genome-wide association of humidity traits. Genome-wide association of relative humidity for three different time periods are visualized as Manhattan plots: three-month (A), one-year (B), three-year (c) durations. Genomic position is plotted on the x-axis against the negative log-transformed p-value on the y-axis. SNVs that are above the Bonferroni-corrected significance threshold, indicated by the dotted grey line, are shown in red and SNVs below the Bonferroni threshold are shown in black. Confidence intervals are represented by the pink bars." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-118663972" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;&lt;span class=&quot;caption-title&quot;&gt;Genome-wide association of humidity traits.&lt;/span&gt; Genome-wide association of relative humidity for three different time periods are visualized as Manhattan plots: three-month (A), one-year (B), three-year (c) durations. Genomic position is plotted on the x-axis against the negative log-transformed &lt;em&gt;p&lt;/em&gt;-value on the y-axis. SNVs that are above the Bonferroni-corrected significance threshold, indicated by the dotted grey line, are shown in red and SNVs below the Bonferroni threshold are shown in black. Confidence intervals are represented by the pink bars.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 3." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2016/09/19/075960/F3.medium.gif" width="440" height="385"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 3." src="https://www.biorxiv.org/content/biorxiv/early/2016/09/19/075960/F3.medium.gif" width="440" height="385"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2016/09/19/075960/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 3." data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2016/09/19/075960/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 3.</span> <span class="caption-title">Genome-wide association of humidity traits.</span><p id="p-17" class="first-child">Genome-wide association of relative humidity for three different time periods are visualized as Manhattan plots: three-month (A), one-year (B), three-year (c) durations. Genomic position is plotted on the x-axis against the negative log-transformed <em>p</em>-value on the y-axis. SNVs that are above the Bonferroni-corrected significance threshold, indicated by the dotted grey line, are shown in red and SNVs below the Bonferroni threshold are shown in black. Confidence intervals are represented by the pink bars.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-7" class="subsection"><h3>Genome-wide association of average temperature</h3><p id="p-18"><em>C. elegans</em> are also found at a variety of average temperatures ranging from 7°C to 25°C, with an average of 15.3°C (File S1). To determine if temperature is associated with genetic variation, we performed a GWA mapping for temperature of isolation location for three months, one year, and three years surrounding the date of isolation (<em>see Methods</em>). We found one significant QTL just right of the center of chromosome V (<a id="xref-fig-4-1" class="xref-fig" href="#F4">Figure 4A</a>; File S2). This QTL is in the same location as that observed for one-year and three-year relative humidity (<a id="xref-fig-3-5" class="xref-fig" href="#F3">Figure 3B, 3C</a>). We found that strains with the reference (N2) allele at the peak marker tend to be isolated from geographic locations with lower temperatures, and strains with an alternative allele at this position tend to be isolated from geographic locations with higher temperatures (<a id="xref-fig-4-2" class="xref-fig" href="#F4">Figure 4B</a>). This QTL suggests that an allele, or alleles nearby this marker, could confer survival advantages to strains that experience different temperatures. To identify the variant(s) that could be driving this QTL, we investigated a region on chromosome V (V:13,845,281-15,332,878) defined by 1.48 Mb that contains 619 total genes. The genes with predicted functional variants are most likely to cause phenotypic differences among diverse strains in species. Therefore, we focused on 363 genes within this region predicted to harbor functional variants of moderate or severe effects on gene function, as determined by SnpEff (<a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">Cingolani <em>et al.</em> 2012</a>). We can further narrow our list of candidate genes to 48 by identifying genes that are highly correlated with differences in temperature (File S3). Although an investigation of these 48 genes did not identify an obvious candidate related to temperature regulation, one or more of these genes could be responsible for the preference of certain strains for specific temperatures.</p><div id="F4" class="fig pos-float type-fig  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2016/09/19/075960/F4.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Genome-wide association of temperature. A) Genome-wide association of three-year average temperature is visualized as a Manhattan plot. Genomic position is plotted on the x-axis against the negative log-transformed p-value on the y-axis. SNVs that are above the Bonferroni-corrected significance threshold, indicated by the dotted grey line, are shown in red, and SNVs below the Bonferroni threshold are shown in black. Confidence intervals are represented by the pink bars. B) Box plots show the strain three-year average temperatures, measured in degrees Celsius, separated by genotype at the peak marker locus. Each point represents one strain. The reference genotype (REF) refers to strains that share the genotype of the reference strain, N2. The alternative genotype (ALT) refers to all other strains that do not have the reference genotype at the peak marker position." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-118663972" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;&lt;span class=&quot;caption-title&quot;&gt;Genome-wide association of temperature.&lt;/span&gt; A) Genome-wide association of three-year average temperature is visualized as a Manhattan plot. Genomic position is plotted on the x-axis against the negative log-transformed &lt;em&gt;p&lt;/em&gt;-value on the y-axis. SNVs that are above the Bonferroni-corrected significance threshold, indicated by the dotted grey line, are shown in red, and SNVs below the Bonferroni threshold are shown in black. Confidence intervals are represented by the pink bars. B) Box plots show the strain three-year average temperatures, measured in degrees Celsius, separated by genotype at the peak marker locus. Each point represents one strain. The reference genotype (REF) refers to strains that share the genotype of the reference strain, N2. The alternative genotype (ALT) refers to all other strains that do not have the reference genotype at the peak marker position.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 4." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2016/09/19/075960/F4.medium.gif" width="440" height="153"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 4." src="https://www.biorxiv.org/content/biorxiv/early/2016/09/19/075960/F4.medium.gif" width="440" height="153"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2016/09/19/075960/F4.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 4." data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2016/09/19/075960/F4.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 4.</span> <span class="caption-title">Genome-wide association of temperature.</span><p id="p-19" class="first-child">A) Genome-wide association of three-year average temperature is visualized as a Manhattan plot. Genomic position is plotted on the x-axis against the negative log-transformed <em>p</em>-value on the y-axis. SNVs that are above the Bonferroni-corrected significance threshold, indicated by the dotted grey line, are shown in red, and SNVs below the Bonferroni threshold are shown in black. Confidence intervals are represented by the pink bars. B) Box plots show the strain three-year average temperatures, measured in degrees Celsius, separated by genotype at the peak marker locus. Each point represents one strain. The reference genotype (REF) refers to strains that share the genotype of the reference strain, N2.</p><p id="p-20">The alternative genotype (ALT) refers to all other strains that do not have the reference genotype at the peak marker position.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-8" class="subsection"><h3>Strains from divergent climates might be adapted to specific temperatures</h3><p id="p-21">Although we have GWA mappings for various weather conditions, validating these QTL would provide more evidence for <em>C. elegans</em> selection of niche based on environmental and geographic factors. Because temperature can be controlled easily and survival at defined temperatures can be tested experimentally, we decided to determine whether two strains from divergent climates are adapted to the respective temperatures nearby their isolation locations using a competition assay. Strains were chosen that had both a different genotype at the peak marker of the QTL on chromosome V identified in the three-year temperature mapping experiment and a large difference in the temperatures nearby the isolation location. JU847, a strain isolated from Northern France in 2005 (File S1), has the reference genotype at the three-year temperature QTL peak marker and is found at a low three-year average temperature (11.3°C). CX11314, a strain isolated from Southern California, USA in 2003 (File S1), has the alternative genotype at the three-year temperature QTL peak marker and is found at a higher three-year average temperature (20.9°C). High fitness at or around the temperature nearby the isolation location of one strain and lower fitness at or around the temperature nearby the isolation location for the other strain would suggest potential adaptive alleles that contribute to a preference and/or better survival at a specific temperature.</p><p id="p-22">Replicate cultures were initiated with an equal number of animals from each strain at either 15°C or 25°C and allowed to compete for at least six generations. After culture transfers two, four, and six, we analyzed the ratios of the two strains found at each temperature (<em>see Methods</em>). CX11314 was found to have higher fitness than JU847 at both temperatures tested (<a id="xref-fig-5-1" class="xref-fig" href="#F5">Figure 5</a>). At high temperature, CX11314 had a clear selective advantage compared to JU847 (for fitness = 1, relative CX11314 fitness s = 2.29), resulting in JU847 alleles comprising less than 1% of the alleles measured after six culture transfers. However, JU847 performed better at the lower temperature than at the higher temperature, comprising almost 8% of the total nematode population after six culture transfers (relative CX11314 fitness s = 1.57). These data suggest that JU847, although not more fit than CX11314 at either temperature, is more fit at 15°C than at 25°C.</p><div id="F5" class="fig pos-float type-fig  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2016/09/19/075960/F5.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Two-strain temperature competition assay. JU847 (isolated at low temperature) was competed against CX11314 (isolated at high temperature) at both 15&#xB0;C (indicated in blue) and 25&#xB0;C (indicated in red). The mean frequency of the JU847 allele in the population is plotted on the y-axis. Error bars represent one standard deviation from the mean. Data were collected from nine experimental and five technical replicates." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-118663972" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;span xmlns=&quot;http://www.w3.org/1999/xhtml&quot; class=&quot;caption-title&quot;&gt;Two-strain temperature competition assay.&lt;/span&gt; JU847 (isolated at low temperature) was competed against CX11314 (isolated at high temperature) at both 15&#xB0;C (indicated in blue) and 25&#xB0;C (indicated in red). The mean frequency of the JU847 allele in the population is plotted on the y-axis. Error bars represent one standard deviation from the mean. Data were collected from nine experimental and five technical replicates.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 5." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2016/09/19/075960/F5.medium.gif" width="440" height="327"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 5." src="https://www.biorxiv.org/content/biorxiv/early/2016/09/19/075960/F5.medium.gif" width="440" height="327"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2016/09/19/075960/F5.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 5." data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2016/09/19/075960/F5.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 5.</span> <span class="caption-title">Two-strain temperature competition assay.</span><p id="p-23" class="first-child">JU847 (isolated at low temperature) was competed against CX11314 (isolated at high temperature) at both 15°C (indicated in blue) and 25°C (indicated in red). The mean frequency of the JU847 allele in the population is plotted on the y-axis. Error bars represent one standard deviation from the mean. Data were collected from nine experimental and five technical replicates.</p><div class="sb-div caption-clear"></div></div></div></div></div><div class="section" id="sec-9"><h2 class="">DISCUSSION</h2><p id="p-24">In this study, we have defined geographic, weather, and climate variables over three different time periods as phenotypic traits for 149 unique wild <em>C. elegans</em> strains and performed genome-wide association mapping for 11 traits. Each phenotype described in this study was obtained using the location, date, and weather of the known isolation location of each isotype in our collection. We found significant correlations between genotype and phenotype for three traits and a total of 10 QTL. However, only temperature displayed strong phenotypic separation associated with genotypic variation that was likely not driven by outlier strains.</p><p id="p-25">Although we found a significant QTL associated with the elevation of isolation location, we did not recapitulate the QTL for latitude observed in the previous study (<a id="xref-ref-1-3" class="xref-bibr" href="#ref-1">Andersen <em>et al.</em> 2012</a>). This difference is likely because the previous mapping appears to be driven by strains with extreme latitude values, similar to our elevation mapping. The larger strain set in this analysis reduces the effect of these outliers on the GWA mapping, and thus the QTL is no longer significant. However, we did observe a QTL associated with latitude just below the Bonferroni-corrected significance threshold on the right arm of chromosome V. This suggestive QTL is in close proximity to the QTL for temperature. It is possible that the moderate association seen with latitude is representative of a real association with temperature, as the two traits are highly correlated. In our dataset, we see a high negative correlation between the absolute value of latitude and temperature (<em>rho</em> = −0.805), and between latitude and temperature (<em>rho</em> = −0.772). This lower-than-expected correlation is likely due to temperature fluctuations throughout the year: temperature at high latitude in the summer would be similar to temperature near the equator in the winter. Additionally, the latitude and longitude recorded for a particular strain is not always precise, especially for strains with older isolation dates. One tenth of a degree of latitude can distinguish between large cities, but could cover up to 11.1 km of distance. The function we used to determine elevation of strain isolation used these imprecise latitude and longitude coordinates, potentially resulting in a range of small to sizeable errors. These estimations could affect not just the geographic traits, but the weather traits as well, which are calculated based on the geographic coordinates. However, we do not expect this estimation to have a large effect, as we found weather data between stations up to 150 km apart to be highly correlated (<em>see Methods</em>).</p><p id="p-26">We chose to assess weather conditions for each of the 149 strains over three time periods: three months, one year, and three years. A duration of three months was chosen to define the weather contemporary to the date of isolation. One year was chosen to evaluate the average weather patterns a strain must be able to survive in nature. Three years was chosen to define the climate of the isolation location. Using data from three years could help us understand the average long-term climate conditions for each strain by eliminating any unusual weather patterns during the year of isolation that are not representative of the overall average environmental conditions. The relative humidity trait maps to the same chromosome positions for all three time periods assayed, suggesting that relative humidity remains constant throughout seasons. Previous studies have also shown that <em>C. elegans</em> tend to be found in humid regions (Frézal and Félix 2015). The same QTL on the right arm of chromosome V was observed for both long-term relative humidity and long-term temperature. Because relative humidity depends on air temperature, these results are expected to be correlated. Although the average temperature maps with only the three-year data set, we observed a QTL at the same position just below the significance threshold for the one-year mapping dataset as well. This suggestive QTL provides more evidence that one or more variants on chromosome V are associated with differences in long-term average temperature of isolation for <em>C. elegans</em>.</p><p id="p-27">The QTL for temperature was evaluated further because it could be controlled in a laboratory setting. The competition assay between JU847 and CX11314 showed that CX11314 had a higher selective advantage than JU847 at both high and low temperatures. This result could be observed because the laboratory environment can not completely recapitulate conditions experienced in the wild. Additionally, it is possible that CX11314 is more fit than JU847 regardless of temperature. To test this QTL more thoroughly and eliminate this possibility, it would be necessary to compete multiple high and low temperature strains. We found that the low temperature strain, JU847, was more competitive with the high temperature strain, CX11314, at the lower temperature. This result provides evidence that one or more genetic variants within the QTL could underlie the hypothesized temperature preference. Furthermore, the duration of this experiment was only six weeks. At the lower temperature, it is possible we could observe a stronger competitive advantage for JU847 over a longer time period.</p><p id="p-28">Although our analyses were unable to identify a single gene or variant that could underlie potential differences in niche specification, our conclusions suggest that different strains are found in unique niches and at least some of the environmental differences in niches are related to genetic variation among strains. As we expand our collection of wild <em>C. elegans</em> strains, we will be able to better define these weather and climate differences. Additionally, longitudinal collection studies with dense sampling, especially in a location with known high species diversity (such as the Hawaiian Islands) would give us more valuable data about how genetic variation in <em>C. elegans</em> is related to environmental conditions. We expect that similar data could be analyzed for other species and allow for investigation of niche specification, specifically in plant species where dense sampling and whole-genome datasets are available.</p></div><div class="section" id="sec-10"><h2 class="">MATERIALS AND METHODS</h2><div id="sec-11" class="subsection"><h3><em>C. elegans</em> wild isolate collection and sequencing</h3><p id="p-29">A collection of 208 wild <em>C. elegans</em> strains have been isolated worldwide and annotated for each strain’s geographic location and date of isolation. Members of the Andersen Lab have carefully and manually curated this dataset to offer the most accurate information possible while accounting for sometimes imprecise sample recording. Whole-genome sequence (WGS) data were collected from all 208 strains (<a id="xref-ref-4-2" class="xref-bibr" href="#ref-4">Cook <em>et al.</em> 2016</a>). The raw Illumina data are deposited with the Short Read Archive under project PRJNA318647. WGS data were analyzed as previously described. In brief, after alignment with BWA (<a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">Li and Durbin 2009</a>) and variant calling using Samtools (<a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">Li <em>et al.</em> 2009</a>), strains with a concordance of 99.93% or higher were grouped as a genome-wide haplotype or isotype. This analysis resulted in 152 unique isotypes (File S1).</p></div><div id="sec-12" class="subsection"><h3>Weather and climate data acquisition</h3><p id="p-30">For each wild strain with a known isolation location, elevation was estimated with the “geosphere” package in R (<a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">Hijmans, Williams, and Vennes 2012</a>) using the geographic coordinates of strain isolation. A correlation test using 74 points of known elevation were used to verify accuracy of the elevation function resulting in a correlation of 0.998. Weather data were downloaded from the Integrated Surface Data (ISD) FTP server (<a href="http://ftp://ftp.ncdc.noaa.gov/pub/data/noaa/">ftp://ftp.ncdc.noaa.gov/pub/data/noaa/</a>) managed by the National Oceanic and Atmospheric Administration (NOAA) and the National Climatic Data Center (NCDC). The ISD dataset comprises worldwide surface weather observations from over 27,447 stations managed by the Automated Weather Network (AWN), the Global Telecommunications System (GTS), the Automated Surface Observing System (ASOS), and others. Data are collected once every three hours for some stations. Some parameters include air quality, atmospheric pressure, atmospheric temperature, dew point, atmospheric winds, clouds, precipitation, ocean waves, and tides.</p><p id="p-31">Three distinct sets of weather station data were collected for analysis: a three-month window, a one-year window, and a three-year window. These data were filtered to include values centered around the date of isolation. For the three-month set, wild isolates with only a known month or year of isolation were not considered. Exact day of isolation is necessary to understand the seasonal environment in which an animal was isolated. For the one-year dataset, wild isolates with a month or day of isolation were used. For the three-year dataset, strains with only a year of isolation were used in addition to those strains with more defined dates of isolation. If only the year was known, the date of isolation defaulted to January 1 of that year, and data were collected surrounding that date. If only the month of isolation was known, the date of isolation was defaulted to the first of that month for data collection.</p><p id="p-32">The 27,447 NOAA weather stations were filtered by their availability of data collected within the years of interest. Stations that had less than ten recordings of any type for any month within the time period of data collection were excluded to avoid misrepresentation by datasets that were averaged from only a few data points. Stations were then filtered by location, and the closest station to location of isolation for each wild strain was selected and downloaded using the “stationaRy” package available at <a href="https://github.com/rich-iannone/stationaRy">https://github.com/rich-iannone/stationaRy</a> (<a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">Iannone 2015</a>). We performed a rank-correlation test for temperature, relative humidity, and atmospheric pressure between two neighboring weather stations (ranging from 0.93-153 km apart) and found high correlations regardless of distance between stations (<em>rho</em> = 0.920, 0.913, and 0.707, respectively). All station-isotype pairs were included in our analysis regardless of the distance between them. The primary fields as well as all additional quantitative data available for each station were downloaded. Some fields (<em>e.g.</em> “AT1” or “Present-weather-observation”) were not downloaded from the NOAA database because the traits are qualitative and would not be conducive to quantitative analyses. The station data were filtered to contain only information from the months surrounding the date of isolation. This process was repeated for each dataset (three-month, one-year, and three-year) in case a closer weather station contained only data for the three-month set but not for the one-year or three-year sets. The station data were meticulously checked by manually removing missing values from each weather category independently and, in certain cases, converting fields to uniform units that can be averaged to form a trait value. For example, precipitation (“AA1”) was downloaded in two columns: 1) time period; 2) depth of precipitation recorded during that time period. Because the variable time periods in which data were recorded, averaging precipitation would lead to skewed results. Precipitation was changed to adapt a “precipitation per hour” model that would be more permissive to our analyses. Each trait was averaged over the time span collected (three months, one year, or three years), and the average value was designated as the phenotypic value for that strain. Only traits with values in more than 90% of strains were analyzed further.</p></div><div id="sec-13" class="subsection"><h3>Association mapping</h3><p id="p-33">Genome-wide association (GWA) mapping was performed using 152 genome-wide <em>C. elegans</em> isotypes using the cegwas R package found at <a href="https://github.com/AndersenLab/cegwas">https://github.com/AndersenLab/cegwas</a>. This package uses the EMMA algorithm for performing association mapping and correcting for population structure (<a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">Kang <em>et al.</em> 2008</a>), which is implemented by the GWAS function in the rrBLUP package (<a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">Endelman 2011</a>). The kinship matrix used for association mapping was generated using a whole-genome high-quality single nucleotide variant (SNV) set (<a id="xref-ref-4-3" class="xref-bibr" href="#ref-4">Cook <em>et al.</em> 2016</a>) and the A.mat function from the rrBLUP package. Single-nucleotide variants identified using RAD-marker sequencing (<a id="xref-ref-1-4" class="xref-bibr" href="#ref-1">Andersen <em>et al.</em> 2012</a>) that had at least 5% minor allele frequency in the 152 isotype set were used for performing GWA mappings. Association mappings that contained at least one SNV that had a −log(<em>p</em>-value) greater than the Bonferroni-corrected <em>p</em>-value were processed further using fine mapping, which entails a Spearman’s rank correlation test with variants from the whole-genome sequence data of moderate to severe predicted effects as determined by the SnpEff function (<a id="xref-ref-3-2" class="xref-bibr" href="#ref-3">Cingolani <em>et al.</em> 2012</a>).</p></div><div id="sec-14" class="subsection"><h3>Temperature competition assays</h3><p id="p-34">We chose two strains, CX11314 and JU847, that had different alleles for the peak QTL marker (chrV: 14,822,276; JU847: T, CX11314: A) in our three-year temperature GWA mapping. JU847 has the reference allele for the peak marker and was isolated at a low temperature, whereas CX11314 has the alternative allele for the peak marker and was isolated at a higher temperature. We designed a Taqman probe (5’-[A]CCGTTTTTTTT[T/A]AATTTT-3’) to measure each of these two alleles from mixed samples of nematodes using the standard software from Applied Biosystems (<a href="https://www.thermofisher.com/order/custom-genomic-products/tools/genotyping/">https://www.thermofisher.com/order/custom-genomic-products/tools/genotyping/</a>) and a corresponding primer set to amplify the region of interest (below).</p><p id="p-35">F: 5’-AAACCCAAGATTTTTATGGTTACTTTAAGATTTGT-3’;</p><p id="p-36">R: 5’-ATCTATAGTTAACTTGGATATATTGTTTGTTTTCGGT-3’</p><p id="p-37">These two strains were chunked to fresh 10 cm NGMA plates seeded with OP50. 48 hours later, seven L4s from each strain were added to each of 45 6 cm NGMA plates seeded with OP50 for both 15°C and 25°C competition experiments. The 45 plates at each temperature represent nine experimental replicates each composed of five independent populations. Plates were placed at either 15°C or 25°C and grown to starvation. After one week for 25°C competition experiments or 10 days for 15°C competition experiments, nematodes were transferred to fresh NGMA plates by cutting a 0.5 cm x 0.5 cm square of agar (containing ~100 worms) and replaced at the appropriate temperature. After culture transfers two, four, and six, starved animals were washed off the plates with M9, and DNA was collected using the Qiagen DNeasy Kit. Genomic DNA from each time point was digested with the <em>Eco</em>RI enzyme and purified using the Zymo DNA Clean &amp; Concentrator Kit. The concentration of fragmented genomic DNA was adjusted to 2 ng/µL by Qubit assay. The number of JU847 and CX11314 alleles in each replicate population was measured using Taqman analysis in a Biorad QZ200 digital droplet PCR system (File S4). Digital PCR was performed following the standard protocol provided by Biorad with the absolute quantification method. The proportion of the JU847 allele and the relative selection coefficients were calculated.</p></div><div id="sec-15" class="subsection"><h3>Data Availability</h3><p id="p-38">Strains are available through the <em>Caenorhabditis elegans</em> Natural Diversity Resource (CeNDR, <a href="http://www.elegansvariation.org">www.elegansvariation.org</a>). File S1 contains the strains, location data, weather station data, and all traits used in mappings. File S2 contains the association mapping data. File S3 contains the interval fine mapping raw data. File S4 contains the count data from the digital droplet PCR for JU847 and CX11314 alleles.</p></div></div><div class="section ref-list" id="ref-list-1"><h2 class="">LITERATURE CITED</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference  in text" id="ref-1">↵</a><div class="cit ref-cit ref-journal" id="cit-075960v1.1" data-doi="10.1038/ng.1050"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Andersen</span>,  <span class="cit-name-given-names">Erik C.</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">Justin P.</span> <span class="cit-name-surname">Gerke</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">Joshua A.</span> <span class="cit-name-surname">Shapiro</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">Jonathan R.</span> <span class="cit-name-surname">Crissman</span></span>, <span class="cit-auth"><span class="cit-name-surname">Rajarshi</span>  <span class="cit-name-given-names">Ghosh</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">Joshua S.</span> <span class="cit-name-surname">Bloom</span></span>, <span class="cit-auth"><span class="cit-name-surname">Marie-Anne</span>  <span class="cit-name-given-names">Félix</span></span>, and <span class="cit-auth"><span class="cit-name-surname">Leonid</span>  <span class="cit-name-given-names">Kruglyak</span></span>. <span class="cit-pub-date">2012</span>. “<span class="cit-article-title">Chromosome-Scale Selective Sweeps Shape Caenorhabditis Elegans Genomic Diversity</span>.” <abbr class="cit-jnl-abbrev">Nature Genetics</abbr> <span class="cit-vol">44</span> (<span class="cit-issue">3</span>): <span class="cit-fpage">285</span>–<span class="cit-lpage">90</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bgenetics%26rft.stitle%253DNat%2BGenet%26rft.aulast%253DAndersen%26rft.auinit1%253DE.%2BC.%26rft.volume%253D44%26rft.issue%253D3%26rft.spage%253D285%26rft.epage%253D290%26rft.atitle%253DChromosome-scale%2Bselective%2Bsweeps%2Bshape%2BCaenorhabditis%2Belegans%2Bgenomic%2Bdiversity.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fng.1050%26rft_id%253Dinfo%253Apmid%252F22286215%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ng.1050&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22286215&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2016%2F09%2F19%2F075960.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference  in text" id="ref-2">↵</a><div class="cit ref-cit ref-journal" id="cit-075960v1.2" data-doi="10.1007/s00360-016-0980-6"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Bozinovic</span></span>, <span class="cit-auth"><span class="cit-name-surname">Francisco</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">Nadia R.</span> <span class="cit-name-surname">Medina</span></span>, <span class="cit-auth"><span class="cit-name-surname">José</span>  <span class="cit-name-given-names">M. Alruiz</span></span>, <span class="cit-auth"><span class="cit-name-surname">Grisel</span>  <span class="cit-name-given-names">Cavieres</span></span>, and <span class="cit-auth"><span class="cit-name-surname">Pablo</span>  <span class="cit-name-given-names">Sabat</span></span>. <span class="cit-pub-date">2016</span>. “<span class="cit-article-title">Thermal Tolerance and Survival Responses to Scenarios of Experimental Climatic Change: Changing Thermal Variability Reduces the Heat and Cold Tolerance in a Fly</span>.” <abbr class="cit-jnl-abbrev">Journal of Comparative Physiology. B, Biochemical, Systemic, and Environmental Physiology</abbr> <span class="cit-vol">186</span> (<span class="cit-issue">5</span>): <span class="cit-fpage">581</span>–<span class="cit-lpage">87</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BComparative%2BPhysiology.%2BB%252C%2BBiochemical%252C%2BSystemic%252C%2Band%2BEnvironmental%2BPhysiology%26rft.volume%253D186%26rft.spage%253D581%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00360-016-0980-6%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00360-016-0980-6&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference  in text" id="ref-3">↵</a><div class="cit ref-cit ref-journal" id="cit-075960v1.3" data-doi="10.4161/fly.19695"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Cingolani</span></span>, <span class="cit-auth"><span class="cit-name-surname">Pablo</span></span>, <span class="cit-auth"><span class="cit-name-surname">Adrian</span>  <span class="cit-name-given-names">Platts</span></span>, <span class="cit-auth"><span class="cit-name-surname">Le</span>  <span class="cit-name-given-names">Lily Wang</span></span>, <span class="cit-auth"><span class="cit-name-surname">Melissa</span>  <span class="cit-name-given-names">Coon</span></span>, <span class="cit-auth"><span class="cit-name-surname">Tung</span>  <span class="cit-name-given-names">Nguyen</span></span>, <span class="cit-auth"><span class="cit-name-surname">Luan</span>  <span class="cit-name-given-names">Wang</span></span>, <span class="cit-auth"><span class="cit-name-surname">Susan</span>  <span class="cit-name-given-names">J.</span></span> <span class="cit-auth"><span class="cit-name-surname">Land</span></span>, <span class="cit-auth"><span class="cit-name-surname">Xiangyi</span>  <span class="cit-name-given-names">Lu</span></span>, and <span class="cit-auth"> <span class="cit-name-given-names">Douglas M.</span> <span class="cit-name-surname">Ruden</span></span>. <span class="cit-pub-date">2012</span>. “<span class="cit-article-title">A Program for Annotating and Predicting the Effects of Single Nucleotide Polymorphisms, SnpEff: SNPs in the Genome of Drosophila Melanogaster Strain w1118; Iso-2; Iso-3</span>.” <abbr class="cit-jnl-abbrev">Fly</abbr> <span class="cit-vol">6</span> (<span class="cit-issue">2</span>): <span class="cit-fpage">80</span>–<span class="cit-lpage">92</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DFly%26rft.volume%253D6%26rft.spage%253D80%26rft_id%253Dinfo%253Adoi%252F10.4161%252Ffly.19695%26rft_id%253Dinfo%253Apmid%252F22728672%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.4161/fly.19695&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22728672&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2016%2F09%2F19%2F075960.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000305965500003&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference  in text" id="ref-4">↵</a><div class="cit ref-cit ref-journal" id="cit-075960v1.4" data-doi="10.1534/genetics.116.191148"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Cook</span></span>, <span class="cit-auth"><span class="cit-name-surname">Daniel</span>  <span class="cit-name-given-names">E.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Stefan</span>  <span class="cit-name-given-names">Zdraljevic</span></span>, <span class="cit-auth"><span class="cit-name-surname">Robyn</span>  <span class="cit-name-given-names">E.</span></span> <span class="cit-auth"><span class="cit-name-surname">Tanny</span></span>, <span class="cit-auth"><span class="cit-name-surname">Beomseok</span>  <span class="cit-name-given-names">Seo</span></span>, <span class="cit-auth"><span class="cit-name-surname">David</span>  <span class="cit-name-given-names">D.</span></span> <span class="cit-auth"><span class="cit-name-surname">Riccardi</span></span>, <span class="cit-auth"><span class="cit-name-surname">Luke</span>  <span class="cit-name-given-names">M.</span></span> <span class="cit-auth"><span class="cit-name-surname">Noble</span></span>, <span class="cit-auth"><span class="cit-name-surname">Matthew</span>  <span class="cit-name-given-names">V.</span></span> <span class="cit-auth"><span class="cit-name-surname">Rockman</span></span>, <span class="cit-etal">et al.</span> <span class="cit-pub-date">2016</span>. “<span class="cit-article-title">The Genetic Basis of Natural Variation in Caenorhabditis Elegans Telomere Length</span>.” <abbr class="cit-jnl-abbrev">Genetics</abbr>, <span class="cit-month">July</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1534/genetics.116.191148</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.116.191148%26rft_id%253Dinfo%253Apmid%252F27449056%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMjA0LzEvMzcxIjtzOjQ6ImF0b20iO3M6Mzc6Ii9iaW9yeGl2L2Vhcmx5LzIwMTYvMDkvMTkvMDc1OTYwLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference  in text" id="ref-5">↵</a><div class="cit ref-cit ref-journal" id="cit-075960v1.5"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Endelman</span>,  <span class="cit-name-given-names">J. B.</span></span> <span class="cit-pub-date">2011</span>. “<span class="cit-article-title">Ridge Regression and Other Kernels for Genomic Selection with R Package rrBLUP</span>.” <abbr class="cit-jnl-abbrev">The Plant Genome</abbr>. <a href="http://dl.sciencesocieties.org">dl.sciencesocieties.org</a>. <a href="https://dl.sciencesocieties.org/publications/tpg/abstracts/4/3/250">https://dl.sciencesocieties.org/publications/tpg/abstracts/4/3/250</a>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference  in text" id="ref-6">↵</a><div class="cit ref-cit ref-journal" id="cit-075960v1.6"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Félix</span></span>, <span class="cit-auth"><span class="cit-name-surname">Marie-Anne</span></span>, and <span class="cit-auth"><span class="cit-name-surname">Christian</span>  <span class="cit-name-given-names">Braendle</span></span>. <span class="cit-pub-date">2010</span>. “<span class="cit-article-title">The Natural History of Caenorhabditis Elegans</span>.” <abbr class="cit-jnl-abbrev">Current Biology: CB</abbr> <span class="cit-vol">20</span> (<span class="cit-issue">22</span>): <span class="cit-fpage">R965</span>–<span class="cit-lpage">69</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCurrent%2BBiology%253A%2BCB%26rft.volume%253D20%26rft.spage%253D965R%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference  in text" id="ref-7">↵</a><div class="cit ref-cit ref-journal" id="cit-075960v1.7" data-doi="10.7554/eLife.05849"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Frézal</span>,  <span class="cit-name-given-names">Lise</span></span>, and <span class="cit-auth"><span class="cit-name-surname">Marie-Anne</span>  <span class="cit-name-given-names">Félix</span></span>. <span class="cit-pub-date">2015</span>. “<span class="cit-article-title">C. Elegans Outside the Petri Dish</span>.” <abbr class="cit-jnl-abbrev">eLife</abbr> <span class="cit-vol">4</span> (<span class="cit-month">March</span>).<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.7554/eLife.05849</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DeLife%26rft_id%253Dinfo%253Adoi%252F10.7554%252FeLife.05849%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.7554/eLife.05849&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference  in text" id="ref-8">↵</a><div class="cit ref-cit ref-journal" id="cit-075960v1.8" data-doi="10.1073/pnas.1503456112"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Gerken</span></span>, <span class="cit-auth"><span class="cit-name-surname">Alison</span>  <span class="cit-name-given-names">R.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Olivia</span>  <span class="cit-name-given-names">C.</span></span> <span class="cit-auth"><span class="cit-name-surname">Eller</span></span>, <span class="cit-auth"><span class="cit-name-surname">Daniel</span>  <span class="cit-name-given-names">A.</span></span> <span class="cit-auth"><span class="cit-name-surname">Hahn</span></span>, and <span class="cit-auth"> <span class="cit-name-given-names">Theodore J.</span> <span class="cit-name-surname">Morgan</span></span>. <span class="cit-pub-date">2015</span>. “<span class="cit-article-title">Constraints, Independence, and Evolution of Thermal Plasticity: Probing Genetic Architecture of Long- and Short-Term Thermal Acclimation</span>.” <abbr class="cit-jnl-abbrev">Proceedings of the National Academy of Sciences of the United States of America</abbr> <span class="cit-vol">112</span> (<span class="cit-issue">14</span>): <span class="cit-fpage">4399</span>–<span class="cit-lpage">4404</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProceedings%2Bof%2Bthe%2BNational%2BAcademy%2Bof%2BSciences%2Bof%2Bthe%2BUnited%2BStates%2Bof%2BAmerica%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.1503456112%26rft_id%253Dinfo%253Apmid%252F25805817%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMToiMTEyLzE0LzQzOTkiO3M6NDoiYXRvbSI7czozNzoiL2Jpb3J4aXYvZWFybHkvMjAxNi8wOS8xOS8wNzU5NjAuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference  in text" id="ref-9">↵</a><div class="cit ref-cit ref-journal" id="cit-075960v1.9"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Hangartner</span></span>, <span class="cit-auth"><span class="cit-name-surname">Sandra</span>  <span class="cit-name-given-names">B.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Ary</span>  <span class="cit-name-given-names">A.</span></span> <span class="cit-auth"><span class="cit-name-surname">Hoffmann</span></span>, <span class="cit-auth"><span class="cit-name-surname">Ailie</span>  <span class="cit-name-given-names">Smith</span></span>, and <span class="cit-auth"> <span class="cit-name-given-names">Philippa C.</span> <span class="cit-name-surname">Griffin</span></span>. <span class="cit-pub-date">2015</span>. “<span class="cit-article-title">A Collection of Australian Drosophila Datasets on Climate Adaptation and Species Distributions</span>.” <abbr class="cit-jnl-abbrev">Scientific Data</abbr> <span class="cit-vol">2</span> (November): <span class="cit-fpage">150067</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScientific%2BData%26rft.volume%253D2%26rft.spage%253D150067%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference  in text" id="ref-10">↵</a><div class="cit ref-cit ref-journal" id="cit-075960v1.10"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Hijmans</span>,  <span class="cit-name-given-names">R. J.</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">E.</span> <span class="cit-name-surname">Williams</span></span>, and <span class="cit-auth"> <span class="cit-name-given-names">C.</span> <span class="cit-name-surname">Vennes</span></span>. <span class="cit-pub-date">2012</span>. “<span class="cit-article-title">Geosphere: Spherical Trigonometry. R Package Version 1.2–28</span>.” <abbr class="cit-jnl-abbrev">CRAN. R-Project. Org/package= Geosphere</abbr>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference  in text" id="ref-11">↵</a><div class="cit ref-cit ref-journal" id="cit-075960v1.11" data-doi="10.1038/nrm1583"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Hirschhorn</span></span>, <span class="cit-auth"><span class="cit-name-surname">Joel</span>  <span class="cit-name-given-names">N.</span></span>, and <span class="cit-auth"> <span class="cit-name-given-names">Mark J.</span> <span class="cit-name-surname">Daly</span></span>. <span class="cit-pub-date">2005</span>. “<span class="cit-article-title">Genome-Wide Association Studies for Common Diseases and Complex Traits</span>.” <abbr class="cit-jnl-abbrev">Nature Reviews. Genetics</abbr> <span class="cit-vol">6</span> (<span class="cit-issue">2</span>): <span class="cit-fpage">95</span>–<span class="cit-lpage">108</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Breviews.%2BGenetics%26rft.stitle%253DNat%2BRev%2BGenet%26rft.aulast%253DHirschhorn%26rft.auinit1%253DJ.%2BN.%26rft.volume%253D6%26rft.issue%253D2%26rft.spage%253D95%26rft.epage%253D108%26rft.atitle%253DGenome-wide%2Bassociation%2Bstudies%2Bfor%2Bcommon%2Bdiseases%2Band%2Bcomplex%2Btraits.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnrm1583%26rft_id%253Dinfo%253Apmid%252F15716906%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nrm1583&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15716906&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2016%2F09%2F19%2F075960.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000226721300010&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference  in text" id="ref-12">↵</a><div class="cit ref-cit ref-journal" id="cit-075960v1.12"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Hodgkin</span>,  <span class="cit-name-given-names">J.</span></span>, and <span class="cit-auth"> <span class="cit-name-given-names">T.</span> <span class="cit-name-surname">Doniach</span></span>. <span class="cit-pub-date">1997</span>. “<span class="cit-article-title">Natural Variation and Copulatory Plug Formation in Caenorhabditis Elegans</span>.” <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">146</span> (<span class="cit-issue">1</span>): <span class="cit-fpage">149</span>–<span class="cit-lpage">64</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DHodgkin%26rft.auinit1%253DJ.%26rft.volume%253D146%26rft.issue%253D1%26rft.spage%253D149%26rft.epage%253D164%26rft.atitle%253DNatural%2BVariation%2Band%2BCopulatory%2BPlug%2BFormation%2Bin%2BCaenorhabditis%2Belegans%26rft_id%253Dinfo%253Apmid%252F9136008%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTQ2LzEvMTQ5IjtzOjQ6ImF0b20iO3M6Mzc6Ii9iaW9yeGl2L2Vhcmx5LzIwMTYvMDkvMTkvMDc1OTYwLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference  in text" id="ref-13">↵</a><div class="cit ref-cit ref-journal" id="cit-075960v1.13"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Hutchinson</span>,  <span class="cit-name-given-names">G. E.</span></span> <span class="cit-pub-date">1957</span>. “<span class="cit-article-title">Cold Spring Harbor Symposium on Quantitative Biology</span>.” <abbr class="cit-jnl-abbrev">Concluding Remarks</abbr>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference  in text" id="ref-14">↵</a><div class="cit ref-cit ref-journal" id="cit-075960v1.14"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Iannone</span></span>, <span class="cit-auth"><span class="cit-name-surname">Richard</span></span>. <span class="cit-pub-date">2015</span>. “<span class="cit-article-title">stationaRy: Get Hourly Meteorological Data from Global Stations. R Package Version 0.4.1</span>.” <abbr class="cit-jnl-abbrev">CRAN. R-Project. Org/package= stationaRy</abbr>, <span class="cit-month">October</span>. Comprehensive R Archive Network (CRAN).</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-075960v1.15"><div class="cit-metadata"><cite>“<span class="cit-article-title">Integrated Surface Global Hourly Data - NOAA Data Catalog</span>.” <span class="cit-pub-date">2016</span>. Accessed September 2. <a href="https://data.noaa.gov/dataset/integrated-surface-global-hourly-data">https://data.noaa.gov/dataset/integrated-surface-global-hourly-data</a>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference  in text" id="ref-16">↵</a><div class="cit ref-cit ref-journal" id="cit-075960v1.16" data-doi="10.1534/genetics.107.080101"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Kang</span></span>, <span class="cit-auth"><span class="cit-name-surname">Hyun</span>  <span class="cit-name-given-names">Min</span></span>, <span class="cit-auth"><span class="cit-name-surname">Noah</span>  <span class="cit-name-given-names">A.</span></span> <span class="cit-auth"><span class="cit-name-surname">Zaitlen</span></span>, <span class="cit-auth"><span class="cit-name-surname">Claire</span>  <span class="cit-name-given-names">M.</span></span> <span class="cit-auth"><span class="cit-name-surname">Wade</span></span>, <span class="cit-auth"><span class="cit-name-surname">Andrew</span>  <span class="cit-name-given-names">Kirby</span></span>, <span class="cit-auth"><span class="cit-name-surname">David</span>  <span class="cit-name-given-names">Heckerman</span></span>, <span class="cit-auth"><span class="cit-name-surname">Mark</span>  <span class="cit-name-given-names">J.</span></span> <span class="cit-auth"><span class="cit-name-surname">Daly</span></span>, and Eleazar Eskin. <span class="cit-pub-date">2008</span>. “<span class="cit-article-title">Efficient Control of Population Structure in Model Organism Association Mapping</span>.” <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">178</span> (<span class="cit-issue">3</span>): <span class="cit-fpage">1709</span>–<span class="cit-lpage">23</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DKang%26rft.auinit1%253DH.%2BM.%26rft.volume%253D178%26rft.issue%253D3%26rft.spage%253D1709%26rft.epage%253D1723%26rft.atitle%253DEfficient%2BControl%2Bof%2BPopulation%2BStructure%2Bin%2BModel%2BOrganism%2BAssociation%2BMapping%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.107.080101%26rft_id%253Dinfo%253Apmid%252F18385116%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE3OC8zLzE3MDkiO3M6NDoiYXRvbSI7czozNzoiL2Jpb3J4aXYvZWFybHkvMjAxNi8wOS8xOS8wNzU5NjAuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference  in text" id="ref-17">↵</a><div class="cit ref-cit ref-journal" id="cit-075960v1.17" data-doi="10.1093/bioinformatics/btp324"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Li</span></span>, <span class="cit-auth"><span class="cit-name-surname">Heng</span></span>, and <span class="cit-auth"><span class="cit-name-surname">Richard</span>  <span class="cit-name-given-names">Durbin</span></span>. <span class="cit-pub-date">2009</span>. “<span class="cit-article-title">Fast and Accurate Short Read Alignment with Burrows–Wheeler Transform</span>.” <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">25</span> (<span class="cit-issue">14</span>): <span class="cit-fpage">1754</span>–<span class="cit-lpage">60</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtp324%26rft_id%253Dinfo%253Apmid%252F19451168%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/btp324&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19451168&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2016%2F09%2F19%2F075960.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000267665900006&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference  in text" id="ref-18">↵</a><div class="cit ref-cit ref-journal" id="cit-075960v1.18" data-doi="10.1093/bioinformatics/btp352"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Li</span></span>, <span class="cit-auth"><span class="cit-name-surname">Heng</span></span>, <span class="cit-auth"><span class="cit-name-surname">Bob</span>  <span class="cit-name-given-names">Handsaker</span></span>, <span class="cit-auth"><span class="cit-name-surname">Alec</span>  <span class="cit-name-given-names">Wysoker</span></span>, <span class="cit-auth"><span class="cit-name-surname">Tim</span>  <span class="cit-name-given-names">Fennell</span></span>, <span class="cit-auth"><span class="cit-name-surname">Jue</span>  <span class="cit-name-given-names">Ruan</span></span>, <span class="cit-auth"><span class="cit-name-surname">Nils</span>  <span class="cit-name-given-names">Homer</span></span>, <span class="cit-auth"><span class="cit-name-surname">Gabor</span>  <span class="cit-name-given-names">Marth</span></span>, <span class="cit-auth"><span class="cit-name-surname">Goncalo</span>  <span class="cit-name-given-names">Abecasis</span></span>, <span class="cit-auth"><span class="cit-name-surname">Richard</span>  <span class="cit-name-given-names">Durbin</span></span>, and <span class="cit-auth cit-collab">1000 Genome Project Data Processing Subgroup</span>. <span class="cit-pub-date">2009</span>. “<span class="cit-article-title">The Sequence Alignment/Map Format and SAMtools</span>.” <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">25</span> (<span class="cit-issue">16</span>): <span class="cit-fpage">2078</span>–<span class="cit-lpage">79</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtp352%26rft_id%253Dinfo%253Apmid%252F19505943%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/btp352&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19505943&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2016%2F09%2F19%2F075960.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000268808600014&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference  in text" id="ref-19">↵</a><div class="cit ref-cit ref-journal" id="cit-075960v1.19" data-doi="10.1111/mec.13446"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Machado</span></span>, <span class="cit-auth"><span class="cit-name-surname">Heather</span>  <span class="cit-name-given-names">E.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Alan</span>  <span class="cit-name-given-names">O.</span></span> <span class="cit-auth"><span class="cit-name-surname">Bergland</span></span>, <span class="cit-auth"><span class="cit-name-surname">Katherine</span>  <span class="cit-name-given-names">R.</span></span> <span class="cit-auth"><span class="cit-name-surname">O’</span>  <span class="cit-name-given-names">Brien</span></span>, <span class="cit-auth"><span class="cit-name-surname">Emily</span>  <span class="cit-name-given-names">L.</span></span> <span class="cit-auth"><span class="cit-name-surname">Behrman</span></span>, <span class="cit-auth"><span class="cit-name-surname">Paul</span>  <span class="cit-name-given-names">S.</span></span> <span class="cit-auth"><span class="cit-name-surname">Schmidt</span></span>, and <span class="cit-auth"> <span class="cit-name-given-names">Dmitri A.</span> <span class="cit-name-surname">Petrov</span></span>. <span class="cit-pub-date">2016</span>. “<span class="cit-article-title">Comparative Population Genomics of Latitudinal Variation in Drosophila Simulans and Drosophila Melanogaster</span>.” <abbr class="cit-jnl-abbrev">Molecular Ecology</abbr> <span class="cit-vol">25</span> (<span class="cit-issue">3</span>): <span class="cit-fpage">723</span>–<span class="cit-lpage">40</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2BEcology%26rft.volume%253D25%26rft.spage%253D723%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fmec.13446%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/mec.13446&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference  in text" id="ref-20">↵</a><div class="cit ref-cit ref-journal" id="cit-075960v1.20"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Saltz</span></span>, <span class="cit-auth"><span class="cit-name-surname">Julia</span>  <span class="cit-name-given-names">B.</span></span>, and <span class="cit-auth"> <span class="cit-name-given-names">Sergey V.</span> <span class="cit-name-surname">Nuzhdin</span></span>. <span class="cit-pub-date">2014</span>. “<span class="cit-article-title">Genetic Variation in Niche Construction: Implications for Development and Evolutionary Genetics</span>.” <abbr class="cit-jnl-abbrev">Trends in Ecology &amp; Evolution</abbr> <span class="cit-vol">29</span> (<span class="cit-issue">1</span>): <span class="cit-fpage">8</span>–<span class="cit-lpage">14</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTrends%2Bin%2BEcology%2B%2526%2BEvolution%26rft.volume%253D29%26rft.spage%253D8%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference  in text" id="ref-21">↵</a><div class="cit ref-cit ref-journal" id="cit-075960v1.21"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Sterken</span></span>, <span class="cit-auth"><span class="cit-name-surname">Mark</span>  <span class="cit-name-given-names">G.</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">L.</span> <span class="cit-name-surname">Basten</span></span> <span class="cit-auth"><span class="cit-name-surname">Snoek</span></span>, <span class="cit-auth"><span class="cit-name-surname">Jan</span>  <span class="cit-name-given-names">E.</span></span> <span class="cit-auth"><span class="cit-name-surname">Kammenga</span></span>, and <span class="cit-auth"> <span class="cit-name-given-names">Erik C.</span> <span class="cit-name-surname">Andersen</span></span>. <span class="cit-pub-date">2015</span>. “<span class="cit-article-title">The Laboratory Domestication of Caenorhabditis Elegans</span>.” <abbr class="cit-jnl-abbrev">Trends in Genetics: TIG</abbr> <span class="cit-vol">31</span> (<span class="cit-issue">5</span>): <span class="cit-fpage">224</span>–<span class="cit-lpage">31</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTrends%2Bin%2BGenetics%253A%2BTIG%26rft.volume%253D31%26rft.spage%253D224%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference  in text" id="ref-22">↵</a><div class="cit ref-cit ref-journal" id="cit-075960v1.22"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Taylor</span></span>, <span class="cit-auth"><span class="cit-name-surname">Michelle</span>  <span class="cit-name-given-names">L.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Alison</span>  <span class="cit-name-given-names">Skeats</span></span>, <span class="cit-auth"><span class="cit-name-surname">Alastair</span>  <span class="cit-name-given-names">J.</span></span> <span class="cit-auth"><span class="cit-name-surname">Wilson</span></span>, <span class="cit-auth"><span class="cit-name-surname">Tom</span>  <span class="cit-name-given-names">A.</span></span> <span class="cit-auth"> <span class="cit-name-given-names">R.</span> <span class="cit-name-surname">Price</span></span>, and <span class="cit-auth"><span class="cit-name-surname">Nina</span>  <span class="cit-name-given-names">Wedell</span></span>. <span class="cit-pub-date">2015</span>. “<span class="cit-article-title">Opposite Environmental and Genetic Influences on Body Size in North American Drosophila Pseudoobscura</span>.” <abbr class="cit-jnl-abbrev">BMC Evolutionary Biology</abbr> <span class="cit-vol">15</span> (March): <span class="cit-fpage">51</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2BEvolutionary%2BBiology%26rft.volume%253D15%26rft.spage%253D51%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference  in text" id="ref-23">↵</a><div class="cit ref-cit ref-journal" id="cit-075960v1.23"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Tyukmaeva</span></span>, <span class="cit-auth"><span class="cit-name-surname">Venera</span>  <span class="cit-name-given-names">I.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Paris</span>  <span class="cit-name-given-names">Veltsos</span></span>, <span class="cit-auth"><span class="cit-name-surname">Jon</span>  <span class="cit-name-given-names">Slate</span></span>, <span class="cit-auth"><span class="cit-name-surname">Emma</span>  <span class="cit-name-given-names">Gregson</span></span>, <span class="cit-auth"><span class="cit-name-surname">Hannele</span>  <span class="cit-name-given-names">Kauranen</span></span>, <span class="cit-auth"><span class="cit-name-surname">Maaria</span>  <span class="cit-name-given-names">Kankare</span></span>, <span class="cit-auth"><span class="cit-name-surname">Michael</span>  <span class="cit-name-given-names">G.</span></span> <span class="cit-auth"><span class="cit-name-surname">Ritchie</span></span>, <span class="cit-auth"><span class="cit-name-surname">Roger</span>  <span class="cit-name-given-names">K.</span></span> <span class="cit-auth"><span class="cit-name-surname">Butlin</span></span>, and <span class="cit-auth"><span class="cit-name-surname">Anneli</span>  <span class="cit-name-given-names">Hoikkala</span></span>. <span class="cit-pub-date">2015</span>. “<span class="cit-article-title">Localization of Quantitative Trait Loci for Diapause and Other Photoperiodically Regulated Life History Traits Important in Adaptation to Seasonally Varying Environments</span>.” <abbr class="cit-jnl-abbrev">Molecular Ecology</abbr> <span class="cit-vol">24</span> (<span class="cit-issue">11</span>): <span class="cit-fpage">2809</span>–<span class="cit-lpage">19</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2BEcology%26rft.volume%253D24%26rft.spage%253D2809%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.biorxiv.org/content/10.1101/075960v1.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-disqus-comment" >
  
      
  
  <div class="pane-content">
    <div id="disqus_thread"><noscript><p><a href="http://biorxivstage.disqus.com/?url=https%3A%2F%2Fwww.biorxiv.org%2Fcontent%2F10.1101%2F075960v1" class="" data-icon-position="" data-hide-link-title="0">View the discussion thread.</a></p></noscript></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/10.1101/076042v1" title="Low-frequency cortical oscillations are modulated by temporal prediction and temporal error coding" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous</span></a></span><span class="pager-next"><a href="/content/10.1101/075754v1" title="Clinically-weighted transcriptomic signatures for protein kinase inhibitor associated cardiotoxicity" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    Posted&nbsp;September 19, 2016.  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-biorxiv-art-tools" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-biorxiv_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/10.1101/075960v1.full.pdf" target="_self" class="article-dl-pdf-link link-icon"><i class="icon-external-link-sign"></i> <span class="title">Download PDF</span></a>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-biorxiv-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="Email this Article" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-biorxiv_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/10.1101/075960v1.full" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for your interest in spreading the word about bioRxiv.</p><p>NOTE: Your email address is requested solely to identify you as the sender of this article.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/10.1101/075960v1" class="active" data-icon-position="" data-hide-link-title="0">Correlations of genotype with climate parameters suggest Caenorhabditis elegans niche preferences</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from bioRxiv
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would like to see this page from the bioRxiv website.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/20978" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-wdqSvD3yt4zNhlVG2aZ_FCs99VAVozq5DeN01Spb2l8" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-biorxiv-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-biorxiv_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="20978" id="node20978--2" data-pisa="biorxiv;075960v1" data-pisa-master="biorxiv;075960" data-seqnum="20978" data-apath="/biorxiv/early/2016/09/19/075960.atom"><div  class="highwire-cite highwire-cite-highwire-article highwire-citation-biorxiv-article-pap-list clearfix" >

      <div class="highwire-cite-title" >
      <div class="highwire-cite-title">Correlations of genotype with climate parameters suggest <em>Caenorhabditis elegans</em> niche preferences</div>    </div>
  
      <div  class="highwire-cite-authors" ><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0"><span class="nlm-given-names">Kathryn S.</span> <span class="nlm-surname">Evans</span></span>, <span class="highwire-citation-author" data-delta="1"><span class="nlm-given-names">Yuehui</span> <span class="nlm-surname">Zhao</span></span>, <span class="highwire-citation-author" data-delta="2"><span class="nlm-given-names">Shannon C.</span> <span class="nlm-surname">Brady</span></span>, <span class="highwire-citation-author" data-delta="3"><span class="nlm-given-names">Lijiang</span> <span class="nlm-surname">Long</span></span>, <span class="highwire-citation-author" data-delta="4"><span class="nlm-given-names">Patrick T.</span> <span class="nlm-surname">McGrath</span></span>, <span class="highwire-citation-author" data-delta="5"><span class="nlm-given-names">Erik C.</span> <span class="nlm-surname">Andersen</span></span></span></div>
  
      <div  class="highwire-cite-metadata" ><span  class="highwire-cite-metadata-journal highwire-cite-metadata">bioRxiv </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">075960; </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata"><span class="doi_label">doi:</span> https://doi.org/10.1101/075960 </span></div>
  
  
  </div>
</div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.biorxiv.org/content/10.1101/075960v1" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.biorxiv.org/content/10.1101/075960v1&amp;title=Correlations%20of%20genotype%20with%20climate%20parameters%20suggest%20Caenorhabditis%20elegans%20niche%20preferences" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.biorxiv.org/content/10.1101/075960v1&amp;title=Correlations%20of%20genotype%20with%20climate%20parameters%20suggest%20Caenorhabditis%20elegans%20niche%20preferences" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.biorxiv.org/content/10.1101/075960v1&amp;text=Correlations%20of%20genotype%20with%20climate%20parameters%20suggest%20Caenorhabditis%20elegans%20niche%20preferences" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.biorxiv.org/content/10.1101/075960v1&amp;title=Correlations%20of%20genotype%20with%20climate%20parameters%20suggest%20Caenorhabditis%20elegans%20niche%20preferences" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.biorxiv.org/content/10.1101/075960v1&amp;t=Correlations%20of%20genotype%20with%20climate%20parameters%20suggest%20Caenorhabditis%20elegans%20niche%20preferences" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.biorxiv.org/content/10.1101/075960v1&amp;title=Correlations%20of%20genotype%20with%20climate%20parameters%20suggest%20Caenorhabditis%20elegans%20niche%20preferences" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.biorxiv.org/content/10.1101/075960v1&amp;title=Correlations%20of%20genotype%20with%20climate%20parameters%20suggest%20Caenorhabditis%20elegans%20niche%20preferences" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-biorxiv-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title="Citation Tools"><i class="icon-globe"></i> <span class="title">Citation Tools</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-biorxiv_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="20978" id="node20978--3" data-pisa="biorxiv;075960v1" data-pisa-master="biorxiv;075960" data-seqnum="20978" data-apath="/biorxiv/early/2016/09/19/075960.atom"><div  class="highwire-cite highwire-cite-highwire-article highwire-citation-biorxiv-article-pap-list clearfix" >

      <div class="highwire-cite-title" >
      <div class="highwire-cite-title">Correlations of genotype with climate parameters suggest <em>Caenorhabditis elegans</em> niche preferences</div>    </div>
  
      <div  class="highwire-cite-authors" ><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0"><span class="nlm-given-names">Kathryn S.</span> <span class="nlm-surname">Evans</span></span>, <span class="highwire-citation-author" data-delta="1"><span class="nlm-given-names">Yuehui</span> <span class="nlm-surname">Zhao</span></span>, <span class="highwire-citation-author" data-delta="2"><span class="nlm-given-names">Shannon C.</span> <span class="nlm-surname">Brady</span></span>, <span class="highwire-citation-author" data-delta="3"><span class="nlm-given-names">Lijiang</span> <span class="nlm-surname">Long</span></span>, <span class="highwire-citation-author" data-delta="4"><span class="nlm-given-names">Patrick T.</span> <span class="nlm-surname">McGrath</span></span>, <span class="highwire-citation-author" data-delta="5"><span class="nlm-given-names">Erik C.</span> <span class="nlm-surname">Andersen</span></span></span></div>
  
      <div  class="highwire-cite-metadata" ><span  class="highwire-cite-metadata-journal highwire-cite-metadata">bioRxiv </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">075960; </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata"><span class="doi_label">doi:</span> https://doi.org/10.1101/075960 </span></div>
  
  
  </div>
</div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage&amp;rft.epage&amp;rft.atitle=Correlations%20of%20genotype%20with%20climate%20parameters%20suggest%20Caenorhabditis%20elegans%20niche%20preferences&amp;rft.volume&amp;rft.issue&amp;rft.date=2016-01-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=bioRxiv&amp;rft.au=Evans%2C+Kathryn+S.&amp;rft.au=Zhao%2C+Yuehui&amp;rft.au=Brady%2C+Shannon+C.&amp;rft.au=Long%2C+Lijiang&amp;rft.au=McGrath%2C+Patrick+T.&amp;rft.au=Andersen%2C+Erik+C."></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/20978/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/20978/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/20978/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/20978/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/20978/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/20978/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/20978/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/20978/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/20978/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/20978/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/20978/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/20978/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><div class="item-list"><ul><li class="first"><a href="http://twitter.com/share?url=https%3A//www.biorxiv.org/content/10.1101/075960v1&amp;count=horizontal&amp;via=&amp;text=Correlations%20of%20genotype%20with%20climate%20parameters%20suggest%20Caenorhabditis%20elegans%20niche%20preferences&amp;counturl=https%3A//www.biorxiv.org/content/10.1101/075960v1" class="twitter-share-button service-links-twitter-widget" id="twitter_widget" title="Tweet This" rel="nofollow" data-icon-position="" data-hide-link-title="0"><span class="element-invisible">Tweet Widget</span></a></li><li><a href="http://www.facebook.com/plugins/like.php?href=https%3A//www.biorxiv.org/content/10.1101/075960v1&amp;layout=button_count&amp;show_faces=false&amp;action=like&amp;colorscheme=light&amp;width=100&amp;height=21&amp;font=&amp;locale=" id="facebook_like" title="I Like it" class="service-links-facebook-like" rel="nofollow" data-icon-position="" data-hide-link-title="0"><span class="element-invisible">Facebook Like</span></a></li><li class="last"><a href="https://www.biorxiv.org/content/10.1101/075960v1" id="google_plus_one" title="Plus it" class="service-links-google-plus-one" rel="nofollow" data-icon-position="" data-hide-link-title="0"><span class="element-invisible">Google Plus One</span></a></li></ul></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-collections" >
  
        <h2 class="pane-title">Subject Area</h2>
    
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-collections"><div class="highwire-list"><ul class="highwire-article-collection-term-list"><li class="first last odd"><span class="highwire-article-collection-term"><a href="/collection/genetics" class="highlight" data-icon-position="" data-hide-link-title="0">Genetics<i class="icon-caret-right"></i>
</a></span></li></ul></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-biorxiv-subject-collections block-style-col2" >
  
      
  
  <div class="pane-content">
    <div class="panel-flexible panels-flexible-8 clearfix" id="mini-panel-biorxiv_subject_collections">
<div class="panel-flexible-inside panels-flexible-8-inside">
<div class="panels-flexible-region panels-flexible-region-8-center panels-flexible-region-first panels-flexible-region-last">
  <div class="inside panels-flexible-region-inside panels-flexible-region-8-center-inside panels-flexible-region-inside-first panels-flexible-region-inside-last">
<div class="panel-pane pane-snippet" >
  
      
  
  <div class="pane-content">
    <div class="snippet biorxiv-subject-areas-table-title" id="biorxiv-subject-areas-table-title">
  
      
  <div class="snippet-content">
    <b>Subject Areas</b>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-snippet" >
  
      
  
  <div class="pane-content">
    <div class="snippet biorxiv-subject-areas-view-papers" id="biorxiv-subject-areas-view-papers">
  
      
  <div class="snippet-content">
    <a href="/content/early/recent"><strong>All Articles</strong></a>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-subject-collections" >
  
      
  
  <div class="pane-content">
    <ul id="collection" class="collection highwire-list-expand"><li class="outer collection depth-2  child first"><div class = "data-wrapper"><a href="/collection/animal-behavior-and-cognition" class="" data-icon-position="" data-hide-link-title="0">Animal Behavior and Cognition</a> <span class = "article-count">(1662)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/biochemistry" class="" data-icon-position="" data-hide-link-title="0">Biochemistry</a> <span class = "article-count">(2789)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/bioengineering" class="" data-icon-position="" data-hide-link-title="0">Bioengineering</a> <span class = "article-count">(1946)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/bioinformatics" class="" data-icon-position="" data-hide-link-title="0">Bioinformatics</a> <span class = "article-count">(10400)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/biophysics" class="" data-icon-position="" data-hide-link-title="0">Biophysics</a> <span class = "article-count">(4243)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/cancer-biology" class="" data-icon-position="" data-hide-link-title="0">Cancer Biology</a> <span class = "article-count">(3272)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/cell-biology" class="" data-icon-position="" data-hide-link-title="0">Cell Biology</a> <span class = "article-count">(4641)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/clinical-trials" class="" data-icon-position="" data-hide-link-title="0">Clinical Trials</a> <span class = "article-count">(135)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/developmental-biology" class="" data-icon-position="" data-hide-link-title="0">Developmental Biology</a> <span class = "article-count">(2889)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/ecology" class="" data-icon-position="" data-hide-link-title="0">Ecology</a> <span class = "article-count">(4544)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/epidemiology" class="" data-icon-position="" data-hide-link-title="0">Epidemiology</a> <span class = "article-count">(2041)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/evolutionary-biology" class="" data-icon-position="" data-hide-link-title="0">Evolutionary Biology</a> <span class = "article-count">(7288)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/genetics" class="" data-icon-position="" data-hide-link-title="0">Genetics</a> <span class = "article-count">(5540)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/genomics" class="" data-icon-position="" data-hide-link-title="0">Genomics</a> <span class = "article-count">(6893)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/immunology" class="" data-icon-position="" data-hide-link-title="0">Immunology</a> <span class = "article-count">(2434)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/microbiology" class="" data-icon-position="" data-hide-link-title="0">Microbiology</a> <span class = "article-count">(7616)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/molecular-biology" class="" data-icon-position="" data-hide-link-title="0">Molecular Biology</a> <span class = "article-count">(3051)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/neuroscience" class="" data-icon-position="" data-hide-link-title="0">Neuroscience</a> <span class = "article-count">(18885)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/paleontology" class="" data-icon-position="" data-hide-link-title="0">Paleontology</a> <span class = "article-count">(142)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/pathology" class="" data-icon-position="" data-hide-link-title="0">Pathology</a> <span class = "article-count">(477)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/pharmacology-and-toxicology" class="" data-icon-position="" data-hide-link-title="0">Pharmacology and Toxicology</a> <span class = "article-count">(791)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/physiology" class="" data-icon-position="" data-hide-link-title="0">Physiology</a> <span class = "article-count">(1178)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/plant-biology" class="" data-icon-position="" data-hide-link-title="0">Plant Biology</a> <span class = "article-count">(2771)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/scientific-communication-and-education" class="" data-icon-position="" data-hide-link-title="0">Scientific Communication and Education</a> <span class = "article-count">(686)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/synthetic-biology" class="" data-icon-position="" data-hide-link-title="0">Synthetic Biology</a> <span class = "article-count">(905)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/systems-biology" class="" data-icon-position="" data-hide-link-title="0">Systems Biology</a> <span class = "article-count">(2881)</span></div></li>
<li class="outer collection depth-2  child last"><div class = "data-wrapper"><a href="/collection/zoology" class="" data-icon-position="" data-hide-link-title="0">Zoology</a> <span class = "article-count">(472)</span></div></li>
</ul>  </div>

  
  </div>
  </div>
</div>
</div>
</div>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__VNH5GD8Zz7g6_hGCOZjVjdMndxxC6naiExSSJKX3k_A__kzmEdtbqCsJx5Z9yg2Qy0PptB__ufXm-TxUOl0KZzUw__QJacVmveFyLIBSPkp21xL0ZvuDW9DMBrD2HB1P4Ry5I.js"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__QJacVmveFyLIBSPkp21xL0ZvuDW9DMBrD2HB1P4Ry5I.js" defer="defer"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__N7ERJBYsOWyRnJgyoM125_Aiez2MOJGaUofG1JdWWBg__bLlcwhzZhT-IwoQ_dvm1_xPxU7E7gecFLC6Fyeqtv8s__QJacVmveFyLIBSPkp21xL0ZvuDW9DMBrD2HB1P4Ry5I.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
function euCookieComplianceLoadScripts() {}
//--><!]]>
</script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var eu_cookie_compliance_cookie_name = "";
//--><!]]>
</script>
<script type="text/javascript" src="//d33xdlntwy0kbs.cloudfront.net/cshl_custom.js"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__BmqjBnkz3MgYCAoc25s1lDRMEjLhC3mEPVonUFIHi08__Unwv5-ZIuHBfFwytsjEx1niBVJ7n1T4lPws7VrkHXM4__QJacVmveFyLIBSPkp21xL0ZvuDW9DMBrD2HB1P4Ry5I.js"></script>
  </body>
</html>
