<html xmlns="http://www.w3.org/1999/xhtml"><head> <style type="text/css">
         	     		 
         	    a                          {background : #ffffff; }
         	    article                    {border-style : dotted; border-width : 2px; }
         		 	
         		 	div                        {background : #ffffcc;}
         		 	div.abstract-title         {font-weight : bold ; font-size : 16pt;}
         		 	div.ack                    {border-style : solid ; border-color : red; margin
         : 2em; }
         		 	div.article-meta           {border-style : solid; border-width : 1 pt; margin
         1em;}
         		 	div.boxed-text             {margin : 5em; border-style : solid;}
         		 	div.contrib-group          {margin : 1em; }
         		 	div.fig                    {border-style : solid; border-width : 2px; margin :
         2em; }
         		 	div.funding                {font-weight : bold ; font-size : 16pt;}
         		 	div.given-names            {font-style : italic;}
         		 	div.intro                  {border-style : inset; margin : 5px;}
         		 	div.introduction           {border-style : inset; margin : 5px;}
         		 	div.journal-meta           {border-style : solid; border-width : 1 pt; margin
         1em;}
         		 	div.journal-title-group    {background : #ffeeee;}
         		 	div.article-title          {font-weight : bold ; font-size : 18pt;}
         		 	div.kwd                    {font-style : italic}
         		 	div.meta-name              {font-weight : bold ; font-size : 16pt;}
         		 	div.name                   {font-weight : bold;}
         		 	div.alt-title              {font-style : italic; font-size : 12px;}
         		 	
         		 	div.sec                    {border : 2px; margin 5px; padding 2px;}
         		 	div.title                  {font-family : courier; font-weight : bold;
         		 	                            font-size : 16pt; margin : 5px;}
         		 	
         		 	div.materials_methods      {border-style : double; margin : 5px; }
         		 	div.methods                {border-style : double; margin : 5px; }
         		 	
         		 	div.results                {border-style : solid; margin : 5px;}
         		 	div.background             {border-style : dotted; margin : 5px;}
         		 	
         		 	div.discussion             {border-style : groove; margin : 5px;}
         		 	div.conclusion             {border-style : ridge; margin : 5px;}
         		 	div.conclusions             {border-style : ridge; margin : 5px;}
         		 	div.supplementary-material {border-style : inset; margin : 5px;}
         		 	div.abbreviations          {border-style : double; border-color : red; margin
         : 5px;}
         		 	div.competinginterests     {border-style : double; border-color : blue; margin
         : 5px;}
         		 	div.acknowledgements       {border-style : double; border-color : green; margin
         : 5px;}
         		 	div.authors_contributions   {border-style : double; border-color : purple; margin
         : 5px;}
         		 	
         		 	div.publisher              {border-style : outset; margin : 5px;}
         		 	div.fn-type-conflict       {background : #f88; }
         		 	div.fn-type-con            {background : #ddf; }
         		 	div.fn-type-other          {background : #ddd; }
         		 	
         		 	div.unknown                {background : #ffd;
         									 	  border-style : solid;
         									 	  border-width : 1px;
         									 	  padding : 2 px;}
         		 	  
         	    table                      {background : #ffffdd;}
         		 	tr                         {background : #ddddff; padding : 1px;}
         		 	
         		 	span                       {background : #ffcccc;}
         		 	
         		 	span.citation-author       {font-family : helvetica; background : #ffeeee;}
         		 	span.collab                {background : #ddffff; }
         		 	span.comment               {font-family : courier; font-size : 6px; background
         : #ffaaff;}
         		 	span.contrib               {background : #ffffff;}
         		 	span.corresp               {background : #ddffdd; }
         		 	span.doi                   {background : #ffffff;}
         		 	span.email                 {font-family : courier; }
         		 	span.etal                  {font-style : italic;}
         		 	span.fpage                 {font-family : courier;}
         		 	span.given-names           {background : #ffffff;}
         		 	span.iso-abbrev            {background : #ffffff;}
         		 	span.issn-epub             {background : #ffffff;}
         		 	span.issn-ppub             {background : #ffffff;}
         		 	span.journal-title         {background : #ffffff;}
         		 	span.lpage                 {font-family : courier;}
         		 	span.mixed-article-title   {font-style : italic ;}
         		 	span.nlm-ta                {background : #ffffff;}
         		 	span.pmc                   {background : #ffffff;}
         		 	span.pmcid                 {background : #ffffff;}
         		 	span.pmid                  {background : #ffffff;}
         		 	span.publisher             {background : #ffffff;}
         		 	span.publisher-id          {background : #ffffff;}
         		 	span.publisher-name        {background : #ffffff;}
         		 	span.source                {background : #ffffff;}
         		 	span.subject               {background : #ffffff;}
         		 	span.surname               {background : #ffffff;}
         		 	span.volume                {font-family : courier; font-weight : bold;}
         		 	span.year                  {font-family : courier ; font-style : italic;}
         			
      </style> </head> <body> <div class="front" title="front"> <div class="journal-meta" tagx="journal-meta" title="journal-meta"> <span class="nlm-ta" title="nlm-ta">PLoS Biol</span> <span class="iso-abbrev" title="iso-abbrev">PLoS Biol</span> <span class="publisher-id" title="publisher-id">plos</span> <span class="pmc" title="pmc">plosbiol</span> <div class="journal-title-group" tagx="journal-title-group" title="journal-title-group"> <span class="journal-title" tagx="journal-title" title="journal-title">PLoS Biology</span> </div> <span class="issn-ppub" tagx="issn" title="issn-ppub">1544-9173</span> <span class="issn-epub" tagx="issn" title="issn-epub">1545-7885</span> <div class="publisher" tagx="publisher" title="publisher"> <span class="publisher-name" tagx="publisher-name" title="publisher-name">Public Library of Science</span> <span class="publisher-loc" tagx="publisher-loc" title="publisher-loc">San Francisco, USA</span> </div> </div> <div class="article-meta" tagx="article-meta" title="article-meta"> <span class="pmcid" title="pmcid"> pmcid: <a href="http://www.ncbi.nlm.nih.gov/pubmed/3071376">3071376</a> </span> <span class="pmid" title="pmid"> pmid: <a href="http://www.ncbi.nlm.nih.gov/pubmed/21483722">21483722</a> </span> <span class="publisher-id" title="publisher-id">PBIOLOGY-D-10-01375</span> <span class="doi" title="doi"> doi: <a href="https://dx.doi.org/10.1371/journal.pbio.1001040">10.1371/journal.pbio.1001040</a> </span> <div class="article-categories" title="article-categories">
: <span class="subject" title="subject">Research Article</span>: <span class="subject" title="subject">Biology</span> </div> <div class="title-group"> <div class="article-title" title="article-title">
A Chaperonin Subunit with Unique Structures Is Essential for Folding of a Specific Substrate </div> <div class="alt-title" title="alt-title">
Function of Cpn60β4 on Folding of NdhH </div> </div> <div class="contrib-group" title="contrib-group"> <span class="contrib" title="contrib"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Peng</span> <span class="given-names" tagx="given-names" title="given-names">Lianwei</span> </span> <a href="#aff1"> <sup>1</sup> </a> </span> <span class="contrib" title="contrib"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Fukao</span> <span class="given-names" tagx="given-names" title="given-names">Yoichiro</span> </span> <a href="#aff2"> <sup>2</sup> </a> </span> <span class="contrib" title="contrib"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Myouga</span> <span class="given-names" tagx="given-names" title="given-names">Fumiyoshi</span> </span> <a href="#aff3"> <sup>3</sup> </a> </span> <span class="contrib" title="contrib"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Motohashi</span> <span class="given-names" tagx="given-names" title="given-names">Reiko</span> </span> <a href="#aff4"> <sup>4</sup> </a> </span> <span class="contrib" title="contrib"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Shinozaki</span> <span class="given-names" tagx="given-names" title="given-names">Kazuo</span> </span> <a href="#aff3"> <sup>3</sup> </a> </span> <span class="contrib" title="contrib"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Shikanai</span> <span class="given-names" tagx="given-names" title="given-names">Toshiharu</span> </span> <a href="#aff1"> <sup>1</sup> </a> <a href="#cor1"> <sup>*</sup> </a> </span> </div> <span class="citation_author_institution" id="aff1">[1], <span class="addr-line" title="addr-line">Department of Botany, Graduate School of Science, Kyoto University, Sakyo-ku, Kyoto, Japan </span> </span> <span class="citation_author_institution" id="aff2">[2], <span class="addr-line" title="addr-line">Plant Global Educational Project, Graduate School of Biological Sciences, Nara Institute of Science and Technology, Takayama, Ikoma, Nara, Japan </span> </span> <span class="citation_author_institution" id="aff3">[3], <span class="addr-line" title="addr-line">Gene Discovery Research Group, RIKEN Plant Science Center, Yokohama, Japan </span> </span> <span class="citation_author_institution" id="aff4">[4], <span class="addr-line" title="addr-line">Faculty of Agriculture, University of Shizuoka, Shizuoka, Japan </span> </span> <div class="contrib-group" title="contrib-group"> <span class="contrib" title="contrib"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Horwich</span> <span class="given-names" tagx="given-names" title="given-names">Arthur L.</span> </span> <span class="role" tagx="role" title="role">Academic Editor</span> <a href="#edit1" /> </span> </div> <span class="citation_author_institution" id="edit1">[], </span> <div class="author-notes" title="author-notes"> <div class="corresp" title="corresp">
* E-mail: <span class="email" tagx="email" title="email">shikanai@pmg.bot.kyoto-u.ac.jp</span> </div> <div class="fn-type-con" title="con"> <p>The author(s) have made the following declarations about their contributions: Conceived and designed the experiments: LP TS. Performed the experiments: LP YF. Analyzed the data: LP YF . Contributed reagents/materials/analysis tools: FM RM KS. Wrote the paper: LP TS. </p> </div> </div> <span class="pub-date-collection" title="pub-date-collection">collection: <span>2011-4</span> </span> <span class="pub-date-epub" title="pub-date-epub">epub: <span>2011-4-4</span> </span> <span class="pub-date-pmc-release" title="pub-date-pmc-release">pmc-release: <span>2011-4-4</span> </span> <span class="volume" tagx="volume" title="volume">9</span> <span class="issue" tagx="issue" title="issue">4</span> <span class="elocation-id" tagx="elocation-id" title="elocation-id">e1001040</span> <span class="history" title="history"> <span class="received" title="received">received: 2010-12-20</span> <span class="accepted" title="accepted">accepted: 2011-2-23</span> </span> <div class="permissions"> <span class="copyright" title="copyright">(C) , <span class="copyright-year" tagx="copyright-year" title="copyright-year">2011</span> </span> <span class="license" title="license"> <span class="license-p" title="license-p">This is an open-access article distributed under the terms of the Creative Commons Attribution License, which permits unrestricted use, distribution, and reproduction in any medium, provided the original author and source are properly credited. </span> </span> </div> <div class="abstract" title="abstract"> <div class="abstract-title" title="abstract-title">
Abstract</div> <p>Type I chaperonins are large, double-ring complexes present in bacteria (GroEL), mitochondria (Hsp60), and chloroplasts (Cpn60), which are involved in mediating the folding of newly synthesized, translocated, or stress-denatured proteins. In <i>Escherichia coli</i>, GroEL comprises 14 identical subunits and has been exquisitely optimized to fold its broad range of substrates. However, multiple Cpn60 subunits with different expression profiles have evolved in chloroplasts. Here, we show that, in <i>Arabidopsis thaliana</i>, the minor subunit Cpn60β4 forms a heterooligomeric Cpn60 complex with Cpn60α1 and Cpn60β1–β3 and is specifically required for the folding of NdhH, a subunit of the chloroplast NADH dehydrogenase-like complex (NDH). Other Cpn60β subunits cannot complement the function of Cpn60β4. Furthermore, the unique C-terminus of Cpn60β4 is required for the full activity of the unique Cpn60 complex containing Cpn60β4 for folding of NdhH. Our findings suggest that this unusual kind of subunit enables the Cpn60 complex to assist the folding of some particular substrates, whereas other dominant Cpn60 subunits maintain a housekeeping chaperonin function by facilitating the folding of other obligate substrates. </p> </div> <div class="abstract" title="abstract"> <div class="title" tagx="title" title="title">
Author Summary</div> <p>Chaperonins assist the folding of some nascent and denatured proteins to their native, functional forms. Each chaperonin consists of a pair of protein complexes resembling two stacked toroids; folding occurs inside the toroid cavity. Chaperonins are ubiquitous in both bacteria and more complex nucleated cells, as well as in the intracellular organelles that have evolved from bacteria by endosymbiosis: mitochondria and, in plants, chloroplasts. They are indispensable for cellular function. Many different chaperonin subunits have evolved in various species of bacteria as well as in most mitochondria and chloroplasts. The physiological and functional relevance of these multiple chaperonin subunits is poorly understood, however. In this study, we have characterized the minor chaperonin subunit Cpn60β4 from <i>Arabidopsis</i> chloroplasts, which differs in structure from other chloroplast chaperonins. When the <i>Cpn60β4</i> gene is defective, the plants fail to accumulate one protein complex in particular: the chloroplast NADH dehydrogenase-like complex (NDH). We discovered that Cpn60β4 forms a complex with other Cpn60 α and <i>β</i> subunits and that this complex is essential for the folding of the NDH subunit NdhH. Cpn60β4 has a unique protein "tail" that is required for the efficient folding of NdhH. Our findings suggest that Cpn60β4 has evolved with distinctive structural features that facilitate the folding of one specific substrate and that this strategy is used by plants to satisfy their conflicting requirements for chaperonins with both specialized and general functions. </p> </div> <div class="counts" title="counts"> <span class="page-count" title="page-count">page-count: </span> </div> </div> </div> <div class="body" title="body"> <div class="introduction" title="sec"> <div class="title" tagx="title" title="title">
Introduction</div> <p>Chaperonins are large double-ring assemblies that assist in the efficient folding of substrate proteins (reviewed in <a href="#pbio.1001040-Frydman1">[1]</a>– <a href="#pbio.1001040-Fenton1">[3]</a>). Two types of chaperonins have been identified: type I in bacteria (GroEL), mitochondria (Hsp60), and chloroplasts (Cpn60), and type II in archaea (thermosome) and eukaryotic (TRiC/CCT) cytosol (reviewed in <a href="#pbio.1001040-Horwich1">[4]</a>). Whereas a type I chaperonin ring is composed of seven subunits, a type II chaperonin ring consists of eight or nine subunits that are not identical but are homologous. Type I chaperonin requires co-chaperonin GroES/Hsp10 for substrate encapsulation, whereas type II chaperonin is independent of GroES/Hsp10 factors. Both types of chaperonins utilize ATP as energy to drive a series of structural rearrangements that allow them to capture, encapsulate, and release the substrate proteins (reviewed in <a href="#pbio.1001040-Horwich1">[4]</a>). </p> <p>The GroEL/GroES complex from <i>Escherichia coli</i> ( <i>E. coli </i>) represents the type I chaperonins and its structure and function have been studied extensively (reviewed in <a href="#pbio.1001040-Horwich1">[4]</a>, <a href="#pbio.1001040-Hartl2">[5]</a>). GroEL consists of 14 identical subunits of ∼57 kDa and these subunits form two heptameric rings stacked back-to-back with a central cavity in each ring <a href="#pbio.1001040-Braig1">[6]</a>. Each subunit contains three domains. An equatorial domain comprises the ATP/ADP binding site and an apical domain contains the hydrophobic surface toward the ring cavity for polypeptide binding. The intermediate domain links the equatorial and apical domains <a href="#pbio.1001040-Braig1">[6]</a>, <a href="#pbio.1001040-Saibil1">[7]</a>. The co-chaperonin GroES is a homoheptameric single-ring composed of 10 kDa subunits <a href="#pbio.1001040-Xu1">[8]</a>. GroES can rapidly bind the substrate-captured GroEL ring ( <i>cis</i> ring) in the presence of ATP; hence, the GroEL/GroES complex provides an encapsulated cavity for protein folding <a href="#pbio.1001040-Mayhew1">[9]</a>, <a href="#pbio.1001040-Weissman1">[10]</a>. Due to structural rearrangements in the apical and intermediate domains, the <i>cis</i> cavity becomes enlarged and the physical features of the cavity wall change. This process lasts about 10–15 s and is accompanied by the hydrolysis of seven ATP molecules. After hydrolysis, ATP and other non-native peptides bind to the GroEL in the <i>trans</i> ring, triggering dissociation of the GroES from the opposite ring. The folded protein is then released from the chaperonin complex (reviewed in <a href="#pbio.1001040-Frydman1">[1]</a>– <a href="#pbio.1001040-Hartl2">[5]</a>). </p> <p>Proteome-wide analysis of chaperonin-dependent protein folding has shown that GroEL interacts with about 250 different proteins and these substrates are categorized into three classes <a href="#pbio.1001040-Kerner1">[11]</a>. The class I substrates are independent of GroEL/GroES, whereas class II substrates are partially dependent on GroEL/GroES, and they can utilize other chaperone systems, such as DnaK, for folding. A total of 84 proteins are grouped into class III and they are potential obligate substrates of GroEL/GroES in vivo <a href="#pbio.1001040-Kerner1">[11]</a>. More recently, Fujiwara et al. <a href="#pbio.1001040-Fujiwara1">[12]</a> employed a more direct approach by testing the solubility of class III substrates in GroE-depleted cells and found that only ∼60% (49 out of 84) of the class III proteins are absolutely dependent on GroEL/GroES for folding. Furthermore, an additional eight proteins that were not identified as class III proteins were also found to be GroEL/GroES obligate substrates and the authors defined these 57 proteins as class IV obligate substrates <a href="#pbio.1001040-Fujiwara1">[12]</a>. The majority of the class IV proteins are involved in metabolic reactions. Bioinformatic analysis has shown that nearly half of the class IV proteins contain TIM-barrel folds. In addition to these TIM-barrel folds, FAD/NAD(P)-binding domains, PLP-dependent transferase-like folds, and thiolase folds are also highly enriched in the group <a href="#pbio.1001040-Fujiwara1">[12]</a>. These data suggest that GroEL/GroES has been optimized to facilitate the folding of a variety of substrates during evolution. </p> <p>The basic features of the mechanisms for GroEL/GroES-mediated folding of the nonnative substrates have been demonstrated by a great number of functional and structural studies. However, these studies have focused primarily on the model chaperonin system that is composed of uniform subunits, such as GroEL from <i>E. coli</i>. In contrast to <i>E. coli</i>, nearly 30% of bacterial genomes contain two or more chaperonin genes <a href="#pbio.1001040-Lund1">[13]</a>. Furthermore, almost all mitochondria and chloroplasts studied in higher plants possess multiple chaperonin subunits <a href="#pbio.1001040-Hill1">[14]</a>. There have been few reports focusing on the role played by multiple chaperonin genes. In <i>Sinorhizobium meliloti</i>, one of the five GroEL paralogs, GroEL1 was shown to be required for NodD protein folding <a href="#pbio.1001040-Ogawa1">[15]</a>. However, overexpression of another GroEL protein can suppress the defect of the <i>groEL1</i> mutant <a href="#pbio.1001040-Ogawa1">[15]</a>. There are similar reports in other bacteria; <i>Bradyrhizobium japonicum</i> possesses at least five highly conserved <i>groESL</i> operons. Although nitrogen fixation activity was reduced to approximately 5% of the wild-type (WT) level in the double mutant defective in <i>groEL3</i> and <i>groEL4</i>, overexpression of two of the other <i>groESL</i> operons partially suppressed this phenotype <a href="#pbio.1001040-Fischer1">[16]</a>. Of the three chaperonin genes of <i>Rhizobium leguminosarum</i>, only <i>Cpn60.1</i> is essential for growth. Overexpression of the <i>Cpn60.3</i> gene in the <i>cpn60.1</i> mutant sustains bacterial growth, but the complemented strain is sensitive to high temperature, suggesting that Cpn60.3 does not facilitate the folding of particular proteins supporting the growth of bacteria at high temperature <a href="#pbio.1001040-Gould1">[17]</a>. By contrast, the specificity of GroEL1 function in <i>Mycobacterium smegmatis</i> seems to be absolute. This chaperonin is not essential for growth but is required for mycolic acid biosynthesis during mature biofilm formation <a href="#pbio.1001040-Ojha1">[18]</a>. GroEL1 may be specifically involved in the folding of two proteins, KasA and SmEG4308, which are required for mycolic acid synthesis, or in converting KasA between two isoforms <a href="#pbio.1001040-Ojha1">[18]</a>. These lines of evidence suggest that the functions of the multiple chaperonin subunits are specialized, although they have different degrees of specificity. </p> <p>Chloroplast type I chaperonin complex (Cpn60) is similar in structure to GroEL and also consists of two stacked heptameric rings <a href="#pbio.1001040-Pushkin1">[19]</a>, <a href="#pbio.1001040-Tsuprun1">[20]</a>. In contrast to GroEL, which is composed of identical subunits, Cpn60 comprises two different subunit types, Cpn60α and Cpn60β <a href="#pbio.1001040-Hemmingsen1">[21]</a>– <a href="#pbio.1001040-Martel1">[23]</a>, and they are only approximately 50% identical to each other <a href="#pbio.1001040-Hill1">[14]</a>. In vitro reconstitution studies of the chloroplast Cpn60 complex suggested a stoichiometry of α7β7 in the Cpn60 complex <a href="#pbio.1001040-Dickson1">[24]</a>, which is in accordance with the observation that roughly equal amounts of α and β subunits are present in chaperonin oligomers purified from spinach chloroplasts <a href="#pbio.1001040-Nishio1">[25]</a>. However, it is still unclear how these subunits are organized within a complex. Furthermore, the <i>Arabidopsis thaliana</i> genome contains two genes encoding Cpn60α subunits and four genes encoding Cpn60β subunits <a href="#pbio.1001040-Hill1">[14]</a>, and they have different expression profiles <a href="#pbio.1001040-Weiss1">[26]</a>, <a href="#pbio.1001040-Peltier1">[27]</a>. Cpn60α1 (At2g28000), Cpn60β1 (At1g55490), and Cpn60β2 (At3g13470) are the dominant Cpn60 subunits, whereas Cpn60α2 (At5g18820), Cpn60β3 (At5g56500), and Cpn60β4 (At1g26230) are present at very low levels. Disruption of the <i>Cpn60α1</i> gene results in general defects in plastid function, leading to embryonic lethality <a href="#pbio.1001040-Apuya1">[28]</a>, which highlights the critical role of Cpn60α1 in maintaining plastid function. The <i>cpn60β1β2</i> double mutant also shows the lethal phenotype <a href="#pbio.1001040-Suzuki1">[29]</a>, suggesting that Cpn60α1 and Cpn60β1–β2 form a heterooligomer that provides the housekeeping chaperonin function in chloroplasts by assisting the folding of a wide range of proteins. </p> <p>Multiple subunits also occur in type II chaperonin CCT and certain subunits are responsible for the binding of specific substrates, such as actin and tubulin <a href="#pbio.1001040-Llorca1">[30]</a>. Recently, it has been suggested that different subunits of CCT play unique roles in determining substrate specificity <a href="#pbio.1001040-Amit1">[31]</a>. However, few reports have focused on the function of the multiple subunits in the chloroplast chaperonin system. In particular, whereas the amino acid sequences of the Cpn60β1–β3 subunits share 90%–95% identity, Cpn60β4 is only 60% identical to each of the other three Cpn60β subunits <a href="#pbio.1001040-Hill1">[14]</a>. So far, there has been no explanation why plants evolved this unusual kind of Cpn60β subunit. In this study, we showed that Cpn60β4 is strictly and specifically required for the folding of the NdhH protein, a subunit of the chloroplast NADH dehydrogenase-like complex (NDH). </p> </div> <div class="results" title="sec"> <div class="title" tagx="title" title="title">
Results</div> <div class="thearabidopsiscrr27mutantisspecificallydefectiveinndhactivity" title="sec"> <div class="title" tagx="title" title="title">
The <i>Arabidopsis crr27</i> Mutant Is Specifically Defective in NDH Activity </div> <p>NDH is a multi-subunit complex embedded in the thylakoid membrane and is involved in chlororespiration and photosystem I (PSI) cyclic electron transport ( <a href="#pbio-1001040-g001">Figure 1A</a>) <a href="#pbio.1001040-Shikanai1">[32]</a>. The activity of NDH can be monitored as a post-illumination rise in chlorophyll fluorescence ( <a href="#pbio-1001040-g001">Figure 1B</a>) due to reduction of the plastoquinone (PQ) pool by the NDH complex in the dark <a href="#pbio.1001040-Shikanai2">[33]</a>. Based on this phenomenon, we isolated dozens of <i>Arabidopsis</i> mutants specifically defective in NDH activity, which we referred to as <i>chlororespiratory reduction</i> ( <i>crr</i>) mutants. Characterization of these mutants led to the identification of several NDH subunits and a large body of proteins involved in the expression of subunit genes and the assembly or stabilization of the NDH complex (reviewed in <a href="#pbio.1001040-Suorsa1">[34]</a>, <a href="#pbio.1001040-Peng1">[35]</a>). Here, we identify two mutants, <i>cpn60α1</i> and <i>crr27</i>, neither of which showed the increase in fluorescence after actinic light (AL) illumination, indicating impaired NDH activity ( <a href="#pbio-1001040-g001">Figure 1B</a>). In contrast to <i>crr</i> mutants, the <i>cpn60α1</i> mutant exhibited retarded growth and pale green-leaf phenotypes ( <a href="#pbio-1001040-g001">Figure 1C</a>). Map-based cloning identified a single amino acid substitution (D335A) at a conserved position in <i>cpn60α1</i> ( <a href="#pbio-1001040-g001">Figure 1D</a>). Although the total levels of Cpn60α and Cpn60β were increased in <i>cpn60α1</i>, possibly due to complementation effects, the levels of many other chloroplast proteins, including NDH subunits, were reduced to various extents ( <a href="#pbio.1001040.s001">Figure S1</a>), supporting the idea that Cpn60 has a diverse set of substrates. The reduction in NDH (25%–50%) in <i>cpn60α1</i> at least partly explains the failure to detect NDH activity by chlorophyll fluorescence. </p> <div class="fig" title="fig"> <div class="figure" title="figure"> <a href="http://doi.org/10.1371/journal.pbio.1001040.g001"> <span class="label" tagx="label" title="label">Figure 1</span> </a> </div>   <div class="title" tagx="title" title="title">
Characterization of <i>cpn60α1</i> and <i>crr27</i> mutants. </div> <p>(A) A schematic model of NDH function. The NDH complex mediates electron transfer from the stromal reducing pool to plastoquinone (PQ). PQ reduction in the dark depends on NDH activity and can be detected by the transient rise of chlorophyll (Chl) fluorescence after illumination with actinic light (AL). For simplicity, this model does not include the information that NDH interacts with PSI. Cyt, cytochrome; PC, plastocyanin; Fd, ferredoxin. (B) Determination of NDH activity using Chl fluorescence analysis. The bottom curve indicates a typical trace of Chl fluorescence in the WT plants. Leaves were exposed to AL for 5 min. AL was turned off and the subsequent transient rise in fluorescence ascribed to NDH activity was monitored using a PAM Chl fluorometer. Insets are magnified traces from the boxed area. <i>crr27-1+Cpn60β4</i> and <i>crr27-1+Cpn60β4-HA</i> represent <i>crr27-1</i> transformed by the WT genomic <i>Cpn60β4</i> and genomic <i>Cpn60β4</i> fused to the HA epitope-tag, respectively. <i>crr27-1+35S::Cpn60β4-HA</i> and <i>crr27-1+35S::Cpn60β1-HA</i> represent <i>crr27-1</i> transformed with Cpn60β4 and Cpn60β1 cDNA, respectively, fused to the sequence encoding the HA-tag expressed under the control of the CaMV <i>35S</i> promoter. Fluorescence levels were standardized to the maximum fluorescence levels of closed PSII ( <i>Fm</i>) by applying saturating-light pulses (SP). ML, measuring light; <i>Fo</i>, minimum fluorescence level of open PSII. (C) Visible phenotype of mutants. Seedlings were cultured at 50 µmol photons m <sup>−2</sup> s <sup>−1</sup> for 4 wk after germination. (D and E) Mutations in <i>cpn60α1</i> (D) and three <i>crr27</i> mutant alleles (E) are indicated. (F) RT-PCR analysis of the <i>Cpn60β4</i> transcript in WT and <i>crr27</i> mutants. <i>ACT8</i> was used as a control. </p>   </div> <p>The <i>crr27</i> mutants were isolated by screening <i>Ds</i> transposon-tagged lines using PAM (pulse amplitude modulation) fluorometry <a href="#pbio.1001040-Kuromori1">[36]</a>, <a href="#pbio.1001040-Okuda1">[37]</a>. Unlike <i>cpn60α1</i>, <i>crr27</i> mutants did not exhibit any visible phenotype besides impaired NDH activity ( <a href="#pbio-1001040-g001">Figure 1B and 1C</a>). The <i>Cpn60β4</i> gene was knocked out by <i>Ds</i> or T-DNA insertions in three <i>crr27</i> alleles, and reverse transcription (RT)-PCR analysis did not detect any <i>Cpn60β4</i> transcripts ( <a href="#pbio-1001040-g001">Figure 1E and 1F </a>). Full NDH activity was rescued by the introduction of the WT <i>Cpn60β4</i> gene into <i>crr27-1</i> ( <a href="#pbio-1001040-g001">Figure 1B</a>). Two chlorophyll fluorescence parameters, ETR (electron transport rate) and NPQ (nonphotochemical quenching), indicate subtle defects in photosynthesis and are therefore often used to characterize mutants with defective photosynthetic apparatus. ETR was only slightly reduced and NPQ was not affected in <i>crr27-1</i> ( <a href="#pbio.1001040.s002">Figure S2 </a>), which is consistent with the phenotypes of other <i>crr</i> mutants with specific defects in NDH activity. </p> </div> <div class="accumulationofndhsubcomplexisimpairedintheabsenceofcpn60β4" title="sec"> <div class="title" tagx="title" title="title">
Accumulation of NDH Subcomplex Is Impaired in the Absence of Cpn60β4 </div> <p>NDH interacts with at least two copies of PSI to form the NDH-PSI supercomplex ( <a href="#pbio-1001040-g002">Figure 2A</a>) <a href="#pbio.1001040-Peng1">[35]</a>, which can be separated by blue native (BN)-PAGE <a href="#pbio.1001040-Peng2">[38]</a>, <a href="#pbio.1001040-Peng3">[39]</a>. To study the role of Cpn60β4 in biogenesis of the NDH complex, thylakoid protein complexes from WT and <i>crr27</i> mutants were separated by BN-PAGE. No difference was found in the major complex bands between WT and <i>crr27</i> ( <a href="#pbio-1001040-g002">Figures 2B</a> and <a href="#pbio.1001040.s003">S3A</a>). However, band I, corresponding to the NDH-PSI supercomplex detected in WT, was replaced by band II, corresponding to the subsupercomplex in <i>crr27,</i> as in the NdhL-defective <i>ndhl (crr23) </i> mutant ( <a href="#pbio-1001040-g002">Figure 2B </a>) <a href="#pbio.1001040-Peng2">[38]</a>. Based on extensive genetic and biochemical characterizations, we divided the NDH complex into four categories: membrane, lumen, and A and B subcomplexes ( <a href="#pbio-1001040-g002">Figure 2A</a>) <a href="#pbio.1001040-Peng3">[39]</a>. Previous mass analysis revealed that only subcomplex A, which is composed of four plastid-encoded subunits (NdhH–NdhK) and four nucleus-encoded subunits (NdhL–NdhO), was absent in band II ( <a href="#pbio-1001040-g002">Figure 2A</a>) <a href="#pbio.1001040-Peng3">[39]</a>. Immunoblot analysis confirmed that the levels of subcomplex A subunits NdhH and NdhL were dramatically decreased in <i>crr27,</i> whereas subunits of the other NDH subcomplexes, NDH18, NDF1, and FKBP16-2, were only slightly reduced ( <a href="#pbio-1001040-g002">Figure 2C</a>). Consistent with the invisible growth phenotype of <i>crr27</i>, identical levels of D1 (PSII complex), PsaA (PSI complex), and cytochrome (Cyt) <i>f</i> (Cyt <i>b</i> <sub>6</sub> <i>f</i> complex) were detected in <i>crr27</i> and WT ( <a href="#pbio-1001040-g002">Figure 2C</a>). In addition, no significant difference in stromal protein levels was detected between WT and <i>crr27-1</i> mutants by clear native (CN)-PAGE and subsequent two-dimensional (2D)/SDS-PAGE ( <a href="#pbio.1001040.s003">Figure S3B </a>). From these results, we conclude that the accumulation of NDH subcomplex A was specifically impaired in the absence of Cpn60β4. </p> <div class="fig" title="fig"> <div class="figure" title="figure"> <a href="http://doi.org/10.1371/journal.pbio.1001040.g002"> <span class="label" tagx="label" title="label">Figure 2</span> </a> </div>   <div class="title" tagx="title" title="title">
Accumulation of NDH subcomplex A was impaired in <i>crr27</i>. </div> <p>(A) A schematic model of the NDH-PSI supercomplex in chloroplasts. The NDH complex is divided into four subcomplexes and interacts with at least two copies of PSI to form the NDH-PSI supercomplex corresponding to Band I, which can be detected by BN-PAGE (B). Eleven plastid-encoded subunits are depicted by green letters; nucleus-encoded subunits are depicted by red letters. Subcomplex A, surrounded by the red line, is missing in the sub-NDH-PSI supercomplex corresponding to Band II, which is detected in <i>crr27</i> and <i>ndhl</i> mutants (B). (B) BN-PAGE analysis of thylakoid protein complexes. After electrophoresis <b>,</b> the gel was stained with CBB. The complexes are identified in the original non-stained gel in <a href="#pbio.1001040.s003">Figure S3A </a>. (C) Immunoblot analysis of the thylakoid proteins from various genetic backgrounds with the indicated antibodies. Thylakoid proteins were loaded on an equal chlorophyll basis, and the series of dilutions is indicated. </p>   </div> </div> <div class="heterooligomericchaperonincomplexformationofcpn60β4withcpn60β1–β3andcpn60α1" title="sec"> <div class="title" tagx="title" title="title">
Heterooligomeric Chaperonin Complex Formation of Cpn60β4 with Cpn60β1–β3 and Cpn60α1 </div> <p>Previous transcriptomic analysis <a href="#pbio.1001040-Weiss1">[26]</a> indicated that the expression level of the <i>Cpn60β</i> <i>4</i> gene is lower than that of other <i>Cpn60β</i> genes. Furthermore, the Cpn60β4 subunit could not be visualized with Coomassie Brilliant Blue (CBB) staining in the 2D CN/SDS-PAGE gel, whereas other Cpn60β subunits and Cpn60α1 were detected <a href="#pbio.1001040-Peltier1">[27]</a>, suggesting that the stoichiometry of Cpn60β4 is extremely low compared to the other Cpn60β subunits. To confirm the accumulation of Cpn60β4 in WT, we separated total stromal protein complexes isolated from WT plants by CN-PAGE. The protein band corresponding to the position of the Cpn60 complex was excised from the gel ( <a href="#pbio.1001040.s004">Figure S4A</a>) and analyzed by liquid chromatography-tandem mass spectrometry (LC-MS/MS) analysis using the linear ion-trap triple quadrupole (LTQ)-Orbitrap XL-HTC-PAL system, which provides high mass accuracy, high resolution, and high sensitivity. The values of Mowse score, Protein match, and emPAI (exponentially modified Protein Abundance Index) are commonly used to estimate relative protein levels. LC-MS/MS analyses detected the Cpn60β4 protein, but its level was significantly lower than those of the other three Cpn60β proteins ( <a href="#pbio.1001040.s004">Figure S4B</a>; <a href="#pbio.1001040.s010">Table S1</a>). Consistent with the apparent mutant phenotype ( <a href="#pbio-1001040-g001">Figures 1</a> and <a href="#pbio-1001040-g002">2</a>), we confirmed the accumulation of Cpn60β4 in WT ( <a href="#pbio.1001040.s004">Figure S4C</a>). </p> <p>In contrast to <i>cpn60α1</i>, the <i>crr27</i> mutation did not affect total Cpn60α or Cpn60β levels ( <a href="#pbio-1001040-g003">Figures 3A</a> and <a href="#pbio.1001040.s001">S1A</a>). To study the role of Cpn60β4, a chimeric gene encoding an HA (influenza hemagglutinin protein epitope) tag fused to the C-terminus of Cpn60β4 was introduced into <i>crr27-1</i>. This transformation fully restored NDH activity ( <a href="#pbio-1001040-g001">Figure 1B</a>), indicating that the HA-tag did not affect the function of Cpn60β4. We also overexpressed HA-tagged Cpn60β1 and Cpn60β4 in <i>crr27-1</i> under control of the cauliflower mosaic virus (CaMV) <i>35S</i> promoter. NDH activity and NDH subcomplex A level were rescued only in the <i>35S::Cpn60β4-HA</i> lines ( <a href="#pbio-1001040-g001">Figures 1B</a> and <a href="#pbio-1001040-g002">2C</a>), indicating that Cpn60β1 cannot complement the function of Cpn60β4, even under the control of the same promoter. </p> <div class="fig" title="fig"> <div class="figure" title="figure"> <a href="http://doi.org/10.1371/journal.pbio.1001040.g003"> <span class="label" tagx="label" title="label">Figure 3</span> </a> </div>   <div class="title" tagx="title" title="title">
Analysis of the Cpn60β4 subunit.</div> <p>(A) Cpn60β4 is localized to the chloroplast stroma. Freshly isolated chloroplasts from various genotypes were separated into membrane and stromal fractions. Immunoblot analysis was performed using the indicated antibodies. RbcL and Cyt <i>f</i> were detected as loading and fractionation controls. (B) Stromal protein complexes isolated from <i>crr27-1</i> complemented by Cpn60β4-HA were separated by CN-PAGE, followed by 2-dimensional SDS-PAGE. The proteins were immunodetected with specific antibodies. A short arrow indicates the position of the Cpn60 complex. (C) Heterooligomeric complex formation between Cpn60β4, Cpn60α1, and Cpn60β1–β3. Chaperonin complex containing Cpn60β4 was purified from the <i>crr27-1</i> mutant plants expressing HA-tagged Cpn60β4 using the µMACS HA isolation kit. After elution, total proteins were separated by 7.5% SDS-PAGE and stained with CBB. The signals were quantitatively analyzed with Imagemaster software (Amersham Pharmacia Biotech). The stoichiometry of Cpn60β4, Cpn60β1–β3, and Cpn60α1 in the specific chaperonin complex containing Cpn60β4 was estimated to be 17:37:46 and 14:39:47 in the two independent purifications (E1 and E2), respectively. </p>   </div> <p>Cpn60β4 localized to the chloroplast stroma ( <a href="#pbio-1001040-g003">Figure 3A</a>) and co-migrated with other Cpn60β and Cpn60α subunits in CN-PAGE ( <a href="#pbio-1001040-g003">Figures 3B</a> and <a href="#pbio.1001040.s004">S4</a>), suggesting that Cpn60β4 is an intrinsic subunit of the Cpn60 complex. To examine this possibility, HA-tagged Cpn60β4 was enriched from the stromal fraction isolated from <i>crr27-1</i> plants complemented with Cpn60β4-HA using the µMACS HA isolation kit (Miltenyi Biotec) under previously established conditions <a href="#pbio.1001040-Kerner1">[11]</a>. Because the additional HA tag in the C-terminus does not affect the function of Cpn60β4 ( <a href="#pbio-1001040-g001">Figure 1B</a>), their interacting proteins might also be co-purified. Total isolated proteins were separated by SDS-PAGE ( <a href="#pbio.1001040.s005">Figure S5A </a>) and further analyzed by LC-MS/MS analysis. Both Cpn60α1 and Cpn60β1–β4 subunits were detected in the purified sample and MS analysis showed that they were the most abundant proteins ( <a href="#pbio-1001040-t001">Tables 1</a> and <a href="#pbio.1001040.s011">S2</a>), implying that Cpn60β4 forms a specific heterooligomeric Cpn60 complex with Cpn60α1 and other Cpn60β subunits. </p> <div class="table-wrap_UNKNOWN"> <a href="https://dx.doi.org/10.1371/journal.pbio.1001040.t001">10.1371/journal.pbio.1001040.t001</a> <span class="label" tagx="label" title="label">Table 1</span>  <div class="title" tagx="title" title="title">
Summary of the chaperonin and NDH subunits detected in the Cpn60β1 and Cpn60β4 IP fractions. </div>  <div class="alternatives" title="alternatives"> <div class="graphic_UNKNOWN" /> <table> <div class="colgroup_UNKNOWN"> <div class="col_UNKNOWN" /></div> <div class="colgroup_UNKNOWN"><div class="col_UNKNOWN" /></div> <div class="colgroup_UNKNOWN"><div class="col_UNKNOWN" /></div> <div class="colgroup_UNKNOWN"><div class="col_UNKNOWN" /></div> <div class="colgroup_UNKNOWN"><div class="col_UNKNOWN" /></div> <div class="colgroup_UNKNOWN"><div class="col_UNKNOWN" /></div> <div class="colgroup_UNKNOWN"><div class="col_UNKNOWN" /></div> <div class="colgroup_UNKNOWN"><div class="col_UNKNOWN" /></div>  <thead> <tr> <td>Protein Name</td> <td /> <td>Cpn60β1 IP</td> <td /> <td /> <td>Cpn60β4 IP</td> <td /> <td>emPAI Ratio <sup>a</sup> </td> </tr> <tr> <td /> <td>Mowse Score</td> <td>Protein Match</td> <td>Coverage (%)</td> <td>Mowse Score</td> <td>Protein Match</td> <td>Coverage (%)</td> <td /> </tr> </thead> <tbody> <tr> <td>Cpn60α1</td> <td>2,177</td> <td>64</td> <td>69</td> <td>6,369</td> <td>188</td> <td>72</td> <td>3.9</td> </tr> <tr> <td>Cpn60β1</td> <td>6,794</td> <td>230</td> <td>70</td> <td>5,591</td> <td>202</td> <td>68</td> <td>0.47</td> </tr> <tr> <td>Cpn60β2</td> <td>5,689</td> <td>178</td> <td>59</td> <td>5,356</td> <td>182</td> <td>67</td> <td>0.87</td> </tr> <tr> <td>Cpn60β3</td> <td>3,624</td> <td>119</td> <td>36</td> <td>3,448</td> <td>123</td> <td>37</td> <td>1.26</td> </tr> <tr> <td>Cpn60β4</td> <td>226 <sup>b</sup> </td> <td>13 <sup>b</sup> </td> <td>3 <sup>b</sup> </td> <td>3,348</td> <td>156</td> <td>74</td> <td>101</td> </tr> <tr> <td>NdhJ</td> <td>57</td> <td>2</td> <td>15</td> <td>—</td> <td>—</td> <td>—</td> <td>—</td> </tr> <tr> <td>NdhH</td> <td>44</td> <td>2</td> <td>4</td> <td>614</td> <td>24</td> <td>53</td> <td>43.4</td> </tr> </tbody> </table> </div> <div class="table-wrap-foot_UNKNOWN"> <div class="fn-type-" title=""> <span class="label" tagx="label" title="label" /> <p>a. emPAI Ratio means the ratio between emPAI score of proteins isolated from Cpn60β4 IP and Cpn60β1 IP fraction. </p> </div> <div class="fn-type-" title=""> <span class="label" tagx="label" title="label" /> <p>b. The sequences of these 13 peptides are identical between Cpn60β1 and Cpn60β4, suggesting that they are sequenced from Cpn60β1. RT-PCR showed that <i>crr27-1</i> is a null mutant ( <a href="#pbio-1001040-g001">Figure 1F </a>). </p> </div> </div> </div> <p>The molecular masses of Cpn60β4-HA, Cpn60β1–β3, and Cpn60α1 are 64.4, 58.2, and 57.1 kDa, respectively, which enables us to distinguish them in SDS-PAGE. The purified proteins were subjected to 7.5% SDS-PAGE and three major bands with molecular masses of approximately 60 kDa were visualized with CBB staining ( <a href="#pbio-1001040-g003">Figure 3C </a>). Based on the mobility, three bands should correspond to Cpn60β4-HA, Cpn60β1–β3, and Cpn60α1 ( <a href="#pbio-1001040-g003">Figure 3C</a>). Quantitative estimation of these signals showed that the level of Cpn60α1 is about 50% of all of the chaperonin subunits, which is consistent with the proposal that the Cpn60 complex is composed in a α7β7 stoichiometry <a href="#pbio.1001040-Dickson1">[24]</a>, <a href="#pbio.1001040-Nishio1">[25]</a>. However, the level of Cpn60β4 in the chaperonin complex was lower (∼15%) ( <a href="#pbio-1001040-g003">Figure 3C</a>), implying that approximately two molecules of Cpn60β4 are included in double rings, as well as five molecules of other Cpn60β subunits. Based on the stoichiometry of the Cpn60β4 subunit in total Cpn60 subunits <a href="#pbio.1001040-Weiss1">[26]</a>, <a href="#pbio.1001040-Peltier1">[27]</a>, the majority of the Cpn60 complex is unlikely to contain Cpn60β4, and we estimated its stoichiometry in the specific complex including Cpn60β4. </p> </div> <div class="ndhhisspecificallyassociatedwithcpn60β4" title="sec"> <div class="title" tagx="title" title="title">
NdhH Is Specifically Associated with Cpn60β4</div> <p>Given that Cpn60β4 is an intrinsic subunit of the chaperonin complex ( <a href="#pbio-1001040-g003">Figure 3</a>) and that the NDH subcomplex A was missing in the <i>crr27</i> mutants ( <a href="#pbio-1001040-g002">Figure 2</a>), it is very likely that the specific chaperonin complex containing Cpn60β4 is required for the folding of at least one subunit of the NDH subcomplex A. If this is the case, the interacting NDH subunit would be copurified with Cpn60β4-HA. To determine differences in substrate specificity, protein complexes containing Cpn60β1 were isolated using the <i>35S::Cpn60β1-HA</i> lines. Neither Cpn60 nor NDH subunits were detected in untransformed WT plants, which were used as a negative control ( <a href="#pbio.1001040.s005">Figure S5A</a>; <a href="#pbio.1001040.s011">Table S2</a>), excluding the possibility of non-specific binding to the magnetic beads. Cpn60α1 and Cpn60β1–β3 were co-purified with both Cpn60β1 and Cpn60β4 ( <a href="#pbio-1001040-t001">Tables 1</a> and <a href="#pbio.1001040.s011">S2</a>). These results confirmed that Cpn60α1 and Cpn60β1–β4 form a heterooligomeric complex in vivo. </p> <p>The ratio between protein emPAI scores can be used to estimate the relative amounts of protein in the different samples <a href="#pbio.1001040-Ishihama1">[40]</a>. The emPAI ratios of Cpn60α1 and Cpn60β1–β3 detected in Cpn60β4- and Cpn60β1-purified samples were 3.9 and 0.47–1.26 (Cpn60β4/Cpn60β1), respectively, suggesting that comparable amounts of Cpn60 complexes were used for MS analysis. This estimation was confirmed by the similar intensity of the Cpn60 subunit bands detected by SDS-PAGE ( <a href="#pbio.1001040.s005">Figure S5A </a>). Interestingly, 24 peptides of an NDH subunit, NdhH, were found in the Cpn60β4-purified fraction. No other NDH subunits or NDH biogenesis factors were found in any sample other than the NdhJ detected in the Cpn60β1-purified extraction ( <a href="#pbio-1001040-t001">Table 1</a>). Although NdhH was also co-purified with Cpn60β1, only two NdhH peptides were detected ( <a href="#pbio-1001040-t001">Table 1</a> and <a href="#pbio.1001040.s005">Figure S5B</a>) and the emPAI ratio of NdhH from Cpn60β4- and Cpn60β1-purified samples was 43.4. Aside from the nonspecific proteins detected in WT as well as chaperonin subunits detected in Cpn60β4-IP sample, NdhH was the most abundant protein found in the Cpn60β4-purified sample ( <a href="#pbio.1001040.s011">Table S2</a>). These results indicate that the Cpn60 complex containing Cpn60β4 can specifically recognize unfolded NdhH. NdhH was also detected in the Cpn60β1-purified sample ( <a href="#pbio-1001040-t001">Table 1</a>), suggesting that Cpn60 complexes containing Cpn60β1 also can interact with NdhH. However, when Cpn60β1-HA was introduced into <i>crr27-1</i>, the transformation did not rescue NDH activity ( <a href="#pbio-1001040-g001">Figures 1</a> and <a href="#pbio-1001040-g002">2</a>), suggesting that Cpn60 complexes lacking Cpn60β4 cannot produce native NdhH even though they can bind to it. This idea is consistent with the fact that <i>crr27</i> accumulates the band II subsupercomplex ( <a href="#pbio-1001040-g002">Figure 2 </a>). </p> </div> <div class="assemblyofndhsubcomplexaisimpairedincrr27" title="sec"> <div class="title" tagx="title" title="title">
Assembly of NDH Subcomplex A Is Impaired in <i>crr27</i> </div> <p>Although the functional NDH complex is localized to the thylakoids, three assembly intermediate complexes including NdhH are present in the chloroplast stroma ( <a href="#pbio.1001040.s006">Figure S6 </a>) <a href="#pbio.1001040-Peng4">[41]</a>. Nuclear-encoded factors CRR6 and CRR7 may be required for integration of these intermediates into thylakoids to form the functional NDH complex. In <i>crr27-1</i>, the level of stroma-localized NdhH was significantly reduced ( <a href="#pbio.1001040.s006">Figure S6A </a>). Furthermore, 2D CN/SDS-PAGE and immunoblot studies showed that the accumulation of the 500 kDa and 400 kDa intermediate complexes was impaired in <i>crr27-1</i> ( <a href="#pbio.1001040.s006">Figure S6B</a>), implying that only NdhH folded by the Cpn60 complex including Cpn60β4 can be efficiently incorporated into these two assembly intermediates and further into thylakoids. These results also suggest that the folding of NdhH via the Cpn60 complex containing Cpn60β4 occurs at the initial step of NDH subcomplex A biogenesis. </p> </div> <div class="theuniquec-terminusofcpn60β4isrequiredforitsspecificfunction" title="sec"> <div class="title" tagx="title" title="title">
The Unique C-Terminus of Cpn60β4 Is Required for Its Specific Function </div> <p>Based on a transcriptome database of <i>Arabidopsis</i>, ATTED-II <a href="#pbio.1001040-Obayashi1">[42]</a>, we found that the <i>Cpn60β4</i> gene, but not other <i>Cpn60</i> genes, is co-expressed with genes encoding NDH subunits and NDH biogenesis factors ( <a href="#pbio.1001040.s012">Table S3 </a>). This pattern is consistent with our findings that Cpn60β4 is specifically required for the folding of NdhH. However, the question remains as to why the other Cpn60β proteins cannot complement the function of Cpn60β4. </p> <p>The mycobacterial GroEL1 has a histidine-rich C-terminus that appears to be critical for its specific function in association with proteins required for bacterial biofilm formation <a href="#pbio.1001040-Ojha1">[18]</a>. Cpn60β4 also contains a C-terminal extension that is not conserved in other Cpn60β proteins ( <a href="#pbio-1001040-g004">Figures 4A</a> and <a href="#pbio.1001040.s007">S7</a>). Although the C-terminus of Cpn60β4 is not conserved in plants, the region is rich in positively charged residues ( <a href="#pbio-1001040-g004">Figure 4A</a>). To study whether the C-terminus is important for Cpn60β4 function, HA-tagged Cpn60β4 lacking the C-terminus or exchanged by the short C-terminal tail of Cpn60β1 was expressed in <i>crr27-1</i> ( <a href="#pbio-1001040-g004">Figure 4B</a>). 2D CN/SDS-PAGE immunoblot analysis showed that the mutant versions of Cpn60β4 can be incorporated into the Cpn60 complex ( <a href="#pbio-1001040-g004">Figure 4C </a>), excluding the possibility that the C-terminus of Cpn60β4 is required for the stabilization or formation of the chaperonin complex. Although the levels of mutant Cpn60β4 were approximately twice those of the WT version of Cpn60β4 in the stroma, NdhH levels in thylakoids were reduced by approximately one half in the Cpn60β4-HA lines ( <a href="#pbio-1001040-g004">Figure 4B</a>), resulting in the reduction of NDH activity ( <a href="#pbio-1001040-g004">Figure 4D</a>). These results indicate that the folding efficiency of NdhH was reduced in the absence of the Cpn60β4-specific C-terminus. </p> <div class="fig" title="fig"> <div class="figure" title="figure"> <a href="http://doi.org/10.1371/journal.pbio.1001040.g004"> <span class="label" tagx="label" title="label">Figure 4</span> </a> </div>   <div class="title" tagx="title" title="title">
C-terminal of Cpn60β4 is required for the Cpn60β4 function. </div> <p>(A) C-terminal alignment of Cpn60 subunits. Cpn60β4 from <i>Arabidopsis thaliana</i> (At), <i>Arabidopsis lyrata </i> (Al), <i>Oryza sativa</i> (Os), <i>Zea mays</i> (Zm), and <i>Populus trichocarpa </i> (Pt) were aligned with the <i>Arabidopsis</i> Cpn60β1–β3 and Cpn60α1 and the <i>E. coli</i> GroEL. Positions of the deletion and the C-terminal swapping are shown. The number of negatively (−) and positively (+) charged residues in the region is indicated. (B) Protein blot analysis of NdhH and Cpn60β4. RbcL and Cyt <i>f</i> were used as loading controls. The series of dilutions is indicated. (C) The C-terminus of Cpn60β4 is not essential for the assembly of the chaperonin complex. Stromal protein complexes isolated from <i>crr27-1</i> complemented by Cpn60β4 truncated in the C-terminus (Cpn60β4(-C)-HA) and from <i>crr27-1</i> transformed by Cpn60β4, in which the C-terminus was exchanged for that of Cpn60β1 (Cpn60β4(+β1C)-HA), were separated by CN-PAGE, followed by 2-D SDS-PAGE. The proteins were immunodetected with specific antibodies. A short arrow indicates the position of the Cpn60 complex. (D) NDH activity monitored by Chl florescence as in <a href="#pbio-1001040-g001">Figure 1B</a>. <i>crr27-1+Cpn60β4-HA</i>, <i>crr27-1</i> transformed by WT genomic <i>Cpn60β4</i>; <i>crr27-1+Cpn60β4(-C)-HA</i>, <i>crr27-1</i> transformed by Cpn60β4 truncated in the C-terminus; <i>crr27-1+Cpn60β4(+β1C)-HA</i>, <i>crr27-1</i> transformed by Cpn60β4, in which the C-terminus was exchanged for that of Cpn60β1; <i>crr27-1+Cpn60β4(CM)-HA</i>, <i>crr27-1</i> transformed by Cpn60β4 containing the amino acid alterations in the wall of central cavity. All of the proteins were fused to the HA-epitope tag on their C-termini. (E) The net charge of each subunit wall exposed to the central cavity in the <i>cis</i> ring. As shown in <a href="#pbio.1001040.s007">Figure S7 </a>, the multiple charged residues, which are highly conserved among Cpn60β4 but not in Cpn60β1–β3 subunits, were altered in AtCpn60β4 to the corresponding amino acids in AtCpn60β1, resulting in a net charge of −4 (AtCpn60β4(CM)). (F) Immunoblot analysis of NdhH and Cpn60β4 from <i>crr27-1</i> expressing the AtCpn60β4(CM)-HA protein. (G) Protein blot analysis of NdhH and Cpn60β4. <i>crr27-1</i>+ <i>Cpn60β4(ΔC+CM)-HA</i>, <i>crr27-1</i> transformed by a multiple mutant version of Cpn60β4 containing the amino acid alterations in the wall of central cavity (AtCpn60β4(CM)) and truncated in its C-terminus (Cpn60β4(-C)). The proteins were fused to the HA-tag on their C-termini. RbcL and Cyt <i>f</i> were used as loading controls and the series of dilutions is indicated. </p>   </div> <p>In the absence of the C-terminal tail, NdhH is still partially assembled ( <a href="#pbio-1001040-g005">Figure 5B</a>). We also transformed <i>crr27</i> with Cpn60β1 fused with the Cpn60β4 C-terminal tail, but NdhH folding activity was not complemented (unpublished data). These results suggest that other features of Cpn60β4 are required for its specific function. Protein sequence alignment revealed that the ATP/ADP and Mg <sup>2+</sup> binding sites are highly conserved between Cpn60β4 and the other three Cpn60β subunits ( <a href="#pbio.1001040.s007">Figure S7</a>), which is consistent with the fact that Cpn60β4 is an intrinsic subunit of the Cpn60 complex ( <a href="#pbio-1001040-g003">Figure 3</a>). However, the proposed substrate-binding residues are less conserved ( <a href="#pbio.1001040.s007">Figure S7</a>), which may explain why Cpn60β4 has a high affinity specifically for NdhH. Protein sequence alignment also showed that up to 31 amino acid residues are highly conserved among the putative Cpn60β4 orthologs, but their properties are different from the corresponding residues in other Cpn60β subunits ( <a href="#pbio.1001040.s007">Figures S7 </a> and <a href="#pbio.1001040.s008">S8</a>). Three-dimensional (3-D) structure analyses mapped these residues to the apical, intermediate, and equatorial domains of the Cpn60β4 subunit ( <a href="#pbio.1001040.s008">Figure S8</a>). </p> <div class="fig" title="fig"> <div class="figure" title="figure"> <a href="http://doi.org/10.1371/journal.pbio.1001040.g005"> <span class="label" tagx="label" title="label">Figure 5</span> </a> </div>   <div class="title" tagx="title" title="title">
Evolutionary analyses of the chloroplast Cpn60β4 subunits. </div> <p>(A) A phylogenetic tree of the chloroplast chaperonin 60 and cyanobacterial GroEL proteins. Chaperonin subunits from fully sequenced genomes of angiosperms ( <i>Arabidopsis thaliana</i>, <i>Arabidopsis lyrata</i>, <i>Populus trichocarpa </i>, <i>Zea mays</i>, and <i>Oryza sativa </i>) and a bryophyte ( <i>Physcomitrella patens </i>) were retrieved from GenBank or Phytozome ( <a href="http://www.phytozome.net/">http://www.phytozome.net/</a>). The GroEL sequences from <i>Synechocystis</i> SP. PCC 6803 and a Cpn60β sequence from <i>Marchantia polymorpha</i>, which is used to complement the <i>crr27-1</i> mutant in (B), are also included. The tree was generated using both Maximum-likelihood (ML) and Bayesian methods. ML and Bayesian consensus trees were topologically congruent and only ML topology is shown and drawn to scale. Numbers at each node in the ML tree signify bootstrap/posterior probability values (&amp;gt;50%) from ML and Bayesian methods, respectively. Support values less than 50 are shown as hyphens (−). The sequence alignments used to generate the ML and Bayesian tree are available in <a href="#pbio.1001040.s013">Dataset S1</a>. Proteins highlighted with yellow boxes were investigated in this study. (B) Immunodetection of chloroplast proteins from WT, <i>crr27-1</i>, <i>crr27-1</i> complemented by AtCpn60β4-HA, and three lines of <i>crr27-1</i> complemented by MpCpn60β-HA (L1–L3). </p>   </div> <p>Of the 31 conserved residues in Cpn60β4, several charged amino acids are located inside the cavity ( <a href="#pbio.1001040.s008">Figure S8</a>). The negatively charged GroEL cavity wall is required for rapid folding of some substrates <a href="#pbio.1001040-Tang1">[43]</a>. In <i>E. coli</i>, each GroEL subunit has 27 negatively and 21 positively charged amino acids exposed to the central cavity in the <i>cis-</i>conformation, resulting in a net charge of −6 <a href="#pbio.1001040-Brocchieri1">[44]</a>. By analogy with <i>E. coli</i> GroEL, Cpn60α1 and Cpn60β1 have net charges of −4 and −6, respectively. However, Cpn60β4 in <i>Arabidopsis</i> has a more positive charge of 0, and this trend is found for Cpn60β4 in other plants (charges ranging from −2 to +2) ( <a href="#pbio-1001040-g004">Figure 4E</a>). To investigate the significance of the positively charged cavity wall in the folding of NdhH, the multiple charged residues, which are highly conserved in Cpn60β4 but not in Cpn60β1–β3 subunits, were converted to the corresponding amino acids of AtCpn60β1 and the mutant genes were introduced into <i>crr27-1</i> plants. The sites correspond to 5 out of 31 amino acid residues indicated in <a href="#pbio.1001040.s008">Figure S8</a>. Although the mutant Cpn60β4 has a net charge of −4 ( <a href="#pbio-1001040-g004">Figure 4E</a>), NdhH level and NDH activity were fully rescued in the transformed plants ( <a href="#pbio-1001040-g004">Figure 4D and 4F </a>). We also transformed a version of Cpn60β4 with the amino acid alterations on the wall of the central cavity and the deletion of the C-terminus into <i>crr27-1</i>. In these lines, NdhH levels in thylakoids were reduced to ∼50% of the Cpn60β4-HA lines ( <a href="#pbio-1001040-g004">Figure 4G</a>), similar to the results in <i>crr27-1</i> transformed by Cpn60β4 lacking its C-terminus ( <a href="#pbio-1001040-g004">Figure 4B</a>). These results suggest that the positive charge of the cavity wall is not crucial for the folding of NdhH. The residues specifically conserved in the putative Cpn60β4 orthologs are dispersed throughout the molecule except for five positively charged sites facing the cavity wall ( <a href="#pbio.1001040.s008">Figure S8</a>), and it is not feasible to determine the sites responsible for the specific function by the site-directed mutagenesis. Proper folding of NdhH may require both the drastic alteration in the sequence as well as the C-terminal extension. </p> </div> <div class="evolutionofthecpn60β4subunit" title="sec"> <div class="title" tagx="title" title="title">
Evolution of the Cpn60β4 Subunit</div> <p>Both chloroplast Cpn60 and NDH complexes are thought to have originated from their cyanobacterial ancestors, GroEL2 <a href="#pbio.1001040-Wastl1">[45]</a> and NDH-1 <a href="#pbio.1001040-Shikanai1">[32]</a>. NdhH is highly conserved in <i>Arabidopsis</i>, <i>Physcomitrella patens </i> (moss) and cyanobacteria ( <a href="#pbio.1001040.s009">Figure S9</a>). NdhH is a 45.5 kDa protein with α+β domains ( <a href="#pbio.1001040.s009">Figure S9</a>) <a href="#pbio.1001040-Arnold1">[46]</a>, and theoretically it can be fully encapsulated within the chaperonin cage. In contrast to NdhH, the structure of Cpn60 is not conserved among organisms. To clarify the evolution and ancestry of Cpn60β subunits in plants, we compared the amino acid sequences of members in several fully sequenced genomes ( <a href="#pbio-1001040-g005">Figure 5A</a>). In addition to the distinct clades of Cpn60α and Cpn60β, Cpn60β proteins were further divided into two clades: putative AtCpn60β4 orthologs and other Cpn60β genes (major Cpn60β). The orthologs of AtCpn60β1–β3 were found in the closely related <i>Arabidopsis lyrata</i>, but two major Cpn60β proteins of poplar ( <i>Populus trichocarpa</i>) were related only to AtCpn60β3. In contrast, the major Cpn60β subunits of monocots form a different subclade, and maize ( <i>Zea mays</i>) and rice ( <i>Oryza sativa</i>) each contain two major Cpn60β subunits ( <a href="#pbio-1001040-g005">Figure 5A</a>). These facts suggest that gene duplication of major Cpn60β subunit genes took place independently both in monocots and eudicots. In contrast, a single copy of the putative Cpn60β4 ortholog was detected in angiosperms ( <a href="#pbio-1001040-g005">Figure 5A</a>). </p> <p>A total of three Cpn60β subunits were found in <i>Physcomitrella patens </i> and they are related to major Cpn60β subunits in angiosperms ( <a href="#pbio-1001040-g005">Figure 5A</a>). Notably, no ortholog of Cpn60β4 was found in <i>P. patens</i>. The phylogenetic tree indicates that the origin of Cpn60β4 can be traced to the origin of land plants and that Cpn60β4 was lost in the descendent lineage of bryophytes. Due to the low bootstrap support of the evolutionary relationships between angiosperm and bryophyte major Cpn60β subunits (72/0.90 as shown in <a href="#pbio-1001040-g005">Figure 5</a>), it is also likely that the Cpn60β4 orthologs were produced by a gene duplication event that took place only in a common ancestor of angiosperms and underwent a rapid rate of evolution to obtain the novel function. In any case, <i>P. patens </i> should use a different mechanism to assist the folding of NdhH, as it also contains the chloroplast NDH complex. </p> <p>To study whether the Cpn60β subunits in bryophytes can facilitate the folding of NdhH, we introduced the <i>Cpn60β</i> gene isolated from liverwort ( <i>Marchantia polymorpha</i>) into <i>crr27-1</i> ( <a href="#pbio-1001040-g005">Figure 5B</a>). We identified only one gene copy encoding Cpn60β in <i>M. polymorpha </i>, possibly due to incomplete genome information. Immunoblots detected a trace amount of NdhH in the thylakoids of three transgenic lines, although the levels of MpCpn60β were comparable to those of AtCpn60β4 in <i>crr27-1</i> transformed by <i>AtCpn60β4-HA</i> ( <a href="#pbio-1001040-g005">Figure 5B</a>), suggesting that MpCpn60β partially rescues the phenotype of <i>crr27-1</i> and that Cpn60β in bryophytes retains its ability in assisting the folding of NdhH. As MpCpn60β forms the heterologous Cpn60 complex with AtCpn60α, we could not compare the efficiency of the NdhH folding with the complex including AtCpn60β. </p> </div> </div> <div class="discussion" title="sec"> <div class="title" tagx="title" title="title">
Discussion</div> <p>Although multiple chaperonin genes are present in a significant proportion of bacteria and eukaryotes, the function and biological significance of this kind of divergent evolution have yet to be revealed <a href="#pbio.1001040-Lund1">[13]</a>. Generally, it is thought that the major subunits fulfill the housekeeping chaperonin function. The minor chaperonin subunits may increase the general chaperoning ability by elevating the chaperonin abundance in response to different environmental conditions. In this study, we demonstrated that the highly divergent chloroplast chaperonin subunit Cpn60β4 is specifically and strictly required for the folding of the NDH subunit NdhH. </p> <p>Although Cpn60β4 is highly divergent from the major Cpn60β subunits ( <a href="#pbio-1001040-g005">Figure 5</a>), it is an intrinsic chaperonin subunit and forms a heterooligomeric Cpn60 complex with Cpn60α1 and other Cpn60β subunits ( <a href="#pbio-1001040-g003">Figure 3 </a> and <a href="#pbio-1001040-t001">Table 1</a>), suggesting the involvement of this specific Cpn60 complex in assisting the folding of some proteins. In line with this idea and the <i>crr27</i> phenotype, the NDH subunit NdhH was copurified with the heterooligomeric Cpn60 complex including Cpn60β4 ( <a href="#pbio-1001040-t001">Table 1 </a>). Although the Cpn60 complex lacking Cpn60β4 also interacts with NdhH with less affinity ( <a href="#pbio-1001040-t001">Tables 1</a> and <a href="#pbio.1001040.s011">S2</a>), it cannot produce native NdhH for further NDH complex assembly ( <a href="#pbio-1001040-g001">Figures 1</a> and <a href="#pbio-1001040-g002">2</a>), implying that Cpn60β4 is required for both high-affinity binding and folding of NdhH. In contrast with the observation in <i>R. meliloti</i>, <i>B. japonicum</i>, and <i>R. leguminosarum </i> <a href="#pbio.1001040-Ogawa1">[15]</a>– <a href="#pbio.1001040-Gould1">[17]</a>, in which the function of the unusual chaperonins can be partially replaced by other chaperonins, Cpn60β4 is absolutely required for the folding of NdhH. These data support the proposal that multiple chaperonin subunits have evolved to assist the folding of specific proteins, although the functional specialization is not absolute in some cases. </p> <p>What is the structural basis for the functional specialization of Cpn60β4? We clarified that the unusual C-terminus of Cpn60β4 is required for the full activity of the Cpn60 complex containing Cpn60β4 for folding NdhH. The aforementioned GroEL1 protein in <i>M</i>. <i>smegmatis</i> also has an unusual histidine-rich C-terminus, which was shown to be essential for the specific function of GroEL1 <a href="#pbio.1001040-Ojha1">[18]</a>. These observations suggest that modification of the C-terminus is necessary to facilitate the folding of specific targets. This idea is consistent with the fact that the many bacterial genomes encode an additional chaperonin with an unusual C-terminus <a href="#pbio.1001040-Lund1">[13]</a>. </p> <p>The C-terminus of Cpn60β4 is not required for the formation of the specific chaperonin complex containing Cpn60β4 ( <a href="#pbio-1001040-g004">Figure 4C</a>). Thus, it should have some special functions in other steps during the folding of NdhH. The chaperonin complex containing Cpn60β4 and its substrate NdhH can be purified via the HA tag fused with the C-terminus of Cpn60β4 ( <a href="#pbio-1001040-t001">Table 1</a> and <a href="#pbio-1001040-g003">Figure 3C </a>). As the <i>cis</i> ring is capped by a co-chaperone, it is likely that the C-terminal tail of Cpn60β4 extends from the <i>trans</i> ring so that the chaperonin complex containing Cpn60β4 can be trapped by microbeads coupled with HA antibody. Alternatively, the chaperonin complex was purified by the C-terminal tail extruding from the <i>cis</i> ring, which was not capped but already associated with the substrate NdhH. The protruded C-terminal tail of Cpn60β4 might promote the high affinity binding with NdhH, ensuring that the nonnative NdhH can be efficiently captured by the Cpn60 complex containing Cpn60β4. In addition, enclosure of the nonnative NdhH protein inside the cavity by a co-chaperonin will lead to the encapsulation of the C-terminus. Consequently, the C-terminus might also contribute to the specific function of Cpn60β4 in assisting the folding of NdhH in the cavity. It has been reported that changing the length of the C-terminus of GroEL can affect the folding speed of some substrates <a href="#pbio.1001040-Tang1">[43]</a>, <a href="#pbio.1001040-Tang2">[47]</a>. Farr et al. provided further evidence that the elongated C-terminus perturbed the ATPase activity, and the disturbance of the rate of ATP hydrolysis resulted in the modification of the folding rate of some substrates <a href="#pbio.1001040-Farr1">[48]</a>. The physical properties of the C-terminus <a href="#pbio.1001040-Suzuki2">[49]</a> such as the length <a href="#pbio.1001040-Tang1">[43]</a>, <a href="#pbio.1001040-Tang2">[47]</a>, hydrophilicity <a href="#pbio.1001040-Machida1">[50]</a>, and hydrophobicity <a href="#pbio.1001040-Tang1">[43]</a> have been proposed to be critical for the substrate folding in the cavity. We also found that the C-terminus of the Cpn60β4 is rich in positively charged residues ( <a href="#pbio-1001040-g004">Figure 4A</a>). Thus, it is likely that these unusual C-termini provide specific environments in the chaperonin cavity and/or modify the chaperonin ATPase rate of the chaperonin complex. </p> <p>The deletion of up to 27 residues of the C-terminal tail of GroEL does not affect the growth of <i>E. coli</i> <a href="#pbio.1001040-McLennan1">[51]</a>, <a href="#pbio.1001040-McLennan2">[52]</a>, suggesting that the C-terminal motif does not play an essential role in assisting the folding of substrates. It is also true that Cpn60β4 lacking its unusual C-terminal tail still can partially assist the folding of NdhH ( <a href="#pbio-1001040-g004">Figure 4B</a>). Our results showed that the charged residues exposed on the cavity wall do not play a critical role for the specific function ( <a href="#pbio-1001040-g004">Figure 4F</a>). In addition, we discovered many residues that are potentially important for the specific function of Cpn60β4 ( <a href="#pbio.1001040.s007">Figures S7</a> and <a href="#pbio.1001040.s008">S8</a>). Among them, several residues are mapped to the intermediate domain or near the ATP/ADP binding site ( <a href="#pbio.1001040.s008">Figure S8 </a>). The <i>E. coli</i> GroEL with specific mutations in this region can improve the folding activity for green fluorescent protein (GFP), most likely through the adjustment of the ATPase activity <a href="#pbio.1001040-Wang1">[53]</a>. Recently, the apical domain of GroEL1 from <i>Mycobacterium tuberculosis</i> was shown to be sufficient for binding the specific substrate KasA <a href="#pbio.1001040-Sielaff1">[54]</a>. Notably, some residues are specifically conserved in the apical domain of the putative Cpn60β4 orthologs ( <a href="#pbio.1001040.s008">Figure S8 </a>). In addition, other conserved residues in AtCpn60β4 orthologs may be required for the formation of the Cpn60 complex including Cpn60β4 with certain stoichiometry or to provide certain physical features of the cavity wall. With the exception of the C-terminal extension, we could not specify the residues that are required for Cpn60β4 function. It is likely that some residues that are specifically conserved in the putative Cpn60β4 orthologs cumulatively contribute to their specific function, and this drastic evolution may have been necessary to assist the folding of NdhH. However, it is puzzling that MpCpn60β, which is related to the major AtCpn60β subunits, partly complemented the function of Cpn60β4 and that bryophytes do not contain the Cpn60β4 orthlogs ( <a href="#pbio-1001040-g005">Figure 5</a>). </p> <p>Specific mutations of GroEL can improve the folding activity of a specific protein <a href="#pbio.1001040-Wang1">[53]</a>. However, mutated GroEL has a reduced ability to fold a variety of natural substrates, suggesting a conflict between the specific ability of GroEL to fold particular substrates and its general ability in assisting the folding of a wide range of substrates <a href="#pbio.1001040-Wang1">[53]</a>. By combining all of the features acquired during the evolution of plants, Cpn60β4 allows the Cpn60 complex to assist the folding of the specific substrate, NdhH. However, other Cpn60 subunits, especially the major Cpn60β proteins, may have become optimized to support the efficient folding of other obligate substrates. Through this kind of divergent evolution, the chaperonin system can resolve the apparent conflict between specialization and generalization of its function. </p> </div> <div class="materials_methods" title="materials|methods"> <div class="title" tagx="title" title="title">
Materials and Methods</div> <div class="plantmaterialandgrowthconditions" title="sec"> <div class="title" tagx="title" title="title">
Plant Material and Growth Conditions</div> <p> <i>Arabidopsis thaliana</i> (ecotypes Col-0 and Nössen) plants were grown in a growth chamber (50 µmol photons m <sup>−2</sup> s <sup>−1</sup>, 16 h photoperiod, 23°C) for 3 to 4 wk. <i>cpn60α1</i> was mutagenized by ethyl methanesulfonate <a href="#pbio.1001040-Hashimoto1">[55]</a>. <i>crr27-3</i> (ecotypes Nössen) was isolated from a collection of <i>Ds</i> transposon insertion lines <a href="#pbio.1001040-Kuromori1">[36]</a>. <i>crr27-1</i> (SALK_136518, Col-0) and <i>crr27-2</i> (SALK_064887, Col-0) were obtained from the ABRC Stock Center. For complementation experiments, vectors were transferred into <i>Agrobacterium tumefaciens </i> C58C by electroporation, and the bacteria were used to transform <i>crr27-1</i> by floral dipping. </p> </div> <div class="chlorophyllfluorescenceanalysis" title="sec"> <div class="title" tagx="title" title="title">
Chlorophyll Fluorescence Analysis</div> <p>Chlorophyll fluorescence was measured using a MINI-PAM (pulse amplitude modulation) portable chlorophyll fluorometer (Walz, Effeltrich, Germany). The transient increase in chlorophyll fluorescence after turning off AL was monitored as previously described <a href="#pbio.1001040-Shikanai2">[33]</a>. Leaves were exposed to AL (50 µmol photons m <sup>−2</sup> s <sup>−1</sup>) for 5 min. AL was turned off and the subsequent transient rise in fluorescence ascribed to NDH activity was monitored. Fluorescence levels were standardized to the maximum fluorescence levels of closed PSII ( <i>Fm</i>) by applying saturating-light pulses (SP). </p> <p>To investigate the light intensity dependence of two chlorophyll fluorescence parameters, ETR and NPQ, measuring light (650 nm, 0.1 µmol photons m <sup>−2</sup> s <sup>−1</sup>) was used to excite the minimum fluorescence at open PSII centers in the dark-adapted state ( <i>F <sub>o</sub> </i>). A saturating pulse of white light (800 ms, 8,000 µmol photons m <sup>−2</sup> s <sup>−1</sup>) was applied to determine the maximum fluorescence at closed PSII centers in the dark ( <i>Fm</i>) or during light illumination ( <i>Fm’</i>). The steady-state fluorescence level ( <i>Fs</i>) was recorded during AL illumination (15–1,000 µmol photons m <sup>−2</sup> s <sup>−1</sup>). These photosynthetic parameters were recoded 2 min after the change of AL intensity. Φ <sub>PSII</sub> was calculated as ( <i>Fm′</i> – <i>Fs</i>)/ <i>Fm′</i>. ETR and NPQ were calculated as Φ <sub>PSII</sub>×photon flux density (µmol photons m <sup>−2</sup> s <sup>−1</sup>) and ( <i>Fm</i> – <i>Fm′</i>)/ <i>Fm′</i>, respectively. </p> </div> <div class="thylakoidmembraneisolation,bn-page,cn-page,andimmunoblotanalysis" title="sec"> <div class="title" tagx="title" title="title">
Thylakoid Membrane Isolation, BN-PAGE, CN-PAGE, and Immunoblot Analysis </div> <p>Freshly isolated chloroplasts were osmotically ruptured in buffer containing 20 mM HEPES–KOH (pH 7.6), 5 mM MgCl <sub>2</sub>, and 2.5 mM EDTA. Thylakoid membranes were separated from the stromal fraction by centrifugation (17,000 g for 10 min at 4°C). CN-PAGE, BN-PAGE, and 2-D CN/SDS-PAGE were performed according to previous reports <a href="#pbio.1001040-Peltier1">[27]</a>, <a href="#pbio.1001040-Peng2">[38]</a>. For immunoblot analysis, thylakoid and stromal proteins were loaded by equal chlorophyll and protein content, respectively. The stromal protein contents were determined with a Bio-Rad Protein Assay Kit (cat. No. 500-0006). Immunoblot signals were detected with an ECL plus Western Blotting Detection Kit (GE Healthcare) and visualized with a Luminescent image analyzer (LAS)-3000 (Fuji Film). Immunoblots were quantified by Imagemaster software (Amersham Pharmacia Biotech). </p> </div> <div class="rnaisolationandrt-pcranalysis" title="sec"> <div class="title" tagx="title" title="title">
RNA Isolation and RT-PCR Analysis</div> <p>Total RNA was isolated from <i>Arabidopsis</i> leaves with an RNeasy Plant Mini Kit (Qiagen) and treated with DNase I (Invitrogen). Total RNA (5 µg) was reverse transcribed using the SuperScript III First-Strand Synthesis System (Invitrogen) in a total volume of 20 µl. The cDNA was used in 35 cycles of PCR with the following primers: <span class="named-content" title="named-content">5′-TGGCTCTGTCACCAAGAAGCTTCAG-3′</span> and <span class="named-content" title="named-content">5′-GCTTTCTGGGTGAATCCGTTGGTAA-3′</span>. RT-PCR products were separated on agarose gels and detected by ethidium bromide staining. </p> </div> <div class="affinitychromatographyofthechaperonin-substratecomplexes" title="sec"> <div class="title" tagx="title" title="title">
Affinity Chromatography of the Chaperonin-Substrate Complexes</div> <p>Cpn60-substrate complexes were isolated from <i>crr27-1</i> plants expressing HA-tagged Cpn60β subunits with the µMACS HA isolation kit under the conditions previously established <a href="#pbio.1001040-Kerner1">[11]</a>, with a minor modification. Freshly isolated chloroplasts were osmotically ruptured in lysis buffer (50 mM Tris-HCl pH 8.0, 0.01% Tween 20, 10 mM MgCl <sub>2</sub>, 20 mM glucose, 20 U/ml hexokinase) plus protease inhibitors (Complete mini, Roche). Within 10 s after lysis, ADP was added to a final concentration of 10 mM. Thylakoid membranes were pelleted by centrifugation at 20,000 g for 10 min at 4°C and the supernatants were transferred to new tubes. NaCl was added to the supernatants to a final concentration of 150 mM and then mixed with 50 µl anti-HA MicroBeads (Miltenyi Biotec.). After incubation for 2 h at 4°C, the beads were transferred to columns placed in a magnetic field. Columns were rinsed four times with 200 µl washing buffer I (50 mM Tris-HCl pH 8.0, 1% Triton, 0.5% Sodium deoxycholate, 150 mM NaCl, 5 mM ADP) and twice with 200 µl washing buffer II (50 mM Tris-HCl pH 8.0, 1% Triton, 150 mM NaCl, 5 mM ADP). After final washing with 200 µl washing buffer III (25 mM Tris-HCl pH 7.5, 5 mM ADP), the chaperonin-substrate complex was eluted with elution buffer (50 mM Tris-HCl pH 6.8, 50 mM DTT, 1% SDS, 1 mM EDTA, 0.005% bromophenol blue, 10% glycerol). Total protein was separated on 12.5% SDS-PAGE gels (Perfect NT Gel, DRC) and stained with CBB. SDS-PAGE lanes were cut into four slices and analyzed by LC-MS/MS analyses. </p> </div> <div class="massspectrometricanalysisanddatabasesearching" title="sec"> <div class="title" tagx="title" title="title">
Mass Spectrometric Analysis and Database Searching</div> <p>Peptide Preparation and LC-MS/MS analyses were performed as previously described <a href="#pbio.1001040-Peng3">[39]</a>. The excised bands were treated twice with 25 mM ammonium bicarbonate in 30% (v/v) acetonitrile for 10 min and 100% (v/v) acetonitrile for 15 min, and then dried in a vacuum concentrator. The dried gel pieces were treated with 0.01 mg/ml trypsin (sequence grade; Promega)/50 mM ammonium bicarbonate and incubated at 37°C for 16 h. The digested peptides were recovered twice with 20 µl 5% (v/v) formic acid/50% (v/v) acetonitrile. The extracted peptides were combined and then dried in a vacuum concentrator. LC-MS/MS analyses were performed on an LTQ-Orbitrap XL-HTC-PAL system. MS/MS spectra were compared by the MASCOT server (v. 2.2) against TAIR8 (The Arabidopsis Information Resource) with the following search parameters: set-off threshold at 0.05 in the ion-score cut-off; peptide tolerance, 10 ppm; MS/MS tolerance, ±0.8 Da; peptide charge, 2+ or 3+; trypsin as enzyme allowing up to one missed cleavage; carboxymethylation on cysteines as a fixed modification; and oxidation on methionine as a variable modification. </p> </div> <div class="phylogeneticanalysis" title="sec"> <div class="title" tagx="title" title="title">
Phylogenetic Analysis</div> <p>Chaperonin protein sequences were first aligned using the CLUSTALX 1.83 program <a href="#pbio.1001040-Thompson1">[56]</a>. The protein alignment was further refined manually and 534 conserved sites were used for phylogenetic analysis. Phylogenetic trees were constructed by using both the maximum likelihood (ML) and Bayesian methods to ensure the robustness of our analysis. ML trees were constructed by using PHYML version 2.4 <a href="#pbio.1001040-Guindon1">[57]</a> with WAG model selected via MODELTEST 3.06 <a href="#pbio.1001040-Posada1">[58]</a>, and support for each branch was assessed using bootstrap analyses with 100 bootstrap replicates. Bayesian trees were constructed using MrBayes software <a href="#pbio.1001040-Huelsenbeck1">[59]</a> with the WAG model. Four chains of Markov chain Monte Carlo were run, sampling one tree every 1,000 generations for 1,000,000 generations, starting with a random tree. The first 50,000 generations were excluded as burn-in to ensure that the chains reached stationary. The posterior probability was used to estimate nodal robustness. </p> </div> <div class="homologymodeling" title="sec"> <div class="title" tagx="title" title="title">
Homology Modeling</div> <p>The structure model of the Cpn60β4 protein was obtained by homology modeling using the SWISS-MODEL server ( <a href="http://swissmodel.expasy.org/">http://swissmodel.expasy.org/</a>) <a href="#pbio.1001040-Arnold2">[60]</a>, and the crystal structure of <i>E. coli</i> GroEL (PDB 1AON, Chain A) was used as a modeling template. The 3-D predicted structure was visualized using the PyMol software. </p> </div> <div class="accessionnumbers" title="sec"> <div class="title" tagx="title" title="title">
Accession Numbers</div> <p>Sequence data from this article can be found in the Arabidopsis Genome Initiative or GenBank/EMBL databases under the following accession numbers: At ( <i>Arabidopsis thaliana</i>) Cpn60α1 (At2g28000), AtCpn60α2 (At5g18820), AtCpn60β1 (At1g55490), AtCpn60β2 (At3g13470), AtCpn60β3 (At5g56500), AtCpn60β4 (At1g26230), AtNdhH (BAA84443), Al ( <i>Arabidopsis lyrata</i>) Cpn60α1 (481708), AlCpn60α2 (488768), AlCpn60β1 (474606), AlCpn60β3 (495739), AlCpn60β4 (890123), Sy ( <i>Synechocystis</i> SP. PCC 6803) GroEL-1 (NP_440731), SyGroEL-2 (NP_442170), SyNdhH (CAA43057), GroEL ( <i>Escherichia coli </i>) (NP_418567), and Pp ( <i>Physcomitrella patens</i>) NdhH (BAC85094). </p> </div> </div> <div class="supplementary-material" title="supplementary-material"> <div class="title" tagx="title" title="title">
Supporting Information</div> <div class="local-data" title="local-data"> <span class="label" tagx="label" title="label">Figure S1</span>  <p>Protein blot analysis of chloroplast proteins. Equal amounts of total protein extracted from the leaves of WT, <i>cpn60α1</i>, and <i>crr27-1</i> plants were separated by SDS-PAGE and immunodetected by antibodies against stromal proteins Cpn60α, Cpn60β, and RbcL (A) or thylakoid proteins NdhH, NDH18, NDF1, FKBP16-2, PsaA, Cyt <i>f</i>, D1, and LHCII (B), which represent the thylakoid protein complexes indicated. </p> <p>(TIF)</p>  <div class="media" title="media"> <a href="pbio.1001040.s001.tif">LINK</a>  <p>Click here for additional data file.</p>  </div> </div> <div class="local-data" title="local-data"> <span class="label" tagx="label" title="label">Figure S2</span>  <p>In vivo analysis of electron transport activity. (A) Light-intensity dependence of ETR. ETR is depicted relative to the maximum value of Φ <sub>PSII</sub> × light intensity (μmol photons m <sup>−2</sup> s <sup>−1</sup>) in the WT (100%). (B) Light-intensity dependence of NPQ of chlorophyll fluorescence. All values represent the mean ± SD ( <i>n</i> = 5) in (A) and (B). </p> <p>(TIF)</p>  <div class="media" title="media"> <a href="pbio.1001040.s002.tif">LINK</a>  <p>Click here for additional data file.</p>  </div> </div> <div class="local-data" title="local-data"> <span class="label" tagx="label" title="label">Figure S3</span>  <p>Analysis of thylakoid and stromal protein complexes by native PAGE. (A) Freshly isolated thylakoid membranes from WT (ecotype Col-0 and Nössen), <i>crr27</i>, and <i>ndhl</i> plants were solubilized in 1% <i>n</i>-dodecyl-β- <div class="sc_UNKNOWN">
d</div>-maltoside (DM) at a chlorophyll concentration of 1 µg μl <sup>−1</sup>, and the protein sample was separated by 5%–12% BN-PAGE. The major protein complexes were assigned based on a previous report <a href="#pbio.1001040-Peng3">[39]</a>. Band I, NDH-PSI supercomplex detected in WT; Band II, sub-NDH-PSI supercomplex detected in the <i>crr27</i> and <i>ndhl</i> mutants. (B) Stromal protein complexes isolated from WT and <i>crr27-1</i> plants were separated by CN-PAGE followed by 2-dimensional SDS-PAGE. After electrophoresis, protein was stained with CBB. The positions of the molecular markers are indicated above the gel. The Cpn60 complex was assigned on the basis of a previous report <a href="#pbio.1001040-Peltier1">[27]</a>. </p> <p>(TIFF)</p>  <div class="media" title="media"> <a href="pbio.1001040.s003.tiff">LINK</a>  <p>Click here for additional data file.</p>  </div> </div> <div class="local-data" title="local-data"> <span class="label" tagx="label" title="label">Figure S4</span>  <p>Detection of Cpn60β4 in WT plants. (A) CN-PAGE analysis of stromal protein complexes isolated from WT plants. After electrophoresis, the CN-gel was stained with CBB. The protein band corresponding to the position of the Cpn60 complex (the boxed area indicated by an arrow) was excised from the gel and analyzed by LC-MS/MS analysis. (B) Summary of the chaperonin subunits detected in the excised band. (C) The peptides of Cpn60β4 detected in the excised band by MS analysis. Total proteins detected are listed in <a href="#pbio.1001040.s010">Table S1</a>. </p> <p>(TIFF)</p>  <div class="media" title="media"> <a href="pbio.1001040.s004.tiff">LINK</a>  <p>Click here for additional data file.</p>  </div> </div> <div class="local-data" title="local-data"> <span class="label" tagx="label" title="label">Figure S5</span>  <p>Immunoaffinity purification of Cpn60-substrate complexes. (A) Chaperonin complexes containing Cpn60β1 and Cpn60β4 subunits were purified from <i>crr27-1</i> mutants expressing HA-tagged Cpn60β1 and Cpn60β4, respectively. After elution, total proteins were separated by SDS-PAGE and stained with CBB. Due to the low abundance of Cpn60β4, more stromal proteins were used for the purification, leading to an increase in non-specific binding of some proteins. Lanes were cut into four slices and total protein was analyzed by LC-MS/MS analysis. CK, WT plants without any transformation were used as a negative control. (B) NdhH peptides detected from the chaperonin complexes containing Cpn60β1 (upper) and Cpn60β4 (bottom) are indicated in red bold. </p> <p>(TIFF)</p>  <div class="media" title="media"> <a href="pbio.1001040.s005.tiff">LINK</a>  <p>Click here for additional data file.</p>  </div> </div> <div class="local-data" title="local-data"> <span class="label" tagx="label" title="label">Figure S6</span>  <p>Analysis of stromal protein complexes isolated from wild-type and <i>crr27-1</i> plants. (A) Immunodetection of NdhH isolated from the stroma of wild type (WT), <i>crr27-1</i>, <i>ndhl</i>, and <i>crr27-1</i> plants complemented with Cpn60β4. Equal amounts of stromal protein (8 µg) were loaded onto each well. WT thylakoid proteins corresponding to 2.5 µg chlorophyll were also analyzed. RbcL was used as a loading control. An asterisk represents non-specific signals. (B) Stromal protein complexes isolated from WT and <i>crr27-1</i> plants were separated by CN-PAGE followed by 2-dimensional SDS-PAGE. The proteins were immunodetected with specific antibodies against NdhH and RbcL. Vertical arrows indicate the positions of three assembly intermediates, which include NdhH. </p> <p>(TIF)</p>  <div class="media" title="media"> <a href="pbio.1001040.s006.tif">LINK</a>  <p>Click here for additional data file.</p>  </div> </div> <div class="local-data" title="local-data"> <span class="label" tagx="label" title="label">Figure S7</span>  <p>Protein alignment of the mature chaperonin proteins. The origin of all chaperonin subunits is shown in <a href="#pbio-1001040-g005">Figure 5</a>. GroEL from <i>E. coli</i> was also included in the alignment. Residues involved in ATP/ADP and Mg <sup>2+</sup> binding are marked with +. Red and blue sharps (#) represent the residues for the substrate binding proposed by Fenton et al. <a href="#pbio.1001040-Fenton2">[61]</a> and Buckle et al. <a href="#pbio.1001040-Buckle1">[62]</a>, respectively. The charged residues exposed to the central cavity in the <i>cis</i> ring are labeled with asterisks. The residues marked with red asterisks were selected for introducing the mutations as shown in <a href="#pbio-1001040-g004">Figure 4F</a>. Amino acid residues that are highly conserved in the putative Cpn60β4 orthologs and with different properties from the corresponding residues in major Cpn60β subunits are marked with solid red circles under the protein sequences. These residues were also highlighted as spheres in the structure model of Cpn60β4 in <a href="#pbio.1001040.s008">Figure S8 </a>. The amino acid mutated in the <i>cpn60α1</i> mutant is labeled with a triangle. Positions of the deletion and the C-terminal swapping are shown. </p> <p>(TIFF)</p>  <div class="media" title="media"> <a href="pbio.1001040.s007.tiff">LINK</a>  <p>Click here for additional data file.</p>  </div> </div> <div class="local-data" title="local-data"> <span class="label" tagx="label" title="label">Figure S8</span>  <p>Predicted structure of the <i>Arabidopsis</i> Cpn60β4 subunit. A 3-dimensional (3D) structural model of <i>Arabidopsis</i> Cpn60β4 was generated with the Swiss Model alignment server. The structure model of <i>E. coli</i> GroEL (PDB 1AON, Chain A) was used as a modeling template. Predicted 3-D structure was displayed by using the PyMOL software. The apical domain (Ap), intermediate domain (Int), and equatorial domain (Eq) are shown in cyan, green, and blue, respectively. Highly conserved ATP/ADP and Mg <sup>2+</sup> binding sites are marked with magenta spheres. Residues that are highly conserved in Cpn60β4 subunits but exhibit different properties with the corresponding residues in other Cpn60β are highlighted with yellow and red spheres. Among them, sites marked with red, blue, and cyan arrows represent residues near the ATP binding site or in the intermediate domain, residues exposed to the central cavity in the <i>cis</i>-conformation of the chaperonin complex, and one residue involved in the substrate binding, respectively. Sites marked with red spheres (indicated with green arrows) were selected for introducing the mutations shown in <a href="#pbio-1001040-g004">Figure 4F</a>. </p> <p>(TIFF)</p>  <div class="media" title="media"> <a href="pbio.1001040.s008.tiff">LINK</a>  <p>Click here for additional data file.</p>  </div> </div> <div class="local-data" title="local-data"> <span class="label" tagx="label" title="label">Figure S9</span>  <p>Structure and evolution of NdhH. (A) Ribbon models of the Complex I Subunit Nqo4 (PDB 2fugV) from <i>Thermus thermophilus</i>. Nqo4 corresponds to NdhH in the chloroplast NDH complex. (B) A predicted structure of the <i>Arabidopsis</i> NdhH. Homology models were generated with the Swiss Model alignment server. The α-helix is shown in red, β-sheet in blue, and other domains in yellow. (C) Protein alignment of the NdhH subunits from <i>Arabidopsis thaliana</i> (AtNdhH), <i>Physcomitrella patens</i> (PpNdhH), and <i>Synechocystis</i> SP. PCC 6803 (SyNdhH). </p> <p>(TIFF)</p>  <div class="media" title="media"> <a href="pbio.1001040.s009.tiff">LINK</a>  <p>Click here for additional data file.</p>  </div> </div> <div class="local-data" title="local-data"> <span class="label" tagx="label" title="label">Table S1</span>  <p>Total protein detected in the excised band corresponding to the Cpn60 complex in CN-gel. </p> <p>(XLS)</p>  <div class="media" title="media"> <a href="pbio.1001040.s010.xls">LINK</a>  <p>Click here for additional data file.</p>  </div> </div> <div class="local-data" title="local-data"> <span class="label" tagx="label" title="label">Table S2</span>  <p>Total protein detected in the WT-IP, Cpn60β1-IP, and Cpn60β4-IP fractions. </p> <p>(XLS)</p>  <div class="media" title="media"> <a href="pbio.1001040.s011.xls">LINK</a>  <p>Click here for additional data file.</p>  </div> </div> <div class="local-data" title="local-data"> <span class="label" tagx="label" title="label">Table S3</span>  <p> <i>r</i> value between <i>Cpn60β4</i> and <i>Cpn60α1</i> with NDH complex-related or chaperonin subunit genes. </p> <p>(DOC)</p>  <div class="media" title="media"> <a href="pbio.1001040.s012.doc">LINK</a>  <p>Click here for additional data file.</p>  </div> </div> <div class="local-data" title="local-data"> <span class="label" tagx="label" title="label">Dataset S1</span>  <p>Protein sequences used for alignment and phylogenetic tree construction.</p> <p>(DOC)</p>  <div class="media" title="media"> <a href="pbio.1001040.s013.doc">LINK</a>  <p>Click here for additional data file.</p>  </div> </div> </div> </div> <div class="back" title="back"> <div class="ack" title="ack"> <p>We are grateful to M. Hayer-Hartl for her critical reading of this manuscript, to D. Gao and M. Hasebe for their help and advice in the phylogenetic analyses, and to F. Kessler for valuable discussion. We would also like to thank T. Kohchi for providing the information regarding the genome of <i>Marchantia polymorpha</i>. We also thank K. Tamura for help in purification of the chaperonin-substrate complexes, A. Tahara and M. Okamura for their technical assistance, and T. Endo, H. Mi, A. Makino, and M. Nakai for providing antibodies. </p> </div> <div class="fn-group" title="fn-group"> <div class="fn-type-COI-statement" title="COI-statement"> <p>The authors have declared that no competing interests exist.</p> </div> <div class="fn-type-financial-disclosure" title="financial-disclosure"> <p>This work was supported by Grants 17GS0316, 22114509, 22247005 from the Ministry of Education, Culture, Sports, Science, and Technology of Japan and a grant from the Ministry of Agriculture, Forestry, and Fisheries of Japan (Genomics for Agricultural Innovation; GPN0008). The funders had no role in study design, data collection and analysis, decision to publish, or preparation of the manuscript. </p> </div> </div> <div class="references">
References</div> <div tag="ref-list"> <ul> <div class="title" tagx="title" title="title">
References</div> <li tag="ref"> <a name="pbio.1001040-Frydman1" /> <span class="label" tagx="label" title="label">1</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Frydman</span> <span class="given-names" tagx="given-names" title="given-names">J</span> </span> </span> <span class="year" tagx="year" title="year">2001</span> <span class="mixed-article-title" title="mixed-article-title">Folding of newly translated proteins in vivo: the role of molecular chaperones. </span> <span class="source" tagx="source" title="source">Annu Rev Biochem</span> <span class="volume" tagx="volume" title="volume">70</span> <span class="fpage" tagx="fpage" title="fpage">603</span> <span class="lpage" tagx="lpage" title="lpage">647</span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/11395418">11395418</a> </span> </span> </li> <li tag="ref"> <a name="pbio.1001040-Hartl1" /> <span class="label" tagx="label" title="label">2</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Hartl</span> <span class="given-names" tagx="given-names" title="given-names">F. U</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Hayer-Hartl</span> <span class="given-names" tagx="given-names" title="given-names">M</span> </span> </span> <span class="year" tagx="year" title="year">2002</span> <span class="mixed-article-title" title="mixed-article-title">Molecular chaperones in the cytosol: from nascent chain to folded protein. </span> <span class="source" tagx="source" title="source">Science</span> <span class="volume" tagx="volume" title="volume">295</span> <span class="fpage" tagx="fpage" title="fpage">1852</span> <span class="lpage" tagx="lpage" title="lpage">1858</span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/11884745">11884745</a> </span> </span> </li> <li tag="ref"> <a name="pbio.1001040-Fenton1" /> <span class="label" tagx="label" title="label">3</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Fenton</span> <span class="given-names" tagx="given-names" title="given-names">W. A</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Horwich</span> <span class="given-names" tagx="given-names" title="given-names">A. L</span> </span> </span> <span class="year" tagx="year" title="year">2003</span> <span class="mixed-article-title" title="mixed-article-title">Chaperonin-mediated protein folding: fate of substrate polypeptide. </span> <span class="source" tagx="source" title="source">Q Rev Biophys</span> <span class="volume" tagx="volume" title="volume">36</span> <span class="fpage" tagx="fpage" title="fpage">229</span> <span class="lpage" tagx="lpage" title="lpage">256</span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/14686103">14686103</a> </span> </span> </li> <li tag="ref"> <a name="pbio.1001040-Horwich1" /> <span class="label" tagx="label" title="label">4</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Horwich</span> <span class="given-names" tagx="given-names" title="given-names">A. L</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Fenton</span> <span class="given-names" tagx="given-names" title="given-names">W. A</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Chapman</span> <span class="given-names" tagx="given-names" title="given-names">E</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Farr</span> <span class="given-names" tagx="given-names" title="given-names">G. W</span> </span> </span> <span class="year" tagx="year" title="year">2007</span> <span class="mixed-article-title" title="mixed-article-title">Two families of chaperonin: physiology and mechanism. </span> <span class="source" tagx="source" title="source">Annu Rev Cell Dev Biol</span> <span class="volume" tagx="volume" title="volume">23</span> <span class="fpage" tagx="fpage" title="fpage">115</span> <span class="lpage" tagx="lpage" title="lpage">145</span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/17489689">17489689</a> </span> </span> </li> <li tag="ref"> <a name="pbio.1001040-Hartl2" /> <span class="label" tagx="label" title="label">5</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Hartl</span> <span class="given-names" tagx="given-names" title="given-names">F. U</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Hayer-Hartl</span> <span class="given-names" tagx="given-names" title="given-names">M</span> </span> </span> <span class="year" tagx="year" title="year">2009</span> <span class="mixed-article-title" title="mixed-article-title">Converging concepts of protein folding <i>in vitro</i> and <i>in vivo</i>. </span> <span class="source" tagx="source" title="source">Nature Struct Mol Biol</span> <span class="volume" tagx="volume" title="volume">16</span> <span class="fpage" tagx="fpage" title="fpage">574</span> <span class="lpage" tagx="lpage" title="lpage">581</span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/19491934">19491934</a> </span> </span> </li> <li tag="ref"> <a name="pbio.1001040-Braig1" /> <span class="label" tagx="label" title="label">6</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Braig</span> <span class="given-names" tagx="given-names" title="given-names">K</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Otwinowski</span> <span class="given-names" tagx="given-names" title="given-names">Z</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Hegde</span> <span class="given-names" tagx="given-names" title="given-names">R</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Boisvert</span> <span class="given-names" tagx="given-names" title="given-names">D. C</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Joachimiak</span> <span class="given-names" tagx="given-names" title="given-names">A</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span> </span> <span class="year" tagx="year" title="year">1994</span> <span class="mixed-article-title" title="mixed-article-title">The crystal structure of the bacterial chaperonin GroEL at 2.8 A. </span> <span class="source" tagx="source" title="source">Nature</span> <span class="volume" tagx="volume" title="volume">371</span> <span class="fpage" tagx="fpage" title="fpage">578</span> <span class="lpage" tagx="lpage" title="lpage">586</span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/7935790">7935790</a> </span> </span> </li> <li tag="ref"> <a name="pbio.1001040-Saibil1" /> <span class="label" tagx="label" title="label">7</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Saibil</span> <span class="given-names" tagx="given-names" title="given-names">H. R</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Ranson</span> <span class="given-names" tagx="given-names" title="given-names">N. A</span> </span> </span> <span class="year" tagx="year" title="year">2002</span> <span class="mixed-article-title" title="mixed-article-title">The chaperonin folding machine.</span> <span class="source" tagx="source" title="source">Trends Biochem Sci</span> <span class="volume" tagx="volume" title="volume">27</span> <span class="fpage" tagx="fpage" title="fpage">627</span> <span class="lpage" tagx="lpage" title="lpage">632</span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/12468232">12468232</a> </span> </span> </li> <li tag="ref"> <a name="pbio.1001040-Xu1" /> <span class="label" tagx="label" title="label">8</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Xu</span> <span class="given-names" tagx="given-names" title="given-names">Z. H</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Horwich</span> <span class="given-names" tagx="given-names" title="given-names">A. L</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Sigler</span> <span class="given-names" tagx="given-names" title="given-names">P. B</span> </span> </span> <span class="year" tagx="year" title="year">1997</span> <span class="mixed-article-title" title="mixed-article-title">The crystal structure of the asymmetric GroEL-GroES-(ADP)7 chaperonin complex. </span> <span class="source" tagx="source" title="source">Nature</span> <span class="volume" tagx="volume" title="volume">388</span> <span class="fpage" tagx="fpage" title="fpage">741</span> <span class="lpage" tagx="lpage" title="lpage">749</span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/9285585">9285585</a> </span> </span> </li> <li tag="ref"> <a name="pbio.1001040-Mayhew1" /> <span class="label" tagx="label" title="label">9</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Mayhew</span> <span class="given-names" tagx="given-names" title="given-names">M</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Da Silva</span> <span class="given-names" tagx="given-names" title="given-names">A. C. R</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Martin</span> <span class="given-names" tagx="given-names" title="given-names">J</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Erdjument-bromage</span> <span class="given-names" tagx="given-names" title="given-names">H</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Tempst</span> <span class="given-names" tagx="given-names" title="given-names">P</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span> </span> <span class="year" tagx="year" title="year">1996</span> <span class="mixed-article-title" title="mixed-article-title">Protein folding in the central cavity of the GroEL-GroES chaperonin complex. </span> <span class="source" tagx="source" title="source">Nature</span> <span class="volume" tagx="volume" title="volume">379</span> <span class="fpage" tagx="fpage" title="fpage">420</span> <span class="lpage" tagx="lpage" title="lpage">426</span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/8559246">8559246</a> </span> </span> </li> <li tag="ref"> <a name="pbio.1001040-Weissman1" /> <span class="label" tagx="label" title="label">10</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Weissman</span> <span class="given-names" tagx="given-names" title="given-names">J. S</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Rye</span> <span class="given-names" tagx="given-names" title="given-names">H. S</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Fenton</span> <span class="given-names" tagx="given-names" title="given-names">W. A</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Beechem</span> <span class="given-names" tagx="given-names" title="given-names">J. M</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Horwich</span> <span class="given-names" tagx="given-names" title="given-names">A. L</span> </span> </span> <span class="year" tagx="year" title="year">1996</span> <span class="mixed-article-title" title="mixed-article-title">Characterization of the active intermediate of a GroEL-GroES-mediated protein folding reaction. </span> <span class="source" tagx="source" title="source">Cell</span> <span class="volume" tagx="volume" title="volume">84</span> <span class="fpage" tagx="fpage" title="fpage">481</span> <span class="lpage" tagx="lpage" title="lpage">490</span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/8608602">8608602</a> </span> </span> </li> <li tag="ref"> <a name="pbio.1001040-Kerner1" /> <span class="label" tagx="label" title="label">11</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Kerner</span> <span class="given-names" tagx="given-names" title="given-names">M. J</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Naylor</span> <span class="given-names" tagx="given-names" title="given-names">D. J</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Ishihaman</span> <span class="given-names" tagx="given-names" title="given-names">Y</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Maier</span> <span class="given-names" tagx="given-names" title="given-names">T</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Chang</span> <span class="given-names" tagx="given-names" title="given-names">H. C</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span> </span> <span class="year" tagx="year" title="year">2005</span> <span class="mixed-article-title" title="mixed-article-title">Proteome-wide analysis of chaperonin-dependent protein folding in <i>Escherichia coli</i>. </span> <span class="source" tagx="source" title="source">Cell</span> <span class="volume" tagx="volume" title="volume">122</span> <span class="fpage" tagx="fpage" title="fpage">209</span> <span class="lpage" tagx="lpage" title="lpage">220</span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/16051146">16051146</a> </span> </span> </li> <li tag="ref"> <a name="pbio.1001040-Fujiwara1" /> <span class="label" tagx="label" title="label">12</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Fujiwara</span> <span class="given-names" tagx="given-names" title="given-names">K</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Ishihama</span> <span class="given-names" tagx="given-names" title="given-names">Y</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Nakahigashi</span> <span class="given-names" tagx="given-names" title="given-names">K</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Soga</span> <span class="given-names" tagx="given-names" title="given-names">T</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Taguchi</span> <span class="given-names" tagx="given-names" title="given-names">H</span> </span> </span> <span class="year" tagx="year" title="year">2010</span> <span class="mixed-article-title" title="mixed-article-title">A systematic survey of in vivo obligate chaperonin-dependent substrates. </span> <span class="source" tagx="source" title="source">EMBO J</span> <span class="volume" tagx="volume" title="volume">29</span> <span class="fpage" tagx="fpage" title="fpage">1552</span> <span class="lpage" tagx="lpage" title="lpage">1564</span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/20360681">20360681</a> </span> </span> </li> <li tag="ref"> <a name="pbio.1001040-Lund1" /> <span class="label" tagx="label" title="label">13</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Lund</span> <span class="given-names" tagx="given-names" title="given-names">P. A</span> </span> </span> <span class="year" tagx="year" title="year">2009</span> <span class="mixed-article-title" title="mixed-article-title">Multiple chaperonins in bacteria—why so many? </span> <span class="source" tagx="source" title="source">FEMS Microbiol Rev</span> <span class="volume" tagx="volume" title="volume">33</span> <span class="fpage" tagx="fpage" title="fpage">785</span> <span class="lpage" tagx="lpage" title="lpage">800</span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/19416363">19416363</a> </span> </span> </li> <li tag="ref"> <a name="pbio.1001040-Hill1" /> <span class="label" tagx="label" title="label">14</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Hill</span> <span class="given-names" tagx="given-names" title="given-names">J. E</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Hemmingsen</span> <span class="given-names" tagx="given-names" title="given-names">S. M</span> </span> </span> <span class="year" tagx="year" title="year">2001</span> <span class="mixed-article-title" title="mixed-article-title"> <i>Arabidopsis thaliana</i> type I and type II chaperonins. </span> <span class="source" tagx="source" title="source">Cell Stress &amp;amp; Chaperones</span> <span class="volume" tagx="volume" title="volume">6</span> <span class="fpage" tagx="fpage" title="fpage">190</span> <span class="lpage" tagx="lpage" title="lpage">200</span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/11599560">11599560</a> </span> </span> </li> <li tag="ref"> <a name="pbio.1001040-Ogawa1" /> <span class="label" tagx="label" title="label">15</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Ogawa</span> <span class="given-names" tagx="given-names" title="given-names">J</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Long</span> <span class="given-names" tagx="given-names" title="given-names">S. R</span> </span> </span> <span class="year" tagx="year" title="year">1995</span> <span class="mixed-article-title" title="mixed-article-title">The <i>Rhizobium meliloti groEL</i>c locus is required for regulation of early <i>nod</i> genes by the transcription activator NodD. </span> <span class="source" tagx="source" title="source">Gene Dev</span> <span class="volume" tagx="volume" title="volume">9</span> <span class="fpage" tagx="fpage" title="fpage">714</span> <span class="lpage" tagx="lpage" title="lpage">729</span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/7729688">7729688</a> </span> </span> </li> <li tag="ref"> <a name="pbio.1001040-Fischer1" /> <span class="label" tagx="label" title="label">16</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Fischer</span> <span class="given-names" tagx="given-names" title="given-names">H. M</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Schneider</span> <span class="given-names" tagx="given-names" title="given-names">K</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Babst</span> <span class="given-names" tagx="given-names" title="given-names">M</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Hennecke</span> <span class="given-names" tagx="given-names" title="given-names">H</span> </span> </span> <span class="year" tagx="year" title="year">1999</span> <span class="mixed-article-title" title="mixed-article-title">GroEL chaperonins are required for the formation of a functional nitrogenase in <i>Bradyrhizobium japonicum</i>. </span> <span class="source" tagx="source" title="source">Arch Microbiol</span> <span class="volume" tagx="volume" title="volume">171</span> <span class="fpage" tagx="fpage" title="fpage">279</span> <span class="lpage" tagx="lpage" title="lpage">289</span> </span> </li> <li tag="ref"> <a name="pbio.1001040-Gould1" /> <span class="label" tagx="label" title="label">17</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Gould</span> <span class="given-names" tagx="given-names" title="given-names">P. S</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Burgar</span> <span class="given-names" tagx="given-names" title="given-names">H. R</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Lund</span> <span class="given-names" tagx="given-names" title="given-names">P. A</span> </span> </span> <span class="year" tagx="year" title="year">2007</span> <span class="mixed-article-title" title="mixed-article-title">Homologous <i>cpn60</i> genes in <i>Rhizobium leguminosarum </i> are not functionally equivalent. </span> <span class="source" tagx="source" title="source">Cell Stress Chaperon</span> <span class="volume" tagx="volume" title="volume">12</span> <span class="fpage" tagx="fpage" title="fpage">123</span> <span class="lpage" tagx="lpage" title="lpage">131</span> </span> </li> <li tag="ref"> <a name="pbio.1001040-Ojha1" /> <span class="label" tagx="label" title="label">18</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Ojha</span> <span class="given-names" tagx="given-names" title="given-names">A</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Anand</span> <span class="given-names" tagx="given-names" title="given-names">M</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Bhatt</span> <span class="given-names" tagx="given-names" title="given-names">A</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Kremer</span> <span class="given-names" tagx="given-names" title="given-names">L</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Jacobs</span> <span class="given-names" tagx="given-names" title="given-names">W. R</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span> </span> <span class="year" tagx="year" title="year">2005</span> <span class="mixed-article-title" title="mixed-article-title">GroEL1: a dedicated chaperone involved in mycolic acid biosynthesis during biofilm formation in mycobacteria. </span> <span class="source" tagx="source" title="source">Cell</span> <span class="volume" tagx="volume" title="volume">123</span> <span class="fpage" tagx="fpage" title="fpage">861</span> <span class="lpage" tagx="lpage" title="lpage">873</span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/16325580">16325580</a> </span> </span> </li> <li tag="ref"> <a name="pbio.1001040-Pushkin1" /> <span class="label" tagx="label" title="label">19</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Pushkin</span> <span class="given-names" tagx="given-names" title="given-names">A. V</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Tsuprun</span> <span class="given-names" tagx="given-names" title="given-names">V. L</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Solovjeva</span> <span class="given-names" tagx="given-names" title="given-names">N. A</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Shubin</span> <span class="given-names" tagx="given-names" title="given-names">V. V</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Evstigneeva</span> <span class="given-names" tagx="given-names" title="given-names">Z. G</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span> </span> <span class="year" tagx="year" title="year">1982</span> <span class="mixed-article-title" title="mixed-article-title">High molecular weight pea leaf protein similar to the <i>gro</i>E protein of <i>Escherichia coli. </i> </span> <span class="source" tagx="source" title="source">Biochim Biophys Acta</span> <span class="volume" tagx="volume" title="volume">704</span> <span class="fpage" tagx="fpage" title="fpage">379</span> <span class="lpage" tagx="lpage" title="lpage">384</span> </span> </li> <li tag="ref"> <a name="pbio.1001040-Tsuprun1" /> <span class="label" tagx="label" title="label">20</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Tsuprun</span> <span class="given-names" tagx="given-names" title="given-names">V. L</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Boekema</span> <span class="given-names" tagx="given-names" title="given-names">E. J</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Samsonidze</span> <span class="given-names" tagx="given-names" title="given-names">T. G</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Pushkin</span> <span class="given-names" tagx="given-names" title="given-names">A. V</span> </span> </span> <span class="year" tagx="year" title="year">1991</span> <span class="mixed-article-title" title="mixed-article-title">Electron microscopy of the complexes of ribulose-1,5-bisphosphate carboxylase (Rubisco) and Rubisco subunit-binding protein from pea leaves. </span> <span class="source" tagx="source" title="source">FEBS Lett</span> <span class="volume" tagx="volume" title="volume">289</span> <span class="fpage" tagx="fpage" title="fpage">205</span> <span class="lpage" tagx="lpage" title="lpage">209</span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/1915848">1915848</a> </span> </span> </li> <li tag="ref"> <a name="pbio.1001040-Hemmingsen1" /> <span class="label" tagx="label" title="label">21</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Hemmingsen</span> <span class="given-names" tagx="given-names" title="given-names">S. M</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Ellis</span> <span class="given-names" tagx="given-names" title="given-names">R. J</span> </span> </span> <span class="year" tagx="year" title="year">1986</span> <span class="mixed-article-title" title="mixed-article-title">Purification and properties of ribulosebisphosphate carboxylase large subunit binding protein. </span> <span class="source" tagx="source" title="source">Plant Physiol</span> <span class="volume" tagx="volume" title="volume">80</span> <span class="fpage" tagx="fpage" title="fpage">269</span> <span class="lpage" tagx="lpage" title="lpage">276</span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/16664596">16664596</a> </span> </span> </li> <li tag="ref"> <a name="pbio.1001040-Musgrove1" /> <span class="label" tagx="label" title="label">22</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Musgrove</span> <span class="given-names" tagx="given-names" title="given-names">J. E</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Johnson</span> <span class="given-names" tagx="given-names" title="given-names">R. A</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Ellis</span> <span class="given-names" tagx="given-names" title="given-names">R. J</span> </span> </span> <span class="year" tagx="year" title="year">1987</span> <span class="mixed-article-title" title="mixed-article-title">Dissociation of the ribulosebisphosphate-carboxylase large-subunit binding protein into dissimilar subunits. </span> <span class="source" tagx="source" title="source">Eur J Biochem</span> <span class="volume" tagx="volume" title="volume">163</span> <span class="fpage" tagx="fpage" title="fpage">529</span> <span class="lpage" tagx="lpage" title="lpage">534</span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/3549295">3549295</a> </span> </span> </li> <li tag="ref"> <a name="pbio.1001040-Martel1" /> <span class="label" tagx="label" title="label">23</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Martel</span> <span class="given-names" tagx="given-names" title="given-names">R</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Cloney</span> <span class="given-names" tagx="given-names" title="given-names">L. P</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Pelcher</span> <span class="given-names" tagx="given-names" title="given-names">L. E</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Hemmingsen</span> <span class="given-names" tagx="given-names" title="given-names">S. M</span> </span> </span> <span class="year" tagx="year" title="year">1990</span> <span class="mixed-article-title" title="mixed-article-title">Unique composition of plastid chaperonin-60: α and β polypeptide-encoding genes are highly divergent. </span> <span class="source" tagx="source" title="source">Gene</span> <span class="volume" tagx="volume" title="volume">94</span> <span class="fpage" tagx="fpage" title="fpage">181</span> <span class="lpage" tagx="lpage" title="lpage">187</span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/1979547">1979547</a> </span> </span> </li> <li tag="ref"> <a name="pbio.1001040-Dickson1" /> <span class="label" tagx="label" title="label">24</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Dickson</span> <span class="given-names" tagx="given-names" title="given-names">R</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Weiss</span> <span class="given-names" tagx="given-names" title="given-names">C</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Howard</span> <span class="given-names" tagx="given-names" title="given-names">R. J</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Alldrick</span> <span class="given-names" tagx="given-names" title="given-names">S. P</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Ellis</span> <span class="given-names" tagx="given-names" title="given-names">R. J</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span> </span> <span class="year" tagx="year" title="year">2000</span> <span class="mixed-article-title" title="mixed-article-title">Reconstitution of higher plant chloroplast chaperonin 60 tetradecamers active in protein folding. </span> <span class="source" tagx="source" title="source">J Biol Chem</span> <span class="volume" tagx="volume" title="volume">275</span> <span class="fpage" tagx="fpage" title="fpage">11829</span> <span class="lpage" tagx="lpage" title="lpage">11835</span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/10766808">10766808</a> </span> </span> </li> <li tag="ref"> <a name="pbio.1001040-Nishio1" /> <span class="label" tagx="label" title="label">25</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Nishio</span> <span class="given-names" tagx="given-names" title="given-names">K</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Hirohashi</span> <span class="given-names" tagx="given-names" title="given-names">T</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Nakai</span> <span class="given-names" tagx="given-names" title="given-names">M</span> </span> </span> <span class="year" tagx="year" title="year">1999</span> <span class="mixed-article-title" title="mixed-article-title">Chloroplast chaperonins: evidence for heterogeneous assembly of α and β Cpn60 polypeptides into a chaperonin oligomer. </span> <span class="source" tagx="source" title="source">Biochem Biophys Res Commun</span> <span class="volume" tagx="volume" title="volume">266</span> <span class="fpage" tagx="fpage" title="fpage">584</span> <span class="lpage" tagx="lpage" title="lpage">587</span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/10600546">10600546</a> </span> </span> </li> <li tag="ref"> <a name="pbio.1001040-Weiss1" /> <span class="label" tagx="label" title="label">26</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Weiss</span> <span class="given-names" tagx="given-names" title="given-names">C</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Bonshtien</span> <span class="given-names" tagx="given-names" title="given-names">A</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Farchi-Pisanty</span> <span class="given-names" tagx="given-names" title="given-names">O</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Vitlin</span> <span class="given-names" tagx="given-names" title="given-names">A</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Azem</span> <span class="given-names" tagx="given-names" title="given-names">A</span> </span> </span> <span class="year" tagx="year" title="year">2009</span> <span class="mixed-article-title" title="mixed-article-title">Cpn20: Siamese twins of the chaperonin world.</span> <span class="source" tagx="source" title="source">Plant Mol Biol</span> <span class="volume" tagx="volume" title="volume">69</span> <span class="fpage" tagx="fpage" title="fpage">227</span> <span class="lpage" tagx="lpage" title="lpage">238</span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/19031045">19031045</a> </span> </span> </li> <li tag="ref"> <a name="pbio.1001040-Peltier1" /> <span class="label" tagx="label" title="label">27</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Peltier</span> <span class="given-names" tagx="given-names" title="given-names">J. B</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Emanuelsson</span> <span class="given-names" tagx="given-names" title="given-names">O</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Kalume</span> <span class="given-names" tagx="given-names" title="given-names">D. E</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Ytterberg</span> <span class="given-names" tagx="given-names" title="given-names">J</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Friso</span> <span class="given-names" tagx="given-names" title="given-names">G</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span> </span> <span class="year" tagx="year" title="year">2006</span> <span class="mixed-article-title" title="mixed-article-title">The oligomeric stromal proteome of <i>Arabidopsis thaliana </i> chloroplasts. </span> <span class="source" tagx="source" title="source">Mol Cell Proteomics</span> <span class="volume" tagx="volume" title="volume">5</span> <span class="fpage" tagx="fpage" title="fpage">114</span> <span class="lpage" tagx="lpage" title="lpage">133</span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/16207701">16207701</a> </span> </span> </li> <li tag="ref"> <a name="pbio.1001040-Apuya1" /> <span class="label" tagx="label" title="label">28</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Apuya</span> <span class="given-names" tagx="given-names" title="given-names">N. R</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Yadegari</span> <span class="given-names" tagx="given-names" title="given-names">R</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Fischer</span> <span class="given-names" tagx="given-names" title="given-names">R. L</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Harada</span> <span class="given-names" tagx="given-names" title="given-names">J. J</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Zimmerman</span> <span class="given-names" tagx="given-names" title="given-names">J. L</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span> </span> <span class="year" tagx="year" title="year">2001</span> <span class="mixed-article-title" title="mixed-article-title">The Arabidopsis embryo mutant <i>schlepperless</i> has a defect in the <i>Chaperonin-60α</i> gene. </span> <span class="source" tagx="source" title="source">Plant Physiol</span> <span class="volume" tagx="volume" title="volume">126</span> <span class="fpage" tagx="fpage" title="fpage">717</span> <span class="lpage" tagx="lpage" title="lpage">730</span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/11402200">11402200</a> </span> </span> </li> <li tag="ref"> <a name="pbio.1001040-Suzuki1" /> <span class="label" tagx="label" title="label">29</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Suzuki</span> <span class="given-names" tagx="given-names" title="given-names">K</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Nakanishi</span> <span class="given-names" tagx="given-names" title="given-names">H</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Bower</span> <span class="given-names" tagx="given-names" title="given-names">J</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Yoder</span> <span class="given-names" tagx="given-names" title="given-names">D. W</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Osteryoung</span> <span class="given-names" tagx="given-names" title="given-names">K. W</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span> </span> <span class="year" tagx="year" title="year">2009</span> <span class="mixed-article-title" title="mixed-article-title">Plastid chaperonin proteins Cpn60α and Cpn60 β are required for plastid division in <i>Arabidopsis thaliana. </i> </span> <span class="source" tagx="source" title="source">BMC Plant Biol</span> <span class="volume" tagx="volume" title="volume">9</span> <span class="fpage" tagx="fpage" title="fpage">38</span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/19344532">19344532</a> </span> </span> </li> <li tag="ref"> <a name="pbio.1001040-Llorca1" /> <span class="label" tagx="label" title="label">30</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Llorca</span> <span class="given-names" tagx="given-names" title="given-names">O</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">McCormack</span> <span class="given-names" tagx="given-names" title="given-names">E. A</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Hynes</span> <span class="given-names" tagx="given-names" title="given-names">G</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Grantham</span> <span class="given-names" tagx="given-names" title="given-names">J</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Cordell</span> <span class="given-names" tagx="given-names" title="given-names">J</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span> </span> <span class="year" tagx="year" title="year">1999</span> <span class="mixed-article-title" title="mixed-article-title">Eukaryotic type II chaperonin CCT interacts with actin through specific subunits. </span> <span class="source" tagx="source" title="source">Nature</span> <span class="volume" tagx="volume" title="volume">402</span> <span class="fpage" tagx="fpage" title="fpage">693</span> <span class="lpage" tagx="lpage" title="lpage">696</span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/10604479">10604479</a> </span> </span> </li> <li tag="ref"> <a name="pbio.1001040-Amit1" /> <span class="label" tagx="label" title="label">31</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Amit</span> <span class="given-names" tagx="given-names" title="given-names">M</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Weisberg</span> <span class="given-names" tagx="given-names" title="given-names">S. J</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Nadler-Holly</span> <span class="given-names" tagx="given-names" title="given-names">M</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">McCormack</span> <span class="given-names" tagx="given-names" title="given-names">E. A</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Feldmesser</span> <span class="given-names" tagx="given-names" title="given-names">E</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span> </span> <span class="year" tagx="year" title="year">2010</span> <span class="mixed-article-title" title="mixed-article-title">Equivalent mutations in the eight subunits of the chaperonin CCT produce dramatically different cellular and gene expression phenotypes. </span> <span class="source" tagx="source" title="source">J Mol Biol</span> <span class="volume" tagx="volume" title="volume">401</span> <span class="fpage" tagx="fpage" title="fpage">532</span> <span class="lpage" tagx="lpage" title="lpage">543</span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/20600117">20600117</a> </span> </span> </li> <li tag="ref"> <a name="pbio.1001040-Shikanai1" /> <span class="label" tagx="label" title="label">32</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Shikanai</span> <span class="given-names" tagx="given-names" title="given-names">T</span> </span> </span> <span class="year" tagx="year" title="year">2007</span> <span class="mixed-article-title" title="mixed-article-title">Cyclic electron transport around photosystem I: genetic approaches. </span> <span class="source" tagx="source" title="source">Annu Rev Plant Biol</span> <span class="volume" tagx="volume" title="volume">58</span> <span class="fpage" tagx="fpage" title="fpage">199</span> <span class="lpage" tagx="lpage" title="lpage">217</span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/17201689">17201689</a> </span> </span> </li> <li tag="ref"> <a name="pbio.1001040-Shikanai2" /> <span class="label" tagx="label" title="label">33</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Shikanai</span> <span class="given-names" tagx="given-names" title="given-names">T</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Endo</span> <span class="given-names" tagx="given-names" title="given-names">T</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Hasimoto</span> <span class="given-names" tagx="given-names" title="given-names">T</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Yamada</span> <span class="given-names" tagx="given-names" title="given-names">Y</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Asada</span> <span class="given-names" tagx="given-names" title="given-names">K</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span> </span> <span class="year" tagx="year" title="year">1998</span> <span class="mixed-article-title" title="mixed-article-title">Directed disruption of the tobacco <i>ndhB</i> gene impairs cyclic electron flow around photosystem I. </span> <span class="source" tagx="source" title="source">Proc Natl Acad Sci U S A</span> <span class="volume" tagx="volume" title="volume">95</span> <span class="fpage" tagx="fpage" title="fpage">9705</span> <span class="lpage" tagx="lpage" title="lpage">9709</span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/9689145">9689145</a> </span> </span> </li> <li tag="ref"> <a name="pbio.1001040-Suorsa1" /> <span class="label" tagx="label" title="label">34</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Suorsa</span> <span class="given-names" tagx="given-names" title="given-names">M</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Sirpiö</span> <span class="given-names" tagx="given-names" title="given-names">S</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Aro</span> <span class="given-names" tagx="given-names" title="given-names">E-M</span> </span> </span> <span class="year" tagx="year" title="year">2009</span> <span class="mixed-article-title" title="mixed-article-title">Towards characterization of the chloroplast NAD(P)H dehydrogenase complex. </span> <span class="source" tagx="source" title="source">Mol Plant</span> <span class="volume" tagx="volume" title="volume">2</span> <span class="fpage" tagx="fpage" title="fpage">1127</span> <span class="lpage" tagx="lpage" title="lpage">1140</span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/19995722">19995722</a> </span> </span> </li> <li tag="ref"> <a name="pbio.1001040-Peng1" /> <span class="label" tagx="label" title="label">35</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Peng</span> <span class="given-names" tagx="given-names" title="given-names">L</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Yamamoto</span> <span class="given-names" tagx="given-names" title="given-names">H</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Shikanai</span> <span class="given-names" tagx="given-names" title="given-names">T</span> </span> </span> <span class="year" tagx="year" title="year">2010</span> <span class="mixed-article-title" title="mixed-article-title">Structure and biogenesis of the chloroplast NAD(P)H dehydrogenase complex. </span> <span class="source" tagx="source" title="source">Biochim Biophys Acta</span> <span class="comment">doi: <a href="http://dx.doi.org/10.1016/j.bbabio.2010.10.015">10.1016/j.bbabio.2010.10.015</a> </span> </span> </li> <li tag="ref"> <a name="pbio.1001040-Kuromori1" /> <span class="label" tagx="label" title="label">36</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Kuromori</span> <span class="given-names" tagx="given-names" title="given-names">T</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Hirayama</span> <span class="given-names" tagx="given-names" title="given-names">T</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Kiyosue</span> <span class="given-names" tagx="given-names" title="given-names">Y</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Takabe</span> <span class="given-names" tagx="given-names" title="given-names">H</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Mizukado</span> <span class="given-names" tagx="given-names" title="given-names">S</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span> </span> <span class="year" tagx="year" title="year">2004</span> <span class="mixed-article-title" title="mixed-article-title">A collection of 11,800 single-copy <i>Ds</i> transposon insertion lines in <i>Arabidopsis</i>. </span> <span class="source" tagx="source" title="source">Plant J</span> <span class="volume" tagx="volume" title="volume">37</span> <span class="fpage" tagx="fpage" title="fpage">897</span> <span class="lpage" tagx="lpage" title="lpage">905</span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/14996221">14996221</a> </span> </span> </li> <li tag="ref"> <a name="pbio.1001040-Okuda1" /> <span class="label" tagx="label" title="label">37</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Okuda</span> <span class="given-names" tagx="given-names" title="given-names">K</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Myouga</span> <span class="given-names" tagx="given-names" title="given-names">F</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Motohashi</span> <span class="given-names" tagx="given-names" title="given-names">R</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Shinozaki</span> <span class="given-names" tagx="given-names" title="given-names">K</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Shikanai</span> <span class="given-names" tagx="given-names" title="given-names">T</span> </span> </span> <span class="year" tagx="year" title="year">2007</span> <span class="mixed-article-title" title="mixed-article-title">Conserved domain structure of pentatricopeptide repeat proteins involved in chloroplast RNA editing. </span> <span class="source" tagx="source" title="source">Proc Natl Acad Sci U S A</span> <span class="volume" tagx="volume" title="volume">104</span> <span class="fpage" tagx="fpage" title="fpage">8178</span> <span class="lpage" tagx="lpage" title="lpage">8183</span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/17483454">17483454</a> </span> </span> </li> <li tag="ref"> <a name="pbio.1001040-Peng2" /> <span class="label" tagx="label" title="label">38</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Peng</span> <span class="given-names" tagx="given-names" title="given-names">L</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Shimizu</span> <span class="given-names" tagx="given-names" title="given-names">H</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Shikanai</span> <span class="given-names" tagx="given-names" title="given-names">T</span> </span> </span> <span class="year" tagx="year" title="year">2008</span> <span class="mixed-article-title" title="mixed-article-title">The chloroplast NAD(P)H dehydrogenase complex interacts with photosystem I in <i>Arabidopsis.</i> </span> <span class="source" tagx="source" title="source">J Biol Chem</span> <span class="volume" tagx="volume" title="volume">83</span> <span class="fpage" tagx="fpage" title="fpage">34873</span> <span class="lpage" tagx="lpage" title="lpage">34879</span> </span> </li> <li tag="ref"> <a name="pbio.1001040-Peng3" /> <span class="label" tagx="label" title="label">39</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Peng</span> <span class="given-names" tagx="given-names" title="given-names">L</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Fukao</span> <span class="given-names" tagx="given-names" title="given-names">Y</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Fujiwara</span> <span class="given-names" tagx="given-names" title="given-names">M</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Takami</span> <span class="given-names" tagx="given-names" title="given-names">T</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Shikanai</span> <span class="given-names" tagx="given-names" title="given-names">T</span> </span> </span> <span class="year" tagx="year" title="year">2009</span> <span class="mixed-article-title" title="mixed-article-title">Efficient operation of NAD(P)H dehydrogenase requires supercomplex formation with photosystem I via minor LHCI in Arabidopsis. </span> <span class="source" tagx="source" title="source">Plant Cell</span> <span class="volume" tagx="volume" title="volume">21</span> <span class="fpage" tagx="fpage" title="fpage">3623</span> <span class="lpage" tagx="lpage" title="lpage">3640</span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/19903870">19903870</a> </span> </span> </li> <li tag="ref"> <a name="pbio.1001040-Ishihama1" /> <span class="label" tagx="label" title="label">40</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Ishihama</span> <span class="given-names" tagx="given-names" title="given-names">Y</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Oda</span> <span class="given-names" tagx="given-names" title="given-names">Y</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Tabata</span> <span class="given-names" tagx="given-names" title="given-names">T</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Sato</span> <span class="given-names" tagx="given-names" title="given-names">T</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Nagasu</span> <span class="given-names" tagx="given-names" title="given-names">T</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span> </span> <span class="year" tagx="year" title="year">2005</span> <span class="mixed-article-title" title="mixed-article-title">Exponentially modified protein abundance index (emPAI) for estimation of absolute protein amount in proteomics by the number of sequenced peptides per protein. </span> <span class="source" tagx="source" title="source">Mol Cell Proteomics</span> <span class="volume" tagx="volume" title="volume">4</span> <span class="fpage" tagx="fpage" title="fpage">1265</span> <span class="lpage" tagx="lpage" title="lpage">1272</span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/15958392">15958392</a> </span> </span> </li> <li tag="ref"> <a name="pbio.1001040-Peng4" /> <span class="label" tagx="label" title="label">41</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Peng</span> <span class="given-names" tagx="given-names" title="given-names">L</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Cai</span> <span class="given-names" tagx="given-names" title="given-names">W</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Shikanai</span> <span class="given-names" tagx="given-names" title="given-names">T</span> </span> </span> <span class="year" tagx="year" title="year">2010</span> <span class="mixed-article-title" title="mixed-article-title">Chloroplast stromal proteins, CRR6 and CRR7, are required for assembly of the NAD(P)H dehydrogenase subcomplex A in Arabidopsis. </span> <span class="source" tagx="source" title="source">Plant J</span> <span class="volume" tagx="volume" title="volume">63</span> <span class="fpage" tagx="fpage" title="fpage">203</span> <span class="lpage" tagx="lpage" title="lpage">211</span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/20444231">20444231</a> </span> </span> </li> <li tag="ref"> <a name="pbio.1001040-Obayashi1" /> <span class="label" tagx="label" title="label">42</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Obayashi</span> <span class="given-names" tagx="given-names" title="given-names">T</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Hayashi</span> <span class="given-names" tagx="given-names" title="given-names">S</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Saeki</span> <span class="given-names" tagx="given-names" title="given-names">M</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Ohta</span> <span class="given-names" tagx="given-names" title="given-names">H</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Kinoshita</span> <span class="given-names" tagx="given-names" title="given-names">K</span> </span> </span> <span class="year" tagx="year" title="year">2009</span> <span class="mixed-article-title" title="mixed-article-title">ATTED-II provides coexpressed gene networks for Arabidopsis. </span> <span class="source" tagx="source" title="source">Nucleic Acids Res</span> <span class="volume" tagx="volume" title="volume">37</span> <span class="fpage" tagx="fpage" title="fpage">D987</span> <span class="lpage" tagx="lpage" title="lpage">D991</span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/18953027">18953027</a> </span> </span> </li> <li tag="ref"> <a name="pbio.1001040-Tang1" /> <span class="label" tagx="label" title="label">43</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Tang</span> <span class="given-names" tagx="given-names" title="given-names">Y. C</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Chang</span> <span class="given-names" tagx="given-names" title="given-names">H. C</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Roeben</span> <span class="given-names" tagx="given-names" title="given-names">A</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Wischnewski</span> <span class="given-names" tagx="given-names" title="given-names">D</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Wischnewski</span> <span class="given-names" tagx="given-names" title="given-names">N</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span> </span> <span class="year" tagx="year" title="year">2006</span> <span class="mixed-article-title" title="mixed-article-title">Structural features of the GroEL-GroES nano-cage required for rapid folding of encapsulated protein. </span> <span class="source" tagx="source" title="source">Cell</span> <span class="volume" tagx="volume" title="volume">125</span> <span class="fpage" tagx="fpage" title="fpage">903</span> <span class="lpage" tagx="lpage" title="lpage">914</span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/16751100">16751100</a> </span> </span> </li> <li tag="ref"> <a name="pbio.1001040-Brocchieri1" /> <span class="label" tagx="label" title="label">44</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Brocchieri</span> <span class="given-names" tagx="given-names" title="given-names">L</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Karlin</span> <span class="given-names" tagx="given-names" title="given-names">S</span> </span> </span> <span class="year" tagx="year" title="year">2000</span> <span class="mixed-article-title" title="mixed-article-title">Conservation among HSP60 sequences in relation to structure, function, and evolution. </span> <span class="source" tagx="source" title="source">Prot Sci</span> <span class="volume" tagx="volume" title="volume">9</span> <span class="fpage" tagx="fpage" title="fpage">476</span> <span class="lpage" tagx="lpage" title="lpage">486</span> </span> </li> <li tag="ref"> <a name="pbio.1001040-Wastl1" /> <span class="label" tagx="label" title="label">45</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Wastl</span> <span class="given-names" tagx="given-names" title="given-names">J</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Fraunholz</span> <span class="given-names" tagx="given-names" title="given-names">M</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Zauner</span> <span class="given-names" tagx="given-names" title="given-names">S</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Douglas</span> <span class="given-names" tagx="given-names" title="given-names">S</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Maier</span> <span class="given-names" tagx="given-names" title="given-names">U. G</span> </span> </span> <span class="year" tagx="year" title="year">1999</span> <span class="mixed-article-title" title="mixed-article-title">Ancient gene duplication and differential gene flow in plastid lineages: the GroEL/Cpn60 example. </span> <span class="source" tagx="source" title="source">J Mol Evol</span> <span class="volume" tagx="volume" title="volume">48</span> <span class="fpage" tagx="fpage" title="fpage">112</span> <span class="lpage" tagx="lpage" title="lpage">117</span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/9873083">9873083</a> </span> </span> </li> <li tag="ref"> <a name="pbio.1001040-Arnold1" /> <span class="label" tagx="label" title="label">46</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Arnold</span> <span class="given-names" tagx="given-names" title="given-names">K</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Bordoli</span> <span class="given-names" tagx="given-names" title="given-names">L</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Kopp</span> <span class="given-names" tagx="given-names" title="given-names">J</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Schwede</span> <span class="given-names" tagx="given-names" title="given-names">T</span> </span> </span> <span class="year" tagx="year" title="year">2006</span> <span class="mixed-article-title" title="mixed-article-title">The SWISS-MODEL workspace: a Web-based environment for protein structure homology modelling. </span> <span class="source" tagx="source" title="source">Bioinformatics</span> <span class="volume" tagx="volume" title="volume">22</span> <span class="fpage" tagx="fpage" title="fpage">195</span> <span class="lpage" tagx="lpage" title="lpage">201</span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/16301204">16301204</a> </span> </span> </li> <li tag="ref"> <a name="pbio.1001040-Tang2" /> <span class="label" tagx="label" title="label">47</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Tang</span> <span class="given-names" tagx="given-names" title="given-names">Y. C</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Chang</span> <span class="given-names" tagx="given-names" title="given-names">H. C</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Chakraborty</span> <span class="given-names" tagx="given-names" title="given-names">K</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Hartl</span> <span class="given-names" tagx="given-names" title="given-names">F. U</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Hayer-Hartl</span> <span class="given-names" tagx="given-names" title="given-names">M</span> </span> </span> <span class="year" tagx="year" title="year">2008</span> <span class="mixed-article-title" title="mixed-article-title">Essential role of the chaperonin folding compartment <i>in vivo </i>. </span> <span class="source" tagx="source" title="source">EMBO J</span> <span class="volume" tagx="volume" title="volume">27</span> <span class="fpage" tagx="fpage" title="fpage">1458</span> <span class="lpage" tagx="lpage" title="lpage">1468</span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/18418386">18418386</a> </span> </span> </li> <li tag="ref"> <a name="pbio.1001040-Farr1" /> <span class="label" tagx="label" title="label">48</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Farr</span> <span class="given-names" tagx="given-names" title="given-names">G. W</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Fenton</span> <span class="given-names" tagx="given-names" title="given-names">W. A</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Horwich</span> <span class="given-names" tagx="given-names" title="given-names">A. L</span> </span> </span> <span class="year" tagx="year" title="year">2007</span> <span class="mixed-article-title" title="mixed-article-title">Perturbed ATPase activity and not "close confinement" of substrate in the cis cavity affects rates of folding by tail-multiplied GroEL. </span> <span class="source" tagx="source" title="source">Proc Natl Acad Sci U S A</span> <span class="volume" tagx="volume" title="volume">104</span> <span class="fpage" tagx="fpage" title="fpage">5342</span> <span class="lpage" tagx="lpage" title="lpage">5347</span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/17372195">17372195</a> </span> </span> </li> <li tag="ref"> <a name="pbio.1001040-Suzuki2" /> <span class="label" tagx="label" title="label">49</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Suzuki</span> <span class="given-names" tagx="given-names" title="given-names">M</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Ueno</span> <span class="given-names" tagx="given-names" title="given-names">T</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Iizuka</span> <span class="given-names" tagx="given-names" title="given-names">R</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Miura</span> <span class="given-names" tagx="given-names" title="given-names">T</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Zako</span> <span class="given-names" tagx="given-names" title="given-names">T</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span> </span> <span class="year" tagx="year" title="year">2008</span> <span class="mixed-article-title" title="mixed-article-title">Effect of the C-terminal truncation on the functional cycle of chaperonin GroEL: implication that the C-terminal region facilitates the transition from the folding-arrested to the folding-competent state. </span> <span class="source" tagx="source" title="source">J Biol Chem</span> <span class="volume" tagx="volume" title="volume">283</span> <span class="fpage" tagx="fpage" title="fpage">23931</span> <span class="lpage" tagx="lpage" title="lpage">23939</span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/18583344">18583344</a> </span> </span> </li> <li tag="ref"> <a name="pbio.1001040-Machida1" /> <span class="label" tagx="label" title="label">50</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Machida</span> <span class="given-names" tagx="given-names" title="given-names">K</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Kono-Okada</span> <span class="given-names" tagx="given-names" title="given-names">A</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Hongo</span> <span class="given-names" tagx="given-names" title="given-names">K</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Mizobata</span> <span class="given-names" tagx="given-names" title="given-names">T</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Kawata</span> <span class="given-names" tagx="given-names" title="given-names">Y</span> </span> </span> <span class="year" tagx="year" title="year">2008</span> <span class="mixed-article-title" title="mixed-article-title">Hydrophilic residues <sup>526</sup>KNDAAD <sup>531</sup> in the flexible C-terminal region of the chaperonin GroEL are critical for substrate protein folding within the central cavity. </span> <span class="source" tagx="source" title="source">J Biol Chem</span> <span class="volume" tagx="volume" title="volume">283</span> <span class="fpage" tagx="fpage" title="fpage">6886</span> <span class="lpage" tagx="lpage" title="lpage">6896</span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/18184659">18184659</a> </span> </span> </li> <li tag="ref"> <a name="pbio.1001040-McLennan1" /> <span class="label" tagx="label" title="label">51</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">McLennan</span> <span class="given-names" tagx="given-names" title="given-names">N. F</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Girshovich</span> <span class="given-names" tagx="given-names" title="given-names">A. S</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Lissin</span> <span class="given-names" tagx="given-names" title="given-names">N. M</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Charters</span> <span class="given-names" tagx="given-names" title="given-names">Y</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Masters</span> <span class="given-names" tagx="given-names" title="given-names">M</span> </span> </span> <span class="year" tagx="year" title="year">1993</span> <span class="mixed-article-title" title="mixed-article-title">The strongly conserved carboxyl-terminus glycine-methionine motif of <i>Escherichia coli</i> GroEL chaperonin is dispensable. </span> <span class="source" tagx="source" title="source">Mol Microbiol</span> <span class="volume" tagx="volume" title="volume">7</span> <span class="fpage" tagx="fpage" title="fpage">49</span> <span class="lpage" tagx="lpage" title="lpage">58</span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/8094879">8094879</a> </span> </span> </li> <li tag="ref"> <a name="pbio.1001040-McLennan2" /> <span class="label" tagx="label" title="label">52</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">McLennan</span> <span class="given-names" tagx="given-names" title="given-names">N. F</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">McAteer</span> <span class="given-names" tagx="given-names" title="given-names">S</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Masters</span> <span class="given-names" tagx="given-names" title="given-names">M</span> </span> </span> <span class="year" tagx="year" title="year">1994</span> <span class="mixed-article-title" title="mixed-article-title">The tail of a chaperonin: the C-terminal region of Escherichia coli GroEL protein. </span> <span class="source" tagx="source" title="source">Mol Microbiol</span> <span class="volume" tagx="volume" title="volume">14</span> <span class="fpage" tagx="fpage" title="fpage">309</span> <span class="lpage" tagx="lpage" title="lpage">321</span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/7830575">7830575</a> </span> </span> </li> <li tag="ref"> <a name="pbio.1001040-Wang1" /> <span class="label" tagx="label" title="label">53</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Wang</span> <span class="given-names" tagx="given-names" title="given-names">J. D</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Herman</span> <span class="given-names" tagx="given-names" title="given-names">C</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Tipton</span> <span class="given-names" tagx="given-names" title="given-names">K. A</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Gross</span> <span class="given-names" tagx="given-names" title="given-names">C. A</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Weissman</span> <span class="given-names" tagx="given-names" title="given-names">J. S</span> </span> </span> <span class="year" tagx="year" title="year">2002</span> <span class="mixed-article-title" title="mixed-article-title">Directed evolution of substrate-optimized GroEL/S chaperonins. </span> <span class="source" tagx="source" title="source">Cell</span> <span class="volume" tagx="volume" title="volume">111</span> <span class="fpage" tagx="fpage" title="fpage">1027</span> <span class="lpage" tagx="lpage" title="lpage">1039</span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/12507429">12507429</a> </span> </span> </li> <li tag="ref"> <a name="pbio.1001040-Sielaff1" /> <span class="label" tagx="label" title="label">54</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Sielaff</span> <span class="given-names" tagx="given-names" title="given-names">B</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Lee</span> <span class="given-names" tagx="given-names" title="given-names">K. S</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Tsai</span> <span class="given-names" tagx="given-names" title="given-names">F. T</span> </span> </span> <span class="year" tagx="year" title="year">2011</span> <span class="mixed-article-title" title="mixed-article-title">Structural and functional conservation of mycobacterium tuberculosis GroEL paralogs suggests that GroEL1 is a chaperonin. </span> <span class="source" tagx="source" title="source">J Mol Biol</span> <span class="volume" tagx="volume" title="volume">405</span> <span class="fpage" tagx="fpage" title="fpage">831</span> <span class="lpage" tagx="lpage" title="lpage">839</span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/21094166">21094166</a> </span> </span> </li> <li tag="ref"> <a name="pbio.1001040-Hashimoto1" /> <span class="label" tagx="label" title="label">55</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Hashimoto</span> <span class="given-names" tagx="given-names" title="given-names">M</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Endo</span> <span class="given-names" tagx="given-names" title="given-names">T</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Peltier</span> <span class="given-names" tagx="given-names" title="given-names">G</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Tasaka</span> <span class="given-names" tagx="given-names" title="given-names">M</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Shikanai</span> <span class="given-names" tagx="given-names" title="given-names">T</span> </span> </span> <span class="year" tagx="year" title="year">2003</span> <span class="mixed-article-title" title="mixed-article-title">A nucleus-encoded factor, CRR2, is essential for the expression of chloroplast <i>ndhB</i> in Arabidopsis. </span> <span class="source" tagx="source" title="source">Plant J</span> <span class="volume" tagx="volume" title="volume">36</span> <span class="fpage" tagx="fpage" title="fpage">541</span> <span class="lpage" tagx="lpage" title="lpage">549</span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/14617084">14617084</a> </span> </span> </li> <li tag="ref"> <a name="pbio.1001040-Thompson1" /> <span class="label" tagx="label" title="label">56</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Thompson</span> <span class="given-names" tagx="given-names" title="given-names">J. D</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Gibson</span> <span class="given-names" tagx="given-names" title="given-names">T. J</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Plewniak</span> <span class="given-names" tagx="given-names" title="given-names">F</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Jeanmougin</span> <span class="given-names" tagx="given-names" title="given-names">F</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Higgins</span> <span class="given-names" tagx="given-names" title="given-names">D. G</span> </span> </span> <span class="year" tagx="year" title="year">1997</span> <span class="mixed-article-title" title="mixed-article-title">The CLUSTAL_X windows interface: flexible strategies for multiple sequence alignment aided by quality analysis tools. </span> <span class="source" tagx="source" title="source">Nucleic Acids Res</span> <span class="volume" tagx="volume" title="volume">25</span> <span class="fpage" tagx="fpage" title="fpage">4876</span> <span class="lpage" tagx="lpage" title="lpage">4882</span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/9396791">9396791</a> </span> </span> </li> <li tag="ref"> <a name="pbio.1001040-Guindon1" /> <span class="label" tagx="label" title="label">57</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Guindon</span> <span class="given-names" tagx="given-names" title="given-names">S</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Gascuel</span> <span class="given-names" tagx="given-names" title="given-names">O</span> </span> </span> <span class="year" tagx="year" title="year">2003</span> <span class="mixed-article-title" title="mixed-article-title">A simple, fast, and accurate algorithm to estimate large phylogenies by maximum likelihood. </span> <span class="source" tagx="source" title="source">Syst Biol</span> <span class="volume" tagx="volume" title="volume">52</span> <span class="fpage" tagx="fpage" title="fpage">696</span> <span class="lpage" tagx="lpage" title="lpage">704</span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/14530136">14530136</a> </span> </span> </li> <li tag="ref"> <a name="pbio.1001040-Posada1" /> <span class="label" tagx="label" title="label">58</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Posada</span> <span class="given-names" tagx="given-names" title="given-names">D</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Crandall</span> <span class="given-names" tagx="given-names" title="given-names">K. A</span> </span> </span> <span class="year" tagx="year" title="year">1998</span> <span class="mixed-article-title" title="mixed-article-title">MODELTEST: testing the model of DNA substitution.</span> <span class="source" tagx="source" title="source">Bioinformatics</span> <span class="volume" tagx="volume" title="volume">14</span> <span class="fpage" tagx="fpage" title="fpage">817</span> <span class="lpage" tagx="lpage" title="lpage">818</span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/9918953">9918953</a> </span> </span> </li> <li tag="ref"> <a name="pbio.1001040-Huelsenbeck1" /> <span class="label" tagx="label" title="label">59</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Huelsenbeck</span> <span class="given-names" tagx="given-names" title="given-names">J. P</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Ronquist</span> <span class="given-names" tagx="given-names" title="given-names">F</span> </span> </span> <span class="year" tagx="year" title="year">2001</span> <span class="mixed-article-title" title="mixed-article-title">MRBAYES: Bayesian inference of phylogenetic trees. </span> <span class="source" tagx="source" title="source">Bioinformatics</span> <span class="volume" tagx="volume" title="volume">17</span> <span class="fpage" tagx="fpage" title="fpage">754</span> <span class="lpage" tagx="lpage" title="lpage">755</span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/11524383">11524383</a> </span> </span> </li> <li tag="ref"> <a name="pbio.1001040-Arnold2" /> <span class="label" tagx="label" title="label">60</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Arnold</span> <span class="given-names" tagx="given-names" title="given-names">K</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Bordoli</span> <span class="given-names" tagx="given-names" title="given-names">L</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Kopp</span> <span class="given-names" tagx="given-names" title="given-names">J</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Schwede</span> <span class="given-names" tagx="given-names" title="given-names">T</span> </span> </span> <span class="year" tagx="year" title="year">2006</span> <span class="mixed-article-title" title="mixed-article-title">The SWISS-MODEL workspace: a web-based environment for protein structure homology modelling. </span> <span class="source" tagx="source" title="source">Bioinformatics</span> <span class="volume" tagx="volume" title="volume">22</span> <span class="fpage" tagx="fpage" title="fpage">195</span> <span class="lpage" tagx="lpage" title="lpage">201</span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/16301204">16301204</a> </span> </span> </li> <li tag="ref"> <a name="pbio.1001040-Fenton2" /> <span class="label" tagx="label" title="label">61</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Fenton</span> <span class="given-names" tagx="given-names" title="given-names">W. A</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Kashi</span> <span class="given-names" tagx="given-names" title="given-names">Y</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Furtak</span> <span class="given-names" tagx="given-names" title="given-names">K</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Horwich</span> <span class="given-names" tagx="given-names" title="given-names">A. L</span> </span> </span> <span class="year" tagx="year" title="year">1994</span> <span class="mixed-article-title" title="mixed-article-title">Residues in chaperonin GroEL required for polypeptide binding and release. </span> <span class="source" tagx="source" title="source">Nature</span> <span class="volume" tagx="volume" title="volume">371</span> <span class="fpage" tagx="fpage" title="fpage">614</span> <span class="lpage" tagx="lpage" title="lpage">619</span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/7935796">7935796</a> </span> </span> </li> <li tag="ref"> <a name="pbio.1001040-Buckle1" /> <span class="label" tagx="label" title="label">62</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Buckle</span> <span class="given-names" tagx="given-names" title="given-names">A. M</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Zahn</span> <span class="given-names" tagx="given-names" title="given-names">R</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Fersht</span> <span class="given-names" tagx="given-names" title="given-names">A. R</span> </span> </span> <span class="year" tagx="year" title="year">1997</span> <span class="mixed-article-title" title="mixed-article-title">A structural model for GroEL-polypeptide recognition. </span> <span class="source" tagx="source" title="source">Proc Natl Acad Sci U S A</span> <span class="volume" tagx="volume" title="volume">94</span> <span class="fpage" tagx="fpage" title="fpage">3571</span> <span class="lpage" tagx="lpage" title="lpage">3575</span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/9108017">9108017</a> </span> </span> </li> </ul> </div> <div class="glossary" title="glossary"> <div class="title" tagx="title" title="title">
Abbreviations</div> <div class="def-list" title="def-list"> <div class="def-item" title="def-item"> <div class="term" title="term">
AL</div> <div class="def" title="def"> <p>actinic light</p> </div> </div> <div class="def-item" title="def-item"> <div class="term" title="term">
BN-PAGE</div> <div class="def" title="def"> <p>blue native PAGE</p> </div> </div> <div class="def-item" title="def-item"> <div class="term" title="term">
CN-PAGE</div> <div class="def" title="def"> <p>clear native PAGE</p> </div> </div> <div class="def-item" title="def-item"> <div class="term" title="term"> <i>crr</i> </div> <div class="def" title="def"> <p> <i>chlororespiratory reduction</i> </p> </div> </div> <div class="def-item" title="def-item"> <div class="term" title="term">
Cyt</div> <div class="def" title="def"> <p>cytochrome</p> </div> </div> <div class="def-item" title="def-item"> <div class="term" title="term">
emPAI</div> <div class="def" title="def"> <p>exponentially modified Protein Abundance Index</p> </div> </div> <div class="def-item" title="def-item"> <div class="term" title="term">
HA</div> <div class="def" title="def"> <p>influenza hemagglutinin protein epitope</p> </div> </div> <div class="def-item" title="def-item"> <div class="term" title="term">
LC-MS/MS</div> <div class="def" title="def"> <p>liquid chromatography-tandem mass spectrometry</p> </div> </div> <div class="def-item" title="def-item"> <div class="term" title="term">
NDH</div> <div class="def" title="def"> <p>NADH dehydrogenase-like complex</p> </div> </div> <div class="def-item" title="def-item"> <div class="term" title="term">
PAM</div> <div class="def" title="def"> <p>pulse amplitude modulation fluorometry</p> </div> </div> <div class="def-item" title="def-item"> <div class="term" title="term">
PQ</div> <div class="def" title="def"> <p>plastoquinone</p> </div> </div> <div class="def-item" title="def-item"> <div class="term" title="term">
PSI</div> <div class="def" title="def"> <p>photosystem I</p> </div> </div> <div class="def-item" title="def-item"> <div class="term" title="term">
PSII</div> <div class="def" title="def"> <p>photosystem II</p> </div> </div> <div class="def-item" title="def-item"> <div class="term" title="term">
WT</div> <div class="def" title="def"> <p>wild type</p> </div> </div> </div> </div> </div>  </body></html>