<html xmlns="http://www.w3.org/1999/xhtml"><head> <style type="text/css">
         	     		 
         	    a                          {background : #ffffff; }
         	    article                    {border-style : dotted; border-width : 2px; }
         		 	
         		 	div                        {background : #ffffcc;}
         		 	div.abstract-title         {font-weight : bold ; font-size : 16pt;}
         		 	div.ack                    {border-style : solid ; border-color : red; margin
         : 2em; }
         		 	div.article-meta           {border-style : solid; border-width : 1 pt; margin
         1em;}
         		 	div.boxed-text             {margin : 5em; border-style : solid;}
         		 	div.contrib-group          {margin : 1em; }
         		 	div.fig                    {border-style : solid; border-width : 2px; margin :
         2em; }
         		 	div.funding                {font-weight : bold ; font-size : 16pt;}
         		 	div.given-names            {font-style : italic;}
         		 	div.intro                  {border-style : inset; margin : 5px;}
         		 	div.introduction           {border-style : inset; margin : 5px;}
         		 	div.journal-meta           {border-style : solid; border-width : 1 pt; margin
         1em;}
         		 	div.journal-title-group    {background : #ffeeee;}
         		 	div.article-title          {font-weight : bold ; font-size : 18pt;}
         		 	div.kwd                    {font-style : italic}
         		 	div.meta-name              {font-weight : bold ; font-size : 16pt;}
         		 	div.name                   {font-weight : bold;}
         		 	div.alt-title              {font-style : italic; font-size : 12px;}
         		 	
         		 	div.sec                    {border : 2px; margin 5px; padding 2px;}
         		 	div.title                  {font-family : courier; font-weight : bold;
         		 	                            font-size : 16pt; margin : 5px;}
         		 	
         		 	div.materials_methods      {border-style : double; margin : 5px; }
         		 	div.methods                {border-style : double; margin : 5px; }
         		 	
         		 	div.results                {border-style : solid; margin : 5px;}
         		 	div.background             {border-style : dotted; margin : 5px;}
         		 	
         		 	div.discussion             {border-style : groove; margin : 5px;}
         		 	div.conclusion             {border-style : ridge; margin : 5px;}
         		 	div.conclusions             {border-style : ridge; margin : 5px;}
         		 	div.supplementary-material {border-style : inset; margin : 5px;}
         		 	div.abbreviations          {border-style : double; border-color : red; margin
         : 5px;}
         		 	div.competinginterests     {border-style : double; border-color : blue; margin
         : 5px;}
         		 	div.acknowledgements       {border-style : double; border-color : green; margin
         : 5px;}
         		 	div.authors_contributions   {border-style : double; border-color : purple; margin
         : 5px;}
         		 	
         		 	div.publisher              {border-style : outset; margin : 5px;}
         		 	div.fn-type-conflict       {background : #f88; }
         		 	div.fn-type-con            {background : #ddf; }
         		 	div.fn-type-other          {background : #ddd; }
         		 	
         		 	div.unknown                {background : #ffd;
         									 	  border-style : solid;
         									 	  border-width : 1px;
         									 	  padding : 2 px;}
         		 	  
         	    table                      {background : #ffffdd;}
         		 	tr                         {background : #ddddff; padding : 1px;}
         		 	
         		 	span                       {background : #ffcccc;}
         		 	
         		 	span.citation-author       {font-family : helvetica; background : #ffeeee;}
         		 	span.collab                {background : #ddffff; }
         		 	span.comment               {font-family : courier; font-size : 6px; background
         : #ffaaff;}
         		 	span.contrib               {background : #ffffff;}
         		 	span.corresp               {background : #ddffdd; }
         		 	span.doi                   {background : #ffffff;}
         		 	span.email                 {font-family : courier; }
         		 	span.etal                  {font-style : italic;}
         		 	span.fpage                 {font-family : courier;}
         		 	span.given-names           {background : #ffffff;}
         		 	span.iso-abbrev            {background : #ffffff;}
         		 	span.issn-epub             {background : #ffffff;}
         		 	span.issn-ppub             {background : #ffffff;}
         		 	span.journal-title         {background : #ffffff;}
         		 	span.lpage                 {font-family : courier;}
         		 	span.mixed-article-title   {font-style : italic ;}
         		 	span.nlm-ta                {background : #ffffff;}
         		 	span.pmc                   {background : #ffffff;}
         		 	span.pmcid                 {background : #ffffff;}
         		 	span.pmid                  {background : #ffffff;}
         		 	span.publisher             {background : #ffffff;}
         		 	span.publisher-id          {background : #ffffff;}
         		 	span.publisher-name        {background : #ffffff;}
         		 	span.source                {background : #ffffff;}
         		 	span.subject               {background : #ffffff;}
         		 	span.surname               {background : #ffffff;}
         		 	span.volume                {font-family : courier; font-weight : bold;}
         		 	span.year                  {font-family : courier ; font-style : italic;}
         			
      </style> </head> <body> <div class="front" title="front"> <div class="journal-meta" tagx="journal-meta" title="journal-meta"> <span class="nlm-ta" title="nlm-ta">BMC Evol Biol</span> <div class="journal-title-group" tagx="journal-title-group" title="journal-title-group"> <span class="journal-title" tagx="journal-title" title="journal-title">BMC Evolutionary Biology</span> </div> <span class="issn-epub" tagx="issn" title="issn-epub">1471-2148</span> <div class="publisher" tagx="publisher" title="publisher"> <span class="publisher-name" tagx="publisher-name" title="publisher-name">BioMed Central</span> </div> </div> <div class="article-meta" tagx="article-meta" title="article-meta"> <span class="pmcid" title="pmcid"> pmcid: <a href="http://www.ncbi.nlm.nih.gov/pubmed/2978018">2978018</a> </span> <span class="publisher-id" title="publisher-id">1471-2148-10-198</span> <span class="pmid" title="pmid"> pmid: <a href="http://www.ncbi.nlm.nih.gov/pubmed/20579371">20579371</a> </span> <span class="doi" title="doi"> doi: <a href="https://dx.doi.org/10.1186/1471-2148-10-198">10.1186/1471-2148-10-198</a> </span> <div class="article-categories" title="article-categories">
: <span class="subject" title="subject">Research Article</span> </div> <div class="title-group"> <div class="article-title" title="article-title">
Molecular evolution of glutamine synthetase II: Phylogenetic evidence of a non-endosymbiotic gene transfer event early in plant evolution </div> </div> <div class="contrib-group" title="contrib-group"> <span class="contrib" title="contrib"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Ghoshroy</span> <span class="given-names" tagx="given-names" title="given-names">Sohini</span> </span> <a href="#I1">1</a> <span class="email" tagx="email" title="email">sghoshroy@clarku.edu</span> </span> <span class="contrib" title="contrib"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Binder</span> <span class="given-names" tagx="given-names" title="given-names">Manfred</span> </span> <a href="#I1">1</a> <span class="email" tagx="email" title="email">mbinder@clarku.edu</span> </span> <span class="contrib" title="contrib"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Tartar</span> <span class="given-names" tagx="given-names" title="given-names">Aurélien</span> </span> <a href="#I2">2</a> <span class="email" tagx="email" title="email">aurelien@NOVA.edu</span> </span> <span class="contrib" title="contrib"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Robertson</span> <span class="given-names" tagx="given-names" title="given-names">Deborah L</span> </span> <a href="#I1">1</a> <span class="email" tagx="email" title="email">debrobertson@clarku.edu</span> </span> </div> <span class="citation_author_institution" id="I1">[1], </span> <span class="citation_author_institution" id="I2">[2], </span> <span class="pub-date-collection" title="pub-date-collection">collection: <span>2010</span> </span> <span class="pub-date-epub" title="pub-date-epub">epub: <span>2010-6-6</span> </span> <span class="volume" tagx="volume" title="volume">10</span> <span class="fpage" tagx="fpage" title="fpage">198</span> <span class="lpage" tagx="lpage" title="lpage">198</span> <span class="history" title="history"> <span class="received" title="received">received: 2009-12-23</span> <span class="accepted" title="accepted">accepted: 2010-6-25</span> </span> <div class="permissions"> <span class="copyright" title="copyright">(C) <span class="copyright-holder" tagx="copyright-holder" title="copyright-holder">Ghoshroy et al; licensee BioMed Central Ltd.</span>, <span class="copyright-year" tagx="copyright-year" title="copyright-year">2010</span> </span> <span class="license" title="license"> <span class="license-p" title="license-p">This is an Open Access article distributed under the terms of the Creative Commons Attribution License ( <a href="http://creativecommons.org/licenses/by/2.0">http://creativecommons.org/licenses/by/2.0</a>), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited. </span> </span> </div> <a class="self-uri" href="http://www.biomedcentral.com/1471-2148/10/198" title="self-uri">http://www.biomedcentral.com/1471-2148/10/198</a> <div class="abstract" title="abstract"> <div class="abstract-title" title="abstract-title">
Abstract</div> <div class="background" title="sec"> <div class="title" tagx="title" title="title">
Background</div> <p>Glutamine synthetase (GS) is essential for ammonium assimilation and the biosynthesis of glutamine. The three GS gene families (GSI, GSII, and GSIII) are represented in both prokaryotic and eukaryotic organisms. In this study, we examined the evolutionary relationship of GSII from eubacterial and eukaryotic lineages and present robust phylogenetic evidence that GSII was transferred from γ-Proteobacteria (Eubacteria) to the Chloroplastida. </p> </div> <div class="results" title="sec"> <div class="title" tagx="title" title="title">
Results</div> <p>GSII sequences were isolated from four species of green algae (Trebouxiophyceae), and additional green algal (Chlorophyceae and Prasinophytae) and streptophyte (Charales, Desmidiales, Bryophyta, Marchantiophyta, Lycopodiophyta and Tracheophyta) sequences were obtained from public databases. In Bayesian and maximum likelihood analyses, eubacterial (GSII <sub>B</sub>) and eukaryotic (GSII <sub>E</sub>) GSII sequences formed distinct clades. Both GSII <sub>B </sub>and GSII <sub>E </sub>were found in chlorophytes and early-diverging streptophytes. The GSII <sub>B </sub>enzymes from these groups formed a well-supported sister clade with the γ-Proteobacteria, providing evidence that GSII <sub>B </sub>in the Chloroplastida arose by horizontal gene transfer (HGT). Bayesian relaxed molecular clock analyses suggest that GSII <sub>B </sub>and GSII <sub>E </sub>coexisted for an extended period of time but it is unclear whether the proposed HGT happened prior to or after the divergence of the primary endosymbiotic lineages (the Archaeplastida). However, GSII <sub>B </sub>genes have not been identified in glaucophytes or red algae, favoring the hypothesis that GSII <sub>B </sub>was gained after the divergence of the primary endosymbiotic lineages. Duplicate copies of the GSII <sub>B </sub>gene were present in <i>Chlamydomonas reinhardtii</i>, <i>Volvox </i> <i>carteri </i>f. <i>nagariensis</i>, and <i>Physcomitrella </i> <i>patens</i>. Both GSII <sub>B </sub>proteins in <i>C. reinhardtii </i>and <i>V. carteri </i>f. <i>nagariensis </i>had N-terminal transit sequences, indicating they are targeted to the chloroplast or mitochondrion. In contrast, GSII <sub>B </sub>proteins of <i>P. patens </i>lacked transit sequences, suggesting a cytosolic function. GSII <sub>B </sub>sequences were absent in vascular plants where the duplication of GSII <sub>E </sub>replaced the function of GSII <sub>B</sub>. </p> </div> <div class="conclusions" title="sec"> <div class="title" tagx="title" title="title">
Conclusions</div> <p>Phylogenetic evidence suggests GSII <sub>B </sub>in Chloroplastida evolved by HGT, possibly after the divergence of the primary endosymbiotic lineages. Thus while multiple GS isoenzymes are common among members of the Chloroplastida, the isoenzymes may have evolved via different evolutionary processes. The acquisition of essential enzymes by HGT may provide rapid changes in biochemical capacity and therefore be favored by natural selection. </p> </div> </div> </div> </div> <div class="body" title="body"> <div class="background" title="sec"> <div class="title" tagx="title" title="title">
Background</div> <p>Glutamine synthetase (GS: E.C. 6.3.1.2) catalyzes the ATP-dependent formation of Gln from Glu and NH <sub>4 </sub> <sup>+ </sup>and is considered one of the oldest functioning enzymes [ <a href="#B1">1</a>, <a href="#B2">2</a>]. The GS gene superfamily includes three distinct classes, GSI, GSII and GSIII, each differing in molecular size and number of subunits in the holoenzyme [ <a href="#B3">3</a>, <a href="#B4">4</a>]. The distribution of the three classes is variable within the three domains of life and instances of multiple GS isoenzymes from different families functioning in the same organism are not uncommon in both eubacteria and eukaryotes [ <a href="#B3">3</a>, <a href="#B5">5</a>- <a href="#B8">8</a>]. These observations suggest the gene families arose early and prior to the divergence of the prokaryotes and eukaryotes [ <a href="#B9">9</a>- <a href="#B11">11</a>]. </p> <p>The evolutionary history of the GS superfamily is complicated and gene transfer events are one among many forces that have shaped the history of these genes. Among prokaryotes, there is phylogenetic evidence of horizontal gene transfer (HGT) of GSI genes between members of the Archaea and Eubacteria [ <a href="#B9">9</a>]. Evidence for secondary endosymbiotic gene transfer of GSII from the red algal endosymbiont to the nucleus of the heterokont host has also been presented [ <a href="#B11">11</a>]. Members of the GSII gene family are found in both eubacterial and eukaryotic lineages. The identification of GSII genes in the plant symbiont <i>Bradyrhizobium japonicum </i>lead Carlson and Chelm [ <a href="#B12">12</a>] to hypothesize that the gene evolved via HGT from vascular plants to bacteria. However, this hypothesis was not supported by subsequent phylogenetic analyses [ <a href="#B11">11</a>, <a href="#B13">13</a>], which established distinct eukaryotic (GSII <sub>E</sub>) and eubacterial (GSII <sub>B</sub>) clades. </p> <p>The supergroup Archaeplastida [ <a href="#B14">14</a>], consisting of Glaucophyta, Rhodophyceae and Chloroplastida, harbors members of GSII gene family that are well characterized in vascular plants but not in other lineages within the supergroup. In general, vascular plants express multiple GS isoenzymes that are localized to either cytosol or chloroplast. The isoenzymes are nuclear encoded, and in most angiosperms a single nuclear gene encodes the chloroplast isoenzyme, while a small nuclear gene family encodes multiple cytosolic isoenzymes that are expressed in tissue-specific and developmentally-regulated patterns [ <a href="#B15">15</a>- <a href="#B18">18</a>]. Previous phylogenetic analyses of chloroplast and cytosolic isoenzymes support the hypothesis that the isoenzymes in angiosperms evolved via a gene duplication event that preceded the divergence of monocots and dicots [ <a href="#B19">19</a>, <a href="#B20">20</a>]. </p> <p>Biochemical studies of green algae provided the first evidence that, as observed in vascular plants, multiple GSII isoenzymes are expressed and localized to the cytosol and chloroplasts within these organisms [ <a href="#B21">21</a>, <a href="#B22">22</a>]. Phylogenetic analyses incorporating the two GSII isoforms characterized in <i>Chlamydomonas reinhardtii </i>[ <a href="#B23">23</a>] uncovered an unusual disparity between the two enzymes [ <a href="#B20">20</a>]. The cytosolic GSII sequence clustered with the vascular plants while the plastid sequence branched more basally and appeared to associate with the eubacterial sequences. </p> <p>Here we examined the evolutionary relationship of the GSII gene family and use increased taxonomic sampling in Chloroplastida to determine if the basally branching, eubacterial-like GSII <sub>B </sub>was broadly distributed. GSII sequences were obtained from four members of the Trebouxiophyceae (Chlorophyta) by PCR amplification using degenerate and gene specific primers. Additional GSII sequences for members of the green algae (Chlorophyta and Prasinophytae) and streptophytes ( <i>Mesostigma</i>, Charales, Desmidiales, Bryophyta, Marchantiophyta, Lycopodiophyta and Tracheophyta) were obtained from publicly available databases, including genome and EST projects. We also increased taxonomic sampling within Eubacteria (to date, GSII genes have not been reported from Archaea). GSII <sub>E </sub>and GSII <sub>B </sub>sequences were identified in members of the green algae and early-diverging streptophytes. Phylogenetic analyses provide support for the hypothesis that GSII <sub>B </sub>was gained in the Chloroplastida from the Eubacteria via a HGT event after the divergence of primary photosynthetic groups. </p> </div> <div class="resultsanddiscussion" title="sec"> <div class="title" tagx="title" title="title">
Results and Discussion</div> <div class="amplificationofgsiigenes" title="sec"> <div class="title" tagx="title" title="title">
Amplification of GSII genes</div> <p>Complete GSII mRNA sequences were obtained from <i>Pseudochlorella </i>sp. CCAP211/1A, <i>Chlorella luteoviridis</i>, <i>Auxenochlorella protothecoides</i>, and <i>Prototheca </i> <i>zopfii</i>. A GSII sequence was also obtained for <i>Pseudochlorella </i>sp. CCAP211/1A that included 912 bp of the ORF and all of the 3'UTR. GenBank accession numbers and characteristics of the transcripts obtained in this study are summarized in Table <a href="#T1">1</a>. </p> <div class="table-wrap_UNKNOWN"> <span class="label" tagx="label" title="label">Table 1</span>  <p>Summary of the GSII sequences characterized in the present study</p>  <table> <thead> <tr> <th>Taxa</th> <th>Sequences Obtained</th> <th>% GC content</th> </tr> </thead> <tbody> <tr> <td /> <td> <b>Accession Number</b> </td> <td> <b>Length (bp)</b> </td> <td> <b>ORF (bp)</b> </td> <td> <b>Amino Acids</b> </td> <td> <b>ORF</b> </td> <td> <b>5' UTR</b> </td> <td> <b>3' UTR</b> </td> </tr> <tr> <td> <hr /> </td> </tr> <tr> <td> <b>GSII <sub>E </sub>Sequences </b> </td> <td /> <td /> <td /> <td /> <td /> <td /> <td /> </tr> <tr> <td>  <i>Pseudochlorella </i>sp. CCAP211/1A (2) </td> <td>GQ465769</td> <td>1486</td> <td>1137</td> <td>378</td> <td>63.32</td> <td>53.19</td> <td>55.12</td> </tr> <tr> <td>  <i>Chlorella luteoviridis </i>UTEX 28 </td> <td>GQ465770</td> <td>1675</td> <td>1146</td> <td>381</td> <td>56.20</td> <td>46.27</td> <td>46.17</td> </tr> <tr> <td>  <i>Auxenochlorella protothecoides</i> </td> <td>GQ465771</td> <td>1621</td> <td>1161</td> <td>386</td> <td>67.96</td> <td>58.18</td> <td>67.28</td> </tr> <tr> <td>  <i>Prototheca zopfii </i>ATCC16527 </td> <td>GQ465772</td> <td>1632</td> <td>1158</td> <td>385</td> <td>69.26</td> <td>71.54</td> <td>72.42</td> </tr> <tr> <td> <hr /> </td> </tr> <tr> <td> <b>GSII <sub>B </sub>Sequences </b> </td> <td /> <td /> <td /> <td /> <td /> <td /> <td /> </tr> <tr> <td>  <i>Pseudochlorella sp</i>. CCAP211/1A (1) </td> <td>GQ491030</td> <td>1266</td> <td>912</td> <td>303</td> <td>56.47</td> <td>n.d.</td> <td>47.93</td> </tr> </tbody> </table> <div class="table-wrap-foot_UNKNOWN"> <p>Four complete sequences of GSIIE and a portion of one GSIIB were obtained from cDNA for the species listed above. NCBI (GenBank) accession numbers are given. Characteristics of the sequences in terms of nucleotide length (Length), size of open reading frame (ORF), and the length of the predicted amino acid sequences (Amino Acids) are presented. The % GC content of open reading frame (ORF) and the 5'and 3' untranslated regions (UTR) of each transcript are presented. </p> </div> </div> </div> <div class="eukaryoticgsiiphylogeny" title="sec"> <div class="title" tagx="title" title="title">
Eukaryotic GSII phylogeny</div> <p>Phylogenetic analyses of GSII amino acid sequences resulted in a well-resolved tree. Assuming the root of the tree lies outside the major eukaryotic clade, there was a clear separation of the eukaryotic (GSII <sub>E</sub>) and eubacterial (GSII <sub>B</sub>) enzymes (Figure <a href="#F1">1</a>). Within the eukaryote clade, the opisthokonts (fungi+animals) and photosynthetic eukaryotes formed separate groups (Figures <a href="#F1">1</a> and <a href="#F2">2</a>). The GSII <sub>E </sub>proteins from streptophytes, chlorophytes, rhodophytes, and chromalveolates formed distinct clades, each with strong to moderate support. The position of the heterokont sequences within this clade is consistent with previous analyses that provide evidence that GSII <sub>E </sub>in heterokonts arose via endosymbiotic gene transfer [ <a href="#B11">11</a>]. Sequences from Chlorophyta (green algae, including representatives of the Chlorophyceae and Trebouxiophyceae) diverged from a basal node within the photosynthetic clade and the Chloroplastida (eukaryotes with chlorophylls <i>a </i>and <i>b</i>) were not monophyletic. However, the deeper nodes within the photosynthetic eukaryotic clade were not well supported and thus the branching pattern within the clade is unresolved (Figure <a href="#F2">2</a>). The streptophyte GSII <sub>E </sub>sequences formed two major groupings; one group contained protein sequences targeted to the chloroplast of angiosperms and the other contained protein sequences of non-vascular and vascular plants that are targeted to the cytosol. Multiple GSII <sub>E </sub>genes were also observed in the gymnosperms ( <i>Pinus </i>spp.) but to date, these appear to function in the cytosol and evidence of plastid targeted isoenzymes is lacking [ <a href="#B24">24</a>]. </p> <div class="fig" title="fig"> <div class="figure" title="figure"> <a href="http://doi.org/"> <span class="label" tagx="label" title="label">Figure 1</span> </a> </div>   <p> <b>Evolutionary relationships among GSII enzymes from prokaryotes and eukaryotes</b>. The phylogenetic analyses were based on 333 amino acid characters from 196 taxa. The 50% majority-rule consensus tree from the Bayesian analyses [ <a href="#B48">48</a>, <a href="#B49">49</a>] is shown as inferred from 20,002 trees as described in the Methods. Nodes with BBP support &amp;gt; 0.95 are represented by thick lines. RAxML [ <a href="#B50">50</a>, <a href="#B51">51</a>] bootstrap values are indicated for nodes recovered in both analyses. RAxML values are not indicated for terminal bifurcations. Eubacterial GSII <sub>B </sub>were used as the outgroup and considered monophyletic. The area of the triangles representing collapsed clades is not proportional to the number of taxa within the clade. </p>   </div> <div class="fig" title="fig"> <div class="figure" title="figure"> <a href="http://doi.org/"> <span class="label" tagx="label" title="label">Figure 2</span> </a> </div>   <p> <b>Evolutionary relationship of GSII <sub>E </sub>genes from eukaryotes, terminal taxa expanded </b>. Phylogenetic analyses are as described in figure <a href="#F1">1</a>. Nodes with BBP support &amp;gt; 0.95 are represented by thick lines. RAxML bootstrap values are indicated for major nodes. RAxML values are not indicated for terminal bifurcations. Sequences characterized in the present study are shown in bold. </p>   </div> </div> <div class="evidenceforthehgtofgsiib" title="sec"> <div class="title" tagx="title" title="title">
Evidence for the HGT of GSII <sub>B</sub> </div> <p>The GSII <sub>B </sub>clade comprised sequences from eubacteria and some members of the Chloroplastida (green algae, liverworts, and mosses; Figures <a href="#F1">1</a> and <a href="#F3">3</a>). The Chloroplastida sequences formed a single clade nested within the eubacterial sequences and branching within the clade was similar to predicted organismal phylogenies [ <a href="#B25">25</a>]. </p> <div class="fig" title="fig"> <div class="figure" title="figure"> <a href="http://doi.org/"> <span class="label" tagx="label" title="label">Figure 3</span> </a> </div>   <p> <b>Evolutionary relationship of GSII <sub>B </sub>genes from prokaryotes and eukaryotes, terminal taxa expanded </b>. The inclusion of GSII <sub>B </sub>genes from eukaryotes (early-diverging Chloroplastida) within prokaryotic GSII <sub>B </sub>clade is evidence of HGT from prokaryotes to eukaryotes within this group. Nodes with BBP support &amp;gt; 0.95 are represented by thick lines. RAxML bootstrap values are indicated for major nodes. RAxML values are not indicated for terminal bifurcations. The sequence characterized in the present study is shown in bold. </p>   </div> <p>GSII <sub>B </sub>sequences are not broadly represented among eubacteria but were identified in members of the Bacteriodetes/Flavobacteria/Cytophaga, Planctomycetes, Verrucomicrobia, Actinobacteria, and the α- and γ-Proteobacteria (Figure <a href="#F3">3</a>; Additional files <a href="#S1">1</a> and <a href="#S2">2</a>). The Chloroplastida GSII <sub>B </sub>was sister to γ-Proteobacteria with strong (Bayesian posterior probability = 1.0) to moderate support (likelihood bootstrap support = 70%). The γ-Proteobacteria + Chloroplastida GSII <sub>B </sub>clade was sister to the Actinobacteria, but this association was not strongly supported. The α-Proteobacteria GSII <sub>B </sub>sequences were not closely related to the γ-Proteobacteria + Chloroplastida GSII <sub>B </sub>clade, which makes the possibility of GSII <sub>B </sub>gain via mitochondrial endosymbiosis unlikely. The α-Proteobacteria GSII <sub>B </sub>were nested within the Verrucomicrobia and thus, we cannot exclude the possibility of an HGT event within the α-Proteobacteria lineage that obscures the mitochondrial origin of the GSII <sub>B </sub>gene in the Chloroplastida. However, the lack of detection of GSII <sub>B </sub>in genomes of other eukaryotic lineages reduces the likelihood of a mitochondrial origin. In addition, EST and genome analyses of other photosynthetic eukaryotes (Glaucophyta, Rhodophyceae and Chromalveolates) and extant cyanobacteria [ <a href="#B26">26</a>, <a href="#B27">27</a>], have not uncovered GSII <sub>B </sub>sequences, reducing the possibility that GSII <sub>B </sub>was acquired via plastid endosymbiosis. Thus, we propose that GSII <sub>B </sub>in the Chloroplastida arose via a HGT from γ-Proteobacteria early in plant evolution. </p> <p>GSII <sub>B </sub>sequences are not broadly distributed among eubacterial lineages and to date, within γ-Proteobacteria, only the genera represented in our analyses have annotated GSII <sub>B </sub>sequences deposited in GenBank. Assuming the GS superfamily evolved prior to the divergence of the three domains of life [ <a href="#B9">9</a>- <a href="#B11">11</a>], the distribution of GSII <sub>B </sub>sequences suggests the gene has been lost in several lineages of Eubacteria and the Archaea. The analysis of GSII <sub>B </sub>may become more robust as additional eubacterial GSII <sub>B </sub>become available through genome sequencing projects. However, gene loss may make the identification of the true donor of GSII <sub>B </sub>to the Chloroplastida difficult. </p> <p>An alternative explanation for the limited distribution of GSII <sub>B </sub>among the eubacteria is that the gene was transferred to the eubacteria from an eukaryotic donor. The possibility of an HGT from Chloroplastida to the γ-Proteobacteria is not supported by our phylogenetic analyses as it implies that the eubacterial sequences would nest within the GSII <sub>E </sub>clade; which has not been observed in our phylogenetic analyses. Eukaryote to eubacterial HGT might be supported if GSII <sub>B </sub>were found in diverse lineages of eukaryotes. Further investigation of GSII diversity in the eukaryotic lineages not represented in our study (e.g., Rhizaria, Excavata and Amoebozoa) will contribute to our understanding of the distribution and evolution of GSII <sub>B</sub>. Given the data at hand, however, the hypothesis that GSII <sub>B </sub>arose in the Chloroplastida via HGT remains the most parsimonious. </p> </div> <div class="estimatingthetimingofthehgt" title="sec"> <div class="title" tagx="title" title="title">
Estimating the timing of the HGT</div> <p>To estimate the relative and absolute timing of the HGT of GSII <sub>B</sub>, we used Bayesian relaxed molecular clock analyses [ <a href="#B28">28</a>]. Both the uncalibrated (Figure <a href="#F4">4</a>) and calibrated analyses (Additional file <a href="#S3">3</a>) show an overlap of the 95% highest density posterior node ranges of the origin of GSII <sub>B </sub>in the early-diverging Chloroplastida coinciding with the GSII <sub>E </sub>divergence in the opisthokonts and in the primary photosynthetic eukaryotes (Archaeplastida). Our analyses indicate that GSII <sub>B </sub>and GSII <sub>E </sub>may have coexisted for an extended period of time and under this scenario, the putative timing of the HGT event from eubacteria to eukaryotes could be placed either prior to or after the divergence of the primary photosynthetic lineages. At present, there is no evidence of GSII <sub>B </sub>in genomes of red algae ( <i>Cyanidioschyzon merolae </i>[ <a href="#B29">29</a>] and <i>Galdieria sulphuraria </i>[ <a href="#B30">30</a>, <a href="#B31">31</a>]), or the glaucophyte <i>Cyanophora paradoxa</i>. We acknowledge that taxon sampling is not extensive within these two lineages and hence cannot exclude the possibility of the existence GSII <sub>B </sub>in these groups. However, given these limited data it is most parsimonious to assume that GSII <sub>B </sub>was acquired only by the Chloroplastida, early after the divergence from the Glaucophyta and Rhodophyceae (red algae). </p> <div class="fig" title="fig"> <div class="figure" title="figure"> <a href="http://doi.org/"> <span class="label" tagx="label" title="label">Figure 4</span> </a> </div>   <p> <b>Maximum clade probability chronogram from the BEAST analysis of the GSII <sub>B </sub>and GSII <sub>E </sub>amino acid sequence alignment </b>. All lineages were allowed to evolve according to a relaxed molecular clock and WAG + Inv + Gamma model. Bars on nodes indicate the width of the 95% highest posterior density for each divergence time. </p>   </div> <p>The distribution of GSII <sub>B </sub>within Chloroplastida covers the major lineages of Chlorophyta (Chlorophyceae, Trebouxiophyceae, and Prasinophyceae; Additional file <a href="#S1">1</a>). In addition, a partial GSII <sub>B </sub>sequence was identified in a member of the Ulvophyceae ( <i>Acetabularia acetabulum</i>; (Additional files <a href="#S2">2</a> and <a href="#S4">4</a>). Within Streptophyta, GSII <sub>B </sub>genes are present in Mesostigmatophyceae ( <i>Mesostigma viride</i>; Additional files <a href="#S2">2</a> and <a href="#S4">4</a>), Zygnemophyceae (Desmidiales; <i>Closterium peracerosum-strigosum-littorale </i>complex), Marchantiophyta (liverworts; <i>Marchantia polymorpha</i>) and Bryophyta (mosses; <i>Physcomitrella patens</i>). GSII <sub>B </sub>is absent from the single Lycopodiophyta genome ( <i>Selaginella moellendorffii</i>) and from all seed plants. Hence, we propose that GSII <sub>B </sub>was lost in the plant lineage after the colonization of land by early plants, marked by the divergence of bryophytes and lycopodiophytes, which is one of the oldest vascular plant lineages [ <a href="#B32">32</a>]. </p> </div> <div class="functionallocalizationandgsiibgeneduplication" title="sec"> <div class="title" tagx="title" title="title">
Functional localization and GSII <sub>B </sub>gene duplication </div> <p>The Chloroplastida lineages that contain the GSII <sub>B </sub>gene also have a GSII <sub>E </sub>counterpart, which attaches to a basal node within the photosynthetic eukaryotes (Figure <a href="#F1">1</a>). Both the GSII <sub>B </sub>and GSII <sub>E </sub>genes are nuclear encoded and thus we identified the cellular location of each of the gene products based on the presence (organelle-localized) or absence (cytosol-localized) of N-terminal transit peptides using TargetP ver. 1.1 ([ <a href="#B33">33</a>], see Additional file <a href="#S5">5</a>). None of the early-diverging Chloroplastida GSII <sub>E </sub>enzymes contained transit peptides. In contrast, chloroplast transit sequences were identified in the GSII <sub>B </sub>protein sequences from <i>Chlorella </i>sp. NC64A, <i>C. vulgaris </i>and the streptophyte, <i>Closterium peracerosum-strigosum-littorale </i>(Zygnemophyceae) but not in the moss ( <i>P. patens</i>) or liverwort ( <i>M. polymorpha</i>). Mitochondrial-targeting transit peptides were predicted in GSII <sub>B </sub>sequences from <i>C. reinhardtii</i>, <i>Volvox carteri </i>f. <i>nagariensis </i>and <i>Scenedesmus obliquus </i>(see Additional file <a href="#S5">5</a>). Previous work indicated that chloroplast transit sequences from <i>C. reinhardtii </i>shared features with both mitochondrial and higher plant chloroplast pre-sequences [ <a href="#B34">34</a>] and thus the prediction of a mitochondrial location of GSII <sub>B </sub>may not reflect its true functional localization. Alternatively, GSII <sub>B </sub>may be targeted to both the mitochondria and chloroplast, similar to what is observed for GSII <sub>E </sub>in leaves of some vascular plants [ <a href="#B35">35</a>, <a href="#B36">36</a>]. While experimental evidence is required to confirm the cellular localization of the GSII <sub>B</sub>, it appears that the GSII <sub>B </sub>enzymes function in either the chloroplast or mitochondrion in the chlorophytes and early-diverging streptophytes ( <i>Closterium </i>sp.) and that GSII <sub>E </sub>functions in the cytosol. </p> <p>The GSII <sub>B </sub>gene is duplicated in <i>C. reinhardtii, V. carteri </i>f <i>nagariensis </i>and <i>P. patens</i>. The duplicated copies of GSII <sub>B </sub>in <i>C. reinhardtii </i>and <i>V. carteri </i>f <i>nagariensis </i>were nearly identical (90% and 95% identical, respectively) and present in the genome in a head-to-head orientation. Similarly, the GSII <sub>B </sub>genes in <i>P. patens </i>were 98% identical but do not appear to be in close genomic proximity. Within our phylogenetic analyses (Figure <a href="#F3">3</a>), the duplicated GSII <sub>B </sub>of <i>C. reinhardtii</i>, <i>V. carteri </i>f. <i>nagariensis </i>and <i>P. patens </i>each formed separate clades, suggesting the genes evolved by independent duplication events. Alternatively, the GSII <sub>B </sub>genes in <i>C. reinhardtii </i>and <i>V. carteri </i>may have evolved via an early duplication within the Chlamydomonadales with subsequent gene conversion following the divergence of these lineages. The GSII <sub>B </sub>are differentially expressed in <i>C. reinhardtii </i>suggesting the need for maintenance of both the copies in the organism [ <a href="#B37">37</a>]. </p> </div> <div class="gsiiblossandreplacementoffunction" title="sec"> <div class="title" tagx="title" title="title">
GSII <sub>B </sub>loss and replacement of function </div> <p>In contrast to the expression of GSII <sub>E </sub>and GSII <sub>B </sub>genes in the early-diverging Chloroplastida, the chloroplast- and cytosolic-localized GSII enzymes in angiosperms are both members of the GSII <sub>E </sub>family and form two distinct clades in our phylogenetic analyses (Figures <a href="#F1">1</a> and <a href="#F2">2</a>). As predicted in earlier studies [ <a href="#B19">19</a>], the genes encoding these enzymes arose via a recent gene duplication event with further expansion in the number of genes encoding cytosolic isoenzymes in several plant lineages (Figure <a href="#F2">2</a>, [ <a href="#B38">38</a>, <a href="#B39">39</a>]). Since GSII <sub>B </sub>is absent from vascular plants, it appears that the chloroplast function of GSII <sub>B </sub>has been replaced by a gene duplication event in higher plants allowing for subsequent loss of the gene from this lineage. There is also an expansion of the GSII <sub>E </sub>gene family in gymnosperms (Figure <a href="#F2">2</a>), but the enzymes are all localized to the cytosol and the plastid targeted isoform appears to have been lost from this group. The expansion of the GSII <sub>E </sub>gene family coincides with the development of vascularization of land plants and maybe correlated with the partitioning of nitrogen assimilation between below and above ground tissue (see Additional file <a href="#S3">3</a>). </p> </div> </div> <div class="conclusions" title="sec"> <div class="title" tagx="title" title="title">
Conclusions</div> <p>We have provided evidence of an ancient HGT event involving the gene for an essential enzyme, GSII. GSII has been well characterized at the molecular level in angiosperms but has been largely overlooked in the early-diverging plant lineages, which were addressed in the present study. Although recent comparative genomic analyses failed to identify bacterial genes in <i>Chlamydomonas reinhardtii </i>[ <a href="#B40">40</a>], our discovery of a eubacterial-like GSII in the chlorophytes and early-diverging streptophytes suggests that further exploration within these lineages is merited. The branching pattern within the monophyletic assemblage of the chlorophytes and early-diverging streptophytes is similar to other molecular and organismal phylogenies, suggesting the occurrence of a single HGT event. As a result, GSII <sub>B </sub>may be useful in resolving taxonomic associations within and among green algal and early-diverging streptophyte lineages. </p> <p>Several genes of bacterial origin have been identified in <i>Dictyostelium discoideum </i>and are thought to be advantageous to organisms living in soil [ <a href="#B40">40</a>]. More recently, Richards et al. [ <a href="#B41">41</a>] identified five genes in plants that appear to be of fungal origin and argue that two may have been advantageous for organisms colonizing a terrestrial environment. We propose that the acquisition of enzymes by HGT results in a more rapid change in enzymatic capacity or kinetic diversity than evolution of isoenzymes by gene duplication and subsequent specialization. Biochemical studies have suggested that GSII <sub>B </sub>has a lower affinity for NH <sub>4 </sub> <sup>+ </sup>and Glu than GSII <sub>E </sub>[ <a href="#B42">42</a>], characteristics that would be advantageous for enzymes assimilating higher concentrations of NH <sub>4 </sub> <sup>+ </sup>from environmental sources, NO <sub>3 </sub> <sup>- </sup>assimilation, or increased rates of photorespiration. Increased taxon sampling and an enlarged fossil age constraint dataset will allow for a more detailed examination of the timing of GSII gains and losses over geological history and coupled with major transitions in plant evolution. </p> </div> <div class="methods" title="methods"> <div class="title" tagx="title" title="title">
Methods</div> <div class="algalculturesandsequencing" title="sec"> <div class="title" tagx="title" title="title">
Algal cultures and sequencing</div> <p>Four members in the class Trebouxiophyceae were selected for GSII gene amplification. Cultures of <i>Pseudochlorella </i>sp. CCAP211/1A, <i>Chlorella luteoviridis</i>, and <i>Auxenochlorella protothecoides </i>were a gift from Dr. Peggy Winter (University of West Florida), and <i>Prototheca zopfii </i>was a gift from Dr. Drion Boucias (University of Florida). Cultures were grown axenically in ATCC medium 847, ( <i>Pseudochlorella </i>sp. CCAP211/1A, <i>C. luteoviridis </i>and <i>A. protothecoides</i>) and in ATCC medium 28: Emmons' modification of Sabouraud's agar ( <i>P. zopfii</i>) at 17°C and 12:12 h light: dark cycle. Cells were collected by centrifugation (approximately 50 mL of culture), flash frozen in liquid nitrogen, ground in a mortar and pestle and subjected to DNA and RNA extraction. DNA was extracted using a hexadecyltrimethylammonium bromide extraction protocol [ <a href="#B43">43</a>]. RNA was extracted using an RNeasy <sup>® </sup>Mini Kit (Qiagen Inc., Valencia, CA) with modifications outlined in Brown et al. [ <a href="#B44">44</a>] for extraction with glass beads using bead beating. Extracted nucleic acids were quantified spectrophotometrically for downstream applications using a MWG BIOTECH Lambda Scan 200×, 96-well Microplate Reader with KCJunior Software (MWG BIOTECH, High Point, NC). cDNA was synthesized using an Omniscript RT kit (Qiagen Inc., Valencia, CA). Total RNA (1.5 μg) was used as a template and the oligo-d (T) primer GCGGCCGCTCTAGACTAG(T) <sub>18 </sub>as the first strand primer. Primers were designed to target specifically GSII <sub>E </sub>and GSII <sub>B </sub>sequences. GSII <sub>E </sub>primers were based on existing sequences from vascular plants, chlorophytes and rhodophytes. GSII <sub>B </sub>primers were based on existing sequences from <i>Chlamydomonas reinhardtii </i>and <i>Physcomitrella patens</i>. PCR was performed in a final volume of 25 μL with Taq PCR core kit (Qiagen) with Q solution to overcome problems associated with high GC content. Primer sequences are listed in Table <a href="#T2">2</a>. Thermal conditions for GSII <sub>E</sub>: 30 cycles of 95°C for 30s, 50°C for 30s, 72°C for 1 min, performed for 30 cycles. Thermal conditions for GSII <sub>B</sub>: Initial denaturation of 94°C for 2 min, followed by 35 cycles of 94°C for 1 min, 51°C for 1 min, 72°C for 1 min and extension at 72°C for 5 min. </p> <div class="table-wrap_UNKNOWN"> <span class="label" tagx="label" title="label">Table 2</span>  <p>Primers used for amplification of GSII genes from green algae</p>  <table> <thead> <tr> <th>Gene</th> <th>Primer name</th> <th>Direction</th> <th>Sequence</th> </tr> </thead> <tbody> <tr> <td /> <td>MossGS2-1F</td> <td>Forward</td> <td>5'-TGGGTTGATGGTMANGARGG-3'</td> </tr> <tr> <td>GSII <sub>B</sub> </td> <td>MossGS2-2R</td> <td>Reverse</td> <td>5'-ATNCCGAAMTCTTCNCC-3'</td> </tr> <tr> <td /> <td>Green UNI 1-F</td> <td>Forward</td> <td>5'-CCIRAITGGWSITTYGAYGG-3'</td> </tr> <tr> <td /> <td>cpGSII(QGPFY)-R</td> <td>Reverse</td> <td>5'-CCRCARTARAAIGGICCYTGIGG-3'</td> </tr> <tr> <td> <hr /> </td> </tr> <tr> <td /> <td>GALG GS F</td> <td>Forward</td> <td>5' - TGC CCA TCC CCA CCA ACA C - 3'</td> </tr> <tr> <td>GSII <sub>E</sub> </td> <td>GALG GS R</td> <td>Reverse</td> <td>5' - TCT CGT GCT TGC CCG TCA GG - 3'</td> </tr> <tr> <td /> <td>GS2ChloroF</td> <td>Forward</td> <td>5' - CGG CWT CGA GCA GGA GTA CAC - 3'</td> </tr> <tr> <td /> <td>GS2ChloroR</td> <td>Reverse</td> <td>5' - CCG AYC TGG WAC TCC CAC TGG - 3'</td> </tr> </tbody> </table> <div class="table-wrap-foot_UNKNOWN"> <p>Sequences of degenerate primers are presented using IUBMB single letter codes. I represents inosine. </p> </div> </div> <p>Nested PCR amplification was used to obtain GSII <sub>B </sub>sequences with first round done with cDNA and primer concentrations of 0.4 μM (MossGS2-1F [forward] and MossGS2-2R [reverse]). The amplicon was used for a second round of amplification with primers Green UNI 1-F (forward) and cpGSII(QGPFY)-R (reverse) and yielded a DNA fragment of 330 bp. Amplified sequences were cleaned and sequenced either commercially (MWG Biotech, Charlotte, NC and Macrogen, Seoul, South Korea) or at Clark University using an automated DNA sequencer (ABI 3130), with ABI Prism Terminator Big Dye ver 3.1 (Applied Biosystems, Carlsbad, CA). Some PCR products were cloned into TOPO vectors following the manufacturer's protocol (TOPO TA Cloning Kit for Sequencing, Invitrogen, Carlsbad, CA) prior to sequencing. Rapid Amplification of cDNA Ends (RACE) methods were used to obtain the entire open reading frame for GSII <sub>E </sub>sequences from <i>Pseudochlorella </i>sp. CCAP211/1A, C. <i>luteoviridis</i>, <i>A. protothecoides </i>and <i>P. zopfii </i>and partial GSII <sub>B </sub>sequence from <i>Pseudochlorella </i>sp. CCAP211/1A. 3' RACE reactions used a combination of gene specific primers and a portion of the oligo-d (T) primer (GCGGCCGCTCTAGACTAGT) used for cDNA synthesis. 5' RACE reactions were performed using 5' RACE System version 2.0 from Invitrogen (Invitrogen) and SMART™ RACE cDNA Amplification Kit (Clontech Laboratories Inc., Mountain View, CA) following manufacturers' recommendations. Contigs were assembled using CodonCode Aligner (CodonCode Corporation, Deadham, MA). All sequences were translated into amino acids <i>in silico</i>. </p> </div> <div class="phylogeneticanalyses" title="sec"> <div class="title" tagx="title" title="title">
Phylogenetic analyses</div> <p>GSII sequences were retrieved from public databases as well as genome and EST projects using the GSII sequence from the diatom <i>Skeletonema costatum </i>(AAC77446) as query, or glutamine synthetase as a keyword. Subsequent queries with eubacterial GSII sequences did not retrieve any additional sequences. Complete information on taxa, database sources and accession numbers is provided in Additional file <a href="#S1">1</a>. The initial alignment of amino acid sequences was done with the web based program CLUSTAL W, using default parameters [ <a href="#B45">45</a>], followed by manual adjustment using BioEdit Sequence Alignment Editor [ <a href="#B46">46</a>] and MacClade 4.08 [ <a href="#B47">47</a>]. The N- and C terminal ends of the proteins along with highly variable regions within the alignments were excluded in the phylogenetic analyses. </p> <p>The final GSII alignment consisted of 196 taxa and 333 characters for Bayesian analysis. Trees were inferred by calculating Bayesian posterior probabilities using MrBayes 3.1.2 [ <a href="#B48">48</a>, <a href="#B49">49</a>]. Two parallel runs, each with four chains (three heated and one cold) were run for 10 <sup>6 </sup>generations. The evolutionary models implemented in MrBayes3.1.2 were explored using the mixed amino acid model. Rate variation across sites was approximated using a gamma distribution with proportion of invariable sites estimated from the data. Trees were sampled every 100 generations. Likelihood tree scores of two independent runs were plotted to estimate the point of convergence to a stable likelihood, and to determine the trees to be excluded via "burnin." Bayesian posterior probabilities of the branches were calculated from trees from both the runs, totaling 20,002 trees. Trees remaining (10,000) after a burnin of 5001 for each run were used to compute a 50% majority-rule consensus. </p> <p>Maximum likelihood (ML) based inference of the phylogenetic trees was done using the software RAxML 7.0.4 [ <a href="#B50">50</a>, <a href="#B51">51</a>]. The analysis used a random starting tree and the rapid hill-climbing algorithm (i.e., option -f d in RAxML) and the WAG model of amino acid substitution were used. A random seed number was used to turn on rapid bootstrapping (-x) and 1000 bootstrap trees were generated by invoking -# 1000 and - x options in RAxML. A majority rule consensus tree was created in PAUP* 4.0 b [ <a href="#B52">52</a>]. The phylogenetic trees in figures <a href="#F1">1</a>, <a href="#F2">2</a> &amp;amp; <a href="#F3">3</a> are the 50% majority rule consensus trees from the Bayesian analyses on which the RAxML bootstrap values have been indicated. The eubacterial GSII <sub>B </sub>sequences were used as the monophyletic outgroup in the graphical representation of the phylogenies. </p> </div> <div class="predictionoffunctionallocalizationofgsiibandgsiieproteinsequencesinearly-divergingchloroplastida" title="sec"> <div class="title" tagx="title" title="title">
Prediction of functional localization of GSII <sub>B </sub>and GSII <sub>E </sub>protein sequences in early-diverging Chloroplastida </div> <p>We used the web-based programs TargetP 1.1[ <a href="#B33">33</a>] and ChloroP 1.1 [ <a href="#B53">53</a>] to identify N-terminal transit peptides in GSII <sub>B </sub>and GSII <sub>E </sub>proteins (see Additional file <a href="#S5">5</a>). </p> </div> <div class="estimationofdivergencetimes" title="sec"> <div class="title" tagx="title" title="title">
Estimation of divergence times</div> <p>We estimated the divergence times using Bayesian approach implemented in BEAST 1.4.8 [ <a href="#B28">28</a>]. We did an un-calibrated and calibrated run. A relaxed molecular clock model of uncorrelated log normal distribution was used. For the un-calibrated analysis, a starting tree generated by RAxML 7.0.4 [ <a href="#B50">50</a>] was used as the input tree with the GS amino acid sequence alignment. For the calibrated analysis we set uniform priors on tmrca parameter. Fossil dates were used as minimum dates and were, as follows, Ascomycota, 400 MYA [ <a href="#B54">54</a>], Bilateria, 550 MYA [ <a href="#B55">55</a>] and streptophytes 475 MYA [ <a href="#B56">56</a>]. Secondary age constraints based on published estimates of divergence times were not used. We used the following models, WAG + Inv + Gamma with priors, birth death speciation on the tree. Markov Chain Monte Carlo was set to default 10 million with sampling at every 1000 generation, resulting in 10,000 trees. Convergence was assessed in Tracer v 1.4 [ <a href="#B57">57</a>] and the first three million samples were excluded as burnin. A maximum clade credibility tree was generated by analyzing the BEAST tree file in TreeAnnotator 1.4.6 [ <a href="#B58">58</a>]. This program determined the 95% highest posterior densities and estimated the node heights as mean heights. </p> </div> </div> <div class="abbreviations" title="sec"> <div class="title" tagx="title" title="title">
Abbreviations</div> <p>GS: Glutamine synthetase; HGT: Horizontal gene transfer; BPP: Bayesian posterior probability;</p> </div> <div class="authors'contributions" title="sec"> <div class="title" tagx="title" title="title">
Authors' contributions</div> <p>SG participated in the cloning and sequencing of GSII genes, assembled the data matrix, conducted all phylogenetic analyses, and drafted the manuscript. MB participated in the execution and interpretation of the relaxed molecular clock analyses and assisted in the writing of the manuscript. AT was responsible for the cloning and sequencing of GSII genes and assisted in writing the manuscript. DLR conceived of the study, coordinated the research, and helped draft the manuscript. All authors read and approved the final manuscript. </p> </div> <div class="supplementary-material" title="supplementary-material"> <div class="title" tagx="title" title="title">
Supplementary Material</div> <div class="local-data" title="local-data">  <div class="title" tagx="title" title="title">
Additional file 1</div> <p> <b>GSII protein sequences used in the present study</b>. GenBank accession numbers and JGI DOE scaffold and protein ID information for the GSII proteins are provided. </p>  <div class="media" title="media"> <a href="1471-2148-10-198-S1.DOC">LINK</a>  <p>Click here for file</p>  </div> </div> <div class="local-data" title="local-data">  <div class="title" tagx="title" title="title">
Additional file 2</div> <p> <b>Identification of GSII <sub>B </sub>sequences from <i>Acetabularia acetabulum </i>and <i>Mesostigma viride</i> </b>. The data matrix used in this analyses is described in the Methods with sequences from the following taxa included in the alignment: <i>Acetabularia acetabulum </i>(Chlorophyta, Ulvophyceae), <i>Mesostigma viride </i>(Streptophyta, Mesostigmatophyceae), which are shown in bold. <i>Pseudochlorella </i>sp. CCAP211/1A (1) GSII <sub>B </sub>amplified in the present study is also shown in bold. Tree used for this illustration is a representative derived from parsimony heuristic search analysis. RAxML bootstrap values are shown for the major clades of GSII genes, which were derived from 1000 bootstrap replicates with the parameters described in the Methods. </p>  <div class="media" title="media"> <a href="1471-2148-10-198-S2.PDF">LINK</a>  <p>Click here for file</p>  </div> </div> <div class="local-data" title="local-data">  <div class="title" tagx="title" title="title">
Additional file 3</div> <p> <b>Maximum clade probability tree displayed as a chronogram from the BEAST analysis of the GS amino acid sequence alignment </b>. All lineages evolved according to a relaxed molecular clock and WAG + Inv. + Gamma model. Node bars indicate the width of the 95% highest posterior density with minimum and maximum values in parentheses. Bold numbers near the nodes indicate node ages. Major lineages are depicted as collapsed triangles. Nodes for which fossil dates were used are marked as A = Acsomycota, 400MYA, B = Bilateria 550MYA and C = Streptophytes 475 MYA. P = Paleozoic; M = Mesozoic; C = Cenozoic; Camb. = Cambrian. </p>  <div class="media" title="media"> <a href="1471-2148-10-198-S3.PDF">LINK</a>  <p>Click here for file</p>  </div> </div> <div class="local-data" title="local-data">  <div class="title" tagx="title" title="title">
Additional file 4</div> <p> <b>Partial GSII <sub>B </sub>sequences were identified in two additional Chloroplastida by preliminary phylogenetic analyses </b>. Partial GSII <sub>B </sub>sequences were obtained for <i>Acetabularia </i> <i>acetabulum </i>(Chlorophyta: Ulvophyceae) and <i>Mesostigma </i> <i>viride </i>(Streptophyta: Mesostigmatophyceae). These sequences were not included in the phylogenetic analyses presented within the paper (figures 1, 2, 3 and 4) due to their short length. Phylogenetic analysis confirming these proteins as members of the GSII <sub>B </sub>clade is presented in Additional file <a href="#S2">2</a>. </p>  <div class="media" title="media"> <a href="1471-2148-10-198-S4.DOC">LINK</a>  <p>Click here for file</p>  </div> </div> <div class="local-data" title="local-data">  <div class="title" tagx="title" title="title">
Additional file 5</div> <p> <b>Predicted cellular localization of GSII proteins in early-diverging Chloroplastida</b>. Only taxa containing both GSII <sub>B </sub>and GSII <sub>E </sub>within Chloroplastida were analyzed. N-terminal sequences of GSII proteins were analyzed for organellar transit peptides, <i>in silico</i>. References are given for established functional localizations. </p>  <div class="media" title="media"> <a href="1471-2148-10-198-S5.DOC">LINK</a>  <p>Click here for file</p>  </div> </div> </div> </div> <div class="back" title="back"> <div class="acknowledgements" title="sec"> <div class="title" tagx="title" title="title">
Acknowledgements</div> <p>We thank Jenna Nguyen and Jacqueline Mitchell for their assistance with this project and David Hibbett for helpful discussions regarding the phylogenetic analyses. We also thank two anonymous reviewers and the editor for their constructive comments and recommendations. This research was supported by an NSF CAREER Award (IBN 0238426) to DLR. </p> </div> <div class="references">
References</div> <div tag="ref-list"> <ul> <li tag="ref"> <a name="B1" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Kumada</span> <span class="given-names" tagx="given-names" title="given-names">Y</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Benson</span> <span class="given-names" tagx="given-names" title="given-names">DR</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Hillemann</span> <span class="given-names" tagx="given-names" title="given-names">D</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Hosted</span> <span class="given-names" tagx="given-names" title="given-names">TJ</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Rochefort</span> <span class="given-names" tagx="given-names" title="given-names">DA</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Thompson</span> <span class="given-names" tagx="given-names" title="given-names">CJ</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Wohlleben</span> <span class="given-names" tagx="given-names" title="given-names">W</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Tateno</span> <span class="given-names" tagx="given-names" title="given-names">Y</span> </span> <span class="mixed-article-title" title="mixed-article-title">Evolution of the glutamine synthetase gene, one of the oldest existing and functioning genes </span> <span class="source" tagx="source" title="source">Proc Natl Acad Sci USA</span> <span class="year" tagx="year" title="year">1993</span> <span class="volume" tagx="volume" title="volume">90</span> <span class="fpage" tagx="fpage" title="fpage">3009</span> <span class="lpage" tagx="lpage" title="lpage">3013</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1073/pnas.90.7.3009">10.1073/pnas.90.7.3009</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/8096645">8096645</a> </span> </span> </li> <li tag="ref"> <a name="B2" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Pesole</span> <span class="given-names" tagx="given-names" title="given-names">G</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Bozzetti</span> <span class="given-names" tagx="given-names" title="given-names">MP</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Lanave</span> <span class="given-names" tagx="given-names" title="given-names">C</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Preparata</span> <span class="given-names" tagx="given-names" title="given-names">G</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Saccone</span> <span class="given-names" tagx="given-names" title="given-names">C</span> </span> <span class="mixed-article-title" title="mixed-article-title">Glutamine synthetase gene evolution: A good molecular clock</span> <span class="source" tagx="source" title="source">Proc Natl Acad Sci USA</span> <span class="year" tagx="year" title="year">1991</span> <span class="volume" tagx="volume" title="volume">88</span> <span class="fpage" tagx="fpage" title="fpage">522</span> <span class="lpage" tagx="lpage" title="lpage">526</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1073/pnas.88.2.522">10.1073/pnas.88.2.522</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/1671172">1671172</a> </span> </span> </li> <li tag="ref"> <a name="B3" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Brown</span> <span class="given-names" tagx="given-names" title="given-names">JR</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Masuchi</span> <span class="given-names" tagx="given-names" title="given-names">Y</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Robb</span> <span class="given-names" tagx="given-names" title="given-names">FT</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Doolittle</span> <span class="given-names" tagx="given-names" title="given-names">WF</span> </span> <span class="mixed-article-title" title="mixed-article-title">Evolutionary relationships of bacterial and archaeal glutamine synthetase genes</span> <span class="source" tagx="source" title="source">J Mol Evol</span> <span class="year" tagx="year" title="year">1994</span> <span class="volume" tagx="volume" title="volume">38</span> <span class="fpage" tagx="fpage" title="fpage">566</span> <span class="lpage" tagx="lpage" title="lpage">576</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1007/BF00175876">10.1007/BF00175876</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/7916055">7916055</a> </span> </span> </li> <li tag="ref"> <a name="B4" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Eisenberg</span> <span class="given-names" tagx="given-names" title="given-names">D</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Gill</span> <span class="given-names" tagx="given-names" title="given-names">HS</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Pfluegl</span> <span class="given-names" tagx="given-names" title="given-names">GMU</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Rotstein</span> <span class="given-names" tagx="given-names" title="given-names">SH</span> </span> <span class="mixed-article-title" title="mixed-article-title">Review: Structure-function relationships of glutamine synthetases</span> <span class="source" tagx="source" title="source">Biochim Biophys Acta</span> <span class="year" tagx="year" title="year">2000</span> <span class="volume" tagx="volume" title="volume">1477</span> <span class="fpage" tagx="fpage" title="fpage">122</span> <span class="lpage" tagx="lpage" title="lpage">145</span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/10708854">10708854</a> </span> </span> </li> <li tag="ref"> <a name="B5" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Garcia-Dominguez</span> <span class="given-names" tagx="given-names" title="given-names">M</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Reyes</span> <span class="given-names" tagx="given-names" title="given-names">JC</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Florencio</span> <span class="given-names" tagx="given-names" title="given-names">FJ</span> </span> <span class="mixed-article-title" title="mixed-article-title">Purification and characterization of a new type of glutamine synthetase from cyanobacteria</span> <span class="source" tagx="source" title="source">Eur J Biochemistry</span> <span class="year" tagx="year" title="year">1997</span> <span class="volume" tagx="volume" title="volume">244</span> <span class="fpage" tagx="fpage" title="fpage">258</span> <span class="lpage" tagx="lpage" title="lpage">264</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1111/j.1432-1033.1997.00258.x">10.1111/j.1432-1033.1997.00258.x</a> </span> </span> </li> <li tag="ref"> <a name="B6" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Reyes</span> <span class="given-names" tagx="given-names" title="given-names">JC</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Florencio</span> <span class="given-names" tagx="given-names" title="given-names">FJ</span> </span> <span class="mixed-article-title" title="mixed-article-title">A new type of glutamine synthetase in cyanobacteria: The protein encoded by the <i>gln</i>N gene supports nitrogen assimilation in <i>Synechocystis </i>sp. strain PCC 6803 </span> <span class="source" tagx="source" title="source">J Bacter</span> <span class="year" tagx="year" title="year">1994</span> <span class="volume" tagx="volume" title="volume">176</span> <span class="fpage" tagx="fpage" title="fpage">1260</span> <span class="lpage" tagx="lpage" title="lpage">1267</span> </span> </li> <li tag="ref"> <a name="B7" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Robertson</span> <span class="given-names" tagx="given-names" title="given-names">DL</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Alberte</span> <span class="given-names" tagx="given-names" title="given-names">RS</span> </span> <span class="mixed-article-title" title="mixed-article-title">Isolation and characterization of glutamine synthetase from the marine diatom <i>Skeletonema costatum</i> </span> <span class="source" tagx="source" title="source">Plant Physiol</span> <span class="year" tagx="year" title="year">1996</span> <span class="volume" tagx="volume" title="volume">111</span> <span class="fpage" tagx="fpage" title="fpage">1169</span> <span class="lpage" tagx="lpage" title="lpage">1175</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1104/pp.111.4.1169">10.1104/pp.111.4.1169</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/8756499">8756499</a> </span> </span> </li> <li tag="ref"> <a name="B8" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Robertson</span> <span class="given-names" tagx="given-names" title="given-names">DL</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Smith</span> <span class="given-names" tagx="given-names" title="given-names">GJ</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Alberte</span> <span class="given-names" tagx="given-names" title="given-names">RS</span> </span> <span class="mixed-article-title" title="mixed-article-title">Glutamine synthetase in marine algae: new surprises from an old enzyme</span> <span class="source" tagx="source" title="source">J Phycol</span> <span class="year" tagx="year" title="year">2001</span> <span class="volume" tagx="volume" title="volume">37</span> <span class="fpage" tagx="fpage" title="fpage">793</span> <span class="lpage" tagx="lpage" title="lpage">795</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1046/j.1529-8817.2001.01057.x">10.1046/j.1529-8817.2001.01057.x</a> </span> </span> </li> <li tag="ref"> <a name="B9" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Brown</span> <span class="given-names" tagx="given-names" title="given-names">JR</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Doolittle</span> <span class="given-names" tagx="given-names" title="given-names">WF</span> </span> <span class="mixed-article-title" title="mixed-article-title">Archaea and the prokaryote-to-eukaryote transition</span> <span class="source" tagx="source" title="source">Microbiol Mol Biol Rev</span> <span class="year" tagx="year" title="year">1997</span> <span class="volume" tagx="volume" title="volume">61</span> <span class="fpage" tagx="fpage" title="fpage">456</span> <span class="lpage" tagx="lpage" title="lpage">502</span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/9409149">9409149</a> </span> </span> </li> <li tag="ref"> <a name="B10" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Mathis</span> <span class="given-names" tagx="given-names" title="given-names">R</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Gamas</span> <span class="given-names" tagx="given-names" title="given-names">P</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Meyer</span> <span class="given-names" tagx="given-names" title="given-names">Y</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Cullimore</span> <span class="given-names" tagx="given-names" title="given-names">JV</span> </span> <span class="mixed-article-title" title="mixed-article-title">The presence of GSI-like genes in higher plants: Support for the paralogous evolution of GSI and GSII </span> <span class="source" tagx="source" title="source">J Mol Evol</span> <span class="year" tagx="year" title="year">2000</span> <span class="volume" tagx="volume" title="volume">50</span> <span class="fpage" tagx="fpage" title="fpage">116</span> <span class="lpage" tagx="lpage" title="lpage">122</span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/10684345">10684345</a> </span> </span> </li> <li tag="ref"> <a name="B11" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Robertson</span> <span class="given-names" tagx="given-names" title="given-names">DL</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Tartar</span> <span class="given-names" tagx="given-names" title="given-names">A</span> </span> <span class="mixed-article-title" title="mixed-article-title">Evolution of glutamine synthetase in heterokonts: evidence for endosymbiotic gene transfer and the early evolution of photosynthesis </span> <span class="source" tagx="source" title="source">Mol Biol Evol</span> <span class="year" tagx="year" title="year">2006</span> <span class="volume" tagx="volume" title="volume">23</span> <span class="fpage" tagx="fpage" title="fpage">1048</span> <span class="lpage" tagx="lpage" title="lpage">1055</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1093/molbev/msj110">10.1093/molbev/msj110</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/16495348">16495348</a> </span> </span> </li> <li tag="ref"> <a name="B12" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Carlson</span> <span class="given-names" tagx="given-names" title="given-names">TA</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Chelm</span> <span class="given-names" tagx="given-names" title="given-names">BK</span> </span> <span class="mixed-article-title" title="mixed-article-title">Apparent eucaryotic origin of glutamine synthetase II from the bacterium <i>Bradyrhizobium japonicum</i> </span> <span class="source" tagx="source" title="source">Nature</span> <span class="year" tagx="year" title="year">1986</span> <span class="fpage" tagx="fpage" title="fpage">568</span> <span class="lpage" tagx="lpage" title="lpage">570</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1038/322568a0">10.1038/322568a0</a> </span> </span> </li> <li tag="ref"> <a name="B13" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Shatters</span> <span class="given-names" tagx="given-names" title="given-names">RG</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Kahn</span> <span class="given-names" tagx="given-names" title="given-names">ML</span> </span> <span class="mixed-article-title" title="mixed-article-title">Glutamine synthetase II in <i>Rhizobium</i>: reexamination of the proposed horizontal transfer of DNA from eukaryotes to prokaryotes </span> <span class="source" tagx="source" title="source">J Mol Evol</span> <span class="year" tagx="year" title="year">1989</span> <span class="volume" tagx="volume" title="volume">29</span> <span class="fpage" tagx="fpage" title="fpage">422</span> <span class="lpage" tagx="lpage" title="lpage">428</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1007/BF02602912">10.1007/BF02602912</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/2575672">2575672</a> </span> </span> </li> <li tag="ref"> <a name="B14" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Adl</span> <span class="given-names" tagx="given-names" title="given-names">SM</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Simpson</span> <span class="given-names" tagx="given-names" title="given-names">AGB</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Farmer</span> <span class="given-names" tagx="given-names" title="given-names">MA</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Andersen</span> <span class="given-names" tagx="given-names" title="given-names">RA</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Anderson</span> <span class="given-names" tagx="given-names" title="given-names">OR</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Barta</span> <span class="given-names" tagx="given-names" title="given-names">JR</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Bowser</span> <span class="given-names" tagx="given-names" title="given-names">SS</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Brugerolle</span> <span class="given-names" tagx="given-names" title="given-names">G</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Fensome</span> <span class="given-names" tagx="given-names" title="given-names">RA</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Fredericq</span> <span class="given-names" tagx="given-names" title="given-names">S</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">James</span> <span class="given-names" tagx="given-names" title="given-names">TY</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Karpov</span> <span class="given-names" tagx="given-names" title="given-names">S</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Kugrens</span> <span class="given-names" tagx="given-names" title="given-names">P</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Krug</span> <span class="given-names" tagx="given-names" title="given-names">J</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Lane</span> <span class="given-names" tagx="given-names" title="given-names">CE</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Lewis</span> <span class="given-names" tagx="given-names" title="given-names">LA</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Lodge</span> <span class="given-names" tagx="given-names" title="given-names">J</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Lynn</span> <span class="given-names" tagx="given-names" title="given-names">DH</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Mann</span> <span class="given-names" tagx="given-names" title="given-names">DG</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Mccourt</span> <span class="given-names" tagx="given-names" title="given-names">RM</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Mendoza</span> <span class="given-names" tagx="given-names" title="given-names">L</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Moestrup</span> <span class="given-names" tagx="given-names" title="given-names">Ø</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Mozley-Standridge</span> <span class="given-names" tagx="given-names" title="given-names">SE</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Nerad</span> <span class="given-names" tagx="given-names" title="given-names">TA</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Shearer</span> <span class="given-names" tagx="given-names" title="given-names">CA</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Smirnov</span> <span class="given-names" tagx="given-names" title="given-names">AV</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Spiegel</span> <span class="given-names" tagx="given-names" title="given-names">FW</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Taylor</span> <span class="given-names" tagx="given-names" title="given-names">MFJR</span> </span> <span class="mixed-article-title" title="mixed-article-title">The new higher level classification of eukaryotes with emphasis on the taxonomy of protists </span> <span class="source" tagx="source" title="source">J Eukaryot Microbiol</span> <span class="year" tagx="year" title="year">2005</span> <span class="volume" tagx="volume" title="volume">5</span> <span class="fpage" tagx="fpage" title="fpage">399</span> <span class="lpage" tagx="lpage" title="lpage">451</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1111/j.1550-7408.2005.00053.x">10.1111/j.1550-7408.2005.00053.x</a> </span> </span> </li> <li tag="ref"> <a name="B15" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Coruzzi</span> <span class="given-names" tagx="given-names" title="given-names">GM</span> </span> <span class="person-group'">Somerville C, Meyerowitz E</span> <span class="mixed-article-title" title="mixed-article-title">Primary N-assimilation into Amino Acids in <i>Arabidopsis</i> </span> <span class="source" tagx="source" title="source">The Arabidopsis Book</span> <span class="year" tagx="year" title="year">2003</span> <span class="publisher-name" tagx="publisher-name" title="publisher-name">Rockville, MD: American Society of Plant Biologists</span> <span class="fpage" tagx="fpage" title="fpage">1</span> <span class="lpage" tagx="lpage" title="lpage">17</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1199/tab.0010">10.1199/tab.0010</a> </span> </span> </li> <li tag="ref"> <a name="B16" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Lam</span> <span class="given-names" tagx="given-names" title="given-names">H-M</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Coschigano</span> <span class="given-names" tagx="given-names" title="given-names">KT</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Oliveira</span> <span class="given-names" tagx="given-names" title="given-names">IC</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Melo-Oliveira</span> <span class="given-names" tagx="given-names" title="given-names">R</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Coruzzi</span> <span class="given-names" tagx="given-names" title="given-names">GM</span> </span> <span class="mixed-article-title" title="mixed-article-title">The molecular-genetics of nitrogen assimilation into amino acids in higher plants</span> <span class="source" tagx="source" title="source">Annu Rev Plant Physiol Mol Biol</span> <span class="year" tagx="year" title="year">1996</span> <span class="volume" tagx="volume" title="volume">47</span> <span class="fpage" tagx="fpage" title="fpage">569</span> <span class="lpage" tagx="lpage" title="lpage">593</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1146/annurev.arplant.47.1.569">10.1146/annurev.arplant.47.1.569</a> </span> </span> </li> <li tag="ref"> <a name="B17" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Sechley</span> <span class="given-names" tagx="given-names" title="given-names">KA</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Yamaya</span> <span class="given-names" tagx="given-names" title="given-names">T</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Oaks</span> <span class="given-names" tagx="given-names" title="given-names">A</span> </span> <span class="mixed-article-title" title="mixed-article-title">Compartmentation of nitrogen assimilation in higher plants</span> <span class="source" tagx="source" title="source">Int Rev Cytol</span> <span class="year" tagx="year" title="year">1992</span> <span class="volume" tagx="volume" title="volume">134</span> <span class="fpage" tagx="fpage" title="fpage">85</span> <span class="lpage" tagx="lpage" title="lpage">163</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1016/S0074-7696(08)62028-8">10.1016/S0074-7696(08)62028-8</a> </span> </span> </li> <li tag="ref"> <a name="B18" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Woodall</span> <span class="given-names" tagx="given-names" title="given-names">J</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Boxall</span> <span class="given-names" tagx="given-names" title="given-names">JG</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Forde</span> <span class="given-names" tagx="given-names" title="given-names">BG</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Pearson</span> <span class="given-names" tagx="given-names" title="given-names">J</span> </span> <span class="mixed-article-title" title="mixed-article-title">Changing perspectives in plant nitrogen metabolism: the central role of glutamine synthetase </span> <span class="source" tagx="source" title="source">Science Progress</span> <span class="year" tagx="year" title="year">1996</span> <span class="volume" tagx="volume" title="volume">79</span> <span class="fpage" tagx="fpage" title="fpage">1</span> <span class="lpage" tagx="lpage" title="lpage">26</span> </span> </li> <li tag="ref"> <a name="B19" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Coruzzi</span> <span class="given-names" tagx="given-names" title="given-names">GM</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Edwards</span> <span class="given-names" tagx="given-names" title="given-names">JW</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Tingey</span> <span class="given-names" tagx="given-names" title="given-names">SV</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Tsai</span> <span class="given-names" tagx="given-names" title="given-names">F-Y</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Walker</span> <span class="given-names" tagx="given-names" title="given-names">EL</span> </span> <span class="person-group'">Goldberg R</span> <span class="mixed-article-title" title="mixed-article-title">Glutamine synthetase: molecular evolution of an eclectic multi-gene family</span> <span class="source" tagx="source" title="source">The Molecular Basis of Plant Development. Proceedings of an E.I. duPont Nermous-UCLA Symposium </span> <span class="year" tagx="year" title="year">1989</span> <span class="publisher-name" tagx="publisher-name" title="publisher-name">New York: Alan R. Liss, Inc</span> <span class="fpage" tagx="fpage" title="fpage">223</span> <span class="lpage" tagx="lpage" title="lpage">232</span> </span> </li> <li tag="ref"> <a name="B20" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Robertson</span> <span class="given-names" tagx="given-names" title="given-names">DL</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Smith</span> <span class="given-names" tagx="given-names" title="given-names">GJ</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Alberte</span> <span class="given-names" tagx="given-names" title="given-names">RS</span> </span> <span class="mixed-article-title" title="mixed-article-title">Characterization of a cDNA encoding glutamine synthetase from the marine diatom <i>Skeletonema costatum </i>(Bacillariophyceae) </span> <span class="source" tagx="source" title="source">J Phycol</span> <span class="year" tagx="year" title="year">1999</span> <span class="volume" tagx="volume" title="volume">35</span> <span class="fpage" tagx="fpage" title="fpage">786</span> <span class="lpage" tagx="lpage" title="lpage">797</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1046/j.1529-8817.1999.3540786.x">10.1046/j.1529-8817.1999.3540786.x</a> </span> </span> </li> <li tag="ref"> <a name="B21" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Ahmad</span> <span class="given-names" tagx="given-names" title="given-names">I</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Hellebust</span> <span class="given-names" tagx="given-names" title="given-names">JA</span> </span> <span class="mixed-article-title" title="mixed-article-title">Nitrogen metabolism of the marine microalga <i>Chlorella autotrophica</i> </span> <span class="source" tagx="source" title="source">Plant Physiol</span> <span class="year" tagx="year" title="year">1984</span> <span class="volume" tagx="volume" title="volume">76</span> <span class="fpage" tagx="fpage" title="fpage">658</span> <span class="lpage" tagx="lpage" title="lpage">663</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1104/pp.76.3.658">10.1104/pp.76.3.658</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/16663902">16663902</a> </span> </span> </li> <li tag="ref"> <a name="B22" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Casselton</span> <span class="given-names" tagx="given-names" title="given-names">PJ</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Chandler</span> <span class="given-names" tagx="given-names" title="given-names">G</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Shah</span> <span class="given-names" tagx="given-names" title="given-names">N</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Stewart</span> <span class="given-names" tagx="given-names" title="given-names">GR</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Sumar</span> <span class="given-names" tagx="given-names" title="given-names">N</span> </span> <span class="mixed-article-title" title="mixed-article-title">Glutamine synthetase isoforms in algae</span> <span class="source" tagx="source" title="source">New Phytol</span> <span class="year" tagx="year" title="year">1986</span> <span class="volume" tagx="volume" title="volume">102</span> <span class="fpage" tagx="fpage" title="fpage">261</span> <span class="lpage" tagx="lpage" title="lpage">270</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1111/j.1469-8137.1986.tb00580.x">10.1111/j.1469-8137.1986.tb00580.x</a> </span> </span> </li> <li tag="ref"> <a name="B23" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Chen</span> <span class="given-names" tagx="given-names" title="given-names">Q</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Silflow</span> <span class="given-names" tagx="given-names" title="given-names">CD</span> </span> <span class="mixed-article-title" title="mixed-article-title">Isolation and characterization of glutamine synthetase genes in <i>Chlamydomonas reinhardtii</i> </span> <span class="source" tagx="source" title="source">Plant Physiol</span> <span class="year" tagx="year" title="year">1996</span> <span class="volume" tagx="volume" title="volume">112</span> <span class="fpage" tagx="fpage" title="fpage">987</span> <span class="lpage" tagx="lpage" title="lpage">996</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1104/pp.112.3.987">10.1104/pp.112.3.987</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/8938407">8938407</a> </span> </span> </li> <li tag="ref"> <a name="B24" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Suarez</span> <span class="given-names" tagx="given-names" title="given-names">MF</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Avila</span> <span class="given-names" tagx="given-names" title="given-names">C</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Gallardo</span> <span class="given-names" tagx="given-names" title="given-names">F</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Canton</span> <span class="given-names" tagx="given-names" title="given-names">FR</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Garcia-Gutierrez</span> <span class="given-names" tagx="given-names" title="given-names">A</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Claros</span> <span class="given-names" tagx="given-names" title="given-names">MG</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Canovas</span> <span class="given-names" tagx="given-names" title="given-names">FM</span> </span> <span class="mixed-article-title" title="mixed-article-title">Molecular and enzymatic analysis of ammonium assimilation in woody plants</span> <span class="source" tagx="source" title="source">J Exp Bot</span> <span class="year" tagx="year" title="year">2002</span> <span class="volume" tagx="volume" title="volume">53</span> <span class="fpage" tagx="fpage" title="fpage">891</span> <span class="lpage" tagx="lpage" title="lpage">904</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1093/jexbot/53.370.891">10.1093/jexbot/53.370.891</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/11912232">11912232</a> </span> </span> </li> <li tag="ref"> <a name="B25" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Bowman</span> <span class="given-names" tagx="given-names" title="given-names">JL</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Floyd</span> <span class="given-names" tagx="given-names" title="given-names">SK</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Sakakibara</span> <span class="given-names" tagx="given-names" title="given-names">K</span> </span> <span class="mixed-article-title" title="mixed-article-title">Green genes - comparative genomics of the green branch of life</span> <span class="source" tagx="source" title="source">Cell</span> <span class="year" tagx="year" title="year">2007</span> <span class="volume" tagx="volume" title="volume">129</span> <span class="fpage" tagx="fpage" title="fpage">229</span> <span class="lpage" tagx="lpage" title="lpage">234</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1016/j.cell.2007.04.004">10.1016/j.cell.2007.04.004</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/17448980">17448980</a> </span> </span> </li> <li tag="ref"> <a name="B26" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Dufresne</span> <span class="given-names" tagx="given-names" title="given-names">A</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Salanoubat</span> <span class="given-names" tagx="given-names" title="given-names">M</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Partensky</span> <span class="given-names" tagx="given-names" title="given-names">Fdr</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Artiguenave</span> <span class="given-names" tagx="given-names" title="given-names">Fo</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Axmann</span> <span class="given-names" tagx="given-names" title="given-names">IM</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Barbe</span> <span class="given-names" tagx="given-names" title="given-names">Vr</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Duprat</span> <span class="given-names" tagx="given-names" title="given-names">S</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Galperin</span> <span class="given-names" tagx="given-names" title="given-names">MY</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Koonin</span> <span class="given-names" tagx="given-names" title="given-names">EV</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Le Gall</span> <span class="given-names" tagx="given-names" title="given-names">F</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Makarova</span> <span class="given-names" tagx="given-names" title="given-names">KS</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Ostrowski</span> <span class="given-names" tagx="given-names" title="given-names">M</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Oztas</span> <span class="given-names" tagx="given-names" title="given-names">S</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Robert</span> <span class="given-names" tagx="given-names" title="given-names">C</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Rogozin</span> <span class="given-names" tagx="given-names" title="given-names">IB</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Scanlan</span> <span class="given-names" tagx="given-names" title="given-names">DJ</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">de Marsac</span> <span class="given-names" tagx="given-names" title="given-names">NT</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Weissenbach</span> <span class="given-names" tagx="given-names" title="given-names">J</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Wincker</span> <span class="given-names" tagx="given-names" title="given-names">P</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Wolf</span> <span class="given-names" tagx="given-names" title="given-names">YI</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Hess</span> <span class="given-names" tagx="given-names" title="given-names">WR</span> </span> <span class="mixed-article-title" title="mixed-article-title">Genome sequence of the cyanobacterium <i>Prochlorococcus marinus </i>SS120, a nearly minimal oxyphototrophic genome </span> <span class="source" tagx="source" title="source">Proc Natl Acad Sci USA</span> <span class="year" tagx="year" title="year">2003</span> <span class="volume" tagx="volume" title="volume">100</span> <span class="fpage" tagx="fpage" title="fpage">10020</span> <span class="lpage" tagx="lpage" title="lpage">10025</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1073/pnas.1733211100">10.1073/pnas.1733211100</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/12917486">12917486</a> </span> </span> </li> <li tag="ref"> <a name="B27" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Palenik</span> <span class="given-names" tagx="given-names" title="given-names">B</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Brahamsha</span> <span class="given-names" tagx="given-names" title="given-names">B</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Larimer</span> <span class="given-names" tagx="given-names" title="given-names">FW</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Land</span> <span class="given-names" tagx="given-names" title="given-names">M</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Hauser</span> <span class="given-names" tagx="given-names" title="given-names">L</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Chain</span> <span class="given-names" tagx="given-names" title="given-names">P</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Lamerdin</span> <span class="given-names" tagx="given-names" title="given-names">J</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Regala</span> <span class="given-names" tagx="given-names" title="given-names">W</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Allen</span> <span class="given-names" tagx="given-names" title="given-names">EE</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">McCarren</span> <span class="given-names" tagx="given-names" title="given-names">J</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Paulsen</span> <span class="given-names" tagx="given-names" title="given-names">I</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Dufresne</span> <span class="given-names" tagx="given-names" title="given-names">A</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Partensky</span> <span class="given-names" tagx="given-names" title="given-names">F</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Webb</span> <span class="given-names" tagx="given-names" title="given-names">EA</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Waterbury</span> <span class="given-names" tagx="given-names" title="given-names">J</span> </span> <span class="mixed-article-title" title="mixed-article-title">The genome of a motile marine <i>Synechococcus</i> </span> <span class="source" tagx="source" title="source">Nature</span> <span class="year" tagx="year" title="year">2003</span> <span class="volume" tagx="volume" title="volume">424</span> <span class="fpage" tagx="fpage" title="fpage">1037</span> <span class="lpage" tagx="lpage" title="lpage">1042</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1038/nature01943">10.1038/nature01943</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/12917641">12917641</a> </span> </span> </li> <li tag="ref"> <a name="B28" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Drummond</span> <span class="given-names" tagx="given-names" title="given-names">AJ</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Rambaut</span> <span class="given-names" tagx="given-names" title="given-names">A</span> </span> <span class="mixed-article-title" title="mixed-article-title">BEAST: Bayesian evolutionary analysis by sampling trees</span> <span class="source" tagx="source" title="source">BMC Evol Biol</span> <span class="year" tagx="year" title="year">2007</span> <span class="volume" tagx="volume" title="volume">7</span> <span class="fpage" tagx="fpage" title="fpage">214</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1186/1471-2148-7-214">10.1186/1471-2148-7-214</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/17996036">17996036</a> </span> </span> </li> <li tag="ref"> <a name="B29" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Matsuzaki</span> <span class="given-names" tagx="given-names" title="given-names">M</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Misumi</span> <span class="given-names" tagx="given-names" title="given-names">O</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Shin-i</span> <span class="given-names" tagx="given-names" title="given-names">T</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Maruyama</span> <span class="given-names" tagx="given-names" title="given-names">S</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Takahara</span> <span class="given-names" tagx="given-names" title="given-names">M</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Miyagishima</span> <span class="given-names" tagx="given-names" title="given-names">S-y</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Mori</span> <span class="given-names" tagx="given-names" title="given-names">T</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Nishida</span> <span class="given-names" tagx="given-names" title="given-names">K</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Yagisawa</span> <span class="given-names" tagx="given-names" title="given-names">F</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Nishida</span> <span class="given-names" tagx="given-names" title="given-names">K</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Yoshida</span> <span class="given-names" tagx="given-names" title="given-names">Y</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Nishimura</span> <span class="given-names" tagx="given-names" title="given-names">Y</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Nakao</span> <span class="given-names" tagx="given-names" title="given-names">S</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Kobayashi</span> <span class="given-names" tagx="given-names" title="given-names">T</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Momoyama</span> <span class="given-names" tagx="given-names" title="given-names">Y</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Higashiyama</span> <span class="given-names" tagx="given-names" title="given-names">T</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Minoda</span> <span class="given-names" tagx="given-names" title="given-names">A</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Sano</span> <span class="given-names" tagx="given-names" title="given-names">M</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Nomoto</span> <span class="given-names" tagx="given-names" title="given-names">H</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Oishi</span> <span class="given-names" tagx="given-names" title="given-names">K</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Hayashi</span> <span class="given-names" tagx="given-names" title="given-names">H</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Ohta</span> <span class="given-names" tagx="given-names" title="given-names">F</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Nishizaka</span> <span class="given-names" tagx="given-names" title="given-names">S</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Haga</span> <span class="given-names" tagx="given-names" title="given-names">S</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Miura</span> <span class="given-names" tagx="given-names" title="given-names">S</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Morishita</span> <span class="given-names" tagx="given-names" title="given-names">T</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Kabeya</span> <span class="given-names" tagx="given-names" title="given-names">Y</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Terasawa</span> <span class="given-names" tagx="given-names" title="given-names">K</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Suzuki</span> <span class="given-names" tagx="given-names" title="given-names">Y</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Ishii</span> <span class="given-names" tagx="given-names" title="given-names">Y</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span> <span class="mixed-article-title" title="mixed-article-title">Genome sequence of the ultrasmall unicellular red alga <i>Cyanidioschyzon merolae </i>10D </span> <span class="source" tagx="source" title="source">Nature</span> <span class="year" tagx="year" title="year">2004</span> <span class="volume" tagx="volume" title="volume">428</span> <span class="fpage" tagx="fpage" title="fpage">653</span> <span class="lpage" tagx="lpage" title="lpage">657</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1038/nature02398">10.1038/nature02398</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/15071595">15071595</a> </span> </span> </li> <li tag="ref"> <a name="B30" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Barbier</span> <span class="given-names" tagx="given-names" title="given-names">G</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Oesterhelt</span> <span class="given-names" tagx="given-names" title="given-names">C</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Larson</span> <span class="given-names" tagx="given-names" title="given-names">MD</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Halgren</span> <span class="given-names" tagx="given-names" title="given-names">RG</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Wilkerson</span> <span class="given-names" tagx="given-names" title="given-names">C</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Garavito</span> <span class="given-names" tagx="given-names" title="given-names">RM</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Benning</span> <span class="given-names" tagx="given-names" title="given-names">C</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Weber</span> <span class="given-names" tagx="given-names" title="given-names">APM</span> </span> <span class="mixed-article-title" title="mixed-article-title">Comparative genomics of two closely related unicellular thermo-acidophilic red algae, <i>Galdieria sulphuraria </i>and <i>Cyanidioschyzon merolae</i>, reveals the molecular basis of the metabolic flexibility of <i>Galdieria sulphuraria </i>and significant differences in carbohydrate metabolism of both algae </span> <span class="source" tagx="source" title="source">Plant Physiol</span> <span class="year" tagx="year" title="year">2005</span> <span class="volume" tagx="volume" title="volume">137</span> <span class="fpage" tagx="fpage" title="fpage">460</span> <span class="lpage" tagx="lpage" title="lpage">474</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1104/pp.104.051169">10.1104/pp.104.051169</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/15710685">15710685</a> </span> </span> </li> <li tag="ref"> <a name="B31" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Weber</span> <span class="given-names" tagx="given-names" title="given-names">A</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Oesterhelt</span> <span class="given-names" tagx="given-names" title="given-names">C</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Gross</span> <span class="given-names" tagx="given-names" title="given-names">W</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Bräutigam</span> <span class="given-names" tagx="given-names" title="given-names">A</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Imboden</span> <span class="given-names" tagx="given-names" title="given-names">L</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Krassovskaya</span> <span class="given-names" tagx="given-names" title="given-names">I</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Linka</span> <span class="given-names" tagx="given-names" title="given-names">N</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Truchina</span> <span class="given-names" tagx="given-names" title="given-names">J</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Schneidereit</span> <span class="given-names" tagx="given-names" title="given-names">J</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Voll</span> <span class="given-names" tagx="given-names" title="given-names">H</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Voll</span> <span class="given-names" tagx="given-names" title="given-names">L</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Zimmermann</span> <span class="given-names" tagx="given-names" title="given-names">M</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Jamai</span> <span class="given-names" tagx="given-names" title="given-names">A</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Riekhof</span> <span class="given-names" tagx="given-names" title="given-names">W</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Yu</span> <span class="given-names" tagx="given-names" title="given-names">B</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Garavito</span> <span class="given-names" tagx="given-names" title="given-names">R</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Benning</span> <span class="given-names" tagx="given-names" title="given-names">C</span> </span> <span class="mixed-article-title" title="mixed-article-title">EST-analysis of the thermo-acidophilic red microalga <i>Galdieria sulphuraria </i>reveals potential for lipid A biosynthesis and unveils the pathway of carbon export from rhodoplasts </span> <span class="source" tagx="source" title="source">Plant Mol Biol</span> <span class="year" tagx="year" title="year">2004</span> <span class="volume" tagx="volume" title="volume">55</span> <span class="fpage" tagx="fpage" title="fpage">17</span> <span class="lpage" tagx="lpage" title="lpage">32</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1007/s11103-004-0376-y">10.1007/s11103-004-0376-y</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/15604662">15604662</a> </span> </span> </li> <li tag="ref"> <a name="B32" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Weng</span> <span class="given-names" tagx="given-names" title="given-names">J-K</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Tanurdzic</span> <span class="given-names" tagx="given-names" title="given-names">M</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Chapple</span> <span class="given-names" tagx="given-names" title="given-names">C</span> </span> <span class="mixed-article-title" title="mixed-article-title">Functional analysis and comparative genomics of expressed sequence tags from the lycophyte <i>Selaginella moellendorffii</i> </span> <span class="source" tagx="source" title="source">BMC Genomics</span> <span class="year" tagx="year" title="year">2005</span> <span class="volume" tagx="volume" title="volume">6</span> <span class="fpage" tagx="fpage" title="fpage">85</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1186/1471-2164-6-85">10.1186/1471-2164-6-85</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/15938755">15938755</a> </span> </span> </li> <li tag="ref"> <a name="B33" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Emanuelsson</span> <span class="given-names" tagx="given-names" title="given-names">O</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Nielsen</span> <span class="given-names" tagx="given-names" title="given-names">H</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Brunak</span> <span class="given-names" tagx="given-names" title="given-names">S</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">von Heijne</span> <span class="given-names" tagx="given-names" title="given-names">G</span> </span> <span class="mixed-article-title" title="mixed-article-title">Predicting subcellular localization of proteins based on their N-terminal amino acid sequence </span> <span class="source" tagx="source" title="source">J Mol Biol</span> <span class="year" tagx="year" title="year">2000</span> <span class="volume" tagx="volume" title="volume">300</span> <span class="fpage" tagx="fpage" title="fpage">1005</span> <span class="lpage" tagx="lpage" title="lpage">1016</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1006/jmbi.2000.3903">10.1006/jmbi.2000.3903</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/10891285">10891285</a> </span> </span> </li> <li tag="ref"> <a name="B34" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Franzén</span> <span class="given-names" tagx="given-names" title="given-names">LG</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Rochaix</span> <span class="given-names" tagx="given-names" title="given-names">JD</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">von Heijne</span> <span class="given-names" tagx="given-names" title="given-names">G</span> </span> <span class="mixed-article-title" title="mixed-article-title">Chloroplast transit peptides from the green alga <i>Chlamydomonas reinhardtii </i>share features with both mitochondrial and higher plant chloroplast presequences </span> <span class="source" tagx="source" title="source">FEBS Lett</span> <span class="year" tagx="year" title="year">1990</span> <span class="volume" tagx="volume" title="volume">260</span> <span class="fpage" tagx="fpage" title="fpage">165</span> <span class="lpage" tagx="lpage" title="lpage">168</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1016/0014-5793(90)80094-Y">10.1016/0014-5793(90)80094-Y</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/2404796">2404796</a> </span> </span> </li> <li tag="ref"> <a name="B35" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Taira</span> <span class="given-names" tagx="given-names" title="given-names">M</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Valtersson</span> <span class="given-names" tagx="given-names" title="given-names">U</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Burkhardt</span> <span class="given-names" tagx="given-names" title="given-names">B</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Ludwig</span> <span class="given-names" tagx="given-names" title="given-names">RA</span> </span> <span class="mixed-article-title" title="mixed-article-title"> <i>Arabidopsis thaliana </i>GLN2-encoded glutamine synthetase is dual targeted to leaf mitochondria and chloroplasts </span> <span class="source" tagx="source" title="source">Plant Cell</span> <span class="year" tagx="year" title="year">2004</span> <span class="volume" tagx="volume" title="volume">16</span> <span class="fpage" tagx="fpage" title="fpage">2048</span> <span class="lpage" tagx="lpage" title="lpage">2058</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1105/tpc.104.022046">10.1105/tpc.104.022046</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/15273293">15273293</a> </span> </span> </li> <li tag="ref"> <a name="B36" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Linka</span> <span class="given-names" tagx="given-names" title="given-names">M</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Weber</span> <span class="given-names" tagx="given-names" title="given-names">APM</span> </span> <span class="mixed-article-title" title="mixed-article-title">Shuffling ammonia between mitochondria and plastids during photorespiration</span> <span class="source" tagx="source" title="source">Trends Plant Sci</span> <span class="year" tagx="year" title="year">2005</span> <span class="volume" tagx="volume" title="volume">10</span> <span class="fpage" tagx="fpage" title="fpage">461</span> <span class="lpage" tagx="lpage" title="lpage">465</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1016/j.tplants.2005.08.002">10.1016/j.tplants.2005.08.002</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/16143558">16143558</a> </span> </span> </li> <li tag="ref"> <a name="B37" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Vallon</span> <span class="given-names" tagx="given-names" title="given-names">O</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Spalding</span> <span class="given-names" tagx="given-names" title="given-names">M</span> </span> <span class="person-group'">Stern D, Harris EH</span> <span class="mixed-article-title" title="mixed-article-title">Amino Acid Metabolism</span> <span class="source" tagx="source" title="source">The Chlamydomonas Source Book</span> <span class="year" tagx="year" title="year">2008</span> <span class="volume" tagx="volume" title="volume">2</span> <span class="edition" tagx="edition" title="edition">2</span> <span class="publisher-name" tagx="publisher-name" title="publisher-name">Boston: Elsevier</span> <span class="fpage" tagx="fpage" title="fpage">115</span> <span class="lpage" tagx="lpage" title="lpage">150</span> </span> </li> <li tag="ref"> <a name="B38" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Doyle</span> <span class="given-names" tagx="given-names" title="given-names">JJ</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Doyle</span> <span class="given-names" tagx="given-names" title="given-names">JL</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Harbison</span> <span class="given-names" tagx="given-names" title="given-names">C</span> </span> <span class="mixed-article-title" title="mixed-article-title">Chloroplast-expressed glutamine synthetase in <i>Glycine </i>and related Leguminosae: Phylogeny, gene duplication, and ancient polyploidy </span> <span class="source" tagx="source" title="source">Systematic Botany</span> <span class="year" tagx="year" title="year">2003</span> <span class="volume" tagx="volume" title="volume">28</span> <span class="fpage" tagx="fpage" title="fpage">567</span> <span class="lpage" tagx="lpage" title="lpage">577</span> </span> </li> <li tag="ref"> <a name="B39" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Walker</span> <span class="given-names" tagx="given-names" title="given-names">EL</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Weeden</span> <span class="given-names" tagx="given-names" title="given-names">NF</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Taylor</span> <span class="given-names" tagx="given-names" title="given-names">CB</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Green</span> <span class="given-names" tagx="given-names" title="given-names">P</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Coruzzi</span> <span class="given-names" tagx="given-names" title="given-names">GM</span> </span> <span class="mixed-article-title" title="mixed-article-title">Molecular evolution of duplicate copies of genes encoding cytosolic glutamine synthetase in <i>Pisum sativum</i> </span> <span class="source" tagx="source" title="source">Plant Mol Biol</span> <span class="year" tagx="year" title="year">1995</span> <span class="volume" tagx="volume" title="volume">29</span> <span class="fpage" tagx="fpage" title="fpage">1111</span> <span class="lpage" tagx="lpage" title="lpage">1125</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1007/BF00020456">10.1007/BF00020456</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/8616212">8616212</a> </span> </span> </li> <li tag="ref"> <a name="B40" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Keeling</span> <span class="given-names" tagx="given-names" title="given-names">PJ</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Palmer</span> <span class="given-names" tagx="given-names" title="given-names">JD</span> </span> <span class="mixed-article-title" title="mixed-article-title">Horizontal gene transfer in eukaryotic evolution</span> <span class="source" tagx="source" title="source">Nat Rev Genet</span> <span class="year" tagx="year" title="year">2008</span> <span class="volume" tagx="volume" title="volume">9</span> <span class="fpage" tagx="fpage" title="fpage">605</span> <span class="lpage" tagx="lpage" title="lpage">618</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1038/nrg2386">10.1038/nrg2386</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/18591983">18591983</a> </span> </span> </li> <li tag="ref"> <a name="B41" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Richards</span> <span class="given-names" tagx="given-names" title="given-names">TA</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Soanes</span> <span class="given-names" tagx="given-names" title="given-names">DM</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Foster</span> <span class="given-names" tagx="given-names" title="given-names">PG</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Leonard</span> <span class="given-names" tagx="given-names" title="given-names">G</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Thornton</span> <span class="given-names" tagx="given-names" title="given-names">CR</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Talbot</span> <span class="given-names" tagx="given-names" title="given-names">NJ</span> </span> <span class="mixed-article-title" title="mixed-article-title">Phylogenomic analysis demonstrates a pattern of rare and ancient horizontal gene transfer between plants and fungi </span> <span class="source" tagx="source" title="source">Plant Cell</span> <span class="year" tagx="year" title="year">2009</span> <span class="volume" tagx="volume" title="volume">21</span> <span class="fpage" tagx="fpage" title="fpage">1897</span> <span class="lpage" tagx="lpage" title="lpage">1911</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1105/tpc.109.065805">10.1105/tpc.109.065805</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/19584142">19584142</a> </span> </span> </li> <li tag="ref"> <a name="B42" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Florencio</span> <span class="given-names" tagx="given-names" title="given-names">FJ</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Vega</span> <span class="given-names" tagx="given-names" title="given-names">JM</span> </span> <span class="mixed-article-title" title="mixed-article-title">Separation, purification, and characterization of two isoforms of glutamine synthetase from <i>Chlamydomonas reinhardii</i> </span> <span class="source" tagx="source" title="source">Z Naturforsch</span> <span class="year" tagx="year" title="year">1983</span> <span class="volume" tagx="volume" title="volume">38c</span> <span class="fpage" tagx="fpage" title="fpage">531</span> <span class="lpage" tagx="lpage" title="lpage">538</span> </span> </li> <li tag="ref"> <a name="B43" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Coyer</span> <span class="given-names" tagx="given-names" title="given-names">JA</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Robertson</span> <span class="given-names" tagx="given-names" title="given-names">DL</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Alberte</span> <span class="given-names" tagx="given-names" title="given-names">RS</span> </span> <span class="mixed-article-title" title="mixed-article-title">Genetic variability within a population and between diploid/haploid tissue of <i>Macrocystis pyrifera </i>(Phaeophyceae) </span> <span class="source" tagx="source" title="source">J Phycol</span> <span class="year" tagx="year" title="year">1994</span> <span class="volume" tagx="volume" title="volume">30</span> <span class="fpage" tagx="fpage" title="fpage">545</span> <span class="lpage" tagx="lpage" title="lpage">542</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1111/j.0022-3646.1994.00545.x">10.1111/j.0022-3646.1994.00545.x</a> </span> </span> </li> <li tag="ref"> <a name="B44" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Brown</span> <span class="given-names" tagx="given-names" title="given-names">KL</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Twing</span> <span class="given-names" tagx="given-names" title="given-names">KI</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Robertson</span> <span class="given-names" tagx="given-names" title="given-names">DL</span> </span> <span class="mixed-article-title" title="mixed-article-title">Unraveling the regulation of nitrogen assimilation in the marine diatom <i>Thalassiosira pseudonana </i>(BACILLARIOPHYCEAE): Diurnal variations in transcript levels for five genes involved in nitrogen assimilation </span> <span class="source" tagx="source" title="source">J Phycol</span> <span class="year" tagx="year" title="year">2009</span> <span class="volume" tagx="volume" title="volume">45</span> <span class="fpage" tagx="fpage" title="fpage">413</span> <span class="lpage" tagx="lpage" title="lpage">426</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1111/j.1529-8817.2009.00648.x">10.1111/j.1529-8817.2009.00648.x</a> </span> </span> </li> <li tag="ref"> <a name="B45" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Thompson</span> <span class="given-names" tagx="given-names" title="given-names">JD</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Higgins</span> <span class="given-names" tagx="given-names" title="given-names">DG</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Gibson</span> <span class="given-names" tagx="given-names" title="given-names">TJ</span> </span> <span class="mixed-article-title" title="mixed-article-title">Clustal W: Improving the sensitivity of progressive multiple sequence alignment through sequence weighting, positions-specific gap penalties and weight matrix choice </span> <span class="source" tagx="source" title="source">Nucleic Acid Res</span> <span class="year" tagx="year" title="year">1994</span> <span class="volume" tagx="volume" title="volume">22</span> <span class="fpage" tagx="fpage" title="fpage">4673</span> <span class="lpage" tagx="lpage" title="lpage">4680</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1093/nar/22.22.4673">10.1093/nar/22.22.4673</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/7984417">7984417</a> </span> </span> </li> <li tag="ref"> <a name="B46" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Hall</span> <span class="given-names" tagx="given-names" title="given-names">TA</span> </span> <span class="mixed-article-title" title="mixed-article-title">BioEdit: A user-friendly biological sequence alignment editor and analysis program for Windows 95/98/NT </span> <span class="source" tagx="source" title="source">Nucl Acids Symp Ser</span> <span class="year" tagx="year" title="year">1999</span> <span class="volume" tagx="volume" title="volume">41</span> <span class="fpage" tagx="fpage" title="fpage">95</span> <span class="lpage" tagx="lpage" title="lpage">98</span> </span> </li> <li tag="ref"> <a name="B47" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Maddison</span> <span class="given-names" tagx="given-names" title="given-names">WP</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Maddison</span> <span class="given-names" tagx="given-names" title="given-names">DR</span> </span> <span class="source" tagx="source" title="source"> <i>MacClade 4: Analysis of phylogeny and character evolution</i> </span> <span class="year" tagx="year" title="year">2000</span> <span class="publisher-name" tagx="publisher-name" title="publisher-name">Sunderland, MA: Sinauer Associates</span> </span> </li> <li tag="ref"> <a name="B48" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Huelsenbeck</span> <span class="given-names" tagx="given-names" title="given-names">J</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Ronquist</span> <span class="given-names" tagx="given-names" title="given-names">F</span> </span> <span class="mixed-article-title" title="mixed-article-title">MRBAYES: Bayesian inference of phylogeny</span> <span class="source" tagx="source" title="source">Bioinformatics</span> <span class="year" tagx="year" title="year">2001</span> <span class="volume" tagx="volume" title="volume">17</span> <span class="fpage" tagx="fpage" title="fpage">754</span> <span class="lpage" tagx="lpage" title="lpage">755</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1093/bioinformatics/17.8.754">10.1093/bioinformatics/17.8.754</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/11524383">11524383</a> </span> </span> </li> <li tag="ref"> <a name="B49" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Ronquist</span> <span class="given-names" tagx="given-names" title="given-names">F</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Huelsenbeck</span> <span class="given-names" tagx="given-names" title="given-names">JP</span> </span> <span class="mixed-article-title" title="mixed-article-title">MRBAYES 3: Bayesian phylogenetic inference under mixed models</span> <span class="source" tagx="source" title="source">Bioinformatics</span> <span class="year" tagx="year" title="year">2003</span> <span class="volume" tagx="volume" title="volume">19</span> <span class="fpage" tagx="fpage" title="fpage">1572</span> <span class="lpage" tagx="lpage" title="lpage">1574</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1093/bioinformatics/btg180">10.1093/bioinformatics/btg180</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/12912839">12912839</a> </span> </span> </li> <li tag="ref"> <a name="B50" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Stamatakis</span> <span class="given-names" tagx="given-names" title="given-names">A</span> </span> <span class="mixed-article-title" title="mixed-article-title">RAxML-VI-HPC: Maximum Likelihood-based phylogenetic analyses with thousands of taxa and mixed models </span> <span class="source" tagx="source" title="source">Bioinformatics</span> <span class="year" tagx="year" title="year">2006</span> <span class="volume" tagx="volume" title="volume">22</span> <span class="fpage" tagx="fpage" title="fpage">2688</span> <span class="lpage" tagx="lpage" title="lpage">2690</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1093/bioinformatics/btl446">10.1093/bioinformatics/btl446</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/16928733">16928733</a> </span> </span> </li> <li tag="ref"> <a name="B51" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Stamatakis</span> <span class="given-names" tagx="given-names" title="given-names">A</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Hoover</span> <span class="given-names" tagx="given-names" title="given-names">P</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Rougemont</span> <span class="given-names" tagx="given-names" title="given-names">J</span> </span> <span class="mixed-article-title" title="mixed-article-title">A rapid bootstrap algorithm for the RAxML web servers</span> <span class="source" tagx="source" title="source">Syst Biol</span> <span class="year" tagx="year" title="year">2008</span> <span class="volume" tagx="volume" title="volume">57</span> <span class="fpage" tagx="fpage" title="fpage">758</span> <span class="lpage" tagx="lpage" title="lpage">771</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1080/10635150802429642">10.1080/10635150802429642</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/18853362">18853362</a> </span> </span> </li> <li tag="ref"> <a name="B52" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Swofford</span> <span class="given-names" tagx="given-names" title="given-names">DL</span> </span> <span class="source" tagx="source" title="source">PAUP: Phylogenetic analysis using parsimony, beta version 4.0b1</span> <span class="year" tagx="year" title="year">1998</span> <span class="publisher-name" tagx="publisher-name" title="publisher-name">Sinauer Associates, Inc., Sunderland</span> </span> </li> <li tag="ref"> <a name="B53" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Emanuelsson</span> <span class="given-names" tagx="given-names" title="given-names">O</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Nielsen</span> <span class="given-names" tagx="given-names" title="given-names">H</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">von Heijne</span> <span class="given-names" tagx="given-names" title="given-names">G</span> </span> <span class="mixed-article-title" title="mixed-article-title">ChloroP, a neural network-based method for predicting chloroplast transit peptides and their cleavage sites </span> <span class="source" tagx="source" title="source">Prot Sci</span> <span class="year" tagx="year" title="year">1999</span> <span class="volume" tagx="volume" title="volume">8</span> <span class="fpage" tagx="fpage" title="fpage">978</span> <span class="lpage" tagx="lpage" title="lpage">984</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1110/ps.8.5.978">10.1110/ps.8.5.978</a> </span> </span> </li> <li tag="ref"> <a name="B54" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Taylor</span> <span class="given-names" tagx="given-names" title="given-names">TN</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Hass</span> <span class="given-names" tagx="given-names" title="given-names">H</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Herp</span> <span class="given-names" tagx="given-names" title="given-names">H</span> </span> <span class="mixed-article-title" title="mixed-article-title">The oldest fossil ascomycetes</span> <span class="source" tagx="source" title="source">Nature</span> <span class="year" tagx="year" title="year">1999</span> <span class="volume" tagx="volume" title="volume">399</span> <span class="fpage" tagx="fpage" title="fpage">648</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1038/21349">10.1038/21349</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/10385115">10385115</a> </span> </span> </li> <li tag="ref"> <a name="B55" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Valentine</span> <span class="given-names" tagx="given-names" title="given-names">JW</span> </span> <span class="person-group'">Lipps JH, Signor PW</span> <span class="mixed-article-title" title="mixed-article-title">The macroevolution of phyla</span> <span class="source" tagx="source" title="source">Origin and Early Evolution of the Metazoa</span> <span class="year" tagx="year" title="year">1992</span> <span class="publisher-name" tagx="publisher-name" title="publisher-name">New York: Plenum Press</span> <span class="fpage" tagx="fpage" title="fpage">525</span> <span class="lpage" tagx="lpage" title="lpage">553</span> </span> </li> <li tag="ref"> <a name="B56" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Gray</span> <span class="given-names" tagx="given-names" title="given-names">J</span> </span> <span class="mixed-article-title" title="mixed-article-title">Major Paleozoic land plant evolutionary bio-events</span> <span class="source" tagx="source" title="source">Palaeogeogr Palaeoclimatol Palaeoecol</span> <span class="year" tagx="year" title="year">1993</span> <span class="volume" tagx="volume" title="volume">104</span> <span class="fpage" tagx="fpage" title="fpage">153</span> <span class="lpage" tagx="lpage" title="lpage">169</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1016/0031-0182(93)90127-5">10.1016/0031-0182(93)90127-5</a> </span> </span> </li> <li tag="ref"> <a name="B57" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Rambaut</span> <span class="given-names" tagx="given-names" title="given-names">A</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Drummond</span> <span class="given-names" tagx="given-names" title="given-names">A</span> </span> <span class="mixed-article-title" title="mixed-article-title">Tracer v1.4</span> <span class="year" tagx="year" title="year">2007</span> <a href="http://beast.bio.ed.ac.uk/Tracer">http://beast.bio.ed.ac.uk/Tracer</a> </span> </li> <li tag="ref"> <a name="B58" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Rambaut</span> <span class="given-names" tagx="given-names" title="given-names">A</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Drummond</span> <span class="given-names" tagx="given-names" title="given-names">AJ</span> </span> <span class="mixed-article-title" title="mixed-article-title">TreeAnnotator 1.4.5</span> <span class="year" tagx="year" title="year">2006</span> <a href="http://beast.bio.ed.ac.uk/TreeAnnotator">http://beast.bio.ed.ac.uk/TreeAnnotator</a> </span> </li> </ul> </div> </div>  </body></html>