<html xmlns="http://www.w3.org/1999/xhtml"> <head> <style type="text/css">
      	     		 
      	    a                          {background : #ffffff; }
      	    article                    {border-style : dotted; border-width : 2px; }
		 	
		 	div                        {background : #ffffcc;}
		 	div.abstract-title         {font-weight : bold ; font-size : 16pt;}
		 	div.ack                    {border-style : solid ; border-color : red; margin : 2em; }
		 	div.article-meta           {border-style : solid; border-width : 1 pt; margin 1em;}
		 	div.boxed-text             {margin : 5em; border-style : solid;}
		 	div.contrib-group          {margin : 1em; }
		 	div.fig                    {border-style : solid; border-width : 2px; margin : 2em; }
		 	div.funding                {font-weight : bold ; font-size : 16pt;}
		 	div.given-names            {font-style : italic;}
		 	div.intro                  {border-style : inset; margin : 5px;}
		 	div.introduction           {border-style : inset; margin : 5px;}
		 	div.journal-meta           {border-style : solid; border-width : 1 pt; margin 1em;}
		 	div.journal-title-group    {background : #ffeeee;}
		 	div.article-title          {font-weight : bold ; font-size : 18pt;}
		 	div.kwd                    {font-style : italic}
		 	div.meta-name              {font-weight : bold ; font-size : 16pt;}
		 	div.name                   {font-weight : bold;}
		 	div.alt-title              {font-style : italic; font-size : 12px;}
		 	
		 	div.sec                    {border : 2px; margin 5px; padding 2px;}
		 	div.title                  {font-family : courier; font-weight : bold;
		 	                            font-size : 16pt; margin : 5px;}
		 	
		 	div.materials_methods      {border-style : double; margin : 5px; }
		 	div.methods                {border-style : double; margin : 5px; }
		 	
		 	div.results                {border-style : solid; margin : 5px;}
		 	div.background             {border-style : dotted; margin : 5px;}
		 	
		 	div.discussion             {border-style : groove; margin : 5px;}
		 	div.conclusion             {border-style : ridge; margin : 5px;}
		 	div.conclusions             {border-style : ridge; margin : 5px;}
		 	div.supplementary-material {border-style : inset; margin : 5px;}
		 	div.abbreviations          {border-style : double; border-color : red; margin : 5px;}
		 	div.competinginterests     {border-style : double; border-color : blue; margin : 5px;}
		 	div.acknowledgements       {border-style : double; border-color : green; margin : 5px;}
		 	div.authors_contributions   {border-style : double; border-color : purple; margin : 5px;}
		 	
		 	div.publisher              {border-style : outset; margin : 5px;}
		 	div.fn-type-conflict       {background : #f88; }
		 	div.fn-type-con            {background : #ddf; }
		 	div.fn-type-other          {background : #ddd; }
		 	
		 	div.unknown                {background : #ffd;
									 	  border-style : solid;
									 	  border-width : 1px;
									 	  padding : 2 px;}
		 	  
      	    table                      {background : #ffffdd;}
		 	tr                         {background : #ddddff; padding : 1px;}
		 	
		 	span                       {background : #ffcccc;}
		 	
		 	span.citation-author       {font-family : helvetica; background : #ffeeee;}
		 	span.collab                {background : #ddffff; }
		 	span.comment               {font-family : courier; font-size : 6px; background : #ffaaff;}
		 	span.contrib               {background : #ffffff;}
		 	span.corresp               {background : #ddffdd; }
		 	span.doi                   {background : #ffffff;}
		 	span.email                 {font-family : courier; }
		 	span.etal                  {font-style : italic;}
		 	span.fpage                 {font-family : courier;}
		 	span.given-names           {background : #ffffff;}
		 	span.iso-abbrev            {background : #ffffff;}
		 	span.issn-epub             {background : #ffffff;}
		 	span.issn-ppub             {background : #ffffff;}
		 	span.journal-title         {background : #ffffff;}
		 	span.lpage                 {font-family : courier;}
		 	span.mixed-article-title   {font-style : italic ;}
		 	span.nlm-ta                {background : #ffffff;}
		 	span.pmc                   {background : #ffffff;}
		 	span.pmcid                 {background : #ffffff;}
		 	span.pmid                  {background : #ffffff;}
		 	span.publisher             {background : #ffffff;}
		 	span.publisher-id          {background : #ffffff;}
		 	span.publisher-name        {background : #ffffff;}
		 	span.source                {background : #ffffff;}
		 	span.subject               {background : #ffffff;}
		 	span.surname               {background : #ffffff;}
		 	span.volume                {font-family : courier; font-weight : bold;}
		 	span.year                  {font-family : courier ; font-style : italic;}
			</style> </head> <body> <div class="front" title="front"> <div class="journal-meta" tagx="journal-meta" title="journal-meta"><span class="nlm-ta" title="nlm-ta">PLoS Genet</span><span class="iso-abbrev" title="iso-abbrev">PLoS Genet</span><span class="publisher-id" title="publisher-id">plos</span><span class="pmc" title="pmc">plosgen</span><div class="journal-title-group" tagx="journal-title-group" title="journal-title-group"><span class="journal-title" tagx="journal-title" title="journal-title">PLoS Genetics</span></div><span class="issn-ppub" tagx="issn" title="issn-ppub">1553-7390</span><span class="issn-epub" tagx="issn" title="issn-epub">1553-7404</span><div class="publisher" tagx="publisher" title="publisher"><span class="publisher-name" tagx="publisher-name" title="publisher-name">Public Library of Science</span><span class="publisher-loc" tagx="publisher-loc" title="publisher-loc">San Francisco, USA</span></div> </div> <div class="article-meta" tagx="article-meta" title="article-meta"><span class="pmid" title="pmid"> pmid: <a href="http://www.ncbi.nlm.nih.gov/pubmed/19730686">19730686</a></span><span class="publisher-id" title="publisher-id">09-PLGE-RA-0854R2</span><span class="doi" title="doi"> doi: <a href="https://dx.doi.org/10.1371/journal.pgen.1000634">10.1371/journal.pgen.1000634</a></span><div class="article-categories" title="article-categories">
: <span class="subject" title="subject">Research Article</span>: <span class="subject" title="subject">Evolutionary Biology</span><span class="subject" title="subject">Evolutionary Biology/Animal Genetics</span><span class="subject" title="subject">Evolutionary Biology/Evolutionary and Comparative Genetics</span><span class="subject" title="subject">Evolutionary Biology/Genomics</span><span class="subject" title="subject">Genetics and Genomics/Animal Genetics</span><span class="subject" title="subject">Genetics and Genomics/Comparative Genomics</span></div> <div class="title-group" tagx="title-group" title="title-group"> <div class="article-title" title="article-title">
Molecular Decay of the Tooth Gene Enamelin (<i>ENAM</i>) Mirrors the Loss of Enamel in the Fossil Record of Placental Mammals</div> <div class="alt-title" title="alt-title">
Enamelin Evolution in Placental Mammals</div> </div> <div class="contrib-group" title="contrib-group"><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Meredith</span><span class="given-names" tagx="given-names" title="given-names">Robert W.</span></span><a href="#aff1"><sup>1</sup></a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gatesy</span><span class="given-names" tagx="given-names" title="given-names">John</span></span><a href="#aff1"><sup>1</sup></a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Murphy</span><span class="given-names" tagx="given-names" title="given-names">William J.</span></span><a href="#aff2"><sup>2</sup></a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ryder</span><span class="given-names" tagx="given-names" title="given-names">Oliver A.</span></span><a href="#aff3"><sup>3</sup></a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Springer</span><span class="given-names" tagx="given-names" title="given-names">Mark S.</span></span><a href="#aff1"><sup>1</sup></a><a href="#cor1"><sup>*</sup></a></span></div><span class="citation_author_institution" id="aff1">[1], <span class="addr-line" title="addr-line">Department of Biology, University of California Riverside, Riverside, California, United States of America</span></span><span class="citation_author_institution" id="aff2">[2], <span class="addr-line" title="addr-line">Department of Veterinary Integrative Biosciences, Texas A&amp;amp;M University, College Station, Texas, United States of America</span></span><span class="citation_author_institution" id="aff3">[3], <span class="addr-line" title="addr-line">San Diego Zoo's Institute for Conservation Research, Escondido, California, United States of America</span></span><div class="contrib-group" title="contrib-group"><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Malik</span><span class="given-names" tagx="given-names" title="given-names">Harmit S.</span></span><span class="role" tagx="role" title="role">Editor</span><a href="#edit1" /></span></div><span class="citation_author_institution" id="edit1">[], </span><div class="author-notes" title="author-notes"> <div class="corresp" title="corresp">
* E-mail: <span class="email" tagx="email" title="email">mark.springer@ucr.edu</span></div> <div class="fn-type-con" title="con"> <p>Conceived and designed the experiments: RWM JG WJM OAR MSS. Performed the experiments: RWM. Analyzed the data: RWM JG MSS. Contributed reagents/materials/analysis tools: RWM JG WJM OAR MSS. Wrote the paper: MSS.</p> </div> </div><span class="pub-date-collection" title="pub-date-collection">collection: <span>2009-9</span></span><span class="pub-date-epub" title="pub-date-epub">epub: <span>2009-9-9</span></span><span class="volume" tagx="volume" title="volume">5</span><span class="issue" tagx="issue" title="issue">9</span><span class="elocation-id" tagx="elocation-id" title="elocation-id">e1000634</span><span class="history" title="history"><span class="received" title="received">received: 2009-5-26</span><span class="accepted" title="accepted">accepted: 2009-8-6</span></span><div class="permissions"><span class="copyright" title="copyright">(C) , <span class="copyright-year" tagx="copyright-year" title="copyright-year">2009</span></span><span class="license" title="license"><span class="license-p" title="license-p">This is an open-access article distributed under the terms of the Creative Commons Attribution License, which permits unrestricted use, distribution, and reproduction in any medium, provided the original author and source are properly credited.</span></span></div><span class="related-article" tagx="related-article" title="related-article"><span class="article-title" tagx="article-title" title="article-title">Molecular Tooth Decay</span></span><div class="abstract" title="abstract"> <div class="abstract-title" title="abstract-title">
Abstract</div> <p>Vestigial structures occur at both the anatomical and molecular levels, but studies documenting the co-occurrence of morphological degeneration in the fossil record and molecular decay in the genome are rare. Here, we use morphology, the fossil record, and phylogenetics to predict the occurrence of "molecular fossils" of the enamelin (<i>ENAM</i>) gene in four different orders of placental mammals (Tubulidentata, Pholidota, Cetacea, Xenarthra) with toothless and/or enamelless taxa. Our results support the "molecular fossil" hypothesis and demonstrate the occurrence of frameshift mutations and/or stop codons in all toothless and enamelless taxa. We then use a novel method based on selection intensity estimates for codons (ω) to calculate the timing of iterated enamel loss in the fossil record of aardvarks and pangolins, and further show that the molecular evolutionary history of <i>ENAM</i> predicts the occurrence of enamel in basal representatives of Xenarthra (sloths, anteaters, armadillos) even though frameshift mutations are ubiquitous in <i>ENAM</i> sequences of living xenarthrans. The molecular decay of <i>ENAM</i> parallels the morphological degeneration of enamel in the fossil record of placental mammals and provides manifest evidence for the predictive power of Darwin's theory.</p> </div> <div class="abstract" title="abstract"> <div class="title" tagx="title" title="title">
Author Summary</div> <p>Enamel is the hardest substance in the vertebrate body. One of the key proteins involved in enamel formation is enamelin. Most placental mammals have teeth that are capped with enamel, but there are also lineages without teeth (anteaters, pangolins, baleen whales) or with enamelless teeth (armadillos, sloths, aardvarks, pygmy and dwarf sperm whales). All toothless and enamelless mammals are descended from ancestral forms that possessed teeth with enamel. Given this ancestry, we predicted that mammalian species without teeth or with teeth that lack enamel would have copies of the gene that codes for the enamelin protein, but that the enamelin gene in these species would contain mutations that render it a nonfunctional pseudogene. To test this hypothesis, we sequenced most of the protein-coding region of the enamelin gene in all groups of placental mammals that lack teeth or have enamelless teeth. In every case, we discovered mutations in the enamelin gene that disrupt the proper reading frame that codes for the enamelin protein. Our results link evolutionary change at the molecular level to morphological change in the fossil record and also provide evidence for the enormous predictive power of Charles Darwin's theory of descent with modification.</p> </div> <div class="counts" title="counts"><span class="page-count" title="page-count">page-count: </span></div> </div> </div> <div class="body" title="body"> <div class="introduction" title="sec"> <div class="title" tagx="title" title="title">
Introduction</div> <p>Vestigial characters occur in a degenerate condition as a result of evolutionary reduction from a more elaborated, functional character state in an ancestral species <a href="#pgen.1000634-Futuyma1">[1]</a>. Darwin <a href="#pgen.1000634-Darwin1">[2]</a>,<a href="#pgen.1000634-Darwin2">[3]</a> identified "vestiges" in human anatomy such as the vermiform appendix that are leftovers from distant evolutionary history, and recognized that vestigial organs are both anticipated and explained by his theory of descent with modification. Other vestiges include remnants of the pelvic girdle and hindlimb within the body wall of baleen whales <a href="#pgen.1000634-Bedjer1">[4]</a> and eyes that are in various stages of degeneration in cave-dwelling animals <a href="#pgen.1000634-Fong1">[5]</a>. At the genomic level, genes that were functional in an ancestral species but which are no longer under selective constraints are predicted to degrade through an accumulation of mutations <a href="#pgen.1000634-Nishikimi1">[6]</a>,<a href="#pgen.1000634-Ohta1">[7]</a>. However, previous studies provide limited evidence linking morphological degeneration in the fossil record to molecular decay in the genome owing to the rarity of soft tissue preservation and the prevalence of pleiotropic genes that influence multiple phenotypic traits.</p> <p>Enamel is the hardest substance found in vertebrates and forms the outer cap of teeth, which comprise the most common and often only preserved remains of extinct mammalian species. Enamel is a derivative of the oral epithelium (ameloblasts), and its development involves secretory and maturation phases that culminate in the formation of an organized network of hydroxyapatite crystals <a href="#pgen.1000634-Hu1">[8]</a>. Multiple extracellular matrix proteins (EMPs) are secreted by mammalian ameloblasts: amelogenin, ameloblastin, and enamelin are structural proteins that direct the formation of hydroxyapatite crystals during the secretory phase of enamel formation; enamelysin and kallikrein 4 are proteinases that process and eventually degrade the structural EMPs during the secretory and maturation phases, respectively. Mutations in the genes for amelogenin (<i>AMELX)</i>, enamelin (<i>ENAM</i>), enamelysin (<i>MMP20</i>), and kallikrein 4 (<i>KLK4</i>) are known to cause defects in enamel that are collectively referred to as amelogenesis imperfectas <a href="#pgen.1000634-Wright1">[9]</a>. The three structural EMP genes belong to the secretory calcium-binding phosphoprotein (SSCP) gene family <a href="#pgen.1000634-Kawasaki1">[10]</a>, and have an evolutionary history that traces at least as far back as the most recent common ancestor of tetrapods <a href="#pgen.1000634-Sire1">[11]</a>,<a href="#pgen.1000634-Kawasaki2">[12]</a>. EMPs that are associated with enameloid formation in chondrichthyans and actinopterygians also belong to the SSCP gene family, but are not orthologous to tetrapod EMPs <a href="#pgen.1000634-Kawasaki2">[12]</a>.</p> <p>Enamelin is the largest EMP in mammals (1104 aa in pig) and is secreted along the mineralization front of developing enamel <a href="#pgen.1000634-Hu2">[13]</a> where it may play a role in crystal elongation <a href="#pgen.1000634-Hu1">[8]</a>. Homozygous <i>ENAM</i> knock-in mice show no true enamel <a href="#pgen.1000634-Hu3">[14]</a>, but are otherwise normal. The combination of the hardness of enamel and the hypothesized ameloblast-specific expression of enamelin <a href="#pgen.1000634-Hu3">[14]</a> provides a model system for investigating the correlated loss of a phenotypic character (enamel) in the fossil record and the molecular decay of a protein-coding gene (<i>ENAM</i>) in the genome.</p> <p>Most placental mammals have teeth with enamel, but there are also edentulous (toothless) mammals (pangolins, baleen whales, anteaters) and mammals with enamelless teeth (sloths, armadillos, pygmy and dwarf sperm whales, aardvark). Among edentulous mammals, there is evidence for aborted tooth bud development in mysticetes <a href="#pgen.1000634-Demr1">[15]</a>, pangolins <a href="#pgen.1000634-Osborn1">[16]</a>,<a href="#pgen.1000634-Spurgin1">[17]</a>, and anteaters <a href="#pgen.1000634-Spurgin1">[17]</a>. Rudimentary teeth are best developed in mysticetes, but enamel is not deposited and the tooth buds are degraded and resorbed prior to parturition (reviewed in reference <a href="#pgen.1000634-Demr1">[15]</a>). Given that all edentulous and enamelless mammals ultimately evolved from ancestors with fully mineralized teeth that included an outer enamel cap, we hypothesized that ameloblast-specific <i>ENAM</i> would be present in these taxa, albeit nonfunctional and in various states of decay with frameshift mutations and stop codons. To test this hypothesis, we generated and analyzed protein-coding sequences for <i>ENAM</i> in all placental orders containing edentulous and/or enamelless taxa. We also used selection intensity estimates for codons (ω) on branches of the tree with functional versus pseudogenic histories to estimate the timing of iterated enamel loss in the fossil record of aardvarks and pangolins. Finally, molecular evolutionary analyses of <i>ENAM</i> were employed to predict the occurrence of teeth with enamel in basal representatives of the order Xenarthra (anteaters, sloths, armadillos).</p> </div> <div class="results/discussion" title="sec"> <div class="title" tagx="title" title="title">
Results/Discussion</div> <p>Our sequence alignment (∼4.1 kb; <a href="#pgen.1000634.s001">Dataset S1</a>) included more than 80% of the protein-coding region for <i>ENAM</i>. Taxon sampling included 20 species that are representative of all placental orders that either lack teeth or have enamelless teeth, 27 additional placental species among which are the closest living relatives to each edentulous/enamelless group, and two marsupial outgroups. The <i>ENAM</i> gene tree (<a href="#pgen.1000634.s003">Figure S1</a> and <a href="#pgen.1000634.s004">Figure S2</a>) is generally concordant with other molecular phylogenies based on larger data sets <a href="#pgen.1000634-Murphy1">[18]</a>,<a href="#pgen.1000634-Springer1">[19]</a>. Three of the four major clades of placental mammals (Afrotheria, Xenarthra, Laurasiatheria) were represented in our study and each was recovered with robust support in the <i>ENAM</i> gene tree. Further, relationships within both Xenarthra and Cetartiodactyla are generally congruent with other molecular phylogenies <a href="#pgen.1000634-Delsuc1">[20]</a>,<a href="#pgen.1000634-OLeary1">[21]</a>. Differences between the <i>ENAM</i> gene tree and other molecular studies pertain to nodes that are not well supported by <i>ENAM</i> alone and typically require larger data sets with increased taxon sampling to subdivide long branches.</p> <p>Given the concordance of our gene tree to mammalian species trees, we mapped 125 different <i>ENAM</i> frameshift mutations onto a composite species tree (<a href="#pgen-1000634-g001">Figure 1</a>). Of these, 123 mapped onto the tree without homoplasy and the remaining two required only two character state changes (<a href="#pgen-1000634-g001">Figure 1</a>, <a href="#pgen.1000634.s011">Table S1</a>). Frameshifts are lacking in all taxa having teeth with enamel whereas 17 of 20 taxa lacking teeth or enamel have at least one frameshift in <i>ENAM</i> (<a href="#pgen-1000634-g001">Figure 1</a>). Frameshifts are evident in all four placental orders that include edentulous or enamelless taxa (<a href="#pgen-1000634-g002">Figure 2</a>).</p> <div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/10.1371/journal.pgen.1000634.g001"><span class="label" tagx="label" title="label">Figure 1</span></a></div>   <div class="title" tagx="title" title="title">
Species tree with frameshift mutations and dN/dS branch coding.</div> <p>Symbols next to taxon names denote taxa having teeth with enamel, taxa having teeth without enamel, and edentulous taxa. Branches are functional (black), pre-mutation (blue), mixed (purple), and pseudogenic (red). Vertical bars on branches represent frameshift mutations (see <a href="#pgen.1000634.s011">Table S1</a>). Frameshifts that map unambiguously onto branches are shown in black. Frameshifts shown in white are unique, but occur in regions where sequences are missing for one or more taxa (<a href="#pgen.1000634.s009">Figure S7</a>) and were arbitrarily mapped onto the youngest possible branch. Homoplastic frameshifts (deltran optimization) are marked by numbers. Numbers after taxon names indicate the minimum number of stop codons in the sequence (before slashes) and the length of the sequence (after slashes).</p>   </div> <div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/10.1371/journal.pgen.1000634.g002"><span class="label" tagx="label" title="label">Figure 2</span></a></div>   <div class="title" tagx="title" title="title">
Examples of frameshift mutations and stop codons in edentulous and enamelless taxa.</div> <p>(A) <i>Orycteropus</i>. (B) <i>Manis</i>. (C) Cetacea. (D) Xenarthra. Frameshift insertions are highlighted in blue, frameshift deletions are highlighted in red, and stop codons are highlighted in green. Also see <a href="#pgen.1000634.s007">Figure S5</a>.</p>   </div> <p><i>ENAM</i> sequences for edentulous and enamelless taxa were translated in all three reading frames and in every case stop codons were present (<a href="#pgen-1000634-g001">Figure 1</a>). Stop codons occur in both of the baleen whale genera (<i>Eubalaena</i>, <i>Megaptera</i>) that lack frameshift mutations. The combination of frameshifts and stop codons provides evidence that all edentulous and enamelless taxa lack a functional copy of full length <i>ENAM</i>. Molecular decay of <i>ENAM</i> in these lineages also corroborates the hypothesis that enamelin exhibits ameloblast-specific gene expression <a href="#pgen.1000634-Hu3">[14]</a>.</p> <p>We calculated dN/dS ratios (ω) for <i>ENAM</i> after dividing phylogenetic branches into four categories [functional, pre-mutation, mixed (functional + pseudogene), and pseudogene] based on ancestral reconstructions of the presence or absence of enamel, and the distribution of frameshifts and stop codons in <i>ENAM</i> sequences (<a href="#pgen-1000634-g001">Figure 1</a>, <a href="#pgen.1000634.s005">Figure S3</a>). Functional branches lead to internal nodes or extant species with teeth having enamel and are expected to have evolved under selective constraints with the lowest ω values unless there is extensive positive selection. Pre-mutation, mixed, and pseudogene branches lead to internal nodes or extant species that lack enamel. Pre-mutation branches lack frameshifts or stop codons in the region of <i>ENAM</i> that was sequenced. Mixed branches are descended from pre-mutation branches and contain the first detected occurrence of a frameshift or stop codon. Mixed branches may be entirely pseudogenic if undetected frameshifts occurred on an earlier (pre-mutation) branch; alternatively, mixed branches may record a history of evolution under functional constraints combined with pseudogenic evolution if the first frameshift or stop codon occurred on the mixed branch. Pseudogene branches occur after mixed branches and are expected to have ω values near the neutral value of 1 given the absence of purifying or positive selection.</p> <p>Results of dN/dS analyses confirm that functional branches have evolved under purifying selection (ω = 0.51) and that pseudogene branches have evolved as predicted for pseudogenes (ω = 1.02). The pseudogene value of 1.02 agrees with the null hypothesis that <i>ENAM</i> evolved under neutral constraints (ω = 1) based on a χ<sup>2</sup> test (0.75</p><p>&amp;lt;0.90). Estimated values for pre-mutation and mixed branches were 0.83 and 0.98, respectively, and are compatible with the hypothesis that <i>ENAM</i> has evolved under progressively more relaxed constraints on branches leading to species that are edentulous or enamelless. The ω value for mixed branches is only slightly less than for pseudogene branches, which suggests that these branches have histories that are almost entirely pseudogenic.</p> <p>The abundance of mammalian teeth in the fossil record provides information on the timing and sequence of events leading to enamel and tooth loss in different mammalian lineages. Complementary data from molecular evolutionary analyses of <i>ENAM</i> provide information on missing segments of the fossil record. Together, as discussed below, the fossil record and the genome facilitate predictive interplay and inform our understanding of the evolutionary history of enamel degeneration in a way that is not possible based on fossils or molecular sequences alone.</p> <p>Teeth in the extant aardvark, <i>Orycteropus afer</i>, lack both enamel and a central pulp cavity and are composed of ∼1500 thin, hexagonal tubes of dentin that are bound together by cementum <a href="#pgen.1000634-Holroyd1">[22]</a>. Whereas molecular data suggest that the aardvark lineage diverged from its closest living relatives (Afrosoricida, Macroscelidea) ∼75.1 million years ago in the Cretaceous <a href="#pgen.1000634-Murphy2">[23]</a>, the oldest fossil aardvarks are <i>O. minutus</i> (19 mya) and <i>Myorycteropus africanus</i> (18 mya) from the early Miocene of Kenya and also lack enamel <a href="#pgen.1000634-Patterson1">[24]</a>,<a href="#pgen.1000634-Milledge1">[25]</a>. The combination of molecular and paleontological data thus suggests that enamel was lost in the aardvark lineage after aardvarks split from their nearest extant relatives 75 million years ago but before the early Miocene when aardvarks first appear in the fossil record. We estimated the value of ω for <i>ENAM</i> at 0.75 for the aardvark branch and then calculated the fraction of functional versus pseudogenic history for <i>ENAM</i> on this branch by comparing its ω value with ωvalues that were calculated for functional and pseudogene branches (see <a href="#s4">Methods</a>). Calculations were performed using two different assumptions. First, functional and pseudogenic segments of the mixed branch were assumed to have equal synonymous substitution rates <a href="#pgen.1000634-Wolfe1">[26]</a>,<a href="#pgen.1000634-Kumar1">[27]</a>. Second, the rate of synonymous substitution on the functional segment of the mixed branch was assumed to be 70% of the rate of synonymous substitution on the pseudogenic segment on the mixed branch <a href="#pgen.1000634-Bustamante1">[28]</a>. Our estimates for the transition from functional gene to pseudogene for aardvark <i>ENAM</i> range from 28.8 mya (two synonymous rates) to 35.3 mya (one synonymous rate) and are ∼10-16 million years older than the first fossil aardvarks. Our analyses predict the occurrence of enamelless fossil aardvarks that are much older than both <i>O. minutus</i> and <i>M. africanus</i> (<a href="#pgen-1000634-g003">Figure 3</a>).</p> <div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/10.1371/journal.pgen.1000634.g003"><span class="label" tagx="label" title="label">Figure 3</span></a></div>   <div class="title" tagx="title" title="title">
A synthetic interpretation of the history of enamel degeneration in Tubulidentata (aardvarks) based on fossils, phylogenetics, molecular clocks, frameshift mutations, and dN/dS ratios.</div> <p>Estimates for the timing of the transition from functional gene to pseudogene are based on decomposition of mixed dN/dS history into two distinct episodes of evolution under purifying selection and neutral evolution, respectively. The divergence time between Tubulidentata and Afrosoricida (golden moles, tenrecs) + Macroscelidea (elephant shrews) is from Murphy and Eizirik <a href="#pgen.1000634-Murphy2">[23]</a>. The age of the oldest aardvark, <i>O. minutus</i>, is from Milledge <a href="#pgen.1000634-Milledge1">[25]</a>.</p>   </div> <p>Numerous frameshifts were reconstructed on the internal branch leading to living pangolins (<i>Manis</i>) and support the hypothesis that <i>ENAM</i> was rendered nonfunctional on this branch (<a href="#pgen-1000634-g001">Figure 1</a>). The internal branch leading to crown-group <i>Manis</i> extends from 79.8 mya <a href="#pgen.1000634-Murphy2">[23]</a> to 27.9 mya (see <a href="#s4">Methods</a>). The earliest definitive pangolin is <i>Eomanis</i> from the Eocene Messel deposits of Germany, which have been dated at ∼47 mya <a href="#pgen.1000634-Franzen1">[29]</a>. <i>Eomanis</i> was edentulous as are modern pangolins. The occurrence of an early pangolin without teeth is consistent with our finding that <i>ENAM</i> became a pseudogene on the stem pangolin branch and further suggests that <i>ENAM</i> has been a pseudogene since at least the Eocene. Given that the branch leading to crown-group pangolins represents ∼52 million years of evolution and that the value of ω on this branch is 0.82, we used the aforementioned method to estimate that the transition from functional gene to pseudogene occurred 54.9 (two synonymous rates) to 59.4 (one synonymous rate) mya, which is ∼8 to 12 million years older than <i>Eomanis</i>. We therefore predict the occurrence of edentulous, or at least enamelless, fossil pangolins in the Paleocene (<a href="#pgen-1000634-g004">Figure 4</a>).</p> <div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/10.1371/journal.pgen.1000634.g004"><span class="label" tagx="label" title="label">Figure 4</span></a></div>   <div class="title" tagx="title" title="title">
A synthetic interpretation of the history of enamel degeneration in Pholidota (pangolins) based on fossils, phylogenetics, molecular clocks, frameshift mutations, and dN/dS ratios.</div> <p>Estimates for the timing of the transition from functional gene to pseudogene are based on decomposition of mixed dN/dS history into two distinct episodes of evolution under purifying selection and neutral evolution, respectively. The divergence time between Pholidota and Carnivora (79.8 mya) is from Murphy and Eizirik <a href="#pgen.1000634-Murphy2">[23]</a>. The divergence time between <i>Manis pentadactyla</i> and <i>M. tricuspis</i> (27.9 mya) is based on molecular dating analyses with <i>ENAM</i> (see <a href="#s4">Methods</a>). The age of the oldest pangolin, <i>Eomanis</i> from the Eocene Messel deposits of Germany, is ∼47 mya <a href="#pgen.1000634-Franzen1">[29]</a>.</p>   </div> <p>Extinct palaeanodonts have been associated with pangolins <a href="#pgen.1000634-Emry1">[30]</a>,<a href="#pgen.1000634-Rose1">[31]</a>, xenarthrans <a href="#pgen.1000634-Simpson1">[32]</a>,<a href="#pgen.1000634-Patterson2">[33]</a>, or both groups <a href="#pgen.1000634-Matthew1">[34]</a>–<a href="#pgen.1000634-Rose2">[36]</a>. Palaeanodonts first appear in the early Paleocene of North America and show a mixture of xenarthran and pholidotan features <a href="#pgen.1000634-Rose3">[37]</a>. The earliest forms have teeth with enamel, but palaeanodonts show trends toward reduction and loss of postcanine teeth from back to front, and loss of enamel on the postcanine teeth <a href="#pgen.1000634-Rose3">[37]</a>–<a href="#pgen.1000634-Gheerbrant1">[39]</a>. Rose <i>et al</i>. (Figure 8.4 in <a href="#pgen.1000634-Rose3">[37]</a>) hypothesized that Palaeanodonta and Pholidota are either sister taxa that split in the latest Cretaceous or that Pholidota evolved from within Palaeanodonta in the early Eocene. The former hypothesis implies that tooth reduction and loss, as well as enamel loss on the postcanine teeth, occurred independently in palaeanodonts and pangolins, whereas these changes may be shared derived features uniting metacheiromyid palaeanodonts and pangolins if the latter hypothesis is correct. Our estimates for the transition from functional <i>ENAM</i> to pseudogene on the basal pangolin branch (54.9–59.4 mya) suggest that ancestral pholidotans without enamel were contemporaneous with, rather than descended from, late Paleocene and early Eocene metacheiromyids such as <i>Palaeanodon</i> that exhibited a reduced postcanine dental arcade, but retained a functional copy of the <i>ENAM</i> gene as inferred by the presence of canines with enamel. The combination of fossil and molecular evolutionary evidence is thus more compatible with the hypothesis that palaeanodonts and pholidotans are sister taxa than the hypothesis that pangolins are derived from within Metacheiromyidae.</p> <p>Cetacea includes both edentulous (Mysticeti) <a href="#pgen.1000634-Demr1">[15]</a> and enamelless (<i>Kogia</i>) <a href="#pgen.1000634-Bianucci1">[40]</a> taxa. Early mysticete fossils record the transition from ancestral forms that possessed teeth, but not baleen, to intermediate forms with teeth and baleen, to forms that are fully edentulous <a href="#pgen.1000634-Demr1">[15]</a>,<a href="#pgen.1000634-Fitzgerald1">[41]</a>. This pattern suggests that purifying selection on enamel-specific genes was relaxed on the stem mysticete branch. The five mysticetes examined here all exhibit frameshifts and/or stop codons in <i>ENAM</i>, which is indicative of pseudogenic evolution, although no frameshifts or stop codons were found that unite all mysticetes. This result agrees with the findings of Deméré <i>et al.</i><a href="#pgen.1000634-Demr1">[15]</a> that the ameloblastin gene (<i>AMBN</i>) and a shorter segment of <i>ENAM</i> (∼530 bp) both contain frameshift mutations that are shared by one or more mysticete taxa, but none in common to all extant mysticetes. Whereas deltran optimization (<a href="#pgen-1000634-g001">Figure 1</a>) suggests an independent origin of the same frameshift mutation in <i>Eschrichtius robustus</i> and <i>Caperea marginata</i>, acctran optimization (not shown) suggests an earlier origin of the frameshift in the common ancestor of <i>E. robustus</i>, <i>C. marginata</i>, and <i>Megaptera novaeangliae</i>, followed by loss of the frameshift in <i>M. novaeangliae</i>. The acctran optimization scenario is compatible with the gain of an ancestral polymorphism on the branch leading to these taxa, followed by subsequent lineage sorting that resulted in loss of the frameshift in <i>M. novaeangliae</i><a href="#pgen.1000634-Demr1">[15]</a>. When all crown mysticete branches were treated as a separate category in dN/dS analyses (<a href="#s4">Methods</a>), the result was in agreement with the null hypothesis that <i>ENAM</i> evolved as a pseudogene (i.e., ω = 1) based on a χ<sup>2</sup> test (0.10</p><p>&amp;lt;0.25). Thus, it appears that <i>ENAM</i> was released from selective constraints in crown mysticetes as predicted by the basal position of the archaic toothless mysticete, <i>Eomysticetus whitmorei</i> (<a href="#pgen-1000634-g005">Figure 5</a>) <a href="#pgen.1000634-Demr1">[15]</a>. If we assume that <i>ENAM</i> evolved as a pseudogene in crown-group mysticetes, which as a group are characterized by generation times and lifespans that equal or exceed those of most other mammals, then the neutral rate of nucleotide substitution is only 3.3×10<sup>−4</sup> substitutions/site/myr, which is almost an order of magnitude lower than Kumar and Subramanian's <a href="#pgen.1000634-Kumar1">[27]</a> estimate of the average mammalian genome rate of 2.2×10<sup>−3</sup> substitutions/site/myr based on an analysis of fourfold degenerate sites.</p> <div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/10.1371/journal.pgen.1000634.g005"><span class="label" tagx="label" title="label">Figure 5</span></a></div>   <div class="title" tagx="title" title="title">
A synthetic interpretation of the history of enamel degeneration in Cetacea based on fossils, phylogenetics, molecular clocks, frameshift mutations, and dN/dS ratios.</div> <p>The common ancestor of Cetacea was reconstructed as having teeth with enamel based on the occurrence of teeth with enamel in stem cetaceans, stem mysticetes, and stem odontocetes. The common ancestor of Mysticeti was reconstructed as toothless based on (1) the fossil record, which records the transition from ancestral forms that possessed teeth but not baleen (e.g., <i>Janjucetus</i>), to intermediate forms with teeth and baleen (e.g., <i>Aetiocetus</i>), to forms that are fully edentulous (e.g., <i>Eomysticetus</i>) and (2) a dN/dS ratio for <i>ENAM</i> in crown mysticetes that agrees with the null hypothesis of neutral evolution (dN/dS = 1). The common ancestor of physeteroids was reconstructed as having teeth with enamel based on the presence of enamel-capped teeth in stem physeteroids (e.g., <i>Naganocetus</i>) and <i>Physeter</i>. Cladistic relationships and divergence times among extant taxa are based on McGowen <i>et al.</i><a href="#pgen.1000634-McGowen1">[69]</a>. Cladistic relationships of basal mysticetes and basal physeteroids are based on Deméré <i>et al.</i><a href="#pgen.1000634-Demr1">[15]</a> and Bianucci and Landini <a href="#pgen.1000634-Bianucci1">[40]</a>, respectively. Frameshift mutations and stop codons were mapped using deltran optimization. All extant mysticete lineages that were sampled have at least one frameshift or stop codon, but none are shared by all mysticetes. Acctran optimization suggests that some homoplastic frameshifts and stop codons within Mysticeti may have resulted from lineage sorting of ancestral polymorphisms <a href="#pgen.1000634-Demr1">[15]</a>.</p>   </div> <p>Stem physeteroids (sperm whales) are known from the Miocene and had teeth with enamel <a href="#pgen.1000634-Bianucci1">[40]</a>. <i>Physeter</i> and <i>Kogia</i> are the only living genera of sperm whales and the teeth in adults belonging to these genera lack enamel. A caveat is that thin, prismless enamel has been observed on the unerupted teeth of <i>P. macrocephalus</i> (giant sperm whale) <a href="#pgen.1000634-Ishiyama1">[42]</a>. Our results provide support for loss of the intact enamelin protein in both <i>Kogia</i> species, but not in <i>P. macrocephalus</i> (<a href="#pgen-1000634-g005">Figure 5</a>, <a href="#pgen.1000634.s012">Text S1</a>).</p> <p>Xenarthrans comprise one of the major clades of placental mammals <a href="#pgen.1000634-Murphy1">[18]</a> and the stem xenarthran branch represents ∼30 million years of evolution <a href="#pgen.1000634-Murphy3">[43]</a>. Even with this long history there are no described fossils that are definitive basal xenarthrans <a href="#pgen.1000634-Rose3">[37]</a>. Almost all living and fossil xenarthrans, which are crown taxa, either lack teeth or have teeth without enamel. The only clear-cut exception is <i>Utaetus</i>, a member of Cingulata (armadillos and extinct relatives) from the Casamayoran (Eocene) that had thin, prismatic enamel (<a href="#pgen.1000634.s012">Text S1</a>) <a href="#pgen.1000634-Simpson2">[44]</a>. The occurrence of enamel in <i>Utaetus</i> suggests that enamel was lost independently in pilosans (anteaters, sloths) and in cingulatans. An alternative hypothesis is that <i>Utaetus</i> re-evolved enamel after earlier loss in the common ancestor of Xenarthra (<a href="#pgen.1000634.s006">Figure S4</a>). We estimated the value of ω for the stem xenarthran branch to determine if <i>ENAM</i> evolved under purifying selection or as a pseudogene on this branch. The estimated value (0.48) is similar to that estimated for functional branches (0.51) and implies that <i>ENAM</i> evolved under purifying selection on the stem xenarthran branch, concordant with the hypothesis that enamel was lost independently in Pilosa and in Cingulata. Ancestral reconstructions based solely on the phenotypes of extant taxa (<a href="#pgen.1000634.s005">Figure S3</a>) imply that enamel was lost on the stem xenarthran branch, but we predict that all stem xenarthran fossils will have teeth with enamel (<a href="#pgen-1000634-g006">Figure 6</a>) based on (1) an estimated value of ω on the stem xenarthran branch that is similar to the estimated value of ω for functional branches elsewhere on the tree, (2) the absence of frameshifts on the stem xenarthran branch, (3) the occurrence of enamel in a fossil armadillo (<i>Utaetus</i>), which would be difficult to regain if enamel had been lost earlier in evolution, and (4) genomics observations on <i>AMELX</i>, <i>AMBN</i>, <i>MMP20</i>, and <i>ENAM</i> in <i>Dasypus novemcinctus</i>, which preclude the possibility of unique, homoplasy-free frameshifts in these genes in the common ancestry of Xenarthra (see below).</p> <div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/10.1371/journal.pgen.1000634.g006"><span class="label" tagx="label" title="label">Figure 6</span></a></div>   <div class="title" tagx="title" title="title">
An integrated interpretation of the history of enamel degeneration in Xenarthra based on fossils, phylogenetics, molecular clocks, frameshift mutations, and dN/dS ratios.</div> <p>No definitive stem xenarthrans have been described, but the common ancestor of Xenarthra was reconstructed as having teeth with enamel based on (1) purifying selection on the stem xenarthran branch (ω = 0.48), (2) a reconstructed ancestral <i>ENAM</i> sequence that contains no frameshifts and no stop codons, (3) at least one crown-group xenarthran (<i>Utaetus</i>) with enamel, which would be improbable to re-evolve if the common ancestor of Xenarthra lacked enamel and had a nonfunctional copy of <i>ENAM</i>, and (4) genomics observations. The common ancestor of Pilosa was reconstructed as having teeth without enamel based on the occurrence of shared frameshifts in <i>ENAM</i> on the stem pilosan branch. The dN/dS ratio in crown clade Dasypodidae agrees with the neutral evolution (dN/dS = 1) hypothesis. There are also frameshifts that result in premature stop codons in all dasypodid <i>ENAM</i> sequences; however, none are shared by all four dasypodids that were sampled in our analysis. We reconstructed the ancestor of living Dasypodidae as having enamel, which may have been very thin as in <i>Utaetus</i>, based on the absence of shared frameshift mutations in protein-coding sequences of <i>AMELX</i>, <i>AMBN</i>, <i>ENAM</i>, and <i>MMP20</i>. The sister group relationship and divergence time between <i>Choloepus hoffmanni</i> and <i>C. didactylus</i> is based on analyses with <i>ENAM</i> (<a href="#pgen.1000634.s003">Figure S1</a>, <a href="#pgen.1000634.s004">Figure S2</a>; <a href="#s4">Methods</a>). All other relationships and divergence times are based on <a href="#pgen-1000634-g001">Figure 1</a> of Delsuc <i>et al.</i><a href="#pgen.1000634-Delsuc1">[20]</a>. <i>Utaetus</i> is derived from Casamayoran age deposits, which are thought to be ?middle to late Eocene in age <a href="#pgen.1000634-Shockey1">[75]</a>,<a href="#pgen.1000634-Croft1">[76]</a>.</p>   </div> <p>Unambiguous fossils of Pilosa are crown rather than stem <a href="#pgen.1000634-Rose3">[37]</a> and the fossil record does not speak directly to whether enamel was lost on the branch leading to extant pilosans. However, the occurrence of 2–4 frameshifts on the stem pilosan branch implies that <i>ENAM</i> transitioned from a functional gene to a pseudogene after Pilosa diverged from Cingulata ∼65 mya, but before Folivora (sloths) separated from Vermilingua (anteaters) ∼55 mya <a href="#pgen.1000634-Delsuc1">[20]</a> (<a href="#pgen-1000634-g006">Figure 6</a>).</p> <p>In Cingulata, all extant dasypodids (armadillos) that we examined have frameshifts and premature stop codons, suggesting that <i>ENAM</i> is a decaying pseudogene in these taxa. However, the only shared frameshifts were in <i>Euphractus</i> and <i>Chaetophractus</i>, which suggests the possibility that enamel was lost independently in different armadillo lineages. As a further test of this hypothesis, we reconstructed the complete coding sequences for the enamelin (<i>ENAM</i>), amelogenin (<i>AMELX</i>), ameloblastin (<i>AMBN</i>), and enamelysin (<i>MMP20</i>) genes in <i>Dasypus novemcinctus</i> based on Ensembl 51 scaffolds and trace files (including inspection of chromatograms). There are no frameshifts or stop codons in the coding sequences of <i>AMELX</i>, <i>AMBN</i>, and <i>MMP20</i> in <i>D. novemcinctus</i>, and the only frameshift in the Ensembl sequence for <i>ENAM</i> is identical to the one that we discovered for <i>D. novemcinctus</i> (<a href="#pgen-1000634-g002">Figure 2</a>, <a href="#pgen.1000634.s007">Figure S5</a>, and <a href="#pgen.1000634.s008">Figure S6</a>). These observations preclude the occurrence of unique, homoplasy-free frameshifts in <i>AMELX</i>, <i>AMBN</i>, <i>MMP20</i>, or <i>ENAM</i> in the common ancestry of either Xenarthra or Dasypodidae and are consistent with the hypothesis that enamel was lost independently in Pilosa and also in more than one dasypodid lineage. When crown-group dasypodid branches were treated as a separate category in dN/dS analyses with <i>ENAM</i>, the pseudogene hypothesis of neutral evolution (ω = 1) could not be rejected (0.05</p><p>&amp;lt;0.10), which suggests that <i>ENAM</i> has evolved as a pseudogene over most of the history of Dasypodidae even though the protein-coding regions of four EMPs appear to have been intact in the most recent common ancestor of Dasypodidae (<a href="#pgen-1000634-g006">Figure 6</a>).</p> <p>Whereas our analyses provide strong support for multiple instances of tooth loss, enamel loss, and <i>ENAM</i> degeneration in placental mammals, they provide no compelling evidence for the re-evolution of enamel following degeneration of the <i>ENAM</i> gene on an earlier branch of the tree. This finding is consistent with Dollo's law, which may be stated as follows: complex organs, once lost, can never be regained in exactly the same form <a href="#pgen.1000634-Lee1">[45, p. 1441]</a>. Dollo's law implies that genes or developmental pathways released from selective constraints will rapidly become nonfunctional <a href="#pgen.1000634-Marshall1">[46]</a>. Marshall <i>et al.</i><a href="#pgen.1000634-Marshall1">[46]</a> showed that there is a significant probability over evolutionary time scales of 0.5 to 6 million years for successful reactivation of neutrally evolving coding sequences in the range of 0.5 to 2.0 kb, but that resurrection of genes of this length that have evolved neutrally for &amp;gt;10 million years is highly improbable. In the case of a three kb exon (e.g., exon 10 of <i>ENAM</i>), the probability of survival after one million years of neutral evolution ranges from 0.65 to 1.6×10<sup>−17</sup> based on calculations using Marshall <i>et al.</i>'s <a href="#pgen.1000634-Marshall1">[46]</a> equation 4 and a range of substitution rates, frameshift accumulation rates, and different values of <i>f</i>, where <i>f</i> is the probability that a substitution will not result in loss of function. Even in the case of very slow substitution and frameshift accumulation rates (i.e., mysticete rates), and a high value of <i>f</i> (i.e., 0.7) <a href="#pgen.1000634-Marshall1">[46]</a>, the probability that a 3 kb exon will remain functional after ten million years is only 0.014. Our finding that enamel has not been regained in any placental mammal lineage is consistent with these predictions for <i>ENAM</i>'s rapid demise in the absence of selective constraints.</p> <p>Among other vertebrates, Kollar and Fisher <a href="#pgen.1000634-Kollar1">[47]</a> performed experiments with recombinant mouse molar mesenchyme and chick dental epithelium that resulted in "hen's teeth" consisting of a dentine cone covered by enamel matrix proteins. This result implies that chick dental epithelium has retained the genetic potential to develop until the last developmental stage (enamel matrix deposition) for 80 to 100 million years. However, possible contamination of mouse mesenchyme by mouse epithelium makes this interpretation uncertain (reviewed in <a href="#pgen.1000634-Sire2">[48]</a>). Further, Sire <i>et al.</i><a href="#pgen.1000634-Sire2">[48]</a> report that all of the genes encoding dental-specific proteins in chicken, including <i>ENAM</i>, have either disappeared from the genome or are pseudogenes. Oral teeth were lost in the common ancestor of cypriniform fishes at least 50 million years ago, possibly in conjunction with the evolution of suction feeding, but have never been regained <a href="#pgen.1000634-Jackman1">[49]</a>. The group has subsequently diversified into ∼3000 extant species that exhibit a wide range of feeding modes, but the loss of oral teeth may have constrained the evolution of large fish-eating forms, which are rare and may be less efficient predators than teleosts that retain oral teeth <a href="#pgen.1000634-Portz1">[50]</a>,<a href="#pgen.1000634-Stock1">[51]</a>. There are examples of the reacquisition of specific teeth, including the polymorphic occurrence of the second lower molar in <i>Lynx lynx</i><a href="#pgen.1000634-Werdelin1">[52]</a>, and of sets of teeth such as the third basibranchial teeth in centrarchid fishes <a href="#pgen.1000634-Gosline1">[53]</a>. However, examples of the reacquisition of individual teeth or sets of teeth only occur in the context of organisms that retain other teeth in the oropharynx <a href="#pgen.1000634-Stock2">[54]</a>.</p> <p>Darwin <a href="#pgen.1000634-Darwin1">[2]</a> stated that "the existence of organs in a rudimentary, imperfect, and useless condition, or quite aborted, far from presenting a strange difficulty, as they assuredly do on the ordinary doctrine of creation, might even have been anticipated, and can be accounted for by the laws of inheritance." Our analysis of <i>ENAM</i> pseudogenes is the first study to combine information from living taxa, the fossil record, phylogenetics, molecular sequences, and selection intensity estimates for codons to make reciprocal predictions about patterns of molecular decay and the parallel loss of a phenotypic character in the fossil record. Given ancestral species with enamel-capped teeth, we correctly predicted the occurrence of vestigial copies of <i>ENAM</i> in four different placental orders with edentulous or enamelless species. dN/dS methods then allowed us to predict the timing of iterated enamel loss in the fossil record of aardvarks and pangolins and the presence of enamel in extinct, basal xenarthrans that may reside in the rock record, but have not yet been described by paleontologists. The molecular decay of <i>ENAM</i> mirrors the morphological degeneration of enamel in edentulous and enamelless taxa and provides manifest evidence for the predictive power of Darwin's theory.</p> </div> <div class="methods" title="sec"> <div class="title" tagx="title" title="title">
Methods</div> <div class="genesequences" title="sec"> <div class="title" tagx="title" title="title">
Gene Sequences</div> <p>Exon 10 <a href="#pgen.1000634-Hart1">[55]</a> is the longest exon in <i>ENAM</i> (2838 base pairs in <i>Sus</i>) and codes for &amp;gt;80% of the total length of the secreted protein after removal of the signal peptide. PCR was used to amplify ∼95% of exon 10 in four overlapping segments. Primer sequences are enumerated in <a href="#pgen.1000634.s009">Figure S7</a>. All segments were amplified with Invitrogen <i>Taq</i> DNA polymerase in 50 µl reactions with the following thermal cycling parameters: initial denaturation at 94°C for 2 minutes; 35 cycles of 1 minute at 94°C (denaturation), 1 minute at 50°C (annealing), and 1 minute at 72°C (extension); and a final extension at 72°C for 10 minutes. Fully nested or half nested PCR reactions, using the aforementioned PCR regime, were performed when the primary PCR reaction did not yield a usable product. One µl of the original PCR product was used as the template DNA in the 50 µl nested PCR reaction. PCR products were run out on a 1% agarose gel, excised, and cleaned with the Bioneer AccuPrep Gel Purification Kit. Cleaned PCR products were sequenced in both directions at the UCR Core Instrumentation Facility using an automated DNA sequencer (ABI 3730xl). Contigs were assembled using Sequencher 4.1. In cases where PCR products did not sequence directly, they were cloned using the CloneJET PCR Cloning Kit (Fermentas) and Top10 chemically competent cells. Transformed cells were grown overnight on SOB agar medium at 37°C. Three to five clone colonies were then picked and used as the template DNA in 50 µl PCR reactions using vector specific primers. PCR amplification, cleaning, and sequencing protocols were identical to the aforementioned regime except that the annealing temperature was 60°C. Sequences for cloned PCR products are consensus sequences based on data from at least three clones. Some xenarthran and <i>Manis</i> segments failed to amplify (<a href="#pgen.1000634.s010">Figure S8</a>). Genbank Accession numbers for new <i>ENAM</i> sequences for 46 taxa are GQ354839-GQ354884; sequences for <i>Bos</i>, <i>Canis</i>, and <i>Monodelphis</i> were obtained from Ensembl 51.</p> </div> <div class="dnaalignments" title="sec"> <div class="title" tagx="title" title="title">
DNA Alignments</div> <p>DNA sequences were aligned with Clustal X 2.0.9 <a href="#pgen.1000634-Larkin1">[56]</a> using default settings and then manually adjusted using Se-Al <a href="#pgen.1000634-Rambaut1">[57]</a>. Two alignments were generated. The first alignment (4089 bp; <a href="#pgen.1000634.s001">Dataset S1</a>), which was used for phylogenetic analyses and visual discovery of frameshift mutations, included complete sequences for all taxa. Frameshift mutations were mapped onto the composite species tree (<a href="#pgen-1000634-g001">Figure 1</a>) using deltran parsimony optimization. The second alignment (2760 bp; <a href="#pgen.1000634.s002">Dataset S2</a>) was used in dN/dS analyses with PAML and omitted marsupial sequences, all frameshift insertions (i.e., insertions not in multiples of three), all insertions that were unique to one taxon, and all insertions that were unique to edentulous or enamelless taxa. Stop codons were recoded as missing.</p> </div> <div class="phylogeneticanalyses" title="sec"> <div class="title" tagx="title" title="title">
Phylogenetic Analyses</div> <p>Modeltest 3.06 <a href="#pgen.1000634-Posada1">[58]</a> was used to select the model of molecular evolution (GTR + Γ) that was implemented in ML and Bayesian analyses with the 4089 bp alignment. RAxML7.0.4 <a href="#pgen.1000634-Stamatakis1">[59]</a> and MrBayes v3.1.1 <a href="#pgen.1000634-Ronquist1">[60]</a>,<a href="#pgen.1000634-Huelsenbeck1">[61]</a> were used to perform the ML and Bayesian analyses, respectively. RAxML analyses employed 500 replicates, randomized MP starting trees, and the fast hill-climbing algorithm; all other free parameters were estimated. The Bayesian analyses used the default priors, random starting trees, and eight chains (seven hot, one cold), and were terminated when the standard deviation of split frequencies for two different simultaneous analyses fell below 0.01 (2.03 million post-burnin generations). Standard deviation of split frequency calculations were performed on the fly after discarding 25% of each chain as burnin. Post-burnin trees were sampled every 1000 generations (i.e., 2030 trees were sampled from each analysis).</p> </div> <div class="dn/dsanalyses" title="sec"> <div class="title" tagx="title" title="title">
dN/dS Analyses</div> <p>PAML 4.2 <a href="#pgen.1000634-Yang1">[62]</a> was used to estimate the ratio (ω) of the nonsynonymous substitution rate (dN) to the synonymous substitution rate (dS) using a composite species tree (<a href="#pgen-1000634-g001">Figure 1</a>) and the PAML alignment after recoding stop codons as missing data. The composite species tree is based on Springer <i>et al.</i><a href="#pgen.1000634-Springer1">[19]</a> for interordinal relationships, Delsuc et al. <a href="#pgen.1000634-Delsuc1">[20]</a> for xenarthrans, and Gatesy <a href="#pgen.1000634-Gatesy1">[63]</a> for cetartiodactyls. Codeml <a href="#pgen.1000634-Yang1">[62]</a> with the branch model was implemented to estimate dN/dS ratios (ω) for four different branch categories (functional, pre-mutation, mixed, pseudogene) (<a href="#pgen-1000634-g001">Figure 1</a> and <a href="#pgen.1000634.s005">Figure S3</a>). Functional branches lead to terminal nodes (i.e., extant taxa) having enamel or internal nodes that were reconstructed as having enamel (<a href="#pgen.1000634.s005">Figure S3</a>) and are expected to have evolved under purifying selection with ω values&amp;lt;1. Pre-mutation, mixed, and pseudogene branches lead to terminal nodes without enamel or internal nodes that were reconstructed as not having enamel (<a href="#pgen.1000634.s005">Figure S3</a>). Pre-mutation branches predate the first detected occurrence of a stop codon or frameshift in <i>ENAM</i>; these branches have uncertain functional versus pseudogene histories and may have evolved under purifying selection or alternatively under relaxed constraints if enamel was already lost or in a state of degeneration. Mixed branches contain the first detected occurrence of a frameshift mutation or stop codon in <i>ENAM</i>. Mixed branches may be entirely pseudogenic if undetected frameshift mutations (i.e., in one of the remaining short exons that comprise the <i>ENAM</i> gene structure that we did not sample here) occurred on an earlier pre-mutation branch; alternatively, mixed branches may reflect a history of evolution that includes a period of evolution under functional constraints followed by a period of pseudogenic evolution if the first frameshift mutation or stop codon in <i>ENAM</i> occurred on the mixed branch. Pseudogene branches post-date the first detected occurrence of a frameshift or stop codon on an earlier branch and are expected to have evolved at the neutral rate (ω = 1). In addition to analyses that recognized these four categories of branches, we performed additional analyses that added a fifth branch category for (a) the branch leading to <i>Orycteropus</i>, (b) the branch leading to <i>Manis</i> species, (c) crown-group mysticete branches, (d) the stem branch leading to crown Xenarthra, and (e) crown-group dasypodid branches, respectively. Codon sites with ambiguous data (i.e., missing data, gaps) were included in the analyses. We used χ<sup>2</sup> tests to compare the estimated number of nonsynonymous and synonymous substitutions (from PAML) with the expected number of nonsynonymous and synonymous substitutions according to a neutral model of evolution with ω = 1. PAML estimates for the number of nonsynonymous and synonymous sites in the alignment were 2004.6 and 755.4, respectively. These estimates were used to calculate expected numbers of nonsynonymous and synonymous changes. In the case that examined all pseudogene branches in the four-category analysis, estimated numbers of nonsynonymous and synonymous changes were 780.8 and 287.9, respectively, whereas expected numbers of nonsynonymous and synonymous changes were 776.2 and 292.5, respectively. In the case of crown-group mysticetes, estimated numbers of nonsynonymous and synonymous changes were 66.3 and 16.3, respectively, whereas expected numbers of nonsynonymous and synonymous changes were 60.0 and 22.6 for ω = 1. Finally, estimated numbers of nonsynonymous and synonymous changes were 308.7 and 93.0, respectively, for crown-group dasypodids, whereas expected numbers or nonsynonymous and synonymous changes were 291.8 and 109.9 for ω = 1. All PAML analyses were run with the CodonFreq = 3 option in PAML.</p> </div> <div class="moleculardatinganalyses" title="sec"> <div class="title" tagx="title" title="title">
Molecular Dating Analyses</div> <p>The relaxed molecular clock method implemented in <i>Multidivtime</i> (version 9-25-03) <a href="#pgen.1000634-Thorne1">[64]</a>–<a href="#pgen.1000634-Kishino1">[66]</a> was used to estimate divergence times for several pairs of taxa that lack published molecular estimates of divergence times: <i>Manis tricuspis</i> and <i>M. pentadactyla</i>; <i>Choloepus didactylus</i> and <i>C. hoffmanni</i>; <i>Heterohyrax brucei</i> and <i>Procavia capensis</i>; and <i>Amblysomus hottentotus</i> and <i>Chrysochloris asiatica</i>. We used the phylogeny in <a href="#pgen-1000634-g001">Figure 1</a> with the following constraints: (1) minimum of 50 mya and maximum of 63 mya for the split between Feliformia and Caniformia <a href="#pgen.1000634-Springer2">[67]</a>; (2) minimum of 54 mya and maximum of 65 mya for the base of Paenungulata <a href="#pgen.1000634-Springer2">[67]</a>; (3) minimum of 52 mya for the base of the hippo and cetacean clade <a href="#pgen.1000634-Waddell1">[68]</a> and a maximum of 65 mya given uncertain relationships between cetartiodactyls and Paleocene mesonychids <a href="#pgen.1000634-OLeary1">[21]</a>.</p> </div> <div class="calculationsofsubstitutionratesinmysticetes" title="sec"> <div class="title" tagx="title" title="title">
Calculations of Substitution Rates in Mysticetes</div> <p>We calculated the neutral substitution rate in crown-group mysticetes as 82.6 substitutions/2760 nucleotide sites/90 myr = 3.3×10<sup>−3</sup> substitutions/site/myr, where 82.6 is the PAML estimate for the total number of substitutions in <i>ENAM</i> on crown-group mysticete branches, 2760 is the number of nucleotide sites in the PAML alignment, and 90 myr is the total duration of crown-group mysticete branches based on McGowen <i>et al.</i><a href="#pgen.1000634-McGowen1">[69]</a>. Similarly, we calculated the rate of frameshift accumulation in mysticetes as 2 frameshifts/2.76 kb/90 myr = 0.0081 frameshifts/kb/myr, where 2 is the minimum number of frameshifts in mysticete <i>ENAM</i> (acctran reconstruction).</p> </div> <div class="resolvingmixedbranchesintofunctionalandpseudogeniccomponents" title="sec"> <div class="title" tagx="title" title="title">
Resolving Mixed Branches into Functional and Pseudogenic Components</div> <p>The mixed branches leading to <i>Orycteropus</i> and to the two species of <i>Manis</i> were assumed to have mixed histories that included both functional and pseudogenic components. For these branches, the lengths of time that <i>ENAM</i> was functional versus pseudogenic were estimated under two different assumptions. First, we assumed that rates of synonymous substitution are neutral and equal on functional and pseudogene branches <a href="#pgen.1000634-Wolfe1">[26]</a>,<a href="#pgen.1000634-Kumar1">[27]</a>. Second, we assumed that the rate of synonymous substitution on functional branches is non-neutral and is 70% of the rate of synonymous substitution on pseudogene branches following Bustamante <i>et al</i>. <a href="#pgen.1000634-Bustamante1">[28]</a>. Possible reasons for a lower rate of synonymous substitution on functional branches include constraints associated with codon usage, splicing, and mRNA stability <a href="#pgen.1000634-Chamary1">[70]</a>.</p> <p>For a single rate of synonymous substitution on functional and pseudogenic branches, the following equation describes the relationship between the lengths of time that <i>ENAM</i> was functional versus pseudogenic on the mixed branch: </p><div class="disp-formula" title="disp-formula"> <div class="graphic" mimetype="image" position="float" /><span class="label" tagx="label" title="label">(1)</span></div>where T is the length of the mixed branch in millions of years, ω<sub>m</sub> is the dN/dS estimate for the mixed branch, T<sub>f</sub> is the amount of time that <i>ENAM</i> was functional on the mixed branch, ω<sub>f</sub> is the dN/dS estimate for functional branches, T<sub>p</sub>  =  (T − T<sub>f</sub>) is the amount of time that <i>ENAM</i> was pseudogenic on the mixed branch, and ω<sub>p</sub> is the dN/dS estimate for pseudogenic branches. T<sub>f</sub> and T<sub>p</sub> are then obtained by rearrangement and substitution: <div class="disp-formula" title="disp-formula"> <div class="graphic" mimetype="image" position="float" /><span class="label" tagx="label" title="label">(2)</span></div> <div class="disp-formula" title="disp-formula"> <div class="graphic" mimetype="image" position="float" /><span class="label" tagx="label" title="label">(3)</span></div> <p /> <p>In the case of two different synonymous substitution rates, one for functional branches and the other for pseudogenic branches, the lengths of time that <i>ENAM</i> was functional versus pseudogenic were estimated using the following equations: </p><div class="disp-formula" title="disp-formula"> <div class="graphic" mimetype="image" position="float" /><span class="label" tagx="label" title="label">(4)</span></div> <div class="disp-formula" title="disp-formula"> <div class="graphic" mimetype="image" position="float" /><span class="label" tagx="label" title="label">(5)</span></div>where T<sub>f(2)</sub> is the amount of time that <i>ENAM</i> was functional on the mixed branch when there are two synonymous rates, T<sub>p(2)</sub> is the amount of time that <i>ENAM</i> was pseudogenic on the mixed branch when there are two synonymous rates, dS<sub>f</sub> is the synonymous (non-neutral) rate of substitution on the functional segment of the mixed branch, dS<sub>p</sub> is the synonymous (neutral) rate of substitution on the pseudogenic segment of the mixed branch, and T<sub>f</sub> is from equation 2. The denominator in equation 4 adjusts for differences in rates of synonymous substitution on functional and pseudogenic segments of the mixed branch, and also introduces a normalization correction so that T  =  T<sub>f(2)</sub> + T<sub>p(2)</sub>. In the case of dS<sub>f</sub>  =  dS<sub>p</sub>, the denominator in equation 4 equals 1 and the equation reduces to T<sub>f(2)</sub>  =  T<sub>f</sub>.<p /> <p>Our approach for dating the transition from functional gene to pseudogene is conceptually related to methods that are based on the overall substitution rate <a href="#pgen.1000634-Li1">[71]</a> and the rate of indel accumulation <a href="#pgen.1000634-Saitou1">[72]</a>,<a href="#pgen.1000634-Springer3">[73]</a>, respectively.</p> </div> <div class="ancestralreconstructions" title="sec"> <div class="title" tagx="title" title="title">
Ancestral Reconstructions</div> <p>Ancestral reconstructions for the presence or absence of enamel (<a href="#pgen.1000634.s005">Figure S3</a>) were performed with SIMMAP Version 1.0 B2.3.2 <a href="#pgen.1000634-Bollback1">[74]</a>.</p> </div> <div class="calculationsofsurvivaltimeunderneutralevolution" title="sec"> <div class="title" tagx="title" title="title">
Calculations of Survival Time under Neutral Evolution</div> <p>Equation 4 from Marshall <i>et al.</i><a href="#pgen.1000634-Marshall1">[46]</a> was used to estimate the probability that a three kb exon will remain functional after one million years of neutral evolution. Calculations were performed using the very slow rate of nucleotide substitution that we calculated for mysticetes (r<sub>s</sub> = 3.3×10<sup>−4</sup> substitutions per site per myr) and Marshall <i>et al</i>.'s <a href="#pgen.1000634-Marshall1">[46]</a> slow, intermediate, and fast rates of nucleotide substitution (<i>r</i><sub>s</sub> = 1×10<sup>−3</sup>, 5×10<sup>−3</sup>, and 10×10<sup>−3</sup> substitutions per site per myr); very slow (i.e., mysticetes), slow, intermediate, and fast rates of frameshift accumulation (<i>r</i><sub>f</sub> = 0.0081, 0.014, 0.093, and 0.287 frameshifts per kb per myr); and with two different values of <i>f</i> (0.3, 0.7), where <i>f</i> is the probability that a substitution will not result in loss of function and the chosen values represent endpoints of Marshall <i>et al</i>.'s <a href="#pgen.1000634-Marshall1">[46]</a> "realistic range" for this variable.</p> </div> </div> <div class="supplementary-material" title="supplementary-material"> <div class="title" tagx="title" title="title">
Supporting Information</div> <div class="local-data" title="local-data"><span class="label" tagx="label" title="label">Dataset S1</span> <p>Complete nexus alignment of DNA sequences for <i>ENAM.</i></p> <p>(0.32 MB TXT)</p>  <div class="media" title="media"><a href="pgen.1000634.s001.txt">LINK</a> <p>Click here for additional data file.</p>  </div> </div> <div class="local-data" title="local-data"><span class="label" tagx="label" title="label">Dataset S2</span> <p>Complete nexus alignment of DNA sequences used in PAML analyses.</p> <p>(0.21 MB TXT)</p>  <div class="media" title="media"><a href="pgen.1000634.s002.txt">LINK</a> <p>Click here for additional data file.</p>  </div> </div> <div class="local-data" title="local-data"><span class="label" tagx="label" title="label">Figure S1</span> <p>ML phylogram based on 4089 bp alignment (<a href="#pgen.1000634.s001">Dataset S1</a>). Marsupial outgroups are not shown.</p> <p>(1.72 MB TIF)</p>  <div class="media" title="media"><a href="pgen.1000634.s003.tif">LINK</a> <p>Click here for additional data file.</p>  </div> </div> <div class="local-data" title="local-data"><span class="label" tagx="label" title="label">Figure S2</span> <p>Maximum posterior probability tree with Bayesian posterior probabilities above branches and ML bootstrap support percentages below branches. Bayesian and ML analyses were performed with the 4089 bp alignment (<a href="#pgen.1000634.s001">Dataset S1</a>).</p> <p>(1.81 MB TIF)</p>  <div class="media" title="media"><a href="pgen.1000634.s004.tif">LINK</a> <p>Click here for additional data file.</p>  </div> </div> <div class="local-data" title="local-data"><span class="label" tagx="label" title="label">Figure S3</span> <p>Ancestral state reconstructions (enamel present or enamel absent) for internal nodes based on SIMMAP (Version 1.0 B2.3.2) <a href="#pgen.1000634-Bollback1">[74]</a>. Parsimony optimization (not shown) agreed with the most probable ML reconstruction. Functional branches lead to extant taxa and internal nodes with enamel. Pre-mutation, mixed, and pseudogene branches lead to extant taxa and internal nodes without enamel. Pre-mutation branches predate the first detected occurrence of a frameshift mutation or stop codon in <i>ENAM</i>. Mixed branches record the first detected occurrence of a frameshift or stop codon in <i>ENAM</i>. Pseudogene branches postdate the first detected occurrence of a frameshift or stop codon in <i>ENAM</i>. Branch colors are as in <a href="#pgen-1000634-g001">Figure 1</a>.</p> <p>(1.95 MB TIF)</p>  <div class="media" title="media"><a href="pgen.1000634.s005.tif">LINK</a> <p>Click here for additional data file.</p>  </div> </div> <div class="local-data" title="local-data"><span class="label" tagx="label" title="label">Figure S4</span> <p>Alternative hypotheses for the loss of enamel in Xenarthra given a basal position for the fossil armadillo <i>Utaetus</i> relative to living armadillos. Deltran parsimony optimization favors the dual loss hypothesis wherein enamel was lost independently in Pilosa and Dasypodidae; acctran parsimony optimization favors loss of enamel in the common ancestor of Xenarthra followed by gain of this feature in <i>Utaetus</i>. A dN/dS ratio of 0.48 suggests that <i>ENAM</i> evolved under purifying selection on the stem branch leading to crown Xenarthra and was a functional gene at this stage in its evolutionary history. Parsimony reconstruction of the partial <i>ENAM</i> sequence for the most recent common ancestor of Xenarthra also suggests that <i>ENAM</i> was functional (i.e., no frameshift mutations or stop codons in reconstructed ancestral sequence). Taken together, the dN/dS ratio for <i>ENAM</i> on the stem xenarthran branch and the reconstructed ancestral <i>ENAM</i> sequence in the last common ancestor of Xenarthra provide support for the dual loss hypothesis. Additional observations from genomics suggest that enamel may have been lost independently in more than one armadillo lineage (see text).</p> <p>(0.74 MB TIF)</p>  <div class="media" title="media"><a href="pgen.1000634.s006.tif">LINK</a> <p>Click here for additional data file.</p>  </div> </div> <div class="local-data" title="local-data"><span class="label" tagx="label" title="label">Figure S5</span> <p>Representative chromatograms showing frameshift insertions and deletions in edentulous and enamelless taxa.</p> <p>(0.29 MB PDF)</p>  <div class="media" title="media"><a href="pgen.1000634.s007.pdf">LINK</a> <p>Click here for additional data file.</p>  </div> </div> <div class="local-data" title="local-data"><span class="label" tagx="label" title="label">Figure S6</span> <p>Pairwise amino acid alignments between <i>Sus</i> and <i>Dasypus</i> for the <i>AMELX, AMBN, ENAM,</i> and <i>MMP20</i> genes.</p> <p>(0.04 MB DOC)</p>  <div class="media" title="media"><a href="pgen.1000634.s008.doc">LINK</a> <p>Click here for additional data file.</p>  </div> </div> <div class="local-data" title="local-data"><span class="label" tagx="label" title="label">Figure S7</span> <p>Primer sequences and location of primer sequences in relationship to nearly complete exonic sequences that were assembled from Ensembl 51 and trace files. F designates forward primers; R designates reverse primers.</p> <p>(0.74 MB DOC)</p>  <div class="media" title="media"><a href="pgen.1000634.s009.doc">LINK</a> <p>Click here for additional data file.</p>  </div> </div> <div class="local-data" title="local-data"><span class="label" tagx="label" title="label">Figure S8</span> <p>Illustration of <i>ENAM</i> segments that amplified (black) and failed to amplify (white) for edentulous and enamelless taxa.</p> <p>(0.87 MB TIF)</p>  <div class="media" title="media"><a href="pgen.1000634.s010.tif">LINK</a> <p>Click here for additional data file.</p>  </div> </div> <div class="local-data" title="local-data"><span class="label" tagx="label" title="label">Table S1</span> <p>Comprehensive list of frameshift insertions and deletions in edentulous and enamelless taxa. Numbering of insertions and deletions corresponds to the alignment in <a href="#pgen.1000634.s001">Dataset S1</a>. Numbers in red font indicate the minimum to maximum number of frameshift indels.</p> <p>(0.03 MB DOC)</p>  <div class="media" title="media"><a href="pgen.1000634.s011.doc">LINK</a> <p>Click here for additional data file.</p>  </div> </div> <div class="local-data" title="local-data"><span class="label" tagx="label" title="label">Text S1</span> <p>Supplementary note on enamel in (a) sperm whales and (b) xenarthrans.</p> <p>(0.07 MB DOC)</p>  <div class="media" title="media"><a href="pgen.1000634.s012.doc">LINK</a> <p>Click here for additional data file.</p>  </div> </div> </div> </div> <div class="back" title="back"> <div class="ack" title="ack"> <p>We thank N. Ayoub, T. Gaudin, S. Gatesy, C. Hayashi, M. Lee, M. McGowen, A. de Queiroz, K. Springer, and two anonymous reviewers for helpful comments. Southwest Fisheries Science Center (specimen numbers Z505, Z3859, Z5750, Z9834, Z10119, Z10124, Z13086, Z15224, Z18928, Z28452, Z35275, Z38274), New York Zoological Society, Northeast Fisheries Science Center Stranding Network, Smithsonian Institution Division of Mammals, Mote Marine Lab, South Australian Museum, G. Braulik (World Wildlife Fund), H. Rosenbaum, U. Arnason, and M. Milinkovitch kindly provided DNA samples.</p> </div> <div class="fn-group" title="fn-group"> <div class="fn-type-COI-statement" title="COI-statement"> <p>The authors have declared that no competing interests exist.</p> </div> <div class="fn-type-financial-disclosure" title="financial-disclosure"> <p>This work was supported in part by the National Science Foundation (EF06298660 to MSS and JG, DEB-0743724 to JG, and EF0629849 to WJM. The funders had no role in study design, data collection and analysis, decision to publish, or preparation of the manuscript.</p> </div> </div> <div class="references">
References</div> <div tag="ref-list"> <ul> <div class="title" tagx="title" title="title">
References</div> <li tag="ref"><a name="pgen.1000634-Futuyma1" /><span class="label" tagx="label" title="label">1</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Futuyma</span><span class="given-names" tagx="given-names" title="given-names">DJ</span></span></span><span class="year" tagx="year" title="year">1998</span><span class="mixed-article-title" title="mixed-article-title">Evolutionary biology.</span><span class="publisher-loc" tagx="publisher-loc" title="publisher-loc">Sunderland</span><span class="publisher-name" tagx="publisher-name" title="publisher-name">Sinauer</span></span></li> <li tag="ref"><a name="pgen.1000634-Darwin1" /><span class="label" tagx="label" title="label">2</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Darwin</span><span class="given-names" tagx="given-names" title="given-names">C</span></span></span><span class="year" tagx="year" title="year">1859</span><span class="mixed-article-title" title="mixed-article-title">On the origin of species by means of natural selection, or the preservation of favoured races in the struggle for life.</span><span class="publisher-loc" tagx="publisher-loc" title="publisher-loc">London</span><span class="publisher-name" tagx="publisher-name" title="publisher-name">John Murray</span></span></li> <li tag="ref"><a name="pgen.1000634-Darwin2" /><span class="label" tagx="label" title="label">3</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Darwin</span><span class="given-names" tagx="given-names" title="given-names">C</span></span></span><span class="year" tagx="year" title="year">1871</span><span class="mixed-article-title" title="mixed-article-title">The descent of man, and selection in relation to sex.</span><span class="publisher-loc" tagx="publisher-loc" title="publisher-loc">London</span><span class="publisher-name" tagx="publisher-name" title="publisher-name">John Murray</span></span></li> <li tag="ref"><a name="pgen.1000634-Bedjer1" /><span class="label" tagx="label" title="label">4</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bedjer</span><span class="given-names" tagx="given-names" title="given-names">L</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hall</span><span class="given-names" tagx="given-names" title="given-names">BK</span></span></span><span class="year" tagx="year" title="year">2002</span><span class="mixed-article-title" title="mixed-article-title">Limbs in whales and limblessness in other vertebrates: mechanisms of evolutionary and developmental transformation and loss.</span><span class="source" tagx="source" title="source">Evol Dev</span><span class="volume" tagx="volume" title="volume">4</span><span class="fpage" tagx="fpage" title="fpage">445</span><span class="lpage" tagx="lpage" title="lpage">458</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12492145">12492145</a></span></span></li> <li tag="ref"><a name="pgen.1000634-Fong1" /><span class="label" tagx="label" title="label">5</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Fong</span><span class="given-names" tagx="given-names" title="given-names">DW</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kane</span><span class="given-names" tagx="given-names" title="given-names">TC</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Culver</span><span class="given-names" tagx="given-names" title="given-names">DC</span></span></span><span class="year" tagx="year" title="year">1995</span><span class="mixed-article-title" title="mixed-article-title">Vestigialization and loss of nonfunctional characters.</span><span class="source" tagx="source" title="source">Annu Rev Ecol Syst</span><span class="volume" tagx="volume" title="volume">26</span><span class="fpage" tagx="fpage" title="fpage">249</span><span class="lpage" tagx="lpage" title="lpage">268</span></span></li> <li tag="ref"><a name="pgen.1000634-Nishikimi1" /><span class="label" tagx="label" title="label">6</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Nishikimi</span><span class="given-names" tagx="given-names" title="given-names">M</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Fukuyama</span><span class="given-names" tagx="given-names" title="given-names">R</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Minoshima</span><span class="given-names" tagx="given-names" title="given-names">S</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Shimizu</span><span class="given-names" tagx="given-names" title="given-names">N</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Yagi</span><span class="given-names" tagx="given-names" title="given-names">K</span></span></span><span class="year" tagx="year" title="year">1994</span><span class="mixed-article-title" title="mixed-article-title">Cloning and chromosomal mapping of the human nonfunctional gene for L-gulono-γ-lactone oxidase, the enzyme for L-ascorbic acid biosynthesis missing in man.</span><span class="source" tagx="source" title="source">J Biol Chem</span><span class="volume" tagx="volume" title="volume">269</span><span class="fpage" tagx="fpage" title="fpage">13685</span><span class="lpage" tagx="lpage" title="lpage">13688</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/8175804">8175804</a></span></span></li> <li tag="ref"><a name="pgen.1000634-Ohta1" /><span class="label" tagx="label" title="label">7</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ohta</span><span class="given-names" tagx="given-names" title="given-names">Y</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Nishikimi</span><span class="given-names" tagx="given-names" title="given-names">M</span></span></span><span class="year" tagx="year" title="year">1999</span><span class="mixed-article-title" title="mixed-article-title">Random nucleotide substitutions in primate nonfunctional gene for L-gulono-γ-lactone oxidase, the missing enzyme in L-ascorbic acid biosynthesis.</span><span class="source" tagx="source" title="source">Biochim Biophys Acta</span><span class="volume" tagx="volume" title="volume">1472</span><span class="fpage" tagx="fpage" title="fpage">408</span><span class="lpage" tagx="lpage" title="lpage">411</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/10572964">10572964</a></span></span></li> <li tag="ref"><a name="pgen.1000634-Hu1" /><span class="label" tagx="label" title="label">8</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hu</span><span class="given-names" tagx="given-names" title="given-names">JC-C</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Chun</span><span class="given-names" tagx="given-names" title="given-names">Y-HP</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Al Hazzazzi</span><span class="given-names" tagx="given-names" title="given-names">T</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Simmer</span><span class="given-names" tagx="given-names" title="given-names">JP</span></span></span><span class="year" tagx="year" title="year">2007</span><span class="mixed-article-title" title="mixed-article-title">Enamel formation and amelogenesis imperfecta.</span><span class="source" tagx="source" title="source">Cells Tissues Organs</span><span class="volume" tagx="volume" title="volume">186</span><span class="fpage" tagx="fpage" title="fpage">78</span><span class="lpage" tagx="lpage" title="lpage">85</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17627121">17627121</a></span></span></li> <li tag="ref"><a name="pgen.1000634-Wright1" /><span class="label" tagx="label" title="label">9</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wright</span><span class="given-names" tagx="given-names" title="given-names">JT</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hart</span><span class="given-names" tagx="given-names" title="given-names">TC</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hart</span><span class="given-names" tagx="given-names" title="given-names">PS</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Simmons</span><span class="given-names" tagx="given-names" title="given-names">D</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Suggs</span><span class="given-names" tagx="given-names" title="given-names">C</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="year" tagx="year" title="year">2009</span><span class="mixed-article-title" title="mixed-article-title">Human and mouse enamel phenotypes resulting from mutation or altered expression of <i>AMEL</i>, <i>ENAM</i>, <i>MMP20</i> and <i>KLK4</i>.</span><span class="source" tagx="source" title="source">Cells Tissues Organs</span><span class="volume" tagx="volume" title="volume">189</span><span class="fpage" tagx="fpage" title="fpage">224</span><span class="lpage" tagx="lpage" title="lpage">229</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18714142">18714142</a></span></span></li> <li tag="ref"><a name="pgen.1000634-Kawasaki1" /><span class="label" tagx="label" title="label">10</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kawasaki</span><span class="given-names" tagx="given-names" title="given-names">K</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Weiss</span><span class="given-names" tagx="given-names" title="given-names">KM</span></span></span><span class="year" tagx="year" title="year">2003</span><span class="mixed-article-title" title="mixed-article-title">Mineralized tissue and vertebrate evolution: the secretory calcium-binding phosphoprotein gene cluster.</span><span class="source" tagx="source" title="source">Proc Natl Acad Sci U S A</span><span class="volume" tagx="volume" title="volume">100</span><span class="fpage" tagx="fpage" title="fpage">4060</span><span class="lpage" tagx="lpage" title="lpage">4065</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12646701">12646701</a></span></span></li> <li tag="ref"><a name="pgen.1000634-Sire1" /><span class="label" tagx="label" title="label">11</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Sire</span><span class="given-names" tagx="given-names" title="given-names">J-Y</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Davit-Beal</span><span class="given-names" tagx="given-names" title="given-names">T</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Delgado</span><span class="given-names" tagx="given-names" title="given-names">S</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gu</span><span class="given-names" tagx="given-names" title="given-names">X</span></span></span><span class="year" tagx="year" title="year">2007</span><span class="mixed-article-title" title="mixed-article-title">The origin and evolution of enamel mineralization genes.</span><span class="source" tagx="source" title="source">Cells Tissues Organs</span><span class="volume" tagx="volume" title="volume">186</span><span class="fpage" tagx="fpage" title="fpage">25</span><span class="lpage" tagx="lpage" title="lpage">48</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17627117">17627117</a></span></span></li> <li tag="ref"><a name="pgen.1000634-Kawasaki2" /><span class="label" tagx="label" title="label">12</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kawasaki</span><span class="given-names" tagx="given-names" title="given-names">K</span></span></span><span class="year" tagx="year" title="year">2009</span><span class="mixed-article-title" title="mixed-article-title">The SCPP gene repertoire in bony vertebrates and graded differences in mineralized tissues.</span><span class="source" tagx="source" title="source">Dev Genes Evol</span><span class="volume" tagx="volume" title="volume">219</span><span class="fpage" tagx="fpage" title="fpage">147</span><span class="lpage" tagx="lpage" title="lpage">157</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19255778">19255778</a></span></span></li> <li tag="ref"><a name="pgen.1000634-Hu2" /><span class="label" tagx="label" title="label">13</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hu</span><span class="given-names" tagx="given-names" title="given-names">C-C</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Fukae</span><span class="given-names" tagx="given-names" title="given-names">M</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Uchida</span><span class="given-names" tagx="given-names" title="given-names">T</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Qian</span><span class="given-names" tagx="given-names" title="given-names">Q</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zhang</span><span class="given-names" tagx="given-names" title="given-names">CH</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="year" tagx="year" title="year">1997</span><span class="mixed-article-title" title="mixed-article-title">Cloning and characterization of porcine enamelin mRNAs.</span><span class="source" tagx="source" title="source">J Dent Res</span><span class="volume" tagx="volume" title="volume">76</span><span class="fpage" tagx="fpage" title="fpage">1720</span><span class="lpage" tagx="lpage" title="lpage">1729</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/9372788">9372788</a></span></span></li> <li tag="ref"><a name="pgen.1000634-Hu3" /><span class="label" tagx="label" title="label">14</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hu</span><span class="given-names" tagx="given-names" title="given-names">JC-C</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hu</span><span class="given-names" tagx="given-names" title="given-names">Y</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Smith</span><span class="given-names" tagx="given-names" title="given-names">CE</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">McKee</span><span class="given-names" tagx="given-names" title="given-names">MD</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wright</span><span class="given-names" tagx="given-names" title="given-names">JT</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="year" tagx="year" title="year">2008</span><span class="mixed-article-title" title="mixed-article-title">Enamel defects and ameloblast-specific expression in <i>Enam</i> knock-out/<i>lacZ</i> knock-in mice.</span><span class="source" tagx="source" title="source">J Biol Chem</span><span class="volume" tagx="volume" title="volume">283</span><span class="fpage" tagx="fpage" title="fpage">10858</span><span class="lpage" tagx="lpage" title="lpage">10871</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18252720">18252720</a></span></span></li> <li tag="ref"><a name="pgen.1000634-Demr1" /><span class="label" tagx="label" title="label">15</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Deméré</span><span class="given-names" tagx="given-names" title="given-names">TA</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">McGowen</span><span class="given-names" tagx="given-names" title="given-names">MR</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Berta</span><span class="given-names" tagx="given-names" title="given-names">A</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gatesy</span><span class="given-names" tagx="given-names" title="given-names">J</span></span></span><span class="year" tagx="year" title="year">2008</span><span class="mixed-article-title" title="mixed-article-title">Morphological and molecular evidence for a stepwise evolutionary transition from teeth to baleen in mysticete whales.</span><span class="source" tagx="source" title="source">Syst Biol</span><span class="volume" tagx="volume" title="volume">57</span><span class="fpage" tagx="fpage" title="fpage">15</span><span class="lpage" tagx="lpage" title="lpage">37</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18266181">18266181</a></span></span></li> <li tag="ref"><a name="pgen.1000634-Osborn1" /><span class="label" tagx="label" title="label">16</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Osborn</span><span class="given-names" tagx="given-names" title="given-names">HF</span></span></span><span class="year" tagx="year" title="year">1893</span><span class="mixed-article-title" title="mixed-article-title">Recent researches upon the succession of teeth in mammals.</span><span class="source" tagx="source" title="source">Amer Nat</span><span class="volume" tagx="volume" title="volume">27</span><span class="fpage" tagx="fpage" title="fpage">493</span><span class="lpage" tagx="lpage" title="lpage">508</span></span></li> <li tag="ref"><a name="pgen.1000634-Spurgin1" /><span class="label" tagx="label" title="label">17</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Spurgin</span><span class="given-names" tagx="given-names" title="given-names">AM</span></span></span><span class="year" tagx="year" title="year">1904</span><span class="mixed-article-title" title="mixed-article-title">Enamel in the teeth of an embryo edentate (<i>Dasypus novemcinctus</i> Linn).</span><span class="source" tagx="source" title="source">Amer J Anat</span><span class="volume" tagx="volume" title="volume">3</span><span class="fpage" tagx="fpage" title="fpage">75</span><span class="lpage" tagx="lpage" title="lpage">84</span></span></li> <li tag="ref"><a name="pgen.1000634-Murphy1" /><span class="label" tagx="label" title="label">18</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Murphy</span><span class="given-names" tagx="given-names" title="given-names">WJ</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Eizirik</span><span class="given-names" tagx="given-names" title="given-names">E</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">O'Brien</span><span class="given-names" tagx="given-names" title="given-names">SJ</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Madsen</span><span class="given-names" tagx="given-names" title="given-names">O</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Scally</span><span class="given-names" tagx="given-names" title="given-names">M</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="year" tagx="year" title="year">2001</span><span class="mixed-article-title" title="mixed-article-title">Resolution of the early placental mammal radiation using Bayesian phylogenetics.</span><span class="source" tagx="source" title="source">Science</span><span class="volume" tagx="volume" title="volume">294</span><span class="fpage" tagx="fpage" title="fpage">2348</span><span class="lpage" tagx="lpage" title="lpage">2351</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11743200">11743200</a></span></span></li> <li tag="ref"><a name="pgen.1000634-Springer1" /><span class="label" tagx="label" title="label">19</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Springer</span><span class="given-names" tagx="given-names" title="given-names">MS</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Burk-Herrick</span><span class="given-names" tagx="given-names" title="given-names">A</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Meredith</span><span class="given-names" tagx="given-names" title="given-names">R</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Eizirik</span><span class="given-names" tagx="given-names" title="given-names">E</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Teeling</span><span class="given-names" tagx="given-names" title="given-names">E</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="year" tagx="year" title="year">2007</span><span class="mixed-article-title" title="mixed-article-title">The adequacy of morphology for reconstructing the early history of placental mammals.</span><span class="source" tagx="source" title="source">Syst Biol</span><span class="volume" tagx="volume" title="volume">56</span><span class="fpage" tagx="fpage" title="fpage">673</span><span class="lpage" tagx="lpage" title="lpage">684</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17661234">17661234</a></span></span></li> <li tag="ref"><a name="pgen.1000634-Delsuc1" /><span class="label" tagx="label" title="label">20</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Delsuc</span><span class="given-names" tagx="given-names" title="given-names">F</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Vizcaino</span><span class="given-names" tagx="given-names" title="given-names">SF</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Douzery</span><span class="given-names" tagx="given-names" title="given-names">EJP</span></span></span><span class="year" tagx="year" title="year">2004</span><span class="mixed-article-title" title="mixed-article-title">Influence of Tertiary paleoenvironmental changes on the diversification of South American mammals: a relaxed molecular clock study within xenarthrans.</span><span class="source" tagx="source" title="source">BMC Evol Biol</span><span class="volume" tagx="volume" title="volume">4</span><span class="fpage" tagx="fpage" title="fpage">11</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15115541">15115541</a></span></span></li> <li tag="ref"><a name="pgen.1000634-OLeary1" /><span class="label" tagx="label" title="label">21</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">O'Leary</span><span class="given-names" tagx="given-names" title="given-names">MA</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gatesy</span><span class="given-names" tagx="given-names" title="given-names">J</span></span></span><span class="year" tagx="year" title="year">2008</span><span class="mixed-article-title" title="mixed-article-title">Impact of increased character sampling on the phylogeny of Cetartiodactyla (Mammalia): combined analysis including fossils.</span><span class="source" tagx="source" title="source">Cladistics</span><span class="volume" tagx="volume" title="volume">24</span><span class="fpage" tagx="fpage" title="fpage">397</span><span class="lpage" tagx="lpage" title="lpage">442</span></span></li> <li tag="ref"><a name="pgen.1000634-Holroyd1" /><span class="label" tagx="label" title="label">22</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Holroyd</span><span class="given-names" tagx="given-names" title="given-names">PA</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Mussell</span><span class="given-names" tagx="given-names" title="given-names">JC</span></span></span><span class="year" tagx="year" title="year">2005</span><span class="mixed-article-title" title="mixed-article-title">Macroscelidea and Tubulidentata.</span><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Rose</span><span class="given-names" tagx="given-names" title="given-names">KD</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Archibald</span><span class="given-names" tagx="given-names" title="given-names">JD</span></span></span><span class="source" tagx="source" title="source">The rise of placental mammals</span><span class="publisher-loc" tagx="publisher-loc" title="publisher-loc">Baltimore</span><span class="publisher-name" tagx="publisher-name" title="publisher-name">The Johns Hopkins Univ Press</span><span class="fpage" tagx="fpage" title="fpage">71</span><span class="lpage" tagx="lpage" title="lpage">83</span></span></li> <li tag="ref"><a name="pgen.1000634-Murphy2" /><span class="label" tagx="label" title="label">23</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Murphy</span><span class="given-names" tagx="given-names" title="given-names">WJ</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Eizirik</span><span class="given-names" tagx="given-names" title="given-names">E</span></span></span><span class="year" tagx="year" title="year">2009</span><span class="mixed-article-title" title="mixed-article-title">Placental mammals (Eutheria).</span><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hedges</span><span class="given-names" tagx="given-names" title="given-names">SB</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kumar</span><span class="given-names" tagx="given-names" title="given-names">S</span></span></span><span class="source" tagx="source" title="source">The timetree of life</span><span class="publisher-loc" tagx="publisher-loc" title="publisher-loc">Oxford</span><span class="publisher-name" tagx="publisher-name" title="publisher-name">Oxford Univ Press</span><span class="fpage" tagx="fpage" title="fpage">471</span><span class="lpage" tagx="lpage" title="lpage">474</span></span></li> <li tag="ref"><a name="pgen.1000634-Patterson1" /><span class="label" tagx="label" title="label">24</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Patterson</span><span class="given-names" tagx="given-names" title="given-names">B</span></span></span><span class="year" tagx="year" title="year">1975</span><span class="mixed-article-title" title="mixed-article-title">The fossil aardvarks (Mammalia: Tubulidentata).</span><span class="source" tagx="source" title="source">Bull Mus Comp Zool</span><span class="volume" tagx="volume" title="volume">147</span><span class="fpage" tagx="fpage" title="fpage">185</span><span class="lpage" tagx="lpage" title="lpage">237</span></span></li> <li tag="ref"><a name="pgen.1000634-Milledge1" /><span class="label" tagx="label" title="label">25</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Milledge</span><span class="given-names" tagx="given-names" title="given-names">S</span></span></span><span class="year" tagx="year" title="year">2003</span><span class="mixed-article-title" title="mixed-article-title">Fossil aardvarks from the Lothagam beds.</span><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Leakey</span><span class="given-names" tagx="given-names" title="given-names">J</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Harris</span><span class="given-names" tagx="given-names" title="given-names">J</span></span></span><span class="source" tagx="source" title="source">Lothagam: The dawn of humanity in Eastern Africa</span><span class="publisher-loc" tagx="publisher-loc" title="publisher-loc">New York</span><span class="publisher-name" tagx="publisher-name" title="publisher-name">Columbia University Press</span><span class="fpage" tagx="fpage" title="fpage">363</span><span class="lpage" tagx="lpage" title="lpage">368</span></span></li> <li tag="ref"><a name="pgen.1000634-Wolfe1" /><span class="label" tagx="label" title="label">26</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wolfe</span><span class="given-names" tagx="given-names" title="given-names">KH</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Sharp</span><span class="given-names" tagx="given-names" title="given-names">PM</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Li</span><span class="given-names" tagx="given-names" title="given-names">W-H</span></span></span><span class="year" tagx="year" title="year">1989</span><span class="mixed-article-title" title="mixed-article-title">Mutation rates differ among regions of the mammalian genome.</span><span class="source" tagx="source" title="source">Nature</span><span class="volume" tagx="volume" title="volume">337</span><span class="fpage" tagx="fpage" title="fpage">283</span><span class="lpage" tagx="lpage" title="lpage">285</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/2911369">2911369</a></span></span></li> <li tag="ref"><a name="pgen.1000634-Kumar1" /><span class="label" tagx="label" title="label">27</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kumar</span><span class="given-names" tagx="given-names" title="given-names">S</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Subramanian</span><span class="given-names" tagx="given-names" title="given-names">S</span></span></span><span class="year" tagx="year" title="year">2002</span><span class="mixed-article-title" title="mixed-article-title">Mutation rates in mammalian genomes.</span><span class="source" tagx="source" title="source">Proc Natl Acad Sci U S A</span><span class="volume" tagx="volume" title="volume">99</span><span class="fpage" tagx="fpage" title="fpage">803</span><span class="lpage" tagx="lpage" title="lpage">808</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11792858">11792858</a></span></span></li> <li tag="ref"><a name="pgen.1000634-Bustamante1" /><span class="label" tagx="label" title="label">28</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bustamante</span><span class="given-names" tagx="given-names" title="given-names">CD</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Nielsen</span><span class="given-names" tagx="given-names" title="given-names">R</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hartl</span><span class="given-names" tagx="given-names" title="given-names">DL</span></span></span><span class="year" tagx="year" title="year">2002</span><span class="mixed-article-title" title="mixed-article-title">A maximum likelihood method analyzing pseudogene evolution: implications for silent site evolution in humans and rodents.</span><span class="source" tagx="source" title="source">Mol Biol Evol</span><span class="volume" tagx="volume" title="volume">19</span><span class="fpage" tagx="fpage" title="fpage">110</span><span class="lpage" tagx="lpage" title="lpage">117</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11752196">11752196</a></span></span></li> <li tag="ref"><a name="pgen.1000634-Franzen1" /><span class="label" tagx="label" title="label">29</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Franzen</span><span class="given-names" tagx="given-names" title="given-names">JL</span></span></span><span class="year" tagx="year" title="year">2005</span><span class="mixed-article-title" title="mixed-article-title">The implications of the numerical dating of the Messel fossil deposit (Eocene, Germany) for mammalian biochronology.</span><span class="source" tagx="source" title="source">Annales de Paleontologie</span><span class="volume" tagx="volume" title="volume">91</span><span class="fpage" tagx="fpage" title="fpage">329</span><span class="lpage" tagx="lpage" title="lpage">335</span></span></li> <li tag="ref"><a name="pgen.1000634-Emry1" /><span class="label" tagx="label" title="label">30</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Emry</span><span class="given-names" tagx="given-names" title="given-names">RJ</span></span></span><span class="year" tagx="year" title="year">1970</span><span class="mixed-article-title" title="mixed-article-title">A North American Oligocene pangolin and other additions to the Pholidota.</span><span class="source" tagx="source" title="source">Bull Amer Mus Nat Hist</span><span class="volume" tagx="volume" title="volume">142</span><span class="fpage" tagx="fpage" title="fpage">455</span><span class="lpage" tagx="lpage" title="lpage">510</span></span></li> <li tag="ref"><a name="pgen.1000634-Rose1" /><span class="label" tagx="label" title="label">31</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Rose</span><span class="given-names" tagx="given-names" title="given-names">KD</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Emry</span><span class="given-names" tagx="given-names" title="given-names">RJ</span></span></span><span class="year" tagx="year" title="year">1993</span><span class="mixed-article-title" title="mixed-article-title">Relationships of Xenarthra, Pholidota, and fossil "edentates": the morphological evidence.</span><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Szalay</span><span class="given-names" tagx="given-names" title="given-names">FS</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Novacek</span><span class="given-names" tagx="given-names" title="given-names">MJ</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">McKenna</span><span class="given-names" tagx="given-names" title="given-names">MC</span></span></span><span class="source" tagx="source" title="source">Mammal phylogeny: Placentals</span><span class="publisher-loc" tagx="publisher-loc" title="publisher-loc">New York</span><span class="publisher-name" tagx="publisher-name" title="publisher-name">Springer-Verlag</span><span class="fpage" tagx="fpage" title="fpage">81</span><span class="lpage" tagx="lpage" title="lpage">102</span></span></li> <li tag="ref"><a name="pgen.1000634-Simpson1" /><span class="label" tagx="label" title="label">32</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Simpson</span><span class="given-names" tagx="given-names" title="given-names">GG</span></span></span><span class="year" tagx="year" title="year">1931</span><span class="mixed-article-title" title="mixed-article-title"><i>Metacheiromys</i> and the Edentata.</span><span class="source" tagx="source" title="source">Bull Amer Mus Nat Hist</span><span class="volume" tagx="volume" title="volume">59</span><span class="fpage" tagx="fpage" title="fpage">295</span><span class="lpage" tagx="lpage" title="lpage">381</span></span></li> <li tag="ref"><a name="pgen.1000634-Patterson2" /><span class="label" tagx="label" title="label">33</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Patterson</span><span class="given-names" tagx="given-names" title="given-names">B</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Segall</span><span class="given-names" tagx="given-names" title="given-names">W</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Turnbull</span><span class="given-names" tagx="given-names" title="given-names">W</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gaudin</span><span class="given-names" tagx="given-names" title="given-names">T</span></span></span><span class="year" tagx="year" title="year">1992</span><span class="mixed-article-title" title="mixed-article-title">The ear region in xenarthrans ( = Edentata: Mammalia). Part II. Pilosa (sloths, anteaters), palaeanodonts, and a miscellany.</span><span class="source" tagx="source" title="source">Fieldiana Geol new series</span><span class="volume" tagx="volume" title="volume">24</span><span class="fpage" tagx="fpage" title="fpage">1</span><span class="lpage" tagx="lpage" title="lpage">79</span></span></li> <li tag="ref"><a name="pgen.1000634-Matthew1" /><span class="label" tagx="label" title="label">34</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Matthew</span><span class="given-names" tagx="given-names" title="given-names">WD</span></span></span><span class="year" tagx="year" title="year">1918</span><span class="mixed-article-title" title="mixed-article-title">A revision of the lower Eocene Wasatch and Wind River faunas. Part V. Insectivora (continued), Glires, Edentata.</span><span class="source" tagx="source" title="source">Bull Amer Mus Nat Hist</span><span class="volume" tagx="volume" title="volume">38</span><span class="fpage" tagx="fpage" title="fpage">565</span><span class="lpage" tagx="lpage" title="lpage">657</span></span></li> <li tag="ref"><a name="pgen.1000634-Patterson3" /><span class="label" tagx="label" title="label">35</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Patterson</span><span class="given-names" tagx="given-names" title="given-names">B</span></span></span><span class="year" tagx="year" title="year">1978</span><span class="mixed-article-title" title="mixed-article-title">Pholidota and Tubulidentata.</span><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Maglio</span><span class="given-names" tagx="given-names" title="given-names">VJ</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Cook</span><span class="given-names" tagx="given-names" title="given-names">HBS</span></span></span><span class="source" tagx="source" title="source">Evolution of African mammals</span><span class="publisher-loc" tagx="publisher-loc" title="publisher-loc">Cambridge</span><span class="publisher-name" tagx="publisher-name" title="publisher-name">Harvard Univ Press</span><span class="fpage" tagx="fpage" title="fpage">268</span><span class="lpage" tagx="lpage" title="lpage">278</span></span></li> <li tag="ref"><a name="pgen.1000634-Rose2" /><span class="label" tagx="label" title="label">36</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Rose</span><span class="given-names" tagx="given-names" title="given-names">KD</span></span></span><span class="year" tagx="year" title="year">1978</span><span class="mixed-article-title" title="mixed-article-title">A new Paleocene epoicotheriid (Mammalia), with comments on the Palaeanodonta.</span><span class="source" tagx="source" title="source">J Paleontol</span><span class="volume" tagx="volume" title="volume">52</span><span class="fpage" tagx="fpage" title="fpage">658</span><span class="lpage" tagx="lpage" title="lpage">674</span></span></li> <li tag="ref"><a name="pgen.1000634-Rose3" /><span class="label" tagx="label" title="label">37</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Rose</span><span class="given-names" tagx="given-names" title="given-names">KD</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Emry</span><span class="given-names" tagx="given-names" title="given-names">RJ</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gaudin</span><span class="given-names" tagx="given-names" title="given-names">TJ</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Storch</span><span class="given-names" tagx="given-names" title="given-names">G</span></span></span><span class="year" tagx="year" title="year">2005</span><span class="mixed-article-title" title="mixed-article-title">Xenarthra and Pholidota.</span><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Rose</span><span class="given-names" tagx="given-names" title="given-names">KD</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Archibald</span><span class="given-names" tagx="given-names" title="given-names">JD</span></span></span><span class="source" tagx="source" title="source">The rise of placental mammals</span><span class="publisher-loc" tagx="publisher-loc" title="publisher-loc">Baltimore</span><span class="publisher-name" tagx="publisher-name" title="publisher-name"> The Johns Hopkins University Press</span><span class="fpage" tagx="fpage" title="fpage">106</span><span class="lpage" tagx="lpage" title="lpage">126</span></span></li> <li tag="ref"><a name="pgen.1000634-Secord1" /><span class="label" tagx="label" title="label">38</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Secord</span><span class="given-names" tagx="given-names" title="given-names">R</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gingerich</span><span class="given-names" tagx="given-names" title="given-names">PD</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bloch</span><span class="given-names" tagx="given-names" title="given-names">JI</span></span></span><span class="year" tagx="year" title="year">2002</span><span class="mixed-article-title" title="mixed-article-title"><i>Mylanodon rosei</i>, a new metacheiromyid (Mammalia, Palaeanodonta) from the late Tiffanian (late Paleocene) of northwestern Wyoming.</span><span class="source" tagx="source" title="source">Contrib Mus Paleontol Univ Michigan</span><span class="volume" tagx="volume" title="volume">30</span><span class="fpage" tagx="fpage" title="fpage">385</span><span class="lpage" tagx="lpage" title="lpage">399</span></span></li> <li tag="ref"><a name="pgen.1000634-Gheerbrant1" /><span class="label" tagx="label" title="label">39</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gheerbrant</span><span class="given-names" tagx="given-names" title="given-names">E</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Rose</span><span class="given-names" tagx="given-names" title="given-names">KD</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Godinot</span><span class="given-names" tagx="given-names" title="given-names">M</span></span></span><span class="year" tagx="year" title="year">2005</span><span class="mixed-article-title" title="mixed-article-title">First palaeanodont (?pholidotan) mammal from the Eocene of Europe.</span><span class="source" tagx="source" title="source">Acta Palaeontol Polon</span><span class="volume" tagx="volume" title="volume">50</span><span class="fpage" tagx="fpage" title="fpage">209</span><span class="lpage" tagx="lpage" title="lpage">218</span></span></li> <li tag="ref"><a name="pgen.1000634-Bianucci1" /><span class="label" tagx="label" title="label">40</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bianucci</span><span class="given-names" tagx="given-names" title="given-names">G</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Landini</span><span class="given-names" tagx="given-names" title="given-names">W</span></span></span><span class="year" tagx="year" title="year">2006</span><span class="mixed-article-title" title="mixed-article-title">Killer sperm whale: a new basal physeteroid (Mammalia, Cetecea) from the Late Miocene of Italy.</span><span class="source" tagx="source" title="source">Zool J Linn Soc</span><span class="volume" tagx="volume" title="volume">148</span><span class="fpage" tagx="fpage" title="fpage">103</span><span class="lpage" tagx="lpage" title="lpage">131</span></span></li> <li tag="ref"><a name="pgen.1000634-Fitzgerald1" /><span class="label" tagx="label" title="label">41</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Fitzgerald</span><span class="given-names" tagx="given-names" title="given-names">EM</span></span></span><span class="year" tagx="year" title="year">2006</span><span class="mixed-article-title" title="mixed-article-title">A bizarre new toothed mysticete (Cetacea) from Australia and the early evolution of baleen whales.</span><span class="source" tagx="source" title="source">Proc R Soc B</span><span class="volume" tagx="volume" title="volume">273</span><span class="fpage" tagx="fpage" title="fpage">2955</span><span class="lpage" tagx="lpage" title="lpage">2963</span></span></li> <li tag="ref"><a name="pgen.1000634-Ishiyama1" /><span class="label" tagx="label" title="label">42</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ishiyama</span><span class="given-names" tagx="given-names" title="given-names">M</span></span></span><span class="year" tagx="year" title="year">1987</span><span class="mixed-article-title" title="mixed-article-title">Enamel structure in odontocete whales.</span><span class="source" tagx="source" title="source">Scanning Microscopy</span><span class="volume" tagx="volume" title="volume">1</span><span class="fpage" tagx="fpage" title="fpage">1071</span><span class="lpage" tagx="lpage" title="lpage">1079</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/3659855">3659855</a></span></span></li> <li tag="ref"><a name="pgen.1000634-Murphy3" /><span class="label" tagx="label" title="label">43</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Murphy</span><span class="given-names" tagx="given-names" title="given-names">WJ</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Pringle</span><span class="given-names" tagx="given-names" title="given-names">TH</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Crider</span><span class="given-names" tagx="given-names" title="given-names">TA</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Springer</span><span class="given-names" tagx="given-names" title="given-names">MS</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Miller</span><span class="given-names" tagx="given-names" title="given-names">W</span></span></span><span class="year" tagx="year" title="year">2007</span><span class="mixed-article-title" title="mixed-article-title">Using genomic data to unravel the root of the placental mammal phylogeny.</span><span class="source" tagx="source" title="source">Gen Res</span><span class="volume" tagx="volume" title="volume">17</span><span class="fpage" tagx="fpage" title="fpage">413</span><span class="lpage" tagx="lpage" title="lpage">421</span></span></li> <li tag="ref"><a name="pgen.1000634-Simpson2" /><span class="label" tagx="label" title="label">44</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Simpson</span><span class="given-names" tagx="given-names" title="given-names">GG</span></span></span><span class="year" tagx="year" title="year">1948</span><span class="mixed-article-title" title="mixed-article-title">The beginning of the age of mammals in South America.</span><span class="source" tagx="source" title="source">Bull Amer Mus Nat Hist</span><span class="volume" tagx="volume" title="volume">91</span><span class="fpage" tagx="fpage" title="fpage">1</span><span class="lpage" tagx="lpage" title="lpage">232</span></span></li> <li tag="ref"><a name="pgen.1000634-Lee1" /><span class="label" tagx="label" title="label">45</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lee</span><span class="given-names" tagx="given-names" title="given-names">MSY</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Shine</span><span class="given-names" tagx="given-names" title="given-names">R</span></span></span><span class="year" tagx="year" title="year">1998</span><span class="mixed-article-title" title="mixed-article-title">Reptilian viviparity and Dollo's law.</span><span class="source" tagx="source" title="source">Evolution</span><span class="volume" tagx="volume" title="volume">52</span><span class="fpage" tagx="fpage" title="fpage">1441</span><span class="lpage" tagx="lpage" title="lpage">1450</span></span></li> <li tag="ref"><a name="pgen.1000634-Marshall1" /><span class="label" tagx="label" title="label">46</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Marshall</span><span class="given-names" tagx="given-names" title="given-names">CR</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Raff</span><span class="given-names" tagx="given-names" title="given-names">EC</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Raff</span><span class="given-names" tagx="given-names" title="given-names">RA</span></span></span><span class="year" tagx="year" title="year">1994</span><span class="mixed-article-title" title="mixed-article-title">Dollo's law and the death and resurrection of genes.</span><span class="source" tagx="source" title="source">Proc Natl Acad Sci U S A</span><span class="volume" tagx="volume" title="volume">91</span><span class="fpage" tagx="fpage" title="fpage">12283</span><span class="lpage" tagx="lpage" title="lpage">12287</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/7991619">7991619</a></span></span></li> <li tag="ref"><a name="pgen.1000634-Kollar1" /><span class="label" tagx="label" title="label">47</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kollar</span><span class="given-names" tagx="given-names" title="given-names">EJ</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Fisher</span><span class="given-names" tagx="given-names" title="given-names">C</span></span></span><span class="year" tagx="year" title="year">1980</span><span class="mixed-article-title" title="mixed-article-title">Tooth induction in chick epithelium: expression of quiescent genes for enamel synthesis.</span><span class="source" tagx="source" title="source">Science</span><span class="volume" tagx="volume" title="volume">207</span><span class="fpage" tagx="fpage" title="fpage">993</span><span class="lpage" tagx="lpage" title="lpage">995</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/7352302">7352302</a></span></span></li> <li tag="ref"><a name="pgen.1000634-Sire2" /><span class="label" tagx="label" title="label">48</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Sire</span><span class="given-names" tagx="given-names" title="given-names">J-Y</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Delgado</span><span class="given-names" tagx="given-names" title="given-names">SC</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Girondot</span><span class="given-names" tagx="given-names" title="given-names">M</span></span></span><span class="year" tagx="year" title="year">2008</span><span class="mixed-article-title" title="mixed-article-title">Hen's teeth with enamel cap: from dream to impossibility.</span><span class="source" tagx="source" title="source">BMC Evol Biol</span><span class="volume" tagx="volume" title="volume">8</span><span class="fpage" tagx="fpage" title="fpage">246</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18775069">18775069</a></span></span></li> <li tag="ref"><a name="pgen.1000634-Jackman1" /><span class="label" tagx="label" title="label">49</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Jackman</span><span class="given-names" tagx="given-names" title="given-names">WR</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Stock</span><span class="given-names" tagx="given-names" title="given-names">DW</span></span></span><span class="year" tagx="year" title="year">2006</span><span class="mixed-article-title" title="mixed-article-title">Transgenic analysis of Dlx regulation in fish tooth development reveals evolutionary retention of enhancer function despite organ loss.</span><span class="source" tagx="source" title="source">Proc Natl Acad Sci U S A</span><span class="volume" tagx="volume" title="volume">103</span><span class="fpage" tagx="fpage" title="fpage">19390</span><span class="lpage" tagx="lpage" title="lpage">19395</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17146045">17146045</a></span></span></li> <li tag="ref"><a name="pgen.1000634-Portz1" /><span class="label" tagx="label" title="label">50</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Portz</span><span class="given-names" tagx="given-names" title="given-names">DE</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Tyus</span><span class="given-names" tagx="given-names" title="given-names">HM</span></span></span><span class="year" tagx="year" title="year">2004</span><span class="mixed-article-title" title="mixed-article-title">Fish humps in two Colorado River fishes: a morphological response to cyprinid predation?</span><span class="source" tagx="source" title="source">Environ Biol Fishes</span><span class="volume" tagx="volume" title="volume">71</span><span class="fpage" tagx="fpage" title="fpage">233</span><span class="lpage" tagx="lpage" title="lpage">245</span></span></li> <li tag="ref"><a name="pgen.1000634-Stock1" /><span class="label" tagx="label" title="label">51</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Stock</span><span class="given-names" tagx="given-names" title="given-names">DW</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Jackman</span><span class="given-names" tagx="given-names" title="given-names">WR</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Trapani</span><span class="given-names" tagx="given-names" title="given-names">J</span></span></span><span class="year" tagx="year" title="year">2006</span><span class="mixed-article-title" title="mixed-article-title">Developmental genetic mechanisms of evolutionary tooth loss in cypriniform fishes.</span><span class="source" tagx="source" title="source">Development</span><span class="volume" tagx="volume" title="volume">133</span><span class="fpage" tagx="fpage" title="fpage">3127</span><span class="lpage" tagx="lpage" title="lpage">3137</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16831836">16831836</a></span></span></li> <li tag="ref"><a name="pgen.1000634-Werdelin1" /><span class="label" tagx="label" title="label">52</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Werdelin</span><span class="given-names" tagx="given-names" title="given-names">L</span></span></span><span class="year" tagx="year" title="year">1987</span><span class="mixed-article-title" title="mixed-article-title">Supernumerary teeth in <i>Lynx lynx</i> and the irreversibility of evolution.</span><span class="source" tagx="source" title="source">J Zool</span><span class="volume" tagx="volume" title="volume">211</span><span class="fpage" tagx="fpage" title="fpage">259</span><span class="lpage" tagx="lpage" title="lpage">266</span></span></li> <li tag="ref"><a name="pgen.1000634-Gosline1" /><span class="label" tagx="label" title="label">53</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gosline</span><span class="given-names" tagx="given-names" title="given-names">WA</span></span></span><span class="year" tagx="year" title="year">1985</span><span class="mixed-article-title" title="mixed-article-title">A possible relationship between aspects of dentition and feeding in the centrarchid and anabantoid fishes.</span><span class="source" tagx="source" title="source">Environ Biol Fishes</span><span class="volume" tagx="volume" title="volume">12</span><span class="fpage" tagx="fpage" title="fpage">161</span><span class="lpage" tagx="lpage" title="lpage">168</span></span></li> <li tag="ref"><a name="pgen.1000634-Stock2" /><span class="label" tagx="label" title="label">54</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Stock</span><span class="given-names" tagx="given-names" title="given-names">DW</span></span></span><span class="year" tagx="year" title="year">2001</span><span class="mixed-article-title" title="mixed-article-title">The genetic basis of modularity in the development and evolution of the vertebrate dentition.</span><span class="source" tagx="source" title="source">Phil Trans R Soc Lond B</span><span class="volume" tagx="volume" title="volume">356</span><span class="fpage" tagx="fpage" title="fpage">1633</span><span class="lpage" tagx="lpage" title="lpage">1653</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11604128">11604128</a></span></span></li> <li tag="ref"><a name="pgen.1000634-Hart1" /><span class="label" tagx="label" title="label">55</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hart</span><span class="given-names" tagx="given-names" title="given-names">PS</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Michalec</span><span class="given-names" tagx="given-names" title="given-names">MD</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Seow</span><span class="given-names" tagx="given-names" title="given-names">WK</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hart</span><span class="given-names" tagx="given-names" title="given-names">TC</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wright</span><span class="given-names" tagx="given-names" title="given-names">JT</span></span></span><span class="year" tagx="year" title="year">2003</span><span class="mixed-article-title" title="mixed-article-title">Identification of the enamelin (g.8344delG) mutation in a new kindred and presentation of a standardized ENAM nomenclature.</span><span class="source" tagx="source" title="source">Arch Oral Biol</span><span class="volume" tagx="volume" title="volume">48</span><span class="fpage" tagx="fpage" title="fpage">589</span><span class="lpage" tagx="lpage" title="lpage">596</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12828988">12828988</a></span></span></li> <li tag="ref"><a name="pgen.1000634-Larkin1" /><span class="label" tagx="label" title="label">56</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Larkin</span><span class="given-names" tagx="given-names" title="given-names">MA</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="year" tagx="year" title="year">2007</span><span class="mixed-article-title" title="mixed-article-title">Clustal W and Clustal X version 2.0.</span><span class="source" tagx="source" title="source">Bioinformatics</span><span class="volume" tagx="volume" title="volume">23</span><span class="fpage" tagx="fpage" title="fpage">2947</span><span class="lpage" tagx="lpage" title="lpage">2948</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17846036">17846036</a></span></span></li> <li tag="ref"><a name="pgen.1000634-Rambaut1" /><span class="label" tagx="label" title="label">57</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Rambaut</span><span class="given-names" tagx="given-names" title="given-names">A</span></span></span><span class="year" tagx="year" title="year">1996</span><span class="mixed-article-title" title="mixed-article-title">Se-Al: Sequence Alignment Editor, 2.0a11.</span><span class="comment"><a href="http://evolve.zoo.ox.ac.uk">http://evolve.zoo.ox.ac.uk</a></span></span></li> <li tag="ref"><a name="pgen.1000634-Posada1" /><span class="label" tagx="label" title="label">58</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Posada</span><span class="given-names" tagx="given-names" title="given-names">D</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Crandall</span><span class="given-names" tagx="given-names" title="given-names">KA</span></span></span><span class="year" tagx="year" title="year">1998</span><span class="mixed-article-title" title="mixed-article-title">MODELTEST: testing the model of DNA substitution.</span><span class="source" tagx="source" title="source">Bioinformatics</span><span class="volume" tagx="volume" title="volume">14</span><span class="fpage" tagx="fpage" title="fpage">817</span><span class="lpage" tagx="lpage" title="lpage">818</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/9918953">9918953</a></span></span></li> <li tag="ref"><a name="pgen.1000634-Stamatakis1" /><span class="label" tagx="label" title="label">59</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Stamatakis</span><span class="given-names" tagx="given-names" title="given-names">A</span></span></span><span class="year" tagx="year" title="year">2006</span><span class="mixed-article-title" title="mixed-article-title">RAxML-VI-HPC: Maximum likelihood-based phylogenetic analyses with thousands of taxa and mixed models.</span><span class="source" tagx="source" title="source">Bioinformatics</span><span class="volume" tagx="volume" title="volume">22</span><span class="fpage" tagx="fpage" title="fpage">2688</span><span class="lpage" tagx="lpage" title="lpage">2690</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16928733">16928733</a></span></span></li> <li tag="ref"><a name="pgen.1000634-Ronquist1" /><span class="label" tagx="label" title="label">60</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ronquist</span><span class="given-names" tagx="given-names" title="given-names">F</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Huelsenbeck</span><span class="given-names" tagx="given-names" title="given-names">JP</span></span></span><span class="year" tagx="year" title="year">2003</span><span class="mixed-article-title" title="mixed-article-title">MrBayes 3: Bayesian phylogenetic inference under mixed models.</span><span class="source" tagx="source" title="source">Bioinformatics</span><span class="volume" tagx="volume" title="volume">19</span><span class="fpage" tagx="fpage" title="fpage">1572</span><span class="lpage" tagx="lpage" title="lpage">1574</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12912839">12912839</a></span></span></li> <li tag="ref"><a name="pgen.1000634-Huelsenbeck1" /><span class="label" tagx="label" title="label">61</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Huelsenbeck</span><span class="given-names" tagx="given-names" title="given-names">JP</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ronquist</span><span class="given-names" tagx="given-names" title="given-names">F</span></span></span><span class="year" tagx="year" title="year">2001</span><span class="mixed-article-title" title="mixed-article-title">MRBAYES: Bayesian inference of phylogenetic trees.</span><span class="source" tagx="source" title="source">Bioinformatics</span><span class="volume" tagx="volume" title="volume">17</span><span class="fpage" tagx="fpage" title="fpage">754</span><span class="lpage" tagx="lpage" title="lpage">755</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11524383">11524383</a></span></span></li> <li tag="ref"><a name="pgen.1000634-Yang1" /><span class="label" tagx="label" title="label">62</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Yang</span><span class="given-names" tagx="given-names" title="given-names">Z</span></span></span><span class="year" tagx="year" title="year">2007</span><span class="mixed-article-title" title="mixed-article-title">PAML 4: a program package for phylogenetic analysis by maximum likelihood.</span><span class="source" tagx="source" title="source">Mol Biol Evol</span><span class="volume" tagx="volume" title="volume">24</span><span class="fpage" tagx="fpage" title="fpage">1586</span><span class="lpage" tagx="lpage" title="lpage">1591</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17483113">17483113</a></span></span></li> <li tag="ref"><a name="pgen.1000634-Gatesy1" /><span class="label" tagx="label" title="label">63</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gatesy</span><span class="given-names" tagx="given-names" title="given-names">J</span></span></span><span class="year" tagx="year" title="year">2009</span><span class="mixed-article-title" title="mixed-article-title">Whales and even-toed ungulates (Cetartiodactyla).</span><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hedges</span><span class="given-names" tagx="given-names" title="given-names">SB</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kumar</span><span class="given-names" tagx="given-names" title="given-names">S</span></span></span><span class="source" tagx="source" title="source">The timetree of life</span><span class="publisher-loc" tagx="publisher-loc" title="publisher-loc">Oxford</span><span class="publisher-name" tagx="publisher-name" title="publisher-name">Oxford Univ Press</span><span class="fpage" tagx="fpage" title="fpage">511</span><span class="lpage" tagx="lpage" title="lpage">515</span></span></li> <li tag="ref"><a name="pgen.1000634-Thorne1" /><span class="label" tagx="label" title="label">64</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Thorne</span><span class="given-names" tagx="given-names" title="given-names">JL</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kishino</span><span class="given-names" tagx="given-names" title="given-names">H</span></span></span><span class="year" tagx="year" title="year">2002</span><span class="mixed-article-title" title="mixed-article-title">Divergence time and evolutionary rate estimation with multilocus data.</span><span class="source" tagx="source" title="source">Syst Biol</span><span class="volume" tagx="volume" title="volume">51</span><span class="fpage" tagx="fpage" title="fpage">689</span><span class="lpage" tagx="lpage" title="lpage">702</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12396584">12396584</a></span></span></li> <li tag="ref"><a name="pgen.1000634-Thorne2" /><span class="label" tagx="label" title="label">65</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Thorne</span><span class="given-names" tagx="given-names" title="given-names">JL</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kishino</span><span class="given-names" tagx="given-names" title="given-names">H</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Painter</span><span class="given-names" tagx="given-names" title="given-names">IS</span></span></span><span class="year" tagx="year" title="year">1998</span><span class="mixed-article-title" title="mixed-article-title">Estimating the rate of evolution of the rate of molecular evolution.</span><span class="source" tagx="source" title="source">Mol Biol Evol</span><span class="volume" tagx="volume" title="volume">15</span><span class="fpage" tagx="fpage" title="fpage">1647</span><span class="lpage" tagx="lpage" title="lpage">1657</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/9866200">9866200</a></span></span></li> <li tag="ref"><a name="pgen.1000634-Kishino1" /><span class="label" tagx="label" title="label">66</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kishino</span><span class="given-names" tagx="given-names" title="given-names">H</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Thorne</span><span class="given-names" tagx="given-names" title="given-names">JL</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bruno</span><span class="given-names" tagx="given-names" title="given-names">WJ</span></span></span><span class="year" tagx="year" title="year">2001</span><span class="mixed-article-title" title="mixed-article-title">Performance of a divergence time estimation method under a probabilistic model of rate evolution.</span><span class="source" tagx="source" title="source">Mol Biol Evol</span><span class="volume" tagx="volume" title="volume">18</span><span class="fpage" tagx="fpage" title="fpage">352</span><span class="lpage" tagx="lpage" title="lpage">361</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11230536">11230536</a></span></span></li> <li tag="ref"><a name="pgen.1000634-Springer2" /><span class="label" tagx="label" title="label">67</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Springer</span><span class="given-names" tagx="given-names" title="given-names">MS</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Murphy</span><span class="given-names" tagx="given-names" title="given-names">WJ</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Eizirik</span><span class="given-names" tagx="given-names" title="given-names">E</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">O'Brien</span><span class="given-names" tagx="given-names" title="given-names">SJ</span></span></span><span class="year" tagx="year" title="year">2003</span><span class="mixed-article-title" title="mixed-article-title">Placental mammal diversification and the Cretaceous-Tertiary boundary.</span><span class="source" tagx="source" title="source">Proc Natl Acad Sci U S A</span><span class="volume" tagx="volume" title="volume">100</span><span class="fpage" tagx="fpage" title="fpage">1056</span><span class="lpage" tagx="lpage" title="lpage">1061</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12552136">12552136</a></span></span></li> <li tag="ref"><a name="pgen.1000634-Waddell1" /><span class="label" tagx="label" title="label">68</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Waddell</span><span class="given-names" tagx="given-names" title="given-names">PJ</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kishino</span><span class="given-names" tagx="given-names" title="given-names">H</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ota</span><span class="given-names" tagx="given-names" title="given-names">R</span></span></span><span class="year" tagx="year" title="year">2001</span><span class="mixed-article-title" title="mixed-article-title">A phylogenetic foundation for comparative mammalian genomics.</span><span class="source" tagx="source" title="source">Genome Inform</span><span class="volume" tagx="volume" title="volume">12</span><span class="fpage" tagx="fpage" title="fpage">141</span><span class="lpage" tagx="lpage" title="lpage">154</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11791233">11791233</a></span></span></li> <li tag="ref"><a name="pgen.1000634-McGowen1" /><span class="label" tagx="label" title="label">69</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">McGowen</span><span class="given-names" tagx="given-names" title="given-names">MR</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Spaulding</span><span class="given-names" tagx="given-names" title="given-names">M</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gatesy</span><span class="given-names" tagx="given-names" title="given-names">J</span></span></span><span class="year" tagx="year" title="year">2009</span><span class="mixed-article-title" title="mixed-article-title">Divergence date estimation and a comprehensive molecular tree of extant cetaceans.</span><span class="source" tagx="source" title="source">Mol Phylogenet Evol</span><span class="comment">In press</span></span></li> <li tag="ref"><a name="pgen.1000634-Chamary1" /><span class="label" tagx="label" title="label">70</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Chamary</span><span class="given-names" tagx="given-names" title="given-names">JV</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Parmley</span><span class="given-names" tagx="given-names" title="given-names">JL</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hurst</span><span class="given-names" tagx="given-names" title="given-names">LD</span></span></span><span class="year" tagx="year" title="year">2006</span><span class="mixed-article-title" title="mixed-article-title">Hearing silence: non-neutral evolution at synonymous sites in mammals.</span><span class="source" tagx="source" title="source">Nature Genet</span><span class="volume" tagx="volume" title="volume">7</span><span class="fpage" tagx="fpage" title="fpage">98</span><span class="lpage" tagx="lpage" title="lpage">108</span></span></li> <li tag="ref"><a name="pgen.1000634-Li1" /><span class="label" tagx="label" title="label">71</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Li</span><span class="given-names" tagx="given-names" title="given-names">W-H</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gojobori</span><span class="given-names" tagx="given-names" title="given-names">T</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Nei</span><span class="given-names" tagx="given-names" title="given-names">M</span></span></span><span class="year" tagx="year" title="year">1981</span><span class="mixed-article-title" title="mixed-article-title">Pseudogenes as a paradigm of neutral evolution.</span><span class="source" tagx="source" title="source">Nature</span><span class="volume" tagx="volume" title="volume">292</span><span class="fpage" tagx="fpage" title="fpage">237</span><span class="lpage" tagx="lpage" title="lpage">239</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/7254315">7254315</a></span></span></li> <li tag="ref"><a name="pgen.1000634-Saitou1" /><span class="label" tagx="label" title="label">72</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Saitou</span><span class="given-names" tagx="given-names" title="given-names">N</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ueda</span><span class="given-names" tagx="given-names" title="given-names">S</span></span></span><span class="year" tagx="year" title="year">1994</span><span class="mixed-article-title" title="mixed-article-title">Evolutionary rates of insertion and deletion in noncoding nucleotide sequences of Primates.</span><span class="source" tagx="source" title="source">Mol Biol Evol</span><span class="volume" tagx="volume" title="volume">11</span><span class="fpage" tagx="fpage" title="fpage">504</span><span class="lpage" tagx="lpage" title="lpage">512</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/8015443">8015443</a></span></span></li> <li tag="ref"><a name="pgen.1000634-Springer3" /><span class="label" tagx="label" title="label">73</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Springer</span><span class="given-names" tagx="given-names" title="given-names">MS</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Burk</span><span class="given-names" tagx="given-names" title="given-names">A</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kavanagh</span><span class="given-names" tagx="given-names" title="given-names">JR</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Waddell</span><span class="given-names" tagx="given-names" title="given-names">VG</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Stanhope</span><span class="given-names" tagx="given-names" title="given-names">MJ</span></span></span><span class="year" tagx="year" title="year">1997</span><span class="mixed-article-title" title="mixed-article-title">The interphotoreceptor retinoid binding protein gene in therian mammals: Implications for higher level relationships and evidence for loss of function in the marsupial mole.</span><span class="source" tagx="source" title="source">Proc Natl Acad Sci U S A</span><span class="volume" tagx="volume" title="volume">94</span><span class="fpage" tagx="fpage" title="fpage">13754</span><span class="lpage" tagx="lpage" title="lpage">13759</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/9391099">9391099</a></span></span></li> <li tag="ref"><a name="pgen.1000634-Bollback1" /><span class="label" tagx="label" title="label">74</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bollback</span><span class="given-names" tagx="given-names" title="given-names">JP</span></span></span><span class="year" tagx="year" title="year">2006</span><span class="mixed-article-title" title="mixed-article-title">SIMMAP: stochastic character mapping of discrete traits on phylogenies.</span><span class="source" tagx="source" title="source">BMC Bioinf</span><span class="volume" tagx="volume" title="volume">7</span><span class="fpage" tagx="fpage" title="fpage">88</span></span></li> <li tag="ref"><a name="pgen.1000634-Shockey1" /><span class="label" tagx="label" title="label">75</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Shockey</span><span class="given-names" tagx="given-names" title="given-names">BJ</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Flynn</span><span class="given-names" tagx="given-names" title="given-names">JJ</span></span></span><span class="year" tagx="year" title="year">2007</span><span class="mixed-article-title" title="mixed-article-title">Morphological diversity in the postcranial skeleton of Casamayoran (?Middle to Late Eocene) Notoungulata and foot posture in notoungulates.</span><span class="source" tagx="source" title="source">Amer Mus Novitat</span><span class="volume" tagx="volume" title="volume">3601</span><span class="fpage" tagx="fpage" title="fpage">1</span><span class="lpage" tagx="lpage" title="lpage">28</span></span></li> <li tag="ref"><a name="pgen.1000634-Croft1" /><span class="label" tagx="label" title="label">76</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Croft</span><span class="given-names" tagx="given-names" title="given-names">D</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Flynn</span><span class="given-names" tagx="given-names" title="given-names">JF</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wyss</span><span class="given-names" tagx="given-names" title="given-names">AR</span></span></span><span class="year" tagx="year" title="year">2008</span><span class="mixed-article-title" title="mixed-article-title">The Tinguiririca fauna of Chile and the early stages of "modernization" of South American mammal faunas.</span><span class="source" tagx="source" title="source">Arquivos do Museu Nacional, Rio de Janeiro</span><span class="volume" tagx="volume" title="volume">66</span><span class="fpage" tagx="fpage" title="fpage">191</span><span class="lpage" tagx="lpage" title="lpage">211</span></span></li> </ul> </div> </div>  </body></html>