<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<link rel="dns-prefetch" href="//d33xdlntwy0kbs.cloudfront.net" />
<link rel="shortcut icon" href="https://www.biorxiv.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<meta name="type" content="article" />
<meta name="category" content="article" />
<meta name="HW.identifier" content="/biorxiv/early/2019/12/13/2019.12.13.870410.atom" />
<meta name="HW.pisa" content="biorxiv;2019.12.13.870410v1" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="mtDNA-based reconstructions of change in effective population sizes of Holarctic birds do not agree with their reconstructed range sizes based on paleoclimates" />
<meta name="DC.Identifier" content="10.1101/2019.12.13.870410" />
<meta name="DC.Date" content="2019-12-13" />
<meta name="DC.Publisher" content="Cold Spring Harbor Laboratory" />
<meta name="DC.Rights" content="© 2019, Posted by Cold Spring Harbor Laboratory. This pre-print is available under a Creative Commons License (Attribution-NonCommercial-NoDerivs 4.0 International), CC BY-NC-ND 4.0, as described at http://creativecommons.org/licenses/by-nc-nd/4.0/" />
<meta name="DC.AccessRights" content="restricted" />
<meta name="DC.Description" content="During the Quaternary, large climate oscillations had profound impacts on the distribution, demography and diversity of species globally. Birds offer a special opportunity for studying these impacts because surveys of geographical distributions, publicly-available genetic sequence data, and the existence of species with adaptations to life in structurally different habitats, permit large-scale comparative analyses. We use Bayesian Skyline Plot (BSP) analysis of mitochondrial DNA to reconstruct profiles depicting how effective population size ( N e) may have changed over time, focussing on variation in the effect of the last deglaciation among 102 Holarctic species. Only 3 species showed a decline in N e since the Last Glacial Maximum (LGM) and 7 showed no sizeable change, whilst 92 profiles revealed an increase in N e. Using bioclimatic Species Distribution Models (SDMs), we also estimated changes in species potential range extent since the LGM. Whilst most modelled ranges also increased, we found no correlation across species between the magnitude of change in range size and change in N e. The lack of correlation between SDM and BSP reconstructions could not be reconciled even when range shifts were considered. We suggest the lack of agreement between these measures might be linked to changes in population densities which can be independent of range changes. We caution that interpreting either SDM or BSPs independently is problematic and potentially misleading. Additionally, we found that N e of wetland species tended to increase later than species from terrestrial habitats, possibly reflecting a delayed increase in the extent of this habitat type after the LGM." />
<meta name="DC.Contributor" content="Eleanor F. Miller" />
<meta name="DC.Contributor" content="Rhys E. Green" />
<meta name="DC.Contributor" content="Andrew Balmford" />
<meta name="DC.Contributor" content="Robert Beyer" />
<meta name="DC.Contributor" content="Marius Somveille" />
<meta name="DC.Contributor" content="Michela Leonardi" />
<meta name="DC.Contributor" content="William Amos" />
<meta name="DC.Contributor" content="Andrea Manica" />
<meta name="article:published_time" content="2019-12-13" />
<meta name="article:section" content="New Results" />
<meta name="citation_title" content="mtDNA-based reconstructions of change in effective population sizes of Holarctic birds do not agree with their reconstructed range sizes based on paleoclimates" />
<meta name="citation_abstract" lang="en" content="&lt;h3&gt;Abstract&lt;/h3&gt;
&lt;p&gt;During the Quaternary, large climate oscillations had profound impacts on the distribution, demography and diversity of species globally. Birds offer a special opportunity for studying these impacts because surveys of geographical distributions, publicly-available genetic sequence data, and the existence of species with adaptations to life in structurally different habitats, permit large-scale comparative analyses. We use Bayesian Skyline Plot (BSP) analysis of mitochondrial DNA to reconstruct profiles depicting how effective population size (&lt;i&gt;N&lt;/i&gt;&lt;sub&gt;e&lt;/sub&gt;) may have changed over time, focussing on variation in the effect of the last deglaciation among 102 Holarctic species. Only 3 species showed a decline in &lt;i&gt;N&lt;/i&gt;&lt;sub&gt;e&lt;/sub&gt; since the Last Glacial Maximum (LGM) and 7 showed no sizeable change, whilst 92 profiles revealed an increase in &lt;i&gt;N&lt;/i&gt;&lt;sub&gt;e&lt;/sub&gt;. Using bioclimatic Species Distribution Models (SDMs), we also estimated changes in species potential range extent since the LGM. Whilst most modelled ranges also increased, we found no correlation across species between the magnitude of change in range size and change in &lt;i&gt;N&lt;/i&gt;&lt;sub&gt;e&lt;/sub&gt;. The lack of correlation between SDM and BSP reconstructions could not be reconciled even when range shifts were considered. We suggest the lack of agreement between these measures might be linked to changes in population densities which can be independent of range changes. We caution that interpreting either SDM or BSPs independently is problematic and potentially misleading. Additionally, we found that &lt;i&gt;N&lt;/i&gt;&lt;sub&gt;e&lt;/sub&gt; of wetland species tended to increase later than species from terrestrial habitats, possibly reflecting a delayed increase in the extent of this habitat type after the LGM.&lt;/p&gt;" />
<meta name="citation_journal_title" content="bioRxiv" />
<meta name="citation_publisher" content="Cold Spring Harbor Laboratory" />
<meta name="citation_publication_date" content="2019/01/01" />
<meta name="citation_mjid" content="biorxiv;2019.12.13.870410v1" />
<meta name="citation_id" content="2019.12.13.870410v1" />
<meta name="citation_public_url" content="https://www.biorxiv.org/content/10.1101/2019.12.13.870410v1" />
<meta name="citation_abstract_html_url" content="https://www.biorxiv.org/content/10.1101/2019.12.13.870410v1.abstract" />
<meta name="citation_full_html_url" content="https://www.biorxiv.org/content/10.1101/2019.12.13.870410v1.full" />
<meta name="citation_pdf_url" content="https://www.biorxiv.org/content/biorxiv/early/2019/12/13/2019.12.13.870410.full.pdf" />
<meta name="citation_doi" content="10.1101/2019.12.13.870410" />
<meta name="citation_num_pages" content="30" />
<meta name="citation_article_type" content="Article" />
<meta name="citation_section" content="New Results" />
<meta name="citation_firstpage" content="2019.12.13.870410" />
<meta name="citation_author" content="Eleanor F. Miller" />
<meta name="citation_author_institution" content="Department of Zoology, University of Cambridge" />
<meta name="citation_author_email" content="em618@cam.ac.uk" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0002-3213-5714" />
<meta name="citation_author" content="Rhys E. Green" />
<meta name="citation_author_institution" content="Department of Zoology, University of Cambridge" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0001-8690-8914" />
<meta name="citation_author" content="Andrew Balmford" />
<meta name="citation_author_institution" content="Department of Zoology, University of Cambridge" />
<meta name="citation_author" content="Robert Beyer" />
<meta name="citation_author_institution" content="Department of Zoology, University of Cambridge" />
<meta name="citation_author" content="Marius Somveille" />
<meta name="citation_author_institution" content="BirdLife International" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0002-6868-5080" />
<meta name="citation_author" content="Michela Leonardi" />
<meta name="citation_author_institution" content="Department of Zoology, University of Cambridge" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0001-8933-9374" />
<meta name="citation_author" content="William Amos" />
<meta name="citation_author_institution" content="Department of Zoology, University of Cambridge" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0002-0971-9914" />
<meta name="citation_author" content="Andrea Manica" />
<meta name="citation_author_institution" content="Department of Zoology, University of Cambridge" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0003-1895-450X" />
<meta name="citation_reference" content="Aiello-Lammens ME, Boria RA, Radosavljevic A, Vilela B, Anderson RP. 2015. spThin: An R package for spatial thinning of species occurrence records for use in ecological niche models. Ecography (Cop). 38(5):541–545. doi:10.1111/ecog.01132." />
<meta name="citation_reference" content="Allen JRM, Brandt U, Brauer A, Hubberten H-W, Huntley B, Keller J, Kraml M, Mackensen A, Mingram J, Negendank JFW, et al. 1999. Rapid environmental changes in southern Europe during the last glacial period. Nature. 400(6746):740–743. doi:10.1038/23432." />
<meta name="citation_reference" content="Allen JRM, Hickler T, Singarayer JS, Sykes MT, Valdes PJ, Huntley B. 2010. Last glacial vegetation of Northern Eurasia. Quat Sci Rev. 29(19–20):2604–2618. doi:10.1016/j.quascirev.2010.05.031." />
<meta name="citation_reference" content="Araújo MB, New M. 2007. Ensemble forecasting of species distributions. Trends Ecol Evol. 22(1):42–47. doi:10.1016/j.tree.2006.09.010." />
<meta name="citation_reference" content="Auer T, Fink D, Strimas-Mackey M. 2019. ebirdst: Tools for loading, plotting, mapping and analysis of eBird Status and Trends data products. R package version 0.1.0. https://cornelllabofornithology.github.io/ebirdst/." />
<meta name="citation_reference" content="Bagchi R, Crosby M, Huntley B, Hole DG, Butchart SHM, Collingham Y, Kalra M, Rajkumar J, Rahmani A, Pandey M, et al. 2013. Evaluating the effectiveness of conservation site networks under climate change: Accounting for uncertainty. Glob Chang Biol. 19(4):1236–1248. doi:10.1111/gcb.12123." />
<meta name="citation_reference" content="Beyer R, Krapp M, Manica A. 2019. A systematic comparison of bias correction methods for paleoclimate simulations. Clim Past Discuss.(February):1–23. doi:10.5194/cp-2019-11." />
<meta name="citation_reference" content="Beyer RM, Krapp M, Manica A. 2019. High-resolution terrestrial climate, bioclimate and vegetation for the last 120,000 years. EarthArXiv." />
<meta name="citation_reference" content="BirdLife International. BirdLife International and Handbook of the Birds of the World. 2018. Bird species distribution maps of the world. Version 2018.1. http://datazone.birdlife.org/species/requestdis." />
<meta name="citation_reference" content="BirdLife International and Handbook of the Birds of the World. 2018. Bird species distribution maps of the world. Version 2018.1." />
<meta name="citation_reference" content="Bouckaert R. 2015. bModelTest: Bayesian site model selection for nucleotide data. bioRxiv.:1–17. doi:http://dx.doi.org/10.1101/020792." />
<meta name="citation_reference" content="Bouckaert R, Heled J, Kühnert D, Vaughan T, Wu CH, Xie D, Suchard MA, Rambaut A, Drummond AJ. 2014. BEAST 2: A Software Platform for Bayesian Evolutionary Analysis. PLoS Comput Biol. 10(4):1–6. doi:10.1371/journal.pcbi.1003537." />
<meta name="citation_reference" content="Burbrink FT, Chan YL, Myers EA, Ruane S, Smith BT, Hickerson MJ. 2016. Asynchronous demographic responses to Pleistocene climate change in Eastern Nearctic vertebrates. Ecol Lett. doi:10.1111/ele.12695. http://doi.wiley.com/10.1111/ele.12695." />
<meta name="citation_reference" content="Calderón L, Campagna L, Wilke T, Lormée H, Eraud C, Dunn J, Rocha G, Zehtindjiev P, Bakaloudis D, Metzger B, et al. 2016. Genome-wide evidence of demographic fluctuations and lack of genetic structure across flyways in a long distance migrant, the European turtle dove. Mol Ecol.:1–11. doi:10.1186/s12862-016-0817-7." />
<meta name="citation_reference" content="Chikhi L, Sousa VC, Luisi P, Goossens B, Beaumont MA. 2010. The confounding effects of population structure, genetic diversity and the sampling scheme on the detection and quantification of population size changes. Genetics. 186(3):983–995. doi:10.1534/genetics.110.118661." />
<meta name="citation_reference" content="Connor EF, Courtney AC, Yoder JM. 2000. Individuals-Area Relationships: The Relationship between Animal Population Density and Area. Ecology. 81(3):734. doi:10.2307/177373." />
<meta name="citation_reference" content="Drummond AJ, Rambaut A. 2007. BEAST: Bayesian evolutionary analysis by sampling trees. BMC Evol Biol. 8:1–8. doi:10.1186/1471-2148-7-214." />
<meta name="citation_reference" content="Drummond AJ, Rambaut A, Shapiro B, Pybus OG. 2005. Bayesian coalescent inference of past population dynamics from molecular sequences. Mol Biol Evol. 22(5):1185–1192. doi:10.1093/molbev/msi103." />
<meta name="citation_reference" content="Dunning J. 2007. CRC Handbook of Avian Body Masses, Second Edition. 2nd ed. CRC Press." />
<meta name="citation_reference" content="Elith J, Leathwick JR. 2009. Species Distribution Models: Ecological Explanation and Prediction Across Space and Time. Annu Rev Ecol Evol Syst. 40(1):677–697. doi:10.1146/annurev.ecolsys.110308.120159." />
<meta name="citation_reference" content="Fan Y, Miguez-Macho G. 2011. A simple hydrologic framework for simulating wetlands in climate and earth system models. Clim Dyn. 37(1):253–278. doi:10.1007/s00382-010-0829-8." />
<meta name="citation_reference" content="Foote AD, Kaschner K, Schultze SE, Garilao C, Ho SYW, Post K, Higham TFG, Stokowska C, Van Der Es H, Embling CB, et al. 2013. Ancient DNA reveals that bowhead whale lineages survived Late Pleistocene climate change and habitat shifts. Nat Commun. 4:1677. doi:10.1038/ncomms2714." />
<meta name="citation_reference" content="Fordham DA, Resit Akçakaya H, Araújo MB, Elith J, Keith DA, Pearson R, Auld TD, Mellin C, Morgan JW, Regan TJ, et al. 2012. Plant extinction risk under climate change: are forecast range shifts alone a good indicator of species vulnerability to global warming? Glob Chang Biol. 18(4):1357–1371. doi:10.1111/j.1365-2486.2011.02614.x." />
<meta name="citation_reference" content="Giesecke T, Brewer S, Finsinger W, Leydet M, Bradshaw RHW. 2017. Patterns and dynamics of European vegetation change over the last 15,000 years. J Biogeogr. 44(7):1441–1456. doi:10.1111/jbi.12974." />
<meta name="citation_reference" content="Grant WS. 2015. Problems and cautions with sequence mismatch analysis and Bayesian skyline plots to infer historical demography. J Hered. 106(4):333–346. doi:10.1093/jhered/esv020." />
<meta name="citation_reference" content="Guisan A, Thuiller W, Zimmermann NE. 2017. Habitat Suitability and Distribution Models. Cambridge: Cambridge University Press. http://ebooks.cambridge.org/ref/id/CBO9781139028271." />
<meta name="citation_reference" content="Heled J, Drummond AJ. 2008. Bayesian inference of population size history from multiple loci. BMC Evol Biol. 15:1–15. doi:10.1186/1471-2148-8-289." />
<meta name="citation_reference" content="citation_journal_title=PLoS One;citation_author=R. Heller;citation_author=L. Chikhi;citation_author=HR. Siegismund;citation_title=The Confounding Effect of Population Structure on Bayesian Skyline Plot Inferences of Demographic History;citation_pages=e62992;citation_volume=8;citation_year=2013;citation_issue=5;citation_pmid=23667558;citation_doi=10.1371/journal.pone.0062992" />
<meta name="citation_reference" content="Hijmans RJ. 2012. Cross-validation of species distribution models: Removing spatial sorting bias and calibration with a null model. Ecology. 93(3):679–688. doi:10.1890/11-0826.1." />
<meta name="citation_reference" content="Hijmans RJ, Cameron SE, Parra JL, Jones PG, Jarvis A. 2005. Very high resolution interpolated climate surfaces for global land areas. Int J Climatol. 25(15):1965–1978. doi:10.1002/joc.1276." />
<meta name="citation_reference" content="Ho SYM. 2007. Calibrating molecular estimates of substitution rates and divergence times in birds. J Avian Biol. 38(4):409–414. doi:10.1111/j.2007.0908-8857.04168.x." />
<meta name="citation_reference" content="Ho SYW, Shapiro B. 2011. Skyline-plot methods for estimating demographic history from nucleotide sequences. Mol Ecol Resour. 11(3):423–434. doi:10.1111/j.1755-0998.2011.02988.x." />
<meta name="citation_reference" content="Hope AG, Ho SYW, Malaney JL, Cook JA, Talbot SL. 2014. Accounting for rate variation among lineages in comparative demographic analyses. Evolution (N Y). 68(9):2689–2700. doi:10.1111/evo.12469." />
<meta name="citation_reference" content="Howard C, Stephens PA, Pearce-Higgins JW, Gregory RD, Willis SG. 2014. Improving species distribution models: The value of data on abundance. Methods Ecol Evol. 5(6):506–513. doi:10.1111/2041-210X.12184." />
<meta name="citation_reference" content="Howell N, Smejkal CB, Mackey DA, Chinnery PF, Turnbull DM, Herrnstadt C. 2003. The Pedigree Rate of Sequence Divergence in the Human Mitochondrial Genome: There Is a Difference Between Phylogenetic and Pedigree Rates. Am J Hum Genet. 72(3):659–670. doi:10.1086/368264." />
<meta name="citation_reference" content="Jetz W, Thomas GH, Joy JB, Hartmann K, Mooers AO. 2012. The global diversity of birds in space and time. Nature. 491(7424):444–448. doi:10.1038/nature11631." />
<meta name="citation_reference" content="Johnston A, Fink D, Reynolds MD, Hochachka WM, Sullivan BL, Bruns NE, Hallstein E, Merrifield MS, Matsumoto S, Kelling S. 2015. Abundance models improve spatial and temporal prioritization of conservation resources. Ecol Appl. 25(7):1749–1756. doi:10.1890/14-1826.1." />
<meta name="citation_reference" content="Kaplan JO. 2002. Wetlands at the Last Glacial Maximum: Distribution and methane emissions. Geophys Res Lett. 29(6):3-1-3–4. doi:10.1029/2001GL013366." />
<meta name="citation_reference" content="Kingman JFC. 1982. On the Genealogy of Large Populations. J Appl Probab. 19(1982):27. doi:10.2307/3213548." />
<meta name="citation_reference" content="Kumar S, Stecher G, Tamura K. 2016. MEGA7: Molecular Evolutionary Genetics Analysis Version 7.0 for Bigger Datasets. Mol Biol Evol. 33(7):1870–1874. doi:10.1093/molbev/msw054." />
<meta name="citation_reference" content="Lafleur PM. 2008. Connecting Atmosphere and Wetland: Energy and Water Vapour Exchange. Geogr Compass. 2(4):1027–1057. doi:10.1111/j.1749-8198.2007.00132.x." />
<meta name="citation_reference" content="Leigh JW, Bryant D. 2015. Popart: Full-Feature Software for Haplotype Network Construction. Methods Ecol Evol. 6(9):1110–1116. doi:10.1111/2041-210X.12410." />
<meta name="citation_reference" content="Lorenzen ED, Nogués-Bravo D, Orlando L, Weinstock J, Binladen J, Marske KA, Ugan A, Borregaard MK, Gilbert MTP, Nielsen R, et al. 2012. Species-specific responses of LateQuaternary megafauna to climate and humans. Nature. 479(7373):359–364. doi:10.1038/nature10574." />
<meta name="citation_reference" content="Lu B, Zheng Y, Murphy RW, Zeng X. 2012. Coalescence patterns of endemic Tibetan species of stream salamanders (Hynobiidae: Batrachuperus). Mol Ecol. 21(13):3308–3324. doi:10.1111/j.1365-294X.2012.05606.x." />
<meta name="citation_reference" content="Mazet O, Rodríguez W, Chikhi L. 2015. Demographic inference using genetic data from a single individual: Separating population size variation from population structure. Theor Popul Biol. 104:46–58. doi:10.1016/j.tpb.2015.06.003." />
<meta name="citation_reference" content="Van Meerbeeck CJ, Renssen H, Roche DM. 2009. How did Marine Isotope Stage 3 and Last Glacial Maximum climates differ? - Perspectives from equilibrium simulations. Clim Past. 5(1):33–51. doi:10.5194/cp-5-33-2009." />
<meta name="citation_reference" content="Miller EF, Manica A, Amos W. 2018. Global demographic history of human populations inferred from whole mitochondrial genomes. R Soc Open Sci. 5(8):180543. doi:10.1098/rsos.180543." />
<meta name="citation_reference" content="Nabholz B, Lanfear R, Fuchs J. 2016. Body mass-corrected molecular rate for bird mitochondrial DNA. Mol Ecol. 25(18):4438–4449. doi:10.1111/mec.13780." />
<meta name="citation_reference" content="Pannell JR. 2003. Coalescence in a Metapopulation with Recurrent Local Extinction and Recolonzation. Evolution (N Y). 57(5):949–961. doi:10.1111/j.0014-3820.2003.tb00307.x." />
<meta name="citation_reference" content="Penny D. 2005. Relativity for molecular clocks. Nature. 436(July):183–184. doi:10.1038/436183a." />
<meta name="citation_reference" content="Potts JM, Elith J. 2006. Comparing species abundance models. Ecol Modell. 199(2):153–163. doi:10.1016/j.ecolmodel.2006.05.025." />
<meta name="citation_reference" content="Pybus OG, Rambaut A, Harvey PH. 2000. An integrated framework for the inference of viral population history from reconstructed genealogies. Genetics. 155(3):1429–37." />
<meta name="citation_reference" content="Roberts DR, Bahn V, Ciuti S, Boyce MS, Elith J, Guillera-Arroita G, Hauenstein S, Lahoz-Monfort JJ, Schröder B, Thuiller W, et al. 2017. Cross-validation strategies for data with temporal, spatial, hierarchical, or phylogenetic structure. Ecography (Cop). 40(8):913–929. doi:10.1111/ecog.02881." />
<meta name="citation_reference" content="Sanchez G, Tomano S, Yamashiro C, Fujita R, Wakabayashi T, Sakai M, Umino T. 2016. Population genetics of the jumbo squid Dosidicus gigas (Cephalopoda: Ommastrephidae) in the northern Humboldt Current system based on mitochondrial and microsatellite DNA markers. Fish Res. 175:1–9. doi:10.1016/j.fishres.2015.11.005." />
<meta name="citation_reference" content="Segawa T, Takeuchi N, Fujita K, Aizen VB, Willerslev E, Yonezawa T. 2018. Demographic analysis of cyanobacteria based on the mutation rates estimated from an ancient ice core. Heredity (Edinb).:1–12. doi:10.1038/s41437-017-0040-3." />
<meta name="citation_reference" content="Stiller M, Baryshnikov G, Bocherens H, Grandal D’Anglade A, Hilpert B, Münzel SC, Pinhasi R, Rabeder G, Rosendahl W, Trinkaus E, et al. 2010. Withering away-25,000 years of genetic decline preceded cave bear extinction. Mol Biol Evol. 27(5):975–978. doi:10.1093/molbev/msq083." />
<meta name="citation_reference" content="Stockwell DR., Peterson AT. 2002. Effects of sample size on accuracy of species distribution models. Ecol Modell. 148(1):1–13. doi:10.1016/S0304-3800(01)00388-X." />
<meta name="citation_reference" content="Svenning J-C, Eiserhardt WL, Normand S, Ordonez A, Sandel B. 2015. The Influence of Paleoclimate on Present-Day Patterns in Biodiversity and Ecosystems. Annu Rev Ecol Evol Syst. 46(1):551–572. doi:10.1146/annurev-ecolsys-112414-054314." />
<meta name="citation_reference" content="Thompson JD, Higgins DG, Gibson TJ. 1994. ClustalW: improving the sensitivity of progressive multiple sequence aligment through sequence weighting, position specific gap penalties and weight matrix choice. Nucleic Acids Res Acids Res. 22(22):4673–4680. doi:10.1093/nar/22.22.4673." />
<meta name="citation_reference" content="Thuiller W, Georges D, Engler R, Breiner F. 2019. biomod2: Ensemble Platform for Species Distribution Modeling. https://cran.r-project.org/package=biomod2." />
<meta name="citation_reference" content="Twitchett RJ. 2006. The palaeoclimatology, palaeoecology and palaeoenvironmental analysis of mass extinction events. Palaeogeogr Palaeoclimatol Palaeoecol. 232(2–4):190–213. doi:10.1016/j.palaeo.2005.05.019." />
<meta name="citation_reference" content="Valavi R, Elith J, Lahoz-Monfort JJ, Guillera-Arroita G. 2019. blockCV: An r package for generating spatially or environmentally separated folds for k-fold cross-validation of species distribution models. Methods Ecol Evol. 10(2):225–232. doi:10.1111/2041-210X.13107." />
<meta name="citation_reference" content="Valdes PJ, Beerling DJ, Jonhson CE. 2005. The ice age methane budget. Geophys Res Lett. 32(2):1–4. doi:10.1029/2004GL021004." />
<meta name="citation_reference" content="Vignaud TM, Maynard JA, Leblois R, Meekan MG, Vázquez-Juárez R, Ramírez-Macías D, Pierce SJ, Rowat D, Berumen ML, Beeravolu C, et al. 2014. Genetic structure of populations of whale sharks among ocean basins and evidence for their historic rise and recent decline. Mol Ecol. doi:10.1111/mec.12754." />
<meta name="citation_reference" content="Villalta I, Amor F, Galarza JA, Dupont S, Ortega P, Hefetz A, Dahbi A, Cerdá X, Boulay R. 2018. Origin and distribution of desert ants across the Gibraltar Straits. Mol Phylogenet Evol. 118(September 2017):122–134. doi:10.1016/j.ympev.2017.09.026." />
<meta name="citation_reference" content="Voous KH. 1977. List of recent Holarctic bird species." />
<meta name="twitter:title" content="mtDNA-based reconstructions of change in effective population sizes of Holarctic birds do not agree with their reconstructed range sizes based on paleoclimates" />
<meta name="twitter:site" content="@biorxivpreprint" />
<meta name="twitter:card" content="summary" />
<meta name="twitter:image" content="https://www.biorxiv.org/sites/default/files/images/biorxiv_logo_homepage7-5-small.png" />
<meta name="twitter:description" content="During the Quaternary, large climate oscillations had profound impacts on the distribution, demography and diversity of species globally. Birds offer a special opportunity for studying these impacts because surveys of geographical distributions, publicly-available genetic sequence data, and the existence of species with adaptations to life in structurally different habitats, permit large-scale comparative analyses. We use Bayesian Skyline Plot (BSP) analysis of mitochondrial DNA to reconstruct profiles depicting how effective population size ( N e) may have changed over time, focussing on variation in the effect of the last deglaciation among 102 Holarctic species. Only 3 species showed a decline in N e since the Last Glacial Maximum (LGM) and 7 showed no sizeable change, whilst 92 profiles revealed an increase in N e. Using bioclimatic Species Distribution Models (SDMs), we also estimated changes in species potential range extent since the LGM. Whilst most modelled ranges also increased, we found no correlation across species between the magnitude of change in range size and change in N e. The lack of correlation between SDM and BSP reconstructions could not be reconciled even when range shifts were considered. We suggest the lack of agreement between these measures might be linked to changes in population densities which can be independent of range changes. We caution that interpreting either SDM or BSPs independently is problematic and potentially misleading. Additionally, we found that N e of wetland species tended to increase later than species from terrestrial habitats, possibly reflecting a delayed increase in the extent of this habitat type after the LGM." />
<meta name="og-title" property="og:title" content="mtDNA-based reconstructions of change in effective population sizes of Holarctic birds do not agree with their reconstructed range sizes based on paleoclimates" />
<meta name="og-url" property="og:url" content="https://www.biorxiv.org/content/10.1101/2019.12.13.870410v1" />
<meta name="og-site-name" property="og:site_name" content="bioRxiv" />
<meta name="og-description" property="og:description" content="During the Quaternary, large climate oscillations had profound impacts on the distribution, demography and diversity of species globally. Birds offer a special opportunity for studying these impacts because surveys of geographical distributions, publicly-available genetic sequence data, and the existence of species with adaptations to life in structurally different habitats, permit large-scale comparative analyses. We use Bayesian Skyline Plot (BSP) analysis of mitochondrial DNA to reconstruct profiles depicting how effective population size ( N e) may have changed over time, focussing on variation in the effect of the last deglaciation among 102 Holarctic species. Only 3 species showed a decline in N e since the Last Glacial Maximum (LGM) and 7 showed no sizeable change, whilst 92 profiles revealed an increase in N e. Using bioclimatic Species Distribution Models (SDMs), we also estimated changes in species potential range extent since the LGM. Whilst most modelled ranges also increased, we found no correlation across species between the magnitude of change in range size and change in N e. The lack of correlation between SDM and BSP reconstructions could not be reconciled even when range shifts were considered. We suggest the lack of agreement between these measures might be linked to changes in population densities which can be independent of range changes. We caution that interpreting either SDM or BSPs independently is problematic and potentially misleading. Additionally, we found that N e of wetland species tended to increase later than species from terrestrial habitats, possibly reflecting a delayed increase in the extent of this habitat type after the LGM." />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.biorxiv.org/sites/default/files/images/biorxiv_logo_homepage7-5-small.png" />
<meta name="citation_date" content="2019-12-13" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/10.1101/2019.12.13.870410v1.full.pdf" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/10.1101/2019.12.13.870410v1.full.txt" />
<link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/10.1101/2019.12.13.870410v1.ppt" />
<meta name="description" content="bioRxiv - the preprint server for biology, operated by Cold Spring Harbor Laboratory, a research and educational institution" />
<meta name="generator" content="Drupal 7 (http://drupal.org)" />
<link rel="canonical" href="https://www.biorxiv.org/content/10.1101/2019.12.13.870410v1" />
<link rel="shortlink" href="https://www.biorxiv.org/node/1035842" />
    <title>mtDNA-based reconstructions of change in effective population sizes of Holarctic birds do not agree with their reconstructed range sizes based on paleoclimates | bioRxiv</title>  
    <link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__jMRAK66KMC1e4TQlwUNn3KiWVDC5AjueUAYEm1xBY_U__KSUjdT4jdcJ4qJz6fKY1K9WkYh1a5EcaWZxt_-zbTis__QJacVmveFyLIBSPkp21xL0ZvuDW9DMBrD2HB1P4Ry5I.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q5p1rr" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__icrP-Y0qw2NP73rJxxkUtrJta0z3VLAABtNjfSdEghA__rlN3rx0wOoswGNHZMeZM96bNn-3cLWHXTztGDoThNlw__QJacVmveFyLIBSPkp21xL0ZvuDW9DMBrD2HB1P4Ry5I.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__q48SDxVDWut_cH6dTPyQe7HjADuSLLmJVFztuiJEFrU__DJqddQGjm5dXCHsaiwNriGytKN5eevIQqyjdq65TZtE__QJacVmveFyLIBSPkp21xL0ZvuDW9DMBrD2HB1P4Ry5I.css" media="all" />
<style type="text/css" media="all">
/* <![CDATA[ */
#sliding-popup.sliding-popup-bottom,#sliding-popup.sliding-popup-bottom .eu-cookie-withdraw-banner,.eu-cookie-withdraw-tab{background:gray}#sliding-popup.sliding-popup-bottom.eu-cookie-withdraw-wrapper{background:transparent}#sliding-popup .popup-content #popup-text h1,#sliding-popup .popup-content #popup-text h2,#sliding-popup .popup-content #popup-text h3,#sliding-popup .popup-content #popup-text p,.eu-cookie-compliance-secondary-button,.eu-cookie-withdraw-tab{color:#fff !important}.eu-cookie-withdraw-tab{border-color:#fff}.eu-cookie-compliance-more-button{color:#fff !important}
/* ]]> */
</style>

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__QJacVmveFyLIBSPkp21xL0ZvuDW9DMBrD2HB1P4Ry5I.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__B6tGdsQSt1vT8d8j-19-oS4FKJe83RrSNBkjPs9jfos__6VkE5WcQmmQpSVTQx-xS2-zn_nlwFlDm-EN1HLJNjrA__QJacVmveFyLIBSPkp21xL0ZvuDW9DMBrD2HB1P4Ry5I.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__I6pBwZrmO51afTZVOjN83FdGH1QRb9t15lSLso0JVV4__F3sVKlGyHKCaAPI_WdiHD7l_-05B6b9bLdUwzi1me90__QJacVmveFyLIBSPkp21xL0ZvuDW9DMBrD2HB1P4Ry5I.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__QJacVmveFyLIBSPkp21xL0ZvuDW9DMBrD2HB1P4Ry5I.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__QJacVmveFyLIBSPkp21xL0ZvuDW9DMBrD2HB1P4Ry5I.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__CRdjIL0HKDL2T9slYvkTSTLypBBPjyoDhwBezBuDy7c__cHebPmPNdg2jyPmpyQpZWssJQ4cPrqfaabp5Or9KlkU__QJacVmveFyLIBSPkp21xL0ZvuDW9DMBrD2HB1P4Ry5I.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://d33xdlntwy0kbs.cloudfront.net/cshl_custom.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/all/libraries/modernizr/modernizr.min.js?q5p1rr"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__yOhrJUBmMpN-jsuQlJYmS8ovkQU6I6KInr295HrSTb0__c--zHhbuOWmkdWkJYXiNWDmkpDRPesmbBjGlQcyqBOY__QJacVmveFyLIBSPkp21xL0ZvuDW9DMBrD2HB1P4Ry5I.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var _prum=[['id', '52e95df3abe53d2e33000000'], ['mark', 'firstbyte',
      (new Date()).getTime()]]; (function() {
      var s=document.getElementsByTagName('script')[0],
      p=document.createElement('script');
      p.async='async'; p.src='//rum-static.pingdom.net/prum.min.js';s.parentNode.insertBefore(p,s);})();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js___KyvuEvfmewWxdOLBXTYsSUo8Pfm_DijV6fWtGvSL_k__HSKkTgIr7nDcdvQYewipf6v4ak6ZLUOQkwPj44wKouA__QJacVmveFyLIBSPkp21xL0ZvuDW9DMBrD2HB1P4Ry5I.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__3uCzzYE5jr6ly9nvsM8PRBRkhh4p0NWCMDja4EKBy4M__dmGGVYcB8LhnbR2iAmVW0ZyXI6QuFVYFRORa5XxglXs__QJacVmveFyLIBSPkp21xL0ZvuDW9DMBrD2HB1P4Ry5I.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__0zdUME0GLauYYyEch2tGPydIw-IG7Uizip0Qb4wUsig__3RgszrCgk4bOnTzo4vaQ-hrWzaG6oD19l-R9CQ1YQ-o__QJacVmveFyLIBSPkp21xL0ZvuDW9DMBrD2HB1P4Ry5I.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__W6wAjopaALytcBam7oT3WhEzllkWaB9R-8hazHyvShM__rK5YBIqBy9qt0di0Sn4DriBx9TJupdmv6uGmLJyZvDs__QJacVmveFyLIBSPkp21xL0ZvuDW9DMBrD2HB1P4Ry5I.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-45638026-1", {"cookieDomain":"auto"});ga("set", "page", location.pathname + location.search + location.hash);ga("send", "pageview");ga('create', 'UA-189672-38', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__1PtAOZ8x7l6QAi8D7X6Kj_Y7HcUOFzvWRvvYHUcCT98__U8IB92u_CnnGwJ-X_xaUa9lzsC8OzUfN0WkK2JQfirg__QJacVmveFyLIBSPkp21xL0ZvuDW9DMBrD2HB1P4Ry5I.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings,{"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"OgOaID5tn1F_5MDZOay36pMb6vChXfJzW1d1xZ6749I","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"modules\/comment\/comment.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"modules\/forum\/forum.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"sites\/default\/modules\/biorxiv\/css\/biorxiv.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/flexible\/flexible.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_collection\/css\/highwire_subject_collection.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire.highwire-list-expand.css":1,"public:\/\/ctools\/css\/bab59090150caf0c64e27afa1b36eef1.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/contrib\/eu_cookie_compliance\/css\/eu_cookie_compliance.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"0":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_biorxiv_styles\/css\/biorxiv-styles.css.less":1,"https:\/\/d33xdlntwy0kbs.cloudfront.net\/cshl_custom.css":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_figures.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"http:\/\/d33xdlntwy0kbs.cloudfront.net\/cshl_custom.js":1,"sites\/all\/modules\/contrib\/eu_cookie_compliance\/js\/eu_cookie_compliance.js":1,"sites\/default\/modules\/jnl_biorxiv_styles\/js\/adv-search-collection-toggle.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/eu_cookie_compliance\/js\/jquery.cookie-1.4.1.min.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/default\/modules\/jnl_biorxiv_styles\/js\/biorxiv-scripts.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_nps_survey\/js\/nps_survey.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"sites\/default\/modules\/contrib\/disqus\/disqus.js":1,"misc\/textarea.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/contrib\/service_links\/js\/twitter_button.js":1,"sites\/all\/modules\/contrib\/service_links\/js\/facebook_like.js":1,"sites\/all\/modules\/contrib\/service_links\/js\/google_plus_one.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.highwire-list-expand.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"1035842","apath":"\/biorxiv\/early\/2019\/12\/13\/2019.12.13.870410.atom","pisa":"biorxiv;2019.12.13.870410v1","processed":["highwire_math"],"markup":[{"requested":"full-text","variant":"full-text","view":"full","pisa":"biorxiv;2019.12.13.870410v1"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"disqus":{"domain":"biorxivstage","url":"https:\/\/www.biorxiv.org\/content\/10.1101\/2019.12.13.870410v1","title":"mtDNA-based reconstructions of change in effective population sizes of Holarctic birds do not agree with their reconstructed range sizes based on paleoclimates","identifier":"node\/1035842"},"panels_ajax_pane":{"new-41":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022aFcaGcwECwhSiJmcPeUBYfbXMsUsnuVSe5PHQDypZvmaT+lFqwRPMQd5a\\\\\\\/WnCon5\\\\\\\/I8KXOInI61rnnCycK4MNtQ5I9a1SKzK83oPxwjUPoZz0dm7FnI9hUDsxQAVRNnmiRmH3zS68MreD4yfrq8pAfg7kGfAeryKuHFILbi3p3eLDHreImSPGYeSSBSBGNUbDzySlmV0g5stHtnD5OZ+TBE9D7DVT6K91aOdzKVJ84N6D+ZbSzXxyk0R2eiDNvuQAaSPrngXwpI30OXkuMr6bMb\\\\\\\/RqSse+lcoCGbrwigmiyVSTkQ2Jw8ivfxGij8+8sr1rLVRxNRs+6R4+YAfi2KdBNJpynVwPM4ouyppwtyChJCMLY2TLHfODYGi592PqVKcI1SDgqDAd4tL2uz0DiEX63oZB55lnJ9toKTkpjkSs6g8KyUnkXWaDFKoScekFrMRbkvlkNinfAcU6eq8yR\\\\\\\/vUjWCB\\\\\\\/LjdqjHayqCp\\\\\\\/4hIUEsz0KIuq4Lw9BF32AzFcZij22HV9gDvpP6plklC6scnkOfkhH\\\\\\\/WsctGyK1+M84coILaiSo5QRnWM3BJKF+5IcyZKBwLaJe2uYnTDdN9wWFunTO9Y5BKJbZXiRyIO75ERXFL57owIOvwPF+NaQQdFIYEeyMiRRaxHGUm2hDp0TJVFbrmPXv80\\\\\\\/apsDy0CJghWUgsr00W2TvPFXrjpzn\\\\\\\/kl\\\\\\\/koQoToHf0zb1I9lR4BGIjJNZ97g8A3wM9yD0TqLORkXCBHon4sVom\\\\\\\/jpc8wO0D56qsGgBmfNeDqNiz72+Bmh3t+tBUb5FbIctThsAgQdC0tJWPvUZtPPNq0ghWwnm9bJPtBrraCOJ9rvAtU5hr42MPjG\\\\\\\/9gD9aIOLDNoBbKW2HKf9gDI9k6ovCMEI+OEsB\\\\\\\/N33muY8erp4gtbGSGvSBPoRh1KJGzLuXEhvbIKU+0gpCZFQ3bfx4YPX1JcuJWvR4y34jL\\\\\\\/6+OxOrXHuHzRgUHRBFrvv6zVeVKBD1me0Sc8qK54SbRBINBCrk8t2jqPsy\\\\\\\/Vw3C5QuppB9mGoRj3Vn4aZN6ItB8CsqM6L35UZ04GlGJ\\\\\\\/qvzXfwUrFIx9L7vYiAblMOaLnF+Lcb5IZd3CFHKbiZYXgEycrMadQicovvzGuK9PVSO5cxxFLy3iq2y3Js3aL+Q1nGNluhsSufHstlV84nfPmRCkrX\\\\\\\/2ZQv329L8sMXvTgZJTl0SIdvC7y1XU7UwNOYZfNlJDDla8CsiO9iERoiVXCTBRrRyZYW1\\\\\\\/tRnsExf2OuJx\\\\\\\/zc4rlq6Ncbsulaw0Il2wmUfB5oq0BkMVM63Ux8R4c3j+oN7BU8F7s4t6nJmP15375hUUdu4zvajV\\\u0022,\\\u0022iv\\\u0022:\\\u0022aINrnOFe4GUHfQSSsrZwnA==\\\u0022,\\\u0022salt\\\u0022:\\\u0022f9d8845712f72c9fee5db7d9ff111eab\\\u0022}\u0022,\u0022hmac\u0022:\u00221b0738066fbccba59ceec315bb37562a4a8c7b0783dbf3e2adc20df1c242ea2b\u0022}"},"urlIsAjaxTrusted":{"\/content\/10.1101\/2019.12.13.870410v1.full":true},"ws_fl":{"width":100,"height":21},"ws_gpo":{"size":"","annotation":"","lang":"","callback":"","width":300},"color":{"logo":"https:\/\/www.biorxiv.org\/sites\/default\/files\/bioRxiv_article.jpg"},"highwire_list_expand":{"is_collapsed":"1"},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"eu_cookie_compliance":{"popup_enabled":1,"popup_agreed_enabled":0,"popup_hide_agreed":0,"popup_clicking_confirmation":1,"popup_scrolling_confirmation":false,"popup_html_info":"\u003Cdiv\u003E\n  \u003Cdiv class =\u0022popup-content info\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n      \u003Cp\u003EWe use cookies on this site to enhance your user experience. By clicking any link on this page you are giving your consent for us to set cookies.\u003C\/p\u003E\n    \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022agree-button eu-cookie-compliance-default-button\u0022\u003EContinue\u003C\/button\u003E\n                    \u003Cbutton type=\u0022button\u0022 class=\u0022find-more-button eu-cookie-compliance-more-button\u0022\u003EFind out more\u003C\/button\u003E\n          \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E","use_mobile_message":false,"mobile_popup_html_info":"\u003Cdiv\u003E\n  \u003Cdiv class =\u0022popup-content info\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n          \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022agree-button eu-cookie-compliance-default-button\u0022\u003EContinue\u003C\/button\u003E\n                    \u003Cbutton type=\u0022button\u0022 class=\u0022find-more-button eu-cookie-compliance-more-button\u0022\u003EFind out more\u003C\/button\u003E\n          \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E\n","mobile_breakpoint":"768","popup_html_agreed":"\u003Cdiv\u003E\n  \u003Cdiv class=\u0022popup-content agreed\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n      \u003Ch2\u003EThank you for accepting cookies\u003C\/h2\u003E\u003Cp\u003EYou can now hide this message or find out more about cookies.\u003C\/p\u003E    \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022hide-popup-button eu-cookie-compliance-hide-button\u0022\u003EHide\u003C\/button\u003E\n              \u003Cbutton type=\u0022button\u0022 class=\u0022find-more-button eu-cookie-compliance-more-button-thank-you\u0022 \u003EMore info\u003C\/button\u003E\n          \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E","popup_use_bare_css":false,"popup_height":"auto","popup_width":"100%","popup_delay":1000,"popup_link":"\/help\/cookie-policy","popup_link_new_window":1,"popup_position":null,"popup_language":"en","store_consent":false,"better_support_for_screen_readers":0,"reload_page":0,"domain":"","popup_eu_only_js":0,"cookie_lifetime":"365","cookie_session":false,"disagree_do_not_show_popup":0,"method":"default","whitelisted_cookies":"","withdraw_markup":"\u003Cbutton type=\u0022button\u0022 class=\u0022eu-cookie-withdraw-tab\u0022\u003EPrivacy settings\u003C\/button\u003E\n\u003Cdiv class=\u0022eu-cookie-withdraw-banner\u0022\u003E\n  \u003Cdiv class=\u0022popup-content info\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n      \u003Cp\u003E\u0026lt;h2\u0026gt;We use cookies on this site to enhance your user experience\u0026lt;\/h2\u0026gt;\u0026lt;p\u0026gt;You have given your consent for us to set cookies.\u0026lt;\/p\u0026gt;\u003C\/p\u003E\n    \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022eu-cookie-withdraw-button\u0022\u003EWithdraw consent\u003C\/button\u003E\n    \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E\n","withdraw_enabled":false},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1,"trackUrlFragments":1},"jnl_biorxiv_styles":{"defaultJCode":"biorxiv"},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-1035842 node-type-highwire-article context-content hw-default-jcode-biorxiv hw-article-type-article hw-article-category-new-results">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-M677548" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-M677548');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
        <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
    
  <div id="zone-branding" class="zone zone-branding clearfix print-display-block container-30">
    <div class="grid-15 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="bioRxiv" src="https://www.biorxiv.org/sites/default/files/bioRxiv_article.jpg" /></a>      </div>
                </div>
          </div>
</div><div class="grid-11 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-system block-menu block-main-menu block-system-main-menu odd block-without-title" id="block-system-main-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/" title="" class="" data-icon-position="" data-hide-link-title="0">Home</a></li>
<li class="leaf" role="menuitem"><a href="/about-biorxiv" class="" data-icon-position="" data-hide-link-title="0">About</a></li>
<li class="leaf" role="menuitem"><a href="/submit-a-manuscript" class="" data-icon-position="" data-hide-link-title="0">Submit</a></li>
<li class="last leaf" role="menuitem"><a href="/content/alertsrss" title="" class="" data-icon-position="" data-hide-link-title="0">ALERTS / RSS</a></li>
</ul></nav>    </div>
  </div>
</div><div class="block block-panels-mini block-biorxiv-search-box block-panels-mini-biorxiv-search-box even block-without-title" id="block-panels-mini-biorxiv-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-biorxiv_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/10.1101/2019.12.13.870410v1.full" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--2" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-SpT7C7XfNQMw6UQ_YV-_vODO6NnsJplzCnW1X7KsO44" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2 advanced-search-link" >
  
      
  
  <div class="pane-content">
    <a href="/search">Advanced Search</a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix container-30">
  	      </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="1035842" id="node1035842" data-pisa="biorxiv;2019.12.13.870410v1" data-pisa-master="biorxiv;2019.12.13.870410" data-apath="/biorxiv/early/2019/12/13/2019.12.13.870410.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div  class="highwire-cite highwire-cite-highwire-article highwire-citation-biorxiv-article-top clearfix has-author-tooltip" >

      <span class="biorxiv-article-type">
        New Results    </span>
  
        <h1 class="highwire-cite-title" id="page-title">mtDNA-based reconstructions of change in effective population sizes of Holarctic birds do not agree with their reconstructed range sizes based on paleoclimates</h1>  
      <div  class="highwire-cite-authors" ><span  class="highwire-citation-authors"><span class="highwire-citation-author first hw-author-orcid-logo-wrapper" data-delta="0"><a href="http://orcid.org/0000-0002-3213-5714" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a><span class="nlm-given-names">Eleanor F.</span> <span class="nlm-surname">Miller</span></span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="1"><a href="http://orcid.org/0000-0001-8690-8914" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a><span class="nlm-given-names">Rhys E.</span> <span class="nlm-surname">Green</span></span>, <span class="highwire-citation-author" data-delta="2"><span class="nlm-given-names">Andrew</span> <span class="nlm-surname">Balmford</span></span>, <span class="highwire-citation-author" data-delta="3"><span class="nlm-given-names">Robert</span> <span class="nlm-surname">Beyer</span></span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="4"><a href="http://orcid.org/0000-0002-6868-5080" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a><span class="nlm-given-names">Marius</span> <span class="nlm-surname">Somveille</span></span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="5"><a href="http://orcid.org/0000-0001-8933-9374" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a><span class="nlm-given-names">Michela</span> <span class="nlm-surname">Leonardi</span></span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="6"><a href="http://orcid.org/0000-0002-0971-9914" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a><span class="nlm-given-names">William</span> <span class="nlm-surname">Amos</span></span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="7"><a href="http://orcid.org/0000-0003-1895-450X" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a><span class="nlm-given-names">Andrea</span> <span class="nlm-surname">Manica</span></span></span></div>
  
      <div  class="highwire-cite-metadata" ><span  class="highwire-cite-metadata-doi highwire-cite-metadata"><span class="label">doi:</span> https://doi.org/10.1101/2019.12.13.870410 </span></div>
  
  
  </div>
<div id="hw-article-author-popups-node1035842" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Eleanor F. Miller </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-institution'>Department of Zoology, University of Cambridge</span>, Downing street, Cambridge, CB2 3EJ, <span class='nlm-country'>UK</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Eleanor%2BF.%2BMiller%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Miller%20EF&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AEleanor%2BF.%2BMiller%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0002-3213-5714" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Eleanor F. Miller</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:em618@cam.ac.uk" class="" data-icon-position="" data-hide-link-title="0">em618@cam.ac.uk</a></span></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Rhys E. Green </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-institution'>Department of Zoology, University of Cambridge</span>, Downing street, Cambridge, CB2 3EJ, <span class='nlm-country'>UK</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Rhys%2BE.%2BGreen%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Green%20RE&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ARhys%2BE.%2BGreen%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0001-8690-8914" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Rhys E. Green</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Andrew Balmford </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-institution'>Department of Zoology, University of Cambridge</span>, Downing street, Cambridge, CB2 3EJ, <span class='nlm-country'>UK</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Andrew%2BBalmford%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Balmford%20A&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AAndrew%2BBalmford%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Robert Beyer </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-institution'>Department of Zoology, University of Cambridge</span>, Downing street, Cambridge, CB2 3EJ, <span class='nlm-country'>UK</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Robert%2BBeyer%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Beyer%20R&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ARobert%2BBeyer%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Marius Somveille </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-institution'>BirdLife International</span>, The David Attenborough Building, Pembroke Street, Cambridge CB2 3QZ, <span class='nlm-country'>UK</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Marius%2BSomveille%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Somveille%20M&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AMarius%2BSomveille%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-6868-5080" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Marius Somveille</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Michela Leonardi </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-institution'>Department of Zoology, University of Cambridge</span>, Downing street, Cambridge, CB2 3EJ, <span class='nlm-country'>UK</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Michela%2BLeonardi%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Leonardi%20M&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AMichela%2BLeonardi%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0001-8933-9374" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Michela Leonardi</a></li></ul></div><div class="author-tooltip-6"><div class="author-tooltip-name">William Amos </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-institution'>Department of Zoology, University of Cambridge</span>, Downing street, Cambridge, CB2 3EJ, <span class='nlm-country'>UK</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=William%2BAmos%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Amos%20W&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AWilliam%2BAmos%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-0971-9914" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for William Amos</a></li></ul></div><div class="author-tooltip-7"><div class="author-tooltip-name">Andrea Manica </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-institution'>Department of Zoology, University of Cambridge</span>, Downing street, Cambridge, CB2 3EJ, <span class='nlm-country'>UK</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Andrea%2BManica%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Manica%20A&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AAndrea%2BManica%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0003-1895-450X" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Andrea Manica</a></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/10.1101/2019.12.13.870410v1" class="panels-ajax-tab-tab" data-panel-name="biorxiv_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:1035842" data-trigger="" data-url-enabled="1">Abstract</a><a href="/panels_ajax_tab/biorxiv_tab_art/node:1035842/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/10.1101/2019.12.13.870410v1.full-text" class="panels-ajax-tab-tab" data-panel-name="article_tab_full_text" data-target-id="highwire_article_tabs" data-entity-context="node:1035842" data-trigger="full-text" data-url-enabled="1">Full Text</a><a href="/panels_ajax_tab/article_tab_full_text/node:1035842/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/10.1101/2019.12.13.870410v1.article-info" class="panels-ajax-tab-tab" data-panel-name="biorxiv_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:1035842" data-trigger="article-info" data-url-enabled="1">Info/History</a><a href="/panels_ajax_tab/biorxiv_tab_info/node:1035842/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/10.1101/2019.12.13.870410v1.article-metrics" class="panels-ajax-tab-tab" data-panel-name="article_tab_metrics" data-target-id="highwire_article_tabs" data-entity-context="node:1035842" data-trigger="article-metrics" data-url-enabled="1">Metrics</a><a href="/panels_ajax_tab/article_tab_metrics/node:1035842/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/10.1101/2019.12.13.870410v1.supplementary-material" class="panels-ajax-tab-tab" data-panel-name="biorxiv_tab_data" data-target-id="highwire_article_tabs" data-entity-context="node:1035842" data-trigger="supplementary-material" data-url-enabled="1">Supplementary material</a><a href="/panels_ajax_tab/biorxiv_tab_data/node:1035842/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/10.1101/2019.12.13.870410v1.full.pdf+html" class="panels-ajax-tab-tab" data-panel-name="biorxiv_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:1035842" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i> Preview PDF</a><a href="/panels_ajax_tab/biorxiv_tab_pdf/node:1035842/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="article_tab_full_text" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.biorxiv.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-article_tab_full_text"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-markup" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2 class="">Abstract</h2><p id="p-2">During the Quaternary, large climate oscillations had profound impacts on the distribution, demography and diversity of species globally. Birds offer a special opportunity for studying these impacts because surveys of geographical distributions, publicly-available genetic sequence data, and the existence of species with adaptations to life in structurally different habitats, permit large-scale comparative analyses. We use Bayesian Skyline Plot (BSP) analysis of mitochondrial DNA to reconstruct profiles depicting how effective population size (<em>N</em><sub>e</sub>) may have changed over time, focussing on variation in the effect of the last deglaciation among 102 Holarctic species. Only 3 species showed a decline in <em>N</em><sub>e</sub> since the Last Glacial Maximum (LGM) and 7 showed no sizeable change, whilst 92 profiles revealed an increase in <em>N</em><sub>e</sub>. Using bioclimatic Species Distribution Models (SDMs), we also estimated changes in species potential range extent since the LGM. Whilst most modelled ranges also increased, we found no correlation across species between the magnitude of change in range size and change in <em>N</em><sub>e</sub>. The lack of correlation between SDM and BSP reconstructions could not be reconciled even when range shifts were considered. We suggest the lack of agreement between these measures might be linked to changes in population densities which can be independent of range changes. We caution that interpreting either SDM or BSPs independently is problematic and potentially misleading. Additionally, we found that <em>N</em><sub>e</sub> of wetland species tended to increase later than species from terrestrial habitats, possibly reflecting a delayed increase in the extent of this habitat type after the LGM.</p></div><div class="section" id="sec-1"><h2 class="">Introduction</h2><p id="p-3">The Quaternary period has been characterised by extensive cycles of glaciation and de-glaciation. The legacy of these ancient large-scale climate alterations is evident today in everything from species’ genetic diversity to population structure (<a id="xref-ref-61-1" class="xref-bibr" href="#ref-61">Twitchett 2006</a>; <a id="xref-ref-58-1" class="xref-bibr" href="#ref-58">Svenning et al. 2015</a>). Despite the profound impact that past climate changes have had on both flora and fauna, there is limited quantitative evidence on which factors determined how different species fared during these cycles, or how species responded to subsequent post-glacial climate amelioration.</p><p id="p-4">One of the most widely used genetic methods for inferring demographic history is the so-called skyline plot, a family of graphical, non-parametric methods first introduced by <a id="xref-ref-52-1" class="xref-bibr" href="#ref-52">Pybus et al. (2000)</a>. Grounded in the principles of Kingman’s coalescent theory (<a id="xref-ref-39-1" class="xref-bibr" href="#ref-39">Kingman 1982</a>), the ‘skyline framework’ aims to use DNA sequence data to reconstruct a gene tree. The rate of coalescent events within the gene tree can then be used to infer how the population changed in size over time: in essence, periods of low coalescent rates imply a large population while a high density of coalescent events implies a small population. Although skyline plots have been used to reconstruct demographic histories for many species, both extant and extinct (<a id="xref-ref-56-1" class="xref-bibr" href="#ref-56">Stiller et al. 2010</a>), and across taxa that include vertebrates (<a id="xref-ref-44-1" class="xref-bibr" href="#ref-44">Lu et al. 2012</a>; <a id="xref-ref-64-1" class="xref-bibr" href="#ref-64">Vignaud et al. 2014</a>), invertebrates (<a id="xref-ref-54-1" class="xref-bibr" href="#ref-54">Sanchez et al. 2016</a>; <a id="xref-ref-65-1" class="xref-bibr" href="#ref-65">Villalta et al. 2018</a>) and even bacteria (<a id="xref-ref-55-1" class="xref-bibr" href="#ref-55">Segawa et al. 2018</a>), comparative studies across many species are only now emerging (<a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">Burbrink et al. 2016</a>).</p><p id="p-5">Skyline plots have been used extensively to infer the response of species during the Last Glacial Maximum (LGM), and they are often paired with climatic reconstructions to infer the changes in available habitat for a given species (<a id="xref-ref-43-1" class="xref-bibr" href="#ref-43">Lorenzen et al. 2012</a>; <a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">Foote et al. 2013</a>; <a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">Calderón et al. 2016</a>). One popular approach for reconstructing possible changes in available habitat for a species through time is the use of bioclimatic Species Distribution Models (SDMs) (<a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">Elith and Leathwick 2009</a>). Modelling algorithms combine data on occurrences with environmental data to describe a species’ natural distribution and then simulate how changes in predictor factors may have influenced this available niche space over a period of interest. The underlying logic in linking these approaches is that, assuming limited population structure and appropriate sampling, a skyline plot could, in principle, provide an indication of changes in total population size, and thus of the range occupied by a species. However, the association between effective population sizes (<em>N</em><sub>e</sub>) as reconstructed by skyline plots and species ranges is generally assumed rather than tested.</p><p id="p-6">There are a number of reasons why reconstructed <em>N</em><sub>e</sub> might not be a good proxy for species ranges. Much attention has been devoted to population structure as a confounding effect, and the recommendation to counter its effects is to pool samples from multiple locations (<a id="xref-ref-28-1" class="xref-bibr" href="#ref-28">Heller et al. 2013</a>). However, even with this sampling scheme, there might be a mismatch in the two quantities if mean population density, and thus <em>N</em><sub>e</sub>, was affected by climate change differently to total range extent. For example an increase in mean population density, and thus population size, might occur without a change in range, if the quality of habitat and its carrying capacity increased without a change in its extent (<a id="xref-fig-1-1" class="xref-fig" href="#F1">Fig. 1A</a>) (<a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">Fordham et al. 2012</a>). Given the positive relationship generally observed between range extent and mean local population density (<a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">Connor et al. 2000</a>), <em>N</em><sub>e</sub> would also be expected to increase by a greater proportion than range extent under climatic amelioration. Another plausible cause of discordance between changes in <em>N</em><sub>e</sub> and range size is that, without substantial gene flow, skyline plots will mostly reconstruct the population dynamics of the sampled locations rather than the whole species (<a id="xref-ref-47-1" class="xref-bibr" href="#ref-47">Miller et al. 2018</a>). Pooling samples from multiple locations can help, but it will not fully resolve the problem (<a id="xref-ref-28-2" class="xref-bibr" href="#ref-28">Heller et al. 2013</a>). A plausible scenario that might lead to a disconnect between local <em>N</em><sub>e</sub> and range size arises during range shifts, as sampled locations, which are suitable for a species at present day, might have been only marginally suitable in the past. In other words, what we now think of as the core area occupied by a species (i.e. where it is abundant, and sampling is more likely) might be inhabited by populations that in the past were at low densities because the local habitat was only marginally suitable. A skyline from such populations would reveal a strong increase in <em>N</em><sub>e</sub> which reflects the local amelioration of conditions for that species, irrespective of broader range changes (<a id="xref-fig-1-2" class="xref-fig" href="#F1">Fig. 1B</a>). A similarly confounded signal will be found in the more extreme scenario where, as a result of a sizeable range shift, the sampled populations inhabit areas which were completely unsuitable in the past, and thus have undergone a founder event after the Last Glacial Maximum (LGM). Such populations would be characterised by a steep increase in <em>N</em><sub>e</sub> as they recovered from the local bottleneck associated with the founder event (<a id="xref-fig-1-3" class="xref-fig" href="#F1">Fig. 1C</a>).</p><div id="F1" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/12/13/2019.12.13.870410/F1.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Three scenarios that might lead to an increase in Ne without a change in range size. The top half of each panel represents a pair of schematic maps of the species range at the LGM (right) and present day (left). The density of colour within each range ellipse shows population density. Circles on the map represent the genetic sampling location. The scenarios are: A) An increase in population size without a change in range recovers an increasing BSP profile. B) Core area today was only marginally suitable in the past, range size remains the same but local amelioration has led to a strong local increase in Ne. C) Sampled populations inhabit areas outside the species range in the past. BSP recovers a steep increase in Ne associated with founder event." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1292071231" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Three scenarios that might lead to an increase in &lt;em&gt;N&lt;/em&gt;&lt;sub&gt;e&lt;/sub&gt; without a change in range size. The top half of each panel represents a pair of schematic maps of the species range at the LGM (right) and present day (left). The density of colour within each range ellipse shows population density. Circles on the map represent the genetic sampling location. The scenarios are: A) An increase in population size without a change in range recovers an increasing BSP profile. B) Core area today was only marginally suitable in the past, range size remains the same but local amelioration has led to a strong local increase in &lt;em&gt;N&lt;/em&gt;&lt;sub&gt;e&lt;/sub&gt;. C) Sampled populations inhabit areas outside the species range in the past. BSP recovers a steep increase in &lt;em&gt;N&lt;/em&gt;&lt;sub&gt;e&lt;/sub&gt; associated with founder event.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 1." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2019/12/13/2019.12.13.870410/F1.medium.gif" width="440" height="167"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 1." src="https://www.biorxiv.org/content/biorxiv/early/2019/12/13/2019.12.13.870410/F1.medium.gif" width="440" height="167"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/12/13/2019.12.13.870410/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1." data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/12/13/2019.12.13.870410/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml"><span class="fig-label">Figure 1.</span> <p id="p-7" class="first-child">Three scenarios that might lead to an increase in <em>N</em><sub>e</sub> without a change in range size. The top half of each panel represents a pair of schematic maps of the species range at the LGM (right) and present day (left). The density of colour within each range ellipse shows population density. Circles on the map represent the genetic sampling location. The scenarios are: A) An increase in population size without a change in range recovers an increasing BSP profile. B) Core area today was only marginally suitable in the past, range size remains the same but local amelioration has led to a strong local increase in <em>N</em><sub>e</sub>. C) Sampled populations inhabit areas outside the species range in the past. BSP recovers a steep increase in <em>N</em><sub>e</sub> associated with founder event.</p><div class="sb-div caption-clear"></div></div></div><p id="p-8">The extent to which changes in population density and bottlenecks related to range shifts can override the signals linked to changes in range size and the overall metapopulation size is unknown. In the current paper, we mined GenBank to compile a comprehensive dataset of publicly-available mtDNA sequence data from many species of Holarctic birds, and reconstruct their population dynamics using Bayesian skyline plots (BSP) in BEAST2 (<a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">Bouckaert et al. 2014</a>). A simple prediction, based on the relative changes of habitat types as reconstructed from the pollen record (<a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">Allen et al. 2010</a>), is that species associated with forests (close habitats) should have increased since the LGM, whilst species from open and semi-closed habitats (such as grasslands and steppes) should show a decrease. However, species have more complex niche requirements than a simple association with a broad habitat type, and a more realistic prediction is that <em>N</em><sub>e</sub> should change in line with changes in extent of the potential geographical range, such as that reconstructed by bioclimatic SDMs. We therefore reconstructed changes in modelled potential range extent between the LGM and the present, using paleoclimate reconstructions and Species Distribution Models, and investigated the relationship between <em>N</em><sub>e</sub> changes and range size changes. In the case of wetland species, for which reconstructing detailed individual ranges in the past is challenging because they depend upon habitats whose extent is not easily reconstructed, we compared their demographic reconstructions to species from other major biomes to investigate whether there was any consistent pattern in their response to climatic amelioration in the Holocene.</p></div><div class="section" id="sec-2"><h2 class="">Results</h2><div id="sec-3" class="subsection"><h3>Summary of available BSPs</h3><p id="p-9">Based on criteria of having a minimum of 10 individuals sequenced for either the NADH dehydrogenase subunit 2 (ND2) or cytochrome b (cytb) genes from the mitochondrial genome (mtDNA), a scan of GenBank yielded a preliminary dataset of 208 species. From these, datasets were discarded for the following reasons: insufficient haplotypes captured for demographic reconstruction (&lt; five), insufficient sequence length (&lt; 200bp), sequences across studies not from comparable sections of the gene, haplotype frequencies not published, an inappropriate sampling strategy used by the original study (e.g. non-random sampling, localised island populations) or extensive population sub-structure (see Materials and Methods for details on the criteria used to select suitable datasets). Application of these criteria left 167 datasets for BSP analyses. All these datasets were analysed with BEAST using a Bayesian Skyline Plot and, with one exception (King Eider, <em>Somateria spectabilis</em>), they converged successfully. We note that in BEAST, we adopted a strategy of resizing the ‘<em>bGroupSizes</em>’ parameter (see Materials and Methods), potentially constraining the level of detail recoverable in the profiles but ensuring that a large volume of variable quality datasets could be analysed using the same settings (and thus providing comparable estimates).</p><p id="p-10">Data for both ND2 and cytb were available for 28 species. For 18 species, expansion times and profiles across both genes were consistent, and a single profile was then selected to illustrate the population history. For ten species, the two genes showed discordant demographic histories but, in 7 of these cases, one dataset was of appreciably lower quality (e.g. fewer samples, shorter sequences, inappropriate sampling strategy) and was removed. Three species were rejected because the two genes gave discordant profiles despite both appearing to be of comparable quality. See Supplementary Table 1 for details of datasets dropped.</p><p id="p-11">Further profiles had to be excluded as: profiles were either too deep (limit = &gt;1,000,000 years before the present, <em>n</em>=9) or too short (limit = &lt;5,000 years before the present, <em>n</em>=4) to be informative for the last deglaciation, or profiles showed patterns of expansion or contraction that predated the time period of interest, Marine Isotope Stage 3 (~60 kya, <em>n</em>=18)(<a id="xref-ref-46-1" class="xref-bibr" href="#ref-46">Van Meerbeeck et al. 2009</a>). Note that, even among the profiles we accepted, there remains great variation in depth due to the sparser and more stochastic branching patterns at the bases of the trees, which cause many profiles either to truncate or to ‘flatline’. Thus, the oldest population size estimates tend to be approximations both because of the reduced information content that impacts all profiles, and the need to use the points of truncation in short profiles or flatline states as the oldest size. For the two species where multiple lineages were identified (pine grosbeak, <em>Pinicola enucleator,</em> and horned lark, <em>Eremophila alpestris</em>), two separate BSP analyses were performed and an average of the estimates was taken for downstream analysis.</p><p id="p-12">Applying the above filters left 102 qualifying species BSP profiles for further analysis. These species inhabit a wide range of habitats Closed (<em>n</em>=43), Open (<em>n</em>=17), Semi-closed (<em>n</em>=25), Wetlands (<em>n</em>=12), and Other (<em>n</em>=5); see Materials and Methods for a description of how habitats were grouped. There was no indication that species associated with particular habitats were more or less likely to be excluded (<em>p</em> = 0.77, Fisher’s Exact Test, excluding the ‘Other’ category as it had too few species for testing). Skyline profiles encompass a wide range of shapes, variously exhibiting a single sharp point of inflection, gradual changes in size and multiple points of change. No significant differences were found in the proportion of ND2/cytb genes in each habitat type (<em>p</em> = 0.78, Fisher’s Exact Test, excluding the ‘Other’ category), nor in the proportions of species from the Palearctic, Nearctic or Holarctic in each habitat type (<em>p</em> = 0.10, Fisher’s Exact Test, excluding the ‘Other’ category).</p></div><div id="sec-4" class="subsection"><h3>Direction and magnitude of demographic change</h3><p id="p-13">Only 3 out of 102 species showed an overall decrease in <em>N</em><sub>e</sub> over time, with 7 showing no sizeable change, all other species (<em>n</em>=92) increasing to some degree. The direction of change was not associated with habitat (<em>p</em> = 0.457, Fisher’s Exact test, excluding the ‘Other’ category), nor was its magnitude (gls <em>p</em> ≥ 0. 402 for backbone E and <em>p</em> ≥ 0.386 for backbone H, lm without phylogenetic correction <em>p</em> = 0.667; <a id="xref-fig-2-1" class="xref-fig" href="#F2">Fig. 2A</a>). This result is rather extreme, but it could be the consequence of most species for which several samples are available in GenBank being relatively common and thus having thrived in the Holocene.</p><div id="F2" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/12/13/2019.12.13.870410/F2.large.jpg?width=800&amp;height=600&amp;carousel=1" title="A) Beanplot showing the log relative difference in effective population size (Ne) from 60kya or start of the profile for species from each habitat type. Kernels represent density (i.e. frequency distribution), each small line an individual population, thick black line is median of species-specific values for the given habitat class. Numbers of species per group are; Closed (n=43), Open (n=17), Semi-closed (n=25), Wetlands (n=12). B) Beanplot showing the log relative difference in modelled range extent from 21kya for species from each habitat type. Kernels represent density (i.e. frequency distribution), each small line an individual population, thick black line is median of species-specific values for the given habitat class. Numbers of species per group are; Closed (n=40), Open (n=15), Semi-closed (n=25), Wetlands (n=11). C) Scatterplot of log ratio of Ne from 60 kya to 5 kya in relation to the log ratio of change in size of climatically suitable area from 21 kya to the present, based upon species&#x2019; individual bioclimate SDMs. D) Scatterplot of log change in Ne in relation to the proportion of the species&#x2019; contemporary range that was also suitable during the LGM. In both scatter plots numbers of species per group are; Closed (n=40), Open (n=15), Other (n=5), Semi-closed (n=25), Wetlands (n=11)." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1292071231" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;A) Beanplot showing the log relative difference in effective population size (&lt;em&gt;N&lt;/em&gt;&lt;sub&gt;e&lt;/sub&gt;) from 60kya or start of the profile for species from each habitat type. Kernels represent density (i.e. frequency distribution), each small line an individual population, thick black line is median of species-specific values for the given habitat class. Numbers of species per group are; Closed (&lt;em&gt;n&lt;/em&gt;=43), Open (&lt;em&gt;n&lt;/em&gt;=17), Semi-closed (&lt;em&gt;n&lt;/em&gt;=25), Wetlands (&lt;em&gt;n&lt;/em&gt;=12). B) Beanplot showing the log relative difference in modelled range extent from 21kya for species from each habitat type. Kernels represent density (i.e. frequency distribution), each small line an individual population, thick black line is median of species-specific values for the given habitat class. Numbers of species per group are; Closed (&lt;em&gt;n&lt;/em&gt;=40), Open (&lt;em&gt;n&lt;/em&gt;=15), Semi-closed (&lt;em&gt;n&lt;/em&gt;=25), Wetlands (&lt;em&gt;n&lt;/em&gt;=11). C) Scatterplot of log ratio of &lt;em&gt;N&lt;/em&gt;&lt;sub&gt;e&lt;/sub&gt; from 60 kya to 5 kya in relation to the log ratio of change in size of climatically suitable area from 21 kya to the present, based upon species&#x2019; individual bioclimate SDMs. D) Scatterplot of log change in &lt;em&gt;N&lt;/em&gt;&lt;sub&gt;e&lt;/sub&gt; in relation to the proportion of the species&#x2019; contemporary range that was also suitable during the LGM. In both scatter plots numbers of species per group are; Closed (&lt;em&gt;n&lt;/em&gt;=40), Open (&lt;em&gt;n&lt;/em&gt;=15), Other (&lt;em&gt;n&lt;/em&gt;=5), Semi-closed (&lt;em&gt;n&lt;/em&gt;=25), Wetlands (&lt;em&gt;n&lt;/em&gt;=11).&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 2." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2019/12/13/2019.12.13.870410/F2.medium.gif" width="440" height="416"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 2." src="https://www.biorxiv.org/content/biorxiv/early/2019/12/13/2019.12.13.870410/F2.medium.gif" width="440" height="416"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/12/13/2019.12.13.870410/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2." data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/12/13/2019.12.13.870410/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 2.</span> <p id="p-14" class="first-child">A) Beanplot showing the log relative difference in effective population size (<em>N</em><sub>e</sub>) from 60kya or start of the profile for species from each habitat type. Kernels represent density (i.e. frequency distribution), each small line an individual population, thick black line is median of species-specific values for the given habitat class. Numbers of species per group are; Closed (<em>n</em>=43), Open (<em>n</em>=17), Semi-closed (<em>n</em>=25), Wetlands (<em>n</em>=12). B) Beanplot showing the log relative difference in modelled range extent from 21kya for species from each habitat type. Kernels represent density (i.e. frequency distribution), each small line an individual population, thick black line is median of species-specific values for the given habitat class. Numbers of species per group are; Closed (<em>n</em>=40), Open (<em>n</em>=15), Semi-closed (<em>n</em>=25), Wetlands (<em>n</em>=11). C) Scatterplot of log ratio of <em>N</em><sub>e</sub> from 60 kya to 5 kya in relation to the log ratio of change in size of climatically suitable area from 21 kya to the present, based upon species’ individual bioclimate SDMs. D) Scatterplot of log change in <em>N</em><sub>e</sub> in relation to the proportion of the species’ contemporary range that was also suitable during the LGM. In both scatter plots numbers of species per group are; Closed (<em>n</em>=40), Open (<em>n</em>=15), Other (<em>n</em>=5), Semi-closed (<em>n</em>=25), Wetlands (<em>n</em>=11).</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-5" class="subsection"><h3>Direction and magnitude of change in extent of the potential geographical range</h3><p id="p-15">To investigate the plausibility of climate-driven changes in the extent of climatically suitable area contributing to the overwhelming majority of profiles showing an increase in <em>N</em><sub>e</sub>, we created individual Species Distribution Models (SDMs) with the R package ‘<em>biomod2</em>’ (<a id="xref-ref-60-1" class="xref-bibr" href="#ref-60">Thuiller et al. 2019</a>) for the each of our 102 species. We used occurrences from the GBIF dataset, keeping samples with coordinate data accurate to 10km, removing observations outside the breeding range (defined as the area mapped by Birdlife (<a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">BirdLife International and Handbook of the Birds of the World 2018</a>) as being occupied by resident and breeding migratory populations). We extracted for each species uncorrelated environmental variables for the present day and the LGM (R.M. <a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">Beyer et al. 2019</a>). We thinned the dataset to reduce spatial sorting bias (<a id="xref-ref-29-1" class="xref-bibr" href="#ref-29">Hijmans 2012</a>) and randomly sampled 5 sets of pseudoabsences in the same number as presences from outside the breeding and residential areas. We ran models following four different algorithms (<a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">Bagchi et al. 2013</a>) and created ensembles (<a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">Araújo and New 2007</a>) by validating each by spatial cross-validation (<a id="xref-ref-53-1" class="xref-bibr" href="#ref-53">Roberts et al. 2017</a>). In the end, we had credible SDMs for 96 species; 5 species had to be excluded as we there were insufficient observation points left for analysis after data thinning, and one species was rejected as its SDMs led to a present day projection much larger than the observed range (Supplementary table 2). For the valid SDMs, we generated projected ranges for the LGM (21 kya) and present day and quantified the changes in range size.</p><p id="p-16">As was the case for <em>N</em><sub>e</sub> changes, the majority of species showed an increase in reconstructed range extent since the LGM (76 out of 96). However, the proportion of species showing an increase in range extent was significantly smaller than the proportion with increased <em>N</em><sub>e</sub> (<em>p</em> = 0.004, data subset to the 96 species for which both analyses were available). Whilst there was variation among groups of species associated with different habitats in the magnitude of change in range extent (gls with phylogenetic correction: <em>p</em>≤0.0001 for all 1000 resolutions of both backbone E and backbone H, lm without phylogenetic correction <em>p</em> = 0.0001, <a id="xref-fig-2-2" class="xref-fig" href="#F2">Fig. 2B</a>) and the direction of change in range extent (<em>p</em> = 0.009, Fisher’s Exact Test, excluding the ‘Other’ category), there was no significant match between the direction of the trend in BSP and SDM reconstructions (Supplementary Fig. 1, Fisher’s Exact Test <em>p</em> = 0.587). Neither was there a significant positive correlation across species between the signed magnitudes of the changes in the two measures (<a id="xref-fig-2-3" class="xref-fig" href="#F2">Fig. 2C</a>; gls with phylogenetic correction: <em>p</em>≥0.994 for all 1000 resolutions of backbone E and 1000 resolutions of backbone H, lm without phylogenetic correction <em>p</em> = 0.994). Furthermore, taking into account changes in the location of the range (i.e. the proportion of overlap between LGM and present range) also failed to explain the changes in <em>N</em><sub>e</sub> as reconstructed by BSP (<a id="xref-fig-2-4" class="xref-fig" href="#F2">Fig. 2D</a>; gls <em>p</em>≥ 0.734 for backbone E and <em>p</em>≥ 0.734 for backbone H, lm without phylogenetic correction <em>p</em> = 0.734).</p></div><div id="sec-6" class="subsection"><h3>Timing of change</h3><p id="p-17">We next explored the relationship between the timing of the dominant population size change and habitat type (excluding the Other category, as it was heterogeneous and only had 5 species). The timings of change in size for each population in the four habitat types are presented in <a id="xref-fig-3-1" class="xref-fig" href="#F3">Fig. 3A</a>. Major size change events in wetland-associated species tended to be more recent than for species from the other three habitats (gls <em>p</em>≤0.044 in 1000 resolutions of backbone E and <em>p</em>≤0.044 in 1000 resolutions of backbone H, lm without phylogenetic correction <em>p</em> = 0.044). Similar results were obtained when we excluded species which changed less than 10% in <em>N</em><sub>e</sub> (which might have added noise) (Supplementary Fig. 2). When using molecular evolution rates from Nabholz et al. (<a id="xref-ref-48-1" class="xref-bibr" href="#ref-48">Nabholz et al. 2016</a>) ‘Calibration set 4’ the timing of all expansions are generally consistent with a response associated with the Last Glacial Maximum (LGM). However, we note that using the rate from ‘Calibration set 2’ (which includes older nodes than set 4) would recover older expansion dates (data not shown); given that such dates would correspond to periods of high ice coverage they seem less likely.</p><div id="F3" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/12/13/2019.12.13.870410/F3.large.jpg?width=800&amp;height=600&amp;carousel=1" title="A) Beanplot showing time of dominant effective population size change events for species from each habitat type. Kernels represent density (i.e. frequency distribution), each small line the time of an individual population&#x2019;s size change event (increase or decrease). Thick black line is median of species-specific change times for a given habitat class. Numbers of species per group are; Closed (n=43), Open (n=17), Semi-closed (n=25), Wetlands (n=12). B) A multi-species index (MSI) depicting normalised changed in size averaged across species within each habitat for each time point." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1292071231" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;A) Beanplot showing time of dominant effective population size change events for species from each habitat type. Kernels represent density (i.e. frequency distribution), each small line the time of an individual population&#x2019;s size change event (increase or decrease). Thick black line is median of species-specific change times for a given habitat class. Numbers of species per group are; Closed (&lt;em&gt;n&lt;/em&gt;=43), Open (&lt;em&gt;n&lt;/em&gt;=17), Semi-closed (&lt;em&gt;n&lt;/em&gt;=25), Wetlands (&lt;em&gt;n&lt;/em&gt;=12). B) A multi-species index (MSI) depicting normalised changed in size averaged across species within each habitat for each time point.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 3." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2019/12/13/2019.12.13.870410/F3.medium.gif" width="298" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 3." src="https://www.biorxiv.org/content/biorxiv/early/2019/12/13/2019.12.13.870410/F3.medium.gif" width="298" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/12/13/2019.12.13.870410/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 3." data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/12/13/2019.12.13.870410/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 3.</span> <p id="p-18" class="first-child">A) Beanplot showing time of dominant effective population size change events for species from each habitat type. Kernels represent density (i.e. frequency distribution), each small line the time of an individual population’s size change event (increase or decrease). Thick black line is median of species-specific change times for a given habitat class. Numbers of species per group are; Closed (<em>n</em>=43), Open (<em>n</em>=17), Semi-closed (<em>n</em>=25), Wetlands (<em>n</em>=12). B) A multi-species index (MSI) depicting normalised changed in size averaged across species within each habitat for each time point.</p><div class="sb-div caption-clear"></div></div></div><p id="p-19">For an alternative view of when the expansions occurred, we further used a multi-species index (MSI). The MSI depicts normalised changes in size averaged across species within each habitat type for each time point (<a id="xref-fig-3-2" class="xref-fig" href="#F3">Fig. 3B</a>). Despite exploiting a different aspect of the BSP profile shape, mean change at each time point rather than a single mean date of maximum change, MSI profiles reveal a pattern that is strongly supportive of the previous result where wetland species expand appreciably later than species in the other three habitats.</p></div></div><div class="section" id="sec-7"><h2 class="">Discussion</h2><p id="p-20">We generated a large collection of mitochondrial DNA datasets from many bird species to look for evidence of habitat-associated trends in population size through time. Although variable data quality may lead to uncertainties about the magnitude of any particular change in population size we detect, the direction of change is relatively robust (<a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">Grant 2015</a>). Out of 102 species, only three species show an overall decrease in effective population size. Changes during the last deglaciation in the modelled extent of the geographical range also indicated increases for most species, though the proportion of increases was lower than for <em>N</em><sub>e</sub>. However, we could find no association across species between the direction or magnitude of change in <em>N</em><sub>e</sub> and habitat or range reconstructions.</p><p id="p-21">Species with very large ranges at present are the likely “winners” in terms of their response to climatic change and are thus more likely to show an increase in range and population size from the LGM. Widespread species might also be more likely to have been sampled. Indeed, we found that the majority of species we sampled showed an increase in range size based on SDMs. However, the proportion of expanding species according to SDMs was much lower than the one observed for BSPs, thus failing to fully explain the ubiquity of expanding BSPs, and there was no statistical association between changes since the LGM as reconstructed by BSP and SDM.</p><p id="p-22">Colonisation bottlenecks during a range shifts can, in principle, lead to an increase BSP irrespective of the overall change in range size, as long as migration is low enough (i.e. if the BSP captures the local dynamics in a given population/small geographic region rather than the whole range). However, if this mechanism was important, we would expect species exhibiting an increase in BSP despite a range contraction to be associated with large shifts; this is not the case (Supplementary Fig. 3). Therefore, colonisation bottlenecks do not seem to explain the ubiquity of expanding BSPs in our dataset.</p><p id="p-23">Increases in migration can also lead to an increasing BSP without any change in census population sizes. The potential role of migration in producing counterintuitive <em>N</em><sub>e</sub> estimates when assuming panmictic populations for a whole species has received much attention recently in the context of interpreting cross-coalescence (MSMC) profiles (<a id="xref-ref-45-1" class="xref-bibr" href="#ref-45">Mazet et al. 2015</a>). However, it is difficult to envision a scenario where migration would increase significantly in the face of a range contraction; the effect of migration is more likely to be seen during an expansion, when previously isolated fragments are reconnected. Thus, it seems unlikely that migration can explain the ubiquity of increasing BSPs in Holarctic birds.</p><p id="p-24">A final, more likely but difficult to test explanation for our results is that population densities have increased since the LGM. Thus, even for species that have experienced a range contraction, there might have been changes in local population dynamics such that the average density is higher at present. This decoupling of range extent and density makes interpretation of the SDMs and BSPs very challenging. To resolve them, there is a need to use species abundance models that explicitly predict population densities rather than presence/absence (<a id="xref-ref-51-1" class="xref-bibr" href="#ref-51">Potts and Elith 2006</a>; <a id="xref-ref-34-1" class="xref-bibr" href="#ref-34">Howard et al. 2014</a>; <a id="xref-ref-37-1" class="xref-bibr" href="#ref-37">Johnston et al. 2015</a>). Whilst fitting such models is possible in principle, they have been little used because extensive population density information is rarely available for any given species. However, recent efforts such as those by the Cornell Lab of Ornithology (<a href="https://ebird.org/science/status-and-trends/">https://ebird.org/science/status-and-trends/</a>)(<a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">Auer et al. 2019</a>) have started collating such information, opening a window in better understanding the link between range size and density. Our results, however, do raise a caveat in the interpretation of SMDs and BSPs for extinct species; arguably, the best strategy is to couple the two approaches, as only their combined results might provide a good overview of the fate of a species.</p><p id="p-25">The timings we find for when population expansions occurred agree broadly with those of changes in climate after the LGM. Dates were based on mitochondrial mutation rates calibrated for body size and based on a calibration set that included relatively young species splits (<a id="xref-ref-48-2" class="xref-bibr" href="#ref-48">Nabholz et al. 2016</a>), and thus likely to give faster mutation rates (i.e. less affected by selection) that were appropriate for within species analysis (<a id="xref-ref-35-1" class="xref-bibr" href="#ref-35">Howell et al. 2003</a>; <a id="xref-ref-50-1" class="xref-bibr" href="#ref-50">Penny 2005</a>; <a id="xref-ref-31-1" class="xref-bibr" href="#ref-31">Ho 2007</a>). We note that using a calibration set that included older species splits (<a id="xref-ref-48-3" class="xref-bibr" href="#ref-48">Nabholz et al. 2016</a>) would lead to much older (well before the last glaciation), and thus less realistic changes in effective population sizes. However, we strongly caution the reader that mutation rates calibrated by bird body size, whilst the best available option for comparative analysis, are likely to be very noisy, and individual species estimates should not overly interpreted. Ideally, we would need taxon-specific mutation rates (<a id="xref-ref-33-1" class="xref-bibr" href="#ref-33">Hope et al. 2014</a>) which are simply not available for the number of species we are investigating in this study. Having said this, the fact that species from the same habitat tend to yield broadly similar profiles gives us confidence that the relative timings are likely robust, even if the absolute values still have room for improvement.</p><p id="p-26">We found that changes in <em>N</em><sub>e</sub> for wetland-associated species have occurred more recently than those from terrestrial habitats. Although the significance is marginal when based on point estimates for the date of most rapidly changing size, the finding is supported by multispecies index analysis, which also reveals a pattern of later expansion among wetland species. Compared to many terrestrial habitats, wetlands tend to be less stable. Factors such as local water table levels, the amount of meltwater from retreating ice-sheets and rates of soil erosion all play into wetland habitat development and could have delayed the establishment of stable wetland habitats after the LGM. Although reconstruction of wetland environments and the modelling of wetland recovery is difficult (<a id="xref-ref-38-1" class="xref-bibr" href="#ref-38">Kaplan 2002</a>; <a id="xref-ref-63-1" class="xref-bibr" href="#ref-63">Valdes et al. 2005</a>; <a id="xref-ref-41-1" class="xref-bibr" href="#ref-41">Lafleur 2008</a>; <a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">Fan and Miguez-Macho 2011</a>), analysis of pollen across Eurasia shows that species associated with wetlands such as Sphagnum moss and Alder trees both exhibit much later expansions compared with terrestrial species (<a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">Allen et al. 1999</a>; <a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">Giesecke et al. 2017</a>). Indeed, the expansion of alder relative to other trees (<a id="xref-ref-2-2" class="xref-bibr" href="#ref-2">Allen et al. 1999</a>) matches closely the relatively later expansion we find for wetland versus non-wetland birds.</p><p id="p-27">BSP analysis is powerful but depends on a number of assumptions that are rarely met in real data, most notably the use of a random sample of individuals drawn from a panmictic population (<a id="xref-ref-49-1" class="xref-bibr" href="#ref-49">Pannell 2003</a>; <a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">Chikhi et al. 2010</a>; <a id="xref-ref-28-3" class="xref-bibr" href="#ref-28">Heller et al. 2013</a>). Consequently, most profiles should be seen as approximations that are easy to over-interpret (<a id="xref-ref-25-2" class="xref-bibr" href="#ref-25">Grant 2015</a>; <a id="xref-ref-47-2" class="xref-bibr" href="#ref-47">Miller et al. 2018</a>). However, increasing numbers of public domain datasets open the door for studies based on characteristics averaged across multiple profiles constructed from species or populations that share a common habitat or other trait. This averaging approach is not without its own significant challenges and the data still need stringent filtering. In our study, we constructed and inspected network diagrams for each dataset, allowing species with genetic outliers and evidence of strong population substructure to be identified and either divided or excluded. The large number of species investigated allowed us to see a clear pattern of population expansion in almost all species following the LGM, irrespective of their range dynamics, and a tendency for the expansion to occur later in wetland species. The near-ubiquitous signal of expansion suggests a decoupling of range size and local densities, implying a need for carefully interpretation of BSP to describe species-wide responses.</p></div><div class="section" id="sec-8"><h2 class="">Materials and Methods</h2><div id="sec-9" class="subsection"><h3>Raw Genetic Data</h3><p id="p-28">We assembled two databases, one for NADH dehydrogenase subunit 2 (ND2) and one for cytochrome b (cytb); these two genes are among the most frequently uploaded avian mtDNA loci in GenBank. We first collated summary information on all available avian ND2 and cytb sequences in GenBank using a custom R script, and then screened it for Holarctic species using the list of Voous et al. (<a id="xref-ref-66-1" class="xref-bibr" href="#ref-66">Voous 1977</a>). We only retained species with more than 10 accession for either gene; when a species had sufficient data for both ND2 and cytb, sequences for each gene were extracted and handled as distinct datasets.</p></div><div id="sec-10" class="subsection"><h3>Alignment</h3><p id="p-29">Sequence data for each species / gene combination come from multiple independent studies and often differ in the gene region they analyse. Comparable regions were found by aligning sequences in MEGA (version 7.0; (<a id="xref-ref-40-1" class="xref-bibr" href="#ref-40">Kumar et al. 2016</a>) using the programme ClustalW (<a id="xref-ref-59-1" class="xref-bibr" href="#ref-59">Thompson et al. 1994</a>). Sequence data for each taxon were then trimmed to the longest common section between all samples. If inclusion of a single sequence required the loss of &gt;200bp from more than 50% of the other sequences, that sample was excluded. Furthermore, all positions containing insertions, deletions or sequencing ambiguities were removed. When studies uploaded only one copy of each haplotype, we used haplotype frequencies from the associated publications to generate the appropriate number of copies in our database. Publications that lacked haplotype frequency were excluded from the analysis. After frequency correction the available sample sizes varied from 11 to 453 sequences per species, with lengths from 236 base pairs (bp) to 1137 bp.</p></div><div id="sec-11" class="subsection"><h3>Median Joining Networks</h3><p id="p-30">For each species / gene combination, we built a median joining network (MJNs) in POPART (<a id="xref-ref-42-1" class="xref-bibr" href="#ref-42">Leigh and Bryant 2015</a>). If the MJN contained long branches, defined as 30 or more nucleotide substitutions on a single branch, the sampling location for that species was reviewed because such long branches are indicative of profound population substructure. If clear geographical separation or grouping was found, the data were divided as appropriate and treated as discrete datasets. Single samples with &gt;30 mutations on a branch were considered extreme outliers and dropped from alignments.</p></div><div id="sec-12" class="subsection"><h3>Mutation Rate</h3><p id="p-31">Recent work (<a id="xref-ref-48-4" class="xref-bibr" href="#ref-48">Nabholz et al. 2016</a>) proposes that body-mass can be used to inform more accurate calculations of taxon-specific substitution rates and provides a correction factor for variation in rates according to body mass as well as major mtDNA loci. We created dataset-specific molecular evolution rates using the body mass / gene correction factors from <a id="xref-ref-48-5" class="xref-bibr" href="#ref-48">Nabholz et al. (2016)</a> ‘Calibration set 4’ (3<sup>rd</sup> codon position), as it includes younger species splits that should lead to estimates more appropriate for within the within species dynamics that we investigated in our paper. Due to the uncertainty surrounding mutation rates, analyses based on ‘Calibration set 2’ were also run (data not shown). Body mass data was taken from <a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">Dunning et al. (2007)</a>.</p></div><div id="sec-13" class="subsection"><h3>BSP Analysis</h3><p id="p-32">Whilst there now exist a range of related skyline plot methods (see <a id="xref-ref-32-1" class="xref-bibr" href="#ref-32">Ho and Shapiro 2011</a> for a technical review) we focus on BSPs (<a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">Drummond et al. 2005</a>). The relative simplicity of this approach, and its inherent robustness, makes it particularly suitable for the heterogenous quality of datasets investigated in this paper. For each dataset, BSP analyses were implemented in BEAST2 (<a id="xref-ref-27-1" class="xref-bibr" href="#ref-27">Heled and Drummond 2008</a>; <a id="xref-ref-12-2" class="xref-bibr" href="#ref-12">Bouckaert et al. 2014</a>) using a strict clock with a taxon-specific body mass / gene mutation rates, run lengths of 300 million steps sampled every 30,000 steps, with the first 10% discarded as burn-in. The integrated Bayesian application bModelTest was used to select the most appropriate site model and parameters for individual analyses (<a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">Bouckaert 2015</a>) and ‘<em>bGroupSizes</em>’ was set to 3. All other parameters were left as default. Each analysis was run twice and convergence verified by both a visual inspection of MCMC trace output in Tracer v1.6 and confirming that the effective sample size (ESS) values exceed 200 (<a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">Drummond and Rambaut 2007</a>). Demographic reconstructions were then summarised in Tracer v1.6 (<a href="http://tree.bio.ed.ac.uk/software/tracer/">http://tree.bio.ed.ac.uk/software/tracer/</a>) with ‘<em>Number of bins</em>’ set to 500, and plotted in R.</p></div><div id="sec-14" class="subsection"><h3>Inclusion Criteria</h3><p id="p-33">Where data were available for both ND2 and cytb BSP profiles were compared along with summary statistics on each dataset. When profiles were in agreement, the best supported dataset was retained, e.g. largest sample size, longest sequences, to represent that species’ history. If profiles were not concordant but there was a clear disparity in the quality of the datasets, we again kept the profile from the better dataset. If the profiles did not show similar trends, and there was no difference in the data quality, we conservatively rejected both profiles. For inclusion in further analysis, profiles needed to have a history deeper than 5 thousand years ago (kya) but shallower than 1 million years ago, and also have recovered a change event (increase and / or decrease) dated within the last 60 kya.</p></div><div id="sec-15" class="subsection"><h3>Habitat Classification</h3><p id="p-34">The species we considered were associated with a wide range of habitat types, especially in terms of the predominant vegetation. Given the need to use a small number of habitat categories, no classification system can be perfect. We used the expert ornithological opinion of one of us (REG) to classify each species according to the major habitat with which each species is currently associated, based upon descriptions of their natural habitats in a standard work (<a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">BirdLife International and Handbook of the Birds of the World 2018</a>). After initial data quality filtering 138 species were available to be classified this way. The habitat classes selected were Closed (forests), Semi-closed (shrubland and open woodlands), Open (grassland, montane and steppe), and Wetlands (freshwater wetlands). Some species could not be placed in one of these classes and there were other classes (e.g. Rivers) with five or fewer species. These exceptions were grouped into a category referred to as ‘Other’.</p></div><div id="sec-16" class="subsection"><h3>Timing of Expansion</h3><p id="p-35">Identifying a population’s point of expansion from a BSP profile proved difficult given the wide range of shapes present: some populations changed little or very gradually in size, while others showed sharp and / or multiple points of inflection. We chose to identify inflection points using a custom algorithm, however, as timings could be confounded by the capture of local optima we also used visual inspection of the plotted data and a custom R script to review and extract exact timings. The ‘algorithmic’ method used a moving window approach, repeated for five different window sizes. In each window a linear regression was fitted, and its equation used to remove any slope, before fitting a second order polynomial and recording the difference in y-axis value between the mid-point and the average start and end of each window. This method usually identifies one point of maximum turn (increase or decrease) but the visual inspection of each plot allowed us to capture any additional size change events that were present.</p></div><div id="sec-17" class="subsection"><h3>Size change</h3><p id="p-36">To compare the magnitude of estimated population expansions within the period of interest, the relative population size change between 60 kya and 5 kya was calculated. <em>N</em><sub>e</sub> at 60 kya and 5 kya was interpolated for each BSP profile and where profiles were shorter than 60 kya the population was assumed to be the size at the start of profile. Given the uncertainty of molecular methods we prefered to use size estimates from 60 kya / the earliest possible point instead of estimating the size at the height of the LGM (21 kya) where it is more likely the analysis would catch profiles already undergoing demographic change as a result of climatic events.</p><p id="p-37">We also created a multispecies index (MSI) to further explore the changes in <em>N</em><sub>e</sub>. MSI is a form of average profile based on the average ratio of estimated population size between adjacent time points. Specifically, for each pair of adjacent time points we calculate the geometric mean of the ratios of all species for which data exist. We reconstructed a profile for each broad habitat grouping based on these ratios, working back from the present which is assigned a value of 1.</p></div><div id="sec-18" class="subsection"><h3>Phylogenetic Correction</h3><p id="p-38">The species included in our study are not phylogenetically independent, so all analyses included a phylogenetic correction based on the most complete molecular phylogeny of extant birds (<a href="http://www.birdtree.org">www.birdtree.org</a>, (<a id="xref-ref-36-1" class="xref-bibr" href="#ref-36">Jetz et al. 2012</a>)). <a id="xref-ref-36-2" class="xref-bibr" href="#ref-36">Jetz et al. (2012)</a> present two phylogenetic backbones, the Ericson and Hackett backbones (‘backbone E’ and ‘backbone H’ from now on), and, as we considered both to be equiprobable, all analyses were repeated with both. For each backbone, we generated 1000 trees, randomly resolving polytomies in each, and repeated all analyses for each tree with a <em>“pgls”</em> phylogenetic correction from the Caper package in R. For all analyses, we also provide the results based Ordinary Least Squares (i.e. without phylogenetic correction).</p></div><div id="sec-19" class="subsection"><h3>Species Distribution Models</h3><p id="p-39">The whole pipeline is provided as a commented R script in the supplementary materials.</p><div id="sec-20" class="subsection"><h4>Present day and LGM paleoclimate reconstructions</h4><p id="p-40">In order to identify areas suitable for species through time, high-resolution climate data from the past is needed. We used a 0.5° resolution dataset for 19 bioclimatic variables; Net Primary productivity (NPP), Leaf Area Index (LAI) and all the BioClim variables (<a id="xref-ref-30-1" class="xref-bibr" href="#ref-30">Hijmans et al. 2005</a>) with the exclusion of BIO2 and BIO3; covering the last 21,000 years in 1,000 year time steps (R.M. <a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">Beyer et al. 2019</a>). This dataset was originally constructed from a combination of HadCM3 climate simulations of the last 120,000 years, high-resolution HadAM3H simulations of the last 21,000 years, and empirical present-day data. The data were downscaled and bias-corrected using the Delta Method (R. <a id="xref-ref-8-2" class="xref-bibr" href="#ref-8">Beyer et al. 2019</a>).</p></div><div id="sec-21" class="subsection"><h4>Species data preparation</h4><p id="p-41">Species occurrences were downloaded from the GBIF database (<a href="https://www.gbif.org">https://www.gbif.org</a>) without any preliminary filtering (download links are available in Supplementary table 2). Occurrences were then filtered based on the accuracy of the coordinates (maximum error: 10 km), keeping only observations within the breeding and resident geographical ranges from Birdlife (BirdLife International). The occurrences were then regridded based on the palaeoclimatic reconstructions (0.5 x 0.5 degrees) and, as the method used works on presence / absence and not frequency, only one presence per grid cell was kept.</p><p id="p-42">For each species, this cleaned dataset of presences was then used to select a subset of bioclimatic variables from the 19 variables available in the paleoclimatic reconstructions (R.M. <a id="xref-ref-7-2" class="xref-bibr" href="#ref-7">Beyer et al. 2019</a>). In order to avoid using highly correlated variables, which may increase noise in the data (<a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">Guisan et al. 2017</a>), a correlation matrix was constructed between the variables associated with each presence. Where two values were highly correlated, the variable with the lowest overall correlation across the matrix was retained and this way a set of uncorrelated variables (threshold = 0.7) were selected.</p><p id="p-43">Opportunistic observations, such as those collected in the GBIF database, tend to have geographic biases in sampling effort. In order to reduce the risk of geographic sample bias affecting the SDMs, the dataset was thinned using the R package <em>spThin</em> (<a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">Aiello-Lammens et al. 2015</a>), a minimum distance of 70 km was enforced between observations. Given the random nature of removing nearest-neighbour data points, this process was repeated 100 times (‘rep’ = 100) but in order to keep as much information as possible the result which kept the maximum number of observations after thinning was retained for downstream analysis.</p></div><div id="sec-22" class="subsection"><h4>Species Distribution Model fitting</h4><p id="p-44">Species Distribution Modelling was performed using the R package <em>biomod2</em> (<a id="xref-ref-60-2" class="xref-bibr" href="#ref-60">Thuiller et al. 2019</a>) for all species with more than 10 occurrences after filtering and thinning (<a id="xref-ref-57-1" class="xref-bibr" href="#ref-57">Stockwell and Peterson 2002</a>). The thinned dataset was used as presences, the whole region (i.e. the land mass of Eurasia or North America) as background, and then the same number of pseudo-absences as presences were randomly drawn of from outside the BirdLife resident and breeding masks 5 times, creating 5 independent datasets for further analysis. We found that the most effective strategy to retrieve SDMs for the present day that were consistent with the best available estimates of species’ modern day range, as provided by BirdLife resident and breeding masks, was to draw pseudo-absences from outside these masks. By confirming that the estimated distributions recovered for the modern day were in accord with the BirdLife range predictions we were confident that the modelled niche being projected to the past was with as accurate as possible (an example using the <em>Passer domesticus</em> dataset is given in Supplementary Fig. 4).</p><p id="p-45">Following Bagchi et al. (<a id="xref-ref-6-2" class="xref-bibr" href="#ref-6">Bagchi et al. 2013</a>), models were run independently for each of the five pseudoabsence datasets using four different algorithms: generalised linear models (GLM), generalized boosting method (GBM, in the mentioned reference defined as “boosted regression tree”), generalised additive models (GAM) and random forest. Model evaluation was performed by spatial cross validation (<a id="xref-ref-53-2" class="xref-bibr" href="#ref-53">Roberts et al. 2017</a>), i.e. splitting the dataset (both presences and all five runs of pseudoabsences) based on latitudinal bands in America and longitudinal bands in Eurasia (Supplementary Fig. 5) with the R package <em>BlockCV</em> (<a id="xref-ref-62-1" class="xref-bibr" href="#ref-62">Valavi et al. 2019</a>), and using 4/5 of the splits to calibrate the model and the remaining 1/5 to evaluate it. A data split cannot be used for evaluation if it contains only absences. For this reason, given the great variety of distribution of the species analysed, we decided to maximise the probability of having at least some presences in all data splits by creating 15 spatial blocks encompassing the whole region of interest, either North America (East-West bands) or Europe (North-South bands). Each block was given an ID, numbered sequentially 1-5, the 15 blocks were then grouped into five working data splits by these ID number.</p><p id="p-46">The models were run based on each of the four mentioned algorithms five times (once for each pseudoabsence run), using in turn four of the five defined data splits to calibrate and one to evaluate based on TSS (threshold = 0.7).</p><p id="p-47">A full ensemble, combining all pseudoabsences sets and algorithms (<a id="xref-ref-4-2" class="xref-bibr" href="#ref-4">Araújo and New 2007</a>), was then built using only models with TSS &gt; 0.7 averaged through four different statistics: mean, median, committee average and weighted mean. The statistic showing the highest TSS was then projected to either Eurasia or Northern America considering both the present-day climate and the palaeoclimatic reconstruction for the LGM (defined as 21 kya).</p></div><div id="sec-23" class="subsection"><h4>Range change and overlap</h4><p id="p-48">Finally, the binary projection was used to estimate the climatically suitable area (in square kilometres) for each species both now and in the LGM. In order to do so, we first re-projected the rasters to the Eckert IV equal-area pseudocylindrical projection setting the grid size to 50×50 km, and then multiplied the number of cells occupied in each period, and their overlap, by the cell area (2500 km<sup>2</sup>).</p></div></div></div><div class="section" id="sec-25"><h2 class="">Funding</h2><p id="p-49">E.F.M was supported by the Biotechnology and Biological Sciences Research Council (BBSRC) Doctoral Training Partnerships program (grant code: BB/M011194/1). A.M., R.B. and M.L. were supported by the ERC Consolidator Grant 647787 (“LocalAdaptation”).</p></div><div class="section" id="sec-26"><h2 class="">Data Availability</h2><p id="p-50">All genetic sequence data used in this study were downloaded from the GenBank website — <a href="https://www.ncbi.nlm.nih.gov/genbank/">https://www.ncbi.nlm.nih.gov/genbank/</a></p><p id="p-51">All species occurrence data were downloaded from the Global Biodiversity Information Facility (GBIF) database; DOIs are available for each individual species dataset in the Supplementary file ‘BSP_and_SDM_info.csv’ (Supplementary Table 2).</p></div><div class="section" id="sec-27"><h2 class="">Author contributions</h2><p id="p-52">E.F.M. performed the analyses, produced the plots and wrote the manuscript. All SDMs were constructed by M.L. Expert ornithological knowledge was provided by R.E.G. who also proposed the MSI analysis. All authors provided supervision and comments on the manuscript.</p></div><div class="section" id="sec-28"><h2 class="">Additional information</h2><p id="p-54">The authors declare no competing financial interests.</p></div><div class="section ack" id="ack-1"><h2 class="">Acknowledgements</h2><p id="p-53">We thank Professor Brian Huntley and a group of paleoecologists gathered to honour him upon his retirement for pointing out the similarity of our finding of a later increase in effective population size of wetland birds and a similar delay in the expansion of some wetland plants during the last deglaciation.</p></div><div class="section ref-list" id="ref-list-1"><h2>References</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference  in text" id="ref-1">↵</a><div class="cit ref-cit ref-journal" id="cit-2019.12.13.870410v1.1" data-doi="10.1111/ecog.01132"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Aiello-Lammens</span>  <span class="cit-name-given-names">ME</span></span>, <span class="cit-auth"><span class="cit-name-surname">Boria</span>  <span class="cit-name-given-names">RA</span></span>, <span class="cit-auth"><span class="cit-name-surname">Radosavljevic</span>  <span class="cit-name-given-names">A</span></span>, <span class="cit-auth"><span class="cit-name-surname">Vilela</span>  <span class="cit-name-given-names">B</span></span>, <span class="cit-auth"><span class="cit-name-surname">Anderson</span>  <span class="cit-name-given-names">RP.</span></span> <span class="cit-pub-date">2015</span>. <span class="cit-article-title">spThin: An R package for spatial thinning of species occurrence records for use in ecological niche models</span>. <abbr class="cit-jnl-abbrev">Ecography (Cop)</abbr>. <span class="cit-vol">38</span>(<span class="cit-issue">5</span>):<span class="cit-fpage">541</span>–<span class="cit-lpage">545</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1111/ecog.01132</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEcography%2B%2528Cop%2529%26rft.volume%253D38%26rft.spage%253D541%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fecog.01132%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/ecog.01132&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference  in text" id="ref-2">↵</a><div class="cit ref-cit ref-journal" id="cit-2019.12.13.870410v1.2" data-doi="10.1038/23432"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Allen</span>  <span class="cit-name-given-names">JRM</span></span>, <span class="cit-auth"><span class="cit-name-surname">Brandt</span>  <span class="cit-name-given-names">U</span></span>, <span class="cit-auth"><span class="cit-name-surname">Brauer</span>  <span class="cit-name-given-names">A</span></span>, <span class="cit-auth"><span class="cit-name-surname">Hubberten</span>  <span class="cit-name-given-names">H-W</span></span>, <span class="cit-auth"><span class="cit-name-surname">Huntley</span>  <span class="cit-name-given-names">B</span></span>, <span class="cit-auth"><span class="cit-name-surname">Keller</span>  <span class="cit-name-given-names">J</span></span>, <span class="cit-auth"><span class="cit-name-surname">Kraml</span>  <span class="cit-name-given-names">M</span></span>, <span class="cit-auth"><span class="cit-name-surname">Mackensen</span>  <span class="cit-name-given-names">A</span></span>, <span class="cit-auth"><span class="cit-name-surname">Mingram</span>  <span class="cit-name-given-names">J</span></span>, <span class="cit-auth"><span class="cit-name-surname">Negendank</span>  <span class="cit-name-given-names">JFW</span></span>, <span class="cit-etal">et al.</span> <span class="cit-pub-date">1999</span>. <span class="cit-article-title">Rapid environmental changes in southern Europe during the last glacial period</span>. <abbr class="cit-jnl-abbrev">Nature</abbr>. <span class="cit-vol">400</span>(<span class="cit-issue">6746</span>):<span class="cit-fpage">740</span>–<span class="cit-lpage">743</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1038/23432</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.volume%253D400%26rft.issue%253D6746%26rft.spage%253D740%26rft.epage%253D743%26rft.atitle%253DRapid%2Benvironmental%2Bchanges%2Bin%2BSouthern%2BEurope%2Bduring%2Bthe%2Blast%2Bglacial%2Bperiod%26rft_id%253Dinfo%253Adoi%252F10.1038%252F23432%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/23432&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=1999067708&amp;link_type=GEOREF" class="cit-ref-sprinkles cit-ref-sprinkles-georef"><span>GeoRef</span></a><a href="/lookup/external-ref?access_num=000082131100043&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference  in text" id="ref-3">↵</a><div class="cit ref-cit ref-journal" id="cit-2019.12.13.870410v1.3" data-doi="10.1016/j.quascirev.2010.05.031"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Allen</span>  <span class="cit-name-given-names">JRM</span></span>, <span class="cit-auth"><span class="cit-name-surname">Hickler</span>  <span class="cit-name-given-names">T</span></span>, <span class="cit-auth"><span class="cit-name-surname">Singarayer</span>  <span class="cit-name-given-names">JS</span></span>, <span class="cit-auth"><span class="cit-name-surname">Sykes</span>  <span class="cit-name-given-names">MT</span></span>, <span class="cit-auth"><span class="cit-name-surname">Valdes</span>  <span class="cit-name-given-names">PJ</span></span>, <span class="cit-auth"><span class="cit-name-surname">Huntley</span>  <span class="cit-name-given-names">B.</span></span> <span class="cit-pub-date">2010</span>. <span class="cit-article-title">Last glacial vegetation of Northern Eurasia</span>. <abbr class="cit-jnl-abbrev">Quat Sci Rev</abbr>. <span class="cit-vol">29</span>(<span class="cit-issue">19–20</span>):<span class="cit-fpage">2604</span>–<span class="cit-lpage">2618</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1016/j.quascirev.2010.05.031</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DQuat%2BSci%2BRev%26rft.volume%253D29%26rft.spage%253D2604%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.quascirev.2010.05.031%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.quascirev.2010.05.031&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000281360500014&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference  in text" id="ref-4">↵</a><div class="cit ref-cit ref-journal" id="cit-2019.12.13.870410v1.4" data-doi="10.1016/j.tree.2006.09.010"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Araújo</span>  <span class="cit-name-given-names">MB</span></span>, <span class="cit-auth"><span class="cit-name-surname">New</span>  <span class="cit-name-given-names">M.</span></span> <span class="cit-pub-date">2007</span>. <span class="cit-article-title">Ensemble forecasting of species distributions</span>. <abbr class="cit-jnl-abbrev">Trends Ecol Evol</abbr>. <span class="cit-vol">22</span>(<span class="cit-issue">1</span>):<span class="cit-fpage">42</span>–<span class="cit-lpage">47</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1016/j.tree.2006.09.010</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTrends%2Bin%2BEcology%2B%2526%2BEvolution%26rft.stitle%253DTrends%2Bin%2BEcology%2B%2526%2BEvolution%26rft.aulast%253DSchwartz%26rft.auinit1%253DM.%2BK.%26rft.volume%253D22%26rft.issue%253D1%26rft.spage%253D42%26rft.epage%253D47%26rft.atitle%253DEnsemble%2Bforecasting%2Bof%2Bspecies%2Bdistributions.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.tree.2006.09.010%26rft_id%253Dinfo%253Apmid%252F17011070%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.tree.2006.09.010&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17011070&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F12%2F13%2F2019.12.13.870410.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000243552900010&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference  in text" id="ref-5">↵</a><div class="cit ref-cit ref-web" id="cit-2019.12.13.870410v1.5"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Auer</span>  <span class="cit-name-given-names">T</span></span>, <span class="cit-auth"><span class="cit-name-surname">Fink</span>  <span class="cit-name-given-names">D</span></span>, <span class="cit-auth"><span class="cit-name-surname">Strimas-Mackey</span>  <span class="cit-name-given-names">M.</span></span> <span class="cit-pub-date">2019</span>. <span class="cit-article-title">ebirdst: Tools for loading, plotting, mapping and analysis of eBird Status and Trends data products</span>. <span class="cit-source">R package version 0.1.0</span>. <a href="https://cornelllabofornithology.github.io/ebirdst/">https://cornelllabofornithology.github.io/ebirdst/</a>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference  in text" id="ref-6">↵</a><div class="cit ref-cit ref-journal" id="cit-2019.12.13.870410v1.6" data-doi="10.1111/gcb.12123"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Bagchi</span>  <span class="cit-name-given-names">R</span></span>, <span class="cit-auth"><span class="cit-name-surname">Crosby</span>  <span class="cit-name-given-names">M</span></span>, <span class="cit-auth"><span class="cit-name-surname">Huntley</span>  <span class="cit-name-given-names">B</span></span>, <span class="cit-auth"><span class="cit-name-surname">Hole</span>  <span class="cit-name-given-names">DG</span></span>, <span class="cit-auth"><span class="cit-name-surname">Butchart</span>  <span class="cit-name-given-names">SHM</span></span>, <span class="cit-auth"><span class="cit-name-surname">Collingham</span>  <span class="cit-name-given-names">Y</span></span>, <span class="cit-auth"><span class="cit-name-surname">Kalra</span>  <span class="cit-name-given-names">M</span></span>, <span class="cit-auth"><span class="cit-name-surname">Rajkumar</span>  <span class="cit-name-given-names">J</span></span>, <span class="cit-auth"><span class="cit-name-surname">Rahmani</span>  <span class="cit-name-given-names">A</span></span>, <span class="cit-auth"><span class="cit-name-surname">Pandey</span>  <span class="cit-name-given-names">M</span></span>, <span class="cit-etal">et al.</span> <span class="cit-pub-date">2013</span>. <span class="cit-article-title">Evaluating the effectiveness of conservation site networks under climate change: Accounting for uncertainty</span>. <abbr class="cit-jnl-abbrev">Glob Chang Biol</abbr>. <span class="cit-vol">19</span>(<span class="cit-issue">4</span>):<span class="cit-fpage">1236</span>–<span class="cit-lpage">1248</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1111/gcb.12123</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGlob%2BChang%2BBiol%26rft.volume%253D19%26rft.spage%253D1236%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fgcb.12123%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/gcb.12123&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference  in text" id="ref-7">↵</a><div class="cit ref-cit ref-journal" id="cit-2019.12.13.870410v1.7" data-doi="10.5194/cp-2019-11"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Beyer</span>  <span class="cit-name-given-names">R</span></span>, <span class="cit-auth"><span class="cit-name-surname">Krapp</span>  <span class="cit-name-given-names">M</span></span>, <span class="cit-auth"><span class="cit-name-surname">Manica</span>  <span class="cit-name-given-names">A.</span></span> <span class="cit-pub-date">2019</span>. <span class="cit-article-title">A systematic comparison of bias correction methods for paleoclimate simulations</span>. <abbr class="cit-jnl-abbrev">Clim Past Discuss</abbr>.(February):<span class="cit-fpage">1</span>–<span class="cit-lpage">23</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.5194/cp-2019-11</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DClim%2BPast%2BDiscuss%26rft_id%253Dinfo%253Adoi%252F10.5194%252Fcp-2019-11%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.5194/cp-2019-11&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference  in text" id="ref-8">↵</a><div class="cit ref-cit ref-journal" id="cit-2019.12.13.870410v1.8"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Beyer</span>  <span class="cit-name-given-names">RM</span></span>, <span class="cit-auth"><span class="cit-name-surname">Krapp</span>  <span class="cit-name-given-names">M</span></span>, <span class="cit-auth"><span class="cit-name-surname">Manica</span>  <span class="cit-name-given-names">A.</span></span> <span class="cit-pub-date">2019</span>. <span class="cit-article-title">High-resolution terrestrial climate, bioclimate and vegetation for the last 120,000 years</span>. <abbr class="cit-jnl-abbrev">EarthArXiv</abbr>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference  in text" id="ref-9">↵</a><div class="cit ref-cit ref-journal" id="cit-2019.12.13.870410v1.9"><div class="cit-metadata"><cite><span class="cit-auth cit-collab">BirdLife International</span>. <span class="cit-article-title">BirdLife International and Handbook of the Birds of the World</span>. <span class="cit-pub-date">2018</span>. <abbr class="cit-jnl-abbrev">Bird species distribution maps of the world</abbr>. Version 2018.1. <a href="http://datazone.birdlife.org/species/requestdis">http://datazone.birdlife.org/species/requestdis</a>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference  in text" id="ref-10">↵</a><div class="cit ref-cit ref-journal" id="cit-2019.12.13.870410v1.10"><div class="cit-metadata"><cite><span class="cit-auth cit-collab">BirdLife International and Handbook of the Birds of the World</span>. <span class="cit-pub-date">2018</span>. <abbr class="cit-jnl-abbrev">Bird species distribution maps of the world</abbr>. Version 2018.1.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference  in text" id="ref-11">↵</a><div class="cit ref-cit ref-web" id="cit-2019.12.13.870410v1.11"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Bouckaert</span>  <span class="cit-name-given-names">R.</span></span> <span class="cit-pub-date">2015</span>. <span class="cit-article-title">bModelTest: Bayesian site model selection for nucleotide data</span>. <span class="cit-source">bioRxiv</span>.:<span class="cit-fpage">1</span>–<span class="cit-lpage">17</span>. doi:<a href="http://dx.doi.org/10.1101/020792">http://dx.doi.org/10.1101/020792</a>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference  in text" id="ref-12">↵</a><div class="cit ref-cit ref-journal" id="cit-2019.12.13.870410v1.12" data-doi="10.1371/journal.pcbi.1003963"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Bouckaert</span>  <span class="cit-name-given-names">R</span></span>, <span class="cit-auth"><span class="cit-name-surname">Heled</span>  <span class="cit-name-given-names">J</span></span>, <span class="cit-auth"><span class="cit-name-surname">Kühnert</span>  <span class="cit-name-given-names">D</span></span>, <span class="cit-auth"><span class="cit-name-surname">Vaughan</span>  <span class="cit-name-given-names">T</span></span>, <span class="cit-auth"><span class="cit-name-surname">Wu</span>  <span class="cit-name-given-names">CH</span></span>, <span class="cit-auth"><span class="cit-name-surname">Xie</span>  <span class="cit-name-given-names">D</span></span>, <span class="cit-auth"><span class="cit-name-surname">Suchard</span>  <span class="cit-name-given-names">MA</span></span>, <span class="cit-auth"><span class="cit-name-surname">Rambaut</span>  <span class="cit-name-given-names">A</span></span>, <span class="cit-auth"><span class="cit-name-surname">Drummond</span>  <span class="cit-name-given-names">AJ.</span></span> <span class="cit-pub-date">2014</span>. <span class="cit-article-title">BEAST 2: A Software Platform for Bayesian Evolutionary Analysis</span>. <abbr class="cit-jnl-abbrev">PLoS Comput Biol</abbr>. <span class="cit-vol">10</span>(<span class="cit-issue">4</span>):<span class="cit-fpage">1</span>–<span class="cit-lpage">6</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1371/journal.pcbi.1003537</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BComput%2BBiol%26rft.volume%253D10%26rft.spage%253D1%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pcbi.1003963%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pcbi.1003963&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference  in text" id="ref-13">↵</a><div class="cit ref-cit ref-web" id="cit-2019.12.13.870410v1.13" data-doi="10.1111/ele.12695"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Burbrink</span>  <span class="cit-name-given-names">FT</span></span>, <span class="cit-auth"><span class="cit-name-surname">Chan</span>  <span class="cit-name-given-names">YL</span></span>, <span class="cit-auth"><span class="cit-name-surname">Myers</span>  <span class="cit-name-given-names">EA</span></span>, <span class="cit-auth"><span class="cit-name-surname">Ruane</span>  <span class="cit-name-given-names">S</span></span>, <span class="cit-auth"><span class="cit-name-surname">Smith</span>  <span class="cit-name-given-names">BT</span></span>, <span class="cit-auth"><span class="cit-name-surname">Hickerson</span>  <span class="cit-name-given-names">MJ.</span></span> <span class="cit-pub-date">2016</span>. <span class="cit-article-title">Asynchronous demographic responses to Pleistocene climate change in Eastern Nearctic vertebrates</span>. <span class="cit-source">Ecol Lett</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1111/ele.12695</span><span class="cit-pub-id-sep cit-pub-id-doi-sep">.</span> <a href="http://doi.wiley.com/10.1111/ele.12695">http://doi.wiley.com/10.1111/ele.12695</a>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEcol%2BLett%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fele.12695%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/ele.12695&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference  in text" id="ref-14">↵</a><div class="cit ref-cit ref-journal" id="cit-2019.12.13.870410v1.14" data-doi="10.1186/s12862-016-0817-7"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Calderón</span>  <span class="cit-name-given-names">L</span></span>, <span class="cit-auth"><span class="cit-name-surname">Campagna</span>  <span class="cit-name-given-names">L</span></span>, <span class="cit-auth"><span class="cit-name-surname">Wilke</span>  <span class="cit-name-given-names">T</span></span>, <span class="cit-auth"><span class="cit-name-surname">Lormée</span>  <span class="cit-name-given-names">H</span></span>, <span class="cit-auth"><span class="cit-name-surname">Eraud</span>  <span class="cit-name-given-names">C</span></span>, <span class="cit-auth"><span class="cit-name-surname">Dunn</span>  <span class="cit-name-given-names">J</span></span>, <span class="cit-auth"><span class="cit-name-surname">Rocha</span>  <span class="cit-name-given-names">G</span></span>, <span class="cit-auth"><span class="cit-name-surname">Zehtindjiev</span>  <span class="cit-name-given-names">P</span></span>, <span class="cit-auth"><span class="cit-name-surname">Bakaloudis</span>  <span class="cit-name-given-names">D</span></span>, <span class="cit-auth"><span class="cit-name-surname">Metzger</span>  <span class="cit-name-given-names">B</span></span>, <span class="cit-etal">et al.</span> <span class="cit-pub-date">2016</span>. <span class="cit-article-title">Genome-wide evidence of demographic fluctuations and lack of genetic structure across flyways in a long distance migrant, the European turtle dove</span>. <abbr class="cit-jnl-abbrev">Mol Ecol</abbr>.:<span class="cit-fpage">1</span>–<span class="cit-lpage">11</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1186/s12862-016-0817-7</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol%2BEcol%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fs12862-016-0817-7%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/s12862-016-0817-7&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference  in text" id="ref-15">↵</a><div class="cit ref-cit ref-journal" id="cit-2019.12.13.870410v1.15" data-doi="10.1534/genetics.110.118661"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Chikhi</span>  <span class="cit-name-given-names">L</span></span>, <span class="cit-auth"><span class="cit-name-surname">Sousa</span>  <span class="cit-name-given-names">VC</span></span>, <span class="cit-auth"><span class="cit-name-surname">Luisi</span>  <span class="cit-name-given-names">P</span></span>, <span class="cit-auth"><span class="cit-name-surname">Goossens</span>  <span class="cit-name-given-names">B</span></span>, <span class="cit-auth"><span class="cit-name-surname">Beaumont</span>  <span class="cit-name-given-names">MA.</span></span> <span class="cit-pub-date">2010</span>. <span class="cit-article-title">The confounding effects of population structure, genetic diversity and the sampling scheme on the detection and quantification of population size changes</span>. <abbr class="cit-jnl-abbrev">Genetics</abbr>. <span class="cit-vol">186</span>(<span class="cit-issue">3</span>):<span class="cit-fpage">983</span>–<span class="cit-lpage">995</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1534/genetics.110.118661</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.110.118661%26rft_id%253Dinfo%253Apmid%252F20739713%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTg2LzMvOTgzIjtzOjQ6ImF0b20iO3M6NDg6Ii9iaW9yeGl2L2Vhcmx5LzIwMTkvMTIvMTMvMjAxOS4xMi4xMy44NzA0MTAuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference  in text" id="ref-16">↵</a><div class="cit ref-cit ref-journal" id="cit-2019.12.13.870410v1.16" data-doi="10.2307/177373"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Connor</span>  <span class="cit-name-given-names">EF</span></span>, <span class="cit-auth"><span class="cit-name-surname">Courtney</span>  <span class="cit-name-given-names">AC</span></span>, <span class="cit-auth"><span class="cit-name-surname">Yoder</span>  <span class="cit-name-given-names">JM.</span></span> <span class="cit-pub-date">2000</span>. <span class="cit-article-title">Individuals-Area Relationships: The Relationship between Animal Population Density and Area</span>. <abbr class="cit-jnl-abbrev">Ecology</abbr>. <span class="cit-vol">81</span>(<span class="cit-issue">3</span>):<span class="cit-fpage">734</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.2307/177373</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEcology%26rft.volume%253D81%26rft.spage%253D734%26rft_id%253Dinfo%253Adoi%252F10.2307%252F177373%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2307/177373&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000085611400012&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference  in text" id="ref-17">↵</a><div class="cit ref-cit ref-journal" id="cit-2019.12.13.870410v1.17" data-doi="10.1186/1471-2148-7-214"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Drummond</span>  <span class="cit-name-given-names">AJ</span></span>, <span class="cit-auth"><span class="cit-name-surname">Rambaut</span>  <span class="cit-name-given-names">A.</span></span> <span class="cit-pub-date">2007</span>. <span class="cit-article-title">BEAST: Bayesian evolutionary analysis by sampling trees</span>. <abbr class="cit-jnl-abbrev">BMC Evol Biol</abbr>. <span class="cit-vol">8</span>:<span class="cit-fpage">1</span>–<span class="cit-lpage">8</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1186/1471-2148-7-214</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2BEvol%2BBiol%26rft.volume%253D8%26rft.spage%253D1%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2148-7-214%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2148-7-214&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference  in text" id="ref-18">↵</a><div class="cit ref-cit ref-journal" id="cit-2019.12.13.870410v1.18" data-doi="10.1093/molbev/msi103"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Drummond</span>  <span class="cit-name-given-names">AJ</span></span>, <span class="cit-auth"><span class="cit-name-surname">Rambaut</span>  <span class="cit-name-given-names">A</span></span>, <span class="cit-auth"><span class="cit-name-surname">Shapiro</span>  <span class="cit-name-given-names">B</span></span>, <span class="cit-auth"><span class="cit-name-surname">Pybus</span>  <span class="cit-name-given-names">OG.</span></span> <span class="cit-pub-date">2005</span>. <span class="cit-article-title">Bayesian coalescent inference of past population dynamics from molecular sequences</span>. <abbr class="cit-jnl-abbrev">Mol Biol Evol</abbr>. <span class="cit-vol">22</span>(<span class="cit-issue">5</span>):<span class="cit-fpage">1185</span>–<span class="cit-lpage">1192</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1093/molbev/msi103</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol%2BBiol%2BEvol%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fmolbev%252Fmsi103%26rft_id%253Dinfo%253Apmid%252F15703244%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/molbev/msi103&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15703244&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F12%2F13%2F2019.12.13.870410.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000228700200004&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference  in text" id="ref-19">↵</a><div class="cit ref-cit ref-book" id="cit-2019.12.13.870410v1.19"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Dunning</span>  <span class="cit-name-given-names">J.</span></span> <span class="cit-pub-date">2007</span>. <span class="cit-chapter-title">CRC Handbook of Avian Body Masses, Second Edition</span>. <span class="cit-edition">2nd</span> ed. <span class="cit-publ-name">CRC Press</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference  in text" id="ref-20">↵</a><div class="cit ref-cit ref-journal" id="cit-2019.12.13.870410v1.20" data-doi="10.1146/annurev.ecolsys.110308.120159"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Elith</span>  <span class="cit-name-given-names">J</span></span>, <span class="cit-auth"><span class="cit-name-surname">Leathwick</span>  <span class="cit-name-given-names">JR.</span></span> <span class="cit-pub-date">2009</span>. <span class="cit-article-title">Species Distribution Models: Ecological Explanation and Prediction Across Space and Time</span>. <abbr class="cit-jnl-abbrev">Annu Rev Ecol Evol Syst</abbr>. <span class="cit-vol">40</span>(<span class="cit-issue">1</span>):<span class="cit-fpage">677</span>–<span class="cit-lpage">697</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1146/annurev.ecolsys.110308.120159</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAnnu%2BRev%2BEcol%2BEvol%2BSyst%26rft.volume%253D40%26rft.spage%253D677%26rft_id%253Dinfo%253Adoi%252F10.1146%252Fannurev.ecolsys.110308.120159%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1146/annurev.ecolsys.110308.120159&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference  in text" id="ref-21">↵</a><div class="cit ref-cit ref-journal" id="cit-2019.12.13.870410v1.21" data-doi="10.1007/s00382-010-0829-8"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Fan</span>  <span class="cit-name-given-names">Y</span></span>, <span class="cit-auth"><span class="cit-name-surname">Miguez-Macho</span>  <span class="cit-name-given-names">G.</span></span> <span class="cit-pub-date">2011</span>. <span class="cit-article-title">A simple hydrologic framework for simulating wetlands in climate and earth system models</span>. <abbr class="cit-jnl-abbrev">Clim Dyn</abbr>. <span class="cit-vol">37</span>(<span class="cit-issue">1</span>):<span class="cit-fpage">253</span>–<span class="cit-lpage">278</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1007/s00382-010-0829-8</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DClim%2BDyn%26rft.volume%253D37%26rft.spage%253D253%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00382-010-0829-8%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00382-010-0829-8&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000293403000016&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference  in text" id="ref-22">↵</a><div class="cit ref-cit ref-journal" id="cit-2019.12.13.870410v1.22" data-doi="10.1038/ncomms2714"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Foote</span>  <span class="cit-name-given-names">AD</span></span>, <span class="cit-auth"><span class="cit-name-surname">Kaschner</span>  <span class="cit-name-given-names">K</span></span>, <span class="cit-auth"><span class="cit-name-surname">Schultze</span>  <span class="cit-name-given-names">SE</span></span>, <span class="cit-auth"><span class="cit-name-surname">Garilao</span>  <span class="cit-name-given-names">C</span></span>, <span class="cit-auth"><span class="cit-name-surname">Ho</span>  <span class="cit-name-given-names">SYW</span></span>, <span class="cit-auth"><span class="cit-name-surname">Post</span>  <span class="cit-name-given-names">K</span></span>, <span class="cit-auth"><span class="cit-name-surname">Higham</span>  <span class="cit-name-given-names">TFG</span></span>, <span class="cit-auth"><span class="cit-name-surname">Stokowska</span>  <span class="cit-name-given-names">C</span></span>, <span class="cit-auth"><span class="cit-name-surname">Van Der Es</span>  <span class="cit-name-given-names">H</span></span>, <span class="cit-auth"><span class="cit-name-surname">Embling</span>  <span class="cit-name-given-names">CB</span></span>, <span class="cit-etal">et al.</span> <span class="cit-pub-date">2013</span>. <span class="cit-article-title">Ancient DNA reveals that bowhead whale lineages survived Late Pleistocene climate change and habitat shifts</span>. <abbr class="cit-jnl-abbrev">Nat Commun</abbr>. <span class="cit-vol">4</span>:<span class="cit-fpage">1677</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1038/ncomms2714</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat%2BCommun%26rft.volume%253D4%26rft.spage%253D1677%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fncomms2714%26rft_id%253Dinfo%253Apmid%252F23575681%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ncomms2714&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23575681&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F12%2F13%2F2019.12.13.870410.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference  in text" id="ref-23">↵</a><div class="cit ref-cit ref-journal" id="cit-2019.12.13.870410v1.23" data-doi="10.1111/j.1365-2486.2011.02614.x"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Fordham</span>  <span class="cit-name-given-names">DA</span></span>, <span class="cit-auth"><span class="cit-name-surname">Resit Akçakaya</span>  <span class="cit-name-given-names">H</span></span>, <span class="cit-auth"><span class="cit-name-surname">Araújo</span>  <span class="cit-name-given-names">MB</span></span>, <span class="cit-auth"><span class="cit-name-surname">Elith</span>  <span class="cit-name-given-names">J</span></span>, <span class="cit-auth"><span class="cit-name-surname">Keith</span>  <span class="cit-name-given-names">DA</span></span>, <span class="cit-auth"><span class="cit-name-surname">Pearson</span>  <span class="cit-name-given-names">R</span></span>, <span class="cit-auth"><span class="cit-name-surname">Auld</span>  <span class="cit-name-given-names">TD</span></span>, <span class="cit-auth"><span class="cit-name-surname">Mellin</span>  <span class="cit-name-given-names">C</span></span>, <span class="cit-auth"><span class="cit-name-surname">Morgan</span>  <span class="cit-name-given-names">JW</span></span>, <span class="cit-auth"><span class="cit-name-surname">Regan</span>  <span class="cit-name-given-names">TJ</span></span>, <span class="cit-etal">et al.</span> <span class="cit-pub-date">2012</span>. <span class="cit-article-title">Plant extinction risk under climate change: are forecast range shifts alone a good indicator of species vulnerability to global warming</span>? <abbr class="cit-jnl-abbrev">Glob Chang Biol</abbr>. <span class="cit-vol">18</span>(<span class="cit-issue">4</span>):<span class="cit-fpage">1357</span>–<span class="cit-lpage">1371</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1111/j.1365-2486.2011.02614.x</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGlob%2BChang%2BBiol%26rft.volume%253D18%26rft.spage%253D1357%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1365-2486.2011.02614.x%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1365-2486.2011.02614.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000301533100013&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference  in text" id="ref-24">↵</a><div class="cit ref-cit ref-journal" id="cit-2019.12.13.870410v1.24" data-doi="10.1111/jbi.12974"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Giesecke</span>  <span class="cit-name-given-names">T</span></span>, <span class="cit-auth"><span class="cit-name-surname">Brewer</span>  <span class="cit-name-given-names">S</span></span>, <span class="cit-auth"><span class="cit-name-surname">Finsinger</span>  <span class="cit-name-given-names">W</span></span>, <span class="cit-auth"><span class="cit-name-surname">Leydet</span>  <span class="cit-name-given-names">M</span></span>, <span class="cit-auth"><span class="cit-name-surname">Bradshaw</span>  <span class="cit-name-given-names">RHW.</span></span> <span class="cit-pub-date">2017</span>. <span class="cit-article-title">Patterns and dynamics of European vegetation change over the last 15,000 years</span>. <abbr class="cit-jnl-abbrev">J Biogeogr</abbr>. <span class="cit-vol">44</span>(<span class="cit-issue">7</span>):<span class="cit-fpage">1441</span>–<span class="cit-lpage">1456</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1111/jbi.12974</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ%2BBiogeogr%26rft.volume%253D44%26rft.spage%253D1441%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fjbi.12974%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/jbi.12974&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference  in text" id="ref-25">↵</a><div class="cit ref-cit ref-journal" id="cit-2019.12.13.870410v1.25" data-doi="10.1093/jhered/esv020"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Grant</span>  <span class="cit-name-given-names">WS.</span></span> <span class="cit-pub-date">2015</span>. <span class="cit-article-title">Problems and cautions with sequence mismatch analysis and Bayesian skyline plots to infer historical demography</span>. <abbr class="cit-jnl-abbrev">J Hered</abbr>. <span class="cit-vol">106</span>(<span class="cit-issue">4</span>):<span class="cit-fpage">333</span>–<span class="cit-lpage">346</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1093/jhered/esv020</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ%2BHered%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fjhered%252Fesv020%26rft_id%253Dinfo%253Apmid%252F25926628%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/jhered/esv020&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25926628&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F12%2F13%2F2019.12.13.870410.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference  in text" id="ref-26">↵</a><div class="cit ref-cit ref-book" id="cit-2019.12.13.870410v1.26"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Guisan</span>  <span class="cit-name-given-names">A</span></span>, <span class="cit-auth"><span class="cit-name-surname">Thuiller</span>  <span class="cit-name-given-names">W</span></span>, <span class="cit-auth"><span class="cit-name-surname">Zimmermann</span>  <span class="cit-name-given-names">NE.</span></span> <span class="cit-pub-date">2017</span>. <span class="cit-chapter-title">Habitat Suitability and Distribution Models</span>. <span class="cit-publ-loc">Cambridge</span>: <span class="cit-publ-name">Cambridge University Press</span>. <a href="http://ebooks.cambridge.org/ref/id/CBO9781139028271">http://ebooks.cambridge.org/ref/id/CBO9781139028271</a>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-27-1" title="View reference  in text" id="ref-27">↵</a><div class="cit ref-cit ref-journal" id="cit-2019.12.13.870410v1.27" data-doi="10.1186/1471-2148-8-289"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Heled</span>  <span class="cit-name-given-names">J</span></span>, <span class="cit-auth"><span class="cit-name-surname">Drummond</span>  <span class="cit-name-given-names">AJ.</span></span> <span class="cit-pub-date">2008</span>. <span class="cit-article-title">Bayesian inference of population size history from multiple loci</span>. <abbr class="cit-jnl-abbrev">BMC Evol Biol</abbr>. <span class="cit-vol">15</span>:<span class="cit-fpage">1</span>–<span class="cit-lpage">15</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1186/1471-2148-8-289</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2BEvol%2BBiol%26rft.volume%253D15%26rft.spage%253D1%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2148-8-289%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2148-8-289&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-28-1" title="View reference  in text" id="ref-28">↵</a><div class="cit ref-cit ref-journal" id="cit-2019.12.13.870410v1.28" data-doi="10.1371/journal.pone.0062992"><div class="cit-metadata"><ol class="duplicate"><li><span class="cit-ed"><span class="cit-name-surname">Mailund</span>  <span class="cit-name-given-names">T</span></span></li></ol><cite><span class="cit-auth"><span class="cit-name-surname">Heller</span>  <span class="cit-name-given-names">R</span></span>, <span class="cit-auth"><span class="cit-name-surname">Chikhi</span>  <span class="cit-name-given-names">L</span></span>, <span class="cit-auth"><span class="cit-name-surname">Siegismund</span>  <span class="cit-name-given-names">HR.</span></span> <span class="cit-pub-date">2013</span>. <span class="cit-article-title">The Confounding Effect of Population Structure on Bayesian Skyline Plot Inferences of Demographic History</span>. <span class="cit-ed"><span class="cit-name-surname">Mailund</span>  <span class="cit-name-given-names">T</span></span>, editor. <abbr class="cit-jnl-abbrev">PLoS One</abbr>. <span class="cit-vol">8</span>(<span class="cit-issue">5</span>):<span class="cit-fpage">e62992</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1371/journal.pone.0062992</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BOne%26rft.volume%253D8%26rft.spage%253De62992%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0062992%26rft_id%253Dinfo%253Apmid%252F23667558%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0062992&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23667558&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F12%2F13%2F2019.12.13.870410.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-29-1" title="View reference  in text" id="ref-29">↵</a><div class="cit ref-cit ref-journal" id="cit-2019.12.13.870410v1.29" data-doi="10.1890/11-0826.1"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Hijmans</span>  <span class="cit-name-given-names">RJ.</span></span> <span class="cit-pub-date">2012</span>. <span class="cit-article-title">Cross-validation of species distribution models: Removing spatial sorting bias and calibration with a null model</span>. <abbr class="cit-jnl-abbrev">Ecology</abbr>. <span class="cit-vol">93</span>(<span class="cit-issue">3</span>):<span class="cit-fpage">679</span>–<span class="cit-lpage">688</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1890/11-0826.1</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEcology%26rft.stitle%253DEcology%26rft.aulast%253DHijmans%26rft.auinit1%253DR.%2BJ.%26rft.volume%253D93%26rft.issue%253D3%26rft.spage%253D679%26rft.epage%253D688%26rft.atitle%253DCross-validation%2Bof%2Bspecies%2Bdistribution%2Bmodels%253A%2Bremoving%2Bspatial%2Bsorting%2Bbias%2Band%2Bcalibration%2Bwith%2Ba%2Bnull%2Bmodel.%26rft_id%253Dinfo%253Adoi%252F10.1890%252F11-0826.1%26rft_id%253Dinfo%253Apmid%252F22624221%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1890/11-0826.1&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22624221&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F12%2F13%2F2019.12.13.870410.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000302957300024&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-30-1" title="View reference  in text" id="ref-30">↵</a><div class="cit ref-cit ref-journal" id="cit-2019.12.13.870410v1.30" data-doi="10.1002/joc.1276"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Hijmans</span>  <span class="cit-name-given-names">RJ</span></span>, <span class="cit-auth"><span class="cit-name-surname">Cameron</span>  <span class="cit-name-given-names">SE</span></span>, <span class="cit-auth"><span class="cit-name-surname">Parra</span>  <span class="cit-name-given-names">JL</span></span>, <span class="cit-auth"><span class="cit-name-surname">Jones</span>  <span class="cit-name-given-names">PG</span></span>, <span class="cit-auth"><span class="cit-name-surname">Jarvis</span>  <span class="cit-name-given-names">A.</span></span> <span class="cit-pub-date">2005</span>. <span class="cit-article-title">Very high resolution interpolated climate surfaces for global land areas</span>. <abbr class="cit-jnl-abbrev">Int J Climatol</abbr>. <span class="cit-vol">25</span>(<span class="cit-issue">15</span>):<span class="cit-fpage">1965</span>–<span class="cit-lpage">1978</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1002/joc.1276</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DInt%2BJ%2BClimatol%26rft.volume%253D25%26rft.spage%253D1965%26rft_id%253Dinfo%253Adoi%252F10.1002%252Fjoc.1276%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1002/joc.1276&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000234519700002&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-31-1" title="View reference  in text" id="ref-31">↵</a><div class="cit ref-cit ref-journal" id="cit-2019.12.13.870410v1.31" data-doi="10.1111/j.2007.0908-8857.04168.x"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Ho</span>  <span class="cit-name-given-names">SYM.</span></span> <span class="cit-pub-date">2007</span>. <span class="cit-article-title">Calibrating molecular estimates of substitution rates and divergence times in birds</span>. <abbr class="cit-jnl-abbrev">J Avian Biol</abbr>. <span class="cit-vol">38</span>(<span class="cit-issue">4</span>):<span class="cit-fpage">409</span>–<span class="cit-lpage">414</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1111/j.2007.0908-8857.04168.x</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ%2BAvian%2BBiol%26rft.volume%253D38%26rft.spage%253D409%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.2007.0908-8857.04168.x%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.2007.0908-8857.04168.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-32-1" title="View reference  in text" id="ref-32">↵</a><div class="cit ref-cit ref-journal" id="cit-2019.12.13.870410v1.32" data-doi="10.1111/j.1755-0998.2011.02988.x"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Ho</span>  <span class="cit-name-given-names">SYW</span></span>, <span class="cit-auth"><span class="cit-name-surname">Shapiro</span>  <span class="cit-name-given-names">B.</span></span> <span class="cit-pub-date">2011</span>. <span class="cit-article-title">Skyline-plot methods for estimating demographic history from nucleotide sequences</span>. <abbr class="cit-jnl-abbrev">Mol Ecol Resour</abbr>. <span class="cit-vol">11</span>(<span class="cit-issue">3</span>):<span class="cit-fpage">423</span>–<span class="cit-lpage">434</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1111/j.1755-0998.2011.02988.x</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DMol%2BEcol%2BResour%26rft.aulast%253DHo%26rft.auinit1%253DS.%2BY.%26rft.volume%253D11%26rft.issue%253D3%26rft.spage%253D423%26rft.epage%253D434%26rft.atitle%253DSkyline-plot%2Bmethods%2Bfor%2Bestimating%2Bdemographic%2Bhistory%2Bfrom%2Bnucleotide%2Bsequences.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1755-0998.2011.02988.x%26rft_id%253Dinfo%253Apmid%252F21481200%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1755-0998.2011.02988.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21481200&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F12%2F13%2F2019.12.13.870410.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-33-1" title="View reference  in text" id="ref-33">↵</a><div class="cit ref-cit ref-journal" id="cit-2019.12.13.870410v1.33" data-doi="10.1111/evo.12469"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Hope</span>  <span class="cit-name-given-names">AG</span></span>, <span class="cit-auth"><span class="cit-name-surname">Ho</span>  <span class="cit-name-given-names">SYW</span></span>, <span class="cit-auth"><span class="cit-name-surname">Malaney</span>  <span class="cit-name-given-names">JL</span></span>, <span class="cit-auth"><span class="cit-name-surname">Cook</span>  <span class="cit-name-given-names">JA</span></span>, <span class="cit-auth"><span class="cit-name-surname">Talbot</span>  <span class="cit-name-given-names">SL.</span></span> <span class="cit-pub-date">2014</span>. <span class="cit-article-title">Accounting for rate variation among lineages in comparative demographic analyses</span>. <abbr class="cit-jnl-abbrev">Evolution (N Y)</abbr>. <span class="cit-vol">68</span>(<span class="cit-issue">9</span>):<span class="cit-fpage">2689</span>–<span class="cit-lpage">2700</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1111/evo.12469</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEvolution%2B%2528N%2BY%2529%26rft.volume%253D68%26rft.spage%253D2689%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fevo.12469%26rft_id%253Dinfo%253Apmid%252F24916007%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/evo.12469&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24916007&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F12%2F13%2F2019.12.13.870410.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-34-1" title="View reference  in text" id="ref-34">↵</a><div class="cit ref-cit ref-journal" id="cit-2019.12.13.870410v1.34" data-doi="10.1111/2041-210X.12184"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Howard</span>  <span class="cit-name-given-names">C</span></span>, <span class="cit-auth"><span class="cit-name-surname">Stephens</span>  <span class="cit-name-given-names">PA</span></span>, <span class="cit-auth"><span class="cit-name-surname">Pearce-Higgins</span>  <span class="cit-name-given-names">JW</span></span>, <span class="cit-auth"><span class="cit-name-surname">Gregory</span>  <span class="cit-name-given-names">RD</span></span>, <span class="cit-auth"><span class="cit-name-surname">Willis</span>  <span class="cit-name-given-names">SG.</span></span> <span class="cit-pub-date">2014</span>. <span class="cit-article-title">Improving species distribution models: The value of data on abundance</span>. <abbr class="cit-jnl-abbrev">Methods Ecol Evol</abbr>. <span class="cit-vol">5</span>(<span class="cit-issue">6</span>):<span class="cit-fpage">506</span>–<span class="cit-lpage">513</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1111/2041-210X.12184</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMethods%2BEcol%2BEvol%26rft.volume%253D5%26rft.spage%253D506%26rft_id%253Dinfo%253Adoi%252F10.1111%252F2041-210X.12184%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/2041-210X.12184&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-35-1" title="View reference  in text" id="ref-35">↵</a><div class="cit ref-cit ref-journal" id="cit-2019.12.13.870410v1.35" data-doi="10.1086/368264"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Howell</span>  <span class="cit-name-given-names">N</span></span>, <span class="cit-auth"><span class="cit-name-surname">Smejkal</span>  <span class="cit-name-given-names">CB</span></span>, <span class="cit-auth"><span class="cit-name-surname">Mackey</span>  <span class="cit-name-given-names">DA</span></span>, <span class="cit-auth"><span class="cit-name-surname">Chinnery</span>  <span class="cit-name-given-names">PF</span></span>, <span class="cit-auth"><span class="cit-name-surname">Turnbull</span>  <span class="cit-name-given-names">DM</span></span>, <span class="cit-auth"><span class="cit-name-surname">Herrnstadt</span>  <span class="cit-name-given-names">C.</span></span> <span class="cit-pub-date">2003</span>. <span class="cit-article-title">The Pedigree Rate of Sequence Divergence in the Human Mitochondrial Genome: There Is a Difference Between Phylogenetic and Pedigree Rates</span>. <abbr class="cit-jnl-abbrev">Am J Hum Genet</abbr>. <span class="cit-vol">72</span>(<span class="cit-issue">3</span>):<span class="cit-fpage">659</span>–<span class="cit-lpage">670</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1086/368264</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAmerican%2Bjournal%2Bof%2Bhuman%2Bgenetics%26rft.stitle%253DAm%2BJ%2BHum%2BGenet%26rft.aulast%253DHowell%26rft.auinit1%253DN.%26rft.volume%253D72%26rft.issue%253D3%26rft.spage%253D659%26rft.epage%253D670%26rft.atitle%253DThe%2Bpedigree%2Brate%2Bof%2Bsequence%2Bdivergence%2Bin%2Bthe%2Bhuman%2Bmitochondrial%2Bgenome%253A%2Bthere%2Bis%2Ba%2Bdifference%2Bbetween%2Bphylogenetic%2Band%2Bpedigree%2Brates.%26rft_id%253Dinfo%253Adoi%252F10.1086%252F368264%26rft_id%253Dinfo%253Apmid%252F12571803%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1086/368264&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=12571803&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F12%2F13%2F2019.12.13.870410.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000181152600015&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-36-1" title="View reference  in text" id="ref-36">↵</a><div class="cit ref-cit ref-journal" id="cit-2019.12.13.870410v1.36" data-doi="10.1038/nature11631"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Jetz</span>  <span class="cit-name-given-names">W</span></span>, <span class="cit-auth"><span class="cit-name-surname">Thomas</span>  <span class="cit-name-given-names">GH</span></span>, <span class="cit-auth"><span class="cit-name-surname">Joy</span>  <span class="cit-name-given-names">JB</span></span>, <span class="cit-auth"><span class="cit-name-surname">Hartmann</span>  <span class="cit-name-given-names">K</span></span>, <span class="cit-auth"><span class="cit-name-surname">Mooers</span>  <span class="cit-name-given-names">AO.</span></span> <span class="cit-pub-date">2012</span>. <span class="cit-article-title">The global diversity of birds in space and time</span>. <abbr class="cit-jnl-abbrev">Nature</abbr>. <span class="cit-vol">491</span>(<span class="cit-issue">7424</span>):<span class="cit-fpage">444</span>–<span class="cit-lpage">448</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1038/nature11631</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DJetz%26rft.auinit1%253DW.%26rft.volume%253D491%26rft.issue%253D7424%26rft.spage%253D444%26rft.epage%253D448%26rft.atitle%253DThe%2Bglobal%2Bdiversity%2Bof%2Bbirds%2Bin%2Bspace%2Band%2Btime.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature11631%26rft_id%253Dinfo%253Apmid%252F23123857%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature11631&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23123857&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F12%2F13%2F2019.12.13.870410.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000311031600046&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-37-1" title="View reference  in text" id="ref-37">↵</a><div class="cit ref-cit ref-journal" id="cit-2019.12.13.870410v1.37" data-doi="10.1890/14-1826.1"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Johnston</span>  <span class="cit-name-given-names">A</span></span>, <span class="cit-auth"><span class="cit-name-surname">Fink</span>  <span class="cit-name-given-names">D</span></span>, <span class="cit-auth"><span class="cit-name-surname">Reynolds</span>  <span class="cit-name-given-names">MD</span></span>, <span class="cit-auth"><span class="cit-name-surname">Hochachka</span>  <span class="cit-name-given-names">WM</span></span>, <span class="cit-auth"><span class="cit-name-surname">Sullivan</span>  <span class="cit-name-given-names">BL</span></span>, <span class="cit-auth"><span class="cit-name-surname">Bruns</span>  <span class="cit-name-given-names">NE</span></span>, <span class="cit-auth"><span class="cit-name-surname">Hallstein</span>  <span class="cit-name-given-names">E</span></span>, <span class="cit-auth"><span class="cit-name-surname">Merrifield</span>  <span class="cit-name-given-names">MS</span></span>, <span class="cit-auth"><span class="cit-name-surname">Matsumoto</span>  <span class="cit-name-given-names">S</span></span>, <span class="cit-auth"><span class="cit-name-surname">Kelling</span>  <span class="cit-name-given-names">S.</span></span> <span class="cit-pub-date">2015</span>. <span class="cit-article-title">Abundance models improve spatial and temporal prioritization of conservation resources</span>. <abbr class="cit-jnl-abbrev">Ecol Appl</abbr>. <span class="cit-vol">25</span>(<span class="cit-issue">7</span>):<span class="cit-fpage">1749</span>–<span class="cit-lpage">1756</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1890/14-1826.1</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEcol%2BAppl%26rft.volume%253D25%26rft.spage%253D1749%26rft_id%253Dinfo%253Adoi%252F10.1890%252F14-1826.1%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1890/14-1826.1&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-38-1" title="View reference  in text" id="ref-38">↵</a><div class="cit ref-cit ref-journal" id="cit-2019.12.13.870410v1.38" data-doi="10.1029/2001GL013366"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Kaplan</span>  <span class="cit-name-given-names">JO.</span></span> <span class="cit-pub-date">2002</span>. <span class="cit-article-title">Wetlands at the Last Glacial Maximum: Distribution and methane emissions</span>. <abbr class="cit-jnl-abbrev">Geophys Res Lett</abbr>. <span class="cit-vol">29</span>(<span class="cit-issue">6</span>):<span class="cit-fpage">3-1</span>-<span class="cit-lpage">3–4</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1029/2001GL013366</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGeophys%2BRes%2BLett%26rft.volume%253D29%26rft.spage%253D3-1%26rft_id%253Dinfo%253Adoi%252F10.1029%252F2001GL013366%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1029/2001GL013366&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-39-1" title="View reference  in text" id="ref-39">↵</a><div class="cit ref-cit ref-journal" id="cit-2019.12.13.870410v1.39" data-doi="10.2307/3213548"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Kingman</span>  <span class="cit-name-given-names">JFC.</span></span> <span class="cit-pub-date">1982</span>. <span class="cit-article-title">On the Genealogy of Large Populations</span>. <abbr class="cit-jnl-abbrev">J Appl Probab</abbr>. <span class="cit-vol">19</span>(<span class="cit-issue">1982</span>):<span class="cit-fpage">27</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.2307/3213548</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ%2BAppl%2BProbab%26rft.volume%253D19%26rft.spage%253D27%26rft_id%253Dinfo%253Adoi%252F10.2307%252F3213548%26rft_id%253Dinfo%253Apmid%252F11338422%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2307/3213548&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=11338422&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F12%2F13%2F2019.12.13.870410.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-40-1" title="View reference  in text" id="ref-40">↵</a><div class="cit ref-cit ref-journal" id="cit-2019.12.13.870410v1.40" data-doi="10.1093/molbev/msw054"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Kumar</span>  <span class="cit-name-given-names">S</span></span>, <span class="cit-auth"><span class="cit-name-surname">Stecher</span>  <span class="cit-name-given-names">G</span></span>, <span class="cit-auth"><span class="cit-name-surname">Tamura</span>  <span class="cit-name-given-names">K.</span></span> <span class="cit-pub-date">2016</span>. <span class="cit-article-title">MEGA7: Molecular Evolutionary Genetics Analysis Version 7.0 for Bigger Datasets</span>. <abbr class="cit-jnl-abbrev">Mol Biol Evol</abbr>. <span class="cit-vol">33</span>(<span class="cit-issue">7</span>):<span class="cit-fpage">1870</span>–<span class="cit-lpage">1874</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1093/molbev/msw054</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol%2BBiol%2BEvol%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fmolbev%252Fmsw054%26rft_id%253Dinfo%253Apmid%252F27004904%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/molbev/msw054&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=27004904&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F12%2F13%2F2019.12.13.870410.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-41-1" title="View reference  in text" id="ref-41">↵</a><div class="cit ref-cit ref-journal" id="cit-2019.12.13.870410v1.41" data-doi="10.1111/j.1749-8198.2007.00132.x"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Lafleur</span>  <span class="cit-name-given-names">PM.</span></span> <span class="cit-pub-date">2008</span>. <span class="cit-article-title">Connecting Atmosphere and Wetland: Energy and Water Vapour Exchange</span>. <abbr class="cit-jnl-abbrev">Geogr Compass</abbr>. <span class="cit-vol">2</span>(<span class="cit-issue">4</span>):<span class="cit-fpage">1027</span>–<span class="cit-lpage">1057</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1111/j.1749-8198.2007.00132.x</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGeogr%2BCompass%26rft.volume%253D2%26rft.spage%253D1027%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1749-8198.2007.00132.x%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1749-8198.2007.00132.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-42-1" title="View reference  in text" id="ref-42">↵</a><div class="cit ref-cit ref-journal" id="cit-2019.12.13.870410v1.42" data-doi="10.1111/2041-210X.12410"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Leigh</span>  <span class="cit-name-given-names">JW</span></span>, <span class="cit-auth"><span class="cit-name-surname">Bryant</span>  <span class="cit-name-given-names">D.</span></span> <span class="cit-pub-date">2015</span>. <span class="cit-article-title">Popart: Full-Feature Software for Haplotype Network Construction</span>. <abbr class="cit-jnl-abbrev">Methods Ecol Evol</abbr>. <span class="cit-vol">6</span>(<span class="cit-issue">9</span>):<span class="cit-fpage">1110</span>–<span class="cit-lpage">1116</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1111/2041-210X.12410</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMethods%2BEcol%2BEvol%26rft.volume%253D6%26rft.spage%253D1110%26rft_id%253Dinfo%253Adoi%252F10.1111%252F2041-210X.12410%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/2041-210X.12410&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-43-1" title="View reference  in text" id="ref-43">↵</a><div class="cit ref-cit ref-journal" id="cit-2019.12.13.870410v1.43" data-doi="10.1038/nature10574"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Lorenzen</span>  <span class="cit-name-given-names">ED</span></span>, <span class="cit-auth"><span class="cit-name-surname">Nogués-Bravo</span>  <span class="cit-name-given-names">D</span></span>, <span class="cit-auth"><span class="cit-name-surname">Orlando</span>  <span class="cit-name-given-names">L</span></span>, <span class="cit-auth"><span class="cit-name-surname">Weinstock</span>  <span class="cit-name-given-names">J</span></span>, <span class="cit-auth"><span class="cit-name-surname">Binladen</span>  <span class="cit-name-given-names">J</span></span>, <span class="cit-auth"><span class="cit-name-surname">Marske</span>  <span class="cit-name-given-names">KA</span></span>, <span class="cit-auth"><span class="cit-name-surname">Ugan</span>  <span class="cit-name-given-names">A</span></span>, <span class="cit-auth"><span class="cit-name-surname">Borregaard</span>  <span class="cit-name-given-names">MK</span></span>, <span class="cit-auth"><span class="cit-name-surname">Gilbert</span>  <span class="cit-name-given-names">MTP</span></span>, <span class="cit-auth"><span class="cit-name-surname">Nielsen</span>  <span class="cit-name-given-names">R</span></span>, <span class="cit-etal">et al.</span> <span class="cit-pub-date">2012</span>. <span class="cit-article-title">Species-specific responses of LateQuaternary megafauna to climate and humans</span>. <abbr class="cit-jnl-abbrev">Nature</abbr>. <span class="cit-vol">479</span>(<span class="cit-issue">7373</span>):<span class="cit-fpage">359</span>–<span class="cit-lpage">364</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1038/nature10574</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.volume%253D479%26rft.spage%253D359%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature10574%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature10574&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-44-1" title="View reference  in text" id="ref-44">↵</a><div class="cit ref-cit ref-journal" id="cit-2019.12.13.870410v1.44" data-doi="10.1111/j.1365-294X.2012.05606.x"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Lu</span>  <span class="cit-name-given-names">B</span></span>, <span class="cit-auth"><span class="cit-name-surname">Zheng</span>  <span class="cit-name-given-names">Y</span></span>, <span class="cit-auth"><span class="cit-name-surname">Murphy</span>  <span class="cit-name-given-names">RW</span></span>, <span class="cit-auth"><span class="cit-name-surname">Zeng</span>  <span class="cit-name-given-names">X.</span></span> <span class="cit-pub-date">2012</span>. <span class="cit-article-title">Coalescence patterns of endemic Tibetan species of stream salamanders (Hynobiidae: Batrachuperus)</span>. <abbr class="cit-jnl-abbrev">Mol Ecol</abbr>. <span class="cit-vol">21</span>(<span class="cit-issue">13</span>):<span class="cit-fpage">3308</span>–<span class="cit-lpage">3324</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1111/j.1365-294X.2012.05606.x</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2BEcology%2B%2528Print%2529%26rft.stitle%253DMolecular%2BEcology%2B%2528Print%2529%26rft.aulast%253DLu%26rft.auinit1%253DB.%26rft.volume%253D21%26rft.issue%253D13%26rft.spage%253D3308%26rft.epage%253D3324%26rft.atitle%253DCoalescence%2Bpatterns%2Bof%2Bendemic%2BTibetan%2Bspecies%2Bof%2Bstream%2Bsalamanders%2B%2528Hynobiidae%253A%2BBatrachuperus%2529.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1365-294X.2012.05606.x%26rft_id%253Dinfo%253Apmid%252F22571598%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1365-294X.2012.05606.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22571598&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F12%2F13%2F2019.12.13.870410.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000305582200018&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-45-1" title="View reference  in text" id="ref-45">↵</a><div class="cit ref-cit ref-journal" id="cit-2019.12.13.870410v1.45" data-doi="10.1016/j.tpb.2015.06.003"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Mazet</span>  <span class="cit-name-given-names">O</span></span>, <span class="cit-auth"><span class="cit-name-surname">Rodríguez</span>  <span class="cit-name-given-names">W</span></span>, <span class="cit-auth"><span class="cit-name-surname">Chikhi</span>  <span class="cit-name-given-names">L.</span></span> <span class="cit-pub-date">2015</span>. <span class="cit-article-title">Demographic inference using genetic data from a single individual: Separating population size variation from population structure</span>. <abbr class="cit-jnl-abbrev">Theor Popul Biol</abbr>. <span class="cit-vol">104</span>:<span class="cit-fpage">46</span>–<span class="cit-lpage">58</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1016/j.tpb.2015.06.003</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTheor%2BPopul%2BBiol%26rft.volume%253D104%26rft.spage%253D46%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.tpb.2015.06.003%26rft_id%253Dinfo%253Apmid%252F26120083%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.tpb.2015.06.003&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=26120083&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F12%2F13%2F2019.12.13.870410.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-46-1" title="View reference  in text" id="ref-46">↵</a><div class="cit ref-cit ref-journal" id="cit-2019.12.13.870410v1.46" data-doi="10.5194/cp-5-33-2009"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Van Meerbeeck</span>  <span class="cit-name-given-names">CJ</span></span>, <span class="cit-auth"><span class="cit-name-surname">Renssen</span>  <span class="cit-name-given-names">H</span></span>, <span class="cit-auth"><span class="cit-name-surname">Roche</span>  <span class="cit-name-given-names">DM.</span></span> <span class="cit-pub-date">2009</span>. <span class="cit-article-title">How did Marine Isotope Stage 3 and Last Glacial Maximum climates differ? - Perspectives from equilibrium simulations</span>. <abbr class="cit-jnl-abbrev">Clim Past</abbr>. <span class="cit-vol">5</span>(<span class="cit-issue">1</span>):<span class="cit-fpage">33</span>–<span class="cit-lpage">51</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.5194/cp-5-33-2009</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DClim%2BPast%26rft.volume%253D5%26rft.spage%253D33%26rft_id%253Dinfo%253Adoi%252F10.5194%252Fcp-5-33-2009%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.5194/cp-5-33-2009&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-47-1" title="View reference  in text" id="ref-47">↵</a><div class="cit ref-cit ref-journal" id="cit-2019.12.13.870410v1.47" data-doi="10.1098/rsos.180543"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Miller</span>  <span class="cit-name-given-names">EF</span></span>, <span class="cit-auth"><span class="cit-name-surname">Manica</span>  <span class="cit-name-given-names">A</span></span>, <span class="cit-auth"><span class="cit-name-surname">Amos</span>  <span class="cit-name-given-names">W.</span></span> <span class="cit-pub-date">2018</span>. <span class="cit-article-title">Global demographic history of human populations inferred from whole mitochondrial genomes</span>. <abbr class="cit-jnl-abbrev">R Soc Open Sci</abbr>. <span class="cit-vol">5</span>(<span class="cit-issue">8</span>):<span class="cit-fpage">180543</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1098/rsos.180543</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DR%2BSoc%2BOpen%2BSci%26rft_id%253Dinfo%253Adoi%252F10.1098%252Frsos.180543%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1098/rsos.180543&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-48-1" title="View reference  in text" id="ref-48">↵</a><div class="cit ref-cit ref-journal" id="cit-2019.12.13.870410v1.48" data-doi="10.1111/mec.13780"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Nabholz</span>  <span class="cit-name-given-names">B</span></span>, <span class="cit-auth"><span class="cit-name-surname">Lanfear</span>  <span class="cit-name-given-names">R</span></span>, <span class="cit-auth"><span class="cit-name-surname">Fuchs</span>  <span class="cit-name-given-names">J.</span></span> <span class="cit-pub-date">2016</span>. <span class="cit-article-title">Body mass-corrected molecular rate for bird mitochondrial DNA</span>. <abbr class="cit-jnl-abbrev">Mol Ecol</abbr>. <span class="cit-vol">25</span>(<span class="cit-issue">18</span>):<span class="cit-fpage">4438</span>–<span class="cit-lpage">4449</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1111/mec.13780</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol%2BEcol%26rft.volume%253D25%26rft.spage%253D4438%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fmec.13780%26rft_id%253Dinfo%253Apmid%252F27483387%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/mec.13780&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=27483387&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F12%2F13%2F2019.12.13.870410.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-49-1" title="View reference  in text" id="ref-49">↵</a><div class="cit ref-cit ref-journal" id="cit-2019.12.13.870410v1.49" data-doi="10.1111/j.0014-3820.2003.tb00307.x"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Pannell</span>  <span class="cit-name-given-names">JR.</span></span> <span class="cit-pub-date">2003</span>. <span class="cit-article-title">Coalescence in a Metapopulation with Recurrent Local Extinction and Recolonzation</span>. <abbr class="cit-jnl-abbrev">Evolution (N Y)</abbr>. <span class="cit-vol">57</span>(<span class="cit-issue">5</span>):<span class="cit-fpage">949</span>–<span class="cit-lpage">961</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1111/j.0014-3820.2003.tb00307.x</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEvolution%2B%2528N%2BY%2529%26rft.volume%253D57%26rft.spage%253D949%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.0014-3820.2003.tb00307.x%26rft_id%253Dinfo%253Apmid%252F12836814%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.0014-3820.2003.tb00307.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=12836814&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F12%2F13%2F2019.12.13.870410.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000183500900002&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-50-1" title="View reference  in text" id="ref-50">↵</a><div class="cit ref-cit ref-journal" id="cit-2019.12.13.870410v1.50" data-doi="10.1038/436183a"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Penny</span>  <span class="cit-name-given-names">D.</span></span> <span class="cit-pub-date">2005</span>. <span class="cit-article-title">Relativity for molecular clocks</span>. <abbr class="cit-jnl-abbrev">Nature</abbr>. <span class="cit-vol">436</span>(July):<span class="cit-fpage">183</span>–<span class="cit-lpage">184</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1038/436183a</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DPenny%26rft.auinit1%253DD.%26rft.volume%253D436%26rft.issue%253D7048%26rft.spage%253D183%26rft.epage%253D184%26rft.atitle%253DEvolutionary%2Bbiology%253A%2Brelativity%2Bfor%2Bmolecular%2Bclocks.%26rft_id%253Dinfo%253Adoi%252F10.1038%252F436183a%26rft_id%253Dinfo%253Apmid%252F16015312%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/436183a&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16015312&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F12%2F13%2F2019.12.13.870410.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000230459500021&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-51-1" title="View reference  in text" id="ref-51">↵</a><div class="cit ref-cit ref-journal" id="cit-2019.12.13.870410v1.51" data-doi="10.1016/j.ecolmodel.2006.05.025"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Potts</span>  <span class="cit-name-given-names">JM</span></span>, <span class="cit-auth"><span class="cit-name-surname">Elith</span>  <span class="cit-name-given-names">J.</span></span> <span class="cit-pub-date">2006</span>. <span class="cit-article-title">Comparing species abundance models</span>. <abbr class="cit-jnl-abbrev">Ecol Modell</abbr>. <span class="cit-vol">199</span>(<span class="cit-issue">2</span>):<span class="cit-fpage">153</span>–<span class="cit-lpage">163</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1016/j.ecolmodel.2006.05.025</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEcol%2BModell%26rft.volume%253D199%26rft.spage%253D153%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.ecolmodel.2006.05.025%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.ecolmodel.2006.05.025&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000241994800004&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-52-1" title="View reference  in text" id="ref-52">↵</a><div class="cit ref-cit ref-journal" id="cit-2019.12.13.870410v1.52"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Pybus</span>  <span class="cit-name-given-names">OG</span></span>, <span class="cit-auth"><span class="cit-name-surname">Rambaut</span>  <span class="cit-name-given-names">A</span></span>, <span class="cit-auth"><span class="cit-name-surname">Harvey</span>  <span class="cit-name-given-names">PH.</span></span> <span class="cit-pub-date">2000</span>. <span class="cit-article-title">An integrated framework for the inference of viral population history from reconstructed genealogies</span>. <abbr class="cit-jnl-abbrev">Genetics</abbr>. <span class="cit-vol">155</span>(<span class="cit-issue">3</span>):<span class="cit-fpage">1429</span>–<span class="cit-lpage">37</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DPybus%26rft.auinit1%253DO.%2BG.%26rft.volume%253D155%26rft.issue%253D3%26rft.spage%253D1429%26rft.epage%253D1437%26rft.atitle%253DAn%2BIntegrated%2BFramework%2Bfor%2Bthe%2BInference%2Bof%2BViral%2BPopulation%2BHistory%2BFrom%2BReconstructed%2BGenealogies%26rft_id%253Dinfo%253Apmid%252F10880500%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE1NS8zLzE0MjkiO3M6NDoiYXRvbSI7czo0ODoiL2Jpb3J4aXYvZWFybHkvMjAxOS8xMi8xMy8yMDE5LjEyLjEzLjg3MDQxMC5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-53-1" title="View reference  in text" id="ref-53">↵</a><div class="cit ref-cit ref-journal" id="cit-2019.12.13.870410v1.53" data-doi="10.1111/ecog.02881"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Roberts</span>  <span class="cit-name-given-names">DR</span></span>, <span class="cit-auth"><span class="cit-name-surname">Bahn</span>  <span class="cit-name-given-names">V</span></span>, <span class="cit-auth"><span class="cit-name-surname">Ciuti</span>  <span class="cit-name-given-names">S</span></span>, <span class="cit-auth"><span class="cit-name-surname">Boyce</span>  <span class="cit-name-given-names">MS</span></span>, <span class="cit-auth"><span class="cit-name-surname">Elith</span>  <span class="cit-name-given-names">J</span></span>, <span class="cit-auth"><span class="cit-name-surname">Guillera-Arroita</span>  <span class="cit-name-given-names">G</span></span>, <span class="cit-auth"><span class="cit-name-surname">Hauenstein</span>  <span class="cit-name-given-names">S</span></span>, <span class="cit-auth"><span class="cit-name-surname">Lahoz-Monfort</span>  <span class="cit-name-given-names">JJ</span></span>, <span class="cit-auth"><span class="cit-name-surname">Schröder</span>  <span class="cit-name-given-names">B</span></span>, <span class="cit-auth"><span class="cit-name-surname">Thuiller</span>  <span class="cit-name-given-names">W</span></span>, <span class="cit-etal">et al.</span> <span class="cit-pub-date">2017</span>. <span class="cit-article-title">Cross-validation strategies for data with temporal, spatial, hierarchical, or phylogenetic structure</span>. <abbr class="cit-jnl-abbrev">Ecography (Cop)</abbr>. <span class="cit-vol">40</span>(<span class="cit-issue">8</span>):<span class="cit-fpage">913</span>–<span class="cit-lpage">929</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1111/ecog.02881</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEcography%2B%2528Cop%2529%26rft.volume%253D40%26rft.spage%253D913%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fecog.02881%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/ecog.02881&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-54-1" title="View reference  in text" id="ref-54">↵</a><div class="cit ref-cit ref-journal" id="cit-2019.12.13.870410v1.54" data-doi="10.1016/j.fishres.2015.11.005"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Sanchez</span>  <span class="cit-name-given-names">G</span></span>, <span class="cit-auth"><span class="cit-name-surname">Tomano</span>  <span class="cit-name-given-names">S</span></span>, <span class="cit-auth"><span class="cit-name-surname">Yamashiro</span>  <span class="cit-name-given-names">C</span></span>, <span class="cit-auth"><span class="cit-name-surname">Fujita</span>  <span class="cit-name-given-names">R</span></span>, <span class="cit-auth"><span class="cit-name-surname">Wakabayashi</span>  <span class="cit-name-given-names">T</span></span>, <span class="cit-auth"><span class="cit-name-surname">Sakai</span>  <span class="cit-name-given-names">M</span></span>, <span class="cit-auth"><span class="cit-name-surname">Umino</span>  <span class="cit-name-given-names">T.</span></span> <span class="cit-pub-date">2016</span>. <span class="cit-article-title">Population genetics of the jumbo squid Dosidicus gigas (Cephalopoda: Ommastrephidae) in the northern Humboldt Current system based on mitochondrial and microsatellite DNA markers</span>. <abbr class="cit-jnl-abbrev">Fish Res</abbr>. <span class="cit-vol">175</span>:<span class="cit-fpage">1</span>–<span class="cit-lpage">9</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1016/j.fishres.2015.11.005</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DFish%2BRes%26rft.volume%253D175%26rft.spage%253D1%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.fishres.2015.11.005%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.fishres.2015.11.005&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-55-1" title="View reference  in text" id="ref-55">↵</a><div class="cit ref-cit ref-journal" id="cit-2019.12.13.870410v1.55" data-doi="10.1038/s41437-017-0040-3"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Segawa</span>  <span class="cit-name-given-names">T</span></span>, <span class="cit-auth"><span class="cit-name-surname">Takeuchi</span>  <span class="cit-name-given-names">N</span></span>, <span class="cit-auth"><span class="cit-name-surname">Fujita</span>  <span class="cit-name-given-names">K</span></span>, <span class="cit-auth"><span class="cit-name-surname">Aizen</span>  <span class="cit-name-given-names">VB</span></span>, <span class="cit-auth"><span class="cit-name-surname">Willerslev</span>  <span class="cit-name-given-names">E</span></span>, <span class="cit-auth"><span class="cit-name-surname">Yonezawa</span>  <span class="cit-name-given-names">T.</span></span> <span class="cit-pub-date">2018</span>. <span class="cit-article-title">Demographic analysis of cyanobacteria based on the mutation rates estimated from an ancient ice core</span>. <abbr class="cit-jnl-abbrev">Heredity (Edinb)</abbr>.:<span class="cit-fpage">1</span>–<span class="cit-lpage">12</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1038/s41437-017-0040-3</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DHeredity%2B%2528Edinb%2529%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fs41437-017-0040-3%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/s41437-017-0040-3&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-56-1" title="View reference  in text" id="ref-56">↵</a><div class="cit ref-cit ref-journal" id="cit-2019.12.13.870410v1.56" data-doi="10.1093/molbev/msq083"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Stiller</span>  <span class="cit-name-given-names">M</span></span>, <span class="cit-auth"><span class="cit-name-surname">Baryshnikov</span>  <span class="cit-name-given-names">G</span></span>, <span class="cit-auth"><span class="cit-name-surname">Bocherens</span>  <span class="cit-name-given-names">H</span></span>, <span class="cit-auth"><span class="cit-name-surname">Grandal D’Anglade</span>  <span class="cit-name-given-names">A</span></span>, <span class="cit-auth"><span class="cit-name-surname">Hilpert</span>  <span class="cit-name-given-names">B</span></span>, <span class="cit-auth"><span class="cit-name-surname">Münzel</span>  <span class="cit-name-given-names">SC</span></span>, <span class="cit-auth"><span class="cit-name-surname">Pinhasi</span>  <span class="cit-name-given-names">R</span></span>, <span class="cit-auth"><span class="cit-name-surname">Rabeder</span>  <span class="cit-name-given-names">G</span></span>, <span class="cit-auth"><span class="cit-name-surname">Rosendahl</span>  <span class="cit-name-given-names">W</span></span>, <span class="cit-auth"><span class="cit-name-surname">Trinkaus</span>  <span class="cit-name-given-names">E</span></span>, <span class="cit-etal">et al.</span> <span class="cit-pub-date">2010</span>. <span class="cit-article-title">Withering away-25,000 years of genetic decline preceded cave bear extinction</span>. <abbr class="cit-jnl-abbrev">Mol Biol Evol</abbr>. <span class="cit-vol">27</span>(<span class="cit-issue">5</span>):<span class="cit-fpage">975</span>–<span class="cit-lpage">978</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1093/molbev/msq083</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol%2BBiol%2BEvol%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fmolbev%252Fmsq083%26rft_id%253Dinfo%253Apmid%252F20335279%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/molbev/msq083&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20335279&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F12%2F13%2F2019.12.13.870410.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000276994800001&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-57-1" title="View reference  in text" id="ref-57">↵</a><div class="cit ref-cit ref-journal" id="cit-2019.12.13.870410v1.57" data-doi="10.1016/S0304-3800(01)00388-X"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Stockwell</span>  <span class="cit-name-given-names">DR.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Peterson</span>  <span class="cit-name-given-names">AT.</span></span> <span class="cit-pub-date">2002</span>. <span class="cit-article-title">Effects of sample size on accuracy of species distribution models</span>. <abbr class="cit-jnl-abbrev">Ecol Modell</abbr>. <span class="cit-vol">148</span>(<span class="cit-issue">1</span>):<span class="cit-fpage">1</span>–<span class="cit-lpage">13</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1016/S0304-3800(01)00388-X</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEcol%2BModell%26rft.volume%253D148%26rft.spage%253D1%26rft_id%253Dinfo%253Adoi%252F10.1016%252FS0304-3800%252801%252900388-X%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/S0304-3800(01)00388-X&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-58-1" title="View reference  in text" id="ref-58">↵</a><div class="cit ref-cit ref-journal" id="cit-2019.12.13.870410v1.58" data-doi="10.1146/annurev-ecolsys-112414-054314"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Svenning</span>  <span class="cit-name-given-names">J-C</span></span>, <span class="cit-auth"><span class="cit-name-surname">Eiserhardt</span>  <span class="cit-name-given-names">WL</span></span>, <span class="cit-auth"><span class="cit-name-surname">Normand</span>  <span class="cit-name-given-names">S</span></span>, <span class="cit-auth"><span class="cit-name-surname">Ordonez</span>  <span class="cit-name-given-names">A</span></span>, <span class="cit-auth"><span class="cit-name-surname">Sandel</span>  <span class="cit-name-given-names">B.</span></span> <span class="cit-pub-date">2015</span>. <span class="cit-article-title">The Influence of Paleoclimate on Present-Day Patterns in Biodiversity and Ecosystems</span>. <abbr class="cit-jnl-abbrev">Annu Rev Ecol Evol Syst</abbr>. <span class="cit-vol">46</span>(<span class="cit-issue">1</span>):<span class="cit-fpage">551</span>–<span class="cit-lpage">572</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1146/annurev-ecolsys-112414-054314</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAnnu%2BRev%2BEcol%2BEvol%2BSyst%26rft.volume%253D46%26rft.spage%253D551%26rft_id%253Dinfo%253Adoi%252F10.1146%252Fannurev-ecolsys-112414-054314%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1146/annurev-ecolsys-112414-054314&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-59-1" title="View reference  in text" id="ref-59">↵</a><div class="cit ref-cit ref-journal" id="cit-2019.12.13.870410v1.59" data-doi="10.1093/nar/22.22.4673"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Thompson</span>  <span class="cit-name-given-names">JD</span></span>, <span class="cit-auth"><span class="cit-name-surname">Higgins</span>  <span class="cit-name-given-names">DG</span></span>, <span class="cit-auth"><span class="cit-name-surname">Gibson</span>  <span class="cit-name-given-names">TJ.</span></span> <span class="cit-pub-date">1994</span>. <span class="cit-article-title">ClustalW: improving the sensitivity of progressive multiple sequence aligment through sequence weighting, position specific gap penalties and weight matrix choice</span>. <abbr class="cit-jnl-abbrev">Nucleic Acids Res Acids Res</abbr>. <span class="cit-vol">22</span>(<span class="cit-issue">22</span>):<span class="cit-fpage">4673</span>–<span class="cit-lpage">4680</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1093/nar/22.22.4673</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes%2BAcids%2BRes%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252F22.22.4673%26rft_id%253Dinfo%253Apmid%252F7984417%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/nar/22.22.4673&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=7984417&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F12%2F13%2F2019.12.13.870410.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1994PU19900018&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-60-1" title="View reference  in text" id="ref-60">↵</a><div class="cit ref-cit ref-web" id="cit-2019.12.13.870410v1.60"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Thuiller</span>  <span class="cit-name-given-names">W</span></span>, <span class="cit-auth"><span class="cit-name-surname">Georges</span>  <span class="cit-name-given-names">D</span></span>, <span class="cit-auth"><span class="cit-name-surname">Engler</span>  <span class="cit-name-given-names">R</span></span>, <span class="cit-auth"><span class="cit-name-surname">Breiner</span>  <span class="cit-name-given-names">F.</span></span> <span class="cit-pub-date">2019</span>. <span class="cit-article-title">biomod2: Ensemble Platform for Species Distribution Modeling</span>. <a href="https://cran.r-project.org/package=biomod2">https://cran.r-project.org/package=biomod2</a>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-61-1" title="View reference  in text" id="ref-61">↵</a><div class="cit ref-cit ref-journal" id="cit-2019.12.13.870410v1.61" data-doi="10.1016/j.palaeo.2005.05.019"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Twitchett</span>  <span class="cit-name-given-names">RJ.</span></span> <span class="cit-pub-date">2006</span>. <span class="cit-article-title">The palaeoclimatology, palaeoecology and palaeoenvironmental analysis of mass extinction events</span>. <abbr class="cit-jnl-abbrev">Palaeogeogr Palaeoclimatol Palaeoecol</abbr>. <span class="cit-vol">232</span>(<span class="cit-issue">2–4</span>):<span class="cit-fpage">190</span>–<span class="cit-lpage">213</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1016/j.palaeo.2005.05.019</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPalaeogeography%252C%2BPalaeoclimatology%252C%2BPalaeoecology%26rft.stitle%253DPalaeogeography%252C%2BPalaeoclimatology%252C%2BPalaeoecology%26rft.volume%253D232%26rft.issue%253D2-4%26rft.spage%253D190%26rft.epage%253D213%26rft.atitle%253DThe%2Bpalaeoclimatology%252C%2Bpalaeoecology%2Band%2Bpalaeoenvironmental%2Banalysis%2Bof%2Bmass%2Bextinction%2Bevents%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.palaeo.2005.05.019%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.palaeo.2005.05.019&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=2006043707&amp;link_type=GEOREF" class="cit-ref-sprinkles cit-ref-sprinkles-georef"><span>GeoRef</span></a><a href="/lookup/external-ref?access_num=000236519100007&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-62-1" title="View reference  in text" id="ref-62">↵</a><div class="cit ref-cit ref-journal" id="cit-2019.12.13.870410v1.62" data-doi="10.1111/2041-210X.13107"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Valavi</span>  <span class="cit-name-given-names">R</span></span>, <span class="cit-auth"><span class="cit-name-surname">Elith</span>  <span class="cit-name-given-names">J</span></span>, <span class="cit-auth"><span class="cit-name-surname">Lahoz-Monfort</span>  <span class="cit-name-given-names">JJ</span></span>, <span class="cit-auth"><span class="cit-name-surname">Guillera-Arroita</span>  <span class="cit-name-given-names">G.</span></span> <span class="cit-pub-date">2019</span>. <span class="cit-article-title">blockCV: An r package for generating spatially or environmentally separated folds for k-fold cross-validation of species distribution models</span>. <abbr class="cit-jnl-abbrev">Methods Ecol Evol</abbr>. <span class="cit-vol">10</span>(<span class="cit-issue">2</span>):<span class="cit-fpage">225</span>–<span class="cit-lpage">232</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1111/2041-210X.13107</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMethods%2BEcol%2BEvol%26rft_id%253Dinfo%253Adoi%252F10.1111%252F2041-210X.13107%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/2041-210X.13107&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-63-1" title="View reference  in text" id="ref-63">↵</a><div class="cit ref-cit ref-journal" id="cit-2019.12.13.870410v1.63" data-doi="10.1029/2004GL021598"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Valdes</span>  <span class="cit-name-given-names">PJ</span></span>, <span class="cit-auth"><span class="cit-name-surname">Beerling</span>  <span class="cit-name-given-names">DJ</span></span>, <span class="cit-auth"><span class="cit-name-surname">Jonhson</span>  <span class="cit-name-given-names">CE.</span></span> <span class="cit-pub-date">2005</span>. <span class="cit-article-title">The ice age methane budget</span>. <abbr class="cit-jnl-abbrev">Geophys Res Lett</abbr>. <span class="cit-vol">32</span>(<span class="cit-issue">2</span>):<span class="cit-fpage">1</span>–<span class="cit-lpage">4</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1029/2004GL021004</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGeophys%2BRes%2BLett%26rft.volume%253D32%26rft.spage%253D1%26rft_id%253Dinfo%253Adoi%252F10.1029%252F2004GL021598%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1029/2004GL021598&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000186442700001&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-64-1" title="View reference  in text" id="ref-64">↵</a><div class="cit ref-cit ref-journal" id="cit-2019.12.13.870410v1.64" data-doi="10.1111/mec.12754"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Vignaud</span>  <span class="cit-name-given-names">TM</span></span>, <span class="cit-auth"><span class="cit-name-surname">Maynard</span>  <span class="cit-name-given-names">JA</span></span>, <span class="cit-auth"><span class="cit-name-surname">Leblois</span>  <span class="cit-name-given-names">R</span></span>, <span class="cit-auth"><span class="cit-name-surname">Meekan</span>  <span class="cit-name-given-names">MG</span></span>, <span class="cit-auth"><span class="cit-name-surname">Vázquez-Juárez</span>  <span class="cit-name-given-names">R</span></span>, <span class="cit-auth"><span class="cit-name-surname">Ramírez-Macías</span>  <span class="cit-name-given-names">D</span></span>, <span class="cit-auth"><span class="cit-name-surname">Pierce</span>  <span class="cit-name-given-names">SJ</span></span>, <span class="cit-auth"><span class="cit-name-surname">Rowat</span>  <span class="cit-name-given-names">D</span></span>, <span class="cit-auth"><span class="cit-name-surname">Berumen</span>  <span class="cit-name-given-names">ML</span></span>, <span class="cit-auth"><span class="cit-name-surname">Beeravolu</span>  <span class="cit-name-given-names">C</span></span>, <span class="cit-etal">et al.</span> <span class="cit-pub-date">2014</span>. <span class="cit-article-title">Genetic structure of populations of whale sharks among ocean basins and evidence for their historic rise and recent decline</span>. <abbr class="cit-jnl-abbrev">Mol Ecol</abbr>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1111/mec.12754</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol%2BEcol%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fmec.12754%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/mec.12754&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-65-1" title="View reference  in text" id="ref-65">↵</a><div class="cit ref-cit ref-journal" id="cit-2019.12.13.870410v1.65" data-doi="10.1016/j.ympev.2017.09.026"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Villalta</span>  <span class="cit-name-given-names">I</span></span>, <span class="cit-auth"><span class="cit-name-surname">Amor</span>  <span class="cit-name-given-names">F</span></span>, <span class="cit-auth"><span class="cit-name-surname">Galarza</span>  <span class="cit-name-given-names">JA</span></span>, <span class="cit-auth"><span class="cit-name-surname">Dupont</span>  <span class="cit-name-given-names">S</span></span>, <span class="cit-auth"><span class="cit-name-surname">Ortega</span>  <span class="cit-name-given-names">P</span></span>, <span class="cit-auth"><span class="cit-name-surname">Hefetz</span>  <span class="cit-name-given-names">A</span></span>, <span class="cit-auth"><span class="cit-name-surname">Dahbi</span>  <span class="cit-name-given-names">A</span></span>, <span class="cit-auth"><span class="cit-name-surname">Cerdá</span>  <span class="cit-name-given-names">X</span></span>, <span class="cit-auth"><span class="cit-name-surname">Boulay</span>  <span class="cit-name-given-names">R.</span></span> <span class="cit-pub-date">2018</span>. <span class="cit-article-title">Origin and distribution of desert ants across the Gibraltar Straits</span>. <abbr class="cit-jnl-abbrev">Mol Phylogenet Evol</abbr>. <span class="cit-vol">118</span>(September 2017):<span class="cit-fpage">122</span>–<span class="cit-lpage">134</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1016/j.ympev.2017.09.026</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol%2BPhylogenet%2BEvol%26rft.volume%253D118%26rft.spage%253D122%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.ympev.2017.09.026%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.ympev.2017.09.026&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-66-1" title="View reference  in text" id="ref-66">↵</a><div class="cit ref-cit ref-journal" id="cit-2019.12.13.870410v1.66"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Voous</span>  <span class="cit-name-given-names">KH.</span></span> <span class="cit-pub-date">1977</span>. <span class="cit-article-title">List of recent Holarctic bird species</span>.</cite></div><div class="cit-extra"></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.biorxiv.org/content/10.1101/2019.12.13.870410v1.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-disqus-comment" >
  
      
  
  <div class="pane-content">
    <div id="disqus_thread"><noscript><p><a href="http://biorxivstage.disqus.com/?url=https%3A%2F%2Fwww.biorxiv.org%2Fcontent%2F10.1101%2F2019.12.13.870410v1" class="" data-icon-position="" data-hide-link-title="0">View the discussion thread.</a></p></noscript></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/10.1101/2019.12.13.875096v1" title="A comprehensive study of Phospholipid fatty acid rearrangements in the early onset of the metabolic syndrome: correlations to organ dysfunction" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous</span></a></span><span class="pager-next"><a href="/content/10.1101/569319v4" title="Neonatal morphometric similarity mapping for predicting brain age and characterizing neuroanatomic variation associated with preterm birth" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    Posted&nbsp;December 13, 2019.  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-biorxiv-art-tools" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-biorxiv_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/10.1101/2019.12.13.870410v1.full.pdf" target="_self" class="article-dl-pdf-link link-icon"><i class="icon-external-link-sign"></i> <span class="title">Download PDF</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-biorxiv-supplementary-fragment" >
  
      
  
  <div class="pane-content">
    <p><a href="/content/10.1101/2019.12.13.870410v1.supplementary-material"><i class="icon-file"></i>Supplementary Material </a></p>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-biorxiv-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="Email this Article" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-biorxiv_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/10.1101/2019.12.13.870410v1.full" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for your interest in spreading the word about bioRxiv.</p><p>NOTE: Your email address is requested solely to identify you as the sender of this article.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/10.1101/2019.12.13.870410v1" class="active" data-icon-position="" data-hide-link-title="0">mtDNA-based reconstructions of change in effective population sizes of Holarctic birds do not agree with their reconstructed range sizes based on paleoclimates</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from bioRxiv
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would like to see this page from the bioRxiv website.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/1035842" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-fQzH07KCWmvI-TVf7ycEcZXJngtXIRvkVT7b6RCvvyg" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-biorxiv-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-biorxiv_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="1035842" id="node1035842--2" data-pisa="biorxiv;2019.12.13.870410v1" data-pisa-master="biorxiv;2019.12.13.870410" data-seqnum="1035842" data-apath="/biorxiv/early/2019/12/13/2019.12.13.870410.atom"><div  class="highwire-cite highwire-cite-highwire-article highwire-citation-biorxiv-article-pap-list clearfix" >

      <div class="highwire-cite-title" >
      <div class="highwire-cite-title">mtDNA-based reconstructions of change in effective population sizes of Holarctic birds do not agree with their reconstructed range sizes based on paleoclimates</div>    </div>
  
      <div  class="highwire-cite-authors" ><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0"><span class="nlm-given-names">Eleanor F.</span> <span class="nlm-surname">Miller</span></span>, <span class="highwire-citation-author" data-delta="1"><span class="nlm-given-names">Rhys E.</span> <span class="nlm-surname">Green</span></span>, <span class="highwire-citation-author" data-delta="2"><span class="nlm-given-names">Andrew</span> <span class="nlm-surname">Balmford</span></span>, <span class="highwire-citation-author" data-delta="3"><span class="nlm-given-names">Robert</span> <span class="nlm-surname">Beyer</span></span>, <span class="highwire-citation-author" data-delta="4"><span class="nlm-given-names">Marius</span> <span class="nlm-surname">Somveille</span></span>, <span class="highwire-citation-author" data-delta="5"><span class="nlm-given-names">Michela</span> <span class="nlm-surname">Leonardi</span></span>, <span class="highwire-citation-author" data-delta="6"><span class="nlm-given-names">William</span> <span class="nlm-surname">Amos</span></span>, <span class="highwire-citation-author" data-delta="7"><span class="nlm-given-names">Andrea</span> <span class="nlm-surname">Manica</span></span></span></div>
  
      <div  class="highwire-cite-metadata" ><span  class="highwire-cite-metadata-journal highwire-cite-metadata">bioRxiv </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">2019.12.13.870410; </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata"><span class="doi_label">doi:</span> https://doi.org/10.1101/2019.12.13.870410 </span></div>
  
  
  </div>
</div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.biorxiv.org/content/10.1101/2019.12.13.870410v1" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.biorxiv.org/content/10.1101/2019.12.13.870410v1&amp;title=mtDNA-based%20reconstructions%20of%20change%20in%20effective%20population%20sizes%20of%20Holarctic%20birds%20do%20not%20agree%20with%20their%20reconstructed%20range%20sizes%20based%20on%20paleoclimates" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.biorxiv.org/content/10.1101/2019.12.13.870410v1&amp;title=mtDNA-based%20reconstructions%20of%20change%20in%20effective%20population%20sizes%20of%20Holarctic%20birds%20do%20not%20agree%20with%20their%20reconstructed%20range%20sizes%20based%20on%20paleoclimates" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.biorxiv.org/content/10.1101/2019.12.13.870410v1&amp;text=mtDNA-based%20reconstructions%20of%20change%20in%20effective%20population%20sizes%20of%20Holarctic%20birds%20do%20not%20agree%20with%20their%20reconstructed%20range%20sizes%20based%20on%20paleoclimates" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.biorxiv.org/content/10.1101/2019.12.13.870410v1&amp;title=mtDNA-based%20reconstructions%20of%20change%20in%20effective%20population%20sizes%20of%20Holarctic%20birds%20do%20not%20agree%20with%20their%20reconstructed%20range%20sizes%20based%20on%20paleoclimates" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.biorxiv.org/content/10.1101/2019.12.13.870410v1&amp;t=mtDNA-based%20reconstructions%20of%20change%20in%20effective%20population%20sizes%20of%20Holarctic%20birds%20do%20not%20agree%20with%20their%20reconstructed%20range%20sizes%20based%20on%20paleoclimates" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.biorxiv.org/content/10.1101/2019.12.13.870410v1&amp;title=mtDNA-based%20reconstructions%20of%20change%20in%20effective%20population%20sizes%20of%20Holarctic%20birds%20do%20not%20agree%20with%20their%20reconstructed%20range%20sizes%20based%20on%20paleoclimates" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.biorxiv.org/content/10.1101/2019.12.13.870410v1&amp;title=mtDNA-based%20reconstructions%20of%20change%20in%20effective%20population%20sizes%20of%20Holarctic%20birds%20do%20not%20agree%20with%20their%20reconstructed%20range%20sizes%20based%20on%20paleoclimates" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-biorxiv-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title="Citation Tools"><i class="icon-globe"></i> <span class="title">Citation Tools</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-biorxiv_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="1035842" id="node1035842--3" data-pisa="biorxiv;2019.12.13.870410v1" data-pisa-master="biorxiv;2019.12.13.870410" data-seqnum="1035842" data-apath="/biorxiv/early/2019/12/13/2019.12.13.870410.atom"><div  class="highwire-cite highwire-cite-highwire-article highwire-citation-biorxiv-article-pap-list clearfix" >

      <div class="highwire-cite-title" >
      <div class="highwire-cite-title">mtDNA-based reconstructions of change in effective population sizes of Holarctic birds do not agree with their reconstructed range sizes based on paleoclimates</div>    </div>
  
      <div  class="highwire-cite-authors" ><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0"><span class="nlm-given-names">Eleanor F.</span> <span class="nlm-surname">Miller</span></span>, <span class="highwire-citation-author" data-delta="1"><span class="nlm-given-names">Rhys E.</span> <span class="nlm-surname">Green</span></span>, <span class="highwire-citation-author" data-delta="2"><span class="nlm-given-names">Andrew</span> <span class="nlm-surname">Balmford</span></span>, <span class="highwire-citation-author" data-delta="3"><span class="nlm-given-names">Robert</span> <span class="nlm-surname">Beyer</span></span>, <span class="highwire-citation-author" data-delta="4"><span class="nlm-given-names">Marius</span> <span class="nlm-surname">Somveille</span></span>, <span class="highwire-citation-author" data-delta="5"><span class="nlm-given-names">Michela</span> <span class="nlm-surname">Leonardi</span></span>, <span class="highwire-citation-author" data-delta="6"><span class="nlm-given-names">William</span> <span class="nlm-surname">Amos</span></span>, <span class="highwire-citation-author" data-delta="7"><span class="nlm-given-names">Andrea</span> <span class="nlm-surname">Manica</span></span></span></div>
  
      <div  class="highwire-cite-metadata" ><span  class="highwire-cite-metadata-journal highwire-cite-metadata">bioRxiv </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">2019.12.13.870410; </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata"><span class="doi_label">doi:</span> https://doi.org/10.1101/2019.12.13.870410 </span></div>
  
  
  </div>
</div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage&amp;rft.epage&amp;rft.atitle=mtDNA-based%20reconstructions%20of%20change%20in%20effective%20population%20sizes%20of%20Holarctic%20birds%20do%20not%20agree%20with%20their%20reconstructed%20range%20sizes%20based%20on%20paleoclimates&amp;rft.volume&amp;rft.issue&amp;rft.date=2019-01-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=bioRxiv&amp;rft.au=Miller%2C+Eleanor+F.&amp;rft.au=Green%2C+Rhys+E.&amp;rft.au=Balmford%2C+Andrew&amp;rft.au=Beyer%2C+Robert&amp;rft.au=Somveille%2C+Marius&amp;rft.au=Leonardi%2C+Michela&amp;rft.au=Amos%2C+William&amp;rft.au=Manica%2C+Andrea"></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/1035842/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/1035842/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/1035842/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/1035842/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/1035842/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/1035842/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/1035842/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/1035842/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/1035842/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/1035842/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/1035842/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/1035842/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><div class="item-list"><ul><li class="first"><a href="http://twitter.com/share?url=https%3A//www.biorxiv.org/content/10.1101/2019.12.13.870410v1&amp;count=horizontal&amp;via=&amp;text=mtDNA-based%20reconstructions%20of%20change%20in%20effective%20population%20sizes%20of%20Holarctic%20birds%20do%20not%20agree%20with%20their%20reconstructed%20range%20sizes%20based%20on%20paleoclimates&amp;counturl=https%3A//www.biorxiv.org/content/10.1101/2019.12.13.870410v1" class="twitter-share-button service-links-twitter-widget" id="twitter_widget" title="Tweet This" rel="nofollow" data-icon-position="" data-hide-link-title="0"><span class="element-invisible">Tweet Widget</span></a></li><li><a href="http://www.facebook.com/plugins/like.php?href=https%3A//www.biorxiv.org/content/10.1101/2019.12.13.870410v1&amp;layout=button_count&amp;show_faces=false&amp;action=like&amp;colorscheme=light&amp;width=100&amp;height=21&amp;font=&amp;locale=" id="facebook_like" title="I Like it" class="service-links-facebook-like" rel="nofollow" data-icon-position="" data-hide-link-title="0"><span class="element-invisible">Facebook Like</span></a></li><li class="last"><a href="https://www.biorxiv.org/content/10.1101/2019.12.13.870410v1" id="google_plus_one" title="Plus it" class="service-links-google-plus-one" rel="nofollow" data-icon-position="" data-hide-link-title="0"><span class="element-invisible">Google Plus One</span></a></li></ul></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-collections" >
  
        <h2 class="pane-title">Subject Area</h2>
    
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-collections"><div class="highwire-list"><ul class="highwire-article-collection-term-list"><li class="first last odd"><span class="highwire-article-collection-term"><a href="/collection/evolutionary-biology" class="highlight" data-icon-position="" data-hide-link-title="0">Evolutionary Biology<i class="icon-caret-right"></i>
</a></span></li></ul></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-biorxiv-subject-collections block-style-col2" >
  
      
  
  <div class="pane-content">
    <div class="panel-flexible panels-flexible-8 clearfix" id="mini-panel-biorxiv_subject_collections">
<div class="panel-flexible-inside panels-flexible-8-inside">
<div class="panels-flexible-region panels-flexible-region-8-center panels-flexible-region-first panels-flexible-region-last">
  <div class="inside panels-flexible-region-inside panels-flexible-region-8-center-inside panels-flexible-region-inside-first panels-flexible-region-inside-last">
<div class="panel-pane pane-snippet" >
  
      
  
  <div class="pane-content">
    <div class="snippet biorxiv-subject-areas-table-title" id="biorxiv-subject-areas-table-title">
  
      
  <div class="snippet-content">
    <b>Subject Areas</b>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-snippet" >
  
      
  
  <div class="pane-content">
    <div class="snippet biorxiv-subject-areas-view-papers" id="biorxiv-subject-areas-view-papers">
  
      
  <div class="snippet-content">
    <a href="/content/early/recent"><strong>All Articles</strong></a>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-subject-collections" >
  
      
  
  <div class="pane-content">
    <ul id="collection" class="collection highwire-list-expand"><li class="outer collection depth-2  child first"><div class = "data-wrapper"><a href="/collection/animal-behavior-and-cognition" class="" data-icon-position="" data-hide-link-title="0">Animal Behavior and Cognition</a> <span class = "article-count">(1662)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/biochemistry" class="" data-icon-position="" data-hide-link-title="0">Biochemistry</a> <span class = "article-count">(2789)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/bioengineering" class="" data-icon-position="" data-hide-link-title="0">Bioengineering</a> <span class = "article-count">(1946)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/bioinformatics" class="" data-icon-position="" data-hide-link-title="0">Bioinformatics</a> <span class = "article-count">(10400)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/biophysics" class="" data-icon-position="" data-hide-link-title="0">Biophysics</a> <span class = "article-count">(4243)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/cancer-biology" class="" data-icon-position="" data-hide-link-title="0">Cancer Biology</a> <span class = "article-count">(3272)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/cell-biology" class="" data-icon-position="" data-hide-link-title="0">Cell Biology</a> <span class = "article-count">(4641)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/clinical-trials" class="" data-icon-position="" data-hide-link-title="0">Clinical Trials</a> <span class = "article-count">(135)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/developmental-biology" class="" data-icon-position="" data-hide-link-title="0">Developmental Biology</a> <span class = "article-count">(2889)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/ecology" class="" data-icon-position="" data-hide-link-title="0">Ecology</a> <span class = "article-count">(4544)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/epidemiology" class="" data-icon-position="" data-hide-link-title="0">Epidemiology</a> <span class = "article-count">(2041)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/evolutionary-biology" class="" data-icon-position="" data-hide-link-title="0">Evolutionary Biology</a> <span class = "article-count">(7288)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/genetics" class="" data-icon-position="" data-hide-link-title="0">Genetics</a> <span class = "article-count">(5540)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/genomics" class="" data-icon-position="" data-hide-link-title="0">Genomics</a> <span class = "article-count">(6893)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/immunology" class="" data-icon-position="" data-hide-link-title="0">Immunology</a> <span class = "article-count">(2434)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/microbiology" class="" data-icon-position="" data-hide-link-title="0">Microbiology</a> <span class = "article-count">(7616)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/molecular-biology" class="" data-icon-position="" data-hide-link-title="0">Molecular Biology</a> <span class = "article-count">(3051)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/neuroscience" class="" data-icon-position="" data-hide-link-title="0">Neuroscience</a> <span class = "article-count">(18885)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/paleontology" class="" data-icon-position="" data-hide-link-title="0">Paleontology</a> <span class = "article-count">(142)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/pathology" class="" data-icon-position="" data-hide-link-title="0">Pathology</a> <span class = "article-count">(477)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/pharmacology-and-toxicology" class="" data-icon-position="" data-hide-link-title="0">Pharmacology and Toxicology</a> <span class = "article-count">(791)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/physiology" class="" data-icon-position="" data-hide-link-title="0">Physiology</a> <span class = "article-count">(1178)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/plant-biology" class="" data-icon-position="" data-hide-link-title="0">Plant Biology</a> <span class = "article-count">(2771)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/scientific-communication-and-education" class="" data-icon-position="" data-hide-link-title="0">Scientific Communication and Education</a> <span class = "article-count">(686)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/synthetic-biology" class="" data-icon-position="" data-hide-link-title="0">Synthetic Biology</a> <span class = "article-count">(905)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/systems-biology" class="" data-icon-position="" data-hide-link-title="0">Systems Biology</a> <span class = "article-count">(2881)</span></div></li>
<li class="outer collection depth-2  child last"><div class = "data-wrapper"><a href="/collection/zoology" class="" data-icon-position="" data-hide-link-title="0">Zoology</a> <span class = "article-count">(472)</span></div></li>
</ul>  </div>

  
  </div>
  </div>
</div>
</div>
</div>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__VNH5GD8Zz7g6_hGCOZjVjdMndxxC6naiExSSJKX3k_A__kzmEdtbqCsJx5Z9yg2Qy0PptB__ufXm-TxUOl0KZzUw__QJacVmveFyLIBSPkp21xL0ZvuDW9DMBrD2HB1P4Ry5I.js"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__QJacVmveFyLIBSPkp21xL0ZvuDW9DMBrD2HB1P4Ry5I.js" defer="defer"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__llgGUcz2-k3Jmyyyog-_4yI1wJfGWwx0eQ2lbnM9IIo__t6UjIjIKIAD6nKc6BAU9qmlVV2VR4CBTNQbxlbJPgN0__QJacVmveFyLIBSPkp21xL0ZvuDW9DMBrD2HB1P4Ry5I.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
function euCookieComplianceLoadScripts() {}
//--><!]]>
</script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var eu_cookie_compliance_cookie_name = "";
//--><!]]>
</script>
<script type="text/javascript" src="//d33xdlntwy0kbs.cloudfront.net/cshl_custom.js"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__BmqjBnkz3MgYCAoc25s1lDRMEjLhC3mEPVonUFIHi08__Unwv5-ZIuHBfFwytsjEx1niBVJ7n1T4lPws7VrkHXM4__QJacVmveFyLIBSPkp21xL0ZvuDW9DMBrD2HB1P4Ry5I.js"></script>
  </body>
</html>
