<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<link rel="dns-prefetch" href="//d33xdlntwy0kbs.cloudfront.net" />
<link rel="shortcut icon" href="https://www.biorxiv.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<meta name="type" content="article" />
<meta name="category" content="article" />
<meta name="HW.identifier" content="/biorxiv/early/2019/04/13/450692.atom" />
<meta name="HW.pisa" content="biorxiv;450692v2" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="The global diversity of the major parasitic nematode Haemonchus contortus is shaped by human intervention and climate" />
<meta name="DC.Identifier" content="10.1101/450692" />
<meta name="DC.Date" content="2019-04-13" />
<meta name="DC.Publisher" content="Cold Spring Harbor Laboratory" />
<meta name="DC.Rights" content="© 2019, Posted by Cold Spring Harbor Laboratory. This pre-print is available under a Creative Commons License (Attribution-NonCommercial-NoDerivs 4.0 International), CC BY-NC-ND 4.0, as described at http://creativecommons.org/licenses/by-nc-nd/4.0/" />
<meta name="DC.AccessRights" content="restricted" />
<meta name="DC.Description" content="The gastrointestinal parasite Haemonchus contortus is an haematophagous parasitic nematode of veterinary interest and a model for the study of drug resistance mechanisms or host-parasite interactions. To understand its evolutionary history, and its ability to adapt in the face of climatic and drug pressure, we have performed an extensive survey of genome-wide diversity using single-worm whole genome sequencing of 223 individuals sampled from 19 isolates spanning five continents. The pattern of global diversity is driven by an African origin for the species, together with contemporary dispersal that is consistent with modern human movement, with evidence for parasites spreading during the transatlantic slave trade and colonisation of Australia presented. Strong selective sweeps were identified in independent populations each surrounding the β-tubulin locus, a target of benzimidazole anthelmintic drug treatment used widely to control H. contortus infections. These signatures of selection were further supported by signals of diversifying selection enriched in genes involved in response to drugs, as well as other anthelmintic-associated biological functions including pharyngeal pumping and oviposition. From these analyses, we identify some known, and previously undescribed, candidate genes that may play a role in ivermectin resistance. Finally, we describe genetic signatures of climate-driven adaptation, revealing a gene acting as an epigenetic regulator and components of the dauer pathway may play a role in adaptation in the face of climatic fluctuations. These results begin to define genetic adaptation to climate for the first time in a parasitic nematode, and provides insight into the ongoing expansion in the range of Haemonchus contortus , which may have consequences for the management of this parasite." />
<meta name="DC.Contributor" content="G. Sallé" />
<meta name="DC.Contributor" content="S.R. Doyle" />
<meta name="DC.Contributor" content="J. Cortet" />
<meta name="DC.Contributor" content="J. Cabaret" />
<meta name="DC.Contributor" content="M. Berriman" />
<meta name="DC.Contributor" content="N. Holroyd" />
<meta name="DC.Contributor" content="J.A Cotton" />
<meta name="article:published_time" content="2019-04-13" />
<meta name="article:section" content="New Results" />
<meta name="citation_title" content="The global diversity of the major parasitic nematode Haemonchus contortus is shaped by human intervention and climate" />
<meta name="citation_abstract" lang="en" content="&lt;h3&gt;Abstract&lt;/h3&gt;
&lt;p&gt;The gastrointestinal parasite &lt;i&gt;Haemonchus contortus&lt;/i&gt; is an haematophagous parasitic nematode of veterinary interest and a model for the study of drug resistance mechanisms or host-parasite interactions. To understand its evolutionary history, and its ability to adapt in the face of climatic and drug pressure, we have performed an extensive survey of genome-wide diversity using single-worm whole genome sequencing of 223 individuals sampled from 19 isolates spanning five continents. The pattern of global diversity is driven by an African origin for the species, together with contemporary dispersal that is consistent with modern human movement, with evidence for parasites spreading during the transatlantic slave trade and colonisation of Australia presented. Strong selective sweeps were identified in independent populations each surrounding the β-tubulin locus, a target of benzimidazole anthelmintic drug treatment used widely to control &lt;i&gt;H. contortus&lt;/i&gt; infections. These signatures of selection were further supported by signals of diversifying selection enriched in genes involved in response to drugs, as well as other anthelmintic-associated biological functions including pharyngeal pumping and oviposition. From these analyses, we identify some known, and previously undescribed, candidate genes that may play a role in ivermectin resistance. Finally, we describe genetic signatures of climate-driven adaptation, revealing a gene acting as an epigenetic regulator and components of the &lt;i&gt;dauer&lt;/i&gt; pathway may play a role in adaptation in the face of climatic fluctuations. These results begin to define genetic adaptation to climate for the first time in a parasitic nematode, and provides insight into the ongoing expansion in the range of &lt;i&gt;Haemonchus contortus&lt;/i&gt;, which may have consequences for the management of this parasite.&lt;/p&gt;" />
<meta name="citation_journal_title" content="bioRxiv" />
<meta name="citation_publisher" content="Cold Spring Harbor Laboratory" />
<meta name="citation_publication_date" content="2019/01/01" />
<meta name="citation_mjid" content="biorxiv;450692v2" />
<meta name="citation_id" content="450692v2" />
<meta name="citation_public_url" content="https://www.biorxiv.org/content/10.1101/450692v2" />
<meta name="citation_abstract_html_url" content="https://www.biorxiv.org/content/10.1101/450692v2.abstract" />
<meta name="citation_full_html_url" content="https://www.biorxiv.org/content/10.1101/450692v2.full" />
<meta name="citation_pdf_url" content="https://www.biorxiv.org/content/biorxiv/early/2019/04/13/450692.full.pdf" />
<meta name="citation_doi" content="10.1101/450692" />
<meta name="citation_num_pages" content="62" />
<meta name="citation_article_type" content="Article" />
<meta name="citation_section" content="New Results" />
<meta name="citation_firstpage" content="450692" />
<meta name="citation_author" content="G. Sallé" />
<meta name="citation_author_institution" content="Wellcome Sanger Institute" />
<meta name="citation_author_institution" content="INRA – U. Tours" />
<meta name="citation_author_email" content="guillaume.salle@inra.fr" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0002-4032-139X" />
<meta name="citation_author" content="S.R. Doyle" />
<meta name="citation_author_institution" content="Wellcome Sanger Institute" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0001-9167-7532" />
<meta name="citation_author" content="J. Cortet" />
<meta name="citation_author_institution" content="INRA – U. Tours" />
<meta name="citation_author" content="J. Cabaret" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0002-7379-9023" />
<meta name="citation_author" content="M. Berriman" />
<meta name="citation_author_institution" content="Wellcome Sanger Institute" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0002-9581-0377" />
<meta name="citation_author" content="N. Holroyd" />
<meta name="citation_author_institution" content="Wellcome Sanger Institute" />
<meta name="citation_author" content="J.A Cotton" />
<meta name="citation_author_institution" content="Wellcome Sanger Institute" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0001-5475-3583" />
<meta name="citation_reference" content="Blaxter, M. &amp; Koutsovoulos, G. The evolution of parasitism in Nematoda. Parasitology 142 Suppl 1, S26–39 (2015)." />
<meta name="citation_reference" content="O'Connor, L.J., Walkden-Brown, S.W. &amp; Kahn, L.P. Ecology of the free-living stages of major trichostrongylid parasites of sheep. Vet Parasitol 142, 1–15 (2006)." />
<meta name="citation_reference" content="Jones, J.T. et al. Top 10 plant-parasitic nematodes in molecular plant pathology. Mol Plant Pathol 14, 946–61 (2013)." />
<meta name="citation_reference" content="Anderson, R.C. The origins of zooparasitic nematodes. Can J Zool 62, 317–328 (1984)." />
<meta name="citation_reference" content="Kaplan, R.M. &amp; Vidyashankar, A.N. An inconvenient truth: Global worming and anthelmintic resistance. Veterinary Parasitology 186, 70–78 (2012)." />
<meta name="citation_reference" content="McKellar, Q.A. &amp; Jackson, F. Veterinary anthelmintics: old and new. Trends Parasitol 20, 456–61 (2004)." />
<meta name="citation_reference" content="Bundy, D.A.P. et al. 100 Years of Mass Deworming Programmes: A Policy Perspective From the World Bank's Disease Control Priorities Analyses. Adv Parasitol 100, 127–154 (2018)." />
<meta name="citation_reference" content="Schulz, J.D., Moser, W., Hurlimann, E. &amp; Keiser, J. Preventive Chemotherapy in the Fight against Soil-Transmitted Helminthiasis: Achievements and Limitations. Trends Parasitol (2018)." />
<meta name="citation_reference" content="Hewitson, J.P. &amp; Maizels, R.M. Vaccination against helminth parasite infections. Expert Rev Vaccines 13, 473–87 (2014)." />
<meta name="citation_reference" content="Sallé, G. et al. Transcriptomic profiling of nematode parasites surviving vaccine exposure. Int J Parasitol (2018)." />
<meta name="citation_reference" content="Gilleard, J.S. Haemonchus contortus as a paradigm and model to study anthelmintic drug resistance. Parasitology 140, 1506–22 (2013)." />
<meta name="citation_reference" content="Doyle, S.R. et al. A Genome Resequencing-Based Genetic Map Reveals the Recombination Landscape of an Outbred Parasitic Nematode in the Presence of Polyploidy and Polyandry. Genome Biol Evol 10, 396–409 (2018)." />
<meta name="citation_reference" content="Laing, R. et al. The genome and transcriptome of Haemonchus contortus, a key model parasite for drug and vaccine discovery. Genome Biol 14, R88 (2013)." />
<meta name="citation_reference" content="International Helminth Genomes, C. Comparative genomics of the major parasitic worms. Nat Genet 51, 163–174 (2019)." />
<meta name="citation_reference" content="Mougin, C. et al. BRC4Env, a network of Biological Resource Centres for research in environmental and agricultural sciences. Environ Sci Pollut Res Int (2018)." />
<meta name="citation_reference" content="Langley, C.H. et al. Genomic variation in natural populations of Drosophila melanogaster. Genetics 192, 533–98 (2012)." />
<meta name="citation_reference" content="Small, S.T. et al. Population genomics of the filarial nematode parasite Wuchereria bancrofti from mosquitoes. Mol Ecol 25, 1465–77 (2016)." />
<meta name="citation_reference" content="Choi, Y.J. et al. Genomic diversity in Onchocerca volvulus and its Wolbachia endosymbiont. Nat Microbiol 2, 16207 (2016)." />
<meta name="citation_reference" content="Perry, G.H. et al. Comparative RNA sequencing reveals substantial genetic variation in endangered primates. Genome Res 22, 602–10 (2012)." />
<meta name="citation_reference" content="Denver, D.R. et al. A genome-wide view of Caenorhabditis elegans base-substitution mutation processes. Proc Natl Acad Sci U S A 106, 16310–4 (2009)." />
<meta name="citation_reference" content="Saccareau, M. et al. Meta-analysis of the parasitic phase traits of Haemonchus contortus infection in sheep. Parasit Vectors 10, 201 (2017)." />
<meta name="citation_reference" content="Geggus, D. The French Slave Trade: An Overview. The William and Mary Quaterly 58, 119–138 (2001)." />
<meta name="citation_reference" content="Anonymous. The wool industry – looking back and forward. (Australian Bureau of Statistics, 2003)." />
<meta name="citation_reference" content="Evesson, B. &amp; Moor, R. The foundation of Australia’s fine wool industry. in Journal of the Royal Australian Historical Society (2000)." />
<meta name="citation_reference" content="Eng, J.K. et al. Ivermectin selection on beta-tubulin: evidence in Onchocerca volvulus and Haemonchus contortus. Mol Biochem Parasitol 150, 229–35 (2006)." />
<meta name="citation_reference" content="de Lourdes Mottier, M. &amp; Prichard, R.K. Genetic analysis of a relationship between macrocyclic lactone and benzimidazole anthelmintic selection on Haemonchus contortus. Pharmacogenet Genomics 18, 129–40 (2008)." />
<meta name="citation_reference" content="Chen, H., Patterson, N. &amp; Reich, D. Population differentiation as a test for selective sweeps. Genome Res 20, 393–402 (2010)." />
<meta name="citation_reference" content="Ardelli, B.F., Stitt, L.E., Tompkins, J.B. &amp; Prichard, R.K. A comparison of the effects of ivermectin and moxidectin on the nematode Caenorhabditis elegans. Vet Parasitol 165, 96–108 (2009)." />
<meta name="citation_reference" content="Janssen, I.J., Krucken, J., Demeler, J. &amp; von Samson-Himmelstjerna, G. Transgenically expressed Parascaris P-glycoprotein-11 can modulate ivermectin susceptibility in Caenorhabditis elegans. Int J Parasitol Drugs Drug Resist 5, 44–7 (2015)." />
<meta name="citation_reference" content="Avery, L. &amp; Horvitz, H.R. Effects of starvation and neuroactive drugs on feeding in Caenorhabditis elegans. J Exp Zool 253, 263–70 (1990)." />
<meta name="citation_reference" content="Scott, E.W., Baxter, P. &amp; Armour, J. Fecundity of anthelmintic resistant adult Haemonchus contortus after exposure to ivermectin or benzimidazoles in vivo. Res Vet Sci 50, 247–9 (1991)." />
<meta name="citation_reference" content="Kotze, A.C. &amp; Prichard, R.K. Anthelmintic Resistance in Haemonchus contortus: History, Mechanisms and Diagnosis. Adv Parasitol 93, 397–428 (2016)." />
<meta name="citation_reference" content="Martin, R.J. et al. Levamisole receptors: a second awakening. Trends Parasitol 28, 289–96 (2012)." />
<meta name="citation_reference" content="Wada, M. et al. Isolation and characterization of a GDP/GTP exchange protein specific for the Rab3 subfamily small G proteins. J Biol Chem 272, 3875–8 (1997)." />
<meta name="citation_reference" content="Cao, R. &amp; Zhang, Y. The functions of E(Z)/EZH2-mediated methylation of lysine 27 in histone H3. Curr Opin Genet Dev 14, 155–64 (2004)." />
<meta name="citation_reference" content="Bolajoko, M.B. et al. The basic reproduction quotient (Q0) as a potential spatial predictor of the seasonality of ovine haemonchosis. Geospat Health 9, 333–50 (2015)." />
<meta name="citation_reference" content="Fitzpatrick, M.C. &amp; Keller, S.R. Ecological genomics meets community-level modelling of biodiversity: mapping the genomic landscape of current and future environmental adaptation. Ecol Lett 18, 1–16 (2015)." />
<meta name="citation_reference" content="Doyle, S.R. et al. Population genomic and evolutionary modelling analyses reveal a single major QTL for ivermectin drug resistance in the pathogenic nematode, Haemonchus contortus. BMC Genomics 20, 218 (2019)." />
<meta name="citation_reference" content="Blouin, M.S., Yowell, C.A., Courtney, C.H. &amp; Dame, J.B. Host movement and the genetic structure of populations of parasitic nematodes. Genetics 141, 1007–14 (1995)." />
<meta name="citation_reference" content="Troell, K., Engstrom, A., Morrison, D.A., Mattsson, J.G. &amp; Hoglund, J. Global patterns reveal strong population structure in Haemonchus contortus, a nematode parasite of domesticated ruminants. Int J Parasitol 36, 1305–16 (2006)." />
<meta name="citation_reference" content="Dey, A., Chan, C.K., Thomas, C.G. &amp; Cutter, A.D. Molecular hyperdiversity defines populations of the nematode Caenorhabditis brenneri. Proc Natl Acad Sci U S A 110, 11056–60 (2013)." />
<meta name="citation_reference" content="Gilleard, J.S. &amp; Redman, E. Genetic Diversity and Population Structure of Haemonchus contortus. Adv Parasitol 93, 31–68 (2016)." />
<meta name="citation_reference" content="Larson, G. et al. Current perspectives and the future of domestication studies. Proc Natl Acad Sci U S A 111, 6139–46 (2014)." />
<meta name="citation_reference" content="Pleurdeau, D. et al. &quot;Of sheep and men&quot;: earliest direct evidence of caprine domestication in southern Africa at Leopard Cave (Erongo, Namibia). PLoS One 7, e40340 (2012)." />
<meta name="citation_reference" content="Campbell, M.C. &amp; Tishkoff, S.A. The evolution of human genetic and phenotypic variation in Africa. Curr Biol 20, R166–73 (2010)." />
<meta name="citation_reference" content="Chessa, B. et al. Revealing the history of sheep domestication using retrovirus integrations. Science 324, 532–6 (2009)." />
<meta name="citation_reference" content="Tapio, M. et al. Sheep mitochondrial DNA variation in European, Caucasian, and Central Asian areas. Mol Biol Evol 23, 1776–83 (2006)." />
<meta name="citation_reference" content="Kijas, J.W. et al. Genome-wide analysis of the world's sheep breeds reveals high levels of historic mixture and strong recent selection. PLoS Biol 10, e1001258 (2012)." />
<meta name="citation_reference" content="Spangler, G.L. et al. Whole genome structural analysis of Caribbean hair sheep reveals quantitative link to West African ancestry. PLoS One 12, e0179021 (2017)." />
<meta name="citation_reference" content="Naves, M., Alexandre, G., Leimbacher, F., Mandonnet, N. &amp; Menendez-buxadera, A. Les ruminants domestiques de la Caraïbe: le point sur les ressources génétiques et leur exploitation. INRA Prod. Anim. 14, 181–192 (2001)." />
<meta name="citation_reference" content="Roman, A. Saint Malo au temps des négriers, 357 (Karthala, 2003)." />
<meta name="citation_reference" content="Pereira, F. et al. Genetic signatures of a Mediterranean influence in Iberian Peninsula sheep husbandry. Mol Biol Evol 23, 1420–6 (2006)." />
<meta name="citation_reference" content="Redman, E. et al. The emergence of resistance to the benzimidazole anthlemintics in parasitic nematodes of livestock is characterised by multiple independent hard and soft selective sweeps. PLoS Negl Trop Dis 9, e0003494 (2015)." />
<meta name="citation_reference" content="Vercruysse, J. et al. Is anthelmintic resistance a concern for the control of human soil-transmitted helminths? Int J Parasitol Drugs Drug Resist 1, 14–27 (2011)." />
<meta name="citation_reference" content="Laing, R., Gillan, V. &amp; Devaney, E. Ivermectin – Old Drug, New Tricks? Trends Parasitol 33, 463–472 (2017)." />
<meta name="citation_reference" content="Osei-Atweneboana, M.Y. et al. Phenotypic evidence of emerging ivermectin resistance in Onchocerca volvulus. PLoS Negl Trop Dis 5, e998 (2011)." />
<meta name="citation_reference" content="Doyle, S.R. et al. Genome-wide analysis of ivermectin response by Onchocerca volvulus reveals that genetic drift and soft selective sweeps contribute to loss of drug sensitivity. PLoS Negl Trop Dis 11, e0005816 (2017)." />
<meta name="citation_reference" content="Rose, H. et al. Climate-driven changes to the spatio-temporal distribution of the parasitic nematode, Haemonchus contortus, in sheep in Europe. Glob Chang Biol 22, 1271–85 (2016)." />
<meta name="citation_reference" content="van Dijk, J., David, G.P., Baird, G. &amp; Morgan, E.R. Back to the future: developing hypotheses on the effects of climate change on ovine parasitic gastroenteritis from historical data. Vet Parasitol 158, 73–84 (2008)." />
<meta name="citation_reference" content="Rajpurohit, S., Oliveira, C.C., Etges, W.J. &amp; Gibbs, A.G. Functional genomic and phenotypic responses to desiccation in natural populations of a desert drosophilid. Mol Ecol 22, 2698–715 (2013)." />
<meta name="citation_reference" content="Sharma, V., Kohli, S. &amp; Brahmachari, V. Correlation between desiccation stress response and epigenetic modifications of genes in Drosophila melanogaster: An example of environment-epigenome interaction. Biochim Biophys Acta 1860, 1058–1068 (2017)." />
<meta name="citation_reference" content="Coustham, V. et al. Quantitative modulation of polycomb silencing underlies natural variation in vernalization. Science 337, 584–7 (2012)." />
<meta name="citation_reference" content="Cassada, R.C. &amp; Russell, R.L. The dauerlarva, a post-embryonic developmental variant of the nematode Caenorhabditis elegans. Dev Biol 46, 326–42 (1975)." />
<meta name="citation_reference" content="Blitz, N.M. &amp; Gibbs, H.C. Studies on the arrested development of Haemonchus contortus in sheep. I. The induction of arrested development. Int J Parasitol 2, 5–12 (1972)." />
<meta name="citation_reference" content="Wood, D.E. &amp; Salzberg, S.L. Kraken: ultrafast metagenomic sequence classification using exact alignments. Genome Biol 15, R46 (2014)." />
<meta name="citation_reference" content="Li, B.W. et al. Transcriptomes and pathways associated with infectivity, survival and immunogenicity in Brugia malayi L3. BMC Genomics 10, 267 (2009)." />
<meta name="citation_reference" content="DePristo, M.A. et al. A framework for variation discovery and genotyping using next-generation DNA sequencing data. Nat Genet 43, 491–8 (2011)." />
<meta name="citation_reference" content="Korneliussen, T.S., Albrechtsen, A. &amp; Nielsen, R. ANGSD: Analysis of Next Generation Sequencing Data. BMC Bioinformatics 15, 356 (2014)." />
<meta name="citation_reference" content="McKenna, A. et al. The Genome Analysis Toolkit: a MapReduce framework for analyzing next-generation DNA sequencing data. Genome Res 20, 1297–303 (2010)." />
<meta name="citation_reference" content="Li, H. et al. The Sequence Alignment/Map format and SAMtools. Bioinformatics 25, 2078–9 (2009)." />
<meta name="citation_reference" content="Rensch, T., Villar, D., Horvath, J., Odom, D.T. &amp; Flicek, P. Mitochondrial heteroplasmy in vertebrates using ChIP-sequencing data. Genome Biol 17, 139 (2016)." />
<meta name="citation_reference" content="Howe, D.K., Baer, C.F. &amp; Denver, D.R. High rate of large deletions in Caenorhabditis briggsae mitochondrial genome mutation processes. Genome Biol Evol 2, 29–38 (2009)." />
<meta name="citation_reference" content="Konrad, A. et al. Mitochondrial Mutation Rate, Spectrum and Heteroplasmy in Caenorhabditis elegans Spontaneous Mutation Accumulation Lines of Differing Population Size. Mol Biol Evol 34, 1319–1334 (2017)." />
<meta name="citation_reference" content="Hazkani-Covo, E., Zeller, R.M. &amp; Martin, W. Molecular poltergeists: mitochondrial DNA copies (numts) in sequenced nuclear genomes. PLoS Genet 6, e1000834 (2010)." />
<meta name="citation_reference" content="Danecek, P. et al. The variant call format and VCFtools. Bioinformatics 27, 2156–8 (2011)." />
<meta name="citation_reference" content="Zheng, X. et al. A high-performance computing toolset for relatedness and principal component analysis of SNP data. Bioinformatics 28, 3326–8 (2012)." />
<meta name="citation_reference" content="R Core Team. R: A Language and Environment for Statistical Computing. (R Foundation for Statistical Computing, Vienna, 2016)." />
<meta name="citation_reference" content="Weir, B.S. &amp; Cockerham, C.C. Estimating F-Statistics for the Analysis of Population Structure. Evolution 38, 1358–1370 (1984)." />
<meta name="citation_reference" content="Svardal, H. et al. Ancient hybridization and strong adaptation to viruses across African vervet monkey populations. Nat Genet 49, 1705–1713 (2017)." />
<meta name="citation_reference" content="Chang, C.C. et al. Second-generation PLINK: rising to the challenge of larger and richer datasets. Gigascience 4, 7 (2015)." />
<meta name="citation_reference" content="Paradis, E., Claude, J. &amp; Strimmer, K. APE: Analyses of Phylogenetics and Evolution in R language. Bioinformatics 20, 289–90 (2004)." />
<meta name="citation_reference" content="Edgar, R.C. MUSCLE: multiple sequence alignment with high accuracy and high throughput. Nucleic Acids Res 32, 1792–7 (2004)." />
<meta name="citation_reference" content="Talavera, G. &amp; Castresana, J. Improvement of phylogenies after removing divergent and ambiguously aligned blocks from protein sequence alignments. Syst Biol 56, 564–77 (2007)." />
<meta name="citation_reference" content="Keane, T.M., Creevey, C.J., Pentony, M.M., Naughton, T.J. &amp; McLnerney, J.O. Assessment of methods for amino acid matrix selection and their use on empirical data shows that ad hoc assumptions for choice of matrix are not justified. BMC Evol Biol 6, 29 (2006)." />
<meta name="citation_reference" content="Guindon, S. &amp; Gascuel, O. A simple, fast, and accurate algorithm to estimate large phylogenies by maximum likelihood. Syst Biol 52, 696–704 (2003)." />
<meta name="citation_reference" content="Skotte, L., Korneliussen, T.S. &amp; Albrechtsen, A. Estimating individual admixture proportions from next generation sequencing data. Genetics 195, 693–702 (2013)." />
<meta name="citation_reference" content="Schiffels, S. &amp; Durbin, R. Inferring human population size and separation history from multiple genome sequences. Nat Genet 46, 919–25 (2014)." />
<meta name="citation_reference" content="Malaspinas, A.S. et al. A genomic history of Aboriginal Australia. Nature 538, 207–214 (2016)." />
<meta name="citation_reference" content="McVean, G.A. &amp; Cardin, N.J. Approximating the coalescent with recombination. Philos Trans R Soc Lond B Biol Sci 360, 1387–93 (2005)." />
<meta name="citation_reference" content="Kingman, J.F.C. The coalescent. Stochastic Processes and their Applications 13, 235–248 (1982)." />
<meta name="citation_reference" content="Emery, D.L., Hunt, P.W. &amp; Le Jambre, L.F. Haemonchus contortus: the then and now, and where to from here? Int J Parasitol 46, 755–769 (2016)." />
<meta name="citation_reference" content="Gutenkunst, R.N., Hernandez, R.D., Williamson, S.H. &amp; Bustamante, C.D. Inferring the joint demographic history of multiple populations from multidimensional SNP frequency data. PLoS Genet 5, e1000695 (2009)." />
<meta name="citation_reference" content="Portik, D.M. et al. Evaluating mechanisms of diversification in a Guineo-Congolian tropical forest frog using demographic model selection. Mol Ecol 26, 5245–5263 (2017)." />
<meta name="citation_reference" content="Coffman, A.J., Hsieh, P.H., Gravel, S. &amp; Gutenkunst, R.N. Computationally Efficient Composite Likelihood Statistics for Demographic Inference. Mol Biol Evol 33, 591–3 (2016)." />
<meta name="citation_reference" content="Hudson, R.R. Generating samples under a Wright-Fisher neutral model of genetic variation. Bioinformatics 18, 337–8 (2002)." />
<meta name="citation_reference" content="Suchard, M.A. et al. Bayesian phylogenetic and phylodynamic data integration using BEAST 1.10. Virus Evol 4, vey016 (2018)." />
<meta name="citation_reference" content="Drummond, A.J., Rambaut, A., Shapiro, B. &amp; Pybus, O.G. Bayesian coalescent inference of past population dynamics from molecular sequences. Mol Biol Evol 22, 1185–92 (2005)." />
<meta name="citation_reference" content="Howe, K.L., Bolt, B.J., Shafie, M., Kersey, P. &amp; Berriman, M. WormBase ParaSite – a comprehensive resource for helminth genomics. Mol Biochem Parasitol 215, 2–10 (2017)." />
<meta name="citation_reference" content="Rufener, L., Kaminsky, R. &amp; Maser, P. In vitro selection of Haemonchus contortus for benzimidazole resistance reveals a mutation at amino acid 198 of beta-tubulin. Mol Biochem Parasitol 168, 120–2 (2009)." />
<meta name="citation_reference" content="Korneliussen, T.S., Moltke, I., Albrechtsen, A. &amp; Nielsen, R. Calculation of Tajima's D and other neutrality test statistics from low depth next-generation sequencing data. BMC Bioinformatics 14, 289 (2013)." />
<meta name="citation_reference" content="Hudson, P.J. et al. Trophic interactions and population growth rates: describing patterns and identifying mechanisms. Philos Trans R Soc Lond B Biol Sci 357, 1259–71 (2002)." />
<meta name="citation_reference" content="Martin, S.H. &amp; Van Belleghem, S.M. Exploring Evolutionary Relationships Across the Genome Using Topology Weighting. Genetics 206, 429–438 (2017)." />
<meta name="citation_reference" content="Köppen, W. Die Wärmezonen der Erde, nach der Dauer der heissen, gemässigten und kalten Zeit und nach der Wirkung der Wärme auf die organische Welt betrachtet Meteorol. Z. 1, 215–226 (1884)." />
<meta name="citation_reference" content="Chen, D. &amp; Chen, H.W. Using the Köppen classification to quantify climate variation and change: An example for 1901–2010. Environmental Development 6, 69–79 (2013)." />
<meta name="citation_reference" content="Ellis, N., Smith, S.J. &amp; Pitcher, C.R. Gradient forests: calculating importance gradients on physical predictors. Ecology 93, 156–68 (2012)." />
<meta name="citation_reference" content="Fick, S. E. &amp; Hijmans, R.J. WorldClim 2: new 1-km spatial resolution climate surfaces for global land areas. International Journal of Climatology 37, 4302–4315 (2017)." />
<meta name="citation_reference" content="Frichot, E., Schoville, S.D., Bouchard, G. &amp; Francois, O. Testing for associations between loci and environmental gradients using latent factor mixed models. Mol Biol Evol 30, 1687–99 (2013)." />
<meta name="citation_reference" content="Alexa, A., Rahnenfuhrer, J. topGO: Enrichment Analysis for Gene Ontology. (2016)." />
<meta name="twitter:title" content="The global diversity of the major parasitic nematode Haemonchus contortus is shaped by human intervention and climate" />
<meta name="twitter:site" content="@biorxivpreprint" />
<meta name="twitter:card" content="summary" />
<meta name="twitter:image" content="https://www.biorxiv.org/sites/default/files/images/biorxiv_logo_homepage7-5-small.png" />
<meta name="twitter:description" content="The gastrointestinal parasite Haemonchus contortus is an haematophagous parasitic nematode of veterinary interest and a model for the study of drug resistance mechanisms or host-parasite interactions. To understand its evolutionary history, and its ability to adapt in the face of climatic and drug pressure, we have performed an extensive survey of genome-wide diversity using single-worm whole genome sequencing of 223 individuals sampled from 19 isolates spanning five continents. The pattern of global diversity is driven by an African origin for the species, together with contemporary dispersal that is consistent with modern human movement, with evidence for parasites spreading during the transatlantic slave trade and colonisation of Australia presented. Strong selective sweeps were identified in independent populations each surrounding the β-tubulin locus, a target of benzimidazole anthelmintic drug treatment used widely to control H. contortus infections. These signatures of selection were further supported by signals of diversifying selection enriched in genes involved in response to drugs, as well as other anthelmintic-associated biological functions including pharyngeal pumping and oviposition. From these analyses, we identify some known, and previously undescribed, candidate genes that may play a role in ivermectin resistance. Finally, we describe genetic signatures of climate-driven adaptation, revealing a gene acting as an epigenetic regulator and components of the dauer pathway may play a role in adaptation in the face of climatic fluctuations. These results begin to define genetic adaptation to climate for the first time in a parasitic nematode, and provides insight into the ongoing expansion in the range of Haemonchus contortus , which may have consequences for the management of this parasite." />
<meta name="og-title" property="og:title" content="The global diversity of the major parasitic nematode Haemonchus contortus is shaped by human intervention and climate" />
<meta name="og-url" property="og:url" content="https://www.biorxiv.org/content/10.1101/450692v2" />
<meta name="og-site-name" property="og:site_name" content="bioRxiv" />
<meta name="og-description" property="og:description" content="The gastrointestinal parasite Haemonchus contortus is an haematophagous parasitic nematode of veterinary interest and a model for the study of drug resistance mechanisms or host-parasite interactions. To understand its evolutionary history, and its ability to adapt in the face of climatic and drug pressure, we have performed an extensive survey of genome-wide diversity using single-worm whole genome sequencing of 223 individuals sampled from 19 isolates spanning five continents. The pattern of global diversity is driven by an African origin for the species, together with contemporary dispersal that is consistent with modern human movement, with evidence for parasites spreading during the transatlantic slave trade and colonisation of Australia presented. Strong selective sweeps were identified in independent populations each surrounding the β-tubulin locus, a target of benzimidazole anthelmintic drug treatment used widely to control H. contortus infections. These signatures of selection were further supported by signals of diversifying selection enriched in genes involved in response to drugs, as well as other anthelmintic-associated biological functions including pharyngeal pumping and oviposition. From these analyses, we identify some known, and previously undescribed, candidate genes that may play a role in ivermectin resistance. Finally, we describe genetic signatures of climate-driven adaptation, revealing a gene acting as an epigenetic regulator and components of the dauer pathway may play a role in adaptation in the face of climatic fluctuations. These results begin to define genetic adaptation to climate for the first time in a parasitic nematode, and provides insight into the ongoing expansion in the range of Haemonchus contortus , which may have consequences for the management of this parasite." />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.biorxiv.org/sites/default/files/images/biorxiv_logo_homepage7-5-small.png" />
<meta name="citation_date" content="2019-04-13" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/10.1101/450692v2.full.pdf" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/10.1101/450692v2.full.txt" />
<link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/10.1101/450692v2.ppt" />
<meta name="description" content="bioRxiv - the preprint server for biology, operated by Cold Spring Harbor Laboratory, a research and educational institution" />
<meta name="generator" content="Drupal 7 (http://drupal.org)" />
<link rel="canonical" href="https://www.biorxiv.org/content/10.1101/450692v2" />
<link rel="shortlink" href="https://www.biorxiv.org/node/608843" />
    <title>The global diversity of the major parasitic nematode Haemonchus contortus is shaped by human intervention and climate | bioRxiv</title>  
    <link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__jMRAK66KMC1e4TQlwUNn3KiWVDC5AjueUAYEm1xBY_U__KSUjdT4jdcJ4qJz6fKY1K9WkYh1a5EcaWZxt_-zbTis__QJacVmveFyLIBSPkp21xL0ZvuDW9DMBrD2HB1P4Ry5I.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q5p1rr" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__icrP-Y0qw2NP73rJxxkUtrJta0z3VLAABtNjfSdEghA__rlN3rx0wOoswGNHZMeZM96bNn-3cLWHXTztGDoThNlw__QJacVmveFyLIBSPkp21xL0ZvuDW9DMBrD2HB1P4Ry5I.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__e8NuQlin4bcnMtI2zOEVRblohOk3xZwfnFAFdibS6pw__XVRsOpcO2N5B6_yYKvNFz_CihNSGO6Yy3SOy-5iyzfE__QJacVmveFyLIBSPkp21xL0ZvuDW9DMBrD2HB1P4Ry5I.css" media="all" />
<style type="text/css" media="all">
/* <![CDATA[ */
#sliding-popup.sliding-popup-bottom,#sliding-popup.sliding-popup-bottom .eu-cookie-withdraw-banner,.eu-cookie-withdraw-tab{background:gray}#sliding-popup.sliding-popup-bottom.eu-cookie-withdraw-wrapper{background:transparent}#sliding-popup .popup-content #popup-text h1,#sliding-popup .popup-content #popup-text h2,#sliding-popup .popup-content #popup-text h3,#sliding-popup .popup-content #popup-text p,.eu-cookie-compliance-secondary-button,.eu-cookie-withdraw-tab{color:#fff !important}.eu-cookie-withdraw-tab{border-color:#fff}.eu-cookie-compliance-more-button{color:#fff !important}
/* ]]> */
</style>

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__QJacVmveFyLIBSPkp21xL0ZvuDW9DMBrD2HB1P4Ry5I.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__B6tGdsQSt1vT8d8j-19-oS4FKJe83RrSNBkjPs9jfos__6VkE5WcQmmQpSVTQx-xS2-zn_nlwFlDm-EN1HLJNjrA__QJacVmveFyLIBSPkp21xL0ZvuDW9DMBrD2HB1P4Ry5I.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__I6pBwZrmO51afTZVOjN83FdGH1QRb9t15lSLso0JVV4__F3sVKlGyHKCaAPI_WdiHD7l_-05B6b9bLdUwzi1me90__QJacVmveFyLIBSPkp21xL0ZvuDW9DMBrD2HB1P4Ry5I.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__QJacVmveFyLIBSPkp21xL0ZvuDW9DMBrD2HB1P4Ry5I.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__QJacVmveFyLIBSPkp21xL0ZvuDW9DMBrD2HB1P4Ry5I.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__CRdjIL0HKDL2T9slYvkTSTLypBBPjyoDhwBezBuDy7c__cHebPmPNdg2jyPmpyQpZWssJQ4cPrqfaabp5Or9KlkU__QJacVmveFyLIBSPkp21xL0ZvuDW9DMBrD2HB1P4Ry5I.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://d33xdlntwy0kbs.cloudfront.net/cshl_custom.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/all/libraries/modernizr/modernizr.min.js?q5p1rr"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__yOhrJUBmMpN-jsuQlJYmS8ovkQU6I6KInr295HrSTb0__c--zHhbuOWmkdWkJYXiNWDmkpDRPesmbBjGlQcyqBOY__QJacVmveFyLIBSPkp21xL0ZvuDW9DMBrD2HB1P4Ry5I.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var _prum=[['id', '52e95df3abe53d2e33000000'], ['mark', 'firstbyte',
      (new Date()).getTime()]]; (function() {
      var s=document.getElementsByTagName('script')[0],
      p=document.createElement('script');
      p.async='async'; p.src='//rum-static.pingdom.net/prum.min.js';s.parentNode.insertBefore(p,s);})();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js___KyvuEvfmewWxdOLBXTYsSUo8Pfm_DijV6fWtGvSL_k__HSKkTgIr7nDcdvQYewipf6v4ak6ZLUOQkwPj44wKouA__QJacVmveFyLIBSPkp21xL0ZvuDW9DMBrD2HB1P4Ry5I.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__3uCzzYE5jr6ly9nvsM8PRBRkhh4p0NWCMDja4EKBy4M__dmGGVYcB8LhnbR2iAmVW0ZyXI6QuFVYFRORa5XxglXs__QJacVmveFyLIBSPkp21xL0ZvuDW9DMBrD2HB1P4Ry5I.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__0zdUME0GLauYYyEch2tGPydIw-IG7Uizip0Qb4wUsig__3RgszrCgk4bOnTzo4vaQ-hrWzaG6oD19l-R9CQ1YQ-o__QJacVmveFyLIBSPkp21xL0ZvuDW9DMBrD2HB1P4Ry5I.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__W6wAjopaALytcBam7oT3WhEzllkWaB9R-8hazHyvShM__rK5YBIqBy9qt0di0Sn4DriBx9TJupdmv6uGmLJyZvDs__QJacVmveFyLIBSPkp21xL0ZvuDW9DMBrD2HB1P4Ry5I.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-45638026-1", {"cookieDomain":"auto"});ga("set", "page", location.pathname + location.search + location.hash);ga("send", "pageview");ga('create', 'UA-189672-38', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__1PtAOZ8x7l6QAi8D7X6Kj_Y7HcUOFzvWRvvYHUcCT98__U8IB92u_CnnGwJ-X_xaUa9lzsC8OzUfN0WkK2JQfirg__QJacVmveFyLIBSPkp21xL0ZvuDW9DMBrD2HB1P4Ry5I.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings,{"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"WfYyLCI9GhweOd1n29K-EzkIe0nLUHwaBEhJYtXkxsc","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"modules\/comment\/comment.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"modules\/forum\/forum.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"sites\/default\/modules\/biorxiv\/css\/biorxiv.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/flexible\/flexible.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_collection\/css\/highwire_subject_collection.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire.highwire-list-expand.css":1,"public:\/\/ctools\/css\/bab59090150caf0c64e27afa1b36eef1.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/contrib\/eu_cookie_compliance\/css\/eu_cookie_compliance.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"0":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_biorxiv_styles\/css\/biorxiv-styles.css.less":1,"https:\/\/d33xdlntwy0kbs.cloudfront.net\/cshl_custom.css":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_figures.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"http:\/\/d33xdlntwy0kbs.cloudfront.net\/cshl_custom.js":1,"sites\/all\/modules\/contrib\/eu_cookie_compliance\/js\/eu_cookie_compliance.js":1,"sites\/default\/modules\/jnl_biorxiv_styles\/js\/adv-search-collection-toggle.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/eu_cookie_compliance\/js\/jquery.cookie-1.4.1.min.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/default\/modules\/jnl_biorxiv_styles\/js\/biorxiv-scripts.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_nps_survey\/js\/nps_survey.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"sites\/default\/modules\/contrib\/disqus\/disqus.js":1,"misc\/textarea.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/contrib\/service_links\/js\/twitter_button.js":1,"sites\/all\/modules\/contrib\/service_links\/js\/facebook_like.js":1,"sites\/all\/modules\/contrib\/service_links\/js\/google_plus_one.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.highwire-list-expand.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"608843","apath":"\/biorxiv\/early\/2019\/04\/13\/450692.atom","pisa":"biorxiv;450692v2","markup":[{"requested":"index","variant":"index","view":"full","pisa":"biorxiv;450692\/DC1"},{"requested":"index","variant":"index","view":"full","pisa":"biorxiv;450692\/DC1"},{"requested":"full-text","variant":"full-text","view":"full","pisa":"biorxiv;450692v2"},{"requested":"index","variant":"index","view":"full","pisa":"biorxiv;450692\/DC1"},{"requested":"index","variant":"index","view":"full","pisa":"biorxiv;450692\/DC1"}],"processed":["highwire_math"],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"disqus":{"domain":"biorxivstage","url":"https:\/\/www.biorxiv.org\/content\/10.1101\/450692v2","title":"The global diversity of the major parasitic nematode Haemonchus contortus is shaped by human intervention and climate","identifier":"node\/608843"},"panels_ajax_pane":{"new-41":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u00225sepAf0y9PCtTsPeLM+PKoouMceqMoF6bRrWf2pnRhLIHzZwiIVlu8VDM\\\\\\\/vQVEq++JP4E\\\\\\\/1N74fM1KtBYkUjBDsdwZgpy9npJ1p\\\\\\\/5SvDo1sraUnOmS1Q48GZoZpMTVOj+7mtxKEmEht0DSQwSLTbdDJGCNsXltH9MPansSv5FyTgBcvDljKo07Ye92flT7l2sTmfqGxnNUFOON6s0SyZ\\\\\\\/Gk4HR4czzW28NTnpoNS1P0dAHbanSPLwBhl21M6VeYM1jQOVqwpUEFlwn7LJnYVhS0RYDAuSVMyInH11rlmCNszz8EAJ0nuvUQlQ2Kgt\\\\\\\/JP3EqSEAPLLn3CA+hEWsCm8ViHk8U0yhmMu07CionxV1D6xsy382FTEp0YwCBFhp7YQPpr3eLOUDk9PYhDVx6lDXzz76nYBt1Z\\\\\\\/jvcK4Xa3gaUbU1xMlEGJRnpe9Lsn4fNDvhu59xz4QcPWOg\\\\\\\/p\\\\\\\/R+jALfu6BOFUGpoTcxQZ3pCMxRR9\\\\\\\/dbk48GHTO4sK05mSjTou0dauaqLzxoTzyvZMUZcv7NlH\\\\\\\/SRZp3Crh5VUE5iIKG5UjuQtLX0aJ6dJdgkwnkFdT0JanxHXD8CJ3CnYBxch8hZUIyufA2e6+9xmksSD+t2qXmCCNm8JwSDjcJzZ33k1ufn3ss65\\\\\\\/JxKlbwYJPqBRQefHYjVUmZ1nIgi0wuLrK7YxBg\\\\\\\/TEKRtfp1orukmLibT4HFeDGVkwQzDHGjc9fM74DmEalvT+wtkWQy72ALdwRhlWtRPHjGgtzD7G7WJrnIVuvKu97mRWuW4ByvNsrwavtbriDPgaHdx6wnrNnhb2TBEIKjhcoS79ZFqN9zx1JvuyiVbtWUtCZRyYjm83jA1UI6nDUV1z0w3unNA7kw5C\\\\\\\/WdO2XDVn+cViBpZjkuSh3Shvm3rVZb0C4XvKGuIg7v0fuZ4YuPvTOQFolGFl2CKKORun\\\\\\\/a26Y2sNhAkjfWPWRijcaVmEvEWTtLuwycWpOsqcF99m5mH8NnsdvaJiFrtb4DCF4Dj6z+u5YHSODCLmdiuEbqjxzCb2qZYp5W2yiWqRLEbp1X9M6fQd67UxqKBRrLGyA5FScLUPnTITnnS8e3YZ4X+n9ztSDsj7fXz2482qUEnRQgPk0L2uK7xXwQeputzonsJZNt+qdiXqmD9sVViN1n4AyNyYOZzMPdRvovLTpC4g\\\\\\\/B1dZN5OjWK4aSQ07qTX2CuZhai61FiTZ1S6zNjbcbO+bPFBWbJXjghH\\\\\\\/LJ\\\\\\\/FdeRKcXednaR5JObna30oYUEujyZ25W1vUYmlmDiQb3nytvTP7iB3vWBjwsSQ+Pnir0mrNOqgKuOT+DJ8=\\\u0022,\\\u0022iv\\\u0022:\\\u0022WVKK6Mr6n\\\\\\\/SE\\\\\\\/6ytR\\\\\\\/+PGQ==\\\u0022,\\\u0022salt\\\u0022:\\\u0022f9d8845712f72c9fee5db7d9ff111eab\\\u0022}\u0022,\u0022hmac\u0022:\u00226e5dbd27e3a84a1667fc4f5aa87b861416ec8aa0d0aa9db0040cd5df6cf4aee3\u0022}"},"urlIsAjaxTrusted":{"\/content\/10.1101\/450692v2.full":true},"ws_fl":{"width":100,"height":21},"ws_gpo":{"size":"","annotation":"","lang":"","callback":"","width":300},"color":{"logo":"https:\/\/www.biorxiv.org\/sites\/default\/files\/bioRxiv_article.jpg"},"highwire_list_expand":{"is_collapsed":"1"},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"eu_cookie_compliance":{"popup_enabled":1,"popup_agreed_enabled":0,"popup_hide_agreed":0,"popup_clicking_confirmation":1,"popup_scrolling_confirmation":false,"popup_html_info":"\u003Cdiv\u003E\n  \u003Cdiv class =\u0022popup-content info\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n      \u003Cp\u003EWe use cookies on this site to enhance your user experience. By clicking any link on this page you are giving your consent for us to set cookies.\u003C\/p\u003E\n    \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022agree-button eu-cookie-compliance-default-button\u0022\u003EContinue\u003C\/button\u003E\n                    \u003Cbutton type=\u0022button\u0022 class=\u0022find-more-button eu-cookie-compliance-more-button\u0022\u003EFind out more\u003C\/button\u003E\n          \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E","use_mobile_message":false,"mobile_popup_html_info":"\u003Cdiv\u003E\n  \u003Cdiv class =\u0022popup-content info\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n          \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022agree-button eu-cookie-compliance-default-button\u0022\u003EContinue\u003C\/button\u003E\n                    \u003Cbutton type=\u0022button\u0022 class=\u0022find-more-button eu-cookie-compliance-more-button\u0022\u003EFind out more\u003C\/button\u003E\n          \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E\n","mobile_breakpoint":"768","popup_html_agreed":"\u003Cdiv\u003E\n  \u003Cdiv class=\u0022popup-content agreed\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n      \u003Ch2\u003EThank you for accepting cookies\u003C\/h2\u003E\u003Cp\u003EYou can now hide this message or find out more about cookies.\u003C\/p\u003E    \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022hide-popup-button eu-cookie-compliance-hide-button\u0022\u003EHide\u003C\/button\u003E\n              \u003Cbutton type=\u0022button\u0022 class=\u0022find-more-button eu-cookie-compliance-more-button-thank-you\u0022 \u003EMore info\u003C\/button\u003E\n          \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E","popup_use_bare_css":false,"popup_height":"auto","popup_width":"100%","popup_delay":1000,"popup_link":"\/help\/cookie-policy","popup_link_new_window":1,"popup_position":null,"popup_language":"en","store_consent":false,"better_support_for_screen_readers":0,"reload_page":0,"domain":"","popup_eu_only_js":0,"cookie_lifetime":"365","cookie_session":false,"disagree_do_not_show_popup":0,"method":"default","whitelisted_cookies":"","withdraw_markup":"\u003Cbutton type=\u0022button\u0022 class=\u0022eu-cookie-withdraw-tab\u0022\u003EPrivacy settings\u003C\/button\u003E\n\u003Cdiv class=\u0022eu-cookie-withdraw-banner\u0022\u003E\n  \u003Cdiv class=\u0022popup-content info\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n      \u003Cp\u003E\u0026lt;h2\u0026gt;We use cookies on this site to enhance your user experience\u0026lt;\/h2\u0026gt;\u0026lt;p\u0026gt;You have given your consent for us to set cookies.\u0026lt;\/p\u0026gt;\u003C\/p\u003E\n    \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022eu-cookie-withdraw-button\u0022\u003EWithdraw consent\u003C\/button\u003E\n    \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E\n","withdraw_enabled":false},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1,"trackUrlFragments":1},"jnl_biorxiv_styles":{"defaultJCode":"biorxiv"},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-608843 node-type-highwire-article context-content hw-default-jcode-biorxiv hw-article-type-article hw-article-category-new-results">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-M677548" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-M677548');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
        <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
    
  <div id="zone-branding" class="zone zone-branding clearfix print-display-block container-30">
    <div class="grid-15 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="bioRxiv" src="https://www.biorxiv.org/sites/default/files/bioRxiv_article.jpg" /></a>      </div>
                </div>
          </div>
</div><div class="grid-11 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-system block-menu block-main-menu block-system-main-menu odd block-without-title" id="block-system-main-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/" title="" class="" data-icon-position="" data-hide-link-title="0">Home</a></li>
<li class="leaf" role="menuitem"><a href="/about-biorxiv" class="" data-icon-position="" data-hide-link-title="0">About</a></li>
<li class="leaf" role="menuitem"><a href="/submit-a-manuscript" class="" data-icon-position="" data-hide-link-title="0">Submit</a></li>
<li class="last leaf" role="menuitem"><a href="/content/alertsrss" title="" class="" data-icon-position="" data-hide-link-title="0">ALERTS / RSS</a></li>
</ul></nav>    </div>
  </div>
</div><div class="block block-panels-mini block-biorxiv-search-box block-panels-mini-biorxiv-search-box even block-without-title" id="block-panels-mini-biorxiv-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-biorxiv_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/10.1101/450692v2.full" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--2" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-DsAwFQt5tWYz2wAhwBKQLz5ci_xZqi1PzJWP2iE1368" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2 advanced-search-link" >
  
      
  
  <div class="pane-content">
    <a href="/search">Advanced Search</a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix container-30">
  	      </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="608843" id="node608843" data-pisa="biorxiv;450692v2" data-pisa-master="biorxiv;450692" data-apath="/biorxiv/early/2019/04/13/450692.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div  class="highwire-cite highwire-cite-highwire-article highwire-citation-biorxiv-article-top clearfix has-author-tooltip" >

      <span class="biorxiv-article-type">
        New Results    </span>
  
        <h1 class="highwire-cite-title" id="page-title">The global diversity of the major parasitic nematode <em>Haemonchus contortus</em> is shaped by human intervention and climate</h1>  
      <div  class="highwire-cite-authors" ><span  class="highwire-citation-authors"><span class="highwire-citation-author first hw-author-orcid-logo-wrapper" data-delta="0"><a href="http://orcid.org/0000-0002-4032-139X" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a><span class="nlm-given-names">G.</span> <span class="nlm-surname">Sallé</span></span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="1"><a href="http://orcid.org/0000-0001-9167-7532" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a><span class="nlm-given-names">S.R.</span> <span class="nlm-surname">Doyle</span></span>, <span class="highwire-citation-author" data-delta="2"><span class="nlm-given-names">J.</span> <span class="nlm-surname">Cortet</span></span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="3"><a href="http://orcid.org/0000-0002-7379-9023" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a><span class="nlm-given-names">J.</span> <span class="nlm-surname">Cabaret</span></span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="4"><a href="http://orcid.org/0000-0002-9581-0377" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a><span class="nlm-given-names">M.</span> <span class="nlm-surname">Berriman</span></span>, <span class="highwire-citation-author" data-delta="5"><span class="nlm-given-names">N.</span> <span class="nlm-surname">Holroyd</span></span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="6"><a href="http://orcid.org/0000-0001-5475-3583" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a><span class="nlm-given-names">J.A</span> <span class="nlm-surname">Cotton</span></span></span></div>
  
      <div  class="highwire-cite-metadata" ><span  class="highwire-cite-metadata-doi highwire-cite-metadata"><span class="label">doi:</span> https://doi.org/10.1101/450692 </span></div>
  
  
  </div>
<div id="hw-article-author-popups-node608843" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">G. Sallé </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-institution'>Wellcome Sanger Institute</span>, Wellcome Genome Campus, Hinxton, Cambridge, CB10 1SA, <span class='nlm-country'>United Kingdom</span></div><div class='author-affiliation'><span class='nlm-institution'>INRA – U. Tours</span>, UMR 1282 ISP Infectiologie et Santé Publique, Centre de recherche Val de Loire, Nouzilly, <span class='nlm-country'>France</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=G.%2BSall%C3%A9%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Sall%C3%A9%20G&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AG.%2BSall%25C3%25A9%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0002-4032-139X" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for G. Sallé</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:guillaume.salle@inra.fr" class="" data-icon-position="" data-hide-link-title="0">guillaume.salle@inra.fr</a></span></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">S.R. Doyle </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-institution'>Wellcome Sanger Institute</span>, Wellcome Genome Campus, Hinxton, Cambridge, CB10 1SA, <span class='nlm-country'>United Kingdom</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=S.R.%2BDoyle%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Doyle%20S&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AS.R.%2BDoyle%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0001-9167-7532" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for S.R. Doyle</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">J. Cortet </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-institution'>INRA – U. Tours</span>, UMR 1282 ISP Infectiologie et Santé Publique, Centre de recherche Val de Loire, Nouzilly, <span class='nlm-country'>France</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=J.%2BCortet%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Cortet%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJ.%2BCortet%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">J. Cabaret </div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=J.%2BCabaret%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Cabaret%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AJ.%2BCabaret%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-7379-9023" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for J. Cabaret</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">M. Berriman </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-institution'>Wellcome Sanger Institute</span>, Wellcome Genome Campus, Hinxton, Cambridge, CB10 1SA, <span class='nlm-country'>United Kingdom</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=M.%2BBerriman%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Berriman%20M&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AM.%2BBerriman%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-9581-0377" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for M. Berriman</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">N. Holroyd </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-institution'>Wellcome Sanger Institute</span>, Wellcome Genome Campus, Hinxton, Cambridge, CB10 1SA, <span class='nlm-country'>United Kingdom</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=N.%2BHolroyd%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Holroyd%20N&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AN.%2BHolroyd%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-6"><div class="author-tooltip-name">J.A Cotton </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-institution'>Wellcome Sanger Institute</span>, Wellcome Genome Campus, Hinxton, Cambridge, CB10 1SA, <span class='nlm-country'>United Kingdom</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=J.A%2BCotton%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Cotton%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AJ.A%2BCotton%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0001-5475-3583" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for J.A Cotton</a></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/10.1101/450692v2" class="panels-ajax-tab-tab" data-panel-name="biorxiv_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:608843" data-trigger="" data-url-enabled="1">Abstract</a><a href="/panels_ajax_tab/biorxiv_tab_art/node:608843/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/10.1101/450692v2.full-text" class="panels-ajax-tab-tab" data-panel-name="article_tab_full_text" data-target-id="highwire_article_tabs" data-entity-context="node:608843" data-trigger="full-text" data-url-enabled="1">Full Text</a><a href="/panels_ajax_tab/article_tab_full_text/node:608843/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/10.1101/450692v2.article-info" class="panels-ajax-tab-tab" data-panel-name="biorxiv_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:608843" data-trigger="article-info" data-url-enabled="1">Info/History</a><a href="/panels_ajax_tab/biorxiv_tab_info/node:608843/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/10.1101/450692v2.article-metrics" class="panels-ajax-tab-tab" data-panel-name="article_tab_metrics" data-target-id="highwire_article_tabs" data-entity-context="node:608843" data-trigger="article-metrics" data-url-enabled="1">Metrics</a><a href="/panels_ajax_tab/article_tab_metrics/node:608843/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/10.1101/450692v2.supplementary-material" class="panels-ajax-tab-tab" data-panel-name="biorxiv_tab_data" data-target-id="highwire_article_tabs" data-entity-context="node:608843" data-trigger="supplementary-material" data-url-enabled="1">Supplementary material</a><a href="/panels_ajax_tab/biorxiv_tab_data/node:608843/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/10.1101/450692v2.full.pdf+html" class="panels-ajax-tab-tab" data-panel-name="biorxiv_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:608843" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i> Preview PDF</a><a href="/panels_ajax_tab/biorxiv_tab_pdf/node:608843/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="article_tab_full_text" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.biorxiv.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-article_tab_full_text"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-markup" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2 class="">Abstract</h2><p id="p-2">The gastrointestinal parasite <em>Haemonchus contortus</em> is an haematophagous parasitic nematode of veterinary interest and a model for the study of drug resistance mechanisms or host-parasite interactions. To understand its evolutionary history, and its ability to adapt in the face of climatic and drug pressure, we have performed an extensive survey of genome-wide diversity using single-worm whole genome sequencing of 223 individuals sampled from 19 isolates spanning five continents. The pattern of global diversity is driven by an African origin for the species, together with contemporary dispersal that is consistent with modern human movement, with evidence for parasites spreading during the transatlantic slave trade and colonisation of Australia presented. Strong selective sweeps were identified in independent populations each surrounding the β-tubulin locus, a target of benzimidazole anthelmintic drug treatment used widely to control <em>H. contortus</em> infections. These signatures of selection were further supported by signals of diversifying selection enriched in genes involved in response to drugs, as well as other anthelmintic-associated biological functions including pharyngeal pumping and oviposition. From these analyses, we identify some known, and previously undescribed, candidate genes that may play a role in ivermectin resistance. Finally, we describe genetic signatures of climate-driven adaptation, revealing a gene acting as an epigenetic regulator and components of the <em>dauer</em> pathway may play a role in adaptation in the face of climatic fluctuations. These results begin to define genetic adaptation to climate for the first time in a parasitic nematode, and provides insight into the ongoing expansion in the range of <em>Haemonchus contortus</em>, which may have consequences for the management of this parasite.</p></div><div class="section" id="sec-1"><h2 class="">Introduction</h2><p id="p-3">Nematodes have evolved to exploit a wide diversity of ecological niches. Although many sustain a free-living lifestyle, parasitic nematodes rely on one or more hosts to complete their life cycle. Many parasitic nematodes undergo a complex series of morphological changes, linked to migration through their hosts to establish a mature infection<sup><a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">1</a></sup>. Their complex life cycles may involve both intermediate hosts, vectors or time spent in the environment, where they face harsh and variable conditions such as frost or drought that they must withstand between infection of their hosts<sup><a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">2</a></sup>. Parasitic nematodes have adapted to a wide range of threats, including predation, climate and the immune responses of a great diversity of both plant and animal host species<sup><a id="xref-ref-34-1" class="xref-bibr" href="#ref-34">34</a></sup>.</p><p id="p-4">The evolutionary success of parasitic nematodes comes at a cost to humans, either directly as they significantly impact human health (amounting to a loss of 10 million disability-adjusted life-years)<sup><a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">3</a></sup>, or indirectly via major economic losses in plant<sup><a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">4</a></sup> and livestock production<sup><a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">5</a></sup>, and parasite control. The control of animal parasitic nematodes relies almost exclusively on anthelmintic drugs, administered on a recurrent basis in livestock<sup><a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">6</a></sup> and through mass-drug administration program in humans<sup><a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">7</a></sup>. Although the success of such strategies was originally undeniable, the emergence of drug resistant veterinary parasites <sup><a id="xref-ref-5-2" class="xref-bibr" href="#ref-5">5</a></sup>, or the reported lack of efficacy in human-infective species<sup><a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">8</a></sup>, threatens ongoing control efforts for many parasitic infections. Vaccines offer an attractive alternate control strategy against these parasites: the extensive genetic diversity and immune-regulatory properties of parasites has, however, greatly hampered vaccine development<sup><a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">9</a></sup>, and although two licensed vaccines are currently available for veterinary purposes<sup><a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">10</a></sup>, transcriptomic plasticity of the parasite following vaccine challenge may contribute to circumvent the vaccinal response of their host<sup><a id="xref-ref-10-2" class="xref-bibr" href="#ref-10">10</a></sup>. It is therefore clear that novel, sustainable control strategies are required. The potential of helminths to adapt to – and thus escape – control measures lies in their underlying genetic diversity. A greater understanding of the extent of this diversity and the processes that shape it throughout their range should provide insight into the mechanisms by which they adapt, and may identify new targets which may be exploited for control.</p><p id="p-5">The trichostrongylid <em>Haemonchus contortus</em> is a gastrointestinal parasite of ruminants in tropical and temperate regions throughout the world, and causes significant economic and animal health burden particularly on sheep husbandry. It is also emerging as a model parasitic nematode system for functional and comparative genomics, largely due to its rapid ability to acquire drug resistance, the relative tractability of its life-cycle under laboratory conditions<sup><a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">11</a></sup>, the development of extensive genomic resources<sup><a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">12</a>,<a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">13</a></sup>, and its relatively close relationship with other clade V parasitic nematodes of both veterinary and medical importance (i.e, other gastro-intestinal nematodes of livestock and human hookworms)<sup><a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">14</a></sup>. We have used whole genome sequencing of 223 individual <em>H. contortus</em> sampled from 19 isolates spanning five continents to characterise genome- and population-wide genetic diversity throughout its range. This survey of genome-wide diversity has revealed old and new genetic connectivity influenced by human history, and signatures of selection in response to anthelmintic exposure and local climatic variation.</p></div><div class="section" id="sec-2"><h2 class="">Results</h2><div id="sec-3" class="subsection"><h3><em>Haemonchus contortus</em> isolates are genetically diverse, with large effective population sizes</h3><p id="p-6">Whole genome sequencing of 223 individuals from 19 isolates (<a id="xref-fig-1-1" class="xref-fig" href="#F1">Fig. 1a</a>; Supplementary Table 1) revealed 23,868,644 SNPs with a genome-wide distribution of 1 SNP per 9.94 bp on average.</p><div id="F1" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/04/13/450692/F1.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Global distribution of Haemonchus contortus isolates. Isolates sampled are coloured by geographical region (sand: South-America, brown: Western-Africa, dark green: Mediterranean area, black: Subtropical Africa, red: Australia). Shape indicates the anthelmintic resistance status of each isolate to fenbendazole (resistant =squares; susceptible =circles) or resistant to both ivermectin and fenbendazole (triangles)." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-691381162" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Global distribution of &lt;em&gt;Haemonchus contortus&lt;/em&gt; isolates. Isolates sampled are coloured by geographical region (sand: South-America, brown: Western-Africa, dark green: Mediterranean area, black: Subtropical Africa, red: Australia). Shape indicates the anthelmintic resistance status of each isolate to fenbendazole (resistant =squares; susceptible =circles) or resistant to both ivermectin and fenbendazole (triangles).&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 1a." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2019/04/13/450692/F1.medium.gif" width="440" height="253"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 1a." src="https://www.biorxiv.org/content/biorxiv/early/2019/04/13/450692/F1.medium.gif" width="440" height="253"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/04/13/450692/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1a." data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/04/13/450692/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml"><span class="fig-label">Figure 1a.</span> <p id="p-7" class="first-child">Global distribution of <em>Haemonchus contortus</em> isolates. Isolates sampled are coloured by geographical region (sand: South-America, brown: Western-Africa, dark green: Mediterranean area, black: Subtropical Africa, red: Australia). Shape indicates the anthelmintic resistance status of each isolate to fenbendazole (resistant =squares; susceptible =circles) or resistant to both ivermectin and fenbendazole (triangles).</p><div class="sb-div caption-clear"></div></div></div><p id="p-8">Only a proportion of the filtered SNPs were called in more than half the individuals (n = 3,338,155 SNPs), with only 411,574 SNPs segregating with a MAF &gt; 5% in those individuals. Estimates of nucleotide diversity (π) in isolates with at least five individuals ranged from 0.44% (STA.2) to 1.3% (NAM; Supplementary Table 2). Variance in n across autosomes among isolates was partly explained by isolate mean coverage (<em>F</em><sub><em>(1,84)</em></sub> = 9.49, <em>P</em> = 0.003; Supplementary note). To account for this bias, estimates were obtained from three subsets of isolates with the highest coverage (greater than 8× on average) from France (FRA.1 and FRA.2), Guadeloupe (GUA) and Namibia (NAM) that yielded slightly higher values that ranged between 0.65 and 1.14%. Overall, these data show equivalent diversity levels to <em>Drosophila melanogaster</em> (ranging between 0.53% and 1.71%)<sup><a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">16</a></sup> but represented approximately 1.6 to 45-fold greater diversity than two filarial nematode species for which similar statistics are available (0.02% for <em>Wuchereria bancrofti</em> larvae<sup><a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">17</a></sup>; 0.01% and 0.4% for <em>π</em><sub><em>S</em></sub> and <em>π</em><sub>N</sub> in <em>Onchocerca volvulus<sup><a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">18</a></sup></em> Supplementary Fig. 1).</p><p id="p-9">To begin to explore the global diversity of <em>H. contortus</em>, we performed a principal component analysis (PCA) of genetic variation, which revealed three broad genetic clusters of isolates (<a id="xref-fig-2-1" class="xref-fig" href="#F2">Fig. 1b</a>) that largely coincided with the geographic region from which they were sampled, including: (i) Sub-tropical African isolates (NAM, STA and ZAI), (ii) Atlantic isolates including Morocco (MOR), São Tomé (STO), Benin (BEN), Brazil (BRA) and Guadeloupe (GUA), and (iii) the remainder from the Mediterranean area (FRA, ACO) and Oceania (AUS.1, AUS.2 and IND). The greatest diversity among samples was identified in the African and South American isolates, with East African samples spread along PC1 and West African and South American samples along PC2.</p><div id="F2" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/04/13/450692/F2.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Principal component analysis based on genotype likelihood inferred from whole genome sequences of 223 individual males (243,012 variants considered). Samples are coloured by geographic region described in panel 1a." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-691381162" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Principal component analysis based on genotype likelihood inferred from whole genome sequences of 223 individual males (243,012 variants considered). Samples are coloured by geographic region described in panel 1a.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 1b." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2019/04/13/450692/F2.medium.gif" width="440" height="272"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 1b." src="https://www.biorxiv.org/content/biorxiv/early/2019/04/13/450692/F2.medium.gif" width="440" height="272"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/04/13/450692/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1b." data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/04/13/450692/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 1b.</span> <p id="p-10" class="first-child">Principal component analysis based on genotype likelihood inferred from whole genome sequences of 223 individual males (243,012 variants considered). Samples are coloured by geographic region described in panel 1a.</p><div class="sb-div caption-clear"></div></div></div><p id="p-11">Using estimates of nucleotide diversity estimates, together with the the <em>C. elegans</em><sup><a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">20</a></sup> mutation rate and assuming a balanced sex ratio<sup><a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">21</a></sup>, we inferred the current effective population size (<em>N</em><sub><em>e</em></sub>) of <em>H. contortus</em> to be between 0.60 and 1.05 million. MSMC analysis, which models past recombination events based on heterozygosity patterns along the genome, revealed the historical <em>N</em><sub><em>e</em></sub> has remained within a slightly lower range of values for most of the sampled time interval, i.e. from 2.5 to 500 thousand years ago (kya), with extreme estimates falling between 1.5 × 10<sup>5</sup> and 6.1 × 10<sup>5</sup> individuals for GUA (633 years ago) and NAM (415 years ago) isolates, respectively (<a id="xref-fig-3-1" class="xref-fig" href="#F3">Fig. 1c</a>). <em>N<sub>e</sub></em> estimates remained relatively constant in most populations until 2.5 Kya; since this time, GUA and FRA.1 isolates suffered a more drastic reduction in <em>N<sub>e</sub></em>, which ranged for each isolate between 0.859 × 10<sup>5</sup> and 1.2 × 10<sup>5</sup> individuals approximately 633 and 452 years ago, respectively (<a id="xref-fig-3-2" class="xref-fig" href="#F3">Fig. 1c</a>). (<a id="xref-fig-3-3" class="xref-fig" href="#F3">Fig. 1c</a>).</p><div id="F3" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/04/13/450692/F3.large.jpg?width=800&amp;height=600&amp;carousel=1" title="MSMC effective population size across time for 6 isolates. Coloured shaded area represents range of values estimated from a cross-validation procedure with five replicates, computed by omitting one chromosome out at a time." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-691381162" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;MSMC effective population size across time for 6 isolates. Coloured shaded area represents range of values estimated from a cross-validation procedure with five replicates, computed by omitting one chromosome out at a time.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 1c." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2019/04/13/450692/F3.medium.gif" width="416" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 1c." src="https://www.biorxiv.org/content/biorxiv/early/2019/04/13/450692/F3.medium.gif" width="416" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/04/13/450692/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1c." data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/04/13/450692/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 1c.</span> <p id="p-12" class="first-child">MSMC effective population size across time for 6 isolates. Coloured shaded area represents range of values estimated from a cross-validation procedure with five replicates, computed by omitting one chromosome out at a time.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-4" class="subsection"><h3>Global population connectivity of Haemonchus contortus is characterised by old and new migration</h3><p id="p-13">To explore the global connectivity between isolates, we used a number of complementary approaches. Phylogenetic relationships determined by nuclear (Supplementary Fig. 2) and mitochondrial (<a id="xref-fig-4-1" class="xref-fig" href="#F4">Fig. 2a</a>; PCA of mtDNA genetic diversity is presented in Supplementary Fig. 3) diversity broadly supported the initial PCA analysis (<a id="xref-fig-2-2" class="xref-fig" href="#F2">Fig. 1b</a>), each revealing three main groups of samples partitioned by broad geographic regions, <em>i.e</em>. Africa, Oceania and Mediterranean groups. However, the mitochondrial data revealed further subdivision of the Oceanian and Mediterranean clades than nuclear data alone.</p><div id="F4" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/04/13/450692/F4.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Unrooted maximum likelihood phylogenetic tree constructed constructed using 3,052 SNPs from 223 individual mitochondrial genomes. Circles indicate bootstrap support for each branch, blue if support was higher than 70%, red elsewhere. Branches leading to a sample identifier are coloured by geographic region described in Figure 1a. Mitogroups are annotated with constitutive sample populations." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-691381162" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Unrooted maximum likelihood phylogenetic tree constructed constructed using 3,052 SNPs from 223 individual mitochondrial genomes. Circles indicate bootstrap support for each branch, blue if support was higher than 70%, red elsewhere. Branches leading to a sample identifier are coloured by geographic region described in Figure 1a. Mitogroups are annotated with constitutive sample populations.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 2a." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2019/04/13/450692/F4.medium.gif" width="405" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 2a." src="https://www.biorxiv.org/content/biorxiv/early/2019/04/13/450692/F4.medium.gif" width="405" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/04/13/450692/F4.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2a." data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/04/13/450692/F4.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 2a.</span> <p id="p-14" class="first-child">Unrooted maximum likelihood phylogenetic tree constructed constructed using 3,052 SNPs from 223 individual mitochondrial genomes. Circles indicate bootstrap support for each branch, blue if support was higher than 70%, red elsewhere. Branches leading to a sample identifier are coloured by geographic region described in <a id="xref-fig-1-2" class="xref-fig" href="#F1">Figure 1a</a>. Mitogroups are annotated with constitutive sample populations.</p><div class="sb-div caption-clear"></div></div></div><p id="p-15">The presence of close genetic relationships between geographically distant isolates, resulting in a weak phylogeographic signal (Mantel’s test <em>r</em> = 0.10, <em>P</em> = 0.001), was inconsistent with a simple isolation-by-distance scenario. This observation was supported by, for example, little genetic differentiation (measured by F<sub>ST</sub>) between geographically distant French (FRA) and Oceanian (AUS.1, AUS.2, IND) isolates (<a id="xref-fig-5-1" class="xref-fig" href="#F5">Fig. 2b</a>).</p><div id="F5" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/04/13/450692/F5.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Matrix showing pairwise dissimilarity between individuals (upper right) and pairwise FST between isolates (lower left)." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-691381162" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Matrix showing pairwise dissimilarity between individuals (upper right) and pairwise F&lt;sub&gt;ST&lt;/sub&gt; between isolates (lower left).&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 2b." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2019/04/13/450692/F5.medium.gif" width="440" height="251"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 2b." src="https://www.biorxiv.org/content/biorxiv/early/2019/04/13/450692/F5.medium.gif" width="440" height="251"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/04/13/450692/F5.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2b." data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/04/13/450692/F5.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 2b.</span> <p id="p-16" class="first-child">Matrix showing pairwise dissimilarity between individuals (upper right) and pairwise F<sub>ST</sub> between isolates (lower left).</p><div class="sb-div caption-clear"></div></div></div><p id="p-17">The greatest genetic dissimilarity was found between subtropical African isolates and the French and Australian isolates, with mean F<sub>ST</sub> across comparisons of 0.21 (F<sub>ST</sub> range according to isolate pairs: 0.06 – 0.42) and 0.24 (F<sub>ST</sub> range: 0.17 – 0.33) respectively (<a id="xref-fig-5-2" class="xref-fig" href="#F5">Fig. 2b</a>, Supplementary Fig. 4). The divergence of African isolates was likely due to the higher within-isolate diversity (+14% higher mitochondrial nucleotide diversity) relative to others (P = 0.004; mitochondrial nucleotide diversity of 0.63% ± 37%, 0.58% ± 0.31%, 0.66 ± 0.32%, 0.6% ± 0.35%, for Mediterranean, Oceanian, American and south-African isolates respectively).</p><p id="p-18">The higher genetic differentiation (8.9% difference, <em>F</em><sub><em>(1,134)</em></sub> = 11.36, <em>P</em> = 0.0009) between STO and other isolates relative to other pairwise comparisons, almost certainly reflects the isolation of an island population relative to continental populations (<a id="xref-fig-5-3" class="xref-fig" href="#F5">Fig. 2b</a>). STO samples also displayed higher genetic dissimilarity (5.7% difference, <em>F</em><sub><em>(1,7567)</em></sub> = 581, <em>P</em> &lt; 10<sup>−4</sup>) to other other samples. Inference of the joint history of STO and other African populations supported this view. The best-fitting model (supplementary Table 3) was consistent with either ancient symmetrical gene flow followed by isolation or an early split followed by secondary contact before isolation with MOR population (supplementary Table 3). This latter demography would underpin the pattern of admixture detected between STO and MOR (<a id="xref-fig-6-1" class="xref-fig" href="#F6">Fig. 2c</a>).</p><div id="F6" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/04/13/450692/F6.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Admixture analysis of 223 individuals. A cluster size of K = 3 is presented, determined from sites with minor allele frequency above 5% and call rate higher than 50%. Admixture pattern for other values of K is provided in supplementary Figure 5. Isolates are presented sorted along their longitudinal range, and samples sorted by assignment to the 3 clusters." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-691381162" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Admixture analysis of 223 individuals. A cluster size of K = 3 is presented, determined from sites with minor allele frequency above 5% and call rate higher than 50%. Admixture pattern for other values of K is provided in supplementary Figure 5. Isolates are presented sorted along their longitudinal range, and samples sorted by assignment to the 3 clusters.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 2c." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2019/04/13/450692/F6.medium.gif" width="440" height="101"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 2c." src="https://www.biorxiv.org/content/biorxiv/early/2019/04/13/450692/F6.medium.gif" width="440" height="101"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/04/13/450692/F6.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2c." data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/04/13/450692/F6.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 2c.</span> <p id="p-19" class="first-child">Admixture analysis of 223 individuals. A cluster size of K = 3 is presented, determined from sites with minor allele frequency above 5% and call rate higher than 50%. Admixture pattern for other values of K is provided in supplementary Figure 5. Isolates are presented sorted along their longitudinal range, and samples sorted by assignment to the 3 clusters.</p><div class="sb-div caption-clear"></div></div></div><p id="p-20">A closer inspection of GUA samples revealed a mixed ancestry, likely derived from West African and Mediterraneanheritage. A subset of GUA samples showed limited genetic differentiation to FRA isolates (F<sub>ST</sub> range = 0.07 – 0.10; <a id="xref-fig-5-4" class="xref-fig" href="#F5">Fig. 2b</a>), whereas the remaining GUA samples were genetically similar to isolates from the West African coast, <em>i.e</em>. ACO, BEN, MOR, STO (<a id="xref-fig-5-5" class="xref-fig" href="#F5">Fig. 2b &amp; c</a>), as indicated by lower F<sub>ST</sub> estimates with these isolates (F<sub>ST</sub> range = 0.07 – 0.13; Wilcoxon’s test, <em>P</em> = 0.07; <a id="xref-fig-6-2" class="xref-fig" href="#F6">Fig. 2b</a>) and evidence of shared ancestry from the admixture analysis (<a id="xref-fig-6-3" class="xref-fig" href="#F6">Fig. 2c</a>, Supplementary Fig. 5 and 6). A particularly close relationship was identified between STO and GUA (F<sub>ST</sub> = 0.10; <a id="xref-fig-5-6" class="xref-fig" href="#F5">Fig. 2b</a>). This conflicting genetic origin among GUA sub-isolates is responsible for the higher nucleotide diversity observed in GUA as a whole (Supplementary Table 2).</p><p id="p-21">The patterns of genetic connectivity support at least three distinct migration events in time and space that we investigated using forward genetic simulations (supplementary Table 3). First, we detect an out-of-Africa scenario, whereby the greatest diversity was sampled within Africa, and that isolates outside of Africa represent a subset of this diversity. Consistent with this hypothesis, nuclear genome variation of non-African isolates experienced a genetic bottleneck that occurred between 2.5 and 10 kya that is not present in the African isolates sampled (<a id="xref-fig-3-4" class="xref-fig" href="#F3">Fig. 1c</a>). Bayesian coalescent estimate of this initial divergence from mitochondrial genome data yielded an overlapping time range of between 3.6 kya and 4.1 kya (Supplementary Fig. 7). Genetic simulations of the joint demography between FRA.1 and African populations also favoured complex scenarios involving early split (maximum likelihood estimates ranging between 13 and 25 Kya) followed by ongoing gene flow with STA populations or more recent isolation with Namibia (supplementary Table 3). Secondly, the genetic connectivity of West African and American isolates is consistent with parasites spreading during the trans-Atlantic slave trade movement. The scenario linking GUA and STO was compatible with initial population division occurring around 1640 Common Era (CE) ± 167 years (supplementary table 3), consistent with migration associated with colonization and slave trade that occurred in the West Indies under French influence during that time<sup><a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">22</a></sup>. The third pattern of connectivity likely reflects British colonisation of Australia in late 1700’s: the interweaving of Australian and South-African worms into the Mediterranean phylogenetic haplogroup (<a id="xref-fig-5-7" class="xref-fig" href="#F5">Fig. 2b</a>) may mirror the foundation of Australian Merino sheep, which were first introduced into Australia from South Africa, before additional contributions from Europe and America were made<sup><a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">23</a>,<a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">24</a></sup>. The admixture pattern observed for worms from the two countries matched their shared ancestry (<a id="xref-fig-6-4" class="xref-fig" href="#F6">Fig. 2c</a>), as well as a genetic connection between America and Australia (seen for the AUS.1 isolate; <a id="xref-fig-6-5" class="xref-fig" href="#F6">Fig. 2c</a>). Although maximum likelihood estimates supported these scenarios with the isolation of European and South-African isolates occurring between 1794 and 1871 CE (supplementary Table 3), broad uncertainty limited our ability to specifically define the timings of these events (Supplementary Table 3). However, the split between Australian isolates occurred in 1895 CE ± 132 years, consistent with the initial foundation of the sheep industry in this country.These complex patterns reiterate that human movement has played an important role in shaping the diversity of this livestock parasite throughout the world.</p></div><div id="sec-5" class="subsection"><h3>Anthelmintic resistance has left distinct patterns of diversity in the Haemonchus contortus genome</h3><p id="p-22">The extensive use of anthelmintics has been and remains the primary means of <em>H. contortus</em> and other gastrointestinal control worldwide. This strategy has resulted in the independent emergence of drug-resistant isolates throughout the world, which now limits farming in some areas. Strong selection should impact the distribution of genetic variation within isolates; signatures of selection may reveal genes associated with drug resistance, knowledge of which may contribute to monitoring the emergence and spread of drug resistant isolates, and the design of new control strategies.</p><p id="p-23">The genetic determinants of benzimidazole resistance is perhaps the best characterised of all anthelmintics, with either one of three amino acid residue changes – F167Y, E198A, and F200Y – in the beta tubulin isotype 1 (<em>Hco-tbb-iso-1)</em> protein capable of mediating phenotypic resistance. We identified indications of a selective sweep in the region surrounding the <em>Hco-tbb-iso-1</em> locus in the resistant isolates analysed. Focusing on three isolates with the highest coverage, we found an average 2.31-fold reduction of Tajima’s <em>D</em> coefficient within 1 Mbp (<a id="xref-fig-7-1" class="xref-fig" href="#F7">Fig. 3a</a>, Supplementary Fig. 8) and an average 33% reduction in nucleotide diversity (Supplementary Fig. 9) within this region relative to the rest of chromosome I.</p><div id="F7" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/04/13/450692/F7.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Analysis of Tajima&#x2019;s D surrounding the Hco-tbb-iso-1 locus on chromosome I. A total of 10-Mbp surrounding was considered (pink), of which 1 Mbp nearest to Hco-tbb-iso-1 is highlighted (blue). Isolates compared included benzimidazole-resistant French (FRA.1), Guadeloupian (GUA), Namibian (NAM) or benzimidazole susceptible Australian (AUS.2) and Zaire (ZAI) isolates. Mean expected Tajima&#x2019;s D (solid grey line) and 99% confidence interval (dotted line) were estimated from a 1,000 simulated 10-Kbp wide sequences following MSMC-inferred demography." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-691381162" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Analysis of Tajima&#x2019;s &lt;em&gt;D&lt;/em&gt; surrounding the &lt;em&gt;Hco-tbb-iso-1&lt;/em&gt; locus on chromosome I. A total of 10-Mbp surrounding was considered (pink), of which 1 Mbp nearest to &lt;em&gt;Hco-tbb-iso-1&lt;/em&gt; is highlighted (blue). Isolates compared included benzimidazole-resistant French (FRA.1), Guadeloupian (GUA), Namibian (NAM) or benzimidazole susceptible Australian (AUS.2) and Zaire (ZAI) isolates. Mean expected Tajima&#x2019;s &lt;em&gt;D&lt;/em&gt; (solid grey line) and 99% confidence interval (dotted line) were estimated from a 1,000 simulated 10-Kbp wide sequences following MSMC-inferred demography.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 3a." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2019/04/13/450692/F7.medium.gif" width="440" height="394"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 3a." src="https://www.biorxiv.org/content/biorxiv/early/2019/04/13/450692/F7.medium.gif" width="440" height="394"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/04/13/450692/F7.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 3a." data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/04/13/450692/F7.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 3a.</span> <p id="p-24" class="first-child">Analysis of Tajima’s <em>D</em> surrounding the <em>Hco-tbb-iso-1</em> locus on chromosome I. A total of 10-Mbp surrounding was considered (pink), of which 1 Mbp nearest to <em>Hco-tbb-iso-1</em> is highlighted (blue). Isolates compared included benzimidazole-resistant French (FRA.1), Guadeloupian (GUA), Namibian (NAM) or benzimidazole susceptible Australian (AUS.2) and Zaire (ZAI) isolates. Mean expected Tajima’s <em>D</em> (solid grey line) and 99% confidence interval (dotted line) were estimated from a 1,000 simulated 10-Kbp wide sequences following MSMC-inferred demography.</p><div class="sb-div caption-clear"></div></div></div><p id="p-25">This signature was most evident in the GUA and NAM isolates, but was weaker in the French isolate (FRA.1) due to both phenotypically susceptible and resistant individuals being present (Supplementary Table 4). Phased genotypic information over the whole <em>Hco-tbb-iso-1</em> (Supplementary Fig. 10) revealed that French and Guadeloupian individuals had little divergence in their haplotypes, suggesting that gene flow of resistant haplotypes between mainland France and the West Indies had occurred (Supplementary Fig. 10). A topology analysis of a 100 Kbp region spanning the <em>Hco-tbb-iso-1</em> locus supported the shared phylogenetic origin between FRA.1 and FRG (topology 3), whereas the surrounding region was in favour of topologies congruent with overall population structure (<a id="xref-fig-8-1" class="xref-fig" href="#F8">Fig. 3b</a>). This finding is consistent when either Moroccan (<a id="xref-fig-8-2" class="xref-fig" href="#F8">Fig. 3b</a>) or São Tomé populations are used as a population with close ancestry with FRG (Supplementary Fig. 11).</p><div id="F8" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/04/13/450692/F8.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Topology weighing analysis of a 100-Kbp window centred on Hco-tbb-iso-1. At each position, the weight of each of the three possible topologies inferred from 50 Kbp-windows are overlaid. Topology 2 (blue) corresponds to an isolation-by-distance history, while topology 3 (brown) would agree with shared introgressed material between worm populations from French mainland into Guadeloupe. The Hco-tbb-iso-1 locus is indicated by vertical dashed lines." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-691381162" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Topology weighing analysis of a 100-Kbp window centred on &lt;em&gt;Hco-tbb-iso-1&lt;/em&gt;. At each position, the weight of each of the three possible topologies inferred from 50 Kbp-windows are overlaid. Topology 2 (blue) corresponds to an isolation-by-distance history, while topology 3 (brown) would agree with shared introgressed material between worm populations from French mainland into Guadeloupe. The &lt;em&gt;Hco-tbb-iso-1&lt;/em&gt; locus is indicated by vertical dashed lines.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 3b." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2019/04/13/450692/F8.medium.gif" width="440" height="302"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 3b." src="https://www.biorxiv.org/content/biorxiv/early/2019/04/13/450692/F8.medium.gif" width="440" height="302"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/04/13/450692/F8.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 3b." data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/04/13/450692/F8.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 3b.</span> <p id="p-26" class="first-child">Topology weighing analysis of a 100-Kbp window centred on <em>Hco-tbb-iso-1</em>. At each position, the weight of each of the three possible topologies inferred from 50 Kbp-windows are overlaid. Topology 2 (blue) corresponds to an isolation-by-distance history, while topology 3 (brown) would agree with shared introgressed material between worm populations from French mainland into Guadeloupe. The <em>Hco-tbb-iso-1</em> locus is indicated by vertical dashed lines.</p><div class="sb-div caption-clear"></div></div></div><p id="p-27">We analysed the frequency of the three well-characterised resistance-associated mutations affecting codon positions 167 (T to A), 198 (A to T) and 200 (T to A) (Supplementary Table 4; Supplementary Table 5; Supplementary Fig. 12 &amp; 13). The F200Y homozygous genotype was the most common and widespread resistant genotype (n = 39), and accounted for all samples from the Guadeloupe (n = 14) and the White-River South-African (STA.3; n = 6) isolates. Variants at codons 167 and 198 were much less common, i.e. 13 mutant allelic carriers were observed at position 198 and only one F167Y mutant was identified in a French isolate. No double homozygous mutants were found, but three individuals from France, Australia and South-Africa were heterozygous at both positions 198 and 200. In each case, inspection of the sequencing reads revealed that the two mutations never appeared together in the same sequencing read, suggesting they cannot co-occur in <em>cis</em> (on the same chromosome copy). Genotype frequencies were consistent with isolate-level benzimidazole efficacies as measured by the percentage of egg excretion after treatment (Supplementary Table 5). Samples analysed from suspected susceptible isolates always presented with the susceptible genotype for each of the three positions considered (n = 20).</p><p id="p-28">The genetic determinants of ivermectin resistance are largely unknown, but many genes have been proposed to be associated with resistance; although one interpretation of this observation is that ivermectin resistance is a multigenic trait, a major locus associated with resistance in Australian and South African <em>H. contortus</em> has recently been mapped to a region approximately 37-40 Mbp along chromosome V <sup><a id="xref-ref-12-2" class="xref-bibr" href="#ref-12">12</a></sup>. To examine for the presence of ivermectin-mediated selection in our data, a pairwise differentiation scan was performed between known ivermectin-resistant isolates and other isolates sharing same genetic ancestry, <em>i.e</em>. STA.1 and STA.3 versus NAM and ZAI in Africa, AUS.1 against AUS.2 in Australia (<a id="xref-fig-9-1" class="xref-fig" href="#F9">Fig. 3c</a>).</p><div id="F9" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/04/13/450692/F9.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Genome-wide differentiation scan between pairs of ivermectin-resistant (AUS.1, STA.1, STA.3) and populations with no evidence of ivermectin resistance. Chromosomes are coloured and ordered by name, i.e. from I (pink) to V (forest green). Horizontal line represents the 0.5% FST quantile cut-off. Vertical line on chromosome I points at Hco-tbb-iso-1 locus." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-691381162" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Genome-wide differentiation scan between pairs of ivermectin-resistant (AUS.1, STA.1, STA.3) and populations with no evidence of ivermectin resistance. Chromosomes are coloured and ordered by name, &lt;em&gt;i.e&lt;/em&gt;. from I (pink) to V (forest green). Horizontal line represents the 0.5% F&lt;sub&gt;ST&lt;/sub&gt; quantile cut-off. Vertical line on chromosome I points at &lt;em&gt;Hco-tbb-iso-1&lt;/em&gt; locus.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 3c." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2019/04/13/450692/F9.medium.gif" width="440" height="401"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 3c." src="https://www.biorxiv.org/content/biorxiv/early/2019/04/13/450692/F9.medium.gif" width="440" height="401"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/04/13/450692/F9.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 3c." data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/04/13/450692/F9.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 3c.</span> <p id="p-29" class="first-child">Genome-wide differentiation scan between pairs of ivermectin-resistant (AUS.1, STA.1, STA.3) and populations with no evidence of ivermectin resistance. Chromosomes are coloured and ordered by name, <em>i.e</em>. from I (pink) to V (forest green). Horizontal line represents the 0.5% F<sub>ST</sub> quantile cut-off. Vertical line on chromosome I points at <em>Hco-tbb-iso-1</em> locus.</p><div class="sb-div caption-clear"></div></div></div><p id="p-30">The previously described region on chromosome V<sup><a id="xref-ref-12-3" class="xref-bibr" href="#ref-12">12</a></sup> appeared differentiated, particularly between pairwise comparisons involving the South-African resistant isolate (<a id="xref-fig-9-2" class="xref-fig" href="#F9">Fig. 3c</a>). A second major differentiation hotspot spanned a 3 Mbp region of chromosome I and encompassed the <em>Hco-tbb-iso-1</em> locus (<a id="xref-fig-9-3" class="xref-fig" href="#F9">Fig. 3c</a>). Although non-synonymous mutations at codon positions 167, 198 and 200 of the β-tubulin isotype 1 are associated with resistance to benzimidazoles, it has been proposed that there may also be an association between this gene and ivermectin resistance<sup><a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">25</a>,<a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">26</a></sup>. Although our data seem to support this association, an attempt to narrow-down the differentiation signal in this region found significant differentiation in only two 10-kbp windows in two comparisons (NAM vs STA.1 and NAM vs. STA.3, and these two windows did not overlap the <em>Hco-tbb-iso-1</em> locus. Furthermore, the fact that all of our ivermectin-resistant isolates were also benzimidazole-resistant suggests that this signal is confounded; pairwise F<sub>ST</sub> analyses between ivermectin-resistant and – susceptible isolates from the field will always be biased toward strong differentiation around the <em>Hco-tbb-iso-1</em> locus due to loss of diversity in benzimidazole-resistant isolates. As such, it was not possible to confirm the putative association between the <em>Hco-tbb-iso-1 locus</em> and ivermectin resistance. We note that the lack of QTL evidence in this region from controlled genetic crosses using ivermectin selection<sup><a id="xref-ref-12-4" class="xref-bibr" href="#ref-12">12</a></sup> supports the conclusion that standing genetic variation at the <em>Hco-tbb-iso-1locus</em> is unlikely to be directly influenced by ivermectin.</p><p id="p-31">To further investigate more subtle signatures of selection, we computed the XP-CLR coefficient, which simultaneously exploits within-isolate departure from neutrality and between-isolate allele-frequency differences <sup><a id="xref-ref-27-1" class="xref-bibr" href="#ref-27">27</a></sup>. This analysis relies on called genotypes rather than genotype likelihoods, but is robust to SNP uncertainty<sup><a id="xref-ref-27-2" class="xref-bibr" href="#ref-27">27</a></sup>. Within continent pairwise comparisons of African (NAM, MOR, STA.1, STA.3, ZAI) and Australian isolates (AUS.1, AUS.2) yielded 1,740 hotspots of diversification, 48% of which were contained within a gene locus (Supplementary Fig. 14, Supplementary Table 6). Among these hotspots, two known candidate genes associated with ivermectin resistance were identified, namely an ivermectin sensitive glutamate-gated chloride channel<sup><a id="xref-ref-28-1" class="xref-bibr" href="#ref-28">28</a></sup> <em>(HCOI00617300, glc-4</em> ortholog) on chromosome and a P-glycoprotein coding gene already involved in ivermectin susceptibility in the equine ascarid <em>Parascaris sp</em> <sup><a id="xref-ref-29-1" class="xref-bibr" href="#ref-29">29</a></sup>, (<em>HCOI00233200, pgp-11</em> ortholog) on chromosome V. While the former showed indication of reduced genetic diversity in its vicinity in resistant isolates relative to others (0.48% ± 0.09% difference in nucleotide diversity between the two groups, <em>t</em> = 5.25, <em>P</em> &lt; 10<sup>−4</sup>; <a id="xref-fig-10-1" class="xref-fig" href="#F10">Fig. 3d</a>), the genetic diversity pattern in the 100 Kbp window surrounding the latter was similar across isolates (t = 0.004, <em>P</em> = 0.99; <a id="xref-fig-10-2" class="xref-fig" href="#F10">Fig. 3d</a>), suggesting its role may not be specifically related to ivermectin resistance.</p><div id="F10" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/04/13/450692/F10.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Nucleotide diversity over 100 Kbp windows surrounding positional candidate genes for ivermectin resistance. Candidate gene position is highlighted by vertical dashed lines. Ivermectin-resistant isolates (IVM-R) appear as triangles and squares indicate susceptible isolates (IVM-S)." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-691381162" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Nucleotide diversity over 100 Kbp windows surrounding positional candidate genes for ivermectin resistance. Candidate gene position is highlighted by vertical dashed lines. Ivermectin-resistant isolates (IVM-R) appear as triangles and squares indicate susceptible isolates (IVM-S).&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 3d." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2019/04/13/450692/F10.medium.gif" width="431" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 3d." src="https://www.biorxiv.org/content/biorxiv/early/2019/04/13/450692/F10.medium.gif" width="431" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/04/13/450692/F10.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 3d." data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/04/13/450692/F10.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 3d.</span> <p id="p-32" class="first-child">Nucleotide diversity over 100 Kbp windows surrounding positional candidate genes for ivermectin resistance. Candidate gene position is highlighted by vertical dashed lines. Ivermectin-resistant isolates (IVM-R) appear as triangles and squares indicate susceptible isolates (IVM-S).</p><div class="sb-div caption-clear"></div></div></div><p id="p-33">GO term enrichment analysis of all XP-CLR significant regions identified 26 significant terms (Supplementary Table 7). The top ten most significant GO terms encompassed enzymatic-related activity, neurotransmitter transporter activity (GO:0005326, <em>P</em> = 5.8 × 10<sup>−4</sup>) and response to drug (GO:0042493, <em>P</em> = 4.3 − 10<sup>−3</sup>). Additional significant biological process associated terms were related to phenotypes tightly linked to anthelmintic effects. For example, pharyngeal pumping<sup><a id="xref-ref-30-1" class="xref-bibr" href="#ref-30">30</a></sup> (GO:0043051, <em>P</em> = 5.8 × 10<sup>−3</sup>) and oviposition (GO:0046662, <em>P</em> = 8.6 × 10<sup>−3</sup>) genes were enriched, both of which are phenotypes linked to ivermectin and its effect on parasite fecundity<sup><a id="xref-ref-31-1" class="xref-bibr" href="#ref-31">31</a></sup>. Neurotransmission is the primary target of anthelmintics such as macrocyclic lactones<sup><a id="xref-ref-32-1" class="xref-bibr" href="#ref-32">32</a></sup> and levamisole<sup><a id="xref-ref-33-1" class="xref-bibr" href="#ref-33">33</a></sup>; genes with GO terms related to neurotransmission (GO:0001505, <em>P</em> = 9.1 × 10<sup>−3</sup>) were significantly enriched across comparisons. Anthelmintic-associated GO terms including “response to drug”, “regulation of neurotransmitter” and “neurotransmitter transporter activity” were significantly enriched in every comparison involving a South-African multi-resistant field isolates (STA.1, Supplementary Table 8). Four candidate genes (<em>HCOI00389600, HCOI00032800, HCOI00243900, HCOI00489500)</em> were found overlapping XP-CLR signals of selection and thus seem candidates for drug resistance loci in <em>H. contortus</em>, as supported by the functions of their <em>C. elegans</em> orthologs (<em>snf-9, unc-24, B0361.4, aex-3</em> respectively). The <em>snf-9</em> gene encodes a neurotransmitter:sodium symporter belonging to a family of proteins involved in neurotransmitter reuptake, and the latter three genes are expressed in neurons. <em>Unc-24</em> and <em>B0361.4</em> are involved in response to lipophilic compounds and <em>aex-3</em> is critical in synaptic vesicle release<sup><a id="xref-ref-34-2" class="xref-bibr" href="#ref-34">34</a></sup>. A reduction in nucleotide diversity surrounding <em>unc-24, B0361.4, aex-3</em> in the STA.1 isolate in comparison to others (<a id="xref-fig-10-3" class="xref-fig" href="#F10">Fig. 3d</a>) may reflect evidence of drug selection.</p></div><div id="sec-6" class="subsection"><h3>Climatic adaptation has shaped genomic variation between isolates</h3><p id="p-34">In addition to putative anthelmintic-related GO terms, response to stress (GO:0006950, <em>P</em> = 5.9 × 10<sup>−3</sup>) was among the top ten significant biological process ontologies associated with genes under diversifying selection. Although anthelmintic exposure would be associated with significant stress on susceptible (and perhaps resistant or tolerant) parasites, all free-living stages of <em>H. contortus</em> will be exposed to and must tolerate abiotic factors such as temperature or humidity prior to infection of a new host. To evaluate the impact of such climatic stressors, a genome scan for genetic differentiation between isolates categorised by by the climactic conditions prevailing at their sampling locations, i.e. arid (Namibia), temperate (France mainland) and tropical (Guadeloupe), was performed (<a id="xref-fig-11-1" class="xref-fig" href="#F11">Fig. 4a</a>).</p><div id="F11" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/04/13/450692/F11.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Genomic signal of climate adaptation Top three panels represent FST values plotted against genomic position (Mbp), colored and ordered by chromosome name (from I to V). Horizontal dashed line represents the 0.5% quantile. Bottom panel shows genomic positions of significant associations between annual precipitation (circle) and temperature annual range (triangle)." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-691381162" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;&lt;span class=&quot;caption-title&quot;&gt;Genomic signal of climate adaptation&lt;/span&gt; Top three panels represent F&lt;sub&gt;ST&lt;/sub&gt; values plotted against genomic position (Mbp), colored and ordered by chromosome name (from I to V). Horizontal dashed line represents the 0.5% quantile. Bottom panel shows genomic positions of significant associations between annual precipitation (circle) and temperature annual range (triangle).&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 4a." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2019/04/13/450692/F11.medium.gif" width="440" height="243"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 4a." src="https://www.biorxiv.org/content/biorxiv/early/2019/04/13/450692/F11.medium.gif" width="440" height="243"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/04/13/450692/F11.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 4a." data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/04/13/450692/F11.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 4a.</span> <span class="caption-title">Genomic signal of climate adaptation</span><p id="p-35" class="first-child">Top three panels represent F<sub>ST</sub> values plotted against genomic position (Mbp), colored and ordered by chromosome name (from I to V). Horizontal dashed line represents the 0.5% quantile. Bottom panel shows genomic positions of significant associations between annual precipitation (circle) and temperature annual range (triangle).</p><div class="sb-div caption-clear"></div></div></div><p id="p-36">A major signal of differentiation formed of two windows (6.925 – 6.995 Mbp and 7.165 – 7.205 Mbp on chromosome I) was shared by all pairwise comparisons. Six genes were found within these two regions (Supplementary Table 9), among which orthologs of <em>C. elegans</em> genes <em>hprt-1, cpb-1, B0205.4</em> were identified. A highly differentiated 260 Kbp region of chromosome III also repeatedly occurred across comparisons. A window between 31.155 Mbp and 31.185 Mbp was common to comparisons involving isolates from arid areas (NAM vs. FRA.1 and NAM vs. GUA), and a second window (31.345 Mbp to 31.415 Mbp) was common to comparisons between tropical isolates and others (GUA <em>vs</em>. FRA.1 and GUA <em>vs</em>. NAM). While the two annotated genes in the latter region were not associated with any biological description, the first window overlapped a chromo-domain containing protein (<em>HCOI01540500</em>; Supplementary Table 89). This gene is an ortholog of <em>Pc</em> (<em>FBgn0003042)</em> in <em>D. melanogaster</em>, a chromo-domain subunit of the Polycomb PRC1 -complex that specifically recognizes trimethylated lysine7 of histone 3 (H3K27me3)<sup><a id="xref-ref-35-1" class="xref-bibr" href="#ref-35">35</a></sup>. Nucleotide diversity over these genomic windows revealed reduced genetic diversity in FRA.1 relative to isolates from hotter climates (<a id="xref-fig-12-1" class="xref-fig" href="#F12">Fig. 4b</a>). The contrasted diversity downstream from <em>HCOI01540500</em> (31.225 Kbp to 31.350 Kbp, <a id="xref-fig-12-2" class="xref-fig" href="#F12">Fig. 4b</a>) found between GUA and NAM certainly contributed to higher F<sub>ST</sub> in these windows. Additional differentiation analyses performed at the gene level with base-pair resolution highlighted a few discrete locations of elevated F<sub>ST</sub> common to every comparison and overlapping intron 4 and exon 5 of this gene (<a id="xref-fig-12-3" class="xref-fig" href="#F12">Fig. 4b</a>). Translated consensus exon 5 sequences revealed the highest divergence in the GUA isolate (92.1% identity with reference sequence; supplementary Fig. 15), characterized by multiple amino acid changes whose putative functional consequences remain unknown.</p><div id="F12" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/04/13/450692/F12.large.jpg?width=800&amp;height=600&amp;carousel=1" title="A Polycomb group protein coding gene underpins major differentiation signal between populations from arid, temperate and tropical areas. Top panel shows nucleotide diversity estimates for 10 Kbp genomic windows spanning windows of chromosome III with high genetic differentiation for climate. Dashed lines indicate the Pc ortholog boundaries. Within these boundaries, FST estimates along every base-pair of the gene sequence are represented below (circle size proportional to FST coefficient), with predicted exon model shown as grey rectangles. The bottom panel shows translated consensus sequences of exon 5 for populations of interest with asterisks marking mutations." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-691381162" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;&lt;span class=&quot;caption-title&quot;&gt;A &lt;em&gt;Polycomb group protein coding&lt;/em&gt; gene underpins major differentiation signal between populations from arid, temperate and tropical areas.&lt;/span&gt; Top panel shows nucleotide diversity estimates for 10 Kbp genomic windows spanning windows of chromosome III with high genetic differentiation for climate. Dashed lines indicate the &lt;em&gt;Pc&lt;/em&gt; ortholog boundaries. Within these boundaries, F&lt;sub&gt;ST&lt;/sub&gt; estimates along every base-pair of the gene sequence are represented below (circle size proportional to F&lt;sub&gt;ST&lt;/sub&gt; coefficient), with predicted exon model shown as grey rectangles. The bottom panel shows translated consensus sequences of exon 5 for populations of interest with asterisks marking mutations.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 4b." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2019/04/13/450692/F12.medium.gif" width="440" height="324"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 4b." src="https://www.biorxiv.org/content/biorxiv/early/2019/04/13/450692/F12.medium.gif" width="440" height="324"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/04/13/450692/F12.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 4b." data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/04/13/450692/F12.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 4b.</span> <span class="caption-title">A <em>Polycomb group protein coding</em> gene underpins major differentiation signal between populations from arid, temperate and tropical areas.</span><p id="p-37" class="first-child">Top panel shows nucleotide diversity estimates for 10 Kbp genomic windows spanning windows of chromosome III with high genetic differentiation for climate. Dashed lines indicate the <em>Pc</em> ortholog boundaries. Within these boundaries, F<sub>ST</sub> estimates along every base-pair of the gene sequence are represented below (circle size proportional to F<sub>ST</sub> coefficient), with predicted exon model shown as grey rectangles. The bottom panel shows translated consensus sequences of exon 5 for populations of interest with asterisks marking mutations.</p><div class="sb-div caption-clear"></div></div></div><p id="p-38">To further explore the impact of climatic conditions of genetic diversity, we used a random forest based statistical approach to quantify the relationship between genetic information along environmental gradients, and relative impact of specific bioclimatic variables on genetic diversity. Bioclimatic variables were derived from monthly temperature and precipitation records, with the aim to represent annual trends or seasonality over the 1970 to 2000 time period (Supplementary Fig. 16; Supplementary Table 10). This analysis randomly samples subsets of sites (encoded as SNP frequencies) that can be partitioned into groups based on differences in climactic variables to estimate the predictive ability of climatic variables. Annual precipitation (BIO12), and temperature annual range (BIO7) were revealed to be the most important bioclimatic variables impacting genetic variation (<a id="xref-fig-13-1" class="xref-fig" href="#F13">Fig. 4c</a>).</p><div id="F13" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/04/13/450692/F13.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Proportion of genetic variance explained by precipitation- (blue) and temperature-associated (orange) climatic variables. Variants from eight isolates with no record of ivermectin resistance (AUS.2, FRA.1, GUA, IND, MOR, NAM, STO, ZAI) were analysed against a set of nine variables (from a total of 19), selected to minimize correlation between variables. Annual precipitation (BIO12) and temperature annual range (BIO7) are main contributors of genetic variance." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-691381162" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Proportion of genetic variance explained by precipitation- (blue) and temperature-associated (orange) climatic variables. Variants from eight isolates with no record of ivermectin resistance (AUS.2, FRA.1, GUA, IND, MOR, NAM, STO, ZAI) were analysed against a set of nine variables (from a total of 19), selected to minimize correlation between variables. Annual precipitation (BIO12) and temperature annual range (BIO7) are main contributors of genetic variance.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 4c." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2019/04/13/450692/F13.medium.gif" width="440" height="390"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 4c." src="https://www.biorxiv.org/content/biorxiv/early/2019/04/13/450692/F13.medium.gif" width="440" height="390"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/04/13/450692/F13.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 4c." data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/04/13/450692/F13.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 4c.</span> <p id="p-39" class="first-child">Proportion of genetic variance explained by precipitation- (blue) and temperature-associated (orange) climatic variables. Variants from eight isolates with no record of ivermectin resistance (AUS.2, FRA.1, GUA, IND, MOR, NAM, STO, ZAI) were analysed against a set of nine variables (from a total of 19), selected to minimize correlation between variables. Annual precipitation (BIO12) and temperature annual range (BIO7) are main contributors of genetic variance.</p><div class="sb-div caption-clear"></div></div></div><p id="p-40">To identify genes that might be impacted by these variables, we performed a genome-wide test for association between SNP variants, and each of these two environmental variables, accounting for genetic structure between isolates. In total, 17 and 25 significant associations (5% FDR; 49,370 SNPs tested) were found with BIO7 and BIO12 respectively (<a id="xref-fig-11-2" class="xref-fig" href="#F11">Fig. 4a</a> bottom panel; Supplementary Table 11). Consistent with the initial differentiation scan, chromosomes I and III harboured most of the associations (n = 13 and n = 11, respectively). On chromosome I, eight (of the 13) associations with BIO12 fell within a 6 Kbp window (7,177,538 bp and 7,183,617 bp), overlapping the region universally differentiated between climatic areas. Chromosome III contained 11 significant associations for both BIO7 (n = 6) and BIO12 (n = 5). The two most significant associations were also found on this chromosome for BIO7 (positions 26,824,240 and 26,824,197 bp; <em>P</em> = 6.34 × 10<sup>−9</sup> and <em>P</em> = 1.39 × 10<sup>−7</sup>, respectively). These two SNPs fell within the <em>HCOI00198200</em> locus, which codes for a metallopeptidase M1, an ortholog of the <em>C. elegans</em> gene <em>anp-1</em>. Additional BIO7-associated SNPs were found within (16,544,889 bp) and in the vicinity (15,905,141 &amp; 15,905,191 bp) of a solute carrier coding gene (<em>HCOI00312500)</em> on chromosome IV, whose orthologs in <em>C. elegans</em> are under the control of <em>daf-12</em>, a key player in dauer formation. Of note, two <em>H. contortus</em> orthologs of components of the dauer pathway in <em>C. elegans</em>, namely <em>tax-4</em> (<em>HCOI00661100)</em> and <em>daf-36</em> (<em>HCOI00015800)</em>, were among genes under diversifying selection (Supplementary Fig. 14), but neither of these genes contained SNPs identified as being associated with the two most significant bioclimatic variables.</p></div></div><div class="section" id="sec-7"><h2 class="">Discussion</h2><p id="p-41">The ecology and epidemiology of gastrointestinal nematodes have been well characterized and exploited to build mathematical models to guide treatment decision in the field<sup><a id="xref-ref-36-1" class="xref-bibr" href="#ref-36">36</a></sup>. On the contrary, knowledge about their genetic diversity remains limited; a better understanding of population structure and selective pressure applied by environmental factors would yield better predictive power of their range dispersion<sup><a id="xref-ref-37-1" class="xref-bibr" href="#ref-37">37</a></sup>. By exploiting a broad collection of samples from globally-distributed isolates, together with chromosome-scale assembly and extensive individual resequencing, we have performed an in-depth characterization of <em>H. contortus</em> genetic diversity, explored historical contributions to its current population structure and identified important drivers shaping the genome of this parasite.</p><p id="p-42"><em>H. contortus</em> populations displayed high levels of nucleotide diversity, consistent with early estimates based on mitochondrial data<sup><a id="xref-ref-39-1" class="xref-bibr" href="#ref-39">39</a>,<a id="xref-ref-40-1" class="xref-bibr" href="#ref-40">40</a></sup>, and recent re-sequencing experiments of inbred isolates<sup><a id="xref-ref-12-5" class="xref-bibr" href="#ref-12">12</a></sup>. These extreme levels of genetic diversity are thought to arise from both a large census population<sup><a id="xref-ref-42-1" class="xref-bibr" href="#ref-42">42</a></sup> and the high fecundity of <em>H. contortus</em> females<sup><a id="xref-ref-21-2" class="xref-bibr" href="#ref-21">21</a></sup>.</p><p id="p-43">An early attempt based on a set of genome-wide AFLP markers obtained for 150 individual worms from 14 countries supported the first exploration of <em>H. contortus</em> population structure at the continent level <sup><a id="xref-ref-40-2" class="xref-bibr" href="#ref-40">40</a></sup>. Analyses of these data identified three to four (Africa, South-East Asia, America and Europe) main phylogenetic clusters as well as evidence for the strong genetic connectivity between Australian, South-African and European isolates<sup><a id="xref-ref-40-3" class="xref-bibr" href="#ref-40">40</a></sup>. Our genome wide data corroborated these early results. However, the use of a chromosome-scale assembly and individual resequencing contributed to identify genome-wide patterns of genetic diversity in its chromosomal context, and in turn, provided sufficient resolution to identify genes likely associated with selective advantage against drug or climate selection pressures. This had not been possible in previous attempts with AFLP markers <sup><a id="xref-ref-40-4" class="xref-bibr" href="#ref-40">40</a></sup>.</p><p id="p-44">Major past human migrations and associated sheep movements have contributed to the mixing of parasite populations, which partly accounts for the limited genetic structure and extensive admixture between some of our globally distributed isolates. Our data supported an out of Africa expansion derived from ancestral populations from Western Africa, with a bottleneck dated back between 2.5 to 10 kya. Sheep domestication originally took place in the Middle East around 10 kya<sup><a id="xref-ref-43-1" class="xref-bibr" href="#ref-43">43</a></sup>, before introduction into Eastern Africa and subsequent spread, likely through cultural diffusion, toward southern Africa approximately 2 to 2.5 kya <sup><a id="xref-ref-44-1" class="xref-bibr" href="#ref-44">44</a></sup>. The timing of the bottleneck identified in our data and occurring between 2.5 and 10 kya is compatible with major migrations of pastoralist populations that ultimately resulted in the import of small ruminants into Southern Africa <sup><a id="xref-ref-44-2" class="xref-bibr" href="#ref-44">44</a></sup>. The simultaneous increase in rainfall in central Africa around 10.5 kya<sup><a id="xref-ref-45-1" class="xref-bibr" href="#ref-45">45</a></sup> would have supported the population expansion and dispersal of <em>H. contortus</em>. In addition, early radiation towards Asia observed in our data was congruent with evidence obtained from the study of retroviral insertions within the sheep genome that suggested direct migration between Africa and southwest Asia<sup><a id="xref-ref-46-1" class="xref-bibr" href="#ref-46">46</a></sup>, and with the timing of Asian sheep expansion between 1.8 and 14 kya<sup><a id="xref-ref-47-1" class="xref-bibr" href="#ref-47">47</a></sup>.</p><p id="p-45">The genetic congruence of parasites in Guadeloupe with those from both West Africa and the Mediterranean region is consistent with co-transportation of livestock, including sheep, during the discovery and subsequent colonization of America.. Woolly Churra sheep were originally brought to the Caribbean by Spanish conquistadors <sup><a id="xref-ref-49-1" class="xref-bibr" href="#ref-49">49</a>,<a id="xref-ref-50-1" class="xref-bibr" href="#ref-50">50</a></sup>, before West African breeds more suited to tropical conditions were transported, resulting in an admixed Carribean sheep population <sup><a id="xref-ref-49-2" class="xref-bibr" href="#ref-49">49</a>,<a id="xref-ref-50-2" class="xref-bibr" href="#ref-50">50</a></sup>. The timing of genetic admixture here overlaps with centuries of human and presumably livestock movement during the transport of slaves, most of whom originated in West Africa and were transported to colonies that included the French West Indies<sup><a id="xref-ref-22-2" class="xref-bibr" href="#ref-22">22</a></sup>. The Mediterranean ancestry of <em>H. contortus</em> isolates in Guadeloupe and the close relationship with French worm populations suggest additional sheep transport occurred between French mainland and its Guadeloupian overseas territory. Although these movements are difficult to track precisely, live sheep were shipped aboard slave trade vessels departing from French harbours<sup><a id="xref-ref-51-1" class="xref-bibr" href="#ref-51">51</a></sup>, and may have introduced European <em>H. contortus</em> to the island. Based on historical data<sup><a id="xref-ref-49-3" class="xref-bibr" href="#ref-49">49</a>,<a id="xref-ref-50-3" class="xref-bibr" href="#ref-50">50</a></sup>, a Spanish lineage of <em>H. contortus</em> on Guadeloupe would also be expected to exist, though data are missing to confirm this. It can be speculated that the shared ancestry between Guadeloupian and Moroccan worm isolates might result from the introduction of Spanish Churra sheep, as this breed emerged in Spain while the region was under Arab influence between the 8<sup>th</sup> and 13<sup>th</sup> century<sup><a id="xref-ref-49-4" class="xref-bibr" href="#ref-49">49</a>,<a id="xref-ref-52-1" class="xref-bibr" href="#ref-52">52</a></sup>. Introduction of contortus-infected sheep from the Maghreb may have been associated with the spreading of African worms in Spain, and ultimately, Guadeloupe.</p><p id="p-46">The widespread use of anthelmintic drugs has been a major selective force shaping standing genetic variation in the isolates analysed. Adaptation to drug exposure was clearly illustrated in the genetic diversity surrounding the β-tubulin coding gene, a target of benzimidazole drugs; strong loss of diversity was observed at this locus in many distinct isolates phenotypically characterised to be resistant to this class of drug. The co-occurrence of the same resistant haplotypes in geographically disconnected isolates is almost certainly due to the independent evolution of benzimidazole resistance as has been described previously<sup><a id="xref-ref-53-1" class="xref-bibr" href="#ref-53">53</a></sup>; however, some evidence of shared ancestry between French and Guadeloupe resistant individuals emphasises the risk of spreading resistance without careful monitoring of parasite populations during livestock trade. Furthermore, this highlights the risk of evolution of resistance in other parasitic nematodes, for example, parasites of medical interest that are treated with benzimidazoles in mass-drug administration programs<sup><a id="xref-ref-54-1" class="xref-bibr" href="#ref-54">54</a></sup>.</p><p id="p-47">Ivermectin is an important anthelmintic for parasite control in both veterinary and medical settings<sup><a id="xref-ref-55-1" class="xref-bibr" href="#ref-55">55</a></sup>. Worldwide emergence of ivermectin-resistant veterinary parasitic nematodes<sup><a id="xref-ref-5-3" class="xref-bibr" href="#ref-5">5</a></sup> and evidence of reduced efficacy in human filarial nematodes<sup><a id="xref-ref-56-1" class="xref-bibr" href="#ref-56">56</a></sup> underline the urgency and importance of a better understanding of the mechanisms involved<sup><a id="xref-ref-55-2" class="xref-bibr" href="#ref-55">55</a></sup>. Our data identified genes previously associated with ivermectin resistance in parasitic nematodes of either veterinary (<em>pgp-11 in Parascaris sp</em>.<sup><a id="xref-ref-29-2" class="xref-bibr" href="#ref-29">29</a></sup>) or medical (<em>aex-3</em> in <em>Onchocerca volvulus</em><sup><a id="xref-ref-57-1" class="xref-bibr" href="#ref-57">57</a></sup>) interest and uncovered new candidates. Additional validation study will determine whether these could serve as that may be useful markers in the field to monitor drug efficacy.. Of note, support for a major QTL for ivermectin resistance on chromosome V recently identified in an introgression experiment in two resistant isolates<sup><a id="xref-ref-38-1" class="xref-bibr" href="#ref-38">38</a></sup> was also present in our genome-wide data and warrants further investigation.</p><p id="p-48">Our broad sampling throughout the global range of <em>H. contortus</em> has enabled the first analyses into climate-driven adaptation in a parasitic nematode. Understanding how climate shapes parasite genetic variation is of primary importance to foresee consequences of climate change on parasite phenology and range dispersion. Parasite dispersal is largely driven by their hosts but <em>H. contortus</em> free-living stages experience climatic conditions that affect their development<sup><a id="xref-ref-2-2" class="xref-bibr" href="#ref-2">2</a></sup> and constrain their spatio-temporal dispersal<sup><a id="xref-ref-58-1" class="xref-bibr" href="#ref-58">58</a></sup>. Observations in Northern Europe suggest that climate change has already altered <em>H. contortus</em> winter phenology <sup><a id="xref-ref-59-1" class="xref-bibr" href="#ref-59">59</a></sup>. Our results suggest that adaptation toward annual precipitation was mostly under the control of variation on chromosome I, however, no obvious candidate genes could be identified. In addition, the <em>Hco-tbb-iso-1</em> locus was close to the region of interest, and linked variation in allelic frequency at this locus as a result of benzimidazole selection cannot be ruled out. A second region of chromosome III was associated with both temperature- and precipitation-related variables, within which biologically relevant genes could be identified; first, the strongest genetic associations with annual temperature range were found in a metallopeptidase with zinc ion binding function (<em>anp-1</em> ortholog), an enzyme linked to drought stress tolerance in <em>Drosophila</em> <sup><a id="xref-ref-60-1" class="xref-bibr" href="#ref-60">60</a></sup>, and second, an ortholog of <em>Pc</em> displayed strong genetic differentiation between arid- and wetter temperate or tropical environmental conditions. This gene has been linked to putative epigenetic regulation of xeric adaptation in <em>Drosophila melanogaster</em>, where <em>Pc</em> mutants display lower resistance to desiccation stress<sup><a id="xref-ref-61-1" class="xref-bibr" href="#ref-61">61</a></sup>. This finding is also corroborated by observations in plants showing Polycomb-mediated regulation of climate-induced phenotypes<sup><a id="xref-ref-62-1" class="xref-bibr" href="#ref-62">62</a></sup>. Further links to climatic adaptation include identification members of the dauer pathway<sup><a id="xref-ref-63-1" class="xref-bibr" href="#ref-63">63</a></sup>; <em>tax-4</em> and <em>daf-36</em> orthologs were under diversifying selection and a <em>daf-12-respondent</em> solute carrier was associated with annual temperature range. Dauer is a developmentally arrested stage in <em>C. elegans</em> that is triggered by environmental stress and mediates tolerance to unfavourable conditions until better conditions are met<sup><a id="xref-ref-63-2" class="xref-bibr" href="#ref-63">63</a></sup>, and can occur in parasitic nematodes like <em>H. contortus</em> under semi-arid conditions<sup><a id="xref-ref-64-1" class="xref-bibr" href="#ref-64">64</a></sup>. Evidence for climatic adaptation suggests that adaptation in the face of climate change will be constrained by available genetic variability at temperature-selected loci, that may both limit or enable range expansion depending on the region. By a better understanding of the interaction between climatic conditions and phenotypes such as hypobiosis (a temporary developmental arrest during unfavorable conditions), optimisation of treatment timing may be possible to maximise control efficacy.</p><p id="p-49">In summary, our data describes the extensive global and genome-wide diversity of the blood-feeding parasitic nematode <em>H. contortus</em>, and how this diversity has been shaped by adaptation to its environment and to drug exposure. Understanding the mechanism(s) by which parasites adapt to fluctuating environmental conditions both within and outside their hosts will have important implications for field management of parasitic nematodes in both veterinary and medical settings. Further characterisation of these putative strategies, together with genetic covariation of drug resistance genes, should contribute to refining epidemiological models and guide treatment decision-trees for a more sustainable management of worm populations in the face of a changing climate.</p></div><div class="section" id="sec-8"><h2 class="">Legends to Figures</h2><p id="p-50"><strong><em>Figure 1. Global diversity of Haemonchus contortus</em>.</strong></p><p id="p-51">(a) Global distribution of <em>Haemonchus contortus</em> isolates. Isolates sampled are coloured by geographical region (sand: South-America, brown: Western-Africa, dark green: Mediterranean area, black: Subtropical Africa, red: Australia). Shape indicates the anthelmintic resistance status of each isolate to fenbendazole (resistant =squares; susceptible =circles) or resistant to both ivermectin and fenbendazole (triangles).</p><p id="p-52">(b) Principal component analysis based on genotype likelihood inferred from whole genome sequences of 223 individual males (243,012 variants considered). Samples are coloured by geographic region described in panel 1a.</p><p id="p-53">(c) MSMC effective population size across time for 6 isolates. Coloured shaded area represents range of values estimated from a cross-validation procedure with five replicates, computed by omitting one chromosome at a time.</p><p id="p-54"><strong><em>Figure 2. Global connectivity of Haemonchus contortus isolates</em>.</strong></p><p id="p-55">(a) Unrooted maximum likelihood phylogenetic tree constructed using 3,052 SNPs from 223 individual mitochondrial genomes. Circles indicate bootstrap support for each branch, blue if support was higher than 70%, red elsewhere. Branches leading to a sample identifier are coloured by geographic region described in <a id="xref-fig-1-3" class="xref-fig" href="#F1">Figure 1a</a>. Mitogroups are annotated with constitutive sample populations.</p><p id="p-56">(b) Matrix showing pairwise dissimilarity between individuals (upper right) and pairwise F<sub>ST</sub> between isolates (lower left).</p><p id="p-57">(c) Admixture analysis of 223 individuals. A cluster size of K = 3 is presented, determined from sites with minor allele frequency above 5% and call rate higher than 50%. Admixture pattern for other values of K is provided in supplementary Figure 5. Isolates are presented sorted along their longitudinal range, and samples sorted by assignment to the 3 clusters.</p><p id="p-58"><strong><em>Figure 3. Anthelmintic-mediated selection is a major driver of genetic variation</em></strong></p><p id="p-59">(<strong>a</strong>) Analysis of Tajima’s <em>D</em> surrounding the <em>Hco-tbb-iso-1</em> locus on chromosome I. A total of 10-Mbp surrounding was considered (pink), of which 1 Mbp nearest to <em>Hco-tbb-iso-1</em> is highlighted (blue). Isolates compared included benzimidazole-resistant French (FRA.1), Guadeloupian (GUA), Namibian (NAM) or benzimidazole susceptible Australian (AUS.2) and Zaire (ZAI) isolates. Mean expected Tajima’s D (solid grey line) and 99% confidence interval (dotted line) were estimated from a 1,000 simulated 10-Kbp wide sequences following MSMC-inferred demography.</p><p id="p-60">(<strong>b</strong>) Topology weighing analysis of a 100-Kbp window centred on <em>Hco-tbb-iso-1</em>. At each position, the weight of each of the three possible topologies inferred from 50 Kbp-windows are overlaid. Topology 2 (blue) corresponds to an isolation-by-distance history, while topology 3 (brown) would agree with shared introgressed material between worm isolates from French mainland into Guadeloupe. The <em>Hco-tbb-iso-1</em> locus is indicated by vertical dashed lines.</p><p id="p-61">(<strong>c</strong>) Genome-wide differentiation scan between pairs of ivermectin-resistant (AUS.1, STA.1, STA.3) and isolates with no evidence for ivermectin resistance. Chromosomes are coloured and ordered by name, i.e. from I (pink) to V (forest green). Horizontal line represents the 0.5% F<sub>ST</sub> quantile cut-off. Vertical line on chromosome I points at <em>Hco-tbb-iso-1</em> locus.</p><p id="p-62">(<strong>d</strong>) Nucleotide diversity over 100 Kbp windows surrounding positional candidate genes for ivermectin resistance. Candidate gene position is highlighted by vertical dashed lines. Ivermectin-resistant isolates (IVM-R) appear as triangles and squares indicate susceptible isolates (IVM-S).</p><p id="p-63"><strong><em>Figure 4. Genomic signal of climate adaptation</em></strong></p><p id="p-64">(<strong>a</strong>) Top three panels represent F<sub>ST</sub> values plotted against genomic position (Mbp), colored and ordered by chromosome name (from I to V). Horizontal dashed line represents the 0.5% quantile. Bottom panel shows genomic positions of significant associations between annual precipitation (circle) and temperature annual range (triangle).</p><p id="p-65">(<strong>b</strong>) A <em>Polycomb group protein coding</em> gene underpins major differentiation signal between populations from arid, temperate and tropical areas. Top panel shows nucleotide diversity estimates for 10 Kbp genomic windows spanning windows of chromosome III with high genetic differentiation for climate. Dashed lines indicate the <em>Pc</em> ortholog boundaries. Within these boundaries, F<sub>ST</sub> estimates along every base-pair of the gene sequence are represented below (circle size proportional to F<sub>ST</sub> coefficient), with predicted exon model shown as grey rectangles. The bottom panel shows translated consensus sequences of exon 5 for populations of interest with asterisks marking mutations.</p><p id="p-66">(<strong>c</strong>) Proportion of genetic variance explained by precipitation- (blue) and temperature-associated (orange) climatic variables. Variants from eight isolates with no record of ivermectin resistance (AUS.2, FRA.1, GUA, IND, MOR, NAM, STO, ZAI) were analysed against a set of nine variables (from a total of 19), selected to minimize correlation between variables. Annual precipitation (BIO12) and temperature annual range (BIO7) are main contributors of genetic variance.</p></div><div class="section" id="sec-9"><h2 class="">Materials and methods</h2><div id="sec-10" class="subsection"><h3>Sample DNA extraction and sequencing</h3><p id="p-67">A total of 267 individual male <em>H. contortus</em> were obtained from a collection held at INRA<sup><a id="xref-ref-16-2" class="xref-bibr" href="#ref-16">16</a></sup> (metadata for all samples is presented in detail in Supplementary Table 1). The sampling regime was motivated to delineate the contribution of major evolutionary forces, <em>i.e</em>. migration and selection (drug and climate) but also constrained by the material available in the collection. Because migration was likely to match human history, isolates from western African countries and southern America were selected to address the contribution of slave trade history to the structuring of <em>H. contortus</em> populations; isolates from former colonies of the British Empire (South-Africa, Australia) were sampled to establish the connectivity between worm populations from Europe and these countries. Ivermectin-resistant isolates (AUS.1, STA.1, STA.3) were also retained to evaluate how anthelmintics had shaped <em>H. contortus</em> genomic variability (drug efficacy data have been provided in supplementary Table 4). Isolates were selected based on available material to ensure minimal sample size (n = 9) per isolate and proper allele frequency estimation. Following these criteria, 19 isolates from 12 countries were available (Supplementary Table 1). Note that the second Australian population (AUS.2) was obtained from an Italian laboratory (labelled ITA_NAP, supplementary Table 1). Samples were gathered between 1995 and 2011 (Supplementary Table 1), and stored in liquid nitrogen upon collection. Four isolates were fenbendazole susceptible (<a id="xref-fig-1-4" class="xref-fig" href="#F1">Fig. 1</a> and supplementary table 4; triangles; FRA.2, FRA.4, STO, ZAI).</p><p id="p-68">DNA was extracted with the NucleoSpin Tissue XS kit (Macherey-Nagel GmbH&amp;Co, France) following the manufacturer’s instruction. Sequencing libraries were prepared as previously described<sup><a id="xref-ref-12-6" class="xref-bibr" href="#ref-12">12</a></sup>. DNA libraries were sequenced with 125 bp paired-end reads on an Illumina Hiseq2500 platform using V4 chemistry (Supplementary Table 1). A second round of sequencing was performed with 75 bp paired-end reads to increase the coverage of 43 samples (Supporting Fig. 15). After sequencing, two samples were identified to be heavily contaminated by kraken-0.10.6-a2d113dc8f<sup><a id="xref-ref-65-1" class="xref-bibr" href="#ref-65">65</a></sup> and were discarded. In total, 18 sequencing lanes consisting of 4,152,170,256 reads were sequenced, the raw data of which are archived under the ENA study accession PRJEB9837.</p></div><div id="sec-11" class="subsection"><h3>Sequencing data processing</h3><p id="p-69">Read mapping to both the mitochondrial and nuclear genomes (v3.0, available at <a href="ftp://ngs.sanger.ac.uk/production/pathogens/Haemonchus_contortus">ftp://ngs.sanger.ac.uk/production/pathogens/Haemonchus_contortus</a>) was performed using SMALT (<a href="http://www.sanger.ac.uk/science/tools/smalt-0">http://www.sanger.ac.uk/science/tools/smalt-0</a>) with a median insert size of 500 bp, k-mer length of 13 bp, and a stringency of 90%. For samples that had two or more BAM files (when split across multiple sequencing lanes), the BAM files were merged using samtools v.0.1.19-44428<sup><a id="xref-ref-66-1" class="xref-bibr" href="#ref-66">66</a></sup>, and duplicated reads removed using Picard v.2_14_0 (<a href="https://github.com/broadinstitute/picard">https://github.com/broadinstitute/picard</a>) before performing realignment around indels using Genome Analysis Toolkit (GATK v3.6)<sup><a id="xref-ref-67-1" class="xref-bibr" href="#ref-67">67</a></sup> RealignerTargetCreator. Mean coverage of the mitochondrial and genomic genomes were estimated using GATK DepthOfCoverage, revealing coverage lower than the estimated target coverage (original 8x) for most samples. Individuals with more than 80% of their mitochondrial sequence with at least 15 reads and a mean mitochondrial genome coverage of at least 20x were retained for population genetic inferences (n = 223 individuals).</p></div><div id="sec-12" class="subsection"><h3>Nuclear genome SNP calling</h3><p id="p-70">To call SNPs, we used GATK HaplotypeCaller in GVCF mode, followed by joint genotyping across samples (GenotypeGVCFs) and extraction of variants (SelectVariants), resulting in a total of 30,040,159 unfiltered SNPs across the five autosomes. Sex determination in <em>H. contortus</em> is based on an XX/XO system; as only male worms were sequenced, their hemizygous X chromosome would have revealed limited phylogenetic information relative to the autosomes, and was henceforth excluded from further analysis.</p><p id="p-71">Low coverage sequencing will inadvertently bias allele sampling at heterozygous SNPs, resulting in excess homozygous genotypes particularly if stringent filtering is applied during SNP calling. To circumvent this issue, we applied the GATK Variant Quality Score Recalibration (<a href="https://gatkforums.broadinstitute.org/gatk/discussion/39/variant-quality-score-recalibration-vqsr">https://gatkforums.broadinstitute.org/gatk/discussion/39/variant-quality-score-recalibration-vqsr</a>), which first uses a reference (truth) SNP set to estimate the covariance between called SNP quality score annotations and SNP probabilities, followed by application of these probabilities to the raw SNPs of interest.</p><p id="p-72">The reference “truth” SNP database was generated from the intersection of variants called from samples with at least a mean of 10x coverage (n = 13) using three independent SNP callers: (i) samtools mpileup (-q20 −Q20 −C50 −uD), (ii) Freebayes<sup><a id="xref-ref-68-1" class="xref-bibr" href="#ref-68">68</a></sup> v.9.9.2-13-gad718f9-dirty (--min-mapping-quality 20 --min-alternate-count 5 --no-indels --min-alternate-qsum 40 --pvar 0.0001 --use-mapping-quality --posterior-integration-limits 1,3 --genotype-variant-threshold 4 --use-mapping-quality --site-selection-max-iterations 3 --genotyping-max-iterations 25 --max-complex-gap 3), and (iii) GATK HaplotypeCaller followed by hard filtering (--QD&lt;2, --DP&gt;10000, --FS&gt;60, --MQ&lt;40, --MQRS &lt;-12.5, --RPRS&lt;-8). The three variant call sets were merged (GATK CombineVariants with --genotypeMergeOptions UNIQUIFY), resulting in an intersecting set of 794,606 SNPs (extracted with GATK SelectVariants). The GATK VariantRecalibrator model was trained with this reference SNP database with a 90% prior likelihood, before being subsequently applied to the raw set of SNPs (n = 30,040,159). The estimation was run for several truth sensitivity threshold values ranging from 90 to 99.9%. After visual inspection of the additional number of SNPs by using sensitivity tranche curves (Supplementary Fig. 16a), a 97% sensitivity threshold was applied to the raw SNP set with GATK ApplyRecalibration, resulting in a total set of 23,868,644 SNPs spanning the five autosomes. Variant depth of coverage (DP) and strand bias (FS) were the main drivers of SNP removal (Supplementary Fig. 16b).</p><p id="p-73">Called SNPs were used for particular analyses (<em>N<sub>e</sub></em> trajectory through time, cross-population composite likelihood-ratio) that could not be performed under the probabilistic framework that relied on genotype likelihoods as implemented in ANGSD<sup><a id="xref-ref-69-1" class="xref-bibr" href="#ref-69">69</a></sup> v. 0.919-20-gb988fab (Supplementary note). These data were also used to estimate average differentiation between isolates. However, within isolate diversity and admixture were analysed using ANGSD<sup><a id="xref-ref-69-2" class="xref-bibr" href="#ref-69">69</a></sup> (Supplementary note).</p></div><div id="sec-13" class="subsection"><h3>Mitochondrial DNA data processing</h3><p id="p-74">The mitochondrial genome exhibited an average coverage depth of 322× (ranging from 24× to 5,868×) per sample (Supplementary Table 1). Mitochondrial reads were extracted and filtered from poorly mapped reads using samtools view (-q 20 -f 0×0002 -F 0×0004 -F 0×0008) and from duplicated reads using Picard v.2_14_0 MarkDuplicates (<a href="https://github.com/broadinstitute/picard">https://github.com/broadinstitute/picard</a>). Realignment around indels was applied with GATK<sup><a id="xref-ref-70-1" class="xref-bibr" href="#ref-70">70</a></sup> and SNP were subsequently called using samtools<sup><a id="xref-ref-71-1" class="xref-bibr" href="#ref-71">71</a></sup> mpileup using only reads that achieved a mapping quality of 30 and base quality of 30. The occurrence of heterozygous sites in mtDNA, known as heteroplasmy, has been described across vertebrate species<sup><a id="xref-ref-72-1" class="xref-bibr" href="#ref-72">72</a></sup> and in other nematode species, including <em>C. briggsae</em><sup><a id="xref-ref-73-1" class="xref-bibr" href="#ref-73">73</a></sup> and <em>C. elegans</em><sup><a id="xref-ref-74-1" class="xref-bibr" href="#ref-74">74</a></sup>. However, heterozygous sites may also occur as technical artefacts as a result of genetically similar sequences shared between the nuclear and mitochondrial genomes, <em>i.e</em>., numts<sup><a id="xref-ref-75-1" class="xref-bibr" href="#ref-75">75</a></sup>. To exclude sites prone to heterozygous signals from further phylogenetic inference analysis, a SNP calling procedure was implemented with the HaplotypeCaller tool to apply hard filtering parameters on the raw SNP sets (QD&gt;=10, FS&lt;=35 MQ&gt;=30 MQRankSum&gt;=−12.5 and ReadPosRankSum &gt;=−8) with a minimum depth of 20 reads. This procedure excluded 1,354 putative heterozygous sites, and retained 72% of the putative SNP sites (3,052 out of 4,234 SNPs). Nucleotide diversity and Tajima’s <em>D</em> were computed by sliding-windows of 100 bp using vcftools v.0.1.15<sup><a id="xref-ref-76-1" class="xref-bibr" href="#ref-76">76</a></sup>. A principal component analysis (PCA) was performed on genotypes using the SNPrelate package<sup><a id="xref-ref-77-1" class="xref-bibr" href="#ref-77">77</a></sup> in R version 3.5<sup><a id="xref-ref-78-1" class="xref-bibr" href="#ref-78">78</a></sup>. A consensus fasta sequence was subsequently generated with GATK FastaAlternateReferenceMaker for each sample using the filtered variant set, which was used for the phylogenetic analyses.</p></div><div id="sec-14" class="subsection"><h3>Diversity and divergence analysis</h3><p id="p-75">Genome-wide nucleotide diversity (π) was computed for each isolate with at least five individuals using ANGSD<sup><a id="xref-ref-69-3" class="xref-bibr" href="#ref-69">69</a></sup>. Using genotype likelihoods (GLs) from samtools<sup><a id="xref-ref-71-2" class="xref-bibr" href="#ref-71">71</a></sup> (option GL=1) as an input, variants were included that had a minimal supporting evidence of 5 reads, and base and mapping quality phred scores of at least 20. As π values were biased by population mean coverage (Supplementary Fig. 17), π was also calculated from a subset of isolates containing individuals with a minimum mean coverage of 5×: this was limited to France (FRA.1, n = 5, mean coverage of 7.66×), Guadeloupe (n = 5, mean coverage of 12.75×) and Namibia (n = 6, mean coverage of 9.85×).</p><p id="p-76">F<sub>ST</sub> was estimated from the VQSR-called genotypes between isolates with at least five individuals using the Weir-Cockerham estimator<sup><a id="xref-ref-79-1" class="xref-bibr" href="#ref-79">79</a></sup> in vcftools v0.1.15<sup><a id="xref-ref-76-2" class="xref-bibr" href="#ref-76">76</a></sup>. To prevent artifactual signal linked to variation in coverage between isolates, F<sub>ST</sub> was calculated on subsets of SNPs, binned based on their minor allele frequency (MAF) in 10% increments. The maximum F<sub>ST</sub> value calculated was retained for comparison (Supplementary Fig. 4) and the maximal value was considered as reported elsewhere<sup><a id="xref-ref-80-1" class="xref-bibr" href="#ref-80">80</a></sup>. The resulting F<sub>ST</sub> estimates were not biased by coverage, as measured by negligible correlation between pairwise F<sub>ST</sub> coefficients and associated population cross-coverage (Pearson’s <em>r<sub>(136)</sub></em> = −0.05, <em>P</em> = 0.55; Supplementary Fig. 18).</p><p id="p-77">The pairwise sequence divergence between individual samples was calculated using the Hamming distance, i.e. 1-IBS, using PLINK<sup><a id="xref-ref-81-1" class="xref-bibr" href="#ref-81">81</a></sup>, considering the only individuals with a mean coverage above 2.5× as failure to do so yielded biased estimates (Supplementary note, Supplementary Fig. 19). A neighbour-joining tree of Hamming distances calculated from the nuclear DNA genotypes was built using the R package <em>ape</em><sup><a id="xref-ref-82-1" class="xref-bibr" href="#ref-82">82</a></sup>.</p><p id="p-78">PCA on genotypes inferred from genotype likelihoods of the 223 samples was generated using ANGSD ngsCovar, filtering for sites with base and mapping quality phred scores less than 30, minimum depth of 5 reads, and a SNP p-value (as computed by ANGSD) below 10<sup>−3</sup>. Clustering was robust to coverage variation and closely matched the PCA from the VQSR-called SNP genotypes (Supplementary Fig. 20).</p></div><div id="sec-15" class="subsection"><h3>Phylogenetic inference</h3><p id="p-79">To determine the phylogenetic structure of the cohort, the 223 consensus mitochondrial fasta sequences were first aligned using Muscle v3.8.31<sup><a id="xref-ref-83-1" class="xref-bibr" href="#ref-83">83</a></sup>, followed by stringent trimming of sequence alignments using Gblocks<sup><a id="xref-ref-84-1" class="xref-bibr" href="#ref-84">84</a></sup>. The most likely evolutionary model, GTR substitution model with rate heterogeneity modelled by a gamma distribution with invariable sites, was determined using modelgenerator v.0.851<sup><a id="xref-ref-85-1" class="xref-bibr" href="#ref-85">85</a></sup>. A maximum-likelihood tree was subsequently generated using PhyML<sup><a id="xref-ref-86-1" class="xref-bibr" href="#ref-86">86</a></sup> v.20120412, with branch supports computed using 100 bootstraps.</p></div><div id="sec-16" class="subsection"><h3>Admixture analysis</h3><p id="p-80">Admixture was determined using NGSAdmix<sup><a id="xref-ref-87-1" class="xref-bibr" href="#ref-87">87</a></sup>. This tool relies on genotype likelihoods to account for data uncertainty, and has been shown to produce robust inferences about population ancestry from low coverage samples alone, or a mixture of low and higher coverage samples<sup><a id="xref-ref-87-2" class="xref-bibr" href="#ref-87">87</a></sup>. This analysis was performed on 223 samples, for <em>K</em> ranging from 2 to 10 clusters (Supplementary Fig. 5 and 6), retaining sites with less than 50% missing data across individuals and minor allele frequency (MAF) above 5%. Five iterations were run omitting one autosome out at a time, and the best K was chosen as the first value that would minimize the median absolute deviation across runs (Supplementary Fig. 6). Sample coverage did not affect the results (Supplementary Fig. 21).</p></div><div id="sec-17" class="subsection"><h3>Effective population size and population divergence dating</h3><p id="p-81">The effective population size (<em>N<sub>e</sub></em>) trajectory through time, and the cross-divergence time between populations, were estimated using MSMC2<sup><a id="xref-ref-88-1" class="xref-bibr" href="#ref-88">88</a>,<a id="xref-ref-89-1" class="xref-bibr" href="#ref-89">89</a></sup>. This approach uses patterns of heterozygosity along the genome to identify past recombination events modelled as Markov processes<sup><a id="xref-ref-90-1" class="xref-bibr" href="#ref-90">90</a></sup>. Mutation density along the sequence mirrors either recent (long tract of limited diversity) or older (enrichment in heterozygosity over short distances) events. According to coalescent theory<sup><a id="xref-ref-91-1" class="xref-bibr" href="#ref-91">91</a></sup>, at any given time, the amount of recombination is proportional to <em>N<sub>e</sub></em>.</p><p id="p-82">MSMC2 was applied to individuals with a mean coverage above 10x, limiting the analysis to six isolates (FRA.1, GUA, IND, MOR, NAM, STA.1), and considering four haplotypes per isolate. Beagle v4.1 was used to impute missing genotypes and to establish phase in VQSR SNP calls. Imputation accuracy analysis revealed a 7.2% and 9.0% discordance rates at the individual and site levels respectively (Supplementary Fig. 22). Input files were created following MSMC recommendations and available msmc-tools (<a href="https://github.com/stschiff/msmc-tools">https://github.com/stschiff/msmc-tools</a>). Briefly, the reference fasta sequence was masked with SNPable (<a href="http://lh3lh3.users.sourceforge.net/snpable.shtml">http://lh3lh3.users.sourceforge.net/snpable.shtml</a>) to extract regions of unambiguous read mapping in chromosome-specific bed files (using the available msmc_create_map_mask.py python script). Negative bed files indicating regions with sufficient coverage at the individual level were created from samples bam files with the bamCaller.py script and filter out sites with coverage below genome-wide average depth. Finally, MSMC2 input files were created for each chromosome with the generate_multihetsep.py script and concatenated into a single input file. For each isolate, estimates were averaged across five runs, leaving one chromosome out at a time for cross-validation, using rho/mu parameter value of 6.22 (average recombination rate of 1.68 cM/Mbp<sup><a id="xref-ref-12-7" class="xref-bibr" href="#ref-12">12</a></sup> and considering a mutation rate similar to that of <em>C. elegans</em> mutation rate, 2.7 × 10<sup>−9</sup> per site per generation<sup><a id="xref-ref-20-2" class="xref-bibr" href="#ref-20">20</a></sup>). MSMC2 times and coalescent rates were scaled to real time and population sizes by assuming the same mutation rate<sup><a id="xref-ref-20-3" class="xref-bibr" href="#ref-20">20</a></sup>, a balanced sex-ratio<sup><a id="xref-ref-21-3" class="xref-bibr" href="#ref-21">21</a></sup> and an inferred generation interval of 40 days (the sum of 10 days to reach mature free-living infective larvae from the egg stage, and a 30-day prepatent period for fully mature egg-laying females)<sup><a id="xref-ref-92-1" class="xref-bibr" href="#ref-92">92</a></sup>.</p><p id="p-83">Migratory scenarios between populations were determined using the forward simulation framework implemented in δaδi<sup><a id="xref-ref-93-1" class="xref-bibr" href="#ref-93">93</a></sup>. Under a given evolutionary scenario, this software models the expected joint site frequency spectrum between multiple populations using a diffusion equation. These expected values are then used to compute the most likely demographic parameters knowing the observed site frequency spectrum. For each model, four rounds of forward simulations were run with 10, 20, 30 and 40 replicates respectively using previously published python scripts<sup><a id="xref-ref-94-1" class="xref-bibr" href="#ref-94">94</a></sup> (<a href="https://github.com/dportik/dadi_pipeline">https://github.com/dportik/dadi_pipeline</a>). Model Akaike Information Criterion (AIC) were compared for ranking scenarios, from which the lower the score, the more likely the outcome.</p><p id="p-84">We first compared a divergence scenario without migration against models including symmetrical and asymmetrical gene flow before isolation. In case migration was the most likely, more complex models (involving split with ancestral (a)symmetrical gene flow, with or without population size change, or models involving secondary contact with/without gene flow and population size change) were tested. However, initial exploration indicated a likely lack of power in our design to accurately estimate parameters of more complex demographic models than the split and isolation model. Nevertheless, these models still provide the most likely scenario and their output have been listed in supplementary Table 3.</p><p id="p-85">Parameters were scaled to real time using same parameters as for MSMC2 inference. Standard deviations of timing estimates for the simple split and isolation models were obtained using the Godambe Information Matrix<sup><a id="xref-ref-95-1" class="xref-bibr" href="#ref-95">95</a></sup> applied to 100 simulated site frequency spectra produced with the <em>ms</em> software<sup><a id="xref-ref-96-1" class="xref-bibr" href="#ref-96">96</a></sup> under the most likely demographic model.</p><p id="p-86">Additional support to the estimates from the nuclear genome were obtained from a phylogenetic analysis of coding sequences using BEASTv1.10<sup><a id="xref-ref-97-1" class="xref-bibr" href="#ref-97">97</a></sup>. Mitochondrial coding sequences were extracted from the consensus sequence of every individual, concatenated per individual, and aligned using Musclev3.8.31<sup><a id="xref-ref-83-2" class="xref-bibr" href="#ref-83">83</a></sup>. A Bayesian skyline model<sup><a id="xref-ref-98-1" class="xref-bibr" href="#ref-98">98</a></sup> was used, with a HKY substitution model and a strict clock model, as other modalities yielded weak effective sample size (ESS) and unstable parameter values. Clock rate was set to <em>C. elegans</em> mitochondrial mutation rate, i.e. 1 × 10<sup>−7</sup> per site per generation<sup><a id="xref-ref-74-2" class="xref-bibr" href="#ref-74">74</a></sup>, as variation in sampling date was not sufficient to estimate molecular rate. Parameters showed sufficient sampling (effective size above 200) after 50,000,000 iterations and a burn-in of the first 20 million steps. Node ages were scaled to years assuming a generation interval of 40 days. A maximum clade credibility tree was generated with TreeAnnotator v1.10.1 (<a href="http://beast.community/treeannotator">http://beast.community/treeannotator</a>).</p></div><div id="sec-18" class="subsection"><h3>Diversifying selection scan</h3><p id="p-87">To further characterize the genetic diversity in <em>H. contortus</em> populations, we identified genomic regions under diversifying selection using XP-CLR<sup><a id="xref-ref-27-3" class="xref-bibr" href="#ref-27">27</a></sup>. This approach takes advantage of both within-population distortion of the allele frequency spectrum, and between-population differences in allele frequencies in the vicinity of selective sweeps<sup><a id="xref-ref-27-4" class="xref-bibr" href="#ref-27">27</a></sup>. Although XP-CLR is robust to SNP ascertainment bias<sup><a id="xref-ref-27-5" class="xref-bibr" href="#ref-27">27</a></sup>, the analysis was restricted to isolates with at least five individuals with a minimum mean coverage of 2× per individual. Unphased VQSR-derived genotypes were filtered to retain SNPs with a within-isolate call rate &gt;80% and MAF &gt;5%. The analysis was run on every one of the 22 possible within-continent pairwise comparisons of the retained isolates (Australia and Africa) with the following options: -w1 0.0001 500 2000 -p0 0. This fit a grid of putatively selected points every 2 Kbp along the genome, with a sliding window size around grid points of 0.01 cM, interpolating SNP position in Morgans from the average recombination rate for each chromosome<sup><a id="xref-ref-12-8" class="xref-bibr" href="#ref-12">12</a></sup>. Down-sampling was applied to windows where more than 500 SNPs were found to keep SNP numbers comparable between regions, and no LD-based down-weighing of the CLR scores was applied. A selection score was subsequently computed at every position as the root mean square of XP-CLR coefficients, and the highest 0.1% of selection scores were deemed significant, as reported elsewhere<sup><a id="xref-ref-80-2" class="xref-bibr" href="#ref-80">80</a></sup>.</p></div><div id="sec-19" class="subsection"><h3>Analysis of β-tubulin isotype 1 (Hco-btub-iso-1) and the genetic architecture of benzimidazole- and ivermectin-resistance</h3><p id="p-88">The genomic coordinates of <em>Hco-btub-iso-1</em> were determined by blasting the gene coding sequence from WormBase Parasite<sup><a id="xref-ref-99-1" class="xref-bibr" href="#ref-99">99</a></sup> against current genome assembly (BLASTN, e-value &lt; 10<sup>−50</sup>). The SNP positions associated with codons 167, 198 and 200 was determined to be at 7027535, 7027755, and 7027758 bp, respectively, along chromosome I after aligning the <em>Hco-btub-iso-1</em> consensus sequence and published sequence<sup><a id="xref-ref-100-1" class="xref-bibr" href="#ref-100">100</a></sup> (GenBank accession <a class="external-ref external-ref-type-gen" href="/lookup/external-ref?link_type=GEN&amp;access_num=FJ981629.1&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F04%2F13%2F450692.atom">FJ981629.1</a>) using muscle v3.8.31<sup><a id="xref-ref-83-3" class="xref-bibr" href="#ref-83">83</a></sup>. Genotype and GLs at these positions were determined with ANGSD<sup><a id="xref-ref-69-4" class="xref-bibr" href="#ref-69">69</a></sup>. Genotypes were only considered for GL &gt;60% (n = 74) as coverage bias occurred otherwise (Supplementary Table 6 and Supplementary Fig. 12). For these samples, phased and imputed genotypes from VQSR SNPs spanning the <em>Hco-btub-iso-1</em> locus were used to compute pairwise number of allele differences. Selection in the vicinity of the <em>Hco-btub-iso-1</em> locus was assessed by computing Tajima’s <em>D</em> with ANGSD<sup><a id="xref-ref-101-1" class="xref-bibr" href="#ref-101">101</a></sup>. For benzimidazole-resistant isolates, neutral state was built with <em>ms<sup><a id="xref-ref-102-1" class="xref-bibr" href="#ref-102">102</a></sup></em> by simulating 10-Kbp wide isolate-specific sequences (n=1,000) following the same coalescent scenario as predicted by MSMC2 for chromosome I (considering a recombination rate of 1.83 cM/Mbp<sup><a id="xref-ref-12-9" class="xref-bibr" href="#ref-12">12</a></sup>). Suitable <em>ms</em> input parameters were derived from MSMC2 output files using the msmc2ms.py script from msmc-tools. This approach was implemented for isolates with sufficient mean depth of coverage, i.e. three benzimidazole-resistant isolates. In case of susceptible isolates, the lack of significant departure in the <em>Hco-btub-iso-1</em> vicinity relative to the rest of chromosome I was tested.</p><p id="p-89">The introgression of resistant haplotypes from France mainland into Guadeloupian worm isolates was tested by a topology weighting analysis implemented with TWISST<sup><a id="xref-ref-103-1" class="xref-bibr" href="#ref-103">103</a></sup>. This method computes phylogenetic trees between a set of isolates, using genetic information from short sliding windows (50 Kbp) and by sampling with replacement individuals from each isolate. At each window, a weight is subsequently computed for each of the possible tree topology, ultimately providing inference of introgression events in discrete locations where tree topology connects phylogenetically distant isolates. Analyses were run using individuals with sufficient mean depth of coverage (5× and more) from French (FRA.1; n = 5; mean depth = 7.62×) and Guadeloupian (GUA; n = 5; mean depth = 12.75×) isolates, adding Namibian isolate (n = 2; mean depth = 10.9×) as an outgroup. A fourth isolate was chosen for its common ancestry with Guadeloupian isolates: first, an analysis was run with worms from São Tomé (n = 2; mean depth = 7.01×), followed by a second analysis using Moroccan samples (n = 2; mean depth = 10.6×), to ensure that evidence of introgression was consistent in both cases.</p><p id="p-90">To investigate the genetic architecture of ivermectin resistance, a differentiation scan was run between ivermectin-resistant isolates and isolates of unknown status sharing same ancestry, i.e. STA.1 and STA.3 versus NAM and ZAI in Africa, AUS.1 against AUS.2 in Australia. Windowed F<sub>ST</sub> estimates<sup><a id="xref-ref-79-2" class="xref-bibr" href="#ref-79">79</a></sup> were calculated every 10 Kbp with a 1 Kbp overlap along the genome with ANGSD<sup><a id="xref-ref-69-5" class="xref-bibr" href="#ref-69">69</a></sup>, retaining sites with minimal depth of 5 reads, mapping and base quality phred scores of at least 30, missing rate below 50%, and windows with at least 1000 sites. Genomic coordinates of the top 0.5% most differentiated windows were extracted and analysed by BLASTN (minimum e-value = 10<sup>−50</sup>) against the published V1 <em>H. contortus</em> assembly<sup><a id="xref-ref-13-2" class="xref-bibr" href="#ref-13">13</a></sup>, and annotated gene identifiers were inferred from the corresponding GFF file from WormBase Parasite<sup><a id="xref-ref-99-2" class="xref-bibr" href="#ref-99">99</a></sup>.</p></div><div id="sec-20" class="subsection"><h3>Identification of SNPs under environmental selection</h3><p id="p-91">To identify SNPs putatively influenced by environmental selection, we first defined the climatic conditions of each isolate using the Köppen-Geiger classification<sup><a id="xref-ref-104-1" class="xref-bibr" href="#ref-104">104</a></sup>, and inferred from isolate geographical coordinates<sup><a id="xref-ref-105-1" class="xref-bibr" href="#ref-105">105</a></sup>. Three isolates with the best coverage, i.e. Namibia, France, and Guadeloupe, were used to contrast dry, temperate and tropical conditions, respectively. We performed a genome-wide scan using pairwise comparisons of F<sub>ST</sub> using ANGSD<sup><a id="xref-ref-69-6" class="xref-bibr" href="#ref-69">69</a></sup>, of which test values greater than the 0.5% quantile in at least two comparisons were analysed further.</p><p id="p-92">To further explore the contribution of environmental climatic variables on standing genetic variation, we applied a machine-learning gradient forest algorithm<sup><a id="xref-ref-106-1" class="xref-bibr" href="#ref-106">106</a></sup> to quantify changes in genetic variation (fit as population SNP MAF) along environmental gradients. The gradients consisted of 19 bioclimatic variables<sup><a id="xref-ref-107-1" class="xref-bibr" href="#ref-107">107</a></sup> summarizing rainfall and temperature information recorded between 1970 and 2000(supplementary table 9). For each SNP, a random forest of 500 trees was grown. For each tree, bootstrapped SNP MAFs were regressed against a random subset of bioclimatic variables to determine the variable that best partitioned the data, thereby building the first node that partitions the data into two sets of homogeneous observations. Iterations follow to determine subsequent nodes by resampling a random subset of bioclimatic variables until no observations are left. The proportion of variance explained by each bioclimatic variable is then averaged across SNPs, and the function of SNP frequency modification along bioclimatic variable is built.</p><p id="p-93">The analysis was performed on isolates with at least 5 individuals, a mean coverage of 2×, and no record of ivermectin resistance (AUS.2, FRA.1, GUA, IND, MOR, NAM, STO, ZAI). SNP MAFs were estimated from genotype likelihoods with ANGSD, and subsequently filtered to ensure a within-isolate MAF &gt;10%, at least 90% within-isolate call-rate, and shared across the eight considered isolates, resulting in 3,758 SNPs retained for further analysis.</p><p id="p-94">Environmental variables were highly correlated (Supplementary Fig. 15a), resulting in instability in the predictor’s importance. To minimise this effect, the gradient forest analysis was restricted to the 11 environmental variables showing least redundancy as assessed by a PCA (Supplementary Fig. 15b). Pair-wise Euclidean distances between variables was computed from their respective coordinates on the first two PCA axes (supplementary Fig. 15a). We selected variables with higher distances (mean distance &gt; 0.85) with any others (BIO4, 7, 2, 15, 5, 10). Other variables defined three clusters (Supplementary Fig. 15b). Within each cluster, we picked the variable with closest distance from every other in the cluster, i.e. the variable summarizing others’ contributions (BIO9, BIO13, BIO19). For the BIO12, 13, 16, 18 cluster, we chose to pick BIO12 (annual precipitation) which is more relevant toward parasite life-cycle across climatic areas. Under temperate areas, quarter-based (BIO16, BIO18) or wettest month (BIO13) statistics would match seasons where hosts are housed. SNP-environment associations were further investigated by focusing on the top two environmental predictors of genomic variations and using a Latent Factor Mixed Model analysis. This analysis was implemented with the lfmm R package<sup><a id="xref-ref-108-1" class="xref-bibr" href="#ref-108">108</a></sup> on genotype calls from the VQSR pipeline, considering SNPs with call rate of at least 70% across isolates and minor genotype frequency above 5%, using K = 3 for the latent factor accounting for underlying population structure. Analyses were run 5 times and <em>P</em> values were combined and adjusted as recommended<sup><a id="xref-ref-108-2" class="xref-bibr" href="#ref-108">108</a></sup>.</p></div><div id="sec-21" class="subsection"><h3>Gene identification and Gene Ontology enrichment analysis</h3><p id="p-95">Annotation of the revised <em>H. contortus</em> genome is ongoing. Therefore, genes underlying major differentiation signals were inferred by blasting the region of interest (10 Kbp window for F<sub>ST</sub> analyses, 2000 bp region around XP-CLR hits) against the published V1 <em>H. contortus</em> genome assembly<sup><a id="xref-ref-13-3" class="xref-bibr" href="#ref-13">13</a></sup>. Any genes falling within most probable blast hit coordinates were retrieved from the <em>H. contortus</em> published assembly available from WormBase Parasite<sup><a id="xref-ref-99-3" class="xref-bibr" href="#ref-99">99</a></sup>. This database was also used to retrieve <em>C. elegans</em> orthologs of positional candidate genes and <em>H. contortus</em> Gene Ontology (GO) terms. GO term enrichment analysis was run with the R topGO<sup><a id="xref-ref-109-1" class="xref-bibr" href="#ref-109">109</a></sup> package, considering nodes with at least 10 annotated genes. The “weight01” algorithm was used to account for existing topology between GO terms. This framework makes P-value of one GO term conditioned on its neighbours, thereby correcting for multiple testing. Enrichment was tested by the Kolmogorov-Smirnov statistic applied to gene selection or F<sub>ST</sub> score accordingly. GO terms with <em>P</em> values below 1% were deemed significant.</p></div></div><div class="section" id="sec-22"><h2 class="">Data availability</h2><p id="p-96">Raw sequencing data are archived under the ENA study accession PRJEB9837. Sequencing data were analysed with publicly available script and software as mentioned in main text. Outputs were analysed using an R script available at: <a href="https://github.com/guiSalle/Haemonchus_diversity">https://github.com/guiSalle/Haemonchus_diversity</a>. Reference assembly used in this project is available at: <a href="ftp://ngs.sanger.ac.uk/production/pathogens/Haemonchus_contortus">ftp://ngs.sanger.ac.uk/production/pathogens/Haemonchus_contortus</a>.</p></div><div class="section" id="sec-24"><h2 class="">Author contributions</h2><p id="p-98">GS, JAC designed the experiment. GS, JAC, SD drafted the manuscript. JCa and JCo sampled and prepared parasite materials. GS performed DNA extraction and data analyses. SD built the reference genome. MB and NH managed and supervised parasite sequencing and the project.</p></div><div class="section COI-statement" id="sec-25"><h2 class="">Competing interests</h2><p id="p-99">Authors declare they have no competing interests.</p></div><div class="section ack" id="ack-1"><h2 class="">Acknowledgements</h2><p id="p-97">JAC, MB, NH, and SD are supported by the Wellcome Trust via their core funding of the Wellcome Trust Sanger Institute (grant 206194). GS has received the support of the EU in the framework of the Marie-Curie FP7 COFUND People Programme, through the award of an AgreenSkills (grant agreement n° 267196) and AgreenSkills+ fellowships (grant agreement n°609398). Authors are grateful to Pr. Beech and Gilleard for insightful discussions.</p></div><div class="section ref-list" id="ref-list-1"><h2 class="">References</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label">1.</span><a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference 1. in text" id="ref-1">↵</a><div class="cit ref-cit ref-journal" id="cit-450692v2.1" data-doi="10.1017/S0031182014000791"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Blaxter</span>,  <span class="cit-name-given-names">M.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Koutsovoulos</span>,  <span class="cit-name-given-names">G.</span></span> <span class="cit-article-title">The evolution of parasitism in Nematoda</span>. <abbr class="cit-jnl-abbrev">Parasitology</abbr> <span class="cit-vol">142</span> <span class="cit-issue">Suppl 1</span>, <span class="cit-fpage">S26</span>–<span class="cit-lpage">39</span> (<span class="cit-pub-date">2015</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DParasitology%26rft.volume%253D142%26rft.spage%253DS26%26rft_id%253Dinfo%253Adoi%252F10.1017%252FS0031182014000791%26rft_id%253Dinfo%253Apmid%252F24963797%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1017/S0031182014000791&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24963797&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F04%2F13%2F450692.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">2.</span><a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference 2. in text" id="ref-2">↵</a><div class="cit ref-cit ref-journal" id="cit-450692v2.2" data-doi="10.1016/j.vetpar.2006.08.035"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">O'Connor</span>,  <span class="cit-name-given-names">L.J.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Walkden-Brown</span>,  <span class="cit-name-given-names">S.W.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Kahn</span>,  <span class="cit-name-given-names">L.P.</span></span> <span class="cit-article-title">Ecology of the free-living stages of major trichostrongylid parasites of sheep</span>. <abbr class="cit-jnl-abbrev">Vet Parasitol</abbr> <span class="cit-vol">142</span>, <span class="cit-fpage">1</span>–<span class="cit-lpage">15</span> (<span class="cit-pub-date">2006</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DVeterinary%2Bparasitology%26rft.stitle%253DVet%2BParasitol%26rft.aulast%253DO%2527connor%26rft.auinit1%253DL.%2BJ.%26rft.volume%253D142%26rft.issue%253D1-2%26rft.spage%253D1%26rft.epage%253D15%26rft.atitle%253DEcology%2Bof%2Bthe%2Bfree-living%2Bstages%2Bof%2Bmajor%2Btrichostrongylid%2Bparasites%2Bof%2Bsheep.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.vetpar.2006.08.035%26rft_id%253Dinfo%253Apmid%252F17011129%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.vetpar.2006.08.035&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17011129&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F04%2F13%2F450692.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">3.</span><a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference 3. in text" id="ref-3">↵</a><div class="cit ref-cit ref-journal" id="cit-450692v2.3" data-doi="10.1111/mpp.12057"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Jones</span>,  <span class="cit-name-given-names">J.T.</span></span> <span class="cit-etal">et al.</span> <span class="cit-article-title">Top 10 plant-parasitic nematodes in molecular plant pathology</span>. <abbr class="cit-jnl-abbrev">Mol Plant Pathol</abbr> <span class="cit-vol">14</span>, <span class="cit-fpage">946</span>–<span class="cit-lpage">61</span> (<span class="cit-pub-date">2013</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol%2BPlant%2BPathol%26rft.volume%253D14%26rft.spage%253D946%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fmpp.12057%26rft_id%253Dinfo%253Apmid%252F23809086%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/mpp.12057&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23809086&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F04%2F13%2F450692.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000326454500010&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">4.</span><a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference 4. in text" id="ref-4">↵</a><div class="cit ref-cit ref-journal" id="cit-450692v2.4" data-doi="10.1139/z84-050"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Anderson</span>,  <span class="cit-name-given-names">R.C.</span></span> <span class="cit-article-title">The origins of zooparasitic nematodes</span>. <abbr class="cit-jnl-abbrev">Can J Zool</abbr> <span class="cit-vol">62</span>, <span class="cit-fpage">317</span>–<span class="cit-lpage">328</span> (<span class="cit-pub-date">1984</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCan%2BJ%2BZool%26rft.volume%253D62%26rft.spage%253D317%26rft_id%253Dinfo%253Adoi%252F10.1139%252Fz84-050%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1139/z84-050&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=A1984SK75100001&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">5.</span><a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference 5. in text" id="ref-5">↵</a><div class="cit ref-cit ref-journal" id="cit-450692v2.5" data-doi="10.1016/j.vetpar.2011.11.048"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Kaplan</span>,  <span class="cit-name-given-names">R.M.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Vidyashankar</span>,  <span class="cit-name-given-names">A.N.</span></span> <span class="cit-article-title">An inconvenient truth: Global worming and anthelmintic resistance</span>. <abbr class="cit-jnl-abbrev">Veterinary Parasitology</abbr> <span class="cit-vol">186</span>, <span class="cit-fpage">70</span>–<span class="cit-lpage">78</span> (<span class="cit-pub-date">2012</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DVeterinary%2Bparasitology%26rft.stitle%253DVet%2BParasitol%26rft.aulast%253DKaplan%26rft.auinit1%253DR.%2BM.%26rft.volume%253D186%26rft.issue%253D1-2%26rft.spage%253D70%26rft.epage%253D78%26rft.atitle%253DAn%2Binconvenient%2Btruth%253A%2Bglobal%2Bworming%2Band%2Banthelmintic%2Bresistance.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.vetpar.2011.11.048%26rft_id%253Dinfo%253Apmid%252F22154968%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.vetpar.2011.11.048&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22154968&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F04%2F13%2F450692.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">6.</span><a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference 6. in text" id="ref-6">↵</a><div class="cit ref-cit ref-journal" id="cit-450692v2.6" data-doi="10.1016/j.pt.2004.08.002"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">McKellar</span>,  <span class="cit-name-given-names">Q.A.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Jackson</span>,  <span class="cit-name-given-names">F.</span></span> <span class="cit-article-title">Veterinary anthelmintics: old and new</span>. <abbr class="cit-jnl-abbrev">Trends Parasitol</abbr> <span class="cit-vol">20</span>, <span class="cit-fpage">456</span>–<span class="cit-lpage">61</span> (<span class="cit-pub-date">2004</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTrends%2Bin%2Bparasitology%26rft.stitle%253DTrends%2BParasitol%26rft.aulast%253DMcKellar%26rft.auinit1%253DQ.%2BA.%26rft.volume%253D20%26rft.issue%253D10%26rft.spage%253D456%26rft.epage%253D461%26rft.atitle%253DVeterinary%2Banthelmintics%253A%2Bold%2Band%2Bnew.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.pt.2004.08.002%26rft_id%253Dinfo%253Apmid%252F15363438%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.pt.2004.08.002&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15363438&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F04%2F13%2F450692.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000224257600002&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">7.</span><a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference 7. in text" id="ref-7">↵</a><div class="cit ref-cit ref-journal" id="cit-450692v2.7"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Bundy</span>,  <span class="cit-name-given-names">D.A.P.</span></span> <span class="cit-etal">et al.</span> <span class="cit-article-title">100 Years of Mass Deworming Programmes: A Policy Perspective From the World Bank's Disease Control Priorities Analyses</span>. <abbr class="cit-jnl-abbrev">Adv Parasitol</abbr> <span class="cit-vol">100</span>, <span class="cit-fpage">127</span>–<span class="cit-lpage">154</span> (<span class="cit-pub-date">2018</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAdv%2BParasitol%26rft.volume%253D100%26rft.spage%253D127%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">8.</span><a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference 8. in text" id="ref-8">↵</a><div class="cit ref-cit ref-journal" id="cit-450692v2.8"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Schulz</span>,  <span class="cit-name-given-names">J.D.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Moser</span>,  <span class="cit-name-given-names">W.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Hurlimann</span>,  <span class="cit-name-given-names">E.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Keiser</span>,  <span class="cit-name-given-names">J.</span></span> <span class="cit-article-title">Preventive Chemotherapy in the Fight against Soil-Transmitted Helminthiasis: Achievements and Limitations</span>. <abbr class="cit-jnl-abbrev">Trends Parasitol</abbr> (<span class="cit-pub-date">2018</span>).</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">9.</span><a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference 9. in text" id="ref-9">↵</a><div class="cit ref-cit ref-journal" id="cit-450692v2.9" data-doi="10.1586/14760584.2014.893195"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Hewitson</span>,  <span class="cit-name-given-names">J.P.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Maizels</span>,  <span class="cit-name-given-names">R.M.</span></span> <span class="cit-article-title">Vaccination against helminth parasite infections</span>. <abbr class="cit-jnl-abbrev">Expert Rev Vaccines</abbr> <span class="cit-vol">13</span>, <span class="cit-fpage">473</span>–<span class="cit-lpage">87</span> (<span class="cit-pub-date">2014</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DExpert%2BRev%2BVaccines%26rft.volume%253D13%26rft.spage%253D473%26rft_id%253Dinfo%253Adoi%252F10.1586%252F14760584.2014.893195%26rft_id%253Dinfo%253Apmid%252F24606541%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1586/14760584.2014.893195&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24606541&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F04%2F13%2F450692.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">10.</span><a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference 10. in text" id="ref-10">↵</a><div class="cit ref-cit ref-journal" id="cit-450692v2.10"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Sallé</span>,  <span class="cit-name-given-names">G.</span></span> <span class="cit-etal">et al.</span> <span class="cit-article-title">Transcriptomic profiling of nematode parasites surviving vaccine exposure</span>. <abbr class="cit-jnl-abbrev">Int J Parasitol</abbr> (<span class="cit-pub-date">2018</span>).</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">11.</span><a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference 11. in text" id="ref-11">↵</a><div class="cit ref-cit ref-journal" id="cit-450692v2.11" data-doi="10.1017/S0031182013001145"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Gilleard</span>,  <span class="cit-name-given-names">J.S.</span></span> <span class="cit-article-title">Haemonchus contortus as a paradigm and model to study anthelmintic drug resistance</span>. <abbr class="cit-jnl-abbrev">Parasitology</abbr> <span class="cit-vol">140</span>, <span class="cit-fpage">1506</span>–<span class="cit-lpage">22</span> (<span class="cit-pub-date">2013</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DParasitology%26rft.volume%253D140%26rft.spage%253D1506%26rft_id%253Dinfo%253Adoi%252F10.1017%252FS0031182013001145%26rft_id%253Dinfo%253Apmid%252F23998513%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1017/S0031182013001145&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23998513&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F04%2F13%2F450692.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">12.</span><a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference 12. in text" id="ref-12">↵</a><div class="cit ref-cit ref-journal" id="cit-450692v2.12"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Doyle</span>,  <span class="cit-name-given-names">S.R.</span></span> <span class="cit-etal">et al.</span> <span class="cit-article-title">A Genome Resequencing-Based Genetic Map Reveals the Recombination Landscape of an Outbred Parasitic Nematode in the Presence of Polyploidy and Polyandry</span>. <abbr class="cit-jnl-abbrev">Genome Biol Evol</abbr> <span class="cit-vol">10</span>, <span class="cit-fpage">396</span>–<span class="cit-lpage">409</span> (<span class="cit-pub-date">2018</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BBiol%2BEvol%26rft.volume%253D10%26rft.spage%253D396%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">13.</span><a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference 13. in text" id="ref-13">↵</a><div class="cit ref-cit ref-journal" id="cit-450692v2.13" data-doi="10.1186/gb-2013-14-8-r88"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Laing</span>,  <span class="cit-name-given-names">R.</span></span> <span class="cit-etal">et al.</span> <span class="cit-article-title">The genome and transcriptome of Haemonchus contortus, a key model parasite for drug and vaccine discovery</span>. <abbr class="cit-jnl-abbrev">Genome Biol</abbr> <span class="cit-vol">14</span>, <span class="cit-fpage">R88</span> (<span class="cit-pub-date">2013</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BBiol%26rft.volume%253D14%26rft.spage%253DR88%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fgb-2013-14-8-r88%26rft_id%253Dinfo%253Apmid%252F23985316%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/gb-2013-14-8-r88&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23985316&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F04%2F13%2F450692.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">14.</span><a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference 14. in text" id="ref-14">↵</a><div class="cit ref-cit ref-journal" id="cit-450692v2.14"><div class="cit-metadata"><cite><span class="cit-auth cit-collab">International Helminth Genomes, C</span>. <span class="cit-article-title">Comparative genomics of the major parasitic worms</span>. <abbr class="cit-jnl-abbrev">Nat Genet</abbr> <span class="cit-vol">51</span>, <span class="cit-fpage">163</span>–<span class="cit-lpage">174</span> (<span class="cit-pub-date">2019</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat%2BGenet%26rft.volume%253D51%26rft.spage%253D163%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">15.</span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-450692v2.15"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Mougin</span>,  <span class="cit-name-given-names">C.</span></span> <span class="cit-etal">et al.</span> <span class="cit-article-title">BRC4Env, a network of Biological Resource Centres for research in environmental and agricultural sciences</span>. <abbr class="cit-jnl-abbrev">Environ Sci Pollut Res Int</abbr> (<span class="cit-pub-date">2018</span>).</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">16.</span><a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference 16. in text" id="ref-16">↵</a><div class="cit ref-cit ref-journal" id="cit-450692v2.16" data-doi="10.1534/genetics.112.142018"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Langley</span>,  <span class="cit-name-given-names">C.H.</span></span> <span class="cit-etal">et al.</span> <span class="cit-article-title">Genomic variation in natural populations of Drosophila melanogaster</span>. <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">192</span>, <span class="cit-fpage">533</span>–<span class="cit-lpage">98</span> (<span class="cit-pub-date">2012</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.112.142018%26rft_id%253Dinfo%253Apmid%252F22673804%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTkyLzIvNTMzIjtzOjQ6ImF0b20iO3M6Mzc6Ii9iaW9yeGl2L2Vhcmx5LzIwMTkvMDQvMTMvNDUwNjkyLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">17.</span><a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference 17. in text" id="ref-17">↵</a><div class="cit ref-cit ref-journal" id="cit-450692v2.17"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Small</span>,  <span class="cit-name-given-names">S.T.</span></span> <span class="cit-etal">et al.</span> <span class="cit-article-title">Population genomics of the filarial nematode parasite Wuchereria bancrofti from mosquitoes</span>. <abbr class="cit-jnl-abbrev">Mol Ecol</abbr> <span class="cit-vol">25</span>, <span class="cit-fpage">1465</span>–<span class="cit-lpage">77</span> (<span class="cit-pub-date">2016</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol%2BEcol%26rft.volume%253D25%26rft.spage%253D1465%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">18.</span><a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference 18. in text" id="ref-18">↵</a><div class="cit ref-cit ref-journal" id="cit-450692v2.18"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Choi</span>,  <span class="cit-name-given-names">Y.J.</span></span> <span class="cit-etal">et al.</span> <span class="cit-article-title">Genomic diversity in Onchocerca volvulus and its Wolbachia endosymbiont</span>. <abbr class="cit-jnl-abbrev">Nat Microbiol</abbr> <span class="cit-vol">2</span>, <span class="cit-fpage">16207</span> (<span class="cit-pub-date">2016</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat%2BMicrobiol%26rft.volume%253D2%26rft.spage%253D16207%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">19.</span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-450692v2.19" data-doi="10.1101/gr.130468.111"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Perry</span>,  <span class="cit-name-given-names">G.H.</span></span> <span class="cit-etal">et al.</span> <span class="cit-article-title">Comparative RNA sequencing reveals substantial genetic variation in endangered primates</span>. <abbr class="cit-jnl-abbrev">Genome Res</abbr> <span class="cit-vol">22</span>, <span class="cit-fpage">602</span>–<span class="cit-lpage">10</span> (<span class="cit-pub-date">2012</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BRes%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.130468.111%26rft_id%253Dinfo%253Apmid%252F22207615%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjg6IjIyLzQvNjAyIjtzOjQ6ImF0b20iO3M6Mzc6Ii9iaW9yeGl2L2Vhcmx5LzIwMTkvMDQvMTMvNDUwNjkyLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">20.</span><a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference 20. in text" id="ref-20">↵</a><div class="cit ref-cit ref-journal" id="cit-450692v2.20" data-doi="10.1073/pnas.0904895106"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Denver</span>,  <span class="cit-name-given-names">D.R.</span></span> <span class="cit-etal">et al.</span> <span class="cit-article-title">A genome-wide view of Caenorhabditis elegans base-substitution mutation processes</span>. <abbr class="cit-jnl-abbrev">Proc Natl Acad Sci U S A</abbr> <span class="cit-vol">106</span>, <span class="cit-fpage">16310</span>–<span class="cit-lpage">4</span> (<span class="cit-pub-date">2009</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc%2BNatl%2BAcad%2BSci%2BU%2BS%2BA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.0904895106%26rft_id%253Dinfo%253Apmid%252F19805298%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMjoiMTA2LzM4LzE2MzEwIjtzOjQ6ImF0b20iO3M6Mzc6Ii9iaW9yeGl2L2Vhcmx5LzIwMTkvMDQvMTMvNDUwNjkyLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">21.</span><a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference 21. in text" id="ref-21">↵</a><div class="cit ref-cit ref-journal" id="cit-450692v2.21"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Saccareau</span>,  <span class="cit-name-given-names">M.</span></span> <span class="cit-etal">et al.</span> <span class="cit-article-title">Meta-analysis of the parasitic phase traits of Haemonchus contortus infection in sheep</span>. <abbr class="cit-jnl-abbrev">Parasit Vectors</abbr> <span class="cit-vol">10</span>, <span class="cit-fpage">201</span> (<span class="cit-pub-date">2017</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DParasit%2BVectors%26rft.volume%253D10%26rft.spage%253D201%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">22.</span><a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference 22. in text" id="ref-22">↵</a><div class="cit ref-cit ref-journal" id="cit-450692v2.22"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Geggus</span>,  <span class="cit-name-given-names">D.</span></span> <span class="cit-article-title">The French Slave Trade: An Overview</span>. <abbr class="cit-jnl-abbrev">The William and Mary Quaterly</abbr> <span class="cit-vol">58</span>, <span class="cit-fpage">119</span>–<span class="cit-lpage">138</span> (<span class="cit-pub-date">2001</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DThe%2BWilliam%2Band%2BMary%2BQuaterly%26rft.volume%253D58%26rft.spage%253D119%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">23.</span><a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference 23. in text" id="ref-23">↵</a><div class="cit ref-cit ref-book" id="cit-450692v2.23"><div class="cit-metadata"><cite><span class="cit-chapter-title">Anonymous</span>. <span class="cit-source">The wool industry – looking back and forward</span>. (<span class="cit-publ-name">Australian Bureau of Statistics</span>, <span class="cit-pub-date">2003</span>).</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">24.</span><a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference 24. in text" id="ref-24">↵</a><div class="cit ref-cit ref-journal" id="cit-450692v2.24"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Evesson</span>,  <span class="cit-name-given-names">B.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Moor</span>,  <span class="cit-name-given-names">R.</span></span> <span class="cit-article-title">The foundation of Australia’s fine wool industry</span>. in <abbr class="cit-jnl-abbrev">Journal of the Royal Australian Historical Society</abbr> (<span class="cit-pub-date">2000</span>).</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">25.</span><a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference 25. in text" id="ref-25">↵</a><div class="cit ref-cit ref-journal" id="cit-450692v2.25" data-doi="10.1016/j.molbiopara.2006.08.007"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Eng</span>,  <span class="cit-name-given-names">J.K.</span></span> <span class="cit-etal">et al.</span> <span class="cit-article-title">Ivermectin selection on beta-tubulin: evidence in Onchocerca volvulus and Haemonchus contortus</span>. <abbr class="cit-jnl-abbrev">Mol Biochem Parasitol</abbr> <span class="cit-vol">150</span>, <span class="cit-fpage">229</span>–<span class="cit-lpage">35</span> (<span class="cit-pub-date">2006</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2Band%2Bbiochemical%2Bparasitology%26rft.stitle%253DMol%2BBiochem%2BParasitol%26rft.aulast%253DEng%26rft.auinit1%253DJ.%2BK.%26rft.volume%253D150%26rft.issue%253D2%26rft.spage%253D229%26rft.epage%253D235%26rft.atitle%253DIvermectin%2Bselection%2Bon%2Bbeta-tubulin%253A%2Bevidence%2Bin%2BOnchocerca%2Bvolvulus%2Band%2BHaemonchus%2Bcontortus.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.molbiopara.2006.08.007%26rft_id%253Dinfo%253Apmid%252F17007942%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.molbiopara.2006.08.007&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17007942&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F04%2F13%2F450692.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">26.</span><a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference 26. in text" id="ref-26">↵</a><div class="cit ref-cit ref-journal" id="cit-450692v2.26" data-doi="10.1097/FPC.0b013e3282f4711d"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">de Lourdes Mottier</span>,  <span class="cit-name-given-names">M.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Prichard</span>,  <span class="cit-name-given-names">R.K.</span></span> <span class="cit-article-title">Genetic analysis of a relationship between macrocyclic lactone and benzimidazole anthelmintic selection on Haemonchus contortus</span>. <abbr class="cit-jnl-abbrev">Pharmacogenet Genomics</abbr> <span class="cit-vol">18</span>, <span class="cit-fpage">129</span>–<span class="cit-lpage">40</span> (<span class="cit-pub-date">2008</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPharmacogenet%2BGenomics%26rft.aulast%253DMottier%26rft.auinit1%253DM.%2Bd.%2Be.%2B%2B.%2BL.%26rft.volume%253D18%26rft.issue%253D2%26rft.spage%253D129%26rft.epage%253D140%26rft.atitle%253DGenetic%2Banalysis%2Bof%2Ba%2Brelationship%2Bbetween%2Bmacrocyclic%2Blactone%2Band%2Bbenzimidazole%2Banthelmintic%2Bselection%2Bon%2BHaemonchus%2Bcontortus.%26rft_id%253Dinfo%253Adoi%252F10.1097%252FFPC.0b013e3282f4711d%26rft_id%253Dinfo%253Apmid%252F18192899%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1097/FPC.0b013e3282f4711d&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18192899&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F04%2F13%2F450692.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000252642800006&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">27.</span><a class="rev-xref-ref" href="#xref-ref-27-1" title="View reference 27. in text" id="ref-27">↵</a><div class="cit ref-cit ref-journal" id="cit-450692v2.27" data-doi="10.1101/gr.100545.109"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Chen</span>,  <span class="cit-name-given-names">H.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Patterson</span>,  <span class="cit-name-given-names">N.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Reich</span>,  <span class="cit-name-given-names">D.</span></span> <span class="cit-article-title">Population differentiation as a test for selective sweeps</span>. <abbr class="cit-jnl-abbrev">Genome Res</abbr> <span class="cit-vol">20</span>, <span class="cit-fpage">393</span>–<span class="cit-lpage">402</span> (<span class="cit-pub-date">2010</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BRes%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.100545.109%26rft_id%253Dinfo%253Apmid%252F20086244%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjg6IjIwLzMvMzkzIjtzOjQ6ImF0b20iO3M6Mzc6Ii9iaW9yeGl2L2Vhcmx5LzIwMTkvMDQvMTMvNDUwNjkyLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">28.</span><a class="rev-xref-ref" href="#xref-ref-28-1" title="View reference 28. in text" id="ref-28">↵</a><div class="cit ref-cit ref-journal" id="cit-450692v2.28" data-doi="10.1016/j.vetpar.2009.06.043"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Ardelli</span>,  <span class="cit-name-given-names">B.F.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Stitt</span>,  <span class="cit-name-given-names">L.E.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Tompkins</span>,  <span class="cit-name-given-names">J.B.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Prichard</span>,  <span class="cit-name-given-names">R.K.</span></span> <span class="cit-article-title">A comparison of the effects of ivermectin and moxidectin on the nematode Caenorhabditis elegans</span>. <abbr class="cit-jnl-abbrev">Vet Parasitol</abbr> <span class="cit-vol">165</span>, <span class="cit-fpage">96</span>–<span class="cit-lpage">108</span> (<span class="cit-pub-date">2009</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DVeterinary%2Bparasitology%26rft.stitle%253DVet%2BParasitol%26rft.aulast%253DArdelli%26rft.auinit1%253DB.%2BF.%26rft.volume%253D165%26rft.issue%253D1-2%26rft.spage%253D96%26rft.epage%253D108%26rft.atitle%253DA%2Bcomparison%2Bof%2Bthe%2Beffects%2Bof%2Bivermectin%2Band%2Bmoxidectin%2Bon%2Bthe%2Bnematode%2BCaenorhabditis%2Belegans.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.vetpar.2009.06.043%26rft_id%253Dinfo%253Apmid%252F19631471%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.vetpar.2009.06.043&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19631471&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F04%2F13%2F450692.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">29.</span><a class="rev-xref-ref" href="#xref-ref-29-1" title="View reference 29. in text" id="ref-29">↵</a><div class="cit ref-cit ref-journal" id="cit-450692v2.29" data-doi="10.1016/j.ijpddr.2015.03.003"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Janssen</span>,  <span class="cit-name-given-names">I.J.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Krucken</span>,  <span class="cit-name-given-names">J.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Demeler</span>,  <span class="cit-name-given-names">J.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">von Samson-Himmelstjerna</span>,  <span class="cit-name-given-names">G.</span></span> <span class="cit-article-title">Transgenically expressed Parascaris P-glycoprotein-11 can modulate ivermectin susceptibility in Caenorhabditis elegans</span>. <abbr class="cit-jnl-abbrev">Int J Parasitol Drugs Drug Resist</abbr> <span class="cit-vol">5</span>, <span class="cit-fpage">44</span>–<span class="cit-lpage">7</span> (<span class="cit-pub-date">2015</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DInt%2BJ%2BParasitol%2BDrugs%2BDrug%2BResist%26rft.volume%253D5%26rft.spage%253D44%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.ijpddr.2015.03.003%26rft_id%253Dinfo%253Apmid%252F25905032%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.ijpddr.2015.03.003&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25905032&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F04%2F13%2F450692.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">30.</span><a class="rev-xref-ref" href="#xref-ref-30-1" title="View reference 30. in text" id="ref-30">↵</a><div class="cit ref-cit ref-journal" id="cit-450692v2.30" data-doi="10.1002/jez.1402530305"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Avery</span>,  <span class="cit-name-given-names">L.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Horvitz</span>,  <span class="cit-name-given-names">H.R.</span></span> <span class="cit-article-title">Effects of starvation and neuroactive drugs on feeding in Caenorhabditis elegans</span>. <abbr class="cit-jnl-abbrev">J Exp Zool</abbr> <span class="cit-vol">253</span>, <span class="cit-fpage">263</span>–<span class="cit-lpage">70</span> (<span class="cit-pub-date">1990</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DThe%2BJournal%2Bof%2Bexperimental%2Bzoology%26rft.stitle%253DJ%2BExp%2BZool%26rft.aulast%253DAvery%26rft.auinit1%253DL.%26rft.volume%253D253%26rft.issue%253D3%26rft.spage%253D263%26rft.epage%253D270%26rft.atitle%253DEffects%2Bof%2Bstarvation%2Band%2Bneuroactive%2Bdrugs%2Bon%2Bfeeding%2Bin%2BCaenorhabditis%2Belegans.%26rft_id%253Dinfo%253Adoi%252F10.1002%252Fjez.1402530305%26rft_id%253Dinfo%253Apmid%252F2181052%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1002/jez.1402530305&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=2181052&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F04%2F13%2F450692.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1990CX80800004&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">31.</span><a class="rev-xref-ref" href="#xref-ref-31-1" title="View reference 31. in text" id="ref-31">↵</a><div class="cit ref-cit ref-journal" id="cit-450692v2.31"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Scott</span>,  <span class="cit-name-given-names">E.W.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Baxter</span>,  <span class="cit-name-given-names">P.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Armour</span>,  <span class="cit-name-given-names">J.</span></span> <span class="cit-article-title">Fecundity of anthelmintic resistant adult Haemonchus contortus after exposure to ivermectin or benzimidazoles in vivo</span>. <abbr class="cit-jnl-abbrev">Res Vet Sci</abbr> <span class="cit-vol">50</span>, <span class="cit-fpage">247</span>–<span class="cit-lpage">9</span> (<span class="cit-pub-date">1991</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DResearch%2Bin%2Bveterinary%2Bscience%26rft.stitle%253DRes%2BVet%2BSci%26rft.aulast%253DScott%26rft.auinit1%253DE.%2BW.%26rft.volume%253D50%26rft.issue%253D2%26rft.spage%253D247%26rft.epage%253D249%26rft.atitle%253DFecundity%2Bof%2Banthelmintic%2Bresistant%2Badult%2BHaemonchus%2Bcontortus%2Bafter%2Bexposure%2Bto%2Bivermectin%2Bor%2Bbenzimidazoles%2Bin%2Bvivo.%26rft_id%253Dinfo%253Apmid%252F2034909%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=2034909&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F04%2F13%2F450692.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">32.</span><a class="rev-xref-ref" href="#xref-ref-32-1" title="View reference 32. in text" id="ref-32">↵</a><div class="cit ref-cit ref-journal" id="cit-450692v2.32"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Kotze</span>,  <span class="cit-name-given-names">A.C.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Prichard</span>,  <span class="cit-name-given-names">R.K.</span></span> <span class="cit-article-title">Anthelmintic Resistance in Haemonchus contortus: History, Mechanisms and Diagnosis</span>. <abbr class="cit-jnl-abbrev">Adv Parasitol</abbr> <span class="cit-vol">93</span>, <span class="cit-fpage">397</span>–<span class="cit-lpage">428</span> (<span class="cit-pub-date">2016</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAdv%2BParasitol%26rft.volume%253D93%26rft.spage%253D397%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">33.</span><a class="rev-xref-ref" href="#xref-ref-33-1" title="View reference 33. in text" id="ref-33">↵</a><div class="cit ref-cit ref-journal" id="cit-450692v2.33" data-doi="10.1016/j.pt.2012.04.003"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Martin</span>,  <span class="cit-name-given-names">R.J.</span></span> <span class="cit-etal">et al.</span> <span class="cit-article-title">Levamisole receptors: a second awakening</span>. <abbr class="cit-jnl-abbrev">Trends Parasitol</abbr> <span class="cit-vol">28</span>, <span class="cit-fpage">289</span>–<span class="cit-lpage">96</span> (<span class="cit-pub-date">2012</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTrends%2Bin%2Bparasitology%26rft.stitle%253DTrends%2BParasitol%26rft.aulast%253DMartin%26rft.auinit1%253DR.%2BJ.%26rft.volume%253D28%26rft.issue%253D7%26rft.spage%253D289%26rft.epage%253D296%26rft.atitle%253DLevamisole%2Breceptors%253A%2Ba%2Bsecond%2Bawakening.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.pt.2012.04.003%26rft_id%253Dinfo%253Apmid%252F22607692%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.pt.2012.04.003&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22607692&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F04%2F13%2F450692.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">34.</span><a class="rev-xref-ref" href="#xref-ref-34-1" title="View reference 34. in text" id="ref-34">↵</a><div class="cit ref-cit ref-journal" id="cit-450692v2.34" data-doi="10.1074/jbc.272.7.3875"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Wada</span>,  <span class="cit-name-given-names">M.</span></span> <span class="cit-etal">et al.</span> <span class="cit-article-title">Isolation and characterization of a GDP/GTP exchange protein specific for the Rab3 subfamily small G proteins</span>. <abbr class="cit-jnl-abbrev">J Biol Chem</abbr> <span class="cit-vol">272</span>, <span class="cit-fpage">3875</span>–<span class="cit-lpage">8</span> (<span class="cit-pub-date">1997</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BBiological%2BChemistry%26rft.stitle%253DJ%2BBiol%2BChem%26rft.aulast%253DWada%26rft.auinit1%253DM.%26rft.volume%253D272%26rft.issue%253D7%26rft.spage%253D3875%26rft.epage%253D3878%26rft.atitle%253DIsolation%2Band%2BCharacterization%2Bof%2Ba%2BGDP%252FGTP%2BExchange%2BProtein%2BSpecific%2Bfor%2Bthe%2BRab3%2BSubfamily%2BSmall%2BG%2BProteins%26rft_id%253Dinfo%253Adoi%252F10.1074%252Fjbc.272.7.3875%26rft_id%253Dinfo%253Apmid%252F9020086%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiamJjIjtzOjU6InJlc2lkIjtzOjEwOiIyNzIvNy8zODc1IjtzOjQ6ImF0b20iO3M6Mzc6Ii9iaW9yeGl2L2Vhcmx5LzIwMTkvMDQvMTMvNDUwNjkyLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">35.</span><a class="rev-xref-ref" href="#xref-ref-35-1" title="View reference 35. in text" id="ref-35">↵</a><div class="cit ref-cit ref-journal" id="cit-450692v2.35" data-doi="10.1016/j.gde.2004.02.001"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Cao</span>,  <span class="cit-name-given-names">R.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Zhang</span>,  <span class="cit-name-given-names">Y.</span></span> <span class="cit-article-title">The functions of E(Z)/EZH2-mediated methylation of lysine 27 in histone H3</span>. <abbr class="cit-jnl-abbrev">Curr Opin Genet Dev</abbr> <span class="cit-vol">14</span>, <span class="cit-fpage">155</span>–<span class="cit-lpage">64</span> (<span class="cit-pub-date">2004</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCurrent%2Bopinion%2Bin%2Bgenetics%2B%2526%2Bdevelopment%26rft.stitle%253DCurr%2BOpin%2BGenet%2BDev%26rft.aulast%253DCao%26rft.auinit1%253DR.%26rft.volume%253D14%26rft.issue%253D2%26rft.spage%253D155%26rft.epage%253D164%26rft.atitle%253DThe%2Bfunctions%2Bof%2BE%2528Z%2529%252FEZH2-mediated%2Bmethylation%2Bof%2Blysine%2B27%2Bin%2Bhistone%2BH3.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.gde.2004.02.001%26rft_id%253Dinfo%253Apmid%252F15196462%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.gde.2004.02.001&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15196462&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F04%2F13%2F450692.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000220832800009&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">36.</span><a class="rev-xref-ref" href="#xref-ref-36-1" title="View reference 36. in text" id="ref-36">↵</a><div class="cit ref-cit ref-journal" id="cit-450692v2.36"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Bolajoko</span>,  <span class="cit-name-given-names">M.B.</span></span> <span class="cit-etal">et al.</span> <span class="cit-article-title">The basic reproduction quotient (Q0) as a potential spatial predictor of the seasonality of ovine haemonchosis</span>. <abbr class="cit-jnl-abbrev">Geospat Health</abbr> <span class="cit-vol">9</span>, <span class="cit-fpage">333</span>–<span class="cit-lpage">50</span> (<span class="cit-pub-date">2015</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGeospat%2BHealth%26rft.volume%253D9%26rft.spage%253D333%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">37.</span><a class="rev-xref-ref" href="#xref-ref-37-1" title="View reference 37. in text" id="ref-37">↵</a><div class="cit ref-cit ref-journal" id="cit-450692v2.37" data-doi="10.1111/ele.12376"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Fitzpatrick</span>,  <span class="cit-name-given-names">M.C.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Keller</span>,  <span class="cit-name-given-names">S.R.</span></span> <span class="cit-article-title">Ecological genomics meets community-level modelling of biodiversity: mapping the genomic landscape of current and future environmental adaptation</span>. <abbr class="cit-jnl-abbrev">Ecol Lett</abbr> <span class="cit-vol">18</span>, <span class="cit-fpage">1</span>–<span class="cit-lpage">16</span> (<span class="cit-pub-date">2015</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEcol%2BLett%26rft.volume%253D18%26rft.spage%253D1%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fele.12376%26rft_id%253Dinfo%253Apmid%252F25270536%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/ele.12376&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25270536&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F04%2F13%2F450692.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">38.</span><a class="rev-xref-ref" href="#xref-ref-38-1" title="View reference 38. in text" id="ref-38">↵</a><div class="cit ref-cit ref-journal" id="cit-450692v2.38"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Doyle</span>,  <span class="cit-name-given-names">S.R.</span></span> <span class="cit-etal">et al.</span> <span class="cit-article-title">Population genomic and evolutionary modelling analyses reveal a single major QTL for ivermectin drug resistance in the pathogenic nematode, Haemonchus contortus</span>. <abbr class="cit-jnl-abbrev">BMC Genomics</abbr> <span class="cit-vol">20</span>, <span class="cit-fpage">218</span> (<span class="cit-pub-date">2019</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2BGenomics%26rft.volume%253D20%26rft.spage%253D218%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">39.</span><a class="rev-xref-ref" href="#xref-ref-39-1" title="View reference 39. in text" id="ref-39">↵</a><div class="cit ref-cit ref-journal" id="cit-450692v2.39"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Blouin</span>,  <span class="cit-name-given-names">M.S.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Yowell</span>,  <span class="cit-name-given-names">C.A.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Courtney</span>,  <span class="cit-name-given-names">C.H.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Dame</span>,  <span class="cit-name-given-names">J.B.</span></span> <span class="cit-article-title">Host movement and the genetic structure of populations of parasitic nematodes</span>. <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">141</span>, <span class="cit-fpage">1007</span>–<span class="cit-lpage">14</span> (<span class="cit-pub-date">1995</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DBlouin%26rft.auinit1%253DM.%2BS.%26rft.volume%253D141%26rft.issue%253D3%26rft.spage%253D1007%26rft.epage%253D1014%26rft.atitle%253DHost%2Bmovement%2Band%2Bthe%2Bgenetic%2Bstructure%2Bof%2Bpopulations%2Bof%2Bparasitic%2Bnematodes.%26rft_id%253Dinfo%253Apmid%252F8582607%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE0MS8zLzEwMDciO3M6NDoiYXRvbSI7czozNzoiL2Jpb3J4aXYvZWFybHkvMjAxOS8wNC8xMy80NTA2OTIuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">40.</span><a class="rev-xref-ref" href="#xref-ref-40-1" title="View reference 40. in text" id="ref-40">↵</a><div class="cit ref-cit ref-journal" id="cit-450692v2.40" data-doi="10.1016/j.ijpara.2006.06.015"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Troell</span>,  <span class="cit-name-given-names">K.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Engstrom</span>,  <span class="cit-name-given-names">A.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Morrison</span>,  <span class="cit-name-given-names">D.A.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Mattsson</span>,  <span class="cit-name-given-names">J.G.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Hoglund</span>,  <span class="cit-name-given-names">J.</span></span> <span class="cit-article-title">Global patterns reveal strong population structure in Haemonchus contortus, a nematode parasite of domesticated ruminants</span>. <abbr class="cit-jnl-abbrev">Int J Parasitol</abbr> <span class="cit-vol">36</span>, <span class="cit-fpage">1305</span>–<span class="cit-lpage">16</span> (<span class="cit-pub-date">2006</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DInternational%2Bjournal%2Bfor%2Bparasitology%26rft.stitle%253DInt%2BJ%2BParasitol%26rft.aulast%253DMuller%26rft.auinit1%253DG.%26rft.volume%253D36%26rft.issue%253D12%26rft.spage%253D1305%26rft.epage%253D1316%26rft.atitle%253DGlobal%2Bpatterns%2Breveal%2Bstrong%2Bpopulation%2Bstructure%2Bin%2BHaemonchus%2Bcontortus%252C%2Ba%2Bnematode%2Bparasite%2Bof%2Bdomesticated%2Bruminants.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.ijpara.2006.06.015%26rft_id%253Dinfo%253Apmid%252F16950266%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.ijpara.2006.06.015&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16950266&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F04%2F13%2F450692.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000241298800009&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">41.</span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-450692v2.41" data-doi="10.1073/pnas.1303057110"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Dey</span>,  <span class="cit-name-given-names">A.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Chan</span>,  <span class="cit-name-given-names">C.K.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Thomas</span>,  <span class="cit-name-given-names">C.G.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Cutter</span>,  <span class="cit-name-given-names">A.D.</span></span> <span class="cit-article-title">Molecular hyperdiversity defines populations of the nematode Caenorhabditis brenneri</span>. <abbr class="cit-jnl-abbrev">Proc Natl Acad Sci U S A</abbr> <span class="cit-vol">110</span>, <span class="cit-fpage">11056</span>–<span class="cit-lpage">60</span> (<span class="cit-pub-date">2013</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc%2BNatl%2BAcad%2BSci%2BU%2BS%2BA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.1303057110%26rft_id%253Dinfo%253Apmid%252F23776215%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMjoiMTEwLzI3LzExMDU2IjtzOjQ6ImF0b20iO3M6Mzc6Ii9iaW9yeGl2L2Vhcmx5LzIwMTkvMDQvMTMvNDUwNjkyLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">42.</span><a class="rev-xref-ref" href="#xref-ref-42-1" title="View reference 42. in text" id="ref-42">↵</a><div class="cit ref-cit ref-journal" id="cit-450692v2.42" data-doi="10.1016/bs.apar.2016.02.009"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Gilleard</span>,  <span class="cit-name-given-names">J.S.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Redman</span>,  <span class="cit-name-given-names">E.</span></span> <span class="cit-article-title">Genetic Diversity and Population Structure of Haemonchus contortus</span>. <abbr class="cit-jnl-abbrev">Adv Parasitol</abbr> <span class="cit-vol">93</span>, <span class="cit-fpage">31</span>–<span class="cit-lpage">68</span> (<span class="cit-pub-date">2016</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAdv%2BParasitol%26rft.volume%253D93%26rft.spage%253D31%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fbs.apar.2016.02.009%26rft_id%253Dinfo%253Apmid%252F27238002%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/bs.apar.2016.02.009&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=27238002&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F04%2F13%2F450692.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">43.</span><a class="rev-xref-ref" href="#xref-ref-43-1" title="View reference 43. in text" id="ref-43">↵</a><div class="cit ref-cit ref-journal" id="cit-450692v2.43" data-doi="10.1073/pnas.1323964111"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Larson</span>,  <span class="cit-name-given-names">G.</span></span> <span class="cit-etal">et al.</span> <span class="cit-article-title">Current perspectives and the future of domestication studies</span>. <abbr class="cit-jnl-abbrev">Proc Natl Acad Sci U S A</abbr> <span class="cit-vol">111</span>, <span class="cit-fpage">6139</span>–<span class="cit-lpage">46</span> (<span class="cit-pub-date">2014</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc%2BNatl%2BAcad%2BSci%2BU%2BS%2BA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.1323964111%26rft_id%253Dinfo%253Apmid%252F24757054%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMToiMTExLzE3LzYxMzkiO3M6NDoiYXRvbSI7czozNzoiL2Jpb3J4aXYvZWFybHkvMjAxOS8wNC8xMy80NTA2OTIuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">44.</span><a class="rev-xref-ref" href="#xref-ref-44-1" title="View reference 44. in text" id="ref-44">↵</a><div class="cit ref-cit ref-journal" id="cit-450692v2.44" data-doi="10.1371/journal.pone.0040340"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Pleurdeau</span>,  <span class="cit-name-given-names">D.</span></span> <span class="cit-etal">et al.</span> <span class="cit-article-title">"Of sheep and men": earliest direct evidence of caprine domestication in southern Africa at Leopard Cave (Erongo, Namibia)</span>. <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">7</span>, <span class="cit-fpage">e40340</span> (<span class="cit-pub-date">2012</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BOne%26rft.volume%253D7%26rft.spage%253De40340%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0040340%26rft_id%253Dinfo%253Apmid%252F22808138%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0040340&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22808138&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F04%2F13%2F450692.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">45.</span><a class="rev-xref-ref" href="#xref-ref-45-1" title="View reference 45. in text" id="ref-45">↵</a><div class="cit ref-cit ref-journal" id="cit-450692v2.45" data-doi="10.1016/j.cub.2009.11.050"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Campbell</span>,  <span class="cit-name-given-names">M.C.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Tishkoff</span>,  <span class="cit-name-given-names">S.A.</span></span> <span class="cit-article-title">The evolution of human genetic and phenotypic variation in Africa</span>. <abbr class="cit-jnl-abbrev">Curr Biol</abbr> <span class="cit-vol">20</span>, <span class="cit-fpage">R166</span>–<span class="cit-lpage">73</span> (<span class="cit-pub-date">2010</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCurrent%2Bbiology%2B%253A%2B%2BCB%26rft.stitle%253DCurr%2BBiol%26rft.aulast%253DCampbell%26rft.auinit1%253DM.%2BC.%26rft.volume%253D20%26rft.issue%253D4%26rft.spage%253DR166%26rft.epage%253DR173%26rft.atitle%253DThe%2Bevolution%2Bof%2Bhuman%2Bgenetic%2Band%2Bphenotypic%2Bvariation%2Bin%2BAfrica.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.cub.2009.11.050%26rft_id%253Dinfo%253Apmid%252F20178763%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.cub.2009.11.050&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20178763&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F04%2F13%2F450692.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000275149200020&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">46.</span><a class="rev-xref-ref" href="#xref-ref-46-1" title="View reference 46. in text" id="ref-46">↵</a><div class="cit ref-cit ref-journal" id="cit-450692v2.46" data-doi="10.1126/science.1170587"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Chessa</span>,  <span class="cit-name-given-names">B.</span></span> <span class="cit-etal">et al.</span> <span class="cit-article-title">Revealing the history of sheep domestication using retrovirus integrations</span>. <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">324</span>, <span class="cit-fpage">532</span>–<span class="cit-lpage">6</span> (<span class="cit-pub-date">2009</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DChessa%26rft.auinit1%253DB.%26rft.volume%253D324%26rft.issue%253D5926%26rft.spage%253D532%26rft.epage%253D536%26rft.atitle%253DRevealing%2Bthe%2BHistory%2Bof%2BSheep%2BDomestication%2BUsing%2BRetrovirus%2BIntegrations%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1170587%26rft_id%253Dinfo%253Apmid%252F19390051%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEyOiIzMjQvNTkyNi81MzIiO3M6NDoiYXRvbSI7czozNzoiL2Jpb3J4aXYvZWFybHkvMjAxOS8wNC8xMy80NTA2OTIuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">47.</span><a class="rev-xref-ref" href="#xref-ref-47-1" title="View reference 47. in text" id="ref-47">↵</a><div class="cit ref-cit ref-journal" id="cit-450692v2.47" data-doi="10.1093/molbev/msl043"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Tapio</span>,  <span class="cit-name-given-names">M.</span></span> <span class="cit-etal">et al.</span> <span class="cit-article-title">Sheep mitochondrial DNA variation in European, Caucasian, and Central Asian areas</span>. <abbr class="cit-jnl-abbrev">Mol Biol Evol</abbr> <span class="cit-vol">23</span>, <span class="cit-fpage">1776</span>–<span class="cit-lpage">83</span> (<span class="cit-pub-date">2006</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol%2BBiol%2BEvol%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fmolbev%252Fmsl043%26rft_id%253Dinfo%253Apmid%252F16782761%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/molbev/msl043&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16782761&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F04%2F13%2F450692.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000239905100016&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">48.</span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-450692v2.48" data-doi="10.1371/journal.pbio.1001258"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Kijas</span>,  <span class="cit-name-given-names">J.W.</span></span> <span class="cit-etal">et al.</span> <span class="cit-article-title">Genome-wide analysis of the world's sheep breeds reveals high levels of historic mixture and strong recent selection</span>. <abbr class="cit-jnl-abbrev">PLoS Biol</abbr> <span class="cit-vol">10</span>, <span class="cit-fpage">e1001258</span> (<span class="cit-pub-date">2012</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2Bbiology%26rft.stitle%253DPLoS%2BBiol%26rft.aulast%253DKijas%26rft.auinit1%253DJ.%2BW.%26rft.volume%253D10%26rft.issue%253D2%26rft.spage%253De1001258%26rft.epage%253De1001258%26rft.atitle%253DGenome-wide%2Banalysis%2Bof%2Bthe%2Bworld%2527s%2Bsheep%2Bbreeds%2Breveals%2Bhigh%2Blevels%2Bof%2Bhistoric%2Bmixture%2Band%2Bstrong%2Brecent%2Bselection.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pbio.1001258%26rft_id%253Dinfo%253Apmid%252F22346734%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pbio.1001258&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22346734&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F04%2F13%2F450692.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">49.</span><a class="rev-xref-ref" href="#xref-ref-49-1" title="View reference 49. in text" id="ref-49">↵</a><div class="cit ref-cit ref-journal" id="cit-450692v2.49"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Spangler</span>,  <span class="cit-name-given-names">G.L.</span></span> <span class="cit-etal">et al.</span> <span class="cit-article-title">Whole genome structural analysis of Caribbean hair sheep reveals quantitative link to West African ancestry</span>. <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">12</span>, <span class="cit-fpage">e0179021</span> (<span class="cit-pub-date">2017</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BOne%26rft.volume%253D12%26rft.spage%253De0179021%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">50.</span><a class="rev-xref-ref" href="#xref-ref-50-1" title="View reference 50. in text" id="ref-50">↵</a><div class="cit ref-cit ref-journal" id="cit-450692v2.50"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Naves</span>,  <span class="cit-name-given-names">M.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Alexandre</span>,  <span class="cit-name-given-names">G.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Leimbacher</span>,  <span class="cit-name-given-names">F.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Mandonnet</span>,  <span class="cit-name-given-names">N.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Menendez-buxadera</span>,  <span class="cit-name-given-names">A.</span></span> <span class="cit-article-title">Les ruminants domestiques de la Caraïbe: le point sur les ressources génétiques et leur exploitation</span>. <abbr class="cit-jnl-abbrev">INRA Prod. Anim.</abbr> <span class="cit-vol">14</span>, <span class="cit-fpage">181</span>–<span class="cit-lpage">192</span> (<span class="cit-pub-date">2001</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DINRA%2BProd.%2BAnim.%26rft.volume%253D14%26rft.spage%253D181%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">51.</span><a class="rev-xref-ref" href="#xref-ref-51-1" title="View reference 51. in text" id="ref-51">↵</a><div class="cit ref-cit ref-book" id="cit-450692v2.51"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Roman</span>,  <span class="cit-name-given-names">A.</span></span> <span class="cit-source">Saint Malo au temps des négriers</span>, <span class="cit-fpage">357</span> (<span class="cit-publ-loc">Karthala</span>, <span class="cit-pub-date">2003</span>).</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">52.</span><a class="rev-xref-ref" href="#xref-ref-52-1" title="View reference 52. in text" id="ref-52">↵</a><div class="cit ref-cit ref-journal" id="cit-450692v2.52" data-doi="10.1093/molbev/msl007"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Pereira</span>,  <span class="cit-name-given-names">F.</span></span> <span class="cit-etal">et al.</span> <span class="cit-article-title">Genetic signatures of a Mediterranean influence in Iberian Peninsula sheep husbandry</span>. <abbr class="cit-jnl-abbrev">Mol Biol Evol</abbr> <span class="cit-vol">23</span>, <span class="cit-fpage">1420</span>–<span class="cit-lpage">6</span> (<span class="cit-pub-date">2006</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol%2BBiol%2BEvol%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fmolbev%252Fmsl007%26rft_id%253Dinfo%253Apmid%252F16672283%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/molbev/msl007&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16672283&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F04%2F13%2F450692.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000238545100011&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">53.</span><a class="rev-xref-ref" href="#xref-ref-53-1" title="View reference 53. in text" id="ref-53">↵</a><div class="cit ref-cit ref-journal" id="cit-450692v2.53"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Redman</span>,  <span class="cit-name-given-names">E.</span></span> <span class="cit-etal">et al.</span> <span class="cit-article-title">The emergence of resistance to the benzimidazole anthlemintics in parasitic nematodes of livestock is characterised by multiple independent hard and soft selective sweeps</span>. <abbr class="cit-jnl-abbrev">PLoS Negl Trop Dis</abbr> <span class="cit-vol">9</span>, <span class="cit-fpage">e0003494</span> (<span class="cit-pub-date">2015</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BNegl%2BTrop%2BDis%26rft.volume%253D9%26rft.spage%253De0003494%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">54.</span><a class="rev-xref-ref" href="#xref-ref-54-1" title="View reference 54. in text" id="ref-54">↵</a><div class="cit ref-cit ref-journal" id="cit-450692v2.54"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Vercruysse</span>,  <span class="cit-name-given-names">J.</span></span> <span class="cit-etal">et al.</span> <span class="cit-article-title">Is anthelmintic resistance a concern for the control of human soil-transmitted helminths?</span> <abbr class="cit-jnl-abbrev">Int J Parasitol Drugs Drug Resist</abbr> <span class="cit-vol">1</span>, <span class="cit-fpage">14</span>–<span class="cit-lpage">27</span> (<span class="cit-pub-date">2011</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DInt%2BJ%2BParasitol%2BDrugs%2BDrug%2BResist%26rft.volume%253D1%26rft.spage%253D14%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">55.</span><a class="rev-xref-ref" href="#xref-ref-55-1" title="View reference 55. in text" id="ref-55">↵</a><div class="cit ref-cit ref-journal" id="cit-450692v2.55" data-doi="10.1016/j.pt.2017.02.004"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Laing</span>,  <span class="cit-name-given-names">R.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Gillan</span>,  <span class="cit-name-given-names">V.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Devaney</span>,  <span class="cit-name-given-names">E.</span></span> <span class="cit-article-title">Ivermectin – Old Drug, New Tricks?</span> <abbr class="cit-jnl-abbrev">Trends Parasitol</abbr> <span class="cit-vol">33</span>, <span class="cit-fpage">463</span>–<span class="cit-lpage">472</span> (<span class="cit-pub-date">2017</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTrends%2BParasitol%26rft.volume%253D33%26rft.spage%253D463%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.pt.2017.02.004%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.pt.2017.02.004&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label">56.</span><a class="rev-xref-ref" href="#xref-ref-56-1" title="View reference 56. in text" id="ref-56">↵</a><div class="cit ref-cit ref-journal" id="cit-450692v2.56" data-doi="10.1371/journal.pntd.0000998"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Osei-Atweneboana</span>,  <span class="cit-name-given-names">M.Y.</span></span> <span class="cit-etal">et al.</span> <span class="cit-article-title">Phenotypic evidence of emerging ivermectin resistance in Onchocerca volvulus</span>. <abbr class="cit-jnl-abbrev">PLoS Negl Trop Dis</abbr> <span class="cit-vol">5</span>, <span class="cit-fpage">e998</span> (<span class="cit-pub-date">2011</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BNegl%2BTrop%2BDis%26rft.aulast%253DOsei-Atweneboana%26rft.auinit1%253DM.%2BY.%26rft.volume%253D5%26rft.issue%253D3%26rft.spage%253De998%26rft.epage%253De998%26rft.atitle%253DPhenotypic%2Bevidence%2Bof%2Bemerging%2Bivermectin%2Bresistance%2Bin%2BOnchocerca%2Bvolvulus.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pntd.0000998%26rft_id%253Dinfo%253Apmid%252F21468315%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pntd.0000998&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21468315&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F04%2F13%2F450692.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">57.</span><a class="rev-xref-ref" href="#xref-ref-57-1" title="View reference 57. in text" id="ref-57">↵</a><div class="cit ref-cit ref-journal" id="cit-450692v2.57"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Doyle</span>,  <span class="cit-name-given-names">S.R.</span></span> <span class="cit-etal">et al.</span> <span class="cit-article-title">Genome-wide analysis of ivermectin response by Onchocerca volvulus reveals that genetic drift and soft selective sweeps contribute to loss of drug sensitivity</span>. <abbr class="cit-jnl-abbrev">PLoS Negl Trop Dis</abbr> <span class="cit-vol">11</span>, <span class="cit-fpage">e0005816</span> (<span class="cit-pub-date">2017</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BNegl%2BTrop%2BDis%26rft.volume%253D11%26rft.spage%253De0005816%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">58.</span><a class="rev-xref-ref" href="#xref-ref-58-1" title="View reference 58. in text" id="ref-58">↵</a><div class="cit ref-cit ref-journal" id="cit-450692v2.58"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Rose</span>,  <span class="cit-name-given-names">H.</span></span> <span class="cit-etal">et al.</span> <span class="cit-article-title">Climate-driven changes to the spatio-temporal distribution of the parasitic nematode, Haemonchus contortus, in sheep in Europe</span>. <abbr class="cit-jnl-abbrev">Glob Chang Biol</abbr> <span class="cit-vol">22</span>, <span class="cit-fpage">1271</span>–<span class="cit-lpage">85</span> (<span class="cit-pub-date">2016</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGlob%2BChang%2BBiol%26rft.volume%253D22%26rft.spage%253D1271%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">59.</span><a class="rev-xref-ref" href="#xref-ref-59-1" title="View reference 59. in text" id="ref-59">↵</a><div class="cit ref-cit ref-journal" id="cit-450692v2.59"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">van Dijk</span>,  <span class="cit-name-given-names">J.</span></span>, <span class="cit-auth"><span class="cit-name-surname">David</span>,  <span class="cit-name-given-names">G.P.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Baird</span>,  <span class="cit-name-given-names">G.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Morgan</span>,  <span class="cit-name-given-names">E.R.</span></span> <span class="cit-article-title">Back to the future: developing hypotheses on the effects of climate change on ovine parasitic gastroenteritis from historical data</span>. <abbr class="cit-jnl-abbrev">Vet Parasitol</abbr> <span class="cit-vol">158</span>, <span class="cit-fpage">73</span>–<span class="cit-lpage">84</span> (<span class="cit-pub-date">2008</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DVeterinary%2Bparasitology%26rft.stitle%253DVet%2BParasitol%26rft.aulast%253Dvan%2BDijk%26rft.auinit1%253DJ.%26rft.volume%253D158%26rft.issue%253D1-2%26rft.spage%253D73%26rft.epage%253D84%26rft.atitle%253DBack%2Bto%2Bthe%2Bfuture%253A%2Bdeveloping%2Bhypotheses%2Bon%2Bthe%2Beffects%2Bof%2Bclimate%2Bchange%2Bon%2Bovine%2Bparasitic%2Bgastroenteritis%2Bfrom%2Bhistorical%2Bdata.%26rft_id%253Dinfo%253Apmid%252F18824303%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=18824303&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F04%2F13%2F450692.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">60.</span><a class="rev-xref-ref" href="#xref-ref-60-1" title="View reference 60. in text" id="ref-60">↵</a><div class="cit ref-cit ref-journal" id="cit-450692v2.60" data-doi="10.1111/mec.12289"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Rajpurohit</span>,  <span class="cit-name-given-names">S.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Oliveira</span>,  <span class="cit-name-given-names">C.C.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Etges</span>,  <span class="cit-name-given-names">W.J.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Gibbs</span>,  <span class="cit-name-given-names">A.G.</span></span> <span class="cit-article-title">Functional genomic and phenotypic responses to desiccation in natural populations of a desert drosophilid</span>. <abbr class="cit-jnl-abbrev">Mol Ecol</abbr> <span class="cit-vol">22</span>, <span class="cit-fpage">2698</span>–<span class="cit-lpage">715</span> (<span class="cit-pub-date">2013</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol%2BEcol%26rft.volume%253D22%26rft.spage%253D2698%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fmec.12289%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/mec.12289&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000318180000008&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">61.</span><a class="rev-xref-ref" href="#xref-ref-61-1" title="View reference 61. in text" id="ref-61">↵</a><div class="cit ref-cit ref-journal" id="cit-450692v2.61"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Sharma</span>,  <span class="cit-name-given-names">V.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Kohli</span>,  <span class="cit-name-given-names">S.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Brahmachari</span>,  <span class="cit-name-given-names">V.</span></span> <span class="cit-article-title">Correlation between desiccation stress response and epigenetic modifications of genes in Drosophila melanogaster: An example of environment-epigenome interaction</span>. <abbr class="cit-jnl-abbrev">Biochim Biophys Acta</abbr> <span class="cit-vol">1860</span>, <span class="cit-fpage">1058</span>–<span class="cit-lpage">1068</span> (<span class="cit-pub-date">2017</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBiochim%2BBiophys%2BActa%26rft.volume%253D1860%26rft.spage%253D1058%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">62.</span><a class="rev-xref-ref" href="#xref-ref-62-1" title="View reference 62. in text" id="ref-62">↵</a><div class="cit ref-cit ref-journal" id="cit-450692v2.62" data-doi="10.1126/science.1221881"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Coustham</span>,  <span class="cit-name-given-names">V.</span></span> <span class="cit-etal">et al.</span> <span class="cit-article-title">Quantitative modulation of polycomb silencing underlies natural variation in vernalization</span>. <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">337</span>, <span class="cit-fpage">584</span>–<span class="cit-lpage">7</span> (<span class="cit-pub-date">2012</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1221881%26rft_id%253Dinfo%253Apmid%252F22798408%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEyOiIzMzcvNjA5NC81ODQiO3M6NDoiYXRvbSI7czozNzoiL2Jpb3J4aXYvZWFybHkvMjAxOS8wNC8xMy80NTA2OTIuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">63.</span><a class="rev-xref-ref" href="#xref-ref-63-1" title="View reference 63. in text" id="ref-63">↵</a><div class="cit ref-cit ref-journal" id="cit-450692v2.63" data-doi="10.1016/0012-1606(75)90109-8"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Cassada</span>,  <span class="cit-name-given-names">R.C.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Russell</span>,  <span class="cit-name-given-names">R.L.</span></span> <span class="cit-article-title">The dauerlarva, a post-embryonic developmental variant of the nematode Caenorhabditis elegans</span>. <abbr class="cit-jnl-abbrev">Dev Biol</abbr> <span class="cit-vol">46</span>, <span class="cit-fpage">326</span>–<span class="cit-lpage">42</span> (<span class="cit-pub-date">1975</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DDevelopmental%2Bbiology%26rft.stitle%253DDev%2BBiol%26rft.aulast%253DCassada%26rft.auinit1%253DR.%2BC.%26rft.volume%253D46%26rft.issue%253D2%26rft.spage%253D326%26rft.epage%253D342%26rft.atitle%253DThe%2Bdauerlarva%252C%2Ba%2Bpost-embryonic%2Bdevelopmental%2Bvariant%2Bof%2Bthe%2Bnematode%2BCaenorhabditis%2Belegans.%26rft_id%253Dinfo%253Adoi%252F10.1016%252F0012-1606%252875%252990109-8%26rft_id%253Dinfo%253Apmid%252F1183723%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/0012-1606(75)90109-8&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=1183723&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F04%2F13%2F450692.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1975AS22600008&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">64.</span><a class="rev-xref-ref" href="#xref-ref-64-1" title="View reference 64. in text" id="ref-64">↵</a><div class="cit ref-cit ref-journal" id="cit-450692v2.64" data-doi="10.1016/0020-7519(72)90028-8"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Blitz</span>,  <span class="cit-name-given-names">N.M.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Gibbs</span>,  <span class="cit-name-given-names">H.C.</span></span> <span class="cit-article-title">Studies on the arrested development of Haemonchus contortus in sheep. I. The induction of arrested development</span>. <abbr class="cit-jnl-abbrev">Int J Parasitol</abbr> <span class="cit-vol">2</span>, <span class="cit-fpage">5</span>–<span class="cit-lpage">12</span> (<span class="cit-pub-date">1972</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DInternational%2Bjournal%2Bfor%2Bparasitology%26rft.stitle%253DInt%2BJ%2BParasitol%26rft.aulast%253DBlitz%26rft.auinit1%253DN.%2BM.%26rft.volume%253D2%26rft.issue%253D1%26rft.spage%253D5%26rft.epage%253D12%26rft.atitle%253DStudies%2Bon%2Bthe%2Barrested%2Bdevelopment%2Bof%2BHaemonchus%2Bcontortus%2Bin%2Bsheep.%2BI.%2BThe%2Binduction%2Bof%2Barrested%2Bdevelopment.%26rft_id%253Dinfo%253Adoi%252F10.1016%252F0020-7519%252872%252990028-8%26rft_id%253Dinfo%253Apmid%252F4652919%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/0020-7519(72)90028-8&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=4652919&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F04%2F13%2F450692.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">65.</span><a class="rev-xref-ref" href="#xref-ref-65-1" title="View reference 65. in text" id="ref-65">↵</a><div class="cit ref-cit ref-journal" id="cit-450692v2.65" data-doi="10.1186/gb-2014-15-3-r46"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Wood</span>,  <span class="cit-name-given-names">D.E.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Salzberg</span>,  <span class="cit-name-given-names">S.L.</span></span> <span class="cit-article-title">Kraken: ultrafast metagenomic sequence classification using exact alignments</span>. <abbr class="cit-jnl-abbrev">Genome Biol</abbr> <span class="cit-vol">15</span>, <span class="cit-fpage">R46</span> (<span class="cit-pub-date">2014</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BBiol%26rft.volume%253D15%26rft.spage%253DR46%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fgb-2014-15-3-r46%26rft_id%253Dinfo%253Apmid%252F24580807%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/gb-2014-15-3-r46&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24580807&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F04%2F13%2F450692.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">66.</span><a class="rev-xref-ref" href="#xref-ref-66-1" title="View reference 66. in text" id="ref-66">↵</a><div class="cit ref-cit ref-journal" id="cit-450692v2.66" data-doi="10.1186/1471-2164-10-267"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Li</span>,  <span class="cit-name-given-names">B.W.</span></span> <span class="cit-etal">et al.</span> <span class="cit-article-title">Transcriptomes and pathways associated with infectivity, survival and immunogenicity in Brugia malayi L3</span>. <abbr class="cit-jnl-abbrev">BMC Genomics</abbr> <span class="cit-vol">10</span>, <span class="cit-fpage">267</span> (<span class="cit-pub-date">2009</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2Bgenomics%2B%255Belectronic%2Bresource%255D%26rft.stitle%253DBMC%2BGenomics%26rft.aulast%253DLi%26rft.auinit1%253DB.%2BW.%26rft.volume%253D10%26rft.spage%253D267%26rft.epage%253D267%26rft.atitle%253DTranscriptomes%2Band%2Bpathways%2Bassociated%2Bwith%2Binfectivity%252C%2Bsurvival%2Band%2Bimmunogenicity%2Bin%2BBrugia%2Bmalayi%2BL3.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2164-10-267%26rft_id%253Dinfo%253Apmid%252F19527522%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2164-10-267&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19527522&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F04%2F13%2F450692.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">67.</span><a class="rev-xref-ref" href="#xref-ref-67-1" title="View reference 67. in text" id="ref-67">↵</a><div class="cit ref-cit ref-journal" id="cit-450692v2.67" data-doi="10.1038/ng.806"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">DePristo</span>,  <span class="cit-name-given-names">M.A.</span></span> <span class="cit-etal">et al.</span> <span class="cit-article-title">A framework for variation discovery and genotyping using next-generation DNA sequencing data</span>. <abbr class="cit-jnl-abbrev">Nat Genet</abbr> <span class="cit-vol">43</span>, <span class="cit-fpage">491</span>–<span class="cit-lpage">8</span> (<span class="cit-pub-date">2011</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bgenetics%26rft.stitle%253DNat%2BGenet%26rft.aulast%253DDepristo%26rft.auinit1%253DM.%2BA.%26rft.volume%253D43%26rft.issue%253D5%26rft.spage%253D491%26rft.epage%253D498%26rft.atitle%253DA%2Bframework%2Bfor%2Bvariation%2Bdiscovery%2Band%2Bgenotyping%2Busing%2Bnext-generation%2BDNA%2Bsequencing%2Bdata.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fng.806%26rft_id%253Dinfo%253Apmid%252F21478889%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ng.806&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21478889&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F04%2F13%2F450692.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000289972600023&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">68.</span><a class="rev-xref-ref" href="#xref-ref-68-1" title="View reference 68. in text" id="ref-68">↵</a><div class="cit ref-cit ref-journal" id="cit-450692v2.68"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Garrison</span>,  <span class="cit-name-given-names">E.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Marth</span>,  <span class="cit-name-given-names">G.</span></span> <span class="cit-article-title">Haplotype-based variant detection from short-read sequencing</span>. <abbr class="cit-jnl-abbrev">arXiv preprint</abbr><span class="cit-pub-id-sep cit-pub-id-arxiv-sep"> </span><span class="cit-pub-id-scheme">arXiv:</span><span class="cit-pub-id cit-pub-id-arxiv">1207.3907</span> [q-bio.GN](<span class="cit-pub-date">2012</span>).</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">69.</span><a class="rev-xref-ref" href="#xref-ref-69-1" title="View reference 69. in text" id="ref-69">↵</a><div class="cit ref-cit ref-journal" id="cit-450692v2.69" data-doi="10.1186/s12859-014-0356-4"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Korneliussen</span>,  <span class="cit-name-given-names">T.S.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Albrechtsen</span>,  <span class="cit-name-given-names">A.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Nielsen</span>,  <span class="cit-name-given-names">R.</span></span> <span class="cit-article-title">ANGSD: Analysis of Next Generation Sequencing Data</span>. <abbr class="cit-jnl-abbrev">BMC Bioinformatics</abbr> <span class="cit-vol">15</span>, <span class="cit-fpage">356</span> (<span class="cit-pub-date">2014</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2BBioinformatics%26rft.volume%253D15%26rft.spage%253D356%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fs12859-014-0356-4%26rft_id%253Dinfo%253Apmid%252F25420514%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/s12859-014-0356-4&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25420514&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F04%2F13%2F450692.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">70.</span><a class="rev-xref-ref" href="#xref-ref-70-1" title="View reference 70. in text" id="ref-70">↵</a><div class="cit ref-cit ref-journal" id="cit-450692v2.70" data-doi="10.1101/gr.107524.110"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">McKenna</span>,  <span class="cit-name-given-names">A.</span></span> <span class="cit-etal">et al.</span> <span class="cit-article-title">The Genome Analysis Toolkit: a MapReduce framework for analyzing next-generation DNA sequencing data</span>. <abbr class="cit-jnl-abbrev">Genome Res</abbr> <span class="cit-vol">20</span>, <span class="cit-fpage">1297</span>–<span class="cit-lpage">303</span> (<span class="cit-pub-date">2010</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BRes%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.107524.110%26rft_id%253Dinfo%253Apmid%252F20644199%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjk6IjIwLzkvMTI5NyI7czo0OiJhdG9tIjtzOjM3OiIvYmlvcnhpdi9lYXJseS8yMDE5LzA0LzEzLzQ1MDY5Mi5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">71.</span><a class="rev-xref-ref" href="#xref-ref-71-1" title="View reference 71. in text" id="ref-71">↵</a><div class="cit ref-cit ref-journal" id="cit-450692v2.71" data-doi="10.1093/bioinformatics/btp352"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Li</span>,  <span class="cit-name-given-names">H.</span></span> <span class="cit-etal">et al.</span> <span class="cit-article-title">The Sequence Alignment/Map format and SAMtools</span>. <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">25</span>, <span class="cit-fpage">2078</span>–<span class="cit-lpage">9</span> (<span class="cit-pub-date">2009</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtp352%26rft_id%253Dinfo%253Apmid%252F19505943%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/btp352&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19505943&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F04%2F13%2F450692.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000268808600014&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">72.</span><a class="rev-xref-ref" href="#xref-ref-72-1" title="View reference 72. in text" id="ref-72">↵</a><div class="cit ref-cit ref-journal" id="cit-450692v2.72"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Rensch</span>,  <span class="cit-name-given-names">T.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Villar</span>,  <span class="cit-name-given-names">D.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Horvath</span>,  <span class="cit-name-given-names">J.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Odom</span>,  <span class="cit-name-given-names">D.T.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Flicek</span>,  <span class="cit-name-given-names">P.</span></span> <span class="cit-article-title">Mitochondrial heteroplasmy in vertebrates using ChIP-sequencing data</span>. <abbr class="cit-jnl-abbrev">Genome Biol</abbr> <span class="cit-vol">17</span>, <span class="cit-fpage">139</span> (<span class="cit-pub-date">2016</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BBiol%26rft.volume%253D17%26rft.spage%253D139%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">73.</span><a class="rev-xref-ref" href="#xref-ref-73-1" title="View reference 73. in text" id="ref-73">↵</a><div class="cit ref-cit ref-journal" id="cit-450692v2.73"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Howe</span>,  <span class="cit-name-given-names">D.K.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Baer</span>,  <span class="cit-name-given-names">C.F.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Denver</span>,  <span class="cit-name-given-names">D.R.</span></span> <span class="cit-article-title">High rate of large deletions in Caenorhabditis briggsae mitochondrial genome mutation processes</span>. <abbr class="cit-jnl-abbrev">Genome Biol Evol</abbr> <span class="cit-vol">2</span>, <span class="cit-fpage">29</span>–<span class="cit-lpage">38</span> (<span class="cit-pub-date">2009</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BBiol%2BEvol%26rft.volume%253D2%26rft.spage%253D29%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">74.</span><a class="rev-xref-ref" href="#xref-ref-74-1" title="View reference 74. in text" id="ref-74">↵</a><div class="cit ref-cit ref-journal" id="cit-450692v2.74"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Konrad</span>,  <span class="cit-name-given-names">A.</span></span> <span class="cit-etal">et al.</span> <span class="cit-article-title">Mitochondrial Mutation Rate, Spectrum and Heteroplasmy in Caenorhabditis elegans Spontaneous Mutation Accumulation Lines of Differing Population Size</span>. <abbr class="cit-jnl-abbrev">Mol Biol Evol</abbr> <span class="cit-vol">34</span>, <span class="cit-fpage">1319</span>–<span class="cit-lpage">1334</span> (<span class="cit-pub-date">2017</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol%2BBiol%2BEvol%26rft.volume%253D34%26rft.spage%253D1319%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">75.</span><a class="rev-xref-ref" href="#xref-ref-75-1" title="View reference 75. in text" id="ref-75">↵</a><div class="cit ref-cit ref-journal" id="cit-450692v2.75" data-doi="10.1371/journal.pgen.1000834"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Hazkani-Covo</span>,  <span class="cit-name-given-names">E.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Zeller</span>,  <span class="cit-name-given-names">R.M.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Martin</span>,  <span class="cit-name-given-names">W.</span></span> <span class="cit-article-title">Molecular poltergeists: mitochondrial DNA copies (numts) in sequenced nuclear genomes</span>. <abbr class="cit-jnl-abbrev">PLoS Genet</abbr> <span class="cit-vol">6</span>, <span class="cit-fpage">e1000834</span> (<span class="cit-pub-date">2010</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BGenet%26rft.aulast%253DHazkani-Covo%26rft.auinit1%253DE.%26rft.volume%253D6%26rft.issue%253D2%26rft.spage%253De1000834%26rft.epage%253De1000834%26rft.atitle%253DMolecular%2Bpoltergeists%253A%2Bmitochondrial%2BDNA%2Bcopies%2B%2528numts%2529%2Bin%2Bsequenced%2Bnuclear%2Bgenomes.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.1000834%26rft_id%253Dinfo%253Apmid%252F20168995%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.1000834&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20168995&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F04%2F13%2F450692.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">76.</span><a class="rev-xref-ref" href="#xref-ref-76-1" title="View reference 76. in text" id="ref-76">↵</a><div class="cit ref-cit ref-journal" id="cit-450692v2.76" data-doi="10.1093/bioinformatics/btr330"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Danecek</span>,  <span class="cit-name-given-names">P.</span></span> <span class="cit-etal">et al.</span> <span class="cit-article-title">The variant call format and VCFtools</span>. <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">27</span>, <span class="cit-fpage">2156</span>–<span class="cit-lpage">8</span> (<span class="cit-pub-date">2011</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtr330%26rft_id%253Dinfo%253Apmid%252F21653522%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/btr330&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21653522&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F04%2F13%2F450692.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000292778700023&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">77.</span><a class="rev-xref-ref" href="#xref-ref-77-1" title="View reference 77. in text" id="ref-77">↵</a><div class="cit ref-cit ref-journal" id="cit-450692v2.77" data-doi="10.1093/bioinformatics/bts606"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Zheng</span>,  <span class="cit-name-given-names">X.</span></span> <span class="cit-etal">et al.</span> <span class="cit-article-title">A high-performance computing toolset for relatedness and principal component analysis of SNP data</span>. <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">28</span>, <span class="cit-fpage">3326</span>–<span class="cit-lpage">8</span> (<span class="cit-pub-date">2012</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbts606%26rft_id%253Dinfo%253Apmid%252F23060615%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/bts606&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23060615&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F04%2F13%2F450692.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000312105300026&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">78.</span><a class="rev-xref-ref" href="#xref-ref-78-1" title="View reference 78. in text" id="ref-78">↵</a><div class="cit ref-cit ref-journal" id="cit-450692v2.78"><div class="cit-metadata"><cite><span class="cit-auth cit-collab">R Core Team</span>. <span class="cit-article-title">R: A Language and Environment for Statistical Computing</span>. (<abbr class="cit-jnl-abbrev">R Foundation for Statistical Computing, Vienna</abbr>, <span class="cit-pub-date">2016</span>).</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">79.</span><a class="rev-xref-ref" href="#xref-ref-79-1" title="View reference 79. in text" id="ref-79">↵</a><div class="cit ref-cit ref-journal" id="cit-450692v2.79" data-doi="10.2307/2408641"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Weir</span>,  <span class="cit-name-given-names">B.S.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Cockerham</span>,  <span class="cit-name-given-names">C.C.</span></span> <span class="cit-article-title">Estimating F-Statistics for the Analysis of Population Structure</span>. <abbr class="cit-jnl-abbrev">Evolution</abbr> <span class="cit-vol">38</span>, <span class="cit-fpage">1358</span>–<span class="cit-lpage">1370</span> (<span class="cit-pub-date">1984</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEvolution%26rft.volume%253D38%26rft.spage%253D1358%26rft_id%253Dinfo%253Adoi%252F10.2307%252F2408641%26rft_id%253Dinfo%253Apmid%252F28563791%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2307/2408641&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=28563791&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F04%2F13%2F450692.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1984TY40400017&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">80.</span><a class="rev-xref-ref" href="#xref-ref-80-1" title="View reference 80. in text" id="ref-80">↵</a><div class="cit ref-cit ref-journal" id="cit-450692v2.80" data-doi="10.1038/ng.3980"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Svardal</span>,  <span class="cit-name-given-names">H.</span></span> <span class="cit-etal">et al.</span> <span class="cit-article-title">Ancient hybridization and strong adaptation to viruses across African vervet monkey populations</span>. <abbr class="cit-jnl-abbrev">Nat Genet</abbr> <span class="cit-vol">49</span>, <span class="cit-fpage">1705</span>–<span class="cit-lpage">1713</span> (<span class="cit-pub-date">2017</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat%2BGenet%26rft.volume%253D49%26rft.spage%253D1705%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fng.3980%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ng.3980&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label">81.</span><a class="rev-xref-ref" href="#xref-ref-81-1" title="View reference 81. in text" id="ref-81">↵</a><div class="cit ref-cit ref-journal" id="cit-450692v2.81" data-doi="10.1186/s13742-015-0047-8"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Chang</span>,  <span class="cit-name-given-names">C.C.</span></span> <span class="cit-etal">et al.</span> <span class="cit-article-title">Second-generation PLINK: rising to the challenge of larger and richer datasets</span>. <abbr class="cit-jnl-abbrev">Gigascience</abbr> <span class="cit-vol">4</span>, <span class="cit-fpage">7</span> (<span class="cit-pub-date">2015</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGigascience%26rft.volume%253D4%26rft.spage%253D7%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fs13742-015-0047-8%26rft_id%253Dinfo%253Apmid%252F25722852%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/s13742-015-0047-8&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25722852&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F04%2F13%2F450692.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">82.</span><a class="rev-xref-ref" href="#xref-ref-82-1" title="View reference 82. in text" id="ref-82">↵</a><div class="cit ref-cit ref-journal" id="cit-450692v2.82" data-doi="10.1093/bioinformatics/btg412"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Paradis</span>,  <span class="cit-name-given-names">E.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Claude</span>,  <span class="cit-name-given-names">J.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Strimmer</span>,  <span class="cit-name-given-names">K.</span></span> <span class="cit-article-title">APE: Analyses of Phylogenetics and Evolution in R language</span>. <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">20</span>, <span class="cit-fpage">289</span>–<span class="cit-lpage">90</span> (<span class="cit-pub-date">2004</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft.stitle%253DBioinformatics%26rft.aulast%253DParadis%26rft.auinit1%253DE.%26rft.volume%253D20%26rft.issue%253D2%26rft.spage%253D289%26rft.epage%253D290%26rft.atitle%253DAPE%253A%2BAnalyses%2Bof%2BPhylogenetics%2Band%2BEvolution%2Bin%2BR%2Blanguage%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtg412%26rft_id%253Dinfo%253Apmid%252F14734327%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/btg412&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=14734327&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F04%2F13%2F450692.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000188389700026&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">83.</span><a class="rev-xref-ref" href="#xref-ref-83-1" title="View reference 83. in text" id="ref-83">↵</a><div class="cit ref-cit ref-journal" id="cit-450692v2.83" data-doi="10.1093/nar/gkh340"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Edgar</span>,  <span class="cit-name-given-names">R.C.</span></span> <span class="cit-article-title">MUSCLE: multiple sequence alignment with high accuracy and high throughput</span>. <abbr class="cit-jnl-abbrev">Nucleic Acids Res</abbr> <span class="cit-vol">32</span>, <span class="cit-fpage">1792</span>–<span class="cit-lpage">7</span> (<span class="cit-pub-date">2004</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BResearch%26rft.stitle%253DNucleic%2BAcids%2BRes%26rft.aulast%253DEdgar%26rft.auinit1%253DR.%2BC.%26rft.volume%253D32%26rft.issue%253D5%26rft.spage%253D1792%26rft.epage%253D1797%26rft.atitle%253DMUSCLE%253A%2Bmultiple%2Bsequence%2Balignment%2Bwith%2Bhigh%2Baccuracy%2Band%2Bhigh%2Bthroughput%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgkh340%26rft_id%253Dinfo%253Apmid%252F15034147%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/nar/gkh340&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15034147&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F04%2F13%2F450692.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000220487200025&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">84.</span><a class="rev-xref-ref" href="#xref-ref-84-1" title="View reference 84. in text" id="ref-84">↵</a><div class="cit ref-cit ref-journal" id="cit-450692v2.84" data-doi="10.1080/10635150701472164"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Talavera</span>,  <span class="cit-name-given-names">G.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Castresana</span>,  <span class="cit-name-given-names">J.</span></span> <span class="cit-article-title">Improvement of phylogenies after removing divergent and ambiguously aligned blocks from protein sequence alignments</span>. <abbr class="cit-jnl-abbrev">Syst Biol</abbr> <span class="cit-vol">56</span>, <span class="cit-fpage">564</span>–<span class="cit-lpage">77</span> (<span class="cit-pub-date">2007</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DSystematic%2BBiology%26rft.stitle%253DSyst%2BBiol%26rft.aulast%253DTalavera%26rft.auinit1%253DG.%26rft.volume%253D56%26rft.issue%253D4%26rft.spage%253D564%26rft.epage%253D577%26rft.atitle%253DImprovement%2Bof%2Bphylogenies%2Bafter%2Bremoving%2Bdivergent%2Band%2Bambiguously%2Baligned%2Bblocks%2Bfrom%2Bprotein%2Bsequence%2Balignments.%26rft_id%253Dinfo%253Adoi%252F10.1080%252F10635150701472164%26rft_id%253Dinfo%253Apmid%252F17654362%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1080/10635150701472164&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17654362&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F04%2F13%2F450692.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000248359900002&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">85.</span><a class="rev-xref-ref" href="#xref-ref-85-1" title="View reference 85. in text" id="ref-85">↵</a><div class="cit ref-cit ref-journal" id="cit-450692v2.85" data-doi="10.1186/1471-2148-6-29"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Keane</span>,  <span class="cit-name-given-names">T.M.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Creevey</span>,  <span class="cit-name-given-names">C.J.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Pentony</span>,  <span class="cit-name-given-names">M.M.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Naughton</span>,  <span class="cit-name-given-names">T.J.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">McLnerney</span>,  <span class="cit-name-given-names">J.O.</span></span> <span class="cit-article-title">Assessment of methods for amino acid matrix selection and their use on empirical data shows that ad hoc assumptions for choice of matrix are not justified</span>. <abbr class="cit-jnl-abbrev">BMC Evol Biol</abbr> <span class="cit-vol">6</span>, <span class="cit-fpage">29</span> (<span class="cit-pub-date">2006</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2Bevolutionary%2Bbiology%2B%255Belectronic%2Bresource%255D%26rft.stitle%253DBMC%2BEvol%2BBiol%26rft.aulast%253DKeane%26rft.auinit1%253DT.%2BM.%26rft.volume%253D6%26rft.spage%253D29%26rft.epage%253D29%26rft.atitle%253DAssessment%2Bof%2Bmethods%2Bfor%2Bamino%2Bacid%2Bmatrix%2Bselection%2Band%2Btheir%2Buse%2Bon%2Bempirical%2Bdata%2Bshows%2Bthat%2Bad%2Bhoc%2Bassumptions%2Bfor%2Bchoice%2Bof%2Bmatrix%2Bare%2Bnot%2Bjustified.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2148-6-29%26rft_id%253Dinfo%253Apmid%252F16563161%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2148-6-29&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16563161&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F04%2F13%2F450692.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">86.</span><a class="rev-xref-ref" href="#xref-ref-86-1" title="View reference 86. in text" id="ref-86">↵</a><div class="cit ref-cit ref-journal" id="cit-450692v2.86" data-doi="10.1080/10635150390235520"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Guindon</span>,  <span class="cit-name-given-names">S.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Gascuel</span>,  <span class="cit-name-given-names">O.</span></span> <span class="cit-article-title">A simple, fast, and accurate algorithm to estimate large phylogenies by maximum likelihood</span>. <abbr class="cit-jnl-abbrev">Syst Biol</abbr> <span class="cit-vol">52</span>, <span class="cit-fpage">696</span>–<span class="cit-lpage">704</span> (<span class="cit-pub-date">2003</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DSyst%2BBiol%26rft_id%253Dinfo%253Adoi%252F10.1080%252F10635150390235520%26rft_id%253Dinfo%253Apmid%252F14530136%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1080/10635150390235520&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=14530136&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F04%2F13%2F450692.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000185732500010&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">87.</span><a class="rev-xref-ref" href="#xref-ref-87-1" title="View reference 87. in text" id="ref-87">↵</a><div class="cit ref-cit ref-journal" id="cit-450692v2.87" data-doi="10.1534/genetics.113.154138"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Skotte</span>,  <span class="cit-name-given-names">L.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Korneliussen</span>,  <span class="cit-name-given-names">T.S.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Albrechtsen</span>,  <span class="cit-name-given-names">A.</span></span> <span class="cit-article-title">Estimating individual admixture proportions from next generation sequencing data</span>. <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">195</span>, <span class="cit-fpage">693</span>–<span class="cit-lpage">702</span> (<span class="cit-pub-date">2013</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.113.154138%26rft_id%253Dinfo%253Apmid%252F24026093%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTk1LzMvNjkzIjtzOjQ6ImF0b20iO3M6Mzc6Ii9iaW9yeGl2L2Vhcmx5LzIwMTkvMDQvMTMvNDUwNjkyLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">88.</span><a class="rev-xref-ref" href="#xref-ref-88-1" title="View reference 88. in text" id="ref-88">↵</a><div class="cit ref-cit ref-journal" id="cit-450692v2.88" data-doi="10.1038/ng.3015"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Schiffels</span>,  <span class="cit-name-given-names">S.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Durbin</span>,  <span class="cit-name-given-names">R.</span></span> <span class="cit-article-title">Inferring human population size and separation history from multiple genome sequences</span>. <abbr class="cit-jnl-abbrev">Nat Genet</abbr> <span class="cit-vol">46</span>, <span class="cit-fpage">919</span>–<span class="cit-lpage">25</span> (<span class="cit-pub-date">2014</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat%2BGenet%26rft.volume%253D46%26rft.spage%253D919%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fng.3015%26rft_id%253Dinfo%253Apmid%252F24952747%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ng.3015&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24952747&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F04%2F13%2F450692.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">89.</span><a class="rev-xref-ref" href="#xref-ref-89-1" title="View reference 89. in text" id="ref-89">↵</a><div class="cit ref-cit ref-journal" id="cit-450692v2.89" data-doi="10.1038/nature18299"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Malaspinas</span>,  <span class="cit-name-given-names">A.S.</span></span> <span class="cit-etal">et al.</span> <span class="cit-article-title">A genomic history of Aboriginal Australia</span>. <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">538</span>, <span class="cit-fpage">207</span>–<span class="cit-lpage">214</span> (<span class="cit-pub-date">2016</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.volume%253D538%26rft.spage%253D207%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature18299%26rft_id%253Dinfo%253Apmid%252F27654914%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature18299&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=27654914&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F04%2F13%2F450692.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">90.</span><a class="rev-xref-ref" href="#xref-ref-90-1" title="View reference 90. in text" id="ref-90">↵</a><div class="cit ref-cit ref-journal" id="cit-450692v2.90" data-doi="10.1098/rstb.2005.1673"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">McVean</span>,  <span class="cit-name-given-names">G.A.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Cardin</span>,  <span class="cit-name-given-names">N.J.</span></span> <span class="cit-article-title">Approximating the coalescent with recombination</span>. <abbr class="cit-jnl-abbrev">Philos Trans R Soc Lond B Biol Sci</abbr> <span class="cit-vol">360</span>, <span class="cit-fpage">1387</span>–<span class="cit-lpage">93</span> (<span class="cit-pub-date">2005</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPhilosophical%2BTransactions%2Bof%2Bthe%2BRoyal%2BSociety%2BB%253A%2BBiological%2BSciences%26rft.stitle%253DPhil%2BTrans%2BR%2BSoc%2BB%26rft.aulast%253DMcVean%26rft.auinit1%253DG.%2BA.%2BT.%26rft.volume%253D360%26rft.issue%253D1459%26rft.spage%253D1387%26rft.epage%253D1393%26rft.atitle%253DApproximating%2Bthe%2Bcoalescent%2Bwith%2Brecombination%26rft_id%253Dinfo%253Adoi%252F10.1098%252Frstb.2005.1673%26rft_id%253Dinfo%253Apmid%252F16048782%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1098/rstb.2005.1673&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16048782&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F04%2F13%2F450692.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">91.</span><a class="rev-xref-ref" href="#xref-ref-91-1" title="View reference 91. in text" id="ref-91">↵</a><div class="cit ref-cit ref-journal" id="cit-450692v2.91" data-doi="10.1016/0304-4149(82)90011-4"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Kingman</span>,  <span class="cit-name-given-names">J.F.C.</span></span> <span class="cit-article-title">The coalescent</span>. <abbr class="cit-jnl-abbrev">Stochastic Processes and their Applications</abbr> <span class="cit-vol">13</span>, <span class="cit-fpage">235</span>–<span class="cit-lpage">248</span> (<span class="cit-pub-date">1982</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DStochastic%2BProcesses%2Band%2Btheir%2BApplications%26rft.volume%253D13%26rft.spage%253D235%26rft_id%253Dinfo%253Adoi%252F10.1016%252F0304-4149%252882%252990011-4%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/0304-4149(82)90011-4&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label">92.</span><a class="rev-xref-ref" href="#xref-ref-92-1" title="View reference 92. in text" id="ref-92">↵</a><div class="cit ref-cit ref-journal" id="cit-450692v2.92" data-doi="10.1016/j.ijpara.2016.07.001"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Emery</span>,  <span class="cit-name-given-names">D.L.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Hunt</span>,  <span class="cit-name-given-names">P.W.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Le Jambre</span>,  <span class="cit-name-given-names">L.F.</span></span> <span class="cit-article-title">Haemonchus contortus: the then and now, and where to from here?</span> <abbr class="cit-jnl-abbrev">Int J Parasitol</abbr> <span class="cit-vol">46</span>, <span class="cit-fpage">755</span>–<span class="cit-lpage">769</span> (<span class="cit-pub-date">2016</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DInt%2BJ%2BParasitol%26rft.volume%253D46%26rft.spage%253D755%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.ijpara.2016.07.001%26rft_id%253Dinfo%253Apmid%252F27620133%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.ijpara.2016.07.001&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=27620133&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F04%2F13%2F450692.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">93.</span><a class="rev-xref-ref" href="#xref-ref-93-1" title="View reference 93. in text" id="ref-93">↵</a><div class="cit ref-cit ref-journal" id="cit-450692v2.93" data-doi="10.1371/journal.pgen.1000695"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Gutenkunst</span>,  <span class="cit-name-given-names">R.N.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Hernandez</span>,  <span class="cit-name-given-names">R.D.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Williamson</span>,  <span class="cit-name-given-names">S.H.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Bustamante</span>,  <span class="cit-name-given-names">C.D.</span></span> <span class="cit-article-title">Inferring the joint demographic history of multiple populations from multidimensional SNP frequency data</span>. <abbr class="cit-jnl-abbrev">PLoS Genet</abbr> <span class="cit-vol">5</span>, <span class="cit-fpage">e1000695</span> (<span class="cit-pub-date">2009</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BGenet%26rft.aulast%253DGutenkunst%26rft.auinit1%253DR.%2BN.%26rft.volume%253D5%26rft.issue%253D10%26rft.spage%253De1000695%26rft.epage%253De1000695%26rft.atitle%253DInferring%2Bthe%2Bjoint%2Bdemographic%2Bhistory%2Bof%2Bmultiple%2Bpopulations%2Bfrom%2Bmultidimensional%2BSNP%2Bfrequency%2Bdata.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.1000695%26rft_id%253Dinfo%253Apmid%252F19851460%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.1000695&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19851460&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F04%2F13%2F450692.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">94.</span><a class="rev-xref-ref" href="#xref-ref-94-1" title="View reference 94. in text" id="ref-94">↵</a><div class="cit ref-cit ref-journal" id="cit-450692v2.94"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Portik</span>,  <span class="cit-name-given-names">D.M.</span></span> <span class="cit-etal">et al.</span> <span class="cit-article-title">Evaluating mechanisms of diversification in a Guineo-Congolian tropical forest frog using demographic model selection</span>. <abbr class="cit-jnl-abbrev">Mol Ecol</abbr> <span class="cit-vol">26</span>, <span class="cit-fpage">5245</span>–<span class="cit-lpage">5263</span> (<span class="cit-pub-date">2017</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol%2BEcol%26rft.volume%253D26%26rft.spage%253D5245%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">95.</span><a class="rev-xref-ref" href="#xref-ref-95-1" title="View reference 95. in text" id="ref-95">↵</a><div class="cit ref-cit ref-journal" id="cit-450692v2.95" data-doi="10.1093/molbev/msv255"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Coffman</span>,  <span class="cit-name-given-names">A.J.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Hsieh</span>,  <span class="cit-name-given-names">P.H.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Gravel</span>,  <span class="cit-name-given-names">S.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Gutenkunst</span>,  <span class="cit-name-given-names">R.N.</span></span> <span class="cit-article-title">Computationally Efficient Composite Likelihood Statistics for Demographic Inference</span>. <abbr class="cit-jnl-abbrev">Mol Biol Evol</abbr> <span class="cit-vol">33</span>, <span class="cit-fpage">591</span>–<span class="cit-lpage">3</span> (<span class="cit-pub-date">2016</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol%2BBiol%2BEvol%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fmolbev%252Fmsv255%26rft_id%253Dinfo%253Apmid%252F26545922%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/molbev/msv255&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=26545922&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F04%2F13%2F450692.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">96.</span><a class="rev-xref-ref" href="#xref-ref-96-1" title="View reference 96. in text" id="ref-96">↵</a><div class="cit ref-cit ref-journal" id="cit-450692v2.96" data-doi="10.1093/bioinformatics/18.2.337"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Hudson</span>,  <span class="cit-name-given-names">R.R.</span></span> <span class="cit-article-title">Generating samples under a Wright-Fisher neutral model of genetic variation</span>. <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">18</span>, <span class="cit-fpage">337</span>–<span class="cit-lpage">8</span> (<span class="cit-pub-date">2002</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft.stitle%253DBioinformatics%26rft.aulast%253DHudson%26rft.auinit1%253DR.%2BR.%26rft.volume%253D18%26rft.issue%253D2%26rft.spage%253D337%26rft.epage%253D338%26rft.atitle%253DGenerating%2Bsamples%2Bunder%2Ba%2BWright-Fisher%2Bneutral%2Bmodel%2Bof%2B%2Bgenetic%2Bvariation%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252F18.2.337%26rft_id%253Dinfo%253Apmid%252F11847089%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/18.2.337&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=11847089&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F04%2F13%2F450692.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000174028100021&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">97.</span><a class="rev-xref-ref" href="#xref-ref-97-1" title="View reference 97. in text" id="ref-97">↵</a><div class="cit ref-cit ref-journal" id="cit-450692v2.97" data-doi="10.1093/ve/vey016"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Suchard</span>,  <span class="cit-name-given-names">M.A.</span></span> <span class="cit-etal">et al.</span> <span class="cit-article-title">Bayesian phylogenetic and phylodynamic data integration using BEAST 1.10</span>. <abbr class="cit-jnl-abbrev">Virus Evol</abbr> <span class="cit-vol">4</span>, <span class="cit-fpage">vey016</span> (<span class="cit-pub-date">2018</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DVirus%2BEvol%26rft.volume%253D4%26rft.spage%253Dvey016%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fve%252Fvey016%26rft_id%253Dinfo%253Apmid%252F29942656%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/ve/vey016&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=29942656&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F04%2F13%2F450692.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">98.</span><a class="rev-xref-ref" href="#xref-ref-98-1" title="View reference 98. in text" id="ref-98">↵</a><div class="cit ref-cit ref-journal" id="cit-450692v2.98" data-doi="10.1093/molbev/msi103"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Drummond</span>,  <span class="cit-name-given-names">A.J.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Rambaut</span>,  <span class="cit-name-given-names">A.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Shapiro</span>,  <span class="cit-name-given-names">B.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Pybus</span>,  <span class="cit-name-given-names">O.G.</span></span> <span class="cit-article-title">Bayesian coalescent inference of past population dynamics from molecular sequences</span>. <abbr class="cit-jnl-abbrev">Mol Biol Evol</abbr> <span class="cit-vol">22</span>, <span class="cit-fpage">1185</span>–<span class="cit-lpage">92</span> (<span class="cit-pub-date">2005</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol%2BBiol%2BEvol%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fmolbev%252Fmsi103%26rft_id%253Dinfo%253Apmid%252F15703244%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/molbev/msi103&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15703244&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F04%2F13%2F450692.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000228700200004&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">99.</span><a class="rev-xref-ref" href="#xref-ref-99-1" title="View reference 99. in text" id="ref-99">↵</a><div class="cit ref-cit ref-journal" id="cit-450692v2.99" data-doi="10.1016/j.molbiopara.2016.11.005"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Howe</span>,  <span class="cit-name-given-names">K.L.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Bolt</span>,  <span class="cit-name-given-names">B.J.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Shafie</span>,  <span class="cit-name-given-names">M.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Kersey</span>,  <span class="cit-name-given-names">P.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Berriman</span>,  <span class="cit-name-given-names">M.</span></span> <span class="cit-article-title">WormBase ParaSite – a comprehensive resource for helminth genomics</span>. <abbr class="cit-jnl-abbrev">Mol Biochem Parasitol</abbr> <span class="cit-vol">215</span>, <span class="cit-fpage">2</span>–<span class="cit-lpage">10</span> (<span class="cit-pub-date">2017</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol%2BBiochem%2BParasitol%26rft.volume%253D215%26rft.spage%253D2%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.molbiopara.2016.11.005%26rft_id%253Dinfo%253Apmid%252F27899279%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.molbiopara.2016.11.005&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=27899279&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F04%2F13%2F450692.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">100.</span><a class="rev-xref-ref" href="#xref-ref-100-1" title="View reference 100. in text" id="ref-100">↵</a><div class="cit ref-cit ref-journal" id="cit-450692v2.100"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Rufener</span>,  <span class="cit-name-given-names">L.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Kaminsky</span>,  <span class="cit-name-given-names">R.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Maser</span>,  <span class="cit-name-given-names">P.</span></span> <span class="cit-article-title">In vitro selection of Haemonchus contortus for benzimidazole resistance reveals a mutation at amino acid 198 of beta-tubulin</span>. <abbr class="cit-jnl-abbrev">Mol Biochem Parasitol</abbr> <span class="cit-vol">168</span>, <span class="cit-fpage">120</span>–<span class="cit-lpage">2</span> (<span class="cit-pub-date">2009</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2Band%2Bbiochemical%2Bparasitology%26rft.stitle%253DMol%2BBiochem%2BParasitol%26rft.aulast%253DRufener%26rft.auinit1%253DL.%26rft.volume%253D168%26rft.issue%253D1%26rft.spage%253D120%26rft.epage%253D122%26rft.atitle%253DIn%2Bvitro%2Bselection%2Bof%2BHaemonchus%2Bcontortus%2Bfor%2Bbenzimidazole%2Bresistance%2Breveals%2Ba%2Bmutation%2Bat%2Bamino%2Bacid%2B198%2Bof%2Bbeta-tubulin.%26rft_id%253Dinfo%253Apmid%252F19616042%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=19616042&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F04%2F13%2F450692.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">101.</span><a class="rev-xref-ref" href="#xref-ref-101-1" title="View reference 101. in text" id="ref-101">↵</a><div class="cit ref-cit ref-journal" id="cit-450692v2.101" data-doi="10.1186/1471-2105-14-289"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Korneliussen</span>,  <span class="cit-name-given-names">T.S.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Moltke</span>,  <span class="cit-name-given-names">I.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Albrechtsen</span>,  <span class="cit-name-given-names">A.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Nielsen</span>,  <span class="cit-name-given-names">R.</span></span> <span class="cit-article-title">Calculation of Tajima's D and other neutrality test statistics from low depth next-generation sequencing data</span>. <abbr class="cit-jnl-abbrev">BMC Bioinformatics</abbr> <span class="cit-vol">14</span>, <span class="cit-fpage">289</span> (<span class="cit-pub-date">2013</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2BBioinformatics%26rft.volume%253D14%26rft.spage%253D289%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2105-14-289%26rft_id%253Dinfo%253Apmid%252F24088262%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2105-14-289&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24088262&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F04%2F13%2F450692.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">102.</span><a class="rev-xref-ref" href="#xref-ref-102-1" title="View reference 102. in text" id="ref-102">↵</a><div class="cit ref-cit ref-journal" id="cit-450692v2.102" data-doi="10.1098/rstb.2002.1126"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Hudson</span>,  <span class="cit-name-given-names">P.J.</span></span> <span class="cit-etal">et al.</span> <span class="cit-article-title">Trophic interactions and population growth rates: describing patterns and identifying mechanisms</span>. <abbr class="cit-jnl-abbrev">Philos Trans R Soc Lond B Biol Sci</abbr> <span class="cit-vol">357</span>, <span class="cit-fpage">1259</span>–<span class="cit-lpage">71</span> (<span class="cit-pub-date">2002</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPhilosophical%2BTransactions%2Bof%2Bthe%2BRoyal%2BSociety%2BB%253A%2BBiological%2BSciences%26rft.stitle%253DPhil%2BTrans%2BR%2BSoc%2BB%26rft.aulast%253DHudson%26rft.auinit1%253DP.%2BJ.%26rft.volume%253D357%26rft.issue%253D1425%26rft.spage%253D1259%26rft.epage%253D1271%26rft.atitle%253DTrophic%2Binteractions%2Band%2Bpopulation%2Bgrowth%2Brates%253A%2Bdescribing%2Bpatterns%2Band%2Bidentifying%2Bmechanisms%26rft_id%253Dinfo%253Adoi%252F10.1098%252Frstb.2002.1126%26rft_id%253Dinfo%253Apmid%252F12396517%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1098/rstb.2002.1126&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=12396517&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F04%2F13%2F450692.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">103.</span><a class="rev-xref-ref" href="#xref-ref-103-1" title="View reference 103. in text" id="ref-103">↵</a><div class="cit ref-cit ref-journal" id="cit-450692v2.103" data-doi="10.1534/genetics.116.194720"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Martin</span>,  <span class="cit-name-given-names">S.H.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Van Belleghem</span>,  <span class="cit-name-given-names">S.M.</span></span> <span class="cit-article-title">Exploring Evolutionary Relationships Across the Genome Using Topology Weighting</span>. <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">206</span>, <span class="cit-fpage">429</span>–<span class="cit-lpage">438</span> (<span class="cit-pub-date">2017</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.116.194720%26rft_id%253Dinfo%253Apmid%252F28341652%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMjA2LzEvNDI5IjtzOjQ6ImF0b20iO3M6Mzc6Ii9iaW9yeGl2L2Vhcmx5LzIwMTkvMDQvMTMvNDUwNjkyLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">104.</span><a class="rev-xref-ref" href="#xref-ref-104-1" title="View reference 104. in text" id="ref-104">↵</a><div class="cit ref-cit ref-journal" id="cit-450692v2.104"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Köppen</span>,  <span class="cit-name-given-names">W. Die Wärmezonen der Erde</span></span>, <span class="cit-article-title">nach der Dauer der heissen, gemässigten und kalten Zeit und nach der Wirkung der Wärme auf die organische Welt betrachtet</span> <abbr class="cit-jnl-abbrev">Meteorol. Z.</abbr> <span class="cit-vol">1</span>, <span class="cit-fpage">215</span>–<span class="cit-lpage">226</span> (<span class="cit-pub-date">1884</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMeteorol.%2BZ.%26rft.volume%253D1%26rft.spage%253D215%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">105.</span><a class="rev-xref-ref" href="#xref-ref-105-1" title="View reference 105. in text" id="ref-105">↵</a><div class="cit ref-cit ref-journal" id="cit-450692v2.105"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Chen</span>,  <span class="cit-name-given-names">D.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Chen</span>,  <span class="cit-name-given-names">H.W.</span></span> <span class="cit-article-title">Using the Köppen classification to quantify climate variation and change: An example for 1901–2010</span>. <abbr class="cit-jnl-abbrev">Environmental Development</abbr> <span class="cit-vol">6</span>, <span class="cit-fpage">69</span>–<span class="cit-lpage">79</span> (<span class="cit-pub-date">2013</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEnvironmental%2BDevelopment%26rft.volume%253D6%26rft.spage%253D69%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">106.</span><a class="rev-xref-ref" href="#xref-ref-106-1" title="View reference 106. in text" id="ref-106">↵</a><div class="cit ref-cit ref-journal" id="cit-450692v2.106" data-doi="10.1890/11-0252.1"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Ellis</span>,  <span class="cit-name-given-names">N.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Smith</span>,  <span class="cit-name-given-names">S.J.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Pitcher</span>,  <span class="cit-name-given-names">C.R.</span></span> <span class="cit-article-title">Gradient forests: calculating importance gradients on physical predictors</span>. <abbr class="cit-jnl-abbrev">Ecology</abbr> <span class="cit-vol">93</span>, <span class="cit-fpage">156</span>–<span class="cit-lpage">68</span> (<span class="cit-pub-date">2012</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEcology%26rft.stitle%253DEcology%26rft.aulast%253DEllis%26rft.auinit1%253DN.%26rft.volume%253D93%26rft.issue%253D1%26rft.spage%253D156%26rft.epage%253D168%26rft.atitle%253DGradient%2Bforests%253A%2Bcalculating%2Bimportance%2Bgradients%2Bon%2Bphysical%2Bpredictors.%26rft_id%253Dinfo%253Adoi%252F10.1890%252F11-0252.1%26rft_id%253Dinfo%253Apmid%252F22486096%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1890/11-0252.1&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22486096&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F04%2F13%2F450692.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">107.</span><a class="rev-xref-ref" href="#xref-ref-107-1" title="View reference 107. in text" id="ref-107">↵</a><div class="cit ref-cit ref-journal" id="cit-450692v2.107"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Fick</span>,  <span class="cit-name-given-names">S. E.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Hijmans</span>,  <span class="cit-name-given-names">R.J.</span></span> <span class="cit-article-title">WorldClim 2: new 1-km spatial resolution climate surfaces for global land areas</span>. <abbr class="cit-jnl-abbrev">International Journal of Climatology</abbr> <span class="cit-vol">37</span>, <span class="cit-fpage">4302</span>–<span class="cit-lpage">4315</span> (<span class="cit-pub-date">2017</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DInternational%2BJournal%2Bof%2BClimatology%26rft.volume%253D37%26rft.spage%253D4302%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">108.</span><a class="rev-xref-ref" href="#xref-ref-108-1" title="View reference 108. in text" id="ref-108">↵</a><div class="cit ref-cit ref-journal" id="cit-450692v2.108" data-doi="10.1093/molbev/mst063"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Frichot</span>,  <span class="cit-name-given-names">E.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Schoville</span>,  <span class="cit-name-given-names">S.D.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Bouchard</span>,  <span class="cit-name-given-names">G.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Francois</span>,  <span class="cit-name-given-names">O.</span></span> <span class="cit-article-title">Testing for associations between loci and environmental gradients using latent factor mixed models</span>. <abbr class="cit-jnl-abbrev">Mol Biol Evol</abbr> <span class="cit-vol">30</span>, <span class="cit-fpage">1687</span>–<span class="cit-lpage">99</span> (<span class="cit-pub-date">2013</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol%2BBiol%2BEvol%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fmolbev%252Fmst063%26rft_id%253Dinfo%253Apmid%252F23543094%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/molbev/mst063&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23543094&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F04%2F13%2F450692.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000321056200018&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">109.</span><a class="rev-xref-ref" href="#xref-ref-109-1" title="View reference 109. in text" id="ref-109">↵</a><div class="cit ref-cit ref-journal" id="cit-450692v2.109"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Alexa</span>,  <span class="cit-name-given-names">A.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Rahnenfuhrer</span>,  <span class="cit-name-given-names">J.</span></span> <abbr class="cit-jnl-abbrev">topGO: Enrichment Analysis for Gene Ontology</abbr>. (<span class="cit-pub-date">2016</span>).</cite></div><div class="cit-extra"></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.biorxiv.org/content/10.1101/450692v2.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-disqus-comment" >
  
      
  
  <div class="pane-content">
    <div id="disqus_thread"><noscript><p><a href="http://biorxivstage.disqus.com/?url=https%3A%2F%2Fwww.biorxiv.org%2Fcontent%2F10.1101%2F450692v2" class="" data-icon-position="" data-hide-link-title="0">View the discussion thread.</a></p></noscript></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/10.1101/420976v2" title="Identification of Kinase Inhibitors that Regulate Nuclear Receptor Nurr1 (NR4A2) Cellular Activity" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous</span></a></span><span class="pager-next"><a href="/content/10.1101/283929v2-0" title="Cell-type heterogeneity in adipose tissue is associated with complex traits and reveals disease-relevant cell-specific eQTLs" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    Posted&nbsp;April 13, 2019.  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-biorxiv-art-tools" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-biorxiv_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/10.1101/450692v2.full.pdf" target="_self" class="article-dl-pdf-link link-icon"><i class="icon-external-link-sign"></i> <span class="title">Download PDF</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-biorxiv-supplementary-fragment" >
  
      
  
  <div class="pane-content">
    <p><a href="/content/10.1101/450692v2.supplementary-material"><i class="icon-file"></i>Supplementary Material </a></p>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-biorxiv-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="Email this Article" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-biorxiv_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/10.1101/450692v2.full" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for your interest in spreading the word about bioRxiv.</p><p>NOTE: Your email address is requested solely to identify you as the sender of this article.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/10.1101/450692v2" class="active" data-icon-position="" data-hide-link-title="0">The global diversity of the major parasitic nematode Haemonchus contortus is shaped by human intervention and climate</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from bioRxiv
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would like to see this page from the bioRxiv website.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/608843" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-YzMZ3oxGVsw30F5xYxUylDd_3j-XpRmUe9MSGXQfC2I" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-biorxiv-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-biorxiv_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="608843" id="node608843--2" data-pisa="biorxiv;450692v2" data-pisa-master="biorxiv;450692" data-seqnum="608843" data-apath="/biorxiv/early/2019/04/13/450692.atom"><div  class="highwire-cite highwire-cite-highwire-article highwire-citation-biorxiv-article-pap-list clearfix" >

      <div class="highwire-cite-title" >
      <div class="highwire-cite-title">The global diversity of the major parasitic nematode <em>Haemonchus contortus</em> is shaped by human intervention and climate</div>    </div>
  
      <div  class="highwire-cite-authors" ><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0"><span class="nlm-given-names">G.</span> <span class="nlm-surname">Sallé</span></span>, <span class="highwire-citation-author" data-delta="1"><span class="nlm-given-names">S.R.</span> <span class="nlm-surname">Doyle</span></span>, <span class="highwire-citation-author" data-delta="2"><span class="nlm-given-names">J.</span> <span class="nlm-surname">Cortet</span></span>, <span class="highwire-citation-author" data-delta="3"><span class="nlm-given-names">J.</span> <span class="nlm-surname">Cabaret</span></span>, <span class="highwire-citation-author" data-delta="4"><span class="nlm-given-names">M.</span> <span class="nlm-surname">Berriman</span></span>, <span class="highwire-citation-author" data-delta="5"><span class="nlm-given-names">N.</span> <span class="nlm-surname">Holroyd</span></span>, <span class="highwire-citation-author" data-delta="6"><span class="nlm-given-names">J.A</span> <span class="nlm-surname">Cotton</span></span></span></div>
  
      <div  class="highwire-cite-metadata" ><span  class="highwire-cite-metadata-journal highwire-cite-metadata">bioRxiv </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">450692; </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata"><span class="doi_label">doi:</span> https://doi.org/10.1101/450692 </span></div>
  
  
  </div>
</div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.biorxiv.org/content/10.1101/450692v2" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.biorxiv.org/content/10.1101/450692v2&amp;title=The%20global%20diversity%20of%20the%20major%20parasitic%20nematode%20Haemonchus%20contortus%20is%20shaped%20by%20human%20intervention%20and%20climate" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.biorxiv.org/content/10.1101/450692v2&amp;title=The%20global%20diversity%20of%20the%20major%20parasitic%20nematode%20Haemonchus%20contortus%20is%20shaped%20by%20human%20intervention%20and%20climate" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.biorxiv.org/content/10.1101/450692v2&amp;text=The%20global%20diversity%20of%20the%20major%20parasitic%20nematode%20Haemonchus%20contortus%20is%20shaped%20by%20human%20intervention%20and%20climate" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.biorxiv.org/content/10.1101/450692v2&amp;title=The%20global%20diversity%20of%20the%20major%20parasitic%20nematode%20Haemonchus%20contortus%20is%20shaped%20by%20human%20intervention%20and%20climate" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.biorxiv.org/content/10.1101/450692v2&amp;t=The%20global%20diversity%20of%20the%20major%20parasitic%20nematode%20Haemonchus%20contortus%20is%20shaped%20by%20human%20intervention%20and%20climate" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.biorxiv.org/content/10.1101/450692v2&amp;title=The%20global%20diversity%20of%20the%20major%20parasitic%20nematode%20Haemonchus%20contortus%20is%20shaped%20by%20human%20intervention%20and%20climate" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.biorxiv.org/content/10.1101/450692v2&amp;title=The%20global%20diversity%20of%20the%20major%20parasitic%20nematode%20Haemonchus%20contortus%20is%20shaped%20by%20human%20intervention%20and%20climate" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-biorxiv-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title="Citation Tools"><i class="icon-globe"></i> <span class="title">Citation Tools</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-biorxiv_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="608843" id="node608843--3" data-pisa="biorxiv;450692v2" data-pisa-master="biorxiv;450692" data-seqnum="608843" data-apath="/biorxiv/early/2019/04/13/450692.atom"><div  class="highwire-cite highwire-cite-highwire-article highwire-citation-biorxiv-article-pap-list clearfix" >

      <div class="highwire-cite-title" >
      <div class="highwire-cite-title">The global diversity of the major parasitic nematode <em>Haemonchus contortus</em> is shaped by human intervention and climate</div>    </div>
  
      <div  class="highwire-cite-authors" ><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0"><span class="nlm-given-names">G.</span> <span class="nlm-surname">Sallé</span></span>, <span class="highwire-citation-author" data-delta="1"><span class="nlm-given-names">S.R.</span> <span class="nlm-surname">Doyle</span></span>, <span class="highwire-citation-author" data-delta="2"><span class="nlm-given-names">J.</span> <span class="nlm-surname">Cortet</span></span>, <span class="highwire-citation-author" data-delta="3"><span class="nlm-given-names">J.</span> <span class="nlm-surname">Cabaret</span></span>, <span class="highwire-citation-author" data-delta="4"><span class="nlm-given-names">M.</span> <span class="nlm-surname">Berriman</span></span>, <span class="highwire-citation-author" data-delta="5"><span class="nlm-given-names">N.</span> <span class="nlm-surname">Holroyd</span></span>, <span class="highwire-citation-author" data-delta="6"><span class="nlm-given-names">J.A</span> <span class="nlm-surname">Cotton</span></span></span></div>
  
      <div  class="highwire-cite-metadata" ><span  class="highwire-cite-metadata-journal highwire-cite-metadata">bioRxiv </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">450692; </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata"><span class="doi_label">doi:</span> https://doi.org/10.1101/450692 </span></div>
  
  
  </div>
</div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage&amp;rft.epage&amp;rft.atitle=The%20global%20diversity%20of%20the%20major%20parasitic%20nematode%20Haemonchus%20contortus%20is%20shaped%20by%20human%20intervention%20and%20climate&amp;rft.volume&amp;rft.issue&amp;rft.date=2019-01-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=bioRxiv&amp;rft.au=Sall%C3%A9%2C+G.&amp;rft.au=Doyle%2C+S.R.&amp;rft.au=Cortet%2C+J.&amp;rft.au=Cabaret%2C+J.&amp;rft.au=Berriman%2C+M.&amp;rft.au=Holroyd%2C+N.&amp;rft.au=Cotton%2C+J.A"></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/608843/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/608843/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/608843/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/608843/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/608843/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/608843/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/608843/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/608843/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/608843/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/608843/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/608843/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/608843/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><div class="item-list"><ul><li class="first"><a href="http://twitter.com/share?url=https%3A//www.biorxiv.org/content/10.1101/450692v2&amp;count=horizontal&amp;via=&amp;text=The%20global%20diversity%20of%20the%20major%20parasitic%20nematode%20Haemonchus%20contortus%20is%20shaped%20by%20human%20intervention%20and%20climate&amp;counturl=https%3A//www.biorxiv.org/content/10.1101/450692v2" class="twitter-share-button service-links-twitter-widget" id="twitter_widget" title="Tweet This" rel="nofollow" data-icon-position="" data-hide-link-title="0"><span class="element-invisible">Tweet Widget</span></a></li><li><a href="http://www.facebook.com/plugins/like.php?href=https%3A//www.biorxiv.org/content/10.1101/450692v2&amp;layout=button_count&amp;show_faces=false&amp;action=like&amp;colorscheme=light&amp;width=100&amp;height=21&amp;font=&amp;locale=" id="facebook_like" title="I Like it" class="service-links-facebook-like" rel="nofollow" data-icon-position="" data-hide-link-title="0"><span class="element-invisible">Facebook Like</span></a></li><li class="last"><a href="https://www.biorxiv.org/content/10.1101/450692v2" id="google_plus_one" title="Plus it" class="service-links-google-plus-one" rel="nofollow" data-icon-position="" data-hide-link-title="0"><span class="element-invisible">Google Plus One</span></a></li></ul></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-collections" >
  
        <h2 class="pane-title">Subject Area</h2>
    
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-collections"><div class="highwire-list"><ul class="highwire-article-collection-term-list"><li class="first last odd"><span class="highwire-article-collection-term"><a href="/collection/genomics" class="highlight" data-icon-position="" data-hide-link-title="0">Genomics<i class="icon-caret-right"></i>
</a></span></li></ul></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-biorxiv-subject-collections block-style-col2" >
  
      
  
  <div class="pane-content">
    <div class="panel-flexible panels-flexible-8 clearfix" id="mini-panel-biorxiv_subject_collections">
<div class="panel-flexible-inside panels-flexible-8-inside">
<div class="panels-flexible-region panels-flexible-region-8-center panels-flexible-region-first panels-flexible-region-last">
  <div class="inside panels-flexible-region-inside panels-flexible-region-8-center-inside panels-flexible-region-inside-first panels-flexible-region-inside-last">
<div class="panel-pane pane-snippet" >
  
      
  
  <div class="pane-content">
    <div class="snippet biorxiv-subject-areas-table-title" id="biorxiv-subject-areas-table-title">
  
      
  <div class="snippet-content">
    <b>Subject Areas</b>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-snippet" >
  
      
  
  <div class="pane-content">
    <div class="snippet biorxiv-subject-areas-view-papers" id="biorxiv-subject-areas-view-papers">
  
      
  <div class="snippet-content">
    <a href="/content/early/recent"><strong>All Articles</strong></a>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-subject-collections" >
  
      
  
  <div class="pane-content">
    <ul id="collection" class="collection highwire-list-expand"><li class="outer collection depth-2  child first"><div class = "data-wrapper"><a href="/collection/animal-behavior-and-cognition" class="" data-icon-position="" data-hide-link-title="0">Animal Behavior and Cognition</a> <span class = "article-count">(1662)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/biochemistry" class="" data-icon-position="" data-hide-link-title="0">Biochemistry</a> <span class = "article-count">(2789)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/bioengineering" class="" data-icon-position="" data-hide-link-title="0">Bioengineering</a> <span class = "article-count">(1946)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/bioinformatics" class="" data-icon-position="" data-hide-link-title="0">Bioinformatics</a> <span class = "article-count">(10400)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/biophysics" class="" data-icon-position="" data-hide-link-title="0">Biophysics</a> <span class = "article-count">(4243)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/cancer-biology" class="" data-icon-position="" data-hide-link-title="0">Cancer Biology</a> <span class = "article-count">(3272)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/cell-biology" class="" data-icon-position="" data-hide-link-title="0">Cell Biology</a> <span class = "article-count">(4641)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/clinical-trials" class="" data-icon-position="" data-hide-link-title="0">Clinical Trials</a> <span class = "article-count">(135)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/developmental-biology" class="" data-icon-position="" data-hide-link-title="0">Developmental Biology</a> <span class = "article-count">(2889)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/ecology" class="" data-icon-position="" data-hide-link-title="0">Ecology</a> <span class = "article-count">(4544)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/epidemiology" class="" data-icon-position="" data-hide-link-title="0">Epidemiology</a> <span class = "article-count">(2041)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/evolutionary-biology" class="" data-icon-position="" data-hide-link-title="0">Evolutionary Biology</a> <span class = "article-count">(7288)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/genetics" class="" data-icon-position="" data-hide-link-title="0">Genetics</a> <span class = "article-count">(5540)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/genomics" class="" data-icon-position="" data-hide-link-title="0">Genomics</a> <span class = "article-count">(6893)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/immunology" class="" data-icon-position="" data-hide-link-title="0">Immunology</a> <span class = "article-count">(2434)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/microbiology" class="" data-icon-position="" data-hide-link-title="0">Microbiology</a> <span class = "article-count">(7616)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/molecular-biology" class="" data-icon-position="" data-hide-link-title="0">Molecular Biology</a> <span class = "article-count">(3051)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/neuroscience" class="" data-icon-position="" data-hide-link-title="0">Neuroscience</a> <span class = "article-count">(18885)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/paleontology" class="" data-icon-position="" data-hide-link-title="0">Paleontology</a> <span class = "article-count">(142)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/pathology" class="" data-icon-position="" data-hide-link-title="0">Pathology</a> <span class = "article-count">(477)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/pharmacology-and-toxicology" class="" data-icon-position="" data-hide-link-title="0">Pharmacology and Toxicology</a> <span class = "article-count">(791)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/physiology" class="" data-icon-position="" data-hide-link-title="0">Physiology</a> <span class = "article-count">(1178)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/plant-biology" class="" data-icon-position="" data-hide-link-title="0">Plant Biology</a> <span class = "article-count">(2771)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/scientific-communication-and-education" class="" data-icon-position="" data-hide-link-title="0">Scientific Communication and Education</a> <span class = "article-count">(686)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/synthetic-biology" class="" data-icon-position="" data-hide-link-title="0">Synthetic Biology</a> <span class = "article-count">(905)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/systems-biology" class="" data-icon-position="" data-hide-link-title="0">Systems Biology</a> <span class = "article-count">(2881)</span></div></li>
<li class="outer collection depth-2  child last"><div class = "data-wrapper"><a href="/collection/zoology" class="" data-icon-position="" data-hide-link-title="0">Zoology</a> <span class = "article-count">(472)</span></div></li>
</ul>  </div>

  
  </div>
  </div>
</div>
</div>
</div>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__VNH5GD8Zz7g6_hGCOZjVjdMndxxC6naiExSSJKX3k_A__kzmEdtbqCsJx5Z9yg2Qy0PptB__ufXm-TxUOl0KZzUw__QJacVmveFyLIBSPkp21xL0ZvuDW9DMBrD2HB1P4Ry5I.js"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__QJacVmveFyLIBSPkp21xL0ZvuDW9DMBrD2HB1P4Ry5I.js" defer="defer"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__llgGUcz2-k3Jmyyyog-_4yI1wJfGWwx0eQ2lbnM9IIo__t6UjIjIKIAD6nKc6BAU9qmlVV2VR4CBTNQbxlbJPgN0__QJacVmveFyLIBSPkp21xL0ZvuDW9DMBrD2HB1P4Ry5I.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
function euCookieComplianceLoadScripts() {}
//--><!]]>
</script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var eu_cookie_compliance_cookie_name = "";
//--><!]]>
</script>
<script type="text/javascript" src="//d33xdlntwy0kbs.cloudfront.net/cshl_custom.js"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__BmqjBnkz3MgYCAoc25s1lDRMEjLhC3mEPVonUFIHi08__Unwv5-ZIuHBfFwytsjEx1niBVJ7n1T4lPws7VrkHXM4__QJacVmveFyLIBSPkp21xL0ZvuDW9DMBrD2HB1P4Ry5I.js"></script>
  </body>
</html>
