<html xmlns="http://www.w3.org/1999/xhtml"> <head> <style type="text/css">
      	     		 
      	    a                          {background : #ffffff; }
      	    article                    {border-style : dotted; border-width : 2px; }
		 	
		 	div                        {background : #ffffcc;}
		 	div.abstract-title         {font-weight : bold ; font-size : 16pt;}
		 	div.ack                    {border-style : solid ; border-color : red; margin : 2em; }
		 	div.article-meta           {border-style : solid; border-width : 1 pt; margin 1em;}
		 	div.boxed-text             {margin : 5em; border-style : solid;}
		 	div.contrib-group          {margin : 1em; }
		 	div.fig                    {border-style : solid; border-width : 2px; margin : 2em; }
		 	div.funding                {font-weight : bold ; font-size : 16pt;}
		 	div.given-names            {font-style : italic;}
		 	div.intro                  {border-style : inset; margin : 5px;}
		 	div.introduction           {border-style : inset; margin : 5px;}
		 	div.journal-meta           {border-style : solid; border-width : 1 pt; margin 1em;}
		 	div.journal-title-group    {background : #ffeeee;}
		 	div.article-title          {font-weight : bold ; font-size : 18pt;}
		 	div.kwd                    {font-style : italic}
		 	div.meta-name              {font-weight : bold ; font-size : 16pt;}
		 	div.name                   {font-weight : bold;}
		 	div.alt-title              {font-style : italic; font-size : 12px;}
		 	
		 	div.sec                    {border : 2px; margin 5px; padding 2px;}
		 	div.title                  {font-family : courier; font-weight : bold;
		 	                            font-size : 16pt; margin : 5px;}
		 	
		 	div.materials_methods      {border-style : double; margin : 5px; }
		 	div.methods                {border-style : double; margin : 5px; }
		 	
		 	div.results                {border-style : solid; margin : 5px;}
		 	div.background             {border-style : dotted; margin : 5px;}
		 	
		 	div.discussion             {border-style : groove; margin : 5px;}
		 	div.conclusion             {border-style : ridge; margin : 5px;}
		 	div.conclusions             {border-style : ridge; margin : 5px;}
		 	div.supplementary-material {border-style : inset; margin : 5px;}
		 	div.abbreviations          {border-style : double; border-color : red; margin : 5px;}
		 	div.competinginterests     {border-style : double; border-color : blue; margin : 5px;}
		 	div.acknowledgements       {border-style : double; border-color : green; margin : 5px;}
		 	div.authors_contributions   {border-style : double; border-color : purple; margin : 5px;}
		 	
		 	div.publisher              {border-style : outset; margin : 5px;}
		 	div.fn-type-conflict       {background : #f88; }
		 	div.fn-type-con            {background : #ddf; }
		 	div.fn-type-other          {background : #ddd; }
		 	
		 	div.unknown                {background : #ffd;
									 	  border-style : solid;
									 	  border-width : 1px;
									 	  padding : 2 px;}
		 	  
      	    table                      {background : #ffffdd;}
		 	tr                         {background : #ddddff; padding : 1px;}
		 	
		 	span                       {background : #ffcccc;}
		 	
		 	span.citation-author       {font-family : helvetica; background : #ffeeee;}
		 	span.collab                {background : #ddffff; }
		 	span.comment               {font-family : courier; font-size : 6px; background : #ffaaff;}
		 	span.contrib               {background : #ffffff;}
		 	span.corresp               {background : #ddffdd; }
		 	span.doi                   {background : #ffffff;}
		 	span.email                 {font-family : courier; }
		 	span.etal                  {font-style : italic;}
		 	span.fpage                 {font-family : courier;}
		 	span.given-names           {background : #ffffff;}
		 	span.iso-abbrev            {background : #ffffff;}
		 	span.issn-epub             {background : #ffffff;}
		 	span.issn-ppub             {background : #ffffff;}
		 	span.journal-title         {background : #ffffff;}
		 	span.lpage                 {font-family : courier;}
		 	span.mixed-article-title   {font-style : italic ;}
		 	span.nlm-ta                {background : #ffffff;}
		 	span.pmc                   {background : #ffffff;}
		 	span.pmcid                 {background : #ffffff;}
		 	span.pmid                  {background : #ffffff;}
		 	span.publisher             {background : #ffffff;}
		 	span.publisher-id          {background : #ffffff;}
		 	span.publisher-name        {background : #ffffff;}
		 	span.source                {background : #ffffff;}
		 	span.subject               {background : #ffffff;}
		 	span.surname               {background : #ffffff;}
		 	span.volume                {font-family : courier; font-weight : bold;}
		 	span.year                  {font-family : courier ; font-style : italic;}
			</style> </head> <body> <div class="front" title="front"> <div class="journal-meta" tagx="journal-meta" title="journal-meta"><span class="nlm-ta" title="nlm-ta">BMC Evol Biol</span><span class="iso-abbrev" title="iso-abbrev">BMC Evol. Biol</span><div class="journal-title-group" tagx="journal-title-group" title="journal-title-group"><span class="journal-title" tagx="journal-title" title="journal-title">BMC Evolutionary Biology</span></div><span class="issn-epub" tagx="issn" title="issn-epub">1471-2148</span><div class="publisher" tagx="publisher" title="publisher"><span class="publisher-name" tagx="publisher-name" title="publisher-name">BioMed Central</span><span class="publisher-loc" tagx="publisher-loc" title="publisher-loc">London</span></div> </div> <div class="article-meta" tagx="article-meta" title="article-meta"><span class="publisher-id" title="publisher-id">1359</span><span class="doi" title="doi"> doi: <a href="https://dx.doi.org/10.1186/s12862-019-1359-6">10.1186/s12862-019-1359-6</a></span><div class="article-categories" title="article-categories">
: <span class="subject" title="subject">Research Article</span></div> <div class="title-group" tagx="title-group" title="title-group"> <div class="article-title" title="article-title">
Odontogenic ameloblast-associated (<i>ODAM</i>) is inactivated in toothless/enamelless placental mammals and toothed whales</div> </div> <div class="contrib-group" title="contrib-group"><span class="contrib" title="contrib"><span class="contrib-id" tagx="contrib-id" title="contrib-id">http://orcid.org/0000-0001-7335-6946</span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Springer</span><span class="given-names" tagx="given-names" title="given-names">Mark S.</span></span><span class="address" title="address"><span class="email" tagx="email" title="email">springer@ucr.edu</span></span><a href="#Aff1">1</a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Emerling</span><span class="given-names" tagx="given-names" title="given-names">Christopher A.</span></span><span class="address" title="address"><span class="email" tagx="email" title="email">drcaemerling@gmail.com</span></span><a href="#Aff2">2</a><a href="#Aff3">3</a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gatesy</span><span class="given-names" tagx="given-names" title="given-names">John</span></span><span class="address" title="address"><span class="email" tagx="email" title="email">jgatesy@amnh.org</span></span><a href="#Aff4">4</a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Randall</span><span class="given-names" tagx="given-names" title="given-names">Jason</span></span><span class="address" title="address"><span class="email" tagx="email" title="email">jrand006@ucr.edu</span></span><a href="#Aff1">1</a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Collin</span><span class="given-names" tagx="given-names" title="given-names">Matthew A.</span></span><span class="address" title="address"><span class="email" tagx="email" title="email">matthew.collin@ucr.edu</span></span><a href="#Aff1">1</a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hecker</span><span class="given-names" tagx="given-names" title="given-names">Nikolai</span></span><span class="address" title="address"><span class="email" tagx="email" title="email">hecker@mpi-cbg.de</span></span><a href="#Aff5">5</a><a href="#Aff6">6</a><a href="#Aff7">7</a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hiller</span><span class="given-names" tagx="given-names" title="given-names">Michael</span></span><span class="address" title="address"><span class="email" tagx="email" title="email">hiller@mpi-cbg.de</span></span><a href="#Aff5">5</a><a href="#Aff6">6</a><a href="#Aff7">7</a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Delsuc</span><span class="given-names" tagx="given-names" title="given-names">Frédéric</span></span><span class="address" title="address"><span class="email" tagx="email" title="email">Frederic.Delsuc@umontpellier.fr</span></span><a href="#Aff2">2</a></span><span class="citation_author_institution" id="Aff1">[1], <div class="institution-wrap" title="institution-wrap"><span class="institution-id" title="institution-id">0000 0001 2222 1582</span><span class="institution-id" title="institution-id">grid.266097.c</span><span class="institution" title="institution">Department of Evolution, Ecology, and Organismal Biology, </span><span class="institution" title="institution">University of California, </span></div></span><span class="citation_author_institution" id="Aff2">[2], <div class="institution-wrap" title="institution-wrap"><span class="institution-id" title="institution-id">0000 0001 2097 0141</span><span class="institution-id" title="institution-id">grid.121334.6</span><span class="institution" title="institution">Institut des Sciences de l’Évolution de Montpellier (ISEM), CNRS, IRD, EPHE, </span><span class="institution" title="institution">Université de Montpellier, </span></div></span><span class="citation_author_institution" id="Aff3">[3], <div class="institution-wrap" title="institution-wrap"><span class="institution-id" title="institution-id">0000 0000 8790 5830</span><span class="institution-id" title="institution-id">grid.422678.d</span><span class="institution" title="institution">Department of Biology, </span><span class="institution" title="institution">Whittier College, </span></div></span><span class="citation_author_institution" id="Aff4">[4], <div class="institution-wrap" title="institution-wrap"><span class="institution-id" title="institution-id">0000 0001 2152 1081</span><span class="institution-id" title="institution-id">grid.241963.b</span><span class="institution" title="institution">Division of Vertebrate Zoology and Sackler Institute for Comparative Genomics, </span><span class="institution" title="institution">American Museum of Natural History, </span></div></span><span class="citation_author_institution" id="Aff5">[5], <div class="institution-wrap" title="institution-wrap"><span class="institution-id" title="institution-id">0000 0001 2113 4567</span><span class="institution-id" title="institution-id">grid.419537.d</span><span class="institution" title="institution">Max Planck Institute of Molecular Cell Biology and Genetics, </span></div></span><span class="citation_author_institution" id="Aff6">[6], <div class="institution-wrap" title="institution-wrap"><span class="institution-id" title="institution-id">0000 0001 2154 3117</span><span class="institution-id" title="institution-id">grid.419560.f</span><span class="institution" title="institution">Max Planck Institute for the Physics of Complex Systems, </span></div></span><span class="citation_author_institution" id="Aff7">[7], <div class="institution-wrap" title="institution-wrap"><span class="institution-id" title="institution-id">grid.495510.c</span><span class="institution" title="institution">Center for Systems Biology Dresden, </span></div></span></div><span class="pub-date-epub" title="pub-date-epub">epub: <span>2019-1-1</span></span><span class="pub-date-pmc-release" title="pub-date-pmc-release">pmc-release: <span>2019-1-1</span></span><span class="pub-date-collection" title="pub-date-collection">collection: <span>2019</span></span><span class="volume" tagx="volume" title="volume">19</span><span class="elocation-id" tagx="elocation-id" title="elocation-id">31</span><span class="history" title="history"><span class="received" title="received">received: 2018-11-18</span><span class="accepted" title="accepted">accepted: 2019-1-11</span></span><div class="permissions"><span class="copyright" title="copyright">(C) , </span><span class="license" title="license"><span class="license-p" title="license-p"><b>Open Access</b>This article is distributed under the terms of the Creative Commons Attribution 4.0 International License (<a href="http://creativecommons.org/licenses/by/4.0/">http://creativecommons.org/licenses/by/4.0/</a>), which permits unrestricted use, distribution, and reproduction in any medium, provided you give appropriate credit to the original author(s) and the source, provide a link to the Creative Commons license, and indicate if changes were made. The Creative Commons Public Domain Dedication waiver (<a href="http://creativecommons.org/publicdomain/zero/1.0/">http://creativecommons.org/publicdomain/zero/1.0/</a>) applies to the data made available in this article, unless otherwise stated.</span></span></div> <div class="abstract" title="abstract"> <div class="abstract-title" title="abstract-title">
Abstract</div> <div class="background" title="sec"> <div class="title" tagx="title" title="title">
Background</div> <p id="Par1">The gene for odontogenic ameloblast-associated (<i>ODAM</i>) is a member of the secretory calcium-binding phosphoprotein gene family. <i>ODAM</i> is primarily expressed in dental tissues including the enamel organ and the junctional epithelium, and may also have pleiotropic functions that are unrelated to teeth. Here, we leverage the power of natural selection to test competing hypotheses that <i>ODAM</i> is tooth-specific versus pleiotropic. Specifically, we compiled and screened complete protein-coding sequences, plus sequences for flanking intronic regions, for <i>ODAM</i> in 165 placental mammals to determine if this gene contains inactivating mutations in lineages that either lack teeth (baleen whales, pangolins, anteaters) or lack enamel on their teeth (aardvarks, sloths, armadillos), as would be expected if the only essential functions of <i>ODAM</i> are related to tooth development and the adhesion of the gingival junctional epithelium to the enamel tooth surface.</p> </div> <div class="results" title="sec"> <div class="title" tagx="title" title="title">
Results</div> <p id="Par2">We discovered inactivating mutations in all species of placental mammals that either lack teeth or lack enamel on their teeth. A surprising result is that <i>ODAM</i> is also inactivated in a few additional lineages including all toothed whales that were examined. We hypothesize that <i>ODAM</i> inactivation is related to the simplified outer enamel surface of toothed whales. An alternate hypothesis is that <i>ODAM</i> inactivation in toothed whales may be related to altered antimicrobial functions of the junctional epithelium in aquatic habitats. Selection analyses on <i>ODAM</i> sequences revealed that the composite dN/dS value for pseudogenic branches is close to 1.0 as expected for a neutrally evolving pseudogene. DN/dS values on transitional branches were used to estimate <i>ODAM</i> inactivation times. In the case of pangolins, <i>ODAM</i> was inactivated ~ 65 million years ago, which is older than the oldest pangolin fossil (<i>Eomanis</i>, 47 Ma) and suggests an even more ancient loss or simplification of teeth in this lineage.</p> </div> <div class="conclusion" title="sec"> <div class="title" tagx="title" title="title">
Conclusion</div> <p id="Par3">Our results validate the hypothesis that the only essential functions of <i>ODAM</i> that are maintained by natural selection are related to tooth development and/or the maintenance of a healthy junctional epithelium that attaches to the enamel surface of teeth.</p> </div> <div class="electronicsupplementarymaterial" title="sec"> <div class="title" tagx="title" title="title">
Electronic supplementary material</div> <p>The online version of this article (10.1186/s12862-019-1359-6) contains supplementary material, which is available to authorized users.</p> </div> </div> <div class="kwd-group"> <div class="title" tagx="title" title="title">
Keywords</div> <div class="kwd" title="kwd">
Edentulism</div> <div class="kwd" title="kwd">
Enamel</div> <div class="kwd" title="kwd">
Junctional epithelium</div> <div class="kwd" title="kwd">
ODAM</div> <div class="kwd" title="kwd">
Pseudogene</div> </div> <div class="funding-group" title="funding-group"> <div class="funding" title="funding">
Funding</div> <div class="award-group" title="award-group"> <div class="funding-source" title="funding-source"> <div class="institution-wrap" title="institution-wrap"><span class="institution-id" title="institution-id">http://dx.doi.org/10.13039/100000001</span><span class="institution" title="institution">National Science Foundation</span></div> </div><span class="award-id" tagx="award-id" title="award-id">DEB-1457735</span><span class="award-id" tagx="award-id" title="award-id">DEB-1457735</span><span class="principal-award-recipient" title="principal-award-recipient"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gatesy</span><span class="given-names" tagx="given-names" title="given-names">John</span></span></span></div> </div> <div class="funding-group" title="funding-group"> <div class="funding" title="funding">
Funding</div> <div class="award-group" title="award-group"> <div class="funding-source" title="funding-source"><span class="institution" title="institution">Marie-Curie</span></div><span class="award-id" tagx="award-id" title="award-id">CEBA: ANR-10-LABX-25-01</span></div> </div> <div class="funding-group" title="funding-group"> <div class="funding" title="funding">
Funding</div> <div class="award-group" title="award-group"> <div class="funding-source" title="funding-source"><span class="institution" title="institution">European Research Council ()</span></div><span class="award-id" tagx="award-id" title="award-id">ERC-2015-CoG-683257</span></div> </div> <div class="funding-group" title="funding-group"> <div class="funding" title="funding">
Funding</div> <div class="award-group" title="award-group"> <div class="funding-source" title="funding-source"><span class="institution" title="institution">Leibniz Association</span></div><span class="award-id" tagx="award-id" title="award-id">SAW-2016-SGN-2</span><span class="principal-award-recipient" title="principal-award-recipient"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hiller</span><span class="given-names" tagx="given-names" title="given-names">Michael</span></span></span></div> </div> <div class="custom-meta-group" title="custom-meta-group"> <meta name="issue-copyright-statement" value="© The Author(s) 2019" /> </div> </div> </div> <div class="body" title="body"> <div class="background" title="sec"> <div class="title" tagx="title" title="title">
Background</div> <p id="Par10">The secretory calcium-binding phosphoprotein (SCPP) gene family contains 23 members in the human genome and achieved its present diversity through an extensive series of gene duplication events [<a href="#CR1">1</a>, <a href="#CR2">2</a>]. In mammals, different SCPP genes are expressed in mineralized tissues, mammary glands, and salivary glands [<a href="#CR2">2</a>]. Among the genes that are expressed in mineralized tissues of vertebrates, some are important for bone and/or dentin formation whereas others are critical for the synthesis of enamel (tetrapods) and/or enameloid (bony fishes) [<a href="#CR2">2</a>]. The gene that encodes odontogenic ameloblast-associated (<i>ODAM</i>) is a member of the SCPP family and is thought to have originated as far back as the common ancestor of tetrapods and bony fishes more than 450 million years ago [<a href="#CR2">2</a>–<a href="#CR4">4</a>]. Early in tooth development, <i>ODAM</i> is highly expressed in the enamel organ that gives rise to ameloblasts that produce enamel. Later in tooth development, <i>ODAM</i> is expressed in the junctional epithelium, which is a specialized epithelium that attaches the soft tissue of the gingiva (gums) to the enamel surface of the tooth and protects against bacteria that promote periodontal disease [<a href="#CR5">5</a>–<a href="#CR9">9</a>]. In this capacity, ODAM plays a critical role in the first line of defense against bacterial invasion [<a href="#CR10">10</a>]. ODAM has also been shown to upregulate the expression of the <i>MMP20</i> gene [<a href="#CR11">11</a>], which encodes a matrix metalloproteinase required for proper enamel development, and to promote the nucleation of hydroxyapatite crystals [<a href="#CR12">12</a>]. Young <i>ODAM</i> knockout mice show no abnormalities in enamel volume, density, and organization, but in older individuals the junctional epithelium shows significant detachment and apical downgrowth [<a href="#CR13">13</a>]. Beyond its expression in ameloblasts and the junctional epithelium, <i>ODAM</i> is expressed in many non-dental tissues in humans [<a href="#CR8">8</a>, <a href="#CR14">14</a>]. Of these, expression levels are strongest in salivary glands, mammary glands, and the trachea [<a href="#CR8">8</a>]. <i>ODAM</i> is also expressed in carcinomas of the stomach, breast, lung, and ovary [<a href="#CR14">14</a>, <a href="#CR15">15</a>].</p> <p id="Par11">Previous studies have shown that one or more of nine tooth-related genes (<i>ACP4, AMBN, AMEL, AMTN, C4orf26</i> (= <i>ODAPH</i>)<i>, DSPP, ENAM, KLK4, MMP20</i>) are inactivated in a variety of toothless vertebrates that have been investigated including birds [<a href="#CR16">16</a>–<a href="#CR19">19</a>], turtles [<a href="#CR20">20</a>], and several mammalian lineages comprising baleen whales, pangolins, anteaters, and Steller’s sea cow [<a href="#CR19">19</a>, <a href="#CR21">21</a>–<a href="#CR28">28</a>]. Many of these genes are also inactivated in mammals with enamelless teeth such as pygmy and dwarf sperm whales, aardvarks, sloths, and armadillos [<a href="#CR19">19</a>, <a href="#CR23">23</a>, <a href="#CR24">24</a>, <a href="#CR28">28</a>–<a href="#CR30">30</a>]. The widespread inactivation of these genes in edentulous and enamelless vertebrates suggests that their only essential functions are related to tooth development even if they are sometimes expressed in non-dental tissues [<a href="#CR19">19</a>].</p> <p id="Par12">Given its important role in amelogenesis, <i>ODAM</i> has emerged as yet another gene that should be screened in toothless and enamelless vertebrates for evidence of inactivating mutations. If <i>ODAM</i> is consistently inactivated in vertebrates that lack teeth or the outer enamel covering on their teeth, then <i>ODAM</i>’s only essential functions would appear to be tooth-related even though <i>ODAM</i> is expressed in other tissues [<a href="#CR14">14</a>]. By contrast, if <i>ODAM</i> gene sequences are intact in toothless and enamelless vertebrates this would provide evidence for pleiotropic, non-dental functions that affect fitness and are protected by natural selection.</p> <p id="Par13">Here, we test the hypothesis that the main function of <i>ODAM</i> is related to enamel formation by investigating whether this gene remains under purifying selection in placental mammals that have lost their teeth or the outer enamel covering on their teeth. To this end, we compiled and screened complete protein coding sequences of the <i>ODAM</i> gene for inactivating mutations in 165 placental mammal species comprising representatives of all extant placental orders. In support of our hypothesis, we found that all lineages without teeth (baleen whales, pangolins, anteaters) or without enamel on their teeth (aardvark, sloths, armadillos) have inactivated <i>ODAM</i>. Finally, selection analyses to estimate when purifying selection on <i>ODAM</i> was relaxed in lineages with inactivated <i>ODAM</i> provide new insights into when teeth or their outer enamel caps were lost in these lineages.</p> </div> <div class="methods" title="sec"> <div class="title" tagx="title" title="title">
Methods</div> <div class="taxonsampling" title="sec"> <div class="title" tagx="title" title="title">
Taxon sampling</div> <p id="Par14">Taxon sampling included 165 placental mammal species, of which 135 are from assembled genomes in public databases, 13 are from unassembled sequence reads in the Sequence Read Archive (SRA) or European Nucleotide Archive (ENA), eight are from a combination of assembled genomes and unassembled sequence reads, and nine are from new Illumina HiSeq2500, HiSeq4000, HiSeqX10, or NovaSeq data (Additional file <a href="#MOESM1">1</a>: Table S1). Taxon sampling included 12 taxa from Afrotheria, ten taxa from Xenarthra, 61 taxa from Euarchontoglires, and 82 taxa from Laurasiatheria. Among the afrotherians, three taxa (<i>Mammut americanum</i> [American mastodon], <i>Mammuthus primigenius</i> [woolly mammoth], <i>Palaeoloxodon antiquus</i> [straight-tusked elephant]) are extinct. Toothless taxa included two vermilinguans (anteaters), two pholidotans (pangolins), and seven mysticetes (baleen whales); taxa that lack enamel on their teeth included one tubulidentate (aardvark), two folivorans (sloths), and six cingulatans (armadillos).</p> </div> <div class="blastsearchesandalignments" title="sec"> <div class="title" tagx="title" title="title">
BLAST searches and alignments</div> <p id="Par15">Genomic sequences encoding <i>ODAM</i> were downloaded from NCBI for representatives of Euarchontoglires (<i>Homo sapiens</i>), Laurasiatheria (<i>Bos taurus</i>), and Afrotheria (<i>Trichechus manatus</i>). These sequences were employed in BLAST searches against other placental mammals in NCBI’s 'RefSeq Genome' and 'Whole-genome shotgun contigs' databases. We used megablast for highly similar sequences and the more sensitive blastn for less similar sequences when megablast searches failed to return results. Complete coding sequences and intervening introns were imported into Geneious 11.1.5 [<a href="#CR31">31</a>] and aligned with MAFFT [<a href="#CR32">32</a>, <a href="#CR33">33</a>] with minor adjustments by eye. For five xenarthrans (<i>Cabassous unicinctus</i>, <i>Tolypeutes matacus</i>, <i>Euphractus sexcinctus</i>, <i>Choloepus didactylus</i>, <i>Tamandua tetradactyla</i>), we imported Discovar de novo (<a href="https://software.broadinstitute.org/software/discovar/blog">https://software.broadinstitute.org/software/discovar/blog</a>/) genome assemblies into Geneious [<a href="#CR12">12</a>] and performed local BLAST searches (discontiguous megablast) using reference xenarthran sequences (<i>Dasypus novemcinctus</i>, <i>Choloepus hoffmanni</i>). With a few exceptions (see below), exon-intron boundaries are conserved among placental mammals. In cases where there were suspected assembly gaps with missing exons, we used sequences from the closest available taxon (exon plus ~ 100 bp of each flanking intron) to perform additional BLAST searches with megablast against the Sequence Read Archive (SRA). Sequences from SRA BLAST searches were imported into Geneious and mapped to the closest reference sequence to assemble missing exons and their flanking intron sequences. A similar strategy was employed to retrieve complete protein-coding sequences for <i>ODAM</i> in taxa with unassembled genomes in the SRA. We also used a mapping to reference approach in Geneious to retrieve <i>ODAM</i> sequences from unpublished Illumina data for <i>Choeropsis liberiensis</i> (pygmy hippopotamus) and three xenarthrans. For <i>C. liberiensis</i>, we mapped reads to the <i>ODAM</i> sequence for <i>Hippopotamus amphibius</i> (river hippopotamus). For the three xenarthrans (<i>Chlamyphorus truncatus</i>, <i>Calyptophractus retusus</i>, <i>Cyclopes didactylus</i>), we imported short reads into Geneious and mapped them to various xenarthran reference sequences. Individual exon sequences (and flanking introns) from each SRA taxon were concatenated into a single sequence with Geneious and aligned to the global alignment.</p> </div> <div class="inactivatingmutations" title="sec"> <div class="title" tagx="title" title="title">
Inactivating mutations</div> <p id="Par16">We inspected the final alignment (165 taxa) for inactivating mutations including exon deletions, frameshift mutations, altered start and stop codons, premature stop codons, and splice site mutations. We also used the alignment program MACSE v2 [<a href="#CR34">34</a>, <a href="#CR35">35</a>] to crosscheck the results of manual inspections for frameshifts and premature stop codons in coding sequences. Parsimony optimizations with delayed transformation (deltran) were performed with PAUP* 4.0a150 [<a href="#CR36">36</a>] to map inactivating mutations to branches of the phylogeny of placental mammals.</p> </div> <div class="phylogeneticanalyses" title="sec"> <div class="title" tagx="title" title="title">
Phylogenetic analyses</div> <p id="Par17">RAxML 8.2.10 [<a href="#CR37">37</a>, <a href="#CR38">38</a>] on CIPRES [<a href="#CR39">39</a>] was used to estimate a maximum likelihood tree for the complete protein-coding sequence alignment. Rapid bootstrap analysis (500 pseudoreplications) [<a href="#CR40">40</a>] and a search for the best tree were performed in a single run. These analyses were performed with a GTR + Γ model of sequence evolution.</p> </div> <div class="selectionanalyses" title="sec"> <div class="title" tagx="title" title="title">
Selection analyses</div> <p id="Par18">We first reconstructed the evolution of the dN/dS ratio of <i>ODAM</i> across the placental mammal phylogeny using the Bayesian approach implemented in Coevol 1.4b [<a href="#CR41">41</a>] on the complete alignment of 165 sequences containing 279 aligned codons. We used the dsom estimation procedure that jointly estimates branch specific dN/dS ratios, divergence times, body sizes, generation times, and ages at sexual maturity modeled as a multivariate Brownian diffusion process [<a href="#CR42">42</a>]. DN/dS analyses with Coevol were performed to detect and visualize patterns of dN/dS variation across a tree. The assumed topology was the <i>ODAM</i> gene tree after making several rearrangements to correct for discrepancies between the gene tree and species tree [<a href="#CR43">43</a>–<a href="#CR46">46</a>] as follows: <i>Chaetophractus</i> sister to other Chlamyphoridae; <i>Orycteropus</i> sister to Afrosoricida + Macroscelidea; <i>Solenodon</i> sister to other Eulipotyphla; <i>Sus</i> sister to Ruminantia + Whippomorpha; <i>Physeter</i> sister to other Odontoceti; Pholidota sister to Carnivora; <i>Rhinolophus</i> + <i>Hipposideros</i> sister to <i>Megaderma</i>; and <i>Nomascus</i> sister to Hominidae. We assumed fossil calibrations from previous studies [<a href="#CR44">44</a>, <a href="#CR46">46</a>, <a href="#CR47">47</a>] and extracted life history traits from PanTHERIA [<a href="#CR48">48</a>]. We set the prior on the root node to 97 Ma with a standard deviation of 20 Ma following the dating of Meredith et al. [<a href="#CR44">44</a>]. We ran two independent MCMC for a total of 5000 cycles sampling parameters every cycle. After checking for convergence by monitoring the effective sample size of the different parameters using the tracecomp command, we excluded the first 500 points of each MCMC as burnin, and made inferences from the remaining 4500 sampled points of each chain.</p> <p id="Par19">Additional dN/dS analyses were performed with the codeml program of PAML 4.4 [<a href="#CR49">49</a>]. Codeml does not employ the autocorrelative model of Brownian motion [<a href="#CR50">50</a>] that is incorporated into Coevol. For computational efficiency and tractability, codeml analyses were performed with 96 taxa after pruning all but one representative for mammalian families with <i>ODAM</i> sequences that are intact for all exemplars included in our study. We used 15 dN/dS categories based on reconstructions of ancestral states for tooth/enamel loss [<a href="#CR23">23</a>, <a href="#CR24">24</a>] and the occurrence of frameshift mutations, premature stop codons, and/or exon deletions in independent lineages. We ignored taxa (<i>Hyaena hyaena, Elaphurus davidianus, Eulemur flavifrons</i> [see below]) that contained just a single splice site mutation except for <i>Physeter macrocephalus.</i> This taxon is of interest because the next possible donor splice site (GT) for intron 4 is 75 bp downstream from the canonical splice site and has a very low MaxEntScan score of − 17 (<a href="http://genes.mit.edu/burgelab/maxent/Xmaxentscan_scoreseq.html">http://genes.mit.edu/burgelab/maxent/Xmaxentscan_scoreseq.html</a> [<a href="#CR51">51</a>]). (For reference, splice sites are predicted to be functional when they have MaxEntScan scores of &amp;gt; 3 [<a href="#CR52">52</a>]). Moreover, all other toothed whales have one or more frameshift mutations and/or premature stop codons in <i>ODAM</i>. The 15 categories that we employed in dN/dS analyses are as follows: one category for branches leading to intact sequences that lack inactivating mutations; one category for pseudogenic branches that postdate tooth or enamel loss (parsimony deltran optimization) and/or the occurrence of an inactivating mutation on an earlier branch; nine categories for transitional branches that record the first inactivating mutation in <i>ODAM</i> (<i>Dasypus novemcinctus</i>; stem Chlamyphoridae; <i>Orycteropus afer</i>; stem Pholidota; <i>Neomonachus schauinslandi</i>; <i>Lipotes vexillifer</i>; stem to Delphinidae + Monodontidae + Phocoenidae; <i>Physeter macrocephalus; Callithrix jacchus</i>); one category for the stem Pilosa branch where enamel was lost [<a href="#CR23">23</a>]; one category for the stem Mysticeti branch where teeth were lost [<a href="#CR24">24</a>]; one category for the stem branch leading to proboscideans with missing exon 6; and one category for proboscidean branches that post-date the loss of exon 6. We employed the same topological corrections for the species tree as detailed above for the Coevol analysis. Codeml analyses were performed with two different codon frequency models, CF1 and CF2. Codon frequencies are estimated from mean nucleotide frequencies across all three codon positions in CF1 and from mean nucleotide frequencies at each of the individual codon positions in CF2. All frameshift insertions were removed from the alignment prior to codeml analyses, as were in-frame insertions that are unique to one or a few taxa. Also, premature stop codons were recoded as missing data. The final alignment for codeml analyses contained 837 aligned positions (279 codons).</p> </div> <div class="estimationofodaminactivationtimes" title="sec"> <div class="title" tagx="title" title="title">
Estimation of ODAM inactivation times</div> <p id="Par20">We used equations from Meredith et al. [<a href="#CR23">23</a>] to estimate when <i>ODAM</i> was inactivated in different placental lineages. Calculations were performed with dN/dS values that were obtained using two different codon models in codeml (CF1, CF2), fixed (1.0) versus estimated values for the dN/dS value on fully pseudogenic branches, and equations that allow for one versus two synonymous substitution rates [<a href="#CR23">23</a>]. Divergence times for relevant nodes in these calculations were taken from Springer et al. [<a href="#CR53">53</a>] for <i>Callithrix</i> to <i>Aotus</i>, Delsuc et al. [<a href="#CR45">45</a>] for all divergences in Xenarthra, Gaubert et al. [<a href="#CR54">54</a>] for <i>Manis pentadactyla</i> to <i>M. javanica</i>, and Foley et al. [<a href="#CR46">46</a>] for all other divergence dates.</p> </div> <div class="screenforconvergentlyinactivatedgenesinedentulousandenamellessmammals" title="sec"> <div class="title" tagx="title" title="title">
Screen for convergently inactivated genes in edentulous and enamelless mammals</div> <p id="Par21">We performed a genomic screen for convergently inactivated genes based on a Forward Genomics approach with a multispecies genome alignment [<a href="#CR28">28</a>]. Briefly, this screen is based on the percentage of the reading frame of the protein-coding gene that is intact and is calculated based on the relative position of inactivating mutations and the partial and/or complete loss of entire exons. This percentage is hereafter denoted as %intact. We considered 61 placental mammals for which inactivating mutations have been computed previously [<a href="#CR28">28</a>]. These species include the edentulous <i>Balaenoptera acutorostrata scammoni</i> and <i>Manis pentadactyla</i> and the enamelless <i>Orycteropus afer</i> and <i>Dasypus novemcinctus</i>, representing four independent lineages. In contrast to a previous screen [<a href="#CR28">28</a>], we performed a more sensitive search by classifying a gene as <i>inactivated</i> if %intact is &amp;lt; 80% and classifying a gene as <i>intact</i> if %intact is ≥90%. We then screened for genes that are <i>inactivated</i> in at least two of the four edentulous or enamelless species, but are <i>intact</i> in at least 90% of species that have tooth enamel. The resulting candidate list of the screen was then sorted (largest to smallest) by the number of species without teeth or enamel that exhibit an <i>inactivated</i> gene (%intact &amp;lt; 80%) and, in case of a tie, sorted (smallest to largest) by the number of species with tooth enamel that do not have an <i>intact</i> version of the gene (%intact &amp;lt; 90%). Finally, we used a phylogenetic generalized least squares approach [<a href="#CR55">55</a>] to compute a phylogeny-corrected <i>P</i>-value for the association between %intact and the presence/absence of enamel.</p> </div> </div> <div class="results" title="sec"> <div class="title" tagx="title" title="title">
Results</div> <div class="multiplealignmentandphylogeneticanalysis" title="sec"> <div class="title" tagx="title" title="title">
Multiple alignment and phylogenetic analysis</div> <p id="Par22">The 165-taxon alignment for the complete protein-coding sequence of <i>ODAM</i> is 1177 nucleotides including frameshift insertions and is available at TreeBASE (<a href="http://purl.org/phylo/treebase/phylows/study/TB2:S23531">http://purl.org/phylo/treebase/phylows/study/TB2:S23531</a>). The shape parameter of the gamma distribution was estimated at 1.98 by RAxML. Figure <a href="#Fig1">1</a> shows the maximum likelihood gene tree (ln L = − 27,926.663886). This gene tree was not rooted with a marsupial, but is consistent with the monophyly of the four major clades of placental mammals (Xenarthra, Afrotheria, Laurasiatheria, Euarchontoglires) [<a href="#CR56">56</a>–<a href="#CR64">64</a>]. Other superordinal groups that were recovered on the <i>ODAM</i> gene tree include Afroinsectiphilia, Paenungulata, Variamana, Euungulata, Glires, and Euarchonta. Thirteen of 19 placental orders included more than one species and in all cases these orders were recovered as monophyletic. Several features of the <i>ODAM</i> gene tree are in minor conflict with generally accepted species trees including the placement of <i>Physeter</i> as the sister group to mysticetes rather than other odontocetes [<a href="#CR65">65</a>, <a href="#CR66">66</a>], an association of Pholidota with Chiroptera rather than Carnivora [<a href="#CR44">44</a>], a sister group relationship between Hipposideridae + Rhinolophidae and yangochiropteran bats instead of with other rhinolophoid bats [<a href="#CR43">43</a>, <a href="#CR67">67</a>], an association of chlamyphorine armadillos (<i>Chlamyphorus, Calyptophractus</i>) with <i>Chaetophractus</i> rather than with tolypeutines (<i>Tolypeutes, Cabassous</i>) [<a href="#CR45">45</a>], a basal split between <i>Condylura</i> and other eulipotyphlans instead of between <i>Solenodon</i> and other eulipotyphlans [<a href="#CR44">44</a>, <a href="#CR68">68</a>, <a href="#CR69">69</a>], and the nonmonophyly of Elephantidae with the mammutid <i>Mammut americanum</i> nested inside of this clade instead of sister to this clade [<a href="#CR70">70</a>, <a href="#CR71">71</a>]. </p><div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/"><span class="label" tagx="label" title="label">Fig. 1</span></a></div>   <p>Maximum likelihood phylogram (ln L = − 27,926.663886) based on <i>ODAM</i> protein-coding sequences for 165 placental mammal species. Pink boxes indicate taxa with inactivating mutations in <i>ODAM</i>; green boxes indicate proboscideans that are missing exon 6 of <i>ODAM</i>. The tree is rooted between Atlantogenata and Boreoeutheria. Some clades for which all of the constituent taxa have intact <i>ODAM</i> sequences were collapsed, and the length of the Glires clade branch was halved to improve aesthetics (see TreeBASE [<a href="http://purl.org/phylo/treebase/phylows/study/TB2:S23531">http://purl.org/phylo/treebase/phylows/study/TB2:S23531</a>] for ML phylogram with all 165 taxa)</p>   </div> <p /> </div> <div class="inactivatingmutations" title="sec"> <div class="title" tagx="title" title="title">
Inactivating mutations</div> <p id="Par23">One or more inactivating mutations (frameshifts, altered start and stop codons, premature stop codons, splice site mutations) were discovered in all placental taxa without teeth or without enamel on their teeth. Table <a href="#Tab1">1</a> provides a complete list of inactivating mutations in these taxa and examples of inactivating mutations are shown in Fig. <a href="#Fig2">2</a>. Shared inactivating mutations were discovered in Vermilingua, <i>Choloepus</i>, Chlamyphoridae, Tolypeutinae, <i>Manis</i>, Balaenidae, Balaenopteroidea, and in the two minke whales (<i>Balaenoptera acutorostrata</i> and <i>B. bonaerensis</i>). Among edentulous forms, taxa with the largest number of inactivating mutations are <i>Manis javanica</i> (15), <i>Manis pentadactyla</i> (14), and <i>Tamandua tetradactyla</i> (14), whereas the fewest inactivating mutations are found in <i>Balaena mysticetus</i> (1) and <i>Balaenoptera musculus</i> (1). Among taxa with enamelless teeth, <i>Orycteropus afer</i> and <i>Tolypeutes matacus</i> each have four inactivating mutations whereas <i>Choloepus didactylus</i> only has one inactivating mutation. However, it should be noted that only 8 of 10 exons had BLAST results for <i>C. didactylus</i>. An association between <i>ODAM</i> inactivation and the loss of teeth or enamel is further supported by a genomic screen for genes that have inactivating mutations and an abolished reading frame in edentulous and enamelless taxa, which uncovered <i>ODAM</i> at rank 7 together with other tooth-related genes (Additional file <a href="#MOESM2">2</a>: Table S2). <div class="table-wrap_UNKNOWN" id="Tab1"><span class="label" tagx="label" title="label">Table 1</span> </div></p><p>Inactivating mutations and whole exon deletions in the <i>ODAM</i> gene of toothless, enamelless, and other placental mammals</p>  <table frame="hsides" rules="groups"> <thead> <tr> <th>Taxon</th> <th>Exon 1</th> <th>Exon 2</th> <th>Exon 3</th> <th>Exon 4</th> <th>Exon 5</th> <th>Exon 6</th> <th>Exon 7</th> <th>Exon 8</th> <th>Exon 9</th> <th>Exon 10</th> <th>Splice site mutations<sup>a</sup></th> </tr> </thead> <tbody> <tr> <td><i>Orycteropus</i></td> <td /> <td /> <td>123-125S</td> <td>256I, 348D</td> <td /> <td /> <td /> <td /> <td>1081I</td> <td>1139-1141S</td> <td>In3Do (AT)</td> </tr> <tr> <td>Vermilingua</td> <td /> <td /> <td /> <td /> <td /> <td /> <td /> <td /> <td>790-792S</td> <td /> <td /> </tr> <tr> <td><i>Tamandua tetradactyla</i></td> <td>29I</td> <td /> <td>105D</td> <td>306-308S, 336I, 374-375D</td> <td>430D</td> <td>546-548S, 573-575S</td> <td>668-670S</td> <td>707-713D</td> <td>797-799S, 1103-1112D</td> <td /> <td>In2Do (AT)</td> </tr> <tr> <td><i>Cyclopes didactylus</i></td> <td /> <td /> <td>127D</td> <td /> <td /> <td /> <td>671I</td> <td>NRM</td> <td /> <td>NRM</td> <td>In1Ac (AT), In4Do (GG), In6Do (AT), In7Do (AT)</td> </tr> <tr> <td><i>Choloepus</i></td> <td /> <td /> <td /> <td /> <td /> <td /> <td /> <td>745-754D</td> <td /> <td /> <td /> </tr> <tr> <td><i>Choloepus hoffmanni</i></td> <td /> <td /> <td /> <td>400-402S</td> <td /> <td /> <td /> <td /> <td /> <td /> <td /> </tr> <tr> <td><i>Choloepus didactylus</i></td> <td /> <td>NBR</td> <td /> <td /> <td /> <td /> <td /> <td /> <td /> <td>NBR</td> <td /> </tr> <tr> <td><i>Dasypus novemcinctus</i></td> <td /> <td /> <td>138-140S</td> <td>400-402S</td> <td /> <td>606-608S</td> <td /> <td /> <td>1127-1129S</td> <td /> <td /> </tr> <tr> <td>Chlamyphoridae</td> <td /> <td>93-98D<sup>b</sup></td> <td /> <td /> <td /> <td /> <td /> <td /> <td /> <td /> <td /> </tr> <tr> <td><i>Chaetophractus vellerosus</i></td> <td /> <td /> <td /> <td /> <td /> <td /> <td>672-674S</td> <td /> <td>820D</td> <td /> <td>In9Ac (CG)</td> </tr> <tr> <td><i>Cabassous</i> + <i>Calyptophractus</i></td> <td /> <td /> <td /> <td /> <td /> <td /> <td /> <td /> <td /> <td /> <td>In4Ac (TG)<sup>c</sup></td> </tr> <tr> <td>Tolypeutinae</td> <td /> <td /> <td>144-146S</td> <td /> <td /> <td>570-573D</td> <td /> <td /> <td /> <td /> <td>In6Ac (AA), In9Ac (AA)</td> </tr> <tr> <td><i>Tolypeutes matacus</i></td> <td>1-3SCD</td> <td /> <td /> <td /> <td>NBR</td> <td /> <td>672-674S</td> <td /> <td /> <td /> <td>In8Do (TT)</td> </tr> <tr> <td><i>Cabassous unicinctus</i></td> <td /> <td /> <td /> <td /> <td /> <td>595-598D</td> <td /> <td>NBR</td> <td>NBR</td> <td /> <td /> </tr> <tr> <td><i>Chlamyphorus truncatus</i></td> <td>NRM</td> <td>NRM</td> <td>NRM</td> <td>NRM</td> <td>NRM</td> <td /> <td>672-674S</td> <td /> <td /> <td /> <td>In8Do (RT)</td> </tr> <tr> <td><i>Calyptophractus retusus</i></td> <td /> <td /> <td /> <td /> <td /> <td /> <td /> <td /> <td>1092D</td> <td /> <td /> </tr> <tr> <td><i>Manis</i></td> <td /> <td /> <td /> <td>195D, 265-268D, 345D</td> <td /> <td>518D, 561D</td> <td>657-658I, 672-674S</td> <td>687I</td> <td>1088D</td> <td /> <td>In4Do (AT), In5Do (CT), In6Do (CT)</td> </tr> <tr> <td><i>Manis pentadactyla</i></td> <td /> <td /> <td>123-125S</td> <td>203I</td> <td /> <td /> <td /> <td /> <td /> <td /> <td /> </tr> <tr> <td><i>Manis javanica</i></td> <td /> <td /> <td /> <td>350D</td> <td /> <td /> <td /> <td /> <td>851-1059I</td> <td /> <td>In9Do (AC)</td> </tr> <tr> <td>Balaenidae</td> <td /> <td /> <td /> <td>420D</td> <td /> <td /> <td /> <td /> <td /> <td /> <td /> </tr> <tr> <td><i>Eubalaena glacialis</i></td> <td /> <td /> <td /> <td /> <td /> <td /> <td /> <td /> <td /> <td /> <td>In6Do (AT)</td> </tr> <tr> <td>Balaenopteroidea</td> <td /> <td /> <td /> <td /> <td /> <td /> <td /> <td /> <td /> <td /> <td>In2Do (AT)</td> </tr> <tr> <td><i>Eschrichtius robustus</i> + <i>Balaenoptera acutorostrata</i> + <i>B. bonaerensis</i> + <i>B. borealis</i></td> <td /> <td /> <td /> <td /> <td /> <td>513-515S</td> <td /> <td /> <td /> <td /> <td /> </tr> <tr> <td><i>Balaenoptera acutorostrata</i> + <i>B. bonaerensis</i></td> <td /> <td /> <td /> <td>263D</td> <td /> <td /> <td /> <td /> <td /> <td /> <td /> </tr> <tr> <td><i>Balaenoptera bonaerensis</i></td> <td /> <td /> <td /> <td>382-384S</td> <td /> <td /> <td /> <td /> <td /> <td /> <td /> </tr> <tr> <td><i>Balaenoptera borealis</i></td> <td /> <td /> <td /> <td /> <td /> <td /> <td /> <td /> <td /> <td>1159-1165D<sup>d</sup></td> <td /> </tr> <tr> <td><i>Physeter macrocephalus</i></td> <td /> <td /> <td /> <td /> <td /> <td /> <td /> <td /> <td /> <td /> <td>In4Do (AT)</td> </tr> <tr> <td><i>Lipotes vexillifer</i></td> <td /> <td /> <td /> <td>195-197S, 264-266S, 288-290S</td> <td /> <td /> <td /> <td /> <td /> <td /> <td /> </tr> <tr> <td>Delphinidae + Monodontidae + Phocoenidae</td> <td /> <td /> <td /> <td>232I, 424-425I</td> <td /> <td /> <td /> <td /> <td /> <td /> <td /> </tr> <tr> <td>Phocoenidae</td> <td /> <td /> <td /> <td /> <td /> <td /> <td /> <td /> <td>793I</td> <td /> <td /> </tr> <tr> <td><i>Phocoena phocoena</i></td> <td /> <td /> <td /> <td /> <td /> <td /> <td /> <td /> <td /> <td /> <td>In2Ac (AT)</td> </tr> <tr> <td><i>Tursiops truncatus</i></td> <td /> <td /> <td /> <td /> <td /> <td /> <td /> <td /> <td>793I</td> <td /> <td /> </tr> <tr> <td><i>Neomonachus schauninslandi</i></td> <td /> <td /> <td>104-105D</td> <td>198-200S</td> <td>429-479D</td> <td /> <td /> <td /> <td /> <td /> <td>In2Ac (AA)</td> </tr> <tr> <td><i>Callithrix jacchus</i></td> <td /> <td /> <td /> <td /> <td>477-479S</td> <td /> <td /> <td /> <td /> <td /> <td>In4Ac (AC)</td> </tr> </tbody> </table> <div class="table-wrap-foot_UNKNOWN"> <p><sup>a</sup>Mutated splice sites are shown in parentheses</p> <p><sup>b</sup>Deletion includes last six bp of exon 2 and first nine bp of intron 2 including donor splice site</p> <p><sup>c</sup>No blast results for <i>Chlamyphorus</i> and <i>Tolypeutes</i></p> <p><sup>d</sup>Eight bp deletion that includes stop codon</p> <p>Abbreviations: <i>Ac</i> acceptor splice site, <i>D</i> frameshift deletion, <i>Do</i> donor splice site, <i>I</i> frameshift insertion, <i>In</i> intron, <i>NBR</i> no blast results, <i>NRM</i> no reads mapped, <i>S</i> premature stop codon relative to original reading frame, <i>SCM</i> start codon mutation</p> <p>Position numbers correspond to the complete protein-coding alignment for 165 placental mammals</p> </div>  <div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/"><span class="label" tagx="label" title="label">Fig. 2</span></a></div>   <p>Examples of inactivating mutations in <i>ODAM</i>. <b>a</b>) <i>Orycteropus afer</i> (aardvark). <b>b</b>) Xenarthra (anteaters, sloths, armadillos). <b>c</b>) <i>Manis</i> spp. (pangolins). <b>d</b>) Cetacea (whales, dolphins, porpoises). Green boxes denote frameshift insertions; blue boxes denote frameshift deletions; and purple boxes denote premature stop codons. <i>Loxodonta</i>, <i>Procavia</i>, <i>Hyaena</i>, <i>Canis</i>, and <i>Hippopotamus</i> have intact <i>ODAM</i> sequences and were included in different alignment panels to provide context for inactivating mutations</p>   </div> <p /> <p id="Par24">In addition to inactivating mutations in edentulous and enamelless taxa, we also discovered inactivating mutations in all eight representatives of Odontoceti including inactivating mutations that are shared by members of the clade Delphinoidea (Phocoenidae [3 spp.], Monodontidae [1 sp.], and Delphinidae [2 spp.]). The number of inactivating mutations among different odontocete species ranges from one in <i>Physeter macrocephalus</i> (splice site mutation) to four in the three phocoenid species (<i>Phocoena phocoena</i>, <i>Neophocaena phocaenoides</i>, <i>N. asiaeorientalis</i>).</p> <p id="Par25">Beyond these odontocetes, additional taxa with enamel-capped teeth had putative inactivating mutations. The phocid (seal) <i>Neomonachus schauinslandi</i> has one 2-bp frameshift deletion, one premature stop codon, an acceptor splice site mutation (intron 2 acceptor, AG = &amp;gt; AA), and a putative deletion of exon 5. However, genomic Illumina data are not available for <i>Neomonachus,</i> and we were unable to validate these mutations. By contrast, the New World primate <i>Callithrix jacchus</i> has a premature stop codon in exon 5 and an acceptor splice site mutation (AC) in intron 4, both of which are confirmed by SRA data. The mustelid <i>Enhydra lutris</i> has a 1-bp frameshift insertion in exon 1. Here, Illumina data for three individuals of <i>Enhydra</i> (SRR5768046, SRR5768052, SRR6450476) suggest that this deletion either has allelic variation or that there is an additional paralog of exon 1 in the genome of this species. Finally, there are splice site mutations (relative to canonical splice sites) in the strepsirrhine primate <i>Eulemur flavifrons</i> (intron 7 acceptor, AG = &amp;gt; TT), in the carnivoran <i>Hyaena hyaena</i> (intron 8 donor, GT = &amp;gt; TT), and in the deer <i>Elaphurus davidianus</i> (intron 5 donor, GT = &amp;gt; CT). However, all of these splice site mutations can be accommodated by alternative splice sites that result in slightly shorter exons (3 bp shorter in <i>Elaphurus</i>, 6 bp shorter in <i>Hyaena</i>, 18 bp shorter in <i>Eulemur</i>), and such evolutionary splice site shifts provide no indication for gene loss [<a href="#CR72">72</a>].</p> <p id="Par26">Finally, there were no BLAST results for exon 6 of <i>ODAM</i> in three elephantids (<i>Mammuthus primigenius, Palaeoloxodon antiquus, Loxodonta cyclotis</i>) and the mammutid <i>Mammut americanum</i> (Fig. <a href="#Fig3">3</a>). By contrast, this exon is present in the elephantids <i>Loxodonta africana</i> and <i>Elephas maximus</i>. </p><div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/"><span class="label" tagx="label" title="label">Fig. 3</span></a></div>   <p>Map to reference coverage of Illumina reads onto a contiguous block of <i>Loxodonta africana</i> (African savannah elephant) <i>ODAM</i> that begins with intron 5 and ends with exon 10. Map to reference results suggest that exon 6 is retained in <i>Elephas maximus</i> (Asian elephant) but is missing in several other proboscideans (<i>Loxodonta cyclotis</i> [African forest elephant]<i>, Mammuthus primigenius</i> [woolly mammoth]<i>, Palaeoloxodon antiquus</i> [straight-tusked elephant]<i>, Mammut americanum</i> [American mastodon])</p>   </div> <p /> </div> <div class="selectionanalyses" title="sec"> <div class="title" tagx="title" title="title">
Selection analyses</div> <p id="Par27">The joint Bayesian reconstruction of dN/dS across the placental phylogeny highlights large variation of selection pressure among lineages. Mean dN/dS values for functional branches range from 0.36 in the kangaroo rat <i>Dipodomys ordii</i> to 0.70 in the bonobo <i>Pan paniscus</i> and are compatible with purifying selection (Fig. <a href="#Fig4">4</a>). At the ordinal level, rodents and bats show the lowest average dN/dS values of 0.41 and 0.48, respectively, whereas functional cetartiodactyl and primate sequences exhibit larger mean values of 0.59 and 0.60, respectively. These results illustrate the influence of different life-history traits. A consequence of relaxed selection constraints is that the non-functional sequences show elevated dN/dS values for pilosans (0.79), cingulatans (0.81), aardvark (0.73), pangolins (0.70), odontocetes (0.75) and mysticetes (0.87). However, the Hawaiian monk seal (<i>Neomonachus schauinslandi</i>) and the common marmoset (<i>Callithrix jacchus</i>), which present inactivating mutations in their <i>ODAM</i> sequences, show only moderately elevated dN/dS values of 0.58 and 0.54, respectively. Transitional branches where enamel loss was inferred based on ancestral reconstructions [<a href="#CR23">23</a>, <a href="#CR24">24</a>] all have elevated dN/dS values: stem Pilosa (0.75), stem Chlamyphoridae (0.81), stem Pholidota (0.70), and stem Mysticeti (0.82). </p><div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/"><span class="label" tagx="label" title="label">Fig. 4</span></a></div>   <p>Bayesian reconstruction of dN/dS for 165 <i>ODAM</i> sequences across the placental phylogeny. The variation of dN/dS was jointly reconstructed with divergence times while controlling the effect of three life-history traits (body mass, longevity, and maturity). Asterisks at the tips of terminal branches indicate non-functional sequences (pseudogenes). Stars indicate branches on which shared inactivating mutations were inferred in toothless or enamelless clades. The tree is rooted between Atlantogenata and Boreoeutheria. Placental orders are delimited to the right of species tree tips</p>   </div> <p /> <p id="Par28">The results of selection (dN/dS) analyses using codeml are summarized in Table <a href="#Tab2">2</a>. The dN/dS values for functional branches that lead to taxa with intact protein-coding sequences for <i>ODAM</i> and enamel-capped teeth are ~ 0.49 and ~ 0.52 with codon frequency model 1 (CF1) and codon frequency model 2 (CF2), respectively, confirming that <i>ODAM</i> has evolved overall under purifying selection. By contrast, the values for pseudogenic branches that post-date the first occurrence of an inactivating mutation(s) on an earlier branch are ~ 1.04 (CF1) and 1.11 (CF2). These values are close to the expected value of 1.00 for neutrally evolving pseudogene sequences. Transitional branches where enamel was lost based on ancestral reconstructions [<a href="#CR23">23</a>, <a href="#CR24">24</a>] include stem Pilosa, stem Pholidota, stem Mysticeti, and <i>Orycteropus</i>. DN/dS values for these branches are ~ 0.12 (CF1) to ~ 0.14 (CF2) for stem Pilosa, ~ 0.93 (CF1) to ~ 1.0 (CF2) for stem Pholidota, and ~ 0.88 (CF1) to ~ 0.93 (CF2) for <i>Orycteropus</i>. The codeml value for Mysticeti is based on only two nucleotide substitutions, both of which are nonsynonymous, leading to parameter unidentifiability. Other transitional branches for enamelless species based on the occurrence of the first inactivating mutation(s) include stem Chlamyphoridae and <i>Dasypus.</i> The dN/dS values for stem Chlamyphoridae are ~ 1.01 (CF1) and ~ 1.10 (CF2), whereas those for <i>Dasypus</i> are ~ 0.96 (CF1) and ~ 1.02 (CF2). <div class="table-wrap_UNKNOWN" id="Tab2"><span class="label" tagx="label" title="label">Table 2</span> </div></p><p>Results of selection (dN/dS=ω) analyses with codon frequency models 1 (CF1) and 2 (CF2) and 15 different branch categories</p>  <table frame="hsides" rules="groups"> <thead> <tr> <th rowspan="2">Branch category</th> <th colspan="3">CF1</th> <th colspan="3">CF2</th> </tr> <tr> <th>dN/dS</th> <th>N * dN</th> <th>S * dS</th> <th>dN/dS</th> <th>N * dN</th> <th>S * dS</th> </tr> </thead> <tbody> <tr> <td>Background</td> <td>0.4904</td> <td>2230.2</td> <td>1906.8</td> <td>0.5202</td> <td>2203.6</td> <td>2015.4</td> </tr> <tr> <td>Pseudogenic</td> <td>1.0446</td> <td>333.7</td> <td>133.9</td> <td>1.1060</td> <td>332.0</td> <td>142.6</td> </tr> <tr> <td><i>Orycteropus afer</i></td> <td>0.8843</td> <td>67.9</td> <td>32.2</td> <td>0.9340</td> <td>66.9</td> <td>34.1</td> </tr> <tr> <td>Stem Pilosa</td> <td>0.1227</td> <td>2.2</td> <td>7.6</td> <td>0.1443</td> <td>2.4</td> <td>8.1</td> </tr> <tr> <td><i>Dasypus novemcinctus</i></td> <td>0.9609</td> <td>46.3</td> <td>20.2</td> <td>1.0190</td> <td>46.5</td> <td>21.7</td> </tr> <tr> <td>Stem Chlamyphoridae</td> <td>1.0083</td> <td>9.2</td> <td>3.8</td> <td>1.1043</td> <td>9.2</td> <td>4.0</td> </tr> <tr> <td>Stem Pholidota</td> <td>0.9333</td> <td>60.7</td> <td>27.2</td> <td>0.9980</td> <td>59.8</td> <td>28.5</td> </tr> <tr> <td>Stem Mysticeti</td> <td>999.0000</td> <td>2.0</td> <td>0.0</td> <td>999.0000</td> <td>2.0</td> <td>0.0</td> </tr> <tr> <td><i>Physeter macrocephalus</i></td> <td>0.9000</td> <td>11.7</td> <td>5.4</td> <td>1.0071</td> <td>11.7</td> <td>5.5</td> </tr> <tr> <td><i>Lipotes vexillifer</i></td> <td>1.1064</td> <td>11.0</td> <td>4.2</td> <td>1.1690</td> <td>10.9</td> <td>4.4</td> </tr> <tr> <td>Stem to Delphinidae + Monodontidae + Phocoenidae</td> <td>999.0000</td> <td>7.7</td> <td>0.0</td> <td>999.0000</td> <td>7.5</td> <td>0.0</td> </tr> <tr> <td><i>Callithrix jacchus</i></td> <td>0.8450</td> <td>13.3</td> <td>6.6</td> <td>0.8965</td> <td>13.1</td> <td>7.0</td> </tr> <tr> <td><i>Neomonachus schauinslandi</i></td> <td>0.4035</td> <td>3.9</td> <td>4.0</td> <td>0.4072</td> <td>3.6</td> <td>4.2</td> </tr> <tr> <td>Stem to <i>Mammut americanum + Loxodonta cyclotis + Palaeoloxodon antiquus + Mammuthius primigenius</i></td> <td>0.7350</td> <td>0.0</td> <td>0.0</td> <td>0.0001</td> <td>0.0</td> <td>0.0</td> </tr> <tr> <td>Proboscidean branches that post-date loss of exon 6</td> <td>0.2133</td> <td>2.4</td> <td>4.6</td> <td>0.2287</td> <td>2.3</td> <td>4.8</td> </tr> </tbody> </table> <div class="table-wrap-foot_UNKNOWN"> <p>Symbols: * is multiplication symbol</p> </div>  <p /> <p id="Par29">Transitional branches that exhibit the first inactivating mutation(s) among odontocetes all have high dN/dS values including <i>Physeter</i> (~ 0.90 [CF1], ~ 1.01 [CF2]), and <i>Lipotes</i> (~ 1.11 [CF1], ~ 1.17 [CF2]). The dN/dS value for the stem branch leading to Delphinidae + Phocoenidae + Monodontidae is based on 7.5–7.7 substitutions, all of which are nonsynonymous, and is undefined. Other transitional branches with putative inactivating mutations are <i>Callithrix</i>, <i>Neomonachus,</i> and the stem branch leading to three elephantids + <i>Mammut americanum</i>. <i>Callithrix</i> has an elevated dN/dS value (~ 0.85 [CF1], ~ 0.90 [CF2]). <i>Neomonachus</i> has dN/dS values of ~ 0.40 (CF1) to ~ 0.41 (CF2) that are below the background ratios for codeml. Finally, there are no inferred substitutions on the stem branch leading to proboscideans that are missing exon 6 so the codeml values for this branch can be ignored. The dN/dS values for crown proboscidean branches that are missing exon 6 are ~ 0.21 (CF1) and ~ 0.23 (CF2) and are below the background ratios of ~ 0.49 and ~ 0.52, respectively.</p> </div> <div class="inactivationtimes" title="sec"> <div class="title" tagx="title" title="title">
Inactivation times</div> <p id="Par30">Estimates of inactivation times for <i>ODAM</i> based on dN/dS ratios and equations in Meredith et al. [<a href="#CR23">23</a>] are provided in Table <a href="#Tab3">3</a> for several transitional branches. The mean estimate for the inactivation of <i>ODAM</i> on the stem Pholidota branch is 64.15 Ma (range = 73.04–57.34 Ma) (Fig. <a href="#Fig5">5</a>). The mean inactivation estimate for <i>Orycteropus ODAM,</i> in turn, is 55.63 Ma (range 58.9–48.2 Ma). Among cingulatans, the mean inactivation time in <i>Dasypus</i> is 40.43 Ma (range 45.45–36.38 Ma) and is similar to the mean estimate of 45.29 Ma for the stem chlamyphorid branch (range from 45.45–44.73 Ma). Inactivation dates for <i>Lipotes</i> and the stem Delphinidae + Phocoenidae + Monodontidae branch were not calculated because dN/dS values for both branches always exceeded the estimates for pseudogenic branches. However, the elevated dN/dS values for these branches suggest that selection on <i>ODAM</i> was relaxed near the base of each branch or even earlier (see Discussion). The mean inactivation date for <i>Callithrix ODAM</i> is 13.12 Ma (range = 14.47–11.17 Ma). <div class="table-wrap_UNKNOWN" id="Tab3"><span class="label" tagx="label" title="label">Table 3</span> </div></p><p>Estimated inactivation times, in millions of years, for select branches where the <i>ODAM</i> gene accumulated its first inactivating mutation</p>  <table frame="hsides" rules="groups"> <thead> <tr> <th rowspan="3">Branch</th> <th colspan="4">CF1</th> <th colspan="4">CF2</th> <th rowspan="3">Mean inactivation date</th> </tr> <tr> <th colspan="2">ω<sub>p</sub> = 1.0 (fixed)</th> <th colspan="2">ω<sub>p</sub> = 1.0446 (estimated)</th> <th colspan="2">ω<sub>p</sub> = 1.0 (fixed)</th> <th colspan="2">ω<sub>p</sub> = 1.106 (estimated)</th> </tr> <tr> <th>1 syn rate</th> <th>2 syn rates</th> <th>1 syn rate</th> <th>2 syn rates</th> <th>1 syn rate</th> <th>2 syn rates</th> <th>1 syn rate</th> <th>2 syn rates</th> </tr> </thead> <tbody> <tr> <td><i>Orycteropus</i></td> <td>58.99</td> <td>53.76</td> <td>54.24</td> <td>48.26</td> <td>65.8</td> <td>62.2</td> <td>53.91</td> <td>47.89</td> <td>55.63</td> </tr> <tr> <td><i>Dasypus</i></td> <td>41.96</td> <td>40.63</td> <td>38.59</td> <td>36.24</td> <td>45.45<sup>a</sup></td> <td>45.45 <sup>a</sup></td> <td>38.7</td> <td>36.38</td> <td>40.43</td> </tr> <tr> <td>Stem Chlamyphoridae</td> <td>45.45<sup>a</sup></td> <td>45.45<sup>a</sup></td> <td>44.93</td> <td>44.73</td> <td>45.45<sup>a</sup></td> <td>45.45 <sup>a</sup></td> <td>45.43</td> <td>45.42</td> <td>45.29</td> </tr> <tr> <td><i>Physeter</i></td> <td>26.07</td> <td>24.04</td> <td>23.97</td> <td>21.56</td> <td>32.43<sup>a</sup></td> <td>32.43<sup>a</sup></td> <td>26.95</td> <td>25.14</td> <td>26.57</td> </tr> <tr> <td>Stem Pholidota</td> <td>65.39</td> <td>62.60</td> <td>61.16</td> <td>57.34</td> <td>73.04</td> <td>72.93</td> <td>62.16</td> <td>58.55</td> <td>64.15</td> </tr> <tr> <td><i>Callithrix</i></td> <td>14.03</td> <td>12.41</td> <td>12.90</td> <td>11.17</td> <td>15.81</td> <td>14.47</td> <td>12.95</td> <td>11.23</td> <td>13.12</td> </tr> </tbody> </table> <div class="table-wrap-foot_UNKNOWN"> <p><sup>a</sup>Estimated inactivation age constrained by divergence time on timetree</p> <p>Abbreviations: <i>CF1</i> codon frequency model 1, <i>CF2</i> codon frequency model 2, <i>syn</i> synonymous, ω<sub>p</sub> =dN/dS for pseudogenic branch category</p> </div>  <div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/"><span class="label" tagx="label" title="label">Fig. 5</span></a></div>   <p>Estimated inactivation times in <i>ENAM</i> versus <i>ODAM</i>. <b>a</b>) <i>Manis</i> spp. (pangolins). <b>b</b>) <i>Orycteropus afer</i> (aardvark). <b>c</b>) Pilosa (anteaters, sloths) and <i>Dasypus novemcinctus</i> (nine-banded armadillo). Inactivation dates are mean values based on eight different combinations of two different codon models in codeml (CF1, CF2), fixed (1.0) versus estimated values for the dN/dS value on fully pseudogenic branches, and equations that permit for one versus two synonymous substitution rates [<a href="#CR23">23</a>]. <i>Manis tricuspis</i> is missing from the <i>ODAM</i> data set and <i>M. javanica</i> was not included in Meredith et al.’s [<a href="#CR23">23</a>] <i>ENAM</i> data set</p>   </div> <p /> </div> </div> <div class="discussion" title="sec"> <div class="title" tagx="title" title="title">
Discussion</div> <div class="patternsofodaminactivationinplacentalmammals" title="sec"> <div class="title" tagx="title" title="title">
Patterns of ODAM inactivation in placental mammals</div> <p id="Par31">Previous studies have documented inactivating mutations in nine different tooth-related genes in toothless and enamelless mammals (<i>ACP4, AMBN, AMEL, AMTN, C4orf26, DSPP, ENAM, KLK4, MMP20</i>). <i>ODAM</i> can now be added to this list based on inactivating mutations in both edentulous (Vermilingua, Pholidota, Mysticeti) and enamelless (<i>Orycteropus</i>, Folivora, Cingulata) clades. Indeed, inactivating mutations were discovered in all 19 edentulous and enamelless species that were investigated. Given the broad phylogenetic spread of mammalian lineages with inactivating mutations, these results suggest that the only essential functions of <i>ODAM</i> that are maintained by natural selection are related to tooth development and/or maintenance of the adhesion of the junctional epithelium to the tooth surface, even though <i>ODAM</i> expression has been reported in other tissues such as salivary gland, trachea, mammary gland, and lacrimal gland [<a href="#CR14">14</a>, <a href="#CR15">15</a>]. The expression of <i>ODAM</i> in mammary and salivary glands may be explained by the location of this gene in the same gene expression neighborhood [<a href="#CR73">73</a>] as other SCPP genes that are expressed in mammary glands (<i>CSN2, CSN3</i>) and salivary glands (<i>STATH</i>).</p> <p id="Par32">Patterns of <i>ODAM</i> inactivation are generally consistent with ancestral reconstructions of tooth and enamel loss and previous studies of other tooth-related genes [<a href="#CR19">19</a>, <a href="#CR23">23</a>, <a href="#CR24">24</a>, <a href="#CR28">28</a>, <a href="#CR30">30</a>]. Our estimates for <i>ODAM</i> inactivation in Pholidota (73.04–57.34 Ma) are slightly older than Meredith et al.’s [<a href="#CR23">23</a>] estimates for <i>ENAM</i> inactivation (59.4–54.9) in this lineage. We recalculated inactivation dates for pangolin <i>ENAM</i> using the same divergence times and codon frequency models that were used for <i>ODAM</i> and the mean inactivation date is 57.7 Ma (Fig. <a href="#Fig5">5</a>). Estimates for <i>ODAM</i> and <i>ENAM</i> inactivation are both older than the oldest fossil pangolin, <i>Eomanis waldi</i>, which is ~ 47 Ma [<a href="#CR23">23</a>, <a href="#CR74">74</a>], and suggest that even older fossil pholidotans that lack teeth or at least tooth enamel may be discovered.</p> <p id="Par33">Inactivation dates for <i>ODAM</i> are also older than inactivation dates for <i>ENAM</i> in the aardvark <i>Orycteropus afer</i>. Meredith et al. [<a href="#CR23">23</a>] estimated an inactivation date of 35.3–28.8 Ma for <i>O. afer ENAM</i>, and we obtained a similar estimate (~ 30.7 Ma) when we used the same divergence dates and codon frequency models as for <i>ODAM</i> (Fig. <a href="#Fig5">5</a>).</p> <p id="Par34">In Xenarthra, ancestral reconstructions of the presence or absence of enamel suggest that enamel was lost on the stem Pilosa branch [<a href="#CR23">23</a>]. This prediction is validated by two inactivating mutations in <i>ENAM</i> that are shared by Vermilingua (anteaters) and Folivora (sloths). In <i>ODAM</i>, we discovered inactivating mutations that are shared by the two vermilinguans included in our study, as well as a mutation that is shared by both folivorans, but did not find any mutations that are shared by all pilosans. The dN/dS value for the stem Pilosa branch (~ 0.12–0.14) is below the background dN/dS ratio of ~ 0.49–0.52 and suggests that <i>ODAM</i> was maintained by natural selection up until the split between Vermilingua and Folivora. We also re-ran codeml analyses with an added category for the stem Vermilingua and stem Folivora branches, which are the immediate descendant branches of stem Pilosa. The resulting dN/dS values (~ 1.12 [CF1], ~ 1.22 [CF2]) are slightly above the dN/dS values for the pseudogenic branch category (~ 1.04 [CF1], ~ 1.11 [CF2]) and suggest that <i>ODAM</i> has evolved neutrally on these branches. Together, the dN/dS values for stem Pilosa, stem Vermilingua, and stem Folivora suggest that selection on <i>ODAM</i> was relaxed very near the most recent common ancestor of Pilosa. By contrast, dN/dS analyses for <i>ENAM</i> suggest that selection was relaxed near the base of the stem pilosan branch at ~ 64.8 Ma (Fig. <a href="#Fig5">5</a>).</p> <p id="Par35">Also in Xenarthra, our estimates of inactivation times in Cingulata (armadillos) suggest that selection on <i>ODAM</i> was relaxed very soon after the split between Dasypodidae and Chlamyphoridae at 45.5 Ma [<a href="#CR45">45</a>]. Specifically, selection on <i>ODAM</i> was relaxed ~ 40 Ma in <i>Dasypus</i> and ~ 45 Ma in stem Chlamyphoridae. Meredith et al. [<a href="#CR23">23</a>] reported inactivating mutations in <i>ENAM</i> for four cingulatan genera that they investigated, but none that were shared by all chlamyphorids. In the case of <i>ENAM</i>, we estimated an inactivation date of ~ 45.5 Ma for this gene in <i>Dasypus</i>. By contrast with pilosans, where inactivating mutations in <i>ENAM</i> preceded inactivating mutations in <i>ODAM</i>, the opposite pattern occurs in chlamyphorids with an inactivating mutation in <i>ODAM</i> preceding the first inactivating mutations in <i>ENAM</i> (or at least the region of <i>ENAM</i> that was targeted by Meredith et al. [<a href="#CR23">23</a>]).</p> <p id="Par36">Whereas <i>ENAM</i> is required for enamel formation, <i>ODAM</i> inactivation in mouse only causes abnormalities in the junctional epithelium [<a href="#CR13">13</a>]. Thus, the more ancient loss of <i>ODAM</i> compared with <i>ENAM</i> in both Pholidota and <i>Orycteropus afer</i> suggests that selection on maintenance of the junctional epithelium was relaxed before enamel loss likely happened. The inactivation of <i>ODAM</i> but not <i>ENAM</i> in toothed whales that retain enamel also supports this hypothesis. However, Pilosa and <i>Dasypus novemcinctus</i> both show the opposite pattern where inactivation dates for <i>ENAM</i> are older than for <i>ODAM</i> (Fig. <a href="#Fig5">5</a>).</p> <p id="Par37">Beyond its inactivation in toothless and enamelless forms, we also discovered inactivating mutations or exon deletions in taxa with enamel-capped teeth. Two of these (odontocetes, some proboscideans) are discussed in separate sections below. Among the remaining taxa with mutations in <i>ODAM</i>, the phocid <i>Neomonachus</i> has four separate mutations (2 bp frameshift deletion, premature stop codon, splice site mutation, exon 5 deletion) and is the best candidate for inactivation of this gene among taxa with enamel-capped teeth. However, short read data for <i>Neomonachus</i> are not available in the SRA and we were unable to validate these mutations. Another phocid, <i>Leptonychotes weddellii</i>, also has problems with the annotated gene sequence, and like <i>Neomonachus</i> is missing exon 5 (as well as exons 6–8). SRA data are available for <i>Leptonychotes</i> and we were able to reconstruct all of the missing exons for this taxon. It remains unclear if the missing exon and other problems with <i>Neomonachus ODAM</i> are real or instead are assembly or annotation errors. <i>Enhydra lutris</i> has a 1 bp frameshift in exon 1 of the annotated sequence, but three different individuals with SRA data have both intact and frameshifted versions of this exonic region, which leaves open the possibility that the mutated form of exon 1 represents a second allele, or more likely, a paralogous locus. <i>Callithrix jacchus</i> (common marmoset) has an acceptor splice site mutation in intron 4, a premature stop codon in exon 5, and an elevated dN/dS ratio (~ 0.85–0.90). Curiously, marmosets exhibit a loss of lingual enamel and a hypertrophy of buccal enamel in their mandibular incisors [<a href="#CR75">75</a>]. However, similar phenotypes are found in the aye-aye (<i>Daubentonia madagascariensis</i>) and rodents, yet they uniformly retain an intact <i>ODAM</i>.</p> </div> <div class="odaminactivationinodontocetes" title="sec"> <div class="title" tagx="title" title="title">
ODAM inactivation in odontocetes</div> <p id="Par38">It is perhaps surprising that <i>ODAM</i> contains inactivating mutations in all of the odontocetes that we investigated given that all of these taxa retain teeth with enamel. We hypothesize that <i>ODAM</i> inactivation in toothed whales may be related to the simplified outer enamel of their teeth. Living odontocetes are characterized by degenerative enamel that is thinner and less complex than the enamel of their archaeocete (stem cetacean) relatives [<a href="#CR76">76</a>–<a href="#CR78">78</a>]. In archaeocetes that have been investigated, there is an inner layer of enamel that is organized into Hunter-Schreger bands (HSB), which are decussating layers of prisms that increase the strength of enamel [<a href="#CR79">79</a>], and an outer layer of radial enamel. This pattern is common in large-bodied, terrestrial mammals and is thought to be a biomechanical adaptation for food processing and crack resistance [<a href="#CR78">78</a>]. By contrast, living odontocetes often swallow prey whole, without mastication, and have variable enamel structure with loss of the HSB in the inner layer. Among Delphinida, which includes all of the odontocetes in our study excepting the sperm whale (<i>Physeter macrocephalus</i>), the simplified enamel is typically comprised of an inner layer of radial enamel and an outer layer of aprismatic enamel [<a href="#CR77">77</a>, <a href="#CR78">78</a>], although in some cases the entire enamel layer is aprismatic [<a href="#CR76">76</a>]. The enamel in <i>Physeter</i> has been characterized as being comprised of pseudoprisms. The simplified enamel microstructure of living odontocetes is associated with the transition from heterodont to homodont dentition wherein the upper and lower teeth are used to grasp and secure prey but not for mastication [<a href="#CR77">77</a>, <a href="#CR78">78</a>]. Given that <i>ODAM</i> is expressed in the junctional epithelium, where the gingiva is in contact with the outer enamel surface of the tooth [<a href="#CR5">5</a>], perhaps <i>ODAM</i> inactivation in odontocetes is related to the simplified outer enamel surface if this surface is not well suited for ODAM-assisted adhesion. Indeed, <i>ODAM</i> is also inactivated in placental mammals that lack enamel on their teeth and instead present dentin on the outer surface (aardvarks, sloths, armadillos).</p> <p id="Par39">An alternate hypothesis is that <i>ODAM</i> inactivation in toothed whales is related to antimicrobial functions of the junctional epithelium that are altered in aquatic habitats. Specifically, the gingival junctional epithelium adheres tightly to the enamel surface of the tooth in terrestrial mammals and presents a first line of defense against invading bacteria [<a href="#CR5">5</a>]. However, the microbiomes of the oral cavity are expected to differ substantially in terrestrial versus aquatic environments, and toothed whales may require a different line of defense against microbes than is required by terrestrial mammals. The coding sequence of <i>ODAM</i> is intact in <i>Trichechus manatus</i> (West Indian manatee), which is the other fully aquatic mammal with teeth that was included in our analysis, but codeml analyses revealed an elevated dN/dS value on the <i>Trichechus</i> branch (CF1 = 1.60, CF2 = 1.89) when this branch was allowed to have its own category. Although these values are &amp;gt; 1 and suggestive of positive selection, there is no significant difference between a free ratio for <i>Trichechus</i> versus a dN/dS value of 1.0 for this branch (<i>p</i> = 0.33 with CF1, <i>p</i> = 0.18 with CF2). Thus, <i>ODAM</i> evolution on the <i>Trichechus</i> branch is also consistent with neutral evolution, as might be expected if a gene is evolving neutrally even though the first inactivating mutation has not yet been fixed. It is worth noting that manatees have a unique system of tooth replacement, hind molar progression, in which the molars march forward until they are worn down and replaced by new molars that emerge at the posterior end of the tooth row [<a href="#CR80">80</a>].</p> </div> <div class="odamdeletionandfunctionalityinproboscidea?" title="sec"> <div class="title" tagx="title" title="title">
ODAM deletion and functionality in Proboscidea?</div> <p id="Par40">Our taxon sampling included six proboscideans, of which five (<i>Loxodonta africana, L. cyclotis, Elephas maximus, Mammuthus primigenius, Palaeoloxodon antiquus</i>) belong to Elephantidae and one belongs to Mammutidae (<i>Mammut americanum</i>). Exon 6 is putatively missing from four of these taxa based on the absence of sequencing reads that map to this region (Fig. <a href="#Fig3">3</a>). The assembled genome sequence for <i>L. africana</i> is also missing exon 6, but we were able to assemble this exon and its flanking intronic regions from SRA data where the Sanger sequencing-based assembly contains a string of Ns (assembly gap). The presence of exon 6 in <i>L. africana</i> and <i>E. maximus</i>, but not in other proboscideans, is unexpected given the phylogenetic relationships of these taxa. Mammutidae and Elephantidae diverged from each other ~ 30–20 Ma [<a href="#CR70">70</a>, <a href="#CR71">71</a>], whereas the five elephantids share a much more recent common ancestor ~ 10–5 Ma [<a href="#CR71">71</a>, <a href="#CR81">81</a>]. Figure <a href="#Fig6">6</a> shows a time tree for the six proboscideans in our study along with presence/absence data for exon 6 of <i>ODAM</i>. Three possible explanations for the presence/absence of exon 6 in these taxa include incomplete lineage sorting (ILS), hybridization, and/or independent deletions of exon 6. Figure <a href="#Fig6">6</a>A shows how ILS can account for the presence/absence of exon 6 in proboscideans, whereas Fig. <a href="#Fig6">6</a>B illustrates how a combination of convergent loss and hybridization can explain the same pattern. Hybridization may play a role in the distribution of <i>ODAM</i> exon 6 in Elephantidae given recent evidence for extensive introgression among elephantids [<a href="#CR71">71</a>], but there is no evidence for hybridization between crown elephantids and <i>Mammut americanum</i> and this explanation seems unlikely. Instead, ILS or the independent deletion of exon 6 in <i>Mammut americanum</i> versus elephantids that lack this exon seems more probable. Is also remains to be determined if individual elephantid species will reveal presence/absence variation for exon 6 when more individuals are sampled, which would not be surprising in view of extensive hybridization among elephantids. </p><div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/"><span class="label" tagx="label" title="label">Fig. 6</span></a></div>   <p>Different hypotheses for the loss of <i>ODAM</i> exon 6 in some but not all proboscideans. <b>a</b>) Loss of <i>ODAM</i> exon 6 based on an ancestral polymorphism in the ancestor of Elephantidae and Mammutidae followed by incomplete lineage sorting of the two allelic variants in Recent and extinct proboscideans. <b>b</b>) One possible scenario for the loss of exon 6 of <i>ODAM</i> based on convergent loss in <i>Mammut americanum</i> and <i>Palaeoloxodon antiquus</i> followed by introgression of the sans exon 6 allele from <i>P. antiquus</i> to <i>Loxodonta cyclotis</i> and <i>Mammuthus primigenius</i>. Other scenarios are also possible based on directions of introgression within Elephantidae that were identified by Palkopoulou et al. [<a href="#CR71">71</a>]</p>   </div> <p /> <p id="Par41">It is also worth noting that the deletion of exon 6 seemingly has an ancient history that may trace as far back as the common ancestor of Elephantidae and Mammutidae if ILS is responsible for the presence/absence of this exon in different proboscideans. Exon 6 is 111 bp in elephantids that have this exon, so complete deletion of this exon will not interrupt the reading frame of the downstream exons (7–10). This leaves open the possibility of an intact version of <i>ODAM</i> based on nine of ten exons. The possibility of an altered yet functional version of <i>ODAM</i> is also supported by the absence of any frameshift mutations, premature stop codons, or splice site mutations in the remaining nine exons (1–5, 7–10) of the <i>ODAM</i> gene of proboscideans that are missing exon 6. Finally, all of the proboscidean <i>ODAM</i> sequences that lack exon 6 cluster together on the <i>ODAM</i> gene tree and the dN/dS value for this clade is only ~ 0.21–0.23, which is below the background value (~ 0.49–0.52) for branches leading to taxa with taxa with enamel-capped teeth and intact <i>ODAM</i> sequences for all ten exons. Thus, beyond the absence of exon 6, there is no evidence for relaxed selection in the <i>ODAM</i> gene of proboscideans that lack this exon. Unfortunately a 3D structure for ODAM is not available and it remains unclear if exon 6 encodes a region of the protein that is on the surface or embedded in the interior of this molecule.</p> </div> <div class="coevolversuscodemlselectionanalyses" title="sec"> <div class="title" tagx="title" title="title">
Coevol versus codeml selection analyses</div> <p id="Par42">DN/dS analyses with Coevol and codeml were performed with different taxon sets and for different purposes. Coevol was used to implement a joint analysis that included all 165 taxa. This analysis was performed to visualize patterns of dN/dS variation across the placental tree and to examine the effects of life history traits on dN/dS values. Importantly, Coevol employs a form of rate smoothing through the incorporation of a Brownian motion model of continuous trait evolution. By contrast, codeml analyses were performed with 96 taxa and 15 different dN/dS categories that were determined a priori based on ancestral reconstructions of tooth/enamel loss [<a href="#CR23">23</a>, <a href="#CR24">24</a>] and patterns of <i>ODAM</i> inactivation. Coevol and codeml results are generally in agreement with each other with lower dN/dS values for functional branches and higher values for transitional and fully pseudogenic branches. At the same time, dN/dS values for both transitional and pseudogenic branches are generally higher for codeml than for Coevol. For example, Coevol dN/dS values are &amp;lt; 1 for pseudogenic branches in clades that post-date the loss of teeth or enamel. By contrast, dN/dS values for the fully pseudogenic branch category are &amp;gt; 1 in codeml analyses (Table <a href="#Tab2">2</a>). Similarly, the dN/dS value for the transitional branch leading to <i>Callithrix</i> is higher for codeml (0.85–0.90) than for Coevol (0.54). We attribute these differences to the autocorrelative or smoothing effect of the Brownian motion model implemented by Coevol [<a href="#CR41">41</a>, <a href="#CR50">50</a>, <a href="#CR82">82</a>].</p> </div> <div class="isodaminactivationneutraloradaptive?" title="sec"> <div class="title" tagx="title" title="title">
Is ODAM inactivation neutral or adaptive?</div> <p id="Par43">The degeneration of morphological structures such as limbs, teeth, and eyes is a complex process that may result from relaxed selection (neutral evolution), adaptive evolution (direct natural selection to conserve energy and/or eliminate the disadvantageous effects of a morphological structure), and/or pleiotropy (indirect selection on another trait) [<a href="#CR83">83</a>]. Adaptive and non-adaptive causes have also been proposed for the inactivation of different genes [<a href="#CR28">28</a>]. In the case of individual lineages of toothless and enamelless placental mammals, it remains unclear if <i>ODAM</i> inactivation resulted from relaxed selection and/or adaptive evolution. For example, was the evolution of edentulism in pangolins driven by the accumulation of random mutations after relaxed selection on tooth development or did natural selection directly favor tooth reduction? If the latter, was this driven by developmental signaling changes that arrested tooth development [<a href="#CR84">84</a>] or were genes that encode structural proteins of the enamel matrix the initial targets of adaptive evolution? This topic is ripe for future studies, but boutique organisms such as pangolins, anteaters, and baleen whales are much less tractable than model organisms such as mice for evo-devo studies of tissue-specific gene expression.</p> </div> </div> <div class="conclusions" title="sec"> <div class="title" tagx="title" title="title">
Conclusions</div> <p id="Par44">Nature’s laboratory has provided us with multiple independent lineages of placental mammals that are either toothless or lack enamel caps on their teeth. Molecular evolutionary analysis of candidate tooth-specific genes in these independent lineages has emerged as a powerful approach to test hypotheses that various tooth-related genes are pleiotropic versus tooth-specific with respect to their essential functions that are maintained by natural selection. Previous studies have demonstrated that nine different tooth-related genes are inactivated in one or more clades of mammals that are either edentulous or lack enamel caps on their teeth. Here, we show that <i>ODAM</i> is also inactivated in all lineages of toothless and enamelless placental mammals that were investigated. These results support the hypothesis that the only essential functions of <i>ODAM</i> that are maintained by natural selection are related to tooth development and the maintenance of a healthy junctional epithelium where the gingivae are in contact with the tooth enamel. Specifically, <i>ODAM</i> has been inactivated in all representatives of three toothless clades (Vermilingua, Mysticeti, Pholidota) and three enamelless clades (Folivora, Cingulata, <i>Orycteropus</i>) that were included in our study. The overlap in naturally occurring gene inactivations and human genetic diseases [<a href="#CR85">85</a>] suggests that <i>ODAM</i> may be linked to some dental or gingival deformities. DN/dS analyses further demonstrate that the <i>ODAM</i> gene has evolved neutrally in clades that post-date the occurrence of an inactivating mutation on an earlier branch of the placental tree. DN/dS values for transitional branches, which record the first inactivating mutation in each lineage, also provide the basis for estimating <i>ODAM</i> inactivation times. In the main, these estimates are similar to estimates for the inactivation of <i>ENAM</i> in these lineages. Estimates of <i>ENAM</i> and <i>ODAM</i> inactivation in Pholidota both suggest that we may discover stem pholidotans that are older than <i>Eomanis</i> (~ 47 Ma) and lack teeth or at least the enamel on their teeth.</p> <p id="Par45">Beyond inactivating mutations in toothless and enamelless mammals, we also discovered mutations in several other groups including odontocetes, four proboscideans, and <i>Callithrix jacchus</i>. In the case of odontocetes, the inactivation of <i>ODAM</i> may be related to a fully aquatic lifestyle where the antimicrobial functions of the junctional epithelium are reduced. An alternate explanation is that the very thin enamel of odontocetes, which is often prismless, may affect the ability of ODAM to contribute to the adhesive properties of the gingival junctional epithelium. The apparent deletion (no BLAST results) of exon 6 of <i>ODAM</i> in three elephantids plus <i>Mammut americanum</i>, but not in two other elephantids, is unexpected and requires some combination of ILS, hybridization, and/or independent deletions of this exon in different proboscideans. However, proboscideans that lack this exon show no other evidence of inactivating mutations and the remaining nine exons have evolved under purifying selection. These results suggest that a version of <i>ODAM</i> that is encoded by nine exons remains functional in these proboscideans.</p> </div> <div class="supplementary-material" title="supplementary-material"> <div class="title" tagx="title" title="title">
Additional files</div> <div class="sec" title="sec"> <p> </p><div class="local-data" title="local-data"> <div class="media" title="media"><a href="12862_2019_1359_MOESM1_ESM.docx">LINK</a><span class="label" tagx="label" title="label">Additional file 1:</span> <p><b>Tables S1.</b> Source of ODAM sequences for 165 placental mammals. (DOCX 40 kb)</p>  </div> </div> <div class="local-data" title="local-data"> <div class="media" title="media"><a href="12862_2019_1359_MOESM2_ESM.xlsx">LINK</a><span class="label" tagx="label" title="label">Additional file 2:</span> <p><b>Tables S2.</b> Genes identified in a screen for genes that are preferentially lost in enamelless or edentulous mammals. (XLSX 13.5 kb)</p>  </div> </div> <p /> </div> </div> </div> <div class="back" title="back"> <div class="glossary" title="glossary"> <div class="title" tagx="title" title="title">
Abbreviations</div> <div class="def-list" title="def-list"> <div class="def-item" title="def-item"> <div class="term" title="term">
bp</div> <div class="def" title="def"> <p id="Par4">Base pairs</p> </div> </div> <div class="def-item" title="def-item"> <div class="term" title="term">
ENA</div> <div class="def" title="def"> <p id="Par5">European nucleotide archive</p> </div> </div> <div class="def-item" title="def-item"> <div class="term" title="term">
ILS</div> <div class="def" title="def"> <p id="Par6">Incomplete lineage sorting</p> </div> </div> <div class="def-item" title="def-item"> <div class="term" title="term">
ODAM</div> <div class="def" title="def"> <p id="Par7">Odontogenic ameloblast-associated</p> </div> </div> <div class="def-item" title="def-item"> <div class="term" title="term">
SCPP</div> <div class="def" title="def"> <p id="Par8">Secretory calcium-binding phosphoprotein</p> </div> </div> <div class="def-item" title="def-item"> <div class="term" title="term">
SRA</div> <div class="def" title="def"> <p id="Par9">Sequence read archive</p> </div> </div> </div> </div> <div class="ack" title="ack"> <p>We thank B. de Thoisy, M. Superina, M. Nachman (Museum of Vertebrate Zoology, University of California, Berkeley, USA), G. Haszprunar, and M. Hiermeier (The Bavarian State Collection of Zoology, Munich, Germany) for providing xenarthran tissue samples and G. Amato (formerly at NYZS) for providing pygmy hippopotamus DNA. We also thank J. Johnson (Broad Institute, Cambridge, USA) for granting access to unpublished xenarthran Discovar assemblies and C. Buell for providing paintings of mammals.</p> <div class="funding" title="sec"> <div class="title" tagx="title" title="title">
Funding</div> <p id="Par46">This work was supported by NSF grant DEB-1457735 to J.G. and M.S.S, a grant from the Leibniz Association (SAW-2016-SGN-2) to M.H., a PRESTIGE Marie-Curie post-doctoral fellowship to C.A.E., and "Investissement d’Avenir" of the "Agence Nationale de la Recherche" (CEBA: ANR-10-LABX-25-01) and European Research Council (ConvergeAnt: ERC-2015-CoG-683257) grants to F.D. This is contribution ISEM 2019-005 of the Institut des Sciences de l’Evolution de Montpellier. These agencies provided funding for generating new data and provided support for personnel.</p> </div> <div class="data-availability" title="data-availability"> <div class="title" tagx="title" title="title">
Availability of data and materials</div> <p id="Par47">The dataset supporting the conclusions of this article is available TreeBASE (<a href="http://purl.org/phylo/treebase/phylows/study/TB2:S23531">http://purl.org/phylo/treebase/phylows/study/TB2:S23531</a>).</p> </div> </div> <div class="title" tagx="title" title="title">
Authors’ contributions</div> <p>Study design (MSS, CAE, JG, MH, FD), data collection (MSS, CAE, JG, JR, MAC, NH, MH, FD), manuscript preparation and editing (MSS, CAE, JG, JR, MAC, NH, MH, FD). All authors read and approved the final manuscript.</p> <div class="ethicsapprovalandconsenttoparticipate" title="sec"> <div class="title" tagx="title" title="title">
Ethics approval and consent to participate</div> <p>Not applicable.</p> </div> <div class="consentforpublication" title="sec"> <div class="title" tagx="title" title="title">
Consent for publication</div> <p>Not applicable.</p> </div> <div class="competinginterests" title="sec"> <div class="title" tagx="title" title="title">
Competing interests</div> <p>The authors declare that they have no competing interests.</p> </div> <div class="publisher’snote" title="sec"> <div class="title" tagx="title" title="title">
Publisher’s Note</div> <p>Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations.</p> </div> <div class="references">
References</div> <div tag="ref-list"> <ul> <div class="title" tagx="title" title="title">
References</div> <li tag="ref"><a name="CR1" /><span class="label" tagx="label" title="label">1.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kawasaki</span><span class="given-names" tagx="given-names" title="given-names">K</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Buchanan</span><span class="given-names" tagx="given-names" title="given-names">AV</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Weiss</span><span class="given-names" tagx="given-names" title="given-names">KM</span></span></span><span class="mixed-article-title" title="mixed-article-title">Biomineralization in humans: making the hard choices in life</span><span class="source" tagx="source" title="source">Annu Rev Genet</span><span class="year" tagx="year" title="year">2009</span><span class="volume" tagx="volume" title="volume">43</span><span class="fpage" tagx="fpage" title="fpage">119</span><span class="lpage" tagx="lpage" title="lpage">142</span><span class="pub-id"><a href="https://dx.doi.org/10.1146/annurev-genet-102108-134242">10.1146/annurev-genet-102108-134242</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19659443">19659443</a></span></span></li> <li tag="ref"><a name="CR2" /><span class="label" tagx="label" title="label">2.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kawasaki</span><span class="given-names" tagx="given-names" title="given-names">K</span></span></span><span class="mixed-article-title" title="mixed-article-title">The SCPP gene family and the complexity of hard tissues in vertebrates</span><span class="source" tagx="source" title="source">Cells Tissues Organs</span><span class="year" tagx="year" title="year">2011</span><span class="volume" tagx="volume" title="volume">194</span><span class="fpage" tagx="fpage" title="fpage">108</span><span class="lpage" tagx="lpage" title="lpage">112</span><span class="pub-id"><a href="https://dx.doi.org/10.1159/000324225">10.1159/000324225</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21576905">21576905</a></span></span></li> <li tag="ref"><a name="CR3" /><span class="label" tagx="label" title="label">3.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kawasaki</span><span class="given-names" tagx="given-names" title="given-names">K</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Weiss</span><span class="given-names" tagx="given-names" title="given-names">KM</span></span></span><span class="mixed-article-title" title="mixed-article-title">Mineralized tissue and vertebrate evolution: the secretory</span><span class="source" tagx="source" title="source">Proc Natl Acad Sci USA</span><span class="year" tagx="year" title="year">2003</span><span class="volume" tagx="volume" title="volume">100</span><span class="fpage" tagx="fpage" title="fpage">4060</span><span class="lpage" tagx="lpage" title="lpage">4065</span><span class="pub-id"><a href="https://dx.doi.org/10.1073/pnas.0638023100">10.1073/pnas.0638023100</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12646701">12646701</a></span></span></li> <li tag="ref"><a name="CR4" /><span class="label" tagx="label" title="label">4.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Sire</span><span class="given-names" tagx="given-names" title="given-names">J-Y</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Davit-Béal</span><span class="given-names" tagx="given-names" title="given-names">T</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Delgado</span><span class="given-names" tagx="given-names" title="given-names">S</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gu</span><span class="given-names" tagx="given-names" title="given-names">X</span></span></span><span class="mixed-article-title" title="mixed-article-title">The origin and evolution of enamel mineralization genes</span><span class="source" tagx="source" title="source">Cells Tissues Organs</span><span class="year" tagx="year" title="year">2007</span><span class="volume" tagx="volume" title="volume">186</span><span class="fpage" tagx="fpage" title="fpage">25</span><span class="lpage" tagx="lpage" title="lpage">48</span><span class="pub-id"><a href="https://dx.doi.org/10.1159/000102679">10.1159/000102679</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17627117">17627117</a></span></span></li> <li tag="ref"><a name="CR5" /><span class="label" tagx="label" title="label">5.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bosshardt</span><span class="given-names" tagx="given-names" title="given-names">DD</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lang</span><span class="given-names" tagx="given-names" title="given-names">NP</span></span></span><span class="mixed-article-title" title="mixed-article-title">The junctional epithelium: from health to disease</span><span class="source" tagx="source" title="source">J Dent Res</span><span class="year" tagx="year" title="year">2005</span><span class="volume" tagx="volume" title="volume">84</span><span class="fpage" tagx="fpage" title="fpage">9</span><span class="lpage" tagx="lpage" title="lpage">20</span><span class="pub-id"><a href="https://dx.doi.org/10.1177/154405910508400102">10.1177/154405910508400102</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15615869">15615869</a></span></span></li> <li tag="ref"><a name="CR6" /><span class="label" tagx="label" title="label">6.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Moffatt</span><span class="given-names" tagx="given-names" title="given-names">P</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Smith</span><span class="given-names" tagx="given-names" title="given-names">CE</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">St-Arnaud</span><span class="given-names" tagx="given-names" title="given-names">R</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Nanci</span><span class="given-names" tagx="given-names" title="given-names">A</span></span></span><span class="mixed-article-title" title="mixed-article-title">Characterization of Apin, a secreted protein highly expressed in tooth-associated epithelia</span><span class="source" tagx="source" title="source">J Cell Biochem</span><span class="year" tagx="year" title="year">2008</span><span class="volume" tagx="volume" title="volume">103</span><span class="fpage" tagx="fpage" title="fpage">941</span><span class="lpage" tagx="lpage" title="lpage">956</span><span class="pub-id"><a href="https://dx.doi.org/10.1002/jcb.21465">10.1002/jcb.21465</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17647262">17647262</a></span></span></li> <li tag="ref"><a name="CR7" /><span class="label" tagx="label" title="label">7.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lee</span><span class="given-names" tagx="given-names" title="given-names">H-K</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lee</span><span class="given-names" tagx="given-names" title="given-names">D-S</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ryoo</span><span class="given-names" tagx="given-names" title="given-names">H-M</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Park</span><span class="given-names" tagx="given-names" title="given-names">J-T</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Park</span><span class="given-names" tagx="given-names" title="given-names">S-J</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bae</span><span class="given-names" tagx="given-names" title="given-names">H-S</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Cho</span><span class="given-names" tagx="given-names" title="given-names">M-I</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Park</span><span class="given-names" tagx="given-names" title="given-names">J-C</span></span></span><span class="mixed-article-title" title="mixed-article-title">The odontogenic ameloblast-associated protein (ODAM) cooperates with RUNX2 and modulates enamel mineralization via regulation of MMP-20</span><span class="source" tagx="source" title="source">J Cell Biochem</span><span class="year" tagx="year" title="year">2010</span><span class="volume" tagx="volume" title="volume">111</span><span class="fpage" tagx="fpage" title="fpage">755</span><span class="lpage" tagx="lpage" title="lpage">767</span><span class="pub-id"><a href="https://dx.doi.org/10.1002/jcb.22766">10.1002/jcb.22766</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20665536">20665536</a></span></span></li> <li tag="ref"><a name="CR8" /><span class="label" tagx="label" title="label">8.</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation">Lee H-K, Ji S, Park S-J, Choung H-W, Choi Y, Lee H-J, Park S-Y, Park J-C. Odontogenic ameloblast-associated protein (ODAM) mediates junctional epithelium attachment to teeth via integrin-ODAM-rho guanine nucleotide exchange factor 5 (ARHGEF5)-RhoA signaling. J Biol Chem. 2015;290:14740–53.</span></li> <li tag="ref"><a name="CR9" /><span class="label" tagx="label" title="label">9.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Nishio</span><span class="given-names" tagx="given-names" title="given-names">C</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wazen</span><span class="given-names" tagx="given-names" title="given-names">R</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kuroda</span><span class="given-names" tagx="given-names" title="given-names">S</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Moffatt</span><span class="given-names" tagx="given-names" title="given-names">P</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Nanci</span><span class="given-names" tagx="given-names" title="given-names">A</span></span></span><span class="mixed-article-title" title="mixed-article-title">Expression pattern of odontogenic ameloblast-associated and amelotin during formation and regeneration of the junctional epithelium</span><span class="source" tagx="source" title="source">Eur Cells Mater.</span><span class="year" tagx="year" title="year">2010</span><span class="volume" tagx="volume" title="volume">20</span><span class="fpage" tagx="fpage" title="fpage">393</span><span class="lpage" tagx="lpage" title="lpage">402</span><span class="pub-id"><a href="https://dx.doi.org/10.22203/eCM.v020a32">10.22203/eCM.v020a32</a></span></span></li> <li tag="ref"><a name="CR10" /><span class="label" tagx="label" title="label">10.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Fouillen</span><span class="given-names" tagx="given-names" title="given-names">A</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Dos Santos Neves</span><span class="given-names" tagx="given-names" title="given-names">J</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Charline</span><span class="given-names" tagx="given-names" title="given-names">M</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Castonguay</span><span class="given-names" tagx="given-names" title="given-names">J-D</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Moffatt</span><span class="given-names" tagx="given-names" title="given-names">P</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Baron</span><span class="given-names" tagx="given-names" title="given-names">C</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Nanci</span><span class="given-names" tagx="given-names" title="given-names">A</span></span></span><span class="mixed-article-title" title="mixed-article-title">Inactivation of AMTN, ODAM and SCPPPQ1 proteins of a specialized basal lamina that attaches epithelial cells to tooth mineral</span><span class="source" tagx="source" title="source">Sci Rep</span><span class="year" tagx="year" title="year">2017</span><span class="volume" tagx="volume" title="volume">7</span><span class="fpage" tagx="fpage" title="fpage">46683</span><span class="pub-id"><a href="https://dx.doi.org/10.1038/srep46683">10.1038/srep46683</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/28436474">28436474</a></span></span></li> <li tag="ref"><a name="CR11" /><span class="label" tagx="label" title="label">11.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Park</span><span class="given-names" tagx="given-names" title="given-names">J-C</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Park</span><span class="given-names" tagx="given-names" title="given-names">J-T</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Son</span><span class="given-names" tagx="given-names" title="given-names">H-H</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kim</span><span class="given-names" tagx="given-names" title="given-names">H-J</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Jeong</span><span class="given-names" tagx="given-names" title="given-names">M-J</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lee</span><span class="given-names" tagx="given-names" title="given-names">C-S</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Dey</span><span class="given-names" tagx="given-names" title="given-names">R</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Cho</span><span class="given-names" tagx="given-names" title="given-names">M-I</span></span></span><span class="mixed-article-title" title="mixed-article-title">The amyloid protein APin is highly expressed during enamel mineralization and maturation in rat incisors</span><span class="source" tagx="source" title="source">Eur J Oral Sci</span><span class="year" tagx="year" title="year">2007</span><span class="volume" tagx="volume" title="volume">115</span><span class="fpage" tagx="fpage" title="fpage">153</span><span class="lpage" tagx="lpage" title="lpage">160</span><span class="pub-id"><a href="https://dx.doi.org/10.1111/j.1600-0722.2007.00435.x">10.1111/j.1600-0722.2007.00435.x</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17451507">17451507</a></span></span></li> <li tag="ref"><a name="CR12" /><span class="label" tagx="label" title="label">12.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ikeda</span><span class="given-names" tagx="given-names" title="given-names">Y</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Neshatian</span><span class="given-names" tagx="given-names" title="given-names">M</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Holcroft</span><span class="given-names" tagx="given-names" title="given-names">J</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ganss</span><span class="given-names" tagx="given-names" title="given-names">B</span></span></span><span class="mixed-article-title" title="mixed-article-title">The enamel protein ODAM promotes mineralization in a collagen matrix</span><span class="source" tagx="source" title="source">Connect Tissue Res</span><span class="year" tagx="year" title="year">2018</span><span class="volume" tagx="volume" title="volume">59</span><span class="issue" tagx="issue" title="issue">sup1</span><span class="fpage" tagx="fpage" title="fpage">62</span><span class="lpage" tagx="lpage" title="lpage">66</span><span class="pub-id"><a href="https://dx.doi.org/10.1080/03008207.2017.1408603">10.1080/03008207.2017.1408603</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/29745811">29745811</a></span></span></li> <li tag="ref"><a name="CR13" /><span class="label" tagx="label" title="label">13.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wazen</span><span class="given-names" tagx="given-names" title="given-names">RM</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Moffatt</span><span class="given-names" tagx="given-names" title="given-names">P</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ponce</span><span class="given-names" tagx="given-names" title="given-names">KJ</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kuroda</span><span class="given-names" tagx="given-names" title="given-names">S</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Nishio</span><span class="given-names" tagx="given-names" title="given-names">C</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Nanci</span><span class="given-names" tagx="given-names" title="given-names">A</span></span></span><span class="mixed-article-title" title="mixed-article-title">Inactivation of the odontogenic ameloblast-associated gene affects the integrity of the junctional epithelium and gingival healing</span><span class="source" tagx="source" title="source">Eur Cells Mater</span><span class="year" tagx="year" title="year">2015</span><span class="volume" tagx="volume" title="volume">30</span><span class="fpage" tagx="fpage" title="fpage">187</span><span class="lpage" tagx="lpage" title="lpage">199</span><span class="pub-id"><a href="https://dx.doi.org/10.22203/eCM.v030a13">10.22203/eCM.v030a13</a></span></span></li> <li tag="ref"><a name="CR14" /><span class="label" tagx="label" title="label">14.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kestler</span><span class="given-names" tagx="given-names" title="given-names">DP</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Foster</span><span class="given-names" tagx="given-names" title="given-names">JS</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Macy</span><span class="given-names" tagx="given-names" title="given-names">SD</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Murphy</span><span class="given-names" tagx="given-names" title="given-names">CL</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Weiss</span><span class="given-names" tagx="given-names" title="given-names">DT</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Solomon</span><span class="given-names" tagx="given-names" title="given-names">A</span></span></span><span class="mixed-article-title" title="mixed-article-title">Expression of odontogenic ameloblast-associated protein (ODAM) in dental and other epithelial neoplasms</span><span class="source" tagx="source" title="source">Mol Med</span><span class="year" tagx="year" title="year">2008</span><span class="volume" tagx="volume" title="volume">14</span><span class="fpage" tagx="fpage" title="fpage">318</span><span class="lpage" tagx="lpage" title="lpage">326</span><span class="pub-id"><a href="https://dx.doi.org/10.2119/2008-00010.Kestler">10.2119/2008-00010.Kestler</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18472969">18472969</a></span></span></li> <li tag="ref"><a name="CR15" /><span class="label" tagx="label" title="label">15.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lee</span><span class="given-names" tagx="given-names" title="given-names">HK</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Park</span><span class="given-names" tagx="given-names" title="given-names">SJ</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Oh</span><span class="given-names" tagx="given-names" title="given-names">HJ</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kim</span><span class="given-names" tagx="given-names" title="given-names">JW</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bae</span><span class="given-names" tagx="given-names" title="given-names">HS</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Park</span><span class="given-names" tagx="given-names" title="given-names">JC</span></span></span><span class="mixed-article-title" title="mixed-article-title">Expression pattern, subcellular localization, and functional implications of ODAM in ameloblasts, odontoblasts, osteoblasts, and various cancer cells</span><span class="source" tagx="source" title="source">Gene Expr Patterns</span><span class="year" tagx="year" title="year">2012</span><span class="volume" tagx="volume" title="volume">12</span><span class="fpage" tagx="fpage" title="fpage">102</span><span class="lpage" tagx="lpage" title="lpage">108</span><span class="pub-id"><a href="https://dx.doi.org/10.1016/j.gep.2012.02.002">10.1016/j.gep.2012.02.002</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22387195">22387195</a></span></span></li> <li tag="ref"><a name="CR16" /><span class="label" tagx="label" title="label">16.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Sire</span><span class="given-names" tagx="given-names" title="given-names">J-Y</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Delgado</span><span class="given-names" tagx="given-names" title="given-names">S</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Girondot</span><span class="given-names" tagx="given-names" title="given-names">M</span></span></span><span class="mixed-article-title" title="mixed-article-title">Hen’s teeth with enamel cap: from dream to impossibility</span><span class="source" tagx="source" title="source">BMC Evol Biol</span><span class="year" tagx="year" title="year">2008</span><span class="volume" tagx="volume" title="volume">8</span><span class="fpage" tagx="fpage" title="fpage">e246</span><span class="pub-id"><a href="https://dx.doi.org/10.1186/1471-2148-8-246">10.1186/1471-2148-8-246</a></span></span></li> <li tag="ref"><a name="CR17" /><span class="label" tagx="label" title="label">17.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Davit-Béal</span><span class="given-names" tagx="given-names" title="given-names">T</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Tucker</span><span class="given-names" tagx="given-names" title="given-names">AS</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Sire</span><span class="given-names" tagx="given-names" title="given-names">J-Y</span></span></span><span class="mixed-article-title" title="mixed-article-title">Loss of teeth and enamel in tetrapods: fossil record, genetic data and morphological adaptations</span><span class="source" tagx="source" title="source">J Anat</span><span class="year" tagx="year" title="year">2009</span><span class="volume" tagx="volume" title="volume">214</span><span class="fpage" tagx="fpage" title="fpage">477</span><span class="lpage" tagx="lpage" title="lpage">501</span><span class="pub-id"><a href="https://dx.doi.org/10.1111/j.1469-7580.2009.01060.x">10.1111/j.1469-7580.2009.01060.x</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19422426">19422426</a></span></span></li> <li tag="ref"><a name="CR18" /><span class="label" tagx="label" title="label">18.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Al-Hashimi</span><span class="given-names" tagx="given-names" title="given-names">N</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lafont</span><span class="given-names" tagx="given-names" title="given-names">A-G</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Delgado</span><span class="given-names" tagx="given-names" title="given-names">S</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kawasaki</span><span class="given-names" tagx="given-names" title="given-names">K</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Sire</span><span class="given-names" tagx="given-names" title="given-names">J-Y</span></span></span><span class="mixed-article-title" title="mixed-article-title">The enamelin genes in lizard, crocodile, and frog and the pseudogene in the chicken provide new insights on enamelin evolution in tetrapods</span><span class="source" tagx="source" title="source">Mol Biol Evol</span><span class="year" tagx="year" title="year">2010</span><span class="volume" tagx="volume" title="volume">27</span><span class="fpage" tagx="fpage" title="fpage">2078</span><span class="lpage" tagx="lpage" title="lpage">2094</span><span class="pub-id"><a href="https://dx.doi.org/10.1093/molbev/msq098">10.1093/molbev/msq098</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20403965">20403965</a></span></span></li> <li tag="ref"><a name="CR19" /><span class="label" tagx="label" title="label">19.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Meredith</span><span class="given-names" tagx="given-names" title="given-names">RW</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zhang</span><span class="given-names" tagx="given-names" title="given-names">G</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gilbert</span><span class="given-names" tagx="given-names" title="given-names">MTP</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Jarvis</span><span class="given-names" tagx="given-names" title="given-names">ED</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Springer</span><span class="given-names" tagx="given-names" title="given-names">MS</span></span></span><span class="mixed-article-title" title="mixed-article-title">Evidence for a single loss of mineralized teeth in the common avian ancestor</span><span class="source" tagx="source" title="source">Science</span><span class="year" tagx="year" title="year">2014</span><span class="volume" tagx="volume" title="volume">346</span><span class="fpage" tagx="fpage" title="fpage">1254390</span><span class="pub-id"><a href="https://dx.doi.org/10.1126/science.1254390">10.1126/science.1254390</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25504730">25504730</a></span></span></li> <li tag="ref"><a name="CR20" /><span class="label" tagx="label" title="label">20.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Meredith</span><span class="given-names" tagx="given-names" title="given-names">RW</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gatesy</span><span class="given-names" tagx="given-names" title="given-names">J</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Springer</span><span class="given-names" tagx="given-names" title="given-names">MS</span></span></span><span class="mixed-article-title" title="mixed-article-title">Molecular decay of enamel matrix protein genes in turtles and other edentulous amniotes</span><span class="source" tagx="source" title="source">BMC Evol Biol</span><span class="year" tagx="year" title="year">2013</span><span class="volume" tagx="volume" title="volume">13</span><span class="fpage" tagx="fpage" title="fpage">20</span><span class="pub-id"><a href="https://dx.doi.org/10.1186/1471-2148-13-20">10.1186/1471-2148-13-20</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23342979">23342979</a></span></span></li> <li tag="ref"><a name="CR21" /><span class="label" tagx="label" title="label">21.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Deméré</span><span class="given-names" tagx="given-names" title="given-names">TA</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">McGowen</span><span class="given-names" tagx="given-names" title="given-names">MR</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Berta</span><span class="given-names" tagx="given-names" title="given-names">A</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gatesy</span><span class="given-names" tagx="given-names" title="given-names">J</span></span></span><span class="mixed-article-title" title="mixed-article-title">Morphological and molecular evidence for a stepwise evolutionary transition from teeth to baleen in mysticete whales</span><span class="source" tagx="source" title="source">Syst Biol</span><span class="year" tagx="year" title="year">2008</span><span class="volume" tagx="volume" title="volume">57</span><span class="fpage" tagx="fpage" title="fpage">15</span><span class="lpage" tagx="lpage" title="lpage">37</span><span class="pub-id"><a href="https://dx.doi.org/10.1080/10635150701884632">10.1080/10635150701884632</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18266181">18266181</a></span></span></li> <li tag="ref"><a name="CR22" /><span class="label" tagx="label" title="label">22.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">McKnight</span><span class="given-names" tagx="given-names" title="given-names">DA</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Fisher</span><span class="given-names" tagx="given-names" title="given-names">LW</span></span></span><span class="mixed-article-title" title="mixed-article-title">Molecular evolution of dentin phosphoprotein among toothed and toothless animals</span><span class="source" tagx="source" title="source">BMC Evol Biol</span><span class="year" tagx="year" title="year">2009</span><span class="volume" tagx="volume" title="volume">9</span><span class="fpage" tagx="fpage" title="fpage">299</span><span class="pub-id"><a href="https://dx.doi.org/10.1186/1471-2148-9-299">10.1186/1471-2148-9-299</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20030824">20030824</a></span></span></li> <li tag="ref"><a name="CR23" /><span class="label" tagx="label" title="label">23.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Meredith</span><span class="given-names" tagx="given-names" title="given-names">RW</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gatesy</span><span class="given-names" tagx="given-names" title="given-names">J</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Murphy</span><span class="given-names" tagx="given-names" title="given-names">WJ</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ryder</span><span class="given-names" tagx="given-names" title="given-names">OA</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Springer</span><span class="given-names" tagx="given-names" title="given-names">MS</span></span></span><span class="mixed-article-title" title="mixed-article-title">Molecular decay of the tooth gene enamelin (<i>ENAM</i>) mirrors the loss of enamel in the fossil record of placental mammals</span><span class="source" tagx="source" title="source">PLoS Genet</span><span class="year" tagx="year" title="year">2009</span><span class="volume" tagx="volume" title="volume">5</span><span class="fpage" tagx="fpage" title="fpage">1</span><span class="lpage" tagx="lpage" title="lpage">12</span><span class="pub-id"><a href="https://dx.doi.org/10.1371/journal.pgen.1000634">10.1371/journal.pgen.1000634</a></span></span></li> <li tag="ref"><a name="CR24" /><span class="label" tagx="label" title="label">24.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Meredith</span><span class="given-names" tagx="given-names" title="given-names">RW</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gatesy</span><span class="given-names" tagx="given-names" title="given-names">J</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Cheng</span><span class="given-names" tagx="given-names" title="given-names">J</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Springer</span><span class="given-names" tagx="given-names" title="given-names">MS</span></span></span><span class="mixed-article-title" title="mixed-article-title">Pseudogenization of the tooth gene enamelysin (<i>MMP20</i>) in the common ancestor of extant baleen whales</span><span class="source" tagx="source" title="source">Proc R Soc B</span><span class="year" tagx="year" title="year">2011</span><span class="volume" tagx="volume" title="volume">278</span><span class="fpage" tagx="fpage" title="fpage">993</span><span class="lpage" tagx="lpage" title="lpage">1002</span><span class="pub-id"><a href="https://dx.doi.org/10.1098/rspb.2010.1280">10.1098/rspb.2010.1280</a></span></span></li> <li tag="ref"><a name="CR25" /><span class="label" tagx="label" title="label">25.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kawasaki</span><span class="given-names" tagx="given-names" title="given-names">K</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">JC-C</span><span class="given-names" tagx="given-names" title="given-names">H</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Simmer</span><span class="given-names" tagx="given-names" title="given-names">JP</span></span></span><span class="mixed-article-title" title="mixed-article-title">Evolution of <i>Klk4</i> and enamel maturation in eutherians</span><span class="source" tagx="source" title="source">Biol Chem</span><span class="year" tagx="year" title="year">2014</span><span class="volume" tagx="volume" title="volume">395</span><span class="fpage" tagx="fpage" title="fpage">1003</span><span class="lpage" tagx="lpage" title="lpage">1013</span><span class="pub-id"><a href="https://dx.doi.org/10.1515/hsz-2014-0122">10.1515/hsz-2014-0122</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25153384">25153384</a></span></span></li> <li tag="ref"><a name="CR26" /><span class="label" tagx="label" title="label">26.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Springer</span><span class="given-names" tagx="given-names" title="given-names">MS</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Signore</span><span class="given-names" tagx="given-names" title="given-names">AV</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Paijmans</span><span class="given-names" tagx="given-names" title="given-names">JLA</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Vélez-Juarbe</span><span class="given-names" tagx="given-names" title="given-names">J</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Domning</span><span class="given-names" tagx="given-names" title="given-names">DP</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bauer</span><span class="given-names" tagx="given-names" title="given-names">CE</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">He</span><span class="given-names" tagx="given-names" title="given-names">K</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Crerar</span><span class="given-names" tagx="given-names" title="given-names">L</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Campos</span><span class="given-names" tagx="given-names" title="given-names">PF</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Murphy</span><span class="given-names" tagx="given-names" title="given-names">WJ</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Meredith</span><span class="given-names" tagx="given-names" title="given-names">RW</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gatesy</span><span class="given-names" tagx="given-names" title="given-names">J</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Willerslev</span><span class="given-names" tagx="given-names" title="given-names">E</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">MacPhee</span><span class="given-names" tagx="given-names" title="given-names">RDE</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hofreiter</span><span class="given-names" tagx="given-names" title="given-names">M</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Campbell</span><span class="given-names" tagx="given-names" title="given-names">KL</span></span></span><span class="mixed-article-title" title="mixed-article-title">2015 Interordinal gene capture, the phylogenetic position of Steller’s sea cow based on molecular and morphological data, and the macroevolutionary history of Sirenia</span><span class="source" tagx="source" title="source">Mol Phylogenet Evol</span><span class="year" tagx="year" title="year">2015</span><span class="volume" tagx="volume" title="volume">91</span><span class="fpage" tagx="fpage" title="fpage">178</span><span class="lpage" tagx="lpage" title="lpage">193</span><span class="pub-id"><a href="https://dx.doi.org/10.1016/j.ympev.2015.05.022">10.1016/j.ympev.2015.05.022</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/26050523">26050523</a></span></span></li> <li tag="ref"><a name="CR27" /><span class="label" tagx="label" title="label">27.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Springer</span><span class="given-names" tagx="given-names" title="given-names">MS</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Starrett</span><span class="given-names" tagx="given-names" title="given-names">J</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Morin</span><span class="given-names" tagx="given-names" title="given-names">PA</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lanzetti</span><span class="given-names" tagx="given-names" title="given-names">A</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hayashi</span><span class="given-names" tagx="given-names" title="given-names">C</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gatesy</span><span class="given-names" tagx="given-names" title="given-names">J</span></span></span><span class="mixed-article-title" title="mixed-article-title">Inactivation of <i>C4orf26</i> in toothless placental mammals</span><span class="source" tagx="source" title="source">Mol Phylogenet Evol</span><span class="year" tagx="year" title="year">2016</span><span class="volume" tagx="volume" title="volume">95</span><span class="fpage" tagx="fpage" title="fpage">34</span><span class="lpage" tagx="lpage" title="lpage">45</span><span class="pub-id"><a href="https://dx.doi.org/10.1016/j.ympev.2015.11.002">10.1016/j.ympev.2015.11.002</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/26596502">26596502</a></span></span></li> <li tag="ref"><a name="CR28" /><span class="label" tagx="label" title="label">28.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Sharma</span><span class="given-names" tagx="given-names" title="given-names">V</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hecker</span><span class="given-names" tagx="given-names" title="given-names">N</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Roscito</span><span class="given-names" tagx="given-names" title="given-names">JG</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Foerster</span><span class="given-names" tagx="given-names" title="given-names">L</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Langer</span><span class="given-names" tagx="given-names" title="given-names">BE</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hiller</span><span class="given-names" tagx="given-names" title="given-names">M</span></span></span><span class="mixed-article-title" title="mixed-article-title">A genomics approach reveals insights into the importance of gene losses for mammalian adaptations</span><span class="source" tagx="source" title="source">Nat Commun</span><span class="year" tagx="year" title="year">2018</span><span class="volume" tagx="volume" title="volume">9</span><span class="fpage" tagx="fpage" title="fpage">1215</span><span class="pub-id"><a href="https://dx.doi.org/10.1038/s41467-018-03667-1">10.1038/s41467-018-03667-1</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/29572503">29572503</a></span></span></li> <li tag="ref"><a name="CR29" /><span class="label" tagx="label" title="label">29.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gasse</span><span class="given-names" tagx="given-names" title="given-names">B</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Silvent</span><span class="given-names" tagx="given-names" title="given-names">J</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Sire</span><span class="given-names" tagx="given-names" title="given-names">J-Y</span></span></span><span class="mixed-article-title" title="mixed-article-title">Evolutionary analysis suggests that <i>AMTN</i> is enamel-specific and a candidate for AI</span><span class="source" tagx="source" title="source">J Dent Res</span><span class="year" tagx="year" title="year">2012</span><span class="volume" tagx="volume" title="volume">91</span><span class="fpage" tagx="fpage" title="fpage">1085</span><span class="lpage" tagx="lpage" title="lpage">1089</span><span class="pub-id"><a href="https://dx.doi.org/10.1177/0022034512460551">10.1177/0022034512460551</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22968158">22968158</a></span></span></li> <li tag="ref"><a name="CR30" /><span class="label" tagx="label" title="label">30.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Delsuc</span><span class="given-names" tagx="given-names" title="given-names">F</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gasse</span><span class="given-names" tagx="given-names" title="given-names">B</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Sire</span><span class="given-names" tagx="given-names" title="given-names">J-Y</span></span></span><span class="mixed-article-title" title="mixed-article-title">Evolutionary analysis of selective constraints identifies ameloblastin (AMBN) as a potential candidate for amelogenesis imperfecta</span><span class="source" tagx="source" title="source">BMC Evol Biol</span><span class="year" tagx="year" title="year">2015</span><span class="volume" tagx="volume" title="volume">15</span><span class="fpage" tagx="fpage" title="fpage">148</span><span class="pub-id"><a href="https://dx.doi.org/10.1186/s12862-015-0431-0">10.1186/s12862-015-0431-0</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/26223266">26223266</a></span></span></li> <li tag="ref"><a name="CR31" /><span class="label" tagx="label" title="label">31.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kearse</span><span class="given-names" tagx="given-names" title="given-names">M</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Moir</span><span class="given-names" tagx="given-names" title="given-names">R</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wilson</span><span class="given-names" tagx="given-names" title="given-names">A</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Stones-Havas</span><span class="given-names" tagx="given-names" title="given-names">S</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Cheung</span><span class="given-names" tagx="given-names" title="given-names">M</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Sturrock</span><span class="given-names" tagx="given-names" title="given-names">S</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Buxton</span><span class="given-names" tagx="given-names" title="given-names">S</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Cooper</span><span class="given-names" tagx="given-names" title="given-names">A</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Markowitz</span><span class="given-names" tagx="given-names" title="given-names">S</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Duran</span><span class="given-names" tagx="given-names" title="given-names">C</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Thierer</span><span class="given-names" tagx="given-names" title="given-names">T</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ashton</span><span class="given-names" tagx="given-names" title="given-names">B</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Mentjies</span><span class="given-names" tagx="given-names" title="given-names">P</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Drummond</span><span class="given-names" tagx="given-names" title="given-names">A</span></span></span><span class="mixed-article-title" title="mixed-article-title">Geneious basic: an integrated and extendable desktop software platform for the organization and analysis of sequence data</span><span class="source" tagx="source" title="source">Bioinformatics</span><span class="year" tagx="year" title="year">2012</span><span class="volume" tagx="volume" title="volume">28</span><span class="fpage" tagx="fpage" title="fpage">1647</span><span class="lpage" tagx="lpage" title="lpage">1649</span><span class="pub-id"><a href="https://dx.doi.org/10.1093/bioinformatics/bts199">10.1093/bioinformatics/bts199</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22543367">22543367</a></span></span></li> <li tag="ref"><a name="CR32" /><span class="label" tagx="label" title="label">32.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Katoh</span><span class="given-names" tagx="given-names" title="given-names">K</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Misawa</span><span class="given-names" tagx="given-names" title="given-names">K</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kuma</span><span class="given-names" tagx="given-names" title="given-names">K</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Miyata</span><span class="given-names" tagx="given-names" title="given-names">TMAFFT</span></span></span><span class="mixed-article-title" title="mixed-article-title">A novel method for rapid multiple sequence alignment based on fast Fourier transform</span><span class="source" tagx="source" title="source">Nucleic Acids Res</span><span class="year" tagx="year" title="year">2002</span><span class="volume" tagx="volume" title="volume">30</span><span class="fpage" tagx="fpage" title="fpage">3059</span><span class="lpage" tagx="lpage" title="lpage">3066</span><span class="pub-id"><a href="https://dx.doi.org/10.1093/nar/gkf436">10.1093/nar/gkf436</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12136088">12136088</a></span></span></li> <li tag="ref"><a name="CR33" /><span class="label" tagx="label" title="label">33.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Katoh</span><span class="given-names" tagx="given-names" title="given-names">K</span></span></span><span class="mixed-article-title" title="mixed-article-title">Stanley DM. MAFFT multiple sequence alignment software version 7: improvements in performance and usability</span><span class="source" tagx="source" title="source">Mol Biol Evol</span><span class="year" tagx="year" title="year">2013</span><span class="volume" tagx="volume" title="volume">30</span><span class="fpage" tagx="fpage" title="fpage">772</span><span class="lpage" tagx="lpage" title="lpage">780</span><span class="pub-id"><a href="https://dx.doi.org/10.1093/molbev/mst010">10.1093/molbev/mst010</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23329690">23329690</a></span></span></li> <li tag="ref"><a name="CR34" /><span class="label" tagx="label" title="label">34.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ranwez</span><span class="given-names" tagx="given-names" title="given-names">V</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Harispe</span><span class="given-names" tagx="given-names" title="given-names">S</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Delsuc</span><span class="given-names" tagx="given-names" title="given-names">F</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Douzery</span><span class="given-names" tagx="given-names" title="given-names">EJP</span></span></span><span class="mixed-article-title" title="mixed-article-title">MACSE: multiple alignment of coding SEquences accounting for frameshifts and stop codons</span><span class="source" tagx="source" title="source">PLoS One</span><span class="year" tagx="year" title="year">2011</span><span class="volume" tagx="volume" title="volume">6</span><span class="fpage" tagx="fpage" title="fpage">e22594</span><span class="pub-id"><a href="https://dx.doi.org/10.1371/journal.pone.0022594">10.1371/journal.pone.0022594</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21949676">21949676</a></span></span></li> <li tag="ref"><a name="CR35" /><span class="label" tagx="label" title="label">35.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ranwez</span><span class="given-names" tagx="given-names" title="given-names">V</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Douzery</span><span class="given-names" tagx="given-names" title="given-names">EJP</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Cambon</span><span class="given-names" tagx="given-names" title="given-names">C</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Chantret</span><span class="given-names" tagx="given-names" title="given-names">N</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Delsuc</span><span class="given-names" tagx="given-names" title="given-names">F</span></span></span><span class="mixed-article-title" title="mixed-article-title">MACSE v2: toolkit for the alignment of coding sequences accounting for frameshifts and stop codons</span><span class="source" tagx="source" title="source">Mol Biol Evol</span><span class="year" tagx="year" title="year">2018</span><span class="volume" tagx="volume" title="volume">35</span><span class="fpage" tagx="fpage" title="fpage">2582</span><span class="lpage" tagx="lpage" title="lpage">2584</span><span class="pub-id"><a href="https://dx.doi.org/10.1093/molbev/msy159">10.1093/molbev/msy159</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/30165589">30165589</a></span></span></li> <li tag="ref"><a name="CR36" /><span class="label" tagx="label" title="label">36.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Swofford</span><span class="given-names" tagx="given-names" title="given-names">DL</span></span><span class="collab'">collab: PAUP*</span></span><span class="source" tagx="source" title="source">Phylogenetic analysis using parsimony (* and other methods)</span><span class="year" tagx="year" title="year">2002</span><span class="publisher-loc" tagx="publisher-loc" title="publisher-loc">Sinauer associates, Sunderland Massachusetts</span><span class="publisher-name" tagx="publisher-name" title="publisher-name">Sinauer Associates</span></span></li> <li tag="ref"><a name="CR37" /><span class="label" tagx="label" title="label">37.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Stamatakis</span><span class="given-names" tagx="given-names" title="given-names">A</span></span></span><span class="mixed-article-title" title="mixed-article-title">RAxML-VI-HPC: maximum likelihood-based phylogenetic analyses with thousands of taxa and mixed models</span><span class="source" tagx="source" title="source">Bioinformatics</span><span class="year" tagx="year" title="year">2006</span><span class="volume" tagx="volume" title="volume">22</span><span class="fpage" tagx="fpage" title="fpage">2688</span><span class="lpage" tagx="lpage" title="lpage">2690</span><span class="pub-id"><a href="https://dx.doi.org/10.1093/bioinformatics/btl446">10.1093/bioinformatics/btl446</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16928733">16928733</a></span></span></li> <li tag="ref"><a name="CR38" /><span class="label" tagx="label" title="label">38.</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation">Stamatakis A. RAxML version 8: a tool for phylogenetic analysis and post-analysis of large phylogenies. Bioinformatics 201;30;1312–1313.</span></li> <li tag="ref"><a name="CR39" /><span class="label" tagx="label" title="label">39.</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation">Miller MA, Pfeiffer W, Schwartz T. Creating the CIPRES science gateway for inference of large phylogenetic trees. Gateway Computing Environments Workshop. 2010:1–8.</span></li> <li tag="ref"><a name="CR40" /><span class="label" tagx="label" title="label">40.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Stamatakis</span><span class="given-names" tagx="given-names" title="given-names">A</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hoover</span><span class="given-names" tagx="given-names" title="given-names">P</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Rougemont</span><span class="given-names" tagx="given-names" title="given-names">J</span></span></span><span class="mixed-article-title" title="mixed-article-title">A rapid bootstrap algorithm for the RAxML web servers</span><span class="source" tagx="source" title="source">Syst Biol</span><span class="year" tagx="year" title="year">2008</span><span class="volume" tagx="volume" title="volume">57</span><span class="fpage" tagx="fpage" title="fpage">758</span><span class="lpage" tagx="lpage" title="lpage">771</span><span class="pub-id"><a href="https://dx.doi.org/10.1080/10635150802429642">10.1080/10635150802429642</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18853362">18853362</a></span></span></li> <li tag="ref"><a name="CR41" /><span class="label" tagx="label" title="label">41.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lartillot</span><span class="given-names" tagx="given-names" title="given-names">N</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Poujol</span><span class="given-names" tagx="given-names" title="given-names">R</span></span></span><span class="mixed-article-title" title="mixed-article-title">A phylogenetic model for investigating correlated evolution of substitution rates and continuous phenotypic characters</span><span class="source" tagx="source" title="source">Mol Biol Evol</span><span class="year" tagx="year" title="year">2010</span><span class="volume" tagx="volume" title="volume">28</span><span class="fpage" tagx="fpage" title="fpage">729</span><span class="lpage" tagx="lpage" title="lpage">744</span><span class="pub-id"><a href="https://dx.doi.org/10.1093/molbev/msq244">10.1093/molbev/msq244</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20926596">20926596</a></span></span></li> <li tag="ref"><a name="CR42" /><span class="label" tagx="label" title="label">42.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lartillot</span><span class="given-names" tagx="given-names" title="given-names">N</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Delsuc</span><span class="given-names" tagx="given-names" title="given-names">F</span></span></span><span class="mixed-article-title" title="mixed-article-title">Joint reconstruction of divergence times and life-history evolution in placental mammals using a phylogenetic covariance model</span><span class="source" tagx="source" title="source">Evolution</span><span class="year" tagx="year" title="year">2012</span><span class="volume" tagx="volume" title="volume">66</span><span class="fpage" tagx="fpage" title="fpage">1773</span><span class="lpage" tagx="lpage" title="lpage">1787</span><span class="pub-id"><a href="https://dx.doi.org/10.1111/j.1558-5646.2011.01558.x">10.1111/j.1558-5646.2011.01558.x</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22671546">22671546</a></span></span></li> <li tag="ref"><a name="CR43" /><span class="label" tagx="label" title="label">43.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Teeling</span><span class="given-names" tagx="given-names" title="given-names">EC</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Springer</span><span class="given-names" tagx="given-names" title="given-names">MS</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Madsen</span><span class="given-names" tagx="given-names" title="given-names">O</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bates</span><span class="given-names" tagx="given-names" title="given-names">P</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">O’Brien</span><span class="given-names" tagx="given-names" title="given-names">SJ</span></span></span><span class="mixed-article-title" title="mixed-article-title">Murphy WJ. A molecular phylogeny for bats illuminates biogeography and the fossil record</span><span class="source" tagx="source" title="source">Science</span><span class="year" tagx="year" title="year">2005</span><span class="volume" tagx="volume" title="volume">307</span><span class="fpage" tagx="fpage" title="fpage">580</span><span class="lpage" tagx="lpage" title="lpage">584</span><span class="pub-id"><a href="https://dx.doi.org/10.1126/science.1105113">10.1126/science.1105113</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15681385">15681385</a></span></span></li> <li tag="ref"><a name="CR44" /><span class="label" tagx="label" title="label">44.</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation">Meredith RW, Janecka JE, Gatesy J, Ryder OA, Fisher CA, Teeling EC, Goodbla A, Eizirik E, Simão TLL, Stadler T, Rabosky DL, Honeycutt RL, Flynn JJ, Ingram CM, Steiner C, Williams TL, Robinson TJ, Burk-Herrick A, Westerman M, Ayoub NA, Springer MS, Murphy WJ. Impacts of the Cretaceous Terrestrial Revolution and KPg extinction on mammal diversification. Science. 2011;334:521–4.</span></li> <li tag="ref"><a name="CR45" /><span class="label" tagx="label" title="label">45.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Delsuc</span><span class="given-names" tagx="given-names" title="given-names">F</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gibb</span><span class="given-names" tagx="given-names" title="given-names">GC</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kuch</span><span class="given-names" tagx="given-names" title="given-names">M</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Billet</span><span class="given-names" tagx="given-names" title="given-names">G</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hautier</span><span class="given-names" tagx="given-names" title="given-names">L</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Southon</span><span class="given-names" tagx="given-names" title="given-names">J</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Rouillard</span><span class="given-names" tagx="given-names" title="given-names">J-M</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Fernicola</span><span class="given-names" tagx="given-names" title="given-names">JC</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Vizcaíno</span><span class="given-names" tagx="given-names" title="given-names">SF</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">MacPhee</span><span class="given-names" tagx="given-names" title="given-names">RDE</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Poinar</span><span class="given-names" tagx="given-names" title="given-names">HN</span></span></span><span class="mixed-article-title" title="mixed-article-title">The phylogenetic affinities of the extinct glyptodonts</span><span class="source" tagx="source" title="source">Curr Biol</span><span class="year" tagx="year" title="year">2016</span><span class="volume" tagx="volume" title="volume">26</span><span class="fpage" tagx="fpage" title="fpage">R141</span><span class="lpage" tagx="lpage" title="lpage">R156</span><span class="pub-id"><a href="https://dx.doi.org/10.1016/j.cub.2016.01.039">10.1016/j.cub.2016.01.039</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/27336092">27336092</a></span></span></li> <li tag="ref"><a name="CR46" /><span class="label" tagx="label" title="label">46.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Foley</span><span class="given-names" tagx="given-names" title="given-names">NM</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Springer</span><span class="given-names" tagx="given-names" title="given-names">MS</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Teeling</span><span class="given-names" tagx="given-names" title="given-names">EC</span></span></span><span class="mixed-article-title" title="mixed-article-title">Mammal madness: is the mammal tree of life not yet resolved?</span><span class="source" tagx="source" title="source">Philos Trans R Soc B</span><span class="year" tagx="year" title="year">2016</span><span class="volume" tagx="volume" title="volume">371</span><span class="fpage" tagx="fpage" title="fpage">20150140</span><span class="pub-id"><a href="https://dx.doi.org/10.1098/rstb.2015.0140">10.1098/rstb.2015.0140</a></span></span></li> <li tag="ref"><a name="CR47" /><span class="label" tagx="label" title="label">47.</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation">Emerling CA, Huynh HT, Nguyen MA, Meredith RW, Springer MS. Spectral shifts of mammalian ultraviolet-sensitive pigments (short wavelength-sensitive opsin 1) are associated with eye length and photic niche evolution. Proc Roy Soc B. 2015;282:20151817.</span></li> <li tag="ref"><a name="CR48" /><span class="label" tagx="label" title="label">48.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Jones</span><span class="given-names" tagx="given-names" title="given-names">KE</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bielby</span><span class="given-names" tagx="given-names" title="given-names">J</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Cardillo</span><span class="given-names" tagx="given-names" title="given-names">M</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Fritz</span><span class="given-names" tagx="given-names" title="given-names">SA</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">O'Dell</span><span class="given-names" tagx="given-names" title="given-names">J</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Orme</span><span class="given-names" tagx="given-names" title="given-names">CDL</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Safi</span><span class="given-names" tagx="given-names" title="given-names">K</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Sechrest</span><span class="given-names" tagx="given-names" title="given-names">W</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Boakes</span><span class="given-names" tagx="given-names" title="given-names">EH</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Carbone</span><span class="given-names" tagx="given-names" title="given-names">C</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Connolly</span><span class="given-names" tagx="given-names" title="given-names">C</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Cutts</span><span class="given-names" tagx="given-names" title="given-names">MJ</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Foster</span><span class="given-names" tagx="given-names" title="given-names">JK</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Grenyer</span><span class="given-names" tagx="given-names" title="given-names">R</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Habib</span><span class="given-names" tagx="given-names" title="given-names">M</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Plaster</span><span class="given-names" tagx="given-names" title="given-names">CA</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Price</span><span class="given-names" tagx="given-names" title="given-names">SA</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Rigby</span><span class="given-names" tagx="given-names" title="given-names">EA</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Rist</span><span class="given-names" tagx="given-names" title="given-names">J</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Teacher</span><span class="given-names" tagx="given-names" title="given-names">A</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bininda-Emonds</span><span class="given-names" tagx="given-names" title="given-names">ORP</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gittleman</span><span class="given-names" tagx="given-names" title="given-names">JL</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Mace</span><span class="given-names" tagx="given-names" title="given-names">GM</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Purvis</span><span class="given-names" tagx="given-names" title="given-names">A</span></span></span><span class="mixed-article-title" title="mixed-article-title">PanTHERIA: a species-level database of life history, ecology, and geography of extant and recently extinct mammals</span><span class="source" tagx="source" title="source">Ecology</span><span class="year" tagx="year" title="year">2009</span><span class="volume" tagx="volume" title="volume">90</span><span class="fpage" tagx="fpage" title="fpage">2648</span><span class="pub-id"><a href="https://dx.doi.org/10.1890/08-1494.1">10.1890/08-1494.1</a></span></span></li> <li tag="ref"><a name="CR49" /><span class="label" tagx="label" title="label">49.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Yang</span><span class="given-names" tagx="given-names" title="given-names">Z</span></span></span><span class="mixed-article-title" title="mixed-article-title">PAML 4: phylogenetic analysis by maximum likelihood</span><span class="source" tagx="source" title="source">Mol Biol Evol</span><span class="year" tagx="year" title="year">2007</span><span class="volume" tagx="volume" title="volume">24</span><span class="fpage" tagx="fpage" title="fpage">1586</span><span class="lpage" tagx="lpage" title="lpage">1591</span><span class="pub-id"><a href="https://dx.doi.org/10.1093/molbev/msm088">10.1093/molbev/msm088</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17483113">17483113</a></span></span></li> <li tag="ref"><a name="CR50" /><span class="label" tagx="label" title="label">50.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Thorne</span><span class="given-names" tagx="given-names" title="given-names">JL</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kishono</span><span class="given-names" tagx="given-names" title="given-names">H</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Painter</span><span class="given-names" tagx="given-names" title="given-names">IS</span></span></span><span class="mixed-article-title" title="mixed-article-title">Estimating the rate of evolution of the rate of molecular evolution</span><span class="source" tagx="source" title="source">Mol Biol Evol</span><span class="year" tagx="year" title="year">1998</span><span class="volume" tagx="volume" title="volume">15</span><span class="fpage" tagx="fpage" title="fpage">1647</span><span class="lpage" tagx="lpage" title="lpage">1657</span><span class="pub-id"><a href="https://dx.doi.org/10.1093/oxfordjournals.molbev.a025892">10.1093/oxfordjournals.molbev.a025892</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/9866200">9866200</a></span></span></li> <li tag="ref"><a name="CR51" /><span class="label" tagx="label" title="label">51.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Yeo</span><span class="given-names" tagx="given-names" title="given-names">G</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Burge</span><span class="given-names" tagx="given-names" title="given-names">CB</span></span></span><span class="mixed-article-title" title="mixed-article-title">Maximum entropy modeling of short sequence motifs with applications to RNA splicing signals</span><span class="source" tagx="source" title="source">J Comp Biol</span><span class="year" tagx="year" title="year">2004</span><span class="volume" tagx="volume" title="volume">11</span><span class="fpage" tagx="fpage" title="fpage">377</span><span class="lpage" tagx="lpage" title="lpage">394</span><span class="pub-id"><a href="https://dx.doi.org/10.1089/1066527041410418">10.1089/1066527041410418</a></span></span></li> <li tag="ref"><a name="CR52" /><span class="label" tagx="label" title="label">52.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Nitsche</span><span class="given-names" tagx="given-names" title="given-names">A</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Rose</span><span class="given-names" tagx="given-names" title="given-names">D</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Fasold</span><span class="given-names" tagx="given-names" title="given-names">M</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Reiche</span><span class="given-names" tagx="given-names" title="given-names">K</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Stadler</span><span class="given-names" tagx="given-names" title="given-names">PF</span></span></span><span class="mixed-article-title" title="mixed-article-title">Comparison of splice sites reveals that long noncoding RNAs are evolutionarily well conserved</span><span class="source" tagx="source" title="source">RNA</span><span class="year" tagx="year" title="year">2015</span><span class="volume" tagx="volume" title="volume">21</span><span class="fpage" tagx="fpage" title="fpage">1</span><span class="lpage" tagx="lpage" title="lpage">12</span><span class="pub-id"><a href="https://dx.doi.org/10.1261/rna.046342.114">10.1261/rna.046342.114</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25516996">25516996</a></span></span></li> <li tag="ref"><a name="CR53" /><span class="label" tagx="label" title="label">53.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Springer</span><span class="given-names" tagx="given-names" title="given-names">MS</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Meredith</span><span class="given-names" tagx="given-names" title="given-names">RW</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gatesy</span><span class="given-names" tagx="given-names" title="given-names">J</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Emerling</span><span class="given-names" tagx="given-names" title="given-names">CA</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Park</span><span class="given-names" tagx="given-names" title="given-names">J</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Rabosky</span><span class="given-names" tagx="given-names" title="given-names">DL</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Stadler</span><span class="given-names" tagx="given-names" title="given-names">T</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Steiner</span><span class="given-names" tagx="given-names" title="given-names">C</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ryder</span><span class="given-names" tagx="given-names" title="given-names">OA</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Janecka</span><span class="given-names" tagx="given-names" title="given-names">JE</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Fisher</span><span class="given-names" tagx="given-names" title="given-names">CA</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Murphy</span><span class="given-names" tagx="given-names" title="given-names">WJ</span></span></span><span class="mixed-article-title" title="mixed-article-title">Macroevolutionary dynamics and historical biogeography of primate diversification inferred from a species supermatrix</span><span class="source" tagx="source" title="source">PLoS One</span><span class="year" tagx="year" title="year">2012</span><span class="volume" tagx="volume" title="volume">7</span><span class="fpage" tagx="fpage" title="fpage">e49521</span><span class="pub-id"><a href="https://dx.doi.org/10.1371/journal.pone.0049521">10.1371/journal.pone.0049521</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23166696">23166696</a></span></span></li> <li tag="ref"><a name="CR54" /><span class="label" tagx="label" title="label">54.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gaubert</span><span class="given-names" tagx="given-names" title="given-names">P</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Antunes</span><span class="given-names" tagx="given-names" title="given-names">A</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Meng</span><span class="given-names" tagx="given-names" title="given-names">H</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Miao</span><span class="given-names" tagx="given-names" title="given-names">L</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Peigné</span><span class="given-names" tagx="given-names" title="given-names">S</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Justy</span><span class="given-names" tagx="given-names" title="given-names">F</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Njiokou</span><span class="given-names" tagx="given-names" title="given-names">F</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Dufour</span><span class="given-names" tagx="given-names" title="given-names">S</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Danquah</span><span class="given-names" tagx="given-names" title="given-names">E</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Alahakoon</span><span class="given-names" tagx="given-names" title="given-names">J</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Verheyen</span><span class="given-names" tagx="given-names" title="given-names">E</span></span></span><span class="mixed-article-title" title="mixed-article-title">The complete phylogeny of pangolins: scaling up resources for the molecular tracing of the most trafficked mammals on earth</span><span class="source" tagx="source" title="source">J Heredity</span><span class="year" tagx="year" title="year">2017</span><span class="volume" tagx="volume" title="volume">109</span><span class="fpage" tagx="fpage" title="fpage">347</span><span class="lpage" tagx="lpage" title="lpage">359</span><span class="pub-id"><a href="https://dx.doi.org/10.1093/jhered/esx097">10.1093/jhered/esx097</a></span></span></li> <li tag="ref"><a name="CR55" /><span class="label" tagx="label" title="label">55.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Prudent</span><span class="given-names" tagx="given-names" title="given-names">X</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Parra</span><span class="given-names" tagx="given-names" title="given-names">G</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Schwede</span><span class="given-names" tagx="given-names" title="given-names">P</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Roscito</span><span class="given-names" tagx="given-names" title="given-names">JG</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hiller</span><span class="given-names" tagx="given-names" title="given-names">M</span></span></span><span class="mixed-article-title" title="mixed-article-title">Controlling for phylogenetic relatedness and evolutionary rates improves the discovery of associations between species’ phenotypic and genomic differences</span><span class="source" tagx="source" title="source">Mol Biol Evol</span><span class="year" tagx="year" title="year">2016</span><span class="volume" tagx="volume" title="volume">33</span><span class="fpage" tagx="fpage" title="fpage">2135</span><span class="lpage" tagx="lpage" title="lpage">2150</span><span class="pub-id"><a href="https://dx.doi.org/10.1093/molbev/msw098">10.1093/molbev/msw098</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/27222536">27222536</a></span></span></li> <li tag="ref"><a name="CR56" /><span class="label" tagx="label" title="label">56.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Madsen</span><span class="given-names" tagx="given-names" title="given-names">O</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Scally</span><span class="given-names" tagx="given-names" title="given-names">M</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Douady</span><span class="given-names" tagx="given-names" title="given-names">CJ</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kao</span><span class="given-names" tagx="given-names" title="given-names">DJ</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">DeBry</span><span class="given-names" tagx="given-names" title="given-names">RW</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Adkins</span><span class="given-names" tagx="given-names" title="given-names">R</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Amrine</span><span class="given-names" tagx="given-names" title="given-names">HM</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Stanhope</span><span class="given-names" tagx="given-names" title="given-names">MJ</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">de Jong</span><span class="given-names" tagx="given-names" title="given-names">WW</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Springer</span><span class="given-names" tagx="given-names" title="given-names">MS</span></span></span><span class="mixed-article-title" title="mixed-article-title">Parallel adaptive radiations in two major clades of placental mammals</span><span class="source" tagx="source" title="source">Nature</span><span class="year" tagx="year" title="year">2001</span><span class="volume" tagx="volume" title="volume">409</span><span class="fpage" tagx="fpage" title="fpage">610</span><span class="lpage" tagx="lpage" title="lpage">614</span><span class="pub-id"><a href="https://dx.doi.org/10.1038/35054544">10.1038/35054544</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11214318">11214318</a></span></span></li> <li tag="ref"><a name="CR57" /><span class="label" tagx="label" title="label">57.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Murphy</span><span class="given-names" tagx="given-names" title="given-names">WJ</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Eizirik</span><span class="given-names" tagx="given-names" title="given-names">E</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Johnson</span><span class="given-names" tagx="given-names" title="given-names">WE</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zhang</span><span class="given-names" tagx="given-names" title="given-names">YP</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ryder</span><span class="given-names" tagx="given-names" title="given-names">OA</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">O’Brien</span><span class="given-names" tagx="given-names" title="given-names">SJ</span></span></span><span class="mixed-article-title" title="mixed-article-title">Molecular phylogenetics and the origins of placental mammals</span><span class="source" tagx="source" title="source">Nature</span><span class="year" tagx="year" title="year">2001</span><span class="volume" tagx="volume" title="volume">409</span><span class="fpage" tagx="fpage" title="fpage">614</span><span class="lpage" tagx="lpage" title="lpage">618</span><span class="pub-id"><a href="https://dx.doi.org/10.1038/35054550">10.1038/35054550</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11214319">11214319</a></span></span></li> <li tag="ref"><a name="CR58" /><span class="label" tagx="label" title="label">58.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Murphy</span><span class="given-names" tagx="given-names" title="given-names">WJ</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Eizirik</span><span class="given-names" tagx="given-names" title="given-names">E</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">O’Brien</span><span class="given-names" tagx="given-names" title="given-names">SJ</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Madsen</span><span class="given-names" tagx="given-names" title="given-names">O</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Scally</span><span class="given-names" tagx="given-names" title="given-names">M</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Douady</span><span class="given-names" tagx="given-names" title="given-names">CJ</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Teeling</span><span class="given-names" tagx="given-names" title="given-names">E</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ryder</span><span class="given-names" tagx="given-names" title="given-names">OA</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Stanhope</span><span class="given-names" tagx="given-names" title="given-names">MJ</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">de Jong</span><span class="given-names" tagx="given-names" title="given-names">WW</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Springer</span><span class="given-names" tagx="given-names" title="given-names">MS</span></span></span><span class="mixed-article-title" title="mixed-article-title">Resolution of the early placental mammal radiation using Bayesian phylogenetics</span><span class="source" tagx="source" title="source">Science</span><span class="year" tagx="year" title="year">2001</span><span class="volume" tagx="volume" title="volume">294</span><span class="fpage" tagx="fpage" title="fpage">2348</span><span class="lpage" tagx="lpage" title="lpage">2351</span><span class="pub-id"><a href="https://dx.doi.org/10.1126/science.1067179">10.1126/science.1067179</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11743200">11743200</a></span></span></li> <li tag="ref"><a name="CR59" /><span class="label" tagx="label" title="label">59.</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation">Scally M, Madsen O, Douady CJ, de Jong WW, Stanhope MJ, Springer MS. Molecular evidence for the major clades of placental mammals. J Mammal Evol 2001;8:239–277.</span></li> <li tag="ref"><a name="CR60" /><span class="label" tagx="label" title="label">60.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Waddell</span><span class="given-names" tagx="given-names" title="given-names">PJ</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kishino</span><span class="given-names" tagx="given-names" title="given-names">H</span></span></span><span class="mixed-article-title" title="mixed-article-title">Ota R. A phylogenetic foundation for comparative mammalian genomics</span><span class="source" tagx="source" title="source">Genome Informatics</span><span class="year" tagx="year" title="year">2001</span><span class="volume" tagx="volume" title="volume">12</span><span class="fpage" tagx="fpage" title="fpage">141</span><span class="lpage" tagx="lpage" title="lpage">154</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11791233">11791233</a></span></span></li> <li tag="ref"><a name="CR61" /><span class="label" tagx="label" title="label">61.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Delsuc</span><span class="given-names" tagx="given-names" title="given-names">F</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Scally</span><span class="given-names" tagx="given-names" title="given-names">M</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Madsen</span><span class="given-names" tagx="given-names" title="given-names">O</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Stanhope</span><span class="given-names" tagx="given-names" title="given-names">MJ</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">de Jong</span><span class="given-names" tagx="given-names" title="given-names">W</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Catzeflis</span><span class="given-names" tagx="given-names" title="given-names">FM</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Springer</span><span class="given-names" tagx="given-names" title="given-names">MS</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Douzery</span><span class="given-names" tagx="given-names" title="given-names">EJP</span></span></span><span class="mixed-article-title" title="mixed-article-title">Molecular phylogeny of living xenarthrans and the impact of character and taxon sampling on the placental tree rooting</span><span class="source" tagx="source" title="source">Mol Biol Evol</span><span class="year" tagx="year" title="year">2002</span><span class="volume" tagx="volume" title="volume">19</span><span class="fpage" tagx="fpage" title="fpage">1656</span><span class="lpage" tagx="lpage" title="lpage">1671</span><span class="pub-id"><a href="https://dx.doi.org/10.1093/oxfordjournals.molbev.a003989">10.1093/oxfordjournals.molbev.a003989</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12270893">12270893</a></span></span></li> <li tag="ref"><a name="CR62" /><span class="label" tagx="label" title="label">62.</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation">Springer MS, Murphy WJ, Eizirik E, O’Brien SJ. Placental mammal diversification and the Cretaceous-Tertiary boundary. Proc Natl Acad Sci U S A. 2003;100:1056–61.</span></li> <li tag="ref"><a name="CR63" /><span class="label" tagx="label" title="label">63.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Springer</span><span class="given-names" tagx="given-names" title="given-names">MS</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Stanhope</span><span class="given-names" tagx="given-names" title="given-names">MJ</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Madsen</span><span class="given-names" tagx="given-names" title="given-names">O</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">de Jong</span><span class="given-names" tagx="given-names" title="given-names">WW</span></span></span><span class="mixed-article-title" title="mixed-article-title">Molecules consolidate the placental mammal tree</span><span class="source" tagx="source" title="source">Trends Ecol Evol</span><span class="year" tagx="year" title="year">2004</span><span class="volume" tagx="volume" title="volume">19</span><span class="fpage" tagx="fpage" title="fpage">430</span><span class="lpage" tagx="lpage" title="lpage">438</span><span class="pub-id"><a href="https://dx.doi.org/10.1016/j.tree.2004.05.006">10.1016/j.tree.2004.05.006</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16701301">16701301</a></span></span></li> <li tag="ref"><a name="CR64" /><span class="label" tagx="label" title="label">64.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Springer</span><span class="given-names" tagx="given-names" title="given-names">MS</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Murphy</span><span class="given-names" tagx="given-names" title="given-names">WJ</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Eizirik</span><span class="given-names" tagx="given-names" title="given-names">E</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">O’Brien</span><span class="given-names" tagx="given-names" title="given-names">SJ</span></span></span><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Rose</span><span class="given-names" tagx="given-names" title="given-names">KD</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Archibald</span><span class="given-names" tagx="given-names" title="given-names">JK</span></span></span><span class="mixed-article-title" title="mixed-article-title">Evidence for major placental clades</span><span class="source" tagx="source" title="source">The rise of placental mammals: origins and relationships of the major extant clades</span><span class="year" tagx="year" title="year">2005</span><span class="publisher-loc" tagx="publisher-loc" title="publisher-loc">Baltimore</span><span class="publisher-name" tagx="publisher-name" title="publisher-name">Johns Hopkins University Press</span><span class="fpage" tagx="fpage" title="fpage">37</span><span class="lpage" tagx="lpage" title="lpage">49</span></span></li> <li tag="ref"><a name="CR65" /><span class="label" tagx="label" title="label">65.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">McGowen</span><span class="given-names" tagx="given-names" title="given-names">MR</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Spaulding</span><span class="given-names" tagx="given-names" title="given-names">M</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gatesy</span><span class="given-names" tagx="given-names" title="given-names">J</span></span></span><span class="mixed-article-title" title="mixed-article-title">Divergence date estimation and a comprehensive molecular tree of extant cetaceans</span><span class="source" tagx="source" title="source">Mol Phylogenet Evol</span><span class="year" tagx="year" title="year">2009</span><span class="volume" tagx="volume" title="volume">53</span><span class="fpage" tagx="fpage" title="fpage">891</span><span class="lpage" tagx="lpage" title="lpage">906</span><span class="pub-id"><a href="https://dx.doi.org/10.1016/j.ympev.2009.08.018">10.1016/j.ympev.2009.08.018</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19699809">19699809</a></span></span></li> <li tag="ref"><a name="CR66" /><span class="label" tagx="label" title="label">66.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gatesy</span><span class="given-names" tagx="given-names" title="given-names">J</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Geisler</span><span class="given-names" tagx="given-names" title="given-names">JH</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Chang</span><span class="given-names" tagx="given-names" title="given-names">J</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Buell</span><span class="given-names" tagx="given-names" title="given-names">C</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Berta</span><span class="given-names" tagx="given-names" title="given-names">A</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Meredith</span><span class="given-names" tagx="given-names" title="given-names">RW</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Springer</span><span class="given-names" tagx="given-names" title="given-names">MS</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">McGowen</span><span class="given-names" tagx="given-names" title="given-names">MR</span></span></span><span class="mixed-article-title" title="mixed-article-title">A phylogenetic blueprint for a modern whale</span><span class="source" tagx="source" title="source">Mol Phylogenet Evol</span><span class="year" tagx="year" title="year">2013</span><span class="volume" tagx="volume" title="volume">66</span><span class="fpage" tagx="fpage" title="fpage">479</span><span class="lpage" tagx="lpage" title="lpage">506</span><span class="pub-id"><a href="https://dx.doi.org/10.1016/j.ympev.2012.10.012">10.1016/j.ympev.2012.10.012</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23103570">23103570</a></span></span></li> <li tag="ref"><a name="CR67" /><span class="label" tagx="label" title="label">67.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Teeling</span><span class="given-names" tagx="given-names" title="given-names">EC</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Madsen</span><span class="given-names" tagx="given-names" title="given-names">O</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Van Den Bussche</span><span class="given-names" tagx="given-names" title="given-names">RA</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">de Jong</span><span class="given-names" tagx="given-names" title="given-names">WW</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Stanhope</span><span class="given-names" tagx="given-names" title="given-names">MJ</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Springer</span><span class="given-names" tagx="given-names" title="given-names">MS</span></span></span><span class="mixed-article-title" title="mixed-article-title">Microbat paraphyly and the convergent evolution of a key innovation in Old World rhinolophoid microbats</span><span class="source" tagx="source" title="source">Proc Natl Acad Sci U S A</span><span class="year" tagx="year" title="year">2002</span><span class="volume" tagx="volume" title="volume">99</span><span class="fpage" tagx="fpage" title="fpage">1431</span><span class="lpage" tagx="lpage" title="lpage">1436</span><span class="pub-id"><a href="https://dx.doi.org/10.1073/pnas.022477199">10.1073/pnas.022477199</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11805285">11805285</a></span></span></li> <li tag="ref"><a name="CR68" /><span class="label" tagx="label" title="label">68.</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation">Roca AL, Bar-Gal GK, Eizirik E, Helgen KM, Maria R, Springer MS, O’Brien SJ, Murphy WJ. Mesozoic origin for West Indian insectivores. Nature. 2004;429:649–51.</span></li> <li tag="ref"><a name="CR69" /><span class="label" tagx="label" title="label">69.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Springer</span><span class="given-names" tagx="given-names" title="given-names">MS</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Murphy</span><span class="given-names" tagx="given-names" title="given-names">WJ</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Roca</span><span class="given-names" tagx="given-names" title="given-names">AL</span></span></span><span class="mixed-article-title" title="mixed-article-title">Appropriate fossil calibrations and tree constraints uphold the Mesozoic divergence of solenodons from other extant mammals</span><span class="source" tagx="source" title="source">Mol Phylogenet Evol</span><span class="year" tagx="year" title="year">2018</span><span class="volume" tagx="volume" title="volume">121</span><span class="fpage" tagx="fpage" title="fpage">158</span><span class="lpage" tagx="lpage" title="lpage">165</span><span class="pub-id"><a href="https://dx.doi.org/10.1016/j.ympev.2018.01.007">10.1016/j.ympev.2018.01.007</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/29331683">29331683</a></span></span></li> <li tag="ref"><a name="CR70" /><span class="label" tagx="label" title="label">70.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Shoshani</span><span class="given-names" tagx="given-names" title="given-names">J</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Walter</span><span class="given-names" tagx="given-names" title="given-names">RC</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Abraha</span><span class="given-names" tagx="given-names" title="given-names">M</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Berhe</span><span class="given-names" tagx="given-names" title="given-names">S</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Tassy</span><span class="given-names" tagx="given-names" title="given-names">P</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Sanders</span><span class="given-names" tagx="given-names" title="given-names">WJ</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Marchant</span><span class="given-names" tagx="given-names" title="given-names">GH</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Libeskal</span><span class="given-names" tagx="given-names" title="given-names">Y</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ghirmai</span><span class="given-names" tagx="given-names" title="given-names">T</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zinner</span><span class="given-names" tagx="given-names" title="given-names">D</span></span></span><span class="mixed-article-title" title="mixed-article-title">A proboscidean from the late Oligocene of Eritrea, a "missing link" between early Elephantiformes and Elephantimorpha, and biogeographic implications</span><span class="source" tagx="source" title="source">Proc Natl Acad Sci U S A</span><span class="year" tagx="year" title="year">2006</span><span class="volume" tagx="volume" title="volume">103</span><span class="fpage" tagx="fpage" title="fpage">17296</span><span class="lpage" tagx="lpage" title="lpage">17301</span><span class="pub-id"><a href="https://dx.doi.org/10.1073/pnas.0603689103">10.1073/pnas.0603689103</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17085582">17085582</a></span></span></li> <li tag="ref"><a name="CR71" /><span class="label" tagx="label" title="label">71.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Palkopoulou</span><span class="given-names" tagx="given-names" title="given-names">E</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lipson</span><span class="given-names" tagx="given-names" title="given-names">M</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Mallick</span><span class="given-names" tagx="given-names" title="given-names">S</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Nielsen</span><span class="given-names" tagx="given-names" title="given-names">S</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Rohland</span><span class="given-names" tagx="given-names" title="given-names">N</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Baleka</span><span class="given-names" tagx="given-names" title="given-names">N</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Karpinski</span><span class="given-names" tagx="given-names" title="given-names">S</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ivancevic</span><span class="given-names" tagx="given-names" title="given-names">AM</span></span><span class="collab'">collab: To T-H</span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kortschak</span><span class="given-names" tagx="given-names" title="given-names">RD</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Raison</span><span class="given-names" tagx="given-names" title="given-names">JM</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Qu</span><span class="given-names" tagx="given-names" title="given-names">Z</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Chin</span><span class="given-names" tagx="given-names" title="given-names">T-J</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Alt</span><span class="given-names" tagx="given-names" title="given-names">KW</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Claesson</span><span class="given-names" tagx="given-names" title="given-names">S</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Dalén</span><span class="given-names" tagx="given-names" title="given-names">L</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">MacPhee</span><span class="given-names" tagx="given-names" title="given-names">RDE</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Meller</span><span class="given-names" tagx="given-names" title="given-names">H</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Roca</span><span class="given-names" tagx="given-names" title="given-names">AL</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ryder</span><span class="given-names" tagx="given-names" title="given-names">OA</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Heiman</span><span class="given-names" tagx="given-names" title="given-names">D</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Young</span><span class="given-names" tagx="given-names" title="given-names">S</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Breen</span><span class="given-names" tagx="given-names" title="given-names">M</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Williams</span><span class="given-names" tagx="given-names" title="given-names">C</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Aken</span><span class="given-names" tagx="given-names" title="given-names">BL</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ruffier</span><span class="given-names" tagx="given-names" title="given-names">M</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Karlsson</span><span class="given-names" tagx="given-names" title="given-names">E</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Johnson</span><span class="given-names" tagx="given-names" title="given-names">J</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Di Palma</span><span class="given-names" tagx="given-names" title="given-names">F</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Alfordi</span><span class="given-names" tagx="given-names" title="given-names">J</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Adelson</span><span class="given-names" tagx="given-names" title="given-names">DL</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Mailund</span><span class="given-names" tagx="given-names" title="given-names">T</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Munch</span><span class="given-names" tagx="given-names" title="given-names">K</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lindblad-Toh</span><span class="given-names" tagx="given-names" title="given-names">K</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hofreiter</span><span class="given-names" tagx="given-names" title="given-names">M</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Poiner</span><span class="given-names" tagx="given-names" title="given-names">H</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Reich</span><span class="given-names" tagx="given-names" title="given-names">D</span></span></span><span class="mixed-article-title" title="mixed-article-title">A comprehensive genomic history of extinct and living elephants</span><span class="source" tagx="source" title="source">Proc Natl Acad Sci U S A</span><span class="year" tagx="year" title="year">2018</span><span class="volume" tagx="volume" title="volume">115</span><span class="fpage" tagx="fpage" title="fpage">E2566</span><span class="lpage" tagx="lpage" title="lpage">E2574</span><span class="pub-id"><a href="https://dx.doi.org/10.1073/pnas.1720554115">10.1073/pnas.1720554115</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/29483247">29483247</a></span></span></li> <li tag="ref"><a name="CR72" /><span class="label" tagx="label" title="label">72.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Sharma</span><span class="given-names" tagx="given-names" title="given-names">V</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Elghafari</span><span class="given-names" tagx="given-names" title="given-names">A</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hiller</span><span class="given-names" tagx="given-names" title="given-names">M</span></span></span><span class="mixed-article-title" title="mixed-article-title">Coding exon-structure aware realigner (CESAR) utilizes genome alignments for accurate comparative gene annotation</span><span class="source" tagx="source" title="source">Nucleic Acids Res</span><span class="year" tagx="year" title="year">2016</span><span class="volume" tagx="volume" title="volume">44</span><span class="fpage" tagx="fpage" title="fpage">e103</span><span class="pub-id"><a href="https://dx.doi.org/10.1093/nar/gkw210">10.1093/nar/gkw210</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/27016733">27016733</a></span></span></li> <li tag="ref"><a name="CR73" /><span class="label" tagx="label" title="label">73.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Oliver</span><span class="given-names" tagx="given-names" title="given-names">B</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Parisi</span><span class="given-names" tagx="given-names" title="given-names">M</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Clark</span><span class="given-names" tagx="given-names" title="given-names">D</span></span></span><span class="mixed-article-title" title="mixed-article-title">Gene expression neighborhoods</span><span class="source" tagx="source" title="source">J Biol</span><span class="year" tagx="year" title="year">2002</span><span class="volume" tagx="volume" title="volume">1</span><span class="fpage" tagx="fpage" title="fpage">4</span><span class="pub-id"><a href="https://dx.doi.org/10.1186/1475-4924-1-4">10.1186/1475-4924-1-4</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12144705">12144705</a></span></span></li> <li tag="ref"><a name="CR74" /><span class="label" tagx="label" title="label">74.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Franzen</span><span class="given-names" tagx="given-names" title="given-names">JL</span></span></span><span class="mixed-article-title" title="mixed-article-title">The implications of the numerical dating of the Messel fossil deposit (Eocene, Germany) for mammalian biochronology</span><span class="source" tagx="source" title="source">Ann Paleontol</span><span class="year" tagx="year" title="year">2005</span><span class="volume" tagx="volume" title="volume">91</span><span class="fpage" tagx="fpage" title="fpage">329</span><span class="lpage" tagx="lpage" title="lpage">335</span><span class="pub-id"><a href="https://dx.doi.org/10.1016/j.annpal.2005.04.002">10.1016/j.annpal.2005.04.002</a></span></span></li> <li tag="ref"><a name="CR75" /><span class="label" tagx="label" title="label">75.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Rosenberger</span><span class="given-names" tagx="given-names" title="given-names">AL</span></span></span><span class="mixed-article-title" title="mixed-article-title">Loss of incisor enamel in marmosets</span><span class="source" tagx="source" title="source">J Mammal</span><span class="year" tagx="year" title="year">1978</span><span class="volume" tagx="volume" title="volume">59</span><span class="fpage" tagx="fpage" title="fpage">207</span><span class="lpage" tagx="lpage" title="lpage">208</span><span class="pub-id"><a href="https://dx.doi.org/10.2307/1379899">10.2307/1379899</a></span></span></li> <li tag="ref"><a name="CR76" /><span class="label" tagx="label" title="label">76.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ishiyama</span><span class="given-names" tagx="given-names" title="given-names">M</span></span></span><span class="mixed-article-title" title="mixed-article-title">Enamel structure in odontocete whales</span><span class="source" tagx="source" title="source">Scan Microsc</span><span class="year" tagx="year" title="year">1987</span><span class="volume" tagx="volume" title="volume">1</span><span class="fpage" tagx="fpage" title="fpage">1071</span><span class="lpage" tagx="lpage" title="lpage">1079</span></span></li> <li tag="ref"><a name="CR77" /><span class="label" tagx="label" title="label">77.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Loch</span><span class="given-names" tagx="given-names" title="given-names">C</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Duncan</span><span class="given-names" tagx="given-names" title="given-names">W</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Simões-Lopes</span><span class="given-names" tagx="given-names" title="given-names">PC</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kieser</span><span class="given-names" tagx="given-names" title="given-names">JA</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Fordyce</span><span class="given-names" tagx="given-names" title="given-names">RE</span></span></span><span class="mixed-article-title" title="mixed-article-title">Ultrastructure of enamel and dentine in extant dolphins (Cetacea: Delphinoidea and Inioidea)</span><span class="source" tagx="source" title="source">Zoomorphology</span><span class="year" tagx="year" title="year">2013</span><span class="volume" tagx="volume" title="volume">132</span><span class="fpage" tagx="fpage" title="fpage">215</span><span class="lpage" tagx="lpage" title="lpage">225</span><span class="pub-id"><a href="https://dx.doi.org/10.1007/s00435-012-0180-1">10.1007/s00435-012-0180-1</a></span></span></li> <li tag="ref"><a name="CR78" /><span class="label" tagx="label" title="label">78.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Loch</span><span class="given-names" tagx="given-names" title="given-names">C</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kieser</span><span class="given-names" tagx="given-names" title="given-names">JA</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Fordyce</span><span class="given-names" tagx="given-names" title="given-names">RE</span></span></span><span class="mixed-article-title" title="mixed-article-title">Enamel ultrastructure in fossil cetaceans (Cetacea: Archaeoceti and Odontoceti)</span><span class="source" tagx="source" title="source">PLoS One</span><span class="year" tagx="year" title="year">2015</span><span class="volume" tagx="volume" title="volume">10</span><span class="fpage" tagx="fpage" title="fpage">e0116557</span><span class="pub-id"><a href="https://dx.doi.org/10.1371/journal.pone.0116557">10.1371/journal.pone.0116557</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25629995">25629995</a></span></span></li> <li tag="ref"><a name="CR79" /><span class="label" tagx="label" title="label">79.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Koenigswald</span><span class="given-names" tagx="given-names" title="given-names">W</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Rensberger</span><span class="given-names" tagx="given-names" title="given-names">JM</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Pretzschner</span><span class="given-names" tagx="given-names" title="given-names">HU</span></span></span><span class="mixed-article-title" title="mixed-article-title">Changes in the tooth enamel of early Paleocene mammals allowing increased diet diversity</span><span class="source" tagx="source" title="source">Nature</span><span class="year" tagx="year" title="year">1987</span><span class="volume" tagx="volume" title="volume">328</span><span class="fpage" tagx="fpage" title="fpage">150</span><span class="lpage" tagx="lpage" title="lpage">152</span><span class="pub-id"><a href="https://dx.doi.org/10.1038/328150a0">10.1038/328150a0</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/3600790">3600790</a></span></span></li> <li tag="ref"><a name="CR80" /><span class="label" tagx="label" title="label">80.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Domning</span><span class="given-names" tagx="given-names" title="given-names">DP</span></span></span><span class="mixed-article-title" title="mixed-article-title">Marching teeth of the manatee</span><span class="source" tagx="source" title="source">Nat Hist</span><span class="year" tagx="year" title="year">1983</span><span class="volume" tagx="volume" title="volume">92</span><span class="fpage" tagx="fpage" title="fpage">8</span><span class="lpage" tagx="lpage" title="lpage">12</span></span></li> <li tag="ref"><a name="CR81" /><span class="label" tagx="label" title="label">81.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Rohland</span><span class="given-names" tagx="given-names" title="given-names">N</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Reich</span><span class="given-names" tagx="given-names" title="given-names">D</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Mallick</span><span class="given-names" tagx="given-names" title="given-names">S</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Meyer</span><span class="given-names" tagx="given-names" title="given-names">M</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Green</span><span class="given-names" tagx="given-names" title="given-names">RE</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Georgiadis</span><span class="given-names" tagx="given-names" title="given-names">NJ</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Roca</span><span class="given-names" tagx="given-names" title="given-names">AL</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hofreiter</span><span class="given-names" tagx="given-names" title="given-names">M</span></span></span><span class="mixed-article-title" title="mixed-article-title">Genomic DNA sequences from mastodon and woolly mammoth reveal deep speciation of forest and savanna elephants</span><span class="source" tagx="source" title="source">PLoS Biol</span><span class="year" tagx="year" title="year">2010</span><span class="volume" tagx="volume" title="volume">8</span><span class="fpage" tagx="fpage" title="fpage">e1000564</span><span class="pub-id"><a href="https://dx.doi.org/10.1371/journal.pbio.1000564">10.1371/journal.pbio.1000564</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21203580">21203580</a></span></span></li> <li tag="ref"><a name="CR82" /><span class="label" tagx="label" title="label">82.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Seo</span><span class="given-names" tagx="given-names" title="given-names">TK</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kishino</span><span class="given-names" tagx="given-names" title="given-names">H</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Thorne</span><span class="given-names" tagx="given-names" title="given-names">JL</span></span></span><span class="mixed-article-title" title="mixed-article-title">Estimating absolute rates of synonymous and nonsynonymous nucleotide substitution in order to characterize natural selection and date species divergences</span><span class="source" tagx="source" title="source">Mol Biol Evol</span><span class="year" tagx="year" title="year">2004</span><span class="volume" tagx="volume" title="volume">21</span><span class="fpage" tagx="fpage" title="fpage">1201</span><span class="lpage" tagx="lpage" title="lpage">1213</span><span class="pub-id"><a href="https://dx.doi.org/10.1093/molbev/msh088">10.1093/molbev/msh088</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15014159">15014159</a></span></span></li> <li tag="ref"><a name="CR83" /><span class="label" tagx="label" title="label">83.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Krishnan</span><span class="given-names" tagx="given-names" title="given-names">J</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Rohner</span><span class="given-names" tagx="given-names" title="given-names">N</span></span></span><span class="mixed-article-title" title="mixed-article-title">Cavefish and the basis for eye loss</span><span class="source" tagx="source" title="source">Philos Trans Roy Soc B</span><span class="year" tagx="year" title="year">2017</span><span class="volume" tagx="volume" title="volume">372</span><span class="fpage" tagx="fpage" title="fpage">20150487</span><span class="pub-id"><a href="https://dx.doi.org/10.1098/rstb.2015.0487">10.1098/rstb.2015.0487</a></span></span></li> <li tag="ref"><a name="CR84" /><span class="label" tagx="label" title="label">84.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Tokita</span><span class="given-names" tagx="given-names" title="given-names">M</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Chaeychomsri</span><span class="given-names" tagx="given-names" title="given-names">W</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Siruntawineti</span><span class="given-names" tagx="given-names" title="given-names">J</span></span></span><span class="mixed-article-title" title="mixed-article-title">Developmental basis of toothlessness in turtles: insight into convergent evolution of vertebrate morphology</span><span class="source" tagx="source" title="source">Evolution</span><span class="year" tagx="year" title="year">2013</span><span class="volume" tagx="volume" title="volume">67</span><span class="fpage" tagx="fpage" title="fpage">260</span><span class="lpage" tagx="lpage" title="lpage">273</span><span class="pub-id"><a href="https://dx.doi.org/10.1111/j.1558-5646.2012.01752.x">10.1111/j.1558-5646.2012.01752.x</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23289576">23289576</a></span></span></li> <li tag="ref"><a name="CR85" /><span class="label" tagx="label" title="label">85.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Emerling</span><span class="given-names" tagx="given-names" title="given-names">CA</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Widjaja</span><span class="given-names" tagx="given-names" title="given-names">AD</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Nguyen</span><span class="given-names" tagx="given-names" title="given-names">NN</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Springer</span><span class="given-names" tagx="given-names" title="given-names">MS</span></span></span><span class="mixed-article-title" title="mixed-article-title">Their loss is our gain: regressive evolution in vertebrates provides genomic models for uncovering human disease loci</span><span class="source" tagx="source" title="source">J Med Genet</span><span class="year" tagx="year" title="year">2017</span><span class="volume" tagx="volume" title="volume">54</span><span class="fpage" tagx="fpage" title="fpage">787</span><span class="lpage" tagx="lpage" title="lpage">794</span><span class="pub-id"><a href="https://dx.doi.org/10.1136/jmedgenet-2017-104837">10.1136/jmedgenet-2017-104837</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/28814606">28814606</a></span></span></li> </ul> </div> </div>  </body></html>