<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<link rel="dns-prefetch" href="//d33xdlntwy0kbs.cloudfront.net" />
<link rel="shortcut icon" href="https://www.biorxiv.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<meta name="article_thumbnail" content="https://www.biorxiv.org/content/biorxiv/early/2019/10/07/634568/embed/inline-graphic-1.gif" />
<meta name="type" content="article" />
<meta name="category" content="article" />
<meta name="HW.identifier" content="/biorxiv/early/2019/10/07/634568.atom" />
<meta name="HW.pisa" content="biorxiv;634568v3" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="Automated data-intensive forecasting of plant phenology throughout the United States" />
<meta name="DC.Identifier" content="10.1101/634568" />
<meta name="DC.Date" content="2019-10-07" />
<meta name="DC.Publisher" content="Cold Spring Harbor Laboratory" />
<meta name="DC.Rights" content="© 2019, Posted by Cold Spring Harbor Laboratory. This pre-print is available under a Creative Commons License (Attribution 4.0 International), CC BY 4.0, as described at http://creativecommons.org/licenses/by/4.0/" />
<meta name="DC.AccessRights" content="restricted" />
<meta name="DC.Description" content="Phenology - the timing of cyclical and seasonal natural phenomena such as flowering and leaf out - is an integral part of ecological systems with impacts on human activities like environmental management, tourism, and agriculture. As a result, there are numerous potential applications for actionable predictions of when phenological events will occur. However, despite the availability of phenological data with large spatial, temporal, and taxonomic extents, and numerous phenology models, there has been no automated species-level forecasts of plant phenology. This is due in part to the challenges of building a system that integrates large volumes of climate observations and forecasts, uses that data to fit models and make predictions for large numbers of species, and consistently disseminates the results of these forecasts in interpretable ways. Here we describe a new near-term phenology forecasting system that makes predictions for the timing of budburst, flowers, ripe fruit, and fall colors for 78 species across the United States up to 6 months in advance and is updated every four days. We use the lessons learned in developing this system to provide guidance developing large-scale near-term ecological forecast systems more generally, to help advance the use of automated forecasting in ecology." />
<meta name="DC.Contributor" content="Shawn D. Taylor" />
<meta name="DC.Contributor" content="Ethan P. White" />
<meta name="article:published_time" content="2019-10-07" />
<meta name="article:section" content="New Results" />
<meta name="citation_title" content="Automated data-intensive forecasting of plant phenology throughout the United States" />
<meta name="citation_abstract" lang="en" content="&lt;h3&gt;Abstract&lt;/h3&gt;
&lt;p&gt;Phenology - the timing of cyclical and seasonal natural phenomena such as flowering and leaf out - is an integral part of ecological systems with impacts on human activities like environmental management, tourism, and agriculture. As a result, there are numerous potential applications for actionable predictions of when phenological events will occur. However, despite the availability of phenological data with large spatial, temporal, and taxonomic extents, and numerous phenology models, there has been no automated species-level forecasts of plant phenology. This is due in part to the challenges of building a system that integrates large volumes of climate observations and forecasts, uses that data to fit models and make predictions for large numbers of species, and consistently disseminates the results of these forecasts in interpretable ways. Here we describe a new near-term phenology forecasting system that makes predictions for the timing of budburst, flowers, ripe fruit, and fall colors for 78 species across the United States up to 6 months in advance and is updated every four days. We use the lessons learned in developing this system to provide guidance developing large-scale near-term ecological forecast systems more generally, to help advance the use of automated forecasting in ecology.&lt;/p&gt;" />
<meta name="citation_journal_title" content="bioRxiv" />
<meta name="citation_publisher" content="Cold Spring Harbor Laboratory" />
<meta name="citation_publication_date" content="2019/01/01" />
<meta name="citation_mjid" content="biorxiv;634568v3" />
<meta name="citation_id" content="634568v3" />
<meta name="citation_public_url" content="https://www.biorxiv.org/content/10.1101/634568v3" />
<meta name="citation_abstract_html_url" content="https://www.biorxiv.org/content/10.1101/634568v3.abstract" />
<meta name="citation_full_html_url" content="https://www.biorxiv.org/content/10.1101/634568v3.full" />
<meta name="citation_pdf_url" content="https://www.biorxiv.org/content/biorxiv/early/2019/10/07/634568.full.pdf" />
<meta name="citation_doi" content="10.1101/634568" />
<meta name="citation_num_pages" content="30" />
<meta name="citation_article_type" content="Article" />
<meta name="citation_section" content="New Results" />
<meta name="citation_firstpage" content="634568" />
<meta name="citation_author" content="Shawn D. Taylor" />
<meta name="citation_author_institution" content="School of Natural Resources and Environment, University of Florida Gainesville" />
<meta name="citation_author_email" content="shawntaylor@weecology.org" />
<meta name="citation_author" content="Ethan P. White" />
<meta name="citation_author_institution" content="Department of Wildlife Ecology and Conservation, University of Florida" />
<meta name="citation_reference" content="Basler, D. 2016. Evaluating phenological models for the prediction of leaf-out dates in six temperate tree species across central europe. Agricultural and Forest Meteorology 217:10–21." />
<meta name="citation_reference" content="Bauer, P., A. Thorpe, and G. Brunet. 2015. The quiet revolution of numerical weather prediction. Nature 525:47–55." />
<meta name="citation_reference" content="Biederman, L., D. Anderson, N. Sather, J. Pearson, J. Beckman, and J. Prekker. 2018. Using phenological monitoring in situ and historical records to determine environmental triggers for emergence and anthesis in the rare orchid Platanthera praeclara Sheviak &amp; Bowles. Global Ecology and Conservation 16:e00461." />
<meta name="citation_reference" content="Bryan, J. 2018. Excuse me, do you have a moment to talk about version control? The American Statistician 72:20–27." />
<meta name="citation_reference" content="Carrassi, A., M. Bocquet, L. Bertino, and G. Evensen. 2018. Data assimilation in the geosciences: An overview of methods, issues, and perspectives. Wiley Interdisciplinary Reviews: Climate Change 9:e535." />
<meta name="citation_reference" content="Carrillo, C. M., T. R. Ault, and D. S. Wilks. 2018. Spring onset predictability in the north american multimodel ensemble. Journal of Geophysical Research: Atmospheres 123:5913–5926." />
<meta name="citation_reference" content="Chen, Y., J. T. Randerson, D. C. Morton, R. S. DeFries, G. J. Collatz, P. S. Kasibhatla, L. Giglio, Y. Jin, and M. E. Marlier. 2011. Forecasting fire season severity in south america using sea surface temperature anomalies. Science 334:787–791." />
<meta name="citation_reference" content="Chuine, I., and J. Régnière. 2017. Process-based models of phenology for plants and animals. Annual Review of Ecology, Evolution, and Systematics 48:159–182." />
<meta name="citation_reference" content="citation_title=Phenology: An integrative environmental science;citation_pages=275-293;citation_year=2013" />
<meta name="citation_reference" content="Clark, J. S., C. Salk, J. Melillo, and J. Mohan. 2014. Tree phenology responses to winter chilling, spring warming, at north and south range limits. Functional Ecology 28:1344–1355." />
<meta name="citation_reference" content="Cook, B. I., A. Terando, and A. Steiner. 2010. Ecological forecasting under climatic data uncertainty: A case study in phenological modeling. Environmental Research Letters 5:044014." />
<meta name="citation_reference" content="Dalcin, L. D., R. R. Paz, P. A. Kler, and A. Cosimo. 2011. Parallel distributed computing using python. Advances in Water Resources 34:1124–1139." />
<meta name="citation_reference" content="Dask Development Team. 2016. Dask: Library for dynamic task scheduling." />
<meta name="citation_reference" content="Denny, E. G., K. L. Gerst, A. J. Miller-Rushing, G. L. Tierney, T. M. Crimmins, C. A. F. Enquist, P. Guertin, A. H. Rosemartin, M. D. Schwartz, K. A. Thomas, and J. F. Weltzin. 2014. Standardized phenology monitoring methods to track plant and animal activity for science and resource management applications. International Journal of Biometeorology 58:591–601." />
<meta name="citation_reference" content="Dietze, M. C. 2017. Prediction in ecology: A first-principles framework. Ecological Applications 27:2048–2060." />
<meta name="citation_reference" content="Dietze, M. C., A. Fox, L. M. Beck-Johnson, J. L. Betancourt, M. B. Hooten, C. S. Jarnevich, T. H. Keitt, M. A. Kenney, C. M. Laney, L. G. Larsen, H. W. Loescher, C. K. Lunch, B. C. Pijanowski, J. T. Randerson, E. K. Read, A. T. Tredennick, R. Vargas, K. C. Weathers, and E. P. White. 2018. Iterative near-term ecological forecasting: Needs, opportunities, and challenges. Proceedings of the National Academy of Sciences 115:1424–1432." />
<meta name="citation_reference" content="Dormann, C. F., J. M. Calabrese, G. Guillera-Arroita, E. Matechou, V. Bahn, K. Bartoń, C. M. Beale, S. Ciuti, J. Elith, K. Gerstner, J. Guelat, P. Keil, J. J. Lahoz-Monfort, L. J. Pollock, B. Reineking, D. R. Roberts, B. Schröder, W. Thuiller, D. I. Warton, B. A. Wintle, S. N. Wood, R. O. Wüest, and F. Hartig. 2018. Model averaging in ecology: A review of bayesian, information-theoretic, and tactical approaches for predictive inference. Ecological Monographs 0:1–20." />
<meta name="citation_reference" content="Grolemund, G., and H. Wickham. 2011. Dates and times made easy with {lubridate}. Journal of Statistical Software 40:1–25." />
<meta name="citation_reference" content="Hart, E. M., and K. Bell. 2015. Prism: Download data from the oregon prism project. http://github.com/ropensci/prism." />
<meta name="citation_reference" content="Hijmans, R. J. 2017. Raster: Geographic data analysis and modeling. r package version 2.6-7. https://CRAN.R-project.org/package=raster." />
<meta name="citation_reference" content="Hoyer, S., and J. J. Hamman. 2017. Xarray: N-d labeled arrays and datasets in python. Journal of Open Research Software 5." />
<meta name="citation_reference" content="Jones, E., T. Oliphant, P. Peterson, and Others. 2001. SciPy: Open source scientific tools for python. http://www.scipy.org." />
<meta name="citation_reference" content="Luo, Y., K. Ogle, C. Tucker, S. Fei, C. Gao, S. LaDeau, J. S. Clark, and D. S. Schimel. 2011. Ecological forecasting and data assimilation in a data-rich era. Ecological Applications 21:1429–1442." />
<meta name="citation_reference" content="McKinney, W. 2010. Data structures for statistical computing in python. Pages 51–56. in Proceedings of the 9th python in science conference. SciPy, Austin, Texas, USA." />
<meta name="citation_reference" content="Melaas, E. K., M. A. Friedl, and A. D. Richardson. 2016. Multiscale modeling of spring phenology across Deciduous Forests in the Eastern United States. Global Change Biology 22:792–805." />
<meta name="citation_reference" content="Ogilvie, J. E., S. R. Griffin, Z. J. Gezon, B. D. Inouye, N. Underwood, D. W. Inouye, and R. E. Irwin. 2017. Interannual bumble bee abundance is driven by indirect climate effects on floral resource phenology. Ecology Letters 20:1507–1515." />
<meta name="citation_reference" content="Oliphant, T. 2006. A guide to numpy. USA: Trelgol Publishing; Trelgol Publishing, Provo, UT." />
<meta name="citation_reference" content="Pebesma, E. J., and R. S. Bivand. 2005. Classes and methods for spatial data in {R}. R News 5:9–13." />
<meta name="citation_reference" content="Petchey, O. L., M. Pontarp, T. M. Massie, S. Kéfi, A. Ozgul, M. Weilenmann, G. M. Palamara, F. Altermatt, B. Matthews, J. M. Levine, D. Z. Childs, B. J. McGill, M. E. Schaepman, B. Schmid, P. Spaak, A. P. Beckerman, F. Pennekamp, and I. S. Pearse. 2015. The ecological forecast horizon, and examples of its uses and determinants. Ecology Letters 18:597–611." />
<meta name="citation_reference" content="Pierce, D. 2017. Ncdf4: Interface to unidata netCDF (version 4 or earlier) format data files}." />
<meta name="citation_reference" content="Prevéy, J., L. Parker, C. Harrington, C. Lamb, and M. Proctor. In revision. Climate change shifts the habitat suitability and phenology of black huckleberry. Agricultural and Forest Meteorology." />
<meta name="citation_reference" content="PRISM Climate Group. 2004. Oregon state university. http://prism.oregonstate.edu; Oregon State University." />
<meta name="citation_reference" content="Python Software Foundation. 2003. Python language reference manual, version 3.6. http://www.python.org." />
<meta name="citation_reference" content="Ram, K. 2013. Git can facilitate greater reproducibility and increased transparency in science. Source Code for Biology and Medicine 8:7." />
<meta name="citation_reference" content="Richardson, A. D., R. S. Anderson, M. A. Arain, A. G. Barr, G. Bohrer, G. Chen, J. M. Chen, P. Ciais, K. J. Davis, A. R. Desai, M. C. Dietze, D. Dragoni, S. R. Garrity, C. M. Gough, R. Grant, D. Y. Hollinger, H. A. Margolis, H. McCaughey, M. Migliavacca, R. K. Monson, J. W. Munger, B. Poulter, B. M. Raczka, D. M. Ricciuto, A. K. Sahoo, K. Schaefer, H. Tian, R. Vargas, H. Verbeeck, J. Xiao, and Y. Xue. 2012. Terrestrial biosphere models need better representation of vegetation phenology: Results from the north american carbon program site synthesis. Global Change Biology 18:566–584." />
<meta name="citation_reference" content="Saha, S., S. Moorthi, H.-L. Pan, X. Wu, J. Wang, S. Nadiga, P. Tripp, R. Kistler, J. Woollen, D. Behringer, H. Liu, D. Stokes, R. Grumbine, G. Gayno, J. Wang, Y.-T. Hou, H.-y. Chuang, H.-M. H. Juang, J. Sela, M. Iredell, R. Treadon, D. Kleist, P. Van Delst, D. Keyser, J. Derber, M. Ek, J. Meng, H. Wei, R. Yang, S. Lord, H. van den Dool, A. Kumar, W. Wang, C. Long, M. Chelliah, Y. Xue, B. Huang, J.-K. Schemm, W. Ebisuzaki, R. Lin, P. Xie, M. Chen, S. Zhou, W. Higgins, C.-Z. Zou, Q. Liu, Y. Chen, Y. Han, L. Cucurull, R. W. Reynolds, G. Rutledge, and M. Goldberg. 2010. The ncep climate forecast system reanalysis. Bulletin of the American Meteorological Society 91:1015–1058." />
<meta name="citation_reference" content="Saha, S., S. Moorthi, X. Wu, J. Wang, S. Nadiga, P. Tripp, D. Behringer, Y.-T. Hou, H.-y. Chuang, M. Iredell, M. Ek, J. Meng, R. Yang, M. P. Mendez, H. van den Dool, Q. Zhang, W. Wang, M. Chen, and E. Becker. 2014. The ncep climate forecast system version 2. Journal of Climate 27:2185–2208." />
<meta name="citation_reference" content="Schwartz, M. D., T. R. Ault, and J. L. Betancourt. 2013. Spring onset variations and trends in the continental United States: past and regional assessment using temperature-based indices. International Journal of Climatology 33:2917–2922." />
<meta name="citation_reference" content="Stoner, A. M. K., K. Hayhoe, X. Yang, and D. J. Wuebbles. 2013. An asynchronous regional regression model for statistical downscaling of daily climate variables. International Journal of Climatology 33:2473–2494." />
<meta name="citation_reference" content="Taylor, S. D. 2018. PyPhenology: A python framework for plant phenology modelling. Journal of Open Source Software 3:827." />
<meta name="citation_reference" content="Taylor, S. D., J. M. Meiners, K. Riemer, M. C. Orr, and E. P. White. 2019. Comparison of large-scale citizen science data and long-term study data for phenology modeling. Ecology 100:e02568." />
<meta name="citation_reference" content="citation_author=US Geological Survey;citation_year=1999" />
<meta name="citation_reference" content="USA National Phenology Network. 2018. Plant and animal phenology data. data type: Individual phenometrics. 01/01/2008-12/31/2017 for region: 49.9375°, −66.4791667° (ur); 24.0625°, −125.0208333° (ll). USA-NPN, Tucson, Arizona, USA. Data set accessed 07/08/2018 at http://doi.org/10.5066/F78S4N1V." />
<meta name="citation_reference" content="USA National Phenology Network. 2019. Plant and animal phenology data. data type: Individual phenometrics. 01/01/2019-05/08/2019 for region: 49.9375°, −66.4791667° (ur); 24.0625°, −125.0208333° (ll). USA-NPN, Tucson, Arizona, USA. Data set accessed 05/09/2019 at http://doi.org/10.5066/F78S4N1V." />
<meta name="citation_reference" content="Van Doren, B. M., and K. G. Horton. 2018. A continental system for forecasting bird migration. Science 361:1115–1118." />
<meta name="citation_reference" content="Weisheimer, A., and T. N. Palmer. 2014. On the reliability of seasonal climate forecasts. Journal of The Royal Society Interface 11:20131162–20131162." />
<meta name="citation_reference" content="Welch, H., E. L. Hazen, S. J. Bograd, M. G. Jacox, S. Brodie, D. Robinson, K. L. Scales, L. Dewitt, and R. Lewison. 2019. Practical considerations for operationalizing dynamic management tools. Journal of Applied Ecology 56:459–469." />
<meta name="citation_reference" content="White, E. P., G. M. Yenni, S. D. Taylor, E. M. Christensen, E. K. Bledsoe, J. L. Simonis, and S. K. Morgan Ernest. 2018. Developing an automated iterative near-term forecasting system for an ecological study. Methods in Ecology and Evolution." />
<meta name="citation_reference" content="Wickham, H. 2016. Ggplot2: Elegant graphics for data analysis. Springer-Verlag New York." />
<meta name="citation_reference" content="Wickham, H., and L. Henry. 2018. Tidyr: Easily tidy data with ‘spread()’ and ‘gather()’ functions." />
<meta name="citation_reference" content="Yenni, G. M., E. M. Christensen, E. K. Bledsoe, S. R. Supp, R. M. Diaz, E. P. White, and S. K. M. Ernest. 2019. Developing a modern data workflow for regularly updated data. PLOS Biology 17:e3000125." />
<meta name="citation_reference" content="Yun, K., J. Hsiao, M.-P. Jung, I.-T. Choi, D. M. Glenn, K.-M. Shim, and S.-H. Kim. 2017. Can a multi-model ensemble improve phenology predictions for climate change studies? Ecological Modelling 362:54–64." />
<meta name="twitter:title" content="Automated data-intensive forecasting of plant phenology throughout the United States" />
<meta name="twitter:site" content="@biorxivpreprint" />
<meta name="twitter:card" content="summary" />
<meta name="twitter:image" content="https://www.biorxiv.org/sites/default/files/images/biorxiv_logo_homepage7-5-small.png" />
<meta name="twitter:description" content="Phenology - the timing of cyclical and seasonal natural phenomena such as flowering and leaf out - is an integral part of ecological systems with impacts on human activities like environmental management, tourism, and agriculture. As a result, there are numerous potential applications for actionable predictions of when phenological events will occur. However, despite the availability of phenological data with large spatial, temporal, and taxonomic extents, and numerous phenology models, there has been no automated species-level forecasts of plant phenology. This is due in part to the challenges of building a system that integrates large volumes of climate observations and forecasts, uses that data to fit models and make predictions for large numbers of species, and consistently disseminates the results of these forecasts in interpretable ways. Here we describe a new near-term phenology forecasting system that makes predictions for the timing of budburst, flowers, ripe fruit, and fall colors for 78 species across the United States up to 6 months in advance and is updated every four days. We use the lessons learned in developing this system to provide guidance developing large-scale near-term ecological forecast systems more generally, to help advance the use of automated forecasting in ecology." />
<meta name="og-title" property="og:title" content="Automated data-intensive forecasting of plant phenology throughout the United States" />
<meta name="og-url" property="og:url" content="https://www.biorxiv.org/content/10.1101/634568v3" />
<meta name="og-site-name" property="og:site_name" content="bioRxiv" />
<meta name="og-description" property="og:description" content="Phenology - the timing of cyclical and seasonal natural phenomena such as flowering and leaf out - is an integral part of ecological systems with impacts on human activities like environmental management, tourism, and agriculture. As a result, there are numerous potential applications for actionable predictions of when phenological events will occur. However, despite the availability of phenological data with large spatial, temporal, and taxonomic extents, and numerous phenology models, there has been no automated species-level forecasts of plant phenology. This is due in part to the challenges of building a system that integrates large volumes of climate observations and forecasts, uses that data to fit models and make predictions for large numbers of species, and consistently disseminates the results of these forecasts in interpretable ways. Here we describe a new near-term phenology forecasting system that makes predictions for the timing of budburst, flowers, ripe fruit, and fall colors for 78 species across the United States up to 6 months in advance and is updated every four days. We use the lessons learned in developing this system to provide guidance developing large-scale near-term ecological forecast systems more generally, to help advance the use of automated forecasting in ecology." />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.biorxiv.org/sites/default/files/images/biorxiv_logo_homepage7-5-small.png" />
<meta name="citation_date" content="2019-10-07" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/10.1101/634568v3.full.pdf" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/10.1101/634568v3.full.txt" />
<link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/10.1101/634568v3.ppt" />
<meta name="description" content="bioRxiv - the preprint server for biology, operated by Cold Spring Harbor Laboratory, a research and educational institution" />
<meta name="generator" content="Drupal 7 (http://drupal.org)" />
<link rel="canonical" href="https://www.biorxiv.org/content/10.1101/634568v3" />
<link rel="shortlink" href="https://www.biorxiv.org/node/939653" />
    <title>Automated data-intensive forecasting of plant phenology throughout the United States | bioRxiv</title>  
    <link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__jMRAK66KMC1e4TQlwUNn3KiWVDC5AjueUAYEm1xBY_U__KSUjdT4jdcJ4qJz6fKY1K9WkYh1a5EcaWZxt_-zbTis__dPhHATyREQrnntSIgxU8XNoYny4rjInxP7lLKSNxQyY.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q5yzns" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__icrP-Y0qw2NP73rJxxkUtrJta0z3VLAABtNjfSdEghA__rlN3rx0wOoswGNHZMeZM96bNn-3cLWHXTztGDoThNlw__dPhHATyREQrnntSIgxU8XNoYny4rjInxP7lLKSNxQyY.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__q48SDxVDWut_cH6dTPyQe7HjADuSLLmJVFztuiJEFrU__DJqddQGjm5dXCHsaiwNriGytKN5eevIQqyjdq65TZtE__dPhHATyREQrnntSIgxU8XNoYny4rjInxP7lLKSNxQyY.css" media="all" />
<style type="text/css" media="all">
/* <![CDATA[ */
#sliding-popup.sliding-popup-bottom,#sliding-popup.sliding-popup-bottom .eu-cookie-withdraw-banner,.eu-cookie-withdraw-tab{background:gray}#sliding-popup.sliding-popup-bottom.eu-cookie-withdraw-wrapper{background:transparent}#sliding-popup .popup-content #popup-text h1,#sliding-popup .popup-content #popup-text h2,#sliding-popup .popup-content #popup-text h3,#sliding-popup .popup-content #popup-text p,.eu-cookie-compliance-secondary-button,.eu-cookie-withdraw-tab{color:#fff !important}.eu-cookie-withdraw-tab{border-color:#fff}.eu-cookie-compliance-more-button{color:#fff !important}
/* ]]> */
</style>

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__dPhHATyREQrnntSIgxU8XNoYny4rjInxP7lLKSNxQyY.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__B6tGdsQSt1vT8d8j-19-oS4FKJe83RrSNBkjPs9jfos__6VkE5WcQmmQpSVTQx-xS2-zn_nlwFlDm-EN1HLJNjrA__dPhHATyREQrnntSIgxU8XNoYny4rjInxP7lLKSNxQyY.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__Cm80tkaB8yhCBXUaefYJe7GLLFWYsVxjke4GKdtw7XY__F3sVKlGyHKCaAPI_WdiHD7l_-05B6b9bLdUwzi1me90__dPhHATyREQrnntSIgxU8XNoYny4rjInxP7lLKSNxQyY.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__dPhHATyREQrnntSIgxU8XNoYny4rjInxP7lLKSNxQyY.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__dPhHATyREQrnntSIgxU8XNoYny4rjInxP7lLKSNxQyY.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__O9AieMsHBlZYeQh-sAOhdoS_nAEDhkj0jDZjGc_kUug__f_9pwrTE8mmshoJRfurwkQ36QonD_Jiwbop2sscnkrE__dPhHATyREQrnntSIgxU8XNoYny4rjInxP7lLKSNxQyY.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://d33xdlntwy0kbs.cloudfront.net/cshl_custom.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/all/libraries/modernizr/modernizr.min.js?q5yzns"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__yOhrJUBmMpN-jsuQlJYmS8ovkQU6I6KInr295HrSTb0__c--zHhbuOWmkdWkJYXiNWDmkpDRPesmbBjGlQcyqBOY__dPhHATyREQrnntSIgxU8XNoYny4rjInxP7lLKSNxQyY.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var _prum=[['id', '52e95df3abe53d2e33000000'], ['mark', 'firstbyte',
      (new Date()).getTime()]]; (function() {
      var s=document.getElementsByTagName('script')[0],
      p=document.createElement('script');
      p.async='async'; p.src='//rum-static.pingdom.net/prum.min.js';s.parentNode.insertBefore(p,s);})();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js___KyvuEvfmewWxdOLBXTYsSUo8Pfm_DijV6fWtGvSL_k__HSKkTgIr7nDcdvQYewipf6v4ak6ZLUOQkwPj44wKouA__dPhHATyREQrnntSIgxU8XNoYny4rjInxP7lLKSNxQyY.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__3uCzzYE5jr6ly9nvsM8PRBRkhh4p0NWCMDja4EKBy4M__dmGGVYcB8LhnbR2iAmVW0ZyXI6QuFVYFRORa5XxglXs__dPhHATyREQrnntSIgxU8XNoYny4rjInxP7lLKSNxQyY.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__0zdUME0GLauYYyEch2tGPydIw-IG7Uizip0Qb4wUsig__3RgszrCgk4bOnTzo4vaQ-hrWzaG6oD19l-R9CQ1YQ-o__dPhHATyREQrnntSIgxU8XNoYny4rjInxP7lLKSNxQyY.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__W6wAjopaALytcBam7oT3WhEzllkWaB9R-8hazHyvShM__rK5YBIqBy9qt0di0Sn4DriBx9TJupdmv6uGmLJyZvDs__dPhHATyREQrnntSIgxU8XNoYny4rjInxP7lLKSNxQyY.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-45638026-1", {"cookieDomain":"auto"});ga("set", "page", location.pathname + location.search + location.hash);ga("send", "pageview");ga('create', 'UA-189672-38', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__1PtAOZ8x7l6QAi8D7X6Kj_Y7HcUOFzvWRvvYHUcCT98__U8IB92u_CnnGwJ-X_xaUa9lzsC8OzUfN0WkK2JQfirg__dPhHATyREQrnntSIgxU8XNoYny4rjInxP7lLKSNxQyY.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings,{"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"w3vnVRSE14kRbQ6Fqgb8JBpzX9KaL9exMsYYOkB9FoM","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"modules\/comment\/comment.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"modules\/forum\/forum.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"sites\/default\/modules\/biorxiv\/css\/biorxiv.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/flexible\/flexible.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_collection\/css\/highwire_subject_collection.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire.highwire-list-expand.css":1,"public:\/\/ctools\/css\/bab59090150caf0c64e27afa1b36eef1.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/contrib\/eu_cookie_compliance\/css\/eu_cookie_compliance.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"0":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_biorxiv_styles\/css\/biorxiv-styles.css.less":1,"https:\/\/d33xdlntwy0kbs.cloudfront.net\/cshl_custom.css":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_figures.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"http:\/\/d33xdlntwy0kbs.cloudfront.net\/cshl_custom.js":1,"sites\/all\/modules\/contrib\/eu_cookie_compliance\/js\/eu_cookie_compliance.js":1,"sites\/default\/modules\/jnl_biorxiv_styles\/js\/adv-search-collection-toggle.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/eu_cookie_compliance\/js\/jquery.cookie-1.4.1.min.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/default\/modules\/jnl_biorxiv_styles\/js\/biorxiv-scripts.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_nps_survey\/js\/nps_survey.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"sites\/default\/modules\/contrib\/disqus\/disqus.js":1,"misc\/textarea.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/contrib\/service_links\/js\/twitter_button.js":1,"sites\/all\/modules\/contrib\/service_links\/js\/facebook_like.js":1,"sites\/all\/modules\/contrib\/service_links\/js\/google_plus_one.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.highwire-list-expand.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"939653","apath":"\/biorxiv\/early\/2019\/10\/07\/634568.atom","pisa":"biorxiv;634568v3","processed":["highwire_math"],"markup":[{"requested":"full-text","variant":"full-text","view":"full","pisa":"biorxiv;634568v3"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"disqus":{"domain":"biorxivstage","url":"https:\/\/www.biorxiv.org\/content\/10.1101\/634568v3","title":"Automated data-intensive forecasting of plant phenology throughout the United States","identifier":"node\/939653"},"panels_ajax_pane":{"new-41":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022F+peYuNjWEGL78aJglf2H3Kj2aHfXlJX9J8QmDFuAcZrJr+Vp6c2\\\\\\\/rS8zpKCBRDAyADB+E0p4AtJQNNzFPQPw+Xo8i0jLY8JVSmU0xy9O6J\\\\\\\/XFKoWmK6OWhJa0PTXiP1Mx1VT4osSTsZH2gZxpIr1XPDx\\\\\\\/W6AvL1gorUFsW6F6CEAFV23zpiBKlqc79cPWQUyFnWqRHkeSOp6TTKiH21U50Xp8XHYd0OZ74RWZ3F1kNMXKU8t8AWsMd1ti7I29uj80aeHlBcu7XEtBmk6Ojt5S3Euwrshh+r24nCWgpsy48bGpK5oDx1LyYbi4ckqa4cmsR0IlngnHIhcgwQBvL4G3MmG6u9PEHy81tKgSGzKMK07h3yrw7dXMh6Fh88HWINBVwO38bNAL3dr6ksyuxFYDBsAIQ84tX\\\\\\\/A2hUYYbK\\\\\\\/WgWuTyC25TwBcnNdkL2Xj2DMUGbDBYnm9\\\\\\\/B38CRrYaK7Sf08WcbXgGvVyNFaaEjW\\\\\\\/NKw4sN2f5wIW6UPp2IBAqvDuBdyp2T1Ls3oDOXUZ2AlXwc9ec8T0t3J6XlaiTsWPYoYbNOQYx6Fr490WpDFn+9JdiWr66aETzP4\\\\\\\/33brB3SE1ZE0JmJdzECIokUWQaUUrkjpfUgN0za3hANK2dZ1WlsMxj9mxP\\\\\\\/huLB0Nr1qDrGnuEQ\\\\\\\/yFpkMD7Wkq6culyFzP3pGn3Mi0zds22vo6K5DT8DmCJiDHza\\\\\\\/mcxsJYYlDcyNisq3E59EFskgE2Isw5k2uQWq29A7z8pSZi6xrsSOx7Wo2kV10YQIXCruL6rcdruJtEW7oc+G7YyMRYyY0YThI9p+Ii6bbv4Z\\\\\\\/b53dhwZt8JZnnSxA\\\\\\\/52+I\\\\\\\/WOEtSIEhkpB6RyT9mVUEGAU5FQzb8dMuUKlSWhKcq6wzJ5LWpSfi0ZhpesnwUVHgDKMDMyskSJA1XlqKK5jgybKu19XW8N+GpRLAvDB2Upb7b+ESFurlCxHGuWdI9v3uHoFSgroSLtQINIXjRnqruj8UPFIVWA11QAuff9gOP9OdJIkMhLOoxoGvW8KL3DMhoqvgERSXrL0p8K3gW5eYYZZGEdJ5bXM84XaSlQWSKJgHlcxZmQs3r9NN8mugyixB99UNErXLLXcBffY8F+oBJKXxQ\\\\\\\/peI3KjsNNbv8tvfKjkUXDPVOBDHdnn\\\\\\\/8QEaZbyoX6Y91v76K6efPZtCzAMuHnkaVf+mbJ7pLZlMO6rb1RxAYSWd4PJ9Ilnq9uNcpBRGrEr4o0y6uq\\\\\\\/tOf4luQYXIMBd5F1YjRrv6mZ92VVOnzNVfjzZbuluBI5Us0vEnTUpRIdKe7KWdC36GpKgPtDf5Cx5ZBmY=\\\u0022,\\\u0022iv\\\u0022:\\\u0022+XCKZqgn0H7CLz+OCIUNDw==\\\u0022,\\\u0022salt\\\u0022:\\\u0022f9d8845712f72c9fee5db7d9ff111eab\\\u0022}\u0022,\u0022hmac\u0022:\u00224a5125506e54606fb407a9edf5cbb6975e84c1175f000261fc9f6acf802edc0f\u0022}"},"urlIsAjaxTrusted":{"\/content\/10.1101\/634568v3.full":true},"ws_fl":{"width":100,"height":21},"ws_gpo":{"size":"","annotation":"","lang":"","callback":"","width":300},"color":{"logo":"https:\/\/www.biorxiv.org\/sites\/default\/files\/bioRxiv_article.jpg"},"highwire_list_expand":{"is_collapsed":"1"},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"eu_cookie_compliance":{"popup_enabled":1,"popup_agreed_enabled":0,"popup_hide_agreed":0,"popup_clicking_confirmation":1,"popup_scrolling_confirmation":false,"popup_html_info":"\u003Cdiv\u003E\n  \u003Cdiv class =\u0022popup-content info\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n      \u003Cp\u003EWe use cookies on this site to enhance your user experience. By clicking any link on this page you are giving your consent for us to set cookies.\u003C\/p\u003E\n    \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022agree-button eu-cookie-compliance-default-button\u0022\u003EContinue\u003C\/button\u003E\n                    \u003Cbutton type=\u0022button\u0022 class=\u0022find-more-button eu-cookie-compliance-more-button\u0022\u003EFind out more\u003C\/button\u003E\n          \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E","use_mobile_message":false,"mobile_popup_html_info":"\u003Cdiv\u003E\n  \u003Cdiv class =\u0022popup-content info\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n          \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022agree-button eu-cookie-compliance-default-button\u0022\u003EContinue\u003C\/button\u003E\n                    \u003Cbutton type=\u0022button\u0022 class=\u0022find-more-button eu-cookie-compliance-more-button\u0022\u003EFind out more\u003C\/button\u003E\n          \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E\n","mobile_breakpoint":"768","popup_html_agreed":"\u003Cdiv\u003E\n  \u003Cdiv class=\u0022popup-content agreed\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n      \u003Ch2\u003EThank you for accepting cookies\u003C\/h2\u003E\u003Cp\u003EYou can now hide this message or find out more about cookies.\u003C\/p\u003E    \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022hide-popup-button eu-cookie-compliance-hide-button\u0022\u003EHide\u003C\/button\u003E\n              \u003Cbutton type=\u0022button\u0022 class=\u0022find-more-button eu-cookie-compliance-more-button-thank-you\u0022 \u003EMore info\u003C\/button\u003E\n          \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E","popup_use_bare_css":false,"popup_height":"auto","popup_width":"100%","popup_delay":1000,"popup_link":"\/help\/cookie-policy","popup_link_new_window":1,"popup_position":null,"popup_language":"en","store_consent":false,"better_support_for_screen_readers":0,"reload_page":0,"domain":"","popup_eu_only_js":0,"cookie_lifetime":"365","cookie_session":false,"disagree_do_not_show_popup":0,"method":"default","whitelisted_cookies":"","withdraw_markup":"\u003Cbutton type=\u0022button\u0022 class=\u0022eu-cookie-withdraw-tab\u0022\u003EPrivacy settings\u003C\/button\u003E\n\u003Cdiv class=\u0022eu-cookie-withdraw-banner\u0022\u003E\n  \u003Cdiv class=\u0022popup-content info\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n      \u003Cp\u003E\u0026lt;h2\u0026gt;We use cookies on this site to enhance your user experience\u0026lt;\/h2\u0026gt;\u0026lt;p\u0026gt;You have given your consent for us to set cookies.\u0026lt;\/p\u0026gt;\u003C\/p\u003E\n    \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022eu-cookie-withdraw-button\u0022\u003EWithdraw consent\u003C\/button\u003E\n    \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E\n","withdraw_enabled":false},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1,"trackUrlFragments":1},"jnl_biorxiv_styles":{"defaultJCode":"biorxiv"},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-939653 node-type-highwire-article context-content hw-default-jcode-biorxiv hw-article-type-article hw-article-category-new-results">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-M677548" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-M677548');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
        <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
    
  <div id="zone-branding" class="zone zone-branding clearfix print-display-block container-30">
    <div class="grid-15 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="bioRxiv" src="https://www.biorxiv.org/sites/default/files/bioRxiv_article.jpg" /></a>      </div>
                </div>
          </div>
</div><div class="grid-11 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-system block-menu block-main-menu block-system-main-menu odd block-without-title" id="block-system-main-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/" title="" class="" data-icon-position="" data-hide-link-title="0">Home</a></li>
<li class="leaf" role="menuitem"><a href="/about-biorxiv" class="" data-icon-position="" data-hide-link-title="0">About</a></li>
<li class="leaf" role="menuitem"><a href="/submit-a-manuscript" class="" data-icon-position="" data-hide-link-title="0">Submit</a></li>
<li class="last leaf" role="menuitem"><a href="/content/alertsrss" title="" class="" data-icon-position="" data-hide-link-title="0">ALERTS / RSS</a></li>
</ul></nav>    </div>
  </div>
</div><div class="block block-panels-mini block-biorxiv-search-box block-panels-mini-biorxiv-search-box even block-without-title" id="block-panels-mini-biorxiv-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-biorxiv_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/10.1101/634568v3.full" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--2" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-KHM38uqneOYoQ4zejSWYbc1Zlgrck8tc78OHvt7FSfk" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2 advanced-search-link" >
  
      
  
  <div class="pane-content">
    <a href="/search">Advanced Search</a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix container-30">
  	      </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="939653" id="node939653" data-pisa="biorxiv;634568v3" data-pisa-master="biorxiv;634568" data-apath="/biorxiv/early/2019/10/07/634568.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div  class="highwire-cite highwire-cite-highwire-article highwire-citation-biorxiv-article-top clearfix has-author-tooltip" >

      <span class="biorxiv-article-type">
        New Results    </span>
  
        <h1 class="highwire-cite-title" id="page-title">Automated data-intensive forecasting of plant phenology throughout the United States</h1>  
      <div  class="highwire-cite-authors" ><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0"><span class="nlm-given-names">Shawn D.</span> <span class="nlm-surname">Taylor</span></span>, <span class="highwire-citation-author" data-delta="1"><span class="nlm-given-names">Ethan P.</span> <span class="nlm-surname">White</span></span></span></div>
  
      <div  class="highwire-cite-metadata" ><span  class="highwire-cite-metadata-doi highwire-cite-metadata"><span class="label">doi:</span> https://doi.org/10.1101/634568 </span></div>
  
  
  </div>
<div id="hw-article-author-popups-node939653" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Shawn D. Taylor </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-institution'>School of Natural Resources and Environment, University of Florida Gainesville</span>, FL, <span class='nlm-country'>United States</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Shawn%2BD.%2BTaylor%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Taylor%20SD&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AShawn%2BD.%2BTaylor%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:shawntaylor@weecology.org" class="" data-icon-position="" data-hide-link-title="0">shawntaylor@weecology.org</a></span></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Ethan P. White </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-institution'>Department of Wildlife Ecology and Conservation, University of Florida</span>, Gainesville, FL, <span class='nlm-country'>United States</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Ethan%2BP.%2BWhite%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=White%20EP&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AEthan%2BP.%2BWhite%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/10.1101/634568v3" class="panels-ajax-tab-tab" data-panel-name="biorxiv_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:939653" data-trigger="" data-url-enabled="1">Abstract</a><a href="/panels_ajax_tab/biorxiv_tab_art/node:939653/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/10.1101/634568v3.full-text" class="panels-ajax-tab-tab" data-panel-name="article_tab_full_text" data-target-id="highwire_article_tabs" data-entity-context="node:939653" data-trigger="full-text" data-url-enabled="1">Full Text</a><a href="/panels_ajax_tab/article_tab_full_text/node:939653/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/10.1101/634568v3.article-info" class="panels-ajax-tab-tab" data-panel-name="biorxiv_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:939653" data-trigger="article-info" data-url-enabled="1">Info/History</a><a href="/panels_ajax_tab/biorxiv_tab_info/node:939653/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/10.1101/634568v3.article-metrics" class="panels-ajax-tab-tab" data-panel-name="article_tab_metrics" data-target-id="highwire_article_tabs" data-entity-context="node:939653" data-trigger="article-metrics" data-url-enabled="1">Metrics</a><a href="/panels_ajax_tab/article_tab_metrics/node:939653/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/10.1101/634568v3.supplementary-material" class="panels-ajax-tab-tab" data-panel-name="biorxiv_tab_data" data-target-id="highwire_article_tabs" data-entity-context="node:939653" data-trigger="supplementary-material" data-url-enabled="1">Supplementary material</a><a href="/panels_ajax_tab/biorxiv_tab_data/node:939653/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/10.1101/634568v3.external-links" class="panels-ajax-tab-tab" data-panel-name="article_tab_data_code" data-target-id="highwire_article_tabs" data-entity-context="node:939653" data-trigger="external-links" data-url-enabled="1">Data/Code</a><a href="/panels_ajax_tab/article_tab_data_code/node:939653/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/10.1101/634568v3.full.pdf+html" class="panels-ajax-tab-tab" data-panel-name="biorxiv_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:939653" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i> Preview PDF</a><a href="/panels_ajax_tab/biorxiv_tab_pdf/node:939653/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="article_tab_full_text" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.biorxiv.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-article_tab_full_text"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-markup" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2 class="">Abstract</h2><p id="p-3">Phenology - the timing of cyclical and seasonal natural phenomena such as flowering and leaf out - is an integral part of ecological systems with impacts on human activities like environmental management, tourism, and agriculture. As a result, there are numerous potential applications for actionable predictions of when phenological events will occur. However, despite the availability of phenological data with large spatial, temporal, and taxonomic extents, and numerous phenology models, there has been no automated species-level forecasts of plant phenology. This is due in part to the challenges of building a system that integrates large volumes of climate observations and forecasts, uses that data to fit models and make predictions for large numbers of species, and consistently disseminates the results of these forecasts in interpretable ways. Here we describe a new near-term phenology forecasting system that makes predictions for the timing of budburst, flowers, ripe fruit, and fall colors for 78 species across the United States up to 6 months in advance and is updated every four days. We use the lessons learned in developing this system to provide guidance developing large-scale near-term ecological forecast systems more generally, to help advance the use of automated forecasting in ecology.</p></div><div class="section" id="sec-1"><h2 class="">Introduction</h2><p id="p-6">Plant phenology - the timing of cyclical and seasonal natural phenomena such as flowering and leaf out - influences many aspects of ecological systems (<a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">Chuine and Régnière 2017</a>) from small scale community interactions (<a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">Ogilvie et al. 2017</a>) to global scale climate feedbacks (<a id="xref-ref-35-1" class="xref-bibr" href="#ref-35">Richardson et al. 2012</a>). Because of the central importance of phenology, advanced forecasts for when phenological events will occur have numerous potential applications including: 1) research on the cascading effects of changing plant phenology on other organisms; 2) tourism planning related to flower blooms and autumn colors; 3) planning for sampling and application of management interventions by researchers and managers; and 4) agricultural decisions on timing for planting, harvesting, and application of pest prevention techniques. However, due to the challenges of automatically integrating, predicting, and disseminating large volumes of data, there are limited examples of applied phenology forecast systems.</p><p id="p-7">Numerous phenology models have been developed to characterize the timing of major plant events and understand their drivers (<a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">Chuine et al. 2013</a>). These models are based on the idea that plant phenology is primarily driven by weather, with seasonal temperatures being the primary driver at temperate latitudes (<a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">Basler 2016</a>, <a id="xref-ref-8-2" class="xref-bibr" href="#ref-8">Chuine and Régnière 2017</a>). Because phenology is driven primarily by weather, it is possible to make predictions for the timing of phenology events based on forecasted weather conditions. The deployment of seasonal climate forecasts (<a id="xref-ref-46-1" class="xref-bibr" href="#ref-46">Weisheimer and Palmer 2014</a>), those beyond just a few weeks, provides the potential to forecast phenology months in advance. This time horizon is long enough to allow meaningful planning and action in response to these forecasts. With well established models, widely available data, and numerous use cases, plant phenology is well suited to serve as an exemplar for near-term ecological forecasting.</p><p id="p-8">For decision making purposes, the most informative plant phenology forecasts will predict the response of large numbers of species and phenophases, over large spatial extents, and at fine spatial resolutions. The only regularly updated phenology forecast in current operation predicts only a single aggregated “spring index” that identifies when early-spring phenological events occur at the level of the entire ecosystem (not individual species) at a resolution of 1° lat/lon grid cells (<a id="xref-ref-38-1" class="xref-bibr" href="#ref-38">Schwartz et al. 2013</a>, <a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">Carrillo et al. 2018</a>). Forecasting individual species and multiple phenological events at higher resolutions is challenging due to the advanced computational tools needed for building and maintaining data-intensive automatic forecasting systems (<a id="xref-ref-48-1" class="xref-bibr" href="#ref-48">White et al. 2018</a>, <a id="xref-ref-47-1" class="xref-bibr" href="#ref-47">Welch et al. 2019</a>). Automated forecasts requires building systems that acquire data, make model-based predictions for the future, and disseminate the forecasts to end-users, all in an automated pipeline (<a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">Dietze et al. 2018</a>, <a id="xref-ref-48-2" class="xref-bibr" href="#ref-48">White et al. 2018</a>, <a id="xref-ref-47-2" class="xref-bibr" href="#ref-47">Welch et al. 2019</a>). This is challenging even for relatively small-scale single site projects with one to several species or response variables due to the need for advanced computational tools to support robust automation (<a id="xref-ref-48-3" class="xref-bibr" href="#ref-48">White et al. 2018</a>, <a id="xref-ref-47-3" class="xref-bibr" href="#ref-47">Welch et al. 2019</a>). Building an automated system to forecast phenology for numerous species at continental scales is even more challenging due to the large-scale data intensive nature of the analyses. Specifically, because phenology is sensitive to local climate conditions, phenology modeling and prediction should be done at high resolutions (<a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">Cook et al. 2010</a>). This requires repeatedly conducting computationally intensive downscaling of seasonal climate forecasts and making large numbers of predictions. To make 4 km resolution spatially explicit forecasts for the 78 species in our study at continental scales requires over 90 million predictions for each updated forecast. To make the forecasts actionable these computational intensive steps need to be repeated in near real-time and disseminated in a way that allows end-users to understand the forecasts and their uncertainties (<a id="xref-ref-16-2" class="xref-bibr" href="#ref-16">Dietze et al. 2018</a>).</p><p id="p-9">Here we describe an automated near-term phenology forecast system we developed to make continental scale forecasts for 78 different plant species. Starting December 1st, and updated every 4 days, this system uses the latest climate information to make forecasts for multiple phenophases and presents the resulting forecasts and their uncertainty on a dynamic website, <a href="https://phenology.naturecast.org/">https://phenology.naturecast.org/</a>. Since the majority of plants complete budburst and/or flowering by the summer solstice in mid-June, this results in lead times of up to six months. We describe the key steps in the system construction, including: 1) fitting phenology models, 2) acquiring and downscaling climate data; 3) making predictions for phenological events; 4) disseminating those predictions; and 5) automating steps 2-4 to update forecasts at a sub-weekly frequency. We follow <a id="xref-ref-47-4" class="xref-bibr" href="#ref-47">Welch et al. (2019</a>)’s framework for describing operationalized dynamic management tools (ie. self-contained tools running automatically and regularly) and describe the major design decisions and lessons learned from implementing this system that will guide improvements to automated ecological forecasting systems. Due to the data-intensive nature of forecasting phenology at fine resolutions over large scales this system serves as a model for large-scale forecasting systems in ecology more broadly.</p><div id="sec-2" class="subsection"><h3>Forecasting Pipeline</h3><p id="p-10"><a id="xref-ref-47-5" class="xref-bibr" href="#ref-47">Welch et al. (2019)</a> break down the process of developing tools for automated prediction into four stages: 1) Acquisition, obtaining and processing the regularly updated data needed for prediction; 2) Prediction, combining the data with models to estimate the outcome of interest; 3) Dissemination, the public presentation of the predictions; and 4) Automation, the tools and approaches used to automatically update the predictions using the newest data on a regular basis. We start by describing our approach to modeling phenology and then describe our approach to each of these stages.</p></div><div id="sec-3" class="subsection"><h3>Phenology Modeling</h3><p id="p-11">Making large spatial scale phenology forecasts for a specific species requires species level observation data from as much of its respective range as possible (<a id="xref-ref-41-1" class="xref-bibr" href="#ref-41">Taylor et al. 2019</a>). We used data from the USA National Phenology Network (USA-NPN), which collects volunteer based data on phenological events and has amassed over 10 million observations representing over 1000 species. The USA-NPN protocol uses status-based monitoring, where observers answer ‘yes,’ ‘no,’ or ‘unsure’ when asked if an individual plant has a specific phenophase present (<a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">Denny et al. 2014</a>). Phenophases refer to specific phases in the annual cycle of a plant, such as the presence of emerging leaves, flowers, fruit, or senescing leaves. We used the “Individual Phenometrics” data product, which provides pre-processed onset dates of individually monitored plants, for the phenophases budburst, flowering, and fall colors for all species with data between 2009 and 2017 (<a id="xref-ref-43-1" class="xref-bibr" href="#ref-43">USA National Phenology Network 2018</a>). We only kept “yes” observations where the individual plant also had a “no” observation within the prior 30 days and dropped any records where a single plant had conflicting records for phenotype status or more than one series of “yes” observations for a phenophase in a 12 month period. We built models for species and phenophase combinations with at least 30 observations (<a id="xref-fig-1-1" class="xref-fig" href="#F1">Figure 1, B</a>) using daily mean temperature data at the location and time of each observation from the PRISM 4km dataset (<a id="xref-ref-32-1" class="xref-bibr" href="#ref-32">PRISM Climate Group 2004</a>). We also included contributed models of budburst, flowering, and/or fruiting for 5 species which were not well represented in the USA-NPN dataset (see Appendix S1: Table S2; Janet S. Prevéy, unpublished data, 2018, Prevéy et al. (In revision); <a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">Biederman et al. (2018)</a>).</p><div id="F1" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/10/07/634568/F1.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Flowchart of initial model building and automated pipeline steps. Letters indicate the associate steps discussed in the main text." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1542700935" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Flowchart of initial model building and automated pipeline steps. Letters indicate the associate steps discussed in the main text.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 1:" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2019/10/07/634568/F1.medium.gif" width="440" height="356"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 1:" src="https://www.biorxiv.org/content/biorxiv/early/2019/10/07/634568/F1.medium.gif" width="440" height="356"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/10/07/634568/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1:" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/10/07/634568/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml"><span class="fig-label">Figure 1:</span> <p id="p-12" class="first-child">Flowchart of initial model building and automated pipeline steps. Letters indicate the associate steps discussed in the main text.</p><div class="sb-div caption-clear"></div></div></div><p id="p-13">For each species and phenophase we fit an ensemble of four models using daily mean temperature as the sole driver (<a id="xref-fig-1-2" class="xref-fig" href="#F1">Figure 1, C</a>). The general model form assumes a phenological event will occur once sufficient thermal forcing units accumulate from a specified start day (<a id="xref-ref-9-2" class="xref-bibr" href="#ref-9">Chuine et al. 2013</a>, <a id="xref-ref-8-3" class="xref-bibr" href="#ref-8">Chuine and Régnière 2017</a>). The specification of forcing units are model specific, but all are derived from the 24-hour daily mean temperature. In a basic model a forcing unit is the maximum of either 0 or the mean temperature above 0°C (ie. growing degree days). The amount of forcing units required, and the date from which they start accumulating are parameterized for each species and phenophase (see Appendix S1: Table S1). Ensembles of multiple models generally improve prediction over any single model by reducing bias and variance, and in a phenology context allow more accurate predictions to be made without knowing the specific physiological processes for each species (<a id="xref-ref-1-2" class="xref-bibr" href="#ref-1">Basler 2016</a>, <a id="xref-ref-52-1" class="xref-bibr" href="#ref-52">Yun et al. 2017</a>, <a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">Dormann et al. 2018</a>). We used a weighted ensemble of four phenology models. We derived the weights for each model within the ensemble using stacking to minimize the root mean squared error on held out test data (100 fold cross-validation) as described in <a id="xref-ref-17-2" class="xref-bibr" href="#ref-17">Dormann et al. (2018)</a> (see Appendix S1: Sec. S1). After determining the weights we fit the core models a final time on the full dataset. Since individual process based phenology models are not probabilistic they do not allow the estimation of uncertainty in the forecasts. Therefore, we used the variance across the five climate models to represent uncertainty (see Prediction). Finally, we also fit a spatially corrected Long Term Average model for use in calculating anomalies (see Dissemination). This uses the past observations in a linear model with latitude as the sole predictor (see Appendix S1: Table S1).</p><p id="p-14">In our pipeline 190 unique phenological models (one for each species and phenophase combination, see see Appendix S1: Table S2) needed to be individually parameterized, evaluated, and stored for future use. To consolidate all these requirements we built a dedicated software package written in Python, pyPhenology, to build, save, and load models, and also apply them to gridded climate datasets (<a id="xref-ref-40-1" class="xref-bibr" href="#ref-40">Taylor 2018</a>). The package also integrates the phenological model ensemble so that the four sub-models can be treated seamlessly as one in the pipeline. After parameterizing each model, its specifications are saved in a text based JSON file that is stored in a git repository along with a metadata file describing all models (<a id="xref-fig-1-3" class="xref-fig" href="#F1">Figure 1, D</a>). This approach allows for the tracking and usage of hundreds of models, allowing models to be easily synchronized across systems, and tracking versions of models as they are updated (or even deleted).</p></div><div id="sec-4" class="subsection"><h3>Acquisition and Downscaling of Climate Data</h3><p id="p-15">Since our phenology models are based on accumulated temperature forcing, making forecasts requires information on both observed temperatures (from Nov. 30 of the prior year up to the date a forecast is made) and forecast temperatures (from the forecast date onward). For observed data we used 4km 24-hour daily mean temperature from PRISM, a gridded climate dataset for the continental U.S.A. which interpolates on the ground measurements and is updated daily (<a id="xref-ref-32-2" class="xref-bibr" href="#ref-32">PRISM Climate Group 2004</a>). These observed data are saved in a netCDF file, which is appended with the most recent data every time the automated forecast is run. For climate forecasts we used the Climate Forecast System Version 2 (CFSv2; a coupled atmosphere-ocean-land global circulation model) 2-m temperature data, which has a 6-hour timestep and a spatial resolution of 0.25 degrees latitude/longitude (<a id="xref-ref-37-1" class="xref-bibr" href="#ref-37">Saha et al. 2014</a>). CFSv2 forecasts are projected out 9 months from the issue date and are updated every 6 hours. The five most recent climate forecasts are downloaded for each updated phenology forecast to accommodate uncertainty (see Prediction).</p><p id="p-16">Because the gridded climate forecasts are issued at large spatial resolutions (0.25 degrees), this data requires downscaling to be used at ecologically relevant scales (<a id="xref-ref-11-2" class="xref-bibr" href="#ref-11">Cook et al. 2010</a>). A downscaling model relates observed values at the smaller scale to the larger scale values generated by the climate forecast during a past time period. We regressed these past conditions from a climate reanalysis of CFSv2 from 1995-2015 (<a id="xref-ref-36-1" class="xref-bibr" href="#ref-36">Saha et al. 2010</a>) against the 4km daily mean temperature from the PRISM dataset for the same time period (<a id="xref-ref-32-3" class="xref-bibr" href="#ref-32">PRISM Climate Group 2004</a>) to build a downscaling model using asynchronous regression (<a id="xref-fig-1-4" class="xref-fig" href="#F1">Figure 1, E-G</a>). The CFSv2 data is first interpolated from the original 0.25 degree grid to a 4km grid using distance weighted sampling, then an asynchronous regression model is applied to each 4km pixel and calendar month (<a id="xref-ref-39-1" class="xref-bibr" href="#ref-39">Stoner et al. 2013</a>, see see Appendix S1: Sec. S2). The two parameters from the regression model for each 4 km cell are saved in a netCFD file by location and calendar month (<a id="xref-fig-1-5" class="xref-fig" href="#F1">Figure 1, H</a>). This downscaling model, at the scale of the continental U.S.A., is used to downscale the most recent CFSv2 forecasts to a 4km resolution during the automated steps.</p><p id="p-17">We used specialized Python packages to overcome the computational challenges inherent in the large CFSv2 climate dataset (<a id="xref-ref-33-1" class="xref-bibr" href="#ref-33">Python Software Foundation 2003</a>). The climate forecast data for each phenology forecast update is 10-40 gigabytes, depending on the time of year (time series are longer later in the year). While it is possible to obtain hardware capable of loading this dataset into memory, a more efficient approach is to perform the downscaling and phenology model operations iteratively by subsetting the climate dataset spatially and performing operations on one chunk at a time. We used the python package xarray (<a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">Hoyer and Hamman 2017</a>), which allows these operations to be efficiently performed in parallel through tight integration with the dask package (<a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">Dask Development Team 2016</a>). The combination of dask and xarray allows the analysis to be run on individual workstations, stand alone servers, and high performance computing systems, and to easily scale to more predictors and higher resolution data.</p></div><div id="sec-5" class="subsection"><h3>Prediction</h3><p id="p-18">The five most recent downscaled climate forecasts are each combined with climate observations to make a five member ensemble of daily mean temperature across the continental USA (<a id="xref-fig-1-6" class="xref-fig" href="#F1">Figure 1, L</a>). These are used to make predictions using the phenology model for each species and phenophase (<a id="xref-fig-1-7" class="xref-fig" href="#F1">Figure 1, M</a>). Each climate ensemble member is a 3d matrix of latitude <em>×</em> longitude <em>×</em> time at daily timesteps extending from Nov. 1 of the prior year to 9 months past the issue date. The pyPhenology package uses this object to make predictions for every 4 km grid cell in the contiguous United States, producing a 2d matrix (latitude <em>×</em> longitude) where each cell represents the predicted Julian day of the phenological event. This results in approximately half a million predictions for each run of each phenology model and 90 million predictions per run of the forecasting pipeline. The output of each model is cropped to the range of the respective species (<a id="xref-ref-42-1" class="xref-bibr" href="#ref-42">US Geological Survey 1999</a>) and saved as a netCDF file (<a id="xref-fig-1-8" class="xref-fig" href="#F1">Figure 1, N</a>) for use in dissemination and later evaluation.</p><p id="p-19">An important aspect of making actionable forecasts is providing decision makers with information on the uncertainty of those predictions (<a id="xref-ref-16-3" class="xref-bibr" href="#ref-16">Dietze et al. 2018</a>). One major component of uncertainty that is often ignored in near-term ecological forecasting studies is the uncertainty in the forecasted drivers. We incorporate information on uncertainty in temperature, the only driver in our phenology models, using the CFSv2 climate ensemble (<a id="xref-fig-1-9" class="xref-fig" href="#F1">Figure 1, I</a>; see Acquisition). The members of the climate ensemble each produce a different temperature forecast due to differences in initial conditions (<a id="xref-ref-46-2" class="xref-bibr" href="#ref-46">Weisheimer and Palmer 2014</a>). For each of the five climate members we make a prediction using the phenology ensemble, and the uncertainty is estimated as the variance of these predictions (see see Appendix S1: Sec. S1). This allows us to present the uncertainty associated with climate, along with a point estimate of the forecast, resulting in a range of dates over which a phenological event is likely to occur.</p></div><div id="sec-6" class="subsection"><h3>Dissemination</h3><p id="p-20">To disseminate the forecasts we built a website that displays maps of the predictions for each unique species and phenophase (<a href="https://phenology.naturecast.org/">https://phenology.naturecast.org/</a>; <a id="xref-fig-1-10" class="xref-fig" href="#F1">Figure 1 Q</a>; <a id="xref-fig-2-1" class="xref-fig" href="#F2">Figure 2</a>). We used the Django web framework and custom JavaScript to allow the user to select forecasts by species, phenophase, and issue date (<a id="xref-fig-2-2" class="xref-fig" href="#F2">Figure 2D</a>). The main map shows the best estimate for when the phenological event will occur for the selected species (<a id="xref-fig-2-3" class="xref-fig" href="#F2">Figure 2A</a>). Actionable forecasts also require an understanding of how much uncertainty is present in the prediction (<a id="xref-ref-16-4" class="xref-bibr" href="#ref-16">Dietze et al. 2018</a>), because knowing the expected date of an annual event such as flowering isn’t particularly useful if the confidence interval stretches over several months. Therefore we also display a map of uncertainty quantified as the 95% prediction interval, the range of days within which the phenology event is expected to fall 95% of the time (<a id="xref-fig-2-4" class="xref-fig" href="#F2">Figure 2C</a>). Finally, to provide context to the current years predictions, we also map the predicted anomaly (<a id="xref-fig-2-5" class="xref-fig" href="#F2">Figure 2B</a>). The anomaly is the difference between the predicted date and the long term, spatially corrected average date of the phenological event (<a id="xref-fig-1-11" class="xref-fig" href="#F1">Figure 1, O</a>; see see Appendix S1: Table S1).</p><div id="F2" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/10/07/634568/F2.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Screenshot of the forecast presentation website (http://phenology.naturecast.org) showing the forecast for the leaf out of Acer saccharinum in Spring, 2019, issued on Feburary 21, 2019. The maps represent the predicted date of leaf out (A), the anomaly compared to prior years (B), and the 95% confidence interval (C). In the upper right is the interface for selecting different species, phenophases, or forecast issue dates via drop down menus (D)." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1542700935" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Screenshot of the forecast presentation website (http://phenology.naturecast.org) showing the forecast for the leaf out of &lt;em&gt;Acer saccharinum&lt;/em&gt; in Spring, 2019, issued on Feburary 21, 2019. The maps represent the predicted date of leaf out (A), the anomaly compared to prior years (B), and the 95% confidence interval (C). In the upper right is the interface for selecting different species, phenophases, or forecast issue dates via drop down menus (D).&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 2:" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2019/10/07/634568/F2.medium.gif" width="380" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 2:" src="https://www.biorxiv.org/content/biorxiv/early/2019/10/07/634568/F2.medium.gif" width="380" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/10/07/634568/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2:" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/10/07/634568/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 2:</span> <p id="p-21" class="first-child">Screenshot of the forecast presentation website (<a href="http://phenology.naturecast.org">http://phenology.naturecast.org</a>) showing the forecast for the leaf out of <em>Acer saccharinum</em> in Spring, 2019, issued on Feburary 21, 2019. The maps represent the predicted date of leaf out (A), the anomaly compared to prior years (B), and the 95% confidence interval (C). In the upper right is the interface for selecting different species, phenophases, or forecast issue dates via drop down menus (D).</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-7" class="subsection"><h3>Automation</h3><p id="p-22">All of the steps in this pipeline, other than phenology and downscaling model fitting, are automatically run every 4 days. To do this we use a cron job running on a local server. Cron jobs automatically rerun code on set intervals. The cron job initiates a python script which runs the major steps in the pipeline. First the latest CFSv2 climate forecasts are acquired, downscaled, and combined with the latest PRISM climate observations (<a id="xref-fig-1-12" class="xref-fig" href="#F1">Figure 1, I-L</a>). This data is then combined with the phenology models using the pyPhenology package to make predictions for the timing of phenological events (<a id="xref-fig-1-13" class="xref-fig" href="#F1">Figure 1, M-N</a>). These forecasts are then converted into maps and uploaded to the website (<a id="xref-fig-1-14" class="xref-fig" href="#F1">Figure 1, O-Q</a>). To ensure that forecasts continue to run even when unexpected events occur it is necessary to develop pipelines that are robust to unexpected errors and missing data, and are also informative when failures inevitably do happen (<a id="xref-ref-47-6" class="xref-bibr" href="#ref-47">Welch et al. 2019</a>). We used status checks and logging to identify and fix problems and separated the website infrastructure from the rest of the pipeline. Data are checked during acquisition to determine if there are data problems and when possible alternate data is used to replace data with issues. For example, members of the CFSv2 ensemble sometimes have insufficient time series lengths. When this is the case that forecast is discarded and a preceding climate forecast obtained. With this setup occasional errors in upstream data can be ignored, and larger problems identified and corrected with minimal downtime. To prevent larger problems from preventing access to the most recent successful forecasts the website is only updated if all other steps run successfully. This ensures that user of the website can always access the latest forecasts.</p><p id="p-23">Software packages used throughout the system include, for the R language, ggplot2 (<a id="xref-ref-49-1" class="xref-bibr" href="#ref-49">Wickham 2016</a>), raster (<a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">Hijmans 2017</a>), prism (<a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">Hart and Bell 2015</a>), sp (<a id="xref-ref-28-1" class="xref-bibr" href="#ref-28">Pebesma and Bivand 2005</a>), tidyr (<a id="xref-ref-50-1" class="xref-bibr" href="#ref-50">Wickham and Henry 2018</a>), lubridate (<a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">Grolemund and Wickham 2011</a>), and ncdf4 (<a id="xref-ref-30-1" class="xref-bibr" href="#ref-30">Pierce 2017</a>). From the python language we also utilized xarray (<a id="xref-ref-21-2" class="xref-bibr" href="#ref-21">Hoyer and Hamman 2017</a>), dask, (<a id="xref-ref-13-2" class="xref-bibr" href="#ref-13">Dask Development Team 2016</a>), scipy (<a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">Jones et al. 2001</a>), numpy (<a id="xref-ref-27-1" class="xref-bibr" href="#ref-27">Oliphant 2006</a>), pandas (<a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">McKinney 2010</a>), and mpi4py (<a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">Dalcin et al. 2011</a>). All code described is available on a GitHub repository (<a href="https://github.com/sdtaylor/phenology_forecasts">https://github.com/sdtaylor/phenology_forecasts</a>). The code as well as 2019 forecasts and observations (see Evaluation) are also permanently archived on Zenodo (<a href="https://doi.org/10.5281/zenodo.2577452">https://doi.org/10.5281/zenodo.2577452</a>).</p></div><div id="sec-8" class="subsection"><h3>Evaluation</h3><p id="p-24">A primary advantage of near-term forecasts is the ability to rapidly evaluate forecast proficiency, thereby shortening the model development cycle (<a id="xref-ref-16-5" class="xref-bibr" href="#ref-16">Dietze et al. 2018</a>). Phenological events happen throughout the growing season, providing a consistent stream of new observations to assess. We evaluated our forecasts (made from Dec. 1, 2018 thru May 1, 2019) using observations from the USA-NPN from Jan. 1, 2019 through May 8, 2019 and subset to species and phenophases represented in our system (<a id="xref-fig-3-1" class="xref-fig" href="#F3">Figure 3</a>; <a id="xref-ref-44-1" class="xref-bibr" href="#ref-44">USA National Phenology Network (2019)</a>). This resulted in 1581 phenological events that our system had forecasts for (588 flowering events, 991 budburst events, and 2 fall coloring across 65 species, see see Appendix S1: Table S3). For each forecast issue date we calculated the root mean square error (RMSE) and average forecast uncertainty for all events and all prior issue dates. We also assessed the distribution of absolute errors <span class="inline-formula" id="inline-formula-1"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2019/10/07/634568/embed/inline-graphic-1.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2019/10/07/634568/embed/inline-graphic-1.gif"/></noscript></span></span> for a subset of issue dates (approximately two a month).</p><div id="F3" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/10/07/634568/F3.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Locations of phenological events which have occurred between Jan. 1, 2019 and May 5, 2019 obtained from the USA National Phenology Network (blue circles), and all sampling locations in the same dataset (red points). Four individual plants are highlighted, with numbers indicating the USA National Phenology Network database ID. The solid line indicates the predicted event date as well as the 95% confidence interval for a specified forecast issue date, and the dashed line indicates the observed event date. The x-axis corresponds to the date a forecast was issued, while the y-axis is the date flowering or budburst was predicted to occur. For example: on Jan. 1, 2019 the P. tremuloides plant was forecast to flower sometime between March, 29 and April, 24 (solid lines). The actual flowering date was March 18 (dashed line)." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1542700935" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Locations of phenological events which have occurred between Jan. 1, 2019 and May 5, 2019 obtained from the USA National Phenology Network (blue circles), and all sampling locations in the same dataset (red points). Four individual plants are highlighted, with numbers indicating the USA National Phenology Network database ID. The solid line indicates the predicted event date as well as the 95% confidence interval for a specified forecast issue date, and the dashed line indicates the observed event date. The x-axis corresponds to the date a forecast was issued, while the y-axis is the date flowering or budburst was predicted to occur. For example: on Jan. 1, 2019 the &lt;em&gt;P. tremuloides&lt;/em&gt; plant was forecast to flower sometime between March, 29 and April, 24 (solid lines). The actual flowering date was March 18 (dashed line).&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 3:" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2019/10/07/634568/F3.medium.gif" width="440" height="163"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 3:" src="https://www.biorxiv.org/content/biorxiv/early/2019/10/07/634568/F3.medium.gif" width="440" height="163"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/10/07/634568/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 3:" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/10/07/634568/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 3:</span> <p id="p-25" class="first-child">Locations of phenological events which have occurred between Jan. 1, 2019 and May 5, 2019 obtained from the USA National Phenology Network (blue circles), and all sampling locations in the same dataset (red points). Four individual plants are highlighted, with numbers indicating the USA National Phenology Network database ID. The solid line indicates the predicted event date as well as the 95% confidence interval for a specified forecast issue date, and the dashed line indicates the observed event date. The x-axis corresponds to the date a forecast was issued, while the y-axis is the date flowering or budburst was predicted to occur. For example: on Jan. 1, 2019 the <em>P. tremuloides</em> plant was forecast to flower sometime between March, 29 and April, 24 (solid lines). The actual flowering date was March 18 (dashed line).</p><div class="sb-div caption-clear"></div></div></div><p id="p-26">Forecast RMSE and uncertainty both decreased for forecasts with shorter lead time (i.e. closer to the date the phenological event occurred), also known as the forecast horizon (<a id="xref-fig-4-1" class="xref-fig" href="#F4">Fig. 4</a>; <a id="xref-ref-29-1" class="xref-bibr" href="#ref-29">Petchey et al. (2015)</a>). Forecasts issued at the start of the year (on Jan. 5, 2019) had a RMSE of 20.9 days, while the most recent forecasts (on May 5, 2019) had an RMSE of only 18.8 days. The average uncertainty for the forecasts were 7.6 and 0.2 days respectively for Jan. 5, and May 5. Errors were normally distributed with a small over-prediction bias (MAE values of 6.8 - 12.1, <a id="xref-fig-5-1" class="xref-fig" href="#F5">Fig. 5</a>). This bias also decreased as spring progressed. These results indicate a generally well performing model, but also one with significant room for improvement that will be facilitated by the iterative nature of the forecasting system.</p><div id="F4" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/10/07/634568/F4.large.jpg?width=800&amp;height=600&amp;carousel=1" title="The root mean square error and the average uncertainty of forecasts issued between Dec. 2, 2018 and May 5, 2019 for 1581 phenological events representing 65 species." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1542700935" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;The root mean square error and the average uncertainty of forecasts issued between Dec. 2, 2018 and May 5, 2019 for 1581 phenological events representing 65 species.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 4:" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2019/10/07/634568/F4.medium.gif" width="440" height="169"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 4:" src="https://www.biorxiv.org/content/biorxiv/early/2019/10/07/634568/F4.medium.gif" width="440" height="169"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/10/07/634568/F4.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 4:" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/10/07/634568/F4.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 4:</span> <p id="p-27" class="first-child">The root mean square error and the average uncertainty of forecasts issued between Dec. 2, 2018 and May 5, 2019 for 1581 phenological events representing 65 species.</p><div class="sb-div caption-clear"></div></div></div><div id="F5" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/10/07/634568/F5.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Distribution of absolute errors (prediction - observed) for 1581 phenological events for 11 selected issue dates. Labels indicate the mean absolute error (MAE)." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1542700935" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Distribution of absolute errors (prediction - observed) for 1581 phenological events for 11 selected issue dates. Labels indicate the mean absolute error (MAE).&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 5:" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2019/10/07/634568/F5.medium.gif" width="440" height="172"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 5:" src="https://www.biorxiv.org/content/biorxiv/early/2019/10/07/634568/F5.medium.gif" width="440" height="172"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/10/07/634568/F5.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 5:" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/10/07/634568/F5.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 5:</span> <p id="p-28" class="first-child">Distribution of absolute errors (prediction - observed) for 1581 phenological events for 11 selected issue dates. Labels indicate the mean absolute error (MAE).</p><div class="sb-div caption-clear"></div></div></div></div></div><div class="section" id="sec-9"><h2 class="">Discussion</h2><p id="p-29">We created an automated plant phenology forecasting system that makes forecasts for 78 species and 4 different phenophases across the entire contiguous United States. Forecasts are updated every four days with the most recent climate observations and forecasts, converted to static maps, and uploaded to a website for dissemination. We used only open source software and data formats, and free publicly available data. While a more comprehensive evaluation of forecast performance is outside the scope of this paper, we note that the majority of forecasts provide realistic phenology estimates across known latitudinal and elevational gradients (<a id="xref-fig-2-6" class="xref-fig" href="#F2">Figure 2</a>), and forecast uncertainty and error decreases as spring progresses (<a id="xref-fig-4-2" class="xref-fig" href="#F4">Figure 4</a>). While there is a bias from over-estimating phenological events, estimates were on-average within 2-3 weeks of the true dates throughout the spring season.</p><p id="p-30">Developing automated forecasting systems in ecology is important both for providing decision makers with near real-time predictions and for improving our understanding of biological systems by allowing repeated tests of, and improvements to, ecological models (<a id="xref-ref-16-6" class="xref-bibr" href="#ref-16">Dietze et al. 2018</a>, <a id="xref-ref-48-4" class="xref-bibr" href="#ref-48">White et al. 2018</a>, <a id="xref-ref-47-7" class="xref-bibr" href="#ref-47">Welch et al. 2019</a>). To facilitate the development of ecological forecasts, we need both active development, descriptions, and discussion of a variety of forecasting systems. These discussions of the tools, philosophies, and challenges involved in forecast pipeline development will advance our understanding of how to most effectively build the systems, thereby lowering the entry barrier of operationalizing ecological models for decision making. Active development and discussion will also help us identify generalizable problems which can be solved with standardized methods, data formats, and software packages. Tools such as this can be used to more efficiently implement new ecological forecast systems, and facilitate synthetic analyses and comparisons across a variety of forecasts.</p><p id="p-31">Automated forecasting systems typically involve multiple major steps in a combined pipeline. We found that breaking the pipeline into modular chunks made maintaining this large number of components more manageable (<a id="xref-ref-48-5" class="xref-bibr" href="#ref-48">White et al. 2018</a>, <a id="xref-ref-47-8" class="xref-bibr" href="#ref-47">Welch et al. 2019</a>). For generalizable pieces of the pipeline we found that turning them into software packages eased maintenance by decoupling dependencies and allowing independent testing. Packaging large components also makes it easier for others to use code developed for a forecasting system. The phenology modelling packge, pyPhenology, was developed for the current system, but is generalized for use in any phenological modelling study (<a id="xref-ref-40-2" class="xref-bibr" href="#ref-40">Taylor 2018</a>). We also found it useful to use different languages for different pieces of the pipeline. Our pipeline involved tasks ranging from automatically processing gigabytes of climate data to visualizing results to disseminating those results through a dynamic website. In such a pipeline no single language will fit all requirements, thus we made use of the strengths of two languages (Python and R) and their associate package ecosystems. Interoperability is facilitated by common data formats (csv and netCDF files), allowing scripts written in one language to communicate results to the next step in the pipeline written in another language.</p><p id="p-32">This phenology forecasting system currently involves 190 different ensemble models, one for each species and phenological stage, each composed of 4 different phenology sub-models and their associated weights for a total of 760 different models. This necessitates having a system for storing and documenting models, and subsequently updating them with new data and/or methods over time. We stored the fitted models in JSON files (a open-standard text format). We used the version control system git to track changes to these text based model specifications. While git was originally designed tracking changes to code, it can also be leveraged for tracking data of many forms, including our model specifications (<a id="xref-ref-34-1" class="xref-bibr" href="#ref-34">Ram 2013</a>, <a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">Bryan 2018</a>, <a id="xref-ref-51-1" class="xref-bibr" href="#ref-51">Yenni et al. 2019</a>). Managing many different models, including different versions of those models and their associate provenance, will likely be a common challenge for ecological forecasting (<a id="xref-ref-48-6" class="xref-bibr" href="#ref-48">White et al. 2018</a>) as one of the goals is iteratively improving the models.</p><p id="p-33">The initial development of this system has highlighted several potential areas for improvement. First, the data-intensive nature of this forecasting system provides challenges and opportunities for disseminating results. Currently static maps show the forecast dates of phenological events across each species respective range. However this only answers one set of questions and makes it difficult for others to build on the forecasts. Additional user interface design, including interactive maps and the potential to view forecasts for a single location, would make it easier to ask other types of questions such as “Which species will be in bloom on this date in a particular location?”. User interface design is vital for successful dissemination, and tools such the python package Django used here, or the R packages Shiny and Rmarkdown provide flexible frameworks for implementation (<a id="xref-ref-48-7" class="xref-bibr" href="#ref-48">White et al. 2018</a>, <a id="xref-ref-47-9" class="xref-bibr" href="#ref-47">Welch et al. 2019</a>). In addition it would be useful to provide access to the raw data underlying each forecast. The sheer number of forecasts makes the bi-weekly forecast data relatively large, presenting some challenges for dissemination through traditional ecological archiving services like Dryad (<a href="https://datadryad.org">https://datadryad.org</a>) and Zenodo (<a href="https://zenodo.org">https://zenodo.org</a>). If stored as csv files every forecast would have generated 15 GB of data. We addressed this by storing the forecasts in compressed netCDF files, which are optimized for large-scale mutli-dimensional data and in our case are 300 times smaller than the csv files (50 MB/forecast).</p><p id="p-34">In addition to areas for improvement in the forecasting system itself, its development has highlighted areas for potential improvement in phenology modeling. Other well-known phenological drivers could be incorporated into the models, such as precipitation and daylength. Precipitation forecasts are available from the CFSv2 dataset, though their accuracy is considerably lower than temperature forecasts (<a id="xref-ref-37-2" class="xref-bibr" href="#ref-37">Saha et al. 2014</a>). Other large-scale phenological datasets, such as remotely-sensed spring greenup could be used to constrain the species level forecasts made here (<a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">Melaas et al. 2016</a>). Our system does not currently integrate observations about how phenology is progressing within a year to update the models. USA-NPN data are available in near real-time after they are submitted by volunteers, thus there is opportunity for data assimilation of phenology observations. Making new forecasts with the latest information not only on the current state of the climate, but also on the current state of the plants themselves would likely be very informative (<a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">Luo et al. 2011</a>, <a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">Dietze 2017</a>). For example, if a species is leafing out sooner than expected in one area it is likely that it will also leaf out sooner than expected in nearby regions. This type of data assimilation is important for making accurate forecasts in other disciplines including meteorology (<a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">Bauer et al. 2015</a>, <a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">Carrassi et al. 2018</a>). However, process based plant phenology models were not designed with data assimilation in mind (<a id="xref-ref-9-3" class="xref-bibr" href="#ref-9">Chuine et al. 2013</a>). <a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">Clark et al. (2014)</a> built a bayesian hierarchical phenology model of budburst which incorporates the discrete observations of phenology data. This could serve as a starting point for a phenology forecasting model that incorporates data assimilation and allows species with relatively few observations to borrow strength from species with a large number of observations. The model from <a id="xref-ref-10-2" class="xref-bibr" href="#ref-10">Clark et al. (2014)</a> also incorporates all stages of the bud development process into a continuous latent state, thus there is also potential for forecasting the current phenological state of plants, instead of just the transition dates as is currently done in this forecast system.</p><p id="p-35">Using recent advances in open source software and large-scale open data collection we have implemented an automated high resolution, continental scale, species-level phenology forecast system. Implementing a system of this scale was made possible by a new phenology data stream and new computational tools that facilitate large scale analysis with limited computing and human resources. Most recent research papers describing ecological forecast systems focus on only the modelling aspect (<a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">Chen et al. 2011</a>, <a id="xref-ref-6-2" class="xref-bibr" href="#ref-6">Carrillo et al. 2018</a>, <a id="xref-ref-45-1" class="xref-bibr" href="#ref-45">Van Doren and Horton 2018</a>), and studies outlining implementation methods and best practices are lacking (but see <a id="xref-ref-48-8" class="xref-bibr" href="#ref-48">White et al. 2018</a>, <a id="xref-ref-47-10" class="xref-bibr" href="#ref-47">Welch et al. 2019</a>). Making a forecast system operational is key to producing applied tools, and requires a significant investment in time and other resources for data logistics and pipeline development. Major challenges here included the automated processing of large meteorological datasets, efficient application of hundreds of phenological models, and stable, consistently updated, and easy to understand dissemination of forecasts. By discussing how we addressed these challenges, and making our code publicly available, we hope to provide guidance for others developing ecological forecasting systems.</p></div><div class="section ack" id="ack-1"><h2 class="">Acknowledgments</h2><p id="p-36">This research was supported by the Gordon and Betty Moore Foundation’s Data-Driven Discovery Initiative through Grant GBMF4563 to E.P. White. We thank the USA National Phenology Network and the many participants who contribute to its Nature’s Notebook program.</p></div><div class="section fn-group" id="fn-group-1"><h2>Footnotes</h2><ul><li class="fn-other" id="fn-1"><p id="p-1"><span class="em-link"><span class="em-addr">ethanwhite{at}ufl.edu</span></span></p></li><li class="fn-update fn-group-summary-of-updates" id="fn-2"><p id="p-4">Corrected Figure 4 as RMSE was not being calculated correctly.</p></li><li class="fn-dataset fn-group-external-links" id="fn-3"><p id="p-5">
<a href="https://doi.org/10.5281/zenodo.2577452">https://doi.org/10.5281/zenodo.2577452</a>
</p></li></ul></div><div class="section ref-list" id="ref-list-1"><h2 class="">References</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference  in text" id="ref-1">↵</a><div class="cit ref-cit ref-journal" id="cit-634568v3.1"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Basler</span>,  <span class="cit-name-given-names">D.</span></span> <span class="cit-pub-date">2016</span>. <span class="cit-article-title">Evaluating phenological models for the prediction of leaf-out dates in six temperate tree species across central europe</span>. <abbr class="cit-jnl-abbrev">Agricultural and Forest Meteorology</abbr> <span class="cit-vol">217</span>:<span class="cit-fpage">10</span>–<span class="cit-lpage">21</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAgricultural%2Band%2BForest%2BMeteorology%26rft.volume%253D217%26rft.spage%253D10%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference  in text" id="ref-2">↵</a><div class="cit ref-cit ref-journal" id="cit-634568v3.2" data-doi="10.1038/nature14956"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Bauer</span>,  <span class="cit-name-given-names">P.</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Thorpe</span></span>, and <span class="cit-auth"> <span class="cit-name-given-names">G.</span> <span class="cit-name-surname">Brunet</span></span>. <span class="cit-pub-date">2015</span>. <span class="cit-article-title">The quiet revolution of numerical weather prediction</span>. <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">525</span>:<span class="cit-fpage">47</span>–<span class="cit-lpage">55</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.volume%253D525%26rft.spage%253D47%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature14956%26rft_id%253Dinfo%253Apmid%252F26333465%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature14956&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=26333465&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F10%2F07%2F634568.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference  in text" id="ref-3">↵</a><div class="cit ref-cit ref-journal" id="cit-634568v3.3"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Biederman</span>,  <span class="cit-name-given-names">L.</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">D.</span> <span class="cit-name-surname">Anderson</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">N.</span> <span class="cit-name-surname">Sather</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Pearson</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Beckman</span></span>, and <span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Prekker</span></span>. <span class="cit-pub-date">2018</span>. <span class="cit-article-title">Using phenological monitoring in situ and historical records to determine environmental triggers for emergence and anthesis in the rare orchid Platanthera praeclara Sheviak &amp; Bowles</span>. <abbr class="cit-jnl-abbrev">Global Ecology and Conservation</abbr> <span class="cit-vol">16</span>:<span class="cit-fpage">e00461</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGlobal%2BEcology%2Band%2BConservation%26rft.volume%253D16%26rft.spage%253D00461e%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference  in text" id="ref-4">↵</a><div class="cit ref-cit ref-journal" id="cit-634568v3.4"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Bryan</span>,  <span class="cit-name-given-names">J.</span></span> <span class="cit-pub-date">2018</span>. <span class="cit-article-title">Excuse me, do you have a moment to talk about version control?</span> <abbr class="cit-jnl-abbrev">The American Statistician</abbr> <span class="cit-vol">72</span>:<span class="cit-fpage">20</span>–<span class="cit-lpage">27</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DThe%2BAmerican%2BStatistician%26rft.volume%253D72%26rft.spage%253D20%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference  in text" id="ref-5">↵</a><div class="cit ref-cit ref-journal" id="cit-634568v3.5"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Carrassi</span>,  <span class="cit-name-given-names">A.</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Bocquet</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">L.</span> <span class="cit-name-surname">Bertino</span></span>, and <span class="cit-auth"> <span class="cit-name-given-names">G.</span> <span class="cit-name-surname">Evensen</span></span>. <span class="cit-pub-date">2018</span>. <span class="cit-article-title">Data assimilation in the geosciences: An overview of methods, issues, and perspectives</span>. <abbr class="cit-jnl-abbrev">Wiley Interdisciplinary Reviews: Climate Change</abbr> <span class="cit-vol">9</span>:<span class="cit-fpage">e535</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DWiley%2BInterdisciplinary%2BReviews%253A%2BClimate%2BChange%26rft.volume%253D9%26rft.spage%253D535e%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference  in text" id="ref-6">↵</a><div class="cit ref-cit ref-journal" id="cit-634568v3.6"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Carrillo</span>,  <span class="cit-name-given-names">C. M.</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">T. R.</span> <span class="cit-name-surname">Ault</span></span>, and <span class="cit-auth"> <span class="cit-name-given-names">D. S.</span> <span class="cit-name-surname">Wilks</span></span>. <span class="cit-pub-date">2018</span>. <span class="cit-article-title">Spring onset predictability in the north american multimodel ensemble</span>. <abbr class="cit-jnl-abbrev">Journal of Geophysical Research: Atmospheres</abbr> <span class="cit-vol">123</span>:<span class="cit-fpage">5913</span>–<span class="cit-lpage">5926</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BGeophysical%2BResearch%253A%2BAtmospheres%26rft.volume%253D123%26rft.spage%253D5913%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference  in text" id="ref-7">↵</a><div class="cit ref-cit ref-journal" id="cit-634568v3.7" data-doi="10.1126/science.1209472"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Chen</span>,  <span class="cit-name-given-names">Y.</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">J. T.</span> <span class="cit-name-surname">Randerson</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">D. C.</span> <span class="cit-name-surname">Morton</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">R. S.</span> <span class="cit-name-surname">DeFries</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">G. J.</span> <span class="cit-name-surname">Collatz</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">P. S.</span> <span class="cit-name-surname">Kasibhatla</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">L.</span> <span class="cit-name-surname">Giglio</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">Y.</span> <span class="cit-name-surname">Jin</span></span>, and <span class="cit-auth"> <span class="cit-name-given-names">M. E.</span> <span class="cit-name-surname">Marlier</span></span>. <span class="cit-pub-date">2011</span>. <span class="cit-article-title">Forecasting fire season severity in south america using sea surface temperature anomalies</span>. <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">334</span>:<span class="cit-fpage">787</span>–<span class="cit-lpage">791</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DChen%26rft.auinit1%253DY.%26rft.volume%253D334%26rft.issue%253D6057%26rft.spage%253D787%26rft.epage%253D791%26rft.atitle%253DForecasting%2BFire%2BSeason%2BSeverity%2Bin%2BSouth%2BAmerica%2BUsing%2BSea%2BSurface%2BTemperature%2BAnomalies%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1209472%26rft_id%253Dinfo%253Apmid%252F22076373%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEyOiIzMzQvNjA1Ny83ODciO3M6NDoiYXRvbSI7czozNzoiL2Jpb3J4aXYvZWFybHkvMjAxOS8xMC8wNy82MzQ1NjguYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference  in text" id="ref-8">↵</a><div class="cit ref-cit ref-journal" id="cit-634568v3.8" data-doi="10.1146/annurev-ecolsys-110316-022706"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Chuine</span>,  <span class="cit-name-given-names">I.</span></span>, and <span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Régnière</span></span>. <span class="cit-pub-date">2017</span>. <span class="cit-article-title">Process-based models of phenology for plants and animals</span>. <abbr class="cit-jnl-abbrev">Annual Review of Ecology, Evolution, and Systematics</abbr> <span class="cit-vol">48</span>:<span class="cit-fpage">159</span>–<span class="cit-lpage">182</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAnnual%2BReview%2Bof%2BEcology%252C%2BEvolution%252C%2Band%2BSystematics%26rft.volume%253D48%26rft.spage%253D159%26rft_id%253Dinfo%253Adoi%252F10.1146%252Fannurev-ecolsys-110316-022706%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1146/annurev-ecolsys-110316-022706&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference  in text" id="ref-9">↵</a><div class="cit ref-cit ref-book" id="cit-634568v3.9"><div class="cit-metadata"><ol class="duplicate"><li><span class="cit-ed"> <span class="cit-name-given-names">M. D.</span> <span class="cit-name-surname">Schwartz</span></span></li></ol><cite><span class="cit-auth"><span class="cit-name-surname">Chuine</span>,  <span class="cit-name-given-names">I.</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">I. G.</span> <span class="cit-name-surname">de Cortazar-Atauri</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">K.</span> <span class="cit-name-surname">Kramer</span></span>, and <span class="cit-auth"> <span class="cit-name-given-names">H.</span> <span class="cit-name-surname">Hänninen</span></span>. <span class="cit-pub-date">2013</span>. <span class="cit-chapter-title">Plant development models</span>. Pages <span class="cit-fpage">275</span>–<span class="cit-lpage">293</span> in <span class="cit-ed"> <span class="cit-name-given-names">M. D.</span> <span class="cit-name-surname">Schwartz</span></span>, <span class="cit-edition">editor</span>. <span class="cit-source">Phenology: An integrative environmental science</span>. <span class="cit-publ-name">Springer</span> <span class="cit-publ-loc">Netherlands, Dordrecht</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference  in text" id="ref-10">↵</a><div class="cit ref-cit ref-journal" id="cit-634568v3.10"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Clark</span>,  <span class="cit-name-given-names">J. S.</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">C.</span> <span class="cit-name-surname">Salk</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Melillo</span></span>, and <span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Mohan</span></span>. <span class="cit-pub-date">2014</span>. <span class="cit-article-title">Tree phenology responses to winter chilling, spring warming, at north and south range limits</span>. <abbr class="cit-jnl-abbrev">Functional Ecology</abbr> <span class="cit-vol">28</span>:<span class="cit-fpage">1344</span>–<span class="cit-lpage">1355</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DFunctional%2BEcology%26rft.volume%253D28%26rft.spage%253D1344%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference  in text" id="ref-11">↵</a><div class="cit ref-cit ref-journal" id="cit-634568v3.11"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Cook</span>,  <span class="cit-name-given-names">B. I.</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Terando</span></span>, and <span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Steiner</span></span>. <span class="cit-pub-date">2010</span>. <span class="cit-article-title">Ecological forecasting under climatic data uncertainty: A case study in phenological modeling</span>. <abbr class="cit-jnl-abbrev">Environmental Research Letters</abbr> <span class="cit-vol">5</span>:<span class="cit-fpage">044014</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEnvironmental%2BResearch%2BLetters%26rft.volume%253D5%26rft.spage%253D044014%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference  in text" id="ref-12">↵</a><div class="cit ref-cit ref-journal" id="cit-634568v3.12" data-doi="10.1016/j.advwatres.2011.04.013"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Dalcin</span>,  <span class="cit-name-given-names">L. D.</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">R. R.</span> <span class="cit-name-surname">Paz</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">P. A.</span> <span class="cit-name-surname">Kler</span></span>, and <span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Cosimo</span></span>. <span class="cit-pub-date">2011</span>. <span class="cit-article-title">Parallel distributed computing using python</span>. <abbr class="cit-jnl-abbrev">Advances in Water Resources</abbr> <span class="cit-vol">34</span>:<span class="cit-fpage">1124</span>–<span class="cit-lpage">1139</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAdvances%2Bin%2BWater%2BResources%26rft.stitle%253DAdvances%2Bin%2BWater%2BResources%26rft.aulast%253DDalcin%26rft.auinit1%253DL.%2BD.%26rft.volume%253D34%26rft.issue%253D9%26rft.spage%253D1124%26rft.epage%253D1139%26rft.atitle%253DParallel%2Bdistributed%2Bcomputing%2Busing%2BPython%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.advwatres.2011.04.013%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.advwatres.2011.04.013&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=2015050401&amp;link_type=GEOREF" class="cit-ref-sprinkles cit-ref-sprinkles-georef"><span>GeoRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference  in text" id="ref-13">↵</a><div class="cit ref-cit ref-journal" id="cit-634568v3.13"><div class="cit-metadata"><cite><span class="cit-auth cit-collab">Dask Development Team</span>. <span class="cit-pub-date">2016</span>. <span class="cit-article-title">Dask: Library for dynamic task scheduling</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference  in text" id="ref-14">↵</a><div class="cit ref-cit ref-journal" id="cit-634568v3.14" data-doi="10.1007/s00484-014-0789-5"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Denny</span>,  <span class="cit-name-given-names">E. G.</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">K. L.</span> <span class="cit-name-surname">Gerst</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">A. J.</span> <span class="cit-name-surname">Miller-Rushing</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">G. L.</span> <span class="cit-name-surname">Tierney</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">T. M.</span> <span class="cit-name-surname">Crimmins</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">C. A. F.</span> <span class="cit-name-surname">Enquist</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">P.</span> <span class="cit-name-surname">Guertin</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">A. H.</span> <span class="cit-name-surname">Rosemartin</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">M. D.</span> <span class="cit-name-surname">Schwartz</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">K. A.</span> <span class="cit-name-surname">Thomas</span></span>, and <span class="cit-auth"> <span class="cit-name-given-names">J. F.</span> <span class="cit-name-surname">Weltzin</span></span>. <span class="cit-pub-date">2014</span>. <span class="cit-article-title">Standardized phenology monitoring methods to track plant and animal activity for science and resource management applications</span>. <abbr class="cit-jnl-abbrev">International Journal of Biometeorology</abbr> <span class="cit-vol">58</span>:<span class="cit-fpage">591</span>–<span class="cit-lpage">601</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DInternational%2BJournal%2Bof%2BBiometeorology%26rft.volume%253D58%26rft.spage%253D591%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00484-014-0789-5%26rft_id%253Dinfo%253Apmid%252F24458770%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00484-014-0789-5&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24458770&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F10%2F07%2F634568.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference  in text" id="ref-15">↵</a><div class="cit ref-cit ref-journal" id="cit-634568v3.15"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Dietze</span>,  <span class="cit-name-given-names">M. C.</span></span> <span class="cit-pub-date">2017</span>. <span class="cit-article-title">Prediction in ecology: A first-principles framework</span>. <abbr class="cit-jnl-abbrev">Ecological Applications</abbr> <span class="cit-vol">27</span>:<span class="cit-fpage">2048</span>–<span class="cit-lpage">2060</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEcological%2BApplications%26rft.volume%253D27%26rft.spage%253D2048%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference  in text" id="ref-16">↵</a><div class="cit ref-cit ref-journal" id="cit-634568v3.16" data-doi="10.1073/pnas.1710231115"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Dietze</span>,  <span class="cit-name-given-names">M. C.</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Fox</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">L. M.</span> <span class="cit-name-surname">Beck-Johnson</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">J. L.</span> <span class="cit-name-surname">Betancourt</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">M. B.</span> <span class="cit-name-surname">Hooten</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">C. S.</span> <span class="cit-name-surname">Jarnevich</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">T. H.</span> <span class="cit-name-surname">Keitt</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">M. A.</span> <span class="cit-name-surname">Kenney</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">C. M.</span> <span class="cit-name-surname">Laney</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">L. G.</span> <span class="cit-name-surname">Larsen</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">H. W.</span> <span class="cit-name-surname">Loescher</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">C. K.</span> <span class="cit-name-surname">Lunch</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">B. C.</span> <span class="cit-name-surname">Pijanowski</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">J. T.</span> <span class="cit-name-surname">Randerson</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">E. K.</span> <span class="cit-name-surname">Read</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">A. T.</span> <span class="cit-name-surname">Tredennick</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">R.</span> <span class="cit-name-surname">Vargas</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">K. C.</span> <span class="cit-name-surname">Weathers</span></span>, and <span class="cit-auth"> <span class="cit-name-given-names">E. P.</span> <span class="cit-name-surname">White</span></span>. <span class="cit-pub-date">2018</span>. <span class="cit-article-title">Iterative near-term ecological forecasting: Needs, opportunities, and challenges</span>. <abbr class="cit-jnl-abbrev">Proceedings of the National Academy of Sciences</abbr> <span class="cit-vol">115</span>:<span class="cit-fpage">1424</span>–<span class="cit-lpage">1432</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProceedings%2Bof%2Bthe%2BNational%2BAcademy%2Bof%2BSciences%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.1710231115%26rft_id%253Dinfo%253Apmid%252F29382745%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMDoiMTE1LzcvMTQyNCI7czo0OiJhdG9tIjtzOjM3OiIvYmlvcnhpdi9lYXJseS8yMDE5LzEwLzA3LzYzNDU2OC5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference  in text" id="ref-17">↵</a><div class="cit ref-cit ref-journal" id="cit-634568v3.17"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Dormann</span>,  <span class="cit-name-given-names">C. F.</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">J. M.</span> <span class="cit-name-surname">Calabrese</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">G.</span> <span class="cit-name-surname">Guillera-Arroita</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">E.</span> <span class="cit-name-surname">Matechou</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">V.</span> <span class="cit-name-surname">Bahn</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">K.</span> <span class="cit-name-surname">Bartoń</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">C. M.</span> <span class="cit-name-surname">Beale</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">Ciuti</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Elith</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">K.</span> <span class="cit-name-surname">Gerstner</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Guelat</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">P.</span> <span class="cit-name-surname">Keil</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">J. J.</span> <span class="cit-name-surname">Lahoz-Monfort</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">L. J.</span> <span class="cit-name-surname">Pollock</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">B.</span> <span class="cit-name-surname">Reineking</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">D. R.</span> <span class="cit-name-surname">Roberts</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">B.</span> <span class="cit-name-surname">Schröder</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">W.</span> <span class="cit-name-surname">Thuiller</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">D. I.</span> <span class="cit-name-surname">Warton</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">B. A.</span> <span class="cit-name-surname">Wintle</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">S. N.</span> <span class="cit-name-surname">Wood</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">R. O.</span> <span class="cit-name-surname">Wüest</span></span>, and <span class="cit-auth"> <span class="cit-name-given-names">F.</span> <span class="cit-name-surname">Hartig</span></span>. <span class="cit-pub-date">2018</span>. <span class="cit-article-title">Model averaging in ecology: A review of bayesian, information-theoretic, and tactical approaches for predictive inference</span>. <abbr class="cit-jnl-abbrev">Ecological Monographs</abbr> <span class="cit-vol">0</span>:<span class="cit-fpage">1</span>–<span class="cit-lpage">20</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEcological%2BMonographs%26rft.volume%253D0%26rft.spage%253D1%26rft.atitle%253DMODEL%2BAVERAGING%2BIN%2BECOLOGY%253A%2BA%2BREVIEW%2BOF%2BBAYESIAN%252C%2BINFORMATION-THEORETIC%252C%2BAND%2BTACTICAL%2BAPPROACHES%2BFOR%2BPREDICTIVE%2BINFERENCE%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference  in text" id="ref-18">↵</a><div class="cit ref-cit ref-journal" id="cit-634568v3.18"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Grolemund</span>,  <span class="cit-name-given-names">G.</span></span>, and <span class="cit-auth"> <span class="cit-name-given-names">H.</span> <span class="cit-name-surname">Wickham</span></span>. <span class="cit-pub-date">2011</span>. <span class="cit-article-title">Dates and times made easy with {lubridate}</span>. <abbr class="cit-jnl-abbrev">Journal of Statistical Software</abbr> <span class="cit-vol">40</span>:<span class="cit-fpage">1</span>–<span class="cit-lpage">25</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BStatistical%2BSoftware%26rft.volume%253D40%26rft.spage%253D1%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference  in text" id="ref-19">↵</a><div class="cit ref-cit ref-web" id="cit-634568v3.19"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Hart</span>,  <span class="cit-name-given-names">E. M.</span></span>, and <span class="cit-auth"> <span class="cit-name-given-names">K.</span> <span class="cit-name-surname">Bell</span></span>. <span class="cit-pub-date">2015</span>. <span class="cit-article-title">Prism: Download data from the oregon prism project</span>. <a href="http://github.com/ropensci/prism">http://github.com/ropensci/prism</a>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference  in text" id="ref-20">↵</a><div class="cit ref-cit ref-web" id="cit-634568v3.20"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Hijmans</span>,  <span class="cit-name-given-names">R. J.</span></span> <span class="cit-pub-date">2017</span>. <span class="cit-article-title">Raster: Geographic data analysis and modeling. r package version 2.6-7</span>. <a href="https://CRAN.R-project.org/package=raster">https://CRAN.R-project.org/package=raster</a>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference  in text" id="ref-21">↵</a><div class="cit ref-cit ref-journal" id="cit-634568v3.21"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Hoyer</span>,  <span class="cit-name-given-names">S.</span></span>, and <span class="cit-auth"> <span class="cit-name-given-names">J. J.</span> <span class="cit-name-surname">Hamman</span></span>. <span class="cit-pub-date">2017</span>. <span class="cit-article-title">Xarray: N-d labeled arrays and datasets in python</span>. <abbr class="cit-jnl-abbrev">Journal of Open Research Software</abbr> <span class="cit-vol">5</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference  in text" id="ref-22">↵</a><div class="cit ref-cit ref-web" id="cit-634568v3.22"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Jones</span>,  <span class="cit-name-given-names">E.</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">T.</span> <span class="cit-name-surname">Oliphant</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">P.</span> <span class="cit-name-surname">Peterson</span></span>, and Others. <span class="cit-pub-date">2001</span>. <span class="cit-article-title">SciPy: Open source scientific tools for python</span>. <a href="http://www.scipy.org">http://www.scipy.org</a>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference  in text" id="ref-23">↵</a><div class="cit ref-cit ref-journal" id="cit-634568v3.23" data-doi="10.1890/09-1275.1"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Luo</span>,  <span class="cit-name-given-names">Y.</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">K.</span> <span class="cit-name-surname">Ogle</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">C.</span> <span class="cit-name-surname">Tucker</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">Fei</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">C.</span> <span class="cit-name-surname">Gao</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">LaDeau</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">J. S.</span> <span class="cit-name-surname">Clark</span></span>, and <span class="cit-auth"> <span class="cit-name-given-names">D. S.</span> <span class="cit-name-surname">Schimel</span></span>. <span class="cit-pub-date">2011</span>. <span class="cit-article-title">Ecological forecasting and data assimilation in a data-rich era</span>. <abbr class="cit-jnl-abbrev">Ecological Applications</abbr> <span class="cit-vol">21</span>:<span class="cit-fpage">1429</span>–<span class="cit-lpage">1442</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEcological%2BApplications%26rft.stitle%253DEcological%2BApplications%26rft.aulast%253DLuo%26rft.auinit1%253DY.%26rft.volume%253D21%26rft.issue%253D5%26rft.spage%253D1429%26rft.epage%253D1442%26rft.atitle%253DEcological%2Bforecasting%2Band%2Bdata%2Bassimilation%2Bin%2Ba%2Bdata-rich%2Bera.%26rft_id%253Dinfo%253Adoi%252F10.1890%252F09-1275.1%26rft_id%253Dinfo%253Apmid%252F21830693%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1890/09-1275.1&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21830693&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F10%2F07%2F634568.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000292766100002&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference  in text" id="ref-24">↵</a><div class="cit ref-cit ref-book" id="cit-634568v3.24"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">McKinney</span>,  <span class="cit-name-given-names">W.</span></span> <span class="cit-pub-date">2010</span>. <span class="cit-chapter-title">Data structures for statistical computing in python</span>. Pages <span class="cit-fpage">51</span>–<span class="cit-lpage">56</span>. in <span class="cit-source">Proceedings of the 9th python in science conference</span>. <span class="cit-publ-name">SciPy</span>, <span class="cit-publ-loc">Austin, Texas, USA</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference  in text" id="ref-25">↵</a><div class="cit ref-cit ref-journal" id="cit-634568v3.25"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Melaas</span>,  <span class="cit-name-given-names">E. K.</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">M. A.</span> <span class="cit-name-surname">Friedl</span></span>, and <span class="cit-auth"> <span class="cit-name-given-names">A. D.</span> <span class="cit-name-surname">Richardson</span></span>. <span class="cit-pub-date">2016</span>. <span class="cit-article-title">Multiscale modeling of spring phenology across Deciduous Forests in the Eastern United States</span>. <abbr class="cit-jnl-abbrev">Global Change Biology</abbr> <span class="cit-vol">22</span>:<span class="cit-fpage">792</span>–<span class="cit-lpage">805</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGlobal%2BChange%2BBiology%26rft.volume%253D22%26rft.spage%253D792%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference  in text" id="ref-26">↵</a><div class="cit ref-cit ref-journal" id="cit-634568v3.26"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Ogilvie</span>,  <span class="cit-name-given-names">J. E.</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">S. R.</span> <span class="cit-name-surname">Griffin</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">Z. J.</span> <span class="cit-name-surname">Gezon</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">B. D.</span> <span class="cit-name-surname">Inouye</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">N.</span> <span class="cit-name-surname">Underwood</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">D. W.</span> <span class="cit-name-surname">Inouye</span></span>, and <span class="cit-auth"> <span class="cit-name-given-names">R. E.</span> <span class="cit-name-surname">Irwin</span></span>. <span class="cit-pub-date">2017</span>. <span class="cit-article-title">Interannual bumble bee abundance is driven by indirect climate effects on floral resource phenology</span>. <abbr class="cit-jnl-abbrev">Ecology Letters</abbr> <span class="cit-vol">20</span>:<span class="cit-fpage">1507</span>–<span class="cit-lpage">1515</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEcology%2BLetters%26rft.volume%253D20%26rft.spage%253D1507%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-27-1" title="View reference  in text" id="ref-27">↵</a><div class="cit ref-cit ref-book" id="cit-634568v3.27"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Oliphant</span>,  <span class="cit-name-given-names">T.</span></span> <span class="cit-pub-date">2006</span>. <span class="cit-chapter-title">A guide to numpy</span>. <span class="cit-publ-loc">USA</span>: <span class="cit-publ-name">Trelgol Publishing; Trelgol Publishing, Provo, UT</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-28-1" title="View reference  in text" id="ref-28">↵</a><div class="cit ref-cit ref-journal" id="cit-634568v3.28"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Pebesma</span>,  <span class="cit-name-given-names">E. J.</span></span>, and <span class="cit-auth"> <span class="cit-name-given-names">R. S.</span> <span class="cit-name-surname">Bivand</span></span>. <span class="cit-pub-date">2005</span>. <span class="cit-article-title">Classes and methods for spatial data in {R}</span>. <abbr class="cit-jnl-abbrev">R News</abbr> <span class="cit-vol">5</span>:<span class="cit-fpage">9</span>–<span class="cit-lpage">13</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DR%2BNews%26rft.volume%253D5%26rft.spage%253D9%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-29-1" title="View reference  in text" id="ref-29">↵</a><div class="cit ref-cit ref-journal" id="cit-634568v3.29" data-doi="10.1111/ele.12443"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Petchey</span>,  <span class="cit-name-given-names">O. L.</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Pontarp</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">T. M.</span> <span class="cit-name-surname">Massie</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">Kéfi</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Ozgul</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Weilenmann</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">G. M.</span> <span class="cit-name-surname">Palamara</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">F.</span> <span class="cit-name-surname">Altermatt</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">B.</span> <span class="cit-name-surname">Matthews</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">J. M.</span> <span class="cit-name-surname">Levine</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">D. Z.</span> <span class="cit-name-surname">Childs</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">B. J.</span> <span class="cit-name-surname">McGill</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">M. E.</span> <span class="cit-name-surname">Schaepman</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">B.</span> <span class="cit-name-surname">Schmid</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">P.</span> <span class="cit-name-surname">Spaak</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">A. P.</span> <span class="cit-name-surname">Beckerman</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">F.</span> <span class="cit-name-surname">Pennekamp</span></span>, and <span class="cit-auth"> <span class="cit-name-given-names">I. S.</span> <span class="cit-name-surname">Pearse</span></span>. <span class="cit-pub-date">2015</span>. <span class="cit-article-title">The ecological forecast horizon, and examples of its uses and determinants</span>. <abbr class="cit-jnl-abbrev">Ecology Letters</abbr> <span class="cit-vol">18</span>:<span class="cit-fpage">597</span>–<span class="cit-lpage">611</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEcology%2BLetters%26rft.volume%253D18%26rft.spage%253D597%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fele.12443%26rft_id%253Dinfo%253Apmid%252F25960188%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/ele.12443&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25960188&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F10%2F07%2F634568.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-30-1" title="View reference  in text" id="ref-30">↵</a><div class="cit ref-cit ref-journal" id="cit-634568v3.30"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Pierce</span>,  <span class="cit-name-given-names">D.</span></span> <span class="cit-pub-date">2017</span>. <span class="cit-article-title">Ncdf4: Interface to unidata netCDF (version 4 or earlier) format data files}</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-634568v3.31"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Prevéy</span>,  <span class="cit-name-given-names">J.</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">L.</span> <span class="cit-name-surname">Parker</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">C.</span> <span class="cit-name-surname">Harrington</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">C.</span> <span class="cit-name-surname">Lamb</span></span>, and <span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Proctor</span></span>. In revision. <span class="cit-article-title">Climate change shifts the habitat suitability and phenology of black huckleberry</span>. <abbr class="cit-jnl-abbrev">Agricultural and Forest Meteorology</abbr>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-32-1" title="View reference  in text" id="ref-32">↵</a><div class="cit ref-cit ref-book" id="cit-634568v3.32"><div class="cit-metadata"><cite><span class="cit-auth cit-collab">PRISM Climate Group</span>. <span class="cit-pub-date">2004</span>. <span class="cit-source">Oregon state university</span>. <a href="http://prism.oregonstate.edu">http://prism.oregonstate.edu</a>; <span class="cit-publ-name">Oregon State University</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-33-1" title="View reference  in text" id="ref-33">↵</a><div class="cit ref-cit ref-web" id="cit-634568v3.33"><div class="cit-metadata"><cite><span class="cit-auth cit-collab">Python Software Foundation</span>. <span class="cit-pub-date">2003</span>. <span class="cit-article-title">Python language reference manual, version 3.6</span>. <a href="http://www.python.org">http://www.python.org</a>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-34-1" title="View reference  in text" id="ref-34">↵</a><div class="cit ref-cit ref-journal" id="cit-634568v3.34" data-doi="10.1186/1751-0473-8-7"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Ram</span>,  <span class="cit-name-given-names">K.</span></span> <span class="cit-pub-date">2013</span>. <span class="cit-article-title">Git can facilitate greater reproducibility and increased transparency in science</span>. <abbr class="cit-jnl-abbrev">Source Code for Biology and Medicine</abbr> <span class="cit-vol">8</span>:<span class="cit-fpage">7</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DSource%2BCode%2Bfor%2BBiology%2Band%2BMedicine%26rft.volume%253D8%26rft.spage%253D7%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1751-0473-8-7%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1751-0473-8-7&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-35-1" title="View reference  in text" id="ref-35">↵</a><div class="cit ref-cit ref-journal" id="cit-634568v3.35" data-doi="10.1111/j.1365-2486.2011.02562.x"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Richardson</span>,  <span class="cit-name-given-names">A. D.</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">R. S.</span> <span class="cit-name-surname">Anderson</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">M. A.</span> <span class="cit-name-surname">Arain</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">A. G.</span> <span class="cit-name-surname">Barr</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">G.</span> <span class="cit-name-surname">Bohrer</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">G.</span> <span class="cit-name-surname">Chen</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">J. M.</span> <span class="cit-name-surname">Chen</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">P.</span> <span class="cit-name-surname">Ciais</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">K. J.</span> <span class="cit-name-surname">Davis</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">A. R.</span> <span class="cit-name-surname">Desai</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">M. C.</span> <span class="cit-name-surname">Dietze</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">D.</span> <span class="cit-name-surname">Dragoni</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">S. R.</span> <span class="cit-name-surname">Garrity</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">C. M.</span> <span class="cit-name-surname">Gough</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">R.</span> <span class="cit-name-surname">Grant</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">D. Y.</span> <span class="cit-name-surname">Hollinger</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">H. A.</span> <span class="cit-name-surname">Margolis</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">H.</span> <span class="cit-name-surname">McCaughey</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Migliavacca</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">R. K.</span> <span class="cit-name-surname">Monson</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">J. W.</span> <span class="cit-name-surname">Munger</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">B.</span> <span class="cit-name-surname">Poulter</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">B. M.</span> <span class="cit-name-surname">Raczka</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">D. M.</span> <span class="cit-name-surname">Ricciuto</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">A. K.</span> <span class="cit-name-surname">Sahoo</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">K.</span> <span class="cit-name-surname">Schaefer</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">H.</span> <span class="cit-name-surname">Tian</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">R.</span> <span class="cit-name-surname">Vargas</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">H.</span> <span class="cit-name-surname">Verbeeck</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Xiao</span></span>, and <span class="cit-auth"> <span class="cit-name-given-names">Y.</span> <span class="cit-name-surname">Xue</span></span>. <span class="cit-pub-date">2012</span>. <span class="cit-article-title">Terrestrial biosphere models need better representation of vegetation phenology: Results from the north american carbon program site synthesis</span>. <abbr class="cit-jnl-abbrev">Global Change Biology</abbr> <span class="cit-vol">18</span>:<span class="cit-fpage">566</span>–<span class="cit-lpage">584</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGlobal%2BChange%2BBiology%26rft.volume%253D18%26rft.spage%253D566%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1365-2486.2011.02562.x%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1365-2486.2011.02562.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-36-1" title="View reference  in text" id="ref-36">↵</a><div class="cit ref-cit ref-journal" id="cit-634568v3.36" data-doi="10.1175/2010BAMS3001.1"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Saha</span>,  <span class="cit-name-given-names">S.</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">Moorthi</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">H.-L.</span> <span class="cit-name-surname">Pan</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">X.</span> <span class="cit-name-surname">Wu</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Wang</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">Nadiga</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">P.</span> <span class="cit-name-surname">Tripp</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">R.</span> <span class="cit-name-surname">Kistler</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Woollen</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">D.</span> <span class="cit-name-surname">Behringer</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">H.</span> <span class="cit-name-surname">Liu</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">D.</span> <span class="cit-name-surname">Stokes</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">R.</span> <span class="cit-name-surname">Grumbine</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">G.</span> <span class="cit-name-surname">Gayno</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Wang</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">Y.-T.</span> <span class="cit-name-surname">Hou</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">H.-y.</span> <span class="cit-name-surname">Chuang</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">H.-M. H.</span> <span class="cit-name-surname">Juang</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Sela</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Iredell</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">R.</span> <span class="cit-name-surname">Treadon</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">D.</span> <span class="cit-name-surname">Kleist</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">P.</span> <span class="cit-name-surname">Van Delst</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">D.</span> <span class="cit-name-surname">Keyser</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Derber</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Ek</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Meng</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">H.</span> <span class="cit-name-surname">Wei</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">R.</span> <span class="cit-name-surname">Yang</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">Lord</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">H.</span> <span class="cit-name-surname">van den Dool</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Kumar</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">W.</span> <span class="cit-name-surname">Wang</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">C.</span> <span class="cit-name-surname">Long</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Chelliah</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">Y.</span> <span class="cit-name-surname">Xue</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">B.</span> <span class="cit-name-surname">Huang</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">J.-K.</span> <span class="cit-name-surname">Schemm</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">W.</span> <span class="cit-name-surname">Ebisuzaki</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">R.</span> <span class="cit-name-surname">Lin</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">P.</span> <span class="cit-name-surname">Xie</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Chen</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">Zhou</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">W.</span> <span class="cit-name-surname">Higgins</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">C.-Z.</span> <span class="cit-name-surname">Zou</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">Q.</span> <span class="cit-name-surname">Liu</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">Y.</span> <span class="cit-name-surname">Chen</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">Y.</span> <span class="cit-name-surname">Han</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">L.</span> <span class="cit-name-surname">Cucurull</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">R. W.</span> <span class="cit-name-surname">Reynolds</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">G.</span> <span class="cit-name-surname">Rutledge</span></span>, and <span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Goldberg</span></span>. <span class="cit-pub-date">2010</span>. <span class="cit-article-title">The ncep climate forecast system reanalysis</span>. <abbr class="cit-jnl-abbrev">Bulletin of the American Meteorological Society</abbr> <span class="cit-vol">91</span>:<span class="cit-fpage">1015</span>–<span class="cit-lpage">1058</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBulletin%2Bof%2Bthe%2BAmerican%2BMeteorological%2BSociety%26rft.volume%253D91%26rft.spage%253D1015%26rft_id%253Dinfo%253Adoi%252F10.1175%252F2010BAMS3001.1%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1175/2010BAMS3001.1&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-37-1" title="View reference  in text" id="ref-37">↵</a><div class="cit ref-cit ref-journal" id="cit-634568v3.37"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Saha</span>,  <span class="cit-name-given-names">S.</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">Moorthi</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">X.</span> <span class="cit-name-surname">Wu</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Wang</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">Nadiga</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">P.</span> <span class="cit-name-surname">Tripp</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">D.</span> <span class="cit-name-surname">Behringer</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">Y.-T.</span> <span class="cit-name-surname">Hou</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">H.-y.</span> <span class="cit-name-surname">Chuang</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Iredell</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Ek</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Meng</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">R.</span> <span class="cit-name-surname">Yang</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">M. P.</span> <span class="cit-name-surname">Mendez</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">H.</span> <span class="cit-name-surname">van den Dool</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">Q.</span> <span class="cit-name-surname">Zhang</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">W.</span> <span class="cit-name-surname">Wang</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Chen</span></span>, and <span class="cit-auth"> <span class="cit-name-given-names">E.</span> <span class="cit-name-surname">Becker</span></span>. <span class="cit-pub-date">2014</span>. <span class="cit-article-title">The ncep climate forecast system version 2</span>. <abbr class="cit-jnl-abbrev">Journal of Climate</abbr> <span class="cit-vol">27</span>:<span class="cit-fpage">2185</span>–<span class="cit-lpage">2208</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BClimate%26rft.volume%253D27%26rft.spage%253D2185%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-38-1" title="View reference  in text" id="ref-38">↵</a><div class="cit ref-cit ref-journal" id="cit-634568v3.38" data-doi="10.1002/joc.3625"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Schwartz</span>,  <span class="cit-name-given-names">M. D.</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">T. R.</span> <span class="cit-name-surname">Ault</span></span>, and <span class="cit-auth"> <span class="cit-name-given-names">J. L.</span> <span class="cit-name-surname">Betancourt</span></span>. <span class="cit-pub-date">2013</span>. <span class="cit-article-title">Spring onset variations and trends in the continental United States: past and regional assessment using temperature-based indices</span>. <abbr class="cit-jnl-abbrev">International Journal of Climatology</abbr> <span class="cit-vol">33</span>:<span class="cit-fpage">2917</span>–<span class="cit-lpage">2922</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DInternational%2BJournal%2Bof%2BClimatology%26rft.volume%253D33%26rft.spage%253D2917%26rft_id%253Dinfo%253Adoi%252F10.1002%252Fjoc.3625%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1002/joc.3625&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-39-1" title="View reference  in text" id="ref-39">↵</a><div class="cit ref-cit ref-journal" id="cit-634568v3.39"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Stoner</span>,  <span class="cit-name-given-names">A. M. K.</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">K.</span> <span class="cit-name-surname">Hayhoe</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">X.</span> <span class="cit-name-surname">Yang</span></span>, and <span class="cit-auth"> <span class="cit-name-given-names">D. J.</span> <span class="cit-name-surname">Wuebbles</span></span>. <span class="cit-pub-date">2013</span>. <span class="cit-article-title">An asynchronous regional regression model for statistical downscaling of daily climate variables</span>. <abbr class="cit-jnl-abbrev">International Journal of Climatology</abbr> <span class="cit-vol">33</span>:<span class="cit-fpage">2473</span>–<span class="cit-lpage">2494</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DInternational%2BJournal%2Bof%2BClimatology%26rft.volume%253D33%26rft.spage%253D2473%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-40-1" title="View reference  in text" id="ref-40">↵</a><div class="cit ref-cit ref-journal" id="cit-634568v3.40"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Taylor</span>,  <span class="cit-name-given-names">S. D.</span></span> <span class="cit-pub-date">2018</span>. <span class="cit-article-title">PyPhenology: A python framework for plant phenology modelling</span>. <abbr class="cit-jnl-abbrev">Journal of Open Source Software</abbr> <span class="cit-vol">3</span>:<span class="cit-fpage">827</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BOpen%2BSource%2BSoftware%26rft.volume%253D3%26rft.spage%253D827%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-41-1" title="View reference  in text" id="ref-41">↵</a><div class="cit ref-cit ref-journal" id="cit-634568v3.41"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Taylor</span>,  <span class="cit-name-given-names">S. D.</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">J. M.</span> <span class="cit-name-surname">Meiners</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">K.</span> <span class="cit-name-surname">Riemer</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">M. C.</span> <span class="cit-name-surname">Orr</span></span>, and <span class="cit-auth"> <span class="cit-name-given-names">E. P.</span> <span class="cit-name-surname">White</span></span>. <span class="cit-pub-date">2019</span>. <span class="cit-article-title">Comparison of large-scale citizen science data and long-term study data for phenology modeling</span>. <abbr class="cit-jnl-abbrev">Ecology</abbr> <span class="cit-vol">100</span>:<span class="cit-fpage">e02568</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEcology%26rft.volume%253D100%26rft.spage%253De02568%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-42-1" title="View reference  in text" id="ref-42">↵</a><div class="cit ref-cit ref-book" id="cit-634568v3.42"><div class="cit-metadata"><ol class="duplicate"><li><span class="cit-ed"><span class="cit-name-surname">Elbert</span>  <span class="cit-name-given-names">L. Little</span>,   <span class="cit-name-suffix">Jr.</span>.</span></li></ol><cite><span class="cit-auth cit-collab">US Geological Survey</span>. <span class="cit-pub-date">1999</span>. <span class="cit-chapter-title">Digital representation of “Atlas of United States Trees”</span> by <span class="cit-ed"><span class="cit-name-surname">Elbert</span>  <span class="cit-name-given-names">L. Little</span>,   <span class="cit-name-suffix">Jr.</span>.</span> <span class="cit-publ-name">US Geological Survey, Lakewood, CO.</span></cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-43-1" title="View reference  in text" id="ref-43">↵</a><div class="cit ref-cit ref-book" id="cit-634568v3.43"><div class="cit-metadata"><cite><span class="cit-auth cit-collab">USA National Phenology Network</span>. <span class="cit-pub-date">2018</span>. <span class="cit-chapter-title">Plant and animal phenology data. data type: Individual phenometrics. 01/01/2008-12/31/2017 for region: 49.9375°, −66.4791667° (ur); 24.0625°, −125.0208333° (ll)</span>. <span class="cit-publ-name">USA-NPN</span>, <span class="cit-publ-loc">Tucson, Arizona, USA</span>. Data set accessed <span class="cit-date-in-citation">07/08/2018</span> at <a href="http://doi.org/10.5066/F78S4N1V">http://doi.org/10.5066/F78S4N1V</a>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-44-1" title="View reference  in text" id="ref-44">↵</a><div class="cit ref-cit ref-book" id="cit-634568v3.44"><div class="cit-metadata"><cite><span class="cit-auth cit-collab">USA National Phenology Network</span>. <span class="cit-pub-date">2019</span>. <span class="cit-chapter-title">Plant and animal phenology data. data type: Individual phenometrics. 01/01/2019-05/08/2019 for region: 49.9375°, −66.4791667° (ur); 24.0625°, −125.0208333° (ll)</span>. <span class="cit-publ-name">USA-NPN</span>, <span class="cit-publ-loc">Tucson, Arizona, USA</span>. Data set accessed <span class="cit-date-in-citation">05/09/2019</span> at <a href="http://doi.org/10.5066/F78S4N1V">http://doi.org/10.5066/F78S4N1V</a>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-45-1" title="View reference  in text" id="ref-45">↵</a><div class="cit ref-cit ref-journal" id="cit-634568v3.45" data-doi="10.1126/science.aat7526"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Van Doren</span>,  <span class="cit-name-given-names">B. M.</span></span>, and <span class="cit-auth"> <span class="cit-name-given-names">K. G.</span> <span class="cit-name-surname">Horton</span></span>. <span class="cit-pub-date">2018</span>. <span class="cit-article-title">A continental system for forecasting bird migration</span>. <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">361</span>:<span class="cit-fpage">1115</span>–<span class="cit-lpage">1118</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DVan%2BDoren%26rft.auinit1%253DB.%2BM.%26rft.volume%253D361%26rft.issue%253D6407%26rft.spage%253D1115%26rft.epage%253D1118%26rft.atitle%253DA%2Bcontinental%2Bsystem%2Bfor%2Bforecasting%2Bbird%2Bmigration%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.aat7526%26rft_id%253Dinfo%253Apmid%252F30213913%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEzOiIzNjEvNjQwNy8xMTE1IjtzOjQ6ImF0b20iO3M6Mzc6Ii9iaW9yeGl2L2Vhcmx5LzIwMTkvMTAvMDcvNjM0NTY4LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-46-1" title="View reference  in text" id="ref-46">↵</a><div class="cit ref-cit ref-journal" id="cit-634568v3.46"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Weisheimer</span>,  <span class="cit-name-given-names">A.</span></span>, and <span class="cit-auth"> <span class="cit-name-given-names">T. N.</span> <span class="cit-name-surname">Palmer</span></span>. <span class="cit-pub-date">2014</span>. <span class="cit-article-title">On the reliability of seasonal climate forecasts</span>. <abbr class="cit-jnl-abbrev">Journal of The Royal Society Interface</abbr> <span class="cit-vol">11</span>:<span class="cit-fpage">20131162</span>–<span class="cit-lpage">20131162</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BThe%2BRoyal%2BSociety%2BInterface%26rft.volume%253D11%26rft.spage%253D20131162%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-47-1" title="View reference  in text" id="ref-47">↵</a><div class="cit ref-cit ref-journal" id="cit-634568v3.47"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Welch</span>,  <span class="cit-name-given-names">H.</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">E. L.</span> <span class="cit-name-surname">Hazen</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">S. J.</span> <span class="cit-name-surname">Bograd</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">M. G.</span> <span class="cit-name-surname">Jacox</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">Brodie</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">D.</span> <span class="cit-name-surname">Robinson</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">K. L.</span> <span class="cit-name-surname">Scales</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">L.</span> <span class="cit-name-surname">Dewitt</span></span>, and <span class="cit-auth"> <span class="cit-name-given-names">R.</span> <span class="cit-name-surname">Lewison</span></span>. <span class="cit-pub-date">2019</span>. <span class="cit-article-title">Practical considerations for operationalizing dynamic management tools</span>. <abbr class="cit-jnl-abbrev">Journal of Applied Ecology</abbr> <span class="cit-vol">56</span>:<span class="cit-fpage">459</span>–<span class="cit-lpage">469</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BApplied%2BEcology%26rft.volume%253D56%26rft.spage%253D459%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-48-1" title="View reference  in text" id="ref-48">↵</a><div class="cit ref-cit ref-journal" id="cit-634568v3.48"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">White</span>,  <span class="cit-name-given-names">E. P.</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">G. M.</span> <span class="cit-name-surname">Yenni</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">S. D.</span> <span class="cit-name-surname">Taylor</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">E. M.</span> <span class="cit-name-surname">Christensen</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">E. K.</span> <span class="cit-name-surname">Bledsoe</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">J. L.</span> <span class="cit-name-surname">Simonis</span></span>, and <span class="cit-auth"> <span class="cit-name-given-names">S. K.</span> <span class="cit-name-surname">Morgan Ernest</span></span>. <span class="cit-pub-date">2018</span>. <span class="cit-article-title">Developing an automated iterative near-term forecasting system for an ecological study</span>. <abbr class="cit-jnl-abbrev">Methods in Ecology and Evolution</abbr>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-49-1" title="View reference  in text" id="ref-49">↵</a><div class="cit ref-cit ref-book" id="cit-634568v3.49"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Wickham</span>,  <span class="cit-name-given-names">H.</span></span> <span class="cit-pub-date">2016</span>. <span class="cit-chapter-title">Ggplot2: Elegant graphics for data analysis</span>. <span class="cit-publ-name">Springer-Verlag</span> <span class="cit-publ-loc">New York</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-50-1" title="View reference  in text" id="ref-50">↵</a><div class="cit ref-cit ref-journal" id="cit-634568v3.50"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Wickham</span>,  <span class="cit-name-given-names">H.</span></span>, and <span class="cit-auth"> <span class="cit-name-given-names">L.</span> <span class="cit-name-surname">Henry</span></span>. <span class="cit-pub-date">2018</span>. <span class="cit-article-title">Tidyr: Easily tidy data with ‘spread()’ and ‘gather()’ functions</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-51-1" title="View reference  in text" id="ref-51">↵</a><div class="cit ref-cit ref-journal" id="cit-634568v3.51"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Yenni</span>,  <span class="cit-name-given-names">G. M.</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">E. M.</span> <span class="cit-name-surname">Christensen</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">E. K.</span> <span class="cit-name-surname">Bledsoe</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">S. R.</span> <span class="cit-name-surname">Supp</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">R. M.</span> <span class="cit-name-surname">Diaz</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">E. P.</span> <span class="cit-name-surname">White</span></span>, and <span class="cit-auth"> <span class="cit-name-given-names">S. K. M.</span> <span class="cit-name-surname">Ernest</span></span>. <span class="cit-pub-date">2019</span>. <span class="cit-article-title">Developing a modern data workflow for regularly updated data</span>. <abbr class="cit-jnl-abbrev">PLOS Biology</abbr> <span class="cit-vol">17</span>:<span class="cit-fpage">e3000125</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLOS%2BBiology%26rft.volume%253D17%26rft.spage%253De3000125%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-52-1" title="View reference  in text" id="ref-52">↵</a><div class="cit ref-cit ref-journal" id="cit-634568v3.52"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Yun</span>,  <span class="cit-name-given-names">K.</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Hsiao</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">M.-P.</span> <span class="cit-name-surname">Jung</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">I.-T.</span> <span class="cit-name-surname">Choi</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">D. M.</span> <span class="cit-name-surname">Glenn</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">K.-M.</span> <span class="cit-name-surname">Shim</span></span>, and <span class="cit-auth"> <span class="cit-name-given-names">S.-H.</span> <span class="cit-name-surname">Kim</span></span>. <span class="cit-pub-date">2017</span>. <span class="cit-article-title">Can a multi-model ensemble improve phenology predictions for climate change studies?</span> <abbr class="cit-jnl-abbrev">Ecological Modelling</abbr> <span class="cit-vol">362</span>:<span class="cit-fpage">54</span>–<span class="cit-lpage">64</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEcological%2BModelling%26rft.volume%253D362%26rft.spage%253D54%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.biorxiv.org/content/10.1101/634568v3.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-disqus-comment" >
  
      
  
  <div class="pane-content">
    <div id="disqus_thread"><noscript><p><a href="http://biorxivstage.disqus.com/?url=https%3A%2F%2Fwww.biorxiv.org%2Fcontent%2F10.1101%2F634568v3" class="" data-icon-position="" data-hide-link-title="0">View the discussion thread.</a></p></noscript></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/10.1101/595603v2" title="A novel approach to localize cortical TMS effects" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous</span></a></span><span class="pager-next"><a href="/content/10.1101/569319v4" title="Neonatal morphometric similarity mapping for predicting brain age and characterizing neuroanatomic variation associated with preterm birth" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    Posted&nbsp;October 07, 2019.  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-biorxiv-art-tools" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-biorxiv_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/10.1101/634568v3.full.pdf" target="_self" class="article-dl-pdf-link link-icon"><i class="icon-external-link-sign"></i> <span class="title">Download PDF</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-biorxiv-supplementary-fragment" >
  
      
  
  <div class="pane-content">
    <p><a href="/content/10.1101/634568v3.supplementary-material"><i class="icon-file"></i>Supplementary Material </a></p>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/early/2019/10/07/634568.external-links" target="_self" class="link-icon"><i class="icon-file"></i> <span class="title">Data/Code</span></a>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-biorxiv-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="Email this Article" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-biorxiv_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/10.1101/634568v3.full" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for your interest in spreading the word about bioRxiv.</p><p>NOTE: Your email address is requested solely to identify you as the sender of this article.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/10.1101/634568v3" class="active" data-icon-position="" data-hide-link-title="0">Automated data-intensive forecasting of plant phenology throughout the United States</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from bioRxiv
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would like to see this page from the bioRxiv website.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/939653" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-sgX35Hmq1Z2pfoptFSq_WrcIRTCKPXs9zhKIDs_xtu8" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-biorxiv-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-biorxiv_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="939653" id="node939653--2" data-pisa="biorxiv;634568v3" data-pisa-master="biorxiv;634568" data-seqnum="939653" data-apath="/biorxiv/early/2019/10/07/634568.atom"><div  class="highwire-cite highwire-cite-highwire-article highwire-citation-biorxiv-article-pap-list clearfix" >

      <div class="highwire-cite-title" >
      <div class="highwire-cite-title">Automated data-intensive forecasting of plant phenology throughout the United States</div>    </div>
  
      <div  class="highwire-cite-authors" ><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0"><span class="nlm-given-names">Shawn D.</span> <span class="nlm-surname">Taylor</span></span>, <span class="highwire-citation-author" data-delta="1"><span class="nlm-given-names">Ethan P.</span> <span class="nlm-surname">White</span></span></span></div>
  
      <div  class="highwire-cite-metadata" ><span  class="highwire-cite-metadata-journal highwire-cite-metadata">bioRxiv </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">634568; </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata"><span class="doi_label">doi:</span> https://doi.org/10.1101/634568 </span></div>
  
  
  </div>
</div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.biorxiv.org/content/10.1101/634568v3" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.biorxiv.org/content/10.1101/634568v3&amp;title=Automated%20data-intensive%20forecasting%20of%20plant%20phenology%20throughout%20the%20United%20States" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.biorxiv.org/content/10.1101/634568v3&amp;title=Automated%20data-intensive%20forecasting%20of%20plant%20phenology%20throughout%20the%20United%20States" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.biorxiv.org/content/10.1101/634568v3&amp;text=Automated%20data-intensive%20forecasting%20of%20plant%20phenology%20throughout%20the%20United%20States" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.biorxiv.org/content/10.1101/634568v3&amp;title=Automated%20data-intensive%20forecasting%20of%20plant%20phenology%20throughout%20the%20United%20States" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.biorxiv.org/content/10.1101/634568v3&amp;t=Automated%20data-intensive%20forecasting%20of%20plant%20phenology%20throughout%20the%20United%20States" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.biorxiv.org/content/10.1101/634568v3&amp;title=Automated%20data-intensive%20forecasting%20of%20plant%20phenology%20throughout%20the%20United%20States" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.biorxiv.org/content/10.1101/634568v3&amp;title=Automated%20data-intensive%20forecasting%20of%20plant%20phenology%20throughout%20the%20United%20States" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-biorxiv-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title="Citation Tools"><i class="icon-globe"></i> <span class="title">Citation Tools</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-biorxiv_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="939653" id="node939653--3" data-pisa="biorxiv;634568v3" data-pisa-master="biorxiv;634568" data-seqnum="939653" data-apath="/biorxiv/early/2019/10/07/634568.atom"><div  class="highwire-cite highwire-cite-highwire-article highwire-citation-biorxiv-article-pap-list clearfix" >

      <div class="highwire-cite-title" >
      <div class="highwire-cite-title">Automated data-intensive forecasting of plant phenology throughout the United States</div>    </div>
  
      <div  class="highwire-cite-authors" ><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0"><span class="nlm-given-names">Shawn D.</span> <span class="nlm-surname">Taylor</span></span>, <span class="highwire-citation-author" data-delta="1"><span class="nlm-given-names">Ethan P.</span> <span class="nlm-surname">White</span></span></span></div>
  
      <div  class="highwire-cite-metadata" ><span  class="highwire-cite-metadata-journal highwire-cite-metadata">bioRxiv </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">634568; </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata"><span class="doi_label">doi:</span> https://doi.org/10.1101/634568 </span></div>
  
  
  </div>
</div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage&amp;rft.epage&amp;rft.atitle=Automated%20data-intensive%20forecasting%20of%20plant%20phenology%20throughout%20the%20United%20States&amp;rft.volume&amp;rft.issue&amp;rft.date=2019-01-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=bioRxiv&amp;rft.au=Taylor%2C+Shawn+D.&amp;rft.au=White%2C+Ethan+P."></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/939653/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/939653/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/939653/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/939653/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/939653/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/939653/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/939653/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/939653/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/939653/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/939653/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/939653/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/939653/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><div class="item-list"><ul><li class="first"><a href="http://twitter.com/share?url=https%3A//www.biorxiv.org/content/10.1101/634568v3&amp;count=horizontal&amp;via=&amp;text=Automated%20data-intensive%20forecasting%20of%20plant%20phenology%20throughout%20the%20United%20States&amp;counturl=https%3A//www.biorxiv.org/content/10.1101/634568v3" class="twitter-share-button service-links-twitter-widget" id="twitter_widget" title="Tweet This" rel="nofollow" data-icon-position="" data-hide-link-title="0"><span class="element-invisible">Tweet Widget</span></a></li><li><a href="http://www.facebook.com/plugins/like.php?href=https%3A//www.biorxiv.org/content/10.1101/634568v3&amp;layout=button_count&amp;show_faces=false&amp;action=like&amp;colorscheme=light&amp;width=100&amp;height=21&amp;font=&amp;locale=" id="facebook_like" title="I Like it" class="service-links-facebook-like" rel="nofollow" data-icon-position="" data-hide-link-title="0"><span class="element-invisible">Facebook Like</span></a></li><li class="last"><a href="https://www.biorxiv.org/content/10.1101/634568v3" id="google_plus_one" title="Plus it" class="service-links-google-plus-one" rel="nofollow" data-icon-position="" data-hide-link-title="0"><span class="element-invisible">Google Plus One</span></a></li></ul></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-collections" >
  
        <h2 class="pane-title">Subject Area</h2>
    
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-collections"><div class="highwire-list"><ul class="highwire-article-collection-term-list"><li class="first last odd"><span class="highwire-article-collection-term"><a href="/collection/ecology" class="highlight" data-icon-position="" data-hide-link-title="0">Ecology<i class="icon-caret-right"></i>
</a></span></li></ul></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-biorxiv-subject-collections block-style-col2" >
  
      
  
  <div class="pane-content">
    <div class="panel-flexible panels-flexible-8 clearfix" id="mini-panel-biorxiv_subject_collections">
<div class="panel-flexible-inside panels-flexible-8-inside">
<div class="panels-flexible-region panels-flexible-region-8-center panels-flexible-region-first panels-flexible-region-last">
  <div class="inside panels-flexible-region-inside panels-flexible-region-8-center-inside panels-flexible-region-inside-first panels-flexible-region-inside-last">
<div class="panel-pane pane-snippet" >
  
      
  
  <div class="pane-content">
    <div class="snippet biorxiv-subject-areas-table-title" id="biorxiv-subject-areas-table-title">
  
      
  <div class="snippet-content">
    <b>Subject Areas</b>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-snippet" >
  
      
  
  <div class="pane-content">
    <div class="snippet biorxiv-subject-areas-view-papers" id="biorxiv-subject-areas-view-papers">
  
      
  <div class="snippet-content">
    <a href="/content/early/recent"><strong>All Articles</strong></a>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-subject-collections" >
  
      
  
  <div class="pane-content">
    <ul id="collection" class="collection highwire-list-expand"><li class="outer collection depth-2  child first"><div class = "data-wrapper"><a href="/collection/animal-behavior-and-cognition" class="" data-icon-position="" data-hide-link-title="0">Animal Behavior and Cognition</a> <span class = "article-count">(1674)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/biochemistry" class="" data-icon-position="" data-hide-link-title="0">Biochemistry</a> <span class = "article-count">(2825)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/bioengineering" class="" data-icon-position="" data-hide-link-title="0">Bioengineering</a> <span class = "article-count">(1963)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/bioinformatics" class="" data-icon-position="" data-hide-link-title="0">Bioinformatics</a> <span class = "article-count">(10456)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/biophysics" class="" data-icon-position="" data-hide-link-title="0">Biophysics</a> <span class = "article-count">(4275)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/cancer-biology" class="" data-icon-position="" data-hide-link-title="0">Cancer Biology</a> <span class = "article-count">(3301)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/cell-biology" class="" data-icon-position="" data-hide-link-title="0">Cell Biology</a> <span class = "article-count">(4686)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/clinical-trials" class="" data-icon-position="" data-hide-link-title="0">Clinical Trials</a> <span class = "article-count">(136)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/developmental-biology" class="" data-icon-position="" data-hide-link-title="0">Developmental Biology</a> <span class = "article-count">(2911)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/ecology" class="" data-icon-position="" data-hide-link-title="0">Ecology</a> <span class = "article-count">(4575)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/epidemiology" class="" data-icon-position="" data-hide-link-title="0">Epidemiology</a> <span class = "article-count">(2041)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/evolutionary-biology" class="" data-icon-position="" data-hide-link-title="0">Evolutionary Biology</a> <span class = "article-count">(7337)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/genetics" class="" data-icon-position="" data-hide-link-title="0">Genetics</a> <span class = "article-count">(5574)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/genomics" class="" data-icon-position="" data-hide-link-title="0">Genomics</a> <span class = "article-count">(6939)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/immunology" class="" data-icon-position="" data-hide-link-title="0">Immunology</a> <span class = "article-count">(2463)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/microbiology" class="" data-icon-position="" data-hide-link-title="0">Microbiology</a> <span class = "article-count">(7697)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/molecular-biology" class="" data-icon-position="" data-hide-link-title="0">Molecular Biology</a> <span class = "article-count">(3078)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/neuroscience" class="" data-icon-position="" data-hide-link-title="0">Neuroscience</a> <span class = "article-count">(19031)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/paleontology" class="" data-icon-position="" data-hide-link-title="0">Paleontology</a> <span class = "article-count">(142)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/pathology" class="" data-icon-position="" data-hide-link-title="0">Pathology</a> <span class = "article-count">(481)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/pharmacology-and-toxicology" class="" data-icon-position="" data-hide-link-title="0">Pharmacology and Toxicology</a> <span class = "article-count">(796)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/physiology" class="" data-icon-position="" data-hide-link-title="0">Physiology</a> <span class = "article-count">(1188)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/plant-biology" class="" data-icon-position="" data-hide-link-title="0">Plant Biology</a> <span class = "article-count">(2805)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/scientific-communication-and-education" class="" data-icon-position="" data-hide-link-title="0">Scientific Communication and Education</a> <span class = "article-count">(699)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/synthetic-biology" class="" data-icon-position="" data-hide-link-title="0">Synthetic Biology</a> <span class = "article-count">(919)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/systems-biology" class="" data-icon-position="" data-hide-link-title="0">Systems Biology</a> <span class = "article-count">(2898)</span></div></li>
<li class="outer collection depth-2  child last"><div class = "data-wrapper"><a href="/collection/zoology" class="" data-icon-position="" data-hide-link-title="0">Zoology</a> <span class = "article-count">(473)</span></div></li>
</ul>  </div>

  
  </div>
  </div>
</div>
</div>
</div>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__VNH5GD8Zz7g6_hGCOZjVjdMndxxC6naiExSSJKX3k_A__kzmEdtbqCsJx5Z9yg2Qy0PptB__ufXm-TxUOl0KZzUw__dPhHATyREQrnntSIgxU8XNoYny4rjInxP7lLKSNxQyY.js"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__dPhHATyREQrnntSIgxU8XNoYny4rjInxP7lLKSNxQyY.js" defer="defer"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__llgGUcz2-k3Jmyyyog-_4yI1wJfGWwx0eQ2lbnM9IIo__t6UjIjIKIAD6nKc6BAU9qmlVV2VR4CBTNQbxlbJPgN0__dPhHATyREQrnntSIgxU8XNoYny4rjInxP7lLKSNxQyY.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
function euCookieComplianceLoadScripts() {}
//--><!]]>
</script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var eu_cookie_compliance_cookie_name = "";
//--><!]]>
</script>
<script type="text/javascript" src="//d33xdlntwy0kbs.cloudfront.net/cshl_custom.js"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__BmqjBnkz3MgYCAoc25s1lDRMEjLhC3mEPVonUFIHi08__Unwv5-ZIuHBfFwytsjEx1niBVJ7n1T4lPws7VrkHXM4__dPhHATyREQrnntSIgxU8XNoYny4rjInxP7lLKSNxQyY.js"></script>
  </body>
</html>
