<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<link rel="dns-prefetch" href="//d33xdlntwy0kbs.cloudfront.net" />
<link rel="shortcut icon" href="https://www.biorxiv.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<meta name="article_thumbnail" content="https://www.biorxiv.org/content/biorxiv/early/2019/03/14/574798/embed/inline-graphic-1.gif" />
<meta name="type" content="article" />
<meta name="category" content="article" />
<meta name="HW.identifier" content="/biorxiv/early/2019/03/14/574798.atom" />
<meta name="HW.pisa" content="biorxiv;574798v2" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="Uncovering mechanisms of global ocean change effects on the Dungeness crab (Cancer magister) through metabolomics analysis" />
<meta name="DC.Identifier" content="10.1101/574798" />
<meta name="DC.Date" content="2019-03-14" />
<meta name="DC.Publisher" content="Cold Spring Harbor Laboratory" />
<meta name="DC.Rights" content="© 2019, Posted by Cold Spring Harbor Laboratory. This article is a US Government work. It is not subject to copyright under 17 USC 105 and is also made available for use under a CC0 license" />
<meta name="DC.AccessRights" content="restricted" />
<meta name="DC.Description" content="The Dungeness crab is an economically and ecologically important species distributed along the North American Pacific coast. To predict how Dungeness crab may physiologically respond to future global ocean change on a molecular level, we performed untargeted metabolomic approaches on individual Dungeness crab juveniles reared in treatments that mimicked current and projected future pH and dissolved oxygen conditions. We found 94 metabolites and 127 lipids responded in a condition-specific manner, with a greater number of known compounds more strongly responding to low oxygen than low pH exposure. Pathway analysis of these compounds revealed that juveniles may respond to low oxygen through evolutionarily conserved processes including downregulating glutathione biosynthesis and upregulating glycogen storage, and may respond to low pH by increasing ATP production. Most interestingly, we found that the response of juveniles to combined low pH and low oxygen exposure was most similar to the low oxygen exposure response, indicating low oxygen may drive the physiology of juvenile crabs more than pH. Our study elucidates metabolic dynamics that expand our overall understanding of how the species might respond to future ocean conditions and provides a comprehensive dataset that could be used in future ocean acidification response studies." />
<meta name="DC.Contributor" content="Shelly A. Trigg" />
<meta name="DC.Contributor" content="Paul McElhany" />
<meta name="DC.Contributor" content="Michael Maher" />
<meta name="DC.Contributor" content="Danielle Perez" />
<meta name="DC.Contributor" content="D. Shallin Busch" />
<meta name="DC.Contributor" content="Krista M. Nichols" />
<meta name="article:published_time" content="2019-03-14" />
<meta name="article:section" content="New Results" />
<meta name="citation_title" content="Uncovering mechanisms of global ocean change effects on the Dungeness crab (Cancer magister) through metabolomics analysis" />
<meta name="citation_abstract" lang="en" content="&lt;h3&gt;ABSTRACT&lt;/h3&gt;
&lt;p&gt;The Dungeness crab is an economically and ecologically important species distributed along the North American Pacific coast. To predict how Dungeness crab may physiologically respond to future global ocean change on a molecular level, we performed untargeted metabolomic approaches on individual Dungeness crab juveniles reared in treatments that mimicked current and projected future pH and dissolved oxygen conditions. We found 94 metabolites and 127 lipids responded in a condition-specific manner, with a greater number of known compounds more strongly responding to low oxygen than low pH exposure. Pathway analysis of these compounds revealed that juveniles may respond to low oxygen through evolutionarily conserved processes including downregulating glutathione biosynthesis and upregulating glycogen storage, and may respond to low pH by increasing ATP production. Most interestingly, we found that the response of juveniles to combined low pH and low oxygen exposure was most similar to the low oxygen exposure response, indicating low oxygen may drive the physiology of juvenile crabs more than pH. Our study elucidates metabolic dynamics that expand our overall understanding of how the species might respond to future ocean conditions and provides a comprehensive dataset that could be used in future ocean acidification response studies.&lt;/p&gt;" />
<meta name="citation_journal_title" content="bioRxiv" />
<meta name="citation_publisher" content="Cold Spring Harbor Laboratory" />
<meta name="citation_publication_date" content="2019/01/01" />
<meta name="citation_mjid" content="biorxiv;574798v2" />
<meta name="citation_id" content="574798v2" />
<meta name="citation_public_url" content="https://www.biorxiv.org/content/10.1101/574798v2" />
<meta name="citation_abstract_html_url" content="https://www.biorxiv.org/content/10.1101/574798v2.abstract" />
<meta name="citation_full_html_url" content="https://www.biorxiv.org/content/10.1101/574798v2.full" />
<meta name="citation_pdf_url" content="https://www.biorxiv.org/content/biorxiv/early/2019/03/14/574798.full.pdf" />
<meta name="citation_doi" content="10.1101/574798" />
<meta name="citation_num_pages" content="24" />
<meta name="citation_article_type" content="Article" />
<meta name="citation_section" content="New Results" />
<meta name="citation_firstpage" content="574798" />
<meta name="citation_author" content="Shelly A. Trigg" />
<meta name="citation_author_institution" content="Division of Biological Sciences, Cell and Developmental Biology Section, University of California San Diego" />
<meta name="citation_author_institution" content="Genomic Analysis Laboratory, The Salk Institute for Biological Studies" />
<meta name="citation_author_institution" content="School of Aquatic and Fishery Sciences, University of Washington" />
<meta name="citation_author_email" content="strigg@uw.edu" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0001-6904-4149" />
<meta name="citation_author" content="Paul McElhany" />
<meta name="citation_author_institution" content="Conservation Biology Division, Northwest Fisheries Science Center, National Marine Fisheries Service, National Oceanic and Atmospheric Administration" />
<meta name="citation_author" content="Michael Maher" />
<meta name="citation_author_institution" content="Conservation Biology Division, Northwest Fisheries Science Center, National Marine Fisheries Service, National Oceanic and Atmospheric Administration" />
<meta name="citation_author" content="Danielle Perez" />
<meta name="citation_author_institution" content="Conservation Biology Division, Northwest Fisheries Science Center, National Marine Fisheries Service, National Oceanic and Atmospheric Administration" />
<meta name="citation_author" content="D. Shallin Busch" />
<meta name="citation_author_institution" content="Conservation Biology Division, Northwest Fisheries Science Center, National Marine Fisheries Service, National Oceanic and Atmospheric Administration" />
<meta name="citation_author_institution" content="Ocean Acidification Program, Office of Oceanic and Atmospheric Research and Northwest Fisheries Science Center, National Marine Fisheries Service, National Oceanic and Atmospheric Administration" />
<meta name="citation_author" content="Krista M. Nichols" />
<meta name="citation_author_institution" content="Conservation Biology Division, Northwest Fisheries Science Center, National Marine Fisheries Service, National Oceanic and Atmospheric Administration" />
<meta name="citation_reference" content="Le Quéré, C. et al. Global Carbon Budget 2016. Earth Syst. Sci. Data 8, 605–649 (2016)." />
<meta name="citation_reference" content="Caldeira, K. &amp; Wickett, M. E. Anthropogenic carbon and ocean pH. Nature 425, 365 (2003)." />
<meta name="citation_reference" content="Feely, R. A. et al. Impact of anthropogenic CO2 on the CaCO3 system in the oceans. Science. 305, 362–366 (2004)." />
<meta name="citation_reference" content="Gobler, C. J. &amp; Baumann, H. Hypoxia and acidification in ocean ecosystems: coupled dynamics and effects on marine life. Biol. Lett. 12, (2016)." />
<meta name="citation_reference" content="National Marine Fisheries Service. Fisheries of the United States. Fisheries of the United States, 2016. (2017)." />
<meta name="citation_reference" content="Marshall, K. N. et al. Risks of ocean acidification in the California Current food web and fisheries: ecosystem model projections. Glob. Chang. Biol. 23, 1525–1539 (2017)." />
<meta name="citation_reference" content="Hunter, K. C. &amp; Rudy, P. P. Osmotic and ionic regulation in the Dungeness crab, Cancer magister dana. Comp. Biochem. Physiol. Part A 51, 439–447 (1975)." />
<meta name="citation_reference" content="Engelhardt, F. R. &amp; Dehnel, P. A. Ionic regulation in the Pacific edible crab, Cancer magister (Dana). Can. J. Zool. 51, 735–743 (1973)." />
<meta name="citation_reference" content="Freire, C. A., Onken, H. &amp; McNamara, J. C. A structure-function analysis of ion transport in crustacean gills and excretory organs. Comparative Biochemistry and Physiology - A Molecular and Integrative Physiology 151, 272–304 (2008)." />
<meta name="citation_reference" content="Jones, L. L. Osmotic regulation in several crabs of the pacific coast of north america. J. Cell. Comp. Physiol. 18, 79–92 (1941)." />
<meta name="citation_reference" content="Henry, R. P. &amp; Wheatly, M. G. Interaction of respiration, ion regulation, and acid-base balance in the everyday life of aquatic crustaceans. Integr. Comp. Biol. 32, 407–416 (1992)." />
<meta name="citation_reference" content="Hans, S., Fehsenfeld, S., Treberg, J. R. &amp; Weihrauch, D. Acid-base regulation in the Dungeness crab (Metacarcinus magister). Mar. Biol. 161, 1179–1193 (2014)." />
<meta name="citation_reference" content="Miller, J. J., Maher, M., Bohaboy, E., Friedman, C. S. &amp; McElhany, P. Exposure to low pH reduces survival and delays development in early life stages of Dungeness crab (Cancer magister). Mar. Biol. 163, 118 (2016)." />
<meta name="citation_reference" content="Bernatis, J. L., Gerstenberger, S. L. &amp; McGaw, I. J. Behavioural responses of the Dungeness crab, Cancer magister, during feeding and digestion in hypoxic conditions. MaiBiol. 150, 941–951 (2007)." />
<meta name="citation_reference" content="Airriess, C. &amp; Mcmahon, B. Cardiovascular adaptations enhance tolerance of environmental hypoxia in the crab Cancer magister. J. Exp. Biol. 190, (1994)." />
<meta name="citation_reference" content="Turi, K. N., Romick-Rosendale, L., Ryckman, K. K. &amp; Hartert, T. V. A review of metabolomics approaches and their application in identifying causal pathways of childhood asthma. Journal of Allergy and Clinical Immunology (2016). doi:10.1016/j.jaci.2017.04.021" />
<meta name="citation_reference" content="Lankadurai, B. P., Nagato, E. G. &amp; Simpson, M. J. Environmental metabolomics: an emerging approach to study organism responses to environmental stressors. Environ. Rev. 21, 180–205 (2013)." />
<meta name="citation_reference" content="Sogin, E. M., Putnam, H. M., Anderson, P. E. &amp; Gates, R. D. Metabolomic signatures of increases in temperature and ocean acidification from the reef-building coral, Pocillopora damicornis. Metabolomics 12, (2016)." />
<meta name="citation_reference" content="Cajka, T., Smilowitz, J. T. &amp; Fiehn, O. Validating Quantitative Untargeted Lipidomics Across Nine Liquid Chromatography–High-Resolution Mass Spectrometry Platforms. Anal. Chem. 89, 12360–12368 (2017)." />
<meta name="citation_reference" content="Kind, T. et al. FiehnLib: mass spectral and retention index libraries for metabolomics based on quadrupole and time-of-flight gas chromatography/mass spectrometry. Anal. Chem. 81, 10038–48 (2009)." />
<meta name="citation_reference" content="Kind, T. et al. LipidBlast in silico tandem mass spectrometry database for lipid identification. Nat. Methods 10, 755–8 (2013)." />
<meta name="citation_reference" content="Skogerson, K., Wohlgemuth, G., Barupal, D. K. &amp; Fiehn, O. The volatile compound BinBase mass spectral database. BMC Bioinformatics 12, 321 (2011)." />
<meta name="citation_reference" content="Wishart, D. S. et al. HMDB 4.0: the human metabolome database for 2018. Nucleic Acids Res. 46, D608–D617 (2018)." />
<meta name="citation_reference" content="Fahy, E., Sud, M., Cotter, D. &amp; Subramaniam, S. LIPID MAPS online tools for lipid research. Nucleic Acids Res. 35, W606–W612 (2007)." />
<meta name="citation_reference" content="Allen, W. V. Lipid transport in the dungeness crab Cancer magister dana. Comp. Biochem. Physiol. Part B Comp. Biochem. 43, 193–IN8 (1972)." />
<meta name="citation_reference" content="Allen, W. V. Amino Acid and Fatty Acid Composition of Tissues of the Dungeness Crab (Cancer magister). J. Fish. Res. Board Canada 28, 1191–1195 (1971)." />
<meta name="citation_reference" content="Grissa, D. et al. Feature Selection Methods for Early Predictive Biomarker Discovery Using Untargeted Metabolomic Data. Front. Mol. Biosci. 3, 30 (2016)." />
<meta name="citation_reference" content="Witte, R. S. &amp; Witte, J. S. Statistics. (J. Wiley &amp; Sons, 2010)." />
<meta name="citation_reference" content="Vinaixa, M. et al. A Guideline to Univariate Statistical Analysis for LC/MS-Based Untargeted Metabolomics-Derived Data. Metabolites 2, 775–95 (2012)." />
<meta name="citation_reference" content="Timmerman, M. E., Hoefsloot, H. C. J., Smilde, A. K. &amp; Ceulemans, E. Scaling in ANOVA-simultaneous component analysis. Metabolomics 11, 1265–1276 (2015)." />
<meta name="citation_reference" content="Broquist, H. P. Lysine-Pipecolic Acid Metabolic Relationships in Microbes and Mammals. Annu. Rev. Nutr. 11, 435–448 (1991)." />
<meta name="citation_reference" content="Cheng, J. et al. Identification of 2-piperidone as a biomarker of CYP2E1 activity through metabolomic phenotyping. Toxicol. Sci. 135, 37–47 (2013)." />
<meta name="citation_reference" content="Orlowski, M. &amp; Meister, A. The gamma-glutamyl cycle: a possible transport system for amino acids. Proc. Natl. Acad. Sci. U. S. A. 67, 1248–55 (1970)." />
<meta name="citation_reference" content="Bodnaryk, R. P. Structure and Function of Insect Peptides. Adv. In Insect Phys. 13, 69–132 (1978)." />
<meta name="citation_reference" content="Mika, A., Skorkowski, E. &amp; Stepnowski, P. The Use of Different MS Techniques to Determine Glutathione Levels in Marine Tissues. Food Anal. Methods 6, 789–802 (2013)." />
<meta name="citation_reference" content="Pelletier, J. et al. Glycogen Synthesis is Induced in Hypoxia by the Hypoxia-Inducible Factor and Promotes Cancer Cell Survival. Front Oncol 2, 18 (2012)." />
<meta name="citation_reference" content="Christman, A. A. Purine and pyrimidine metabolism. Physiol. Rev. 32, 303–48 (1952)." />
<meta name="citation_reference" content="Winefield, C. S., Farnden, K. J. F., Reynolds, P. H. S. &amp; Marshall, C. J. Evolutionary analysis of aspartate aminotransferases. J. Mol. Evol. 40, 455–463 (1995)." />
<meta name="citation_reference" content="Scher, W. &amp; Jakoby, W. B. Maleate isomerase. J. Biol. Chem. 244, 1878–1882 (1969)." />
<meta name="citation_reference" content="Mamer, O. et al. The complete targeted profile of the organic acid intermediates of the citric acid cycle using a single stable isotope dilution analysis, sodium borodeuteride reduction and selected ion monitoring GC/MS. Metabolomics 9, 1019–1030 (2013)." />
<meta name="citation_reference" content="Engelking, L. R. &amp; Engelking, L. R. in Textbook of Veterinary Physiological Chemistry 208–213 (Elsevier, 2015). doi:10.1016/B978-0-12-391909-0.50034-7" />
<meta name="citation_reference" content="Menon, G. K. K., Friedman, D. L. &amp; Stern, J. R. Enzymic synthesis of glutaryl-coenzyme A. Biochim. Biophys. Acta 44, 375–377 (1960)." />
<meta name="citation_reference" content="Sauer, S. W. et al. Bioenergetics in glutaryl-coenzyme A dehydrogenase deficiency: a role for glutaryl-coenzyme A. J. Biol. Chem. 280, 21830–6 (2005)." />
<meta name="citation_reference" content="Gibellini, F. &amp; Smith, T. K. The Kennedy pathway-De novo synthesis of phosphatidylethanolamine and phosphatidylcholine. IUBMB Life 62, n/a-n/a (2010)." />
<meta name="citation_reference" content="Mathews, C. K., Van Holde, K. E. (Kensal E. &amp; Ahern, K. G. Biochemistry. (Benjamin Cummings, 2000)." />
<meta name="citation_reference" content="Courtney-Martin, G. &amp; Pencharz, P. B. in The Molecular Nutrition of Amino Acids and Proteins 265–286 (Elsevier, 2016). doi:10.1016/B978-0-12-802167-5.00019-0" />
<meta name="citation_reference" content="Jackson, T. M. &amp; O’Malley, K. G. Comparing genetic connectivity among Dungeness crab (Cancer magister) inhabiting Puget Sound and coastal Washington. Mar. Biol. 164, (2017)." />
<meta name="citation_reference" content="Jackson, T. M., Roegner, G. C. &amp; O’Malley, K. G. Evidence for interannual variation in genetic structure of Dungeness crab (Cancer magister) along the California Current System. Mol. Ecol. (2018). doi:10.1111/mec.14443" />
<meta name="citation_reference" content="Chong, E. Y. et al. Local false discovery rate estimation using feature reliability in LC/MS metabolomics data. Sci. Rep. 5, 17221 (2015)." />
<meta name="citation_reference" content="Ding, M.-Z. et al. Proteomic Research Reveals the Stress Response and Detoxification of Yeast to Combined Inhibitors. PLoS One 7, e43474 (2012)." />
<meta name="citation_reference" content="Harding, H. P. et al. An Integrated Stress Response Regulates Amino Acid Metabolism and Resistance to Oxidative Stress. Mol. Cell 11, 619–633 (2003)." />
<meta name="citation_reference" content="Murray, A. J. Metabolic adaptation of skeletal muscle to high altitude hypoxia: How new technologies could resolve the controversies. Genome Med. 1, (2009)." />
<meta name="citation_reference" content="Nava, G. M., Lee, D. Y., Ospina, J. H., Cai, S.-Y. &amp; Gaskins, H. R. Genomic analyses reveal a conserved glutathione homeostasis pathway in the invertebrate chordate Ciona intestinalis. Physiol. Genomics 39, 183–94 (2009)." />
<meta name="citation_reference" content="Fraser, J. A., Saunders, R. D. C. &amp; McLellan, L. I. Drosophila melanogaster glutamate-cysteine ligase activity is regulated by a modifier subunit with a mechanism of action similar to that of the mammalian form. J. Biol. Chem. 277, 1158–65 (2002)." />
<meta name="citation_reference" content="Mansfield, K. D., Simon, M. C. &amp; Keith, B. Hypoxic reduction in cellular glutathione levels requires mitochondrial reactive oxygen species. J. Appl. Physiol. 97, 1358–1366 (2004)." />
<meta name="citation_reference" content="Cargnoni, A., Ceconi, C., Gaia, G., Agnoletti, L. &amp; Ferrari, R. Cellular thiols redox status: A switch for NF-κB activation during myocardial post-ischaemic reperfusion. J. Mol. Cell. Cardiol. 34, 997–1005 (2002)." />
<meta name="citation_reference" content="Murata, Y., Ohteki, T., Koyasu, S. &amp; Hamuro, J. IFN-gamma and pro-inflammatory cytokine production by antigen-presenting cells is dictated by intracellular thiol redox status regulated by oxygen tension. Eur. J. Immunol. 32, 2866–2873 (2002)." />
<meta name="citation_reference" content="Rajpurohit, R., Koch, C. J., Tao, Z., Teixeira, C. M. &amp; Shapiro, I. M. Adaptation of chondrocytes to low oxygen tension: Relationship between hypoxia and cellular metabolism. J. Cell. Physiol. 168, 424–432 (1996)." />
<meta name="citation_reference" content="Pescador, N. et al. Hypoxia Promotes Glycogen Accumulation through Hypoxia Inducible Factor (HIF)-Mediated Induction of Glycogen Synthase 1. PLoS One 5, e9644 (2010)." />
<meta name="citation_reference" content="Shen, G. M., Zhang, F. L., Liu, X. L. &amp; Zhang, J. W. Hypoxia-inducible factor 1-mediated regulation of PPP1R3C promotes glycogen accumulation in human MCF-7 cells under hypoxia. FEBS Lett. 584, 4366–4372 (2010)." />
<meta name="citation_reference" content="Phlippen, M. K., Webster, S. G., Chung, J. S. &amp; Dircksen, H. Ecdysis of decapod crustaceans is associated with a dramatic release of crustacean cardioactive peptide into the haemolymph. J. Exp. Biol. 203, 521–36 (2000)." />
<meta name="citation_reference" content="Kuballa, A. V, Holton, T. A., Paterson, B. &amp; Elizur, A. Moult cycle specific differential gene expression profiling of the crab Portunus pelagicus. BMC Genomics 12, 147 (2011)." />
<meta name="citation_reference" content="Fiehn, O. &amp; Kind, T. in 3–17 (Humana Press, 2007). doi:10.1007/978-1-59745-244-1_1" />
<meta name="citation_reference" content="Matyash, V., Liebisch, G., Kurzchalia, T. V., Shevchenko, A. &amp; Schwudke, D. Lipid extraction by methyl- tert -butyl ether for high-throughput lipidomics. J. Lipid Res. 49, 1137–1146 (2008)." />
<meta name="citation_reference" content="Fiehn, O., Wohlgemuth, G. &amp; Scholz, M. in 224–239 (Springer, Berlin, Heidelberg, 2005). doi:10.1007/11530084_18" />
<meta name="citation_reference" content="Fiehn, O. et al. Quality control for plant metabolomics: reporting MSI-compliant studies. Plant J. 53, 691–704 (2008)." />
<meta name="citation_reference" content="Breiman, L. &amp; Cutler, A. Breiman and Cutler’s random forests for classification and regression. Packag. ‘randomForest’ 29 (2012). doi:10.5244/C.22.54" />
<meta name="citation_reference" content="Wohlgemuth, G., Haldiya, P. K., Willighagen, E., Kind, T. &amp; Fiehn, O. The chemical translation service-a web-based tool to improve standardization of metabolomic reports. Bioinformatics 26, 2647–2648 (2010)." />
<meta name="citation_reference" content="Kim, S. et al. PubChem substance and compound databases. Nucleic Acids Res. 44, D1202–D1213 (2016)." />
<meta name="citation_reference" content="Barupal, D. K. et al. MetaMapp: mapping and visualizing metabolomic data by integrating information from biochemical pathways and chemical and mass spectral similarity. BMC Bioinformatics 13, (2012)." />
<meta name="citation_reference" content="Shannon, P. et al. Cytoscape: a software environment for integrated models of biomolecular interaction networks. Genome Res. 13, 2498–504 (2003)." />
<meta name="citation_reference" content="Torchiano, M. Efficient Effect Size Computation [R package effsize version 0.7.1]." />
<meta name="twitter:title" content="Uncovering mechanisms of global ocean change effects on the Dungeness crab (Cancer magister) through metabolomics analysis" />
<meta name="twitter:site" content="@biorxivpreprint" />
<meta name="twitter:card" content="summary" />
<meta name="twitter:image" content="https://www.biorxiv.org/sites/default/files/images/biorxiv_logo_homepage7-5-small.png" />
<meta name="twitter:description" content="The Dungeness crab is an economically and ecologically important species distributed along the North American Pacific coast. To predict how Dungeness crab may physiologically respond to future global ocean change on a molecular level, we performed untargeted metabolomic approaches on individual Dungeness crab juveniles reared in treatments that mimicked current and projected future pH and dissolved oxygen conditions. We found 94 metabolites and 127 lipids responded in a condition-specific manner, with a greater number of known compounds more strongly responding to low oxygen than low pH exposure. Pathway analysis of these compounds revealed that juveniles may respond to low oxygen through evolutionarily conserved processes including downregulating glutathione biosynthesis and upregulating glycogen storage, and may respond to low pH by increasing ATP production. Most interestingly, we found that the response of juveniles to combined low pH and low oxygen exposure was most similar to the low oxygen exposure response, indicating low oxygen may drive the physiology of juvenile crabs more than pH. Our study elucidates metabolic dynamics that expand our overall understanding of how the species might respond to future ocean conditions and provides a comprehensive dataset that could be used in future ocean acidification response studies." />
<meta name="og-title" property="og:title" content="Uncovering mechanisms of global ocean change effects on the Dungeness crab (Cancer magister) through metabolomics analysis" />
<meta name="og-url" property="og:url" content="https://www.biorxiv.org/content/10.1101/574798v2" />
<meta name="og-site-name" property="og:site_name" content="bioRxiv" />
<meta name="og-description" property="og:description" content="The Dungeness crab is an economically and ecologically important species distributed along the North American Pacific coast. To predict how Dungeness crab may physiologically respond to future global ocean change on a molecular level, we performed untargeted metabolomic approaches on individual Dungeness crab juveniles reared in treatments that mimicked current and projected future pH and dissolved oxygen conditions. We found 94 metabolites and 127 lipids responded in a condition-specific manner, with a greater number of known compounds more strongly responding to low oxygen than low pH exposure. Pathway analysis of these compounds revealed that juveniles may respond to low oxygen through evolutionarily conserved processes including downregulating glutathione biosynthesis and upregulating glycogen storage, and may respond to low pH by increasing ATP production. Most interestingly, we found that the response of juveniles to combined low pH and low oxygen exposure was most similar to the low oxygen exposure response, indicating low oxygen may drive the physiology of juvenile crabs more than pH. Our study elucidates metabolic dynamics that expand our overall understanding of how the species might respond to future ocean conditions and provides a comprehensive dataset that could be used in future ocean acidification response studies." />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.biorxiv.org/sites/default/files/images/biorxiv_logo_homepage7-5-small.png" />
<meta name="citation_date" content="2019-03-14" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/10.1101/574798v2.full.pdf" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/10.1101/574798v2.full.txt" />
<link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/10.1101/574798v2.ppt" />
<meta name="description" content="bioRxiv - the preprint server for biology, operated by Cold Spring Harbor Laboratory, a research and educational institution" />
<meta name="generator" content="Drupal 7 (http://drupal.org)" />
<link rel="canonical" href="https://www.biorxiv.org/content/10.1101/574798v2" />
<link rel="shortlink" href="https://www.biorxiv.org/node/249808" />
    <title>Uncovering mechanisms of global ocean change effects on the Dungeness crab (Cancer magister) through metabolomics analysis | bioRxiv</title>  
    <link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__jMRAK66KMC1e4TQlwUNn3KiWVDC5AjueUAYEm1xBY_U__KSUjdT4jdcJ4qJz6fKY1K9WkYh1a5EcaWZxt_-zbTis__QJacVmveFyLIBSPkp21xL0ZvuDW9DMBrD2HB1P4Ry5I.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q5p1rr" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__icrP-Y0qw2NP73rJxxkUtrJta0z3VLAABtNjfSdEghA__rlN3rx0wOoswGNHZMeZM96bNn-3cLWHXTztGDoThNlw__QJacVmveFyLIBSPkp21xL0ZvuDW9DMBrD2HB1P4Ry5I.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__q48SDxVDWut_cH6dTPyQe7HjADuSLLmJVFztuiJEFrU__DJqddQGjm5dXCHsaiwNriGytKN5eevIQqyjdq65TZtE__QJacVmveFyLIBSPkp21xL0ZvuDW9DMBrD2HB1P4Ry5I.css" media="all" />
<style type="text/css" media="all">
/* <![CDATA[ */
#sliding-popup.sliding-popup-bottom,#sliding-popup.sliding-popup-bottom .eu-cookie-withdraw-banner,.eu-cookie-withdraw-tab{background:gray}#sliding-popup.sliding-popup-bottom.eu-cookie-withdraw-wrapper{background:transparent}#sliding-popup .popup-content #popup-text h1,#sliding-popup .popup-content #popup-text h2,#sliding-popup .popup-content #popup-text h3,#sliding-popup .popup-content #popup-text p,.eu-cookie-compliance-secondary-button,.eu-cookie-withdraw-tab{color:#fff !important}.eu-cookie-withdraw-tab{border-color:#fff}.eu-cookie-compliance-more-button{color:#fff !important}
/* ]]> */
</style>

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__QJacVmveFyLIBSPkp21xL0ZvuDW9DMBrD2HB1P4Ry5I.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__B6tGdsQSt1vT8d8j-19-oS4FKJe83RrSNBkjPs9jfos__6VkE5WcQmmQpSVTQx-xS2-zn_nlwFlDm-EN1HLJNjrA__QJacVmveFyLIBSPkp21xL0ZvuDW9DMBrD2HB1P4Ry5I.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__I6pBwZrmO51afTZVOjN83FdGH1QRb9t15lSLso0JVV4__F3sVKlGyHKCaAPI_WdiHD7l_-05B6b9bLdUwzi1me90__QJacVmveFyLIBSPkp21xL0ZvuDW9DMBrD2HB1P4Ry5I.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__QJacVmveFyLIBSPkp21xL0ZvuDW9DMBrD2HB1P4Ry5I.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__QJacVmveFyLIBSPkp21xL0ZvuDW9DMBrD2HB1P4Ry5I.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__CRdjIL0HKDL2T9slYvkTSTLypBBPjyoDhwBezBuDy7c__cHebPmPNdg2jyPmpyQpZWssJQ4cPrqfaabp5Or9KlkU__QJacVmveFyLIBSPkp21xL0ZvuDW9DMBrD2HB1P4Ry5I.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://d33xdlntwy0kbs.cloudfront.net/cshl_custom.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/all/libraries/modernizr/modernizr.min.js?q5p1rr"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__yOhrJUBmMpN-jsuQlJYmS8ovkQU6I6KInr295HrSTb0__c--zHhbuOWmkdWkJYXiNWDmkpDRPesmbBjGlQcyqBOY__QJacVmveFyLIBSPkp21xL0ZvuDW9DMBrD2HB1P4Ry5I.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var _prum=[['id', '52e95df3abe53d2e33000000'], ['mark', 'firstbyte',
      (new Date()).getTime()]]; (function() {
      var s=document.getElementsByTagName('script')[0],
      p=document.createElement('script');
      p.async='async'; p.src='//rum-static.pingdom.net/prum.min.js';s.parentNode.insertBefore(p,s);})();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js___KyvuEvfmewWxdOLBXTYsSUo8Pfm_DijV6fWtGvSL_k__HSKkTgIr7nDcdvQYewipf6v4ak6ZLUOQkwPj44wKouA__QJacVmveFyLIBSPkp21xL0ZvuDW9DMBrD2HB1P4Ry5I.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__3uCzzYE5jr6ly9nvsM8PRBRkhh4p0NWCMDja4EKBy4M__dmGGVYcB8LhnbR2iAmVW0ZyXI6QuFVYFRORa5XxglXs__QJacVmveFyLIBSPkp21xL0ZvuDW9DMBrD2HB1P4Ry5I.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__0zdUME0GLauYYyEch2tGPydIw-IG7Uizip0Qb4wUsig__3RgszrCgk4bOnTzo4vaQ-hrWzaG6oD19l-R9CQ1YQ-o__QJacVmveFyLIBSPkp21xL0ZvuDW9DMBrD2HB1P4Ry5I.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__W6wAjopaALytcBam7oT3WhEzllkWaB9R-8hazHyvShM__rK5YBIqBy9qt0di0Sn4DriBx9TJupdmv6uGmLJyZvDs__QJacVmveFyLIBSPkp21xL0ZvuDW9DMBrD2HB1P4Ry5I.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-45638026-1", {"cookieDomain":"auto"});ga("set", "page", location.pathname + location.search + location.hash);ga("send", "pageview");ga('create', 'UA-189672-38', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__1PtAOZ8x7l6QAi8D7X6Kj_Y7HcUOFzvWRvvYHUcCT98__U8IB92u_CnnGwJ-X_xaUa9lzsC8OzUfN0WkK2JQfirg__QJacVmveFyLIBSPkp21xL0ZvuDW9DMBrD2HB1P4Ry5I.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings,{"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"qraAoatQ-1rtEcDO2w5hTmPnP6kDVaeBEBwUlGkvAoY","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"modules\/comment\/comment.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"modules\/forum\/forum.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"sites\/default\/modules\/biorxiv\/css\/biorxiv.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/flexible\/flexible.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_collection\/css\/highwire_subject_collection.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire.highwire-list-expand.css":1,"public:\/\/ctools\/css\/bab59090150caf0c64e27afa1b36eef1.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/contrib\/eu_cookie_compliance\/css\/eu_cookie_compliance.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"0":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_biorxiv_styles\/css\/biorxiv-styles.css.less":1,"https:\/\/d33xdlntwy0kbs.cloudfront.net\/cshl_custom.css":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_figures.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_tables.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"http:\/\/d33xdlntwy0kbs.cloudfront.net\/cshl_custom.js":1,"sites\/all\/modules\/contrib\/eu_cookie_compliance\/js\/eu_cookie_compliance.js":1,"sites\/default\/modules\/jnl_biorxiv_styles\/js\/adv-search-collection-toggle.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/eu_cookie_compliance\/js\/jquery.cookie-1.4.1.min.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/default\/modules\/jnl_biorxiv_styles\/js\/biorxiv-scripts.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_nps_survey\/js\/nps_survey.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"sites\/default\/modules\/contrib\/disqus\/disqus.js":1,"misc\/textarea.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/contrib\/service_links\/js\/twitter_button.js":1,"sites\/all\/modules\/contrib\/service_links\/js\/facebook_like.js":1,"sites\/all\/modules\/contrib\/service_links\/js\/google_plus_one.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.highwire-list-expand.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"249808","apath":"\/biorxiv\/early\/2019\/03\/14\/574798.atom","pisa":"biorxiv;574798v2","processed":["highwire_math"],"markup":[{"requested":"full-text","variant":"full-text","view":"full","pisa":"biorxiv;574798v2"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"disqus":{"domain":"biorxivstage","url":"https:\/\/www.biorxiv.org\/content\/10.1101\/574798v2","title":"Uncovering mechanisms of global ocean change effects on the Dungeness crab (Cancer magister) through metabolomics analysis","identifier":"node\/249808"},"panels_ajax_pane":{"new-41":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022c6HV9HkuXz3nQvyBOei9Emu6DODtKn+ydUl4zKEhtCtguNAT8d0Qjxm2O42fTiei9jFTQdlfx6CvLKWM2Fg2tn1z6DiP7VmJWKk+d618JKbv71iHjsRtdfaIAi3roBJxAEAah8\\\\\\\/jtJelr81tsC1vD37ZF6p6oyFOCbAttopKdjIDxI\\\\\\\/oVjLB5PScfiFD0x0DWXkPfV0OdS5uz5b6ZjYQGyDH6J6ZOG9GLtuWEmjHZqUCG6t+gvooHwnTc4JcLnMdYIkCZ97opessv8KS7+BwuifqXzhdTR\\\\\\\/RZ0IVQ8jdM5CSK\\\\\\\/5DybROttcFefDz\\\\\\\/zusX\\\\\\\/6C8SKQCTbSNBqx3CKarJiBpm3A0qr6S+b5cg9B9X9SanwtF5sE+ewbp3vTD5ooqRTClCenpszmYWR7nfe7\\\\\\\/kVwhZnopgX3xvY3ROrkQpFs+yHFJVckBqTGYytybtRPx\\\\\\\/O0fT7RVomZieBeUjM7u+LdXWRGZq6RqDA4HtfF3KQvsAmTFanp3Ezf6vQTowSyQiTRyvFokFgiBeAOLC3LeAjB+bSSLLUQB7sUfUrXKnfggt+QRSNbvOIHpt8sT8GpEAN+Tm3UraEayrjQkPSTBjsG53HDpm4Z8bL\\\\\\\/ZpeRmvjur0OMnEZwFufGLbcBQre75SX550VElnzbtO9TOCXn65Kro18lXmJ9DvvVW4iEgNC4TOmQ8Ey+Gg5vxuDHEipSxrkQGtcDZXxjo+NVP6\\\\\\\/TdicBB1bo5FipkbKVeF4pkd1sVOmp5ZBu6hoeSKTaQ4G8\\\\\\\/sXg5FgZk5eB4yKvtDLJ56JxcDSum6KmwxntaqykJelRKyklenVrVDt++s60fB34PqcqcSgBreEikbjOOEGroVB1u5BrTSV\\\\\\\/YD44+zJWen\\\\\\\/ATf31SB7R8tEnGQ395bBQyMxuZn92PsCchEFtYwdvIVABlUt7FEjWgpcyfP3pCQ\\\\\\\/UzUeUXoscP\\\\\\\/AHfzfiIEdJY2yTsD0YgqAHxei0JQr\\\\\\\/IDl9+TqH5636\\\\\\\/mnKhLn2\\\\\\\/appSHpZ5lMJq7D3GMTIDTucigaYvYcJS5W22EuP4rhJSpTlBRU8zBu0taF3hXgltpY7Tf3e9xs6yy62vTcapEcXmehY7th3tdJaKBrT0rygb0s+wfcNQX4WU8F1+fxHLmW+snVRMTS2CtlImJQVSxKqwX+qyDIyEjdW+h093s5E\\\\\\\/lcKTndvwfg4KYS7OeMq+uWgNCHRs\\\\\\\/J+aYuJaeGj7+QhcjMCaQScibmMhUzknawdd7AZg4RMCTe0bjP\\\\\\\/KwP50mJc+g2buKvXstBFUPQkOHusWV0aCuR9Mr\\\\\\\/p4QIXg4bjoq4EN27m\\\\\\\/nGKgQ9JnO\\\\\\\/hJV8=\\\u0022,\\\u0022iv\\\u0022:\\\u0022vv4c+A2mg4x3gQs7iQ2vzg==\\\u0022,\\\u0022salt\\\u0022:\\\u0022f9d8845712f72c9fee5db7d9ff111eab\\\u0022}\u0022,\u0022hmac\u0022:\u0022dae2f03343d85375e5b3677d286e5843353b63c2c194c1540906f73a6ec12206\u0022}"},"urlIsAjaxTrusted":{"\/content\/10.1101\/574798v2.full":true},"ws_fl":{"width":100,"height":21},"ws_gpo":{"size":"","annotation":"","lang":"","callback":"","width":300},"color":{"logo":"https:\/\/www.biorxiv.org\/sites\/default\/files\/bioRxiv_article.jpg"},"highwire_list_expand":{"is_collapsed":"1"},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"eu_cookie_compliance":{"popup_enabled":1,"popup_agreed_enabled":0,"popup_hide_agreed":0,"popup_clicking_confirmation":1,"popup_scrolling_confirmation":false,"popup_html_info":"\u003Cdiv\u003E\n  \u003Cdiv class =\u0022popup-content info\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n      \u003Cp\u003EWe use cookies on this site to enhance your user experience. By clicking any link on this page you are giving your consent for us to set cookies.\u003C\/p\u003E\n    \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022agree-button eu-cookie-compliance-default-button\u0022\u003EContinue\u003C\/button\u003E\n                    \u003Cbutton type=\u0022button\u0022 class=\u0022find-more-button eu-cookie-compliance-more-button\u0022\u003EFind out more\u003C\/button\u003E\n          \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E","use_mobile_message":false,"mobile_popup_html_info":"\u003Cdiv\u003E\n  \u003Cdiv class =\u0022popup-content info\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n          \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022agree-button eu-cookie-compliance-default-button\u0022\u003EContinue\u003C\/button\u003E\n                    \u003Cbutton type=\u0022button\u0022 class=\u0022find-more-button eu-cookie-compliance-more-button\u0022\u003EFind out more\u003C\/button\u003E\n          \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E\n","mobile_breakpoint":"768","popup_html_agreed":"\u003Cdiv\u003E\n  \u003Cdiv class=\u0022popup-content agreed\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n      \u003Ch2\u003EThank you for accepting cookies\u003C\/h2\u003E\u003Cp\u003EYou can now hide this message or find out more about cookies.\u003C\/p\u003E    \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022hide-popup-button eu-cookie-compliance-hide-button\u0022\u003EHide\u003C\/button\u003E\n              \u003Cbutton type=\u0022button\u0022 class=\u0022find-more-button eu-cookie-compliance-more-button-thank-you\u0022 \u003EMore info\u003C\/button\u003E\n          \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E","popup_use_bare_css":false,"popup_height":"auto","popup_width":"100%","popup_delay":1000,"popup_link":"\/help\/cookie-policy","popup_link_new_window":1,"popup_position":null,"popup_language":"en","store_consent":false,"better_support_for_screen_readers":0,"reload_page":0,"domain":"","popup_eu_only_js":0,"cookie_lifetime":"365","cookie_session":false,"disagree_do_not_show_popup":0,"method":"default","whitelisted_cookies":"","withdraw_markup":"\u003Cbutton type=\u0022button\u0022 class=\u0022eu-cookie-withdraw-tab\u0022\u003EPrivacy settings\u003C\/button\u003E\n\u003Cdiv class=\u0022eu-cookie-withdraw-banner\u0022\u003E\n  \u003Cdiv class=\u0022popup-content info\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n      \u003Cp\u003E\u0026lt;h2\u0026gt;We use cookies on this site to enhance your user experience\u0026lt;\/h2\u0026gt;\u0026lt;p\u0026gt;You have given your consent for us to set cookies.\u0026lt;\/p\u0026gt;\u003C\/p\u003E\n    \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022eu-cookie-withdraw-button\u0022\u003EWithdraw consent\u003C\/button\u003E\n    \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E\n","withdraw_enabled":false},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1,"trackUrlFragments":1},"jnl_biorxiv_styles":{"defaultJCode":"biorxiv"},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-249808 node-type-highwire-article context-content hw-default-jcode-biorxiv hw-article-type-article hw-article-category-new-results">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-M677548" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-M677548');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
        <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
    
  <div id="zone-branding" class="zone zone-branding clearfix print-display-block container-30">
    <div class="grid-15 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="bioRxiv" src="https://www.biorxiv.org/sites/default/files/bioRxiv_article.jpg" /></a>      </div>
                </div>
          </div>
</div><div class="grid-11 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-system block-menu block-main-menu block-system-main-menu odd block-without-title" id="block-system-main-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/" title="" class="" data-icon-position="" data-hide-link-title="0">Home</a></li>
<li class="leaf" role="menuitem"><a href="/about-biorxiv" class="" data-icon-position="" data-hide-link-title="0">About</a></li>
<li class="leaf" role="menuitem"><a href="/submit-a-manuscript" class="" data-icon-position="" data-hide-link-title="0">Submit</a></li>
<li class="last leaf" role="menuitem"><a href="/content/alertsrss" title="" class="" data-icon-position="" data-hide-link-title="0">ALERTS / RSS</a></li>
</ul></nav>    </div>
  </div>
</div><div class="block block-panels-mini block-biorxiv-search-box block-panels-mini-biorxiv-search-box even block-without-title" id="block-panels-mini-biorxiv-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-biorxiv_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/10.1101/574798v2.full" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--2" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-Au54GPzWeV12YJwb6QUex21onuuSgutiplGtNrpIrsM" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2 advanced-search-link" >
  
      
  
  <div class="pane-content">
    <a href="/search">Advanced Search</a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix container-30">
  	      </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="249808" id="node249808" data-pisa="biorxiv;574798v2" data-pisa-master="biorxiv;574798" data-apath="/biorxiv/early/2019/03/14/574798.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div  class="highwire-cite highwire-cite-highwire-article highwire-citation-biorxiv-article-top clearfix has-author-tooltip" >

      <span class="biorxiv-article-type">
        New Results    </span>
  
        <h1 class="highwire-cite-title" id="page-title">Uncovering mechanisms of global ocean change effects on the Dungeness crab (<em>Cancer magister</em>) through metabolomics analysis</h1>  
      <div  class="highwire-cite-authors" ><span  class="highwire-citation-authors"><span class="highwire-citation-author first hw-author-orcid-logo-wrapper" data-delta="0"><a href="http://orcid.org/0000-0001-6904-4149" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a><span class="nlm-given-names">Shelly A.</span> <span class="nlm-surname">Trigg</span></span>, <span class="highwire-citation-author" data-delta="1"><span class="nlm-given-names">Paul</span> <span class="nlm-surname">McElhany</span></span>, <span class="highwire-citation-author" data-delta="2"><span class="nlm-given-names">Michael</span> <span class="nlm-surname">Maher</span></span>, <span class="highwire-citation-author" data-delta="3"><span class="nlm-given-names">Danielle</span> <span class="nlm-surname">Perez</span></span>, <span class="highwire-citation-author" data-delta="4"><span class="nlm-given-names">D. Shallin</span> <span class="nlm-surname">Busch</span></span>, <span class="highwire-citation-author" data-delta="5"><span class="nlm-given-names">Krista M.</span> <span class="nlm-surname">Nichols</span></span></span></div>
  
      <div  class="highwire-cite-metadata" ><span  class="highwire-cite-metadata-doi highwire-cite-metadata"><span class="label">doi:</span> https://doi.org/10.1101/574798 </span></div>
  
  
  </div>
<div id="hw-article-author-popups-node249808" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Shelly A. Trigg </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-institution'>Division of Biological Sciences, Cell and Developmental Biology Section, University of California San Diego</span>, La Jolla, California, <span class='nlm-country'>USA</span></div><div class='author-affiliation'><span class='nlm-institution'>Genomic Analysis Laboratory, The Salk Institute for Biological Studies</span>, La Jolla, California, <span class='nlm-country'>USA</span></div><div class='author-affiliation'>Present address: <span class='nlm-institution'>School of Aquatic and Fishery Sciences, University of Washington</span>, Seattle, Washington, <span class='nlm-country'>USA</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Shelly%2BA.%2BTrigg%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Trigg%20SA&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AShelly%2BA.%2BTrigg%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0001-6904-4149" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Shelly A. Trigg</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:strigg@uw.edu" class="" data-icon-position="" data-hide-link-title="0">strigg@uw.edu</a></span></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Paul McElhany </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-institution'>Conservation Biology Division, Northwest Fisheries Science Center, National Marine Fisheries Service, National Oceanic and Atmospheric Administration</span>, Seattle, Washington, <span class='nlm-country'>USA</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Paul%2BMcElhany%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=McElhany%20P&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3APaul%2BMcElhany%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Michael Maher </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-institution'>Conservation Biology Division, Northwest Fisheries Science Center, National Marine Fisheries Service, National Oceanic and Atmospheric Administration</span>, Seattle, Washington, <span class='nlm-country'>USA</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Michael%2BMaher%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Maher%20M&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AMichael%2BMaher%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Danielle Perez </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-institution'>Conservation Biology Division, Northwest Fisheries Science Center, National Marine Fisheries Service, National Oceanic and Atmospheric Administration</span>, Seattle, Washington, <span class='nlm-country'>USA</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Danielle%2BPerez%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Perez%20D&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ADanielle%2BPerez%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">D. Shallin Busch </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-institution'>Conservation Biology Division, Northwest Fisheries Science Center, National Marine Fisheries Service, National Oceanic and Atmospheric Administration</span>, Seattle, Washington, <span class='nlm-country'>USA</span></div><div class='author-affiliation'><span class='nlm-institution'>Ocean Acidification Program, Office of Oceanic and Atmospheric Research and Northwest Fisheries Science Center, National Marine Fisheries Service, National Oceanic and Atmospheric Administration</span>, Seattle, Washington, <span class='nlm-country'>USA</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=D.%2BShallin%2BBusch%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Busch%20DS&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AD.%2BShallin%2BBusch%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Krista M. Nichols </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-institution'>Conservation Biology Division, Northwest Fisheries Science Center, National Marine Fisheries Service, National Oceanic and Atmospheric Administration</span>, Seattle, Washington, <span class='nlm-country'>USA</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Krista%2BM.%2BNichols%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Nichols%20KM&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AKrista%2BM.%2BNichols%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/10.1101/574798v2" class="panels-ajax-tab-tab" data-panel-name="biorxiv_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:249808" data-trigger="" data-url-enabled="1">Abstract</a><a href="/panels_ajax_tab/biorxiv_tab_art/node:249808/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/10.1101/574798v2.full-text" class="panels-ajax-tab-tab" data-panel-name="article_tab_full_text" data-target-id="highwire_article_tabs" data-entity-context="node:249808" data-trigger="full-text" data-url-enabled="1">Full Text</a><a href="/panels_ajax_tab/article_tab_full_text/node:249808/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/10.1101/574798v2.article-info" class="panels-ajax-tab-tab" data-panel-name="biorxiv_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:249808" data-trigger="article-info" data-url-enabled="1">Info/History</a><a href="/panels_ajax_tab/biorxiv_tab_info/node:249808/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/10.1101/574798v2.article-metrics" class="panels-ajax-tab-tab" data-panel-name="article_tab_metrics" data-target-id="highwire_article_tabs" data-entity-context="node:249808" data-trigger="article-metrics" data-url-enabled="1">Metrics</a><a href="/panels_ajax_tab/article_tab_metrics/node:249808/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/10.1101/574798v2.supplementary-material" class="panels-ajax-tab-tab" data-panel-name="biorxiv_tab_data" data-target-id="highwire_article_tabs" data-entity-context="node:249808" data-trigger="supplementary-material" data-url-enabled="1">Supplementary material</a><a href="/panels_ajax_tab/biorxiv_tab_data/node:249808/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/10.1101/574798v2.full.pdf+html" class="panels-ajax-tab-tab" data-panel-name="biorxiv_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:249808" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i> Preview PDF</a><a href="/panels_ajax_tab/biorxiv_tab_pdf/node:249808/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="article_tab_full_text" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.biorxiv.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-article_tab_full_text"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-markup" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2 class="">ABSTRACT</h2><p id="p-2">The Dungeness crab is an economically and ecologically important species distributed along the North American Pacific coast. To predict how Dungeness crab may physiologically respond to future global ocean change on a molecular level, we performed untargeted metabolomic approaches on individual Dungeness crab juveniles reared in treatments that mimicked current and projected future pH and dissolved oxygen conditions. We found 94 metabolites and 127 lipids responded in a condition-specific manner, with a greater number of known compounds more strongly responding to low oxygen than low pH exposure. Pathway analysis of these compounds revealed that juveniles may respond to low oxygen through evolutionarily conserved processes including downregulating glutathione biosynthesis and upregulating glycogen storage, and may respond to low pH by increasing ATP production. Most interestingly, we found that the response of juveniles to combined low pH and low oxygen exposure was most similar to the low oxygen exposure response, indicating low oxygen may drive the physiology of juvenile crabs more than pH. Our study elucidates metabolic dynamics that expand our overall understanding of how the species might respond to future ocean conditions and provides a comprehensive dataset that could be used in future ocean acidification response studies.</p></div><div class="section" id="sec-1"><h2 class="">INTRO</h2><p id="p-3">The continued increase in anthropogenic carbon dioxide emissions is leading to ocean acidification, with the ocean absorbing an average of 25% of human-caused emissions annually<sup><a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">1</a></sup>. If atmospheric carbon dioxide concentration continues to rise at the current rate, the pH of oceans is predicted to fall 0.3-0.4 units by the end of the century<sup><a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">2</a>,<a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">3</a></sup>. This pH drop could exacerbate conditions in the U.S. Pacific Northwest, where the ocean pH is lower than that of the global ocean due to natural oceanographic processes including regional upwelling, and could pose a greater challenge to the marine inhabitants already coping with this lower pH. An additional and compounding factor to ocean acidification is ocean deoxygenation, which co-varies with pH and temperature. Given that global ocean temperatures will rise with global warming from continued greenhouse gas emissions, hypoxic zones are expected to increase in duration, intensity, and frequency<sup><a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">4</a></sup>. It is not certain how future ocean acidification and deoxygenation environmental stress might affect important Pacific Northwest fisheries like the Dungeness crab fishery, which is the most lucrative and valued at more than $200 million annually<sup><a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">5</a></sup>.</p><p id="p-4">Ocean acidification is predicted to have negative indirect effects on Dungeness crab through loss of prey directly affected by ocean acidification<sup><a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">6</a></sup>, but knowledge of how ocean acidification might directly impact Dungeness crab is limited. It has been postulated that Dungeness crab may exhibit limited tolerance for acid-base disturbances given past observations of Dungeness crab as weak osmoregulators<sup><a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">7</a>–<a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">10</a></sup> and that acid-base balance and osmoregulation are tightly coupled in decapod crustaceans<sup><a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">11</a></sup>. However, it was shown that following a brief two-week exposure to a future-predicted seawater pH of 7.4, adult Dungeness crab are able to acclimate by increasing hemolymph ion levels (bicarbonate, calcium, chloride, sulfate, and sodium), and by decreasing both oxygen consumption and nitrogen excretion<sup><a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">12</a></sup>. On the other hand, Dungeness crab larvae have shown reduced survival and development rate in response to a 45-day exposure to the projected future seawater pH of 7.5 and 7.1<sup><a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">13</a></sup>. Moreover, adult Dungeness crab have shown behavioral responses to declining oxygen conditions (21-1.5 kPa pO<sub>2</sub> over a 5-hour period), including reduced feeding and a preference for the area with the highest pO<sub>2</sub> level when placed in a seawater oxygen gradient (2.5-10.5 kPa pO<sub>2</sub> for 1 hour)<sup><a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">14</a></sup>. Dungeness crab also have shown physiological responses to declining oxygen conditions (18-3 kPa pO2 over a 6-hour period), including redistributing hemolymph to high-energy-demand tissues<sup><a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">15</a></sup>. Despite these clear biological responses to low pH and oxygen, the biochemical mechanisms underlying Dungeness crab response to combined pH and oxygen stress have not yet been defined.</p><p id="p-5">One way to gain broad insight into the biochemical processes underlying the physiological status of organisms is through surveying metabolomes, which is now possible for nearly any species with the advancement of high-throughput metabolite profiling, also known as metabolomics. Exploratory untargeted metabolomics approaches can offer unbiased analyses of the composition of all detectable metabolites for the rapid and quantitative detection of stress responses, which often leads to the development of targeted approaches and identification of stress-indicating biomarkers<sup><a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">16</a>,<a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">17</a></sup>. Functional analyses using pathway inference can subsequently be performed by integrating metabolomics data with databases and other “-omics” datasets using bioinformatics tools to ultimately establish causal networks between different experimental conditions and outcomes. Untargeted metabolomics in the context of understanding ocean acidification has recently been applied to reef-building coral, and revealed metabolite profiles that were predictive of primary production activity and molecules that could be used as potential biomarkers of ocean acidification<sup><a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">18</a></sup>.</p><p id="p-6">To better understand which biochemical pathways might be altered in the response of Dungeness crab to ocean acidification, we applied untargeted metabolomics and lipidomics to individual Puget Sound juveniles exposed to current pH (7.85) and future pH (7.45) conditions for an average of 32 days. To account for the dissolved oxygen (DO) levels that naturally co-vary with pH, we included both ambient oxygen (8.9 mg/L or saturated O<sub>2</sub>) and low oxygen (3.0 mg/L or 33% O<sub>2</sub> saturation) treatments with our pH conditions in a factorial design to understand how pH, oxygen, and/or their interaction might influence metabolite abundances.</p></div><div class="section" id="sec-2"><h2 class="">RESULTS</h2><div id="sec-3" class="subsection"><h3>Juvenile Dungeness crab general metabolome and lipidome composition</h3><p id="p-7">Untargeted lipidomics and metabolomics were carried out on 60 individual juvenile crabs exposed to ocean acidification treatments (<strong><a id="xref-table-wrap-1-1" class="xref-table" href="#T1">Table 1</a></strong> and <strong>Supplementary Table 1</strong>) through the entire duration of their first juvenile instar until 2 days after molting to their second juvenile instar (<strong>Supplementary Table 2</strong>). From the lipidomics and metabolomics profiles generated by the West Coast Metabolomics Core using in-house open-access bioinformatics pipelines<sup><a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">19</a>,<a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">20</a></sup> (see Methods for additional details), a total of 3113 lipids and 651 general metabolites were detected of which 88% (3320/3764 total detected compounds) had spectra that did not match LipidBlast<sup><a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">21</a></sup> or BinBase<sup><a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">22</a></sup> records and were classified as unknown compounds (<strong>Supplementary Table 3</strong> and <strong>4</strong>). Eighteen lipids were detected in fewer than 50% of individual profiles and were therefore excluded from all downstream analyses. The MS identification techniques found 29/281 metabolite classes among the 246 compounds detected and listed in Human Metabolome Database (HMDB)<sup><a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">23</a></sup> and 13/77 lipid classes among 195 compounds detected and listed in LIPID MAPS Structural Database<sup><a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">24</a></sup> (<strong>Supplementary Fig. 1</strong>). Among the largest classes represented in all identified compounds were LIPID MAPS classes glycerophosphocholines, triradylglycerols, and fatty acids, and HMDB classes organic acids, carboxylic acids, and organic oxygen compounds, consistent with previously observed <em>Cancer magister</em> body composition proportions of these compounds<sup><a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">25</a>,<a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">26</a></sup>.</p><div id="T1" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/262860/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup"><a href="/highwire/markup/262860/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/262860" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 1.</span> <p id="p-8" class="first-child">Summary of water chemistry for experimental treatments showing the mean ± standard deviation for each parameter. Temperature (temp), pH, and DO were continuously measured by logger probes. Total alkalinity (TA) and salinity were discrete measurements, with TA ranging from 2003.4 to 2043.3 mol/kg and salinity ranging from 29.42 to 29.96 ppt throughout the course of the experiment. Refer to <strong>Supplementary Table 1</strong> for complete data table.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-4" class="subsection"><h3>A combination of statistical methods reveal condition-specific responsive compounds</h3><p id="p-9">In general, individual metabolite and lipid profiles showed high variation regardless of treatment group, with no obvious trends revealed by clustering the relative abundance of all compounds by treatment groups (<strong>Supplementary Fig. 2</strong>). To identify specific compounds affected by treatments, we applied both univariate and multivariate statistical approaches, combining the strengths of different methods<sup><a id="xref-ref-27-1" class="xref-bibr" href="#ref-27">27</a></sup>. To assess individual effects as well as interaction effects from pH and DO treatments on individual compounds, we used a two-way analysis of variance (ANOVA) on the raw abundance data (<strong>Supplementary Table 3 and 4</strong>). Prior to performing ANOVA, we verified compounds showed homogeneity of variances across treatment groups (&gt;96% compounds showed a Levene test <em>P</em> value &gt; 0.05), and that metabolite and lipid abundances were mostly (on average 69% compounds showed a Shapiro-Wilks test <em>P</em> value &gt; 0.05) normally distributed within treatment groups (<strong>Supplementary Table 5</strong>). Although about one-third of compounds violated the ANOVA normality assumption, ANOVA can be considered robust to violations of this assumption when datasets have more than 10 samples per treatment group<sup><a id="xref-ref-28-1" class="xref-bibr" href="#ref-28">28</a>,<a id="xref-ref-29-1" class="xref-bibr" href="#ref-29">29</a></sup>. Of all compounds analyzed, 56/651 metabolites (including 24/160 known metabolites) and 98/3095 lipids (including 7/284 known lipids) showed overall model significance at <em>P</em> &lt; 0.1 and at least one model term (pH, DO, and/or pH x DO interaction) significant at <em>P</em> &lt; 0.05 (<strong>Supplementary Fig. 3 and 4, and Supplementary Table 6</strong>). Because no metabolite or lipid overall model <em>P</em> values passed their respective 10% false discovery rate Benjamini-Hochberg <em>P</em> value threshold of 1 x 10<sup>-4</sup> and less than 3 x 10<sup>-5</sup>, we chose to use their uncorrected <em>P</em> values.</p><p id="p-10">To identify discriminatory compounds and model the relationship between metabolite profiles and exposure treatments using a multivariate linear regression approach, we used partial least square discriminatory analysis (PLS-DA). For facilitating comparisons of metabolite composition among treatment groups, compound abundances were centered around the mean and scaled by the reference group (ambient pH and DO) compound standard deviations prior to multivariate analysis<sup><a id="xref-ref-30-1" class="xref-bibr" href="#ref-30">30</a></sup>. The PLS-DA model for the metabolite data shows partial separation of low pH-treated samples from ambient pH-treated samples by the first and second components accounting for a total of 30.9% (<strong><a id="xref-fig-1-1" class="xref-fig" href="#F1">Fig. 1a</a></strong>). Moreover, the metabolite data shows partial separation of low DO-treated samples in the second and third components, explaining an additional 6.8% of variation (<strong><a id="xref-fig-1-2" class="xref-fig" href="#F1">Fig. 1b</a></strong>). For the lipid data, the PLS-DA model shows less clear treatment group separation on the first component (<strong><a id="xref-fig-1-3" class="xref-fig" href="#F1">Fig. 1c</a></strong>) but shows partial separation of low pH-treated samples from ambient pH-treated samples on the second and third components explaining 14.7% of variation (<strong><a id="xref-fig-1-4" class="xref-fig" href="#F1">Fig. 1d</a></strong>). Because the overall PLS-DA model had weak predictive power (<strong>Supplementary Fig. 5</strong>), we used importance thresholds defined by the point of diminishing returns in the PLS-DA component loadings plots (<strong>Supplementary Fig 6.</strong> and <strong>Methods</strong>), and identified a total of 45 metabolites (including 14/160 known metabolites) and 18 unknown lipids as compounds important in discriminating between treatment groups.</p><div id="F1" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/03/14/574798/F1.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Summary of PLS-DA analysis. Metabolite data show partial separation of low pH treatment groups (blue and purple) from low DO treatment groups (pink and green) by (a) Components 1 and 2 and (b) components 2 and 3. Lipid data show partial separation of treatment groups by (c) components 1 and 2 and (d) components 2 and 3. Shapes correspond to treatment groups (ambient pH and DO, &#x394;; ambient pH and low DO, +; low pH and ambient DO, x; low pH and low DO, &#x25C7;. Numbers refer to the sample number." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1332576508" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Summary of PLS-DA analysis. Metabolite data show partial separation of low pH treatment groups (blue and purple) from low DO treatment groups (pink and green) by (&lt;strong&gt;a&lt;/strong&gt;) Components 1 and 2 and (&lt;strong&gt;b&lt;/strong&gt;) components 2 and 3. Lipid data show partial separation of treatment groups by (&lt;strong&gt;c&lt;/strong&gt;) components 1 and 2 and (&lt;strong&gt;d&lt;/strong&gt;) components 2 and 3. Shapes correspond to treatment groups (ambient pH and DO, &#x394;; ambient pH and low DO, +; low pH and ambient DO, x; low pH and low DO, &#x25C7;. Numbers refer to the sample number.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 1." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2019/03/14/574798/F1.medium.gif" width="440" height="435"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 1." src="https://www.biorxiv.org/content/biorxiv/early/2019/03/14/574798/F1.medium.gif" width="440" height="435"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/03/14/574798/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1." data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/03/14/574798/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml"><span class="fig-label">Figure 1.</span> <p id="p-11" class="first-child">Summary of PLS-DA analysis. Metabolite data show partial separation of low pH treatment groups (blue and purple) from low DO treatment groups (pink and green) by (<strong>a</strong>) Components 1 and 2 and (<strong>b</strong>) components 2 and 3. Lipid data show partial separation of treatment groups by (<strong>c</strong>) components 1 and 2 and (<strong>d</strong>) components 2 and 3. Shapes correspond to treatment groups (ambient pH and DO, Δ; ambient pH and low DO, +; low pH and ambient DO, x; low pH and low DO, ◇. Numbers refer to the sample number.</p><div class="sb-div caption-clear"></div></div></div><p id="p-12">Finally, we used a multivariate nonlinear-based supervised random forest classification to predict treatment classes from the metabolomic and lipidomic data (<strong>Supplementary Fig. 7a-b</strong>). A total of 9 metabolites (including 4/160 known metabolites) and 15 lipids (including 1/284 known lipids) were considered compounds important to predicting treatments based on how much their removal from the model decreased prediction accuracy (<strong>Supplementary Fig. 7c-d, Supplementary Table 8</strong>, and <strong>Methods</strong>). While no compounds were commonly identified by all 3 statistical methods as was expected given the fundamental differences of the methods, 7 metabolites were commonly identified by ANOVA and PLS-DA, 8 metabolites were in common between ANOVA and random forest, 1 metabolite overlapped between random forest and PLS-DA, and 4 lipids overlapped between ANOVA and random forest (<strong><a id="xref-fig-2-1" class="xref-fig" href="#F2">Fig. 2a-b</a></strong>). This combination of analyses yielded a comprehensive set of compounds affected by treatments that no one method was capable of capturing on its own.</p><div id="F2" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/03/14/574798/F2.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Venn diagrams DA, and/or random forest a" class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1332576508" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Venn diagrams DA, and/or random forest a&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 2." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2019/03/14/574798/F2.medium.gif" width="304" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 2." src="https://www.biorxiv.org/content/biorxiv/early/2019/03/14/574798/F2.medium.gif" width="304" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/03/14/574798/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2." data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/03/14/574798/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 2.</span> <p id="p-13" class="first-child">Venn diagrams DA, and/or random forest a</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-5" class="subsection"><h3>Low oxygen more strongly affects compound abundance than pH</h3><p id="p-14">Heatmaps of the statistically identified 94 (including 35 known) metabolites and 127 (including 7 known) lipids, show several compounds respond to low pH and oxygen treatments by commonly increasing or decreasing abundance relative to the ambient treatment (<strong><a id="xref-fig-3-1" class="xref-fig" href="#F3">Fig. 3a-b</a></strong>). This is also summarized by violin plots of known compound abundances in <strong><a id="xref-fig-4-1" class="xref-fig" href="#F4">Figure 4a-c</a></strong>. For example, 5-methoxytryptamine, butyrolactam, cysteine, cysteine, homoserine, pipecolinic acid, piperidone, ribose, and xanthine commonly show a decrease in abundance in response to low DO treatments relative to ambient DO treatments. Glutamic acid, maltose, and maltotriose commonly show an increase in abundance in response to low DO treatments relative to ambient DO treatments (<strong><a id="xref-fig-4-2" class="xref-fig" href="#F4">Figure 4a</a></strong>). Overall, metabolite abundance tends to decrease rather than increase in response to low DO treatments relative to ambient DO treatments, exemplified by the mostly blue color of compound abundance averages for the low DO treatment group (green) in <strong><a id="xref-fig-3-2" class="xref-fig" href="#F3">Figure 3a</a></strong>. Specific to the low pH treatment, more metabolites show average abundances similar to ambient treatment signifying pH has a less dramatic effect on metabolites compared to low DO treatment (<strong><a id="xref-fig-3-3" class="xref-fig" href="#F3">Fig. 3a-b</a> and <a id="xref-fig-4-3" class="xref-fig" href="#F4">Fig. 4a-b</a></strong>). This is also apparent in the combined low pH and low DO treatment, where metabolites and lipids show average abundances similar to the low DO treatment signifying low DO has a more dominant effect on compounds than low pH (<strong><a id="xref-fig-3-4" class="xref-fig" href="#F3">Fig. 3a-b</a></strong>).</p><div id="F3" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/03/14/574798/F3.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Heatmap plots showing the average compound abundance (average peak intensity) for each treatment group for the (a) 94 general metabolites and (b) 127 lipids selected by multivariate and univariate statistical methods used to evaluate treatment effects. Individual known and unknown (&#x201C;unk&#x201D;) compound names are listed over the columns and treatment groups are listed over the rows (orange, ambient pH, ambient O2; green, ambient pH, low O2; purple, low pH, ambient O2; and blue, low pH, low O2). Compound average abundances are shown as auto-scaled within each compound (&#x201C;relative abundance&#x201D;)." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1332576508" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Heatmap plots showing the average compound abundance (average peak intensity) for each treatment group for the (&lt;strong&gt;a&lt;/strong&gt;) 94 general metabolites and (&lt;strong&gt;b&lt;/strong&gt;) 127 lipids selected by multivariate and univariate statistical methods used to evaluate treatment effects. Individual known and unknown (&#x201C;unk&#x201D;) compound names are listed over the columns and treatment groups are listed over the rows (orange, ambient pH, ambient O&lt;sub&gt;2&lt;/sub&gt;; green, ambient pH, low O&lt;sub&gt;2&lt;/sub&gt;; purple, low pH, ambient O&lt;sub&gt;2&lt;/sub&gt;; and blue, low pH, low O&lt;sub&gt;2&lt;/sub&gt;). Compound average abundances are shown as auto-scaled within each compound (&#x201C;relative abundance&#x201D;).&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 3." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2019/03/14/574798/F3.medium.gif" width="440" height="156"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 3." src="https://www.biorxiv.org/content/biorxiv/early/2019/03/14/574798/F3.medium.gif" width="440" height="156"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/03/14/574798/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 3." data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/03/14/574798/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 3.</span> <p id="p-15" class="first-child">Heatmap plots showing the average compound abundance (average peak intensity) for each treatment group for the (<strong>a</strong>) 94 general metabolites and (<strong>b</strong>) 127 lipids selected by multivariate and univariate statistical methods used to evaluate treatment effects. Individual known and unknown (“unk”) compound names are listed over the columns and treatment groups are listed over the rows (orange, ambient pH, ambient O<sub>2</sub>; green, ambient pH, low O<sub>2</sub>; purple, low pH, ambient O<sub>2</sub>; and blue, low pH, low O<sub>2</sub>). Compound average abundances are shown as auto-scaled within each compound (“relative abundance”).</p><div class="sb-div caption-clear"></div></div></div><div id="F4" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/03/14/574798/F4.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Compound abundance levels (peak intensities) of known metabolites and lipids selected by univariate and multivariate statistical methods used to evaluate treatment effects. Violin plots with boxplot insets show the distribution of compound abundances within each treatment group for compounds statistically showing (a) DO treatment effect, (b) pH treatment effect, and (c) DO:pH interaction effect. Binbase names for each known compound are listed across the top of each plot. Abundance levels (peak intensities, noted as &#x201C;quant&#x201D;) are listed on the y-axis while treatments are listed along the x-axis. Statistical methods that each known compound was identified by are noted by shapes in the upper corners of each plot (ANOVA, triangle; PLS-DA, square; and random forest, diamond)." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1332576508" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Compound abundance levels (peak intensities) of known metabolites and lipids selected by univariate and multivariate statistical methods used to evaluate treatment effects. Violin plots with boxplot insets show the distribution of compound abundances within each treatment group for compounds statistically showing (&lt;strong&gt;a&lt;/strong&gt;) DO treatment effect, (&lt;strong&gt;b&lt;/strong&gt;) pH treatment effect, and (c) DO:pH interaction effect. Binbase names for each known compound are listed across the top of each plot. Abundance levels (peak intensities, noted as &#x201C;quant&#x201D;) are listed on the y-axis while treatments are listed along the x-axis. Statistical methods that each known compound was identified by are noted by shapes in the upper corners of each plot (ANOVA, triangle; PLS-DA, square; and random forest, diamond).&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 4." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2019/03/14/574798/F4.medium.gif" width="440" height="284"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 4." src="https://www.biorxiv.org/content/biorxiv/early/2019/03/14/574798/F4.medium.gif" width="440" height="284"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/03/14/574798/F4.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 4." data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/03/14/574798/F4.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 4.</span> <p id="p-16" class="first-child">Compound abundance levels (peak intensities) of known metabolites and lipids selected by univariate and multivariate statistical methods used to evaluate treatment effects. Violin plots with boxplot insets show the distribution of compound abundances within each treatment group for compounds statistically showing (<strong>a</strong>) DO treatment effect, (<strong>b</strong>) pH treatment effect, and (c) DO:pH interaction effect. Binbase names for each known compound are listed across the top of each plot. Abundance levels (peak intensities, noted as “quant”) are listed on the y-axis while treatments are listed along the x-axis. Statistical methods that each known compound was identified by are noted by shapes in the upper corners of each plot (ANOVA, triangle; PLS-DA, square; and random forest, diamond).</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-6" class="subsection"><h3>Pathway analysis reveals potential mechanisms of pH and oxygen stress tolerance</h3><p id="p-17">To explore the physiological relevance of treatment-responsive compounds, we performed biochemical pathway analysis on all known compounds. While compounds classified as sugars and fatty acids were affected by the low pH and DO treatments, most of the significantly affected compounds were part of the amino acids class, indicating amino acid metabolism was most significantly altered by the treatments. Abbreviated biochemical pathway networks focusing on affected amino acids are shown in <strong><a id="xref-fig-5-1" class="xref-fig" href="#F5">Figure 5</a></strong>, and the complete biochemical pathway networks affected by low pH, low DO, and the combined low pH and DO treatments can be found in <strong>Supplementary Figures 8-10</strong>. The trends in amino acid abundances resulting from either low pH, low DO, or the combined low pH and DO treatment (<strong><a id="xref-fig-5-2" class="xref-fig" href="#F5">Fig. 5a-c</a></strong>), suggest different amino acid metabolic pathways are affected in response to each factor or combination of factors.</p><div id="F5" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/03/14/574798/F5.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Differential amino acid network effects from (a) low DO, (b) low pH, and (c) combined low pH and low DO. Compounds are clustered by chemical similarity. Node fill color is colored by Cohen D effect size comparing the treatment group to the ambient pH,ambient DO group, and node borders are colored by the treatment group mean fold change relative to the ambient pH,ambient DO group. Node shapes indicate the statistical method(s) from which the compound was classified as important. Node shape and label size are enlarged if the compound was identified as important by a statistical method. Gray edges indicate nodes sharing a KEGG biochemical pathway(s)." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1332576508" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Differential amino acid network effects from (&lt;strong&gt;a&lt;/strong&gt;) low DO, (&lt;strong&gt;b&lt;/strong&gt;) low pH, and (&lt;strong&gt;c&lt;/strong&gt;) combined low pH and low DO. Compounds are clustered by chemical similarity. Node fill color is colored by Cohen D effect size comparing the treatment group to the ambient pH,ambient DO group, and node borders are colored by the treatment group mean fold change relative to the ambient pH,ambient DO group. Node shapes indicate the statistical method(s) from which the compound was classified as important. Node shape and label size are enlarged if the compound was identified as important by a statistical method. Gray edges indicate nodes sharing a KEGG biochemical pathway(s).&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 5." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2019/03/14/574798/F5.medium.gif" width="440" height="269"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 5." src="https://www.biorxiv.org/content/biorxiv/early/2019/03/14/574798/F5.medium.gif" width="440" height="269"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/03/14/574798/F5.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 5." data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/03/14/574798/F5.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 5.</span> <p id="p-18" class="first-child">Differential amino acid network effects from (<strong>a</strong>) low DO, (<strong>b</strong>) low pH, and (<strong>c</strong>) combined low pH and low DO. Compounds are clustered by chemical similarity. Node fill color is colored by Cohen D effect size comparing the treatment group to the ambient pH,ambient DO group, and node borders are colored by the treatment group mean fold change relative to the ambient pH,ambient DO group. Node shapes indicate the statistical method(s) from which the compound was classified as important. Node shape and label size are enlarged if the compound was identified as important by a statistical method. Gray edges indicate nodes sharing a KEGG biochemical pathway(s).</p><div class="sb-div caption-clear"></div></div></div><p id="p-19">Specific to low DO treatment response compared to the normoxic treatment, energy conservation pathways appear to be most affected (<strong><a id="xref-fig-5-3" class="xref-fig" href="#F5">Fig. 5a</a></strong>). The increased lysine with decreased pipecolinic acid and the piperidine derivative piperidone abundance is suggestive of downregulation of the lysine degradation pathway<sup><a id="xref-ref-31-1" class="xref-bibr" href="#ref-31">31</a>,<a id="xref-ref-32-1" class="xref-bibr" href="#ref-32">32</a></sup>. The decreased abundance of both cysteine and the cysteine homodimer, cystine, suggests that cysteine catabolism is upregulated. This coupled with the increased abundance of glutamic acid suggests the glutathione synthesis pathway could be downregulated, as both cysteine and glutamic acid are precursor molecules in glutathione synthesis<sup><a id="xref-ref-33-1" class="xref-bibr" href="#ref-33">33</a>,<a id="xref-ref-34-1" class="xref-bibr" href="#ref-34">34</a></sup>. Glutathione itself was not detected, but is typically challenging to detect by MS techniques in marine animals due to its reactivity<sup><a id="xref-ref-35-1" class="xref-bibr" href="#ref-35">35</a></sup>. The glycogen intermediates maltose and maltotriose show an increase in abundance consistent with previously observed glycogen synthesis pathway upregulation during hypoxic stress<sup><a id="xref-ref-36-1" class="xref-bibr" href="#ref-36">36</a></sup>. Purine and pyrimidine metabolic intermediates orotic acid, ribose, and xanthine show decreased abundance suggesting purine and pyrimidine metabolic pathways could be downregulated<sup><a id="xref-ref-37-1" class="xref-bibr" href="#ref-37">37</a></sup>.</p><p id="p-20">Specific to low pH treatment response compared to ambient pH treatment, ATP generation pathways appeared most affected (<strong><a id="xref-fig-5-4" class="xref-fig" href="#F5">Fig 5b</a></strong>). The decreased abundance of aspartic acid and the subtle decreased abundance of maleic acid may suggest the citric acid cycle intermediates that they form (oxaloacetate, malic acid, and fumaric acid)<sup><a id="xref-ref-38-1" class="xref-bibr" href="#ref-38">38</a>,<a id="xref-ref-39-1" class="xref-bibr" href="#ref-39">39</a></sup> are favored. Although, oxaloacetate was not detected (likely due to the instability of the alpha keto acid compound)<sup><a id="xref-ref-40-1" class="xref-bibr" href="#ref-40">40</a></sup>, and malic acid and fumaric acid did not show a significant difference across treatments. However, citric acid did show an increase in abundance. Taken all together, the altered abundance of these compounds suggest that citric acid cycle activity could be upregulated in response to low pH<sup><a id="xref-ref-41-1" class="xref-bibr" href="#ref-41">41</a></sup>. An increase in glutaric acid in response to low pH suggests the catabolism of its parent molecule, glutaryl CoA, which produces ATP in addition to glutaric acid<sup><a id="xref-ref-42-1" class="xref-bibr" href="#ref-42">42</a></sup>. Glutaryl CoA catabolism also supports citric acid cycle activity because glutaryl CoA can uncompetitively inhibit the alpha-ketoglutarate dehydrogenase complex that facilitates a rate limiting step in the citric acid cycle<sup><a id="xref-ref-43-1" class="xref-bibr" href="#ref-43">43</a></sup>.</p><p id="p-21">Among the compounds affected by the combined low pH and DO treatment, phosphoethanolamine and phophatidylethanolamine (PE(p-34:2) or PE(o-34:3)) show increased abundance compared to ambient treatment (<strong>Supplementary Fig. 10</strong>), suggesting that low pH and DO might alter the glycerophospholipid synthesis pathway of which phosphatidulethanolamine is a product and phosphoethanolamine is a substrate intermediate in a rate limiting step<sup><a id="xref-ref-44-1" class="xref-bibr" href="#ref-44">44</a></sup>. Alanine shows a decrease in abundance, suggesting that alanine synthesis is decreased and that its substrate for synthesis, pyruvate, may be limited<sup><a id="xref-ref-45-1" class="xref-bibr" href="#ref-45">45</a></sup>. The increased abundance of cystathionine and subtle increase of methionine suggest cysteine and homoserine synthesis are likely stalled<sup><a id="xref-ref-46-1" class="xref-bibr" href="#ref-46">46</a></sup> in response to combined low pH and low DO exposure. This overlaps with the decrease in homoserine abundance and decrease in cysteine synthesis products (cysteine and cystine) observed in low DO treatment (<strong><a id="xref-fig-5-5" class="xref-fig" href="#F5">Fig. 5b-c</a></strong>).</p></div></div><div class="section" id="sec-7"><h2 class="">DISCUSSION</h2><p id="p-22">We used metabolomics to explore how the Dungeness crab might respond to the simultaneous change in oxygen and carbon dioxide in predicted future climate change scenarios in the Pacific Northwest. Although the untargeted metabolomics and lipidomics approaches rapidly identified hundreds of different molecular species, the majority of compounds identified in our study, including most compounds selected by statistical methods, lack annotations in existing metabolite databases. Thus, our understanding of their role in response to low pH and low DO treatment is limited without extensive further validation. Still, through shedding light on the simultaneous activity of hundreds of known compounds, we were able to observe a dynamic range of metabolic responses among individuals within treatment groups that indicates that Dungeness crab have flexibility in how their biochemistry compensates for environmental change. While we attempted to control for variation between individuals by collecting animals from the same location within a 2-month period, we were unable to control for prior environmental exposure or genetic background of the wild-caught animals we used. Where prior studies have found high genetic variation among individuals within one sampling site<sup><a id="xref-ref-47-1" class="xref-bibr" href="#ref-47">47</a>,<a id="xref-ref-48-1" class="xref-bibr" href="#ref-48">48</a></sup>, we suspect that both genetics and prior environmental exposure likely contributed to the high variation in metabolite abundances among individuals within treatment groups, which may have obscured the different treatment effects. Within-sample variation could be reduced by sampling from a specific tissue rather than using whole animals, however this may be challenging in early stage juvenile Dungeness crabs given their small size.</p><p id="p-23">We applied three different statistical methods (ANOVA, PLS-DA, and random forest) for selecting important metabolites in order to combine the strengths of powerful univariate and multivariate analyses<sup><a id="xref-ref-27-2" class="xref-bibr" href="#ref-27">27</a></sup>. Although using univariate statistics in the strictest sense with an FDR correction to control for false discoveries showed no statistically significant variation for compounds across treatments, not all compounds were identified with the same confidence level and were subject to the sensitivity of the mass spectrometry method used. Applying an FDR correction to all detected compounds assumes that all compounds had equal chance for discovery, which can mask important biology in highly variable datasets<sup><a id="xref-ref-49-1" class="xref-bibr" href="#ref-49">49</a></sup>. For this reason and due to our use of wild-caught whole-animal samples, we chose a more liberal focus on all compounds showing an overall ANOVA model <em>P</em> value &lt; 0.1 and at least one model term <em>P</em> value of &lt; 0.05, or identified as important in PLS-DA or random forest models in our pathway analysis.</p><p id="p-24">While we acknowledge a level of uncertainty in our pathway analysis due to liberal selection of important treatment-responsive compounds, the resulting proposed affected pathways are consistent with previous observations of pH and hypoxia effects on different organisms. In general, amino acid metabolism is a well-documented mechanism for stress tolerance<sup><a id="xref-ref-50-1" class="xref-bibr" href="#ref-50">50</a>,<a id="xref-ref-51-1" class="xref-bibr" href="#ref-51">51</a></sup>. This class of compounds contains versatile chemical structures that serve as buffering molecules, antioxidants, signaling molecules, and chemical building blocks for the synthesis of proteins important in stress response (i.e., heat shock proteins, unfolded protein response proteins, ion channels). Under low oxygen conditions, it is in the best interest of the animal to limit non-essential energy consuming pathways<sup><a id="xref-ref-52-1" class="xref-bibr" href="#ref-52">52</a></sup>. One way Dungeness crab may do this is through reducing the activity of evolutionarily conserved gamma-glutamyl cycle that synthesizes glutathione and consumes ATP<sup><a id="xref-ref-53-1" class="xref-bibr" href="#ref-53">53</a>,<a id="xref-ref-54-1" class="xref-bibr" href="#ref-54">54</a></sup> by limiting cysteine availability through catabolism. Interestingly, glutathione reduction in response to hypoxia has been observed in multiple mammalian cell lines<sup><a id="xref-ref-55-1" class="xref-bibr" href="#ref-55">55</a>–<a id="xref-ref-58-1" class="xref-bibr" href="#ref-58">58</a></sup>. Also under low oxygen conditions in nature, food can be scarce, and it is theorized that glycogen storage during low oxygen can help prepare cells for low nutrient conditions. In multiple mouse and human cancer cell lines<sup><a id="xref-ref-36-2" class="xref-bibr" href="#ref-36">36</a>,<a id="xref-ref-59-1" class="xref-bibr" href="#ref-59">59</a>,<a id="xref-ref-60-1" class="xref-bibr" href="#ref-60">60</a></sup>, increased glycogen storage has been observed in response to hypoxia, and is induced by highly evolutionarily conserved hypoxia-inducible factor transcriptional signalling<sup><a id="xref-ref-36-3" class="xref-bibr" href="#ref-36">36</a></sup>. It seems that Dungeness crab may also adopt this strategy in combating low oxygen conditions. Under low pH, adult Dungeness crab initially develop hypercapnia which then abates over time via elevated hemolymph bicarbonate likely generated through gill restructuring and the upregulation of energy consuming ion-exchange proteins<sup><a id="xref-ref-12-2" class="xref-bibr" href="#ref-12">12</a></sup>. Our results indicate that this process may occur in early juveniles in low pH conditions given that we found metabolite profiles that support energy generation via increased citric acid cycle activity.</p><p id="p-25">Having parsed out effects from low oxygen and low pH, we found that in combined low oxygen and pH conditions low oxygen has a more dramatic effect on metabolite abundance. However, the animals in this treatment group still showed pathway alterations similar to the low pH treatment group including increased abundance of citric-acid-cycle-related metabolites to potentially increase ATP production. It is not yet clear what the longer-term consequences are of these metabolic adjustments or how long these responses could be sustained. Future avenues of research to expand on these findings should include targeted metabolomics to confirm the compounds identified in this study as well as capture more antioxidants and citric acid cycle intermediates that were not detected in this study. Targeted expression profiling would also be helpful in confirming the biochemical pathway activity predictions from this study. Ultimately, longer-term exposure experiments on crabs reared over generations might best reveal the maximal duration that these metabolic responses can be sustained and any long-term consequences of low pH and oxygen exposure. This exploratory metabolomics and lipidomics analysis uncovered potential biochemical pathways affected by experiments simulating ocean acidification and hypoxia, and can now serve as preliminary hypotheses for deeper investigations of how the Dungeness crab (and even other crustaceans) may tolerate global ocean change.</p></div><div class="section" id="sec-8"><h2 class="">METHODS</h2><div id="sec-9" class="subsection"><h3>Animals</h3><p id="p-26"><em>Cancer magister</em> megaolopae were collected from a single site in Puget Sound (47.950232, - 122.301784) on several days between the June and September 2016 using light traps that were set overnight. The contents of each trap were immediately transferred into a 5-gallon bucket that was, within 5 minutes, pooled into a cooler with ice packs and an air bubbler. After transferring contents from 7 traps in approximately 1 hour, the cooler containing megalopae was brought within 5 minutes into the lab and megalopae were individually transferred onto the water flow-through system described below. The total time for transferring megalopae from the cooler onto the seawater flow-through system was about 2 hours.</p></div><div id="sec-10" class="subsection"><h3>CO<sub>2</sub> exposure experiments</h3><p id="p-27">Megalopae were held in individual 250 mL customized jars on Mobile Ocean Acidification Treatment Systems (MOATS). These systems flowed one-micron-filtered, UV-sterilized, Puget Sound seawater maintained at 12°C. Prior to flowing through jars, seawater was degassed and oxygen, nitrogen and carbon dioxide gases were resupplied to finely control dissolved gas levels. Temperature, pH, and dissolved oxygen were continuously monitored throughout the duration of the experiment by Omega thermistors, Honeywell Durafet III probes, and Vernier optical dissolved oxygen probes, respectively. The pH was additionally validated by periodic sampling of water for dissolved inorganic carbon and total alkalinity, and by bi-weekly spectrophotometric pH measurements using an Ocean Optics USB 230 2000+ Fiber Optic Spectometer with SpectraSuite software and a 5mM solution of Sigma Aldritch m-cresol purple indicator dye. MOATS chemistry parameters were automatically adjusted through a data-driven feedback system. Megalopae were fed <em>Artermia salina</em> (San Francisco Bay brand) at a target concentration of 1 nauplius per mL every 3 days. Once megalopae transitioned to the first juvenile instar, they were fed small pieces of squid. Upon transitioning to the second juvenile instar, crabs were held on the MOATS for an additional 48 hours post-molt in attempt to reduce variation due to potential stochastic physiological processes associated with molting <sup><a id="xref-ref-61-1" class="xref-bibr" href="#ref-61">61</a>,<a id="xref-ref-62-1" class="xref-bibr" href="#ref-62">62</a></sup>. Juveniles were then immediately frozen and stored at −80°C after lightly blotting with a paper towel to remove excess seawater. To reduce variation from length of exposure, which in total ranged from 20-65 days, only 60 crabs with the average exposure time (30-33 days) were chosen for metabolomics analysis with 15 individuals from each treatment group.</p></div><div id="sec-11" class="subsection"><h3>Water chemistry analysis</h3><p id="p-28">Mean and standard deviation calculations for pH, DO, and temperature (<strong><a id="xref-table-wrap-1-2" class="xref-table" href="#T1">Table 1</a></strong>) were based on logger data (measured every 10 minutes) with the exception of pH and DO data from MOATS 5 and 12, where Spectrophotometric pH and Presens DO measurements were instead used due to incorrectly calibrated pH and DO probes in these MOATS. Temperature logger readings above 25°C and below 4°C were excluded from the mean and standard deviation analysis because these were out of the achievable range for the equipment used and were indicative of a thermistor malfunction. Total alkalinity (TA) and salinity were discrete measurements, with TA ranging from 2003.4 to 2036.1 μmol/kg and salinity ranging from 29.42 to 29.96 ppt throughout the course of the experiment. The complete data table is included as <strong>Supplementary Table 1</strong>.</p></div><div id="sec-12" class="subsection"><h3>Sample preparation</h3><p id="p-29">Frozen samples were sent to the West Coast Metabolomics Center, Davis, CA for sample preparation, metabolomics and lipidomics profiling. Whole animals were thawed, weighed, and derivatized as previously described<sup><a id="xref-ref-63-1" class="xref-bibr" href="#ref-63">63</a>,<a id="xref-ref-64-1" class="xref-bibr" href="#ref-64">64</a></sup> (<strong>Supplementary Table 2</strong>). Briefly, samples were extracted at −20°C with 2 mL of degassed acetonitrile/isopropanol/water (3:3:2) solution and solvents were evaporated to complete dryness with a Labconco Centrivap cold trap concentrator. Membrane lipids and triglycerides were subsequently removed from dried samples with 50% acetonitrile, and samples were again concentrated to complete dryness. 15 mg of each sample preparation was used for metabolomic profiling. For lipidomic profiling, 15 mg of the same sample preparation was also used to which internal standards, C8-C30 fatty acid methyl esters were added. Aliquoted samples were derivatized with methoxyamine hydrochloride (Sigma-Aldrich) in pyridine (Acros Organics) and then by N-methyl-N-(trimethylsilyl) trifluoroacetamide (Sigma-Aldrich) for trimethylsilylation of acidic protons.</p></div><div id="sec-13" class="subsection"><h3>Metabolite and lipid data acquisition</h3><p id="p-30">General metabolite and lipid abundances were quantified from derivatized samples by gas-chromatography, time-of-flight mass spectrometry (GC-TOF/MS) and charged-surface, hybrid-column, electrospray-quadrupole, time-of-flight mass spectrometry (CSH-ESI QTOF MS/MS), respectively. For metabolites, an Agilent 6890 gas chromatograph (Santa Clara, CA) was used with a Leco Pagasus IV time-of-flight mass spectrometer running Leco ChromaTOF software 2.32 (St. Joseph, MI). The following temperature profile was used: 50°C to 275°C final temperature at a rate of 12°C/s and hold for 3 minutes. Injection volume was 0.5 μl with 10 μl/s injection speed on a splitless injector with a purge time of 25 seconds. Liner (Gerstel #011711-010-00) was changed after every 10 samples, (using the Maestro1 Gerstel software vs. 1.1.4.18). Before and after each injection, the 10 μL injection syringe was washed 3 times with 10 μL ethyl acetate. For gas chromatography, a 30 m long, 0.25 mm i.d. Rtx-5Sil MS column (0.25 μm 95% dimethyl 5% diphenyl polysiloxane film) with additional 10 m integrated guard column was used (Restek, Bellefonte PA). 99.9999% pure Helium with a built-in purifier (Airgas, Radnor PA) was set at a constant flow of 1 mL/minute. The oven temperature was held constant at 50°C for 1 minute and then ramped at 20°C/minute to 330°C at which it is held constant for 5 minutes. The transfer line temperature between gas chromatograph and mass spectrometer was set to 280°C. Electron-impact ionization at 70V was employed with an ion source temperature of 250°C. Acquisition rate was 17 spectra/second, with a scan mass range of 85-500 Da.</p><p id="p-31">For positively charged lipids, an Agilent 6530 QTOF mass spectrometer with resolution 10,000 was used and for negatively charged lipids, an Agilent 6550 QTOF mass spectrometer with resolution 20,000 was used. Electrospray ionization was used to ionize column elutants in both positive and negative modes. Compounds were separated using a Waters Acquity ultra-high-pressure, liquid-chromatography charged surface hybrid column C18 (100 mm length x 2.1 mm internal diameter; 1.7 um particles) using the following conditions: mobile phase A (60:40 acetonitrile:water + 10 mM ammonium formiate + 0.1% formic acid, mobile phase B (90:10 isopropanol:acetonitrile + 10 mM ammonium formiate + 0.1% formic acid), 65°C column temperature, a flow rate of 0.6 mL/minute, an injection volume of 3 uL, an injection temperature of 4 C, and a gradient of 0 minutes 15%, 0-2 minutes 30%, 2-2.5 minutes 48%, 2.5-11 minutes 82%, 11-11.5 minutes 99%, 11.5-12 minutes 99%, 12-12.1 minutes 15%, and 12.1-15 minutes 15%. The capillary voltage was set to +3.5 and −3.5 kV, and the collision energy to 25 or 40 eV for positive and negative modes. Mass-to-charge ratios (m/z) were scanned from 60 to 1700 Da and spectra acquired every 2 seconds.</p></div><div id="sec-14" class="subsection"><h3>Spectral data processing</h3><p id="p-32">Acquired metabolite data were processed using UC Davis’s BinBase workflow, which performs data processing including peak detection at signal-to-noise levels of 5:1 throughout the chromatogram. Resulting apex masses are reported for use in the BinBase algorithm to facilitate metabolite identification and quantification<sup><a id="xref-ref-65-1" class="xref-bibr" href="#ref-65">65</a></sup>. Metabolites were identified through comparison to the BinBase database<sup><a id="xref-ref-22-2" class="xref-bibr" href="#ref-22">22</a></sup> and peak heights were normalized to total metabolite content<sup><a id="xref-ref-66-1" class="xref-bibr" href="#ref-66">66</a></sup>.</p><p id="p-33">For acquired lipid data, MassHunter (Qual v. B05.00) was used to find peaks from the raw data in up to 300 chromatograms. These peaks were imported into Mass Profiler Professional for alignment to determine which peaks occur in at least 30 % of the chromatograms. These peaks were then quantified with MassHunter. Resulting accurate mass data and tandem MS/MS spectra were compared to LipidBlast libraries for compound identification<sup><a id="xref-ref-21-2" class="xref-bibr" href="#ref-21">21</a></sup>. All spectra mapping to previously identified compounds are accessible in the public database MassBank of North America (<a href="http://mona.fiehnlab.ucdavis.edu/">http://mona.fiehnlab.ucdavis.edu/</a>). Spectra that did not map to previously identified compounds are accessible by searching their BinBase identifiers listed in <strong>Supplementary Tables 3 and 4</strong> in the BinVestigate database (<a href="http://binvestigate.fiehnlab.ucdavis.edu">http://binvestigate.fiehnlab.ucdavis.edu</a>).</p></div><div id="sec-15" class="subsection"><h3>Statistical analyses</h3><p id="p-34">All statistical analyses excluded 18 compounds that were detected in less than half of the individual crabs surveyed, and were carried out in R, except for PLS-DA and random forest analyses which were carried out using the MetaboAnalyst web interface.</p></div><div id="sec-16" class="subsection"><h3>Univariate statistics</h3><p id="p-35">Prior to applying a univariate statistical test, normality and heteroscedasticity of compound abundances within treatment groups was first assessed using the Shapiro-Wilks test and the Levene test, respectively (<strong>Supplementary Table 5</strong>). After confirming normality and heteroscedasticity assumptions were correct, a two-way ANOVA was then applied to each compound. A Benjamini-Hochberg FDR correction was applied to ANOVA <em>P</em> values to correct for multiple testing, but corrected <em>P</em> values were not used in selecting important compounds. Metabolites and lipids were selected for pathway analyses if they had an overall model ANOVA <em>P</em> value less than 0.1 and an effect <em>P</em> value with less than 0.05 without an FDR correction applied. Results from ANOVA tests are reported in <strong>Supplementary Table 6</strong>.</p></div><div id="sec-17" class="subsection"><h3>Multivariate statistics</h3><p id="p-36">Prior to applying multivariate testing, data were normalized by mean-centering and scaling by the reference-group standard-deviations of each compound in order to make compounds more comparable to one another<sup><a id="xref-ref-30-2" class="xref-bibr" href="#ref-30">30</a></sup>. Normalized data was then uploaded to MetaboAnalyst with no further normalizations applied. The PLS-DA function was run with default settings and a table of component loadings for each compound were exported from MetaboAnalyst (<strong>Supplementary Table 7</strong>). Compound loadings were ordered by from greatest variance explained to lowest, and plotted for PLS-DA components. Importance thresholds were placed at the point of diminishing returns in the plot curves (<strong>Supplementary Fig. 6</strong>). Specifically for metabolites, loadings were plotted PLS-DA components 1, 2, and 3 (<strong>Supplementary Fig. 6a-c</strong>) since these PLS-DA components gave the greatest separation between treatment groups (<strong><a id="xref-fig-1-5" class="xref-fig" href="#F1">Fig. 1a-b</a></strong>). The points of diminishing returns that importance thresholds were placed in the metabolite loadings plots were as follows: component 1, loadings threshold &gt; 0.05; component 2, loadings threshold &gt; 0.1; and component 3, loadings threshold &gt; 0.05. Specifically for lipids, loadings were plotted for PLS-DA components 2 and 3 (<strong>Supplementary Fig. 6d-e</strong>) since these gave the best separation between treatment groups, while component 1 could only distinguish the ambient from the altered treatment groups (<strong><a id="xref-fig-1-6" class="xref-fig" href="#F1">Fig. 1c-d</a></strong>). The points of diminishing returns that importance thresholds were placed in the lipids loadings plots were as follows: component 2, loadings threshold &gt; 0.05; and component 3, loadings threshold &gt; 0.05. For the random forest analysis, the Random Forest function was run with 2000 decision trees and either 25 predictors for the metabolite dataset or 56 predictors for the lipid dataset, conforming to the default classification value being the square root of the number of variables<sup><a id="xref-ref-67-1" class="xref-bibr" href="#ref-67">67</a></sup>. The complete table of important variables and their mean decrease in random forest accuracy prediction exported from MetaboAnalyst (<strong>Supplementary Table 8</strong>). The mean decrease in prediction accuracy for each compound was ordered from largest to smallest and plotted (<strong>Supplementary Fig. 7c-d</strong>). Importance thresholds were drawn from the plots at the points of diminishing returns, which for metabolites was a mean decrease in accuracy &gt; 0.001 and for lipids was a mean decrease in accuracy &gt; 0.00047.</p><p id="p-37">Heatmaps in <strong><a id="xref-fig-3-5" class="xref-fig" href="#F3">Figure 3</a></strong> and <strong>Supplementary Figure 2</strong> were made using the heatmap function in the MetaboAnalyst web interface with Euclidean distance, ward clustering, plotting auto-scaled compound abundance averages for each treatment group.</p></div><div id="sec-18" class="subsection"><h3>Pathway analysis</h3><p id="p-38">Pubchem and KEGG identifiers were obtained for metabolites and lipid international chemical identifier keys provided by the West Coast Metabolomics Core using Chemical Translation Service<sup><a id="xref-ref-68-1" class="xref-bibr" href="#ref-68">68</a></sup> and Pubchem Identifier exchange<sup><a id="xref-ref-69-1" class="xref-bibr" href="#ref-69">69</a></sup>. Metamapp<sup><a id="xref-ref-70-1" class="xref-bibr" href="#ref-70">70</a></sup> was then used to map metabolites by chemical and biochemical relationships. The generated SIF file (<strong>Supplementary File 1</strong>) was imported into Cytoscape<sup><a id="xref-ref-71-1" class="xref-bibr" href="#ref-71">71</a></sup>. For visualizing the pH, DO, and pH:DO interaction effects, both mean abundance fold change and Cohen D effect size values were calculated for each compound within a treatment group relative to the ambient treatment group. Cohen D effect size <span class="inline-formula" id="inline-formula-1"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2019/03/14/574798/embed/inline-graphic-1.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2019/03/14/574798/embed/inline-graphic-1.gif"/></noscript></span></span> was calculated for each compound raw abundance within a treatment group relative to the ambient treatment group using the R package effsize<sup><a id="xref-ref-72-1" class="xref-bibr" href="#ref-72">72</a></sup>. The treatment group -log<sub>2</sub> fold change values and the Cohen D effsize output from R were added to a node attribute file (<strong>Supplementary Table 9</strong>) that was also imported into Cytoscape. Low pH, low DO, and low pH:low DO networks were styled using the following settings: node shapes were set according to statistical method used to select the compound (ANOVA, triangle; PLS-DA, square; random forest, diamond; both PLS-DA and ANOVA, hexagon; not selected as important by any method, circle); node border was color by the treatment group mean compound abundance fold change relative to ambient (1.25X-2X fold change, yellow gradient’ −1.25X – 1.25X fold change, grey; −1.25X - -10X fold change, blue gradient); node fill was colored by the Cohen D effect size of treatment on compound abundance relative to ambient (effect size of 0.5-1, yellow gradient; −0.5.-0.5, white; and −0.5- −1, blue gradient); node shape size and label size were set to fixed values (150 height and width with size 50 font for statistically selected compounds, and 30 height and width with size 12 font for compounds not identified by statistics (<strong>Supplementary Fig. 8-10</strong>).</p></div></div><div class="section" id="sec-20"><h2 class="">DATA AVAILABILITY</h2><p id="p-39">All mass spectra mapping to previously identified compounds are accessible in the public database MassBank of North America (<a href="http://mona.fiehnlab.ucdavis.edu/">http://mona.fiehnlab.ucdavis.edu/</a>). Spectra that did not map to previously identified compounds are accessible by searching their BinBase identifiers in the BinVestigate database (<a href="http://binvestigate.fiehnlab.ucdavis.edu">http://binvestigate.fiehnlab.ucdavis.edu</a>). Processed metabolite and lipid spectral data are included as <strong>Supplementary Table 3 and 4</strong>.</p></div><div class="section" id="sec-21"><h2 class="">AUTHOR CONTRIBUTIONS</h2><p id="p-41">K.M.N., P.M., and D.S.B conceived the project. K.M.N., P.M., and D.S.B. advised research. M.M. and D.P. performed experiments. S.A.T. performed statistical analyses and pathway analysis. S.A.T. prepared the manuscript with edits from K.M.N., P.M, and D.S.B.</p></div><div class="section COI-statement" id="sec-22"><h2 class="">COMPETING INTERESTS</h2><p id="p-42">The authors declare no competing interests.</p></div><div class="section ack" id="ack-1"><h2 class="">ACKNOWLEDGEMENTS</h2><p id="p-40">This material is based upon work supported by the Ocean Acidification Program and Northwest Fisheries Science Center of the National Oceanic and Atmospheric Administration, and in part by the National Science Foundation Graduate Research Fellowship Program and Graduate Research Internship Program (to S.A.T.). S.A.T. is supported in part by the Mary K. Chapman Foundation. We thank the current and former members of the Ocean Acidification lab at the Northwest Fisheries Science Center Mukilteo Research Station, members of the West Coast Metabolomics Core and Fiehn lab at University of California Davis.</p></div><div class="section ref-list" id="ref-list-1"><h2 class="">REFERENCES</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label">1.</span><a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference 1. in text" id="ref-1">↵</a><div class="cit ref-cit ref-journal" id="cit-574798v2.1" data-doi="10.5194/essd-8-605-2016"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Le Quéré</span>,  <span class="cit-name-given-names">C.</span></span> <span class="cit-etal">et al.</span> <span class="cit-article-title">Global Carbon Budget 2016</span>. <abbr class="cit-jnl-abbrev">Earth Syst. Sci. Data</abbr> <span class="cit-vol">8</span>, <span class="cit-fpage">605</span>–<span class="cit-lpage">649</span> (<span class="cit-pub-date">2016</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEarth%2BSyst.%2BSci.%2BData%26rft.volume%253D8%26rft.spage%253D605%26rft_id%253Dinfo%253Adoi%252F10.5194%252Fessd-8-605-2016%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.5194/essd-8-605-2016&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label">2.</span><a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference 2. in text" id="ref-2">↵</a><div class="cit ref-cit ref-journal" id="cit-574798v2.2" data-doi="10.1038/425365a"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Caldeira</span>,  <span class="cit-name-given-names">K.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Wickett</span>,  <span class="cit-name-given-names">M. E.</span></span> <span class="cit-article-title">Anthropogenic carbon and ocean pH</span>. <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">425</span>, <span class="cit-fpage">365</span> (<span class="cit-pub-date">2003</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.volume%253D425%26rft.issue%253D6956%26rft.spage%253D365%26rft.atitle%253DAnthropogenic%2Bcarbon%2Band%2Bocean%2BpH%26rft_id%253Dinfo%253Adoi%252F10.1038%252F425365a%26rft_id%253Dinfo%253Apmid%252F14508477%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/425365a&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=2004082942&amp;link_type=GEOREF" class="cit-ref-sprinkles cit-ref-sprinkles-georef"><span>GeoRef</span></a><a href="/lookup/external-ref?access_num=14508477&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F03%2F14%2F574798.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000185502300029&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">3.</span><a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference 3. in text" id="ref-3">↵</a><div class="cit ref-cit ref-journal" id="cit-574798v2.3" data-doi="10.1126/science.1097329"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Feely</span>,  <span class="cit-name-given-names">R. A.</span></span> <span class="cit-etal">et al.</span> <span class="cit-article-title">Impact of anthropogenic CO2 on the CaCO3 system in the oceans</span>. <abbr class="cit-jnl-abbrev">Science</abbr>. <span class="cit-vol">305</span>, <span class="cit-fpage">362</span>–<span class="cit-lpage">366</span> (<span class="cit-pub-date">2004</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DFeely%26rft.auinit1%253DR.%2BA.%26rft.volume%253D305%26rft.issue%253D5682%26rft.spage%253D362%26rft.epage%253D366%26rft.atitle%253DImpact%2Bof%2BAnthropogenic%2BCO2%2Bon%2Bthe%2BCaCO3%2BSystem%2Bin%2Bthe%2BOceans%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1097329%26rft_id%253Dinfo%253Apmid%252F15256664%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEyOiIzMDUvNTY4Mi8zNjIiO3M6NDoiYXRvbSI7czozNzoiL2Jpb3J4aXYvZWFybHkvMjAxOS8wMy8xNC81NzQ3OTguYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">4.</span><a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference 4. in text" id="ref-4">↵</a><div class="cit ref-cit ref-journal" id="cit-574798v2.4"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Gobler</span>,  <span class="cit-name-given-names">C. J.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Baumann</span>,  <span class="cit-name-given-names">H.</span></span> <span class="cit-article-title">Hypoxia and acidification in ocean ecosystems: coupled dynamics and effects on marine life</span>. <abbr class="cit-jnl-abbrev">Biol. Lett</abbr>. <span class="cit-vol">12</span>, (<span class="cit-pub-date">2016</span>).</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">5.</span><a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference 5. in text" id="ref-5">↵</a><div class="cit ref-cit ref-journal" id="cit-574798v2.5"><div class="cit-metadata"><cite><span class="cit-auth cit-collab">National Marine Fisheries Service</span>. <abbr class="cit-jnl-abbrev">Fisheries of the United States. Fisheries of the United States, 2016</abbr>. (<span class="cit-pub-date">2017</span>).</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">6.</span><a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference 6. in text" id="ref-6">↵</a><div class="cit ref-cit ref-journal" id="cit-574798v2.6"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Marshall</span>,  <span class="cit-name-given-names">K. N.</span></span> <span class="cit-etal">et al.</span> <span class="cit-article-title">Risks of ocean acidification in the California Current food web and fisheries: ecosystem model projections</span>. <abbr class="cit-jnl-abbrev">Glob. Chang. Biol</abbr>. <span class="cit-vol">23</span>, <span class="cit-fpage">1525</span>–<span class="cit-lpage">1539</span> (<span class="cit-pub-date">2017</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGlob.%2BChang.%2BBiol%26rft.volume%253D23%26rft.spage%253D1525%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">7.</span><a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference 7. in text" id="ref-7">↵</a><div class="cit ref-cit ref-journal" id="cit-574798v2.7"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Hunter</span>,  <span class="cit-name-given-names">K. C.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Rudy</span>,  <span class="cit-name-given-names">P. P.</span></span> <span class="cit-article-title">Osmotic and ionic regulation in the Dungeness crab, Cancer magister dana</span>. <abbr class="cit-jnl-abbrev">Comp. Biochem. Physiol. Part A</abbr> <span class="cit-vol">51</span>, <span class="cit-fpage">439</span>–<span class="cit-lpage">447</span> (<span class="cit-pub-date">1975</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DComparative%2BBiochemistry%2Band%2BPhysiology.%2BPart%2BB%252C%2BBiochemistry%2B%2526%2BMolecular%2BBiology%26rft.stitle%253DComparative%2BBiochemistry%2Band%2BPhysiology.%2BPart%2BB%252C%2BBiochemistry%2B%2526%2BMolecular%2BBiology%26rft.aulast%253DLafont%26rft.auinit1%253DR.%26rft.volume%253D51%26rft.issue%253D4%26rft.spage%253D439%26rft.epage%253D444%26rft.atitle%253DDevelopmental%2Bstudies%2Bin%2BPieris%2Bbrassicae%2B%2528Lepidoptera%2529--I.%2BGrowth%2Bof%2Bvarious%2Btissues%2Bduring%2Bthe%2Blast%2Blarval%2Binstar.%26rft_id%253Dinfo%253Apmid%252F1149430%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=1149430&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F03%2F14%2F574798.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">8.</span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-574798v2.8"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Engelhardt</span>,  <span class="cit-name-given-names">F. R.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Dehnel</span>,  <span class="cit-name-given-names">P. A.</span></span> <span class="cit-article-title">Ionic regulation in the Pacific edible crab, <em>Cancer magister</em> (Dana)</span>. <abbr class="cit-jnl-abbrev">Can. J. Zool</abbr>. <span class="cit-vol">51</span>, <span class="cit-fpage">735</span>–<span class="cit-lpage">743</span> (<span class="cit-pub-date">1973</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCan.%2BJ.%2BZool%26rft.volume%253D51%26rft.spage%253D735%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">9.</span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-574798v2.9"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Freire</span>,  <span class="cit-name-given-names">C. A.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Onken</span>,  <span class="cit-name-given-names">H.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">McNamara</span>,  <span class="cit-name-given-names">J. C.</span></span> <span class="cit-article-title">A structure-function analysis of ion transport in crustacean gills and excretory organs</span>. <abbr class="cit-jnl-abbrev">Comparative Biochemistry and Physiology - A Molecular and Integrative Physiology</abbr> <span class="cit-vol">151</span>, <span class="cit-fpage">272</span>–<span class="cit-lpage">304</span> (<span class="cit-pub-date">2008</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DComparative%2BBiochemistry%2Band%2BPhysiology%2B-%2BA%2BMolecular%2Band%2BIntegrative%2BPhysiology%26rft.volume%253D151%26rft.spage%253D272%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">10.</span><a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference 10. in text" id="ref-10">↵</a><div class="cit ref-cit ref-journal" id="cit-574798v2.10"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Jones</span>,  <span class="cit-name-given-names">L. L.</span></span> <span class="cit-article-title">Osmotic regulation in several crabs of the pacific coast of north america</span>. <abbr class="cit-jnl-abbrev">J. Cell. Comp. Physiol</abbr>. <span class="cit-vol">18</span>, <span class="cit-fpage">79</span>–<span class="cit-lpage">92</span> (<span class="cit-pub-date">1941</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BCell.%2BComp.%2BPhysiol%26rft.volume%253D18%26rft.spage%253D79%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=000200174800008&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">11.</span><a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference 11. in text" id="ref-11">↵</a><div class="cit ref-cit ref-journal" id="cit-574798v2.11" data-doi="10.1093/icb/32.3.407"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Henry</span>,  <span class="cit-name-given-names">R. P.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Wheatly</span>,  <span class="cit-name-given-names">M. G.</span></span> <span class="cit-article-title">Interaction of respiration, ion regulation, and acid-base balance in the everyday life of aquatic crustaceans</span>. <abbr class="cit-jnl-abbrev">Integr. Comp. Biol</abbr>. <span class="cit-vol">32</span>, <span class="cit-fpage">407</span>–<span class="cit-lpage">416</span> (<span class="cit-pub-date">1992</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DIntegrative%2Band%2BComparative%2BBiology%26rft.stitle%253DIntegr.%2BComp.%2BBiol.%26rft.volume%253D32%26rft.issue%253D3%26rft.spage%253D407%26rft.epage%253D416%26rft.atitle%253DInteraction%2Bof%2BRespiration%252C%2BIon%2BRegulation%252C%2Band%2BAcid-Base%2BBalance%2Bin%2Bthe%2BEveryday%2BLife%2Bof%2BAquatic%2BCrustaceans%26rft_id%253Dinfo%253Adoi%252F10.1093%252Ficb%252F32.3.407%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/icb/32.3.407&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label">12.</span><a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference 12. in text" id="ref-12">↵</a><div class="cit ref-cit ref-journal" id="cit-574798v2.12" data-doi="10.1007/s00227-014-2409-7"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Hans</span>,  <span class="cit-name-given-names">S.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Fehsenfeld</span>,  <span class="cit-name-given-names">S.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Treberg</span>,  <span class="cit-name-given-names">J. R.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Weihrauch</span>,  <span class="cit-name-given-names">D.</span></span> <span class="cit-article-title">Acid-base regulation in the Dungeness crab (Metacarcinus magister)</span>. <abbr class="cit-jnl-abbrev">Mar. Biol</abbr>. <span class="cit-vol">161</span>, <span class="cit-fpage">1179</span>–<span class="cit-lpage">1193</span> (<span class="cit-pub-date">2014</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMar.%2BBiol%26rft.volume%253D161%26rft.spage%253D1179%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00227-014-2409-7%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00227-014-2409-7&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label">13.</span><a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference 13. in text" id="ref-13">↵</a><div class="cit ref-cit ref-journal" id="cit-574798v2.13"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Miller</span>,  <span class="cit-name-given-names">J. J.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Maher</span>,  <span class="cit-name-given-names">M.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Bohaboy</span>,  <span class="cit-name-given-names">E.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Friedman</span>,  <span class="cit-name-given-names">C. S.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">McElhany</span>,  <span class="cit-name-given-names">P.</span></span> <span class="cit-article-title">Exposure to low pH reduces survival and delays development in early life stages of Dungeness crab (Cancer magister)</span>. <abbr class="cit-jnl-abbrev">Mar. Biol</abbr>. <span class="cit-vol">163</span>, <span class="cit-fpage">118</span> (<span class="cit-pub-date">2016</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMar.%2BBiol%26rft.volume%253D163%26rft.spage%253D118%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">14.</span><a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference 14. in text" id="ref-14">↵</a><div class="cit ref-cit ref-journal" id="cit-574798v2.14"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Bernatis</span>,  <span class="cit-name-given-names">J. L.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Gerstenberger</span>,  <span class="cit-name-given-names">S. L.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">McGaw</span>,  <span class="cit-name-given-names">I. J.</span></span> <span class="cit-article-title">Behavioural responses of the Dungeness crab, Cancer magister, during feeding and digestion in hypoxic conditions</span>. <abbr class="cit-jnl-abbrev">MaiBiol</abbr>. <span class="cit-vol">150</span>, <span class="cit-fpage">941</span>–<span class="cit-lpage">951</span> (<span class="cit-pub-date">2007</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMaiBiol%26rft.volume%253D150%26rft.spage%253D941%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">15.</span><a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference 15. in text" id="ref-15">↵</a><div class="cit ref-cit ref-journal" id="cit-574798v2.15"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Airriess</span>,  <span class="cit-name-given-names">C.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Mcmahon</span>,  <span class="cit-name-given-names">B.</span></span> <span class="cit-article-title">Cardiovascular adaptations enhance tolerance of environmental hypoxia in the crab Cancer magister</span>. <abbr class="cit-jnl-abbrev">J. Exp. Biol</abbr>. <span class="cit-vol">190</span>, (<span class="cit-pub-date">1994</span>).</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">16.</span><a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference 16. in text" id="ref-16">↵</a><div class="cit ref-cit ref-journal" id="cit-574798v2.16" data-doi="10.1016/j.jaci.2017.04.021"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Turi</span>,  <span class="cit-name-given-names">K. N.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Romick-Rosendale</span>,  <span class="cit-name-given-names">L.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Ryckman</span>,  <span class="cit-name-given-names">K. K.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Hartert</span>,  <span class="cit-name-given-names">T. V.</span></span> <span class="cit-article-title">A review of metabolomics approaches and their application in identifying causal pathways of childhood asthma</span>. <abbr class="cit-jnl-abbrev">Journal of Allergy and Clinical Immunology</abbr> (<span class="cit-pub-date">2016</span>).<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1016/j.jaci.2017.04.021</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BAllergy%2Band%2BClinical%2BImmunology%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.jaci.2017.04.021%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.jaci.2017.04.021&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label">17.</span><a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference 17. in text" id="ref-17">↵</a><div class="cit ref-cit ref-journal" id="cit-574798v2.17" data-doi="10.1139/er-2013-0011"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Lankadurai</span>,  <span class="cit-name-given-names">B. P.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Nagato</span>,  <span class="cit-name-given-names">E. G.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Simpson</span>,  <span class="cit-name-given-names">M. J.</span></span> <span class="cit-article-title">Environmental metabolomics: an emerging approach to study organism responses to environmental stressors</span>. <abbr class="cit-jnl-abbrev">Environ. Rev</abbr>. <span class="cit-vol">21</span>, <span class="cit-fpage">180</span>–<span class="cit-lpage">205</span> (<span class="cit-pub-date">2013</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEnviron.%2BRev%26rft.volume%253D21%26rft.spage%253D180%26rft_id%253Dinfo%253Adoi%252F10.1139%252Fer-2013-0011%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1139/er-2013-0011&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label">18.</span><a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference 18. in text" id="ref-18">↵</a><div class="cit ref-cit ref-journal" id="cit-574798v2.18"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Sogin</span>,  <span class="cit-name-given-names">E. M.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Putnam</span>,  <span class="cit-name-given-names">H. M.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Anderson</span>,  <span class="cit-name-given-names">P. E.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Gates</span>,  <span class="cit-name-given-names">R. D.</span></span> <span class="cit-article-title">Metabolomic signatures of increases in temperature and ocean acidification from the reef-building coral, Pocillopora damicornis</span>. <abbr class="cit-jnl-abbrev">Metabolomics</abbr> <span class="cit-vol">12</span>, (<span class="cit-pub-date">2016</span>).</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">19.</span><a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference 19. in text" id="ref-19">↵</a><div class="cit ref-cit ref-journal" id="cit-574798v2.19" data-doi="10.1021/acs.analchem.7b03404"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Cajka</span>,  <span class="cit-name-given-names">T.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Smilowitz</span>,  <span class="cit-name-given-names">J. T.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Fiehn</span>,  <span class="cit-name-given-names">O.</span></span> <span class="cit-article-title">Validating Quantitative Untargeted Lipidomics Across Nine Liquid Chromatography–High-Resolution Mass Spectrometry Platforms</span>. <abbr class="cit-jnl-abbrev">Anal. Chem</abbr>. <span class="cit-vol">89</span>, <span class="cit-fpage">12360</span>–<span class="cit-lpage">12368</span> (<span class="cit-pub-date">2017</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAnal.%2BChem%26rft.volume%253D89%26rft.spage%253D12360%26rft_id%253Dinfo%253Adoi%252F10.1021%252Facs.analchem.7b03404%26rft_id%253Dinfo%253Apmid%252F29064229%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1021/acs.analchem.7b03404&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=29064229&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F03%2F14%2F574798.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">20.</span><a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference 20. in text" id="ref-20">↵</a><div class="cit ref-cit ref-journal" id="cit-574798v2.20" data-doi="10.1021/ac9019522"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Kind</span>,  <span class="cit-name-given-names">T.</span></span> <span class="cit-etal">et al.</span> <span class="cit-article-title">FiehnLib: mass spectral and retention index libraries for metabolomics based on quadrupole and time-of-flight gas chromatography/mass spectrometry</span>. <abbr class="cit-jnl-abbrev">Anal. Chem</abbr>. <span class="cit-vol">81</span>, <span class="cit-fpage">10038</span>–<span class="cit-lpage">48</span> (<span class="cit-pub-date">2009</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAnalytical%2BChemistry%2B%2528Washington%252C%2BDC%2529%26rft.stitle%253DAnalytical%2BChemistry%2B%2528Washington%252C%2BDC%2529%26rft.aulast%253DKind%26rft.auinit1%253DT.%26rft.volume%253D81%26rft.issue%253D24%26rft.spage%253D10038%26rft.epage%253D10048%26rft.atitle%253DFiehnLib%253A%2Bmass%2Bspectral%2Band%2Bretention%2Bindex%2Blibraries%2Bfor%2Bmetabolomics%2Bbased%2Bon%2Bquadrupole%2Band%2Btime-of-flight%2Bgas%2Bchromatography%252Fmass%2Bspectrometry.%26rft_id%253Dinfo%253Adoi%252F10.1021%252Fac9019522%26rft_id%253Dinfo%253Apmid%252F19928838%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1021/ac9019522&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19928838&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F03%2F14%2F574798.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">21.</span><a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference 21. in text" id="ref-21">↵</a><div class="cit ref-cit ref-journal" id="cit-574798v2.21" data-doi="10.1038/nmeth.2551"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Kind</span>,  <span class="cit-name-given-names">T.</span></span> <span class="cit-etal">et al.</span> <span class="cit-article-title">LipidBlast in silico tandem mass spectrometry database for lipid identification</span>. <abbr class="cit-jnl-abbrev">Nat. Methods</abbr> <span class="cit-vol">10</span>, <span class="cit-fpage">755</span>–<span class="cit-lpage">8</span> (<span class="cit-pub-date">2013</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat.%2BMethods%26rft.volume%253D10%26rft.spage%253D755%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnmeth.2551%26rft_id%253Dinfo%253Apmid%252F23817071%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nmeth.2551&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23817071&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F03%2F14%2F574798.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000322453600025&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">22.</span><a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference 22. in text" id="ref-22">↵</a><div class="cit ref-cit ref-journal" id="cit-574798v2.22" data-doi="10.1186/1471-2105-12-321"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Skogerson</span>,  <span class="cit-name-given-names">K.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Wohlgemuth</span>,  <span class="cit-name-given-names">G.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Barupal</span>,  <span class="cit-name-given-names">D. K.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Fiehn</span>,  <span class="cit-name-given-names">O.</span></span> <span class="cit-article-title">The volatile compound BinBase mass spectral database</span>. <abbr class="cit-jnl-abbrev">BMC Bioinformatics</abbr> <span class="cit-vol">12</span>, <span class="cit-fpage">321</span> (<span class="cit-pub-date">2011</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2Bbioinformatics%2B%255Belectronic%2Bresource%255D%26rft.stitle%253DBMC%2BBioinformatics%26rft.aulast%253DSkogerson%26rft.auinit1%253DK.%26rft.volume%253D12%26rft.spage%253D321%26rft.epage%253D321%26rft.atitle%253DThe%2Bvolatile%2Bcompound%2BBinBase%2Bmass%2Bspectral%2Bdatabase.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2105-12-321%26rft_id%253Dinfo%253Apmid%252F21816034%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2105-12-321&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21816034&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F03%2F14%2F574798.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">23.</span><a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference 23. in text" id="ref-23">↵</a><div class="cit ref-cit ref-journal" id="cit-574798v2.23" data-doi="10.1093/nar/gkx1089"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Wishart</span>,  <span class="cit-name-given-names">D. S.</span></span> <span class="cit-etal">et al.</span> <span class="cit-article-title">HMDB 4.0: the human metabolome database for 2018</span>. <abbr class="cit-jnl-abbrev">Nucleic Acids Res</abbr>. <span class="cit-vol">46</span>, <span class="cit-fpage">D608</span>–<span class="cit-lpage">D617</span> (<span class="cit-pub-date">2018</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes%26rft.volume%253D46%26rft.spage%253DD608%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgkx1089%26rft_id%253Dinfo%253Apmid%252F29140435%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/nar/gkx1089&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=29140435&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F03%2F14%2F574798.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">24.</span><a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference 24. in text" id="ref-24">↵</a><div class="cit ref-cit ref-journal" id="cit-574798v2.24" data-doi="10.1093/nar/gkm324"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Fahy</span>,  <span class="cit-name-given-names">E.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Sud</span>,  <span class="cit-name-given-names">M.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Cotter</span>,  <span class="cit-name-given-names">D.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Subramaniam</span>,  <span class="cit-name-given-names">S.</span></span> <span class="cit-article-title">LIPID MAPS online tools for lipid research</span>. <abbr class="cit-jnl-abbrev">Nucleic Acids Res</abbr>. <span class="cit-vol">35</span>, <span class="cit-fpage">W606</span>–<span class="cit-lpage">W612</span> (<span class="cit-pub-date">2007</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgkm324%26rft_id%253Dinfo%253Apmid%252F17584797%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/nar/gkm324&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17584797&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F03%2F14%2F574798.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000255311500113&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">25.</span><a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference 25. in text" id="ref-25">↵</a><div class="cit ref-cit ref-journal" id="cit-574798v2.25"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Allen</span>,  <span class="cit-name-given-names">W. V.</span></span> <span class="cit-article-title">Lipid transport in the dungeness crab Cancer magister dana</span>. <abbr class="cit-jnl-abbrev">Comp. Biochem. Physiol. Part B Comp. Biochem</abbr>. <span class="cit-vol">43</span>, <span class="cit-fpage">193</span>–<span class="cit-lpage">IN8</span> (<span class="cit-pub-date">1972</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DComp.%2BBiochem.%2BPhysiol.%2BPart%2BB%2BComp.%2BBiochem%26rft.volume%253D43%26rft.spage%253D193%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">26.</span><a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference 26. in text" id="ref-26">↵</a><div class="cit ref-cit ref-journal" id="cit-574798v2.26"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Allen</span>,  <span class="cit-name-given-names">W. V.</span></span> <span class="cit-article-title">Amino Acid and Fatty Acid Composition of Tissues of the Dungeness Crab (<em>Cancer magister</em>)</span>. <abbr class="cit-jnl-abbrev">J. Fish. Res. Board Canada</abbr> <span class="cit-vol">28</span>, <span class="cit-fpage">1191</span>–<span class="cit-lpage">1195</span> (<span class="cit-pub-date">1971</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BFish.%2BRes.%2BBoard%2BCanada%26rft.volume%253D28%26rft.spage%253D1191%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">27.</span><a class="rev-xref-ref" href="#xref-ref-27-1" title="View reference 27. in text" id="ref-27">↵</a><div class="cit ref-cit ref-journal" id="cit-574798v2.27"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Grissa</span>,  <span class="cit-name-given-names">D.</span></span> <span class="cit-etal">et al.</span> <span class="cit-article-title">Feature Selection Methods for Early Predictive Biomarker Discovery Using Untargeted Metabolomic Data</span>. <abbr class="cit-jnl-abbrev">Front. Mol. Biosci</abbr>. <span class="cit-vol">3</span>, <span class="cit-fpage">30</span> (<span class="cit-pub-date">2016</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DFront.%2BMol.%2BBiosci%26rft.volume%253D3%26rft.spage%253D30%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">28.</span><a class="rev-xref-ref" href="#xref-ref-28-1" title="View reference 28. in text" id="ref-28">↵</a><div class="cit ref-cit ref-book" id="cit-574798v2.28"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Witte</span>,  <span class="cit-name-given-names">R. S.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Witte</span>,  <span class="cit-name-given-names">J. S.</span></span> <span class="cit-source">Statistics</span>. (<span class="cit-publ-name">J. Wiley &amp; Sons</span>, <span class="cit-pub-date">2010</span>).</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">29.</span><a class="rev-xref-ref" href="#xref-ref-29-1" title="View reference 29. in text" id="ref-29">↵</a><div class="cit ref-cit ref-journal" id="cit-574798v2.29" data-doi="10.3390/metabo2040775"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Vinaixa</span>,  <span class="cit-name-given-names">M.</span></span> <span class="cit-etal">et al.</span> <span class="cit-article-title">A Guideline to Univariate Statistical Analysis for LC/MS-Based Untargeted Metabolomics-Derived Data</span>. <abbr class="cit-jnl-abbrev">Metabolites</abbr> <span class="cit-vol">2</span>, <span class="cit-fpage">775</span>–<span class="cit-lpage">95</span> (<span class="cit-pub-date">2012</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMetabolites%26rft.volume%253D2%26rft.spage%253D775%26rft_id%253Dinfo%253Adoi%252F10.3390%252Fmetabo2040775%26rft_id%253Dinfo%253Apmid%252F24957762%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3390/metabo2040775&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24957762&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F03%2F14%2F574798.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">30.</span><a class="rev-xref-ref" href="#xref-ref-30-1" title="View reference 30. in text" id="ref-30">↵</a><div class="cit ref-cit ref-journal" id="cit-574798v2.30"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Timmerman</span>,  <span class="cit-name-given-names">M. E.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Hoefsloot</span>,  <span class="cit-name-given-names">H. C. J.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Smilde</span>,  <span class="cit-name-given-names">A. K.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Ceulemans</span>,  <span class="cit-name-given-names">E.</span></span> <span class="cit-article-title">Scaling in ANOVA-simultaneous component analysis</span>. <abbr class="cit-jnl-abbrev">Metabolomics</abbr> <span class="cit-vol">11</span>, <span class="cit-fpage">1265</span>–<span class="cit-lpage">1276</span> (<span class="cit-pub-date">2015</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMetabolomics%26rft.volume%253D11%26rft.spage%253D1265%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">31.</span><a class="rev-xref-ref" href="#xref-ref-31-1" title="View reference 31. in text" id="ref-31">↵</a><div class="cit ref-cit ref-journal" id="cit-574798v2.31" data-doi="10.1146/annurev.nu.11.070191.002251"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Broquist</span>,  <span class="cit-name-given-names">H. P.</span></span> <span class="cit-article-title">Lysine-Pipecolic Acid Metabolic Relationships in Microbes and Mammals</span>. <abbr class="cit-jnl-abbrev">Annu. Rev. Nutr</abbr>. <span class="cit-vol">11</span>, <span class="cit-fpage">435</span>–<span class="cit-lpage">448</span> (<span class="cit-pub-date">1991</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAnnual%2Breview%2Bof%2Bnutrition%26rft.stitle%253DAnnu%2BRev%2BNutr%26rft.aulast%253DBroquist%26rft.auinit1%253DH.%26rft.volume%253D11%26rft.issue%253D1%26rft.spage%253D435%26rft.epage%253D448%26rft.atitle%253DLysine-Pipecolic%2BAcid%2BMetabolic%2BRelationships%2BIn%2BMicrobes%2BAnd%2BMammals%26rft_id%253Dinfo%253Adoi%252F10.1146%252Fannurev.nu.11.070191.002251%26rft_id%253Dinfo%253Apmid%252F1909881%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1146/annurev.nu.11.070191.002251&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=1909881&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F03%2F14%2F574798.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1991FX66500020&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">32.</span><a class="rev-xref-ref" href="#xref-ref-32-1" title="View reference 32. in text" id="ref-32">↵</a><div class="cit ref-cit ref-journal" id="cit-574798v2.32" data-doi="10.1093/toxsci/kft143"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Cheng</span>,  <span class="cit-name-given-names">J.</span></span> <span class="cit-etal">et al.</span> <span class="cit-article-title">Identification of 2-piperidone as a biomarker of CYP2E1 activity through metabolomic phenotyping</span>. <abbr class="cit-jnl-abbrev">Toxicol. Sci</abbr>. <span class="cit-vol">135</span>, <span class="cit-fpage">37</span>–<span class="cit-lpage">47</span> (<span class="cit-pub-date">2013</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DToxicol.%2BSci%26rft_id%253Dinfo%253Adoi%252F10.1093%252Ftoxsci%252Fkft143%26rft_id%253Dinfo%253Apmid%252F23811823%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/toxsci/kft143&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23811823&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F03%2F14%2F574798.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">33.</span><a class="rev-xref-ref" href="#xref-ref-33-1" title="View reference 33. in text" id="ref-33">↵</a><div class="cit ref-cit ref-journal" id="cit-574798v2.33" data-doi="10.1073/pnas.67.3.1248"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Orlowski</span>,  <span class="cit-name-given-names">M.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Meister</span>,  <span class="cit-name-given-names">A.</span></span> <span class="cit-article-title">The gamma-glutamyl cycle: a possible transport system for amino acids</span>. <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. U. S. A</abbr>. <span class="cit-vol">67</span>, <span class="cit-fpage">1248</span>–<span class="cit-lpage">55</span> (<span class="cit-pub-date">1970</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPNAS%26rft.stitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft.aulast%253DOrlowski%26rft.auinit1%253DM.%26rft.volume%253D67%26rft.issue%253D3%26rft.spage%253D1248%26rft.epage%253D1255%26rft.atitle%253DThe%2B%257Bgamma%257D-Glutamyl%2BCycle%253A%2BA%2BPossible%2BTransport%2BSystem%2Bfor%2BAmino%2BAcids%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.67.3.1248%26rft_id%253Dinfo%253Apmid%252F5274454%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czo5OiI2Ny8zLzEyNDgiO3M6NDoiYXRvbSI7czozNzoiL2Jpb3J4aXYvZWFybHkvMjAxOS8wMy8xNC81NzQ3OTguYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">34.</span><a class="rev-xref-ref" href="#xref-ref-34-1" title="View reference 34. in text" id="ref-34">↵</a><div class="cit ref-cit ref-journal" id="cit-574798v2.34"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Bodnaryk</span>,  <span class="cit-name-given-names">R. P.</span></span> <span class="cit-article-title">Structure and Function of Insect Peptides</span>. <abbr class="cit-jnl-abbrev">Adv. In Insect Phys</abbr>. <span class="cit-vol">13</span>, <span class="cit-fpage">69</span>–<span class="cit-lpage">132</span> (<span class="cit-pub-date">1978</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAdv.%2BIn%2BInsect%2BPhys%26rft.volume%253D13%26rft.spage%253D69%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">35.</span><a class="rev-xref-ref" href="#xref-ref-35-1" title="View reference 35. in text" id="ref-35">↵</a><div class="cit ref-cit ref-journal" id="cit-574798v2.35"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Mika</span>,  <span class="cit-name-given-names">A.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Skorkowski</span>,  <span class="cit-name-given-names">E.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Stepnowski</span>,  <span class="cit-name-given-names">P.</span></span> <span class="cit-article-title">The Use of Different MS Techniques to Determine Glutathione Levels in Marine Tissues</span>. <abbr class="cit-jnl-abbrev">Food Anal. Methods</abbr> <span class="cit-vol">6</span>, <span class="cit-fpage">789</span>–<span class="cit-lpage">802</span> (<span class="cit-pub-date">2013</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DFood%2BAnal.%2BMethods%26rft.volume%253D6%26rft.spage%253D789%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">36.</span><a class="rev-xref-ref" href="#xref-ref-36-1" title="View reference 36. in text" id="ref-36">↵</a><div class="cit ref-cit ref-journal" id="cit-574798v2.36"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Pelletier</span>,  <span class="cit-name-given-names">J.</span></span> <span class="cit-etal">et al.</span> <span class="cit-article-title">Glycogen Synthesis is Induced in Hypoxia by the Hypoxia-Inducible Factor and Promotes Cancer Cell Survival</span>. <abbr class="cit-jnl-abbrev">Front Oncol</abbr> <span class="cit-vol">2</span>, <span class="cit-fpage">18</span> (<span class="cit-pub-date">2012</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DFront%2BOncol%26rft.aulast%253DPelletier%26rft.auinit1%253DJ.%26rft.volume%253D2%26rft.spage%253D18%26rft.epage%253D18%26rft.atitle%253DGlycogen%2BSynthesis%2Bis%2BInduced%2Bin%2BHypoxia%2Bby%2Bthe%2BHypoxia-Inducible%2BFactor%2Band%2BPromotes%2BCancer%2BCell%2BSurvival.%26rft_id%253Dinfo%253Apmid%252F22649778%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=22649778&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F03%2F14%2F574798.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">37.</span><a class="rev-xref-ref" href="#xref-ref-37-1" title="View reference 37. in text" id="ref-37">↵</a><div class="cit ref-cit ref-journal" id="cit-574798v2.37"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Christman</span>,  <span class="cit-name-given-names">A. A.</span></span> <span class="cit-article-title">Purine and pyrimidine metabolism</span>. <abbr class="cit-jnl-abbrev">Physiol. Rev</abbr>. <span class="cit-vol">32</span>, <span class="cit-fpage">303</span>–<span class="cit-lpage">48</span> (<span class="cit-pub-date">1952</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPhysiological%2BReviews%26rft.stitle%253DPhysiol.%2BRev.%26rft.aulast%253DChristman%26rft.auinit1%253DA.%2BA.%26rft.volume%253D32%26rft.issue%253D3%26rft.spage%253D303%26rft.epage%253D348%26rft.atitle%253DPurine%2Band%2BPyrimidine%2BMetabolism%26rft_id%253Dinfo%253Apmid%252F12983226%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=12983226&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F03%2F14%2F574798.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1952UH07000002&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">38.</span><a class="rev-xref-ref" href="#xref-ref-38-1" title="View reference 38. in text" id="ref-38">↵</a><div class="cit ref-cit ref-journal" id="cit-574798v2.38" data-doi="10.1007/BF00164031"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Winefield</span>,  <span class="cit-name-given-names">C. S.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Farnden</span>,  <span class="cit-name-given-names">K. J. F.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Reynolds</span>,  <span class="cit-name-given-names">P. H. S.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Marshall</span>,  <span class="cit-name-given-names">C. J.</span></span> <span class="cit-article-title">Evolutionary analysis of aspartate aminotransferases</span>. <abbr class="cit-jnl-abbrev">J. Mol. Evol</abbr>. <span class="cit-vol">40</span>, <span class="cit-fpage">455</span>–<span class="cit-lpage">463</span> (<span class="cit-pub-date">1995</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BMolecular%2BEvolution%26rft.stitle%253DJournal%2Bof%2BMolecular%2BEvolution%26rft.aulast%253DWinefield%26rft.auinit1%253DC.%2BS.%26rft.volume%253D40%26rft.issue%253D4%26rft.spage%253D455%26rft.epage%253D463%26rft.atitle%253DEvolutionary%2Banalysis%2Bof%2Baspartate%2Baminotransferases.%26rft_id%253Dinfo%253Adoi%252F10.1007%252FBF00164031%26rft_id%253Dinfo%253Apmid%252F7769621%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/BF00164031&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=7769621&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F03%2F14%2F574798.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1995QR37000012&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">39.</span><a class="rev-xref-ref" href="#xref-ref-39-1" title="View reference 39. in text" id="ref-39">↵</a><div class="cit ref-cit ref-journal" id="cit-574798v2.39"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Scher</span>,  <span class="cit-name-given-names">W.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Jakoby</span>,  <span class="cit-name-given-names">W. B.</span></span> <span class="cit-article-title">Maleate isomerase</span>. <abbr class="cit-jnl-abbrev">J. Biol. Chem</abbr>. <span class="cit-vol">244</span>, <span class="cit-fpage">1878</span>–<span class="cit-lpage">1882</span> (<span class="cit-pub-date">1969</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BBiological%2BChemistry%26rft.stitle%253DJ%2BBiol%2BChem%26rft.aulast%253DScher%26rft.auinit1%253DW.%26rft.volume%253D244%26rft.issue%253D7%26rft.spage%253D1878%26rft.epage%253D1882%26rft.atitle%253DMaleate%2BIsomerase%26rft_id%253Dinfo%253Apmid%252F5780844%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiamJjIjtzOjU6InJlc2lkIjtzOjEwOiIyNDQvNy8xODc4IjtzOjQ6ImF0b20iO3M6Mzc6Ii9iaW9yeGl2L2Vhcmx5LzIwMTkvMDMvMTQvNTc0Nzk4LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">40.</span><a class="rev-xref-ref" href="#xref-ref-40-1" title="View reference 40. in text" id="ref-40">↵</a><div class="cit ref-cit ref-journal" id="cit-574798v2.40" data-doi="10.1007/s11306-013-0521-1"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Mamer</span>,  <span class="cit-name-given-names">O.</span></span> <span class="cit-etal">et al.</span> <span class="cit-article-title">The complete targeted profile of the organic acid intermediates of the citric acid cycle using a single stable isotope dilution analysis, sodium borodeuteride reduction and selected ion monitoring GC/MS</span>. <abbr class="cit-jnl-abbrev">Metabolomics</abbr> <span class="cit-vol">9</span>, <span class="cit-fpage">1019</span>–<span class="cit-lpage">1030</span> (<span class="cit-pub-date">2013</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMetabolomics%26rft.volume%253D9%26rft.spage%253D1019%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs11306-013-0521-1%26rft_id%253Dinfo%253Apmid%252F24348278%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s11306-013-0521-1&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24348278&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F03%2F14%2F574798.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">41.</span><a class="rev-xref-ref" href="#xref-ref-41-1" title="View reference 41. in text" id="ref-41">↵</a><div class="cit ref-cit ref-book" id="cit-574798v2.41" data-doi="10.1016/B978-0-12-391909-0.50034-7"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Engelking</span>,  <span class="cit-name-given-names">L. R.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Engelking</span>,  <span class="cit-name-given-names">L. R.</span></span> in <span class="cit-source">Textbook of Veterinary Physiological Chemistry</span> <span class="cit-fpage">208</span>–<span class="cit-lpage">213</span> (<span class="cit-publ-name">Elsevier</span>, <span class="cit-pub-date">2015</span>).<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1016/B978-0-12-391909-0.50034-7</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTextbook%2Bof%2BVeterinary%2BPhysiological%2BChemistry%26rft_id%253Dinfo%253Adoi%252F10.1016%252FB978-0-12-391909-0.50034-7%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/B978-0-12-391909-0.50034-7&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label">42.</span><a class="rev-xref-ref" href="#xref-ref-42-1" title="View reference 42. in text" id="ref-42">↵</a><div class="cit ref-cit ref-journal" id="cit-574798v2.42"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Menon</span>,  <span class="cit-name-given-names">G. K. K.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Friedman</span>,  <span class="cit-name-given-names">D. L.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Stern</span>,  <span class="cit-name-given-names">J. R.</span></span> <span class="cit-article-title">Enzymic synthesis of glutaryl-coenzyme A</span>. <abbr class="cit-jnl-abbrev">Biochim. Biophys. Acta</abbr> <span class="cit-vol">44</span>, <span class="cit-fpage">375</span>–<span class="cit-lpage">377</span> (<span class="cit-pub-date">1960</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBiochimica%2Bet%2BBiophysica%2BActa.%2BProtein%2BStructure%2Band%2BMolecular%2BEnzymology%26rft.stitle%253DBiochimica%2Bet%2BBiophysica%2BActa.%2BProtein%2BStructure%2Band%2BMolecular%2BEnzymology%26rft.aulast%253DMenon%26rft.auinit1%253DG.%2BK.%26rft.volume%253D44%26rft.spage%253D375%26rft.epage%253D377%26rft.atitle%253DEnzymic%2Bsynthesis%2Bof%2Bglutaryl-coenzyme%2BA.%26rft_id%253Dinfo%253Apmid%252F13769477%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=13769477&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F03%2F14%2F574798.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">43.</span><a class="rev-xref-ref" href="#xref-ref-43-1" title="View reference 43. in text" id="ref-43">↵</a><div class="cit ref-cit ref-journal" id="cit-574798v2.43" data-doi="10.1074/jbc.M502845200"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Sauer</span>,  <span class="cit-name-given-names">S. W.</span></span> <span class="cit-etal">et al.</span> <span class="cit-article-title">Bioenergetics in glutaryl-coenzyme A dehydrogenase deficiency: a role for glutaryl-coenzyme A</span>. <abbr class="cit-jnl-abbrev">J. Biol. Chem</abbr>. <span class="cit-vol">280</span>, <span class="cit-fpage">21830</span>–<span class="cit-lpage">6</span> (<span class="cit-pub-date">2005</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BBiol.%2BChem%26rft_id%253Dinfo%253Adoi%252F10.1074%252Fjbc.M502845200%26rft_id%253Dinfo%253Apmid%252F15840571%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiamJjIjtzOjU6InJlc2lkIjtzOjEyOiIyODAvMjMvMjE4MzAiO3M6NDoiYXRvbSI7czozNzoiL2Jpb3J4aXYvZWFybHkvMjAxOS8wMy8xNC81NzQ3OTguYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">44.</span><a class="rev-xref-ref" href="#xref-ref-44-1" title="View reference 44. in text" id="ref-44">↵</a><div class="cit ref-cit ref-journal" id="cit-574798v2.44"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Gibellini</span>,  <span class="cit-name-given-names">F.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Smith</span>,  <span class="cit-name-given-names">T. K.</span></span> <span class="cit-article-title">The Kennedy pathway-De novo synthesis of phosphatidylethanolamine and phosphatidylcholine</span>. <abbr class="cit-jnl-abbrev">IUBMB Life</abbr> <span class="cit-vol">62</span>, n/a-n/a (<span class="cit-pub-date">2010</span>).</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">45.</span><a class="rev-xref-ref" href="#xref-ref-45-1" title="View reference 45. in text" id="ref-45">↵</a><div class="cit ref-cit ref-book" id="cit-574798v2.45"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Mathews</span>,  <span class="cit-name-given-names">C. K.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Van Holde</span>,  <span class="cit-name-given-names">K. E.</span></span> (<span class="cit-auth"><span class="cit-name-surname">Kensal</span>  <span class="cit-name-given-names">E.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Ahern</span>,  <span class="cit-name-given-names">K. G.</span></span> <span class="cit-source">Biochemistry</span>. (<span class="cit-publ-name">Benjamin Cummings</span>, <span class="cit-pub-date">2000</span>).</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">46.</span><a class="rev-xref-ref" href="#xref-ref-46-1" title="View reference 46. in text" id="ref-46">↵</a><div class="cit ref-cit ref-book" id="cit-574798v2.46" data-doi="10.1016/B978-0-12-802167-5.00019-0"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Courtney-Martin</span>,  <span class="cit-name-given-names">G.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Pencharz</span>,  <span class="cit-name-given-names">P. B.</span></span> in <span class="cit-source">The Molecular Nutrition of Amino Acids and Proteins</span> <span class="cit-fpage">265</span>–<span class="cit-lpage">286</span> (<span class="cit-publ-name">Elsevier</span>, <span class="cit-pub-date">2016</span>).<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1016/B978-0-12-802167-5.00019-0</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DThe%2BMolecular%2BNutrition%2Bof%2BAmino%2BAcids%2Band%2BProteins%26rft_id%253Dinfo%253Adoi%252F10.1016%252FB978-0-12-802167-5.00019-0%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/B978-0-12-802167-5.00019-0&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label">47.</span><a class="rev-xref-ref" href="#xref-ref-47-1" title="View reference 47. in text" id="ref-47">↵</a><div class="cit ref-cit ref-journal" id="cit-574798v2.47"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Jackson</span>,  <span class="cit-name-given-names">T. M.</span></span> &amp; O’<span class="cit-auth"><span class="cit-name-surname">Malley</span>,  <span class="cit-name-given-names">K. G.</span></span> <span class="cit-article-title">Comparing genetic connectivity among Dungeness crab (Cancer magister) inhabiting Puget Sound and coastal Washington</span>. <abbr class="cit-jnl-abbrev">Mar. Biol</abbr>. <span class="cit-vol">164</span>, (<span class="cit-pub-date">2017</span>).</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">48.</span><a class="rev-xref-ref" href="#xref-ref-48-1" title="View reference 48. in text" id="ref-48">↵</a><div class="cit ref-cit ref-journal" id="cit-574798v2.48" data-doi="10.1111/mec.14443"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Jackson</span>,  <span class="cit-name-given-names">T. M.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Roegner</span>,  <span class="cit-name-given-names">G. C.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">O’Malley</span>,  <span class="cit-name-given-names">K. G.</span></span> <span class="cit-article-title">Evidence for interannual variation in genetic structure of Dungeness crab (Cancer magister) along the California Current System</span>. <abbr class="cit-jnl-abbrev">Mol. Ecol</abbr>. (<span class="cit-pub-date">2018</span>).<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1111/mec.14443</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BEcol%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fmec.14443%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/mec.14443&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label">49.</span><a class="rev-xref-ref" href="#xref-ref-49-1" title="View reference 49. in text" id="ref-49">↵</a><div class="cit ref-cit ref-journal" id="cit-574798v2.49"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Chong</span>,  <span class="cit-name-given-names">E. Y.</span></span> <span class="cit-etal">et al.</span> <span class="cit-article-title">Local false discovery rate estimation using feature reliability in LC/MS metabolomics data</span>. <abbr class="cit-jnl-abbrev">Sci. Rep</abbr>. <span class="cit-vol">5</span>, <span class="cit-fpage">17221</span> (<span class="cit-pub-date">2015</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DSci.%2BRep%26rft.volume%253D5%26rft.spage%253D17221%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">50.</span><a class="rev-xref-ref" href="#xref-ref-50-1" title="View reference 50. in text" id="ref-50">↵</a><div class="cit ref-cit ref-journal" id="cit-574798v2.50" data-doi="10.1371/journal.pone.0043474"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Ding</span>,  <span class="cit-name-given-names">M.-Z.</span></span> <span class="cit-etal">et al.</span> <span class="cit-article-title">Proteomic Research Reveals the Stress Response and Detoxification of Yeast to Combined Inhibitors</span>. <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">7</span>, <span class="cit-fpage">e43474</span> (<span class="cit-pub-date">2012</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BONE%26rft.aulast%253DDing%26rft.auinit1%253DM.%2BZ.%26rft.volume%253D7%26rft.issue%253D8%26rft.spage%253De43474%26rft.epage%253De43474%26rft.atitle%253DProteomic%2Bresearch%2Breveals%2Bthe%2Bstress%2Bresponse%2Band%2Bdetoxification%2Bof%2Byeast%2Bto%2Bcombined%2Binhibitors.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0043474%26rft_id%253Dinfo%253Apmid%252F22952687%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0043474&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22952687&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F03%2F14%2F574798.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">51.</span><a class="rev-xref-ref" href="#xref-ref-51-1" title="View reference 51. in text" id="ref-51">↵</a><div class="cit ref-cit ref-journal" id="cit-574798v2.51" data-doi="10.1016/S1097-2765(03)00105-9"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Harding</span>,  <span class="cit-name-given-names">H. P.</span></span> <span class="cit-etal">et al.</span> <span class="cit-article-title">An Integrated Stress Response Regulates Amino Acid Metabolism and Resistance to Oxidative Stress</span>. <abbr class="cit-jnl-abbrev">Mol. Cell</abbr> <span class="cit-vol">11</span>, <span class="cit-fpage">619</span>–<span class="cit-lpage">633</span> (<span class="cit-pub-date">2003</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2Bcell%26rft.stitle%253DMol%2BCell%26rft.aulast%253DHarding%26rft.auinit1%253DH.%2BP.%26rft.volume%253D11%26rft.issue%253D3%26rft.spage%253D619%26rft.epage%253D633%26rft.atitle%253DAn%2Bintegrated%2Bstress%2Bresponse%2Bregulates%2Bamino%2Bacid%2Bmetabolism%2Band%2Bresistance%2Bto%2Boxidative%2Bstress.%26rft_id%253Dinfo%253Adoi%252F10.1016%252FS1097-2765%252803%252900105-9%26rft_id%253Dinfo%253Apmid%252F12667446%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/S1097-2765(03)00105-9&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=12667446&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F03%2F14%2F574798.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000182049700009&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">52.</span><a class="rev-xref-ref" href="#xref-ref-52-1" title="View reference 52. in text" id="ref-52">↵</a><div class="cit ref-cit ref-journal" id="cit-574798v2.52"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Murray</span>,  <span class="cit-name-given-names">A. J.</span></span> <span class="cit-article-title">Metabolic adaptation of skeletal muscle to high altitude hypoxia: How new technologies could resolve the controversies</span>. <abbr class="cit-jnl-abbrev">Genome Med</abbr>. <span class="cit-vol">1</span>, (<span class="cit-pub-date">2009</span>).</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">53.</span><a class="rev-xref-ref" href="#xref-ref-53-1" title="View reference 53. in text" id="ref-53">↵</a><div class="cit ref-cit ref-journal" id="cit-574798v2.53" data-doi="10.1152/physiolgenomics.00025.2009"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Nava</span>,  <span class="cit-name-given-names">G. M.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Lee</span>,  <span class="cit-name-given-names">D. Y.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Ospina</span>,  <span class="cit-name-given-names">J. H.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Cai</span>,  <span class="cit-name-given-names">S.-Y.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Gaskins</span>,  <span class="cit-name-given-names">H. R.</span></span> <span class="cit-article-title">Genomic analyses reveal a conserved glutathione homeostasis pathway in the invertebrate chordate Ciona intestinalis</span>. <abbr class="cit-jnl-abbrev">Physiol. Genomics</abbr> <span class="cit-vol">39</span>, <span class="cit-fpage">183</span>–<span class="cit-lpage">94</span> (<span class="cit-pub-date">2009</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPhysiol.%2BGenomics%26rft_id%253Dinfo%253Adoi%252F10.1152%252Fphysiolgenomics.00025.2009%26rft_id%253Dinfo%253Apmid%252F19470804%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1152/physiolgenomics.00025.2009&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19470804&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F03%2F14%2F574798.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000271525900006&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">54.</span><a class="rev-xref-ref" href="#xref-ref-54-1" title="View reference 54. in text" id="ref-54">↵</a><div class="cit ref-cit ref-journal" id="cit-574798v2.54" data-doi="10.1074/jbc.M106683200"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Fraser</span>,  <span class="cit-name-given-names">J. A.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Saunders</span>,  <span class="cit-name-given-names">R. D. C.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">McLellan</span>,  <span class="cit-name-given-names">L. I.</span></span> <span class="cit-article-title">Drosophila melanogaster glutamate-cysteine ligase activity is regulated by a modifier subunit with a mechanism of action similar to that of the mammalian form</span>. <abbr class="cit-jnl-abbrev">J. Biol. Chem</abbr>. <span class="cit-vol">277</span>, <span class="cit-fpage">1158</span>–<span class="cit-lpage">65</span> (<span class="cit-pub-date">2002</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BBiol.%2BChem%26rft_id%253Dinfo%253Adoi%252F10.1074%252Fjbc.M106683200%26rft_id%253Dinfo%253Apmid%252F11698394%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiamJjIjtzOjU6InJlc2lkIjtzOjEwOiIyNzcvMi8xMTU4IjtzOjQ6ImF0b20iO3M6Mzc6Ii9iaW9yeGl2L2Vhcmx5LzIwMTkvMDMvMTQvNTc0Nzk4LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">55.</span><a class="rev-xref-ref" href="#xref-ref-55-1" title="View reference 55. in text" id="ref-55">↵</a><div class="cit ref-cit ref-journal" id="cit-574798v2.55" data-doi="10.1152/japplphysiol.00449.2004"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Mansfield</span>,  <span class="cit-name-given-names">K. D.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Simon</span>,  <span class="cit-name-given-names">M. C.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Keith</span>,  <span class="cit-name-given-names">B.</span></span> <span class="cit-article-title">Hypoxic reduction in cellular glutathione levels requires mitochondrial reactive oxygen species</span>. <abbr class="cit-jnl-abbrev">J. Appl. Physiol</abbr>. <span class="cit-vol">97</span>, <span class="cit-fpage">1358</span>–<span class="cit-lpage">1366</span> (<span class="cit-pub-date">2004</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BAppl.%2BPhysiol%26rft_id%253Dinfo%253Adoi%252F10.1152%252Fjapplphysiol.00449.2004%26rft_id%253Dinfo%253Apmid%252F15180977%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1152/japplphysiol.00449.2004&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15180977&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F03%2F14%2F574798.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000224247700025&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">56.</span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-574798v2.56" data-doi="10.1006/jmcc.2002.2046"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Cargnoni</span>,  <span class="cit-name-given-names">A.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Ceconi</span>,  <span class="cit-name-given-names">C.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Gaia</span>,  <span class="cit-name-given-names">G.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Agnoletti</span>,  <span class="cit-name-given-names">L.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Ferrari</span>,  <span class="cit-name-given-names">R.</span></span> <span class="cit-article-title">Cellular thiols redox status: A switch for NF-κB activation during myocardial post-ischaemic reperfusion</span>. <abbr class="cit-jnl-abbrev">J. Mol. Cell. Cardiol</abbr>. <span class="cit-vol">34</span>, <span class="cit-fpage">997</span>–<span class="cit-lpage">1005</span> (<span class="cit-pub-date">2002</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2Bmolecular%2Band%2Bcellular%2Bcardiology%26rft.stitle%253DJ%2BMol%2BCell%2BCardiol%26rft.aulast%253DCargnoni%26rft.auinit1%253DA.%26rft.volume%253D34%26rft.issue%253D8%26rft.spage%253D997%26rft.epage%253D1005%26rft.atitle%253DCellular%2Bthiols%2Bredox%2Bstatus%253A%2Ba%2Bswitch%2Bfor%2BNF-kappaB%2Bactivation%2Bduring%2Bmyocardial%2Bpost-ischaemic%2Breperfusion.%26rft_id%253Dinfo%253Adoi%252F10.1006%252Fjmcc.2002.2046%26rft_id%253Dinfo%253Apmid%252F12234769%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1006/jmcc.2002.2046&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=12234769&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F03%2F14%2F574798.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000177761000010&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">57.</span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-574798v2.57" data-doi="10.1002/1521-4141(2002010)32:10&lt;2866::AID-IMMU2866&gt;3.0.CO;2-V"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Murata</span>,  <span class="cit-name-given-names">Y.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Ohteki</span>,  <span class="cit-name-given-names">T.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Koyasu</span>,  <span class="cit-name-given-names">S.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Hamuro</span>,  <span class="cit-name-given-names">J.</span></span> <span class="cit-article-title">IFN-gamma and pro-inflammatory cytokine production by antigen-presenting cells is dictated by intracellular thiol redox status regulated by oxygen tension</span>. <abbr class="cit-jnl-abbrev">Eur. J. Immunol</abbr>. <span class="cit-vol">32</span>, <span class="cit-fpage">2866</span>–<span class="cit-lpage">2873</span> (<span class="cit-pub-date">2002</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEuropean%2Bjournal%2Bof%2Bimmunology%26rft.stitle%253DEur%2BJ%2BImmunol%26rft.aulast%253DMurata%26rft.auinit1%253DY.%26rft.volume%253D32%26rft.issue%253D10%26rft.spage%253D2866%26rft.epage%253D2873%26rft.atitle%253DIFN-gamma%2Band%2Bpro-inflammatory%2Bcytokine%2Bproduction%2Bby%2Bantigen-presenting%2Bcells%2Bis%2Bdictated%2Bby%2Bintracellular%2Bthiol%2Bredox%2Bstatus%2Bregulated%2Bby%2Boxygen%2Btension.%26rft_id%253Dinfo%253Adoi%252F10.1002%252F1521-4141%25282002010%252932%253A10%253C2866%253A%253AAID-IMMU2866%253E3.0.CO%253B2-V%26rft_id%253Dinfo%253Apmid%252F12355439%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1002/1521-4141(2002010)32:10&lt;2866::AID-IMMU2866&gt;3.0.CO;2-V&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=12355439&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F03%2F14%2F574798.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000178746600018&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">58.</span><a class="rev-xref-ref" href="#xref-ref-58-1" title="View reference 58. in text" id="ref-58">↵</a><div class="cit ref-cit ref-journal" id="cit-574798v2.58" data-doi="10.1002/(SICI)1097-4652(199608)168:2&lt;424::AID-JCP21&gt;3.0.CO;2-1"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Rajpurohit</span>,  <span class="cit-name-given-names">R.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Koch</span>,  <span class="cit-name-given-names">C. J.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Tao</span>,  <span class="cit-name-given-names">Z.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Teixeira</span>,  <span class="cit-name-given-names">C. M.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Shapiro</span>,  <span class="cit-name-given-names">I. M.</span></span> <span class="cit-article-title">Adaptation of chondrocytes to low oxygen tension: Relationship between hypoxia and cellular metabolism</span>. <abbr class="cit-jnl-abbrev">J. Cell. Physiol</abbr>. <span class="cit-vol">168</span>, <span class="cit-fpage">424</span>–<span class="cit-lpage">432</span> (<span class="cit-pub-date">1996</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2Bcellular%2Bphysiology%26rft.stitle%253DJ%2BCell%2BPhysiol%26rft.aulast%253DRajpurohit%26rft.auinit1%253DR.%26rft.volume%253D168%26rft.issue%253D2%26rft.spage%253D424%26rft.epage%253D432%26rft.atitle%253DAdaptation%2Bof%2Bchondrocytes%2Bto%2Blow%2Boxygen%2Btension%253A%2Brelationship%2Bbetween%2Bhypoxia%2Band%2Bcellular%2Bmetabolism.%26rft_id%253Dinfo%253Adoi%252F10.1002%252F%2528SICI%25291097-4652%2528199608%2529168%253A2%253C424%253A%253AAID-JCP21%253E3.0.CO%253B2-1%26rft_id%253Dinfo%253Apmid%252F8707878%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1002/(SICI)1097-4652(199608)168:2&lt;424::AID-JCP21&gt;3.0.CO;2-1&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=8707878&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F03%2F14%2F574798.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1996VA30800021&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">59.</span><a class="rev-xref-ref" href="#xref-ref-59-1" title="View reference 59. in text" id="ref-59">↵</a><div class="cit ref-cit ref-journal" id="cit-574798v2.59" data-doi="10.1371/journal.pone.0009644"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Pescador</span>,  <span class="cit-name-given-names">N.</span></span> <span class="cit-etal">et al.</span> <span class="cit-article-title">Hypoxia Promotes Glycogen Accumulation through Hypoxia Inducible Factor (HIF)-Mediated Induction of Glycogen Synthase 1</span>. <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">5</span>, <span class="cit-fpage">e9644</span> (<span class="cit-pub-date">2010</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BONE%26rft.aulast%253DPescador%26rft.auinit1%253DN.%26rft.volume%253D5%26rft.issue%253D3%26rft.spage%253De9644%26rft.epage%253De9644%26rft.atitle%253DHypoxia%2Bpromotes%2Bglycogen%2Baccumulation%2Bthrough%2Bhypoxia%2Binducible%2Bfactor%2B%2528HIF%2529-mediated%2Binduction%2Bof%2Bglycogen%2Bsynthase%2B1.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0009644%26rft_id%253Dinfo%253Apmid%252F20300197%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0009644&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20300197&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F03%2F14%2F574798.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">60.</span><a class="rev-xref-ref" href="#xref-ref-60-1" title="View reference 60. in text" id="ref-60">↵</a><div class="cit ref-cit ref-journal" id="cit-574798v2.60" data-doi="10.1016/j.febslet.2010.09.040"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Shen</span>,  <span class="cit-name-given-names">G. M.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Zhang</span>,  <span class="cit-name-given-names">F. L.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Liu</span>,  <span class="cit-name-given-names">X. L.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Zhang</span>,  <span class="cit-name-given-names">J. W.</span></span> <span class="cit-article-title">Hypoxia-inducible factor 1-mediated regulation of PPP1R3C promotes glycogen accumulation in human MCF-7 cells under hypoxia</span>. <abbr class="cit-jnl-abbrev">FEBS Lett</abbr>. <span class="cit-vol">584</span>, <span class="cit-fpage">4366</span>–<span class="cit-lpage">4372</span> (<span class="cit-pub-date">2010</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DFEBS%2BLetters%26rft.stitle%253DFEBS%2BLetters%26rft.aulast%253DShen%26rft.auinit1%253DG.%2BM.%26rft.volume%253D584%26rft.issue%253D20%26rft.spage%253D4366%26rft.epage%253D4372%26rft.atitle%253DHypoxia-inducible%2Bfactor%2B1-mediated%2Bregulation%2Bof%2BPPP1R3C%2Bpromotes%2Bglycogen%2Baccumulation%2Bin%2Bhuman%2BMCF-7%2Bcells%2Bunder%2Bhypoxia.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.febslet.2010.09.040%26rft_id%253Dinfo%253Apmid%252F20888814%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.febslet.2010.09.040&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20888814&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F03%2F14%2F574798.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">61.</span><a class="rev-xref-ref" href="#xref-ref-61-1" title="View reference 61. in text" id="ref-61">↵</a><div class="cit ref-cit ref-journal" id="cit-574798v2.61"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Phlippen</span>,  <span class="cit-name-given-names">M. K.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Webster</span>,  <span class="cit-name-given-names">S. G.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Chung</span>,  <span class="cit-name-given-names">J. S.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Dircksen</span>,  <span class="cit-name-given-names">H.</span></span> <span class="cit-article-title">Ecdysis of decapod crustaceans is associated with a dramatic release of crustacean cardioactive peptide into the haemolymph</span>. <abbr class="cit-jnl-abbrev">J. Exp. Biol</abbr>. <span class="cit-vol">203</span>, <span class="cit-fpage">521</span>–<span class="cit-lpage">36</span> (<span class="cit-pub-date">2000</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BExperimental%2BBiology%26rft.stitle%253DJ.%2BExp.%2BBiol.%26rft.aulast%253DPhlippen%26rft.auinit1%253DM.%26rft.volume%253D203%26rft.issue%253D3%26rft.spage%253D521%26rft.epage%253D536%26rft.atitle%253DEcdysis%2Bof%2Bdecapod%2Bcrustaceans%2Bis%2Bassociated%2Bwith%2Ba%2Bdramatic%2Brelease%2Bof%2Bcrustacean%2Bcardioactive%2Bpeptide%2Binto%2Bthe%2Bhaemolymph%26rft_id%253Dinfo%253Apmid%252F10637181%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiamV4YmlvIjtzOjU6InJlc2lkIjtzOjk6IjIwMy8zLzUyMSI7czo0OiJhdG9tIjtzOjM3OiIvYmlvcnhpdi9lYXJseS8yMDE5LzAzLzE0LzU3NDc5OC5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span></span></a></div></div></li><li><span class="ref-label">62.</span><a class="rev-xref-ref" href="#xref-ref-62-1" title="View reference 62. in text" id="ref-62">↵</a><div class="cit ref-cit ref-journal" id="cit-574798v2.62" data-doi="10.1186/1471-2164-12-147"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Kuballa</span>,  <span class="cit-name-given-names">A. V</span></span>, <span class="cit-auth"><span class="cit-name-surname">Holton</span>,  <span class="cit-name-given-names">T. A.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Paterson</span>,  <span class="cit-name-given-names">B.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Elizur</span>,  <span class="cit-name-given-names">A.</span></span> <span class="cit-article-title">Moult cycle specific differential gene expression profiling of the crab Portunus pelagicus</span>. <abbr class="cit-jnl-abbrev">BMC Genomics</abbr> <span class="cit-vol">12</span>, <span class="cit-fpage">147</span> (<span class="cit-pub-date">2011</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2Bgenomics%2B%255Belectronic%2Bresource%255D%26rft.stitle%253DBMC%2BGenomics%26rft.aulast%253DKuballa%26rft.auinit1%253DA.%2BV.%26rft.volume%253D12%26rft.spage%253D147%26rft.epage%253D147%26rft.atitle%253DMoult%2Bcycle%2Bspecific%2Bdifferential%2Bgene%2Bexpression%2Bprofiling%2Bof%2Bthe%2Bcrab%2BPortunus%2Bpelagicus.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2164-12-147%26rft_id%253Dinfo%253Apmid%252F21396120%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2164-12-147&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21396120&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F03%2F14%2F574798.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">63.</span><a class="rev-xref-ref" href="#xref-ref-63-1" title="View reference 63. in text" id="ref-63">↵</a><div class="cit ref-cit ref-book" id="cit-574798v2.63" data-doi="10.1007/978-1-59745-244-1_1"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Fiehn</span>,  <span class="cit-name-given-names">O.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Kind</span>,  <span class="cit-name-given-names">T.</span></span> in <span class="cit-fpage">3</span>–<span class="cit-lpage">17</span> (<span class="cit-publ-name">Humana Press</span>, <span class="cit-pub-date">2007</span>).<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1007/978-1-59745-244-1_1</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMethods%2Bin%2Bmolecular%2Bbiology%2B%2528Clifton%252C%2BN.J.%2529%26rft.stitle%253DMethods%2BMol%2BBiol%26rft.aulast%253DFiehn%26rft.auinit1%253DO.%26rft.volume%253D358%26rft.spage%253D3%26rft.epage%253D17%26rft.atitle%253DMetabolite%2Bprofiling%2Bin%2Bblood%2Bplasma.%26rft_id%253Dinfo%253Adoi%252F10.1007%252F978-1-59745-244-1_1%26rft_id%253Dinfo%253Apmid%252F17035677%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/978-1-59745-244-1_1&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17035677&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F03%2F14%2F574798.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">64.</span><a class="rev-xref-ref" href="#xref-ref-64-1" title="View reference 64. in text" id="ref-64">↵</a><div class="cit ref-cit ref-journal" id="cit-574798v2.64" data-doi="10.1194/jlr.D700041-JLR200"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Matyash</span>,  <span class="cit-name-given-names">V.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Liebisch</span>,  <span class="cit-name-given-names">G.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Kurzchalia</span>,  <span class="cit-name-given-names">T. V.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Shevchenko</span>,  <span class="cit-name-given-names">A.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Schwudke</span>,  <span class="cit-name-given-names">D.</span></span> <span class="cit-article-title">Lipid extraction by methyl- <em>tert</em> -butyl ether for high-throughput lipidomics</span>. <abbr class="cit-jnl-abbrev">J. Lipid Res</abbr>. <span class="cit-vol">49</span>, <span class="cit-fpage">1137</span>–<span class="cit-lpage">1146</span> (<span class="cit-pub-date">2008</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BLipid%2BRes%26rft_id%253Dinfo%253Adoi%252F10.1194%252Fjlr.D700041-JLR200%26rft_id%253Dinfo%253Apmid%252F18281723%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiamxyIjtzOjU6InJlc2lkIjtzOjk6IjQ5LzUvMTEzNyI7czo0OiJhdG9tIjtzOjM3OiIvYmlvcnhpdi9lYXJseS8yMDE5LzAzLzE0LzU3NDc5OC5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">65.</span><a class="rev-xref-ref" href="#xref-ref-65-1" title="View reference 65. in text" id="ref-65">↵</a><div class="cit ref-cit ref-book" id="cit-574798v2.65" data-doi="10.1007/11530084_18"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Fiehn</span>,  <span class="cit-name-given-names">O.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Wohlgemuth</span>,  <span class="cit-name-given-names">G.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Scholz</span>,  <span class="cit-name-given-names">M.</span></span> in <span class="cit-fpage">224</span>–<span class="cit-lpage">239</span> (<span class="cit-publ-name">Springer</span>, <span class="cit-publ-loc">Berlin, Heidelberg</span>, <span class="cit-pub-date">2005</span>).<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1007/11530084_18</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DLECT%2BNOT%2BCOMP%2BSCI%26rft.volume%253D3615%26rft.spage%253D224%26rft_id%253Dinfo%253Adoi%252F10.1007%252F11530084_18%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/11530084_18&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label">66.</span><a class="rev-xref-ref" href="#xref-ref-66-1" title="View reference 66. in text" id="ref-66">↵</a><div class="cit ref-cit ref-journal" id="cit-574798v2.66" data-doi="10.1111/j.1365-313X.2007.03387.x"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Fiehn</span>,  <span class="cit-name-given-names">O.</span></span> <span class="cit-etal">et al.</span> <span class="cit-article-title">Quality control for plant metabolomics: reporting MSI-compliant studies</span>. <abbr class="cit-jnl-abbrev">Plant J</abbr>. <span class="cit-vol">53</span>, <span class="cit-fpage">691</span>–<span class="cit-lpage">704</span> (<span class="cit-pub-date">2008</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DThe%2BPlant%2Bjournal%2B%253A%2B%2Bfor%2Bcell%2Band%2Bmolecular%2Bbiology%26rft.stitle%253DPlant%2BJ%26rft.aulast%253DFiehn%26rft.auinit1%253DO.%26rft.volume%253D53%26rft.issue%253D4%26rft.spage%253D691%26rft.epage%253D704%26rft.atitle%253DQuality%2Bcontrol%2Bfor%2Bplant%2Bmetabolomics%253A%2Breporting%2BMSI-compliant%2Bstudies.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1365-313X.2007.03387.x%26rft_id%253Dinfo%253Apmid%252F18269577%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1365-313X.2007.03387.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18269577&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F03%2F14%2F574798.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000252931800007&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">67.</span><a class="rev-xref-ref" href="#xref-ref-67-1" title="View reference 67. in text" id="ref-67">↵</a><div class="cit ref-cit ref-journal" id="cit-574798v2.67" data-doi="10.5244/C.22.54"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Breiman</span>,  <span class="cit-name-given-names">L.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Cutler</span>,  <span class="cit-name-given-names">A.</span></span> <span class="cit-article-title">Breiman and Cutler’s random forests for classification and regression</span>. <abbr class="cit-jnl-abbrev">Packag. ‘randomForest’</abbr> <span class="cit-vol">29</span> (<span class="cit-pub-date">2012</span>).<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.5244/C.22.54</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPackag.%2B%253FrandomForest%253F%26rft_id%253Dinfo%253Adoi%252F10.5244%252FC.22.54%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.5244/C.22.54&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label">68.</span><a class="rev-xref-ref" href="#xref-ref-68-1" title="View reference 68. in text" id="ref-68">↵</a><div class="cit ref-cit ref-journal" id="cit-574798v2.68" data-doi="10.1093/bioinformatics/btq476"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Wohlgemuth</span>,  <span class="cit-name-given-names">G.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Haldiya</span>,  <span class="cit-name-given-names">P. K.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Willighagen</span>,  <span class="cit-name-given-names">E.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Kind</span>,  <span class="cit-name-given-names">T.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Fiehn</span>,  <span class="cit-name-given-names">O.</span></span> <span class="cit-article-title">The chemical translation service-a web-based tool to improve standardization of metabolomic reports</span>. <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">26</span>, <span class="cit-fpage">2647</span>–<span class="cit-lpage">2648</span> (<span class="cit-pub-date">2010</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtq476%26rft_id%253Dinfo%253Apmid%252F20829444%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/btq476&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20829444&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F03%2F14%2F574798.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000282749700031&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">69.</span><a class="rev-xref-ref" href="#xref-ref-69-1" title="View reference 69. in text" id="ref-69">↵</a><div class="cit ref-cit ref-journal" id="cit-574798v2.69" data-doi="10.1093/nar/gkv951"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Kim</span>,  <span class="cit-name-given-names">S.</span></span> <span class="cit-etal">et al.</span> <span class="cit-article-title">PubChem substance and compound databases</span>. <abbr class="cit-jnl-abbrev">Nucleic Acids Res</abbr>. <span class="cit-vol">44</span>, <span class="cit-fpage">D1202</span>–<span class="cit-lpage">D1213</span> (<span class="cit-pub-date">2016</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgkv951%26rft_id%253Dinfo%253Apmid%252F26400175%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/nar/gkv951&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=26400175&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F03%2F14%2F574798.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">70.</span><a class="rev-xref-ref" href="#xref-ref-70-1" title="View reference 70. in text" id="ref-70">↵</a><div class="cit ref-cit ref-journal" id="cit-574798v2.70"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Barupal</span>,  <span class="cit-name-given-names">D. K.</span></span> <span class="cit-etal">et al.</span> <span class="cit-article-title">MetaMapp: mapping and visualizing metabolomic data by integrating information from biochemical pathways and chemical and mass spectral similarity</span>. <abbr class="cit-jnl-abbrev">BMC Bioinformatics</abbr> <span class="cit-vol">13</span>, (<span class="cit-pub-date">2012</span>).</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">71.</span><a class="rev-xref-ref" href="#xref-ref-71-1" title="View reference 71. in text" id="ref-71">↵</a><div class="cit ref-cit ref-journal" id="cit-574798v2.71" data-doi="10.1101/gr.1239303"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Shannon</span>,  <span class="cit-name-given-names">P.</span></span> <span class="cit-etal">et al.</span> <span class="cit-article-title">Cytoscape: a software environment for integrated models of biomolecular interaction networks</span>. <abbr class="cit-jnl-abbrev">Genome Res</abbr>. <span class="cit-vol">13</span>, <span class="cit-fpage">2498</span>–<span class="cit-lpage">504</span> (<span class="cit-pub-date">2003</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BResearch%26rft.stitle%253DGenome%2BRes%26rft.aulast%253DShannon%26rft.auinit1%253DP.%26rft.volume%253D13%26rft.issue%253D11%26rft.spage%253D2498%26rft.epage%253D2504%26rft.atitle%253DCytoscape%253A%2BA%2BSoftware%2BEnvironment%2Bfor%2BIntegrated%2BModels%2Bof%2BBiomolecular%2BInteraction%2BNetworks%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.1239303%26rft_id%253Dinfo%253Apmid%252F14597658%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjEwOiIxMy8xMS8yNDk4IjtzOjQ6ImF0b20iO3M6Mzc6Ii9iaW9yeGl2L2Vhcmx5LzIwMTkvMDMvMTQvNTc0Nzk4LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">72.</span><a class="rev-xref-ref" href="#xref-ref-72-1" title="View reference 72. in text" id="ref-72">↵</a><div class="cit ref-cit ref-journal" id="cit-574798v2.72"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Torchiano</span>,  <span class="cit-name-given-names">M.</span></span> <span class="cit-article-title">Efficient Effect Size Computation [R package effsize version 0.7.1]</span>.</cite></div><div class="cit-extra"></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.biorxiv.org/content/10.1101/574798v2.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-disqus-comment" >
  
      
  
  <div class="pane-content">
    <div id="disqus_thread"><noscript><p><a href="http://biorxivstage.disqus.com/?url=https%3A%2F%2Fwww.biorxiv.org%2Fcontent%2F10.1101%2F574798v2" class="" data-icon-position="" data-hide-link-title="0">View the discussion thread.</a></p></noscript></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/10.1101/578245v1" title="Biological age is a universal marker of aging, stress, and frailty" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous</span></a></span><span class="pager-next"><a href="/content/10.1101/283929v2-0" title="Cell-type heterogeneity in adipose tissue is associated with complex traits and reveals disease-relevant cell-specific eQTLs" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    Posted&nbsp;March 14, 2019.  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-biorxiv-art-tools" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-biorxiv_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/10.1101/574798v2.full.pdf" target="_self" class="article-dl-pdf-link link-icon"><i class="icon-external-link-sign"></i> <span class="title">Download PDF</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-biorxiv-supplementary-fragment" >
  
      
  
  <div class="pane-content">
    <p><a href="/content/10.1101/574798v2.supplementary-material"><i class="icon-file"></i>Supplementary Material </a></p>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-biorxiv-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="Email this Article" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-biorxiv_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/10.1101/574798v2.full" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for your interest in spreading the word about bioRxiv.</p><p>NOTE: Your email address is requested solely to identify you as the sender of this article.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/10.1101/574798v2" class="active" data-icon-position="" data-hide-link-title="0">Uncovering mechanisms of global ocean change effects on the Dungeness crab (Cancer magister) through metabolomics analysis</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from bioRxiv
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would like to see this page from the bioRxiv website.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/249808" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-I77CFaRRKK7gDIDph5VSJ3q6KhcZORx-bzotlfJXBKc" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-biorxiv-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-biorxiv_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="249808" id="node249808--2" data-pisa="biorxiv;574798v2" data-pisa-master="biorxiv;574798" data-seqnum="249808" data-apath="/biorxiv/early/2019/03/14/574798.atom"><div  class="highwire-cite highwire-cite-highwire-article highwire-citation-biorxiv-article-pap-list clearfix" >

      <div class="highwire-cite-title" >
      <div class="highwire-cite-title">Uncovering mechanisms of global ocean change effects on the Dungeness crab (<em>Cancer magister</em>) through metabolomics analysis</div>    </div>
  
      <div  class="highwire-cite-authors" ><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0"><span class="nlm-given-names">Shelly A.</span> <span class="nlm-surname">Trigg</span></span>, <span class="highwire-citation-author" data-delta="1"><span class="nlm-given-names">Paul</span> <span class="nlm-surname">McElhany</span></span>, <span class="highwire-citation-author" data-delta="2"><span class="nlm-given-names">Michael</span> <span class="nlm-surname">Maher</span></span>, <span class="highwire-citation-author" data-delta="3"><span class="nlm-given-names">Danielle</span> <span class="nlm-surname">Perez</span></span>, <span class="highwire-citation-author" data-delta="4"><span class="nlm-given-names">D. Shallin</span> <span class="nlm-surname">Busch</span></span>, <span class="highwire-citation-author" data-delta="5"><span class="nlm-given-names">Krista M.</span> <span class="nlm-surname">Nichols</span></span></span></div>
  
      <div  class="highwire-cite-metadata" ><span  class="highwire-cite-metadata-journal highwire-cite-metadata">bioRxiv </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">574798; </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata"><span class="doi_label">doi:</span> https://doi.org/10.1101/574798 </span></div>
  
  
  </div>
</div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.biorxiv.org/content/10.1101/574798v2" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.biorxiv.org/content/10.1101/574798v2&amp;title=Uncovering%20mechanisms%20of%20global%20ocean%20change%20effects%20on%20the%20Dungeness%20crab%20%28Cancer%20magister%29%20through%20metabolomics%20analysis" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.biorxiv.org/content/10.1101/574798v2&amp;title=Uncovering%20mechanisms%20of%20global%20ocean%20change%20effects%20on%20the%20Dungeness%20crab%20%28Cancer%20magister%29%20through%20metabolomics%20analysis" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.biorxiv.org/content/10.1101/574798v2&amp;text=Uncovering%20mechanisms%20of%20global%20ocean%20change%20effects%20on%20the%20Dungeness%20crab%20%28Cancer%20magister%29%20through%20metabolomics%20analysis" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.biorxiv.org/content/10.1101/574798v2&amp;title=Uncovering%20mechanisms%20of%20global%20ocean%20change%20effects%20on%20the%20Dungeness%20crab%20%28Cancer%20magister%29%20through%20metabolomics%20analysis" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.biorxiv.org/content/10.1101/574798v2&amp;t=Uncovering%20mechanisms%20of%20global%20ocean%20change%20effects%20on%20the%20Dungeness%20crab%20%28Cancer%20magister%29%20through%20metabolomics%20analysis" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.biorxiv.org/content/10.1101/574798v2&amp;title=Uncovering%20mechanisms%20of%20global%20ocean%20change%20effects%20on%20the%20Dungeness%20crab%20%28Cancer%20magister%29%20through%20metabolomics%20analysis" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.biorxiv.org/content/10.1101/574798v2&amp;title=Uncovering%20mechanisms%20of%20global%20ocean%20change%20effects%20on%20the%20Dungeness%20crab%20%28Cancer%20magister%29%20through%20metabolomics%20analysis" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-biorxiv-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title="Citation Tools"><i class="icon-globe"></i> <span class="title">Citation Tools</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-biorxiv_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="249808" id="node249808--2" data-pisa="biorxiv;574798v2" data-pisa-master="biorxiv;574798" data-seqnum="249808" data-apath="/biorxiv/early/2019/03/14/574798.atom"><div  class="highwire-cite highwire-cite-highwire-article highwire-citation-biorxiv-article-pap-list clearfix" >

      <div class="highwire-cite-title" >
      <div class="highwire-cite-title">Uncovering mechanisms of global ocean change effects on the Dungeness crab (<em>Cancer magister</em>) through metabolomics analysis</div>    </div>
  
      <div  class="highwire-cite-authors" ><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0"><span class="nlm-given-names">Shelly A.</span> <span class="nlm-surname">Trigg</span></span>, <span class="highwire-citation-author" data-delta="1"><span class="nlm-given-names">Paul</span> <span class="nlm-surname">McElhany</span></span>, <span class="highwire-citation-author" data-delta="2"><span class="nlm-given-names">Michael</span> <span class="nlm-surname">Maher</span></span>, <span class="highwire-citation-author" data-delta="3"><span class="nlm-given-names">Danielle</span> <span class="nlm-surname">Perez</span></span>, <span class="highwire-citation-author" data-delta="4"><span class="nlm-given-names">D. Shallin</span> <span class="nlm-surname">Busch</span></span>, <span class="highwire-citation-author" data-delta="5"><span class="nlm-given-names">Krista M.</span> <span class="nlm-surname">Nichols</span></span></span></div>
  
      <div  class="highwire-cite-metadata" ><span  class="highwire-cite-metadata-journal highwire-cite-metadata">bioRxiv </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">574798; </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata"><span class="doi_label">doi:</span> https://doi.org/10.1101/574798 </span></div>
  
  
  </div>
</div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage&amp;rft.epage&amp;rft.atitle=Uncovering%20mechanisms%20of%20global%20ocean%20change%20effects%20on%20the%20Dungeness%20crab%20%28Cancer%20magister%29%20through%20metabolomics%20analysis&amp;rft.volume&amp;rft.issue&amp;rft.date=2019-01-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=bioRxiv&amp;rft.au=Trigg%2C+Shelly+A.&amp;rft.au=McElhany%2C+Paul&amp;rft.au=Maher%2C+Michael&amp;rft.au=Perez%2C+Danielle&amp;rft.au=Busch%2C+D.+Shallin&amp;rft.au=Nichols%2C+Krista+M."></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/249808/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/249808/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/249808/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/249808/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/249808/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/249808/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/249808/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/249808/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/249808/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/249808/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/249808/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/249808/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><div class="item-list"><ul><li class="first"><a href="http://twitter.com/share?url=https%3A//www.biorxiv.org/content/10.1101/574798v2&amp;count=horizontal&amp;via=&amp;text=Uncovering%20mechanisms%20of%20global%20ocean%20change%20effects%20on%20the%20Dungeness%20crab%20%28Cancer%20magister%29%20through%20metabolomics%20analysis&amp;counturl=https%3A//www.biorxiv.org/content/10.1101/574798v2" class="twitter-share-button service-links-twitter-widget" id="twitter_widget" title="Tweet This" rel="nofollow" data-icon-position="" data-hide-link-title="0"><span class="element-invisible">Tweet Widget</span></a></li><li><a href="http://www.facebook.com/plugins/like.php?href=https%3A//www.biorxiv.org/content/10.1101/574798v2&amp;layout=button_count&amp;show_faces=false&amp;action=like&amp;colorscheme=light&amp;width=100&amp;height=21&amp;font=&amp;locale=" id="facebook_like" title="I Like it" class="service-links-facebook-like" rel="nofollow" data-icon-position="" data-hide-link-title="0"><span class="element-invisible">Facebook Like</span></a></li><li class="last"><a href="https://www.biorxiv.org/content/10.1101/574798v2" id="google_plus_one" title="Plus it" class="service-links-google-plus-one" rel="nofollow" data-icon-position="" data-hide-link-title="0"><span class="element-invisible">Google Plus One</span></a></li></ul></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-collections" >
  
        <h2 class="pane-title">Subject Area</h2>
    
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-collections"><div class="highwire-list"><ul class="highwire-article-collection-term-list"><li class="first last odd"><span class="highwire-article-collection-term"><a href="/collection/systems-biology" class="highlight" data-icon-position="" data-hide-link-title="0">Systems Biology<i class="icon-caret-right"></i>
</a></span></li></ul></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-biorxiv-subject-collections block-style-col2" >
  
      
  
  <div class="pane-content">
    <div class="panel-flexible panels-flexible-8 clearfix" id="mini-panel-biorxiv_subject_collections">
<div class="panel-flexible-inside panels-flexible-8-inside">
<div class="panels-flexible-region panels-flexible-region-8-center panels-flexible-region-first panels-flexible-region-last">
  <div class="inside panels-flexible-region-inside panels-flexible-region-8-center-inside panels-flexible-region-inside-first panels-flexible-region-inside-last">
<div class="panel-pane pane-snippet" >
  
      
  
  <div class="pane-content">
    <div class="snippet biorxiv-subject-areas-table-title" id="biorxiv-subject-areas-table-title">
  
      
  <div class="snippet-content">
    <b>Subject Areas</b>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-snippet" >
  
      
  
  <div class="pane-content">
    <div class="snippet biorxiv-subject-areas-view-papers" id="biorxiv-subject-areas-view-papers">
  
      
  <div class="snippet-content">
    <a href="/content/early/recent"><strong>All Articles</strong></a>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-subject-collections" >
  
      
  
  <div class="pane-content">
    <ul id="collection" class="collection highwire-list-expand"><li class="outer collection depth-2  child first"><div class = "data-wrapper"><a href="/collection/animal-behavior-and-cognition" class="" data-icon-position="" data-hide-link-title="0">Animal Behavior and Cognition</a> <span class = "article-count">(1662)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/biochemistry" class="" data-icon-position="" data-hide-link-title="0">Biochemistry</a> <span class = "article-count">(2789)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/bioengineering" class="" data-icon-position="" data-hide-link-title="0">Bioengineering</a> <span class = "article-count">(1946)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/bioinformatics" class="" data-icon-position="" data-hide-link-title="0">Bioinformatics</a> <span class = "article-count">(10400)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/biophysics" class="" data-icon-position="" data-hide-link-title="0">Biophysics</a> <span class = "article-count">(4243)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/cancer-biology" class="" data-icon-position="" data-hide-link-title="0">Cancer Biology</a> <span class = "article-count">(3272)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/cell-biology" class="" data-icon-position="" data-hide-link-title="0">Cell Biology</a> <span class = "article-count">(4641)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/clinical-trials" class="" data-icon-position="" data-hide-link-title="0">Clinical Trials</a> <span class = "article-count">(135)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/developmental-biology" class="" data-icon-position="" data-hide-link-title="0">Developmental Biology</a> <span class = "article-count">(2889)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/ecology" class="" data-icon-position="" data-hide-link-title="0">Ecology</a> <span class = "article-count">(4544)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/epidemiology" class="" data-icon-position="" data-hide-link-title="0">Epidemiology</a> <span class = "article-count">(2041)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/evolutionary-biology" class="" data-icon-position="" data-hide-link-title="0">Evolutionary Biology</a> <span class = "article-count">(7288)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/genetics" class="" data-icon-position="" data-hide-link-title="0">Genetics</a> <span class = "article-count">(5540)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/genomics" class="" data-icon-position="" data-hide-link-title="0">Genomics</a> <span class = "article-count">(6893)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/immunology" class="" data-icon-position="" data-hide-link-title="0">Immunology</a> <span class = "article-count">(2434)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/microbiology" class="" data-icon-position="" data-hide-link-title="0">Microbiology</a> <span class = "article-count">(7616)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/molecular-biology" class="" data-icon-position="" data-hide-link-title="0">Molecular Biology</a> <span class = "article-count">(3051)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/neuroscience" class="" data-icon-position="" data-hide-link-title="0">Neuroscience</a> <span class = "article-count">(18885)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/paleontology" class="" data-icon-position="" data-hide-link-title="0">Paleontology</a> <span class = "article-count">(142)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/pathology" class="" data-icon-position="" data-hide-link-title="0">Pathology</a> <span class = "article-count">(477)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/pharmacology-and-toxicology" class="" data-icon-position="" data-hide-link-title="0">Pharmacology and Toxicology</a> <span class = "article-count">(791)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/physiology" class="" data-icon-position="" data-hide-link-title="0">Physiology</a> <span class = "article-count">(1178)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/plant-biology" class="" data-icon-position="" data-hide-link-title="0">Plant Biology</a> <span class = "article-count">(2771)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/scientific-communication-and-education" class="" data-icon-position="" data-hide-link-title="0">Scientific Communication and Education</a> <span class = "article-count">(686)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/synthetic-biology" class="" data-icon-position="" data-hide-link-title="0">Synthetic Biology</a> <span class = "article-count">(905)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/systems-biology" class="" data-icon-position="" data-hide-link-title="0">Systems Biology</a> <span class = "article-count">(2881)</span></div></li>
<li class="outer collection depth-2  child last"><div class = "data-wrapper"><a href="/collection/zoology" class="" data-icon-position="" data-hide-link-title="0">Zoology</a> <span class = "article-count">(472)</span></div></li>
</ul>  </div>

  
  </div>
  </div>
</div>
</div>
</div>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__VNH5GD8Zz7g6_hGCOZjVjdMndxxC6naiExSSJKX3k_A__kzmEdtbqCsJx5Z9yg2Qy0PptB__ufXm-TxUOl0KZzUw__QJacVmveFyLIBSPkp21xL0ZvuDW9DMBrD2HB1P4Ry5I.js"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__QJacVmveFyLIBSPkp21xL0ZvuDW9DMBrD2HB1P4Ry5I.js" defer="defer"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__N7ERJBYsOWyRnJgyoM125_Aiez2MOJGaUofG1JdWWBg__bLlcwhzZhT-IwoQ_dvm1_xPxU7E7gecFLC6Fyeqtv8s__QJacVmveFyLIBSPkp21xL0ZvuDW9DMBrD2HB1P4Ry5I.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
function euCookieComplianceLoadScripts() {}
//--><!]]>
</script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var eu_cookie_compliance_cookie_name = "";
//--><!]]>
</script>
<script type="text/javascript" src="//d33xdlntwy0kbs.cloudfront.net/cshl_custom.js"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__BmqjBnkz3MgYCAoc25s1lDRMEjLhC3mEPVonUFIHi08__Unwv5-ZIuHBfFwytsjEx1niBVJ7n1T4lPws7VrkHXM4__QJacVmveFyLIBSPkp21xL0ZvuDW9DMBrD2HB1P4Ry5I.js"></script>
  </body>
</html>
