<html xmlns="http://www.w3.org/1999/xhtml"> <head> <style type="text/css">
      	     		 
      	    a                          {background : #ffffff; }
      	    article                    {border-style : dotted; border-width : 2px; }
		 	
		 	div                        {background : #ffffcc;}
		 	div.abstract-title         {font-weight : bold ; font-size : 16pt;}
		 	div.ack                    {border-style : solid ; border-color : red; margin : 2em; }
		 	div.article-meta           {border-style : solid; border-width : 1 pt; margin 1em;}
		 	div.boxed-text             {margin : 5em; border-style : solid;}
		 	div.contrib-group          {margin : 1em; }
		 	div.fig                    {border-style : solid; border-width : 2px; margin : 2em; }
		 	div.funding                {font-weight : bold ; font-size : 16pt;}
		 	div.given-names            {font-style : italic;}
		 	div.intro                  {border-style : inset; margin : 5px;}
		 	div.introduction           {border-style : inset; margin : 5px;}
		 	div.journal-meta           {border-style : solid; border-width : 1 pt; margin 1em;}
		 	div.journal-title-group    {background : #ffeeee;}
		 	div.article-title          {font-weight : bold ; font-size : 18pt;}
		 	div.kwd                    {font-style : italic}
		 	div.meta-name              {font-weight : bold ; font-size : 16pt;}
		 	div.name                   {font-weight : bold;}
		 	div.alt-title              {font-style : italic; font-size : 12px;}
		 	
		 	div.sec                    {border : 2px; margin 5px; padding 2px;}
		 	div.title                  {font-family : courier; font-weight : bold;
		 	                            font-size : 16pt; margin : 5px;}
		 	
		 	div.materials_methods      {border-style : double; margin : 5px; }
		 	div.methods                {border-style : double; margin : 5px; }
		 	
		 	div.results                {border-style : solid; margin : 5px;}
		 	div.background             {border-style : dotted; margin : 5px;}
		 	
		 	div.discussion             {border-style : groove; margin : 5px;}
		 	div.conclusion             {border-style : ridge; margin : 5px;}
		 	div.conclusions             {border-style : ridge; margin : 5px;}
		 	div.supplementary-material {border-style : inset; margin : 5px;}
		 	div.abbreviations          {border-style : double; border-color : red; margin : 5px;}
		 	div.competinginterests     {border-style : double; border-color : blue; margin : 5px;}
		 	div.acknowledgements       {border-style : double; border-color : green; margin : 5px;}
		 	div.authors_contributions   {border-style : double; border-color : purple; margin : 5px;}
		 	
		 	div.publisher              {border-style : outset; margin : 5px;}
		 	div.fn-type-conflict       {background : #f88; }
		 	div.fn-type-con            {background : #ddf; }
		 	div.fn-type-other          {background : #ddd; }
		 	
		 	div.unknown                {background : #ffd;
									 	  border-style : solid;
									 	  border-width : 1px;
									 	  padding : 2 px;}
		 	  
      	    table                      {background : #ffffdd;}
		 	tr                         {background : #ddddff; padding : 1px;}
		 	
		 	span                       {background : #ffcccc;}
		 	
		 	span.citation-author       {font-family : helvetica; background : #ffeeee;}
		 	span.collab                {background : #ddffff; }
		 	span.comment               {font-family : courier; font-size : 6px; background : #ffaaff;}
		 	span.contrib               {background : #ffffff;}
		 	span.corresp               {background : #ddffdd; }
		 	span.doi                   {background : #ffffff;}
		 	span.email                 {font-family : courier; }
		 	span.etal                  {font-style : italic;}
		 	span.fpage                 {font-family : courier;}
		 	span.given-names           {background : #ffffff;}
		 	span.iso-abbrev            {background : #ffffff;}
		 	span.issn-epub             {background : #ffffff;}
		 	span.issn-ppub             {background : #ffffff;}
		 	span.journal-title         {background : #ffffff;}
		 	span.lpage                 {font-family : courier;}
		 	span.mixed-article-title   {font-style : italic ;}
		 	span.nlm-ta                {background : #ffffff;}
		 	span.pmc                   {background : #ffffff;}
		 	span.pmcid                 {background : #ffffff;}
		 	span.pmid                  {background : #ffffff;}
		 	span.publisher             {background : #ffffff;}
		 	span.publisher-id          {background : #ffffff;}
		 	span.publisher-name        {background : #ffffff;}
		 	span.source                {background : #ffffff;}
		 	span.subject               {background : #ffffff;}
		 	span.surname               {background : #ffffff;}
		 	span.volume                {font-family : courier; font-weight : bold;}
		 	span.year                  {font-family : courier ; font-style : italic;}
			</style> </head> <body> <div class="front" title="front"> <div class="journal-meta" tagx="journal-meta" title="journal-meta"><span class="nlm-ta" title="nlm-ta">Philos Trans R Soc Lond B Biol Sci</span><span class="iso-abbrev" title="iso-abbrev">Philos. Trans. R. Soc. Lond., B, Biol. Sci</span><span class="publisher-id" title="publisher-id">RSTB</span><span class="hwp" title="hwp">royptb</span><div class="journal-title-group" tagx="journal-title-group" title="journal-title-group"><span class="journal-title" tagx="journal-title" title="journal-title">Philosophical Transactions of the Royal Society B: Biological Sciences</span></div><span class="issn-ppub" tagx="issn" title="issn-ppub">0962-8436</span><span class="issn-epub" tagx="issn" title="issn-epub">1471-2970</span><div class="publisher" tagx="publisher" title="publisher"><span class="publisher-name" tagx="publisher-name" title="publisher-name">The Royal Society</span></div> </div> <div class="article-meta" tagx="article-meta" title="article-meta"><span class="pmid" title="pmid"> pmid: <a href="http://www.ncbi.nlm.nih.gov/pubmed/27325833">27325833</a></span><span class="doi" title="doi"> doi: <a href="https://dx.doi.org/10.1098/rstb.2015.0136">10.1098/rstb.2015.0136</a></span><span class="publisher-id" title="publisher-id">rstb20150136</span><div class="article-categories" title="article-categories">
: <span class="subject" title="subject">1001</span><span class="subject" title="subject">70</span>: <span class="subject" title="subject">Articles</span>: <span class="subject" title="subject">Research Article</span></div> <div class="title-group" tagx="title-group" title="title-group"> <div class="article-title" title="article-title">
Closing the gap between rocks and clocks using total-evidence dating</div> <div class="alt-title" title="alt-title">
Closing the gap between rocks and clocks</div> </div> <div class="contrib-group" title="contrib-group"><span class="contrib" title="contrib"><span class="contrib-id" tagx="contrib-id" title="contrib-id">http://orcid.org/0000-0002-3929-251X</span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ronquist</span><span class="given-names" tagx="given-names" title="given-names">Fredrik</span></span><a href="#af1">1</a><a href="#cor1" /></span><span class="contrib" title="contrib"><span class="contrib-id" tagx="contrib-id" title="contrib-id">http://orcid.org/0000-0002-9973-7760</span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lartillot</span><span class="given-names" tagx="given-names" title="given-names">Nicolas</span></span><a href="#af2">2</a></span><span class="contrib" title="contrib"><span class="contrib-id" tagx="contrib-id" title="contrib-id">http://orcid.org/0000-0003-1532-449X</span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Phillips</span><span class="given-names" tagx="given-names" title="given-names">Matthew J.</span></span><a href="#af3">3</a></span></div><span class="citation_author_institution" id="af1">[1], <span class="addr-line" title="addr-line">Department of Bioinformatics and Genetics</span><span class="institution" title="institution">Swedish Museum of Natural History</span><span class="addr-line" title="addr-line">PO Box 50007, 104 05 Stockholm</span><span class="country" tagx="country" title="country">Sweden</span></span><span class="citation_author_institution" id="af2">[2], <span class="addr-line" title="addr-line">Laboratoire de Biométrie et Biologie Evolutive, UMR CNRS 5558</span><span class="institution" title="institution">Université Claude Bernard Lyon 1</span><span class="addr-line" title="addr-line">F-69622 Villeurbanne Cedex</span><span class="country" tagx="country" title="country">France</span></span><span class="citation_author_institution" id="af3">[3], <span class="addr-line" title="addr-line">School of Earth, Environmental and Biological Sciences</span><span class="institution" title="institution">Queensland University of Technology</span><span class="addr-line" title="addr-line">2 George Street, Brisbane, Queensland 4000</span><span class="country" tagx="country" title="country">Australia</span></span><div class="author-notes" title="author-notes"> <div class="corresp" title="corresp">
e-mail: <span class="email" tagx="email" title="email">fredrik.ronquist@nrm.se</span></div> <div class="fn-type-other" title="other"> <p>One contribution of 15 to a discussion meeting issue ‘<a href="http://dx.doi.org/10.1098/rstb/371/1699">Dating species divergences using rocks and clocks</a>’.</p> </div> </div><span class="pub-date-ppub" title="pub-date-ppub">ppub: <span>2016-7-7</span></span><span class="pub-date-pmc-release" title="pub-date-pmc-release">pmc-release: <span>2016-7-7</span></span><span class="volume" tagx="volume" title="volume">371</span><span class="issue" tagx="issue" title="issue">1699</span><span class="issue-title" tagx="issue-title" title="issue-title">Discussion meeting issue ‘Dating species divergences using rocks and clocks’ organized and edited by Ziheng Yang and Philip Donoghue</span><span class="elocation-id" tagx="elocation-id" title="elocation-id">20150136</span><span class="history" title="history"><span class="accepted" title="accepted">accepted: 2016-5-10</span></span><div class="permissions"><span class="copyright" title="copyright">(C) , <span class="copyright-year" tagx="copyright-year" title="copyright-year">2016</span></span><span class="license" title="license"><span class="license-p" title="license-p">Published by the Royal Society under the terms of the Creative Commons Attribution License <a href="http://creativecommons.org/licenses/by/4.0/">http://creativecommons.org/licenses/by/4.0/</a>, which permits unrestricted use, provided the original author and source are credited.</span></span></div><a class="self-uri" href="rstb20150136.pdf" title="self-uri">rstb20150136.pdf</a><div class="abstract" title="abstract"> <div class="abstract-title" title="abstract-title">
Abstract</div> <p>Total-evidence dating (TED) allows evolutionary biologists to incorporate a wide range of dating information into a unified statistical analysis. One might expect this to improve the agreement between rocks and clocks but this is not necessarily the case. We explore the reasons for such discordance using a mammalian dataset with rich molecular, morphological and fossil information. There is strong conflict in this dataset between morphology and molecules under standard stochastic models. This causes TED to push divergence events back in time when using inadequate models or vague priors, a phenomenon we term ‘deep root attraction’ (DRA). We identify several causes of DRA. Failure to account for diversified sampling results in dramatic DRA, but this can be addressed using existing techniques. Inadequate morphological models also appear to be a major contributor to DRA. The major reason seems to be that current models do not account for dependencies among morphological characters, causing distorted topology and branch length estimates. This is particularly problematic for huge morphological datasets, which may contain large numbers of correlated characters. Finally, diversification and fossil sampling priors that do not incorporate all the available background information can contribute to DRA, but these priors can also be used to compensate for DRA. Specifically, we show that DRA in the mammalian dataset can be addressed by introducing a modest extra penalty for ghost lineages that are unobserved in the fossil record, for instance by assuming rapid diversification, rare extinction or high fossil sampling rate; any of these assumptions produces highly congruent divergence time estimates with a minimal gap between rocks and clocks. Under these conditions, fossils have a stabilizing influence on divergence time estimates and significantly increase the precision of those estimates, which are generally close to the dates suggested by palaeontologists.</p> <p>This article is part of the themed issue ‘Dating species divergences using rocks and clocks’.</p> </div> <div class="kwd-group"> <div class="kwd" title="kwd">
total-evidence dating</div> <div class="kwd" title="kwd">
mammals</div> <div class="kwd" title="kwd">
fossilized birth–death</div> <div class="kwd" title="kwd">
deep root attraction</div> </div> <div class="funding-group" title="funding-group"> <div class="funding" title="funding">
Funding</div> <div class="award-group" title="award-group"> <div class="funding-source" title="funding-source"> <div class="institution-wrap" title="institution-wrap"><span class="institution" title="institution">Vetenskapsrådet</span><span class="institution-id" title="institution-id">http://dx.doi.org/10.13039/501100004359</span></div> </div><span class="award-id" tagx="award-id" title="award-id">2014-5901</span></div> <div class="award-group" title="award-group"> <div class="funding-source" title="funding-source"> <div class="institution-wrap" title="institution-wrap"><span class="institution" title="institution">Agence Nationale de la Recherche</span><span class="institution-id" title="institution-id">http://dx.doi.org/10.13039/501100001665</span></div> </div><span class="award-id" tagx="award-id" title="award-id">ANR-10-BINF-01-01 "Ancestrome"</span></div> </div> <div class="custom-meta-group" title="custom-meta-group"> <meta name="cover-date" value="July 19, 2016" /> </div> </div> </div> <div class="body" title="body"> <div class="introduction" title="sec"><span class="label" tagx="label" title="label">1.</span><div class="title" tagx="title" title="title">
Introduction</div> <p>Until recently, dating phylogenies with fossils has been based on the notion of associating particular calibration nodes in molecular-clock trees with age estimates derived from the fossil record. Nowadays, such ‘node dating’ is usually done within a Bayesian framework, using relaxed clock models and sophisticated prior probability distributions describing the information in the fossil record about the age of the calibration nodes. Node dating involves a stepwise analysis of divergence times: initially, calibration priors are derived from the fossil record and associated with particular tree nodes, and then these priors are combined with molecular data to produce posterior distributions on dated trees.</p> <p>Expressing the dating information in the fossil record as probability distributions on node ages is quite challenging [<a href="#RSTB20150136C1">1</a>]. Each calibration distribution should summarize the information from multiple fossils, while accommodating the uncertainty in the dating of each fossil, in its phylogenetic placement, and in the length of any side branches connecting it to the tree of extant taxa. Ideally, the calibration prior should also factor in the spatial and temporal sampling of fossils, and the implications of the fossil record with respect to likely diversification processes in the past.</p> <p>In recent years, evolutionary biologists have started exploring an alternative approach to dating phylogenies with fossils that we will refer to as ‘total-evidence dating’ (TED) [<a href="#RSTB20150136C2">2</a>–<a href="#RSTB20150136C4">4</a>]. TED is often called ‘tip dating’, even though TED or ‘integrative dating’ are more descriptive. TED involves simultaneous analysis of fossil and recent taxa, allowing evolutionary biologists to incorporate a wide range of sources of dating information into a unified statistical analysis.</p> <p>TED requires explicit coding of the character evidence that informs fossil affinities, which is typically hard work. However, the payoff is considerable. TED involves direct analysis of the available evidence; there is no need for secondarily derived calibration nodes, although it is of course possible to combine TED with node dating. TED integrates over the topological uncertainty in the placement of fossils, and the length of any side branches connecting them to the extant tree. This allows fossils with more certain placements and with traits that are closer to ancestral forms to exert more influence over the dating than fossils with uncertain affinities and many unique features. TED allows one to explicitly represent the uncertainty concerning the age of each individual fossil and, using fossilized birth–death (FBD) models, TED can also incorporate information about speciation, extinction and sampling processes [<a href="#RSTB20150136C5">5</a>].</p> <p>In one of the first TED analyses, we showed that divergence time estimates from TED were more robust to variation in prior assumptions and more precise (and presumably more accurate) than estimates from node dating [<a href="#RSTB20150136C4">4</a>]. We also showed that apparently erroneous or questionable assumptions about fossil placements can seriously bias divergence time estimates in node dating. This supports the expectation that an integrative approach, like TED, should be less sensitive to potential methodological problems than a complex stepwise procedure like node dating, in which it is more difficult to capture uncertainty in the initial steps (phylogenetic inference, placement of fossils in the tree, derivation of calibration distributions), so that it can be carried over properly to the final analysis. Thus, TED would appear to have the potential to close the gap between rocks and clocks often observed in node dating studies.</p> <p>Recent work has shown that this is not always the case in practice. In fact, TED is often reported to result in divergence time estimates that are more uncertain and more at odds with the fossil record than those from node dating or a combination of TED and node dating [<a href="#RSTB20150136C6">6</a>,<a href="#RSTB20150136C7">7</a>]. The aim of this paper is to examine why this might be the case.</p> <p>For several reasons, we chose to focus on the radiation of mammals, more specifically on the radiation of the Eutheria (crown-group and stem-group placentals). There may be no other group with so much data available for a TED analysis, and so much controversy surrounding the divergence time estimates. Recently, an exceptional morphological (phenomic) dataset (the O'Leary <i>et al</i>. dataset) was published for fossil and recent mammals [<a href="#RSTB20150136C8">8</a>]. With 4541 characters in total, scored for 46 extant and 40 fossil taxa, this may be the largest phenomic dataset published to date: it includes 1284 cranial characters, 1451 dental characters, 925 postcranial characters and 881 soft characters (including behavioural and other characters). There are also rich genetic and genomic data available for extant placentals [<a href="#RSTB20150136C9">9</a>,<a href="#RSTB20150136C10">10</a>].</p> <p>The debate concerning the age of crown placentals has a long history and is not settled yet. Some palaeontologists [<a href="#RSTB20150136C8">8</a>,<a href="#RSTB20150136C11">11</a>] argue for an origin adjacent to the Cretaceous–Palaeogene mass extinction (K–Pg event) around 66 Ma, followed by an explosive radiation, producing extant mammal orders during a time interval of a few hundred thousand years. This explosive radiation scenario is supported if one minimizes the length of any postulated lineages that are unobserved in the fossil record (‘ghost lineages’) [<a href="#RSTB20150136C8">8</a>]. In contrast, node dating based on genetic or genomic data place the placental origin at around 101 [<a href="#RSTB20150136C9">9</a>] or 90 Ma [<a href="#RSTB20150136C10">10</a>]; these trees postulate that a substantial portion of the interordinal placental diversification took place before the K–Pg event. The abundance of placental fossils in the Palaeogene and the absence of them in the Cretaceous provide one of the strongest pieces of evidence supporting the explosive radiation scenario ([<a href="#RSTB20150136C8">8</a>] but see [<a href="#RSTB20150136C12">12</a>]), whereas the high rates of molecular evolution around the K–Pg boundary required by such a scenario is one of the most powerful arguments against it ([<a href="#RSTB20150136C13">13</a>] but see [<a href="#RSTB20150136C14">14</a>]).</p> <p>The O'Leary <i>et al</i>. dataset [<a href="#RSTB20150136C8">8</a>] has not been subjected yet to TED analysis. However, a TED analysis of the placental radiation based on a similar dataset was recently published [<a href="#RSTB20150136C15">15</a>]. The phenomic dataset in the latter study comprised 421 characters coded for 102 taxa, most of them fossils. The divergence time estimates obtained when these morphological data were analysed alone or in combination with molecular data under an unconstrained TED model were unrealistically old (around 164 Ma), suggesting that the placental radiation might be a suitable target for investigating why TED misbehaves in some cases.</p> <p>Using a slightly modified version of the O'Leary <i>et al</i>. dataset, we show here that there is a tendency for TED divergence time estimates, under certain conditions, to be pushed back to very old strata and to be very imprecise. We term this phenomenon ‘deep root attraction’ (DRA). By exploring various prior models, we characterize the causes of DRA. DRA occurs when there are model errors or conflicts in the data that are difficult to reconcile, and ghost lineages carry little cost. A major driver of DRA appears to be inadequacies of current morphological models, a problem that is difficult if not impossible to address. However, by increasing the cost of ghost lineages, we show that it is possible to address DRA and obtain a stable TED solution that is robust to variation in prior assumptions. This solution places the radiation of crown placentals at around 85 Ma, which is intermediate between the explosive radiation scenario and the age estimates suggested by most previous node dating analyses (for exceptions, see [<a href="#RSTB20150136C16">16</a>,<a href="#RSTB20150136C17">17</a>]).</p> </div> <div class="methods" title="methods"><span class="label" tagx="label" title="label">2.</span><div class="title" tagx="title" title="title">
Methods</div> <div class="data" title="sec"><span class="label" tagx="label" title="label">(a)</span><div class="title" tagx="title" title="title">
Data</div> <p>The dataset of O'Leary <i>et al</i>. [<a href="#RSTB20150136C8">8</a>] includes 46 recent taxa and 40 fossil taxa. We were worried that it would be difficult to model the radiation of mammals into stem forms of monotremes, marsupials and placentals because these early events involve few lineages and long time periods according to both molecular and palaeontological studies. Therefore, we removed all extant and fossil taxa outside of the Eutheria (placentals plus stem-group placentals). The fossil <i>Eomaia scansoria</i> was placed outside of Theria (placentals + marsupials) in the analysis of O'Leary <i>et al</i>. [<a href="#RSTB20150136C8">8</a>] but it has previously been considered a member of Eutheria or Placentalia in analyses that densely sample stem therians [<a href="#RSTB20150136C18">18</a>,<a href="#RSTB20150136C19">19</a>], and we therefore included it in our dataset. To improve the fit between the data and our model of diversified sampling, we removed a very recent split among extant taxa by deleting <i>Galeopterus variegatus</i> from the dataset. Similarly, we omitted the very young (16.4 Ma) fossil <i>Hapalops elongatus</i>. The diversified sampling model [<a href="#RSTB20150136C20">20</a>] assumes that all deep splits in the tree have been sampled. The actual age of the most recent split in the tree, beyond which no more lineages are sampled, is inferred from data. Spurious young speciation events can seriously distort the inference of BD parameters under this model because they erroneously extend the period during which the process is assumed to be completely sampled. The resulting dataset included 74 eutherian or potentially eutherian taxa, 33 fossils and 41 extant taxa. The character data comprised 4541 discrete morphological characters and 36 860 DNA sites from 27 nuclear genes, including 22 protein-coding genes and five untranslated regions (UTRs) [<a href="#RSTB20150136C8">8</a>].</p> </div> <div class="substitutionmodels" title="sec"><span class="label" tagx="label" title="label">(b)</span><div class="title" tagx="title" title="title">
Substitution models</div> <p>The morphological data were modelled using the Mk model [<a href="#RSTB20150136C21">21</a>], correcting for the coding bias resulting from only variable characters being present in the matrix. The matrix actually did include some constant characters, but they were removed prior to analysis. Rate variation across morphological characters was modelled using a discrete gamma model with four categories.</p> <p>The 22 protein-coding genes were divided into three partitions corresponding to different codon positions, with the five UTRs treated in a separate partition. Each of the four molecular data partitions was associated with a unique nucleotide substitution model. Stationary state frequencies were allowed to differ, and we sampled across all different ways of partitioning the exchangeability rates [<a href="#RSTB20150136C22">22</a>]. Rate variation across sites was modelled using a gamma model (four discrete rate categories) with a proportion of invariable sites. Each of the four molecular data partitions was associated with a unique set of exchangeability rates, stationary state frequencies, gamma shape and proportion of invariable sites. The base rate of each partition was drawn from a Dirichlet distribution, under the constraint that the average rate across characters was 1.0.</p> </div> <div class="non-clockandclockmodels" title="sec"><span class="label" tagx="label" title="label">(c)</span><div class="title" tagx="title" title="title">
Non-clock and clock models</div> <p>For non-clock analyses, we assumed equal prior probability for all topologies. In all strict and relaxed clock analyses, we used a diffuse offset exponential prior on the age of the root of the tree, with a mean of 164 Ma and an offset of 64 Ma based on the oldest generally accepted crown placentals [<a href="#RSTB20150136C8">8</a>]. When fossils were included in the analyses, the minimum age of the induced prior was constrained by the age of the oldest fossil, <i>Eomaia scansoria</i>, which is dated to 122 Ma. The clock rate was associated with a lognormal prior with a mean of −6.0 and a standard deviation of 0.5 on the natural logarithm scale. This prior has an expectation of 0.25% substitutions per site per million years.</p> <p>All relaxed clock analyses assumed an independent gamma rates (white noise) model [<a href="#RSTB20150136C23">23</a>] with the variance increase parameter having an exponential prior with rate 10 (expectation 0.1). Because of rooting problems (see Results), we constrained all relaxed clock analyses to have a monophyletic Laurasiatheria + Euarchontoglires in the tree of extant taxa, unless noted otherwise. The positions of fossil taxa were never constrained.</p> </div> <div class="datingpriors" title="sec"><span class="label" tagx="label" title="label">(d)</span><div class="title" tagx="title" title="title">
Dating priors</div> <p>For dating, we used a uniform tree prior [<a href="#RSTB20150136C4">4</a>], a BD prior and an FBD prior [<a href="#RSTB20150136C24">24</a>]. The uniform tree prior does not have any parameters. For the naive BD and FBD priors, we used an exponential prior with rate 10 (expectation 0.1) for the net diversification rate, <i>d</i> (<i>d</i> = <i>λ</i> – <i>μ</i>, where <i>λ</i> is the birth rate and <i>μ</i> is the death rate), and a uniform beta prior, that is, a Beta(1,1) prior, for the turnover, <i>r</i> (<i>r</i> = <i>μ</i>/<i>λ</i>). For the fossil sampling probability, <i>f</i> (<i>f</i> = <i>ψ</i>/(<i>μ</i> + <i>ψ</i>), where <i>ψ</i> is the fossilization rate times the probability of subsequent discovery of the fossil), we also used a Beta(1,1) prior.</p> <p>The ages of fossils were updated from O'Leary <i>et al.</i> [<a href="#RSTB20150136C8">8</a>] (see electronic supplementary material, appendix S1). We did not associate the age estimates with any uncertainty, assuming that this would be a negligible source of error in our analyses, and we take our results to confirm this assumption at least with respect to our main conclusions. However, the uncertainty of age estimates may clearly be important in many contexts [<a href="#RSTB20150136C6">6</a>].</p> </div> <div class="tipsamplingassumptions" title="sec"><span class="label" tagx="label" title="label">(e)</span><div class="title" tagx="title" title="title">
Tip sampling assumptions</div> <p>For both the BD and FBD priors, we explored three different assumptions concerning tip sampling, namely that the tips represent: (i) a complete sample, (ii) a random sample or (iii) a diversified sample of the entire tree [<a href="#RSTB20150136C5">5</a>,<a href="#RSTB20150136C20">20</a>]. Given that there are approximately 5000 described species of placentals, we considered the 41 extant taxa in our dataset to represent approximately 1% of the total diversity of the group in the randomized and diversified tip sampling scenarios.</p> </div> <div class="informativedatingpriors" title="sec"><span class="label" tagx="label" title="label">(f)</span><div class="title" tagx="title" title="title">
Informative dating priors</div> <p>To increase the penalty for ghost lineages, we explored four different variants of the basic BD and FBD models with vague priors. First, we increased the prior probability of a low extinction rate by using a Beta(1,100) prior instead of a flat prior for the turnover, <i>r</i>. Second, we increased the prior probability of a high fossilization rate by using a Beta(100,1) prior instead of a flat prior for the fossil sampling probability, <i>f</i>. Third, we combined these two assumptions in a low extinction rate/high fossil sampling rate prior. Finally, we explored the effects of assuming a rapid net diversification rate, <i>d</i>, by fixing it to 0.1 while using flat Beta(1,1) priors for <i>r</i> and <i>f</i>. All analyses were run both with fossils under the FBD model, and without fossils under the corresponding BD model.</p> </div> <div class="alternativemodels" title="sec"><span class="label" tagx="label" title="label">(g)</span><div class="title" tagx="title" title="title">
Alternative models</div> <p>We explored two alternative models that could potentially address DRA by providing more realistic assumptions about diversification processes or rate variation across the tree. We first allowed the diversification process to vary across time by using a skyline FBD model [<a href="#RSTB20150136C25">25</a>] with three intervals: older than 70, 70–55 and younger than 55 Ma. FBD parameters were estimated separately for each of these intervals, using the standard priors (Exp(10) for <i>d</i> and Beta(1,1) for <i>r</i> and <i>f</i>). Second, we decoupled the relaxed clocks for morphology and molecules, allowing morphological and molecular rates to have different trajectories across the tree.</p> </div> <div class="markovchainmontecarlosampling" title="sec"><span class="label" tagx="label" title="label">(h)</span><div class="title" tagx="title" title="title">
Markov chain Monte Carlo sampling</div> <p>All analyses were performed with pre-release versions of M <div class="sc_UNKNOWN">
r</div>B <div class="sc_UNKNOWN">
ayes</div> v. 3.2.6 (svn revision 1067 and following) [<a href="#RSTB20150136C26">26</a>], the source code of which is available from the program website (<span class="uri" title="uri"><a href="http://mrbayes.net">http://mrbayes.net</a></span>). The code is now part of the latest M <div class="sc_UNKNOWN">
r</div>B <div class="sc_UNKNOWN">
ayes</div> release. To improve topological convergence when fossils are included, we added a couple of topology moves that specifically target fossil subtrees but that are, otherwise, identical to existing topology moves.</p> <p>For each model, we ran four independent analyses, each using four Metropolis-coupled chains. We ran the chains for 10–30 M generations each and sampled them every 500 generations, discarding 25% of the samples as burn-in. When fossils were included in the analysis, tree samples were summarized both with fossils included and with fossils first pruned away from all tree samples. Data files and M <div class="sc_UNKNOWN">
r</div>B <div class="sc_UNKNOWN">
ayes</div> run files for all analyses are provided in the electronic supplementary material.</p> <p>All tree figures were drawn using F <div class="sc_UNKNOWN">
ig</div>T <div class="sc_UNKNOWN">
ree</div> [<a href="#RSTB20150136C27">27</a>]. Kernel densities of posterior distributions were estimated from the sampled values using the defaults of the ‘density’ function in R [<a href="#RSTB20150136C28">28</a>], with the default bandwidth adjusted by a factor of 2.0–5.0.</p> </div> </div> <div class="results" title="sec"><span class="label" tagx="label" title="label">3.</span><div class="title" tagx="title" title="title">
Results</div> <div class="conflictbetweenmorphologyandmolecules" title="sec"><span class="label" tagx="label" title="label">(a)</span><div class="title" tagx="title" title="title">
Conflict between morphology and molecules</div> <p>The non-clock analyses demonstrate strong conflict between the molecular and the morphological signal in the eutherian dataset (<a href="#RSTB20150136F1">figure 1</a>). Whereas the total-evidence (<a href="#RSTB20150136F1">figure 1</a><i>a</i>) and molecular (<a href="#RSTB20150136F1">figure 1</a><i>b</i>) trees both retrieve the four mammalian superorders that have been recognized in most recent analyses (Xenarthra, Afrotheria, Laurasiatheria and Euarchontoglires), the morphological tree (<a href="#RSTB20150136F1">figure 1</a><i>c</i>) presents quite different groupings, often consistent with results from classical morphological cladistic analyses of placental relationships [<a href="#RSTB20150136C29">29</a>]. The morphological evidence sprinkles the afrotherian taxa across the tree, and even if we disregard the Afrotheria, the Euarchontoglires and Laurasiatheria remain intermixed. Among the Laurasiatheria, the pangolin (<i>Manis</i>) occupies a basal position in the tree together with Xenarthra (anteaters, sloths and armadillos). Furthermore, primates and their relatives (Euarchonta) group with bats (Chiroptera) rather than with Glires (rodents and lagomorphs), splitting the Euarchontoglires in two. </p><div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/"><span class="label" tagx="label" title="label">Figure 1.</span></a></div>   <p>Phylogenetic signal in the dataset. (<i>a</i>) Non-clock analysis of extant taxa using combined data. Posterior probabilities (in %) indicated on branches if below 100%. (<i>b</i>) Ditto using molecular data only. (<i>c</i>) Ditto using morphological data only. Note that the morphological tree (<i>c</i>) has longer branches and is more poorly supported than the molecular tree (<i>b</i>), indicating weak and potentially misleading phylogenetic signal owing to morphological convergence. The morphological tree is also strongly in conflict with monophyly of the four placental superorders (indicated by coloured bars). Nevertheless, the morphological signal is strong enough to change a few details in the combined tree (<i>a</i>), namely the positions of Scandentia (<i>Ptilocercus</i> and <i>Tupaia</i>) and of the rock hyrax (<i>Procavia</i>; arrows). (Online version in colour.)</p>   </div> <p /> <p>Many of the higher groupings in the morphology tree are poorly supported, with posterior probabilities well below 80% (<a href="#RSTB20150136F1">figure 1</a><i>c</i>). The branches in the tree are also long (electronic supplementary material, figure S1<i>b</i>), suggesting that there is considerable plasticity in morphology, with a potential for misleading convergence. This contrasts with the molecular tree, which is well resolved and has a much smaller proportion of characters changing along each branch (<a href="#RSTB20150136F1">figure 1</a><i>b</i> and electronic supplementary material, figure S1<i>a</i>). The branch length estimates are quite precise both for the morphological and molecular data (electronic supplementary material, figure S1<i>c</i>,<i>d</i>). Adding fossils to the morphology tree did not change the inferred relationships among extant taxa in any major way (electronic supplementary material, figure S2<i>a</i>).</p> <p>The analysis based on combined data resulted in a tree that was largely the same as the molecular tree (<a href="#RSTB20150136F1">figure 1</a><i>a</i>). However, despite the morphological characters being fewer (4.5 versus 36.9 k) and more homoplastic than the molecular characters, they were nevertheless able to change a few details in the molecular tree. Specifically, the combined tree supported the monophyly of Euarchonta (primates with tree shrews and flying lemurs), whereas the molecular tree grouped tree shrews with rodents and lagomorphs (arrows, <a href="#RSTB20150136F1">figure 1</a><i>a</i>). The morphological data also changed the position of the rock hyrax (<i>Procavia</i>) slightly. Adding fossils to the combined-data analysis only resulted in one minor topological change in the extant tree, affecting the position of the aardvark (<i>Orycteropus</i>) within Afrotheria (electronic supplementary material, figure S2<i>b</i>).</p> </div> <div class="rootingclocktrees" title="sec"><span class="label" tagx="label" title="label">(b)</span><div class="title" tagx="title" title="title">
Rooting clock trees</div> <p>Non-clock trees based on molecular data and combined data can readily be rooted, so that the four mammalian superorders are monophyletic (<a href="#RSTB20150136F2">figure 2</a><i>a</i>). Unconstrained strict-clock and relaxed-clock analyses, however, produce rooted trees with various apparent topological artefacts owing to incorrect rooting (<a href="#RSTB20150136F2">figure 2</a><i>b</i>,<i>c</i>). Depending on the exact model used, the clock trees may be rooted between Euarchontoglires and Laurasiatheria, or within one of these two superorders (often within Rodentia or Eulipotyphla (hedgehogs, shrews and moles)). The artefacts occur both in strict-clock trees and in relaxed-clock trees. Adding a single rooting constraint in the relaxed-clock analyses, forcing Boreoeutheria (Euarchontoglires + Laurasiatheria) to be monophyletic, is sufficient to stabilize the rooting and retrieve a tree topology of extant taxa that is fully congruent with the non-clock tree, and consistent with monophyly of all four mammalian superorders (<a href="#RSTB20150136F2">figure 2</a><i>d</i>). Such a rooting constraint was used for extant taxa in all subsequent relaxed-clock analyses; fossil taxa were allowed to attach to any part of the extant tree. </p><div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/"><span class="label" tagx="label" title="label">Figure 2.</span></a></div>   <p>Rooting artefacts in clock trees. (<i>a</i>) Analysis of relationships among extant taxa under a non-clock model. (<i>b</i>) Ditto under a strict-clock model. (<i>c</i>) Ditto under a relaxed-clock model (independent gamma rates). (<i>d</i>) Ditto under a relaxed-clock model (independent gamma rates), enforcing the topological constraint that Boreoeutheria (Euarchontoglires + Laurasiatheria) are monophyletic. The non-clock tree (<i>a</i>) can be rooted by appropriate choice of outgroup, so that placental superorders (coloured bars) are monophyletic. Clock models produce rooted trees without the need for specifying an outgroup. However, both strict (<i>b</i>) and relaxed clocks (<i>c</i>) result in topological artefacts close to the root when no rooting constraints are enforced. This can be solved in the relaxed-clock analysis by adding a single topological constraint close to the root of the tree (<i>d</i>). (Online version in colour.)</p>   </div> <p /> </div> <div class="tipsamplingeffects" title="sec"><span class="label" tagx="label" title="label">(c)</span><div class="title" tagx="title" title="title">
Tip sampling effects</div> <p>The assumption concerning the tip sampling procedure has a major influence on the divergence time estimates obtained under BD models (<a href="#RSTB20150136F3">figure 3</a>). When fossils are not included in the analysis, the root date is not affected much for the studied dataset, but the estimates of the most recent divergence times differ substantially. The effect is most clearly seen in the estimated age of the split between the whale (<i>Caperea</i>) and the dolphin (<i>Tursiops</i>), the most recent of all speciation events (arrows in <a href="#RSTB20150136F3">figure 3</a>). If tips are assumed to represent a complete sample of lineages, then the split is estimated to (all posterior age distributions are summarized using the median, followed by the 95% region of highest posterior density) 13 (5, 26) Ma (<a href="#RSTB20150136F3">figure 3</a><i>a</i>). If we account for the fact that the tips only represent approximately 1% of all known species, but still assume that they represent a random sample, the same split is estimated to 16 (5, 27) Ma (<a href="#RSTB20150136F3">figure 3</a><i>b</i>). However, when we also account for the tips being chosen to maximize diversity, the estimate is pushed back to 35 (22, 60) Ma (<a href="#RSTB20150136F3">figure 3</a><i>c</i>), which is more in line with expectations based on the fossil record [<a href="#RSTB20150136C8">8</a>]. </p><div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/"><span class="label" tagx="label" title="label">Figure 3.</span></a></div>   <p>The effects of biases in tip sampling when dating under a birth–death model. (<i>a</i>) Relaxed-clock dating analysis of extant taxa assuming complete sampling of tips. (<i>b</i>) Ditto assuming random sampling of tips. (<i>c</i>) Ditto assuming diversified sampling of tips. In studies of higher taxa, it is standard practice to choose exemplars that span as much as possible of the phylogenetic diversity of the studied group. Such diversified sampling results in recent lineage splits being absent in the tree. If we analyse diversified data assuming that the tips represent a complete or a random sample, major dating errors may result. Specifically, there will be a tendency to spread splits evenly over time, so that deep divergences become too old and/or recent divergences too young. For instance, note that the most recent split (arrow)—between the whale (<i>Caperea marginata</i>) and the dolphin (<i>Tursiops truncatus</i>)—is dated to 35 Ma under diversified sampling (<i>c</i>), consistent with the fossil record [<a href="#RSTB20150136C8">8</a>], whereas it is dated to 12 Ma under complete sampling (<i>a</i>) and 16 Ma under random sampling (<i>b</i>).</p>   </div> <p /> </div> <div class="deeprootattraction" title="sec"><span class="label" tagx="label" title="label">(d)</span><div class="title" tagx="title" title="title">
Deep root attraction</div> <p>When fossils are added to the analysis under an FBD model with vague priors, the effect of changing the tip sampling assumption becomes dramatic (<a href="#RSTB20150136F4">figure 4</a>). Specifically, erroneous assumptions about the sampling procedure (<a href="#RSTB20150136F4">figure 4</a><i>a</i>,<i>b</i>) push divergence time estimates back in time to strata that would be considered completely unrealistic by most workers. Even under the diversified sampling assumption, divergence time estimates appear old (<a href="#RSTB20150136F4">figure 4</a><i>c</i>), especially for the deepest splits in the tree, and they are quite uncertain (<a href="#RSTB20150136TB1">table 1</a>). For instance, the crown Placentalia are dated in this analysis to 118 (95, 178) Ma, which is older than the estimates resulting from many molecular-clock analyses and much older than the ages suggested by palaeontologists [<a href="#RSTB20150136C8">8</a>], but the credible interval is quite wide. </p><div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/"><span class="label" tagx="label" title="label">Figure 4.</span></a></div>   <p>The effects of model inadequacies in total-evidence dating (TED). (<i>a</i>) TED under a birth–death model assuming complete sampling of tips (only extant tree shown). (<i>b</i>) Ditto assuming random sampling of tips. (<i>c</i>) Ditto assuming diversified sampling of tips. Under TED, there is a tendency for some model inadequacies to push divergence time estimates towards unreasonably old strata, a phenomenon we term ‘deep root attraction’. For the eutherian data, it appears that the major underlying cause of DRA is the difficulty of reconciling the conflict between molecular and morphological signals under standard models. The problem is exaggerated by inappropriate modelling of the tip sampling protocol (<i>a,b</i>) but it is presumably also influencing the results under a more realistic tip sampling model (<i>c</i>).</p>   </div> <div class="table-wrap_UNKNOWN" id="RSTB20150136TB1" orientation="portrait" position="float"><span class="label" tagx="label" title="label">Table 1.</span> <p>Prior and posterior distributions for the ages of key clades under the uninformative and rapid diversification TED model priors. The induced joint prior distributions were estimated under the full model, with fossils included, using Markov chain Monte Carlo sampling with log-likelihood of characters set to 0.0. Defined in this way, the prior distribution contains a fair amount of dating information from FBD model parameter priors and from the number of sampled fossils and their ages. Estimated distributions are summarized by the median followed by the 95% region of highest posterior density.</p>  <table frame="hsides" rules="groups"> <colgroup span="1"> <col align="left" span="1" /></colgroup> <colgroup><col align="left" span="1" /></colgroup> <colgroup><col align="left" span="1" /></colgroup> <colgroup><col align="left" span="1" /></colgroup> <colgroup><col align="left" span="1" /></colgroup>  <thead valign="bottom"> <tr> <th rowspan="2" align="left" colspan="1">clade</th> <th colspan="2" align="left" rowspan="1">uninformative <hr /> </th> <th colspan="2" align="left" rowspan="1">rapid diversification <hr /> </th> </tr> <tr> <th align="left" rowspan="1" colspan="1">induced prior</th> <th align="left" rowspan="1" colspan="1">posterior</th> <th align="left" rowspan="1" colspan="1">induced prior</th> <th align="left" rowspan="1" colspan="1">posterior</th> </tr> </thead> <tbody> <tr> <td rowspan="1" colspan="1">Placentalia</td> <td rowspan="1" colspan="1">111 (86, 141)</td> <td rowspan="1" colspan="1">118 (95, 178)</td> <td rowspan="1" colspan="1">85 (66, 106)</td> <td rowspan="1" colspan="1">85 (76, 93)</td> </tr> <tr> <td rowspan="1" colspan="1">Xenarthra</td> <td rowspan="1" colspan="1">46 (33, 84)</td> <td rowspan="1" colspan="1">72 (44, 109)</td> <td rowspan="1" colspan="1">39 (30, 63)</td> <td rowspan="1" colspan="1">43 (35, 55)</td> </tr> <tr> <td rowspan="1" colspan="1">Afrotheria</td> <td rowspan="1" colspan="1">70 (44, 99)</td> <td rowspan="1" colspan="1">86 (70, 120)</td> <td rowspan="1" colspan="1">52 (37, 74)</td> <td rowspan="1" colspan="1">66 (61, 74)</td> </tr> <tr> <td rowspan="1" colspan="1">Afroinsectiphila</td> <td rowspan="1" colspan="1">54 (37, 80)</td> <td rowspan="1" colspan="1">79 (62, 112)</td> <td rowspan="1" colspan="1">42 (33, 60)</td> <td rowspan="1" colspan="1">56 (45, 66)</td> </tr> <tr> <td rowspan="1" colspan="1">Paenungulata</td> <td rowspan="1" colspan="1">50 (35, 77)</td> <td rowspan="1" colspan="1">76 (63, 101)</td> <td rowspan="1" colspan="1">41 (32, 59)</td> <td rowspan="1" colspan="1">61 (47, 68)</td> </tr> <tr> <td rowspan="1" colspan="1">Boreoeutheria</td> <td rowspan="1" colspan="1">95 (73, 119)</td> <td rowspan="1" colspan="1">113 (92, 174)</td> <td rowspan="1" colspan="1">72 (58, 89)</td> <td rowspan="1" colspan="1">81 (74, 90)</td> </tr> <tr> <td rowspan="1" colspan="1">Laurasiatheria</td> <td rowspan="1" colspan="1">86 (65, 107)</td> <td rowspan="1" colspan="1">106 (85, 162)</td> <td rowspan="1" colspan="1">64 (48, 79)</td> <td rowspan="1" colspan="1">75 (68, 82)</td> </tr> <tr> <td rowspan="1" colspan="1">Eulipotyphla</td> <td rowspan="1" colspan="1">57 (38, 81)</td> <td rowspan="1" colspan="1">91 (71, 156)</td> <td rowspan="1" colspan="1">43 (34, 60)</td> <td rowspan="1" colspan="1">56 (44, 69)</td> </tr> <tr> <td rowspan="1" colspan="1">Euungulata</td> <td rowspan="1" colspan="1">65 (43, 88)</td> <td rowspan="1" colspan="1">94 (74, 134)</td> <td rowspan="1" colspan="1">47 (37, 64)</td> <td rowspan="1" colspan="1">65 (59, 72)</td> </tr> <tr> <td rowspan="1" colspan="1">Cetartiodactyla</td> <td rowspan="1" colspan="1">56 (39, 77)</td> <td rowspan="1" colspan="1">74 (57, 100)</td> <td rowspan="1" colspan="1">43 (34, 56)</td> <td rowspan="1" colspan="1">54 (47, 60)</td> </tr> <tr> <td rowspan="1" colspan="1">Chiroptera</td> <td rowspan="1" colspan="1">62 (42, 84)</td> <td rowspan="1" colspan="1">75 (55, 106)</td> <td rowspan="1" colspan="1">46 (36, 63)</td> <td rowspan="1" colspan="1">55 (43, 65)</td> </tr> <tr> <td rowspan="1" colspan="1">Ferae</td> <td rowspan="1" colspan="1">52 (35, 76)</td> <td rowspan="1" colspan="1">94 (74, 137)</td> <td rowspan="1" colspan="1">41 (32, 57)</td> <td rowspan="1" colspan="1">68 (61, 75)</td> </tr> <tr> <td rowspan="1" colspan="1">Carnivora</td> <td rowspan="1" colspan="1">39 (32, 59)</td> <td rowspan="1" colspan="1">71 (38, 102)</td> <td rowspan="1" colspan="1">36 (30, 45)</td> <td rowspan="1" colspan="1">44 (35, 61)</td> </tr> <tr> <td rowspan="1" colspan="1">Euarchontoglires</td> <td rowspan="1" colspan="1">82 (60, 105)</td> <td rowspan="1" colspan="1">104 (83, 146)</td> <td rowspan="1" colspan="1">61 (42, 76)</td> <td rowspan="1" colspan="1">76 (69, 85)</td> </tr> <tr> <td rowspan="1" colspan="1">Glires</td> <td rowspan="1" colspan="1">65 (41, 89)</td> <td rowspan="1" colspan="1">98 (79, 130)</td> <td rowspan="1" colspan="1">47 (35, 65)</td> <td rowspan="1" colspan="1">73 (66, 81)</td> </tr> <tr> <td rowspan="1" colspan="1">Rodentia</td> <td rowspan="1" colspan="1">52 (36, 74)</td> <td rowspan="1" colspan="1">89 (70, 116)</td> <td rowspan="1" colspan="1">41 (32, 55)</td> <td rowspan="1" colspan="1">67 (61, 73)</td> </tr> <tr> <td rowspan="1" colspan="1">Euarchonta</td> <td rowspan="1" colspan="1">69 (46, 91)</td> <td rowspan="1" colspan="1">91 (70, 128)</td> <td rowspan="1" colspan="1">50 (37, 67)</td> <td rowspan="1" colspan="1">65 (58, 73)</td> </tr> <tr> <td rowspan="1" colspan="1">Primates</td> <td rowspan="1" colspan="1">50 (36, 72)</td> <td rowspan="1" colspan="1">77 (59, 109)</td> <td rowspan="1" colspan="1">40 (32, 54)</td> <td rowspan="1" colspan="1">58 (52, 65)</td> </tr> </tbody> </table> </div> <p /> <p>The same pull towards unrealistic deep-root scenarios occurs under the uniform clock tree prior, which must also be described as vague or uninformative about divergence times. Without fossils, the estimated crown age of Placentalia under this prior is 84 (64, 151) Ma, which is not unreasonable. However, inclusion of fossils in the analysis, without changing any of the prior settings, moves the estimate back to 284 (203, 374) Ma, which would imply more than 200 Myr of evolution of crown placentals without leaving a trace in the fossil record.</p> <p>We term the pull towards deep-root scenarios that we observe under TED with vague or erroneous priors DRA. To explore the causes of DRA in the study dataset, we examined four informative diversification models that put less prior probability on deep-root scenarios with long ghost lineages: a model with a low expected extinction rate, a model with a high expected fossil sampling rate, a model combining both of these expectations and a model assuming a high net diversification rate (up to the time of the last speciation event observed in the diversified tree; after that event, the tree is no longer informative about diversification rates because of the absence of speciation events).</p> <p>Detailed analysis of divergence time estimates for three key clades under these models (<a href="#RSTB20150136F5">figure 5</a>) shows that the vague prior is associated with a posterior that is both diffuse and bimodal. The multiple peaks in the distribution are caused by different topological placements of key fossils. For instance, the age estimate for crown placentals (<a href="#RSTB20150136F5">figure 5</a><i>a</i>) is affected by the position of <i>Eomaia</i>, which can be placed either within or outside of Placentalia, and both solutions have significant posterior probability. </p><div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/"><span class="label" tagx="label" title="label">Figure 5.</span></a></div>   <p>The influence of diversification and fossil sampling parameter priors. (<i>a</i>) Divergence time estimates (posterior densities) for crown Placentalia under different priors on diversification and fossil sampling parameters. (<i>b</i>) Ditto for crown Rodentia. (<i>c</i>) Ditto for crown Chiroptera. Ideally, with a large tree and a rich fossil sample, there will be enough information to infer diversification and fossil sampling parameters under an uninformative prior, but the signal in the eutherian data is not strong enough to resist DRA under such conditions. Instead, an uninformative prior results in a poorly defined, bimodal deep-root scenario in the posterior (solid line). It is sufficient to introduce a moderate extra penalty for unobserved ghost lineages to shift the posterior to a well-defined solution with a minimal gap between rocks and clocks. Such penalties may involve assumptions of low extinction rates, high fossil sampling probabilities, high net diversification or any combination of these (various dashed lines); the result is similar under all these scenarios. Bimodal distributions correspond to cases where there are alternative likely placements for key fossils.</p>   </div> <p /> <p>In contrast, the models with informative priors all give posterior distributions that have rather small variance and that are often unimodal (<a href="#RSTB20150136F5">figure 5</a>). When they are bimodal, the cause is the same as for the model with vague priors. For instance, the bimodal posterior for the age of bats (<a href="#RSTB20150136F5">figure 5</a><i>c</i>) is due to the uncertainty in placing the two bat fossils: either inside or outside crown bats.</p> <p>Importantly, although the informative-prior models are quite different among themselves, they produce similar posterior distributions. This is true not only for the key clades examined here, but also for the age estimates in general and for parameters affecting rates of morphological and molecular evolution (<a href="#RSTB20150136TB1">table 1</a> and electronic supplementary material, tables S1, S2). Making the informative priors penalize ghost lineages even more than shown here (for instance, by changing the prior on <i>f</i> from Beta(100,1) to Beta(10 000,1)) did not change parameter estimates, including divergence time estimates, appreciably.</p> </div> <div class="fossilcontributiontodating" title="sec"><span class="label" tagx="label" title="label">(e)</span><div class="title" tagx="title" title="title">
Fossil contribution to dating</div> <p>To assess the fossil contribution to dating, we compared divergence time estimates obtained with and without fossils included, using identical priors (except for the fossil sampling prior required by the FBD model, which is inapplicable when fossils are excluded). The results show that fossils make divergence time estimates more precise and more robust to variations in prior assumptions, given that DRA is addressed properly (<a href="#RSTB20150136F6">figure 6</a>). </p><div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/"><span class="label" tagx="label" title="label">Figure 6.</span></a></div>   <p>The influence of fossils on divergence time estimates. (<i>a</i>) Divergence time estimates (posterior densities) for crown Placentalia in analyses with fossils (black lines) and without fossils (red lines), and under different priors on diversification and fossil sampling parameters. Scenarios with fossils correspond to those in <a href="#RSTB20150136F5">figure 5</a>, except that the uninformative-prior scenario is omitted for clarity. Scenarios with fossils are equivalent, when applicable, except that fossil-related parameters are excluded. (<i>b</i>) Ditto for crown Rodentia. (<i>c</i>) Ditto for crown Chiroptera. Despite the conflict between morphology and molecules, making it challenging to place the fossils correctly in the extant tree, the fossils nevertheless contribute important dating information. Specifically, they tend to make divergence time estimates more robust to variations in prior assumptions and more precise (black lines) than if dating is solely based on information from extant taxa (red lines). (Online version in colour.)</p>   </div> <p /> <p>Without fossils, a vague prior or a low-extinction prior both give diffuse posterior distributions that do not contain much dating information, whereas a rapid-diversification prior pulls the dates so strongly towards the recent that the tree gets inappropriately compressed against the minimum root age constraint (which was set to 64 Ma; see the inferred age of Placentalia, <a href="#RSTB20150136F6">figure 6</a><i>a</i>). The result is divergence time estimates that are quite precise but probably too young given the fossil record. For instance, without fossils, the rapid diversification prior suggests an age of 42 (33, 53) Ma for crown-group rodents (<a href="#RSTB20150136F6">figure 6</a><i>b</i>) while even conservative inference from the fossil record suggests at least 52.5 Ma (based on several ‘primitive’ hystricognaths [<a href="#RSTB20150136C10">10</a>]). Furthermore, the same prior places bats at 40 (31, 50) Ma (<a href="#RSTB20150136F6">figure 6</a><i>c</i>), whereas the fossil record indicates a minimum of 45 Ma based on the age of <i>Tanzanycteris</i> or possibly 47 Ma based on the slightly older <i>Tachypteron</i>. When fossils are included in the analysis, the divergence times arguably agree better with the fossil record; at least, they meet the expected minimum age requirements for both rodents and bats (<a href="#RSTB20150136F6">figure 6</a><i>b</i>,<i>c</i>).</p> </div> <div class="effectofconflictbetweenmorphologyandmolecules" title="sec"><span class="label" tagx="label" title="label">(f)</span><div class="title" tagx="title" title="title">
Effect of conflict between morphology and molecules</div> <p>To tease apart the DRA effect caused by a vague FBD prior and the part caused by the difficulty of reconciling the morphological data and fossil ages with the extant tree, we fixed the tree topology using the ending tree from one of the informative FBD prior analyses (the rapid diversification prior), and then inferred the divergence times with morphological data excluded. When topology was unconstrained (except for the rooting constraint) and morphology was included, vague priors placed the placental origin at approximately 118 Ma, whereas informative priors placed the origin at around 85 Ma (figures <a href="#RSTB20150136F5">5<i>a</i></a> and <a href="#RSTB20150136F6">6</a><i>a</i>). The topologically constrained analysis under the vague FBD prior and without morphological data placed the origin at around 88 Ma. Thus, the main driver of DRA in this dataset appears to be the difficulty of fitting the required amount of morphological evolution under the Mk model onto what would otherwise appear to be the most reasonable clock tree.</p> </div> <div class="insightsfromalternativemodels" title="sec"><span class="label" tagx="label" title="label">(g)</span><div class="title" tagx="title" title="title">
Insights from alternative models</div> <p>Unlinking morphological and molecular-relaxed clocks had no major effect on the inferred divergence times, indicating that it is problems with the morphological model itself that drive DRA, and not inappropriate linking of morphological and molecular rate variation. The skyline FBD model with vague priors was not able to pick up a signal of rapid placental radiation around the K–Pg event. Instead, the skyline FBD model actually made the DRA effect worse, presumably by adding more independent parameters with vague priors.</p> </div> <div class="divergencetimeestimates" title="sec"><span class="label" tagx="label" title="label">(h)</span><div class="title" tagx="title" title="title">
Divergence time estimates</div> <p>Overall, the total-evidence analysis (under informative priors) provides divergence time estimates that are younger than those obtained under most molecular-clock analyses published to date, even though the inferred ages are still older than what is obtained from analyses that minimize ghost lineages, especially for the deepest splits in the tree (<a href="#RSTB20150136F7">figure 7</a> and <a href="#RSTB20150136TB1">table 1</a>). Some shallow divergences have young means, but the highest posterior density almost always overlaps the fossil record expectations. No placental orders are inferred to predate the K–Pg boundary except for Rodentia, which barely reaches into the Cretaceous. In total, 12 extant placental lineages are inferred to cross the K–Pg boundary but only eight of them pass beyond 70 Ma. </p><div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/"><span class="label" tagx="label" title="label">Figure 7.</span></a></div>   <p>Divergence time estimates for the placental radiation under TED. Results are shown for the rapid diversification model, but results are similar for other informative priors on the diversification process (figures <a href="#RSTB20150136F5">5</a> and <a href="#RSTB20150136F6">6</a>). The estimated age for placentals (crown eutherians) is around 86 Ma, younger than in most molecular-clock analyses but still a considerable distance from the post-K–Pg-boundary estimate favoured by some palaeontologists. Note that the crown groups of most placental orders postdate the K–Pg boundary, consistent with the fossil record. (Online version in colour.)</p>   </div> <p /> <p>The placental crown radiation is dated to 85 (76, 93) Ma under the rapid diversification prior, and the other informative total-evidence priors give very similar results (<a href="#RSTB20150136F6">figure 6</a><i>a</i> and electronic supplementary material, table S1). Because the same rapid diversification prior places the radiation of placental mammals at 67 (64, 77) Ma when fossils are excluded from the analysis (<a href="#RSTB20150136F6">figure 6</a><i>a</i>), it is clear that the fossil data provide considerable resistance against a scenario where the entire placental radiation took place in the Palaeogene.</p> </div> <div class="positionsoffossils" title="sec"><span class="label" tagx="label" title="label">(i)</span><div class="title" tagx="title" title="title">
Positions of fossils</div> <p>Under a total-evidence analysis, some fossils float around in the tree while the rich morphological data succeed in placing many others with considerable certainty (<a href="#RSTB20150136F8">figure 8</a>). Most of the fossil placements are quite consistent with palaeontological interpretations of the fossil record. The more notable exceptions include a monophyletic grouping of ‘condylarths’ usually thought of as paraphyletically or polyphyletically distributed in the tree, from presumed basal or stem placentals (<i>Protungulatum</i>) to South American (<i>Didolodus</i>, <i>Protolipterna</i>) and Northern Hemisphere (<i>Hyopsodus</i>, <i>Phenacodus</i>, <i>Apheliscus</i>) relatives of afrotherian or laurasiatherian ungulates. Another pair of presumed ungulate fossils (<i>Thomashuxleya</i>, <i>Carodnia</i>) are attracted towards basal Afrotheria lineages (the same effect is seen in O'Leary <i>et al</i>. [<a href="#RSTB20150136C8">8</a>]). Furthermore, three fossils that are usually considered to be crown or stem cetaceans [<a href="#RSTB20150136C30">30</a>], <i>Rhodocetus</i>, <i>Artiocetus</i> and <i>Basilosaurus</i>, are pulled deeper down into the tree in our analysis, forming a group of stem artiodactyls together with <i>Mesonyx</i>, a traditional mesonychid sister taxon to cetaceans. Most other fossil placements would seem to be in line with expectations. <i>Leptictis</i>, for example, is sister to crown placentals, which is more consistent with the traditional view that its closest affinities are with Late Cretaceous eutherians, such as <i>Gypsonictos</i> [<a href="#RSTB20150136C31">31</a>]. However, it seems that the large number of cranial and dental characters in the dataset might have resulted in the grouping of <i>Mesonyx</i> with fossil cetaceans, which is contradicted by major basicranial and ankle character complexes. </p><div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/"><span class="label" tagx="label" title="label">Figure 8.</span></a></div>   <p>Inferred phylogenetic affinities of fossils. Despite the plasticity in the morphological data and/or the imperfection of our models of morphological evolution, the placements of fossils suggested by total-evidence analyses largely agree with the views expressed by palaeontologists (see text for more detailed discussion). Results are shown for the model assuming rapid diversification, but fossil placements were similar for other informative priors on the diversification and fossil sampling processes.</p>   </div> <p /> </div> </div> <div class="discussion" title="sec"><span class="label" tagx="label" title="label">4.</span><div class="title" tagx="title" title="title">
Discussion</div> <div class="reconcilingphenomicsandgenomics" title="sec"><span class="label" tagx="label" title="label">(a)</span><div class="title" tagx="title" title="title">
Reconciling phenomics and genomics</div> <p>TED is based on simultaneous treatment of all available evidence—including morphology (phenomics), molecules (genomics) and the fossil record—in an integrated statistical analysis [<a href="#RSTB20150136C2">2</a>–<a href="#RSTB20150136C4">4</a>]. The strong conflict between morphological and molecular signal in mammals is clearly problematic in such a context, but at least a unified statistical framework provides the platform needed to understand the nature of the different signals and to make progress in reconciling them.</p> <p>Springer <i>et al</i>. [<a href="#RSTB20150136C13">13</a>] highlighted the fact that the impressively large phenomic dataset assembled by O'Leary <i>et al</i>. [<a href="#RSTB20150136C8">8</a>]—when analysed using parsimony methods ([<a href="#RSTB20150136C8">8</a>]: electronic supplementary material, figure S2; [<a href="#RSTB20150136C13">13</a>]: <a href="#RSTB20150136F1">figure 1</a>)—supports a number of groups that many workers today interpret as reflecting convergent ecological adaptations rather than as genealogical relationships. The problematic assemblages include an ‘insectivore’ group (where most workers would probably agree that the small Madagascar hedgehog is misplaced), an ‘ant and termite eating’ group (pulling together unrelated forms like the pangolin, aardvark, anteater and armadillo), a ‘tree-dwelling group’ (placing bats with primates and their relatives) and an ‘ungulate’ group (including taxa such as elephants, sirenians and the rock hyrax alongside artiodactyls and perissodactyls) [<a href="#RSTB20150136C13">13</a>].</p> <p>The results from our statistical reanalysis of the phenomic data (<a href="#RSTB20150136F1">figure 1</a><i>c</i>) differ in minor details from the parsimony analysis, but confirm that the phylogenetic signal is weak and apparently influenced by convergent adaptation to similar life histories. Adding fossils to the statistical analysis (electronic supplementary material, figure S2<i>a</i>) helps improve some parts of the morphology tree, but the tree still contains most of the unexpected groupings found in the parsimony analysis and in the statistical analysis of extant taxa.</p> <p>Adding molecular data to the statistical analysis appears to resolve most of the phylogenetic artefacts caused by morphological convergence (<a href="#RSTB20150136F1">figures 1<i>a</i></a> and <a href="#RSTB20150136F8">8</a>; electronic supplementary material, figure S2<i>b</i>). Nevertheless, as long as there is significant mismatch between the results of statistical phylogenetic analysis of the morphological data and the expert evaluation of the same data by comparative anatomists and palaeontologists, we should be concerned that our stochastic models of morphological evolution might be inadequate. Indeed, it is easy to see a number of serious potential shortcomings in the standard Mk model for the evolution of discrete morphological characters, which we used here.</p> <p>First, the Mk model does not accommodate directional evolution, which is readily shown to occur in some morphological datasets over the timescales involved here [<a href="#RSTB20150136C32">32</a>]. Perhaps more importantly, the variation in the rate of morphological evolution across characters and lineages is poorly understood and has probably been inadequately modelled so far. For instance, we explored models assuming that morphological and molecular clocks are either the same or completely independent. In reality, however, the morphological and molecular rates are likely to be neither identical nor independent but correlated. Furthermore, we used an uncorrelated relaxed clock model even though there is much to suggest that a significant component of rate variation across lineages is constrained by long-term autocorrelation, and failure to accommodate for this is likely to bias inferred divergence times. Separately, we explore more sophisticated models of rate variation that account for both long-term and short-term rate variation and their effects on dating the placental radiation [<a href="#RSTB20150136C33">33</a>]. Analyses under such models suggest that the naive uncorrelated model used in this paper does not fully recognize the rapid evolutionary rate of rodents, presumably owing to their small body size and therefore short generation time, which is strongly phylogenetically constrained. The effect is that the current analysis probably overestimates the age of crown rodents, so that the conclusion that they predate the K–Pg event may be erroneous [<a href="#RSTB20150136C34">34</a>].</p> <p>Another significant source of error in current stochastic models of morphological evolution is the assumption that characters evolve independently of each other. In reality, morphological evolution is likely to be strongly constrained by various types of character dependencies. When single evolutionary events can affect large groups of correlated characters, the probability of unrelated lineages evolving similar morphologies is much higher than if all characters evolve independently, and failure to account for this is likely to lead to errors in phylogenetic inference akin to the effect of long-branch attraction. The symptoms may include erroneous or overconfident topological inference, as well as excessively precise and biased branch length estimates, both of which may be problematic in TED. The grouping of ecologically similar but unrelated forms in the phenomic tree (<a href="#RSTB20150136F1">figure 1</a><i>c</i>), and the fact that some fossil placements seem to put undue weight on highly correlated dental features (<a href="#RSTB20150136F8">figure 8</a>), could both be explained as artefacts in our analyses caused by an inappropriate assumption of evolutionary independence among morphological characters.</p> <p>Accounting for interdependence among morphological characters is challenging, but such work will definitely be worthwhile given the importance of realistic models of morphological evolution for accurate dating of phylogenies with fossils. Denser taxon sampling, especially of fossils, might help to some extent but to really solve the problem we need to get at the correlation structure itself, either by inferring it or by modelling it in the prior. Methods for learning the correlation structure of small numbers of discrete (morphological) characters are well known [<a href="#RSTB20150136C35">35</a>], and there has been considerable progress in sampling from models accounting for large sets of discrete (molecular) characters with known correlation structure [<a href="#RSTB20150136C36">36</a>,<a href="#RSTB20150136C37">37</a>]. The challenge is to find computationally feasible ways of accommodating large numbers of morphological characters with unknown correlation structure, or to find reasonable <i>a priori</i> representations of the way in which morphological characters are likely to coevolve.</p> </div> <div class="deeprootattraction" title="sec"><span class="label" tagx="label" title="label">(b)</span><div class="title" tagx="title" title="title">
Deep root attraction</div> <p>Our results suggest that there is, indeed, a tendency for TED to produce unrealistically old divergence time estimates under certain conditions, the effect we term DRA. It appears that DRA is caused by the combination of vague priors with various types of model inadequacies. Specifically, it seems that the strong conflict between morphological and molecular signals is one of the primary drivers of DRA in the data of O'Leary <i>et al</i>. The conflict appears to be caused to a large extent by a failure to account for the evolutionary dependencies among morphological characters, resulting in positively misleading signal due to functional convergence when morphology is analysed alone. Even though most of the topological artefacts are corrected when the morphological data are combined with molecular data, there is a large amount of unlikely convergent evolution in morphology to explain in the combined-data tree. This results in a push towards older dates, giving more evolutionary time for unlikely events to occur. If this interpretation is correct, then we would expect DRA to be more severe the larger the morphological dataset, with massive phenomic datasets like the one analysed here representing the worst case as they are likely to include large numbers of correlated characters.</p> <p>It is obvious from our results that a failure to account for diversified sampling of extant taxa can also cause severe DRA in the dating of higher-level phylogenies. This is consistent with results for the dating of the early radiation of Hymenoptera (ants, wasps and bees) [<a href="#RSTB20150136C5">5</a>] and an earlier analysis of the placental radiation [<a href="#RSTB20150136C15">15</a>]. Thus, accounting for tip sampling biases is important in addressing DRA. The smaller the fraction of taxa sampled for the analysis, the more important it probably becomes to model the sampling biases correctly. In contrast, making the TED model more realistic by addressing speciation, extinction and fossil sampling using the FBD process is not sufficient to remove the effects of DRA, at least not under uninformative priors. When the possibility of different (elevated) diversification rates just after the K–Pg event is accommodated using the skyline FBD model, DRA becomes even stronger in our analyses, presumably because of the addition of more parameters with vague priors. Thus, it appears that the signal in the data concerning diversification processes is not strong enough to overcome DRA under standard FBD priors. Similarly, uncoupling the morphological and molecular clocks has little effect on DRA.</p> <p>Fixing the placement of fossils based on palaeontological expertise and not using the morphological data in divergence time estimation appears to be one way to address DRA. Doing so, one would still rely on a principled approach in integrating information on fossil ages, molecular data and diversification processes in the analysis [<a href="#RSTB20150136C33">33</a>,<a href="#RSTB20150136C38">38</a>]. However, ultimately, the purpose of TED is to explicitly formalize all aspects of the macroevolutionary process, including the evolution of morphological characters, and to use this information in integrating over the uncertainty in the placement of fossils.</p> <p>While waiting for better models of morphological evolution, our analyses show that a simple way to control DRA is to increase the penalty for unobserved ghost lineages in order to correct for model inadequacies and to augment weak signal in the data. The exact method for penalizing ghost lineages appears to be unimportant; the end result is very similar with respect to the posterior probability distribution. Increasing the penalty beyond the point where a phase shift occurs in the posterior appears to have no effect; thus, it is certainly not possible to get whichever divergence time estimates you like by these types of prior modifications.</p> <p>Of the informative priors we explored for the placental data, presumably the assumption of rapid net diversification up to the last sampled speciation event is the one that comes closest to a reasonable prior model assumption. However, we do not argue that any of the informative priors we used here is necessarily a reasonable model prior for the placental data; the penalizing effect on ghost lineages is the important factor, which can be used as an <i>ad hoc</i> fix for DRA when the primary interest is divergence time estimation.</p> <p>Clearly, the informative priors are partly compensating for the fact that we are not accounting for all of the relevant background information in the rest of the model. Consider, for instance, that the strongest evidence we have for an exclusively or largely Palaeogene radiation of placentals is the observation that we have found so many placental fossils in the Palaeogene but none in the Cretaceous (at least not definitive crown placentals). The ratio of the number of fossils found before and after the critical time period is one of the critical factors determining the odds against a placental origin in the Cretaceous. In the analysis, we discuss in this paper, the magnitude of this factor is estimated from the distribution in time of a sample of some 30 fossils, a miniscule subset of the placental species actually present in the fossil record. Clearly, the odds against a Cretaceous origin of placentals will be grossly underestimated using such a small fossil sample. We should really correct for this by using a more informative prior (representing more of the background information) or by including more fossils in the analysis.</p> <p>There are several other ways of making the FBD prior more informative. One interesting possibility is to consider models that would accommodate spatially structured fossil sampling probabilities. This would allow for ghost lineages to be penalized less when fossil records are expected to be poor, such as for the afrotherian stem; there are no known mammal fossils from the Late Cretaceous of Africa. A fairly obvious improvement would be to consider diversification models allowing diversification rates to vary over time, and extend the sampling of extant and extinct taxa to increase our ability to infer such patterns.</p> <p>It remains an open question whether a more powerful empirical signal concerning the diversification and sampling processes is enough to compensate for the DRA effect caused by other model inadequacies, such as the failure to account for non-independence in morphological evolution. However, it is clear that we can learn much from studying all these model aspects in an integrative context, such as that of TED.</p> <p>Although we have pointed to several reasons to expect DRA in TED, there are also TED effects that counteract DRA. In particular, when morphological evolution can be modelled accurately, fossil morphology is well known, and fossils are close to predicted ancestral phenotypes, fossils will tend to pull speciation events towards the recent. Thus, although it is quite possible that DRA is a common problem in TED [<a href="#RSTB20150136C6">6</a>], each case is potentially unique and warrants a fresh look before any definite conclusions are drawn.</p> </div> <div class="datingtheplacentalradiation" title="sec"><span class="label" tagx="label" title="label">(c)</span><div class="title" tagx="title" title="title">
Dating the placental radiation</div> <p>We find it encouraging that the total-evidence analysis presented here produces divergence time estimates that match the fossil record closely despite the fact that we did not use any direct constraints on node dates. The explosive radiation scenario favoured by O'Leary <i>et al</i>. [<a href="#RSTB20150136C8">8</a>,<a href="#RSTB20150136C14">14</a>] places the placental origin just after the K–Pg mass extinction at 66 Ma. It requires both net diversification rates and rates of molecular evolution during the early phases of placental evolution that are unlikely considering other sources of evidence on mammalian evolution [<a href="#RSTB20150136C13">13</a>]. O'Leary <i>et al</i>. contrasts the explosive radiation scenario with a much-cited molecular-clock study that uses node dating and places the origin of crown placentals at 101 Ma [<a href="#RSTB20150136C9">9</a>]. Our TED analysis, under informative priors, halves the gap between these two estimates, from approximately 36 to 16–19 Myr (<a href="#RSTB20150136TB1">table 1</a> and electronic supplementary material, table S1). Our estimates for the timing of the subsequent Palaeogene radiation of placentals come even closer to the dates inferred from the fossil record according to O'Leary <i>et al</i>. [<a href="#RSTB20150136C8">8</a>]. Our estimate for the timing of the primary interordinal radiation of placental mammals that Meredith <i>et al</i>. [<a href="#RSTB20150136C9">9</a>] date at approximately 83 Ma, falls close to the 66 Ma K–Pg boundary, with 95% highest posterior densities of several major clades—including Euarchonta, Eulipotyphla, Afrotheria, Euungulata (artiodactyls, perissodactyls) and Ferae (carnivorans, pangolins)—overlapping this boundary.</p> <p>We note that our divergence time estimates agree fairly well with a recent phylogenomic study using a rich set of ‘soft’ node calibrations [<a href="#RSTB20150136C10">10</a>], but our estimates tend to have more uncertainty associated with them. The increased uncertainty is not due to the fact that we used a smaller molecular dataset than dos Reis <i>et al</i>. [<a href="#RSTB20150136C10">10</a>]. In our analyses, effective branch lengths (measured in expected substitutions per site or character) in the extant tree were estimated with high precision (e.g. electronic supplementary material, figure S1<i>c</i>) and were almost invariant across models. Thus, almost all of the dating uncertainty comes from other parts of the TED model. So are the estimates from our analyses more or less reliable than those from the node dating study based on phylogenomic data?</p> <p>This is not an easy question to answer. Although TED is an elegant and theoretically satisfying approach, TED is not necessarily better than node dating. It should be possible to summarize the information in the fossil record quite accurately, particularly in a set of soft node calibrations as used in dos Reis <i>et al</i>. [<a href="#RSTB20150136C10">10</a>]. However, deriving these calibrations is a difficult exercise that is prone to error and various biases, and that could possibly result in exaggerated precision. Thus, it is conceivable at least that the general uncertainty levels in our analyses are more realistic, despite the modelling problems that we encountered.</p> <p>Regardless of this particular comparison, it is clear that the general advantage of the TED approach is that it offers a unified statistical framework for understanding the relative contributions of different sources of evidence, and for combining them using the common arbiter of probability. This should make it easier to identify modelling problems, to improve the reconciliation among conflicting data signals, to successively account for more of the available evidence and ultimately to improve the accuracy of divergence time estimation.</p> </div> </div> <div class="supplementary-material" title="supplementary-material"> <div class="title" tagx="title" title="title">
Supplementary Material</div> <div class="local-data" title="local-data">  <div class="title" tagx="title" title="title">
Supplementary figures and tables</div>  <div class="media" title="media"><a href="rstb20150136supp1.pdf">LINK</a></div> </div> </div> <div class="supplementary-material" title="supplementary-material"> <div class="title" tagx="title" title="title">
Supplementary Material</div> <div class="local-data" title="local-data">  <div class="title" tagx="title" title="title">
Appendix S1</div>  <div class="media" title="media"><a href="rstb20150136supp2.zip">LINK</a></div> </div> </div> </div> <div class="back" title="back"> <div class="ack" title="ack"> <div class="title" tagx="title" title="title">
Acknowledgements</div> <p>We thank Chi Zhang for assistance with some of the analyses, and Mario Reis and an anonymous reviewer for valuable comments on the manuscript.</p> </div> <div class="dataaccessibility" title="sec"> <div class="title" tagx="title" title="title">
Data accessibility</div> <p>The datasets and M <div class="sc_UNKNOWN">
r</div>B <div class="sc_UNKNOWN">
ayes</div> scripts used in this paper were uploaded as part of the electronic supplementary material.</p> </div> <div class="authors'contributions" title="sec"> <div class="title" tagx="title" title="title">
Authors' contributions</div> <p>The authors jointly designed the study. F.R. ran the analyses and drafted a first version of the manuscript. The final manuscript was a joint effort.</p> </div> <div class="competinginterests" title="sec"> <div class="title" tagx="title" title="title">
Competing interests</div> <p>The authors have no competing interests.</p> </div> <div class="funding" title="sec"> <div class="title" tagx="title" title="title">
Funding</div> <p>F.R. was supported by the Swedish Research Council (grant no. 2014-5901). N.L. was supported by the French National Research Agency, grant no. ANR-10-BINF-01-01 ‘Ancestrome’. Computational resources were kindly provided by the Swedish National Infrastructure for Computing (project 2014-1-323).</p> </div> <div class="references">
References</div> <div tag="ref-list"> <ul> <div class="title" tagx="title" title="title">
References</div> <li tag="ref"><a name="RSTB20150136C1" /><span class="label" tagx="label" title="label">1</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Parham</span><span class="given-names" tagx="given-names" title="given-names">JF</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="year" tagx="year" title="year">2012</span><span class="mixed-article-title" title="mixed-article-title">Best practices for justifying fossil calibrations</span>. <span class="source" tagx="source" title="source">Syst. Biol</span>. <span class="volume" tagx="volume" title="volume">61</span>, <span class="fpage" tagx="fpage" title="fpage">346</span>–<span class="lpage" tagx="lpage" title="lpage">359</span>. (<span class="pub-id"><a href="https://dx.doi.org/10.1093/sysbio/syr107">10.1093/sysbio/syr107</a></span>)<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22105867">22105867</a></span></span></li> <li tag="ref"><a name="RSTB20150136C2" /><span class="label" tagx="label" title="label">2</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lee</span><span class="given-names" tagx="given-names" title="given-names">MSY</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Oliver</span><span class="given-names" tagx="given-names" title="given-names">PM</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hutchinson</span><span class="given-names" tagx="given-names" title="given-names">MN</span></span></span><span class="year" tagx="year" title="year">2009</span><span class="mixed-article-title" title="mixed-article-title">Phylogenetic uncertainty and molecular clock calibrations: a case study of legless lizards (Pygopodidae, Gekkota)</span>. <span class="source" tagx="source" title="source">Mol. Phylogenet. Evol.</span><span class="volume" tagx="volume" title="volume">50</span>, <span class="fpage" tagx="fpage" title="fpage">661</span>–<span class="lpage" tagx="lpage" title="lpage">666</span>. (<span class="pub-id"><a href="https://dx.doi.org/10.1016/j.ympev.2008.11.024">10.1016/j.ympev.2008.11.024</a></span>)<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19111935">19111935</a></span></span></li> <li tag="ref"><a name="RSTB20150136C3" /><span class="label" tagx="label" title="label">3</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Pyron</span><span class="given-names" tagx="given-names" title="given-names">RA</span></span></span><span class="year" tagx="year" title="year">2011</span><span class="mixed-article-title" title="mixed-article-title">Divergence time estimation using fossils as terminal taxa and the origins of Lissamphibia</span>. <span class="source" tagx="source" title="source">Syst. Biol.</span><span class="volume" tagx="volume" title="volume">60</span>, <span class="fpage" tagx="fpage" title="fpage">466</span>–<span class="lpage" tagx="lpage" title="lpage">481</span>. (<span class="pub-id"><a href="https://dx.doi.org/10.1093/sysbio/syr047">10.1093/sysbio/syr047</a></span>)<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21540408">21540408</a></span></span></li> <li tag="ref"><a name="RSTB20150136C4" /><span class="label" tagx="label" title="label">4</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ronquist</span><span class="given-names" tagx="given-names" title="given-names">F</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Klopfstein</span><span class="given-names" tagx="given-names" title="given-names">S</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Vilhelmsen</span><span class="given-names" tagx="given-names" title="given-names">L</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Schulmeister</span><span class="given-names" tagx="given-names" title="given-names">S</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Murray</span><span class="given-names" tagx="given-names" title="given-names">DL</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Rasnitsyn</span><span class="given-names" tagx="given-names" title="given-names">AP</span></span></span><span class="year" tagx="year" title="year">2012</span><span class="mixed-article-title" title="mixed-article-title">A total-evidence approach to dating with fossils, applied to the early radiation of the Hymenoptera</span>. <span class="source" tagx="source" title="source">Syst. Biol</span>. <span class="volume" tagx="volume" title="volume">61</span>, <span class="fpage" tagx="fpage" title="fpage">973</span>–<span class="lpage" tagx="lpage" title="lpage">999</span>. (<span class="pub-id"><a href="https://dx.doi.org/10.1093/sysbio/sys058">10.1093/sysbio/sys058</a></span>)<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22723471">22723471</a></span></span></li> <li tag="ref"><a name="RSTB20150136C5" /><span class="label" tagx="label" title="label">5</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zhang</span><span class="given-names" tagx="given-names" title="given-names">C</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Stadler</span><span class="given-names" tagx="given-names" title="given-names">T</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Klopfstein</span><span class="given-names" tagx="given-names" title="given-names">S</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Heath</span><span class="given-names" tagx="given-names" title="given-names">TA</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ronquist</span><span class="given-names" tagx="given-names" title="given-names">F</span></span></span><span class="year" tagx="year" title="year">2015</span><span class="mixed-article-title" title="mixed-article-title">Total-evidence dating under the fossilized birth–death process</span>. <span class="source" tagx="source" title="source">Syst. Biol.</span><span class="volume" tagx="volume" title="volume">65</span>, <span class="fpage" tagx="fpage" title="fpage">228</span>–<span class="lpage" tagx="lpage" title="lpage">249</span>. (<span class="pub-id"><a href="https://dx.doi.org/10.1093/sysbio/syv080">10.1093/sysbio/syv080</a></span>)<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/26493827">26493827</a></span></span></li> <li tag="ref"><a name="RSTB20150136C6" /><span class="label" tagx="label" title="label">6</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">O'Reilly</span><span class="given-names" tagx="given-names" title="given-names">JE</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">dos</span><span class="given-names" tagx="given-names" title="given-names">Reis M</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Donoghue</span><span class="given-names" tagx="given-names" title="given-names">PCJ</span></span></span><span class="year" tagx="year" title="year">2015</span><span class="mixed-article-title" title="mixed-article-title">Dating tips for divergence-time estimation</span>. <span class="source" tagx="source" title="source">Trends Genet</span>. <span class="volume" tagx="volume" title="volume">31</span>, <span class="fpage" tagx="fpage" title="fpage">637</span>–<span class="lpage" tagx="lpage" title="lpage">650</span>. (<span class="pub-id"><a href="https://dx.doi.org/10.1016/j.tig.2015.08.001">10.1016/j.tig.2015.08.001</a></span>)<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/26439502">26439502</a></span></span></li> <li tag="ref"><a name="RSTB20150136C7" /><span class="label" tagx="label" title="label">7</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lee</span><span class="given-names" tagx="given-names" title="given-names">MSY</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Palci</span><span class="given-names" tagx="given-names" title="given-names">A</span></span></span><span class="year" tagx="year" title="year">2015</span><span class="mixed-article-title" title="mixed-article-title">Morphological phylogenetics in the genomic age</span>. <span class="source" tagx="source" title="source">Curr. Biol</span>. <span class="volume" tagx="volume" title="volume">25</span>, <span class="fpage" tagx="fpage" title="fpage">R922</span>–<span class="lpage" tagx="lpage" title="lpage">R929</span>. (<span class="pub-id"><a href="https://dx.doi.org/10.1016/j.cub.2015.07.009">10.1016/j.cub.2015.07.009</a></span>)<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/26439355">26439355</a></span></span></li> <li tag="ref"><a name="RSTB20150136C8" /><span class="label" tagx="label" title="label">8</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">O'Leary</span><span class="given-names" tagx="given-names" title="given-names">MA</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="year" tagx="year" title="year">2013</span><span class="mixed-article-title" title="mixed-article-title">The placental mammal ancestor and the post-K–Pg radiation of placentals</span>. <span class="source" tagx="source" title="source">Science</span><span class="volume" tagx="volume" title="volume">339</span>, <span class="fpage" tagx="fpage" title="fpage">662</span>–<span class="lpage" tagx="lpage" title="lpage">667</span>. (<span class="pub-id"><a href="https://dx.doi.org/10.1126/science.1229237">10.1126/science.1229237</a></span>)<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23393258">23393258</a></span></span></li> <li tag="ref"><a name="RSTB20150136C9" /><span class="label" tagx="label" title="label">9</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Meredith</span><span class="given-names" tagx="given-names" title="given-names">RW</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="year" tagx="year" title="year">2011</span><span class="mixed-article-title" title="mixed-article-title">Impacts of the Cretaceous terrestrial revolution and KPg extinction on mammal diversification</span>. <span class="source" tagx="source" title="source">Science</span><span class="volume" tagx="volume" title="volume">334</span>, <span class="fpage" tagx="fpage" title="fpage">521</span>–<span class="lpage" tagx="lpage" title="lpage">524</span>. (<span class="pub-id"><a href="https://dx.doi.org/10.1126/science.1211028">10.1126/science.1211028</a></span>)<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21940861">21940861</a></span></span></li> <li tag="ref"><a name="RSTB20150136C10" /><span class="label" tagx="label" title="label">10</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">dos</span><span class="given-names" tagx="given-names" title="given-names">Reis M</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Inoue</span><span class="given-names" tagx="given-names" title="given-names">J</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hasegawa</span><span class="given-names" tagx="given-names" title="given-names">M</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Asher</span><span class="given-names" tagx="given-names" title="given-names">RJ</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Donoghue</span><span class="given-names" tagx="given-names" title="given-names">PC</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Yang</span><span class="given-names" tagx="given-names" title="given-names">Z</span></span></span><span class="year" tagx="year" title="year">2012</span><span class="mixed-article-title" title="mixed-article-title">Phylogenomic datasets provide both precision and accuracy in estimating the timescale of placental mammal phylogeny</span>. <span class="source" tagx="source" title="source">Proc. R. Soc. B</span><span class="volume" tagx="volume" title="volume">279</span>, <span class="fpage" tagx="fpage" title="fpage">3491</span>–<span class="lpage" tagx="lpage" title="lpage">3500</span>. (<span class="pub-id"><a href="https://dx.doi.org/10.1098/rspb.2012.0683">10.1098/rspb.2012.0683</a></span>)</span></li> <li tag="ref"><a name="RSTB20150136C11" /><span class="label" tagx="label" title="label">11</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wible</span><span class="given-names" tagx="given-names" title="given-names">JR</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Rougier</span><span class="given-names" tagx="given-names" title="given-names">GW</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Novacek</span><span class="given-names" tagx="given-names" title="given-names">MJ</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Asher</span><span class="given-names" tagx="given-names" title="given-names">RJ</span></span></span><span class="year" tagx="year" title="year">2007</span><span class="mixed-article-title" title="mixed-article-title">Cretaceous eutherians and Laurasian origin for placental mammals near the K/T boundary</span>. <span class="source" tagx="source" title="source">Nature</span><span class="volume" tagx="volume" title="volume">447</span>, <span class="fpage" tagx="fpage" title="fpage">1003</span>–<span class="lpage" tagx="lpage" title="lpage">1006</span>. (<span class="pub-id"><a href="https://dx.doi.org/10.1038/nature05854">10.1038/nature05854</a></span>)<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17581585">17581585</a></span></span></li> <li tag="ref"><a name="RSTB20150136C12" /><span class="label" tagx="label" title="label">12</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">dos</span><span class="given-names" tagx="given-names" title="given-names">Reis M</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Donoghue</span><span class="given-names" tagx="given-names" title="given-names">PCJ</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Yang</span><span class="given-names" tagx="given-names" title="given-names">Z</span></span></span><span class="year" tagx="year" title="year">2014</span><span class="mixed-article-title" title="mixed-article-title">Neither phylogenomic nor palaeontological data support a Palaeogene origin of placental mammals</span>. <span class="source" tagx="source" title="source">Biol. Lett</span>. <span class="volume" tagx="volume" title="volume">10</span>, <span class="fpage" tagx="fpage" title="fpage">20131003</span> (<span class="pub-id"><a href="https://dx.doi.org/10.1098/rsbl.2013.1003">10.1098/rsbl.2013.1003</a></span>)<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24429684">24429684</a></span></span></li> <li tag="ref"><a name="RSTB20150136C13" /><span class="label" tagx="label" title="label">13</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Springer</span><span class="given-names" tagx="given-names" title="given-names">MS</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Meredith</span><span class="given-names" tagx="given-names" title="given-names">RW</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Teeling</span><span class="given-names" tagx="given-names" title="given-names">EC</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Murphy</span><span class="given-names" tagx="given-names" title="given-names">WJ</span></span></span><span class="year" tagx="year" title="year">2013</span><span class="mixed-article-title" title="mixed-article-title">Technical comment on ‘The placental mammal ancestor and the post-K–Pg radiation of placentals’</span>. <span class="source" tagx="source" title="source">Science</span><span class="volume" tagx="volume" title="volume">341</span>, <span class="fpage" tagx="fpage" title="fpage">613</span> (<span class="pub-id"><a href="https://dx.doi.org/10.1126/science.1238025">10.1126/science.1238025</a></span>)</span></li> <li tag="ref"><a name="RSTB20150136C14" /><span class="label" tagx="label" title="label">14</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">O'Leary</span><span class="given-names" tagx="given-names" title="given-names">MA</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="year" tagx="year" title="year">2013</span><span class="mixed-article-title" title="mixed-article-title">Response to comment on ‘The placental mammal ancestor and the post-K–Pg radiation of placentals’</span>. <span class="source" tagx="source" title="source">Science</span><span class="volume" tagx="volume" title="volume">341</span>, <span class="comment">613–613</span> (<span class="pub-id"><a href="https://dx.doi.org/10.1126/science.1238162">10.1126/science.1238162</a></span>)</span></li> <li tag="ref"><a name="RSTB20150136C15" /><span class="label" tagx="label" title="label">15</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Beck</span><span class="given-names" tagx="given-names" title="given-names">RMD</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lee</span><span class="given-names" tagx="given-names" title="given-names">MSY</span></span></span><span class="year" tagx="year" title="year">2014</span><span class="mixed-article-title" title="mixed-article-title">Ancient dates or accelerated rates? Morphological clocks and the antiquity of placental mammals</span>. <span class="source" tagx="source" title="source">Proc. R. Soc. B</span><span class="volume" tagx="volume" title="volume">281</span>, <span class="fpage" tagx="fpage" title="fpage">20141278</span> (<span class="pub-id"><a href="https://dx.doi.org/10.1098/rspb.2014.1278">10.1098/rspb.2014.1278</a></span>)</span></li> <li tag="ref"><a name="RSTB20150136C16" /><span class="label" tagx="label" title="label">16</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kitazoe</span><span class="given-names" tagx="given-names" title="given-names">Y</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kishino</span><span class="given-names" tagx="given-names" title="given-names">H</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Waddell</span><span class="given-names" tagx="given-names" title="given-names">PJ</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Nakajima</span><span class="given-names" tagx="given-names" title="given-names">N</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Okabayashi</span><span class="given-names" tagx="given-names" title="given-names">T</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Watanabe</span><span class="given-names" tagx="given-names" title="given-names">T</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Okuhara</span><span class="given-names" tagx="given-names" title="given-names">Y</span></span></span><span class="year" tagx="year" title="year">2007</span><span class="mixed-article-title" title="mixed-article-title">Robust time estimation reconciles views of the antiquity of placental mammals</span>. <span class="source" tagx="source" title="source">PLoS ONE</span><span class="volume" tagx="volume" title="volume">2</span>, <span class="fpage" tagx="fpage" title="fpage">e384</span> (<span class="pub-id"><a href="https://dx.doi.org/10.1371/journal.pone.0000384">10.1371/journal.pone.0000384</a></span>)<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17440620">17440620</a></span></span></li> <li tag="ref"><a name="RSTB20150136C17" /><span class="label" tagx="label" title="label">17</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Phillips</span><span class="given-names" tagx="given-names" title="given-names">MJ</span></span></span><span class="year" tagx="year" title="year">2016</span><span class="mixed-article-title" title="mixed-article-title">Geomolecular dating and the origin of placental mammals</span>. <span class="source" tagx="source" title="source">Syst. Biol</span>. <span class="volume" tagx="volume" title="volume">65</span>, <span class="fpage" tagx="fpage" title="fpage">546</span>–<span class="lpage" tagx="lpage" title="lpage">557</span>. (<span class="pub-id"><a href="https://dx.doi.org/10.1093/sysbio/syv115">10.1093/sysbio/syv115</a></span>)<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/26658702">26658702</a></span></span></li> <li tag="ref"><a name="RSTB20150136C18" /><span class="label" tagx="label" title="label">18</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ji</span><span class="given-names" tagx="given-names" title="given-names">Q</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Luo</span><span class="given-names" tagx="given-names" title="given-names">Z-X</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Yuan</span><span class="given-names" tagx="given-names" title="given-names">C-X</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wible</span><span class="given-names" tagx="given-names" title="given-names">JR</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zhang</span><span class="given-names" tagx="given-names" title="given-names">J-P</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Georgi</span><span class="given-names" tagx="given-names" title="given-names">JA</span></span></span><span class="year" tagx="year" title="year">2002</span><span class="mixed-article-title" title="mixed-article-title">The earliest known eutherian mammal</span>. <span class="source" tagx="source" title="source">Nature</span><span class="volume" tagx="volume" title="volume">416</span>, <span class="fpage" tagx="fpage" title="fpage">816</span>–<span class="lpage" tagx="lpage" title="lpage">822</span>. (<span class="pub-id"><a href="https://dx.doi.org/10.1038/416816a">10.1038/416816a</a></span>)<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11976675">11976675</a></span></span></li> <li tag="ref"><a name="RSTB20150136C19" /><span class="label" tagx="label" title="label">19</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Luo</span><span class="given-names" tagx="given-names" title="given-names">Z-X</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Yuan</span><span class="given-names" tagx="given-names" title="given-names">C-X</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Meng</span><span class="given-names" tagx="given-names" title="given-names">Q-J</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ji</span><span class="given-names" tagx="given-names" title="given-names">Q</span></span></span><span class="year" tagx="year" title="year">2011</span><span class="mixed-article-title" title="mixed-article-title">A Jurassic eutherian mammal and divergence of marsupials and placentals</span>. <span class="source" tagx="source" title="source">Nature</span><span class="volume" tagx="volume" title="volume">476</span>, <span class="fpage" tagx="fpage" title="fpage">442</span>–<span class="lpage" tagx="lpage" title="lpage">445</span>. (<span class="pub-id"><a href="https://dx.doi.org/10.1038/nature10291">10.1038/nature10291</a></span>)<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21866158">21866158</a></span></span></li> <li tag="ref"><a name="RSTB20150136C20" /><span class="label" tagx="label" title="label">20</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Höhna</span><span class="given-names" tagx="given-names" title="given-names">S</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Stadler</span><span class="given-names" tagx="given-names" title="given-names">T</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ronquist</span><span class="given-names" tagx="given-names" title="given-names">F</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Britton</span><span class="given-names" tagx="given-names" title="given-names">T</span></span></span><span class="year" tagx="year" title="year">2011</span><span class="mixed-article-title" title="mixed-article-title">Inferring speciation and extinction rates under different sampling schemes</span>. <span class="source" tagx="source" title="source">Mol. Biol. Evol</span>. <span class="volume" tagx="volume" title="volume">28</span>, <span class="fpage" tagx="fpage" title="fpage">2577</span>–<span class="lpage" tagx="lpage" title="lpage">2589</span>. (<span class="pub-id"><a href="https://dx.doi.org/10.1093/molbev/msr095">10.1093/molbev/msr095</a></span>)<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21482666">21482666</a></span></span></li> <li tag="ref"><a name="RSTB20150136C21" /><span class="label" tagx="label" title="label">21</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lewis</span><span class="given-names" tagx="given-names" title="given-names">P</span></span></span><span class="year" tagx="year" title="year">2001</span><span class="mixed-article-title" title="mixed-article-title">A likelihood approach to estimating phylogeny from discrete morphological character data</span>. <span class="source" tagx="source" title="source">Syst. Biol</span>. <span class="volume" tagx="volume" title="volume">50</span>, <span class="fpage" tagx="fpage" title="fpage">913</span>–<span class="lpage" tagx="lpage" title="lpage">925</span>. (<span class="pub-id"><a href="https://dx.doi.org/10.1080/106351501753462876">10.1080/106351501753462876</a></span>)<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12116640">12116640</a></span></span></li> <li tag="ref"><a name="RSTB20150136C22" /><span class="label" tagx="label" title="label">22</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Huelsenbeck</span><span class="given-names" tagx="given-names" title="given-names">JP</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Larget</span><span class="given-names" tagx="given-names" title="given-names">B</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Alfaro</span><span class="given-names" tagx="given-names" title="given-names">ME</span></span></span><span class="year" tagx="year" title="year">2004</span><span class="mixed-article-title" title="mixed-article-title">Bayesian phylogenetic model selection using reversible jump Markov chain Monte Carlo</span>. <span class="source" tagx="source" title="source">Mol. Biol. Evol</span>. <span class="volume" tagx="volume" title="volume">21</span>, <span class="fpage" tagx="fpage" title="fpage">1123</span>–<span class="lpage" tagx="lpage" title="lpage">1133</span>. (<a href="http://dx.doi.org/0.1093/molbev/msh123">doi:0.1093/molbev/msh123</a>)<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15034130">15034130</a></span></span></li> <li tag="ref"><a name="RSTB20150136C23" /><span class="label" tagx="label" title="label">23</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lepage</span><span class="given-names" tagx="given-names" title="given-names">T</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bryant</span><span class="given-names" tagx="given-names" title="given-names">D</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Philippe</span><span class="given-names" tagx="given-names" title="given-names">H</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lartillot</span><span class="given-names" tagx="given-names" title="given-names">N</span></span></span><span class="year" tagx="year" title="year">2007</span><span class="mixed-article-title" title="mixed-article-title">A general comparison of relaxed molecular clock models</span>. <span class="source" tagx="source" title="source">Mol. Biol. Evol</span>. <span class="volume" tagx="volume" title="volume">24</span>, <span class="fpage" tagx="fpage" title="fpage">2669</span>–<span class="lpage" tagx="lpage" title="lpage">2680</span>. (<span class="pub-id"><a href="https://dx.doi.org/10.1093/molbev/msm193">10.1093/molbev/msm193</a></span>)<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17890241">17890241</a></span></span></li> <li tag="ref"><a name="RSTB20150136C24" /><span class="label" tagx="label" title="label">24</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Stadler</span><span class="given-names" tagx="given-names" title="given-names">T</span></span></span><span class="year" tagx="year" title="year">2010</span><span class="mixed-article-title" title="mixed-article-title">Sampling-through-time in birth–death trees</span>. <span class="source" tagx="source" title="source">J. Theor. Biol</span>. <span class="volume" tagx="volume" title="volume">267</span>, <span class="fpage" tagx="fpage" title="fpage">396</span>–<span class="lpage" tagx="lpage" title="lpage">404</span>. (<span class="pub-id"><a href="https://dx.doi.org/10.1016/j.jtbi.2010.09.010">10.1016/j.jtbi.2010.09.010</a></span>)<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20851708">20851708</a></span></span></li> <li tag="ref"><a name="RSTB20150136C25" /><span class="label" tagx="label" title="label">25</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gavryushkina</span><span class="given-names" tagx="given-names" title="given-names">A</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Welch</span><span class="given-names" tagx="given-names" title="given-names">D</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Stadler</span><span class="given-names" tagx="given-names" title="given-names">T</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Drummond</span><span class="given-names" tagx="given-names" title="given-names">AJ</span></span></span><span class="year" tagx="year" title="year">2014</span><span class="mixed-article-title" title="mixed-article-title">Bayesian inference of sampled ancestor trees for epidemiology and fossil calibration</span>. <span class="source" tagx="source" title="source">PLoS Comput. Biol</span>. <span class="volume" tagx="volume" title="volume">10</span>, <span class="fpage" tagx="fpage" title="fpage">e1003919</span> (<span class="pub-id"><a href="https://dx.doi.org/10.1371/journal.pcbi.1003919">10.1371/journal.pcbi.1003919</a></span>)<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25474353">25474353</a></span></span></li> <li tag="ref"><a name="RSTB20150136C26" /><span class="label" tagx="label" title="label">26</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ronquist</span><span class="given-names" tagx="given-names" title="given-names">F</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="year" tagx="year" title="year">2012</span><span class="mixed-article-title" title="mixed-article-title">MrBayes 3.2: efficient Bayesian phylogenetic inference and model choice across a large model space</span>. <span class="source" tagx="source" title="source">Syst. Biol</span>. <span class="volume" tagx="volume" title="volume">61</span>, <span class="fpage" tagx="fpage" title="fpage">539</span>–<span class="lpage" tagx="lpage" title="lpage">542</span>. (<span class="pub-id"><a href="https://dx.doi.org/10.1093/sysbio/sys029">10.1093/sysbio/sys029</a></span>)<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22357727">22357727</a></span></span></li> <li tag="ref"><a name="RSTB20150136C27" /><span class="label" tagx="label" title="label">27</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Rambaut</span><span class="given-names" tagx="given-names" title="given-names">A</span></span></span><span class="year" tagx="year" title="year">2014</span><span class="comment">FigTree version 1.4.2. Software downloaded from <span class="uri" title="uri"><a href="http://tree.bio.ed.ac.uk/software/figtree/">http://tree.bio.ed.ac.uk/software/figtree/</a></span>.</span></span></li> <li tag="ref"><a name="RSTB20150136C28" /><span class="label" tagx="label" title="label">28</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="collab'">collab: R Development Core Team</span>. <span class="year" tagx="year" title="year">2009</span><span class="source" tagx="source" title="source">R: a language and environment for statistical computing</span>. <span class="publisher-loc" tagx="publisher-loc" title="publisher-loc">Vienna, Austria</span>: <span class="publisher-name" tagx="publisher-name" title="publisher-name">R Foundation for Statistical Computing</span>.</span></li> <li tag="ref"><a name="RSTB20150136C29" /><span class="label" tagx="label" title="label">29</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Novacek</span><span class="given-names" tagx="given-names" title="given-names">MJ</span></span></span><span class="year" tagx="year" title="year">1992</span><span class="mixed-article-title" title="mixed-article-title">Mammalian phylogeny: shaking the tree</span>. <span class="source" tagx="source" title="source">Nature</span><span class="volume" tagx="volume" title="volume">356</span>, <span class="fpage" tagx="fpage" title="fpage">121</span>–<span class="lpage" tagx="lpage" title="lpage">125</span>. (<span class="pub-id"><a href="https://dx.doi.org/10.1038/356121a0">10.1038/356121a0</a></span>)<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/1545862">1545862</a></span></span></li> <li tag="ref"><a name="RSTB20150136C30" /><span class="label" tagx="label" title="label">30</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Thewissen</span><span class="given-names" tagx="given-names" title="given-names">JGM</span></span></span><span class="year" tagx="year" title="year">1998</span><span class="source" tagx="source" title="source">The emergence of whales</span>. <span class="publisher-loc" tagx="publisher-loc" title="publisher-loc">New York, NY</span>: <span class="publisher-name" tagx="publisher-name" title="publisher-name">Plenum Press</span>.</span></li> <li tag="ref"><a name="RSTB20150136C31" /><span class="label" tagx="label" title="label">31</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wible</span><span class="given-names" tagx="given-names" title="given-names">JR</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Rougier</span><span class="given-names" tagx="given-names" title="given-names">GW</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Novacek</span><span class="given-names" tagx="given-names" title="given-names">MJ</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Asher</span><span class="given-names" tagx="given-names" title="given-names">RJ</span></span></span><span class="year" tagx="year" title="year">2009</span><span class="mixed-article-title" title="mixed-article-title">The eutherian mammal <i>Maelestes gobiensis</i> from the Late Cretaceous of Mongolia and the phylogeny of Cretaceous Eutheria</span>. <span class="source" tagx="source" title="source">Bull. Am. Mus. Nat. Hist</span>. <span class="volume" tagx="volume" title="volume">327</span>, <span class="fpage" tagx="fpage" title="fpage">1</span>–<span class="lpage" tagx="lpage" title="lpage">123</span>. (<span class="pub-id"><a href="https://dx.doi.org/10.1206/623.1">10.1206/623.1</a></span>)</span></li> <li tag="ref"><a name="RSTB20150136C32" /><span class="label" tagx="label" title="label">32</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Klopfstein</span><span class="given-names" tagx="given-names" title="given-names">S</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Vilhelmsen</span><span class="given-names" tagx="given-names" title="given-names">L</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ronquist</span><span class="given-names" tagx="given-names" title="given-names">F</span></span></span><span class="year" tagx="year" title="year">2015</span><span class="mixed-article-title" title="mixed-article-title">A nonstationary Markov model detects directional evolution in hymenopteran morphology</span>. <span class="source" tagx="source" title="source">Syst. Biol</span>. <span class="volume" tagx="volume" title="volume">64</span>, <span class="fpage" tagx="fpage" title="fpage">1089</span>–<span class="lpage" tagx="lpage" title="lpage">1103</span>. (<span class="pub-id"><a href="https://dx.doi.org/10.1093/sysbio/syv052">10.1093/sysbio/syv052</a></span>)<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/26272507">26272507</a></span></span></li> <li tag="ref"><a name="RSTB20150136C33" /><span class="label" tagx="label" title="label">33</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lartillot</span><span class="given-names" tagx="given-names" title="given-names">N</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Phillips</span><span class="given-names" tagx="given-names" title="given-names">MJ</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ronquist</span><span class="given-names" tagx="given-names" title="given-names">F</span></span></span><span class="year" tagx="year" title="year">2016</span><span class="mixed-article-title" title="mixed-article-title">A mixed relaxed clock model</span>. <span class="source" tagx="source" title="source">Phil. Trans. R. Soc. B</span><span class="volume" tagx="volume" title="volume">371</span>, <span class="fpage" tagx="fpage" title="fpage">20150132</span> (<span class="pub-id"><a href="https://dx.doi.org/10.1098/rstb.2015.0132">10.1098/rstb.2015.0132</a></span>)<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/27325829">27325829</a></span></span></li> <li tag="ref"><a name="RSTB20150136C34" /><span class="label" tagx="label" title="label">34</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wu</span><span class="given-names" tagx="given-names" title="given-names">S</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wu</span><span class="given-names" tagx="given-names" title="given-names">W</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zhang</span><span class="given-names" tagx="given-names" title="given-names">F</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ye</span><span class="given-names" tagx="given-names" title="given-names">J</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ni</span><span class="given-names" tagx="given-names" title="given-names">X</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Sun</span><span class="given-names" tagx="given-names" title="given-names">J</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Edwards</span><span class="given-names" tagx="given-names" title="given-names">SV</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Meng</span><span class="given-names" tagx="given-names" title="given-names">J</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Organ</span><span class="given-names" tagx="given-names" title="given-names">CL</span></span></span><span class="year" tagx="year" title="year">2012</span><span class="mixed-article-title" title="mixed-article-title">Molecular and paleontological evidence for a post-Cretaceous origin of rodents</span>. <span class="source" tagx="source" title="source">PLoS ONE</span><span class="volume" tagx="volume" title="volume">7</span>, <span class="fpage" tagx="fpage" title="fpage">e46445</span> (<span class="pub-id"><a href="https://dx.doi.org/10.1371/journal.pone.0046445">10.1371/journal.pone.0046445</a></span>)<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23071573">23071573</a></span></span></li> <li tag="ref"><a name="RSTB20150136C35" /><span class="label" tagx="label" title="label">35</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Pagel</span><span class="given-names" tagx="given-names" title="given-names">M</span></span></span><span class="year" tagx="year" title="year">1994</span><span class="mixed-article-title" title="mixed-article-title">Detecting correlated evolution on phylogenies: a general method for the comparative analysis of discrete characters</span>. <span class="source" tagx="source" title="source">Proc. R. Soc. Lond. B</span><span class="volume" tagx="volume" title="volume">255</span>, <span class="fpage" tagx="fpage" title="fpage">37</span>–<span class="lpage" tagx="lpage" title="lpage">45</span>. (<span class="pub-id"><a href="https://dx.doi.org/10.1098/rspb.1994.0006">10.1098/rspb.1994.0006</a></span>)</span></li> <li tag="ref"><a name="RSTB20150136C36" /><span class="label" tagx="label" title="label">36</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Robinson</span><span class="given-names" tagx="given-names" title="given-names">DM</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Jones</span><span class="given-names" tagx="given-names" title="given-names">DT</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kishino</span><span class="given-names" tagx="given-names" title="given-names">H</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Goldman</span><span class="given-names" tagx="given-names" title="given-names">N</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Thorne</span><span class="given-names" tagx="given-names" title="given-names">JL</span></span></span><span class="year" tagx="year" title="year">2003</span><span class="mixed-article-title" title="mixed-article-title">Protein evolution with dependence among codons due to tertiary structure</span>. <span class="source" tagx="source" title="source">Mol. Biol. Evol</span>. <span class="volume" tagx="volume" title="volume">20</span>, <span class="fpage" tagx="fpage" title="fpage">1692</span>–<span class="lpage" tagx="lpage" title="lpage">1704</span>. (<span class="pub-id"><a href="https://dx.doi.org/10.1093/molbev/msg184">10.1093/molbev/msg184</a></span>)<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12885968">12885968</a></span></span></li> <li tag="ref"><a name="RSTB20150136C37" /><span class="label" tagx="label" title="label">37</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Rodrigue</span><span class="given-names" tagx="given-names" title="given-names">N</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Philippe</span><span class="given-names" tagx="given-names" title="given-names">H</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lartillot</span><span class="given-names" tagx="given-names" title="given-names">N</span></span></span><span class="year" tagx="year" title="year">2006</span><span class="mixed-article-title" title="mixed-article-title">Assessing site-interdependent phylogenetic models of sequence evolution</span>. <span class="source" tagx="source" title="source">Mol. Biol. Evol</span>. <span class="volume" tagx="volume" title="volume">23</span>, <span class="fpage" tagx="fpage" title="fpage">1762</span>–<span class="lpage" tagx="lpage" title="lpage">1775</span>. (<span class="pub-id"><a href="https://dx.doi.org/10.1093/molbev/msl041">10.1093/molbev/msl041</a></span>)<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16787998">16787998</a></span></span></li> <li tag="ref"><a name="RSTB20150136C38" /><span class="label" tagx="label" title="label">38</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Heath</span><span class="given-names" tagx="given-names" title="given-names">TA</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Huelsenbeck</span><span class="given-names" tagx="given-names" title="given-names">JP</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Stadler</span><span class="given-names" tagx="given-names" title="given-names">T</span></span></span><span class="year" tagx="year" title="year">2014</span><span class="mixed-article-title" title="mixed-article-title">The fossilized birth–death process for coherent calibration of divergence-time estimates</span>. <span class="source" tagx="source" title="source">Proc. Natl. Acad. Sci. USA</span><span class="volume" tagx="volume" title="volume">111</span>, <span class="fpage" tagx="fpage" title="fpage">E2957</span>–<span class="lpage" tagx="lpage" title="lpage">E2966</span>. (<span class="pub-id"><a href="https://dx.doi.org/10.1073/pnas.1319091111">10.1073/pnas.1319091111</a></span>)<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25009181">25009181</a></span></span></li> </ul> </div> </div>  </body></html>