<html xmlns="http://www.w3.org/1999/xhtml"> <head> <style type="text/css">
      	     		 
      	    a                          {background : #ffffff; }
      	    article                    {border-style : dotted; border-width : 2px; }
		 	
		 	div                        {background : #ffffcc;}
		 	div.abstract-title         {font-weight : bold ; font-size : 16pt;}
		 	div.ack                    {border-style : solid ; border-color : red; margin : 2em; }
		 	div.article-meta           {border-style : solid; border-width : 1 pt; margin 1em;}
		 	div.boxed-text             {margin : 5em; border-style : solid;}
		 	div.contrib-group          {margin : 1em; }
		 	div.fig                    {border-style : solid; border-width : 2px; margin : 2em; }
		 	div.funding                {font-weight : bold ; font-size : 16pt;}
		 	div.given-names            {font-style : italic;}
		 	div.intro                  {border-style : inset; margin : 5px;}
		 	div.introduction           {border-style : inset; margin : 5px;}
		 	div.journal-meta           {border-style : solid; border-width : 1 pt; margin 1em;}
		 	div.journal-title-group    {background : #ffeeee;}
		 	div.article-title          {font-weight : bold ; font-size : 18pt;}
		 	div.kwd                    {font-style : italic}
		 	div.meta-name              {font-weight : bold ; font-size : 16pt;}
		 	div.name                   {font-weight : bold;}
		 	div.alt-title              {font-style : italic; font-size : 12px;}
		 	
		 	div.sec                    {border : 2px; margin 5px; padding 2px;}
		 	div.title                  {font-family : courier; font-weight : bold;
		 	                            font-size : 16pt; margin : 5px;}
		 	
		 	div.materials_methods      {border-style : double; margin : 5px; }
		 	div.methods                {border-style : double; margin : 5px; }
		 	
		 	div.results                {border-style : solid; margin : 5px;}
		 	div.background             {border-style : dotted; margin : 5px;}
		 	
		 	div.discussion             {border-style : groove; margin : 5px;}
		 	div.conclusion             {border-style : ridge; margin : 5px;}
		 	div.conclusions             {border-style : ridge; margin : 5px;}
		 	div.supplementary-material {border-style : inset; margin : 5px;}
		 	div.abbreviations          {border-style : double; border-color : red; margin : 5px;}
		 	div.competinginterests     {border-style : double; border-color : blue; margin : 5px;}
		 	div.acknowledgements       {border-style : double; border-color : green; margin : 5px;}
		 	div.authors_contributions   {border-style : double; border-color : purple; margin : 5px;}
		 	
		 	div.publisher              {border-style : outset; margin : 5px;}
		 	div.fn-type-conflict       {background : #f88; }
		 	div.fn-type-con            {background : #ddf; }
		 	div.fn-type-other          {background : #ddd; }
		 	
		 	div.unknown                {background : #ffd;
									 	  border-style : solid;
									 	  border-width : 1px;
									 	  padding : 2 px;}
		 	  
      	    table                      {background : #ffffdd;}
		 	tr                         {background : #ddddff; padding : 1px;}
		 	
		 	span                       {background : #ffcccc;}
		 	
		 	span.citation-author       {font-family : helvetica; background : #ffeeee;}
		 	span.collab                {background : #ddffff; }
		 	span.comment               {font-family : courier; font-size : 6px; background : #ffaaff;}
		 	span.contrib               {background : #ffffff;}
		 	span.corresp               {background : #ddffdd; }
		 	span.doi                   {background : #ffffff;}
		 	span.email                 {font-family : courier; }
		 	span.etal                  {font-style : italic;}
		 	span.fpage                 {font-family : courier;}
		 	span.given-names           {background : #ffffff;}
		 	span.iso-abbrev            {background : #ffffff;}
		 	span.issn-epub             {background : #ffffff;}
		 	span.issn-ppub             {background : #ffffff;}
		 	span.journal-title         {background : #ffffff;}
		 	span.lpage                 {font-family : courier;}
		 	span.mixed-article-title   {font-style : italic ;}
		 	span.nlm-ta                {background : #ffffff;}
		 	span.pmc                   {background : #ffffff;}
		 	span.pmcid                 {background : #ffffff;}
		 	span.pmid                  {background : #ffffff;}
		 	span.publisher             {background : #ffffff;}
		 	span.publisher-id          {background : #ffffff;}
		 	span.publisher-name        {background : #ffffff;}
		 	span.source                {background : #ffffff;}
		 	span.subject               {background : #ffffff;}
		 	span.surname               {background : #ffffff;}
		 	span.volume                {font-family : courier; font-weight : bold;}
		 	span.year                  {font-family : courier ; font-style : italic;}
			</style> </head> <body> <div class="front" title="front"> <div class="journal-meta" tagx="journal-meta" title="journal-meta"><span class="nlm-ta" title="nlm-ta">Eur Ann Otorhinolaryngol Head Neck Dis</span><span class="iso-abbrev" title="iso-abbrev">Eur Ann Otorhinolaryngol Head Neck Dis</span><div class="journal-title-group" tagx="journal-title-group" title="journal-title-group"><span class="journal-title" tagx="journal-title" title="journal-title">European Annals of Otorhinolaryngology, Head and Neck Diseases</span></div><span class="issn-ppub" tagx="issn" title="issn-ppub">1879-7296</span><span class="issn-epub" tagx="issn" title="issn-epub">1879-730X</span><div class="publisher" tagx="publisher" title="publisher"><span class="publisher-name" tagx="publisher-name" title="publisher-name">Elsevier Masson SAS.</span></div> </div> <div class="article-meta" tagx="article-meta" title="article-meta"><span class="pmcid" title="pmcid"> pmcid: <a href="http://www.ncbi.nlm.nih.gov/pubmed/7409732">7409732</a></span><span class="publisher-id" title="publisher-id">S1879-7296(20)30138-1</span><span class="doi" title="doi"> doi: <a href="https://dx.doi.org/10.1016/j.anorl.2020.05.014">10.1016/j.anorl.2020.05.014</a></span><div class="article-categories" title="article-categories">
: <span class="subject" title="subject">Article</span></div> <div class="title-group" tagx="title-group" title="title-group"> <div class="article-title" title="article-title">
Viruses and viral epidemics in the metabolic theory of evolution</div> </div> <div class="contrib-group" title="contrib-group"><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Jankowski</span><span class="given-names" tagx="given-names" title="given-names">R.</span></span><span class="email" tagx="email" title="email">r.jankowski@chru-nancy.fr</span><a href="#aff0005">a</a><a href="#aff0010">b</a><a href="#aff0015">c</a><a href="#cor0005">*</a></span><span class="citation_author_institution" id="aff0005">[a], </span><span class="citation_author_institution" id="aff0010">[b], </span><span class="citation_author_institution" id="aff0015">[c], </span></div> <div class="author-notes" title="author-notes"> <div class="corresp" title="corresp"><span class="label" tagx="label" title="label">⁎</span>Corresponding address: ORL et Chirurgie Cervico-Faciale, Hôpital de Brabois, CHRU de Nancy, Université de Lorraine, Bâtiment Louis-Mathieu, 54500 Vandoeuvre-les-Nancy, France. <span class="email" tagx="email" title="email">r.jankowski@chru-nancy.fr</span></div> </div><span class="pub-date-pmc-release" title="pub-date-pmc-release">pmc-release: <span>2020-8-8</span></span><span class="pub-date-ppub" title="pub-date-ppub">ppub: <span>2020-9</span></span><span class="pub-date-epub" title="pub-date-epub">epub: <span>2020-8-8</span></span><span class="volume" tagx="volume" title="volume">137</span><span class="issue" tagx="issue" title="issue">4</span><span class="fpage" tagx="fpage" title="fpage">297</span><span class="lpage" tagx="lpage" title="lpage">301</span><div class="permissions"><span class="copyright" title="copyright">(C) <span class="copyright-holder" tagx="copyright-holder" title="copyright-holder">Elsevier Masson SAS</span>, <span class="copyright-year" tagx="copyright-year" title="copyright-year">2020</span></span><span class="license" title="license"><span class="license-p" title="license-p">Since January 2020 Elsevier has created a COVID-19 resource centre with free information in English and Mandarin on the novel coronavirus COVID-19. The COVID-19 resource centre is hosted on Elsevier Connect, the company's public news and information website. Elsevier hereby grants permission to make all its COVID-19-related research that is available on the COVID-19 resource centre - including this research content - immediately available in PubMed Central and other publicly funded repositories, such as the WHO COVID database with rights for unrestricted research re-use and analyses in any form or by any means with acknowledgement of the original source. These permissions are granted for free by Elsevier for as long as the COVID-19 resource centre remains active.</span></span></div> <div class="abstract" title="abstract"> <div class="abstract-title" title="abstract-title">
Abstract</div> <p>Viruses, including the SARS-CoV-2 virus responsible for the current COVID-19 epidemic, are a key to the understanding of life and evolution. Cells may have arisen from aqueous sequestration inside a lipid envelope studded with chromophores capable of capturing solar photons. Nitrogen incorporation in the primordial cell chemistry allowed synthesis of amino acids and nucleic acids, a prelude to RNA and subsequently DNA. Metagenomics provides access to nucleoprotein sediments synthesised by a googol of metabolically differentiated cells that have marked the evolution of life. Replication of a virus, a nucleoprotein particle, occurs passively in competent cells. Viruses are only identified in the context of the epidemic that they induce as a result of transmission from one host to another. By breaking down the viral particle, the host cell appears to resurrect the metabolic function of the nucleic acid, which synthesises its components without any form of control. Viral products undergo self-assembly and are exported by either exocytosis or cytolysis. In the absence of cells, viruses appear to be inert. However, intracellular contamination of a virus does not always result in replication: the viral genome can disappear, remain latent, wake up, remain embedded in the cellular genome, become an oncogene or induce auto-immunity. The presence of endogenous retroviruses in eukaryotic cells raises the question of their possible role in evolution.</p> </div> <div class="kwd-group"> <div class="title" tagx="title" title="title">
Keywords</div> <div class="kwd" title="kwd">
COVID-19</div> <div class="kwd" title="kwd">
Protists</div> <div class="kwd" title="kwd">
Prokaryotes</div> <div class="kwd" title="kwd">
Cell</div> <div class="kwd" title="kwd">
Infection</div> </div> </div> </div> <div class="body" title="body"> <div class="introduction" title="sec"><span class="label" tagx="label" title="label">1</span><div class="title" tagx="title" title="title">
Introduction</div> <p id="par0005">The current COVID-19 pandemic, characterised by acute inflammatory damage to the human respiratory tract (mild to severe and life-threatening), is caused by a so-called viral particle of matter called SARS-CoV-2, which is transmitted from one individual to another by droplets of saliva or respiratory mucus projected into the air physiologically by breathing, speaking or sneezing or coughing. These virus-laden droplets are deposited on the objects that we touch and can also be transmitted by the hands to the mucous membranes of the face. But, what is a virus? And, what is an epidemic?</p> <p id="par0010">A virus is conventionally defined as an ultramicroscopic particle of organic matter that can only multiply with the help of a cell. Viruses that multiply inside bacteria (non-nucleated cells or prokaryotes) are called bacteriophages. Viruses that multiply inside animal or plant cells (nucleated cells or eukaryotes) are considered to be pathogenic when they cause disease. However, scientists still do not know whether viruses are living things, because if the cell (nucleated or non-nucleated) did not exist, viruses would only be inert matter.</p> <p id="par0015">According to the metabolic theory of evolution, primordial cells arose about 4 billion years ago by sequestration inside lipid microfilms, isolating a microscopic <i>milieu intérieur</i> of the ocean environment within closed vesicles. Trapped inside a membrane phospholipid bilayer, chromophore molecules capable of capturing the energy of solar photons provide the energy for intracellular chemical reactions. The first cells were born, each characterised by its own internal metabolism that depends on transmembrane exchanges. The elementary metabolism of all living cells (glycolysis, citric acid cycle, etc.) comprises only four basic atoms: carbon, hydrogen, oxygen and phosphorus. Incorporation of the nitrogen atom in the course of evolution allowed the synthesis of amino acids, as well as nucleic acids. The physicochemical affinity between proteins and the purine or pyrimidine bases of nucleic acids probably led to an ancient RNA world before the development of the DNA world, as we know it <a href="#bib0070">[1]</a>. Prokaryotic cells preceded eukaryotic cells in the evolution of life. The cell's capacity to multiply on its own by using solar energy to produce organic molecules (or the geochemical energy of the centre of the Earth for certain prokaryotes) helps to define living organisms. Cells are able to reproduce identical copies of them or can differentiate into other types of cells, constituting the origin of all forms, varieties or species of living organisms. The first description of a plant cell is attributed to Hooke, who, in 1635, discovered the structure of cork by using the microscope invented by van Leeuwenhoek several years earlier. Schwann's publication of microscopic investigations on the similarity of structure and growth of animals and plants in 1839 marked the birth of cell theory. Remak in 1855 and Virchow in 1858 subsequently showed that all cells are derived from pre-existing cells by division of their nucleus. French histologists, who rejected the cell theory for a long time, only admitted, at the beginning of the 20th century, that the cell was both the morphological and physiological unit of every living organism (<a href="http://www.universalis.fr/encyclopedie/theorie-cellulaire">http://www.universalis.fr/encyclopedie/theorie-cellulaire</a><i>).</i></p> <p id="par0020">Despite their organic nature, viral particles therefore do not seem to be the origin of life, but, on the contrary, appear to be a product of living organisms. How can we explain this very special place of viruses in nature, somewhere between inanimate objects and living organisms? How can these organic particles only become pathogenic by means of epidemics transmitted by living organisms? The data concerning the structure and passive replication of viral particles reported here are unanimously accepted, but often presented as if it were certain that viral particles were living organisms. The description of viruses, detached from this often implicit hypothesis, corresponds to the debate on the nature of viruses initiated in 1957 by André Lwoff <a href="#bib0075">[2]</a>, Nobel Prize in Medicine in 1965.</p> </div> <div class="structureofviralparticles" title="sec"><span class="label" tagx="label" title="label">2</span><div class="title" tagx="title" title="title">
Structure of viral particles</div> <p id="par0025">Viruses, like living cells, associate nucleic acids with proteins. However, "when suspended in culture medium, viruses cannot metabolise, produce or use energy, grow, or replicate, all functions that are common to living organisms" (François Jacob) <a href="#bib0080">[3]</a>. Unlike living cells, viruses do not contain the two types of nucleic acids, and are either RNA viruses or DNA viruses. For greater simplicity and ease of understanding, this paper will not discuss subviral particles, such as viroids or satellite viruses (which are entities comprising at least a nucleic acid, but which are unable to replicate without the assistance of another virus), or giruses (contraction of giant viruses) such as mimiviruses (contraction of microbe mimicking viruses) (which are viral entities that can be infected by other viruses called virophages). However, the following approach, considering usual viruses as a model, allows the integration of both subviral particles and giruses.</p> <p id="par0030">This distinguishes between double-stranded DNA viruses, single-stranded DNA viruses, positive-sense double-stranded RNA viruses, negative-sense single-stranded RNA viruses (the genome of RNA viruses can be coded in two different directions: either in the 5’–3’ direction, like messenger RNA, corresponding to positive-sense, or in the reverse direction corresponding to negative-sense) and retroviruses <a href="#bib0085">[4]</a>. One of the smallest known viruses measures 10 nanometres and its genome contains only one gene: hepatitis delta virus (which, moreover, is considered to be a satellite virus of the hepatitis B virus, as it can only replicate in a patient already infected with hepatitis B virus or in the presence of simultaneous contamination with hepatitis B virus and hepatitis D virus) <a href="#bib0090">[5]</a>. One of the largest known viruses, mimivirus, measures 400 nanometres and its genome contains 1200 genes, more than certain bacteria <a href="#bib0095">[6]</a>.</p> <p id="par0035">The nucleic acid of a virus is usually stabilised on a skeleton of basic proteins (like histones in the nucleus of eukaryotic cells), which are all surrounded by a protein shell called a capsid, which gives the viral particle its characteristic morphology. The genome and capsid complex is called the nucleocapsid and forms a naked virus. Adenoviruses, discovered in 1953 from adenoid tissues (hence their name), are naked viruses. Their genome consists of 33 to 45 kilobases of linear, non-segmented, double-stranded DNA, subdivided into about one hundred transcription units, including regions encoding early proteins and late proteins. The icosahedral capsid (20 surfaces and 12 peaks) is a protein, consisting of 252 subunits called capsomers: 240 hexons carrying several genera and types of antigens and 12 pentons located at the spikes, carrying antigens common to all members of the same family. Haemagglutinin fibres (glycoproteins) of variable lengths extend beyond pentons, allowing attachment of certain adenoviruses to receptors present on the membrane of their target cells. These receptors are mostly the same as those used by Coxsackieviruses (which are enteroviruses) and are called CAR (Coxsackievirus and Adenovirus Receptor). CAR receptors belong to the immunoglobulin family. Adenoviruses are highly resistant in the outdoor environment, where they can survive for days or weeks on inert surfaces or in aqueous media. In addition to direct individual-to-individual transmission by droplets, indirect transmission by dirty hands, contaminated material, water or food is also very common. The portal of entry is nasopharyngeal (in which the main site of replication is the respiratory epithelium), conjunctival or gastrointestinal (for enteric strains with faeco-oral transmission). Human contamination by adenoviruses is thought to be asymptomatic in more than one-half of cases. However, adenoviruses have a pathogenic affinity for many tissues and, in immunocompetent subjects, are responsible for respiratory tract (sore throat, rhinopharyngitis, pneumonia), ophthalmological (conjunctivitis, keratoconjunctivitis) and gastrointestinal (febrile diarrhoea) infections.</p> <p id="par0040">The nucleocapsid of some viruses may be surrounded by an envelope. These viral particles belong to the group of enveloped viruses. The most remarkable feature is that this envelope is a lipid membrane derived from the host cell membrane system, but it is encrusted with viral proteins. Enveloped viruses leave the host cell by budding, which accounts for the lipid composition of their envelope. The genome and capsid of an enveloped virus, such as the influenza virus, self-assembles into nucleocapsids underneath the cytoplasmic membrane before being externalised by the formation of buds at the expense of the host cell cytoplasmic membrane, which consequently supplies the viral envelope. <i>Herpesviridae</i> assemble the genome and capsid in the host cell nucleus and are enveloped by budding of the nuclear membrane and then exported outside of the cell, which would explain certain differences in the lipid composition of viral envelopes (in contrast, release of naked viruses is secondary to rupture of the host cell membrane). Viral proteins may be inserted into the lipid bilayer immediately after their synthesis in ribosomes of the rough endoplasmic reticulum or after being transported via the Golgi apparatus. The original intrinsic proteins inserted into the host's lipid membrane are then expelled and replaced by viral proteins. In certain viruses, such as the hepatitis B virus, the fraction of viral proteins inserted into the lipid envelope is so abundant that a barrier of proteins tightly pressed against each other may be formed on the surface, totally masking the lipid membrane, which makes the virus highly resistant to external influences and detergents. The higher the proportion of lipids in the viral envelope (e.g. SARS-CoV-2), the easier it is to destroy the virus by hydroalcoholic solution or soap. In reality, the viral envelope makes the virus more fragile by making it less resistant to desiccation or attack by enzymes of the gastrointestinal tract. On the other hand, the presence of even small quantities of lipid bilayer fragments, by merging with the host cell cytoplasmic membrane, allows passive intracellular penetration of the viral particle.</p> <p id="par0045">We can therefore see signs of a possible evolution of the structure of viral particles, from a simple nucleic acid to an enveloped viral particle, via the naked viral particle. Viral particles actually appear to be completely passive in the phenomenon of their own replication.</p> </div> <div class="passivereplicationofviralparticles" title="sec"><span class="label" tagx="label" title="label">3</span><div class="title" tagx="title" title="title">
Passive replication of viral particles</div> <p id="par0050">"Viruses are viruses" (André Lwoff). The process of multiplication of virus particles inside cells is called replication: the viral particle is initially broken down, then reconstructed (with possible errors) by assembling its various components that the cell manufactures in multiple copies. A viral particle never descends directly from another viral particle.</p> <p id="par0055">Attachment of the viral particle to the cell membrane is passive, and results from the physicochemical forces that bind molecules together (adsorption) or a cell membrane receptor (protein or glycoprotein) involved in recognition of the external environment. Some viruses can infect many different cell types, while others target host cells defined by a specific receptor, such as the HIV virus that binds to the human CD4 molecule. Penetration of the viral particle can be either a passive or active process. Enveloped viruses often enter cells passively by chemical fusion of their lipid envelope with the host cell lipid membrane, releasing the capsid into the cytoplasm. Enveloped viruses, but with a protein barrier, or naked viruses enter the cell by endocytosis: it is therefore the cell and not the virus that provides the work necessary for internalisation of the capsid, by providing the capsid with a lipid bilayer envelope allowing it to merge with the cell membrane metabolic machinery.</p> <p id="par0060">Uncoating often plays a central role in the mechanisms of penetration and intracellular transport of the nucleocapsid. This energy-intensive step is ensured by host cell uncoating enzymes (only a few rare viruses, such as poxviruses, carry an uncoating enzyme in their genome). The released viral genome is then in direct contact with the cytoplasmic metabolic machinery. Positive-sense viral RNA behaves like messenger RNA to produce capsid proteins and to replicate (however, replication of viral mRNA is a complex process that will not be discussed here). Negative-sense viral RNA is transcribed into a single strand of complementary RNA, which is then translated by ribosomes, but only when the viral genome contains a transcriptase (which is an RNA-dependent RNA polymerase). Double-stranded RNA viruses must also carry a transcriptase in order to be translated metabolically. Retrovirus RNA is copied into double-stranded complementary DNA by an RNA-dependent DNA polymerase DNA (or reverse transcriptase) that must be present in the viral genome; the resulting DNA is then inserted into the DNA of the cell nucleus by means of an integrase, often associated with reverse transcriptase in the viral genome; the retrovirus genome then becomes an integral part of the cell genome and behaves as such. The genome of DNA viruses is transcribed into messenger RNA by RNA-dependent RNA polymerases that, depending on the case, are either host cell enzymes or enzymes included in the viral genome.</p> <p id="par0065">The natural physicochemical affinity between the various components of the nucleocapsid allows the self-assembly of countless new viral particles. However, self-assembly can be modulated by cellular enzymes that can alter the structure of viral proteins by modifying chemical groups, for example, by adding a sugar.</p> <p id="par0070">Naked viruses are released by rupture of the infected cell, while enveloped viruses are released by budding from one of the membrane compartments of the cell. The viral particles released therefore exponentially contaminate the other healthy cells. The immune system usually keeps a memory of the expelled viral particle after replication, which can then be eliminated before it enters the cell in the event of subsequent mucosal contamination.</p> <p id="par0075">However, intracellular contamination by a viral particle does not always lead to viral replication. The intracellular viral genome may disappear after a variable interval or may remain latent in the cell. It can then sometimes wake up, like the herpes virus. In the case of oncogenic viruses, certain viral proteins or viral genome insertions into the chromosomes of the host cell appear to be able to suppress the normal mechanisms of control of cell division. In other cases, the latent virus alters the membrane proteins of the host cell, which is no longer recognised by the immune system, resulting in autoimmune diseases. More surprisingly, retroviruses have been found in the genomes of all eukaryotic cells studied and have been called "endogenous retroviruses"; human endogenous retroviruses have been demonstrated by sequencing of the human genome, which would contain about half a million of these viruses <a href="#bib0100">[7]</a>, <a href="#bib0105">[8]</a>.</p> <p id="par0080">Viral replication therefore appears to be a passive viral phenomenon different from the process of reproduction that characterises living organisms, as it is the host cell that is the agent of viral replication. Moreover, not all viral particles necessarily behave like infectious particles. So, what is a viral infection?</p> </div> <div class="responsetotheconceptofvirusandviralinfection" title="sec"><span class="label" tagx="label" title="label">4</span><div class="title" tagx="title" title="title">
Response to the concept of virus and viral infection</div> <p id="par0085">An infection can be defined by "penetration into a living organism of a generally pathogenic foreign entity that is able to reproduce itself within this organism". A brief review of history can provide a better understanding of this current definition proposed by the <i>Centre National de Ressources Textuelles et Lexicales</i> [French centre for textual and lexical resources].</p> <p id="par0090">Communicable diseases have been known to man since ancient times <a href="#bib0075">[2]</a>. The Chinese identified smallpox 2500 years ago and knew that it was transmitted between humans, without, of course, attributing the disease to a virus. Sexual transmission of syphilis had been known since the 15th century, but <i>Treponema pallidum</i>, the bacterial species responsible for syphilis, was not identified until 1905. The work by Pasteur (1822–1895) established the microbial origin of infectious diseases. All microbes, whether bacteria, protists or fungi, were called "viruses" at the time. Pasteur and Roux easily demonstrated, even without identifying the causative agent, that rabies was a specific infectious disease probably caused by a microbe too small to be seen. Iwanowsky subsequently discovered that the juice of tobacco plants affected by tobacco mosaic disease remained contagious even after ultrafiltration. Many invisible but ultrafiltrable infectious agents were therefore suspected and were called viruses. At the same time, the term "microbe" was reserved for infectious agents that could be observed visually. Almost all viruses can only be visualised by electron microscopy <a href="#bib0110">[9]</a>, with the exception of giruses that are visible under the light microscope <a href="#bib0095">[6]</a>.</p> <p id="par0095">André Lwoff highlighted the ambivalence of the concepts of viruses and viral infection as early as 1957: are viruses living organisms, because they do not reproduce but replicate at the expense of the cell (like parasites)? What is the origin of viruses, since it is established that living organisms always reproduce from living organisms? <a href="#bib0075">[2]</a>.</p> <p id="par0100">The only paradox of Lwoff's formal and masterful demonstration of the "concept of virus" would appear to be the use of the term "parasite". By definition, a parasite is a living organism that lives at the expense of another living organism.</p> <p id="par0105">However, the simple facts detached from any underlying hypothesis allow a different interpretation from that of a parasite. The hypothesis of a deficiency of the host cell's intrinsic defences towards an organic foreign body can be proposed. Endocytosis is only a particular form of phagocytosis, which, in single-celled eukaryotes such as amoeba, plays a nutritional role, allowing the capture and ingestion of bacteria. Phagocytosis is an essential mechanism of immunity in multicellular animals, in which it is ensured by specialised cells, such as macrophages, dendritic cells, glial cells, neutrophils etc. Take, for example, contamination of the respiratory epithelium of the nasal cavities by naked adenoviruses present in a spray of respiratory mucus, carried by mucociliary transport allowing viral particles to attach to the cytoplasmic membrane of epithelial cells simply by selective physicochemical forces. One or several viral particles, for a variety of reasons, may not be detected sufficiently early by the mucosa-associated lymphoid tissue (MALT) to be eliminated. Persistence of this inert foreign body on the epithelium may lead to phagocytosis of the viral particle by the epithelial cell. However, the digestive capacities of the lysosomes of epithelial cells that have become specialised in mucociliary clearance (which, to my knowledge, have not been specifically studied) do not allow or no longer allow degradation of adenoviral DNA. Adenoviral DNA can be found in the cell nucleus, although no mechanisms of intranuclear transport of adenoviruses have been demonstrated. Several viral particles may also be phagocytosed by the same epithelial cell and viral DNA then persists in the cytoplasm. A single intranuclear molecule of viral DNA may actually be sufficient to trigger the production of messenger RNA, leading to the synthesis of both capsid proteins and viral DNA in the cytoplasm, where, as we have seen, these two constituents assemble naturally by means of physicochemical affinity. Intracellular accumulation of nucleocapsids leads to cell lysis in the case of adenoviruses, which are naked viruses, and exponential contamination of new epithelial cells. At the same time, immune cells develop specific immunity to this viral particle.</p> <p id="par0110">In the case of enveloped viruses, such as SARS-CoV-2, replacement of the intrinsic proteins of the cell membrane by viral proteins, resulting in the viral envelope structure, may be simply due to the abundance of unregulated production of these proteins (in contrast with physiological protein synthesis, which is regulated) and their insertion in appropriate proportions would be facilitated by their physicochemical affinity for the proteins of the previously self-assembled nucleocapsid.</p> <p id="par0115">In both cases, intracellular penetration of inert particles consisting of chains of potentially functional nucleic acid units, i.e. readable, transcriptable and translatable, results in uncontrolled functioning of the cell's metabolism. The main difficulty in relation to antiviral treatments actually concerns their necessary interference with the host cell metabolism and therefore a risk of cytotoxicity.</p> <p id="par0120">How can organic particles corresponding to potentially functional chains of nucleic acids, either comprising a single unit (a single gene, as in the case of the satellite hepatitis D virus) or hundreds or thousands of functional units assembled around basic, histone-like proteins and associated with capsid-forming proteins, end up in the air, water, soil and on objects? We have seen that intracellular penetration of viral particles may be necessary and sufficient to recreate the conditions of their activity.</p> <p id="par0125">Life on Earth is not limited to plants, animals, fungi and the small number of identified protists. Protists that do not have any of the characteristics of plants, animals or fungi live invisibly in the soil, water, air and virtually all terrestrial objects (such as dishwasher gaskets or roller shutters, etc.) and form the dark matter of life, probably more than 75% of the biomass, the role of which in the large cycles of matter (carbon cycle, nitrogen cycle, etc.) and the balance of various ecosystems is only beginning to be understood <a href="#bib0115">[10]</a>.</p> <p id="par0130">Metagenomics, i.e. analysis of all nucleic acids present in a given environment without involving a culture phase, was inaugurated in 1987 by Pace et al. <a href="#bib0120">[11]</a>, who proposed to directly clone the <i>5S</i> and <i>16S</i> ribosomal RNA genes from samples taken from the environment (soil, water, etc.). This new molecular approach to the infinitely small field of the invisible, even by electron microscopy, tends to suggest that human knowledge about life would represent only 1% of all micro-organisms, and these microbes would comprise a large number of viruses, including many human commensal viruses found in faeces ("faecal viroma"), on the skin or in the lungs. <a href="#bib0125">[12]</a>. However, it is difficult to identify unknown viruses by means of this method because viruses do not appear to have any universally conserved sequences and because we also need to be able to assemble long "contigs" (blocks of continuous nucleic acid sequences, whose overlapping terminals allow them to be assembled) within complex biological samples.</p> <p id="par0135">Another important finding is that identification of pathogenic viruses by means of this method is much easier in the context of acute diseases than in chronic diseases. To date, this method has only been able to identify the human oncogenic polyomavirus, the Merckel virus <a href="#bib0130">[13]</a>. This virus appears to be present on the surface of the skin in 70% of individuals, but only one to three people in a million develop a malignant Merckel tumour each year, often in the context of immunodepression. In contrast, SARS-CoV-2 was rapidly identified following the onset of the COVID-19 epidemic, as viruses appear to be identified only by the infectious diseases that they cause. These diseases are always contagious.</p> <p id="par0140">Epidemics therefore appear to be an essential phenomenon for viral replication. As soon as the chain of transmission is interrupted and the virus is no longer transported from one living organism to another, in which it can find the cells essential for its replication, the epidemic dies out and the virus disappears.</p> <p id="par0145">Thus, in the metabolic theory of evolution and the functional logic of living systems, viruses may correspond to nucleic acid particles that have been synthesised during the course of life by countless species, varieties and forms of cells that have developed on Earth since the nitrogen atom was incorporated into the protocellular metabolism, leading to the synthesis of amino acids and nucleic acids, and have become partners for life. But, life ends in death, and the origin of viruses, these fossils of coding sequences with no cytoplasmic metabolism to activate them, can be found in the sediments of past life at the bottom of the seas, in soils and probably the atmosphere and which, when by chance they meet a competent cell, divert the host cell's metabolism for a brief period of resurrection.</p> </div> <div class="conclusion" title="sec"><span class="label" tagx="label" title="label">5</span><div class="title" tagx="title" title="title">
Conclusion</div> <p id="par0150">Viruses, probably fossilised inert particles composed of nucleic acids and proteins, appear to be the physical support for temporarily resurrected biological functions by a competent cell as a result of a fortuitous physicochemical attachment. An epidemic is the result of transmission of these particles from one host to another and appears to be the clinical manifestation of their passive replication. The end of an epidemic, with disappearance of the virus, may entirely depend on the rapidity of complete interruption of the chain of transmission between hosts.</p> </div> <div class="disclosureofinterest" title="sec"> <div class="title" tagx="title" title="title">
Disclosure of interest</div> <p id="par0155">The author declares that he has no competing interest.</p> </div> </div> <div class="back" title="back"> <div class="references">
References</div> <div tag="ref-list"> <ul> <div class="title" tagx="title" title="title">
References</div> <li tag="ref"><a name="bib0070" /><span class="label" tagx="label" title="label">1</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Morowitz</span><span class="given-names" tagx="given-names" title="given-names">H.</span></span></span><span class="chapter-title" tagx="chapter-title" title="chapter-title">Beginnings of cellular life: metabolism recapitulates biogenesis</span><span class="year" tagx="year" title="year">1992</span><span class="publisher-name" tagx="publisher-name" title="publisher-name">Yale University Press</span><span class="publisher-loc" tagx="publisher-loc" title="publisher-loc">New Haven London</span></span></li> <li tag="ref"><a name="bib0075" /><span class="label" tagx="label" title="label">2</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lwoff</span><span class="given-names" tagx="given-names" title="given-names">A.</span></span></span><span class="mixed-article-title" title="mixed-article-title">The concept of virus</span><span class="source" tagx="source" title="source">Microbiology</span><span class="volume" tagx="volume" title="volume">17</span><span class="year" tagx="year" title="year">1957</span><span class="fpage" tagx="fpage" title="fpage">239</span><span class="lpage" tagx="lpage" title="lpage">253</span></span></li> <li tag="ref"><a name="bib0080" /><span class="label" tagx="label" title="label">3</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Jacob</span><span class="given-names" tagx="given-names" title="given-names">F.</span></span></span><span class="chapter-title" tagx="chapter-title" title="chapter-title">Qu’est-ce que la vie ?</span><span class="source" tagx="source" title="source">Université de tous les savoirs – La vie</span><span class="year" tagx="year" title="year">2002</span><span class="publisher-name" tagx="publisher-name" title="publisher-name">Odile Jacob</span></span></li> <li tag="ref"><a name="bib0085" /><span class="label" tagx="label" title="label">4</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Baltimore</span><span class="given-names" tagx="given-names" title="given-names">D.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Viruses, polymerases, and cancer (Nobel lecture)</span><span class="source" tagx="source" title="source">Science</span><span class="volume" tagx="volume" title="volume">192</span><span class="year" tagx="year" title="year">1976</span><span class="fpage" tagx="fpage" title="fpage">632</span><span class="lpage" tagx="lpage" title="lpage">636</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/57644">57644</a></span></span></li> <li tag="ref"><a name="bib0090" /><span class="label" tagx="label" title="label">5</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Rizzetto</span><span class="given-names" tagx="given-names" title="given-names">M.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Hepatitis D: thirty years after</span><span class="source" tagx="source" title="source">J Hepatol</span><span class="volume" tagx="volume" title="volume">50</span><span class="year" tagx="year" title="year">2009</span><span class="fpage" tagx="fpage" title="fpage">1043</span><span class="lpage" tagx="lpage" title="lpage">1050</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19285743">19285743</a></span></span></li> <li tag="ref"><a name="bib0095" /><span class="label" tagx="label" title="label">6</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">La Scola</span><span class="given-names" tagx="given-names" title="given-names">B.</span></span></span><span class="mixed-article-title" title="mixed-article-title">A giant virus in <i>amoebae</i></span><span class="source" tagx="source" title="source">Science</span><span class="volume" tagx="volume" title="volume">299</span><span class="year" tagx="year" title="year">2003</span><span class="fpage" tagx="fpage" title="fpage">2033</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12663918">12663918</a></span></span></li> <li tag="ref"><a name="bib0100" /><span class="label" tagx="label" title="label">7</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lander</span><span class="given-names" tagx="given-names" title="given-names">E.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Initial sequencing and analysis of the human genome</span><span class="source" tagx="source" title="source">Nature</span><span class="volume" tagx="volume" title="volume">409</span><span class="year" tagx="year" title="year">2001</span><span class="fpage" tagx="fpage" title="fpage">820</span><span class="lpage" tagx="lpage" title="lpage">921</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11236995">11236995</a></span></span></li> <li tag="ref"><a name="bib0105" /><span class="label" tagx="label" title="label">8</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">De Parseval</span><span class="given-names" tagx="given-names" title="given-names">N.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Heidmann</span><span class="given-names" tagx="given-names" title="given-names">T.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Human endogenous retroviruses: from infectious elements to human genes</span><span class="source" tagx="source" title="source">Cytogenet Genome Res</span><span class="volume" tagx="volume" title="volume">110</span><span class="year" tagx="year" title="year">2005</span><span class="fpage" tagx="fpage" title="fpage">318</span><span class="lpage" tagx="lpage" title="lpage">332</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16093684">16093684</a></span></span></li> <li tag="ref"><a name="bib0110" /><span class="label" tagx="label" title="label">9</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Roingeard</span><span class="given-names" tagx="given-names" title="given-names">P.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Détection des virus par microscopie électronique : passé, présent et futur</span><span class="source" tagx="source" title="source">Virologie</span><span class="volume" tagx="volume" title="volume">13</span><span class="year" tagx="year" title="year">2009</span><span class="fpage" tagx="fpage" title="fpage">249</span><span class="lpage" tagx="lpage" title="lpage">258</span></span></li> <li tag="ref"><a name="bib0115" /><span class="label" tagx="label" title="label">10</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Silar</span><span class="given-names" tagx="given-names" title="given-names">P.</span></span></span><span class="chapter-title" tagx="chapter-title" title="chapter-title">Protistes eucaryotes : origine, evolution et biologie des microbes eucaryotes</span><span class="year" tagx="year" title="year">2016</span><span class="comment">[Vol. 978-2-9555841-0-1. hal-01263138]</span></span></li> <li tag="ref"><a name="bib0120" /><span class="label" tagx="label" title="label">11</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Pace</span><span class="given-names" tagx="given-names" title="given-names">N.</span></span></span><span class="chapter-title" tagx="chapter-title" title="chapter-title">The analysis of natural microbial populations by ribosomal RNA sequences</span><span class="source" tagx="source" title="source">Advances in microbial ecology</span><span class="year" tagx="year" title="year">1986</span><span class="publisher-name" tagx="publisher-name" title="publisher-name">Springer</span><span class="fpage" tagx="fpage" title="fpage">1</span><span class="lpage" tagx="lpage" title="lpage">55</span></span></li> <li tag="ref"><a name="bib0125" /><span class="label" tagx="label" title="label">12</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bernardo</span><span class="given-names" tagx="given-names" title="given-names">P.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Métagénomique virale et pathologie – une histoire récente</span><span class="source" tagx="source" title="source">Médecine/sciences</span><span class="volume" tagx="volume" title="volume">29</span><span class="year" tagx="year" title="year">2013</span><span class="fpage" tagx="fpage" title="fpage">501</span><span class="lpage" tagx="lpage" title="lpage">508</span></span></li> <li tag="ref"><a name="bib0130" /><span class="label" tagx="label" title="label">13</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Feng</span><span class="given-names" tagx="given-names" title="given-names">H.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Clonal integration of a polyomavirus in human Merkel cell carcinoma</span><span class="source" tagx="source" title="source">Science</span><span class="volume" tagx="volume" title="volume">319</span><span class="year" tagx="year" title="year">2008</span><span class="fpage" tagx="fpage" title="fpage">1096</span><span class="lpage" tagx="lpage" title="lpage">1100</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18202256">18202256</a></span></span></li> </ul> </div> </div>  </body></html>