<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<link rel="dns-prefetch" href="//d33xdlntwy0kbs.cloudfront.net" />
<link rel="shortcut icon" href="https://www.biorxiv.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<meta name="article_thumbnail" content="https://www.biorxiv.org/content/biorxiv/early/2019/05/18/641183/embed/graphic-1.gif" />
<meta name="type" content="article" />
<meta name="category" content="article" />
<meta name="HW.identifier" content="/biorxiv/early/2019/05/18/641183.atom" />
<meta name="HW.pisa" content="biorxiv;641183v2" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="cRacle: R Tools for Estimating Climate from Vegetation" />
<meta name="DC.Identifier" content="10.1101/641183" />
<meta name="DC.Date" content="2019-05-18" />
<meta name="DC.Publisher" content="Cold Spring Harbor Laboratory" />
<meta name="DC.Rights" content="© 2019, Posted by Cold Spring Harbor Laboratory. This pre-print is available under a Creative Commons License (Attribution-NonCommercial 4.0 International), CC BY-NC 4.0, as described at http://creativecommons.org/licenses/by-nc/4.0/" />
<meta name="DC.AccessRights" content="restricted" />
<meta name="DC.Description" content="Premise of the study The Climate Reconstruction Analysis using Coexistence Likelihood Estimation (CRACLE) method for the estimation of climate from vegetation is a robust set of modeling tools for estimating climate and paleoclimate that makes use of large repositories of biodiversity data and open-access R software.

Methods Here we implement a new R library for the estimation of climate from vegetation.The ‘cRacle’ library implements functions for data access, aggregation, and models to estimate climate from plant community composition. ‘cRacle’ is modular and features many best-practice features.

Results Performance tests using modern vegetation survey data from North and South America shows that CRACLE outperforms alternative methods. CRACLE estimates of mean annual temperature (MAT) are usually within 1°C of the actual when optimal model parameters are used. Generalized Boosted Regression (GBR) model correction is also shown here to improve on CRACLE models by reducing bias.

Discussion CRACLE provides accurate estimates of climate from modern plant communities. Non-parametric CRACLE modeling coupled to GBR model correction produces the most accurate results to date. The ‘cRacle’ R library streamlines the estimation of climate from plant community data, and will make this modeling more accessible to a wider range of users." />
<meta name="DC.Contributor" content="Robert S. Harbert" />
<meta name="DC.Contributor" content="Alex A. Baryiames" />
<meta name="article:published_time" content="2019-05-18" />
<meta name="article:section" content="New Results" />
<meta name="citation_title" content="cRacle: R Tools for Estimating Climate from Vegetation" />
<meta name="citation_abstract" lang="en" content="&lt;h3&gt;ABSTRACT&lt;/h3&gt;
&lt;h3&gt;Premise of the study&lt;/h3&gt;
&lt;p&gt;The Climate Reconstruction Analysis using Coexistence Likelihood Estimation (CRACLE) method for the estimation of climate from vegetation is a robust set of modeling tools for estimating climate and paleoclimate that makes use of large repositories of biodiversity data and open-access R software.&lt;/p&gt;&lt;h3&gt;Methods&lt;/h3&gt;
&lt;p&gt;Here we implement a new R library for the estimation of climate from vegetation.The ‘cRacle’ library implements functions for data access, aggregation, and models to estimate climate from plant community composition. ‘cRacle’ is modular and features many best-practice features.&lt;/p&gt;&lt;h3&gt;Results&lt;/h3&gt;
&lt;p&gt;Performance tests using modern vegetation survey data from North and South America shows that CRACLE outperforms alternative methods. CRACLE estimates of mean annual temperature (MAT) are usually within 1°C of the actual when optimal model parameters are used. Generalized Boosted Regression (GBR) model correction is also shown here to improve on CRACLE models by reducing bias.&lt;/p&gt;&lt;h3&gt;Discussion&lt;/h3&gt;
&lt;p&gt;CRACLE provides accurate estimates of climate from modern plant communities. Non-parametric CRACLE modeling coupled to GBR model correction produces the most accurate results to date. The ‘cRacle’ R library streamlines the estimation of climate from plant community data, and will make this modeling more accessible to a wider range of users.&lt;/p&gt;" />
<meta name="citation_journal_title" content="bioRxiv" />
<meta name="citation_publisher" content="Cold Spring Harbor Laboratory" />
<meta name="citation_publication_date" content="2019/01/01" />
<meta name="citation_mjid" content="biorxiv;641183v2" />
<meta name="citation_id" content="641183v2" />
<meta name="citation_public_url" content="https://www.biorxiv.org/content/10.1101/641183v2" />
<meta name="citation_abstract_html_url" content="https://www.biorxiv.org/content/10.1101/641183v2.abstract" />
<meta name="citation_full_html_url" content="https://www.biorxiv.org/content/10.1101/641183v2.full" />
<meta name="citation_pdf_url" content="https://www.biorxiv.org/content/biorxiv/early/2019/05/18/641183.full.pdf" />
<meta name="citation_doi" content="10.1101/641183" />
<meta name="citation_num_pages" content="21" />
<meta name="citation_article_type" content="Article" />
<meta name="citation_section" content="New Results" />
<meta name="citation_firstpage" content="641183" />
<meta name="citation_author" content="Robert S. Harbert" />
<meta name="citation_author_institution" content="Department of Biology, Stonehill College" />
<meta name="citation_author_institution" content="American Museum of Natural History" />
<meta name="citation_author_email" content="rharbert@stonehill.edu" />
<meta name="citation_author" content="Alex A. Baryiames" />
<meta name="citation_author_institution" content="Department of Biology, Stonehill College" />
<meta name="citation_reference" content="Aiello□Lammens, M.E., R.A. Boria, A. Radosavljevic, B. Vilela, and R.P. Anderson. 2015. spThin: an R package for spatial thinning of species occurrence records for use in ecological niche models. Ecography, 38(5): 541–545." />
<meta name="citation_reference" content="Barve, V., and E. Hart. 2017. ‘rinat’: Access iNaturalist Data Through APIs. R package version 0.1.5. https://cran.r-project.org/package=rinat" />
<meta name="citation_reference" content="Fick, S.E. and R.J. Hijmans. 2017. WorldClim 2: new 1□km spatial resolution climate surfaces for global land areas. International journal of climatology 37(12):4302–4315." />
<meta name="citation_reference" content="Fletcher, T., R. Feng, A.M. Telka, J.V. Matthews Jr, and A. Ballantyne. 2017. Floral Dissimilarity and the Influence of Climate in the Pliocene High Arctic: Biotic and Abiotic Influences on Five Sites on the Canadian Arctic Archipelago. Frontiers in Ecology and Evolution 5:19." />
<meta name="citation_reference" content="Fletcher, T.L., A.Z. Csank, and A.P. Ballantyne. 2018. Identifying bias in cold season temperature reconstructions by beetle mutual climatic range methods in the Pliocene Canadian High Arctic. Palaeogeography, Palaeoclimatology, Palaeoecology 514: 672–676." />
<meta name="citation_reference" content="Greenwell, B., B. Boehmke, J. Cunningham, and GBM Developers. 2019. ‘gbm’: Generalized Boosted Regression Models. R package version 2.1.5 https://cran.r-project.org/package=gbm" />
<meta name="citation_reference" content="Harbert, R.S. and K.C. Nixon. 2015. Climate reconstruction analysis using coexistence likelihood estimation (CRACLE): A method for the estimation of climate using vegetation. American Journal of Botany 102(8): 1277–1289." />
<meta name="citation_reference" content="Harbert, R.S. and Nixon, K.C., 2018. Quantitative Late Quaternary Climate Reconstruction from Plant Macrofossil Communities in Western North America. Open Quaternary 4(1):8." />
<meta name="citation_reference" content="Harris, A.J., Papeş, M., Gao, Y.D. and Watson, L., 2014. Estimating paleoenvironments using ecological niche models of nearest living relatives: A case study of Eocene Aesculus L. Journal of Systematics and Evolution, 52(1): 16–34." />
<meta name="citation_reference" content="Hijmans, R.J., S.E. Cameron, J.L. Parra, P.G. Jones, A. Jarvis. 2005. Very high resolution interpolated climate surfaces for global land areas. International Journal of Climatology: A Journal of the Royal Meteorological Society 25(15):1965–1978." />
<meta name="citation_reference" content="Hijmans, R.J. (2018). raster: Geographic Data Analysis and Modeling. R package version 2.8-4. https://CRAN.R-project.org/package=raster" />
<meta name="citation_reference" content="Hyland, E.G., K.W. Huntington, N.D. Sheldon, and T. Reichgelt. 2018. Temperature seasonality in the North American continental interior during the Early Eocene Climatic Optimum. Climate of the Past 14(10): 1391–1404." />
<meta name="citation_reference" content="Karger, D.N., O. Conrad, J. Böhner, T. Kawohl, H. Kreft, R.W. Soria-Auza, N.E. Zimmermann, H.P. Linder, and M. Kessler. 2017. Climatologies at high resolution for the earth’s land surface areas. Scientific Data 4:170122." />
<meta name="citation_reference" content="Maitner, B.S., B. Boyle, N. Casler, R. Condit, J. Donoghue, S.M. Durán, D. Guaderrama, C.E. Hinchliff, P.M. Jørgensen, N.J. Kraft, and B. McGill. 2018. The bien r package: A tool to access the Botanical Information and Ecology Network (BIEN) database. Methods in Ecology and Evolution 9(2):373–379" />
<meta name="citation_reference" content="Montade, V., O. Peyron, C. Favier, J.P Francois, and S.G. Haberle. 2019. A pollen– climate calibration from western Patagonia for palaeoclimatic reconstructions. Journal of Quaternary Science 34(1): 76–86." />
<meta name="citation_reference" content="Mosbrugger, V. and T. Utescher. 1997. The coexistence approach—a method for quantitative reconstructions of Tertiary terrestrial palaeoclimate data using plant fossils. Palaeogeography, Palaeoclimatology, Palaeoecology 134(1-4):61–86." />
<meta name="citation_reference" content="R Core Team (2018). R: A language and environment for statistical computing. R Foundation for Statistical Computing, Vienna, Austria. URL https://www.R-project.org/." />
<meta name="citation_reference" content="Silverman, BW. 1986. Density estimation for statistics and data analysis. Routledge. DOI: https://doi.org/10.1201/9781315140919" />
<meta name="citation_reference" content="Thompson, R.S., K.H. Anderson, R.T. Pelltier, L.E. Strickland, P.J. Bartlein, and S.L. Shafer. 2012. Quantitative estimation of climatic parameters from vegetation data in North America by the mutual climatic range technique. Quaternary Science Reviews 51:18–39." />
<meta name="citation_reference" content="Title, P.O. and J.B. Bemmels. 2018. ENVIREM: an expanded set of bioclimatic and topographic variables increases flexibility and improves performance of ecological niche modeling. Ecography 41(2):291–307." />
<meta name="citation_reference" content="Utescher, T., A.A. Bruch, B. Erdei, L. François, D. Ivanov, F.M.B. Jacques, A.K. Kern, V. Mosbrugger, and R.A. Spicer. 2014. The Coexistence Approach—theoretical background and practical considerations of using plant fossils for climate quantification. Palaeogeography, Palaeoclimatology, Palaeoecology, 410: 58–73." />
<meta name="citation_reference" content="Willard, D.A., T.H. Donders, T. Reichgelt, D.R. Greenwood, F. Sangiorgi, F. Peterse, K.G. Nierop, j. Frieling, S. Schouten, and A. Sluijs. 2019. Arctic vegetation, temperature, and hydrology during Early Eocene transient global warming events. Global and Planetary Change 178:139–152." />
<meta name="citation_reference" content="Zaborac-Reed, S.J. and E.B. Leopold. 2016. Determining the paleoclimate and elevation of the late Eocene Florissant flora: support from the coexistence approach. Canadian Journal of Earth Sciences 53(6): 565–573." />
<meta name="twitter:title" content="cRacle: R Tools for Estimating Climate from Vegetation" />
<meta name="twitter:site" content="@biorxivpreprint" />
<meta name="twitter:card" content="summary" />
<meta name="twitter:image" content="https://www.biorxiv.org/sites/default/files/images/biorxiv_logo_homepage7-5-small.png" />
<meta name="twitter:description" content="Premise of the study The Climate Reconstruction Analysis using Coexistence Likelihood Estimation (CRACLE) method for the estimation of climate from vegetation is a robust set of modeling tools for estimating climate and paleoclimate that makes use of large repositories of biodiversity data and open-access R software.

Methods Here we implement a new R library for the estimation of climate from vegetation.The ‘cRacle’ library implements functions for data access, aggregation, and models to estimate climate from plant community composition. ‘cRacle’ is modular and features many best-practice features.

Results Performance tests using modern vegetation survey data from North and South America shows that CRACLE outperforms alternative methods. CRACLE estimates of mean annual temperature (MAT) are usually within 1°C of the actual when optimal model parameters are used. Generalized Boosted Regression (GBR) model correction is also shown here to improve on CRACLE models by reducing bias.

Discussion CRACLE provides accurate estimates of climate from modern plant communities. Non-parametric CRACLE modeling coupled to GBR model correction produces the most accurate results to date. The ‘cRacle’ R library streamlines the estimation of climate from plant community data, and will make this modeling more accessible to a wider range of users." />
<meta name="og-title" property="og:title" content="cRacle: R Tools for Estimating Climate from Vegetation" />
<meta name="og-url" property="og:url" content="https://www.biorxiv.org/content/10.1101/641183v2" />
<meta name="og-site-name" property="og:site_name" content="bioRxiv" />
<meta name="og-description" property="og:description" content="Premise of the study The Climate Reconstruction Analysis using Coexistence Likelihood Estimation (CRACLE) method for the estimation of climate from vegetation is a robust set of modeling tools for estimating climate and paleoclimate that makes use of large repositories of biodiversity data and open-access R software.

Methods Here we implement a new R library for the estimation of climate from vegetation.The ‘cRacle’ library implements functions for data access, aggregation, and models to estimate climate from plant community composition. ‘cRacle’ is modular and features many best-practice features.

Results Performance tests using modern vegetation survey data from North and South America shows that CRACLE outperforms alternative methods. CRACLE estimates of mean annual temperature (MAT) are usually within 1°C of the actual when optimal model parameters are used. Generalized Boosted Regression (GBR) model correction is also shown here to improve on CRACLE models by reducing bias.

Discussion CRACLE provides accurate estimates of climate from modern plant communities. Non-parametric CRACLE modeling coupled to GBR model correction produces the most accurate results to date. The ‘cRacle’ R library streamlines the estimation of climate from plant community data, and will make this modeling more accessible to a wider range of users." />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.biorxiv.org/sites/default/files/images/biorxiv_logo_homepage7-5-small.png" />
<meta name="citation_date" content="2019-05-18" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/10.1101/641183v2.full.pdf" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/10.1101/641183v2.full.txt" />
<link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/10.1101/641183v2.ppt" />
<meta name="description" content="bioRxiv - the preprint server for biology, operated by Cold Spring Harbor Laboratory, a research and educational institution" />
<meta name="generator" content="Drupal 7 (http://drupal.org)" />
<link rel="canonical" href="https://www.biorxiv.org/content/10.1101/641183v2" />
<link rel="shortlink" href="https://www.biorxiv.org/node/739126" />
    <title>cRacle: R Tools for Estimating Climate from Vegetation | bioRxiv</title>  
    <link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__jMRAK66KMC1e4TQlwUNn3KiWVDC5AjueUAYEm1xBY_U__KSUjdT4jdcJ4qJz6fKY1K9WkYh1a5EcaWZxt_-zbTis__QJacVmveFyLIBSPkp21xL0ZvuDW9DMBrD2HB1P4Ry5I.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q5p1rr" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__icrP-Y0qw2NP73rJxxkUtrJta0z3VLAABtNjfSdEghA__rlN3rx0wOoswGNHZMeZM96bNn-3cLWHXTztGDoThNlw__QJacVmveFyLIBSPkp21xL0ZvuDW9DMBrD2HB1P4Ry5I.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__q48SDxVDWut_cH6dTPyQe7HjADuSLLmJVFztuiJEFrU__DJqddQGjm5dXCHsaiwNriGytKN5eevIQqyjdq65TZtE__QJacVmveFyLIBSPkp21xL0ZvuDW9DMBrD2HB1P4Ry5I.css" media="all" />
<style type="text/css" media="all">
/* <![CDATA[ */
#sliding-popup.sliding-popup-bottom,#sliding-popup.sliding-popup-bottom .eu-cookie-withdraw-banner,.eu-cookie-withdraw-tab{background:gray}#sliding-popup.sliding-popup-bottom.eu-cookie-withdraw-wrapper{background:transparent}#sliding-popup .popup-content #popup-text h1,#sliding-popup .popup-content #popup-text h2,#sliding-popup .popup-content #popup-text h3,#sliding-popup .popup-content #popup-text p,.eu-cookie-compliance-secondary-button,.eu-cookie-withdraw-tab{color:#fff !important}.eu-cookie-withdraw-tab{border-color:#fff}.eu-cookie-compliance-more-button{color:#fff !important}
/* ]]> */
</style>

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__QJacVmveFyLIBSPkp21xL0ZvuDW9DMBrD2HB1P4Ry5I.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__B6tGdsQSt1vT8d8j-19-oS4FKJe83RrSNBkjPs9jfos__6VkE5WcQmmQpSVTQx-xS2-zn_nlwFlDm-EN1HLJNjrA__QJacVmveFyLIBSPkp21xL0ZvuDW9DMBrD2HB1P4Ry5I.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__I6pBwZrmO51afTZVOjN83FdGH1QRb9t15lSLso0JVV4__F3sVKlGyHKCaAPI_WdiHD7l_-05B6b9bLdUwzi1me90__QJacVmveFyLIBSPkp21xL0ZvuDW9DMBrD2HB1P4Ry5I.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__QJacVmveFyLIBSPkp21xL0ZvuDW9DMBrD2HB1P4Ry5I.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__QJacVmveFyLIBSPkp21xL0ZvuDW9DMBrD2HB1P4Ry5I.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__CRdjIL0HKDL2T9slYvkTSTLypBBPjyoDhwBezBuDy7c__cHebPmPNdg2jyPmpyQpZWssJQ4cPrqfaabp5Or9KlkU__QJacVmveFyLIBSPkp21xL0ZvuDW9DMBrD2HB1P4Ry5I.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://d33xdlntwy0kbs.cloudfront.net/cshl_custom.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/all/libraries/modernizr/modernizr.min.js?q5p1rr"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__yOhrJUBmMpN-jsuQlJYmS8ovkQU6I6KInr295HrSTb0__c--zHhbuOWmkdWkJYXiNWDmkpDRPesmbBjGlQcyqBOY__QJacVmveFyLIBSPkp21xL0ZvuDW9DMBrD2HB1P4Ry5I.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var _prum=[['id', '52e95df3abe53d2e33000000'], ['mark', 'firstbyte',
      (new Date()).getTime()]]; (function() {
      var s=document.getElementsByTagName('script')[0],
      p=document.createElement('script');
      p.async='async'; p.src='//rum-static.pingdom.net/prum.min.js';s.parentNode.insertBefore(p,s);})();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js___KyvuEvfmewWxdOLBXTYsSUo8Pfm_DijV6fWtGvSL_k__HSKkTgIr7nDcdvQYewipf6v4ak6ZLUOQkwPj44wKouA__QJacVmveFyLIBSPkp21xL0ZvuDW9DMBrD2HB1P4Ry5I.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__3uCzzYE5jr6ly9nvsM8PRBRkhh4p0NWCMDja4EKBy4M__dmGGVYcB8LhnbR2iAmVW0ZyXI6QuFVYFRORa5XxglXs__QJacVmveFyLIBSPkp21xL0ZvuDW9DMBrD2HB1P4Ry5I.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__0zdUME0GLauYYyEch2tGPydIw-IG7Uizip0Qb4wUsig__3RgszrCgk4bOnTzo4vaQ-hrWzaG6oD19l-R9CQ1YQ-o__QJacVmveFyLIBSPkp21xL0ZvuDW9DMBrD2HB1P4Ry5I.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__W6wAjopaALytcBam7oT3WhEzllkWaB9R-8hazHyvShM__rK5YBIqBy9qt0di0Sn4DriBx9TJupdmv6uGmLJyZvDs__QJacVmveFyLIBSPkp21xL0ZvuDW9DMBrD2HB1P4Ry5I.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-45638026-1", {"cookieDomain":"auto"});ga("set", "page", location.pathname + location.search + location.hash);ga("send", "pageview");ga('create', 'UA-189672-38', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__1PtAOZ8x7l6QAi8D7X6Kj_Y7HcUOFzvWRvvYHUcCT98__U8IB92u_CnnGwJ-X_xaUa9lzsC8OzUfN0WkK2JQfirg__QJacVmveFyLIBSPkp21xL0ZvuDW9DMBrD2HB1P4Ry5I.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings,{"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"F8kJ030Q_iubIm7ZqsGnHMXd66WbuW8sB4gMPvjYj1A","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"modules\/comment\/comment.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"modules\/forum\/forum.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"sites\/default\/modules\/biorxiv\/css\/biorxiv.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/flexible\/flexible.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_collection\/css\/highwire_subject_collection.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire.highwire-list-expand.css":1,"public:\/\/ctools\/css\/bab59090150caf0c64e27afa1b36eef1.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/contrib\/eu_cookie_compliance\/css\/eu_cookie_compliance.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"0":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_biorxiv_styles\/css\/biorxiv-styles.css.less":1,"https:\/\/d33xdlntwy0kbs.cloudfront.net\/cshl_custom.css":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_figures.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_tables.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"http:\/\/d33xdlntwy0kbs.cloudfront.net\/cshl_custom.js":1,"sites\/all\/modules\/contrib\/eu_cookie_compliance\/js\/eu_cookie_compliance.js":1,"sites\/default\/modules\/jnl_biorxiv_styles\/js\/adv-search-collection-toggle.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/eu_cookie_compliance\/js\/jquery.cookie-1.4.1.min.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/default\/modules\/jnl_biorxiv_styles\/js\/biorxiv-scripts.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_nps_survey\/js\/nps_survey.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"sites\/default\/modules\/contrib\/disqus\/disqus.js":1,"misc\/textarea.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/contrib\/service_links\/js\/twitter_button.js":1,"sites\/all\/modules\/contrib\/service_links\/js\/facebook_like.js":1,"sites\/all\/modules\/contrib\/service_links\/js\/google_plus_one.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.highwire-list-expand.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"739126","apath":"\/biorxiv\/early\/2019\/05\/18\/641183.atom","pisa":"biorxiv;641183v2","processed":["highwire_math"],"markup":[{"requested":"full-text","variant":"full-text","view":"full","pisa":"biorxiv;641183v2"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"disqus":{"domain":"biorxivstage","url":"https:\/\/www.biorxiv.org\/content\/10.1101\/641183v2","title":"cRacle: R Tools for Estimating Climate from Vegetation","identifier":"node\/739126"},"panels_ajax_pane":{"new-41":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022INYIrDQ0DdZryKNK1Ao9f3HBXpFz1quoO1Dt4cnEum\\\\\\\/T9XKxGC0Ba07V8+AZfzIPXHb7uCo0uodUAhuu2XLFyjRktwn8+yGrXIuswuJyf\\\\\\\/CNbm6lv8n1Llh91+ijMsz5UQz+wmcKVXdoAkNp9GIlft1W8BodjBlV0wyOZhn\\\\\\\/uQ45HJY2i\\\\\\\/6Dfn9R9auQv8MXoXhCBWci8Q1YO7t0+TiRxEXIaj6sAfM2sy5R7PGKIVcGBcq2K6F2\\\\\\\/cp7V85UYUVILzcRDz2LJAPSHdAurbxfAADXW7GuqkGbCVM46JqoGLryr2WuWgZzdjusw1ASg5rTObudDtRMXxRJpc3YTEjtUALm5uoesTjHzMbToRyO3l9OrPZnIR5CsZb8UcCjp+9Pl+jmY1a+ezQPCrorNETbbR+TLhYbv\\\\\\\/QAAQpVo6aR6NByW2hO0FAzKT9n4gtWeeokZcgIU\\\\\\\/+6ZLMJNUS6+kQCBmGN+OWA0sLJrWoN\\\\\\\/1Rm26E2Kd26utziIJT8SRxDxazAsz4HuVKZjLvmYRTGkhsdlNWM0Kau0eDCH+1IwC4D4r\\\\\\\/ArkKKB6Rct9D1l6Droc4NtdMF+U\\\\\\\/xNN0HhXE+fI7SDHAJIDZ2mzWug4qtzrfgW\\\\\\\/G3Px5z9Y0KdnzWntPwHAiAOKo5zgho\\\\\\\/\\\\\\\/4EZZWSYKk6WIuIple+JftY8UjT\\\\\\\/UVnpGO9IsZCRKe4fsPqGyJvBYbvw1\\\\\\\/ZDgiCx1q4\\\\\\\/q+hmWWLhO\\\\\\\/pK+aDh891hYRl8UeVMp8XGryfy8Kh64k7lqzImX7br3SvIpoR2WOsLXPLVoVG6lWl7vTyzuNH21SvxYh5AwaBgEfEBC7d0WBqKnpOfXOOCvQidnfTaUysHYuPlHNz0zBwEx45NbXLfASmJGHJxNfmMDAlNGmNPGtJ2+wac8BRFkjW9rcV01ggItX38E4U0ut3nMF+94iYHjkCxymSGfJk7HKGly5HbCtFfa2SBn\\\\\\\/llKFRmBbsGO+cEPjyAq9RhvUiBhNEPTHMEpiYiR\\\\\\\/uKao1Hyqv6HS2kVMLf2XDnXCLQLaTMyspntly7inhDLqaf1tWn6y0Q3LlStZwBceXwIahU57qsx03lM41kQSTFys2nWIi5h8rfHhST+flUixfpfI10Rvaw3YYHfpRKLLO7AXTb2FSfK1cAYWYzCemvLGjgcsbYVyYMZYZhyn01KoL0NxIOGsc8vjrBaa0u3ulF69jlcVoD\\\\\\\/8iWjZga11WIMZB6232LVKUmCH6zo8daR7xMb9eRmSrw5gNlurBmkHifL2+eBPvc3Kix6vYbuPYlOTqAmkIB1mA6G0u4gNc1bRavHApL5OYzCBqKxQuxjzZp5M=\\\u0022,\\\u0022iv\\\u0022:\\\u0022kOEfUQdpJeuxVjxDbqodow==\\\u0022,\\\u0022salt\\\u0022:\\\u0022f9d8845712f72c9fee5db7d9ff111eab\\\u0022}\u0022,\u0022hmac\u0022:\u00229b5b96509778761f23ddbdbdb3232537cf04dc56da2d88e07986ebffb02bf575\u0022}"},"urlIsAjaxTrusted":{"\/content\/10.1101\/641183v2.full":true},"ws_fl":{"width":100,"height":21},"ws_gpo":{"size":"","annotation":"","lang":"","callback":"","width":300},"color":{"logo":"https:\/\/www.biorxiv.org\/sites\/default\/files\/bioRxiv_article.jpg"},"highwire_list_expand":{"is_collapsed":"1"},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"eu_cookie_compliance":{"popup_enabled":1,"popup_agreed_enabled":0,"popup_hide_agreed":0,"popup_clicking_confirmation":1,"popup_scrolling_confirmation":false,"popup_html_info":"\u003Cdiv\u003E\n  \u003Cdiv class =\u0022popup-content info\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n      \u003Cp\u003EWe use cookies on this site to enhance your user experience. By clicking any link on this page you are giving your consent for us to set cookies.\u003C\/p\u003E\n    \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022agree-button eu-cookie-compliance-default-button\u0022\u003EContinue\u003C\/button\u003E\n                    \u003Cbutton type=\u0022button\u0022 class=\u0022find-more-button eu-cookie-compliance-more-button\u0022\u003EFind out more\u003C\/button\u003E\n          \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E","use_mobile_message":false,"mobile_popup_html_info":"\u003Cdiv\u003E\n  \u003Cdiv class =\u0022popup-content info\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n          \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022agree-button eu-cookie-compliance-default-button\u0022\u003EContinue\u003C\/button\u003E\n                    \u003Cbutton type=\u0022button\u0022 class=\u0022find-more-button eu-cookie-compliance-more-button\u0022\u003EFind out more\u003C\/button\u003E\n          \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E\n","mobile_breakpoint":"768","popup_html_agreed":"\u003Cdiv\u003E\n  \u003Cdiv class=\u0022popup-content agreed\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n      \u003Ch2\u003EThank you for accepting cookies\u003C\/h2\u003E\u003Cp\u003EYou can now hide this message or find out more about cookies.\u003C\/p\u003E    \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022hide-popup-button eu-cookie-compliance-hide-button\u0022\u003EHide\u003C\/button\u003E\n              \u003Cbutton type=\u0022button\u0022 class=\u0022find-more-button eu-cookie-compliance-more-button-thank-you\u0022 \u003EMore info\u003C\/button\u003E\n          \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E","popup_use_bare_css":false,"popup_height":"auto","popup_width":"100%","popup_delay":1000,"popup_link":"\/help\/cookie-policy","popup_link_new_window":1,"popup_position":null,"popup_language":"en","store_consent":false,"better_support_for_screen_readers":0,"reload_page":0,"domain":"","popup_eu_only_js":0,"cookie_lifetime":"365","cookie_session":false,"disagree_do_not_show_popup":0,"method":"default","whitelisted_cookies":"","withdraw_markup":"\u003Cbutton type=\u0022button\u0022 class=\u0022eu-cookie-withdraw-tab\u0022\u003EPrivacy settings\u003C\/button\u003E\n\u003Cdiv class=\u0022eu-cookie-withdraw-banner\u0022\u003E\n  \u003Cdiv class=\u0022popup-content info\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n      \u003Cp\u003E\u0026lt;h2\u0026gt;We use cookies on this site to enhance your user experience\u0026lt;\/h2\u0026gt;\u0026lt;p\u0026gt;You have given your consent for us to set cookies.\u0026lt;\/p\u0026gt;\u003C\/p\u003E\n    \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022eu-cookie-withdraw-button\u0022\u003EWithdraw consent\u003C\/button\u003E\n    \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E\n","withdraw_enabled":false},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1,"trackUrlFragments":1},"jnl_biorxiv_styles":{"defaultJCode":"biorxiv"},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-739126 node-type-highwire-article context-content hw-default-jcode-biorxiv hw-article-type-article hw-article-category-new-results">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-M677548" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-M677548');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
        <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
    
  <div id="zone-branding" class="zone zone-branding clearfix print-display-block container-30">
    <div class="grid-15 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="bioRxiv" src="https://www.biorxiv.org/sites/default/files/bioRxiv_article.jpg" /></a>      </div>
                </div>
          </div>
</div><div class="grid-11 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-system block-menu block-main-menu block-system-main-menu odd block-without-title" id="block-system-main-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/" title="" class="" data-icon-position="" data-hide-link-title="0">Home</a></li>
<li class="leaf" role="menuitem"><a href="/about-biorxiv" class="" data-icon-position="" data-hide-link-title="0">About</a></li>
<li class="leaf" role="menuitem"><a href="/submit-a-manuscript" class="" data-icon-position="" data-hide-link-title="0">Submit</a></li>
<li class="last leaf" role="menuitem"><a href="/content/alertsrss" title="" class="" data-icon-position="" data-hide-link-title="0">ALERTS / RSS</a></li>
</ul></nav>    </div>
  </div>
</div><div class="block block-panels-mini block-biorxiv-search-box block-panels-mini-biorxiv-search-box even block-without-title" id="block-panels-mini-biorxiv-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-biorxiv_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/10.1101/641183v2.full" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--2" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-fgAvBgRh9NZqgKPQjhICNJA2KZyBrNFKic2rznifrDI" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2 advanced-search-link" >
  
      
  
  <div class="pane-content">
    <a href="/search">Advanced Search</a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix container-30">
  	      </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="739126" id="node739126" data-pisa="biorxiv;641183v2" data-pisa-master="biorxiv;641183" data-apath="/biorxiv/early/2019/05/18/641183.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div  class="highwire-cite highwire-cite-highwire-article highwire-citation-biorxiv-article-top clearfix has-author-tooltip" >

      <span class="biorxiv-article-type">
        New Results    </span>
  
        <h1 class="highwire-cite-title" id="page-title">cRacle: R Tools for Estimating Climate from Vegetation</h1>  
      <div  class="highwire-cite-authors" ><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0"><span class="nlm-given-names">Robert S.</span> <span class="nlm-surname">Harbert</span></span>, <span class="highwire-citation-author" data-delta="1"><span class="nlm-given-names">Alex A.</span> <span class="nlm-surname">Baryiames</span></span></span></div>
  
      <div  class="highwire-cite-metadata" ><span  class="highwire-cite-metadata-doi highwire-cite-metadata"><span class="label">doi:</span> https://doi.org/10.1101/641183 </span></div>
  
  
  </div>
<div id="hw-article-author-popups-node739126" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Robert S. Harbert </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-institution'>Department of Biology, Stonehill College</span>, 320 Washington St., North Easton, MA 02357</div><div class='author-affiliation'><span class='nlm-institution'>American Museum of Natural History</span>, 79<span class='nlm-sup'>th</span> St &#38; Central Park West, New York, NY 10024</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Robert%2BS.%2BHarbert%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Harbert%20RS&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ARobert%2BS.%2BHarbert%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:rharbert@stonehill.edu" class="" data-icon-position="" data-hide-link-title="0">rharbert@stonehill.edu</a></span></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Alex A. Baryiames </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-institution'>Department of Biology, Stonehill College</span>, 320 Washington St., North Easton, MA 02357</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Alex%2BA.%2BBaryiames%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Baryiames%20AA&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AAlex%2BA.%2BBaryiames%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/10.1101/641183v2" class="panels-ajax-tab-tab" data-panel-name="biorxiv_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:739126" data-trigger="" data-url-enabled="1">Abstract</a><a href="/panels_ajax_tab/biorxiv_tab_art/node:739126/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/10.1101/641183v2.full-text" class="panels-ajax-tab-tab" data-panel-name="article_tab_full_text" data-target-id="highwire_article_tabs" data-entity-context="node:739126" data-trigger="full-text" data-url-enabled="1">Full Text</a><a href="/panels_ajax_tab/article_tab_full_text/node:739126/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/10.1101/641183v2.article-info" class="panels-ajax-tab-tab" data-panel-name="biorxiv_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:739126" data-trigger="article-info" data-url-enabled="1">Info/History</a><a href="/panels_ajax_tab/biorxiv_tab_info/node:739126/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/10.1101/641183v2.article-metrics" class="panels-ajax-tab-tab" data-panel-name="article_tab_metrics" data-target-id="highwire_article_tabs" data-entity-context="node:739126" data-trigger="article-metrics" data-url-enabled="1">Metrics</a><a href="/panels_ajax_tab/article_tab_metrics/node:739126/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/10.1101/641183v2.external-links" class="panels-ajax-tab-tab" data-panel-name="article_tab_data_code" data-target-id="highwire_article_tabs" data-entity-context="node:739126" data-trigger="external-links" data-url-enabled="1">Data/Code</a><a href="/panels_ajax_tab/article_tab_data_code/node:739126/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/10.1101/641183v2.full.pdf+html" class="panels-ajax-tab-tab" data-panel-name="biorxiv_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:739126" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i> Preview PDF</a><a href="/panels_ajax_tab/biorxiv_tab_pdf/node:739126/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="article_tab_full_text" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.biorxiv.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-article_tab_full_text"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-markup" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2 class="">ABSTRACT</h2><div id="sec-1" class="subsection"><p id="p-2"><strong>Premise of the study</strong> The Climate Reconstruction Analysis using Coexistence Likelihood Estimation (CRACLE) method for the estimation of climate from vegetation is a robust set of modeling tools for estimating climate and paleoclimate that makes use of large repositories of biodiversity data and open-access R software.</p></div><div id="sec-2" class="subsection"><p id="p-3"><strong>Methods</strong> Here we implement a new R library for the estimation of climate from vegetation.The ‘cRacle’ library implements functions for data access, aggregation, and models to estimate climate from plant community composition. ‘cRacle’ is modular and features many best-practice features.</p></div><div id="sec-3" class="subsection"><p id="p-4"><strong>Results</strong> Performance tests using modern vegetation survey data from North and South America shows that CRACLE outperforms alternative methods. CRACLE estimates of mean annual temperature (MAT) are usually within 1°C of the actual when optimal model parameters are used. Generalized Boosted Regression (GBR) model correction is also shown here to improve on CRACLE models by reducing bias.</p></div><div id="sec-4" class="subsection"><p id="p-5"><strong>Discussion</strong> CRACLE provides accurate estimates of climate from modern plant communities. Non-parametric CRACLE modeling coupled to GBR model correction produces the most accurate results to date. The ‘cRacle’ R library streamlines the estimation of climate from plant community data, and will make this modeling more accessible to a wider range of users.</p></div></div><div class="section" id="sec-5"><h2 class="">INTRODUCTION</h2><p id="p-8">‘cRacle’ is an open-source R software package that performs the Climate Reconstruction Analysis using Coexistence Likelihood Estimation (CRACLE) model for the estimation of climate from biological community data (<a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">Harbert &amp; Nixon, 2015</a>). CRACLE performs an empirical likelihood estimation of climatic parameters given a set of taxa (species, genera, or families) and their modern distributions via point-locality data from the Global Biodiversity Information Facility (GBIF) or similar primary biodiversity databases (e.g., iNaturalist, iDigBio, BIEN).</p><p id="p-9">The primary application of CRACLE and related methods is in the reconstruction of paleoclimate from fossil floras. All current applications of CRACLE make assumptions about climatic niche stability through time because the rely on the Nearest Living Relative (NLR) approach to infer elements of fossil taxon niche dimensions by observing the niche space occupied by a closely related extant relative (<a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">Mosbrugger and Utescher, 1997</a>; Utescher, 2014; <a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">Harris <em>et al</em>. 2014</a>). The NLR assumptions will be most valid when the fossil taxa can be placed as members of extant species and least accurate when fossils are members of extinct lineages. For this reason, NLR and CRACLE will produce the most reliable reconstructions of relatively recent (e.g., Pleistocene and Pliocene) paleoclimate, but may produce reasonable results for much older fossil assemblages where the plant fossil taxa can be placed reliably in extant groups with well sampled modern distributions (e.g., <a id="xref-ref-9-2" class="xref-bibr" href="#ref-9">Harris <em>et al.</em> 2014</a>).</p><p id="p-10">CRACLE modeling was recently applied to the estimation of paleoclimate using Late Quaternary (&lt;50ka – <em>kilo anuum</em>) plant macrofossils from packrat (<em>Neotoma</em> spp.) middens in Western North America (<a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">Harbert, Nixon 2018</a>). Analysis of the packrat midden paleoclimate estimates revealed a history of rapid climate change during and just after the Late Pleistocene deglaciation, followed by Holocene warm and dry periods of 1-2°C above the modern (1950-2000) averages (<a id="xref-ref-8-2" class="xref-bibr" href="#ref-8">Harbert and Nixon, 2018</a>)</p><p id="p-11">Community assemblages from excavated permafrost at five Pliocene sites in the Canadian Arctic Archipelago analyzed with CRACLE (<a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">Fletcher <em>et al</em>., 2017</a>). This work revealed that the Canadian arctic was up to 22°C warmer than today during the Early Pliocene (~3.6Ma – <em>mega anuum</em>) and supported many cool-temperate plant taxa. However, <a id="xref-ref-4-2" class="xref-bibr" href="#ref-4">Fletcher <em>et al.</em> (2017)</a> carefully compared results derived from CRACLE estimates using species only identifications vs. generic level identifications and suggest that care must be taken when interpreting CRACLE results when generic level identifications are used to avoid being misled by model imprecision.</p><p id="p-12">Modified bioclimatic envelope method that are closely related to CRACLE were recently used to reconstruct the Paleocene-Eocene Thermal Maximum (~55Ma) and the Eocene Thermal Maximum (53.5Ma) by way of plant macrofossils from the Green River Basin in southwestern Wyoming and plant microfossils found in Arctic marine sediment cores (<a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">Hyland et al., 2018</a>; <a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">Willard et al., 2019</a>).</p><p id="p-13">CRACLE models have been applied to non-plant communities as well. Pliocene Arctic climate estimates based on fossil beetle communities using CRACLE, revealed notable biases in beetle based climate reconstructions relative to plant proxy and stable isotope methods (<a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">Fletcher <em>et al</em>., 2018</a>).</p><p id="p-14">Despite CRACLE existing for the first four years as only a single R script published as a supplement to the original paper (<a id="xref-ref-7-2" class="xref-bibr" href="#ref-7">Harbert and Nixon, 2015</a>) the method has proven to be a useful tool to the community (presently 16 Google Scholar citations), yielding informative results for several fossil systems. Here, we present the ‘cRacle’ R package. ‘cRacle’ is designed to streamline the process of CRACLE modeling, data acquisition and pre-processing of primary biodiversity data (from GBIF and other related databases), and CRACLE result visualization. The ‘cRacle’ R package is built from modular functions that perform segments of the modeling protocol that allow for easy to read R scripts as well as future modular developments of related methods.</p><p id="p-15">For comparison with the CRACLE methods, we have also implemented the related Thompson’s Mutual Climatic Range (MCR; <a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">Thompson et al., 2012</a>). We include both the weighted and unweighted MCR methods as representatives of the many available MCR or Coexistence Approach (CA) range-intersect methods (<a id="xref-ref-16-2" class="xref-bibr" href="#ref-16">Mosbrugger and Utescher, 1997</a>; Sinka and Atkinson, 1999; <a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">Utescher et al., 2014</a>) for the estimation of climate from vegetation (ECV) as part of the ‘cRacle’ R package. The ‘cRacle’ package is distributed under the open MIT License and all code is available on GitHub (<a href="https://www.github.com/rsh249/cRacle">https://www.github.com/rsh249/cRacle</a>).</p></div><div class="section" id="sec-6"><h2 class="">METHODS</h2><p id="p-16">Full documentation of ‘cRacle’ R code is available with the package code (<a href="https://github.com/rsh249/cRacle">https://github.com/rsh249/cRacle</a>). Below is an outline of the major functionality of the ‘cRacle’ package and explanation of many of the core functions available.</p><div id="sec-7" class="subsection"><h3>‘cRacle’ Package Functions – Data Acquisition</h3><p id="p-17">Downloading primary biodiversity data from GBIF (<a href="http://www.gbif.org">www.gbif.org</a>), iNaturalist (<a href="http://www.inaturalist.org">www.inaturalist.org</a>), and BISON (<a href="http://www.bison.usgs.gov">www.bison.usgs.gov</a>) are supported in ‘cRacle’ through the functions <em>gbif_get(), get_bison()</em>, and <em>inat()</em>. Each of these functions accepts arguments for a single taxon (genus, species, or family) to query and a maximum number of records to return. Data pre-processing, cleaning for outliers, spatial bias reduction, and climate data extraction for a set of georeferenced occurrence data is done through the <em>extraction()</em> function. Data downloading for multiple taxa and preprocessing functionality can be done in one step using the <em>getextr()</em> function, which is implemented with parallel computing capability to reduce user wait time.</p></div><div id="sec-8" class="subsection"><h3>Likelihood Modeling</h3><p id="p-18">The primary modeling that CRACLE requires is the calculation of probability density functions using both parametric (Gaussian Normal) and non-parametric (Kernel Density) estimates. These calculations can be performed with ‘cRacle’ directly from the output of <em>extraction()</em> using the function(s) <em>dens_obj()</em> and <em>densform()</em> for multiple and single taxa respectively. Note that <em>dens_obj()</em> is simply a wrapper function for <em>densform()</em> designed to make multi-species likelihood model building simpler. Both parametric and non-parametric likelihood functions are always calculated and stored in the object produced by <em>densform()</em> and <em>dens_obj()</em>. This way all calculations are done in tandem for a dataset and results for both methods can be examined at the end of the CRACLE process.</p><p id="p-19">The likelihood modeling carried out in the <em>densform()</em> function includes several options that can be manipulated by the user. The Kernel Density Estimation (KDE) employed by CRACLE can now be implemented with several kernel estimators including the standard Gaussian, Epanechnikov, cosine, optcosine, biweight, rectangular, and triangular kernels based on the R ‘stats’ package function <em>density()</em> (<a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">R Core Team, 2018</a>). The kernel bandwidth can be optimized using standard criteria via the argument ‘bw’ in the <em>densform()</em> and <em>dens_obj()</em> functions. Likelihoods can be calculated based on a standard or conditional probability via the ‘manip’ argument where conditional probabilities take into account the background distribution of each climate parameter using a set of randomly sampled points from within a normalized distance from each primary occurrence location or from a user defined set of climate data. Finally, the resulting likelihood distributions can be trimmed to empirical ranges or confidence intervals to avoid unintended extrapolation using the ‘clip’ argument.</p></div><div id="sec-9" class="subsection"><h3>CRACLE Joint Likelihood</h3><p id="p-20">After likelihood functions are estimated the next key CRACLE step is to calculate first the joint likelihood and then the maximum of the joint likelihood distribution. ‘cRacle’ implements the function <em>and_fun()</em> for calculating the joint likelihood and <em>or_fun()</em> for creating unions between taxa for merging likelihood features between groups (i.e., between species in a clade). The output of <em>and_fun()</em> is a set of joint likelihood features for all parameters, and this can be summarized by the function <em>get_optim()</em> which finds the optimal climate values for each parameter. The output of <em>get_optim()</em> provides the user with CRACLE results.</p></div><div id="sec-10" class="subsection"><h3>Example Pseudocode</h3><p id="p-21">The general CRACLE modeling flow will use the following functions in order: <em>getextr() -&gt; dens_obj()-&gt; and_fun()-&gt; get_optim();</em> to produce climate estimates from a list of taxa. This code would: download data and extract climate data, estimate likelihood distributions, calculate the joint likelihood, and find the optimum of the joint likelihood for each climate variable analyzed.</p></div><div id="sec-11" class="subsection"><h3>Visualizing Likelihood and Joint Likelihood Functions</h3><p id="p-22">‘cRacle’ provides a set of functions for visualizing the likelihood functions as standard probability density distributions. These functions are <em>densplot()</em> for plotting single distribution objects (i.e., for one taxon/climate parameter pair), <em>addplot()</em> for adding to an existing <em>densplot()</em> figure, and <em>multiplot()</em> for plotting distributions for multiple taxa given a single climate parameter. For example, a user may plot the distributions for all taxa in their study for Mean Annual Temperature using <em>multiplot()</em> and the output from <em>dens_obj()</em>, and then add the joint likelihood function using <em>addplot()</em> and the model object output from <em>and_fun()</em>.</p></div><div id="sec-12" class="subsection"><h3>Issue Tracking and Versioning</h3><p id="p-23">The repository at <a href="https://github.com/rsh249/cRacle.git">https://github.com/rsh249/cRacle.git</a> will maintain the current development version of ‘cRacle’. Future major releases will be submitted to CRAN. Issues with running ‘cRacle’ code are accepted on the Git repository’s ‘Issues’ feature at <a href="https://github.com/rsh249/cRacle/issues">https://github.com/rsh249/cRacle/issues</a> or by email with the corresponding author of this publication.</p></div><div id="sec-13" class="subsection"><h3>Data access</h3><p id="p-24">Primary biodiversity data, occurrence coordinates for your target taxa, and climate data are required for CRACLE modeling. We recommend primary biodiversity data downloaded from the Global Biodiversity Information Facility (GBIF) and support data access through ‘cRacle’ functions. Climate data must be in the form of a raster object readable by the R ‘raster’ package (<a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">Hijmans, 2018</a>). Climate data from the WorldClim project (<a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">Hijmans et al., 2005</a>; <a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">Fick and Hijmans, 2017</a>) are recommended, but users may also use raster data downloaded from other sources such as CHELSA (<a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">Karger et al., 2017</a>), and ENVIREM (<a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">Title and Bemmels, 2018</a>).</p></div><div id="sec-14" class="subsection"><h3>Estimation Climate from Vegetation – (ECV)</h3><p id="p-25">As originally described CRACLE consists of two methods for likelihood estimation: parametric and non-parametric probability functions. ‘cRacle’ implements these two approaches in tandem for each step of the likelihood analysis, and has implemented options about whether to calculate the likelihoods from standard probability or conditional probability. For comparison we have implemented calculations for the Mutual Climatic Range (MCR) method for ECV described by <a id="xref-ref-19-2" class="xref-bibr" href="#ref-19">Thompson <em>et al.</em> (2012)</a>, though the unweighted MCR described is analogous to the Coexistence Approach (CA) described elsewhere (<a id="xref-ref-16-3" class="xref-bibr" href="#ref-16">Mosbrugger and Utescher, 1997</a>)</p><p id="p-26">CRACLE (<a id="xref-ref-7-3" class="xref-bibr" href="#ref-7">Harbert and Nixon, 2015</a>) defines both a parametric and non-parametric probability density function for each taxon – climate parameter pair. CRACLE estimates likelihood for a set of taxa (t<sub>1:i</sub>). The CRACLE Parametric –Normal – probabilities are estimated as:
<span class="disp-formula" id="disp-formula-1"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2019/05/18/641183/embed/graphic-1.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2019/05/18/641183/embed/graphic-1.gif"/></noscript></span>

</span></p><p id="p-27">Whereas the CRACLE Non-Parametric – Kernel Density Estimation (KDE) – probabilities are estimated as:
<span class="disp-formula" id="disp-formula-2"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2019/05/18/641183/embed/graphic-2.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2019/05/18/641183/embed/graphic-2.gif"/></noscript></span>

</span></p><p id="p-28">Where K is a Kernel function of properties XXXX, and ‘h’ is the kernel bandwidth, a smoothing parameter with value &gt; 0. We recommend using either the “optcosine” or “gaussian” kernels and a bandwidth calculated using Silverman’s Rule of Thumb (<a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">Silverman, 1986</a>). ‘cRacle’ allows for the calculation of probability density functions conditioned by the probability density of a random background sample for each climate parameter defined as:
<span class="disp-formula" id="disp-formula-3"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2019/05/18/641183/embed/graphic-3.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2019/05/18/641183/embed/graphic-3.gif"/></noscript></span>

</span></p><p id="p-29">Where ‘c’ is the set of background climate given by a sample of points from within the study area. The conditional probability for a taxon (t) is defined as:
<span class="disp-formula" id="disp-formula-4"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2019/05/18/641183/embed/graphic-4.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2019/05/18/641183/embed/graphic-4.gif"/></noscript></span>

</span></p><p id="p-30">For the estimation of climate from vegetation CRACLE modeling calculates the joint likelihood for multiple taxon probability functions as:
<span class="disp-formula" id="disp-formula-5"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2019/05/18/641183/embed/graphic-5.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2019/05/18/641183/embed/graphic-5.gif"/></noscript></span>

</span></p><p id="p-31">Where the maximum value of L(x| t<sub>1:i</sub>)corresponds to the value of the climate parameter ‘x’ that is most likely to lead to the coexistence of that set of taxa (t<sub>1:i</sub>).</p><p id="p-32">Note that ‘cRacle’ also implements the calculation of a weighted mean by variance that can be substituted for the parametric CRACLE functions. Weighted means <span class="inline-formula" id="inline-formula-1"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2019/05/18/641183/embed/inline-graphic-1.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2019/05/18/641183/embed/inline-graphic-1.gif"/></noscript></span></span> is calculated given a set of taxon means (x<sub>1:i</sub>) and standard deviations (σ<sub>1:i</sub>):
<span class="disp-formula" id="disp-formula-6"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2019/05/18/641183/embed/graphic-6.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2019/05/18/641183/embed/graphic-6.gif"/></noscript></span>

</span></p></div><div id="sec-15" class="subsection"><h3>Modern Validation</h3><p id="p-33">To test the performance of CRACLE that can be expected for paleoclimate modeling we set two experimental analyses to test aspects of the CRACLE modeling process.</p><p id="p-34">First, we sample iNaturalist data (<a href="http://www.inaturalist.org">www.inaturalist.org</a>) by a grid search of North America for plant species coexisting in 0.1×0.1 degree bounded search areas using the ‘rinat’ R package (<a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">Barve and Hart, 2017</a>). Then we build CRACLE estimates for Mean Annual Temperature using the WorldClim 2.0 dataset at 2.5 arcminutes resolution (<a id="xref-ref-3-2" class="xref-bibr" href="#ref-3">Fick and Hijmans, 2017</a>) for parametric and non-parametric CRACLE as well as MCR and MCR un (unweighted) methods. Primary biodiversity data were obtained from GBIF (<a href="http://www.gbif.org">www.gbif.org</a>) using the ‘cRacle’ <em>get_dist_all()</em> function and the GBIF RESTful JSON based API. Model overfitting for the iNaturalist dataset was tested by spatially thinning the raw GBIF data using the ‘cRacle’ <em>extraction()</em> function for factors of 2,4,6,8, and 10 times the dimensions of the 2.5 arcminute climate raster cells. The ‘cRacle’ thinning procedure is an imperfect, but efficient, spatial thinning method that is consistent with best-practices in Ecological Niche Modeling literature (<a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">Aiello-Lammens <em>et al.</em> 2015</a>). CRACLE and MCR model output for each thinning level were summarized and reported for general guidance. The R code for the iNaturalist grid search test is available at <a href="https://github.com/rsh249/cracle_examples/inat_grid_search">https://github.com/rsh249/cracle_examples/inat_grid_search</a>.</p><p id="p-35">Second, the Botanical Information and Ecology Network (BIEN 4.0) database was queried for published vegetation plot surveys from North and South America using the ‘rbien’ R package (<a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">Maitner <em>et al.</em> 2017</a>). For plot surveys with &gt;5 plant taxa with distribution data available from GBIF we estimated climate parameters for all 19 bioclimatic variables from the WorldClim 2.0 data (<a id="xref-ref-3-3" class="xref-bibr" href="#ref-3">Fick and Hijmans, 2017</a>) using parametric CRACLE methods and spatial thinning to a factor of 2 times the 2.5 arcminute climate grid. Error in CRACLE are non-random (<a id="xref-ref-7-4" class="xref-bibr" href="#ref-7">Harbert and Nixon, 2015</a>), therefore non-linear regression modeling may be able to account for and correct common CRACLE errors. CRACLE results for 70,391 plot surveys were then partitioned 50:50 into training and testing sets by random sampling with replacement and Generalized Boosted Regression (GBR) models were fit to the training data for each of the 19 climate parameters using the R ‘gbm’ package (<a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">Greenwell <em>et al.</em> 2019</a>). The GBR models made adjusted climate estimates using the CRACLE results in the test set to determine whether error patterns in CRACLE can be predicted by GBR and therefore may be corrected for in independent analyses. The R code for the iNaturalist grid search is available: <a href="https://github.com/rsh249/cracle_examples/cracle_bien">https://github.com/rsh249/cracle_examples/cracle_bien</a>.</p></div></div><div class="section" id="sec-16"><h2 class="">RESULTS</h2><div id="sec-17" class="subsection"><h3>Modern Validation</h3><div id="sec-18" class="subsection"><h4>iNaturalist</h4><p id="p-36">The grid search of iNaturalist in North America yielded climate estimations by CRACLE (both parametric and non-parametric) and the Mutual Climatic Range (MCR) unweighted and weighted approaches for 285 sites at spatial thinning factors of 0, 2, 4, 6, 8, and 10 times the 2.5 arcminute climate raster. A total of 1,204,925 unique records were accessed from GBIF for this analysis for a total of 1541 species. The sampled localities ranged in mean annual temperature (MAT) from 0.4-26.1°C. The top performing (lowest error rates) estimates were the CRACLE non-parametric (‘kde’ in <a id="xref-fig-1-1" class="xref-fig" href="#F1">Figure 1</a>) and the MCR weighted (‘mcr.w’ in <a id="xref-fig-1-2" class="xref-fig" href="#F1">Figure 1</a>) methods. Mean errors for CRACLE were ~1°C and median errors were generally less than the means. Spatial thinning did not change mean or median error rates for any of the tested methods (<a id="xref-fig-1-3" class="xref-fig" href="#F1">Fig. 1B,C</a>).</p><div id="F1" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/05/18/641183/F1.large.jpg?width=800&amp;height=600&amp;carousel=1" title="iNaturalist Grid Search Climate Estimation Results. A) Geographic search area (shading) with successful climate estimation locations marked (black squares). B) Mean Anomaly Rates, and C) Median Anomaly Rates for CRACLE (&#x2018;kde&#x2019; non-parametric, and &#x2018;means&#x2019; parametric) and MCR (&#x2018;uw&#x2019; unweighted and &#x2018;w&#x2019; weighted) results for factor levels of 0 (&#x2018;raw&#x2019;), 2, 4, 6, 8, and 10 times the 2.5 arcminute climate raster." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-828798454" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;iNaturalist Grid Search Climate Estimation Results. A) Geographic search area (shading) with successful climate estimation locations marked (black squares). B) Mean Anomaly Rates, and C) Median Anomaly Rates for CRACLE (&#x2018;kde&#x2019; non-parametric, and &#x2018;means&#x2019; parametric) and MCR (&#x2018;uw&#x2019; unweighted and &#x2018;w&#x2019; weighted) results for factor levels of 0 (&#x2018;raw&#x2019;), 2, 4, 6, 8, and 10 times the 2.5 arcminute climate raster.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 1." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2019/05/18/641183/F1.medium.gif" width="196" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 1." src="https://www.biorxiv.org/content/biorxiv/early/2019/05/18/641183/F1.medium.gif" width="196" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/05/18/641183/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1." data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/05/18/641183/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml"><span class="fig-label">Figure 1.</span> <p id="p-37" class="first-child">iNaturalist Grid Search Climate Estimation Results. A) Geographic search area (shading) with successful climate estimation locations marked (black squares). B) Mean Anomaly Rates, and C) Median Anomaly Rates for CRACLE (‘kde’ non-parametric, and ‘means’ parametric) and MCR (‘uw’ unweighted and ‘w’ weighted) results for factor levels of 0 (‘raw’), 2, 4, 6, 8, and 10 times the 2.5 arcminute climate raster.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-19" class="subsection"><h4>BIEN</h4><p id="p-38">Non-parametric CRACLE estimates of all 19 WorldClim Bioclim variables (<a href="http://www.worldclim.org/bioclim">www.worldclim.org/bioclim</a>) from the WorldClim 2.0 dataset (<a id="xref-ref-3-4" class="xref-bibr" href="#ref-3">Fick and Hijmans, 2017</a>) were generated for 70,391 vegetation plot surveys accessed through the BIEN 4.0 database using a total of 2,560,261 georeferenced specimen records from GBIF.</p><p id="p-39">Mean absolute anomaly rates for temperature parameters were between 1-2°C with Pearson’s Correlation coefficients greater than 0.9 in the best scenarios (<a id="xref-table-wrap-1-1" class="xref-table" href="#T1">Table 1</a>), but up to 5°C and correlation coefficients &lt;0.7 for estimates of mean temperature during the wettest and driest quarters of the year (BIO8 and BIO9). The mean absolute anomaly rates for mean annual precipitation (MAP – BIO12) was 179mm with a correlation of 0.79 (<a id="xref-table-wrap-1-2" class="xref-table" href="#T1">Table 1</a>).</p><div id="T1" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/741136/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup"><a href="/highwire/markup/741136/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/741136" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 1:</span> <p id="p-40" class="first-child">CRACLE performance statistics for 19 Bioclim parameters estimated for BIEN vegetation plot data (n=35,195 plots)</p><div class="sb-div caption-clear"></div></div></div><p id="p-41">Generalized Boosted Regression (GBR) models were used to correct CRACLE estimates using independent random samples without replacement of 50% of the data for training and testing datasets. GBR correction improved CRACLE performance in the test set by 20-50% (<a id="xref-table-wrap-1-3" class="xref-table" href="#T1">Table 1</a>, <a id="xref-table-wrap-2-1" class="xref-table" href="#T2">2</a>; <a id="xref-fig-2-1" class="xref-fig" href="#F2">Figure 2</a>). CRACLE+GBM corrected anomalies are smaller and more symmetrically distributed around ‘0’ than the non-corrected CRACLE results (<a id="xref-fig-2-2" class="xref-fig" href="#F2">Figure 2D</a>), and CRACLE+GBM estimates are more strongly correlated with the WorldClim data than the raw CRACLE results (<a id="xref-table-wrap-1-4" class="xref-table" href="#T1">Table 1</a>, <a id="xref-table-wrap-2-2" class="xref-table" href="#T2">2</a>).</p><div id="T2" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/741139/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/741139/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 2:</span> <p id="p-42" class="first-child">GBR Corrected CRACLE performance statistics for 19 Bioclim parameters estimated for BIEN vegetation plot data (n=35,195 plots)</p><div class="sb-div caption-clear"></div></div></div><div id="F2" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/05/18/641183/F2.large.jpg?width=800&amp;height=600&amp;carousel=1" title="CRACLE estimates and GBR error correction demonstration for BIEN vegetation plot surveys and representative climate parameters. BIEN vegetation plot surveys from 70,391 unique localities (A) were analyzed with CRACLE (B) to estimate the 19 Bioclim variables (showing BIO1, 5, 6, 12, 17, and 18 here as top performing examples) from the WorldClim2.0 dataset (Fick and Hijmans, 2017). Generalized Boosted Regression error correction modeling was trained and tested (C) on independent subsets of plot data. GBR correction yielded overall reduced error rates and less biased estimates in many cases (D)." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-828798454" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;span xmlns=&quot;http://www.w3.org/1999/xhtml&quot; class=&quot;caption-title&quot;&gt;CRACLE estimates and GBR error correction demonstration for BIEN vegetation plot surveys and representative climate parameters.&lt;/span&gt; BIEN vegetation plot surveys from 70,391 unique localities (A) were analyzed with CRACLE (B) to estimate the 19 Bioclim variables (showing BIO1, 5, 6, 12, 17, and 18 here as top performing examples) from the WorldClim2.0 dataset (Fick and Hijmans, 2017). Generalized Boosted Regression error correction modeling was trained and tested (C) on independent subsets of plot data. GBR correction yielded overall reduced error rates and less biased estimates in many cases (D).&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 2." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2019/05/18/641183/F2.medium.gif" width="379" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 2." src="https://www.biorxiv.org/content/biorxiv/early/2019/05/18/641183/F2.medium.gif" width="379" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/05/18/641183/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2." data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/05/18/641183/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 2.</span> <span class="caption-title">CRACLE estimates and GBR error correction demonstration for BIEN vegetation plot surveys and representative climate parameters.</span><p id="p-43" class="first-child">BIEN vegetation plot surveys from 70,391 unique localities (A) were analyzed with CRACLE (B) to estimate the 19 Bioclim variables (showing BIO1, 5, 6, 12, 17, and 18 here as top performing examples) from the WorldClim2.0 dataset (<a id="xref-ref-3-5" class="xref-bibr" href="#ref-3">Fick and Hijmans, 2017</a>). Generalized Boosted Regression error correction modeling was trained and tested (C) on independent subsets of plot data. GBR correction yielded overall reduced error rates and less biased estimates in many cases (D).</p><div class="sb-div caption-clear"></div></div></div></div></div></div><div class="section" id="sec-20"><h2 class="">DISCUSSION</h2><p id="p-44">The modern validation analyses conducted here should provide a good baseline for expected CRACLE performance going forward. More thorough testing in future studies is certainly welcome, but most modeling choices are should be made by the user to reflect any unique properties of their study system. The results we provide here indicate that the best estimates that are generated from the ‘cRacle’ software are the CRACLE non-parametric estimates with spatially thinned data, these yielding mean errors of approximately 0.5°C less than the Mutual Climatic Range (MCR) method, an analogous method to the widely used Coexistence Approach (<a id="xref-ref-16-4" class="xref-bibr" href="#ref-16">Mosbrugger and Utescher, 1997</a>). Furthermore, CRACLE results presented here compare favorably, though indirectly, to recent applications of the Weighted Average – Partial Least Squares (WA-PLS) modeling that is common for analysis of fossil pollen samples that uses modern analog communities to build proxy models (<a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">Montade <em>et al.</em>, 2019</a>). A direct comparison of CRACLE and WA-PLS methods is necessary future work.</p><p id="p-45">Through our analysis of BIEN hosted vegetation plot surveys we show that CRACLE can produce accurate climate estimates for a variety of both temperature and precipitation parameters (<a id="xref-fig-2-3" class="xref-fig" href="#F2">Figure 2</a>), though some parameters are better predicted than others (<a id="xref-table-wrap-1-5" class="xref-table" href="#T1">Table 1</a>). Notably, BIO8 (Mean Temperature of the Warmest Quarter), BIO9 (Mean Temperature of the Coldest Quarter), BIO13 (Precipitation of the Wettest Month), and BIO15 (Precipitation Seasonality – Coeff. Var.) are relatively poorly predicted by CRACLE, likely due to less direct impact of those variables on plant distributions. Whereas estimates of BIO1 (MAT), BIO5 (Max Temperature), BIO6 (Min Temperature), and BIO18 (Precipitation of the Driest Quarter) yield the highest correlation with known values (<a id="xref-table-wrap-1-6" class="xref-table" href="#T1">Table 1</a>). We also show that CRACLE model correction via Generalized Boosted Regression (GBR) can lower error rates from the CRACLE baseline (<a id="xref-fig-2-4" class="xref-fig" href="#F2">Figure 2</a>; <a id="xref-table-wrap-2-3" class="xref-table" href="#T2">Table 2</a>). We expect that further regression model fitting on expanded datasets may yield further refinement, however, the GBR model correction is implemented in ‘cRacle’ <em>get_optim()</em> as an option for users to correct non-parameteric CRACLE estimates for the WorldClim 2.0 Bioclim variables only.</p><div id="sec-21" class="subsection"><h3>Resources and Tutorials</h3><p id="p-46">We are actively developing resources and tutorials in support of the ‘cRacle’ R package. Demonstration code and short projects will be maintained on a continuing basis at <a href="https://github.com/rsh249/cracle_examples">https://github.com/rsh249/cracle_examples</a>. We aim to provide a series of web tutorials for various CRACLE modeling tasks here to guide beginner users through the process. Tutorials, and issue tracking are distributed through the main ‘cRacle’ repository: <a href="https://github.com/rsh249/cRacle.git">https://github.com/rsh249/cRacle.git</a>.</p></div><div id="sec-22" class="subsection"><h3>Conclusion</h3><p id="p-47">‘cRacle’ is a new and actively maintained resource for climate estimation from biological community composition. These estimates are particularly relevant to the study of fossil systems where often the best indication of past climate is the community of plant and animal fossils. We show that users of ‘cRacle’ should expect estimates that are accurate (e.g., within 1°C for MAT) when applying best-practices.</p></div></div><div class="section" id="sec-23"><h2 class="">AUTHOR CONTRIBUTIONS</h2><p id="p-49">AAB conceived of and implemented the iNaturalist grid search and CRACLE analysis and provided valuable testing, benchmarking, and feedback data on the ‘cRacle’ R library. RSH designed the BIEN vegetation plots CRACLE study and implemented the Generalized Boosted Regression model correction analysis. RSH designed and wrote the ‘cRacle’ R package code.</p></div><div class="section ack" id="ack-1"><h2 class="">Acknowledgements</h2><p id="p-48">The authors would like to thank Stonehill College for their support of undergraduate research that made this study possible.</p></div><div class="section fn-group" id="fn-group-1"><h2>Footnotes</h2><ul><li class="fn-update fn-group-summary-of-updates" id="fn-1"><p id="p-6">An error in the abstract formatting to a paragraph format compatible with bioRxiv</p></li><li class="fn-dataset fn-group-external-links" id="fn-2"><p id="p-7">
<a href="https://github.com/rsh249/cRacle.git">https://github.com/rsh249/cRacle.git</a>
</p></li></ul></div><div class="section ref-list" id="ref-list-1"><h2 class="">LITERATURE CITED</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference  in text" id="ref-1">↵</a><div class="cit ref-cit ref-journal" id="cit-641183v2.1" data-doi="10.1111/ecog.01132"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Aiello□Lammens</span>,  <span class="cit-name-given-names">M.E.</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">R.A.</span> <span class="cit-name-surname">Boria</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Radosavljevic</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">B.</span> <span class="cit-name-surname">Vilela</span></span>, and <span class="cit-auth"> <span class="cit-name-given-names">R.P.</span> <span class="cit-name-surname">Anderson</span></span>. <span class="cit-pub-date">2015</span>. <span class="cit-article-title">spThin: an R package for spatial thinning of species occurrence records for use in ecological niche models</span>. <abbr class="cit-jnl-abbrev">Ecography</abbr>, <span class="cit-vol">38</span>(<span class="cit-issue">5</span>): <span class="cit-fpage">541</span>–<span class="cit-lpage">545</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEcography%26rft.volume%253D38%26rft.spage%253D541%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fecog.01132%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/ecog.01132&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference  in text" id="ref-2">↵</a><div class="cit ref-cit ref-web" id="cit-641183v2.2"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Barve</span>,  <span class="cit-name-given-names">V.</span></span>, and <span class="cit-auth"> <span class="cit-name-given-names">E.</span> <span class="cit-name-surname">Hart</span></span>. <span class="cit-pub-date">2017</span>. ‘<span class="cit-article-title">rinat’: Access iNaturalist Data Through APIs</span>. R package version 0.1.5. <a href="https://cran.r-project.org/package=rinat">https://cran.r-project.org/package=rinat</a></cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference  in text" id="ref-3">↵</a><div class="cit ref-cit ref-journal" id="cit-641183v2.3"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Fick</span>,  <span class="cit-name-given-names">S.E.</span></span> and <span class="cit-auth"> <span class="cit-name-given-names">R.J.</span> <span class="cit-name-surname">Hijmans</span></span>. <span class="cit-pub-date">2017</span>. <span class="cit-article-title">WorldClim 2: new 1□km spatial resolution climate surfaces for global land areas</span>. <abbr class="cit-jnl-abbrev">International journal of climatology</abbr> <span class="cit-vol">37</span>(<span class="cit-issue">12</span>):<span class="cit-fpage">4302</span>–<span class="cit-lpage">4315</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DInternational%2Bjournal%2Bof%2Bclimatology%26rft.volume%253D37%26rft.spage%253D4302%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference  in text" id="ref-4">↵</a><div class="cit ref-cit ref-journal" id="cit-641183v2.4"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Fletcher</span>,  <span class="cit-name-given-names">T.</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">R.</span> <span class="cit-name-surname">Feng</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">A.M.</span> <span class="cit-name-surname">Telka</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">J.V.</span> <span class="cit-name-surname">Matthews</span>   <span class="cit-name-suffix">Jr</span>.</span>, and <span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Ballantyne</span></span>. <span class="cit-pub-date">2017</span>. <span class="cit-article-title">Floral Dissimilarity and the Influence of Climate in the Pliocene High Arctic: Biotic and Abiotic Influences on Five Sites on the Canadian Arctic Archipelago</span>. <abbr class="cit-jnl-abbrev">Frontiers in Ecology and Evolution</abbr> <span class="cit-vol">5</span>:<span class="cit-fpage">19</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DFrontiers%2Bin%2BEcology%2Band%2BEvolution%26rft.volume%253D5%26rft.spage%253D19%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference  in text" id="ref-5">↵</a><div class="cit ref-cit ref-journal" id="cit-641183v2.5"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Fletcher</span>,  <span class="cit-name-given-names">T.L.</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">A.Z.</span> <span class="cit-name-surname">Csank</span></span>, and <span class="cit-auth"> <span class="cit-name-given-names">A.P.</span> <span class="cit-name-surname">Ballantyne</span></span>. <span class="cit-pub-date">2018</span>. <span class="cit-article-title">Identifying bias in cold season temperature reconstructions by beetle mutual climatic range methods in the Pliocene Canadian High Arctic</span>. <abbr class="cit-jnl-abbrev">Palaeogeography, Palaeoclimatology, Palaeoecology</abbr> <span class="cit-vol">514</span>: <span class="cit-fpage">672</span>–<span class="cit-lpage">676</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPalaeogeography%252C%2BPalaeoclimatology%252C%2BPalaeoecology%26rft.volume%253D514%26rft.spage%253D672%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference  in text" id="ref-6">↵</a><div class="cit ref-cit ref-web" id="cit-641183v2.6"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Greenwell</span>,  <span class="cit-name-given-names">B.</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">B.</span> <span class="cit-name-surname">Boehmke</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Cunningham</span></span>, and <span class="cit-auth"> <span class="cit-name-given-names">GBM</span> <span class="cit-name-surname">Developers</span></span>. <span class="cit-pub-date">2019</span>. ‘<span class="cit-article-title">gbm’: Generalized Boosted Regression Models</span>. R package version 2.1.5 <a href="https://cran.r-project.org/package=gbm">https://cran.r-project.org/package=gbm</a></cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference  in text" id="ref-7">↵</a><div class="cit ref-cit ref-journal" id="cit-641183v2.7" data-doi="10.3732/ajb.1400500"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Harbert</span>,  <span class="cit-name-given-names">R.S.</span></span> and <span class="cit-auth"> <span class="cit-name-given-names">K.C.</span> <span class="cit-name-surname">Nixon</span></span>. <span class="cit-pub-date">2015</span>. <span class="cit-article-title">Climate reconstruction analysis using coexistence likelihood estimation (CRACLE): A method for the estimation of climate using vegetation</span>. <abbr class="cit-jnl-abbrev">American Journal of Botany</abbr> <span class="cit-vol">102</span>(<span class="cit-issue">8</span>): <span class="cit-fpage">1277</span>–<span class="cit-lpage">1289</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAmerican%2BJournal%2Bof%2BBotany%26rft_id%253Dinfo%253Adoi%252F10.3732%252Fajb.1400500%26rft_id%253Dinfo%253Apmid%252F26290551%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiYW1qYm90IjtzOjU6InJlc2lkIjtzOjEwOiIxMDIvOC8xMjc3IjtzOjQ6ImF0b20iO3M6Mzc6Ii9iaW9yeGl2L2Vhcmx5LzIwMTkvMDUvMTgvNjQxMTgzLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference  in text" id="ref-8">↵</a><div class="cit ref-cit ref-journal" id="cit-641183v2.8"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Harbert</span>,  <span class="cit-name-given-names">R.S.</span></span> and <span class="cit-auth"><span class="cit-name-surname">Nixon</span>,  <span class="cit-name-given-names">K.C.</span></span>, <span class="cit-pub-date">2018</span>. <span class="cit-article-title">Quantitative Late Quaternary Climate Reconstruction from Plant Macrofossil Communities in Western North America</span>. <abbr class="cit-jnl-abbrev">Open Quaternary</abbr> <span class="cit-vol">4</span>(<span class="cit-issue">1</span>):<span class="cit-fpage">8</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DOpen%2BQuaternary%26rft.volume%253D4%26rft.spage%253D8%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference  in text" id="ref-9">↵</a><div class="cit ref-cit ref-journal" id="cit-641183v2.9"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Harris</span>,  <span class="cit-name-given-names">A.J.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Papeş</span>,  <span class="cit-name-given-names">M.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Gao</span>,  <span class="cit-name-given-names">Y.D.</span></span> and <span class="cit-auth"><span class="cit-name-surname">Watson</span>,  <span class="cit-name-given-names">L.</span></span>, <span class="cit-pub-date">2014</span>. <span class="cit-article-title">Estimating paleoenvironments using ecological niche models of nearest living relatives: A case study of Eocene Aesculus L</span>. <abbr class="cit-jnl-abbrev">Journal of Systematics and Evolution</abbr>, <span class="cit-vol">52</span>(<span class="cit-issue">1</span>): <span class="cit-fpage">16</span>–<span class="cit-lpage">34</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BSystematics%2Band%2BEvolution%26rft.volume%253D52%26rft.spage%253D16%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference  in text" id="ref-10">↵</a><div class="cit ref-cit ref-journal" id="cit-641183v2.10"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Hijmans</span>,  <span class="cit-name-given-names">R.J.</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">S.E.</span> <span class="cit-name-surname">Cameron</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">J.L.</span> <span class="cit-name-surname">Parra</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">P.G.</span> <span class="cit-name-surname">Jones</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Jarvis</span></span>. <span class="cit-pub-date">2005</span>. <span class="cit-article-title">Very high resolution interpolated climate surfaces for global land areas</span>. <abbr class="cit-jnl-abbrev">International Journal of Climatology: A Journal of the Royal Meteorological Society</abbr> <span class="cit-vol">25</span>(<span class="cit-issue">15</span>):<span class="cit-fpage">1965</span>–<span class="cit-lpage">1978</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DInternational%2BJournal%2Bof%2BClimatology%253A%2BA%2BJournal%2Bof%2Bthe%2BRoyal%2BMeteorological%2BSociety%26rft.volume%253D25%26rft.spage%253D1965%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference  in text" id="ref-11">↵</a><div class="cit ref-cit ref-web" id="cit-641183v2.11"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Hijmans</span>,  <span class="cit-name-given-names">R.J.</span></span> (<span class="cit-pub-date">2018</span>). <span class="cit-article-title">raster: Geographic Data Analysis and Modeling</span>. R package version 2.8-4. <a href="https://CRAN.R-project.org/package=raster">https://CRAN.R-project.org/package=raster</a></cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference  in text" id="ref-12">↵</a><div class="cit ref-cit ref-journal" id="cit-641183v2.12"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Hyland</span>,  <span class="cit-name-given-names">E.G.</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">K.W.</span> <span class="cit-name-surname">Huntington</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">N.D.</span> <span class="cit-name-surname">Sheldon</span></span>, and <span class="cit-auth"> <span class="cit-name-given-names">T.</span> <span class="cit-name-surname">Reichgelt</span></span>. <span class="cit-pub-date">2018</span>. <span class="cit-article-title">Temperature seasonality in the North American continental interior during the Early Eocene Climatic Optimum</span>. <abbr class="cit-jnl-abbrev">Climate of the Past</abbr> <span class="cit-vol">14</span>(<span class="cit-issue">10</span>): <span class="cit-fpage">1391</span>–<span class="cit-lpage">1404</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DClimate%2Bof%2Bthe%2BPast%26rft.volume%253D14%26rft.spage%253D1391%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference  in text" id="ref-13">↵</a><div class="cit ref-cit ref-journal" id="cit-641183v2.13"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Karger</span>,  <span class="cit-name-given-names">D.N.</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">O.</span> <span class="cit-name-surname">Conrad</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Böhner</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">T.</span> <span class="cit-name-surname">Kawohl</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">H.</span> <span class="cit-name-surname">Kreft</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">R.W.</span> <span class="cit-name-surname">Soria</span></span>-<span class="cit-auth"><span class="cit-name-surname">Auza</span>,  <span class="cit-name-given-names">N.</span></span><span class="cit-auth"> <span class="cit-name-given-names">E.</span> <span class="cit-name-surname">Zimmermann</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">H.P.</span> <span class="cit-name-surname">Linder</span></span>, and <span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Kessler</span></span>. <span class="cit-pub-date">2017</span>. <span class="cit-article-title">Climatologies at high resolution for the earth’s land surface areas</span>. <abbr class="cit-jnl-abbrev">Scientific Data</abbr> <span class="cit-vol">4</span>:<span class="cit-fpage">170122</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScientific%2BData%26rft.volume%253D4%26rft.spage%253D170122%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference  in text" id="ref-14">↵</a><div class="cit ref-cit ref-journal" id="cit-641183v2.14"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Maitner</span>,  <span class="cit-name-given-names">B.S.</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">B.</span> <span class="cit-name-surname">Boyle</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">N.</span> <span class="cit-name-surname">Casler</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">R.</span> <span class="cit-name-surname">Condit</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Donoghue</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">S.M.</span> <span class="cit-name-surname">Durán</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">D.</span> <span class="cit-name-surname">Guaderrama</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">C.E.</span> <span class="cit-name-surname">Hinchliff</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">P.M.</span> <span class="cit-name-surname">Jørgensen</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">N.J.</span> <span class="cit-name-surname">Kraft</span></span>, and <span class="cit-auth"> <span class="cit-name-given-names">B.</span> <span class="cit-name-surname">McGill</span></span>. <span class="cit-pub-date">2018</span>. <span class="cit-article-title">The bien r package: A tool to access the Botanical Information and Ecology Network (BIEN) database</span>. <abbr class="cit-jnl-abbrev">Methods in Ecology and Evolution</abbr> <span class="cit-vol">9</span>(<span class="cit-issue">2</span>):<span class="cit-fpage">373</span>–<span class="cit-lpage">379</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMethods%2Bin%2BEcology%2Band%2BEvolution%26rft.volume%253D9%26rft.spage%253D373%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference  in text" id="ref-15">↵</a><div class="cit ref-cit ref-journal" id="cit-641183v2.15"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Montade</span>,  <span class="cit-name-given-names">V.</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">O.</span> <span class="cit-name-surname">Peyron</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">C.</span> <span class="cit-name-surname">Favier</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">J.P</span> <span class="cit-name-surname">Francois</span></span>, and <span class="cit-auth"> <span class="cit-name-given-names">S.G.</span> <span class="cit-name-surname">Haberle</span></span>. <span class="cit-pub-date">2019</span>. <span class="cit-article-title">A pollen– climate calibration from western Patagonia for palaeoclimatic reconstructions</span>. <abbr class="cit-jnl-abbrev">Journal of Quaternary Science</abbr> <span class="cit-vol">34</span>(<span class="cit-issue">1</span>): <span class="cit-fpage">76</span>–<span class="cit-lpage">86</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BQuaternary%2BScience%26rft.volume%253D34%26rft.spage%253D76%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference  in text" id="ref-16">↵</a><div class="cit ref-cit ref-journal" id="cit-641183v2.16" data-doi="10.1016/S0031-0182(96)00154-X"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Mosbrugger</span>,  <span class="cit-name-given-names">V.</span></span> and <span class="cit-auth"> <span class="cit-name-given-names">T.</span> <span class="cit-name-surname">Utescher</span></span>. <span class="cit-pub-date">1997</span>. <span class="cit-article-title">The coexistence approach—a method for quantitative reconstructions of Tertiary terrestrial palaeoclimate data using plant fossils</span>. <abbr class="cit-jnl-abbrev">Palaeogeography, Palaeoclimatology, Palaeoecology</abbr> <span class="cit-vol">134</span>(<span class="cit-issue">1-4</span>):<span class="cit-fpage">61</span>–<span class="cit-lpage">86</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPalaeogeography%252C%2BPalaeoclimatology%252C%2BPalaeoecology%26rft.stitle%253DPalaeogeography%252C%2BPalaeoclimatology%252C%2BPalaeoecology%26rft.volume%253D134%26rft.issue%253D1-4%26rft.spage%253D61%26rft.epage%253D86%26rft.atitle%253DThe%2Bcoexistence%2Bapproach%253B%2Ba%2Bmethod%2Bfor%2Bquantitative%2Breconstructions%2Bof%2BTertiary%2Bterrestrial%2Bpalaeoclimate%2Bdata%2Busing%2Bplant%2Bfossils%26rft_id%253Dinfo%253Adoi%252F10.1016%252FS0031-0182%252896%252900154-X%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/S0031-0182(96)00154-X&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=1997064755&amp;link_type=GEOREF" class="cit-ref-sprinkles cit-ref-sprinkles-georef"><span>GeoRef</span></a><a href="/lookup/external-ref?access_num=A1997YK90800003&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference  in text" id="ref-17">↵</a><div class="cit ref-cit ref-journal" id="cit-641183v2.17"><div class="cit-metadata"><cite><span class="cit-auth cit-collab">R Core Team</span> (<span class="cit-pub-date">2018</span>). <span class="cit-article-title">R: A language and environment for statistical computing</span>. <abbr class="cit-jnl-abbrev">R Foundation for Statistical Computing, Vienna, Austria</abbr>. URL <a href="https://www.R-project.org/">https://www.R-project.org/</a>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference  in text" id="ref-18">↵</a><div class="cit ref-cit ref-journal" id="cit-641183v2.18"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Silverman</span>,  <span class="cit-name-given-names">BW.</span></span> <span class="cit-pub-date">1986</span>. <span class="cit-article-title">Density estimation for statistics and data analysis</span>. <abbr class="cit-jnl-abbrev">Routledge</abbr>. DOI: <a href="https://doi.org/10.1201/9781315140919">https://doi.org/10.1201/9781315140919</a></cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference  in text" id="ref-19">↵</a><div class="cit ref-cit ref-journal" id="cit-641183v2.19" data-doi="10.1016/j.quascirev.2012.07.003"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Thompson</span>,  <span class="cit-name-given-names">R.S.</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">K.H.</span> <span class="cit-name-surname">Anderson</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">R.T.</span> <span class="cit-name-surname">Pelltier</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">L.E.</span> <span class="cit-name-surname">Strickland</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">P.J.</span> <span class="cit-name-surname">Bartlein</span></span>, and <span class="cit-auth"> <span class="cit-name-given-names">S.L.</span> <span class="cit-name-surname">Shafer</span></span>. <span class="cit-pub-date">2012</span>. <span class="cit-article-title">Quantitative estimation of climatic parameters from vegetation data in North America by the mutual climatic range technique</span>. <abbr class="cit-jnl-abbrev">Quaternary Science Reviews</abbr> <span class="cit-vol">51</span>:<span class="cit-fpage">18</span>–<span class="cit-lpage">39</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DQuaternary%2BScience%2BReviews%26rft.stitle%253DQuaternary%2BScience%2BReviews%26rft.aulast%253DThompson%26rft.auinit1%253DR.%2BS.%26rft.volume%253D51%26rft.spage%253D18%26rft.epage%253D39%26rft.atitle%253DQuantitative%2Bestimation%2Bof%2Bclimatic%2Bparameters%2Bfrom%2Bvegetation%2Bdata%2Bin%2BNorth%2BAmerica%2Bby%2Bthe%2Bmutual%2Bclimatic%2Brange%2Btechnique%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.quascirev.2012.07.003%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.quascirev.2012.07.003&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=2013020637&amp;link_type=GEOREF" class="cit-ref-sprinkles cit-ref-sprinkles-georef"><span>GeoRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference  in text" id="ref-20">↵</a><div class="cit ref-cit ref-journal" id="cit-641183v2.20"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Title</span>,  <span class="cit-name-given-names">P.O.</span></span> and <span class="cit-auth"> <span class="cit-name-given-names">J.B.</span> <span class="cit-name-surname">Bemmels</span></span>. <span class="cit-pub-date">2018</span>. <span class="cit-article-title">ENVIREM: an expanded set of bioclimatic and topographic variables increases flexibility and improves performance of ecological niche modeling</span>. <abbr class="cit-jnl-abbrev">Ecography</abbr> <span class="cit-vol">41</span>(<span class="cit-issue">2</span>):<span class="cit-fpage">291</span>–<span class="cit-lpage">307</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEcography%26rft.volume%253D41%26rft.spage%253D291%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference  in text" id="ref-21">↵</a><div class="cit ref-cit ref-journal" id="cit-641183v2.21" data-doi="10.1016/j.palaeo.2014.05.031"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Utescher</span>,  <span class="cit-name-given-names">T.</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">A.A.</span> <span class="cit-name-surname">Bruch</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">B.</span> <span class="cit-name-surname">Erdei</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">L.</span> <span class="cit-name-surname">François</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">D.</span> <span class="cit-name-surname">Ivanov</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">F.M.B.</span> <span class="cit-name-surname">Jacques</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">A.K.</span> <span class="cit-name-surname">Kern</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">V.</span> <span class="cit-name-surname">Mosbrugger</span></span>, and <span class="cit-auth"> <span class="cit-name-given-names">R.A.</span> <span class="cit-name-surname">Spicer</span></span>. <span class="cit-pub-date">2014</span>. <span class="cit-article-title">The Coexistence Approach—theoretical background and practical considerations of using plant fossils for climate quantification</span>. <abbr class="cit-jnl-abbrev">Palaeogeography, Palaeoclimatology, Palaeoecology</abbr>, <span class="cit-vol">410</span>: <span class="cit-fpage">58</span>–<span class="cit-lpage">73</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPalaeogeography%252C%2BPalaeoclimatology%252C%2BPalaeoecology%26rft.stitle%253DPalaeogeography%252C%2BPalaeoclimatology%252C%2BPalaeoecology%26rft.aulast%253DUtescher%26rft.auinit1%253DT.%26rft.volume%253D410%26rft.spage%253D58%26rft.epage%253D73%26rft.atitle%253DThe%2Bcoexistence%2Bapproach%253B%2Btheoretical%2Bbackground%2Band%2Bpractical%2Bconsiderations%2Bof%2Busing%2Bplant%2Bfossils%2Bfor%2Bclimate%2Bquantification%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.palaeo.2014.05.031%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.palaeo.2014.05.031&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=2014071586&amp;link_type=GEOREF" class="cit-ref-sprinkles cit-ref-sprinkles-georef"><span>GeoRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference  in text" id="ref-22">↵</a><div class="cit ref-cit ref-journal" id="cit-641183v2.22"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Willard</span>,  <span class="cit-name-given-names">D.A.</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">T.H.</span> <span class="cit-name-surname">Donders</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">T.</span> <span class="cit-name-surname">Reichgelt</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">D.R.</span> <span class="cit-name-surname">Greenwood</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">F.</span> <span class="cit-name-surname">Sangiorgi</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">F.</span> <span class="cit-name-surname">Peterse</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">K.G.</span> <span class="cit-name-surname">Nierop</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">j.</span> <span class="cit-name-surname">Frieling</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">Schouten</span></span>, and <span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Sluijs.</span></span> <span class="cit-pub-date">2019</span>. <span class="cit-article-title">Arctic vegetation, temperature, and hydrology during Early Eocene transient global warming events</span>. <abbr class="cit-jnl-abbrev">Global and Planetary Change</abbr> <span class="cit-vol">178</span>:<span class="cit-fpage">139</span>–<span class="cit-lpage">152</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGlobal%2Band%2BPlanetary%2BChange%26rft.volume%253D178%26rft.spage%253D139%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-641183v2.23" data-doi="10.1139/cjes-2015-0165"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Zaborac-Reed</span>,  <span class="cit-name-given-names">S.J.</span></span> and <span class="cit-auth"> <span class="cit-name-given-names">E.B.</span> <span class="cit-name-surname">Leopold</span></span>. <span class="cit-pub-date">2016</span>. <span class="cit-article-title">Determining the paleoclimate and elevation of the late Eocene Florissant flora: support from the coexistence approach</span>. <abbr class="cit-jnl-abbrev">Canadian Journal of Earth Sciences</abbr> <span class="cit-vol">53</span>(<span class="cit-issue">6</span>): <span class="cit-fpage">565</span>–<span class="cit-lpage">573</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCanadian%2BJournal%2Bof%2BEarth%2BSciences%26rft.stitle%253DCan.%2BJ.%2BEarth%2BSci.%26rft.aulast%253DZaborac-Reed%26rft.auinit1%253DS.%2BJ.%26rft.volume%253D53%26rft.issue%253D6%26rft.spage%253D565%26rft.epage%253D573%26rft.atitle%253DDetermining%2Bthe%2Bpaleoclimate%2Band%2Belevation%2Bof%2Bthe%2Blate%2BEocene%2BFlorissant%2Bflora%253A%2Bsupport%2Bfrom%2Bthe%2Bcoexistence%2Bapproach%26rft_id%253Dinfo%253Adoi%252F10.1139%252Fcjes-2015-0165%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ3NjamVzIjtzOjU6InJlc2lkIjtzOjg6IjUzLzYvNTY1IjtzOjQ6ImF0b20iO3M6Mzc6Ii9iaW9yeGl2L2Vhcmx5LzIwMTkvMDUvMTgvNjQxMTgzLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.biorxiv.org/content/10.1101/641183v2.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-disqus-comment" >
  
      
  
  <div class="pane-content">
    <div id="disqus_thread"><noscript><p><a href="http://biorxivstage.disqus.com/?url=https%3A%2F%2Fwww.biorxiv.org%2Fcontent%2F10.1101%2F641183v2" class="" data-icon-position="" data-hide-link-title="0">View the discussion thread.</a></p></noscript></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/10.1101/639963v2" title="Binding partner regulation of Myosin VI: Loss of tumour-suppressor Dab2 leads to enhanced activity of nuclear myosin" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous</span></a></span><span class="pager-next"><a href="/content/10.1101/283929v2-0" title="Cell-type heterogeneity in adipose tissue is associated with complex traits and reveals disease-relevant cell-specific eQTLs" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    Posted&nbsp;May 18, 2019.  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-biorxiv-art-tools" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-biorxiv_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/10.1101/641183v2.full.pdf" target="_self" class="article-dl-pdf-link link-icon"><i class="icon-external-link-sign"></i> <span class="title">Download PDF</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/early/2019/05/18/641183.external-links" target="_self" class="link-icon"><i class="icon-file"></i> <span class="title">Data/Code</span></a>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-biorxiv-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="Email this Article" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-biorxiv_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/10.1101/641183v2.full" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for your interest in spreading the word about bioRxiv.</p><p>NOTE: Your email address is requested solely to identify you as the sender of this article.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/10.1101/641183v2" class="active" data-icon-position="" data-hide-link-title="0">cRacle: R Tools for Estimating Climate from Vegetation</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from bioRxiv
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would like to see this page from the bioRxiv website.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/739126" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-UvjcHHyYrShdxHz6xDe-5FUN7ujEvW7voifGP4oTyT0" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-biorxiv-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-biorxiv_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="739126" id="node739126--2" data-pisa="biorxiv;641183v2" data-pisa-master="biorxiv;641183" data-seqnum="739126" data-apath="/biorxiv/early/2019/05/18/641183.atom"><div  class="highwire-cite highwire-cite-highwire-article highwire-citation-biorxiv-article-pap-list clearfix" >

      <div class="highwire-cite-title" >
      <div class="highwire-cite-title">cRacle: R Tools for Estimating Climate from Vegetation</div>    </div>
  
      <div  class="highwire-cite-authors" ><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0"><span class="nlm-given-names">Robert S.</span> <span class="nlm-surname">Harbert</span></span>, <span class="highwire-citation-author" data-delta="1"><span class="nlm-given-names">Alex A.</span> <span class="nlm-surname">Baryiames</span></span></span></div>
  
      <div  class="highwire-cite-metadata" ><span  class="highwire-cite-metadata-journal highwire-cite-metadata">bioRxiv </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">641183; </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata"><span class="doi_label">doi:</span> https://doi.org/10.1101/641183 </span></div>
  
  
  </div>
</div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.biorxiv.org/content/10.1101/641183v2" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.biorxiv.org/content/10.1101/641183v2&amp;title=cRacle%3A%20R%20Tools%20for%20Estimating%20Climate%20from%20Vegetation" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.biorxiv.org/content/10.1101/641183v2&amp;title=cRacle%3A%20R%20Tools%20for%20Estimating%20Climate%20from%20Vegetation" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.biorxiv.org/content/10.1101/641183v2&amp;text=cRacle%3A%20R%20Tools%20for%20Estimating%20Climate%20from%20Vegetation" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.biorxiv.org/content/10.1101/641183v2&amp;title=cRacle%3A%20R%20Tools%20for%20Estimating%20Climate%20from%20Vegetation" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.biorxiv.org/content/10.1101/641183v2&amp;t=cRacle%3A%20R%20Tools%20for%20Estimating%20Climate%20from%20Vegetation" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.biorxiv.org/content/10.1101/641183v2&amp;title=cRacle%3A%20R%20Tools%20for%20Estimating%20Climate%20from%20Vegetation" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.biorxiv.org/content/10.1101/641183v2&amp;title=cRacle%3A%20R%20Tools%20for%20Estimating%20Climate%20from%20Vegetation" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-biorxiv-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title="Citation Tools"><i class="icon-globe"></i> <span class="title">Citation Tools</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-biorxiv_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="739126" id="node739126--3" data-pisa="biorxiv;641183v2" data-pisa-master="biorxiv;641183" data-seqnum="739126" data-apath="/biorxiv/early/2019/05/18/641183.atom"><div  class="highwire-cite highwire-cite-highwire-article highwire-citation-biorxiv-article-pap-list clearfix" >

      <div class="highwire-cite-title" >
      <div class="highwire-cite-title">cRacle: R Tools for Estimating Climate from Vegetation</div>    </div>
  
      <div  class="highwire-cite-authors" ><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0"><span class="nlm-given-names">Robert S.</span> <span class="nlm-surname">Harbert</span></span>, <span class="highwire-citation-author" data-delta="1"><span class="nlm-given-names">Alex A.</span> <span class="nlm-surname">Baryiames</span></span></span></div>
  
      <div  class="highwire-cite-metadata" ><span  class="highwire-cite-metadata-journal highwire-cite-metadata">bioRxiv </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">641183; </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata"><span class="doi_label">doi:</span> https://doi.org/10.1101/641183 </span></div>
  
  
  </div>
</div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage&amp;rft.epage&amp;rft.atitle=cRacle%3A%20R%20Tools%20for%20Estimating%20Climate%20from%20Vegetation&amp;rft.volume&amp;rft.issue&amp;rft.date=2019-01-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=bioRxiv&amp;rft.au=Harbert%2C+Robert+S.&amp;rft.au=Baryiames%2C+Alex+A."></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/739126/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/739126/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/739126/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/739126/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/739126/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/739126/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/739126/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/739126/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/739126/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/739126/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/739126/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/739126/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><div class="item-list"><ul><li class="first"><a href="http://twitter.com/share?url=https%3A//www.biorxiv.org/content/10.1101/641183v2&amp;count=horizontal&amp;via=&amp;text=cRacle%3A%20R%20Tools%20for%20Estimating%20Climate%20from%20Vegetation&amp;counturl=https%3A//www.biorxiv.org/content/10.1101/641183v2" class="twitter-share-button service-links-twitter-widget" id="twitter_widget" title="Tweet This" rel="nofollow" data-icon-position="" data-hide-link-title="0"><span class="element-invisible">Tweet Widget</span></a></li><li><a href="http://www.facebook.com/plugins/like.php?href=https%3A//www.biorxiv.org/content/10.1101/641183v2&amp;layout=button_count&amp;show_faces=false&amp;action=like&amp;colorscheme=light&amp;width=100&amp;height=21&amp;font=&amp;locale=" id="facebook_like" title="I Like it" class="service-links-facebook-like" rel="nofollow" data-icon-position="" data-hide-link-title="0"><span class="element-invisible">Facebook Like</span></a></li><li class="last"><a href="https://www.biorxiv.org/content/10.1101/641183v2" id="google_plus_one" title="Plus it" class="service-links-google-plus-one" rel="nofollow" data-icon-position="" data-hide-link-title="0"><span class="element-invisible">Google Plus One</span></a></li></ul></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-collections" >
  
        <h2 class="pane-title">Subject Area</h2>
    
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-collections"><div class="highwire-list"><ul class="highwire-article-collection-term-list"><li class="first last odd"><span class="highwire-article-collection-term"><a href="/collection/bioinformatics" class="highlight" data-icon-position="" data-hide-link-title="0">Bioinformatics<i class="icon-caret-right"></i>
</a></span></li></ul></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-biorxiv-subject-collections block-style-col2" >
  
      
  
  <div class="pane-content">
    <div class="panel-flexible panels-flexible-8 clearfix" id="mini-panel-biorxiv_subject_collections">
<div class="panel-flexible-inside panels-flexible-8-inside">
<div class="panels-flexible-region panels-flexible-region-8-center panels-flexible-region-first panels-flexible-region-last">
  <div class="inside panels-flexible-region-inside panels-flexible-region-8-center-inside panels-flexible-region-inside-first panels-flexible-region-inside-last">
<div class="panel-pane pane-snippet" >
  
      
  
  <div class="pane-content">
    <div class="snippet biorxiv-subject-areas-table-title" id="biorxiv-subject-areas-table-title">
  
      
  <div class="snippet-content">
    <b>Subject Areas</b>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-snippet" >
  
      
  
  <div class="pane-content">
    <div class="snippet biorxiv-subject-areas-view-papers" id="biorxiv-subject-areas-view-papers">
  
      
  <div class="snippet-content">
    <a href="/content/early/recent"><strong>All Articles</strong></a>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-subject-collections" >
  
      
  
  <div class="pane-content">
    <ul id="collection" class="collection highwire-list-expand"><li class="outer collection depth-2  child first"><div class = "data-wrapper"><a href="/collection/animal-behavior-and-cognition" class="" data-icon-position="" data-hide-link-title="0">Animal Behavior and Cognition</a> <span class = "article-count">(1662)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/biochemistry" class="" data-icon-position="" data-hide-link-title="0">Biochemistry</a> <span class = "article-count">(2789)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/bioengineering" class="" data-icon-position="" data-hide-link-title="0">Bioengineering</a> <span class = "article-count">(1946)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/bioinformatics" class="" data-icon-position="" data-hide-link-title="0">Bioinformatics</a> <span class = "article-count">(10400)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/biophysics" class="" data-icon-position="" data-hide-link-title="0">Biophysics</a> <span class = "article-count">(4243)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/cancer-biology" class="" data-icon-position="" data-hide-link-title="0">Cancer Biology</a> <span class = "article-count">(3272)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/cell-biology" class="" data-icon-position="" data-hide-link-title="0">Cell Biology</a> <span class = "article-count">(4641)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/clinical-trials" class="" data-icon-position="" data-hide-link-title="0">Clinical Trials</a> <span class = "article-count">(135)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/developmental-biology" class="" data-icon-position="" data-hide-link-title="0">Developmental Biology</a> <span class = "article-count">(2889)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/ecology" class="" data-icon-position="" data-hide-link-title="0">Ecology</a> <span class = "article-count">(4544)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/epidemiology" class="" data-icon-position="" data-hide-link-title="0">Epidemiology</a> <span class = "article-count">(2041)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/evolutionary-biology" class="" data-icon-position="" data-hide-link-title="0">Evolutionary Biology</a> <span class = "article-count">(7288)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/genetics" class="" data-icon-position="" data-hide-link-title="0">Genetics</a> <span class = "article-count">(5540)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/genomics" class="" data-icon-position="" data-hide-link-title="0">Genomics</a> <span class = "article-count">(6893)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/immunology" class="" data-icon-position="" data-hide-link-title="0">Immunology</a> <span class = "article-count">(2434)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/microbiology" class="" data-icon-position="" data-hide-link-title="0">Microbiology</a> <span class = "article-count">(7616)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/molecular-biology" class="" data-icon-position="" data-hide-link-title="0">Molecular Biology</a> <span class = "article-count">(3051)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/neuroscience" class="" data-icon-position="" data-hide-link-title="0">Neuroscience</a> <span class = "article-count">(18885)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/paleontology" class="" data-icon-position="" data-hide-link-title="0">Paleontology</a> <span class = "article-count">(142)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/pathology" class="" data-icon-position="" data-hide-link-title="0">Pathology</a> <span class = "article-count">(477)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/pharmacology-and-toxicology" class="" data-icon-position="" data-hide-link-title="0">Pharmacology and Toxicology</a> <span class = "article-count">(791)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/physiology" class="" data-icon-position="" data-hide-link-title="0">Physiology</a> <span class = "article-count">(1178)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/plant-biology" class="" data-icon-position="" data-hide-link-title="0">Plant Biology</a> <span class = "article-count">(2771)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/scientific-communication-and-education" class="" data-icon-position="" data-hide-link-title="0">Scientific Communication and Education</a> <span class = "article-count">(686)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/synthetic-biology" class="" data-icon-position="" data-hide-link-title="0">Synthetic Biology</a> <span class = "article-count">(905)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/systems-biology" class="" data-icon-position="" data-hide-link-title="0">Systems Biology</a> <span class = "article-count">(2881)</span></div></li>
<li class="outer collection depth-2  child last"><div class = "data-wrapper"><a href="/collection/zoology" class="" data-icon-position="" data-hide-link-title="0">Zoology</a> <span class = "article-count">(472)</span></div></li>
</ul>  </div>

  
  </div>
  </div>
</div>
</div>
</div>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__VNH5GD8Zz7g6_hGCOZjVjdMndxxC6naiExSSJKX3k_A__kzmEdtbqCsJx5Z9yg2Qy0PptB__ufXm-TxUOl0KZzUw__QJacVmveFyLIBSPkp21xL0ZvuDW9DMBrD2HB1P4Ry5I.js"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__QJacVmveFyLIBSPkp21xL0ZvuDW9DMBrD2HB1P4Ry5I.js" defer="defer"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__N7ERJBYsOWyRnJgyoM125_Aiez2MOJGaUofG1JdWWBg__bLlcwhzZhT-IwoQ_dvm1_xPxU7E7gecFLC6Fyeqtv8s__QJacVmveFyLIBSPkp21xL0ZvuDW9DMBrD2HB1P4Ry5I.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
function euCookieComplianceLoadScripts() {}
//--><!]]>
</script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var eu_cookie_compliance_cookie_name = "";
//--><!]]>
</script>
<script type="text/javascript" src="//d33xdlntwy0kbs.cloudfront.net/cshl_custom.js"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__BmqjBnkz3MgYCAoc25s1lDRMEjLhC3mEPVonUFIHi08__Unwv5-ZIuHBfFwytsjEx1niBVJ7n1T4lPws7VrkHXM4__QJacVmveFyLIBSPkp21xL0ZvuDW9DMBrD2HB1P4Ry5I.js"></script>
  </body>
</html>
