<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<link rel="dns-prefetch" href="//d33xdlntwy0kbs.cloudfront.net" />
<link rel="shortcut icon" href="https://www.biorxiv.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<meta name="type" content="article" />
<meta name="category" content="article" />
<meta name="HW.identifier" content="/biorxiv/early/2019/02/15/549675.atom" />
<meta name="HW.pisa" content="biorxiv;549675v1" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="Climate change is predicted to disrupt patterns of local adaptation in wild and cultivated maize" />
<meta name="DC.Identifier" content="10.1101/549675" />
<meta name="DC.Date" content="2019-02-15" />
<meta name="DC.Publisher" content="Cold Spring Harbor Laboratory" />
<meta name="DC.Rights" content="© 2019, Posted by Cold Spring Harbor Laboratory. This pre-print is available under a Creative Commons License (Attribution-NonCommercial-NoDerivs 4.0 International), CC BY-NC-ND 4.0, as described at http://creativecommons.org/licenses/by-nc-nd/4.0/" />
<meta name="DC.AccessRights" content="restricted" />
<meta name="DC.Description" content="Climate change is one of the most important threats to biodiversity and crop sustainability. The impact of climate change is often evaluated on the basis of expected changes in species’ geographical distributions. Genomic diversity, local adaptation, and migration are seldom integrated into projections of species’ responses to climate change. Here we predict that climate change will impact populations of two wild relatives of maize, the teosintes Zea mays ssp. mexicana and Z. mays ssp. parviglumis , by altering patterns of local adaptation and decreasing migration probabilities. These alterations appear to be geographically heterogeneous across populations, suggesting that the possible impacts of climate change will vary considerably among populations. This in spite that most populations exhibit high levels of genetic diversity and are predicted to lie within future suitable areas. The heterogeneous patterns of local adaptation uncovered in teosintes are also evident across maize landraces, which suggests that climate change may give way to future maladaptation of several landraces within currently cultivated areas, possibly leading to increased chances of production shocks under increased temperatures. The predicted alterations to habitat distribution, migration potential, and patterns of local adaptation in wild and cultivated maize, raises a red flag for the future of populations. This underscores the need for continued integration of agronomical practices, genomic data, and climate models to better understand the impacts of a rapidly changing climate on cultivated and wild species." />
<meta name="DC.Contributor" content="Jonás A Aguirre-Liguori" />
<meta name="DC.Contributor" content="Santiago Ramírez-Barahona" />
<meta name="DC.Contributor" content="Peter Tiffin" />
<meta name="DC.Contributor" content="Luis E Eguiarte" />
<meta name="article:published_time" content="2019-02-15" />
<meta name="article:section" content="New Results" />
<meta name="citation_title" content="Climate change is predicted to disrupt patterns of local adaptation in wild and cultivated maize" />
<meta name="citation_abstract" lang="en" content="&lt;h3&gt;Abstract&lt;/h3&gt;
&lt;p&gt;Climate change is one of the most important threats to biodiversity and crop sustainability. The impact of climate change is often evaluated on the basis of expected changes in species’ geographical distributions. Genomic diversity, local adaptation, and migration are seldom integrated into projections of species’ responses to climate change. Here we predict that climate change will impact populations of two wild relatives of maize, the teosintes &lt;i&gt;Zea mays&lt;/i&gt; ssp. &lt;i&gt;mexicana&lt;/i&gt; and &lt;i&gt;Z. mays&lt;/i&gt; ssp. &lt;i&gt;parviglumis&lt;/i&gt;, by altering patterns of local adaptation and decreasing migration probabilities. These alterations appear to be geographically heterogeneous across populations, suggesting that the possible impacts of climate change will vary considerably among populations. This in spite that most populations exhibit high levels of genetic diversity and are predicted to lie within future suitable areas. The heterogeneous patterns of local adaptation uncovered in teosintes are also evident across maize landraces, which suggests that climate change may give way to future maladaptation of several landraces within currently cultivated areas, possibly leading to increased chances of production shocks under increased temperatures. The predicted alterations to habitat distribution, migration potential, and patterns of local adaptation in wild and cultivated maize, raises a red flag for the future of populations. This underscores the need for continued integration of agronomical practices, genomic data, and climate models to better understand the impacts of a rapidly changing climate on cultivated and wild species.&lt;/p&gt;" />
<meta name="citation_journal_title" content="bioRxiv" />
<meta name="citation_publisher" content="Cold Spring Harbor Laboratory" />
<meta name="citation_publication_date" content="2019/01/01" />
<meta name="citation_mjid" content="biorxiv;549675v1" />
<meta name="citation_id" content="549675v1" />
<meta name="citation_public_url" content="https://www.biorxiv.org/content/10.1101/549675v1" />
<meta name="citation_abstract_html_url" content="https://www.biorxiv.org/content/10.1101/549675v1.abstract" />
<meta name="citation_full_html_url" content="https://www.biorxiv.org/content/10.1101/549675v1.full" />
<meta name="citation_pdf_url" content="https://www.biorxiv.org/content/biorxiv/early/2019/02/15/549675.full.pdf" />
<meta name="citation_doi" content="10.1101/549675" />
<meta name="citation_num_pages" content="29" />
<meta name="citation_article_type" content="Article" />
<meta name="citation_section" content="New Results" />
<meta name="citation_firstpage" content="549675" />
<meta name="citation_author" content="Jonás A Aguirre-Liguori" />
<meta name="citation_author_institution" content="Laboratorio de Evolución Molecular y Experimental, Departamento de Ecología Evolutiva, Instituto de Ecología. Universidad Nacional Autónoma de México" />
<meta name="citation_author" content="Santiago Ramírez-Barahona" />
<meta name="citation_author_institution" content="Departamento de Botánica, Instituto de Biología, Universidad Nacional Autónoma de México" />
<meta name="citation_author_email" content="santiago.ramirez@ib.unam.mx" />
<meta name="citation_author_email" content="fruns@unam.mx" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0003-3999-6952" />
<meta name="citation_author" content="Peter Tiffin" />
<meta name="citation_author_institution" content="Department of Plant and Microbial Biology, University of Minnesota" />
<meta name="citation_author" content="Luis E Eguiarte" />
<meta name="citation_author_institution" content="Laboratorio de Evolución Molecular y Experimental, Departamento de Ecología Evolutiva, Instituto de Ecología. Universidad Nacional Autónoma de México" />
<meta name="citation_author_email" content="santiago.ramirez@ib.unam.mx" />
<meta name="citation_author_email" content="fruns@unam.mx" />
<meta name="citation_reference" content="Allendorf FW, Hohenlohe PA, Luikart G. 2010. Genomics and the future of conservation genetics. Nat Rev Genet 11: 697–709." />
<meta name="citation_reference" content="Dai A. 2012. Increasing drought under global warming in observations and models. Nat Clim Change 1633: 52–57." />
<meta name="citation_reference" content="Pauls SU, Nowak C, Bálint M, Pfenninger M. 2012. The impact of global climate change on genetic diversity within populations and species. Mol Ecol 22: 925–946." />
<meta name="citation_reference" content="Hellin J, Bellon MR, Hearne SJ. 2014. Maize landraces and adaptation to climate change in Mexico. J Crop Improv 28: 482–501." />
<meta name="citation_reference" content="Schafer ABA, et al. 2015. Genomics and the challenging translation into conservation practice. Trends Ecol Evol 30: 78–87." />
<meta name="citation_reference" content="Pacifici M, et al. 2017. Species’ traits influenced their response to recent climate change. Nat Clim Change 3223: 1–4." />
<meta name="citation_reference" content="Tigcheelar M, Battisti DS, Naylor RL, Ray DK. 2018. Future warming increases probability of globally synchronized maize production shocks. Proc Natl Acad Sci 115: 6644–6649." />
<meta name="citation_reference" content="Palmgren MG, et al. 2014. Are we ready for back-to-nature crop breeding□? Trends Plant Sci. 20: 155–164." />
<meta name="citation_reference" content="Warschefsky E, Penmetsa RV, Cook DR, Wettberg EJ. 2014. Back to the wilds: Tapping evolutionary adaptations for resilient crops through systematic hybridization with crop wild relatives. Am J Bot 101: 1791–1800." />
<meta name="citation_reference" content="Aguirre-Liguori JA, Aguirre-planter E, Eguiarte LE. 2016. Genetics and ecology of wild and cultivated maize: domestication and introgression. In: Ethnobotany of Mexico. New York: Springer, pp 403–416" />
<meta name="citation_reference" content="Zhang H, Mittal N, Leamy LJ, Barazani O, Song BH. 2016. Back into the wild — Apply untapped genetic diversity of wild relatives for crop improvement. Evol Appl 10: 5–24." />
<meta name="citation_reference" content="Dempewolf H, et al. 2017. Past and future use of wild relatives in crop breeding. Crop Sci 57: 1070–1082." />
<meta name="citation_reference" content="Matsuoka Y, et al. 2002. A single domestication for maize shown by multilocusmicrosatellite genotyping. Proc Natl Acad Sci 99: 6080–6084." />
<meta name="citation_reference" content="CONABIO. 2011. Base de datos del proyecto global “Recopilación, generación, actualización y análisis de información acerca de la diversidad genética de maíces y sus parientes silvestres en México”. Comisión Nacional para el Conocimiento y Uso de la Biodiversidad. México, DF." />
<meta name="citation_reference" content="Sánchez GJJ. 2011. Diversidad del maíz y teocintle. Informe preparado para el proyecto global “Recopilación, generación, actualización y análisis de información acerca de la diversidad genética de maíces y sus parientes silvestres en México” de la Comisión Nacional para el Conocimiento y Uso de la Biodiversidad. Comisión Nacional para el Conocimiento y Uso de la Biodiversidad. México, DF." />
<meta name="citation_reference" content="Hufford MB, Martínez-Meyer E, Gaut BS, Eguiarte LE, Tenaillon MI. 2012. Inferences from the historical distribution of wild and domesticated maize provide ecological and evolutionary insight. PLoS ONE 11: e47659." />
<meta name="citation_reference" content="Sánchez González JJ, et al. 2018. Ecogeography of teosinte. PLoS ONE 13: e0192676." />
<meta name="citation_reference" content="van Heerwaarden J, et al. 2011. Genetic signals of origin, spread, and introgression in a large sample of maize landraces. Proc Natl Acad Sci 108: 1088–1092." />
<meta name="citation_reference" content="Ureta C, Martínez-Meyer E, Perales HR, Álvarez-Buylla ER. 2012. Projecting the effects of climate change on the distribution of maize races and their wild relatives in Mexico. Glob Change Biol 18: 1073–108." />
<meta name="citation_reference" content="Baltazar BM, Sanchez-Gonzalez JJ, de la Cruz-Larios L, Schoper J B. 2005. Pollination between maize and teosinte: an important determinant of gene flow in Mexico. Theor Appl Genet 110: 519–526." />
<meta name="citation_reference" content="Kato TAY, Sánchez JJG. 2002. Introgression of chromosome knobs from Zea diploperennis into maize. Maydica 47: 33–50." />
<meta name="citation_reference" content="Warburton ML, et al. 2011. Gene flow among different teosinte taxa and into the domesticated maize gene pool. Genet Resour Crop Ev 58:1243–1261." />
<meta name="citation_reference" content="Wilkes HG. 2006. Urgent notice to all maize researchers: disappearance and extinction of the last wild teosinte populations is more than half completed. A modest proposal for teosinte evolution and conservation in situ: the Balsas, Guerrero, Mexico. Maydica 52:49–58" />
<meta name="citation_reference" content="Pyhäjärvi T, Hufford MB, Mezmouk S, Ross-Ibarra J. 2013. Complex patterns of local adaptation in teosinte. Genome Biol Evol 5: 1594–1609." />
<meta name="citation_reference" content="Aguirre-Liguori JA, et al. 2017. Connecting genomic patterns of local adaptation and niche suitability in teosintes. Mol Ecol 26: 4226–4240." />
<meta name="citation_reference" content="Fustier M-A, et al. 2017. Signatures of local adaptation in lowland and highland teosintes from whole genome sequencing of pooled samples. Mol Ecol 26: 2738–D2756." />
<meta name="citation_reference" content="Peterson AT, et al. 2011. Ecological niches and geographic distributions. Princeton University Press, New Jersey." />
<meta name="citation_reference" content="Pagel J, Schurr FM. 2012. Forecasting species ranges by statistical estimation of ecological niches and spatial populations dynamics. Glob Ecol Biogeogr 21: 293–304." />
<meta name="citation_reference" content="Gotelli NJ, Stanton-Geddes J. 2015. Climate change, genetic markers and species distribution modelling. J Biogeogr 42: 1577–1585." />
<meta name="citation_reference" content="Fitzpatrick MC, Keller SR. 2015. Ecological genomics meets community-level modelling of biodiversity: mapping the genomic landscape of current and future environmental adaptation. Ecol Lett 18: 1–16." />
<meta name="citation_reference" content="Peterson ML, Doak DF, Morris WF. 2019. Incorporating local adaptation into forecasts of species’ distribution and abundance under climate change. Global Change Biol early view: https://doi.org/10.1111/gcb.14562." />
<meta name="citation_reference" content="Exposito-Alonso M, et al. 2017. Genomic basis and evolutionary potential for extreme drought adaptation in Arabidopsis thaliana. Nat Ecol Evol 2: 352–358." />
<meta name="citation_reference" content="Bay RA, et al. 2018. Genomic signals of selection predict climate-driven populations declines in a migratory bird. Science 359: 83–86." />
<meta name="citation_reference" content="Phillips SJ, Anderson RP, Schapire RE. 2006. Maximum entropy modeling of species geographic distributions. Ecol Model 190: 231–259." />
<meta name="citation_reference" content="Hijmans RJ, Cameron SE, Parra JL, Jones PG, Jarvis A 2005. Very high-resolution interpolated climate surfaces for global land areas. Int J Climatology, 25: 1965–1978." />
<meta name="citation_reference" content="Villemereuil P, Gaggiotti OE. 2015. A new FST-based method to uncover local adaptation using environmental variables. Methods Ecol Evol 6: 1248–1258." />
<meta name="citation_reference" content="Coop G, Witonsky D, Di Rienzo A, Pritchard JK. 2010. Using environmental correlations to identify loci underlying local adaptation. Genetics 185: 1411–1423." />
<meta name="citation_reference" content="Jombart T. 2008. adegenet: a R package for the multivariate analysis of genetic markers. Bioinformatics 24: 1403–1405." />
<meta name="citation_reference" content="R Core Team. 2018. R: A language and environment for statistical computing. R Foundation for Statistical Computing, Vienna, Austria." />
<meta name="citation_reference" content="Goudet J. 2005. Hierfstat, a package for R to compute and test variance components and F-statistics. Mol Ecol Notes 5: 184–186." />
<meta name="citation_reference" content="Ellis N, Smith SJ, Pitcher CR. 2012. Calculating importance gradients on physical predictors. Ecology 93: 156–168." />
<meta name="citation_reference" content="Hijmans RJ. 2017. raster: Geographic Data Analysis and Modeling. R package version 2.6–7. https://CRAN.R-project.org/package=raster" />
<meta name="citation_reference" content="McRae BH, Dickson BG, Keitt T. 2008. Using circuit theory to model connectivity in ecology, evolution, and conservation. Ecology 89: 2712–2724." />
<meta name="citation_reference" content="Arteaga MC, et al. 2016. Genomic variation in recently collected maize landraces from Mexico. Genomics Data 7: 38–45." />
<meta name="citation_reference" content="Hoffmann AA, Sgrò CM. 2011. Climate change and evolutionary adaptation. Nature 470: 479–485." />
<meta name="citation_reference" content="Siol M, Wright SI, Barrett SC. 2010. The population genomics of plant adaptation. New Phytol 188: 313–332." />
<meta name="citation_reference" content="Frankham R. 2010. Challenges and opportunities of genetic approaches to biological conservation. Biol Conserv 143: 1919–1927." />
<meta name="citation_reference" content="Jezkova T, Wiens JJ. 2016. Rates of change in climatic niches in plant and animal populations are much slower than projected climate change. Proc Royal Soc B 283: 20162104." />
<meta name="citation_reference" content="Ackerly DD, et al. 2010. The geography of climate change: implications for conservation biogeography. Divers Distrib 16:476–487." />
<meta name="citation_reference" content="Manel S, Schwartz MK, Luikart G, Taberlet P. 2003. Landscape genetics: combining landscape ecology and population genetics. Trends Ecol Evol 18: 189–197." />
<meta name="citation_reference" content="Loarie SR, et al. 2009. The velocity of climate change. Nature 462: 1052–1055." />
<meta name="citation_reference" content="Wang L, Yang A, He C, Qu M, Zhang J. 2008. Creation of new maize germplasm using alien introgression from Zea mays ssp. mexicana. Euphytica 164: 789–801." />
<meta name="citation_reference" content="Moreno-Letelier A, et al. 2017. Was maize domesticated in the Balsas Basin? Complex patterns of genetic divergence, gene flow and ancestral introgressions among Zea subspecies suggest an alternative scenario. bioRxiv: 239707." />
<meta name="citation_reference" content="Yoder JB, et al. 2014. Genomic signature of adaptation to climate in Medicago truncatula. Genetics 196: 1263–1275." />
<meta name="citation_reference" content="Tiffin P, Ross-Ibarra J. 2014. Advances and limits of using population genetics to understand local adaptation. Trends Ecol Evol 29: 673–680." />
<meta name="citation_reference" content="Kawecki TJ, Ebert D. 2004. Conceptual issues in local adaptation. Ecol Lett 7: 1225–1241." />
<meta name="citation_reference" content="Yeaman S. 2015. Local adaptation by alleles of small effect. Am Nat 186: S74–S89." />
<meta name="citation_reference" content="Fitzpatrick MC, Keller SR, Lotterhos KE. 2018. Comment on “Genomic signals of selection predict climate-driven population declines in a migratory bird”. Science 361: eaat7279." />
<meta name="twitter:title" content="Climate change is predicted to disrupt patterns of local adaptation in wild and cultivated maize" />
<meta name="twitter:site" content="@biorxivpreprint" />
<meta name="twitter:card" content="summary" />
<meta name="twitter:image" content="https://www.biorxiv.org/sites/default/files/images/biorxiv_logo_homepage7-5-small.png" />
<meta name="twitter:description" content="Climate change is one of the most important threats to biodiversity and crop sustainability. The impact of climate change is often evaluated on the basis of expected changes in species’ geographical distributions. Genomic diversity, local adaptation, and migration are seldom integrated into projections of species’ responses to climate change. Here we predict that climate change will impact populations of two wild relatives of maize, the teosintes Zea mays ssp. mexicana and Z. mays ssp. parviglumis , by altering patterns of local adaptation and decreasing migration probabilities. These alterations appear to be geographically heterogeneous across populations, suggesting that the possible impacts of climate change will vary considerably among populations. This in spite that most populations exhibit high levels of genetic diversity and are predicted to lie within future suitable areas. The heterogeneous patterns of local adaptation uncovered in teosintes are also evident across maize landraces, which suggests that climate change may give way to future maladaptation of several landraces within currently cultivated areas, possibly leading to increased chances of production shocks under increased temperatures. The predicted alterations to habitat distribution, migration potential, and patterns of local adaptation in wild and cultivated maize, raises a red flag for the future of populations. This underscores the need for continued integration of agronomical practices, genomic data, and climate models to better understand the impacts of a rapidly changing climate on cultivated and wild species." />
<meta name="og-title" property="og:title" content="Climate change is predicted to disrupt patterns of local adaptation in wild and cultivated maize" />
<meta name="og-url" property="og:url" content="https://www.biorxiv.org/content/10.1101/549675v1" />
<meta name="og-site-name" property="og:site_name" content="bioRxiv" />
<meta name="og-description" property="og:description" content="Climate change is one of the most important threats to biodiversity and crop sustainability. The impact of climate change is often evaluated on the basis of expected changes in species’ geographical distributions. Genomic diversity, local adaptation, and migration are seldom integrated into projections of species’ responses to climate change. Here we predict that climate change will impact populations of two wild relatives of maize, the teosintes Zea mays ssp. mexicana and Z. mays ssp. parviglumis , by altering patterns of local adaptation and decreasing migration probabilities. These alterations appear to be geographically heterogeneous across populations, suggesting that the possible impacts of climate change will vary considerably among populations. This in spite that most populations exhibit high levels of genetic diversity and are predicted to lie within future suitable areas. The heterogeneous patterns of local adaptation uncovered in teosintes are also evident across maize landraces, which suggests that climate change may give way to future maladaptation of several landraces within currently cultivated areas, possibly leading to increased chances of production shocks under increased temperatures. The predicted alterations to habitat distribution, migration potential, and patterns of local adaptation in wild and cultivated maize, raises a red flag for the future of populations. This underscores the need for continued integration of agronomical practices, genomic data, and climate models to better understand the impacts of a rapidly changing climate on cultivated and wild species." />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.biorxiv.org/sites/default/files/images/biorxiv_logo_homepage7-5-small.png" />
<meta name="citation_date" content="2019-02-15" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/10.1101/549675v1.full.pdf" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/10.1101/549675v1.full.txt" />
<link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/10.1101/549675v1.ppt" />
<meta name="description" content="bioRxiv - the preprint server for biology, operated by Cold Spring Harbor Laboratory, a research and educational institution" />
<meta name="generator" content="Drupal 7 (http://drupal.org)" />
<link rel="canonical" href="https://www.biorxiv.org/content/10.1101/549675v1" />
<link rel="shortlink" href="https://www.biorxiv.org/node/176518" />
    <title>Climate change is predicted to disrupt patterns of local adaptation in wild and cultivated maize | bioRxiv</title>  
    <link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__jMRAK66KMC1e4TQlwUNn3KiWVDC5AjueUAYEm1xBY_U__KSUjdT4jdcJ4qJz6fKY1K9WkYh1a5EcaWZxt_-zbTis__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q8002p" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__icrP-Y0qw2NP73rJxxkUtrJta0z3VLAABtNjfSdEghA__rlN3rx0wOoswGNHZMeZM96bNn-3cLWHXTztGDoThNlw__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__q48SDxVDWut_cH6dTPyQe7HjADuSLLmJVFztuiJEFrU__J9zYrFgVU9w_9DRO_7M268JyEPb2If_fHKWgrFly-Tw__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<style type="text/css" media="all">
/* <![CDATA[ */
#sliding-popup.sliding-popup-bottom,#sliding-popup.sliding-popup-bottom .eu-cookie-withdraw-banner,.eu-cookie-withdraw-tab{background:gray}#sliding-popup.sliding-popup-bottom.eu-cookie-withdraw-wrapper{background:transparent}#sliding-popup .popup-content #popup-text h1,#sliding-popup .popup-content #popup-text h2,#sliding-popup .popup-content #popup-text h3,#sliding-popup .popup-content #popup-text p,.eu-cookie-compliance-secondary-button,.eu-cookie-withdraw-tab{color:#fff !important}.eu-cookie-withdraw-tab{border-color:#fff}.eu-cookie-compliance-more-button{color:#fff !important}
/* ]]> */
</style>

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__B6tGdsQSt1vT8d8j-19-oS4FKJe83RrSNBkjPs9jfos__6VkE5WcQmmQpSVTQx-xS2-zn_nlwFlDm-EN1HLJNjrA__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__78WRxe8YsWkYp6RcQ_VwTjFG__x9mi_NTPqoxAwFUsA__F3sVKlGyHKCaAPI_WdiHD7l_-05B6b9bLdUwzi1me90__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__dAeSAHswzRXxn-2n4D6zNPmcZDmwaqqLzE-2ab_8jKw__f_9pwrTE8mmshoJRfurwkQ36QonD_Jiwbop2sscnkrE__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://d33xdlntwy0kbs.cloudfront.net/cshl_custom.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/all/libraries/modernizr/modernizr.min.js?q8002p"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__yOhrJUBmMpN-jsuQlJYmS8ovkQU6I6KInr295HrSTb0__c--zHhbuOWmkdWkJYXiNWDmkpDRPesmbBjGlQcyqBOY__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var _prum=[['id', '52e95df3abe53d2e33000000'], ['mark', 'firstbyte',
      (new Date()).getTime()]]; (function() {
      var s=document.getElementsByTagName('script')[0],
      p=document.createElement('script');
      p.async='async'; p.src='//rum-static.pingdom.net/prum.min.js';s.parentNode.insertBefore(p,s);})();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js___KyvuEvfmewWxdOLBXTYsSUo8Pfm_DijV6fWtGvSL_k__HSKkTgIr7nDcdvQYewipf6v4ak6ZLUOQkwPj44wKouA__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__3uCzzYE5jr6ly9nvsM8PRBRkhh4p0NWCMDja4EKBy4M__dmGGVYcB8LhnbR2iAmVW0ZyXI6QuFVYFRORa5XxglXs__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__0zdUME0GLauYYyEch2tGPydIw-IG7Uizip0Qb4wUsig__3RgszrCgk4bOnTzo4vaQ-hrWzaG6oD19l-R9CQ1YQ-o__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__W6wAjopaALytcBam7oT3WhEzllkWaB9R-8hazHyvShM__rK5YBIqBy9qt0di0Sn4DriBx9TJupdmv6uGmLJyZvDs__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-45638026-1", {"cookieDomain":"auto"});ga("set", "page", location.pathname + location.search + location.hash);ga("send", "pageview");ga('create', 'UA-189672-38', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__1PtAOZ8x7l6QAi8D7X6Kj_Y7HcUOFzvWRvvYHUcCT98__U8IB92u_CnnGwJ-X_xaUa9lzsC8OzUfN0WkK2JQfirg__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings,{"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"4NIZyW_IKXmXGLWcI23vhBWd9NgWM16P-K4oEJ40ZLc","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"modules\/comment\/comment.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"modules\/forum\/forum.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"sites\/default\/modules\/biorxiv\/css\/biorxiv.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/flexible\/flexible.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_collection\/css\/highwire_subject_collection.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire.highwire-list-expand.css":1,"public:\/\/ctools\/css\/bab59090150caf0c64e27afa1b36eef1.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/contrib\/eu_cookie_compliance\/css\/eu_cookie_compliance.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"0":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_biorxiv_styles\/css\/biorxiv-styles.css.less":1,"https:\/\/d33xdlntwy0kbs.cloudfront.net\/cshl_custom.css":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_figures.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"http:\/\/d33xdlntwy0kbs.cloudfront.net\/cshl_custom.js":1,"sites\/all\/modules\/contrib\/eu_cookie_compliance\/js\/eu_cookie_compliance.js":1,"sites\/default\/modules\/jnl_biorxiv_styles\/js\/adv-search-collection-toggle.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/eu_cookie_compliance\/js\/jquery.cookie-1.4.1.min.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/default\/modules\/jnl_biorxiv_styles\/js\/biorxiv-scripts.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_nps_survey\/js\/nps_survey.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"sites\/default\/modules\/contrib\/disqus\/disqus.js":1,"misc\/textarea.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/contrib\/service_links\/js\/twitter_button.js":1,"sites\/all\/modules\/contrib\/service_links\/js\/facebook_like.js":1,"sites\/all\/modules\/contrib\/service_links\/js\/google_plus_one.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.highwire-list-expand.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"176518","apath":"\/biorxiv\/early\/2019\/02\/15\/549675.atom","pisa":"biorxiv;549675v1","processed":["highwire_math"],"markup":[{"requested":"full-text","variant":"full-text","view":"full","pisa":"biorxiv;549675v1"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"disqus":{"domain":"biorxivstage","url":"https:\/\/www.biorxiv.org\/content\/10.1101\/549675v1","title":"Climate change is predicted to disrupt patterns of local adaptation in wild and cultivated maize","identifier":"node\/176518"},"panels_ajax_pane":{"new-41":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022sgiyKZdSXYAQtLK1y1UXkK1Rv9xubypqY58KZWYhmVzWA4a\\\\\\\/mClxJ915B+bSmtrAXZOb3uNxiNWjNBA3Xc32dLqnITEthIKMLgifT3SQxWnBKo5JA6ST2X+RAn7AzwC27Ymxrr9bf4xz29YdCuA0O0JbxxrpT0wggJfZzMZkAsgT\\\\\\\/dXQlCE8aX9O+YynOj4jQDj8yYUtgm0FipflmOlmHb03NVFzBPTSVXmiMC7f5bIFQGunFqCHDVKbbTPR5mfGrMyWXsGLi8nrqD9PqY\\\\\\\/Tq3rTC0hWfX+cc1i23J1uHTEo2QVpuRF3IZtmnFC2ao9+PTrn3nD+iQnrceiPBO7SSEtCXL3q8\\\\\\\/tvXRQ7W3yrP+NVIn+V5QIL544X1\\\\\\\/x0Leb1TJNAXT8o4TGxvpCLopp2n0jfh1aBlJwb5YGupYRYO+BT6zrR5pHseGmnmuUm82vJIBd290uXI3fndRmMfUG0tdvKnZvmjxe+j3N958OuHzdh9w3YKm\\\\\\\/O\\\\\\\/Pi\\\\\\\/Lp9EzeCMnM8DH0o4q4yn4q8JYhPhpCn0+oZjQXijh8r\\\\\\\/Rr4bVt0\\\\\\\/DIQOQahmfzIEu7p53bgo3XHnIZzO6eb08bI741RhJcqOg989yKEi7kgu\\\\\\\/8J9ANUPLy72jtyIzWmNkh7GQ7OgVKYRu+l1AkDxbs8vibGN5jzNS5dFbfYyzOwU4QWDRXhrKp2JUzfAf1FjGAYtjsFlAE0u74Rh7YMPs7y+DZUvR3ql7DzNy4Q9c\\\\\\\/DIwjK6nefCYJHpxNTzZqLxYO\\\\\\\/Pzrzw\\\\\\\/SWSYVhyfM0TKML98E565oX1ZVVTlYDGea9vj5qxpZ971y7hiVw9JmnHV4DbaqqC2SuqhqZ8M6OX2xuTb79G8uK4vWWn0wycgxsfAsWC0kqaR12iVYMts\\\\\\\/eu5zAxdc\\\\\\\/xOOfwqnMbrmVUUmoi7LsnsDHY1HM\\\\\\\/GvlXp88sps\\\\\\\/9WOS2Koov3JIOlcsnNXAbsG6KKQwiZ9h6l2s+fj3PnK84YF03YtcEIBl+WhgXpua3dqGlCrW3Mz6cBsbW3Aowj8S6hhpO9bhdGFiRVLY3\\\\\\\/BXQQt3Ps7LROVWHu9lXnuG77G\\\\\\\/qe9C7lPQqJfyWRXYP4Dthm27EgqFz6ja4\\\\\\\/HTgfF3OaOB2E3jxhZgl99JrmLaLflq3NRzqxv4gSGRhORNqHHrWYoFAqDyO06w5S07D6oQpEQ66G48SkYeB2uXrivg\\\\\\\/molmFnR1wCi\\\\\\\/VXxF3S2O1DRhUP79PJLuRYVIsxS+jloI6EIE3Lc5Hgoq0j+qMH7Ua0owylTWztoFOcDvgg2VIESLy1lrzp+pfJbYZw8+XW5\\\\\\\/foRFrRBW+wm11FXCD7S84sU=\\\u0022,\\\u0022iv\\\u0022:\\\u00223in8Jfxm1VwqnxU2XjwxNQ==\\\u0022,\\\u0022salt\\\u0022:\\\u0022f9d8845712f72c9fee5db7d9ff111eab\\\u0022}\u0022,\u0022hmac\u0022:\u0022ecd2dcb5a8cbc4ed410e65097318ad15583acc9156f2c2b46ed1034dd8b27ca1\u0022}"},"urlIsAjaxTrusted":{"\/content\/10.1101\/549675v1.full":true},"ws_fl":{"width":100,"height":21},"ws_gpo":{"size":"","annotation":"","lang":"","callback":"","width":300},"color":{"logo":"https:\/\/www.biorxiv.org\/sites\/default\/files\/bioRxiv_article.jpg"},"highwire_list_expand":{"is_collapsed":"1"},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"eu_cookie_compliance":{"popup_enabled":1,"popup_agreed_enabled":0,"popup_hide_agreed":0,"popup_clicking_confirmation":1,"popup_scrolling_confirmation":false,"popup_html_info":"\u003Cdiv\u003E\n  \u003Cdiv class =\u0022popup-content info\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n      \u003Cp\u003EWe use cookies on this site to enhance your user experience. By clicking any link on this page you are giving your consent for us to set cookies.\u003C\/p\u003E\n    \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022agree-button eu-cookie-compliance-default-button\u0022\u003EContinue\u003C\/button\u003E\n                    \u003Cbutton type=\u0022button\u0022 class=\u0022find-more-button eu-cookie-compliance-more-button\u0022\u003EFind out more\u003C\/button\u003E\n          \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E","use_mobile_message":false,"mobile_popup_html_info":"\u003Cdiv\u003E\n  \u003Cdiv class =\u0022popup-content info\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n          \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022agree-button eu-cookie-compliance-default-button\u0022\u003EContinue\u003C\/button\u003E\n                    \u003Cbutton type=\u0022button\u0022 class=\u0022find-more-button eu-cookie-compliance-more-button\u0022\u003EFind out more\u003C\/button\u003E\n          \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E\n","mobile_breakpoint":"768","popup_html_agreed":"\u003Cdiv\u003E\n  \u003Cdiv class=\u0022popup-content agreed\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n      \u003Ch2\u003EThank you for accepting cookies\u003C\/h2\u003E\u003Cp\u003EYou can now hide this message or find out more about cookies.\u003C\/p\u003E    \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022hide-popup-button eu-cookie-compliance-hide-button\u0022\u003EHide\u003C\/button\u003E\n              \u003Cbutton type=\u0022button\u0022 class=\u0022find-more-button eu-cookie-compliance-more-button-thank-you\u0022 \u003EMore info\u003C\/button\u003E\n          \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E","popup_use_bare_css":false,"popup_height":"auto","popup_width":"100%","popup_delay":1000,"popup_link":"\/help\/cookie-policy","popup_link_new_window":1,"popup_position":null,"popup_language":"en","store_consent":false,"better_support_for_screen_readers":0,"reload_page":0,"domain":"","popup_eu_only_js":0,"cookie_lifetime":"365","cookie_session":false,"disagree_do_not_show_popup":0,"method":"default","whitelisted_cookies":"","withdraw_markup":"\u003Cbutton type=\u0022button\u0022 class=\u0022eu-cookie-withdraw-tab\u0022\u003EPrivacy settings\u003C\/button\u003E\n\u003Cdiv class=\u0022eu-cookie-withdraw-banner\u0022\u003E\n  \u003Cdiv class=\u0022popup-content info\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n      \u003Cp\u003E\u0026lt;h2\u0026gt;We use cookies on this site to enhance your user experience\u0026lt;\/h2\u0026gt;\u0026lt;p\u0026gt;You have given your consent for us to set cookies.\u0026lt;\/p\u0026gt;\u003C\/p\u003E\n    \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022eu-cookie-withdraw-button\u0022\u003EWithdraw consent\u003C\/button\u003E\n    \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E\n","withdraw_enabled":false},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1,"trackUrlFragments":1},"jnl_biorxiv_styles":{"defaultJCode":"biorxiv"},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-176518 node-type-highwire-article context-content hw-default-jcode-biorxiv hw-article-type-article hw-article-category-new-results">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-M677548" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-M677548');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
        <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
    
  <div id="zone-branding" class="zone zone-branding clearfix print-display-block container-30">
    <div class="grid-15 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="bioRxiv" src="https://www.biorxiv.org/sites/default/files/bioRxiv_article.jpg" /></a>      </div>
                </div>
          </div>
</div><div class="grid-11 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-system block-menu block-main-menu block-system-main-menu odd block-without-title" id="block-system-main-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/" title="" class="" data-icon-position="" data-hide-link-title="0">Home</a></li>
<li class="leaf" role="menuitem"><a href="/about-biorxiv" class="" data-icon-position="" data-hide-link-title="0">About</a></li>
<li class="leaf" role="menuitem"><a href="/submit-a-manuscript" class="" data-icon-position="" data-hide-link-title="0">Submit</a></li>
<li class="last leaf" role="menuitem"><a href="/content/alertsrss" title="" class="" data-icon-position="" data-hide-link-title="0">ALERTS / RSS</a></li>
</ul></nav>    </div>
  </div>
</div><div class="block block-panels-mini block-biorxiv-search-box block-panels-mini-biorxiv-search-box even block-without-title" id="block-panels-mini-biorxiv-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-biorxiv_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/10.1101/549675v1.full" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--2" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-iwi3KtUYAVy5_E1yvOCOodsFBMGkcElQN3hNM2O8hv8" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2 advanced-search-link" >
  
      
  
  <div class="pane-content">
    <a href="/search">Advanced Search</a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix container-30">
  	      </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="176518" id="node176518" data-pisa="biorxiv;549675v1" data-pisa-master="biorxiv;549675" data-apath="/biorxiv/early/2019/02/15/549675.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div  class="highwire-cite highwire-cite-highwire-article highwire-citation-biorxiv-article-top clearfix has-author-tooltip" >

      <span class="biorxiv-article-type">
        New Results    </span>
  
        <h1 class="highwire-cite-title" id="page-title">Climate change is predicted to disrupt patterns of local adaptation in wild and cultivated maize</h1>  
      <div  class="highwire-cite-authors" ><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0"><span class="nlm-given-names">Jonás A</span> <span class="nlm-surname">Aguirre-Liguori</span></span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="1"><a href="http://orcid.org/0000-0003-3999-6952" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a><span class="nlm-given-names">Santiago</span> <span class="nlm-surname">Ramírez-Barahona</span></span>, <span class="highwire-citation-author" data-delta="2"><span class="nlm-given-names">Peter</span> <span class="nlm-surname">Tiffin</span></span>, <span class="highwire-citation-author" data-delta="3"><span class="nlm-given-names">Luis E</span> <span class="nlm-surname">Eguiarte</span></span></span></div>
  
      <div  class="highwire-cite-metadata" ><span  class="highwire-cite-metadata-doi highwire-cite-metadata"><span class="label">doi:</span> https://doi.org/10.1101/549675 </span></div>
  
  
  </div>
<div id="hw-article-author-popups-node176518" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Jonás A Aguirre-Liguori </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>1</span><span class='nlm-institution'>Laboratorio de Evolución Molecular y Experimental, Departamento de Ecología Evolutiva, Instituto de Ecología. Universidad Nacional Autónoma de México</span>, Circuito Exterior s/n, Ciudad de México, 04510, <span class='nlm-country'>Mexico</span>.</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jon%C3%A1s%2BA%2BAguirre-Liguori%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Aguirre-Liguori%20JA&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJon%25C3%25A1s%2BA%2BAguirre-Liguori%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Santiago Ramírez-Barahona </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>2</span><span class='nlm-institution'>Departamento de Botánica, Instituto de Biología, Universidad Nacional Autónoma de México</span>, Circuito Exterior s/n, Ciudad de México, 04510, <span class='nlm-country'>Mexico</span>.</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Santiago%2BRam%C3%ADrez-Barahona%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Ram%C3%ADrez-Barahona%20S&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ASantiago%2BRam%25C3%25ADrez-Barahona%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0003-3999-6952" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Santiago Ramírez-Barahona</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:santiago.ramirez@ib.unam.mx" class="" data-icon-position="" data-hide-link-title="0">santiago.ramirez@ib.unam.mx</a>
<a href="mailto:fruns@unam.mx" class="" data-icon-position="" data-hide-link-title="0">fruns@unam.mx</a></span></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Peter Tiffin </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>3</span><span class='nlm-institution'>Department of Plant and Microbial Biology, University of Minnesota</span>, St. Paul, MN 55108, <span class='nlm-country'>USA</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Peter%2BTiffin%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Tiffin%20P&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3APeter%2BTiffin%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Luis E Eguiarte </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>1</span><span class='nlm-institution'>Laboratorio de Evolución Molecular y Experimental, Departamento de Ecología Evolutiva, Instituto de Ecología. Universidad Nacional Autónoma de México</span>, Circuito Exterior s/n, Ciudad de México, 04510, <span class='nlm-country'>Mexico</span>.</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Luis%2BE%2BEguiarte%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Eguiarte%20LE&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ALuis%2BE%2BEguiarte%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:santiago.ramirez@ib.unam.mx" class="" data-icon-position="" data-hide-link-title="0">santiago.ramirez@ib.unam.mx</a>
<a href="mailto:fruns@unam.mx" class="" data-icon-position="" data-hide-link-title="0">fruns@unam.mx</a></span></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/10.1101/549675v1" class="panels-ajax-tab-tab" data-panel-name="biorxiv_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:176518" data-trigger="" data-url-enabled="1">Abstract</a><a href="/panels_ajax_tab/biorxiv_tab_art/node:176518/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/10.1101/549675v1.full-text" class="panels-ajax-tab-tab" data-panel-name="article_tab_full_text" data-target-id="highwire_article_tabs" data-entity-context="node:176518" data-trigger="full-text" data-url-enabled="1">Full Text</a><a href="/panels_ajax_tab/article_tab_full_text/node:176518/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/10.1101/549675v1.article-info" class="panels-ajax-tab-tab" data-panel-name="biorxiv_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:176518" data-trigger="article-info" data-url-enabled="1">Info/History</a><a href="/panels_ajax_tab/biorxiv_tab_info/node:176518/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/10.1101/549675v1.article-metrics" class="panels-ajax-tab-tab" data-panel-name="article_tab_metrics" data-target-id="highwire_article_tabs" data-entity-context="node:176518" data-trigger="article-metrics" data-url-enabled="1">Metrics</a><a href="/panels_ajax_tab/article_tab_metrics/node:176518/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/10.1101/549675v1.supplementary-material" class="panels-ajax-tab-tab" data-panel-name="biorxiv_tab_data" data-target-id="highwire_article_tabs" data-entity-context="node:176518" data-trigger="supplementary-material" data-url-enabled="1">Supplementary material</a><a href="/panels_ajax_tab/biorxiv_tab_data/node:176518/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/10.1101/549675v1.full.pdf+html" class="panels-ajax-tab-tab" data-panel-name="biorxiv_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:176518" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i> Preview PDF</a><a href="/panels_ajax_tab/biorxiv_tab_pdf/node:176518/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="article_tab_full_text" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.biorxiv.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-article_tab_full_text"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-markup" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2 class="">Abstract</h2><p id="p-2">Climate change is one of the most important threats to biodiversity and crop sustainability. The impact of climate change is often evaluated on the basis of expected changes in species’ geographical distributions. Genomic diversity, local adaptation, and migration are seldom integrated into projections of species’ responses to climate change. Here we predict that climate change will impact populations of two wild relatives of maize, the teosintes <em>Zea mays</em> ssp. <em>mexicana</em> and <em>Z. mays</em> ssp. <em>parviglumis</em>, by altering patterns of local adaptation and decreasing migration probabilities. These alterations appear to be geographically heterogeneous across populations, suggesting that the possible impacts of climate change will vary considerably among populations. This in spite that most populations exhibit high levels of genetic diversity and are predicted to lie within future suitable areas. The heterogeneous patterns of local adaptation uncovered in teosintes are also evident across maize landraces, which suggests that climate change may give way to future maladaptation of several landraces within currently cultivated areas, possibly leading to increased chances of production shocks under increased temperatures. The predicted alterations to habitat distribution, migration potential, and patterns of local adaptation in wild and cultivated maize, raises a red flag for the future of populations. This underscores the need for continued integration of agronomical practices, genomic data, and climate models to better understand the impacts of a rapidly changing climate on cultivated and wild species.</p></div><div class="section" id="sec-1"><h2 class="">Introduction</h2><p id="p-3">Climate change is having detrimental effects on the geographic distribution, local abundance and levels of genetic diversity of species<sup><a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">1</a>–<a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">6</a></sup>. In particular, climate change induced population declines are one of the major contributors to the current loss of biodiversity. In cultivated species, these impacts will have significant economic and alimentary repercussions at local and global scales<sup><a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">2</a>,<a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">4</a>,<a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">7</a></sup>. The development of strategies to mitigate the impacts of climate change on cultivated species has been a high priority, with recent proposals highlighting the importance of genetic diversity and local adaptation of wild relatives for crop improvement and mitigation<sup><a id="xref-ref-4-2" class="xref-bibr" href="#ref-4">4</a>,<a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">8</a>–<a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">12</a></sup>.</p><p id="p-4">Maize (<em>Zea mays</em> ssp. <em>mays</em>) was domesticated in Mexico from its wild relatives, the teosintes, nearly 9,000 years ago<sup><a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">13</a></sup>. More than 200 Latin-American landraces of maize have been described, which are the result of past and on-going sociocultural processes that shape and maintain extraordinary biodiversity in this species<sup><a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">14</a>,<a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">15</a></sup>. Teosintes grow wild under a wide range of climatic conditions, from very hot and humid coastal environments to temperate and dry inland regions<sup><a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">10</a>,<a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">16</a>,<a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">17</a></sup>. Maize land races have an even wider geographic and environmental range<sup><a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">5</a>,<a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">18</a>,<a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">19</a></sup>. Evidence of gene flow from teosintes into maize and the presence of fertile hybrids<sup><a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">20</a>,<a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">21</a></sup> indicates that gene introgression into landraces is possible and could be important to the continued evolution of maize to respond to new environmental challenges<sup><a id="xref-ref-16-2" class="xref-bibr" href="#ref-16">16</a>,<a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">22</a></sup>.</p><p id="p-5">Both the teosintes and several maize landraces are threatened by habitat loss and climate change<sup><a id="xref-ref-4-3" class="xref-bibr" href="#ref-4">4</a>,<a id="xref-ref-7-2" class="xref-bibr" href="#ref-7">7</a>,<a id="xref-ref-19-2" class="xref-bibr" href="#ref-19">19</a>,<a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">23</a></sup>, primarily increasing temperature and aridity<sup><a id="xref-ref-4-4" class="xref-bibr" href="#ref-4">4</a>,<a id="xref-ref-6-2" class="xref-bibr" href="#ref-6">6</a></sup>. Local adaptation to warm and dry climates is common in teosintes<sup><a id="xref-ref-16-3" class="xref-bibr" href="#ref-16">16</a>,<a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">24</a>–<a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">26</a></sup>. Populations locally adapted to warm and dry environments are likely to contain alleles that could reduce the negative effects of global warming and increased aridity. Thus, the local adaptation of teosintes populations, which has not been directed by cultural and agronomical practices, might prove useful as a blueprint of local adaptation for crop improvement and mitigation in maize landraces.</p><p id="p-6">Ecological niche modeling has become a powerful tool to assess the impacts of climate change on species’ geographic distribution and persistence<sup><a id="xref-ref-27-1" class="xref-bibr" href="#ref-27">27</a>–<a id="xref-ref-29-1" class="xref-bibr" href="#ref-29">29</a></sup>. Commonly used ecological niche models assume that populations within a species are identical and share the same environmental tolerance limits, disregarding differences in the local adaptive response of populations to climate<sup><a id="xref-ref-30-1" class="xref-bibr" href="#ref-30">30</a>,<a id="xref-ref-31-1" class="xref-bibr" href="#ref-31">31</a></sup>. However, given that local adaptation is common, among-population variation in climatic tolerances and varying levels of climate-gene relationships across the genome are expected<sup><a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">25</a>,<a id="xref-ref-30-2" class="xref-bibr" href="#ref-30">30</a></sup>. To our knowledge only a couple of studies have explicitly integrated ecological niche modeling and genomic data to more realistically assess species’ response to climate change<sup><a id="xref-ref-30-3" class="xref-bibr" href="#ref-30">30</a>,<a id="xref-ref-32-1" class="xref-bibr" href="#ref-32">32</a>,<a id="xref-ref-33-1" class="xref-bibr" href="#ref-33">33</a></sup>.</p><p id="p-7">Here we examined how climate change is expected to affect the geographic distribution and the frequency of single nucleotide polymorphisms (SNPs) associated with climate-related traits in two teosintes species: <em>Z. mays</em> ssp. <em>mexicana</em> and <em>Z. mays</em> ssp. <em>parviglumis</em> (hereafter referred to as <em>mexicana</em> and <em>parviglumis</em>, respectively). These teosintes thrive under distinct environmental conditions: <em>mexicana</em> is native to the highland regions of Central Mexico, <em>parviglumis</em> is native to the lowlands of South-western Mexico<sup><a id="xref-ref-10-2" class="xref-bibr" href="#ref-10">10</a>,<a id="xref-ref-16-4" class="xref-bibr" href="#ref-16">16</a>,<a id="xref-ref-17-2" class="xref-bibr" href="#ref-17">17</a>,<a id="xref-ref-25-2" class="xref-bibr" href="#ref-25">25</a></sup>.</p></div><div class="section" id="sec-2"><h2 class="">Materials and Methods</h2><div id="sec-3" class="subsection"><h3>Ecological niche models</h3><p id="p-8">We used present day climatic variables to build species distribution models for the two teosintes separately as implemented in Maxent v.3.3.3<sup><a id="xref-ref-34-1" class="xref-bibr" href="#ref-34">34</a></sup>. We used 254 (<em>mexicana</em>) and 329 (<em>parviglumis</em>) occurrence data points and the bioclimatic variables available in WorldClim<sup><a id="xref-ref-35-1" class="xref-bibr" href="#ref-35">35</a></sup> at a resolution of 30arc/sec (~1 Km<sup>2</sup>), with an area extent of 90°-120° W, 10°-30° N, and background sampling points randomly chosen across the study area (SI Appendix). The eight climate models included: two general circulation models (Community Climate System Model (CCSM), Model for Interdisciplinary Research on Climate, MIROC), two greenhouse gas concentration models adopted by the Intergovernmental Panel on Climate Change (Representative Conservation Pathways: RCP 4.5, RCP 8.5) and two future projections (2050, 2070). After validating the models, we generated binary presence/absence maps to quantify the degree of change in the geographic distribution of the two teosintes species (SI Appendix).</p></div><div id="sec-4" class="subsection"><h3>Population genomic data</h3><p id="p-9">We merged the datasets of ref.<sup><a id="xref-ref-24-2" class="xref-bibr" href="#ref-24">24</a></sup> and ref.<sup><a id="xref-ref-25-3" class="xref-bibr" href="#ref-25">25</a></sup> (available at: <a href="http://dx.doi.org/10.5061/dryad.8m648">http://dx.doi.org/10.5061/dryad.8m648</a> and <a href="https://doi.org/10.5061/dryad.tf556">https://doi.org/10.5061/dryad.tf556</a>), both obtained using the MaizeSNP50 Illumina Bead chip, to construct a dataset of 33,454 high quality SNPs distributed along the ten chromosomes of maize<sup><a id="xref-ref-24-3" class="xref-bibr" href="#ref-24">24</a>,<a id="xref-ref-25-4" class="xref-bibr" href="#ref-25">25</a></sup>. The dataset comprises 23 populations for <em>mexicana</em> and 24 populations for <em>parviglumis</em>, covering the entire geographic and environmental distribution of teosintes<sup><a id="xref-ref-25-5" class="xref-bibr" href="#ref-25">25</a></sup>. The sets of locally adapted SNPs used here are described in detail by ref.<sup><a id="xref-ref-25-6" class="xref-bibr" href="#ref-25">25</a></sup>, who used <em>Bayescenv</em><sup><a id="xref-ref-36-1" class="xref-bibr" href="#ref-36">36</a></sup> and <em>Bayenv</em><sup><a id="xref-ref-37-1" class="xref-bibr" href="#ref-37">37</a></sup> to identify locally adapted SNPs with significant genetic differentiation among populations (<em>F</em><sub><em>ST</em></sub>) and with significant associations with temperature and precipitation after controlling for geographic clines (SI Appendix). To define the set of candidate SNPs (canSNPs) and putative adaptive SNPs (paSNPs), we performed a Discriminant Analyses on Principal Components (DAPC) using climatically defined groups of populations and the locally adapted SNPs for each species separately using the <em>adegenet</em> package<sup><a id="xref-ref-38-1" class="xref-bibr" href="#ref-38">38</a></sup> in R<sup><a id="xref-ref-39-1" class="xref-bibr" href="#ref-39">39</a></sup> (SI Appendix). We define the set of paSNPs as those with a high power to discriminate populations (<em>i.e.</em>, values greater than the third quartile of distribution of loadings) (see SI Appendix, fig. S1), and the set of canSNPs as those with a lower discrimination power.</p><p id="p-10">We estimated levels of genetic diversity (<em>Hs</em>) for refSNPs using the <em>adegenet</em> package<sup><a id="xref-ref-37-2" class="xref-bibr" href="#ref-37">37</a></sup> and <em>hierfstat</em> package<sup><a id="xref-ref-40-1" class="xref-bibr" href="#ref-40">40</a></sup> in R<sup><a id="xref-ref-39-2" class="xref-bibr" href="#ref-39">39</a></sup>. We characterized genetically poor populations as those with an <em>Hs</em> lower than 0.2 for refSNPs, because lower <em>Hs</em> appears to be the probable result of recent bottlenecks<sup><a id="xref-ref-24-4" class="xref-bibr" href="#ref-24">24</a></sup>.</p></div><div id="sec-5" class="subsection"><h3>Gradient Forest (GF)</h3><p id="p-11">Recently, species ecological distribution modeling techniques have been developed to consider intraspecific genetic variation in the assessment of the impact of climate change<sup><a id="xref-ref-29-2" class="xref-bibr" href="#ref-29">29</a>,<a id="xref-ref-30-4" class="xref-bibr" href="#ref-30">30</a>,<a id="xref-ref-32-2" class="xref-bibr" href="#ref-32">32</a>,<a id="xref-ref-33-2" class="xref-bibr" href="#ref-33">33</a></sup>. Among these, GF<sup><a id="xref-ref-41-1" class="xref-bibr" href="#ref-41">41</a></sup> is a regression tree-based approach that implements non-linear regressions to estimate patterns of turnover in species (or allelic) composition as a function of environmental variables<sup><a id="xref-ref-30-5" class="xref-bibr" href="#ref-30">30</a></sup> (SI Appendix). The algorithm identifies regions along an environmental gradient associated with a high rate of change in allele frequencies, irrespective of the underlying allele frequencies. GF uses this information to construct allele turnover functions<sup><a id="xref-ref-30-6" class="xref-bibr" href="#ref-30">30</a></sup>, which can be used to model the change in allele frequencies throughout a species’ range as a function of climatic variables.</p><p id="p-12">We used the <em>gradientforest</em> package<sup><a id="xref-ref-41-2" class="xref-bibr" href="#ref-41">41</a></sup> in R<sup><a id="xref-ref-39-3" class="xref-bibr" href="#ref-39">39</a></sup> to construct turnover functions for each SNP dataset separately (e.g., refSNPs, canSNPs, paSNPs) with the 19 bioclimatic variables used for the species distribution models (see above). We estimated the allelic frequencies of all SNPs in each dataset and implemented GF using 100 trees and a correlation threshold of 0.5<sup><a id="xref-ref-30-7" class="xref-bibr" href="#ref-30">30</a></sup>. Based on the best allele turnover models for each set of SNPs, we constructed landscape models for the present and for the eight future climate models using the <em>raster</em> package<sup><a id="xref-ref-42-1" class="xref-bibr" href="#ref-42">42</a></sup> in R<sup><a id="xref-ref-39-4" class="xref-bibr" href="#ref-39">39</a></sup>. We compared present-day and future landscape models to identify regions (and populations) where the current gene–environment relationships will be most disrupted (genomic offset) (SI Appendix), which measures the relative magnitude of allele frequency changes expected as result of climate change<sup><a id="xref-ref-30-8" class="xref-bibr" href="#ref-30">30</a>,<a id="xref-ref-32-3" class="xref-bibr" href="#ref-32">32</a>,<a id="xref-ref-33-3" class="xref-bibr" href="#ref-33">33</a></sup>.</p><p id="p-13">To gauge the robustness of the GF models to the use of different sets of SNPs, we ran additional analyses using: (1) a set of reference SNPs with an allele frequency distribution similar to those of canSNPs and paSNPs; (2) different sets of outlier SNPs identified by ref.<sup><a id="xref-ref-25-7" class="xref-bibr" href="#ref-25">25</a></sup>; and (3) on the complete set of 33,454 SNPs (SI Appendix).</p></div><div id="sec-6" class="subsection"><h3>Allele distribution models</h3><p id="p-14">We predicted the geographic distribution for putative adaptive alleles at paSNPs using Maxent v.3.3.3<sup><a id="xref-ref-34-2" class="xref-bibr" href="#ref-34">34</a></sup>. Since we were interested in modeling the distribution of the putatively warm-adapted alleles (SI Appendix), we used the corresponding populations’ geographic coordinates where these alleles were recorded as input. We used the same settings and validation procedures used for the species distribution models (see above).</p><p id="p-15">We generated binary presence/absence distribution models for each paSNP for the present and for future climate models (SI Appendix). We estimated the geographic overlap between the present model and each future model to define suitable areas for local adaptation as those with at least five paSNPs predicted to occur. We cross validated the allele distribution models using the GF models for paSNPs, basically inspecting the genomic offset of populations for three sets of regions: present-only, future-only, and overlap.</p></div><div id="sec-7" class="subsection"><h3>Barriers to migration</h3><p id="p-16">Based on the overlapped distribution models for the paSNPs, we tested the potential capacity of populations to migrate into the new regions predicted under future climate scenarios using circuit theory<sup><a id="xref-ref-43-1" class="xref-bibr" href="#ref-43">43</a></sup>. This represents a simplified model of population migration based on the distribution of paSNPs and assumes that migration would be mostly limited by local adaptation. We constructed maps of potential migration using the present and future distribution models for paSNPs to determine landscape resistance (environmental distances) as a proxy of limitation to the successful migration, where increasing resistance indicates decreasing probabilities of migration (SI Appendix). For each sampled population, we constrained our analyses to a 1°x1° degree grid-cell centered on each sampled population. We used the 10-percentile of resistance values as a minimum threshold to estimate migration potential, which can be interpreted as the resistance to successful migration into at least 10% of the future areas of potential settlement (SI Appendix).</p></div><div id="sec-8" class="subsection"><h3>Identifying putative adaptive SNPs in maize landraces</h3><p id="p-17">We explored whether the paSNPs identified for teosintes have been documented in particular landraces of maize<sup><a id="xref-ref-44-1" class="xref-bibr" href="#ref-44">44</a></sup>. For this, we downloaded the genotype data for maize described by ref.<sup><a id="xref-ref-44-2" class="xref-bibr" href="#ref-44">44</a></sup>, consisting of 36,931 SNPs across 46 landraces in Mexico, with 1 to 16 accessions per landrace. After identifying putative adaptive alleles at paSNPs across landraces, we estimated a relative ‘adaptive’ score by adding up individual allele frequencies in each landrace. Due to the varying number of accessions per land race, we corrected the adaptive score by the maximum number of alleles possible in each landrace (i.e., for each locus, four alleles for two accessions) (SI Appendix).</p><p id="p-18">We evaluated the correlation between the adaptive scores and temperature, genomic offset and the geographic extension of each landrace (SI Appendix). For this, we employed simple linear regressions using the adaptive score as the response variable and climatic and geographic data for landraces as the predictors. We used the estimated genomic offset for the two teosintes under the less extreme climate change models (CCSM_2050_RCP4.5) to extract mean genomic offset values from the geographic occurrences of each landrace. To assess the possibility of uncovering positive correlations between predictors and the adaptive score of landraces irrespective of their potential for adaptation, we tested the correlations using adaptive scores estimated from 1000 different sets of randomly sampled refSNPs (13 per set).</p></div></div><div class="section" id="sec-9"><h2 class="">Results and Discussion</h2><p id="p-19">We employed ecological niche modeling and climate averages from 1960 to 1990<sup><a id="xref-ref-35-2" class="xref-bibr" href="#ref-35">35</a></sup> to build geographic distribution models for the two teosintes species under eight projections of climate change. As expected due to geographic constraints<sup><a id="xref-ref-6-3" class="xref-bibr" href="#ref-6">6</a>,<a id="xref-ref-19-3" class="xref-bibr" href="#ref-19">19</a>,<a id="xref-ref-45-1" class="xref-bibr" href="#ref-45">45</a></sup>, the highland teosintes (<em>mexicana</em>) are predicted to be more susceptible to climate change than the lowland teosintes (<em>parviglumis</em>)<sup><a id="xref-ref-17-3" class="xref-bibr" href="#ref-17">17</a>,<a id="xref-ref-19-4" class="xref-bibr" href="#ref-19">19</a></sup>. These models predict that the potentially suitable geographic range for <em>mexicana</em> will be reduced by 30-39% in the next 30 to 40 years, whereas <em>parviglumis</em> range is predicted to be reduced by 4-9% (see SI Appendix, fig. S2, table S1). Consistent with the predicted contraction in geographic range, 29% and 15% of populations in <em>mexicana</em> and <em>parviglumis</em>, respectively, are predicted to fall outside of suitable habitat by 2050 (45% and 29% by 2070).</p><p id="p-20">Ecological niche models can be used to predict suitable geographic ranges<sup><a id="xref-ref-27-2" class="xref-bibr" href="#ref-27">27</a>,<a id="xref-ref-29-3" class="xref-bibr" href="#ref-29">29</a></sup>. Intuitively, the predicted shifts and shrinkage in the teosintes’ geographic range represent a potential threat to their persistence under climate change<sup><a id="xref-ref-19-5" class="xref-bibr" href="#ref-19">19</a></sup>. However, these models do not provide direct information on the potential effects that climate change will have on the genetic composition of local populations<sup><a id="xref-ref-31-2" class="xref-bibr" href="#ref-31">31</a></sup>. To examine the potential effect of climate change on the genetic composition of local populations we applied an extension of Gradient Forest<sup><a id="xref-ref-41-3" class="xref-bibr" href="#ref-41">41</a></sup> developed by ref.<sup><a id="xref-ref-30-9" class="xref-bibr" href="#ref-30">30</a></sup> to predict how climate change will alter the frequencies of three sets of SNPs selected from a dataset of 33,454 SNPs (see SI Appendix) previously identified to be segregating within a sample of 23 populations of <em>mexicana</em> and 24 populations of <em>parviglumis</em> <sup><a id="xref-ref-24-5" class="xref-bibr" href="#ref-24">24</a>,<a id="xref-ref-25-8" class="xref-bibr" href="#ref-25">25</a></sup>. The sampled populations were collected from throughout the teosintes’ distributions.</p><p id="p-21">We subdivided the genomic dataset into three sets of SNPs. The first set is comprised by nine <em>parviglumis</em> and eight <em>mexicana</em> SNPs with frequencies that differ strongly among populations living either in warm or cold environments, a signal of local adaptation. Several of these SNPs are in or near genes involved in responses to heat, drought, and other environmental stress and we hereafter referred to these as putatively adaptive SNPs, or paSNP (see SI Appendix, table S2). The second set, hereafter referred to as candidate SNPs or canSNPs, include 24 <em>parviglumis</em> and 25 <em>mexicana</em> SNPs that also are significantly associated with climate, but not as strongly as paSNPs (see SI Appendix). The third set is comprised of 500 reference SNPs (refSNPs) that we randomly selected from the 33,454 SNPs that showed some, but not particularly strong population differentiation (<em>F</em><sub><em>ST</em></sub> values within the 10% and 90% percentiles of the distribution of per-SNP pairwise population <em>F</em><sub><em>ST</em></sub>). Our expectations are that the paSNPs will not only be most strongly affected by climate change, but also will have a greater contribution to population adaptation to new climatic conditions than can or refSNPs.</p><p id="p-22">Using the extension of Gradient Forest, we estimated environment-SNP associations by modeling allele frequency changes along present climatic gradients<sup><a id="xref-ref-41-4" class="xref-bibr" href="#ref-41">41</a></sup>. These models, in combination with projections of future climate change, can then be used to predict temporal patterns of within-population genetic turnover throughout a species geographic range<sup><a id="xref-ref-30-10" class="xref-bibr" href="#ref-30">30</a>,<a id="xref-ref-41-5" class="xref-bibr" href="#ref-41">41</a></sup>. The extent of genetic turnover of the paSNPs and the canSNPs relative to refSNPs, which we hereafter refer to as genomic offset<sup><a id="xref-ref-30-11" class="xref-bibr" href="#ref-30">30</a>,<a id="xref-ref-32-4" class="xref-bibr" href="#ref-32">32</a>,<a id="xref-ref-33-4" class="xref-bibr" href="#ref-33">33</a></sup>, provides insight into the disruptive effect of climate change on the frequencies of alleles that bear the strongest signatures of local adaptation among contemporary populations. Higher genomic offset is indicative of stronger shifts in frequencies of locally adapted alleles than populations having a lower genomic offset.</p><p id="p-23">Although a more pronounced geographic contraction is predicted for <em>mexicana</em> than <em>parviglumis</em>, genomic offset at paSNPs and canSNPs is higher in <em>parviglumis</em> than in <em>mexicana</em> (mean = 0.27 vs. 0.06 across all models, respectively; see SI Appendix, tables S3, S4, S5). The higher genome-wide offset in <em>parviglumis</em> (see SI Appendix, figs. S3, S4) likely reflects greater population structure in <em>parviglumis</em> than <em>mexicana</em><sup><a id="xref-ref-25-9" class="xref-bibr" href="#ref-25">25</a></sup> and might also indicate that the genetic composition of <em>parviglumis</em> populations will be more affected by climate change than <em>mexicana</em> populations. The levels of genomic offset are, however, highly variable among populations (<a id="xref-fig-1-1" class="xref-fig" href="#F1">fig. 1a-b</a>), with genomic offset lower for populations either with higher frequencies (&gt; 0.5) of the adaptive alleles at paSNPs or having little expected change in climatic conditions (see SI Appendix, fig. S5). The among-population variation underscores that both the selective impact of climate change, and the genomic consequences of subsequent evolutionary response, may both vary among populations and are affected by past local adaptation.</p><div id="F1" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/02/15/549675/F1.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Genomic offset in climate-related SNPs and limited migratory potential for sampled populations of two teosintes in Mexico: Zea mays spp. mexicana and Zea mays spp. parviglumis a-d, Distribution of genomic offset (a-b) and relative migration costs (c-d) for sampled populations of the two species of teosintes estimated under the climate change model CCSM_2050_RCP4.5. Light and dark gray areas represent the species distribution models for mexicana and parviglumis, respectively. Circle size is proportional to the estimated genomic offset and migration costs, with white circles representing populations with the lowest genomic offset and migration costs. Black circles in panels c and d represent populations with no migration potential (infinite migration costs) according the allele distribution models." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1529288064" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;&lt;span class=&quot;caption-title&quot;&gt;Genomic offset in climate-related SNPs and limited migratory potential for sampled populations of two teosintes in Mexico: &lt;em&gt;Zea mays&lt;/em&gt; spp. &lt;em&gt;mexicana&lt;/em&gt; and &lt;em&gt;Zea mays&lt;/em&gt; spp. &lt;em&gt;parviglumis&lt;/em&gt;&lt;/span&gt; &lt;strong&gt;a-d&lt;/strong&gt;, Distribution of genomic offset (a-b) and relative migration costs (c-d) for sampled populations of the two species of teosintes estimated under the climate change model CCSM_2050_RCP4.5. Light and dark gray areas represent the species distribution models for &lt;em&gt;mexicana&lt;/em&gt; and &lt;em&gt;parviglumis&lt;/em&gt;, respectively. Circle size is proportional to the estimated genomic offset and migration costs, with white circles representing populations with the lowest genomic offset and migration costs. Black circles in panels c and d represent populations with no migration potential (infinite migration costs) according the allele distribution models.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 1." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2019/02/15/549675/F1.medium.gif" width="423" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 1." src="https://www.biorxiv.org/content/biorxiv/early/2019/02/15/549675/F1.medium.gif" width="423" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/02/15/549675/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1." data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/02/15/549675/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml"><span class="fig-label">Figure 1.</span> <span class="caption-title">Genomic offset in climate-related SNPs and limited migratory potential for sampled populations of two teosintes in Mexico: <em>Zea mays</em> spp. <em>mexicana</em> and <em>Zea mays</em> spp. <em>parviglumis</em></span><p id="p-24" class="first-child"><strong>a-d</strong>, Distribution of genomic offset (a-b) and relative migration costs (c-d) for sampled populations of the two species of teosintes estimated under the climate change model CCSM_2050_RCP4.5. Light and dark gray areas represent the species distribution models for <em>mexicana</em> and <em>parviglumis</em>, respectively. Circle size is proportional to the estimated genomic offset and migration costs, with white circles representing populations with the lowest genomic offset and migration costs. Black circles in panels c and d represent populations with no migration potential (infinite migration costs) according the allele distribution models.</p><div class="sb-div caption-clear"></div></div></div><p id="p-25">The considerable levels of genomic offset predicted in most populations have the potential to greatly alter patterns of local adaptation, or at least the genetic basis of local adaptation. The genomic offset may also limit populations’ adaptive potential, thereby leading to reduced population sizes and possibly higher probabilities of local extinction<sup><a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">3</a>,<a id="xref-ref-25-10" class="xref-bibr" href="#ref-25">25</a>,<a id="xref-ref-30-12" class="xref-bibr" href="#ref-30">30</a>,<a id="xref-ref-32-5" class="xref-bibr" href="#ref-32">32</a>,<a id="xref-ref-33-5" class="xref-bibr" href="#ref-33">33</a>,<a id="xref-ref-45-2" class="xref-bibr" href="#ref-45">45</a></sup>. This points towards more complex impacts of climate change on wild populations than those predicted using geographical distribution models alone<sup><a id="xref-ref-31-3" class="xref-bibr" href="#ref-31">31</a></sup>.</p><p id="p-26">To complement the use of Gradient Forest, we used the same modeling tools used to predict species distributions to predict the present and future geographic distribution of the putatively adaptive alleles at each of the paSNPs (see SI Appendix). This modeling showed that by 2050, the geographic distribution of adaptive alleles is expected to contract by ~ 45% in <em>mexicana</em> and by 36% <em>parviglumis</em> (table S6). The geographic areas not suitable for adaptive alleles in the future also show higher levels of genomic offset (see SI Appendix, fig. S6), indicating that areas outside the predicted distribution of adaptive alleles are expected to have big shifts in climate-gene relationships, either due to a lower starting frequency of adaptive alleles or to a larger change in climate.</p><p id="p-27">Adaptive alleles at paSNPs are expected to confer selective advantages to expanding warmer and drier environments, yet populations lacking these adaptive alleles might follow a different evolutionary trajectory to adapt to climate change. In this context, standing ‘neutral’ genetic diversity could act as an important source of variation for future local adaptation<sup><a id="xref-ref-1-2" class="xref-bibr" href="#ref-1">1</a>,<a id="xref-ref-5-2" class="xref-bibr" href="#ref-5">5</a>,<a id="xref-ref-46-1" class="xref-bibr" href="#ref-46">46</a>,<a id="xref-ref-47-1" class="xref-bibr" href="#ref-47">47</a></sup>. We found a significantly negative association between genetic diversity at refSNPs (<em>Hs</em>) and the predicted genomic offset for paSNPs (see SI Appendix, fig. S7). These results suggest that, on average, genetically rich populations will be less vulnerable to climate change, probably because the adaptive potential of populations is maximized with intermediate allele frequencies. This result is consistent with the importance usually given to standing genetic variation to allow responding to a changing climate<sup><a id="xref-ref-46-2" class="xref-bibr" href="#ref-46">46</a>,<a id="xref-ref-47-2" class="xref-bibr" href="#ref-47">47</a></sup>. However, more than two-thirds of teosintes populations (71% in <em>parviglumis</em> and 91% in <em>mexicana</em>) show high levels of genetic diversity (<em>Hs</em> &gt; 0.2, ref. <sup><a id="xref-ref-24-6" class="xref-bibr" href="#ref-24">24</a></sup>), yet even under the most conservative climate model, these populations will experience widely varying levels of genomic offset (<a id="xref-fig-1-2" class="xref-fig" href="#F1">fig. 1a-b</a>).</p><p id="p-28">While some populations might adapt to the conditions of a changing climate, other populations might migrate to track the climatic conditions to which they are currently adapted<sup><a id="xref-ref-48-1" class="xref-bibr" href="#ref-48">48</a>,<a id="xref-ref-49-1" class="xref-bibr" href="#ref-49">49</a></sup>. With a focus on the alleles of major effect, we evaluated the migratory potential<sup><a id="xref-ref-43-2" class="xref-bibr" href="#ref-43">43</a>,<a id="xref-ref-50-1" class="xref-bibr" href="#ref-50">50</a></sup> for sampled populations from their current location into sites with climatic conditions that are predicted to be suitable for the current frequencies of adaptive alleles. To do this, we used the distribution models for adaptive alleles to estimate environmental costs (i.e., lower probabilities of populations successfully migrating<sup><a id="xref-ref-43-3" class="xref-bibr" href="#ref-43">43</a>,<a id="xref-ref-50-2" class="xref-bibr" href="#ref-50">50</a></sup>) along all potential migration routes radiating from each sampled population.</p><p id="p-29">Depending on the climate model used, as the climate changes, 46-54% of <em>parviglumis</em> populations and 22-52% of <em>mexicana</em> populations are expected to have the possibility of migrating into sites in which the alleles of major effect will remain advantageous (see SI Appendix, table S7). Although more <em>mexicana</em> populations are predicted to have no migration capabilities (<em>i.e.</em>, infinite migration costs) into climatically suitable sites, probably due to isolation from areas predicted to have suitable conditions in the future, the remaining populations of <em>mexicana</em> actually showed lower migration costs than populations of <em>parviglumis</em> (<a id="xref-fig-1-3" class="xref-fig" href="#F1">fig. 1c-d</a>). Thus, although migration costs are expected to be higher for <em>parviglumis</em> than for <em>mexicana</em>, populations inhabiting the Central Highlands of Mexico are expected to have the highest migratory constraints (<a id="xref-fig-1-4" class="xref-fig" href="#F1">fig. 1c</a>). The variation in migrations costs might result from the differing rates of temperature change estimated between highland environments compared to lowland regions<sup><a id="xref-ref-51-1" class="xref-bibr" href="#ref-51">51</a></sup>. The predicted likelihood of successful migration into a climatically suitable location was positively correlated with within-population frequency of adaptive alleles (see SI Appendix, fig. S8).</p><p id="p-30">Under the most conservative model of climate change (CCSM_4.5_2050), our species distribution models predict 88% and 84% of sampled populations within future suitable areas for <em>parviglumis</em> and <em>mexicana</em>, respectively (see SI Appendix, table S1). In addition, most of these populations exhibit high levels of genetic diversity<sup><a id="xref-ref-25-11" class="xref-bibr" href="#ref-25">25</a></sup>. However, the heterogeneous geographical patterns of genomic offset and migration potential suggest that the impacts of climate change will vary considerably among populations. Overall, eastern populations of <em>mexicana</em> and western populations of <em>parviglumis</em> show high levels of genomic offset and lower migration potential (<a id="xref-fig-1-5" class="xref-fig" href="#F1">fig. 1</a>), thus we expect these populations will experience stronger pressures to adapt to climate change than other teosintes populations. From this simple comparison, we conclude that although highly useful, distribution models are an oversimplification of the complex ecological and evolutionary consequences of climate change<sup><a id="xref-ref-28-1" class="xref-bibr" href="#ref-28">28</a>,<a id="xref-ref-29-4" class="xref-bibr" href="#ref-29">29</a></sup>.</p><p id="p-31">Locally adapted populations of teosintes can be used as a source of advantageous genetic diversity for landraces of maize<sup><a id="xref-ref-4-5" class="xref-bibr" href="#ref-4">4</a>,<a id="xref-ref-17-4" class="xref-bibr" href="#ref-17">17</a>,<a id="xref-ref-22-2" class="xref-bibr" href="#ref-22">22</a></sup>, through the introduction of ‘new wild alleles’ into cultivated populations<sup><a id="xref-ref-8-2" class="xref-bibr" href="#ref-8">8</a>,<a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">9</a>,<a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">11</a>,<a id="xref-ref-12-2" class="xref-bibr" href="#ref-12">12</a></sup>. Experimental evidence has proven that teosintes germplasm can be successfully transferred into maize and can increase crop yields and drought resistance<sup><a id="xref-ref-14-2" class="xref-bibr" href="#ref-14">14</a>,<a id="xref-ref-15-2" class="xref-bibr" href="#ref-15">15</a>,<a id="xref-ref-21-2" class="xref-bibr" href="#ref-21">21</a>,<a id="xref-ref-52-1" class="xref-bibr" href="#ref-52">52</a></sup>. Alternatively, maize improvement can be achieved more easily by selecting adaptive alleles already present in maize landraces. We explored the frequency at which paSNPs identified for teosintes have been documented in some Mexican landraces of maize<sup><a id="xref-ref-44-3" class="xref-bibr" href="#ref-44">44</a></sup>, finding adaptive alleles in 46 landraces of maize (see SI Appendix, table S8). This search allowed us to estimate a relative ‘adaptive’ score by adding up the individual frequencies of adaptive alleles for available accessions in each landrace (SI Appendix).</p><p id="p-32">Expanding on the varying levels of geographic contraction predicted for different maize landraces<sup><a id="xref-ref-19-6" class="xref-bibr" href="#ref-19">19</a></sup>, we found an uneven presence of adaptive alleles across distinct landraces in Mexico (<a id="xref-fig-2-1" class="xref-fig" href="#F2">fig. 2a</a>), with a positive trend of increasing frequency of adaptive alleles (<em>i.e.</em>, adaptive score) in larger geographic ranges and higher mean annual temperatures (<a id="xref-fig-2-2" class="xref-fig" href="#F2">fig. 2c-d</a>). Although the positive trend uncovered for range size is non-significant, several landraces with narrow distribution ranges, such as the almost extinct ‘Tehua’ landrace, and other highly restricted landraces<sup><a id="xref-ref-14-3" class="xref-bibr" href="#ref-14">14</a></sup> (depicted as open circles in <a id="xref-fig-2-3" class="xref-fig" href="#F2">fig. 2b-e</a>), tend to have adaptive alleles at fewer paSNPs and with lower frequencies. In turn, we found a negative relationship between adaptive scores and the genomic offset (<a id="xref-fig-2-4" class="xref-fig" href="#F2">fig. 2e</a>), but this was statistically significant only for the CCSM_2070_RCP8.5 model. However, adaptive scores estimated with refSNPs showed a weaker association with temperature and genomic offset than those estimated for paSNPs, with less than 0.5% of the refSNPs subsamples showing a stronger correlation (see SI Appendix, fig. S9).</p><div id="F2" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/02/15/549675/F2.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Adaptive scores and genomic offset in climate-related SNPs among landraces of maize accessions in Mexico a, Distribution of adaptive scores for accessions of 46 maize landraces in Mexico estimated from the climate-SNP models for the two teosintes species under the climate change model CCSM_2050_RCP4.5. Circle size is proportional to the estimated adaptive scores, with white circles representing accessions with the adaptive scores. b, Known geographic occurrences for the 46 maize landraces in Mexico colored by mean annual temperature. Black stars represent accessions for landraces with high adaptive scores that have been used for maize improvement and white circles represent accessions for restricted landraces with low adaptive potential. c-e, Association of adaptive scores versus the geographic range size, mean annual temperature and genomic offset estimated for 46 maize landraces. Regression coefficients for range size: &#x3B2; = 0.02, F = 1.1, d.f. = 45, p = 0.31; for temperature: &#x3B2; = 0.01, F = 17.7, d.f. = 45, p &lt; 0.001, R2 = 0.27; for genomic offset: &#x3B2; = &#x2212;3.31, F = 3.76, d.f. = 45, p = 0.06. &#x2018;Conico Norte&#xF1;o&#x2019; (CN), &#x2018;Rat&#xF3;n&#x2019; (R), &#x2018;Tabloncillo&#x2019; (Ta), &#x2018;Tabloncillo Perla&#x2019; (TaP), &#x2018;Tuxpe&#xF1;o&#x2019; (Tx), &#x2018;Tuxpe&#xF1;o Norte&#xF1;o&#x2019; (TxN), &#x2018;Vande&#xF1;o&#x2019; (V), &#x2018;Zapolote Chico&#x2019; (ZC), &#x2018;Arrocillo&#x2019; (A), &#x2018;Bofo&#x2019; (B), &#x2018;Cristalino de Chihuahua&#x2019; (CC), &#x2018;Palomero Toluque&#xF1;o&#x2019; (PT) and &#x2018;Tehua&#x2019; (Te)." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1529288064" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;&lt;span class=&quot;caption-title&quot;&gt;Adaptive scores and genomic offset in climate-related SNPs among landraces of maize accessions in Mexico&lt;/span&gt; &lt;strong&gt;a&lt;/strong&gt;, Distribution of adaptive scores for accessions of 46 maize landraces in Mexico estimated from the climate-SNP models for the two teosintes species under the climate change model CCSM_2050_RCP4.5. Circle size is proportional to the estimated adaptive scores, with white circles representing accessions with the adaptive scores. &lt;strong&gt;b&lt;/strong&gt;, Known geographic occurrences for the 46 maize landraces in Mexico colored by mean annual temperature. Black stars represent accessions for landraces with high adaptive scores that have been used for maize improvement and white circles represent accessions for restricted landraces with low adaptive potential. &lt;strong&gt;c-e&lt;/strong&gt;, Association of adaptive scores versus the geographic range size, mean annual temperature and genomic offset estimated for 46 maize landraces. Regression coefficients for range size: &lt;em&gt;&#x3B2;&lt;/em&gt; = 0.02, &lt;em&gt;F&lt;/em&gt; = 1.1, &lt;em&gt;d.f.&lt;/em&gt; = 45, &lt;em&gt;p&lt;/em&gt; = 0.31; for temperature: &lt;em&gt;&#x3B2;&lt;/em&gt; = 0.01, &lt;em&gt;F&lt;/em&gt; = 17.7, &lt;em&gt;d.f.&lt;/em&gt; = 45, &lt;em&gt;p&lt;/em&gt; &lt; 0.001, R&lt;sup&gt;2&lt;/sup&gt; = 0.27; for genomic offset: &lt;em&gt;&#x3B2;&lt;/em&gt; = &#x2212;3.31, &lt;em&gt;F&lt;/em&gt; = 3.76, &lt;em&gt;d.f.&lt;/em&gt; = 45, &lt;em&gt;p&lt;/em&gt; = 0.06. &#x2018;Conico Norte&#xF1;o&#x2019; (CN), &#x2018;Rat&#xF3;n&#x2019; (R), &#x2018;Tabloncillo&#x2019; (Ta), &#x2018;Tabloncillo Perla&#x2019; (TaP), &#x2018;Tuxpe&#xF1;o&#x2019; (Tx), &#x2018;Tuxpe&#xF1;o Norte&#xF1;o&#x2019; (TxN), &#x2018;Vande&#xF1;o&#x2019; (V), &#x2018;Zapolote Chico&#x2019; (ZC), &#x2018;Arrocillo&#x2019; (A), &#x2018;Bofo&#x2019; (B), &#x2018;Cristalino de Chihuahua&#x2019; (CC), &#x2018;Palomero Toluque&#xF1;o&#x2019; (PT) and &#x2018;Tehua&#x2019; (Te).&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 2." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2019/02/15/549675/F2.medium.gif" width="440" height="329"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 2." src="https://www.biorxiv.org/content/biorxiv/early/2019/02/15/549675/F2.medium.gif" width="440" height="329"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/02/15/549675/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2." data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/02/15/549675/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 2.</span> <span class="caption-title">Adaptive scores and genomic offset in climate-related SNPs among landraces of maize accessions in Mexico</span><p id="p-33" class="first-child"><strong>a</strong>, Distribution of adaptive scores for accessions of 46 maize landraces in Mexico estimated from the climate-SNP models for the two teosintes species under the climate change model CCSM_2050_RCP4.5. Circle size is proportional to the estimated adaptive scores, with white circles representing accessions with the adaptive scores. <strong>b</strong>, Known geographic occurrences for the 46 maize landraces in Mexico colored by mean annual temperature. Black stars represent accessions for landraces with high adaptive scores that have been used for maize improvement and white circles represent accessions for restricted landraces with low adaptive potential. <strong>c-e</strong>, Association of adaptive scores versus the geographic range size, mean annual temperature and genomic offset estimated for 46 maize landraces. Regression coefficients for range size: <em>β</em> = 0.02, <em>F</em> = 1.1, <em>d.f.</em> = 45, <em>p</em> = 0.31; for temperature: <em>β</em> = 0.01, <em>F</em> = 17.7, <em>d.f.</em> = 45, <em>p</em> &lt; 0.001, R<sup>2</sup> = 0.27; for genomic offset: <em>β</em> = −3.31, <em>F</em> = 3.76, <em>d.f.</em> = 45, <em>p</em> = 0.06. ‘Conico Norteño’ (CN), ‘Ratón’ (R), ‘Tabloncillo’ (Ta), ‘Tabloncillo Perla’ (TaP), ‘Tuxpeño’ (Tx), ‘Tuxpeño Norteño’ (TxN), ‘Vandeño’ (V), ‘Zapolote Chico’ (ZC), ‘Arrocillo’ (A), ‘Bofo’ (B), ‘Cristalino de Chihuahua’ (CC), ‘Palomero Toluqueño’ (PT) and ‘Tehua’ (Te).</p><div class="sb-div caption-clear"></div></div></div><p id="p-34">The statistically significant relationships between paSNPs identified in teosintes and the climatic environment in which maize landraces are grown, could result from past introgression of adaptive genes into maize via gene flow from teosintes<sup><a id="xref-ref-16-5" class="xref-bibr" href="#ref-16">16</a>,<a id="xref-ref-18-2" class="xref-bibr" href="#ref-18">18</a>,<a id="xref-ref-22-3" class="xref-bibr" href="#ref-22">22</a>,<a id="xref-ref-53-1" class="xref-bibr" href="#ref-53">53</a></sup>. However, hybridization between maize and teosintes appears to be limited to areas of sympatry<sup><a id="xref-ref-22-4" class="xref-bibr" href="#ref-22">22</a>,<a id="xref-ref-53-2" class="xref-bibr" href="#ref-53">53</a></sup> and thus it seem more likely that the higher adaptive scores for landraces growing in warmer climates reflect past adaptation to these conditions<sup><a id="xref-ref-4-6" class="xref-bibr" href="#ref-4">4</a></sup>. The low regression coefficients recovered for paSNPs in maize may partially result from the transferability of models of teosintes into maize, or from the quantitative nature of phenotypic traits associated with adaptation to climate<sup><a id="xref-ref-54-1" class="xref-bibr" href="#ref-54">54</a>–<a id="xref-ref-57-1" class="xref-bibr" href="#ref-57">57</a></sup>. Nonetheless, our results emphasize that climate change might give way to future maladaptation of landraces within currently cultivated areas<sup><a id="xref-ref-4-7" class="xref-bibr" href="#ref-4">4</a></sup>, possibly leading to increased chances of yield declines in many of these landraces under increase temperatures<sup><a id="xref-ref-7-3" class="xref-bibr" href="#ref-7">7</a></sup>.</p><p id="p-35">Based on our analyses, we identified several landraces (depicted as black stars in <a id="xref-fig-2-5" class="xref-fig" href="#F2">fig. 2b-e</a>) with moderate to high adaptive scores and known to be drought resistant<sup><a id="xref-ref-14-4" class="xref-bibr" href="#ref-14">14</a></sup> (see SI Appendix, table S9). Indeed, several of these landraces have been used to improve maize lines with enhanced heat and drought resistance, with increased yields<sup><a id="xref-ref-14-5" class="xref-bibr" href="#ref-14">14</a>,<a id="xref-ref-52-2" class="xref-bibr" href="#ref-52">52</a></sup>. We acknowledge that adaptation to climate very likely involves quantitative traits<sup><a id="xref-ref-46-3" class="xref-bibr" href="#ref-46">46</a>,<a id="xref-ref-54-2" class="xref-bibr" href="#ref-54">54</a>–<a id="xref-ref-57-2" class="xref-bibr" href="#ref-57">57</a></sup>, yet identifying alleles with small effects on polygenic traits remain a major challenge<sup><a id="xref-ref-57-3" class="xref-bibr" href="#ref-57">57</a></sup>, especially in species with complex genetic structures, such as teosintes. Nonetheless, the paSNPs and canSNPs we identified are promising candidates to test the response of maize and teosintes to changing climate under experimental settings. The convergence between the adaptive score of landraces, together with historical agronomical practices, opens up the opportunity to fully integrate the vast cultural knowledge on maize landraces with analyses of genomic data in an ecological context.</p><p id="p-36">In an era in which population-level genomic data is more accessible for an increasing number of non-model species, approaches integrating genomic and environmental data are powerful tools to better understand the impacts of climate change on the viability of natural populations<sup><a id="xref-ref-5-3" class="xref-bibr" href="#ref-5">5</a>,<a id="xref-ref-30-13" class="xref-bibr" href="#ref-30">30</a>–<a id="xref-ref-33-6" class="xref-bibr" href="#ref-33">33</a></sup>. Predictions of the impact of climate change on species based solely on projected shifts in species’ ranges, although valuable, only provide partial information<sup><a id="xref-ref-28-2" class="xref-bibr" href="#ref-28">28</a>,<a id="xref-ref-29-5" class="xref-bibr" href="#ref-29">29</a></sup>. By building spatiotemporal models of climate-gene relationships, we show that the future of wild populations of teosintes and possibly of several landraces of maize appears to be much harsher than previously estimated<sup><a id="xref-ref-19-7" class="xref-bibr" href="#ref-19">19</a></sup>.</p><p id="p-37">Although it is tempting to directly relate our results to future declines in population fitness<sup><a id="xref-ref-33-7" class="xref-bibr" href="#ref-33">33</a></sup>, the association between climate and fitness should be properly validated using experimental or simulation approaches<sup><a id="xref-ref-32-6" class="xref-bibr" href="#ref-32">32</a>,<a id="xref-ref-58-1" class="xref-bibr" href="#ref-58">58</a></sup>. Nevertheless, the fact that future climate change predicts significant alterations to habitat distribution, migration potential, and patterns of local adaptation, raises a red flag for the future of wild and cultivated populations of maize.</p><p id="p-38">Our results represent an important advance moving beyond the standard species distribution models to assess climate change impacts<sup><a id="xref-ref-30-14" class="xref-bibr" href="#ref-30">30</a>–<a id="xref-ref-33-8" class="xref-bibr" href="#ref-33">33</a></sup>, yet our approach still remains an oversimplification of the complex evolutionary and ecological processes affecting populations<sup><a id="xref-ref-31-4" class="xref-bibr" href="#ref-31">31</a>,<a id="xref-ref-45-3" class="xref-bibr" href="#ref-45">45</a>,<a id="xref-ref-46-4" class="xref-bibr" href="#ref-46">46</a>,<a id="xref-ref-57-4" class="xref-bibr" href="#ref-57">57</a></sup>. Our methodological proposal is a step forward in proving how the analysis of genomic data can identify relevant genetic resources to aid wildlife conservation and crop sustainability under a rapidly changing climate.</p></div><div class="section" id="sec-11"><h2 class="">Author contributions</h2><p id="p-40">J.A-L., S.R.-B. and L.E.E. conceived and designed the framework for the analyses. J.A-L. and S.R.-B. collected the data, conducted analyses and drafted the manuscript. L.E.E. secured the funding. All authors discussed results and wrote the manuscript.</p></div><div class="section COI-statement" id="sec-12"><h2 class="">Competing financial interests</h2><p id="p-41">The authors declare no competing financial interests.</p></div><div class="section" id="sec-13"><h2 class="">Data availability</h2><p id="p-42">Genomic data are available at the Dryad Digital Repository: <a href="http://dx.doi.org/10.5061/dryad.8m648">http://dx.doi.org/10.5061/dryad.8m648</a>, <a href="https://doi.org/10.5061/dryad.tf556">https://doi.org/10.5061/dryad.tf556</a>, and <a href="https://doi.org/10.5061/dryad.4t20n">https://doi.org/10.5061/dryad.4t20n</a>. The list of candidate SNPs at <a href="https://doi.org/10.1111/mec.14203">https://doi.org/10.1111/mec.14203</a>. Geographic occurrences of teosintes and maize landraces at <a href="http://www.biodiversidad.gob.mx">http://www.biodiversidad.gob.mx</a>. R data and code at <a href="https://github.com/spiritu-santi/teosintes">https://github.com/spiritu-santi/teosintes</a>.</p></div><div class="section ack" id="ack-1"><h2 class="">Acknowledgements</h2><p id="p-39">We thank Eria A. Rebollar Caudillo, Erika Aguirre Planter and Laura Espinosa Asuar for technical and logistic support during this project. This study was funded by CONACYT Investigación Científica Básica CB2011-167826 and SEP-CONACYT-ANUIES-ECOS France M12-A03-CONACYT-ANUIES 207571 granted to L.E.E; and postdoctoral fellowship from DGAPA-UNAM granted to S.R.-B. This article was written during a sabbatical leave of L.E.E. in the Dept. of Plant and Microbial Biology at the Univ. of Minnesota, with support from the program PASPA-DGAPA, UNAM.</p></div><div class="section ref-list" id="ref-list-1"><h2 class="">References</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label">1.</span><a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference 1. in text" id="ref-1">↵</a><div class="cit ref-cit ref-journal" id="cit-549675v1.1" data-doi="10.1038/nrg2844"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Allendorf</span>  <span class="cit-name-given-names">FW</span></span>, <span class="cit-auth"><span class="cit-name-surname">Hohenlohe</span>  <span class="cit-name-given-names">PA</span></span>, <span class="cit-auth"><span class="cit-name-surname">Luikart</span>  <span class="cit-name-given-names">G.</span></span> <span class="cit-pub-date">2010</span>. <span class="cit-article-title">Genomics and the future of conservation genetics</span>. <abbr class="cit-jnl-abbrev">Nat Rev Genet</abbr> <span class="cit-vol">11</span>: <span class="cit-fpage">697</span>–<span class="cit-lpage">709</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Breviews.%2BGenetics%26rft.stitle%253DNat%2BRev%2BGenet%26rft.aulast%253DAllendorf%26rft.auinit1%253DF.%2BW.%26rft.volume%253D11%26rft.issue%253D10%26rft.spage%253D697%26rft.epage%253D709%26rft.atitle%253DGenomics%2Band%2Bthe%2Bfuture%2Bof%2Bconservation%2Bgenetics.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnrg2844%26rft_id%253Dinfo%253Apmid%252F20847747%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nrg2844&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20847747&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F02%2F15%2F549675.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">2.</span><a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference 2. in text" id="ref-2">↵</a><div class="cit ref-cit ref-journal" id="cit-549675v1.2"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Dai</span>  <span class="cit-name-given-names">A.</span></span> <span class="cit-pub-date">2012</span>. <span class="cit-article-title">Increasing drought under global warming in observations and models</span>. <abbr class="cit-jnl-abbrev">Nat Clim Change</abbr> <span class="cit-vol">1633</span>: <span class="cit-fpage">52</span>–<span class="cit-lpage">57</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat%2BClim%2BChange%26rft.volume%253D1633%26rft.spage%253D52%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">3.</span><a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference 3. in text" id="ref-3">↵</a><div class="cit ref-cit ref-journal" id="cit-549675v1.3"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Pauls</span>  <span class="cit-name-given-names">SU</span></span>, <span class="cit-auth"><span class="cit-name-surname">Nowak</span>  <span class="cit-name-given-names">C</span></span>, <span class="cit-auth"><span class="cit-name-surname">Bálint</span>  <span class="cit-name-given-names">M</span></span>, <span class="cit-auth"><span class="cit-name-surname">Pfenninger</span>  <span class="cit-name-given-names">M.</span></span> <span class="cit-pub-date">2012</span>. <span class="cit-article-title">The impact of global climate change on genetic diversity within populations and species</span>. <abbr class="cit-jnl-abbrev">Mol Ecol</abbr> <span class="cit-vol">22</span>: <span class="cit-fpage">925</span>–<span class="cit-lpage">946</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol%2BEcol%26rft.volume%253D22%26rft.spage%253D925%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">4.</span><a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference 4. in text" id="ref-4">↵</a><div class="cit ref-cit ref-journal" id="cit-549675v1.4"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Hellin</span>  <span class="cit-name-given-names">J</span></span>, <span class="cit-auth"><span class="cit-name-surname">Bellon</span>  <span class="cit-name-given-names">MR</span></span>, <span class="cit-auth"><span class="cit-name-surname">Hearne</span>  <span class="cit-name-given-names">SJ.</span></span> <span class="cit-pub-date">2014</span>. <span class="cit-article-title">Maize landraces and adaptation to climate change in Mexico</span>. <abbr class="cit-jnl-abbrev">J Crop Improv</abbr> <span class="cit-vol">28</span>: <span class="cit-fpage">482</span>–<span class="cit-lpage">501</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ%2BCrop%2BImprov%26rft.volume%253D28%26rft.spage%253D482%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">5.</span><a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference 5. in text" id="ref-5">↵</a><div class="cit ref-cit ref-journal" id="cit-549675v1.5" data-doi="10.1016/j.tree.2014.11.009"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Schafer</span>  <span class="cit-name-given-names">ABA</span></span>, <span class="cit-etal">et al.</span> <span class="cit-pub-date">2015</span>. <span class="cit-article-title">Genomics and the challenging translation into conservation practice</span>. <abbr class="cit-jnl-abbrev">Trends Ecol Evol</abbr> <span class="cit-vol">30</span>: <span class="cit-fpage">78</span>–<span class="cit-lpage">87</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTrends%2BEcol%2BEvol%26rft.volume%253D30%26rft.spage%253D78%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.tree.2014.11.009%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.tree.2014.11.009&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label">6.</span><a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference 6. in text" id="ref-6">↵</a><div class="cit ref-cit ref-journal" id="cit-549675v1.6"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Pacifici</span>  <span class="cit-name-given-names">M</span></span>, <span class="cit-etal">et al.</span> <span class="cit-pub-date">2017</span>. <span class="cit-article-title">Species’ traits influenced their response to recent climate change</span>. <abbr class="cit-jnl-abbrev">Nat Clim Change</abbr> <span class="cit-vol">3223</span>: <span class="cit-fpage">1</span>–<span class="cit-lpage">4</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat%2BClim%2BChange%26rft.volume%253D3223%26rft.spage%253D1%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">7.</span><a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference 7. in text" id="ref-7">↵</a><div class="cit ref-cit ref-journal" id="cit-549675v1.7" data-doi="10.1073/pnas.1718031115"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Tigcheelar</span>  <span class="cit-name-given-names">M</span></span>, <span class="cit-auth"><span class="cit-name-surname">Battisti</span>  <span class="cit-name-given-names">DS</span></span>, <span class="cit-auth"><span class="cit-name-surname">Naylor</span>  <span class="cit-name-given-names">RL</span></span>, <span class="cit-auth"><span class="cit-name-surname">Ray</span>  <span class="cit-name-given-names">DK.</span></span> <span class="cit-pub-date">2018</span>. <span class="cit-article-title">Future warming increases probability of globally synchronized maize production shocks</span>. <abbr class="cit-jnl-abbrev">Proc Natl Acad Sci</abbr> <span class="cit-vol">115</span>: <span class="cit-fpage">6644</span>–<span class="cit-lpage">6649</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc%2BNatl%2BAcad%2BSci%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.1718031115%26rft_id%253Dinfo%253Apmid%252F29891651%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMToiMTE1LzI2LzY2NDQiO3M6NDoiYXRvbSI7czozNzoiL2Jpb3J4aXYvZWFybHkvMjAxOS8wMi8xNS81NDk2NzUuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">8.</span><a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference 8. in text" id="ref-8">↵</a><div class="cit ref-cit ref-journal" id="cit-549675v1.8"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Palmgren</span>  <span class="cit-name-given-names">MG</span></span>, <span class="cit-etal">et al.</span> <span class="cit-pub-date">2014</span>. <span class="cit-article-title">Are we ready for back-to-nature crop breeding□?</span> <abbr class="cit-jnl-abbrev">Trends Plant Sci.</abbr> <span class="cit-vol">20</span>: <span class="cit-fpage">155</span>–<span class="cit-lpage">164</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTrends%2BPlant%2BSci.%26rft.volume%253D20%26rft.spage%253D155%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">9.</span><a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference 9. in text" id="ref-9">↵</a><div class="cit ref-cit ref-journal" id="cit-549675v1.9" data-doi="10.3732/ajb.1400116"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Warschefsky</span>  <span class="cit-name-given-names">E</span></span>, <span class="cit-auth"><span class="cit-name-surname">Penmetsa</span>  <span class="cit-name-given-names">RV</span></span>, <span class="cit-auth"><span class="cit-name-surname">Cook</span>  <span class="cit-name-given-names">DR</span></span>, <span class="cit-auth"><span class="cit-name-surname">Wettberg</span>  <span class="cit-name-given-names">EJ.</span></span> <span class="cit-pub-date">2014</span>. <span class="cit-article-title">Back to the wilds: Tapping evolutionary adaptations for resilient crops through systematic hybridization with crop wild relatives</span>. <abbr class="cit-jnl-abbrev">Am J Bot</abbr> <span class="cit-vol">101</span>: <span class="cit-fpage">1791</span>–<span class="cit-lpage">1800</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAm%2BJ%2BBot%26rft_id%253Dinfo%253Adoi%252F10.3732%252Fajb.1400116%26rft_id%253Dinfo%253Apmid%252F25326621%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiYW1qYm90IjtzOjU6InJlc2lkIjtzOjExOiIxMDEvMTAvMTc5MSI7czo0OiJhdG9tIjtzOjM3OiIvYmlvcnhpdi9lYXJseS8yMDE5LzAyLzE1LzU0OTY3NS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">10.</span><a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference 10. in text" id="ref-10">↵</a><div class="cit ref-cit ref-book" id="cit-549675v1.10"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Aguirre-Liguori</span>  <span class="cit-name-given-names">JA</span></span>, <span class="cit-auth"><span class="cit-name-surname">Aguirre-planter</span>  <span class="cit-name-given-names">E</span></span>, <span class="cit-auth"><span class="cit-name-surname">Eguiarte</span>  <span class="cit-name-given-names">LE.</span></span> <span class="cit-pub-date">2016</span>. <span class="cit-chapter-title">Genetics and ecology of wild and cultivated maize: domestication and introgression</span>. In: <span class="cit-source">Ethnobotany of Mexico</span>. <span class="cit-publ-loc">New York</span>: <span class="cit-publ-name">Springer</span>, pp <span class="cit-fpage">403</span>–<span class="cit-lpage">416</span></cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">11.</span><a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference 11. in text" id="ref-11">↵</a><div class="cit ref-cit ref-journal" id="cit-549675v1.11" data-doi="10.1111/eva.12434"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">H</span></span>, <span class="cit-auth"><span class="cit-name-surname">Mittal</span>  <span class="cit-name-given-names">N</span></span>, <span class="cit-auth"><span class="cit-name-surname">Leamy</span>  <span class="cit-name-given-names">LJ</span></span>, <span class="cit-auth"><span class="cit-name-surname">Barazani</span>  <span class="cit-name-given-names">O</span></span>, <span class="cit-auth"><span class="cit-name-surname">Song</span>  <span class="cit-name-given-names">BH.</span></span> <span class="cit-pub-date">2016</span>. <span class="cit-article-title">Back into the wild — Apply untapped genetic diversity of wild relatives for crop improvement</span>. <abbr class="cit-jnl-abbrev">Evol Appl</abbr> <span class="cit-vol">10</span>: <span class="cit-fpage">5</span>–<span class="cit-lpage">24</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEvol%2BAppl%26rft.volume%253D10%26rft.spage%253D5%26rft_id%253Dinfo%253Adoi%252F10.1111%252Feva.12434%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/eva.12434&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label">12.</span><a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference 12. in text" id="ref-12">↵</a><div class="cit ref-cit ref-journal" id="cit-549675v1.12" data-doi="10.2135/cropsci2016.10.0885"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Dempewolf</span>  <span class="cit-name-given-names">H</span></span>, <span class="cit-etal">et al.</span> <span class="cit-pub-date">2017</span>. <span class="cit-article-title">Past and future use of wild relatives in crop breeding</span>. <abbr class="cit-jnl-abbrev">Crop Sci</abbr> <span class="cit-vol">57</span>: <span class="cit-fpage">1070</span>–<span class="cit-lpage">1082</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCrop%2BSci%26rft.volume%253D57%26rft.spage%253D1070%26rft_id%253Dinfo%253Adoi%252F10.2135%252Fcropsci2016.10.0885%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2135/cropsci2016.10.0885&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label">13.</span><a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference 13. in text" id="ref-13">↵</a><div class="cit ref-cit ref-journal" id="cit-549675v1.13" data-doi="10.1073/pnas.052125199"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Matsuoka</span>  <span class="cit-name-given-names">Y</span></span>, <span class="cit-etal">et al.</span> <span class="cit-pub-date">2002</span>. <span class="cit-article-title">A single domestication for maize shown by multilocusmicrosatellite genotyping</span>. <abbr class="cit-jnl-abbrev">Proc Natl Acad Sci</abbr> <span class="cit-vol">99</span>: <span class="cit-fpage">6080</span>–<span class="cit-lpage">6084</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPNAS%26rft.stitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft.aulast%253DMatsuoka%26rft.auinit1%253DY.%26rft.volume%253D99%26rft.issue%253D9%26rft.spage%253D6080%26rft.epage%253D6084%26rft.atitle%253DA%2Bsingle%2Bdomestication%2Bfor%2Bmaize%2Bshown%2Bby%2Bmultilocus%2Bmicrosatellite%2Bgenotyping%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.052125199%26rft_id%253Dinfo%253Apmid%252F11983901%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czo5OiI5OS85LzYwODAiO3M6NDoiYXRvbSI7czozNzoiL2Jpb3J4aXYvZWFybHkvMjAxOS8wMi8xNS81NDk2NzUuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">14.</span><a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference 14. in text" id="ref-14">↵</a><div class="cit ref-cit ref-book" id="cit-549675v1.14"><div class="cit-metadata"><cite><span class="cit-auth cit-collab">CONABIO</span>. <span class="cit-pub-date">2011</span>. <span class="cit-chapter-title">Base de datos del proyecto global “Recopilación, generación, actualización y análisis de información acerca de la diversidad genética de maíces y sus parientes silvestres en México”</span>. <span class="cit-source">Comisión Nacional para el Conocimiento y Uso de la Biodiversidad</span>. <span class="cit-publ-loc">México, DF</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">15.</span><a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference 15. in text" id="ref-15">↵</a><div class="cit ref-cit ref-book" id="cit-549675v1.15"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Sánchez</span>  <span class="cit-name-given-names">GJJ.</span></span> <span class="cit-pub-date">2011</span>. <span class="cit-chapter-title">Diversidad del maíz y teocintle</span>. <span class="cit-source">Informe preparado para el proyecto global “Recopilación, generación, actualización y análisis de información acerca de la diversidad genética de maíces y sus parientes silvestres en México” de la Comisión Nacional para el Conocimiento y Uso de la Biodiversidad</span>. <span class="cit-publ-name">Comisión Nacional para el Conocimiento y Uso de la Biodiversidad</span>. <span class="cit-publ-loc">México, DF</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">16.</span><a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference 16. in text" id="ref-16">↵</a><div class="cit ref-cit ref-journal" id="cit-549675v1.16"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Hufford</span>  <span class="cit-name-given-names">MB</span></span>, <span class="cit-auth"><span class="cit-name-surname">Martínez-Meyer</span>  <span class="cit-name-given-names">E</span></span>, <span class="cit-auth"><span class="cit-name-surname">Gaut</span>  <span class="cit-name-given-names">BS</span></span>, <span class="cit-auth"><span class="cit-name-surname">Eguiarte</span>  <span class="cit-name-given-names">LE</span></span>, <span class="cit-auth"><span class="cit-name-surname">Tenaillon</span>  <span class="cit-name-given-names">MI.</span></span> <span class="cit-pub-date">2012</span>. <span class="cit-article-title">Inferences from the historical distribution of wild and domesticated maize provide ecological and evolutionary insight</span>. <abbr class="cit-jnl-abbrev">PLoS ONE</abbr> <span class="cit-vol">11</span>: <span class="cit-fpage">e47659</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BONE%26rft.volume%253D11%26rft.spage%253De47659%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">17.</span><a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference 17. in text" id="ref-17">↵</a><div class="cit ref-cit ref-journal" id="cit-549675v1.17"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Sánchez González</span>  <span class="cit-name-given-names">JJ</span></span>, <span class="cit-etal">et al.</span> <span class="cit-pub-date">2018</span>. <span class="cit-article-title">Ecogeography of teosinte</span>. <abbr class="cit-jnl-abbrev">PLoS ONE</abbr> <span class="cit-vol">13</span>: <span class="cit-fpage">e0192676</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BONE%26rft.volume%253D13%26rft.spage%253De0192676%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">18.</span><a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference 18. in text" id="ref-18">↵</a><div class="cit ref-cit ref-journal" id="cit-549675v1.18" data-doi="10.1073/pnas.1013011108"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">van Heerwaarden</span>  <span class="cit-name-given-names">J</span></span>, <span class="cit-etal">et al.</span> <span class="cit-pub-date">2011</span>. <span class="cit-article-title">Genetic signals of origin, spread, and introgression in a large sample of maize landraces</span>. <abbr class="cit-jnl-abbrev">Proc Natl Acad Sci</abbr> <span class="cit-vol">108</span>: <span class="cit-fpage">1088</span>–<span class="cit-lpage">1092</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc%2BNatl%2BAcad%2BSci%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.1013011108%26rft_id%253Dinfo%253Apmid%252F21189301%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMDoiMTA4LzMvMTA4OCI7czo0OiJhdG9tIjtzOjM3OiIvYmlvcnhpdi9lYXJseS8yMDE5LzAyLzE1LzU0OTY3NS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">19.</span><a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference 19. in text" id="ref-19">↵</a><div class="cit ref-cit ref-journal" id="cit-549675v1.19"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Ureta</span>  <span class="cit-name-given-names">C</span></span>, <span class="cit-auth"><span class="cit-name-surname">Martínez-Meyer</span>  <span class="cit-name-given-names">E</span></span>, <span class="cit-auth"><span class="cit-name-surname">Perales</span>  <span class="cit-name-given-names">HR</span></span>, <span class="cit-auth"><span class="cit-name-surname">Álvarez-Buylla</span>  <span class="cit-name-given-names">ER.</span></span> <span class="cit-pub-date">2012</span>. <span class="cit-article-title">Projecting the effects of climate change on the distribution of maize races and their wild relatives in Mexico</span>. <abbr class="cit-jnl-abbrev">Glob Change Biol</abbr> <span class="cit-vol">18</span>: <span class="cit-fpage">1073</span>–<span class="cit-lpage">108</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGlob%2BChange%2BBiol%26rft.volume%253D18%26rft.spage%253D1073%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">20.</span><a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference 20. in text" id="ref-20">↵</a><div class="cit ref-cit ref-journal" id="cit-549675v1.20" data-doi="10.1007/s00122-004-1859-6"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Baltazar</span>  <span class="cit-name-given-names">BM</span></span>, <span class="cit-auth"><span class="cit-name-surname">Sanchez-Gonzalez</span>  <span class="cit-name-given-names">JJ</span></span>, <span class="cit-auth"><span class="cit-name-surname">de la Cruz-Larios</span>  <span class="cit-name-given-names">L</span></span>, <span class="cit-auth"><span class="cit-name-surname">Schoper</span>  <span class="cit-name-given-names">J B.</span></span> <span class="cit-pub-date">2005</span>. <span class="cit-article-title">Pollination between maize and teosinte: an important determinant of gene flow in Mexico</span>. <abbr class="cit-jnl-abbrev">Theor Appl Genet</abbr> <span class="cit-vol">110</span>: <span class="cit-fpage">519</span>–<span class="cit-lpage">526</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTAG.%2BTheoretical%2Band%2Bapplied%2Bgenetics.%2B%2BTheoretische%2Bund%2Bangewandte%2BGenetik%26rft.stitle%253DTheor%2BAppl%2BGenet%26rft.aulast%253DBaltazar%26rft.auinit1%253DB.%2BM.%26rft.volume%253D110%26rft.issue%253D3%26rft.spage%253D519%26rft.epage%253D526%26rft.atitle%253DPollination%2Bbetween%2Bmaize%2Band%2Bteosinte%253A%2Ban%2Bimportant%2Bdeterminant%2Bof%2Bgene%2Bflow%2Bin%2BMexico.%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00122-004-1859-6%26rft_id%253Dinfo%253Apmid%252F15592808%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00122-004-1859-6&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15592808&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F02%2F15%2F549675.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000227049900012&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">21.</span><a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference 21. in text" id="ref-21">↵</a><div class="cit ref-cit ref-journal" id="cit-549675v1.21"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Kato</span>  <span class="cit-name-given-names">TAY</span></span>, <span class="cit-auth"><span class="cit-name-surname">Sánchez</span>  <span class="cit-name-given-names">JJG.</span></span> <span class="cit-pub-date">2002</span>. <span class="cit-article-title">Introgression of chromosome knobs from Zea diploperennis into maize</span>. <abbr class="cit-jnl-abbrev">Maydica</abbr> <span class="cit-vol">47</span>: <span class="cit-fpage">33</span>–<span class="cit-lpage">50</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMaydica%26rft.volume%253D47%26rft.spage%253D33%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">22.</span><a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference 22. in text" id="ref-22">↵</a><div class="cit ref-cit ref-journal" id="cit-549675v1.22"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Warburton</span>  <span class="cit-name-given-names">ML</span></span>, <span class="cit-etal">et al.</span> <span class="cit-pub-date">2011</span>. <span class="cit-article-title">Gene flow among different teosinte taxa and into the domesticated maize gene pool</span>. <abbr class="cit-jnl-abbrev">Genet Resour Crop Ev</abbr> <span class="cit-vol">58</span>:<span class="cit-fpage">1243</span>–<span class="cit-lpage">1261</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenet%2BResour%2BCrop%2BEv%26rft.volume%253D58%26rft.spage%253D1243%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">23.</span><a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference 23. in text" id="ref-23">↵</a><div class="cit ref-cit ref-journal" id="cit-549675v1.23"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Wilkes</span>  <span class="cit-name-given-names">HG.</span></span> <span class="cit-pub-date">2006</span>. <span class="cit-article-title">Urgent notice to all maize researchers: disappearance and extinction of the last wild teosinte populations is more than half completed. A modest proposal for teosinte evolution and conservation in situ: the Balsas, Guerrero, Mexico</span>. <abbr class="cit-jnl-abbrev">Maydica</abbr> <span class="cit-vol">52</span>:<span class="cit-fpage">49</span>–<span class="cit-lpage">58</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMaydica%26rft.volume%253D52%26rft.spage%253D49%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">24.</span><a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference 24. in text" id="ref-24">↵</a><div class="cit ref-cit ref-journal" id="cit-549675v1.24" data-doi="10.1093/gbe/evt109"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Pyhäjärvi</span>  <span class="cit-name-given-names">T</span></span>, <span class="cit-auth"><span class="cit-name-surname">Hufford</span>  <span class="cit-name-given-names">MB</span></span>, <span class="cit-auth"><span class="cit-name-surname">Mezmouk</span>  <span class="cit-name-given-names">S</span></span>, <span class="cit-auth"><span class="cit-name-surname">Ross-Ibarra</span>  <span class="cit-name-given-names">J.</span></span> <span class="cit-pub-date">2013</span>. <span class="cit-article-title">Complex patterns of local adaptation in teosinte</span>. <abbr class="cit-jnl-abbrev">Genome Biol Evol</abbr> <span class="cit-vol">5</span>: <span class="cit-fpage">1594</span>–<span class="cit-lpage">1609</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BBiol%2BEvol%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fgbe%252Fevt109%26rft_id%253Dinfo%253Apmid%252F23902747%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/gbe/evt109&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23902747&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F02%2F15%2F549675.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">25.</span><a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference 25. in text" id="ref-25">↵</a><div class="cit ref-cit ref-journal" id="cit-549675v1.25"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Aguirre-Liguori</span>  <span class="cit-name-given-names">JA</span></span>, <span class="cit-etal">et al.</span> <span class="cit-pub-date">2017</span>. <span class="cit-article-title">Connecting genomic patterns of local adaptation and niche suitability in teosintes</span>. <abbr class="cit-jnl-abbrev">Mol Ecol</abbr> <span class="cit-vol">26</span>: <span class="cit-fpage">4226</span>–<span class="cit-lpage">4240</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol%2BEcol%26rft.volume%253D26%26rft.spage%253D4226%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">26.</span><a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference 26. in text" id="ref-26">↵</a><div class="cit ref-cit ref-journal" id="cit-549675v1.26" data-doi="10.1111/mec.14082"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Fustier</span>  <span class="cit-name-given-names">M-A</span></span>, <span class="cit-etal">et al.</span> <span class="cit-pub-date">2017</span>. <span class="cit-article-title">Signatures of local adaptation in lowland and highland teosintes from whole genome sequencing of pooled samples</span>. <abbr class="cit-jnl-abbrev">Mol Ecol</abbr> <span class="cit-vol">26</span>: <span class="cit-fpage">2738</span>–<span class="cit-lpage">D2756</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol%2BEcol%26rft.volume%253D26%26rft.spage%253D2738%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fmec.14082%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/mec.14082&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label">27.</span><a class="rev-xref-ref" href="#xref-ref-27-1" title="View reference 27. in text" id="ref-27">↵</a><div class="cit ref-cit ref-book" id="cit-549675v1.27"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Peterson</span>  <span class="cit-name-given-names">AT</span></span>, <span class="cit-etal">et al.</span> <span class="cit-pub-date">2011</span>. <span class="cit-source">Ecological niches and geographic distributions</span>. <span class="cit-publ-name">Princeton University Press</span>, <span class="cit-publ-loc">New Jersey</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">28.</span><a class="rev-xref-ref" href="#xref-ref-28-1" title="View reference 28. in text" id="ref-28">↵</a><div class="cit ref-cit ref-journal" id="cit-549675v1.28"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Pagel</span>  <span class="cit-name-given-names">J</span></span>, <span class="cit-auth"><span class="cit-name-surname">Schurr</span>  <span class="cit-name-given-names">FM.</span></span> <span class="cit-pub-date">2012</span>. <span class="cit-article-title">Forecasting species ranges by statistical estimation of ecological niches and spatial populations dynamics</span>. <abbr class="cit-jnl-abbrev">Glob Ecol Biogeogr</abbr> <span class="cit-vol">21</span>: <span class="cit-fpage">293</span>–<span class="cit-lpage">304</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGlob%2BEcol%2BBiogeogr%26rft.volume%253D21%26rft.spage%253D293%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">29.</span><a class="rev-xref-ref" href="#xref-ref-29-1" title="View reference 29. in text" id="ref-29">↵</a><div class="cit ref-cit ref-journal" id="cit-549675v1.29"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Gotelli</span>  <span class="cit-name-given-names">NJ</span></span>, <span class="cit-auth"><span class="cit-name-surname">Stanton-Geddes</span>  <span class="cit-name-given-names">J.</span></span> <span class="cit-pub-date">2015</span>. <span class="cit-article-title">Climate change, genetic markers and species distribution modelling</span>. <abbr class="cit-jnl-abbrev">J Biogeogr</abbr> <span class="cit-vol">42</span>: <span class="cit-fpage">1577</span>–<span class="cit-lpage">1585</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ%2BBiogeogr%26rft.volume%253D42%26rft.spage%253D1577%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">30.</span><a class="rev-xref-ref" href="#xref-ref-30-1" title="View reference 30. in text" id="ref-30">↵</a><div class="cit ref-cit ref-journal" id="cit-549675v1.30" data-doi="10.1111/ele.12376"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Fitzpatrick</span>  <span class="cit-name-given-names">MC</span></span>, <span class="cit-auth"><span class="cit-name-surname">Keller</span>  <span class="cit-name-given-names">SR.</span></span> <span class="cit-pub-date">2015</span>. <span class="cit-article-title">Ecological genomics meets community-level modelling of biodiversity: mapping the genomic landscape of current and future environmental adaptation</span>. <abbr class="cit-jnl-abbrev">Ecol Lett</abbr> <span class="cit-vol">18</span>: <span class="cit-fpage">1</span>–<span class="cit-lpage">16</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEcol%2BLett%26rft.volume%253D18%26rft.spage%253D1%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fele.12376%26rft_id%253Dinfo%253Apmid%252F25270536%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/ele.12376&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25270536&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F02%2F15%2F549675.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">31.</span><a class="rev-xref-ref" href="#xref-ref-31-1" title="View reference 31. in text" id="ref-31">↵</a><div class="cit ref-cit ref-journal" id="cit-549675v1.31"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Peterson</span>  <span class="cit-name-given-names">ML</span></span>, <span class="cit-auth"><span class="cit-name-surname">Doak</span>  <span class="cit-name-given-names">DF</span></span>, <span class="cit-auth"><span class="cit-name-surname">Morris</span>  <span class="cit-name-given-names">WF.</span></span> <span class="cit-pub-date">2019</span>. <span class="cit-article-title">Incorporating local adaptation into forecasts of species’ distribution and abundance under climate change</span>. <abbr class="cit-jnl-abbrev">Global Change Biol</abbr> <span class="cit-vol">early view</span>: <a href="https://doi.org/10.1111/gcb.14562">https://doi.org/10.1111/gcb.14562</a>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">32.</span><a class="rev-xref-ref" href="#xref-ref-32-1" title="View reference 32. in text" id="ref-32">↵</a><div class="cit ref-cit ref-journal" id="cit-549675v1.32"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Exposito-Alonso</span>  <span class="cit-name-given-names">M</span></span>, <span class="cit-etal">et al.</span> <span class="cit-pub-date">2017</span>. <span class="cit-article-title">Genomic basis and evolutionary potential for extreme drought adaptation in Arabidopsis thaliana</span>. <abbr class="cit-jnl-abbrev">Nat Ecol Evol</abbr> <span class="cit-vol">2</span>: <span class="cit-fpage">352</span>–<span class="cit-lpage">358</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat%2BEcol%2BEvol%26rft.volume%253D2%26rft.spage%253D352%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">33.</span><a class="rev-xref-ref" href="#xref-ref-33-1" title="View reference 33. in text" id="ref-33">↵</a><div class="cit ref-cit ref-journal" id="cit-549675v1.33" data-doi="10.1126/science.aan4380"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Bay</span>  <span class="cit-name-given-names">RA</span></span>, <span class="cit-etal">et al.</span> <span class="cit-pub-date">2018</span>. <span class="cit-article-title">Genomic signals of selection predict climate-driven populations declines in a migratory bird</span>. <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">359</span>: <span class="cit-fpage">83</span>–<span class="cit-lpage">86</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DBay%26rft.auinit1%253DR.%2BA.%26rft.volume%253D359%26rft.issue%253D6371%26rft.spage%253D83%26rft.epage%253D86%26rft.atitle%253DGenomic%2Bsignals%2Bof%2Bselection%2Bpredict%2Bclimate-driven%2Bpopulation%2Bdeclines%2Bin%2Ba%2Bmigratory%2Bbird%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.aan4380%26rft_id%253Dinfo%253Apmid%252F29302012%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjExOiIzNTkvNjM3MS84MyI7czo0OiJhdG9tIjtzOjM3OiIvYmlvcnhpdi9lYXJseS8yMDE5LzAyLzE1LzU0OTY3NS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">34.</span><a class="rev-xref-ref" href="#xref-ref-34-1" title="View reference 34. in text" id="ref-34">↵</a><div class="cit ref-cit ref-journal" id="cit-549675v1.34" data-doi="10.1016/j.ecolmodel.2005.03.026"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Phillips</span>  <span class="cit-name-given-names">SJ</span></span>, <span class="cit-auth"><span class="cit-name-surname">Anderson</span>  <span class="cit-name-given-names">RP</span></span>, <span class="cit-auth"><span class="cit-name-surname">Schapire</span>  <span class="cit-name-given-names">RE.</span></span> <span class="cit-pub-date">2006</span>. <span class="cit-article-title">Maximum entropy modeling of species geographic distributions</span>. <abbr class="cit-jnl-abbrev">Ecol Model</abbr> <span class="cit-vol">190</span>: <span class="cit-fpage">231</span>–<span class="cit-lpage">259</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEcol%2BModel%26rft.volume%253D190%26rft.spage%253D231%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.ecolmodel.2005.03.026%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.ecolmodel.2005.03.026&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000233859600001&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">35.</span><a class="rev-xref-ref" href="#xref-ref-35-1" title="View reference 35. in text" id="ref-35">↵</a><div class="cit ref-cit ref-journal" id="cit-549675v1.35"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Hijmans</span>  <span class="cit-name-given-names">RJ</span></span>, <span class="cit-auth"><span class="cit-name-surname">Cameron</span>  <span class="cit-name-given-names">SE</span></span>, <span class="cit-auth"><span class="cit-name-surname">Parra</span>  <span class="cit-name-given-names">JL</span></span>, <span class="cit-auth"><span class="cit-name-surname">Jones</span>  <span class="cit-name-given-names">PG</span></span>, <span class="cit-auth"><span class="cit-name-surname">Jarvis</span>  <span class="cit-name-given-names">A</span></span> <span class="cit-pub-date">2005</span>. <span class="cit-article-title">Very high-resolution interpolated climate surfaces for global land areas</span>. <abbr class="cit-jnl-abbrev">Int J Climatology</abbr>, <span class="cit-vol">25</span>: <span class="cit-fpage">1965</span>–<span class="cit-lpage">1978</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DInt%2BJ%2BClimatology%26rft.volume%253D25%26rft.spage%253D1965%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">36.</span><a class="rev-xref-ref" href="#xref-ref-36-1" title="View reference 36. in text" id="ref-36">↵</a><div class="cit ref-cit ref-journal" id="cit-549675v1.36" data-doi="10.1111/2041-210X.12418"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Villemereuil</span>  <span class="cit-name-given-names">P</span></span>, <span class="cit-auth"><span class="cit-name-surname">Gaggiotti</span>  <span class="cit-name-given-names">OE.</span></span> <span class="cit-pub-date">2015</span>. <span class="cit-article-title">A new FST-based method to uncover local adaptation using environmental variables</span>. <abbr class="cit-jnl-abbrev">Methods Ecol Evol</abbr> <span class="cit-vol">6</span>: <span class="cit-fpage">1248</span>–<span class="cit-lpage">1258</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMethods%2BEcol%2BEvol%26rft.volume%253D6%26rft.spage%253D1248%26rft_id%253Dinfo%253Adoi%252F10.1111%252F2041-210X.12418%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/2041-210X.12418&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label">37.</span><a class="rev-xref-ref" href="#xref-ref-37-1" title="View reference 37. in text" id="ref-37">↵</a><div class="cit ref-cit ref-journal" id="cit-549675v1.37" data-doi="10.1534/genetics.110.114819"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Coop</span>  <span class="cit-name-given-names">G</span></span>, <span class="cit-auth"><span class="cit-name-surname">Witonsky</span>  <span class="cit-name-given-names">D</span></span>, <span class="cit-auth"><span class="cit-name-surname">Di Rienzo</span>  <span class="cit-name-given-names">A</span></span>, <span class="cit-auth"><span class="cit-name-surname">Pritchard</span>  <span class="cit-name-given-names">JK.</span></span> <span class="cit-pub-date">2010</span>. <span class="cit-article-title">Using environmental correlations to identify loci underlying local adaptation</span>. <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">185</span>: <span class="cit-fpage">1411</span>–<span class="cit-lpage">1423</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.110.114819%26rft_id%253Dinfo%253Apmid%252F20516501%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE4NS80LzE0MTEiO3M6NDoiYXRvbSI7czozNzoiL2Jpb3J4aXYvZWFybHkvMjAxOS8wMi8xNS81NDk2NzUuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">38.</span><a class="rev-xref-ref" href="#xref-ref-38-1" title="View reference 38. in text" id="ref-38">↵</a><div class="cit ref-cit ref-journal" id="cit-549675v1.38" data-doi="10.1093/bioinformatics/btn129"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Jombart</span>  <span class="cit-name-given-names">T.</span></span> <span class="cit-pub-date">2008</span>. <span class="cit-article-title">adegenet: a R package for the multivariate analysis of genetic markers</span>. <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">24</span>: <span class="cit-fpage">1403</span>–<span class="cit-lpage">1405</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtn129%26rft_id%253Dinfo%253Apmid%252F18397895%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/btn129&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18397895&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F02%2F15%2F549675.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000256169300015&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">39.</span><a class="rev-xref-ref" href="#xref-ref-39-1" title="View reference 39. in text" id="ref-39">↵</a><div class="cit ref-cit ref-journal" id="cit-549675v1.39"><div class="cit-metadata"><cite><span class="cit-auth cit-collab">R Core Team</span>. <span class="cit-pub-date">2018</span>. <span class="cit-article-title">R: A language and environment for statistical computing</span>. <abbr class="cit-jnl-abbrev">R Foundation for Statistical Computing, Vienna, Austria</abbr>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">40.</span><a class="rev-xref-ref" href="#xref-ref-40-1" title="View reference 40. in text" id="ref-40">↵</a><div class="cit ref-cit ref-journal" id="cit-549675v1.40" data-doi="10.1111/j.1471-8286.2004.00828.x"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Goudet</span>  <span class="cit-name-given-names">J.</span></span> <span class="cit-pub-date">2005</span>. <span class="cit-article-title">Hierfstat, a package for R to compute and test variance components and F-statistics</span>. <abbr class="cit-jnl-abbrev">Mol Ecol Notes</abbr> <span class="cit-vol">5</span>: <span class="cit-fpage">184</span>–<span class="cit-lpage">186</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol%2BEcol%2BNotes%26rft.volume%253D5%26rft.spage%253D184%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1471-8286.2004.00828.x%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1471-8286.2004.00828.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000227598500059&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">41.</span><a class="rev-xref-ref" href="#xref-ref-41-1" title="View reference 41. in text" id="ref-41">↵</a><div class="cit ref-cit ref-journal" id="cit-549675v1.41" data-doi="10.1890/11-0252.1"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Ellis</span>  <span class="cit-name-given-names">N</span></span>, <span class="cit-auth"><span class="cit-name-surname">Smith</span>  <span class="cit-name-given-names">SJ</span></span>, <span class="cit-auth"><span class="cit-name-surname">Pitcher</span>  <span class="cit-name-given-names">CR.</span></span> <span class="cit-pub-date">2012</span>. <span class="cit-article-title">Calculating importance gradients on physical predictors</span>. <abbr class="cit-jnl-abbrev">Ecology</abbr> <span class="cit-vol">93</span>: <span class="cit-fpage">156</span>–<span class="cit-lpage">168</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEcology%26rft.stitle%253DEcology%26rft.aulast%253DEllis%26rft.auinit1%253DN.%26rft.volume%253D93%26rft.issue%253D1%26rft.spage%253D156%26rft.epage%253D168%26rft.atitle%253DGradient%2Bforests%253A%2Bcalculating%2Bimportance%2Bgradients%2Bon%2Bphysical%2Bpredictors.%26rft_id%253Dinfo%253Adoi%252F10.1890%252F11-0252.1%26rft_id%253Dinfo%253Apmid%252F22486096%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1890/11-0252.1&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22486096&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F02%2F15%2F549675.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">42.</span><a class="rev-xref-ref" href="#xref-ref-42-1" title="View reference 42. in text" id="ref-42">↵</a><div class="cit ref-cit ref-journal" id="cit-549675v1.42"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Hijmans</span>  <span class="cit-name-given-names">RJ.</span></span> <span class="cit-pub-date">2017</span>. <span class="cit-article-title">raster: Geographic Data Analysis and Modeling</span>. <abbr class="cit-jnl-abbrev">R package version 2.6–7</abbr>. <a href="https://CRAN.R-project.org/package=raster">https://CRAN.R-project.org/package=raster</a></cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">43.</span><a class="rev-xref-ref" href="#xref-ref-43-1" title="View reference 43. in text" id="ref-43">↵</a><div class="cit ref-cit ref-journal" id="cit-549675v1.43" data-doi="10.1890/07-1861.1"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">McRae</span>  <span class="cit-name-given-names">BH</span></span>, <span class="cit-auth"><span class="cit-name-surname">Dickson</span>  <span class="cit-name-given-names">BG</span></span>, <span class="cit-auth"><span class="cit-name-surname">Keitt</span>  <span class="cit-name-given-names">T.</span></span> <span class="cit-pub-date">2008</span>. <span class="cit-article-title">Using circuit theory to model connectivity in ecology, evolution, and conservation</span>. <abbr class="cit-jnl-abbrev">Ecology</abbr> <span class="cit-vol">89</span>: <span class="cit-fpage">2712</span>–<span class="cit-lpage">2724</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEcology%26rft.stitle%253DEcology%26rft.aulast%253DMcRae%26rft.auinit1%253DB.%2BH.%26rft.volume%253D89%26rft.issue%253D10%26rft.spage%253D2712%26rft.epage%253D2724%26rft.atitle%253DUsing%2Bcircuit%2Btheory%2Bto%2Bmodel%2Bconnectivity%2Bin%2Becology%252C%2Bevolution%252C%2Band%2Bconservation.%26rft_id%253Dinfo%253Adoi%252F10.1890%252F07-1861.1%26rft_id%253Dinfo%253Apmid%252F18959309%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1890/07-1861.1&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18959309&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F02%2F15%2F549675.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000259995100006&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">44.</span><a class="rev-xref-ref" href="#xref-ref-44-1" title="View reference 44. in text" id="ref-44">↵</a><div class="cit ref-cit ref-journal" id="cit-549675v1.44"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Arteaga</span>  <span class="cit-name-given-names">MC</span></span>, <span class="cit-etal">et al.</span> <span class="cit-pub-date">2016</span>. <span class="cit-article-title">Genomic variation in recently collected maize landraces from Mexico</span>. <abbr class="cit-jnl-abbrev">Genomics Data</abbr> <span class="cit-vol">7</span>: <span class="cit-fpage">38</span>–<span class="cit-lpage">45</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenomics%2BData%26rft.volume%253D7%26rft.spage%253D38%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">45.</span><a class="rev-xref-ref" href="#xref-ref-45-1" title="View reference 45. in text" id="ref-45">↵</a><div class="cit ref-cit ref-journal" id="cit-549675v1.45" data-doi="10.1038/nature09670"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Hoffmann</span>  <span class="cit-name-given-names">AA</span></span>, <span class="cit-auth"><span class="cit-name-surname">Sgrò</span>  <span class="cit-name-given-names">CM.</span></span> <span class="cit-pub-date">2011</span>. <span class="cit-article-title">Climate change and evolutionary adaptation</span>. <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">470</span>: <span class="cit-fpage">479</span>–<span class="cit-lpage">485</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DHoffmann%26rft.auinit1%253DA.%2BA.%26rft.volume%253D470%26rft.issue%253D7335%26rft.spage%253D479%26rft.epage%253D485%26rft.atitle%253DClimate%2Bchange%2Band%2Bevolutionary%2Badaptation.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature09670%26rft_id%253Dinfo%253Apmid%252F21350480%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature09670&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21350480&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F02%2F15%2F549675.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000287652900031&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">46.</span><a class="rev-xref-ref" href="#xref-ref-46-1" title="View reference 46. in text" id="ref-46">↵</a><div class="cit ref-cit ref-journal" id="cit-549675v1.46" data-doi="10.1111/j.1469-8137.2010.03401.x"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Siol</span>  <span class="cit-name-given-names">M</span></span>, <span class="cit-auth"><span class="cit-name-surname">Wright</span>  <span class="cit-name-given-names">SI</span></span>, <span class="cit-auth"><span class="cit-name-surname">Barrett</span>  <span class="cit-name-given-names">SC.</span></span> <span class="cit-pub-date">2010</span>. <span class="cit-article-title">The population genomics of plant adaptation</span>. <abbr class="cit-jnl-abbrev">New Phytol</abbr> <span class="cit-vol">188</span>: <span class="cit-fpage">313</span>–<span class="cit-lpage">332</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNew%2BPhytologist%26rft.stitle%253DNew%2BPhytologist%26rft.aulast%253DSiol%26rft.auinit1%253DM.%26rft.volume%253D188%26rft.issue%253D2%26rft.spage%253D313%26rft.epage%253D332%26rft.atitle%253DThe%2Bpopulation%2Bgenomics%2Bof%2Bplant%2Badaptation.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1469-8137.2010.03401.x%26rft_id%253Dinfo%253Apmid%252F20696011%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1469-8137.2010.03401.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20696011&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F02%2F15%2F549675.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000282225000004&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">47.</span><a class="rev-xref-ref" href="#xref-ref-47-1" title="View reference 47. in text" id="ref-47">↵</a><div class="cit ref-cit ref-journal" id="cit-549675v1.47"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Frankham</span>  <span class="cit-name-given-names">R.</span></span> <span class="cit-pub-date">2010</span>. <span class="cit-article-title">Challenges and opportunities of genetic approaches to biological conservation</span>. <abbr class="cit-jnl-abbrev">Biol Conserv</abbr> <span class="cit-vol">143</span>: <span class="cit-fpage">1919</span>–<span class="cit-lpage">1927</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBiol%2BConserv%26rft.volume%253D143%26rft.spage%253D1919%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">48.</span><a class="rev-xref-ref" href="#xref-ref-48-1" title="View reference 48. in text" id="ref-48">↵</a><div class="cit ref-cit ref-journal" id="cit-549675v1.48" data-doi="10.1098/rspb.2016.2104"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Jezkova</span>  <span class="cit-name-given-names">T</span></span>, <span class="cit-auth"><span class="cit-name-surname">Wiens</span>  <span class="cit-name-given-names">JJ.</span></span> <span class="cit-pub-date">2016</span>. <span class="cit-article-title">Rates of change in climatic niches in plant and animal populations are much slower than projected climate change</span>. <abbr class="cit-jnl-abbrev">Proc Royal Soc B</abbr> <span class="cit-vol">283</span>: <span class="cit-fpage">20162104</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc%2BRoyal%2BSoc%2BB%26rft_id%253Dinfo%253Adoi%252F10.1098%252Frspb.2016.2104%26rft_id%253Dinfo%253Apmid%252F27881748%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1098/rspb.2016.2104&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=27881748&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F02%2F15%2F549675.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">49.</span><a class="rev-xref-ref" href="#xref-ref-49-1" title="View reference 49. in text" id="ref-49">↵</a><div class="cit ref-cit ref-journal" id="cit-549675v1.49"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Ackerly</span>  <span class="cit-name-given-names">DD</span></span>, <span class="cit-etal">et al.</span> <span class="cit-pub-date">2010</span>. <span class="cit-article-title">The geography of climate change: implications for conservation biogeography</span>. <abbr class="cit-jnl-abbrev">Divers Distrib</abbr> <span class="cit-vol">16</span>:<span class="cit-fpage">476</span>–<span class="cit-lpage">487</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DDivers%2BDistrib%26rft.volume%253D16%26rft.spage%253D476%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">50.</span><a class="rev-xref-ref" href="#xref-ref-50-1" title="View reference 50. in text" id="ref-50">↵</a><div class="cit ref-cit ref-journal" id="cit-549675v1.50" data-doi="10.1016/S0169-5347(03)00008-9"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Manel</span>  <span class="cit-name-given-names">S</span></span>, <span class="cit-auth"><span class="cit-name-surname">Schwartz</span>  <span class="cit-name-given-names">MK</span></span>, <span class="cit-auth"><span class="cit-name-surname">Luikart</span>  <span class="cit-name-given-names">G</span></span>, <span class="cit-auth"><span class="cit-name-surname">Taberlet</span>  <span class="cit-name-given-names">P.</span></span> <span class="cit-pub-date">2003</span>. <span class="cit-article-title">Landscape genetics: combining landscape ecology and population genetics</span>. <abbr class="cit-jnl-abbrev">Trends Ecol Evol</abbr> <span class="cit-vol">18</span>: <span class="cit-fpage">189</span>–<span class="cit-lpage">197</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTrends%2BEcol%2BEvol%26rft.volume%253D18%26rft.spage%253D189%26rft_id%253Dinfo%253Adoi%252F10.1016%252FS0169-5347%252803%252900008-9%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/S0169-5347(03)00008-9&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000182266700012&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">51.</span><a class="rev-xref-ref" href="#xref-ref-51-1" title="View reference 51. in text" id="ref-51">↵</a><div class="cit ref-cit ref-journal" id="cit-549675v1.51" data-doi="10.1038/nature08649"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Loarie</span>  <span class="cit-name-given-names">SR</span></span>, <span class="cit-etal">et al.</span> <span class="cit-pub-date">2009</span>. <span class="cit-article-title">The velocity of climate change</span>. <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">462</span>: <span class="cit-fpage">1052</span>–<span class="cit-lpage">1055</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DLoarie%26rft.auinit1%253DS.%2BR.%26rft.volume%253D462%26rft.issue%253D7276%26rft.spage%253D1052%26rft.epage%253D1055%26rft.atitle%253DThe%2Bvelocity%2Bof%2Bclimate%2Bchange.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature08649%26rft_id%253Dinfo%253Apmid%252F20033047%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature08649&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=2010018076&amp;link_type=GEOREF" class="cit-ref-sprinkles cit-ref-sprinkles-georef"><span>GeoRef</span></a><a href="/lookup/external-ref?access_num=20033047&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F02%2F15%2F549675.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000272996000046&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">52.</span><a class="rev-xref-ref" href="#xref-ref-52-1" title="View reference 52. in text" id="ref-52">↵</a><div class="cit ref-cit ref-journal" id="cit-549675v1.52"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">L</span></span>, <span class="cit-auth"><span class="cit-name-surname">Yang</span>  <span class="cit-name-given-names">A</span></span>, <span class="cit-auth"><span class="cit-name-surname">He</span>  <span class="cit-name-given-names">C</span></span>, <span class="cit-auth"><span class="cit-name-surname">Qu</span>  <span class="cit-name-given-names">M</span></span>, <span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">J.</span></span> <span class="cit-pub-date">2008</span>. <span class="cit-article-title">Creation of new maize germplasm using alien introgression from Zea mays ssp. mexicana</span>. <abbr class="cit-jnl-abbrev">Euphytica</abbr> <span class="cit-vol">164</span>: <span class="cit-fpage">789</span>–<span class="cit-lpage">801</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEuphytica%26rft.volume%253D164%26rft.spage%253D789%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">53.</span><a class="rev-xref-ref" href="#xref-ref-53-1" title="View reference 53. in text" id="ref-53">↵</a><div class="cit ref-cit ref-journal" id="cit-549675v1.53"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Moreno-Letelier</span>  <span class="cit-name-given-names">A</span></span>, <span class="cit-etal">et al.</span> <span class="cit-pub-date">2017</span>. <span class="cit-article-title">Was maize domesticated in the Balsas Basin? Complex patterns of genetic divergence, gene flow and ancestral introgressions among Zea subspecies suggest an alternative scenario</span>. <abbr class="cit-jnl-abbrev">bioRxiv</abbr>: <span class="cit-fpage">239707</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">54.</span><a class="rev-xref-ref" href="#xref-ref-54-1" title="View reference 54. in text" id="ref-54">↵</a><div class="cit ref-cit ref-journal" id="cit-549675v1.54" data-doi="10.1534/genetics.113.159319"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Yoder</span>  <span class="cit-name-given-names">JB</span></span>, <span class="cit-etal">et al.</span> <span class="cit-pub-date">2014</span>. <span class="cit-article-title">Genomic signature of adaptation to climate in Medicago truncatula</span>. <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">196</span>: <span class="cit-fpage">1263</span>–<span class="cit-lpage">1275</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.113.159319%26rft_id%253Dinfo%253Apmid%252F24443444%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE5Ni80LzEyNjMiO3M6NDoiYXRvbSI7czozNzoiL2Jpb3J4aXYvZWFybHkvMjAxOS8wMi8xNS81NDk2NzUuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">55.</span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-549675v1.55" data-doi="10.1016/j.tree.2014.10.004"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Tiffin</span>  <span class="cit-name-given-names">P</span></span>, <span class="cit-auth"><span class="cit-name-surname">Ross-Ibarra</span>  <span class="cit-name-given-names">J.</span></span> <span class="cit-pub-date">2014</span>. <span class="cit-article-title">Advances and limits of using population genetics to understand local adaptation</span>. <abbr class="cit-jnl-abbrev">Trends Ecol Evol</abbr> <span class="cit-vol">29</span>: <span class="cit-fpage">673</span>–<span class="cit-lpage">680</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTrends%2BEcol%2BEvol%26rft.volume%253D29%26rft.spage%253D673%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.tree.2014.10.004%26rft_id%253Dinfo%253Apmid%252F25454508%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.tree.2014.10.004&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25454508&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F02%2F15%2F549675.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">56.</span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-549675v1.56" data-doi="10.1111/j.1461-0248.2004.00684.x"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Kawecki</span>  <span class="cit-name-given-names">TJ</span></span>, <span class="cit-auth"><span class="cit-name-surname">Ebert</span>  <span class="cit-name-given-names">D.</span></span> <span class="cit-pub-date">2004</span>. <span class="cit-article-title">Conceptual issues in local adaptation</span>. <abbr class="cit-jnl-abbrev">Ecol Lett</abbr> <span class="cit-vol">7</span>: <span class="cit-fpage">1225</span>–<span class="cit-lpage">1241</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEcol%2BLett%26rft.volume%253D7%26rft.spage%253D1225%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1461-0248.2004.00684.x%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1461-0248.2004.00684.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000225078000013&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">57.</span><a class="rev-xref-ref" href="#xref-ref-57-1" title="View reference 57. in text" id="ref-57">↵</a><div class="cit ref-cit ref-journal" id="cit-549675v1.57" data-doi="10.1086/682405"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Yeaman</span>  <span class="cit-name-given-names">S.</span></span> <span class="cit-pub-date">2015</span>. <span class="cit-article-title">Local adaptation by alleles of small effect</span>. <abbr class="cit-jnl-abbrev">Am Nat</abbr> <span class="cit-vol">186</span>: <span class="cit-fpage">S74</span>–<span class="cit-lpage">S89</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAm%2BNat%26rft.volume%253D186%26rft.spage%253DS74%26rft_id%253Dinfo%253Adoi%252F10.1086%252F682405%26rft_id%253Dinfo%253Apmid%252F26656219%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1086/682405&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=26656219&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F02%2F15%2F549675.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">58.</span><a class="rev-xref-ref" href="#xref-ref-58-1" title="View reference 58. in text" id="ref-58">↵</a><div class="cit ref-cit ref-journal" id="cit-549675v1.58" data-doi="10.1126/science.aat7279"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Fitzpatrick</span>  <span class="cit-name-given-names">MC</span></span>, <span class="cit-auth"><span class="cit-name-surname">Keller</span>  <span class="cit-name-given-names">SR</span></span>, <span class="cit-auth"><span class="cit-name-surname">Lotterhos</span>  <span class="cit-name-given-names">KE.</span></span> <span class="cit-pub-date">2018</span>. <span class="cit-article-title">Comment on “Genomic signals of selection predict climate-driven population declines in a migratory bird”</span>. <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">361</span>: <span class="cit-fpage">eaat7279</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DFitzpatrick%26rft.auinit1%253DM.%2BC.%26rft.volume%253D361%26rft.issue%253D6401%26rft.spage%253Deaat7279%26rft.epage%253Deaat7279%26rft.atitle%253DComment%2Bon%2B%2522Genomic%2Bsignals%2Bof%2Bselection%2Bpredict%2Bclimate-driven%2Bpopulation%2Bdeclines%2Bin%2Ba%2Bmigratory%2Bbird%2522%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.aat7279%26rft_id%253Dinfo%253Apmid%252F30072513%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjE3OiIzNjEvNjQwMS9lYWF0NzI3OSI7czo0OiJhdG9tIjtzOjM3OiIvYmlvcnhpdi9lYXJseS8yMDE5LzAyLzE1LzU0OTY3NS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.biorxiv.org/content/10.1101/549675v1.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-disqus-comment" >
  
      
  
  <div class="pane-content">
    <div id="disqus_thread"><noscript><p><a href="http://biorxivstage.disqus.com/?url=https%3A%2F%2Fwww.biorxiv.org%2Fcontent%2F10.1101%2F549675v1" class="" data-icon-position="" data-hide-link-title="0">View the discussion thread.</a></p></noscript></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/10.1101/552083v1" title="Retinoic acid promotes in vitro development of haploid germ cells from pre-pubertal porcine spermatogenic cells" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous</span></a></span><span class="pager-next"><a href="/content/10.1101/283929v2-0" title="Cell-type heterogeneity in adipose tissue is associated with complex traits and reveals disease-relevant cell-specific eQTLs" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    Posted&nbsp;February 15, 2019.  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-biorxiv-art-tools" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-biorxiv_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/10.1101/549675v1.full.pdf" target="_self" class="article-dl-pdf-link link-icon"><i class="icon-external-link-sign"></i> <span class="title">Download PDF</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-biorxiv-supplementary-fragment" >
  
      
  
  <div class="pane-content">
    <p><a href="/content/10.1101/549675v1.supplementary-material"><i class="icon-file"></i>Supplementary Material </a></p>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-biorxiv-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="Email this Article" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-biorxiv_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/10.1101/549675v1.full" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for your interest in spreading the word about bioRxiv.</p><p>NOTE: Your email address is requested solely to identify you as the sender of this article.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/10.1101/549675v1" class="active" data-icon-position="" data-hide-link-title="0">Climate change is predicted to disrupt patterns of local adaptation in wild and cultivated maize</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from bioRxiv
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would like to see this page from the bioRxiv website.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/176518" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-kuB5ngzRVri7VJaT9yonMzcLWcXM1ts_LFgiotUlgUM" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-biorxiv-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-biorxiv_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="176518" id="node176518--2" data-pisa="biorxiv;549675v1" data-pisa-master="biorxiv;549675" data-seqnum="176518" data-apath="/biorxiv/early/2019/02/15/549675.atom"><div  class="highwire-cite highwire-cite-highwire-article highwire-citation-biorxiv-article-pap-list clearfix" >

      <div class="highwire-cite-title" >
      <div class="highwire-cite-title">Climate change is predicted to disrupt patterns of local adaptation in wild and cultivated maize</div>    </div>
  
      <div  class="highwire-cite-authors" ><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0"><span class="nlm-given-names">Jonás A</span> <span class="nlm-surname">Aguirre-Liguori</span></span>, <span class="highwire-citation-author" data-delta="1"><span class="nlm-given-names">Santiago</span> <span class="nlm-surname">Ramírez-Barahona</span></span>, <span class="highwire-citation-author" data-delta="2"><span class="nlm-given-names">Peter</span> <span class="nlm-surname">Tiffin</span></span>, <span class="highwire-citation-author" data-delta="3"><span class="nlm-given-names">Luis E</span> <span class="nlm-surname">Eguiarte</span></span></span></div>
  
      <div  class="highwire-cite-metadata" ><span  class="highwire-cite-metadata-journal highwire-cite-metadata">bioRxiv </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">549675; </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata"><span class="doi_label">doi:</span> https://doi.org/10.1101/549675 </span></div>
  
  
  </div>
</div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.biorxiv.org/content/10.1101/549675v1" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.biorxiv.org/content/10.1101/549675v1&amp;title=Climate%20change%20is%20predicted%20to%20disrupt%20patterns%20of%20local%20adaptation%20in%20wild%20and%20cultivated%20maize" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.biorxiv.org/content/10.1101/549675v1&amp;title=Climate%20change%20is%20predicted%20to%20disrupt%20patterns%20of%20local%20adaptation%20in%20wild%20and%20cultivated%20maize" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.biorxiv.org/content/10.1101/549675v1&amp;text=Climate%20change%20is%20predicted%20to%20disrupt%20patterns%20of%20local%20adaptation%20in%20wild%20and%20cultivated%20maize" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.biorxiv.org/content/10.1101/549675v1&amp;title=Climate%20change%20is%20predicted%20to%20disrupt%20patterns%20of%20local%20adaptation%20in%20wild%20and%20cultivated%20maize" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.biorxiv.org/content/10.1101/549675v1&amp;t=Climate%20change%20is%20predicted%20to%20disrupt%20patterns%20of%20local%20adaptation%20in%20wild%20and%20cultivated%20maize" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.biorxiv.org/content/10.1101/549675v1&amp;title=Climate%20change%20is%20predicted%20to%20disrupt%20patterns%20of%20local%20adaptation%20in%20wild%20and%20cultivated%20maize" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.biorxiv.org/content/10.1101/549675v1&amp;title=Climate%20change%20is%20predicted%20to%20disrupt%20patterns%20of%20local%20adaptation%20in%20wild%20and%20cultivated%20maize" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-biorxiv-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title="Citation Tools"><i class="icon-globe"></i> <span class="title">Citation Tools</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-biorxiv_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="176518" id="node176518--3" data-pisa="biorxiv;549675v1" data-pisa-master="biorxiv;549675" data-seqnum="176518" data-apath="/biorxiv/early/2019/02/15/549675.atom"><div  class="highwire-cite highwire-cite-highwire-article highwire-citation-biorxiv-article-pap-list clearfix" >

      <div class="highwire-cite-title" >
      <div class="highwire-cite-title">Climate change is predicted to disrupt patterns of local adaptation in wild and cultivated maize</div>    </div>
  
      <div  class="highwire-cite-authors" ><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0"><span class="nlm-given-names">Jonás A</span> <span class="nlm-surname">Aguirre-Liguori</span></span>, <span class="highwire-citation-author" data-delta="1"><span class="nlm-given-names">Santiago</span> <span class="nlm-surname">Ramírez-Barahona</span></span>, <span class="highwire-citation-author" data-delta="2"><span class="nlm-given-names">Peter</span> <span class="nlm-surname">Tiffin</span></span>, <span class="highwire-citation-author" data-delta="3"><span class="nlm-given-names">Luis E</span> <span class="nlm-surname">Eguiarte</span></span></span></div>
  
      <div  class="highwire-cite-metadata" ><span  class="highwire-cite-metadata-journal highwire-cite-metadata">bioRxiv </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">549675; </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata"><span class="doi_label">doi:</span> https://doi.org/10.1101/549675 </span></div>
  
  
  </div>
</div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage&amp;rft.epage&amp;rft.atitle=Climate%20change%20is%20predicted%20to%20disrupt%20patterns%20of%20local%20adaptation%20in%20wild%20and%20cultivated%20maize&amp;rft.volume&amp;rft.issue&amp;rft.date=2019-01-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=bioRxiv&amp;rft.au=Aguirre-Liguori%2C+Jon%C3%A1s+A&amp;rft.au=Ram%C3%ADrez-Barahona%2C+Santiago&amp;rft.au=Tiffin%2C+Peter&amp;rft.au=Eguiarte%2C+Luis+E"></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/176518/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/176518/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/176518/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/176518/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/176518/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/176518/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/176518/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/176518/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/176518/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/176518/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/176518/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/176518/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><div class="item-list"><ul><li class="first"><a href="http://twitter.com/share?url=https%3A//www.biorxiv.org/content/10.1101/549675v1&amp;count=horizontal&amp;via=&amp;text=Climate%20change%20is%20predicted%20to%20disrupt%20patterns%20of%20local%20adaptation%20in%20wild%20and%20cultivated%20maize&amp;counturl=https%3A//www.biorxiv.org/content/10.1101/549675v1" class="twitter-share-button service-links-twitter-widget" id="twitter_widget" title="Tweet This" rel="nofollow" data-icon-position="" data-hide-link-title="0"><span class="element-invisible">Tweet Widget</span></a></li><li><a href="http://www.facebook.com/plugins/like.php?href=https%3A//www.biorxiv.org/content/10.1101/549675v1&amp;layout=button_count&amp;show_faces=false&amp;action=like&amp;colorscheme=light&amp;width=100&amp;height=21&amp;font=&amp;locale=" id="facebook_like" title="I Like it" class="service-links-facebook-like" rel="nofollow" data-icon-position="" data-hide-link-title="0"><span class="element-invisible">Facebook Like</span></a></li><li class="last"><a href="https://www.biorxiv.org/content/10.1101/549675v1" id="google_plus_one" title="Plus it" class="service-links-google-plus-one" rel="nofollow" data-icon-position="" data-hide-link-title="0"><span class="element-invisible">Google Plus One</span></a></li></ul></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-collections" >
  
        <h2 class="pane-title">Subject Area</h2>
    
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-collections"><div class="highwire-list"><ul class="highwire-article-collection-term-list"><li class="first last odd"><span class="highwire-article-collection-term"><a href="/collection/evolutionary-biology" class="highlight" data-icon-position="" data-hide-link-title="0">Evolutionary Biology<i class="icon-caret-right"></i>
</a></span></li></ul></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-biorxiv-subject-collections block-style-col2" >
  
      
  
  <div class="pane-content">
    <div class="panel-flexible panels-flexible-8 clearfix" id="mini-panel-biorxiv_subject_collections">
<div class="panel-flexible-inside panels-flexible-8-inside">
<div class="panels-flexible-region panels-flexible-region-8-center panels-flexible-region-first panels-flexible-region-last">
  <div class="inside panels-flexible-region-inside panels-flexible-region-8-center-inside panels-flexible-region-inside-first panels-flexible-region-inside-last">
<div class="panel-pane pane-snippet" >
  
      
  
  <div class="pane-content">
    <div class="snippet biorxiv-subject-areas-table-title" id="biorxiv-subject-areas-table-title">
  
      
  <div class="snippet-content">
    <b>Subject Areas</b>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-snippet" >
  
      
  
  <div class="pane-content">
    <div class="snippet biorxiv-subject-areas-view-papers" id="biorxiv-subject-areas-view-papers">
  
      
  <div class="snippet-content">
    <a href="/content/early/recent"><strong>All Articles</strong></a>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-subject-collections" >
  
      
  
  <div class="pane-content">
    <ul id="collection" class="collection highwire-list-expand"><li class="outer collection depth-2  child first"><div class = "data-wrapper"><a href="/collection/animal-behavior-and-cognition" class="" data-icon-position="" data-hide-link-title="0">Animal Behavior and Cognition</a> <span class = "article-count">(1781)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/biochemistry" class="" data-icon-position="" data-hide-link-title="0">Biochemistry</a> <span class = "article-count">(3107)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/bioengineering" class="" data-icon-position="" data-hide-link-title="0">Bioengineering</a> <span class = "article-count">(2119)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/bioinformatics" class="" data-icon-position="" data-hide-link-title="0">Bioinformatics</a> <span class = "article-count">(11020)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/biophysics" class="" data-icon-position="" data-hide-link-title="0">Biophysics</a> <span class = "article-count">(4543)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/cancer-biology" class="" data-icon-position="" data-hide-link-title="0">Cancer Biology</a> <span class = "article-count">(3547)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/cell-biology" class="" data-icon-position="" data-hide-link-title="0">Cell Biology</a> <span class = "article-count">(5028)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/clinical-trials" class="" data-icon-position="" data-hide-link-title="0">Clinical Trials</a> <span class = "article-count">(137)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/developmental-biology" class="" data-icon-position="" data-hide-link-title="0">Developmental Biology</a> <span class = "article-count">(3100)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/ecology" class="" data-icon-position="" data-hide-link-title="0">Ecology</a> <span class = "article-count">(4884)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/epidemiology" class="" data-icon-position="" data-hide-link-title="0">Epidemiology</a> <span class = "article-count">(2044)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/evolutionary-biology" class="" data-icon-position="" data-hide-link-title="0">Evolutionary Biology</a> <span class = "article-count">(7665)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/genetics" class="" data-icon-position="" data-hide-link-title="0">Genetics</a> <span class = "article-count">(5808)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/genomics" class="" data-icon-position="" data-hide-link-title="0">Genomics</a> <span class = "article-count">(7279)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/immunology" class="" data-icon-position="" data-hide-link-title="0">Immunology</a> <span class = "article-count">(2698)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/microbiology" class="" data-icon-position="" data-hide-link-title="0">Microbiology</a> <span class = "article-count">(8312)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/molecular-biology" class="" data-icon-position="" data-hide-link-title="0">Molecular Biology</a> <span class = "article-count">(3313)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/neuroscience" class="" data-icon-position="" data-hide-link-title="0">Neuroscience</a> <span class = "article-count">(20149)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/paleontology" class="" data-icon-position="" data-hide-link-title="0">Paleontology</a> <span class = "article-count">(147)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/pathology" class="" data-icon-position="" data-hide-link-title="0">Pathology</a> <span class = "article-count">(522)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/pharmacology-and-toxicology" class="" data-icon-position="" data-hide-link-title="0">Pharmacology and Toxicology</a> <span class = "article-count">(856)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/physiology" class="" data-icon-position="" data-hide-link-title="0">Physiology</a> <span class = "article-count">(1293)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/plant-biology" class="" data-icon-position="" data-hide-link-title="0">Plant Biology</a> <span class = "article-count">(3001)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/scientific-communication-and-education" class="" data-icon-position="" data-hide-link-title="0">Scientific Communication and Education</a> <span class = "article-count">(731)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/synthetic-biology" class="" data-icon-position="" data-hide-link-title="0">Synthetic Biology</a> <span class = "article-count">(971)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/systems-biology" class="" data-icon-position="" data-hide-link-title="0">Systems Biology</a> <span class = "article-count">(3023)</span></div></li>
<li class="outer collection depth-2  child last"><div class = "data-wrapper"><a href="/collection/zoology" class="" data-icon-position="" data-hide-link-title="0">Zoology</a> <span class = "article-count">(502)</span></div></li>
</ul>  </div>

  
  </div>
  </div>
</div>
</div>
</div>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__VNH5GD8Zz7g6_hGCOZjVjdMndxxC6naiExSSJKX3k_A__kzmEdtbqCsJx5Z9yg2Qy0PptB__ufXm-TxUOl0KZzUw__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js" defer="defer"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__llgGUcz2-k3Jmyyyog-_4yI1wJfGWwx0eQ2lbnM9IIo__t6UjIjIKIAD6nKc6BAU9qmlVV2VR4CBTNQbxlbJPgN0__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
function euCookieComplianceLoadScripts() {}
//--><!]]>
</script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var eu_cookie_compliance_cookie_name = "";
//--><!]]>
</script>
<script type="text/javascript" src="//d33xdlntwy0kbs.cloudfront.net/cshl_custom.js"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__BmqjBnkz3MgYCAoc25s1lDRMEjLhC3mEPVonUFIHi08__Unwv5-ZIuHBfFwytsjEx1niBVJ7n1T4lPws7VrkHXM4__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
  </body>
</html>
