<html xmlns="http://www.w3.org/1999/xhtml"><head> <style type="text/css">
         	     		 
         	    a                          {background : #ffffff; }
         	    article                    {border-style : dotted; border-width : 2px; }
         		 	
         		 	div                        {background : #ffffcc;}
         		 	div.abstract-title         {font-weight : bold ; font-size : 16pt;}
         		 	div.ack                    {border-style : solid ; border-color : red; margin
         : 2em; }
         		 	div.article-meta           {border-style : solid; border-width : 1 pt; margin
         1em;}
         		 	div.boxed-text             {margin : 5em; border-style : solid;}
         		 	div.contrib-group          {margin : 1em; }
         		 	div.fig                    {border-style : solid; border-width : 2px; margin :
         2em; }
         		 	div.funding                {font-weight : bold ; font-size : 16pt;}
         		 	div.given-names            {font-style : italic;}
         		 	div.intro                  {border-style : inset; margin : 5px;}
         		 	div.introduction           {border-style : inset; margin : 5px;}
         		 	div.journal-meta           {border-style : solid; border-width : 1 pt; margin
         1em;}
         		 	div.journal-title-group    {background : #ffeeee;}
         		 	div.article-title          {font-weight : bold ; font-size : 18pt;}
         		 	div.kwd                    {font-style : italic}
         		 	div.meta-name              {font-weight : bold ; font-size : 16pt;}
         		 	div.name                   {font-weight : bold;}
         		 	div.alt-title              {font-style : italic; font-size : 12px;}
         		 	
         		 	div.sec                    {border : 2px; margin 5px; padding 2px;}
         		 	div.title                  {font-family : courier; font-weight : bold;
         		 	                            font-size : 16pt; margin : 5px;}
         		 	
         		 	div.materials_methods      {border-style : double; margin : 5px; }
         		 	div.methods                {border-style : double; margin : 5px; }
         		 	
         		 	div.results                {border-style : solid; margin : 5px;}
         		 	div.background             {border-style : dotted; margin : 5px;}
         		 	
         		 	div.discussion             {border-style : groove; margin : 5px;}
         		 	div.conclusion             {border-style : ridge; margin : 5px;}
         		 	div.conclusions             {border-style : ridge; margin : 5px;}
         		 	div.supplementary-material {border-style : inset; margin : 5px;}
         		 	div.abbreviations          {border-style : double; border-color : red; margin
         : 5px;}
         		 	div.competinginterests     {border-style : double; border-color : blue; margin
         : 5px;}
         		 	div.acknowledgements       {border-style : double; border-color : green; margin
         : 5px;}
         		 	div.authors_contributions   {border-style : double; border-color : purple; margin
         : 5px;}
         		 	
         		 	div.publisher              {border-style : outset; margin : 5px;}
         		 	div.fn-type-conflict       {background : #f88; }
         		 	div.fn-type-con            {background : #ddf; }
         		 	div.fn-type-other          {background : #ddd; }
         		 	
         		 	div.unknown                {background : #ffd;
         									 	  border-style : solid;
         									 	  border-width : 1px;
         									 	  padding : 2 px;}
         		 	  
         	    table                      {background : #ffffdd;}
         		 	tr                         {background : #ddddff; padding : 1px;}
         		 	
         		 	span                       {background : #ffcccc;}
         		 	
         		 	span.citation-author       {font-family : helvetica; background : #ffeeee;}
         		 	span.collab                {background : #ddffff; }
         		 	span.comment               {font-family : courier; font-size : 6px; background
         : #ffaaff;}
         		 	span.contrib               {background : #ffffff;}
         		 	span.corresp               {background : #ddffdd; }
         		 	span.doi                   {background : #ffffff;}
         		 	span.email                 {font-family : courier; }
         		 	span.etal                  {font-style : italic;}
         		 	span.fpage                 {font-family : courier;}
         		 	span.given-names           {background : #ffffff;}
         		 	span.iso-abbrev            {background : #ffffff;}
         		 	span.issn-epub             {background : #ffffff;}
         		 	span.issn-ppub             {background : #ffffff;}
         		 	span.journal-title         {background : #ffffff;}
         		 	span.lpage                 {font-family : courier;}
         		 	span.mixed-article-title   {font-style : italic ;}
         		 	span.nlm-ta                {background : #ffffff;}
         		 	span.pmc                   {background : #ffffff;}
         		 	span.pmcid                 {background : #ffffff;}
         		 	span.pmid                  {background : #ffffff;}
         		 	span.publisher             {background : #ffffff;}
         		 	span.publisher-id          {background : #ffffff;}
         		 	span.publisher-name        {background : #ffffff;}
         		 	span.source                {background : #ffffff;}
         		 	span.subject               {background : #ffffff;}
         		 	span.surname               {background : #ffffff;}
         		 	span.volume                {font-family : courier; font-weight : bold;}
         		 	span.year                  {font-family : courier ; font-style : italic;}
         			
      </style> </head> <body> <div class="front" title="front"> <div class="journal-meta" tagx="journal-meta" title="journal-meta"> <span class="nlm-ta" title="nlm-ta">Sci Rep</span> <span class="iso-abbrev" title="iso-abbrev">Sci Rep</span> <div class="journal-title-group" tagx="journal-title-group" title="journal-title-group"> <span class="journal-title" tagx="journal-title" title="journal-title">Scientific Reports</span> </div> <span class="issn-epub" tagx="issn" title="issn-epub">2045-2322</span> <div class="publisher" tagx="publisher" title="publisher"> <span class="publisher-name" tagx="publisher-name" title="publisher-name">Nature Publishing Group UK</span> <span class="publisher-loc" tagx="publisher-loc" title="publisher-loc">London</span> </div> </div> <div class="article-meta" tagx="article-meta" title="article-meta"> <span class="pmcid" title="pmcid"> pmcid: <a href="http://www.ncbi.nlm.nih.gov/pubmed/5548756">5548756</a> </span> <span class="publisher-id" title="publisher-id">7891</span> <span class="doi" title="doi"> doi: <a href="https://dx.doi.org/10.1038/s41598-017-07891-5">10.1038/s41598-017-07891-5</a> </span> <div class="article-categories" title="article-categories">
: <span class="subject" title="subject">Article</span> </div> <div class="title-group"> <div class="article-title" title="article-title">
Chloroplast genomes of <i>Arabidopsis halleri</i> ssp. <i>gemmifera</i> and <i>Arabidopsis lyrata</i> ssp. <i>petraea:</i> Structures and comparative analysis </div> </div> <div class="contrib-group" title="contrib-group"> <span class="contrib" title="contrib"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Asaf</span> <span class="given-names" tagx="given-names" title="given-names">Sajjad</span> </span> <a href="#Aff1">1</a> </span> <span class="contrib" title="contrib"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Khan</span> <span class="given-names" tagx="given-names" title="given-names">Abdul Latif</span> </span> <a href="#Aff2">2</a> </span> <span class="contrib" title="contrib"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Khan</span> <span class="given-names" tagx="given-names" title="given-names">Muhammad Aaqil</span> </span> <a href="#Aff1">1</a> </span> <span class="contrib" title="contrib"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Waqas</span> <span class="given-names" tagx="given-names" title="given-names">Muhammad</span> </span> <a href="#Aff1">1</a> </span> <span class="contrib" title="contrib"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Kang</span> <span class="given-names" tagx="given-names" title="given-names">Sang-Mo</span> </span> <a href="#Aff1">1</a> </span> <span class="contrib" title="contrib"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Yun</span> <span class="given-names" tagx="given-names" title="given-names">Byung-Wook</span> </span> <a href="#Aff1">1</a> </span> <span class="contrib" title="contrib"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Lee</span> <span class="given-names" tagx="given-names" title="given-names">In-Jung</span> </span> <span class="address" title="address"> <span class="email" tagx="email" title="email">ijlee@knu.ac.kr</span> </span> <a href="#Aff1">1</a> </span> <span class="citation_author_institution" id="Aff1">[1], <div class="institution-wrap" title="institution-wrap"> <span class="institution-id" title="institution-id">0000 0001 0661 1556</span> <span class="institution-id" title="institution-id">grid.258803.4</span> <span class="institution" title="institution">School of Applied Biosciences, </span> <span class="institution" title="institution">Kyungpook National University, </span> </div> </span> <span class="citation_author_institution" id="Aff2">[2], <div class="institution-wrap" title="institution-wrap"> <span class="institution-id" title="institution-id">grid.444752.4</span> <span class="institution" title="institution">Chair of Oman’s Medicinal Plants &amp;amp; Marine Natural Products, </span> <span class="institution" title="institution">University of Nizwa, </span> </div> </span> </div> <span class="pub-date-epub" title="pub-date-epub">epub: <span>2017-8-8</span> </span> <span class="pub-date-pmc-release" title="pub-date-pmc-release">pmc-release: <span>2017-8-8</span> </span> <span class="pub-date-collection" title="pub-date-collection">collection: <span>2017</span> </span> <span class="volume" tagx="volume" title="volume">7</span> <span class="elocation-id" tagx="elocation-id" title="elocation-id">7556</span> <span class="history" title="history"> <span class="received" title="received">received: 2016-10-25</span> <span class="accepted" title="accepted">accepted: 2017-7-5</span> </span> <div class="permissions"> <span class="copyright" title="copyright">(C) , </span> <span class="license" title="license"> <span class="license-p" title="license-p"> <b>Open Access</b> This article is licensed under a Creative Commons Attribution 4.0 International License, which permits use, sharing, adaptation, distribution and reproduction in any medium or format, as long as you give appropriate credit to the original author(s) and the source, provide a link to the Creative Commons license, and indicate if changes were made. The images or other third party material in this article are included in the article’s Creative Commons license, unless indicated otherwise in a credit line to the material. If material is not included in the article’s Creative Commons license and your intended use is not permitted by statutory regulation or exceeds the permitted use, you will need to obtain permission directly from the copyright holder. To view a copy of this license, visit <a href="http://creativecommons.org/licenses/by/4.0/">http://creativecommons.org/licenses/by/4.0/</a>. </span> </span> </div> <div class="abstract" title="abstract"> <div class="abstract-title" title="abstract-title">
Abstract</div> <p>We investigated the complete chloroplast (cp) genomes of non-model <i>Arabidopsis halleri</i> ssp. <i>gemmifera</i> and <i>Arabidopsis lyrata</i> ssp. <i>petraea</i> using Illumina paired-end sequencing to understand their genetic organization and structure. Detailed bioinformatics analysis revealed genome sizes of both subspecies ranging between 154.4~154.5 kbp, with a large single-copy region (84,197~84,158 bp), a small single-copy region (17,738~17,813 bp) and pair of inverted repeats (IRa/IRb; 26,264~26,259 bp). Both cp genomes encode 130 genes, including 85 protein-coding genes, eight ribosomal RNA genes and 37 transfer RNA genes. Whole cp genome comparison of <i>A</i>. <i>halleri</i> ssp. <i>gemmifera</i> and <i>A</i>. <i>lyrata</i> ssp. <i>petraea</i>, along with ten other <i>Arabidopsis</i> species, showed an overall high degree of sequence similarity, with divergence among some intergenic spacers. The location and distribution of repeat sequences were determined, and sequence divergences of shared genes were calculated among related species. Comparative phylogenetic analysis of the entire genomic data set and 70 shared genes between both cp genomes confirmed the previous phylogeny and generated phylogenetic trees with the same topologies. The sister species of <i>A</i>. <i>halleri</i> ssp. <i>gemmifera is A</i>. <i>umezawana</i>, whereas the closest relative of <i>A</i>. <i>lyrata spp</i>. <i>petraea</i> is <i>A</i>. <i>arenicola</i>. </p> </div> <div class="custom-meta-group" title="custom-meta-group"> <meta name="issue-copyright-statement" value="© The Author(s) 2017" /> </div> </div> </div> <div class="body" title="body"> <div class="introduction" title="introduction"> <div class="title" tagx="title" title="title">
Introduction</div> <p>The chloroplast is an active metabolic centre that paves the way for sustaining plant growth and development by converting solar energy to carbohydrates through the process of photosynthesis <sup> <a href="#CR1">1</a>, <a href="#CR2">2</a> </sup>. It serves as a metabolic epicentre in cellular metabolic reactions <sup> <a href="#CR2">2</a> </sup>. The chloroplast (cp) genome is composed of genetic information that encodes synthesis of many key proteins involved in photosynthesis and metabolomic processes <sup> <a href="#CR1">1</a>, <a href="#CR2">2</a> </sup>. In addition, the cp genome contains valuable information for plant systematics, as it is inherited maternally in most angiosperms <sup> <a href="#CR3">3</a>, <a href="#CR4">4</a> </sup>. Substitution rates in plant cp genomes are much lower than those in nuclear genomes <sup> <a href="#CR5">5</a> </sup>. Plant cp genomes are also valuable sources of genetic markers for phylogenetic analyses because of their very low levels of recombination <sup> <a href="#CR6">6</a>, <a href="#CR7">7</a> </sup>. </p> <p>The advent of high-throughput sequencing technologies has facilitated rapid progress in the field of chloroplast genomics <sup> <a href="#CR7">7</a>, <a href="#CR8">8</a> </sup>. While the first complete nucleotide sequences of <i>Nicotiana tabacum</i> and <i>Marchantia polymorpha</i> were painstakingly generated using clone sequencing of plasmid and cosmid libraries <sup> <a href="#CR9">9</a>, <a href="#CR10">10</a> </sup>, there are now over 800 cp genomes (including 300 from crops and trees) that have been sequenced and deposited in the NCBI Organelle Genome Resources database. The evolution of cp genomes in terrestrial plants may now be studied using these database resources <sup> <a href="#CR11">11</a> </sup>. Chloroplast genome sequences have shown significant variation within and between plant species in terms of both sequence and structural variations <sup> <a href="#CR12">12</a> </sup>. Such information in chloroplast genomes is essential for understanding the climatic adaptation of economically important crops, for the selection of alternatives to breeding closely related species and for the identification and conservation of valuable traits <sup> <a href="#CR8">8</a>, <a href="#CR13">13</a> </sup>. </p> <p>Previous studies have suggested that cp genomes sequences increase phylogenetic resolution at lower taxonomic levels and that they are effective tools for plant phylogenetic and population genetic analyses <sup> <a href="#CR6">6</a>, <a href="#CR8">8</a>, <a href="#CR14">14</a> </sup>. The typical chloroplast genome in angiosperms has a conserved quadripartite structure, with one large single-copy (LSC) region, one small single-copy (SSC) region, and two copies of an inverted repeat (IR) <sup> <a href="#CR15">15</a> </sup>. Despite its conservation of structure, the size of cp genomes fluctuates between species, ranging from 107 kb ( <i>Cathaya argyrophylla</i>) to 218 kb ( <i>Pelargonium hortorum</i>) <sup> <a href="#CR11">11</a>, <a href="#CR16">16</a>, <a href="#CR17">17</a> </sup>. The current rapid development of DNA sequencing technology has resulted in the extensive use of cp genomes as molecular markers in numerous molecular phylogenetic studies. </p> <p> <i>Arabidopsis thaliana</i> is a well-known model organism because of its rapid life cycle, small nuclear genome, amenability to genetic analysis and practical use in breeding biology <sup> <a href="#CR18">18</a>– <a href="#CR20">20</a> </sup>. The steadily increasing knowledge of <i>Arabidopsis</i> has benefited a new era of functional genomics and evolutionary analyses focused on several taxa in <i>Brassicaceae</i> with varying divergence from the model species <sup> <a href="#CR21">21</a>, <a href="#CR22">22</a> </sup>. In general, three major lineages have been recognized, namely, <i>A</i>. <i>arenosa</i>, <i>A</i>. <i>halleri</i> and <i>A</i>. <i>lyrata</i> <sup> <a href="#CR23">23</a> </sup>, and most species or subspecies can be categorized within these three lineages. </p> <p> <i>A</i>. <i>lyrata</i> ssp. <i>petraea</i> (2 <i>n</i> = 16/32) is a perennial herb that diverged from a common ancestor with <i>A</i>. <i>thaliana</i> approximately 10 million years ago <sup> <a href="#CR24">24</a> </sup>. Despite its evolutionary proximity, its nuclear genome size is estimated to be between 230 to 245 Mb, or one and a half times larger than that of the <i>A</i>. <i>thaliana</i> <sup> <a href="#CR25">25</a>, <a href="#CR26">26</a> </sup>. <i>A</i>. <i>halleri</i> ssp. <i>gemmifera</i> (2 <i>n</i> = 16) is a stoloniferous diploid perennial herb with a disjunctive distribution in eastern Asia, including Japan and Taiwan, and its nuclear genome is 40–60% larger than that of <i>A</i>. <i>thaliana</i> <sup> <a href="#CR25">25</a>– <a href="#CR28">28</a> </sup>. In contrast to <i>A</i>. <i>lyrata</i>, the geographical distribution of <i>A</i>. <i>halleri</i> rarely extends into northern latitudes, and the species is more tolerant of competition. <i>A</i>. <i>halleri</i> grows on acidic, neutral and oligotrophic soils, as well as on soils with high heavy metal contents <sup> <a href="#CR28">28</a>, <a href="#CR29">29</a> </sup>. <i>A</i>. <i>halleri</i> can accumulate zinc and cadmium in its aerial parts. On the other hand, <i>A</i>. <i>lyrata</i> is sensitive to metals, suggesting that metal hyperaccumulation in <i>A</i>. <i>halleri</i> is mainly derived from its ecological niche. Moreover, the available data suggest that plants growing in contaminated metallic and non-metallic soils could share such traits <sup> <a href="#CR30">30</a> </sup>. </p> <p>The genus <i>Arabidopsis</i> is frequently affected by the processes of hybridization and introgression <sup> <a href="#CR31">31</a> </sup>, as both <i>A</i>. <i>lyrata</i> and <i>A</i>. <i>halleri</i> possess diploid structures that are reported hybrids from <i>A</i>. <i>kamchatica</i> on the basis of either nuclear genes or self-incompatibility alleles <sup> <a href="#CR31">31</a>– <a href="#CR34">34</a> </sup>. There has been clear evidence of admixture between tetraploid <i>A</i>. <i>lyrata</i> ssp. <i>petraea</i> and <i>A</i>. <i>arenosa</i> ssp. <i>Borbasii</i> <sup> <a href="#CR34">34</a> </sup>, whereas <i>A</i>. <i>halleri</i> ssp. <i>gemmifera</i> is a close relative of <i>Arabidopsis thaliana</i> <sup> <a href="#CR35">35</a> </sup>. Precise phylogenetic placement has proven difficult according to Al-Shehbaz and OKane (2002) <sup> <a href="#CR36">36</a> </sup>, Schmickl <i>et al</i>. <sup> <a href="#CR37">37</a> </sup>, Hohmann <i>et al</i>. <sup> <a href="#CR38">38</a> </sup>, and Novikova <i>et al</i>. <sup> <a href="#CR34">34</a> </sup>, and requires further investigation. <i>A</i>. <i>halleri</i> and <i>A</i>. <i>lyrata</i>, along with their subspecies, on the other hand, are extensively used as outgroups in comparative plant systematic studies <sup> <a href="#CR24">24</a>, <a href="#CR39">39</a>, <a href="#CR40">40</a> </sup>. However, there is little information available on their genetic structure, especially their chloroplast genomes or their detailed phylogenetic placement. The current study presents the cp genomes of <i>A</i>. <i>lyrata</i> ssp. <i>petraea</i> and <i>A</i>. <i>halleri</i> ssp. <i>gemmifera</i> for the first time. In this study, we sequenced the complete chloroplast genomes of <i>A</i>. <i>halleri</i> ssp. <i>gemmifera</i> (GenBank accession number: KU764767) and <i>A</i>. <i>lyrata</i> ssp. <i>petraea</i> (GenBank accession number: KU764768). We aimed to elucidate the global patterns of structural variations in the cp genomes of <i>A</i>. <i>halleri</i> ssp. <i>gemmifera</i> and <i>A</i>. <i>lyrata</i> ssp. <i>petraea</i> and compare them with cp genomes of other <i>Arabidopsis</i> species available from NCBI. </p> </div> <div class="results" title="results"> <div class="title" tagx="title" title="title">
Results</div> <div class="chloroplastgenomefeaturesandthestructureoftwoarabidopsissubspecies" title="sec"> <div class="title" tagx="title" title="title">
Chloroplast genome features and the structure of two <i>Arabidopsis</i> subspecies </div> <p>Two <i>Arabidopsis</i> species, <i>A</i>. <i>halleri</i> ssp. <i>gemmifera</i> and <i>A</i>. <i>lyrata</i> ssp. <i>petraea</i>, were sequenced on an Illumina HiSeq. 2000 to produce 63,528,604 and 67,938,537 bp paired-end raw reads, respectively. The average read length was 101 bp, and the cp genomes of <i>A</i>. <i>halleri</i> ssp. <i>gemmifera</i> and <i>A</i>. <i>lyrata</i> ssp. <i>petraea</i> received 2232.81x and 2173.5x coverage, respectively. The four junction regions for each resulting cp genome were also confirmed by PCR-based Sanger sequencing with four pairs of primers (Table <a href="#MOESM1">S2</a>). The cp genome sizes of <i>A</i>. <i>halleri</i> ssp. <i>gemmifera</i> and <i>A</i>. <i>lyrata</i> ssp. <i>petraea</i> were 154,473 and 154,489 bp, respectively (Table <a href="#Tab1">1</a>, Fig. <a href="#Fig1">1</a>). The genomes displayed a typical quadripartite structure, as shown by most angiosperms. This included a large single-copy region (LSC; 84197 bp and 84158 bp), a small single-copy region (SSC; 17738 bp and 17813 bp) and a pair of inverted repeats (IRa/IRb; 26264 bp and 26259 bp) in <i>A</i>. <i>halleri</i> ssp. <i>gemmifera</i> and <i>A</i>. <i>lyrata</i> ssp. <i>petraea</i>, respectively. The GC contents (36.4%) of both <i>A</i>. <i>halleri</i> ssp. <i>gemmifera</i> and <i>A</i>. <i>lyrata</i> ssp. <i>petraea</i> were very similar to those in the cp genomes of other <i>Arabidopsis</i> species (Table <a href="#Tab1">1</a>). However, the GC contents were unequally distributed in different fragments of the cp genomes, with the highest values in the IR regions (42.3%), median values in the LSC regions (34.1%) and the lowest values in the SSC regions (29.4%) (Table <a href="#Tab1">1</a>). The high GC content of the IR regions might be due to the presence of eight ribosomal RNA (rRNA) sequences in these regions. <div class="table-wrap_UNKNOWN"> <span class="label" tagx="label" title="label">Table 1</span>  </div></p><p>Summary of complete chloroplast genomes for twelve Arabidopsis species.</p>  <table> <thead> <tr> <th>Region</th> <th> <i>A. are</i> </th> <th> <i>A. ceb</i> </th> <th> <i>A. h. gem</i> </th> <th> <i>A. l. pet</i> </th> <th> <i>A. ped</i> </th> <th> <i>A. tha</i> </th> <th> <i>A. aren</i> </th> <th> <i>A. cro</i> </th> <th> <i>A. neg</i> </th> <th> <i>A. pet</i> </th> <th> <i>A. sue</i> </th> <th> <i>A. ume</i> </th> </tr> </thead> <tbody> <tr> <td> <b>LSC</b> </td> </tr> <tr> <td>Length (bp)</td> <td>84464</td> <td>84160</td> <td>84197</td> <td>84158</td> <td>84478</td> <td>84170</td> <td>84234</td> <td>84336</td> <td>84397</td> <td>84478</td> <td>84090</td> <td>84251</td> </tr> <tr> <td>GC (%)</td> <td>34.2</td> <td>34.1</td> <td>34.1</td> <td>34.1</td> <td>34</td> <td>34</td> <td>34.1</td> <td>34.1</td> <td>34.2</td> <td>34.2</td> <td>34</td> <td>34.1</td> </tr> <tr> <td>Length (%)</td> <td>54.53</td> <td>54.47</td> <td>54.50</td> <td>54.47</td> <td>54.53</td> <td>54.48</td> <td>54.4</td> <td>54.5</td> <td>54.5</td> <td>54.5</td> <td>54.4</td> <td>54.4</td> </tr> <tr> <td> <b>SSC</b> </td> </tr> <tr> <td>Length (bp)</td> <td>17885</td> <td>17830</td> <td>17738</td> <td>17813</td> <td>17873</td> <td>17780</td> <td>17859</td> <td>17862</td> <td>17882</td> <td>18875</td> <td>17755</td> <td>17872</td> </tr> <tr> <td>GC (%)</td> <td>29.4</td> <td>29.4</td> <td>29.4</td> <td>29.4</td> <td>29.5</td> <td>29.3</td> <td>29.4</td> <td>29.4</td> <td>29.4</td> <td>29.4</td> <td>29.4</td> <td>29.4</td> </tr> <tr> <td>Length (%)</td> <td>11.54</td> <td>11.54</td> <td>11.48</td> <td>11.53</td> <td>11.53</td> <td>11.50</td> <td>11.5</td> <td>11.54</td> <td>11.55</td> <td>12.1</td> <td>11.5</td> <td>11.55</td> </tr> <tr> <td> <b>IR</b> </td> </tr> <tr> <td>Length (bp)</td> <td>26261</td> <td>26257</td> <td>26264</td> <td>26259</td> <td>26272</td> <td>26264</td> <td>26258</td> <td>26265</td> <td>26260</td> <td>26256</td> <td>26259</td> <td>26261</td> </tr> <tr> <td>GC (%)</td> <td>42.3</td> <td>42.3</td> <td>42.3</td> <td>42.3</td> <td>42.3</td> <td>42.3</td> <td>42.3</td> <td>42.3</td> <td>42.3</td> <td>42.3</td> <td>42.3</td> <td>42.3</td> </tr> <tr> <td>Length (%)</td> <td>16.95</td> <td>16.99</td> <td>17.00</td> <td>16.99</td> <td>16.96</td> <td>17.00</td> <td>16.9</td> <td>16.9</td> <td>16.9</td> <td>16.9</td> <td>17</td> <td>16.9</td> </tr> <tr> <td> <b>Total</b> </td> </tr> <tr> <td>GC (%)</td> <td>36.4</td> <td>36.4</td> <td>36.4</td> <td>36.4</td> <td>36.3</td> <td>36.3</td> <td>36.4</td> <td>36.4</td> <td>36.4</td> <td>36.4</td> <td>36.3</td> <td>36.4</td> </tr> <tr> <td>Length (%)</td> <td>154871</td> <td>154504</td> <td>154473</td> <td>154489</td> <td>154895</td> <td>154478</td> <td>154610</td> <td>154728</td> <td>154799</td> <td>154865</td> <td>154366</td> <td>154645</td> </tr> </tbody> </table> <div class="table-wrap-foot_UNKNOWN"> <p> <b>A. are</b> =  <i>A</i>. <i>arenosa</i>; <b>A. ceb</b> =  <i>A</i>. <i>cebennensis</i>; <b>A. h. gem</b> =  <i>A</i>. <i>halleri ssp</i>. <i>gemmifera</i>; <b>A. l. pet</b> =  <i>A</i>. <i>lyrata</i> ssp. <i>petraea</i>; <b>A. ped</b> =  <i>A</i>. <i>pedemontana</i>; <b>A. tha</b> =  <i>A</i>. <i>thanliana</i>; <b>A. aren</b> =  <i>A</i>. <i>arenicola</i>; <b>A. cro</b> =  <i>A</i>. <i>croatica</i>; <b>A. neg</b> =  <i>A</i>. <i>neglecta</i>; <b>A. pet</b> =  <i>A</i>. <i>petrogenea</i>; <b>A. sue</b> =  <i>A</i>. <i>suecia</i>; <b>A. ume</b> =  <i>A</i>. <i>umezawana</i>. </p> </div>  <div class="fig" title="fig"> <div class="figure" title="figure"> <a href="http://doi.org/"> <span class="label" tagx="label" title="label">Figure 1</span> </a> </div>   <p>Gene map of the <i>A</i>. <i>halleri</i> ssp. <i>gemmifera</i> and <i>A</i>. <i>lyrata</i> ssp. <i>petraea</i> chloroplast genomes. Genes drawn inside the circle are transcribed clockwise, and those outside the circle are transcribed counter clockwise. The asterisks indicate intron-containing genes. Genes belonging to different functional groups are colour-coded. The darker grey in the inner circle corresponds to GC content, and the lighter grey corresponds to AT content. </p>   </div> <p /> <p>Both cp genomes were composed of 130 genes, including 86 protein-coding genes, eight ribosomal RNA (rRNA) genes and 37 transfer RNA (tRNA) genes (Figs <a href="#Fig1">1</a>, <a href="#Fig2">2</a>; Table <a href="#Tab2">2</a>). Five protein-coding genes ( <i>rpl2</i>, <i>rpl23</i>, <i>ycf2</i>, <i>ndhB</i> and <i>rps7</i>) are duplicated and the inverted repeats contain truncated copies of <i>rps19</i>, <i>ycf1</i>, <i>ndhF</i> and duplicated exons of <i>rps12</i> (exon 2 and exon 3). The LSC region was composed of 62 protein-coding and 22 tRNA genes, whereas the SSC region was composed of 12 protein-coding genes and 1 tRNA gene. The protein-coding genes present in the <i>A</i>. <i>halleri</i> ssp. <i>gemmifera</i> and <i>A</i>. <i>lyrata</i> ssp. <i>petraea</i> genomes included nine genes for large ribosomal proteins ( <i>rpl2</i>, 14, 16, 20, 22, 23, 32, 33, 36), 12 genes for small ribosomal proteins ( <i>rps2</i>, 3, 4, 7, 8, 11, 12, 14, 15, 16, 18, 19), five genes for photosystem I ( <i>psaA</i>, <i>B</i>, <i>C</i>, <i>I</i>, <i>J</i>), 15 genes related to photosystem II ( <i>psbA</i>, <i>B</i>, <i>C</i>, <i>D</i>, <i>E</i>, <i>F</i>, <i>H</i>, <i>I</i>, <i>J</i>, <i>K</i>, <i>L</i>, <i>M</i>, <i>N</i>, <i>T</i>, <i>Z</i>), and six genes ( <i>atpA</i>, <i>B</i>, <i>E</i>, <i>F</i>, <i>H</i>, <i>I</i>) for ATP synthase and the electron transport chain (Fig. <a href="#Fig2">2</a>, Table <a href="#Tab1">1</a>). </p><div class="fig" title="fig"> <div class="figure" title="figure"> <a href="http://doi.org/"> <span class="label" tagx="label" title="label">Figure 2</span> </a> </div>   <p>Gene contents of the <i>A</i>. <i>halleri</i> ssp. <i>gemmifera</i> and <i>A</i>. <i>lyrata</i> ssp. <i>petraea</i> chloroplast genomes, grouped by gene family. The colour of each gene is unique within its gene family. Horizontal axis indicates each box is proportional to the size of the gene (bp), including introns. </p>   </div> <div class="table-wrap_UNKNOWN"> <span class="label" tagx="label" title="label">Table 2</span>  <p>List of genes in the <i>A</i>. <i>halleri</i> ssp. <i>gemmifera</i> and <i>A</i>. <i>lyrata</i> ssp. <i>petraea</i> chloroplast genomes. </p>  <table> <thead> <tr> <th>Category</th> <th>Group of genes</th> <th>Name of genes</th> </tr> </thead> <tbody> <tr> <td>Self-replication</td> <td>Large subunit of ribosomal proteins</td> <td> <i>rpl2</i> <sup> <i>*</i>,a </sup> <i>, 14, 16</i> <sup> <i>*</i> </sup> <i>, 20, 22, 23</i> <sup>a</sup> <i>, 32, 33, 36</i> </td> </tr> <tr> <td>Small subunit of ribosomal proteins</td> <td> <i>rps2</i>, <i>3</i>, <i>4</i>, <i>7</i> <sup>a</sup>, <i>8</i>, <i>11</i>, <i>12</i> <sup> <i>*</i>,a </sup>, <i>14</i>, <i>15</i>, <i>16</i> <sup> <i>*</i> </sup>, <i>18</i>, <i>19</i> </td> </tr> <tr> <td>DNA dependent RNA polymerase</td> <td> <i>rpoA</i>, <i>B</i>, <i>C1</i> <sup> <i>*</i> </sup>, <i>C2</i> </td> </tr> <tr> <td>rRNA genes</td> <td> <i>rrn16</i> <sup>a</sup>, <i>rrn23</i> <sup>a</sup>, <i>rrn4</i>. <i>5</i> <sup>a</sup>, <i>rrn5</i> <sup>a</sup> </td> </tr> <tr> <td>tRNA genes</td> <td> <i>trnA-UGC</i> <sup> <i>*</i>,a </sup>, <i>trnC-GCA</i>, <i>trnD-GUC</i>, <i>trnE-UUC trnF-GAA</i>, <i>trnfM-CAU</i>, <i>trnG-UCC*</i>, <i>trnH-GUG</i>, <i>trnI-CAU</i>, <i>trnI-GAU</i> <sup> <i>*</i>,a </sup>, <i>trnK-UUU*</i>, <i>trnL-CAA</i>, <i>trnL-UAA*</i>, <i>trnL-UAG</i>, <i>trnM-CAU</i>, <i>trnN-GUU</i>, <i>trnP-GGG</i>, <i>trnP-UGG</i>, <i>trnQ-UUG</i>, <i>trnR-ACG</i>, <i>trnR-UCU</i>, <i>trnS-GCU</i>, <i>trnS-GGA</i>, <i>trnS-UGA</i>, <i>trnT-GGU</i>, <i>trnT-UGU</i>, <i>trnV-GAC</i>, <i>trnV-UAC*</i>, <i>trnW-CCA</i>, <i>trnY-GUA</i> </td> </tr> <tr> <td>Photosynthesis</td> <td>Photosystem I</td> <td> <i>psaA</i>, <i>B</i>, <i>C</i>, <i>I</i>, <i>J</i> </td> </tr> <tr> <td>Photosystem II</td> <td> <i>psbA</i>, <i>B</i>, <i>C</i>, <i>D</i>, <i>E</i>, <i>F</i>, <i>H</i>, <i>I</i>, <i>J</i>, <i>K</i>, <i>L</i>, <i>M</i>, <i>N</i>, <i>T</i>, <i>lhbA</i> </td> </tr> <tr> <td>NadH oxidoreductase</td> <td> <i>ndhA</i> <sup> <i>*</i> </sup>, <i>B</i> <sup> <i>*</i>,a </sup>, <i>C</i>, <i>D</i>, <i>E</i>, <i>F</i>, <i>G</i>, <i>H</i>, <i>I</i>, <i>J</i>, <i>K</i> </td> </tr> <tr> <td>Cytochrome b6/f complex</td> <td> <i>petA</i>, <i>B</i> <sup> <i>*</i> </sup>, <i>D</i> <sup> <i>*</i> </sup>, <i>G</i>, <i>L</i>, <i>N</i> </td> </tr> <tr> <td>ATP synthase</td> <td> <i>atpA</i>, <i>B</i>, <i>E</i>, <i>F</i> <sup> <i>*</i> </sup>, <i>H</i>, <i>I</i> </td> </tr> <tr> <td>Rubisco</td> <td> <i>rbcL</i> </td> </tr> <tr> <td>Other genes</td> <td>Maturase</td> <td> <i>matK</i> </td> </tr> <tr> <td>Protease</td> <td> <i>clpP</i> <sup> <i>*</i> </sup> </td> </tr> <tr> <td>Envelop membrane protein</td> <td> <i>cemA</i> </td> </tr> <tr> <td>Subunit Acetyl- CoA-Carboxylate</td> <td> <i>accD</i> </td> </tr> <tr> <td>c-type cytochrome synthesis gene</td> <td> <i>ccsA</i> </td> </tr> <tr> <td>Conserved Open reading frames</td> <td> <i>ycf1</i>, <i>2</i> <sup>a</sup>, <i>3</i> <sup> <i>*</i> </sup>, <i>4</i> </td> </tr> </tbody> </table> <div class="table-wrap-foot_UNKNOWN"> <p>*Genes containing introns; <sup>a</sup>Duplicated gene (Genes present in the IR regions). </p> </div> </div> <p /> <p>Proteins are encoded by 51.80% ( <i>A</i>. <i>halleri</i>) and 51.79% ( <i>A</i>. <i>lyrata</i>) of the cp genomes, while rRNAs and tRNAs comprise 5.85% and 1.79%, respectively, of both genomes, with the remaining 40.5% made up of non-coding regions (Table <a href="#Tab3">3</a>). The total protein-coding sequences (CDS) were 80,019 and 80,013 bp in length in these two cp genomes, respectively, and composed of 86 protein-coding genes that code for 26,907 and 26,905 codons, respectively (Tables <a href="#Tab3">3</a> and <a href="#Tab4">4</a>). The codon usage frequency was calculated for tRNA and protein-coding gene sequences in the <i>A</i>. <i>halleri</i> ssp. <i>gemmifera</i> (Table <a href="#MOESM1">S3</a>) and <i>A</i>. <i>lyrata</i> ssp. <i>petraea</i> (Table <a href="#MOESM1">S4</a>) cp genomes. In these cp genomes, leucine (10.7%) was the most common amino acid, while cysteine (1.2%) was least common. Furthermore, isoleucine, serine, glycine, arginine and alanine were present at 7.9, 7.5, 7.4, 6.5, and 6.1%, respectively. <div class="table-wrap_UNKNOWN"> <span class="label" tagx="label" title="label">Table 3</span>  </div></p><p>Comparison of coding and non-coding region size among twelve <i>Arabidopsis</i> species. </p>  <table> <thead> <tr> <th>Region</th> <th> <i>A. are</i> </th> <th> <i>A. ceb</i> </th> <th> <i>A.h. gem</i> </th> <th> <i>A. l. pet</i> </th> <th> <i>A. ped</i> </th> <th> <i>A. tha</i> </th> <th> <i>A. aren</i> </th> <th> <i>A. cro</i> </th> <th> <i>A. neg</i> </th> <th> <i>A. pet</i> </th> <th> <i>A. sue</i> </th> <th> <i>A. ume</i> </th> </tr> </thead> <tbody> <tr> <td> <b>Protein Coding</b> </td> </tr> <tr> <td>Length (bp)</td> <td>78561</td> <td>78564</td> <td>80019</td> <td>80013</td> <td>78540</td> <td>79368</td> <td>78648</td> <td>78672</td> <td>78675</td> <td>78675</td> <td>78666</td> <td>78699</td> </tr> <tr> <td>GC (%)</td> <td>37.1</td> <td>37.1</td> <td>37</td> <td>37</td> <td>37.1</td> <td>37</td> <td>37.1</td> <td>37</td> <td>37.1</td> <td>37.1</td> <td>37</td> <td>37.1</td> </tr> <tr> <td>Length (%)</td> <td>50.7</td> <td>50.8</td> <td>51.80</td> <td>51.79</td> <td>50.70</td> <td>51.3</td> <td>50.8</td> <td>50.8</td> <td>50.8</td> <td>50.8</td> <td>50.9</td> <td>50.8</td> </tr> <tr> <td> <b>tRNA</b> </td> </tr> <tr> <td>Length (bp)</td> <td>2790</td> <td>2790</td> <td>2775</td> <td>2775</td> <td>2796</td> <td>3325</td> <td>2790</td> <td>2790</td> <td>2790</td> <td>2791</td> <td>2789</td> <td>2791</td> </tr> <tr> <td>GC (%)</td> <td>52.6</td> <td>52.6</td> <td>52.2</td> <td>52.3</td> <td>52.5</td> <td>49.2</td> <td>52.6</td> <td>52.6</td> <td>52.6</td> <td>52.6</td> <td>52.5</td> <td>52.6</td> </tr> <tr> <td>Length (%)</td> <td>1.80</td> <td>1.80</td> <td>1.79</td> <td>1.79</td> <td>1.80</td> <td>2.15</td> <td>1.80</td> <td>1.80</td> <td>1.80</td> <td>1.80</td> <td>1.80</td> <td>1.80</td> </tr> <tr> <td> <b>rRNA</b> </td> </tr> <tr> <td>Length (bp)</td> <td>9050</td> <td>9050</td> <td>9050</td> <td>9050</td> <td>9050</td> <td>8929</td> <td>9050</td> <td>9050</td> <td>9050</td> <td>9050</td> <td>9050</td> <td>9050</td> </tr> <tr> <td>GC (%)</td> <td>55.4</td> <td>55.4</td> <td>55.4</td> <td>55.4</td> <td>55.4</td> <td>55.4</td> <td>55.4</td> <td>55.4</td> <td>55.4</td> <td>55.4</td> <td>55.4</td> <td>55.4</td> </tr> <tr> <td>Length (%)</td> <td>5.84</td> <td>5.85</td> <td>5.85</td> <td>5.85</td> <td>5.84</td> <td>5.78</td> <td>5.8</td> <td>5.84</td> <td>5.84</td> <td>5.84</td> <td>5.84</td> <td>5.84</td> </tr> <tr> <td>Intergenic</td> <td>64470</td> <td>64100</td> <td>62629</td> <td>62629</td> <td>64509</td> <td>62256</td> <td>64302</td> <td>64216</td> <td>64284</td> <td>64347</td> <td>63861</td> <td>64205</td> </tr> <tr> <td>GC (%)</td> <td>31.6</td> <td>31.6</td> <td>31.8</td> <td>31.7</td> <td>31.7</td> <td>31</td> <td>31.3</td> <td>31.1</td> <td>31.5</td> <td>31.3</td> <td>31.2</td> <td>31.6</td> </tr> <tr> <td>Length (%)</td> <td>41.6</td> <td>41.48</td> <td>40.56</td> <td>40.53</td> <td>41.64</td> <td>40.3</td> <td>41.5</td> <td>41.5</td> <td>41.52</td> <td>41.55</td> <td>41.36</td> <td>41.51</td> </tr> </tbody> </table> <div class="table-wrap-foot_UNKNOWN"> <p> <b>A. are</b> =  <i>A</i>. <i>arenosa;</i> <b>A. ceb</b> =  <i>A</i>. <i>cebennensis;</i> <b>A. h. gem</b> =  <i>A</i>. <i>halleri ssp</i>. <i>gemmifera;</i> <b>A. l. pet</b> =  <i>A</i>. <i>lyrata</i> ssp. <i>petraea;</i> <b>A. ped</b> =  <i>A</i>. <i>pedemontana</i>; <b>A. tha</b> =  <i>A</i>. <i>thanliana;</i> <b>A. aren</b> =  <i>A</i>. <i>arenicola;</i> <b>A. cro</b> =  <i>A</i>. <i>croatica;</i> <b>A. neg</b> =  <i>A</i>. <i>neglecta;</i> <b>A. pet</b> =  <i>A</i>. <i>petrogenea;</i> <b>A. sue</b> =  <i>A</i>. <i>suecia;</i> <b>A. ume</b> =  <i>A</i>. <i>umezawana</i>. </p> </div>  <div class="table-wrap_UNKNOWN"> <span class="label" tagx="label" title="label">Table 4</span>  <p>Base compositions in the <i>A</i>. <i>halleri</i> ssp. <i>gemmifera</i> (Ahg) and <i>A</i>. <i>lyrata</i> ssp. <i>petraea</i> (Alp) cp genome. </p>  <table> <thead> <tr> <th /> <th>T/U</th> <th>C</th> <th>A</th> <th>G</th> <th>Length (bp)</th> </tr> <tr> <th> <i>A</i>. <i>h</i>. <i>gem</i> </th> <th> <i>A</i>. <i>l</i>. <i>pet</i> </th> <th> <i>A</i>. <i>h</i>. <i>gem</i> </th> <th> <i>A</i>. <i>l</i>. <i>pet</i> </th> <th> <i>A</i>. <i>h</i>. <i>gem</i> </th> <th> <i>A</i>. <i>l</i>. <i>pet</i> </th> <th> <i>A</i>. <i>h</i>. <i>gem</i> </th> <th> <i>A</i>. <i>l</i>. <i>pet</i> </th> <th> <i>A</i>. <i>h</i>. <i>gem</i> </th> <th> <i>A</i>. <i>l</i>. <i>pet</i> </th> </tr> </thead> <tbody> <tr> <td>Genome</td> <td>32.3</td> <td>32.3</td> <td>18.5</td> <td>18.5</td> <td>31.4</td> <td>31.4</td> <td>17.9</td> <td>17.9</td> <td>154473</td> <td>154489</td> </tr> <tr> <td>LSC</td> <td>33.8</td> <td>33.8</td> <td>17.5</td> <td>17.5</td> <td>32.1</td> <td>32.1</td> <td>16.6</td> <td>16.6</td> <td>84197</td> <td>84158</td> </tr> <tr> <td>SSC</td> <td>35.2</td> <td>35.2</td> <td>15.2</td> <td>15.2</td> <td>35.4</td> <td>35.4</td> <td>14.2</td> <td>14.2</td> <td>17739</td> <td>17814</td> </tr> <tr> <td>IR</td> <td>28.8</td> <td>28.8</td> <td>22</td> <td>22.0</td> <td>28.9</td> <td>29.0</td> <td>20.3</td> <td>20.3</td> <td>26270</td> <td>26259</td> </tr> <tr> <td>tRNA</td> <td>23.2</td> <td>23.2</td> <td>26.3</td> <td>26.3</td> <td>24.5</td> <td>24.5</td> <td>25.9</td> <td>25.9</td> <td>2775</td> <td>2775</td> </tr> <tr> <td>rRNA</td> <td>22.3</td> <td>22.3</td> <td>27.7</td> <td>27.7</td> <td>22.3</td> <td>22.3</td> <td>27.7</td> <td>27.7</td> <td>9050</td> <td>9050</td> </tr> <tr> <td>Protein Coding genes</td> <td>31.5</td> <td>31.9</td> <td>17.3</td> <td>17.3</td> <td>31</td> <td>31.1</td> <td>19.7</td> <td>19.7</td> <td>80019</td> <td>80013</td> </tr> <tr> <td>1st position</td> <td>24.24</td> <td>24.26</td> <td>16.8</td> <td>18.6</td> <td>30.34</td> <td>30.3</td> <td>26.75</td> <td>28.4</td> <td>26907</td> <td>26905</td> </tr> <tr> <td>2nd position</td> <td>33.05</td> <td>33.03</td> <td>20.2</td> <td>20.2</td> <td>28.83</td> <td>28.8</td> <td>17.84</td> <td>17.8</td> <td>26907</td> <td>26905</td> </tr> <tr> <td>3rd position</td> <td>38.6</td> <td>36.6</td> <td>13.3</td> <td>14.03</td> <td>32.13</td> <td>32.12</td> <td>15.54</td> <td>15.5</td> <td>26907</td> <td>26905</td> </tr> </tbody> </table> <div class="table-wrap-foot_UNKNOWN"> <p> <b>A. h. gem</b> =  <i>A</i>. <i>halleri ssp</i>. <i>gemmifera;</i> <b>A. l. pet</b> =  <i>A</i>. <i>lyrata</i> ssp. <i>petraea</i>. </p> </div> </div> <p /> <p>Among these, the most common codon used was ATT (1,077), encoding isoleucine, and the least common codons were CTG (1) in <i>A</i>. <i>lyrata</i> ssp. <i>petraea</i> and ATT (1) in <i>A</i>. <i>halleri</i> ssp. <i>gemmifera</i>, which encodes methionine. The AT content was 54.58%, 61.88%, and 70.73% at the 1st, 2nd, and 3rd codon positions within the <i>A</i>. <i>halleri</i> ssp. <i>gemmifera</i> CDS region (Table <a href="#Tab4">4</a>). In <i>A</i>. <i>lyrata</i> ssp. <i>petraea</i>, the AT content was 54.56%, 61.03%, and 68.72% at the 1st, 2nd, and 3rd codon positions within the CDS region (Table <a href="#Tab4">4</a>). </p> </div> <div class="repeatanalysisandcomparisonofitsdistributioninarabidopsissubspecies" title="sec"> <div class="title" tagx="title" title="title">
Repeat Analysis and comparison of its distribution in <i>Arabidopsis</i> subspecies </div> <p>A total of 71 and 75 repeats were detected in the <i>A</i>. <i>halleri</i> ssp. <i>gemmifera</i> and <i>A</i>. <i>lyrata</i> ssp. <i>petraea</i> genomes, respectively, using REPuter, including direct, reversed and palindromic repeats (Tables <a href="#MOESM1">S5</a>, <a href="#MOESM1">S6</a>). In these cp genomes, the repeat analysis detected 11 and 12 palindromic repeats, 26 and 31 forward repeats, and 34 and 32 tandem repeats in <i>A</i>. <i>halleri</i> ssp. <i>gemmifera</i> and <i>A</i>. <i>lyrata</i> ssp. <i>petraea</i>, respectively (Fig. <a href="#Fig3">3</a>). Among these, 22 forward repeats were 30–44 bp in length, 5 tandem repeats were of the same length, and 27 were 15–29 bp in length (Fig. <a href="#Fig3">3A–D</a>). Similarly, in <i>A</i>. <i>halleri</i> ssp. <i>gemmifera</i>, 8 palindromic repeats were 30–44 bp, and 2 repeats were 45–59 bp in length (Fig. <a href="#Fig3">3D</a>). However, in <i>A</i>. <i>lyrata</i> ssp. <i>petraea</i>, 25 forward repeats and nine palindromic repeats were 30–44 bp in length, and 27 tandem repeats were 15–29 bp in length (Fig. <a href="#Fig3">3B–D</a>). </p><div class="fig" title="fig"> <div class="figure" title="figure"> <a href="http://doi.org/"> <span class="label" tagx="label" title="label">Figure 3</span> </a> </div>   <p>Analysis of repeated sequences in twelve <i>Arabidopsis</i> cp genomes. ( <b>A</b>) Totals of three repeat types; ( <b>B</b>) Frequency of forward repeats by length; ( <b>C</b>) Frequency of palindromic repeats by length; ( <b>D</b>) Frequency of tandem repeats by length. </p>   </div> <p /> <p>Similarly, 63, 77, 61, 61, 74, 75, 82, 81, 76 and 72 repeat pairs were found in previously reported <i>A</i>. <i>thaliana</i>, <i>A</i>. <i>arenosa</i>, <i>A</i>. <i>cebennensis</i>, <i>A</i>. <i>pedemontana</i>, <i>A</i>. <i>arenicola</i>, <i>A</i>. <i>croatica</i>, <i>A</i>. <i>neglecta</i>, <i>A</i>. <i>petrogena</i>, <i>A</i>. <i>suecica</i> and <i>A</i>. <i>umezawana</i> genomes, respectively (Fig. <a href="#Fig3">3A</a>). This suggests that <i>A</i>. <i>halleri</i> and <i>A</i>. <i>lyrata</i> are more similar to <i>A</i>. <i>arenosa</i>, <i>A</i>. <i>croatica</i>, <i>A</i>. <i>suecica</i> and <i>A</i>. <i>umezawana</i> with respect to repeats. The lengths of direct and palindromic repeats in the <i>A</i>. <i>halleri</i> ssp. <i>gemmifera</i> and <i>A</i>. <i>lyrata</i> ssp. <i>petraea</i> cp genomes were much shorter, ranging from 30 to 101 bp (Tables <a href="#MOESM1">S5</a>, <a href="#MOESM1">S6</a>). In <i>A</i>. <i>halleri</i> ssp. <i>gemmifera</i>, a minority of repeats were found in introns (7.89%), while the majority were located in IGS (55.2%) and coding sequence (CDS) regions (36.8%), while in <i>A</i>. <i>lyrata</i> ssp. <i>petraea</i> these values were 33.3%, 11.90% and 52.3%, respectively (Tables <a href="#MOESM1">S5</a> and <a href="#MOESM1">S6</a>). </p> </div> <div class="simplesequencerepeat(ssr)analysisandcomparison" title="sec"> <div class="title" tagx="title" title="title">
Simple Sequence Repeat (SSR) Analysis and comparison</div> <p>In this study, we detected perfect SSRs in the cp genomes of <i>A</i>. <i>halleri</i> ssp. <i>gemmifera</i> and <i>A</i>. <i>lyrata</i> ssp. <i>petraea</i> cp together with ten other <i>Arabidopsis</i> species (Fig. <a href="#Fig4">4A–D</a>). Certain parameters were set because SSRs of 10 bp or longer are prone to slipped strand mis-pairing, which is believed to be the main mutational mechanism of SSR polymorphisms. A total of 227 and 229 microsatellites were found in the <i>A</i>. <i>halleri</i> ssp. <i>gemmifera</i> and <i>A</i>. <i>lyrata</i> ssp. <i>petraea</i> cp genomes based on SSR analysis, respectively (Fig. <a href="#Fig4">4A</a>). Among these, 78 and 76 were found in coding regions, while 144 and 148 microsatellites were found in intergenic regions (Fig. <a href="#Fig4">4A</a>). Similarly, 226, 215, 214, 221, 214, 216, 213, 216, 220, and 216 SSRs were detected in <i>A</i>. <i>thaliana</i>, <i>A</i>. <i>arenosa</i>, <i>A</i>. <i>cebennensis</i>, <i>A</i>. <i>pedemontana</i>, <i>A</i>. <i>arenicola</i>, <i>A</i>. <i>croatica</i>, <i>A</i>. <i>neglecta</i>, <i>A</i>. <i>petrogena</i>, <i>A</i>. <i>suecica</i> and <i>A</i>. <i>umezawana</i>, respectively (Fig. <a href="#Fig4">4A</a>). The majority of the SSRs in these cp genomes consist of mono- and dinucleotide repeat motifs, varying from 65 in <i>A</i>. <i>suecica</i> to 78 in <i>A</i>. <i>halleri</i> ssp. <i>gemmifera</i> for mononucleotide repeats, while dinucleotide repeats varied from 70 in <i>A</i>. <i>cebennensis</i> and <i>A</i>. <i>croatica</i> to 83 in <i>A</i>. <i>thaliana</i> (Fig. <a href="#Fig4">4A</a>). Trinucleotide SSRs are the second most common, ranging from 57 in <i>A</i>. <i>arenosa</i> and <i>A</i>. <i>neglecta</i> to 64 in <i>A</i>. <i>halleri</i> ssp. <i>gemmifera</i>. Furthermore, two pentanucleotide SSRs are present in <i>A</i>. <i>thaliana</i>, <i>A</i>. <i>arenosa</i>, <i>A</i>. <i>pedemontana</i>, <i>A</i>. <i>croatica</i>, and <i>A</i>. <i>petrogena</i>, with one present in <i>A</i>. <i>halleri</i> ssp. <i>gemmifera</i>, <i>A</i>. <i>arenicola</i>, <i>A</i>. <i>neglecta</i>, and <i>A</i>. <i>umezawana</i>, and four in <i>A</i>. <i>cebennensis</i>. Additionally, four hexanucleotide repeats were found in <i>A</i>. <i>lyrata</i> ssp. <i>petraea</i>, one in <i>A</i>. <i>thaliana</i>, <i>A</i>. <i>arenosa</i>, <i>A</i>. <i>arenicola</i>, <i>A</i>. <i>neglecta</i>, <i>A</i>. <i>suecica</i>, and two in <i>A</i>. <i>halleri</i> ssp. <i>gemmifera</i> using our search criterion (Fig. <a href="#Fig4">4A</a>, Table <a href="#MOESM1">S7</a>). In <i>A</i>. <i>halleri</i> ssp. <i>gemmifera</i> and <i>A</i>. <i>lyrata</i> ssp. <i>petraea</i>, most mononucleotide SSRs were A (98.7%, 97.4%) motifs, with the majority of dinucleotide SSRs being A/T (71.05%, 69.44%) and A/G (27.77%, 26.31%) motifs (Fig. <a href="#Fig4">4B</a>, Tables <a href="#MOESM1">S7</a> and <a href="#MOESM1">S8</a>). </p><div class="fig" title="fig"> <div class="figure" title="figure"> <a href="http://doi.org/"> <span class="label" tagx="label" title="label">Figure 4</span> </a> </div>   <p>Analysis of simple sequence repeats (SSR) in the twelve <i>Arabidopsis</i> cp genomes. ( <b>A</b>) Number of different SSR types detected in the six genomes; ( <b>B</b>) Frequency of identified SSR motifs in different repeat class types; ( <b>C</b>) Frequency of identified SSRs in coding regions; ( <b>D</b>) Frequency of identified intergenic regions. </p>   </div> <p /> </div> <div class="structuralcomparativeassessmentofcpgenomesinarabidopsis" title="sec"> <div class="title" tagx="title" title="title">
Structural comparative assessment of cp genomes in <i>Arabidopsis</i> </div> <p>Ten complete cp genomes within the genus <i>Arabidopsis</i> ( <i>A</i>. <i>thaliana</i>, <i>A</i>. <i>arenosa</i>, <i>A</i>. <i>cebennensis</i>, <i>A</i>. <i>pedemontana</i>, <i>A</i>. <i>arenicola</i>, <i>A</i>. <i>croatica</i>, <i>A</i>. <i>neglecta</i>, <i>A</i>. <i>petrogena</i>, <i>A</i>. <i>suecica</i> and <i>A</i>. <i>umezawana</i>) were selected for comparison with <i>A</i>. <i>halleri</i> ssp. <i>gemmifera</i> (154,473 bp) and <i>A</i>. <i>lyrata</i> ssp. <i>petraea</i> (154,489 bp). The genome size of <i>A</i>. <i>pedemontana</i> (154,895 bp) is the largest of these, and this difference was mostly attributed to variation in the length of the LSC region (Table <a href="#Tab1">1</a>), as reported previously in angiosperms cp genomes. Analysis of genes with known functions showed that <i>A</i>. <i>halleri</i> ssp. <i>gemmifera</i> and <i>A</i>. <i>lyrata</i> ssp. <i>petraea</i> shared 70 protein-coding genes with the other ten <i>Arabidopsis</i> species cp genomes. The number of unique genes found in these cp genomes was 79 (Fig. <a href="#Fig5">5</a>). </p><div class="fig" title="fig"> <div class="figure" title="figure"> <a href="http://doi.org/"> <span class="label" tagx="label" title="label">Figure 5</span> </a> </div>   <p>Alignment visualization of the twelve <i>Arabidopsis</i> chloroplast genome sequences. VISTA-based identity plot showing sequence identity among the six-species using <i>A</i>. <i>halleri</i> ssp. <i>gemmifera</i> as a reference. Vertical scale indicates the percentage of identity, ranging from 50% to 100%. Horizontal axis indicates the coordinates within the chloroplast genome. Arrows indicate the annotated genes and their transcriptional direction. The thick black lines show the inverted repeats (IRs) in the chloroplast genomes. </p>   </div> <p /> <p>Pairwise cp genomic alignment of <i>A</i>. <i>halleri</i> ssp. <i>gemmifera</i> and <i>A</i>. <i>lyrata</i> ssp. <i>petraea</i> with ten other <i>Arabidopsis</i> cp genomes uncovered a high degree of synteny. Annotations from cp genomes of <i>A</i>. <i>halleri</i> ssp. <i>gemmifera</i> and <i>A</i>. <i>lyrata</i> ssp. <i>petraea</i> were used as a reference to plot. The sequences of eleven <i>Arabidopsis</i> species cp genomes were compared using mVISTA (Fig. <a href="#Fig5">5</a> and Fig. <a href="#MOESM1">S1</a>). Furthermore, we compared the <i>Arabidopsis</i> cp genomes and calculated the average pairwise sequence divergence among these twelve species (Table <a href="#MOESM2">S9</a>). Of these genomes, <i>A</i>. <i>halleri</i> ssp. <i>gemmifera</i> and <i>A</i>. <i>lyrata</i> ssp. <i>petraea</i> exhibited 0.0039 and 0.0045 average sequence divergence, respectively, and both species were highly divergent (0.0123 and 0.0128, respectively) with <i>A</i>. <i>thaliana</i> (Table <a href="#MOESM2">S9</a>). For both <i>A</i>. <i>halleri</i> ssp. <i>gemmifera</i> and <i>A</i>. <i>lyrata</i> ssp. <i>petraea</i>, the lowest average sequence divergences were found with <i>A</i>. <i>umezawana</i> (0.0005) and <i>A</i>. <i>arenicola</i> (0.0019). Additionally, the ten most divergent genes among these <i>Arabidopsis</i> cp genomes were <i>rps8</i>, <i>psbK</i>, <i>petD</i>, <i>psbM</i>, <i>ndhD</i>, <i>E</i>, <i>accD</i>, <i>rps12</i>, <i>rpl33</i> and <i>atpH</i> (Figs <a href="#MOESM1">S2</a> and <a href="#MOESM1">S3</a>). The highest average sequence distance of <i>A</i>. <i>halleri</i> ssp. <i>gemmifera</i> from the other cp genomes was observed in <i>psbM</i> (0.0292), followed by <i>rpl22</i> (0.0210), with <i>A</i>. <i>thaliana</i> and <i>A</i>. <i>suecica</i>, respectively (Fig. <a href="#MOESM1">S2</a>). In <i>A</i>. <i>lyrata</i> ssp. <i>petraea</i> the highest average sequence distances were observed for <i>psbK</i> (0.021), rpl22 (0.021) and <i>psbM</i> (0.0194) (Fig. <a href="#MOESM1">S3</a>). </p> </div> <div class="invertedrepeat(ir)contractionandexpansionacrosstwosubspecies" title="sec"> <div class="title" tagx="title" title="title">
Inverted repeat (IR) contraction and expansion across two subspecies</div> <p>A detailed comparison was performed of four junctions (J <sub>LA</sub>, J <sub>LB</sub>, J <sub>SA</sub> and J <sub>SB</sub>) between the two IRs (IRa and IRb) and the two single-copy regions (LSC and SSC) among <i>A</i>. <i>thaliana</i>, <i>A</i>. <i>arenosa</i>, <i>A</i>. <i>cebennensis</i>, <i>A</i>. <i>pedemontana</i>, <i>A</i>. <i>arenicola</i>, <i>A</i>. <i>croatica</i>, <i>A</i>. <i>neglecta</i>, <i>A</i>. <i>petrogena</i>, <i>A</i>. <i>suecica</i> and <i>A</i>. <i>umezawana</i> in comparison to <i>A</i>. <i>halleri</i> ssp. <i>gemmifera</i> and <i>A</i>. <i>lyrata</i> ssp. <i>petraea</i> (Fig. <a href="#Fig6">6</a>). We carefully analysed and compared the exact IR border positions and the adjacent genes among the <i>Arabidopsis</i> species cp genomes (Fig. <a href="#Fig6">6</a>). In this study, despite the similar length of the IR regions in <i>A</i>. <i>halleri</i> ssp. <i>gemmifera</i> and <i>A</i>. <i>lyrata</i> ssp. <i>petraea</i> with those in the other ten <i>Arabidopsis</i> species (from 26,256 bp in <i>A</i>. <i>petrogena</i> to 26,272 bp in <i>A</i>. <i>pedemontana</i>), some IR contraction and expansion was observed. The LSC/IRb junction was located in the <i>rps19</i> region in all <i>Arabidopsis</i> species plastid genomes, which extended 113 bp into the IRb region in all genomes. The IRa ends up with the truncated copy of 113 bp of <i>rps19</i> gene in all twelve species. Similarly, the <i>trnH</i> gene was located in LSC region, 3 bp away from the IRa/LSC border across compared genomes. The <i>ndhF</i> gene was located in the J <sub>SB</sub> border and extended 35 and 36 bp in <i>A</i>. <i>lyrata</i> ssp. <i>petraea</i> and <i>A</i>. <i>halleri</i> ssp. <i>gemmifera</i>, respectively, and 37 bp in the other <i>Arabidopsis</i> species in the IRb region. The pseudogene <i>ycf</i> extended 3 bp into the SSC region at the border of J <sub>SB</sub> for <i>A</i>. <i>halleri</i> ssp. <i>gemmifera</i>, and 2 bp in <i>A</i>. <i>thaliana</i>, <i>A</i>. <i>arenosa</i>, <i>A</i>. <i>cebennensis</i>, <i>A</i>. <i>pedemontana</i>, <i>A</i>. <i>arenicola</i>, <i>A</i>. <i>croatica</i>, <i>A</i>. <i>neglecta</i>, <i>A</i>. <i>petrogena</i>, <i>A</i>. <i>suecica</i> and <i>A</i>. <i>umezawana</i>. A similar gene composition was found in the J <sub>SA</sub> border. In <i>A</i>. <i>lyrata</i> ssp. <i>petraea</i>, the <i>ycf1</i> gene was located 1030 and 1028 bp in the IR regions, and in other <i>Arabidopsis</i> species, this distance was 1030 bp away from the J <sub>SA</sub> border in IR. </p><div class="fig" title="fig"> <div class="figure" title="figure"> <a href="http://doi.org/"> <span class="label" tagx="label" title="label">Figure 6</span> </a> </div>   <p>Comparison of border distance between adjacent genes and junctions of LSC, SSC, and two IR regions among the chloroplast genomes of twelve <i>Arabidopsis</i> species. Boxes above or below the main line indicate the adjacent border genes. The figure is not to scale with respect to sequence length and only shows relative changes at or near the IR/SC borders. </p>   </div> <p /> </div> <div class="phylogeneticanalysisofa.hallerissp.gemmiferaanda.lyratassp.petraea" title="sec"> <div class="title" tagx="title" title="title">
Phylogenetic analysis of <i>A</i>. <i>halleri</i> ssp. <i>gemmifera</i> and <i>A</i>. <i>lyrata</i> ssp. <i>petraea</i> </div> <p>In this study, the phylogenetic position of <i>A</i>. <i>halleri</i> ssp. <i>gemmifera</i> and <i>A</i>. <i>lyrata</i> ssp. <i>petraea</i> within the family <i>Brassicaceae</i> was established by analysing multiple alignments of complete cp genomes and 70 shared genes of 29 Brassicaceae members representing 12 genera (Fig. <a href="#Fig7">7</a> and Fig. <a href="#MOESM1">S4</a>). <i>Carica papaya</i> was set as the outgroup. Phylogenetic analyses using Bayesian inference (BI), maximum parsimony (MP), maximum likelihood (ML) and neighbour-joining (NJ) were performed. The results revealed that complete cp genomes and 70 shared genes of <i>A</i>. <i>halleri</i> ssp. <i>gemmifera</i> and <i>A</i>. <i>lyrata</i> ssp. <i>petraea</i> contain the same phylogenetic signals; the complete genome sequence and the 70 shared genes (from all species) generated phylogenetic trees with identical topologies (Fig. <a href="#Fig7">7</a> and Fig. <a href="#MOESM1">S4</a>). Maximum likelihood (ML) analysis revealed 22 out of 26 nodes with bootstrap values ≥ 99%, and most of these nodes had 100% bootstrap values. In these phylogenetic trees based on the entire genome data set and the 70 shared genes, <i>A</i>. <i>halleri</i> ssp. <i>gemmifera</i> formed a single clade with <i>A</i>. <i>umezawana</i>, and <i>A</i>. <i>lyrata</i> ssp. <i>petraea</i> formed a single clade with <i>A</i>. <i>arenicola</i> for high Bayesian inference (BI), and bootstrap support using four different methods (Fig. <a href="#Fig7">7</a> and Fig. <a href="#MOESM1">S4</a>). </p><div class="fig" title="fig"> <div class="figure" title="figure"> <a href="http://doi.org/"> <span class="label" tagx="label" title="label">Figure 7</span> </a> </div>   <p>Phylogenetic trees were constructed for twenty-eight species from the family Brassicaceae using several different methods, and the tree shown is for the 70 shared protein coding genes. The following four different methods were used for the 70 shared genes data set: Bayesian inference (BI), maximum parsimony (MP), maximum likelihood (ML) and neighbour-joining (NJ). Numbers above the branches are the posterior probabilities of BI and bootstrap values for NJ, MP and ML. Stars represent the positions of <i>A</i>. <i>halleri</i> ssp. <i>gemmifera</i> and <i>A</i>. <i>lyrata</i> ssp. <i>petraea</i>. </p>   </div> <p /> </div> </div> <div class="discussion" title="discussion"> <div class="title" tagx="title" title="title">
Discussion</div> <p>This study reports the complete chloroplast genomes of <i>A</i>. <i>halleri</i> ssp. <i>gemmifera</i> and <i>A</i>. <i>lyrata</i> ssp. <i>petraea</i>, ranging from 154.4~154.5 kbp in length. Both cp genomes exhibit a typical quadripartite structure, as reported for other angiosperms. Both genomes encode ~130 genes, including 85 protein-coding genes, 8 ribosomal RNA genes and 37 transfer RNA genes, with 227 to 229 microsatellites distributed randomly throughout their genomes, respectively. In addition, approximately 26/31 forward, 34/32 tandem and 11/12 palindromic repeats were found in both cp genomes. This conformed with the protein coding genes found in other <i>Brassicaceae</i> members, such as <i>A</i>. <i>thaliana</i> <sup> <a href="#CR16">16</a> </sup>, <i>Brassica nigra</i>, <i>Brassica oleracea</i>, <sup> <a href="#CR41">41</a> </sup> and <i>Capsella rubella</i> <sup> <a href="#CR42">42</a> </sup>. Among the coding genes, <i>rps12</i> is an unequally divided gene, with its 5′ terminal exon located in the LSC region, while two copies of the 3′ terminal exon and intron are located in IRs, which is similar to other angiosperm cp genomes. The pseudogene <i>ycf1</i> was located in the boundary regions between IRb and SSC, leading to incomplete duplication of the gene within IR regions. We found twelve intron-containing genes in both of the cp genomes, nine of which contained one intron, whereas the genes <i>ycf3</i>, <i>clpP</i>, and <i>rps12</i> contained two introns. The gene <i>ndhA</i> had the longest intron in both <i>A</i>. <i>halleri</i> ssp. <i>gemmifera</i> (1,089 bp) and <i>A</i>. <i>lyrata</i> ssp. <i>petraea</i> (1,084) (Tables <a href="#MOESM1">S10</a> and <a href="#MOESM1">11</a>). This intron plays an important role in the regulation of gene expression, and some recent research revealed that various introns improve exogenous gene expression at specific positions. Therefore, the intron can be a valuable tool to improve transformational efficiency <sup> <a href="#CR43">43</a> </sup>. It was observed that <i>ycf1</i>, <i>ycf2</i> <sup> <a href="#CR44">44</a>, <a href="#CR45">45</a> </sup>, <i>rpl23</i> <sup> <a href="#CR46">46</a> </sup> and <i>accD</i> <sup> <a href="#CR47">47</a>, <a href="#CR48">48</a> </sup> are often absent in plants <sup> <a href="#CR46">46</a> </sup>, but they were detected in the reported <i>Arabidopsis</i> cp genomes. The pair of genes <i>atpB-atpE</i> was observed to overlap each other by ~4 bp. However, <i>psbC-psbD</i> had a 92 bp overlap in the <i>A</i>. <i>halleri</i> ssp. <i>gemmifera</i> and <i>A</i>. <i>lyrata</i> ssp. <i>petraea</i> cp genomes, while this overlap was 53 bp in <i>A</i>. <i>thaliana</i>, 17 bp in <i>A</i>. <i>arenosa</i>, 53 bp in <i>Gossypium</i> <sup> <a href="#CR49">49</a> </sup> and 52 bp in <i>Camellia</i> cp genomes <sup> <a href="#CR50">50</a> </sup>. Similar ratios of amino acids were observed in comparison to the previously reported cp genomes <sup> <a href="#CR50">50</a>– <a href="#CR52">52</a> </sup>. The preference for a high AT content at the 3 <sup>rd</sup> codon position is consistent with the A and T concentrations reported in various terrestrial plant cp genomes <sup> <a href="#CR17">17</a>, <a href="#CR51">51</a>, <a href="#CR53">53</a>, <a href="#CR54">54</a> </sup>. </p> <p>We found 71 and 75 repeats in the cp genomes of <i>A</i>. <i>halleri</i> ssp. <i>gemmifera</i> and <i>A</i>. <i>lyrata</i> ssp. <i>petraea</i>, respectively, which included reversed, direct and palindromic repeats. Repeat sequences are very helpful in phylogenetic studies and play a role in genome rearrangements <sup> <a href="#CR53">53</a>, <a href="#CR55">55</a> </sup>. Furthermore, analyses of the various cp genomes concluded that repeat sequences are essential to inducing indels and substitutions <sup> <a href="#CR56">56</a> </sup>. The length of direct and palindromic repeats in the <i>A</i>. <i>halleri</i> ssp. <i>gemmifera</i> and <i>A</i>. <i>lyrata</i> ssp. <i>petraea</i> cp genomes were much shorter, ranging from 30–101 bp., and similar results have been previously reported in <i>Camellia</i>, which has an 82 bp repeat. However, much longer repeats have been observed, such as the 132 bp and 287 bp repeats found in <i>Poaceae</i> and <i>Fabaceae</i>, respectively <sup> <a href="#CR57">57</a> </sup>. Previous reports suggested that sequence variation and genome rearrangement occur due to slipped strand mispairing and the improper recombination of these repeat sequences <sup> <a href="#CR58">58</a>, <a href="#CR59">59</a> </sup>. Furthermore, the presence of these repeats indicates that the region is a crucial hotspot for genome reconfiguration <sup> <a href="#CR59">59</a> </sup>. Additionally, these repeats are an informative source for developing genetic markers for <i>A</i>. <i>halleri</i> ssp. <i>gemmifera</i> and <i>A</i>. <i>lyrata</i> ssp. <i>petraea</i> for phylogenetic and population studies <sup> <a href="#CR55">55</a> </sup>. </p> <p>During genome annotation, we noted perfect SSRs in <i>A</i>. <i>halleri</i> ssp. <i>gemmifera</i> and <i>A</i>. <i>lyrata</i> ssp. <i>petraea</i> compared to the ten other <i>Arabidopsis</i> species cp genomes. SSRs usually have a higher mutation rate compared to other neutral DNA regions due to slipped DNA strands. These are present in the cp genome at the highest diversity in copy numbers and are important molecular markers for plant population genetics, evolutionary, and ecological studies <sup> <a href="#CR60">60</a> </sup>. We looked for SSRs of 10 bp or longer, as these have been suggested to be prone to slipped strand mispairing. This is believed to be the main mutational mechanism for SSR polymorphisms <sup> <a href="#CR61">61</a>, <a href="#CR62">62</a> </sup>. From our SSR analysis, 227 and 229 microsatellites were found in the <i>A</i>. <i>halleri</i> ssp. <i>gemmifera</i> and <i>A</i>. <i>lyrata</i> ssp. <i>petraea</i> cp genomes, respectively. In addition, 226, 215, 214, 221, 214, 216, 213, 216, 220, and 216 SSRs were detected in <i>A</i>. <i>thaliana</i>, <i>A</i>. <i>arenosa</i>, <i>A</i>. <i>cebennensis</i>, <i>A</i>. <i>pedemontana</i>, <i>A</i>. <i>arenicola</i>, <i>A</i>. <i>croatica</i>, <i>A</i>. <i>neglecta</i>, <i>A</i>. <i>petrogena</i>, <i>A</i>. <i>suecica</i> and <i>A</i>. <i>umezawana</i>, respectively. </p> <p>These findings are consistent with the previous observation that the SSRs of cp genomes are dominated by ‘A’ or ‘T’ mononucleotide repeats <sup> <a href="#CR16">16</a>, <a href="#CR63">63</a> </sup>. Mononucleotide, pentanucleotide and hexanucleotides repeats were also composed of ‘A’ or ‘T’ at greater frequencies, which reflects a biased base composition, with an overall A-T richness in the cp genomes <sup> <a href="#CR64">64</a>, <a href="#CR65">65</a> </sup>. Our findings are comparable to previous reports that showed that SSRs found in the cp genome are generally composed of polythymine (polyT) or polyadenine (polyA) repeats and infrequently contain tandem cytosine (C) and guanine (G) repeats <sup> <a href="#CR65">65</a> </sup>. Therefore, these SSRs contribute to the ‘AT’ richness of the <i>A</i>. <i>halleri</i> ssp. <i>gemmifera</i> and <i>A</i>. <i>lyrata</i> ssp. <i>petraea</i> cp genomes, as previously reported for different species <sup> <a href="#CR52">52</a>, <a href="#CR66">66</a> </sup>. The current analysis showed that approximately 69% ( <i>A</i>. <i>halleri</i> ssp. g <i>emmifera</i>) and 77% ( <i>A</i>. <i>lyrata</i> ssp. <i>petraea</i>) of SSRs were detected in non-coding regions. These results agree with previous reports that SSRs are unevenly distributed in cp genomes and might provide more information for selecting effective molecular markers for the detection of intra- and interspecific polymorphisms <sup> <a href="#CR67">67</a>, <a href="#CR68">68</a> </sup>. </p> <p>Our results reveal that both the <i>A</i>. <i>halleri</i> ssp. <i>gemmifera</i> and <i>A</i>. <i>lyrata</i> ssp. <i>petraea</i> cp genomes share high sequence similarity with all ten <i>Arabidopsis</i> species. However, relatively lower identity was also observed with these species in several comparable genomic regions. In addition, similar to previously reported cp genomes <sup> <a href="#CR51">51</a>, <a href="#CR52">52</a>, <a href="#CR69">69</a>– <a href="#CR71">71</a> </sup>, the LSC and SSC regions were less similar than the two IR regions in all <i>Arabidopsis</i> species cp genomes. Similar results were reported previously in various higher plant cp genomes, which suggests that the lower sequence divergence in the IR regions compared to the SC and LSC regions is possibly due to copy correction between IR sequences by gene conversion <sup> <a href="#CR72">72</a> </sup>. Furthermore, the non-coding regions showed greater divergence than the coding regions. The divergent regions included <i>trnK</i>- <i>rps16</i>, <i>rpoC1</i>, <i>TrnL</i>- <i>TrnF</i>, <i>atpB</i>- <i>rbcL</i>, <i>accD</i>, <i>petA</i>- <i>psbJ</i>, <i>petD</i>- <i>rpoA</i>, <i>ccsA</i>, <i>rpl33</i>, <i>rps12</i>, <i>psbM</i>, <i>ndhD</i> and <i>ycf2</i>. Similar results for these genes were reported previously <sup> <a href="#CR51">51</a>, <a href="#CR52">52</a> </sup>, and these results also confirm similar differences among various coding regions in the analysed species suggested by Yang <i>et al</i>. <sup> <a href="#CR73">73</a> </sup>. These results are consistent with previous reports that these divergent genes are mostly present in LSC regions and show a trend towards more rapid evolution <sup> <a href="#CR52">52</a> </sup>. </p> <p>Regarding the IR regions, the expansion and contraction at the borders are the main reasons for size variations among cp genomes, and it plays a vital role in evolution <sup> <a href="#CR71">71</a>, <a href="#CR74">74</a>, <a href="#CR75">75</a> </sup>. A detailed comparison between the two IRs and two single-copy regions was performed among <i>A</i>. <i>thaliana</i>, <i>A</i>. <i>arenosa</i>, <i>A</i>. <i>cebennensis</i>, <i>A</i>. <i>pedemontana</i>, <i>A</i>. <i>arenicola</i>, <i>A</i>. <i>croatica</i>, <i>A</i>. <i>neglecta</i>, <i>A</i>. <i>petrogena</i>, <i>A</i>. <i>suecica</i> and <i>A</i>. <i>umezawana</i> in comparison to <i>A</i>. <i>halleri</i> ssp. <i>gemmifera</i> and <i>A</i>. <i>lyrata</i> ssp. <i>petraea</i>. In <i>A</i>. <i>halleri</i> ssp. <i>gemmifera</i> and <i>A</i>. <i>lyrata</i> ssp. <i>petraea</i>, the <i>ycf1</i> gene is located 1030 and 1028 bp in the IR regions, respectively. This gene is the second largest gene in the plastid genome and encodes a protein of approximately 1,800 amino acids. Recently, researchers reported that <i>ycf1</i> is essential for plant viability and encodes Tic214, a vital component of the <i>Arabidopsis</i> TIC complex <sup> <a href="#CR76">76</a>, <a href="#CR77">77</a> </sup>. </p> <p>Recently, Dong <i>et al</i>. found that two regions of the plastid gene <i>ycf1</i> were highly variable in flowering plants <sup> <a href="#CR77">77</a> </sup>. Specifically, the <i>ycf1</i> (pseudogene) located in the IRb region is conserved, while the <i>ycf1</i> in the SSC is highly variable. This region of the ycf1 gene is more variable than <i>matK</i> in most taxa investigated thus far <sup> <a href="#CR44">44</a>, <a href="#CR45">45</a> </sup>. Therefore, researchers used the two regions of <i>ycf1</i> as a new tool to solve phylogenetic problems at the species level and for DNA barcoding of some closely related flowering plant species <sup> <a href="#CR78">78</a>– <a href="#CR81">81</a> </sup>. Furthermore, two regions within <i>ycf1</i>, <i>ycf1a and ycf1b</i> have been predicted to have the highest nucleotide diversity (π) at the species level within angiosperm plastid genomes <sup> <a href="#CR77">77</a>, <a href="#CR82">82</a> </sup>. </p> <p>Chloroplast genomes have shown substantial power in studies of phylogenetics, evolution and molecular systematics. During the last decade, there have been many analyses to address phylogenetic questions at deep nodes based on comparisons of multiple protein-coding genes <sup> <a href="#CR83">83</a>, <a href="#CR84">84</a> </sup> and complete cp genome sequences that enhance our understanding of enigmatic evolutionary relationships among angiosperms <sup> <a href="#CR38">38</a> </sup>. The genus <i>Arabidopsis</i> has been estimated to comprise at least 9 species and 6 subspecies <sup> <a href="#CR85">85</a> </sup> or up to 13 species and 9 subspecies <sup> <a href="#CR86">86</a> </sup> depending on the taxonomic approach and the identifier. According to the Hohmann <i>et al</i>. <sup> <a href="#CR38">38</a> </sup> taxonomy, the genus <i>Arabidopsis</i> has been proposed to consist of as many as 26 taxa, including the model plant <i>A</i>. <i>thaliana</i> <sup> <a href="#CR34">34</a>, <a href="#CR86">86</a> </sup>. Continued efforts have enhanced our ability to differentiate lineages and to understand the genomic structure and phylogenetic relationships of <i>Arabidopsis</i> species <sup> <a href="#CR34">34</a> </sup>. Previous evolutionary relationships among different <i>Arabidopsis</i> genomes and species were estimated using nuclear and chloroplast DNA <sup> <a href="#CR87">87</a>, <a href="#CR88">88</a> </sup> restriction fragment-length polymorphisms, but complete genome sequencing provides more detailed insights <sup> <a href="#CR34">34</a> </sup>. Recently, Novikova <i>et al</i>. <sup> <a href="#CR34">34</a> </sup> reported Illumina sequencing and phylogenetic analysis based on polymorphism data and chloroplast genome data for 26 taxa, including those presented in the current study, that constitute the genus <i>Arabidopsis</i> <sup> <a href="#CR34">34</a> </sup>. In our study, the phylogenetic positions of <i>A</i>. <i>halleri</i> ssp. <i>gemmifera</i> and <i>A</i>. <i>lyrata</i> ssp. <i>petraea</i> within the family <i>Brassicaceae</i> were established by sequencing complete cp genomes and analysing 70 shared genes of 29 <i>Brassicaceae</i> members, representing 12 genera. The phylogenetic analysis showed that the complete cp genomes and the 70 shared gene data set exhibit identical phylogenetic signals. In both the entire genome data set and the 70 shared genes data set, <i>A</i>. <i>halleri</i> ssp. <i>gemmifera</i> forms a single clade with <i>A</i>. <i>umezawana</i>, and <i>A</i>. <i>lyrata</i> ssp. <i>petraea</i> forms a single clade with <i>A</i>. <i>arenicola</i>. Similar results were described by Hohmann <i>et al</i>. <sup> <a href="#CR38">38</a> </sup> based on <i>trnLF</i> and ribosomal ITS data, finding that <i>A</i>. <i>halleri</i> ssp. <i>gemmifera</i> is a sister to <i>A</i>. <i>umezawana</i>. The results also confirmed a previous report by Hohmann <i>et al</i>. <sup> <a href="#CR38">38</a> </sup> reporting that <i>A</i>. <i>lyrata</i> ssp. <i>petraea</i> and <i>A</i>. <i>arenicola</i> are close relatives and that <i>A</i>. <i>arenicola</i> probably originated postglacially from an <i>A</i>. <i>lyrata</i> population <sup> <a href="#CR37">37</a> </sup>. Furthermore, these results are also in broad agreement with previous results reported by Novikoa <i>et al</i>. <sup> <a href="#CR34">34</a> </sup>, where <i>A</i>. <i>halleri</i> ssp. <i>gemmifera</i> was most closely related to <i>A</i>. <i>umezawana</i> and <i>A</i>. <i>lyrata</i> ssp. <i>petraea</i> formed a clade with <i>A</i>. <i>arenicola</i> <sup> <a href="#CR34">34</a> </sup>. </p> <p>In conclusion, we assembled and analysed the complete chloroplast genomes of <i>A</i>. <i>halleri</i> ssp. <i>gemmifera</i> and <i>A</i>. <i>lyrata</i> ssp. <i>petraea</i> and compared them with other <i>Arabidopsis</i> species for the first time. The genome organization, gene order, GC content and codon usage were similar to those of previously reported cp genomes from the genus <i>Arabidopsis</i>. The location and distribution of repeat sequences was determined, and sequence divergences of cp genomes and 70 shared genes were calculated with related species. The phylogenetic analysis based on whole cp genomes and 70 shared genes yielded identical phylogenetic trees, with <i>A</i>. <i>halleri</i> ssp. <i>gemmifera</i> and <i>A</i>. <i>lyrata</i> ssp. <i>petraea</i> forming single clades with <i>A</i>. <i>umezawana</i> and <i>A</i>. <i>arenicola</i>, respectively. </p> </div> <div class="materials_methods" title="materials|methods"> <div class="title" tagx="title" title="title">
Materials and Methods</div> <div class="genomesequencingandassembly" title="sec"> <div class="title" tagx="title" title="title">
Genome Sequencing and Assembly</div> <p>A standard protocol of DNA extraction was followed, as described in detail by Hu <i>et al</i>. <sup> <a href="#CR24">24</a> </sup>. Pure DNA was sequenced on an Illumina HiSeq. 2000. A total of 63,528,604 and 67,938,537 raw reads were generated for <i>A</i>. <i>halleri</i> ssp. <i>gemmifera</i> and <i>A</i>. <i>lyrata</i> ssp. <i>petraea</i>, respectively, which were then trimmed and filtered using CLC Genomics Workbench v7.0 (CLC Bio, Aarhus, Denmark). Then, CLC Genomics Workbench v7.0 (CLC Bio, Aarhus, Denmark) was used for <i>de novo</i> genome assembly. Different k-mer sizes were evaluated, and a k-mer size of 66 provided the best results in terms of minimum members and longest average length of scaffolds. These parameters were used to generate the final assembly. Then, the resulting contigs were compared against the <i>A</i>. <i>thaliana</i> chloroplast genome using BLASTN with an E-value cutoff of 1e-5. Six and seven contigs of <i>A</i>. <i>halleri</i> ssp. <i>gemmifera</i> and <i>A</i>. <i>lyrata</i> ssp. <i>petraea</i>, respectively, were identified and were temporarily arranged based on their mapping positions on the reference genome. Then, primers were designed (Table <a href="#MOESM1">S1</a>) based on the sequence at the ends of the adjacent contigs. PCR amplification and subsequent DNA sequencing were used to fill the gaps. PCR amplification was performed in a total volume of 20 μl containing 1 × reaction buffer, 0.1 μl Taq DNA Polymerase, 0.4 μl dNTP (10 mM) and 1 μl (10 ng/μl) of DNA. The PCR programme was composed of an initial denaturation at 95 °C for 5 min followed by 32 cycles at 95 °C for 30 s, 60 °C for 20 s and 72 °C for 30 s, with a final extension step at 72 °C for 5 min. After incorporation of the Sanger sequencing results, the completed cp genome was used as a reference to map the initial short reads to refine the assembly based on maximum sequence coverage. </p> </div> <div class="genomeannotationandsequencearchitecture" title="sec"> <div class="title" tagx="title" title="title">
Genome Annotation and Sequence Architecture</div> <p>A program (DOGMA) was used to annotate the <i>A</i>. <i>halleri</i> ssp. <i>gemmifera</i> and <i>A</i>. <i>lyrata</i> ssp. <i>petraea</i> cp genomes <sup> <a href="#CR89">89</a> </sup>. The annotation results were checked manually, and codon positions were adjusted by comparing to homologues from the database <i>A</i>. <i>thaliana</i> cp genome. All transfer RNAs were verified using tRNAscan-SE version 1.21 <sup> <a href="#CR90">90</a> </sup> using the default settings. OGDRAW <sup> <a href="#CR91">91</a> </sup> was used to illustrate structural features of the <i>A</i>. <i>halleri</i> ssp. <i>gemmifera</i> and <i>A</i>. <i>lyrata</i> ssp. <i>petraea</i> cp genomes. The relative synonymous codon usage (RSCU) was determined using MEGA 6.0 <sup> <a href="#CR92">92</a> </sup> to examine deviations in synonymous codon usage by avoiding the influence of amino acid composition. The software mVISTA was used in Shuffle-LAGAN mode to compare the whole genome variations of <i>A</i>. <i>halleri</i> ssp. <i>gemmifera</i> and <i>A</i>. <i>lyrata</i> ssp. <i>petraea</i> genome with four other cp genomes using the <i>A</i>. <i>halleri</i> ssp. <i>gemmifera</i> and <i>A</i>. <i>lyrata</i> ssp. <i>petraea</i> annotation as a reference <sup> <a href="#CR93">93</a> </sup>. </p> </div> <div class="characterizationofrepeatsequenceandssrs" title="sec"> <div class="title" tagx="title" title="title">
Characterization of Repeat sequence and SSRs</div> <p>We used REPuter to identify repeat sequences, including palindromic, reverse, and direct repeats, within the cp genome <sup> <a href="#CR94">94</a> </sup>. The following settings for repeat identification were used in REPuter: 1) Hamming distance of 3, 2) 90% or greater sequence identity, and 3) a minimum repeat size of 30 bp. Phobos version 3.3.12 <sup> <a href="#CR95">95</a> </sup> was used to detect (SSRs) within the cp genome, with the search parameters set at ≥10 repeat units for mononucleotides, ≥8 repeat units for dinucleotides, ≥4 repeat units for trinucleotides and tetranucleotides, and ≥3 repeat units for pentanucleotide and hexanucleotide SSRs. Tandem repeats in the <i>A</i>. <i>lyrata</i> and <i>A</i>. <i>halleri</i> cp genomes were identified using Tandem Repeats Finder version 4.07 b with default settings <sup> <a href="#CR96">96</a> </sup>. </p> </div> <div class="sequencedivergenceandphylogeneticanalysis" title="sec"> <div class="title" tagx="title" title="title">
Sequence Divergence and Phylogenetic Analysis</div> <p>Complete cp genomes and a separate partition containing only the 70 shared genes were used to analyse the average pairwise sequence divergence for ten <i>Arabidopsis</i> species: <i>A</i>. <i>thaliana</i>, <i>A</i>. <i>arenosa</i>, <i>A</i>. <i>cebennensis</i>, <i>A</i>. <i>pedemontana</i>, <i>A</i>. <i>arenicola</i>, <i>A</i>. <i>croatica</i>, <i>A</i>. <i>neglecta</i>, <i>A</i>. <i>petrogena</i>, <i>A</i>. <i>suecica</i> and <i>A</i>. <i>umezawana</i>. Missing and ambiguous gene annotations were confirmed by comparative sequence analysis after a multiple sequence alignment and gene order comparison. These regions were aligned using MAFFT (version 7.222) <sup> <a href="#CR97">97</a> </sup> with default parameters. Kimura’s two parameter (K2P) model was selected to calculate pairwise sequence divergences <sup> <a href="#CR98">98</a> </sup>. To resolve the <i>A</i>. <i>halleri</i> ssp. <i>gemmifera</i> and <i>A</i>. <i>lyrata</i> ssp. <i>petraea</i> phylogenetic positions within the family <i>Brassicaceae</i>, twenty-nine published cp genomes were downloaded from the NCBI database for analysis. First, multiple alignments were performed using complete cp genomes based on the conserved structure and gene order of chloroplast genomes <sup> <a href="#CR98">98</a> </sup>. Four methods were employed to construct the phylogenetic trees, including Bayesian inference (BI) implemented with MrBayes 3.1.2 <sup> <a href="#CR99">99</a> </sup>, maximum parsimony (MP) with PAUP 4.0 <sup> <a href="#CR100">100</a> </sup>, maximum likelihood (ML) and neighbour-joining (NJ) with MEGA 6 <sup> <a href="#CR92">92</a> </sup> using settings derived from Wu <i>et al</i>. <sup> <a href="#CR101">101</a> </sup>. MP was run using a heuristic search with 1000 random addition sequence replicates with the tree-bisection-reconnection (TBR) branch-swapping tree search criterion. Parameters for the ML analysis were optimized with a BIONJ tree as the starting tree with 1000 bootstrap replicates using the Kimura 2-parameter model with gamma-distributed rate heterogeneity and invariant sites. For Bayesian posterior probabilities (PP) in the BI analyses, the best substitution model GTR + G model was tested according to the Akaike information criterion (AIC) by jModelTest verion 2 <sup> <a href="#CR102">102</a> </sup>. The Markov Chain Monto Carlo (MCMC) was run for 1,000,000 generations with 4 incrementally heated chains, starting from random trees and sampling 1 out of every 100 generations. The first 25% of trees were discarded as burn-in to estimate the value of posterior probabilities. In the second phylogenetic analysis, 70 shared genes from the cp genomes of the twenty-two <i>Brassicaceae</i> members, with <i>Carica papaya</i> as the outgroup species, were aligned in ClustalX using the default settings, followed by manual adjustment to preserve reading frames. The above four phylogenetic-inference methods were used to infer trees from these 65 concatenated genes using the same settings described above and in Yao <i>et al</i>. <sup> <a href="#CR103">103</a> </sup>. </p> </div> </div> <div class="supplementary-material" title="supplementary-material"> <div class="title" tagx="title" title="title">
Electronic supplementary material</div> <div class="sec" title="sec"> <p> </p><div class="local-data" title="local-data"> <div class="media" title="media"> <a href="41598_2017_7891_MOESM1_ESM.docx">LINK</a>  <p>SI All files</p>  </div> </div> <div class="local-data" title="local-data"> <div class="media" title="media"> <a href="41598_2017_7891_MOESM2_ESM.xls">LINK</a>  <p>SI Table 9</p>  </div> </div> <p /> </div> </div> </div> <div class="back" title="back"> <div class="fn-group" title="fn-group"> <div class="fn-type-" title=""> <p> <b>Electronic supplementary material</b> </p> <p> <b>Supplementary information</b> accompanies this paper at doi:10.1038/s41598-017-07891-5 </p> </div> <div class="fn-type-" title=""> <p> <b>Publisher's note:</b> Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations. </p> </div> </div> <div class="ack" title="ack"> <div class="title" tagx="title" title="title">
Acknowledgements</div> <p>This study was supported by a research grant (311058-05-3-CG000) from the Ministry for Food, Agriculture, Forestry, and Fisheries, Republic of Korea. </p> </div> <div class="title" tagx="title" title="title">
Author Contributions</div> <p>S.A. and A.L.K. planned, designed, and performed the research and wrote the findings; B.W.Y. and I.J.L. edited the manuscript and arranged the experimental resources; M.A.K., analysed the data. S.M.K. and M.W. edited the language. All authors read and approved the final manuscript. </p> <div class="competinginterests" title="sec"> <div class="title" tagx="title" title="title">
Competing Interests</div> <p>The authors declare that they have no competing interests.</p> </div> <div class="references">
References</div> <div tag="ref-list"> <ul> <div class="title" tagx="title" title="title">
References</div> <li tag="ref"> <a name="CR1" /> <span class="label" tagx="label" title="label">1.</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation">Bock, R. Cell and molecular biology of plastids (Springer, 2007).</span> </li> <li tag="ref"> <a name="CR2" /> <span class="label" tagx="label" title="label">2.</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation">Daniell, H., Chan, H.-T. &amp;amp; Pasoreck, E. K. Vaccination Through Chloroplast Genetics: Affordable Protein Drugs for the Prevention and Treatment of Inherited or Infectious Human Diseases. <i>Annu Rev Genet</i> <b>50</b>, null, doi:10.1146/annurev-genet-120215-035349 (2016). </span> </li> <li tag="ref"> <a name="CR3" /> <span class="label" tagx="label" title="label">3.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Corriveau</span> <span class="given-names" tagx="given-names" title="given-names">JL</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Coleman</span> <span class="given-names" tagx="given-names" title="given-names">AW</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Rapid Screening Method to Detect Potential Biparental Inheritance of Plastid DNA and Results for Over 200 Angiosperm Species </span> <span class="source" tagx="source" title="source">Am J Bot</span> <span class="year" tagx="year" title="year">1988</span> <span class="volume" tagx="volume" title="volume">75</span> <span class="fpage" tagx="fpage" title="fpage">1443</span> <span class="lpage" tagx="lpage" title="lpage">1458</span> <span class="pub-id"> <a href="https://dx.doi.org/10.2307/2444695">10.2307/2444695</a> </span> </span> </li> <li tag="ref"> <a name="CR4" /> <span class="label" tagx="label" title="label">4.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Zhang</span> <span class="given-names" tagx="given-names" title="given-names">Q</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Liu</span> <span class="given-names" tagx="given-names" title="given-names">Y</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Sodmergen.</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Examination of the cytoplasmic DNA in male reproductive cells to determine the potential for cytoplasmic inheritance in 295 angiosperm species </span> <span class="source" tagx="source" title="source">Plant Cell Physiol</span> <span class="year" tagx="year" title="year">2003</span> <span class="volume" tagx="volume" title="volume">44</span> <span class="fpage" tagx="fpage" title="fpage">941</span> <span class="lpage" tagx="lpage" title="lpage">951</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1093/pcp/pcg121">10.1093/pcp/pcg121</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/14519776">14519776</a> </span> </span> </li> <li tag="ref"> <a name="CR5" /> <span class="label" tagx="label" title="label">5.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Wolfe</span> <span class="given-names" tagx="given-names" title="given-names">KH</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Li</span> <span class="given-names" tagx="given-names" title="given-names">WH</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Sharp</span> <span class="given-names" tagx="given-names" title="given-names">PM</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Rates of nucleotide substitution vary greatly among plant mitochondrial, chloroplast, and nuclear DNAs </span> <span class="source" tagx="source" title="source">Proc Natl Acad Sci USA</span> <span class="year" tagx="year" title="year">1987</span> <span class="volume" tagx="volume" title="volume">84</span> <span class="fpage" tagx="fpage" title="fpage">9054</span> <span class="lpage" tagx="lpage" title="lpage">9058</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1073/pnas.84.24.9054">10.1073/pnas.84.24.9054</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/3480529">3480529</a> </span> </span> </li> <li tag="ref"> <a name="CR6" /> <span class="label" tagx="label" title="label">6.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Provan</span> <span class="given-names" tagx="given-names" title="given-names">J</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Powell</span> <span class="given-names" tagx="given-names" title="given-names">W</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Hollingsworth</span> <span class="given-names" tagx="given-names" title="given-names">PM</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Chloroplast microsatellites: new tools for studies in plant ecology and evolution</span> <span class="source" tagx="source" title="source">Trends in ecology &amp;amp; evolution</span> <span class="year" tagx="year" title="year">2001</span> <span class="volume" tagx="volume" title="volume">16</span> <span class="fpage" tagx="fpage" title="fpage">142</span> <span class="lpage" tagx="lpage" title="lpage">147</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1016/S0169-5347(00)02097-8">10.1016/S0169-5347(00)02097-8</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/11179578">11179578</a> </span> </span> </li> <li tag="ref"> <a name="CR7" /> <span class="label" tagx="label" title="label">7.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Ravi</span> <span class="given-names" tagx="given-names" title="given-names">V</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Khurana</span> <span class="given-names" tagx="given-names" title="given-names">JP</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Tyagi</span> <span class="given-names" tagx="given-names" title="given-names">AK</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Khurana</span> <span class="given-names" tagx="given-names" title="given-names">P</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">An update on chloroplast genomes</span> <span class="source" tagx="source" title="source">Plant Syst Evol</span> <span class="year" tagx="year" title="year">2008</span> <span class="volume" tagx="volume" title="volume">271</span> <span class="fpage" tagx="fpage" title="fpage">101</span> <span class="lpage" tagx="lpage" title="lpage">122</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1007/s00606-007-0608-0">10.1007/s00606-007-0608-0</a> </span> </span> </li> <li tag="ref"> <a name="CR8" /> <span class="label" tagx="label" title="label">8.</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation">Bock, R., Knoop, V. &amp;amp; SpringerLink (Online service). Genomics of Chloroplasts and Mitochondria. </span> </li> <li tag="ref"> <a name="CR9" /> <span class="label" tagx="label" title="label">9.</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation">Shinozaki, K. <i>et al</i>. The complete nucleotide sequence of the tobacco chloroplast genome: its gene organization and expression. <i>EMBO J</i> <b>5</b> (1986). </span> </li> <li tag="ref"> <a name="CR10" /> <span class="label" tagx="label" title="label">10.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Ohyama</span> <span class="given-names" tagx="given-names" title="given-names">K</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Chloroplast gene organization deduced from complete sequence of liverwort Marchantia polymorpha chloroplast DNA </span> <span class="source" tagx="source" title="source">Nature</span> <span class="year" tagx="year" title="year">1986</span> <span class="volume" tagx="volume" title="volume">322</span> <span class="fpage" tagx="fpage" title="fpage">572</span> <span class="lpage" tagx="lpage" title="lpage">574</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1038/322572a0">10.1038/322572a0</a> </span> </span> </li> <li tag="ref"> <a name="CR11" /> <span class="label" tagx="label" title="label">11.</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation">Daniell, H., Lin, C. S., Yu, M. &amp;amp; Chang, W. J. Chloroplast genomes: diversity, evolution, and applications in genetic engineering. <i>Genome Biol</i> <b>17</b>, doi:10.1186/S13059-016-1004-2 (2016). </span> </li> <li tag="ref"> <a name="CR12" /> <span class="label" tagx="label" title="label">12.</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation">Wambugu, P., Brozynska, M., Furtado, A., Waters, D. &amp;amp; Henry, R. Relationships of wild and domesticated rices (Oryza AA genome species) based upon whole chloroplast genome sequences. <i>Sci Rep</i> <b>5</b>, doi:10.1038/srep13957 (2015). </span> </li> <li tag="ref"> <a name="CR13" /> <span class="label" tagx="label" title="label">13.</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation">Brozynska, M., Furtado, A. &amp;amp; Henry, R. J. Genomics of crop wild relatives: expanding the gene pool for crop improvement. <i>Plant Biotechnol J</i> <b>14</b>, doi:10.1111/pbi.12454 (2016). </span> </li> <li tag="ref"> <a name="CR14" /> <span class="label" tagx="label" title="label">14.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Lei</span> <span class="given-names" tagx="given-names" title="given-names">W</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Intraspecific and heteroplasmic variations, gene losses and inversions in the chloroplast genome of Astragalus membranaceus </span> <span class="source" tagx="source" title="source">Sci Rep</span> <span class="year" tagx="year" title="year">2016</span> <span class="volume" tagx="volume" title="volume">6</span> <span class="fpage" tagx="fpage" title="fpage">21669</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1038/srep21669">10.1038/srep21669</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/26899134">26899134</a> </span> </span> </li> <li tag="ref"> <a name="CR15" /> <span class="label" tagx="label" title="label">15.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Kolodner</span> <span class="given-names" tagx="given-names" title="given-names">R</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Tewari</span> <span class="given-names" tagx="given-names" title="given-names">KK</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Inverted repeats in chloroplast DNA from higher plants</span> <span class="source" tagx="source" title="source">Proceedings of the National Academy of Sciences</span> <span class="year" tagx="year" title="year">1979</span> <span class="volume" tagx="volume" title="volume">76</span> <span class="fpage" tagx="fpage" title="fpage">41</span> <span class="lpage" tagx="lpage" title="lpage">45</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1073/pnas.76.1.41">10.1073/pnas.76.1.41</a> </span> </span> </li> <li tag="ref"> <a name="CR16" /> <span class="label" tagx="label" title="label">16.</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation">Sato, S., Nakamura, Y., Kaneko, T., Asamizu, E. &amp;amp; Tabata, S. Complete structure of the chloroplast genome of Arabidopsis thaliana. <i>DNA Res</i> <b>6</b>, doi:10.1093/dnares/6.5.283 (1999). </span> </li> <li tag="ref"> <a name="CR17" /> <span class="label" tagx="label" title="label">17.</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation">Lin, C. P., Huang, J. P., Wu, C. S., Hsu, C. Y. &amp;amp; Chaw, S. M. Comparative chloroplast genomics reveals the evolution of Pinaceae genera and subfamilies. <i>Genome Biol Evol</i> <b>2</b>, doi:10.1093/gbe/evq036 (2010). </span> </li> <li tag="ref"> <a name="CR18" /> <span class="label" tagx="label" title="label">18.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Pruitt</span> <span class="given-names" tagx="given-names" title="given-names">RE</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Bowman</span> <span class="given-names" tagx="given-names" title="given-names">JL</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Grossniklaus</span> <span class="given-names" tagx="given-names" title="given-names">U</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Plant genetics: a decade of integration</span> <span class="source" tagx="source" title="source">Nat Genet</span> <span class="year" tagx="year" title="year">2003</span> <span class="volume" tagx="volume" title="volume">33</span> <span class="issue" tagx="issue" title="issue">Suppl</span> <span class="fpage" tagx="fpage" title="fpage">294</span> <span class="lpage" tagx="lpage" title="lpage">304</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1038/ng1108">10.1038/ng1108</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/12610539">12610539</a> </span> </span> </li> <li tag="ref"> <a name="CR19" /> <span class="label" tagx="label" title="label">19.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Koornneef</span> <span class="given-names" tagx="given-names" title="given-names">M</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Alonso-Blanco</span> <span class="given-names" tagx="given-names" title="given-names">C</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Vreugdenhil</span> <span class="given-names" tagx="given-names" title="given-names">D</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Naturally occurring genetic variation in Arabidopsis thaliana</span> <span class="source" tagx="source" title="source">Annu Rev Plant Biol</span> <span class="year" tagx="year" title="year">2004</span> <span class="volume" tagx="volume" title="volume">55</span> <span class="fpage" tagx="fpage" title="fpage">141</span> <span class="lpage" tagx="lpage" title="lpage">172</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1146/annurev.arplant.55.031903.141605">10.1146/annurev.arplant.55.031903.141605</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/15377217">15377217</a> </span> </span> </li> <li tag="ref"> <a name="CR20" /> <span class="label" tagx="label" title="label">20.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Tonsor</span> <span class="given-names" tagx="given-names" title="given-names">SJ</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Alonso-Blanco</span> <span class="given-names" tagx="given-names" title="given-names">C</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Koornneef</span> <span class="given-names" tagx="given-names" title="given-names">M</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Gene function beyond the single trait: natural variation, gene effects, and evolutionary ecology in Arabidopsis thaliana </span> <span class="source" tagx="source" title="source">Plant Cell Environ</span> <span class="year" tagx="year" title="year">2005</span> <span class="volume" tagx="volume" title="volume">28</span> <span class="fpage" tagx="fpage" title="fpage">2</span> <span class="lpage" tagx="lpage" title="lpage">20</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1111/j.1365-3040.2004.01264.x">10.1111/j.1365-3040.2004.01264.x</a> </span> </span> </li> <li tag="ref"> <a name="CR21" /> <span class="label" tagx="label" title="label">21.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Nasrallah</span> <span class="given-names" tagx="given-names" title="given-names">ME</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Liu</span> <span class="given-names" tagx="given-names" title="given-names">P</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Nasrallah</span> <span class="given-names" tagx="given-names" title="given-names">JB</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Generation of self-incompatible Arabidopsis thaliana by transfer of two S locus genes from A-lyrata </span> <span class="source" tagx="source" title="source">Science</span> <span class="year" tagx="year" title="year">2002</span> <span class="volume" tagx="volume" title="volume">297</span> <span class="fpage" tagx="fpage" title="fpage">247</span> <span class="lpage" tagx="lpage" title="lpage">249</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1126/science.1072205">10.1126/science.1072205</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/12114625">12114625</a> </span> </span> </li> <li tag="ref"> <a name="CR22" /> <span class="label" tagx="label" title="label">22.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Osborn</span> <span class="given-names" tagx="given-names" title="given-names">TC</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Understanding mechanisms of novel gene expression in polyploids</span> <span class="source" tagx="source" title="source">Trends Genet</span> <span class="year" tagx="year" title="year">2003</span> <span class="volume" tagx="volume" title="volume">19</span> <span class="fpage" tagx="fpage" title="fpage">141</span> <span class="lpage" tagx="lpage" title="lpage">147</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1016/S0168-9525(03)00015-5">10.1016/S0168-9525(03)00015-5</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/12615008">12615008</a> </span> </span> </li> <li tag="ref"> <a name="CR23" /> <span class="label" tagx="label" title="label">23.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Clauss</span> <span class="given-names" tagx="given-names" title="given-names">MJ</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Koch</span> <span class="given-names" tagx="given-names" title="given-names">MA</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Poorly known relatives of Arabidopsis thaliana</span> <span class="source" tagx="source" title="source">Trends Plant Sci</span> <span class="year" tagx="year" title="year">2006</span> <span class="volume" tagx="volume" title="volume">11</span> <span class="fpage" tagx="fpage" title="fpage">449</span> <span class="lpage" tagx="lpage" title="lpage">459</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1016/j.tplants.2006.07.005">10.1016/j.tplants.2006.07.005</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/16893672">16893672</a> </span> </span> </li> <li tag="ref"> <a name="CR24" /> <span class="label" tagx="label" title="label">24.</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation">Hu, T. T. <i>et al</i>. The Arabidopsis lyrata genome sequence and the basis of rapid genome size change. <i>Nat Genet</i> <b>43</b>, 476–+, doi:10.1038/ng.807 (2011). </span> </li> <li tag="ref"> <a name="CR25" /> <span class="label" tagx="label" title="label">25.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Johnston</span> <span class="given-names" tagx="given-names" title="given-names">JS</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Evolution of genome size in Brassicaceae</span> <span class="source" tagx="source" title="source">Ann Bot-London</span> <span class="year" tagx="year" title="year">2005</span> <span class="volume" tagx="volume" title="volume">95</span> <span class="fpage" tagx="fpage" title="fpage">229</span> <span class="lpage" tagx="lpage" title="lpage">235</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1093/aob/mci016">10.1093/aob/mci016</a> </span> </span> </li> <li tag="ref"> <a name="CR26" /> <span class="label" tagx="label" title="label">26.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Lysak</span> <span class="given-names" tagx="given-names" title="given-names">MA</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Koch</span> <span class="given-names" tagx="given-names" title="given-names">MA</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Beaulieu</span> <span class="given-names" tagx="given-names" title="given-names">JM</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Meister</span> <span class="given-names" tagx="given-names" title="given-names">A</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Leitch</span> <span class="given-names" tagx="given-names" title="given-names">IJ</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">The Dynamic Ups and Downs of Genome Size Evolution in Brassicaceae</span> <span class="source" tagx="source" title="source">Mol Biol Evol</span> <span class="year" tagx="year" title="year">2009</span> <span class="volume" tagx="volume" title="volume">26</span> <span class="fpage" tagx="fpage" title="fpage">85</span> <span class="lpage" tagx="lpage" title="lpage">98</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1093/molbev/msn223">10.1093/molbev/msn223</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/18842687">18842687</a> </span> </span> </li> <li tag="ref"> <a name="CR27" /> <span class="label" tagx="label" title="label">27.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Kolník</span> <span class="given-names" tagx="given-names" title="given-names">M</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Marhold</span> <span class="given-names" tagx="given-names" title="given-names">K</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Distribution, chromosome numbers and nomenclature conspect of Arabidopsis halleri (Brassicaceae) in the Carpathians </span> <span class="source" tagx="source" title="source">Biologia</span> <span class="year" tagx="year" title="year">2006</span> <span class="volume" tagx="volume" title="volume">61</span> <span class="fpage" tagx="fpage" title="fpage">41</span> <span class="lpage" tagx="lpage" title="lpage">50</span> <span class="pub-id"> <a href="https://dx.doi.org/10.2478/s11756-006-0007-y">10.2478/s11756-006-0007-y</a> </span> </span> </li> <li tag="ref"> <a name="CR28" /> <span class="label" tagx="label" title="label">28.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Hoffmann</span> <span class="given-names" tagx="given-names" title="given-names">MH</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Evolution of the realized climatic niche in the genus Arabidopsis (Brassicaceae)</span> <span class="source" tagx="source" title="source">Evolution</span> <span class="year" tagx="year" title="year">2005</span> <span class="volume" tagx="volume" title="volume">59</span> <span class="fpage" tagx="fpage" title="fpage">1425</span> <span class="lpage" tagx="lpage" title="lpage">1436</span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/16153029">16153029</a> </span> </span> </li> <li tag="ref"> <a name="CR29" /> <span class="label" tagx="label" title="label">29.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Bert</span> <span class="given-names" tagx="given-names" title="given-names">V</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">MacNair</span> <span class="given-names" tagx="given-names" title="given-names">MR</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">DeLaguerie</span> <span class="given-names" tagx="given-names" title="given-names">P</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Saumitou-Laprade</span> <span class="given-names" tagx="given-names" title="given-names">P</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Petit</span> <span class="given-names" tagx="given-names" title="given-names">D</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Zinc tolerance and accumulation in metallicolous and nonmetallicolous populations of Arabidopsis halleri (Brassicaceae) </span> <span class="source" tagx="source" title="source">New Phytol</span> <span class="year" tagx="year" title="year">2000</span> <span class="volume" tagx="volume" title="volume">146</span> <span class="fpage" tagx="fpage" title="fpage">225</span> <span class="lpage" tagx="lpage" title="lpage">233</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1046/j.1469-8137.2000.00634.x">10.1046/j.1469-8137.2000.00634.x</a> </span> </span> </li> <li tag="ref"> <a name="CR30" /> <span class="label" tagx="label" title="label">30.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Roux</span> <span class="given-names" tagx="given-names" title="given-names">C</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Does speciation between Arabidopsis halleri and Arabidopsis lyrata coincide with major changes in a molecular target of adaptation? </span> <span class="source" tagx="source" title="source">Plos One</span> <span class="year" tagx="year" title="year">2011</span> <span class="volume" tagx="volume" title="volume">6</span> <span class="fpage" tagx="fpage" title="fpage">e26872</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1371/journal.pone.0026872">10.1371/journal.pone.0026872</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/22069475">22069475</a> </span> </span> </li> <li tag="ref"> <a name="CR31" /> <span class="label" tagx="label" title="label">31.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Schmickl</span> <span class="given-names" tagx="given-names" title="given-names">R</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Koch</span> <span class="given-names" tagx="given-names" title="given-names">MA</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Arabidopsis hybrid speciation processes</span> <span class="source" tagx="source" title="source">P Natl Acad Sci USA</span> <span class="year" tagx="year" title="year">2011</span> <span class="volume" tagx="volume" title="volume">108</span> <span class="fpage" tagx="fpage" title="fpage">14192</span> <span class="lpage" tagx="lpage" title="lpage">14197</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1073/pnas.1104212108">10.1073/pnas.1104212108</a> </span> </span> </li> <li tag="ref"> <a name="CR32" /> <span class="label" tagx="label" title="label">32.</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation">Castric, V., Bechsgaard, J., Schierup, M. H. &amp;amp; Vekemans, X. Repeated Adaptive Introgression at a Gene under Multiallelic Balancing Selection. <i>Plos Genet</i> <b>4</b>, doi:10.1371/journal.pgen.1000168 (2008). </span> </li> <li tag="ref"> <a name="CR33" /> <span class="label" tagx="label" title="label">33.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Wang</span> <span class="given-names" tagx="given-names" title="given-names">W-K</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Multilocus analysis of genetic divergence between outcrossing Arabidopsis species: evidence of genome-wide admixture </span> <span class="source" tagx="source" title="source">The New Phytologist</span> <span class="year" tagx="year" title="year">2010</span> <span class="volume" tagx="volume" title="volume">188</span> <span class="fpage" tagx="fpage" title="fpage">488</span> <span class="lpage" tagx="lpage" title="lpage">500</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1111/j.1469-8137.2010.03383.x">10.1111/j.1469-8137.2010.03383.x</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/20673288">20673288</a> </span> </span> </li> <li tag="ref"> <a name="CR34" /> <span class="label" tagx="label" title="label">34.</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation">Novikova, P. Y. <i>et al</i>. Sequencing of the genus Arabidopsis identifies a complex history of nonbifurcating speciation and abundant trans-specific polymorphism. <i>Nat Genet</i> <b>48</b>, 1077-1082, doi:10.1038/ng.3617, <a href="http://www.nature.com/ng/journal/v48/n9/abs/ng.3617.html#supplementary-information">http://www.nature.com/ng/journal/v48/n9/abs/ng.3617.html#supplementary-information</a> (2016). </span> </li> <li tag="ref"> <a name="CR35" /> <span class="label" tagx="label" title="label">35.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Kawabe</span> <span class="given-names" tagx="given-names" title="given-names">A</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Miyashita</span> <span class="given-names" tagx="given-names" title="given-names">NT</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">DNA polymorphism in active gene and pseudogene of the cytosolic phosphoglucose isomerase (PgiC) loci in Arabidopsis halleri ssp gemmifera </span> <span class="source" tagx="source" title="source">Mol Biol Evol</span> <span class="year" tagx="year" title="year">2003</span> <span class="volume" tagx="volume" title="volume">20</span> <span class="fpage" tagx="fpage" title="fpage">1043</span> <span class="lpage" tagx="lpage" title="lpage">1050</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1093/molbev/msg124">10.1093/molbev/msg124</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/12716977">12716977</a> </span> </span> </li> <li tag="ref"> <a name="CR36" /> <span class="label" tagx="label" title="label">36.</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation">Al-Shehbaz, I. A. &amp;amp; O’Kane, S. L. Taxonomy and Phylogeny of Arabidopsis (Brassicaceae). <i>The Arabidopsis Book</i>, e0001 doi:10.1199/tab.0001 (2002). </span> </li> <li tag="ref"> <a name="CR37" /> <span class="label" tagx="label" title="label">37.</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation">Schmickl, R., Jørgensen, M. H., Brysting, A. K. &amp;amp; Koch, M. A. The evolutionary history of the Arabidopsis lyrata complex: a hybrid in the amphi-Beringian area closes a large distribution gap and builds up a genetic barrier. <i>Bmc Evol Biol</i> <b>10</b>, 98, doi:10.1186/1471-2148-10-98 (2010). </span> </li> <li tag="ref"> <a name="CR38" /> <span class="label" tagx="label" title="label">38.</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation">Hohmann, N. <i>et al</i>. Taming the wild: resolving the gene pools of non-model Arabidopsis lineages. <i>Bmc Evol Biol</i> <b>14</b>, doi:10.1186/S12862-014-0224-X (2014). </span> </li> <li tag="ref"> <a name="CR39" /> <span class="label" tagx="label" title="label">39.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Clauss</span> <span class="given-names" tagx="given-names" title="given-names">MJ</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Koch</span> <span class="given-names" tagx="given-names" title="given-names">MA</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Poorly known relatives of Arabidopsis thaliana</span> <span class="source" tagx="source" title="source">Trends Plant Sci</span> <span class="year" tagx="year" title="year">2006</span> <span class="volume" tagx="volume" title="volume">11</span> <span class="fpage" tagx="fpage" title="fpage">449</span> <span class="lpage" tagx="lpage" title="lpage">459</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1016/j.tplants.2006.07.005">10.1016/j.tplants.2006.07.005</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/16893672">16893672</a> </span> </span> </li> <li tag="ref"> <a name="CR40" /> <span class="label" tagx="label" title="label">40.</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation">Briskine, R. V. <i>et al</i>. Genome assembly and annotation of Arabidopsis halleri, a model for heavy metal hyperaccumulation and evolutionary ecology. <i>Mol Ecol Resour</i>, doi:10.1111/1755-0998.12604 (2016). </span> </li> <li tag="ref"> <a name="CR41" /> <span class="label" tagx="label" title="label">41.</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation">Seol, Y. J. <i>et al</i>. The complete chloroplast genome of two Brassica species, Brassica nigra and B. Oleracea. <i>Mitochondr DNA</i>, 1–2, doi:10.3109/19401736.2015.1115493 (2015). </span> </li> <li tag="ref"> <a name="CR42" /> <span class="label" tagx="label" title="label">42.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Wu</span> <span class="given-names" tagx="given-names" title="given-names">Z</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">The complete chloroplast genome of Capsella rubella</span> <span class="source" tagx="source" title="source">Mitochondrial DNA. Part A, DNA mapping, sequencing, and analysis</span> <span class="year" tagx="year" title="year">2016</span> <span class="volume" tagx="volume" title="volume">27</span> <span class="fpage" tagx="fpage" title="fpage">2561</span> <span class="lpage" tagx="lpage" title="lpage">2562</span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/26024136">26024136</a> </span> </span> </li> <li tag="ref"> <a name="CR43" /> <span class="label" tagx="label" title="label">43.</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation">Xu, J. W. <i>et al</i>. The first intron of rice EPSP synthase enhances expression of foreign gene. Sci China Ser C <b>46</b>, 561–+, doi:10.1360/02yc0120 (2003). </span> </li> <li tag="ref"> <a name="CR44" /> <span class="label" tagx="label" title="label">44.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Wolf</span> <span class="given-names" tagx="given-names" title="given-names">PG</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span> </span> <span class="mixed-article-title" title="mixed-article-title">The evolution of chloroplast genes and genomes in ferns</span> <span class="source" tagx="source" title="source">Plant Mol Biol</span> <span class="year" tagx="year" title="year">2011</span> <span class="volume" tagx="volume" title="volume">76</span> <span class="fpage" tagx="fpage" title="fpage">251</span> <span class="lpage" tagx="lpage" title="lpage">261</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1007/s11103-010-9706-4">10.1007/s11103-010-9706-4</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/20976559">20976559</a> </span> </span> </li> <li tag="ref"> <a name="CR45" /> <span class="label" tagx="label" title="label">45.</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation">Oliver, M. J. <i>et al</i>. Chloroplast genome sequence of the moss Tortula ruralis: gene content, polymorphism, and structural arrangement relative to other green plant chloroplast genomes. <i>Bmc Genomics</i> <b>11</b>, doi:10.1186/1471-2164-11-143 (2010). </span> </li> <li tag="ref"> <a name="CR46" /> <span class="label" tagx="label" title="label">46.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Wicke</span> <span class="given-names" tagx="given-names" title="given-names">S</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Schneeweiss</span> <span class="given-names" tagx="given-names" title="given-names">GM</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">dePamphilis</span> <span class="given-names" tagx="given-names" title="given-names">CW</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Müller</span> <span class="given-names" tagx="given-names" title="given-names">KF</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Quandt</span> <span class="given-names" tagx="given-names" title="given-names">D</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">The evolution of the plastid chromosome in land plants: gene content, gene order, gene function </span> <span class="source" tagx="source" title="source">Plant Mol Biol</span> <span class="year" tagx="year" title="year">2011</span> <span class="volume" tagx="volume" title="volume">76</span> <span class="fpage" tagx="fpage" title="fpage">273</span> <span class="lpage" tagx="lpage" title="lpage">297</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1007/s11103-011-9762-4">10.1007/s11103-011-9762-4</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/21424877">21424877</a> </span> </span> </li> <li tag="ref"> <a name="CR47" /> <span class="label" tagx="label" title="label">47.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Jansen</span> <span class="given-names" tagx="given-names" title="given-names">RK</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Analysis of 81 genes from 64 plastid genomes resolves relationships in angiosperms and identifies genome-scale evolutionary patterns </span> <span class="source" tagx="source" title="source">Proceedings of the National Academy of Sciences</span> <span class="year" tagx="year" title="year">2007</span> <span class="volume" tagx="volume" title="volume">104</span> <span class="fpage" tagx="fpage" title="fpage">19369</span> <span class="lpage" tagx="lpage" title="lpage">19374</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1073/pnas.0709121104">10.1073/pnas.0709121104</a> </span> </span> </li> <li tag="ref"> <a name="CR48" /> <span class="label" tagx="label" title="label">48.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Nakkaew</span> <span class="given-names" tagx="given-names" title="given-names">A</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Chotigeat</span> <span class="given-names" tagx="given-names" title="given-names">W</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Eksomtramage</span> <span class="given-names" tagx="given-names" title="given-names">T</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Phongdara</span> <span class="given-names" tagx="given-names" title="given-names">A</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Cloning and expression of a plastid-encoded subunit, beta-carboxyltransferase gene (accD) and a nuclear-encoded subunit, biotin carboxylase of acetyl-CoA carboxylase from oil palm (Elaeis guineensis Jacq.) </span> <span class="source" tagx="source" title="source">Plant Sci</span> <span class="year" tagx="year" title="year">2008</span> <span class="volume" tagx="volume" title="volume">175</span> <span class="fpage" tagx="fpage" title="fpage">497</span> <span class="lpage" tagx="lpage" title="lpage">504</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1016/j.plantsci.2008.05.023">10.1016/j.plantsci.2008.05.023</a> </span> </span> </li> <li tag="ref"> <a name="CR49" /> <span class="label" tagx="label" title="label">49.</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation">Xu, Q. <i>et al</i>. Analysis of complete nucleotide sequences of 12 Gossypium chloroplast genomes: origin and evolution of allotetraploids. Plos One <b>7</b>, doi:10.1371/journal.pone.0037128 (2012). </span> </li> <li tag="ref"> <a name="CR50" /> <span class="label" tagx="label" title="label">50.</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation">Huang, H., Shi, C., Liu, Y., Mao, S. Y. &amp;amp; Gao, L. Z. Thirteen Camellia chloroplast genome sequences determined by high-throughput sequencing: genome structure and phylogenetic relationships. <i>Bmc Evol Biol</i> <b>14</b>, doi:10.1186/1471-2148-14-151 (2014). </span> </li> <li tag="ref"> <a name="CR51" /> <span class="label" tagx="label" title="label">51.</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation">Asaf, S. <i>et al</i>. Complete Chloroplast Genome of Nicotiana otophora and its Comparison with Related Species. Front Plant Sci <b>7</b>, doi:10.3389/fpls.2016.00843 (2016). </span> </li> <li tag="ref"> <a name="CR52" /> <span class="label" tagx="label" title="label">52.</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation">Chen, J. H. <i>et al</i>. The complete chloroplast genome sequence of the relict woody plant Metasequoia glyptostroboides Hu et Cheng. Front Plant Sci <b>6</b>, doi: 10.3389/Fpls.2015.00447 (2015). </span> </li> <li tag="ref"> <a name="CR53" /> <span class="label" tagx="label" title="label">53.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Cavalier-Smith</span> <span class="given-names" tagx="given-names" title="given-names">T</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Chloroplast evolution: Secondary symbiogenesis and multiple losses</span> <span class="source" tagx="source" title="source">Curr Biol</span> <span class="year" tagx="year" title="year">2002</span> <span class="volume" tagx="volume" title="volume">12</span> <span class="fpage" tagx="fpage" title="fpage">R62</span> <span class="lpage" tagx="lpage" title="lpage">R64</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1016/S0960-9822(01)00675-3">10.1016/S0960-9822(01)00675-3</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/11818081">11818081</a> </span> </span> </li> <li tag="ref"> <a name="CR54" /> <span class="label" tagx="label" title="label">54.</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation">Asaf, S. <i>et al</i>. The Complete Chloroplast Genome of Wild Rice (Oryza minuta) and Its Comparison to Related Species. <i>Front Plant Sci</i> <b>8</b>, doi:10.3389/fpls.2017.00304 (2017). </span> </li> <li tag="ref"> <a name="CR55" /> <span class="label" tagx="label" title="label">55.</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation">Nie, X. J. <i>et al</i>. Complete Chloroplast Genome Sequence of a Major Invasive Species, Crofton Weed (Ageratina adenophora). <i>Plos One</i> <b>7</b>, doi: 10.1371/journal.pone.0036869 (2012). </span> </li> <li tag="ref"> <a name="CR56" /> <span class="label" tagx="label" title="label">56.</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation">Saski, C. <i>et al</i>. Complete chloroplast genome sequences of Hordeum vulgare, Sorghum bicolor and Agrostis stolonifera, and comparative analyses with other grass genomes. <i>Theor Appl Genet</i> <b>115</b>, doi:10.1007/s00122-007-0567-4 (2007). </span> </li> <li tag="ref"> <a name="CR57" /> <span class="label" tagx="label" title="label">57.</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation">Tangphatsornruang, S. <i>et al</i>. The chloroplast genome sequence of mungbean (Vigna radiata) determined by high-throughput pyrosequencing: structural organization and phylogenetic relationships. <i>DNA Res</i>, dsp025 (2009). </span> </li> <li tag="ref"> <a name="CR58" /> <span class="label" tagx="label" title="label">58.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Timme</span> <span class="given-names" tagx="given-names" title="given-names">RE</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Kuehl</span> <span class="given-names" tagx="given-names" title="given-names">JV</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Boore</span> <span class="given-names" tagx="given-names" title="given-names">JL</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Jansen</span> <span class="given-names" tagx="given-names" title="given-names">RK</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">A comparative analysis of the Lactuca and Helianthus (Asteraceae) plastid genomes: Identification of divergent regions and categorization of shared repeats </span> <span class="source" tagx="source" title="source">Am J Bot</span> <span class="year" tagx="year" title="year">2007</span> <span class="volume" tagx="volume" title="volume">94</span> <span class="fpage" tagx="fpage" title="fpage">302</span> <span class="lpage" tagx="lpage" title="lpage">312</span> <span class="pub-id"> <a href="https://dx.doi.org/10.3732/ajb.94.3.302">10.3732/ajb.94.3.302</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/21636403">21636403</a> </span> </span> </li> <li tag="ref"> <a name="CR59" /> <span class="label" tagx="label" title="label">59.</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation">Gao, L., Yi, X., Yang, Y. X., Su, Y. J. &amp;amp; Wang, T. Complete chloroplast genome sequence of a tree fern Alsophila spinulosa: insights into evolutionary changes in fern chloroplast genomes. <i>Bmc Evol Biol</i> <b>9</b>, doi:10.1186/1471-2148-9-130 (2009). </span> </li> <li tag="ref"> <a name="CR60" /> <span class="label" tagx="label" title="label">60.</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation">Zhao, Y. B. <i>et al</i>. The complete chloroplast genome provides insight into the evolution and polymorphism of Panax ginseng. <i>Front Plant Sci</i> <b>5</b>, doi: 10.3389/Fpls.2014.00696 (2015). </span> </li> <li tag="ref"> <a name="CR61" /> <span class="label" tagx="label" title="label">61.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Rose</span> <span class="given-names" tagx="given-names" title="given-names">O</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Falush</span> <span class="given-names" tagx="given-names" title="given-names">D</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">A threshold size for microsatellite expansion</span> <span class="source" tagx="source" title="source">Mol Biol Evol</span> <span class="year" tagx="year" title="year">1998</span> <span class="volume" tagx="volume" title="volume">15</span> <span class="fpage" tagx="fpage" title="fpage">613</span> <span class="lpage" tagx="lpage" title="lpage">615</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1093/oxfordjournals.molbev.a025964">10.1093/oxfordjournals.molbev.a025964</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/9580993">9580993</a> </span> </span> </li> <li tag="ref"> <a name="CR62" /> <span class="label" tagx="label" title="label">62.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Huotari</span> <span class="given-names" tagx="given-names" title="given-names">T</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Korpelainen</span> <span class="given-names" tagx="given-names" title="given-names">H</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Complete chloroplast genome sequence of Elodea canadensis and comparative analyses with other monocot plastid genomes </span> <span class="source" tagx="source" title="source">Gene</span> <span class="year" tagx="year" title="year">2012</span> <span class="volume" tagx="volume" title="volume">508</span> <span class="fpage" tagx="fpage" title="fpage">96</span> <span class="lpage" tagx="lpage" title="lpage">105</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1016/j.gene.2012.07.020">10.1016/j.gene.2012.07.020</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/22841789">22841789</a> </span> </span> </li> <li tag="ref"> <a name="CR63" /> <span class="label" tagx="label" title="label">63.</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation">Qian, J. <i>et al</i>. The complete chloroplast genome sequence of the medicinal plant Salvia miltiorrhiza. <i>Plos One</i> <b>8</b>, doi:10.1371/journal.pone.0057607 (2013). </span> </li> <li tag="ref"> <a name="CR64" /> <span class="label" tagx="label" title="label">64.</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation">Zhang, Y. J., Ma, P. F. &amp;amp; Li, D. Z. High-throughput sequencing of six bamboo chloroplast genomes: phylogenetic implications for temperate woody bamboos (Poaceae: Bambusoideae). <i>Plos One</i> <b>6</b>, doi:10.1371/journal.pone.0020596 (2011). </span> </li> <li tag="ref"> <a name="CR65" /> <span class="label" tagx="label" title="label">65.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Yi</span> <span class="given-names" tagx="given-names" title="given-names">X</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Gao</span> <span class="given-names" tagx="given-names" title="given-names">L</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Wang</span> <span class="given-names" tagx="given-names" title="given-names">B</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Su</span> <span class="given-names" tagx="given-names" title="given-names">YJ</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Wang</span> <span class="given-names" tagx="given-names" title="given-names">T</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">The Complete Chloroplast Genome Sequence of Cephalotaxus oliveri (Cephalotaxaceae): Evolutionary Comparison of Cephalotaxus Chloroplast DNAs and Insights into the Loss of Inverted Repeat Copies in Gymnosperms </span> <span class="source" tagx="source" title="source">Genome Biol Evol</span> <span class="year" tagx="year" title="year">2013</span> <span class="volume" tagx="volume" title="volume">5</span> <span class="fpage" tagx="fpage" title="fpage">688</span> <span class="lpage" tagx="lpage" title="lpage">698</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1093/gbe/evt042">10.1093/gbe/evt042</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/23538991">23538991</a> </span> </span> </li> <li tag="ref"> <a name="CR66" /> <span class="label" tagx="label" title="label">66.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Kuang</span> <span class="given-names" tagx="given-names" title="given-names">DY</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Complete chloroplast genome sequence of Magnolia kwangsiensis (Magnoliaceae): implication for DNA barcoding and population genetics </span> <span class="source" tagx="source" title="source">Genome</span> <span class="year" tagx="year" title="year">2011</span> <span class="volume" tagx="volume" title="volume">54</span> <span class="fpage" tagx="fpage" title="fpage">663</span> <span class="lpage" tagx="lpage" title="lpage">673</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1139/g11-026">10.1139/g11-026</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/21793699">21793699</a> </span> </span> </li> <li tag="ref"> <a name="CR67" /> <span class="label" tagx="label" title="label">67.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Powell</span> <span class="given-names" tagx="given-names" title="given-names">W</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Morgante</span> <span class="given-names" tagx="given-names" title="given-names">M</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Mcdevitt</span> <span class="given-names" tagx="given-names" title="given-names">R</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Vendramin</span> <span class="given-names" tagx="given-names" title="given-names">GG</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Rafalski</span> <span class="given-names" tagx="given-names" title="given-names">JA</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Polymorphic Simple Sequence Repeat Regions in Chloroplast Genomes - Applications to the Population-Genetics of Pines </span> <span class="source" tagx="source" title="source">P Natl Acad Sci USA</span> <span class="year" tagx="year" title="year">1995</span> <span class="volume" tagx="volume" title="volume">92</span> <span class="fpage" tagx="fpage" title="fpage">7759</span> <span class="lpage" tagx="lpage" title="lpage">7763</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1073/pnas.92.17.7759">10.1073/pnas.92.17.7759</a> </span> </span> </li> <li tag="ref"> <a name="CR68" /> <span class="label" tagx="label" title="label">68.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Pauwels</span> <span class="given-names" tagx="given-names" title="given-names">M</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Nuclear and chloroplast DNA phylogeography reveals vicariance among European populations of the model species for the study of metal tolerance, Arabidopsis halleri (Brassicaceae) </span> <span class="source" tagx="source" title="source">New Phytol</span> <span class="year" tagx="year" title="year">2012</span> <span class="volume" tagx="volume" title="volume">193</span> <span class="fpage" tagx="fpage" title="fpage">916</span> <span class="lpage" tagx="lpage" title="lpage">928</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1111/j.1469-8137.2011.04003.x">10.1111/j.1469-8137.2011.04003.x</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/22225532">22225532</a> </span> </span> </li> <li tag="ref"> <a name="CR69" /> <span class="label" tagx="label" title="label">69.</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation">Nazareno, A. G., Carlsen, M. &amp;amp; Lohmann, L. G. Complete Chloroplast Genome of Tanaecium tetragonolobum: The First Bignoniaceae Plastome. <i>Plos One</i> <b>10</b>, doi:10.1371/journal.pone.0129930 (2015). </span> </li> <li tag="ref"> <a name="CR70" /> <span class="label" tagx="label" title="label">70.</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation">Hao, Z. D. <i>et al</i>. The Complete Chloroplast Genome Sequence of a Relict Conifer Glyptostrobus pensilis: Comparative Analysis and Insights into Dynamics of Chloroplast Genome Rearrangement in Cupressophytes and Pinaceae. <i>Plos One</i> <b>11</b>, doi:10.1371/journal.pone.0161809 (2016). </span> </li> <li tag="ref"> <a name="CR71" /> <span class="label" tagx="label" title="label">71.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Raubeson</span> <span class="given-names" tagx="given-names" title="given-names">LA</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Comparative chloroplast genomics: analyses including new sequences from the angiosperms Nuphar advena and Ranunculus macranthus </span> <span class="source" tagx="source" title="source">Bmc Genomics</span> <span class="year" tagx="year" title="year">2007</span> <span class="volume" tagx="volume" title="volume">8</span> <span class="fpage" tagx="fpage" title="fpage">174</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1186/1471-2164-8-174">10.1186/1471-2164-8-174</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/17573971">17573971</a> </span> </span> </li> <li tag="ref"> <a name="CR72" /> <span class="label" tagx="label" title="label">72.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Khakhlova</span> <span class="given-names" tagx="given-names" title="given-names">O</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Bock</span> <span class="given-names" tagx="given-names" title="given-names">R</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Elimination of deleterious mutations in plastid genomes by gene conversion</span> <span class="source" tagx="source" title="source">Plant J</span> <span class="year" tagx="year" title="year">2006</span> <span class="volume" tagx="volume" title="volume">46</span> <span class="fpage" tagx="fpage" title="fpage">85</span> <span class="lpage" tagx="lpage" title="lpage">94</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1111/j.1365-313X.2006.02673.x">10.1111/j.1365-313X.2006.02673.x</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/16553897">16553897</a> </span> </span> </li> <li tag="ref"> <a name="CR73" /> <span class="label" tagx="label" title="label">73.</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation">Yang, M. <i>et al</i>. The complete chloroplast genome sequence of date palm (Phoenix dactylifera L.). <i>Plos One</i> <b>5</b>, doi:10.1371/journal.pone.0012762 (2010). </span> </li> <li tag="ref"> <a name="CR74" /> <span class="label" tagx="label" title="label">74.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Kode</span> <span class="given-names" tagx="given-names" title="given-names">V</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Mudd</span> <span class="given-names" tagx="given-names" title="given-names">EA</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Iamtham</span> <span class="given-names" tagx="given-names" title="given-names">S</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Day</span> <span class="given-names" tagx="given-names" title="given-names">A</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">The tobacco plastid accD gene is essential and is required for leaf development</span> <span class="source" tagx="source" title="source">Plant J</span> <span class="year" tagx="year" title="year">2005</span> <span class="volume" tagx="volume" title="volume">44</span> <span class="fpage" tagx="fpage" title="fpage">237</span> <span class="lpage" tagx="lpage" title="lpage">244</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1111/j.1365-313X.2005.02533.x">10.1111/j.1365-313X.2005.02533.x</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/16212603">16212603</a> </span> </span> </li> <li tag="ref"> <a name="CR75" /> <span class="label" tagx="label" title="label">75.</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation">Yao, X. <i>et al</i>. The first complete chloroplast genome sequences in Actinidiaceae: genome structure and comparative analysis. Plos One <b>10</b>, doi:10.1371/journal.pone.0129347 (2015). </span> </li> <li tag="ref"> <a name="CR76" /> <span class="label" tagx="label" title="label">76.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Kikuchi</span> <span class="given-names" tagx="given-names" title="given-names">S</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Uncovering the Protein Translocon at the Chloroplast Inner Envelope Membrane</span> <span class="source" tagx="source" title="source">Science</span> <span class="year" tagx="year" title="year">2013</span> <span class="volume" tagx="volume" title="volume">339</span> <span class="fpage" tagx="fpage" title="fpage">571</span> <span class="lpage" tagx="lpage" title="lpage">574</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1126/science.1229262">10.1126/science.1229262</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/23372012">23372012</a> </span> </span> </li> <li tag="ref"> <a name="CR77" /> <span class="label" tagx="label" title="label">77.</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation">Dong, W. P. <i>et al</i>. ycf1, the most promising plastid DNA barcode of land plants. Sci Rep-Uk <b>5</b>, doi:10.1038/Srep08348 (2015). </span> </li> <li tag="ref"> <a name="CR78" /> <span class="label" tagx="label" title="label">78.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">De las Rivas</span> <span class="given-names" tagx="given-names" title="given-names">J</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Lozano</span> <span class="given-names" tagx="given-names" title="given-names">JJ</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Ortiz</span> <span class="given-names" tagx="given-names" title="given-names">AR</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Comparative analysis of chloroplast genomes: Functional annotation, genome-based phylogeny, and deduced evolutionary patterns </span> <span class="source" tagx="source" title="source">Genome Res</span> <span class="year" tagx="year" title="year">2002</span> <span class="volume" tagx="volume" title="volume">12</span> <span class="fpage" tagx="fpage" title="fpage">567</span> <span class="lpage" tagx="lpage" title="lpage">583</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1101/gr.209402">10.1101/gr.209402</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/11932241">11932241</a> </span> </span> </li> <li tag="ref"> <a name="CR79" /> <span class="label" tagx="label" title="label">79.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Neubig</span> <span class="given-names" tagx="given-names" title="given-names">KM</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Phylogenetic utility of ycf1 in orchids: a plastid gene more variable than matK</span> <span class="source" tagx="source" title="source">Plant Syst Evol</span> <span class="year" tagx="year" title="year">2009</span> <span class="volume" tagx="volume" title="volume">277</span> <span class="fpage" tagx="fpage" title="fpage">75</span> <span class="lpage" tagx="lpage" title="lpage">84</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1007/s00606-008-0105-0">10.1007/s00606-008-0105-0</a> </span> </span> </li> <li tag="ref"> <a name="CR80" /> <span class="label" tagx="label" title="label">80.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Gernandt</span> <span class="given-names" tagx="given-names" title="given-names">DS</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Phylogenetic Relationships of Pinus Subsection Ponderosae Inferred from Rapidly Evolving cpDNA Regions </span> <span class="source" tagx="source" title="source">Syst Bot</span> <span class="year" tagx="year" title="year">2009</span> <span class="volume" tagx="volume" title="volume">34</span> <span class="fpage" tagx="fpage" title="fpage">481</span> <span class="lpage" tagx="lpage" title="lpage">491</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1600/036364409789271290">10.1600/036364409789271290</a> </span> </span> </li> <li tag="ref"> <a name="CR81" /> <span class="label" tagx="label" title="label">81.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Drew</span> <span class="given-names" tagx="given-names" title="given-names">BT</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Sytsma</span> <span class="given-names" tagx="given-names" title="given-names">KJ</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Testing the Monophyly and Placement of Lepechinia in the Tribe Mentheae (Lamiaceae)</span> <span class="source" tagx="source" title="source">Syst Bot</span> <span class="year" tagx="year" title="year">2011</span> <span class="volume" tagx="volume" title="volume">36</span> <span class="fpage" tagx="fpage" title="fpage">1038</span> <span class="lpage" tagx="lpage" title="lpage">1049</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1600/036364411X605047">10.1600/036364411X605047</a> </span> </span> </li> <li tag="ref"> <a name="CR82" /> <span class="label" tagx="label" title="label">82.</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation">Dong, W. P., Liu, J., Yu, J., Wang, L. &amp;amp; Zhou, S. L. Highly Variable Chloroplast Markers for Evaluating Plant Phylogeny at Low Taxonomic Levels and for DNA Barcoding. <i>Plos One</i> <b>7</b>, doi:10.1371/journal.pone.0035071 (2012). </span> </li> <li tag="ref"> <a name="CR83" /> <span class="label" tagx="label" title="label">83.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Moore</span> <span class="given-names" tagx="given-names" title="given-names">MJ</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Soltis</span> <span class="given-names" tagx="given-names" title="given-names">PS</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Bell</span> <span class="given-names" tagx="given-names" title="given-names">CD</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Burleigh</span> <span class="given-names" tagx="given-names" title="given-names">JG</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Soltis</span> <span class="given-names" tagx="given-names" title="given-names">DE</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Phylogenetic analysis of 83 plastid genes further resolves the early diversification of eudicots </span> <span class="source" tagx="source" title="source">Proceedings of the National Academy of Sciences</span> <span class="year" tagx="year" title="year">2010</span> <span class="volume" tagx="volume" title="volume">107</span> <span class="fpage" tagx="fpage" title="fpage">4623</span> <span class="lpage" tagx="lpage" title="lpage">4628</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1073/pnas.0907801107">10.1073/pnas.0907801107</a> </span> </span> </li> <li tag="ref"> <a name="CR84" /> <span class="label" tagx="label" title="label">84.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Goremykin</span> <span class="given-names" tagx="given-names" title="given-names">VV</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Hirsch-Ernst</span> <span class="given-names" tagx="given-names" title="given-names">KI</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Wolfl</span> <span class="given-names" tagx="given-names" title="given-names">S</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Hellwig</span> <span class="given-names" tagx="given-names" title="given-names">FH</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">The chloroplast genome of Nymphaea alba: Whole-genome analyses and the problem of identifying the most basal angiosperm </span> <span class="source" tagx="source" title="source">Mol Biol Evol</span> <span class="year" tagx="year" title="year">2004</span> <span class="volume" tagx="volume" title="volume">21</span> <span class="fpage" tagx="fpage" title="fpage">1445</span> <span class="lpage" tagx="lpage" title="lpage">1454</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1093/molbev/msh147">10.1093/molbev/msh147</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/15084683">15084683</a> </span> </span> </li> <li tag="ref"> <a name="CR85" /> <span class="label" tagx="label" title="label">85.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">OKane</span> <span class="given-names" tagx="given-names" title="given-names">SL</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">AlShehbaz</span> <span class="given-names" tagx="given-names" title="given-names">IA</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">A synopsis of Arabidopsis (Brassicaceae)</span> <span class="source" tagx="source" title="source">Novon</span> <span class="year" tagx="year" title="year">1997</span> <span class="volume" tagx="volume" title="volume">7</span> <span class="fpage" tagx="fpage" title="fpage">323</span> <span class="lpage" tagx="lpage" title="lpage">327</span> <span class="pub-id"> <a href="https://dx.doi.org/10.2307/3391949">10.2307/3391949</a> </span> </span> </li> <li tag="ref"> <a name="CR86" /> <span class="label" tagx="label" title="label">86.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Koch</span> <span class="given-names" tagx="given-names" title="given-names">MA</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Wernisch</span> <span class="given-names" tagx="given-names" title="given-names">M</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Schmickl</span> <span class="given-names" tagx="given-names" title="given-names">R</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Arabidopsis thaliana’s wild relatives: an updated overview on systematics, taxonomy and evolution </span> <span class="source" tagx="source" title="source">Taxon</span> <span class="year" tagx="year" title="year">2008</span> <span class="volume" tagx="volume" title="volume">57</span> <span class="fpage" tagx="fpage" title="fpage">933</span> <span class="lpage" tagx="lpage" title="lpage">943</span> </span> </li> <li tag="ref"> <a name="CR87" /> <span class="label" tagx="label" title="label">87.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Koch</span> <span class="given-names" tagx="given-names" title="given-names">MA</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Matschinger</span> <span class="given-names" tagx="given-names" title="given-names">M</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Evolution and genetic differentiation among relatives of Arabidopsis thaliana</span> <span class="source" tagx="source" title="source">Proceedings of the National Academy of Sciences</span> <span class="year" tagx="year" title="year">2007</span> <span class="volume" tagx="volume" title="volume">104</span> <span class="fpage" tagx="fpage" title="fpage">6272</span> <span class="lpage" tagx="lpage" title="lpage">6277</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1073/pnas.0701338104">10.1073/pnas.0701338104</a> </span> </span> </li> <li tag="ref"> <a name="CR88" /> <span class="label" tagx="label" title="label">88.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Beck</span> <span class="given-names" tagx="given-names" title="given-names">JB</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Al-Shehbaz</span> <span class="given-names" tagx="given-names" title="given-names">IA</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">O’Kane</span> <span class="given-names" tagx="given-names" title="given-names">SL</span> <span class="suffix" tagx="suffix" title="suffix">Jr.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Schaal</span> <span class="given-names" tagx="given-names" title="given-names">BA</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Further insights into the phylogeny of Arabidopsis (Brassicaceae) from nuclear Atmyb2 flanking sequence </span> <span class="source" tagx="source" title="source">Mol Phylogenet Evol</span> <span class="year" tagx="year" title="year">2007</span> <span class="volume" tagx="volume" title="volume">42</span> <span class="fpage" tagx="fpage" title="fpage">122</span> <span class="lpage" tagx="lpage" title="lpage">130</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1016/j.ympev.2006.06.011">10.1016/j.ympev.2006.06.011</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/16908202">16908202</a> </span> </span> </li> <li tag="ref"> <a name="CR89" /> <span class="label" tagx="label" title="label">89.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Wyman</span> <span class="given-names" tagx="given-names" title="given-names">SK</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Jansen</span> <span class="given-names" tagx="given-names" title="given-names">RK</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Boore</span> <span class="given-names" tagx="given-names" title="given-names">JL</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Automatic annotation of organellar genomes with DOGMA</span> <span class="source" tagx="source" title="source">Bioinformatics</span> <span class="year" tagx="year" title="year">2004</span> <span class="volume" tagx="volume" title="volume">20</span> <span class="fpage" tagx="fpage" title="fpage">3252</span> <span class="lpage" tagx="lpage" title="lpage">3255</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1093/bioinformatics/bth352">10.1093/bioinformatics/bth352</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/15180927">15180927</a> </span> </span> </li> <li tag="ref"> <a name="CR90" /> <span class="label" tagx="label" title="label">90.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Schattner</span> <span class="given-names" tagx="given-names" title="given-names">P</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Brooks</span> <span class="given-names" tagx="given-names" title="given-names">AN</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Lowe</span> <span class="given-names" tagx="given-names" title="given-names">TM</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">The tRNAscan-SE, snoscan and snoGPS web servers for the detection of tRNAs and snoRNAs</span> <span class="source" tagx="source" title="source">Nucleic Acids Res</span> <span class="year" tagx="year" title="year">2005</span> <span class="volume" tagx="volume" title="volume">33</span> <span class="fpage" tagx="fpage" title="fpage">W686</span> <span class="lpage" tagx="lpage" title="lpage">W689</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1093/nar/gki366">10.1093/nar/gki366</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/15980563">15980563</a> </span> </span> </li> <li tag="ref"> <a name="CR91" /> <span class="label" tagx="label" title="label">91.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Lohse</span> <span class="given-names" tagx="given-names" title="given-names">M</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Drechsel</span> <span class="given-names" tagx="given-names" title="given-names">O</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Bock</span> <span class="given-names" tagx="given-names" title="given-names">R</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">OrganellarGenomeDRAW (OGDRAW): a tool for the easy generation of high-quality custom graphical maps of plastid and mitochondrial genomes </span> <span class="source" tagx="source" title="source">Curr Genet</span> <span class="year" tagx="year" title="year">2007</span> <span class="volume" tagx="volume" title="volume">52</span> <span class="fpage" tagx="fpage" title="fpage">267</span> <span class="lpage" tagx="lpage" title="lpage">274</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1007/s00294-007-0161-y">10.1007/s00294-007-0161-y</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/17957369">17957369</a> </span> </span> </li> <li tag="ref"> <a name="CR92" /> <span class="label" tagx="label" title="label">92.</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation">Kumar, S., Nei, M., Dudley, J. &amp;amp; Tamura, K. MEGA: a biologist-centric software for evolutionary analysis of DNA and protein sequences. <i>Brief Bioinform</i> <b>9</b>, 299–306, doi:10.1093/bib/bbn017 (2008). </span> </li> <li tag="ref"> <a name="CR93" /> <span class="label" tagx="label" title="label">93.</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation">Frazer, K. A., Pachter, L., Poliakov, A., Rubin, E. M. &amp;amp; Dubchak, I. VISTA: computational tools for comparative genomics. <i>Nucleic Acids Res</i> <b>32</b>, W273-W279, doi:10.1093/nar/gkh458 (2004). </span> </li> <li tag="ref"> <a name="CR94" /> <span class="label" tagx="label" title="label">94.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Kurtz</span> <span class="given-names" tagx="given-names" title="given-names">S</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span> </span> <span class="mixed-article-title" title="mixed-article-title">REPuter: the manifold applications of repeat analysis on a genomic scale</span> <span class="source" tagx="source" title="source">Nucleic Acids Res</span> <span class="year" tagx="year" title="year">2001</span> <span class="volume" tagx="volume" title="volume">29</span> <span class="fpage" tagx="fpage" title="fpage">4633</span> <span class="lpage" tagx="lpage" title="lpage">4642</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1093/nar/29.22.4633">10.1093/nar/29.22.4633</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/11713313">11713313</a> </span> </span> </li> <li tag="ref"> <a name="CR95" /> <span class="label" tagx="label" title="label">95.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Kraemer</span> <span class="given-names" tagx="given-names" title="given-names">L</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span> </span> <span class="mixed-article-title" title="mixed-article-title">STAMP: Extensions to the STADEN sequence analysis package for high throughput interactive microsatellite marker design </span> <span class="source" tagx="source" title="source">Bmc Bioinformatics</span> <span class="year" tagx="year" title="year">2009</span> <span class="volume" tagx="volume" title="volume">10</span> <span class="fpage" tagx="fpage" title="fpage">41</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1186/1471-2105-10-41">10.1186/1471-2105-10-41</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/19183437">19183437</a> </span> </span> </li> <li tag="ref"> <a name="CR96" /> <span class="label" tagx="label" title="label">96.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Benson</span> <span class="given-names" tagx="given-names" title="given-names">G</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Tandem repeats finder: a program to analyze DNA sequences</span> <span class="source" tagx="source" title="source">Nucleic Acids Res</span> <span class="year" tagx="year" title="year">1999</span> <span class="volume" tagx="volume" title="volume">27</span> <span class="fpage" tagx="fpage" title="fpage">573</span> <span class="lpage" tagx="lpage" title="lpage">580</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1093/nar/27.2.573">10.1093/nar/27.2.573</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/9862982">9862982</a> </span> </span> </li> <li tag="ref"> <a name="CR97" /> <span class="label" tagx="label" title="label">97.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Katoh</span> <span class="given-names" tagx="given-names" title="given-names">K</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Standley</span> <span class="given-names" tagx="given-names" title="given-names">DM</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">MAFFT Multiple Sequence Alignment Software Version 7: Improvements in Performance and Usability </span> <span class="source" tagx="source" title="source">Mol Biol Evol</span> <span class="year" tagx="year" title="year">2013</span> <span class="volume" tagx="volume" title="volume">30</span> <span class="fpage" tagx="fpage" title="fpage">772</span> <span class="lpage" tagx="lpage" title="lpage">780</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1093/molbev/mst010">10.1093/molbev/mst010</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/23329690">23329690</a> </span> </span> </li> <li tag="ref"> <a name="CR98" /> <span class="label" tagx="label" title="label">98.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Kimura</span> <span class="given-names" tagx="given-names" title="given-names">M</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">A simple method for estimating evolutionary rates of base substitutions through comparative studies of nucleotide sequences </span> <span class="source" tagx="source" title="source">J Mol Evol</span> <span class="year" tagx="year" title="year">1980</span> <span class="volume" tagx="volume" title="volume">16</span> <span class="fpage" tagx="fpage" title="fpage">111</span> <span class="lpage" tagx="lpage" title="lpage">120</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1007/BF01731581">10.1007/BF01731581</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/7463489">7463489</a> </span> </span> </li> <li tag="ref"> <a name="CR99" /> <span class="label" tagx="label" title="label">99.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Ronquist</span> <span class="given-names" tagx="given-names" title="given-names">F</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Huelsenbeck</span> <span class="given-names" tagx="given-names" title="given-names">JP</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">MrBayes 3: Bayesian phylogenetic inference under mixed models</span> <span class="source" tagx="source" title="source">Bioinformatics</span> <span class="year" tagx="year" title="year">2003</span> <span class="volume" tagx="volume" title="volume">19</span> <span class="fpage" tagx="fpage" title="fpage">1572</span> <span class="lpage" tagx="lpage" title="lpage">1574</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1093/bioinformatics/btg180">10.1093/bioinformatics/btg180</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/12912839">12912839</a> </span> </span> </li> <li tag="ref"> <a name="CR100" /> <span class="label" tagx="label" title="label">100.</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation">Swofford, D. L. PAUP*: Phylogenetic Analysis Using Parsimony (*and other methods), Version 4. (Sinauer Associates, 2002). </span> </li> <li tag="ref"> <a name="CR101" /> <span class="label" tagx="label" title="label">101.</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation">Wu, Z. Q., Tembrock, L. R. &amp;amp; Ge, S. Are Differences in Genomic Data Sets due to True Biological Variants or Errors in Genome Assembly: An Example from Two Chloroplast Genomes. <i>Plos One</i> <b>10</b>, doi:10.1371/journal.pone.0118019 (2015). </span> </li> <li tag="ref"> <a name="CR102" /> <span class="label" tagx="label" title="label">102.</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation">Darriba, D., Taboada, G. L., Doallo, R. &amp;amp; Posada, D. jModelTest 2: more models, new heuristics and parallel computing. <i>Nat Meth</i> <b>9</b>, 772–772, <a href="http://www.nature.com/nmeth/journal/v9/n8/abs/nmeth.2109.html#supplementary-information">http://www.nature.com/nmeth/journal/v9/n8/abs/nmeth.2109.html#supplementary-information</a> (2012). </span> </li> <li tag="ref"> <a name="CR103" /> <span class="label" tagx="label" title="label">103.</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation">Yao, X. <i>et al</i>. Chloroplast genome structure in Ilex (Aquifoliaceae). <i>Sci Rep-Uk</i> <b>6</b> (2016). </span> </li> </ul> </div> </div>  </body></html>