<html xmlns="http://www.w3.org/1999/xhtml"> <head> <style type="text/css">
      	     		 
      	    a                          {background : #ffffff; }
      	    article                    {border-style : dotted; border-width : 2px; }
		 	
		 	div                        {background : #ffffcc;}
		 	div.abstract-title         {font-weight : bold ; font-size : 16pt;}
		 	div.ack                    {border-style : solid ; border-color : red; margin : 2em; }
		 	div.article-meta           {border-style : solid; border-width : 1 pt; margin 1em;}
		 	div.boxed-text             {margin : 5em; border-style : solid;}
		 	div.contrib-group          {margin : 1em; }
		 	div.fig                    {border-style : solid; border-width : 2px; margin : 2em; }
		 	div.funding                {font-weight : bold ; font-size : 16pt;}
		 	div.given-names            {font-style : italic;}
		 	div.intro                  {border-style : inset; margin : 5px;}
		 	div.introduction           {border-style : inset; margin : 5px;}
		 	div.journal-meta           {border-style : solid; border-width : 1 pt; margin 1em;}
		 	div.journal-title-group    {background : #ffeeee;}
		 	div.article-title          {font-weight : bold ; font-size : 18pt;}
		 	div.kwd                    {font-style : italic}
		 	div.meta-name              {font-weight : bold ; font-size : 16pt;}
		 	div.name                   {font-weight : bold;}
		 	div.alt-title              {font-style : italic; font-size : 12px;}
		 	
		 	div.sec                    {border : 2px; margin 5px; padding 2px;}
		 	div.title                  {font-family : courier; font-weight : bold;
		 	                            font-size : 16pt; margin : 5px;}
		 	
		 	div.materials_methods      {border-style : double; margin : 5px; }
		 	div.methods                {border-style : double; margin : 5px; }
		 	
		 	div.results                {border-style : solid; margin : 5px;}
		 	div.background             {border-style : dotted; margin : 5px;}
		 	
		 	div.discussion             {border-style : groove; margin : 5px;}
		 	div.conclusion             {border-style : ridge; margin : 5px;}
		 	div.conclusions             {border-style : ridge; margin : 5px;}
		 	div.supplementary-material {border-style : inset; margin : 5px;}
		 	div.abbreviations          {border-style : double; border-color : red; margin : 5px;}
		 	div.competinginterests     {border-style : double; border-color : blue; margin : 5px;}
		 	div.acknowledgements       {border-style : double; border-color : green; margin : 5px;}
		 	div.authors_contributions   {border-style : double; border-color : purple; margin : 5px;}
		 	
		 	div.publisher              {border-style : outset; margin : 5px;}
		 	div.fn-type-conflict       {background : #f88; }
		 	div.fn-type-con            {background : #ddf; }
		 	div.fn-type-other          {background : #ddd; }
		 	
		 	div.unknown                {background : #ffd;
									 	  border-style : solid;
									 	  border-width : 1px;
									 	  padding : 2 px;}
		 	  
      	    table                      {background : #ffffdd;}
		 	tr                         {background : #ddddff; padding : 1px;}
		 	
		 	span                       {background : #ffcccc;}
		 	
		 	span.citation-author       {font-family : helvetica; background : #ffeeee;}
		 	span.collab                {background : #ddffff; }
		 	span.comment               {font-family : courier; font-size : 6px; background : #ffaaff;}
		 	span.contrib               {background : #ffffff;}
		 	span.corresp               {background : #ddffdd; }
		 	span.doi                   {background : #ffffff;}
		 	span.email                 {font-family : courier; }
		 	span.etal                  {font-style : italic;}
		 	span.fpage                 {font-family : courier;}
		 	span.given-names           {background : #ffffff;}
		 	span.iso-abbrev            {background : #ffffff;}
		 	span.issn-epub             {background : #ffffff;}
		 	span.issn-ppub             {background : #ffffff;}
		 	span.journal-title         {background : #ffffff;}
		 	span.lpage                 {font-family : courier;}
		 	span.mixed-article-title   {font-style : italic ;}
		 	span.nlm-ta                {background : #ffffff;}
		 	span.pmc                   {background : #ffffff;}
		 	span.pmcid                 {background : #ffffff;}
		 	span.pmid                  {background : #ffffff;}
		 	span.publisher             {background : #ffffff;}
		 	span.publisher-id          {background : #ffffff;}
		 	span.publisher-name        {background : #ffffff;}
		 	span.source                {background : #ffffff;}
		 	span.subject               {background : #ffffff;}
		 	span.surname               {background : #ffffff;}
		 	span.volume                {font-family : courier; font-weight : bold;}
		 	span.year                  {font-family : courier ; font-style : italic;}
			</style> </head> <body> <div class="front" title="front"> <div class="journal-meta" tagx="journal-meta" title="journal-meta"><span class="nlm-ta" title="nlm-ta">Nat Commun</span><span class="iso-abbrev" title="iso-abbrev">Nat Commun</span><div class="journal-title-group" tagx="journal-title-group" title="journal-title-group"><span class="journal-title" tagx="journal-title" title="journal-title">Nature Communications</span></div><span class="issn-epub" tagx="issn" title="issn-epub">2041-1723</span><div class="publisher" tagx="publisher" title="publisher"><span class="publisher-name" tagx="publisher-name" title="publisher-name">Nature Publishing Group</span></div> </div> <div class="article-meta" tagx="article-meta" title="article-meta"><span class="pii" title="pii">ncomms12758</span><span class="doi" title="doi"> doi: <a href="https://dx.doi.org/10.1038/ncomms12758">10.1038/ncomms12758</a></span><span class="pmid" title="pmid"> pmid: <a href="http://www.ncbi.nlm.nih.gov/pubmed/27585543">27585543</a></span><div class="article-categories" title="article-categories">
: <span class="subject" title="subject">Article</span></div> <div class="title-group" tagx="title-group" title="title-group"> <div class="article-title" title="article-title">
Morphological and molecular convergences in mammalian phylogenetics</div> </div> <div class="contrib-group" title="contrib-group"><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zou</span><span class="given-names" tagx="given-names" title="given-names">Zhengting</span></span><a href="#a1">1</a><span class="contrib-id" tagx="contrib-id" title="contrib-id">http://orcid.org/0000-0003-1716-5090</span></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zhang</span><span class="given-names" tagx="given-names" title="given-names">Jianzhi</span></span><a href="#c1">a</a><a href="#a2">2</a><span class="contrib-id" tagx="contrib-id" title="contrib-id">http://orcid.org/0000-0001-6141-1290</span></span><span class="citation_author_institution" id="a1">[1], <span class="institution" title="institution">Department of Computational Medicine and Bioinformatics, University of Michigan</span><span class="country" tagx="country" title="country">USA</span></span><span class="citation_author_institution" id="a2">[2], <span class="institution" title="institution">Department of Ecology and Evolutionary Biology, University of Michigan</span><span class="country" tagx="country" title="country">USA</span></span></div> <div class="author-notes" title="author-notes"> <div class="corresp" title="corresp"><span class="label" tagx="label" title="label">a</span><span class="email" tagx="email" title="email">jianzhi@umich.edu</span></div> </div><span class="pub-date-epub" title="pub-date-epub">epub: <span>2016-09-09</span></span><span class="pub-date-collection" title="pub-date-collection">collection: <span>2016</span></span><span class="volume" tagx="volume" title="volume">7</span><span class="elocation-id" tagx="elocation-id" title="elocation-id">12758</span><span class="history" title="history"><span class="received" title="received">received: 2016-03-02</span><span class="accepted" title="accepted">accepted: 2016-07-29</span></span><div class="permissions"><span class="copyright" title="copyright">(C) <span class="copyright-holder" tagx="copyright-holder" title="copyright-holder">The Author(s)</span>, <span class="copyright-year" tagx="copyright-year" title="copyright-year">2016</span></span><span class="license" title="license"><span class="license-p" title="license-p">This work is licensed under a Creative Commons Attribution 4.0 International License. The images or other third party material in this article are included in the article's Creative Commons license, unless indicated otherwise in the credit line; if the material is not included under the Creative Commons license, users will need to obtain permission from the license holder to reproduce the material. To view a copy of this license, visit <a href="http://creativecommons.org/licenses/by/4.0/">http://creativecommons.org/licenses/by/4.0/</a></span></span></div> <div class="abstract" title="abstract"> <div class="abstract-title" title="abstract-title">
Abstract</div> <p>Phylogenetic trees reconstructed from molecular sequences are often considered more reliable than those reconstructed from morphological characters, in part because convergent evolution, which confounds phylogenetic reconstruction, is believed to be rarer for molecular sequences than for morphologies. However, neither the validity of this belief nor its underlying cause is known. Here comparing thousands of characters of each type that have been used for inferring the phylogeny of mammals, we find that on average morphological characters indeed experience much more convergences than amino acid sites, but this disparity is explained by fewer states per character rather than an intrinsically higher susceptibility to convergence for morphologies than sequences. We show by computer simulation and actual data analysis that a simple method for identifying and removing convergence-prone characters improves phylogenetic accuracy, potentially enabling, when necessary, the inclusion of morphologies and hence fossils for reliable tree inference.</p> </div> <div class="abstract" title="abstract"> <p><span class="inline-graphic" title="inline-graphic" />Morphological characters are generally thought to have higher rates of convergence than molecular characters. Here, Zou and Zhang provide empirical evidence for this assumption and devise a method to improve the accuracy of phylogenetic reconstruction through identifying and removing convergence-prone characters.</p> </div> </div> </div> <div class="body" title="body"> <p>Having a reliable species tree is prerequisite for understanding evolution, which is necessary for making sense of virtually every biological phenomenon. Traditionally, species trees are inferred using morphological, physiological or behavioural characters, collectively called morphological characters hereinafter. The advent of molecular biology supplied numerous molecular characters in the form of DNA and protein sequences, which are often (although not universally) considered more suitable than morphological characters for phylogenetic inference<a href="#b1">1</a><a href="#b2">2</a><a href="#b3">3</a><a href="#b4">4</a><a href="#b5">5</a><a href="#b6">6</a>. A major reason of this consideration concerns convergence, which refers to repeated origins of the same character state in multiple evolutionary lineages and is a primary source of error in phylogenetic reconstruction. Compared with morphological characters, molecular characters are believed by many (but not all) to be less susceptible to convergence<a href="#b1">1</a><a href="#b3">3</a><a href="#b4">4</a><a href="#b5">5</a><a href="#b7">7</a><a href="#b8">8</a><a href="#b9">9</a><a href="#b10">10</a><a href="#b11">11</a><a href="#b12">12</a>. Nevertheless, this belief appears to arise in the early days of molecular systematics when morphological convergence had long been known while molecular convergence had not. Recent genetic and genomic studies, however, revealed a large number of convergences in protein sequence evolution<a href="#b13">13</a><a href="#b14">14</a><a href="#b15">15</a><a href="#b16">16</a><a href="#b17">17</a><a href="#b18">18</a><a href="#b19">19</a><a href="#b20">20</a><a href="#b21">21</a><a href="#b22">22</a><a href="#b23">23</a>, casting a doubt on the above belief. Determining whether morphological characters are more prone to convergence than molecular characters is important for several reasons. First, although morphological and molecular trees are often concordant with each other, this is not always the case<a href="#b2">2</a><a href="#b5">5</a><a href="#b6">6</a><a href="#b24">24</a><a href="#b25">25</a>. Knowledge of the relative prevalence of convergence in the two types of characters helps decide which tree is more trustable. Furthermore, it helps decide whether total evidence trees reconstructed jointly from the two types of characters<a href="#b5">5</a><a href="#b9">9</a><a href="#b10">10</a><a href="#b25">25</a><a href="#b26">26</a><a href="#b27">27</a><a href="#b28">28</a> are preferred over trees based on any one type. Second, phylogenetic analysis that includes fossils can help understand evolutionary relation, time and process for fossils as well as extant species<a href="#b2">2</a><a href="#b5">5</a><a href="#b9">9</a><a href="#b10">10</a><a href="#b12">12</a><a href="#b25">25</a><a href="#b26">26</a><a href="#b29">29</a>. Because molecular characters are inaccessible in the vast majority of fossils, knowing the frequency of morphological convergence is critical to assessing the reliability of phylogenies involving fossils. Third, convergence is caused by either repeated adaptations of different evolutionary lineages to similar environmental challenges or chance. Recent studies suggested that most molecular convergence events are attributable to chance<a href="#b18">18</a><a href="#b19">19</a><a href="#b30">30</a><a href="#b31">31</a>. A comparison between morphological and molecular characters may provide information about the relative roles of selection and drift in morphological evolution.</p> <p>Because not all morphological or molecular characters are employed by phylogeneticists, a fair comparison between the two character types in the context of phylogenetics should concentrate on characters used for phylogenetic reconstruction. To this end, we analysed a large data set including 3,414 parsimony informative morphological characters and 5,722 parsimony informative amino acid sites that was previously compiled for the inference of mammalian phylogeny of 46 extant and 40 fossil species<a href="#b25">25</a>. Our analysis focused on extant species because they have both types of characters. We found that morphological characters experience much more convergences than molecular characters. We devised a method to identify and remove convergence-prone characters, enabling the inclusion of morphologies and hence fossils for reliable tree inference.</p> <div class="results" title="sec"> <div class="title" tagx="title" title="title">
Results</div> <div class="whole-treeanalysis" title="sec"> <div class="title" tagx="title" title="title">
Whole-tree analysis</div> <p>Analysing character convergence requires a species tree. Because the mammalian tree is not completely resolved, we used three trees, respectively, reconstructed using the morphological characters, molecular characters, and both types of characters in the data set. Under each tree, we inferred the ancestral states at all interior nodes for each character. For every pair of independent branches, we identified characters that showed convergence (<a href="#f1">Fig. 1a</a>; see Methods) and compared the mean number of convergences per character between morphological and molecular characters. For example, under the morphological tree, the exterior branches, respectively, leading to wolf (<i>Canis lupus</i>) and aardvark (<i>Orycteropus afer</i>) form an independent branch pair (<a href="#S1">Supplementary Fig. 1a</a>), where 0.0072 convergences per morphological character was observed, significantly exceeding that (0.0038) per molecular character (<i>P</i>=0.03, Fisher's exact test; see Methods). Among 3,396 pairs of independent branches in the morphological tree, 79.1% exhibit a higher convergence per morphological character than that per molecular character (<a href="#f1">Fig. 1b</a>), significantly exceeding the chance expectation (<i>P</i>&amp;lt;1 × 10<sup>−4</sup>, bootstrap test; see Methods). There are 645 branch pairs with significantly higher per character morphological convergence than molecular convergence (<i>Q</i>-value &amp;lt;0.05, Fisher's exact test; blue dots in <a href="#f1">Fig. 1b</a>), whereas the opposite is true for only 61 branch pairs (orange dots in <a href="#f1">Fig. 1b</a>). The mean number of convergence per morphological character is 1.7 times that per molecular character.</p> <p>It was proposed that convergence is more fairly compared among characters or branch pairs by the ratio between the number of <i>c</i>on<i>v</i>ergence and that of <i>d</i>i<i>v</i>ergence (<i>Cv</i>/<i>Dv</i>; <a href="#f1">Fig. 1a</a>)<a href="#b15">15</a><a href="#b30">30</a> because both <i>Cv</i> and <i>Dv</i> increase with the amount of evolution. Hence, we identified divergence events for each branch pair (see Methods) and then calculated the total number of convergence events relative to the total number of divergence events for the branch pair for each type of characters. We found that morphological characters exhibit overwhelmingly larger <i>Cv</i>/<i>Dv</i>, compared with molecular characters (<a href="#f1">Fig. 1c</a>). The mean <i>Cv</i>/<i>Dv</i> ratio of morphological characters is 4.0 times that of molecular characters.</p> <p>If the morphological tree used differs from the unknown true tree, inferring convergence under the morphological tree underestimates morphological convergence and hence the conclusion of a higher convergence for morphological characters than molecular characters should be conservative. As expected, when the above analyses were repeated under the molecular tree (<a href="#S1">Supplementary Fig. 1b</a>) or the total evidence tree (<a href="#S1">Supplementary Fig. 1c</a>), we found even higher convergences (<a href="#f1">Fig. 1d</a>; <a href="#S1">Supplementary Fig. 2a</a>) and <i>Cv</i>/<i>Dv</i> ratios (<a href="#f1">Fig. 1e</a>; <a href="#S1">Supplementary Fig. 2b</a>) for morphological characters than for molecular characters. Similar results were obtained using conventional measures of homoplasy such as the consistency index (<i>ci</i>) and rescaled consistency index (<i>rc</i>). That is, regardless of the tree topology used, morphological characters show lower <i>ci</i> and <i>rc</i>, thus higher homoplasy, than molecular characters (<a href="#S1">Supplementary Fig. 3</a>).</p> <p>DNA sequences instead of amino acid sequences are sometimes used as molecular characters in phylogenetics. We, therefore, also conducted a whole-tree analysis of the 19,227 parsimony informative nucleotide sites in the data set, with the tree inferred from the nucleotide sequences as the molecular tree. Regardless of whether the morphological or molecular tree is used, we observed higher convergence per character and higher <i>Cv/Dv</i> ratio for morphological characters than nucleotide sites (<a href="#S1">Supplementary Fig. 4a–d</a>).</p> </div> <div class="quartetanalysis" title="sec"> <div class="title" tagx="title" title="title">
Quartet analysis</div> <p>Because the true mammalian tree is unknown, to ensure a fair comparison between morphological and molecular characters, we further examined every four species in the data that show the same phylogenetic relationship in the morphological and molecular trees, which we refer to as quartets (<a href="#f2">Fig. 2a</a>). Given a quartet and their phylogenetic relationship, a parsimony-informative character is said to be convergent if at least two changes are required to explain the observed states (<a href="#f2">Fig. 2a</a>). We identified all convergence events for each quartet. Averaged across 7,146 quartets that can be examined, we observed 0.026 convergences per morphological character, which is three times that per molecular character (0.0085). Higher morphological convergence than molecular convergence is found in 93.9% of quartets (<a href="#f2">Fig. 2b</a>), significantly exceeding the chance expectation (<i>P</i>&amp;lt;1 × 10<sup>−4</sup>, bootstrap test). A total of 6,087 quartets show significantly higher per character morphological convergence than molecular convergence (<i>Q</i>-value &amp;lt;0.05), while only 104 quartets show the opposite (<a href="#f2">Fig. 2b</a>).</p> <p>Given a quartet and their phylogenetic relationship, a parsimony-informative character is said to be consistent when only one change is needed to explain the observed states. Convergence provides an erroneous phylogenetic signal for the quartet, whereas consistency offers the correct signal. We thus computed, for each quartet, the ratio between the total number of <i>c</i>on<i>v</i>ergences and that of <i>c</i>on<i>s</i>istencies (<i>Cv</i>/<i>Cs</i> ratio) for each type of characters, which may be viewed as the noise/signal ratio. Again, morphological characters tend to have higher <i>Cv</i>/<i>Cs</i> ratios than molecular characters (<a href="#f2">Fig. 2c</a>). The above results also hold when nucleotide sites instead of amino acid sites are used as molecular characters (<a href="#S1">Supplementary Fig. 4e,f</a>).</p> </div> <div class="numberofstatespercharacter" title="sec"> <div class="title" tagx="title" title="title">
Number of states per character</div> <p>We found that 75.2% of parsimony-informative morphological characters are binary (<a href="#f3">Fig. 3a</a>). Because binary characters can only have one kind of change given an ancestral state, it is obvious that they are susceptible to convergence once multiple changes occur. By contrast, only a small fraction (12.4%) of molecular characters are binary (<a href="#f3">Fig. 3a</a>). The median number of states is five for molecular characters, significantly higher than that (two) for morphological characters (<i>P</i>&amp;lt;10<sup>−300</sup>, Mann–Whitney <i>U</i>-test).</p> <p>The probability of convergence relative to that of divergence for a character is expected to decrease with the number of states. Indeed, the <i>Cv</i>/<i>Dv</i> ratio decreases with the number of states for both types of characters (<a href="#f3">Fig. 3b</a>; <a href="#S1">Supplementary Fig. 5</a>) and this trend remains after the control of evolutionary rate (<a href="#S1">Supplementary Table 1</a>). We estimated that the <i>Cv/Dv</i> ratio of an average morphological character is 0.89 times that of a molecular character with the same number of states, and the corresponding number is 0.55 for <i>Cv/Cs</i> (see Methods). These results indicate that, compared with molecular characters, the higher convergence of morphological characters is caused by having fewer states rather than intrinsically higher susceptibilities to adaptive convergent evolution, because morphological characters are no more prone to convergence than molecular characters once the number of states is controlled for.</p> <p>The above patterns remain unchanged even when nucleotide sites instead of amino acid sites are used as molecular characters (<a href="#S1">Supplementary Table 2</a>). Interestingly, although there can be no more than four states at each nucleotide site, the median number of states (three) per nucleotide site is still significantly higher than that (two) per morphological character (<i>P</i>&amp;lt;10<sup>−300</sup>).</p> </div> <div class="removingconvergence-pronecharactersimprovesphylogenetics" title="sec"> <div class="title" tagx="title" title="title">
Removing convergence-prone characters improves phylogenetics</div> <p>Because the vast majority of molecular convergences are explainable by chance<a href="#b18">18</a><a href="#b19">19</a><a href="#b30">30</a><a href="#b31">31</a>, the fact that average morphological characters have even smaller <i>Cv</i>/<i>Dv</i> and <i>Cv</i>/<i>Cs</i> ratios than those of molecular characters of the same numbers of states suggest that most morphological convergences observed in the data analysed are probably also attributable to chance. If convergence is owing to chance rather than lineage-specific selection, it is possible to identify and remove convergence-prone characters using species with reliable phylogenetic relations and then infer the tree for species of uncertain relations using the remaining characters. This approach would be especially beneficial to phylogenetic inference that includes morphological data because of the relatively frequent convergence in such data. We propose the following procedure when analysing a data set with both morphological and molecular characters. First, we infer the morphological and molecular trees separately. Second, quartets (that is, groups of four species with the same phylogenetic relations in the two trees) are identified and the <i>Cv</i>/<i>Cs</i> ratio is calculated based on these quartets for each character. Third, we remove all characters whose <i>Cv</i>/<i>Cs</i> ratio exceeds a cutoff and infer the tree using all remaining morphological and molecular characters combined.</p> <p>To investigate whether the above approach improves phylogenetic accuracy, we conducted 50 simulations of mammalian morphological and molecular characters based on their respective empirical distributions of the number of states (<a href="#S1">Supplementary Fig. 6a</a>). Quartet analysis demonstrates that the simulated data have similar properties as the real data (<a href="#S1">Supplementary Fig. 6b,c</a>; <a href="#S1">Supplementary Table 3</a>). We measured the Robinson-Foulds distance (<i>d</i><sub>RF</sub>) between an inferred tree and the known true tree in simulation; <i>d</i><sub>RF</sub> is twice the fraction of branch partitions that differ between the two trees<a href="#b32">32</a>; the smaller the <i>d</i><sub>RF</sub>, the more accurate the inferred tree. We found that <i>d</i><sub>RF</sub> is significantly greater for the 50 morphological trees than the 50 molecular trees (<i>P</i>=1.6 × 10<sup>−14</sup>, Mann–Whitney <i>U</i>-test), confirming the damage of random convergence on phylogenetic accuracy. We set 10 <i>Cv</i>/<i>Cs</i> cutoffs from 5 to 0.03 and inferred ten low-convergence total evidence trees for each simulated data set using the above proposed procedure (see Methods). We found that <i>d</i><sub>RF</sub> to the true tree is generally smaller for low-convergence trees than the original tree reconstructed using all characters (green symbols in <a href="#f4">Fig. 4a</a>), and the improvement in phylogenetic accuracy plateaus when the cutoff reaches 0.3. By contrast, trees based on a random removal of the same number of characters do not show smaller <i>d</i><sub>RF</sub> when compared with the original tree (pink symbols in <a href="#f4">Fig. 4a</a>). As expected, the mean number of states is higher for the remaining low-convergence characters than for the same number of characters randomly picked from the original simulated data (<a href="#S1">Supplementary Fig. 6d</a>).</p> </div> <div class="removingconvergence-pronecharactersaltersthebattree" title="sec"> <div class="title" tagx="title" title="title">
Removing convergence-prone characters alters the bat tree</div> <p>We applied the above pipeline to the mammalian data set including both morphological characters and amino acid sequences. The same 10 <i>Cv</i>/<i>Cs</i> ratio cutoffs as in the simulation were used in removing high-convergence characters, and low-convergence total evidence trees of all 86 extant and fossil species were inferred using the remaining morphological and molecular characters. For the 46 extant species that can be compared, the resultant low-convergence trees are generally more similar than trees based on the same numbers of randomly selected characters to the original molecular tree (<a href="#S1">Supplementary Fig. 7</a>). The low-convergence trees are also generally more different than trees based on the same numbers of randomly selected characters from the original morphological tree (<a href="#S1">Supplementary Fig. 7</a>). Although the true mammalian tree is unknown, these observations are consistent with our finding that convergence is less frequent in molecular characters than morphological characters.</p> <p>Regarding intra-order relationships, the phylogeny of bats has been highly controversial. Specifically, all echolocating bats typically form a monophyletic group in morphological trees, suggesting a single origin of bat echolocation<a href="#b25">25</a>. But they tend to form a paraphyly in molecular trees<a href="#b33">33</a><a href="#b34">34</a><a href="#b35">35</a><a href="#b36">36</a>, suggesting the possibility of two origins of bat echolocation or one origin followed by a loss. In the original total evidence tree (<a href="#S1">Supplementary Fig. 8a</a>) reconstructed using the data analysed here, all five extant species of echolocating bats form a monophyly to the exclusion of the only non-echolocating extant bat <i>Pteropus giganteus</i>, with a 99.2% bootstrap support (<a href="#f4">Fig. 4b</a>). When the 3,930 characters (1,007 morphological and 2,923 molecular) with <i>Cv</i>/<i>Cs</i> ratio &amp;lt;0.2 are used after the removal of 2,407 morphological characters and 2,799 molecular characters (<a href="#S1">Supplementary Fig. 8b</a>), echolocating bats become paraphyletic; the echolocating <i>Rhinopoma hardwickii</i> and non-echolocating <i>P. giganteus</i> are grouped with a 95.0% bootstrap support (<a href="#f4">Fig. 4c</a>). Note that using low-convergence morphological characters alone does not result in this new topology. For comparison, we generated 50 randomly subsampled data sets, each with 1,007 morphological and 2,923 molecular characters. Although 18 of them also yielded the same topology as in <a href="#f4">Fig. 4c</a>, the corresponding bootstrap support ranged between 18 and 70%, suggesting that the strong support for the paraphyly of echolocators in <a href="#f4">Fig. 4c</a> is not explained simply by subsampling of the original data. Our results are not sensitive to the <i>Cv</i>/<i>Cs</i> ratio cutoff, because the same bat relationships were recovered when any <i>Cv</i>/<i>Cs</i> cutoff of 0.3 or smaller was used.</p> </div> </div> <div class="discussion" title="sec"> <div class="title" tagx="title" title="title">
Discussion</div> <p>Our analysis of comparably large numbers of morphological and molecular characters previously used in inferring the mammalian tree showed that morphological characters experienced more convergent evolution than molecular characters, confirming a long-held belief of the phylogenetics community. Nevertheless, we caution that our conclusion should be further scrutinized using additional data from additional groups of species, because they are currently based on only one, albeit very large, data set of one group of species. There are three potential sources of error in our inference of convergence. First, use of a wrong species tree could bias our inference. But, as demonstrated, our results are robust to different species trees used. Second, our inference of convergence relies on ancestral state reconstruction by parsimony that may contain errors<a href="#b37">37</a>. But, such errors should be comparable between the two types of characters. Third, it was recently proposed that some inferred convergences may be caused by incomplete lineage sorting rather than genuine convergent changes<a href="#b38">38</a>. Similar to genuine convergence, apparent convergence owing to incomplete lineage sorting also confounds phylogenetic inference and thus need not be separated from our estimates of convergence. Hence, the three potential errors do not affect our conclusion.</p> <p>Regarding the reason behind the higher convergence of morphological characters than molecular characters, our results do not support the common view that morphological characters are intrinsically more prone to convergence because they are more frequently subject to positive selection. Instead, we found the probability of convergence for a character to decrease with the number of states and found no greater intrinsic propensities for convergence (as measured by <i>Cv</i>/<i>Dv</i> and <i>Cv</i>/<i>Cs</i> ratios) among morphological characters than molecular characters after the control of the number of states. A likely explanation for this unexpected finding is that phylogeneticists have removed morphological characters that are subject to frequent positive selection (for example, body size and coat colour) from phylogenetic analysis, because such characters are known to lack reliable phylogenetic signals<a href="#b39">39</a>. As a result, the morphological characters used for phylogenetic inference have relatively low intrinsic propensities for convergence. If most convergences of the morphological characters in the data analysed are not manifestations of repeated adaptations but pure chance, one wonders what morphological characters are responsible for the clustering of species with seemingly adaptive convergences in the morphological tree, such as the clade of the four ant- and termite-eaters: the nine-banded armadillo <i>Dasypus novemcinctus</i>, collared anteater <i>Tamandua tetradactyla</i>, Chinese pangolin <i>Manis pentadactyla</i>, and aardvark <i>Orycteropus afer</i> (<a href="#S1">Supplementary Fig. 1a</a>). These species form three independent lineages (<i>Dasypus</i> + <i>Tamandua</i>, <i>Manis</i>, and <i>Orycteropus</i>) in the molecular tree (<a href="#S1">Supplementary Fig. 1b</a>) as well as the total evidence tree (<a href="#S1">Supplementary Fig. 1c</a>). We found that, even on the basis of the molecular tree, at most 14 morphological characters are inferred to have experienced convergence among the three lineages, and the actual number is likely much smaller because, for 13 of the 14 characters, convergence is but one of several equally parsimonious evolutionary scenarios. However, none of the 14 characters are apparently related to ant- and termite-eating or are specific to these four species. For instance, the only character for which the sole parsimonious reconstruction indicates convergence among the three lineages describes the shape of the medial border of humerus trochlea. The humerus is a long bone in the arm or forelimb that runs from the shoulder to the elbow and trochlea refers to a grooved structure reminiscent of a pulley's wheel. This character does not appear to be related to ant- and termite-eating. In fact, manatee (<i>Trichechus manatus</i>) and ring-tailed lemur (<i>Lemur catta</i>) also have the same state as the four ant- and termite-eating mammals for this character. These findings are consistent with our conclusion that most morphological convergences observed here are caused by chance rather than repeated adaptations. Of course, we cannot exclude the possibility that a small number of morphological convergences observed in this data set are adaptive.</p> <p>Nevertheless, morphological characters experience more convergences than molecular characters, because of much fewer states in the former than the latter. The low number of states per morphological character may be related to one or both of the following reasons<a href="#b7">7</a><a href="#b10">10</a>. First, curating multistate morphological characters may be more subjective and error-prone, resulting in a reduced use of such characters in phylogenetics<a href="#b40">40</a>. Second, most morphological characters may have a small state space, rendering finding multistate characters difficult<a href="#b41">41</a>.</p> <p>Because of the higher prevalence of convergence among morphological characters than molecular characters and the rapid accumulation of molecular sequence data, we suggest that phylogenetic reconstruction should normally use only molecular data. In the event that molecular data are inaccessible for some taxa such as fossils, one should consider using morphological characters with relatively large numbers of states to minimize convergence in phylogenetic analysis.</p> <p>Given a data set of morphological and molecular characters, we proposed a method to reconstruct more accurate total evidence trees by identifying and removing convergence-prone characters in the data set, and demonstrated its validity by computer simulation. Homoplasy, which interferes with phylogenetic inference, also includes reversal in addition to convergence. While our study focuses on convergence, it is worth noting that convergence-prone characters are also expected to be reversal-prone if most convergences are chance events owing to the availability of only few states, as indicated by the present data. Thus, in removing convergence-prone characters, we effectively also take out many reversal-prone characters; the success of our method may be in part attributable to this effect. Because our method relies on the assumption that characters that are convergence-prone in the quartets analysed are also convergence-prone in other species, it is not effective in removing characters that are convergence-prone in a few specific lineages such as those subject to adaptive convergence. In principle, one could also downweight instead of removing convergence-prone characters, but the appropriate weights are unknown. Future studies can investigate how to acquire the best weights for improving phylogenetic accuracy.</p> <p>We showed that the original total evidence mammalian tree in which all echolocating bats form a monophyly is altered upon the removal of convergence-prone characters. The low-convergence tree shows a paraphyly of echolocating bats, identical to the recently published genome-based bat phylogeny<a href="#b34">34</a>. Assuming that the genome-based tree is correct, our results demonstrated the utility of our method in actual phylogenetic inference with the total evidence approach. Besides, our low-convergence tree also supports the monophyly of pangolin (<i>Manis pentadactyla</i>) and carnivores (<a href="#S1">Supplementary Fig. 8b</a>), which is not reflected in the original total evidence tree (<a href="#S1">Supplementary Fig. 8a</a>) but is supported by previous molecular studies<a href="#b33">33</a><a href="#b42">42</a>. As shown by our computer simulation, although removing convergence-prone characters improves phylogenetic accuracy, low-convergence trees may still contain errors. Identifying and removing convergence-prone characters is by no means a panacea for phylogenetics. While rapidly accumulating genome sequences will eventually dwarf the morphological data of any extant species, morphological data will remain useful in phylogenetic analysis that needs to contain fossils, whose value to understanding evolution is indispensable. For this reason, understanding and remedying convergence, which is more prevalent in morphological than molecular characters, will remain an important task in phylogenetics. Of course, morphological characters that can be studied in fossils do not represent a random sample of all morphological characters. Whether this nonrandomness will bias phylogenetic inference<a href="#b43">43</a> is also worth investigation.</p> </div> <div class="methods" title="sec"> <div class="title" tagx="title" title="title">
Methods</div> <div class="datasetused" title="sec"> <div class="title" tagx="title" title="title">
Data set used</div> <p>The original data set is composed of 4,541 morphological characters and 11,365 amino acid sites<a href="#b25">25</a>. It includes 86 species, with 40 fossil taxa having only morphological characters and 46 extant species having both types of characters. We focused on extant species in this study because they have both types of characters for comparison. The morphological tree, molecular tree, and total evidence tree (that is, based on both types of characters) built using the parsimony method were provided by the original study (see <a href="#S1">Supplementary Fig. 1</a>). We removed all parsimony-uninformative characters for the 46 extant species. A parsimony-informative character has at least two states, each represented by at least two taxa. Parsimony trees of the 46 extant species based on the remaining 3,414 morphological characters or 5,722 amino acid sites agree with those based on all characters of the same types.</p> </div> <div class="whole-treeanalysis" title="sec"> <div class="title" tagx="title" title="title">
Whole-tree analysis</div> <p>Ancestral states of each parsimony-informative character were inferred for all interior nodes in the morphological tree, molecular tree or total evidence tree by parsimony using Mesquite (V. 3.03) (<a href="http://mesquiteproject.org/">http://mesquiteproject.org/</a>). Equal weights were given to equally parsimonious pathways in counting convergence events, such that if only one of <i>n</i> equally parsimonious pathways for a character shows convergence for a branch pair, 1/<i>n</i> convergence events are counted. Missing extant states of a character were inferred simultaneously during the inference by parsimony, such that no additional changes are required due to the missing state assignment. Mesquite also output the number of states appearing in the 46 extant species for each character and the number of changes each character experienced along the entire tree.</p> <p>An independent branch pair refers to two branches that are not ancestral to each other and contain no common node. For example, let the starting and end states of one branch (node 1 to node 3) be X<sub>1</sub> and X<sub>3</sub>, and let those of another branch (node 2 to node 4) be X<sub>2</sub> and X<sub>4</sub>, respectively. These two branches form an independent branch pair if (i) the four nodes are all distinct from one another, (ii) node 3 is not on the path from the tree root to node 4 and (iii) node 4 is not on the path from the tree root to node 3. For an independent pair of branches, there is a convergence if and only if X<sub>1</sub>≠X<sub>3</sub>, X<sub>2</sub>≠X<sub>4</sub>, and X<sub>3</sub>=X<sub>4</sub>. This definition includes both parallel and convergent changes previously defined<a href="#b13">13</a>. Similarly, there is a divergence in the independent branch pair if and only if X<sub>1</sub>≠X<sub>3</sub>, X<sub>2</sub>≠X<sub>4</sub>, and X<sub>3</sub>≠X<sub>4</sub>. Thus, once ancestral states are inferred, we know whether a character experiences convergence, divergence, or neither for a branch pair. For a character, the consistency index (<i>ci</i>) is the smallest minimal number of changes required to explain the observed states by any tree (Min) divided by the minimal number of changes required by the tree under evaluation (Obs). Retention index (<i>ri</i>)=(Max−Obs)/(Max−Min), where Max is the largest minimal number of changes required to explain the observed states by any tree. Rescaled consistency index (<i>rc</i>) equals consistency index multiplied by retention index<a href="#b44">44</a>. Values of <i>ri</i> and <i>ci</i> were calculated by Mesquite.</p> <p>We used Fisher's exact test to compare the number of convergences per character or <i>Cv</i>/<i>Dv</i> ratio between morphological and molecular characters. For example, in the branch pair leading to wolf and aardvark, we inferred 24.67 convergences among 3,414 morphological characters and 21.88 convergences among 5,722 molecular characters. We rounded the decimal number of convergence to the nearest integer and tested the null hypothesis that the probability of experiencing convergence is the same for the two types of characters using the following 2 × 2 contingency table: 25, 3414-25, 22 and 5722-22. We obtained a two-tailed <i>P</i> value of 0.0332 using Fisher's exact test, indicating that the frequency of convergence is significantly higher for morphologies than for sequences for the branch pair. For the same branch pair, we inferred 9.84 and 88.57 divergence events for morphological and molecular characters, respectively. We tested the null hypothesis that <i>Cv</i>/<i>Dv</i> ratio is the same for the two types of characters using the contingency table of 25, 10, 22 and 89. The obtained two-tailed <i>P</i> value from Fisher's exact test is 3.9 × 10<sup>−8</sup>, indicating that <i>Cv</i>/<i>Dv</i> ratio is significantly higher for morphological characters than for molecular characters for this branch pair. There were two branch pairs with no convergence and no divergence for molecular characters under the morphological tree, and three such branch pairs under either the molecular tree or the total evidence tree. These branch pairs had undefined <i>Cv/Dv</i> ratios for molecular characters and could not be tested in Fisher's exact test. Hence, they were excluded from the analysis and corresponding figures.</p> <p>Because branch pairs (or quartets) are not independent from one another, simple parametric statistic tests cannot be used. We thus used a bootstrap method to test the null hypothesis that per character convergence is lower for morphological characters than molecular characters. First, we generated one bootstrap sample containing the same number of both morphological and molecular characters as in the original data. Second, we analysed all branch pairs using the bootstrap sample and examined if &amp;gt;50% branch pairs show a lower morphological convergence than molecular convergence. We repeated the above two steps 10,000 times and computed the fraction of bootstrap samples in which &amp;gt;50% branch pairs show a lower morphological convergence than molecular convergence. This fraction is an estimate of the probability that the null hypothesis is correct, hence is the <i>P</i> value of this bootstrap test. The same bootstrap method was used to test the null hypothesis that <i>Cv</i>/<i>Dv</i> ratio and <i>Cv/Cs</i> ratio is lower for morphological characters than molecular characters in respective analyses.</p> </div> <div class="quartetanalysis" title="sec"> <div class="title" tagx="title" title="title">
Quartet analysis</div> <p>Four extant taxa Y<sub>1</sub>, Y<sub>2</sub>, Y<sub>3</sub> and Y<sub>4</sub> are selected if they satisfy the following conditions: (i) Y<sub>1</sub> and Y<sub>2</sub> form a monophyletic group in exclusion of Y<sub>3</sub> and Y<sub>4</sub> in both the morphological and molecular trees of all extant taxa examined; (ii) Y<sub>3</sub> and Y<sub>4</sub> form a monophyletic group in exclusion of Y<sub>1</sub> and Y<sub>2</sub> in both the morphological and molecular trees; and (iii) the root of this four-species tree is located on the internal branch in both the morphological and molecular trees. Mapping a parsimony-informative character onto this quartet tree, we say that the character shows a convergence if the states of (Y<sub>1</sub>, Y<sub>2</sub>, Y<sub>3</sub>, Y<sub>4</sub>) are (A, B, A, B) or (A, B, B, A), where A and B are two observed states of the character in the four species. By contrast, we say that the character shows a consistency if (A, A, B, B) is observed. Statistical tests followed those in the whole-tree analysis, except that quartets replaced branch pairs. There were 103 quartets with zero convergence and zero consistency for molecular characters. These quartets had undefined <i>Cv/Cs</i> ratios for molecular characters and could not be tested by Fisher's exact test. Hence, they were excluded from the analysis and corresponding figures.</p> </div> <div class="comparisonofcv/dvgiventhenumberofstates" title="sec"> <div class="title" tagx="title" title="title">
Comparison of <i>Cv</i>/<i>Dv</i> given the number of states</div> <p>The <i>Cv/Dv</i> ratio of a character is calculated by the sum of <i>Cv</i> across all branch pairs divided by the sum of <i>Dv</i> across all branch pairs for the character. Morphological and molecular characters are divided into bins according to the number of states. For each bin, a ratio between mean morphological <i>Cv/Dv</i> and mean molecular <i>Cv/Dv</i> is calculated. Finally, this ratio is averaged across bins, weighted by the number of morphological characters in each bin. Hence, the weighted average reflects <i>Cv/Dv</i> of morphological characters relative to that of molecular characters of the same numbers of states. When the measure of <i>Cv/Cs</i> is used, the same procedure is followed except that quartets instead of branch pairs are used.</p> </div> <div class="simulationofcharacterevolution" title="sec"> <div class="title" tagx="title" title="title">
Simulation of character evolution</div> <p>The evolution of morphological and molecular characters was simulated according to Markov processes, based on the tree topology and branch lengths of the nucleotide maximum likelihood tree from the original study<a href="#b25">25</a> (<a href="#S1">Supplementary Fig. 6a</a>). The Newick format of the tree is ((((((((1:0.0759,(2:0.0568,3:0.0467)47:0.0234)48:0.00318,(((4:0.0448,5:0.0626)49:0.00468,((6:0.0656,7:0.0707)50:0.00570,(8:0.0634,9:0.0616)51:0.00210)52:0.0142)53:0.0150,((10:0.0602,(((11:0.0383,(12:0.0233,13:0.0128)54:0.0165)55:0.00491,14:0.0721)56:0.00919,15:0.0666)57:0.00518)58:0.0222,16:0.0559)59:0.00143)60:0.000543)61:0.00249,(17:0.1007,(18:0.0849,(19:0.1390,20:0.1468)62:0.0152)63:0.00163)64:0.00940)65:0.0110,(((21:0.0989,22:0.0676)66:0.0303,(23:0.1102,((24:0.0777,25:0.1875)67:0.00944,(26:0.0941,27:0.1660)68:0.00225)69:0.0131)70:0.00528)71:0.00149,((28:0.0618,(29:0.0913,(30:0.0414,31:0.0231)72:0.0368)73:0.00438)74:0.00775,(32:0.00806,33:0.00966)75:0.0581)76:0.00177)77:0.00997)78:0.0107,(34:0.0664,35:0.0869)79:0.0309)80:0.00230,((36:0.0429,(37:0.1000,38:0.0439)81:0.00304)82:0.0133,(39:0.0695,((40:0.1506,41:0.0760)83:0.00679,42:0.1275)84:0.00331)85:0.00283)86:0.0300)87:0.1278,(43:0.0834,44:0.0739)88:0.1754)89:0.1518,(45:0.0454,46:0.0378)90:0.1518)91:0.0000. In the simulated evolution, a model equivalent to the Jukes–Cantor model assuming equal equilibrium frequencies of all states and equal exchange rates among all states was used. For each morphological character, its number of states <i>N</i> is a randomly drawn number from the empirical distribution of the number of states in the original morphological data (<a href="#f3">Fig. 3a</a>). The 1 PAM transition matrix for this character is an <i>N</i> × <i>N</i> square matrix <i>M</i> with each non-diagonal item equal to 0.01/<i>N</i>. The relative evolutionary rate <i>r</i> of the character is randomly drawn according to a Pearson correlation of 0.64 with the number of states <i>n</i>, as was observed in the actual data. Specifically, we draw a random variable <i>n'</i> from the empirical distribution of the number of states and compute <span class="inline-formula" title="inline-formula"><span class="inline-graphic" title="inline-graphic" /></span>. We then normalize <i>r</i> such that the mean <i>r</i> from all characters equals 1.</p> <p>The character evolution then starts from a random initial state at the tree root and evolves by a Markov chain along tree branches. Molecular characters were similarly simulated. Fifty simulations were conducted, each composed of 20,000 morphological characters and 40,000 molecular characters. The number of states used to generate each character and the number of substitution steps in evolution were recorded for downstream analysis. Quartet analysis based on a randomly picked simulation showed that the properties of these characters resemble those of the real data. Specifically, for almost all quartets, convergence per character and <i>Cv</i>/<i>Cs</i> ratio are higher for morphological characters than molecular character (<a href="#S1">Supplementary Fig. 6b,c</a>). In addition, a significant negative partial correlation was observed between the number of states and <i>Cv</i>/<i>Cs</i> ratio when the number of steps was controlled (<a href="#S1">Supplementary Table 3</a>).</p> </div> <div class="inferenceofparsimonytrees" title="sec"> <div class="title" tagx="title" title="title">
Inference of parsimony trees</div> <p>Because the evolutionary models of morphological characters have not been well established, model-based tree inference is not used here. Instead, we inferred maximum parsimony trees using PAUP4.0 (<a href="http://people.sc.fsu.edu/~dswofford/paup_test/">http://people.sc.fsu.edu/~dswofford/paup_test/</a>) for both morphological and molecular data to allow fair comparisons. When analysing the real data, 1,000 replicated heuristic searches were performed with parameters from the original study<a href="#b25">25</a>. All fossil taxa were included when morphological characters were used in the inference. Consensus trees were derived when multiple equally parsimonious topologies were found, with a strict collapse of branches and equal weights of all topologies. In the analysis of simulated characters, 5,000 replications were used instead of 1,000. Bootstrap tests were conducted in PAUP with 1,000 replicates unless otherwise mentioned. Bootstrap values were calculated and mapped by custom Python scripts; equal weights were given to all equally parsimonious trees resulting from each bootstrapped data set.</p> </div> <div class="phylogeneticanalysesoflow-convergencecharacters" title="sec"> <div class="title" tagx="title" title="title">
Phylogenetic analyses of low-convergence characters</div> <p>We used various <i>Cv</i>/<i>Cs</i> ratio cutoffs to remove characters whose <i>Cv</i>/<i>Cs</i> ratios are higher than the cutoffs. For example, in the real data of 9136 parsimony-informative characters, 5,206 characters showed <i>Cv</i>/<i>Cs</i>&amp;gt;0.2, according to quartet analysis. Hence, under the cutoff of <i>Cv</i>/<i>Cs</i>=0.2, we retained 9,136−5,206=3930 characters for tree inference, including 1,007 morphological and 2,923 molecular characters. As a control, we randomly drew 1,007 morphological and 2,923 molecular characters from all 9,136 characters and conducted a phylogenetic analysis. This control was repeated 50 times.</p> </div> <div class="dataavailability" title="sec"> <div class="title" tagx="title" title="title">
Data availability</div> <p>All morphological and molecular data analysed were previously published<a href="#b25">25</a>, and the data matrices and related files were retrieved from MorphoBank Project 773 (<a href="http://www.morphobank.org/">http://www.morphobank.org/</a>).</p> </div> </div> <div class="additionalinformation" title="sec"> <div class="title" tagx="title" title="title">
Additional information</div> <p><b>How to cite this article:</b> Zou, Z. &amp;amp; Zhang, J. Morphological and molecular convergences in mammalian phylogenetics. <i>Nat. Commun.</i> 7:12758 doi: 10.1038/ncomms12758 (2016).</p> </div> <div class="supplementary-material" title="supplementary-material"> <div class="title" tagx="title" title="title">
Supplementary Material</div> <div class="local-data" title="local-data">  <div class="title" tagx="title" title="title">
Supplementary Information</div> <p>Supplementary Figures 1 - 8 and Supplementary Tables 1 - 3</p>  <div class="media" title="media"><a href="ncomms12758-s1.pdf">LINK</a></div> </div> <div class="local-data" title="local-data">  <div class="title" tagx="title" title="title">
Peer Review File</div>  <div class="media" title="media"><a href="ncomms12758-s2.pdf">LINK</a></div> </div> </div> </div> <div class="back" title="back"> <div class="ack" title="ack"> <p>We thank Wei-Chin Ho, Bryan Moyers and Jian-Rong Yang for constructive comments. This work was supported in part by a research grant from the U.S. National Institutes of Health (GM103232) to J.Z.</p> </div> <div class="references">
References</div> <div tag="ref-list"> <ul> <li tag="ref"><a name="b1" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Springer</span><span class="given-names" tagx="given-names" title="given-names">M. S.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Meredith</span><span class="given-names" tagx="given-names" title="given-names">R. W.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Teeling</span><span class="given-names" tagx="given-names" title="given-names">E. C.</span></span> &amp;amp; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Murphy</span><span class="given-names" tagx="given-names" title="given-names">W. J.</span></span><span class="mixed-article-title" title="mixed-article-title">Technical comment on ‘The placental mammal ancestor and the post-K-Pg radiation of placentals'</span>. <span class="source" tagx="source" title="source">Science</span><span class="volume" tagx="volume" title="volume">341</span>, <span class="fpage" tagx="fpage" title="fpage">613</span> (<span class="year" tagx="year" title="year">2013</span>).<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23929967">23929967</a></span></span></li> <li tag="ref"><a name="b2" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Jarvis</span><span class="given-names" tagx="given-names" title="given-names">E. D.</span></span><i>et al.</i><span class="mixed-article-title" title="mixed-article-title">Whole-genome analyses resolve early branches in the tree of life of modern birds</span>. <span class="source" tagx="source" title="source">Science</span><span class="volume" tagx="volume" title="volume">346</span>, <span class="fpage" tagx="fpage" title="fpage">1320</span>–<span class="lpage" tagx="lpage" title="lpage">1331</span> (<span class="year" tagx="year" title="year">2014</span>).<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25504713">25504713</a></span></span></li> <li tag="ref"><a name="b3" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wake</span><span class="given-names" tagx="given-names" title="given-names">D. B.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wake</span><span class="given-names" tagx="given-names" title="given-names">M. H.</span></span> &amp;amp; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Specht</span><span class="given-names" tagx="given-names" title="given-names">C. D.</span></span><span class="mixed-article-title" title="mixed-article-title">Homoplasy: from detecting pattern to determining process and mechanism of evolution</span>. <span class="source" tagx="source" title="source">Science</span><span class="volume" tagx="volume" title="volume">331</span>, <span class="fpage" tagx="fpage" title="fpage">1032</span>–<span class="lpage" tagx="lpage" title="lpage">1035</span> (<span class="year" tagx="year" title="year">2011</span>).<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21350170">21350170</a></span></span></li> <li tag="ref"><a name="b4" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Jousselin</span><span class="given-names" tagx="given-names" title="given-names">E.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Rasplus</span><span class="given-names" tagx="given-names" title="given-names">J. Y.</span></span> &amp;amp; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kjellberg</span><span class="given-names" tagx="given-names" title="given-names">F.</span></span><span class="mixed-article-title" title="mixed-article-title">Convergence and coevolution in a mutualism: evidence from a molecular phylogeny of Ficus</span>. <span class="source" tagx="source" title="source">Evolution</span><span class="volume" tagx="volume" title="volume">57</span>, <span class="fpage" tagx="fpage" title="fpage">1255</span>–<span class="lpage" tagx="lpage" title="lpage">1269</span> (<span class="year" tagx="year" title="year">2003</span>).<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12894934">12894934</a></span></span></li> <li tag="ref"><a name="b5" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Legg</span><span class="given-names" tagx="given-names" title="given-names">D. A.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Sutton</span><span class="given-names" tagx="given-names" title="given-names">M. D.</span></span> &amp;amp; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Edgecombe</span><span class="given-names" tagx="given-names" title="given-names">G. D.</span></span><span class="mixed-article-title" title="mixed-article-title">Arthropod fossil data increase congruence of morphological and molecular phylogenies</span>. <span class="source" tagx="source" title="source">Nat. Commun.</span><span class="volume" tagx="volume" title="volume">4</span>, <span class="fpage" tagx="fpage" title="fpage">2485</span> (<span class="year" tagx="year" title="year">2013</span>).<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24077329">24077329</a></span></span></li> <li tag="ref"><a name="b6" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Perelman</span><span class="given-names" tagx="given-names" title="given-names">P.</span></span><i>et al.</i><span class="mixed-article-title" title="mixed-article-title">A molecular phylogeny of living primates</span>. <span class="source" tagx="source" title="source">PLoS Genet.</span><span class="volume" tagx="volume" title="volume">7</span>, <span class="fpage" tagx="fpage" title="fpage">e1001342</span> (<span class="year" tagx="year" title="year">2011</span>).<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21436896">21436896</a></span></span></li> <li tag="ref"><a name="b7" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Davalos</span><span class="given-names" tagx="given-names" title="given-names">L. M.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Cirranello</span><span class="given-names" tagx="given-names" title="given-names">A. L.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Geisler</span><span class="given-names" tagx="given-names" title="given-names">J. H.</span></span> &amp;amp; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Simmons</span><span class="given-names" tagx="given-names" title="given-names">N. B.</span></span><span class="mixed-article-title" title="mixed-article-title">Understanding phylogenetic incongruence: lessons from phyllostomid bats</span>. <span class="source" tagx="source" title="source">Biol. Rev. Camb. Philos. Soc.</span><span class="volume" tagx="volume" title="volume">87</span>, <span class="fpage" tagx="fpage" title="fpage">991</span>–<span class="lpage" tagx="lpage" title="lpage">1024</span> (<span class="year" tagx="year" title="year">2012</span>).<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22891620">22891620</a></span></span></li> <li tag="ref"><a name="b8" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gaubert</span><span class="given-names" tagx="given-names" title="given-names">P.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wozencraft</span><span class="given-names" tagx="given-names" title="given-names">W. C.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Cordeiro-Estrela</span><span class="given-names" tagx="given-names" title="given-names">P.</span></span> &amp;amp; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Veron</span><span class="given-names" tagx="given-names" title="given-names">G.</span></span><span class="mixed-article-title" title="mixed-article-title">Mosaics of convergences and noise in morphological phylogenies: what's in a viverrid-like carnivoran?</span><span class="source" tagx="source" title="source">Syst. Biol.</span><span class="volume" tagx="volume" title="volume">54</span>, <span class="fpage" tagx="fpage" title="fpage">865</span>–<span class="lpage" tagx="lpage" title="lpage">894</span> (<span class="year" tagx="year" title="year">2005</span>).<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16282167">16282167</a></span></span></li> <li tag="ref"><a name="b9" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wiens</span><span class="given-names" tagx="given-names" title="given-names">J. J.</span></span><i>et al.</i><span class="mixed-article-title" title="mixed-article-title">Combining phylogenomics and fossils in higher-level squamate reptile phylogeny: molecular data change the placement of fossil taxa</span>. <span class="source" tagx="source" title="source">Syst. Biol.</span><span class="volume" tagx="volume" title="volume">59</span>, <span class="fpage" tagx="fpage" title="fpage">674</span>–<span class="lpage" tagx="lpage" title="lpage">688</span> (<span class="year" tagx="year" title="year">2010</span>).<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20930035">20930035</a></span></span></li> <li tag="ref"><a name="b10" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Davalos</span><span class="given-names" tagx="given-names" title="given-names">L. M.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Velazco</span><span class="given-names" tagx="given-names" title="given-names">P. M.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Warsi</span><span class="given-names" tagx="given-names" title="given-names">O. M.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Smits</span><span class="given-names" tagx="given-names" title="given-names">P. D.</span></span> &amp;amp; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Simmons</span><span class="given-names" tagx="given-names" title="given-names">N. B.</span></span><span class="mixed-article-title" title="mixed-article-title">Integrating incomplete fossils by isolating conflicting signal in saturated and non-independent morphological characters</span>. <span class="source" tagx="source" title="source">Syst. Biol.</span><span class="volume" tagx="volume" title="volume">63</span>, <span class="fpage" tagx="fpage" title="fpage">582</span>–<span class="lpage" tagx="lpage" title="lpage">600</span> (<span class="year" tagx="year" title="year">2014</span>).<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24817532">24817532</a></span></span></li> <li tag="ref"><a name="b11" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Givnish</span><span class="given-names" tagx="given-names" title="given-names">T. J.</span></span> &amp;amp; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Sytsma</span><span class="given-names" tagx="given-names" title="given-names">K. J.</span></span><span class="mixed-article-title" title="mixed-article-title">Consistency, characters, and the likelihood of correct phylogenetic inference</span>. <span class="source" tagx="source" title="source">Mol. Phylogenet. Evol.</span><span class="volume" tagx="volume" title="volume">7</span>, <span class="fpage" tagx="fpage" title="fpage">320</span>–<span class="lpage" tagx="lpage" title="lpage">330</span> (<span class="year" tagx="year" title="year">1997</span>).<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/9187091">9187091</a></span></span></li> <li tag="ref"><a name="b12" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Page</span><span class="given-names" tagx="given-names" title="given-names">R. D. M.</span></span> &amp;amp; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Holmes</span><span class="given-names" tagx="given-names" title="given-names">E. C.</span></span><span class="source" tagx="source" title="source">Molecular Evolution: a Phylogenetic Approach</span> Blackwell Science (<span class="year" tagx="year" title="year">1998</span>).</span></li> <li tag="ref"><a name="b13" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zhang</span><span class="given-names" tagx="given-names" title="given-names">J.</span></span> &amp;amp; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kumar</span><span class="given-names" tagx="given-names" title="given-names">S.</span></span><span class="mixed-article-title" title="mixed-article-title">Detection of convergent and parallel evolution at the amino acid sequence level</span>. <span class="source" tagx="source" title="source">Mol. Biol. Evol.</span><span class="volume" tagx="volume" title="volume">14</span>, <span class="fpage" tagx="fpage" title="fpage">527</span>–<span class="lpage" tagx="lpage" title="lpage">536</span> (<span class="year" tagx="year" title="year">1997</span>).<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/9159930">9159930</a></span></span></li> <li tag="ref"><a name="b14" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Li</span><span class="given-names" tagx="given-names" title="given-names">Y.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Liu</span><span class="given-names" tagx="given-names" title="given-names">Z.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Shi</span><span class="given-names" tagx="given-names" title="given-names">P.</span></span> &amp;amp; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zhang</span><span class="given-names" tagx="given-names" title="given-names">J.</span></span><span class="mixed-article-title" title="mixed-article-title">The hearing gene <i>Prestin</i> unites echolocating bats and whales</span>. <span class="source" tagx="source" title="source">Curr. Biol.</span><span class="volume" tagx="volume" title="volume">20</span>, <span class="fpage" tagx="fpage" title="fpage">R55</span>–<span class="lpage" tagx="lpage" title="lpage">R56</span> (<span class="year" tagx="year" title="year">2010</span>).<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20129037">20129037</a></span></span></li> <li tag="ref"><a name="b15" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Castoe</span><span class="given-names" tagx="given-names" title="given-names">T. A.</span></span><i>et al.</i><span class="mixed-article-title" title="mixed-article-title">Evidence for an ancient adaptive episode of convergent molecular evolution</span>. <span class="source" tagx="source" title="source">Proc. Natl Acad. Sci. USA</span><span class="volume" tagx="volume" title="volume">106</span>, <span class="fpage" tagx="fpage" title="fpage">8986</span>–<span class="lpage" tagx="lpage" title="lpage">8991</span> (<span class="year" tagx="year" title="year">2009</span>).<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19416880">19416880</a></span></span></li> <li tag="ref"><a name="b16" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ujvari</span><span class="given-names" tagx="given-names" title="given-names">B.</span></span><i>et al.</i><span class="mixed-article-title" title="mixed-article-title">Widespread convergence in toxin resistance by predictable molecular evolution</span>. <span class="source" tagx="source" title="source">Proc. Natl Acad. Sci. USA</span><span class="volume" tagx="volume" title="volume">112</span>, <span class="fpage" tagx="fpage" title="fpage">11911</span>–<span class="lpage" tagx="lpage" title="lpage">11916</span> (<span class="year" tagx="year" title="year">2015</span>).<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/26372961">26372961</a></span></span></li> <li tag="ref"><a name="b17" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Parker</span><span class="given-names" tagx="given-names" title="given-names">J.</span></span><i>et al.</i><span class="mixed-article-title" title="mixed-article-title">Genome-wide signatures of convergent evolution in echolocating mammals</span>. <span class="source" tagx="source" title="source">Nature</span><span class="volume" tagx="volume" title="volume">502</span>, <span class="fpage" tagx="fpage" title="fpage">228</span>–<span class="lpage" tagx="lpage" title="lpage">231</span> (<span class="year" tagx="year" title="year">2013</span>).<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24005325">24005325</a></span></span></li> <li tag="ref"><a name="b18" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Foote</span><span class="given-names" tagx="given-names" title="given-names">A. D.</span></span><i>et al.</i><span class="mixed-article-title" title="mixed-article-title">Convergent evolution of the genomes of marine mammals</span>. <span class="source" tagx="source" title="source">Nat. Genet.</span><span class="volume" tagx="volume" title="volume">47</span>, <span class="fpage" tagx="fpage" title="fpage">272</span>–<span class="lpage" tagx="lpage" title="lpage">275</span> (<span class="year" tagx="year" title="year">2015</span>).<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25621460">25621460</a></span></span></li> <li tag="ref"><a name="b19" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zou</span><span class="given-names" tagx="given-names" title="given-names">Z.</span></span> &amp;amp; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zhang</span><span class="given-names" tagx="given-names" title="given-names">J.</span></span><span class="mixed-article-title" title="mixed-article-title">Are convergent and parallel amino acid substitutions in protein evolution more prevalent than neutral expectations?</span><span class="source" tagx="source" title="source">Mol. Biol. Evol.</span><span class="volume" tagx="volume" title="volume">32</span>, <span class="fpage" tagx="fpage" title="fpage">2085</span>–<span class="lpage" tagx="lpage" title="lpage">2096</span> (<span class="year" tagx="year" title="year">2015</span>).<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25862140">25862140</a></span></span></li> <li tag="ref"><a name="b20" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Rokas</span><span class="given-names" tagx="given-names" title="given-names">A.</span></span> &amp;amp; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Carroll</span><span class="given-names" tagx="given-names" title="given-names">S. B.</span></span><span class="mixed-article-title" title="mixed-article-title">Frequent and widespread parallel evolution of protein sequences</span>. <span class="source" tagx="source" title="source">Mol. Biol. Evol.</span><span class="volume" tagx="volume" title="volume">25</span>, <span class="fpage" tagx="fpage" title="fpage">1943</span>–<span class="lpage" tagx="lpage" title="lpage">1953</span> (<span class="year" tagx="year" title="year">2008</span>).<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18583353">18583353</a></span></span></li> <li tag="ref"><a name="b21" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Christin</span><span class="given-names" tagx="given-names" title="given-names">P. A.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Weinreich</span><span class="given-names" tagx="given-names" title="given-names">D. M.</span></span> &amp;amp; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Besnard</span><span class="given-names" tagx="given-names" title="given-names">G.</span></span><span class="mixed-article-title" title="mixed-article-title">Causes and evolutionary significance of genetic convergence</span>. <span class="source" tagx="source" title="source">Trends Genet.</span><span class="volume" tagx="volume" title="volume">26</span>, <span class="fpage" tagx="fpage" title="fpage">400</span>–<span class="lpage" tagx="lpage" title="lpage">405</span> (<span class="year" tagx="year" title="year">2010</span>).<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20685006">20685006</a></span></span></li> <li tag="ref"><a name="b22" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Projecto-Garcia</span><span class="given-names" tagx="given-names" title="given-names">J.</span></span><i>et al.</i><span class="mixed-article-title" title="mixed-article-title">Repeated elevational transitions in hemoglobin function during the evolution of Andean hummingbirds</span>. <span class="source" tagx="source" title="source">Proc. Natl Acad. Sci. USA</span><span class="volume" tagx="volume" title="volume">110</span>, <span class="fpage" tagx="fpage" title="fpage">20669</span>–<span class="lpage" tagx="lpage" title="lpage">20674</span> (<span class="year" tagx="year" title="year">2013</span>).<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24297909">24297909</a></span></span></li> <li tag="ref"><a name="b23" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zhen</span><span class="given-names" tagx="given-names" title="given-names">Y.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Aardema</span><span class="given-names" tagx="given-names" title="given-names">M. L.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Medina</span><span class="given-names" tagx="given-names" title="given-names">E. M.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Schumer</span><span class="given-names" tagx="given-names" title="given-names">M.</span></span> &amp;amp; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Andolfatto</span><span class="given-names" tagx="given-names" title="given-names">P.</span></span><span class="mixed-article-title" title="mixed-article-title">Parallel molecular evolution in an herbivore community</span>. <span class="source" tagx="source" title="source">Science</span><span class="volume" tagx="volume" title="volume">337</span>, <span class="fpage" tagx="fpage" title="fpage">1634</span>–<span class="lpage" tagx="lpage" title="lpage">1637</span> (<span class="year" tagx="year" title="year">2012</span>).<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23019645">23019645</a></span></span></li> <li tag="ref"><a name="b24" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Pisani</span><span class="given-names" tagx="given-names" title="given-names">D.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Benton</span><span class="given-names" tagx="given-names" title="given-names">M. J.</span></span> &amp;amp; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wilkinson</span><span class="given-names" tagx="given-names" title="given-names">M.</span></span><span class="mixed-article-title" title="mixed-article-title">Congruence of morphological and molecular phylogenies</span>. <span class="source" tagx="source" title="source">Acta Biotheor.</span><span class="volume" tagx="volume" title="volume">55</span>, <span class="fpage" tagx="fpage" title="fpage">269</span>–<span class="lpage" tagx="lpage" title="lpage">281</span> (<span class="year" tagx="year" title="year">2007</span>).<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17657570">17657570</a></span></span></li> <li tag="ref"><a name="b25" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">O'Leary</span><span class="given-names" tagx="given-names" title="given-names">M. A.</span></span><i>et al.</i><span class="mixed-article-title" title="mixed-article-title">The placental mammal ancestor and the post-K-Pg radiation of placentals</span>. <span class="source" tagx="source" title="source">Science</span><span class="volume" tagx="volume" title="volume">339</span>, <span class="fpage" tagx="fpage" title="fpage">662</span>–<span class="lpage" tagx="lpage" title="lpage">667</span> (<span class="year" tagx="year" title="year">2013</span>).<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23393258">23393258</a></span></span></li> <li tag="ref"><a name="b26" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Pyron</span><span class="given-names" tagx="given-names" title="given-names">R. A.</span></span><span class="mixed-article-title" title="mixed-article-title">Post-molecular systematics and the future of phylogenetics</span>. <span class="source" tagx="source" title="source">Trends Ecol. Evol.</span><span class="volume" tagx="volume" title="volume">30</span>, <span class="fpage" tagx="fpage" title="fpage">384</span>–<span class="lpage" tagx="lpage" title="lpage">389</span> (<span class="year" tagx="year" title="year">2015</span>).<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/26014093">26014093</a></span></span></li> <li tag="ref"><a name="b27" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bieler</span><span class="given-names" tagx="given-names" title="given-names">R.</span></span><i>et al.</i><span class="mixed-article-title" title="mixed-article-title">Investigating the Bivalve Tree of Life—an exemplar-based approach combining molecular and novel morphological characters</span>. <span class="source" tagx="source" title="source">Invertebr. Syst.</span><span class="volume" tagx="volume" title="volume">28</span>, <span class="fpage" tagx="fpage" title="fpage">32</span>–<span class="lpage" tagx="lpage" title="lpage">115</span> (<span class="year" tagx="year" title="year">2014</span>).</span></li> <li tag="ref"><a name="b28" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lee</span><span class="given-names" tagx="given-names" title="given-names">M. S.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Soubrier</span><span class="given-names" tagx="given-names" title="given-names">J.</span></span> &amp;amp; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Edgecombe</span><span class="given-names" tagx="given-names" title="given-names">G. D.</span></span><span class="mixed-article-title" title="mixed-article-title">Rates of phenotypic and genomic evolution during the Cambrian explosion</span>. <span class="source" tagx="source" title="source">Curr. Biol.</span><span class="volume" tagx="volume" title="volume">23</span>, <span class="fpage" tagx="fpage" title="fpage">1889</span>–<span class="lpage" tagx="lpage" title="lpage">1895</span> (<span class="year" tagx="year" title="year">2013</span>).<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24035543">24035543</a></span></span></li> <li tag="ref"><a name="b29" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lee</span><span class="given-names" tagx="given-names" title="given-names">M. S.</span></span> &amp;amp; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Palci</span><span class="given-names" tagx="given-names" title="given-names">A.</span></span><span class="mixed-article-title" title="mixed-article-title">Morphological phylogenetics in the genomic age</span>. <span class="source" tagx="source" title="source">Curr. Biol.</span><span class="volume" tagx="volume" title="volume">25</span>, <span class="fpage" tagx="fpage" title="fpage">R922</span>–<span class="lpage" tagx="lpage" title="lpage">R929</span> (<span class="year" tagx="year" title="year">2015</span>).<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/26439355">26439355</a></span></span></li> <li tag="ref"><a name="b30" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Thomas</span><span class="given-names" tagx="given-names" title="given-names">G. W.</span></span> &amp;amp; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hahn</span><span class="given-names" tagx="given-names" title="given-names">M. W.</span></span><span class="mixed-article-title" title="mixed-article-title">Determining the null model for detecting adaptive convergence from genomic data: a case study using echolocating mammals</span>. <span class="source" tagx="source" title="source">Mol. Biol. Evol.</span><span class="volume" tagx="volume" title="volume">32</span>, <span class="fpage" tagx="fpage" title="fpage">1232</span>–<span class="lpage" tagx="lpage" title="lpage">1236</span> (<span class="year" tagx="year" title="year">2015</span>).<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25631926">25631926</a></span></span></li> <li tag="ref"><a name="b31" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zou</span><span class="given-names" tagx="given-names" title="given-names">Z.</span></span> &amp;amp; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zhang</span><span class="given-names" tagx="given-names" title="given-names">J.</span></span><span class="mixed-article-title" title="mixed-article-title">No genome-wide protein sequence convergence for echolocation</span>. <span class="source" tagx="source" title="source">Mol. Biol. Evol.</span><span class="volume" tagx="volume" title="volume">32</span>, <span class="fpage" tagx="fpage" title="fpage">1237</span>–<span class="lpage" tagx="lpage" title="lpage">1241</span> (<span class="year" tagx="year" title="year">2015</span>).<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25631925">25631925</a></span></span></li> <li tag="ref"><a name="b32" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Robinson</span><span class="given-names" tagx="given-names" title="given-names">D. F.</span></span> &amp;amp; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Foulds</span><span class="given-names" tagx="given-names" title="given-names">L. R.</span></span><span class="mixed-article-title" title="mixed-article-title">Comparison of phylogenetic trees</span>. <span class="source" tagx="source" title="source">Math. Biosci.</span><span class="volume" tagx="volume" title="volume">53</span>, <span class="fpage" tagx="fpage" title="fpage">131</span>–<span class="lpage" tagx="lpage" title="lpage">147</span> (<span class="year" tagx="year" title="year">1981</span>).</span></li> <li tag="ref"><a name="b33" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Meredith</span><span class="given-names" tagx="given-names" title="given-names">R. W.</span></span><i>et al.</i><span class="mixed-article-title" title="mixed-article-title">Impacts of the cretaceous terrestrial revolution and KPg extinction on mammal diversification</span>. <span class="source" tagx="source" title="source">Science</span><span class="volume" tagx="volume" title="volume">334</span>, <span class="fpage" tagx="fpage" title="fpage">521</span>–<span class="lpage" tagx="lpage" title="lpage">524</span> (<span class="year" tagx="year" title="year">2011</span>).<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21940861">21940861</a></span></span></li> <li tag="ref"><a name="b34" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Tsagkogeorga</span><span class="given-names" tagx="given-names" title="given-names">G.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Parker</span><span class="given-names" tagx="given-names" title="given-names">J.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Stupka</span><span class="given-names" tagx="given-names" title="given-names">E.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Cotton</span><span class="given-names" tagx="given-names" title="given-names">J. A.</span></span> &amp;amp; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Rossiter</span><span class="given-names" tagx="given-names" title="given-names">S. J.</span></span><span class="mixed-article-title" title="mixed-article-title">Phylogenomic analyses elucidate the evolutionary relationships of bats</span>. <span class="source" tagx="source" title="source">Curr. Biol.</span><span class="volume" tagx="volume" title="volume">23</span>, <span class="fpage" tagx="fpage" title="fpage">2262</span>–<span class="lpage" tagx="lpage" title="lpage">2267</span> (<span class="year" tagx="year" title="year">2013</span>).<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24184098">24184098</a></span></span></li> <li tag="ref"><a name="b35" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Teeling</span><span class="given-names" tagx="given-names" title="given-names">E. C.</span></span><i>et al.</i><span class="mixed-article-title" title="mixed-article-title">A molecular phylogeny for bats illuminates biogeography and the fossil record</span>. <span class="source" tagx="source" title="source">Science</span><span class="volume" tagx="volume" title="volume">307</span>, <span class="fpage" tagx="fpage" title="fpage">580</span>–<span class="lpage" tagx="lpage" title="lpage">584</span> (<span class="year" tagx="year" title="year">2005</span>).<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15681385">15681385</a></span></span></li> <li tag="ref"><a name="b36" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Teeling</span><span class="given-names" tagx="given-names" title="given-names">E. C.</span></span><i>et al.</i><span class="mixed-article-title" title="mixed-article-title">Molecular evidence regarding the origin of echolocation and flight in bats</span>. <span class="source" tagx="source" title="source">Nature</span><span class="volume" tagx="volume" title="volume">403</span>, <span class="fpage" tagx="fpage" title="fpage">188</span>–<span class="lpage" tagx="lpage" title="lpage">192</span> (<span class="year" tagx="year" title="year">2000</span>).<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/10646602">10646602</a></span></span></li> <li tag="ref"><a name="b37" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zhang</span><span class="given-names" tagx="given-names" title="given-names">J.</span></span> &amp;amp; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Nei</span><span class="given-names" tagx="given-names" title="given-names">M.</span></span><span class="mixed-article-title" title="mixed-article-title">Accuracies of ancestral amino acid sequences inferred by the parsimony, likelihood, and distance methods</span>. <span class="source" tagx="source" title="source">J. Mol. Evol.</span><span class="volume" tagx="volume" title="volume">44</span>, (Suppl 1): <span class="fpage" tagx="fpage" title="fpage">S139</span>–<span class="lpage" tagx="lpage" title="lpage">S146</span> (<span class="year" tagx="year" title="year">1997</span>).<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/9071022">9071022</a></span></span></li> <li tag="ref"><a name="b38" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hahn</span><span class="given-names" tagx="given-names" title="given-names">M. W.</span></span> &amp;amp; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Nakhleh</span><span class="given-names" tagx="given-names" title="given-names">L.</span></span><span class="mixed-article-title" title="mixed-article-title">Irrational exuberance for resolved species trees</span>. <span class="source" tagx="source" title="source">Evolution</span><span class="volume" tagx="volume" title="volume">70</span>, <span class="fpage" tagx="fpage" title="fpage">7</span>–<span class="lpage" tagx="lpage" title="lpage">17</span> (<span class="year" tagx="year" title="year">2016</span>).<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/26639662">26639662</a></span></span></li> <li tag="ref"><a name="b39" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wiley</span><span class="given-names" tagx="given-names" title="given-names">E. O.</span></span> &amp;amp; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lieberman</span><span class="given-names" tagx="given-names" title="given-names">B. S.</span></span><span class="source" tagx="source" title="source">Phylogenetics: Theory and Practice of Phylogenetic Systematics</span> John Wiley &amp;amp; Sons (<span class="year" tagx="year" title="year">2011</span>).</span></li> <li tag="ref"><a name="b40" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wiens</span><span class="given-names" tagx="given-names" title="given-names">J. J.</span></span><span class="mixed-article-title" title="mixed-article-title">Character analysis in morphological phylogenetics: problems and solutions</span>. <span class="source" tagx="source" title="source">Syst. Biol.</span><span class="volume" tagx="volume" title="volume">50</span>, <span class="fpage" tagx="fpage" title="fpage">689</span>–<span class="lpage" tagx="lpage" title="lpage">699</span> (<span class="year" tagx="year" title="year">2001</span>).<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12116939">12116939</a></span></span></li> <li tag="ref"><a name="b41" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wagner</span><span class="given-names" tagx="given-names" title="given-names">P. J.</span></span><span class="mixed-article-title" title="mixed-article-title">Exhaustion of morphologic character states among fossil taxa</span>. <span class="source" tagx="source" title="source">Evolution</span><span class="volume" tagx="volume" title="volume">54</span>, <span class="fpage" tagx="fpage" title="fpage">365</span>–<span class="lpage" tagx="lpage" title="lpage">386</span> (<span class="year" tagx="year" title="year">2000</span>).<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/10937214">10937214</a></span></span></li> <li tag="ref"><a name="b42" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Du Toit</span><span class="given-names" tagx="given-names" title="given-names">Z.</span></span><i>et al.</i><span class="mixed-article-title" title="mixed-article-title">The complete mitochondrial genome of temminck's ground pangolin (<i>Smutsia temminckii</i>; Smuts, 1832) and phylogenetic position of the Pholidota (Weber, 1904)</span>. <span class="source" tagx="source" title="source">Gene</span><span class="volume" tagx="volume" title="volume">551</span>, <span class="fpage" tagx="fpage" title="fpage">49</span>–<span class="lpage" tagx="lpage" title="lpage">54</span> (<span class="year" tagx="year" title="year">2014</span>).<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25158133">25158133</a></span></span></li> <li tag="ref"><a name="b43" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Sansom</span><span class="given-names" tagx="given-names" title="given-names">R. S.</span></span> &amp;amp; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wills</span><span class="given-names" tagx="given-names" title="given-names">M. A.</span></span><span class="mixed-article-title" title="mixed-article-title">Fossilization causes organisms to appear erroneously primitive by distorting evolutionary trees</span>. <span class="source" tagx="source" title="source">Sci. Rep.</span><span class="volume" tagx="volume" title="volume">3</span>, <span class="fpage" tagx="fpage" title="fpage">2545</span> (<span class="year" tagx="year" title="year">2013</span>).<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23985991">23985991</a></span></span></li> <li tag="ref"><a name="b44" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Nei</span><span class="given-names" tagx="given-names" title="given-names">M.</span></span> &amp;amp; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kumar</span><span class="given-names" tagx="given-names" title="given-names">S.</span></span><span class="source" tagx="source" title="source">Molecular Evolution and Phylogenetics</span> Oxford University Press (<span class="year" tagx="year" title="year">2000</span>).</span></li> </ul> </div> <div class="fn-group" title="fn-group"> <div class="fn-type-" title=""> <p><b>Author contributions</b> J.Z. conceived the project; Z.Z. and J.Z. designed the project; Z.Z. conducted the experiments and analysed the data; Z.Z. and J.Z. wrote the manuscript.</p> </div> </div> </div> <div class="floats-group" title="floats-group"> <div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/"><span class="label" tagx="label" title="label">Figure 1</span></a></div>   <div class="title" tagx="title" title="title">
Whole-tree analysis of morphological and molecular convergences in mammals.</div> <p>(<b>a</b>) Schematic examples of convergence and divergence. Given the states of the interior and exterior nodes of the tree, the blue and green branch pairs each experienced a convergence event, while the orange branch pair experienced a divergence event. A, L, N and V are four different states of a character. (<b>b</b>) Mean number of convergences per morphological character and that per molecular character for each branch pair examined under the morphological tree. (<b>c</b>) Convergence/divergence (<i>Cv</i>/<i>Dv</i>) ratio for each branch pair under the morphological tree. (<b>d</b>) Mean number of convergences per morphological character and that per molecular character for each branch pair examined under the molecular tree. (<b>e</b>) <i>Cv</i>/<i>Dv</i> ratio for each branch pair under the molecular tree. In <b>b</b>–<b>e</b>, each dot represents a branch pair. In the grey box of each panel, ‘total' refers to the numbers of dots above and below the diagonal (dots on the diagonal are not counted), respectively, and ‘significant' refers to the numbers of dots significantly (at <i>Q</i>-value of 0.05) above (blue) and below (orange) the diagonal, respectively. Total number of dots above the diagonal significantly exceeds that below the diagonal in <b>b</b>–<b>e</b> (<i>P</i>&amp;lt;1 × 10<sup>−4</sup>, bootstrap test). For <b>c</b> and <b>e</b>, branch pairs with infinite <i>Cv/Dv</i> values are not plotted but included in the comparison.</p>   </div> <div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/"><span class="label" tagx="label" title="label">Figure 2</span></a></div>   <div class="title" tagx="title" title="title">
Quartet analysis of morphological and molecular convergences in mammals.</div> <p>(<b>a</b>) A schematic example of a quartet, which are four species (2, 3, 5 and 6) showing the same phylogenetic relationship in the morphological (left) and molecular (right) trees. Examples of character states exhibiting convergence and consistency are shown. (<b>b</b>) Mean number of convergences per morphological character and that per molecular character for each quartet examined. (<b>c</b>) Convergence/consistency (<i>Cv</i>/<i>Cs</i>) ratio for each quartet. In <b>b</b> and <b>c</b>, each dot represents a quartet. In the grey box of each panel, ‘total' refers to the numbers of dots above and below the diagonal (dots on the diagonal are not counted), respectively, and ‘significant' refers to the numbers of dots significantly (at <i>Q</i>-value of 0.05) above (blue) and below (orange) the diagonal, respectively. Total number of dots above the diagonal significantly exceeds that below the diagonal in <b>b</b> and <b>c</b> (<i>P</i>&amp;lt;1 × 10<sup>−4</sup>, bootstrap test). In <b>c</b>, quartets with infinite <i>Cv/Cs</i> values are not plotted but included in the comparison.</p>   </div> <div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/"><span class="label" tagx="label" title="label">Figure 3</span></a></div>   <div class="title" tagx="title" title="title">
Morphological characters tend to have fewer states than molecular characters.</div> <p>(<b>a</b>) Frequency distribution of the number of states per character. (<b>b</b>) <i>Cv</i>/<i>Dv</i> ratio of a character decreases as the number of states increases. <i>Cv/Dv</i> ratio of a character is the sum of convergences across all branch pairs divided by that of divergences. The top and bottom edges of a box represent the first and third quartiles of the distribution, respectively, while the thick line inside the box represents the median. The two whiskers show the maximum value not greater than the first quartile plus 1.5 times the box height and the minimum value not smaller than the third quartile minus 1.5 times the box height, respectively. <i>Cv</i>/<i>Dv</i> ratios are calculated under the morphological tree. The same pattern is observed when <i>Cv</i>/<i>Dv</i> ratios are calculated under the molecular tree (<a href="#S1">Supplementary Fig. 5</a>).</p>   </div> <div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/"><span class="label" tagx="label" title="label">Figure 4</span></a></div>   <div class="title" tagx="title" title="title">
Removing convergence-prone characters improves phylogenetic accuracy.</div> <p>(<b>a</b>) Simulation results showing that using characters with <i>Cv</i>/<i>Cs</i> ratios below certain cutoffs reduces the Robinson-Foulds distance (<i>d</i><sub>RF</sub>) between the true tree and the inferred tree, while using the same number of randomly picked characters does not. The top and bottom edges of a box, respectively, represent the first and third quartiles of the distribution from 50 simulations, while the thick line inside the box represents the median. The two whiskers show the maximum value not greater than the first quartile plus 1.5 times the box height and the minimum value not smaller than the third quartile minus 1.5 times the box height, respectively. <i>Cv</i>/<i>Cs</i> ratios are estimated based on quartets (sets of four species with the same phylogenetic relationships in the inferred morphological and molecular trees of the simulated data). *<i>P</i>&amp;lt;0.05, paired Mann–Whitney <i>U</i>-test from 50 simulations; **<i>P</i>&amp;lt;0.01. (<b>b</b>) Bat relationships based on the parsimony tree reconstructed using 9,136 informative morphological and molecular characters. Echolocating species are marked with an echo sign. Extant bats are in black, while fossil bats and other taxa are in grey. Bootstrap percentage is presented for each internal node, with the red colour highlighting the bootstrap percentage for the monophyly of echolocators. (<b>c</b>) Topology based on 3,930 informative characters with <i>Cv</i>/<i>Cs</i> ratios &amp;lt;0.2. The red colour highlights the bootstrap percentage for the sister relationship between the non-echolocator <i>Pteropus giganteus</i> and the echolocator <i>Rhinopoma hardwickii</i>.</p>   </div> </div>  </body></html>