<html xmlns="http://www.w3.org/1999/xhtml"> <head> <style type="text/css">
      	     		 
      	    a                          {background : #ffffff; }
      	    article                    {border-style : dotted; border-width : 2px; }
		 	
		 	div                        {background : #ffffcc;}
		 	div.abstract-title         {font-weight : bold ; font-size : 16pt;}
		 	div.ack                    {border-style : solid ; border-color : red; margin : 2em; }
		 	div.article-meta           {border-style : solid; border-width : 1 pt; margin 1em;}
		 	div.boxed-text             {margin : 5em; border-style : solid;}
		 	div.contrib-group          {margin : 1em; }
		 	div.fig                    {border-style : solid; border-width : 2px; margin : 2em; }
		 	div.funding                {font-weight : bold ; font-size : 16pt;}
		 	div.given-names            {font-style : italic;}
		 	div.intro                  {border-style : inset; margin : 5px;}
		 	div.introduction           {border-style : inset; margin : 5px;}
		 	div.journal-meta           {border-style : solid; border-width : 1 pt; margin 1em;}
		 	div.journal-title-group    {background : #ffeeee;}
		 	div.article-title          {font-weight : bold ; font-size : 18pt;}
		 	div.kwd                    {font-style : italic}
		 	div.meta-name              {font-weight : bold ; font-size : 16pt;}
		 	div.name                   {font-weight : bold;}
		 	div.alt-title              {font-style : italic; font-size : 12px;}
		 	
		 	div.sec                    {border : 2px; margin 5px; padding 2px;}
		 	div.title                  {font-family : courier; font-weight : bold;
		 	                            font-size : 16pt; margin : 5px;}
		 	
		 	div.materials_methods      {border-style : double; margin : 5px; }
		 	div.methods                {border-style : double; margin : 5px; }
		 	
		 	div.results                {border-style : solid; margin : 5px;}
		 	div.background             {border-style : dotted; margin : 5px;}
		 	
		 	div.discussion             {border-style : groove; margin : 5px;}
		 	div.conclusion             {border-style : ridge; margin : 5px;}
		 	div.conclusions             {border-style : ridge; margin : 5px;}
		 	div.supplementary-material {border-style : inset; margin : 5px;}
		 	div.abbreviations          {border-style : double; border-color : red; margin : 5px;}
		 	div.competinginterests     {border-style : double; border-color : blue; margin : 5px;}
		 	div.acknowledgements       {border-style : double; border-color : green; margin : 5px;}
		 	div.authors_contributions   {border-style : double; border-color : purple; margin : 5px;}
		 	
		 	div.publisher              {border-style : outset; margin : 5px;}
		 	div.fn-type-conflict       {background : #f88; }
		 	div.fn-type-con            {background : #ddf; }
		 	div.fn-type-other          {background : #ddd; }
		 	
		 	div.unknown                {background : #ffd;
									 	  border-style : solid;
									 	  border-width : 1px;
									 	  padding : 2 px;}
		 	  
      	    table                      {background : #ffffdd;}
		 	tr                         {background : #ddddff; padding : 1px;}
		 	
		 	span                       {background : #ffcccc;}
		 	
		 	span.citation-author       {font-family : helvetica; background : #ffeeee;}
		 	span.collab                {background : #ddffff; }
		 	span.comment               {font-family : courier; font-size : 6px; background : #ffaaff;}
		 	span.contrib               {background : #ffffff;}
		 	span.corresp               {background : #ddffdd; }
		 	span.doi                   {background : #ffffff;}
		 	span.email                 {font-family : courier; }
		 	span.etal                  {font-style : italic;}
		 	span.fpage                 {font-family : courier;}
		 	span.given-names           {background : #ffffff;}
		 	span.iso-abbrev            {background : #ffffff;}
		 	span.issn-epub             {background : #ffffff;}
		 	span.issn-ppub             {background : #ffffff;}
		 	span.journal-title         {background : #ffffff;}
		 	span.lpage                 {font-family : courier;}
		 	span.mixed-article-title   {font-style : italic ;}
		 	span.nlm-ta                {background : #ffffff;}
		 	span.pmc                   {background : #ffffff;}
		 	span.pmcid                 {background : #ffffff;}
		 	span.pmid                  {background : #ffffff;}
		 	span.publisher             {background : #ffffff;}
		 	span.publisher-id          {background : #ffffff;}
		 	span.publisher-name        {background : #ffffff;}
		 	span.source                {background : #ffffff;}
		 	span.subject               {background : #ffffff;}
		 	span.surname               {background : #ffffff;}
		 	span.volume                {font-family : courier; font-weight : bold;}
		 	span.year                  {font-family : courier ; font-style : italic;}
			</style> </head> <body> <div class="front" title="front"> <div class="journal-meta" tagx="journal-meta" title="journal-meta"><span class="nlm-ta" title="nlm-ta">PLoS One</span><span class="iso-abbrev" title="iso-abbrev">PLoS ONE</span><span class="publisher-id" title="publisher-id">plos</span><span class="pmc" title="pmc">plosone</span><div class="journal-title-group" tagx="journal-title-group" title="journal-title-group"><span class="journal-title" tagx="journal-title" title="journal-title">PLoS ONE</span></div><span class="issn-epub" tagx="issn" title="issn-epub">1932-6203</span><div class="publisher" tagx="publisher" title="publisher"><span class="publisher-name" tagx="publisher-name" title="publisher-name">Public Library of Science</span><span class="publisher-loc" tagx="publisher-loc" title="publisher-loc">San Francisco, USA</span></div> </div> <div class="article-meta" tagx="article-meta" title="article-meta"><span class="pmid" title="pmid"> pmid: <a href="http://www.ncbi.nlm.nih.gov/pubmed/21187920">21187920</a></span><span class="publisher-id" title="publisher-id">PONE-D-10-00552</span><span class="doi" title="doi"> doi: <a href="https://dx.doi.org/10.1371/journal.pone.0015571">10.1371/journal.pone.0015571</a></span><div class="article-categories" title="article-categories">
: <span class="subject" title="subject">Research Article</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Biology</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Biophysics</span>: <span class="subject" title="subject">Biophysics Simulations</span></div> <div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Computational Biology</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Population Modeling</span>: <span class="subject" title="subject">Infectious Disease Modeling</span></div> </div> <div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Microbiology</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Virology</span>: <span class="subject" title="subject">Viral Transmission and Infection</span></div>: <span class="subject" title="subject">Host-Pathogen Interaction</span></div> <div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Population Biology</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Epidemiology</span>: <span class="subject" title="subject">Infectious Disease Epidemiology</span><span class="subject" title="subject">Spatial Epidemiology</span></div>: <span class="subject" title="subject">Population Modeling</span></div> </div> <div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Mathematics</span>: <span class="subject" title="subject">Applied Mathematics</span></div> <div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Medicine</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Epidemiology</span>: <span class="subject" title="subject">Infectious Disease Epidemiology</span></div> <div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Infectious Diseases</span>: <span class="subject" title="subject">Infectious Disease Modeling</span></div> </div> </div> <div class="title-group" tagx="title-group" title="title-group"> <div class="article-title" title="article-title">
Viral Epidemics in a Cell Culture: Novel High Resolution Data and Their Interpretation by a Percolation Theory Based Model</div> <div class="alt-title" title="alt-title">
Viral Epidemics in a Cell Culture</div> </div> <div class="contrib-group" title="contrib-group"><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gönci</span><span class="given-names" tagx="given-names" title="given-names">Balázs</span></span><a href="#aff1"><sup>1</sup></a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Németh</span><span class="given-names" tagx="given-names" title="given-names">Valéria</span></span><a href="#aff1"><sup>1</sup></a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Balogh</span><span class="given-names" tagx="given-names" title="given-names">Emeric</span></span><a href="#aff1"><sup>1</sup></a><a href="#aff2"><sup>2</sup></a><a href="#aff3"><sup>3</sup></a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Szabó</span><span class="given-names" tagx="given-names" title="given-names">Bálint</span></span><a href="#aff1"><sup>1</sup></a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Dénes</span><span class="given-names" tagx="given-names" title="given-names">Ádám</span></span><a href="#aff4"><sup>4</sup></a><a href="#aff5"><sup>5</sup></a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Környei</span><span class="given-names" tagx="given-names" title="given-names">Zsuzsanna</span></span><a href="#aff4"><sup>4</sup></a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Vicsek</span><span class="given-names" tagx="given-names" title="given-names">Tamás</span></span><a href="#aff1"><sup>1</sup></a><a href="#aff6"><sup>6</sup></a><a href="#cor1"><sup>*</sup></a></span></div><span class="citation_author_institution" id="aff1">[1], <span class="addr-line" title="addr-line">Department of Biological Physics, Eötvös University, Budapest, Hungary</span></span><span class="citation_author_institution" id="aff2">[2], <span class="addr-line" title="addr-line">Department of Theoretical Physics, Babes-Bolyai University, Cluj-Napoca, Romania</span></span><span class="citation_author_institution" id="aff3">[3], <span class="addr-line" title="addr-line">Department of Optics and Quantum Electronics, University of Szeged, Szeged, Hungary</span></span><span class="citation_author_institution" id="aff4">[4], <span class="addr-line" title="addr-line">Institute of Experimental Medicine of the Hungarian Academy of Sciences, Budapest, Hungary</span></span><span class="citation_author_institution" id="aff5">[5], <span class="addr-line" title="addr-line">Faculty of Life Sciences, University of Manchester, Manchester, United Kingdom</span></span><span class="citation_author_institution" id="aff6">[6], <span class="addr-line" title="addr-line">Statistical and Biological Physics Research Group of the Hungarian Academy of Sciences, Budapest, Hungary</span></span><div class="contrib-group" title="contrib-group"><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ooi</span><span class="given-names" tagx="given-names" title="given-names">Eng Eong</span></span><span class="role" tagx="role" title="role">Editor</span><a href="#edit1" /></span></div><span class="citation_author_institution" id="edit1">[], </span><div class="author-notes" title="author-notes"> <div class="corresp" title="corresp">
* E-mail: <span class="email" tagx="email" title="email">vicsek@hal.elte.hu</span></div> <div class="fn-type-con" title="con"> <p>Conceived and designed the experiments: ÁD ZK TV. Performed the experiments: VN ZV. Analyzed the data: VN BG EB TV BS. Contributed reagents/materials/analysis tools: EB BG. Wrote the paper: BG ZK TV.</p> </div> </div><span class="pub-date-collection" title="pub-date-collection">collection: <span>2010</span></span><span class="pub-date-epub" title="pub-date-epub">epub: <span>2010-12-12</span></span><span class="volume" tagx="volume" title="volume">5</span><span class="issue" tagx="issue" title="issue">12</span><span class="elocation-id" tagx="elocation-id" title="elocation-id">e15571</span><span class="history" title="history"><span class="received" title="received">received: 2010-8-3</span><span class="accepted" title="accepted">accepted: 2010-11-13</span></span><div class="permissions"><span class="copyright" title="copyright">(C) , <span class="copyright-year" tagx="copyright-year" title="copyright-year">2010</span></span><span class="license" title="license"><span class="license-p" title="license-p">This is an open-access article distributed under the terms of the Creative Commons Attribution License, which permits unrestricted use, distribution, and reproduction in any medium, provided the original author and source are properly credited.</span></span></div> <div class="abstract" title="abstract"> <div class="abstract-title" title="abstract-title">
Abstract</div> <p>Because of its relevance to everyday life, the spreading of viral infections has been of central interest in a variety of scientific communities involved in fighting, preventing and theoretically interpreting epidemic processes. Recent large scale observations have resulted in major discoveries concerning the overall features of the spreading process in systems with highly mobile susceptible units, but virtually no data are available about observations of infection spreading for a very large number of immobile units. Here we present the first detailed quantitative documentation of percolation-type viral epidemics in a highly reproducible <i>in vitro</i> system consisting of tens of thousands of virtually motionless cells. We use a confluent astroglial monolayer in a Petri dish and induce productive infection in a limited number of cells with a genetically modified herpesvirus strain. This approach allows extreme high resolution tracking of the spatio-temporal development of the epidemic. We show that a simple model is capable of reproducing the basic features of our observations, i.e., the observed behaviour is likely to be applicable to many different kinds of systems. Statistical physics inspired approaches to our data, such as fractal dimension of the infected clusters as well as their size distribution, seem to fit into a percolation theory based interpretation. We suggest that our observations may be used to model epidemics in more complex systems, which are difficult to study in isolation.</p> </div> <div class="counts" title="counts"><span class="page-count" title="page-count">page-count: </span></div> </div> </div> <div class="body" title="body"> <div class="introduction" title="sec"> <div class="title" tagx="title" title="title">
Introduction</div> <p>Epidemic is a common phenomenon in a wide variety of systems ranging from flora through domestic animals to human populations. Understanding the statistics of the dynamics and spatial aspects of spreading infection could be crucial in its containment <a href="#pone.0015571-Brauer1">[1]</a>–<a href="#pone.0015571-AlainBarrat1">[4]</a>. Appropriate predictions about epidemics of various kinds can lead to enormous savings of both material goods as well as human lives <a href="#pone.0015571-Wang1">[5]</a>–<a href="#pone.0015571-Ferguson1">[8]</a>. In spite of its unquestionable importance, relatively little is known about the details of the spreading of a disease in a natural environment. In particular, observations of epidemics on a large scale (involving tens of thousands of organisms) are extremely scarce and incomplete <a href="#pone.0015571-Brauer1">[1]</a>, <a href="#pone.0015571-Daley1">[2]</a>, <a href="#pone.0015571-Lai1">[9]</a>, <a href="#pone.0015571-Pfeiffer1">[10]</a>. There are obvious reasons for this: i) in the case of individuals having either a considerable size (e.g., trees or animals), or moving around extensively (e.g., birds or people), the area to be covered in a controlled experiment is huge. ii) In addition, to infect and study living subjects is a risky and ethically problematic procedure. Therefore, a majority of the related studies involves modelling <a href="#pone.0015571-Riley1">[3]</a>, <a href="#pone.0015571-Keeling1">[11]</a> instead of experimenting. Many studies have pointed out the relevance of spatial inhomogeneities, to the specific (e.g., scale-free) patterns of connectivity and transport of the subjects. However, in the case of static units <a href="#pone.0015571-Haseltine1">[12]</a>, <a href="#pone.0015571-Lee1">[13]</a> the possibility of a detailed comparison between the predictions of the models and the actual observations for situations involving inhomogeneities has remained limited.</p> <p>In general, there are two major types of infection spreading: i) the position of the infected units does not change significantly in time, e.g., in cells in a tissue exposed to viral or bacterial infection <a href="#pone.0015571-Marquis1">[14]</a> or trees in a forest <a href="#pone.0015571-Rhodes1">[15]</a> ii) the organisms carrying the infection are moving around and thus are acquiring new sets of connections (this is perhaps the more common case, in the case of people, e.g., involving long distance flights, etc.). The latter case has two variants <a href="#pone.0015571-Wang1">[5]</a>, the units only diffusing around, or, in the case of birds or people, they can make huge distances in a short time so that the spatial aspect is less relevant <a href="#pone.0015571-Brockmann1">[6]</a>–<a href="#pone.0015571-Ferguson1">[8]</a> and a network description becomes more appropriate <a href="#pone.0015571-Hu1">[16]</a>, <a href="#pone.0015571-Colizza1">[17]</a>. The former case applies to serious diseases caused by microorganisms such as Burkholderia spp., Listeria monocytogenes <a href="#pone.0015571-Tilney1">[18]</a>, Mycobacterium marinum <a href="#pone.0015571-Stamm1">[19]</a>, Shigella spp. <a href="#pone.0015571-Heindl1">[20]</a>, and Rickettsia spp <a href="#pone.0015571-Teysseire1">[21]</a>. These bacterial pathogens enter host cells and spread through tissues by moving directly from one cell into adjacent cells using an actin polymerization driven tail propelling them inside the cytoplasm. The <i>in vitro</i> spreading of Listeria <a href="#pone.0015571-Marquis1">[14]</a> and Shigella <a href="#pone.0015571-Way1">[22]</a> in 2D cell cultures was shown to lead to circular or more complex plaques. Finally, a standard paradigm used to explain infection spreading through a percolation-like process involves a forest in which some of the trees become infected by a pathogen that can spread to a neighboring tree only with a probability smaller than 1.</p> <p>In order to handle the above mentioned constraints we have designed experiments allowing us to study in detail the outbreak dynamics and the patterns of infection spreading in a system containing tens of thousand of essentially static units. The progression of the epidemics was followed by both making video recordings and high resolution photos (i.e., tracking the infection by both GFP expression and making preparations with various selective, immuno-histochemistry based stainings). With the help of a computerized system at each given time 3×3 = 9, 4×4 = 16 (movies) and 10×10 = 100 (microscopic fields) images were stitched together to obtain uniquely high resolution (e.g., the merged pictures of the stained plates have appr. 13000×9000, i.e., around 100 million pixels.).</p> <p>To interpret our observations we use models, quantities and expressions primarily associated with percolation theory <a href="#pone.0015571-Stauffer1">[23]</a>. This theory predicts that for the simplest version of a growing percolation process (in which occupied sites are randomly added to an initially empty set of lattice sites), close to the transition (or critical) point, when the size of the largest cluster becomes compatible with the system size, the distribution of the cluster sizes follows a power law with corrections. In particular, if n<sub>s</sub> denotes the normalized number of clusters as a function of their size s than </p><div class="disp-formula" title="disp-formula"> <div class="graphic" mimetype="image" position="float" /> </div>where p is the proportion of occupied sites, p<sub>c</sub> is the critical proportion (or probability of being occupied), <i>σ</i> is a positive number and f(x) is a scaling function being about constant for x&amp;lt;&amp;lt;1 and exhibiting a fast (exponential) decay for large values of its argument. Thus, close to the transition point (p is about p<sub>c</sub>) n<sub>s</sub> is expected to exhibit as a power law decay with an exponent <i>−τ</i> followed by an exponentially fast cut-off depending on how far the system is from its critical point. We test the validity of this and other related predictions of the theory and build and simulate a model which is in agreement with the experiments.<p /> <p>In short, i) our study is aimed at exploring a common and important but yet not fully documented phenomenon, ii) we use an image processing technology far beyond those used in the present context before and iii) we successfully interpret our findings using numerical simulations.</p> </div> <div class="results" title="sec"> <div class="title" tagx="title" title="title">
Results</div> <p>Our experimental setup (<a href="#pone-0015571-g001"><b>Fig. 1</b></a>) was based on a tissue cell culture (confluent monolayer of astrocytes) exposed uniformly to a recombinant pseudorabies virus (PRV) Ba-DupGreen (BDG), which expresses green fluorescent protein (GFP) in the infected cells with immediate early kinetics. We have chosen astrocytes from a number of cell types examined in preliminary experiments, as these cells show restricted release of infectious viral particles whereas the spread of infection among interconnected cells is not compromised <a href="#pone.0015571-Card1">[24]</a>.</p> <div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/10.1371/journal.pone.0015571.g001"><span class="label" tagx="label" title="label">Figure 1</span></a></div>   <div class="title" tagx="title" title="title">
Flow diagram of the experimental design.</div> <p>All steps are discussed in details throughout the text as well as in <a href="#s4">Materials and Methods</a>.</p>   </div> <p>First, we initiated productive infection provoking GFP expression in a limited number of cells (see <a href="#s4"><b>Materials and Methods</b></a>). After removing the infective viral particles from the culture medium (<b><a href="#pone.0015571.s006">Table S1</a></b>) we obtained an extensive series of images of the developing epidemics (growing clusters of infected cells; <a href="#pone-0015571-g002"><b>Fig. 2</b></a>). These images, especially after computer aided processing and analysis revealed a set of highly complex geometrical patterns. In <a href="#pone-0015571-g003"><b>Fig. 3</b></a> we displayed a representative example showing the rich spatial structure of the infected zones close to a point when the largest cluster spans the whole culture. An approximate self-similarity could be observed being visually demonstrated in the figure by including consecutively enlarged parts of the original image (supporting information <b><a href="#pone.0015571.s001">Fig. S1</a></b>). Since the images we obtained were reminiscent of those generated by simple statistical physics models of percolation, we determined both the distribution of the cluster sizes and the fractal dimension of the largest clusters (<a href="#pone-0015571-g004"><b>Fig. 4</b></a><b> and </b><a href="#pone-0015571-g005"><b>Fig. 5</b></a>, respectively). These two quantities are considered to be the most characteristic features of percolating systems and are known to be universal over a wide selection of model variants. Indeed, our experimental images also displayed features very much like percolation models, i.e., the largest cluster had a well defined fractal dimension and the cluster size distribution function could be interpreted as an algebraically decaying function with some characteristic modifications (<a href="#pone-0015571-g006"><b>Fig. 6</b></a>).</p> <div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/10.1371/journal.pone.0015571.g002"><span class="label" tagx="label" title="label">Figure 2</span></a></div>   <div class="title" tagx="title" title="title">
Mosaic images of astrocytic cultures.</div> <p>The cultures were fixed 18 h (<b>A</b>), 30 h (<b>B</b>) and 42 hours (<b>C</b>) after infection with the BDG-PRV virus. In the bottom row (<b>D</b>) higher magnification images can be seen taken 36 hours after infection. The last image shows accurate overlap between immunostained PRV virus particles (red) and GFP (green) expession by infected cells. Scale bars are 300 µm and 25 µm, respectively. High resolution file can be downloaded from: <a href="http://amur.elte.hu/BDGVirus/">http://amur.elte.hu/BDGVirus/</a>.</p>   </div> <div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/10.1371/journal.pone.0015571.g003"><span class="label" tagx="label" title="label">Figure 3</span></a></div>   <div class="title" tagx="title" title="title">
An image processed representation of a large infected region of the monolayer of astrocytes.</div> <p>100 (10×10) microscopic field images were taken from fixed cultures with GFP expressing infected cells and stitched together to obtain a high resolution single picture (approximately having 13,000×9,000 pixels). Then, the originally black and white image was analysed by finding clusters of cells (connected regions of larger than 1000 pixels: which number was associated with the average cell size). Next, each separate cluster was coloured by choosing a colour from a random set of 8 possible ones. The obtained clusters and their perimeters display a highly complex spatial structure reminiscent of mathematical fractals and possessing some level of self-similarity illustrated by sequential magnification of parts of the structure. Scale bar: 100 µm. (See original image in <b><a href="#pone.0015571.s001">Fig. S1</a></b>.)</p>   </div> <div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/10.1371/journal.pone.0015571.g004"><span class="label" tagx="label" title="label">Figure 4</span></a></div>   <div class="title" tagx="title" title="title">
The distribution of cluster sizes.</div> <p>Panels <b>A</b>, <b>C</b> and <b>E</b> show the cluster size distribution after 18, 30 and 42 hours, respectively, as determined from a series of pictures analogous to <b><a href="#pone.0015571.s001">Fig. S1</a></b>. Corresponding cluster size distribution of the model is presented in panels <b>B</b>, <b>D</b> and <b>F</b>. Experimental graphs are consistent with the predictions of our percolation based simple spreading model: both the experimental scaling behaviour at 42 hours and the spreading of the infection before that are well characterized by the model. A scaling (power law) distribution is built up in time, i.e., the number of infected clusters <i>vs</i> cluster size follows a power function instead of an exponential drop. Model time was scaled to fit to the infection spreading rate of high titer experiments.</p>   </div> <div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/10.1371/journal.pone.0015571.g005"><span class="label" tagx="label" title="label">Figure 5</span></a></div>   <div class="title" tagx="title" title="title">
Determination of the fractal dimension of the largest clusters.</div> <p>This figure (squares) shows the number of cells in the largest clusters of infected cells as a function of the linear size (L) of the regions in which they were counted. The fitted line indicates that the geometry of the largest infected regions is analogous to those of fractals with a dimension of about 1.78. Remarkably, the simulation data (crosses) are almost indistinguishable from those obtained from the observations.</p>   </div> <div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/10.1371/journal.pone.0015571.g006"><span class="label" tagx="label" title="label">Figure 6</span></a></div>   <div class="title" tagx="title" title="title">
Results from the simulations of our model.</div> <p>A representative set of clusters (bottom 20% of the original picture has been omitted to save space) is displayed (top, <b>A</b>) for the following parameters: size 500×500 lattice units, 500 starting seed particles with a "switching on" time distribution following a lognormal fitted to the experimentally observed parameters (mean = 7.5 h, standard deviation = 2.5 h) plus 2000 further seeds with a constantly increasing rate in time between 20 h and 42 h (corresponding to the observed second wave of infection, see <b><a href="#pone.0015571.s004">Fig. S4</a></b>). The implemented values of the two model parameters were ρ = 0.0182 and α = 3. The bottom left (<b>B</b>) figure is a schematic demonstration of the model where the coloured circles correspond to the experiment as follows: black dots → not susceptible or missing cells; red dots → primarily infected cells; brown dots → cells infected by one of their neighbours; yellow dots → cells potentially infected in the next time step; green hollow: a cluster of infected cells. The bottom right (<b>C</b>) plot demonstrates that the cluster size distribution in the model also displays a scaling behaviour for the parameters corresponding to the experimental conditions (the same as for the picture in the top).</p>   </div> <p>Associating the infected cells with the occupied sites, we saw that the observations are rather reminiscent of the extremely simple percolation model. There were some differences, however, to be discussed later. The value of approximately 1.7 we found for the exponent <i>τ</i> (<a href="#pone-0015571-g004"><b>Fig. 4</b></a>) defined in the <a href="#s1">introduction</a> is consistent with both the value obtained from our simulations (approximately 1.65) and with the predictions for the distribution of cluster sizes in systems under a non-equilibrium, cluster growth regime (in equilibrium systems <i>τ</i>&amp;gt;2) <a href="#pone.0015571-Stauffer1">[23]</a>.</p> <p>In addition to static configurations, we have also investigated the dynamics of the infection spreading in our tissue cultures (<a href="#pone-0015571-g007"><b>Fig. 7</b></a> and supplementary <b><a href="#pone.0015571.s008">Videos S1</a></b>, <b><a href="#pone.0015571.s009">S2</a></b>, <b><a href="#pone.0015571.s010">S3</a></b>, <b><a href="#pone.0015571.s011">S4</a></b>, <b><a href="#pone.0015571.s012">S5</a> and <a href="#pone.0015571.s013">S6</a>.</b>). Long term video microscopy data have enabled us to keep track of the number of infected cells after all of them had been exposed to a uniformly distributed infection for a short (as compared to the experiments' time scale) time interval. The Supplementary <b><a href="#pone.0015571.s012">Video S5</a></b> clearly shows that the individual astrocytes witin the monolayers do not shift from their original position more, than a fraction of their diameter, within a considerable amount of time (more than 24 h).</p> <div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/10.1371/journal.pone.0015571.g007"><span class="label" tagx="label" title="label">Figure 7</span></a></div>   <div class="title" tagx="title" title="title">
Images demonstrating the spreading of the infection.</div> <p>Screenshots (<b>A–D</b>) and graphical representations (<b>E</b>,<b>F</b>) of videos on virus propagation within astroglial cultures treated with BDG-PRV virus. Supplementary <b><a href="#pone.0015571.s008">Videos S1</a></b>, <b><a href="#pone.0015571.s009">S2</a></b> and <b><a href="#pone.0015571.s010">S3</a></b> were recorded using cultures infected with different titers showing virus spread on large scale. Titers in these experiments were <b>A</b>) 2,5×10<sup>4</sup> PFU/ml ("low" titer) <b>B</b>) 2.5×10<sup>5</sup> PFU/ml ("high" titer) <b>C</b>) 1,25×10<sup>6</sup> PFU/ml ("highest" titer). Image series <b>D</b>) and <b>E</b>) and supplementary <b><a href="#pone.0015571.s011">Videos S4</a> and <a href="#pone.0015571.s012">S5</a></b> show the appearance of GFP in "low titer" infected cells over time within a representative microscopic field. Formation of cell clusters with more advanced infection can be seen in image series <b>F</b>) and <b>Video 6,</b> representing 9 stiched microscopic fields. Red cells indicate the foci of the developing clusters. High resolution videos and images can be downloaded from: <a href="http://amur.elte.hu/BDGVirus/">http://amur.elte.hu/BDGVirus/</a>.</p>   </div> <p>We have determined the rate by which cells acquire a "primary" infection (directly related to the initial exposure), and also the dynamics by which the number of infected cells grows as a result of "secondary" infections (infection due to being a neighbour of an already infectious cell; <b><a href="#pone.0015571.s007">Table S2</a></b>, <b><a href="#pone.0015571.s002">Fig. S2</a>)</b>. The motion picture recordings allowed us to gather some unique statistical data about the time development aspects of the phenomenon as well (<a href="#pone-0015571-g008"><b>Fig. 8</b></a>).</p> <div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/10.1371/journal.pone.0015571.g008"><span class="label" tagx="label" title="label">Figure 8</span></a></div>   <div class="title" tagx="title" title="title">
Visualization of the spreading of the infection around a single infected seed.</div> <p>The upper picture (<b>A</b>) demonstrates the progression of the infection around a single infected cell (see <b><a href="#pone.0015571.s011">Videos S4</a></b>, <b><a href="#pone.0015571.s012">S5</a></b>, <b><a href="#pone.0015571.s013">S6</a>.</b>). The colour coding is such that lighter colours correspond to later times and the movie frames (also image processed here) were taken at time 15, 18, 21, 24 hours after the initial exposure to the virus. The bottom plot (<b>B</b>) shows the probability distribution function for a single isolated cell (playing the role of a seed for the infection spreading) to become infected. Fitted is a log-normal function giving a significantly better agreement than a Gaussian. Scale bar: 50 µm.</p>   </div> <p>Next, in order to both qualitatively and quantitatively interpret our large scale spatio-temporal epidemiological data, we have designed a model. This approach enabled us to estimate to what degree the global behaviour is sensitive to details of the processes taking place on the local (cellular) level. Our model was, on one hand based on the simplest prior epidemic models (percolation), on the other hand it took into account some of the specifics of the present experiment in a new way. The two major features of the model were: i) neighbours (of the already infected cells) were infected with a probability taken from an algebraic (non-linear) distribution, ii) a secondary wave of infection was represented by the gradual introduction of new infectious sites (see <b><a href="#pone.0015571.s007">Table S2</a></b> and <a href="#s4">MATERIALS AND METHODS</a> for details). These simulations allowed us to calculate the same quantities which we have determined for the experimental images and thus, a quantitative comparison could be made concerning the essential parameters of the epidemics.</p> <p>We expected that a cluster grown from a single seed would have had a well defined fractal dimension <a href="#pone.0015571-Vicsek1">[25]</a>. However in our case there were many clusters in the system growing simultaneously and sometimes these clusters merged. This second mechanism would have changed the fractal dimension of the formed clusters considerably. The merging of the clusters could be observed also in the experiment. The simulation was stopped when the ratio of the infected cells reaches a given value calculated from the experimental results. After this point the infected cells began to die. A typical set of clusters obtained for parameters corresponding to the experiments is shown in <a href="#pone-0015571-g006"><b>Fig. 6</b></a> with the different clusters having selected colours by size from a table of eight. The dynamics of cluster growth of the model can be seen in supplementary <b><a href="#pone.0015571.s014">Video S7</a></b>.</p> </div> <div class="discussion" title="sec"> <div class="title" tagx="title" title="title">
Discussion</div> <p>Our infection spreading experiments have been possible because the viral particles are able to spread within the compromised cell and enter its processes. Newly formed virions have the potential to infect nearby cells following exocytosis from the infected cell and a subsequent uptake through the cell membrane. In the case of productive infection, virus containing cells produce sufficient progeny virions to infect their surroundings and to initiate a focal spread of infection over time. Compromised (but viable) as well as dying cells are also capable of releasing infectious virus particles into the media, which may attach to and infect further cells from a large distance to the place of the original infection <a href="#pone.0015571-Granzow1">[26]</a>–<a href="#pone.0015571-Garner1">[31]</a>.</p> <p>Detailed documentation of the spreading of viral infection in a layer of many thousands of cells has been carried out in a very limited number of papers only <a href="#pone.0015571-Lam1">[32]</a>, <a href="#pone.0015571-Duca2">[33]</a>. These prior investigations involved systems in which an infected cell was likely to infect all of its neighbours in a time scale much shorter then that of the whole experiment. In addition, the infected cells introduced back to the medium significant amounts of viral particles. Under these conditions the infected regions of the culture possess interesting ring-like or elongated shapes, allowing quantitative analysis of some of the features of the propagating zone of infection. We observed such convection current-induced disturbance of the infection spread in the case of some cell types used in preliminary experiments, such as swine testis cells (ST) or canine (MDCK) and human kidney cells (HEK293) (data not shown). The phenomena we observed for the above cell types were very much like those described in refences <a href="#pone.0015571-Appleyard1">[34]</a>–<a href="#pone.0015571-Zhu1">[36]</a>. In the case of the PRV virus versus astrocytes system, however, neither the infection of all of the neighbours nor the secondary infection of the medium played a dominant role and, in fact, this is why we were able to observe the growth of percolation-cluster-like infected regions. In contrast to the cell lines mentioned above, which - due to rapid cell lysis - released a very high titer of infectious viral particles into the culture medium (data not shown), low virus titers in the supernatant of infected astrocytes did not lead to convection current-induced disturbance of the infection spread and, in line with simulation data, the growth of newly formed clusters after 18 hours of infection did not have a relevant impact on the percolation-type growth. Another possible advantage of using primary astrocytes over cell lines may be that due to cell to cell contact inhibition, the proliferation of primary astrocytes is much slower than that of cell lines. This made possible to image these cultures even beyond 48 hours, without the disturbing effects of an over-confluent multilayer, which would also involve significant cell death.</p> <p>Several biological and physical parameters are known to affect the cells' susceptibility to infection and/or lead to inhomogeneous spreading of infection in both <i>in vivo</i> and cell cultures. Variability in cell cycle (both in cell lines and primary cells), expression levels and availability of different virus receptors, the variable activation state of individual cells, changes in fluid flow and diffusion can all contribute to changes in spreading characteristics <a href="#pone.0015571-Fitting1">[37]</a>–<a href="#pone.0015571-Khlifa1">[41]</a>. In addition, in our case the actual geometry of the astrocytes the varying width and extension of their processes are likely to result in a highly variable number of contacts (through which the infection can take place) between them. The non-linear term (power law) we use in our model can account for such type of details because it corresponds to a situation, in which the probability of becoming infected in the next time step by a neighbour is not scattered around a well defined average but is very unevenly distributed.</p> <p>Based on our experimental observations and the above description the following scenario can be proposed for the large scale spreading in a situation when i) the susceptible units are spatially fixed ii) the process of spreading is highly inhomogeneous. The inhomogeneities are likely to be due to the highly variable geometry of the individual cells as well as their log-normally distributed susceptibility to become infected after a sudden initial exposure to infection. We find that the actual spreading is a combination of the growth of individual plaques around their corresponding "seed" (initially infected cell) and the subsequent fusion of these plaques. However, the plaques themselves have a complex geometry since the infection rate of a cell is highly varying from cell to cell. In such situations, many small and a fewer (but not exponentially decaying number of) larger connected infected regions are expected to appear in the system. All this can be interpreted in terms of simultaneously growing percolation clusters. The model we proposed is extremely simple and is designed to account for the above experimental details. Both the visual appearance and the quantitative statistical features of the simulated clusters agree reasonably well with the observations.</p> <p>Since our model possesses general features of live systems and takes into account the specificities of the currently used astroglial cell culture to a minimal degree only, we are motivated to suppose that our observations are likely to be applicable to a very wide variety of experimental situations being dominated by the infection taking place at spatially scattered points and spreading to neighbours with a rate highly varying from one susceptible units to another. In such cases the largest infected regions are likely to have fractal geometry and the size distribution of the clusters follow a scaling law. Such features have long been suggested by population dynamics models but here we present the first large scale experimental verification of the conjectured behaviours.</p> <p>An interesting extension of the present work would be to consider cell populations in which the migration of the infectious cells cannot be ignored. In principle, even the network aspects of infection spreading could be studied in a similar setup with cells having very long processes capable of reaching out to and through them eventually infect distance cells.</p> </div> <div class="materials_methods" title="materials|methods"> <div class="title" tagx="title" title="title">
Materials and Methods</div> <div class="ethicsstatement" title="sec"> <div class="title" tagx="title" title="title">
Ethics Statement</div> <p>This study was carried out in strict accordance with 86/609/EEC/2 Directives of European Community. Based on the permissions issued by the Food and Animal Health Authority, Budapest, Hungary, under the licenses No.: 2296/003/2006-2011 and 3668/003/2008-2014, the Animal Care and Experimentation Committee of the Institute of Experimental Medicine, Hungarian Academy of Sciences approves investigations, such as ours, complying with the above licenses during their period of validity.</p> </div> <div class="materials" title="sec"> <div class="title" tagx="title" title="title">
Materials</div> <p>Trypsin, DNase I, poly-L-lysine, MEM, foetal calf serum, glutamine, Hoechst 33342 nuclear dye and anti-GFP antibody were purchased from Sigma-Aldrich Kft. Hungary; tissue culture dishes either from Greiner Bio One Co., Hungary or IBIDI GmbH, Germany. CellMask™ Orange, anti-rabbit-Alexa594 and anti-mouse-Alexa488 fluorescent dyes were obtained from Invitrogen Co., Hungary. Primary antibody to PRV virion proteins was a kind gift of Prof. Lynn Enquist.</p> </div> <div class="experimentalsetup" title="sec"> <div class="title" tagx="title" title="title">
Experimental setup</div> <p>The Methods section follows the Experimental setup explained in the Flow diagram of <a href="#pone-0015571-g001"><b>Fig. 1</b></a>.</p> </div> <div class="astroglialcultures" title="sec"> <div class="title" tagx="title" title="title">
Astroglial cultures</div> <p>Astrocytes were isolated from whole brains of neonatal (P0-P3) mice, as described earlier <a href="#pone.0015571-Krnyei1">[42]</a>. In brief, meninges were removed and the tissue pieces were subjected to enzymatic dissociation, using 0.05% w/v trypsin and 0.05% w/v DNase for 10 minutes at room temperature. The cells were plated onto poly-L-lysine coated plastic surfaces and were grown in Minimal Essential Medium supplemented with 10% foetal calf serum, 4 mM glutamine and 40 µg/ml gentamycin in humidified air atmosphere containing 5% CO<sub>2</sub>, at 37°C. The culture medium was changed on the first two days and every second or third day afterwards. Confluent primary cultures were harvested by trypsinization and re-plated onto pLL coated glass coverslips or into petri dishes, according to the actual experimental design. Secondary astrocytic cultures reaching confluency and displaying mosaic-like pattern were infected with PRV viruses, as it follows.</p> </div> <div class="viralinfection" title="sec"> <div class="title" tagx="title" title="title">
Viral infection</div> <p>We aimed to choose a virus strain for the experiments, which has relatively slow and well-controlled spreading characteristic in order to analyse the evolution of plaque size and distribution very precisely over time. To this end, Ba-DupGreen (BDG), a recombinant PRV-Bartha-derived pseudorabies virus was used. The virus genome was altered by insertion of the green fluorescent protein (GFP) gene expression cassette to the putative antisense promoter (ASP) located at the inverted repeat region of the virus <a href="#pone.0015571-Boldogki1">[43]</a>. Due to this modification, BDG multiplicates with a significantly lower rate <i>in vitro</i> and infects fewer numbers of cells <i>in vivo</i> than the parent PRV-Ba strain <a href="#pone.0015571-Boldogki2">[44]</a>. The reporter gene was placed under the control of the human cytomegalovirus major immediate early (IE)1 promoter (CMVP). Therefore, GFP was expressed at a very early stage of infection in the compromised cells that made possible to precisely follow the spread of infection over time. The detailed explanation of the mechanisms underlying the gene expression profile and the spreading kinetics of the virus was described previously <a href="#pone.0015571-Boldogki1">[43]</a>, <a href="#pone.0015571-Boldogki2">[44]</a>. The virus was grown in porcine kidney PK-15 cells to a titer of 6×10<sup>8</sup> plaque-forming units (PFU)/ml and was further concentrated to 1.5×10<sup>9.8</sup> PFU/ml by ultracentrifugation at 70,000×<i>g</i>. Virus stocks were aliquoted and stored at −80°C. Aliquots were thawed immediately prior to infection of astrocyte cultures. During preliminary experiments, the appropriate virus titer to obtain a low number of plaques was determined. The astroglia cultures were infected with BDG at a final titer of 2.5×10<sup>5</sup> PFU/ml ("high" titer) or 2,5×10<sup>4</sup> PFU/ml ("low" titer). The cells, grown in 35 mm diameter Petri dishes were incubated with 1 ml of the virus containing medium, for 1 h, at room temperature. The multiplicity of infection (MOI) was ∼0,17 or 0,017 PFU/cell, respectively.</p> </div> <div class="removalofinfectiveviralparticlesfromtheculturemedium" title="sec"> <div class="title" tagx="title" title="title">
Removal of infective viral particles from the culture medium</div> <p>In order to remove infective virus particles from the culture medium we washed all the cultures at least three times after the 1 h viral exposure. To check whether three changes of medium was sufficient to completely remove viruses from the supernatant we transferred the first, second and third washes to intact cultures. 24 hours after either a 1 hour or a day long incubation period we fixed the cells and counted the GFP expressing plaques or cells within 30-30 random microscopic fields in each dish. Both the size of the plaques and the number of infected cells or cell groups was strongly reduced due to the washing procedure. The data indicated that 3 washes were sufficient to remove the "original" viral particles from the infected cultures (see supplementary <b><a href="#pone.0015571.s006">Table S1</a></b>).</p> </div> <div class="releaseofinfectiveviralparticlestotheculturemediumbyastrocytes" title="sec"> <div class="title" tagx="title" title="title">
Release of infective viral particles to the culture medium by astrocytes</div> <p>To determine, whether the virus-treated cells released infective virus particles into the medium during culture maintenance we analyzed the infective capacity of the culture media removed after 1, 5, 24, 30 or 48 hour long incubation periods. Equal volumes of the media were transferred to non-treated sister cultures, which were further incubated for either 1 or 24 hours. Finally, we fixed the cells and counted the GFP expressing plaques or cells within 30-30 random microscopic fields on each coverslip (duplicates). The data indicate that ∼24 hrs after the infection astrocytes began to release infective viral particles to the culture medium (<b><a href="#pone.0015571.s007">Table S2</a></b>).</p> </div> <div class="determinationofthenumberofneighbouringastrocytes" title="sec"> <div class="title" tagx="title" title="title">
Determination of the number of neighbouring astrocytes</div> <p>To determine the average number of neighbouring astrocytes we stained the cultures both with CellMask™ Orange, a fluorescent plasma membrane stain of living cells, and Hoechst-33342, a membrane-permeable nuclear dye. With the help of the visualized cell contours and the nuclear stainings we determined the average number of the neighbours of any individual astroglial cell (n = 100) (<b><a href="#pone.0015571.s003">Fig. S3</a></b>). These data were further used in the development of the virus propagation model.</p> </div> <div class="immunocytochemistryandimageprocessingoffixedcultures" title="sec"> <div class="title" tagx="title" title="title">
Immunocytochemistry and image processing of fixed cultures</div> <p>We used a Zeiss Axiovert 200 M microscope equipped with AxioCamMrm to make images of astrocytic cultures fixed at various moments (3, 6, 9, 12, 18, 24, 30, 36, 42, 48, 72 hours) after infection. For detailed analyses we used 2-2 parallel cultures infected either with low or high virus titers. Immunocytochemical stainings were performed as described earlier <a href="#pone.0015571-Krnyei1">[42]</a>. In the present work, primary antibodies to structural virus proteins (PRV, rabbit, kind gift of Prof. Lynn Enquist) and to GFP (mouse) were used in 1∶1000 and 1∶2000 dilutions, respectively. As secondary antibodies anti-rabbit-Alexa594 and anti-mouse-Alexa488 fluorescent dyes were used in a dilution of 1∶1000. DAPI was used to label cell nuclei. The stainings revealed that GFP expression within the infected cells overlaps with the PRV immunostaining. From each culture we made 10*10 overlapping images in four channels (DAPI, GFP, PRV and Phase contrast). The background of each microscopic field was subtracted with the Subtract Background method of the ImageJ <a href="#pone.0015571-ImageJ1">[45]</a> software using a rolling ball radius of 300 <a href="#pone.0015571-Sternberg1">[46]</a>. Before analyzing we manually stitched the images together using the Photoshop 7.0 software and smoothed them running 3 times the Smooth algorithm of the ImageJ software to decrease noise. This filter replaces each pixel with the average of its 3×3 neighborhood. After stitching the whole field of view was as large as 13000*10000 pixels (8,5*6,5 mm) containing approx. 100–150 000 astrocytes. Virus infection of single cells monitored by GFP expression develops in time with an increasing fluorescent intensity. Depending on the initial expression level of infected cells and the sensitivity of the detection system an intensity threshold has to be declared in order to decide whether a cell is already infected. On the basis of visual image analysis we tuned the threshold to find all infected cells even in a very early state of infection appearing brighter than the background noise. As a result we gained a binary black and white image. (See <a href="#pone-0015571-g003"><b>Fig. 3</b></a>)</p> </div> <div class="realtimemonitoringofinfectionspreading" title="sec"> <div class="title" tagx="title" title="title">
Real time monitoring of infection spreading</div> <p>Dynamics of virus propagation was monitored in astroglial cultures treated with different titers of BDG virus. GFP expression within infected cells allowed the real time monitoring of the spread of infection. Time-lapse recordings were performed on a computer-controlled Leica DM IRB inverted microscope equipped with 10x and 20x objectives and an Olympus DP70 cooled CCD camera. We used Märzhäuser Scan IM 120×100 motorized stage and fine focus motor. A maximum number of 4 cell cultures in 35 mm Greiner or Ibidi Petri dishes were kept at 37°C in a humidified 5% CO2 atmosphere within a CellMovie microscope stage incubator. For time-lapse imaging we used the CellMovie control software (<a href="http://cellmovie.eu">http://cellmovie.eu</a>). Phase-contrast and fluorescent images were acquired every 10–20 min for 2–5 days. We scanned 3×3 or 4×4 adjoining fields of view of cell cultures in every cycle of the time-lapse program. Automated focus correction was carried out in phase contrast mode in each time-lapse cycle.</p> </div> <div class="analysisofviruspropagationdynamics" title="sec"> <div class="title" tagx="title" title="title">
Analysis of virus propagation dynamics</div> <p>The image processing of the movies was carried out in a partially different way. After the experiment the background of each microscopic field was subtracted with the Subtract Background method of the ImageJ software using a rolling ball radius of 300. To gain mosaic-like large scale jpeg images from the 3×3 or 4×4 microscopic fields we stitched them on the basis of the <i>xy</i> coordinates of the motorized microscope stage. Stitched jpeg images were also converted to mpeg videos with the help of a Linux script using the mjpegtools graphical software package. See supplementary <b><a href="#pone.0015571.s008">Videos S1</a></b>, <b><a href="#pone.0015571.s009">S2</a></b>, and <b><a href="#pone.0015571.s010">S3</a></b>. The dynamics of virus spreading is considered to be unaffected by the intensity threshold described above if we apply a constant value for all the consecutive time-lapse images: increasing or decreasing the threshold in the range of early infection results only in a time shift of virus spreading. To test whether the intensity threshold influences our result<b>s</b> we applied different values and found the same behaviour of graphs. Microscopic fields in each culture were than analyzed with the WTrack software developed by B. Gönci (unpublished). The geometric centres of GFP+ cells were tagged manually one by one on the basis of visual detection resulting in the xy position of infected cells in each time-lapse frame during the first 24 ("high" titer) or 30 hours ("low" titer). To get maximal precision we used Wacom Intuos Pen Tablet for cell tracking. The number of infected cells grew in a quasi-exponential manner. These data were used in the development of the virus propagation model. See supplementary <b><a href="#pone.0015571.s011">Videos S4</a></b>, <b><a href="#pone.0015571.s012">S5</a></b>, <b>and <a href="#pone.0015571.s013">S6</a></b>.</p> </div> <div class="simulations,modeling" title="sec"> <div class="title" tagx="title" title="title">
Simulations, modeling</div> <p>The model we designed to simulate infection spreading is based on the combination of the well known concepts of percolation and the SI or SIS (susceptible-infected-susceptible) type epidemiological models [1–2.] or, in other words, it is a simple infection spreading model in an inhomogeneous environment with a simple assumption about the distribution of the susceptibilities of the individual cells. Our goal was to reproduce the dynamics of infection spreading and the structure and shape of the resulting clusters of infected cells as observed in the experiments using a model as simple (thus, as universal) as possible. The simulations have been carried out on a hexagonal lattice to minimize the effect of lattice geometry-dependent, preferred orientations. The time during the simulation proceeds in discrete steps. In our case a single step corresponds to one minute in the experiment.</p> <div class="virusspreadingwasmodelledbymontecarlosimulation" title="sec"> <div class="title" tagx="title" title="title">
Virus spreading was modelled by Monte Carlo simulation</div> <p>1. There are two stages when infected cells appear because of the infection from the surrounding medium. At the beginning of the experiment the whole system is exposed to the infection uniformly, thus, infected cells appear randomly in time and space according to a rate taken as input from the experiments. The number of cells became infected due to the initial exposure to the infection was approximated from <a href="#pone-0015571-g008"><b>Fig. 8B</b>:</a> the frequency with which the randomly positioned infected cells appear during this stage can be well approximated with a lognormal distribution. This is the dominant mechanism for infection from the medium. At later stages cells (primarily during lysis) release virus particles into the medium resulting in a constantly increasing, but relatively low titer, leading to a second stage of infection beginning at around 18 hours (See also the increase in the number of appearing clusters at around 18 hours in <a href="#pone-0015571-g008"><b>Fig. 8</b></a>). This stage was approximated by a linearly increasing probability that new, infected cells will appear, starting at 20 hours, and lasting until 42 hours, when the simulation was stopped, because by this time the death rate of cells becomes non-negligible. To account for the death of the cells a more complicated model would be needed, and is out of the scope of this study.</p> <p>2. In every simulation step, the yet healthy immediate neighbours of the set of already infected cells are considered for possible infection. Cell A infects its not yet infected neighbour B in a certain cycle of the simulation if a random number r chosen from the [0,1] interval in this cycle is smaller than the P<sub>i</sub> susceptibility value of cell B. Susceptibility value of each cell is chosen randomly and fixed at the beginning of the simulation: <span class="inline-formula" title="inline-formula"><span class="inline-graphic" title="inline-graphic" /></span>. Here p<sub>i</sub> is a random number generated from a random variable X having continuous uniform distribution on the [0,1] interval. The random variable <span class="inline-formula" title="inline-formula"><span class="inline-graphic" title="inline-graphic" /></span> will have a probability density function of </p><div class="disp-formula" title="disp-formula"> <div class="graphic" mimetype="image" position="float" /> </div>on the [0,1] interval, zero everywhere else. This distribution infers that the low value of susceptibility is more frequent than the high value if α&amp;gt;1. Either power law or lognormal distribution, the final results are very similar. The constant ρ was chosen (fitted to the observations) in such a way that the largest cluster of infected cells percolated at about the same time as in the experiments. (see supplementary <b><a href="#pone.0015571.s004">Fig. S4</a></b> and <b><a href="#pone.0015571.s014">Video S7</a></b>). The picturesque meaning is <span class="inline-formula" title="inline-formula"><span class="inline-graphic" title="inline-graphic" /></span>, where dt is the time step of the simulation, T is the time needed to infect the most susceptible cells having an infected neighbour. Parameter <i>α</i> was tested for different integer values from 0 to 5, and the one for which the results had the best overall agreement with the experiment was used in the final model (α = 3).<p /> <p>To minimize the effect of the system's finite size, the number of particles was double of the number of cells' appearing on the stitched experimental image [<a href="#pone-0015571-g003">Fig. 3</a>]. The fractal dimension (<a href="#pone-0015571-g005"><b>Fig. 5</b></a>) and cluster size distribution (<a href="#pone-0015571-g004"><b>Fig. 4</b></a><b> and </b><a href="#pone-0015571-g006"><b>Fig. 6</b></a>) are averaged results of 10 distinct simulations, done with the same parameters, in order to avoid accidental good/bad agreement with the experiment due to the variance of the simulation results. The parameters of the model are summarized in <a href="#pone-0015571-t001"><b>Table 1</b></a>.</p> <div class="table-wrap_UNKNOWN" id="pone-0015571-t001" position="float"><a href="https://dx.doi.org/10.1371/journal.pone.0015571.t001">10.1371/journal.pone.0015571.t001</a><span class="label" tagx="label" title="label">Table 1</span> <div class="title" tagx="title" title="title">
The main parameters of the simulation.</div>  <div class="alternatives" title="alternatives"> <div class="graphic" id="pone-0015571-t001-1" /> <table frame="hsides" rules="groups"> <colgroup span="1"> <col align="left" span="1" /></colgroup> <colgroup><col align="center" span="1" /></colgroup> <colgroup><col align="center" span="1" /></colgroup>  <thead> <tr> <td align="left" rowspan="1" colspan="1">Parameter type</td> <td align="left" rowspan="1" colspan="1">Value</td> <td align="left" rowspan="1" colspan="1">Mode of determination</td> </tr> </thead> <tbody> <tr> <td align="left" rowspan="1" colspan="1">System size</td> <td align="left" rowspan="1" colspan="1">500×500 particles in a hexagonal lattice</td> <td align="left" rowspan="1" colspan="1">Double of the size of the partial system captured by the images</td> </tr> <tr> <td align="left" rowspan="1" colspan="1">Number of particles appearing during the first stage of infection</td> <td align="left" rowspan="1" colspan="1">500</td> <td align="left" rowspan="1" colspan="1">Approximated from <a href="#pone-0015571-g008"><b>Fig. 8</b></a></td> </tr> <tr> <td align="left" rowspan="1" colspan="1">Number of particles appearing in the second stage</td> <td align="left" rowspan="1" colspan="1">2000</td> <td align="left" rowspan="1" colspan="1">Best fit</td> </tr> <tr> <td align="left" rowspan="1" colspan="1"><i>ρ</i></td> <td align="left" rowspan="1" colspan="1">0.0182</td> <td align="left" rowspan="1" colspan="1">Best fit</td> </tr> <tr> <td align="left" rowspan="1" colspan="1">Power law exponent: α</td> <td align="left" rowspan="1" colspan="1">3</td> <td align="left" rowspan="1" colspan="1">Best fit</td> </tr> <tr> <td align="left" rowspan="1" colspan="1">Expected value of the lognormal distribution (used for the first phase of infection)</td> <td align="left" rowspan="1" colspan="1">7.5 hours</td> <td align="left" rowspan="1" colspan="1">Fitted to experimental results (see <a href="#pone-0015571-g008"><b>Fig. 8</b></a>)</td> </tr> <tr> <td align="left" rowspan="1" colspan="1">Standard deviation of the lognormal distribution (used for the first phase of infection)</td> <td align="left" rowspan="1" colspan="1">2.5 hours</td> <td align="left" rowspan="1" colspan="1">Fitted to experimental results (see <a href="#pone-0015571-g008"><b>Fig. 8</b></a>)</td> </tr> </tbody> </table> </div> <div class="table-wrap-foot_UNKNOWN"> <div class="fn-type-" title=""><span class="label" tagx="label" title="label" /><p>Best fit means that the parameter was optimized to give best overall agreement with the experimental results.</p> </div> </div> </div> <p>Unfortunately, standard tests for comparing the very complex experimental and computer generated fractal-like clusters are not available. Thus, to obtain results supporting the validity of our model, we have decided to compare the experimental results with two possible models, one being the model we propose, the other one being a standard percolation model, and evaluate the differences between the numbers we calculated from the observations with those obtained from the simulations. When applying the standard model, we assumed the same specific dynamics for the spatial and temporal dependence of the new centres of infection. However - instead of the non-linear function describing the probability that an infected cell spreads the viruses to its neighbours - we used a common and simpler assumption of this probability being a constant p = 0.7. In addition we have also visually compared the actual appearance of the cluster shapes. The results of these studies can be summarized as follows:</p> <p>The best fit to the fractal dimension of the largest cluster in <a href="#pone-0015571-g005"><b>Fig. 5</b></a> results in a value D = 1.78±0.03. Our non-linear model gives a result D = 1.813±0.027. The standard percolation model leads to the value D = 1.861±0.019. Thus, we conclude that regarding our model versus the experiment, the error bars overlap (there is a reasonable agreement), while a standard percolation model gives a result which is outside of the region allowed by the error bars. In the language of the t-test, the agreement of our model is still not good enough to establish a 95% confidence of its validity, but the probability that our model is superior to the simpler standard version is orders of magnitude higher. The situation is very similar in the case of the exponent describing the cluster size distribution. Here the experimentally obtained value of −1.7±0.04 can be compared with the values −1,643±0,03 (our model) and −1,578±0,02 (standard model). Again, the results are such that there is an overlap within the error bars for our model and there is no overlap for the standard model. <b><a href="#pone.0015571.s005">Fig. S5</a></b> shows the visual result gained using the standard percolation model.</p> <p>Visual observation of the experimental and the simulated clusters suggests that the relatively smooth and rounded hull of the largest cluster for the standard model (p = 0.7) would result in a fractal line with a significantly smaller dimension and a much smaller scaling region (length versus overall extension) than hulls of the experimental and the simulated (our model) clusters. Indeed, the numbers we obtain are for the fractal dimensions of the hulls, D<sub>H</sub>, D<sub>H</sub>(experiment) = 1.54±0.06, D<sub>H</sub>(our model) = 1.57±0.06 and D<sub>H</sub>(standard model) = 1.41±0.06.</p> </div> </div> <div class="analysisoftheclusters" title="sec"> <div class="title" tagx="title" title="title">
Analysis of the clusters</div> <p>The resulting configurations (clusters of connected infected cells) from the simulation and also from the experiment have been analyzed by methods of statistical physics. The cluster size distribution has been determined using the Hoshen-Kopelman algorithm to find clusters <a href="#pone.0015571-Hoshen1">[47]</a>. The fractal dimension of the biggest cluster and the average of the fractal dimensions of all clusters above a given size were calculated from density correlations <a href="#pone.0015571-Vicsek1">[25]</a>. In the case of the experimental results the pixels of the pictures belonging to a cluster were considered as particles. (see <a href="#pone-0015571-g004"><b>Fig. 4b,d,f</b></a> and <b><a href="#pone.0015571.s002">Fig. S2</a></b>).</p> </div> <div class="percolationtheoryessentials" title="sec"> <div class="title" tagx="title" title="title">
Percolation theory essentials</div> <p>One of the classic explanatory examples for standard percolation processes involves a forest, in which a few trees spontaneously become infected. The probability that a neighbouring tree becomes infected by an already infected one depends on several factors and is modelled with allowing infection between two trees with a given probability p. In the case of a tree nursery the trees would be, for example, arranged as the nodes of a square lattice. According to percolation theory if p is smaller than a given, lattice dependent p<sub>c</sub>, the infection cannot spread, above this critical probability a finite proportion of the trees become infected, while at the critical point the infected trees form a large very irregular "cluster" of neighbouring trees. This cluster can be described in terms of fractal geometry, because the number of trees N(L) in a region of linear size L scales as <span class="inline-formula" title="inline-formula"><span class="inline-graphic" title="inline-graphic" /></span>, i.e., instead of a power equal to 2 (corresponding to a 2 dimensional cluster) the fractal dimension of the cluster, D, is a number smaller than 2 <a href="#pone.0015571-Vicsek1">[25]</a>. Here L is measured using a unit being equal to the average distance between two neighbours or, in the case of lattice geometry, it is the lattice constant. Another helpful thought experiment for demonstrating the phenomena taking place in percolation systems involves coins dropped with some rate at randomly chosen spots of a given area. As the number of coins on the ground grows the probability that two nearby coins eventually overlap increases, thus, larger and larger clusters appear. At a critical number of the coins (corresponding to the critical probability above) the largest cluster (of overlapping coins) spreads over the whole area. Meanwhile the distribution of the cluster sizes is very uneven, there will be many small ones, a lesser number of larger ones, and a single so called giant cluster. This distribution is denoted by n<sub>s</sub> (where n<sub>s</sub> is the number of clusters consisting of s units) and is described by a power law with an exponent <i>τ</i>, i.e., <span class="inline-formula" title="inline-formula"><span class="inline-graphic" title="inline-graphic" /></span>which is very different from the usual Gaussian distribution since it means that the cluster sizes are not scattered around a well defined average <a href="#pone.0015571-Stauffer1">[23]</a>.</p> </div> </div> <div class="supplementary-material" title="supplementary-material"> <div class="title" tagx="title" title="title">
Supporting Information</div> <div class="local-data" title="local-data"><span class="label" tagx="label" title="label">Figure S1</span> <p>Original image of <a href="#pone-0015571-g003"><b>Figure 3</b></a>. 10×10 microscopic field images were taken from fixed cultures with GFP expressing infected cells, 48 hours after the onset of infection with "high" virus titer. High resolution file can be downloaded from: <a href="http://amur.elte.hu/BDGVirus/">http://amur.elte.hu/BDGVirus/</a></p> <p>(TIF)</p>  <div class="media" title="media"><a href="pone.0015571.s001.tif">LINK</a> <p>Click here for additional data file.</p>  </div> </div> <div class="local-data" title="local-data"><span class="label" tagx="label" title="label">Figure S2</span> <p>Rate of appearance of new cluster seeds as a function of time in the model. Simulation starts with 500 seeds with a "switching on" time following a log-normal distribution fitted to the experimentally observed parameters (mean =7.5 h, standard deviation =2.5 h), and continues with 2000 further seeds with a constantly increasing rate in time between 20 h and 42 h corresponding to the observed second wave of infection.</p> <p>(TIF)</p>  <div class="media" title="media"><a href="pone.0015571.s002.tif">LINK</a> <p>Click here for additional data file.</p>  </div> </div> <div class="local-data" title="local-data"><span class="label" tagx="label" title="label">Figure S3</span> <p>Features of the astrocytic cultures. Astrocytic cultures were live-stained with a fluorescent plasma membrane stain (<b>A</b>,<b>B</b>,<b>C</b>,<b>H</b>) and the nuclei were labeled with DAPI (<b>C</b>,<b>D</b>,<b>E</b>,<b>H</b>). Both the membrane-stained and the phase contrast (<b>F</b>,<b>G</b>) images show that astrocytes are arranged to a mosaic like monolayer. Scale bars: 25 µm (<b>A</b>,<b>D</b>,<b>F</b>,<b>H</b>), 10 µm (<b>B</b>,<b>C</b>,<b>E</b>,<b>G</b>). High resolution file can be downloaded from: <a href="http://amur.elte.hu/BDGVirus/">http://amur.elte.hu/BDGVirus/</a></p> <p>(TIF)</p>  <div class="media" title="media"><a href="pone.0015571.s003.tif">LINK</a> <p>Click here for additional data file.</p>  </div> </div> <div class="local-data" title="local-data"><span class="label" tagx="label" title="label">Figure S4</span> <p>Comparing the experimentally and computationally obtained clusters. Comparison of clusters found by the Hoshen – Koppelman algorithm on photo montages (<b>A</b>,<b>B</b>,<b>C</b>) and clusters developed in the model (<b>D</b>,<b>E</b>,<b>F</b>). Time elapsed after infection was 18 h (<b>A</b>,<b>D</b>), 30 h (<b>B</b>,<b>E</b>) and 42 h (<b>C</b>,<b>F</b>), respectively. Cluster growth, imitating virus propagation in the model can be seen in supplementary <b>Video S7.</b> High resolution file can be downloaded from: <a href="http://amur.elte.hu/BDGVirus/">http://amur.elte.hu/BDGVirus/</a></p> <p>(TIF)</p>  <div class="media" title="media"><a href="pone.0015571.s004.tif">LINK</a> <p>Click here for additional data file.</p>  </div> </div> <div class="local-data" title="local-data"><span class="label" tagx="label" title="label">Figure S5</span> <p>Clusters obtained in the simulations of a simplified model. The clusters generated with seeds of infection appearing at a rate corresponding to the experiment, but the infection spreading according to the p=0.7 simple percolation rule leads to images like this one. The clusters, on the scale being about 20 to 50 lattice units (a lattice unit corresponding to the size of a cell), are much more compact than the ones observed in the experiments, and their percolation-like features start to show up only at a much larger scale than in the experiments.</p> <p>(TIF)</p>  <div class="media" title="media"><a href="pone.0015571.s005.tif">LINK</a> <p>Click here for additional data file.</p>  </div> </div> <div class="local-data" title="local-data"><span class="label" tagx="label" title="label">Table S1</span> <p>Removal of infective viral particles from the culture medium. Percentage of infected (GFP expressing) cell groups developed in astrocytic cultures incubated with the first 3 washes of virus treated "original" sister‐cultures for 1 h or 24 hours. "Low titer": 2,5×10<sup>4</sup> PFU/ml, "High titer": 2,5×10<sup>5</sup> PFU/ml.</p> <p>(DOC)</p>  <div class="media" title="media"><a href="pone.0015571.s006.doc">LINK</a> <p>Click here for additional data file.</p>  </div> </div> <div class="local-data" title="local-data"><span class="label" tagx="label" title="label">Table S2</span> <p>Release of infective viral particles to the culture medium by astrocytes. Percentage of infected (GFP expressing) cell groups developed in astrocytic cultures incubated for 1 h or 24 hrs with conditioned media (CM) taken from virus treated sister‐cultures. nd: not detected</p> <p>(DOC)</p>  <div class="media" title="media"><a href="pone.0015571.s007.doc">LINK</a> <p>Click here for additional data file.</p>  </div> </div> <div class="local-data" title="local-data"><span class="label" tagx="label" title="label">Video S1</span> <p>Virus spreading (development of GFP expression) in an astroglial cell culture infected with "low" titer (2,5×10<sup>4</sup> PFU/ml) BDG‐PRV virus. High resolution videos can be downloaded from: <a href="http://amur.elte.hu/BDGVirus/">http://amur.elte.hu/BDGVirus/</a></p> <p>(MPG)</p>  <div class="media" title="media"><a href="pone.0015571.s008.mpg">LINK</a> <p>Click here for additional data file.</p>  </div> </div> <div class="local-data" title="local-data"><span class="label" tagx="label" title="label">Video S2</span> <p>Virus spreading (development of GFP expression) in an astroglial cell culture infected with "high" titer (2.5×10<sup>5</sup> PFU/ml) BDG‐PRV virus. High resolution videos can be downloaded from: <a href="http://amur.elte.hu/BDGVirus/">http://amur.elte.hu/BDGVirus/</a></p> <p>(MPG)</p>  <div class="media" title="media"><a href="pone.0015571.s009.mpg">LINK</a> <p>Click here for additional data file.</p>  </div> </div> <div class="local-data" title="local-data"><span class="label" tagx="label" title="label">Video S3</span> <p>Virus spreading (development of GFP expression) in an astroglial cell culture infected with "highest" titer (1,25×10<sup>6</sup> PFU/ml) BDG‐PRV virus. High resolution videos can be downloaded from: <a href="http://amur.elte.hu/BDGVirus/">http://amur.elte.hu/BDGVirus/</a></p> <p>(MPG)</p>  <div class="media" title="media"><a href="pone.0015571.s010.mpg">LINK</a> <p>Click here for additional data file.</p>  </div> </div> <div class="local-data" title="local-data"><span class="label" tagx="label" title="label">Video S4</span> <p>The video shows the appearance of GFP in infected cells over time within a representative microscopic field. High resolution file can be downloaded from: <a href="http://amur.elte.hu/BDGVirus/">http://amur.elte.hu/BDGVirus/</a></p> <p>(MPG)</p>  <div class="media" title="media"><a href="pone.0015571.s011.mpg">LINK</a> <p>Click here for additional data file.</p>  </div> </div> <div class="local-data" title="local-data"><span class="label" tagx="label" title="label">Video S5</span> <p>Graphical representation of <b>Video 4</b>. showing virus propagation within an astroglial culture treated with low titer (2,5×10<sup>4</sup> PFU/ml) BDG‐PRV virus. Red cells indicate the foci of the developing clusters. High resolution file can be downloaded from: <a href="http://amur.elte.hu/BDGVirus/">http://amur.elte.hu/BDGVirus/</a></p> <p>(MPG)</p>  <div class="media" title="media"><a href="pone.0015571.s012.mpg">LINK</a> <p>Click here for additional data file.</p>  </div> </div> <div class="local-data" title="local-data"><span class="label" tagx="label" title="label">Video S6</span> <p>Graphical representation of virus propagation within 9 adjacent microscopic fields of an astroglial culture treated with low titer (2,5×10<sup>4</sup> PFU/ml) BDG‐PRV virus. Graphical representations of virus propagation within an astroglial culture treated with low titer (2,5×10<sup>4</sup> PFU/ml) BDG‐PRV virus. High resolution file can be downloaded from: <a href="http://amur.elte.hu/BDGVirus/">http://amur.elte.hu/BDGVirus/</a></p> <p>(MPG)</p>  <div class="media" title="media"><a href="pone.0015571.s013.mpg">LINK</a> <p>Click here for additional data file.</p>  </div> </div> <div class="local-data" title="local-data"><span class="label" tagx="label" title="label">Video S7</span> <p>Cluster growth, imitating virus propagation in the model.</p> <p>(MPG)</p>  <div class="media" title="media"><a href="pone.0015571.s014.mpg">LINK</a> <p>Click here for additional data file.</p>  </div> </div> </div> </div> <div class="back" title="back"> <div class="ack" title="ack"> <p>We thank Dr. Zsolt Boldogkői and Dr. Ákos Hornyák for the recombinant BDG virus. We are grateful to Emília Madarász and Krisztina Kovács for supporting our work.</p> </div> <div class="fn-group" title="fn-group"> <div class="fn-type-COI-statement" title="COI-statement"> <p><b>Competing Interests: </b>The authors have declared that no competing interests exist.</p> </div> <div class="fn-type-financial-disclosure" title="financial-disclosure"> <p><b>Funding: </b>National Office of Research and Technology, Hungary, <a href="http://www.nkth.gov.hu/english">http://www.nkth.gov.hu/english</a>, Grant: NKTH ELTE CellCom RET <a href="http://cellkom.elte.hu/projects.html">http://cellkom.elte.hu/projects.html</a>, European Research Council, <a href="http://erc.europa.eu/">http://erc.europa.eu/</a> Grant: EU ERC COLLMOT No: 227878 <a href="http://erc.europa.eu/index.cfm?fuseaction=page.display&amp;topicID=517">http://erc.europa.eu/index.cfm?fuseaction=page.display&amp;amp;topicID=517</a>. The funders had no role in study design, data collection and analysis, decision to publish, or preparation of the manuscript.</p> </div> </div> <div class="references">
References</div> <div tag="ref-list"> <ul> <div class="title" tagx="title" title="title">
References</div> <li tag="ref"><a name="pone.0015571-Brauer1" /><span class="label" tagx="label" title="label">1</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Brauer</span><span class="given-names" tagx="given-names" title="given-names">F</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Castillo-Chávez</span><span class="given-names" tagx="given-names" title="given-names">C</span></span></span><span class="year" tagx="year" title="year">2001</span><span class="mixed-article-title" title="mixed-article-title">Mathematical Models in Population Biology and Epidemiology.</span><span class="publisher-loc" tagx="publisher-loc" title="publisher-loc">New York</span><span class="publisher-name" tagx="publisher-name" title="publisher-name">Springer-Verlag</span></span></li> <li tag="ref"><a name="pone.0015571-Daley1" /><span class="label" tagx="label" title="label">2</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Daley</span><span class="given-names" tagx="given-names" title="given-names">DJ</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gani</span><span class="given-names" tagx="given-names" title="given-names">J</span></span></span><span class="year" tagx="year" title="year">1999</span><span class="mixed-article-title" title="mixed-article-title">Epidemic Modelling: an Introduction.</span><span class="publisher-loc" tagx="publisher-loc" title="publisher-loc">Cambridge</span><span class="publisher-name" tagx="publisher-name" title="publisher-name">Cambridge University Press</span></span></li> <li tag="ref"><a name="pone.0015571-Riley1" /><span class="label" tagx="label" title="label">3</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Riley</span><span class="given-names" tagx="given-names" title="given-names">S</span></span></span><span class="year" tagx="year" title="year">2007</span><span class="mixed-article-title" title="mixed-article-title">Large-Scale Spatial-Transmission Models of Infectious Disease.</span><span class="source" tagx="source" title="source">Science</span><span class="volume" tagx="volume" title="volume">316</span><span class="fpage" tagx="fpage" title="fpage">1298</span><span class="lpage" tagx="lpage" title="lpage">1301</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17540894">17540894</a></span></span></li> <li tag="ref"><a name="pone.0015571-AlainBarrat1" /><span class="label" tagx="label" title="label">4</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Alain Barrat</span><span class="given-names" tagx="given-names" title="given-names">A</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Barthelemy</span><span class="given-names" tagx="given-names" title="given-names">M</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Vespignani</span><span class="given-names" tagx="given-names" title="given-names">A</span></span></span><span class="year" tagx="year" title="year">2008</span><span class="mixed-article-title" title="mixed-article-title">Dynamical processes on complex networks.</span><span class="publisher-loc" tagx="publisher-loc" title="publisher-loc">New York</span><span class="publisher-name" tagx="publisher-name" title="publisher-name">Cambridge University Press</span></span></li> <li tag="ref"><a name="pone.0015571-Wang1" /><span class="label" tagx="label" title="label">5</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wang</span><span class="given-names" tagx="given-names" title="given-names">P</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">González</span><span class="given-names" tagx="given-names" title="given-names">MC</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hidalgo</span><span class="given-names" tagx="given-names" title="given-names">CA</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Barabási</span><span class="given-names" tagx="given-names" title="given-names">A-L</span></span></span><span class="year" tagx="year" title="year">2009</span><span class="mixed-article-title" title="mixed-article-title">Understanding the Spreading Patterns of Mobile Phone Viruses.</span><span class="source" tagx="source" title="source">Science</span><span class="volume" tagx="volume" title="volume">324</span><span class="fpage" tagx="fpage" title="fpage">1071</span><span class="lpage" tagx="lpage" title="lpage">1076</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19342553">19342553</a></span></span></li> <li tag="ref"><a name="pone.0015571-Brockmann1" /><span class="label" tagx="label" title="label">6</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Brockmann</span><span class="given-names" tagx="given-names" title="given-names">D</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hufnagel</span><span class="given-names" tagx="given-names" title="given-names">L</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Geisel</span><span class="given-names" tagx="given-names" title="given-names">T</span></span></span><span class="year" tagx="year" title="year">2006</span><span class="mixed-article-title" title="mixed-article-title">The scaling laws of human travel.</span><span class="source" tagx="source" title="source">Nature</span><span class="volume" tagx="volume" title="volume">439</span><span class="fpage" tagx="fpage" title="fpage">462</span><span class="lpage" tagx="lpage" title="lpage">465</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16437114">16437114</a></span></span></li> <li tag="ref"><a name="pone.0015571-Hufnagel1" /><span class="label" tagx="label" title="label">7</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hufnagel</span><span class="given-names" tagx="given-names" title="given-names">L</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Brockmann</span><span class="given-names" tagx="given-names" title="given-names">D</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Geisel</span><span class="given-names" tagx="given-names" title="given-names">T</span></span></span><span class="year" tagx="year" title="year">2004</span><span class="mixed-article-title" title="mixed-article-title">Forecast and control of epidemics in a globalized world.</span><span class="source" tagx="source" title="source">Proc Natl Acad Sci USA</span><span class="volume" tagx="volume" title="volume">101</span><span class="fpage" tagx="fpage" title="fpage">15124</span><span class="lpage" tagx="lpage" title="lpage">15129</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15477600">15477600</a></span></span></li> <li tag="ref"><a name="pone.0015571-Ferguson1" /><span class="label" tagx="label" title="label">8</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ferguson</span><span class="given-names" tagx="given-names" title="given-names">NM</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Cummings</span><span class="given-names" tagx="given-names" title="given-names">DA</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Cauchemez</span><span class="given-names" tagx="given-names" title="given-names">S</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Fraser</span><span class="given-names" tagx="given-names" title="given-names">C</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Riley</span><span class="given-names" tagx="given-names" title="given-names">S</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="year" tagx="year" title="year">2005</span><span class="mixed-article-title" title="mixed-article-title">Strategies for containing an emerging influenza pandemic in Southeast Asia.</span><span class="source" tagx="source" title="source">Nature</span><span class="volume" tagx="volume" title="volume">437</span><span class="fpage" tagx="fpage" title="fpage">209</span><span class="lpage" tagx="lpage" title="lpage">214</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16079797">16079797</a></span></span></li> <li tag="ref"><a name="pone.0015571-Lai1" /><span class="label" tagx="label" title="label">9</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lai</span><span class="given-names" tagx="given-names" title="given-names">PC</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">So</span><span class="given-names" tagx="given-names" title="given-names">FM</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Chan</span><span class="given-names" tagx="given-names" title="given-names">K-W</span></span></span><span class="year" tagx="year" title="year">2009</span><span class="mixed-article-title" title="mixed-article-title">Spatial Epidemiological Approaches in Disease Mapping and Analysis.</span><span class="publisher-loc" tagx="publisher-loc" title="publisher-loc">Boca Raton</span><span class="publisher-name" tagx="publisher-name" title="publisher-name">CRC Press, Taylor &amp;amp; Francis Group</span></span></li> <li tag="ref"><a name="pone.0015571-Pfeiffer1" /><span class="label" tagx="label" title="label">10</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Pfeiffer</span><span class="given-names" tagx="given-names" title="given-names">DU</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Robinson</span><span class="given-names" tagx="given-names" title="given-names">TP</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Stevenson</span><span class="given-names" tagx="given-names" title="given-names">M</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Stevens</span><span class="given-names" tagx="given-names" title="given-names">KB</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Rogers</span><span class="given-names" tagx="given-names" title="given-names">DJ</span></span></span><span class="year" tagx="year" title="year">2008</span><span class="mixed-article-title" title="mixed-article-title">Spatial Analysis in Epidemiology.</span><span class="publisher-loc" tagx="publisher-loc" title="publisher-loc">Oxford</span><span class="publisher-name" tagx="publisher-name" title="publisher-name">Oxford University Press</span></span></li> <li tag="ref"><a name="pone.0015571-Keeling1" /><span class="label" tagx="label" title="label">11</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Keeling</span><span class="given-names" tagx="given-names" title="given-names">MJ</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Eames</span><span class="given-names" tagx="given-names" title="given-names">KTD</span></span></span><span class="year" tagx="year" title="year">2005</span><span class="mixed-article-title" title="mixed-article-title">Networks and epidemic models.</span><span class="source" tagx="source" title="source">J R Soc Interface</span><span class="volume" tagx="volume" title="volume">2</span><span class="fpage" tagx="fpage" title="fpage">295</span><span class="lpage" tagx="lpage" title="lpage">307</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16849187">16849187</a></span></span></li> <li tag="ref"><a name="pone.0015571-Haseltine1" /><span class="label" tagx="label" title="label">12</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Haseltine</span><span class="given-names" tagx="given-names" title="given-names">EL</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lam</span><span class="given-names" tagx="given-names" title="given-names">V</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Yin</span><span class="given-names" tagx="given-names" title="given-names">J</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Rawlings</span><span class="given-names" tagx="given-names" title="given-names">JB</span></span></span><span class="year" tagx="year" title="year">2008</span><span class="mixed-article-title" title="mixed-article-title">Image-guided modelling of virus growth and spread.</span><span class="source" tagx="source" title="source">Bull Math Biol</span><span class="volume" tagx="volume" title="volume">70</span><span class="fpage" tagx="fpage" title="fpage">1730</span><span class="lpage" tagx="lpage" title="lpage">1748</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18437499">18437499</a></span></span></li> <li tag="ref"><a name="pone.0015571-Lee1" /><span class="label" tagx="label" title="label">13</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lee</span><span class="given-names" tagx="given-names" title="given-names">Y</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Yin</span><span class="given-names" tagx="given-names" title="given-names">J</span></span></span><span class="year" tagx="year" title="year">1996</span><span class="mixed-article-title" title="mixed-article-title">Detection of evolving viruses. Nat. Biotechnol.</span><span class="volume" tagx="volume" title="volume">14</span><span class="fpage" tagx="fpage" title="fpage">491</span><span class="lpage" tagx="lpage" title="lpage">493</span></span></li> <li tag="ref"><a name="pone.0015571-Marquis1" /><span class="label" tagx="label" title="label">14</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Marquis</span><span class="given-names" tagx="given-names" title="given-names">H</span></span></span><span class="year" tagx="year" title="year">2006</span><span class="mixed-article-title" title="mixed-article-title">Tissue Culture Cell Assays Used to Analyze Listeria monocytogenes.</span><span class="source" tagx="source" title="source">Curr Prot in Microbiol.</span><span class="comment">Unit 9B.4</span></span></li> <li tag="ref"><a name="pone.0015571-Rhodes1" /><span class="label" tagx="label" title="label">15</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Rhodes</span><span class="given-names" tagx="given-names" title="given-names">CJ</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Anderson</span><span class="given-names" tagx="given-names" title="given-names">RM</span></span></span><span class="year" tagx="year" title="year">1998</span><span class="mixed-article-title" title="mixed-article-title">Forest-fire as a model for the dynamics of disease epidemics.</span><span class="source" tagx="source" title="source">J Franklin Inst</span><span class="volume" tagx="volume" title="volume">335</span><span class="fpage" tagx="fpage" title="fpage">199</span><span class="lpage" tagx="lpage" title="lpage">211</span></span></li> <li tag="ref"><a name="pone.0015571-Hu1" /><span class="label" tagx="label" title="label">16</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hu</span><span class="given-names" tagx="given-names" title="given-names">H</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Myers</span><span class="given-names" tagx="given-names" title="given-names">S</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Colizza</span><span class="given-names" tagx="given-names" title="given-names">V</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Vespignani</span><span class="given-names" tagx="given-names" title="given-names">A</span></span></span><span class="year" tagx="year" title="year">2009</span><span class="mixed-article-title" title="mixed-article-title">WiFi networks and malware epidemiology.</span><span class="source" tagx="source" title="source">Proc Natl Acad Sci USA</span><span class="volume" tagx="volume" title="volume">106</span><span class="fpage" tagx="fpage" title="fpage">1318</span><span class="lpage" tagx="lpage" title="lpage">1323</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19171909">19171909</a></span></span></li> <li tag="ref"><a name="pone.0015571-Colizza1" /><span class="label" tagx="label" title="label">17</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Colizza</span><span class="given-names" tagx="given-names" title="given-names">V</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Vespignani</span><span class="given-names" tagx="given-names" title="given-names">A</span></span></span><span class="year" tagx="year" title="year">2008</span><span class="mixed-article-title" title="mixed-article-title">Epidemic modelling in metapopulation systems with heterogeneous coupling pattern: theory and simulations.</span><span class="source" tagx="source" title="source">J Theor Biol</span><span class="volume" tagx="volume" title="volume">251</span><span class="fpage" tagx="fpage" title="fpage">450</span><span class="lpage" tagx="lpage" title="lpage">467</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18222487">18222487</a></span></span></li> <li tag="ref"><a name="pone.0015571-Tilney1" /><span class="label" tagx="label" title="label">18</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Tilney</span><span class="given-names" tagx="given-names" title="given-names">LG</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Portnoy</span><span class="given-names" tagx="given-names" title="given-names">DA</span></span></span><span class="year" tagx="year" title="year">1989</span><span class="mixed-article-title" title="mixed-article-title">Actin filaments and the growth, movement, and spread of the intracellular bacterial parasite, Listeria monocytogenes.</span><span class="source" tagx="source" title="source">J Cell Biol</span><span class="volume" tagx="volume" title="volume">109</span><span class="fpage" tagx="fpage" title="fpage">1597</span><span class="lpage" tagx="lpage" title="lpage">1608</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/2507553">2507553</a></span></span></li> <li tag="ref"><a name="pone.0015571-Stamm1" /><span class="label" tagx="label" title="label">19</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Stamm</span><span class="given-names" tagx="given-names" title="given-names">LM</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Morisaki</span><span class="given-names" tagx="given-names" title="given-names">JH</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gao</span><span class="given-names" tagx="given-names" title="given-names">LY</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Jeng</span><span class="given-names" tagx="given-names" title="given-names">RL</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">McDonald</span><span class="given-names" tagx="given-names" title="given-names">KL</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="year" tagx="year" title="year">2003</span><span class="mixed-article-title" title="mixed-article-title">Mycobacterium marinum escapes from phagosomes and is propelled by actin-based motility.</span><span class="source" tagx="source" title="source">J Exp Med</span><span class="volume" tagx="volume" title="volume">198</span><span class="fpage" tagx="fpage" title="fpage">1361</span><span class="lpage" tagx="lpage" title="lpage">1368</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/14597736">14597736</a></span></span></li> <li tag="ref"><a name="pone.0015571-Heindl1" /><span class="label" tagx="label" title="label">20</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Heindl</span><span class="given-names" tagx="given-names" title="given-names">JE</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Saran</span><span class="given-names" tagx="given-names" title="given-names">I</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Yi</span><span class="given-names" tagx="given-names" title="given-names">C</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lesser</span><span class="given-names" tagx="given-names" title="given-names">CF</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Goldberg</span><span class="given-names" tagx="given-names" title="given-names">MB</span></span></span><span class="year" tagx="year" title="year">2010</span><span class="mixed-article-title" title="mixed-article-title">Requirement for Formin-Induced Actin Polymerization during Spread of Shigella flexneri.</span><span class="source" tagx="source" title="source">Infect Immun</span><span class="volume" tagx="volume" title="volume">78</span><span class="fpage" tagx="fpage" title="fpage">193</span><span class="lpage" tagx="lpage" title="lpage">203</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19841078">19841078</a></span></span></li> <li tag="ref"><a name="pone.0015571-Teysseire1" /><span class="label" tagx="label" title="label">21</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Teysseire</span><span class="given-names" tagx="given-names" title="given-names">N</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Chiche-Portiche</span><span class="given-names" tagx="given-names" title="given-names">C</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Raoult</span><span class="given-names" tagx="given-names" title="given-names">D</span></span></span><span class="year" tagx="year" title="year">1992</span><span class="mixed-article-title" title="mixed-article-title">Intracellular movements of Rickettsia conorii and R. typhi based on actin polymerization.</span><span class="source" tagx="source" title="source">Res Microbiol</span><span class="volume" tagx="volume" title="volume">143</span><span class="fpage" tagx="fpage" title="fpage">821</span><span class="lpage" tagx="lpage" title="lpage">829</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/1299836">1299836</a></span></span></li> <li tag="ref"><a name="pone.0015571-Way1" /><span class="label" tagx="label" title="label">22</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Way</span><span class="given-names" tagx="given-names" title="given-names">SS</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Borczuk</span><span class="given-names" tagx="given-names" title="given-names">AC</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Dominitz</span><span class="given-names" tagx="given-names" title="given-names">R</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Goldberg</span><span class="given-names" tagx="given-names" title="given-names">MB</span></span></span><span class="year" tagx="year" title="year">1998</span><span class="mixed-article-title" title="mixed-article-title">An essential role for gamma interferon in innate resistance to Shigella flexneri infection.</span><span class="source" tagx="source" title="source">Infect Immun</span><span class="volume" tagx="volume" title="volume">66</span><span class="fpage" tagx="fpage" title="fpage">1342</span><span class="lpage" tagx="lpage" title="lpage">1348</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/9529051">9529051</a></span></span></li> <li tag="ref"><a name="pone.0015571-Stauffer1" /><span class="label" tagx="label" title="label">23</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Stauffer</span><span class="given-names" tagx="given-names" title="given-names">D</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Aharony</span><span class="given-names" tagx="given-names" title="given-names">A</span></span></span><span class="year" tagx="year" title="year">1991</span><span class="mixed-article-title" title="mixed-article-title">Introduction to Percolation Theory.</span><span class="publisher-loc" tagx="publisher-loc" title="publisher-loc">London</span><span class="publisher-name" tagx="publisher-name" title="publisher-name">Taylor and Francis</span></span></li> <li tag="ref"><a name="pone.0015571-Card1" /><span class="label" tagx="label" title="label">24</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Card</span><span class="given-names" tagx="given-names" title="given-names">JP</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Rinaman</span><span class="given-names" tagx="given-names" title="given-names">L</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lynn</span><span class="given-names" tagx="given-names" title="given-names">RB</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lee</span><span class="given-names" tagx="given-names" title="given-names">BH</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Meade</span><span class="given-names" tagx="given-names" title="given-names">RP</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="year" tagx="year" title="year">1993</span><span class="mixed-article-title" title="mixed-article-title">Pseudorabies virus infection of the rat central nervous system: ultrastructural characterization of viral replication, transport, and pathogenesis.</span><span class="source" tagx="source" title="source">J Neurosci</span><span class="volume" tagx="volume" title="volume">13</span><span class="fpage" tagx="fpage" title="fpage">2515</span><span class="lpage" tagx="lpage" title="lpage">39</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/8388923">8388923</a></span></span></li> <li tag="ref"><a name="pone.0015571-Vicsek1" /><span class="label" tagx="label" title="label">25</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Vicsek</span><span class="given-names" tagx="given-names" title="given-names">T</span></span></span><span class="year" tagx="year" title="year">1992</span><span class="mixed-article-title" title="mixed-article-title">Fractal Growth Phenomena.</span><span class="publisher-loc" tagx="publisher-loc" title="publisher-loc">Singapore</span><span class="publisher-name" tagx="publisher-name" title="publisher-name">World Scientific Publishing Co. Pte. Ltd</span></span></li> <li tag="ref"><a name="pone.0015571-Granzow1" /><span class="label" tagx="label" title="label">26</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Granzow</span><span class="given-names" tagx="given-names" title="given-names">H</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Weiland</span><span class="given-names" tagx="given-names" title="given-names">F</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Jons</span><span class="given-names" tagx="given-names" title="given-names">A</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Klupp</span><span class="given-names" tagx="given-names" title="given-names">BG</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Karger</span><span class="given-names" tagx="given-names" title="given-names">A</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="year" tagx="year" title="year">1997</span><span class="mixed-article-title" title="mixed-article-title">Ultrastructural analysis of the replication cycle of pseudorabies virus in cell culture: a reassessment.</span><span class="source" tagx="source" title="source">J Virol</span><span class="volume" tagx="volume" title="volume">71</span><span class="fpage" tagx="fpage" title="fpage">2072</span><span class="lpage" tagx="lpage" title="lpage">2082</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/9032339">9032339</a></span></span></li> <li tag="ref"><a name="pone.0015571-Dingwell1" /><span class="label" tagx="label" title="label">27</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Dingwell</span><span class="given-names" tagx="given-names" title="given-names">KS</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Brunetti</span><span class="given-names" tagx="given-names" title="given-names">CR</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hendricks</span><span class="given-names" tagx="given-names" title="given-names">RL</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Tang</span><span class="given-names" tagx="given-names" title="given-names">Q</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Tang</span><span class="given-names" tagx="given-names" title="given-names">M</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="year" tagx="year" title="year">1994</span><span class="mixed-article-title" title="mixed-article-title">Herpes simplex virus glycoproteins E and I facilitate cell-to-cell spread in vivo and across junctions of cultured cells.</span><span class="source" tagx="source" title="source">J Virol</span><span class="volume" tagx="volume" title="volume">68</span><span class="fpage" tagx="fpage" title="fpage">834</span><span class="lpage" tagx="lpage" title="lpage">845</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/8289387">8289387</a></span></span></li> <li tag="ref"><a name="pone.0015571-Duca1" /><span class="label" tagx="label" title="label">28</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Duca</span><span class="given-names" tagx="given-names" title="given-names">KA</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lam</span><span class="given-names" tagx="given-names" title="given-names">V</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Keren</span><span class="given-names" tagx="given-names" title="given-names">I</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Endler</span><span class="given-names" tagx="given-names" title="given-names">EE</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Letchworth</span><span class="given-names" tagx="given-names" title="given-names">GJ</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="year" tagx="year" title="year">2001</span><span class="mixed-article-title" title="mixed-article-title">Quantifying viral propagation in vitro: toward a method for characterization of complex phenotypes.</span><span class="source" tagx="source" title="source">Biotechnol Prog</span><span class="volume" tagx="volume" title="volume">17</span><span class="fpage" tagx="fpage" title="fpage">1156</span><span class="lpage" tagx="lpage" title="lpage">1165</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11735454">11735454</a></span></span></li> <li tag="ref"><a name="pone.0015571-Katsumoto1" /><span class="label" tagx="label" title="label">29</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Katsumoto</span><span class="given-names" tagx="given-names" title="given-names">T</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hirano</span><span class="given-names" tagx="given-names" title="given-names">A</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kurimura</span><span class="given-names" tagx="given-names" title="given-names">T</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Takagi</span><span class="given-names" tagx="given-names" title="given-names">A</span></span></span><span class="year" tagx="year" title="year">1981</span><span class="mixed-article-title" title="mixed-article-title">In situ electron microscopical observation of cells infected with herpes simplex virus.</span><span class="source" tagx="source" title="source">J Gen Virol</span><span class="volume" tagx="volume" title="volume">52</span><span class="fpage" tagx="fpage" title="fpage">267</span><span class="lpage" tagx="lpage" title="lpage">278</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/6270236">6270236</a></span></span></li> <li tag="ref"><a name="pone.0015571-Price1" /><span class="label" tagx="label" title="label">30</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Price</span><span class="given-names" tagx="given-names" title="given-names">RW</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Rubenstein</span><span class="given-names" tagx="given-names" title="given-names">R</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Khan</span><span class="given-names" tagx="given-names" title="given-names">A</span></span></span><span class="year" tagx="year" title="year">1982</span><span class="mixed-article-title" title="mixed-article-title">Herpes simplex virus infection of isolated autonomic neurons in culture: viral replication and spread in a neuronal network.</span><span class="source" tagx="source" title="source">Arch Virol</span><span class="volume" tagx="volume" title="volume">71</span><span class="fpage" tagx="fpage" title="fpage">127</span><span class="lpage" tagx="lpage" title="lpage">140</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/6279059">6279059</a></span></span></li> <li tag="ref"><a name="pone.0015571-Garner1" /><span class="label" tagx="label" title="label">31</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Garner</span><span class="given-names" tagx="given-names" title="given-names">JA</span></span></span><span class="year" tagx="year" title="year">2003</span><span class="mixed-article-title" title="mixed-article-title">Herpes simplex virion entry into and intracellular transport within mammalian cells.</span><span class="source" tagx="source" title="source">Adv Drug Deliv Rev</span><span class="volume" tagx="volume" title="volume">14</span><span class="fpage" tagx="fpage" title="fpage">1497</span><span class="lpage" tagx="lpage" title="lpage">1513</span></span></li> <li tag="ref"><a name="pone.0015571-Lam1" /><span class="label" tagx="label" title="label">32</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lam</span><span class="given-names" tagx="given-names" title="given-names">V</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Duca</span><span class="given-names" tagx="given-names" title="given-names">KA</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Yin</span><span class="given-names" tagx="given-names" title="given-names">J</span></span></span><span class="year" tagx="year" title="year">2005</span><span class="mixed-article-title" title="mixed-article-title">Arrested spread of vesicular stomatitis virus infections in vitro depends on interferon-mediated antiviral activity.</span><span class="source" tagx="source" title="source">Biotechnol Bioeng</span><span class="volume" tagx="volume" title="volume">90</span><span class="issue" tagx="issue" title="issue">7</span><span class="fpage" tagx="fpage" title="fpage">793</span><span class="lpage" tagx="lpage" title="lpage">804</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15834946">15834946</a></span></span></li> <li tag="ref"><a name="pone.0015571-Duca2" /><span class="label" tagx="label" title="label">33</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Duca</span><span class="given-names" tagx="given-names" title="given-names">KA</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lam</span><span class="given-names" tagx="given-names" title="given-names">V</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Keren</span><span class="given-names" tagx="given-names" title="given-names">I</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Endler</span><span class="given-names" tagx="given-names" title="given-names">EE</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Letchworth</span><span class="given-names" tagx="given-names" title="given-names">GJ</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="year" tagx="year" title="year">2001</span><span class="mixed-article-title" title="mixed-article-title">Quantifying viral propagation in vitro: toward a method for characterization of complex phenotypes.</span><span class="source" tagx="source" title="source">Biotechnol Prog</span><span class="volume" tagx="volume" title="volume">17</span><span class="issue" tagx="issue" title="issue">6</span><span class="fpage" tagx="fpage" title="fpage">1156</span><span class="lpage" tagx="lpage" title="lpage">65</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11735454">11735454</a></span></span></li> <li tag="ref"><a name="pone.0015571-Appleyard1" /><span class="label" tagx="label" title="label">34</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Appleyard</span><span class="given-names" tagx="given-names" title="given-names">G</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hapel</span><span class="given-names" tagx="given-names" title="given-names">AJ</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Boulter</span><span class="given-names" tagx="given-names" title="given-names">EA</span></span></span><span class="year" tagx="year" title="year">1971</span><span class="mixed-article-title" title="mixed-article-title">An antigenic difference between intracellular and extracellular rabbitpox virus.</span><span class="source" tagx="source" title="source">J Gen Virol</span><span class="volume" tagx="volume" title="volume">13</span><span class="issue" tagx="issue" title="issue">1</span><span class="fpage" tagx="fpage" title="fpage">9</span><span class="lpage" tagx="lpage" title="lpage">17</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/4108676">4108676</a></span></span></li> <li tag="ref"><a name="pone.0015571-Law1" /><span class="label" tagx="label" title="label">35</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Law</span><span class="given-names" tagx="given-names" title="given-names">M</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hollinshead</span><span class="given-names" tagx="given-names" title="given-names">R</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Smith</span><span class="given-names" tagx="given-names" title="given-names">GL</span></span></span><span class="year" tagx="year" title="year">2002</span><span class="mixed-article-title" title="mixed-article-title">Antibody-sensitive and antibody-resistant cell-to-cell spread by vaccinia virus: role of the A33R protein in antibody-resistant spread.</span><span class="source" tagx="source" title="source">J Gen Virol</span><span class="volume" tagx="volume" title="volume">83</span><span class="fpage" tagx="fpage" title="fpage">209</span><span class="lpage" tagx="lpage" title="lpage">22</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11752718">11752718</a></span></span></li> <li tag="ref"><a name="pone.0015571-Zhu1" /><span class="label" tagx="label" title="label">36</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zhu</span><span class="given-names" tagx="given-names" title="given-names">Y</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Yin</span><span class="given-names" tagx="given-names" title="given-names">J</span></span></span><span class="year" tagx="year" title="year">2007</span><span class="mixed-article-title" title="mixed-article-title">A quantitative comet assay: Imaging and analysis of virus plaques formed with a liquid overlay.</span><span class="source" tagx="source" title="source">Journal of Virological Methods</span><span class="volume" tagx="volume" title="volume">139</span><span class="fpage" tagx="fpage" title="fpage">100</span><span class="lpage" tagx="lpage" title="lpage">102</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17092573">17092573</a></span></span></li> <li tag="ref"><a name="pone.0015571-Fitting1" /><span class="label" tagx="label" title="label">37</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Fitting</span><span class="given-names" tagx="given-names" title="given-names">S</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zou</span><span class="given-names" tagx="given-names" title="given-names">S</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Chen</span><span class="given-names" tagx="given-names" title="given-names">W</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Vo</span><span class="given-names" tagx="given-names" title="given-names">P</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hauser</span><span class="given-names" tagx="given-names" title="given-names">KF</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="year" tagx="year" title="year">2010</span><span class="mixed-article-title" title="mixed-article-title">Regional heterogeneity and diversity in cytokine and chemokine production by astroglia: differential responses to HIV-1 Tat, gp120, and morphine revealed by multiplex analysis.</span><span class="source" tagx="source" title="source">J Proteome Res</span><span class="volume" tagx="volume" title="volume">9</span><span class="issue" tagx="issue" title="issue">4</span><span class="fpage" tagx="fpage" title="fpage">1795</span><span class="lpage" tagx="lpage" title="lpage">804</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20121167">20121167</a></span></span></li> <li tag="ref"><a name="pone.0015571-vanderMeulen1" /><span class="label" tagx="label" title="label">38</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">van der Meulen</span><span class="given-names" tagx="given-names" title="given-names">KM</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Nauwynck</span><span class="given-names" tagx="given-names" title="given-names">HJ</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Pensaert</span><span class="given-names" tagx="given-names" title="given-names">MB</span></span></span><span class="year" tagx="year" title="year">2002</span><span class="mixed-article-title" title="mixed-article-title">Increased susceptibility of peripheral blood mononuclear cells to equine herpes virus type 1 infection upon mitogen stimulation: a role of the cell cycle and of cell-to-cell transmission of the virus.</span><span class="source" tagx="source" title="source">Vet Microbiol</span><span class="volume" tagx="volume" title="volume">86</span><span class="issue" tagx="issue" title="issue">1-2</span><span class="fpage" tagx="fpage" title="fpage">157</span><span class="lpage" tagx="lpage" title="lpage">63</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11888698">11888698</a></span></span></li> <li tag="ref"><a name="pone.0015571-Carrascosa1" /><span class="label" tagx="label" title="label">39</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Carrascosa</span><span class="given-names" tagx="given-names" title="given-names">AL</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bustos</span><span class="given-names" tagx="given-names" title="given-names">MJ</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Galindo</span><span class="given-names" tagx="given-names" title="given-names">I</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Viñuela</span><span class="given-names" tagx="given-names" title="given-names">E</span></span></span><span class="year" tagx="year" title="year">1999</span><span class="mixed-article-title" title="mixed-article-title">Virus-specific cell receptors are necessary, but not sufficient, to confer cell susceptibility to African swine fever virus.</span><span class="source" tagx="source" title="source">Arch Virol</span><span class="volume" tagx="volume" title="volume">144</span><span class="issue" tagx="issue" title="issue">7</span><span class="fpage" tagx="fpage" title="fpage">1309</span><span class="lpage" tagx="lpage" title="lpage">21</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/10481739">10481739</a></span></span></li> <li tag="ref"><a name="pone.0015571-Li1" /><span class="label" tagx="label" title="label">40</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Li</span><span class="given-names" tagx="given-names" title="given-names">D</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Duan</span><span class="given-names" tagx="given-names" title="given-names">L</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Freimuth</span><span class="given-names" tagx="given-names" title="given-names">P</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">O'Malley</span><span class="given-names" tagx="given-names" title="given-names">BW</span><span class="suffix" tagx="suffix" title="suffix">Jr</span></span></span><span class="year" tagx="year" title="year">1999</span><span class="mixed-article-title" title="mixed-article-title">Variability of adenovirus receptor density influences gene transfer efficiency and therapeutic response in head and neck cancer.</span><span class="source" tagx="source" title="source">Clin Cancer Res</span><span class="volume" tagx="volume" title="volume">5</span><span class="issue" tagx="issue" title="issue">12</span><span class="fpage" tagx="fpage" title="fpage">4175</span><span class="lpage" tagx="lpage" title="lpage">81</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/10632357">10632357</a></span></span></li> <li tag="ref"><a name="pone.0015571-Khlifa1" /><span class="label" tagx="label" title="label">41</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Khélifa</span><span class="given-names" tagx="given-names" title="given-names">R</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Menezes</span><span class="given-names" tagx="given-names" title="given-names">J</span></span></span><span class="year" tagx="year" title="year">1982</span><span class="mixed-article-title" title="mixed-article-title">Epstein-Barr virus-lymphoid cell interactions. III. Effect of concanavalin A and saccharides on Epstein-Barr virus penetration.</span><span class="source" tagx="source" title="source">J Virol</span><span class="volume" tagx="volume" title="volume">42</span><span class="issue" tagx="issue" title="issue">2</span><span class="fpage" tagx="fpage" title="fpage">402</span><span class="lpage" tagx="lpage" title="lpage">10</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/6283164">6283164</a></span></span></li> <li tag="ref"><a name="pone.0015571-Krnyei1" /><span class="label" tagx="label" title="label">42</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Környei</span><span class="given-names" tagx="given-names" title="given-names">Z</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Szlávik</span><span class="given-names" tagx="given-names" title="given-names">V</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Szabó</span><span class="given-names" tagx="given-names" title="given-names">B</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gócza</span><span class="given-names" tagx="given-names" title="given-names">E</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Czirók</span><span class="given-names" tagx="given-names" title="given-names">A</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="year" tagx="year" title="year">2005</span><span class="mixed-article-title" title="mixed-article-title">Humoral and contact interactions in astroglia/stem cell co-cultures in the course of glia-induced neurogenesis.</span><span class="source" tagx="source" title="source">Glia</span><span class="volume" tagx="volume" title="volume">49</span><span class="fpage" tagx="fpage" title="fpage">430</span><span class="lpage" tagx="lpage" title="lpage">444</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15546152">15546152</a></span></span></li> <li tag="ref"><a name="pone.0015571-Boldogki1" /><span class="label" tagx="label" title="label">43</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Boldogkői</span><span class="given-names" tagx="given-names" title="given-names">Z</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Reichart</span><span class="given-names" tagx="given-names" title="given-names">A</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Tóth</span><span class="given-names" tagx="given-names" title="given-names">IE</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Sik</span><span class="given-names" tagx="given-names" title="given-names">A</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Erdélyi</span><span class="given-names" tagx="given-names" title="given-names">F</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="year" tagx="year" title="year">2002</span><span class="mixed-article-title" title="mixed-article-title">Construction of recombinant pseudorabies viruses optimized for labelling and neurochemical characterization of neural circuitry.</span><span class="source" tagx="source" title="source">Mol Brain Res</span><span class="volume" tagx="volume" title="volume">109</span><span class="fpage" tagx="fpage" title="fpage">105</span><span class="lpage" tagx="lpage" title="lpage">118</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12531520">12531520</a></span></span></li> <li tag="ref"><a name="pone.0015571-Boldogki2" /><span class="label" tagx="label" title="label">44</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Boldogkői</span><span class="given-names" tagx="given-names" title="given-names">Z</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Sík</span><span class="given-names" tagx="given-names" title="given-names">A</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Dénes</span><span class="given-names" tagx="given-names" title="given-names">Á</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Reichart</span><span class="given-names" tagx="given-names" title="given-names">A</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Toldi</span><span class="given-names" tagx="given-names" title="given-names">J</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="year" tagx="year" title="year">2004</span><span class="mixed-article-title" title="mixed-article-title">Novel tracing paradigms-genetically engineered herpesviruses as tools for mapping functional circuits within the CNS: present status and future prospects,</span><span class="source" tagx="source" title="source">Prog Neurobiol</span><span class="volume" tagx="volume" title="volume">72</span><span class="fpage" tagx="fpage" title="fpage">417</span><span class="lpage" tagx="lpage" title="lpage">445</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15177785">15177785</a></span></span></li> <li tag="ref"><a name="pone.0015571-ImageJ1" /><span class="label" tagx="label" title="label">45</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="comment">ImageJ website (accessed 2010). <a href="http://rsbweb.nih.gov/ij/docs/menus/process.html#background">http://rsbweb.nih.gov/ij/docs/menus/process.html#background</a></span></span></li> <li tag="ref"><a name="pone.0015571-Sternberg1" /><span class="label" tagx="label" title="label">46</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Sternberg</span><span class="given-names" tagx="given-names" title="given-names">SR</span></span></span><span class="year" tagx="year" title="year">1983</span><span class="mixed-article-title" title="mixed-article-title">Biomedical image processing.</span><span class="source" tagx="source" title="source">Computer</span><span class="volume" tagx="volume" title="volume">16</span><span class="fpage" tagx="fpage" title="fpage">22</span><span class="lpage" tagx="lpage" title="lpage">34</span></span></li> <li tag="ref"><a name="pone.0015571-Hoshen1" /><span class="label" tagx="label" title="label">47</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hoshen</span><span class="given-names" tagx="given-names" title="given-names">J</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kopelman</span><span class="given-names" tagx="given-names" title="given-names">R</span></span></span><span class="year" tagx="year" title="year">1976</span><span class="mixed-article-title" title="mixed-article-title">Percolation and cluster distribution. I. Cluster multiple labelling technique and critical concentration algorithm,</span><span class="source" tagx="source" title="source">Phys Rev B</span><span class="volume" tagx="volume" title="volume">14</span><span class="fpage" tagx="fpage" title="fpage">3438</span><span class="lpage" tagx="lpage" title="lpage">3445</span></span></li> </ul> </div> </div>  </body></html>