<html xmlns="http://www.w3.org/1999/xhtml"> <head> <style type="text/css">
      	     		 
      	    a                          {background : #ffffff; }
      	    article                    {border-style : dotted; border-width : 2px; }
		 	
		 	div                        {background : #ffffcc;}
		 	div.abstract-title         {font-weight : bold ; font-size : 16pt;}
		 	div.ack                    {border-style : solid ; border-color : red; margin : 2em; }
		 	div.article-meta           {border-style : solid; border-width : 1 pt; margin 1em;}
		 	div.boxed-text             {margin : 5em; border-style : solid;}
		 	div.contrib-group          {margin : 1em; }
		 	div.fig                    {border-style : solid; border-width : 2px; margin : 2em; }
		 	div.funding                {font-weight : bold ; font-size : 16pt;}
		 	div.given-names            {font-style : italic;}
		 	div.intro                  {border-style : inset; margin : 5px;}
		 	div.introduction           {border-style : inset; margin : 5px;}
		 	div.journal-meta           {border-style : solid; border-width : 1 pt; margin 1em;}
		 	div.journal-title-group    {background : #ffeeee;}
		 	div.article-title          {font-weight : bold ; font-size : 18pt;}
		 	div.kwd                    {font-style : italic}
		 	div.meta-name              {font-weight : bold ; font-size : 16pt;}
		 	div.name                   {font-weight : bold;}
		 	div.alt-title              {font-style : italic; font-size : 12px;}
		 	
		 	div.sec                    {border : 2px; margin 5px; padding 2px;}
		 	div.title                  {font-family : courier; font-weight : bold;
		 	                            font-size : 16pt; margin : 5px;}
		 	
		 	div.materials_methods      {border-style : double; margin : 5px; }
		 	div.methods                {border-style : double; margin : 5px; }
		 	
		 	div.results                {border-style : solid; margin : 5px;}
		 	div.background             {border-style : dotted; margin : 5px;}
		 	
		 	div.discussion             {border-style : groove; margin : 5px;}
		 	div.conclusion             {border-style : ridge; margin : 5px;}
		 	div.conclusions             {border-style : ridge; margin : 5px;}
		 	div.supplementary-material {border-style : inset; margin : 5px;}
		 	div.abbreviations          {border-style : double; border-color : red; margin : 5px;}
		 	div.competinginterests     {border-style : double; border-color : blue; margin : 5px;}
		 	div.acknowledgements       {border-style : double; border-color : green; margin : 5px;}
		 	div.authors_contributions   {border-style : double; border-color : purple; margin : 5px;}
		 	
		 	div.publisher              {border-style : outset; margin : 5px;}
		 	div.fn-type-conflict       {background : #f88; }
		 	div.fn-type-con            {background : #ddf; }
		 	div.fn-type-other          {background : #ddd; }
		 	
		 	div.unknown                {background : #ffd;
									 	  border-style : solid;
									 	  border-width : 1px;
									 	  padding : 2 px;}
		 	  
      	    table                      {background : #ffffdd;}
		 	tr                         {background : #ddddff; padding : 1px;}
		 	
		 	span                       {background : #ffcccc;}
		 	
		 	span.citation-author       {font-family : helvetica; background : #ffeeee;}
		 	span.collab                {background : #ddffff; }
		 	span.comment               {font-family : courier; font-size : 6px; background : #ffaaff;}
		 	span.contrib               {background : #ffffff;}
		 	span.corresp               {background : #ddffdd; }
		 	span.doi                   {background : #ffffff;}
		 	span.email                 {font-family : courier; }
		 	span.etal                  {font-style : italic;}
		 	span.fpage                 {font-family : courier;}
		 	span.given-names           {background : #ffffff;}
		 	span.iso-abbrev            {background : #ffffff;}
		 	span.issn-epub             {background : #ffffff;}
		 	span.issn-ppub             {background : #ffffff;}
		 	span.journal-title         {background : #ffffff;}
		 	span.lpage                 {font-family : courier;}
		 	span.mixed-article-title   {font-style : italic ;}
		 	span.nlm-ta                {background : #ffffff;}
		 	span.pmc                   {background : #ffffff;}
		 	span.pmcid                 {background : #ffffff;}
		 	span.pmid                  {background : #ffffff;}
		 	span.publisher             {background : #ffffff;}
		 	span.publisher-id          {background : #ffffff;}
		 	span.publisher-name        {background : #ffffff;}
		 	span.source                {background : #ffffff;}
		 	span.subject               {background : #ffffff;}
		 	span.surname               {background : #ffffff;}
		 	span.volume                {font-family : courier; font-weight : bold;}
		 	span.year                  {font-family : courier ; font-style : italic;}
			</style> </head> <body> <div class="front" title="front"> <div class="journal-meta" tagx="journal-meta" title="journal-meta"><span class="nlm-ta" title="nlm-ta">F1000Res</span><span class="iso-abbrev" title="iso-abbrev">F1000Res</span><span class="pmc" title="pmc">F1000Research</span><div class="journal-title-group" tagx="journal-title-group" title="journal-title-group"><span class="journal-title" tagx="journal-title" title="journal-title">F1000Research</span></div><span class="issn-epub" tagx="issn" title="issn-epub">2046-1402</span><div class="publisher" tagx="publisher" title="publisher"><span class="publisher-name" tagx="publisher-name" title="publisher-name">F1000Research</span><span class="publisher-loc" tagx="publisher-loc" title="publisher-loc">London, UK</span></div> </div> <div class="article-meta" tagx="article-meta" title="article-meta"><span class="pmcid" title="pmcid"> pmcid: <a href="http://www.ncbi.nlm.nih.gov/pubmed/4184305">4184305</a></span><span class="doi" title="doi"> doi: <a href="https://dx.doi.org/10.12688/f1000research.4711.2">10.12688/f1000research.4711.2</a></span><div class="article-categories" title="article-categories">
: <span class="subject" title="subject">Correspondence</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Articles</span>: <span class="subject" title="subject">Evolutionary/Comparative Genetics</span>: <span class="subject" title="subject">Genetics of the Immune System</span>: <span class="subject" title="subject">Leukocyte Development</span>: <span class="subject" title="subject">Leukocyte Signaling &amp;amp; Gene Expression</span></div> </div> <div class="title-group" tagx="title-group" title="title-group"> <div class="article-title" title="article-title">
Non-human <i>lnc-DC</i> orthologs encode Wdnm1-like protein</div> <div class="fn-group" title="fn-group"> <div class="fn-type-" title=""> <p>v2; ref status: indexed</p> </div> </div> </div> <div class="contrib-group" title="contrib-group"><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Dijkstra</span><span class="given-names" tagx="given-names" title="given-names">Johannes M.</span></span><a href="#c1">a</a><a href="#a1">1</a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ballingall</span><span class="given-names" tagx="given-names" title="given-names">Keith T.</span></span><a href="#a2">2</a></span><span class="citation_author_institution" id="a1">[1], </span><span class="citation_author_institution" id="a2">[2], </span></div> <div class="author-notes" title="author-notes"> <div class="corresp" title="corresp"><span class="label" tagx="label" title="label">a</span><span class="email" tagx="email" title="email">Dijkstra@fujita-hu.ac.jp</span></div> <div class="fn-type-con" title="con"> <p>JMD did most of the research and wrote the manuscript. KTB analyzed sequence databases of extinct hominids and carefully checked the manuscript.</p> </div> <div class="fn-type-conflict" title="conflict"> <p><b>Competing interests: </b>No competing interests were disclosed.</p> </div> </div><span class="pub-date-epub" title="pub-date-epub">epub: <span>2014-9-9</span></span><span class="pub-date-collection" title="pub-date-collection">collection: <span>2014</span></span><span class="volume" tagx="volume" title="volume">3</span><span class="elocation-id" tagx="elocation-id" title="elocation-id">160</span><span class="history" title="history"><span class="accepted" title="accepted">accepted: 2014-9-19</span></span><div class="permissions"><span class="copyright" title="copyright">(C) , <span class="copyright-year" tagx="copyright-year" title="copyright-year">2014</span></span><span class="license" title="license"><span class="license-p" title="license-p">This is an open access article distributed under the terms of the Creative Commons Attribution Licence, which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited.</span></span><span class="license" title="license"><span class="license-p" title="license-p">Data associated with the article are available under the terms of the Creative Commons Zero "No rights reserved" data waiver (CC0 1.0 Public domain dedication).</span></span></div><a class="self-uri" href="f1000research-3-5709.pdf" title="self-uri">f1000research-3-5709.pdf</a><div class="abstract" title="abstract"> <div class="abstract-title" title="abstract-title">
Abstract</div> <p>In a recent publication in <i>Science</i>, Wang <i>et al</i>. found a long noncoding RNA (lncRNA) expressed in human dendritic cells (DC), which they designated <i>lnc-DC</i>. Based on lentivirus-mediated RNA interference (RNAi) experiments in human and murine systems, they concluded that <i>lnc-DC</i> is important in differentiation of monocytes into DC. However, Wang <i>et al</i>. did not mention that their so-called "mouse <i>lnc-DC ortholog</i>" gene was already designated " <i>Wdnm1-like</i>" and is known to encode a small secreted protein. We found that incapacitation of the <i>Wdnm1-like</i> open reading frame (ORF) is very rare among mammals, with all investigated primates except for hominids having an intact ORF. The null-hypothesis by Wang et al. therefore should have been that the human <i>lnc-DC</i> transcript might only represent a non-functional relatively young evolutionary remnant of a protein coding locus. Whether this null-hypothesis can be rejected by the experimental data presented by Wang <i>et al</i>. depends in part on the possible off-target (immunogenic or otherwise) effects of their RNAi procedures, which were not exhaustive in regard to the number of analyzed RNAi sequences and control sequences. If, however, the conclusions by Wang <i>et al</i>. on their human model are correct, and they may be, current knowledge regarding the <i>Wdnm1-like</i> locus suggests an intriguing combination of different functions mediated by transcript and protein in the maturation of several cell types at some point in evolution. We feel that the article by Wang <i>et al</i>. tends to be misleading without the discussion presented here.</p> </div> <div class="funding-group" title="funding-group"> <div class="funding" title="funding">
Funding</div> <div class="funding-statement" title="funding-statement">
The author(s) declared that no grants were involved in the funding of this work.</div> </div> </div> <div class="amendmentsfromversion1" title="sec"><span class="label" tagx="label" title="label">Revised</span><div class="title" tagx="title" title="title">
Amendments from Version 1</div> <p>The excellent referee reports were mostly positive, but included a set of extra details and thoughts. For that we now refer to those reports. We now also mention mass spectrometry reports that support the existence of Wdnm1-like protein.</p> </div> </div> <div class="body" title="body"> <div class="correspondence" title="sec"> <div class="title" tagx="title" title="title">
Correspondence</div> <p>In their recent publication in <i>Science</i>, Wang <i>et al.</i><sup><a href="#ref-1">1</a></sup> aimed to identify lncRNAs involved in DC differentiation and function. In order to do this they used an established model of human DC differentiation from peripheral blood monocytes (Mo), based on addition of recombinant cytokines. They found that transcription of the human <i>Wdnm1-like</i> pseudogene ( <i>Wdnm1-like-ψ</i>), or <i>lnc-DC</i> as they call it, was robustly induced by the Mo-DC differentiation process. Furthermore, they found <i>Wdnm1-like-ψ</i> highly transcribed in other dendritic cells, and confirmed correlation of <i>Wdnm1-like-ψ</i> transcription with DC differentiation in several ways. To investigate a functional role of <i>Wdnm1-like-ψ</i> in their Mo-DC differentiation model, they used a lentivirus-mediated RNA interference (RNAi) system. The RNAi interference with <i>Wdnm1-like-ψ</i> fragments resulted in a pronounced effect on Mo-DC differentiation as measured by expression of genes and molecules involved in the immune system, the ability to take up antigen, and the capacity to stimulate T-helper cells. Wang <i>et al.</i> showed by a number of experiments that the <i>Wdnm1-like-ψ</i> transcript, in particular the 3’-end, has some specificity for binding to the STAT3 transcription factor and can reduce STAT3 dephosphorylation by phosphatase SHP1. And, importantly, they showed that in their human Mo-DC differentiation model the effect of STAT3 inhibition caused similar effects as knockdown of <i>Wdnm1-like-ψ</i>. They therefore postulated that human <i>Wdnm1-like-ψ</i> transcript is an important regulator of DC differentiation by enhancing STAT3 activity through prevention of STAT3 dephosphorylation by SHP1. The results and human model presented by Wang <i>et al.</i> are generally convincing, yet some questions remain, such as to why not for all experiments both "no transfection control" (used in a few experiments) and "control RNAi" (used in all experiments) were included, and why they only used a single RNAi control sequence. RNAi control sequences are relevant because off-target genes might be knocked down (e.g. Jackson <i>et al.</i><sup><a href="#ref-3">3</a></sup>), but also because the lentivirus system using short hairpin RNA (shRNA) can have immunogenic properties in an shRNA-sequence-dependent manner (e.g. Kenworthy <i>et al.</i><sup><a href="#ref-4">4</a></sup>). Notably, in some experiments Wang <i>et al.</i><sup><a href="#ref-1">1</a></sup> independently knocked down two different fragments of <i>Wdnm1-like-ψ</i>, with similar experimental results, thus reducing the chance that off-target effects of their RNAi systems influenced their conclusions. On the other hand, since the use of two positive RNAi systems suggests that Wang <i>et al.</i> were aware of the potential weaknesses of the system, this raises the question as to why they only used a single sequence for their RNAi control experiments. Regardless, we consider the part of their manuscript on human <i>Wdnm1-like-ψ</i> to be mostly solid and interesting, and the main reason why we are so (overly) critical is that acceptance of the model for human <i>Wdnm1-like-ψ</i> function as proposed by Wang <i>et al.</i> leads to a quite spectacular evolutionary model, as outlined below. Our view, which is supported in the accompanying referee report provided by Dr. Burchard, is that such a spectacular claim requires very robust evidence which in this case probably requires a higher number of RNAi controls.</p> <p>Whereas the presentation of their human data appears to be mostly correct, we feel that the way Wang <i>et al.</i><sup><a href="#ref-1">1</a></sup> present their mouse model is inappropriate. Wang <i>et al.</i> used a mouse model to confirm that knockdown of <i>Wdnm1-like(-ψ)</i> results in impaired DC differentiation. Technically these experiments in mice worked as they expected, indicated both by <i>in vitro</i> and <i>in vivo</i> results, and they also found that knockdown of murine <i>Wdnm1-like</i> could lead to reduction of STAT3 phosphorylation, although they apparently did not check if murine <i>Wdnm1-like</i> transcript can bind STAT3. However, even though Wang <i>et al.</i> refer to Gene symbol 110000G20Rik which mentions "Wdnm1-like", they only present the readers with the term "mouse lnc-DC ortholog". This is highly misleading as it suggests that the transcript also relates to a long noncoding RNA in mice. The authors even state " <i>Taken together, our data suggest that lnc-DC is vital for DC differentiation in both human and mice</i>". However, in mice the gene encodes a functional Wdnm1-like protein, as shown by recombinant analysis <sup><a href="#ref-2">2</a></sup>, and our extensive analysis of mammalian sequence databases indicates that the <i>Wdnm1-like</i> ORF incapacitation is very rare among mammals. Actually, among the eutherian mammals that we investigated and for which the relevant genomic region information was available, only humans (and Neanderthals and Denisovans) lacked the capacity to encode the otherwise highly conserved Wdnm1-like protein sequence ( <a href="#f1">Figure 1</a>). At the level of the genus <i>Pan</i> (chimpanzee and bonobo) the N-terminus of the predicted mature protein differs from consensus, but even in gorilla and orangutan the encoded Wdnm1-like protein appears fully normal. So possibly the function of the Wdnm1-like protein started to lose importance after separation of <i>Homo</i>/ <i>Pan</i> from the other apes, which is quite recent in evolutionary terms. Calculation of synonymous (ds) versus nonsynonymous (dn) nucleotide substitution rates, using software available at <a href="http://www.hiv.lanl.gov/content/sequence/SNAP/SNAP.html">http://www.hiv.lanl.gov/content/sequence/SNAP/SNAP.html</a>, indicates conservation of Wdnm1-like protein function after most of the animals shown in <a href="#f1">Figure 1</a> had separated in evolution. Namely, in pairwise comparisons, for the depicted set of eutherian mammals except <i>Pan</i>/ <i>Homo</i> the average ds/dn ratio is 3.5, and for the set of primates except <i>Pan</i>/ <i>Homo</i> this value is 3.0. Thus, although in each individual species experimental evidence would still be required, it is expected that most eutherian mammals possess functional Wdnm1-like protein. Probably because of lack of directed investigations, naturally expressed endogenous full-size Wdnm1-like proteins have not yet been reported. However, our search of the PeptideAtlas database of peptides identified by mass spectrometry identified a rat ( <i>Rattus Norwegicus</i>) Wdnm1-like fragment encoded by properly spliced <i>Wdnm1-like</i> transcript ( <a href="http://www.peptideatlas.org">http://www.peptideatlas.org</a>, peptide PAp03984316).</p> <div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/"><span class="label" tagx="label" title="label">Figure 1. </span></a></div>   <div class="title" tagx="title" title="title">
The highly conserved coding sequences of mammalian <i>Wdnm1-like</i>.</div> <p>The figure shows deduced Wdnm1-like amino acid sequences plus their coding nucleotide sequences in representative mammals.</p> <p>After evolutionary separation from gorilla, in an ancestor common to the genera <i>Pan</i> (including chimpanzee and bonobo) and <i>Homo</i> (including human, Neanderthal and Denisovan), the nucleotide region coding the N-terminus of the mature Wdnm1-like protein was modified by deletions (yellow shading). Nevertheless, in the genus <i>Pan</i> the <i>Wdnm1-like</i> open reading frame (ORF) remained intact. Only in <i>Homo</i> the Wdnm1-like coding sequence was interrupted by a frameshift through a single nucleotide deletion (red shading) within the leader peptide coding region (the resulting change in amino acids is shaded grey). For the human <i>Wdnm1-like</i> locus several transcripts (splicoforms) were found (Ensembl reports <a href="http://asia.ensembl.org/Homo_sapiens/Transcript/Summary?db=core;g=ENSG00000261040;r=17:58160927-58169246;t=ENST00000590346">ENST00000590346</a>, <a href="http://asia.ensembl.org/Homo_sapiens/Transcript/Summary?db=core;g=ENSG00000261040;r=17:58160927-58169246;t=ENST00000588180">ENST00000588180</a>, <a href="http://asia.ensembl.org/Homo_sapiens/Transcript/Summary?db=core;g=ENSG00000261040;r=17:58160927-58169246;t=ENST00000587298">ENST00000587298</a>, <a href="http://asia.ensembl.org/Homo_sapiens/Transcript/Summary?db=core;g=ENSG00000261040;r=17:58160927-58169246;t=ENST00000590012">ENST00000590012</a>, <a href="http://asia.ensembl.org/Homo_sapiens/Transcript/Summary?db=core;g=ENSG00000261040;r=17:58160927-58169246;t=ENST00000589987">ENST00000589987</a>, <a href="http://asia.ensembl.org/Homo_sapiens/Transcript/Summary?db=core;g=ENSG00000261040;r=17:58160927-58169246;t=ENST00000592556">ENST00000592556</a>, <a href="http://asia.ensembl.org/Homo_sapiens/Transcript/Summary?db=core;g=ENSG00000261040;r=17:58160927-58169246;t=ENST00000566140">ENST00000566140</a>, and <a href="http://asia.ensembl.org/Homo_sapiens/Transcript/Summary?db=core;g=ENSG00000261040;r=17:58160927-58169246;t=ENST00000589777">ENST00000589777</a>); however, we agree with Wang <i>et al.</i><sup><a href="#ref-1">1</a></sup> that software investigation of the known transcripts suggests that the human <i>Wdnm1-like</i> locus does not code a functional protein (analyses not shown).</p> <p>The marsupial <i>Monodelphis domestica</i> (opossum) was the only non-eutherian mammal for which we could identify <i>Wdnm1-like</i>, situated upstream of the gene <i>HEAT Repeat Containing 6</i> ( <i>HEATR6</i>) like its ortholog in eutherian mammals. To avoid gaps in the bulk of the figure, the N-terminus of the opossum sequence is not perfectly aligned with Wdnm1-like of eutherian mammals.</p> <p>Except for rabbit (see Methods section), the figure shows the ORFs of sequences corresponding to the murine Wdnm1-like protein coding transcript of NCBI accession NM_183249, while other (possible) splicoforms are neglected. The intron site is indicated by a downward triangle. Intron sequences are not shown, but the below listed genomic sequence reports agree with GT-AG borders. For most of the species, the depicted sequences were supported by transcript reports, as exemplified per species in the Methods section. In the figure, dashes indicate gaps that were introduced for optimal sequence alignment. The alignments were performed by hand.</p> <p>Amino acid sequences are indicated above the second nucleotides of codons. Basic residues are indicated in red, acidic residues in blue, and green residues are more hydrophilic than the orange ones (following reference <sup><a href="#ref-9">9</a></sup>). Cysteines are in violet. Asterisks correspond with stop codons. Predicted leader sequences are underlined.</p> <p>The mouse <i>Wdnm1-like</i> sequence was designated "mouse lnc-DC ortholog" by Wang <i>et al.</i><sup><a href="#ref-1">1</a></sup>, and they targeted the regions shaded blue and green for transcript knockdown by "RNAi-1" and "RNAi-2", respectively, using a lentivirus-mediated RNA interference system.</p>   </div> <p>The name <i>Wdnm1-like</i> was first coined by Adachi <i>et al.</i><sup><a href="#ref-5">5</a></sup>, who found that <i>Wdnm1-like</i> transcript was differentially expressed in limbal versus central corneal epithelia in rat, and who observed similarity of the encoded protein with Wdnm1. Within the serial analysis of gene expression (SAGE) experiment by Adachi <i>et al.</i><sup><a href="#ref-5">5</a></sup>, <i>Wdnm1-like</i> comprised the most abundant SAGE tag present exclusively in the limbal library, and the authors hypothesized that Wdnm1-like might be a marker of limbal stem cells. They could, however, not rule out the possibility that Wdnm1-like was expressed by other cell types present in limbal epithelia, such as for example dendritic cells. A later study on rodent <i>Wdnm1-like</i> was performed in mice by Wu and Smas <sup><a href="#ref-2">2</a></sup>. Wu and Smas got interested in <i>Wdnm1-like</i> after they found it highly upregulated upon differentiation of preadipocytes into adipocytes. They found <i>Wdnm1-like</i> to be selectively expressed in liver and adipose tissue, and enriched in white adipose depots versus brown. Recombinant expression of tagged murine Wdnm1-like in HT1080 human fibrosarcoma cells revealed a small secreted protein <sup><a href="#ref-2">2</a></sup>. Because Wdnm1-like is a distant member of the whey acidic protein/four-disulfide core (WAP/4-DSC) family, of which several members have roles as proteinase inhibitors, Wu and Smas speculated that Wdnm1-like might have a similar function. An important class of extracellular proteases involved in adipocyte differentiation are the matrix metalloproteinases (MMPs), which can degrade extracellular matrix (ECM) components. Therefore, Wu and Smas investigated whether MMPs expressed by HT1080 were affected by the recombinant Wdnm1-like expression, and they found an increased amount of the active form of MMP-2 <sup><a href="#ref-2">2</a></sup>. Thus, rather than having an inhibitory effect, Wdnm1-like appears to enhance activation of a protease. Wu and Smas conclude with " <i>Future studies are required to address the mechanism(s) underlying the function and regulation of adipocyte-secreted Wdnm1-like</i>" <sup><a href="#ref-2">2</a></sup>, and according to literature this situation has not changed since then.</p> <p>Looking at the combined publications, a very complicated picture emerges. In most mammals the <i>Wdnm1-like</i> locus encodes a protein, with humans as an exception which is possibly unique. In rat, <i>Wdnm1-like</i> is differentially expressed in limbal versus central corneal epithelia <sup><a href="#ref-4">4</a></sup>. In mouse, <i>Wdnm1-like</i> is expressed upon adipogenesis, and Wdnm1-like protein enhances the production of active MMP-2 <sup><a href="#ref-2">2</a></sup>. In human and mouse, the <i>Wdnm1-like(-ψ</i>) transcript appears functionally associated with dendritic cell differentiation, and at least in humans this may be mediated by binding of the transcript to STAT3 <sup><a href="#ref-1">1</a></sup>. This leaves questions for future research such as, for example, whether human <i>Wdnm1-like-ψ</i> transcript is also associated with adipogenesis, and whether murine <i>Wdnm1-like</i> transcript exerts its function on DC differentiation by binding to STAT3 or by encoding Wdnm1-like protein. Supporting that the Wdnm1-like proteins and transcripts in some extinct or extant animals may have (had) synergetic functions, is the fact that differentiation of both adipocytes and limbal epithelial cells can involve STAT3 <sup><a href="#ref-6">6</a>, <a href="#ref-7">7</a></sup>. So, despite our points of criticism, we think that the results and human model by Wang <i>et al.</i> may be valid and part of a more complex evolutionary scenario that involves distinct functions at the transcript and protein level, and a number of different tissues and cell types. In general, we think that studies on long noncoding RNAs typically require discussion of the evolutionary context <sup><a href="#ref-8">8</a></sup>, especially when dealing with wide species borders such as between human and mouse.</p> <div class="additionalnote1" title="sec"> <div class="title" tagx="title" title="title">
Additional note 1</div> <p>A nice speculation allowed by the combined referenced articles is that Wdnm1-like protein might promote Mo-DC differentiation in humans. After all, murine Wdnm1-like protein was found to enhance MMP-2 activity of human HT1080 cells <sup><a href="#ref-2">2</a></sup>, concluding that humans did not lose their sensitivity to Wdnm1-like protein.</p> </div> <div class="additionalnote2" title="sec"> <div class="title" tagx="title" title="title">
Additional note 2</div> <p>We did not feel comfortable with the amount of space and visibility the editors of the journal <i>Science</i> were able to offer us via their commenting mechanism for the discussion presented here. Therefore, we declined their offer, and instead deemed publication in <i>F1000Research</i> a more appropriate vehicle. Through F1000Research we asked specialists and the corresponding authors of several of the referenced articles, including the article by Wang <i>et al.</i>, to provide referee reports (which may also include broad views) on our discussion. We are pleased with the excellent referee reports received from Dr. Smas and Dr. Ren, and Dr. Burchard, and recommend them to the readers of our article. Importantly, Drs. Smas and Ren confirmed that we correctly summarized reports on rodent <i>Wdnm1-like</i> and listed additional evidence to that matter, while Dr. Burchard substantiated and detailed our notion that the RNAi experiments by Wang <i>et al.</i> were inconsistent and probably incomplete. We would welcome comments from the group of Wang <i>et al.</i> and also encourage other researchers to leave comments.</p> </div> <div class="additionalnote3" title="sec"> <div class="title" tagx="title" title="title">
Additional note 3</div> <p>Wdnm1-like protein appears to be very interesting. Not only may it be involved in differentiation of several cell types, it also is intriguing because it appears highly conserved throughout eutherian mammals and (rather) uniquely lost in hominids. It may help determine what makes us human.</p> </div> </div> <div class="methods" title="methods"> <div class="title" tagx="title" title="title">
Methods</div> <p>The partial <i>Wdnm1-like</i> sequence information available for extinct hominids, namely, for Neanderthal and Denisovan, was retrieved using the UCSC genome browser ( <a href="http://genome.ucsc.edu">http://genome.ucsc.edu</a>). All other sequences shown in the figure were retrieved from Ensembl ( <a href="http://www.ensembl.org/">www.ensembl.org/</a>) or NCBI ( <a href="http://www.ncbi.nlm.nih.gov/">http://www.ncbi.nlm.nih.gov/</a>) databases. For a representative list of model species, we investigated database sequences of all mammals for which genomic sequences are available in the Ensembl database, and also of <i>Pan paniscus</i> (bonobo). For some of those animals sequence information for the <i>Wdnm1-like</i> ORF or for its expected genomic site was incomplete, and in such case the sequence is not included in the alignment figure.</p> <p>Leader peptide sequences were predicted by SignalP software ( <a href="http://www.cbs.dtu.dk/services/SignalP/">www.cbs.dtu.dk/services/SignalP/</a>) and are underlined. For the species <i>Panio anubis</i> (olive baboon), <i>Heterocephalus glaber</i> (naked mole-rat), <i>Myotis lucifugus</i> (microbat), and <i>Dasypus novemcintus</i> (armadillo), besides the here indicated evolutionary conserved cleavage site, the SignalP software also predicted an alternative cleavage site with a calculated higher likelihood (not shown).</p> <p><b>Species-specific information related to sequences depicted in <a href="#f1">Figure 1</a>:</b></p> <div class="homosapiens(human)" title="sec"> <div class="title" tagx="title" title="title"><i>Homo sapiens</i> (human)</div> <p>The depicted human <i>Wdnm1-like</i> pseudogene sequence maps within Ensembl database GRCh37, Chr.17 positions 58162470-to-58165647, reverse orientation. Furthermore, the depicted sequence corresponds with positions 182-to-366 of the transcript sequence of NCBI accession <a href="http://www.google.co.jp/url?sa=t&amp;rct=j&amp;q=&amp;esrc=s&amp;source=web&amp;cd=1&amp;cad=rja&amp;uact=8&amp;ved=0CB4QFjAA&amp;url=http://www.ncbi.nlm.nih.gov/nuccore/267844894&amp;ei=Oji2U_LBHY-E8gXqk4HgAQ&amp;usg=AFQjCNH61bIObnUQh8yn7dWh0KJjCEnWNA&amp;sig2=hYUI7tCB0LWUqQ_4Tv5KNw&amp;bvm=bv.70138588,d.dGc">NR_030732.1</a>.</p> </div> <div class="homosapiens(neanderthal)(whetherneanderthalshouldbeconsideredasubspeciesofhomosapiensisamatterofdebate)" title="sec"> <div class="title" tagx="title" title="title"><i>Homo sapiens</i> (Neanderthal) (whether Neanderthal should be considered a subspecies of <i>Homo sapiens</i> is a matter of debate)</div> <p>The depicted Neanderthal sequence was identified from genomic DNA fragments (Read names: M_SL-XAT_0004_FC30PMDAAXX:1:87:384:343, M_BIOLAB29_Run_PE51_1:2:9:981:262 and C_M_SOLEXA-GA04_JK_PE_SL21:8:99:944:526) isolated from the Vi33.16 and Vi33.25 Neanderthal samples <sup><a href="#ref-10">10</a></sup> using the UCSC genome browser by comparison with the human Wdnm1-like sequence. The depicted sequence fragment is identical to that of human.</p> </div> <div class="homosapiens(denisovan)(whetherdenisovanshouldbeconsideredasubspeciesofhomosapiensisamatterofdebate)" title="sec"> <div class="title" tagx="title" title="title"><i>Homo sapiens</i> (Denisovan) (whether Denisovan should be considered a subspecies of <i>Homo sapiens</i> is a matter of debate)</div> <p>The depicted Denisovan sequence was obtained as described for the Neanderthal sequences, and corresponds to part of the read M_SOLEXA-GA02_00040_PEdi_MM_3:8:112:19220:10730#AACCATG,CTCGATG (Meyer <i>et al.</i><sup><a href="#ref-11">11</a></sup>). The depicted sequence fragment is identical to that of human.</p> </div> <div class="pantroglodytes(chimpanzee)" title="sec"> <div class="title" tagx="title" title="title"><i>Pan troglodytes</i> (chimpanzee)</div> <p>The depicted chimpanzee <i>Wdnm1-like</i> sequence maps within Ensembl database CHIMP2.1.4, Chr.17 positions 588018940-to-58805072, reverse orientation. Transcription is supported by NCBI SRA database sequence reports, such as for example gnl|SRA|DRR003370.54864751.1 of experiment set DRX002694.</p> </div> <div class="panpaniscus(bonobo,orpygmychimpanzee)" title="sec"> <div class="title" tagx="title" title="title"><i>Pan paniscus</i> (bonobo, or pygmy chimpanzee)</div> <p>The depicted bonobo <i>Wdnm1-like</i> sequence maps within the genomic sequence of NCBI database accession <a href="http://www.ncbi.nlm.nih.gov/Traces/wgs/?val=AJFE01&amp;page=1&amp;display=contig&amp;search=AJFE01016111">gb|AJFE01016111.1|</a>, positions 11414-to-14585, forward orientation. Transcription is supported by NCBI SRA database sequence reports, such as for example gnl|SRA|SRR873628.59588401.2 of experiment set SRX290737.</p> </div> <div class="gorillagorilla(gorilla)" title="sec"> <div class="title" tagx="title" title="title"><i>Gorilla gorilla</i> (gorilla)</div> <p>The depicted gorilla <i>Wdnm1-like</i> sequence maps within Ensembl database gorGor3.1, Chr.5 positions 23775798-to-23778981, forward orientation. Transcription is supported by NCBI SRA database sequence reports, such as for example gnl|SRA|SRR306801.5146816.1 of experiment set SRX081945.</p> </div> <div class="pongoabelii(sumatranorangutan)" title="sec"> <div class="title" tagx="title" title="title"><i>Pongo abelii</i> (Sumatran orangutan)</div> <p>The depicted orangutan <i>Wdnm1-like</i> sequence maps within Ensembl database PPYG2, Chr.17 32711864-to-32715478, forward orientation. This is a recent intrachromosomal duplication of the original <i>Wdnm1-like</i> gene. The Ensembl database shows that Sumatran orangutan still has at least part of that original <i>Wdnm1-like</i> gene upstream of <i>HEATR6</i>, but information of that region is incomplete. Evidence for transcription of <i>Wdnm1-like</i> in orangutan is provided by NCBI SRA database sequence reports <i>for Pongo pygmaeus</i> (Bornean orangutan), such as for example gnl|SRA|SRR306799.12707499.1 of experiment set SRX081943.</p> </div> <div class="nomascusleucogenys(gibbon)" title="sec"> <div class="title" tagx="title" title="title"><i>Nomascus leucogenys</i> (gibbon)</div> <p>The depicted gibbon <i>Wdnm1-like</i> sequence maps within the genomic sequence of NCBI database accession <a href="http://www.ncbi.nlm.nih.gov/Traces/wgs/?val=ADFV01&amp;page=1&amp;display=contig&amp;search=ADFV01146912">gb|ADFV01146912.1|</a>, positions 1414-to-4561, reverse orientation.</p> </div> <div class="papioanubis(olivebaboon)" title="sec"> <div class="title" tagx="title" title="title"><i>Papio anubis</i> (olive baboon)</div> <p>The depicted olive <i>Wdnm1-like</i> sequence maps within Ensembl database Panu_2.0, scaffold JH685681 positions 60156-to-64601, reverse orientation. Transcription is supported by NCBI SRA database sequence reports, such as for example gnl|SRA|SRR1045089.118535973.1 of experiment set SRR1045089.</p> </div> <div class="macacamulatta(rhesusmacaque)" title="sec"> <div class="title" tagx="title" title="title"><i>Macaca mulatta</i> (rhesus macaque)</div> <p>The depicted rhesus macaque <i>Wdnm1-like</i> sequence maps within Ensembl database MMUL_1, scaffold 1099548049739 positions 121534-to-124737, forward orientation. Transcription is supported by NCBI SRA database sequence reports, such as for example gnl|SRA|SRR1240160.28991243.2 of experiment set SRR1240160.</p> </div> <div class="macacafascicularis(crab-eatingmacaque)" title="sec"> <div class="title" tagx="title" title="title"><i>Macaca fascicularis</i> (crab-eating macaque)</div> <p>The depicted crab-eating macaque <i>Wdnm1-like</i> sequence maps within the genomic sequence of NCBI database accession <a href="http://www.ncbi.nlm.nih.gov/Traces/wgs/?val=AEHL01&amp;page=1&amp;display=contig&amp;search=AEHL01027073">gb|AEHL01027073.1|</a>, positions 5255-to-8524, forward orientation. Transcription is supported by NCBI SRA database sequence reports, such as for example gnl|SRA|DRR001354.3296367.1 of experiment set DRX000951.</p> </div> <div class="chlorocebussabaeus(vervetmonkey)" title="sec"> <div class="title" tagx="title" title="title"><i>Chlorocebus sabaeus</i> (vervet monkey)</div> <p>The depicted <i>Wdnm1-like</i> sequence maps within Ensembl database ChlSab1.0, Chr.16 positions 29807079-to-29810262, reverse orientation. Transcription is supported by NCBI SRA database sequence reports for the closely related species <i>Chlorocebus aethiops</i> (green monkey), such as for example gnl|SRA|SRR1178509.592424.2 of experiment set SRR1178509.</p> </div> <div class="saimiriboliviensis(boliviansquirrelmonkey)" title="sec"> <div class="title" tagx="title" title="title"><i>Saimiri boliviensis</i> (Bolivian squirrel monkey)</div> <p>The depicted squirrel monkey <i>Wdnm1-like</i> sequence maps within Ensembl database SalBol1.0, scaffold JH378137 positions 636410-to-639575, forward orientation. Transcription is supported by NCBI SRA database sequence reports, such as for example gnl|SRA|SRR500949.3269772.2 of experiment set SRX149650.</p> </div> <div class="callithrixjacchus(marmoset)" title="sec"> <div class="title" tagx="title" title="title"><i>Callithrix jacchus</i> (marmoset)</div> <p>The depicted marmoset <i>Wdnm1-like</i> sequence maps within Ensembl database C_jacchus3.2.1, Chr.5 positions 88345809-to-88348914, reverse orientation. Transcription is supported by NCBI database accession <a href="http://www.ncbi.nlm.nih.gov/Traces/wgs/?val=GAMR01&amp;page=1&amp;display=contig&amp;search=GAMR01043615">gb|GAMR01043615.1|</a>.</p> </div> <div class="tarsiussyrichta(tarsier)" title="sec"> <div class="title" tagx="title" title="title"><i>Tarsius syrichta</i> (tarsier)</div> <p>The depicted tarsier <i>Wdnm1-like</i> sequence maps within Ensembl database tarSyr1, scaffold_1716 positions 51738-to-55873, reverse orientation.</p> </div> <div class="otolemur_garnettii(bushbaby)" title="sec"> <div class="title" tagx="title" title="title"><i>Otolemur_garnettii</i> (bushbaby)</div> <p>The depicted bushbaby <i>Wdnm1-like</i> sequence maps within Ensembl database OtoGar3, scaffold GL873613 positions 7509108-to-7514627, reverse orientation.</p> </div> <div class="microcebusmurinus(mouselemur)" title="sec"> <div class="title" tagx="title" title="title"><i>Microcebus murinus</i> (mouse lemur)</div> <p>The depicted mouse lemur <i>Wdnm1-like</i> sequence maps within Ensembl database micMur1, GeneScaffold_1067 positions 49762-to-53887, reverse orientation. Transcription is supported by NCBI SRA database sequence reports, such as for example gnl|SRA|SRR832933.720157201.1 of experiment set SRX270644.</p> </div> <div class="tupaiachinensis(chinesetreeshrew)" title="sec"> <div class="title" tagx="title" title="title"><i>Tupaia chinensis</i> (Chinese tree shrew)</div> <p>The depicted Chinese tree shrew <i>Wdnm1-like</i> sequence maps within Ensembl database TREESHREW, scaffold_15853 positions 2941-to-6216, reverse orientation. Transcription is supported by NCBI SRA database sequence reports, such as for example gnl|SRA|SRR518934.53798716.1 of experiment set SRX157966.</p> </div> <div class="musmusculus(mouse)" title="sec"> <div class="title" tagx="title" title="title"><i>Mus musculus</i> (mouse)</div> <p>The depicted mouse <i>Wdnm1-like</i> sequence maps within Ensembl database GRCm38, Chr.11 positions 83747027-to-83749327, forward orientation. Transcription is supported by for example NCBI accession <a href="http://www.google.co.jp/url?sa=t&amp;rct=j&amp;q=&amp;esrc=s&amp;source=web&amp;cd=2&amp;cad=rja&amp;uact=8&amp;ved=0CCUQFjAB&amp;url=http://www.ncbi.nlm.nih.gov/nuccore/34304021&amp;ei=B0G2U7PdLcmk8AW-h4DACA&amp;usg=AFQjCNFopx34_IgH7lUmpfK7fBQGHhBKZw&amp;sig2=vUTStbQFO-Sub3v0Vkt1xA&amp;bvm=bv.70138588,d.dGc">NM_183249.1</a>.</p> </div> <div class="rattusnorvegicus(rat)" title="sec"> <div class="title" tagx="title" title="title"><i>Rattus norvegicus</i> (rat)</div> <p>The depicted rat <i>Wdnm1-like</i> sequence maps within Ensembl database Rnor_5.0, Chr.10 positions 70671110-to-70673427, forward orientation. Transcription is supported by for example NCBI accession <a href="http://www.ncbi.nlm.nih.gov/nucleotide/119351088?report=genbank&amp;log$=nuclalign&amp;blast_rank=2&amp;RID=VCGEA52501R">gb|EF122001.1|</a>.</p> </div> <div class="microtusochrogaster(prairievole)" title="sec"> <div class="title" tagx="title" title="title"><i>Microtus ochrogaster</i> (prairie vole)</div> <p>The depicted prairie vole <i>Wdnm1-like</i> sequence maps within Ensembl database MicOch1.0, Chr.7 positions 15310620-to-15312860, reverse orientation. According to the Ensembl database the prairie vole also has an intronless copy of <i>Wdnm1-like</i> gene on Chr.X (not shown). Transcription is supported by NCBI SRA database sequence reports, such as for example gnl|SRA|SRR058428.108679.2 of experiment set SRX018513.</p> </div> <div class="cricetulusgriseus(chinesehamster)" title="sec"> <div class="title" tagx="title" title="title"><i>Cricetulus griseus</i> (Chinese hamster)</div> <p>The depicted hamster <i>Wdnm1-like</i> sequence maps within the genomic sequence of NCBI database accession <a href="http://www.ncbi.nlm.nih.gov/Traces/wgs/?val=AMDS01&amp;page=1&amp;display=contig&amp;search=AMDS01007412">gb|AMDS01007412.1|</a>, positions 15363-to-17750, forward orientation.</p> </div> <div class="dipodomysordii(kangaroorat)" title="sec"> <div class="title" tagx="title" title="title"><i>Dipodomys ordii</i> (kangaroo rat)</div> <p>The depicted kangaroo rat <i>Wdnm1-like</i> sequence maps within Ensembl database dipOrd1, scaffold_2778 positions 48464-to-52516, reverse orientation.</p> </div> <div class="ictidomystridecemlineatus(thirteen-linedgroundsquirrel)" title="sec"> <div class="title" tagx="title" title="title"><i>Ictidomys tridecemlineatus</i> (thirteen-lined ground squirrel)</div> <p>The depicted squirrel <i>Wdnm1-like</i> sequence maps within Ensembl database spetri2, scaffold JH393300 positions 533158-to-536139, forward orientation.</p> </div> <div class="heterocephalusglaber(nakedmole-rat)" title="sec"> <div class="title" tagx="title" title="title"><i>Heterocephalus glaber</i> (naked mole-rat)</div> <p>The depicted naked mole-rat <i>Wdnm1-like</i> sequence maps within Ensembl database HetGla_female_1.0, scaffold JH602188 positions 3555009-to-3557720, forward orientation.</p> </div> <div class="caviaporcellus(domesticguineapig)" title="sec"> <div class="title" tagx="title" title="title"><i>Cavia porcellus</i> (domestic guinea pig)</div> <p>The depicted guinea pig <i>Wdnm1-like</i> sequence maps within Ensembl database cavPor3, scaffold_32 positions 10788821-to-10791159, reverse orientation.</p> </div> <div class="oryctolaguscuniculus(rabbit)" title="sec"> <div class="title" tagx="title" title="title"><i>Oryctolagus cuniculus</i> (rabbit)</div> <p>The depicted rabbit <i>Wdnm1-like</i> sequence maps within Ensembl database OryCun2.0, Chr.19 positions 25079843-to-25084775, forward orientation. The underlined part in Italic font at the 3’end belongs to a third exon. Transcription is supported by for example NCBI accession <a href="http://www.ncbi.nlm.nih.gov/Traces/wgs/?val=GBCH01&amp;page=1&amp;display=contig&amp;search=GBCH01008538">gb|GBCH01008538.1|</a>.</p> </div> <div class="ochotonaprinceps(pika)" title="sec"> <div class="title" tagx="title" title="title"><i>Ochotona princeps</i> (pika)</div> <p>The depicted pika <i>Wdnm1-like</i> sequence maps within Ensembl database OchPri3, scaffold JH802106 positions 113719-to-116807, forward orientation. Transcription is supported by NCBI SRA database sequence reports, such as for example gnl|SRA|SRR850200.108627.2 of experiment set SRX277346.</p> </div> <div class="bostaurus(cattle)" title="sec"> <div class="title" tagx="title" title="title"><i>Bos taurus</i> (cattle)</div> <p>The depicted cattle <i>Wdnm1-like</i> sequence maps within Ensembl database UMD3.1, Chr.19 positions 14485956-to-14490393, reverse orientation. Transcription is supported by for example NCBI accession <a href="http://www.ncbi.nlm.nih.gov/nucleotide/7054708?report=genbank&amp;log$=nuclalign&amp;blast_rank=4&amp;RID=VCH7BVKU01R">gb|AW484602.1|</a>.</p> </div> <div class="ovisaries(sheep)" title="sec"> <div class="title" tagx="title" title="title"><i>Ovis aries</i> (sheep)</div> <p>The depicted sheep <i>Wdnm1-like</i> sequence maps within Ensembl database Oar_v3.1, Chr.11 positions 13759463-to-13763966, reverse orientation. Transcription is supported by for example NCBI accession <a href="http://www.ncbi.nlm.nih.gov/nucleotide/44903219?report=genbank&amp;log$=nuclalign&amp;blast_rank=1&amp;RID=VCHNF29T015">gb|CK830678.1|</a>.</p> </div> <div class="tursiopstruncatus(dolphin)" title="sec"> <div class="title" tagx="title" title="title"><i>Tursiops truncatus</i> (dolphin)</div> <p>The depicted dolphin <i>Wdnm1-like</i> sequence maps within the genomic sequence of NCBI database accession <a href="http://www.ncbi.nlm.nih.gov/Traces/wgs/?val=ABRN02&amp;page=1&amp;display=contig&amp;search=ABRN02348024">gb|ABRN02348024.1|</a>, positions 2742-to-5945, forward orientation.</p> </div> <div class="susscrofa(pig)" title="sec"> <div class="title" tagx="title" title="title"><i>Sus scrofa</i> (pig)</div> <p>The depicted pig <i>Wdnm1-like</i> sequence maps within the genomic sequence of NCBI database accession <a href="http://www.ncbi.nlm.nih.gov/Traces/wgs/?val=AJKK01&amp;page=1&amp;display=contig&amp;search=AJKK01193461">gb|AJKK01193461.1|</a>, positions 3786-to-7100, reverse orientation. Transcription is supported by for example NCBI accession <a href="http://www.ncbi.nlm.nih.gov/nucleotide/371921287?report=genbank&amp;log$=nuclalign&amp;blast_rank=1&amp;RID=VCJ1JR2B015">dbj|AK399701.1|</a>.</p> </div> <div class="vicugnapacos(alpaca)" title="sec"> <div class="title" tagx="title" title="title"><i>Vicugna pacos</i> (alpaca)</div> <p>The depicted alpaca <i>Wdnm1-like</i> sequence maps within Ensembl database vicPac1, GeneScaffold_1352 positions 716864-to-719675, reverse orientation.</p> </div> <div class="equuscaballus(horse)" title="sec"> <div class="title" tagx="title" title="title"><i>Equus caballus</i> (horse)</div> <p>The depicted horse Wdnm1-like sequence maps within Ensembl database EquCab2, Chr.11 positions 36986601-to-36989473, reverse orientation. The Ensembl database indicates additional copies of <i>Wdnm1-like</i> on Chr.11 (not shown). Transcription is supported by for example NCBI accession <a href="http://www.ncbi.nlm.nih.gov/nucleotide/60718810?report=genbank&amp;log$=nuclalign&amp;blast_rank=1&amp;RID=VCJN66DX014">gb|DN508620.1|</a>.</p> </div> <div class="ceratotheriumsimum(rhinoceros)" title="sec"> <div class="title" tagx="title" title="title"><i>Ceratotherium simum</i> (rhinoceros)</div> <p>The depicted rhinoceros <i>Wdnm1-like</i> sequence maps within Ensembl database CerSimSim1, scaffold JH767772 positions 17445128-to-17447968, reverse orientation.</p> </div> <div class="myotislucifugus(microbat)" title="sec"> <div class="title" tagx="title" title="title"><i>Myotis lucifugus</i> (microbat)</div> <p>The depicted microbat <i>Wdnm1-like</i> sequence maps within Ensembl database Myoluc2.0, scaffold_GL430154 positions 92198-to-94525, reverse orientation.</p> <p>Transcription is supported by NCBI SRA database sequence reports, such as for example gnl|SRA|SRR1013468.27145136.1 of experiment set SRR1013468.</p> </div> <div class="feliscatus(cat)" title="sec"> <div class="title" tagx="title" title="title"><i>Felis catus</i> (cat)</div> <p>The depicted cat <i>Wdnm1-like</i> sequence maps within the genomic sequence of NCBI database accession <a href="http://www.ncbi.nlm.nih.gov/Traces/wgs/?val=AANG02&amp;page=1&amp;display=contig&amp;search=AANG02057756">gb|AANG02057756.1|</a>, positions 9507-to-13123, forward orientation. Transcription is supported by NCBI SRA database sequence reports, such as for example gnl|SRA|SRR835496.27404932.1 of experiment set SRX272142.</p> </div> <div class="canislupusfamiliaris(dog)" title="sec"> <div class="title" tagx="title" title="title"><i>Canis lupus familiaris</i> (dog)</div> <p>The depicted dog <i>Wdnm1-like</i> sequence maps within Ensembl database CanFam3.1, Chr.9 positions 37619501-to-37622242, reverse orientation. Transcription is supported by for example NCBI accession <a href="http://www.ncbi.nlm.nih.gov/nucleotide/67566410?report=genbank&amp;log$=nuclalign&amp;blast_rank=1&amp;RID=VCK1PJY701R">gb|DR107055.1|</a>.</p> </div> <div class="ailuropodamelanoleuca(panda)" title="sec"> <div class="title" tagx="title" title="title"><i>Ailuropoda melanoleuca</i> (panda)</div> <p>The depicted panda <i>Wdnm1-like</i> sequence maps within Ensembl database ailMel1, scaffold GL193203 positions 54404-to-57462, forward orientation.</p> </div> <div class="mustelaputorius(ferret)" title="sec"> <div class="title" tagx="title" title="title"><i>Mustela putorius</i> (ferret)</div> <p>The depicted ferret <i>Wdnm1-like</i> sequence maps within Ensembl database MusPutFur1.0, scaffold GL896917 positions 9435086-to-9438171, forward orientation. Transcription is supported by for example NCBI accession <a href="http://www.ncbi.nlm.nih.gov/nucleotide/372307839?report=genbank&amp;log$=nuclalign&amp;blast_rank=1&amp;RID=VCK517UJ015">gb|JR792458.1|</a>.</p> </div> <div class="echinopstelfairi(lesserhedgehogtenrec)" title="sec"> <div class="title" tagx="title" title="title"><i>Echinops telfairi</i> (lesser hedgehog tenrec)</div> <p>The depicted tenrec <i>Wdnm1-like</i> sequence maps within the genomic sequence of NCBI database accession <a href="http://www.ncbi.nlm.nih.gov/Traces/wgs/?val=AAIY02&amp;page=1&amp;display=contig&amp;search=AAIY02150441">gb|AAIY02150441.1|</a>, positions 1061-to-5393, forward orientation.</p> </div> <div class="loxodontaafricana(elephant)" title="sec"> <div class="title" tagx="title" title="title"><i>Loxodonta africana</i> (elephant)</div> <p>The depicted elephant <i>Wdnm1-like</i> sequence maps within Ensembl database loxAfr3, scaffold_31 positions 5685863-to-5689773, forward orientation. Transcription is supported by NCBI SRA database sequence reports, such as for example gnl|SRA|SRR1041765.37646273.1 of experiment set SRR1041765.</p> </div> <div class="orycteropusafer(aardvark)" title="sec"> <div class="title" tagx="title" title="title"><i>Orycteropus afer</i> (aardvark)</div> <p>The depicted aardvark <i>Wdnm1-like</i> sequence maps within Ensembl database OryAfe1, scaffold JH863914 positions 5948889-to-5951515, reverse orientation.</p> </div> <div class="dasypusnovemcinctus(nine-bandedarmadillo)" title="sec"> <div class="title" tagx="title" title="title"><i>Dasypus novemcinctus</i> (nine-banded armadillo)</div> <p>The depicted armadillo <i>Wdnm1-like</i> sequence maps within Ensembl database Dasnov3.0, scaffold JH562945 positions 1888971-to-1892017, forward orientation. Transcription is supported by NCBI SRA database sequence reports, such as for example gnl|SRA|SRR494776.6845635.2 of experiment set SRX146634.</p> </div> <div class="monodelphisdomestica(opossum)" title="sec"> <div class="title" tagx="title" title="title"><i>Monodelphis domestica</i> (opossum)</div> <p>The depicted opossum <i>Wdnm1-like</i> sequence maps within Ensembl database BROADO5, Chr.2 positions 498828348-to-498830609, reverse orientation. Transcription is supported by NCBI SRA database sequence reports, such as for example gnl|SRA|SRR908062.57922637.2, gnl|SRA|SRR873400.62402918.1, gnl|SRA|SRR943348.21681365, gnl|SRA|SRR943348.11424624, and gnl|SRA|SRR943348.9801988 of experiment sets SRX310006 and SRX290643 (because other <i>Wdnm1-like</i> transcript information appears lacking for marsupials, we here provide SRA database accessions that together cover the full ORF).</p> </div> </div> </div> <div class="back" title="back"> <div class="references">
References</div> <div tag="ref-list"> <ul> <li tag="ref"><a name="ref-1" /><span class="label" tagx="label" title="label">1</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wang</span><span class="given-names" tagx="given-names" title="given-names">P</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Xue</span><span class="given-names" tagx="given-names" title="given-names">Y</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Han</span><span class="given-names" tagx="given-names" title="given-names">Y</span></span><span class="etal" title="etal"><i>et al.</i></span></span>: <span class="mixed-article-title" title="mixed-article-title">The STAT3-binding long noncoding RNA lnc-DC controls human dendritic cell differentiation.</span><span class="source" tagx="source" title="source"><i>Science.</i></span><span class="year" tagx="year" title="year">2014</span>;<span class="volume" tagx="volume" title="volume">344</span>(<span class="issue" tagx="issue" title="issue">6181</span>):<span class="fpage" tagx="fpage" title="fpage">310</span>–<span class="lpage" tagx="lpage" title="lpage">3</span><span class="pub-id"><a href="https://dx.doi.org/10.1126/science.1251456">10.1126/science.1251456</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24744378">24744378</a></span></span></li> <li tag="ref"><a name="ref-2" /><span class="label" tagx="label" title="label">2</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wu</span><span class="given-names" tagx="given-names" title="given-names">Y</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Smas</span><span class="given-names" tagx="given-names" title="given-names">CM</span></span></span>: <span class="mixed-article-title" title="mixed-article-title">Wdnm1-like, a new adipokine with a role in MMP-2 activation.</span><span class="source" tagx="source" title="source"><i>Am J Physiol Endocrinol Metab.</i></span><span class="year" tagx="year" title="year">2008</span>;<span class="volume" tagx="volume" title="volume">295</span>(<span class="issue" tagx="issue" title="issue">1</span>):<span class="fpage" tagx="fpage" title="fpage">E205</span>–<span class="lpage" tagx="lpage" title="lpage">15</span><span class="pub-id"><a href="https://dx.doi.org/10.1152/ajpendo.90316.2008">10.1152/ajpendo.90316.2008</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18492766">18492766</a></span></span></li> <li tag="ref"><a name="ref-3" /><span class="label" tagx="label" title="label">3</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Jackson</span><span class="given-names" tagx="given-names" title="given-names">AL</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Burchard</span><span class="given-names" tagx="given-names" title="given-names">J</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Schelter</span><span class="given-names" tagx="given-names" title="given-names">J</span></span><span class="etal" title="etal"><i>et al.</i></span></span>: <span class="mixed-article-title" title="mixed-article-title">Widespread siRNA "off-target" transcript silencing mediated by seed region sequence complementarity.</span><span class="source" tagx="source" title="source"><i>RNA.</i></span><span class="year" tagx="year" title="year">2006</span>;<span class="volume" tagx="volume" title="volume">12</span>(<span class="issue" tagx="issue" title="issue">7</span>):<span class="fpage" tagx="fpage" title="fpage">1179</span>–<span class="lpage" tagx="lpage" title="lpage">87</span><span class="pub-id"><a href="https://dx.doi.org/10.1261/rna.25706">10.1261/rna.25706</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16682560">16682560</a></span></span></li> <li tag="ref"><a name="ref-4" /><span class="label" tagx="label" title="label">4</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kenworthy</span><span class="given-names" tagx="given-names" title="given-names">R</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lambert</span><span class="given-names" tagx="given-names" title="given-names">D</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Yang</span><span class="given-names" tagx="given-names" title="given-names">F</span></span><span class="etal" title="etal"><i>et al.</i></span></span>: <span class="mixed-article-title" title="mixed-article-title">Short-hairpin RNAs delivered by lentiviral vector transduction trigger RIG-I-mediated IFN activation.</span><span class="source" tagx="source" title="source"><i>Nucleic Acids Res.</i></span><span class="year" tagx="year" title="year">2009</span>;<span class="volume" tagx="volume" title="volume">37</span>(<span class="issue" tagx="issue" title="issue">19</span>):<span class="fpage" tagx="fpage" title="fpage">6587</span>–<span class="lpage" tagx="lpage" title="lpage">99</span><span class="pub-id"><a href="https://dx.doi.org/10.1093/nar/gkp714">10.1093/nar/gkp714</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19729514">19729514</a></span></span></li> <li tag="ref"><a name="ref-5" /><span class="label" tagx="label" title="label">5</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Adachi</span><span class="given-names" tagx="given-names" title="given-names">W</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ulanovsky</span><span class="given-names" tagx="given-names" title="given-names">H</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Li</span><span class="given-names" tagx="given-names" title="given-names">Y</span></span><span class="etal" title="etal"><i>et al.</i></span></span>: <span class="mixed-article-title" title="mixed-article-title">Serial analysis of gene expression (SAGE) in the rat limbal and central corneal epithelium.</span><span class="source" tagx="source" title="source"><i>Invest Ophthalmol Vis Sci.</i></span><span class="year" tagx="year" title="year">2006</span>;<span class="volume" tagx="volume" title="volume">47</span>(<span class="issue" tagx="issue" title="issue">9</span>):<span class="fpage" tagx="fpage" title="fpage">3801</span>–<span class="lpage" tagx="lpage" title="lpage">10</span><span class="pub-id"><a href="https://dx.doi.org/10.1167/iovs.06-0216">10.1167/iovs.06-0216</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16936091">16936091</a></span></span></li> <li tag="ref"><a name="ref-6" /><span class="label" tagx="label" title="label">6</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Derecka</span><span class="given-names" tagx="given-names" title="given-names">M</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gornicka</span><span class="given-names" tagx="given-names" title="given-names">A</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Koralov</span><span class="given-names" tagx="given-names" title="given-names">SB</span></span><span class="etal" title="etal"><i>et al.</i></span></span>: <span class="mixed-article-title" title="mixed-article-title">Tyk2 and Stat3 regulate brown adipose tissue differentiation and obesity.</span><span class="source" tagx="source" title="source"><i>Cell Metab.</i></span><span class="year" tagx="year" title="year">2012</span>;<span class="volume" tagx="volume" title="volume">16</span>(<span class="issue" tagx="issue" title="issue">6</span>):<span class="fpage" tagx="fpage" title="fpage">814</span>–<span class="lpage" tagx="lpage" title="lpage">24</span><span class="pub-id"><a href="https://dx.doi.org/10.1016/j.cmet.2012.11.005">10.1016/j.cmet.2012.11.005</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23217260">23217260</a></span></span></li> <li tag="ref"><a name="ref-7" /><span class="label" tagx="label" title="label">7</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hsueh</span><span class="given-names" tagx="given-names" title="given-names">YJ</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Chen</span><span class="given-names" tagx="given-names" title="given-names">HC</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Chu</span><span class="given-names" tagx="given-names" title="given-names">WK</span></span><span class="etal" title="etal"><i>et al.</i></span></span>: <span class="mixed-article-title" title="mixed-article-title">STAT3 regulates the proliferation and differentiation of rabbit limbal epithelial cells via a ΔNp63-dependent mechanism.</span><span class="source" tagx="source" title="source"><i>Invest Ophthalmol Vis Sci.</i></span><span class="year" tagx="year" title="year">2011</span>;<span class="volume" tagx="volume" title="volume">52</span>(<span class="issue" tagx="issue" title="issue">7</span>):<span class="fpage" tagx="fpage" title="fpage">4685</span>–<span class="lpage" tagx="lpage" title="lpage">93</span><span class="pub-id"><a href="https://dx.doi.org/10.1167/iovs.10-6103">10.1167/iovs.10-6103</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21447682">21447682</a></span></span></li> <li tag="ref"><a name="ref-8" /><span class="label" tagx="label" title="label">8</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Necsulea</span><span class="given-names" tagx="given-names" title="given-names">A</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Soumillon</span><span class="given-names" tagx="given-names" title="given-names">M</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Warnefors</span><span class="given-names" tagx="given-names" title="given-names">M</span></span><span class="etal" title="etal"><i>et al.</i></span></span>: <span class="mixed-article-title" title="mixed-article-title">The evolution of lncRNA repertoires and expression patterns in tetrapods.</span><span class="source" tagx="source" title="source"><i>Nature.</i></span><span class="year" tagx="year" title="year">2014</span>;<span class="volume" tagx="volume" title="volume">505</span>(<span class="issue" tagx="issue" title="issue">7485</span>):<span class="fpage" tagx="fpage" title="fpage">635</span>–<span class="lpage" tagx="lpage" title="lpage">40</span><span class="pub-id"><a href="https://dx.doi.org/10.1038/nature12943">10.1038/nature12943</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24463510">24463510</a></span></span></li> <li tag="ref"><a name="ref-9" /><span class="label" tagx="label" title="label">9</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hopp</span><span class="given-names" tagx="given-names" title="given-names">TP</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Woods</span><span class="given-names" tagx="given-names" title="given-names">KR</span></span></span>: <span class="mixed-article-title" title="mixed-article-title">Prediction of protein antigenic determinants from amino acid sequences.</span><span class="source" tagx="source" title="source"><i>Proc Natl Acad Sci U S A.</i></span><span class="year" tagx="year" title="year">1981</span>;<span class="volume" tagx="volume" title="volume">78</span>(<span class="issue" tagx="issue" title="issue">6</span>):<span class="fpage" tagx="fpage" title="fpage">3824</span>–<span class="lpage" tagx="lpage" title="lpage">28</span><span class="pub-id"><a href="https://dx.doi.org/10.1073/pnas.78.6.3824">10.1073/pnas.78.6.3824</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/6167991">6167991</a></span></span></li> <li tag="ref"><a name="ref-10" /><span class="label" tagx="label" title="label">10</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Green</span><span class="given-names" tagx="given-names" title="given-names">RE</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Krause</span><span class="given-names" tagx="given-names" title="given-names">J</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Briggs</span><span class="given-names" tagx="given-names" title="given-names">AW</span></span><span class="etal" title="etal"><i>et al.</i></span></span>: <span class="mixed-article-title" title="mixed-article-title">A Draft Sequence of the Neandertal genome.</span><span class="source" tagx="source" title="source"><i>Science.</i></span><span class="year" tagx="year" title="year">2010</span>;<span class="volume" tagx="volume" title="volume">328</span>(<span class="issue" tagx="issue" title="issue">5979</span>):<span class="fpage" tagx="fpage" title="fpage">710</span>–<span class="lpage" tagx="lpage" title="lpage">22</span><span class="pub-id"><a href="https://dx.doi.org/10.1126/science.1188021">10.1126/science.1188021</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20448178">20448178</a></span></span></li> <li tag="ref"><a name="ref-11" /><span class="label" tagx="label" title="label">11</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Meyer</span><span class="given-names" tagx="given-names" title="given-names">M</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kircher</span><span class="given-names" tagx="given-names" title="given-names">M</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gansauge</span><span class="given-names" tagx="given-names" title="given-names">MT</span></span><span class="etal" title="etal"><i>et al.</i></span></span>: <span class="mixed-article-title" title="mixed-article-title">A high-coverage genome sequence from an archaic Denisovan individual.</span><span class="source" tagx="source" title="source"><i>Science.</i></span><span class="year" tagx="year" title="year">2012</span>;<span class="volume" tagx="volume" title="volume">338</span>(<span class="issue" tagx="issue" title="issue">6104</span>):<span class="fpage" tagx="fpage" title="fpage">222</span>–<span class="lpage" tagx="lpage" title="lpage">26</span><span class="pub-id"><a href="https://dx.doi.org/10.1126/science.1224344">10.1126/science.1224344</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22936568">22936568</a></span></span></li> </ul> </div> </div>  </body></html>