<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<link rel="dns-prefetch" href="//d33xdlntwy0kbs.cloudfront.net" />
<link rel="shortcut icon" href="https://www.biorxiv.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<meta name="article_thumbnail" content="https://www.biorxiv.org/content/biorxiv/early/2016/02/17/039818/embed/graphic-3.gif" />
<meta name="type" content="article" />
<meta name="category" content="article" />
<meta name="HW.identifier" content="/biorxiv/early/2016/02/17/039818.atom" />
<meta name="HW.pisa" content="biorxiv;039818v1" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="Assessing the relationship between height growth and molecular genetic variation in Douglas-fir (Pseudotsuga menziesii) provenances" />
<meta name="DC.Identifier" content="10.1101/039818" />
<meta name="DC.Date" content="2016-02-17" />
<meta name="DC.Publisher" content="Cold Spring Harbor Laboratory" />
<meta name="DC.Rights" content="© 2016, Posted by Cold Spring Harbor Laboratory. The copyright holder for this pre-print is the author. All rights reserved. The material may not be redistributed, re-used or adapted without the author's permission." />
<meta name="DC.AccessRights" content="restricted" />
<meta name="DC.Description" content="Douglas-fir ( Pseudotsuga menziesii ) is a conifer tree native to western North America. In central Europe, it shows superior growth performance and is considered a suitable substitute for tree species impaired in vitality due to climate change. Maintenance and improvement of growth performance in a changing environment is a main challenge for forest tree breeders. In this context, genetic variation as a factor underlying phenotypic variation, but also as the basis for future adaptation, is of particular interest. The aims of this study were to analyse (i) genetic diversity of selected Douglas-fir provenances, (ii) variation in height growth among provenances, and (iii) to assess the link between genetic and phenotypic variation height growth. Genotyping was done on microsatellite loci. Effects of ‘provenance’, ‘genotype’, and ‘site’ on height growth were assessed by fitting mixed linear models. The most significant genetic differentiation was observed between provenances of the coastal variety, versus a provenance of the interior variety originating from British Columbia. Although genetic differentiation among provenances of the coastal variety was lower, genetic structures within this variety were identified. Moreover, genetic diversity showed a latitudinal gradient with the southernmost provenances being more diverse, probably reflecting the species' evolutionary history. The modelling approach revealed that height growth differed significantly by provenance, site, and the interaction between site and provenance, demonstrating that height growth is under strong genetic control. Additionally, this analysis showed that genetic variation captured by the genotyped microsatellite loci was significantly related to variation in height growth, providing statistical evidence for a genetic component in the observed phenotypic variation." />
<meta name="DC.Contributor" content="Charalambos Neophytou" />
<meta name="DC.Contributor" content="Anna-Maria Weisser" />
<meta name="DC.Contributor" content="Daniel Landwehr" />
<meta name="DC.Contributor" content="Muhidin Šeho" />
<meta name="DC.Contributor" content="Ulrich Kohnle" />
<meta name="DC.Contributor" content="Ingo Ensminger" />
<meta name="DC.Contributor" content="Henning Wildhagen" />
<meta name="article:published_time" content="2016-02-17" />
<meta name="article:section" content="New Results" />
<meta name="citation_title" content="Assessing the relationship between height growth and molecular genetic variation in Douglas-fir (Pseudotsuga menziesii) provenances" />
<meta name="citation_abstract" lang="en" content="&lt;h3&gt;Abstract&lt;/h3&gt;
&lt;p&gt;Douglas-fir (&lt;i&gt;Pseudotsuga menziesii&lt;/i&gt;) is a conifer tree native to western North America. In central Europe, it shows superior growth performance and is considered a suitable substitute for tree species impaired in vitality due to climate change. Maintenance and improvement of growth performance in a changing environment is a main challenge for forest tree breeders. In this context, genetic variation as a factor underlying phenotypic variation, but also as the basis for future adaptation, is of particular interest. The aims of this study were to analyse (i) genetic diversity of selected Douglas-fir provenances, (ii) variation in height growth among provenances, and (iii) to assess the link between genetic and phenotypic variation height growth. Genotyping was done on microsatellite loci. Effects of ‘provenance’, ‘genotype’, and ‘site’ on height growth were assessed by fitting mixed linear models. The most significant genetic differentiation was observed between provenances of the coastal variety, versus a provenance of the interior variety originating from British Columbia. Although genetic differentiation among provenances of the coastal variety was lower, genetic structures within this variety were identified. Moreover, genetic diversity showed a latitudinal gradient with the southernmost provenances being more diverse, probably reflecting the species9 evolutionary history. The modelling approach revealed that height growth differed significantly by provenance, site, and the interaction between site and provenance, demonstrating that height growth is under strong genetic control. Additionally, this analysis showed that genetic variation captured by the genotyped microsatellite loci was significantly related to variation in height growth, providing statistical evidence for a genetic component in the observed phenotypic variation.&lt;/p&gt;" />
<meta name="citation_journal_title" content="bioRxiv" />
<meta name="citation_publisher" content="Cold Spring Harbor Laboratory" />
<meta name="citation_publication_date" content="2016/01/01" />
<meta name="citation_mjid" content="biorxiv;039818v1" />
<meta name="citation_id" content="039818v1" />
<meta name="citation_public_url" content="https://www.biorxiv.org/content/10.1101/039818v1" />
<meta name="citation_abstract_html_url" content="https://www.biorxiv.org/content/10.1101/039818v1.abstract" />
<meta name="citation_full_html_url" content="https://www.biorxiv.org/content/10.1101/039818v1.full" />
<meta name="citation_pdf_url" content="https://www.biorxiv.org/content/biorxiv/early/2016/02/17/039818.full.pdf" />
<meta name="citation_doi" content="10.1101/039818" />
<meta name="citation_num_pages" content="25" />
<meta name="citation_article_type" content="Article" />
<meta name="citation_section" content="New Results" />
<meta name="citation_firstpage" content="039818" />
<meta name="citation_author" content="Charalambos Neophytou" />
<meta name="citation_author_institution" content="Forest Research Institute Baden-Württemberg (FVA)" />
<meta name="citation_author" content="Anna-Maria Weisser" />
<meta name="citation_author_institution" content="Forest Research Institute Baden-Württemberg (FVA)" />
<meta name="citation_author" content="Daniel Landwehr" />
<meta name="citation_author_institution" content="Forest Research Institute Baden-Württemberg (FVA)" />
<meta name="citation_author" content="Muhidin Šeho" />
<meta name="citation_author_institution" content="Forest Research Institute Baden-Württemberg (FVA)" />
<meta name="citation_author" content="Ulrich Kohnle" />
<meta name="citation_author_institution" content="Forest Research Institute Baden-Württemberg (FVA)" />
<meta name="citation_author" content="Ingo Ensminger" />
<meta name="citation_author_institution" content="Forest Research Institute Baden-Württemberg (FVA)" />
<meta name="citation_author_institution" content="Department of Biology, Graduate Programs in Cell &amp; Systems Biology and Ecology and Evolutionary Biology, University of Toronto" />
<meta name="citation_author" content="Henning Wildhagen" />
<meta name="citation_author_institution" content="Forest Research Institute Baden-Württemberg (FVA)" />
<meta name="citation_author_institution" content="Büsgen-Institut, Department of Forest Botany and Tree Physiology, Georg-August-Universität Göttingen" />
<meta name="citation_author_email" content="hwildha@gwdg.de" />
<meta name="citation_reference" content="Aas G (2008) Die Douglasie (Pseudotsuga menziesii) in Nordamerika: Verbreitung, Variabilität und Ökologie. Die Douglasie-Perspektiven im Klimawandel. LWF Wissen 59:7–11" />
<meta name="citation_reference" content="Aitken SN, Yeaman S, Holliday J A, Wang T, Curtis-McLane S (2008) Adaptation, migration or extirpation: climate change outcomes for tree populations. Evol Appl 1:95–111" />
<meta name="citation_reference" content="Alberto FJ, Aitken SN, Alía R, González-Martínez SC, Hänninen H, Kremer A, Lefèvre F, Lenormand T, Yeaman S, Whetten R, Savolainen O (2013) Potential for evolutionary responses to climate change–evidence from tree populations. Glob Chang Biol 19:1645–1661" />
<meta name="citation_reference" content="Albrecht A, Kohnle U, Hanewinkel, M, Bauhus J (2013) Storm damage in Douglas-fir unexpectedly high compared to Norway spruce. Ann For Sci 70:195–207" />
<meta name="citation_reference" content="Albrecht A, Hanewinkel M, Bauhus J, Kohnle U (2012) How does silviculture affect storm damage in forests of south-western Germany? Results from empirical modeling based on long-term observations. Eur J For Res 131:229–247" />
<meta name="citation_reference" content="Andolfatto P (2001) Adaptive hitchhiking effects on genome variability. Curr Opin Genet Dev 11:635–641" />
<meta name="citation_reference" content="Belaj A, Muñoz-Diez C, Baldoni L, Porceddu A, Barranco D, Satovic Z (2007) Genetic diversity and population structure of wild olives from the north-western Mediterranean assessed by SSR Markers. Ann Bot 100:449–458" />
<meta name="citation_reference" content="Belkhir K, Borsa P, Chikhi L, Raufaste N, Bonhomme F (2004) GENETIX 4.05.2, Windows™ Software for Population Genetics. Laboratoire génome, populations, interactions, CNRS UMR, 5000, University of Montpellier" />
<meta name="citation_reference" content="Bryja J, Smith C, Konevcnỳ A, Reichard M (2010) Range-wide population genetic structure of the European bitterling (Rhodeus amarus) based on microsatellite and mitochondrial DNA analysis. Mol Ecol 19:4708–4722" />
<meta name="citation_reference" content="Chakraborty D, Wang T, Andre K, Konnert M, Lexer MJ, Matulla C, Schueler S (2015) Selecting Populations for Non-Analogous Climate Conditions Using Universal Response Functions: The Case of Douglas-Fir in Central Europe. PloS One, 10(8):e0136357" />
<meta name="citation_reference" content="Darychuk N, Hawkins BJ, Stoehr M (2012) Trade-offs between growth and cold and drought hardiness in submaritime Douglas-fir. Can J For Res 42:1530–1541" />
<meta name="citation_reference" content="Duplantier JM, Granjon L, Mathieu E, Bonhomme F (1990) Structures génétiques comparées de trois espèces de rongeurs africains du genre Mastomys au Sénégal. Genetica 81:179–192" />
<meta name="citation_reference" content="Earl DA, vonHoldt BM (2012) STRUCTURE HARVESTER: a website and program for visualizing STRUCTURE output and implementing the Evanno method. Conserv Genet Resour 4:359–361" />
<meta name="citation_reference" content="Eckert AJ, Bower AD, Wegrzyn JL, Pande B, Jermstad KD, Krutovsky KV, St. Clair JB, Neale DB (2009) Association genetics of coastal Douglas fir (Pseudotsuga menziesii var. menziesii, Pinaceae). I. Cold-hardiness related traits. Genetics 182:1289–1302" />
<meta name="citation_reference" content="Ehring A, Klädtke J, Yue C (1999) Ein interaktives Programm zur Erstellung von Bestandeshöhenkurven. Centralblatt für das gesamte Forstwesen 116:47–52" />
<meta name="citation_reference" content="Eilmann B, Rigling A (2012) Tree-growth analyses to estimate tree species’ drought tolerance. Tree Phys 32:178–187" />
<meta name="citation_reference" content="Eilmann B, de Vries SMG, den Ouden J, Mohren GMJ, Sauren P, Sass-Klaasen U (2013) Origin matters! Difference in drought tolerance and productivity of coastal Douglas-fir (Pseudotsuga menziesii (Mirb.)) provenances. Forest Ecol Manag 302:133–143" />
<meta name="citation_reference" content="Evanno G, Regnaut S, Goudet J (2005) Detecting the number of clusters of individuals using the software STRUCTURE: a simulation study. Mol Ecol 14:2611–2620" />
<meta name="citation_reference" content="González-Martínez SC, Wheeler NC, Ersoz E, Nelson CD, Neale DB (2007) Association genetics in Pinus taeda L. I. Wood property traits. Genetics 175:399–409" />
<meta name="citation_reference" content="Goudet J (2001) FSTAT (version 2.9.3.2): FSTAT, a program to estimate and test gene diversities and fixation indices (version 2.9.3.2). Available at http://www2.unil.ch/popgen/softwares/fstat.html." />
<meta name="citation_reference" content="Goudet J, Raymond M, de Meeüs T, Rousset F (1996) Testing Differentiation in Diploid Populations. Genetics 144:1933–1940" />
<meta name="citation_reference" content="Gould PJ, Harrington CA, St. Clair JB (2012) Growth phenology of coast Douglas-fir seed sources planted in diverse environments. Tree Phys 32:1482–1496" />
<meta name="citation_reference" content="Grivet D, Sork VL, Westfall RD, Davis FW (2008) Conserving the evolutionary potential of California valley oak (Quercus lobata Née): A multivariate genetic approach to conservation planning. Mol Ecol 17:139–156" />
<meta name="citation_reference" content="Gugger PF, Sugita S (2010) Glacial populations and postglacial migration of Douglas-fir based on fossil pollen and macrofossil evidence. Quaternary Sci Rev 29:2052–2070" />
<meta name="citation_reference" content="Gugger PF, Sugita S, Cavender-Bares J (2010) Phylogeography of Douglas-fir based on mitochondrial and chloroplast DNA sequences: testing hypotheses from the fossil record. Mol Ecol 19:1877–1897" />
<meta name="citation_reference" content="Guichoux E, Lagache L, Wagner S, Chaumiel P, Léger P, Lepais O, Lepoittevin C, Malausa T, Revardel E, Salin F, Petit J (2011) Current trends in microsatellite genotyping. Mol Ecol Res 11:591–611" />
<meta name="citation_reference" content="citation_title=Silvics of North America: 1. Conifers, USDA Forest Service Agriculture Handbook 654;citation_pages=527-540;citation_year=1990" />
<meta name="citation_reference" content="Hermann RK, Lavender DP (1999) Douglas-fir planted forests. New Forests 17:53–70" />
<meta name="citation_reference" content="Hubisz MJ, Falush D, Stephens M, Pritchard JK (2009) Inferring weak population structure with the assistance of sample group information. Mol Ecol Res 9:1322–1332" />
<meta name="citation_reference" content="Jakobsson M, Rosenberg NA (2007) CLUMPP: A cluster matching and permutation program for dealing with label switching and multimodality in analysis of population structure. Bioinformatics 23:1801–1806" />
<meta name="citation_reference" content="Jansen K, Sohrt J, Kohnle U, Ensminger I, Gessler A (2013) Tree ring isotopic composition, radial increment and height growth reveal provenance-specific reactions of Douglas-fir towards environmental parameters. Trees 27:37–52" />
<meta name="citation_reference" content="Kenk G, Thren M (1984) Ergebnisse verschiedener Douglasienprovenienzversuche in Baden-Württemberg. Teil I: Der Internationale Douglasien-Provenienzversuch 1958. Allg Forst Jagdztg 155:165–184" />
<meta name="citation_reference" content="Kohnle U, Hein S, Sorensen FC, Weiskittel AR (2012) Effects of seed source origin on bark thickness of Douglas-fir (Pseudotsuga menziesii) growing in south-western Germany. Can J For Res 42:382–399" />
<meta name="citation_reference" content="Konnert M, Ruetz W, Schirmer R (2008) Fragen zum forstlichen Vermehrungsgut bei Douglasie. LWF-Wissen 59:22–26" />
<meta name="citation_reference" content="Krakowski J, Stoehr MU (2009) Coastal Douglas-fir provenance variation: patterns and predictions for British Columbia seed transfer. Ann For Sci 66:811." />
<meta name="citation_reference" content="Kremer A, Potts BM, Delzon S (2014) Genetic divergence in forest trees: understanding the consequences of climate change. Funct Ecol 28:22–36" />
<meta name="citation_reference" content="Krutovsky KV, Neale DB (2005) Nucleotide Diversity and Linkage Disequilibrium in Cold-Hardiness- and Wood Quality-Related Candidate Genes in Douglas Fir. Genetics 171:2029–2041" />
<meta name="citation_reference" content="Krutovsky KV, St. Clair JB, Saich R, Hipkins VD, Neale DB (2009) Estimation of population structure in coastal Douglas-fir [Pseudotsuga menziesii (Mirb.) Franco var. menziesii] using allozyme and microsatellite markers. Tree Genet Genomes 5:641–658" />
<meta name="citation_reference" content="Leites LP, Robinson AP, Rehfeldt GE, Marshall JD, Crookston NL (2012) Height-growth response to climatic changes differs among populations of Douglas-fir: a novel analysis of historic data. Ecol Appl 22:154–165" />
<meta name="citation_reference" content="Li P, Adams WT (1989) Range-wide patterns of allozyme variation in Douglas-fir (Pseudotsuga menziesii). Can J For Res 19:149–161" />
<meta name="citation_reference" content="Little EL (1971) Atlas of United States trees, volume 1, conifers and important hardwoods: U.S. Department of Agriculture Miscellaneous Publication 1146" />
<meta name="citation_reference" content="Martin A, Herrera MA, Martin LM (2012) In situ conservation and landscape genetics in forest species. J Nat Resour Develop 2:1–5" />
<meta name="citation_reference" content="Matschiner M, Salzburger W (2009) TANDEM: integrating automated allele binning into genetics and genomics workflows. Bioinformatics 25:1982–1983" />
<meta name="citation_reference" content="Messaoud Y, Chen HYH (2011) The influence of recent climate change on tree height growth differs with species and spatial environment. PLoS One 6(2):e14691. doi:101371/journal.pone.0014691" />
<meta name="citation_reference" content="Mosca E, González-Martínez SC, Neale DB (2014) Environmental versus geographical determinants of genetic structure in two subalpine conifers. New Phytol 201:180–192" />
<meta name="citation_reference" content="Müller T, Ensminger I, Schmid KJ (2012) A catalogue of putative unique transcripts from Douglas-fir (Pseudotsuga menziesii) based on 454 transcriptome sequencing of genetically diverse, drought stressed seedlings. BMC Genomics 13:673. doi: 10.1186/1471-2164-13-673" />
<meta name="citation_reference" content="Müller T, Freund F, Wildhagen H, Schmid KJ (2015) Targeted re-sequencing of five Douglas-fir provenances reveals population structure and putative target genes of positive selection. Tree Genet Genomes 11:816" />
<meta name="citation_reference" content="Neale DB, Kremer A (2011) Forest tree genomics: growing resources and applications. Nat Rev Gen 12:111–122" />
<meta name="citation_reference" content="Nei M (1972) Genetic distance between populations. Am Nat 106:283–292" />
<meta name="citation_reference" content="Nosil P, Egan SP, Funk DJ (2008) Heterogeneous genomic differentiation between walking-stick ecotypes: “Isolation by adaptation” and multiple roles for divergent selection. Evolution 62:316–336" />
<meta name="citation_reference" content="Nosil P, Funk DJ, Ortiz-Barrientos D (2009) Divergent selection and heterogeneous genomic divergence. Mol Ecol 18:375–402" />
<meta name="citation_reference" content="O’Brien IEW, Smith DR, Gardner RC, Murray BG (1996) Flow cytometric determination of genome size in Pinus. Plant Sci 115:91–99" />
<meta name="citation_reference" content="Oksanen J, Guillaume Blanchet F, Kindt R Legendre, P Minchin PR, O’Hara RB, Simpson GL, Solymos P, Henry H, Stevens H, Wagner H (2013) Community Ecology Package. R package version 2.0-10. http://CRAN.R-project.org/package=vegan." />
<meta name="citation_reference" content="Rehfeldt GE (1989) Ecological adaptations in Douglas-fir (Pseudotsuga menziesii var. glauca): a synthesis. Forest Ecol Manag 28:203–215" />
<meta name="citation_reference" content="Van Oosterhout C, Hutchinson WF, Wills DPM, Shipley P (2004) Micro-Checker: Software for identifying and correcting genotyping errors in microsatellite data. Mol Ecol Notes 4:535–538" />
<meta name="citation_reference" content="Pinheiro J, Bates D, DebRoy S, Sarkar D, and the R Development Core Team (2012). nlme: Linear and Nonlinear Mixed Effects Models. R package version 3.1-103" />
<meta name="citation_reference" content="Pritchard JK, Stephens M, Donnelly P (2000) Inference of population structure using multilocus genotype data. Genetics 155:945–959" />
<meta name="citation_reference" content="Pritchard JK, Pickrell JK, Coop G (2010) The Genetics of Human Adaptation: Hard Sweeps, Soft Sweeps and Polygenic Adaptation. Curr Biol 20:R208–R215. doi: 10.1016/j.cub.2009.11.055" />
<meta name="citation_reference" content="Puettmann K, Coates KD, Messier C (2009) A critique of silviculture: Managing for complexity. Island Press, Washington D. C." />
<meta name="citation_reference" content="R Development Core Team (2011) R: A language and environment for statistical computing. R Foundation for Statistical Computing, Vienna, Austria, http://www.R-project.org" />
<meta name="citation_reference" content="Ramírez-Valiente JA, Lorenzo Z, Soto A, Valladares F, Gil L, Aranda I (2010) Natural selection on cork oak: allele frequency reveals divergent selection in cork oak populations along a temperature cline. Evol Ecol 24:1031–1044" />
<meta name="citation_reference" content="Rosenberg NA (2004) Distruct: a program for the graphical display of population structure. Mol Ecol Notes 4:137–138" />
<meta name="citation_reference" content="Scotti-Saintagne C, Mariette S, Porth I, Goicoechea PG, Barreneche T, Bodénès C, Burg K, Kremer A (2004) Genome scanning for interspecific differentiation between two closely related oak species [Quercus robur L. and Q. petraea (Matt.) Liebl.]. Genetics 168:1615–1626" />
<meta name="citation_reference" content="Schmidt M, Hanewinkel M, Kändler G, Kublin E, Kohnle U (2010) An inventory-based approach for modeling single-tree storm damage-experiences with the winter storm of 1999 in south-western Germany. Can J For Res 40:1636–1652" />
<meta name="citation_reference" content="Schütt P, Schuck HJ, Stimm B (2002) Lexikon der Baum-und Straucharten: Das Standardwerk der Forstbotanik. Nikol Verlagsgesellschaft mbH &amp; Co. KG, Hamburg" />
<meta name="citation_reference" content="Šeho M, Kohnle U (2014) Der Internationale Douglasien-Provenienzversuch 1958: Unterschiede in der Ausprägung von Ast-und Stammmerkmalen auf den südwestdeutschen Versuchsflächen. Allg Forst Jagdztg 185:27–42" />
<meta name="citation_reference" content="Selkoe K A, Toonen RJ (2006) Microsatellites for ecologists: a practical guide to using and evaluating microsatellite markers. Ecol Lett 9:615–629" />
<meta name="citation_reference" content="She JX, Autem M, Kotulas G, Pasteur N, Bonhomme F (1987) Multivariate analysis of genetic exchanges between Solea aegyptiaca and Solea senegalensis (Teleosts, Soleidae). Biol J Linn Soc 32:357–371" />
<meta name="citation_reference" content="Slavov GT, Howe GT, Yakovlev I, Edwards KJ, Krutovsky KV, Tuskan GA, Carlson JE, Strauss SH, Adams WT (2004). Highly variable SSR markers in Douglas-fir: Mendelian inheritance and map locations. Theor Appl Genet 108:873–880" />
<meta name="citation_reference" content="Sork VL, Davis FW, Westfall R, Flint A, Ikegami M, Wang H, Grivet D (2010) Gene movement and genetic association with regional climate gradients in California valley oak (Quercus lobata Née) in the face of climate change. Mol Ecol 19:3806–3823" />
<meta name="citation_reference" content="Strehlke B (1959) Die Ernte von Douglasiensamen in USA und Kanada. Folgerungen für die deutsche Forstwirtschaft. Der Forst-und Holzwirt 14:295–300" />
<meta name="citation_reference" content="Takezaki N, Nei M, Tamura K (2010) POPTREE2: Software for Constructing Population Trees from Allele Frequency Data and Computing Other Population Statistics with Windows Interface. Mol Biol Evol 27:747–752" />
<meta name="citation_reference" content="Temunović M, Franjić J, Satovic Z, Grgurev M, Frascaria-Lacoste N, Fernández-Manjarrés JF (2012) Environmental Heterogeneity Explains the Genetic Structure of Continental and Mediterranean Populations of Fraxinus angustifolia Vahl. PLoS One 7:e42764. doi: 10.1371/journal.pone.0042764" />
<meta name="citation_reference" content="Tsukada M (1982) Pseudotsuga menziesii (Mirb.) Franco: Its pollen dispersal and late Quaternary history in the Pacific Northwest [USA]. Japanese J Ecol 32:159–187" />
<meta name="citation_reference" content="Tsumura Y, Uchiyama K, Moriguchi Y, Ueno S, Ihara-Ujino T (2012) Genome scanning for detecting adaptive genes along environmental gradients in the Japanese conifer, Cryptomeria japonica. Heredity 109:349–360" />
<meta name="citation_reference" content="Vasemägi A, Gross R, Paaver T, Kangur M, Nilsson J, Eriksson LO (2001) Identification of the origin of an Atlantic salmon (Salmo salar L.) population in a recently recolonized river in the Baltic Sea. Mol Ecol 10:2877–2882" />
<meta name="citation_reference" content="Weir BS, Cockerham CC (1984) Estimating F-Statistics for the Analysis of Population Structure. Evolution 38:1358–1370" />
<meta name="citation_reference" content="Zuur A, Ieno EN, Walker N, Saveliev AA, Smith GM (2007) Mixed Effects Models and Extensions in Ecology with R. Springer, New York" />
<meta name="twitter:title" content="Assessing the relationship between height growth and molecular genetic variation in Douglas-fir (Pseudotsuga menziesii) provenances" />
<meta name="twitter:site" content="@biorxivpreprint" />
<meta name="twitter:card" content="summary" />
<meta name="twitter:image" content="https://www.biorxiv.org/sites/default/files/images/biorxiv_logo_homepage7-5-small.png" />
<meta name="twitter:description" content="Douglas-fir ( Pseudotsuga menziesii ) is a conifer tree native to western North America. In central Europe, it shows superior growth performance and is considered a suitable substitute for tree species impaired in vitality due to climate change. Maintenance and improvement of growth performance in a changing environment is a main challenge for forest tree breeders. In this context, genetic variation as a factor underlying phenotypic variation, but also as the basis for future adaptation, is of particular interest. The aims of this study were to analyse (i) genetic diversity of selected Douglas-fir provenances, (ii) variation in height growth among provenances, and (iii) to assess the link between genetic and phenotypic variation height growth. Genotyping was done on microsatellite loci. Effects of ‘provenance’, ‘genotype’, and ‘site’ on height growth were assessed by fitting mixed linear models. The most significant genetic differentiation was observed between provenances of the coastal variety, versus a provenance of the interior variety originating from British Columbia. Although genetic differentiation among provenances of the coastal variety was lower, genetic structures within this variety were identified. Moreover, genetic diversity showed a latitudinal gradient with the southernmost provenances being more diverse, probably reflecting the species' evolutionary history. The modelling approach revealed that height growth differed significantly by provenance, site, and the interaction between site and provenance, demonstrating that height growth is under strong genetic control. Additionally, this analysis showed that genetic variation captured by the genotyped microsatellite loci was significantly related to variation in height growth, providing statistical evidence for a genetic component in the observed phenotypic variation." />
<meta name="og-title" property="og:title" content="Assessing the relationship between height growth and molecular genetic variation in Douglas-fir (Pseudotsuga menziesii) provenances" />
<meta name="og-url" property="og:url" content="https://www.biorxiv.org/content/10.1101/039818v1" />
<meta name="og-site-name" property="og:site_name" content="bioRxiv" />
<meta name="og-description" property="og:description" content="Douglas-fir ( Pseudotsuga menziesii ) is a conifer tree native to western North America. In central Europe, it shows superior growth performance and is considered a suitable substitute for tree species impaired in vitality due to climate change. Maintenance and improvement of growth performance in a changing environment is a main challenge for forest tree breeders. In this context, genetic variation as a factor underlying phenotypic variation, but also as the basis for future adaptation, is of particular interest. The aims of this study were to analyse (i) genetic diversity of selected Douglas-fir provenances, (ii) variation in height growth among provenances, and (iii) to assess the link between genetic and phenotypic variation height growth. Genotyping was done on microsatellite loci. Effects of ‘provenance’, ‘genotype’, and ‘site’ on height growth were assessed by fitting mixed linear models. The most significant genetic differentiation was observed between provenances of the coastal variety, versus a provenance of the interior variety originating from British Columbia. Although genetic differentiation among provenances of the coastal variety was lower, genetic structures within this variety were identified. Moreover, genetic diversity showed a latitudinal gradient with the southernmost provenances being more diverse, probably reflecting the species' evolutionary history. The modelling approach revealed that height growth differed significantly by provenance, site, and the interaction between site and provenance, demonstrating that height growth is under strong genetic control. Additionally, this analysis showed that genetic variation captured by the genotyped microsatellite loci was significantly related to variation in height growth, providing statistical evidence for a genetic component in the observed phenotypic variation." />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.biorxiv.org/sites/default/files/images/biorxiv_logo_homepage7-5-small.png" />
<meta name="citation_date" content="2016-02-17" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/10.1101/039818v1.full.pdf" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/10.1101/039818v1.full.txt" />
<link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/10.1101/039818v1.ppt" />
<meta name="description" content="bioRxiv - the preprint server for biology, operated by Cold Spring Harbor Laboratory, a research and educational institution" />
<meta name="generator" content="Drupal 7 (http://drupal.org)" />
<link rel="canonical" href="https://www.biorxiv.org/content/10.1101/039818v1" />
<link rel="shortlink" href="https://www.biorxiv.org/node/10804" />
    <title>Assessing the relationship between height growth and molecular genetic variation in Douglas-fir (Pseudotsuga menziesii) provenances | bioRxiv</title>  
    <link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__jMRAK66KMC1e4TQlwUNn3KiWVDC5AjueUAYEm1xBY_U__KSUjdT4jdcJ4qJz6fKY1K9WkYh1a5EcaWZxt_-zbTis__QJacVmveFyLIBSPkp21xL0ZvuDW9DMBrD2HB1P4Ry5I.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q5p1rr" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__icrP-Y0qw2NP73rJxxkUtrJta0z3VLAABtNjfSdEghA__rlN3rx0wOoswGNHZMeZM96bNn-3cLWHXTztGDoThNlw__QJacVmveFyLIBSPkp21xL0ZvuDW9DMBrD2HB1P4Ry5I.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__e8NuQlin4bcnMtI2zOEVRblohOk3xZwfnFAFdibS6pw__XVRsOpcO2N5B6_yYKvNFz_CihNSGO6Yy3SOy-5iyzfE__QJacVmveFyLIBSPkp21xL0ZvuDW9DMBrD2HB1P4Ry5I.css" media="all" />
<style type="text/css" media="all">
/* <![CDATA[ */
#sliding-popup.sliding-popup-bottom,#sliding-popup.sliding-popup-bottom .eu-cookie-withdraw-banner,.eu-cookie-withdraw-tab{background:gray}#sliding-popup.sliding-popup-bottom.eu-cookie-withdraw-wrapper{background:transparent}#sliding-popup .popup-content #popup-text h1,#sliding-popup .popup-content #popup-text h2,#sliding-popup .popup-content #popup-text h3,#sliding-popup .popup-content #popup-text p,.eu-cookie-compliance-secondary-button,.eu-cookie-withdraw-tab{color:#fff !important}.eu-cookie-withdraw-tab{border-color:#fff}.eu-cookie-compliance-more-button{color:#fff !important}
/* ]]> */
</style>

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__QJacVmveFyLIBSPkp21xL0ZvuDW9DMBrD2HB1P4Ry5I.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__B6tGdsQSt1vT8d8j-19-oS4FKJe83RrSNBkjPs9jfos__6VkE5WcQmmQpSVTQx-xS2-zn_nlwFlDm-EN1HLJNjrA__QJacVmveFyLIBSPkp21xL0ZvuDW9DMBrD2HB1P4Ry5I.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__I6pBwZrmO51afTZVOjN83FdGH1QRb9t15lSLso0JVV4__F3sVKlGyHKCaAPI_WdiHD7l_-05B6b9bLdUwzi1me90__QJacVmveFyLIBSPkp21xL0ZvuDW9DMBrD2HB1P4Ry5I.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__QJacVmveFyLIBSPkp21xL0ZvuDW9DMBrD2HB1P4Ry5I.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__QJacVmveFyLIBSPkp21xL0ZvuDW9DMBrD2HB1P4Ry5I.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__CRdjIL0HKDL2T9slYvkTSTLypBBPjyoDhwBezBuDy7c__cHebPmPNdg2jyPmpyQpZWssJQ4cPrqfaabp5Or9KlkU__QJacVmveFyLIBSPkp21xL0ZvuDW9DMBrD2HB1P4Ry5I.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://d33xdlntwy0kbs.cloudfront.net/cshl_custom.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/all/libraries/modernizr/modernizr.min.js?q5p1rr"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__yOhrJUBmMpN-jsuQlJYmS8ovkQU6I6KInr295HrSTb0__c--zHhbuOWmkdWkJYXiNWDmkpDRPesmbBjGlQcyqBOY__QJacVmveFyLIBSPkp21xL0ZvuDW9DMBrD2HB1P4Ry5I.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var _prum=[['id', '52e95df3abe53d2e33000000'], ['mark', 'firstbyte',
      (new Date()).getTime()]]; (function() {
      var s=document.getElementsByTagName('script')[0],
      p=document.createElement('script');
      p.async='async'; p.src='//rum-static.pingdom.net/prum.min.js';s.parentNode.insertBefore(p,s);})();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js___KyvuEvfmewWxdOLBXTYsSUo8Pfm_DijV6fWtGvSL_k__HSKkTgIr7nDcdvQYewipf6v4ak6ZLUOQkwPj44wKouA__QJacVmveFyLIBSPkp21xL0ZvuDW9DMBrD2HB1P4Ry5I.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__3uCzzYE5jr6ly9nvsM8PRBRkhh4p0NWCMDja4EKBy4M__dmGGVYcB8LhnbR2iAmVW0ZyXI6QuFVYFRORa5XxglXs__QJacVmveFyLIBSPkp21xL0ZvuDW9DMBrD2HB1P4Ry5I.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__0zdUME0GLauYYyEch2tGPydIw-IG7Uizip0Qb4wUsig__3RgszrCgk4bOnTzo4vaQ-hrWzaG6oD19l-R9CQ1YQ-o__QJacVmveFyLIBSPkp21xL0ZvuDW9DMBrD2HB1P4Ry5I.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__W6wAjopaALytcBam7oT3WhEzllkWaB9R-8hazHyvShM__rK5YBIqBy9qt0di0Sn4DriBx9TJupdmv6uGmLJyZvDs__QJacVmveFyLIBSPkp21xL0ZvuDW9DMBrD2HB1P4Ry5I.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-45638026-1", {"cookieDomain":"auto"});ga("set", "page", location.pathname + location.search + location.hash);ga("send", "pageview");ga('create', 'UA-189672-38', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__1PtAOZ8x7l6QAi8D7X6Kj_Y7HcUOFzvWRvvYHUcCT98__U8IB92u_CnnGwJ-X_xaUa9lzsC8OzUfN0WkK2JQfirg__QJacVmveFyLIBSPkp21xL0ZvuDW9DMBrD2HB1P4Ry5I.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings,{"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"4i6_PXg9CyJH02z-N_LTjHaKYIyIs79JVTG6BAuUOjo","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"modules\/comment\/comment.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"modules\/forum\/forum.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"sites\/default\/modules\/biorxiv\/css\/biorxiv.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/flexible\/flexible.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_collection\/css\/highwire_subject_collection.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire.highwire-list-expand.css":1,"public:\/\/ctools\/css\/bab59090150caf0c64e27afa1b36eef1.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/contrib\/eu_cookie_compliance\/css\/eu_cookie_compliance.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"0":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_biorxiv_styles\/css\/biorxiv-styles.css.less":1,"https:\/\/d33xdlntwy0kbs.cloudfront.net\/cshl_custom.css":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_figures.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_tables.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"http:\/\/d33xdlntwy0kbs.cloudfront.net\/cshl_custom.js":1,"sites\/all\/modules\/contrib\/eu_cookie_compliance\/js\/eu_cookie_compliance.js":1,"sites\/default\/modules\/jnl_biorxiv_styles\/js\/adv-search-collection-toggle.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/eu_cookie_compliance\/js\/jquery.cookie-1.4.1.min.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/default\/modules\/jnl_biorxiv_styles\/js\/biorxiv-scripts.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_nps_survey\/js\/nps_survey.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"sites\/default\/modules\/contrib\/disqus\/disqus.js":1,"misc\/textarea.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/contrib\/service_links\/js\/twitter_button.js":1,"sites\/all\/modules\/contrib\/service_links\/js\/facebook_like.js":1,"sites\/all\/modules\/contrib\/service_links\/js\/google_plus_one.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.highwire-list-expand.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"10804","apath":"\/biorxiv\/early\/2016\/02\/17\/039818.atom","pisa":"biorxiv;039818v1","markup":[{"requested":"index","variant":"index","view":"full","pisa":"biorxiv;039818\/DC1"},{"requested":"index","variant":"index","view":"full","pisa":"biorxiv;039818\/DC1"},{"requested":"full-text","variant":"full-text","view":"full","pisa":"biorxiv;039818v1"},{"requested":"index","variant":"index","view":"full","pisa":"biorxiv;039818\/DC1"},{"requested":"index","variant":"index","view":"full","pisa":"biorxiv;039818\/DC1"}],"processed":["highwire_math"],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"disqus":{"domain":"biorxivstage","url":"https:\/\/www.biorxiv.org\/content\/10.1101\/039818v1","title":"Assessing the relationship between height growth and molecular genetic variation in Douglas-fir (Pseudotsuga menziesii) provenances","identifier":"node\/10804"},"panels_ajax_pane":{"new-41":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022v5Zl8pXnJfOwYJXkQg1E8I104wfYwxZMeaKflslzqQC0W6HKXTx1UbBdBc6X2wix6+FmYDx64PFrF2EdT2qRi\\\\\\\/JRkoTT+sJsC4lZ+YksllYR7sNdOXVGPlAzGP6T9p1QlnZ0n3YNwLADwjw\\\\\\\/cz87ybonJzZ8798uGvb3iIWMdGJTAMXoHB6YYu1UuZbpxHSbzwGE69wQ3Ybjv116SmLGwn8KssOCRIVSO25TG\\\\\\\/3i5nCd31I1zXjKondgJas7IjZHzYAFBMqsK8H\\\\\\\/QxCcNgAsanU+FYPMgABHEcUoi6Mj1hEaf9ePdkASGLsBVfvXKDSrXOKuWK+jYJR2jHGNXUcPLT+cudpoZlONG\\\\\\\/ij8QTaPufwB5tuvlO2m2M2mHGHzGN7XD3UPYq6JzCMZXY8m27apaR+5XwEUff2dR3Jp+K90zVnhRzmRD\\\\\\\/2jokalFaysTOVahr4IO1OMhxUM7x1tL2NsLRVGJ+nRnVxJ9mZuoU5JuQ2ABTXtX1Xd3wt7IamP9mohUOrKFwGfoG98g26cQXLfEXAfijZE1SZKbKk6zC2YQH8FtmikUwvi0cBN6H3xL7GxyOggYEqOK9pWcTYT6jhO5n+lJAG0R7xUvUVKnoN1D33TwoboRHtzS+7rnh5DYe5Syeol8JMeq9W+lsT5h+8Bqj13ch9E1vuBMFl+diepw8raf53ltOZA\\\\\\\/u\\\\\\\/S5strbvtKlw591UFBhMj4k760TBaU1oFLAwWtgiE5mx0ARAraaW5cvC6\\\\\\\/YQQ5\\\\\\\/QBbF20Rg+OgAhsxmWfC4kJrNUHOS8NxA5sW6ossjleVVZ5sXmhRTl06+uT933EVbrccpTehWRMuxUz3pZKCHEJHR8yxRdQvsDqfd2KrX2Hu99A2mvHxzX1+lvR6wWxVp\\\\\\\/7DHi701\\\\\\\/v+fyu0Z3\\\\\\\/k8NfqdDetlEmopUOgxOoJPLeGIgZYuhAWx0js4e+JnUExTRoS\\\\\\\/h2kjAVoZnXjm2\\\\\\\/gfYugukcI2XNRsHA4AByaVeQBSXwd7Zl2uZBb0xC68hdCsmSAO8XuQUV2SfSEEZccGH8YWR0EI+dKuMRz4IyMbFul2u+jD1HgeAdrOiF4Bt6A6gvYJ90RJHUz16b2qxJFBFMvO0rI30eP3FSIUPVlCtSh3zcwdD97ZYHHboFvdcMvw4nHeP+y8pklLrebieQ5KLqddvwK3KzrjQM+o3dW7lF5uQroNBQZvbOdjeRpC15gsYbXMEBTJLo8Dp91ESH40gsc72gwSmWO5RNR2ow1GOK4g+Y1tdkQa0bXUN5ikoO4W5EjAcITTMtxfrpPtNwzPoE4hjS8IM8zabOlfua2x808NBBe7b1pWs=\\\u0022,\\\u0022iv\\\u0022:\\\u0022mdoV3XL5cBA3rvLxIqIkTQ==\\\u0022,\\\u0022salt\\\u0022:\\\u0022f9d8845712f72c9fee5db7d9ff111eab\\\u0022}\u0022,\u0022hmac\u0022:\u0022e1cfbb7f9389c04c5db03defd5015044d741717400105319c398ddbe78bff65b\u0022}"},"urlIsAjaxTrusted":{"\/content\/10.1101\/039818v1.full":true},"ws_fl":{"width":100,"height":21},"ws_gpo":{"size":"","annotation":"","lang":"","callback":"","width":300},"color":{"logo":"https:\/\/www.biorxiv.org\/sites\/default\/files\/bioRxiv_article.jpg"},"highwire_list_expand":{"is_collapsed":"1"},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"eu_cookie_compliance":{"popup_enabled":1,"popup_agreed_enabled":0,"popup_hide_agreed":0,"popup_clicking_confirmation":1,"popup_scrolling_confirmation":false,"popup_html_info":"\u003Cdiv\u003E\n  \u003Cdiv class =\u0022popup-content info\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n      \u003Cp\u003EWe use cookies on this site to enhance your user experience. By clicking any link on this page you are giving your consent for us to set cookies.\u003C\/p\u003E\n    \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022agree-button eu-cookie-compliance-default-button\u0022\u003EContinue\u003C\/button\u003E\n                    \u003Cbutton type=\u0022button\u0022 class=\u0022find-more-button eu-cookie-compliance-more-button\u0022\u003EFind out more\u003C\/button\u003E\n          \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E","use_mobile_message":false,"mobile_popup_html_info":"\u003Cdiv\u003E\n  \u003Cdiv class =\u0022popup-content info\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n          \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022agree-button eu-cookie-compliance-default-button\u0022\u003EContinue\u003C\/button\u003E\n                    \u003Cbutton type=\u0022button\u0022 class=\u0022find-more-button eu-cookie-compliance-more-button\u0022\u003EFind out more\u003C\/button\u003E\n          \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E\n","mobile_breakpoint":"768","popup_html_agreed":"\u003Cdiv\u003E\n  \u003Cdiv class=\u0022popup-content agreed\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n      \u003Ch2\u003EThank you for accepting cookies\u003C\/h2\u003E\u003Cp\u003EYou can now hide this message or find out more about cookies.\u003C\/p\u003E    \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022hide-popup-button eu-cookie-compliance-hide-button\u0022\u003EHide\u003C\/button\u003E\n              \u003Cbutton type=\u0022button\u0022 class=\u0022find-more-button eu-cookie-compliance-more-button-thank-you\u0022 \u003EMore info\u003C\/button\u003E\n          \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E","popup_use_bare_css":false,"popup_height":"auto","popup_width":"100%","popup_delay":1000,"popup_link":"\/help\/cookie-policy","popup_link_new_window":1,"popup_position":null,"popup_language":"en","store_consent":false,"better_support_for_screen_readers":0,"reload_page":0,"domain":"","popup_eu_only_js":0,"cookie_lifetime":"365","cookie_session":false,"disagree_do_not_show_popup":0,"method":"default","whitelisted_cookies":"","withdraw_markup":"\u003Cbutton type=\u0022button\u0022 class=\u0022eu-cookie-withdraw-tab\u0022\u003EPrivacy settings\u003C\/button\u003E\n\u003Cdiv class=\u0022eu-cookie-withdraw-banner\u0022\u003E\n  \u003Cdiv class=\u0022popup-content info\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n      \u003Cp\u003E\u0026lt;h2\u0026gt;We use cookies on this site to enhance your user experience\u0026lt;\/h2\u0026gt;\u0026lt;p\u0026gt;You have given your consent for us to set cookies.\u0026lt;\/p\u0026gt;\u003C\/p\u003E\n    \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022eu-cookie-withdraw-button\u0022\u003EWithdraw consent\u003C\/button\u003E\n    \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E\n","withdraw_enabled":false},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1,"trackUrlFragments":1},"jnl_biorxiv_styles":{"defaultJCode":"biorxiv"},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-10804 node-type-highwire-article context-content hw-default-jcode-biorxiv hw-article-type-article hw-article-category-new-results">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-M677548" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-M677548');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
        <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
    
  <div id="zone-branding" class="zone zone-branding clearfix print-display-block container-30">
    <div class="grid-15 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="bioRxiv" src="https://www.biorxiv.org/sites/default/files/bioRxiv_article.jpg" /></a>      </div>
                </div>
          </div>
</div><div class="grid-11 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-system block-menu block-main-menu block-system-main-menu odd block-without-title" id="block-system-main-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/" title="" class="" data-icon-position="" data-hide-link-title="0">Home</a></li>
<li class="leaf" role="menuitem"><a href="/about-biorxiv" class="" data-icon-position="" data-hide-link-title="0">About</a></li>
<li class="leaf" role="menuitem"><a href="/submit-a-manuscript" class="" data-icon-position="" data-hide-link-title="0">Submit</a></li>
<li class="last leaf" role="menuitem"><a href="/content/alertsrss" title="" class="" data-icon-position="" data-hide-link-title="0">ALERTS / RSS</a></li>
</ul></nav>    </div>
  </div>
</div><div class="block block-panels-mini block-biorxiv-search-box block-panels-mini-biorxiv-search-box even block-without-title" id="block-panels-mini-biorxiv-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-biorxiv_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/10.1101/039818v1.full" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--2" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-tA0OC-oyIAP6QNmtSvpszhA6x8hWbyMPOkYrv9kx5D4" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2 advanced-search-link" >
  
      
  
  <div class="pane-content">
    <a href="/search">Advanced Search</a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix container-30">
  	      </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="10804" id="node10804" data-pisa="biorxiv;039818v1" data-pisa-master="biorxiv;039818" data-apath="/biorxiv/early/2016/02/17/039818.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div  class="highwire-cite highwire-cite-highwire-article highwire-citation-biorxiv-article-top clearfix has-author-tooltip" >

      <span class="biorxiv-article-type">
        New Results    </span>
  
        <h1 class="highwire-cite-title" id="page-title">Assessing the relationship between height growth and molecular genetic variation in Douglas-fir (<em>Pseudotsuga menziesii</em>) provenances</h1>  
      <div  class="highwire-cite-authors" ><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0"><span class="nlm-given-names">Charalambos</span> <span class="nlm-surname">Neophytou</span></span>, <span class="highwire-citation-author" data-delta="1"><span class="nlm-given-names">Anna-Maria</span> <span class="nlm-surname">Weisser</span></span>, <span class="highwire-citation-author" data-delta="2"><span class="nlm-given-names">Daniel</span> <span class="nlm-surname">Landwehr</span></span>, <span class="highwire-citation-author" data-delta="3"><span class="nlm-given-names">Muhidin</span> <span class="nlm-surname">Šeho</span></span>, <span class="highwire-citation-author" data-delta="4"><span class="nlm-given-names">Ulrich</span> <span class="nlm-surname">Kohnle</span></span>, <span class="highwire-citation-author" data-delta="5"><span class="nlm-given-names">Ingo</span> <span class="nlm-surname">Ensminger</span></span>, <span class="highwire-citation-author" data-delta="6"><span class="nlm-given-names">Henning</span> <span class="nlm-surname">Wildhagen</span></span></span></div>
  
      <div  class="highwire-cite-metadata" ><span  class="highwire-cite-metadata-doi highwire-cite-metadata"><span class="label">doi:</span> https://doi.org/10.1101/039818 </span></div>
  
  
  </div>
<div id="hw-article-author-popups-node10804" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Charalambos Neophytou </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-institution'>Forest Research Institute Baden-Württemberg (FVA)</span>, Freiburg, <span class='nlm-country'>Germany</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Charalambos%2BNeophytou%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Neophytou%20C&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ACharalambos%2BNeophytou%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Anna-Maria Weisser </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-institution'>Forest Research Institute Baden-Württemberg (FVA)</span>, Freiburg, <span class='nlm-country'>Germany</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Anna-Maria%2BWeisser%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Weisser%20A&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AAnna-Maria%2BWeisser%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Daniel Landwehr </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-institution'>Forest Research Institute Baden-Württemberg (FVA)</span>, Freiburg, <span class='nlm-country'>Germany</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Daniel%2BLandwehr%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Landwehr%20D&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ADaniel%2BLandwehr%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Muhidin Šeho </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-institution'>Forest Research Institute Baden-Württemberg (FVA)</span>, Freiburg, <span class='nlm-country'>Germany</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Muhidin%2B%C5%A0eho%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=%C5%A0eho%20M&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AMuhidin%2B%25C5%25A0eho%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Ulrich Kohnle </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-institution'>Forest Research Institute Baden-Württemberg (FVA)</span>, Freiburg, <span class='nlm-country'>Germany</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Ulrich%2BKohnle%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Kohnle%20U&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AUlrich%2BKohnle%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Ingo Ensminger </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-institution'>Forest Research Institute Baden-Württemberg (FVA)</span>, Freiburg, <span class='nlm-country'>Germany</span></div><div class='author-affiliation'><span class='nlm-institution'>Department of Biology, Graduate Programs in Cell &#38; Systems Biology and Ecology and Evolutionary Biology, University of Toronto</span>, Mississauga Campus, Mississauga, Ontario, <span class='nlm-country'>Canada</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Ingo%2BEnsminger%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Ensminger%20I&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AIngo%2BEnsminger%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-6"><div class="author-tooltip-name">Henning Wildhagen </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-institution'>Forest Research Institute Baden-Württemberg (FVA)</span>, Freiburg, <span class='nlm-country'>Germany</span></div><div class='author-affiliation'>Present address: <span class='nlm-institution'>Büsgen-Institut, Department of Forest Botany and Tree Physiology, Georg-August-Universität Göttingen</span>, Göttingen, <span class='nlm-country'>Germany</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Henning%2BWildhagen%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Wildhagen%20H&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AHenning%2BWildhagen%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:hwildha@gwdg.de" class="" data-icon-position="" data-hide-link-title="0">hwildha@gwdg.de</a></span></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/10.1101/039818v1" class="panels-ajax-tab-tab" data-panel-name="biorxiv_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:10804" data-trigger="" data-url-enabled="1">Abstract</a><a href="/panels_ajax_tab/biorxiv_tab_art/node:10804/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/10.1101/039818v1.full-text" class="panels-ajax-tab-tab" data-panel-name="article_tab_full_text" data-target-id="highwire_article_tabs" data-entity-context="node:10804" data-trigger="full-text" data-url-enabled="1">Full Text</a><a href="/panels_ajax_tab/article_tab_full_text/node:10804/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/10.1101/039818v1.article-info" class="panels-ajax-tab-tab" data-panel-name="biorxiv_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:10804" data-trigger="article-info" data-url-enabled="1">Info/History</a><a href="/panels_ajax_tab/biorxiv_tab_info/node:10804/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/10.1101/039818v1.article-metrics" class="panels-ajax-tab-tab" data-panel-name="article_tab_metrics" data-target-id="highwire_article_tabs" data-entity-context="node:10804" data-trigger="article-metrics" data-url-enabled="1">Metrics</a><a href="/panels_ajax_tab/article_tab_metrics/node:10804/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/10.1101/039818v1.supplementary-material" class="panels-ajax-tab-tab" data-panel-name="biorxiv_tab_data" data-target-id="highwire_article_tabs" data-entity-context="node:10804" data-trigger="supplementary-material" data-url-enabled="1">Supplementary material</a><a href="/panels_ajax_tab/biorxiv_tab_data/node:10804/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/10.1101/039818v1.full.pdf+html" class="panels-ajax-tab-tab" data-panel-name="biorxiv_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:10804" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i> Preview PDF</a><a href="/panels_ajax_tab/biorxiv_tab_pdf/node:10804/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="article_tab_full_text" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.biorxiv.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-article_tab_full_text"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-markup" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2 class="">Abstract</h2><p id="p-2">Douglas-fir (<em>Pseudotsuga menziesii</em>) is a conifer tree native to western North America. In central Europe, it shows superior growth performance and is considered a suitable substitute for tree species impaired in vitality due to climate change. Maintenance and improvement of growth performance in a changing environment is a main challenge for forest tree breeders. In this context, genetic variation as a factor underlying phenotypic variation, but also as the basis for future adaptation, is of particular interest. The aims of this study were to analyse (i) genetic diversity of selected Douglas-fir provenances, (ii) variation in height growth among provenances, and (iii) to assess the link between genetic and phenotypic variation height growth. Genotyping was done on microsatellite loci. Effects of ‘provenance’, ‘genotype’, and ‘site’ on height growth were assessed by fitting mixed linear models. The most significant genetic differentiation was observed between provenances of the coastal variety, versus a provenance of the interior variety originating from British Columbia. Although genetic differentiation among provenances of the coastal variety was lower, genetic structures within this variety were identified. Moreover, genetic diversity showed a latitudinal gradient with the southernmost provenances being more diverse, probably reflecting the species' evolutionary history. The modelling approach revealed that height growth differed significantly by provenance, site, and the interaction between site and provenance, demonstrating that height growth is under strong genetic control. Additionally, this analysis showed that genetic variation captured by the genotyped microsatellite loci was significantly related to variation in height growth, providing statistical evidence for a genetic component in the observed phenotypic variation.</p></div><div class="section" id="sec-1"><h2 class="">Introduction</h2><p id="p-3">Douglas-fir (<em>Pseudotsuga menziesii</em> (Mirb.) Franco) is one of the most remarkable conifer species of western North American forests. It occurs naturally in a large geographic area ranging from northern latitude of 19° in Mexico to 55° in the Canadian province of British Columbia, being adapted to a wide variety of ecological conditions. Within its natural distribution area, it can attain majestic dimensions, reaching a height of more than 100 meters and a trunk diameter of more than 4.5 meters (<a id="xref-ref-65-1" class="xref-bibr" href="#ref-65">Schütt et al. 2002</a>, <a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">Aas 2008</a>). Two varieties of Douglas-fir, the coastal (<em>P. menziesii</em> var. <em>menziesii</em>) and the interior (<em>P. menziesii</em> var. <em>glauca</em>) variety, can be well distinguished due to morphological, ecophysiological and growth differences. In addition, a series of transitional forms between these two varieties, as well as a large number of ecotypes can be found within the species’ range (<a id="xref-ref-1-2" class="xref-bibr" href="#ref-1">Aas 2008</a>, <a id="xref-ref-34-1" class="xref-bibr" href="#ref-34">Konnert et al. 2008</a>).</p><p id="p-4">Due to its superior wood quality and high productivity, Douglas-fir has become one of the most important timber trees not only in North America, but also in several countries worldwide (e.g. throughout Europe, in New Zealand, Australia, Chile and Argentina), where it has been successfully introduced (<a id="xref-ref-28-1" class="xref-bibr" href="#ref-28">Hermann and Lavender 1999</a>). In these countries, its growth performance can exceed that of indigenous species. In Europe, for instance, Douglas-fir often displays higher volume increment and yield performance compared to economically significant forest tree species of local origin (<a id="xref-ref-28-2" class="xref-bibr" href="#ref-28">Hermann and Lavander 1999</a>), especially under adverse growth conditions (<a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">Eilmann and Rigling 2012</a>). Therefore, it is of high interest as a source of timber in times of an increased demand on woody biomass.</p><p id="p-5">Soon after the introduction of Douglas-fir to Europe, the factor ‘provenance’ was recognized as an important determinant for phenotypic traits (<a id="xref-ref-33-1" class="xref-bibr" href="#ref-33">Kohnle et al. 2012</a>, <a id="xref-ref-66-1" class="xref-bibr" href="#ref-66">Šeho and Kohnle 2014</a>).</p><p id="p-6">Often, effects of ‘provenance’, i.e. seed source origin, on phenotypic traits are considered to be related to genetic variation among the respective provenances. Provenance research has revealed clinal variation in several adaptive traits like bud flushing, bud set, freezing tolerance and height growth along altitudinal and other climatic variables in Douglas-fir (<a id="xref-ref-54-1" class="xref-bibr" href="#ref-54">Rehfeldt 1989</a>, <a id="xref-ref-39-1" class="xref-bibr" href="#ref-39">Leites et al. 2012</a>, <a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">Chakraborty et al. 2015</a>), but also in other conifer and broadleaved species (<a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">Aitken et al. 2008</a>, <a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">Alberto et al. 2013</a>, <a id="xref-ref-36-1" class="xref-bibr" href="#ref-36">Kremer et al. 2014</a>). However, only in recent years, this putative causal link was subject to systematic investigations in studies testing for an association between variation in phenotypes and allelic variation in candidate genes (for conifers e.g. <a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">González-Martínez et al. 2007</a>, <a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">Eckert et al. 2009</a>). The analysis of associations between phenotypic traits and functional genetic markers such as SNPs is complemented by the analysis of genetic diversity and stratification by means of putatively neutral genetic markers (i.e. occurring in non-expressed genomic regions), like microsatellite markers (Single Sequence Repeats; SSRs).</p><p id="p-7">Microsatellites are appropriate markers for addressing a multitude of population genetic research questions. They are widely used in ecology and forest genetics (<a id="xref-ref-67-1" class="xref-bibr" href="#ref-67">Selkoe and Toonen 2006</a>, <a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">Guichoux et al. 2011</a>, <a id="xref-ref-42-1" class="xref-bibr" href="#ref-42">Martin et al. 2012</a>). Due to their high variability, they provide insights into population genetic structures and evolutionary history of populations. On the other hand, genetic variation of SSR loci may also reflect selection and adaptation processes if these loci are located in the vicinity of genes subjected to natural selection. In this case, allele frequency changes at the gene region as a post-selection effect may also alter genetic differentiation at the SSR locus (<a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">Andolfatto 2001</a>). Genome scans using a large number of loci have shown that divergent selection does not uniformly affect the genome, but is rather indicated by genome regions with an increased differentiation between populations, exceeding the neutral expectation. Specific microsatellite loci have often been classified as such ‘outlier’ loci, which is probably due to the indirect effects of natural selection (<a id="xref-ref-63-1" class="xref-bibr" href="#ref-63">Scotti-Saintagne et al. 2004</a>, <a id="xref-ref-51-1" class="xref-bibr" href="#ref-51">Nosil et al. 2009</a>, <a id="xref-ref-75-1" class="xref-bibr" href="#ref-75">Tsumura et al. 2012</a>). Moreover, recent SSR based genetic studies conducted in natural populations of forest tree species showed an association between genetic variation on the one hand and ecological (e.g. climatic or topographic parameters) or phenotypic variation (e.g. leaf size and growth) on the other (<a id="xref-ref-61-1" class="xref-bibr" href="#ref-61">Ramírez-Valiente et al. 2010</a>, <a id="xref-ref-70-1" class="xref-bibr" href="#ref-70">Sork et al. 2010</a>, <a id="xref-ref-73-1" class="xref-bibr" href="#ref-73">Temunović et al. 2012</a>).</p><p id="p-8">In our study, we focused on a common garden experiment installed in Central Europe, in order to study the genetic and phenotypic variation among Douglas-fir provenances planted there. In particular, we aimed to analyse the effects of genotype and site conditions on growth performance of the provenances. Given that the source populations reside within a large geographic area, we expected to detect genetic differentiation due to isolation and drift among the provenances. Moreover, we hypothesize that genetic adaptation to variable local environments will be reflected in variation in height growth among the provenances. To test these hypotheses, we (i) analysed genetic diversity and differentiation among selected provenances based on data from SSR loci, (ii) we analysed the effects of the factor ‘provenance’ and of climate at provenance origins on variation in height growth, (iii) we assessed whether a provenance effect on height growth is related to genetic variation among the provenances and (iv) we discussed evolutionary implications by assessing statistical relationships between growth, molecular genetic variation, climate and geographic location of the provenances.</p></div><div class="section" id="sec-2"><h2 class="">Materials and methods</h2><div id="sec-3" class="subsection"><h3>Experimental plots and plant material</h3><p id="p-9">The sampled experimental sites are located in south-western Germany and were established in 1961 as part of an international provenance experiment with Douglas-fir (<em>Pseudotsuga menziesii</em> (Mirbel) Franco) with three-year-old saplings. Seeds had been collected from wind-pollinated parent trees in naturally regenerated stands in British Columbia (Canada), Washington, and Oregon (USA) (<a id="xref-ref-71-1" class="xref-bibr" href="#ref-71">Strehlke 1959</a>, <a id="xref-ref-32-1" class="xref-bibr" href="#ref-32">Kenk and Thren 1984</a>). From the various locations of the experimental series in south-western Germany, we selected along a gradient in elevation the experimental sites ‘Schluchsee’ (Dgl123; 1050 m above sea level (a.s.l.)), ‘Sindelfingen’ (Dgl91; 490 m a.s.l.), and ‘Wiesloch’ (Dgl122; 105 m a.s.l.). A brief description of these sites is presented in <a id="xref-table-wrap-1-1" class="xref-table" href="#T1">Table 1</a>. We studied the coastal provenances ‘BC Cameron Lake’, ‘WA Darrington 3 Conrad Creek’, ‘OR Timber’ and ‘OR Santiam River’, as well as the interior provenance ‘BC Salmon Arm 31/102’. The selected provenances a wide latitudinal and longitudinal range of the North American seed source locations present at all three sites; for a detailed description of the origin of these provenances and site conditions, see <a id="xref-ref-33-2" class="xref-bibr" href="#ref-33">Kohnle et al. (2012)</a> and <a id="xref-ref-32-2" class="xref-bibr" href="#ref-32">Kenk and Thren (1984)</a>. The latter reference includes a map showing the location of the experimental sites in Germany. Geographic positions of the provenances’ native habitats are presented in <a id="xref-fig-1-1" class="xref-fig" href="#F1">Figure 1</a>. On each experimental site, each provenance was planted on two to four plots each of 0.1 ha size. Planting density was 3300 trees ha<sup>−1</sup> at a spacing of 1.5 by 2 m. The arrangement of plots on each site did not follow either a systematic or random design in a strict sense, but plots were irregularly arranged so identical provenances never adjoined (<a id="xref-ref-33-3" class="xref-bibr" href="#ref-33">Kohnle et al. 2012</a>). A detailed description of the arrangement of the plots of the studied provenances is given in Supplementary Figure 1. The three selected sites are part of the network of long-term growth and yield experiments of the Forest Research Institute (FVA) Baden-Württemberg. The establishment and height-driven thinning regime of the plots is described in detail in <a id="xref-ref-33-4" class="xref-bibr" href="#ref-33">Kohnle et al. (2012)</a>. The applied height-driven thinning regime resulted in rather uniform stand density dynamics. As a rule, the experiments displayed similar stand densities at identical stand heights. There were, however, some exceptions. As a consequence of the storm damage during the gales of 1990 and 1999, stand densities on some plots at the sites at ‘Wiesloch’ and ‘Sindelfingen’ were reduced below the height-specific target densities (see section <a id="xref-sec-10-1" class="xref-bibr" href="#sec-10"><em>Collection of tree growth data</em></a> for details).</p><div id="F1" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2016/02/17/039818/F1.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Map of the origin of provenances in British Columbia (Canada), Washington (USA) and Oregon (USA). Douglas-fir distribution range (Little 1971) is designated with dark grey (shapefile recovered by USGS publicly available at http://esp.cr.usgs.gov/data/atlas/little/). Shapefiles for political boundaries and coast lines were recovered by the website of the Commission for Environmental Cooperation of North America (CEC; data publicly available at http://www.cec.org/Page.asp?PageID=122&amp;ContentID=2312&amp;SiteNodeID=497&amp;BL_ExpandID=). AR, Salmon Arm; CR, Conrad Creek; LA, Cameron Lake; TI, Timber; RI, Santiam River." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1119361156" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Map of the origin of provenances in British Columbia (Canada), Washington (USA) and Oregon (USA). Douglas-fir distribution range (Little 1971) is designated with dark grey (shapefile recovered by USGS publicly available at http://esp.cr.usgs.gov/data/atlas/little/). Shapefiles for political boundaries and coast lines were recovered by the website of the Commission for Environmental Cooperation of North America (CEC; data publicly available at http://www.cec.org/Page.asp?PageID=122&amp;ContentID=2312&amp;SiteNodeID=497&amp;BL_ExpandID=). AR, Salmon Arm; CR, Conrad Creek; LA, Cameron Lake; TI, Timber; RI, Santiam River.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Fig. 1" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2016/02/17/039818/F1.medium.gif" width="401" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Fig. 1" src="https://www.biorxiv.org/content/biorxiv/early/2016/02/17/039818/F1.medium.gif" width="401" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2016/02/17/039818/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Fig. 1" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2016/02/17/039818/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml"><span class="fig-label">Fig. 1</span> <p id="p-10" class="first-child">Map of the origin of provenances in British Columbia (Canada), Washington (USA) and Oregon (USA). Douglas-fir distribution range (<a id="xref-ref-41-1" class="xref-bibr" href="#ref-41">Little 1971</a>) is designated with dark grey (shapefile recovered by USGS publicly available at <a href="http://esp.cr.usgs.gov/data/atlas/little/">http://esp.cr.usgs.gov/data/atlas/little/</a>). Shapefiles for political boundaries and coast lines were recovered by the website of the Commission for Environmental Cooperation of North America (CEC; data publicly available at <a href="http://www.cec.org/Page.asp?PageID=122&amp;ContentID=2312&amp;SiteNodeID=497&amp;BL_ExpandID=">http://www.cec.org/Page.asp?PageID=122&amp;ContentID=2312&amp;SiteNodeID=497&amp;BL_ExpandID=</a>). AR, Salmon Arm; CR, Conrad Creek; LA, Cameron Lake; TI, Timber; RI, Santiam River.</p><div class="sb-div caption-clear"></div></div></div><div id="T1" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/199113/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup"><a href="/highwire/markup/199113/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/199113" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 1</span> <p id="p-11" class="first-child">Characteristics of the three selected sites and of the source populations represented in these sites of the International Douglas-fir provenance trial</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-4" class="subsection"><h3>Analysis of genetic diversity and differentiation</h3><div id="sec-5" class="subsection"><h4>Sample collection for genotyping of microsatellite loci</h4><p id="p-12">For each provenance at each site needles or cambium was collected from 31 to 40 randomly chosen trees (see Supplementary Table 2 for the number of trees sampled per provenance and site).</p><p id="p-13">These trees represent all plots per provenance and site, with the exception of provenance ‘Salmon Arm’ on site ‘Schluchsee’, where trees of only one of two plots were harvested. Until further utilisation the sampled material was stored at – 80°C.</p></div><div id="sec-6" class="subsection"><h4>Laboratory procedures</h4><p id="p-14">The needle and cambium tissue was first frozen in liquid nitrogen and subsequently homogenised in a mixer mill (MM30, Qiagen, Hilden, Germany). The DNA was extracted using the DNeasy<sup>®</sup> 96 plant kit (Qiagen, Hilden, Germany) following the manufacturers’ protocol for extraction of frozen plant material with the following modifications: After grinding in lysis solution, samples were incubated for 40 min at 65 °C in a water bath; after the addition of the manufacturer’s precipitation buffer AP2, the samples were incubated for 30 min on ice instead of 10 min at −20 °C. The quantification of DNA was performed with a NanoDrop™1000 (Thermo Scientific, Karlsruhe, Germany). For genotyping, six unlinked and highly polymorphic dinucleotide simple sequence repeat (SSR) loci were amplified by polymerase chain reaction (PCR). The marker loci (PmOSU_1C3, PmOSU_2G12, PmOSU_3B2, PmOSU_3F1, PmOSU_3G9 and PmOSU_4A7) were selected from a previous study (<a id="xref-ref-69-1" class="xref-bibr" href="#ref-69">Slavov et al. 2004</a>). As in <a id="xref-ref-38-1" class="xref-bibr" href="#ref-38">Krutovsky et al. (2009)</a>, we refer to these loci without the generic prefix “PmOSU”. PCRs were prepared using the components of the Multiplex PCR kit (Qiagen, Hilden, Germany) according to the manufacturers’ instructions. Oligonucleotide primers for the amplification of these loci were obtained from Applied Biosystems (Darmstadt, Germany) using the sequences given in <a id="xref-ref-69-2" class="xref-bibr" href="#ref-69">Slavov et al. (2004)</a> with the following set-up of fluorescent labelling: 6-Fam for 1C3, NED for 2G12, VIC for 3B2 and 4A7, PET for 3F1, NED for 3G9. The PCR program included an initial denaturation step at 95 °C for 15 min, 36 cycles at 94 °C for 30 s, an annealing step at 55 °C for 90 s, an elongation step at 72 °C for 45 s. Final elongation was performed at 60 °C for 30 min. The analysis of the fragment lengths was performed using the ABI-Prim<sup>®</sup> 3100-Avant Genetic Analyser (Applied Biosystems, Darmstadt, Germany) by means of polymer 3100 Pop-4™, the Dye Set DS-33 and LIZ<sup>®</sup> size standard (Applied Biosystems, Darmstadt, Germany).</p></div><div id="sec-7" class="subsection"><h4>Genotyping</h4><p id="p-15">The software GeneMapper<sup>®</sup> version 4.0 (Applied Biosystems, Darmstadt, Germany) was used for genotyping. Allele bins were set by the software and manually corrected, if necessary. In order to check for rounding errors, the software TANDEM (<a id="xref-ref-43-1" class="xref-bibr" href="#ref-43">Matschiner and Salzburger 2009</a>) was used. This program hampers rounding errors that can occur during manual binning by applying a formula that rounds the allele sizes consistently (<a id="xref-ref-43-2" class="xref-bibr" href="#ref-43">Matschiner and Salzburger 2009</a>). Subsequently, data were checked for null alleles (non-amplifiable) and scoring errors (large allele drop-out and errors due to stuttering) using the software Microchecker v. 2.2.3 (<a id="xref-ref-55-1" class="xref-bibr" href="#ref-55">Van Oosterhout et al. 2004</a>) with the algorithm Brookfield 1. Within each locus, we applied 1,000 randomisations of alleles for the tests. The adjusted genotype lists were used for further analysis.</p></div><div id="sec-8" class="subsection"><h4>Statistical analysis of genetic diversity and differentiation among provenances</h4><p id="p-16">Observed (H<sub>o</sub>) and expected (H<sub>e</sub>) heterozygosity, as well as inbreeding coefficients (F<sub>IS</sub>; <a id="xref-ref-77-1" class="xref-bibr" href="#ref-77">Weir and Cockerham 1984</a>) were calculated using the software Genetix v. 4.05.2 (<a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">Belkhir et al. 2004</a>). We defined 15 populations resulting from the combination of three sites and five provenances. Significance of F<sub>IS</sub> within each population was tested by performing 1,000 random permutations of alleles over individuals using the same software. Number of alleles (n<sub>a</sub>), allelic richness (A) and pairwise fixation indices (F<sub>ST</sub>; <a id="xref-ref-77-2" class="xref-bibr" href="#ref-77">Weir and Cockerham 1984</a>) were computed using the software FSTAT v. 2.9.3.2 (<a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">Goudet 2001</a>). Rarefaction size for the calculation of the allelic richness was set to 10. Following <a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">Goudet et al. (1996)</a>, a test of pairwise differentiation based on 1,000 random permutations of genotypes between populations was carried out with the same computer program. Given that multiple pairwise population comparisons were made, we applied a Bonferroni correction for multiple tests in order to find adjusted rejection probability values, following the instructions included in the software manual. The software Poptree2 (<a id="xref-ref-72-1" class="xref-bibr" href="#ref-72">Takezaki et al. 2010</a>) was used to calculate genetic distance D (<a id="xref-ref-49-1" class="xref-bibr" href="#ref-49">Nei 1972</a>) and for the assembly of a UPGMA phylogenetic tree. Reliability of clades was tested by bootstrapping with 1,000 repetitions.</p><p id="p-17">In order to analyse genetic stratification and assign individuals to <em>K</em> subpopulations (genetic clusters), we employed a model-based Bayesian clustering analysis using the software STRUCTURE 2.3.3 (<a id="xref-ref-57-1" class="xref-bibr" href="#ref-57">Pritchard et al. 2000</a>). This software uses a Markov chain Monte Carlo procedure (MCMC) to infer unstructured subpopulations, which approach the Hardy-Weinberg equilibrium. Membership proportions of each individual to each one of <em>K</em> modelled subpopulations were calculated based on the individuals’ genotype. For the analysis, the admixture model was chosen which assumes that individuals may have mixed ancestry and could thus be assigned to more than one subpopulation. We used the locprior-function (<a id="xref-ref-29-1" class="xref-bibr" href="#ref-29">Hubisz et al. 2009</a>), which was developed to improve the clustering in situations where individuals are sampled from different geographic locations. We assigned the 15 populations to the five provenances. 100,000 updates of the Markov chain and 100,000 MCMC iterations were performed. The procedure was run for <em>K</em> = 1, …, 15. Ten runs were performed for each <em>K</em>. To detect the uppermost hierarchical level of structure and to provide an accurate estimation of population clustering the parameter Δ<em>K</em> after <a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">Evanno et al. (2005)</a> was calculated with the software Structure-Harvester (<a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">Earl and vonHoldt 2012</a>). The maximal Δ<em>K</em> was found for two assumed subpopulations (<em>K</em> = 2). The second highest Δ<em>K</em> was found for of <em>K</em> = 3.</p><p id="p-18">To find the optimal alignment of the replicate cluster analysis (10 runs for each <em>K</em>) the software CLUMPP v. 1.1.2 (<a id="xref-ref-30-1" class="xref-bibr" href="#ref-30">Jakobsson and Rosenberg 2007</a>) was applied. The output from CLUMPP was used as input for the cluster visualisation program DISTRUCT v.1.1 (<a id="xref-ref-62-1" class="xref-bibr" href="#ref-62">Rosenberg 2004</a>).</p><p id="p-19">To validate our results from clustering with STRUCTURE, we used an alternative way to detect structures by applying a multivariate statistical method suited for the detection of clusters and for the reduction of multivariate data to a smaller number of dimensions. For this purpose, single locus genotypes are transformed into allelic variables by assigning scores of 0, 1, or 2 for individuals not carrying a particular allele, carrying it in a heterozygous state, or a homozygous state, respectively (as proposed by <a id="xref-ref-68-1" class="xref-bibr" href="#ref-68">She et al. 1987</a> and <a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">Duplantier et al. 1990</a>). A widely used method for the identification of population stratification, which accommodates the discrete nature of the allelic variables is correspondence analysis. This analysis is implemented in the software Genetix v. 4.05.2 (<a id="xref-ref-8-2" class="xref-bibr" href="#ref-8">Belkhir et al. 2004</a>), which is frequently used in population genetics (e.g. <a id="xref-ref-76-1" class="xref-bibr" href="#ref-76">Vasemägi et al. 2001</a>, <a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">Belaj et al. 2007</a>, <a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">Bryja et al. 2010</a>). We opted to apply such a factorial correspondence analysis (FCA) to our data by employing the function “AFC sur populations” of the Genetix software, where the predefined populations represented by their allele frequency vectors are used as the object for the derivation of the new composite variables (factors) and individuals are subsequently given new coordinates within this composite hyperspace.</p><p id="p-20">Besides their utility for identification of genetic structures, composite variables derived by multivariate analyses may also better reflect natural selection and adaptation processes than allele frequencies themselves (<a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">Grivet et al. 2008</a>, <a id="xref-ref-70-2" class="xref-bibr" href="#ref-70">Sork et al. 2010</a>). This is partly because adaptation processes often involve polygenic traits causing moderate effects across the genome (<a id="xref-ref-58-1" class="xref-bibr" href="#ref-58">Pritchard et al. 2010</a>).</p></div></div><div id="sec-9" class="subsection"><h3>Analysis of tree height data in relation to ‘site’, ‘provenance’, ‘genotype’, and climate</h3><div id="sec-10" class="subsection"><h4>Collection of tree growth data</h4><p id="p-21">The long-term experiments have been routinely monitored by periodic re-measurements in intervals of usually five years commencing at a stand height of approximately 10 m (stand age 18-22 years). The last measurement took place in winter 2011/2012 when the stands were 54 years old; data presented here originate from this last measurement. The height of the trees was determined as the average of two hypsometer measurements (Vertex IV, Haglöf, Langsele, Sweden). In addition to the trees measured in winter 2011/2012, height data of trees cut for a study on branch and stem characteristics, measured in winter 2010/2011 were included in the analysis. The height of these trees was adjusted by adding the annual height increment measured for the last growing season to the determined height. The increment of the last growing season was defined as the length from the tip of the crown to the topmost whorl. All further data analyses on possible provenance and site specific influences on height growth were exclusively based on these measured or projected height data. In total, among the 598 trees measured individually for height, the height of 63 trees was estimated through projection of the height increment of the last year (see Supplementary Table 1 for a summary of the number of trees measured for height per provenance and site).</p><p id="p-22">As it is the rule in even-aged stands, the social (competition) status affects the expression of tree height in the stands of the trial: thick, dominant trees tend to be taller than co-dominant and suppressed trees with smaller diameter. This correlation is captured by the stand height curve. As we were using height data from trees of different diameter, we had to account for possible confounding diameter-related impacts on height. For this purpose, we derived plot-specific height curves by (i) measuring all trees present on the plot for diameter at breast height (at about 1.3 m height; dbh) by averaging two rectangular calliper measurements to the closest mm, and (ii) by measuring approximately 30 trees selected across the range of the dbh for height. From these measurements, the plot specific height curves were derived using the program and function described by <a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">Ehring et al. (1999)</a>.</p><p id="p-23">Since the diameters of all trees were measured, stand-specific diameter characteristic could be calculated from the individual diameters: for example, d200 is calculated as the diameter of the mean basal area tree of the 200 thickest trees ha<sup>−1</sup> (ca. 20 trees per plot). Corresponding stand-specific height characteristics can then be calculated from the plot-specific height curve (e.g. h200 as the height curve value corresponding to d200). Based on these plot-specific height growth curves, we were able to account for possible competition-dependent effects on tree heights: firstly, we restricted all further analyses to height measurements of the dominant trees of the d200 collective (200 thickest trees·ha<sup>−1</sup>; restriction to dominant trees was based on the rationale, that the trajectory of the stand height curve is usually relatively flat among the dominant trees of a stand). Secondly, we developed a correction term (cor_h<sub><em>status</em></sub>) that was added to a tree’s actually measured height. Based on the stand height curves constructed from the measurements in winter 2011/2012, this correction term calculates as follows:
<span class="disp-formula" id="disp-formula-1"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2016/02/17/039818/embed/graphic-3.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2016/02/17/039818/embed/graphic-3.gif"/></noscript></span></span>
where <em>h</em><sub><em>ikl</em></sub> is the height of tree <em>i</em> on plot <em>k</em> at site <em>l</em> derived from the plot-specific stand height curve and the <em>dbh</em> of tree <em>i</em> and h200<sub><em>kl</em></sub> is the height of the d200 tree on plot <em>k</em> at site <em>l</em> derived from the plot-specific stand height curve and d200.</p><p id="p-24">A number of plots on the sites ‘Wiesloch’ and ‘Sindelfingen’ had suffered from wind-throw damage during gales in 1990 or 1999, respectively. In tendency tall trees had suffered more as to be expected from general models on storm damage demonstrating an increase of damage probability with increasing tree height (<a id="xref-ref-64-1" class="xref-bibr" href="#ref-64">Schmidt et al. 2010</a>, <a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">Albrecht et al. 2012</a>, <a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">2013</a>): based on the height data obtained from the measurement prior to the storm, h200 of the trees present before the storm and h200 of the trees actually surviving the storm revealed a reduction of stand height ranging between 0 and 1.4 m with an average of 0.41 m and a standard deviation of 0.43 m. On plots where such storm damage-induced reductions of h200 exceeded 0.1 m (on 22 out of 33 plots), we added another correction term (cor_h<sub><em>storm</em></sub>) as a plot-specific offset value to the individual tree height data in order to compensate for the damage-induced effects on stand height:
<span class="disp-formula" id="disp-formula-2"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2016/02/17/039818/embed/graphic-4.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2016/02/17/039818/embed/graphic-4.gif"/></noscript></span></span>
where h200<sub><em>kl</em></sub> is the h200 of plot <em>k</em> at site <em>l</em> calculated from the plot-specific height curve based on all trees present at the measurement before the storm, and h200_storm<sub><em>kl</em></sub> is h200 of plot <em>k</em> at site <em>l</em> calculated from the plot-specific height curve based on the trees actually surviving the storm until the last measurement. For the 22 plots where a correction term was introduced cor_h<sub><em>storm</em></sub> ranged from 0.1 – 1.4 m and averaged 0.61 m (standard deviation: 0.38 m). Statistical analyses of the proportion of the cumulated damage-related removals of the plots’ total basal area production and of the correction term cor_h<sub><em>storm</em></sub> revealed that the damaging effect was not significantly different among provenances (data not shown).</p></div><div id="sec-11" class="subsection"><h4>Statistical analysis of tree height data in relation to ‘site’, ‘provenance’, and ‘genotype’</h4><p id="p-25">Since not all genotyped trees were scored for height growth, we first matched the genotype and height growth data sets, yielding 315 complete observations (see Supplementary Table 3 for the number of matching trees per provenance and site). Normal distribution of this set of height data was verified graphically. Linear mixed-effects models estimating the height of individual trees were fitted with the function ‘lme’ provided within the ‘nlme’ library (<a id="xref-ref-56-1" class="xref-bibr" href="#ref-56">Pinheiro et al. 2012</a>) within unix version 2.14.0 of the statistical software environment R (<a id="xref-ref-60-1" class="xref-bibr" href="#ref-60">R Development Core Team 2011</a>). The aim of the modelling approach was two-fold: First, we wanted to assess if height growth differed among provenances and if these differences depend on the factor ‘site’. Second we wanted to test if differences in height growth found among populations are related to genetic variation in SSR loci. To pursue these two aims the modelling was done in two independent approaches: To address the first aim, the factors ‘provenance’, ‘site’, as well as the interaction between ‘provenance’ and ‘site’ were included as fixed explanatory factors in the model (referred to as the provenance-based model). To address the second aim, another modelling procedure was done where we omitted ‘provenance’ but included the membership proportions of individual trees in <em>K</em> = 5 clusters as calculated by STRUCTURE (see above for details of this analysis), and the factor ‘site’ (referred to as the STRUCTURE-based model).</p><p id="p-26">Following <a id="xref-ref-78-1" class="xref-bibr" href="#ref-78">Zuur et al. (2007)</a>, we started the model building process with full, potentially beyond-optimal models, including a random intercept for ‘plot’ as a random effect. Next, appropriate fixed-effects structures were selected by subsequently dropping non-significant fixed effects (P &gt; 0.05) with highest P-values as computed with the function ‘anova’. Parameter estimation was computed with the maximum-likelihood technique. The reduced model was compared to the respective full model using BIC. This procedure was iterated until all factors retained in the model were significant. The parameters for the final models were re-estimated with the restricted-maximum-likelihood method.</p><p id="p-27">In the provenance-based approach including ‘provenance’ and ‘site’, the height of tree <em>i</em> belonging to provenance <em>j</em> growing on plot <em>k</em> at site <em>l</em> was modelled as
<span class="disp-formula" id="disp-formula-3"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2016/02/17/039818/embed/graphic-5.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2016/02/17/039818/embed/graphic-5.gif"/></noscript></span></span>
with <em>i</em> = 1, …, 315; <em>j</em> = 1, …, 5; <em>k</em> = 1, …, 31; <em>l</em> = 1, 2, 3. The fixed effects are represented by β<sub>0</sub> (intercept), β<sub>1</sub> (slope for ‘provenance’), β<sub>2</sub> (slope for ‘site’) and β<sub>3</sub> (slope for interaction ‘provenance × site’).</p><p id="p-28">In the second modelling approach omitting ‘provenance’ but including memberships to the genetic clusters identified by STRUCTURE under <em>K</em> = 5 (STRUCTURE-based model), model selection was started with the following full model describing the height of tree <em>i</em> growing on plot <em>k</em> at site <em>l</em> as:
<span class="disp-formula" id="disp-formula-4"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2016/02/17/039818/embed/graphic-6.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2016/02/17/039818/embed/graphic-6.gif"/></noscript></span></span>
with <em>i</em> = 1, …, 315; <em>k</em> = 1, …, 31. The fixed effects are represented by γ<sub>0</sub> (intercept), γ<sub>1</sub> (slope for membership in cluster 1), γ<sub>2</sub> (slope for membership in cluster 2), γ<sub>3</sub> (slope for membership in cluster 3), γ<sub>4</sub> (slope for membership in cluster 4), γ<sub>5</sub> (slope for membership in cluster 5) and γ<sub>6</sub> (slope for ‘site’).</p><p id="p-29">Model selection resulted in a model describing the height of tree <em>i</em> growing on plot <em>k</em> at site <em>l</em> as:
<span class="disp-formula" id="disp-formula-5"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2016/02/17/039818/embed/graphic-7.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2016/02/17/039818/embed/graphic-7.gif"/></noscript></span></span>
with <em>i</em> = 1, …, 315; <em>k</em> = 1, …, 31. The fixed effects are represented by γ<sub>0</sub> (intercept), γ<sub>1</sub> (slope for membership in cluster 1), γ<sub>2</sub> (slope for membership in cluster 2) and γ<sub>3</sub> (slope for ‘site’).</p><p id="p-30">For both models, the random effect for ‘plot’ is denoted by b<sub>ik</sub>. Within-group errors are assumed to be independent identically normally distributed.</p><p id="p-31">Random effects are assumed to be normally distributed. Within-group errors and random effects are assumed to be independent.</p><p id="p-32">Correlation between observations of the same level of ‘plot’ was calculated as
<span class="disp-formula" id="disp-formula-6"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2016/02/17/039818/embed/graphic-8.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2016/02/17/039818/embed/graphic-8.gif"/></noscript></span></span>
where SD denotes the standard deviation.</p><p id="p-33">The validity of the final models was evaluated by graphically inspecting the residuals for normality, constant variance and centring around zero (Supplementary Figure 3, 4). The assumption of normally distributed random effects was assessed graphically, as well (Supplementary Figure 5). The explanatory power of the models was calculated as the squared correlation between the observed values and the predicted values based on fixed-effects and random effects.</p><p id="p-34">In order to compare height growth among provenances within each site, separate models for each site including only ‘provenance’ as a main effect were fitted.</p><p id="p-35">In addition, we applied Mantel tests and partial Mantel tests as a further approach to study the relationship between growth, on the one hand, and site condition or genetic differentiation on the other. For this purpose, we computed dissimilarity matrices comparing all 15 populations pairwise for all these three factors. For calculating growth (phenotypic) dissimilarity, we used height and dbh. After standardising both variables (using the function ‘scale’ in R), we calculated Euclidean distances between populations using the function ‘dist’ in R. In the same way, we produced a matrix of climatic dissimilarity among sites based on all four climatic variables presented in <a id="xref-table-wrap-1-2" class="xref-table" href="#T1">Table 1</a>. As a matrix of genetic dissimilarity, we used Nei distances calculated with the software POPTREE2. We performed the Mantel Tests using the functions ‘mantel’ and ‘partial.mantel’ of the package vegan 2.0-10 (<a id="xref-ref-53-1" class="xref-bibr" href="#ref-53">Oksanen et al. 2013</a>) based on 9999 permutations and Pearson’s correlation coefficient.</p></div><div id="sec-12" class="subsection"><h4>Statistical analysis of tree height data in relation to climatic conditions at the provenances origin</h4><p id="p-36">Finally, we tested whether there are effects of evolutionary processes from the place of origin on growth performance or genetic variation. Following the procedure described above, we studied the correlation between the matrices of (1) genetic differentiation or (2) phenotypic on the one hand and climatic or geographic differentiation among provenances on the other. In the first case, we aimed to analyse whether there is an effect of isolation by distance (IBD) or isolation by adaptation (IBA) on genetic variation at putatively neutral loci. In the second case, we aimed to test whether there are similar effects on growth, a trait of adaptive relevance.</p></div></div></div><div class="section" id="sec-13"><h2 class="">Results</h2><div id="sec-14" class="subsection"><h3>Genetic diversity across loci</h3><p id="p-37">Two of the six loci could not be used for the population genetic analysis. Although we attempted to optimise PCR conditions for the primers of locus 1C3, no evaluable electropherogram could be generated. Besides excessive stuttering, additional peaks, corresponding to additional PCR amplicons, occurred in a large number of samples at this locus. For locus 3B2 no bin-set could be generated. Fragment sizes displayed a rather continuous distribution, whereas a systematic shift of two base pairs would be expected for this locus. Thus, an assignment of the observed peaks to different allele clusters (bins) was not possible and the locus was removed from further analyses. For the remaining loci, genotype lists were adjusted using the software Microchecker 2.2.3, in case of scoring errors (<a id="xref-ref-55-2" class="xref-bibr" href="#ref-55">Van Oosterhout et al. 2004</a>). Genotyping errors detected by Microchecker included ‘null’ alleles and excessive stuttering (Supplementary Table 4).</p><p id="p-38">An analysis of the variability of the four analysed loci over all populations after genotypic adjustments for null alleles Revealed that the number of alleles per locus (without considering null alleles) ranged from 32 at locus 3G9 to 50 at locus 4A7 (<a id="xref-table-wrap-2-1" class="xref-table" href="#T2">Table 2</a>). For the loci 3F1, 2G12 and 4A7 allelic richness was equal or higher than 12.04, while at locus 3G9 it was only 9.85 (<a id="xref-table-wrap-2-2" class="xref-table" href="#T2">Table 2</a>). The observed heterozygosity (H<sub>o</sub>) varied from 0.724 to 0.857. Expected heterozygosity (H<sub>e</sub>) was generally high (at least 0.887). Even after genotypic adjustments for null alleles, inbreeding coefficient F<sub>IS</sub> averaged over all loci was 0.145.</p><div id="T2" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/199112/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup"><a href="/highwire/markup/199112/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/199112" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 2</span> <p id="p-39" class="first-child">Diversity measures for each locus in the whole sample after genotype adjustments for null alleles. Genotype adjustments for null alleles were performed with Microchecker v. 2.2.3 (<a id="xref-ref-55-3" class="xref-bibr" href="#ref-55">van Oosterhout et al. 2004</a>). Number of alleles per locus (n<sub>a</sub>) and allelic richness (A; rarefaction size = 10) were calculated with FSTAT v. 2.9.3.2 (<a id="xref-ref-20-2" class="xref-bibr" href="#ref-20">Goudet 2001</a>), observed (H<sub>o</sub>) and expected heterozygosity (H<sub>e</sub>), inbreeding coefficient (F<sub>IS</sub>) with Genetix v. 4.05.2 (<a id="xref-ref-8-3" class="xref-bibr" href="#ref-8">Belkhir et al. 2004</a>) and null allele frequencies (f<sub>0</sub>) with Microchecker v. 2.2.3</p><div class="sb-div caption-clear"></div></div></div><p id="p-40">The lowest F<sub>IS</sub> value was found for locus 2G12 and the highest value for 4A7 (<a id="xref-table-wrap-2-3" class="xref-table" href="#T2">Table 2</a>). Given the high estimated frequencies of null alleles, only adjusted genotypic data were used for further genetic analyses.</p><p id="p-41">The genetic diversity of the five study provenances is presented in <a id="xref-table-wrap-3-1" class="xref-table" href="#T3">Table 3</a>. With the exception of H<sub>o</sub>, the highest diversity values were measured for provenances ‘Santiam River’ and ‘Timber’. In general, these two provenances showed relatively high diversity across loci. In the remaining three provenances, high diversity values were observed only at specific loci. For instance, provenance ‘Conrad Creek’ displayed high diversity at locus 3G9 and provenance ‘Salmon Arm’ at locus 2G12 (data not shown). On average, the provenances ‘Salmon Arm’ (AR) and ‘Cameron Lake’ (LA) showed the lowest diversity values. However, H<sub>o</sub> for ‘Cameron Lake’ was the highest due to low heterozygote deficiency marked by the lowest inbreeding coefficient (F<sub>IS</sub>).</p><div id="T3" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/199222/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup"><a href="/highwire/markup/199222/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/199222" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 3</span> <p id="p-42" class="first-child">Diversity measures for each provenance averaged over loci and field sites. Average number of alleles per locus (n<sub>a</sub>) and allelic richness (A; rarefaction size = 10) were calculated with FSTAT v. 2.9.3.2 (<a id="xref-ref-20-3" class="xref-bibr" href="#ref-20">Goudet 2001</a>), while observed (H<sub>o</sub>) and expected heterozygosity (H<sub>e</sub>), inbreeding coefficient (F<sub>IS</sub>) were calculated with Genetix v. 4.05.2 (<a id="xref-ref-8-4" class="xref-bibr" href="#ref-8">Belkhir et al. 2004</a>). Abbreviations for provenances: AR, ‘Salmon Arm’; CR, ‘Conrad Creek’; LA, ‘Cameron Lake’; RI, ‘Santiam River’; TI, ‘Timber’. Abbreviations for sites: W, ‘Wiesloch’, SS; ‘Schluchsee’; SF, Sindelfingen.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-15" class="subsection"><h3>Genetic variation, stratification, and differentiation among populations</h3><p id="p-43">F<sub>ST</sub> values between pairs of provenances are presented in <a id="xref-table-wrap-4-1" class="xref-table" href="#T4">Table 4</a>. The most significant differentiation in terms of pairwise F<sub>ST</sub> values was found between the provenance ‘Salmon Arm’ (AR) on the one hand and the other four provenances on the other (<a id="xref-table-wrap-4-2" class="xref-table" href="#T4">Table 4</a>).</p><div id="T4" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/199241/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup"><a href="/highwire/markup/199241/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/199241" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 4</span> <p id="p-44" class="first-child">Pairwise F<sub>ST</sub>-values among provenances (below diagonal) and probability based on test of differentiation (above diagonal) according to <a id="xref-ref-21-2" class="xref-bibr" href="#ref-21">Goudet et al. (1996)</a> computed with FSTAT v. 2.9.3.2 (<a id="xref-ref-20-4" class="xref-bibr" href="#ref-20">Goudet et al. 2001</a>). The indicative adjusted nominal level (5%) for multiple comparisons after Bonferroni correction was 0.005. Abbreviations for provenances (Prov.): AR, ‘Salmon Arm’; CR, ‘Conrad Creek’; LA, ‘Cameron Lake’; RI, ‘Santiam River’; TI, ‘Timber’. Abbreviations for sites: W, ‘Wiesloch’, SS; ‘Schluchsee’; SF, Sindelfingen.</p><div class="sb-div caption-clear"></div></div></div><p id="p-45">Genetic variation among the 15 populations was analysed by computing genetic distances after <a id="xref-ref-49-2" class="xref-bibr" href="#ref-49">Nei (1972)</a>. Genetic distances between the populations were low when pairs of the same provenance (but from different sites) were considered (highest D = 0.37, Supplementary Table 5). For most pairs involving one of the populations of provenance ‘Salmon Arm’, the genetic distances were high (lowest D = 0.29, highest D = 0.69, Supplementary Table 5). To visualise and statistically validate the genetic distances between the populations a phylogenetic tree was constructed (Supplementary Figure 1). It shows that the main division (94 % of the bootstraps) is the branch of the three populations of the provenance ‘Salmon Arm’ (AR), whereas all populations of the other four provenances are clustered on a second main branch.</p><p id="p-46"><a id="xref-fig-2-1" class="xref-fig" href="#F2">Figure 2</a> shows the genetic structure of the populations under the assumption of <em>K</em> = 2, …, 5 as computed with the software STRUCTURE (<a id="xref-ref-57-2" class="xref-bibr" href="#ref-57">Pritchard et al. 2000</a>). For <em>K</em> = 2 obviously the individuals of the provenance ‘Salmon Arm’ at all field sites are assigned to one subpopulation whereas the individuals of the four other provenances over all field sites were assigned together to the second subpopulation. In both cases, membership coefficients were high. The genetic structure assuming <em>K</em> = 3 shows once again that trees of the provenance ‘Salmon Arm’ are clearly assigned together to one subpopulation (marked blue) with high membership coefficients. Individuals of the provenance ‘Cameron Lake’ were assigned to a second subpopulation with little admixture with other clusters. A third subpopulation was mainly represented by the provenances ‘Santiam River’ and ‘Timber’, which showed a limited degree of admixture with the second subpopulation. Trees of the provenance ‘Conrad Creek’, displayed mixed coancestry, with equal membership to both aforementioned clusters. By setting <em>K</em> = 4, provenances ‘Santiam River’ and ‘Timber’ constitute a fourth cluster with comparably low admixture. No further structure was revealed by setting <em>K</em> = 5 or higher.</p><div id="F2" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2016/02/17/039818/F2.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Genetic structure Douglas-fir trees of five provenances growing on three site in south-western Germany computed on the basis of four microsatellite loci for K = 2 up to K = 5 assumed subpopulations. With the software STRUCTURE 2.3.3 (Pritchard et al. 2000) the 15 populations and their individuals were assigned to the subpopulations. Each individual is represented with a vertical bar and each inferred cluster is marked with a different colour (colours are visible only in the online version of this article). Abbreviations for provenances: AR, &#x2018;Salmon Arm&#x2019;; CR, &#x2018;Conrad Creek&#x2019;; LA, &#x2018;Cameron Lake&#x2019;; RI, &#x2018;Santiam River&#x2019;; TI, &#x2018;Timber&#x2019;." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1119361156" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Genetic structure Douglas-fir trees of five provenances growing on three site in south-western Germany computed on the basis of four microsatellite loci for &lt;em&gt;K&lt;/em&gt; = 2 up to &lt;em&gt;K&lt;/em&gt; = 5 assumed subpopulations. With the software STRUCTURE 2.3.3 (Pritchard et al. 2000) the 15 populations and their individuals were assigned to the subpopulations. Each individual is represented with a vertical bar and each inferred cluster is marked with a different colour (colours are visible only in the online version of this article). Abbreviations for provenances: AR, &#x2018;Salmon Arm&#x2019;; CR, &#x2018;Conrad Creek&#x2019;; LA, &#x2018;Cameron Lake&#x2019;; RI, &#x2018;Santiam River&#x2019;; TI, &#x2018;Timber&#x2019;.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Fig. 2" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2016/02/17/039818/F2.medium.gif" width="440" height="207"/><noscript><img class="highwire-fragment fragment-image" alt="Fig. 2" src="https://www.biorxiv.org/content/biorxiv/early/2016/02/17/039818/F2.medium.gif" width="440" height="207"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2016/02/17/039818/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Fig. 2" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2016/02/17/039818/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Fig. 2</span> <p id="p-47" class="first-child">Genetic structure Douglas-fir trees of five provenances growing on three site in south-western Germany computed on the basis of four microsatellite loci for <em>K</em> = 2 up to <em>K</em> = 5 assumed subpopulations. With the software STRUCTURE 2.3.3 (<a id="xref-ref-57-3" class="xref-bibr" href="#ref-57">Pritchard et al. 2000</a>) the 15 populations and their individuals were assigned to the subpopulations. Each individual is represented with a vertical bar and each inferred cluster is marked with a different colour (colours are visible only in the online version of this article). Abbreviations for provenances: AR, ‘Salmon Arm’; CR, ‘Conrad Creek’; LA, ‘Cameron Lake’; RI, ‘Santiam River’; TI, ‘Timber’.</p><div class="sb-div caption-clear"></div></div></div><p id="p-48">To validate the results of population structure calculated by STRUCTURE, we analysed the genotype data with a non-model-based factorial correspondence analysis (FCA) as implemented in the software Genetix (v. 4.05.2, <a id="xref-ref-8-5" class="xref-bibr" href="#ref-8">Belkhir et al. 2004</a>). The first four factors accounted for 23.17, 12.17, 9.02 and 8.48% (a cumulative 52.84%) of the total variation respectively. The most prominent pattern detected with this analysis was a separation along the first axis (Factor-1, see Supplementary Figure 6): Trees of the provenance ‘Salmon Arm’ showed higher scores on this axis, while individuals of the other four provenances clustered around a score of zero. There was a tendency for a separation of provenances ‘Santiam River’ and ‘Cameron Lake’ along Factor-2 (Supplementary Figure 6). Trees of both provenances, ‘Santiam River’ and ‘Salmon Arm’, were spread over a wider range of Factor-3 compared to individuals of the other provenances (Supplementary Figure 6).</p></div><div id="sec-16" class="subsection"><h3>Height of Douglas-fir trees as affected by ‘provenance’, ‘site’ and ‘genotype’</h3><p id="p-49">Exploratory analysis of data on height growth of Douglas-fir trees approximately at the age of 50 years indicates effects of site, provenance, and interaction between these two factors (<a id="xref-fig-3-1" class="xref-fig" href="#F3">Figure 3</a>). Averaged over the three sites investigated in this study, trees of provenance ‘Cameron Lake’ reached a height of 30.7 ± 2.2 m (mean ± standard deviation), those of provenance ‘Conrad Creek’ 32.2 ± 2.3 m, of provenance ‘Salmon Arm’ 30.1 ± 2.2 m, of provenance ‘Santiam River’ 30.4 ± 2.3 m, and those of provenance ‘Timber’ 31.7 ± 1.8 m. The mean heights of trees on the three sites averaged over provenances were 32.7 ± 1.4 m for site ‘Wiesloch’, 29.0 ± 1.8 m for site ‘Schluchsee’, and 30.6 ± 2.2 m for site ‘Sindelfingen’.</p><div id="F3" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2016/02/17/039818/F3.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Height data of five Douglas-fir provenances growing on three sites in south-western Germany. (a) Heights (mean &#xB1; standard error, N= 9 to 34) plotted over sites to illustrate the genotype-environment-interaction effect. (b) Boxplots of heights with the mean per provenance and site depicted by an asterisk (*). The number of trees per provenance and site is indicated by the width of the boxes. The P-value for a significant provenance effect within each site is given in the top left corner of each panel. Numbers at the x-axis denote the provenance ranking within each site. As a measure of stability of provenances rankings across environments, the variance of ranks across sites (var_r) is given in the legend." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1119361156" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Height data of five Douglas-fir provenances growing on three sites in south-western Germany. (a) Heights (mean &#xB1; standard error, N= 9 to 34) plotted over sites to illustrate the genotype-environment-interaction effect. (b) Boxplots of heights with the mean per provenance and site depicted by an asterisk (*). The number of trees per provenance and site is indicated by the width of the boxes. The P-value for a significant provenance effect within each site is given in the top left corner of each panel. Numbers at the x-axis denote the provenance ranking within each site. As a measure of stability of provenances rankings across environments, the variance of ranks across sites (var_r) is given in the legend.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Fig. 3" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2016/02/17/039818/F3.medium.gif" width="440" height="385"/><noscript><img class="highwire-fragment fragment-image" alt="Fig. 3" src="https://www.biorxiv.org/content/biorxiv/early/2016/02/17/039818/F3.medium.gif" width="440" height="385"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2016/02/17/039818/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Fig. 3" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2016/02/17/039818/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Fig. 3</span> <p id="p-50" class="first-child">Height data of five Douglas-fir provenances growing on three sites in south-western Germany. (a) Heights (mean ± standard error, N= 9 to 34) plotted over sites to illustrate the genotype-environment-interaction effect. (b) Boxplots of heights with the mean per provenance and site depicted by an asterisk (*). The number of trees per provenance and site is indicated by the width of the boxes. The P-value for a significant provenance effect within each site is given in the top left corner of each panel. Numbers at the x-axis denote the provenance ranking within each site. As a measure of stability of provenances rankings across environments, the variance of ranks across sites (var_r) is given in the legend.</p><div class="sb-div caption-clear"></div></div></div><p id="p-51">In order to test whether these differences were significant, we fitted mixed linear models to the height data, taking two different approaches. <a id="xref-table-wrap-5-1" class="xref-table" href="#T5">Table 5</a> shows the significant explanatory variables of the two models. In the provenance-based model (<a id="xref-disp-formula-1-1" class="xref-disp-formula" href="#disp-formula-1">eq. 3</a>), both, ‘provenance’- and ‘site’-main effects (P=0.027 and P &lt; 0.001, respectively), as well as the interaction between these two factors (P=0.0175), were significant. To validate that these results, which are based on the set of trees analysed for both height growth and genotypes (n=315), are representative for the entire set of trees measured for height (n=598), we also fitted the provenance-based model to this larger data set. As a result, we obtained the same pattern of statistical significance, albeit with lower P-values (P-values of 0.0029, &lt; 0.001, and 0.0013 for ‘provenance’, ‘site’, and ‘provenance’-×-‘site’ effects) demonstrating that the merged data set of 315 trees was indeed a representative subsample. The visualisation of the interaction effect and the variance of provenance ranks across sites (<a id="xref-fig-3-2" class="xref-fig" href="#F3">Figure 3</a>) revealed that the high growth performance of ‘Conrad Creek’ and ‘Timber’ is quite stable across sites. Both provenances are among the top three ranks on all sites. By contrast, ‘Salmon Arm’ was relatively stable in its low growth performance across sites. Only on site ‘Sindelfingen’ this provenance was able to grow better (29.9 ± 0.3 m) than ‘Santiam River’ (28.7 ± 0.3 m) and ‘Cameron Lake’ (28.9 ± 0.3 m). These latter two provenances showed highly site-dependent growth, reflected in the highest variance of ranks across sites (1.33 and 3.0 respectively). While on site ‘Schluchsee’ provenance ‘Cameron Lake’ was best performing (30.8± 0.5 m), its growth was poor on site ‘Sindelfingen’ (28.9 ± 0.3 m). Similarly, provenance ‘Santiam River’ showed comparable growth on sites ‘Sindelfingen’ (28.7 ± 0.3 m) and ‘Schluchsee’ (28.7 ± 0.4 m), in contrast to the general trend of lowest growth at site ‘Schluchsee’.</p><div id="T5" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/199154/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup"><a href="/highwire/markup/199154/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/199154" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 5</span> <p id="p-52" class="first-child">Significance of terms of the fixed-effects part of the models according to <em>F</em>-Tests. DFnum, degrees of freedom of the numerator; DFdenom, degrees of freedom of the denominator; Cluster-1, Cluster-2, membership to cluster 1 and cluster 2 as identified in the population genetic analysis using STRUCTURE with <em>K</em> = 5 assumed subpopulations</p><div class="sb-div caption-clear"></div></div></div><p id="p-53">In order to analyse whether the differences among provenances within sites were significant, separate models per site were fitted. On sites ‘Schluchsee’ and ‘Sindelfingen’, a significant effect of ‘provenance’ was detected (P=0.03 and P=0.05, respectively), while the differences between provenances on site ‘Wiesloch’ were not significant (P=0.36).</p><p id="p-54">For the two-factorial provenance-based model, the correlation between observations within the same level of ‘plot’, Corr<sub>Plot</sub>, was 0.37, i.e. the random effect accounted for 37%, the within-group error for 63% of the variability not explained by the fixed effects.</p><p id="p-55">In the STRUCTURE-based model (<a id="xref-disp-formula-5-1" class="xref-disp-formula" href="#disp-formula-5">eq. 5</a>), the predictors ‘site’, membership to STRUCTURE ‘cluster-1’ and membership to STRUCTURE ‘cluster-2’ were significant. Corr<sub>Plot</sub> was 0.44, i.e. the random effect accounted for 44%, the within-group error for 56% of the variance not explained by the fixed effects.</p><p id="p-56">The explanatory power of the models was analysed by calculating correlations between observed and predicted values. The squared correlation between observed values and predicted values for the provenance-based model was 0.689 (Supplementary Figure 7), and 0.695 for the STRUCTURE-based model (Supplementary Figure 7).</p><p id="p-57">The assumptions of independent, normally distributed residuals and normally distributed random effects were assessed graphically (Supplementary Figures 3, 4, 5). There were no indications that any of these assumptions were violated.</p><p id="p-58">A significant correlation between growth and field site climatic differentiation was found based on a Mantel test (R=0.51, P=0.0006). The correlation between growth and genetic variation was significant (R=0.25, P=0.0404). In addition, the question was raised whether there was a correlation between growth and genetic differentiation after taking differentiation of field site conditions into account. To address this question, we performed a partial Mantel test. In this case, no significant correlation was found (R=0.19, P=0.0822).</p></div><div id="sec-17" class="subsection"><h3>Height of Douglas-fir trees in relation to climatic conditions at the provenances origin</h3><p id="p-59">Moreover, significant correlations were found between genetic variation and climate of the provenance (R=0.59, P=0.0003) and between genetic variation and geographic distance between provenances (R=0.42, P=0.0009). A significant correlation between genetic variation and climatic differentiation was also found when geographic distance was taken into account by applying a partial Mantel test (R=0.46, P=0.0008). Finally, Mantel tests revealed no significant correlation either between growth and original climate of the provenances (R=0.07, P=0.2163), or between growth and geographic distance between provenances (R=-0.03, P=0.5969). Given that none of the aforementioned tests were significant, we did not perform a partial Mantel test.</p></div></div><div class="section" id="sec-18"><h2 class="">Discussion</h2><div id="sec-19" class="subsection"><h3>Genetic variation among provenances as revealed by microsatellite markers</h3><p id="p-60">Genetic variation at the four analysed SSR loci was high and comparable to earlier studies (<a id="xref-ref-69-3" class="xref-bibr" href="#ref-69">Slavov et al. 2004</a>, <a id="xref-ref-38-2" class="xref-bibr" href="#ref-38">Krutovsky et al. 2009</a>). A characteristic of the used markers is the presence of ‘null’ (non-amplified) alleles. These reached frequencies up to 31 % in our study. In addition to ‘null’ alleles, errors due to excessive stuttering might have occurred, thus explaining the fact that F<sub>IS</sub> values remained high and significant even after genotypic adjustments. These results are comparable to the results reported by <a id="xref-ref-38-3" class="xref-bibr" href="#ref-38">Krutovsky et al. (2009)</a>, who used the same microsatellite markers.</p><p id="p-61">Different levels of genetic diversity among provenances are suggested by our results. Both the highest variability (in terms of n<sub>a</sub>; see <a id="xref-table-wrap-3-2" class="xref-table" href="#T3">Table 3</a>) and diversity (in terms of A, H<sub>o</sub> and H<sub>e</sub>; see <a id="xref-table-wrap-3-3" class="xref-table" href="#T3">Tables 3</a> and <a id="xref-table-wrap-4-3" class="xref-table" href="#T4">4</a>) were measured in provenances ‘Santiam River’ and ‘Timber’. The lowest variability and diversity were observed in the two northernmost provenances, ‘Salmon Arm’ and ‘Cameron Lake’, while provenance ‘Conrad Creek’ displayed median values. This pattern agrees with the glacial history and post-glacial migration of the species. Pollen and macrofossil evidence place a significant refugial population in the Willamette Valley (northern Oregon), in a region where both ‘Santiam River’ and ‘Timber’ provenances reside (<a id="xref-ref-74-1" class="xref-bibr" href="#ref-74">Tsukada 1982</a>, <a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">Gugger and Sugita 2010</a>). A relatively high genetic diversity for this area, which is in concordance with the existence of a refugial population, has also been indicated by a range wide study of <a id="xref-ref-40-1" class="xref-bibr" href="#ref-40">Li and Adams (1989)</a>. Long persistence of these populations on their current locations might have allowed for accumulation of genetic diversity. In contrast, post-glacial recolonization may have led to losses of genetic diversity due to genetic drift, resulting in a reduced genetic diversity in the northernmost provenances of our study.</p><p id="p-62">A significant differentiation of ‘Salmon Arm’ from all other provenances is shown by pairwise measures of genetic differentiation, as well as by the Bayesian Analysis of population structure (based on the STRUCTURE software) and the Factorial Correspondence Analysis (FCA). Even though the number of loci genotyped in the present study is not high, we consider the presented results of the STRUCTURE analysis as robust, since <a id="xref-ref-29-2" class="xref-bibr" href="#ref-29">Hubisz et al. (2009)</a> reported that by using the locprior-Option, the true population structure will be well approximated with a number of loci as small as two. The STRUCTURE analysis clearly shows that irrespective of the chosen clustering solution, individuals of provenance ‘Salmon Arm’ form a distinct subpopulation. This supports the different origin of the source population, which is located in an area where the two varieties – coastal and interior – may intermingle (<a id="xref-ref-33-5" class="xref-bibr" href="#ref-33">Kohnle et al. 2012</a>, <a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">Gugger et al. 2010</a>).</p><p id="p-63">On the other hand, genetic differentiation among the four remaining, coastal provenances was more limited, but could also be identified by means of the aforementioned methods. In particular, provenances ‘Santiam River’ and ‘Timber’, which originate from habitats relatively close to each other in Oregon, were genetically homogenous. These two southernmost provenances could be distinguished from the northernmost coastal provenance ‘Cameron Lake’ from British Columbia. ‘Conrad Creek’, originating from a region between the two extremes, showed an admixed gene pool between the northernmost and the southernmost coastal provenances (see the clustering solution of STRUCTURE for three assumed subpopulations; <em>K</em> = 3), which is in agreement with a latitudinal gradient. Remarkably, very similar patterns of genetic differentiation were found in a complementary study, where the genetic diversity of a subset of trees of the same populations were analysed on the basis of approximately 80,000 SNP markers (<a id="xref-ref-47-1" class="xref-bibr" href="#ref-47">Müller et al. 2015</a>). Both isolation-by-distance (IBD) and geographic barriers acting currently and historically might account for this differentiation pattern among provenances studied in this work. <a id="xref-ref-38-4" class="xref-bibr" href="#ref-38">Krutovsky et al. (2009)</a> analysed the relationship between genetic differentiation and geographic distance with a larger number of populations covering the whole range of the coastal variety within the United States and found a positive and significant correlation between genetic differentiation and distance.</p><p id="p-64">Besides migration and drift, an additional potential cause of the revealed genetic differentiation among the study provenances could be natural selection, leading to isolation-by-adaptation (IBA). In particular, adaptive divergence can also increase genome-wide differentiation by promoting general barriers to neutral gene flow, thereby facilitating genomic divergence via genetic drift. This latter process can yield a positive correlation between adaptive phenotypic divergence and neutral genetic differentiation (<a id="xref-ref-50-1" class="xref-bibr" href="#ref-50">Nosil et al. 2008</a>). Mantel tests carried out in our study showed that there was a correlation between genetic differentiation and climate at origin, even when the effects of IBD were taken into account. IBA may occur due to several ecological differences like the intensity of summer drought, or increasingly continental climate, which characterises the interior part of the region where the source populations originate from (<a id="xref-ref-27-1" class="xref-bibr" href="#ref-27">Hermann and Lavender 1990</a>, <a id="xref-ref-50-2" class="xref-bibr" href="#ref-50">Nosil et al. 2008</a>, <a id="xref-ref-45-1" class="xref-bibr" href="#ref-45">Mosca et al. 2014</a>). A genetic basis of drought tolerance of Douglas-fir has been recently indicated based on physiological data and growth in provenance trials (<a id="xref-ref-31-1" class="xref-bibr" href="#ref-31">Jansen et al. 2013</a>), as well as genetic differentiation in drought related genes (<a id="xref-ref-47-2" class="xref-bibr" href="#ref-47">Müller et al. 2015</a>). Additionally, a number of genes have been associated with cold hardiness in the frame of association studies (<a id="xref-ref-37-1" class="xref-bibr" href="#ref-37">Krutovsky and Neale 2005</a>, <a id="xref-ref-14-2" class="xref-bibr" href="#ref-14">Eckert et al. 2009</a>). Even if the loci of our study are considered as selectively neutral, it cannot be precluded that they carry the imprints of natural selection due to selection at closely linked genes due to genetic hitchhiking (<a id="xref-ref-6-2" class="xref-bibr" href="#ref-6">Andolfatto 2001</a>).</p></div><div id="sec-20" class="subsection"><h3>Significant effects of ‘site’, ‘provenance’, and their interaction on height growth</h3><p id="p-65">The statistical analysis of height growth of Douglas-fir revealed significant differences between sites. Height growth is a well-established indicator for site productivity, that is, besides being putatively affected by the genetic background of the trees, it is largely determined by growth relevant environmental factors such as temperature, precipitation and soil properties (<a id="xref-ref-59-1" class="xref-bibr" href="#ref-59">Puettmann et al. 2009</a>, <a id="xref-ref-44-1" class="xref-bibr" href="#ref-44">Messaoud and Chen 2011</a>, <a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">Darychuk et al. 2012</a>). Accordingly, differences in height growth among sites found in our study clearly reflect gradients in environmental factors that will determine height growth. This correlation between growth and among-site climatic variation was also confirmed by means of Mantel test and was highly significant. The site with the lowest growth potential, ‘Schluchsee’, is located at a high elevation (1050 m a.s.l.), with accordingly lower mean annual temperature (6.1 °C) and a shorter growing period, whereas the site ‘Wiesloch’ is located in the lowland (105 m a.s.l) and is characterised by a mean annual air temperature of 9.9 °C. Among the three sites studied here, ‘Sindelfingen’ is situated at a medium elevation and is characterised by intermediate growth potential as well as intermediate site conditions (8.2 °C, 490 m a.s.l.). In a study on growth phenology in seedlings of coastal Douglas-fir, <a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">Gould et al. (2012)</a> reported increased height growth in environments with higher mean annual temperatures, regardless of seed source origin. Interestingly, the rather low annual precipitation on site ‘Wiesloch’, in combination with a sandy soil type does obviously not override the positive effect of the higher temperatures typical for this site. This is in agreement with recently published results from field trials in Austria and Bavaria (SE Germany) showing that increasing temperature at the planting site had a significant positive effect on height growth of Douglas-fir, whereas precipitation variation did not explain growth differences among sites (<a id="xref-ref-10-2" class="xref-bibr" href="#ref-10">Chakraborty et al. 2015</a>).</p><p id="p-66">Attempts to analyse the effect of seed source origin (i.e. the effect of the factor ‘provenance’) on phenotypic traits usually assume that such differences of geographic position reflect differences of genetic variation in that trait. One objective of this study was to analyse the effect of seed source origin on height growth. Though less pronounced than the effect of field site, we showed that provenance has also an effect on growth (<a id="xref-table-wrap-5-2" class="xref-table" href="#T5">Table 5</a>). This is supported by the Mantel test between phenotypic and genetic distances, too, but it is not confirmed by the partial Mantel test which takes the climatic differentiation among field sites into account. Therefore, our results do not support a link between climate of origin and growth performance which would be indicative of adaptation. However, it should be taken into account that no more than five provenances planted on three sites could be included in our study, which is probably too few to draw general conclusions. Significant effects of the environment of origin on height growth have been shown elsewhere based on larger numbers of Douglas-fir provenances (<a id="xref-ref-35-1" class="xref-bibr" href="#ref-35">Krakowski &amp; Stoehr 2009</a>, <a id="xref-ref-39-2" class="xref-bibr" href="#ref-39">Leites et al. 2012</a>, <a id="xref-ref-10-3" class="xref-bibr" href="#ref-10">Chakraborty et al. 2015</a>). For example, previous reports suggested that provenances from sites with high mean annual temperature show a better growth potential (<a id="xref-ref-39-3" class="xref-bibr" href="#ref-39">Leites et al. 2012</a>, <a id="xref-ref-10-4" class="xref-bibr" href="#ref-10">Chakraborty et al. 2015</a>), which is in agreement with the constantly high growth potential of ‘Conrad Creek’ observed in our study. In contrast, provenance ‘Cameron Lake’ showed highly site-dependent growth performance. The high growth potential on site ‘Schluchsee’ along with a near-average or low height growth on sites ‘Sindelfingen’ and ‘Wiesloch’, respectively, may suggest that on the latter two sites, the low annual precipitation might restrict height growth of ‘Cameron Lake’ (climate of origin is wetter; see <a id="xref-table-wrap-1-3" class="xref-table" href="#T1">Table 1</a>). Recent measurements on other sites of the provenance tests confirm these trends; ‘Conrad Creek’ was always above average, whereas the height of ‘Cameron Lake’ was site specific elsewhere, too.</p></div><div id="sec-21" class="subsection"><h3>Genetic variation in microsatellite loci is related to phenotypic variation in height growth</h3><p id="p-67">The superior growth of either ‘Cameron Lake’ (on site ‘Schluchsee’) or ‘Conrad Creek’ (on sites ‘Sindelfingen’ and ‘Wiesloch’) may suggest a north-south gradient of growth performance for the coastal provenances, which agrees with results reported by <a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">Eilmann et al. (2013)</a>. Interestingly, the high growth performance of the provenances ‘Conrad Creek’ and ‘Cameron Lake’ coincides with a low genetic diversity of these provenances (<a id="xref-table-wrap-3-4" class="xref-table" href="#T3">Table 3</a>). With the accompanying genetic analysis, we are able to show that this – pronounced and highly significant – phenotypic difference might indeed be related to a genetic differentiation (compare <a id="xref-fig-2-2" class="xref-fig" href="#F2">Figures 2</a>, <a id="xref-fig-3-3" class="xref-fig" href="#F3">3</a>, Supplementary Table 5, Supplementary Figure 2). In the STRUCTURE-based modelling approach, the membership proportions to Cluster 1 and Cluster 2 were identified as significant predictors, providing statistical evidence for a genetic component in the observed phenotypic differentiation in height growth (<a id="xref-table-wrap-5-3" class="xref-table" href="#T5">Table 5</a>). Cluster 1 (indicated in blue in <a id="xref-fig-2-3" class="xref-fig" href="#F2">Figure 2</a>) largely represents the genetic differentiation of the provenance ‘Salmon Arm’ from the coastal provenances, while the membership proportions to Cluster 2 (indicated in red in <a id="xref-fig-2-4" class="xref-fig" href="#F2">Figure 2</a>) separate Santiam River and Timber on the one hand, and Cameron Lake and Conrad Creek on the other. Given the fact that trees of provenance ‘Salmon Arm’ perform low in height growth on all three sites, it is not surprising that the membership to Cluster 1 is a significant explanatory variable for height growth. The pattern of membership proportions of the four coastal provenances to Cluster 2 indicate that genetic differentiation correlates with the significant differences in height growth observed between ‘Conrad Creek’ and ‘Santiam River’.</p><p id="p-68">An interesting result is the quasi-identical explanatory power of the two modelling approaches. While the provenance-based model can only account for among-provenance variation, the STRUCTURE-based model potentially reflects both among-and within-provenance genetic variation. However, the quasi-identical explanatory power of the two models argues for a low extent of within-provenance genetic variation in the analysed loci. Given the putative nature of height growth as a quantitative trait, the relatively small number of microsatellite loci genotyped in this study, the large genome size of Douglas-fir (<a id="xref-ref-52-1" class="xref-bibr" href="#ref-52">O’Brien et al. 1996</a>) and the rapid decay of linkage disequilibrium in forest trees (<a id="xref-ref-48-1" class="xref-bibr" href="#ref-48">Neale and Kremer, 2011</a>) it appears reasonable that genetic variation underlying the observed differences in height growth is only partially captured by the genetic parameters derived from the SSR genotypes. As a consequence, the STRUCTURE-based model does not outperform the provenance-based approach. With the set of single nucleotide polymorphism markers developed for Douglas-fir recently (<a id="xref-ref-46-1" class="xref-bibr" href="#ref-46">Müller et al. 2012</a>; <a id="xref-ref-47-3" class="xref-bibr" href="#ref-47">Müller et al. 2015</a>), we now have the tools available to analyse the effect of adaptive genetic variation on height growth (and other phenotypic traits) directly.</p></div></div><div class="section" id="sec-22"><h2 class="">Conclusion</h2><p id="p-69">Considering the importance of genetic diversity for the adaptability of tree populations, and the fact that our results on height growth suggest a tradeoff between genetic diversity and growth, this study is of significance for forest managers. For a sustainable management of forest ecosystems under conditions of a changing climate, it is of particular significance to learn more about adaptedness and adaptability of tree species on sites where they are introduced. In our study, we did this by assessing phenotypic variation along environmental gradients and by relating it to climate of origin and genotypic variation. An innovative aspect of the study was to directly test a genetic component in height growth variation by deriving explanatory variables from molecular data. The use of growth data from stands of an age of over 50 years – higher than most provenance research studies known to us – increases the significance and confidence of the results. On the other hand, this study was restricted to five provenances and three sites. In order to enable the development of climate-and genotype-sensitive models of height growth for Douglas-fir, future research should be based on larger sets of sites, provenances, and genetic markers.</p></div><div class="section" id="sec-23"><h2 class="">Conflict of Interest Statement</h2><p id="p-71">The authors declare that the research was conducted in the absence of any commercial or financial relationship that could be construed as a potential conflict of interest.</p></div><div class="section ack" id="ack-1"><h2 class="">Acknowledgements</h2><p id="p-70">The study sites are part of the International Douglas-fir provenance trial, initiated by Prof. R. Schober after a decision of the Unit of Forest Growth of the German Association of Forest Research Stations (DFFVA). The German federal state of Baden-Württemberg joined this provenance trial series in 1961 with test plantations at 15 locations, of which ten are currently still active and supervised by the Forest Research Institute of Baden-Württemberg (FVA). Seeds for the experiments were collected in 1955 by B. Strehlke from autochthonous stands in North America and from selected cone-bearing stands growing in Southwestern Germany judged by forest practitioners to display vigorous growth. We thank Dr. Axel Albrecht for helpful discussions. This study was financially supported by the Forest Research Institute Baden-Württemberg with funding to I.E., U.K., H.W. and by the German Science Foundation (DFG) with grants to U.K. (KO 931/8-1) and I.E. (EN 829/4-1) as part of the collaborative project DougAdapt (PAK 583/468).</p></div><div class="section ref-list" id="ref-list-1"><h2 class="">References</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference  in text" id="ref-1">↵</a><div class="cit ref-cit ref-book" id="cit-039818v1.1"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Aas</span>  <span class="cit-name-given-names">G</span></span> (<span class="cit-pub-date">2008</span>) <span class="cit-chapter-title">Die Douglasie (Pseudotsuga menziesii) in Nordamerika: Verbreitung, Variabilität und Ökologie</span>. <span class="cit-source">Die Douglasie-Perspektiven im Klimawandel</span>. <span class="cit-publ-name">LWF Wissen</span> <span class="cit-vol">59</span>:<span class="cit-fpage">7</span>–<span class="cit-lpage">11</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DDie%2BDouglasie-Perspektiven%2Bim%2BKlimawandel%26rft.volume%253D59%26rft.spage%253D7%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference  in text" id="ref-2">↵</a><div class="cit ref-cit ref-journal" id="cit-039818v1.2" data-doi="10.1111/j.1752-4571.2007.00013.x"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Aitken</span>  <span class="cit-name-given-names">SN</span></span>, <span class="cit-auth"><span class="cit-name-surname">Yeaman</span>  <span class="cit-name-given-names">S</span></span>, <span class="cit-auth"><span class="cit-name-surname">Holliday</span>  <span class="cit-name-given-names">J A</span></span>, <span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">T</span></span>, <span class="cit-auth"><span class="cit-name-surname">Curtis-McLane</span>  <span class="cit-name-given-names">S</span></span> (<span class="cit-pub-date">2008</span>) <span class="cit-article-title">Adaptation, migration or extirpation: climate change outcomes for tree populations</span>. <abbr class="cit-jnl-abbrev">Evol Appl</abbr> <span class="cit-vol">1</span>:<span class="cit-fpage">95</span>–<span class="cit-lpage">111</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEvol%2BAppl%26rft.volume%253D1%26rft.spage%253D95%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1752-4571.2007.00013.x%26rft_id%253Dinfo%253Apmid%252F25567494%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1752-4571.2007.00013.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25567494&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2016%2F02%2F17%2F039818.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000262783400009&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference  in text" id="ref-3">↵</a><div class="cit ref-cit ref-journal" id="cit-039818v1.3"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Alberto</span>  <span class="cit-name-given-names">FJ</span></span>, <span class="cit-auth"><span class="cit-name-surname">Aitken</span>  <span class="cit-name-given-names">SN</span></span>, <span class="cit-auth"><span class="cit-name-surname">Alía</span>  <span class="cit-name-given-names">R</span></span>, <span class="cit-auth"><span class="cit-name-surname">González-Martínez</span>  <span class="cit-name-given-names">SC</span></span>, <span class="cit-auth"><span class="cit-name-surname">Hänninen</span>  <span class="cit-name-given-names">H</span></span>, <span class="cit-auth"><span class="cit-name-surname">Kremer</span>  <span class="cit-name-given-names">A</span></span>, <span class="cit-auth"><span class="cit-name-surname">Lefèvre</span>  <span class="cit-name-given-names">F</span></span>, <span class="cit-auth"><span class="cit-name-surname">Lenormand</span>  <span class="cit-name-given-names">T</span></span>, <span class="cit-auth"><span class="cit-name-surname">Yeaman</span>  <span class="cit-name-given-names">S</span></span>, <span class="cit-auth"><span class="cit-name-surname">Whetten</span>  <span class="cit-name-given-names">R</span></span>, <span class="cit-auth"><span class="cit-name-surname">Savolainen</span>  <span class="cit-name-given-names">O</span></span> (<span class="cit-pub-date">2013</span>) <span class="cit-article-title">Potential for evolutionary responses to climate change–evidence from tree populations</span>. <abbr class="cit-jnl-abbrev">Glob Chang Biol</abbr> <span class="cit-vol">19</span>:<span class="cit-fpage">1645</span>–<span class="cit-lpage">1661</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGlob%2BChang%2BBiol%26rft.volume%253D19%26rft.spage%253D1645%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference  in text" id="ref-4">↵</a><div class="cit ref-cit ref-journal" id="cit-039818v1.4" data-doi="10.1007/s13595-012-0244-x"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Albrecht</span>  <span class="cit-name-given-names">A</span></span>, <span class="cit-auth"><span class="cit-name-surname">Kohnle</span>  <span class="cit-name-given-names">U</span></span>, <span class="cit-auth"><span class="cit-name-surname">Hanewinkel</span>,  <span class="cit-name-given-names">M</span></span>, <span class="cit-auth"><span class="cit-name-surname">Bauhus</span>  <span class="cit-name-given-names">J</span></span> (<span class="cit-pub-date">2013</span>) <span class="cit-article-title">Storm damage in Douglas-fir unexpectedly high compared to Norway spruce</span>. <abbr class="cit-jnl-abbrev">Ann For Sci</abbr> <span class="cit-vol">70</span>:<span class="cit-fpage">195</span>–<span class="cit-lpage">207</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAnn%2BFor%2BSci%26rft.volume%253D70%26rft.spage%253D195%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs13595-012-0244-x%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s13595-012-0244-x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference  in text" id="ref-5">↵</a><div class="cit ref-cit ref-journal" id="cit-039818v1.5" data-doi="10.1007/s10342-010-0432-x"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Albrecht</span>  <span class="cit-name-given-names">A</span></span>, <span class="cit-auth"><span class="cit-name-surname">Hanewinkel</span>  <span class="cit-name-given-names">M</span></span>, <span class="cit-auth"><span class="cit-name-surname">Bauhus</span>  <span class="cit-name-given-names">J</span></span>, <span class="cit-auth"><span class="cit-name-surname">Kohnle</span>  <span class="cit-name-given-names">U</span></span> (<span class="cit-pub-date">2012</span>) <span class="cit-article-title">How does silviculture affect storm damage in forests of south-western Germany? Results from empirical modeling based on long-term observations</span>. <abbr class="cit-jnl-abbrev">Eur J For Res</abbr> <span class="cit-vol">131</span>:<span class="cit-fpage">229</span>–<span class="cit-lpage">247</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEur%2BJ%2BFor%2BRes%26rft.volume%253D131%26rft.spage%253D229%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs10342-010-0432-x%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s10342-010-0432-x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference  in text" id="ref-6">↵</a><div class="cit ref-cit ref-journal" id="cit-039818v1.6" data-doi="10.1016/S0959-437X(00)00246-X"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Andolfatto</span>  <span class="cit-name-given-names">P</span></span> (<span class="cit-pub-date">2001</span>) <span class="cit-article-title">Adaptive hitchhiking effects on genome variability</span>. <abbr class="cit-jnl-abbrev">Curr Opin Genet Dev</abbr> <span class="cit-vol">11</span>:<span class="cit-fpage">635</span>–<span class="cit-lpage">641</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCurrent%2Bopinion%2Bin%2Bgenetics%2B%2526%2Bdevelopment%26rft.stitle%253DCurr%2BOpin%2BGenet%2BDev%26rft.aulast%253DAndolfatto%26rft.auinit1%253DP.%26rft.volume%253D11%26rft.issue%253D6%26rft.spage%253D635%26rft.epage%253D641%26rft.atitle%253DAdaptive%2Bhitchhiking%2Beffects%2Bon%2Bgenome%2Bvariability.%26rft_id%253Dinfo%253Adoi%252F10.1016%252FS0959-437X%252800%252900246-X%26rft_id%253Dinfo%253Apmid%252F11682306%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/S0959-437X(00)00246-X&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=11682306&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2016%2F02%2F17%2F039818.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000171989300006&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference  in text" id="ref-7">↵</a><div class="cit ref-cit ref-journal" id="cit-039818v1.7" data-doi="10.1093/aob/mcm132"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Belaj</span>  <span class="cit-name-given-names">A</span></span>, <span class="cit-auth"><span class="cit-name-surname">Muñoz-Diez</span>  <span class="cit-name-given-names">C</span></span>, <span class="cit-auth"><span class="cit-name-surname">Baldoni</span>  <span class="cit-name-given-names">L</span></span>, <span class="cit-auth"><span class="cit-name-surname">Porceddu</span>  <span class="cit-name-given-names">A</span></span>, <span class="cit-auth"><span class="cit-name-surname">Barranco</span>  <span class="cit-name-given-names">D</span></span>, <span class="cit-auth"><span class="cit-name-surname">Satovic</span>  <span class="cit-name-given-names">Z</span></span> (<span class="cit-pub-date">2007</span>) <span class="cit-article-title">Genetic diversity and population structure of wild olives from the north-western Mediterranean assessed by SSR Markers</span>. <abbr class="cit-jnl-abbrev">Ann Bot</abbr> <span class="cit-vol">100</span>:<span class="cit-fpage">449</span>–<span class="cit-lpage">458</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAnn%2BBot%26rft_id%253Dinfo%253Adoi%252F10.1093%252Faob%252Fmcm132%26rft_id%253Dinfo%253Apmid%252F17613587%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/aob/mcm132&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17613587&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2016%2F02%2F17%2F039818.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000249654600003&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference  in text" id="ref-8">↵</a><div class="cit ref-cit ref-journal" id="cit-039818v1.8"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Belkhir</span>  <span class="cit-name-given-names">K</span></span>, <span class="cit-auth"><span class="cit-name-surname">Borsa</span>  <span class="cit-name-given-names">P</span></span>, <span class="cit-auth"><span class="cit-name-surname">Chikhi</span>  <span class="cit-name-given-names">L</span></span>, <span class="cit-auth"><span class="cit-name-surname">Raufaste</span>  <span class="cit-name-given-names">N</span></span>, <span class="cit-auth"><span class="cit-name-surname">Bonhomme</span>  <span class="cit-name-given-names">F</span></span> (<span class="cit-pub-date">2004</span>) <span class="cit-article-title">GENETIX 4.05.2, Windows™ Software for Population Genetics</span>. <abbr class="cit-jnl-abbrev">Laboratoire génome, populations, interactions, CNRS UMR, 5000, University of Montpellier</abbr></cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference  in text" id="ref-9">↵</a><div class="cit ref-cit ref-journal" id="cit-039818v1.9" data-doi="10.1111/j.1365-294X.2010.04844.x"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Bryja</span>  <span class="cit-name-given-names">J</span></span>, <span class="cit-auth"><span class="cit-name-surname">Smith</span>  <span class="cit-name-given-names">C</span></span>, <span class="cit-auth"><span class="cit-name-surname">Konevcnỳ</span>  <span class="cit-name-given-names">A</span></span>, <span class="cit-auth"><span class="cit-name-surname">Reichard</span>  <span class="cit-name-given-names">M</span></span> (<span class="cit-pub-date">2010</span>) <span class="cit-article-title">Range-wide population genetic structure of the European bitterling (Rhodeus amarus) based on microsatellite and mitochondrial DNA analysis</span>. <abbr class="cit-jnl-abbrev">Mol Ecol</abbr> <span class="cit-vol">19</span>:<span class="cit-fpage">4708</span>–<span class="cit-lpage">4722</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2BEcology%2B%2528Print%2529%26rft.stitle%253DMolecular%2BEcology%2B%2528Print%2529%26rft.aulast%253DBryja%26rft.auinit1%253DJ.%26rft.volume%253D19%26rft.issue%253D21%26rft.spage%253D4708%26rft.epage%253D4722%26rft.atitle%253DRange-wide%2Bpopulation%2Bgenetic%2Bstructure%2Bof%2Bthe%2BEuropean%2Bbitterling%2B%2528Rhodeus%2Bamarus%2529%2Bbased%2Bon%2Bmicrosatellite%2Band%2Bmitochondrial%2BDNA%2Banalysis.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1365-294X.2010.04844.x%26rft_id%253Dinfo%253Apmid%252F20958813%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1365-294X.2010.04844.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20958813&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2016%2F02%2F17%2F039818.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000283163700011&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference  in text" id="ref-10">↵</a><div class="cit ref-cit ref-journal" id="cit-039818v1.10"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Chakraborty</span>  <span class="cit-name-given-names">D</span></span>, <span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">T</span></span>, <span class="cit-auth"><span class="cit-name-surname">Andre</span>  <span class="cit-name-given-names">K</span></span>, <span class="cit-auth"><span class="cit-name-surname">Konnert</span>  <span class="cit-name-given-names">M</span></span>, <span class="cit-auth"><span class="cit-name-surname">Lexer</span>  <span class="cit-name-given-names">MJ</span></span>, <span class="cit-auth"><span class="cit-name-surname">Matulla</span>  <span class="cit-name-given-names">C</span></span>, <span class="cit-auth"><span class="cit-name-surname">Schueler</span>  <span class="cit-name-given-names">S</span></span> (<span class="cit-pub-date">2015</span>) <span class="cit-article-title">Selecting Populations for Non-Analogous Climate Conditions Using Universal Response Functions: The Case of Douglas-Fir in Central Europe</span>. <abbr class="cit-jnl-abbrev">PloS One</abbr>, <span class="cit-vol">10</span>(<span class="cit-issue">8</span>):<span class="cit-fpage">e0136357</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPloS%2BOne%26rft.volume%253D10%26rft.spage%253De0136357%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference  in text" id="ref-11">↵</a><div class="cit ref-cit ref-journal" id="cit-039818v1.11" data-doi="10.1139/x2012-092"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Darychuk</span>  <span class="cit-name-given-names">N</span></span>, <span class="cit-auth"><span class="cit-name-surname">Hawkins</span>  <span class="cit-name-given-names">BJ</span></span>, <span class="cit-auth"><span class="cit-name-surname">Stoehr</span>  <span class="cit-name-given-names">M</span></span> (<span class="cit-pub-date">2012</span>) <span class="cit-article-title">Trade-offs between growth and cold and drought hardiness in submaritime Douglas-fir</span>. <abbr class="cit-jnl-abbrev">Can J For Res</abbr> <span class="cit-vol">42</span>:<span class="cit-fpage">1530</span>–<span class="cit-lpage">1541</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCan%2BJ%2BFor%2BRes%26rft.volume%253D42%26rft.spage%253D1530%26rft_id%253Dinfo%253Adoi%252F10.1139%252Fx2012-092%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1139/x2012-092&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference  in text" id="ref-12">↵</a><div class="cit ref-cit ref-journal" id="cit-039818v1.12"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Duplantier</span>  <span class="cit-name-given-names">JM</span></span>, <span class="cit-auth"><span class="cit-name-surname">Granjon</span>  <span class="cit-name-given-names">L</span></span>, <span class="cit-auth"><span class="cit-name-surname">Mathieu</span>  <span class="cit-name-given-names">E</span></span>, <span class="cit-auth"><span class="cit-name-surname">Bonhomme</span>  <span class="cit-name-given-names">F</span></span> (<span class="cit-pub-date">1990</span>) <span class="cit-article-title">Structures génétiques comparées de trois espèces de rongeurs africains du genre Mastomys au Sénégal</span>. <abbr class="cit-jnl-abbrev">Genetica</abbr> <span class="cit-vol">81</span>:<span class="cit-fpage">179</span>–<span class="cit-lpage">192</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetica%26rft.volume%253D81%26rft.spage%253D179%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference  in text" id="ref-13">↵</a><div class="cit ref-cit ref-journal" id="cit-039818v1.13" data-doi="10.1007/s12686-011-9548-7"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Earl</span>  <span class="cit-name-given-names">DA</span></span>, <span class="cit-auth"><span class="cit-name-surname">vonHoldt</span>  <span class="cit-name-given-names">BM</span></span> (<span class="cit-pub-date">2012</span>) <span class="cit-article-title">STRUCTURE HARVESTER: a website and program for visualizing STRUCTURE output and implementing the Evanno method</span>. <abbr class="cit-jnl-abbrev">Conserv Genet Resour</abbr> <span class="cit-vol">4</span>:<span class="cit-fpage">359</span>–<span class="cit-lpage">361</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DConserv%2BGenet%2BResour%26rft.volume%253D4%26rft.spage%253D359%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs12686-011-9548-7%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s12686-011-9548-7&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference  in text" id="ref-14">↵</a><div class="cit ref-cit ref-journal" id="cit-039818v1.14" data-doi="10.1534/genetics.109.102350"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Eckert</span>  <span class="cit-name-given-names">AJ</span></span>, <span class="cit-auth"><span class="cit-name-surname">Bower</span>  <span class="cit-name-given-names">AD</span></span>, <span class="cit-auth"><span class="cit-name-surname">Wegrzyn</span>  <span class="cit-name-given-names">JL</span></span>, <span class="cit-auth"><span class="cit-name-surname">Pande</span>  <span class="cit-name-given-names">B</span></span>, <span class="cit-auth"><span class="cit-name-surname">Jermstad</span>  <span class="cit-name-given-names">KD</span></span>, <span class="cit-auth"><span class="cit-name-surname">Krutovsky</span>  <span class="cit-name-given-names">KV</span></span>, <span class="cit-auth"><span class="cit-name-surname">St. Clair</span>  <span class="cit-name-given-names">JB</span></span>, <span class="cit-auth"><span class="cit-name-surname">Neale</span>  <span class="cit-name-given-names">DB</span></span> (<span class="cit-pub-date">2009</span>) <span class="cit-article-title">Association genetics of coastal Douglas fir (Pseudotsuga menziesii var. menziesii, Pinaceae). I. Cold-hardiness related traits</span>. <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">182</span>:<span class="cit-fpage">1289</span>–<span class="cit-lpage">1302</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.109.102350%26rft_id%253Dinfo%253Apmid%252F19487566%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE4Mi80LzEyODkiO3M6NDoiYXRvbSI7czozNzoiL2Jpb3J4aXYvZWFybHkvMjAxNi8wMi8xNy8wMzk4MTguYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference  in text" id="ref-15">↵</a><div class="cit ref-cit ref-journal" id="cit-039818v1.15"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Ehring</span>  <span class="cit-name-given-names">A</span></span>, <span class="cit-auth"><span class="cit-name-surname">Klädtke</span>  <span class="cit-name-given-names">J</span></span>, <span class="cit-auth"><span class="cit-name-surname">Yue</span>  <span class="cit-name-given-names">C</span></span> (<span class="cit-pub-date">1999</span>) <span class="cit-article-title">Ein interaktives Programm zur Erstellung von Bestandeshöhenkurven</span>. <abbr class="cit-jnl-abbrev">Centralblatt für das gesamte Forstwesen</abbr> <span class="cit-vol">116</span>:<span class="cit-fpage">47</span>–<span class="cit-lpage">52</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCentralblatt%2Bf%25FCr%2Bdas%2Bgesamte%2BForstwesen%26rft.volume%253D116%26rft.spage%253D47%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference  in text" id="ref-16">↵</a><div class="cit ref-cit ref-journal" id="cit-039818v1.16" data-doi="10.1093/treephys/tps004"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Eilmann</span>  <span class="cit-name-given-names">B</span></span>, <span class="cit-auth"><span class="cit-name-surname">Rigling</span>  <span class="cit-name-given-names">A</span></span> (<span class="cit-pub-date">2012</span>) <span class="cit-article-title">Tree-growth analyses to estimate tree species’ drought tolerance</span>. <abbr class="cit-jnl-abbrev">Tree Phys</abbr> <span class="cit-vol">32</span>:<span class="cit-fpage">178</span>–<span class="cit-lpage">187</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTree%2BPhys%26rft_id%253Dinfo%253Adoi%252F10.1093%252Ftreephys%252Ftps004%26rft_id%253Dinfo%253Apmid%252F22363071%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/treephys/tps004&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22363071&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2016%2F02%2F17%2F039818.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000300988800008&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference  in text" id="ref-17">↵</a><div class="cit ref-cit ref-journal" id="cit-039818v1.17"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Eilmann</span>  <span class="cit-name-given-names">B</span></span>, <span class="cit-auth"><span class="cit-name-surname">de Vries</span>  <span class="cit-name-given-names">SMG</span></span>, <span class="cit-auth"><span class="cit-name-surname">den Ouden</span>  <span class="cit-name-given-names">J</span></span>, <span class="cit-auth"><span class="cit-name-surname">Mohren</span>  <span class="cit-name-given-names">GMJ</span></span>, <span class="cit-auth"><span class="cit-name-surname">Sauren</span>  <span class="cit-name-given-names">P</span></span>, <span class="cit-auth"><span class="cit-name-surname">Sass-Klaasen</span>  <span class="cit-name-given-names">U</span></span> (<span class="cit-pub-date">2013</span>) <span class="cit-article-title">Origin matters! Difference in drought tolerance and productivity of coastal Douglas-fir (Pseudotsuga menziesii (Mirb.)) provenances</span>. <abbr class="cit-jnl-abbrev">Forest Ecol Manag</abbr> <span class="cit-vol">302</span>:<span class="cit-fpage">133</span>–<span class="cit-lpage">143</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DForest%2BEcol%2BManag%26rft.volume%253D302%26rft.spage%253D133%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference  in text" id="ref-18">↵</a><div class="cit ref-cit ref-journal" id="cit-039818v1.18" data-doi="10.1111/j.1365-294X.2005.02553.x"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Evanno</span>  <span class="cit-name-given-names">G</span></span>, <span class="cit-auth"><span class="cit-name-surname">Regnaut</span>  <span class="cit-name-given-names">S</span></span>, <span class="cit-auth"><span class="cit-name-surname">Goudet</span>  <span class="cit-name-given-names">J</span></span> (<span class="cit-pub-date">2005</span>) <span class="cit-article-title">Detecting the number of clusters of individuals using the software STRUCTURE: a simulation study</span>. <abbr class="cit-jnl-abbrev">Mol Ecol</abbr> <span class="cit-vol">14</span>:<span class="cit-fpage">2611</span>–<span class="cit-lpage">2620</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2BEcology%2B%2528Print%2529%26rft.stitle%253DMolecular%2BEcology%2B%2528Print%2529%26rft.aulast%253DEvanno%26rft.auinit1%253DG.%26rft.volume%253D14%26rft.issue%253D8%26rft.spage%253D2611%26rft.epage%253D2620%26rft.atitle%253DDetecting%2Bthe%2Bnumber%2Bof%2Bclusters%2Bof%2Bindividuals%2Busing%2Bthe%2Bsoftware%2BSTRUCTURE%253A%2Ba%2Bsimulation%2Bstudy.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1365-294X.2005.02553.x%26rft_id%253Dinfo%253Apmid%252F15969739%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1365-294X.2005.02553.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15969739&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2016%2F02%2F17%2F039818.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000229961500029&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference  in text" id="ref-19">↵</a><div class="cit ref-cit ref-journal" id="cit-039818v1.19" data-doi="10.1534/genetics.106.061127"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">González-Martínez</span>  <span class="cit-name-given-names">SC</span></span>, <span class="cit-auth"><span class="cit-name-surname">Wheeler</span>  <span class="cit-name-given-names">NC</span></span>, <span class="cit-auth"><span class="cit-name-surname">Ersoz</span>  <span class="cit-name-given-names">E</span></span>, <span class="cit-auth"><span class="cit-name-surname">Nelson</span>  <span class="cit-name-given-names">CD</span></span>, <span class="cit-auth"><span class="cit-name-surname">Neale</span>  <span class="cit-name-given-names">DB</span></span> (<span class="cit-pub-date">2007</span>) <span class="cit-article-title">Association genetics in Pinus taeda L. I. Wood property traits</span>. <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">175</span>:<span class="cit-fpage">399</span>–<span class="cit-lpage">409</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.106.061127%26rft_id%253Dinfo%253Apmid%252F17110498%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTc1LzEvMzk5IjtzOjQ6ImF0b20iO3M6Mzc6Ii9iaW9yeGl2L2Vhcmx5LzIwMTYvMDIvMTcvMDM5ODE4LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference  in text" id="ref-20">↵</a><div class="cit ref-cit ref-web" id="cit-039818v1.20"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Goudet</span>  <span class="cit-name-given-names">J</span></span> (<span class="cit-pub-date">2001</span>) <span class="cit-article-title">FSTAT (version 2.9.3.2): FSTAT, a program to estimate and test gene diversities and fixation indices (version 2.9.3.2)</span>. Available at <a href="http://www2.unil.ch/popgen/softwares/fstat.html">http://www2.unil.ch/popgen/softwares/fstat.html</a>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference  in text" id="ref-21">↵</a><div class="cit ref-cit ref-journal" id="cit-039818v1.21"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Goudet</span>  <span class="cit-name-given-names">J</span></span>, <span class="cit-auth"><span class="cit-name-surname">Raymond</span>  <span class="cit-name-given-names">M</span></span>, <span class="cit-auth"><span class="cit-name-surname">de Meeüs</span>  <span class="cit-name-given-names">T</span></span>, <span class="cit-auth"><span class="cit-name-surname">Rousset</span>  <span class="cit-name-given-names">F</span></span> (<span class="cit-pub-date">1996</span>) <span class="cit-article-title">Testing Differentiation in Diploid Populations</span>. <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">144</span>:<span class="cit-fpage">1933</span>–<span class="cit-lpage">1940</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DGoudet%26rft.auinit1%253DJ.%26rft.volume%253D144%26rft.issue%253D4%26rft.spage%253D1933%26rft.epage%253D1940%26rft.atitle%253DTesting%2BDifferentiation%2Bin%2BDiploid%2BPopulations%26rft_id%253Dinfo%253Apmid%252F8978076%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE0NC80LzE5MzMiO3M6NDoiYXRvbSI7czozNzoiL2Jpb3J4aXYvZWFybHkvMjAxNi8wMi8xNy8wMzk4MTguYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference  in text" id="ref-22">↵</a><div class="cit ref-cit ref-journal" id="cit-039818v1.22" data-doi="10.1093/treephys/tps106"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Gould</span>  <span class="cit-name-given-names">PJ</span></span>, <span class="cit-auth"><span class="cit-name-surname">Harrington</span>  <span class="cit-name-given-names">CA</span></span>, <span class="cit-auth"><span class="cit-name-surname">St. Clair</span>  <span class="cit-name-given-names">JB</span></span> (<span class="cit-pub-date">2012</span>) <span class="cit-article-title">Growth phenology of coast Douglas-fir seed sources planted in diverse environments</span>. <abbr class="cit-jnl-abbrev">Tree Phys</abbr> <span class="cit-vol">32</span>:<span class="cit-fpage">1482</span>–<span class="cit-lpage">1496</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTree%2BPhys%26rft_id%253Dinfo%253Adoi%252F10.1093%252Ftreephys%252Ftps106%26rft_id%253Dinfo%253Apmid%252F23135739%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/treephys/tps106&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23135739&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2016%2F02%2F17%2F039818.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference  in text" id="ref-23">↵</a><div class="cit ref-cit ref-journal" id="cit-039818v1.23" data-doi="10.1111/j.1365-294X.2007.03498.x"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Grivet</span>  <span class="cit-name-given-names">D</span></span>, <span class="cit-auth"><span class="cit-name-surname">Sork</span>  <span class="cit-name-given-names">VL</span></span>, <span class="cit-auth"><span class="cit-name-surname">Westfall</span>  <span class="cit-name-given-names">RD</span></span>, <span class="cit-auth"><span class="cit-name-surname">Davis</span>  <span class="cit-name-given-names">FW</span></span> (<span class="cit-pub-date">2008</span>) <span class="cit-article-title">Conserving the evolutionary potential of California valley oak (Quercus lobata Née): A multivariate genetic approach to conservation planning</span>. <abbr class="cit-jnl-abbrev">Mol Ecol</abbr> <span class="cit-vol">17</span>:<span class="cit-fpage">139</span>–<span class="cit-lpage">156</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2BEcology%2B%2528Print%2529%26rft.stitle%253DMolecular%2BEcology%2B%2528Print%2529%26rft.aulast%253DGrivet%26rft.auinit1%253DD.%26rft.volume%253D17%26rft.issue%253D1%26rft.spage%253D139%26rft.epage%253D156%26rft.atitle%253DConserving%2Bthe%2Bevolutionary%2Bpotential%2Bof%2BCalifornia%2Bvalley%2Boak%2B%2528Quercus%2Blobata%2BNee%2529%253A%2Ba%2Bmultivariate%2Bgenetic%2Bapproach%2Bto%2Bconservation%2Bplanning.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1365-294X.2007.03498.x%26rft_id%253Dinfo%253Apmid%252F17868293%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1365-294X.2007.03498.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17868293&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2016%2F02%2F17%2F039818.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference  in text" id="ref-24">↵</a><div class="cit ref-cit ref-journal" id="cit-039818v1.24"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Gugger</span>  <span class="cit-name-given-names">PF</span></span>, <span class="cit-auth"><span class="cit-name-surname">Sugita</span>  <span class="cit-name-given-names">S</span></span> (<span class="cit-pub-date">2010</span>) <span class="cit-article-title">Glacial populations and postglacial migration of Douglas-fir based on fossil pollen and macrofossil evidence</span>. <abbr class="cit-jnl-abbrev">Quaternary Sci Rev</abbr> <span class="cit-vol">29</span>:<span class="cit-fpage">2052</span>–<span class="cit-lpage">2070</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DQuaternary%2BSci%2BRev%26rft.volume%253D29%26rft.spage%253D2052%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference  in text" id="ref-25">↵</a><div class="cit ref-cit ref-journal" id="cit-039818v1.25" data-doi="10.1111/j.1365-294X.2010.04622.x"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Gugger</span>  <span class="cit-name-given-names">PF</span></span>, <span class="cit-auth"><span class="cit-name-surname">Sugita</span>  <span class="cit-name-given-names">S</span></span>, <span class="cit-auth"><span class="cit-name-surname">Cavender-Bares</span>  <span class="cit-name-given-names">J</span></span> (<span class="cit-pub-date">2010</span>) <span class="cit-article-title">Phylogeography of Douglas-fir based on mitochondrial and chloroplast DNA sequences: testing hypotheses from the fossil record</span>. <abbr class="cit-jnl-abbrev">Mol Ecol</abbr> <span class="cit-vol">19</span>:<span class="cit-fpage">1877</span>–<span class="cit-lpage">1897</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2BEcology%2B%2528Print%2529%26rft.stitle%253DMolecular%2BEcology%2B%2528Print%2529%26rft.aulast%253DGugger%26rft.auinit1%253DP.%2BF.%26rft.volume%253D19%26rft.issue%253D9%26rft.spage%253D1877%26rft.epage%253D1897%26rft.atitle%253DPhylogeography%2Bof%2BDouglas-fir%2Bbased%2Bon%2Bmitochondrial%2Band%2Bchloroplast%2BDNA%2Bsequences%253A%2Btesting%2Bhypotheses%2Bfrom%2Bthe%2Bfossil%2Brecord.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1365-294X.2010.04622.x%26rft_id%253Dinfo%253Apmid%252F20374486%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1365-294X.2010.04622.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20374486&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2016%2F02%2F17%2F039818.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000276794300011&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference  in text" id="ref-26">↵</a><div class="cit ref-cit ref-journal" id="cit-039818v1.26"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Guichoux</span>  <span class="cit-name-given-names">E</span></span>, <span class="cit-auth"><span class="cit-name-surname">Lagache</span>  <span class="cit-name-given-names">L</span></span>, <span class="cit-auth"><span class="cit-name-surname">Wagner</span>  <span class="cit-name-given-names">S</span></span>, <span class="cit-auth"><span class="cit-name-surname">Chaumiel</span>  <span class="cit-name-given-names">P</span></span>, <span class="cit-auth"><span class="cit-name-surname">Léger</span>  <span class="cit-name-given-names">P</span></span>, <span class="cit-auth"><span class="cit-name-surname">Lepais</span>  <span class="cit-name-given-names">O</span></span>, <span class="cit-auth"><span class="cit-name-surname">Lepoittevin</span>  <span class="cit-name-given-names">C</span></span>, <span class="cit-auth"><span class="cit-name-surname">Malausa</span>  <span class="cit-name-given-names">T</span></span>, <span class="cit-auth"><span class="cit-name-surname">Revardel</span>  <span class="cit-name-given-names">E</span></span>, <span class="cit-auth"><span class="cit-name-surname">Salin</span>  <span class="cit-name-given-names">F</span></span>, <span class="cit-auth"><span class="cit-name-surname">Petit</span>  <span class="cit-name-given-names">J</span></span> (<span class="cit-pub-date">2011</span>) <span class="cit-article-title">Current trends in microsatellite genotyping</span>. <abbr class="cit-jnl-abbrev">Mol Ecol Res</abbr> <span class="cit-vol">11</span>:<span class="cit-fpage">591</span>–<span class="cit-lpage">611</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol%2BEcol%2BRes%26rft.volume%253D11%26rft.spage%253D591%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-27-1" title="View reference  in text" id="ref-27">↵</a><div class="cit ref-cit ref-book" id="cit-039818v1.27"><div class="cit-metadata"><ol class="duplicate"><li><span class="cit-ed"><span class="cit-name-surname">Burns</span>  <span class="cit-name-given-names">R M</span></span>, </li><li><span class="cit-ed"><span class="cit-name-surname">Barbara</span>  <span class="cit-name-given-names">H</span></span>, </li><li><span class="cit-ed"><span class="cit-name-surname">Honkala</span>  <span class="cit-name-given-names">BH</span></span></li></ol><cite><span class="cit-auth"><span class="cit-name-surname">Hermann</span>  <span class="cit-name-given-names">RK</span></span>, <span class="cit-auth"><span class="cit-name-surname">Lavender</span>  <span class="cit-name-given-names">DP</span></span> (<span class="cit-pub-date">1990</span>) <span class="cit-chapter-title">Pseudotsuga menziesii (Mirb.) Franco</span>. In: <span class="cit-ed"><span class="cit-name-surname">Burns</span>  <span class="cit-name-given-names">R M</span></span>, <span class="cit-ed"><span class="cit-name-surname">Barbara</span>  <span class="cit-name-given-names">H</span></span>, <span class="cit-ed"><span class="cit-name-surname">Honkala</span>  <span class="cit-name-given-names">BH</span></span> (ed) <span class="cit-source">Silvics of North America: 1. Conifers, USDA Forest Service Agriculture Handbook 654</span>, <span class="cit-publ-loc">Washington, D.C</span>., pp <span class="cit-fpage">527</span>–<span class="cit-lpage">540</span></cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-28-1" title="View reference  in text" id="ref-28">↵</a><div class="cit ref-cit ref-journal" id="cit-039818v1.28"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Hermann</span>  <span class="cit-name-given-names">RK</span></span>, <span class="cit-auth"><span class="cit-name-surname">Lavender</span>  <span class="cit-name-given-names">DP</span></span> (<span class="cit-pub-date">1999</span>) <span class="cit-article-title">Douglas-fir planted forests</span>. <abbr class="cit-jnl-abbrev">New Forests</abbr> <span class="cit-vol">17</span>:<span class="cit-fpage">53</span>–<span class="cit-lpage">70</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNew%2BForests%26rft.volume%253D17%26rft.spage%253D53%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-29-1" title="View reference  in text" id="ref-29">↵</a><div class="cit ref-cit ref-journal" id="cit-039818v1.29" data-doi="10.1111/j.1755-0998.2009.02591.x"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Hubisz</span>  <span class="cit-name-given-names">MJ</span></span>, <span class="cit-auth"><span class="cit-name-surname">Falush</span>  <span class="cit-name-given-names">D</span></span>, <span class="cit-auth"><span class="cit-name-surname">Stephens</span>  <span class="cit-name-given-names">M</span></span>, <span class="cit-auth"><span class="cit-name-surname">Pritchard</span>  <span class="cit-name-given-names">JK</span></span> (<span class="cit-pub-date">2009</span>) <span class="cit-article-title">Inferring weak population structure with the assistance of sample group information</span>. <abbr class="cit-jnl-abbrev">Mol Ecol Res</abbr> <span class="cit-vol">9</span>:<span class="cit-fpage">1322</span>–<span class="cit-lpage">1332</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol%2BEcol%2BRes%26rft.volume%253D9%26rft.spage%253D1322%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1755-0998.2009.02591.x%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1755-0998.2009.02591.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-30-1" title="View reference  in text" id="ref-30">↵</a><div class="cit ref-cit ref-journal" id="cit-039818v1.30" data-doi="10.1093/bioinformatics/btm233"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Jakobsson</span>  <span class="cit-name-given-names">M</span></span>, <span class="cit-auth"><span class="cit-name-surname">Rosenberg</span>  <span class="cit-name-given-names">NA</span></span> (<span class="cit-pub-date">2007</span>) <span class="cit-article-title">CLUMPP: A cluster matching and permutation program for dealing with label switching and multimodality in analysis of population structure</span>. <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">23</span>:<span class="cit-fpage">1801</span>–<span class="cit-lpage">1806</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtm233%26rft_id%253Dinfo%253Apmid%252F17485429%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/btm233&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17485429&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2016%2F02%2F17%2F039818.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000249248300012&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-31-1" title="View reference  in text" id="ref-31">↵</a><div class="cit ref-cit ref-journal" id="cit-039818v1.31" data-doi="10.1007/s00468-012-0765-9"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Jansen</span>  <span class="cit-name-given-names">K</span></span>, <span class="cit-auth"><span class="cit-name-surname">Sohrt</span>  <span class="cit-name-given-names">J</span></span>, <span class="cit-auth"><span class="cit-name-surname">Kohnle</span>  <span class="cit-name-given-names">U</span></span>, <span class="cit-auth"><span class="cit-name-surname">Ensminger</span>  <span class="cit-name-given-names">I</span></span>, <span class="cit-auth"><span class="cit-name-surname">Gessler</span>  <span class="cit-name-given-names">A</span></span> (<span class="cit-pub-date">2013</span>) <span class="cit-article-title">Tree ring isotopic composition, radial increment and height growth reveal provenance-specific reactions of Douglas-fir towards environmental parameters</span>. <abbr class="cit-jnl-abbrev">Trees</abbr> <span class="cit-vol">27</span>:<span class="cit-fpage">37</span>–<span class="cit-lpage">52</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTrees%26rft.volume%253D27%26rft.spage%253D37%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00468-012-0765-9%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00468-012-0765-9&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-32-1" title="View reference  in text" id="ref-32">↵</a><div class="cit ref-cit ref-journal" id="cit-039818v1.32"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Kenk</span>  <span class="cit-name-given-names">G</span></span>, <span class="cit-auth"><span class="cit-name-surname">Thren</span>  <span class="cit-name-given-names">M</span></span> (<span class="cit-pub-date">1984</span>) <span class="cit-article-title">Ergebnisse verschiedener Douglasienprovenienzversuche in Baden-Württemberg. Teil I: Der Internationale Douglasien-Provenienzversuch 1958</span>. <abbr class="cit-jnl-abbrev">Allg Forst Jagdztg</abbr> <span class="cit-vol">155</span>:<span class="cit-fpage">165</span>–<span class="cit-lpage">184</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAllg%2BForst%2BJagdztg%26rft.volume%253D155%26rft.spage%253D165%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-33-1" title="View reference  in text" id="ref-33">↵</a><div class="cit ref-cit ref-journal" id="cit-039818v1.33"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Kohnle</span>  <span class="cit-name-given-names">U</span></span>, <span class="cit-auth"><span class="cit-name-surname">Hein</span>  <span class="cit-name-given-names">S</span></span>, <span class="cit-auth"><span class="cit-name-surname">Sorensen</span>  <span class="cit-name-given-names">FC</span></span>, <span class="cit-auth"><span class="cit-name-surname">Weiskittel</span>  <span class="cit-name-given-names">AR</span></span> (<span class="cit-pub-date">2012</span>) <span class="cit-article-title">Effects of seed source origin on bark thickness of Douglas-fir (Pseudotsuga menziesii) growing in south-western Germany</span>. <abbr class="cit-jnl-abbrev">Can J For Res</abbr> <span class="cit-vol">42</span>:<span class="cit-fpage">382</span>–<span class="cit-lpage">399</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCan%2BJ%2BFor%2BRes%26rft.volume%253D42%26rft.spage%253D382%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-34-1" title="View reference  in text" id="ref-34">↵</a><div class="cit ref-cit ref-journal" id="cit-039818v1.34"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Konnert</span>  <span class="cit-name-given-names">M</span></span>, <span class="cit-auth"><span class="cit-name-surname">Ruetz</span>  <span class="cit-name-given-names">W</span></span>, <span class="cit-auth"><span class="cit-name-surname">Schirmer</span>  <span class="cit-name-given-names">R</span></span> (<span class="cit-pub-date">2008</span>) <span class="cit-article-title">Fragen zum forstlichen Vermehrungsgut bei Douglasie</span>. <abbr class="cit-jnl-abbrev">LWF-Wissen</abbr> <span class="cit-vol">59</span>:<span class="cit-fpage">22</span>–<span class="cit-lpage">26</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DLWF-Wissen%26rft.volume%253D59%26rft.spage%253D22%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-35-1" title="View reference  in text" id="ref-35">↵</a><div class="cit ref-cit ref-journal" id="cit-039818v1.35"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Krakowski</span>  <span class="cit-name-given-names">J</span></span>, <span class="cit-auth"><span class="cit-name-surname">Stoehr</span>  <span class="cit-name-given-names">MU</span></span> (<span class="cit-pub-date">2009</span>) <span class="cit-article-title">Coastal Douglas-fir provenance variation: patterns and predictions for British Columbia seed transfer</span>. <abbr class="cit-jnl-abbrev">Ann For Sci</abbr> <span class="cit-vol">66</span>:<span class="cit-fpage">811</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAnn%2BFor%2BSci%26rft.volume%253D66%26rft.spage%253D811%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-36-1" title="View reference  in text" id="ref-36">↵</a><div class="cit ref-cit ref-journal" id="cit-039818v1.36"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Kremer</span>  <span class="cit-name-given-names">A</span></span>, <span class="cit-auth"><span class="cit-name-surname">Potts</span>  <span class="cit-name-given-names">BM</span></span>, <span class="cit-auth"><span class="cit-name-surname">Delzon</span>  <span class="cit-name-given-names">S</span></span> (<span class="cit-pub-date">2014</span>) <span class="cit-article-title">Genetic divergence in forest trees: understanding the consequences of climate change</span>. <abbr class="cit-jnl-abbrev">Funct Ecol</abbr> <span class="cit-vol">28</span>:<span class="cit-fpage">22</span>–<span class="cit-lpage">36</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DFunct%2BEcol%26rft.volume%253D28%26rft.spage%253D22%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-37-1" title="View reference  in text" id="ref-37">↵</a><div class="cit ref-cit ref-journal" id="cit-039818v1.37" data-doi="10.1534/genetics.105.044420"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Krutovsky</span>  <span class="cit-name-given-names">KV</span></span>, <span class="cit-auth"><span class="cit-name-surname">Neale</span>  <span class="cit-name-given-names">DB</span></span> (<span class="cit-pub-date">2005</span>) <span class="cit-article-title">Nucleotide Diversity and Linkage Disequilibrium in Cold-Hardiness- and Wood Quality-Related Candidate Genes in Douglas Fir</span>. <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">171</span>:<span class="cit-fpage">2029</span>–<span class="cit-lpage">2041</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.105.044420%26rft_id%253Dinfo%253Apmid%252F16157674%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE3MS80LzIwMjkiO3M6NDoiYXRvbSI7czozNzoiL2Jpb3J4aXYvZWFybHkvMjAxNi8wMi8xNy8wMzk4MTguYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-38-1" title="View reference  in text" id="ref-38">↵</a><div class="cit ref-cit ref-journal" id="cit-039818v1.38" data-doi="10.1007/s11295-009-0216-y"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Krutovsky</span>  <span class="cit-name-given-names">KV</span></span>, <span class="cit-auth"><span class="cit-name-surname">St. Clair</span>  <span class="cit-name-given-names">JB</span></span>, <span class="cit-auth"><span class="cit-name-surname">Saich</span>  <span class="cit-name-given-names">R</span></span>, <span class="cit-auth"><span class="cit-name-surname">Hipkins</span>  <span class="cit-name-given-names">VD</span></span>, <span class="cit-auth"><span class="cit-name-surname">Neale</span>  <span class="cit-name-given-names">DB</span></span> (<span class="cit-pub-date">2009</span>) <span class="cit-article-title">Estimation of population structure in coastal Douglas-fir [Pseudotsuga menziesii (Mirb.) Franco var. menziesii] using allozyme and microsatellite markers</span>. <abbr class="cit-jnl-abbrev">Tree Genet Genomes</abbr> <span class="cit-vol">5</span>:<span class="cit-fpage">641</span>–<span class="cit-lpage">658</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTree%2BGenet%2BGenomes%26rft.volume%253D5%26rft.spage%253D641%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs11295-009-0216-y%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s11295-009-0216-y&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000269436200009&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-39-1" title="View reference  in text" id="ref-39">↵</a><div class="cit ref-cit ref-journal" id="cit-039818v1.39"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Leites</span>  <span class="cit-name-given-names">LP</span></span>, <span class="cit-auth"><span class="cit-name-surname">Robinson</span>  <span class="cit-name-given-names">AP</span></span>, <span class="cit-auth"><span class="cit-name-surname">Rehfeldt</span>  <span class="cit-name-given-names">GE</span></span>, <span class="cit-auth"><span class="cit-name-surname">Marshall</span>  <span class="cit-name-given-names">JD</span></span>, <span class="cit-auth"><span class="cit-name-surname">Crookston</span>  <span class="cit-name-given-names">NL</span></span> (<span class="cit-pub-date">2012</span>) <span class="cit-article-title">Height-growth response to climatic changes differs among populations of Douglas-fir: a novel analysis of historic data</span>. <abbr class="cit-jnl-abbrev">Ecol Appl</abbr> <span class="cit-vol">22</span>:<span class="cit-fpage">154</span>–<span class="cit-lpage">165</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEcol%2BAppl%26rft.volume%253D22%26rft.spage%253D154%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-40-1" title="View reference  in text" id="ref-40">↵</a><div class="cit ref-cit ref-journal" id="cit-039818v1.40" data-doi="10.1139/x89-022"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">P</span></span>, <span class="cit-auth"><span class="cit-name-surname">Adams</span>  <span class="cit-name-given-names">WT</span></span> (<span class="cit-pub-date">1989</span>) <span class="cit-article-title">Range-wide patterns of allozyme variation in Douglas-fir (Pseudotsuga menziesii)</span>. <abbr class="cit-jnl-abbrev">Can J For Res</abbr> <span class="cit-vol">19</span>:<span class="cit-fpage">149</span>–<span class="cit-lpage">161</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCan%2BJ%2BFor%2BRes%26rft.volume%253D19%26rft.spage%253D149%26rft_id%253Dinfo%253Adoi%252F10.1139%252Fx89-022%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1139/x89-022&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-41-1" title="View reference  in text" id="ref-41">↵</a><div class="cit ref-cit ref-journal" id="cit-039818v1.41"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Little</span>  <span class="cit-name-given-names">EL</span></span> (<span class="cit-pub-date">1971</span>) <span class="cit-article-title">Atlas of United States trees, volume 1, conifers and important hardwoods: U.S. Department of Agriculture Miscellaneous Publication</span> <span class="cit-fpage">1146</span></cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-42-1" title="View reference  in text" id="ref-42">↵</a><div class="cit ref-cit ref-journal" id="cit-039818v1.42"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Martin</span>  <span class="cit-name-given-names">A</span></span>, <span class="cit-auth"><span class="cit-name-surname">Herrera</span>  <span class="cit-name-given-names">MA</span></span>, <span class="cit-auth"><span class="cit-name-surname">Martin</span>  <span class="cit-name-given-names">LM</span></span> (<span class="cit-pub-date">2012</span>) <span class="cit-article-title">In situ conservation and landscape genetics in forest species</span>. <abbr class="cit-jnl-abbrev">J Nat Resour Develop</abbr> <span class="cit-vol">2</span>:<span class="cit-fpage">1</span>–<span class="cit-lpage">5</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ%2BNat%2BResour%2BDevelop%26rft.volume%253D2%26rft.spage%253D1%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-43-1" title="View reference  in text" id="ref-43">↵</a><div class="cit ref-cit ref-journal" id="cit-039818v1.43" data-doi="10.1093/bioinformatics/btp303"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Matschiner</span>  <span class="cit-name-given-names">M</span></span>, <span class="cit-auth"><span class="cit-name-surname">Salzburger</span>  <span class="cit-name-given-names">W</span></span> (<span class="cit-pub-date">2009</span>) <span class="cit-article-title">TANDEM: integrating automated allele binning into genetics and genomics workflows</span>. <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">25</span>:<span class="cit-fpage">1982</span>–<span class="cit-lpage">1983</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtp303%26rft_id%253Dinfo%253Apmid%252F19420055%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/btp303&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19420055&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2016%2F02%2F17%2F039818.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000268107100027&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-44-1" title="View reference  in text" id="ref-44">↵</a><div class="cit ref-cit ref-journal" id="cit-039818v1.44" data-doi="10.1371/journal.pone.0014691"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Messaoud</span>  <span class="cit-name-given-names">Y</span></span>, <span class="cit-auth"><span class="cit-name-surname">Chen</span>  <span class="cit-name-given-names">HYH</span></span> (<span class="cit-pub-date">2011</span>) <span class="cit-article-title">The influence of recent climate change on tree height growth differs with species and spatial environment</span>. <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">6</span>(<span class="cit-issue">2</span>):<span class="cit-fpage">e14691</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">101371/journal.pone.0014691</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BONE%26rft.aulast%253DMessaoud%26rft.auinit1%253DY.%26rft.volume%253D6%26rft.issue%253D2%26rft.spage%253De14691%26rft.epage%253De14691%26rft.atitle%253DThe%2Binfluence%2Bof%2Brecent%2Bclimate%2Bchange%2Bon%2Btree%2Bheight%2Bgrowth%2Bdiffers%2Bwith%2Bspecies%2Band%2Bspatial%2Benvironment.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0014691%26rft_id%253Dinfo%253Apmid%252F21358817%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0014691&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21358817&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2016%2F02%2F17%2F039818.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-45-1" title="View reference  in text" id="ref-45">↵</a><div class="cit ref-cit ref-journal" id="cit-039818v1.45"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Mosca</span>  <span class="cit-name-given-names">E</span></span>, <span class="cit-auth"><span class="cit-name-surname">González-Martínez</span>  <span class="cit-name-given-names">SC</span></span>, <span class="cit-auth"><span class="cit-name-surname">Neale</span>  <span class="cit-name-given-names">DB</span></span> (<span class="cit-pub-date">2014</span>) <span class="cit-article-title">Environmental versus geographical determinants of genetic structure in two subalpine conifers</span>. <abbr class="cit-jnl-abbrev">New Phytol</abbr> <span class="cit-vol">201</span>:<span class="cit-fpage">180</span>–<span class="cit-lpage">192</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNew%2BPhytol%26rft.volume%253D201%26rft.spage%253D180%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-46-1" title="View reference  in text" id="ref-46">↵</a><div class="cit ref-cit ref-journal" id="cit-039818v1.46" data-doi="10.1186/1471-2164-13-673"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Müller</span>  <span class="cit-name-given-names">T</span></span>, <span class="cit-auth"><span class="cit-name-surname">Ensminger</span>  <span class="cit-name-given-names">I</span></span>, <span class="cit-auth"><span class="cit-name-surname">Schmid</span>  <span class="cit-name-given-names">KJ</span></span> (<span class="cit-pub-date">2012</span>) <span class="cit-article-title">A catalogue of putative unique transcripts from Douglas-fir (Pseudotsuga menziesii) based on 454 transcriptome sequencing of genetically diverse, drought stressed seedlings</span>. <abbr class="cit-jnl-abbrev">BMC Genomics</abbr> <span class="cit-vol">13</span>:<span class="cit-fpage">673</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">doi: </span><span class="cit-pub-id cit-pub-id-doi">10.1186/1471-2164-13-673</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2Bgenomics%2B%255Belectronic%2Bresource%255D%26rft.stitle%253DBMC%2BGenomics%26rft.aulast%253DMuller%26rft.auinit1%253DT.%26rft.volume%253D13%26rft.issue%253D1%26rft.spage%253D673%26rft.epage%253D673%26rft.atitle%253DA%2Bcatalogue%2Bof%2Bputative%2Bunique%2Btranscripts%2Bfrom%2BDouglas-fir%2B%2528Pseudotsuga%2Bmenziesii%2529%2Bbased%2Bon%2B454%2Btranscriptome%2Bsequencing%2Bof%2Bgenetically%2Bdiverse%252C%2Bdrought%2Bstressed%2Bseedlings.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2164-13-673%26rft_id%253Dinfo%253Apmid%252F23190494%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2164-13-673&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23190494&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2016%2F02%2F17%2F039818.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-47-1" title="View reference  in text" id="ref-47">↵</a><div class="cit ref-cit ref-journal" id="cit-039818v1.47"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Müller</span>  <span class="cit-name-given-names">T</span></span>, <span class="cit-auth"><span class="cit-name-surname">Freund</span>  <span class="cit-name-given-names">F</span></span>, <span class="cit-auth"><span class="cit-name-surname">Wildhagen</span>  <span class="cit-name-given-names">H</span></span>, <span class="cit-auth"><span class="cit-name-surname">Schmid</span>  <span class="cit-name-given-names">KJ</span></span> (<span class="cit-pub-date">2015</span>) <span class="cit-article-title">Targeted re-sequencing of five Douglas-fir provenances reveals population structure and putative target genes of positive selection</span>. <abbr class="cit-jnl-abbrev">Tree Genet Genomes</abbr> <span class="cit-vol">11</span>:<span class="cit-fpage">816</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTree%2BGenet%2BGenomes%26rft.volume%253D11%26rft.spage%253D816%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-48-1" title="View reference  in text" id="ref-48">↵</a><div class="cit ref-cit ref-journal" id="cit-039818v1.48" data-doi="10.1038/nrg2931"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Neale</span>  <span class="cit-name-given-names">DB</span></span>, <span class="cit-auth"><span class="cit-name-surname">Kremer</span>  <span class="cit-name-given-names">A</span></span> (<span class="cit-pub-date">2011</span>) <span class="cit-article-title">Forest tree genomics: growing resources and applications</span>. <abbr class="cit-jnl-abbrev">Nat Rev Gen</abbr> <span class="cit-vol">12</span>:<span class="cit-fpage">111</span>–<span class="cit-lpage">122</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Breviews.%2BGenetics%26rft.stitle%253DNat%2BRev%2BGenet%26rft.aulast%253DNeale%26rft.auinit1%253DD.%2BB.%26rft.volume%253D12%26rft.issue%253D2%26rft.spage%253D111%26rft.epage%253D122%26rft.atitle%253DForest%2Btree%2Bgenomics%253A%2Bgrowing%2Bresources%2Band%2Bapplications.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnrg2931%26rft_id%253Dinfo%253Apmid%252F21245829%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nrg2931&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21245829&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2016%2F02%2F17%2F039818.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-49-1" title="View reference  in text" id="ref-49">↵</a><div class="cit ref-cit ref-journal" id="cit-039818v1.49" data-doi="10.1086/282771"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Nei</span>  <span class="cit-name-given-names">M</span></span> (<span class="cit-pub-date">1972</span>) <span class="cit-article-title">Genetic distance between populations</span>. <abbr class="cit-jnl-abbrev">Am Nat</abbr> <span class="cit-vol">106</span>:<span class="cit-fpage">283</span>–<span class="cit-lpage">292</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAm%2BNat%26rft.volume%253D106%26rft.spage%253D283%26rft_id%253Dinfo%253Adoi%252F10.1086%252F282771%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1086/282771&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=A1972M475000002&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-50-1" title="View reference  in text" id="ref-50">↵</a><div class="cit ref-cit ref-journal" id="cit-039818v1.50" data-doi="10.1111/j.1558-5646.2007.00299.x"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Nosil</span>  <span class="cit-name-given-names">P</span></span>, <span class="cit-auth"><span class="cit-name-surname">Egan</span>  <span class="cit-name-given-names">SP</span></span>, <span class="cit-auth"><span class="cit-name-surname">Funk</span>  <span class="cit-name-given-names">DJ</span></span> (<span class="cit-pub-date">2008</span>) <span class="cit-article-title">Heterogeneous genomic differentiation between walking-stick ecotypes: “Isolation by adaptation” and multiple roles for divergent selection</span>. <abbr class="cit-jnl-abbrev">Evolution</abbr> <span class="cit-vol">62</span>:<span class="cit-fpage">316</span>–<span class="cit-lpage">336</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEvolution%26rft.stitle%253DEvolution%26rft.aulast%253DNosil%26rft.auinit1%253DP.%26rft.volume%253D62%26rft.issue%253D2%26rft.spage%253D316%26rft.epage%253D336%26rft.atitle%253DHeterogeneous%2Bgenomic%2Bdifferentiation%2Bbetween%2Bwalking-stick%2Becotypes%253A%2B%2526quot%253Bisolation%2Bby%2Badaptation%2526quot%253B%2Band%2Bmultiple%2Broles%2Bfor%2Bdivergent%2Bselection.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1558-5646.2007.00299.x%26rft_id%253Dinfo%253Apmid%252F17999721%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1558-5646.2007.00299.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17999721&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2016%2F02%2F17%2F039818.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000253373900006&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-51-1" title="View reference  in text" id="ref-51">↵</a><div class="cit ref-cit ref-journal" id="cit-039818v1.51" data-doi="10.1111/j.1365-294X.2008.03946.x"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Nosil</span>  <span class="cit-name-given-names">P</span></span>, <span class="cit-auth"><span class="cit-name-surname">Funk</span>  <span class="cit-name-given-names">DJ</span></span>, <span class="cit-auth"><span class="cit-name-surname">Ortiz-Barrientos</span>  <span class="cit-name-given-names">D</span></span> (<span class="cit-pub-date">2009</span>) <span class="cit-article-title">Divergent selection and heterogeneous genomic divergence</span>. <abbr class="cit-jnl-abbrev">Mol Ecol</abbr> <span class="cit-vol">18</span>:<span class="cit-fpage">375</span>–<span class="cit-lpage">402</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2BEcology%2B%2528Print%2529%26rft.stitle%253DMolecular%2BEcology%2B%2528Print%2529%26rft.aulast%253DNosil%26rft.auinit1%253DP.%26rft.volume%253D18%26rft.issue%253D3%26rft.spage%253D375%26rft.epage%253D402%26rft.atitle%253DDivergent%2Bselection%2Band%2Bheterogeneous%2Bgenomic%2Bdivergence.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1365-294X.2008.03946.x%26rft_id%253Dinfo%253Apmid%252F19143936%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1365-294X.2008.03946.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19143936&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2016%2F02%2F17%2F039818.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000262517400001&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-52-1" title="View reference  in text" id="ref-52">↵</a><div class="cit ref-cit ref-journal" id="cit-039818v1.52" data-doi="10.1016/0168-9452(96)04356-7"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">O’Brien</span>  <span class="cit-name-given-names">IEW</span></span>, <span class="cit-auth"><span class="cit-name-surname">Smith</span>  <span class="cit-name-given-names">DR</span></span>, <span class="cit-auth"><span class="cit-name-surname">Gardner</span>  <span class="cit-name-given-names">RC</span></span>, <span class="cit-auth"><span class="cit-name-surname">Murray</span>  <span class="cit-name-given-names">BG</span></span> (<span class="cit-pub-date">1996</span>) <span class="cit-article-title">Flow cytometric determination of genome size in Pinus</span>. <abbr class="cit-jnl-abbrev">Plant Sci</abbr> <span class="cit-vol">115</span>:<span class="cit-fpage">91</span>–<span class="cit-lpage">99</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BSci%26rft.volume%253D115%26rft.spage%253D91%26rft_id%253Dinfo%253Adoi%252F10.1016%252F0168-9452%252896%252904356-7%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/0168-9452(96)04356-7&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-53-1" title="View reference  in text" id="ref-53">↵</a><div class="cit ref-cit ref-web" id="cit-039818v1.53"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Oksanen</span>  <span class="cit-name-given-names">J</span></span>, <span class="cit-auth"><span class="cit-name-surname">Guillaume</span>  <span class="cit-name-given-names">Blanchet F</span></span>, <span class="cit-auth"><span class="cit-name-surname">Kindt</span>  <span class="cit-name-given-names">R Legendre</span></span>, <span class="cit-auth"><span class="cit-name-surname">P Minchin</span>  <span class="cit-name-given-names">PR</span></span>, <span class="cit-auth"><span class="cit-name-surname">O’Hara</span>  <span class="cit-name-given-names">RB</span></span>, <span class="cit-auth"><span class="cit-name-surname">Simpson</span>  <span class="cit-name-given-names">GL</span></span>, <span class="cit-auth"><span class="cit-name-surname">Solymos</span>  <span class="cit-name-given-names">P</span></span>, <span class="cit-auth"><span class="cit-name-surname">Henry</span>  <span class="cit-name-given-names">H</span></span>, <span class="cit-auth"><span class="cit-name-surname">Stevens</span>  <span class="cit-name-given-names">H</span></span>, <span class="cit-auth"><span class="cit-name-surname">Wagner</span>  <span class="cit-name-given-names">H</span></span> (<span class="cit-pub-date">2013</span>) <span class="cit-article-title">Community Ecology Package</span>. <span class="cit-source">R package version 2.0-10</span>. <a href="http://CRAN.R-project.org/package=vegan">http://CRAN.R-project.org/package=vegan</a>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-54-1" title="View reference  in text" id="ref-54">↵</a><div class="cit ref-cit ref-journal" id="cit-039818v1.54" data-doi="10.1016/0378-1127(89)90004-2"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Rehfeldt</span>  <span class="cit-name-given-names">GE</span></span> (<span class="cit-pub-date">1989</span>) <span class="cit-article-title">Ecological adaptations in Douglas-fir (Pseudotsuga menziesii var. glauca): a synthesis</span>. <abbr class="cit-jnl-abbrev">Forest Ecol Manag</abbr> <span class="cit-vol">28</span>:<span class="cit-fpage">203</span>–<span class="cit-lpage">215</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DForest%2BEcol%2BManag%26rft.volume%253D28%26rft.spage%253D203%26rft_id%253Dinfo%253Adoi%252F10.1016%252F0378-1127%252889%252990004-2%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/0378-1127(89)90004-2&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-55-1" title="View reference  in text" id="ref-55">↵</a><div class="cit ref-cit ref-journal" id="cit-039818v1.55" data-doi="10.1111/j.1471-8286.2004.00684.x"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Van Oosterhout</span>  <span class="cit-name-given-names">C</span></span>, <span class="cit-auth"><span class="cit-name-surname">Hutchinson</span>  <span class="cit-name-given-names">WF</span></span>, <span class="cit-auth"><span class="cit-name-surname">Wills</span>  <span class="cit-name-given-names">DPM</span></span>, <span class="cit-auth"><span class="cit-name-surname">Shipley</span>  <span class="cit-name-given-names">P</span></span> (<span class="cit-pub-date">2004</span>) <span class="cit-article-title">Micro-Checker: Software for identifying and correcting genotyping errors in microsatellite data</span>. <abbr class="cit-jnl-abbrev">Mol Ecol Notes</abbr> <span class="cit-vol">4</span>:<span class="cit-fpage">535</span>–<span class="cit-lpage">538</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol%2BEcol%2BNotes%26rft.volume%253D4%26rft.spage%253D535%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1471-8286.2004.00684.x%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1471-8286.2004.00684.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000223649300069&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-56-1" title="View reference  in text" id="ref-56">↵</a><div class="cit ref-cit ref-journal" id="cit-039818v1.56"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Pinheiro</span>  <span class="cit-name-given-names">J</span></span>, <span class="cit-auth"><span class="cit-name-surname">Bates</span>  <span class="cit-name-given-names">D</span></span>, <span class="cit-auth"><span class="cit-name-surname">DebRoy</span>  <span class="cit-name-given-names">S</span></span>, <span class="cit-auth"><span class="cit-name-surname">Sarkar</span>  <span class="cit-name-given-names">D</span></span>, and <span class="cit-article-title">the R Development Core Team</span> (<span class="cit-pub-date">2012</span>). <abbr class="cit-jnl-abbrev">nlme: Linear and Nonlinear Mixed Effects Models</abbr>. R package version <span class="cit-vol">3.1</span>-<span class="cit-fpage">103</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253Dnlme%253A%2BLinear%2Band%2BNonlinear%2BMixed%2BEffects%2BModels%26rft.volume%253D31%26rft.spage%253D103%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-57-1" title="View reference  in text" id="ref-57">↵</a><div class="cit ref-cit ref-journal" id="cit-039818v1.57" data-doi="10.1111/j.1471-8286.2005.01155.x"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Pritchard</span>  <span class="cit-name-given-names">JK</span></span>, <span class="cit-auth"><span class="cit-name-surname">Stephens</span>  <span class="cit-name-given-names">M</span></span>, <span class="cit-auth"><span class="cit-name-surname">Donnelly</span>  <span class="cit-name-given-names">P</span></span> (<span class="cit-pub-date">2000</span>) <span class="cit-article-title">Inference of population structure using multilocus genotype data</span>. <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">155</span>:<span class="cit-fpage">945</span>–<span class="cit-lpage">959</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DPritchard%26rft.auinit1%253DJ.%2BK.%26rft.volume%253D155%26rft.issue%253D2%26rft.spage%253D945%26rft.epage%253D959%26rft.atitle%253DInference%2Bof%2BPopulation%2BStructure%2BUsing%2BMultilocus%2BGenotype%2BData%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1471-8286.2005.01155.x%26rft_id%253Dinfo%253Apmid%252F10835412%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTU1LzIvOTQ1IjtzOjQ6ImF0b20iO3M6Mzc6Ii9iaW9yeGl2L2Vhcmx5LzIwMTYvMDIvMTcvMDM5ODE4LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-58-1" title="View reference  in text" id="ref-58">↵</a><div class="cit ref-cit ref-journal" id="cit-039818v1.58" data-doi="10.1016/j.cub.2009.11.055"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Pritchard</span>  <span class="cit-name-given-names">JK</span></span>, <span class="cit-auth"><span class="cit-name-surname">Pickrell</span>  <span class="cit-name-given-names">JK</span></span>, <span class="cit-auth"><span class="cit-name-surname">Coop</span>  <span class="cit-name-given-names">G</span></span> (<span class="cit-pub-date">2010</span>) <span class="cit-article-title">The Genetics of Human Adaptation: Hard Sweeps, Soft Sweeps and Polygenic Adaptation</span>. <abbr class="cit-jnl-abbrev">Curr Biol</abbr> <span class="cit-vol">20</span>:<span class="cit-fpage">R208</span>–<span class="cit-lpage">R215</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">doi: </span><span class="cit-pub-id cit-pub-id-doi">10.1016/j.cub.2009.11.055</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCurrent%2Bbiology%2B%253A%2B%2BCB%26rft.stitle%253DCurr%2BBiol%26rft.aulast%253DPritchard%26rft.auinit1%253DJ.%2BK.%26rft.volume%253D20%26rft.issue%253D4%26rft.spage%253DR208%26rft.epage%253DR215%26rft.atitle%253DThe%2Bgenetics%2Bof%2Bhuman%2Badaptation%253A%2Bhard%2Bsweeps%252C%2Bsoft%2Bsweeps%252C%2Band%2Bpolygenic%2Badaptation.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.cub.2009.11.055%26rft_id%253Dinfo%253Apmid%252F20178769%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.cub.2009.11.055&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20178769&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2016%2F02%2F17%2F039818.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000275149200026&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-59-1" title="View reference  in text" id="ref-59">↵</a><div class="cit ref-cit ref-book" id="cit-039818v1.59"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Puettmann</span>  <span class="cit-name-given-names">K</span></span>, <span class="cit-auth"><span class="cit-name-surname">Coates</span>  <span class="cit-name-given-names">KD</span></span>, <span class="cit-auth"><span class="cit-name-surname">Messier</span>  <span class="cit-name-given-names">C</span></span> (<span class="cit-pub-date">2009</span>) <span class="cit-chapter-title">A critique of silviculture: Managing for complexity</span>. <span class="cit-publ-name">Island Press</span>, <span class="cit-publ-loc">Washington D. C</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-60-1" title="View reference  in text" id="ref-60">↵</a><div class="cit ref-cit ref-web" id="cit-039818v1.60"><div class="cit-metadata"><cite><span class="cit-auth cit-collab">R Development Core Team</span> (<span class="cit-pub-date">2011</span>) <span class="cit-chapter-title">R: A language and environment for statistical computing</span>. <span class="cit-publ-name">R Foundation for Statistical Computing</span>, <span class="cit-publ-loc">Vienna, Austria</span>, <a href="http://www.R-project.org">http://www.R-project.org</a></cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-61-1" title="View reference  in text" id="ref-61">↵</a><div class="cit ref-cit ref-journal" id="cit-039818v1.61" data-doi="10.1007/s10682-010-9365-6"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Ramírez-Valiente</span>  <span class="cit-name-given-names">JA</span></span>, <span class="cit-auth"><span class="cit-name-surname">Lorenzo</span>  <span class="cit-name-given-names">Z</span></span>, <span class="cit-auth"><span class="cit-name-surname">Soto</span>  <span class="cit-name-given-names">A</span></span>, <span class="cit-auth"><span class="cit-name-surname">Valladares</span>  <span class="cit-name-given-names">F</span></span>, <span class="cit-auth"><span class="cit-name-surname">Gil</span>  <span class="cit-name-given-names">L</span></span>, <span class="cit-auth"><span class="cit-name-surname">Aranda</span>  <span class="cit-name-given-names">I</span></span> (<span class="cit-pub-date">2010</span>) <span class="cit-article-title">Natural selection on cork oak: allele frequency reveals divergent selection in cork oak populations along a temperature cline</span>. <abbr class="cit-jnl-abbrev">Evol Ecol</abbr> <span class="cit-vol">24</span>:<span class="cit-fpage">1031</span>–<span class="cit-lpage">1044</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEvol%2BEcol%26rft.volume%253D24%26rft.spage%253D1031%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs10682-010-9365-6%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s10682-010-9365-6&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-62-1" title="View reference  in text" id="ref-62">↵</a><div class="cit ref-cit ref-journal" id="cit-039818v1.62" data-doi="10.1046/j.1471-8286.2003.00566.x"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Rosenberg</span>  <span class="cit-name-given-names">NA</span></span> (<span class="cit-pub-date">2004</span>) <span class="cit-article-title">Distruct: a program for the graphical display of population structure</span>. <abbr class="cit-jnl-abbrev">Mol Ecol Notes</abbr> <span class="cit-vol">4</span>:<span class="cit-fpage">137</span>–<span class="cit-lpage">138</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol%2BEcol%2BNotes%26rft.volume%253D4%26rft.spage%253D137%26rft_id%253Dinfo%253Adoi%252F10.1046%252Fj.1471-8286.2003.00566.x%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1046/j.1471-8286.2003.00566.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000189159500042&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-63-1" title="View reference  in text" id="ref-63">↵</a><div class="cit ref-cit ref-journal" id="cit-039818v1.63" data-doi="10.1534/genetics.104.026849"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Scotti-Saintagne</span>  <span class="cit-name-given-names">C</span></span>, <span class="cit-auth"><span class="cit-name-surname">Mariette</span>  <span class="cit-name-given-names">S</span></span>, <span class="cit-auth"><span class="cit-name-surname">Porth</span>  <span class="cit-name-given-names">I</span></span>, <span class="cit-auth"><span class="cit-name-surname">Goicoechea</span>  <span class="cit-name-given-names">PG</span></span>, <span class="cit-auth"><span class="cit-name-surname">Barreneche</span>  <span class="cit-name-given-names">T</span></span>, <span class="cit-auth"><span class="cit-name-surname">Bodénès</span>  <span class="cit-name-given-names">C</span></span>, <span class="cit-auth"><span class="cit-name-surname">Burg</span>  <span class="cit-name-given-names">K</span></span>, <span class="cit-auth"><span class="cit-name-surname">Kremer</span>  <span class="cit-name-given-names">A</span></span> (<span class="cit-pub-date">2004</span>) <span class="cit-article-title">Genome scanning for interspecific differentiation between two closely related oak species [Quercus robur L. and Q. petraea (Matt.) Liebl.]</span>. <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">168</span>:<span class="cit-fpage">1615</span>–<span class="cit-lpage">1626</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DScotti-Saintagne%26rft.auinit1%253DC.%26rft.volume%253D168%26rft.issue%253D3%26rft.spage%253D1615%26rft.epage%253D1626%26rft.atitle%253DGenome%2BScanning%2Bfor%2BInterspecific%2BDifferentiation%2BBetween%2BTwo%2BClosely%2BRelated%2BOak%2BSpecies%2B%255BQuercus%2Brobur%2BL.%2Band%2BQ.%2Bpetraea%2B%2528Matt.%2529%2BLiebl.%255D%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.104.026849%26rft_id%253Dinfo%253Apmid%252F15579711%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE2OC8zLzE2MTUiO3M6NDoiYXRvbSI7czozNzoiL2Jpb3J4aXYvZWFybHkvMjAxNi8wMi8xNy8wMzk4MTguYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-64-1" title="View reference  in text" id="ref-64">↵</a><div class="cit ref-cit ref-journal" id="cit-039818v1.64" data-doi="10.1139/X10-099"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Schmidt</span>  <span class="cit-name-given-names">M</span></span>, <span class="cit-auth"><span class="cit-name-surname">Hanewinkel</span>  <span class="cit-name-given-names">M</span></span>, <span class="cit-auth"><span class="cit-name-surname">Kändler</span>  <span class="cit-name-given-names">G</span></span>, <span class="cit-auth"><span class="cit-name-surname">Kublin</span>  <span class="cit-name-given-names">E</span></span>, <span class="cit-auth"><span class="cit-name-surname">Kohnle</span>  <span class="cit-name-given-names">U</span></span> (<span class="cit-pub-date">2010</span>) <span class="cit-article-title">An inventory-based approach for modeling single-tree storm damage-experiences with the winter storm of 1999 in south-western Germany</span>. <abbr class="cit-jnl-abbrev">Can J For Res</abbr> <span class="cit-vol">40</span>:<span class="cit-fpage">1636</span>–<span class="cit-lpage">1652</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCan%2BJ%2BFor%2BRes%26rft.volume%253D40%26rft.spage%253D1636%26rft_id%253Dinfo%253Adoi%252F10.1139%252FX10-099%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1139/X10-099&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-65-1" title="View reference  in text" id="ref-65">↵</a><div class="cit ref-cit ref-book" id="cit-039818v1.65"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Schütt</span>  <span class="cit-name-given-names">P</span></span>, <span class="cit-auth"><span class="cit-name-surname">Schuck</span>  <span class="cit-name-given-names">HJ</span></span>, <span class="cit-auth"><span class="cit-name-surname">Stimm</span>  <span class="cit-name-given-names">B</span></span> (<span class="cit-pub-date">2002</span>) <span class="cit-chapter-title">Lexikon der Baum-und Straucharten: Das Standardwerk der Forstbotanik</span>. <span class="cit-publ-name">Nikol Verlagsgesellschaft mbH &amp; Co</span>. <span class="cit-publ-loc">KG, Hamburg</span></cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-66-1" title="View reference  in text" id="ref-66">↵</a><div class="cit ref-cit ref-journal" id="cit-039818v1.66"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Šeho</span>  <span class="cit-name-given-names">M</span></span>, <span class="cit-auth"><span class="cit-name-surname">Kohnle</span>  <span class="cit-name-given-names">U</span></span> (<span class="cit-pub-date">2014</span>) <span class="cit-article-title">Der Internationale Douglasien-Provenienzversuch 1958: Unterschiede in der Ausprägung von Ast-und Stammmerkmalen auf den südwestdeutschen Versuchsflächen</span>. <abbr class="cit-jnl-abbrev">Allg Forst Jagdztg</abbr> <span class="cit-vol">185</span>:<span class="cit-fpage">27</span>–<span class="cit-lpage">42</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAllg%2BForst%2BJagdztg%26rft.volume%253D185%26rft.spage%253D27%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-67-1" title="View reference  in text" id="ref-67">↵</a><div class="cit ref-cit ref-journal" id="cit-039818v1.67" data-doi="10.1111/j.1461-0248.2006.00889.x"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Selkoe</span>  <span class="cit-name-given-names">K A</span></span>, <span class="cit-auth"><span class="cit-name-surname">Toonen</span>  <span class="cit-name-given-names">RJ</span></span> (<span class="cit-pub-date">2006</span>) <span class="cit-article-title">Microsatellites for ecologists: a practical guide to using and evaluating microsatellite markers</span>. <abbr class="cit-jnl-abbrev">Ecol Lett</abbr> <span class="cit-vol">9</span>:<span class="cit-fpage">615</span>–<span class="cit-lpage">629</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DEcol%2BLett%26rft.aulast%253DSelkoe%26rft.auinit1%253DK.%2BA.%26rft.volume%253D9%26rft.issue%253D5%26rft.spage%253D615%26rft.epage%253D629%26rft.atitle%253DMicrosatellites%2Bfor%2Becologists%253A%2Ba%2Bpractical%2Bguide%2Bto%2Busing%2Band%2Bevaluating%2Bmicrosatellite%2Bmarkers.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1461-0248.2006.00889.x%26rft_id%253Dinfo%253Apmid%252F16643306%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1461-0248.2006.00889.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16643306&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2016%2F02%2F17%2F039818.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000236799600011&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-68-1" title="View reference  in text" id="ref-68">↵</a><div class="cit ref-cit ref-journal" id="cit-039818v1.68" data-doi="10.1111/j.1095-8312.1987.tb00437.x"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">She</span>  <span class="cit-name-given-names">JX</span></span>, <span class="cit-auth"><span class="cit-name-surname">Autem</span>  <span class="cit-name-given-names">M</span></span>, <span class="cit-auth"><span class="cit-name-surname">Kotulas</span>  <span class="cit-name-given-names">G</span></span>, <span class="cit-auth"><span class="cit-name-surname">Pasteur</span>  <span class="cit-name-given-names">N</span></span>, <span class="cit-auth"><span class="cit-name-surname">Bonhomme</span>  <span class="cit-name-given-names">F</span></span> (<span class="cit-pub-date">1987</span>) <span class="cit-article-title">Multivariate analysis of genetic exchanges between Solea aegyptiaca and Solea senegalensis (Teleosts, Soleidae)</span>. <abbr class="cit-jnl-abbrev">Biol J Linn Soc</abbr> <span class="cit-vol">32</span>:<span class="cit-fpage">357</span>–<span class="cit-lpage">371</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBiol%2BJ%2BLinn%2BSoc%26rft.volume%253D32%26rft.spage%253D357%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1095-8312.1987.tb00437.x%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1095-8312.1987.tb00437.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-69-1" title="View reference  in text" id="ref-69">↵</a><div class="cit ref-cit ref-journal" id="cit-039818v1.69" data-doi="10.1007/s00122-003-1490-y"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Slavov</span>  <span class="cit-name-given-names">GT</span></span>, <span class="cit-auth"><span class="cit-name-surname">Howe</span>  <span class="cit-name-given-names">GT</span></span>, <span class="cit-auth"><span class="cit-name-surname">Yakovlev</span>  <span class="cit-name-given-names">I</span></span>, <span class="cit-auth"><span class="cit-name-surname">Edwards</span>  <span class="cit-name-given-names">KJ</span></span>, <span class="cit-auth"><span class="cit-name-surname">Krutovsky</span>  <span class="cit-name-given-names">KV</span></span>, <span class="cit-auth"><span class="cit-name-surname">Tuskan</span>  <span class="cit-name-given-names">GA</span></span>, <span class="cit-auth"><span class="cit-name-surname">Carlson</span>  <span class="cit-name-given-names">JE</span></span>, <span class="cit-auth"><span class="cit-name-surname">Strauss</span>  <span class="cit-name-given-names">SH</span></span>, <span class="cit-auth"><span class="cit-name-surname">Adams</span>  <span class="cit-name-given-names">WT</span></span> (<span class="cit-pub-date">2004</span>). <span class="cit-article-title">Highly variable SSR markers in Douglas-fir: Mendelian inheritance and map locations</span>. <abbr class="cit-jnl-abbrev">Theor Appl Genet</abbr> <span class="cit-vol">108</span>:<span class="cit-fpage">873</span>–<span class="cit-lpage">880</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTAG.%2BTheoretical%2Band%2Bapplied%2Bgenetics.%2B%2BTheoretische%2Bund%2Bangewandte%2BGenetik%26rft.stitle%253DTheor%2BAppl%2BGenet%26rft.aulast%253DSlavov%26rft.auinit1%253DG.%2BT.%26rft.volume%253D108%26rft.issue%253D5%26rft.spage%253D873%26rft.epage%253D880%26rft.atitle%253DHighly%2Bvariable%2BSSR%2Bmarkers%2Bin%2BDouglas-fir%253A%2BMendelian%2Binheritance%2Band%2Bmap%2Blocations.%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00122-003-1490-y%26rft_id%253Dinfo%253Apmid%252F14625671%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00122-003-1490-y&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=14625671&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2016%2F02%2F17%2F039818.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000220424400013&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-70-1" title="View reference  in text" id="ref-70">↵</a><div class="cit ref-cit ref-journal" id="cit-039818v1.70" data-doi="10.1111/j.1365-294X.2010.04726.x"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Sork</span>  <span class="cit-name-given-names">VL</span></span>, <span class="cit-auth"><span class="cit-name-surname">Davis</span>  <span class="cit-name-given-names">FW</span></span>, <span class="cit-auth"><span class="cit-name-surname">Westfall</span>  <span class="cit-name-given-names">R</span></span>, <span class="cit-auth"><span class="cit-name-surname">Flint</span>  <span class="cit-name-given-names">A</span></span>, <span class="cit-auth"><span class="cit-name-surname">Ikegami</span>  <span class="cit-name-given-names">M</span></span>, <span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">H</span></span>, <span class="cit-auth"><span class="cit-name-surname">Grivet</span>  <span class="cit-name-given-names">D</span></span> (<span class="cit-pub-date">2010</span>) <span class="cit-article-title">Gene movement and genetic association with regional climate gradients in California valley oak (Quercus lobata Née) in the face of climate change</span>. <abbr class="cit-jnl-abbrev">Mol Ecol</abbr> <span class="cit-vol">19</span>:<span class="cit-fpage">3806</span>–<span class="cit-lpage">3823</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2BEcology%2B%2528Print%2529%26rft.stitle%253DMolecular%2BEcology%2B%2528Print%2529%26rft.aulast%253DSork%26rft.auinit1%253DV.%2BL.%26rft.volume%253D19%26rft.issue%253D17%26rft.spage%253D3806%26rft.epage%253D3823%26rft.atitle%253DGene%2Bmovement%2Band%2Bgenetic%2Bassociation%2Bwith%2Bregional%2Bclimate%2Bgradients%2Bin%2BCalifornia%2Bvalley%2Boak%2B%2528Quercus%2Blobata%2BNee%2529%2Bin%2Bthe%2Bface%2Bof%2Bclimate%2Bchange.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1365-294X.2010.04726.x%26rft_id%253Dinfo%253Apmid%252F20723054%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1365-294X.2010.04726.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20723054&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2016%2F02%2F17%2F039818.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000281285200022&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-71-1" title="View reference  in text" id="ref-71">↵</a><div class="cit ref-cit ref-book" id="cit-039818v1.71"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Strehlke</span>  <span class="cit-name-given-names">B</span></span> (<span class="cit-pub-date">1959</span>) <span class="cit-chapter-title">Die Ernte von Douglasiensamen in USA und Kanada</span>. <span class="cit-source">Folgerungen für die deutsche Forstwirtschaft</span>. <span class="cit-publ-loc">Der Forst-und Holzwirt</span> <span class="cit-vol">14</span>:<span class="cit-fpage">295</span>–<span class="cit-lpage">300</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DFolgerungen%2Bf%25FCr%2Bdie%2Bdeutsche%2BForstwirtschaft%26rft.volume%253D14%26rft.spage%253D295%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-72-1" title="View reference  in text" id="ref-72">↵</a><div class="cit ref-cit ref-journal" id="cit-039818v1.72" data-doi="10.1093/molbev/msp312"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Takezaki</span>  <span class="cit-name-given-names">N</span></span>, <span class="cit-auth"><span class="cit-name-surname">Nei</span>  <span class="cit-name-given-names">M</span></span>, <span class="cit-auth"><span class="cit-name-surname">Tamura</span>  <span class="cit-name-given-names">K</span></span> (<span class="cit-pub-date">2010</span>) <span class="cit-article-title">POPTREE2: Software for Constructing Population Trees from Allele Frequency Data and Computing Other Population Statistics with Windows Interface</span>. <abbr class="cit-jnl-abbrev">Mol Biol Evol</abbr> <span class="cit-vol">27</span>:<span class="cit-fpage">747</span>–<span class="cit-lpage">752</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol%2BBiol%2BEvol%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fmolbev%252Fmsp312%26rft_id%253Dinfo%253Apmid%252F20022889%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/molbev/msp312&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20022889&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2016%2F02%2F17%2F039818.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000276639800001&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-73-1" title="View reference  in text" id="ref-73">↵</a><div class="cit ref-cit ref-journal" id="cit-039818v1.73" data-doi="10.1371/journal.pone.0042764"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Temunović</span>  <span class="cit-name-given-names">M</span></span>, <span class="cit-auth"><span class="cit-name-surname">Franjić</span>  <span class="cit-name-given-names">J</span></span>, <span class="cit-auth"><span class="cit-name-surname">Satovic</span>  <span class="cit-name-given-names">Z</span></span>, <span class="cit-auth"><span class="cit-name-surname">Grgurev</span>  <span class="cit-name-given-names">M</span></span>, <span class="cit-auth"><span class="cit-name-surname">Frascaria-Lacoste</span>  <span class="cit-name-given-names">N</span></span>, <span class="cit-auth"><span class="cit-name-surname">Fernández-Manjarrés</span>  <span class="cit-name-given-names">JF</span></span> (<span class="cit-pub-date">2012</span>) <span class="cit-article-title">Environmental Heterogeneity Explains the Genetic Structure of Continental and Mediterranean Populations of Fraxinus angustifolia Vahl</span>. <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">7</span>:<span class="cit-fpage">e42764</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">doi: </span><span class="cit-pub-id cit-pub-id-doi">10.1371/journal.pone.0042764</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BONE%26rft.aulast%253DTemunovic%26rft.auinit1%253DM.%26rft.volume%253D7%26rft.issue%253D8%26rft.spage%253De42764%26rft.epage%253De42764%26rft.atitle%253DEnvironmental%2Bheterogeneity%2Bexplains%2Bthe%2Bgenetic%2Bstructure%2Bof%2BContinental%2Band%2BMediterranean%2Bpopulations%2Bof%2BFraxinus%2Bangustifolia%2BVahl.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0042764%26rft_id%253Dinfo%253Apmid%252F22905171%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0042764&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22905171&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2016%2F02%2F17%2F039818.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-74-1" title="View reference  in text" id="ref-74">↵</a><div class="cit ref-cit ref-journal" id="cit-039818v1.74"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Tsukada</span>  <span class="cit-name-given-names">M</span></span> (<span class="cit-pub-date">1982</span>) <span class="cit-article-title">Pseudotsuga menziesii (Mirb.) Franco: Its pollen dispersal and late Quaternary history in the Pacific Northwest [USA]</span>. <abbr class="cit-jnl-abbrev">Japanese J Ecol</abbr> <span class="cit-vol">32</span>:<span class="cit-fpage">159</span>–<span class="cit-lpage">187</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJapanese%2BJ%2BEcol%26rft.volume%253D32%26rft.spage%253D159%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-75-1" title="View reference  in text" id="ref-75">↵</a><div class="cit ref-cit ref-journal" id="cit-039818v1.75" data-doi="10.1038/hdy.2012.50"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Tsumura</span>  <span class="cit-name-given-names">Y</span></span>, <span class="cit-auth"><span class="cit-name-surname">Uchiyama</span>  <span class="cit-name-given-names">K</span></span>, <span class="cit-auth"><span class="cit-name-surname">Moriguchi</span>  <span class="cit-name-given-names">Y</span></span>, <span class="cit-auth"><span class="cit-name-surname">Ueno</span>  <span class="cit-name-given-names">S</span></span>, <span class="cit-auth"><span class="cit-name-surname">Ihara-Ujino</span>  <span class="cit-name-given-names">T</span></span> (<span class="cit-pub-date">2012</span>) <span class="cit-article-title">Genome scanning for detecting adaptive genes along environmental gradients in the Japanese conifer, Cryptomeria japonica</span>. <abbr class="cit-jnl-abbrev">Heredity</abbr> <span class="cit-vol">109</span>:<span class="cit-fpage">349</span>–<span class="cit-lpage">360</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DHeredity%26rft.stitle%253DHeredity%26rft.aulast%253DTsumura%26rft.auinit1%253DY.%26rft.volume%253D109%26rft.issue%253D6%26rft.spage%253D349%26rft.epage%253D360%26rft.atitle%253DGenome%2Bscanning%2Bfor%2Bdetecting%2Badaptive%2Bgenes%2Balong%2Benvironmental%2Bgradients%2Bin%2Bthe%2BJapanese%2Bconifer%252C%2BCryptomeria%2Bjaponica.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fhdy.2012.50%26rft_id%253Dinfo%253Apmid%252F22929151%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/hdy.2012.50&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22929151&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2016%2F02%2F17%2F039818.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000311098300005&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-76-1" title="View reference  in text" id="ref-76">↵</a><div class="cit ref-cit ref-journal" id="cit-039818v1.76"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Vasemägi</span>  <span class="cit-name-given-names">A</span></span>, <span class="cit-auth"><span class="cit-name-surname">Gross</span>  <span class="cit-name-given-names">R</span></span>, <span class="cit-auth"><span class="cit-name-surname">Paaver</span>  <span class="cit-name-given-names">T</span></span>, <span class="cit-auth"><span class="cit-name-surname">Kangur</span>  <span class="cit-name-given-names">M</span></span>, <span class="cit-auth"><span class="cit-name-surname">Nilsson</span>  <span class="cit-name-given-names">J</span></span>, <span class="cit-auth"><span class="cit-name-surname">Eriksson</span>  <span class="cit-name-given-names">LO</span></span> (<span class="cit-pub-date">2001</span>) <span class="cit-article-title">Identification of the origin of an Atlantic salmon (Salmo salar L.) population in a recently recolonized river in the Baltic Sea</span>. <abbr class="cit-jnl-abbrev">Mol Ecol</abbr> <span class="cit-vol">10</span>:<span class="cit-fpage">2877</span>–<span class="cit-lpage">2882</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2BEcology%2B%2528Print%2529%26rft.stitle%253DMolecular%2BEcology%2B%2528Print%2529%26rft.aulast%253DVasem%25C3%25A4gi%26rft.auinit1%253DA.%26rft.volume%253D10%26rft.issue%253D12%26rft.spage%253D2877%26rft.epage%253D2882%26rft.atitle%253DIdentification%2Bof%2Bthe%2Borigin%2Bof%2Ban%2BAtlantic%2Bsalmon%2B%2528Salmo%2Bsalar%2BL.%2529%2Bpopulation%2Bin%2Ba%2Brecently%2Brecolonized%2Briver%2Bin%2Bthe%2BBaltic%2BSea.%26rft_id%253Dinfo%253Apmid%252F11903899%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=11903899&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2016%2F02%2F17%2F039818.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-77-1" title="View reference  in text" id="ref-77">↵</a><div class="cit ref-cit ref-journal" id="cit-039818v1.77" data-doi="10.2307/2408641"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Weir</span>  <span class="cit-name-given-names">BS</span></span>, <span class="cit-auth"><span class="cit-name-surname">Cockerham</span>  <span class="cit-name-given-names">CC</span></span> (<span class="cit-pub-date">1984</span>) <span class="cit-article-title">Estimating F-Statistics for the Analysis of Population Structure</span>. <abbr class="cit-jnl-abbrev">Evolution</abbr> <span class="cit-vol">38</span>:<span class="cit-fpage">1358</span>–<span class="cit-lpage">1370</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEvolution%26rft.volume%253D38%26rft.spage%253D1358%26rft_id%253Dinfo%253Adoi%252F10.2307%252F2408641%26rft_id%253Dinfo%253Apmid%252F28563791%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2307/2408641&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=28563791&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2016%2F02%2F17%2F039818.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1984TY40400017&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-78-1" title="View reference  in text" id="ref-78">↵</a><div class="cit ref-cit ref-book" id="cit-039818v1.78"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Zuur</span>  <span class="cit-name-given-names">A</span></span>, <span class="cit-auth"><span class="cit-name-surname">Ieno</span>  <span class="cit-name-given-names">EN</span></span>, <span class="cit-auth"><span class="cit-name-surname">Walker</span>  <span class="cit-name-given-names">N</span></span>, <span class="cit-auth"><span class="cit-name-surname">Saveliev</span>  <span class="cit-name-given-names">AA</span></span>, <span class="cit-auth"><span class="cit-name-surname">Smith</span>  <span class="cit-name-given-names">GM</span></span> (<span class="cit-pub-date">2007</span>) <span class="cit-source">Mixed Effects Models and Extensions in Ecology with R</span>. <span class="cit-publ-name">Springer</span>, <span class="cit-publ-loc">New York</span></cite></div><div class="cit-extra"></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.biorxiv.org/content/10.1101/039818v1.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-disqus-comment" >
  
      
  
  <div class="pane-content">
    <div id="disqus_thread"><noscript><p><a href="http://biorxivstage.disqus.com/?url=https%3A%2F%2Fwww.biorxiv.org%2Fcontent%2F10.1101%2F039818v1" class="" data-icon-position="" data-hide-link-title="0">View the discussion thread.</a></p></noscript></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/10.1101/039875v1" title="Validation of a habitat suitability index for oyster restoration" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous</span></a></span><span class="pager-next"><a href="/content/10.1101/039339v1" title="Integrating regulatory information via combinatorial control of the transcription cycle" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    Posted&nbsp;February 17, 2016.  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-biorxiv-art-tools" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-biorxiv_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/10.1101/039818v1.full.pdf" target="_self" class="article-dl-pdf-link link-icon"><i class="icon-external-link-sign"></i> <span class="title">Download PDF</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-biorxiv-supplementary-fragment" >
  
      
  
  <div class="pane-content">
    <p><a href="/content/10.1101/039818v1.supplementary-material"><i class="icon-file"></i>Supplementary Material </a></p>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-biorxiv-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="Email this Article" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-biorxiv_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/10.1101/039818v1.full" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for your interest in spreading the word about bioRxiv.</p><p>NOTE: Your email address is requested solely to identify you as the sender of this article.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/10.1101/039818v1" class="active" data-icon-position="" data-hide-link-title="0">Assessing the relationship between height growth and molecular genetic variation in Douglas-fir (Pseudotsuga menziesii) provenances</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from bioRxiv
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would like to see this page from the bioRxiv website.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/10804" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-g4_4eT1YWMUsHjkxrpTgrwE3CxTlqflM3-TzLPBE8L4" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-biorxiv-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-biorxiv_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="10804" id="node10804--2" data-pisa="biorxiv;039818v1" data-pisa-master="biorxiv;039818" data-seqnum="10804" data-apath="/biorxiv/early/2016/02/17/039818.atom"><div  class="highwire-cite highwire-cite-highwire-article highwire-citation-biorxiv-article-pap-list clearfix" >

      <div class="highwire-cite-title" >
      <div class="highwire-cite-title">Assessing the relationship between height growth and molecular genetic variation in Douglas-fir (<em>Pseudotsuga menziesii</em>) provenances</div>    </div>
  
      <div  class="highwire-cite-authors" ><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0"><span class="nlm-given-names">Charalambos</span> <span class="nlm-surname">Neophytou</span></span>, <span class="highwire-citation-author" data-delta="1"><span class="nlm-given-names">Anna-Maria</span> <span class="nlm-surname">Weisser</span></span>, <span class="highwire-citation-author" data-delta="2"><span class="nlm-given-names">Daniel</span> <span class="nlm-surname">Landwehr</span></span>, <span class="highwire-citation-author" data-delta="3"><span class="nlm-given-names">Muhidin</span> <span class="nlm-surname">Šeho</span></span>, <span class="highwire-citation-author" data-delta="4"><span class="nlm-given-names">Ulrich</span> <span class="nlm-surname">Kohnle</span></span>, <span class="highwire-citation-author" data-delta="5"><span class="nlm-given-names">Ingo</span> <span class="nlm-surname">Ensminger</span></span>, <span class="highwire-citation-author" data-delta="6"><span class="nlm-given-names">Henning</span> <span class="nlm-surname">Wildhagen</span></span></span></div>
  
      <div  class="highwire-cite-metadata" ><span  class="highwire-cite-metadata-journal highwire-cite-metadata">bioRxiv </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">039818; </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata"><span class="doi_label">doi:</span> https://doi.org/10.1101/039818 </span></div>
  
  
  </div>
</div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.biorxiv.org/content/10.1101/039818v1" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.biorxiv.org/content/10.1101/039818v1&amp;title=Assessing%20the%20relationship%20between%20height%20growth%20and%20molecular%20genetic%20variation%20in%20Douglas-fir%20%28Pseudotsuga%20menziesii%29%20provenances" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.biorxiv.org/content/10.1101/039818v1&amp;title=Assessing%20the%20relationship%20between%20height%20growth%20and%20molecular%20genetic%20variation%20in%20Douglas-fir%20%28Pseudotsuga%20menziesii%29%20provenances" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.biorxiv.org/content/10.1101/039818v1&amp;text=Assessing%20the%20relationship%20between%20height%20growth%20and%20molecular%20genetic%20variation%20in%20Douglas-fir%20%28Pseudotsuga%20menziesii%29%20provenances" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.biorxiv.org/content/10.1101/039818v1&amp;title=Assessing%20the%20relationship%20between%20height%20growth%20and%20molecular%20genetic%20variation%20in%20Douglas-fir%20%28Pseudotsuga%20menziesii%29%20provenances" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.biorxiv.org/content/10.1101/039818v1&amp;t=Assessing%20the%20relationship%20between%20height%20growth%20and%20molecular%20genetic%20variation%20in%20Douglas-fir%20%28Pseudotsuga%20menziesii%29%20provenances" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.biorxiv.org/content/10.1101/039818v1&amp;title=Assessing%20the%20relationship%20between%20height%20growth%20and%20molecular%20genetic%20variation%20in%20Douglas-fir%20%28Pseudotsuga%20menziesii%29%20provenances" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.biorxiv.org/content/10.1101/039818v1&amp;title=Assessing%20the%20relationship%20between%20height%20growth%20and%20molecular%20genetic%20variation%20in%20Douglas-fir%20%28Pseudotsuga%20menziesii%29%20provenances" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-biorxiv-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title="Citation Tools"><i class="icon-globe"></i> <span class="title">Citation Tools</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-biorxiv_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="10804" id="node10804--3" data-pisa="biorxiv;039818v1" data-pisa-master="biorxiv;039818" data-seqnum="10804" data-apath="/biorxiv/early/2016/02/17/039818.atom"><div  class="highwire-cite highwire-cite-highwire-article highwire-citation-biorxiv-article-pap-list clearfix" >

      <div class="highwire-cite-title" >
      <div class="highwire-cite-title">Assessing the relationship between height growth and molecular genetic variation in Douglas-fir (<em>Pseudotsuga menziesii</em>) provenances</div>    </div>
  
      <div  class="highwire-cite-authors" ><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0"><span class="nlm-given-names">Charalambos</span> <span class="nlm-surname">Neophytou</span></span>, <span class="highwire-citation-author" data-delta="1"><span class="nlm-given-names">Anna-Maria</span> <span class="nlm-surname">Weisser</span></span>, <span class="highwire-citation-author" data-delta="2"><span class="nlm-given-names">Daniel</span> <span class="nlm-surname">Landwehr</span></span>, <span class="highwire-citation-author" data-delta="3"><span class="nlm-given-names">Muhidin</span> <span class="nlm-surname">Šeho</span></span>, <span class="highwire-citation-author" data-delta="4"><span class="nlm-given-names">Ulrich</span> <span class="nlm-surname">Kohnle</span></span>, <span class="highwire-citation-author" data-delta="5"><span class="nlm-given-names">Ingo</span> <span class="nlm-surname">Ensminger</span></span>, <span class="highwire-citation-author" data-delta="6"><span class="nlm-given-names">Henning</span> <span class="nlm-surname">Wildhagen</span></span></span></div>
  
      <div  class="highwire-cite-metadata" ><span  class="highwire-cite-metadata-journal highwire-cite-metadata">bioRxiv </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">039818; </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata"><span class="doi_label">doi:</span> https://doi.org/10.1101/039818 </span></div>
  
  
  </div>
</div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage&amp;rft.epage&amp;rft.atitle=Assessing%20the%20relationship%20between%20height%20growth%20and%20molecular%20genetic%20variation%20in%20Douglas-fir%20%28Pseudotsuga%20menziesii%29%20provenances&amp;rft.volume&amp;rft.issue&amp;rft.date=2016-01-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=bioRxiv&amp;rft.au=Neophytou%2C+Charalambos&amp;rft.au=Weisser%2C+Anna-Maria&amp;rft.au=Landwehr%2C+Daniel&amp;rft.au=%C5%A0eho%2C+Muhidin&amp;rft.au=Kohnle%2C+Ulrich&amp;rft.au=Ensminger%2C+Ingo&amp;rft.au=Wildhagen%2C+Henning"></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/10804/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/10804/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/10804/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/10804/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/10804/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/10804/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/10804/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/10804/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/10804/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/10804/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/10804/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/10804/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><div class="item-list"><ul><li class="first"><a href="http://twitter.com/share?url=https%3A//www.biorxiv.org/content/10.1101/039818v1&amp;count=horizontal&amp;via=&amp;text=Assessing%20the%20relationship%20between%20height%20growth%20and%20molecular%20genetic%20variation%20in%20Douglas-fir%20%28Pseudotsuga%20menziesii%29%20provenances&amp;counturl=https%3A//www.biorxiv.org/content/10.1101/039818v1" class="twitter-share-button service-links-twitter-widget" id="twitter_widget" title="Tweet This" rel="nofollow" data-icon-position="" data-hide-link-title="0"><span class="element-invisible">Tweet Widget</span></a></li><li><a href="http://www.facebook.com/plugins/like.php?href=https%3A//www.biorxiv.org/content/10.1101/039818v1&amp;layout=button_count&amp;show_faces=false&amp;action=like&amp;colorscheme=light&amp;width=100&amp;height=21&amp;font=&amp;locale=" id="facebook_like" title="I Like it" class="service-links-facebook-like" rel="nofollow" data-icon-position="" data-hide-link-title="0"><span class="element-invisible">Facebook Like</span></a></li><li class="last"><a href="https://www.biorxiv.org/content/10.1101/039818v1" id="google_plus_one" title="Plus it" class="service-links-google-plus-one" rel="nofollow" data-icon-position="" data-hide-link-title="0"><span class="element-invisible">Google Plus One</span></a></li></ul></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-collections" >
  
        <h2 class="pane-title">Subject Area</h2>
    
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-collections"><div class="highwire-list"><ul class="highwire-article-collection-term-list"><li class="first last odd"><span class="highwire-article-collection-term"><a href="/collection/plant-biology" class="highlight" data-icon-position="" data-hide-link-title="0">Plant Biology<i class="icon-caret-right"></i>
</a></span></li></ul></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-biorxiv-subject-collections block-style-col2" >
  
      
  
  <div class="pane-content">
    <div class="panel-flexible panels-flexible-8 clearfix" id="mini-panel-biorxiv_subject_collections">
<div class="panel-flexible-inside panels-flexible-8-inside">
<div class="panels-flexible-region panels-flexible-region-8-center panels-flexible-region-first panels-flexible-region-last">
  <div class="inside panels-flexible-region-inside panels-flexible-region-8-center-inside panels-flexible-region-inside-first panels-flexible-region-inside-last">
<div class="panel-pane pane-snippet" >
  
      
  
  <div class="pane-content">
    <div class="snippet biorxiv-subject-areas-table-title" id="biorxiv-subject-areas-table-title">
  
      
  <div class="snippet-content">
    <b>Subject Areas</b>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-snippet" >
  
      
  
  <div class="pane-content">
    <div class="snippet biorxiv-subject-areas-view-papers" id="biorxiv-subject-areas-view-papers">
  
      
  <div class="snippet-content">
    <a href="/content/early/recent"><strong>All Articles</strong></a>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-subject-collections" >
  
      
  
  <div class="pane-content">
    <ul id="collection" class="collection highwire-list-expand"><li class="outer collection depth-2  child first"><div class = "data-wrapper"><a href="/collection/animal-behavior-and-cognition" class="" data-icon-position="" data-hide-link-title="0">Animal Behavior and Cognition</a> <span class = "article-count">(1662)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/biochemistry" class="" data-icon-position="" data-hide-link-title="0">Biochemistry</a> <span class = "article-count">(2789)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/bioengineering" class="" data-icon-position="" data-hide-link-title="0">Bioengineering</a> <span class = "article-count">(1946)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/bioinformatics" class="" data-icon-position="" data-hide-link-title="0">Bioinformatics</a> <span class = "article-count">(10400)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/biophysics" class="" data-icon-position="" data-hide-link-title="0">Biophysics</a> <span class = "article-count">(4243)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/cancer-biology" class="" data-icon-position="" data-hide-link-title="0">Cancer Biology</a> <span class = "article-count">(3272)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/cell-biology" class="" data-icon-position="" data-hide-link-title="0">Cell Biology</a> <span class = "article-count">(4641)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/clinical-trials" class="" data-icon-position="" data-hide-link-title="0">Clinical Trials</a> <span class = "article-count">(135)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/developmental-biology" class="" data-icon-position="" data-hide-link-title="0">Developmental Biology</a> <span class = "article-count">(2889)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/ecology" class="" data-icon-position="" data-hide-link-title="0">Ecology</a> <span class = "article-count">(4544)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/epidemiology" class="" data-icon-position="" data-hide-link-title="0">Epidemiology</a> <span class = "article-count">(2041)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/evolutionary-biology" class="" data-icon-position="" data-hide-link-title="0">Evolutionary Biology</a> <span class = "article-count">(7288)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/genetics" class="" data-icon-position="" data-hide-link-title="0">Genetics</a> <span class = "article-count">(5540)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/genomics" class="" data-icon-position="" data-hide-link-title="0">Genomics</a> <span class = "article-count">(6893)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/immunology" class="" data-icon-position="" data-hide-link-title="0">Immunology</a> <span class = "article-count">(2434)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/microbiology" class="" data-icon-position="" data-hide-link-title="0">Microbiology</a> <span class = "article-count">(7616)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/molecular-biology" class="" data-icon-position="" data-hide-link-title="0">Molecular Biology</a> <span class = "article-count">(3051)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/neuroscience" class="" data-icon-position="" data-hide-link-title="0">Neuroscience</a> <span class = "article-count">(18885)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/paleontology" class="" data-icon-position="" data-hide-link-title="0">Paleontology</a> <span class = "article-count">(142)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/pathology" class="" data-icon-position="" data-hide-link-title="0">Pathology</a> <span class = "article-count">(477)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/pharmacology-and-toxicology" class="" data-icon-position="" data-hide-link-title="0">Pharmacology and Toxicology</a> <span class = "article-count">(791)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/physiology" class="" data-icon-position="" data-hide-link-title="0">Physiology</a> <span class = "article-count">(1178)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/plant-biology" class="" data-icon-position="" data-hide-link-title="0">Plant Biology</a> <span class = "article-count">(2771)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/scientific-communication-and-education" class="" data-icon-position="" data-hide-link-title="0">Scientific Communication and Education</a> <span class = "article-count">(686)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/synthetic-biology" class="" data-icon-position="" data-hide-link-title="0">Synthetic Biology</a> <span class = "article-count">(905)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/systems-biology" class="" data-icon-position="" data-hide-link-title="0">Systems Biology</a> <span class = "article-count">(2881)</span></div></li>
<li class="outer collection depth-2  child last"><div class = "data-wrapper"><a href="/collection/zoology" class="" data-icon-position="" data-hide-link-title="0">Zoology</a> <span class = "article-count">(472)</span></div></li>
</ul>  </div>

  
  </div>
  </div>
</div>
</div>
</div>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__VNH5GD8Zz7g6_hGCOZjVjdMndxxC6naiExSSJKX3k_A__kzmEdtbqCsJx5Z9yg2Qy0PptB__ufXm-TxUOl0KZzUw__QJacVmveFyLIBSPkp21xL0ZvuDW9DMBrD2HB1P4Ry5I.js"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__QJacVmveFyLIBSPkp21xL0ZvuDW9DMBrD2HB1P4Ry5I.js" defer="defer"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__N7ERJBYsOWyRnJgyoM125_Aiez2MOJGaUofG1JdWWBg__bLlcwhzZhT-IwoQ_dvm1_xPxU7E7gecFLC6Fyeqtv8s__QJacVmveFyLIBSPkp21xL0ZvuDW9DMBrD2HB1P4Ry5I.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
function euCookieComplianceLoadScripts() {}
//--><!]]>
</script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var eu_cookie_compliance_cookie_name = "";
//--><!]]>
</script>
<script type="text/javascript" src="//d33xdlntwy0kbs.cloudfront.net/cshl_custom.js"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__BmqjBnkz3MgYCAoc25s1lDRMEjLhC3mEPVonUFIHi08__Unwv5-ZIuHBfFwytsjEx1niBVJ7n1T4lPws7VrkHXM4__QJacVmveFyLIBSPkp21xL0ZvuDW9DMBrD2HB1P4Ry5I.js"></script>
  </body>
</html>
